<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD colspan="3" align="left" valign="center"><FONT size="7" color="#787878"><B style="color: #51a7f9">Labbook</B></FONT></TD></TR><TR><TD colspan="3" align="left"><P class="introduction">

We documented our work chronologically on a daily basis, linked the respective protocols and added all lab results. You can find the entries sorted according to month and sub-project, which can be selected separately. To do actual experiments it is also important to prepare buffer and all kind of stock solutions, agar plates, gels and so on. This basic work is partially not mentioned here, but was also done by the team. To ç
To find what was done in each day, select the month you want to see and then select which category you want to check out.
		</P></TD></TR><TR><TD colspan="3" align="center" valign="center"><DIV class="calendarcontainer"><DIV class="ui grid"><LABEL for="Cas13a">Cas13a</LABEL><LABEL for="Readout">Read-out</LABEL><LABEL for="Target">Target</LABEL><LABEL for="Other">Other</LABEL><DIV class="thirteen wide column" id="labbook-content"><DIV class="month" id="April"><DIV class="date">Tuesday 25th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Plating of DH5α p2CT-His-MBP-Lbu-Cas13a-WT and DH5α p2CT-His-MBP-Lsh-Cas13a-WT on 
						LB<SUB>Carb</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">-</A></LI><LI class="myParticipants">
			    		Participants: Ludwig        

			    	</LI><LI class="myObservations">
			    		Observations:</LI><LI>
			    		Incubation at 37 °C over night.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    Bacteria grew and single colonies were visible. 
			    	</LI><DIV class="date">Wednesday 26th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Preparation of 6 ml LB<SUB>Carb</SUB> cultures of DH5α p2CT-His-MBP-Lbu-Cas13a-WT and DH5α p2CT-His-MBP-Lsh-Cas13a-WT on LB<SUB>Carb</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">-</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Incubation at 37 °C over night.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    Bacteria cultures grew. 
			    	</LI><DIV class="date">Thursday 27th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of cryo stocks for DH5α p2CT-His-MBP-Lbu-Cas13a-WT and 
						DH5α p2CT-His-MBP-Lsh-Cas13a-WT</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cryo stocks</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α Cas13a overnight cultures</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT 
						in <I>E. coli</I> Rosetta.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		1 µl of plasmid DNA was used.
			    			</LI><LI>
						 Bacteria were plated on LB<SUB>Carb</SUB> plates without additional Cm. 
							</LI></UL><LI class="myResults"><H4>Results</H4>
                    Bacteria lost second plasmid probably.  
			    	</LI></DIV><DIV class="month" id="May"><DIV class="date">Tuesday 2nd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT 
						in <I>E. coli</I> Rosetta.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		1 µl of plasmid DNA was used.
			    			</LI><LI>
						 Bacteria were plated on LB<SUB>Carb + Cm</SUB> plates
							</LI></UL><LI class="myResults"><H4>Results</H4>
                    Bacteria grew and single colonies were visible.  
			    	</LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparations of buffers for Cas13a purification.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Protein purification</A></LI><LI class="myParticipants">
			    		Participants: Chris, Ludwig, Julian    

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		TCEP changes the pH. Readjustement after adding neccessary. 
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT 
						in <I>E. coli</I> Rosetta.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		1 µl of plasmid DNA was used.
			    			</LI><LI>
						 Bacteria were plated on LB<SUB>Carb + Cm</SUB> plates
							</LI></UL><LI class="myResults"><H4>Results</H4>
                    Bacteria grew, but only few single colonies were visible.  
			    	</LI><DIV class="date">Wednesday 3rd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT 
						in <I>E. coli</I> Rosetta.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris      

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		1 µl of plasmid DNA was used.
			    			</LI><LI>
						 Bacteria were plated on LB<SUB>Carb + Cm</SUB> plates
							</LI></UL><LI class="myResults"><H4>Results</H4>
                    This time bacteria grew well and a lot of single colonies were visible.   
			    	</LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 25 ml LB<SUB>Carb + Cm</SUB> cultures of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT and Rosetta p2CT-His-MBP-Lsh-Cas13a-WT and incubated at 37 °C overnight.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Protein Expression</A></LI><LI class="myParticipants">
			    		Participants: Ludwig       

			    	</LI></UL><DIV class="date">Thursday 4th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting of Rosetta cells with overexpressed Cas13a Lbu and Cas13a Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Protein expression</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myResults"><H4>Results</H4>
                    Bacterial pellets were stored at -80 °C.
			    	</LI></UL><DIV class="date">Friday 5th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Protein purification of Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		One test sample was run on the Äkta before the actual run. Only the second run was analyzed with a SDS PAGE. 
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Protein purification of Cas13a Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Monday 8th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Transformation of TEV plasmid (form lab in the chemistry department) in DH5α </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro Transformation</A></LI><LI class="myParticipants">
			    		Participants: Sven
			    	</LI></UL><DIV class="date">Tuesday 9th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Dialysis of pooled peak fractions of Cas13a Lbu and Cas13a Lsh into ion exchange buffer</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris, Sven
			    	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>PCR of NT and T crDNA (for Cas13a Lbu and Cas13a Lsh)</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Sven
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
							PCR didn't work. 
			    			</LI></UL><DIV class="date">Wednesday 10th</DIV><UL class="myEntry other"><LI class="myTitel"><B>Test of different lysis methods for total RNA extraction with Norgen kit</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Total RNA Purification with Norgen Kit</A></LI><LI class="myParticipants">
			    		Participants: Julian, Milica, Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Three lysis methods were used: sonifcation, lysozyme digestion and heat lysis in SDS.
			    			</LI><LI>
			    				Lysozyme digestion: According to protocol.
			    			</LI><LI>
			    				Heat lysis: Sample was incubated in 2% SDS at 75 °C for 15 min.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                                                                               See results from &quot;Phenol/Chloroform purification of RNA from in-vitro transcription&quot; from 11.05.17 for gel picture.
			    	</LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Upconcentration fo Cas13a Lbu and Cas13 Lsh after dialysis</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Thursday 11th</DIV><UL class="myEntry target"><LI class="myTitel"><B>Test of different lysis methods for Phenol/Chloroform total RNA extraction.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Phenol/Chloroform purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Patrick, Julian
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Three lysis method were used: sonification, lysozyme digestion and heat lysis in SDS.
			    			</LI><LI>
			    				Lysozyme digestion: according to the total RNA purification with Norgen Kit protocol. 
			    			</LI><LI>
			    				Heat lysis: Sample was incubated in 2% SDS at 75 °C for 15 min.
			    			</LI><LI>
			    				The samples were incubated at -80 °C overnight and the purification finished the next day.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                                         See results from &quot;Phenol/Chloroform purification of RNA from in-vitro transcription&quot; from 11.05.17.
			    	</LI><UL class="myEntry target"><LI class="myTitel"><B>Phenol/Chloroform purification of RNA from in-vitro transcription</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">Phenol/Chloroform purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti, Sven
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>
			    				The samples were incubated at -80 °C overnight and the purification finished the next day.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Friday 12th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Urea PAGE of <I>in vitro</I> transcription samples </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">UREA PAGE</A></LI><LI class="myParticipants">
			    		Participants: Sven
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Monday 15th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation fo buffers for Heparin purification of Cas13a Lbu and Cas13a Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI></UL><DIV class="date">Tuesday 16th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Heparin purification of Cas13a Lbu and Cas13 Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Dawa
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Tuesday 23rd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>PCR of Cas13a Lwa part1 and part2 and backbone pSB1C3_ccdB</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">PCR</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris, Rob
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
						Q5 polymerase was used 
							</LI><LI>
			    		Primers Lwa part1 and part2: VF and VR
			    			</LI><LI>
						Primers pSB1C3_ccdB: p-pSB-GG-Fwd, p-pSB-GG-Rev
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Assembly of Cas13a Lwa part1 and part2 with pSB1C3_ccdB using GoldenGate Cloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">GoldenGate Assembly</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris, Rob
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Insert to vector ratio: 2:1 and 3:1
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro Transformation of assembled Lwa parts and pSB1C3 into DH5α and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro Transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris, Rob
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Next day: Only a few colonies. 
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro Transformation of TEV biobricks (pSB1C3-BBa_K1319008 and pSB1C3-BBa_K1319004) and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro Transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris, Rob
			    	</LI></UL><DIV class="date">Wednesday 24th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Repeat: Assembly of Cas13a Lwa part1 and part2 with pSB1C3_ccdB using GoldenGate Cloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">GoldenGate Assembly</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Dawa, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Insert to vector ratio: 2:1 and 3:1
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro Transformation of assembled Lwa parts and pSB1C3 into DH5α and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro Transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Dawa, Chris
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Repeat: PCR of Cas13a Lwa part1 and part2 and backbone pSB1C3_ccdB</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">PCR</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Dawa, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
						Q5 polymerase was used 
							</LI><LI>
			    		Primers Lwa part1 and part2: VF and VR
			    			</LI><LI>
						Primers pSB1C3_ccdB: p-pSB-GG-Fwd, p-pSB-GG-Rev
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of one picked DH5α pSB1C3-Cas13a-LwaLwa colony (from first GoldenGate assembly)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Dawa, Chris
			    	</LI></UL><DIV class="date">Thursday 25th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Miniprep of DH5α pSB1C3-Cas13a-Lwa overnight culture</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Dawa, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		pSB1C3-Cas13a-Lwa was then used for an analytical restriction digest and was sent for sequencing (with primer VR)
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Analytical restriction digest with pSB1C3-Cas13a-LwaLwa with XbaI (single digest) and XbaI + SpeI (double digest)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Restriction digest</A></LI><LI class="myParticipants">
			    		Participants: Dawa, Chris
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Repeat: PCR of Lwa part2 and also GoldenGate was redone with new amplified Lwa part2 </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">PCR and GoldenGate</A></LI><LI class="myParticipants">
			    		Participants: 
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		PCR Primers: VF, VR
			    			</LI><LI>
						Insert:Vecor 3:1 (GoldenGate)
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Friday 26th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Transformation of 1 µl GoldenGate sample in DH5α and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Dawa, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		No colonies visible
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>PCR of BBa_K1319004 and BBa_K1319008 (TEV) to check if biobricks are in plasmid</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">PCR</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Dawa, Chris
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro transformation of BBa_K1319004 and BBa_K1319008 (TEV) in DH5α and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: 
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		No colonies for BBa_K1319008
			    			</LI></UL><DIV class="date">Monday 29th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Chemical transformation of BBa_K1319008 (TEV) in DH5α and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Chemical Transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Dawa, Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		3 µl (= 750 pg) were used
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Chemical transformation of GolenGate sample from 24.05. and also from 26.05. in DH5α and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Chemical Transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Dawa, Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		5 µl for each transformation were used
			    			</LI></UL><DIV class="date">Tuesday 30th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Colony PCR of DH5α pSB1C3-Cas13a-LwaLwa (from GoldenGate)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">cPCR</A></LI><LI class="myParticipants">
			    		Participants: Chris, Dawa
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Colony PCR of DH5α pSB1C3-BBa_K1319004 and pSB1C3-BBa_K1319008 (TEV)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">cPCR</A></LI><LI class="myParticipants">
			    		Participants: Chris, Dawa
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Primers: VF, VR
			    			</LI><LI>
			    		Overnight cultures from cultures that showed right cPCR bands were set up 
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Wednesday 31st</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of buffers for Cas13a purification (Elution, Wash, Ion exchange and SEC)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta purification</A></LI><LI class="myParticipants">
			    		Participants: Milica, Ludwig, Flo, Sven, Jorge 
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α pSB1C3-BBa_K1319004 and pSB1C3-BBa_K1319008 (TEV) overnight cultures</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Miniprep</A></LI><LI class="myParticipants">
			    		Participants: Milica, Ludwig, Flo, Sven, Jorge 
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Each construct was sent for sequencing with primers VR and VF
			    			</LI></UL></DIV><DIV class="month" id="June"><DIV class="date firstOfMonth">Thursday 1st</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Colony PCR of DH5α pSB1C3-Cas13a-Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">cPCR</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Sven
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI> 		Primers: VR, VF
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Äkta purification of Cas13a Lbu and Cas13a Lwa (Affinity chromatography)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Sven
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Friday 2nd</DIV><UL class="myEntry other"><LI class="myTitel"><B>Total RNA extraction of <I>E. coli</I> W3110 with Norgen kit</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols#lolxd">Total RNA Purification with Norgen Kit</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Instead of using TE and lysozyme to lyse the cells, they were resuspenden in 2% SDS in TAE and incubated at 75 ºC for 15 min.
			    			</LI><LI>
			    			        The 6 samples were stored at -80 ºC and labeled T RNA #1-6
			    			</LI></UL><DIV class="date">Monday 5th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of overnight cultures from Rosetta pSB1C3-His-MBP-Lbu-Cas13a-WT and Rosetta p2CT-His-MBP-Lsh-Cas13a-WT glycerol stocks, DH5α pSB-Cas13a-Lwa cPCR colony 5 and 
						DH5α pSB1C3-BBa_K1319004 cPCR colony 1 and DH5α pSB1C3-BBa_K1319008 cPCR colony 1</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Sven
			    	</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>RNA Adsorption Test on first 3D print</B></LI><LI class="myParticipants">
			    		Participants: Jorge

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI> Total RNA sample 5 used, prepared on Friday 02/06 by Jorge (Concentration: 126 ng/µl)</LI><LI> Procedure:
                            </LI><LI>Take 1 µl presample from stock ; add with 2 µl nf H2O and 3 µl 2x RNA LD</LI><LI>Put 5 µl of sample on device</LI><LI>Take 1 µl of sample immediately, add with 2 µl nf H2O and 3 µl 2x RNA LD</LI><LI>Wait 15 minutes</LI><LI>Take 1 µl of sample immediately, add with 2 µl nf H2O and 3 µl 2x RNA LD</LI><LI>Store at -80 °C</LI><LI> What still needs to be done:
</LI><LI>
Cook samples at 95 °C for 10 minutes and put directly on ice afterwards (to prevent refolding)
</LI><LI>
Load on Gel and quantify</LI></UL><DIV class="date">Tuesday 6th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Inoculation of Rosetta pSB1C3-His-MBP-Lbu-Cas13a-WT and Rosetta p2CT-His-MBP-Lsh-Cas13a-WT overnight cultures in each 1 l LB<SUB>Carb + Cm</SUB> medium</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Sven, Patrick
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of glycerol stocks for DH5α pSB1C3-BBa_K1319004 cPCR and DH5α pSB1C3-BBa_K1319008</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Glycerol cryo stocks</A></LI><LI class="myParticipants">
			    		Participants: Sven, Patrick
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α pSB-Cas13a-Lwa cPCR colony 5, DH5α pSB1C3-BBa_K1319004 and DH5α pSB1C3-BBa_K1319008</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Miniprep</A></LI><LI class="myParticipants">
			    		Participants:  Sven, Patrick
			    	</LI></UL><DIV class="date">Wednesday 7th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting of bacteria with overexpressed Cas13a Lbu and Cas13a Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5 </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI></UL><DIV class="date">Thursday 8th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of wash and elution buffer for Cas13a protein purification</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		pH was checked and adapted after adding of TCEP
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Purification of Cas13a Lbu and Cas13a Lsh (Affinity chromatography)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Sven
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α pSB-Cas13a-Lwa cPCR colony 5 and preparation of glycerol stock</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation and glycerol stock</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Erika, Sven
			    	</LI></UL><DIV class="date">Friday 9th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>PCR of pSB1C3-BBa_K1319008 (TEV) for adding a His tag </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">PCR</A></LI><LI class="myParticipants">
			    		Participants: Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Primers: p-TEV-His-fwd and p-TEV-His-rev
			    			</LI><LI>
						Procedure was follwed by PCR cleanup and DpnI digestion
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Monday 12th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Purification of Cas13a Lbu and Cas13a Lsh (Size exclusion chromatography)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta purification</A></LI><LI class="myParticipants">
			    		Participants: Sven, Ludwig, Benedikt, Milica
			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
			    		Cas13a Lbu and Cas13a were upconcentrated to 1 ml each (centrifugal filters: MWCO: 30 kDa). 
						2 runs for each protein were necessary
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>PCR of Lwa part2b (reordered gBlock without internal BsaI restriction site)</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Sven, Ludwig, Benedikt, Milica
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Primers: VF, VR
			    			</LI><LI>
						Procedure was follwed by PCR cleanup 
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Assembly of Lwa part1, part2b and pSB1C3-ccdB with GoldenGate and transformation in DH5α </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">GoldenGate Cloning</A></LI><LI class="myParticipants">
			    		Participants: Sven, Ludwig, Benedikt, Milica
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Ligation of pSB1C3-BBa_K1319008 (TEV) after adding the His tag by overhang PCR</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Sven, Ludwig, Benedikt, Milica
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Procedure was follwed by chemicial transformation in DH5α and plating on Lb<SUB>Cm</SUB> agar plates
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Tuesday 13th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Colony PCR of DH5α pSB-Cas13a-Lwa from GoldenGate assembly (with part2b)</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myResults"><H4>Results</H4></LI><LI>
						cPCR showed positive results. Cloning seemed to work, although there was the internal BsaI restriction site 
						</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Redone: Transformation of 15 µl ligation sample (of pSB1C3-BBa_K1319008 (TEV) after adding the His tag by overhang PCR) </B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Colonies were visible the next day this time. 2 colonies were picked and resuspended in each 5 ml LB<SUB>Cm</SUB> for overnight cultures
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B> Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5 </B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: 
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Inoculation from cyro stock
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry other"><LI class="myTitel"><B>Lysis-tests with acidic phenol chlorophorm extraction </B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Alkaline lysis</A>
			    		Protocol:  <A href="#">Phenol-Chlorophorm extraction</A>
			    		Protocol:  <A href="#">Trizol Reagenz protocol</A></LI><LI class="myParticipants">
			    		Participants: Julian, Patrick

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    				Three lysis methods were used: Alkaline Lysis, heat lysis with SDS and Trizol-Lyse.
			    			</LI><LI>
			    				 No vortexing for lysis (may break up cells and won't be available in our device), only room-temperature (apart form SDS as reference)
			    			</LI><LI>
			    				Cells had OD of 5.97 -&gt; 4.78*10<SUP>9</SUP> cells according to http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=1029158
			    			</LI><LI>
			    				Cells had OD of 5.97 -&gt; 4.78*10<SUP>9</SUP> cells according to http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=1029158 
			    				-&gt; dilluted 1:100 (to V=10mL) and pelleted 209 miL to get to 10<SUP>7</SUP> cells 
			    			</LI></UL></DIV></DIV></DIV></DIV></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">B</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">C</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">D</SPAN></DIV></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">1</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-0-0" data-namespace="table-ns3555"><SPAN>Method</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-0-1" data-namespace="table-ns3555"><SPAN>SDS</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-0-2" data-namespace="table-ns3555"><SPAN>NaOH</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-0-3" data-namespace="table-ns3555"><SPAN>TRIzol</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">2</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-1-0" data-namespace="table-ns3555"><SPAN>prepare</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-1-1" data-namespace="table-ns3555"><SPAN>1 mL TES (10mM Tris-HCl, 10mM EDTA, pH 8.0, 2% SDS -&gt; TE-Buffer + SDS...</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">3</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-2-0" data-namespace="table-ns3555"><SPAN>----Lysis</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-2-1" data-namespace="table-ns3555"><SPAN>1x</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-2-2" data-namespace="table-ns3555"><SPAN>3x</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-2-3" data-namespace="table-ns3555"><SPAN>1x</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">4</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-3-0" data-namespace="table-ns3555"><SPAN>separate to tubes Spin 300xG, 5 min -&gt;Ice, TROzol to -20! (100 miL at OD 1 -&gt; 10<SUP>7</SUP> cells, Max. according to TRIzolPaper</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-3-1" data-namespace="table-ns3555"><SPAN>100 miL bactSusp</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-3-2" data-namespace="table-ns3555"><SPAN>100 miL bact-susp</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-3-3" data-namespace="table-ns3555"><SPAN>100 miL bact-suspension</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">5</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-4-0" data-namespace="table-ns3555"><SPAN>No washing step, bad for mRNA (TRIzolP.)</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-4-1" data-namespace="table-ns3555"><SPAN> 300 µl (0.45 ml) TES resuspend pellet</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">6</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-5-0" data-namespace="table-ns3555"><SPAN>12,5 uL Inhibitor (40k U/ml, 1 U/miL required)</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-5-1" data-namespace="table-ns3555"><SPAN>15 min, 75 °C lysis, vortex</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-5-2" data-namespace="table-ns3555"><SPAN>one probe after another</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">7</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-6-1" data-namespace="table-ns3555"><SPAN>(Addition of inhibitor); Transform total volume to gel tube</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-6-2" data-namespace="table-ns3555"><SPAN>resuspend thorougly in 66 µl (100 miL) Solution1</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">8</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-7-2" data-namespace="table-ns3555"><SPAN>132 µl (200 miL) Sol2, INVERT 4x</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">9</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-8-2" data-namespace="table-ns3555"><SPAN>wait 0, 1 ,3 min</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">10</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-9-2" data-namespace="table-ns3555"><SPAN>add 99 µl (150 miL) Sol3, INVERT 4x</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">11</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-10-2" data-namespace="table-ns3555"><SPAN>Total volume: 297 µl (0.45 ml)</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">12</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-11-0" data-namespace="table-ns3555"><SPAN>--phenChloExtr</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">13</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" rowspan="1" colspan="2" data-cid="table-note-part2402-12-1" data-namespace="table-ns3555"><SPAN>measure pH, add acid...</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-12-3" data-namespace="table-ns3555"><SPAN>meassure Trizol PH, add acid if needed (pH &lt;5)</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">14</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" rowspan="1" colspan="2" data-cid="table-note-part2402-13-1" data-namespace="table-ns3555"><SPAN>add 600 µl [same V (0.45 ml)] Phe:chloro.</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-13-3" data-namespace="table-ns3555"><SPAN>add 1 mL TRIzol to 10<SUP>7</SUP> cells</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">15</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-14-3" data-namespace="table-ns3555"><SPAN>5 min incubation</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">16</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-15-3" data-namespace="table-ns3555"><SPAN>add 0.2 mL CHLOROPHORM</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">17</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-16-0" data-namespace="table-ns3555" style="color: rgb(52, 64, 67);"><SPAN>incub 2.5 (2-3) min, RT</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">18</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-17-0" data-namespace="table-ns3555" style="color: rgb(52, 64, 67);"><SPAN>centri 12k g 15 min 4°C</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">19</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-18-0" data-namespace="table-ns3555"><SPAN>angle 45°, extract top phase carefully</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-18-1" data-namespace="table-ns3555"><SPAN>-&gt; discard everything else, prot &amp; DNA</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-18-3" data-namespace="table-ns3555"><SPAN>-&gt; which Vol for the TRIzol aqueous phase?</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">20</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-19-0" data-namespace="table-ns3555"><SPAN>add 0.4 (0.375) ml Isopropanol, wait 10 min, RT</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">21</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-20-0" data-namespace="table-ns3555"><SPAN>centrifuge 10 min 12k , 4°C -&gt;</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">22</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-21-0" data-namespace="table-ns3555" style="color: rgb(225, 69, 22);"><SPAN>Aditional step, because RNA pellet was not entierly spinned down: Centrifugation (16000 rcf, 5 min, 4 °C)</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-21-1" data-namespace="table-ns3555"><SPAN>big-pellet assay in fridge, but 1. interphase was punctured/gel(DNA?) stuck to pipette 2. s.u.</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">23</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-22-0" data-namespace="table-ns3555" style="color: rgb(225, 69, 22);"><SPAN>Aditional step, because suspected RNA fog was not entirely spinned down: remove upper and lower RNA fog phase</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">24</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-23-0" data-namespace="table-ns3555" style="color: rgb(225, 69, 22);"><SPAN>Additional step: Centrifugation (16000 rcf, 5 min, 4 °C)</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">25</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-24-0" data-namespace="table-ns3555" style="color: rgb(225, 69, 22);"><SPAN>No RNA pellet or RNA fog any more :(</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">26</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-25-0" data-namespace="table-ns3555" style="color: rgb(225, 69, 22);"><SPAN>Addition of 400 µl isopropanol -&gt; No RNA pellet or fog any more -&gt;prob. removed too much</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">27</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-26-0" data-namespace="table-ns3555"><SPAN>discard supernatant -&gt; RNA in gel-like pellet!</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-26-1" data-namespace="table-ns3555"><SPAN>-&gt; gel hardly visible, nearly draged it out of tube (stuck to pipette...)</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">28</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-27-0" data-namespace="table-ns3555" style="color: rgb(0, 0, 0);"><SPAN>resuspend in 0.75 ml 70 % (75% ideally) ethanol</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-27-1" data-namespace="table-ns3555"><SPAN>-&gt; storeed RNA at -20°C is stable up to 1 year...</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">29</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-28-0" data-namespace="table-ns3555"><SPAN>vortex, centri 5 min 7,5k 4°</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">30</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-29-0" data-namespace="table-ns3555"><SPAN>discard supernatant, air dry for 5-10 min (do NOT let completely dry out)</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">31</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-30-0" data-namespace="table-ns3555"><SPAN>resuspend in RNAse free water 30 µl (20-50 miL) </SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2402-30-1" data-namespace="table-ns3555"><SPAN>-&gt;measurement...</SPAN></TD></TR></TBODY></TABLE><LI class="myResults"><H4>Results</H4>
                        No pellets seen during procedure, concentration too low for photometer -&gt; repeat with more cells
			    	</LI><DIV class="date">Wednesday 14th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Up-concentration of Cas13a Lbu and Cas13 Lsh after SEC and concentration measurement</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: Chris, Patrick
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Centrifual filters: MWCO: 30 kDa
			    			</LI><LI>
						Concentrations were measured using a Bradford assay
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>
							Cas13 Lbu: 746 mg/ml, Cas13 Lsh: 120 mg/ml
							</LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Miniprep of overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Chris, Patrick
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Plasmid was again sent for sequencing with VR and VF 
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>
						First sequencing looked good
						</LI><DIV class="date">Friday 16th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Sending pSB-Cas13a-Lwa cPCR colony 5 for sequencing again with more primers</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Sequencing</A></LI><LI class="myParticipants">
			    		Participants: Chris, Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Primer: p-seq-lwa 1-4
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Preparation of first readout experiment with Cas13 Lbu and Cas13 Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Readout</A></LI><LI class="myParticipants">
			    		Participants: Chris, Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Cleavage activity was tested with Aptamers (Malachite, Spinach) and RNase alert
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>
						The Cas13a proteins were activated and cleaved the aptamers. This resulted in a decrease in fluorescence intensity. 
						</LI><UL class="myEntry other"><LI class="myTitel"><B>Lysis-tests with acidic phenol chlorophorm extraction </B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Lysis test pipetting scheme</A>
			    		Protocol:  <A href="#">Phenol-Chlorophorm extraction</A></LI><LI class="myParticipants">
			    		Participants: Julian, Patrick

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    				Cell suspension had OD600 (extrapolated) of  2,220 -&gt; 1.78 x 10<SUP>9</SUP> cells/ml according to http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=1029158
			    			</LI><LI>
			    				Experiments were done with 108 and 109 cells per sample, because it failed with 10<SUP>7</SUP> cells last time
			    			</LI><LI>
                                Aspiration of supernatant in phenol-chlorophorm extraction: about 2/3 aspirated (200 µl) and 1/3 (100 µl) left to not destroy interphase
			    			</LI><LI>
                                No pellet after RNA precipitation (-80 °C) and subsequent centrifugation in:
                               </LI><LI>NaOH, 1 min, 10<SUP>8</SUP></LI><LI>SDS, 10<SUP>9</SUP></LI><LI>Trizol, 10<SUP>8</SUP></LI></UL><LI> Mistakes:
                                
			    			    </LI><LI>SDS, 10<SUP>9</SUP>: aspirated 100 µl instead of 200 µl supernatant</LI><LI>NaOH, 10<SUP>8</SUP>, 3 min: contamination with DNA (&lt;10% DNA interphase transfered) </LI><LI>NaOH, 10<SUP>9</SUP>, 3 min: added + 200 µl of SDS 109 tube</LI><LI>! Did not calibrate alkaline lysis solution 3 to given pH -&gt; probably more RNA degradation</LI><LI>! Did store RNA at -20 °C instead of -80 -&gt; probably more RNA degradation</LI><LI class="myResults"><H4>Results</H4><TABLE class="mediocre-tableEditable-table u-small-spacing-right" style="top: 0px; width: 988px;"><TBODY><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">Sample</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">[] [µg/ml]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">adjusted C**</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A260/A280</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A260/A230</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A230 [A]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A260 [A]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A280 [A]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A320 [A]</SPAN></DIV></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">1</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-0-0" data-namespace="table-ns3555"><SPAN>SDS, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-1" data-namespace="table-ns3555"><SPAN>36.8</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>55</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-3" data-namespace="table-ns3555"><SPAN>1.704</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-4" data-namespace="table-ns3555"><SPAN>2.044</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-5" data-namespace="table-ns3555"><SPAN>0.047</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-6" data-namespace="table-ns3555"><SPAN>0.094</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-7" data-namespace="table-ns3555"><SPAN>0.056</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-0-8" data-namespace="table-ns3555"><SPAN>0.002</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">2</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-1-0" data-namespace="table-ns3555"><SPAN>SDS, 10<SUP>9</SUP> cells*</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-1" data-namespace="table-ns3555"><SPAN>119</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>357</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-3" data-namespace="table-ns3555"><SPAN>1.693</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-4" data-namespace="table-ns3555"><SPAN>2.069</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-5" data-namespace="table-ns3555"><SPAN>0.146</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-6" data-namespace="table-ns3555"><SPAN>0.300</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-7" data-namespace="table-ns3555"><SPAN>0.178</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-1-8" data-namespace="table-ns3555"><SPAN>0.002</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">3</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-2-0" data-namespace="table-ns3555"><SPAN>NaOH, 0 min, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-1" data-namespace="table-ns3555"><SPAN>35.2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>53</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-3" data-namespace="table-ns3555"><SPAN>1.725</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-4" data-namespace="table-ns3555"><SPAN>2.146</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-5" data-namespace="table-ns3555"><SPAN>0.042</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-6" data-namespace="table-ns3555"><SPAN>0.089</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-7" data-namespace="table-ns3555"><SPAN>0.052</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-2-8" data-namespace="table-ns3555"><SPAN>0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">4</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-3-0" data-namespace="table-ns3555"><SPAN>NaOH, 0 min, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-1" data-namespace="table-ns3555"><SPAN>586</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>879</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-3" data-namespace="table-ns3555" style="background-color: rgb(132, 198, 32);"><SPAN>1.923</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-4" data-namespace="table-ns3555"><SPAN>1.993</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-5" data-namespace="table-ns3555"><SPAN>0.740</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-6" data-namespace="table-ns3555"><SPAN>1.470</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-7" data-namespace="table-ns3555"><SPAN>0.767</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-3-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">5</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-4-0" data-namespace="table-ns3555"><SPAN>NaOH, 1 min, 10^8 cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-1" data-namespace="table-ns3555"><SPAN>73.6</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>110</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-3" data-namespace="table-ns3555"><SPAN>1.235</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-4" data-namespace="table-ns3555"><SPAN>0.586</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-5" data-namespace="table-ns3555"><SPAN>0.317</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-6" data-namespace="table-ns3555"><SPAN>0.187</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-7" data-namespace="table-ns3555"><SPAN>0.152</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-4-8" data-namespace="table-ns3555"><SPAN>0.003</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">6</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-5-0" data-namespace="table-ns3555"><SPAN>NaOH, 1 min, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-1" data-namespace="table-ns3555"><SPAN>370</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>555</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-3" data-namespace="table-ns3555" style="background-color: rgb(132, 198, 32);"><SPAN>1.945</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-4" data-namespace="table-ns3555"><SPAN>1.958</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-5" data-namespace="table-ns3555"><SPAN>0.475</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-6" data-namespace="table-ns3555"><SPAN>0.927</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-7" data-namespace="table-ns3555"><SPAN>0.478</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-5-8" data-namespace="table-ns3555"><SPAN>0.003</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">7</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-6-0" data-namespace="table-ns3555"><SPAN>NaOH, 3 min, 10<SUP>8</SUP> cells*</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-1" data-namespace="table-ns3555"><SPAN>259</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>389</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-3" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(166, 212, 97);"><SPAN>1.849</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-4" data-namespace="table-ns3555"><SPAN>1.491</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-5" data-namespace="table-ns3555"><SPAN>0.439</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-6" data-namespace="table-ns3555"><SPAN>0.652</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-7" data-namespace="table-ns3555"><SPAN>0.355</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-6-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">8</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-7-0" data-namespace="table-ns3555"><SPAN>NaOH, 3 min, 10<SUP>9</SUP> cells*</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-1" data-namespace="table-ns3555"><SPAN>275</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>413</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-3" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(166, 212, 97);"><SPAN>1.850</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-4" data-namespace="table-ns3555"><SPAN>2.212</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-5" data-namespace="table-ns3555"><SPAN>0.316</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-6" data-namespace="table-ns3555"><SPAN>0.693</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-7" data-namespace="table-ns3555"><SPAN>0.368</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-7-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">9</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-8-0" data-namespace="table-ns3555"><SPAN>TRIzol, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-1" data-namespace="table-ns3555"><SPAN>194</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>265</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-3" data-namespace="table-ns3555"><SPAN>1.362</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-4" data-namespace="table-ns3555"><SPAN>0.221</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-5" data-namespace="table-ns3555"><SPAN>2.200</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-6" data-namespace="table-ns3555"><SPAN>0.487</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-7" data-namespace="table-ns3555"><SPAN>0.358</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-8-8" data-namespace="table-ns3555"><SPAN>0.002</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">10</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2455-9-0" data-namespace="table-ns3555"><SPAN>TRIzol, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-1" data-namespace="table-ns3555"><SPAN>160</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-2" data-namespace="table-ns3555" style="color: rgb(0, 0, 0); background-color: rgb(246, 217, 77);"><SPAN>218</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-3" data-namespace="table-ns3555"><SPAN>1.429</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-4" data-namespace="table-ns3555"><SPAN>0.488</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-5" data-namespace="table-ns3555"><SPAN>0.822</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-6" data-namespace="table-ns3555"><SPAN>0.403</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-7" data-namespace="table-ns3555"><SPAN>0.283</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2455-9-8" data-namespace="table-ns3555"><SPAN>0.003</SPAN></TD></TR></TBODY></TABLE></LI><LI>Measured [RNA] by Implen Nanophotometer (20170616).</LI><LI>**adjusted for different supernatant V aspirated at extraction step</LI>
			    	see also &quot;Denaturing PAGE for ssRNA&quot;  Monday, the 26th
			    <DIV class="date">Monday 19th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5 </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Ludwig
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Inoculation from cryo stock
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of DH5α pSB1C3-BBa_K1319008-His (TEV) 1 and 2 </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Ludwig
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Inoculation from cryo stock
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro transformation of pSB1C3-Cas13a-Lwa (cPCR colony 5) into <I>E. coli</I> BL21star</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		 pSB1C3-Cas13a-Lwa (cPCR colony 5) was verified by sequencing and is named now pSB1C3-Cas13a-Lwa 
			    			</LI></UL><DIV class="date">Tuesday 20th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α pSB-Cas13a-Lwa cPCR colony 5</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		The plasmid was sent for sequencing with VR, VF, p-seq-lwa 1-4
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α pSB1C3-BBa_K1319008-His (TEV) 1 and 2</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		The plasmids were sent for sequencing with VR, VF
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of one picked BL21star pSB1C3-Cas13a-Lwa colony</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI></UL><DIV class="date">Wednesday 21st</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of BL21star pSB1C3-Cas13a-Lwa overnight culture and send for sequencing </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Erika, Igor, Milica
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    	Primers: VF, VR, p-seq-lwa 1-4
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of glycerol stock for BL21star pSB1C3-Cas13a-Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Glycerol stock</A></LI><LI class="myParticipants">
			    		Participants: Erika, Igor, Milica
			    	</LI></UL><DIV class="date">Friday 23rd</DIV><UL class="myEntry other"><LI class="myTitel"><B>Time-series of alkaline lysis, in triplets </B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Lysis test pipetting scheme</A></LI><LI class="myParticipants">
			    		Participants: Julian, Patrick

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    				Did alkaline Lysis with inbubation/lysis times of 0 min (~10 sec ), 1 min, 3 min, 10 min, Heat+SDS as reference, no Trizol
			    			</LI><LI>
                            Cell suspension with OD600 (extrapolated) of 2.182 -&gt; 1.78 x 109 cells/ml
			    			</LI><LI>
                            Phenol-Chlorophorm Precipitation at -80 was done until Wednesday
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                        see &quot;Denaturing PAGE for ssRNA&quot;  Thursday (not Monday!), the 29th
			    	</LI><DIV class="date">Monday 26th</DIV><UL class="myEntry other"><LI class="myTitel"><B>Gel analysis of extracted RNA (July 16th !)</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Denaturing PAGE for ssRNA</A></LI><LI class="myParticipants">
			    		Participants: Julian, Patrick

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
                            see Friday, 16th
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Tuesday 27th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Electro Transformation of pSB1C3-BBa_K1319008-His (TEV) in BL21star and plating on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		1 µl was used
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm + Carb</SUB> overnight culture of <I>E. coli</I> Rosetta p2CT-His-MBP-Lbu-Cas13a-WT </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of <I>E. coli</I> DH5α pSB1C3-BBa_K1319008-His 2</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI></UL><DIV class="date">Wednesday 28th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Inoculation of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT in 4x 1 l 2xYT<SUB>Cm + Carb</SUB> medium for expression of Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Chris, Ludwig,Jorge
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of DH5α pSB1C3-BBa_K1319008-His 2 and send for sequencing</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation and Sequencing</A></LI><LI class="myParticipants">
			    		Participants: Chris, Ludwig, Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Primers: VF, VR
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of BL21star pSB1C3-BBa_K1319008-His from picked colony</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Chris, Ludwig, Jorge
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Plating of DH5α pSB1C3-Cas13a-Lwa glycerol stock on LB<SUB>Cm</SUB> agar plates</B></LI><LI class="myProtocol">
			    		Protocol:  </LI><LI class="myParticipants">
			    		Participants: 
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Colonies should be checked if they grow the same and a cPCR should confirm that there is only one plasmid
			    			</LI></UL><DIV class="date">Thursday 29th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of glycerol stock of BL21star pSB1C3-BBa_K1319008-His </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Glycerol stock</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		From this glycerol stock a 5 ml LB<SUB>Cm</SUB> overnight culture was prepared
			    			</LI><LI>
						A 5 ml LB<SUB>Cm</SUB> overnight culture was prepared directly from this cryo stock
							</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT with overexpressed Cas13a Lbu </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		The pellet was stored at -80 °C
			    			</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Colony PCR of DH5α pSB1C3-Cas13a-Lwa (plated glycerol stock)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">cPCR</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Chris
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Time-series of alkaline lysis, in triplets (continued)</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Phenol-Chlorophorm extraction</A></LI><LI class="myParticipants">
			    		Participants: Patrick

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
                            Continuing after -80 °C precipitation
			    			</LI><LI>
                            Extremely big pellets after 99 % EtOH precipitation for all samples with SDS: Possibly contaminated with DNA
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>
                        SDS 10⁹ seems to have lower yield than NaOH -&gt; maybe more SDS? (10⁸ has no problems...)
			    			</LI><LI>
                        NaOH is done after &quot;0&quot; minutes (about 10 sec for adding base to all eppis and turning 3 times)
			    			</LI><LI><TABLE class="mediocre-tableEditable-table u-small-spacing-right" style="top: 0px; width: 1031px;"><TBODY><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">Sample</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">Number of sample</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">[] [µg/ml]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A260/A280</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A260/A230</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A230 [A]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A260 [A]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A280 [A]</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A320 [A]</SPAN></DIV></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">1</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-0-0" data-namespace="table-ns3555"><SPAN>SDS, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-2" data-namespace="table-ns3555"><SPAN>62.4</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-3" data-namespace="table-ns3555"><SPAN>2.080</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-4" data-namespace="table-ns3555"><SPAN>0.118</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-5" data-namespace="table-ns3555"><SPAN>1.343</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-6" data-namespace="table-ns3555"><SPAN>0.174</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-7" data-namespace="table-ns3555"><SPAN>0.093</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-0-8" data-namespace="table-ns3555"><SPAN>0.018</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">2</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-2" data-namespace="table-ns3555"><SPAN>56.8</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-3" data-namespace="table-ns3555"><SPAN>2.029</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-4" data-namespace="table-ns3555"><SPAN>0.128</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-5" data-namespace="table-ns3555"><SPAN>1.123</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-6" data-namespace="table-ns3555"><SPAN>0.153</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-7" data-namespace="table-ns3555"><SPAN>0.081</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-1-8" data-namespace="table-ns3555"><SPAN>0.011</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">3</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-2" data-namespace="table-ns3555"><SPAN>47.6</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-3" data-namespace="table-ns3555"><SPAN>1.983</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-4" data-namespace="table-ns3555"><SPAN>0.102</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-5" data-namespace="table-ns3555"><SPAN>1.178</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-6" data-namespace="table-ns3555"><SPAN>0.127</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-7" data-namespace="table-ns3555"><SPAN>0.068</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-2-8" data-namespace="table-ns3555"><SPAN>0.008</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">4</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-3-0" data-namespace="table-ns3555"><SPAN>SDS, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-3-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-3-2" data-namespace="table-ns3555"><SPAN>354</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-3-3" data-namespace="table-ns3555"><SPAN>1.782</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-3-4" data-namespace="table-ns3555"><SPAN>&gt;0.4</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-3-5" data-namespace="table-ns3555"><SPAN>&gt;2.5</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-3-6" data-namespace="table-ns3555"><SPAN>0.913</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-3-7" data-namespace="table-ns3555"><SPAN>0.525</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-3-8" data-namespace="table-ns3555"><SPAN>0.029</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">5</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-2" data-namespace="table-ns3555"><SPAN>260</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-3" data-namespace="table-ns3555"><SPAN>1.857</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-4" data-namespace="table-ns3555"><SPAN>0.295</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-5" data-namespace="table-ns3555"><SPAN>2.222</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-6" data-namespace="table-ns3555"><SPAN>0.665</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-7" data-namespace="table-ns3555"><SPAN>0.365</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-4-8" data-namespace="table-ns3555"><SPAN>0.015</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">6</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-2" data-namespace="table-ns3555"><SPAN>376</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-3" data-namespace="table-ns3555"><SPAN>1.801</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-4" data-namespace="table-ns3555"><SPAN>0.392</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-5" data-namespace="table-ns3555"><SPAN>2.422</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-6" data-namespace="table-ns3555"><SPAN>0.962</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-7" data-namespace="table-ns3555"><SPAN>0.544</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-5-8" data-namespace="table-ns3555"><SPAN>0.022</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">7</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-6-0" data-namespace="table-ns3555"><SPAN>NaOH, 0 min, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-2" data-namespace="table-ns3555"><SPAN>39.2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-3" data-namespace="table-ns3555" style="color: rgb(0, 0, 0);"><SPAN>1.690</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-4" data-namespace="table-ns3555"><SPAN>2.042</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-5" data-namespace="table-ns3555"><SPAN>0.056</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-6" data-namespace="table-ns3555"><SPAN>0.106</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-7" data-namespace="table-ns3555"><SPAN>0.066</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-6-8" data-namespace="table-ns3555"><SPAN>0.008</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">8</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-2" data-namespace="table-ns3555"><SPAN>53.6</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-3" data-namespace="table-ns3555" style="color: rgb(0, 0, 0);"><SPAN>1.614</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-4" data-namespace="table-ns3555"><SPAN>1.887</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-5" data-namespace="table-ns3555"><SPAN>0.086</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-6" data-namespace="table-ns3555"><SPAN>0.149</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-7" data-namespace="table-ns3555"><SPAN>0.098</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-7-8" data-namespace="table-ns3555"><SPAN>0.015</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">9</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-2" data-namespace="table-ns3555"><SPAN>68.8</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-3" data-namespace="table-ns3555"><SPAN>1.720</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-4" data-namespace="table-ns3555"><SPAN>2.457</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-5" data-namespace="table-ns3555"><SPAN>0.070</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-6" data-namespace="table-ns3555"><SPAN>0.172</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-7" data-namespace="table-ns3555"><SPAN>0.100</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-8-8" data-namespace="table-ns3555"><SPAN>0.000</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">10</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-9-0" data-namespace="table-ns3555"><SPAN>NaOH, 0 min, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-2" data-namespace="table-ns3555"><SPAN>629</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-3" data-namespace="table-ns3555"><SPAN>1.911</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-4" data-namespace="table-ns3555"><SPAN>2.466</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-5" data-namespace="table-ns3555"><SPAN>0.643</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-6" data-namespace="table-ns3555"><SPAN>1.578</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-7" data-namespace="table-ns3555"><SPAN>0.828</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-9-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">11</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-2" data-namespace="table-ns3555" style="background-color: rgb(249, 220, 220);"><SPAN>17.6</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-3" data-namespace="table-ns3555"><SPAN>1.692</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-4" data-namespace="table-ns3555"><SPAN>2.200</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-5" data-namespace="table-ns3555"><SPAN>0.019</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-6" data-namespace="table-ns3555"><SPAN>0.043</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-7" data-namespace="table-ns3555"><SPAN>0.025</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-10-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">12</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-2" data-namespace="table-ns3555"><SPAN>619</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-3" data-namespace="table-ns3555"><SPAN>1.897</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-4" data-namespace="table-ns3555"><SPAN>2.453</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-5" data-namespace="table-ns3555"><SPAN>0.636</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-6" data-namespace="table-ns3555"><SPAN>1.553</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-7" data-namespace="table-ns3555"><SPAN>0.821</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-11-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">13</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-12-0" data-namespace="table-ns3555"><SPAN>NaOH, 1 min, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-2" data-namespace="table-ns3555"><SPAN>52</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-3" data-namespace="table-ns3555"><SPAN>1.806</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-4" data-namespace="table-ns3555"><SPAN>2.203</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-5" data-namespace="table-ns3555"><SPAN>0.059</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-6" data-namespace="table-ns3555"><SPAN>0.130</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-7" data-namespace="table-ns3555"><SPAN>0.072</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-12-8" data-namespace="table-ns3555"><SPAN>0.000</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">14</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-2" data-namespace="table-ns3555"><SPAN>33.2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-3" data-namespace="table-ns3555"><SPAN>1.729</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-4" data-namespace="table-ns3555"><SPAN>2.371</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-5" data-namespace="table-ns3555"><SPAN>0.035</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-6" data-namespace="table-ns3555"><SPAN>0.083</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-7" data-namespace="table-ns3555"><SPAN>0.048</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-13-8" data-namespace="table-ns3555"><SPAN>0.000</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">15</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-2" data-namespace="table-ns3555"><SPAN>58.4</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-3" data-namespace="table-ns3555"><SPAN>1.738</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-4" data-namespace="table-ns3555"><SPAN>2.393</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-5" data-namespace="table-ns3555"><SPAN>0.061</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-6" data-namespace="table-ns3555"><SPAN>0.146</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-7" data-namespace="table-ns3555"><SPAN>0.084</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-14-8" data-namespace="table-ns3555"><SPAN>0.000</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">16</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-15-0" data-namespace="table-ns3555"><SPAN>NaOH, 1 min, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-2" data-namespace="table-ns3555"><SPAN>606</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-3" data-namespace="table-ns3555"><SPAN>1.893</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-4" data-namespace="table-ns3555"><SPAN>2.469</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-5" data-namespace="table-ns3555"><SPAN>0.620</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-6" data-namespace="table-ns3555"><SPAN>1.522</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-7" data-namespace="table-ns3555"><SPAN>0.807</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-15-8" data-namespace="table-ns3555"><SPAN>0.006</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">17</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-2" data-namespace="table-ns3555"><SPAN>627</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-3" data-namespace="table-ns3555"><SPAN>1.898</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-4" data-namespace="table-ns3555"><SPAN>2.431</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-5" data-namespace="table-ns3555"><SPAN>0.651</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-6" data-namespace="table-ns3555"><SPAN>1.574</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-7" data-namespace="table-ns3555"><SPAN>0.832</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-16-8" data-namespace="table-ns3555"><SPAN>0.006</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">18</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-2" data-namespace="table-ns3555" style="background-color: rgb(249, 220, 220);"><SPAN>22.4</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-3" data-namespace="table-ns3555"><SPAN>1.806</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-4" data-namespace="table-ns3555"><SPAN>2.435</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-5" data-namespace="table-ns3555"><SPAN>0.023</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-6" data-namespace="table-ns3555"><SPAN>0.056</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-7" data-namespace="table-ns3555"><SPAN>0.031</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-17-8" data-namespace="table-ns3555"><SPAN>0.000</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">19</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-18-0" data-namespace="table-ns3555"><SPAN>NaOH, 3 min, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-2" data-namespace="table-ns3555"><SPAN>44</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-3" data-namespace="table-ns3555"><SPAN>1.864</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-4" data-namespace="table-ns3555"><SPAN>2.444</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-5" data-namespace="table-ns3555"><SPAN>0.044</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-6" data-namespace="table-ns3555"><SPAN>0.109</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-7" data-namespace="table-ns3555"><SPAN>0.058</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-18-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">20</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-2" data-namespace="table-ns3555"><SPAN>61.2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-3" data-namespace="table-ns3555"><SPAN>1.779</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-4" data-namespace="table-ns3555"><SPAN>2.593</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-5" data-namespace="table-ns3555"><SPAN>0.058</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-6" data-namespace="table-ns3555"><SPAN>0.152</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-7" data-namespace="table-ns3555"><SPAN>0.085</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-19-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">21</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-2" data-namespace="table-ns3555" style="background-color: rgb(249, 220, 220);"><SPAN>5.6</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-3" data-namespace="table-ns3555"><SPAN>1.400</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-4" data-namespace="table-ns3555"><SPAN>2.800</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-5" data-namespace="table-ns3555"><SPAN>0.004</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-6" data-namespace="table-ns3555"><SPAN>0.013</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-7" data-namespace="table-ns3555"><SPAN>0.009</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-20-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">22</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-21-0" data-namespace="table-ns3555"><SPAN>NaOH, 3 min, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-2" data-namespace="table-ns3555"><SPAN>619</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-3" data-namespace="table-ns3555"><SPAN>1.908</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-4" data-namespace="table-ns3555"><SPAN>2.432</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-5" data-namespace="table-ns3555"><SPAN>0.643</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-6" data-namespace="table-ns3555"><SPAN>1.554</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-7" data-namespace="table-ns3555"><SPAN>0.818</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-21-8" data-namespace="table-ns3555"><SPAN>0.007</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">23</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-2" data-namespace="table-ns3555"><SPAN>570</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-3" data-namespace="table-ns3555"><SPAN>1.927</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-4" data-namespace="table-ns3555"><SPAN>2.468</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-5" data-namespace="table-ns3555"><SPAN>0.583</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-6" data-namespace="table-ns3555"><SPAN>1.430</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-7" data-namespace="table-ns3555"><SPAN>0.745</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-22-8" data-namespace="table-ns3555"><SPAN>0.006</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">24</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-2" data-namespace="table-ns3555" style="background-color: rgb(249, 220, 220);"><SPAN>20.4</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-3" data-namespace="table-ns3555"><SPAN>1.759</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-4" data-namespace="table-ns3555"><SPAN>2.429</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-5" data-namespace="table-ns3555"><SPAN>0.020</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-6" data-namespace="table-ns3555"><SPAN>0.050</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-7" data-namespace="table-ns3555"><SPAN>0.028</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-23-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">25</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-24-0" data-namespace="table-ns3555"><SPAN>NaOH, 10 min, 10<SUP>8</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-2" data-namespace="table-ns3555"><SPAN>10.8</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-3" data-namespace="table-ns3555"><SPAN>1.588</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-4" data-namespace="table-ns3555"><SPAN>2.250</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-5" data-namespace="table-ns3555"><SPAN>0.012</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-6" data-namespace="table-ns3555"><SPAN>0.027</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-7" data-namespace="table-ns3555"><SPAN>0.017</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-24-8" data-namespace="table-ns3555"><SPAN>0.000</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">26</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-2" data-namespace="table-ns3555"><SPAN>8.8</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-3" data-namespace="table-ns3555"><SPAN>1.571</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-4" data-namespace="table-ns3555"><SPAN>3.143</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-5" data-namespace="table-ns3555"><SPAN>0.006</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-6" data-namespace="table-ns3555"><SPAN>0.021</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-7" data-namespace="table-ns3555"><SPAN>0.013</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-25-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">27</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-2" data-namespace="table-ns3555"><SPAN>4.4</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-3" data-namespace="table-ns3555"><SPAN>1.375</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-4" data-namespace="table-ns3555"><SPAN>3.667</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-5" data-namespace="table-ns3555"><SPAN>0.002</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-6" data-namespace="table-ns3555"><SPAN>0.010</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-7" data-namespace="table-ns3555"><SPAN>0.007</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-26-8" data-namespace="table-ns3555"><SPAN>-0.001</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">28</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part2640-27-0" data-namespace="table-ns3555"><SPAN>NaOH, 10 min, 10<SUP>9</SUP> cells</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-1" data-namespace="table-ns3555"><SPAN>1</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-2" data-namespace="table-ns3555"><SPAN>503</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-3" data-namespace="table-ns3555"><SPAN>1.916</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-4" data-namespace="table-ns3555"><SPAN>2.499</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-5" data-namespace="table-ns3555"><SPAN>0.508</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-6" data-namespace="table-ns3555"><SPAN>1.262</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-7" data-namespace="table-ns3555"><SPAN>0.661</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-27-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">29</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-1" data-namespace="table-ns3555"><SPAN>2</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-2" data-namespace="table-ns3555"><SPAN>564</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-3" data-namespace="table-ns3555"><SPAN>1.911</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-4" data-namespace="table-ns3555"><SPAN>2.487</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-5" data-namespace="table-ns3555"><SPAN>0.572</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-6" data-namespace="table-ns3555"><SPAN>1.415</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-7" data-namespace="table-ns3555"><SPAN>0.743</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-28-8" data-namespace="table-ns3555"><SPAN>0.005</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">30</SPAN></DIV></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-1" data-namespace="table-ns3555"><SPAN>3</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-2" data-namespace="table-ns3555"><SPAN>611</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-3" data-namespace="table-ns3555"><SPAN>1.902</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-4" data-namespace="table-ns3555"><SPAN>2.467</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-5" data-namespace="table-ns3555"><SPAN>0.625</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-6" data-namespace="table-ns3555"><SPAN>1.533</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-7" data-namespace="table-ns3555"><SPAN>0.809</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part2640-29-8" data-namespace="table-ns3555"><SPAN>0.006</SPAN></TD></TR></TBODY></TABLE></LI><DIV class="date">Friday 30th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Minipreparation of BL21star pSB1C3-BBa_K1319008-His </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		The plasmid was sent for sequencing with primers VR and VF
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Gel analysis of alkaline-extracted RNA time-series (July 23th !)</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Denaturing PAGE for ssRNA</A></LI><LI class="myParticipants">
			    		Participants: Julian, Patrick

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
                            see Friday, 23th
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI></DIV><DIV class="month" id="July"><DIV class="date firstOfMonth">Monday 3rd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight culture of BL21star pSB-Cas13a-Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI></UL><DIV class="date">Tuesday 4th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Inoculation of BL21star pSB-Cas13a-Lwa in 4x 1 l 2xYT<SUB>Cm</SUB> medium for expression of Cas13a Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Ni NTA agarose protein purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Dawa
			    	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Purification of Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Ni NTA agarose purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Dawa
			    	</LI><LI class="myObservations">
			    		Observations:</LI><LI>
			    		Fresh DTT was added to the lysis buffer
			    			</LI><LI>
						1 ml Ni-NTA agarose was used per 4 ml lysate 
							</LI><LI>
						Sample was incubated 1 h at 4 °C while shaking
							</LI><LI>
						The procedure was followed by TEV cleavage overnigth while dialyzing
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Wednesday 5th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting of BL21star pSB-Cas13a-Lwa with overexpressed Cas13a Lwa </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Ni NTA protein purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Dawa, Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Pellet was incubated for 1 h at -80 °C before it was used for purification
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Purification of Cas13a Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Ni NTA agarose purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Dawa, Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Fresh DTT was added to the lysis buffer
			    			</LI><LI>
						1 ml Ni-NTA agarose was used per 4 ml lysate 
							</LI><LI>
						Sample was incubated 1 h at 4 °C while shaking
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Precipitation of TEV plasmid for European meetup in Delft</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Max, Dawa, Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		pSB1C3-BBa_K1319008-His was lyophylized by Ethanol precipitation and evaporation
			    			</LI></UL><DIV class="date">Friday 7th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Reverse Ni-NTA purification of cut Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Ni NTA agarose protein purification</A></LI><LI class="myParticipants">
			    		Participants: Rob
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    		Ni NTA agarose binds the cleaved His-MBP tag, Cas13a Lbu is in the flow-through
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Monday 10th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 50 ml LB<SUB>Cm</SUB> overnight culture of BL21star pSB1C3-BBa_K1319008-His (TEV)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Rob, Ludwig, Rob
			    	</LI></UL><DIV class="date">Tuesday 11th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Inoculation of BL21star pSB1C3-BBa_K1319008-His (TEV) in 3x 1 l 2xYT<SUB>Cm</SUB> medium for expression of the TEV protease</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Max
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
						Induction with 1 mM IPTG at OD<SUB>600</SUB> of 0.6
							</LI><LI>
			    		Expression at 16 °C overnight
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>RNase alert assay with Cas13a Lbu and Cas13a Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor, Rob
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Wednesday 12th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting of BL21star pSB1C3-BBa_K1319008-His with overexpressed TEV protease</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Milica
			    	</LI></UL><DIV class="date">Friday 14th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Protein purification of TEV protease</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Max, Sven
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Cas13a activity assay with RNase alert </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13 activity assay</A></LI><LI class="myParticipants">
			    		Participants: Dawa, Rob
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>RNA Extraction with commercial kit </B></LI><LI class="myParticipants">
			    		Participants: Patrick
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		used ReliaPrep miRNA Cell and Tissue Miniprep System (Promega) according to instructions
			    		</LI><LI>
			    		Specification: Use in range of 10<SUP>2</SUP> to 10<SUP>6</SUP> cells. Tested 10<SUP>6</SUP> and 10<SUP>8</SUP> cells
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
			    	No nanodrop-measureable amount of RNA/DNA with 10<SUP>6</SUP> cell, &lt;10 ug/ml RNA with 10<SUP>8</SUP> cells
			    	Terrible yield, repeat again to be sure
                    </LI><DIV class="date">Monday 17th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>RNA Extraction with commercial kit </B></LI><LI class="myParticipants">
			    		Participants: Patrick
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		used ReliaPrep miRNA Cell and Tissue Miniprep System (Promega) according to instructions
			    		</LI><LI>
			    		Specification: Use in range of 10<SUP>2</SUP> to 10<SUP>6</SUP> cells. Tested 10<SUP>6</SUP> and 10<SUP>8</SUP> cells
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
                    Again no measureable amount of RNA for 10<SUP>6</SUP> cells.
                    For 10<SUP>8</SUP> cells, RNA concentration was increased through column-binding and elution (small elution volume)
                    Elution seems effective, second elution has only 20 % of first elution's amount of RNA
                    </LI><DIV class="date">Tuesday 18th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Preparation of 40 ml LB<SUB>Cm + Carb</SUB> overnight culture of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Chris, Max, Dawa
			    	</LI></UL><DIV class="date">Wednesday 19th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Inoculation of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT in 3x 1 l 2xYT<SUB>Cm + Carb</SUB> medium for expression of Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Dawa 
			    	</LI></UL><DIV class="date">Thursday 20th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Size exclusion chromatography of affinity purified TEV protease</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Sven, Ludwig
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT with overexpressed Cas13 Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Sven, Ludwig
			    	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B><I>In vitro</I> Transcription of target RNA, Lbu crRNA and Lsh crRNA</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols"><I>In vitro</I> Transcription</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
					Procedure was followed by DNAase treatment and phenol chloroform extraction
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Friday 21st</DIV><UL class="myEntry readout"><LI class="myTitel"><B>PCR of gBlock of aeBlue</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">PCR</A></LI><LI class="myParticipants">
			    		Participants: Chris
			    	</LI></UL><DIV class="date">Monday 24th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Digestion of insert and pSB1C3 with Pst1-HF and EcoRI-HF</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Digestion and Ligation</A></LI><LI class="myParticipants">
			    		Participants: Chris
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
					Plasmid was dephosporylated with arctic phosphatase and insert was ligated with plasmid overnight at 16 °C
							</LI></UL><DIV class="date">Tuesday 25th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Electro transformation of aeBlue ligation sample in <I>E. coli</I> Turbo cells</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Electro transformation</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI></UL><DIV class="date">Wednesday 26th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Colony PCR of plated pSB1C3-aeBlue</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">cPCR</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Preparation of 5 ml LB<SUB>Cm</SUB> overnight cultures (picked colonies of aeBlue ligation)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI></UL><DIV class="date">Thursday 27th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Minipreparation of Turbo pSB1C3-aeBlue overnight cultures and send for sequencing</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Minipreparation and sequencing</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI></UL><DIV class="date">Friday 28th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Upconcentration of TEV protease to 2 mg/ml and stored in TEV storage buffer</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Äkta protein purification</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI></UL><DIV class="date">Monday 31th</DIV></DIV><DIV class="month" id="August"><DIV class="date firstOfMonth">Tuesday 1st</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">DNA Gel-Purification and Genomic DNA Purification</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Purification of b-Gal fragments from 1 % Agarose Gel   </LI><LI>
								60 µl per lane 
			    			</LI>
								Genomic DNA purification from E.coli W3110+
			    		</UL><LI class="myResults"><H4>Results</H4>
						DNA was lost during the purification
			    	</LI><DIV class="date">Wednesday 2nd</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Q5-PCR and PCR-Cleanup kit</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Repeated PCR of beta-Gal N and C Terminal fragments from 31.7.   </LI><LI>
								3 different Templates:
								</LI><LI>
			    				W3110 genomic DNA
								</LI><LI>
			    				W3110 genomic DNA diluted 1:100
								</LI><LI>
			    				W3110 2 µl of cryostock
								</LI></UL><LI>
			    				Clean-up of PCR with Purification Kit and elution in 33 µl nf water
			    			</LI><UL class="myEntry other"><LI class="myTitel"><B>InterLab Study calibration</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">IGEM HQ InterLab Study Protocol</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
                                                        Patch length correction was on in the plate reader.
			    			</LI><LI>
			    				Calibration for the plate reader experiment done with LUDOX and Fluorescein.
			    			</LI><LI>
			    				We saw no difference between LUDOX and water when measuring at an OD of 600 nm.
			    			</LI><LI>
			    				For the fluorescein measurement, the best gain was 600 using 485 nm/520 nm.

			    			</LI><LI>
			    				Plasmids from the plate 7 (positive and negative control, and devices 1-6) were resuspended.
			    			</LI></UL><DIV class="date">Thursday 3rd</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Agarose-Gel</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Gel of purified PCR   </LI><LI>
								1,5% small Gel, 120 V, 30 minutes
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>PCR with undiluted extracted DNA didn't work.</LI><DIV class="date">Monday 7th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Protein-Purifcation of Lbu and Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols"> - </A></LI><LI class="myParticipants">
			    		Participants: Max, Ludwig, Teeradon                  </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Inoculation of main cultures of Rosetta p2CT-His-MBP-Lbu_C13a_WT and Lsh in LB(Carb+Cm) </LI></UL><UL class="myEntry other"><LI class="myTitel"><B>InterLab Study Day 1: Transformation</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">IGEM HQ InterLab Study Protocol</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Each plasmids was transformed two times: with chemical transformation and with electroporation, for a total of 16 plasmids.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Every plate except the ones for Device 1 had colonies. </LI><DIV class="date">Tuesday 8th</DIV><UL class="myEntry other"><LI class="myTitel"><B>First test with 3D printed 96-well plate</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Tested different concentrations of fluorescein in a 3D printed 96 well plate on paper.    </LI><LI>
								Scanned at differnt gains at a constant focal height.
			    			</LI><LI>
			    				Dilutions were 1:20; 1:400 and 1:8000 of Fluorescein-FITC in water
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    No light bleeding was observed in the experiment
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>InterLab Study Day 2: Colony transfer to liquid culture</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">IGEM HQ InterLab Study Protocol</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    			        Device 1 was again transformed, as no colonies grew on the plate.
			    			</LI><LI>
			    				Liquid cultures for the 7 plasmids whose plates showed colonies (all except Device 1) were made in duplicate.
			    			</LI><LI>
			    				The plates were stored at -4 °C.
			    			</LI></UL><DIV class="date">Wednesday 9th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Expression and Harvesting of Cas13a variants</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Protein Purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Ludwig                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Harvested Cas13a Lbu overexpression.  </LI><LI>
								Inoculated 2 litres 2xYT media with Rosetta Cas13a Lsh and let them grow at 16 °C overnight with 1 mM IPTG
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>InterLab Study Day 3: Plate reader experiments</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">IGEM HQ InterLab Study Protocol</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    			        After measuring the OD600 of the O/N cultures, they were diluted with LB+Cm to an OD600 of 0.02, in duplicates.
			    			</LI><LI>
			    				The absorbance and fluorescence of the diluted cultures were measured at 0, 2, 4 and 6 hours after the cultures were set.
			    			</LI><LI>
			    				Patch length correction was on in the plate reader.
			    			</LI><LI>
			    				Device 1 was again transformed, as no colonies grew on the plate.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Transformation of Device 1 was successful.</LI><DIV class="date">Thursday 10th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>His Purification of Cas13a Lbu and Harvesting of Cas13a Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">His-Purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Ludwig, Milica, Sven, Benedikt                    </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				His purification of Cas13a Lbu according to protocol.
			    			</LI><LI>
								Elution in 4 times 4 ml.
			    			</LI><LI>
			    				ON Dialysis with TEV protease in 2 litres of Lysis buffer.
			    			</LI><LI>
								Harvesting of Cas13a Lsh according to protocol
							</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>InterLab Study Day 4: Preparation of O/N cultures for plate reader experiments</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">IGEM HQ InterLab Study Protocol</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    			        O/N liquid cultures in duplicate were set up from all 8 Devices.
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Annealing of Invitro Transcription Targe Inhibitor Circuit 3a (IC3a)</B></LI><LI class="myProtocol">
			    		Protocol: 
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Invitro Transcription of IC3a </B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">In vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Incubated over night at 37 ºC</LI></UL><LI class="myResults"><H4>Results</H4>
                                       See results of PEC from 11.10.2017
			    	</LI><DIV class="date">Friday 11th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Reverse HIS purification and SEC of Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">His purification and SEC protein purification</A></LI><LI class="myParticipants">
			    		Participants: Ludwig, Benedikt, Max, Sven                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Incubation  of cleaved Lbu sample with Ni-NTA beads (ca. 7 mL) (1 h, 4 °C)   </LI><LI>
			    			Application on column, keep FT (there is our protein)
			    			</LI><LI>
			    			Elution of His-MBP with ca. 10 mL Elution buffer (for gel analysis)
			    			</LI><LI>
			    			Washing of Ni-NTA bead (3x with water) and storage in 20 % EtOH	
			    			</LI><LI>
			    			Up-concentration of Cas13a Lbu and buffer exchange in SEC buffer (centrifugal filter: MWCO: 30 kDa, 4 °C, 5000 RCF)	
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
						See Gel image of 14.8. for results.
			    	</LI><UL class="myEntry readout"><LI class="myTitel"><B>o</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">Restriction digest, Phenol-chlorophorm purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Phenol-Chloroform with the wrong pH was used.</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Sample was lost due to the use of the wrong Phenol-Chloroform solution.</LI><UL class="myEntry other"><LI class="myTitel"><B>InterLab Study Day 5: Plate reader calibration and experiments</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://static.igem.org/mediawiki/2017/8/85/InterLab_2017_Plate_Reader_Protocol.pdf">IGEM HQ InterLab Study Protocol</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti Erika
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
                                                        Patch length correction was off in the plate reader.
			    			</LI><LI>
			    				Calibration for the plate reader experiment done with LUDOX and Fluorescein.
			    			</LI><LI>
			    				We saw the difference in measurements between LUDOX and water.
			    			</LI><LI>
                                                        After measuring the OD600 of the O/N cultures, they were diluted with LB+Cm to an OD600 of 0.02, in duplicates.			    				
			    			</LI><LI>
			    				The absorbance and fluorescence of the diluted cultures were measured at 0, 2, 4 and 6 hours after the cultures were set.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Monday 14th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>SDS PAGE of Lbu purifcations</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">SDS-PAGE</A></LI><LI class="myParticipants">
			    		Participants: Ludwig                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				SDS-PAGE of Lbu  purification from last week  </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Invitro Transcription of IC3a</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">In vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4>
                                       See results of Restriction digestion and PCE from 15.08.17
			    	</LI></UL><DIV class="date">Tuesday 15th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>DNAse I digest and PCE of the RNA</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">Restriction digest, Phenol-chlorophorm purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Phenol-Chloroform with the wrong pH was used.</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>IC3A has a length of 33 bp = band on top.</LI><DIV class="date">Wednesday 16th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Cas13a preparation</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Benedikt, Ludwig                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Concentration measurment of Lbu, Lsh and Lwa with Bradford essay   </LI></UL><LI class="myResults"><H4>Results</H4>
						Concentration of proteins measured(µg/ml):
							</LI><LI>
			    			718 - Lbu 13.6 (pooled fractions 6, 7, 8)
			    			</LI><LI>
			    			387 - Lbu 12.7 
			    			</LI><LI>
			    			90,8 - Lsh 13.6 (pooled fraction 7,8,9)
			    			</LI><LI>
			    			379 -Lwa 5.7 (elution 1)	
			    			</LI><LI>
			    			1620 - Lwa 5.7 (elution 2)
			    			</LI><LI>
			    			818 - Lwa 5.7 (elution 3)	
			    			</LI><LI>
							405 -Lwa 5.7 (elution 4)
							</LI><DIV class="date">Thursday 17th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>DNA RNA Hybrid Annealing of IC3A with Circuit 3 Activator (C3A) – IC3a concentration screen</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Incubation at room temperature for 1 h of DNA and RNA with varied concentrations of IC3a (RNA).</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>4% Agarose does not seem to work for separation. No oligos at all visible after staining with SybrGreen II and SybrGold.</LI><UL class="myEntry cas13a"><LI class="myTitel"><B>Protein-Expression of Lbu and Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Protein-Expression</A></LI><LI class="myParticipants">
			    		Participants: Benedikt, Ludwig, Milica                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Expression of Rosetta Cas13a Lbu (3L 2xYT Medium, 16°C ON)   </LI><LI>
								Expression of Rosetta Cas13a Lwa (2L 2xYT Medium, 16°C ON) 	
			    			</LI></UL><DIV class="date">Friday 18th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Harvesting and Purification of Cas13a Lbu and Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Protein Purification</A></LI><LI class="myParticipants">
			    		Participants: Benedikt, Milica                 </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Harvesting of both expressions   </LI><LI>
								Purification of Cas13a Lbu according to protocol.
			    			</LI><LI>
			    				Elution in 4 times 3 ml elution buffer.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Sunday 20th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Monday 21st</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Continuation of Cas13a purification</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">His-Purification</A></LI><LI class="myParticipants">
			    		Participants: Benedikt, Milica, Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Continued His purification of Cas13a Lbu     </LI><LI>
								Overnight dialysis with TEV protesae
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>As visible on the gel the elutions 1 and 2 contained the highest amount of protein and were pooled for the TEV cleavage.</LI><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for DNA with <I>E. coli</I> W3110 pre-purified gDNA</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA Extraction for DNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				The gDNA was pre-purified with the Phenol/Chloroform method.
			    			</LI><LI>
			    				The gDNA solution was diluted 1:100, 1:1000, 1:100000, 1:10e6 and 1:10e8. Each dilution was then used as a sample for the FINA extraction.
			    			</LI><LI>
			    				From each dilution, two extractions were performed. One as in protocol (1N-5N), the other without the NaOH washing step (1-5).
			    			</LI><LI>
			    				As negative control (KK) FINA extraction was performed as in protocol with nuclease free water as sample.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for beta-galactosidase from <I>E. coli</I> with FINA-DNA samples and purified gDNA from 21.08.17</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>For the membranes, no template was directly pipette. Instead, the membranes were put in the tube.</LI><LI>p-bGal_N_N was used as forward primer</LI><LI>p-bGal_N_C was used as reverse primer</LI><LI>Annealing was performed at 61 ºC</LI><LI>The amplification step was 90 s long</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products </LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>1N-5N: 1:100-1:10e8 dilutions FINA extraction performed as in protocol.</LI><LI>1-5: 1:100-1:10e8 dilutions FINA extraction without washing step.</LI><LI>1K-5K: 1:100-1:10e8 dilutions.</LI><LI>KK: negative control for FINA extraction (nuclease free water as sample).</LI><DIV class="date">Tuesday 22nd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Reverse His purification of Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">His purification</A></LI><LI class="myParticipants">
			    		Participants: Max, Benedikt, Milica                    </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Incubation (shaker!!) of cleaved Lbu sample with Ni-NTA beads (ca. 7 mL) (1 h, ice) </LI><LI>
			    			Application on column, keep FT (there is our protein) (on ice)
			    			</LI><LI>
			    			Washing with 10 mL lysis-buffer	
			    			</LI><LI>
			    			Elution of His-MBP with ca. 10 mL Elution buffer (for gel analysis)
			    			</LI><LI>
			    			Washing of Ni-NTA bead (3x5mL with water) and storage in 20 % EtOH	
			   </LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for DNA with <I>E. coli</I> W3110 cell culture</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA Extraction for DNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				The extraction was done when the culture had an OD600 of 1.2.
			    			</LI><LI>
			    				The cells were diluted 1:1, 1:100 and 1:1000 before the extraction.
			    			</LI><LI>
			    				Each dilution was used as sample twice: as in protocol (1N-3N and P1N-P3N) and washed with 300 ul lysis buffer from RNA extraction kit (Norgen, P/N 17200) (1G-3G and P1G-P3G).
			    			</LI><LI>
			    				Membranes 1N-3N and 1G-3G were put in 40 ul nuclease free water and incubated at RT for 10 min in order for the bounded DNA to go into solution.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for beta-galactosidase from <I>E. coli</I> with FINA-DNA samples from 22.08.17</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>The membranes from the FINA extraction were directly put into the PCR-mix.
 No template DNA was directly pipetted.</LI><LI>p-bGal_N_N was used as forward primer.</LI><LI>p-bGal_N_C was used as reverse primer.</LI><LI>Annealing was performed at 61 ºC.</LI><LI>The amplification step was 90 s long.</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products. 
</LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>1N-3N: 1:1-1:1000 dilutions FINA extraction performed as in protocol, loaded in gel after elution (no PCR).</LI><LI>1G-3G: 1:1-1:1000 dilutions FINA extraction performed with lysis buffer as washing step, loaded in gel after elution (no PCR).</LI><LI>P1N-P3N: 1:1-1:1000 dilutions FINA extraction performed as in protocol.</LI><LI>P1G-P3G: 1:1-1:1000 dilutions FINA extraction with lysis buffer as washing buffer.</LI><LI>KK: negative control for FINA extraction (nuclease free water as sample).</LI><DIV class="date">Wednesday 23rd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>Cas13a purification</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Size Exclusion Chromatography</A></LI><LI class="myParticipants">
			    		Participants: Benedikt, Milica                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Measuring concentration after Reverse His Lbu    </LI><LI>
								Upconcentration and buffer exchange  of Lbu in 30 kDa Centricon
			    			</LI><LI>
			    				SEC Run of Lbu according to protcol
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
						See SDS PAGE of 24.8. to see the SEC result.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for DNA with  <I>E. coli</I> W3110 cell culture</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA Extraction for DNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Benedikt
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				The extraction was done when the culture had an OD600 of 2.33.
			    			</LI><LI>
                                                        The cells were diluted 1:1, 1:100, 1:10000, 1:10e6 and 1:10e8 before the extraction.
			    			</LI><LI>
                                                        Each dilution was used as a sample for the FINA extraction. In addition, three 1:1 dilutions were used as samples for FINA extraction: two according to protocol and the third with 300 ul lysis buffer as washing buffer. One of the 1:1 membranes was put in nuclease free water and incubated at RT for 20 min in order for the bounded DNA to go into solution.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for beta-galactosidase from <I>E. coli</I> with FINA-DNA samples from 23.08.17</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Benedikt 
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>The membranes from the FINA extraction were directly put into the PCR-mix.
 No template DNA was directly pipetted.</LI><LI>p-bGal_N_N was used as forward primer.</LI><LI>p-bGal_N_C was used as reverse primer.</LI><LI>Annealing was performed at 61 ºC.</LI><LI>The amplification step was 90 s long.</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products. 
</LI><LI>Afte the PCR, only 2,5,L and KK were loaded in the gel, as the other tubes had no liquid.</LI><LI>The gel was loaded before it was submerged in TAE-buffer.</LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>2: 1:100 dilution FINA extraction performed as in protocol</LI><LI>L: 1:1 dilution FINA extraction performed with 300 ul lysis buffer as washing buffer. </LI><LI>KK: negative control for FINA extraction (LB-medium as sample).</LI><DIV class="date">Thursday 24th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B>SDS-Gel of SEC-Run from 23.8.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">SDS-PAGE</A></LI><LI class="myParticipants">
			    		Participants: Chris                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				10% SDS PAGE of SEC Run samples   </LI><LI>
								Marker; Sample prior to run; Fractions 5-16;  FT Centricon
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>The samples with the highest protein concentration were pooled and used for further experiments.</LI><UL class="myEntry readout"><LI class="myTitel"><B>DNA-RNA-Hybrid-Building</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Native PAGE for short DNA oligos</B></LI><LI class="myProtocol">
			    		Protocol: 
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>Slight band shift of the oligo between 4 and 5 µM.</LI></UL><UL><LI>Slight band shift of the oligo between 4 and 5 µM.</LI><LI>Looks better than 30% PAGE in its resolution of the DNA oligos.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for DNA with  <I>E. coli</I> W3110 cell culture (Repetition of the experiment from 23.08.17)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA extraction for DNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Benedikt
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				As there were problems with the gel from the 23th, we repeated the experiment.
			    			</LI><LI>
			    				The extraction was done when the culture had an OD600 of 2.78.
			    			</LI><LI>
                                                        The cells were diluted 1:1, 1:100, 1:10000, 1:10e6 and 1:10e8 before the extraction.
			    			</LI><LI>
                                                        Each dilution was used as a sample for the FINA extraction. In addition, three 1:1 dilutions were used as samples for FINA extraction: two according to protocol and the third with 300 ul lysis buffer as washing buffer. One of the 1:1 membranes was put in nuclease free water and incubated at RT for 20 min in order for the bounded DNA to go into solution.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for beta-galactosidase from <I>E. coli</I> with FINA-DNA samples from 24.08.17</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Benedikt
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>The membranes from the FINA extraction were directly put into the PCR-mix.
 No template DNA was directly pipetted.</LI><LI>p-bGal_N_N was used as forward primer.</LI><LI>p-bGal_N_C was used as reverse primer.</LI><LI>Annealing was performed at 61 ºC.</LI><LI>The amplification step was 90 s long.</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products. 
</LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>1-5: 1:1-1:10e8 dilutions FINA extraction performed as in protocol</LI><LI>L: 1:1 dilution FINA extraction performed with 300 ul lysis buffer as washing buffer. </LI><LI>KK: negative control for FINA extraction (LB-medium as sample).</LI><UL class="myEntry other"><LI class="myTitel"><B>Total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT with Norgen kit</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Total RNA Purification with Norgen Kit</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Benedikt
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    			        2 tubes stored at -80 ºC. Labelled gRNA #1, gRNA #2.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry other"><LI class="myTitel"><B>Phenol/Chloroform total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Phenol/Chloroform purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    			        The incubation step at -80 ºC was done, O/N.
			    			</LI><LI>
			    			        2 tubes stored at -80 ºC. Labelled gRNA PEC #1, gRNA PEC #2.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Friday 25th</DIV><UL class="myEntry other"><LI class="myTitel"><B>Continuation of Phenol/Chloroform total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT from yesterday </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Phenol/Chloroform purification for RNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				See entry from 24.08.17
			    			</LI></UL><DIV class="date">Monday 28th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>DNA RNA Hybrid Annealing of IC3A with Circuit 3 Activator (C3A) – IC3A concentration screen (high concentration)</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>Native PAGE 20 %</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Tuesday 29th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">PCR with Q5 Polymerase, PCR Cleanup Kit, Restriction digest and Ligation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
								Resuspension of gblocks Split_Uni C and Split_Uni N at a conc. of 100 nMol
			    			</LI><LI>
			    				PCR of SPlit Uni C und Split Uni N with Q5 Polymerase    </LI><LI>
								Clean-up with Monarch Kit
			    			</LI><LI>
								Digestion of PCR products and psB1C3 with PstI-HF und EcoRI-HF in Cutsmart buffer
			    			</LI><LI>
								Dephosphorilation with Antarctic Phosphatase
			    			</LI><LI>
			    				Ligation ON with T4 DNA Ligase
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for DNA with <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA extraction for DNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				The extraction was done when the culture had an OD600 of 0.88.
			    			</LI><LI>
                                                        Na: Extraction as in protocol. Cl: Washed with 10 mM HCl instead of NaOH. H: incubated 10 min at 80 ºC. L: 10 min incubated in 1 mg/ml lysozyme. LH: 10 min incubated in 1 mg/ml lysozyme at 80 ºC.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for beta-galactosidase from <I>E. coli</I> with FINA-DNA samples from 24.08.17</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>The membranes from the FINA extraction were directly put into the PCR-mix.
 No template DNA was directly pipetted.</LI><LI>p-bGal_N_N was used as forward primer.</LI><LI>p-bGal_N_C was used as reverse primer.</LI><LI>Annealing was performed at 61 ºC.</LI><LI>The amplification step was 90 s long.</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products. 
</LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4>
                                     See Q5-PCR from 30.08.17
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for RNA with purified total RNA (gRNA PEC #1)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA extraction for RNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>First strand cDNA synthesis from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT purified total RNA (Phenol/Chloroform extraction from 24.08.17) and FINA-RNA eluate</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">First strand cDNA synthesis</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				p-bGal_N_C was used as primer.
			    			</LI><LI>
			    				Sample gRNA PEC #1 from 24.08.17 was diluted 1:4 and used as sample P.
			    			</LI><LI>
			    				The eluate from the FINA extraction for RNA from 29.08.17 was used as sample S.
			    			</LI><LI>
			    				For both reactions (S and P), 6 ul sample were pipetted.
			    			</LI></UL><DIV class="date">Wednesday 30th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>DNA and RNA hybrid of 3C-A and IC3-A</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Concentration screen IC3a between 1 and 5 µM. Incubation for dimerization again for 1 hour and at room temperature.</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>20% Native PAGE, 12.5 mM MgCl2</LI><UL><LI>Shift of DNA-oligo band with higher concentration of IC3a RNA until 3.5 µM. No band of single DNA oligo C3a at 5 µM visible anymore. Hypothesis of full dimerization, but with a big background of single stranded IC3a RNA.</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for beta-galactosidase from <I>E. coli</I> with cDNA samples from 29.08.17</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>p-bGal_N_N was used as forward primer.</LI><LI>p-bGal_N_C was used as reverse primer.</LI><LI>Annealing was performed at 61 ºC.</LI><LI>The amplification step was 90 s long.</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products. 
</LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Na: FINA extraction performed as in protocol.</LI><LI>Cl: FINA extraction performed with 10 mM HCl instead of NaOH.</LI><LI>L: Cell solution incubated 10 min in 1 mg/ml lysozyme. No washing step in FINA</LI><LI>H: Cell solution incubated 10 min at 80 ºC. No washing step in FINA</LI><LI>LH: Cell solution incubated 10 min at 80 ºC in 1 mg/ml lysozyme. No washing step in FINA</LI><LI>K-: negative control FINA (LB-medium used as sample).</LI><LI>P:  1:4 dilution of gRNA PEC #1 after reverse transcription.</LI><LI>S:  1:4 dilution of gRNA PEC #1 after FINA extraction for RNA and reverse transcription.</LI><LI>KK:  Negative control for the cDNA synthesis (water instead of MuLV-RT used)</LI><DIV class="date">Thursday 31st</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">E. comp Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				E. comp transformation of E. coli Turbo with 1 µl of overnight ligation of 27.8.    </LI><LI>
								Platted on Cm plates overnight.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>RNA extraction from induced Lbu-strain</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Lysis test pipetting scheme</A>
                        Protocol:  <A href="#">Phenol-Chlorophorm extraction</A>
                        Protocol:  <A href="#">Denaturing PAGE for ssRNA</A></LI><LI class="myParticipants">
			    		Participants: Dawa

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
                            Induced pSB1C3_lbu with IPTG before lysis
			    			</LI><LI>
                            Two cultures lysed with SDS + Heat only
			    			</LI><LI>
                            No OD600 measurement
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                        See Friday, 1st Sept.
			    	</LI></DIV><DIV class="month" id="September"><DIV class="date firstOfMonth">Friday 1st</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Ligation</A></LI><LI class="myParticipants">
			    		Participants: Max

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Plates showed no colonies, as there wasn't enough backbone used.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
						Repeated digestion and Ligation from Thursday 31.8 and ligated 1 C-Terminal and 3 N-Terminal fragments respectively.
			    	</LI><UL class="myEntry target"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">-</A></LI><LI class="myParticipants">
			    		Participants: Max
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Plates showed no colonies, as there was no SOC step for outgrowth
			    			</LI><LI>
			    				Repeated digestion and Ligation
			    			</LI><LI>
                                                        Ligated 1 C-Terminal and 3 N-Terminal fragments into psb1C3 and psb4A5
                                                </LI></UL><UL class="myEntry other"><LI class="myTitel"><B>RNA extraction from induced Lbu-strain (continued)</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Denaturing PAGE for ssRNA</A></LI><LI class="myParticipants">
			    		Participants: Dawa, Julian

			    	</LI><LI class="myObservations">		Notes:
			    		</LI><LI>
                            Induced pSB1C3_lbu with IPTG before lysis
			    			</LI><LI>
                            Two cultures lysed with SDS + Heat only
			    			</LI><LI>
                            No OD600 measurement
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI><TABLE class="mediocre-tableEditable-table u-small-spacing-right" style="top: 0px; width: 345px;"><TBODY><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">A</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">B</SPAN></DIV></TD><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--col"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">C</SPAN></DIV></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">1</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-0-0" data-namespace="table-ns3555"><SPAN>sample</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-0-1" data-namespace="table-ns3555"><SPAN>conc ~Nanodrop</SPAN></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-0-2" data-namespace="table-ns3555"><SPAN>260/280</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">2</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-1-0" data-namespace="table-ns3555"><SPAN>1 (non-induced)</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-1-1" data-namespace="table-ns3555"><SPAN>2192</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-1-2" data-namespace="table-ns3555"><SPAN>1.8</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">3</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-2-0" data-namespace="table-ns3555"><SPAN>2 (non-induced)</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-2-1" data-namespace="table-ns3555"><SPAN>2320</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-2-2" data-namespace="table-ns3555"><SPAN>1.8</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">4</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-3-0" data-namespace="table-ns3555"><SPAN>3 (induced)</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-3-1" data-namespace="table-ns3555"><SPAN>508</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-3-2" data-namespace="table-ns3555"><SPAN>1.69</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">5</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" data-cid="table-note-part3169-4-0" data-namespace="table-ns3555"><SPAN>4 (induced)</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-4-1" data-namespace="table-ns3555"><SPAN>442</SPAN></TD><TD class="cell is-value is-alignRight cell-contents" data-cid="table-note-part3169-4-2" data-namespace="table-ns3555"><SPAN>1.66</SPAN></TD></TR><TR><TD class="mediocre-tableEditable-axisCell mediocre-tableEditable-axisCell--row"><DIV><SPAN class="mediocre-tableEditable-axisCell-num">6</SPAN></DIV></TD><TD class="cell is-text is-alignLeft cell-contents" rowspan="1" colspan="3" data-cid="table-note-part3169-5-0" data-namespace="table-ns3555"><SPAN> 1,2 showed precipitation of RNA -&gt;measurement/gel wrong if not vortexed completely </SPAN></TD></TR></TBODY></TABLE></LI><LI>
                            A260/280 ration hints to impurities or DNA  -&gt;maybe due to too many cells
			    			</LI><UL class="myEntry other"><LI class="myTitel"><B>Nukleotide-silica binding with different buffers</B></LI><LI class="myProtocol">
                        Protocol:  
                        </LI><LI class="myParticipants">
			    		Participants: Julian
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Used 10 ul of RNA-solution from Aug 31 RNA-extraction by Dawa per sample
			    		</LI><LI>
			    		Did NOT wash silica particles before with binding buffer (pH) and NOT during procedure with ethanol.
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
			    	Both buffers show terrible A260/280 ration, need washing step and need to adjust for acidic silica-solution
			    	</LI><DIV class="date">Saturday 2nd</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Chem_Trafo</A></LI><LI class="myParticipants">
			    		Participants: Julian, Milica, Jorge                        

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Three lysis methods were used: sonifcation, lysozyme digestion and heat lysis in SDS.        All
			    			</LI><LI>
			    				Lysozyme digestion: According to protocol.
			    			</LI><LI>
			    				Heat lysis: Sample was incubated in 2% SDS at 75 °C for 15 min.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    See results from &quot;Phenol/Chloroform purification of RNA from in-vitro transcription&quot; from 11.05.17 for gel picture.     
                    
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Nukleotide-silica binding with different buffers</B></LI><LI class="myProtocol">
                        Protocol:  
</LI><LI class="myParticipants">
			    		Participants: Julian
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Used 10 ul of RNA-solution from Aug 31 RNA-extraction by Dawa 
			    		</LI><LI>
			    		Washed according to protocol
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
			    	Acetate-buffer results acceptable now, GuSCN-buffer still a problem.GuSCN probably too difficult clean up, needs too many washing steps.
			    	</LI><DIV class="date">Monday 4th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">E. comp Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                       </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Plates showed no colonies as there was no SOC step.    </LI><LI>
			    				Retransformation of Ligation from Friday into E.Coli Turbo e. comp.
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Total RNA extraction of <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT with Norgen kit</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Total RNA Purification with Norgen Kit</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Stored at -80 ºC as TRNA Kit #1-4
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Phenol/Chloroform total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">Protocol</A></LI><LI class="myParticipants">
			    		Participants: Jorge, Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>
                                                     Stored at -80 ºC as TRNA PC #1-4.
                                                </LI></UL><LI class="myResults"><H4>Results</H4>
			    		See results Urea-SDS-PAGE from 05.09.17.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Nukleotide-silica binding with different buffers</B></LI><LI class="myProtocol">
                        Protocol:  
</LI><LI class="myParticipants">
			    		Participants: Julian
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Did NOT incubate silica with acetate buffer for a day, prepared it fresh
			    		</LI><LI>
			    		Used 10 ul of RNA-solution from Aug 31 RNA-extraction by Dawa 
			    		</LI><LI>
			    		Washed according to protocol, but did NOT wait a day for silica-buffer interaction with Acetate-buffer
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
			    	Terrible yield, need buffer-silica incubation in protocol.
			    	</LI><DIV class="date">Tuesday 5th</DIV><UL class="myEntry other"><LI class="myTitel"><B>DNA Amplification Test using Recombinase Polymerase Amplification </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				DNA Amplification with RPA was tested using the provided Control Reaction by TwistDx and our BioBrick His6-TEV plasmid 
			    				incubation times of 40 minutes for Control and Sample 1 and 60 minutes for sample 2. Differences due to incubation time 
			    				was tested. 
			    			</LI><LI>
			    				RPA according to protocol
			    			</LI><LI>
			    				PCR cleanup according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Positive Control worked.</LI><LI>RPA of TEV could not produce the whole (~1300 bp) construct but would produce amplicons of shorter size (100-500 bp) which was anticipated since TwistDx protocol says optimal amplicon length is 500 bp.</LI><LI>Full length double-stranded DNA could consequently not be formed.</LI><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Agarose-Gel</A></LI><LI class="myParticipants">
			    		Participants: Max                      </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Ran a small 1.75% TAE-Agarose Gel at 150 V for 50 minutes and post stained with Sybr Gold        </LI></UL><LI class="myTitel"><B>Overnight Ligation of Intein-Extein-Construct</B></LI><LI class="myProtocol">		    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Ligation of DNA-Fragments</A></LI><LI class="myParticipants">
			    		Participants: Max                      </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Overnight ligation in ice bath with psb1C3 and psb4A5 as backbones and a 3:1 molar excess of insert        </LI><LI>
			    				200 ng backbone
			    			</LI><UL class="myEntry other"><LI class="myTitel"><B>Preparation of chemical competent Turbo cells</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Preparation of chemical competent cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Plated one aliquot as negative controll       </LI></UL><LI class="myResults"><H4>Results</H4>
                    No colonies on plates, therefore usefull competent cells.    </LI><UL class="myEntry readout"><LI class="myTitel"><B>Invitro Transcription IC3a RNA</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">In vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4>
                                          See results of DNAseI digestion and PCE from 06.09.17
			    	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Phenol/Chloroform total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT (continued)</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Denaturing PAGE for ssRNA</A></LI><LI class="myParticipants">
			    		Participants: Dawa, Jorge

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
                            Ran urea gel
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry other"><LI class="myTitel"><B>DNA Amplification Test using Recombinase Polymerase Amplification </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				DNA Amplification with RPA was tested using the provided Control Reaction by TwistDx and our BioBrick His6-TEV plasmid 
			    				incubation times of 40 minutes for Control and Sample 1 and 60 minutes for sample 2. Differences due to incubation time 
			    				was tested. 
			    			</LI><LI>
			    				RPA according to protocol
			    			</LI><LI>
			    				PCR cleanup according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>
                    Positive Control worked. RPA of TEV could not produce the whole (~1300 bp) construct but would produce amplicons of shorter size (100-500 bp)
                    which was anticipated since TwistDx protocol says optimal amplicon length is 500 bp. Full length double-stranded DNA could consequently not be formed. 
                                          </LI><DIV class="date">Wednesday 6th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> RNaseAlert Experiment with E.coli RNA with different concentrations of target E.coli RNA </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa, Jorge, Igor
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			Run1: Amount of RNA: 1 nm Cas13a, 10 nm crRNA
			</LI><LI>
			Run2: Amount of RNA: 1 nm Cas13a, 10 nm crRNA
			</LI><LI>
			img src=&quot;https://static.igem.org/mediawiki/2017/b/be/T--Munich--pic--platereader_pipScheme_6_09_Ecoli.png&quot; &gt;
			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Chem and electro Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                       </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Transformation into chem and e. competent E.coli Turbo and DH5 alpha cells.        </LI></UL><LI class="myResults"><H4>Results</H4>
                    No colonies in the evening, therefore the plates were left at room temperature overnight.    </LI><UL class="myEntry readout"><LI class="myTitel"><B>DNAse I digest and PCE of invitro transcribed IC3a RNA</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">Restriction digest,Phenol-chlorophorm purification of DNA</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>6 M UREA-PAGE 15% of IC3a RNA</LI></UL><DIV class="date">Thursday 7th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>RNA Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Colony PCR</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Picked 128 colonies from plates of 06.09.17        </LI><LI>
			    				4 strip tubes of each construct( psB1C3_C; psB1c3_N; psB4A5_c; psB4A5_N)
			    			</LI><LI>
			    				Colony PCR with OneTaq Polymerase
			    			</LI><LI> 
								Samples loaded onto monster Gel (Samples order is interlaced)
							</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry readout"><LI class="myTitel"><B>First try of Cas13a digestion of IC3a/C3a dimer over 1 hour</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">RNAse activity protocol</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Incubation for dimerization at room temperature for 1 h. Afterwards 1 hour digestion with Cas13a.</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>No real difference visible after digestion for 1 h.</LI><DIV class="date">Friday 8th</DIV><UL class="myEntry other"><LI class="myTitel"><B>Test heat-only lysis (1,3 and 7 minutes))</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Phenol-Chlorophorm extraction</A></LI><LI class="myParticipants">
			    		Participants: Julian

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Diluted E. coli culture to 10<SUP>8</SUP> cells/ml, used 300 uL as sample
			    		</LI><LI>
                            Heat-only at 93 °C for 15 min, diluted wiht tap water to simulate actual usage scenario (saliva etc..)
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
			    				    		Hardly any RNA detectable (concentration &lt; 10 ug/ml). Abysmal 260/280-ratio 
			    				    		-&gt; Heat-only lysis is too inefficient for that low cell counts, use 10<SUP>9</SUP> next time.
                        			    	</LI><DIV class="date">Monday 11th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols"> - </A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Precultures of 16 colonies from plates of 06.09.17.      </LI><LI>
			    				Precultures in 5 ml LB with corresponding antibiotic
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Second try of Cas13a digestion of IC3a/C3a dimer for several hours and overnight</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">RNAse activity protocol</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Tuesday 12th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Mini-Prep with Qiagen Kit and Sequencing Sample preparation </A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Mini Prep of precultures from 11.09.17   </LI><LI>
								Send all 16 plasmids for sequencing with primer VR
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
						psB4A5_C-Terminal 1-7 and 3-4 were positive.</LI><DIV class="date">Wednesday 13th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Restriction digest and PCR protocol with Q5 polymerase</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Send Samples psB4A5_C-Terminal 1-7 and 3-4 for sequencing with primer VF </LI><LI>
								New digestion of unamplified N-Terminal g-block (200 ng in 50 µl with EcoRI-HF and SpeI-HF)
			    			</LI><LI>
			    				PCR of psB1C3 with psb1A3_fw and psb1A3_rev , to get an opened backbone (Q5 polymerase protocol with 66 °C as annealing temerature and 1 minute elongation time)
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    PsB4A5_C terminal 1-7 was sequenced completly and used for further cloning steps.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Heat-lysis with E. coli</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Phenol-Chlorophorm extraction</A></LI><LI class="myParticipants">
			    		Participants: Julian

			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Lysis at 90°C, used 300 uL of E. coli suspension diluted to 2*10<SUP>9</SUP> cells/ml  
			    		</LI><LI>
			    		SDS-Lysis as reference was impossible with this cell count, interphase during extraction too broad 
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                        
                        Extraction-yield is down 5- to 10- fold  compared to SDS (calculated from earlier SDS-lysis results with 10<SUP>9</SUP> (not 2*10<SUP>9</SUP>)
                        </LI><UL class="myEntry other"><LI class="myTitel"><B>RPA stability test and Reaction Temperature Screening on paper </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification on Paper</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Preparation of stability assay of RPA on paper. Lyophilisation and storage.
			    			</LI><LI>
			    				RPA on paper according to protocol
			    			</LI></UL><DIV class="date">Thursday 14th</DIV><UL class="myEntry other"><LI class="myTitel"><B>RPA stability test and Reaction Temperature Screening on paper </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification on Paper</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Preparation of stability assay of RPA on paper. Lyophilisation and storage.
			    			</LI><LI>
			    				RPA on paper according to protocol
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Restriction digest, Ligation of DNA fragments, Gibbson-Assembly and E and chem Trafo</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				New digestion of unamplified N-Terminal gBlock and psB1C3 with SpeI-HF and EcoRI-Hf   </LI><LI>
								Dephosphorylation with Antarctic Phosphatase
			    			</LI><LI>
			    				Ligation of digested sequences overnight with T4 DNA Ligase 
			    			</LI><LI>
								Gibbson-Assembly of psB1C3with N-Term fragment
							</LI><LI>
							Transformation of chem and e. competent Turbo cells
							</LI></UL><DIV class="date">Friday 15th</DIV><UL class="myEntry other"><LI class="myTitel"><B>RPA stability test and Reaction Temperature Screening on paper </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification on Paper</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Stability of RPA on paper was tested using His6-TEV plasmid after 0, 2 and 24 hours post-lyophilisation. Reaction temperature was tested at 37 °C 
			    				as suggested by TwistDx and room temperature. Reaction was positively controlled using non-lyophilised reaction mixture. 
			    			</LI><LI>
			    				RPA according to protocol
			    			</LI><LI>
			    				PCR cleanup according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Lyophilisation of RPA on paper worked.</LI><LI>Directly after lyophilisation, the reaction
                    efficiency seems to be similar to fresh RPA reaction mixture.</LI><LI>Lyophilisation of RPA on paper worked. However, activity loss is observed already after 2 hours and after 24 hours, 
                    almost no activity is observable anymore. </LI><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Colony PCR</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Picked 32 colonies from plates of 14.09.17 </LI><LI>
								Screened with Colony PCR 
			    			</LI><LI>
			    				Gel 200 V 2 % Agarose 30 minutes
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>No positive colononies in the Col PCR.</LI><UL class="myEntry other"><LI class="myTitel"><B>Phenol/Chloroform total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Phenol/Chloroform purification of RNA</A></LI><LI class="myParticipants">		    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Two samples stored at -80 ºC labeled TRNA Lsh #1 15.09.17 and TRNA Lsh #2 15.09.17
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Monday 18th</DIV><UL class="myEntry other"><LI class="myTitel"><B>RPA stability test on paper </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification on Paper</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Preparation of temperature and air-sensitivity assay of RPA on paper. 
			    			</LI><LI>
			    				RPA on paper according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Lyophilisation of RPA on paper.</LI><LI>Storage with and without Parafilm enclosure at 4 °C or room temperature.</LI><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Q5-PCR and Restriction digestion and Col-PCR and e and chem comp Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 New PCR of backbone with psB1A3-fw and psB1A3-rev. Sample was subsequently DpnI digested and purified    </LI><LI>
			    			Col-Pcr of 16 non green colonies from plates from thursday 14.9.17
			    			</LI><LI>
			    			Transformation of cells with ON Ligation from Friday 15.9.17 into E.coli DH5 alpha and E.coli Turbo 
			    			</LI></UL><DIV class="date">Tuesday 19th</DIV><UL class="myEntry other"><LI class="myTitel"><B> RNA extraction from 5ml overnight culture of B. subtilis  </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">Phenol-chlorophorm extraction RNA</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			used heat lysis at 90 °C for 15 mins
			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Coupling <I>In-Vitro Transcription</I> to RPA </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Coupling RPA to In-Vitro Transcription</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Transcription efficiency when coupled to RPA was determined using a reaction without T7 RNA Polymerase as negative control.
			    				The construct as template used was His6-TEV plasmid. 
			    		 
			    			</LI><LI>
			    				RPA-Tx according to protocol.
			    			</LI><LI>
			    				Phenol-Chloroform-Extraction according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>In-Vitro Transcription could not be shown.</LI><LI>RNA was detected in the samples in which no T7 RNA Polymerase was added while there are no
                   	bands in the samples that had T7 RNA Polymerase in them. Though unlikely, a labelling error might have occured. </LI><UL class="myEntry other"><LI class="myTitel"><B>RPA stability test on paper </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Recombinase Polymerase Amplification on Paper</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Stability of RPA on paper was tested using His6-TEV plasmid after 24 hours post-lyophilisation. Storage conditions were altered.
			    				Storage at room temperature and at 4 °C were tested. Also, duplicates were done at these temperatures, one sample being always enclosed 
			    				with Parafilm to avoid air exposure. 
			    		 
			    			</LI><LI>
			    				RPA on Paper according to protocol
			    			</LI><LI>
			    				PCR cleanup according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>RPA activity clearly showed that storage temperature did not have a significant effect on the integrity of the RPA reaction mixture.</LI><LI>Exposure to air, however, was identified as the bottleneck for RPA stability. The RPA samples that were protected from exposure to air showed significantly higher</LI><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout and Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Gibbson-Assembly and Ligation</A></LI><LI class="myParticipants">
			    		Participants: Max, Rob                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Gel of Col-PCR from 18.09.17     </LI><LI>
			    			Clean-up from backbone PCR and digest from 18.09.17 followed by	Gibbsonassembly with N-terminal insert.
			    			</LI><LI>
							Started the Recloning of the Biobricks by PCR with phosphorylated primers and T4 Ligation and transformed into Turbo chem competent cells.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Sample 1-3 and 2-1 had the correct length, and therefore precultures of these colonies were started overnight.</LI><UL class="myEntry readout"><LI class="myTitel"><B>IC3a invitro transcription</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">In vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4>
                                           See results of DNaseI digestion and PCE from 20.09.17.
			    	</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Heat-lysis with B. subtilis</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">Phenol-Chlorophorm extraction</A></LI><LI class="myParticipants">
			    		Participants: Julian
			    	</LI><LI class="myObservations">
			    		Notes:
			    		<UL>
			    		2x SDS-Heat lysis (protocol), 2x heat-only at 90 °C for 20 min
			    		</UL></LI><LI class="myResults"><H4>Results</H4>
			    	SDS-lysis  works a lot worse for gram+ bacteria, Heat-only now better/same than SDS-lysis (more incubation time...)
			    	A260/280 ratio relatively bad, probably a lot of cell-wall impurities
                    </LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Heat lysis and isothermal PCR: temperature dependence</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">RPA-pcr</A></LI><LI class="myParticipants">
			    		Participants: Julian, Sven
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Diluted E. coli (DH5a-pSB1C3-Tev-008-His) suspension to 10<SUP>6</SUP> cells / ml and heat-lysed in 50 uL aliquots
			    		</LI><LI>
			    		Lysis-temperature ranged from 65 °C to 90 °C in 5 °C steps
			    		</LI><LI>
			    		Added 2 uL of lysed sample to 10 uL RPA-solution and TEV-protease primers
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
			    	Heat differences in this range and this cell concentration have only minor effect. Best lysis seems to be around 80 °C, which is consistens with literature.
			    	</LI><DIV class="date">Wednesday 20th</DIV><UL class="myEntry other"><LI class="myTitel"><B> Gel analysis of B.subtilis RNA samples </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">Denaturating Urea PAGE</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			 8% gel
			</LI></UL><LI class="myResults"><H4>Results</H4>
	 B. subtilis-PCE 190 ng/ul, B.subtilis heat/lysis-378 ng/ul
    </LI><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Miniprep with Qiagen Kit</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Minipreped overnight cultures from 1-3 and 2-1.   </LI><LI>
								Send samples for sequencing with VF and VR.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                       N-Terminal sample 1-3 was positive.  </LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Col PCR and preparation of e comp. cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Colony PCR of  36 colonies from each of the 3 constructs His-Sumo-Lwa (Lwa(l)), Lwa(lwa(k) and His-Sumo-Lbu each in psB1C3.   </LI><LI>
								Precultures of 4 colonies from Lwa(l) and Lwa(k)
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry readout"><LI class="myTitel"><B>DNAse I digest and Phenol-Chloroform-Precipitation</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">Restriction digest,Phenol-chlorophorm purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>6 M UREA-PAGE of invitro transcribed and pooled IC3a.</LI></UL><UL><LI>Quantitative gel analysis calculated a concentration of 35 µM.</LI></UL><DIV class="date">Thurdsay 21st</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Restriction digest, Gibbson-Assembly and Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Digestion of 1 µg of each backbone (psb1c3_N-term_uni and psb4A5_C-Term_uni with BBS1-HF to open them for Gibbson-Assembly </LI><LI>
								Gibbson-Assembly of Col_PCR fragments from 03.08.2017
			    			</LI><UL><LI>
									1:3 backbone to insert concentration
								</LI><LI>
									30 minutes at 50 °C
								</LI></UL><LI>
			    				Transformation of E.coli Turbo e and chem competent cells according to protocol
			    			</LI><LI>
								Preculture for Cryostock from psB1C3_Split_uni_N_term 1-3
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Col-PCR and Sample Sequencing</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Send samples for sequencing   </LI><LI>
								Col_PCR of 4 samples from Lbu plate (all negative)
			    			</LI></UL><DIV class="date">Wednesday 20th</DIV><UL class="myEntry cs13a"><LI class="myTitel"><B> RNase Alert Experiment with Bacillus subtilis using varying concentrations of target RNA </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			Pippeting scheme: </LI></UL><DIV class="date">Thursday 21th</DIV><UL class="myEntry other"><LI class="myTitel"><B> B. subtilis RNA extraction using lysozyme </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">Phenol-chlorophorm RNA extraction</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI></UL><DIV class="date">Monday 25th</DIV><UL class="myEntry other"><LI class="myTitel"><B> crRNA in vitro transcription (for B.subtilis, Norovirus. Q5 beta, Hepatitis)
 </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">In vitro Transcription</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Restriction digest, PCR Cleanup Kit, Gibbson Assembly and Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Redigest of 1 µg of each backbone with BBS1 and dephoshorilation according to protocol
								</LI><LI>
								Cleanup with cleanup kit
			    			</LI><LI>
			    				Gibbson assembly with 100 µg backbone and 3:1 excess of insert ( calculated based on length ratio) Inserts 2 and 3 from PCR of 05.08.17
			    			</LI><LI>
			    				Transformation into E.coli Turbo e. comp
			    			</LI><LI>
			    				COlPCR of plates from Thursday. 21.9.17=&gt; 36 samples 
			    			</LI><LI>
			    				Gel at 1 % 45 mins 150 V
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Total</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				COlPCR and Gel of samples from trafo of Friday 22.09.17 =&gt; 30 samples (all non green) psB1C3_LBU   </LI><LI>
								Gel at 1 % 45 mins 150 V
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    All negative.
			    	</LI><DIV class="date">Tuesday 26th</DIV><UL class="myEntry other"><LI class="myTitel"><B> crRNA quantification, Urea gel run </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">Denaturating Urea Page</A></LI><LI class="myParticipants">
        Participants: Dawa, Erika
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			2 samples each from B. subtilis, Norovirus, Q5 beta and HCV(first diluted 1:5 and second undiluted) 
			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">ColPCR, Agarosegel</A></LI><LI class="myParticipants">
			    		Participants: Max, Milica                   </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				ColPCR of 12 colonies from N-term plate from 25.9.17    </LI><LI>
								Gel at 1 % 30 mins 150 V
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
					Samples 1-8 and 2-1 of the N-Terminus looked good =&gt; preculture for sequencing.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Total</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				COlPCR and Gel of samples from trafo of Friday 22.09.17 =&gt; 30 samples (all non green) psB1C3_Lwa_kurz   </LI><LI>
								Gel at 1 % 30 mins 150 V
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    All negative.
			    	</LI><DIV class="date">Wednesday 27th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Minipreparation and Sample Sequencing</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Miniprep of  psB1C3_Split_N_term_bGal 1-8 and 2-1   </LI><LI>
								Send  Minipreps for sequencing with primer  VF and VR
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    Both sequences positive.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Phenol/Chloroform total RNA purification from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">Phenol/Chloroform purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Four samples stored at -80 ºC labeled TRNA Lsh #1 27.09.17 and TRNA Lsh #2 27.09.17, TRNA Lsh #3 27.09.17, TRNA Lsh #4 27.09.17.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Thursday 28th</DIV><UL class="myEntry other"><LI class="myTitel"><B> Agarose and Urea gel run with RNA </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">PAGE, Urea PAGE</A></LI><LI class="myParticipants">
        Participants: Dawa, Erika
	</LI><LI class="myResults"><H4>Results</H4>
	Agarose Gel didn’t work – The samples seems to be degraded in agarose gel , may be due to contamination of the gel chamber and other materials with Rnase.
    </LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">ColPCR, ON culture</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				ColPCR from samples of 22.09.17    </LI><LI>
								32x Lwa short 
			    			</LI><LI>
			    				Overnight cultures from colonies 1-8 and 2-3 started 
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry readout"><LI class="myTitel"><B>RNA/DNA hybrid dimerization with Cy5 DNA-oligo C3a</B></LI><LI class="myProtocol">
                                       Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>Native PAGE 20%, 12.5 mM MgCl2</LI></UL><UL><LI>C3a too high concentrated (Red bands). 5 µM instead of 500 nM. Better results with band shift see 29-09-2017.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Screen of LBU activity on RNA/DNA dimer</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">RNAse activity protocol</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>Native PAGE 20%, 12.5 mM MgCl2</LI></UL><UL><LI>No visible shift of activator DNA oligo with addition of RNA under the use of Cy5 modified DNA activator oligo. Results are bad because of too high C3A DNA oligo concentration. (C3a = red bands).</LI></UL><DIV class="date">Friday 29th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> RNaseAlert experiment with extracted RNA targets from E.coli and in vitro transcribed RNAs. </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa, Erika
	</LI>
    Notes:
		<UL><LI>
			Pipetting scheme: </LI></UL><LI class="myResults"><H4>Results</H4>
	The results for E.coli look good. Excel table saved in lrz as: 2017_09_29_rnase alert_ecolitarget_invitrotarget.xlsx
    </LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Mini-Prepatation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
								Mini prep of 1-8 and 2-3.
			    			</LI><LI>
			    				Lwa short send for sequencing with VF and VR    </LI></UL><LI class="myResults"><H4>Results</H4>
                    Lwa long positive.
			    	</LI><UL class="myEntry readout"><LI class="myTitel"><B>DNA and RNA hybrid of 3C-A and IC3-A</B></LI><LI class="myProtocol">
                                       Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>Native PAGE 20%, 12.5 mM MgCl2</LI></UL><UL><LI>Visible small up shift of RNA/DNA dimer versus samples, which only contain the DNA oligo by itself. (DNA oligo C3a marked with Cy5 visible as red bands.)</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>FINA extraction for RNA with purified total RNA (TRNA Lsh #2-4 from 27.09.17)</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">FINA Extraction for RNA</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				FINA was performed with 25 ul sample instead of 50 ul.
			    			</LI><LI>
			    				The extraction for TRNA #2 was performed as in protocol.
			    			</LI><LI>
			    				Before the FINA extraction, TRNA #3 was digested with DNase I according to <A href="DNase">DNase I Reaction Protocol</A></LI><LI>
			    				After the FINA extraction, the two eluate from TRNA #4 were digested with DNase I according to <A href="DNase">DNase I Reaction Protocol</A></LI></UL><UL class="myEntry other"><LI class="myTitel"><B>First strand cDNA synthesis from <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT FINA-RNA eluate</B></LI><LI class="myProtocol">
			    		Protocol: <A href="#">First strand cDNA synthesis</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations
			    		</LI><LI>
			    				pseq-Lsh-06rev was used as primer.
			    			</LI><LI>
			    				The eluates from the FINA extractions for RNA from 29.09.17 were used as samples. 
			    			</LI><LI>
			    				From each pair of samples, one was used as a negative control (nuclease free water instead of MuLV-RT).
			    			</LI><LI>
			    				For all reactions, 6 ul sample were pipetted.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Q5-PCR for Lsh-fragment in <I>E. coli</I> Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT  with cDNA samples from 29.09.17</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="#">PCR amplification with Q5 Master Mix</A></LI><LI class="myParticipants">
			    		Participants: Jorge
			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>pseq-Lsh-05fwd was used as forward primer.</LI><LI>pseq-Lsh-06rev was used as reverse primer.</LI><LI>Annealing was performed at 58 ºC.</LI><LI>The amplification step was 110 s long.</LI><LI>A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products. 
</LI><LI>Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>N: FINA extraction performed as in protocol.</LI><LI>NK: FINA extraction as in protocol. No RT in cDNA synthesis (negative control).</LI><LI>P: Digestion of DNA before FINA extraction.</LI><LI>PK: Digestion of DNA before FINA extraction. No RT in cDNA synthesis (negative control).</LI><LI>A: Digestion of DNA after FINA extraction.</LI><LI>AK: Digestion of DNA after FINA extraction. No RT in cDNA synthesis (negative control).</LI></DIV><DIV class="month" id="October"><DIV class="date fistOfMonth">Monday 2nd</DIV><UL class="myEntry other"><LI class="myTitel"><B> Phenol chloroform purification of crRNAs of B. subtilis, NoroV, HCV and Q5 beta
 </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">Phenol-chloroform extraction RNA</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Live Invitro Transcription test with LBU and Inhibitor-RNA/Activator-DNA dimer with different T7 concentration</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">In vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI>Forgot to add C3ds with single strand overhang, which is needed for activation of invitro transcription after binding of DNA oligo C3a (activator of C3ds).</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>Plate reader data showed no activity for all samples, as expected without DNA template to transcribe.</LI><DIV class="date">Tuesday 3rd</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> various Rnase alert experiments </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			with Noro Virus (10 ul total volume used for the experiment) </LI><LI>
			with Q5 beta (10 ul total volume used for the experiment)
			</LI><LI>
			with Bacillus ( 10 ul total volume used for the experiment) and 
			</LI><LI>
			with HCV and E.coli RNA (FINA method) samples (10 ul total volume used for the experiment)
			</LI><LI>
			Pipetting Scheme Norovirus and Q5 beta: </LI><LI>
			Pipetting Scheme JVC and B. subtilis: </LI><LI>
			Pipetting Scheme E.coli RNA (FINA method): </LI></UL><LI class="myResults"><H4>Results</H4>
	Bacillus, Noro virus and HCV seeem to work. However Q5 beta needs to be repeated and also HCV could be repeated!
    </LI><UL class="myEntry other"><LI class="myTitel"><B> FINA extraction of RNA from E. coli with and without filters </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">FINA RNA extraction</A></LI><LI class="myParticipants">
        Participants: Jorge
	</LI><LI class="myResults"><H4>Results</H4>
	Works, Target-sequence with PCR detected.
    </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Live Invitro Transcription test with LBU and Inhibitor-RNA/Activator-DNA dimer with different T7 concentration</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">In vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>No Invitro transcription of RNA at any T7-Polymerase concentration. LBU does either not set activator DNA free or does not work at all. Same result as on the gel.</LI><LI>RNAse H shows low level fluorescence over the whole experiment, which can not be related to an invitro transcription.</LI></UL><DIV class="date">Wednesday 4th</DIV><UL class="myEntry other"><LI class="myTitel"><B> Bacillus RNA extraction </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Erika, Dawa
	</LI><LI class="myObservations">
    Notes:
		<UL>

			Resuspension Buffer (20 ml):
			</UL></LI><LI>
			Sucrose (0.3 M) -&gt; 2.1 g
			</LI><LI>
			Sodium Acetate (10 mM) -&gt; 16.4 mg

			</LI><LI>
			Adjust pH to 4.2

			</LI><LI>
			Stored at 4°C

			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Send Lwa short colony 2-3 for sequencing with pseqLwa primer   </LI></UL><LI class="myResults"><H4>Results</H4>
						Full read with no mutations.
			    	</LI><DIV class="date">Thursday 5th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> RNaseAlert </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants:  Dawa, Rob, Erika
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			RNaseAlert reaction mix ( in glass fiber filter paper) lyophilisation. Introduction and first trial
			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Transformation of e. comp cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Repeated Gibbson-Assembly for Lbu with 4:1 ratio</LI><LI>
								Transformation into E.coli Turbo
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    
				Empty plates =&gt; need to repeat PCR.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>In-vitro transcription of AuNP linkers U0, U5, U10, U15</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">in vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI></UL><DIV class="date">Friday 6th</DIV><UL class="myEntry other"><LI class="myTitel"><B> RNA extraction from Bacillus overnight culture </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">Phenol-chlorophorm extraction</A></LI><LI class="myParticipants">
        Participants: Erika
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			following SIGMA Genosys Protocol from Step 1 to 7. Then, followed Lab´s protocol steps. 
			</LI><LI>
			RNA sample stored at -80°C and needs to be quantified.
			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>PCE and gel analysis of of AuNP linkers U0, U5, U10, U15 and new in vitro transcription</B></LI><LI class="myProtocol">
							Protocol:  <A href="/Team:Munich/Protocols">phenol-chlorophorm purification of RNA</A></LI><LI class="myProtocol">
							Protocol:  <A href="/Team:Munich/Protocols">Urea gel for RNA detection </A></LI><LI class="myProtocol">
							Protocol:  <A href="/Team:Munich/Protocols">in vitro transcription</A></LI><LI class="myParticipants">
							Participants: Rob

						</LI><LI class="myObservations">
							Observations: 
							</LI><LI>
									There was no RNA visible.
								</LI><LI>
									 The in-vitro transcription was repeated.
								</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Saturday 7th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>SybrGreen II sensitivity test</B></LI><LI class="myProtocol">
                                       Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>1:10.000 seems to be the right dilution of SybrGreen II dye for a 15 µl Sample.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>PCE and gel analysis of of AuNP linkers U0, U5, U10, U15</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">phenol-chlorophorm purification of RNA</A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Urea gel for RNA detection </A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">in vitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
			    				There was no RNA visible.
			    			</LI><LI>
			    				 The in-vitro transcription was repeated with a new batch of T7 RNA polymerase.
			    			</LI></UL><DIV class="date">Sunday 8th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>IC3A invitro transcription</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">Invitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4>
                                          See results of Digestion and PCE from 09.10.17.
			    	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>T7 polymerase sensitivity test</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">Invitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>Higher concentrations of T7-polymerase causes the maximal concentration of RNA to be reached faster.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>PCE and gel analysis of of AuNP linkers U0, U5, U10, U15</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">phenol-chlorophorm purification of RNA</A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Urea gel for RNA detection </A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP preparation</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
			    				The IVTX was successful.
			    			</LI><LI> 
							3-day incubation of AuNP from stock was started.
						&lt;\li&gt;
			    			
			    		</LI></UL><LI class="myResults"><H4>Results</H4></LI><DIV class="date">Monday 9th</DIV><UL class="myEntry other"><LI class="myTitel"><B> Quantification of viral RNAs and Bacillus RNA extracted from last week </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNA urea gel</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI></UL><UL class="myEntry cas13a"><LI class="myTitel"><B>Lbu-Recloning with Gibbson-Assembly</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">PCR,Gibbson and Trafo in chem and e comp</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				PCR for Gibbson with Biobrick fwd and rev primer.    </LI><LI>
								Lbu Gibbson Assembly with 3x excess of insert.
			    			</LI><LI>
			    				Trafo into E. coli chem and e comp.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
						No colonies
			    	</LI><UL class="myEntry readout"><LI class="myTitel"><B>DNAse I digest and PCE of 4 pooled invitro transcriptions of IC3a</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">Restriction digest,Phenol-chlorophorm purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>6 M UREA-PAGE 15% of IC3a RNA</LI></UL><DIV class="date">Tuesday 10th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">PCR and Gibbson Assembly</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				PCR of Lbu and psB1C3 as backbone   </LI><LI>
								Gibbson Assembly with 5:1 excess of insert
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry other"><LI class="myTitel"><B>Heat lysis and isothermal PCR: cell concentration dependence s</B></LI><LI class="myProtocol">
                        Protocol:  <A href="#">RPA-pcr</A></LI><LI class="myParticipants">
			    		Participants: Julian, Sven
			    	</LI><LI class="myObservations">
			    		Notes:
			    		</LI><LI>
			    		Diluted E. coli (DH5a-pSB1C3-Tev-008-His) suspension in a range of 10<SUP>6</SUP> to 10<SUP>2</SUP> cells / ml and heat-lysed in 50 uL aliquots
			    		</LI><LI>
			    		Added 2 uL of lysed sample to 10 uL RPA-solution and TEV-protease primers
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
			    	Bands visible down to a dillution of 5*10<SUP>4</SUP> or 10<SUP>4</SUP> cells/ml, which corresponds to an average of 100 or 20 cells' DNA per PCR
			    	</LI><DIV class="date">Wednesday 11th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Plate reader experiments </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa, Jorge
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			with Noro virus samples ( worked!)
			</LI><LI>
			with Bacillus ( seems to have Rnase contamination)
			</LI><LI>
			with the paper strips (glass fiber filter paper) 1.8 ul reaction mix pipetted with different amounts of invitro transcribed RNAs.
			</LI><LI>
			with E.coli RNA samples( heat lysed)- ( seems to have RNase contamination)
			</LI><LI>
			Pipetting Scheme 1: </LI><LI>
			Pipetting Scheme 2: </LI></UL><UL class="myEntry other"><LI class="myTitel"><B>Coupling <I>In-Vitro Transcription</I> to RPA in Bulk and on Paper</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Coupling RPA to In-Vitro Transcription</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Transcription efficiency when coupled to RPA was determined using a reaction without T7 RNA Polymerase as negative control.
			    				The construct as template used was RNA Amp G-Block 200 bp double-stranded DNA using VF and VR as primers. 
			    		 
			    			</LI><LI>
			    				RPA-Tx and RPA-Tx on Paper according to protocol.
			    			</LI></UL><UL class="myEntry other"><LI class="myTitel"><B>G-Block cloning of RNA-Amplification test plasmid via Restriction Cloning and Gibson Assembly</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">G-Block Cloning</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Preparation: 
			    				Solubilisation of G-Block to reach 10 ng/ul concentration. 
			    				PCR reaction using VF and VR as primers resulted in 50 ul of a 24 ng/ul solution. </LI><LI>
			    				Gibson Assembly: 
			    				PCR reaction using pSB1A3_fw and pSB1A3_rev for backbone amplification of pSB1C3-GFP yielded 54 ng/ul. 
			    				Ratio for Gibson Assembly: 0.375 pmol Insert : 0.125 pmol backbone. Incubation for 15 minutes at 50 °C.
			    			</LI><LI>
			    				Restriction Cloning:
			    				Restriction of backbone and insert using EcoRI-HF and SpeI-HF in NEB Cutsmart Buffer and heat inactivation. 
			    				Subsequent dephosphorylation of backbone using Antartic Phosphatase. 
			    				T4 DNA Ligation with a ratio of 0.02 pmol backbone : 0.06 pmol insert. </LI><LI>
			    				Transformation of Ligation and Gibson Assembly into chemocompetent and electrocompetent Turbo cells. Plate-out on 
			    				chloramphenicol agar plates.
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                   	The investigation of the agar plates the next day using UV-light revealed that the only plasmid that was taken up by the Turbo cells
                   	was the initial pSB1C3-GFP plasmid. No positive inserts could be identified.         
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Transformation of chem comp cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
								Transformation of Turbo cells with Gibbson of 10.10.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Dye Test for live imaging of invitro transcription (1st row Controls (with SybrGold partially), 2nd row SybrGold dilution series, 3rd row SybrGreen I dilution series, 4th row EvaGreen dilution series)</B></LI><LI class="myProtocol">
                                       Protocol: <A href="/Team:Munich/Protocols">invitro transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI><LI class="myResults"><H4>Results</H4></LI><LI>6 M UREA-PAGE 15% of IC3a RNA</LI></UL><UL><LI>Mistake made: Added T7 Polymerase to Control 2. Was left out in Graph.</LI><LI>Only Eva Green seems to not influence the invitro transcription, because it was especially designed for live imaging with ongoing transcription reactions.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Cas13a activity assay</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Cas13a activity assay</A></LI><LI class="myParticipants">
			    		Participants: Igor
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Preparation and DNA-conjugation of of new AuNP</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP preparation</A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP-DNA conjugation</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI> Preparation of AuNP was successfully completed.
			    				
			    			</LI><LI> 
							DNA-conjugation with &quot;AuNP 5' primer&quot; and &quot;AuNP 3' primer&quot;


						&lt;\li&gt;
			    			
			    		</LI></UL><LI class="myResults"><H4>Results</H4>
                 			See results from &quot;conjugation test&quot;
			    	</LI><DIV class="date">Thursday 12th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Plate reader experiment with paper strips blocked ovenight with BSA </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			1.8 ul of reaction mix pipetted in each paper with different amounts of in vitro target RNA.
			</LI><LI>
			The overnight blocked paper strips were rinsed with nuclease free water and then dried in the oven at 70 degree for 30 mins and then used for the experiments. 
			</LI><LI>
			Pipetting scheme: </LI></UL><LI class="myResults"><H4>Results</H4>
	Seems to work! Needs to be repeated for the confirmation of results.
    </LI><UL class="myEntry other"><LI class="myTitel"><B>UREA-PAGE of Coupling <I>In-Vitro Transcription</I> to RPA in Bulk and on Paper</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Coupling RPA to In-Vitro Transcription</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Transcription efficiency when coupled to RPA was determined using a reaction without T7 RNA Polymerase as negative control.
			    				The construct as template used was RNA Amp G-Block 200 bp double-stranded DNA using VF and VR as primers. 
			    		 
			    			</LI><LI>
			    				RPA-Tx and RPA-Tx on Paper according to protocol.
			    			</LI><LI>
			    				UREA-PAGE Gel of Samples extracted via Phenol-Choloform-Extraction; both according to protocol.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>T7 RNA-Polymerase was, by accident, also added to the negative control of the RPA-Tx reaction on Paper. </LI><LI>First, one can see that the gel was overloaded. Also, it seems like there is additional bands at the expected size of around 150 bp.</LI><LI>Questionable is that this seems also to be the case for the negative Control of the RPA-Tx reaction in Bulk. </LI><LI>The experiment on paper, however, looks quite promising since it seems like there are distinct bands at the expected approximate size of the RNA transcript. Cas13a cleavage assay will provide evidence if these bands actually consist of the desired target RNA. 
            </LI><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Transformation of e. comp cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Transformation of BL21 with psB4A5 C-Term and psB1C3 N-Term   </LI></UL><LI class="myResults"><H4>Results</H4>
                    No colonies for psB4A5 C-Term.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Col-PCR for Biobrick Recloning</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Col PCR of 6x N-Term in psB1C3, 16x C-Term in psB1C3, 14x Lbu in psB1C3    </LI></UL><LI class="myResults"><H4>Results</H4>
                    All negative.
			    	</LI><UL class="myEntry readout"><LI class="myTitel"><B>Eva Green Sensitivity Test with RNA background</B></LI><LI class="myProtocol">
                                       Protocol: 
			    	</LI><LI class="myParticipants">
			    		Participants: Florian
			    	</LI><LI class="myObservations">
			    		Observations:

			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>AuNP linkage asssay with DNA-linker</B></LI><LI class="myProtocol">  		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
			    				1x Buffer was used first, which resulted in no aggregation. Increasing the concentration by adding buffer to a final concentration of 2x afterwards resulted in aggregation and thus was set as the standard.
			    			</LI></UL><DIV class="date">Friday 13th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Plate reader experiment with paper strips blocked ovenight with BSA </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Rob
	</LI><LI class="myObservations">
	 

    Notes:
		</LI><LI>
			1.8 ul of reaction mix pipetted in each paper with different amounts of in vitro target RNA.
			</LI><LI>
			The overnight blocked paper strips were rinsed with nuclease free water and then dried in the oven at 70 degree for 30 mins and then used for the experiments. (Results seem to work!)
			</LI></UL><LI>
			Pipetting scheme: </LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">PCR with Q5-Polymerase</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Repeated PCR of N and C-term part for Recloning    </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>AuNP linkage asssay with DNA-linker and varying buffer concentrations</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							2x, 4x, and 6x buffer was used, which resulted in unspecific aggregation from 4x to 6x buffer, thus 2x was kept as standard.
			    			</LI></UL><DIV class="date">Saturday 14th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Restriction digest Gibbson-Assembly and Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				DpnI digest of PCR from 31.10.17    </LI><LI>
								Clean-up with PCR purification kit
			    			</LI><LI>
			    				Gibbson-Assembly with 3:1 excess
			    			</LI><LI>
			    				Transformation into E.coli Turbo chem comp.
			    			</LI></UL><DIV class="date">Sunday 15th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> RNA extraction of E. coli and Bacillus + plate reader experiment</B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa, Jorge, Erika
	</LI><LI class="myObservations">
    Notes:
		</LI><LI>
			HCV assay and Q5beta in Fluorostar and Clariostar for 1 hour each (HCV assay worked!)
			</LI><LI>
			RNA urea gel for RNA extracted from E.coli and Bacillus
			</LI><LI>
			Plate reader experiment of RNA extracted from the E.coli in Fluorostar for 2 hours
			</LI><LI>
			Pipetting scheme: </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Transformation of e. comp cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Repeated Transformation of Bl21 star with psB4A5-Split-Intein-C-Term    </LI></UL><LI class="myResults"><H4>Results</H4>
                    No colonies.=&gt; Transformation in other expression strains 
			    	</LI><DIV class="date">Monday 16th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative controls. Plus assay with cas13a with TDPs and assay with RPA paper samples.
 </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI class="myObservations">
    Pipetting scheme: </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Transformation of e. comp cells</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Transformed all available Bl21 strains with psB4A5-C-Term    </LI><LI>
								Strains were:
			    			</LI><LI>
			    				Bl21 pLYs; BL21 star; BL21 DE3, BL21 RIPL, Rosetta and E.coli Turbo as pos control
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
                    No colonies for any BL21 strain. The Turbo cells grew fine. =&gt; Expression seems to be toxic.
			    	</LI><UL class="myEntry readout"><LI class="myTitel"><B>New DNA-conjugation and AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP-DNA conjugation</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							New AuNP were DNA-conjugated
			    			</LI><LI>
							Water was used instead of linker-RNA.			    			</LI></UL><DIV class="date">Tuesday 17th</DIV><UL class="myEntry cas13a"><LI class="myTitel"><B> Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative controls. Plus assay with cas13a with TDPs and assay with RPA paper samples.
 </B></LI><LI class="myProtocol">
        Protocol:  <A href="#">RNase activity</A></LI><LI class="myParticipants">
        Participants: Dawa
	</LI><LI>
			Pipetting Scheme B. subtilis:</LI><LI>
			Pipetting Scheme RPA paper:</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Intein-Extein-Readout</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">-</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				 Midipreparation of psB4A5-C-Term with Macherey-Nagel Xtra Midi   </LI><LI>
								Preparation done according to protocol
			    			</LI><LI>
			    				Plasmid was intended for In vitro Expression
			    			</LI></UL><LI class="myResults"><H4>Results</H4>
						In vitro expression was not done, due to lack of time.
			    	</LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Q5-PCR, Gibbson-Assembly and Transformation</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				PCR of all GFP degradation tag library variants with Biobrick sequence binding primers, for transfer from psB1A3 to psB1C3.   </LI><LI>
								Gibbson-Assembly and Transformation according to protocol
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry"><LI class="myTitel"><B>AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							The experiment from the day before resulted in unspecific aggregation in the negative control. To have comparable samples and negative control for the cleavage assay, the experiment was repeated with a mock-DNA in the negative control.
 repeated with mock-RNA as negative control. A possible explanation for this is that negatively charched, non-bound RNA in the solution increases repulsion of AuNP.
			    			</LI></UL><DIV class="date">Wednesday 18th</DIV><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Overnight culture</A></LI><LI class="myParticipants">
			    		Participants: Max                     </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Picked 2 green colonies from each plate for overnight culture   </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>observation of AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							After cooling over night and spinning down for 1 min, just minor aggregation was achieved and samples were kept at 4°C for 2 days to allow for potential further aggregation.
			    			</LI></UL><DIV class="date">Thursday 19th</DIV><UL class="myEntry other"><LI class="myTitel"><B>G-Block cloning of RNA-Amplification test plasmid via Restriction Cloning and Gibson Assembly</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">G-Block Cloning</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Preparation: 
			    				PCR reaction of G-Block using VF and VR as primers resulted in 50 ul of a 27 ng/ul solution. </LI><LI>
			    				Gibson Assembly: 
			    				PCR reaction using pSB1A3_fw and pSB1A3_rev for backbone amplification of pSB1C3-GFP yielded 52 ng/ul. 
			    				Ratio for Gibson Assembly: 0.375 pmol Insert : 0.125 pmol backbone. Incubation for 15 minutes at 50 °C.
			    			</LI><LI>
			    				Restriction Cloning:
			    				Restriction of backbone and insert using EcoRI-HF and SpeI-HF in NEB Cutsmart Buffer and heat inactivation. 
			    				Simultanous dephosphorylation of backbone using Antartic Phosphatase. 
			    				T4 DNA Ligation with a ratio of 0.02 pmol backbone : 0.06 pmol insert. 
			    			</LI><LI>
			    				Transformation of Ligation and Gibson Assembly into chemocompetent and electrocompetent Turbo cells. Plate-out on 
			    				chloramphenicol agar plates.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>The investigation of the agar plates the next day using UV-light showed that for the Gibson Assembly, positive clones were visible. </LI><LI>Colony PCR was performed and showed inserts for several plasmids.</LI><LI>Overnight cultures were inocculated for sequencing.</LI><UL class="myEntry other"><LI class="myTitel"><B>Biobrick Recloning</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">Sequencing</A></LI><LI class="myParticipants">
			    		Participants: Max              </LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Send samples from degradation tag library for sequencing   </LI></UL><LI class="myResults"><H4>Results</H4>
                    pdt2-E 2 was positive
			    	</LI><DIV class="date">Friday 20th</DIV><UL class="myEntry other"><LI class="myTitel"><B>G-Block cloning of RNA-Amplification test plasmid</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">G-Block Cloning</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Mini-prep of previously inocculated overnight cultures and sent to sequencing.
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>The investigation of the agar plates the next day using UV-light showed that for the Gibson Assembly, positive clones were visible. 
</LI><UL class="myEntry readout"><LI class="myTitel"><B>In vitro transcription of crRNA Lbu, crRNA Lsh and target RNA at 37 ºC overnight</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">In vitro Transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian

			    	</LI><LI class="myResults">
                                      See results of digestion and PEC from 21.10.17
			    	</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>observation of AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							The 2-day incubation did not result in higher aggregation, so a new over-night linkage was started.
						&lt;\li&gt;
			    			
			       			</LI><LI>
							After cooling over night and spinning down for 1 min, just minor aggregation was achieved and samples were kept at 4°C for 3 days over the weekend to allow for aggregat
			    			</LI></UL><DIV class="date">Saturday 21st</DIV><UL class="myEntry readout"><LI class="myTitel"><B>DNAse I digest and PCE of the RNA crRNA from Lbu and Lsh</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Restriction digest,Phenol/Chlorophorm purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Florian

			    	</LI><LI class="myObservations">
			    		Observations: 
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>In vitro transcription of crRNA Lwa at 37 C overnight</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">In vitro Transcription</A></LI><LI class="myParticipants">
			    		Participants: Florian

			    	</LI><LI class="myResults">
                                      See results of digestion and PEC from 23.10.17
			    	</LI></UL><DIV class="date">Sunday 22nd</DIV><UL class="myEntry other"><LI class="myTitel"><B>RPA on PDMS Chip</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="https://2017.igem.org/Team:Munich/Protocols">RPA on Paper</A></LI><LI class="myParticipants">
			    		Participants: Sven

			    	</LI><LI class="myObservations">
			    		Observations:
			    		</LI><LI>
			    				Since the PDMS chip was leaky, no real sample could be extracted from it. 
			    			</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>                   	Positive Controls as well as colony-PCR using RPA showed clear bands on a 2%-agarose gel. The RPA performed on Chip, however, was not showing any bands though Nanodrop concentration showed 27 ng/ul in the sample. </LI><UL class="myEntry readout"><LI class="myTitel"><B>observation of AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							After cooling over night and spinning down for 10 min, aggregates could be observed more clearly.
			    			</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>DNAse I digest and PCE of the RNA crRNA from Lwa</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">Restriction digest,Phenol/Chlorophorm purification of RNA</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti

			    	</LI><LI class="myObservations">
			    		Observations: 
			    	</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative controls with Lsh Cas13a </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI><LI>No activity of Lsh Cas13a visible.</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative control with TDPs from team TU Delft</B></LI><LI class="myProtocol">  		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with different concentrations of RNA extracted with chemical lysis and phenol chloroform purification from the E.coli culture</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Repeat of plate reader experiment with Lsh Cas13a with Elution 1 and Elution 4,2</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI><LI>No activity of Lsh Cas13a visible</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Cross target  plate reader experiment using crRNA E.coli with different target RNAs from B.subtilis and Noro virus</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI><LI>It didn't work</LI></UL><DIV class="date">Monday 23rd</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with dried and not dried Cas13a and TDPs with different target concentrations and positive and negative controls</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with with Cas13a Lbu and different concentrations of RPA samples </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with old Lwa Cas13a elution</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI>An elution from July, stored at 4 ºC of Lwa Cas13a was used</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>It didn't work</LI><DIV class="date">Tuesday 24th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with Lwa Cas13a, old elutions 2 and 3</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI>An elution from July, stored at 4 ºC of Lwa Cas13a was used</LI></UL><LI class="myResults"><H4>Results</H4></LI><LI>It didn't work</LI><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with cross targets using crRNA form E.coli and different target RNAs form B. subtilis, Noro virus  and E.coli</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with Spinach aptamer using Lbu Cas13a and different concentrations of in vitro targets</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI><LI>It didn't work</LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>observation of AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP-DNA conjugation</A></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP linkage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							To have optimal conditions and high DNA density of DNA on the AuNP for linkage and cleavage, a new AuNP-DNA conjugation was performed.
						&lt;\li&gt;
									    			
			       			</LI><LI>
							Over-night linkage was started accoriding to the final protocol.
			    			</LI></UL><DIV class="date">Wednesday 25th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiments with lyophilized reaction mix with Cas13a Lbu with different concentrations of the in vitro target. </B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI>Cas13a was added only immediately before starting the measurement.</LI></UL><LI class="myResults"><H4>Results</H4></LI><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with different concentration of heat lysed E.coli samples with Cas13a Lbu</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI><LI>The activity of Cas13a is visible, but additional RNase activity was also seen. </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>General control plate reader experiments with Cas13a Lbu with everything, without crRNA, without target, without Cas13a, positive and negative control.</B></LI><LI class="myProtocol">
			    		Protocol:
			    	</LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>observation of AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP cleavage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							After spinning down the linked AuNP, all supernatant except for 5 μl was removed, pellet resuspended and spotted on paper.
 						&lt;\li&gt;
									    			
			       			</LI><LI>
							The spotted AuNP were visible as blue aggregates and red, unlinked AuNP. 
			    			</LI></UL><DIV class="date">Thursday 26th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with Noro virus crRNA, Lbu Cas13a and different RNA targets from E.coli, HCV and Noro virus.</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiment with Lwa Cas13a purified from the stored Lwa pellet using different concentrations of invitro transcribed RNAs </B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiments with the lyophilized reaction mix on paper using different concentrations of target RNAs</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI><LI>It didn’t work since the lyophilisation was done overnight which was way too long for a small amount of 15 ul reaction mix </LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiments with the RNA aptamer Spinach with different concentrations of in vitro RNA samples</B></LI><LI class="myProtocol">
			    		Protocol: 
			    	</LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><UL class="myEntry readout"><LI class="myTitel"><B>observation of AuNP linkage asssay with RNA-linker and DNA-linker</B></LI><LI class="myProtocol">
			    		Protocol:  <A href="/Team:Munich/Protocols">AuNP cleavage</A></LI><LI class="myParticipants">
			    		Participants: Rob

			    	</LI><LI class="myObservations">
			    		Observations: 
			    		</LI><LI>
							To assure removal of unbound AuNP from , this time, after spinning down the linked AuNP, all supernatant except for was removed, pellet resuspended in 2 μl 1x Cas13a reaction buffer and spotted on paper.
 						&lt;\li&gt;
									    			
			       			</LI><LI>
							The spotted AuNP were visible as blue aggregates and red, unlinked AuNP. 
			    			</LI></UL><DIV class="date">Friday 27th</DIV><UL class="myEntry readout"><LI class="myTitel"><B>Plate reader experiments with Lwa ( elution 1 and 2 and wash 1 ) purified using Nickel NTA</B></LI><LI class="myProtocol">
			    		Protocol: <A href="/Team:Munich/Protocols">RNase activity</A></LI><LI class="myParticipants">
			    		Participants: Dawafuti
			    	</LI><LI class="myObservations">
			    		Observations:
</LI><LI class="myResults"><H4>Results</H4></LI></UL><DIV class="date">Tuesday 31st</DIV></DIV></DIV><DIV class="three wide column" id="monthsList"><UL><LI><A href="#April">April</A></LI><LI><A href="#May">May</A></LI><LI><A href="#June">June</A></LI><LI><A href="#July">July</A></LI><LI><A href="#August">August</A></LI><LI><A href="#September">September</A></LI><LI><A href="#October">October</A></LI></UL></DIV></DIV></DIV></BODY></HTML>