<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Amplification skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Amplification</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD style="background-color: #51a7f9;" colspan="6" align="left"><UL class="menuList" id="menu"><LI><A href="/Team:Munich/Results">Overview</A></LI><LI><A href="/Team:Munich/Cas13a">Cas13a</A></LI><LI><A href="/Team:Munich/Readouts">Readouts</A></LI><LI><A href="/Team:Munich/Targets">Targets</A></LI><LI><A href="/Team:Munich/DetectionOnChip">Detection Chip</A></LI><LI><A href="/Team:Munich/Amplification">Amplification</A></LI></UL></TD></TR><TR><TD colspan="6" align="left" valign="center"><DIV style="margin-top: 40px;"><FONT size="7" color="#51a7f9"><B style="color: #51a7f9">Results: Amplification</B></FONT></DIV></TD></TR><TR><TD width="900%"><H3>What worked:</H3><UL class="listResults"><LI>We used RPA to  <A class="myLink" href="https://2017.igem.org/Team:Munich/Amplification#heatlysed">amplify DNA</A> from heat lysed <I>E. coli</I>.</LI><LI>We conducted RPA and transcription from an in vitro DNA <A class="myLink" href="https://2017.igem.org/Team:Munich/Amplification#onpaper">on paper</A>.</LI><LI>We amplified and transcribed an in vitro DNA target to RNA concentrations <A class="myLink" href="https://2017.igem.org/Team:Munich/Amplification#transcriptionreadout">detectable by our readout circuit</A>.</LI></UL></TD></TR><TR><TD width="900%"><H3>What presented issues:</H3><UL class="listResults"><LI>Amplifying long sequences with RPA.</LI></UL></TD></TR><TR class="lastRow"><TD colspan="6" align="left"><P class="introduction">
Since real-world patient samples contain only traces of pathogens (attomolar at the lowest), any detection method relying on binding affinity (such as Cas13a with target RNA) needs prior amplification of the detected molecule to concentrations within the range of the K<SUB>d</SUB>.
After we confirmed that the detection limit of Cas13a is in the nanomolar region, we had to tackle the problem
of low target concentrations in medical samples.  
We debated different amplification methods that would be simple and stable on our portable platform. Traditional PCR, using a proper thermocycler and well-defined cycling times, also requires a trained personnel to use them, which dioes not fit to our goal of a distributable diagnostic device. We therefore chose to explore isothermal amplification reactions, to facilitate hardware design, leading to decreased production and development costs. 
While Cas13a detects RNA, rather than DNA, we would eventually need to be able to detect both DNA and RNA samples. We therefore chose a combination of <B>reverse-transcriptase</B>-coupled isothermal <B>recombinase polymerase amplification</B> and subsequent <B><I>in vitro</I> transcription </B> for amplification. We term this cascade of amplification reactions <B>RT-RPA-TX</B>. For detection of DNA samples, the RT step can be omitted. Finally, we  lyophilize the reaction mixture on paper, to stabilize the sensitive enzymes for transport.
                </P><DIV class="captionPicture"><P>
Our amplification cascade consisting of RT-RPA-TX can be coupled to the Cas13a-based readout circuit to improve the sensitivity of CascAID.
</P></DIV></TD></TR><TR class="lastRow"><TD colspan="4"><H3 id="heatlysed">Recombinase Polymerase Amplification</H3><P>
The <B>Recombinase Polymerase Amplification</B> (RPA) developed by TwistDx is an isothermal amplification method for DNA. 
Rather than melting the double strand and annealing the primers through temperature cycles, it uses a recombinase
that binds the primers and assists  in the annealing process. Another protein, single-strand DNA binding protein (SSB), 
promotes the binding of the primers to the recombinase in this process.
Hence, the first step in the development of an isothermal amplification method of an RNA signal via RT-RPA-TX was testing the RPA itself in bulk. For this, we took our previously cloned His-tagged TEV Protease (<A class="myLink" href="http://parts.igem.org/Part:BBa_K2323002">BBa_K2323002</A> improved from <A class="myLink" href="http://parts.igem.org/Part:BBa_K1319008">BBa_K1319008</A>)
and ran dummy PCR reactions using the TwistDx RPA kit and the universal biobrick primers VF2 (<A class="myLink" href="http://parts.igem.org/Part:BBa_G00100">BBa_G00100</A>) and VR (<A class="myLink" href="http://parts.igem.org/Part:BBa_G00101">BBa_G00101</A>). We ran the provided test reaction of the TwistDx RPA kit as a positive control (<A class="myLink" href="https://www.twistdx.co.uk/en/products/product/twistamp-exo">TwistAmp exo</A>). The results are shown in <B>Figure 1</B>. 
The first point to note is that PCR primers and RPA primers have different requirements. Due to the need for affinity of the recombinase to the primers, RPA primers tend to be shorter with a higher GC content. Therefore VF2 and VR might not be ideal primers for RPA. The second point is that RPA's optimum amplification length is 500 bp. For longer amplifications, it is possible that the polymerase falls off the strand. This could explain why we see at maximum a band of 500 bp, and shorter bands, although our amplicon should be 1262 bp. We could therefore see that amplification using RPA was possible in bulk, but that primer design would need to be optimized for our final product.

To better assess the RPA reaction, we therefore constructed a benchmark pSB1C3 plasmid coding for 3 of our <A class="myLink" href="https://2017.igem.org/Team:Munich/Targets">target sequences</A> (<I>E. coli</I>, <I>B. subtilis</I> and Norovirus) under the control of a T7 promoter. It is flanked by VF2 and VR primer binding sequences, so that it can be amplified similarly to our first His-TEV construct, but the amplicon is only 194 bp. This was a great benchmarking sequence, as the plasmid could be used for amplification from lysed cells, as would be the case with a real sample. The sequence was short enough, could be coupled with transcription and consisted of relevant targets for our readout.</P><H3 id="onpaper">RPA on Paper</H3><P>  
The next step was to bring the RPA reaction on paper. For this, we lyophilized the reaction mixture provided by TwistDx 
on paper and tried to run RPA reactions directly from it by inserting the blotting paper into the PCR tube. 
Activity could be shown on an agarose gel, but time-stability experiments showed that the activity of the RPA kit decreases quickly.  
In the user's manual, it is stated that after breaking the seal, one should use the reaction mix within one hour. 
We can confirm this statement since activity declined rapidly after as little as two hours and was non-existent after 
24 hours, as is visible in <B>Figure 2</B>.
Furthermore, temperature seems to be a very important factor during amplification using RPA. The samples incubated at 20°C, though still active, showed dramatically decreased activity compared to the same samples at 37°C.

For usage of RPA on a detection device that does not require purification of the sample, the reaction needs to take
place in the environment of a lysate with cell debris and released proteins from the cell. In order to see whether this
is achievable, we performed a colony-PCR like setting of RPA using an <I>E. coli</I> culture in LB-medium with OD=1.0
and ran a regular RPA from purified plasmid as a control. As our first PCR produced a too long amplicon, we changed to a shorter PCR scheme, which should lead to an amplicon of 194 bp. The results are shown in <B>Figure 3</B>. 
A band at the expected length is visible in both the control condition and the <I>E. coli</I> environment. We observed that the yield measured by gel quantification was approximately the same. 
Since the stability of RPA on paper for a long period of time would be crucial to our diagnostic device, we considered the possible reasons for the instability of our RPA mix. We could think of two main factors affecting the stability: humidity and temperature. As we want our platform to be distributable across the world, we needed to test those hypotheses. We found out, that the instability is mainly caused by exposure to air, presumably humidity. We could dramatically increase stability when covering the paperstrip in a plastic Petri dish and sealing it with Parafilm. The stability was increased when keeping the paperstrip at 4°C, but the RPA was also still active after 24 hours storage at room temperature, when protected in a sealed environment. The results are shown in <B>Figure 4</B>. We did not have time to conduct long-term storage tests, but we are hopeful that with the right sealing strategy, the RPA might still be active on paper after a year. 

</P></TD><TD colspan="2"><DIV class="captionPicture" align="center"><P><B>Figure 1:</B> RPA reaction at 37 °C of Control by TwistDx and His<SUB>6</SUB>-TEV using VF2 and VR primers.
</P><DIV class="captionPicture"><P><B>Figure 2:</B> RPA reaction of lyophilised RPA mixture at 37 °C or 20 °C of His<SUB>6</SUB>-TEV using VF2 and VR primers. 
The control is a reaction mixture without the lyophilisation step.
</P><DIV class="captionPicture"><P><B>Figure 3:</B> Colony-PCR of Cas13a Benchmark plasmid using RPA. Control is a standard RPA reaction of purified plasmid. The cells were lysed for 10 minutes at 95 °C prior to the RPA reaction.
</P></DIV><DIV class="captionPicture"><P><B>Figure 4:</B> RPA reaction after freeze-dried storage on paper at different conditions. Conditions that were taken 
into consideration are temperature and air accessibility. Air determines the samples that were accessible to air.
The other samples were stored in a Petri dish sealed with parafilm.
</P></DIV></DIV></DIV></TD></TR><TR class="lastRow"><TD colspan="6"><H3>Coupling to Transcription</H3><P>
The coupling of TX to RPA is in theory feasible because both reaction are conducted at 37°C, and still active at room temperature. We thus proceeded to try RPA-TX on the His<SUB>6</SUB>-TEV construct. However, we never succeeded to get the expected bands in the urea-PAGE following phenol-chloroform extraction. As our initial amplicon from the His-TEV construct was too long (1262 bp compared to 500 bp optimal amplification length from RPA), we believe that the enzyme did not manage to produce double-stranded DNA, and therefore transcription was not completed. At this point, we decided to create our benchmark plasmid described above, which has an amplicon of only 194 bp. We could characterize it for RPA, but unfortunately we did not have time to conduct RPA-TX experiments in bulk. Instead, we directly proceeded to attempting RPA-TX lyophilized on paper.
</P></TD></TR><TR><TD colspan="6"><H3 id="transcriptionreadout">Bringing RPA and TX on Paper</H3><P>
Our final goal was to bring together RPA and TX lyophilized on paper and prove the possibility of amplifying target RNA from our benchmark plasmid. The paper should then be sealed inside the <A class="mylink" href="https://2017.igem.org/Team:Munich/Hardware/SampleProcessing">sample processing unit</A>. For this, we ran experiments and analyzed the RNA expression with urea-PAGE (<B>Figure 5</B>). The expected transcript length is 132 nucleotides. The reaction was ran for 30, 60 and 120 minutes on paper, and the control was a bulk overnight reaction, all at 37°C. We see a clear band at the right length, with increasing intensity as the reaction is ran for longer. As the intensity of the band seems to be linearly increasing, this suggests that the transcription by T7 polymerase is the rate limiting step. 
 </P><DIV class="captionPicture" align="center"><P><B>Figure 5:</B> 15%-Urea-PAGE for concentration determination of the RPA-TX amplifications. The time gives incubation time at 37 °C before ending the reaction by phenol-chloroform extraction. Control is given by a RPA reaction where T7 RNA Polymerase was not added.
</P></DIV><P>
Finally, we proceeded to test the ability of our product from RPA-TX to trigger Cas13a collateral RNase activity. We conducted the RPA-TX from our benchmark plasmid on paper, then phenol-chloroform extracted the RNA, quantified the RNA concentration, and tested the amplified RNA with our Cas13a detection circuit (<B>Figure 6</B>). In this experiment and others, the positive control was lower than the higher concentrations of target RNA, which we attribute to low activity of the RNaseA used in the positive control. We found that the Cas13a response was as good as from normally <I>in vitro</I> or <I>in vivo</I> sourced target RNA, and we could similarly detect 10 nM of amplified RNA. Our control contained the result of an RPA amplification without TX, that was extracted similarly as the RPA-TX sample, and added in the same volume as the 10 nM amplified RNA sample. 
</P><DIV class="captionPicture" align="center"><P><B>Figure 6:</B> Detection of the RPA-TX with varying sample concentration using Cas13a.
</P></DIV><H3>Reproducibility</H3><P>
The RPA amplification is a well standardized method, which is commercially available as a kit. We used our RPA-TX repeatedly on paper with different incubation times and stability conditions, and we found that the circuit worked well, even though the sequence length of the amplicon should be carefully designed. We however did not have time to try amplification from a variety of sequences or primers.</P><H3>Discussion and Conclusion</H3><P>
We successfully conducted RPA from DNA <I>in vitro</I> and in cell lysate context. We joined RPA and TX in a one batch reaction, and conducted it on paper, using purified DNA. We could detect RNA with gel quantification within 60 minutes. Finally, we were able to perform Cas13a detection from RPA-TX amplified target, both on paper and in bulk. The stability of RPA on paper needs to be improved, but we showed that protection from air provided a reasonable protection of the activity. One issue that still needs to be solved is that the coupled RPA-TX reaction was only tested for purified samples. All the parts we tested (RPA in cell lysate, RPA combined with TX, stability of the whole circuit on paper, detection from an amplified target) now need to be combined and associated with the <A class="mylink" href="https://2017.igem.org/Team:Munich/Hardware/SampleProcessing">sample processing unit</A> and our <A class="mylink" href="https://2017.igem.org/Team:Munich/Hardware/Detector">detector</A>. Our modular approach proved successful to develop parallel working units that have to be assembled into a fully functioning platform. 
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>References</H3><P><OL style="text-align: left"><LI id="ref_1">Gao, Zhu and Huang. &quot;Recombinase Polymerase Amplification: A New DNA/RNA Amplification Strategy.&quot; (2016)
    	<I>Chinese journal of Biochemistry and Molecular Biology</I> 32(6): 627-634.</LI><LI id="ref_2">Daher, Stewart, Boissinot and Bergeron. &quot;Recombinase Polymerase Amplification for Diagnostic Applications&quot; 
       (2016) <I>Clinical Chemistry</I> 62(7): 947-958.</LI><LI id="ref_3">Beckert and Masquida. &quot;Synthesis of RNA by In Vitro Transcription.&quot; (2010) <I> Methods in Molecular Biology </I>
        Volume 703</LI></OL></P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>