Team:Munich/Protocols
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Protocols
For our project we used different methods which are described in respective protocols. We also included a detailed list of all chemicals, materials and the aim of each experiment. This page is for everybody who wants to reproduce our work or wants to get an overview, but also to share our experiences with the iGEM community. Simply click on the section of interest to see the respective protocols.
General
Cloning
Cas13a
Readout
Other
Agarose gel electrophoresis
PCR
Minipreparation
Restriction digest
Ligation
Glycerol stock
Ni-NTA agarose protein purification
Äkta protein purification
SDS PAGE
RNase activity
Lyophilization
Expression of Cas13a and TEV Protease
Phenol/Chlorophorm purification of RNA
Phenol/Chlorophorm purification of genomic DNA
Denaturing Urea PAGE
In vitro Transcription
RNA purification with Norgen Kit
FINA method for RNA extraction
FINA method for DNA extraction
First strand cDNA syntesis
RPA
RPA on paper
RPA TX
Chemical transformation
Electro transformation
Chemically competent cells
Electro competent cells
Dephosphorylation
Golden Gate Assembly
Gibson Assembly
PCR clean-up
Sequencing sample preparation
Soft litography
AuNP preparation
AuNP-DNA conjugation
AuNP linkage
AuNP cleavage
Cell lysis
Protein and nucleic acid sequences
Proteins
Nucleic acids
Cloning
We used different approaches to assemble genetic parts dependent on their design. Golden Gate cloning was chosen to insert His-SUMO-Cas13a-Lwa into pSB1C3. Digestion and ligation of DNA parts was used for the "aeBlue" cloning. For the Intein-Extein constructs we finally combined different cloning methods to assembly the DNA fragments.
We amplified DNA fragments with standard PCR approaches by using Q5, Phusion and Taq polymerases. Cells were transformed by chemical or electro transformation and outgrown at 37 °C for 1 h in SOC medium before plating on selective LB agar plates. We isolated plasmid DNA with the help of a commercial available kit and always eluted with nuclease-free water. With analytical restriction digest, colony PCR and agarose gels we checked the success of our cloning work. Final glycerol cryo stocks were always verified by sequencing.
RNA extraction
We tested different methods to lyse cells and purify RNA. We compared these methods with commercial available kits. RNA purity was checked in denaturating urea gels or agarose gels. To quantify RNA after FINA extraction we transcribed it into cDNA, amplified it using PCR, and then tried to quantify the products on an agarose gel. To test our system, we first used defined transcribed RNAs which mimicked our targets. After in vitro transcription, RNA was further purified using a phenol chloroform based method.
Protein purification
We expressed our His-tagged proteins in E.coli strains and purified them using the Äkta purification system or Ni-NTA agarose. The latter had the advantage to incubate the cell lysate a certain time to get a better binding. To cleave off the His-SUMO or His-MBP tags from Cas13a proteins, we incubated them with the SUMO or TEV protease during dialysis over night, respectively. In some cases, we reloaded the cleaved protein solution again on Ni-NTA agarose to get rid of the thereby binding His-tag. The proteins were purified further via size exclusion chromatography. Protein purity was always checked in SDS PAGE gels.
Cas13a Lbu is the central component of our diagnostic platform. The TEV Protease is part of our idea to the Intein-Extein readout, but apart from that, served as molecular tool for cleaving off the protein tags. A summary of all protein information can also be found.
Readout
For our experiments, we chose Cas13a Lbu since it was reported to be the most active one. We first focused to get a running system and detect a fragment of E.coli 16S rRNA in plate reader experiments. We in vitro transcribed these target sequences, purified them using a phenol/chloroform based method and quantified the resulting RNA in a denaturing SDS PAGE. As a reporter system we chose the RNase alert kit from IDT. Further testing included the detection of different targets, both bacteria and viruses. Next step was to detect RNA, which we isolated form actual bacterial sources. We then brought the system onto paper by lyophilization of our components.
Other
Methods related to other readouts, for example how to deal with gold nanoparticles, were added. Additionaly, a detailed list of all in this project used nucleotide sequences was also linked.
