Team:Munich/Description
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Description
Thanks to advances in molecular biology and biochemistry, scientists have been able to consistently detect lower and lower concentration of molecules 1 , to the point where single molecules can be reliably recognized using methods such as polymerase chain reaction (PCR) 2 , fluorescence in situ hybridization (FISH) 3 and enzyme-linked immunosorbent assays (ELISA) 4 . This has opened doors for synthetic biology to create better and more accurate diagnostic tests that use biomarkers like nucleic acids and proteins as a target 5 , 6 . These advances have led to development of the field of molecular diagnostics. Unfortunately, current standard diagnostic methods require expensive equipment or trained personnel, which limits their usability to hospitals or laboratories. Recently, there has been a push to develop new tests that fuse the reliability of standard methods with affordable platforms such as lab-on-a-chip or paper strips  to overcome these restrictions 7-9 . We wanted to help seal this gap and thus set out to engineer a diagnosis principle for the detection of a wide array of targets that could be used at the point-of-care.
Overview of our three major hardware modules. Shown are, starting from the left: the processing unit, the paper strip and our fluorescence detector.
Problem Definition
Antibiotic resistance is a global public health risk with high severity. Overuse of antibiotics happening probably since discovery of penicillin has led to appearance of multi-resistant strains of pathogens. About 90% of all antibiotics prescription are issued by general practitioners (GP) and most of them account for upper respiratory tract infections (57% of all prescribed antibiotics in Europe) 11 . Astonishing is that from 70% of patients in U.S. having sore throat receive antibiotics prescription, while only about 20-30% are likely to have a bacterial infection recommended for antibiotic treatment 12 . Besides, results of survey in UK showed that 55% of GPs felt under pressure, mainly from patients, to prescribe antibiotics and 44% admitted that they issued prescription to get a patient to leave the surgery. 11
Solution Statement
We set out to develop highly specific highly sensitive rapid in vitro diagnostic (IVD) device that will provide solid foundation for treatment. We strongly believe that point-of-care (POC) IVD device, which is able to distinguish pathogens (including both viral and bacterial), will significantly contribute to resolution of antibiotics overprescription and respectively of antibiotic resistance crisis. Besides that our device is easily configurable and can be quickly adopted to detect nucleic acid sequence of pathogen of choice.
As a basis for design of our solution we used WHO guideline for POC testing devices known as “ASSURED” 13 , which stands for: affordable, sensitive, specific, user-friendly, rapid and robust, equipment-free, deliverable to the person in need. Moreover, we decided to create all-in-one portable solution that includes all steps of analysis: sample processing, nucleic acid amplification and target detection.
CascAID+
Our project, named Cas13a controlled assay for infectious diseases (CascAID), features the recently identified CRISPR/Cas effector Cas13a 10 . Unlike other proteins in the familiy, Cas13a has the unique ability to bind and cleave specific RNA rather than DNA targets.  Moreover, after cleaving its target, Cas13a is able to unspecifically cleave RNA molecules. By using this collateral activity of Cas13a, our system is capable of detecting virtually any RNA target. This is done by changing the crRNA in the protein, which is a short RNA sequence that determines what is recognized as target.
The 3D structure of Cas13a Lbu shows the crRNA (blue) which binds the complementary target RNA (green).
Cas13a binds specific target RNA depending on the crRNA sequence. After activation, Cas13a cleaves RNA indiscriminately, which we use for our detection mechanisms. The scheme vizualizes the possibilty to design new crRNAs to sense any kind of target sequence.
Our project is divided in the following 3 general parts:
Sample Processing Unit
Tackling the challenge of sample pre-processing on field, we started developing a portable fluidic system featuring a temperature control unit for lysis and isothermal PCR (RPA). Conceiving a platform independent of lab infrastructure, we demonstrate the feasibility of controlling fluid flow control with the simplest tools possible using bike tires and air balloons.
Paper Strip Reaction Unit
After pre-processing, the idea was to combine all diagnostic reactions into one easy-to-use format. We chose to embed all the reactions into the format of a paper strip of the size of a typical post-it, where our full readout reaction cascade takes place. This enables to freeze-dry all reaction agents to make them fit for long-term storage. An additional advantage of the paper format are the low sample volumes needed for a reaction assey. To enable transport of the sample-containing fluid to the areas containing the detection mixture, we chose to use the paper-fluidics technology. The whole printing mechanism of the paper fluidics is based around a regular office printer to pattern the paper with hydrophobic wax channels. The detection circuit is first assessed in bulk, the Cas13a is characterized using the RNaseAlert standard, its detection limit is determined and the differentiation between viral and bacterial targets is verified, before the mechanism is transferred into a paper strip application. Three advanced readout methods are designed and explored, all of which propose an amplification cascade following Cas13a target detection. Those readout methods, combined with the fluidics, should give us the possibility to lower the detection limit and improve the on-field use.
Detector Unit
Starting from the fact that suitable measurement instruments with sufficient sensitivity for field use are too expensive for mass production, we constructed a portable low-cost fluorescence detector, which can be easily assembled with a few standard worldwide available electronic parts and a 3D-printer. Driving the development even further, we pushed the sensitivity of our detector into the range of commercial plate readers, while conserving an assembly cost of under 15$. A detailed documentation of the detector development and sensitivity determination can be found under Measurement and Detector . On top of our hardware technology we provide a software for a crRNA databank, secondary structure verification of crRNAs and off-target verification of designed crRNAs. In combination with the detector unit, we supply a program code to evaluate data acquired with our detector.
References
Cohen, Limor, and David R. Walt. "Single-Molecule Arrays for Protein and Nucleic Acid Analysis." Annual Review of Analytical Chemistry 0 (2017).
Nakano, Michihiko, et al. "Single-molecule PCR using water-in-oil emulsion." Journal of biotechnology 102.2 (2003): 117-124.
Taniguchi, Yuichi, et al. "Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells." science 329.5991 (2010): 533-538.
Rissin, David M., et al. "Single-molecule enzyme-linked immunosorbent assay detects serum proteins at subfemtomolar concentrations." Nature biotechnology 28.6 (2010): 595-599.
Pardee, Keith, et al. "Rapid, low-cost detection of Zika virus using programmable biomolecular components." Cell 165.5 (2016): 1255-1266.
Slomovic, Shimyn, Keith Pardee, and James J. Collins. "Synthetic biology devices for in vitro and in vivo diagnostics." Proceedings of the National Academy of Sciences 112.47 (2015): 14429-14435.
Tang, Ruihua, et al. "A fully disposable and integrated paper-based device for nucleic acid extraction, amplification and detection." Lab on a Chip 17.7 (2017): 1270-1279.
Vashist, Sandeep Kumar, et al. "Emerging technologies for next-generation point-of-care testing." Trends in biotechnology 33.11 (2015): 692-705.
Gubala, Vladimir, et al. "Point of care diagnostics: status and future." Analytical chemistry 84.2 (2011): 487-515.
Abudayyeh, Omar O., et al. "C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector." Science 353.6299 (2016): aaf5573.
Llor, C., & Bjerrum, L. (2014). Antimicrobial resistance: risk associated with antibiotic overuse and initiatives to reduce the problem. Therapeutic Advances in Drug Safety, 5(6), 229–241.
Shulman ST1, Bisno AL, Clegg HW, Gerber MA, Kaplan EL, Lee G, Martin JM, Van Beneden C. Clinical practice guideline for the diagnosis and management of group A streptococcal pharyngitis: 2012 update by the Infectious Diseases Society of America. Clin Infect Dis. 2012 Nov 15;55(10):1279-82. doi: 10.1093/cid/cis847.
Grace Wu & Muhammad H Zaman. Low-cost tools for diagnosing and monitoring HIV infection in low-resource settings. Bulletin of the World Health Organization 2012;90:914-920. doi: 10.2471/BLT.12.102780
