<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_DetectionOnChip skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/DetectionOnChip</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD style="background-color: #51a7f9;" colspan="6" align="left"><UL class="menuList" id="menu"><LI><A href="/Team:Munich/Results">Overview</A></LI><LI><A href="/Team:Munich/Cas13a">Cas13a</A></LI><LI><A href="/Team:Munich/Readouts">Readouts</A></LI><LI><A href="/Team:Munich/Targets">Targets</A></LI><LI><A href="/Team:Munich/DetectionOnChip">Detection Chip</A></LI><LI><A href="/Team:Munich/Amplification">Amplification</A></LI></UL></TD></TR><TR><TD colspan="6" align="left" valign="center"><DIV style="margin-top: 40px"><FONT size="7" color="#51a7f9"><B style="color: #51a7f9">Results: Detection on Chip</B></FONT></DIV></TD></TR><TR><TD colspan="6"><H3>What worked:</H3><UL class="listResults"><LI>We characterized Cas13a and its detection limit with native and <A class="myLink" href="https://2017.igem.org/Team:Munich/DetectionOnChip#lyophi">lyophilized</A> protein, with 
<A class="myLink" href="https://2017.igem.org/Team:Munich/Cas13a#Figure_1">in vitro</A> and <A class="myLink" href="https://2017.igem.org/Team:Munich/Cas13a#vivo">in vivo</A> sources of RNA, in bulk and <A class="myLink" href="https://2017.igem.org/Team:Munich/DetectionOnChip#onpaper2">on paper</A>. </LI><LI>We optimized the <A class="myLink" href="https://2017.igem.org/Team:Munich/DetectionOnChip#onpaper2"> paper support</A> for Cas13a functionality.</LI><LI>We built the <A class="myLink" href="https://2017.igem.org/Team:Munich/Hardware/Detector#sensitivity"> most sensitive</A> and  <A class="myLink" href="https://2017.igem.org/Team:Munich/Hardware/Detector#cost">cheapest</A> fluorescence detector <A class="myLink" href="https://2017.igem.org/Team:Munich/DetectionOnChip#detectorB">ever created by an iGEM team</A>   to our knowledge. 
</LI><LI>We successfully used our fluorescence detector to <A class="myLink" href="https://2017.igem.org/Team:Munich/DetectionOnChip#detectorA"> detect Cas13a activity</A> on paper.</LI></UL></TD></TR><TR><TD colspan="6"><H3>What presented issues:</H3><UL class="listResults"><LI>Optimizing the lyophilization and stability of Cas13a.</LI></UL></TD></TR><TR><TD colspan="6" align="left"><P class="introduction">
We used a step-wise experimental method to determine if our detection circuit could be applied on-chip. We first put our bulk solution on paper and followed the fluorescence in a plate reader, we then lyophilized the Cas13a on paper, and finally we checked the bulk reaction on paper in our <A class="myLink" href="/Team:Munich/Hardware/Detector">self-built detector</A>.  
                </P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3 id="onpaper2">Feasibility of detection on paper</H3><P>  
First, we needed to optimize the paper on which the detection was done. We found that our detection circuit was hindered on nitrocellulose paper, and that the best support was glass fiber filter paper blocked with 5% BSA overnight, rinsed with RNase-free water and dried in the oven (70°C, 20 minutes). </P><P>
As a first simple test, we pipetted our reaction mixture on the paper, sandwiched the strip in coverslips and pinched the paper in a <A class="myLink" href="https://github.com/igemsoftware2017/igem_munich_2017/tree/master/3D_Prints">3D-printed 96-well plate</A>, to avoid any artefacts caused by evaporation, curling of the paper or similar effects. We could then measure the fluorescence in a plate reader. We found the kinetics to be similar to bulk, and the detection limit to be between 10 nM and 50 nM <B>(Figure 1)</B>. This is similar to the detection limit in bulk experiments, but as in this particular set of experiment, we did not have a positive control, we cannot normalise the amount of cleaved RNaseAlert and directly compare the titration curves. </P><DIV class="captionPicture"><P><B>Figure 1</B>: The detection circuit is pipetted on paper, with different concentrations of target RNA. The type of paper and its treatment are crucial to the correct processing of the circuit. 
</P></DIV></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3 id="lyophi">Lyophilization</H3><P>  
We then looked into lyophilizing the protein. As we intend our paper strips to be distributable, we need the detection circuit to resist a range of temperatures and humidity, and lyophilization was found to be the optimal method for this purpose<SUP><A class="myLink" href="#ref_1">1,2</A></SUP>. We mixed all components for the reaction but the target RNA and applied them on paper, froze the paper at -80°C or in liquid nitrogen, and lyophilized the sample. The target was then pipetted on the paper with water to reach the intended final concentration, the paper was treated as described above, and the fluorescence was tracked in the plate reader. We found that the detection efficiency was lower than for pure bulk experiments, and we could only detect 100nM of target. All other concentrations of target gave a signal below that of the negative control <B>(Figure 2)</B>, which can happen as shown before due to the design of our negative control. We think that the decrease in detection limit is caused by inactivation  of the Cas13a protein during the lyophilization process. An optimisation of the concentrations of Cas13a and crRNA could help improve our detection limit on lyophilized samples.</P><DIV class="captionPicture"><P><B>Figure 2</B>: Fluorescence intensity at 30 minutes, with lyophilized detection circuit, for different target concentrations. 
</P></DIV></TD></TR><TR><TD colspan="3" align="center" valign="center"><H3>Tardigrade Proteins</H3><P>  
When we interviewed him about the possibility to use Cas13a as a paper strip based pathogen detector, <A class="myLink" href="/Team:Munich/Gold_Integrated/KeithPardee">Dr. Pardee</A> advised us to use a cryoprotectant together with our Cas13a to avoid loss of function or bad stability when freeze-dried. Specifically, he recommended trehalose which is a carbohydrate present in Tardigrade Proteins (TDPs). As the iGEM team from TU Delft (Case 13a) is working on associating TDPs and Cas13a to create an antibiotic resistance test, we asked for a sample and characterized the functionality of our detection circuit when dried together with TDPs <B>(Figure 3)</B>. We found that the basal activity of Cas13a was increased and that the target-specific activation was undetectable, even for 60 nM target concentration. Based on these experiments, we think it more likely that lyophilization methods will give us a better activity of the Cas13a on paper. </P></TD><TD colspan="3" align="center" valing="center"><DIV class="captionPicture"><P><B>Figure 3</B>: Cas13a activity at 30 minutes when initially dried with TDPs. We find that basal activity without target is roughly one fourth of the activity of the positive control. We think TDPs cause Cas13a to be active without target. 
</P></DIV></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3 id="detectorA">Portable Fluorescence Detector</H3><P>  
Having our RNaseAlert based readout functioning on paper, we created a portable paper-based fluorescence detector to make this readout fit for in field usage. Our detector costs less than 15$, is reusable and can measure time lapses with a sensitivity in the range of a commercial plate reader. To have reproducible measurements we calibrated our detector by measuring dilution series of fluorescein with on chip. Our detector therefore measures fluorescence in equivalent fluorescein concentrations. As a first proof of principle we reproduced the plate reader experiments for Cas13a on paper. We were able to measure a time trace of target-activated Cas13a digesting RNaseAlert with our detector. For comparison, we also measured a positive control containing RNaseA and a negative control containing only RNaseAlert. The data are displayed in <B>Figure 4</B>.</P><DIV class="captionPicture"><P><B>Figure 4</B>: Time lapse measurement of Cas13a digesting RNase Alert on paper using our detector. The positive control contains RNaseA and RNaseAlert. The negative control contains only RNaseAlert. Data points are connected with lines for the readability. Error bars represent the measurement uncertainties of the detector.
</P></DIV><P>
The data show typical curves of enzyme kinetics. It can be seen that RNaseA is more active than Cas13a. The negative control shows that our detector was free of RNase contaminations. This proves that our detector is in fact able to quantitatively measure different levels of enzyme activity and can easily distinguish between the negative control and active Cas13a. By assuming that RNaseA digested all RNaseAlert, we conclude that 185 nM of RNaseAlert have an equivalent fluorescence to 10 µM fluorescein. Our detection limit for RNaseAlert is therefore around 50 times lower than the limit for fluorescein, which corresponds to a RNaseAlert concentration lower than 10 nM. When characterizing Cas13a, we chose a cut-off of 15% of the total RNaseAlert cleaved to accept a signal as positive, which corresponds to a concentration of roughly 28 nM, so our detector limit is good enough for our diagnosis test.
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Reproducibility</H3><P>
Our glass paper fiber treatment method was reproducible, and the Cas13a showed good activity on this support. Our lyophilization method proved successful, and this should be reproduced and optimized for our application. The basal activity of Cas13a when dried with TDPs and in the absence of target, was consistent with the results from the <A class="myLink" href="/Team:TU_Delft">TU Delft team</A>. 
</P><P>  
We created a <A class="myLink" href="https://2017.igem.org/wiki/index.php?title=Team:Munich/Hardware/Detector"> detailed documentation</A> of our detector including derivations of equations, a detailed consideration of measurement uncertainties and a complete description of the assembly of our detector. This should enable other iGEM teams to rebuild and use our detector. As we intend our detector to be easy to <A href="Team:Munich/Measurement">assemble and use</A>, we are confident that it could be used to characterise fluorescence circuits in a reproducible manner. However, seeing the difference in detection limit between fluorescein and RNaseAlert, we think our detector should be calibrated with cleaved RNaseAlert.
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3 id="detectorB">Discussion and Conclusion</H3><P>  
We found that Cas13a detection is functional on paper, with or without lyophilization. This module is validated and was successfully integrated with the <A href="Team:Munich/Detector" class="myLink">detector</A>. However, optimisation of the lyophilization process, and study of the stability of the lyophilized detection circuit should be further pursued. 
We classified our detector by creating a cost vs. sensitivity diagram in <B>Figure 5</B>. We compared commercial detectors, detectors from other iGEM teams and low-cost detectors from publications to our detector. The other detectors roughly fall along a line of cost vs. sensitivity. Our detector, however, shows a better sensitivity than all iGEM-built detectors we found, and a lower cost than all other detectors from publications or companies.</P><DIV class="captionPicture"><P><B>Figure 5</B>: Cost vs. sensitivity diagram of several fluorescence detectors. We compared commercially available detectors (orange dots), low-cost detectors from publications<SUP><A class="myLink" href="#ref_7">7-12</A></SUP> (green dots) and  detectors from other iGEM teams (blue dots) to our fluorescence detector (red dot).
</P></DIV><P>
Price is an important factor for the viability of any kind of diagnostic device. For these reason, we ran the calculations of the cost of a single CascAID reaction, assuming economy of scales could be use to produce a batch of 1000 reactions efficiently. According to our calculations, we come under the 1$ price, which was one of the requirements that we set for ourselves after our <A class="myLink" href="https://2017.igem.org/Team:Munich/HP/Gold_Integrated/Doctors_Without_Borders">interview with Dr. De La Tour from Doctors without Borders.</A></P></TD></TR><TR><TH>Part</TH><TH>Cost</TH><TH>Fraction used/reaction</TH><TH>Cost/1000 ($)</TH><TH>Cost/1 ($)</TH></TR><TR><TD>Cas13a protein purification</TD><TD>1448,95</TD><TD>2,54E-06</TD><TD>3,68</TD><TD>0,00368</TD></TR><TR><TD>1 NEB transcription kit</TD><TD>217</TD><TD>4,00E-07</TD><TD>0,0868</TD><TD>0,0000868</TD></TR><TR><TD>RPA kit</TD><TD>410</TD><TD>5,20E-04</TD><TD>213,54</TD><TD>0,21354</TD></TR><TR><TD>RNase alert</TD><TD>295</TD><TD>6,67E-05</TD><TD>19,67</TD><TD>0,01967</TD></TR><TR><TD>T7/RNTPs</TD><TD>67</TD><TD>2,40E-04</TD><TD>16,08</TD><TD>0,01608</TD></TR><TR><TD>RNase inhibitor</TD><TD>276</TD><TD>5,33E-04</TD><TD>147,2</TD><TD>0,1472</TD></TR><TR><TD>PDMS</TD><TD>0,1</TD></TR><TR><TD>glass fiber paper</TD><TD>0,5</TD><TD>0,002</TD><TD>1</TD><TD>0,001</TD></TR><TR><TD><B>Lightbringer</B></TD></TR><TR><TD>Geekreit Atmega 328p</TD><TD>2,92</TD><TD>1,00E-03</TD><TD>2,92</TD><TD>0,00292</TD></TR><TR><TD>USB cable</TD><TD>3,49</TD><TD>1,00E-03</TD><TD>3,49</TD><TD>0,00349</TD></TR><TR><TD>LDR GL5516 NT00183</TD><TD>0,17</TD><TD>1,00E-03</TD><TD>0,17</TD><TD>0,00017</TD></TR><TR><TD>Blue LED</TD><TD>0,12</TD><TD>1,00E-03</TD><TD>0,12</TD><TD>0,00012</TD></TR><TR><TD>Green LED</TD><TD>0,12</TD><TD>1,00E-03</TD><TD>0,12</TD><TD>0,00012</TD></TR><TR><TD>BC635 transistor</TD><TD>0,14</TD><TD>1,00E-03</TD><TD>0,14</TD><TD>0,00014</TD></TR><TR><TD>Hole raster board</TD><TD>0,49</TD><TD>1,00E-03</TD><TD>0,49</TD><TD>0,00049</TD></TR><TR><TD>Resistors</TD><TD>0,56</TD><TD>1,00E-03</TD><TD>0,56</TD><TD>0,00056</TD></TR><TR><TD>light filter foils</TD><TD>0,12</TD><TD>1,00E-03</TD><TD>0,12</TD><TD>0,00012</TD></TR><TR><TD>Neodym magnets</TD><TD>5,58</TD><TD>1,00E-03</TD><TD>5,58</TD><TD>0,00558</TD></TR><TR><TD><B>Plate</B></TD></TR><TR><TD>button red </TD><TD>23,65</TD><TD>1,00E-03</TD><TD>23,65</TD><TD>0,02365</TD></TR><TR><TD>button green</TD><TD>23,65</TD><TD>1,00E-03</TD><TD>23,65</TD><TD>0,02365</TD></TR><TR><TD>recessed socket</TD><TD>1,58</TD><TD>1,00E-03</TD><TD>1,58</TD><TD>0,00158</TD></TR><TR><TD>U.S solid mini 12V 2 way valve</TD><TD>104,44</TD><TD>1,00E-03</TD><TD>104,44</TD><TD>0,10444</TD></TR><TR><TD>valve connector</TD><TD>4,68</TD><TD>1,00E-03</TD><TD>4,68</TD><TD>0,00468</TD></TR><TR><TD>chipboard</TD><TD>4,07</TD><TD>1,00E-03</TD><TD>4,07</TD><TD>0,00407</TD></TR><TR><TD>ACa platte</TD><TD>10,34</TD><TD>1,00E-03</TD><TD>10,34</TD><TD>0,01034</TD></TR><TR><TD>Tygon tubing</TD><TD>27,88</TD><TD>1,00E-03</TD><TD>27,88</TD><TD>0,02788</TD></TR><TR><TD>Syringe 12 ml</TD><TD>6,42</TD><TD>1,00E-03</TD><TD>6,42</TD><TD>0,00642</TD></TR><TR><TD>Syringe vent</TD><TD>3,49</TD><TD>1,00E-03</TD><TD>3,49</TD><TD>0,00349</TD></TR><TR><TD><B>Heatbringer</B></TD></TR><TR><TD>Peltier element</TD><TD>2,3</TD><TD>1,00E-03</TD><TD>2,3</TD><TD>0,0023</TD></TR><TR><TD>pinboard</TD><TD>5,81</TD><TD>1,00E-03</TD><TD>5,81</TD><TD>0,00581</TD></TR><TR><TD>Thermosensor tmp006</TD><TD>8,13</TD><TD>1,00E-03</TD><TD>8,13</TD><TD>0,00813</TD></TR><TR><TD>Thermosensor si7021</TD><TD>9,29</TD><TD>1,00E-03</TD><TD>9,29</TD><TD>0,00929</TD></TR><TR><TD>Rasberry pi3b</TD><TD>40,66</TD><TD>1,00E-03</TD><TD>40,66</TD><TD>0,04066</TD></TR><TR><TD><B>Together for Hardware</B></TD></TR><TR><TD>jumper cable</TD><TD>1,74</TD><TD>1,00E-03</TD><TD>1,74</TD><TD>0,00174</TD></TR><TR><TD>wires</TD><TD>6,39</TD><TD>1,00E-03</TD><TD>6,39</TD><TD>0,00639</TD></TR><TR><TD>silicon blocks</TD><TD>27,3</TD><TD>1,00E-03</TD><TD>27,3</TD><TD>0,0273</TD></TR><TR><TD>3d printed plastic (pla)</TD><TD>13,94</TD><TD>1,00E-03</TD><TD>13,94</TD><TD>0,01394</TD></TR><TR><TD>M3 screws</TD><TD>2,9</TD><TD>1,00E-03</TD><TD>2,9</TD><TD>0,0029</TD></TR><TR><TD>M3 screw-nuts</TD><TD>5,29</TD><TD>1,00E-03</TD><TD>5,29</TD><TD>0,00529</TD></TR><TR><TD><B>0,85</B></TD></TR></TBODY></TABLE><P>
The detector is an excellent alternative to commercial fluorescence detectors. It is not limited to our specific application but can be used for the detection of any fluorescence signal in biological or chemical systems. We therefore think that our detector can benefit other iGEM teams and research groups that want to make fluorescence based detection fit for in-field applications.
</P><H3>References</H3><P><OL style="text-align: left"><LI id="ref_1">Pardee, K et al. &quot;Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components&quot;. Cell (2016) 165(5):1255-1266. doi: 10.1016/j.cell.2016.04.059. Epub 2016 May 6.</LI><LI id="ref_2">Gootenberg, J. S.et al. &quot;Nucleic acid detection with CRISPR-Cas13a/C2c2.&quot;Science (2017). eaam9321.</LI><LI id="ref_3">Cohen, Limor, and David R. Walt. &quot;Single-Molecule Arrays for Protein and Nucleic Acid Analysis.&quot; Annual Review of Analytical Chemistry  (2017).</LI><LI id="ref_4">Nakano, Michihiko, et al. &quot;Single-molecule PCR using water-in-oil emulsion.&quot; Journal of biotechnology 102.2 (2003): 117-124.</LI><LI id="ref_5">Taniguchi, Yuichi, et al. &quot;Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells.&quot; Science 329.5991 (2010): 533-538.</LI><LI id="ref_6">Rissin, David M., et al. &quot;Single-molecule enzyme-linked immunosorbent assay detects serum proteins at subfemtomolar concentrations.&quot; Nature biotechnology 28.6 (2010): 595-599.</LI><LI id="ref_7">Wu, Jing, et al. &quot;An economical fluorescence detector for lab-on-a-chip devices with a light emitting photodiode and a low-cost avalanche photodiode.&quot; Analyst 137.2 (2012): 519-525.</LI><LI id="ref_8">Walczak, Rafał, and Krzysztof Adamski. &quot;Inkjet 3D printing of microfluidic structures—on the selection of the printer towards printing your own microfluidic chips.&quot; Journal of Micromechanics and Microengineering 25.8 (2015): 085013.</LI><LI id="ref_9">Yang, Feng-Bo, et al. &quot;A low-cost light-emitting diode induced fluorescence detector for capillary electrophoresis based on an orthogonal optical arrangement.&quot; Talanta 78.3 (2009): 1155-1158.0039-9140, https://doi.org/10.1016/j.talanta.2009.01.033.</LI><LI id="ref_10">Novak, Lukas, et al. &quot;An integrated fluorescence detection system for lab-on-a-chip applications.&quot; Lab on a Chip 7.1 (2007): 27-29.</LI><LI id="ref_11">Pais, Andrea, et al. &quot;High-sensitivity, disposable lab-on-a-chip with thin-film organic electronics for fluorescence detection.&quot; Lab on a Chip 8.5 (2008): 794-800.</LI></OL></P></DIV></DIV></DIV></DIV></BODY></HTML>