Team:Munich/Readouts
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Overview
Cas13a
Readouts
Targets
Detection Chip
Amplification
Results: Readouts
What worked:
We designed fluorescent and colorimetric readouts, and used a synthetic aptamer as a detection tool.
What presented issues:
Developing colorimetric read-outs.
For our experimental design, we used different fluorescence and colorimetric readouts as stated below. The fluorescence read-outs worked well, and especially the RNaseAlert allowed us to characterize our Cas13a detection in a variety of contexts: native protein, lyophlized, in bulk, on paper, from virus target, bacterial target, in vitro and in vivo RNA. For our colorimetric read-outs, we could not reach a full integration with target RNA detection, but where able to assemble the necessary constructs and start characterizing them.
RNaseAlert Readout
To characterize our Cas13a, we first turned to the standard of the field, namely the RNaseAlert detection kit, which uses the short modified RNA sequence RNaseAlert. This was used in recent work to characterize Cas13a and detect pathogen RNA sequences 1,2 . In the absence of Cas13a activation, there is no fluorescence detectable due to physical proximity of the quencher to the fluorophore. Upon binding of a matching target RNA to the crRNA, Cas13a develops a promiscuous RNase activity and cleaves off the quencher from the fluorophore. This spatial separation in solution now allows the fluorophore to emit light, leading to a detectable fluorescent signal.  We showed that the increase in signal is a measure of the Cas13a activation by target RNA and therefore used it for most of our experiments. The corresponding results are specifically described in the Cas13a and targets subsections.
In general, the RNaseAlert assay works very well, but one has to consider that a diagnostic device relying on this system is dependent on a fluorescence detector. For this reason, we built and evaluated the fluorescence detector “Lightbringer”, which is the most sensitive fluorescence detector ever built by an iGEM team to our knowledge. Since a colorimetric readout would overcome this problem by making an easy visualization possible, we also worked on different colorimetric readouts.
Spinach Aptamer Readout
Here, we used the Spinach aptamer (a 80-nucleotides RNA) which binds the DFHBI fluorophore, changing its spatial conformation and thereby enables fluorescence 3 . Activated Cas13a cleaves the Spinach aptamer leading to the release of DFHBI. This process is detectable as a decreasing fluorescence intensity (Figure 1).
Figure 1: The fluorescence activity of the Spinach aptamer is inversely proportional to the target concentration.
ssDNA Readout
For this readout, we wanted to link the cleavage of an RNA strand (due to Cas13a activation) to an amplification scheme based on ssDNA. A dimer is formed between a ssDNA sequence and an inhibitor ssRNA sequence. This RNA is composed of three regions binding to the ssDNA separated by polyU loops (Figure 2), so that upon cleavage of the polyU loops by Cas13a, the melting temperature of the dimer is lowered and the cleaved ssRNA falls off. The ssDNA is freed and can be used into an amplification scheme: we envisioned that it would either complete a linear transcription template (known as genelet) that is single-stranded in its promoter region, and activate its transcription 4 , or it would bind the PCR DNA template. In both cases, either transcription or PCR would lead to amplification of the signal. A transcription signal could be read with a nucleic acid binding dye, or could be further linked to translation, to create a colored protein read-out such as aeBlue. Using transcription translation as an detection amplification into a colorimetric readout was successfully shown by Pardee et al. 5 . Similarly, DNA amplification could be signaled with nucleic acid binding dyes and fluorescence could be read with our detector .
Figure 2: Working principle of ssDNA
We designed a fitting inhibitor RNA and complementary DNA activator, and confirmed with Nupack 6 that cleaving of the polyU loops would cause the dimer to melt at room temperature. The functional assembly of the RNA/DNA dimer could be proved by native PAGE. We designed a double-stranded DNA template that is only single-stranded in its promoter region, so that it could be activated by the released ssDNA activator. Furthermore, the cornerstone for the transfer of the circuit to a colorimetric readout was laid by the successful cloning of aeBlue into a pSB1C3 backbone. This construct can be amplified, then cleaved with a type II restriction enzyme and a nuclease, so that the promoter region can be rendered single stranded. However, to this point we were not able to demonstrate that Cas13a activity, or even RNaseH, can successfully free the ssDNA activator. We think that the ssDNA/ssRNA ratio and the sequences could be optimized so that the dimer can be melted after RNA cleavage. We initially tried to prove that the dimer could be formed, and we may have overshot the design in the direction of dimer stability and binding efficiency. We also did not find a dye that gave us a very good read of the nucleic acids concentrations in such a dynamic system. We do not see a fundamental blockage to the possibility to develop this readout to the colorimetric readout, but we did not reach the full proof-of-concept within the time of our project.
Intein-Extein Readout
The Intein-Extein-Readout consists of 2 devices. A N- and a C- Terminal fragment of beta-Gal that are both linked to the respective halfs of an inactivated split intein, that can be activated by TEV-cleavage. The sequence of the split-intein was ordered as gblocks from IDT and inserted into psB1C3 and psB4A5 respectively.  The successful insertion was confirmed by sequencing at GATC. The plasmids were then opened by BbsI and the coding sequence of beta-galactosidase was inserted via a Gibson-Assembly. The coding sequence originated from W3110 and was extracted by PCR with overlap primers. Once again, we confirmed the sequences at GATC and got full length reads for both constructs. However, the two biobricks could neither be submitted nor characterized, as the recloning to remove illegal cuts sites within the sequence was not finished prior to the deadline. The characterization was not possible, as the C-terminal fragment could not be expressed in any of our E. coli expression strains, as it seemed to be toxic. The toxicity seemed to not affect E. coli Turbo, which was used for cloning. The N-terminal part alone could not be characterized as the parts only work together.
We plan to express the C-terminal part with an in vitro expression system and are confident, that the purified proteins will work together as planned.
Figure 3: Colony-PCR of C-Terminal fragment. Samples 9 and 12 had the correct length and were confirmed with sequencing.
Figure 4: Colony-PCR of N-Terminal fragment. Samples 8 and 9 had the correct length and were confirmed with sequencing.
Gold nanoparticle readout (AuNP)
We designed a gold nanoparticle (AuNP)-based readout system, which consists of AuNPs of ~10 nm diameter cross-linked by DNA-labels and complementary RNA linkers which remain single-stranded in their center. In a paper-based test, upon encounter with either target-activated Cas13a or another RNAse cleaving this single-stranded segment, the aggregates are expected to partially and evenly dissolve into the surrounding area and develop a reddish color. When first testing AuNP cleavage on paper, a positive result, i.e. a color change mediated by spread of AuNPs, was not visible for Cas13a, but for the positive control with RNaseA. In the follow-up experiment, RNA-linked and DNA-linked AuNPs were examined. Cleavage of aggregates did occur in the RNAseA-containing positive control for the RNA-linkers but not in the DNA-linked negative control. These preliminary results indicate that our AuNP system can be used for selective detection of RNases.
Figure 5: Upon nuclease activity for either target-activated Cas13a or RNase A , an even circular distribution of diffused, red-shifted AuNP around the spotted aggregate was expected (see left panels). This could be observed in the RNaseA-containing positive controls (upper panels) for the AuNPs with RNA-linkers U5, U10 and U15, containing either 5, 10 or 15 Uracil-containing single-stranded linker segments, but not for Cas13a (lower panels) or negative control with DNA-linked AuNP.
However, some improvements of the assay should be conducted. First, aggregation should be optimized to avoid any unspecific aggregation, while facilitating specific aggregation trough extraction of full-length in-vitro-transcribed RNA. Second, it would be useful to quantify the kinetics of AuNP-resuspension by RNaseA and Cas13a in a plate-reader based assay, like our experiments using RNaseAlert. Last, to optimize test conditions on the paper platform, a variety of paper materials, coatings and sealing materials should be tested. After all, looking at the exposed position of the Cas13a promiscuous cleavage site and our results on Cas13a and AuNPs, we are confident that an optimized version of this readout will present a functional tool for RNA detection.
Reproducibility
We successfully integrated the fluorescence readouts RNaseAlert and Spinach Aptamer in our Cas13a detection system for different bacterial and viral targets. We successfully tested Cas13a Lbu and Lwa with RNaseAlert and in parallel the Spinach Aptamer with the Cas13a Lbu. Based on our extensive experience with RNaseAlert, we can say that this fluorescence readouts can be implemented in variety of systems and is highly reproducible. However, our work on colorimetric readouts was too preliminary to discuss reproducibility of those circuits.
Discussion and conclusion
The results we obtained using the RNaseAlert and Spinach Aptamer with our Cas13a system conclude that having a fluorescence readout is an efficient system. This is why we constructed our detector , and we successfully used it to characterize Cas13a activity with RNaseAlert. Additionally, we were also able to try explore different colorimetric readouts and lay the groundwork for their success. The idea of using AuNP for a colorimetric readout is quite promising, taking into account the positive result it gave us with the RNaseA.
References
Gootenberg, J. S., Abudayyeh, O. O., Lee, J. W., Essletzbichler, P., Dy, A. J., Joung, J., ... & Myhrvold, C. (2017). Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, eaam9321.
East-Seletsky, A., O’Connell, M. R., Knight, S. C., Burstein, D., Cate, J. H., Tjian, R., & Doudna, J. A. (2016). Two distinct RNase activities of CRISPR-C2c2 enable guide-RNA processing and RNA detection. Nature, 538(7624), 270-273.
Paige, J. S., Wu, K. Y., & Jaffrey, S. R. (2011). RNA mimics of green fluorescent protein. Science, 333(6042), 642-646.
Franco, E., Friedrichs, E. Kim, J., Jungmann, R., Murray, R., Winfree, E., Simmel, F.C. (2011). Timing molecular motion and production with a synthetic transcriptional clock. PNAS, 108(40), E784-E793.
Pardee, K., Green, A.A., Takahashi, M.K., Braff, D., Lambert, G., Lee, J.W., Ferrante, T., Ma, D., Donghia, N., Fan, M., Daringer, B.M., Bosch, I., Dudley, D.M., O'Connor, D.H., Gehrke, L., Collins, J.J. (2016). Rapid, Low-Cost Detection of Zika Virus Using Programmable Biomolecular Components. Cell, 165, 1255-1266.
J. N. Zadeh, C. D. Steenberg, J. S. Bois, B. R. Wolfe, M. B. Pierce, A. R. Khan, R. M. Dirks, N. A. Pierce. NUPACK: analysis and design of nucleic acid systems. J Comput Chem, 32:170–173, 2011.
