Team:Munich/Results/Detector
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Description
Thanks to advances in molecular biology and biochemistry, scientists have been able to consistently detect lower and lower concentration of molecules 1 , to the point that single molecules can be reliably recognized with methods such as polymerase chain reaction (PCR) 2 , fluorescence in situ hybridization (FISH) 3 and enzyme-linked immunosorbent assays (ELISA) 4 . This has opened doors for synthetic biology to create better and more accurate diagnostic tests that use biomarkers like nucleic acids and proteins as targets 5 , 6 . Through such advances, the field of molecular diagnostics developed. Unfortunately, current standard methods require expensive equipment or trained personnel, which generally limits their usability to hospitals or laboratories. Recently, there has been a push to develop new tests that fuse the reliability of standard methods with affordable platforms such as lab-on-a-chip or paper strips  to overcome this restrictions 7-9 . We wanted to help close this gap and set out to engineer a diagnosis principle for the detection of a wide array of targets that could be used without difficult-to-meet technical requirements.
Applications and reasons for a field-use detection device
In times of increasing antibiotic resistances and the accompanying problems to fight bacterial infections against multi-resistant bacterial strains, it is of especial need to conserve antibiotic use to reasonable utilization, to avert a post-antibiotic era and its fatal aftermath. Freely available antibiotics for big parts of the world population leads to lots of misuse and wrong practice in combination with antibiotics, which will not change without an easy entry point for the public, when and what antibiotic is of need.
Key parts a home diagnostic method must fulfill
Therefore, an easy to use home diagnostic tool must work based on the public standard state of knowledge. What is commonly known by the public, is that antibiotics only have effect against bacterial infections. Working of this knowledge state a low cost, easy to use, reliable home diagnostic tool, which could determine the reason of an infection could give the public a straightforward way to reduce the previously stated misuse of antibiotics. They could determine their etiology by themselves without the entry barriers of visiting their general practitioner or a hospital. From the test result on, the path to find the right treatment for their disease would be extremely simplified. By the switch of diagnostic treatment to everyone’s home step, an extremely synergistic effect for the whole population could be generated.
Problem definition for a reliable home diagnostic
Today, pathogens are discriminated by cell culture or PCR-based methods, requiring expensive equipment, trained personal, and time. Point-of-care tests currently on the market, like pregnancy tests, target certain metabolites and are therefore restricted to one specific application and one detection target. For a reliable home diagnostic test it would be of need to combine the portability, affordability, and usability of a smartphone like gadget with the universality and sensitivity of typical laboratory experimental setup of PCR-based nucleic acid detection methods and their accompanying equipment.
Solution statement
Based on these stated problems the solution seems only accomplishable by the combination of the transfer of all diagnostic reaction parts into the already previously used paper strip format and a readout and sample processing based around a portable processing unit with an integrated sensor with a high sensitivity/cost ratio.
CascAID+
Our project, which we named Cas13a controlled assay for infectious diseases (CascAID), features the recently identified CRISPR/Cas effector Cas13a 10 . Unlike other proteins in the familiy, Cas13a has the unique ability to bind and cleave specific RNA targets rather than DNA ones.  Moreover, after cleaving its target, Cas13a is able to unspecifically cleave RNA molecules. By using this collateral activity from Cas13a, our system is capable of detecting virtually any RNA target. This is done by changing the crRNA in the protein, that is a short RNA sequence that determines what is recognized as target.
Cas13a binds specific target RNA depending on the crRNA sequence. After activation, Cas13a cleaves RNA indiscriminately.
Our project is divided in the following 3 general parts:
Sample processing unit
Tackling the challenge of sample pre-processing in field, we started developing a portable fluidic system featuring a temperature control unit for lysis and isothermal PCR (RPA). Conceiving a platform independent of lab infrastructure, we demonstrate the feasibility of controlling fluid flow control with the simplest tools possible using bike tires and air balloons.
Paper strip reaction unit
After pre-processing, the idea was to combine all diagnostic reactions into an easy to use format. We chose to imbed all the reactions into the format of a paper strip of the size of a typical post-it. Our full readout producing reaction chain takes place on this small paper strip. This enables us to freeze-dry all reaction agents in a small proximity and further provides also long-time storage possibility. In addition, the advantages of the paper format are the low sample volumes needed for a reaction asset. To enable transport of the sample-containing fluid to the areas containing the detection mixture, we chose to use the paper-fluidics technology. The whole printing mechanism of the paper fluidics is based around a regular office printer to pattern the paper with hydrophobic wax channels. The detection circuit is first assessed in bulk, the Cas13a is characterized using the RNaseAlert standard, its detection limit is determined, and the differentiation between viral and bacterial targets is verified; before the mechanism is transferred into a paper strip application. Three advanced readout methods are designed and explored, all of which propose an amplification cascade following Cas13a target detection. Those readout methods, combined with the fluidics, should give us the possibility to lower the detection limit and improve the on-field use.
Detector unit
Starting from the fact that suitable measurement instruments with high enough sensitivity for field use are too expensive for mass production, we constructed a portable low-cost fluorescence detector, which can be easily assembled with a few standard worldwide available electronic parts and a 3D-printer. Driving the development even further, we pushed the sensitivity of our detector into the range of commercial plate reader, while conserving an assembly cost of around 15 $. A detailed documentation of the detector development and sensitivity determination can be found under Measurement & Detector Furthermore, with our hardware technology we provide a software for a crRNA databank, secondary structure verification of crRNAs and off target verification of designed crRNAs. In combination with the detector unit, we supply a program code for data evaluation of data acquired with our detector.
References
Cohen, Limor, and David R. Walt. "Single-Molecule Arrays for Protein and Nucleic Acid Analysis." Annual Review of Analytical Chemistry 0 (2017).
Nakano, Michihiko, et al. "Single-molecule PCR using water-in-oil emulsion." Journal of biotechnology 102.2 (2003): 117-124.
Taniguchi, Yuichi, et al. "Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells." science 329.5991 (2010): 533-538.
Rissin, David M., et al. "Single-molecule enzyme-linked immunosorbent assay detects serum proteins at subfemtomolar concentrations." Nature biotechnology 28.6 (2010): 595-599.
Pardee, Keith, et al. "Rapid, low-cost detection of Zika virus using programmable biomolecular components." Cell 165.5 (2016): 1255-1266.
Slomovic, Shimyn, Keith Pardee, and James J. Collins. "Synthetic biology devices for in vitro and in vivo diagnostics." Proceedings of the National Academy of Sciences 112.47 (2015): 14429-14435.
Tang, Ruihua, et al. "A fully disposable and integrated paper-based device for nucleic acid extraction, amplification and detection." Lab on a Chip 17.7 (2017): 1270-1279.
Vashist, Sandeep Kumar, et al. "Emerging technologies for next-generation point-of-care testing." Trends in biotechnology 33.11 (2015): 692-705.
Gubala, Vladimir, et al. "Point of care diagnostics: status and future." Analytical chemistry 84.2 (2011): 487-515.
Abudayyeh, Omar O., et al. "C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector." Science 353.6299 (2016): aaf5573.
Gootenberg, Jonathan S., et al. "Nucleic acid detection with CRISPR-Cas13a/C2c2." Science (2017): eaam9321.
https://www.idtdna.com/pages/docs/technical-reports/in_vitro_nuclease_detectionD325FDB69855.pdf (retrieved: 13.10.17)
Anraku, Yasuhiro, Ryuta Mizutani, and Yoshinori Satow. "Protein splicing: its discovery and structural insight into novel chemical mechanisms." IUBMB life 57.8 (2005): 563-574.
Link, Stephan, and Mostafa A. El-Sayed. "Size and temperature dependence of the plasmon absorption of colloidal gold nanoparticles." The Journal of Physical Chemistry B 103.21 (1999): 4212-4217.
Zhao, W., Ali, M.M., Aguirre, S.D., Brook, M.A., and Li, Y. (2008). "Paper-based bioassays using gold nanoparticle colorimetric probes." Analytical Chemistry 80, 8431–8437.
