Team:Munich/Cas13a
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Overview
Cas13a
Readouts
Targets
Detection Chip
Amplification
Results: Cas13a
What worked:
We characterized Cas13a and its detection limit with native and lyophilized protein, with in vitro and in vivo sources of RNA, in bulk and on paper .
What presented issues:
Optimizing the purification protocol for Cas13a.
Demonstrating functionality of Lsh Cas13a.
Ruling out RNase contamination from heat-lysed in vivo samples.
Protein Cloning, Expression and Purification
We decided to compare three versions of Cas13a that were previously characterized in the literature 1 : Lbu, Lsh, and Lwa. We ordered the Lbu and Lsh plasmids from Addgene, and we cloned Lwa using Golden Gate assembly (sequence was taken from Gootenberg et al., 2017). Lbu and Lsh were expressed in E.coli Rosetta2, as the sequences were not codon-optimized, and Lwa was expressed in E.coli BL21 (DE3) star. We created three BioBricks from the Lwa sequence: BBa_K2323000 (containing the Lwa coding sequence and a Tphi terminator), BBa_K2323001 (where a 6xHis/Twin strep tag and a SUMO tag are added to the N-terminal end of BBa_K2323000), and BBa_K2323004 (where BBa_K2323001 is preceded by the T7 promoter and the Elowitz RBS). We improved the TEV-protease BBa_K1319008 by tagging it with a 6xHis tag, purified it and successfully used it for the TEV cleavage of our Cas13a proteins.
Gel pictures of the final steps in the purification of our four proteins
We followed the purification protocols from literature, and found that although the His-purification and the tag cleavage steps worked as expected, the cation-exchange purification step failed, and we systematically lost our proteins. We still completed the size-exclusion purification, and our proteins with some amount of contamination. Protein purification took most of the first month of our project, due to the failure of the cation-exchange chromatography, but we eventually purified functional, if not perfectly clean, proteins.
Proof-of-Concept
To prove the functionality of Cas13a, we used the 16S rRNA sequence from E.coli as a target sequence, given that is highly conserved in all bacterial species and can be easily extracted from bacterial cultures in large concentrations. For our first experiments, we used only 130 nucleotides of the 16S rRNA sequence and transcribed in vitro from a DNA template (since the whole 16S rRNA is 1500 nucleotides, therefore too large to be transcribed). Our crRNA DNA template was designed so that the target-binding region could easily be changed to detect new targets . We found that both Lbu and Lwa were functional and degraded the read-out RNase Alert in presence of both the target and the crRNA. An example time plot is shown in Figure 1, where the specific activity of Lbu was controlled by taking out the crRNA and Lbu, alternatively.
Lbu showed higher cleaving efficiency at equal concentrations compared to Lwa (contradicting what was shown in Gootenberg et al., 2017), and Lsh was not functional (we assume that the purification process inactivated the protein), see Figure 2. We therefore decided to use Lbu for the rest of our experiments.
Figure 1: Plot of a typical experiment with 10 nM Lbu Cas13a, 100 nM crRNA, 50 nM target, 185 nM RNase Alert and 1U/µL RNase inhibitor. For analysis, we typically considered the fluorescence intensity of samples after 30 minutes, and normalized it to obtain the ratio of cleaved RNase Alert, assuming that our negative control (with neither crRNA and Cas13a) had 0% cleavage and our positive control (with RNaseA) had 100% cleavage. We should note that we occasionally found that high target concentrations led to above positive control signals (which could be due to the degradation and lesser activity of RNaseA) and that low target concentrations led to below negative control signals (which could be due to noise at low fluorescence intensities).
Figure 2: Target RNA concentration was screened for all three Cas13a proteins and their matching crRNA. A conservative cut-off of at least 15% of the RNaseAlert cleavage was chosen to determine the detection limit of our system.
Figure 3: The activity of Lwa Cas13a was found to be similar before and after His purification.
Interestingly, we found that Lwa was active even without purification: after lysing cells expressing the Cas13a, we used the supernatant in our detection system, and found similar activity as after purification, see Figure 3. This result, along with further characterization, showed us that Cas13a is a relatively robust enzyme that works in a variety of contexts.
We screened the cleavage efficiency dependence on Cas13a and target concentrations, and found that for high Cas13a concentration, the background activity of Cas13a was overlaying with the target [plot of ratio vs Cas13a concentration] specific activation (Figure 4). As our device should detect low target RNA concentrations in less than 30 minutes, we optimized the concentration of Cas13a: at high concentrations of the enzyme, the background activity hid the target-dependent signal; at low concentrations, the enzyme was too slow and a detectable signal could not be obtained in 30 mins unless large amounts of target RNA were added. A compromise was found at 10nM of Cas13a, and in these conditions, we found our target detection limit to be around 10nM (Figure 1).
Figure 4: At constant target RNA concentration, as Lbu Cas13a concentration increases, the background activity of the enzyme reduces the on/off ratio of activation by the target RNA.
Cell lysis and RNA extraction
For RNA extraction from our bacterial targets, we looked at several possible lysis methods. We tried and abandoned Guanidine-salts as lysis agent, since its strong chaotropic power makes extensive purification necessary. For the same reason regarding the need for purification, we used detergent/ heat lysis only in our lab work. While we investigated RNA-silica binding properties (see labbook Sept. 1st to 5th, section "other") and tested commercial silica-based kits for such purifications, we decided against adding unnecessary complexity for our prototype.
Figure 5: Lysis-RNA yield of detergent/heat and alkaline lysis.
Figure 6: Degradation of RNA due alkaline lysis with different incubation times.
Alkaline lysis is well-known for DNA-, but not for RNA-extraction due to the rapid hydrolysis of RNA under alkaline conditions. Since our protein responds to a very short part of our target sequence (<30 bp), compared to the resulting RNA fragments (most >300 pb, see Figure 6), it should work none the less and with better efficiency (Figure 5) and superior speed (seconds) compared to detergent/ heat lysis.
Since microfluidic mixing of liquids is a rather complicated process, we settled for an isothermal PCR-based approach (RPA). With the exceptional sensitivity of PCR, we can even use an inefficient heat-only lysis (5-10 times less efficient than detergent/ heat) and still detect RNA with an amount of 100 cells in the PCR reaction volume.
Figure 7: PCR with varying cell density [1] log2 DNA ladder, [2] 106 cells/ml, [3] 105 cells/ml, [4] 5*104 cells/ml, [5] 104 cells/ml,[6] 5*103 cells/ml, [7] 103 cells/ml, [8] 102 cells/ml,[9] 106 cells/ml (no heat-lysis step. only PCR at 37 °C)
Detection of Pathogenic RNA from in vivo Source
We then set out to detect RNA from in vivo samples rather than from in vitro transcribed RNA. As we had chosen the 16S rRNA sequence of E. coli as a target, we used E. coli DH5α cultures as in vivo samples. We performed two kinds of treatment on the cells (from an overnight culture):
We lysed the cells with 10% SDS and heated them between 80°C or 95°C for 10 minutes, and then extracted the RNA with phenol-chloroform extraction. We used this purified RNA to perform the detection tests.
We lysed the cells with heat (80°C or 95°C for 10 minutes) and used this directly for our detection tests. As the sample was not purified, we expect to have some amount of RNase present here, and it is unclear whether the RNase inhibitor we used was enough to prevent activity from the E. coli native RNases.
To have an estimation for the 16S rRNA concentration for our first extraction method, we did the following calculations. We assumed that a concentration of 10 fM of 16S rRNA would be equivalent to a cell concentration of 100 CFU/mL, which is the conservative end of the range given by Esfandiari et al 2 .  We then assumed that our overnight culture would have an O.D. 600 nm of 2, corresponding to 1,6 * 109 CFU/mL. We assumed no loss of RNA during phenol-chloroform extraction (which is again, a conservative estimation of the concentration), and considered a concentrating factor of 40, as we extracted the RNA from a 2 mL culture and resuspended it in 50 µL. We estimated that our extracted RNA would have a concentration of 6,4 µM of 16S rRNA, and tested our detection circuit with dilutions from this source, see Figure 8. We found that we had a higher detection limit for our in vivo source, which could be caused by our conservative calculation of the extracted RNA concentration.
Figure 8: Titration curve for the detection of the 16S rRNA from E.coli, from an in vitro or an in vivo source.
Our second extraction method is closest to what we want to achieve on our chip: the cells are lysed and the target is amplified. As we did not manage to bring together our amplification module with our in vivo extraction module (due to lack of time), we set out to directly detect the RNA from the lysed cells. Assuming the same O.D. as for our first extraction method, the concentration of 16S rRNA in a saturated culture would be around 160 nM. In this experiment, we found that the fluorescence was maximum for an intermediate concentration of the lysed cells (equivalent to an estimated 48 nM of 16S rRNA). As expected, the fluorescence was lower as the lysed cells concentration decreased (Figure 9), but we could not explain why the signal also went down for the higher concentration (equivalent to 80 nM 16S rRNA). In all samples with cells, the fluorescence was higher than the positive control, which could indicate that the fluorescence is not due to Cas13a activity but rather to RNAse activity. However, the positive control was significantly lower here than in our first in vivo experiment (around 3*104 a.u. of fluorescence compared to 6*104 a.u. for the same gain), which could be due to a loss of activity of RNaseA. Besides, our Lwa experiments have shown a similar activity for the enzyme directly pipetted from lysed cells as for a His-purified enzyme. We therefore think that there is good indication that we can directly detect the 16S rRNA from heat-lysed cells. However, it is clear that this experiment should be reproduced and confirmed. A control experiment could consist of an unnatural target that will be added to E.coli via a plasmid. We could then compare cells with and without the plasmid, i.e. with and without the target, but where the RNase contamination from cell lysis should be identical.
Figure 9: Direct detection of 16S rRNA from heat-lysed cells led to a peak response depending on concentration.
Reproducibility
As we characterized the Cas13a thoroughly, we found that the enzyme was extremely robust in its activity. It showed reproducible cleaving activity through batches of purification of both the enzyme and the purified RNAs, with different target concentrations, and especially when handled by different experimenters, more or less trained. However, we did find that as the kinetics of Cas13a in these conditions are relatively fast, the signal had already reached saturation when the slower experimenter was done assembling the reaction into the reading plate. We recommend that all parts of the sample be assembled, and that the target RNA (or its source) be added at the very last minute, just before starting fluorescence acquisition, so that the kinetics can be properly followed. In the context of our diagnosis device, this would not cause a problem, as we want the fastest possible result reading by the patient or doctor.
Discussion and Conclusion
We purified and proved the functionality of the Cas13a enzyme, chose Lbu for its better activity, optimized the concentrations in our detection scheme and found the detection limit to be in the range of 10 nM target RNA. We found that we could detect RNA from in vivo sources, with full RNA extraction, but possibly also from simply lysed cells. This makes this module (the Cas13a detection circuit) the best characterized and most promising module of our platform. It gives fast, high fluorescence signals for low target RNA concentration, and can be combined with our amplification module, which would use heat lysis (80°C) followed by reverse transcription, RPA and transcription (room temperature).
References
Gootenberg, J. S., Abudayyeh, O. O., Lee, J. W., Essletzbichler, P., Dy, A. J., Joung, J., ... & Myhrvold, C. (2017). Nucleic acid detection with CRISPR-Cas13a/C2c2. Science, eaam9321.
Esfandiari, L., Wang, S., Wang, S., Banda, A., Lorenzini, M., Kocharyan, G., ... & Schmidt, J. J. (2016). PCR-Independent Detection of Bacterial Species-Specific 16S rRNA at 10 fM by a Pore-Blockage Sensor. Biosensors, 6(3), 37.
