Team:Munich/Software/Collaborations
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Software Collaboration
For TU Delft
We started collaborating early after the European iGEM Meet-Up with TU Delft in the wetlab. Since we were both employing Cas13a as the molecular heart of our project, it made sense to  exchange data and experiences from wet lab experiments. At some point during a skype meeting, we realised that we were doing similar things in the dry lab and thus, decided to collaborate and exchange ideas in that field as well.
Secondary Structure Prediction
In our drylab collaboration with Team Delft, we performed a Cas13a design verification  of crRNAs they used in their project with our developed software. Since the secondary  structure of the crRNA is essential for its affinity to Cas13a, one can make assumptions  on the post-binding RNase activity of the protein based on secondary structure predictions.  We then compared these predicted structures  to secondary structures of crRNAs that have  been shown to work experimentally. For the five crRNA sequences Team Delft provided,  the predicted secondary structure matched one of the secondary structures in our databank.  The only one that was not recognized by neither the NUPACK nor the mFOLD databank was  crRNA 2. It's predicted secondary structure in Vienna notation is depicted below:
GGAUUUAGACUACCCCAAAAACGAAGGGGACUAAAACAAGACAGUCAUAAGUGCGGCGACGAU ..(((((((((.((((.........)))).((......))..)))).))))).((....))..
It looks like the backbone structure needed in the first 35 bases is not constructed completely.  When looking at the graphical output of NUPACK depicted in Figure 1, however, it is visible that the binding probability  seems to be rather low regarding the structures that interact with the terminal 28 bp target sequence.  Thus, crRNA 2 would most probably still be active when tested in experiment, but might show decreased activity.
Figure 1: Graphical output of the secondary structure prediction of crRNA 2 in NUPACK.
Figure 2: Cleavage experiments of crRNA 2, crRNA 3 and crRNA 4 as provided by TU Delft.
Indeed, decreased activity using crRNA 2 in comparison to other crRNA is observed when performing experiments using  these crRNA designs. Though this has to be treated with caution since crRNA 4's secondary structure was predicted  to be correct but the Cas13a-crRNA complex of that crRNA does not show higher activity. We conclude that further testing of the software has to be done to show whether it can predict activity of Cas13a.
Off-target Effects
Furthermore, in order to determine off-target effects, we constructed a database from Transcriptome data obtained  from the Ensembl and Ensembl Bacteria databank. We tried to make the search as tailor-made to the Delft project as  possible and thus considered species that were relevant to their project of detecting bacterial resistances related  to cattle and milk production.  The included species were:
Bos Taurus (Cattle)
Clostridium perfringens
Corynebacterium diphtheria
Fusobacterium necrophorum
Lactobacillus casei
Lactococcus lactis
Providencia stuartii
Staphylococcus aureus
Streptococcus pneumoniae
Three of the five crRNAs showed no off-targets in the constructed database. For crRNA 3 and crRNA 4,  possible off-targets were detected but just as a result of the identity parameter being rather low  in the BLAST-short run. When looking at the results, it is clearly visible that the sequences are not  identical enough to show any RNase activity of Cas13a since the 28 bases target is specific to up to 2 mutations,  but all the hits that have been found are with a match of 17 bases far from the 28 bases long.
Finally, we can conclude that our software did not find any problems in the crRNA design of Team Delft.  It seems that their constructs will most probably show activity and, at least for the mentioned species,  it will not show any off-target effects.
From TU Delft
The TU Delft team developed a simulation code for modeling the on and off-target activities for sequences on all possible frames on the genome. In this, they employ a kinetic model by Depken et al. in 2017  to determine whether off-target activities are high enough to give a false positive  collateral RNase activity of the Cas13a protein. They deducted from these model cleavage probabilities for different crRNA sequences and could thus make statements of the possibility to distinguish  between our two bacterial targets, the 16S rRNA of E. Coli and B. subtillis. By running the crRNA against both genomes of these species, they determined the off-target probability to be very low for all our crRNA design as shown in Figure 3. Thus, they deduced from these that our crRNA design would be able to differentiate between E. Coli and B. subtilis.
Figure 3: Off-target probability of activation of the Cas13a protein for our crRNAs sensing E.Coli (1.3) and B. subtillis crRNA 1-3 are taken into one since the spacer sequence of the crRNA was identical for these constructs.
We would like to thank Team Delft for the fun collaboration and wish you all the best for your project.  We are looking forward to seeing you in Boston.
References
Klein, M., Eslami-Mossallam, B., Gonzalez Arroyo, D., Depken, M. (2017).       "The kinetic basis of CRISPR-Cas off-targeting rules." doi: 10.1101/143602.
