Team:Munich/Notebook
-_-
Project
Description
Design
Demonstrate
Measurement
Applied Design
Final Results
Achievements
Wetlab
Protocols
Notebook
Safety
Parts
Improved Part
Part Collection
InterLab
Model/Software
Model
Software
Collaboration
Hardware
Detector
Quake Valve
Paperstrips
Sample Processing
Human Practices
Silver
Gold
Engagement
Collaborations
Team
Team members
Sponsors
Attributions
Gallery
Labbook
We documented our work chronologically on a daily basis, linked the respective protocols and added all lab results. You can find the entries sorted according to month and sub-project, which can be selected separately. To do actual experiments it is also important to prepare buffer and all kind of stock solutions, agar plates, gels and so on. This basic work is partially not mentioned here, but was also done by the team. To
ç To find what was done in each day, select the month you want to see and then select which category you want to check out.
Cas13a
Read-out
Target
Other
Tuesday 25th
Plating of DH5α p2CT-His-MBP-Lbu-Cas13a-WT and DH5α p2CT-His-MBP-Lsh-Cas13a-WT on        LBCarb agar plates
Protocol: -
Participants: Ludwig
Observations:
Incubation at 37 °C over night.
Results
Bacteria grew and single colonies were visible.
Wednesday 26th
Preparation of 6 ml LBCarb cultures of DH5α p2CT-His-MBP-Lbu-Cas13a-WT and DH5α p2CT-His-MBP-Lsh-Cas13a-WT on LBCarb agar plates
Protocol: -
Participants: Ludwig
Observations:
Incubation at 37 °C over night.
Results
Bacteria cultures grew.
Thursday 27th
Preparation of cryo stocks for DH5α p2CT-His-MBP-Lbu-Cas13a-WT and        DH5α p2CT-His-MBP-Lsh-Cas13a-WT
Protocol: Cryo stocks
Participants: Ludwig
Minipreparation of DH5α Cas13a overnight cultures
Protocol: Minipreparation
Participants: Ludwig
Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT in E. coli Rosetta.
Protocol: Electro transformation
Participants: Ludwig
Observations:
1 µl of plasmid DNA was used.
Bacteria were plated on LBCarb plates without additional Cm.
Results
Bacteria lost second plasmid probably.
Tuesday 2nd
Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT        in E. coli Rosetta.
Protocol: Electro transformation
Participants: Ludwig
Observations:
1 µl of plasmid DNA was used.
Bacteria were plated on LBCarb + Cm plates
Results
Bacteria grew and single colonies were visible.
Preparations of buffers for Cas13a purification.
Protocol: Protein purification
Participants: Chris, Ludwig, Julian
Observations:
TCEP changes the pH. Readjustement after adding neccessary.
Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT        in E. coli Rosetta.
Protocol: Electro transformation
Participants: Ludwig
Observations:
1 µl of plasmid DNA was used.
Bacteria were plated on LBCarb + Cm plates
Results
Bacteria grew, but only few single colonies were visible.
Wednesday 3rd
Electro transformation of p2CT-His-MBP-Lbu-Cas13a-WT and p2CT-His-MBP-Lsh-Cas13a-WT        in E. coli Rosetta.
Protocol: Electro transformation
Participants: Ludwig, Chris
Observations:
1 µl of plasmid DNA was used.
Bacteria were plated on LBCarb + Cm plates
Results
This time bacteria grew well and a lot of single colonies were visible.
Preparation of 25 ml LBCarb + Cm cultures of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT and Rosetta p2CT-His-MBP-Lsh-Cas13a-WT and incubated at 37 °C overnight.
Protocol: Protein Expression
Participants: Ludwig
Thursday 4th
Harvesting of Rosetta cells with overexpressed Cas13a Lbu and Cas13a Lsh
Protocol: Protein expression
Participants: Ludwig, Chris
Results
Bacterial pellets were stored at -80 °C.
Friday 5th
Protein purification of Cas13a Lbu
Protocol: Äkta protein purification
Participants: Ludwig, Chris
Observations:
One test sample was run on the Äkta before the actual run. Only the second run was analyzed with a SDS PAGE.
Results
Protein purification of Cas13a Lsh
Protocol: Äkta protein purification
Participants: Ludwig, Chris
Results
Monday 8th
Transformation of TEV plasmid (form lab in the chemistry department) in DH5α
Protocol: Electro Transformation
Participants: Sven
Tuesday 9th
Dialysis of pooled peak fractions of Cas13a Lbu and Cas13a Lsh into ion exchange buffer
Protocol: Äkta protein purification
Participants: Ludwig, Chris, Sven
PCR of NT and T crDNA (for Cas13a Lbu and Cas13a Lsh)
Protocol:
Participants: Sven
Observations:
PCR didn't work.
Wednesday 10th
Test of different lysis methods for total RNA extraction with Norgen kit
Protocol: Total RNA Purification with Norgen Kit
Participants: Julian, Milica, Jorge
Observations:
Three lysis methods were used: sonifcation, lysozyme digestion and heat lysis in SDS.
Lysozyme digestion: According to protocol.
Heat lysis: Sample was incubated in 2% SDS at 75 °C for 15 min.
Results
See results from "Phenol/Chloroform purification of RNA from in-vitro transcription" from 11.05.17 for gel picture.
Upconcentration fo Cas13a Lbu and Cas13 Lsh after dialysis
Protocol: Äkta protein purification
Participants: Ludwig, Chris
Results
Thursday 11th
Test of different lysis methods for Phenol/Chloroform total RNA extraction.
Protocol: Phenol/Chloroform purification of RNA
Participants: Patrick, Julian
Observations:
Three lysis method were used: sonification, lysozyme digestion and heat lysis in SDS.
Lysozyme digestion: according to the total RNA purification with Norgen Kit protocol.
Heat lysis: Sample was incubated in 2% SDS at 75 °C for 15 min.
The samples were incubated at -80 °C overnight and the purification finished the next day.
Results
See results from "Phenol/Chloroform purification of RNA from in-vitro transcription" from 11.05.17.
Phenol/Chloroform purification of RNA from in-vitro transcription
Protocol: Phenol/Chloroform purification of RNA
Participants: Dawafuti, Sven
Observations
The samples were incubated at -80 °C overnight and the purification finished the next day.
Results
Friday 12th
Urea PAGE of in vitro transcription samples
Protocol: UREA PAGE
Participants: Sven
Results
Monday 15th
Preparation fo buffers for Heparin purification of Cas13a Lbu and Cas13a Lsh
Protocol: Äkta protein purification
Participants: Ludwig, Chris
Tuesday 16th
Heparin purification of Cas13a Lbu and Cas13 Lsh
Protocol: Äkta protein purification
Participants: Ludwig, Dawa
Results
Tuesday 23rd
PCR of Cas13a Lwa part1 and part2 and backbone pSB1C3_ccdB
Protocol: PCR
Participants: Ludwig, Chris, Rob
Observations:
Q5 polymerase was used
Primers Lwa part1 and part2: VF and VR
Primers pSB1C3_ccdB: p-pSB-GG-Fwd, p-pSB-GG-Rev
Results
Assembly of Cas13a Lwa part1 and part2 with pSB1C3_ccdB using GoldenGate Cloning
Protocol: GoldenGate Assembly
Participants: Ludwig, Chris, Rob
Observations:
Insert to vector ratio: 2:1 and 3:1
Electro Transformation of assembled Lwa parts and pSB1C3 into DH5α and plating on LBCm agar plates
Protocol: Electro Transformation
Participants: Ludwig, Chris, Rob
Observations:
Next day: Only a few colonies.
Electro Transformation of TEV biobricks (pSB1C3-BBa_K1319008 and pSB1C3-BBa_K1319004) and plating on LBCm agar plates
Protocol: Electro Transformation
Participants: Ludwig, Chris, Rob
Wednesday 24th
Repeat: Assembly of Cas13a Lwa part1 and part2 with pSB1C3_ccdB using GoldenGate Cloning
Protocol: GoldenGate Assembly
Participants: Ludwig, Erika, Dawa, Chris
Observations:
Insert to vector ratio: 2:1 and 3:1
Electro Transformation of assembled Lwa parts and pSB1C3 into DH5α and plating on LBCm agar plates
Protocol: Electro Transformation
Participants: Ludwig, Erika, Dawa, Chris
Repeat: PCR of Cas13a Lwa part1 and part2 and backbone pSB1C3_ccdB
Protocol: PCR
Participants: Ludwig, Erika, Dawa, Chris
Observations:
Q5 polymerase was used
Primers Lwa part1 and part2: VF and VR
Primers pSB1C3_ccdB: p-pSB-GG-Fwd, p-pSB-GG-Rev
Results
Preparation of 5 ml LBCm overnight culture of one picked DH5α pSB1C3-Cas13a-LwaLwa colony (from first GoldenGate assembly)
Protocol: -
Participants: Ludwig, Erika, Dawa, Chris
Thursday 25th
Miniprep of DH5α pSB1C3-Cas13a-Lwa overnight culture
Protocol: Minipreparation
Participants: Dawa, Chris
Observations:
pSB1C3-Cas13a-Lwa was then used for an analytical restriction digest and was sent for sequencing (with primer VR)
Analytical restriction digest with pSB1C3-Cas13a-LwaLwa with XbaI (single digest) and XbaI + SpeI (double digest)
Protocol: Restriction digest
Participants: Dawa, Chris
Results
Repeat: PCR of Lwa part2 and also GoldenGate was redone with new amplified Lwa part2
Protocol: PCR and GoldenGate
Participants:
Observations:
PCR Primers: VF, VR
Insert:Vecor 3:1 (GoldenGate)
Results
Friday 26th
Transformation of 1 µl GoldenGate sample in DH5α and plating on LBCm agar plates
Protocol: Transformation
Participants: Ludwig, Dawa, Chris
Observations:
No colonies visible
PCR of BBa_K1319004 and BBa_K1319008 (TEV) to check if biobricks are in plasmid
Protocol: PCR
Participants: Ludwig, Dawa, Chris
Results
Electro transformation of BBa_K1319004 and BBa_K1319008 (TEV) in DH5α and plating on LBCm agar plates
Protocol:
Participants:
Observations:
No colonies for BBa_K1319008
Monday 29th
Chemical transformation of BBa_K1319008 (TEV) in DH5α and plating on LBCm agar plates
Protocol: Chemical Transformation
Participants: Ludwig, Dawa, Erika
Observations:
3 µl (= 750 pg) were used
Chemical transformation of GolenGate sample from 24.05. and also from 26.05. in DH5α and plating on LBCm agar plates
Protocol: Chemical Transformation
Participants: Ludwig, Dawa, Erika
Observations:
5 µl for each transformation were used
Tuesday 30th
Colony PCR of DH5α pSB1C3-Cas13a-LwaLwa (from GoldenGate)
Protocol: cPCR
Participants: Chris, Dawa
Results
Colony PCR of DH5α pSB1C3-BBa_K1319004 and pSB1C3-BBa_K1319008 (TEV)
Protocol: cPCR
Participants: Chris, Dawa
Observations:
Primers: VF, VR
Overnight cultures from cultures that showed right cPCR bands were set up
Results
Wednesday 31st
Preparation of buffers for Cas13a purification (Elution, Wash, Ion exchange and SEC)
Protocol: Äkta purification
Participants: Milica, Ludwig, Flo, Sven, Jorge
Observations:
Minipreparation of DH5α pSB1C3-BBa_K1319004 and pSB1C3-BBa_K1319008 (TEV) overnight cultures
Protocol: Miniprep
Participants: Milica, Ludwig, Flo, Sven, Jorge
Observations:
Each construct was sent for sequencing with primers VR and VF
Thursday 1st
Colony PCR of DH5α pSB1C3-Cas13a-Lwa
Protocol: cPCR
Participants: Ludwig, Erika, Sven
Observations:
Primers: VR, VF
Results
Äkta purification of Cas13a Lbu and Cas13a Lwa (Affinity chromatography)
Protocol: Äkta protein purification
Participants: Ludwig, Erika, Sven
Results
Friday 2nd
Total RNA extraction of E. coli W3110 with Norgen kit
Protocol: Total RNA Purification with Norgen Kit
Participants: Jorge
Observations:
Instead of using TE and lysozyme to lyse the cells, they were resuspenden in 2% SDS in TAE and incubated at 75 ºC for 15 min.
The 6 samples were stored at -80 ºC and labeled T RNA #1-6
Monday 5th
Preparation of overnight cultures from Rosetta pSB1C3-His-MBP-Lbu-Cas13a-WT and Rosetta p2CT-His-MBP-Lsh-Cas13a-WT glycerol stocks, DH5α pSB-Cas13a-Lwa cPCR colony 5 and        DH5α pSB1C3-BBa_K1319004 cPCR colony 1 and DH5α pSB1C3-BBa_K1319008 cPCR colony 1
Protocol: -
Participants: Sven
RNA Adsorption Test on first 3D print
Participants: Jorge
Notes:
Total RNA sample 5 used, prepared on Friday 02/06 by Jorge (Concentration: 126 ng/µl)
Procedure:
Take 1 µl presample from stock ; add with 2 µl nf H2O and 3 µl 2x RNA LD
Put 5 µl of sample on device
Take 1 µl of sample immediately, add with 2 µl nf H2O and 3 µl 2x RNA LD
Wait 15 minutes
Take 1 µl of sample immediately, add with 2 µl nf H2O and 3 µl 2x RNA LD
Store at -80 °C
What still needs to be done:
Cook samples at 95 °C for 10 minutes and put directly on ice afterwards (to prevent refolding)
Load on Gel and quantify
Tuesday 6th
Inoculation of Rosetta pSB1C3-His-MBP-Lbu-Cas13a-WT and Rosetta p2CT-His-MBP-Lsh-Cas13a-WT overnight cultures in each 1 l LBCarb + Cm medium
Protocol:
Participants: Sven, Patrick
Preparation of glycerol stocks for DH5α pSB1C3-BBa_K1319004 cPCR and DH5α pSB1C3-BBa_K1319008
Protocol: Glycerol cryo stocks
Participants: Sven, Patrick
Minipreparation of DH5α pSB-Cas13a-Lwa cPCR colony 5, DH5α pSB1C3-BBa_K1319004 and DH5α pSB1C3-BBa_K1319008
Protocol: Miniprep
Participants:  Sven, Patrick
Wednesday 7th
Harvesting of bacteria with overexpressed Cas13a Lbu and Cas13a Lsh
Protocol: Äkta purification
Participants: Ludwig
Preparation of 5 ml LBCm overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5
Protocol: -
Participants: Jorge
Thursday 8th
Preparation of wash and elution buffer for Cas13a protein purification
Protocol: Äkta protein purification
Participants: Ludwig, Erika
Observations:
pH was checked and adapted after adding of TCEP
Purification of Cas13a Lbu and Cas13a Lsh (Affinity chromatography)
Protocol: Äkta protein purification
Participants: Ludwig, Erika, Sven
Results
Minipreparation of DH5α pSB-Cas13a-Lwa cPCR colony 5 and preparation of glycerol stock
Protocol: Minipreparation and glycerol stock
Participants: Ludwig, Erika, Sven
Friday 9th
PCR of pSB1C3-BBa_K1319008 (TEV) for adding a His tag
Protocol: PCR
Participants: Chris
Observations:
Primers: p-TEV-His-fwd and p-TEV-His-rev
Procedure was follwed by PCR cleanup and DpnI digestion
Results
Monday 12th
Purification of Cas13a Lbu and Cas13a Lsh (Size exclusion chromatography)
Protocol: Äkta purification
Participants: Sven, Ludwig, Benedikt, Milica
Observations:
Cas13a Lbu and Cas13a were upconcentrated to 1 ml each (centrifugal filters: MWCO: 30 kDa).        2 runs for each protein were necessary
Results
PCR of Lwa part2b (reordered gBlock without internal BsaI restriction site)
Protocol:
Participants: Sven, Ludwig, Benedikt, Milica
Observations:
Primers: VF, VR
Procedure was follwed by PCR cleanup
Results
Assembly of Lwa part1, part2b and pSB1C3-ccdB with GoldenGate and transformation in DH5α
Protocol: GoldenGate Cloning
Participants: Sven, Ludwig, Benedikt, Milica
Ligation of pSB1C3-BBa_K1319008 (TEV) after adding the His tag by overhang PCR
Protocol:
Participants: Sven, Ludwig, Benedikt, Milica
Observations:
Procedure was follwed by chemicial transformation in DH5α and plating on LbCm agar plates
Results
Tuesday 13th
Colony PCR of DH5α pSB-Cas13a-Lwa from GoldenGate assembly (with part2b)
Protocol:
Participants: Ludwig, Chris
Results
cPCR showed positive results. Cloning seemed to work, although there was the internal BsaI restriction site
Redone: Transformation of 15 µl ligation sample (of pSB1C3-BBa_K1319008 (TEV) after adding the His tag by overhang PCR)
Protocol:
Participants: Ludwig, Chris
Observations:
Colonies were visible the next day this time. 2 colonies were picked and resuspended in each 5 ml LBCm for overnight cultures
Preparation of 5 ml LBCm overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5
Protocol:
Participants:
Observations:
Inoculation from cyro stock
Results
Lysis-tests with acidic phenol chlorophorm extraction
Protocol: Alkaline lysis Protocol: Phenol-Chlorophorm extraction Protocol: Trizol Reagenz protocol
Participants: Julian, Patrick
Notes:
Three lysis methods were used: Alkaline Lysis, heat lysis with SDS and Trizol-Lyse.
No vortexing for lysis (may break up cells and won't be available in our device), only room-temperature (apart form SDS as reference)
Cells had OD of 5.97 -> 4.78*109 cells according to http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=1029158
Cells had OD of 5.97 -> 4.78*109 cells according to http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=1029158 -> dilluted 1:100 (to V=10mL) and pelleted 209 miL to get to 107 cells
A
B
C
D
1
Method
SDS
NaOH
TRIzol
2
prepare
1 mL TES (10mM Tris-HCl, 10mM EDTA, pH 8.0, 2% SDS -> TE-Buffer + SDS...
3
----Lysis
1x
3x
1x
4
separate to tubes Spin 300xG, 5 min ->Ice, TROzol to -20! (100 miL at OD 1 -> 107 cells, Max. according to TRIzolPaper
100 miL bactSusp
100 miL bact-susp
100 miL bact-suspension
5
No washing step, bad for mRNA (TRIzolP.)
300 µl (0.45 ml) TES resuspend pellet
6
12,5 uL Inhibitor (40k U/ml, 1 U/miL required)
15 min, 75 °C lysis, vortex
one probe after another
7
(Addition of inhibitor); Transform total volume to gel tube
resuspend thorougly in 66 µl (100 miL) Solution1
8
132 µl (200 miL) Sol2, INVERT 4x
9
wait 0, 1 ,3 min
10
add 99 µl (150 miL) Sol3, INVERT 4x
11
Total volume: 297 µl (0.45 ml)
12
--phenChloExtr
13
measure pH, add acid...
meassure Trizol PH, add acid if needed (pH <5)
14
add 600 µl [same V (0.45 ml)] Phe:chloro.
add 1 mL TRIzol to 107 cells
15
5 min incubation
16
add 0.2 mL CHLOROPHORM
17
incub 2.5 (2-3) min, RT
18
centri 12k g 15 min 4°C
19
angle 45°, extract top phase carefully
-> discard everything else, prot & DNA
-> which Vol for the TRIzol aqueous phase?
20
add 0.4 (0.375) ml Isopropanol, wait 10 min, RT
21
centrifuge 10 min 12k , 4°C ->
22
Aditional step, because RNA pellet was not entierly spinned down: Centrifugation (16000 rcf, 5 min, 4 °C)
big-pellet assay in fridge, but 1. interphase was punctured/gel(DNA?) stuck to pipette 2. s.u.
23
Aditional step, because suspected RNA fog was not entirely spinned down: remove upper and lower RNA fog phase
24
Additional step: Centrifugation (16000 rcf, 5 min, 4 °C)
25
No RNA pellet or RNA fog any more :(
26
Addition of 400 µl isopropanol -> No RNA pellet or fog any more ->prob. removed too much
27
discard supernatant -> RNA in gel-like pellet!
-> gel hardly visible, nearly draged it out of tube (stuck to pipette...)
28
resuspend in 0.75 ml 70 % (75% ideally) ethanol
-> storeed RNA at -20°C is stable up to 1 year...
29
vortex, centri 5 min 7,5k 4°
30
discard supernatant, air dry for 5-10 min (do NOT let completely dry out)
31
resuspend in RNAse free water 30 µl (20-50 miL)
->measurement...
Results
No pellets seen during procedure, concentration too low for photometer -> repeat with more cells
Wednesday 14th
Up-concentration of Cas13a Lbu and Cas13 Lsh after SEC and concentration measurement
Protocol:
Participants: Chris, Patrick
Observations:
Centrifual filters: MWCO: 30 kDa
Concentrations were measured using a Bradford assay
Results
Cas13 Lbu: 746 mg/ml, Cas13 Lsh: 120 mg/ml
Miniprep of overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5
Protocol: Minipreparation
Participants: Chris, Patrick
Observations:
Plasmid was again sent for sequencing with VR and VF
Results
First sequencing looked good
Friday 16th
Sending pSB-Cas13a-Lwa cPCR colony 5 for sequencing again with more primers
Protocol: Sequencing
Participants: Chris, Jorge
Observations:
Primer: p-seq-lwa 1-4
Preparation of first readout experiment with Cas13 Lbu and Cas13 Lsh
Protocol: Readout
Participants: Chris, Jorge
Observations:
Cleavage activity was tested with Aptamers (Malachite, Spinach) and RNase alert
Results
The Cas13a proteins were activated and cleaved the aptamers. This resulted in a decrease in fluorescence intensity.
Lysis-tests with acidic phenol chlorophorm extraction
Protocol: Lysis test pipetting scheme Protocol: Phenol-Chlorophorm extraction
Participants: Julian, Patrick
Notes:
Cell suspension had OD600 (extrapolated) of  2,220 -> 1.78 x 109 cells/ml according to http://www.genomics.agilent.com/biocalculators/calcODBacterial.jsp?_requestid=1029158
Experiments were done with 108 and 109 cells per sample, because it failed with 107 cells last time
Aspiration of supernatant in phenol-chlorophorm extraction: about 2/3 aspirated (200 µl) and 1/3 (100 µl) left to not destroy interphase
No pellet after RNA precipitation (-80 °C) and subsequent centrifugation in:
NaOH, 1 min, 108
SDS, 109
Trizol, 108
Mistakes:
SDS, 109: aspirated 100 µl instead of 200 µl supernatant
NaOH, 108, 3 min: contamination with DNA (<10% DNA interphase transfered)
NaOH, 109, 3 min: added + 200 µl of SDS 109 tube
! Did not calibrate alkaline lysis solution 3 to given pH -> probably more RNA degradation
! Did store RNA at -20 °C instead of -80 -> probably more RNA degradation
Results
Sample
[] [µg/ml]
adjusted C**
A260/A280
A260/A230
A230 [A]
A260 [A]
A280 [A]
A320 [A]
1
SDS, 108 cells
36.8
55
1.704
2.044
0.047
0.094
0.056
0.002
2
SDS, 109 cells*
119
357
1.693
2.069
0.146
0.300
0.178
0.002
3
NaOH, 0 min, 108 cells
35.2
53
1.725
2.146
0.042
0.089
0.052
0.001
4
NaOH, 0 min, 109 cells
586
879
1.923
1.993
0.740
1.470
0.767
0.005
5
NaOH, 1 min, 10^8 cells
73.6
110
1.235
0.586
0.317
0.187
0.152
0.003
6
NaOH, 1 min, 109 cells
370
555
1.945
1.958
0.475
0.927
0.478
0.003
7
NaOH, 3 min, 108 cells*
259
389
1.849
1.491
0.439
0.652
0.355
0.005
8
NaOH, 3 min, 109 cells*
275
413
1.850
2.212
0.316
0.693
0.368
0.005
9
TRIzol, 108 cells
194
265
1.362
0.221
2.200
0.487
0.358
0.002
10
TRIzol, 109 cells
160
218
1.429
0.488
0.822
0.403
0.283
0.003
Measured [RNA] by Implen Nanophotometer (20170616).
**adjusted for different supernatant V aspirated at extraction step
see also "Denaturing PAGE for ssRNA"  Monday, the 26th
Monday 19th
Preparation of 5 ml LBCm overnight culture of DH5α pSB-Cas13a-Lwa cPCR colony 5
Protocol: Overnight culture
Participants: Ludwig
Observations:
Inoculation from cryo stock
Preparation of 5 ml LBCm overnight culture of DH5α pSB1C3-BBa_K1319008-His (TEV) 1 and 2
Protocol: Overnight culture
Participants: Ludwig
Observations:
Inoculation from cryo stock
Electro transformation of pSB1C3-Cas13a-Lwa (cPCR colony 5) into E. coli BL21star
Protocol: Electro transformation
Participants: Ludwig, Chris
Observations:
pSB1C3-Cas13a-Lwa (cPCR colony 5) was verified by sequencing and is named now pSB1C3-Cas13a-Lwa
Tuesday 20th
Minipreparation of DH5α pSB-Cas13a-Lwa cPCR colony 5
Protocol: Minipreparation
Participants: Ludwig, Chris
Observations:
The plasmid was sent for sequencing with VR, VF, p-seq-lwa 1-4
Minipreparation of DH5α pSB1C3-BBa_K1319008-His (TEV) 1 and 2
Protocol: Minipreparation
Participants: Ludwig, Chris
Observations:
The plasmids were sent for sequencing with VR, VF
Preparation of 5 ml LBCm overnight culture of one picked BL21star pSB1C3-Cas13a-Lwa colony
Protocol: Overnight culture
Participants: Ludwig, Chris
Wednesday 21st
Minipreparation of BL21star pSB1C3-Cas13a-Lwa overnight culture and send for sequencing
Protocol: Minipreparation
Participants: Erika, Igor, Milica
Observations:
Primers: VF, VR, p-seq-lwa 1-4
Preparation of glycerol stock for BL21star pSB1C3-Cas13a-Lwa
Protocol: Glycerol stock
Participants: Erika, Igor, Milica
Friday 23rd
Time-series of alkaline lysis, in triplets
Protocol: Lysis test pipetting scheme
Participants: Julian, Patrick
Notes:
Did alkaline Lysis with inbubation/lysis times of 0 min (~10 sec ), 1 min, 3 min, 10 min, Heat+SDS as reference, no Trizol
Cell suspension with OD600 (extrapolated) of 2.182 -> 1.78 x 109 cells/ml
Phenol-Chlorophorm Precipitation at -80 was done until Wednesday
Results
see "Denaturing PAGE for ssRNA"  Thursday (not Monday!), the 29th
Monday 26th
Gel analysis of extracted RNA (July 16th !)
Protocol: Denaturing PAGE for ssRNA
Participants: Julian, Patrick
Notes:
see Friday, 16th
Results
Tuesday 27th
Electro Transformation of pSB1C3-BBa_K1319008-His (TEV) in BL21star and plating on LBCm agar plates
Protocol: Electro transformation
Participants: Ludwig, Chris
Observations:
1 µl was used
Preparation of 5 ml LBCm + Carb overnight culture of E. coli Rosetta p2CT-His-MBP-Lbu-Cas13a-WT
Protocol: Overnight culture
Participants: Ludwig, Chris
Preparation of 5 ml LBCm overnight culture of E. coli DH5α pSB1C3-BBa_K1319008-His 2
Protocol: Overnight culture
Participants: Ludwig, Chris
Wednesday 28th
Inoculation of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT in 4x 1 l 2xYTCm + Carb medium for expression of Cas13a Lbu
Protocol: Äkta protein purification
Participants: Chris, Ludwig,Jorge
Minipreparation of DH5α pSB1C3-BBa_K1319008-His 2 and send for sequencing
Protocol: Minipreparation and Sequencing
Participants: Chris, Ludwig, Jorge
Observations:
Primers: VF, VR
Preparation of 5 ml LBCm overnight culture of BL21star pSB1C3-BBa_K1319008-His from picked colony
Protocol: Overnight culture
Participants: Chris, Ludwig, Jorge
Plating of DH5α pSB1C3-Cas13a-Lwa glycerol stock on LBCm agar plates
Protocol:
Participants:
Observations:
Colonies should be checked if they grow the same and a cPCR should confirm that there is only one plasmid
Thursday 29th
Preparation of glycerol stock of BL21star pSB1C3-BBa_K1319008-His
Protocol: Glycerol stock
Participants: Ludwig, Chris
Observations:
From this glycerol stock a 5 ml LBCm overnight culture was prepared
A 5 ml LBCm overnight culture was prepared directly from this cryo stock
Harvesting of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT with overexpressed Cas13a Lbu
Protocol: Äkta protein purification
Participants: Ludwig, Chris
Observations:
The pellet was stored at -80 °C
Colony PCR of DH5α pSB1C3-Cas13a-Lwa (plated glycerol stock)
Protocol: cPCR
Participants: Ludwig, Chris
Results
Time-series of alkaline lysis, in triplets (continued)
Protocol: Phenol-Chlorophorm extraction
Participants: Patrick
Notes:
Continuing after -80 °C precipitation
Extremely big pellets after 99 % EtOH precipitation for all samples with SDS: Possibly contaminated with DNA
Results
SDS 10⁹ seems to have lower yield than NaOH -> maybe more SDS? (10⁸ has no problems...)
NaOH is done after "0" minutes (about 10 sec for adding base to all eppis and turning 3 times)
Sample
Number of sample
[] [µg/ml]
A260/A280
A260/A230
A230 [A]
A260 [A]
A280 [A]
A320 [A]
1
SDS, 108 cells
1
62.4
2.080
0.118
1.343
0.174
0.093
0.018
2
2
56.8
2.029
0.128
1.123
0.153
0.081
0.011
3
3
47.6
1.983
0.102
1.178
0.127
0.068
0.008
4
SDS, 109 cells
1
354
1.782
>0.4
>2.5
0.913
0.525
0.029
5
2
260
1.857
0.295
2.222
0.665
0.365
0.015
6
3
376
1.801
0.392
2.422
0.962
0.544
0.022
7
NaOH, 0 min, 108 cells
1
39.2
1.690
2.042
0.056
0.106
0.066
0.008
8
2
53.6
1.614
1.887
0.086
0.149
0.098
0.015
9
3
68.8
1.720
2.457
0.070
0.172
0.100
0.000
10
NaOH, 0 min, 109 cells
1
629
1.911
2.466
0.643
1.578
0.828
0.005
11
2
17.6
1.692
2.200
0.019
0.043
0.025
-0.001
12
3
619
1.897
2.453
0.636
1.553
0.821
0.005
13
NaOH, 1 min, 108 cells
1
52
1.806
2.203
0.059
0.130
0.072
0.000
14
2
33.2
1.729
2.371
0.035
0.083
0.048
0.000
15
3
58.4
1.738
2.393
0.061
0.146
0.084
0.000
16
NaOH, 1 min, 109 cells
1
606
1.893
2.469
0.620
1.522
0.807
0.006
17
2
627
1.898
2.431
0.651
1.574
0.832
0.006
18
3
22.4
1.806
2.435
0.023
0.056
0.031
0.000
19
NaOH, 3 min, 108 cells
1
44
1.864
2.444
0.044
0.109
0.058
-0.001
20
2
61.2
1.779
2.593
0.058
0.152
0.085
-0.001
21
3
5.6
1.400
2.800
0.004
0.013
0.009
-0.001
22
NaOH, 3 min, 109 cells
1
619
1.908
2.432
0.643
1.554
0.818
0.007
23
2
570
1.927
2.468
0.583
1.430
0.745
0.006
24
3
20.4
1.759
2.429
0.020
0.050
0.028
-0.001
25
NaOH, 10 min, 108 cells
1
10.8
1.588
2.250
0.012
0.027
0.017
0.000
26
2
8.8
1.571
3.143
0.006
0.021
0.013
-0.001
27
3
4.4
1.375
3.667
0.002
0.010
0.007
-0.001
28
NaOH, 10 min, 109 cells
1
503
1.916
2.499
0.508
1.262
0.661
0.005
29
2
564
1.911
2.487
0.572
1.415
0.743
0.005
30
3
611
1.902
2.467
0.625
1.533
0.809
0.006
Friday 30th
Minipreparation of BL21star pSB1C3-BBa_K1319008-His
Protocol: Minipreparation
Participants: Max
Observations:
The plasmid was sent for sequencing with primers VR and VF
Gel analysis of alkaline-extracted RNA time-series (July 23th !)
Protocol: Denaturing PAGE for ssRNA
Participants: Julian, Patrick
Notes:
see Friday, 23th
Results
Monday 3rd
Preparation of 5 ml LBCm overnight culture of BL21star pSB-Cas13a-Lwa
Protocol: Overnight culture
Participants: Max
Tuesday 4th
Inoculation of BL21star pSB-Cas13a-Lwa in 4x 1 l 2xYTCm medium for expression of Cas13a Lwa
Protocol: Ni NTA agarose protein purification
Participants: Max, Dawa
Purification of Cas13a Lbu
Protocol: Ni NTA agarose purification
Participants: Max, Dawa
Observations:
Fresh DTT was added to the lysis buffer
1 ml Ni-NTA agarose was used per 4 ml lysate
Sample was incubated 1 h at 4 °C while shaking
The procedure was followed by TEV cleavage overnigth while dialyzing
Results
Wednesday 5th
Harvesting of BL21star pSB-Cas13a-Lwa with overexpressed Cas13a Lwa
Protocol: Ni NTA protein purification
Participants: Max, Dawa, Erika
Observations:
Pellet was incubated for 1 h at -80 °C before it was used for purification
Results
Purification of Cas13a Lwa
Protocol: Ni NTA agarose purification
Participants: Max, Dawa, Erika
Observations:
Fresh DTT was added to the lysis buffer
1 ml Ni-NTA agarose was used per 4 ml lysate
Sample was incubated 1 h at 4 °C while shaking
Results
Precipitation of TEV plasmid for European meetup in Delft
Protocol: -
Participants: Max, Dawa, Erika
Observations:
pSB1C3-BBa_K1319008-His was lyophylized by Ethanol precipitation and evaporation
Friday 7th
Reverse Ni-NTA purification of cut Cas13a Lbu
Protocol: Ni NTA agarose protein purification
Participants: Rob
Observations:
Ni NTA agarose binds the cleaved His-MBP tag, Cas13a Lbu is in the flow-through
Results
Monday 10th
Preparation of 50 ml LBCm overnight culture of BL21star pSB1C3-BBa_K1319008-His (TEV)
Protocol: Overnight culture
Participants: Rob, Ludwig, Rob
Tuesday 11th
Inoculation of BL21star pSB1C3-BBa_K1319008-His (TEV) in 3x 1 l 2xYTCm medium for expression of the TEV protease
Protocol: Äkta protein purification
Participants: Ludwig, Max
Observations:
Induction with 1 mM IPTG at OD600 of 0.6
Expression at 16 °C overnight
RNase alert assay with Cas13a Lbu and Cas13a Lsh
Protocol: Cas13a activity assay
Participants: Igor, Rob
Results
Wednesday 12th
Harvesting of BL21star pSB1C3-BBa_K1319008-His with overexpressed TEV protease
Protocol: Äkta protein purification
Participants: Ludwig, Milica
Friday 14th
Protein purification of TEV protease
Protocol: Äkta protein purification
Participants: Ludwig, Max, Sven
Results
Cas13a activity assay with RNase alert
Protocol: Cas13 activity assay
Participants: Dawa, Rob
Results
RNA Extraction with commercial kit
Participants: Patrick
Notes:
used ReliaPrep miRNA Cell and Tissue Miniprep System (Promega) according to instructions
Specification: Use in range of 102 to 106 cells. Tested 106 and 108 cells
Results
No nanodrop-measureable amount of RNA/DNA with 106 cell, <10 ug/ml RNA with 108 cells         Terrible yield, repeat again to be sure
Monday 17th
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
RNA Extraction with commercial kit
Participants: Patrick
Notes:
used ReliaPrep miRNA Cell and Tissue Miniprep System (Promega) according to instructions
Specification: Use in range of 102 to 106 cells. Tested 106 and 108 cells
Results
Again no measureable amount of RNA for 106 cells.                     For 108 cells, RNA concentration was increased through column-binding and elution (small elution volume)                     Elution seems effective, second elution has only 20 % of first elution's amount of RNA
Tuesday 18th
Preparation of 40 ml LBCm + Carb overnight culture of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT
Protocol: Äkta protein purification
Participants: Chris, Max, Dawa
Wednesday 19th
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
Inoculation of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT in 3x 1 l 2xYTCm + Carb medium for expression of Cas13a Lbu
Protocol: Äkta protein purification
Participants: Max, Dawa
Thursday 20th
Size exclusion chromatography of affinity purified TEV protease
Protocol: Äkta protein purification
Participants: Max, Sven, Ludwig
Results
Harvesting of Rosetta p2CT-His-MBP-Lbu-Cas13a-WT with overexpressed Cas13 Lbu
Protocol: Äkta protein purification
Participants: Max, Sven, Ludwig
In vitro Transcription of target RNA, Lbu crRNA and Lsh crRNA
Protocol: In vitro Transcription
Participants: Igor
Observations:
Procedure was followed by DNAase treatment and phenol chloroform extraction
Results
Friday 21st
PCR of gBlock of aeBlue
Protocol: PCR
Participants: Chris
Monday 24th
Digestion of insert and pSB1C3 with Pst1-HF and EcoRI-HF
Protocol: Digestion and Ligation
Participants: Chris
Observations:
Plasmid was dephosporylated with arctic phosphatase and insert was ligated with plasmid overnight at 16 °C
Tuesday 25th
Electro transformation of aeBlue ligation sample in E. coli Turbo cells
Protocol: Electro transformation
Participants: Max
Wednesday 26th
Colony PCR of plated pSB1C3-aeBlue
Protocol: cPCR
Participants: Max
Results
Preparation of 5 ml LBCm overnight cultures (picked colonies of aeBlue ligation)
Protocol: Overnight culture
Participants: Max
Thursday 27th
Minipreparation of Turbo pSB1C3-aeBlue overnight cultures and send for sequencing
Protocol: Minipreparation and sequencing
Participants: Max
Friday 28th
Upconcentration of TEV protease to 2 mg/ml and stored in TEV storage buffer
Protocol: Äkta protein purification
Participants: Max
Monday 31th
Tuesday 1st
Intein-Extein-Readout
Protocol: DNA Gel-Purification and Genomic DNA Purification
Participants: Max
Observations:
Purification of b-Gal fragments from 1 % Agarose Gel
60 µl per lane
Genomic DNA purification from E.coli W3110+
Results
DNA was lost during the purification
Wednesday 2nd
Intein-Extein-Readout
Protocol: Q5-PCR and PCR-Cleanup kit
Participants: Max
Observations:
Repeated PCR of beta-Gal N and C Terminal fragments from 31.7.
3 different Templates:
W3110 genomic DNA
W3110 genomic DNA diluted 1:100
W3110 2 µl of cryostock
Clean-up of PCR with Purification Kit and elution in 33 µl nf water
InterLab Study calibration
Protocol: IGEM HQ InterLab Study Protocol
Participants: Dawafuti Erika
Observations:
Patch length correction was on in the plate reader.
Calibration for the plate reader experiment done with LUDOX and Fluorescein.
We saw no difference between LUDOX and water when measuring at an OD of 600 nm.
For the fluorescein measurement, the best gain was 600 using 485 nm/520 nm.
Plasmids from the plate 7 (positive and negative control, and devices 1-6) were resuspended.
Thursday 3rd
Intein-Extein-Readout
Protocol: Agarose-Gel
Participants: Max
Observations:
Gel of purified PCR
1,5% small Gel, 120 V, 30 minutes
Results
PCR with undiluted extracted DNA didn't work.
Monday 7th
Protein-Purifcation of Lbu and Lsh
Protocol: -
Participants: Max, Ludwig, Teeradon
Observations:
Inoculation of main cultures of Rosetta p2CT-His-MBP-Lbu_C13a_WT and Lsh in LB(Carb+Cm)
InterLab Study Day 1: Transformation
Protocol: IGEM HQ InterLab Study Protocol
Participants: Dawafuti Erika
Observations:
Each plasmids was transformed two times: with chemical transformation and with electroporation, for a total of 16 plasmids.
Results
Every plate except the ones for Device 1 had colonies.
Tuesday 8th
First test with 3D printed 96-well plate
Protocol: -
Participants: Max
Observations:
Tested different concentrations of fluorescein in a 3D printed 96 well plate on paper.
Scanned at differnt gains at a constant focal height.
Dilutions were 1:20; 1:400 and 1:8000 of Fluorescein-FITC in water
Results
No light bleeding was observed in the experiment
InterLab Study Day 2: Colony transfer to liquid culture
Protocol: IGEM HQ InterLab Study Protocol
Participants: Dawafuti Erika
Observations:
Device 1 was again transformed, as no colonies grew on the plate.
Liquid cultures for the 7 plasmids whose plates showed colonies (all except Device 1) were made in duplicate.
The plates were stored at -4 °C.
Wednesday 9th
Expression and Harvesting of Cas13a variants
Protocol: Protein Purification
Participants: Max, Ludwig
Observations:
Harvested Cas13a Lbu overexpression.
Inoculated 2 litres 2xYT media with Rosetta Cas13a Lsh and let them grow at 16 °C overnight with 1 mM IPTG
InterLab Study Day 3: Plate reader experiments
Protocol: IGEM HQ InterLab Study Protocol
Participants: Dawafuti Erika
Observations:
After measuring the OD600 of the O/N cultures, they were diluted with LB+Cm to an OD600 of 0.02, in duplicates.
The absorbance and fluorescence of the diluted cultures were measured at 0, 2, 4 and 6 hours after the cultures were set.
Patch length correction was on in the plate reader.
Device 1 was again transformed, as no colonies grew on the plate.
Results
Transformation of Device 1 was successful.
Thursday 10th
His Purification of Cas13a Lbu and Harvesting of Cas13a Lsh
Protocol: His-Purification
Participants: Max, Ludwig, Milica, Sven, Benedikt
Observations:
His purification of Cas13a Lbu according to protocol.
Elution in 4 times 4 ml.
ON Dialysis with TEV protease in 2 litres of Lysis buffer.
Harvesting of Cas13a Lsh according to protocol
InterLab Study Day 4: Preparation of O/N cultures for plate reader experiments
Protocol: IGEM HQ InterLab Study Protocol
Participants: Dawafuti Erika
Observations:
O/N liquid cultures in duplicate were set up from all 8 Devices.
Annealing of Invitro Transcription Targe Inhibitor Circuit 3a (IC3a)
Protocol:
Participants: Florian
Observations:
Invitro Transcription of IC3a
Protocol: In vitro transcription
Participants: Florian
Observations:
Incubated over night at 37 ºC
Results
See results of PEC from 11.10.2017
Friday 11th
Reverse HIS purification and SEC of Cas13a Lbu
Protocol: His purification and SEC protein purification
Participants: Ludwig, Benedikt, Max, Sven
Observations:
Incubation  of cleaved Lbu sample with Ni-NTA beads (ca. 7 mL) (1 h, 4 °C)
Application on column, keep FT (there is our protein)
Elution of His-MBP with ca. 10 mL Elution buffer (for gel analysis)
Washing of Ni-NTA bead (3x with water) and storage in 20 % EtOH
Up-concentration of Cas13a Lbu and buffer exchange in SEC buffer (centrifugal filter: MWCO: 30 kDa, 4 °C, 5000 RCF)
Results
See Gel image of 14.8. for results.
o
Protocol: Restriction digest, Phenol-chlorophorm purification of RNA
Participants: Florian
Observations:
Phenol-Chloroform with the wrong pH was used.
Results
Sample was lost due to the use of the wrong Phenol-Chloroform solution.
InterLab Study Day 5: Plate reader calibration and experiments
Protocol: IGEM HQ InterLab Study Protocol
Participants: Dawafuti Erika
Observations:
Patch length correction was off in the plate reader.
Calibration for the plate reader experiment done with LUDOX and Fluorescein.
We saw the difference in measurements between LUDOX and water.
After measuring the OD600 of the O/N cultures, they were diluted with LB+Cm to an OD600 of 0.02, in duplicates.
The absorbance and fluorescence of the diluted cultures were measured at 0, 2, 4 and 6 hours after the cultures were set.
Results
Monday 14th
SDS PAGE of Lbu purifcations
Protocol: SDS-PAGE
Participants: Ludwig
Observations:
SDS-PAGE of Lbu  purification from last week
Invitro Transcription of IC3a
Protocol: In vitro transcription
Participants: Florian
Observations:
Results
See results of Restriction digestion and PCE from 15.08.17
Tuesday 15th
DNAse I digest and PCE of the RNA
Protocol: Restriction digest, Phenol-chlorophorm purification of RNA
Participants: Florian
Observations:
Phenol-Chloroform with the wrong pH was used.
Results
IC3A has a length of 33 bp = band on top.
Wednesday 16th
Cas13a preparation
Protocol: -
Participants: Benedikt, Ludwig
Observations:
Concentration measurment of Lbu, Lsh and Lwa with Bradford essay
Results
Concentration of proteins measured(µg/ml):
718 - Lbu 13.6 (pooled fractions 6, 7, 8)
387 - Lbu 12.7
90,8 - Lsh 13.6 (pooled fraction 7,8,9)
379 -Lwa 5.7 (elution 1)
1620 - Lwa 5.7 (elution 2)
818 - Lwa 5.7 (elution 3)
405 -Lwa 5.7 (elution 4)
Thursday 17th
DNA RNA Hybrid Annealing of IC3A with Circuit 3 Activator (C3A) – IC3a concentration screen
Protocol:
Participants: Florian
Observations:
Incubation at room temperature for 1 h of DNA and RNA with varied concentrations of IC3a (RNA).
Results
4% Agarose does not seem to work for separation. No oligos at all visible after staining with SybrGreen II and SybrGold.
Protein-Expression of Lbu and Lwa
Protocol: Protein-Expression
Participants: Benedikt, Ludwig, Milica
Observations:
Expression of Rosetta Cas13a Lbu (3L 2xYT Medium, 16°C ON)
Expression of Rosetta Cas13a Lwa (2L 2xYT Medium, 16°C ON)
Friday 18th
Harvesting and Purification of Cas13a Lbu and Lsh
Protocol: Protein Purification
Participants: Benedikt, Milica
Observations:
Harvesting of both expressions
Purification of Cas13a Lbu according to protocol.
Elution in 4 times 3 ml elution buffer.
Results
Sunday 20th
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
Monday 21st
Continuation of Cas13a purification
Protocol: His-Purification
Participants: Benedikt, Milica, Max
Observations:
Continued His purification of Cas13a Lbu
Overnight dialysis with TEV protesae
Results
As visible on the gel the elutions 1 and 2 contained the highest amount of protein and were pooled for the TEV cleavage.
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
FINA extraction for DNA with E. coli W3110 pre-purified gDNA
Protocol: FINA Extraction for DNA
Participants: Jorge
Observations:
The gDNA was pre-purified with the Phenol/Chloroform method.
The gDNA solution was diluted 1:100, 1:1000, 1:100000, 1:10e6 and 1:10e8. Each dilution was then used as a sample for the FINA extraction.
From each dilution, two extractions were performed. One as in protocol (1N-5N), the other without the NaOH washing step (1-5).
As negative control (KK) FINA extraction was performed as in protocol with nuclease free water as sample.
Q5-PCR for beta-galactosidase from E. coli with FINA-DNA samples and purified gDNA from 21.08.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge
Observations
For the membranes, no template was directly pipette. Instead, the membranes were put in the tube.
p-bGal_N_N was used as forward primer
p-bGal_N_C was used as reverse primer
Annealing was performed at 61 ºC
The amplification step was 90 s long
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
1N-5N: 1:100-1:10e8 dilutions FINA extraction performed as in protocol.
1-5: 1:100-1:10e8 dilutions FINA extraction without washing step.
1K-5K: 1:100-1:10e8 dilutions.
KK: negative control for FINA extraction (nuclease free water as sample).
Tuesday 22nd
Reverse His purification of Cas13a Lbu
Protocol: His purification
Participants: Max, Benedikt, Milica
Observations:
Incubation (shaker!!) of cleaved Lbu sample with Ni-NTA beads (ca. 7 mL) (1 h, ice)
Application on column, keep FT (there is our protein) (on ice)
Washing with 10 mL lysis-buffer
Elution of His-MBP with ca. 10 mL Elution buffer (for gel analysis)
Washing of Ni-NTA bead (3x5mL with water) and storage in 20 % EtOH
Results
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
FINA extraction for DNA with E. coli W3110 cell culture
Protocol: FINA Extraction for DNA
Participants: Jorge
Observations:
The extraction was done when the culture had an OD600 of 1.2.
The cells were diluted 1:1, 1:100 and 1:1000 before the extraction.
Each dilution was used as sample twice: as in protocol (1N-3N and P1N-P3N) and washed with 300 ul lysis buffer from RNA extraction kit (Norgen, P/N 17200) (1G-3G and P1G-P3G).
Membranes 1N-3N and 1G-3G were put in 40 ul nuclease free water and incubated at RT for 10 min in order for the bounded DNA to go into solution.
Q5-PCR for beta-galactosidase from E. coli with FINA-DNA samples from 22.08.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge
Observations
The membranes from the FINA extraction were directly put into the PCR-mix.  No template DNA was directly pipetted.
p-bGal_N_N was used as forward primer.
p-bGal_N_C was used as reverse primer.
Annealing was performed at 61 ºC.
The amplification step was 90 s long.
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products.
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
1N-3N: 1:1-1:1000 dilutions FINA extraction performed as in protocol, loaded in gel after elution (no PCR).
1G-3G: 1:1-1:1000 dilutions FINA extraction performed with lysis buffer as washing step, loaded in gel after elution (no PCR).
P1N-P3N: 1:1-1:1000 dilutions FINA extraction performed as in protocol.
P1G-P3G: 1:1-1:1000 dilutions FINA extraction with lysis buffer as washing buffer.
KK: negative control for FINA extraction (nuclease free water as sample).
Wednesday 23rd
Cas13a purification
Protocol: Size Exclusion Chromatography
Participants: Benedikt, Milica
Observations:
Measuring concentration after Reverse His Lbu
Upconcentration and buffer exchange  of Lbu in 30 kDa Centricon
SEC Run of Lbu according to protcol
Results
See SDS PAGE of 24.8. to see the SEC result.
FINA extraction for DNA with E. coli W3110 cell culture
Protocol: FINA Extraction for DNA
Participants: Jorge, Benedikt
Observations:
The extraction was done when the culture had an OD600 of 2.33.
The cells were diluted 1:1, 1:100, 1:10000, 1:10e6 and 1:10e8 before the extraction.
Each dilution was used as a sample for the FINA extraction. In addition, three 1:1 dilutions were used as samples for FINA extraction: two according to protocol and the third with 300 ul lysis buffer as washing buffer. One of the 1:1 membranes was put in nuclease free water and incubated at RT for 20 min in order for the bounded DNA to go into solution.
Q5-PCR for beta-galactosidase from E. coli with FINA-DNA samples from 23.08.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge, Benedikt
Observations
The membranes from the FINA extraction were directly put into the PCR-mix.  No template DNA was directly pipetted.
p-bGal_N_N was used as forward primer.
p-bGal_N_C was used as reverse primer.
Annealing was performed at 61 ºC.
The amplification step was 90 s long.
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products.
Afte the PCR, only 2,5,L and KK were loaded in the gel, as the other tubes had no liquid.
The gel was loaded before it was submerged in TAE-buffer.
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
2: 1:100 dilution FINA extraction performed as in protocol
L: 1:1 dilution FINA extraction performed with 300 ul lysis buffer as washing buffer.
KK: negative control for FINA extraction (LB-medium as sample).
Thursday 24th
SDS-Gel of SEC-Run from 23.8.
Protocol: SDS-PAGE
Participants: Chris
Observations:
10% SDS PAGE of SEC Run samples
Marker; Sample prior to run; Fractions 5-16;  FT Centricon
Results
The samples with the highest protein concentration were pooled and used for further experiments.
DNA-RNA-Hybrid-Building
Protocol:
Participants: Florian
Observations:
Native PAGE for short DNA oligos
Protocol:
Participants: Florian
Observations:
Results
Slight band shift of the oligo between 4 and 5 µM.
Slight band shift of the oligo between 4 and 5 µM.
Looks better than 30% PAGE in its resolution of the DNA oligos.
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
FINA extraction for DNA with E. coli W3110 cell culture (Repetition of the experiment from 23.08.17)
Protocol: FINA extraction for DNA
Participants: Jorge, Benedikt
Observations:
As there were problems with the gel from the 23th, we repeated the experiment.
The extraction was done when the culture had an OD600 of 2.78.
The cells were diluted 1:1, 1:100, 1:10000, 1:10e6 and 1:10e8 before the extraction.
Each dilution was used as a sample for the FINA extraction. In addition, three 1:1 dilutions were used as samples for FINA extraction: two according to protocol and the third with 300 ul lysis buffer as washing buffer. One of the 1:1 membranes was put in nuclease free water and incubated at RT for 20 min in order for the bounded DNA to go into solution.
Q5-PCR for beta-galactosidase from E. coli with FINA-DNA samples from 24.08.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge, Benedikt
Observations
The membranes from the FINA extraction were directly put into the PCR-mix.  No template DNA was directly pipetted.
p-bGal_N_N was used as forward primer.
p-bGal_N_C was used as reverse primer.
Annealing was performed at 61 ºC.
The amplification step was 90 s long.
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products.
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
1-5: 1:1-1:10e8 dilutions FINA extraction performed as in protocol
L: 1:1 dilution FINA extraction performed with 300 ul lysis buffer as washing buffer.
KK: negative control for FINA extraction (LB-medium as sample).
Total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT with Norgen kit
Protocol: Total RNA Purification with Norgen Kit
Participants: Jorge, Benedikt
Observations:
2 tubes stored at -80 ºC. Labelled gRNA #1, gRNA #2.
Results
Phenol/Chloroform total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT
Protocol: Phenol/Chloroform purification of RNA
Participants: Jorge
Observations:
The incubation step at -80 ºC was done, O/N.
2 tubes stored at -80 ºC. Labelled gRNA PEC #1, gRNA PEC #2.
Results
Friday 25th
Continuation of Phenol/Chloroform total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT from yesterday
Protocol: Phenol/Chloroform purification for RNA
Participants: Jorge
Observations:
See entry from 24.08.17
Monday 28th
DNA RNA Hybrid Annealing of IC3A with Circuit 3 Activator (C3A) – IC3A concentration screen (high concentration)
Protocol:
Participants: Florian
Observations:
Results
Native PAGE 20 %
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
Tuesday 29th
Intein-Extein-Readout
Protocol: PCR with Q5 Polymerase, PCR Cleanup Kit, Restriction digest and Ligation
Participants: Max
Observations:
Resuspension of gblocks Split_Uni C and Split_Uni N at a conc. of 100 nMol
PCR of SPlit Uni C und Split Uni N with Q5 Polymerase
Clean-up with Monarch Kit
Digestion of PCR products and psB1C3 with PstI-HF und EcoRI-HF in Cutsmart buffer
Dephosphorilation with Antarctic Phosphatase
Ligation ON with T4 DNA Ligase
FINA extraction for DNA with E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT
Protocol: FINA extraction for DNA
Participants: Jorge
Observations:
The extraction was done when the culture had an OD600 of 0.88.
Na: Extraction as in protocol. Cl: Washed with 10 mM HCl instead of NaOH. H: incubated 10 min at 80 ºC. L: 10 min incubated in 1 mg/ml lysozyme. LH: 10 min incubated in 1 mg/ml lysozyme at 80 ºC.
Q5-PCR for beta-galactosidase from E. coli with FINA-DNA samples from 24.08.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge
Observations
The membranes from the FINA extraction were directly put into the PCR-mix.  No template DNA was directly pipetted.
p-bGal_N_N was used as forward primer.
p-bGal_N_C was used as reverse primer.
Annealing was performed at 61 ºC.
The amplification step was 90 s long.
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products.
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
See Q5-PCR from 30.08.17
FINA extraction for RNA with purified total RNA (gRNA PEC #1)
Protocol: FINA extraction for RNA
Participants: Jorge
Observations:
First strand cDNA synthesis from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT purified total RNA (Phenol/Chloroform extraction from 24.08.17) and FINA-RNA eluate
Protocol: First strand cDNA synthesis
Participants: Jorge
Observations:
p-bGal_N_C was used as primer.
Sample gRNA PEC #1 from 24.08.17 was diluted 1:4 and used as sample P.
The eluate from the FINA extraction for RNA from 29.08.17 was used as sample S.
For both reactions (S and P), 6 ul sample were pipetted.
Wednesday 30th
DNA and RNA hybrid of 3C-A and IC3-A
Protocol:
Participants: Florian
Observations:
Concentration screen IC3a between 1 and 5 µM. Incubation for dimerization again for 1 hour and at room temperature.
Results
20% Native PAGE, 12.5 mM MgCl2
Shift of DNA-oligo band with higher concentration of IC3a RNA until 3.5 µM. No band of single DNA oligo C3a at 5 µM visible anymore. Hypothesis of full dimerization, but with a big background of single stranded IC3a RNA.
Q5-PCR for beta-galactosidase from E. coli with cDNA samples from 29.08.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge
Observations:
p-bGal_N_N was used as forward primer.
p-bGal_N_C was used as reverse primer.
Annealing was performed at 61 ºC.
The amplification step was 90 s long.
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products.
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
Na: FINA extraction performed as in protocol.
Cl: FINA extraction performed with 10 mM HCl instead of NaOH.
L: Cell solution incubated 10 min in 1 mg/ml lysozyme. No washing step in FINA
H: Cell solution incubated 10 min at 80 ºC. No washing step in FINA
LH: Cell solution incubated 10 min at 80 ºC in 1 mg/ml lysozyme. No washing step in FINA
K-: negative control FINA (LB-medium used as sample).
P:  1:4 dilution of gRNA PEC #1 after reverse transcription.
S:  1:4 dilution of gRNA PEC #1 after FINA extraction for RNA and reverse transcription.
KK:  Negative control for the cDNA synthesis (water instead of MuLV-RT used)
Thursday 31st
Intein-Extein-Readout
Protocol: E. comp Transformation
Participants: Max
Observations:
E. comp transformation of E. coli Turbo with 1 µl of overnight ligation of 27.8.
Platted on Cm plates overnight.
RNA extraction from induced Lbu-strain
Protocol: Lysis test pipetting scheme Protocol: Phenol-Chlorophorm extraction Protocol: Denaturing PAGE for ssRNA
Participants: Dawa
Notes:
Induced pSB1C3_lbu with IPTG before lysis
Two cultures lysed with SDS + Heat only
No OD600 measurement
Results
See Friday, 1st Sept.
Friday 1st
Intein-Extein-Readout
Protocol: Ligation
Participants: Max
Observations:
Plates showed no colonies, as there wasn't enough backbone used.
Results
Repeated digestion and Ligation from Thursday 31.8 and ligated 1 C-Terminal and 3 N-Terminal fragments respectively.
Intein-Extein-Readout
Protocol: -
Participants: Max
Observations:
Plates showed no colonies, as there was no SOC step for outgrowth
Repeated digestion and Ligation
Ligated 1 C-Terminal and 3 N-Terminal fragments into psb1C3 and psb4A5
RNA extraction from induced Lbu-strain (continued)
Protocol: Denaturing PAGE for ssRNA
Participants: Dawa, Julian
Notes:
Induced pSB1C3_lbu with IPTG before lysis
Two cultures lysed with SDS + Heat only
No OD600 measurement
Results
A
B
C
1
sample
conc ~Nanodrop
260/280
2
1 (non-induced)
2192
1.8
3
2 (non-induced)
2320
1.8
4
3 (induced)
508
1.69
5
4 (induced)
442
1.66
6
1,2 showed precipitation of RNA ->measurement/gel wrong if not vortexed completely
A260/280 ration hints to impurities or DNA  ->maybe due to too many cells
Nukleotide-silica binding with different buffers
Protocol:
Participants: Julian
Notes:
Used 10 ul of RNA-solution from Aug 31 RNA-extraction by Dawa per sample
Did NOT wash silica particles before with binding buffer (pH) and NOT during procedure with ethanol.
Results
Both buffers show terrible A260/280 ration, need washing step and need to adjust for acidic silica-solution
Saturday 2nd
Intein-Extein-Readout
Protocol: Chem_Trafo
Participants: Julian, Milica, Jorge
Observations:
Three lysis methods were used: sonifcation, lysozyme digestion and heat lysis in SDS.        All
Lysozyme digestion: According to protocol.
Heat lysis: Sample was incubated in 2% SDS at 75 °C for 15 min.
Results
See results from "Phenol/Chloroform purification of RNA from in-vitro transcription" from 11.05.17 for gel picture.
Nukleotide-silica binding with different buffers
Protocol:
Participants: Julian
Notes:
Used 10 ul of RNA-solution from Aug 31 RNA-extraction by Dawa
Washed according to protocol
Results
Acetate-buffer results acceptable now, GuSCN-buffer still a problem.GuSCN probably too difficult clean up, needs too many washing steps.
Monday 4th
Intein-Extein-Readout
Protocol: E. comp Transformation
Participants: Max
Observations:
Plates showed no colonies as there was no SOC step.
Retransformation of Ligation from Friday into E.Coli Turbo e. comp.
Total RNA extraction of E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT with Norgen kit
Protocol: Total RNA Purification with Norgen Kit
Participants: Jorge, Dawafuti
Observations:
Stored at -80 ºC as TRNA Kit #1-4
Phenol/Chloroform total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT
Protocol: Protocol
Participants: Jorge, Dawafuti
Observations
Stored at -80 ºC as TRNA PC #1-4.
Results
See results Urea-SDS-PAGE from 05.09.17.
Nukleotide-silica binding with different buffers
Protocol:
Participants: Julian
Notes:
Did NOT incubate silica with acetate buffer for a day, prepared it fresh
Used 10 ul of RNA-solution from Aug 31 RNA-extraction by Dawa
Washed according to protocol, but did NOT wait a day for silica-buffer interaction with Acetate-buffer
Results
Terrible yield, need buffer-silica incubation in protocol.
Tuesday 5th
DNA Amplification Test using Recombinase Polymerase Amplification
Protocol: Recombinase Polymerase Amplification
Participants: Sven
Observations:
DNA Amplification with RPA was tested using the provided Control Reaction by TwistDx and our BioBrick His6-TEV plasmid             incubation times of 40 minutes for Control and Sample 1 and 60 minutes for sample 2. Differences due to incubation time             was tested.
RPA according to protocol
PCR cleanup according to protocol.
Results
Positive Control worked.
RPA of TEV could not produce the whole (~1300 bp) construct but would produce amplicons of shorter size (100-500 bp) which was anticipated since TwistDx protocol says optimal amplicon length is 500 bp.
Full length double-stranded DNA could consequently not be formed.
Intein-Extein-Readout
Protocol: Agarose-Gel
Participants: Max
Observations:
Ran a small 1.75% TAE-Agarose Gel at 150 V for 50 minutes and post stained with Sybr Gold
Overnight Ligation of Intein-Extein-Construct
Protocol: Ligation of DNA-Fragments
Participants: Max
Observations:
Overnight ligation in ice bath with psb1C3 and psb4A5 as backbones and a 3:1 molar excess of insert
200 ng backbone
Preparation of chemical competent Turbo cells
Protocol: Preparation of chemical competent cells
Participants: Max
Observations:
Plated one aliquot as negative controll
Results
No colonies on plates, therefore usefull competent cells.
Invitro Transcription IC3a RNA
Protocol: In vitro transcription
Participants: Florian
Observations:
Results
See results of DNAseI digestion and PCE from 06.09.17
Phenol/Chloroform total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT (continued)
Protocol: Denaturing PAGE for ssRNA
Participants: Dawa, Jorge
Notes:
Ran urea gel
Results
DNA Amplification Test using Recombinase Polymerase Amplification
Protocol: Recombinase Polymerase Amplification
Participants: Sven
Observations:
DNA Amplification with RPA was tested using the provided Control Reaction by TwistDx and our BioBrick His6-TEV plasmid             incubation times of 40 minutes for Control and Sample 1 and 60 minutes for sample 2. Differences due to incubation time             was tested.
RPA according to protocol
PCR cleanup according to protocol.
Results
Positive Control worked. RPA of TEV could not produce the whole (~1300 bp) construct but would produce amplicons of shorter size (100-500 bp)                     which was anticipated since TwistDx protocol says optimal amplicon length is 500 bp. Full length double-stranded DNA could consequently not be formed.
Wednesday 6th
RNaseAlert Experiment with E.coli RNA with different concentrations of target E.coli RNA
Protocol: RNase activity
Participants: Dawa, Jorge, Igor
Notes:
Run1: Amount of RNA: 1 nm Cas13a, 10 nm crRNA
Run2: Amount of RNA: 1 nm Cas13a, 10 nm crRNA
img src="https://static.igem.org/mediawiki/2017/b/be/T--Munich--pic--platereader_pipScheme_6_09_Ecoli.png" >
Intein-Extein-Readout
Protocol: Chem and electro Transformation
Participants: Max
Observations:
Transformation into chem and e. competent E.coli Turbo and DH5 alpha cells.
Results
No colonies in the evening, therefore the plates were left at room temperature overnight.
DNAse I digest and PCE of invitro transcribed IC3a RNA
Protocol: Restriction digest,Phenol-chlorophorm purification of DNA
Participants: Florian
Observations:
Results
6 M UREA-PAGE 15% of IC3a RNA
Thursday 7th
RNA Intein-Extein-Readout
Protocol: Colony PCR
Participants: Max
Observations:
Picked 128 colonies from plates of 06.09.17
4 strip tubes of each construct( psB1C3_C; psB1c3_N; psB4A5_c; psB4A5_N)
Colony PCR with OneTaq Polymerase
Samples loaded onto monster Gel (Samples order is interlaced)
Results
First try of Cas13a digestion of IC3a/C3a dimer over 1 hour
Protocol: RNAse activity protocol
Participants: Florian
Observations:
Incubation for dimerization at room temperature for 1 h. Afterwards 1 hour digestion with Cas13a.
Results
No real difference visible after digestion for 1 h.
Friday 8th
Test heat-only lysis (1,3 and 7 minutes))
Protocol: Phenol-Chlorophorm extraction
Participants: Julian
Notes:
Diluted E. coli culture to 108 cells/ml, used 300 uL as sample
Heat-only at 93 °C for 15 min, diluted wiht tap water to simulate actual usage scenario (saliva etc..)
Results
Hardly any RNA detectable (concentration < 10 ug/ml). Abysmal 260/280-ratio -> Heat-only lysis is too inefficient for that low cell counts, use 109 next time.
Monday 11th
Intein-Extein-Readout
Protocol: -
Participants: Max
Observations:
Precultures of 16 colonies from plates of 06.09.17.
Precultures in 5 ml LB with corresponding antibiotic
Second try of Cas13a digestion of IC3a/C3a dimer for several hours and overnight
Protocol: RNAse activity protocol
Participants: Florian
Observations:
Results
Tuesday 12th
Intein-Extein-Readout
Protocol: Mini-Prep with Qiagen Kit and Sequencing Sample preparation
Participants: Max
Observations:
Mini Prep of precultures from 11.09.17
Send all 16 plasmids for sequencing with primer VR
Results
psB4A5_C-Terminal 1-7 and 3-4 were positive.
Wednesday 13th
Intein-Extein-Readout
Protocol: Restriction digest and PCR protocol with Q5 polymerase
Participants: Max
Observations:
Send Samples psB4A5_C-Terminal 1-7 and 3-4 for sequencing with primer VF
New digestion of unamplified N-Terminal g-block (200 ng in 50 µl with EcoRI-HF and SpeI-HF)
PCR of psB1C3 with psb1A3_fw and psb1A3_rev , to get an opened backbone (Q5 polymerase protocol with 66 °C as annealing temerature and 1 minute elongation time)
Results
PsB4A5_C terminal 1-7 was sequenced completly and used for further cloning steps.
Heat-lysis with E. coli
Protocol: Phenol-Chlorophorm extraction
Participants: Julian
Notes:
Lysis at 90°C, used 300 uL of E. coli suspension diluted to 2*109 cells/ml
SDS-Lysis as reference was impossible with this cell count, interphase during extraction too broad
Results
Extraction-yield is down 5- to 10- fold  compared to SDS (calculated from earlier SDS-lysis results with 109 (not 2*109)
RPA stability test and Reaction Temperature Screening on paper
Protocol: Recombinase Polymerase Amplification on Paper
Participants: Sven
Observations:
Preparation of stability assay of RPA on paper. Lyophilisation and storage.
RPA on paper according to protocol
Thursday 14th
RPA stability test and Reaction Temperature Screening on paper
Protocol: Recombinase Polymerase Amplification on Paper
Participants: Sven
Observations:
Preparation of stability assay of RPA on paper. Lyophilisation and storage.
RPA on paper according to protocol
Intein-Extein-Readout
Protocol: Restriction digest, Ligation of DNA fragments, Gibbson-Assembly and E and chem Trafo
Participants: Max
Observations:
New digestion of unamplified N-Terminal gBlock and psB1C3 with SpeI-HF and EcoRI-Hf
Dephosphorylation with Antarctic Phosphatase
Ligation of digested sequences overnight with T4 DNA Ligase
Gibbson-Assembly of psB1C3with N-Term fragment
Transformation of chem and e. competent Turbo cells
Friday 15th
RPA stability test and Reaction Temperature Screening on paper
Protocol: Recombinase Polymerase Amplification on Paper
Participants: Sven
Observations:
Stability of RPA on paper was tested using His6-TEV plasmid after 0, 2 and 24 hours post-lyophilisation. Reaction temperature was tested at 37 °C             as suggested by TwistDx and room temperature. Reaction was positively controlled using non-lyophilised reaction mixture.
RPA according to protocol
PCR cleanup according to protocol.
Results
Lyophilisation of RPA on paper worked.
Directly after lyophilisation, the reaction                     efficiency seems to be similar to fresh RPA reaction mixture.
Lyophilisation of RPA on paper worked. However, activity loss is observed already after 2 hours and after 24 hours,                      almost no activity is observable anymore.
Intein-Extein-Readout
Protocol: Colony PCR
Participants: Max
Observations:
Picked 32 colonies from plates of 14.09.17
Screened with Colony PCR
Gel 200 V 2 % Agarose 30 minutes
Results
No positive colononies in the Col PCR.
Phenol/Chloroform total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT
Protocol: Phenol/Chloroform purification of RNA
Participants: Jorge
Observations:
Two samples stored at -80 ºC labeled TRNA Lsh #1 15.09.17 and TRNA Lsh #2 15.09.17
Results
Monday 18th
RPA stability test on paper
Protocol: Recombinase Polymerase Amplification on Paper
Participants: Sven
Observations:
Preparation of temperature and air-sensitivity assay of RPA on paper.
RPA on paper according to protocol.
Results
Lyophilisation of RPA on paper.
Storage with and without Parafilm enclosure at 4 °C or room temperature.
Intein-Extein-Readout
Protocol: Q5-PCR and Restriction digestion and Col-PCR and e and chem comp Transformation
Participants: Max
Observations:
New PCR of backbone with psB1A3-fw and psB1A3-rev. Sample was subsequently DpnI digested and purified
Col-Pcr of 16 non green colonies from plates from thursday 14.9.17
Transformation of cells with ON Ligation from Friday 15.9.17 into E.coli DH5 alpha and E.coli Turbo
Tuesday 19th
RNA extraction from 5ml overnight culture of B. subtilis
Protocol: Phenol-chlorophorm extraction RNA
Participants: Dawa
Notes:
used heat lysis at 90 °C for 15 mins
Coupling In-Vitro Transcription to RPA
Protocol: Coupling RPA to In-Vitro Transcription
Participants: Sven
Observations:
Transcription efficiency when coupled to RPA was determined using a reaction without T7 RNA Polymerase as negative control.            The construct as template used was His6-TEV plasmid.
RPA-Tx according to protocol.
Phenol-Chloroform-Extraction according to protocol.
Results
In-Vitro Transcription could not be shown.
RNA was detected in the samples in which no T7 RNA Polymerase was added while there are no                     bands in the samples that had T7 RNA Polymerase in them. Though unlikely, a labelling error might have occured.
RPA stability test on paper
Protocol: Recombinase Polymerase Amplification on Paper
Participants: Sven
Observations:
Stability of RPA on paper was tested using His6-TEV plasmid after 24 hours post-lyophilisation. Storage conditions were altered.            Storage at room temperature and at 4 °C were tested. Also, duplicates were done at these temperatures, one sample being always enclosed             with Parafilm to avoid air exposure.
RPA on Paper according to protocol
PCR cleanup according to protocol.
Results
RPA activity clearly showed that storage temperature did not have a significant effect on the integrity of the RPA reaction mixture.
Exposure to air, however, was identified as the bottleneck for RPA stability. The RPA samples that were protected from exposure to air showed significantly higher
Intein-Extein-Readout and Biobrick Recloning
Protocol: Gibbson-Assembly and Ligation
Participants: Max, Rob
Observations:
Gel of Col-PCR from 18.09.17
Clean-up from backbone PCR and digest from 18.09.17 followed by Gibbsonassembly with N-terminal insert.
Started the Recloning of the Biobricks by PCR with phosphorylated primers and T4 Ligation and transformed into Turbo chem competent cells.
Results
Sample 1-3 and 2-1 had the correct length, and therefore precultures of these colonies were started overnight.
IC3a invitro transcription
Protocol: In vitro transcription
Participants: Florian
Observations:
Results
See results of DNaseI digestion and PCE from 20.09.17.
Heat-lysis with B. subtilis
Protocol: Phenol-Chlorophorm extraction
Participants: Julian
Notes:
2x SDS-Heat lysis (protocol), 2x heat-only at 90 °C for 20 min
Results
SDS-lysis  works a lot worse for gram+ bacteria, Heat-only now better/same than SDS-lysis (more incubation time...)         A260/280 ratio relatively bad, probably a lot of cell-wall impurities
Heat lysis and isothermal PCR: temperature dependence
Protocol: RPA-pcr
Participants: Julian, Sven
Notes:
Diluted E. coli (DH5a-pSB1C3-Tev-008-His) suspension to 106 cells / ml and heat-lysed in 50 uL aliquots
Lysis-temperature ranged from 65 °C to 90 °C in 5 °C steps
Added 2 uL of lysed sample to 10 uL RPA-solution and TEV-protease primers
Results
Heat differences in this range and this cell concentration have only minor effect. Best lysis seems to be around 80 °C, which is consistens with literature.
Wednesday 20th
Gel analysis of B.subtilis RNA samples
Protocol: Denaturating Urea PAGE
Participants: Dawa
Notes:
8% gel
Results
B. subtilis-PCE 190 ng/ul, B.subtilis heat/lysis-378 ng/ul
Intein-Extein-Readout
Protocol: Miniprep with Qiagen Kit
Participants: Max
Observations:
Minipreped overnight cultures from 1-3 and 2-1.
Send samples for sequencing with VF and VR.
Results
N-Terminal sample 1-3 was positive.
Biobrick Recloning
Protocol: Col PCR and preparation of e comp. cells
Participants: Max
Observations:
Colony PCR of  36 colonies from each of the 3 constructs His-Sumo-Lwa (Lwa(l)), Lwa(lwa(k) and His-Sumo-Lbu each in psB1C3.
Precultures of 4 colonies from Lwa(l) and Lwa(k)
Results
DNAse I digest and Phenol-Chloroform-Precipitation
Protocol: Restriction digest,Phenol-chlorophorm purification of RNA
Participants: Florian
Observations:
Results
6 M UREA-PAGE of invitro transcribed and pooled IC3a.
Quantitative gel analysis calculated a concentration of 35 µM.
Thurdsay 21st
Intein-Extein-Readout
Protocol: Restriction digest, Gibbson-Assembly and Transformation
Participants: Max
Observations:
Digestion of 1 µg of each backbone (psb1c3_N-term_uni and psb4A5_C-Term_uni with BBS1-HF to open them for Gibbson-Assembly
Gibbson-Assembly of Col_PCR fragments from 03.08.2017
1:3 backbone to insert concentration
30 minutes at 50 °C
Transformation of E.coli Turbo e and chem competent cells according to protocol
Preculture for Cryostock from psB1C3_Split_uni_N_term 1-3
Results
Biobrick Recloning
Protocol: Col-PCR and Sample Sequencing
Participants: Max
Observations:
Send samples for sequencing
Col_PCR of 4 samples from Lbu plate (all negative)
Wednesday 20th
RNase Alert Experiment with Bacillus subtilis using varying concentrations of target RNA
Protocol: RNase activity
Participants: Dawa
Notes:
Pippeting scheme:
Thursday 21th
B. subtilis RNA extraction using lysozyme
Protocol: Phenol-chlorophorm RNA extraction
Participants: Dawa
Monday 25th
crRNA in vitro transcription (for B.subtilis, Norovirus. Q5 beta, Hepatitis)
Protocol: In vitro Transcription
Participants: Dawa
Intein-Extein-Readout
Protocol: Restriction digest, PCR Cleanup Kit, Gibbson Assembly and Transformation
Participants: Max
Observations:
Redigest of 1 µg of each backbone with BBS1 and dephoshorilation according to protocol
Cleanup with cleanup kit
Gibbson assembly with 100 µg backbone and 3:1 excess of insert ( calculated based on length ratio) Inserts 2 and 3 from PCR of 05.08.17
Transformation into E.coli Turbo e. comp
COlPCR of plates from Thursday. 21.9.17=> 36 samples
Gel at 1 % 45 mins 150 V
Biobrick Recloning
Protocol: Total
Participants: Max
Observations:
COlPCR and Gel of samples from trafo of Friday 22.09.17 => 30 samples (all non green) psB1C3_LBU
Gel at 1 % 45 mins 150 V
Results
All negative.
Tuesday 26th
crRNA quantification, Urea gel run
Protocol: Denaturating Urea Page
Participants: Dawa, Erika
Notes:
2 samples each from B. subtilis, Norovirus, Q5 beta and HCV(first diluted 1:5 and second undiluted)
Intein-Extein-Readout
Protocol: ColPCR, Agarosegel
Participants: Max, Milica
Observations:
ColPCR of 12 colonies from N-term plate from 25.9.17
Gel at 1 % 30 mins 150 V
Results
Samples 1-8 and 2-1 of the N-Terminus looked good => preculture for sequencing.
Biobrick Recloning
Protocol: Total
Participants: Max
Observations:
COlPCR and Gel of samples from trafo of Friday 22.09.17 => 30 samples (all non green) psB1C3_Lwa_kurz
Gel at 1 % 30 mins 150 V
Results
All negative.
Wednesday 27th
Intein-Extein-Readout
Protocol: Minipreparation and Sample Sequencing
Participants: Max
Observations:
Miniprep of  psB1C3_Split_N_term_bGal 1-8 and 2-1
Send  Minipreps for sequencing with primer  VF and VR
Results
Both sequences positive.
Phenol/Chloroform total RNA purification from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT
Protocol: Phenol/Chloroform purification of RNA
Participants: Jorge
Observations:
Four samples stored at -80 ºC labeled TRNA Lsh #1 27.09.17 and TRNA Lsh #2 27.09.17, TRNA Lsh #3 27.09.17, TRNA Lsh #4 27.09.17.
Results
Thursday 28th
Agarose and Urea gel run with RNA
Protocol: PAGE, Urea PAGE
Participants: Dawa, Erika
Results
Agarose Gel didn’t work – The samples seems to be degraded in agarose gel , may be due to contamination of the gel chamber and other materials with Rnase.
Biobrick Recloning
Protocol: ColPCR, ON culture
Participants: Max
Observations:
ColPCR from samples of 22.09.17
32x Lwa short
Overnight cultures from colonies 1-8 and 2-3 started
Results
RNA/DNA hybrid dimerization with Cy5 DNA-oligo C3a
Protocol:
Participants: Florian
Observations:
Results
Native PAGE 20%, 12.5 mM MgCl2
C3a too high concentrated (Red bands). 5 µM instead of 500 nM. Better results with band shift see 29-09-2017.
Screen of LBU activity on RNA/DNA dimer
Protocol: RNAse activity protocol
Participants: Florian
Observations:
Results
Native PAGE 20%, 12.5 mM MgCl2
No visible shift of activator DNA oligo with addition of RNA under the use of Cy5 modified DNA activator oligo. Results are bad because of too high C3A DNA oligo concentration. (C3a = red bands).
Friday 29th
RNaseAlert experiment with extracted RNA targets from E.coli and in vitro transcribed RNAs.
Protocol: RNase activity
Participants: Dawa, Erika
Notes:
Pipetting scheme:
Results
The results for E.coli look good. Excel table saved in lrz as: 2017_09_29_rnase alert_ecolitarget_invitrotarget.xlsx
Biobrick Recloning
Protocol: Mini-Prepatation
Participants: Max
Observations:
Mini prep of 1-8 and 2-3.
Lwa short send for sequencing with VF and VR
Results
Lwa long positive.
DNA and RNA hybrid of 3C-A and IC3-A
Protocol:
Participants: Florian
Observations:
Results
Native PAGE 20%, 12.5 mM MgCl2
Visible small up shift of RNA/DNA dimer versus samples, which only contain the DNA oligo by itself. (DNA oligo C3a marked with Cy5 visible as red bands.)
FINA extraction for RNA with purified total RNA (TRNA Lsh #2-4 from 27.09.17)
Protocol: FINA Extraction for RNA
Participants: Jorge
Observations:
FINA was performed with 25 ul sample instead of 50 ul.
The extraction for TRNA #2 was performed as in protocol.
Before the FINA extraction, TRNA #3 was digested with DNase I according to DNase I Reaction Protocol
After the FINA extraction, the two eluate from TRNA #4 were digested with DNase I according to DNase I Reaction Protocol
First strand cDNA synthesis from E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT FINA-RNA eluate
Protocol: First strand cDNA synthesis
Participants: Jorge
Observations
pseq-Lsh-06rev was used as primer.
The eluates from the FINA extractions for RNA from 29.09.17 were used as samples.
From each pair of samples, one was used as a negative control (nuclease free water instead of MuLV-RT).
For all reactions, 6 ul sample were pipetted.
Q5-PCR for Lsh-fragment in E. coli Rosetta2 DE3 p2CT-His-MBP-Lsh_C13a_WT  with cDNA samples from 29.09.17
Protocol: PCR amplification with Q5 Master Mix
Participants: Jorge
Observations:
pseq-Lsh-05fwd was used as forward primer.
pseq-Lsh-06rev was used as reverse primer.
Annealing was performed at 58 ºC.
The amplification step was 110 s long.
A 1.5% agarose gel was run in TAE-buffer at 110 V for 80 min to check for PCR-products.
Quick-Load Purple 2-Log DNA Ladder (N0550S) was used as ladder
Results
N: FINA extraction performed as in protocol.
NK: FINA extraction as in protocol. No RT in cDNA synthesis (negative control).
P: Digestion of DNA before FINA extraction.
PK: Digestion of DNA before FINA extraction. No RT in cDNA synthesis (negative control).
A: Digestion of DNA after FINA extraction.
AK: Digestion of DNA after FINA extraction. No RT in cDNA synthesis (negative control).
Monday 2nd
Phenol chloroform purification of crRNAs of B. subtilis, NoroV, HCV and Q5 beta
Protocol: Phenol-chloroform extraction RNA
Participants: Dawa
Live Invitro Transcription test with LBU and Inhibitor-RNA/Activator-DNA dimer with different T7 concentration
Protocol: In vitro transcription
Participants: Florian
Observations:
Forgot to add C3ds with single strand overhang, which is needed for activation of invitro transcription after binding of DNA oligo C3a (activator of C3ds).
Results
Plate reader data showed no activity for all samples, as expected without DNA template to transcribe.
Tuesday 3rd
various Rnase alert experiments
Protocol: RNase activity
Participants: Dawa
Notes:
with Noro Virus (10 ul total volume used for the experiment)
with Q5 beta (10 ul total volume used for the experiment)
with Bacillus ( 10 ul total volume used for the experiment) and
with HCV and E.coli RNA (FINA method) samples (10 ul total volume used for the experiment)
Pipetting Scheme Norovirus and Q5 beta:
Pipetting Scheme JVC and B. subtilis:
Pipetting Scheme E.coli RNA (FINA method):
Results
Bacillus, Noro virus and HCV seeem to work. However Q5 beta needs to be repeated and also HCV could be repeated!
FINA extraction of RNA from E. coli with and without filters
Protocol: FINA RNA extraction
Participants: Jorge
Results
Works, Target-sequence with PCR detected.
Live Invitro Transcription test with LBU and Inhibitor-RNA/Activator-DNA dimer with different T7 concentration
Protocol: In vitro transcription
Participants: Florian
Observations:
Results
No Invitro transcription of RNA at any T7-Polymerase concentration. LBU does either not set activator DNA free or does not work at all. Same result as on the gel.
RNAse H shows low level fluorescence over the whole experiment, which can not be related to an invitro transcription.
Wednesday 4th
Bacillus RNA extraction
Protocol: RNase activity
Participants: Erika, Dawa
Notes:
Resuspension Buffer (20 ml):
Sucrose (0.3 M) -> 2.1 g
Sodium Acetate (10 mM) -> 16.4 mg
Adjust pH to 4.2
Stored at 4°C
Biobrick Recloning
Protocol: -
Participants: Max
Observations:
Send Lwa short colony 2-3 for sequencing with pseqLwa primer
Results
Full read with no mutations.
Thursday 5th
RNaseAlert
Protocol: RNase activity
Participants:  Dawa, Rob, Erika
Notes:
RNaseAlert reaction mix ( in glass fiber filter paper) lyophilisation. Introduction and first trial
Intein-Extein-Readout
Protocol: Transformation of e. comp cells
Participants: Max
Observations:
Repeated Gibbson-Assembly for Lbu with 4:1 ratio
Transformation into E.coli Turbo
Results
Empty plates => need to repeat PCR.
In-vitro transcription of AuNP linkers U0, U5, U10, U15
Protocol: in vitro transcription
Participants: Rob
Friday 6th
RNA extraction from Bacillus overnight culture
Protocol: Phenol-chlorophorm extraction
Participants: Erika
Notes:
following SIGMA Genosys Protocol from Step 1 to 7. Then, followed Lab´s protocol steps.
RNA sample stored at -80°C and needs to be quantified.
PCE and gel analysis of of AuNP linkers U0, U5, U10, U15 and new in vitro transcription
Protocol: phenol-chlorophorm purification of RNA
Protocol: Urea gel for RNA detection
Protocol: in vitro transcription
Participants: Rob
Observations:
There was no RNA visible.
The in-vitro transcription was repeated.
Results
Saturday 7th
SybrGreen II sensitivity test
Protocol:
Participants: Florian
Observations:
Results
1:10.000 seems to be the right dilution of SybrGreen II dye for a 15 µl Sample.
PCE and gel analysis of of AuNP linkers U0, U5, U10, U15
Protocol: phenol-chlorophorm purification of RNA
Protocol: Urea gel for RNA detection
Protocol: in vitro transcription
Participants: Rob
Observations:
There was no RNA visible.
The in-vitro transcription was repeated with a new batch of T7 RNA polymerase.
Sunday 8th
IC3A invitro transcription
Protocol: Invitro transcription
Participants: Florian
Observations:
Results
See results of Digestion and PCE from 09.10.17.
T7 polymerase sensitivity test
Protocol: Invitro transcription
Participants: Florian
Observations:
Results
Higher concentrations of T7-polymerase causes the maximal concentration of RNA to be reached faster.
PCE and gel analysis of of AuNP linkers U0, U5, U10, U15
Protocol: phenol-chlorophorm purification of RNA
Protocol: Urea gel for RNA detection
Protocol: AuNP preparation
Participants: Rob
Observations:
The IVTX was successful.
3-day incubation of AuNP from stock was started. <\li>
Results
Monday 9th
Quantification of viral RNAs and Bacillus RNA extracted from last week
Protocol: RNA urea gel
Participants: Dawa
Lbu-Recloning with Gibbson-Assembly
Protocol: PCR,Gibbson and Trafo in chem and e comp
Participants: Max
Observations:
PCR for Gibbson with Biobrick fwd and rev primer.
Lbu Gibbson Assembly with 3x excess of insert.
Trafo into E. coli chem and e comp.
Results
No colonies
DNAse I digest and PCE of 4 pooled invitro transcriptions of IC3a
Protocol: Restriction digest,Phenol-chlorophorm purification of RNA
Participants: Florian
Observations:
Results
6 M UREA-PAGE 15% of IC3a RNA
Tuesday 10th
Intein-Extein-Readout
Protocol: PCR and Gibbson Assembly
Participants: Max
Observations:
PCR of Lbu and psB1C3 as backbone
Gibbson Assembly with 5:1 excess of insert
Results
Heat lysis and isothermal PCR: cell concentration dependence s
Protocol: RPA-pcr
Participants: Julian, Sven
Notes:
Diluted E. coli (DH5a-pSB1C3-Tev-008-His) suspension in a range of 106 to 102 cells / ml and heat-lysed in 50 uL aliquots
Added 2 uL of lysed sample to 10 uL RPA-solution and TEV-protease primers
Results
Bands visible down to a dillution of 5*104 or 104 cells/ml, which corresponds to an average of 100 or 20 cells' DNA per PCR
Wednesday 11th
Plate reader experiments
Protocol: RNase activity
Participants: Dawa, Jorge
Notes:
with Noro virus samples ( worked!)
with Bacillus ( seems to have Rnase contamination)
with the paper strips (glass fiber filter paper) 1.8 ul reaction mix pipetted with different amounts of invitro transcribed RNAs.
with E.coli RNA samples( heat lysed)- ( seems to have RNase contamination)
Pipetting Scheme 1:
Pipetting Scheme 2:
Coupling In-Vitro Transcription to RPA in Bulk and on Paper
Protocol: Coupling RPA to In-Vitro Transcription
Participants: Sven
Observations:
Transcription efficiency when coupled to RPA was determined using a reaction without T7 RNA Polymerase as negative control.            The construct as template used was RNA Amp G-Block 200 bp double-stranded DNA using VF and VR as primers.
RPA-Tx and RPA-Tx on Paper according to protocol.
G-Block cloning of RNA-Amplification test plasmid via Restriction Cloning and Gibson Assembly
Protocol: G-Block Cloning
Participants: Sven
Observations:
Preparation:
Solubilisation of G-Block to reach 10 ng/ul concentration.
PCR reaction using VF and VR as primers resulted in 50 ul of a 24 ng/ul solution.
Gibson Assembly:
PCR reaction using pSB1A3_fw and pSB1A3_rev for backbone amplification of pSB1C3-GFP yielded 54 ng/ul.             Ratio for Gibson Assembly: 0.375 pmol Insert : 0.125 pmol backbone. Incubation for 15 minutes at 50 °C.
Restriction Cloning:
Restriction of backbone and insert using EcoRI-HF and SpeI-HF in NEB Cutsmart Buffer and heat inactivation.
Subsequent dephosphorylation of backbone using Antartic Phosphatase.
T4 DNA Ligation with a ratio of 0.02 pmol backbone : 0.06 pmol insert.
Transformation of Ligation and Gibson Assembly into chemocompetent and electrocompetent Turbo cells. Plate-out on             chloramphenicol agar plates.
Results
The investigation of the agar plates the next day using UV-light revealed that the only plasmid that was taken up by the Turbo cells                     was the initial pSB1C3-GFP plasmid. No positive inserts could be identified.
Biobrick Recloning
Protocol: Transformation of chem comp cells
Participants: Max
Observations:
Transformation of Turbo cells with Gibbson of 10.10.
Dye Test for live imaging of invitro transcription (1st row Controls (with SybrGold partially), 2nd row SybrGold dilution series, 3rd row SybrGreen I dilution series, 4th row EvaGreen dilution series)
Protocol: invitro transcription
Participants: Florian
Observations:
Results
6 M UREA-PAGE 15% of IC3a RNA
Mistake made: Added T7 Polymerase to Control 2. Was left out in Graph.
Only Eva Green seems to not influence the invitro transcription, because it was especially designed for live imaging with ongoing transcription reactions.
Cas13a activity assay
Protocol: Cas13a activity assay
Participants: Igor
Observations:
Results
Preparation and DNA-conjugation of of new AuNP
Protocol: AuNP preparation
Protocol: AuNP-DNA conjugation
Participants: Rob
Observations:
Preparation of AuNP was successfully completed.
DNA-conjugation with "AuNP 5' primer" and "AuNP 3' primer" <\li>
Results
See results from "conjugation test"
Thursday 12th
Plate reader experiment with paper strips blocked ovenight with BSA
Protocol: RNase activity
Participants: Dawa
Notes:
1.8 ul of reaction mix pipetted in each paper with different amounts of in vitro target RNA.
The overnight blocked paper strips were rinsed with nuclease free water and then dried in the oven at 70 degree for 30 mins and then used for the experiments.
Pipetting scheme:
Results
Seems to work! Needs to be repeated for the confirmation of results.
UREA-PAGE of Coupling In-Vitro Transcription to RPA in Bulk and on Paper
Protocol: Coupling RPA to In-Vitro Transcription
Participants: Sven
Observations:
Transcription efficiency when coupled to RPA was determined using a reaction without T7 RNA Polymerase as negative control.            The construct as template used was RNA Amp G-Block 200 bp double-stranded DNA using VF and VR as primers.
RPA-Tx and RPA-Tx on Paper according to protocol.
UREA-PAGE Gel of Samples extracted via Phenol-Choloform-Extraction; both according to protocol.
Results
T7 RNA-Polymerase was, by accident, also added to the negative control of the RPA-Tx reaction on Paper.
First, one can see that the gel was overloaded. Also, it seems like there is additional bands at the expected size of around 150 bp.
Questionable is that this seems also to be the case for the negative Control of the RPA-Tx reaction in Bulk.
The experiment on paper, however, looks quite promising since it seems like there are distinct bands at the expected approximate size of the RNA transcript. Cas13a cleavage assay will provide evidence if these bands actually consist of the desired target RNA.
Intein-Extein-Readout
Protocol: Transformation of e. comp cells
Participants: Max
Observations:
Transformation of BL21 with psB4A5 C-Term and psB1C3 N-Term
Results
No colonies for psB4A5 C-Term.
Biobrick Recloning
Protocol: Col-PCR for Biobrick Recloning
Participants: Max
Observations:
Col PCR of 6x N-Term in psB1C3, 16x C-Term in psB1C3, 14x Lbu in psB1C3
Results
All negative.
Eva Green Sensitivity Test with RNA background
Protocol:
Participants: Florian
Observations:
AuNP linkage asssay with DNA-linker
Protocol: AuNP linkage
Participants: Rob
Observations:
1x Buffer was used first, which resulted in no aggregation. Increasing the concentration by adding buffer to a final concentration of 2x afterwards resulted in aggregation and thus was set as the standard.
Friday 13th
Plate reader experiment with paper strips blocked ovenight with BSA
Protocol: RNase activity
Participants: Rob
Notes:
1.8 ul of reaction mix pipetted in each paper with different amounts of in vitro target RNA.
The overnight blocked paper strips were rinsed with nuclease free water and then dried in the oven at 70 degree for 30 mins and then used for the experiments. (Results seem to work!)
Pipetting scheme:
Biobrick Recloning
Protocol: PCR with Q5-Polymerase
Participants: Max
Observations:
Repeated PCR of N and C-term part for Recloning
AuNP linkage asssay with DNA-linker and varying buffer concentrations
Protocol: AuNP linkage
Participants: Rob
Observations:
2x, 4x, and 6x buffer was used, which resulted in unspecific aggregation from 4x to 6x buffer, thus 2x was kept as standard.
Saturday 14th
Intein-Extein-Readout
Protocol: Restriction digest Gibbson-Assembly and Transformation
Participants: Max
Observations:
DpnI digest of PCR from 31.10.17
Clean-up with PCR purification kit
Gibbson-Assembly with 3:1 excess
Transformation into E.coli Turbo chem comp.
Sunday 15th
RNA extraction of E. coli and Bacillus + plate reader experiment
Protocol: RNase activity
Participants: Dawa, Jorge, Erika
Notes:
HCV assay and Q5beta in Fluorostar and Clariostar for 1 hour each (HCV assay worked!)
RNA urea gel for RNA extracted from E.coli and Bacillus
Plate reader experiment of RNA extracted from the E.coli in Fluorostar for 2 hours
Pipetting scheme:
Intein-Extein-Readout
Protocol: Transformation of e. comp cells
Participants: Max
Observations:
Repeated Transformation of Bl21 star with psB4A5-Split-Intein-C-Term
Results
No colonies.=> Transformation in other expression strains
Monday 16th
Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative controls. Plus assay with cas13a with TDPs and assay with RPA paper samples.
Protocol: RNase activity
Participants: Dawa
Pipetting scheme:
Intein-Extein-Readout
Protocol: Transformation of e. comp cells
Participants: Max
Observations:
Transformed all available Bl21 strains with psB4A5-C-Term
Strains were:
Bl21 pLYs; BL21 star; BL21 DE3, BL21 RIPL, Rosetta and E.coli Turbo as pos control
Results
No colonies for any BL21 strain. The Turbo cells grew fine. => Expression seems to be toxic.
New DNA-conjugation and AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP linkage
Protocol: AuNP-DNA conjugation
Participants: Rob
Observations:
New AuNP were DNA-conjugated
Water was used instead of linker-RNA.
Tuesday 17th
Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative controls. Plus assay with cas13a with TDPs and assay with RPA paper samples.
Protocol: RNase activity
Participants: Dawa
Pipetting Scheme B. subtilis:
Pipetting Scheme RPA paper:
Intein-Extein-Readout
Protocol: -
Participants: Max
Observations:
Midipreparation of psB4A5-C-Term with Macherey-Nagel Xtra Midi
Preparation done according to protocol
Plasmid was intended for In vitro Expression
Results
In vitro expression was not done, due to lack of time.
Biobrick Recloning
Protocol: Q5-PCR, Gibbson-Assembly and Transformation
Participants: Max
Observations:
PCR of all GFP degradation tag library variants with Biobrick sequence binding primers, for transfer from psB1A3 to psB1C3.
Gibbson-Assembly and Transformation according to protocol
Results
AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP linkage
Participants: Rob
Observations:
The experiment from the day before resulted in unspecific aggregation in the negative control. To have comparable samples and negative control for the cleavage assay, the experiment was repeated with a mock-DNA in the negative control.  repeated with mock-RNA as negative control. A possible explanation for this is that negatively charched, non-bound RNA in the solution increases repulsion of AuNP.
Wednesday 18th
Biobrick Recloning
Protocol: Overnight culture
Participants: Max
Observations:
Picked 2 green colonies from each plate for overnight culture
observation of AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP linkage
Participants: Rob
Observations:
After cooling over night and spinning down for 1 min, just minor aggregation was achieved and samples were kept at 4°C for 2 days to allow for potential further aggregation.
Thursday 19th
G-Block cloning of RNA-Amplification test plasmid via Restriction Cloning and Gibson Assembly
Protocol: G-Block Cloning
Participants: Sven
Observations:
Preparation:
PCR reaction of G-Block using VF and VR as primers resulted in 50 ul of a 27 ng/ul solution.
Gibson Assembly:
PCR reaction using pSB1A3_fw and pSB1A3_rev for backbone amplification of pSB1C3-GFP yielded 52 ng/ul.             Ratio for Gibson Assembly: 0.375 pmol Insert : 0.125 pmol backbone. Incubation for 15 minutes at 50 °C.
Restriction Cloning:
Restriction of backbone and insert using EcoRI-HF and SpeI-HF in NEB Cutsmart Buffer and heat inactivation.
Simultanous dephosphorylation of backbone using Antartic Phosphatase.
T4 DNA Ligation with a ratio of 0.02 pmol backbone : 0.06 pmol insert.
Transformation of Ligation and Gibson Assembly into chemocompetent and electrocompetent Turbo cells. Plate-out on             chloramphenicol agar plates.
Results
The investigation of the agar plates the next day using UV-light showed that for the Gibson Assembly, positive clones were visible.
Colony PCR was performed and showed inserts for several plasmids.
Overnight cultures were inocculated for sequencing.
Biobrick Recloning
Protocol: Sequencing
Participants: Max
Observations:
Send samples from degradation tag library for sequencing
Results
pdt2-E 2 was positive
Friday 20th
G-Block cloning of RNA-Amplification test plasmid
Protocol: G-Block Cloning
Participants: Sven
Observations:
Mini-prep of previously inocculated overnight cultures and sent to sequencing.
Results
The investigation of the agar plates the next day using UV-light showed that for the Gibson Assembly, positive clones were visible.
In vitro transcription of crRNA Lbu, crRNA Lsh and target RNA at 37 ºC overnight
Protocol: In vitro Transcription
Participants: Florian
See results of digestion and PEC from 21.10.17
observation of AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP linkage
Participants: Rob
Observations:
The 2-day incubation did not result in higher aggregation, so a new over-night linkage was started. <\li>
After cooling over night and spinning down for 1 min, just minor aggregation was achieved and samples were kept at 4°C for 3 days over the weekend to allow for aggregat
Saturday 21st
DNAse I digest and PCE of the RNA crRNA from Lbu and Lsh
Protocol: Restriction digest,Phenol/Chlorophorm purification of RNA
Participants: Florian
Observations:
Results
In vitro transcription of crRNA Lwa at 37 C overnight
Protocol: In vitro Transcription
Participants: Florian
See results of digestion and PEC from 23.10.17
Sunday 22nd
RPA on PDMS Chip
Protocol: RPA on Paper
Participants: Sven
Observations:
Since the PDMS chip was leaky, no real sample could be extracted from it.
Results
Positive Controls as well as colony-PCR using RPA showed clear bands on a 2%-agarose gel. The RPA performed on Chip, however, was not showing any bands though Nanodrop concentration showed 27 ng/ul in the sample.
observation of AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP linkage
Participants: Rob
Observations:
After cooling over night and spinning down for 10 min, aggregates could be observed more clearly.
DNAse I digest and PCE of the RNA crRNA from Lwa
Protocol: Restriction digest,Phenol/Chlorophorm purification of RNA
Participants: Dawafuti
Observations:
Results
Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative controls with Lsh Cas13a
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
No activity of Lsh Cas13a visible.
Plate reader experiment with different concentrations of RNA in vitro targets, positive and negative control with TDPs from team TU Delft
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Plate reader experiment with different concentrations of RNA extracted with chemical lysis and phenol chloroform purification from the E.coli culture
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Repeat of plate reader experiment with Lsh Cas13a with Elution 1 and Elution 4,2
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
No activity of Lsh Cas13a visible
Cross target  plate reader experiment using crRNA E.coli with different target RNAs from B.subtilis and Noro virus
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
It didn't work
Monday 23rd
Plate reader experiment with dried and not dried Cas13a and TDPs with different target concentrations and positive and negative controls
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Plate reader experiment with with Cas13a Lbu and different concentrations of RPA samples
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Plate reader experiment with old Lwa Cas13a elution
Protocol: RNase activity
Participants: Dawafuti
Observations:
An elution from July, stored at 4 ºC of Lwa Cas13a was used
Results
It didn't work
Tuesday 24th
Plate reader experiment with Lwa Cas13a, old elutions 2 and 3
Protocol: RNase activity
Participants: Dawafuti
Observations:
An elution from July, stored at 4 ºC of Lwa Cas13a was used
Results
It didn't work
Plate reader experiment with cross targets using crRNA form E.coli and different target RNAs form B. subtilis, Noro virus  and E.coli
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Plate reader experiment with Spinach aptamer using Lbu Cas13a and different concentrations of in vitro targets
Protocol:
Participants: Dawafuti
Observations:
Results
It didn't work
observation of AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP-DNA conjugation
Protocol: AuNP linkage
Participants: Rob
Observations:
To have optimal conditions and high DNA density of DNA on the AuNP for linkage and cleavage, a new AuNP-DNA conjugation was performed. <\li>
Over-night linkage was started accoriding to the final protocol.
Wednesday 25th
Plate reader experiments with lyophilized reaction mix with Cas13a Lbu with different concentrations of the in vitro target.
Protocol:
Participants: Dawafuti
Observations:
Cas13a was added only immediately before starting the measurement.
Results
Plate reader experiment with different concentration of heat lysed E.coli samples with Cas13a Lbu
Protocol:
Participants: Dawafuti
Observations:
Results
The activity of Cas13a is visible, but additional RNase activity was also seen.
General control plate reader experiments with Cas13a Lbu with everything, without crRNA, without target, without Cas13a, positive and negative control.
Protocol:
Participants: Dawafuti
Observations:
Results
observation of AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP cleavage
Participants: Rob
Observations:
After spinning down the linked AuNP, all supernatant except for 5 μl was removed, pellet resuspended and spotted on paper. <\li>
The spotted AuNP were visible as blue aggregates and red, unlinked AuNP.
Thursday 26th
Plate reader experiment with Noro virus crRNA, Lbu Cas13a and different RNA targets from E.coli, HCV and Noro virus.
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Plate reader experiment with Lwa Cas13a purified from the stored Lwa pellet using different concentrations of invitro transcribed RNAs
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Plate reader experiments with the lyophilized reaction mix on paper using different concentrations of target RNAs
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
It didn’t work since the lyophilisation was done overnight which was way too long for a small amount of 15 ul reaction mix
Plate reader experiments with the RNA aptamer Spinach with different concentrations of in vitro RNA samples
Protocol:
Participants: Dawafuti
Observations:
Results
observation of AuNP linkage asssay with RNA-linker and DNA-linker
Protocol: AuNP cleavage
Participants: Rob
Observations:
To assure removal of unbound AuNP from , this time, after spinning down the linked AuNP, all supernatant except for was removed, pellet resuspended in 2 μl 1x Cas13a reaction buffer and spotted on paper. <\li>
The spotted AuNP were visible as blue aggregates and red, unlinked AuNP.
Friday 27th
Plate reader experiments with Lwa ( elution 1 and 2 and wash 1 ) purified using Nickel NTA
Protocol: RNase activity
Participants: Dawafuti
Observations:
Results
Tuesday 31st
April
May
June
July
August
September
October
