<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD colspan="6" align="left" valign="center"><FONT size="7" color="#51a7f9"><B style="color: #51a7f9">Project Design</B></FONT></TD></TR><TR class="lastRow"><TD colspan="6" align="left"><P class="introduction">
We started our iGEM journey with brainstorming sessions resulting in several great ideas, that were framed into written project proposals by teams of two to five students. After a constructive peer-review process with our supervisors and PIs, the projects were presented to the whole team. The final two ideas were subjected to SWOT-analysis, after which we settled on CascAID, our CRISPR-based diagnostic device for point-of-care testing.
Our project was to build an actual, functional diagnosis chip, and was therefore quite ambitious and complex. We used an engineering approach and broke down our project into modules we could work on in parallel, to eventually integrate them into a final product. For readout circuit module, Cas13a had to be purified and characterized in the lab. For the sample processing module, the chosen pathogens had to be lysed and their target sequences amplified to yield an amount of target RNA that could be detectable by Cas13a. Finally, we needed a detection module so the signal would be made visible to the user. Finally, everything had to be framed into a portable and user-friendly format, by designing appropriate hardware. 
We therefore formed sub-teams who focused their work on these modules. A high degree of modularization requires a high level of cooperativity between the sub-teams, but also provides an increased flexibility in design and better possibilities for customization, which eventually became a key advantage of our platform. Similarly, many ideas can be developed and tested in parallel, which speeds up the development process. 

                </P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H3>Sample processing</H3><P>  
The goal of this module is to obtain a detectable amount of target RNA from a sample of <I>E. coli</I> cells, which we used as a dummy target. The extraction method should be efficient and not require the use of hazardous chemicals. Therefore, the first step was to test common lysis techniques differing in the required equipment and use of chemicals. We compared their efficiencies by agarose gel electrophoresis.  
The standard approach for RNA extraction, Guanidinium-Phenol-Chloroform extraction, using chaotropic salts as a lytic agent, requires purification of the lysis product and involves the use of toxic chemicals. Another commonly used lysis process: incubation at high temperature with detergents such as SDS, gave good results, but again, required separation of SDS from the RNA afterwards.  For RNA purification purposes we investigated silica based procedures, as a less harmful solution. However, a point-of-care device has to be robust and reliable. In this regard, a simple design is considered superior to complex ones. We therefore abandoned chemical lysis methods that require purification and chose a combination of heat lysis followed by isothermal PCR. 
Recombinase Polymerase Amplification (RPA), as an isothermal alternative to PCR, is conducted at 37°C without the need for thermocycling and therefore reduces the requirements and costs of the accompanying hardware significantly. Since Cas13a targets RNA rather than DNA, we coupled RPA to <I>in vitro</I> transcription (TX). Since both reactions take place at the same temperature of 37°C, this can be done in a one-pot reaction. To render the RPA-TX distributable, we lyophilized the enzymes on filter paper, which, when sealed in a tight container, stabilizes the assay for storage. 
</P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H3>Cas13a, Purification and Characterization</H3><P>  
We chose Cas13a, an RNA-guided RNAse, because it could be practically optimized to detect any sequence of interest, this way allowing customization of device for any pathogen. It is also highly specific and sensitive, which enables detection even when the sequence of interest is only present in traces in sample. 
Our plan was to investigate proteins of the Cas13a family from different organisms - <I>Leptotrichia buccalis</I> (Lbu), <I>Leptotrichia shahii</I> (Lsh) and <I>Leptotrichia wadei</I> (Lwa) - and choose the one that performs best. Lbu and Lsh were available from addgene, while we cloned and codon-optimised Lwa ourselves. We therefore used two different <I>E. coli</I> strains for protein expression. We used BL21 Star for codon-optimized sequences and for not optimized ones we chose the Rosetta strain, which is a BL21 derivate that provides tRNAs for codons that are usually not present in <I>E. coli</I>. In order to prevent the large Cas-protein from forming inclusion bodies we expressed it together with SUMO (Small Ubiquitin-like Modifier) or MBP (Maltose Binding Protein) proteins, which help Cas13a to stay in solution. We opted for Ni-NTA purification followed tag cleavage and by size exclusion chromatography, with the His-tag remained attached to the cleaved SUMO/MBP. To determine the purity of protein and see if there are differences between investigated proteins we used SDS-PAGE. 
For characterization of Cas13a proteins we used the RNaseAlert assay, a cleavage assay in which Cas13a is activated through the presence of appropriate crRNA and target RNA. We transcribed target RNA <I>in vitro</I> to accelerate the process and then screened for optimum conditions. Using this assay for different target sequences from Noro Virus, Hepatitis C Virus and <I>Bacillus subtilis</I>, we investigated the specificity of the Cas13a activity.

</P></TD></TR><TR class="lastRow"><TD colspan="3" align="center" valign="center"><H3>Readouts</H3><P>  
CascAID utilizes the collateral RNase activity of Cas13a to produce a highly amplified signal. The purpose of this module was to translate the cleavage of RNA into a readout that is easy to measure and interpret by the user. We identified this part as a key challenge of the project, so we opted for a redundant project design and tried different approaches.
</P><P>  
We used two well-established fluorescence assays to quantify Cas13a's activity:
</P><P><B>RNaseAlert Assay:</B> For our first prototyping and  characterisation of Cas13a we used RNaseAlert, which is a commercially available fluorescence beacon for monitoring RNase activity. Given the appropriate equipment, fluorescence can be measured down to low concentrations (around 10nM), which is very practical for laboratory work. Due to the lack of an existing portable, affordable and sufficiently sensitive fluorescent detector, our hardware team built our own device, as described in the following section.
</P></TD><TD colspan="3" align="center" valign="center"><DIV class="captionPicture"><P>Working principle of <B>RNaseAlert Assay</B></P></DIV></TD></TR><TR class="lastRow"><TD colspan="3" align="center" valign="center"><DIV class="captionPicture"><P>Scheme of the transcription of the <B>Spinach aptamer</B> and subsequent DFHBI binding, increasing fluorescence</P></DIV></TD><TD colspan="3" align="center" valign="center"><P><B>Spinach Aptamer Assay</B>: The Spinach aptamer assay is based on a fluorophore DFHBI which was the first molecule against which a SELEX experiment was run. The synthetic Spinach RNA sequence binds DFHBI with a dissociation constant of K<SUB>d</SUB> = 390 nM. It increases the quantum yield of DFHBI from 0.0007 when free to 0.72 when bound to the aptamer. Spinach has been used for imaging protein and gene expression, and it has been also modified in order to be used as a sensor of biological reactions or synthetic circuits. We considered the Spinach aptamer as a cheaper alternative to the RNaseAlert kit, which would still rely on fluorescence in the most commonly used GFP channel.
</P></TD></TR><TR><TD colspan="3" align="center" valign="center"><P>
However, the most practical readout would be a colorimetric assay, which produces a test result readable by the human eye.  We tested three alternative methods for this purpose. 
</P><P><B>ssDNA amplification readout</B>: The scheme shows the basic idea of the ssDNA oligo based readout. Three different routes for a final signal readout have been thought of. All three readouts are based around the formation of an RNA/ DNA dimer and the liberation of the DNA oligo by digestion of the RNA part by Cas13a. The RNA oligo has 2 poly-U loops, while bearing the complementary sequence for the small DNA oligo in between these poly-U loops.
</P></TD><TD colspan="3" align="center" valign="center"><DIV class="captionPicture"><P>Working principle of our <B>ssDNA amplification based readout</B></P></DIV></TD></TR><TR><TD colspan="3" align="center" valign="center"><DIV class="captionPicture"><P>Working principle of our <B>Intein-Extein readout</B></P></DIV></TD><TD colspan="3" align="center" valign="center"><P>
The Cas13a should preferably digest the poly-U loops of the annealed RNA, which results in three 6 base pair interactions being left between the RNA and the DNA oligo. Those 6bp binding regions can melt at room temperature and dissociate in a short time frame. This enables the release of the DNA oligo and therefore the interaction with another DNA template to generate one of the three thought-of amplification methods. For this readout the RNA serves as an inhibitor strand, while the DNA oligo serves as the activator strand for another DNA template (<A href="http://parts.igem.org/Part:BBa_K864401">aeBlue</A>). This readout opens many coupling possibilities to existing cell-free synthetic biological circuits, such as genelet oscillators or toehold switches. 

Design of the RNA/ DNA complex required detailed tuning of free energies and melting temperatures, which we performed in silico using NUPACK. The design was confirmed and optimized by native PAGE using a labelled version of the DNA oligo.
</P><P><B>Intein-Extein</B>: This readout depends on a TEV protease that is immobilized by an RNA aptamer, which gets released after cleavage by Cas13a. Once free, the TEV cleaves a linker within the intein-extein-device that then creates a fully functional beta-galactosidase. The active enzyme is then able to cleave 5-Brom-4-chlor-3-indoxyl-β-D-galactopyranoside, an artificial substrate which when cut forms a deep blue chromophore. This would allow for a visual readout, without the need of a detection device.
</P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><P><B>Gold nanoparticles</B>: Contrary to our first two colorimetric readouts, ssDNA and Intein-Extein, the only protein involved in the gold nanoparticle (AuNP)-readout is Cas13a, like in our RNaseAlert readout. This reduces the necessary fine tuning of the biochemical circuit to a minimum, favoring high robustness of the readout. Due to the phenomenon of Localized Surface Plasmon Resonance, AuNPs appear in a distinct color, ranging from intense red to blue, black and colorless, depending on aggregation state.
In our project we use AuNPs with a diameter of roughly 10 nm, giving them a bright red color in solution. Our first step is to functionalize AuNPs with either 5’- or 3’- thiolated DNA and, through addition of a RNA linker which hybridizes with two thiolated DNA strands, form aggregates, changing the color from red to blue. The design of the RNA linker includes a uracil-rich segment, prone to Cas13a-mediated promiscuous cleavage. We varied this segment in length as the plasmonic effect is strongly distance dependent.</P></TD></TR><TR><TD class="verticalColumn" colspan="3" align="center" valing="center"><P>
It has been shown that AuNPs can be aggregated mediated by DNA-based hybridizaiton, spotted on filter paper, dried and redissolved by addition of a nuclease-containing solution. To ensure specific aggregation mediated by RNA-linkers we performed excessive screenings for salt conditions and optimum linker-to-AuNP ratios. We then spot RNA-linked AuNPs on paper, dry them alongside the Cas13a mixture and detect specific target RNAs and resulting Cas13a activity with a simple change from blue to red. 
</P></TD><TD colspan="3" align="center" valign="center"><DIV class="captionPicture"><P>Working principle of our <B>Gold nanoparticles readout</B></P></DIV></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H3>Hardware and Paperfluidics</H3><P>  
We developed an open source fluorescence detector as a portable solution for fluorescence based readout. With the rise of 3D printing and microcontroller-based DIY electronics, the open source philosophy has spread from software engineering into the world of hardware. By providing detailed documentations and CAD-drawings of our hardware, we want to enable other engineers and hobbyists to improve our designs, democratize access to the system and empower the user to self-educate about the working principle of our hardware.
We chose to perform our detection reaction on filter paper, which enables us to freeze-dry the mixture for long-time stability, and use very small reaction volumes per test. To enable transport of the sample-containing fluid to the spots containing the detection mixture, we used a technique called paperfluidics. Thereto, we hacked our regular office printer, to pattern the paper with hydrophobic wax. Defined areas were left uncoated in order to form hydrophilic channels that promote fluid transport by capillary forces. This technique enabled us to rapidly prototype and test different channel geometries for optimal flow, with very short iteration cycles of about one hour. 
After the first tests, we recognized that bleached cellulose paper limits the resolution of fluorescence readout due to its auto-fluorescence. This led us to integrate glass microfiber filter paper into the paper strip in the respective detection areas, which shows little to no auto-fluorescence.  Further, we used adhesive tape to seal the paper strip to avoid RNase contamination and keep the glass microfiber filter paper in place. 
In a similar way, we prototyped our sample preparation module, consisting of a fluid control unit, a heating system and a fluidic chip. For the fluidic chip we used 3D-printed masters, rather than photo-lithography, to achieve faster prototyping cycles to optimize our design, while being open to later miniaturization. The heating system is controlled by a microcontroller, and for the fluid control unit we chose to build valves driven with the pressure from a bike tire to direct flow without the need of carrying pressurized air. The design of our hardware components allows their integration in a single portable device.

</P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>