<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD colspan="6" align="left" valign="center"><FONT size="7" color="#51a7f9"><B style="color: #51a7f9">Software</B></FONT><PRE>
                    ###################################################################
                    #                                                                 #
                    #                        CascAID V1.0                             #
                    #                                                                 #
                    #                   Thu Nov  2 04:23:54 2017                      #		
                    #                                                                 #
                    #                      IGEM Munich 2017                           # 
                    #                                                                 #
                    ###################################################################


</PRE></TD></TR><TR class="lastRow"><TD colspan="6" align="left"><P class="introduction">
CascAID is a potentially universal tool for nucleic acid detection. 
Fast adaptation of our platform to new targets requires <I>in silico</I> verification of the crRNA design. 
Crucial factors for the development of these crRNA designs are the binding of the crRNA to Cas13a, which is
mainly determined by its secondary structure, and the uniqueness of the targeting sequence in the transcriptome (to rule out false positive results). To ensure the integrity of the Cas13a-crRNA complex, we developed
a python script that uses the established program packages for secondary structures, NUPACK and Mfold. 
In order to verify the specificity of the targeting sequence, we used the BLASTN-short program to 
check for similar structures in a transcriptome databank. Additionally, we created a database of crRNA designs 
that have already worked and made it 
as extensive as possible given the limited time, checking for collaboration with other teams working with Cas13a, 
mainly TU Delft. 
The second branch of software we developed is needed for hardware control in our project. 
They allow user's devices such as computers and smartphones to control 
our hardware, Heatbringer and Lightbringer.
The repository to our software can be found <A class="myLink" href="https://github.com/igemsoftware2017/igem_munich_2017">here</A>.
                </P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H3>crRNA Design Verification</H3><P>  
There are two main problems regarding the design of crRNA for a diagnostic test. 
First, the secondary structure of the crRNA needed for Cas13a activity needs to be verified. 
Secondly, the sequence targeted by the crRNA has to be specific, i.e. there must be no identical sequence in the 
reference transcriptome of an healthy patient. Otherwise, off-target effects will lead to 
false positive results since Cas13a is activated even though the pathogen is not present. 
To address these issues, we developed a software relying on bioinformatic principles such as 
secondary structure prediction and Basic Local Alignment Search Tool (BLAST). 
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Secondary Structure Prediction</H3><P>  
For secondary structure prediction of the crRNA we utilised the two established program packages 
in the field, NUPACK and Mfold to compare newly designed crRNA with secondary structures of crRNAs that 
were already known to be active. These reference crRNA structures were either obtained from actual 
crystallography data of crRNA in complex with Cas13a, or from structure prediction data of experimentally 
tested crRNAs. Using secondary structure verification we were able to rule out misfolding crRNA
designs prior to experiment. We developed a script for the end user automating this procedure. 
</P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H4>NUPACK</H4><P>  
NUPACK is a RNA Secondary Structure Prediction package developed 
by several contributors under the guidance of Prof. Niles A. Pierce at the California Insitute of Technology (Caltech). 
The source-code is available free-of-charge for academic usage. 
NUPACK allows the analysis of the partition function, the minimum free energy and the equillibrium base-pairing 
probabilities of a RNA sequence. 
For offline usage, we implemented NUPACK locally. We proceeded to implement Mfold as a webserver request. 
This decision was made because we experienced that in certain cases, only one of the program packages 
was able to predict the secondary structure of crRNA as described in previous papers, predominantly the paper of Liu et al. published in <I>Cell</I> in 2017 
&quot;Two Distant Catalytic Sites Are Responsible for C2c2 RNase Activities&quot;. Also, a local run gives you 
the possibility of using the full spectrum of NUPACK's programs.
By the use of several of the the final structure prediction, we estimated whether the 
crRNA would be active in Cas13a. 
Furthermore, we experienced that NUPACK sometimes predicts the right secondary structure, it just doesn't represent
the most stable structure. With NUPACK's subopt, it is possible to predict more than just 
the most stable structure. This enables looking at less stable structures which might be more favourable when bound to the protein and comparing these to the 
structure databank. The output of a suboptimal prediction 
is given below as the second example. Explanations are included as comments after '#': 


<PRE style="text-align: left;">
% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% %
66                                                                 # length of the sequence
-9.400                                                             # free energy
.....................(.((((((.((((....)))).)))))).)............... # secondary structure
22      51                                                         # IDs of bases that form basepairs
24      49                                                         # form basepairs
25      48                                                         # this would mean base 22 
26      47                                                         # pairs with base 51
27      46
28      45
29      44
31      42
32      41
33      40
34      39
% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% %

% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% %
66
-9.300
.....................(((((((..((((....)))).)))))))................
22      50
23      49
24      48
25      47
26      46
27      45
28      44
31      42
32      41
33      40
34      39
% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% %

</PRE></P><P>

From this, we can extract the secondary structure in Vienna notation as well as the free energies 
of the RNA structure to predict the probability of formation in solution with help of the calculation 
of the full partition function. Using these results, the user can make qualitative assumptions about 
the activity of the corresponding Cas13a-crRNA complex. 
</P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H4>Mfold</H4><P>  
Mfold is a webserver for RNA secondary structure prediction developed by Michael Zuker based on his paper 
&quot;Mfold web server for nucleic acid folding and hybridization prediction&quot; that was published in <I>Nucleic Acids Research</I>  
in 2003. Since Mfold is not available as a locally buildable binary for every operating system, we developed a 
script that automatically requests a standardised RNA Fold job from the server, therefore making it available 
throughout all operating systems. Using the result obtained from this request, the secondary structure is 
checked via a string comparison in so-called &quot;Vienna&quot; notation. This notation gives base pairing as a string 
of dots and brackets where a dot represents a non-bonded base and brackets represents paired bases, clarified by 
a opening bracket &quot;(&quot; at the 5'-end and a closing bracket &quot;)&quot; at the 3'-end of each paired sequence. An example for the output
of the program is given below:
<PRE style="text-align: left;">

#######################################################################################
#################### CascAID Secondary Structure Verification #########################
#######################################################################################



#######################################################################################
##################### NUPACK Secondary Structure Verification #########################
#######################################################################################
	

GOOD NEWS! YOU'VE GOT THE RIGHT SECONDARY STRUCTURE!
YOUR SEQUENCE WAS:

5' GGAUUUAGACUACCCCAAAAACGAAGGGGACUAAAACUGAUAAAGAAGACAGUCAUAAGUGCGGC  3'

      (((((....((((.........)))).)))))                               ####    MATCHED SECONDARY STRUCTURE
   ...(((((....((((.........)))).)))))..((((..........)))).........  ####  PREDICTED SECONDARY STRUCTURE
___________________________________________________________________

		YOUR BACKBONE SEQUENCE HAS BEEN FOUND IN THE DATABANK
		IT CORRESPONDS TO THE BACKBONE SEQUENCE OF: lwaCas13a
______________________________________________________________________________________

Job ended normally. Sun Oct 29 23:46:28 2017
Do you have internet connectivity? [yes/no]yes


#######################################################################################
#################### MFOLD SECONDARY STRUCTURE VERIFICATION ###########################
#######################################################################################



		#################### CAUTION! ##################### 
		mFOLD SECONDARY STRUCTURE DOES NOT FIT OUR DATA BANK
		#################### CAUTION! #####################


YOUR SEQUENCE AND MOST STABLE PREDICTED STRUCTURE IS:

5' GGAUUUAGACUACCCCAAAAACGAAGGGGACUAAAACUGAUAAAGAAGACAGUCAUAAGUGCGG 3'
   ..(((((((((.((((.........)))).......((.....)).....)))).)))))....
______________________________________________________________________________________

Job ended normally. Sun Oct 29 23:47:06 2017


</PRE></P><P>
This is also a good example to show that it might happen that one program recognizes the 
crRNA secondary structure while the other does not. In this case, NUPACK has predicted the structure
while Mfold is not able to predict the structure. Even though this is an experimental construct 
that worked, we did not put the secondary structure prediction of this into the database for Mfold,
since it was unable to predict the right structure.
</P></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H3>Off-Target Effects</H3><P>  

In order to rule out off-target effects for the designed crRNA in diagnostic applications, 
we developed a script that is able to BLAST the sequence either against whole databases 
online or a custom database we compiled. This database contains the human transcriptome and those of bacteria common in the human nasal tract as well as modell organisms used in our project:
<OL style="list-style-type:disc; list-style-position:left; text-align: left;"><LI>Homo Sapiens</LI><LI>Escherichia Coli</LI><LI>Bacillus subtilis</LI><LI>Staphylococcus aureus</LI><LI>Corynebacterium diphtheriae</LI><LI>Streptococcus diphtheriae</LI><LI>Haemophillus influenzae</LI></OL></P><P>
Transcriptomes that are common in the nasal tract but were not available are,
among others:
</P><OL style="list-style-type:disc; list-style-position:left; text-align: left;"><LI>Neisseria family</LI><LI>Staphylococcus epidermidis</LI><LI>Streptococcus pyogenes</LI></OL><P>
All data was retreived from the Transcriptome Release #90 of the ENSEMBL project. The output is generated
from the output of a blastn-short run and consists in the example below of all sequences that show sequence identity of 18 bp or 
higher. For an actual run, the identity would need to be 26 bp or higher in order to actually show off-target effects since Cas13a is
selective up to 2 point mutations regarding the binding of crRNA and subsequent RNase activity. 
The expectation value here describes the number of hits one can expect to find
in a random database the same size as the database used for the blastn-short run. 
</P><PRE style="text-align: left;">
##################################################################
####################### Input Sequence ###########################
##################################################################

Your Sequence was:

GGAUUUAGACUACCCCAAAAACGAAGGGGACUAAAACUGAUAAAGAAGACAGUCAUAAGUGCGGC

Your target sequence thus is:
UGAUAAAGAAGACAGUCAUAAGUGCGGC

Sun Oct 29 23:49:22 2017

##################################################################



##################################################################
####### Following possible off-targets have been identified ######
##################################################################

&gt;seq 0                                                               # ID of alignment
sequence:gnl|BL_ORD_ID|91933 ENSBTAT00000042836.3 cds chromosome:    # location of gene
UMD3.1:3:107697178:107698719:1 gene:ENSBTAG00000030338.3             # gene name and ID 
gene_biotype:protein_coding transcript_biotype:protein_coding        # type of gene
gene_symbol:GJA9 description:gap junction protein alpha 9            # Gene description
[Source:HGNC Symbol;Acc:HGNC:19155]
length:1542                                                          # Length of gene
e value:1.50536                                                      # expectation value
identity:18                                                          # sequence identity count
ATAAAGAAGACAGTCATAA...                                               # alignment
|||||||||||| ||||||...                                               # match/mismatches
ATAAAGAAGACACTCATAA...                                               # alignment

&gt;seq 1
sequence:gnl|BL_ORD_ID|69018 ENSBTAT00000042836.3 cdna 
chromosome:UMD3.1:3:107697178:107698719:1 gene:ENSBTAG00000030338.3 
gene_biotype:protein_coding transcript_biotype:protein_coding gene_symbol:GJA9 
description:gap junction protein alpha 9 
[Source:HGNC Symbol;Acc:HGNC:19155]
length:1542
e value:1.50536
identity:18
ATAAAGAAGACAGTCATAA...
|||||||||||| ||||||...
ATAAAGAAGACACTCATAA... 
___________________________________________________________

Job ended normally. Sun Oct 29 23:49:22 2017


These results have also been saved in:              off_target.out
The full BLAST output can be found in:                     seq.xml
</PRE></TD></TR><TR class="lastRow"><TD colspan="6" align="center" valign="center"><H3>Database</H3><P>  
The database program gives you an interface to interact with the MySQL database created for 
crRNAs that have been shown to work experimentally . 
</P><PRE style="text-align: left;">

############         Available Detection Targets         #################

[1] Virus
[2] Bacteria

[0] Go back one step

What would you like to detect?2

############         Available Detection Targets         #################

[1] Escherichia coli
[2] Bacillus subtillis

[0] Go back one step

What would you like to detect?1

############             Choose your Target              #################
	
[1] E. Coli 16s rRNA

[0] Go back one step

What would you like to detect?1

###########      The sequence thou art looking for is : ################
	
ACUUUACUCCCUUCCUCCCCGCUGAAA



[9] Exit
[0] Go back one step


</PRE><P>

However, these still need to be tested for off-target effects experimentally since <I>in silico</I> 
screening can only confirm specificity to a certain amount of certainty. 
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Hardware control</H3><P>  
The software for reading out the fluorescence detector is described in the 
<A class="myLink" href="https://2017.igem.org/Team:Munich/Hardware/Detector">Hardware</A> section.
All software developed for hardware can be found in our <A class="myLink" href="https://github.com/igemsoftware2017/igem_munich_2017">GitHub</A> repository. 
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>References</H3><P><OL style="text-align: left"><LI id="ref_1">M. Dirks, J. S. Bois, J. M. Schaeffer, E. Winfree, and N. A. Pierce.
    	&quot;Thermodynamic analysis of interacting nucleic acid strands.&quot;(2007) <I>SIAM Rev</I>, 49:65-88.</LI><LI id="ref_2">R. M. Dirks and N. A. Pierce. &quot;An algorithm for computing nucleic acid base-pairing probabilities including pseudoknots.&quot;
		(2004) <I>J. Comput. Chem.</I>, 25:1295-1304.</LI><LI id="ref_3">R. M. Dirks and N. A. Pierce. &quot;A partition function algorithm for nucleic acid secondary structure including pseudoknots.&quot;
		(2003) <I>J Comput Chem</I>, 24:1664-1677.</LI><LI id="ref_4">M. Zuker, D. H. Mathews and D. H. Turner. &quot;Algorithms and Thermodynamics for RNA Secondary Structure Prediction: A Practical Guide&quot;
		(1999) <I>RNA Biochemistry and Biotechnology</I> 11-43 J. Barciszewski and B. F. C. Clark, eds.,
		NATO ASI Series, Kluwer Academic Publishers, Dordrecht, NL </LI><LI id="ref_5">J.-M. Rouillard, M. Zuker and E. Gulari. &quot;OligoArray 2.0: Thermodynamicaly improved 
		oligonucleotide design for microarrays.&quot; (2003) <I>Nucleic Acids Res.</I> 31:12, 3057-3062. </LI><LI>S.F. Altschul, W. Gish, W. Miller, E.W. Myers and D.J. Lipman &quot;Basic local alignment search tool.&quot; (1990)
		 <I>J. Mol. Biol. </I> 215:403-410.</LI></OL></P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>