<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Munich_Software_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Munich/Software/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><B style=" color: #ffffff !important">-_-</B></P></DIV><DIV class="dropdown"><BUTTON class="dropbtnProject dropbtn"><A href="/Team:Munich/Description"><P><B>Project</B></P></A></BUTTON><DIV class="dropdown-contentDescription dropdown-content"><A href="/Team:Munich/Description"><P><B>Description</B></P></A></DIV><DIV class="dropdown-contentDesign dropdown-content"><A href="/Team:Munich/Design"><P><B>Design</B></P></A></DIV><DIV class="dropdown-contentDemonstration dropdown-content"><A href="/Team:Munich/Demonstrate"><P><B>Demonstrate</B></P></A></DIV><DIV class="dropdown-contentMeasurement dropdown-content"><A href="/Team:Munich/Measurement"><P><B>Measurement</B></P></A></DIV><DIV class="dropdown-contentApplication dropdown-content"><A href="/Team:Munich/Applied_Design"><P><B>Applied Design</B></P></A></DIV><DIV class="dropdown-contentFinalResults dropdown-content"><A href="/Team:Munich/Results"><P><B>Final Results</B></P></A></DIV><DIV class="dropdown-contentAchievements dropdown-content"><A href="/Team:Munich/Achievements"><P><B>Achievements</B></P></A></DIV></DIV><DIV class="dropdown" valign="center"><BUTTON class="dropbtnWetlab dropbtn" valign="center"><A href="/Team:Munich/Protocols"><P><B>Wetlab</B></P></A></BUTTON><DIV class="dropdown-contentProtocolsMethods dropdown-content"><A href="/Team:Munich/Protocols"><P><B>Protocols</B></P></A></DIV><DIV class="dropdown-contentLabJournal dropdown-content"><A href="/Team:Munich/Notebook"><P><B>Notebook</B></P></A></DIV><DIV class="dropdown-contentSafety dropdown-content"><A href="/Team:Munich/Safety"><P><B>Safety</B></P></A></DIV><DIV class="dropdown-contentParts dropdown-content"><A href="/Team:Munich/Parts"><P><B>Parts</B></P></A></DIV><DIV class="dropdown-contentImprove dropdown-content"><A href="/Team:Munich/Improve"><P><B>Improved Part</B></P></A></DIV><DIV class="dropdown-contentCollection dropdown-content"><A href="/Team:Munich/Part_Collection"><P><B>Part Collection</B></P></A></DIV><DIV class="dropdown-contentInterlab dropdown-content"><A href="/Team:Munich/InterLab"><P><B>InterLab</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnModel dropbtn"><A href="/Team:Munich/Model"><P><B>Model/Software</B></P></A></BUTTON><DIV class="dropdown-contentModel dropdown-content"><A href="/Team:Munich/Model"><P><B>Model</B></P></A></DIV><DIV class="dropdown-contentSoftware dropdown-content"><A href="/Team:Munich/Software"><P><B>Software</B></P></A></DIV><DIV class="dropdown-contentSoftwareCollaborations dropdown-content"><A href="/Team:Munich/Software/Collaborations"><P><B>Collaboration</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnSoftHardware dropbtn"><A href="/Team:Munich/Hardware"><P><B>Hardware</B></P></A></BUTTON><DIV class="dropdown-contentDetector dropdown-content"><A href="/Team:Munich/Hardware/Detector"><P><B>Detector</B></P></A></DIV><DIV class="dropdown-contentQuakeValve dropdown-content"><A href="/Team:Munich/Hardware/QuakeValve"><P><B>Quake Valve</B></P></A></DIV><DIV class="dropdown-contentPaperstrips dropdown-content"><A href="/Team:Munich/Hardware/Paperstrip"><P><B>Paperstrips</B></P></A></DIV><DIV class="dropdown-contentPCRChip dropdown-content"><A href="/Team:Munich/Hardware/SampleProcessing"><P><B>Sample Processing</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnHP dropbtn"><A href="/Team:Munich/HP/Silver"><P><B>Human Practices</B></P></A></BUTTON><DIV class="dropdown-contentHPSilver dropdown-content"><A href="/Team:Munich/HP/Silver"><P><B>Silver</B></P></A></DIV><DIV class="dropdown-contentIHPgold dropdown-content"><A href="/Team:Munich/HP/Gold_Integrated"><P><B>Gold</B></P></A></DIV><DIV class="dropdown-contentPublicEngagement dropdown-content"><A href="/Team:Munich/Engagement"><P><B>Engagement</B></P></A></DIV><DIV class="dropdown-contentCollaborations dropdown-content"><A href="/Team:Munich/Collaborations"><P><B>Collaborations</B></P></A></DIV></DIV><DIV class="dropdown"><BUTTON class="dropbtnTeam dropbtn"><A href="/Team:Munich/Team"><P><B>Team</B></P></A></BUTTON><DIV class="dropdown-contentTeamMembers dropdown-content"><A href="/Team:Munich/Team"><P><B>Team members</B></P></A></DIV><DIV class="dropdown-contentSponsors dropdown-content"><A href="/Team:Munich/Sponsors"><P><B>Sponsors</B></P></A></DIV><DIV class="dropdown-contentAttributions dropdown-content"><A href="/Team:Munich/Attributions"><P><B>Attributions</B></P></A></DIV><DIV class="dropdown-contentGallery dropdown-content"><A href="/Team:Munich/Gallery"><P><B>Gallery</B></P></A></DIV></DIV><TABLE width="100%" height="100%" cellpadding="0" cellspacing="0" border="0"><TBODY><TR><TD colspan="6" align="left" valign="center"><FONT size="7" color="#51a7f9"><B style="color: #51a7f9">Software Collaboration</B></FONT></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>For TU Delft</H3><P>  
We started collaborating early after the European iGEM Meet-Up with TU Delft in the wetlab.
Since we were both employing Cas13a as the molecular heart of our project, it made sense to 
exchange data and experiences from wet lab experiments. At some point during a skype meeting,
we realised that we were doing similar things in the dry lab and thus, decided
to collaborate and exchange ideas in that field as well. </P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Secondary Structure Prediction</H3><P>  
In our drylab collaboration with Team Delft, we performed a Cas13a design verification 
of crRNAs they used in their project with our developed software. Since the secondary 
structure of the crRNA is essential for its affinity to Cas13a, one can make assumptions 
on the post-binding RNase activity of the protein based on secondary structure predictions. 
We then compared these predicted structures  to secondary structures of crRNAs that have 
been shown to work experimentally. For the five crRNA sequences Team Delft provided, 
the predicted secondary structure matched one of the secondary structures in our databank. 
The only one that was not recognized by neither the NUPACK nor the mFOLD databank was 
crRNA 2. It's predicted secondary structure in Vienna notation is depicted below:   
</P><PRE>
GGAUUUAGACUACCCCAAAAACGAAGGGGACUAAAACAAGACAGUCAUAAGUGCGGCGACGAU
..(((((((((.((((.........)))).((......))..)))).))))).((....))..
</PRE><P>
It looks like the backbone structure needed in the first 35 bases is not constructed completely. 
When looking at the graphical output of NUPACK depicted in Figure 1, however, it is visible that the binding probability 
seems to be rather low regarding the structures that interact with the terminal 28 bp target sequence. 
Thus, crRNA 2 would most probably still be active when tested in experiment, but might show decreased activity. 
</P><DIV class="captionPicture" align="center"><P><B>Figure 1</B>: Graphical output of the secondary structure prediction of crRNA 2 in NUPACK.
</P></DIV><DIV class="captionPicture" align="center"><P><B>Figure 2</B>: Cleavage experiments of crRNA 2, crRNA 3 and crRNA 4 as provided by TU Delft.
</P></DIV><P>
Indeed, decreased activity using crRNA 2 in comparison to other crRNA is observed when performing experiments using 
these crRNA designs. Though this has to be treated with caution since crRNA 4's secondary structure was predicted 
to be correct but the Cas13a-crRNA complex of that crRNA does not show higher activity. We conclude that further testing
of the software has to be done to show whether it can predict activity of Cas13a. </P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>Off-target Effects</H3><P>  
Furthermore, in order to determine off-target effects, we constructed a database from Transcriptome data obtained 
from the Ensembl and Ensembl Bacteria databank. We tried to make the search as tailor-made to the Delft project as 
possible and thus considered species that were relevant to their project of detecting bacterial resistances related 
to cattle and milk production.  The included species were:

<OL style="list-style-type:disc; list-style-position:left; text-align: left;"><LI>Bos Taurus (Cattle)</LI><LI>Clostridium perfringens</LI><LI>Corynebacterium diphtheria</LI><LI>Fusobacterium necrophorum</LI><LI>Lactobacillus casei</LI><LI>Lactococcus lactis</LI><LI>Providencia stuartii</LI><LI>Staphylococcus aureus</LI><LI>Streptococcus pneumoniae</LI></OL></P><P>
Three of the five crRNAs showed no off-targets in the constructed database. For crRNA 3 and crRNA 4, 
possible off-targets were detected but just as a result of the identity parameter being rather low 
in the BLAST-short run. When looking at the results, it is clearly visible that the sequences are not 
identical enough to show any RNase activity of Cas13a since the 28 bases target is specific to up to 2 mutations, 
but all the hits that have been found are with a match of 17 bases far from the 28 bases long. 
</P><P>
Finally, we can conclude that our software did not find any problems in the crRNA design of Team Delft. 
It seems that their constructs will most probably show activity and, at least for the mentioned species, 
it will not show any off-target effects.
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>From TU Delft</H3><P>  
The TU Delft team developed a simulation code for modeling the on and off-target
activities for sequences on all possible frames on the genome. In this, they employ a kinetic model
by Depken et al. in 2017  to determine whether off-target activities are high enough to give a false positive 
collateral RNase activity of the Cas13a protein. They deducted from these model cleavage probabilities
for different crRNA sequences and could thus make statements of the possibility to distinguish 
between our two bacterial targets, the 16S rRNA of <I>E. Coli</I> and <I>B. subtillis</I>. By running the crRNA
against both genomes of these species, they determined the off-target probability to be very low for all our crRNA
design as shown in <B>Figure 3</B>. Thus, they deduced from these that our crRNA design would be able to differentiate between <I>E. Coli</I> and <I>B. subtilis</I>.
</P><DIV class="captionPicture" align="center"><P><B>Figure 3:</B> Off-target probability of activation of the Cas13a protein for our crRNAs sensing <I>E.Coli (1.3)</I> and <I>B. subtillis</I>
crRNA 1-3 are taken into one since the spacer sequence of the crRNA was identical for these constructs. 
</P></DIV><P>
We would like to thank Team Delft for the fun collaboration and wish you all the best for your project. 
We are looking forward to seeing you in Boston. 
</P></TD></TR><TR><TD colspan="6" align="center" valign="center"><H3>References</H3><P><OL style="text-align: left"><LI id="ref_1">Klein, M., Eslami-Mossallam, B., Gonzalez Arroyo, D., Depken, M. (2017). 
    	&quot;The kinetic basis of CRISPR-Cas off-targeting rules.&quot; doi: 10.1101/143602.</LI></OL></P></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></BODY></HTML>