"
Page
Discussion
View source
History
teams
Log in
 
Team:Toulouse/Project/binding
From 2014.igem.org
            
       
       
HOME
TEAM
Our Team
Official team profile
Gallery
Fun Facts
PROJECT
Project context
Overview
Chemotaxis
Binding
Fungicides
Spreading
RESULTS
Experimental results
Modeling
Parts
Achievements
HUMAN PRACTICE
Safety
Ethics
Communication
NOTEBOOK
Calendar
Project monitoring
Protocols
ACKNOWLEDGEMENTS
Attributions
Sponsors
   
Binding
To be attached to the fungal cell wall
Project   >   Binding
Figure 1: Schema of the binding module
In order to be highly efficient in the fight against the pythopathogen Ceratocystis platani, our optimized bacterium has to be anchored to the fungus.    Thus, we designed a chimeric protein ( BBa_K1364005 ) capable of building    a bridge between the bacterial peptidoglycan and the fungal chitin, the main component of the pathogen’s cell wall.    According to the work of the Imperial College 2010 iGEM team,    we used the Cell Wall Binding (CWB) domain of the LytC protein (coding for a N-acetylmuramoyl-L-alanine amidase) to attach our chimeric protein    to the Bacillus subtilis cell wall. On the other side of our protein, we added the domain 4 of GbpA from Vibrio cholerae, which is known to    recognize chitin.
More information about this module
The open reading frame of the Binding Module is composed of 3 sections:
Anchor section: the CWB (Cell Wall Binding) domain is extracted from LytC gene and composes the 5' side of our binding module. As previously described by the Imperial College of London 2010  iGEM team, we kept the first 318 bp. We can note the presence of the signal peptide at the beginning of the sequence, from 1 to 24 bp.
Chitin Binding Domain (CBD) section:  the domain 4 of GbpA from V. cholerae is able to bind to N-Acetyl Glucosamine oligosacchararides. Also,  the 3' side of our gene is composed by a part of the GbpA sequence (from 423 to 484 bp).
Helical Linker: According to the work of the 2010 Imperial College of London iGEM team, we used the same six amino acids sequence (SRGSRA) to make a bridge between the Anchor section and the Chitin Binding section.
Figure 2: Binding gene composition
The sequence has been designed in silico and codon optimized for the transcription in B. subtilis.
Final construction (More details about the intermediate parts Here )
The binding module has been placed under the control of Pveg ( BBa_K143012 ),  a strong constitutive promoter and we used a consensus RBS ( BBa_K090505 ) as well as a  double terminator ( B0015 ).
Figure 3: Binding gene construction
References
M. Desvaux, E. Dumas, I. Chafsey, and M. Hébraud. Protein cell surface display in Gram-positive bacteria: from single protein to macromolecular protein structure . FEMS Microbiol. Lett. 256, 1–15 (2006).
E. Wong, G. Vaaje-Kolstad, A. Ghosh, R. Hurtado-Guerrero, PV. Konarev, AF. Ibrahim, DI. Svergun, VG. Eijsink, NS. Chatterjee, and DM. van Aalten. The Vibrio cholerae colonization factor GbpA possesses a modular structure that governs binding to different host surfaces. PLoS Pathog. 8, e1002373 (2012).
H. Yamamoto, S. Kurosawa, and J. Sekiguchi. Localization of the vegetative cell wall hydrolases LytC, LytE, and LytF on the Bacillus subtilis cell surface and stability of these enzymes to cell wall-bound or extracellular proteases.  J. Bacteriol. 185, 6666–6677 (2003).
Chemotaxis
Fungicides
iGEM Toulouse 2014
INSA Toulouse
135, Avenue de Rangueil
31400 TOULOUSE
Let's talk
Feel free to contact us for any question about our SubtiTree project
CONTACT
Socialize
iGEM Toulouse 2014
Retrieved from " http://2014.igem.org/Team:Toulouse/Project/binding "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
