<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Toulouse_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Toulouse/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Toulouse/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Toulouse/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Toulouse/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Toulouse/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Toulouse/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="l-header">
      
      <DIV class="navbar-header">

        
        <DIV id="navigation"><DIV class="navigation"><DIV class="centering"><UL class="topnav" id="topnav"><LI><A href="https://2014.igem.org/Team:Toulouse" class="navlink">HOME</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Team" class="navlink">TEAM</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Team">Our Team</A></LI><LI><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Toulouse">Official team profile</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Team/Gallery">Gallery</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Team/Fun_facts">Fun Facts</A></LI></UL><LI><A href="https://2014.igem.org/Team:Toulouse/Project/Overviews" class="navlink">PROJECT</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Project/project-context">Project context</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Project/Overviews">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Project/Chemotaxis">Chemotaxis</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Project/binding">Binding</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Project/Fungicides">Fungicides</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Project/Spreading">Spreading</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Result/experimental-results" class="navlink">RESULTS</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Result/experimental-results">Experimental results</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Modelling">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Result/parts">Parts</A></LI><LI class="Achievements"><A href="https://2014.igem.org/Team:Toulouse/Result/achievement">Achievements</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Safety" class="navlink">HUMAN PRACTICE</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/ethics">Ethics</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Communication">Communication</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Notebook/Calendar" class="navlink">NOTEBOOK</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Notebook/Calendar">Calendar</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Notebook/project-monitoring">Project monitoring</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Acknowledgements/Attributions" class="navlink">ACKNOWLEDGEMENTS</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Acknowledgements/Attributions">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Toulouse/Acknowledgements/Sponsors">Sponsors</A></LI></DIV></DIV></DIV>


      </DIV><DIV class="fils-ariane" style="width:100%; height:60px; background:#ededed; margin-top:30px;"><DIV style="margin:0 auto; width:960px;"><P style="color:#696969; padding-top:20px; font-size:16px; float:left;"> Notebook   &gt;   Protocols</P></DIV></DIV><DIV id="innercontenthome"><DIV class="centering" style="padding-top: 85px; padding-bottom:40px;"><DIV id="column-left"><H3 class="title2" style="margin-top:10px; color:#333;">Summary :</H3><UL class="menuleft"><LI style="margin-top:25px;"><A href="#select1"><I>E. coli</I> competent cells</A></LI><LI><A href="#select2"><I>E. coli</I> transformation protocol</A></LI><LI><A href="#select3">Miniprep and alcaline lysis</A></LI><LI><A href="#select4">Cloning</A></LI><LI><A href="#select5">Checking of the genetic constructions</A></LI><LI><A href="#select6"><I>B. subtilis</I> transformation</A></LI><LI><A href="#select7">Test of the pSB<SUB>BS</SUB>4S plasmid integration in <I>Bacillus subtilis</I> genome on the threonine site</A></LI><LI><A href="#select8">Final Tests</A></LI></UL></DIV><DIV class="column-right" style="width:75%; float:right;"><P class="texte"> All the following protocols were inspired by one or several protocols, used, improved and optimized (which took more or less time...). Finally they gave us some <A href="https://2014.igem.org/Team:Toulouse/Result/experimental-results">results</A> :-).</P><P class="title1" id="select1"><I>E. coli</I> competent cells</P><P class="texte"><CENTER><I> MANIPULATION IN ICE  </I></CENTER></P><P class="texte"><B> Day 0 </B>-	Make an <I>Escherichia coli</I> cell culture in LB medium overnight
</P><P class="texte"><B> Day 1 </B> 
- 	Freeze 0.1M CaCl<SUB>2</SUB> and 4 Falcon tubes of 50mL at 4°C
 
-	Streak 2% culture in LB to get an OD of 0.3 to 0.4 (it takes approximately 1h30)
 
-	Centrifuge 10 minutes at 4500 RPM

-	Remove the supernatant

-	Resuspend the pellet in 7.5 mL of 0.1 M frozen CaCl<SUB>2</SUB>
-	Centrifuge 10 minutes at 4500 RPM

-	Resuspend the pellet in 500 µL of 0.1 M CaCl<SUB>2</SUB>
-	Add glycerol to a final concentration of 15%

-	Keep the tubes at -80°C
</P><P class="title1" id="select2"><I>E. coli</I> transformation protocol </P><P class="texte">
-	Let the LB agar medium plates dry in a sterile area

-	Thaw out the competent cell aliquotes for about  10 to 20 minutes

-	Add 20 to 100 ng of plasmid or 3 µL of kit plate DNA 
<I>NB: for kit plate, resuspend the well in 10 µL of sterile water</I>
-	Put the tubes 20 minutes in the ice 
-	Put the tubes 2 minutes at 42°C in the water bath to create the thermic shock

-	Put the tubes back in ice immediately 

-	Add 1mL of LB medium

-	Put the tube 2 hours in the 37°C water bath (1 hour if it concerns an ampicillin resistant strain) to allow the phenotypic expression

-	Centrifuge for 1 minute at 13000 RPM

-	Remove the supernatant 

-	Resuspend in 250 µL of LB medium

-	Streak the final mix on LB agar selective medium: 200 µL on one plate, 50 µL on the second plate
 </P><P class="title1" id="select3"> Miniprep and alcaline lysis </P><P class="texte"><B> Day 0 </B>-	Resuspend 4 to 12 colonies from the plate and name each colony taken on the tubes and on the plate (A, B, C, …)

-	Resuspend one colony per culture tube in 5 mL of LB medium with antibiotic

-	Let the culture grow overnight at 37°C in a shaking incubator</P><P class="texte"><B> Day 1 </B>-	Use the QIAprep spin Miniprep Kit for each culture tube. The last step consisting in the elution of the DNA is made with elution buffer or water at 55°C.

-	Keep the tubes at -20°C 
<I>NB: It is possible to purify the plasmid with an alcaline lysis without any purification column. For 2 mL of culture, 200 µL of buffer 1 is added to resuspend the pellet, 400 µL of buffer 2 to allow the lysis of the cells and the denaturation of the protein and 300 µL of buffer 3 to precipitate the DNA and the proteins. The solution is then centrifuged 10 minutes at 13 000 RPM.
600 µL of isopropanol is added to the supernatant and the solution is centrifuged again. The pellet is then resuspended in 100 µL of pH 7.4 TE buffer. A part of the contamination by the RNA can avoid by the addition of pH 7.4 TE buffer + 0.2 µL of RNAse. </I><B>Buffer 1:</B> Tris 10 mM pH 8 + EDTA 1mM
<B>Buffer 2:</B> NaOH 2 mM + SDS 1%
<B>Buffer 3:</B> Potassium acetate 3 M + 15% glacial acetic acid
</P><P class="title1" id="select4">Cloning </P><P class="texte">Cloning is the step after taking the competent cells, transforming the BioBricks and miniprep them.
</P><P class="title2">First step</P><P class="title3">Both parts have the same antibiotic resistance</P><P class="texte"><B>1) Digestion mix</B> For the vector :
-	5 µL of miniprep plasmid 

-	2 µL of each restriction enzymes

-	2 µL of Green Buffer

-	9 µL of Milli-Q water 
 For the insert :
-	10 µL of miniprep plasmid 

-	2 µL of each restriction enzymes

-	2 µL of Green Buffer

-	4 µL of Milli-Q water 

-	Incubate 15 minutes at 37°C 

<B>2) Gel extraction</B>
-	Prepare a 1% or 2% electrophoresis agarose gel with 0.5x TAE buffer

-	Put 20 µL of sample + 6 µL of marker (1 kb for 1% gel and 100 pb for 2%) into the well

-	Migration for 30 min at 100 V or 1 hour at 50V

-	The revelation is made in BET (10 minutes). Then wash in water for 5 minutes

- The gel extraction is realized thanks to the THERMO SCIENTIFIC GeneJET Gel Extraction and DNA Clean Up Microkit

3) Inactivation of the enzymes for the vector
There are two ways to inactivate the enzymes:
- Use of DNA Clean up kit for the DNA fragment above 200 pb
- Heat inactivation at 95°C for 10 minutes.</P><P class="title3">The two parts have a different antibiotic resistance</P><P class="texte"><B>1)	Digestion mix</B>For each part, add: 
-	5 µL of miniprep plasmid 
-	1 µL of each restriction enzymes
-	2 µL of Green Buffer
-	9 µL of Milli-Q water 
-	Incubate 15 minutes at 37°C 

</P><P class="texte"><B>2) Inactivation of the enzymes for the vector</B>There are two ways to inactivate the enzymes:
- Use of DNA Clean up kit for the DNA fragment above 200 pb
- Heat inactivation at 95°C for 10 minutes.</P><P class="title2">Second step</P><P class="title3">Ligation</P><P class="texte">-	Mix 10 µL of insert + 4 µL of vector + 2 µL of 10x T4 buffer + 0.5 µL of T4 ligase + 3.5 µL of Milli-Q water

A control without insert must be made

-	Incubate the ligation mix 15 minutes at room temperature (22°C) and keep the tubes in ice or at -20°C to prepare the transformation</P><P class="title3">Transformation</P><P class="texte">
-	Take 5µL of the ligation mix for 50 µL of competent cells and use the <A href="https://2014.igem.org/Team:Toulouse/Notebook/Protocols#select2">Toulouse iGEM Team 2014 transformation protocol</A>.

-	Plate the solution on selective medium overnight at 37°C.</P><P class="title1 " id="select5">Checking of the genetic constructions </P><P class="title2">1)	Colony PCR</P><P class="texte">
-	Add 0.5 µL of plasmid + 25 µL of DreamTAQ MasterMix + 2 µL of each 10 µM primer (VR and VF2) + H<SUB>2</SUB>0 qsp 25 µL and take a colony.

-	Look for the number of necessary cycles and the proper temperature thanks to AmplifX or Serial Cloner 2.1 softwares.
The following cycles have been used : 
- 94°C - 5 min
- (94°C 45 sec  ; 55°C 45 sec ; 72°C 1min/kb ) *  25 cycles 
- 72°C - 5min
- Then 4°C</P><P class="title2">2)	Analytic digestion</P><P class="texte">
-	Put a colony in 5 mL of LB selective medium and wait for 6 hours

-	Make a purification thanks to the Miniprep kit

-	Mix 2 µL of plasmid + 2 µL of Fast Digest Green Buffer + 1µL of each enzyme +  Milli-Q water qsp 20µL

-	Wait 15 minutes at 37°C and put the mix on a 1% or 2% gel for 30 minutes at 100 V.</P><P class="title2">3)	Sequencing</P><P class="texte">The sequencing of the genetic constructions was performed by Eurofins Genomics Company by mixing 15µL of pure plasmid solution with 2µL of one primer.</P><P class="title1" id="select6"><I>B. subtilis</I> transformation</P><P class="texte"><B> Day 0 </B>-	 Streak out the <I>B. subtilis</I> strain and plate this on an LB agar plate overnight at 37°C</P><P class="texte"><B> Day 1 </B>-	Pick up a nice big colony of <I>B. subtilis </I> strain and drop it in 2ml of completed 1x MC

-	Grow at 37°C for 5 hours

-	Mix 400µl of culture in a fresh tube (tubes loosely closed for the aeration) and put 5µL of Miniprep DNA.

-	Grow the cells at 37°C for an additional 2 hours

-	Spread the complete 400 µl reaction mix on selective antibiotic plates (100µl per plate), and incubate at 37°C overnight 
</P><P class="texte"><B> Preparation of solutions: </B><I> 300 mM Tri-Na Citrate:</I>- 0.88 g Tri-Na Citrate
- 10mL MQ water
<I>Ferric NH4 citrate:</I>- 0.22g Ferric NH4
- 10mL MQ water
<I>10x Competence Medium </I> For 10mL:
- 1.40g K2HPO4
- 0.52g KH2PO4
- 2g glucose
- 1 mL 300 mM Tri-Na citrate 
- 0.1 mL Ferric NH4 citrate
- 0.1g Casein Hydrolysate
- 0.2 g Potassium glutamate
The complete mixture should be dissolved in 10 ml. First add 5 ml milliQ water and mix. When everything is dissolved add MQ water till 10 ml. Filter sterilize the complete mixture and store at -20°C.
<I>1x Competence Medium </I>- 1.8 mL MQ water
- 200 µL 10x Competence Medium solution (previously filter sterilized)
- 6.7 µL 1M MgSO4 (previously autoclaved)
- 10 µL 1% tryptophan (previously filter sterilized and stored in aluminium foil)
The complete mixture should be dissolved in 100 ml. First add 50 ml milliQ water and mix. When everything is dissolved add MQ water till 100 ml. Filter sterilize the complete mixture and store at -20°C.


</P><P class="title1" id="select7">Test of the pSB<SUB>BS</SUB>4S plasmid integration in <I>Bacillus subtilis</I> genome on the threonine site</P><P class="texte">- Plate the transformed <I>B. subtilis</I> strain on a selective medium (LB + spectinomycin) overnight 
- The obtained clones are then plated on different media:  Medium Competence (Thr+), Medium Competence (Thr-) and LB + Spectinomycin. 
When the plasmid is integrated, the clone can grow on minimum medium with threonine and on LB + Spectinomycin but can not grow on the minimum medium without thronine.
Moreover, a colony PCR can be performed with the same protocol as previously presented. The used primers are VFBS and VRBS.
</P><P class="legend">Threnonine test (Left: MC Thr+; Right: MC Thr -)</P><P class="title1" id="select8">Final Tests</P><P class="title2">Chemotaxis test</P><P class="texte">
Many chemotaxis tests exist such as plate tests or capillary tests. We tried all of them and we decided to optimize the <A href="https://2011.igem.org/Team:Imperial_College_London/Protocols_Chemotaxis">« Capillary essay »</A> from Imperial College 2011 iGEM team.
-	Prepare the bacteria in LB medium until they reach an OD between 0.5 and 0.6 (exponential growth phase). This step takes about 5 hours.
-	Prepare the multichannel pipette: improve the cohesion between the tips and the pipette with Blu-Tack to avoid air and the possibility of leakage.
</P><P class="texte">-	Put 200 µl of the different chemoattractants in the wells of the ELISA plate and pipet 15 µL of each with the multichannel pipette: galactose which represents our negative control, glucose which represents our positive control and N-acetylglucosamine (NAG). The volume in the tips must be marked.<I>NB: The NAG is the most important test because it is the monosaccharide which composes the chitin on <I>Ceratocystis platani</I> wall.</I>-	Put the tips with chemoattractants in 300 µL of the bacterial solution in exponential growth phase in the ELISA plate.
-	Let the installation settle for 1 hour at room temperature.
-	After an hour, put the volume of the tips on parafilm. 
-	Each solution is diluted 1/10,000 and 100 µL is spread on LA medium.
-	The plates are then incubated overnight at 37°C.</P><P class="title2">Binding test</P><P class="texte"><I>CBB (Chitin Binding Buffer):</I>- 500 mM NaCl
- 20 mM Tris-HCl
- 1 mM EDTA
- 0,05% Triton X-100, 25°C, pH=8
</P><P class="title3">Column activation:</P><P class="texte">- Vortex the beads 
- Put 50 µL of beads in a 1.5mL centrifuge tube
- Wash with 500 µL of CBB
- Put the centrifuge tube on a magnetic rack
- Wait 30 seconds
- Remove supernatant
- Repeat the wash
</P><P class="title3">Bacterial fixation on the chitin beads:</P><P class="texte">
- Add 200 µL of bacteria solution (10<SUP>5</SUP> bacteria/mL) to the washed beads 
- Shake during 1h at 4°C
- Add 500µL of CBB (washing A)
- Put the centrifuge tube on a magnetic rack
- Wait 30 seconds
- Remove supernatant
- Add 500µL of CBB (washing B)
- Put the centrifuge tube on a magnetic rack
- Wait 30 seconds
- Remove supernatant
- Add 500µL of CBB to recover the beads directly
</P><P class="title3">Bacteria count:</P><P class="texte">- Make different dilutions : 10<SUP>-1</SUP>, 10<SUP>-3</SUP>, 10<SUP>-5</SUP> of the first bacterial culture and spread on LA plates
- Make different dilutions : 1, 10<SUP>-2</SUP>, 10<SUP>-4</SUP> of washings (A and B) and of the beads in CBB medium and spread on LA plates
- Place the plates at 37°C overnight
- Count colonies on different plates
</P><P class="title2">Fungicide test: anti-fungal activities</P><P class="texte">
CAUTION : all the lab equipment must be desinfected before and after the manipulations with the fungi. Three different fungus strains were used : <I>Aspergillus brasiliensis</I>, <I>Aspergillus nidulans</I> and <I>Trichoderma reesei</I>- The conidia can be taken by adding one drop of Tween 80 on the fungus plate.
- Then the drop is mixed with 1mL of sterile water in an Eppedorf.
- A microscopy count can be performed thanks to Thoma cell to determine the conidia concentration.
<I>NB : The conidia solutions are then diluted and spread on sap medium to get 10,000 conidia/plate.</I>- After 72 hours of liquid culture of the different clones of <I>B. subtilis</I> with the fungicides module, the culture can be centrifugated.
- 130µL of the supernatant is used to soak a pad placed on the conidia plate. The bacterial pellet is resuspended in 130µL of LB medium and also put on a pad. 
- The plates containing 10,000 conidia and the soaked pads are then put at room temperature for a few days according to the growth speed of the fungi. Controls are also realized with wild type strains or copper sulfate at 10 and 20mg/mL.
</P><P class="texte"><B> Sap-like Medium (250 mL) (see references)</B>: 
 
2,5g de tryptone

1,25g de YE

2,5g de NaCl

Glucose : 1,175g

Fructose : 1,125g

Sucrose : 0,125g

Inositol : 0,084 g

</P><P class="title2">Fungicide test: <I>in planta</I> assay</P><P class="title3">First step</P><P class="texte">
The first step is related to the inoculation of SubtiTree in plants through stomata (opened in wet condition). We diluted our bacterial samples to get two concentrations:  5.10<SUP>6</SUP> and 10<SUP>8</SUP> bacteria per mL. The WT and transformed bacteria are introduced into plants (a control test without bacteria is performed). Thanks to a 1 ml syringe (without needle), the plant was injected with bacteria by pressure. Five leaves of each plant were used, marked with a marker point. It is necessary to repeat the operation on both sides of the leaf and the excess is wipe off. The plants are placed in a growth chamber (phytotron) to control light, high humidity, temperature and bacterial non-proliferation. Bacterial growth in the plant is left for 24 h.
</P><P class="title3">Second step</P><P class="texte">
The next step begins with the preparation of the fungal samples. Fungal culture is crushed and mixed with PDB (Potato Dextrose Broth). Then the mix passes through a 100 µm filter (to remove large aggregates) and  through a 40 µm filter. The caught hyphae are mixed with PDB for 24 to 48 hours until it reaches an OD of 2.5 at 600nm. The previously seeded leaves are taken from the plant using a scalpel and placed in boxes above wet absorbent paper (leaves are kept alive for a week). Above each leaf, 5µl of the fungal suspension is deposited (using beveled tips because it is too viscous). As control, we kept inoculated leaves without fungus and leaves with only fungus. Pictures are taken at different times. All the plants are destroyed by autoclaving.
</P><P class="texte"><B>References</B>
Véronique Amiard, Annette Morvan-Bertrand, Jean-Bernard Cliquet, Jean-Pierre Billard,
Claude Huault, Jonas P. Sandström, and Marie-Pascale Prud’homme. <B>Carbohydrate and amino
acid composition in phloem sap of Lolium perenne L. before and after defoliation</B>. Can. J. Bot.
Vol. 82: 1594–1601, 2004.</P></DIV></DIV></DIV><DIV id="footer"><DIV class="wrapper-footer"><DIV class="block-footer" style="width:300px;"><DIV class="block-footer-text" style="padding-right:65px;"><H3>iGEM Toulouse 2014</H3><P>INSA Toulouse 135, Avenue de Rangueil 
         31400 TOULOUSE </P></DIV></DIV><DIV class="block-footer" style="width:378px;"><DIV class="block-footer-text" style="padding-right:65px; padding-left:75px;"><H3>Let's talk</H3><P>Feel free to contact us for any question about our SubtiTree project</P><A href="mailto:igemtoulouse2014@gmail.com" style="border:1px solid #fff;-webkit-border-radius: 5px; -moz-border-radius: 5px; border-radius: 5px; padding: 13px 25px 12px; color:white; text-decoration:none; font-size:17px; background:none; margin-top:30px; display:inline-block; ">CONTACT</A></DIV></DIV><DIV class="block-footer-last"><DIV class="block-footer-text" style="padding-left:75px"><H3>Socialize</H3></DIV></DIV></DIV></DIV><DIV class="nav-footer"><P style="text-align:center; padding-top:6px; color:#ccc;">iGEM Toulouse 2014</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Toulouse/Notebook/Protocols">http://2014.igem.org/Team:Toulouse/Notebook/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Toulouse/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Toulouse/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Toulouse/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Toulouse/Notebook/Protocols&amp;oldid=398640" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>