<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aalto-Helsinki_Research"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aalto-Helsinki/Research">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aalto-Helsinki/Research&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aalto-Helsinki/Research&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aalto-Helsinki/Research&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aalto-Helsinki/Research" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aalto-Helsinki/Research</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navbar navbar-default navbar-fixed-top"><DIV class="text-center col-sm-12" id="navigation"><NAV><A href="https://2014.igem.org/Team:Aalto-Helsinki" id="home">Main</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Team">Team</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Business">Business</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Research">Research</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Modeling">Modeling</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Cooperation">Cooperation</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Outreach">Outreach</A><A href="https://2014.igem.org/Team:Aalto-Helsinki/Journal">Journal</A></NAV></DIV></DIV><DIV id="" page-content-wrapper=""><DIV class="splash" id="researchsplash"><DIV class="row splashesittely"><DIV class="col-md-6"><H1>Research</H1><P class="bigsplashtext">
				Everything about our customisable light-controlled three-channel switch.
			</P></DIV></DIV><DIV class="row splashala"><DIV class="col-md-12 transp scroller"><A href="#Switch">
                Scroll down to read more
                </A></DIV></DIV><DIV class="splashalavalikko"><A href="#Switch">The Switch</A><A href="#Background">Background</A><A href="#Hypotheses">Hypotheses</A><A href="#Parts">Parts</A><A href="#Methods">Methods</A><A href="#Results">Results</A><A href="#Discussion">Discussion</A><A href="#Conclusions">Conclusions</A><A href="#References">References</A></DIV><DIV class="splashcopyright"><P class="transp"><A href="http://i-see-faces.deviantart.com/">Photo © Tanja Maria</A></P></DIV></DIV><DIV class="navbar navbar-default extrabar" data-spy="affix"><NAV><A id="Switch-submenu" class="active" href="#Switch">The Switch</A><A id="Background-submenu" href="#Background">Background</A><A id="Hypotheses-submenu" href="#Hypotheses">Hypotheses</A><A id="Parts-submenu" href="#Parts">Parts</A><A id="Methods-submenu" href="#Methods">Methods</A><A id="Results-submenu" href="#Results">Results</A><A id="Discussion-submenu" href="#Discussion">Discussion</A><A id="Conclusions-submenu" href="#Conclusions">Conclusions</A><A id="References-submenu" href="#References">References</A></NAV></DIV><ARTICLE><H2>The Three-Channel Switch</H2><H3>Introduction</H3><P>
		We engineered a three-channel switch that can be controlled with the intensity of blue light. Using a modified lambda (λ) repressor mechanism linked to a blue light sensor protein, we would be able to switch swiftly between the expressions of three different genes. This mechanism could provide a nearly real-time control over the chosen genes, which could advance a variety of industrial bioprocesses, speed up research projects and benefit metabolic engineering. The switch has a modular structure so the users can select the genes. This does not require any additional changes to the mechanism itself.
	</P><H3>Features</H3><DIV class="row"><DIV class="col-md-8 text-col-left"><P>
				We chose <B>YF1</B> fusion protein to be the light receptor protein. The receptor autophosphorylates in darkness but in blue light, it is unphosphorylated. A phosphorylated YF1 protein acts as a kinase and activates <B>FixJ</B> transcription factor, which can then bind to a <B><EM>FixK<SUB>2</SUB></EM></B> promoter's binding site and activate the production of <B>λ repressor protein CI</B>. The kinase activity of YF1 is inversely proportional to blue light intensity and the effect is carried on to the active FixJ concentration. Thus, CI protein is not produced in bright blue light but the production increases as the blue light is dimmed down.
			</P><P>
				We used lambda repressor protein CI to regulate the genes <I>A, B</I> and <I>C</I> in the switch. Only <I>gene A</I> is active when there is no repressor protein CI. At medium concentrations of CI, <I>gene A</I> becomes deactivated and <I>gene B</I> is activated. At high concentrations of CI, only <I>gene C</I> is active. The activity and inactivity of the <I>gene C</I> is based on the tetracycline repressor protein that is produced with <I>genes A</I> and <I>B</I>. <I>Gene C</I> is active only when both <I>genes (A and B)</I> are inactive. The activities of <I>genes A</I> and <I>B</I> are controlled with a modified lambda repressor mechanism, which is explained thoroughly later on this page.
			</P><P>
				So, to put it all together, in blue light the switch activates <I>gene A</I>, in dim blue light <I>gene B</I>, and in darkness <I>gene C</I> as illustrated in .
			</P></DIV><DIV class="col-md-4 img-100"><P class="kuvateksti"><B><SPAN ref="lamppukuva">.</SPAN></B>: In blue light the switch activates <I>gene A</I>, in dim blue light <I>gene B</I>, and in darkness <I>gene C</I>. The changes are based on differences in the concentration of the λ repressor protein CI.
			</P></DIV></DIV><H3>Glossary</H3><UL><LI><STRONG>YF1 fusion protein</STRONG> = blue light sensor that becomes unphosphorylated in blue light and phosphorylated (activated) in darkness</LI><LI><STRONG>FixJ protein</STRONG> = after being phosphorylated by YF1, this activates the promoter <I>FixK<SUB>2</SUB></I></LI><LI><STRONG><EM>FixK</EM><SUB>2</SUB> binding site</STRONG> = initiates the transcription of CI when FixJ binds to it</LI><LI><STRONG>lambda (λ) repressor protein CI</STRONG> = a protein that can repress or activate the transcription of two different genes</LI><LI><STRONG><EM>O</EM><SUB>R</SUB> right tripartite operator site</STRONG> = an operator site to which CI binds, downstream from <EM>O</EM><SUB>L</SUB> in the λ phage</LI><LI><STRONG><EM>O</EM><SUB>L</SUB> left tripartite operator site</STRONG> = an operator site to which CI binds, upstream from <EM>O</EM><SUB>R</SUB> in the λ phage</LI><LI><STRONG><EM>P</EM><SUB>R</SUB> promoter</STRONG> = a promoter that is active only when there’s no CI protein</LI><LI><STRONG><EM>P</EM><SUB>RM</SUB> promoter</STRONG> = a promoter that is active only when there’s a little CI protein. Too high or too low CI concentration inhibits the production</LI><LI><STRONG><EM>genes A-C</EM></STRONG> = the three genes that you can choose. Our system would regulated them as explained earlier</LI><LI><STRONG>Tetracycline repressor TetR</STRONG> = a protein that can bind to a TetR repressible promoter site and inhibit downstream gene transcription</LI></UL></ARTICLE><ARTICLE><H2 id="developing-out-idea">Background</H2><H3>Developing Our Idea</H3><DIV class="row"><DIV class="col-md-8 text-col-left"><P>
        		In the beginning of our project, from March 2014, we focused solely on coming up with new ideas for the project, building the <A href="https://2014.igem.org/Team:Aalto-Helsinki/Outreach#Media">public team website</A> and applying to <A href="https://2014.igem.org/Team:Aalto-Helsinki/Business#Sos">Summer of Startups 2014</A> program and foundations with deadlines during the spring. In our team meetings, we had different types of brainstorming sessions where all of us could present even the craziest ideas. We came up with disco bacteria that glow in sync with music, new kinds of bacterial drinks that change flavour with the storing temperature, remote controllable bacteria, oil-eating bacteria, tooth paste bacteria that keeps up the mouth hygiene despite forgetting to brush the teeth, etc. On ane Friday evening we had a vote and everybody gave a score from 1 to 5 for each idea. We made an <A href="https://static.igem.org/mediawiki/2014/5/5e/Aalto_Helsinki_Ideagradientti.jpg">idea gradient</A> where the ideas were sorted according to the score from bottom left to top right.
    		</P><P>
        		We chose the top five ideas and presented them to our advisors and researchers from Aalto University, VTT and University of Helsinki to get feedback and comments on the feasibility. After all, none of us had never done anything even close to this before. As a result of this feedback, further idea development and admission to Summer of Startups, we decided to work on the idea with a project name &quot;nest box&quot;. It's a container resembling a traditional Finnish bird nest box (a man-made home for birds) but it contains bacteria producing different fragrances. These bacteria could be controlled using light concealed inside the box. The user can decide which fragrance to produce by a press of a button and this action would be translated into a light signal to control the gene expression in the bacteria. Such a system is concrete and understandable for the public audience and it would help to build a positive image of synthetic biology in Finland. In theory, such system could be used in conference rooms, hotels, living rooms, saunas, movie theaters... The scent marketing is an expanding business and our idea had the potential to add something new to it. Furthermore, it was a concrete product which made our participation in Summer of Startups a lot easier.
			</P><P>
        		We started our BioBrick practice project mid May and the Summer of Startups programme began in early June. As we got forward with the idea, we realized that the nest box concept would be very difficult to execute in practice. Implementation of such a separate unmaintained container would be very challenging and in addition, we cannot bring genetically modified bacteria outside laboratory conditions (it's defined in the gene technology legislation). We decided to solely focus on one element of our nest box idea. Constructing the interface between the bacteria and the user or a computer is not trivial. So, we redefined our direction which was maintained to the end. We wanted to develop a three channel switch that would allow us to activate one of the three genes at a time using only the intensity of single wavelength of light. We could then connect this switch with fragrance producing genes, for example.
    		</P><P>
				Later we realized that it would fit better in a <A href="https://2014.igem.org/Team:Aalto-Helsinki/Business#our-business-idea">bioreactor</A> in order to get precise control of gene transcription.
			</P></DIV><DIV class="col-md-4 img-100"><P class="kuvateksti">
				Lassi and Niklas figuring out the gene circuit.
			</P><P class="kuvateksti">
				Our ideas on whiteboard. Photo by Stewart Dowden.
			</P></DIV></DIV><H3>Earlier Research</H3><DIV class="row"><DIV class="col-md-8 text-col-left"><H4 class="nopaddingtop">Why to Use Light</H4><P>
		        Previously gene regulation was controlled by chemical effectors, such as tetracycline and IPTG. In many bacterial cultures, a constant and uniform level of concentration is hard to maintain as the chemicals get absorbed, degraded and modified by the cells and environment. Moreover, the export of these chemicals is hard and time consuming or even impossible and the response time is limited by the rate of chemical addition and diffusion. On top of all, the effector chemicals may even be toxic or they might bind to unknown receptors leading to unwanted effects in the bacteria. (Schmidl et al., 2014)
		    </P><P>
		        Light signals, on the other hand, are non-toxic and we have full control over them: it is very easy to increase and decrease the light intensity. In many cases, light is orthogonal signal that does not affect to the indigenous cell mechanisms and pathways. Such is the case in for example <EM>E. coli</EM>. We are also able to target light very specifically to single colonies or even single cells allowing unprecedented accuracy. This allows very precise spatial and temporal control over the light signal. Lastly, the light signals can be emitted using LEDs which are widely available, durable and very inexpensive even in a large scale.
		    </P><H4>Light Regulated Gene Expression in Earlier Research</H4><P>
		        Optical control over molecular interactions has been harnessed in metabolic signaling, dimerization systems, light activated adrenergic receptors and neuronal ion channels (Tabor et al., 2011). Most importantly in terms of synthetic biology we can now regulate transcription with light. One of the foundational papers were written in 2005 by Levskaya et al.: <EM>Synthetic biology: engineering Escherichia coli to see light</EM>. Since then, many different light sensors and light regulators have been built. This toolbox enables more precise spatial and temporal control over the synthetic biological systems. Hence, light regulation has been part of many iGEM projects. Light sensors are arguably one of the most important elements in controlling gene expression in synthetic biology. In contrast to chemicals, we can have full control over light intensity.
		    </P><P>
		        The applications of the previous iGEM teams include communication, kill switches and metabolic engineering, among many others. Previous teams have been very active in making BioBricks out of new light sensors published in research. Uppsala 2011 team designed a multichromatic light sensor system which regulates three independent genes. In their project they used blue, green and red light sensors and linked them to different chromoproteins. This way, different light signals produce different colors in bacteria. They added many useful BioBricks to the registry which has made also our project more feasible.
		    </P><P>
        		STJU BioX Shanghai 2013 team aimed to create a universal system to regulate genomic genes using blue, red and green light and CRISPR and <EM>dCas9</EM> protein. Their fundamental idea was to regulate production of sgRNA with different wavelengths. This would enable regulation of three different genomic genes within the chromosome, presumably independently of each other. Switching the regulatory targets would be done by changing the sgRNAs. The Shanghai team also built a box with computer controlled LEDs of all three different colors. In addition, they shared the software as well as the instructions how to build such a light regulation box. This was very interesting project as we built a light regulation system for three genes using LEDs and a microcontroller. STJU BioX Shanghai gave some inspiration to our LED control system.
    		</P><P>
        		Interestingly, also Freiburg 2013 team made a universal regulatory system using CRISPR and dCas9 protein but they targeted mammalian cells. In their concept, <EM>dCas9</EM> is guided to the site and the function can be chosen remotely using different wavelengths of light. The effector either activates or represses the target gene or methylates a histone tail affecting the histone code at the site. The three effectors: repressor (KRAB), activator (VP16) and methyltransferase (G9a), can be controlled with red, blue and UV light. Such system would be able to regulate the target genes remotely in diverse ways.
    		</P><BLOCKQUOTE>
                &quot;We program living cells to sense, compute, and respond to information in their environments. Our programmable cells have applications in basic science, industry, and medicine.&quot;
                <CITE>Tabor Lab</CITE></BLOCKQUOTE><P>
        		Maybe the most interesting research group in this field is Jeffey Tabor's group. On their website they write: &quot;We program living cells to sense, compute, and respond to information in their environments. Our programmable cells have applications in basic science, industry, and medicine.&quot; (http://www.taborlab.rice.edu/home, 20.5.2014). They have been involved in developing new light sensors, including CcaS/CcaR and Cph8. They also constructed a multichromatic gene regulation system combining both these receptors. In a very recent paper (Schmidl et al., 2014) they re-engineered Cph8 and CcaS/CcaR sensors to reduce cross-talk and leakiness and to increase the dynamic range.
    		</P></DIV><DIV class="col-md-4 img-100"><P class="kuvateksti">
				Laura and Oskari looking for something between alpha and omega.
			</P><P class="kuvateksti"><A href="https://2011.igem.org/Team:Uppsala-Sweden">Uppsala 2011</A> chromoprotein collection.
			</P><P class="kuvateksti">
				Image of <A href="https://2013.igem.org/Team:SJTU-BioX-Shanghai">STJU BioX Shanghai 2013</A> slightly modified.
			</P><P class="kuvateksti">
				Image of <A href="https://2013.igem.org/Team:Freiburg">Freiburg 2013 Team</A>.
			</P></DIV></DIV><P>
       	Our setup was heavily inspired by Tabor Lab. In their experiments they exposed bacterial cultures to light using a Light Tube Array (LTA). The LTA consists of 64 individually controlled slots for tubes and it can be placed inside an incubator in order to gather big amounts of experiments with precise control.
    </P><P>
		Only very little technical information about the rig hardware was available, so we designed our own. They still work using the same principles.
	</P><P>
		Their rig is much better calibrated than ours. They precondition the cultures with a &quot;staggered start&quot;, which assures similar starting conditions for each tube (Olson et al., 2014, Supplementary Note 1). We did not perform this kind of starting procedure, but all our cultures were grown in the dark in similar conditions and diluted to same OD<SUB>600</SUB> to minimize the sample variability in the beginning.
    </P><P>
       When handling our data, we should take into account that the LED rig runs on batteries. The depletion of the battery charge results in lower power supply, although this should be covered to some extent by the on-board voltage regulator. Tabor Lab points out that newly produced fluorescent proteins do not exhibit the fluorescence signature until maturation has completed (Olson et al., 2014, Supplementary Note 2). This means that the actual measured fluorescence does not represent the exact protein concentration. Their approach was to do a &quot;post treatment&quot; for the grown cultures to ensure protein maturation. We entirely ignored this fact, mainly due to time restrictions and to the possible mis-exposure that could occur during such treatment.
   	</P><P>
        In contrast to the previous research, our system attempts to regulate multiple targets with only one wavelength of light.The regulatory effect of this light depends on its intensity. In the paper by Tabor et al. (2011) they say: &quot;The development of multichromatic gene regulatory systems, where different light wavelengths regulate the expression of different genes, will allow more advanced control of synthetic and natural gene regulatory networks.&quot; Whereas we completely agree with this, we also claim if we're able to use single wavelengths more efficiently, we could achieve even greater level of control of the synthetic gene regulatory networks.
    </P><H3>In Search of the Perfect Light Sensing System</H3><P>
        First of all, to be able to control gene expression with light, we needed a light receptor protein. It would react to a certain wavelength of light and mediate the signal to the DNA level and initiate the response. We found different alternatives in the iGEM Registry but most of them were not properly characterized. We could not find reliable experiences of the use of such light receptors or, when we found them, the receptor protein was not available. Our requirements for the light receptor system were following: 1) it has worked for other iGEM teams 2) it should work in <I>E. coli</I> 3) it is available in the Registry (and preferably in the 2014 distribution) 4) it does not require any cofactors or special strains 5) short response time and sensitivity. In this section, we discuss about the different alternatives we found and what made us choose the YF1 protein. We also introduce the mechanism how the light signal is conveyed in our system.
    </P><P>
        Phytochrome interacting factors (PIF's) were used by some previous iGEM teams, including Freiburg 2013 and TU Munich 2012. The actual red light sensing protein is called Phytochrome B and in the active form it binds to the PIF (<A href="http://parts.igem.org/Part:BBa_K1150005">BBa_K1150005 or </A><A href="http://parts.igem.org/Part:BBa_K365000">BBa_K365000</A>). Thus, the cell also needs to express Phytochrome B. However, these PIFs were derived from <I>Arabidopsis thaliana</I>, which is an eukaryotic organism. Moreover, the response mechanism would require fusing the two proteins to the actual DNA binding domain and transcription factor. Unfortunately, Freiburg team only succeeded in 1.3-fold induction of gene expression after 48 hours of incubation in red light and TU Munich did not manage to implement the light sensing part in their kill switch.
    </P><P>
        The Peking 2012 team designed an ultra-sensitive light sensor for luminescence. They combined a LOV domain, a dimerizing domain and a DNA-binding LexA domain. Such protein becomes active when the cells are illuminated with 440 - 480 nm light. In active form, the dimerizing domains facilitate formation of LexA dimer which is able to bind DNA and repress transcription in only such form. Whereas their light sensing system seemed to work very well there were three issues with it. Firstly, their light sensing system was not available and secondly, it might have been too sensitive; controlling the light at very low intensities could be rather challenging. Furthermore, LexA is a part of SOS system which is integral part of the <I>E. coli</I> genome so it might interfere with indigenous <I>E. coli</I> genes. However, according to Peking 2012, mutated version of LexA could solve this problem.
    </P><P>
        Cph8 (<A href="http://parts.igem.org/Part:BBa_I15010">BBa_I15010</A>) is a fusion of Cph1 red photoreceptor domain (660nm) and EnvZ histidine kinase domain. In absence of red light, EnvZ is active and it phosphorylates endogenous OmpR which in turn activates OmpC promoter (<A href="http://parts.igem.org/Part:BBa_R0082">BBa_R0082</A>). Red light activates Cph1 which inhibits EnvZ kinase domain function. However, as both EnvZ, OmpR and OmpC are endogenous in <I>E. coli</I>, reliable use would require EnvZ deficient strain. The activation Cph1 requires Phycocyanobilin which can be produced with heme oxygenase (<A href="http://parts.igem.org/Part:BBa_I15008">BBa_I15008</A>) and ferrodoxin oxidoreductase (<A href="http://parts.igem.org/Part:BBa_I15009">BBa_I15009</A>). The Cph8 was described by Levskaya et al. (2005) and they achieved nearly 10-fold induction. However, as EnvZ and OmpR are involved in indigenous osmolarity response, phosphorylated OmpR might also interfere with promoters encoding membrane porins.
    </P><P>
        CcaS (<A href="http://parts.igem.org/Part:BBa_K592001">BBa_K592001</A>) is a cyanobacteriochrome that gets activated by autophosphorylation. This is upregulated by green light (535nm). The activated CcaS phosphorylates its response regulator CCaR (<A href="http://parts.igem.org/Part:BBa_K592002">BBa_K592002</A>) which then induces the promoter <EM>P</EM><SUB>cpcG2</SUB> (Hirose et al., 2008; Tabor et al., 2011). As Cph8 system, this light receptor also requires Phycocyanobilin to function. In addition, CcaS is inactivated in red light (672nm) allowing higher degree of control and fast response times. According to the experiments by Tabor et al. (2011), the promoter <EM>P</EM><SUB>cpcG2</SUB> showed only 2-fold induction in green light. The transcription response is somewhat linear to the light intensity of 0.01 W/m<SUP>2</SUP> after which it rapidly saturates. Although CcaR and CcaS BioBricks are indeed available although the experiences of other iGEM teams were very few and poorly described.
    </P><H3 id="yf1-title">YF1 Light Sensor</H3><P>
        YF1 (<A href="http://parts.igem.org/Part:BBa_K592004">BBa_K592004</A>) is an engineered blue light (previous experiments in 430nm) sensor protein that consists of two initially separate domains, LOV domain (sensory domain) from YtvA and histidine kinase domain (effector domain) from FixL. This fusionprotein was designed and constructed by Möglich et al. (2009). YF1, nor it's precursors, do not exist naturally in <I>E. coli</I> nor reportedly interfere with indigenous <I>E. coli</I> genes but according to Möglich et al. (2009) the sensor works well in <I>E. coli</I>: it's capable of 70-fold supression of the target promoter when illuminated with blue light. The sensor does not require additional, foreign molecules in order to get activated by blue light. The gene is only 1132 bp long and it has been used by previous iGEM teams, including <A href="https://2011.igem.org/Team:Uppsala-Sweden">Uppsala 2011</A>, <A href="https://2013.igem.org/Team:SJTU-BioX-Shanghai">STJU BioX Shanghai 2013</A> and <A href="https://2013.igem.org/Team:Tokyo-NoKoGen/light">Tokyo NoKoGen 2013</A>. Thus, we evaluated YF1 to be the most suitable light sensor protein for our three channel switch.
    </P><P>
        Based on the work of Möglich et al. (2009) we describe the mechanism of YF1 blue light sensor in greater detail. YF1 is a water soluble chimeric fusionprotein that contains two domains linked together with a Jα linker sequence. The N terminal domain is a flavin nucleotide binding LOV domain (light-oxygen-voltage, a sub group of PAS sensor domains) from <I>Bacillus subtilis</I> YtvA. This domain incorporates flavin mononucleotide noncovalently and this induces a conformational change in this light sensory domain.
    </P><BLOCKQUOTE>
        YF1 light sensor protein is a combination of two protein domains from two different organisms.
    </BLOCKQUOTE><P>
        The C terminal domain of YF1 is a histidine kinase, the FixL effector domain from <I>Bradyrhizobium japonicum</I>. The original protein, FixL, is involved in regulating the nitrate respiration and nitrate fixation. The histidine kinase is functioning only when it is phosphorylated. It has autophosphorylation activity that is regulated by the sensor domain. Autophosphorylated histidine kinase activates a FixJ (<A href="http://parts.igem.org/Part:BBa_K592005">BBa_K592005</A>) response regulator by transferring a phosphoryl group. However, when the histidine kinase domain is unphosphorylated, it has actually phosphatase activity for the response regulator (Diensthuber et al., 2013). Phosphorylated FixJ binds to <EM>FixK</EM><SUB>2</SUB> promoter (<A href="http://parts.igem.org/Part:BBa_K592006">BBa_K592006</A>) initiating the expression of the corresponding genes. This system should be completely orthogonal in <I>E. coli</I>.
    </P><P>
        Möglich et al. (2009) proposed that the YF1 proteins form dimers. This would also bring the histidine kinase domains close to each other. It is proposed that the autophosphorylation actually works in trans within the dimer. They hypothesized the Jα helix linker would form a coiled coil with another YF1 protein. The linker would mediate the signal from the sensor domain to the effector domain inhibiting/enabling its kinase activity. The conformational change from the incorporated flavin domain is suggested to rotate the linker helix. The relative angle of the two domains greatly affect the mechanism. If one amino acid was added to the Jα linker, the relative angle changes and the effect of the sensor domain would be inverted: in presence of light the histidine kinase is active but in darkness, it is inactive. This presents interesting approaches and easy solution to invert the YF1 signal.
    </P><P>
        In our three channel switch, we fused a CI protein to the <I>Bradyrhizobium japonicum</I><EM>FixK</EM><SUB>2</SUB> promoter. Thus, in the absence of blue light (470 nm in our experiments), the angle between YF1 domains induces autophosphorylation of histidine kinase of YF1, which then transfers a phosphate group to FixJ activating it. The P-FixJ binds to <EM>FixK</EM><SUB>2</SUB> promoter activating the CI transcription. The role of this CI protein in our switch is described in the next section.
    </P><H3 id="lambda-title">Lambda (λ) Repressor</H3><P>
		Lambda repressor plays a crucial role in our switch. The repressor is a part of the lambda phage and it regulates the lysogenic and lytic states of the virus. The lambda phage genetic material can be integrated in the host genome as prophage. This phase is called lysogenic cycle and during that state, the host cell can proliferate and function normally. However, in lytic cycle the phage genetic material is separate from the host genome and it is heavily expressed to produce more viruses which eventually lyse the host cell (hence the name). Lambda repressor is the region in the lambda phage that is responsible for regulating the current state in infected host. This phage has some very interesting characteristics that are described in this subsection.
    </P><DIV class="row"><DIV class="col-md-8 text-col-left"><P>
        		In the center of the lambda repressor there is a lambda repressor protein CI (<A href="http://parts.igem.org/Part:BBa_K105004">BBa_K105004</A>). This protein represses the genes in the lytic cycle and maintains the lysogenic state. Moreover, this protein is autoregulated in a very special manner. CI dimerizes () and binds into its own promoter (<EM>P</EM><SUB>RM</SUB>) upregulating its own production. However, in high concentrations, CI represses its production maintaining it on an appropriate level. Dimerized CI represses a lytic cycle transcription factor Cro by binding to operator regions before <EM>P</EM><SUB>R</SUB>. The lambda phage changes the state from lysogenic cycle to lytic cycle (i.e. lysogenic induction) when the host is exposed to DNA damaging factors, such as UV light. The damage activates the SOS response and the production of RecA protein involved in repair and maintenance of DNA. RecA has protease activity and it cleaves CI resulting in a drop of CI dimer concentration, derepression of Cro and lysogenic induction. Dodd et al. (2001) showed that the concentration of CI affects the capability for lysogenic induction. Thus, it is important that lambda phage maintains the CI concentration in specific frame in order to be able to switch to lytic cycle when the opportunity arises. High concentration leads to inability of lysogenic induction whereas low concentration results in too high induction sensitivity.
    		</P><BLOCKQUOTE>
                Lambda repressor protein CI is able to both upregulate and repress its own production.
            </BLOCKQUOTE><P>
				The interaction of CI protein and lambda repressor promoters is well illustrated in the . The repressor region consists of two promoters, <EM>P</EM><SUB>RM</SUB> and <EM>P</EM><SUB>R</SUB> separated by common operator sites (<EM>O</EM><SUB>R</SUB> 1, 2 and 3). The CI protein dimers bind to the DNA at these three operator sites. Johnson et al. (1979) showed that CI binds cooperatively on the three operator sites (). The <EM>O</EM><SUB>R</SUB>1 has the highest affinity for the CI dimer which recruits another dimer to the <EM>O</EM><SUB>R</SUB>2 site. This blocks the transcription of the <EM>P</EM><SUB>R</SUB> promoter and activates the <EM>P</EM><SUB>RM</SUB> to produce CI. Bell et al. (2000) suggested that these CI dimers at the two operator sites form a tetramer. Very similar interaction is observed between two of the three <EM>O</EM><SUB>L</SUB> operator sites which are located 2,4kb away from <EM>O</EM><SUB>R</SUB> operator. In addition, CI binds to <EM>O</EM><SUB>R</SUB>3 and <EM>O</EM><SUB>L</SUB>3 sites, but only in substantially higher CI concentrations (Johnson et al., 1979). This leads to the repression of the <EM>P</EM><SUB>RM</SUB> promoter. Maurer et al. (1980) showed that 50% repression of <EM>P</EM><SUB>RM</SUB> requires 15 times the normal lysogenic concentration. However, the mechanism of <EM>P</EM><SUB>RM</SUB> repression is not result of only CI dimer binding in the <EM>O</EM><SUB>R</SUB>3 site.
    		</P><P>
        		Dodd et al. (2001) showed that two tetramers at <EM>O</EM><SUB>L</SUB> and <EM>O</EM><SUB>R</SUB> sites form an octamer by looping the DNA (). They proposed that this DNA looping juxtaposes <EM>O</EM><SUB>R</SUB>3 and <EM>O</EM><SUB>L</SUB>3 sites and allows them to be linked by a CI tetramer. This, in turn, would silence the <EM>P</EM><SUB>RM</SUB> promoter. Thus, the lambda repressor has three different states regulated by CI concentration: lytic (low CI concentration, <EM>P</EM><SUB>R</SUB> is active and <EM>P</EM><SUB>RM</SUB> is inactive), lysogenic (moderate CI concentration, <EM>P</EM><SUB>R</SUB> is inactive and <EM>P</EM><SUB>RM</SUB> is active) and repressing lysogenic (high CI concentration, <EM>P</EM><SUB>R</SUB> is inactive and <EM>P</EM><SUB>RM</SUB> is inactive). This is the underlying principle of our three-channel switch. In our approach, we regulate the CI concentration with blue light intensity and replace the <I>cro</I> and <I>cI</I> genes of lambda repressor with other genes of interest while producing CI elsewhere. This way, we are able to use the characteristics of the lambda repressor to construct the mechanism behind the three-channel switch.
    		</P></DIV><DIV class="col-md-4 img-100"><P class="kuvateksti"><B><SPAN ref="cidimerization">.</SPAN></B>: Repressor protein CI forms dimers.
			</P><P class="kuvateksti"><B><SPAN ref="ciconcentration">.</SPAN></B>: The <EM>O</EM><SUB>R</SUB>1 has the highest affinity for the CI dimer which recruits another dimer to <EM>O</EM><SUB>R</SUB>2 site. <EM>O</EM><SUB>R</SUB>3 has the lowest affinity for CI and CI binds to it only in a high enough concentration.
			</P></DIV></DIV><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="lambdaloop">.</SPAN></B>: Two tetramers at <EM>O</EM><SUB>L</SUB> and <EM>O</EM><SUB>R</SUB> sites form an octamer by looping the DNA. DNA looping juxtaposes <EM>O</EM><SUB>R</SUB>3 and <EM>O</EM><SUB>L</SUB>3 sites and allows them to be linked by a CI tetramer.
			</P></DIV></DIV></ARTICLE><ARTICLE class="abg-dnaseq"><H2>Hypotheses</H2><P>
  		The very first and critical assumption is that the concentration of CI would be controlled by light when it is put behind <EM>FixK</EM><SUB>2</SUB> promoter. Moreover, we assume that this effect will be more or less linear on the relevant region so that double light intensity leads eventually to double concentration. It is also very crucial that <EM>FixK</EM><SUB>2</SUB> promoter does not show significant leaky expression in absence of phosphorylated FixJ. We need to be able to keep the CI concentration low enough to keep the <EM>P</EM><SUB>R</SUB> promoter active in the presence of strong blue light.
    </P><P>
        To better match the BioBrick standards and the idea about being able to ligate BioBricks together as is, we wanted to make our promoters unidirectional. The <EM>P</EM><SUB>RM</SUB> and the <EM>P</EM><SUB>R</SUB> promoters, however, point to opposite directions. This would require flipping the BioBricks added to the preceding strand or using reverse complement BioBricks. This would not be very convenient. Hence, we made a duplication of the OR sites, one coupled with the <EM>P</EM><SUB>RM</SUB> and the other connected to the <EM>P</EM><SUB>R</SUB> promoter. We suggest we are able to repress the <EM>P</EM><SUB>R</SUB> promoter and activate the <EM>P</EM><SUB>RM</SUB> promoter with CI dimers despite this duplication.
    </P><P>
        The third hypothesis is related to the DNA looping as a result of CI octamerization at the operator sites (). Originally, DNA looping takes place between <EM>O</EM><SUB>R</SUB> and <EM>O</EM><SUB>L</SUB> sites by octamerization of CI dimers bound to operator sites 1 and 2 at both locations. As <EM>O</EM><SUB>R</SUB> and <EM>O</EM><SUB>L</SUB> regions resemble each other in terms of CI affinity and as the interacting factor seems to be CI and not the DNA sequence, we hypothesize that <EM>O</EM><SUB>L</SUB> could be replaced by <EM>O</EM><SUB>R</SUB> region without losing the DNA looping capability. This would also underline the significance of the previous assumption: as we have duplicated the <EM>O</EM><SUB>R</SUB> sites we already have the hypothetical factors required for DNA looping and <EM>P</EM><SUB>RM</SUB> repression. Therefore, if we duplicate the <EM>O</EM><SUB>R</SUB> region and separate them with correct number of bases, we might be able to form the DNA loop between these regions without having an <EM>O</EM><SUB>L</SUB> region as illustrated in .
    </P><P>
        The lambda repressor functions in lambda prophage when the virus is in the lysogenic state. This means the lambda repressor is integrated as a part of the genome. It has been shown that the DNA looping takes place in the infected <EM>E. coli</EM> cells (Manzo et al., 2012). However, we have not found evidence that this would happen also in the plasmids containing the lambda repressor region. It is possible the supercoiled structure of the plasmid prevents the normal function of the lambda repressor. We have identified three possible scenarios. Firstly, the lambda repressor could function within the plasmid without any issues. Alternatively, the loop size formed by lambda repressor may be changed or it may be easier or more difficult to be formed but it may work with some modifications. The worst scenario is that the lambda repressor does not work at all in the plasmid context. In such case, our three channel switch could be integrated to the <EM>E. coli</EM> genome.
  	</P><P>
  		Lastly, we made some hypotheses regarding the CI concentration. The wild type CI half-life has been measured to be longer than 10 hours (Parsell et al., 1990). This would mean that it takes very long time to reduce the CI concentration that is, switch back from the channel C in our system. However, we can fuse an ssrA tag to the protein an target it for faster degradation within the cell. According to Keiler et al. (1996), such tagging can reduce the half-life to 4 minutes. Hence, we used the LVA tagged version of the CI lambda repressor protein (<A href="http://parts.igem.org/Part:BBa_K327018">BBa_K327018</A>). This enables much shorter reaction time and nearly real-time control of the CI concentration. However, it also lowers the highest possible concentration of CI. We hypothesize that by using a strong enough promoter, we can reach sufficient CI concentrations for <EM>P</EM><SUB>RM</SUB> repression.
  	</P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="geneaactive">.</SPAN></B>: This is our unidirectional version of the lambda repressor with duplicated <EM>O</EM><SUB>R</SUB> sites and planned <I>genes A and B</I>. When there is no CI present, only <I>gene A</I> will be active.
			</P></DIV><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="genebactive">.</SPAN></B>: As a CI dimer binds to the <EM>O</EM><SUB>R</SUB>1 site it immediately recruits another dimer to the <EM>O</EM><SUB>R</SUB>2 site. In this situation, only <I>gene B</I> will be active.
			</P></DIV><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="switchloop">.</SPAN></B>: As CI octamerizes, the DNA will be looped and both <I>genes A</I> and <I>B</I> are deactivated. <I>Gene C</I> is then activated.
			</P></DIV></DIV></ARTICLE><ARTICLE class="abg-pcb"><H2>Parts</H2><P>
        The gene circuit consists of two different segments: the light sensor that produces CI according to the intensity of light and the actual switch that responds to the differences in the concentration of the repressor protein CI. You can find the complete sequence of the switch from <A href="https://static.igem.org/mediawiki/2014/4/47/Aalto_Helsinki_Gene_Switch.txt" id="sequence">this file (click)</A>.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="genecircuit">.</SPAN></B>: This is our gene circuit. The upper part is the light sensing segment that produces the CI protein and the lower part reacts to differences in the CI concentration and switches the gene channel. The turquoise arrows are promoters, the turquoise circles are operator sites, the light blue circles are ribosome binding sites and the gray squares are expressed genes. The promoter of the YF1 gene can be any constitutive promoter.
			</P></DIV></DIV><H4>Parts in Our Gene Switch</H4><P>
        Here’s a list of all the BioBricks that we used in our gene switch. They are in the same order as they are in our gene circuit (), and the previously mentioned <A href="#Parts">sequence file</A>.
    </P><UL><LI><A href="http://parts.igem.org/Part:BBa_K823008 ">BBa_K823008</A> Anderson Promoter J23106</LI><LI><A href="http://parts.igem.org/Part:BBa_K592016">BBa_K592016</A> B0034-YF1-B0034-FixJ, blue light sensor and RR with RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> Double terminator</LI><LI><A href="http://parts.igem.org/Part:BBa_K592006">BBa_K592006</A><EM>FixK</EM><SUB>2</SUB> Promoter</LI><LI><A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> Elowitz Reference RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_K105004 ">BBa_K105004</A> CI protein CDS</LI><LI><A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> Double terminator</LI><LI><A href="http://parts.igem.org/Part:BBa_K1443002">BBa_K1443002</A> Lambda Repressor <EM>O</EM><SUB>R</SUB>1-<EM>O</EM><SUB>R</SUB>2-<EM>O</EM><SUB>R</SUB>3-<EM>P</EM><SUB>RM</SUB><STRONG>(Created by Aalto-Helsinki!)</STRONG></LI><LI><A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> Elowitz Reference RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_E0022 ">BBa_E0022</A> Cyan Fluorescent protein + LVA (as <I>gene B</I>)</LI><LI><A href="http://parts.igem.org/Part:BBa_B0032">BBa_B0032</A> Medium Strength RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_C0040">BBa_C0040</A> Tetracycline repressor protein + LVA</LI><LI><A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> Double terminator</LI><LI><A href="http://parts.igem.org/Part:BBa_K1443001 ">BBa_K1443001</A> Lambda Repressor <EM>O</EM><SUB>R</SUB>3-<EM>O</EM><SUB>R</SUB>2-<EM>O</EM><SUB>R</SUB>1-<EM>P</EM><SUB>R</SUB><STRONG>(Created by Aalto-Helsinki!)</STRONG></LI><LI><A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> Elowitz Reference RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_E0032 ">BBa_E0032</A> Yellow fluorescent protein + LVA (as <I>gene A</I>)</LI><LI><A href="http://parts.igem.org/Part:BBa_B0032">BBa_B0032</A> Medium Strength RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_C0040">BBa_C0040</A> Tetracycline repressor protein + LVA</LI><LI><A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> Double terminator</LI><LI><A href="http://parts.igem.org/Part:BBa_R0040 ">BBa_R0040</A> Tetracycline repressible promoter</LI><LI><A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A> Elowitz Reference RBS</LI><LI><A href="http://parts.igem.org/Part:BBa_K082003 ">BBa_K082003</A> Green Fluorescent protein + LVA (as <I>gene C</I>)</LI><LI><A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A> Double terminator</LI></UL><H4>Parts That We Created</H4><P>
		Here's the <A href="https://2014.igem.org/Team:Aalto-Helsinki/Parts">link to our official group parts table (click)</A>.
	</P><P>
		List of the parts:
	</P><UL><LI><A href="http://parts.igem.org/Part:BBa_K1443000">BBa_K1443000</A> Blue light response element</LI><LI><A href="http://parts.igem.org/Part:BBa_K1443001">BBa_K1443001</A> Lambda Repressor <EM>O</EM><SUB>R</SUB>1-<EM>O</EM><SUB>R</SUB>2-<EM>O</EM><SUB>R</SUB>3-<EM>P</EM><SUB>RM</SUB></LI><LI><A href="http://parts.igem.org/Part:BBa_K1443002">BBa_K1443002</A> Lambda Repressor <EM>O</EM><SUB>R</SUB>3-<EM>O</EM><SUB>R</SUB>2-<EM>O</EM><SUB>R</SUB>1-<EM>P</EM><SUB>R</SUB></LI><LI><A href="http://parts.igem.org/Part:BBa_K1443003">BBa_K1443003</A> RFP binding forward primer for backbone amplification</LI><LI><A href="http://parts.igem.org/Part:BBa_K1443004">BBa_K1443004</A> RFP binding reverse primer for backbone amplification</LI></UL></ARTICLE><ARTICLE><H2>Methods</H2><H3>Lab Protocols</H3><P>
  		Here are the protocols we used in the lab as a <A href="https://static.igem.org/mediawiki/2014/0/02/Aalto_Helsinki_Protocols.pdf">PDF file (click)</A>.
  	</P><H4>The PDF contains protocols for the following</H4><UL><LI>Making Heat-Shock Competent Cells</LI><LI>Transformation of Heat-Shock Competent Cells</LI><LI>Making Electrocompetent Cells</LI><LI>Transformation of Electrocompetent Cells</LI><LI>Making SOC Broth</LI><LI>Restriction Digestion</LI><LI>Ligation</LI><LI>PCR</LI><LI>Agarose Gel Electrophoresis</LI></UL><H3 id="ledrig-title">LED Rig</H3><DIV class="row"><DIV class="col-md-8 text-col-left"><P>
		  		To be able to expose our bacteria to light in a controlled manner, we designed a new kind of device from scratch. Inspired by the LTA developed by Tabor Lab (Olson et al., 2014) we decided to build a simpler one, only with blue LEDs.
		  	</P><BLOCKQUOTE>
                We built a portable device that can illuminate 16 bacterial cultures with different intensities of blue light.
            </BLOCKQUOTE><P>
		  		Luckily we had some knowledge of electronics in our team, too. Pietu designed the rig and built a foam-padded transportable casing that can be put in an incubator overnight.
		  	</P><H4>Microcontroller</H4><P>
		  		The core of the rig is an Arduino Nano microcontroller. The Arduino is responsible for the logic behind our illumination patterns.
		  	</P><P>
		  		The patterns in our experiments are fairly simple and could in theory be done without the Arduino, but we chose to use it due to its wide popularity and because we now could make advanced (and cool) animations on our rig. Also the Arduino acts as a 5V voltage regulator which provides the voltage for the second part of the rig: the Adafruit 16-channel 16-bit PWM controller.
		  	</P><H4>LEDs</H4><P>
		        Light Emitting Diodes(LEDs) are current controlled components, and their intensity depends on the current passing through them. LEDs are great because they consume little power and don't generate much heat. It is important that the LEDs wont heat up the bacterial culture remarkably.
		    </P><P>
		    	Controlling the intensity of these LEDs is trivial by varying the current, but this will cause the emission spectra to shift  (Muthu et al., 2002). Hence we chose a different, and very popular method for varying the LED intensity: Pulse Width Modulation (PWM). The idea behind PWM is to turn the LED on and off in very short intervals, up to frequencies of 1.6kHz. By varying the time the LED is on and off, we will be able to make a perceived difference in the intensity. The bigger the ratio of on-time versus off-time the brighter the LED.
		    </P><P>
		    	Due to the hurry of making this rig, we chose to use a pre-made Arduino &quot;shield&quot; for handling the PWM. We chose a 16-channel 16-bit PWM shield from Adafruit. The 16 bits provide us much more range than the standard 8 bits of the Arduino (light intensity is an integer from 0 to 4095). Additionally, the Nano does not support as many channels, so having 16 LEDs provides more flexibility in our measurements. The LEDs we used are 5mm Blue 470nm LEDs with an intensity of 1800mcd, 15° angle, Vf@20mA:3,5V (Liteon: LTL-2P3TBK). If you are building the same setup as us, we recommend you to get good quality LEDs from a known manufacturer who can supply a specifications sheet for the components.
		    </P><H4>Constructing</H4><P>
		  		We wanted to be able to illuminate the bacterial culture with multiple LEDs with different intensities. We chose to grow our bacteria on microtiter plates. This choice was also affected by the fact that we had an access to a fluorometer (Thermo Scientific Varioskan) that measures fluorescence out of such microtiter plates.
		  	</P><P>
		  		We designed our LED rig so that an unmodified 96-well microtiter plate with the test cultures could be easily inserted. Our LEDs are attached to a lid that can be placed on a standard microtiter plate. 16 LEDs will illuminate the wells on columns 11,9,7,5 and rows B, D, F, H.
		  	</P><P>
		  		Our LEDs' spectrum peak is at the value of 470nm. This value is specified by the manufacturer but the spectra of individual LEDs can vary. Our aim was to measure the emission spectra of LEDs with a spectrophotometer to be sure of the excitation wavelength, but unfortunately the spectrophotometer at the aiding lab was not functioning and we were not able to get the emission spectra.
				We did however measure the intensities of our LEDs at 470nm wavelength.
			</P><P>
				We measured intensities for two randomly selected LEDs (nr. 9 and 6) out of the total 16.
				Measurements were done with a UDT instruments S450 Optometer with 1cm<SUP>2</SUP> detector.
				The measurements were done in a dimmed room and measured at 470nm. We measured both LEDs when the LED rig was supplied with a USB voltage of 5V and using 4 AAA-type batteries.
				One of the LEDs was also measured with nearly depleted batteries for reference.
			</P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="ledintensitymeasurements">.</SPAN></B>: LED intensity as a function of intensity parameter. The intensities were measured for two LEDs (nr. 9 and 6) with different power supplies attached to the rig. USB refers to 5V input from a computer and newbatt refers to 4 fresh batteries of AAA-type (LR3). Intensity of LED nr. 9 was also measured with nearly depleted batteries for reference (oldbatt).
            </P></DIV></DIV><P><A ref="ledintensitymeasurements">.</A>  maps the intensity parameters to absolute intensity values. We can conclude the measured LEDs are very close in measured intensities and this suggests that the LEDs' intensities of the different components don't vary much. We can also note the LED intensity is a function of the battery level, which is unfortunate. The design of the LED rig is not perfect. However, we can tackle this problem by always using fresh batteries or USB power when available. A proper voltage regulator on the LED supply voltage should be considered in the future experiments. We can see from <I>LED9 oldbatt</I> graph that the intensity decreases to a tenth of its original value when the batteries are depleted.
		  	</P><H4>Programming</H4><P>
		        The Arduino source code used for the measurements is available on our <A href="http://github.com/iGEM-QSF/pwm-led-rig/">GitHub pages</A> . An example of the source code used can be seen in the .
		  	</P><P>
		  		The most important thing to understand when varying the LED brightness is that the perceived brightness is logarithmic. So the difference between 10 and 20 is clearly visible to the eye, but the difference between 3000 and 4000 is not.
		  	</P><H4>Technical Details</H4><UL><LI>Arduino Nano microcontroller, <A href="http://arduino.cc/en/Main/arduinoBoardNano">http://arduino.cc/en/Main/arduinoBoardNano</A></LI><LI>Adafruit 16-channel PWM shield, <A href="https://learn.adafruit.com/adafruit-16-channel-pwm-slash-servo-shield?view=all">https://learn.adafruit.com/adafruit-16-channel-pwm-slash-servo-shield?view=all</A></LI><LI><A href="http://www.thermoscientific.com/content/tfs/en/product/varioskan-flash-multimode-reader.html">http://www.thermoscientific.com/content/tfs/en/product/varioskan-flash-multimode-reader.html</A></LI></UL></DIV><DIV class="col-md-4 img-100"><P class="kuvateksti">
                This is the LED rig we constructed.
            </P><P class="kuvateksti">
            	Pietu presenting the LED rig.
            </P><P class="kuvateksti">
            	LED rig close-up.
            </P><P class="kuvateksti">
            	Measuring the intensity levels of the LEDs. Did you expect to see something? We measured our LEDs in the dark!
            </P><P class="kuvateksti"><SPAN ref="sourcecode">.</SPAN>: Source code of a basic program that initializes the LED-intensities to certain values.
            </P><P class="kuvateksti">
            	The LED rig is <I>very</I> customisable.
            </P></DIV></DIV><DIV class="row"><DIV class="img-center"><P class="kuvateksti">
                Here's a short <A href="http://www.youtube.com/embed/azzMXd8zj1E" target="_blank">video of the rig</A>.
            </P></DIV></DIV><H3 id="fluorescence-title">Fluorescence Measurements</H3><P>
        Fluorescence measurements were carried out in Thermo Scientific Varioskan. It is a microtiter plate reader capable of exitation and measuring the corresponding fluorescense at user defined wavelengths. All measurements were made with overnight cell cultures diluted to the same cell density. In the LED rig there are 16 different LEDs but also many additional wells that can be used for control samples. That way we can ensure identical conditions between our test and control samples. We always used two controls: blue chromoprotein BioBrick (<A href="http://parts.igem.org/Part:BBa_K592009">BBa_K592009</A>) without a promoter as the negative control and a constitutive GFP generator from Measurement Interlabs study (<A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A>) as the positive control. All experiments were carried out in <EM>E. coli</EM> Top10 strain.
    </P><P>
        Our LED rig is controlled with programmable Arduino Nano computer. The light intensity parameters vary between 0 and 4095 where 0 corresponds to the dark state and 4095 the maximal intensity. Whereas measured intensity depends linearly on this value, the light intensity observed with naked eye (or bacterial light receptor), was logarithmically dependent on the intensity parameter. Thus, having equispaced intensity parameters for the LEDs led to biased distribution of light. To get equal difference in the LED intensities, we set the parameters linearly according to the log<SUB>10</SUB> of the intensity parameter value. In other words, the intensity of the LED lights was proportional to the log<SUB>10</SUB> of the intensity parameter.
    </P><H4>Choosing the Correct Parameters</H4><P>
        Cormack et al. (1996) reported the wavelengths of green fluorescent protein mutant 3 to be the following
        <UL><LI>Excitation wavelength: 501 nm</LI><LI>Emission wavelength: 511 nm</LI></UL></P><P>
        In addition Ashikawa et al. (2011) had used the following parameters in measuring GFP
        <UL><LI>Bandwidth: 12 nm</LI><LI>Measurement time: 500 ms</LI></UL></P><P>
        We tried measuring GFP using the parameters provided by Cormack et al., (1996) and Ashikawa et al. (2011), but we got strange results. We assumed that that this was because the excitation and emission wavelengths were so close to each other. Therefore we decided to change the excitation wavelength to 470 nm, as it seemed to be another peak of excitation for GFP mut 3 according to Cormack et al. (1996) (). For the same reason we also tried a narrower bandwidth of 5 nm, which is the other option available on Varioskan.
    </P><P class="kuvateksti"><B><SPAN ref="GFPCormack">.</SPAN></B>: Excitation and emission spectra for wild type and mutant GFPs. (A) Excitation spectra (emission 540 nm). Excitation maxima: wt GFP 395 nm, GFP mut 1 488 nm, GFP mut 2 481 nm and GFP mut 3 501 nm. (B) Emission spectra (excitation 395 nm for wild type and 450 nm for mutant types). Emission maxima: wt GFP 508 nm, GFP mut 1 507 nm, GFP mut 2 507 nm and GFP mut 3 511 nm (Cormack et al. 1996).
    </P><P>In the end we settled for the following:
        <UL><LI>Excitation wavelength: 470 nm</LI><LI>Emission wavelength: 511 nm</LI><LI>Bandwidth: 5nm</LI><LI>Measurement time: 500 ms</LI></UL></P><P>
        With the chosen values we got sensible results and our blank samples gave practically zero emission.
    </P></ARTICLE><ARTICLE><H2>Results</H2><H3>GFP Fluorescence in High OD<SUB>600</SUB> Under Different Light Intensities</H3><P>
        In this experiment, we measured how our cells with light response mechanism will react to different intensities of blue light when grown in same conditions and very high cell density. In such conditions, the cells are not able to proliferate in a way that would affect to the measurements in short time period. This was the first experiment using different intensities of LED lights at 470 nm wavelength. The light response element was fused with GFP reporter gene. Our purpose was to find out, how cells would respond to light in steady and high cell density and how the preceding conditions affect to the fluorescence during a long time period. In addition, we were to observe the relative change between the highest and the lowest level of fluorescence. We had two kinds of samples: bacteria grown  in the darkness and in lit environment (usual lab lightning in addition to desk lamp directed to our sample). We also had a negative and a positive control for the fluorescence measurements.
    </P><P>
        We constructed a testing device in order to estimate the concentration and the dynamics of the CI protein. In the desting device, we replaced CI encoding sequence with GFP encoding sequence. The GFP was fused with LVA degradation tag. We excluded the lambda repressor and gene C parts. As a result, our testing device consisted of a blue light sensing element (constitutive promoter (<A href="http://parts.igem.org/Part:BBa_J23115">BBa_J23115</A>), RBS-YF1-RBS-FixJ (<A href="http://parts.igem.org/Part:BBa_K592016">BBa_K592016</A>) and a double terminator (<A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>) ) and the testing element (FixK2 promoter (<A href="http://parts.igem.org/Part:BBa_K592006">BBa_K592006</A>), RBS (<A href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</A>), GFP-LVA (<A href="http://parts.igem.org/Part:BBa_K082003">BBa_K082003</A>) and double terminator (<A href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</A>) ).
    </P><P>
        At first, all the samples were diluted to OD<SUB>600</SUB> 2,4. They were inserted in different wells of microtiter plates. Control samples both were incubated in dark at two different wells each, and dark and light samples were incubated in 9 different intensities of blue 470 nm light (8 intensities in addition to darkness, ). The first measurements were taken after 4 hours of incubation and a second set of measurements was taken after 23 hours of incubation.  All the samples had LB as growth medium and they were cultured overnight at 37 Celsius degrees and 200 rpm.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN tab="intensities1">.</SPAN></B>: Different light intensity parameters and corresponding log<SUB>10</SUB> values used in this experiment.
            </P></DIV></DIV><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="results4h">.</SPAN></B>: GFP fluorescence of light and dark sample after 4 hours of incubation in different blue light intensities. Control samples were measured only in darkness. Positive control value (101,5) was excluded from the picture to preserve resolution.
            </P></DIV></DIV><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="results23h">.</SPAN></B>: GFP fluorescence of light and dark sample after 23 hours of incubation in different blue light intensities. Control samples were measured only in darkness. Positive control value (176,9) was excluded from the picture to preserve resolution.
            </P></DIV></DIV><P>
        From both figures we can make speculation that bacteria grown in the darkness are more fluorescent compared to the light cultures. This may be because of many different reasons. One possible explanation is that the bacteria have simply accumulated GFP during the preceding overnight incubation. Therefore, all measurement points for dark cultures would be shifted to higher level of fluorescence. However, we also know that the GFP-LVA fusion half-life, around 40 minutes (Andersen et al., 1998), is rather short compared to the time period. Thus, such higher GFP concentrations should fade over time and settle close to the actual production rate. These figures suggest that long term elevation of GFP production might take place in our system. We would need further experiments to verify this.
    </P><P>
        When we observe the , we see a clear descending trend in the fluorescence towards the higher intensities of light. This suggests that the bacteria indeed alter the rate of GFP production in response to the light even in very high cell density. The brighter the light, the lower the GFP production and hence, the lower the fluorescence. The difference between bright light and darkness seems to be 1.5-2 fold. However, this effect could also be a result of the LED lights heating up the bacterial cultures and inhibiting proliferation and thus, leading to lower total GFP amount even if the per cell concentration would be constant. We also observe leaky production of GFP even in the bright light when we compare the fluorescence to the negative control strain without a GFP gene.
    </P><P>
        When compared to the 4 hour measurements, the  does not have such a clear trend as in the . However, we also notice that the fluorescence values vary greatly and they are clearly lower compared to the values after 4 hours of incubation. This suggests that the response to the light stimuli is not as strong after long incubation and that the linear dependency on the light intensity seems to disappear. We must also take into account the length of the incubation and its effects on the cell culture. After such a long time in high cell density without adding fresh media will inherently lead to low welfare, low transcription and protein synthesis rate and poor response to the light stimuli. In other words, the 23 hour incubation is clearly too long and leads to bad sample quality.
    </P><H3>GFP Fluorescence of Our Light Response Element Compared to Constitutive GFP Producing Strain and Non-Fluorescent Strain</H3><P>
        We designed new experiments based on the previous results. First of all, we need to address how positive and negative control respond to light. We need to address the possibility that some of the effects are due to warming effects of the LED lights. Moreover, we need to measure the light response of bacterial cultures in logarithmic growth state with lower cell density. These experiments were carried out in transparent minimal medium to avoid fluorescent bias by LB medium. In the beginning of the experiment, all the cells were diluted to OD<SUB>600</SUB> of 0.5.
    </P><P>
        We prepared three different samples (light response element, positive control and negative control) and placed them under five different intensities of blue light (). The lowest intensity is actually a well without a led light, that is,near-zero intensity. However, we designate this intensity with parameter value of 1 instead of zero (which would translate to -infinity on the logarithmic scale). We had duplicate samples of our light response element and we measured the their average fluorescence in this experiment.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN tab="intensities1">.</SPAN></B>: Different light intensity parameters and corresponding log<SUB>10</SUB> values used in this experiment.
            </P></DIV></DIV><P>
        We aimed to measure the GFP fluorescence regularly, once per hour throughout the experiment. The experiments lasted for approximately six hours yielding six different time points. This means 30 data points per sample. However, as the fluorometer was reserved at times, the timepoints are not exactly regular. In addition, we noticed that between our 4th and 5th measurement, the batteries of our LED rig had died and the LED intensities had all dimmed significantly. At this point, the batteries were immediately changed to fully charged ones.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="timeseries">.</SPAN></B>: GFP fluorescence of our light response element at different intensities in different times (upper figure). The perceived intensity is a 10 base logarithm of the parameter values, given in  and the fluorescence is an average of two different samples. Negative control was calculated as the average of all the time points at the given intensity. The legend is in the figure below. The positive control is in the , and the intensity values are in the . The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><P>
        The  is very good example of the light response element’s capabilities. The first measurement is taken right after the dilutions before the incubation in blue light. We notice the fluorescence is equal for each sample (blue line, 10:43). However, already after one hour of incubation (green, 11:53), we can see a clear response to incubation in blue light. Bacteria grown in low intensity have a much higher fluorescence level compared to the those grown in high intensity of blue light. The difference is over fourfold. We also note that the effect of light intensity to the fluorescence is nearly linear. However, the intensity  here is 10 base logarithm of the actual intensity value. This means that bacteria perceive light intensity logarithmically, that is, doubling the intensity value leads to weaker effects in higher values of intensity. We also need to take into account that the intensity level 0 is not comparable to the other values as the difference between darkness and very dim light was hard to evaluate at the logarithmic scale.
    </P><BLOCKQUOTE>
        This means that bacteria perceive light intensity logarithmically.
    </BLOCKQUOTE><P>
        When the incubation is continued (red and turqoise), the fluorescence also tend to increase at every point of measurement. The fluorescence at the highest blue light intensity increases but so does the fluorescence at the lowest intensity. In fact, they change at the same rate relative to each other so that the fourfold difference is maintained. This suggests that this change in fluorescence between the timepoints is caused by the growth of bacteria in the sample leading to a GFP expression multiplied by a scalar denoting the cell density. In other words, the effect of blue light on the single cell level may be constant after the second measurement. Therefore, the full response to blue light intensity may take place already between the first and second measurement. This would imply very fast response times.
    </P><P>
        However, we observe a great difference at the 5th timepoint (violet, 15:43) because the fluorescence rockets even at higher blue light intensities. This is due to batteries running low in our LED rig. Interestingly, this incident reveals some very interesting dynamics in our light response element. Clearly, the effect of the batteries running low has emerged after the measurement at 14 o'clock. Independently of the intensity, our light response element has achieved the same fluorescence values as the samples incubated in dark since the beginning. This has happened in a time shorter than one and half hours which is the time difference between the measurements. As the main component of the fluorescence is GFP-LVA, the samples incubated in bright light have reached the full production capacity when compared to the most fluorescent samples. Moreover, this time has been enough to accumulate GFP long enough to reach seemingly steady state (degradation rate and production rate being equal). Therefore, the response is likely to be a lot shorter than the time between the measurements (one and half hours).
    </P><P>
        The last timepoint also reveals some very interesting features in our response element. As the light intensities were restored after the perturbation, we incubated the samples in the original light intensities for 75 minutes prior to doing the last measurement. When we examine these fluorescence values (orange, 16:49) we can see that the values were restored back to the same level as the measurements before the perturbation. Also the linear dependency and fourfold fluorescence difference was restored. This means that our system seems to react very fast to increase in the blue light intensity.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="samplevspositive">.</SPAN></B>: GFP fluorescence of our light response element compared to the positive control. The 3D plot with respect to time, fluorescence and intensity shows the difference in fluorescence between our sample and the constant GFP expressing strain (Top10 transformed with Measurement Interlabs plasmid with <A href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</A>). The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><P>
        One question is still yet to be addressed: does the LEDs interact with heat or is light indeed the mechanism that affects do the fluorescence? Or maybe the blue light interferes straight with GFP inhibiting its fluorescence? To rule out these effects, we compare our samples to a positive and negative control strains. We observe the effect of the blue light on the strain producing GFP at a constant rate and on strain without GFP encoding sequence.
    </P><P>
        In the , we have plotted average value of negative control strain fluorescence as a function of blue light intensity. The values were averaged over all six timepoints. As we see, the fluorescence levels are clearly lower than any of the measurements with our light response element. Moreover, the value is nearly constant, only decreasing slightly towards higher intensities. This suggests that light intensity may have some minor decreasing effects on fluorescence measured at 511 nm. However, these differences are negligible when compared to the differences in fluorescence observed with the light response element. Moreover, this also suggests that the fluorescence levels are clearly elevated even at the highest blue light intensities.
    </P><P>
        In  we have compared our light response element with a strain producing GFP at a constant rate. First of all, we see that the fluorescence levels are on the different level of magnitude. This is presumably due to significantly longer half-life and presumably higher transcription rate. This is supported by the fact that the fluorescence accumulates through the time (although the effect is largely contributed by the dividing cells). The main interest is, however, to compare the effect of blue light to the fluorescence. First of all, we observe inclination to higher fluorescence levels at higher blue light intensities. This is opposing our previous observation with negative control. Moreover, this change is the opposite to what is observed in our light response element, which supports our hypothesis. The higher fluorescence may be, among many other possible reasons, result of LEDs heating up the sample a little bit and enabling the cells to proliferate a little bit faster during the measurements carried out in room temperature. We also need to take into account that as we only had one sample per intensity, the fluorescence values are subject to fluctuation and we cannot reliably conclude that there would be an increasing trend towards higher intensities in fluorescence of the positive control strain.
    </P><H3>Induction Dynamics and GFP Fluorescence Under Varying Light Intensity</H3><P>
        As we noticed in the previous measurement results, the response dynamics are too fast to be observed in the time scale of one hour. We designed a new experiment which aims to capture how fast the bacteria are able to respond to the differences in light intensity. There are two main factors in this feature: delay in transcriptional activation/deactivation and half-life of the protein being measured. We can approximate that the delay includes all the steps from YF1 phosphorylation to protein translation. These experiments were carried out in transparent minimal medium to avoid fluorescent bias by LB medium. In the beginning of the experiment, all the cells were diluted to OD<SUB>600</SUB> of 0.5.
    </P><P>
        In addition, we needed to examine how the bacteria would respond to varying intensity of light, which would be the conditions for example inside a bioreactor. The biomass in the reactor would decrease the light intensity as distance to the nearest LED increases. The culture in the bioreactor would be constantly mixed and we assumed that one cycle in the bioreactor takes approximately 8 minutes. If there are 4 horizontally spaced LED rigs, travelling from LED rig to another would take 1 minute. Thus, we set 8 LEDs in our LED rig to change the intensities linearly from zero to maximum intensity in one minute cycle. Here, we need to state that the linear fit of the parameter is done in logarithmic scale as the log<SUB>10</SUB> of the parameter is directly proportional to the perceived light intensity. We hypothesize the fluorescence of the blinking light will be on the same level as the fluorescence under a constant blue light with half of the maximum intensity of the blinking light. That is, we suggest the effect to be proportional to the average intensity and that the blinking would not affect to the results.
    </P><P>
        In this experiment, we had three black microtiter plates for three different sets of samples. On each microtiter plate, we had two positive and two negative control samples in wells without a LED. In addition, we had 20 light response element samples where 8 samples were incubated under constant light, 8 incubated under blinking LEDs and 4 samples which were without a LED. Each plate had the same order of samples. We defined different conditions for each plate. One plate (called light) was incubated in 37 Celsius degrees and constant lab environment lightning. The second plate (called dark-LED) was first incubated in complete darkness under an aluminum foil cover and after one hour, it was placed in the LED rig. The third plate (called LED-dark) was placed in the LED rig in the beginning of the experiment and after one hour it was placed in darkness under an aluminum foil cover. This installation was set to examine the dynamics of the light response element. In case of dark-LED, we can observe how bacteria will respond to different intensities of light, that is, how fast they are able to repress the production and what is the half-life of GFP-LVA. And vice versa: LED-dark will show us how fast the bacteria will be able to initiate the production when they are brought to darkness. To measure this dynamics properly, we must have much more frequent measurements compared to two previous experiments. All the plates were incubated at 37 Celsius degrees and 220 rpm.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="darksteady">.</SPAN></B>: Fluorescence of our light response element at different intensities. During the first four time points the samples were in identical conditions in complete darkness. After the fourth measurement, the samples were put in eight different constant intensities of blue light. The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="darkblink">.</SPAN></B>: Fluorescence of our light response element at different intensities. During the first four time points the samples were in identical conditions in complete darkness. After the fourth measurement, the samples were put in eight different blinking intensities of blue light. The LED lights were blinking from dark to maximum intensity (see legend) with frequency of one cycle per minute. The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><P>
        The dark-LED plate was first incubated in darkness. The  and  show clearly that all the samples have very similar expression profile in the first four timepoints. After this point, the fluorescence values diverge. In both cases, the higher the intensity (darker lines) the lower the fluorescence. All fluorescence levels are significantly higher than that of the control sample. The dark sample (dotted line) conditions should be same throughout the measurement but despite this fact, the fluorescence in the 4th measurement point seems to decrease. We hypothesize that the sample cooled down to room temperature while changing the plate environment causing stress to the bacteria. If the response time is short with respect to the total length of the experiment, we could state that the dotted line represents the effect of bacterial growth, not change in gene expression.
    </P><P>
        Let's pay closer attention to . Between the 4th and 5th timepoints, the fluorescence levels diverge as a result of change in conditions, just as expected. We see again the fourfold difference between the lowest and highest fluorescence values as a result of intensity difference. What is noteworthy, it seems like the fluorescence levels reach their new levels before the 5th timepoint and stay on that level, especially in case of the darker lines (greater intensities). We do see some increase in intensity, but this is likely to be result of cell growth. This suggests that our measuring frequency was not good enough to capture the response dynamics! We cannot make reliable estimations on the average response times but we can limit the maximum response times to 25 minutes which is the time difference between 4th and 5th timepoint. The response time may even be 10 seconds and we would get very similar data.
    </P><P>
        We can now evaluate our hypothesis of the effect of the blinking blue light on our light response element. Compared to the previous figure, we see a clear shift of fluorescence values at higher intensities in the . The high intensity LEDs have now weaker impact on the fluorescence. At the first glance we could say that the difference is around 0.7 times the maximum intensity value (for example, the 3.61 blinking intensity has an effect approximately comparable to constant intensity of 2,65). Let's call this constant efficiency multiplier. However, we need to once again take into account the bias in the low values of the intensities on the logarithmic scale. We are unable to estimate the parameter for complete darkness. If all the values would be set relative to complete darkness value, the efficiency multiplier would be lower and more accurate.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="ledsteady">.</SPAN></B>: Fluorescence of our light response element at different intensities. During the first four time points the samples were in eight different constant intensities of blue light. After the fourth measurement, the samples were put in identical conditions in complete darkness. The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="ledblink">.</SPAN></B>: Fluorescence of our light response element at different intensities. During the first four time points the samples were in eight different blinking intensities of blue light. The LED lights were blinking from dark to maximum intensity (see legend) with frequency of one cycle per minute. After the fourth measurement, the samples were put in identical conditions in complete darkness. The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><P>
        The LED-dark plate was first incubated in the LED rig and after the first four timepoints, the plate was put into complete darkness. In the  we see that the fluorescence levels diverge immediately and they are separate already in the second timepoint. Higher intensity levels lead again to lower fluorescence. However, each line also has an increasing trend during the 4 first timepoints which again may be result of bacterial growth. Already in the 5th timepoint the correlation between the intensities and fluorescence has disappeared. This state is comparable to the samples incubated in environmental light . In contrast to dark-LED measurements, the fourfold difference is present only in the 2nd and 4th timepoint. This may be due to fluctuations in temperature between the measurements.
    </P><BLOCKQUOTE>
        This strengthens our hypothesis of the fluorescence depending on the average intensity.
    </BLOCKQUOTE><P>
        The fluorescence profiles in the  are very similar to that of . Whereas the fluorescence at the low intensity levels seem to remain somewhat unchanged, the high intensity curves have higher fluorescence in 2nd to 4th timepoint compared to the constant light. The separation of different fluorescence levels is not as big as in the steady light. This strengthens our former hypothesis of the fluorescence depending on the average intensity, not the highest or lowest value. The efficiency multiplier seems to be on the same scale as in the dark-LED measurements.
    </P><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="light">.</SPAN></B>: Fluorescence of our light response element in environmental light. The samples were incubated in normal lab lightning. The times indicated are times of the day, not incubation times.
            </P></DIV></DIV><P>
        The last plate was incubated in constant conditions (37 Celsius degrees and constant environmental light) throughout the experiment. Moreover, the conditions for all 20 samples were identical. The results are shown in the . Our first observation is that all the samples' fluorescence levels are very close to each other and all the samples have the same profile. It seems like only one sample has values different to the others (the sample with highest fluorescence in the end of the experiment). We also note that the fluorescence is very low in the first timepoint but already second timepoint has the same fluorescence levels as the rest of the points. This also suggest fast response to the change in conditions (here, change from the room temperature to 37 Celsius degrees right after the dilution). The fluorescence levels seems to correspond the constant blue light intensity 2.49 in .
    </P><P>
        When we examine the results from constant environmental light incubation, we see that the fluorescence level does not change significantly after the second timepoint. This contrasts to our hypothesis that cellular growth would affect the fluorescence levels. Otherwise we should able to see steady rise in fluorescence throughout time. As we see a steady growth in control sample fluorescence we can confirm that the cells are indeed growing in the experiment (assuming that the fluorescence in control sample is proportional to the cell density). The reason for such a steady fluorescence level remains to be undiscovered.
    </P><P>
        Even though we aimed to have as frequent measurements as possible, 20 minutes was the shortest possible interval in our measurements. This limit is affected by many different factors. First of all, we need to be able to maintain the incubation temperature as steady as possible. The measurements were carried out in room temperature which cools down our samples. We need to have long enough incubation period in between the measurements in order to let the cells thrive. Moreover, the measurements take some time, about 3 minutes per plate. It is reasonable to have incubation in 37 Celsius degrees for at least twice the time taken for measurements. Finally, there were other scientist using the Varioskan fluorometer which caused variability in our measurement interval.
    </P></ARTICLE><ARTICLE><H2>Discussion</H2><P>
        According to our experiments, the light response element seems to be capable of fourfold repression of GFP-LVA with blue light. This is already very good result but we need to evaluate whether this is enough for our lambda repressor system. Dodd et al. (2001) made experiments constructing two mutated versions of <EM>O</EM><SUB>R</SUB>3 operator site. The other mutant (λr1) led to CI concentration of 2.3 compared to the wild type and the other mutation (λc12) led to 0.6 times the lysogenic concentration. This difference is of similar scale as in our experiments. They compared the effect of these mutations to prophage induction by UV light. λr1 was induced at 10% efficiency compared to wild type whereas λc12 efficiency was 170%. CI protein represses lytic genes including <EM>cro</EM> under <EM>P</EM><SUB>R</SUB> promoter. If the probability of prophage induction is proportional to CI concentration (as they showed in the paper), we can hypothesize that such difference in CI concentration would lead to 17-fold repression in <EM>P</EM><SUB>R</SUB> activity. According to the  the increase in CI concentration from 0.6 WLU to 2.3 WLU (wild-type lysogenic units) would be sufficient for <EM>P</EM><SUB>RM</SUB> repression in the wild type lambda repressor. Based on these observations, we estimate fourfold repression with blue light to be sufficient range for our three channel system.
    </P><BLOCKQUOTE>
        Our light response element seems to be capable of fourfold repression of GFP-LVA with blue light.
    </BLOCKQUOTE><DIV class="row"><DIV class="img-center"><P class="kuvateksti"><B><SPAN ref="cieffect">.</SPAN></B>: <EM>P</EM><SUB>RM</SUB> promoter activity with <EM>lacZ</EM> reporter system. In this experiment,<EM>cI</EM> was fused with <EM>P</EM><SUB>lac</SUB> promoter to control CI concentration with IPTG concentration. The figure shows the effect of CI concentration in wild type lysogenic units (WLU) to <EM>P</EM><SUB>RM</SUB> promoter (Dodd et al., 2001).
            </P></DIV></DIV><P>
        As it can be clearly seen from the , <EM>P</EM><SUB>RM</SUB> promoter will be derepressed efficiently already in low concentrations. In addition, we also observed that even at very high light intensities, the measurement device for light response element shows clearly elevated fluorescence compared to negative control. This implies leakiness in the <EM>FixK<SUB>2</SUB></EM> promoter. As this promoter also controls the CI expression, we might experience significant levels of CI in high blue light intensity. This might pose a problem by repressing <EM>P</EM><SUB>R</SUB> promoter and derepressing <EM>P</EM><SUB>RM</SUB> promoter even at the highest intensity of blue light. We need to keep this possibility in mind in future experiments and possibly look for solutions to eliminate <EM>FixK<SUB>2</SUB></EM> leakiness.
    </P><P>
        In contrast to our initial expectations, the response time of our light response element was very short. This means that we can control the gene expression at <EM>FixK<SUB>2</SUB></EM> almost real time. Phosphorylation of YF1 and as well as FixJ seems to be rapid and responsive process and presumably most of the delay occur in transcription and translation of our reporter protein. The response to increase in light intensity is contributed by YF1 phosphatase activity (Diensthuber et al., 2013) but to be able observe the effect in the reporter concentration, we need to have short enough half-life. Indeed, it seems like our reporter, GFP-LVA, has short half-life. Although Andersen et al. (1998) suggest this half-life to be around 40 minutes, we observed a lot faster degradation of GFP-LVA. In the  between 4th and 5th timepoint, the fluorescence drops from 2.9 to 0.6 in 25 minutes. According to these values, the half-life of the GFP-LVA we used in our experiments, would be around 11 minutes, at maximum. Moreover, in our actual light response element <EM>gfp</EM> will be replaced by <EM>cI</EM>. The GFP half-life (26 hours, Corish et al., 1999) is approximately 2.6 times longer than that of CI (10 hours, Parsell et al., 1990). As CI is not as stable as GFP, CI-LVA may be less stable compared to GFP-LVA potentially leading to even shorter half-life. Therefore, we estimate response to be around 15 minutes at promoter level in our three channel gene switch. We hypothesize that this enables sufficient temporal control for several industrial applications.
    </P><P>
        In our experiments to address the effect of varying light intensity on our light response element, we observed that the average intensity seems to be the dominating factor for the response. However, we also noticed that the effect of blinking light is not exactly comparable to constant blue light with same average intensity. We may use an efficiency multiplier to compare the blinking blue LED (from zero to certain maximum intensity) to constant blue light. The maximum intensity of the blinking LED multiplied by the efficiency multiplier tells the corresponding constant blue light intensity. We estimated the value of this efficiency multiplier empirically to be at the scale of 0.65 to 0.7 in our experiments. As long as the blinking frequency is short enough compared to the half-life of the CI protein, the perturbations will be averaged out in the response of our system. This would enable use of the three channel gene switch in bioreactors where the intensity of the LEDs will be nonuniform. If the bacterial culture is mixed constantly in such conditions we should have a uniform response to blue light.
    </P><P>
        There are several options to optimize and improve our three channel gene switch. First of all, as Dodd et al. (2001) have shown, the operator sites can be mutated to modify the function of the lambda repressor element. We can tune the <EM>P</EM><SUB>RM</SUB> and <EM>P</EM><SUB>R</SUB> promoters to enhance or suppress the rate of transcription associated with the promoter. This is very important as the promoters cannot be changed and different users have different needs for the promoter strengths. Another option is to modify the promoters to be as strong as possible and then tune the translation by using weaker RBS sites to get the desired level of protein production.
    </P><P>
        To address the leakiness of our system, we could attempt to engineer the <EM>FixK<SUB>2</SUB></EM> promoter to eliminate the leaky activity. We could also attempt to repress the basal expression level by duplicating the operator site in front of it. The proper function of the promoter is crucial to our system and intended dynamics of the lambda repressor element. However, as we have observed in our experiments, we have not yet reached the saturating intensity of blue light. In each figure of the results section, we see that the fluorescence decreases even from the second highest to the highest intensity. We might be able to drive down the leakiness of the promoter by using even stronger LEDs. Of course, we also need to try to minimize the heat generated by these LEDs.
    </P><P>
        There are several hypotheses that are yet to be addressed and questions to remain unanswered. Despite our best efforts we have not been able to finish our construct at the moment of the wiki freeze. We have not been able to test our hypotheses regarding the lambda repressor and we have not had enough time to test the light response element in greater detail. We are still working on it and possible future results will be published on the <A href="http://2014.aaltohelsinki.com/">team website</A>.
    </P><P>
        All our experiments have been carried out in the population level which averages out the possible individual differences. As Zopf et al. (2013) have described, cell cycle affects greatly in the rate of transcription. Especially after division, transcription rates tend to be significantly lower than before the division. In proliferating cultures this may pose a problem. Our mechanism works on the cellular level but instead, we have measured the average level. It is true that we can affect to the expression of the GFP-LVA protein but how does this rate vary from cell to cell? Are we able to keep it stable enough in cell resolution to be able to reliably control protein levels to switch between genes? This issue needs to be taken seriously and it requires further analysis. Unfortunately, the single cell experiments require equipment that we did not have access to.
    </P><P>
        The lambda phage integrates its DNA to the <EM>E. coli</EM> genome upon infection. The lambda repressor works robustly in the <EM>E. coli</EM> genome, but is it able to function properly in the plasmid? We know from experience that individual parts of the lambda repressor have been used successfully in synthetic plasmids in <EM>E. coli</EM> (this can be verified by searching these parts in the BioBrick foundation's registry of standard biological parts). For example DNA looping is complex enough mechanism to be affected by the plasmid context. Supercoiled plasmids may not have enough flexibility in order to enable such DNA looping. Even if it was possible, it might require very precise placement of <EM>O</EM><SUB>R</SUB> sites with respect to each other.
    </P><P>
        Our experiments were based on the assumption that GFP-LVA would have similar dynamics as the CI-LVA would. However, this should be evaluated in future experiments. CI-LVA half-life needs to be determined by using common protein concentration measurement methods. To monitor the effect of CI in the lambda repressor context we could fuse CI with GFP-LVA and associate <EM>P</EM><SUB>RM</SUB> or <EM>P</EM><SUB>R</SUB> promoter with a complementary reporter, such as <EM>lacZ</EM>. This would enable us to observe the effect of CI concentration. Another possibility is to use similar setting as Dodd et al. (2001) and produce CI from a separate plasmid using lac promoter induced by IPTG. This enables us to monitor the effect of high CI concentration and evaluate, whether DNA looping takes place or not.
    </P><P>
        As discussed multiple times in the results section, we need to identify how much proliferation accounts for the rise in fluorescence in our measurements. We should include OD<SUB>600</SUB> measurements to our future experiments and have proper controls to evaluate, to what extent the fluorescence differences are caused by blue light intensity and what is effect of cell proliferation on these measurements. Moreover, carrying out the experiments with 5 minute measurement interval in a room with 37 Celsius degrees would allow us to examine better the dynamics without affecting the growing conditions.
    </P><P>
        The experiments done for the blue light response element are very similar to those required for testing the entire three channel gene switch system. We would use LED rig to control the blue light intensity and would measure the effect of three different reporter systems. If feasible, these reporters could be three different fluorescent proteins whose excitation spectrum overlaps as little as possible. Alternatively, a LacZ reporter system could be used. The response of each different reporter system would be measured regularly during a several hour experiment. A second experiment would measure the dynamics of the system and evaluate, how fast and reliably it is possible to switch between the active genes.
    </P><P>
        Lastly, as one may note in our <A href="https://2014.igem.org/Team:Aalto-Helsinki/Modeling#Simulation">simulation</A>, we have also planned to include a red light sensing element to our system. A good candidate for the light receptor would be engineered version of Cph8. The role of the red light would be regulating the rate of transcription of the the three target genes whereas the blue light defines the gene being transcribed. The purpose is to have control over both the rate of transcription and the gene being active. The red light response could be mediated by CRISPR/dCas9 system by associating the complex with light activated Krüppel-associated Box (KRAB) as described by Freiburg 2013 team. The system would identify a sequence associated with each target gene. Therefore, we could regulate the rate of transcription at each three genes using red light. However, the exact mechanism of such a system is yet to be designed and tested.
    </P></ARTICLE><ARTICLE><H2>Conclusions</H2><P>
      Our experiments show that our light response element works very well. Based on the observations we can conclude that the system is capable of fourfold repression with blue light at 470 nm wavelength and its response time is likely to be less than 20 minutes. In addition, the system is resistant to perturbations in light intensity which makes it robust and applicable to conditions in a bioreactor, including non uniform light intensity. These are very good results and they form a good basis for our three channel switch. In addition, we were able to build an excellent device to control the light intensities during the incubations. However, as a consequence of not being able to ligate our lambda repressor element together before wiki freeze, the gene switch system at its entire form remains to be evaluated at a later stage.
  </P><BLOCKQUOTE>
      The system is resistant to perturbations in light intensity which makes it robust and applicable to conditions in a bioreactor.
  </BLOCKQUOTE></ARTICLE><ARTICLE><H2>References</H2><UL id="references"><LI>
            Andersen, J. B., Sternberg, C., Poulsen, L. K., Bjorn, S. P., Givskov, M., &amp; Molin, S. (1998). <EM>New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria.</EM> Applied and Environmental Microbiology, 64(6), 2240–6. Retrieved from http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=106306&amp;tool=pmcentrez&amp;rendertype=abstract
        </LI><LI>
            Ashikawa, Y., Ihara, M., Matsuura, N., Fukunaga, Y., Kusakabe, Y., &amp; Yamashita, A. (2011). <EM>GFP-based evaluation system of recombinant expression through the secretory pathway in insect cells and its application to the extracellular domains of class C GPCRs.</EM> Protein Science : A Publication of the Protein Society, 20(10), 1720–34. doi:10.1002/pro.707
        </LI><LI>
            Bell, C. E., Frescura, P., Hochschild, A., &amp; Lewis, M. (2000). <EM>Crystal Structure of the λ Repressor C-Terminal Domain Provides a Model for Cooperative Operator Binding.</EM> Cell, 101(7), 801–811. doi:10.1016/S0092-8674(00)80891-0
        </LI><LI>Corish, P., &amp; Tyler-Smith, C. (1999). <EM>Attenuation of green fluorescent protein half-life in mammalian cells.</EM> Protein Engineering, 12(12), 1035–40. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/10611396
        </LI><LI>
            Cormack, B. P., Valdivia, R. H., &amp; Falkow, S. (1996). <EM>FACS-optimized mutants of the green fluorescent protein (GFP).</EM> Gene, 173(1), 33–8. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8707053
        </LI><LI>
            Diensthuber, R. P., Bommer, M., Gleichmann, T., &amp; Möglich, A. (2013). <EM>Full-length structure of a sensor histidine kinase pinpoints coaxial coiled coils as signal transducers and modulators.</EM> Structure (London, England : 1993), 21(7), 1127–36. doi:10.1016/j.str.2013.04.024
        </LI><LI>
            Dodd, I. B., Perkins, A. J., Tsemitsidis, D., &amp; Egan, J. B. (2001). OcDodd, I. B., Perkins, A. J., Tsemitsidis, D., &amp; Egan, J. B. (2001). <EM>Octamerization of lambda CI repressor is needed for effective repression of P(RM) and efficient switching from lysogeny.</EM> Genes &amp; Development, 15(22), 3013–22. doi:10.1101/gad.937301tame. Genes &amp; Development, 15(22), 3013–22. doi:10.1101/gad.937301
        </LI><LI>
            Hirose, Y., Shimada, T., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2008). <EM>Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein</EM>. Proceedings of the National Academy of Sciences of the United States of America, 105(28), 9528–33. doi:10.1073/pnas.0801826105
        </LI><LI>
            Johnson, A. D., Meyer, B. J., &amp; Ptashne, M. (1979). <EM>Interactions between DNA-bound repressors govern regulation by the lambda phage repressor.</EM> Proceedings of the National Academy of Sciences of the United States of America, 76(10), 5061–5. Retrieved from http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=413079&amp;tool=pmcentrez&amp;rendertype=abstract
        </LI><LI>
            Johnson, A. D., Poteete, A. R., Lauer, G., Sauer, R. T., Ackers, G. K., &amp; Ptashne, M. (1981). <EM>λ Repressor and cro—components of an efficient molecular switch.</EM> Nature, 294(5838), 217–223. doi:10.1038/294217a0
        </LI><LI>
            Keiler, K. C., Waller, P. R., &amp; Sauer, R. T. (1996). <EM>Role of a peptide tagging system in degradation of proteins synthesized from damaged messenger RNA.</EM> Science (New York, N.Y.), 271(5251), 990–3. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8584937
        </LI><LI>
            Levskaya, A., Chevalier, A. A., Tabor, J. J., Simpson, Z. B., Lavery, L. A., Levy, M., … Voigt, C. A. (2005). <EM>Synthetic biology: engineering Escherichia coli to see light</EM>. Nature, 438(7067), 441–2. doi:10.1038/nature04405
        </LI><LI>
            Manzo, C., Zurla, C., Dunlap, D. D., &amp; Finzi, L. (2012). The effect of nonspecific binding of lambda repressor on DNA looping dynamics. Biophysical Journal, 103(8), 1753–61. doi:10.1016/j.bpj.2012.09.006
        </LI><LI>
            Möglich, A., Ayers, R. A., &amp; Moffat, K. (2009). <EM>Design and signaling mechanism of light-regulated histidine kinases</EM>. Journal of Molecular Biology, 385(5), 1433–44. doi:10.1016/j.jmb.2008.12.017
        </LI><LI>
            Olson, E. J., Hartsough, L. A., Landry, B. P., Shroff, R., &amp; Tabor, J. J. (2014). <EM>Characterizing bacterial gene circuit dynamics with optically programmed gene expression signals.</EM> Nature Methods, 11(4), 449–55. doi:10.1038/nmeth.2884
        </LI><LI>
            Parsell, D. A., Silber, K. R., &amp; Sauer, R. T. (1990). <EM>Carboxy-terminal determinants of intracellular protein degradation.</EM> Genes &amp; Development, 4(2), 277–286. doi:10.1101/gad.4.2.277
        </LI><LI>
            Tabor, J. J., Levskaya, A., &amp; Voigt, C. A. (2011). <EM>Multichromatic control of gene expression in Escherichia coli.</EM> Journal of Molecular Biology, 405(2), 315–24. doi:10.1016/j.jmb.2010.10.038
        </LI><LI>
            Zopf, C. J., Quinn, K., Zeidman, J., &amp; Maheshri, N. (2013). <EM>Cell-cycle dependence of transcription dominates noise in gene expression</EM>. PLoS Computational Biology, 9(7), e1003161. doi:10.1371/journal.pcbi.1003161
        </LI><LI>
            Schmidl S., Sheth R. U., Wu S., Tabor J.J. (2014). <EM>Refactoring and Optimization of Light-Switchable Escherichia coli Two-Component Systems.</EM> ACS Synthetic Biology Web. DOI: 10.102/sb500273n
        </LI><LI>
            Muthu S., Schuurmans F. and Pashley M. (2002). <EM>Red, Green and Blue LED based white light generation: Issues and Control</EM> 37th Annual IEEEIAS meeting, Vol. 1, pp. 327 – 333 (2002). DOI: 10.1109/IAS.2002.1044108
        </LI></UL><H3>References to iGEM teams:</H3><UL><LI>
            Peking iGEM Team (2012), <A href="https://2012.igem.org/Team:Peking">https://2012.igem.org/Team:Peking</A></LI><LI>
            Uppsala-Sweden iGEM Team (2011), <A href="https://2011.igem.org/Team:Uppsala-Sweden">https://2011.igem.org/Team:Uppsala-Sweden</A></LI><LI>
            SJTU BioX Shanghai iGEM Team, (2013), <A href="https://2013.igem.org/Team:SJTU-BioX-Shanghai">https://2013.igem.org/Team:SJTU-BioX-Shanghai</A></LI><LI>
            Freiburg iGEM Team, (2013), <A href="https://2013.igem.org/Team:Freiburg">https://2013.igem.org/Team:Freiburg</A></LI></UL></ARTICLE></DIV></DIV><FOOTER><DIV id="yhteystiedot"><P>
      This wiki is designed and constructed by Aalto-Helsinki.<A href="https://static.igem.org/mediawiki/2014/7/73/Aalto_Helsinki_Wiki_Sitemap.png" target="_blank">Sitemap</A> | <A href="http://twitter.com/AaltoHelsinki">Twitter</A> | <A href="http://www.facebook.com/AaltoHelsinki">Facebook</A> | <A href="mailto:oskari.vinko@gmail.com" target="_top">Email</A></P></DIV></FOOTER><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Aalto-Helsinki/Research">http://2014.igem.org/Team:Aalto-Helsinki/Research</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aalto-Helsinki/Research" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aalto-Helsinki/Research" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aalto-Helsinki/Research&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aalto-Helsinki/Research&amp;oldid=406271" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>