<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Model_FBA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Model/FBA</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page_wrap"><HEADER><DIV id="logo_evryigem"><A href="https://2016.igem.org/Team:Evry" title="Homepage"><SPAN>Homepage</SPAN></A></DIV><DIV id="bar-1"><DIV id="igem-logo"><A href="https://igem.org" title="Visit the iGEM website">Visit the iGEM website!</A></DIV><DIV id="social"><P>Feel free to contact us! </P><UL><LI><A class="email" href="mailto:evryigem2016@gmail.com?Subject=Wiki " title="E-mail - evryigem2016@gmail.com">E-mail</A></LI><LI><A class="twitter" href="https://twitter.com/iGEM_Evry" title="Twitter - @iGEM_evry">Twitter</A></LI><LI><A class="facebook" href="https://www.facebook.com/iGEM.Evry.2016/" title="Facebook - /iGEM.Evry.2016/">Facebook</A></LI><LI><A class="insta" href="https://www.instagram.com/igem_evry/" title="Instagram - @igem_evry">Instagram</A></LI></UL></DIV><DIV id="letsplay"><EM>Let's PLAy project</EM> - Bioproduction of PLA</DIV></DIV><NAV id="navbar"><DIV id="nav"><UL id="nav_menu"><LI class="cat_menu show_more"><A href="#">Show menu</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI class="cat_menu dropdown xproj"><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Improvements">Improvements</A></LI></UL><LI class="cat_menu dropdown xachiv"><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Medals">Medal fulfillments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Results">Results</A></LI><LI class="cat_menu dropdown xilab"><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab">Results</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Labbook">Lab book</A></LI><LI class="cat_menu dropdown xsafe"><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety">Project safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Lab">Lab safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Shipment">Shipment safety</A></LI><LI class="cat_menu dropdown xhp"><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Integrated_Practices"> Integrated HP</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Engagement">Engagements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/HP/Events">iGEM events</A></LI><LI class="cat_menu dropdown xteam"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Previous_Evry">iGEM in Evry</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Campus">Campus</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Sponsors">Sponsors</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></DIV></NAV></HEADER><UL id="show_more"><LI><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI class="lastl"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></UL><MAIN><DIV id="content-wrap"><DIV id="content"><DIV id="breadcrumbs"><UL><LI><A href="https://2016.igem.org/Team:Evry">Homepage</A></LI><LI><A href="">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model">Modeling</A></LI><LI>Flux Balance Analysis</LI></UL></DIV><H1 class="red">Flux Balance Analysis</H1><P><UL id="project_menu" class="pm_modeling"><LI class="current_page"><A href="https://2016.igem.org/Team:Evry/Model/FBA"><SPAN>Flux Balance Analysis</SPAN></A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model/Dynamic"><SPAN>Dynamic Regulation</SPAN></A></LI></UL></P><DIV class="grid-2"><DIV><P>Genome-scale Metabolic Network Models (GMMs) are the most promising tools taking into account stoichiometric whole cell metabolism used to study, optimize and manipulate the cell metabolism.  For this purpose, the  main approach is  Flux Balance Analysis (FBA) <A href="#ref1">[1]</A>.</P><P>In Metabolic Engineering, GMMs can be applied for analyzing and manipulating the flux distribution in order to optimize the yield of the desired product. Therefore, we used FBA to study the production of PLA in a synthetic pathway consisting of 2 exogenous enzymes.</P><P>The first step was to find the SBML file of the GMM corresponding to our chassis, <I>Pseudomonas putida</I> KT2440, which has been reconstructed before <A href="#ref2">[2]</A>. Then, we implemented our pathway with its enzymes, metabolites and reactions into the model. This constituted an in silico PLA producing <I>P. putida</I> (Table 1). The last and foremost step was to challenge our model with several variables such as different carbon sources and objective functions. The optimization process was done by FBA algorithm using the OptFlux toolbox <A href="#ref3">[3]</A>. </P><P class="center"><TABLE class="center"><TBODY><TR><TH>Number of genes</TH><TH>Number of reactions</TH><TH>Number of metabolites</TH></TR><TR><TD>962</TD><TD>980</TD><TD>899</TD></TR></TBODY></TABLE></P><P class="figure"><U>Table 1.</U> Number of parts in the final PLA-producing <I>P. putida</I> KT2440 metabolic network.</P><P>In our project, we tested glucose (<A href="https://static.igem.org/mediawiki/2016/6/61/T--Evry--Pputida_Glucose.txt" target="_blank">SBML file</A>) and fructose (<A href="https://static.igem.org/mediawiki/2016/d/de/T--Evry--Pputida_Fructose.txt" target="_blank">SBML file</A>) as substrate with two objective functions, that are:</P><UL><LI>PLA producing reaction</LI><LI>Biomass, which is a hypothetical reaction in which precursors of cellular biomass are reactants and biomass is the product. The biomass flux is identical  with cell growth rate</LI></UL></DIV><DIV><P>Moreover, since FBA is a biased approach to optimize the objective function (PLA or Biomass), we implemented PLA as a precursor of biomass, to obtain a more realistic view on cell growth and PLA production simultaneously.  </P><P>The first experiment shows both independent FBA on biomass (blue fluxes) and PLA producing reaction (red fluxes) as objective functions, using glucose as sole carbon source (Figure 1). As mentioned, due to the biased optimization of FBA, the yield of PLA production equals zero  when the biomass is maximized, and vice versa. Besides, comparison of flux distribution in the central metabolism of these two independent FBA demonstrates that the main bottleneck of PLA production locates where pyruvate transforms into lactate, which is a fermentation reaction. </P><P class="center"><A href="https://static.igem.org/mediawiki/2016/3/35/T--Evry--modeling--glu.png" class="fancybox" title="Figure 1"><SPAN> Click to enlarge</SPAN></A></P><P class="figure"><U>Figure 1.</U> Schematic representation of the central metabolism with PLA pathway. In this flux distribution, Glucose is the carbon source. The Blue flux values are associated with Biomass optimized FBA and the red flux values are associated with PLA FBA. These two FBAs were done in independent experiments.</P></DIV></DIV><DIV class="grid-2"><DIV><P>Not surprisingly, in “real life” cellular conditions, in which cells try to grow and duplicate as much as they can, the flux of the fermentation pathway is approximately zero. As a result, the main challenge in PLA production is the first out of 3 main reactions of the PLA pathway, in which lactate is supplied from the central metabolism. Thus, we first focused on finding means to overcome this challenging bottleneck. </P><P>In this direction, we discovered a mutant of LDH enzyme (lactate dehydrogenase) in the literature, which utilizes both NADH and NADPH efficiently <A href="#ref4">[4]</A>. More importantly, this mutant produces the enantiomer of lactate, which is used by the second enzyme of the PLA pathway. This novel affinity of the enzyme to NADPH gives it access to a higher quantity of substrate (since the natural ratio of NADPH/NADP+ is much higher than NADH/NAD+), which reduces the main bottleneck of the pathway, as we described before.
Comparing the flux distribution in these two conditions (biomass and PLA optimization) led us to discover strategies to increase the yield of PLA (Figure 1). For instance, the oxygen uptake flux for PLA production is 6-fold less than biomass FBA. This shows that low levels of oxygen are sufficient for PLA production, since lower oxygen levels  cause less biomass, which lead to carbon being turned via fermentation into lactate, and finally  to PLA.</P><P class="center"><A href="https://static.igem.org/mediawiki/2016/9/91/T--Evry--modeling--fru.png" class="fancybox" title="Figure 2"><SPAN> Click to enlarge</SPAN></A></P><P class="figure"><U>Figure 2.</U> Schematic representation of the central metabolism with PLA pathway. In this flux distribution, Fructose is the carbon source. The Blue flux values are associated with Biomass optimized FBA and the red flux values are associated with PLA FBA. These two FBAs were done in independent experiments.</P><P>However, due to the necessity of cellular biomass as the cell factory, we thought that the best solution was to design a two step fermentation:</P><UL><LI>First, we increase the oxygen level with high aeration in order to increase the biomass</LI><LI>Then, we use microaerobic conditions to redirect the most of the carbon and energy into production of PLA</LI></UL><P>The second experiment is almost the same as the previous one, except we used fructose as the sole carbon source, instead of glucose. As illustrated in the Figure 2, flux distributions and flux values are different than pictured in Figure 1. More importantly, both the biomass and PLA fluxes are increased compared to the previous experiment. 
Thus, we have been informed through FBA that fructose is a more suited substrate to promote both growth and PLA production.</P></DIV><DIV><P>Finally, in the last experiment, we tried to  integrate both Biomass and PLA into one main objective function, which means that when the exogenous pathway is implemented to  the cell, the PLA will be part of the cellular biomass. Therefore, the PLA was included into the biomass as one of its precursors (<A href="https://static.igem.org/mediawiki/2016/8/88/T--Evry--Pputida_PLABiomass.txt">SBML file</A>). The stoichiometric ratio of PLA with regards to the whole biomass was put the same as other polymers of <I>P. putida</I> KT2440 GMM.  This integration enables further investigating of the pathway and finding the mutants with higher yield, and provides more realistic perspectives on the whole cell metabolism, emphasizing on PLA production while keeping economic growth rate (Figure 3).</P><P class="center"><A href="https://static.igem.org/mediawiki/2016/3/30/T--Evry--fba-3-biopla.png" class="fancybox" title="Figure 3"><SPAN> Click to enlarge</SPAN></A></P><P class="figure"><U>Figure 3.</U>Schematic representation of the central metabolism with implementation of the PLA pathway. In this flux distribution, Glucose is the carbon source. Here PLA has been set as a precursor of biomass. .</P><H2>Perspective</H2><P>GMM analysis is a promising tool to study and manipulate cells <I>in-silico</I>. Here, we presented some  applications of our modified <I>P. putida</I> KT2440 metabolic network.</P><P>To take this theory into practice, we need to tune cellular metabolism through genetic engineering, enzyme engineering and biochemical optimization of growth conditions. Moreover, since cells are dedicated to growth and not satisfy our synthetic phenotype, implementing exogenous enzymes is not sufficient to produce fine chemicals. </P><P>Hence, using synthetic biology tools, we have to tune number of gene copies, number of mRNAs and proteins inside the cell, as well as controlling the pathway’s components with gene expression network and protein-protein interactions network. All of these could allow us to reach higher flux distribution into the producing pathway and to let it balance itself with the cell growth. </P><P>In other words, dynamic regulation makes the pathway tight to the cellular components in a natural way. To do this, we have designed a genetic-metabolic circuit regulating the pathway dynamically in a way that the cell finds its own equilibrium to make PLA functionally (see <A href="https://2016.igem.org/Team:Evry/Model/Dynamic">dynamic regulation part</A>).</P></DIV></DIV><DIV id="references"><H2 class="red">References</H2><OL><LI>Schellenberger, J. et al. Quantitative prediction of cellular metabolism with constraint-based models: the COBRA Toolbox v2.0. Nat Protoc 6, 1290–1307 (2011).</LI><LI>Nogales, J., Palsson, B. Ø. &amp; Thiele, I. A genome-scale metabolic reconstruction of <I>Pseudomonas putida</I> KT2440: iJN746 as a cell factory. BMC Syst Biol 2, 79 (2008).</LI><LI>Rocha, I. et al. OptFlux: an open-source software platform for in silico metabolic engineering. BMC Syst Biol 4, 45 (2010).</LI><LI>Meng, H. et al. Engineering a d-lactate dehydrogenase that can super-efficiently utilize NADPH and NADH as cofactors. Scientific Reports 6, 24887 (2016).</LI></OL></DIV></DIV></DIV></MAIN></DIV><FOOTER><DIV id="footer_content"><DIV id="sponsors"><UL><LI><A target="_black" class="genopole" href="http://www.genopole.fr/?lang=en" title="Genopole">Genopole</A></LI><LI class="wide"><A target="_black" class="idt" href="http://eu.idtdna.com/site" title="Integrated DNA technologies">IDT</A></LI><LI><A target="_black" class="issb" href="http://www.issb.genopole.fr/" title="Institute of Systems &amp; Systhetic biology">iSSB</A></LI><LI><A target="_black" class="ueve" href="http://www.univ-evry.fr/fr/index.html" title="University of Evry-Val-d'Essonne">UEVE</A></LI><LI><A target="_black" class="ibisc" href="https://www.ibisc.univ-evry.fr/en/start" title="Informatique, Biologie Intégrative et Systèmes Complexes">IBISC</A></LI><LI><A target="_black" class="rf" href="http://fr.franceintheus.org/" title="French Ambassy in the United-States">République Française</A></LI><LI><A target="_black" class="crous" href="http://www.crous-paris.fr/" title="Crous">Crous</A></LI><LI class="wide"><A target="_black" class="neb" href="https://www.neb.com/" title="New-England Biolabs">New-England Biolab</A></LI><LI class="wide"><A target="_black" class="geneious" href="http://www.geneious.com/" title="Geneious">Geneinous</A></LI><LI class="wide"><A target="_black" class="mathworks" href="http://mathworks.com/" title="Mathworks">Mathworks</A></LI><LI><A target="_black" class="mie" href="http://www.paris.fr/services-et-infos-pratiques/famille-et-education/etudier-a-paris/maison-des-initiatives-etudiantes-2521" title="Maison des initiatives étudiantes (House of Student initiatives)">MIE</A></LI><LI><A target="_black" class="papet" href="http://www.grandparissud.fr/" title="Grand Paris Sud">Papeterie GPS</A></LI></UL></DIV><SPAN id="pvl"><STRONG>Evry</STRONG>2016.</SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>