<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Design_PLAsystem skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Design/PLAsystem</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page_wrap"><HEADER><DIV id="logo_evryigem"><A href="https://2016.igem.org/Team:Evry" title="Homepage"><SPAN>Homepage</SPAN></A></DIV><DIV id="bar-1"><DIV id="igem-logo"><A href="https://igem.org" title="Visit the iGEM website">Visit the iGEM website!</A></DIV><DIV id="social"><P>Feel free to contact us! </P><UL><LI><A class="email" href="mailto:evryigem2016@gmail.com?Subject=Wiki " title="E-mail - evryigem2016@gmail.com">E-mail</A></LI><LI><A class="twitter" href="https://twitter.com/iGEM_Evry" title="Twitter - @iGEM_evry">Twitter</A></LI><LI><A class="facebook" href="https://www.facebook.com/iGEM.Evry.2016/" title="Facebook - /iGEM.Evry.2016/">Facebook</A></LI><LI><A class="insta" href="https://www.instagram.com/igem_evry/" title="Instagram - @igem_evry">Instagram</A></LI></UL></DIV><DIV id="letsplay"><EM>Let's PLAy project</EM> - Bioproduction of PLA</DIV></DIV><NAV id="navbar"><DIV id="nav"><UL id="nav_menu"><LI class="cat_menu show_more"><A href="#">Show menu</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI class="cat_menu dropdown xproj"><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Improvements">Improvements</A></LI></UL><LI class="cat_menu dropdown xachiv"><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Medals">Medal fulfillments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Results">Results</A></LI><LI class="cat_menu dropdown xilab"><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab">Results</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Labbook">Lab book</A></LI><LI class="cat_menu dropdown xsafe"><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety">Project safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Lab">Lab safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Shipment">Shipment safety</A></LI><LI class="cat_menu dropdown xhp"><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Integrated_Practices"> Integrated HP</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Engagement">Engagements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/HP/Events">iGEM events</A></LI><LI class="cat_menu dropdown xteam"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Previous_Evry">iGEM in Evry</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Campus">Campus</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Sponsors">Sponsors</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></DIV></NAV></HEADER><UL id="show_more"><LI><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI class="lastl"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></UL><MAIN><DIV id="content-wrap"><DIV id="content"><DIV id="breadcrumbs"><UL><LI><A href="https://2016.igem.org/Team:Evry">Homepage</A></LI><LI><A href="">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Design">Design</A></LI><LI>The PLA system</LI></UL></DIV><H1 class="red">The PLA system</H1><P>Starting from the simple idea of producing a biological alternative to regular plastic, we had to think about how to engineer bacteria to reach this goal in the most efficient way. As explained in <A href="https://2016.igem.org/Team:Evry/Design">Design</A>, we first carefully chose our chassis; we proceeded with a basic analysis of the pathways to produce PLA; and we designed the implementation of the necessary genes. </P><H2> Choosing a chassis </H2><DIV class="grid-2"><DIV><P> In literature, bioproduction of PLA had already been reported in <I>E. coli</I>. However, we detected several criteria that would set other organisms as better chassis for PLA synthesis, and thus accomplish our purpose: improve PLA bioproduction. </P></DIV><DIV><P>Over different potential candidates, we set  <I>P. putida </I>KT2440 as the best chassis, as argumented in <A href="https://2016.igem.org/Team:Evry/Description/Chassis">Project Description</A>. </P></DIV></DIV><H2> Basic pathway analysis </H2><DIV class="grid-2"><DIV><P> Metabolism of <I>P. putida </I>KT2440 has been well described in literature <A href="#ref1">[1]</A>. Taking into account the reaction steps indicated in Jung et al., 2010 <A href="#ref2">[2]</A> for bioproduction of PLA in <I>E. coli</I>, we studied <I>P. putida </I>KT2440 pathways and clarified which were the minimal genes to express to have PLA. The schema in Figure 1 indicates the reactions that we would need to introduce in our chassis. </P><P>From this, we proceeded to think about the design of our biobricks, which would contain the PLA involved genes.</P></DIV><DIV><P class="figure"><U>Figure 1.</U> Engineered Pct and PhaC. Propionate Co-A transferase and PHA synthase are the two genes needed for PLA production from lactic acid.</P></DIV></DIV><H5>Biobrick design</H5><DIV class="grid-2"><DIV><P>Clearly, we needed two genes for PLA biosynthesis: an evolved Propionate CoA transferase who can take lactate as subtract and convert it into Lactyl-CoA, and an evolved PHA synthase which can polymerize monomer of Lactyl-CoA into PLA polymer. To do so, we started by looking for the sequence of <I>Clostridium propionicum</I> propionate-CoA transferase and we codon optimized the sequence for our bacteria <I>P. putida</I>. The wild type enzyme normally catalyzes the reaction: acetyl-CoA + propanoate &lt;--&gt; acetate + propanoyl-CoA. We introduced one amino acid mutation (A243T) found to allow efficiently convert lactate into lactyl-CoA <A href="#ref3">[3]</A>.</P><P>So, we used the same method for PHA synthase, we found two potential enzymes from different organisms. The first one was used in the iGEM Yale 2013 project (<I>Pseudomonas resinovorans</I> PHA synthase 1) and the second is <I>Pseudomonas sp.</I> MBEL 6-19 PHA synthase 1. We optimized their sequence for <I>P. putida</I> and performed quadruple important amino acid mutation (E130D,S325T, S477F, and Q481K) found for altering the substrate specificity and having enhanced activity towards (D)-lactyl-CoA <A href="#ref3">[3]</A>.</P></DIV><DIV><P>Finally, to increase the production of our compounds, we considered increasing the initial subtract as an important step. We decided to implement an evolved D-lactate dehydrogenase which could be useful for our system. We based the design on an article reporting a engineering NADH-dependent dehydrogenase (D-LDH*) from <I>Bacillus delbrueckii </I>11842 that can super-efficiently utilize NADPH and NADH as cofactors. We looked for the sequence, we codon optimized it for <I>P. putida</I> and did the tree point amino acids mutation (D176S, I177R, F178T) <A href="#ref4">[4]</A>.</P></DIV></DIV><H2> Implementation of the basic system </H2><DIV class="grid-2"><DIV><P>Once our basic sequences were designed, we had to think about how to express them in the cell. For that, we started with the design of our plasmid. In order to choose the best plasmid for <I>P. putida</I>, we asked for advice to <A href="http://www.cnb.csic.es/index.php/es/investigacion/departamentos-de-investigacion/biologia-de-sistemas/microbiologia-medioambiental-molecular">VÃ­ctor De Lorenzo lab (CNB-CSIC)</A>, which are working in <I>Pseudomonas putida</I> KT2440. He recommended the use of two SEVA (Standard European Vector Architecture) plasmids <A href="#ref5">[5]</A>, which were: </P><UL><LI>pSEVA224: IPTG inducible promoter and Kanamycin resistant gene</LI><LI>pSEVA2311: Cyclohexanone inducible promoter and Kanamycin resistant gene</LI></UL></DIV><DIV><P>An efficiently way of building our plasmid was indeed making an operon with the two genes (PhaC and Pct). According to their length we had to synthetize them separately with our IDT gBlocks. In the synthesis, we also included RBS from <I>Pseudomonas putida</I> genomic DNA before and close to the genes to prevent missing initial codon during translation. Finally, we added biobrick prefix and suffix in the synthesis. For the next steps, we worked in parallel with the two versions of the PhaC gene that we had.</P></DIV></DIV><H4>Standard assembly method</H4><DIV class="grid-2"><DIV><P>Thinking on the design of the operon, we decided to put the PhaC gene before the Pct, as the polymerization is the bottleneck of our system and we assumed that proximity to the promoter was more necessary than for Pct, as it could provide more expression.</P><P>We wanted to put this operon in pSEVA224 plasmid in order to express it in <I>P. putida</I>. Figure 3 is a map of the region between promoter and terminator in pSEVA224.</P><P>We included the operon in this region by digesting the plasmid with Ecor1 and Pst1. In the final construct, the operon is under control of the inducible promoter of pSEVA224. Figure 4 is the final plasmid map containing the operon.</P></DIV><DIV><P class="figure"><U>Figure 2.</U> Standard assembly for construction of operon. We would use restriction enzymes to construct our PhaC-Pct operon from the separate fragments on gBlocks. </P></DIV></DIV><P class="figure"><U>Figure 3.</U> pSEVA 224: region between promoter and terminator. </P><P class="figure"><U>Figure 4.</U> Designed plasmid. It is the pSEVA 224 vector, with the PhaC-Pct operon inserted by standard digestion assembly. </P><DIV class="grid-2"><DIV><P>For the D-LDH* gene, we aimed to express it in the same cell to increase the yield of PLA. The constructed plasmid pSEVA 224 with our operon was &gt;8kb, for what we decided that it was more convenient to use a different plasmid to express LDH. The first strategy was to use pSEVA2311 plasmid and take profit of playing with two different inducers to control when we wanted more precursor (lactate). To do so, we would change the antibiotic resistance gene, so we could co-transform <I>Pseudomonas </I>and select cells by using the two different antibiotics.</P><P>For the synthesis of this gene, we included a constitutive promoter (pTN8-P1) from <I>P. putida</I> genomic DNA in upstream of the sequence. Indeed, separated from the translation unit with BamH1 restriction site, this design allowed us the possibility to choose between expressing the gene under control of the SEVA promoter or using a constitutive one.</P></DIV><DIV><P>In parallel, we ask also to De Lorenzo lab another plasmid, the pSEVA424 which is a plasmid with the save IPTG inducible promoter as pSEVA224 but with a Spectinomycin resistant gene. Indeed, using a single inductor instead of two could be better for the cell in terms of avoiding toxicity. This would allow us to select cell transformed with both plasmid using Kanamycin and Spectinomycin antibiotic. Figure 6 is the plasmid map of the LDH* in pSEVA424.</P><P class="figure"><U>Figure 5.</U> Design of D-LDH* gene block. </P></DIV></DIV><P class="figure"><U>Figure 6.</U> Designed plasmid for LDH*. It is the pSEVA424 vector with the LDH* designed block inserted by standard digestion assembly. </P><DIV class="grid-2"><DIV><P>Finally, we would have <I>Pseudomonas putida</I> co-transformed with two plasmids: one would allow to take lactate as the substrate and convert it efficiently into lactyl-CoA, and then polymerize the monomer of lactyl-coA into PLA homopolymer; the other one would allow to increase lactate production in the cell and, in consequence, the PLA yield.</P></DIV><DIV><P>To go further, we extended the pathway analysis by performing computational <A href="https://2016.igem.org/Team:Evry/Model/FBA">Flux Balance Analysis</A>, and we also designed a <FONT color="blue"><A href="https://2016.igem.org/Team:Evry/Model/Dynamic">dynamic regulation system</A></FONT> to optimize the PLA production, and studied it <I>in silico</I>.</P></DIV></DIV><DIV id="references"><H2 class="red">References</H2><OL><LI>Nikel, P.I. et al. Biotechnological domestication of pseudomonads using synthetic biology. Nature Reviews Microbiology 12, 368â379 (2014)</LI><LI>Jung, Y.K. et al. Tailor-made type II Pseudomonas PHA synthases and their use for the biosynthesis of polylactic acid and its copolymer in recombinant <I>Escherichia coli</I>. Appl Microbiol Biotechnol 90(2):603-14 (2011).</LI><LI>Yang, T. H. et al. Biosynthesis of Polylactic Acid and Its Copolymers
Using Evolved Propionate CoA Transferase and PHA Synthase. Biotechnol Bioeng. 1;105(1):150-60 (2010)
</LI><LI>Meng, H. et al. Engineering a D-lactate dehydrogenase that can super-efficiently utilize NADPH and NADH as cofactors Sci Rep 6: 24887( 2016)</LI><LI>Silva-Rocha, E. et al. The Standard European Vector Architecture (SEVA): a coherent platform for the analysis and deployment of complex prokaryotic phenotypes. Nucleic Acids Res. 41, D666âD675 (2013)</LI></OL></DIV></DIV></DIV></MAIN></DIV><FOOTER><DIV id="footer_content"><DIV id="sponsors"><UL><LI><A target="_black" class="genopole" href="http://www.genopole.fr/?lang=en" title="Genopole">Genopole</A></LI><LI class="wide"><A target="_black" class="idt" href="http://eu.idtdna.com/site" title="Integrated DNA technologies">IDT</A></LI><LI><A target="_black" class="issb" href="http://www.issb.genopole.fr/" title="Institute of Systems &amp; Systhetic biology">iSSB</A></LI><LI><A target="_black" class="ueve" href="http://www.univ-evry.fr/fr/index.html" title="University of Evry-Val-d'Essonne">UEVE</A></LI><LI><A target="_black" class="ibisc" href="https://www.ibisc.univ-evry.fr/en/start" title="Informatique, Biologie IntÃ©grative et SystÃ¨mes Complexes">IBISC</A></LI><LI><A target="_black" class="rf" href="http://fr.franceintheus.org/" title="French Ambassy in the United-States">RÃ©publique FranÃ§aise</A></LI><LI><A target="_black" class="crous" href="http://www.crous-paris.fr/" title="Crous">Crous</A></LI><LI class="wide"><A target="_black" class="neb" href="https://www.neb.com/" title="New-England Biolabs">New-England Biolab</A></LI><LI class="wide"><A target="_black" class="geneious" href="http://www.geneious.com/" title="Geneious">Geneinous</A></LI><LI class="wide"><A target="_black" class="mathworks" href="http://mathworks.com/" title="Mathworks">Mathworks</A></LI><LI><A target="_black" class="mie" href="http://www.paris.fr/services-et-infos-pratiques/famille-et-education/etudier-a-paris/maison-des-initiatives-etudiantes-2521" title="Maison des initiatives Ã©tudiantes (House of Student initiatives)">MIE</A></LI><LI><A target="_black" class="papet" href="http://www.grandparissud.fr/" title="Grand Paris Sud">Papeterie GPS</A></LI></UL></DIV><SPAN id="pvl"><STRONG>Evry</STRONG>2016.</SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>