Team:Evry/Notebook
Homepage
Visit the iGEM website!
Feel free to contact us!
E-mail
Twitter
Facebook
Instagram
Let's PLAy project - Bioproduction of PLA
Show menu
Home
Project
Description
Design
Modeling
Experiments
Notebook
Improvements
Achivements
Parts
Medal fulfillments
Results
Interlab
Results
Protocol
Lab book
Safety
Project safety
Lab safety
Shipment safety
Human practices
Integrated HP
Engagements
iGEM events
Team
Our team
iGEM in Evry
Collaborations
Campus
Sponsors
Attributions
Home
Project
Achivements
Interlab
Safety
Human practices
Team
Attributions
Homepage
Project
Note book
Note book
Complete version of our note book (.pdf)
Mon
Tue
Wed
Thu
Fri
Sat
Sun
July
Week 1
4
5
6
7
8
9
10
Week 2
11
12
13
14
15
16
17
Week 3
18
19
20
21
22
23
24
Week 4
25
26
27
28
29
30
31
August
Week 5
1
2
3
4
5
6
7
Week 6
8
9
10
11
12
13
14
Week 7
15
16
17
18
19
20
21
Week 8
22
23
24
25
26
27
28
Week 9
29
30
31
September
Week 9
1
2
3
4
Week 10
5
6
7
8
9
10
11
Week 11
12
13
14
15
16
17
18
Rest of the lab book (.pdf)
July 2016
Week 1
4th of July
Bottles of LB and LB-Agar (200 mL and 400 mL)
LB-Agar: 14 g for 400 mL of water
LB: 8 g for 400 mL of water
Autoclave
Resistance test of P. putida
2 plates of LB-Agar with either Kanamycin (50 µg/mL) or Spectinomycin (50 µg/mL).
Preparation of HEPES solution 0.5 M (for electroporation)
11 g of HEPES (solid) in 64 mL of water
NaOH (0.1 M) for pH7
Refresh P. putida (the Petri dish got old)
E. coli C118 with plasmids pSEVA2311 (KanR) and pSEVA224 (KanR) were separately spread in LB.
cells with pSEVA2311  were observed
cells with pSEVA224 were not observed!!
Jonathan asked for another sample
5th of July
LB culture of E. coli C118
HEPES filtration
Check the inventory of the fridge
Plate of P. putida at 30 °C and 37 °C → We didn’t do this step
Liquide culture (5 mL of preculture) of P. putida to have competent cells
6th of July
Preculture for transformation (5 mL) of E. coli DH5 α
7th of July
Miniprep of E. coli C118 with pSEVA2311
Protocol “Midiprep” with 50 mL of culture
C = 80 ng/µL in 200 µL of elution buffer
OD260 = 1.616
OD280 = 0.8
Transformation of Pseudomonas with pSEVA2311
40 mL of culture (competent cell):
When OD600nm = 0.6
20 mL for Heat Shock
20 mL for Electroporation
After electroporation and heat-shock spread on Amp-Plates
8th of July
The transformation of 07/07/2016 didn’t work because the plasmid given by Robert didn’t match with Pseudomonas→ Second transformation (Heat shock and electroporation)
Preparation of  plates (LB + Antibiotics)
10 plates LB-Agar, Kanamycin (1 µL/mL)
10 plates LB-Agar, Chloramphenicol (1 µL/mL)
10 plates LB-Agar, Ampicillin (1 µL/mL)
10 plates LB-Agar, without antibiotic
Week 2
12th of July
Preparation of M9-media (10X, V= 600 mL) with different kind of C-sources
Glucose → DONE
Glycerol → DONE
Fructose → NO YET
Benzoate → NO YET
Lactic acid → NO YET
Methane → NO YET
Compounds
Cf
Masses (g) or volumes (ml)
CaCl2.H2O
1 mM
0.0882 g
MgSO4
20 mM
1.44 g
FeSO4.H2O
0.1 mM
0.01668 g
M9 (salt)
10 X
67.68 g
Glycerol (100 %) *
8 % v/v
60.48 g
Casamino acid
2 % w/v
12 g
Thiamine 10 mg/mL **
10 µg/mL
(0.6 g)
Uracil
200 µg/mL
0.12 g
Leucine
300 µg/mL
0.18 g
NaOH
pH 6.6
-
Table 1. *Glucose : 59.22 g;
** Thiamine has not been added but it has to be before the preculture of P. putida.
→ problem : precipitation
13th of July
Preparation of the deletion process – Transformation of E. coli Mach1 competent cells (CaCl2 treatment) by heat shock
Plasmids pKD46, pCP20 and pKD4
3 tubes in which 25 µL of cells were mixed with 1.5 µL of each plasmid (pKD4, pCP20 or pKD46)
Incubation on ice for 20 min
Heat at 42 °C (thermobloc) 1 min
120 µL of LB added to each tube
Incubation: 30 min at 37 °C for the plasmid pKD4, 30 °C for the plasmids pCP20 and pKD46
Spread on plates (Ampicilline 100 μg/mL)
Incubation overnight at 37 °C for pKD4, 30 °C for pCP20 and pKD46
Preparation for the deletion process – Preparation of E. coli DH5 α competent cells
5 mL of preculture of DH5 α were prepared
Incubation overnight at 37 °C
14th of July
pKD46 and pCP20 transformation properly worked
pKD4 didn’t succeed
Preparation of 5 mL culture of pKD46 and pCP20 to purify plasmid
pKD4 since it is not needed for transformation will be used w/o amplification
DH5 α competent cells were prepared and stored
15th of July
Midiprep of pCP20 and pKD46
pCP20: 65 ng/µL
pKD46: 50 ng/µL
Transformation of P. putida KT2440 with pKD46 plasmid to start deletion process
Incubation overnight at 30 °C (Amp Plates)
Spread KT2440 WT and check growth at 37 °C
16th of July
Transformation results
cells grew on Ampicillin at 30 °C
Pick 4 clones from previous plate and spread them in a new plate (Amp with proper concentration)
Incubation at 30 °C
Pseudomonas putida : Working concentration of Antibiotics
Kan (50 µg/mL) → 1.7 µL of stock solution/mL
Amp (500 µg/mL) → 5 µL of stock solution/mL
Spec (50 µg/mL) →1 µL of stock solution/mL
KT2440 grew well at 37 °C
17th of July
Week 3
18th of July
Electroporation of P. putida : pKD46
Electro-competent cells have been prepared
1.5 µL of DNA were added to ~ 50 µL of electro-competent cells (all the tube)
1 mL of LB was transfer in a 1.5 mL tube
500 µL of the LB were prepare in the pipette-tip and put rapidly in the electro-cuve  after electroporation (at 1800V)
All the cells+LB of the cuve were transferred to the 1.5 mL tube containing LB
Incubation + Shaking  1 h at 30 °C
Centrifugation 3 min 90 rpm
Cells were plated in LB Agar Ampicillin (100 μg/μL) and incubated at 30 °C
Spread C118 (pSEVA224) on plates with Kanamycin
19th of July
The transformation (Electroporation of 18/08) didn’t work → maybe the replicon doesn’t match with P. putida
Colony PCR: to check the transformation of P. putida with p2311
Tick one colony and with the same tip, spread a little area on LB plate and dissolve the rest in 10 µL ddH2O
Gently vortex and briefly centrifuge DreamTaq Green PCR Master Mix (2X) after thawing
Place a thin-walled PCR tube on ice and add the following components for each 50 µL reaction
DreamTaq Green PCR Master Mix (2X)
1X
25 µL
Forward primer
0.5 µM
0.25 µL
Reverse primer
0.5 µM
0.25 µL
Colony water
----
2 µL
Water, nuclease-free
----
22.5 uL
Gently vortex the samples and spin down
Perform PCR using the recommended thermal cycling conditions outlined below
Step
Temperature °C
Time
Number of cycles
Initial denaturation
95 °C
5 min
1
Denaturation
95 °C
30 s
30
Annealing
55(Tm)-5 °C
30
Extension
72 °C
1 min/kb
Final extension
72 °C
10 min
1
Pause
4 °C
“00”
---
Load 5-15 µL of PCR mixture directly on a gel
Electrophoresis to check the transformation
1 % of agarose
Duplicates have been done
See complete version of the notebook for details on the conditions.
PCR of pKD4
Mix for 100 µL
Mix 1
Reaction buffer
20 µL
dNTP mix
2 µL
Primers (reverse and forward)
0.5 µL
Template (pKD4)
1 µL
Q5 polymerase
1 µL
H2O
Qs 100 µL (75.5 µL)
Cycles
Temperature
Time
Number of cycles
98 °C
30”
1
90 °C
30”
60 °C
30”
32
72 °C
1’
72 °C
5’
1
4 °C
“00”
---
Culture of E. coli C118 pSEVA for midi prep’
20th of July
Midiprep’ of pSEVA224 (from E. coli)
C= 59.9 ng/µL in 200 µL of elution buffer
DO260/DO280 = 1.97
Alternative to delete gene
Transformation of Pseudomonas by a PCR product : Kanamycin resistance cassette + homologous region 5’ and 3’ using Heat Shock and Electroporation.
See protocols for both transformations
We used 10 µL of DNA
Preparation of 7 plates for Pseudomonas (1.7 µL/mL of kanamycin stock solution)
Liquid culture of E. coli pSEVA224 (2 mL)
2 mL of LB
3.4 µL of Kan - It was a mistake: we should have put 2 µL of Kan (since we used E. coli)
Preculture to amplify the vector given by Cyril (BBa.R0010 pLac)
E. coli G3A
50 mL of LB
50 µL of Amp
21th of July
!!! The fridge and freezer went off !!!
The transformation of the PCR product (for the deletion process) didn’t worked
We will focus on the construction of the plasmid
Store the culture
Storage in glycerol = 1 vol of glycerol + 1 vol of the culture and at -80 °C
Midiprep of the vector R0010
325 ng/µL
DO260/DO280 = 1.8
Preparation of Ampicillin solution stock 50 mg/mL in H2O (Helix) aliquoted in 5 tubes of 2 mL
In sterile condition
0.5 g of Amp + 10 mL of H2O
Filtration (0.2µm)
Storage away from light
Digestion of
the vector (with Ba.R0010 pLac from Cyril) by EcoRI (E) and SpeI (S)
the vector (with Ba.R0010 pLac from Cyril)  by EcoRI and PstI (P)
the phaC sequence by EcoRI and SpeI
the Propionyl-CoA transferase sequence by EcoRI and PstI
By the way, we tried to remove the promoter (BBa.R0010 pLac) from the vector
100 µL of Water were added to the phaC sequence and the Propionyl-CoA transferase sequence
Vector (E/S)
phaC (E/S)
Vector (E/P)
propCoA(E/P)
DNA
2 µL
10 µL
2 µL
10 µL
Buffer
Cutsmart 10X: 3 µL
Cutsmart 10X: 3 µL
2.1: 3 µL
2.1: 3 µL
Enzymes
EcoRI: 1 µL
EcoRI: 1 µL
EcoRI: 1 µL
EcoRI: 1 µL
SpeI: 1 µL
SpeI: 1 µL
PstI: 1.5 µL
PstI: 1.5 µL
H2O
23 µL
15 µL
22.5 µL
14.5 µL
1 h at 37 °C
Electrophoresis
Wells:
Marker
Control vector with pLac
Vector  E/S (30 µL DNA + 6 µL loading buffer)
Vector E/S (30 µL DNA + 6 µL loading buffer)
(8 µL)
(5 µL DNA + 1 µL loading buffer)
Expected results
__
1 band (2270 bp)
2 bands (2070 bp + 200 bp)
2 bands (2070 bp + 200 bp)
1 % agarose
Vector (with pLac): 2270 bp
Promoter pLac ~ 200 bp
Results
* : circular plasmid (3 forms: compacted, intermediate and loose forms)
-We get rid of the promoter and we kept the empty vectors
Purification of the DNA sequences phaC and prop-CoA transf.
Zymoclean Gel DNA Recovery Kit
Elution with 6 µL of water
Wait 1-2 min before the last spin
The check at the spectrophotometer nanodrop showed a pic around 240 nm
Maybe: problem with the wash buffer reminded in the column
Inversion of the 2 columns before the second spin
22th of July
Ligations
Empty vector (R0010)
10 µL
Insert phaC
20 µL
T4 buffer 10X
4 µL
T4 ligase
2 µL
H2O
5 µL
Empty vector (R0010)
10 µL
Insert propCoA transf.
20 µL
T4 buffer 10X
4 µL
T4 ligase
2 µL
H2O
5µL
Incubation 1 h at RT
Transformation of E. coli DH5 α, Top 10 and BL21 by Heat Shock
Same protocol
We didn’t recover enough DNA, insert from previous midiprep, purification
These strains were spread on labeling plates (see table below)
Incubation overnight at 37 °C
22/07/16 E. coli phaC DH5 α
22/07/16 E. coli phaC Top10
22/07/16 E. coli phaC BL21
22/07/16 E. coli propCoA DH5 α
22/07/16 E. coli propCoA Top10
22/07/16 E. coli propCoA BL21
23th of July
“Transformed” clones were observed
Colony PCR
Week 4
25th of July
Media
5 bottle of LB
3 x 400 mL
2 x 200 mL
5 bottle of LB-Agar
3 x 400 mL
2 x 200 mL
Previous transformations (22/07) worked
Electrophoresis (PCR colony 23/07)
Gel 1 %
propCoA (in Cyril’s vector)
phaC (in Cyril’s vector)
Precultures
2 mL of LB
Amp (1 µL stock solution/mL)
A good colony
phaC : 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 19, 20
propCoA :1, 3, 4, 5, 6, 8, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20
Incubation overnight at 37 °C
26th of July
Miniprep’ kit Sigma-Aldrich : extraction of the plasmids from the precultures of the 26th of july
For the propCoA transferase : problem with the 8th tube we lost a lot of bacteria*
Colonies (with the phaC )
Concentrations of DNA (ng/µL)
Colonies (with the phaC )
Concentrations of DNA (ng/µL)
1
282.3
11
160.9
2
170.4
12
173.0
4
174.6
13
170
5
176.5
14
185.3
6
188.9
15
162
7
167.8
16
189.5
8
185.9
17
193.6
9
191.4
19
183
10
173.4
20
163
Colonies (with the propCoA )
Concentrations of DNA (ng/µL)
Colonies (with the propCoA )
Concentrations of DNA (ng/µL)
1
150.5
13
169.4
3
143.4
14
282.8
4
156
15
192.3
5
151.8
16
226
6
234.2
17
203.3
8
29.4*
18
175.1
10
194.5
19
256.5
11
167.2
20
181.7
12
162.8
-
-
M9 medium 2 x 50 mL of 2.5 X
Without C-sources, thiamine
Problem: precipitation (the same than previous M9 media)
27th of July
Stock solutions
CaCl2 1 M: 27.745 g in 250 mL of milliQ
MgSO4 1 M: 30.1 g in 250 mL of milliQ
FeSO4 10 mM: 0.379 g in 250 mL of milliQ
Preculture of Pseudomonas putida KT2440 (2 mL)
28th of July
Aliquots for sequencing  V= 20 µL
Medium M9 1X without Carbone source and filtred (2 µm) in 2 falcon of 50 mL
Casamino acid : 1.2 g
Leucine: 3 mg
Uracil: 2 mg
Thiamine 10 mg/mL: 10 µL
M9 salt 5X: 20 mL
CaCl2 1 M: 10 µL
MgSO4 1 M: 100 µL
FeSO4 10 mM: 1 µL
Water milliQ qs 100 mL
NaOH for pH between 6 and 7.4 → no need since the pH was already at 6.8
No precipitation
10 mL of Media with C-source
0.10037 g of Glycerol (100 %) (0.16 put) in 10 mL of milliQ water
0.0982 g of Glucose in 10 mL of milliQ water
0.057 g of Benzoic acid in 10 mL of milliQ water
Preparation for plate (96 wells) and Growth rate with the CLARIO star machine
26.7 µL of the overnight preculture of Pseudomonas in medium M9 (with either Benzoic acid, Glucose or Glycerol) qs 1 mL to have around OD= 0.005
Plate mode, slow kinetic, DO600nm, T= 30 °C, shaking (200 rpm) before cycle of 10 min
1
2
3
4
A
-
Benzoic acid M9 medium
Glucose M9 medium
Glycerol M9 medium
B
Benzoic acid M9 medium
BE-M9 with cells
BE-M9 with cells
BE-M9 with cells
C
Glucose M9 medium
GLU-M9 with cells
GLU-M9 with cells
GLU-M9 with cells
D
Glycerol M9 medium
GLY-M9 with cells
GLY-M9 with cells
GLY-M9 with cells
August 2016
Week 5
2nd of August
Resolubilisation of IDT gBlocks in 100 µL water (Cf= 10 ng/µL)
Digestion
Mix:
PhaC2
PCT1
LDH1
pSB1A3
GBlock gene
10 µL
10 µL
10 µL
2 µL (≈600ng)
Buffer NEB
3 µL (2.1)
3 µL (CutSmart)
3 µL (2.1)
3 µL (CutSmart or 2.1)*
Enzyme 1
1 µL (EcoRI)
1 µL (EcoRI)
1 µL (XbaI)
1 µL (EcoRI or XbaI)
Enzyme 2
1 µL (PstI)
1 µL (SpeI)
1 µL (PstI)
1 µL (PstI or SpeI)
H2O (qs 30 µL)
15 µL
15 µL
15 µL
23 µL
*CutSmart when we doesn’t digest by PstI and 2.1 when PstI is used
Incubation 1 h 37 °C
Agarose gel 1 %
Digestion
Mix:
-
PhaC2
PCT1
LDH1
GBlock gene
10 µL
10 µL
10 µL
Buffer NEB
3 µL (2.1)
3 µL (CutSmart)
3 µL (2.1)
Enzyme 1
1 µL (EcoRI)
1 µL (EcoRI)
1 µL (XbaI)**
Enzyme 2
1 µL (PstI)
1 µL (SpeI)*
1 µL (PstI)
H2O (qs 30 µL)
15 µL
15 µL
15 µL
* PCT1 has a PstI restriction site in its sequence so the digestion was done by SpeI
** LDH1 has a EcoRI restriction site site in its sequence so the digestion was done by XbaI
Incubation at 37 °C
DNA extraction from gel
DNA clean-up → kit Monarch PCR and DNA cleanup (#T1030G) NEB
→ DNA Gel extraction kit (#T1020G) NEB
Incubation at 55 °C to dissolve gel
400 µL buffer for 100 mg of agarose gel
DNA elution in 40 µL water
Ligation
10 µL Vector (pSB1A3 digested)
20 µL Insert digested
4 µL 10X T4 DNA ligase buffer
2 µL T4 DNA ligase
5 µL H2O
Incubation RT 1 h
Transformation
Thaw 25 µL of competent DH5α E. coli
Add 10 µL of ligated DNA
20 min incubation in ice
1 min 42 °C
Add 120 µL of LB medium
Incubation 40 min 37 °C
Spread on plates LBA (Ampicillin)
Incubation 37 °C overnight
3rd of August
Colony PCR
Mix (50 µL total volume reaction)
25 µL DreamTaq PCR Mastermix 2X (ThermoScientific)
5 µL 10X DreamTaq Green Buffer (ThermoScientific)
0.25 µL Forward Primer (0.5 µM) iG001
0.25 µL Reverse Primer (0.5 µM) iG002
19.5 µL H2O filtrated (qs 50 µL)
Mix prepared for 15 reactions (plus one reaction prepared without Green buffer)
PCR program:
95 °C 5 min
(95 °C 30 s, 50 °C 30 s, 72 °C 2 min) 30 times
72 °C 10 min
10 °C ∞
Nanodrop
PhaC2 3.5 ng/µL
PCT1 3.8 ng/µL
LDH1 8.7 ng/µL
pSB1A3 (digested by X+P) 3.3 ng/µL
Gel electrophoresis
The results were not conclusive
4th of August
Colony PCR
Mix (25 µL total volume reaction)
12.5 µL DreamTaq PCR Mastermix 2X (ThermoScientific)
2.5 µL 10X DreamTaq Green Buffer (ThermoScientific)
0.25 µL Forward Primer (0.5 µM) iG001
0.25 µL Reverse Primer (0.5 µM) iG002
9.5 µL H2O filtrated (qs 25 µL)
Mix prepared for 7 reactions
PCR program:
95 °C 5 min
(95 °C 30 s, 50 °C 30 s, 72 °C 2 min) 30 times
72 °C 10 min
10 °C ∞
The results were not conclusive
Week 6
8th of August
Ligation
50 ng vector
5 µL T4 DNA ligase Buffer 10X
3 µL T4 DNA
H2O qs 50 µL
DNA
PhaC 2
11.7 µL
Vector E/P
7.8 µL
H2O 22.5 µL
(3.5 ng/µL)
(6.4 ng/µL)
PCT 1
30.4 µL
Vector E/S
4.1 µL
H2O 7.5 µL
(3.8 ng/µL)
(12.2 ng/µL)
LDH 1
13.9 µL
Vector X/P
8.5 µL
H2O 19.6 µL
(8.7 ng/µL)
(5.9 ng/µL)
We put 27.5 µL of Pct1 DNA because there was not enough.
Transformation
Thaw DH5-α competent bacteria in the ice.
Using the protocol processing by thermal shock.
25 µL of bacteria + Mix of ligation (2 µL each time)
Incubation 20 minutes in ice and then 1 min at 42 °C
Adding 120 μl LB and incubate at 37 ° for 1 hour
Spread on LB Ampicillin with the öse
9th of August
Colony PCR
Mix PCR with 25 µL of total volume reaction prepared for 12 reactions prepared as before (DreamTaq #K1071 Thermo Scientific).
PCR program:
95 °C 5 min
(95 °C 30 s, 50 °C 30 s, 72 °C 1 min) 30 times
72 °C 10 min
10 °C ∞
No bands on the gel (maybe problems with the enzyme)
10th of August
Digestion
Mix:
PhaC2
PhaC3
PCT2
LDH1
GBlock gene
20 µL
20 µL
20 µL
20 µL
Buffer NEB
3 µL (2.1)
3 µL (2.1)
3 µL (CutSmart)
3 µL (2.1)
Enzyme 1
1 µL (EcoRI)
1 µL (EcoRI)
1 µL (EcoRI)
1 µL (XbaI)*
Enzyme 2
1 µL (PstI)
1 µL (PstI)
1 µL (PstI)
1 µL (PstI)
H2O (qs 30µL)
5 µL
5 µL
5 µL
5 µL
* LDH1 has a EcoRI restriction site site in its sequence so the digestion was done by XbaI
pSB1A3
pSB1A3
Vector DNA
2 µL (≈600 ng)
6 µL (≈2 µg)
Buffer NEB
3 µL (CutSmart or 2.1)*
9 µL (CutSmart or 2.1)*
Enzyme 1
1 µL (EcoRI or XbaI)
3 µL (EcoRI or XbaI)
Enzyme 2
1 µL (PstI or SpeI)
3 µL (PstI or SpeI)
H2O (qs 30 µL)
23 µL
69 µL
Divided in 3 reaction tubes
Incubation 1 h 37 °C
1 % agarose gel
2 trash nucleotides after the PstI restriction site in the suffix were forgotten so the digestion risks to fail → the digestion will be done by SpeI!
PhaC genes doesn’t match the gene in our reference publications but it corresponds to the gene used by the YALE IGEM team in 2013
11th of August
Extraction of digested vectors (pSB1A3 X+P and pSB1A3 E+P) from agarose gel (kit Zymoclean gel DNA Recovery kit #D4001S)
Concentration of the samples
pSB1A3 digested by X+P => 3.9 ng/µL
pSB1A3 digested by E+P => 7.2 ng/µL
Digestion by SpeI
Mix:
20 µL genes digested by E+P or X+P or vector digested by X+P
3 µL buffer NEB CutSmart
1 µL enzyme SpeI
6 µL H20 (qs 30 µL)
Or mix:
20 µL genes digested by E+P or X+P or vector digested by X+P
3 µL buffer NEB CutSmart
1 µL enzyme SpeI
6 µL H20 (qs 30 µL)
DNA purification
DNA clean & concentrator-5 (#D4003S Zymo Research)
For the vector: 2 volumes of binding buffer add to DNA (ratio 2:1)
For the genes: 5 volumes of binding buffer add to DNA (ratio 5:1)
Elution with 20 µL of water and incubation 2 min before centrifugation for DNA elution
Ligation
Mix:
PhaC2
PhaC3
PCT2
Vector pSB1A3
4.5 µL
4.5 µL
4.5 µL
Gene
14 µL
11 µL
10 µL
T4 DNA ligase Buffer
3 µL
3 µL
3 µL
T4 DNA ligase
1 µL
1 µL
1 µL
H2O (qs 30 µL)
7.5 µL
10.5 µL
11.5 µL
Incubation 1 h RT
Sample frozen at -20 °C until transformation
12th of August
Transformation
Thaw DH5-α competent bacteria in the ice.
Using the protocol processing by heat shock.
25 µL of bacteria + 5 µL ligation mix
Incubation 20 minutes in ice
1 min at 42 ° C
3 min in ice
Adding 120 μl of LB
Incubation at 37 °C for around 50 min
Spread on LB Ampicillin plates
Incubation overnight 37 °C
13th of August
Results transformation
We got clones from the transformation. We took 3 clones from Pct(v2), 2 from phaC(v2) and 2 from phaC(v3) and performed PCR colony.
PCR colonies
20 µL mix per sample with:
10 µL Green Taq Master Mix 2X (#K1081 Thermo Scientific)
0.25 µL Forward primer iG001 (10 µM)
0.25 µL Forward primer iG002 (10 µM)
9.5 µL H2O (qs 20 µL)
Mix for 9 reactions + clones
PCR program:
95 °C 5 min
(95 °C 30 s, 50 °C 30 s, 72 °C 2 min) 30 times
72 °C 10 min
10 °C ∞
1 % agarose electrophoresis (90 V, 111 mA, 40 min)
=> Result of PCR Colony: 1 clone of each was good
Precultures
3 mL LB medium + positive clones for each gene (PhaC2, PhaC3, PCT2)
Incubation 37 °C with shaking (around 16 h incubation)
Transformation (E. coli DH5-α with pSB1A3 ligated with gene) by heat shock
Thaw DH5-α competent bacteria in ice
100 µL of bacteria + 10 µL ligation mix
Incubation 20 minutes in ice
45 s at 42 ° C
3 min in ice
Adding 900 μl of LB
Incubation 1 h, 37 °C
Spread on LB Ampicillin plates
Incubation overnight 37 °C
14th of August
No colony for PCT2 transformation!
Miniprep from precultures
Transformed bacteria were stored at -80 °C in 50 % glycerol (2 aliquotes for each preculture)
GenElute Plasmid Miniprep Kit (Sigma, PLN350-1KT)
Incubation 1 min for elution of DNA before centrifugation
pSB1A3-PhaC2 144 ng/µL
pSB1A3-PhaC3 96.1 ng/µL
pSB1A3-PCT2 143.3 ng/µL
Digestion
Mix
-
LDH1 gBlock gene (10 ng/µL)
pSB1A3 (X+P)
DNA
20 µL (200 ng)
50 µL
10X Buffer NEB CutSmart
3 µL
6 µL
enzyme XbaI
1 µL
2 µL
enzyme SpeI
1 µL
-
H2O (qs 30 µL)
5 µL
2 µL (qs 60 µL)
Incubation 1 h 37 °C
Dephosphorylation
Add 1 µL alkaline phosphatase (FastAP #EF0654 Thermo Scientific) to the digestion mix (for LDH1 digestion or vector pSB1A3 digestion)
Incubation 30 min 37 °C
Inactivationof the enzyme: incubation 75 °C, 10 min
Recombinant vector digestion
Mix
-
pSB1A3-PhaC2
pSB1A3-PhaC3
pSB1A3-PCT2
DNA (around 200 ng or 600 ng)
2 µL
2.5 µL
6 µL (around 600 ng)
Buffer NEB CutSmart 10X
3 µL (2.1)
3 µL (2.1)
3 µL (CutSmart)
Enzyme 1
1 µL (EcoRI)
1 µL (EcoRI)
1 µL (EcoRI)
Enzyme 2
1 µL (SpeI)
1 µL (SpeI)
1 µL (XbaI)
H2O (qs 20 µL)
14 µL
13.5 µL
10 µL
Incubation 1 h 37 °C
DNA purification (LDH gene)
DNA clean & concentrator-5 (#D4003S Zymo Research)
Ratio 5:1 of DNA binding buffer for LDH gene fragment
Elution 30 µL water with incubation 3 min before centrifugation
LDH1 =>
6.9 ng/µL
Agarose gel for digested recombinant vectors
All of the digested samples is charged on the gel.
Digestion of the recombinant vector. Digestion was a failure for the 2 plasmids containing the PhaC2 and PhaC3 genes. The vector containing PCT2 is open but we have no way to be sure that the digestion was efficient for the 2 enzymes or just for one of them.
Colony PCR on the clones obtained after the 2nd transformation of the ligation mixes:
PCR colony. PCR on colony failed.
Week 7
16th of August
Digestion
-
LDH1 gBlock gene (10 ng/µL)
pSB1A3 (352 ng/µL)
DNA
30 µL (300 ng)
2 µL
10X Fastdigest buffer (ThermoScientific)
4 µL (clear buffer)
2 µL (green buffer)
enzyme XbaI
1 µL
1 µL
enzyme SpeI
1 µL
1 µL
H2O (qs 40 µL)
4 µL
14 µL (qs 20 µL)
-
pSB1A3-PhaC2
pSB1A3-PhaC3
pSB1A3-PCT2
DNA (around 200 ng or 600 ng)
2 µL
2.5 µL
6 µL (around 600 ng)
Buffer Fastdigest green 10X (ThermoScientific)
2 µL
2 µL
2 µL
Enzyme 1
1 µL (EcoRI)
1 µL (EcoRI)
1 µL (EcoRI)
Enzyme 2
1 µL (SpeI)
1 µL (SpeI)
1 µL (XbaI)
H2O (qs 20 µL)
14 µL
13.5 µL
10 µL
Agarose gel 1 %
DNA purification (digested by X + S LDH1 gene)
DNA clean & concentrator-5 (#D4003S Zymo Research)
Ratio 5:1 of DNA binding buffer for LDH gene fragment
17th of August
Gel purification via the « Zymoclean Gel DNA Recovery » kit
weight of the empty tube: 1 g
weight of the tube + vector PSB1A3 (X+S)
n°1 : 1.1 g is 100 mg
n°2 : 1.1 is 100 mg
vector size: 2155 bp
Step 1 : For 100 mg d’agarose we add 300 µL of ADB
Step 2 : Put samples at 55 °C for 10 minutes and centrifuge at 300 rpm
Step 3 : Transfert in the column
Step 4 : Centrifugation while 1 minute at 13 000 rpm then suppression of the flow-through
Step 5 : Add 200 µL of DNA Wash Buffer and centrifugation while 30 seconds X2
Step 6 : Centrifugation empty for 2 minutes
Step 7 : Add 10 µL DNA Elution Buffer, wait 2 minutes
Step 8 : Centrifuge 1 minutes at 13 000 rpm
Nano drop
DO vector Psb1a3 n°1 : 39 ng/µL
DO vector Psb1a3 n°2 : 43.2 ng/ µL
18th of August
Chloramphenicol stock
50 ng/mL chloramphenicol stock (2.5 g in 50 mL ethanol 100 %)
30 aliquots stored at -20 °C
pSB1C3-mRFP
Solubilisation of pSB1C3-mRFP (2016 IGEM plate 5 - 1F) with 10 µL water nuclease-free (ThermoScientific)
Incubation 5 min RT
Transformation in E. coli DH5-α strain
Transformation
Thaw 25 µL DH5-α competent bacteria in ice
Add 2 µL of resolubilized pSB1C3-mRFP DNA from 2016 IGEM plate 5
Incubation 20 minutes on ice
1 min at 42 ° C
3 min in ice
Adding 120 μl of LB
Incubation 1 h, 37 °C with shaking
Spread on LB Chloramphenicol plates
Incubation overnight 37 °C
Solubilisation of the primers iG051, iG052, iG049 and iG050 and aliquots of 10 times dilutions.
19th of August
Precultures
3 mL LB medium + 1.8 µL Chloramphenicol (50 mg/mL) => plates at 30 µg/mL chloramphenicol
No conclusive results for PCR on colonies for pSB1C3-mRFP plates so we did precultures with 4 other clones (A, B, C and D)
The Mastermix Taq DNA polymerase seems to not work anymore (after freezer problem, the polymerases lost their activities)
Q5 PCR for changing antibiotic resistance in pSEVA 224 plasmid
10 µL reaction buffer
1 µL dNTP mix
1 µL 10 times diluted primers (X2)
0.5 µL DNA template
0.5 µL Q5 DNA polymerase
H20 qs 50 µL (= 36 µL)
2 different reactions:
Amplification of spectinomycin resistance gene with pCDF vector as DNA template (length expected = 1220 bp) primers used: iG050 and iG052 (1 min of DNA polymerization and primers annealing at 60 °C)
Amplification of pSEVA224 backbone without kanamycin resistance gene (pSEVA224 as DNA template; length expected = 4253 bp) primers used: iG049 and iG051 (3 min of DNA polymerization and primers annealing at 60 °C)
95 °C 5 min
(95 °C 30 s, 60 °C 30 s, 72 °C 1 min or 3 min) * 35 times
72 °C 5 min
10 °C ∞
20th of August
DNA miniprep of pSB1C3-mRFP (clone A, B, C and D)
Stock in 50 % glycerol stored at -80 °C
GenElute Plasmid Miniprep Kit (Sigma, PLN350-1KT)
Incubation 2 min for DNA elution before centrifugation and centrifugation of 1 min 30 s instead of 1 min.
Optional Wash Buffer step performed and use of nuclease-free water for elution.
Clone A: 240.3 ng/µL
Clone B: 150.3 ng/µL
Clone C: 111.0 ng/µL
Clone D: 76.6 ng/µL
PCR on DNA miniprep - Primers iG001 and iG002
20 µL mix per sample with:
10 µL Green Taq Master Mix 2X (#K1081 Thermo Scientific)
0.25 µL Forward primer iG001 (10 µM)
0.25 µL Forward primer iG002 (10 µM)
9.5 µL H2O (qs 20 µL)
PCR program:
95 °C 5 min
(95 °C 30 s, 50 °C 30 s, 72 °C 1 min) * 30 times
72 °C 5 min
10 °C ∞
Agarose gel 1 % for changing antibiotic PCR
The amplification of pSEVA => new attempt
Digestion vectors pSB1C3-mRFP (on DNA miniprep)
10 µL plasmid DNA
3 µL Buffer 10X green FD (FastDigest)
1 µL EcoRI
1 µL SpeI
15 µL H20 (qs 30 µL)
Size expected: 2051 bp
Digestion was checked on agarose gel and the band corresponding to pSB1C3 digested by E+S was cut in order to recover the DNA.
Week 8
22th of August
Digestion efficiency test of restriction enzymes EcoRI and SpeI
Mix
4 µL vector pSB1C3
3 µL buffer 10X CutSmart or FastDigest Green
1 µL EcoRI and/or SpeI
21 µL or 22 µL H2O
Incubation 1 h 37 °C
Agarose gel 1 %
DNA extraction from agarose gel (linear vector extracted from clones A, B and C)
Zymoclean gel DNA Recovery kit #D4001S
Vector A: 20.9 ng/µL
Vector B: 13.3 ng/µL
Vector C: 13.1 ng/µL
Expected sizes after digestion by E+S
pSB1C3 2051 bp
mRFP 742 bp
23th of August
Digestion gBlocks by E+S
20 µL gBlocks DNA
3 µL 10X NEB Buffer (Cutmart)
1 µL EcoRI
1 µL SpeI
5 µL H2O (qs 30 µL)
Incubation 1 h at 37 °C
Purification Agarose Gel with the kit « Zymoclean Gel DNA Recovery kit » (#D4001S Zymo Research)
Cf protocol august 17th 2016
Elution with 20µL H2O
Nanodrop 1
LDH2: 10.3 ng/µL
Phac3: 5.9 ng/µL
Phac 4: 8.5 ng/µL
Pct 2: 8.9 ng/µL
too low concentration for ligation so I put at the Speed Vae. Protocole "Vaccum Drive"; resuspend with 10 µL H2O
Nanodrop 2
LDH2 = 15 ng/µL - Ratio 260/280 : 1.92
Phac 3 = 8.8 ng/µL - Ratio 260/280 : 2.20
PhaC4 = 9.2 ng/µL - Ratio 260/280 : 1.05
Pct2 = 10.8 ng/µL - Ratio 260/280 : 1.88
Gene
Length
Amount (ng)
Concentration (ng/µL)
Volume (µL)
Plasmid psB1C3
2051 pb
20.9
2.4
PhaC3
1717 pb
209.3
8.8
23.8
LDH2
1227 pb
149.6
15
10
Phac 4
1720 pb
209.7
9.2
22.8
Pct 2
1612 pb
196.5
10.8
18.2
Amount of insert: 50*5 x length gBlock / length plasmid
Volume of insert: amount of insert / concentration of insert
As we suspended in 10 μl I do not have sufficient quantities.
Ligation mix
Insert
DNA (µL)
Vector (µL)
Buffer T4 Ligase 10X (µL)
T4 Ligase (µL)
H2O (qs 20 µL)
Pct2
9
0.96
2
0.5
7.54
LDH2
9
1.29
2
0.5
7.21
Phac4
9
0.96
2
0.5
7.54
Phac3
9
0.96
2
0.5
7.54
I put at 22 °C the ligation Mix for 10 min
I take 5 µL for the transformation of 50 µL competent cells
Transformation
Use protocol of Bacterial transformation by heat shock
Thaw competent cells on ice
mixed 50 µL of competent bacteria with 5 µL of DNA
incubate on ice for 20 minutes
placing the tubes at 42 ° C for 1 min then put 3 min in ice
Add 250 µL of LB and incubate 1 h at 37 °C
Spread on box LB Chloramphenicol (30 µg/mL)
incubation overnight 37 °C
Poored 6 plates of 25 mL of LB agar + chloramphenicol (30 µg/µL)
Incubation at 37 °C
24th of August
Digestion gBlocks by E+S
20 µL gBlocks DNA
3 µL 10X NEB Buffer (Cutmart)
1 µL EcoRI
1 µL SpeI
5 µL H2O (qs 30 µL)
Incubation 1 h at 37 °C
Purification Agarose Gel with the kit « Zymoclean Gel DNA Recovery kit » (#D4001S Zymo Research)
Elution with 20 µL nuclease-free water
PhaC3 (E+S) => 12.2 ng/µL
PhaC4 (E+S) => 6.3 ng/µL
LDH2 (E+S) => 16.8 ng/µL
PCT2 (E+S) => 12.4 ng/µL
Concentration of the DNA: around 1 h in evaporator
DNA resuspension with 11 µL water
First concentration failed so second concentration of DNA performed in evaporator (1 h)
25th of August
Resuspension of DNA with 11 µL water
PhaC3 (E+S) => 10.4 ng/µL
PhaC4 (E+S) => 10.6 ng/µL
LDH2 (E+S) => 27.0 ng/µL
PCT2 (E+S) => 20.1 ng/µL
DNA concentration by water evaporation failed!
Ligation genes inside pSB1C3 (ratio 5:1)
-
PhaC4
PhaC3
PCT2
LDH2
Buffer T4 DNA ligase (10X)
2 µL
2 µL
2 µL
2 µL
T4 DNA ligase
0.5 µL
0.5 µL
0.5 µL
0.5 µL
Vector
0.96 µL
0.96 µL
0.96 µL
0.96 µL
Insert (gene)
7.9 µL
8.05 µL
3.91 µL
2.2 µL
H20 (qs 20 µL)
8.63 µL
8.49 µL
12.6 µL
14.3 µL
10 min incubation at 22 °C
Transformation
Thaw 50 µL bacteria (E. coli DH5-α) chemo-competent
Add 5 µL DNA (mix ligation)
Incubation 20 min on ice
Heat shock: 45 s at 42 °C
Incubation 3 min on ice
Add 950 µL LB
Incubation 1 h at 37 °C with shaking
Centrifugation 1 min at 4000 rpm
Throw away supernatant
Resuspension with the rest of supernatant (around 50 µL)
Spread on plates LB Chloramphenicol (30 µg/mL)
26th of August
Preparation plates LBC (30 µg/µL)
Ligation (ratio 1:1)
-
PhaC4
PCT2
PhaC3
LDH2
Buffer T4 DNA ligase (10X)
1 µL
1 µL
1 µL
1 µL
T4 DNA ligase
0.5 µL
0.5 µL
0.5 µL
0.5 µL
Vector
2.39 µL
2.39 µL
2.39 µL
2.39 µL
Insert (gene)
3.96 µL
1.95 µL
4.02 µL
1.11 µL
H20 (qs 10 µL)
2.15 µL
4.15 µL
2.08 µL
5.0 µL
Incubation 16h 16 °C
28th of August
Transformation E. coli DH5-α
Thaw 25 µL chemo-competent bacteria
Add 5 µL DNA (mix ligation) (2 µL of pSB1C3-mRFP from clone A as positive transformation control)
Incubation 20 min on ice
Heat shock: 45 s at 42 °C
Incubation 3 min on ice
Add 900 µL LB
Incubation 1h20 at 37 °C with shaking
Centrifugation 1 min 30 s at 5000 rpm
Throw away all the supernatant
Resuspension with 100 µL LB medium
Spread on plates LB Chloramphenicol (30 µg/mL)
Incubation overnight 37 °C
Week 9
29th of August
The GFP gene contained in Gen4_BNR vector (provided by Cécile) was amplified by PCR with Q5 DNA polymerase with primers allowing to add the prefix and suffix regions at both sides of the amplified region.
Mix
10 µL Q5 reaction buffer 5X
1 µL dNTP mix
2.5 µL of Forward Primer (iG063)
2.5 µL of Reverse Primer (iG064)
1 µL DNA template
0.5 µL Q5 DNA polymerase
32.5 µL Nuclease-free water (qs 50 µL)
PCR program:
98 °C 30 s
(98 °C 30 s, 55 °C 30 s, 72 °C 1 min) 35 times
72 °C 2 min
10 °C ∞
After checking on agarose 1 % gel, we determined that the GFP was successfully amplified.
Digestion
The amplified fragment was digested by EcoRI and PstI during 1 h at 37 °C. The reaction was performed in NEB CutSmart buffer.
The digested mix was purified by using the kit Wizard SV Gel and PCR Clean-Up System (Ref: A9282, Promega).
30th of August
Ligation
The GFP fragment was then ligated separately in pSEVA 224 or pSEVA 2311 digested by E+P in a ratio 5:1.
The ligation mix was stored at -20 °C until transformation.
31th of August
September 2016
Week 9
2nd of September
Digestion
Genes
Quantity of DNA (µL)
Restriction Enzymes (µL)
Buffer
Water
Digest NEB 2.1 (µL)
qs 50 µL (µL)
LDH2 (109.5 ng/µL)
10
EcoR1 HF : 1 µL
5
33
Pst1 :1 µL
PCT2 (56.5 ng/µL)
20
EcoR1 HF : 1 µL
5
23
Pst1 :1 µL
PhaC3 (63.9 ng/µL)
17.5
EcoR1 HF : 1 µL
5
25.5
Pst1 :1 µL
PhaC4 (88.1 ng/µL)
12.5
EcoR1 HF : 1µL
5
30.5
Pst1 :1µL
Vector pSB1C3 (240.3 ng/µL)
4.5
EcoR1 HF : 1 µL
5
38.5
Pst1 :1 µL
Incubation 1 h at 37 °C and after incubation 20 min at 80 °C.
Gel Extraction
50 mL of Agarose 1 % with 2.5 µL of midorigreen.
Migration 25 min at 100 mV.
*Digestion mixes with EcoRI and PstI were loaded on the gel.
3rd of September
DNA extraction from agarose gel
Kit: Wizard SV Gel and PCR Clean-Up System (Ref: A9282, Promega)
pSB1C3 => 6.4 ng/µL
LDH2 => 4.8 ng/µL
PCT2 => 2.9 ng/µL
PhaC3 => 4 ng/µL
PhaC4 => 5.8 ng/µL
DNA concentration by water evaporation
pSB1C3 => 7.1 ng/µL
LDH2 => 6.5 ng/µL
PCT2 => 5.5 ng/µL
PhaC3 => 4.0 ng/µL
PhaC4 => 5.0 ng/µL
DNA concentration still seems not efficient
DNA transformation
Thaw 50 µL chemo-competent DH5-α E. coli
Add 5 µL DNA (mix ligation with pSEVA 2311 or 224)
Incubation 25 min on ice
Heat shock: 45 s at 42 °C
Incubation 3 min on ice
Add 250 µL LB
Incubation 1h15 at 37 °C with shaking
Spread on plates LB Kanamycin (50 µg/mL) (50 µL of bacteria on one plate and 10 µL on another one for each vector)
Incubation overnight 37 °C
4th of September
Ligation in pSB1C3
-
LDH2 (6.5 ng/µL)
PCT2 (6.5 ng/µL)
PhaC3 (4.0 ng/µL)
PhaC4 (5.0 ng/µL)
Vector
4.2 µL
4.2 µL
4.2 µL
4.2 µL
Insert (gene)
2.8 µL
0.5 µL
0.5 µL
0.5 µL
Buffer T4 DNA ligase (10X)
2.39 µL
2.39 µL
2.39 µL
2.39 µL
T4 DNA ligase
3.96 µL
1.95 µL
4.02 µL
1.11 µL
H20 (qs 10 µL)
2.15 µL
4.15 µL
2.08 µL
5.0 µL
Incubation 1 h at 16 °C
Poor plates LBC (30 µg/mL chloramphenicol)
Transformation
Thaw 25 µL chemo-competent DH5-α E. coli
Add 4 µL DNA (mix ligation)
Incubation 25 min on ice
Heat shock: 45 s at 42 °C
Incubation 3 min on ice
Add 225 µL LB
Incubation 1 h at 37 °C with shaking
Spread 120 µL on plates LBC
Incubation overnight 37 °C
Week 10
7th of September
Digestion of genes
See complete version of the notebook (pdf) for details on the conditions.
Incubation 1 h at 37 °C and after incubation 20 min at 80 °C.
8th of September
Digestion of bacbone Igem
Mix: Enzyme Master Mix E+P
0.5 µL NEB 2.1 Buffer
0.5 µL BSA (Bovine Serum Albumin)
0.5 µL EcoR1-HF
0.5 µL Pst1
0.5 µL Dnp1
18 µL H2O
Mix of 4 µL of linearized backbone and 4 µL of master mix enzyme
Incubation of all the solution at 37 °C during 30 min and then, heat deactivation during 20 min at 80 °C.
Ligation
Genes
length genes
Volume Final
Concentration Final
amount for 1 µL (Cf/Vf)
pSB1C3
2037 bp
8 µL
100 ng
25 ng= 2 µL
LDH_V2
1227 bp
50 µL
1095 ng
21.9 ng/µL
PCT_V2
1612 bp
50 µL
1130 ng
22.6 ng/µL
PhaC_V3
1717 bp
50 µL
1118.25 ng
22.365 ng/µL
PhacC_V4
1720 bp
50 µL
1101.25 ng
22.025 ng/µL
Amount calculation of insert for Ligation with NEBio Calculator:
Genes
Amount
Volume for deduced
LDH_V2
15.06 ng
0.7 µL
PCT_V2
19.78 ng
0.9 µL
PhaC_V3
21.07 ng
0.9 µL
PhaC_V4
21.11 ng
0.96 µL
Mix of ligation
2 µL of pSB1C3
Volume calculate of digested insert
1 µL T4 Ligase Buffer
0.5 µL T4 Ligase
qs 10 µL
I put the mix at 16 °C while 30 min and after heat kill for 20 min at 80 °C
Transformation by heat shock
With 2 µL of Ligation mix and 25 µL of Competent Cells.
2 µL of Ligation with 25 µL of competent cells.
Put the mix 20 min in the ice.
Put the mix at 42 °C while 45 seconds
2 min on the ice.
Add 950 µL of LB
Recovery 1 h at 37 °C with agitation
7000 rpm during 1 min
Suppression of the flow-throw
Resuspend with the remaining
Spread on plates (put the simple in the middle of the plates)
Incubation overnight 37 °C
Plates preparation
1 plates = 25mL of LB Agar + 15 µL of chloramphenicol [30 µg/mL]
Preculture
3 mL LB liquid medium + 1 µL E. coli DH5-α bacteria from stock at -80 °C
9th of September
Preparation of chemo-competent E. coli DH5-α by heat shock
WORK STERILE
Launch a pre-culture of the strain in LB at 37 ° C
Inoculate 25 mL of LB 1/50 (500 μl) with the pre-culture
Let it growth at 37 ° C with stirring until OD(600 nm) = 0.5-0.7
After 1h30, OD= 0.9
For Spectrometer measurements, 1 mL of LB for the blank
A dilution was performed to obtain 0.5 OD(600 nm) and the culture was split into 2 falcons with 27.5 mL of culture
Cells are cooled 10 min on ice
Centrifugate culture 6 min at 4000 rpm at 4 °C
Suppression of the flow-through
Dilution of the pellet in 1/2 volume of cold CaCl2 0.1M (13.75 mL)
Cells cooled on ice during 20 min
Culture centrifugated 6 min at 4000 rpm at 4 °C
Suppression of the flow-through
Resuspension the pellet in 1 / 50e of final volume (550 μl final volume) of cold CaCl2 + 10 % glycerol
Vi= 110 µL
The mix is 440 µL CaCl2+ 110 µL glycerol 50 %
I multiplied by 2 the doses for have more mix
22 aliquots of 50 µL bacteria
11th of September
PCR Amplification of LDH2, PCT2, PHAC3, PHAC4 with Q5 DNA polymerase
For 50 µL reaction
25 µL of Q5 Master Mix
2.5 µL of Forward Primer (IG063)
2.5 µL of Reverse Primer (IG064)
0.5 of Template DNA
19.5 of H2O
PCR program
98 °C 30 s
(98 °C 30 s, 55 °C 30 s, 72 °C 1 min) 35 times
72 °C 2 min
10 °C ∞
PCR Colonie LDH2 with One Taq
For 25 µL to 1 Reaction --> Vf= 125 µL
Mix
25 µL of One Taq Standard Reaction Buffer
dNTPs = 2.5 µL
2.5 µL of Forward Primer (IG001)
2.5 µL of Reverse Primer (IG002)
0.625 µL of One Taq DNA Polymerase
91.9 µL of Nuclease Free Water
PCR program
94 °C 30 s
(94 °C 30 s, 50 °C 1 min, 68 °C 1 min 45 sec) 30 times
68 °C 5 min
10 °C ∞
Week 11
12st of September
Miniprep colonies LDH
3 µL of LB
Take one of the colony in the plates
Incubated at 37 °C overnight
Digestion Backbone IGEM pSB1C3 linearized by E+P
Mix: Enzyme Master Mix E+P for 22 reaction
2 µL NEB 2.1 Buffer
2 µL BSA (Bovine Serum Albumin)
2 µL EcoRI-HF
2 µL PstI
2 µL DnpI
72 µL H2O
16 µL of linearized backbone and 16 µL of the MasterMix enzyme
Incubation at 37 °C during 30 min
Enzyme inactivation with 20 min incubation at 80 °C.
Precultures
5 mL LB medium
+ 5 µL antibiotic (Kanamycin 50 mg/mL for pSEVA 212S, Spectinomycin 50 mg/mL for pSEVA 424 and Gentamycin 10 mg/mL for pSEVA 628S)
+ some bacteria (sent as bacteria aliquots in LB medium by de Lorenzo)
Incubation 37 °C overnight
13st of September
Miniprep of pSEVA plasmids
2 aliquots stocks in 50 % glycerol stored at -80 °C
GenElute Plasmid Miniprep Kit (Sigma, PLN350-1KT)
3 DNA vectors: pSEVA 424 (equivalent to the pSEVA 224 but with spectinomycin resistance gene instead of kanamycin, pSEVA 212S and pSEVA 628S used for genes deletion in genomic DNA in Pseudomonas putida)
Digestion of the “old” gene by E+P
Ratio 1 : 2
Genes
Amount
Volume for deduced
LDH_V2
30.12 ng
1.37 µL
PCT_V2
39.57 ng
1.71 µL
PhaC_V3
42.15 ng
1.88 µL
PhaC_V4
42.22 ng
1.92 µL
Ligation mix
2 µL of pSB1C3
Volume calculate of digested insert
1 µL T4 Ligase Buffer
0.5 µL T4 Ligase
qs 10 µL
I put the mix at 16 °C while 30 min and after heat kill for 20 min at 80 °C
Minipreps DNA Purification System kit “Wizard Plus”
Take 2 µL of overnight culture
Centrifuge at top speed for 1 min
Suppression of the flow-throw
Resuspend pellet with 250 µL of Cell Resuspension Solution
Add 250 µL of Cell Lysis Solution and invert 4 times to mix
Add 10 µL of Alkaline Protease Solution and invert 4 time to mix
Incubate 5 min at room temperature
Add 350 µL of Neutralization Solution and invert 4 times to mix
Centrifuge at top speed for 10 min at room temperature
Insert Spin Column into Collection Tube
Decant cleared lysate into Spin Column
Centrifuge at top speed for 1 min
Discard flowthrough
Add 750 µL of Wash Solution and centrifuge at top speed 1 min
Discard flowthrough
Repeat with 250 µL of Wash Solution
Centrifuge at top speed for 2 min
Transfert Spin Column to 1.5 mL tube
Add 100 µL of Nuclease free water to the Spin Column
Centrifuge at the top speed for 1 min
Discard column
Ratio 1 : 5
Genes
Amount
Volume for deduced
LDH_V2
75.29 ng
3.44 µL
PCT_V2
98.92 ng
4.38 µL
PhaC_V3
105.4 ng
4.71 µL
PhaC_V4
105.5 ng
4.79 µL
Purification of Amplification PCR LDH2, PCT2, PHAC3, PHAC4
Nanodrop
For the genes
Genes
Concentration (ng/µL)
Ratio 260/280
LDH2
62.7
1.78
PCT2
52.8
1.79
PHAC3
50.3
1.76
PHAC4
69.7
1.78
For the LDH colonies
Colonies
Concentration (ng/µL)
Ratio 260/280
LDH_n°1
70
1.92
LDH_n°2
31
2.06
Digestion “New” genes LDH2, PCT2, PHAC3, PHAC4
Digestion by EcoR1-HF+PSt1
Genes
Quantity of DNA (µL)
Restriction Enzymes
Buffer Digest NEB 2.1 (µL)
H2O
LDH2 (62.7 ng/µL)
16
1 µL EcoR1-HF
5
27
1 µL Pst1
PCT2 (52.8 ng/µL)
19
1 µL EcoR1-HF
5
24
1 µL Pst1
PHAC3 (63.9 ng/µL)
16
1 µL EcoR1-HF
5
27
1µL Pst1
PhAC4 (69.7 ng/µL)
14.5
1 µL EcoR1-HF
5
28.5
1 µL Pst1
pSB1C3 (240.3 ng/µL)
4.5
1 µL EcoR1-HF
5
38.5
1 µL Pst1
Other digestion
Genes
Quantity of DNA (µL)
Restriction Enzymes
Buffer Digesr NEB 2.1 (µL)
H2O
PHAC3
16
1 µL Xba1
5
27
1 µL Pst1
PHAC4
14.5
1 µL Xba1
5
28.5
1 µL Pst1
PCT2
19
1 µL EcoR1-HF
5
24
1 µL Spe1
Incubation at 37 °C during 30 min and then, enzymes deactivation by heat kill for 20 min at 80 °C.
PCR Purification
Nanodrop Vf = 50 µL
Genes
Concentration (ng/µL)
Ratio 260/280
LDH_V2
62.7
1.78
PCT_V2
52.8
1.79
PHAC_V3
50.3
1.76
PHAC_V4
69.7
1.78
14st of September
The promoter region of the vectors pSEVA 224 and pSEVA 2311 were amplified by PCR with Q5 DNA polymerase with primers allowing to add the prefix and suffix regions at both sides of the amplified region.
Mix
10 µL Q5 reaction buffer 5X
1 µL dNTP mix
2.5 µL 10 µM Forward Primer (iG063)
2.5 µL 10 µM Reverse Primer (iG064)
1 µL DNA template
0.5 µL Q5 DNA polymerase
32.5 µL Nuclease-free water (qs 50 µL)
PCR program:
98 °C 30 s
(98 °C 30 s, 55 °C 30 s, 72 °C 1 min) 35 times
72 °C 2 min
10 °C ∞
After checking on agarose 1 % gel, we determined that the fragments were successfully amplified.
Digestion
The amplified fragments were digested by EcoRI and PstI during 1 h at 37 °C. The reaction was performed in NEB CutSmart buffer.
The digested mixes were purified by using the kit Wizard SV Gel and PCR Clean-Up System (Ref: A9282, Promega).
Design Primer for Gibson Assembly -> PR_IG0075 -> 81
15st of September
Ligation
The 2 fragments were then ligated separately in linear pSB1C3 provided by IGEM Headquarters in a ratio 5:1.
The ligation mix was used for transformation in E. coli DH5-alpha.
Transformation
Thaw 25 µL chemo-competent DH5-alpha E. coli
Add 3 µL DNA (mix ligation)
Incubation 20 min on ice
Heat shock: 45 s at 42 °C
Incubation 3 min on ice
Add 225 µL of LB medium
Incubation 1 h at 37 °C with shaking
Spread 120 µL on plates LBC
Incubation overnight 37 °C
16th of September
PCR colony
Mix (25 µL total volume reaction):
12.5 µL DreamTaq PCR Mastermix 2X (ThermoScientific)
2.5 µL 10X DreamTaq Green Buffer (ThermoScientific)
0.25 µL Forward Primer (0.5 µM) iG001
0.25 µL Reverse Primer (0.5 µM) iG002
9.5 µL H2O filtrated (qs 25 µL)
+ clones
Agarose gel 1 %
Precultures
Positive clones selected from the agarose gel results were picked and used to inoculate 3 mL of LBC (25 µg/mL chloramphenicol) medium.
Observation
All the precultures have grown
PCR colony
Mix (25 µL total volume reaction):
10 µL DreamTaq PCR Mastermix 2X (ThermoScientific)
1.5 µL Forward Primer (0.5 µM) iG001
1.5 µL Reverse Primer (0.5 µM) iG002
6 µL H2O filtrated (qs 25 µL)
+ clones
Migration map
October 2016
Gel shown the last results corresponding to the genes selected
The promoters isolated from vectors pSEVA2311 and pSEVA224 were also sent to iGEM HQ after checking by sequencing.
For information, PhaC3 and PCT2 without RBS were successfully cloned in pSB1C3 and sent to iGEM HQ (verified by sequencing)
Check the rest of our note book (.pdf)
Genopole
IDT
iSSB
UEVE
IBISC
République Française
Crous
New-England Biolab
Geneinous
Mathworks
MIE
Papeterie GPS
Evry2016.
