Team:Evry/Design/PLAsystem
Homepage
Visit the iGEM website!
Feel free to contact us!
E-mail
Twitter
Facebook
Instagram
Let's PLAy project - Bioproduction of PLA
Show menu
Home
Project
Description
Design
Modeling
Experiments
Notebook
Improvements
Achivements
Parts
Medal fulfillments
Results
Interlab
Results
Protocol
Lab book
Safety
Project safety
Lab safety
Shipment safety
Human practices
Integrated HP
Engagements
iGEM events
Team
Our team
iGEM in Evry
Collaborations
Campus
Sponsors
Attributions
Home
Project
Achivements
Interlab
Safety
Human practices
Team
Attributions
Homepage
Project
Design
The PLA system
The PLA system
Starting from the simple idea of producing a biological alternative to regular plastic, we had to think about how to engineer bacteria to reach this goal in the most efficient way. As explained in Design , we first carefully chose our chassis; we proceeded with a basic analysis of the pathways to produce PLA; and we designed the implementation of the necessary genes.
Choosing a chassis
In literature, bioproduction of PLA had already been reported in E. coli. However, we detected several criteria that would set other organisms as better chassis for PLA synthesis, and thus accomplish our purpose: improve PLA bioproduction.
Over different potential candidates, we set P. putida KT2440 as the best chassis, as argumented in Project Description .
Basic pathway analysis
Metabolism of P. putida KT2440 has been well described in literature [1] . Taking into account the reaction steps indicated in Jung et al., 2010 [2] for bioproduction of PLA in E. coli, we studied P. putida KT2440 pathways and clarified which were the minimal genes to express to have PLA. The schema in Figure 1 indicates the reactions that we would need to introduce in our chassis.
From this, we proceeded to think about the design of our biobricks, which would contain the PLA involved genes.
Figure 1. Engineered Pct and PhaC. Propionate Co-A transferase and PHA synthase are the two genes needed for PLA production from lactic acid.
Biobrick design
Clearly, we needed two genes for PLA biosynthesis: an evolved Propionate CoA transferase who can take lactate as subtract and convert it into Lactyl-CoA, and an evolved PHA synthase which can polymerize monomer of Lactyl-CoA into PLA polymer. To do so, we started by looking for the sequence of Clostridium propionicum propionate-CoA transferase and we codon optimized the sequence for our bacteria P. putida. The wild type enzyme normally catalyzes the reaction: acetyl-CoA + propanoate <--> acetate + propanoyl-CoA. We introduced one amino acid mutation (A243T) found to allow efficiently convert lactate into lactyl-CoA [3] .
So, we used the same method for PHA synthase, we found two potential enzymes from different organisms. The first one was used in the iGEM Yale 2013 project (Pseudomonas resinovorans PHA synthase 1) and the second is Pseudomonas sp. MBEL 6-19 PHA synthase 1. We optimized their sequence for P. putida and performed quadruple important amino acid mutation (E130D,S325T, S477F, and Q481K) found for altering the substrate specificity and having enhanced activity towards (D)-lactyl-CoA [3] .
Finally, to increase the production of our compounds, we considered increasing the initial subtract as an important step. We decided to implement an evolved D-lactate dehydrogenase which could be useful for our system. We based the design on an article reporting a engineering NADH-dependent dehydrogenase (D-LDH*) from Bacillus delbrueckii 11842 that can super-efficiently utilize NADPH and NADH as cofactors. We looked for the sequence, we codon optimized it for P. putida and did the tree point amino acids mutation (D176S, I177R, F178T) [4] .
Implementation of the basic system
Once our basic sequences were designed, we had to think about how to express them in the cell. For that, we started with the design of our plasmid. In order to choose the best plasmid for P. putida, we asked for advice to Víctor De Lorenzo lab (CNB-CSIC) , which are working in Pseudomonas putida KT2440. He recommended the use of two SEVA (Standard European Vector Architecture) plasmids [5] , which were:
pSEVA224: IPTG inducible promoter and Kanamycin resistant gene
pSEVA2311: Cyclohexanone inducible promoter and Kanamycin resistant gene
An efficiently way of building our plasmid was indeed making an operon with the two genes (PhaC and Pct). According to their length we had to synthetize them separately with our IDT gBlocks. In the synthesis, we also included RBS from Pseudomonas putida genomic DNA before and close to the genes to prevent missing initial codon during translation. Finally, we added biobrick prefix and suffix in the synthesis. For the next steps, we worked in parallel with the two versions of the PhaC gene that we had.
Standard assembly method
Thinking on the design of the operon, we decided to put the PhaC gene before the Pct, as the polymerization is the bottleneck of our system and we assumed that proximity to the promoter was more necessary than for Pct, as it could provide more expression.
We wanted to put this operon in pSEVA224 plasmid in order to express it in P. putida. Figure 3 is a map of the region between promoter and terminator in pSEVA224.
We included the operon in this region by digesting the plasmid with Ecor1 and Pst1. In the final construct, the operon is under control of the inducible promoter of pSEVA224. Figure 4 is the final plasmid map containing the operon.
Figure 2. Standard assembly for construction of operon. We would use restriction enzymes to construct our PhaC-Pct operon from the separate fragments on gBlocks.
Figure 3. pSEVA 224: region between promoter and terminator.
Figure 4. Designed plasmid. It is the pSEVA 224 vector, with the PhaC-Pct operon inserted by standard digestion assembly.
For the D-LDH* gene, we aimed to express it in the same cell to increase the yield of PLA. The constructed plasmid pSEVA 224 with our operon was >8kb, for what we decided that it was more convenient to use a different plasmid to express LDH. The first strategy was to use pSEVA2311 plasmid and take profit of playing with two different inducers to control when we wanted more precursor (lactate). To do so, we would change the antibiotic resistance gene, so we could co-transform Pseudomonas and select cells by using the two different antibiotics.
For the synthesis of this gene, we included a constitutive promoter (pTN8-P1) from P. putida genomic DNA in upstream of the sequence. Indeed, separated from the translation unit with BamH1 restriction site, this design allowed us the possibility to choose between expressing the gene under control of the SEVA promoter or using a constitutive one.
In parallel, we ask also to De Lorenzo lab another plasmid, the pSEVA424 which is a plasmid with the save IPTG inducible promoter as pSEVA224 but with a Spectinomycin resistant gene. Indeed, using a single inductor instead of two could be better for the cell in terms of avoiding toxicity. This would allow us to select cell transformed with both plasmid using Kanamycin and Spectinomycin antibiotic. Figure 6 is the plasmid map of the LDH* in pSEVA424.
Figure 5. Design of D-LDH* gene block.
Figure 6. Designed plasmid for LDH*. It is the pSEVA424 vector with the LDH* designed block inserted by standard digestion assembly.
Finally, we would have Pseudomonas putida co-transformed with two plasmids: one would allow to take lactate as the substrate and convert it efficiently into lactyl-CoA, and then polymerize the monomer of lactyl-coA into PLA homopolymer; the other one would allow to increase lactate production in the cell and, in consequence, the PLA yield.
To go further, we extended the pathway analysis by performing computational Flux Balance Analysis , and we also designed a dynamic regulation system to optimize the PLA production, and studied it in silico.
References
Nikel, P.I. et al. Biotechnological domestication of pseudomonads using synthetic biology. Nature Reviews Microbiology 12, 368–379 (2014)
Jung, Y.K. et al. Tailor-made type II Pseudomonas PHA synthases and their use for the biosynthesis of polylactic acid and its copolymer in recombinant Escherichia coli. Appl Microbiol Biotechnol 90(2):603-14 (2011).
Yang, T. H. et al. Biosynthesis of Polylactic Acid and Its Copolymers Using Evolved Propionate CoA Transferase and PHA Synthase. Biotechnol Bioeng. 1;105(1):150-60 (2010)
Meng, H. et al. Engineering a D-lactate dehydrogenase that can super-efficiently utilize NADPH and NADH as cofactors Sci Rep 6: 24887( 2016)
Silva-Rocha, E. et al. The Standard European Vector Architecture (SEVA): a coherent platform for the analysis and deployment of complex prokaryotic phenotypes. Nucleic Acids Res. 41, D666–D675 (2013)
Genopole
IDT
iSSB
UEVE
IBISC
République Française
Crous
New-England Biolab
Geneinous
Mathworks
MIE
Papeterie GPS
Evry2016.
