<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page_wrap"><HEADER><DIV id="logo_evryigem"><A href="https://2016.igem.org/Team:Evry" title="Homepage"><SPAN>Homepage</SPAN></A></DIV><DIV id="bar-1"><DIV id="igem-logo"><A href="https://igem.org" title="Visit the iGEM website">Visit the iGEM website!</A></DIV><DIV id="social"><P>Feel free to contact us! </P><UL><LI><A class="email" href="mailto:evryigem2016@gmail.com?Subject=Wiki " title="E-mail - evryigem2016@gmail.com">E-mail</A></LI><LI><A class="twitter" href="https://twitter.com/iGEM_Evry" title="Twitter - @iGEM_evry">Twitter</A></LI><LI><A class="facebook" href="https://www.facebook.com/iGEM.Evry.2016/" title="Facebook - /iGEM.Evry.2016/">Facebook</A></LI><LI><A class="insta" href="https://www.instagram.com/igem_evry/" title="Instagram - @igem_evry">Instagram</A></LI></UL></DIV><DIV id="letsplay"><EM>Let's PLAy project</EM> - Bioproduction of PLA</DIV></DIV><NAV id="navbar"><DIV id="nav"><UL id="nav_menu"><LI class="cat_menu show_more"><A href="#">Show menu</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI class="cat_menu dropdown xproj"><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Improvements">Improvements</A></LI></UL><LI class="cat_menu dropdown xachiv"><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Medals">Medal fulfillments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Results">Results</A></LI><LI class="cat_menu dropdown xilab"><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab">Results</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Labbook">Lab book</A></LI><LI class="cat_menu dropdown xsafe"><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety">Project safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Lab">Lab safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Shipment">Shipment safety</A></LI><LI class="cat_menu dropdown xhp"><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Integrated_Practices"> Integrated HP</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Engagement">Engagements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/HP/Events">iGEM events</A></LI><LI class="cat_menu dropdown xteam"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Previous_Evry">iGEM in Evry</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Campus">Campus</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Sponsors">Sponsors</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></DIV></NAV></HEADER><UL id="show_more"><LI><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI class="lastl"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></UL><MAIN><DIV id="content-wrap"><DIV id="breadcrumbs"><UL><LI><A href="https://2016.igem.org/Team:Evry">Homepage</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Description">Project</A></LI><LI>Note book</LI></UL></DIV><DIV id="content"><H1 class="red">Note book</H1><SPAN style="font-size:1.15em; "><A href="https://static.igem.org/mediawiki/2016/3/35/T--Evry--LabBook_iGEM_complete.pdf" target="_blank"><U>Complete version of our note book (.pdf)</U></A></SPAN><DIV id="calendar"><TABLE class="center"><TBODY><TR><TH>Mon</TH><TH>Tue</TH><TH>Wed</TH><TH>Thu</TH><TH>Fri</TH><TH>Sat</TH><TH>Sun</TH></TR><TR><TD colspan="7">July</TD></TR><TR><TD><A href="#w1">Week 1</A></TD><TD><A href="#d407">4</A></TD><TD><A href="#d507">5</A></TD><TD><A href="#d607">6</A></TD><TD><A href="#d707">7</A></TD><TD><A href="#d807">8</A></TD><TD><I>9</I></TD><TD><I>10</I></TD></TR><TR><TD><A href="#w2">Week 2</A></TD><TD><I>11</I></TD><TD><A href="#d1207">12</A></TD><TD><A href="#d1307">13</A></TD><TD><A href="#d1407">14</A></TD><TD><A href="#d1507">15</A></TD><TD><A href="#d1607">16</A></TD><TD><I>17</I></TD></TR><TR><TD><A href="#w3">Week 3</A></TD><TD><A href="#d1807">18</A></TD><TD><A href="#d1907">19</A></TD><TD><A href="#d2007">20</A></TD><TD><A href="#d2107">21</A></TD><TD><A href="#d2207">22</A></TD><TD><A href="#d2307">23</A></TD><TD><I>24</I></TD></TR><TR><TD><A href="#w4">Week 4</A></TD><TD><A href="#d2507">25</A></TD><TD><A href="#d2607">26</A></TD><TD><A href="#d2707">27</A></TD><TD><A href="#d2807">28</A></TD><TD><I>29</I></TD><TD><I>30</I></TD><TD><A href="#d3107">31</A></TD></TR><TR><TD colspan="7">August</TD></TR><TR><TD><A href="#w5">Week 5</A></TD><TD><I>1</I></TD><TD><A href="#d208">2</A></TD><TD><A href="#d308">3</A></TD><TD><A href="#d408">4</A></TD><TD><I>5</I></TD><TD><I>6</I></TD><TD><I>7</I></TD></TR><TR><TD><A href="#w6">Week 6</A></TD><TD><A href="#d808">8</A></TD><TD><A href="#d908">9</A></TD><TD><A href="#d1008">10</A></TD><TD><A href="#d1108">11</A></TD><TD><A href="#d1208">12</A></TD><TD><A href="#d1308">13</A></TD><TD><A href="#d1408">14</A></TD></TR><TR><TD><A href="#w7">Week 7</A></TD><TD><I>15</I></TD><TD><A href="#d1608">16</A></TD><TD><A href="#d1708">17</A></TD><TD><A href="#d1808">18</A></TD><TD><A href="#d1908">19</A></TD><TD><A href="#d2008">20</A></TD><TD><I>21</I></TD></TR><TR><TD><A href="#w8">Week 8</A></TD><TD><A href="#d2208">22</A></TD><TD><A href="#d2308">23</A></TD><TD><A href="#d2408">24</A></TD><TD><A href="#d2508">25</A></TD><TD><A href="#d2608">26</A></TD><TD><I>27</I></TD><TD><A href="#d2808">28</A></TD></TR><TR><TD><A href="#w9">Week 9</A></TD><TD><A href="#d2908">29</A></TD><TD><A href="#d3008">30</A></TD><TD><I>31</I></TD></TR><TR><TD colspan="7">September</TD></TR><TR><TD><A href="#w9b">Week 9</A></TD><TD><I>1</I></TD><TD><A href="#d209">2</A></TD><TD><A href="#d309">3</A></TD><TD><A href="#d409">4</A></TD></TR><TR><TD><A href="#w10">Week 10</A></TD><TD><I>5</I></TD><TD><I>6</I></TD><TD><A href="#d709">7</A></TD><TD><A href="#d809">8</A></TD><TD><A href="#d909">9</A></TD><TD><I>10</I></TD><TD><A href="#d1109">11</A></TD></TR><TR><TD><A href="#w11">Week 11</A></TD><TD><A href="#d1209">12</A></TD><TD><A href="#d1309">13</A></TD><TD><A href="#d1409">14</A></TD><TD><A href="#d1509">15</A></TD><TD><A href="#d1609">16</A></TD><TD><I>17</I></TD><TD><I>18</I></TD></TR><TR><TD colspan="8"><A href="https://static.igem.org/mediawiki/2016/c/cd/T--Evry--end-labbook.pdf" target="_blank">Rest of the lab book (.pdf)</A></TD></TR></TBODY></TABLE></DIV><DIV id="actual_notebook"><H1>July 2016</H1><DIV class="grid-2"><DIV><H3 class="yellow">Week 1</H3><H5 class="blue">4<SUP>th</SUP> of July</H5><H6>Bottles of LB and LB-Agar (200 mL and 400 mL)</H6><UL><LI>LB-Agar: 14 g for 400 mL of water</LI><LI>LB: 8 g for 400 mL of water</LI><LI>Autoclave</LI></UL><H6>Resistance test of <I>P. putida</I></H6><P>2 plates of LB-Agar with either Kanamycin (50 µg/mL) or Spectinomycin (50 µg/mL).</P><H6>Preparation of HEPES solution 0.5 M (for electroporation)</H6><UL><LI>11 g of HEPES (solid) in 64 mL of water</LI><LI>NaOH (0.1 M) for pH7</LI></UL><H6>Refresh <I>P. putida</I> (the Petri dish got old)</H6><P><I>E. coli</I> C118 with plasmids pSEVA2311 (KanR) and pSEVA224 (KanR) were separately spread in LB. </P><P><UL><LI>cells with pSEVA2311  were observed</LI><LI>cells with pSEVA224 were not observed!!</LI><LI>Jonathan asked for another sample</LI></UL></P><H5 class="blue">5<SUP>th</SUP> of July</H5><P><UL><LI>LB culture of <I>E. coli</I> C118 </LI><LI>HEPES filtration</LI><LI>Check the inventory of the fridge</LI><LI>Plate of <I>P. putida</I> at 30 °C and 37 °C → We didn’t do this step</LI><LI>Liquide culture (5 mL of preculture) of <I>P. putida</I> to have competent cells</LI></UL></P><H5 class="blue">6<SUP>th</SUP> of July</H5><P>Preculture for transformation (5 mL) of <I>E. coli</I> DH5 α</P><H5 class="blue">7<SUP>th</SUP> of July</H5><H6>Miniprep of <I>E. coli</I> C118 with pSEVA2311</H6><UL><LI>Protocol “Midiprep” with 50 mL of culture</LI><LI>C = 80 ng/µL in 200 µL of elution buffer</LI><LI>OD<SUB>260</SUB> = 1.616</LI><LI>OD<SUB>280</SUB> = 0.8</LI></UL><H6>Transformation of <I>Pseudomonas</I> with pSEVA2311</H6><UL><LI>40 mL of culture (competent cell):</LI><LI>When OD<SUB>600nm </SUB>= 0.6</LI></UL><UL style="margin-left:10%"><LI>20 mL for Heat Shock</LI><LI>20 mL for Electroporation</LI></UL><P>After electroporation and heat-shock spread on Amp-Plates</P><H5 class="blue">8<SUP>th</SUP> of July</H5><P>The transformation of 07/07/2016 didn’t work because the plasmid given by Robert didn’t match with <I>Pseudomonas</I>→ Second transformation (Heat shock and electroporation)</P><H6>Preparation of  plates (LB + Antibiotics)</H6><UL><LI>10 plates LB-Agar, Kanamycin (1 µL/mL)</LI><LI>10 plates LB-Agar, Chloramphenicol (1 µL/mL)</LI><LI>10 plates LB-Agar, Ampicillin (1 µL/mL)</LI><LI>10 plates LB-Agar, without antibiotic</LI></UL><H3 class="yellow">Week 2</H3><H5 class="blue">12<SUP>th</SUP> of July</H5><H6>Preparation of M9-media (10X, V= 600 mL) with different kind of C-sources</H6><UL><LI>Glucose → <SPAN class="green">DONE</SPAN></LI><LI>Glycerol → <SPAN class="green">DONE</SPAN></LI><LI>Fructose → <SPAN class="red">NO YET</SPAN></LI><LI>Benzoate → <SPAN class="red">NO YET</SPAN></LI><LI>Lactic acid → <SPAN class="red">NO YET</SPAN></LI><LI>Methane → <SPAN class="red">NO YET</SPAN></LI></UL><TABLE class="center" style="width:80%"><TBODY><TR><TH>Compounds</TH><TH>Cf</TH><TH>Masses (g) or volumes (ml)</TH></TR><TR><TD>CaCl2.H<SUB>2</SUB>O</TD><TD>1 mM</TD><TD>0.0882 g</TD></TR><TR><TD>MgSO4</TD><TD>20 mM</TD><TD>1.44 g</TD></TR><TR><TD>FeSO4.H<SUB>2</SUB>O</TD><TD>0.1 mM</TD><TD>0.01668 g</TD></TR><TR><TD>M9 (salt)</TD><TD>10 X</TD><TD>67.68 g</TD></TR><TR><TD>Glycerol (100 %) *</TD><TD>8 % v/v</TD><TD>60.48 g</TD></TR><TR><TD>Casamino acid</TD><TD>2 % w/v</TD><TD>12 g</TD></TR><TR><TD>Thiamine 10 mg/mL **</TD><TD>10 µg/mL</TD><TD>(0.6 g)</TD></TR><TR><TD>Uracil</TD><TD>200 µg/mL</TD><TD>0.12 g</TD></TR><TR><TD>Leucine</TD><TD>300 µg/mL</TD><TD>0.18 g</TD></TR><TR><TD>NaOH</TD><TD>pH 6.6</TD><TD>-</TD></TR></TBODY></TABLE><P class="figure"><U>Table 1.</U> *Glucose : 59.22 g; 
	** Thiamine has not been added but it has to be before the preculture of <I>P. putida</I>.</P><SPAN class="red">→ problem : precipitation </SPAN><H5 class="blue">13<SUP>th</SUP> of July</H5><H6>Preparation of the deletion process – Transformation of <I>E. coli</I> Mach1 competent cells (CaCl2 treatment) by heat shock</H6><UL><LI>Plasmids pKD46, pCP20 and pKD4</LI><LI>3 tubes in which 25 µL of cells were mixed with 1.5 µL of each plasmid (pKD4, pCP20 or pKD46)</LI><LI>Incubation on ice for 20 min</LI><LI>Heat at 42 °C (thermobloc) 1 min</LI><LI>120 µL of LB added to each tube</LI><LI>Incubation: 30 min at 37 °C for the plasmid pKD4, 30 °C for the plasmids pCP20 and pKD46</LI><LI>Spread on plates (Ampicilline 100 μg/mL)</LI><LI>Incubation overnight at 37 °C for pKD4, 30 °C for pCP20 and pKD46</LI></UL><H6>Preparation for the deletion process – Preparation of <I>E. coli</I> DH5 α competent cells</H6><UL><LI>5 mL of preculture of DH5 α were prepared</LI><LI>Incubation overnight at 37 °C</LI></UL><H5 class="blue">14<SUP>th</SUP> of July</H5><UL><LI>pKD46 and pCP20 transformation properly worked</LI><LI>pKD4 didn’t succeed</LI><LI>Preparation of 5 mL culture of pKD46 and pCP20 to purify plasmid</LI><LI>pKD4 since it is not needed for transformation will be used w/o amplification</LI><LI>DH5 α competent cells were prepared and stored</LI></UL><H5 class="blue">15<SUP>th</SUP> of July</H5><H6>Midiprep of pCP20 and pKD46</H6><UL><LI>pCP20: 65 ng/µL</LI><LI>pKD46: 50 ng/µL</LI></UL><H6>Transformation of <I>P. putida</I> KT2440 with pKD46 plasmid to start deletion process</H6><UL><LI>Incubation overnight at 30 °C (Amp Plates)</LI><LI>Spread KT2440 WT and check growth at 37 °C</LI></UL><H5 class="blue">16<SUP>th</SUP> of July</H5><H6>Transformation results</H6><UL><LI>cells grew on Ampicillin at 30 °C</LI><LI>Pick 4 clones from previous plate and spread them in a new plate (Amp with proper concentration)</LI><LI>Incubation at 30 °C</LI></UL><H6><I>Pseudomonas putida</I> : Working concentration of Antibiotics</H6><UL><LI>Kan (50 µg/mL) → 1.7 µL of stock solution/mL</LI><LI>Amp (500 µg/mL) → 5 µL of stock solution/mL</LI><LI>Spec (50 µg/mL) →1 µL of stock solution/mL</LI></UL><P>KT2440 grew well at 37 °C</P><H5 class="blue">17<SUP>th</SUP> of July</H5><H3 class="yellow">Week 3</H3><H5 class="blue">18<SUP>th</SUP> of July</H5><H6>Electroporation of <I>P. putida</I> : pKD46 </H6><UL><LI>Electro-competent cells have been prepared</LI><LI>1.5 µL of DNA were added to ~ 50 µL of electro-competent cells (all the tube)</LI><LI>1 mL of LB was transfer in a 1.5 mL tube </LI><LI>500 µL of the LB were prepare in the pipette-tip and put rapidly in the electro-cuve  after electroporation (at 1800V)</LI><LI>All the cells+LB of the cuve were transferred to the 1.5 mL tube containing LB</LI><LI>Incubation + Shaking  1 h at 30 °C</LI><LI>Centrifugation 3 min 90 rpm</LI><LI>Cells were plated in LB Agar Ampicillin (100 μg/μL) and incubated at 30 °C</LI></UL><H6>Spread C118 (pSEVA224) on plates with Kanamycin</H6><H5 class="blue">19<SUP>th</SUP> of July</H5><P>The transformation (Electroporation of 18/08) didn’t work → maybe the replicon doesn’t match with <I>P. putida</I></P><H6>Colony PCR: to check the transformation of <I>P. putida</I> with p2311 </H6><UL><LI>Tick one colony and with the same tip, spread a little area on LB plate and dissolve the rest in 10 µL ddH<SUB>2</SUB>O</LI><LI>Gently vortex and briefly centrifuge DreamTaq Green PCR Master Mix (2X) after thawing</LI><LI>Place a thin-walled PCR tube on ice and add the following components for each 50 µL reaction</LI></UL><TABLE class="center" style="width:80%"><TBODY><TR><TH>DreamTaq Green PCR Master Mix (2X)</TH><TH>1X</TH><TH>25 µL</TH></TR><TR><TD>Forward primer</TD><TD>0.5 µM</TD><TD>0.25 µL</TD></TR><TR><TD>Reverse primer</TD><TD>0.5 µM</TD><TD>0.25 µL</TD></TR><TR><TD>Colony water</TD><TD>----</TD><TD>2 µL</TD></TR><TR><TD>Water, nuclease-free</TD><TD>----</TD><TD>22.5 uL</TD></TR></TBODY></TABLE><UL><LI>Gently vortex the samples and spin down</LI><LI>Perform PCR using the recommended thermal cycling conditions outlined below</LI></UL><TABLE class="center" style="width:80%"><TBODY><TR><TH>Step</TH><TH>Temperature °C</TH><TH>Time</TH><TH>Number of cycles</TH></TR><TR><TD>Initial denaturation</TD><TD>95 °C</TD><TD>5 min</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95 °C</TD><TD>30 s</TD><TD>30</TD></TR><TR><TD>Annealing</TD><TD>55(Tm)-5 °C</TD><TD>30</TD></TR><TR><TD>Extension</TD><TD>72 °C</TD><TD>1 min/kb</TD></TR><TR><TD>Final extension</TD><TD>72 °C</TD><TD>10 min</TD><TD>1</TD></TR><TR><TD>Pause</TD><TD>4 °C</TD><TD>“00”</TD><TD>---</TD></TR></TBODY></TABLE><UL><LI>Load 5-15 µL of PCR mixture directly on a gel</LI></UL><H6>Electrophoresis to check the transformation</H6><UL><LI>1 % of agarose </LI><LI>Duplicates have been done</LI></UL><P>See complete version of the notebook for details on the conditions.</P><H6>PCR of pKD4</H6><P><STRONG>Mix for 100 µL</STRONG></P><TABLE class="center" style="width:80%"><TBODY><TR><TH colspan="2">Mix 1</TH></TR><TR><TD>Reaction buffer</TD><TD>20 µL</TD></TR><TR><TD>dNTP mix</TD><TD>2 µL</TD></TR><TR><TD>Primers (reverse and forward)</TD><TD>0.5 µL</TD></TR><TR><TD>Template (pKD4)</TD><TD>1 µL</TD></TR><TR><TD>Q5 polymerase</TD><TD>1 µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>Qs 100 µL (75.5 µL)</TD></TR></TBODY></TABLE><H6>Cycles</H6><TABLE class="center" style="width:80%"><TBODY><TR><TH>Temperature</TH><TH>Time</TH><TH>Number of cycles</TH></TR><TR><TD>98 °C</TD><TD>30”</TD><TD>1</TD></TR><TR><TD>90 °C</TD><TD>30”</TD></TR><TR><TD>60 °C</TD><TD>30”</TD><TD>32</TD></TR><TR><TD>72 °C</TD><TD>1’</TD></TR><TR><TD>72 °C</TD><TD>5’</TD><TD>1</TD></TR><TR><TD>4 °C</TD><TD>“00”</TD><TD>---</TD></TR></TBODY></TABLE><H6>Culture of <I>E. coli</I> C118 pSEVA for midi prep’</H6><H5 class="blue">20<SUP>th</SUP> of July</H5><H6>Midiprep’ of pSEVA224 (from <I>E. coli</I>)</H6><P>C= 59.9 ng/µL in 200 µL of elution buffer</P><P><STRONG>DO<SUB>260</SUB>/DO<SUB>280</SUB> = 1.97</STRONG></P><H6>Alternative to delete gene</H6><P>Transformation of <I>Pseudomonas</I> by a PCR product : Kanamycin resistance cassette + homologous region 5’ and 3’ using Heat Shock and Electroporation.</P><UL><LI>See protocols for both transformations</LI><LI>We used 10 µL of DNA</LI></UL><H6>Preparation of 7 plates for <I>Pseudomonas</I> (1.7 µL/mL of kanamycin stock solution)</H6><H6>Liquid culture of <I>E. coli</I> pSEVA224 (2 mL)</H6><UL><LI>2 mL of LB</LI><LI>3.4 µL of Kan - <SPAN class="red">It was a mistake: we should have put 2 µL of Kan (since we used <I>E. coli</I>)</SPAN></LI></UL><H6>Preculture to amplify the vector given by Cyril (BBa.R0010 pLac)</H6><UL><LI><I>E. coli</I> G3A</LI><LI>50 mL of LB</LI><LI>50 µL of Amp</LI></UL></DIV><DIV><H5 class="blue">21<SUP>th</SUP> of July</H5><SPAN class="red">!!! The fridge and freezer went off !!!</SPAN><H6>The transformation of the PCR product (for the deletion process) didn’t worked</H6><UL><LI>We will focus on the construction of the plasmid</LI></UL><H6>Store the culture</H6><P>Storage in glycerol = 1 vol of glycerol + 1 vol of the culture and at -80 °C
	</P><H6>Midiprep of the vector R0010</H6><UL><LI>325 ng/µL</LI><LI>DO<SUB>260</SUB>/DO<SUB>280</SUB> = 1.8
	</LI></UL><H6>Preparation of Ampicillin solution stock 50 mg/mL in H<SUB>2</SUB>O (Helix) aliquoted in 5 tubes of 2 mL</H6><UL><LI>In sterile condition</LI><LI>0.5 g of Amp + 10 mL of H<SUB>2</SUB>O</LI><LI>Filtration (0.2µm)</LI><LI>Storage away from light</LI></UL><H6>Digestion of</H6><UL><LI>the vector (with Ba.R0010 pLac from Cyril) by EcoRI (E) and SpeI (S)</LI><LI>the vector (with Ba.R0010 pLac from Cyril)  by EcoRI and PstI (P)</LI><LI>the phaC sequence by EcoRI and SpeI</LI><LI>the Propionyl-CoA transferase sequence by EcoRI and PstI</LI></UL><UL><LI>By the way, we tried to remove the promoter (BBa.R0010 pLac) from the vector</LI><LI>100 µL of Water were added to the phaC sequence and the Propionyl-CoA transferase sequence</LI></UL><TABLE class="center"><TBODY><TR><TH>Vector (E/S)</TH><TH>phaC (E/S)</TH><TH>Vector (E/P)</TH><TH>propCoA(E/P)</TH></TR><TR><TD>DNA</TD><TD>2 µL</TD><TD>10 µL</TD><TD>2 µL</TD><TD>10 µL</TD></TR><TR><TD>Buffer</TD><TD>Cutsmart 10X: 3 µL</TD><TD>Cutsmart 10X: 3 µL</TD><TD>2.1: 3 µL</TD><TD>2.1: 3 µL</TD></TR><TR><TD>Enzymes</TD><TD>EcoRI: 1 µL</TD><TD>EcoRI: 1 µL</TD><TD>EcoRI: 1 µL</TD><TD>EcoRI: 1 µL</TD></TR><TR><TD>SpeI: 1 µL</TD><TD>SpeI: 1 µL</TD><TD>PstI: 1.5 µL</TD><TD>PstI: 1.5 µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>23 µL</TD><TD>15 µL</TD><TD>22.5 µL</TD><TD>14.5 µL</TD></TR></TBODY></TABLE><P>1 h at 37 °C</P><H6>Electrophoresis</H6><TABLE class="center"><TBODY><TR><TH rowspan="2">Wells:</TH><TH>Marker</TH><TH>Control vector with pLac</TH><TH rowspan="2">Vector  E/S (30 µL DNA + 6 µL loading buffer)</TH><TH rowspan="2">Vector E/S (30 µL DNA + 6 µL loading buffer)</TH></TR><TR><TD>(8 µL)</TD><TD>(5 µL DNA + 1 µL loading buffer)</TD></TR><TR><TD>Expected results</TD><TD>__</TD><TD>1 band (2270 bp)</TD><TD>2 bands (2070 bp + 200 bp)</TD><TD>2 bands (2070 bp + 200 bp)</TD></TR></TBODY></TABLE><UL><LI>1 % agarose</LI><LI>Vector (with pLac): 2270 bp</LI><LI>Promoter pLac ~ 200 bp</LI></UL><H6>Results</H6><P class="figure">* : circular plasmid (3 forms: compacted, intermediate and loose forms)</P><UL><LI>-We get rid of the promoter and we kept the empty vectors</LI></UL><H6>Purification of the DNA sequences phaC and prop-CoA transf.</H6><UL><LI>Zymoclean Gel DNA Recovery Kit</LI><LI>Elution with 6 µL of water</LI><LI>Wait 1-2 min before the last spin</LI><LI>The check at the spectrophotometer nanodrop showed a pic around 240 nm</LI><LI><SPAN class="red">Maybe: problem with the wash buffer reminded in the column</SPAN></LI><LI><SPAN class="red">Inversion of the 2 columns before the second spin</SPAN></LI></UL><H5 class="blue">22<SUP>th</SUP> of July</H5><H6>Ligations</H6><TABLE class="center" style="width:70%"><TBODY><TR><TH>Empty vector (R0010)</TH><TH>10 µL</TH></TR><TR><TD>Insert phaC</TD><TD>20 µL</TD></TR><TR><TD>T4 buffer 10X</TD><TD>4 µL</TD></TR><TR><TD>T4 ligase</TD><TD>2 µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>5 µL</TD></TR></TBODY></TABLE><TABLE class="center" style="width:70%"><TBODY><TR><TH>Empty vector (R0010)</TH><TH>10 µL</TH></TR><TR><TD>Insert propCoA transf.</TD><TD>20 µL</TD></TR><TR><TD>T4 buffer 10X</TD><TD>4 µL</TD></TR><TR><TD>T4 ligase</TD><TD>2 µL</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>5µL</TD></TR></TBODY></TABLE><UL><LI>Incubation 1 h at RT</LI></UL><H6>Transformation of <I>E. coli</I> DH5 α, Top 10 and BL21 by Heat Shock </H6><UL><LI>Same protocol</LI><LI>We didn’t recover enough DNA, insert from previous midiprep, purification</LI><LI>These strains were spread on labeling plates (see table below)</LI><LI>Incubation overnight at 37 °C</LI></UL><TABLE class="center" style="width:80%"><TBODY><TR><TH>22/07/16 <I>E. coli</I> phaC DH5 α</TH><TH>22/07/16 <I>E. coli</I> phaC Top10</TH><TH>22/07/16 <I>E. coli</I> phaC BL21</TH></TR><TR><TD>22/07/16 <I>E. coli</I> propCoA DH5 α</TD><TD>22/07/16 <I>E. coli</I> propCoA Top10</TD><TD>22/07/16 <I>E. coli</I> propCoA BL21</TD></TR></TBODY></TABLE><H5 class="blue">23<SUP>th</SUP> of July</H5><UL><LI>“Transformed” clones were observed</LI><LI>Colony PCR</LI></UL><H3 class="yellow">Week 4</H3><H5 class="blue">25<SUP>th</SUP> of July</H5><H6>Media</H6><UL><LI>5 bottle of LB</LI><LI style="padding-left:10%">3 x 400 mL</LI><LI style="padding-left:10%">2 x 200 mL</LI><LI>5 bottle of LB-Agar</LI><LI style="padding-left:10%">3 x 400 mL</LI><LI style="padding-left:10%">2 x 200 mL</LI></UL><H6>Previous transformations (22/07) worked</H6><H6>Electrophoresis (PCR colony 23/07)</H6><UL><LI>Gel 1 %</LI><LI>propCoA (in Cyril’s vector)</LI></UL><UL><LI>phaC (in Cyril’s vector)</LI></UL><H6>Precultures</H6><UL><LI>2 mL of LB</LI><LI>Amp (1 µL stock solution/mL) </LI><LI>A good colony</LI><LI style="margin-left:10%">phaC : 1, 2, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 19, 20</LI><LI style="margin-left:10%">propCoA :1, 3, 4, 5, 6, 8, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20</LI><LI>Incubation overnight at 37 °C</LI></UL><H5 class="blue">26<SUP>th</SUP> of July</H5><H6>Miniprep’ kit Sigma-Aldrich : extraction of the plasmids from the precultures of the 26<SUP>th</SUP> of july</H6><UL><LI>For the propCoA transferase : problem with the 8<SUP>th</SUP>  tube we lost a lot of bacteria*</LI></UL><TABLE class="center"><TBODY><TR><TH>Colonies (with the phaC )</TH><TH>Concentrations of DNA (ng/µL)</TH><TH>Colonies (with the phaC )</TH><TH>Concentrations of DNA (ng/µL)</TH></TR><TR><TD>1</TD><TD>282.3</TD><TD>11</TD><TD>160.9</TD></TR><TR><TD>2</TD><TD>170.4</TD><TD>12</TD><TD>173.0</TD></TR><TR><TD>4</TD><TD>174.6</TD><TD>13</TD><TD>170</TD></TR><TR><TD>5</TD><TD>176.5</TD><TD>14</TD><TD>185.3</TD></TR><TR><TD>6</TD><TD>188.9</TD><TD>15</TD><TD>162</TD></TR><TR><TD>7</TD><TD>167.8</TD><TD>16</TD><TD>189.5</TD></TR><TR><TD>8</TD><TD>185.9</TD><TD>17</TD><TD>193.6</TD></TR><TR><TD>9</TD><TD>191.4</TD><TD>19</TD><TD>183</TD></TR><TR><TD>10</TD><TD>173.4</TD><TD>20</TD><TD>163</TD></TR></TBODY></TABLE><TABLE class="center"><TBODY><TR><TH>Colonies (with the propCoA )</TH><TH>Concentrations of DNA (ng/µL)</TH><TH>Colonies (with the propCoA )</TH><TH>Concentrations of DNA (ng/µL)</TH></TR><TR><TD>1</TD><TD>150.5</TD><TD>13</TD><TD>169.4</TD></TR><TR><TD>3</TD><TD>143.4</TD><TD>14</TD><TD>282.8</TD></TR><TR><TD>4</TD><TD>156</TD><TD>15</TD><TD>192.3</TD></TR><TR><TD>5</TD><TD>151.8</TD><TD>16</TD><TD>226</TD></TR><TR><TD>6</TD><TD>234.2</TD><TD>17</TD><TD>203.3</TD></TR><TR><TD>8</TD><TD>29.4*</TD><TD>18</TD><TD>175.1</TD></TR><TR><TD>10</TD><TD>194.5</TD><TD>19</TD><TD>256.5</TD></TR><TR><TD>11</TD><TD>167.2</TD><TD>20</TD><TD>181.7</TD></TR><TR><TD>12</TD><TD>162.8</TD><TD>-</TD><TD>-</TD></TR></TBODY></TABLE><H6>M9 medium 2 x 50 mL of 2.5 X</H6><UL><LI>Without C-sources, thiamine</LI><LI><SPAN class="red">Problem: precipitation (the same than previous M9 media)</SPAN></LI></UL><H5 class="blue">27<SUP>th</SUP> of July</H5><H6>Stock solutions</H6><UL><LI>CaCl2 1 M: 27.745 g in 250 mL of milliQ</LI><LI>MgSO4 1 M: 30.1 g in 250 mL of milliQ</LI><LI>FeSO4 10 mM: 0.379 g in 250 mL of milliQ</LI></UL><H6>Preculture of <I>Pseudomonas putida</I> KT2440 (2 mL)</H6><H5 class="blue">28<SUP>th</SUP> of July</H5><H6>Aliquots for sequencing  V= 20 µL</H6><H6>Medium M9 1X without Carbone source and filtred (2 µm) in 2 falcon of 50 mL</H6><UL><LI>Casamino acid : 1.2 g</LI><LI>Leucine: 3 mg</LI><LI>Uracil: 2 mg</LI><LI>Thiamine 10 mg/mL: 10 µL</LI><LI>M9 salt 5X: 20 mL</LI><LI>CaCl2 1 M: 10 µL</LI><LI>MgSO4 1 M: 100 µL</LI><LI>FeSO4 10 mM: 1 µL</LI><LI>Water milliQ qs 100 mL</LI><LI>NaOH for pH between 6 and 7.4 → no need since the pH was already at 6.8</LI></UL><P><STRONG>No precipitation</STRONG></P><H6>10 mL of Media with C-source</H6><UL><LI>0.10037 g of Glycerol (100 %) (0.16 put) in 10 mL of milliQ water</LI><LI>0.0982 g of Glucose in 10 mL of milliQ water</LI><LI>0.057 g of Benzoic acid in 10 mL of milliQ water</LI></UL><H6>Preparation for plate (96 wells) and Growth rate with the CLARIO star machine</H6><UL><LI>26.7 µL of the overnight preculture of <I>Pseudomonas</I> in medium M9 (with either Benzoic acid, Glucose or Glycerol) qs 1 mL to have around OD= 0.005</LI><LI>Plate mode, slow kinetic, DO<SUB>600nm</SUB>, T= 30 °C, shaking (200 rpm) before cycle of 10 min</LI></UL><TABLE class="center"><TBODY><TR><TH>1</TH><TH>2</TH><TH>3</TH><TH>4</TH></TR><TR><TD>A</TD><TD>-</TD><TD>Benzoic acid M9 medium</TD><TD>Glucose M9 medium</TD><TD>Glycerol M9 medium</TD></TR><TR><TD>B</TD><TD>Benzoic acid M9 medium</TD><TD>BE-M9 with cells</TD><TD>BE-M9 with cells</TD><TD>BE-M9 with cells</TD></TR><TR><TD>C</TD><TD>Glucose M9 medium</TD><TD>GLU-M9 with cells</TD><TD>GLU-M9 with cells</TD><TD>GLU-M9 with cells</TD></TR><TR><TD>D</TD><TD>Glycerol M9 medium</TD><TD>GLY-M9 with cells</TD><TD>GLY-M9 with cells</TD><TD>GLY-M9 with cells</TD></TR></TBODY></TABLE></DIV></DIV><H1>August 2016</H1><DIV class="grid-2"><DIV><H3 class="yellow">Week 5</H3><H5 class="blue">2<SUP>nd</SUP> of August</H5><P>Resolubilisation of IDT gBlocks in 100 µL water (Cf= 10 ng/µL)</P><H6>Digestion</H6><P>Mix:</P><TABLE class="center"><TBODY><TR><TH>PhaC2</TH><TH>PCT1</TH><TH>LDH1</TH><TH>pSB1A3</TH></TR><TR><TD>GBlock gene</TD><TD>10 µL</TD><TD>10 µL</TD><TD>10 µL</TD><TD>2 µL (≈600ng)</TD></TR><TR><TD>Buffer NEB</TD><TD>3 µL (2.1)</TD><TD>3 µL (CutSmart)</TD><TD>3 µL (2.1)</TD><TD>3 µL (CutSmart or 2.1)*</TD></TR><TR><TD>Enzyme 1</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (XbaI)</TD><TD>1 µL (EcoRI or XbaI)</TD></TR><TR><TD>Enzyme 2</TD><TD>1 µL (PstI)</TD><TD>1 µL (SpeI)</TD><TD>1 µL (PstI)</TD><TD>1 µL (PstI or SpeI)</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 30 µL)</TD><TD>15 µL</TD><TD>15 µL</TD><TD>15 µL</TD><TD>23 µL</TD></TR></TBODY></TABLE><UL><LI>*CutSmart when we doesn’t digest by PstI and 2.1 when PstI is used</LI><LI>Incubation 1 h 37 °C
</LI></UL><H6>Agarose gel 1 %</H6><H6>Digestion</H6><P>Mix:</P><TABLE class="center"><TBODY><TR><TH>-</TH><TH>PhaC2</TH><TH>PCT1</TH><TH>LDH1</TH></TR><TR><TD>GBlock gene</TD><TD>10 µL</TD><TD>10 µL</TD><TD>10 µL</TD></TR><TR><TD>Buffer NEB</TD><TD>3 µL (2.1)</TD><TD>3 µL (CutSmart)</TD><TD>3 µL (2.1)</TD></TR><TR><TD>Enzyme 1</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (XbaI)**</TD></TR><TR><TD>Enzyme 2</TD><TD>1 µL (PstI)</TD><TD>1 µL (SpeI)*</TD><TD>1 µL (PstI)</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 30 µL)</TD><TD>15 µL</TD><TD>15 µL</TD><TD>15 µL</TD></TR></TBODY></TABLE><P class="figure">* PCT1 has a PstI restriction site in its sequence so the digestion was done by SpeI** LDH1 has a EcoRI restriction site site in its sequence so the digestion was done by XbaI
</P><UL><LI>Incubation at 37 °C</LI></UL><H6>DNA extraction from gel</H6><P>DNA clean-up → kit Monarch PCR and DNA cleanup (#T1030G) NEB</P><P>→ DNA Gel extraction kit (#T1020G) NEB</P><UL><LI>Incubation at 55 °C to dissolve gel</LI><LI>400 µL buffer for 100 mg of agarose gel</LI><LI>DNA elution in 40 µL water</LI></UL><H6>Ligation</H6><UL><LI>10 µL Vector (pSB1A3 digested)</LI><LI>20 µL Insert digested</LI><LI>4 µL 10X T4 DNA ligase buffer</LI><LI>2 µL T4 DNA ligase</LI><LI>5 µL H<SUB>2</SUB>O</LI><LI>Incubation RT 1 h</LI></UL><H6>Transformation</H6><UL><LI>Thaw 25 µL of competent DH5α <I>E. coli</I></LI><LI>Add 10 µL of ligated DNA</LI><LI>20 min incubation in ice</LI><LI>1 min 42 °C</LI><LI>Add 120 µL of LB medium</LI><LI>Incubation 40 min 37 °C</LI><LI>Spread on plates LBA (Ampicillin)</LI><LI>Incubation 37 °C overnight</LI></UL><H5 class="blue">3<SUP>rd</SUP> of August</H5><H6>Colony PCR</H6><P><STRONG>Mix (50 µL total volume reaction)</STRONG></P><UL><LI>25 µL DreamTaq PCR Mastermix 2X (ThermoScientific)</LI><LI>5 µL 10X DreamTaq Green Buffer (ThermoScientific)</LI><LI>0.25 µL Forward Primer (0.5 µM) iG001</LI><LI>0.25 µL Reverse Primer (0.5 µM) iG002</LI><LI>19.5 µL H<SUB>2</SUB>O filtrated (qs 50 µL)</LI></UL><P>Mix prepared for 15 reactions (plus one reaction prepared without Green buffer)</P><P><STRONG>PCR program:</STRONG></P><UL><LI>95 °C 5 min</LI><LI>(95 °C 30 s, 50 °C 30 s, 72 °C 2 min) 30 times</LI><LI>72 °C 10 min</LI><LI>10 °C ∞</LI></UL><H6>Nanodrop</H6><UL><LI>PhaC2 3.5 ng/µL</LI><LI>PCT1 3.8 ng/µL</LI><LI>LDH1 8.7 ng/µL</LI><LI>pSB1A3 (digested by X+P) 3.3 ng/µL</LI></UL><H6>Gel electrophoresis</H6><P>The results were not conclusive</P><H5 class="blue">4<SUP>th</SUP> of August</H5><H6>Colony PCR</H6><P><STRONG>Mix (25 µL total volume reaction)</STRONG></P><UL><LI>12.5 µL DreamTaq PCR Mastermix 2X (ThermoScientific)</LI><LI>2.5 µL 10X DreamTaq Green Buffer (ThermoScientific)</LI><LI>0.25 µL Forward Primer (0.5 µM) iG001</LI><LI>0.25 µL Reverse Primer (0.5 µM) iG002</LI><LI>9.5 µL H<SUB>2</SUB>O filtrated (qs 25 µL)</LI></UL><P>Mix prepared for 7 reactions</P><P><STRONG>PCR program:</STRONG></P><UL><LI>95 °C 5 min</LI><LI>(95 °C 30 s, 50 °C 30 s, 72 °C 2 min) 30 times</LI><LI>72 °C 10 min</LI><LI>10 °C ∞</LI></UL><P>The results were not conclusive</P><H3 class="yellow">Week 6</H3><H5 class="blue">8<SUP>th</SUP> of August</H5><H6>Ligation</H6><UL><LI>50 ng vector</LI><LI>5 µL T4 DNA ligase Buffer 10X</LI><LI>3 µL T4 DNA</LI><LI>H<SUB>2</SUB>O qs 50 µL</LI><LI>DNA </LI></UL><TABLE class="center"><TBODY><TR><TD>PhaC 2</TD><TD rowspan="2">11.7 µL</TD><TD>Vector E/P</TD><TD rowspan="2">7.8 µL</TD><TD rowspan="2">H<SUB>2</SUB>O 22.5 µL</TD></TR><TR><TD>(3.5 ng/µL)</TD><TD>(6.4 ng/µL)</TD></TR><TR><TD>PCT 1</TD><TD rowspan="2">30.4 µL</TD><TD>Vector E/S</TD><TD rowspan="2">4.1 µL</TD><TD rowspan="2">H<SUB>2</SUB>O 7.5 µL</TD></TR><TR><TD>(3.8 ng/µL)</TD><TD>(12.2 ng/µL)</TD></TR><TR><TD>LDH 1</TD><TD rowspan="2">13.9 µL</TD><TD>Vector X/P</TD><TD rowspan="2">8.5 µL</TD><TD rowspan="2">H<SUB>2</SUB>O 19.6 µL</TD></TR><TR><TD>(8.7 ng/µL)</TD><TD>(5.9 ng/µL)</TD></TR></TBODY></TABLE><P>We put 27.5 µL of Pct1 DNA because there was not enough.</P><H6>Transformation</H6><UL><LI>Thaw DH5-α competent bacteria in the ice.</LI><LI>Using the protocol processing by thermal shock.</LI><LI>25 µL of bacteria + Mix of ligation (2 µL each time)</LI><LI>Incubation 20 minutes in ice and then 1 min at 42 °C</LI><LI>Adding 120 μl LB and incubate at 37 ° for 1 hour</LI><LI>Spread on LB Ampicillin with the öse</LI></UL><H5 class="blue">9<SUP>th</SUP> of August</H5><H6>Colony PCR</H6><P>Mix PCR with 25 µL of total volume reaction prepared for 12 reactions prepared as before (DreamTaq #K1071 Thermo Scientific).</P><P><STRONG>PCR program:</STRONG></P><UL><LI>95 °C 5 min</LI><LI>(95 °C 30 s, 50 °C 30 s, 72 °C 1 min) 30 times</LI><LI>72 °C 10 min</LI><LI>10 °C ∞</LI></UL><P>No bands on the gel (maybe problems with the enzyme)</P><H5 class="blue">10<SUP>th</SUP> of August</H5><H6>Digestion</H6><P><STRONG>Mix:</STRONG></P><TABLE class="center"><TBODY><TR><TH>PhaC2</TH><TH>PhaC3</TH><TH>PCT2</TH><TH>LDH1</TH></TR><TR><TD>GBlock gene</TD><TD>20 µL</TD><TD>20 µL</TD><TD>20 µL</TD><TD>20 µL</TD></TR><TR><TD>Buffer NEB</TD><TD>3 µL (2.1)</TD><TD>3 µL (2.1)</TD><TD>3 µL (CutSmart)</TD><TD>3 µL (2.1)</TD></TR><TR><TD>Enzyme 1</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (XbaI)*</TD></TR><TR><TD>Enzyme 2</TD><TD>1 µL (PstI)</TD><TD>1 µL (PstI)</TD><TD>1 µL (PstI)</TD><TD>1 µL (PstI)</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 30µL)</TD><TD>5 µL</TD><TD>5 µL</TD><TD>5 µL</TD><TD>5 µL</TD></TR></TBODY></TABLE><P class="figure">* LDH1 has a EcoRI restriction site site in its sequence so the digestion was done by XbaI</P><TABLE class="center"><TBODY><TR><TH>pSB1A3</TH><TH>pSB1A3</TH></TR><TR><TD>Vector DNA</TD><TD>2 µL (≈600 ng)</TD><TD>6 µL (≈2 µg)</TD></TR><TR><TD>Buffer NEB</TD><TD>3 µL (CutSmart or 2.1)*</TD><TD>9 µL (CutSmart or 2.1)*</TD></TR><TR><TD>Enzyme 1</TD><TD>1 µL (EcoRI or XbaI)</TD><TD>3 µL (EcoRI or XbaI)</TD></TR><TR><TD>Enzyme 2</TD><TD>1 µL (PstI or SpeI)</TD><TD>3 µL (PstI or SpeI)</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 30 µL)</TD><TD>23 µL</TD><TD>69 µL</TD></TR><TR><TD>Divided in 3 reaction tubes</TD></TR></TBODY></TABLE><P>Incubation 1 h 37 °C</P><H6>1 % agarose gel</H6><SPAN class="red"><UL><LI>2 trash nucleotides after the PstI restriction site in the suffix were forgotten so the digestion risks to fail → the digestion will be done by SpeI!</LI><LI>PhaC genes doesn’t match the gene in our reference publications but it corresponds to the gene used by the YALE IGEM team in 2013</LI></UL></SPAN><H5 class="blue">11<SUP>th</SUP> of August</H5><P>Extraction of digested vectors (pSB1A3 X+P and pSB1A3 E+P) from agarose gel (kit Zymoclean gel DNA Recovery kit #D4001S)</P><H6>Concentration of the samples</H6><UL><LI>pSB1A3 digested by X+P =&gt; 3.9 ng/µL</LI><LI>pSB1A3 digested by E+P =&gt; 7.2 ng/µL</LI></UL><H6>Digestion by SpeI</H6><P><STRONG>Mix:</STRONG></P><UL><LI>20 µL genes digested by E+P or X+P or vector digested by X+P</LI><LI>3 µL buffer NEB CutSmart</LI><LI>1 µL enzyme SpeI</LI><LI>6 µL H20 (qs 30 µL)</LI></UL><P><STRONG>Or mix:</STRONG></P><UL><LI>20 µL genes digested by E+P or X+P or vector digested by X+P</LI><LI>3 µL buffer NEB CutSmart</LI><LI>1 µL enzyme SpeI</LI><LI>6 µL H20 (qs 30 µL)</LI></UL><H6>DNA purification</H6><UL><LI>DNA clean &amp; concentrator-5 (#D4003S Zymo Research)</LI><LI>For the vector: 2 volumes of binding buffer add to DNA (ratio 2:1)</LI><LI>For the genes: 5 volumes of binding buffer add to DNA (ratio 5:1)</LI><LI>Elution with 20 µL of water and incubation 2 min before centrifugation for DNA elution</LI></UL><H6>Ligation</H6><P><STRONG>Mix:</STRONG></P><TABLE><TBODY><TR><TH>PhaC2</TH><TH>PhaC3</TH><TH>PCT2</TH></TR><TR><TD>Vector pSB1A3</TD><TD>4.5 µL</TD><TD>4.5 µL</TD><TD>4.5 µL</TD></TR><TR><TD>Gene</TD><TD>14 µL</TD><TD>11 µL</TD><TD>10 µL</TD></TR><TR><TD>T4 DNA ligase Buffer</TD><TD>3 µL</TD><TD>3 µL</TD><TD>3 µL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 30 µL)</TD><TD>7.5 µL</TD><TD>10.5 µL</TD><TD>11.5 µL</TD></TR></TBODY></TABLE><UL><LI>Incubation 1 h RT</LI><LI>Sample frozen at -20 °C until transformation</LI></UL><H5 class="blue">12<SUP>th</SUP> of August</H5><H6>Transformation</H6><UL><LI>Thaw DH5-α competent bacteria in the ice.</LI><LI>Using the protocol processing by heat shock.</LI><LI>25 µL of bacteria + 5 µL ligation mix</LI><LI>Incubation 20 minutes in ice</LI><LI>1 min at 42 ° C</LI><LI>3 min in ice</LI><LI>Adding 120 μl of LB</LI><LI>Incubation at 37 °C for around 50 min</LI><LI>Spread on LB Ampicillin plates</LI><LI>Incubation overnight 37 °C</LI></UL><H5 class="blue">13<SUP>th</SUP> of August</H5><H6>Results transformation</H6><P>We got clones from the transformation. We took 3 clones from Pct(v2), 2 from phaC(v2) and 2 from phaC(v3) and performed PCR colony.</P><H6>PCR colonies</H6><P><STRONG>20 µL mix per sample with:</STRONG></P><UL><LI>10 µL Green Taq Master Mix 2X (#K1081 Thermo Scientific)</LI><LI>0.25 µL Forward primer iG001 (10 µM)</LI><LI>0.25 µL Forward primer iG002 (10 µM)</LI><LI>9.5 µL H<SUB>2</SUB>O (qs 20 µL)</LI></UL><P>Mix for 9 reactions + clones</P><P><STRONG>PCR program:</STRONG></P><UL><LI>95 °C 5 min</LI><LI>(95 °C 30 s, 50 °C 30 s, 72 °C 2 min) 30 times</LI><LI>72 °C 10 min</LI><LI>10 °C ∞</LI></UL><P>1 % agarose electrophoresis (90 V, 111 mA, 40 min)</P><P><SPAN class="red">=&gt;	Result of PCR Colony: 1 clone of each was good</SPAN></P><H6>Precultures</H6><P>3 mL LB medium + positive clones for each gene (PhaC2, PhaC3, PCT2)</P><P>Incubation 37 °C with shaking (around 16 h incubation)</P><H6>Transformation (<I>E. coli</I> DH5-α with pSB1A3 ligated with gene) by heat shock</H6><UL><LI>Thaw DH5-α competent bacteria in ice</LI><LI>100 µL of bacteria + 10 µL ligation mix</LI><LI>Incubation 20 minutes in ice</LI><LI>45 s at 42 ° C</LI><LI>3 min in ice</LI><LI>Adding 900 μl of LB</LI><LI>Incubation 1 h, 37 °C</LI><LI>Spread on LB Ampicillin plates</LI><LI>Incubation overnight 37 °C</LI></UL><H5 class="blue">14<SUP>th</SUP> of August</H5><P><SPAN class="red">No colony for PCT2 transformation!</SPAN></P><H6>Miniprep from precultures</H6><P>Transformed bacteria were stored at -80 °C in 50 % glycerol (2 aliquotes for each preculture)</P><UL><LI>GenElute Plasmid Miniprep Kit (Sigma, PLN350-1KT)</LI><LI>Incubation 1 min for elution of DNA before centrifugation</LI><LI>pSB1A3-PhaC2 144 ng/µL</LI><LI>pSB1A3-PhaC3 96.1 ng/µL</LI><LI>pSB1A3-PCT2 143.3 ng/µL</LI></UL><H6>Digestion</H6><P><STRONG>Mix</STRONG></P><TABLE class="center"><TBODY><TR><TH>-</TH><TH>LDH1 gBlock gene (10 ng/µL)</TH><TH>pSB1A3 (X+P)</TH></TR><TR><TD>DNA</TD><TD>20 µL (200 ng)</TD><TD>50 µL</TD></TR><TR><TD>10X Buffer NEB CutSmart</TD><TD>3 µL</TD><TD>6 µL</TD></TR><TR><TD>enzyme XbaI</TD><TD>1 µL</TD><TD>2 µL</TD></TR><TR><TD>enzyme SpeI</TD><TD>1 µL</TD><TD>-</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 30 µL)</TD><TD>5 µL</TD><TD>2 µL (qs 60 µL)</TD></TR></TBODY></TABLE><P>Incubation 1 h 37 °C</P><H6>Dephosphorylation</H6><UL><LI>Add 1 µL alkaline phosphatase (FastAP #EF0654 Thermo Scientific) to the digestion mix (for LDH1 digestion or vector pSB1A3 digestion)</LI><LI>Incubation 30 min 37 °C</LI><LI>Inactivationof the enzyme: incubation 75 °C, 10 min</LI></UL><H6>Recombinant vector digestion</H6><P><STRONG>Mix</STRONG></P><TABLE><TBODY><TR><TH>-</TH><TH>pSB1A3-PhaC2</TH><TH>pSB1A3-PhaC3</TH><TH>pSB1A3-PCT2</TH></TR><TR><TD>DNA (around 200 ng or 600 ng)</TD><TD>2 µL</TD><TD>2.5 µL</TD><TD>6 µL (around 600 ng)</TD></TR><TR><TD>Buffer NEB CutSmart 10X</TD><TD>3 µL (2.1)</TD><TD>3 µL (2.1)</TD><TD>3 µL (CutSmart)</TD></TR><TR><TD>Enzyme 1</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD></TR><TR><TD>Enzyme 2</TD><TD>1 µL (SpeI)</TD><TD>1 µL (SpeI)</TD><TD>1 µL (XbaI)</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 20 µL)</TD><TD>14 µL</TD><TD>13.5 µL</TD><TD>10 µL</TD></TR></TBODY></TABLE><P>Incubation 1 h 37 °C</P><H6>DNA purification (LDH gene)</H6><UL><LI>DNA clean &amp; concentrator-5 (#D4003S Zymo Research)</LI><LI>Ratio 5:1 of DNA binding buffer for LDH gene fragment</LI><LI>Elution 30 µL water with incubation 3 min before centrifugation</LI><LI>LDH1 =&gt;</LI> 6.9 ng/µL</UL><H6>Agarose gel for digested recombinant vectors</H6><P>All of the digested samples is charged on the gel.</P><P class="figure"><U>Digestion of the recombinant vector.</U>	Digestion was a failure for the 2 plasmids containing the PhaC2 and PhaC3 genes. The vector containing PCT2 is open but we have no way to be sure that the digestion was efficient for the 2 enzymes or just for one of them.</P><P>Colony PCR on the clones obtained after the 2<SUP>nd</SUP> transformation of the ligation mixes:</P><P class="figure"><U>PCR colony.</U>	PCR on colony failed.</P></DIV><DIV><H3 class="yellow">Week 7</H3><H5 class="blue">16<SUP>th</SUP> of August</H5><H6>Digestion</H6><TABLE class="center" style="width:80%"><TBODY><TR><TH>-</TH><TH>LDH1 gBlock gene (10 ng/µL)</TH><TH>pSB1A3 (352 ng/µL)</TH></TR><TR><TD>DNA</TD><TD>30 µL (300 ng)</TD><TD>2 µL</TD></TR><TR><TD>10X Fastdigest buffer (ThermoScientific)</TD><TD>4 µL (clear buffer)</TD><TD>2 µL (green buffer)</TD></TR><TR><TD>enzyme XbaI</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD>enzyme SpeI</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 40 µL)</TD><TD>4 µL</TD><TD>14 µL (qs 20 µL)</TD></TR></TBODY></TABLE><TABLE class="center" style="width:80%"><TBODY><TR><TH>-</TH><TH>pSB1A3-PhaC2</TH><TH>pSB1A3-PhaC3</TH><TH>pSB1A3-PCT2</TH></TR><TR><TD>DNA (around 200 ng or 600 ng)</TD><TD>2 µL</TD><TD>2.5 µL</TD><TD>6 µL (around 600 ng)</TD></TR><TR><TD>Buffer Fastdigest green 10X (ThermoScientific)</TD><TD>2 µL</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD>Enzyme 1</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD><TD>1 µL (EcoRI)</TD></TR><TR><TD>Enzyme 2</TD><TD>1 µL (SpeI)</TD><TD>1 µL (SpeI)</TD><TD>1 µL (XbaI)</TD></TR><TR><TD>H<SUB>2</SUB>O (qs 20 µL)</TD><TD>14 µL</TD><TD>13.5 µL</TD><TD>10 µL</TD></TR></TBODY></TABLE><H6>Agarose gel 1 %</H6><H6>DNA purification (digested by X + S LDH1 gene)</H6><UL><LI>DNA clean &amp; concentrator-5 (#D4003S Zymo Research)</LI><LI>Ratio 5:1 of DNA binding buffer for LDH gene fragment</LI></UL><H5 class="blue">17<SUP>th</SUP> of August</H5><H6>Gel purification via the « Zymoclean Gel DNA Recovery » kit</H6><UL><LI>weight of the empty tube: 1 g</LI><LI>weight of the tube + vector PSB1A3 (X+S)
			</LI><LI style="margin-left:10%">n°1 : 1.1 g is 100 mg</LI><LI style="margin-left:10%">n°2 : 1.1 is 100 mg</LI><LI>vector size: 2155 bp</LI><LI>Step 1 : For 100 mg d’agarose we add 300 µL of ADB</LI><LI>Step 2 : Put samples at 55 °C for 10 minutes and centrifuge at 300 rpm</LI><LI>Step 3 : Transfert in the column</LI><LI>Step 4 : Centrifugation while 1 minute at 13 000 rpm then suppression of the flow-through</LI><LI>Step 5 : Add 200 µL of DNA Wash Buffer and centrifugation while 30 seconds X2</LI><LI>Step 6 : Centrifugation empty for 2 minutes</LI><LI>Step 7 : Add 10 µL DNA Elution Buffer, wait 2 minutes</LI><LI>Step 8 : Centrifuge 1 minutes at 13 000 rpm</LI></UL><H6>Nano drop</H6><UL><LI>DO vector Psb1a3 n°1 : 39 ng/µL</LI><LI>DO vector Psb1a3 n°2 : 43.2 ng/ µL</LI></UL><H5 class="blue">18<SUP>th</SUP> of August</H5><H6>Chloramphenicol stock</H6><UL><LI>50 ng/mL chloramphenicol stock (2.5 g in 50 mL ethanol 100 %)</LI><LI>30 aliquots stored at -20 °C</LI></UL><H6>pSB1C3-mRFP</H6><UL><LI>Solubilisation of pSB1C3-mRFP (2016 IGEM plate 5 - 1F) with 10 µL water nuclease-free (ThermoScientific)</LI><LI>Incubation 5 min RT</LI><LI>Transformation in <I>E. coli</I> DH5-α strain</LI></UL><H6>Transformation</H6><UL><LI>Thaw 25 µL DH5-α competent bacteria in ice</LI><LI>Add 2 µL of resolubilized pSB1C3-mRFP DNA from 2016 IGEM plate 5</LI><LI>Incubation 20 minutes on ice</LI><LI>1 min at 42 ° C</LI><LI>3 min in ice</LI><LI>Adding 120 μl of LB</LI><LI>Incubation 1 h, 37 °C with shaking</LI><LI>Spread on LB Chloramphenicol plates</LI><LI>Incubation overnight 37 °C</LI></UL><P>Solubilisation of the primers iG051, iG052, iG049 and iG050 and aliquots of 10 times dilutions.</P><H5 class="blue">19<SUP>th</SUP> of August</H5><H6>Precultures</H6><P>3 mL LB medium + 1.8 µL Chloramphenicol (50 mg/mL) =&gt; plates at 30 µg/mL chloramphenicol</P><UL><LI>No conclusive results for PCR on colonies for pSB1C3-mRFP plates so we did precultures with 4 other clones (A, B, C and D)</LI><LI>The Mastermix Taq DNA polymerase seems to not work anymore (after freezer problem, the polymerases lost their activities)</LI></UL><H6>Q5 PCR for changing antibiotic resistance in pSEVA 224 plasmid</H6><UL><LI>10 µL reaction buffer</LI><LI>1 µL dNTP mix</LI><LI>1 µL 10 times diluted primers (X2)</LI><LI>0.5 µL DNA template</LI><LI>0.5 µL Q5 DNA polymerase</LI><LI>H20 qs 50 µL (= 36 µL)</LI></UL><P><STRONG>2 different reactions:</STRONG></P><OL><LI>Amplification of spectinomycin resistance gene with pCDF vector as DNA template (length expected = 1220 bp) primers used: iG050 and iG052 (1 min of DNA polymerization and primers annealing at 60 °C)</LI><LI>Amplification of pSEVA224 backbone without kanamycin resistance gene (pSEVA224 as DNA template; length expected = 4253 bp) primers used: iG049 and iG051 (3 min of DNA polymerization and primers annealing at 60 °C)</LI></OL><UL><LI>95 °C 5 min</LI><LI>(95 °C 30 s, 60 °C 30 s, 72 °C 1 min or 3 min) * 35 times </LI><LI>72 °C 5 min</LI><LI>10 °C ∞</LI></UL><H5 class="blue">20<SUP>th</SUP> of August</H5><H6>DNA miniprep of pSB1C3-mRFP (clone A, B, C and D)</H6><UL><LI>Stock in 50 % glycerol stored at -80 °C</LI><LI>GenElute Plasmid Miniprep Kit (Sigma, PLN350-1KT)</LI><LI>Incubation 2 min for DNA elution before centrifugation and centrifugation of 1 min 30 s instead of 1 min.</LI><LI>Optional Wash Buffer step performed and use of nuclease-free water for elution.</LI><LI>Clone A: 240.3 ng/µL</LI><LI>Clone B: 150.3 ng/µL</LI><LI>Clone C: 111.0 ng/µL</LI><LI>Clone D: 76.6 ng/µL</LI></UL><H6>PCR on DNA miniprep - Primers iG001 and iG002</H6><P><STRONG>20 µL mix per sample with:</STRONG></P><UL><LI>10 µL Green Taq Master Mix 2X (#K1081 Thermo Scientific)</LI><LI>0.25 µL Forward primer iG001 (10 µM)</LI><LI>0.25 µL Forward primer iG002 (10 µM)</LI><LI>9.5 µL H<SUB>2</SUB>O (qs 20 µL)</LI></UL><P><STRONG>PCR program:</STRONG></P><UL><LI>95 °C 5 min</LI><LI>(95 °C 30 s, 50 °C 30 s, 72 °C 1 min) * 30 times</LI><LI>72 °C 5 min</LI><LI>10 °C ∞</LI></UL><H6>Agarose gel 1 % for changing antibiotic PCR</H6><P>The amplification of pSEVA =&gt; new attempt</P><P><STRONG>Digestion vectors pSB1C3-mRFP (on DNA miniprep)</STRONG></P><UL><LI>10 µL plasmid DNA</LI><LI>3 µL Buffer 10X green FD (FastDigest)</LI><LI>1 µL EcoRI</LI><LI>1 µL SpeI</LI><LI>15 µL H20 (qs 30 µL)</LI></UL><P>Size expected: 2051 bp</P><P>Digestion was checked on agarose gel and the band corresponding to pSB1C3 digested by E+S was cut in order to recover the DNA.</P><H3 class="yellow">Week 8</H3><H5 class="blue">22<SUP>th</SUP> of August</H5><H6>Digestion efficiency test of restriction enzymes EcoRI and SpeI</H6><P><STRONG>Mix</STRONG></P><UL><LI>4 µL vector pSB1C3</LI><LI>3 µL buffer 10X CutSmart or FastDigest Green</LI><LI>1 µL EcoRI and/or SpeI</LI><LI>21 µL or 22 µL H<SUB>2</SUB>O</LI></UL><P>Incubation 1 h 37 °C</P><H6>Agarose gel 1 %</H6><H6>DNA extraction from agarose gel (linear vector extracted from clones A, B and C)</H6><UL><LI>Zymoclean gel DNA Recovery kit #D4001S</LI><LI>Vector A: 20.9 ng/µL</LI><LI>Vector B: 13.3 ng/µL</LI><LI>Vector C: 13.1 ng/µL</LI></UL><P><STRONG>Expected sizes after digestion by E+S</STRONG></P><UL><LI>pSB1C3 2051 bp</LI><LI>mRFP 742 bp</LI></UL><H5 class="blue">23<SUP>th</SUP> of August</H5><H6>Digestion gBlocks by E+S</H6><UL><LI>20 µL gBlocks DNA</LI><LI>3 µL 10X NEB Buffer (Cutmart)</LI><LI>1 µL EcoRI</LI><LI>1 µL SpeI</LI><LI>5 µL H<SUB>2</SUB>O (qs 30 µL)</LI><LI>Incubation 1 h at 37 °C</LI></UL><H6>Purification Agarose Gel with the kit « Zymoclean Gel DNA Recovery kit » (#D4001S Zymo Research)</H6><UL><LI>Cf protocol august 17<SUP>th</SUP> 2016</LI><LI>Elution with 20µL H<SUB>2</SUB>O</LI></UL><H6>Nanodrop 1</H6><UL><LI>LDH2: 10.3 ng/µL</LI><LI>Phac3: 5.9 ng/µL</LI><LI>Phac 4: 8.5 ng/µL</LI><LI>Pct 2: 8.9 ng/µL</LI></UL><P>too low concentration for ligation so I put at the Speed Vae. Protocole &quot;Vaccum Drive&quot;; resuspend with 10 µL H<SUB>2</SUB>O</P><H6>Nanodrop 2</H6><UL><LI>LDH2 = 15 ng/µL - Ratio 260/280 : 1.92</LI><LI>Phac 3 = 8.8 ng/µL - Ratio 260/280 : 2.20</LI><LI>PhaC4 = 9.2 ng/µL - Ratio 260/280 : 1.05</LI><LI>Pct2 = 10.8 ng/µL - Ratio 260/280 : 1.88</LI></UL><TABLE class="center" style="width:80%"><TBODY><TR><TH>Gene</TH><TH>Length</TH><TH>Amount (ng)</TH><TH>Concentration (ng/µL)</TH><TH>Volume (µL)</TH></TR><TR><TD>Plasmid psB1C3</TD><TD>2051 pb</TD><TD>20.9</TD><TD>2.4</TD></TR><TR><TD>PhaC3</TD><TD>1717 pb</TD><TD>209.3</TD><TD>8.8</TD><TD>23.8</TD></TR><TR><TD>LDH2</TD><TD>1227 pb</TD><TD>149.6</TD><TD>15</TD><TD>10</TD></TR><TR><TD>Phac 4</TD><TD>1720 pb</TD><TD>209.7</TD><TD>9.2</TD><TD>22.8</TD></TR><TR><TD>Pct 2</TD><TD>1612 pb</TD><TD>196.5</TD><TD>10.8</TD><TD>18.2</TD></TR></TBODY></TABLE><UL><LI>Amount of insert: 50*5 x length gBlock / length plasmid</LI><LI>Volume of insert: amount of insert / concentration of insert</LI></UL><P>As we suspended in 10 μl I do not have sufficient quantities.</P><P><STRONG>Ligation mix</STRONG></P><TABLE class="center" style="width:80%"><TBODY><TR><TH>Insert</TH><TH>DNA (µL)</TH><TH>Vector (µL)</TH><TH>Buffer T4 Ligase 10X (µL)</TH><TH>T4 Ligase (µL)</TH><TH>H<SUB>2</SUB>O (qs 20 µL)</TH></TR><TR><TD>Pct2</TD><TD>9</TD><TD>0.96</TD><TD>2</TD><TD>0.5</TD><TD>7.54</TD></TR><TR><TD>LDH2</TD><TD>9</TD><TD>1.29</TD><TD>2</TD><TD>0.5</TD><TD>7.21</TD></TR><TR><TD>Phac4</TD><TD>9</TD><TD>0.96</TD><TD>2</TD><TD>0.5</TD><TD>7.54</TD></TR><TR><TD>Phac3</TD><TD>9</TD><TD>0.96</TD><TD>2</TD><TD>0.5</TD><TD>7.54</TD></TR></TBODY></TABLE><UL><LI>I put at 22 °C the ligation Mix for 10 min</LI><LI>I take 5 µL for the transformation of 50 µL competent cells</LI></UL><H6>Transformation</H6><UL><LI>Use protocol of Bacterial transformation by heat shock</LI><LI>Thaw competent cells on ice</LI><LI>mixed 50 µL of competent bacteria with 5 µL of DNA</LI><LI>incubate on ice for 20 minutes</LI><LI>placing the tubes at 42 ° C for 1 min then put 3 min in ice</LI><LI>Add 250 µL of LB and incubate 1 h at 37 °C </LI><LI>Spread on box LB Chloramphenicol (30 µg/mL)</LI><LI>incubation overnight 37 °C</LI><LI>Poored 6 plates of 25 mL of LB agar + chloramphenicol (30 µg/µL)</LI><LI>Incubation at 37 °C </LI></UL><H5 class="blue">24<SUP>th</SUP> of August</H5><H6>Digestion gBlocks by E+S</H6><UL><LI>20 µL gBlocks DNA</LI><LI>3 µL 10X NEB Buffer (Cutmart)</LI><LI>1 µL EcoRI</LI><LI>1 µL SpeI</LI><LI>5 µL H<SUB>2</SUB>O (qs 30 µL)</LI></UL><P>Incubation 1 h at 37 °C</P><H6>Purification Agarose Gel with the kit « Zymoclean Gel DNA Recovery kit » (#D4001S Zymo Research)</H6><P>Elution with 20 µL nuclease-free water</P><UL><LI>PhaC3 (E+S) =&gt; 12.2 ng/µL</LI><LI>PhaC4 (E+S) =&gt; 6.3 ng/µL</LI><LI>LDH2 (E+S) =&gt; 16.8 ng/µL</LI><LI>PCT2 (E+S) =&gt; 12.4 ng/µL</LI><LI>Concentration of the DNA: around 1 h in evaporator</LI><LI>DNA resuspension with 11 µL water</LI></UL><P>First concentration failed so second concentration of DNA performed in evaporator (1 h)</P><H5 class="blue">25<SUP>th</SUP> of August</H5><UL><LI>Resuspension of DNA with 11 µL water</LI><LI>PhaC3 (E+S) =&gt; 10.4 ng/µL</LI><LI>PhaC4 (E+S) =&gt; 10.6 ng/µL</LI><LI>LDH2 (E+S) =&gt; 27.0 ng/µL</LI><LI>PCT2 (E+S) =&gt; 20.1 ng/µL</LI></UL><P>DNA concentration by water evaporation failed!</P><H6>Ligation genes inside pSB1C3 (ratio 5:1)</H6><TABLE class="center" style="width:80%"><TBODY><TR><TH>-</TH><TH>PhaC4</TH><TH>PhaC3</TH><TH>PCT2</TH><TH>LDH2</TH></TR><TR><TD>Buffer T4 DNA ligase (10X)</TD><TD>2 µL</TD><TD>2 µL</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>0.5 µL</TD><TD>0.5 µL</TD><TD>0.5 µL</TD><TD>0.5 µL</TD></TR><TR><TD>Vector</TD><TD>0.96 µL</TD><TD>0.96 µL</TD><TD>0.96 µL</TD><TD>0.96 µL</TD></TR><TR><TD>Insert (gene)</TD><TD>7.9 µL</TD><TD>8.05 µL</TD><TD>3.91 µL</TD><TD>2.2 µL</TD></TR><TR><TD>H20 (qs 20 µL)</TD><TD>8.63 µL</TD><TD>8.49 µL</TD><TD>12.6 µL</TD><TD>14.3 µL</TD></TR></TBODY></TABLE><P>10 min incubation at 22 °C</P><H6>Transformation</H6><UL><LI>Thaw 50 µL bacteria (<I>E. coli</I> DH5-α) chemo-competent</LI><LI>Add 5 µL DNA (mix ligation)</LI><LI>Incubation 20 min on ice</LI><LI>Heat shock: 45 s at 42 °C</LI><LI>Incubation 3 min on ice</LI><LI>Add 950 µL LB</LI><LI>Incubation 1 h at 37 °C with shaking</LI><LI>Centrifugation 1 min at 4000 rpm</LI><LI>Throw away supernatant</LI><LI>Resuspension with the rest of supernatant (around 50 µL)</LI><LI>Spread on plates LB Chloramphenicol (30 µg/mL)</LI></UL><H5 class="blue">26<SUP>th</SUP> of August</H5><P>Preparation plates LBC (30 µg/µL)</P><H6>Ligation (ratio 1:1)</H6><TABLE class="center" style="width:80%"><TBODY><TR><TH>-</TH><TH>PhaC4</TH><TH>PCT2</TH><TH>PhaC3</TH><TH>LDH2</TH></TR><TR><TD>Buffer T4 DNA ligase (10X)</TD><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>0.5 µL</TD><TD>0.5 µL</TD><TD>0.5 µL</TD><TD>0.5 µL</TD></TR><TR><TD>Vector</TD><TD>2.39 µL</TD><TD>2.39 µL</TD><TD>2.39 µL</TD><TD>2.39 µL</TD></TR><TR><TD>Insert (gene)</TD><TD>3.96 µL</TD><TD>1.95 µL</TD><TD>4.02 µL</TD><TD>1.11 µL</TD></TR><TR><TD>H20 (qs 10 µL)</TD><TD>2.15 µL</TD><TD>4.15 µL</TD><TD>2.08 µL</TD><TD>5.0 µL</TD></TR></TBODY></TABLE><P>Incubation 16h 16 °C</P><H5 class="blue">28<SUP>th</SUP> of August</H5><H6>Transformation <I>E. coli</I> DH5-α</H6><UL><LI>Thaw 25 µL chemo-competent bacteria</LI><LI>Add 5 µL DNA (mix ligation) (2 µL of pSB1C3-mRFP from clone A as positive transformation control)</LI><LI>Incubation 20 min on ice</LI><LI>Heat shock: 45 s at 42 °C</LI><LI>Incubation 3 min on ice</LI><LI>Add 900 µL LB</LI><LI>Incubation 1h20 at 37 °C with shaking</LI><LI>Centrifugation 1 min 30 s at 5000 rpm</LI><LI>Throw away all the supernatant</LI><LI>Resuspension with 100 µL LB medium</LI><LI>Spread on plates LB Chloramphenicol (30 µg/mL)</LI><LI>Incubation overnight 37 °C</LI></UL><H3 class="yellow">Week 9</H3><H5 class="blue">29<SUP>th</SUP> of August</H5><P>The GFP gene contained in Gen4_BNR vector (provided by Cécile) was amplified by PCR with Q5 DNA polymerase with primers allowing to add the prefix and suffix regions at both sides of the amplified region.</P><H6>Mix</H6><UL><LI>10 µL Q5 reaction buffer 5X</LI><LI>1 µL dNTP mix</LI><LI>2.5 µL of Forward Primer (iG063)</LI><LI>2.5 µL of Reverse Primer (iG064)</LI><LI>1 µL DNA template</LI><LI>0.5 µL Q5 DNA polymerase</LI><LI>32.5 µL Nuclease-free water (qs 50 µL)</LI></UL><H6>PCR program:</H6><UL><LI>98 °C 30 s</LI><LI>(98 °C 30 s, 55 °C 30 s, 72 °C 1 min) 35 times</LI><LI>72 °C 2 min</LI><LI>10 °C ∞</LI></UL><P>After checking on agarose 1 % gel, we determined that the GFP was successfully amplified.</P><H6>Digestion</H6><P>The amplified fragment was digested by EcoRI and PstI during 1 h at 37 °C. The reaction was performed in NEB CutSmart buffer.</P><P>The digested mix was purified by using the kit Wizard SV Gel and PCR Clean-Up System (Ref: A9282, Promega).</P><H5 class="blue">30<SUP>th</SUP> of August</H5><H6>Ligation</H6><P>The GFP fragment was then ligated separately in pSEVA 224 or pSEVA 2311 digested by E+P in a ratio 5:1.</P><P>The ligation mix was stored at -20 °C until transformation.</P><H5 class="blue">31<SUP>th</SUP> of August</H5></DIV></DIV><H1>September 2016</H1><DIV class="grid-2"><DIV><H3 class="yellow">Week 9</H3><H5 class="blue">2<SUP>nd</SUP> of September</H5><H6>Digestion</H6><TABLE class="center"><TBODY><TR><TH rowspan="2">Genes</TH><TH rowspan="2">Quantity of DNA (µL)</TH><TH rowspan="2">Restriction Enzymes (µL)</TH><TH>Buffer</TH><TH>Water</TH></TR><TR><TD>Digest NEB 2.1 (µL)</TD><TD>qs 50 µL (µL)</TD></TR><TR><TD rowspan="2">LDH2 (109.5 ng/µL)</TD><TD rowspan="2">10</TD><TD>EcoR1 HF : 1 µL</TD><TD rowspan="2">5</TD><TD rowspan="2">33</TD></TR><TR><TD>Pst1 :1 µL</TD></TR><TR><TD rowspan="2">PCT2 (56.5 ng/µL)</TD><TD rowspan="2">20</TD><TD>EcoR1 HF : 1 µL</TD><TD rowspan="2">5</TD><TD rowspan="2">23</TD></TR><TR><TD>Pst1 :1 µL</TD></TR><TR><TD rowspan="2">PhaC3 (63.9 ng/µL)</TD><TD rowspan="2">17.5</TD><TD>EcoR1 HF : 1 µL</TD><TD rowspan="2">5</TD><TD rowspan="2">25.5</TD></TR><TR><TD>Pst1 :1 µL</TD></TR><TR><TD rowspan="2">PhaC4 (88.1 ng/µL)</TD><TD rowspan="2">12.5</TD><TD>EcoR1 HF : 1µL</TD><TD rowspan="2">5</TD><TD rowspan="2">30.5</TD></TR><TR><TD>Pst1 :1µL</TD></TR><TR><TD rowspan="2">Vector pSB1C3 (240.3 ng/µL)</TD><TD rowspan="2">4.5</TD><TD>EcoR1 HF : 1 µL</TD><TD rowspan="2">5</TD><TD rowspan="2">38.5</TD></TR><TR><TD>Pst1 :1 µL</TD></TR></TBODY></TABLE><P>Incubation 1 h at 37 °C and after incubation 20 min at 80 °C.</P><H6>Gel Extraction</H6><UL><LI>50 mL of Agarose 1 % with 2.5 µL of midorigreen.</LI><LI>Migration 25 min at 100 mV.</LI><P class="figure">*Digestion mixes with EcoRI and PstI were loaded on the gel.</P></UL><H5 class="blue">3<SUP>rd</SUP> of September</H5><H6>DNA extraction from agarose gel</H6><P>Kit: Wizard SV Gel and PCR Clean-Up System (Ref: A9282, Promega)</P><UL><LI>pSB1C3 =&gt; 6.4 ng/µL</LI><LI>LDH2 =&gt; 4.8 ng/µL</LI><LI>PCT2 =&gt; 2.9 ng/µL</LI><LI>PhaC3 =&gt; 4 ng/µL</LI><LI>PhaC4 =&gt; 5.8 ng/µL</LI></UL><H6>DNA concentration by water evaporation</H6><UL><LI>pSB1C3 =&gt; 7.1 ng/µL</LI><LI>LDH2 =&gt; 6.5 ng/µL</LI><LI>PCT2 =&gt; 5.5 ng/µL</LI><LI>PhaC3 =&gt; 4.0 ng/µL</LI><LI>PhaC4 =&gt; 5.0 ng/µL</LI></UL><P>DNA concentration still seems not efficient</P><H6>DNA transformation</H6><UL><LI>Thaw 50 µL chemo-competent DH5-α <I>E. coli</I></LI><LI>Add 5 µL DNA (mix ligation with pSEVA 2311 or 224)</LI><LI>Incubation 25 min on ice</LI><LI>Heat shock: 45 s at 42 °C</LI><LI>Incubation 3 min on ice</LI><LI>Add 250 µL LB</LI><LI>Incubation 1h15 at 37 °C with shaking</LI><LI>Spread on plates LB Kanamycin (50 µg/mL) (50 µL of bacteria on one plate and 10 µL on another one for each vector)</LI><LI>Incubation overnight 37 °C</LI></UL><H5 class="blue">4<SUP>th</SUP> of September</H5><H6>Ligation in pSB1C3</H6><TABLE class="center"><TBODY><TR><TH>-</TH><TH>LDH2 (6.5 ng/µL)</TH><TH>PCT2 (6.5 ng/µL)</TH><TH>PhaC3 (4.0 ng/µL)</TH><TH>PhaC4 (5.0 ng/µL)</TH></TR><TR><TD>Vector</TD><TD>4.2 µL</TD><TD>4.2 µL</TD><TD>4.2 µL</TD><TD>4.2 µL</TD></TR><TR><TD>Insert (gene)</TD><TD>2.8 µL</TD><TD>0.5 µL</TD><TD>0.5 µL</TD><TD>0.5 µL</TD></TR><TR><TD>Buffer T4 DNA ligase (10X)</TD><TD>2.39 µL</TD><TD>2.39 µL</TD><TD>2.39 µL</TD><TD>2.39 µL</TD></TR><TR><TD>T4 DNA ligase</TD><TD>3.96 µL</TD><TD>1.95 µL</TD><TD>4.02 µL</TD><TD>1.11 µL</TD></TR><TR><TD>H20 (qs 10 µL)</TD><TD>2.15 µL</TD><TD>4.15 µL</TD><TD>2.08 µL</TD><TD>5.0 µL</TD></TR></TBODY></TABLE><P>Incubation 1 h at 16 °C</P><P>Poor plates LBC (30 µg/mL chloramphenicol)</P><H6>Transformation</H6><UL><LI>Thaw 25 µL chemo-competent DH5-α <I>E. coli</I></LI><LI>Add 4 µL DNA (mix ligation)</LI><LI>Incubation 25 min on ice</LI><LI>Heat shock: 45 s at 42 °C</LI><LI>Incubation 3 min on ice</LI><LI>Add 225 µL LB</LI><LI>Incubation 1 h at 37 °C with shaking</LI><LI>Spread 120 µL on plates LBC</LI><LI>Incubation overnight 37 °C</LI></UL><H3 class="yellow">Week 10</H3><H5 class="blue">7<SUP>th</SUP> of September</H5><H6>Digestion of genes</H6><P>See complete version of the notebook (pdf) for details on the conditions.</P><P>Incubation 1 h at 37 °C and after incubation 20 min at 80 °C.</P><H5 class="blue">8<SUP>th</SUP> of September</H5><H6>Digestion of bacbone Igem</H6><P>Mix: Enzyme Master Mix E+P</P><UL><LI>0.5 µL NEB 2.1 Buffer</LI><LI>0.5 µL BSA (Bovine Serum Albumin)</LI><LI>0.5 µL EcoR1-HF</LI><LI>0.5 µL Pst1</LI><LI>0.5 µL Dnp1</LI><LI>18 µL H<SUB>2</SUB>O</LI></UL><P>Mix of 4 µL of linearized backbone and 4 µL of master mix enzyme</P><P>Incubation of all the solution at 37 °C during 30 min and then, heat deactivation during 20 min at 80 °C.</P><H6>Ligation</H6><TABLE class="center"><TBODY><TR><TH>Genes</TH><TH>length genes</TH><TH>Volume Final</TH><TH>Concentration Final</TH><TH>amount for 1 µL (Cf/Vf)</TH></TR><TR><TD>pSB1C3</TD><TD>2037 bp</TD><TD>8 µL</TD><TD>100 ng</TD><TD>25 ng= 2 µL</TD></TR><TR><TD>LDH_V2</TD><TD>1227 bp</TD><TD>50 µL</TD><TD>1095 ng</TD><TD>21.9 ng/µL</TD></TR><TR><TD>PCT_V2</TD><TD>1612 bp</TD><TD>50 µL</TD><TD>1130 ng</TD><TD>22.6 ng/µL</TD></TR><TR><TD>PhaC_V3</TD><TD>1717 bp</TD><TD>50 µL</TD><TD>1118.25 ng</TD><TD>22.365 ng/µL</TD></TR><TR><TD>PhacC_V4</TD><TD>1720 bp</TD><TD>50 µL</TD><TD>1101.25 ng</TD><TD>22.025 ng/µL</TD></TR></TBODY></TABLE><P>Amount calculation of insert for Ligation with NEBio Calculator:</P><TABLE class="center" style="width:80%"><TBODY><TR><TH>Genes</TH><TH>Amount</TH><TH>Volume for deduced</TH></TR><TR><TD>LDH_V2</TD><TD>15.06 ng</TD><TD>0.7 µL</TD></TR><TR><TD>PCT_V2</TD><TD>19.78 ng</TD><TD>0.9 µL</TD></TR><TR><TD>PhaC_V3</TD><TD>21.07 ng</TD><TD>0.9 µL</TD></TR><TR><TD>PhaC_V4</TD><TD>21.11 ng</TD><TD>0.96 µL</TD></TR></TBODY></TABLE><P><STRONG>Mix of ligation</STRONG></P><UL><LI>2 µL of pSB1C3</LI><LI>Volume calculate of digested insert</LI><LI>1 µL T4 Ligase Buffer</LI><LI>0.5 µL T4 Ligase</LI><LI>qs 10 µL</LI></UL><P>I put the mix at 16 °C while 30 min and after heat kill for 20 min at 80 °C</P><H6>Transformation by heat shock</H6><P>With 2 µL of Ligation mix and 25 µL of Competent Cells.</P><UL><LI>2 µL of Ligation with 25 µL of competent cells.</LI><LI>Put the mix 20 min in the ice. </LI><LI>Put the mix at 42 °C while 45 seconds</LI><LI>2 min on the ice.</LI><LI>Add 950 µL of LB</LI><LI>Recovery 1 h at 37 °C with agitation</LI><LI>7000 rpm during 1 min</LI><LI>Suppression of the flow-throw</LI><LI>Resuspend with the remaining</LI><LI>Spread on plates (put the simple in the middle of the plates)</LI><LI>Incubation overnight 37 °C</LI></UL><H6>Plates preparation</H6><P>1 plates = 25mL of LB Agar + 15 µL of chloramphenicol [30 µg/mL]</P><H6>Preculture</H6><P>3 mL LB liquid medium + 1 µL <I>E. coli</I> DH5-α bacteria from stock at -80 °C</P><H5 class="blue">9<SUP>th</SUP> of September</H5><H6>Preparation of chemo-competent <I>E. coli</I> DH5-α by heat shock</H6><UL><LI>WORK STERILE</LI><LI>Launch a pre-culture of the strain in LB at 37 ° C</LI><LI>Inoculate 25 mL of LB 1/50 (500 μl) with the pre-culture</LI><LI>Let it growth at 37 ° C with stirring until OD(600 nm) = 0.5-0.7</LI><LI>After 1h30, OD= 0.9</LI><LI>For Spectrometer measurements, 1 mL of LB for the blank</LI><LI>A dilution was performed to obtain 0.5 OD(600 nm) and the culture was split into 2 falcons with 27.5 mL of culture</LI><LI>Cells are cooled 10 min on ice</LI><LI>Centrifugate culture 6 min at 4000 rpm at 4 °C</LI><LI>Suppression of the flow-through</LI><LI>Dilution of the pellet in 1/2 volume of cold CaCl2 0.1M (13.75 mL)</LI><LI>Cells cooled on ice during 20 min</LI><LI>Culture centrifugated 6 min at 4000 rpm at 4 °C</LI><LI>Suppression of the flow-through</LI><LI>Resuspension the pellet in 1 / 50e of final volume (550 μl final volume) of cold CaCl2 + 10 % glycerol </LI><LI>Vi= 110 µL</LI><LI>The mix is 440 µL CaCl2+ 110 µL glycerol 50 %</LI><LI>I multiplied by 2 the doses for have more mix</LI><LI>22 aliquots of 50 µL bacteria</LI></UL><H5 class="blue">11<SUP>th</SUP> of September</H5><H6>PCR Amplification of LDH2, PCT2, PHAC3, PHAC4 with Q5 DNA polymerase</H6><P><STRONG>For 50 µL reaction</STRONG></P><UL><LI>25 µL of Q5 Master Mix</LI><LI>2.5 µL of Forward Primer (IG063)</LI><LI>2.5 µL of Reverse Primer (IG064)</LI><LI>0.5 of Template DNA</LI><LI>19.5 of H<SUB>2</SUB>O</LI></UL><P><STRONG>PCR program</STRONG></P><UL><LI>98 °C 30 s</LI><LI>(98 °C 30 s, 55 °C 30 s, 72 °C 1 min) 35 times</LI><LI>72 °C 2 min</LI><LI>10 °C ∞</LI></UL><H6>PCR Colonie LDH2 with One Taq</H6><P>For 25 µL to 1 Reaction --&gt; Vf= 125 µL</P><P><STRONG>Mix</STRONG></P><UL><LI>25 µL of One Taq Standard Reaction Buffer</LI><LI>dNTPs = 2.5 µL</LI><LI>2.5 µL of Forward Primer (IG001)</LI><LI>2.5 µL of Reverse Primer (IG002)</LI><LI>0.625 µL of One Taq DNA Polymerase </LI><LI>91.9 µL of Nuclease Free Water</LI></UL><P><STRONG>PCR program</STRONG></P><UL><LI>94 °C 30 s</LI><LI>(94 °C 30 s, 50 °C 1 min, 68 °C 1 min 45 sec) 30 times</LI><LI>68 °C 5 min</LI><LI>10 °C ∞</LI></UL><H3 class="yellow">Week 11</H3><H5 class="blue">12<SUP>st</SUP> of September</H5><H6>Miniprep colonies LDH</H6><UL><LI>3 µL of LB </LI><LI>Take one of the colony in the plates</LI><LI>Incubated at 37 °C overnight</LI></UL><H6>Digestion Backbone IGEM pSB1C3 linearized by E+P</H6><P><STRONG>Mix: Enzyme Master Mix E+P for 22 reaction</STRONG></P><UL><LI>2 µL NEB 2.1 Buffer</LI><LI>2 µL BSA (Bovine Serum Albumin)</LI><LI>2 µL EcoRI-HF</LI><LI>2 µL PstI</LI><LI>2 µL DnpI</LI><LI>72 µL H<SUB>2</SUB>O</LI></UL><UL><LI>16 µL of linearized backbone and 16 µL of the MasterMix enzyme</LI><LI>Incubation at 37 °C during 30 min </LI><LI>Enzyme inactivation with 20 min incubation at 80 °C.</LI></UL><H6>Precultures</H6><UL><LI>5 mL LB medium</LI><LI>+ 5 µL antibiotic (Kanamycin 50 mg/mL for pSEVA 212S, Spectinomycin 50 mg/mL for pSEVA 424 and Gentamycin 10 mg/mL for pSEVA 628S)</LI><LI>+ some bacteria (sent as bacteria aliquots in LB medium by de Lorenzo)</LI><LI>Incubation 37 °C overnight</LI></UL></DIV><DIV><H5 class="blue">13<SUP>st</SUP> of September</H5><H6>Miniprep of pSEVA plasmids</H6><P>2 aliquots stocks in 50 % glycerol stored at -80 °C</P><P>GenElute Plasmid Miniprep Kit (Sigma, PLN350-1KT)</P><P>3 DNA vectors: pSEVA 424 (equivalent to the pSEVA 224 but with spectinomycin resistance gene instead of kanamycin, pSEVA 212S and pSEVA 628S used for genes deletion in genomic DNA in <I>Pseudomonas putida</I>) </P><H6>Digestion of the “old” gene by E+P</H6><TABLE class="center"><TBODY><TR><TH colspan="3">Ratio 1 : 2</TH></TR><TR><TD>Genes</TD><TD>Amount</TD><TD>Volume for deduced</TD></TR><TR><TD>LDH_V2</TD><TD>30.12 ng</TD><TD>1.37 µL</TD></TR><TR><TD>PCT_V2</TD><TD>39.57 ng</TD><TD>1.71 µL</TD></TR><TR><TD>PhaC_V3</TD><TD>42.15 ng</TD><TD>1.88 µL</TD></TR><TR><TD>PhaC_V4</TD><TD>42.22 ng</TD><TD>1.92 µL</TD></TR></TBODY></TABLE><P><STRONG>Ligation mix</STRONG></P><UL><LI>2 µL of pSB1C3</LI><LI>Volume calculate of digested insert</LI><LI>1 µL T4 Ligase Buffer</LI><LI>0.5 µL T4 Ligase</LI><LI>qs 10 µL</LI></UL><P>I put the mix at 16 °C while 30 min and after heat kill for 20 min at 80 °C</P><H6>Minipreps DNA Purification System kit “Wizard Plus”</H6><UL><LI>Take 2 µL of overnight culture</LI><LI>Centrifuge at top speed for 1 min </LI><LI>Suppression of the flow-throw</LI><LI>Resuspend pellet with 250 µL of Cell Resuspension Solution</LI><LI>Add 250 µL of Cell Lysis Solution and invert 4 times to mix</LI><LI>Add 10 µL of Alkaline Protease Solution and invert 4 time to mix</LI><LI>Incubate 5 min at room temperature</LI><LI>Add 350 µL of Neutralization Solution and invert 4 times to mix</LI><LI>Centrifuge at top speed for 10 min at room temperature</LI><LI>Insert Spin Column into Collection Tube</LI><LI>Decant cleared lysate into Spin Column</LI><LI>Centrifuge at top speed for 1 min</LI><LI>Discard flowthrough</LI><LI>Add 750 µL of Wash Solution and centrifuge at top speed 1 min</LI><LI>Discard flowthrough</LI><LI>Repeat with 250 µL of Wash Solution</LI><LI>Centrifuge at top speed for 2 min </LI><LI>Transfert Spin Column to 1.5 mL tube</LI><LI>Add 100 µL of Nuclease free water to the Spin Column</LI><LI>Centrifuge at the top speed for 1 min</LI><LI>Discard column</LI></UL><TABLE class="center"><TBODY><TR><TH colspan="3">Ratio 1 : 5</TH></TR><TR><TD>Genes</TD><TD>Amount</TD><TD>Volume for deduced</TD></TR><TR><TD>LDH_V2</TD><TD>75.29 ng</TD><TD>3.44 µL</TD></TR><TR><TD>PCT_V2</TD><TD>98.92 ng</TD><TD>4.38 µL</TD></TR><TR><TD>PhaC_V3</TD><TD>105.4 ng</TD><TD>4.71 µL</TD></TR><TR><TD>PhaC_V4</TD><TD>105.5 ng</TD><TD>4.79 µL</TD></TR></TBODY></TABLE><H6>Purification of Amplification PCR LDH2, PCT2, PHAC3, PHAC4</H6><H6>Nanodrop</H6><P><STRONG>For the genes</STRONG></P><TABLE class="center" style="width:80%"><TBODY><TR><TH>Genes</TH><TH>Concentration (ng/µL)</TH><TH>Ratio 260/280</TH></TR><TR><TD>LDH2</TD><TD>62.7</TD><TD>1.78</TD></TR><TR><TD>PCT2</TD><TD>52.8</TD><TD>1.79</TD></TR><TR><TD>PHAC3</TD><TD>50.3</TD><TD>1.76</TD></TR><TR><TD>PHAC4</TD><TD>69.7</TD><TD>1.78</TD></TR></TBODY></TABLE><P><STRONG>For the LDH colonies</STRONG></P><TABLE class="center" style="width:80%"><TBODY><TR><TH>Colonies</TH><TH>Concentration (ng/µL)</TH><TH>Ratio 260/280</TH></TR><TR><TD>LDH_n°1</TD><TD>70</TD><TD>1.92</TD></TR><TR><TD>LDH_n°2</TD><TD>31</TD><TD>2.06</TD></TR></TBODY></TABLE><H6>Digestion “New” genes LDH2, PCT2, PHAC3, PHAC4</H6><P>Digestion by EcoR1-HF+PSt1</P><TABLE class="center"><TBODY><TR><TH>Genes</TH><TH>Quantity of DNA (µL)</TH><TH>Restriction Enzymes</TH><TH>Buffer Digest NEB 2.1 (µL)</TH><TH>H<SUB>2</SUB>O</TH></TR><TR><TD rowspan="2">LDH2 (62.7 ng/µL)</TD><TD rowspan="2">16</TD><TD>1 µL EcoR1-HF</TD><TD rowspan="2">5</TD><TD rowspan="2">27</TD></TR><TR><TD>1 µL Pst1</TD></TR><TR><TD rowspan="2">PCT2 (52.8 ng/µL)</TD><TD rowspan="2">19</TD><TD>1 µL EcoR1-HF</TD><TD rowspan="2">5</TD><TD rowspan="2">24</TD></TR><TR><TD>1 µL Pst1</TD></TR><TR><TD rowspan="2">PHAC3 (63.9 ng/µL)</TD><TD rowspan="2">16</TD><TD>1 µL EcoR1-HF</TD><TD rowspan="2">5</TD><TD rowspan="2">27</TD></TR><TR><TD>1µL Pst1</TD></TR><TR><TD rowspan="2">PhAC4 (69.7 ng/µL)</TD><TD rowspan="2">14.5</TD><TD>1 µL EcoR1-HF</TD><TD rowspan="2">5</TD><TD rowspan="2">28.5</TD></TR><TR><TD>1 µL Pst1</TD></TR><TR><TD rowspan="2">pSB1C3 (240.3 ng/µL)</TD><TD rowspan="2">4.5</TD><TD>1 µL EcoR1-HF</TD><TD rowspan="2">5</TD><TD rowspan="2">38.5</TD></TR><TR><TD>1 µL Pst1</TD></TR></TBODY></TABLE><P>Other digestion</P><TABLE class="center"><TBODY><TR><TH>Genes</TH><TH>Quantity of DNA (µL)</TH><TH>Restriction Enzymes</TH><TH>Buffer Digesr NEB 2.1 (µL)</TH><TH>H<SUB>2</SUB>O</TH></TR><TR><TD rowspan="2">PHAC3</TD><TD rowspan="2">16</TD><TD>1 µL Xba1</TD><TD rowspan="2">5</TD><TD rowspan="2">27</TD></TR><TR><TD>1 µL Pst1</TD></TR><TR><TD rowspan="2">PHAC4</TD><TD rowspan="2">14.5</TD><TD>1 µL Xba1</TD><TD rowspan="2">5</TD><TD rowspan="2">28.5</TD></TR><TR><TD>1 µL Pst1</TD></TR><TR><TD rowspan="2">PCT2</TD><TD rowspan="2">19</TD><TD>1 µL EcoR1-HF</TD><TD rowspan="2">5</TD><TD rowspan="2">24</TD></TR><TR><TD>1 µL Spe1</TD></TR></TBODY></TABLE><P>Incubation at 37 °C during 30 min and then, enzymes deactivation by heat kill for 20 min at 80 °C.</P><H6>PCR Purification</H6><H6>Nanodrop Vf = 50 µL</H6><TABLE class="center"><TBODY><TR><TH>Genes</TH><TH>Concentration (ng/µL)</TH><TH>Ratio 260/280</TH></TR><TR><TD>LDH_V2</TD><TD>62.7</TD><TD>1.78</TD></TR><TR><TD>PCT_V2</TD><TD>52.8</TD><TD>1.79</TD></TR><TR><TD>PHAC_V3</TD><TD>50.3</TD><TD>1.76</TD></TR><TR><TD>PHAC_V4</TD><TD>69.7</TD><TD>1.78</TD></TR></TBODY></TABLE><H5 class="blue">14<SUP>st</SUP> of September</H5><P>The promoter region of the vectors pSEVA 224 and pSEVA 2311 were amplified by PCR with Q5 DNA polymerase with primers allowing to add the prefix and suffix regions at both sides of the amplified region.</P><P><STRONG>Mix</STRONG></P><UL><LI>10 µL Q5 reaction buffer 5X</LI><LI>1 µL dNTP mix</LI><LI>2.5 µL 10 µM Forward Primer (iG063)</LI><LI>2.5 µL 10 µM Reverse Primer (iG064)</LI><LI>1 µL DNA template</LI><LI>0.5 µL Q5 DNA polymerase</LI><LI>32.5 µL Nuclease-free water (qs 50 µL)</LI></UL><P><STRONG>PCR program:</STRONG></P><UL><LI>98 °C 30 s</LI><LI>(98 °C 30 s, 55 °C 30 s, 72 °C 1 min) 35 times</LI><LI>72 °C 2 min</LI><LI>10 °C ∞</LI></UL><P>After checking on agarose 1 % gel, we determined that the fragments were successfully amplified.</P><H6>Digestion</H6><P>The amplified fragments were digested by EcoRI and PstI during 1 h at 37 °C. The reaction was performed in NEB CutSmart buffer.</P><P>The digested mixes were purified by using the kit Wizard SV Gel and PCR Clean-Up System (Ref: A9282, Promega).</P><P>Design Primer for Gibson Assembly -&gt; PR_IG0075 -&gt; 81</P><H5 class="blue">15<SUP>st</SUP> of September</H5><H6>Ligation</H6><P>The 2 fragments were then ligated separately in linear pSB1C3 provided by IGEM Headquarters in a ratio 5:1.</P><P>The ligation mix was used for transformation in E. coli DH5-alpha.</P><H6>Transformation</H6><LI>Thaw 25 µL chemo-competent DH5-alpha E. coli</LI><LI>Add 3 µL DNA (mix ligation)</LI><LI>Incubation 20 min on ice</LI><LI>Heat shock: 45 s at 42 °C</LI><LI>Incubation 3 min on ice</LI><LI>Add 225 µL of LB medium</LI><LI>Incubation 1 h at 37 °C with shaking</LI><LI>Spread 120 µL on plates LBC</LI><LI>Incubation overnight 37 °C</LI><H5 class="blue">16<SUP>th</SUP> of September</H5><H6>PCR colony</H6><P><STRONG>Mix (25 µL total volume reaction):</STRONG></P><LI>12.5 µL DreamTaq PCR Mastermix 2X (ThermoScientific)</LI><LI>2.5 µL 10X DreamTaq Green Buffer (ThermoScientific)</LI><LI>0.25 µL Forward Primer (0.5 µM) iG001</LI><LI>0.25 µL Reverse Primer (0.5 µM) iG002</LI><LI>9.5 µL H2O filtrated (qs 25 µL)</LI><P>+ clones</P><H6>Agarose gel 1 %</H6><H6>Precultures</H6><P>Positive clones selected from the agarose gel results were picked and used to inoculate 3 mL of LBC (25 µg/mL chloramphenicol) medium.</P><P><I>Observation</I></P><P>All the precultures have grown</P><H6>PCR colony</H6><P><STRONG>Mix (25 µL total volume reaction):</STRONG></P><LI>10 µL DreamTaq PCR Mastermix 2X (ThermoScientific)</LI><LI>1.5 µL Forward Primer (0.5 µM) iG001</LI><LI>1.5 µL Reverse Primer (0.5 µM) iG002</LI><LI>6 µL H2O filtrated (qs 25 µL)</LI><P>+ clones</P><H6>Migration map</H6></DIV></DIV><H1>October 2016</H1><DIV class="grid-2"><DIV><H6>Gel shown the last results corresponding to the genes selected</H6><P>The promoters isolated from vectors pSEVA2311 and pSEVA224 were also sent to iGEM HQ after checking by sequencing.</P><H5 class="red">For information, PhaC3 and PCT2 without RBS were successfully cloned in pSB1C3 and sent to iGEM HQ (verified by sequencing)</H5><SPAN style="font-size:1.5em; "><A href="https://static.igem.org/mediawiki/2016/c/cd/T--Evry--end-labbook.pdf" target="_blank"><U>Check the rest of our note book (.pdf)</U></A></SPAN></DIV></DIV></DIV></DIV></DIV></MAIN></DIV><FOOTER><DIV id="footer_content"><DIV id="sponsors"><UL><LI><A target="_black" class="genopole" href="http://www.genopole.fr/?lang=en" title="Genopole">Genopole</A></LI><LI class="wide"><A target="_black" class="idt" href="http://eu.idtdna.com/site" title="Integrated DNA technologies">IDT</A></LI><LI><A target="_black" class="issb" href="http://www.issb.genopole.fr/" title="Institute of Systems &amp; Systhetic biology">iSSB</A></LI><LI><A target="_black" class="ueve" href="http://www.univ-evry.fr/fr/index.html" title="University of Evry-Val-d'Essonne">UEVE</A></LI><LI><A target="_black" class="ibisc" href="https://www.ibisc.univ-evry.fr/en/start" title="Informatique, Biologie Intégrative et Systèmes Complexes">IBISC</A></LI><LI><A target="_black" class="rf" href="http://fr.franceintheus.org/" title="French Ambassy in the United-States">République Française</A></LI><LI><A target="_black" class="crous" href="http://www.crous-paris.fr/" title="Crous">Crous</A></LI><LI class="wide"><A target="_black" class="neb" href="https://www.neb.com/" title="New-England Biolabs">New-England Biolab</A></LI><LI class="wide"><A target="_black" class="geneious" href="http://www.geneious.com/" title="Geneious">Geneinous</A></LI><LI class="wide"><A target="_black" class="mathworks" href="http://mathworks.com/" title="Mathworks">Mathworks</A></LI><LI><A target="_black" class="mie" href="http://www.paris.fr/services-et-infos-pratiques/famille-et-education/etudier-a-paris/maison-des-initiatives-etudiantes-2521" title="Maison des initiatives étudiantes (House of Student initiatives)">MIE</A></LI><LI><A target="_black" class="papet" href="http://www.grandparissud.fr/" title="Grand Paris Sud">Papeterie GPS</A></LI></UL></DIV><SPAN id="pvl"><STRONG>Evry</STRONG>2016.</SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>