<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Evry_Model_Dynamic_Circuit skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Evry/Model/Dynamic/Circuit</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="page_wrap"><HEADER><DIV id="logo_evryigem"><A href="https://2016.igem.org/Team:Evry" title="Homepage"><SPAN>Homepage</SPAN></A></DIV><DIV id="bar-1"><DIV id="igem-logo"><A href="https://igem.org" title="Visit the iGEM website">Visit the iGEM website!</A></DIV><DIV id="social"><P>Feel free to contact us! </P><UL><LI><A class="email" href="mailto:evryigem2016@gmail.com?Subject=Wiki " title="E-mail - evryigem2016@gmail.com">E-mail</A></LI><LI><A class="twitter" href="https://twitter.com/iGEM_Evry" title="Twitter - @iGEM_evry">Twitter</A></LI><LI><A class="facebook" href="https://www.facebook.com/iGEM.Evry.2016/" title="Facebook - /iGEM.Evry.2016/">Facebook</A></LI><LI><A class="insta" href="https://www.instagram.com/igem_evry/" title="Instagram - @igem_evry">Instagram</A></LI></UL></DIV><DIV id="letsplay"><EM>Let's PLAy project</EM> - Bioproduction of PLA</DIV></DIV><NAV id="navbar"><DIV id="nav"><UL id="nav_menu"><LI class="cat_menu show_more"><A href="#">Show menu</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI class="cat_menu dropdown xproj"><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model">Modeling</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Improvements">Improvements</A></LI></UL><LI class="cat_menu dropdown xachiv"><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Medals">Medal fulfillments</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Results">Results</A></LI><LI class="cat_menu dropdown xilab"><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab">Results</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Labbook">Lab book</A></LI><LI class="cat_menu dropdown xsafe"><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety">Project safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Lab">Lab safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Shipment">Shipment safety</A></LI><LI class="cat_menu dropdown xhp"><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Integrated_Practices"> Integrated HP</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Engagement">Engagements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/HP/Events">iGEM events</A></LI><LI class="cat_menu dropdown xteam"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team">Our team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Previous_Evry">iGEM in Evry</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Campus">Campus</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Team/Sponsors">Sponsors</A></LI><LI class="cat_menu"><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></DIV></NAV></HEADER><UL id="show_more"><LI><A href="https://2016.igem.org/Team:Evry">Home</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Project">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Achivements">Achivements</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Interlab/Intro">Interlab</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Safety/Intro">Safety</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Human_Practices/Intro">Human practices</A></LI><LI class="lastl"><A href="https://2016.igem.org/Team:Evry/Team/Intro">Team</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Attributions">Attributions</A></LI></UL><MAIN><DIV id="content-wrap"><DIV id="breadcrumbs"><UL><LI><A href="https://2016.igem.org/Team:Evry">Homepage</A></LI><LI><A href="">Project</A></LI><LI><A href="https://2016.igem.org/Team:Evry/Model">Modeling</A></LI><LI>Dynamic Regulation</LI></UL></DIV><DIV id="content"><H1 class="red">Dynamic Regulation</H1><P><UL id="project_menu" class="pm_modeling"><LI><A href="https://2016.igem.org/Team:Evry/Model/FBA"><SPAN>Flux Balance Analysis</SPAN></A></LI><LI class="current_page"><A href="https://2016.igem.org/Team:Evry/Model/Dynamic"><SPAN>Dynamic Regulation</SPAN></A></LI></UL></P><UL id="project_menu"><LI><A href="https://2016.igem.org/Team:Evry/Model/Dynamic/Kappa"><SPAN>Kappa</SPAN></A></LI><LI class="current_page"><A href="https://2016.igem.org/Team:Evry/Model/Dynamic/Circuit"><SPAN>Circuit</SPAN></A></LI></UL><H2> Dynamic circuit</H2><DIV class="grid-2"><DIV><P>Living systems are problem-solving systems. Multidisciplinary field of Synthetic Biology is trying to develop scalable platforms to engineer biological systems enabling sophisticated sense-and-respond behaviors vastly more efficient, safe, understandable, and predictable. Due to the deep similarities between living cellular modules and electronic parts, biological electronic-like circuits have been developed during the past ten years as main tool in order to design novel biological parts and process information in living cells <A href="#ref1">[1]</A>. 
Practical applications of biological electronic-like circuits include biosensors for pollutant detection and degradation, biosensors for medical diagnostic, smart therapeutics enabling sensing and regulation of drug delivery, and dynamic regulation in metabolic engineering.</P><P>Information processing in living cells involves integrating multiple input signals, performing computations on these signals, storing information in memory, and actuating outputs, as how electrical circuits accomplish. Most circuits take as input small molecules, which are used to activate or inhibit genes expression via riboswitches or transcription factors. Just considering progress made in the last two years, signal processing in biological systems can now rely on elements such as logic gates, load drivers, memory systems, amplifiers, coupling systems or bow-tie architectures, to name a few. </P><P>At the cellular level, biochemical pathways can be modeled as electrical circuits where signals are produced, propagated and consumed with the following equivalents <A href="#ref2">[2]</A>: (i) mass flux as electric current, (ii) concentration as voltage, (iii) stoichiometric conservation as Kirchhoff’s voltage law, and (iv) mass conservation as Kirchhoff’s current law. Additionally, the deep similarity between the Boltzmann thermodynamic equations that describe noisy electronic current flow in sub-threshold transistors and noisy molecular flux in biochemical reactions has been used to map analog circuit motifs in electronics to analog circuit motifs in cells and vice versa <A href="#ref3">[3]</A>. For instance, the increase in gate voltage in a transistor (which lowers energy barriers and increases current flow) is analogous to the increases of enzyme/catalyst concentration in a metabolic reaction (which increases reaction rates).</P><P>In our project, the aim of electrical circuits-like modeling was to integrate metabolic and gene expression levels in order to obtain a profound sight on our production pathway, considering the pathway tightly connected into the cell metabolism and more importantly cell gene expression and regulation network.
In the electronic, possessing the differential equation of many kind of the systems (an electrical chip, a string or a biochemical reaction or regulation), enables one to model the system in a circuit schema. Then, the circuit is solved to study or optimize the system. </P><P>Here we presented the first electrical circuit modelling of the dynamic regulation in the metabolic engineering in the iGEM. Our approach can be applied in order to design regulating system applying biosensors (Using the tool from <A href="#ref5">[5]</A>) and making synthetic feedback loops sophisticating and optimizing the processes in the metabolic engineering. </P></DIV><DIV><P class="center"><A href="https://static.igem.org/mediawiki/2016/d/d8/T--Evry--DynamicCircuit1.png" title="Figure 1" class="fancybox"><SPAN> Click to enlarge</SPAN></A></P><P class="figure"><U>Figure 1.</U> Electrical circuit interpretation of the designed dynamic regulation system.</P><P>The first effort was to interpret the dynamic regulation circuit in an electrical circuit language (Figure 1. Derived from <A href="https://2016.igem.org/Team:Evry/Model/Dynamic" target="_blank"><U>dynamic regulation map</U></A>, the annotations are based on the equations which have been represented in the table). Because we did not have any electrical engineer in our team, we confine ourselves in extracting the differential equation (see the equations) of the evolution of each component from <A href="https://2016.igem.org/Team:Evry/Model/Dynamic">dynamic regulation map</A> and solving them using Python (see the code). Figure 2. illustrates the dynamic behaviour of each component proportional to the time. The mathematical method used for extracting the dynamic equations is from Brian Ingalls lab, University of Waterloo <A href="#ref4">[4]</A>. </P><P class="center"><A href="https://static.igem.org/mediawiki/2016/9/97/T--Evry--DynamicCircuit2.png" title="Figure 2" class="fancybox"><SPAN> Click to enlarge</SPAN></A></P><P class="figure"><U>Figure 2.</U> Output of the Python code solving the equations. (A) The red curve associated with evolution of PLA indicates that the network maximizes the PLA. (B) Increasing the time range of the solution confirms the stability of the all components of the system after a period of time, while PLA is still increasing.</P><P>Figure 2. demonstrates that evolution of the PLA (red curve) increases while all the other components reach a balance after a period of time. Even though we used approximation for the value of all the constants (Ki , di …), we achieved an expected curve equilibrating the enzymes and metabolites as well as maximizing the PLA production. This astonishing results obviously suggest that, natural behaviour of each part is dramatically more important than the value of each enzymatic, interaction or regulation (thermodynamic and kinetic constants). However, in order to model the circuits more precisely, and include more details, the necessity of associating the exact values is unavoidable. Nowadays, there are many computational and practical methods for measuring these constants. It must be emphasized that, using electrical circuit modelling, one will consider more details such as gain, delay, amplification and even memory, along with applying different electronic equivalents as mentioned before. </P></DIV></DIV><H2 class="yellow">Equations</H2><DIV class="grid-2"><DIV><H5>Part 1</H5><P class="center"><A href="https://static.igem.org/mediawiki/2016/0/0b/T--Evry--Equations1DC.png" title="Equations 1" class="fancybox"><SPAN> Click to enlarge</SPAN></A></P></DIV><DIV><H5>Part 2</H5><P class="center"><A href="https://static.igem.org/mediawiki/2016/4/47/T--Evry--Equations2DC.png" title="Equations 1" class="fancybox"><SPAN> Click to enlarge</SPAN></A></P></DIV><DIV><TABLE class="center"><TBODY><TR><TH>Abbreviations and Constants  </TH><TH>Description</TH></TR><TR><TD>Pyr ↔ Pyruvate</TD></TR><TR><TD>Lac ↔ Lactate</TD></TR><TR><TD>LacCoA ↔ Lactyl-CoA</TD></TR><TR><TD>mRNA_Ei  ↔ mRNA of the enzyme Ei</TD></TR><TR><TD>apoLldR </TD><TD>free LldR</TD></TR><TR><TD>aloLldR </TD><TD>LldR bound to the Ligand (Lactate)</TD></TR><TR><TD>di </TD><TD>degradation constants of mRNAs and proteins </TD></TR><TR><TD>β / (1 + [McbR]^n1)</TD><TD>Kinetics of repressed gene transcription by McbR
(and n1 are constants) </TD></TR><TR><TD>α / (1 + [aloLldR]^n2)</TD><TD>Kinetics of activated gene transcription by LldR
( and n2 are constants)</TD></TR><TR><TD> Ki</TD><TD>Thermodynamic constant of the reactions, first-order (translation of mRNA) or second-order reaction </TD></TR><TR><TD> Kcat /Km</TD><TD>Kinetic constants of the enzymes</TD></TR></TBODY></TABLE></DIV></DIV><H2 class="yellow">Python code</H2><DIV id="code_kappa"><P align="center">@author: Clément Nonlinear problem solving using Euler method  for biological engineering purpose &quot;&quot;&quot; # in the current program, dX will be the derivate  # of the concentration of X over time #defining of the global constants Kcat1=1 Km1=1 K=1 K1=1 K2=1 K3=1 K4=1 K5=1 K_1=1 Kcat2=1 Km2=1 Km3=1 Kcat3=1 d1=1 d2=1 d3=1 d4=1 d5=1 d6=1 d7=1 d8=1 n_1=1 n_2=1 beta=1 alpha=1 #defining the basic functions on which the Euler method will be applied #dX must be understood as &quot;the derivate of the concentration of X over time&quot;</P><P align="center"> # pyruvate  def dpyr (t,E1,pyr,Glu):  dpyr = -Kcat1*E1*pyr/(Km1+pyr)+K*Glu  return (dpyr) # lactate def dlactate (t,E1,E2,pyr,lactate,apoLldR,aloLldr):  dlactate = Kcat1*E1*pyr/(Km1+pyr)-K1*lactate*apoLldR+K_1*aloLldr -Kcat2*E2*lactate/(Km2+lactate)  return (dlactate) def dlactylCoa (t,E2,E3,lactate,lactylCoa):  dlactylCoa = Kcat2*E2*lactate/(Km2+lactate)-Kcat3*E3*lactylCoa/(Km3+lactylCoa)  return (dlactylCoa) # PolyLacticAcid def dPLA (t,E3,lactylCoa):  dPLA = Kcat3*E3*lactylCoa/(Km3+lactylCoa)  return (dPLA) # Messenger RNA of the E1 enzyme def dmRNA_E1 (t,repressor,mRNA_E1):  dmRNA_E1 = beta/(1+repressor**n_1)-d1*mRNA_E1  return (dmRNA_E1) # Messenger RNA of the E2 enzyme def dmRNA_E2 (t,aloLldR,mRNA_E2):  dmRNA_E2 = alpha/(1+aloLldR**n_2)-d2*mRNA_E2  return (dmRNA_E2) # Messenger RNA of the E3 enzyme def dmRNA_E3 (t,aloLldR,mRNA_E3):  dmRNA_E3 = alpha/(1+aloLldR**n_2)-d3*mRNA_E3  return (dmRNA_E3) # Messenger RNA of the Repressor def dmRNA_Repressor (t,aloLldR,mRNA_Repressor):  dmRNA_Repressor = alpha/(1+aloLldR**n_2)-d4*mRNA_Repressor  return (dmRNA_Repressor) # Repressor def drepressor (t,mRNA_Repressor,repressor):  drepressor = K2*mRNA_Repressor-d5*repressor  return (drepressor) # def dapoLldR (t,apoLldR,lactate):  dapoLldR = -K1*apoLldR*lactate+K_1*al</P><P>oLldR  return (dapoLldR) # def daloLldR (t,apoLldR,lactate):  daloLldR = K1*apoLldR*lactate-K_1*aloLldR  return (daloLldR) # def dE1 (t,E1,mRNA_E1):  dE1 = K3*mRNA_E1-d6*E1  return (dE1) # def dE2 (t,E2,mRNA_E2):  dE2 = K4*mRNA_E2-d7*E2  return (dE2) # def dE3 (t,E3,mRNA_E3):  dE3 = K5*mRNA_E3-d8*E3  return (dE3) #importing a library allowing of the application of plotting function  import matplotlib.pyplot as plt #defining the Euler method function and its variables # t0 = start time of the simulation # tf = end time of the simulation # Glu0 = concentration of Glucose (considered as constant in this simulation) # n = resolution of the simulation  # N.B: curves will not match with a realistic experiment if &quot;n&quot; is taken  #too small but the program will be too memory consuming if taken too big def Euler(dE1,dE2,dE3,dmRNA_E1,dmRNA_E2,dmRNA_E3,dmRNA_Repressor,dpyr, dlactate,dlactylCoa,drepressor,daloLldR,dapoLldR,dPLA,t0,tf,Glu0,n) : #initialising variables  t=t0  Glu=Glu0  E1,E2,E3=1,1,1  mRNA_E1,mRNA_E2,mRNA_E3,pyr,lactate,lactylCoa,PLA,mRNA_Repressor,repressor,apoLldR,aloLldR=0,0,0,0,0,0,0,0,0,0,0 #slicing time in n parts. For each part, the evolution of derivate will be # considered as null  h=(tf-t0)/float(n) #creating a list of the time stamps (used as absciss for plotting the curves)  temps=[t0] #introducing lists that will store the consecutive values of the concentrations  # for each of the products  LE1=[1]  LE2=[1]  LE3=[1]  LmRNA_E1=[0]  LmRNA_E2=[0]  LmRNA_E3=[0]  Lpyr=[0]  Llactate=[0]  LlactylCoa=[0]  LPLA=[0]  LmRNA_Repressor=[0]  Lrepressor=[0]  LapoLldR=[0]  LaloLldR=[0] #Starting of the real simulation: #runs n times the computation  for i in range(n) : # &quot;+=&quot; means increment, X will take the value of X+h*dX # where &quot;h&quot; is the little &quot;time slice&quot; on which dX is # considered as the constant  pyr += h*dpyr (t,E1,pyr,Glu)  lactate += h*dlactate (t,E1,E2,pyr,lactate,apoLldR,aloLldR)  lactylCoa += h*dlactylCoa (t,E2,E3,lactate,lactylCoa)  PLA += h*dPLA (t,E3,lactylCoa)   mRNA_E1 += h*dmRNA_E1 (t,repressor,mRNA_E1)  mRNA_E2 += h*dmRNA_E2 (t,aloLldR,mRNA_E2)  mRNA_E3 += h*dmRNA_E3 (t,aloLldR,mRNA_E3)  mRNA_Repressor += h*drepressor (t,mRNA_Repressor,repressor)  repressor += h*drepressor (t,mRNA_Repressor,repressor)  apoLldR += h*dapoLldR (t,apoLldR,lactate)  aloLldR += h*daloLldR (t,apoLldR,lactate)  E1 += h*dE1 (t,E1,mRNA_E1)  E2 += h*dE2 (t,E2,mRNA_E2)  E3 += h*dE3 (t,E3,mRNA_E3) #storing current time value in the &quot;time-list&quot;  temps.append(t) # storing concentration values in each dedicated list  LE1.append(E1)  LE2.append(E2)  LE3.append(E3)  LmRNA_E1.append(mRNA_E1)  LmRNA_E2.append(mRNA_E2)  LmRNA_E3.append(mRNA_E3)  Lpyr.append(pyr)  Llactate.append(lactate)  LlactylCoa.append(lactylCoa)  LPLA.append(PLA)  LmRNA_Repressor.append(mRNA_Repressor)  Lrepressor.append(repressor)  LapoLldR.append(apoLldR)  LaloLldR.append(aloLldR) #increasing time by adding &quot;h&quot; for the next run  t+=h #plotting all the curves # if you want to see selected curves (not all of them), # you just have to put a &quot;#&quot; before the curves you don't want to see  plt.plot(temps,LE1)  plt.plot(temps,LE2)  plt.plot(temps,LE3)  plt.plot(temps,LmRNA_E1)  plt.plot(temps,LmRNA_E2)  plt.plot(temps,LmRNA_E3)  plt.plot(temps,Lpyr)  plt.plot(temps,Llactate)  plt.plot(temps,LlactylCoa)  plt.plot(temps,LPLA)  plt.plot(temps,LmRNA_Repressor)  plt.plot(temps,Lrepressor)  plt.plot(temps,LapoLldR)  plt.plot(temps,LaloLldR) &quot;&quot;&quot; USER GUIDE: to run the simulation, just copy and paste the following line Euler(dE1,dE2,dE3,dmRNA_E1,dmRNA_E2,dmRNA_E3,dmRNA_Repressor,dpyr,dlactate, dlactylCoa,drepressor,daloLldR,dapoLldR,dPLA,t0,tf,Glu0,n) in the console, making sure you choose values for &quot;t0&quot;, &quot;tf&quot;, &quot;Glu&quot; and &quot;n&quot; &quot;&quot;&quot;</P></DIV><DIV id="references"><H2 class="red">References</H2><OL><LI>Brophy, J.A. and C.A. Voigt, Principles of genetic circuit design. Nat Methods, 2014. 11(5): 508-20.</LI><LI>May, E.E. and R.L. Schiek, BioXyce: an engineering platform for the study of cellular systems. IET Syst Biol, 2009. 3(2): 77-89.</LI><LI>Sarpeshkar, R., Analog synthetic biology. Philos Trans A Math Phys Eng Sci, 2014. 372(2012): p. 20130110.</LI><LI>Brian Ingalls, from University of Waterloo. Website: https://uwaterloo.ca/applied-mathematics/people-profiles/brian-ingalls (10/10/2016)</LI><LI>Delépine, Baudoin, et al. &quot;SensiPath: computer-aided design of sensing-enabling metabolic pathways.&quot; Nucleic acids research (2016): gkw305.</LI></OL></DIV></DIV></DIV></MAIN></DIV><FOOTER><DIV id="footer_content"><DIV id="sponsors"><UL><LI><A target="_black" class="genopole" href="http://www.genopole.fr/?lang=en" title="Genopole">Genopole</A></LI><LI class="wide"><A target="_black" class="idt" href="http://eu.idtdna.com/site" title="Integrated DNA technologies">IDT</A></LI><LI><A target="_black" class="issb" href="http://www.issb.genopole.fr/" title="Institute of Systems &amp; Systhetic biology">iSSB</A></LI><LI><A target="_black" class="ueve" href="http://www.univ-evry.fr/fr/index.html" title="University of Evry-Val-d'Essonne">UEVE</A></LI><LI><A target="_black" class="ibisc" href="https://www.ibisc.univ-evry.fr/en/start" title="Informatique, Biologie Intégrative et Systèmes Complexes">IBISC</A></LI><LI><A target="_black" class="rf" href="http://fr.franceintheus.org/" title="French Ambassy in the United-States">République Française</A></LI><LI><A target="_black" class="crous" href="http://www.crous-paris.fr/" title="Crous">Crous</A></LI><LI class="wide"><A target="_black" class="neb" href="https://www.neb.com/" title="New-England Biolabs">New-England Biolab</A></LI><LI class="wide"><A target="_black" class="geneious" href="http://www.geneious.com/" title="Geneious">Geneinous</A></LI><LI class="wide"><A target="_black" class="mathworks" href="http://mathworks.com/" title="Mathworks">Mathworks</A></LI><LI><A target="_black" class="mie" href="http://www.paris.fr/services-et-infos-pratiques/famille-et-education/etudier-a-paris/maison-des-initiatives-etudiantes-2521" title="Maison des initiatives étudiantes (House of Student initiatives)">MIE</A></LI><LI><A target="_black" class="papet" href="http://www.grandparissud.fr/" title="Grand Paris Sud">Papeterie GPS</A></LI></UL></DIV><SPAN id="pvl"><STRONG>Evry</STRONG>2016.</SPAN></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>