<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Stockholm 2018 Wiki</TITLE></P><DIV id="main-content" class="container"><DIV class="row justify-content-center"><DIV class="col-md-12 col-lg-8"><DIV class="content"><H1><SPAN class="contentHeading"><SPAN class="bigFirst">R</SPAN>esults</SPAN></H1><P class="lead">Our research was essentially divided into four different areas. In this page, we reveal which results we obtained from each individual step. Click in the names of the sections to learn more.</P><A class="subheading blue" data-toggle="collapse" data-target="#laccase">Laccase Production</A><DIV class="innerp collapse" id="laccase"><H4>Key Achievements</H4><UL><LI>Production of active wild-type <I>Trametes versicolor (T. versicolor)</I> laccase.</LI><LI>Partial purification of wild-type <I>T. versicolor</I> laccase.</LI><LI>Integration of mutant <I>T. versicolor</I> laccase gene into <I>Pichia pastoris (P. pastoris)</I> genome.</LI></UL><H4>Cloning of the Wild-Type Laccase Gene</H4><P>We successfully isolated the BioBrick (BBa_K500002) from the DNA distribution plate. Using overhang PCR, we modified it by adding restriction sites, a stop codon and a 6xHis tag (Figure 1A). The modified BioBrick was then cloned into the pPICZa A vector and confirmed by colony PCR and sequencing (Figure 1B).</P><P class="legend"><B>Figure 1.</B> (A) (left) PCR product (modified BioBrick) on a 1% agarose gel. Both wells contain the same reaction. (B) (right) Colony PCR on colonies 1-10 and a control colony (from a transformation plate with only vector + ligase). Colonies 2, 3 and 5-9 show a band of the expected size (2053bp).</P><H4>Transformation and Expression of Wild-Type Laccase</H4><P>The plasmid containing the modified BioBrick was transformed into <I>P. pastoris</I> X33 by electroporation and integration of the plasmid into the genome was confirmed by colony PCR (Figure 2).</P><P class="legend"><B>Figure 2.</B> Colony PCR products on a 1% agarose gel. The lanes 1 to 28 contain the transformed clones 1 to 28 and C is the control, which was the untransformed X33 strain. In the upper lanes (A), the PCR done with both AOX1 primers can be seen, and the expected size of the band is 2053 bp. In the lower lanes (B) are the PCR products from the 5’AOXI and the gene-specific reverse primer and the expected band size is 1892 bp. </P><P>Fourteen clones were picked to screen for enzyme production by cultivation in 
<A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title="Buffered-Methanol complex Medium; methanol induces expression from the AOX1 promoter">BMGY &amp; BMMY medium</A>, with daily addition of 1% methanol and 0.2 mM copper sulfate. Samples were taken and analyzed daily by measuring ABTS oxidation. On the second day of screening, we started observing a colour change in the cuvettes after 10 to 30 minutes for most clones (Figure 3). On the fourth day, a clear increase in absorbance was observed after 30 minutes. On the fifth day the increase in absorbance was even higher (Figure 4). </P><P class="legend"><B>Figure 3.</B> ABTS assay cuvettes after 16 hours at room temperature, containing 100µl culture supernatant (clones 1-14, day 2), 100µl 2mM ABTS and 800µl citric acid-phosphate buffer pH 4.0.   Cuvette labelled 31 contains only ABTS and buffer, cuvette labelled 32 contains only supernatant and buffer. Cuvette labelled C contains the wild-type control supernatant.</P><P class="legend"><B>Figure 4.</B> Difference in absorbance at 420nm measured at t=0 min vs. t=30 min for clone 1 to 14 and the control on day 4 and 5 of screening. Activity assay performed with 0.2mM ABTS at pH 4.0 and room temperature.</P><P>Unfortunately, we could not detect any bands of the expected size for our protein on SDS-PAGE or Western blot compared to the control culture. It is likely that the enzyme is expressed at low levels, explaining why it is not observed on an SDS-PAGE gel with Coomassie staining. Moreover, it is possible that the His-tag is obscured by the protein fold, making it difficult to detect with anti-His antibodies in Western blot [1].</P><P>The highest enzyme activity was observed for clone 2 on day 5. Therefore, we selected this clone for enzyme production and cultivated it on a larger scale. A control culture (untransformed X33 <I>P. pastoris</I>) was also cultivated in the same way. After five days, the supernatant was collected by centrifugation. We performed the ABTS activity assay again on the supernatant and observed a clear difference compared to the control supernatant (Figure 5).</P><P class="legend"><B>Figure 5.</B> Oxidized ABTS product concentration over time in 40 µL citrate phosphate buffer pH 4, 30 µL 2mM ABTS and 30 µL culture supernatant. The assay was performed at 30°C. Triplicates were performed for both clone 2 and the control. Error bars (grey) indicate the standard deviation (n=3). Statistically significant result is denoted with ****p≤0.0001 (comparing the mean of each group with a Mann-Whitney U test).</P><P>We also performed a second screening with more clones, but did not find a clone showing a higher level of enzyme activity than clone 2 in the ABTS assay.</P><H4>Purification</H4><P>To purify our enzyme from the supernatant, we performed <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title="A method used to purify proteins fused with a polyhistidine tag, based on affinity of the tag to metal ions chelated to the column resin">immobilized metal affinity chromatography</A> (IMAC). After IMAC, we could see more than one band on the SDS-PAGE gel (Figure 6). Our protein had an expected size of 55 kDa, without glycosylation. However, since <I>P. pastoris</I> is known to hyperglycosylate proteins [2], it is likely that our enzyme would migrate at a higher band size. To separate our protein from the other purified proteins in the fraction, we also performed <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title=" Also known as gel filtration, separates proteins based on size.">size exclusion chromatography</A> (SEC) and <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title="Technique used to identify proteins by peptide mass fingerprinting">MALDI-TOF</A>. However, we did not succeed in purifying our protein using these methods. It is possible that our protein had precipitated at this point, as we noticed precipitation in the tubes. Also, storage in imidazole and the multiple freezing/thawing cycles could have affected the protein stability.</P><P class="legend"><B>Figure 6.</B> Coomassie staining of SDS-PAGE gel on fraction 4-7 obtained from IMAC with the culture supernatant.</P><H4>Cloning of the Mutant Laccase Gene</H4><P>For the production of <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title=" laccase-2 with Phe-162, Phe-332 and Phe-337 mutated to isoleucine">our mutant laccase</A> we amplified the gBlock we received from IDT and cloned it into the pPICZa A vector. We confirmed the cloning by colony PCR (Figure 7) and sequencing.</P><P class="legend"><B>Figure 7.</B> Colony PCR on colonies 1-10, a positive control (plasmid with mutant laccase BioBrick) and a negative control (plasmid without insert). Colonies 1-3, 5, 7-8 and 10 show a band of the expected size (2053 bp).</P><H4>Transformation and Expression of Mutant Laccase</H4><P>The plasmid containing the mutant laccase gene was transformed into X33 <I>P. pastoris</I> by electroporation. The integration of the plasmid into the genome was confirmed by colony PCR (Figure 8).</P><P class="legend"><B>Figure 8.</B> Colony PCR products on a 1% agarose gel. The lanes 1 to 13 contain the clones 8 to 20 with the 5’AOXI and the reverse gene specific primer. C1 (control 1) is the plasmid with the same primers. C2 is the negative control with the untransformed X33 strain and C3 is the untransformed X33 strain with both AOXI primers. </P><P>Eleven clones were picked to screen for enzyme production by cultivation in BMGY and BMMY. Samples were taken and analyzed daily. However, during five days of screening, we could not measure any change in absorbance. Also, we did not observe any color change after leaving the cuvettes overnight, indicating that no active enzyme was expressed by any of the clones.</P><P>Since the transformation was successful and colony PCR showed that the construct integrated into the genome, we believe it is likely that the mutant laccase is inactive. As the plasmid construct was identical to the construct used for (successful) expression of the wild-type laccase, we consider it unlikely that expression was the problem. Instead, we believe that the enzyme was misfolded, either due to the introduced mutations or codon optimization [3]. In fact, we conducted further computational analysis to investigate whether our suspicions were correct. You can find a discussion about this <A href="/Team:Stockholm/Model"><U>on the modeling results page</U></A>.</P><H4>References</H4><OL><LI>Debeljak N, Feldman L, Davis KL, Komel R, Sytkowski AJ. Variability in the Immunodetection of His-tagged Recombinant Proteins. Analytical biochemistry. 2006;359(2):216-223.</LI><LI>Darby R.A.J., Cartwright S.P., Dilworth M.V., Bill R.M. Which Yeast Species Shall I Choose? Saccharomyces cerevisiae Versus Pichia pastoris (Review). In: Bill R., editor. Recombinant Protein Production in Yeast. Methods in Molecular Biology (Methods and Protocols), vol 866. Humana Press; 2012.</LI><LI>Zhao F, Yu C, Liu Y. Codon usage regulates protein structure and function by affecting translation elongation speed in Drosophila cells. Nucleic Acids Research. 2017;45(14):8484-8492. doi:10.1093/nar/gkx501.</LI></OL></DIV><A class="subheading cyan" data-toggle="collapse" data-target="#activity">Enzymatic Activity</A><DIV class="innerp collapse" id="activity"><H4>Key Achievements</H4><UL><LI>Determination of kinetic parameters of our wild-type laccase for the model substrate ABTS.</LI><LI>Determination of the optimal conditions for the <I>T. versicolor</I> laccase.</LI></UL><H4>Kinetic Assay (Michaelis-Menten)</H4><P>We determined <I>K<SUB>M</SUB></I> and <I>v<SUB>max</SUB></I>  for our wild type laccase by performing non-linear regression of a saturation curve on experimental data (Michaelis-Menten model, Figure 1).
</P><P class="legend"><B>Figure 1.</B> A comparison of the enzymatic activity in different substrate concentrations at pH 4.0, 30 °C. Enzyme concentration was approximately 0.0069 mg/ml in the reactions. Error bars indicate standard deviation (n=3). The blue line represents the Michaelis-Menten model obtained from non-linear regression (R<SUP>2</SUP> = 0.9952).</P><P>The kinetic parameters found with non-linear regression were:</P><UL><LI><I>K<SUB>M</SUB></I> =  0.15 ± 0.0091 mg/ml ABTS</LI><LI><I>v<SUB>max</SUB></I> = 29 ± 0.84 μM/min</LI></UL><P>To determine the turnover number <I>k<SUB>cat</SUB></I>, the enzyme concentration must be known.  After IMAC purification of our wild-type laccase, another protein remained still in the eluted fractions. To determine the concentration of our laccase, we attempted to further purify the protein using preparative <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title=" A chromatography method separating proteins based on size.">Size Exclusion Chromatography</A>. However, the concentration of the enzyme was too low to be detected by the machine. One reason could be that the protein had precipitated due to freezing and thawing several times in imidazole (the elution buffer used in IMAC) and therefore was removed when the fractions were filtered before the chromatography. </P><P>After this, we purified more protein by IMAC and made a standard curve of different concentrations of the standard protein BSA (bovine serum albumin) on SDS-PAGE (Figure 2). However, the protein concentrations of our eluted fractions were too low to determine the concentration in this way.  </P><P class="legend"><B>Figure 2.</B> Relationship of band intensity and protein concentration in SDS-PAGE. Blue line represents linear regression of experimental data (n=1, R<SUP>2</SUP> = 0.9636).</P><P>Instead, the laccase concentration was determined by the fraction of total intensity on the gel. Using Nanodrop, the total protein concentration in the elution fraction from IMAC was determined to 0.65 mg/ml. The fraction of laccase on the gel was 35% based on the band intensities. In this way, the laccase concentration in the fraction was estimated to be 0.23 mg/ml. The molar mass of laccase is estimated to be 55 kDa, without glycosylations.</P><P>From the Michaelis-Menten model fitted to our experimental data, and the estimated laccase concentration, <I>k<SUB>cat</SUB></I> was determined to be 4.2 s<SUP>-1</SUP>. In addition, with the help of a supercomputer we theoretically calculated that the wild-type enzyme has a <I>k<SUB>cat</SUB></I> of 4.9 s<SUP>-1</SUP>. These values are strikingly similar, which confirms the accuracy of our model!
</P><H4>Optimal pH for Laccase Activity</H4><P>To determine the optimal pH for the enzymatic activity of laccase, we used a commercial laccase from <I>T. versicolor</I> for multiple assays. </P><P>The highest observed activity was at pH 4 (Figure 3).  The laccase seems to favor an acidic environment in general, most likely because the residues which can form hydrogen bonds to the substrate are protonated then (read more about this on our <A class="link" href="https://2018.igem.org/Team:Stockholm/Model">Model page</A>). </P><P class="legend"><B>Figure 3.</B> A comparison of enzymatic activity in different pH. The assay was performed at room temperature with 0.05 mg/ml ABTS and 0.05 mg/ml commercial laccase (not pure). Error bars represent the standard deviation. A statistically  significant difference in activity between groups is denoted as ** for p&lt;0.01 and **** for p&lt;0.0001 (one-way ANOVA followed by Dunnett’s post hoc test to correct for multiple comparisons, n=3).</P><P>In addition, we performed a stability assessment around the laccase’s optimum pH (Figure 4). The initial enzymatic activity was measured at pH 3, 4 and 5. The enzymatic activity was measured again after a 30 minute incubation period. We did not find a statistically significant difference in activity before and after incubation (one-way ANOVA multiple comparisons test).</P><P class="legend"><B>Figure 4.</B> A comparison of enzyme stability after incubation at different pH. The assay was performed at room temperature with 0.05 mg/ml ABTS and 0.05 mg/ml commercial laccase (not pure). Error bars indicate standard deviation. We did not find a significant difference in activity before and after incubations (one-way ANOVA multiple comparisons test followed by Dunnet’s post hoc test, n=3).</P><H4>Optimal Temperature for Laccase Activity</H4><P>To determine the optimal temperature for the enzymatic activity of laccase, we used a commercial laccase from <I>T. versicolor</I> for multiple assays. </P><P>The highest observed enzymatic activity occurred at 50 °C (Figure 5). As we mentioned before in the experiments section, these findings could also be attributed to the oxygen dissolving and diffusion rate. However, according to our model the rate-determining step does not involve oxygen, and the liquid is saturated with oxygen. Therefore, the oxygen dissolving rate and diffusion rate should not affect the reaction rate.</P><P class="legend"><B>Figure 5.</B> A comparison of enzymatic activity in different temperatures with 0.05 mg/ml ABTS and 0.05 mg/ml commercial laccase (not pure). The assay was performed in a temperature-controlled spectrophotometer, in citric acid-phosphate buffer which had been heated or cooled in a water bath prior to the assay to the desired temperature. Error bars represent standard deviation. When increasing the temperature of the reaction, we observed a statistically significant increase in activity (linear regression, p&lt;0.0001).</P><P>From the obtained results, high temperature seems to give high initial enzymatic activity. However, we are also interested in the long term activity, and therefore we did a stability assessment around the enzyme’s optimum temperature (Figure 6). The initial activity was measured in room temperature followed by a 30 minute incubation of aliquots at 40 °C, 50 °C and 60 °C respectively. After this, the enzymes were cooled down on ice and the activity was again measured in room temperature. </P><P class="legend"><B>Figure 6.</B> A comparison of enzyme stability after incubation at different temperatures. The assay was performed in room temperature in citric acid-phosphate buffer pH 4. ABTS concentration was 0,05 mg/ml, and concentration of commercial laccase (not pure) was 0,05 mg/ml. Error bars represent standard deviation. A statistically  significant difference in activity is denoted as ****p≤0.0001 (one-way ANOVA followed by Dunnett’s post hoc test to correct for multiple comparisons, n=3).</P><H4>Deactivation Methods</H4><P>We could successfully inactivate the commercial laccase by both increasing and lowering the pH to 7 and 1 respectively, followed by incubation for one hour at that pH and subsequent restoration of the original pH. To further validate our quenching method, this was tested at different laccase and ABTS concentrations. Representative results of some of the tested setups are shown in Figure 7. As shown, activity of the enzyme was significantly eradicated after our pH quenching treatments.</P><P class="legend"><B>Figure 7.</B> Enzymatic activity shown by ABTS oxidation (0.1 mM ABTS, other concentrations were also tested and similar results were obtained). Activity before quenching refers to the initial measured enzymatic activity before treating the sample. Activity after restoring the pH refers to the measured enzymatic activity after incubation for one hour at one of the specific pHs followed by restoration of the initial pH value. Activity was also tested 1.5 hours after pH restoration. Statistically significant results are denoted with ***p≤0.005 (one-way ANOVA test followed by Dunnet’s post hoc test).</P><P>As demonstrated, our method for laccase inactivation works and therefore was applied when necessary throughout the project.</P><H4>Sulfamethoxazole Degradation</H4><P>The aim of our project is to develop a super enzyme that can remove the antibiotic sulfamethoxazole  (SMX) faster than the wild type enzyme. Natural degradation of pharmaceuticals such as SMX by laccases has been reported in literature, however degradation is carried out at a very slow rate. [1]  We attempted to test the rate of removal of SMX by commercial laccase in the optimal conditions found from the activity and stability assays described earlier on this page (in 40 °C and pH 4). Three reactions and three controls, containing only SMX, were set up for each reaction time (72 hours, 1 week, 2 weeks), giving a total of 18 reactions and controls. </P><P>Since the reactions will run a long time in warm conditions, it is important to both consider oxygen availability and evaporation. A closed lid on the reaction would mean a closed environment, risking a shortage of oxygen. No lid would mean fast evaporation, which changes the concentration of SMX in the reaction vial. We tried a compromise with half-closed lids, not tightly sealed to let oxygen in. This still gives a potentially uneven evaporation of buffer.</P><P>The concentration of SMX in the different vials were determined with <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title="A chromatography method where the stationary phase is hydrophobic, which separates compounds based on their hydrophobicity">reverse phase HPLC</A>.
The results from the removal test (Figure 8) showed a significant change in SMX concentration after one week, when comparing the vials containing laccase and SMX with the controls only containing SMX (One-way ANOVA and multiple comparisons test, p&lt;0.0001, n=3). After only 72 hours no significant change is observed. Strangely, after two weeks we didn’t observe any significant change between the control and reaction vials. The removal of SMX after one week could be a false positive. The inconclusive results could also occur due to uneven evaporation or oxygen limitations in different vials.</P><P class="legend"><B>Figure 8.</B> Sulfamethoxazole (SMX) removal assay. The reactions were performed at 40 °C and pH 4 with 0.1 mg/ml SMX and 0.1 mg/ml commercial laccase (not pure). Error bars indicate standard deviation. Significant difference is marked with ****p≤0.0001 (comparing to control by one-way ANOVA followed by Dunnet’s post hoc test, n=3).</P><H4>Outlook</H4><P>To determine K<SUB>cat</SUB> and K<SUB>M</SUB> more accurately, the enzyme concentration needs to be exactly determined. Ideally, the elution fraction from the IMAC purification could be further purified with SEC to separate the two proteins. After that, the exact laccase concentration could be determined.</P><P>When comparing the SMX removal assay with the eco-toxicity assay, the only difference in the reactions are the concentrations of SMX and laccase. Since we observed significant results in the ecotoxicity assays but inconclusive data in the removal assay, we suspect that this could be due to the difference in concentration. We expect that if we analyze the reactions in higher concentrations with HPLC, we would be able to see a consistent removal of SMX by laccase.


            </P></DIV><A class="subheading purple" data-toggle="collapse" data-target="#toxicity">Ecotoxicity Testing</A><DIV class="innerp collapse" id="toxicity"><H4>Key Achievements</H4><UL><LI>Showing a decrease in growth inhibition (reduced ecotoxic effect) for transformation products (TPs) generated from commercial laccase compared to pure SMX.</LI><LI>Showing a reduced ecotoxic potency (lower <A href="" data-toggle="tooltip" onclick="return false;" data-placement="top" title="EC50 is a measurement of the potency of a drug or a toxicant. The EC50 is the value in which the drug has reached 50% of its maximal capacity">EC50</A>) of TPs generated from wild-type laccase compared to pure SMX.</LI></UL><H4>Growth Inhibition Assay with Commercial Laccase</H4><P>We successfully showed that TPs generated after 3 days from a reaction with SMX and commercial laccase from <I>T. versicolor</I> are less toxic than pure SMX and do not inhibit growth to the same extent as pure SMX. This suggests that SMX is modified to a less harmful product (see figure 1B). These results can be seen for the reactions with an SMX concentration of 10.1 mg/mL, but not for the reactions with 5 mg/mL (see figure 1A).</P><P class="legend"><B>Figure 1.</B> TPs generated from reaction with 10.1 mg/mL SMX shows reduced ecotoxic effect compared to pure SMX, while 5 mg/mL do not. (A) Reactions containing initial concentration of 5 mg/mL SMX. There is no observed difference between TPs and pure SMX, while pure SMX is significantly reduced compared to <I>E. coli</I>.  (B) Reactions containing initial concentration of 10.1 mg/mL SMX. TPs are seen to be significantly less ecotoxic than pure SMX, not inhibiting bacterial growth to the same extent. All TPs are taken from day 3, pH 4.5 and 50°C. Growth inhibition assays were analysed using a non-parametric ANOVA comparing the mean of each column with the mean of every column (columns <I>E. coli</I>, SMX, TPs). A Dunnett’s post hoc test was used to correct for multiple comparisons. Statistically significant results are denoted with *p≤0.05, **p≤0.01, ***p≤0.001, ****p≤0.0001</P><P>The reason for why the concentration of 5 mg/mL did not show any differences is due to the lower concentration, and that the time frame was not long enough to render the nontoxic TPs.</P><P>However, we could not determine a significant difference between the TP reaction generated by our wild-type laccase, compared to pure SMX (see figure 2A-B). This could be due to several reasons:</P><UL><LI>Too low concentrations of the enzyme. Since we used purified fractions with unknown concentrations, it was difficult to estimate how much would be needed of the fractions for the reactions</LI><LI>Too short time of interaction between the wild-type laccase to the SMX reactions. The time points tested were both 3 days and 7 days. The results from 3 days exposure is seen in figure 2A-B below</LI><LI>Decreased temperature compared to the commercial laccase reactions, which were performed in 50 °C, whereas the wild-type laccase reactions were performed in 40 °C</LI></UL><P class="legend"><B>Figure 2.</B> No observed difference between TPs generated from reaction with 5 mg/mL or 7.6 mg/mL SMX compared to pure SMX using wild-type laccase. (A) Reactions containing initial concentration of 5 mg/mL SMX. TPs are seen to significantly reduce the growth of <I>E. coli</I> compared to <I>E. coli</I> alone. This is also the case for pure SMX compared to <I>E. coli</I>. However, no difference is observed between TPs and pure SMX. 
(B)  Reactions containing initial concentration of 7.6 mg/mL SMX. TPs are seen to significantly reduce the growth of <I>E. coli</I> compared to <I>E. coli</I> alone. This is also the case for pure SMX compared to <I>E. coli</I>. However, no difference is observed between TPs and pure SMX. All TPs are taken from day 3, pH 4.5 and 40°C. Growth inhibition assays were analysed using a non-parametric ANOVA comparing the mean of each column with the mean of every column (columns <I>E. coli</I>, SMX, TPs). A Dunnett’s post hoc test was used to correct for multiple comparisons. Statistically significant results are denoted with *p≤0.05, **p≤0.01, ***p≤0.001, ****p≤0.0001
</P><H4>Testing the Ecotoxic Potential of TPs and Pure SMX on <I>Aliivibrio fischeri</I></H4><P>We were able to successfully measure the EC50 values for both the transformation reactions with our wild-type laccase and pure SMX. We tested TPs generated from two different concentrations of SMX and wild-type laccase: a low concentration of 5 mg/mL and a high concentration of 7.6 mg/mL SMX. This was to see whether there is a difference in the activity of the laccase or a difference in ecotoxic profile in the different concentrations. Using the bioluminescent bacteria <I>A. fischeri</I>, we were able to measure light inhibition which correlates with <I>A. fischeri</I> death. SMX is toxic for <I>A. fischeri</I> which is shown since the EC50 values were approximately 50 mg/L for pure SMX for both 5 mg/mL and 7.6 mg/mL of SMX. For the TPs, we observed a 7-fold increase of the EC50. This means that there is significant proof that the WT laccase is degrading SMX, and the products of the reaction are not toxic for the environment.</P><P>After establishing that the ecotoxic potency of TPs are lower than pure SMX we observed a possible saturation of the laccase when adding the high SMX concentration compared to the lower one. The EC50 of the low SMX concentration was seen to be higher than the EC50 of high SMX. To further assess the saturation point more dilutions would be necessary, as well as further increasing the number of the sample size (TPs tested and replicates).</P><P>Below we give the EC50 values that we got using the Excel sheet provided by the BioTox™ WaterTox™ EVO Kit 1243-500 from EBPI’s line of toxicity testing kits. The Excel sheet follows the ISO Standard 11348-3: 2007 to determine toxicity in water-soluble samples.</P><P><B>Table 1.</B> EC50 values obtained from pure SMX and TPs generated from wild-type laccase and SMX</P><DIV class="table-responsive-sm"><TABLE class="table"><THEAD><TR><TH scope="col">Sample</TH><TH scope="col">EC50 value</TH></TR></THEAD><TBODY><TR><TD>WT laccase + SMX (16.8 mg/mL)</TD><TD>362.77 mg/L</TD></TR><TR><TD>SMX (16.8 mg/mL)</TD><TD>50.32 mg/L</TD></TR><TR><TD>WT laccase + SMX (25.3 mg/mL)</TD><TD>339.01 mg/L</TD></TR><TR><TD>SMX (25.3 mg/mL)</TD><TD>52.36 mg/L</TD></TR><TR><TD>DMSO</TD><TD>No EC50 value obtained</TD></TR></TBODY></TABLE></DIV></DIV><A class="subheading blue" data-toggle="collapse" data-target="#immobilization">Laccase Immobilization</A><DIV class="innerp collapse" id="immobilization"><H4>Key Achievements</H4><UL><LI>Developed a protocol for the immobilization of a laccase. </LI><LI>Proved that amine coupling with our laccase is possible on magnetic beads and that the immobilized enzyme remains active.</LI></UL><H4>Laccase Immobilization on Magnetite Nanospheres</H4><P>We tested the coated beads with laccase (Fe<SUB>3</SUB>O<SUB>4</SUB>-PSS-Chitosan-GA-Lac) in ABTS solution for activity at room temperature. For that, we separated the laccase solution containing unbound laccase from the beads with a magnet. We washed the beads three times with PBS and mixed them with 100µl 2mM ABTS and 800µl citric acid-phosphate buffer pH 4.0. Figure 1 shows the beads in the ABTS solution and the separated laccase solution.</P><P class="legend"><B>Figure 1.</B>  Left tube: magnetic beads with applied magnet in ABTS solution. Right tube: laccase solution in ABTS solution. The  laccase solution with not immobilized laccases on the right has the characteristic blue color of oxidized ABTS. The solution containing the beads does not show a blue but yellow color.</P><P>Due to technical difficulties during the drying step of the beads, it was not clear whether we successfully immobilized the enzyme. The ABTS solution turned yellow and not blue like the laccase solution with the unbound laccase indicating enzyme activity. Therefore, we could not clearly detect the immobilized laccase with the ABTS activity assay in this method. Due to time constraints, we, unfortunately, did not have the chance to repeat the experiment. 
Nevertheless, we still wanted to show that the immobilization of this laccase is possible with amine coupling. We, therefore, chose a fast protocol with streptavidin-coated Dynabeads® (superparamagnetic particles) to immobilize the laccase via biotinylation. Biotin also binds to primary amines of the enzyme provided by lysines, which would support that the laccase is feasible for the principle we followed by amine coupling with Glutaraldehyde.</P><H4>Laccase Immobilization on Streptavidin-coated Dynabeads® </H4><P>After biotinylation of the laccase we bound it to streptavidin-coated Dynabeads® and tested them in the ABTS solution. The magnetic particles with the biotinylated laccase showed the characteristic blue color in the ABTS solution. After the second washing step, there is no significant color change visible. We, therefore, can conclude that the laccase is successfully immobilised on the beads and assume that the laccase is feasible for this immobilization principle and remains active.</P><P class="legend"><B>Figure 2.</B> The positive controls show the expected color change. The second washing step clearly shows only a light color change in comparison to the washed beads with the biotinylated laccase.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>