<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stockholm_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stockholm/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Stockholm 2018 Wiki</TITLE></P><DIV id="main-content" class="container"><DIV class="row justify-content-center"><DIV class="col-md-12 col-lg-8"><DIV class="content"><H1><SPAN class="contentHeading"><SPAN class="bigFirst">B</SPAN>ioBrick Improvement</SPAN></H1><P class="lead">We decided to improve a BioBrick with the help of a recently developed mutagenesis technique called <A href="" data-toggle="tooltip" onclick="return false;" data-html="true" title="Solid-phase Assisted Mutagenesis by Uracil Restricted Ablation &lt;i&gt;In vitro&lt;/i&gt;">SAMURAI.</A>The method was developed by Hu et al. [1] at KTH Royal Institute of Technology (under the name SPUX, later renamed SAMURAI [2]) and is a novel method for easily and quickly introducing the site-specific mutations. SAMURAI allows the user to introduce multiple mutations in the same gene in only one experiment. We wanted to explore the potential of this technique in relation to our project, but also to investigate how this method can be used within the frames of iGEM. Can it be a valuable resource for future iGEM teams? To answer this question, we first of all need to explain the SAMURAI procedure. After that follows a section about how we applied this technique to improve a BioBrick.</P><A class="subheading blue" data-toggle="collapse" data-target="#what">What is SAMURAI?</A><DIV class="innerp collapse" id="what"><P>SAMURAI (Solid-phase Assisted Mutagenesis by Uracil Restricted Ablation <I>In vitro</I>) is an alternative site-directed mutagenesis method which utilizes uracil amplification, biotinylated primers and magnetic beads. SAMURAI allows the user to introduce multiple single-site mutations at once in the gene of interest, with high purity of the final product due to the total absence of wild-type DNA. [2]</P></DIV><A class="subheading purple" data-toggle="collapse" data-target="#how">How Does SAMURAI Work?</A><DIV class="innerp collapse" id="how"><OL><LI>Amplify the wild-type DNA with a <A href="" data-toggle="tooltip" onclick="return false;" data-html="true" title="Biotin is a vitamin also known as B7. It binds to streptavidin with one of the strongest non-covalent interactions found in nature.">biotinylated</A> forward primer and dAGCU (including uracil) instead of dAGCT (including thymine).</LI><LI>Bind the biotinylated top strand to <A href="" data-toggle="tooltip" onclick="return false;" data-html="true" title="Streptavidin is a protein with an extraordinarily high affinity for biotin.">streptavidin</A> coated magnetic beads.</LI><LI>Separate the uracil-containing top strand from the bottom strand and discard the bottom strand. This is done by separating the strands with NaOH, collecting the beads with a magnet and washing away the rest.</LI><LI>Introduce the mutations via PCR. The mutations are introduced using <A href="" data-toggle="tooltip" onclick="return false;" data-html="true" title="A phosphate group has been added to the 5' end of the DNA oligos to enable ligation by a ligase in the later on PCR produced bottom strand.">5’ phosphorylated</A> DNA oligos (containing the mutations) annealing to the top strand, regular dNTPs and Phusion U polymerase, a polymerase which is able to work in both directions. Ligase is also added in this PCR step, to ligate the nicks arising in the bottom strand at the end of each oligo in the PCR reaction.</LI><LI>Release everything from the beads by heat incubation.</LI><LI>Dissolve the uracil-containing top strand (wild-type DNA not containing the mutations). For this, a restriction enzyme called USER enzyme is added, which cuts specifically at every uracil it encounters.</LI><LI>Do regular PCR with an ordinary polymerase for amplification of the remaining mutated ssDNA. The final product will be a pure sample containing only mutated dsDNA.</LI></OL></DIV><A class="subheading cyan" data-toggle="collapse" data-target="#activity">Improvement of a BioBrick</A><DIV class="innerp collapse" id="activity"><P>Our original idea was to learn the SAMURAI procedure in order to apply it to our laccase, the main part of our project, in a later stage. We knew that we wanted to mutate our laccase based on computational rational enzyme design. While we were working on the modeling, we also started with the production of the native laccase. As the weeks passed we realized that we needed a quick way of introducing our mutations in the laccase gene, otherwise, we would not have time to express, purify and test it. In our search, we came across the recently developed SAMURAI method. This technique suited our needs perfectly since it would solve our problem of inserting multiple mutations with limited time on our hands.</P><P>We decided to try out the SAMURAI technique to learn how it works. To do this, we chose to introduce two distantly spaced mutations to an already existing BioBrick, BBa_I13502, designed by the MIT team 2005. This BioBrick consists of a monomeric fluorescent protein (mRF1) gene with a ribosome binding site (RBS). The part is not compatible with assembly standard RFC25 since it includes two AgeI restriction sites, the first at 573 bp and the second at 685 bp. </P><P class="legend"><B>Figure 1.</B> A map of the the BioBrick BBa_I13502, containing an RBS and an mRFP1 gene. Mutation 1 and 2 show the positions of the point mutations we chose to introduce to make the part compatible with assembly standard RFC25 by removing the two AgeI sites.</P><P>To remove the two AgeI sites with SAMURAI, we designed two types of forward primers (one with biotin and one without) and one reverse primer. We also designed two oligos where we introduced our point mutations to remove the AgeI sites: G → A at 576 bp and C → G at 687 bp (Figure 1). We chose these bases since they do not change the amino acid sequence, and therefore should not affect the function of the protein. As we performed the SAMURAI method, we encountered some problems, troubleshooted these and found solutions. Our main problem was the annealing temperature of the oligos introducing the mutations in PCR. After increasing this temperature, everything worked perfectly. We had now created a mutated version of the original BioBrick (BBa_I13502), compatible with all RFC assembly standards. We named this BioBrick 
<A class="link" href="http://parts.igem.org/Part:BBa_K2835000">BBa_K2835000</A>. This part was ligated into the plasmid pSB1C3 by making use of the restriction sites in the BioBrick prefix and suffix (EcoRI and PstI). Transformation into TOP10 <I>E.coli</I> followed, whereby colony PCR showed the expected size of the SAMURAI insert in two out of four colonies (Figure 2).
</P><P class="legend"><B>Figure 2.</B> Agarose gel showing the products from the colony PCR. Colony 1 and 2 have a SAMURAI insert of the same size as the original BioBrick BBa_I13502 (WT in the figure), which is expected after a successful SAMURAI procedure. Colony 3a and 3b do not contain a SAMURAI insert.</P><P>To express the SAMURAI product and prove that our mutations did not affect the protein function of mRFP1, we assembled our improved BioBrick with a strong constitutive promoter, BBa_J23119 (designed by team Berkeley 2006). The same assembly was performed with the original BioBrick (BBa_I13502) for experimental comparison. After several failed attempts to accomplish this with 3A assembly, we decided to try out a different approach. Instead of trying to assemble three different parts, as in 3A assembly, we simply PCR amplified our improved BioBrick and ligated this PCR product (digested with XbaI and PstI) into a pSB1C3 vector from the Distribution Kit already carrying the chosen promoter (BioBrick BBa_J23119) (digested with SpeI and PstI). The following transformation resulted in red colonies on the first try, with the red color indicating a successful expression and assembly. See our lab book for a more detailed workflow.
</P><P class="legend"><B>Figure 3.</B> The pSB1C3 plasmid backbone containing a constitutive promoter (BBa_J23119) and RBS + mRFP1 gene (BBa_K2835000), constructed by digestion and ligation.</P></DIV><A class="subheading blue" data-toggle="collapse" data-target="#proof">Proof of Improvement</A><DIV class="innerp collapse" id="proof"><P>Liquid cultures of the inoculated red colonies showed a bright red color after one-night incubation, as can be seen in Figure 4, where cultures with the original BioBrick (BBa_I13502) and our improved version (BBa_K2835000) are shown side by side. To prove that the introduced mutations did not affect the fluorescence of the protein, absorbance measurements were performed. A wavelength of 600 nm was used to check the cell density, while 584 nm (excitation peak of mRFP1 [3]) was used to measure the red fluorescent protein. The results showed close to identical absorbance for both versions of mRFP1, with values of OD<SUB>584</SUB>/OD<SUB>600</SUB> = 1.058 for the original BioBrick and OD<SUB>584</SUB>/OD<SUB>600</SUB> = 1.068 for our mutated version. These results, together with sequencing results confirming the introduced mutations, prove that our mutated BioBrick is an improved version of the original part. It has equal fluorescent properties and is compatible with assembly standard RFC25. RFC25 enables in-frame protein assembly, which could be beneficial when this BioBrick is used as a fluorescent reporter linked to another protein.</P><P class="legend"><B>Figure 4.</B> Liquid overnight cultures (TOP10 <I>E.coli</I>) with the original BioBrick encoding RBS + mRFP1, BBa_I13502, and our improved version of this BioBrick, BBa_K2835000. Both parts have been assembled with a strong constitutive promoter (BBa_J23119).</P></DIV><A class="subheading cyan" data-toggle="collapse" data-target="#future">SAMURAI and Future iGEM teams</A><DIV class="innerp collapse" id="future"><P>Now to the question: Can the SAMURAI technique be a valuable resource for future iGEM teams? We believe so! The strongest argument for this is <B>time</B>. As we have experienced, one of the most limiting factors in the iGEM competition is the short time frame of the project. SAMURAI is a quick site-directed mutagenesis technique, especially when you wish to introduce point mutations in different parts of a gene. Another argument is <B>specificity</B>. When performing site-directed mutagenesis with conventional methods there will most likely be some wild-type DNA not containing the mutations left in the mix, and lots of samples have to be sent for sequencing due to this. By elimination of the uracil-containing wild-type DNA not including the mutations, the chance of keeping any non-mutated products is significantly lower. The third argument is <B>low cost</B>. Not only will the method drastically reduce the number of samples having to be sent for sequencing, but all the reagents needed, except for the primers and oligos, are also practically a one-time investment. This results in economic benefits in the long run if the mutagenesis is to be repeated. We believe that this definitely could be of interest for future iGEM teams since expensive mutagenesis kits are not an alternative for many teams.</P><P>Furthermore, the SAMURAI procedure has only three PCR steps, where multiple mutations can be introduced at the same time. As an example, SAMURAI can be really useful for introducing mutations in the active site of an enzyme, since these amino acids are often dispersed across the protein sequence. Hence, this technique has the possibility to work well together with rational enzyme design, one of the main parts of our project. </P></DIV><H4>References</H4><OL><LI>Hu, F. (2017). Utilizing Solid Phase Cloning, Surface Display And Epitope Information for Antibody Generation and Characterization (TRITA-BIO-Report, 2017:10). KTH Royal Institute of Technology.</LI><LI>Lundqvist, M. (2018). Methods for cell line and protein engineering (TRITA-CBH-FOU ; 2018:14). KTH Royal Institute of Technology.</LI><LI>Vrzheshch EP, Dmitrienko D V, Rudanov GS, Zagidullin VE, Paschenko VZ, Razzhivin AP, et al. Optical Properties of the Monomeric Red Fluorescent Protein mRFP1. 2008;63(3):109–12.
</LI></OL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>