Team:Stockholm/Experiments
iGEM Stockholm 2018 Wiki
Experiments
Our research was essentially divided into four different areas. In this page, we describe in detail how we achieved our results in each individual step. Click on the name of each section to learn more.
Laccase Production
Aim
Producing and purifying the wild-type and mutant laccases from the fungus Trametes versicolor (T. versicolor)
Cloning of the Wild-Type Laccase Gene
To produce our wild-type laccase enzyme, we started with cloning the BioBrick BBa_K500002 into an expression vector for Pichia pastoris (P. pastoris). This was done by restriction cloning, using the restriction sites EcoRI and XbaI. First of all, we modified the BioBrick by overhang PCR to remove the native signal sequence, add a double stop codon and an N-terminal 6xHis-Tag. We chose to include the polyhistidine tag for purification of the enzyme later on. The restriction sites for cloning were also included at the end of the primer overhangs.
Following that, we cloned the BioBrick into the pPICZɑ A vector (Figure 1). In this vector, the BioBrick is fused to the ɑ-factor secretion signal from Saccharomyces cerevisiae and placed under control of the methanol-inducible AOX1 promoter. The plasmid was then transformed into TOP10 E.coli for amplification, using the antibiotic zeocin for selection.
Figure 1. Vector map of pPICZa A containing the wild-type laccase BioBrick (BBa_K2835003).
Cloning of the Mutant Laccase Gene
The sequence for the mutant laccase was based on our modified wild-type laccase BioBrick sequence, but mutations were introduced according to our modeling results . The amino acid residues changed were Phe-162, Phe-332 and Phe-337, of which the first was mutated leucine and the last two to isoleucine. We ordered the sequence from IDT and codon optimized it for P. pastoris. After amplification by PCR, we cloned the mutant laccase sequence into the pPICZɑ A vector by restriction cloning (Figure 2), using the same methods as for the wild-type sequence.
Figure 2. Vector map of pPICZa A containing the mutant laccase BioBrick (BBa_K2835004).
Transformation & Expression
After extracting and linearizing the plasmid, we transformed the yeast by electroporation. Successful transformants were selected using zeocin and further confirmed by colony PCR.
For the expression of the enzyme, we first grew the clones for 1 day in BMGY . The media was then changed to BMMY for another 5 days of cultivation, adding 1% methanol and 0.1 mM copper sulfate every 24 hours.
First, we screened several clones by taking samples every 24 hours and testing for enzymatic activity with an ABTS assay. Laccases oxidate ABTS, resulting in a change in color that can be measured as a change in absorbance at 420 nm. Thus, this assay can be used to screen for the presence of laccase in the culture supernatant. We also ran the samples on SDS-PAGE and Western blot.
After the screening, the clone producing the highest levels of enzyme was selected as well as the optimal harvesting time. Using this clone and harvesting time production of the enzyme was set up on a larger scale in BMGY and BMMY, in the same way as the screening. On the last production day, we harvested the culture supernatant (in which the enzyme is secreted) by centrifugation.
Purification
The enzyme was purified from the supernatant using immobilized metal ion affinity chromatography (IMAC) . Afterwards, we confirmed the purification by running the fractions on SDS-PAGE. For further purification, we also performed size exclusion chromatography (SEC) and MALDI-TOF mass spectrometry (with the help of our advisor Christiane Stiller).
The full production workflow for laccases in P. pastoris is shown in Figure 3.
Figure 3. Laccase production workflow.
Enzymatic Activity
Aim
Determine the kinetic parameters (KM and kcat) of our enzyme
Characterize the optimal conditions in which laccase from the fungus T. versicolor operates
Characterize  the potential of T. versicolor laccase to remove sulfamethoxazole (SMX)
Testing the Activity of Our Enzyme
To characterize the preferences of our copper-fueled enzymes and determine their kinetic parameters, we developed several assays. The degradation of our target substrate sulfamethoxazole by the laccase cannot be measured spectrophotometrically due to the slow reaction rate. For simplicity, we used the model substrate ABTS for activity assays, which is an established method to evaluate oxidoreductases [1]. It is a convenient method since the oxidized product of ABTS, here called ABTSox, displays a beautiful blue color and absorbs strongly at 420 nm.
What is the KM and kcat of the Enzyme?
In biochemistry, the standard way of comparing different enzymes is to look at the kinetic constants KM and kcat. KM is the dissociation constant, which characterizes the enzyme’s ability to bind the substrate. kcat is the turnover number, or the amount of substrate that the enzyme can turn into product per second.
The reaction between laccase and ABTS can be summarized as reaction 1 below. In reactions 2-4 we try to break down the reaction in smaller parts to make it easier to understand. If one would sum up reactions 2-4, reaction 1 would be obtained. First, four ABTS molecules are oxidized sequentially, this is the rate-determining step according to our model. This is shown in reaction 2 below.  The electrons from ABTS are stored by the copper ions in the enzyme, which is shown by reaction 3 (the enzyme works like a battery!). When all four copper ions are reduced, oxygen and hydrogen ions will pick up the four electrons by forming water, and the enzyme will return to its starting state, which is shown in reaction 4.
4 ABTS + O2 + 4 H+ → 4 ABTSox + 2 H2O    (1)
4 ABTS → 4 ABTSox + 4 e- (2)
4 Cu2+ + 4 e- → 4 Cu+ (3)
4 Cu+ +  O2 + 4 H+ → 4 Cu2+ + 2 H2O        (4)
We cannot control the saturation of the second substrate (dissolved oxygen), however, the first part of the reaction (oxidation of ABTS) is rate determining according to our model. We can assume that oxygen is in excess in the reaction, and therefore it should not affect the reaction speed or the kinetic parameters.
To calculate KM and kcat accurately, we also need to know the amount of enzyme that is added to the reaction. We attempted to determine the enzyme concentration with SEC and SDS-PAGE.
At Which Conditions Is Laccase from T. versicolor Most Active?
To establish the optimal conditions for our laccase, we performed several assays with a commercial laccase from T. versicolor (38429 Sigma),  which is similar to the enzymes we expressed.
1. Optimum pH
To investigate how the enzymatic activity depends on pH, we tested the activity of the enzyme in citric acid-phosphate buffers with pH 3, 4, 4.5, 5, 6, and 7 using ABTS as substrate.
2. Optimum Temperature
We investigated the enzyme’s activity in different temperatures; 5, 10, 20, 30, 40 and 50 ºC.
As we mentioned before, the availability of oxygen in the reaction vessel will affect the reaction rate. The solubility of oxygen in liquid increases with temperature, which means that the mass transfer rate in the buffer also increases in higher temperatures, which could also affect the reaction rate. This means that if the reaction rate is faster in higher temperatures compared to lower, it could both be due to the properties of dissolved oxygen in the solution, or due to the enzymatic activity increasing. However, since the reaction step involving oxygen is not rate determining, the availability of oxygen should not affect the reaction rate if the solution is saturated.
3. Enzyme Stability
To be sure that our enzyme is robust in the optimal conditions we found from the pH and temperature assays, we made a stability assessment by incubating the enzyme in pH and temperatures in the conditions that seem to be the most favorable for the enzymatic activity. If the enzyme is robust in these conditions, the activity should also be preserved.
Can We Deactivate our Enzyme if Needed?
Sometimes the analysis of experiments cannot be done at the desired time points due to schedule limitations or unavailability of the required analytical devices at that specific moment. Therefore, to carry out our experiments, we designed a method to quench the reactions (i.e. inactivate the laccase and thus stop its activity) when needed.  This provided us with both further knowledge of our enzyme’s characteristics, as well as a powerful tool to make the design of our experiments more flexible. For this purpose, we estimated whether pH changes can achieve this goal.
The ability to inactivate the fungal laccase was tested by incubating the laccase in pH 1 and pH 7 for one hour. After that incubation period, the pH was restored to the original value and activity was measured by ABTS assay at two time points: right after restoring the initial pH and 90 minutes afterwards. Different setups were also tested for further validation (e.g. different laccase quantities added or different ABTS concentrations).
How Well Can the Enzyme Remove Sulfamethoxazole (SMX)?
To estimate how much SMX our enzyme can degrade, we ran several reactions and negative controls in the ideal reaction conditions. The reactions contained SMX and laccase, and the negative control contained only SMX. The ideal conditions were determined by our previous assays, which investigated the optimum temperature and pH for the enzyme. The removal of SMX was analyzed by reverse phase HPLC (by our advisor Zhe Li), using a standard curve to quantify the amount of SMX in the different samples.
References
Childs RE, Bardsley WG (1975) The steady-state kinetics of peroxidase with 2,2′-azino-di-(3-ethyl-benzthiazoline-6-sulphonic acid) as chromogen. Biochem J 145: 93–103.
Ecotoxicity Testing
Aim
Testing the ecotoxicity of transformation products (TPs) of sulfamethoxazole (SMX) after enzymatic degradation with laccases from Trametes versicolor
Testing whether laccases reduce the ecotoxic potency of SMX
How Safe is Enzymatic Degradation of Pharmaceuticals?
Using laccases to degrade antibiotics has been proven to work successfully [1]. However, transformation products (TPs) generated from the enzymatic degradation might possess ecotoxic properties, posing a threat to the environment [2].
We wanted to make sure that the commercial, wild-type and mutated laccases do not produce ecotoxic TPs, and that our enzyme reduces the ecotoxicity of SMX. To do this, we aimed at testing the ecotoxicity of TPs generated from a reaction with laccase and SMX. We tested the ecotoxicity in two ways:
1. Investigating the effect of TPs on E.coli to see whether the antibiotic effect is reduced
2. Performing the Aliivibrio fischeri luminescence inhibition test, one of the most common and well-standardized bacterial bioassays [3]. A. fischeri is found in seawaters, and is bioluminescent by itself. Since SMX is toxic, luminescence inhibition is correlated with cell death. This can be used to establish a possible ecotoxic effect of TPs generated from the laccase and SMX reaction, measuring the potency of SMX by calculating the EC50 .
Before testing the ecotoxic effects of the products formed by our enzymatic degradation, we first set up transformation reactions. The set-up of these mixtures was designed after extensive literature research in the field [4], as well as by using the optimization results obtained from the activity assays that we performed previously .
1. Testing the Effect of Transformation Products on E. coli
To test the effect of TPs on bacteria, we developed a protocol using E. coli as a model. E. coli is easy to cultivate and serve well as a model bacterium to show proof of concept. The protocol is based on a method adapted from Becker et. al. (2016), but was changed and improved to fit our specific needs [1]. In brief, E. coli is cultivated and used in a growth inhibition assay set-up with the different transformation reactions (performed in triplicates). The plates are read in a spectrophotometer, comparing the effect of TPs and pure SMX on E. coli growth. The full protocol of this assay can be found here .
Figure 1. Growth inhibition testing work flow with E. coli treated with TPs and pure SMX
2. Assessing the Ecotoxic Potency of Transformation Products and SMX
We want to make sure that enzymatic degradation with our laccases does not generate more ecotoxic TPs than pure antibiotics alone. Therefore, we searched for a method to assess their ecotoxic potency. Looking at what other researchers had done, we concluded that using the bioluminescent assay with A. fischeri is the best method, as it is one of the most standardized bioassays for this purpose [2]. A. fischeri was introduced to TPs from a reaction with wild-type laccase and two different concentrations of SMX, to determine the EC50 based on luminescence differences. The kit used was the BioTox™ WaterTox™ EVO [5].
Figure 2. Bioluminescence inhibition testing work flow with A. fischeri treated with TPs and pure SMX, and the estimation of the EC50 value through linear regression.
References
Yang, J., Li, W., Ng, T. B., Deng, X., Lin, J., & Ye, X. (2017). Laccases: Production, Expression Regulation, and Applications in Pharmaceutical Biodegradation. Frontiers in Microbiology, 8, 832.
Becker, D., Varela Della Giustina, S., Rodriguez-Mozaz, S., Schoevaart, R., Barceló, D., de Cazes, M., … Wagner, M. (2016). Removal of antibiotics in wastewater by enzymatic treatment with fungal laccase – Degradation of compounds does not always eliminate toxicity. Bioresource Technology, 219, 500–509.
Microtox® Acute Toxicity Solid-Phase Test. Azur Environmental; 1995. 20 pp.
Margot, J., Bennati-Granier, C., Maillard, J., Blánquez, P., Barry, D. A., & Holliger, C. (2013). Bacterial versus fungal laccase: potential for micropollutant degradation. AMB Express, 3(1), 63.
EBPI. (2018). Aliivibrio fischeri Toxicity Tests (formerly Vibrio Fischeri ). Retrieved September 15, 2018, from http://www.ebpi-kits.com/ebpi-line-of-biotoxicity-tests/vibrio-fischeri-toxicity- tests.html
Laccase Immobilization
Aim
Immobilizing the laccase on a magnetic bead by glutaraldehyde-mediated amine coupling and make it reusable and applicable for our desired implementation
Immobilising the laccase on streptavidin-coated magnetic beads by biotinylation to demonstrate the feasibility of our enzyme to be immobilized
Principle of Amine Coupling
As the last step in our experimental work, we wanted to immobilize our enzyme. For this, we chose to use glutaraldehyde, a frequently used reagent for amine coupling. In our approach, we used magnetic beads and coated them with chitosan to provide a surface with primary amines. Glutaraldehyde binds to these amines covalently and can be coupled via the aldehyde group (pointing outwards) to other primary amines provided by lysines on the outer part of the protein.  As a proof of concept, we also used biotinylation for amine coupling, to show the feasibility of the laccase for amine coupling. Biotin binds to primary amines of the laccase. The biotinylated laccase can then be bound to streptavidin, which is coated on the surface of magnetic beads making it feasible to be easily removed from solutions [3].
Immobilization of Laccase with Glutaraldehyde
To immobilize the laccase on magnetic beads we used nanospheres made out of magnetite  (Fe3O4), which had a diameter of ~200 nm. We applied a protocol which was used for immobilization of enzymes on this material before for a similar purpose [1]. We coated the beads first with Poly(sodium 4-styrenesulfonate) (PSS). Subsequently, Fe3O4-PSS was coated with chitosan and Fe3O4-PSS-Chitosan was collected with a magnet. We linked glutaraldehyde to Fe3O4-PSS-Chitosan and performed the immobilization with a commercial laccase from T. versicolor to demonstrate the potential of our produced laccase to be immobilized [2]. We prepared a solution with the commercial laccase and mixed it with Fe3O4-PSS-Chitosan-GA. Afterwards, we tested them for immobilized laccase using the ABTS activity assay.
Figure 1. The immobilization process of enzymes on the magnetic bead.
Immobilization of Laccase on Streptavidin-coated Magnetic Beads
We mixed the laccase with biotin to achieve biotinylation of the protein. The laccase-biotin solution was then mixed with Dynabeads® (superparamagnetic particles) and placed on a magnetic rack for separation [4]. To ensure the successful immobilization and activity of the enzyme, the beads were tested with the ABTS activity assay.
References
Yeon K-M, Lee C-H, Kim J. Magnetic Enzyme Carrier for Effective Biofouling Control in the Membrane Bioreactor Based on Enzymatic Quorum Quenching. Environmental Science & Technology. 2009;43(19):7403–9.
Yufang Zhu, Stefan Kaskel, Jianlin Shi, Tobias Wage, and Karl-Heinz van Pée Immobilization of Trametes versicolor Laccase on Magnetically Separable Mesoporous Silica Spheres Chemistry of Materials 2007 19 (26), 6408-6413
Chivers CE, Koner AL, Lowe ED, Howarth M. How the biotin-streptavidin interaction was made even stronger: investigation via crystallography and a chimaeric tetramer. Biochemical Journal. 2011;435(Pt 1):55-63. doi:10.1042/BJ20101593.
Thermo Fisher Scientific, available at: thermofisher.com/order/catalog/product/B30010, accessed 2018-10-16
