Team:Stockholm/Project Design
iGEM Stockholm 2018 Wiki
Project Design
We created our enzymatic solution - Biotic Blue - to inactivate the persistent antibiotic sulfamethoxazole (SMX). The basis for Biotic Blue is formed by laccases, which have a high oxidizing power for phenolic compounds such as SMX. However, their applicability is currently limited because of the long time required for the reaction. Therefore, we aimed to engineer this enzyme to degrade SMX with higher affinity in a shorter amount of time.
Figure 1. The original wild-type laccase (PDB: 1GYC).
Designing Our Enzyme
Our enzyme is based on the laccase-2 enzyme from the fungus Trametes versicolor. We chose this laccase for its high redox potential [1], the available crystal structure (PDB: 1GYC) and its availability as a BioBrick (BBa_K500002, designed by iGEM Tianjin 2010). The 1GYC laccase thus acted as an excellent starting point to tackle SMX degradation. While the enzyme has evolved to degrade a wide range of phenolic compounds, our hope is that small, specific changes in the amino acid sequence can drastically improve its specificity and ability to degrade SMX.
A unique aspect of our project is the use of rational enzyme design: by analyzing and understanding how our enzyme works in silico, we can introduce carefully selected mutations in the enzyme's active site. This approach holds several advantages over more conventional methods. Directed evolution is a well-established protein engineering technique, but it is a slow procedure as many clones have to be screened. This is complicated by the fact that laccases require a complex expression system. Moreover, it is likely that sulfamethoxazole would interfere with the process due to its antibiotic effect on the host organism.
Our project thus started with computationally selecting a few promising enzyme variants, followed by expression and testing of the enzymes. After testing the enzyme, we used the experimental data obtained to improve our model and make better suggestions for mutations, forming a continuous cycle (Figure 2).
Throughout the course of our project we also integrated our modeling and experimental results in several other ways. For example, from our modeling we concluded which protein terminus was most suitable for adding polyhistidine tag without disturbing the enzymatic activity. Also, quantum mechanical modeling of the enzyme’s mechanism influenced the experimental design of our activity assays.
Figure 2. The application of the synthetic biology 'Design-Build-Test' cycle in our project.
Building Our Production Platform
In order to express the wild-type enzyme, as well as the enzyme variants that we designed in silico, we required a eukaryotic expression system. Since the T. versicolor laccase is a eukaryotic enzyme, it is post-translationally modified in the native host - containing two disulfide bonds and N-glycosylations at three sites. These modifications are important for proper folding and activity of the enzyme [2]. In prokaryotic expression systems, such as Escherichia coli, such modifications would not be introduced into the protein.
Instead we decided to use yeast, a widely used eukaryotic expression host with many genetic engineering tools. We chose the methylotrophic yeast Pichia pastoris over the more commonly used Saccharomyces cerevisiae (Baker’s yeast) for several reasons. First of all, a higher protein yield has been reported for laccases in P. pastoris than in S. cerevisiae [3]. Furthermore, it secretes few endogenous proteins, simplifying purification of our recombinant protein. Lastly, hyperglycosylation can be a problem in both yeast species and could impair enzyme activity. However, in P. pastoris N-linked oligosaccharides are typically composed of 20 residues compared to 50 to 150 residues in S. cerevisiae [4].
For expression of our enzyme in P. pastoris, we used the pPICZα A vector from Invitrogen. This vector includes the α-factor secretion signal from S. cerevisiae under control of the methanol-inducible AOX1 promoter. The vector directly integrates into the P. pastoris genome at the native AOX1 locus. Upon addition of methanol to the culture medium, the promoter will be activated and the yeast will start producing the enzyme.
For purification of our enzymes we used immobilized metal affinity chromatography (IMAC), a method relying on the affinity of polyhistidine tags to metal ions chelated to the column resin. A 6xHis tag was included at the N-terminal end of all enzymes, to make it easier to purify them with the same method.
Figure 3. A schematic workflow of our production and testing platform.
Exploring Site-Directed Mutagenesis
To facilitate the engineering of our enzyme, we also investigated a rapid, site-directed mutagenesis (SDM) technique called SAMURAI [5]. This novel method can be used to easily introduce the various site-specific mutations suggested by modeling results. It allows the user to introduce multiple or single mutations in the same gene in only one experiment. In projects such as ours, where one wants many variants of the same gene with multiple modifications, this is a very useful tool. Read more about it here .
Putting Our Enzyme Under Scrutiny
Finally, to test our enzyme we developed several assays that can be used to characterize the wild-type and mutant enzymes. We performed kinetic testing using the model substrate ABTS, to measure the enzyme’s activity and determine its Michaelis-Menten parameters. To estimate the ability of our enzymes to remove SMX, we used reverse phase HPLC to analyze SMX-containing samples that were incubated with laccase for different time periods. In addition, as it has been reported that the degradation of antibiotics can actually result in toxic products itself [6], we decided to develop two assays testing the ecotoxicity of the reaction products: a growth inhibition assay in E. coli and the standardized bioluminescence assay with A. fischeri.
Bringing Our Enzyme Out of the Lab
To make our enzyme ready for implementation, we experimented with several ways to immobilize it. First, we considered using the EnginZyme immobilization system, which relies on the affinity of polyhistidine tags to the EziG resin. The advantage of this method is the low mass transfer limitation, resulting in a higher retained enzyme activity after immobilization. However, this immobilization is not stable enough to withstand the chemically and mechanically harsh conditions in wastewater treatment plants.
The demands on our carrier to be used multiple times before it is reloaded with enzyme, led us to an alternative method that was previously used for this purpose [7]. We chose to immobilize our enzyme on magnetic beads, using an amine-to-amine cross-linking approach with gluteraldehyde as a cross-linking agent. As a proof of concept, we also used biotinylation to immobilized laccase on streptavidin-coated magnetic beads.
References
Jones SM, Solomon EI. Electron Transfer and Reaction Mechanism of Laccases. Cellular and molecular life sciences: CMLS. 2015;72(5):869-883.
Brijwani K, Rigdon A, Vadlani, PV. Fungal Laccases: Production, Function, and Applications in Food Processing. Enzyme Research. 149748. Published online August 2010.
Gomes AR, Byregowda SM, Veeregowda BM, Balamurugan V. An overview of heterologous expression host systems for the production of recombinant proteins. Adv. Anim. Vet. Sci. 2016 June;4(7):346-356.
Piscitelli A, Pezzella C, Giardina P, Faraco V, Sannia G. Heterologous laccase production and its role in industrial applications. Bioengineered Bugs. 2010;1(4):252-262.
Darby RAJ, Cartwright SP, Dilworth MV, Bill RM Which Yeast Species Shall I Choose? Saccharomyces cerevisiae Versus Pichia pastoris (Review). In: Bill R., editor. Recombinant Protein Production in Yeast. Methods in Molecular Biology (Methods and Protocols), vol 866. Humana Press; 2012.
Hu, F. Utilizing Solid Phase Cloning, Surface Display And Epitope Information for Antibody Generation and Characterization. TRITA-BIO-Report. 2017:10. KTH Royal Institute of Technology.
Becker D, Varela Della Giustina S, Rodriguez-Mozaz S, et al. Removal of antibiotics in wastewater by enzymatic treatment with fungal laccase – Degradation of compounds does not always eliminate toxicity. Bioresource Technology. 2016;219:500–509.
Yeon KM, Lee CH & Kim, J. Magnetic enzyme carrier for effective biofouling control in the membrane bioreactor based on enzymatic quorum quenching. Environmental Science and Technology, 2016;43(19):7403–7409.
