"
Page
Discussion
View source
History
teams
Log in
 
Team:Northwestern/Project/Results
From 2012.igem.org
Team
Official
Profiles
Attributions
Project
At a Glance
Background
Results
Parts
Modelling
Notebook
Protocols
Weekly Reports
Outreach
High School
Collaboration
Safety
Quick Section Shortcuts:
Phytase
Design
Construct
Assay
pH Sensor
Design
Construct
Assay
Phytase
Design
At the core of Northwestern’s Phytastic Probiotic project are two main subprojects.
The first is to clone different phytases from different organisms into E. coli and then to engineer E. coli to constitutively express the phytases at varying levels. We used a lab strain of E. coli as a model chassis for this project, but our technology could be adopted to function in probiotic strains as well. The four phytases we chose were phytases from E. coli, citrobacter braaki, aspergillus niger, and bacillus subtilus. These phytases were selected based upon varying specific activities, pH optima, and temperature optima that are expected to be most compatible with physiological conditions of the stomach, as compared to other phytases. Our general methods comprised PCR from genomic DNA, gibson assembly,  and standard RFC10 biobrick assembly methods.
Construct
Each of the four phytases was PCR-amplified out of its respective organism’s genome.  However, due to the introns present in the the Aspergillus niger phytase coding gene, we had an intron-free version of this gene synthesized by IDT. We assembled these parts together using Gibson assembly. After PCR cleanup, gel electrophoresis was performed to verify that the expected band patterns were observed.
After gel verifying and sequencing each of the phytases, an unexpected restriction site was discovered in the middle of the Bacillus subtilus phytase gene that prohibited biobrick assembly. Thus, we decided not to pursue this phytase construct further at this time. The three remaining phytases were cloned into protein expression backbones from the parts registry with constitutive promoters and ribosome binding sites already cloned into the vector. Several promoters from the same promoter library were used as each has varying relative expression strengths. Gel electrophoresis and sequencing was again performed to verify that the parts were assembled as expected.
Assay
To obtain a relative activity of the phytase produced by the cells with the constitutive promoter/citrobacter phytase part, we performed a phosphate assay on the produced phytase. In order to remove the phytase from the cells, the cells were overnighted and diluted 1:500 and then regrown into the end of log phase. Once at the end of log phase, the cells were centrifuged and sonicated in order to obtain the intracellular protein(with the produced phytase)to use in a phosphate assay. To perform the phosphate assay, .1 nmol phytic acid was added to the sonicated cell lysate at both a pH of 7 and a pH of 4.5. The pH of 4.5 was the literature value of pH that citrobacter phytase works best at. A colormetric phosphate assay solution was added and the OD of the solution was recorded in comparison to phosphate standards in order to obtain a measure of free phosphate in the solution.
From the above phosphate assay results, we concluded that the produced Citrobacter phytase grown in E. coli managed to liberate phosphate from the phytic acid solution. We are continuing work on performing more phosphate assays to obtain results with more statistical rigor.
As the regional jamboree approaches, we are continuing to obtain results for the phosphate assays performed on both the Aspergillus niger and E. coli phytase in order to better characterize our new phytase parts for the registry. Specifically , it would be beneficial to know which phytase is the best at liberating phosphate and iron from phytic acid. As each phytase has a different active pH range, optimum temperature, and specific activity we expect that each phytase will have drastically different results in the physiological conditions of the stomach. Our best guess of which phytase will perform the best in the conditions of the stomach is the Citrobacter braaki phytase, due to its high specific activity[1] and low pH optimum.  [1] http://www.ncbi.nlm.nih.gov/pubmed/17302159
pH Sensor
Design
The second is a system that will inducibly release the phytase into the cell’s surroundings when the cells reach the stomach of the host.  The system is composed of an inducible promoter (Pgad) that is activated at high intracellular chloride ion concentration upstream of a lysis device. In order to convert Pgad from a chloride-sensitive promoter to an effective pH-sensitive promoter, a chloride-hydrogen antiporter known as CLC-ec1 is constitutively produced. The CLC-ec1 antiporter is activated at low intracellular pH and pumps 2 chloride ions into the cell in order to move one hydrogen ion out of the cell. Thus the CLC-ec1 antiporter will be activated at low intracellular pH which will cause the antiporter to pump chloride ions into the cell and hydrogen ions out of the cell. The increasing intracellular chloride ion concentration will activate the inducible Pgad promoter which in turn will activate the lysis device.
Construct
The clc-ec1 antiporter gene was PCR amplified from K12 E. coli’s genome obtained from Dr. Tyo’s Lab at Northwestern University. We also collaborated with the CUHK 2011 team in order to improve the characterization and submit their Pgad part to the registry as it was not submitted last year.  We obtained the Pgad part from the CUHK team in an unknown vector. Because the vector was unknown, we PCR amplified the Pgad gene. After PCR cleanup, gel electrophoresis was performed to verify that the expected band pattern was observed.
After gel verifying the clc-antiporter and the Pgad promoter, we then set out to construct the pH inducible promoter for use in our phytase delivery system. To do this, we transformed and miniprepped a lysis cassette, the protein expression plasmids described before, and a double terminator from the registry. We assembled and cloned together each part piece by piece, gel verifying after each step in the process to avoid errors. The final construct created was Pgad/lysis/(4 different CP+RBS)/CLC-ec1/Double terminator. Another construct replacing lysis with GFP was used to measure the activity of the Pgad promoter in an assay described in our system characterization section. Each of these constructs was again gel verified and sequenced.
Assay
To obtain a relative measure of the Pgad activity at varying pH’s of our pH sensor, cells containing the assembled Pgad/GFP nsCP/clc-ec1/xx in pSB1A3 plasmid were grown overnight, diluted 1:500 and then regrown to log phase (all in LB). Once in log phase, the cells were transferred into a 96-well plate with solutions of LB pre-adjusted to various pH levels. Every 5 minutes, the OD and GFP-fluorescence of each well were recorded.
Each data point was normalized for LB autofluorescence and for OD (OD shifts were observed based upon the amount of HCl in solution).  These abnormalities are seen in the unusually high fluoresence of all wells due to LB autofluoresence and abnormally high OD of the pH 2 wells due to HCl in solution. The results were then normalized again by calculating the fluorescence per OD of cells (pictured below). A t-test was then performed on the two populations of fluoresence/OD numbers, with the null hypothesis stating that the two populations did not have different means and the alternative hypothesis stating that the two populations did have different means. When the t-test was performed, the p-value was 4.9E-009, suggesting that the null hypothesis should be rejected and that there is a statistically significant difference between the mean Fluoresence/OD of the Pgad/GFP + nsCP/clc/xx at pH7 and at pH2. This means that the clc antiporter was activated at low pH conditions, causing chloride ions to flow into the cell, which in turn activated the Pgad promoter, producing the fluorescence found in the assay.
After quantifying the amount of Pgad expression through fluorescence, the next logical assay to perform was a test lysis assay to see if the cells would lyse when exposed to the physiological conditions of the stomach(low pH, high Cl). Cells containing the assembled Pgad/lysis nsCP/clc-ec1/xx in pSB1A3 plasmid were overnighted and then regrown in LB to log phase. Once at log phase, the cells were placed into a 96 well plate with varying pH solutions of LB. Every 15 minutes the OD and of the cells was recorded.
Simply put, cells with the Pgad/lysis nsCP/clc/xx part at a pH of 7 grew exponentially in solution as according to their OD’s over time. The cells at a pH of 4 and 3 however did not grow exponentially in solution. Their OD’s stayed relatively constant over time. The cells at a pH of 2 however did lyse, and their OD’s dropped over time.
Retrieved from " http://2012.igem.org/Team:Northwestern/Project/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
