<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Project_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Project/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Project/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Project/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Project/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Project/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Edinburgh/Notebook&amp;redirect=no" title="Team:Edinburgh/Notebook">Team:Edinburgh/Notebook</A>)</DIV><DIV class="menu"><TABLE style="text-align: center;"><TBODY><TR><TD align="right"><A href="/File:Edinburgh_uni.png" title="File:Edinburgh uni.png">class=&quot;nomargin&quot;</A></TD></TR><TR><TD><A href="/Team:Edinburgh" title="Team:Edinburgh" class="mw-redirect">Home</A></TD><TD><A href="/Team:Edinburgh/Team" title="Team:Edinburgh/Team" class="mw-redirect">Team</A></TD><TD><A href="/Team:Edinburgh/Project" title="Team:Edinburgh/Project">Lab work</A></TD><TD><A href="/Team:Edinburgh/Modeling" title="Team:Edinburgh/Modeling">Modeling</A></TD><TD><A href="/Team:Edinburgh/Human_Aspects" title="Team:Edinburgh/Human Aspects">Human practices</A></TD><TD><A href="/Team:Edinburgh/GenBrick" title="Team:Edinburgh/GenBrick">GenBrick</A></TD></TR></TBODY></TABLE></DIV><DIV class="main_frame"><DIV id="side_menu"><UL id="side_menu_items"><LI class="side_menu_item"><A href="/Team:Edinburgh/Project/Results" title="Team:Edinburgh/Project/Results">Results</A></LI><LI class="side_menu_subitem"><A href="/Team:Edinburgh/Project/Results/Chassis" title="Team:Edinburgh/Project/Results/Chassis">Chassis and vector characterisation</A></LI><LI class="side_menu_subitem"><A href="/Team:Edinburgh/Project/Results/Metal_promoters_Results" title="Team:Edinburgh/Project/Results/Metal promoters Results">Metal promoters</A></LI><LI class="side_menu_subitem"><A href="/Team:Edinburgh/Project/Results/Metal_binding_Results" title="Team:Edinburgh/Project/Results/Metal binding Results">Metal binding</A></LI><LI class="side_menu_subitem"><A href="/Team:Edinburgh/Project/Results/Bioethanol_Results" title="Team:Edinburgh/Project/Results/Bioethanol Results">Bioethanol</A></LI><LI class="side_menu_subitem"><A href="/Team:Edinburgh/Project/Results/Aggregation_Results" title="Team:Edinburgh/Project/Results/Aggregation Results">Aggregation</A></LI></UL><LI class="side_menu_item"><STRONG class="selflink">Notebook</STRONG></LI><LI class="side_menu_item"><A href="/Team:Edinburgh/Project/Parts" title="Team:Edinburgh/Project/Parts">Parts submitted</A></LI><LI class="side_menu_item"><A href="/Team:Edinburgh/Project/Conclusions" title="Team:Edinburgh/Project/Conclusions">Conclusions and future work</A></LI><LI class="side_menu_item"><A href="/Team:Edinburgh/Project/References" title="Team:Edinburgh/Project/References">Reference list</A></LI><LI class="side_menu_item"><A href="/Team:Edinburgh/Project/Appendix" title="Team:Edinburgh/Project/Appendix">Appendix</A></LI><LI class="side_menu_subitem"><A href="/Team:Edinburgh/Project/Appendix#Protocols" title="Team:Edinburgh/Project/Appendix">Protocols</A></LI></DIV><DIV class="content"><H2><SPAN class="mw-headline" id="Chronological_notebook"><B>Chronological notebook</B></SPAN></H2><DIV id="27-June" class="notebookEntry"><UL><LI><B>27th of June</B>: 
</LI></UL><P><I>B. subtilis</I> (strain: 168) culture inoculated into 10 ml LB medium. Overnight incubation in 37 deg.C. 
</P><P>Preparation of metal ion stock solutions: Ni - 0.1M; Zn - 0.1M; Fe - 0.5M; Cu - 0.5M; Co - 0.1M; Mn - 1M (Aleksandra, Harry and Jan)
</P></DIV><DIV id="28-June" class="notebookEntry"><UL><LI><B>28th of June</B>: 
</LI></UL><P>0.5 ml units of O/N culture diluted to 50ml (x100 dilution in LB medium). 3 ml of diluted culture aliquots poured into universal bottles (enables adequate aeration of the culture). Those were inoculated with 30 ul of differently concentrated metal ion solutions giving final concentrations of: 
</P><P><B>Ni</B> - 1mM; 0.75 mM; 0.5 mM and 0.25mM;
<B>Zn</B> - 1mM; 0.75 mM; 0.5 mM and 0.25mM;
<B>Fe</B> - 5mM; 3.75mM; 2.5mM and 1.25mM;
<B>Cu</B> - 5mM; 3.75mM; 2.5mM and 1.25mM;
<B>Co</B> - 1mM; 0.75 mM; 0.5 mM and 0.25mM;
<B>Mn</B> - 10mM; 7.5mM; 5mM; 2.5mM; 1.25mM and 0.625mM
</P><P>0.4 ml of O/N culture diluted to 40ml (x100 dilution in 2xLB medium). 3 ml of diluted culture aliquots poured into universal bottles (enables adequate aeration of the culture). Those were inoculated with 1ml of differently concentrated ethanol solutions giving final concentrations of: 
2%; 4%; 6%; 8%; 10%; 15%; 20% and 25%.
</P><P>The initial Absorbance readings at OD600 for diluted cultures were taken. Following 7h. incubation in 37 deg.C absorbance of each sample was measured and compared to samples inoculated only with H2O (Kyle, Harry, Aleksandra, Dainius and Jan). 
</P></DIV><DIV id="1-July" class="notebookEntry"><UL><LI><B>1st of July</B>: 
</LI></UL><P><I>B. subtilis</I> EtOH resistance assay was continued in a narrow concentration range (Jan). 
</P><P><I>E. coli</I> (DH5alpha) cells were transformed using BioBricks required for EtOH production (Aleksandra). 
</P><P>O/N cultures of <I>E. coli</I> and <I>B. subtilis</I> were set up to continue with the experiments tomorrow (Harry).  
</P></DIV><DIV id="2-July" class="notebookEntry"><UL><LI><B>2nd of July</B>: 
</LI></UL><P>Transformation was successful. Four colonies were selected and propagated into 5ml LB bottles supplemented with adequate antibiotic (Jan). After 8h incubation in 37 deg. C cells were pelleted and plasmids were extracted using Qiagen MiniPrep Kit (Aleksandra). Concentration  and quality of DNA was measured using NanoDrop. 
</P><P><I>E. coli</I> cells plated for competitors were harvested and propagated into 50 ml LB medium. Following 4h incubation they were used to make competent cells for future transformations (Aleksandra and Jan). 
</P><P>B. subtilis resistance to heavy metals experiment was continued in a narrow metal ion concentration range (Harry).
</P></DIV><DIV id="3-July" class="notebookEntry"><UL><LI><B>3rd of July</B>: 
</LI></UL><P>Since we propose using B. subtilis as a chassis for our waste water treatment bacteria, we first set about determining the concentration of different metal ions that the non genetically altered chassis can cope with. We initially observed cell density of cultures after 8 hours incubation with varying metal concentrations. Following this we recorded culture density every hour to obtain growth data for appropriate concentrations of metals, as indicated by the initial 8 hour cultures. These experiments continue... (Harry, Danius, Hugo)
</P></DIV><DIV id="4-July" class="notebookEntry"><UL><LI><B>4th of July</B>: 
</LI></UL><P>The competent <I>E. coli</I> cells were tested by transforming them with parts for ethanol production. The negative control was performed to confirm that cells have no indigenous antibiotic resistance and that prepared antibiotic plates have antibacterial properties (Aleksandra and Jan). 
</P><P>Time-lapse measurements of <I>B. subtilis</I> growth in specified metal concentrations (based on previous data) were performed (Harry, Hugo, Dainius and Kyle)
</P><P>Team started preparing poster (Kyle) and presentation (Dainius and Hugo) for the Young Synthetic Biologist meeting held in London over the next weekend: 12-14/07/2013
</P></DIV><DIV id="5-July" class="notebookEntry"><UL><LI><B>5th of July</B>: 
</LI></UL><P>Outcome of the transformation was positive with the negative control showing no growth (as expected). Four colonies were collected and grown in 5ml LB supplemented with adequate antibiotic. Following 7h incubation the entire culture was pelleted and plasmid was purified using Qiagen MiniPrep Kit. NanoDrop was used to measure the DNA concentration (Harry and Jan)
</P><P>Time-lapse measurements of <I>B. subtilis</I> growth in specified metal and ethanol concentrations (based on previous data)were performed (Harry, Hugo, Dainius and Jan).
</P></DIV><DIV id="8-July" class="notebookEntry"><UL><LI><B>8th of July</B>: 
</LI></UL><P>Two PCR reactions were set up to clone out genes of interest from <I>B. subtilis</I> (Harry and Hugo; Dainius and Kyle).
</P><P>The ethanol production part was combined with a promoter - overnight ligation(Jan)
</P></DIV><DIV id="9-July" class="notebookEntry"><UL><LI><B>9th of July</B>:
</LI></UL><P>The PCR product was run on the gel to confirm the presence of the desired product (Aleksandra, Dainius, Jan)
</P><P>The ligation product was transformed into the competent <I>E. coli</I> cells. Those were plated on the KAN+IPTG+X-gal plates (Aleksandra, Jan).
</P><P><I>B. subtilis</I> genomic DNA was purified from the O/N culture for the use in the future PCR reaction (Harry)
</P></DIV><DIV id="10-July" class="notebookEntry"><UL><LI><B>10th of July</B>:
</LI></UL><P>DNA gel electrophoresis of the PCR product was performed once more due to inconclusive results from 9/07/2013 (Jan). Bands of expected size were observed. Additional PCR was set up to obtain more DNA (Harry and Hugo). Following gel electrophoresis resulting products were restricted using correct enzyme and an O/N ligation reaction was set-up (Aleksandra and Jan)
</P><P>The transformation using the EdiBrick and ethanol production parts ligation product was successful - numerous blue colonies were obtained. Two colonies were selected and propagated into 5ml LB +KAN medium. Two more colonies (4 in total) were propagated onto LB+KAN+X-gal plates and incubated O/N in 37 deg.C (Aleksandra, Jan)
</P></DIV><DIV id="11-July" class="notebookEntry"><UL><LI><B>11th of July</B>:
</LI></UL><P>The ligation products of vector and two transcription factors amplified using PCR were transformed into E. coli and plated on LB+AMP+X-gal+IPTG plates (Harry and Weike)
</P></DIV><DIV id="15-July" class="notebookEntry"><UL><LI><B>15th of July</B>:
</LI></UL><P>The transformation done using the ligation product gave negative results. The restriction of SinR; Fur and AmpR vector was repeated. O/N ligation was set up using new restriction products (Harry, Weike and Jan)
</P><P>Cells from four different plates of the EdiBrick_pET ligation were resuspended in 5 ml LB medium + KAN and grown in 37 deg.C. The plasmid was purified following 4 hours of incubation using Qiagen MiniPrep Kit (Aleksandra and Jan)
</P></DIV><DIV id="16-July" class="notebookEntry"><UL><LI><B>16th of July</B>:
</LI></UL><P>The DNA resulting from ligation was used for transformation of competent <I>E. coli</I>. Cells were plated on LB+AMP+IPTG (20 ul) plates (Weike and Hugo)
</P><P>The EdiBrick_pET DNA was restricted using XbaI and PstI in CutSmart buffer. 2 h. incubation in 37 deg. C was used for restriction reaction.The resulting proucts were run on the gel (15ul product + 3ul loading dye). Three out of four DNA samples showed correct band sizes (Aleksandra).
</P><P>Reagents required for ethanol assay were prepared and a sample data for 0; 0.5 mM and 1 mM were obtained in order to plot a standard curve. O/N cultures of <I>E. coli</I> with pET and EdiBrick_pET were set up in order to analyse those tomorrow (Aleksandra and Jan).
</P><P>Timelapse analysis of <I>B. subtilis</I> growth in varying concentrations of three different heavy metals was performed (Harry and Dainius)
</P><P>2xLB was prepared to perform the time-lapse analysis of <I>B. subtilis</I> growth in higher EtOh concentrations (Jan).
</P></DIV><DIV id="17-July" class="notebookEntry"><UL><LI><B>17th of July</B>:
</LI></UL><P>Due to negative results of ligation all DNA involved in the process was analysed using DNA gel electrophoresis to identify the faulty step (Kyle, Dainius and Hugo)
</P><P>Supernatant of the pET and EdiBrick_pET O/N cultures was subjected to the ethanol assay. Due to encouraging results the pET plasmid was isolated and restricted using XbaI and PstI to confirm it definitely has the correct insert (Aleksandra and Jan)
</P></DIV><DIV id="18-July" class="notebookEntry"><UL><LI><B>18th of July</B>:
</LI></UL><P><I>B. subtilis</I> growth in higher [EtOH] was analysed (Aleksandra and Jan)
</P><P>PCR reaction was set up to obtain the DNA fragments required to perform deletions in the <I>B. subtilis</I> sporulation pathway (Dainius and Kyle)
</P><P>PCR reaction for the MABEL based fusion of adhB and pdc was performed. Product was analysed on a gel. An O/N blunt-end ligation reaction was set up to obtain the plasmid with fusion proteins(Aleksandra and Jan)
</P><P>Restriction of the Fur and SinR PCR products was performed using an enhanced protocol with an O/N incubation in 37 deg. C (Dainius)
</P></DIV><DIV id="19-July" class="notebookEntry"><UL><LI><B>19th of July</B>:
</LI></UL><P>Due to low amount of product obtained second PCR reaction was set up to amplify the DNA fragments required to perform deletions in the <I>B. subtilis</I> sporulation pathway (Dainius and Kyle)
</P><P>The O/N restriction product was purified from the reaction mix. This was used for two separate ligation reactions using different ligant-to-vector ratios and [vactor] (Hugo and Dainius)
</P><P>New GenBrick linkers were prepared to be tested over the next week (Aleksandra)
</P></DIV><DIV id="20-July" class="notebookEntry"><UL><LI><B>20th of July</B>:
</LI></UL><P>The ligation products were used for transformation of super-competent <I>E. coli</I> cells (Hugo and Dainius)
</P></DIV><DIV id="22-July" class="notebookEntry"><UL><LI><B>22nd of July</B>:
</LI></UL><P>The transformation result was positive. Four colonies per insert were selected and subjected to colony PCR. Moreover, those were propagated into 5ml LB + AMP cultures. Cultures were grown for 7 h and entire 5 ml was used for plasmid purification using the Qiagen MiniPrep kit. The resulting DNA was quantified using NanoDrop (Hugo and Dainius)
</P><P>The MABEL blunt end ligation product was transformed into competent <I>E. coli</I> cells and plated on LB+KAN+IPTG+X-gal plates. Those were incubated in 37 deg. C O/N. Moreover, the purified linear MABEL product was run on the DNA electrophoresis gel to assess the quality of the large fragment (&gt;7 kb) purification process when using Qiagen PCR product purification kit (Aleksandra and Jan)
</P></DIV><DIV id="23-July" class="notebookEntry"><UL><LI><B>23rd of July</B>:
</LI></UL><P>Cloned parts were excised from working vector and inserted into pSB1C3. A 2h restriction with XbaI/ PstI and an O/N ligation was performed to achieve that. Moreover, cloned parts were mixed with sequencing vectors so they are ready to be submitted to a sequencing facility tomorrow (Hugo and Dainius)
</P><P>The transformation resulting from the blunt end ligation was successful. Six colonies were chosen and propagated into 5ml LB medium + KAN. Following 7h incubation in 37 deg. C the cultures were propagated into fresh 5ml LB + KAN + 2% lactose to induce expression of pdc_adhB fusion protein. The non fusion cultures were set up and induced at the same time to have a point of reference (Aleksandra and Jan)
</P><P><I>E. coli</I> JM109 cells were made chemically competent. <I>E. coli</I> and <I>B. subtilis</I> were patched onto fresh LB medium to be used in other experiments. Some CML plates were prepared (Aleksandra and Jan)
</P></DIV><DIV id="24-July" class="notebookEntry"><UL><LI><B>24th of July</B>:
</LI></UL><P>The ligation mix was used for transformation of competent <I>E. coli</I> cells. X-gal and IPTG were included in the plates (Hugo and Dainius)
</P><P>A PCR reaction of the flanking regions required for deletion in the <I>B. subtilis</I> genome was performed (Kyle)
</P><P>SDS-PAGE experiment was run to analyse the cell lysate from un-induced and induced <I>E. coli</I> bearing a plasmid coding for pET operon and pET operon fusion. The gel was left for staining O/N. The samples were also analysed for ethanol production (Aleksandra and Jan)
</P></DIV><DIV id="25-July" class="notebookEntry"><UL><LI><B>25th of July</B>:
</LI></UL><P>The transformation was successful. About half of the cells had inserts of interest in pSB1C3. Those were propagated into 5ml LB + CML. A MimiPrep was performed in the afternoon to obtain a pure DNA sample for sequencing (Hugo and Dainius)
</P><P>SDS-PAGE was destained for an hour and photographed. De-staining was continued o/n (Aleksandra and Jan)
</P></DIV><DIV id="26-July" class="notebookEntry"><UL><LI><B>26th of July</B>:
</LI></UL><P>The pSB1C3 with Fur and SinR was purified from o/n cultures using Qiagem MiniPrep Kit. The DNA samples are ready for the submission to the Registry (Hugo and Dainius)
</P><P>Three cultures of pET operon fusion showing correct band on the SDS-PAGE were subjected to Qiagen MiniPrep kit. The purified plasmid was restricted using PstI and XbaI and used for ligation (4 deg. C incubation) with the pSB1C3 - RFP restriction product. All restriction products were analysed on an agarose gel (Aleksandra and Jan)
</P></DIV><DIV id="29-July" class="notebookEntry"><UL><LI><B>29th of July</B>:
</LI></UL><P>The pET fusion_pSB1C3 ligation product was used for transformation of competent <I>E. coli</I> cells. Cells were plated on LB+CML+X-gal+IPTG plates to enable blue- red screening (Aleksandra and Jan)
</P><P>A set of eight O/N <I>E. coli</I> cultures was set up to test influence of varying concentrations of lactose, xylose and glucose on ethanol production (Aleksandra and Jan)
</P><P>GenBrick assemblies prepared last week were used for transformation of competent <I>E. coli</I> cells (Hugo and Weike)
</P><P><I>B. subtilis</I> wild strain was recovered from a freeze-dried ampule and plated on nutrient and LB agar (Kyle)
</P></DIV><DIV id="30-July" class="notebookEntry"><UL><LI><B>30th of July</B>:
</LI></UL><P>Ethanol production in media containing different sugars was analysed using an ethanol assay (Aleksandra and Jan)
</P><P>Colonies resulting from the transformation were picked and propagated into 5ml LB + CML medium in order to purify plasmid with insert and pSB1C3-RFP tomorrow (Aleksandra)
</P></DIV><DIV id="31-July" class="notebookEntry"><UL><LI><B>31st of July</B>:
</LI></UL><P>O/N cultures expressing the Reporter genes for GenBrick assembly and pET fusion in pSB1C3 were subjected to Qiagen MiniPrep Kit in order to purify plasmids. Obtained plasmids are ready for the submission to the Registry (Aleksandra and Jan)
</P><P>New reagent: Nitro Blue Tetrazolium was tested as a component of ethanol assay. Standard measurements were taken at both 530 and 570 nm in order to evaluate the suitable wavelength (Aleksandra and Jan)
</P><P>Following the protocol for competent Bacillus subtillis cells taken from <A href="http://openwetware.org/wiki/Cfrench:BacTrans2" class="external free" rel="nofollow">http://openwetware.org/wiki/Cfrench:BacTrans2</A> . Made some caramel minimal medium, the recipe is uncertain at the moment. The new B. subtilis was put to grow O/N in it. (Dainius and Kyle)
</P><P>PCR and Gel of SinI and SinR (Hugo)
</P></DIV><DIV id="1-August" class="notebookEntry"><UL><LI><B>1st of August</B>:
</LI></UL><P>Ethanol production in media containing different sugars was analysed using an alternative ethanol assay with NBT as a dye (Jan)
</P><P>GenBrick linkers with and without Fur box were phosphorylated and annealed. An O/N reaction of fusing the linkers to parts required for Fur box characterisation was set up (Hugo and Jan). 
</P><P>The caramel minimal medium is not toxic to B. subtilis cells. So i continued working with it and aliquoted the overnight culture into fresh medium for 3 hours, then added starvation medium for 3 hours, aliquoted 0.3ml into ependorfs and put into -80C. (Dainius)
</P></DIV><DIV id="2-August" class="notebookEntry"><UL><LI><B>2nd of August</B>:
</LI></UL><P>Dainius and Kyle made some more caramel minimal medium. Great success!
1 part salt, 1 part sugar, 1 part love, 200ml of DI water, try to measure pH and fail, put to autoclave for 2hours, serve hot.
</P><P>God gave us some real minimal salts, so we worked with that instead, following the protocol used with caramel medium on 31st July. (Dainius and Kyle)
</P><P>PCR purification of fusions from yesterday were performed. Four assemblies were made and used for E. coli transformations on CML plates. peace. (Hugo)
</P></DIV><DIV id="3-August" class="notebookEntry"><UL><LI><B>3rd of August</B>:
</LI></UL><P>Three colonies of 3 different assemblies from yesterday were put in LB broth. The rest of the competent transformed cells were concentrated and plated on CML plates. Will it work? Will it not? The suspense is great. Tune in tomorrow to see if the plates grew. (Hugo)
</P><P>Meanwhile biofilm forming Bacillus subtilis competent cells were made and put in -80C, ready for action. (Dainius)
</P></DIV><DIV id="5-August" class="notebookEntry"><UL><LI><B>5th of August</B>:
</LI></UL><P>Competent biofilm forming B. subtilis competent cells were transformed with David's kanamycin resistance plasmid and plated out. (Dainius and Kyle)
</P><P>Some transformations were made with GFP and RFP. (Hugo)
</P><P>A native PAGE gel was casted for functional analysis of pET and pET fusion (Aleksandra)
</P><P>A PCR was run and analysed on a 2% agarose gel to establish the size difference in the spacing between fused and non-fused pET operon genes (Jan)
</P><P>Set of nine O/N cultures was set up. Those are both induced and not induced for ethanol production parts expression. The cultures will be analysed on a native PAGE tomorrow (Jan)
</P></DIV><DIV id="6-August" class="notebookEntry"><UL><LI><B>6th of August</B>:
</LI></UL><P>A PCR was run and analysed on a 2% agarose gel to establish the size difference in the spacing between fused and non-fused pET operon genes. This time a different DNA polymerase and highier primer concentrations were used. A sucesfull reaction indicated bands of size corresponding to fusion and non-fusion genes on the gel (Jan)
</P><P>Two native PAGEs were run. Firs one was subjected to o/n comassie staining. The second gel was subjected to functional staining for AdhB activity (Aleksandra and Jan)
</P><P>A PCR was run to mutagenise a shuttle vector using GenBrick based EMMA technique. The aim is to create a plasmid which could be used in both <I>E. coli</I> and <I>B. subtilis</I> and would be BioBrick compatible (Harry and Gavin)
</P><P>A transformation of RFP and GFP parts required for GenBrick assembly characterising the Fur binding sequence was plated out (Hugo)
</P><P>Nothing grew on the kanamycin plates with Bacillus subtilis. We will try a new method tomorrow. (Dainius and Kyle).
</P></DIV><DIV id="7-August" class="notebookEntry"><UL><LI><B>7th of August</B>:
</LI></UL><P>To obtain even more GenBrick reporter parts two more transformations were performed (Hugo).
</P><P>A PCR was run to amplify genes coding for SinR and Fur transcription factors. Set of new primers was used with both BioBrick and GenBrick standard overhangs. PCR products were analysed on a gel and restricted using PstI and XbaI. An O/N ligation was set up which would insert those into pSB1C3 (Jan).
</P><P>Two native PAGE were casted to be used in functional analysis of AdhB (Aleksandra)
</P><P>Set of 12 <I>E. coli</I> o/n cultures with appropriate additives were set up. Those will be analysed tomorrow for both ethanol content and adhB activity, using native PAGE (Aleksandra and Jan)
</P><P>Three more PCRs were run to mutagenise a shuttle vector using EMMA technique. All products were analysed on a gel. Fragments of expected size were observed (Harry and Gavin)
</P><P>Competent <I>E. coli</I> cells were transformed with a vector on which new shuttle will be based in order to obtain more DNA to work with (Harry and Gavin)
</P><P>A new method for transforming B. subtilis was used for introducing kanamycin resistance (Yasbin method). (Dainius)
</P></DIV><DIV id="8-August" class="notebookEntry"><UL><LI><B>8th of August</B>:
</LI></UL><P>Proteins were extracted from a set of 12 o/n cultures and prepared for an analysis on a native PAGE. Tween 20 detergent was included to increase amount of protein in the sample (Aleksandra)
</P><P>A PCR was run to amplify entD; entE; entF genes and flanking region of SinR gene. PCR products were analysed on a gel and stored in -20 deg. C for purification  and restriction tomorrow (Kyle and Jan)
</P><P>O/N ligation product was used for transformation of competent <I>E. coli</I> cells and plated on CML+IPTG plates (Jan)
</P><P>PCR products required for construction of <I>B. subtilis</I> - <I>E. coli</I> shuttle vector were purified using Qiagen PCR product purification kit (Harry and Gavin)
</P><P>Failed. (Dainius)
</P></DIV><DIV id="9-August" class="notebookEntry"><UL><LI><B>9th of August</B>:
</LI></UL><P>Trying it again, but also with <I>B. subtilis</I> 168 and concentrating the cells 4x before adding DNA. (Dainius)
</P><P>Colonies resulting from a transformation using Fur_pSB1C3 and SinR_pSB1C3 ligation products were propagated into 5ml LB + CML and grown in 37 deg. C shaking. Following 6 h incubation the cultures were minipreped (Aleksandra and Jan)
</P><P>Native PAGE was run and analysed for adhB activity (Aleksandra)
</P><P><I>E. coli</I> cultures grown in 4% glucose were analysed for ethanol production. Both induced/ un-induced and fusion/ non-fusion samples were assayed (Aleksandra and Jan)
</P><P>8 dsRED minipreps, more competent cells, and GFP 5 transformations were made. Love. (Hugo)
</P></DIV><DIV id="10-August" class="notebookEntry"><UL><LI><B>10th of August</B>:
</LI></UL><P>Competent cells done. Made some supercompetent transformations. (Hugo)
</P><P>Nothing worked again. I think the plasmid/the antibiotic used may be wrong all along. (Dainius)
</P></DIV><DIV id="12-August" class="notebookEntry"><UL><LI><B>12th of August</B>:
</LI></UL><P>Native PAGE gels were casted in order to analyse the cultures for adhB and pdc activity tomorrow (Aleksandra)
</P><P>Various <I>E. coli</I> cultures expressing fused and non-fused pdc and adhB were set-up. Both anaerobic and aerobic conditions were used for incubation (Jan)
</P><P>Ligation products of enterobactin genes into pSB1C3 were transformed into competent <I>E. coli</I> cells (Hugo and Jan)
</P></DIV><DIV id="13-August" class="notebookEntry"><UL><LI><B>13th of August</B>:
</LI></UL><P>Pdc activity assay was developed based on the aldehyde assay using Shiffs reagent (Jan)
</P><P>Native PAGE was performed on the cell extracts from pET and pET fusion. AdhB and pdc activity staining was performed. Separate gel was left for O/N Comassie staining (Aleksandra)
</P><P>Cell extracts from pET and pET fusion were analysed using NAD+ assay (Jan and Aleksandra)
</P><P>SinR and Fur in pSB1C3 was MiniPreped from O/N cultures (Jan)
</P></DIV><DIV id="14-August" class="notebookEntry"><UL><LI><B>14th of August</B>:
</LI></UL><P>PCR was set up to amplify the adhE gene from <I>Clostridium acetobutylicum</I> gDNA. The product was purified and analysed on a gel (Aleksandra)
</P><P>Protein content of the isolates from cells expressing the fusion and non-fusion pET operons was measured using Pierce reagent (Jan)
</P><P>The gel stained for adhB activity was finally visualised. The Comassie stained gel was destained (Jan)
</P></DIV><DIV id="15-August" class="notebookEntry"><UL><LI><B>15th of August</B>:
</LI></UL><P>PCR was set up to amplify the adhE; entD; entE and entF genes. <I>E. coli</I> MG1655 cells were used as a template for the enterobactin synthesis genes. All of the products were analysed on 20 ml agarose gel. Additionally, the adhE was analysed on 40 ml agarose gel. EntD and EntF samples were subjected to the Qiagen PCR product purification kit and the quality of samples was assessed using NanoDrop (Aleksandra and Jan). 
</P><P>More 2xLB medium was prepared. The medium was autoclaved and used to set up two <I>E. coli</I> cultures with fused and non-fused pET cassette (Aleksandra). 
</P><P>Larger volumes of cell extracts from pET and pET fusion were analysed using NAD+ assay (Jan)
</P><P>More investigation was done into the best way of performing the aldehyde assay using Shiffs reagent. A new ratio of reagents was determined to give a more sensitive response (Jan)
</P></DIV><DIV id="16-August" class="notebookEntry"><UL><LI><B>16th of August</B>:
</LI></UL><P>PCR was run to amplify adhE and entE genes. PCR products were analysed on a gel (Jan)
</P><P>O/N <I>E. coli</I> cultures with fused and non-fused pET cassette were processed. First the OD600 was measured. Following that cell suspension was concentrated and sonicated. Amount of protein was quantified in each crude cell extract. Moreover, the lysate was analysed on a Native PAGE. Gels were stained for the adhB and pdc activities and an O/N Comassie staining was set-up (Jan and Aleksandra)
</P><P>NAD+ assay was performed on the crude extracts of <I>E. coli</I> cultures expressing fused and non-fused pET cassette. A set of valuable data was obtained which enable determination of kinetic parameters for the adhB in the fusion state (Aleksandra and Jan). 
</P><P>Amount of ethanol was measured in O/N <I>E. coli</I> cultures with fused and non-fused pET cassette. Moreover, the remaining culture was propagated into anaerobic conditions and incubated in 37 deg. C; slow rotation over the weekend (Aleksandra)
</P></DIV><DIV id="17-August" class="notebookEntry"><UL><LI><B>17th of August</B>:
</LI></UL><P>Restriction products of EntD; EntF and pSB1C3 were purified and analysed on a NanoDrop. O/N ligation reactions were set-up (Jan)
</P><P>The PCR products of EntE and AdhE were analysed on 40 ml 1.6% agarose gel. An intriguing band was observed in an EntE line. This was excised and purified using a gel extraction kit (Jan)
</P><P>The o/n comassie staining for the native PAGE was destained (Jan)
</P></DIV><DIV id="18-August" class="notebookEntry"><UL><LI><B>18th of August</B>:
</LI></UL><P>The EntD and EntF ligation products were transformed into competent <I>E. coli</I> cells (Jan)
</P><P>The 48h anaerobic fusion and non-fusion cultures were analysed for ethanol production (Jan)
</P><P>Fur and EdiBrick plasmids were restricted in order to obtain an induicable expression of iron responsive transcription factor. An O/N ligation reaction was set-up (Jan)
</P></DIV><DIV id="19-August" class="notebookEntry"><UL><LI><B>19th of August</B>:
</LI></UL><P>The 72h anaerobic fusion and non-fusion cultures were analysed for ethanol production and growth (Aleksandra)
</P><P>A PCR was run to fuse pdc and adhB without EdiBrick using MABEL technology (Jan). MABEL results were analysed on a gel (Aleksandra) and used for an O/N phosphorylation/ ligation reaction (Jan)
</P><P>Results from the EntD and EntF transformation were obtained. Positive (white) colonies were patched on CML + IPTG plate and grown for 8h. After good growth was obtained the colonies were used as a template for the colony PCR using GoTaq polymerase. The colonies were also propagated into 5ml liquid cultures (Jan)
</P><P>Set of 10 <I>E. coli</I> cultures grown in different [Fe2+] was made. Those will be used to asess the toxicity of iron before Fur TF is over-expressed (Jan)
</P><P>O/N cultures with plasmid coding for fused and non-fused pET casette were set-up. Both inducing and non-inducing conditions were used (Jan)
</P></DIV><DIV id="20-August" class="notebookEntry"><UL><LI><B>20th of August</B>:
</LI></UL><P>A colony PCR was run on selected colonies suspected to have EdiBrick_Fur construct. The PCR products were analysed on a gel(Aleksandra)
</P><P>O/N cultures with plasmid coding for fused and non-fused pET casette were analysed. OD600 was measured and phase contrast microscopy was performed. Interesting observations prompted us to perform cell count of all the cultures to establish actual number of the cells. Cultures were dilluted by DF = 10^5 and 10^6 and plated out (Aleksandra and Jan)
</P><P>OD600 of 10 <I>E. coli</I> cultures grown in different [Fe2+] was measured to establish basic toxicity levels of iron (Jan)
</P><P>The ligation product resulting from O/N blunt end ligation of pET only plasmid was used for transformation of competent <I>E. coli</I> cells. The cells were plated on antibiotic plates and incubated in 37 deg. C O/N (Aleksandra)
</P><P>A PCR was set up to amplify the AdhE gene from both C. acetobutylicum gDNA and glycerol stock. As a positive control a reaction amplifying the rRNA region of the genome was made. The PCR products were analysed on agel (Jan)
</P><P>A set of four O/N anaerobic cultures of Weizmann organism was made using LB medium. The cultures were incubated in 37 deg. C; 200 rpm(Jan)
</P><P>Furbox assemblies were made once more and transformed. (Hugo and Kyle)
</P></DIV><DIV id="21-August" class="notebookEntry"><UL><LI><B>21st of August</B>:
</LI></UL><P>It seems like no assemblies worked but made cultures of possible assemblies nonetheless. (Hugo)
</P><P>Made a gel of uncut and cut plasmids of DsRed, GFP and Acc_RFP to see which part of the assembly does not work. (Hugo and Kyle)
</P><P>A cell counts were done on cultures with plasmid coding for fused and non-fused pET casette. The phase contrast microscopy pictures were obtained. The combined result suggest an interesting feature of cells expressing fused pET cassette (Aleksandra and Jan)
</P><P>A EdiBrick_Fur construct was purified and analysed using XbaI and PstI restriction (Aleksandra, Hugo and Jan)
</P><P>5 O/N <I>E. coli</I> cultures induced to express <I>B. subtilis</I> Fur TF were grown in different [Fe] (Hugo and Jan)
</P><P>A MABEL reaction was performed to change pLac upstream of pET fusion into pSpac (Jan). PCR product was analysed on a gel (Aleksandra), purified and used as a substrate for an O/N blunt end ligation (Jan)
</P><P>8 colonies resulting from transformation using a blunt end ligation product of pET fusion only were selected. Those were grown in 5 ml cultures and MiniPreped in the afternoon. Purified DNA was used for an O/N restriction reaction (Aleksandra and Jan)
</P><P>Obtained cultures of Weizmann organism were used as a template in PCR amplifying AdhE coding sequence. A control amplifying 23S rRNA region was used. GoTaq from PROMEGA was used for the reaction (Aleksandra and Jan)
</P></DIV><DIV id="22-August" class="notebookEntry"><UL><LI><B>22nd of August</B>:
</LI></UL><P>The PCR products of AdhE and restriction products of pET only were analysed on an agarose gel (Aleksandra)
</P><P>Four pET only restriction reactions were selected and ligated into pSB1C3 (Jan)
</P><P>The pET casette fusion pLac to pSpac MABEl blunt end ligation products were used for the transformation of competent cells and plate on CML, IPTG, Xgal plates (Jan)
</P><P>Protein samples were taken from induced fusion and non-fusion pET cassette and uninduced one. Those were prepared and run on a SDS PAGE (Aleksandra and Jan)
</P><P>OD600 was measured for O/N <I>E. coli</I> cultures over- expressing Fur TF  and grown in various [Fe] (Jan)
</P><P>A MABEL PCR was set up to change pLac to pSpac in both EdiBrick and non-fusion pET under the control of EDiBrick (Aleksandra) the PCR products were analysed on a gel, purified and used for an O/N blunt end ligation (Jan)
</P><P>Protein isolates from induced fusion and non-fusion pET cassette and uninduced one were assayed using Pierce reagent (Jan)
</P><P>Miniprep of the second set of assemblies done my Hugo and Kyle (Kyle)
</P></DIV><DIV id="23-August" class="notebookEntry"><UL><LI><B>23rd of August</B>:
</LI></UL><P>An excursion was made to a certain location from which distillery waste water samples were obtained (Aleksandra and Jan)
</P></DIV><DIV id="24-August" class="notebookEntry"><UL><LI><B>24th of August</B>:
</LI></UL><P>1 ml of each water sample from the distillery was mixed with 20 ml of molten agar and poured on a plate. Those were incubated in 37 deg. C (Jan)
</P><P>Samples of water from the distillery were sterile filtered and 10M NaOH was added to them in order to perticipate all insoluble Me hydroxydies (Jan)
</P><P>Induced pET fusion and non-fusion cultures were set up and incubated o/n in 37 deg. C shaking (Jan)
</P><P>Two white colonies were selected from pLac to pSpac MABEL and propagated into 5ml LB + CML (Jan)
</P><P>Transformation was performed using 2 ul of purified pSB1C3 RFP plasmid in order to evaluate the transformation efficiency of new batch of competent cells (Jan)
</P></DIV><DIV id="25-August" class="notebookEntry"><UL><LI><B>25th of August</B>:
</LI></UL><P>The ligation products of: fusion only - pSB1C3; pSpac and pSpac - non fusion were transformed into competent <I>E. coli</I> cells (Jan)
</P><P>A restriction of EdiBrick- non fusion and of pSB1C3- RFP was performed using PstI and XbaI. The restriction products were purified and used for an O/ N ligation reaction (Jan)
</P><P>The o/ n Induced pET fusion and non-fusion cultures were transferred into anaerobic incubation in order to enable alcoholic fermentation to proceed (Jan)
</P><P>O/N cultures of <I>E. coli</I> MG1655 and <I>B. subtilis</I> 168 were set up and incubated in 37 deg.C; 200 rpm (Jan)
</P></DIV><DIV id="26-August" class="notebookEntry"><UL><LI><B>26th of August</B>:
</LI></UL><P>Twelve sequencing reactions were set up using pSBNX3 and other primers (Aleksandra)
</P><P>Colonies were picked up from the transformations done yesterday. pSpac colony was propagated into LB + AMP and Fusion only in pSB1C3 into LB + CML. The cultures were grown in 37 deg. C O/N (Aleksandra)
</P><P>Total protein content was measured in an uninduced and induced <I>E. coli</I> cells having a plasmid coding for pET cassette fusion (Jan)
</P><P>A restriction analysis was performed to observe the size difference between EdiBrick_pET_Fusion and pSpac_lacZ_pET_fusion. The 36 bp difference between the fragments was resolved using 2.5% agarose gel run for 50 min at 110V (Jan)
</P><P>1 ml of each water sample from the distillery was mixed with 20 ml of molten agar and poured on a plate. Those were incubated in 16 deg. C (Aleksandra and Jan)
</P><P>Transformation was performed using 10 ul of EdiBrick_pET and pSB1C3 ligation product. The cells were plated on LB + CML + IPTG plates and incubated O/N in 37 deg. C (Aleksandra and Jan)
</P><P>Six more colonies were picked from the pSpac_Fusion plate and propagated int 5 ml LB + CML medium (Jan)
</P><P>O/N cultures of <I>E. coli</I> MG1655 and <I>B. subtilis</I> 168 were set up and incubated in 37 deg.C; 200 rpm (Aleksandra)
</P></DIV><DIV id="27-August" class="notebookEntry"><UL><LI><B>27th of August</B>:
</LI></UL><P>Four cultures of pSpac_Fusion, a culture of pSpac and a culture of Fusion only in pSB1C3 were minipreped. NanoDrop was used to asses the quality of obtained DNA (Aleksandra) 
</P><P>Dried Whatmann filters with the entire hydroxide participate of upstream and downstream water samples were weighted. The masses were compered to identify any obvious differences in  heavy metal ion contents of the water samples (Jan)
</P><P>An ethanol assay was performed on 24h aerobic fusion and non-fusion cultures and on 48h anaerobic fusion and non- fusion cultures. A set of appropriate standard measurements was made in order to obtain reliable results (Jan)
</P><P>A MABEL PCR was performed to obtain a pSpac_lacZ_Fur construct. The reaction products were analysed on a gel and used for an O/N blunt- end ligation reaction (Aleksandra)
</P><P>A transformation was performed using DH5alpha competent cells and remaining 10 ul of the pSB1C3 - EdiBrick_pdc_adhB ligation product. Transformed cells were plated on CML + IPTG plates and incubated in 37 deg. C O/N (Jan)
</P><P>A glycerol stock of EdiBrick_Fur was revived by propagating an inoculum of cells into 20 ml LB + CML medium and incubating in 37 deg. C; 200 rpm O/N (Jan)
</P></DIV><DIV id="28-August" class="notebookEntry"><UL><LI><B>28th of August</B>:
</LI></UL><P>Selected colonies from the transformation were propagated into 5 ml LB + CML and incubated o/n in 37 deg. C (Jan)
</P><P>Eight cultures were set up using different iron concentrations and cells over expressing or not the Fur TF ( Jan)
</P></DIV><DIV id="29-August" class="notebookEntry"><UL><LI><B>29th of August</B>:
</LI></UL><P>An ethanol assay was performed  on 96h anaerobic fusion and non- fusion cultures. A set of appropriate standard measurements was made in order to obtain reliable results (Jan)
</P></DIV><DIV id="10-September" class="notebookEntry"><UL><LI><B>10th of September</B>:
</LI></UL><P>Prepared medium for growth for Christianas measurements of Fusion and Nonfusion in LB+4%glucose. (Dainius)
</P></DIV><DIV id="11-September" class="notebookEntry"><UL><LI><B>11th of September</B>:
</LI></UL><P>Did the growth curves and prepared media for tomorrow's work (0%, 1%, 2% glucose stuff). (Dainius)
</P></DIV><DIV id="16-September" class="notebookEntry"><UL><LI><B>16th of September</B>:
</LI></UL><P>Four transformations were performed using competent JM109 cells. Following parts from the Registry Spring 2013 distribution were used: BBa_B0032; BBa_B0030; BBa_B0033 - all of those contain RBS of varying strength. Additionally a pLac_LacZ_RBS (BBa_B0034) was transformed (Aleksandra)
</P><P>Set of four O/N cultures with fusion and non-fusion pET cassette were set-up (Jan)
</P></DIV><DIV id="17-September" class="notebookEntry"><UL><LI><B>17th of September</B>:
</LI></UL><P>Colonies were selected from the transformation plates and propagated into 5ml LB medium supplemented with an adequate antibiotic (Aleksandra)
</P><P>Using o/n cultures expressing fusion and non-fusion pET cassette ethanol production cultures were set up (Jan)
</P></DIV><DIV id="18-September" class="notebookEntry"><UL><LI><B>18th of September</B>:
</LI></UL><P>Obtained o/n cultures containing desired RBS parts were MiniPreped (Jan)
</P><P>MiniPreps of RBS parts were restricted using XbaI and EcoRI. PLac-LacZ construct was restricted using SpeI and EcoRI. O/N ligation was set up using obtained restriction products (Aleksandra)
</P><P>O/N ethanol production cultures were propagated into fermentative conditions and incubated in slow rotation (Jan)
</P></DIV><DIV id="19-September" class="notebookEntry"><UL><LI><B>19th of September</B>:
</LI></UL><P>A PCR was performed to amplify LDH-D and LDH-L coding sequences from <I>B. subtilis</I> and <I>E. coli</I> gDNA (Jan)
</P><P>PCR products were analysed on an agarose gel. Bands of correct size were observed. PCR products were purified and restricted using XbaI and PstI. Restricted fragments were used for O/N ligation into restricted pSB1C3-RFP (Aleksandra)
</P></DIV><DIV id="20-September" class="notebookEntry"><UL><LI><B>20th of September</B>:
</LI></UL><P>Ligation products of LDH genes and pSB1C3, RBS and pLac_LacZ were transformed into competent <I>E. coli</I> cells and plated on LB plates supplemented with CML, IPTG and X-gal where required (Jan)
</P><P>48h fermentative <I>E. coli</I> cultures expressing both fused and non-fused pET cassette were pelleted and a supernatant was assayed for ethanol content (Jan)
</P></DIV><DIV id="21-September" class="notebookEntry"><UL><LI><B>21st of September</B>:
</LI></UL><P>Colonies obtained from transformation were transfered into 5ml LB supplemented with an antibiotic and incubated in 37 de. C O/N (Aleksandra)
</P><P>An o/n culture of <I>E. coli</I> MG1655 and JM109 was set-up (Jan)
</P></DIV><DIV id="22-September" class="notebookEntry"><UL><LI><B>22nd of September</B>:
</LI></UL><P>An o/n culture of <I>E. coli</I> MG1655 was set-up (Jan)
</P><P>Pure plasmid DNA was isolated from 5ml cultures using Qiagen MiniPrep kit. Obtained DNA samples were analysed using NanoDrop and are ready for the submission to the Registry (Aleksandra and Jan)
</P></DIV><DIV id="23-September" class="notebookEntry"><UL><LI><B>23rd of September</B>:
</LI></UL><P>Set of 7 parts was packed and sent to the Registry of Standard Biological Parts (Aleksandra and Jan)
</P></DIV><DIV id="24-September" class="notebookEntry"><UL><LI><B>26th of September</B>:
</LI></UL><P>Five restriction reactions were set up. Those included: L-LDH, D-LDH, pLac-LacZ, pSpac-Fusion and pTG262-RFP(Jan).
</P><P>Set of three o/n ligation reactions was set-up. Following products of interest will be generated: pLac_LacZ_L-LDH; pLac_LacZ_D-LDH; pSpac_Fusion in pTG262 (Aleksandra)
</P></DIV><DIV id="27-September" class="notebookEntry"><UL><LI><B>27th of September</B>:
</LI></UL><P>Three transformations were performed using competent JM109 <I>E. coli</I> cells. Cells bearing pLac_LacZ-LDH construct were plated on CML, IPTG, XGAL plates and those having pSpac_Fusion in pTG262 were plated on Kan25, IPTG plates (Aleksandra and Jan)
</P></DIV><DIV id="28-September" class="notebookEntry"><UL><LI><B>28th of September</B>:
</LI></UL><P>Blue colonies resulting from transformation of pLac_LacZ-LDH constructs were picked and propagated into 5ml LB sumpplemented with CML (Jan)
</P></DIV><DIV id="29-September" class="notebookEntry"><UL><LI><B>29th of September</B>:
</LI></UL><P>O/N cultures were MiniPreded in order to obtain a pure sample of pLac_LacZ_RBS_D/L_LDH (Jan)
</P></DIV><DIV id="30-September" class="notebookEntry"><UL><LI><B>30th of September</B>:
</LI></UL><P>A double transformation was performer using pLac_LacZ_RBS_D/L_LDH in pSB1C3 and pLac_LacZ_pET_Fusion in pSB1K3. Resulting cells were plated on CML+KAN plates and incubated o/n in 37 deg. C (Jan)
</P><P>A transformation was performer using pLac_LacZ_RBS_FbpA1/2 and pLAc_LacZ_RBS_Fur ligation products. Resulting cells were plated on CML+IPTG+XGAL plates (Jan)
</P><P>A set of four o/n 5ml cultures was set-up. Single colonies of pLac_LacZ_RBS_L_LDH and pLac_LacZ_RBS_D_LDH were used (two cultures each). Cultures were supplemented with CML only (un-induced) or CML+IPTG (induced)and incubated in 37 deg. C, shaking O/N (Jan)   
</P></DIV><DIV id="1-October" class="notebookEntry"><UL><LI><B>1st of October</B>:
</LI></UL><P>A SDS-PAGE was performer on cel-lysates from cultures which were induced and non-induced for D and L - LDH expression. Gel was left for Coomassie Brilliant Blue G-250 staining o/n (Aleksandra and Jan)
</P><P>Colonies resulting from transformations performer using pLac_LacZ_RBS_D/L_LDH in pSB1C3 and pLac_LacZ_pET_Fusion in pSB1K3 constructs were collected and propagated into 5ml LB + KAN + CML (Jan)
</P><P>Colony of cells having a pLac_LacZ_RBS_FUR construct was propagated into 5ml LB + CML (Jan)
</P><P>Miniprep of the fur box assemblies (Kyle)
</P></DIV><DIV id="2-October" class="notebookEntry"><UL><LI><B>2nd of October</B>:
</LI></UL><P>SDS-PAGE was destained (Jan)
</P><P>The cultures with plasmids containing pLac_LacZ_RBS_D/L_LDH in pSB1C3 and pLac_LacZ_pET_Fusion in pSB1K3 constructs were propagated into a fermentative culture supplemented with D/L lactic acid racemate. Cultures were incubated o/n in 37 deg. C, shaking (Jan) 
</P><P>Digestion and gel comparing undigested assembled plasmid to digested assembly (Kyle)
</P></DIV><DIV id="3-October" class="notebookEntry"><UL><LI><B>3rd of October</B>:
</LI></UL><P>32 sequencing reactions were set-up and submitted to the GenePool sequencing facility (Harry)
</P><P>Cells overexpressing Fur transcription factor, FbpA (iron binding protein) were plated on CML +IPTG and supplemented with ferric citrate (Jan)
</P></DIV></DIV></DIV><TABLE align="center" class="footer"><TBODY><TR><TD colspan="6" align="center"> This iGEM team has been funded by the MSD Scottish Life Sciences Fund. The opinions expressed by this iGEM team are those of the team members and do not necessarily represent those of MSD
</TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Edinburgh/Project/Notebook">http://2013.igem.org/Team:Edinburgh/Project/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Project/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Project/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Project/Notebook&amp;oldid=306627" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>