Team:William and Mary/Speed Control
Toggle navigation
Description
Project Overview
Demonstrate
Measurement
Results
Results Overview
Degradation
Speed Control
Readjustment
Dynamic Control
Proof of Concept
Enabling Teams
Modeling
Human Practices
Overview
Consult Community
Build Connections
Sustainable Impact
HP/Silver
Activities Booklet
Judging
Achievements
Medal Requirements
Parts
Composite Parts
Part Collection
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Overview
To measure speed, we measured our pTet mScarlet-I pdt reporter constructs (on pSB1C3) along with our pLac mf-Lon construct (on pSB3K3) into NEB 10-Beta cells. After growing them for 8 hours and diluting them to an OD600 of .005, we simultaneously induced each construct with 0.1mM IPTG and 50ng/mL ATC. Simultaneous induction was chosen because our preliminary math modeling suggested that a greater speed change would be observed when using simultaneous induction rather than inducing reporters with mf-Lon already at a steady state concentration.
Before we took speed measurements, we first tested a variety of different time course methods for use with flow cytometry. We tried cryogenic freezing, staggering cells, and fixation with 1% PFA for various lengths of time. Ultimately we found that the most robust method was simply taking aliquots of cells at each time point, adding them to PBS on ice and measuring them immediately with FACS. We found that this method was robust across different days, as well as robust to experimental conditions and other errors. We noted that fluorescence per cell did not change significantly after 20 minutes on ice, which represents the longest any sample sat on ice before being measured. All measurements were performed in NEB 10-Beta cells and M9 media with glucose and 0.1% casamino acids added, and dilutions were made to maintain cells in the mid-log growth phase for the entirety of the experiment. A full protocol can be found here .
All data on this page represents the geometric means of at least three biological replicates (colonies) taken on the same day. Each biological replicateâ€™s fluorescence was determined on the FL3 channel of an S3e cell sorter, and the geometric mean of at least 10,000 (typically 20,000) cells was used. Conversion to absolute units (MECY) was performed using spherotek rainbow calibration beads and Flowcal. The shaded region represents one geometric standard deviation above and below the mean of the biological replicates.
Figure 1: Schematic of a generic reporter construct used to test gene expression speed. mScarlet-I is produced in the prescence of ATC, and is degraded by mf-Lon
Results
In our experiment we observed a robust tag dependent speed change during the course of our experiments. Figure 2A shows plots of normalized fluorescence over time. Steady state was defined as the point where the next two consecutive data points did not exhibit any increase in fluorescence. That data point was then used to normalize the previous values of that time course. Figure 2B shows these same results without truncation, and as predicted by mathematical modeling we see that fluorescence starts to decay down to a second lower steady state. This result is because of the choice of simultaneous induction, and was later used to help create our IFFL circuits.
Figure 2: (A) Truncated speed graph of tagged mScarlet-I constructs. Stronger tags (earlier letters), show greater speed change.
(B) Full speed graph. Stronger tags show greater speed change, but sufficiently strong tags show descent to second steady state as predicted by our math model .
Results cont.
We then graphed the t1/2 of each pdt against its degradation rate, and found that the resulting curve was in the shape of 1/degradation rate, which is the same form as the predicted log(2)/gamma (degradation rate). This suggests that the relationship between degradation rate and gene expression speed is similar to what we would predict. Please see our model page for information on how we attempt to fit this data to our mathmatical model.
Figure 3: Unguided (left) and guided (right) comparison of degradation rate (fluorescence of tag/fluorescence of untagged control) vs t1/2 (the point at which the regression reaches half of max). Each point represents one of the biological replicates from the above experiment. Blue line in graph is a guide for the eye, and does not represent a fit to the data.
Top
Contact us at igem@wm.edu
2017 William & Mary iGEM
