Team:William and Mary/Collaborations
Toggle navigation
Description
Project Overview
Demonstrate
Measurement
Results
Results Overview
Degradation
Speed Control
Readjustment
Dynamic Control
Proof of Concept
Enabling Teams
Modeling
Human Practices
Overview
Consult Community
Build Connections
Sustainable Impact
HP/Silver
Activities Booklet
Judging
Achievements
Medal Requirements
Parts
Composite Parts
Part Collection
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Background
This year our team has focused on creating a system that will allow other iGEM teams to easily control the gene expression speed of an arbitrary protein without requiring a costly wholesale redesign of existing architecture. To that end, we characterized how protein degradation tags (pdt) can be used to modularly change the speed of mScarlet-I, a fast folding, high quantum yield red fluorescent protein. We also created a set of easy-cloning parts which enable future iGEM teams to clone our parts without needing intermediates. At the UVA Mid-Atlantic meetup, we were particularly intrigued by University of Maryland iGEM’s copper sensor. After discussing with them, we agreed that we would collaborate on improving the function of their copper sensor ( Bba_K2477013 ). Since they primarily wanted to use their copper sensor as an educational tool, reducing the time required to get results would help them keep students engaged, as well as enable them to use their device in labs or outreach activities that did not have a lot of time.
To help UMD increase their gene expression speed, we proposed that we jointly create and characterize modified versions of their copper sensor parts with pdts (Figure 1). Since time was short and Maryland did not have access to a cell sorter, we simply had them ship us their parts, and then performed the modifications and characterizations at William and Mary, before sending the modified parts back for use. On the other end, William and Mary sent blinded speed constructs to Maryland for testing. We hoped that this would enable us to confirm degradation characterization in platforms other than our own, and potentially show speed change as well.
Figure 1: Schematic of University of Maryland's modified copper sensing part
Results
We successfully cloned and sequenced all 6 pdt variants of University of Maryland’s copper circuit ( K2333437 - K2333442 ). We confirmed that they are inducible with copper sulfate and that the RFP produced can be degraded by mf-Lon (Figure 2). We then performed preliminary speed characterization of a subset of the parts before sending them back to Maryland (Figure 3). We also analyzed Maryland’s data of our constructs and found that degradation was in fact occurring (Figure 4). Together these results show that not only can our system degrade in different strains and media conditions, but that it can degrade and change the speed of gene expression for arbitrary proteins. While we didn’t get to characterize the modified parts to the extent that we desired, we did see a noticeable and qualitatively similar speed change in our characterization. We hope that with more testing, we’ll be able to make these parts as consistent as our test parts.
We would also like to note that due to a judging form mishap, we did not fill in the Silver Medal parts requirement with any BioBrick IDs. However, since these parts serve as both new parts and functional proofs of concept, they would be sufficient to fulfill the Silver Medal requirement, we would humbly ask that judges evaluate these parts (among others), as proof of our fulfillment of Silver Medal requirement I. More information on this can be found on our for Judges page, as well as our other part related pages.
Degradation Functionality
Figure 2 shows the results from our plate reader functionality assay.  Cells were grown for 4 hours then induced with either 500µM or 1mM of CuSO4 (to induce copper sensing parts). After 2 hours cells were induced further with .01mM IPTG (to induce pLac mf-Lon). Introduction of mf-Lon clearly decreased the fluorescence of the cells, and based upon the results from this assay, we decided to perform speed tests on these constructs with 500µM CuSO4, and .01mM IPTG
Figure 2: Functional plate reader assay of Copper Sensor pdt parts, values are geometric means of the fluorescence/OD600 of three biological replicates, excitation/emission 584/612, shaded region represents one geometric std above and below mean
Preliminary Speed Tests
Figure 3 shows the results from our FACS speed tests, which were performed in the same manner as previously described . We were able to see a significant, but somewhat noisy speed change. While it was clear that parts with increased degradation rate were reaching steady state faster, it was fairly hard to distinguish them from one another. This is likely because we experienced cell death and toxicity issues with higher concentrations of copper sulfate (1mM lead to extreme cell death), and because we were unable to test parameters as thoroughly as we would have liked due to time concerns. Never the less, we feel that this data still shows that our work is generalizable, and usable in circuits under imperfect real-world conditions
Figure 3: Time course assay of Copper Sensor pdt parts, values are steady state normalized geometric means of of three biological replicates taken on the FL3 channel. Shaded region represents one geometric std above and below mean
Maryland's Tests
We also had University of Maryland confirm degradation of our pTet mScarlet-I constructs in their plate reader. They transformed each one of our blinded constructs into a DH5-alpha derivative along with our pLac mf-Lon construct, and then did fluorescence time course measurements of cells in LB in their plate reader. Since their plate reader could only take density measurements at the beginning and end of the time course, we analyzed the relative expression of our constructs (fluorescence/OD700) at the final measurement. When unblinded, we observed that all but one of the pdts that we sent them degraded as expected. The difference in the remaining construct could be due to differences in growth conditions (LB vs. M9 media), plate reader conditions, or growth phase of measurement (Figure 4).
Figure 4: Geometric mean of three biological replicates of each of the tested constructs.
Top
Contact us at igem@wm.edu
2017 William & Mary iGEM
