Team:William and Mary/Description
Toggle navigation
Description
Project Overview
Demonstrate
Measurement
Results
Results Overview
Degradation
Speed Control
Readjustment
Dynamic Control
Proof of Concept
Enabling Teams
Modeling
Human Practices
Overview
Consult Community
Build Connections
Sustainable Impact
HP/Silver
Activities Booklet
Judging
Achievements
Medal Requirements
Parts
Composite Parts
Part Collection
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Motivation
One of the main goals of synthetic biology is to create a modular genetic basis for the independent control of circuit behavior properties. Much progress has been made in achieving this aim for properties like gene expression strength (where well-characterized ribosome binding sites (RBSs) can be swapped within a genetic part), circuit architecture (where promoters can be swapped out to introduce connections and feedback architectures), and even gene expression noise (through a combination of the above two modulations). However, in order to move into the next phase of synthetic biology, we need to be able to control the dynamical properties of circuits— we want to move beyond circuits that focus on endpoint, steady-state values and explore the rich variety of dynamical systems. Fundamentally, gaining controlling of dynamical systems implies gaining control of temporal dynamics.
Currently, there is no good way to control the temporal dynamics of gene expression. Current control strategies require either a rewiring of the circuit architecture to achieve different time-dependent dynamics [1, 2] or a complete circumvention of transcriptional circuitry altogether, relying on post-translational dynamics like phosphorylation [3] or protein-protein interactions [4] to transmit information through a circuit. These approaches are often inaccessible to iGEM teams because they require too drastic an overhaul of existing circuit implementations. To alleviate this issue, and to enable future iGEM teams to create robust dynamical circuits, we created a protein degradation based ‘plug-and-play’ style system that allows modular and predictable control of the gene expression speed of a given circuit without requiring a fundamental redesign of existing circuit architecture.
Degradation Based Control of Gene Expression Speed
The relationship between the degradation rate of a gene's protein product and the response speed of its expression is a well-established concept in the theoretical biology community. Consider the following simple mathematical model presented by Uri Alon [5]. X is an inducible gene which, once activated, produces its protein product x at rate α. If x is degraded at rate γ, then we can use the following differential equation to track the concentration of protein x over time:
Figure 1: A simple kinetic model tracking the changing concentration of protein x over time.
Gene circuit component images adapted from Newcastle iGEM 2010; protein images adapted from Cameron and Collins 2014 [7].
We solve the above differential equation to obtain the following function, which gives the protein concentration at a given time t after activation of X:
Figure 2: Concentration of protein x as a function of time; the solution to the differential equation in Figure 1.
We are able to graph this function and determine that the steady state concentration for x is equal to production rate over degradation rate. Finally, defining т1/2 as the time it takes for x to reach half its steady-state concentration, we find the following:
Figure 3: We find that т1/2 is a function of γ, the degradation rate for protein x.
The above expression reveals that in this model, it is not only sufficient but also necessary to tune the degradation rate γ in order to control the gene expression speed (represented here as т1/2). This property remains true for more complex circuit architectures like feedforward loops, provided that the steady-state concentration is held constant [5].
Thus in order to create a plug-and-play style system to control gene expression speed, we developed and characterized a suite of BioBrick parts which allow for simple, modular and predictable changes to the gene expression speed of arbitrary proteins via protein degradation.
Orthogonal Degradation Tags
When we created our gene expression control system, we wanted to make sure that it was both usable across a variety of biological systems and circuits, and easily accessible to other iGEM teams. To this end, we chose to use the Mesoplasma florum Lon (mf-Lon) protease system which was characterized by Gur and Sauer in 2008 [6] and developed into a modular suite of genetic parts by Cameron and Collins in 2014 [7]. This system consists of of a AAA+ protease and its associated protein degradation tags (pdt), which operate in a mechanistically similar manner to the E. Coli endogenous protease ClpXP and its associated ssrA tags. However, unlike ClpXP and ssrA tags, mf-Lon and pdts are completely orthogonal to the endogenous protein degradation systems in E. coli. Using this orthogonal degradation system helps eliminate cross-talk between our system and endogenous E. coli proteins. Further, since there are pdts with a wide range of different affinities, we are able to tune degradation rate, and thus gene expression speed to a wide variety of values. This represents not only a practical advancement in the tuning of circuits, but also an important contribution in our theoretical understanding of biological processes, as to our knowledge there has so far been no direct experimental measurement in the literature of the predicted 1/γ scaling of a gene's speed with its degradation rate. See our results page for more details.
Modeling
Finally, by constructing a mechanistic model of our degradation system, we were able to rigorously analyze our timeseries datasets using Bayesian Parameter Inference and obtain parameter distributions (Figure 4). We then used this analysis to feed back into our predictive saturation model, which we used to develop the beginnings of a theoretical framework for understanding the relationship between protease-driven speed control and saturation effects on the protease. By combining our characterization measurements with our conceptual insights, future iGEM teams are now enabled to use the pdt system to predictably control the temporal dynamics of their genetic circuits. See our modeling page for details
Figure 4: MCMC parameter estimation for a simple protein production model correctly estimates the simulated values for both beta and gamma (15 and .03 respectively). In addition, the MCMC identifies a strong positive correlation between beta and gamma. This makes sense as steady-state levels of protein are set by the ratio Beta/Gamma. Therefore as beta goes up we should see a corresponding increase in gamma to best fit the model.
References
[1]  Nitzan Rosenfeld, Michael B Elowitz, and Uri Alon. Negative autoregulation speeds the response times of transcription networks. Journal of molecular biology, 323(5):785–793, 2002.
[2] Shmoolik Mangan and Uri Alon. Structure and function of the feed-forward loop network motif. Proceedings of the National Academy of Sciences, 100(21):11980–11985, 2003.
[3] Russell M Gordley, Reid E Williams, Caleb J Bashor, Jared E Toettcher, Shude Yan, and Wendell A Lim. Engineering dynamical control of cell fate switching using synthetic phospho- regulons. Proceedings of the National Academy of Sciences, 113(47):13528–13533, 2016.
[4] Anselm Levskaya, Orion D Weiner, Wendell A Lim, and Christopher A Voigt. Spatiotemporal control of cell signalling using a light-switchable protein interaction. Nature, 461(7266):997– 1001, 2009.
[5]  Uri Alon. An introduction to systems biology: design principles of biological circuits. CRC press, 2006
[6] Eyal Gur and Robert T Sauer. Evolution of the ssra degradation tag in mycoplasma: specificity switch to a different protease. Proceedings of the National Academy of Sciences, 105(42):16113– 16118, 2008.
[7] D Ewen Cameron and James J Collins. Tunable protein degradation in bacteria. Nature biotechnology, 32(12):1276–1281, 20
Top
Contact us at igem@wm.edu
2017 William & Mary iGEM
