Team:William and Mary/Measurement
Toggle navigation
Description
Project Overview
Demonstrate
Measurement
Results
Results Overview
Degradation
Speed Control
Readjustment
Dynamic Control
Proof of Concept
Enabling Teams
Modeling
Human Practices
Overview
Consult Community
Build Connections
Sustainable Impact
HP/Silver
Activities Booklet
Judging
Achievements
Medal Requirements
Parts
Composite Parts
Part Collection
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Measuring Temporal Dynamics
For our project this year we successfully designed and characterized an accessible and modular degradation-based system for the control of gene expression speed. Utilizing an E. coli orthogonal tmRNA degradation system consisting of a Mesoplasma florum Lon (mf-Lon) protease [1] and highly engineered tmRNA tags [2] with a range of protease affinities, we were able to engineer gene expression speed change driven by degradation rate. To do this we first had developed a time course measurement protocol that would allow robust and reproducible single cell measurements. Developing this method was time intensive, and meant that we spent a large portion of the summer without getting high-quality gene expression speed data, but ultimately our final time course protocol ensured that we got robust, reproducible data that we could feel confident in.
Beyond simply tracking gene expression over time to obtain basic speed measurements , we were able to successfully measure and exert control over more complex aspects of the temporal dynamics of gene expression. Informed by our mathematical model , we not only predicted but executed a functional incoherent feedforward loop enabling control over pulse-like behavior using our protein degradation system. To that end, we provided a thorough characterization of gene expression dynamics over time, comparing simultaneous induction to pre-induction of protease to yield qualitatively different results.
We believe that our team should be considered for this award not only for the quality of our characterization but also for our choice of what properties to measure. Rigorous measurements of the temporal aspects of genetic processes are overshadowed by an emphasis on the characterization of concentration- or strength-dependent features. This bias is so strong that even though the mf-Lon protein degradation tags have been available to the community since 2014, no direct experimental demonstrations of the relationship between genetic response speed and degradation rate existed in the synthetic biology literature before our project. By emphasizing the importance of temporal circuit dynamics and providing future iGEM teams with a modular, simple and predictable system to control and measure those dynamical properties by altering gene expression speed, our project helps to drive iGEM forward as a continual innovator in making the kinds of enabling measurements which open the door to new technologies. We also hope that this characterization can be used as a spring-board to do further single cell characterizations using our new inverted microscope. We've already refined our methodologies, and have taken some videos of our pTet mScarlet-I constructs. This should enable us to track single cells over a longer period of time, and account for heterogeneous circuit behavior in single cells.
Finally, we ensured throughout our projects that our graphs were designed to depict the underlying data in as genuine a way as possible. In particular, following the guidelines in [3], we chose specifically to replace all instances of bar graphs with univariate scatterplots, so that others would be able to explicitly see the inter-replicate variation in our measurements. Additionally, because single-cell fluorescence measurements are expected to be log-normally distributed [4], we chose to report our results using the geometric mean and geometric standard deviation in order to ensure that our statistical measures are consonant with our theoretical understanding of the processes which generated our observations.
References
[1] Eyal Gur and Robert T Sauer. Evolution of the ssra degradation tag in mycoplasma: specificity switch to a different protease. Proceedings of the National Academy of Sciences, 105(42):16113– 16118, 2008.
[2] D Ewen Cameron and James J Collins. Tunable protein degradation in bacteria. Nature biotechnology, 32(12):1276–1281, 20
[3] Weissgerber TL, Milic NM, Winham SJ, Garovic VD. Beyond bar and line graphs: time for a new data presentation paradigm. PLoS biology. 2015 Apr 22;13(4):e1002128.
[4] Beal J. Biochemical complexity drives log-normal variation in genetic expression. Engineering Biology. 2017 Jul 11;1(1):55-60.
Top
Contact us at igem@wm.edu
2017 William & Mary iGEM
