Team:William and Mary/Parts
Toggle navigation
Description
Project Overview
Demonstrate
Measurement
Results
Results Overview
Degradation
Speed Control
Readjustment
Dynamic Control
Proof of Concept
Enabling Teams
Modeling
Human Practices
Overview
Consult Community
Build Connections
Sustainable Impact
HP/Silver
Activities Booklet
Judging
Achievements
Medal Requirements
Parts
Composite Parts
Part Collection
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
This year we are proud to submit a series of mf-Lon/pdt based parts which allow modular control of gene expression speed via degradation. These parts enable future iGEM teams to easily and predictably tune the temporal and dynamical parameters of arbitrary circuits. Our submitted parts include cloning ready constructs containing codon optimized and standardized protein degradation tags so that our speed-control system can be seamlessly and easily incorporated into a user circuit of interest. We also include well-characterized inducible and constitutive reporter constructs with two different colors of fast folding reporter proteins to aide those seeking to expand or replicate our results. Additionally, every composite part submitted is flanked by the Unique Nucleotide Sequences (UNS’s) introduced to iGEM by W&M’s 2016 team to improve ease and reliability of cloning.
Judging note: We would like to note that due to a judging form mishap, we did not fill in the Silver Medal parts requirement with any BioBrick IDs. However, since any of our mScarlet, copper sensor, or mf-Lon parts could fulfill Silver Medal criteria I, we humbly ask that judges evaluate those parts, and/or the parts listed on our judging form for best part collection and best composite part as proof of our fulfillment of the criteria. More information on this can be found on our Medal Requirements page, as well as our other part related pages.
Cloning-Ready Protein Degradation Tags
In order to harness degradation to control gene expression speed, we utilize a suite of six E. coli orthogonal protein degradation tags (pdts) originally designed by Cameron and Collins in 2014 [1]. Each tag confers a distinct level of protein degradation; tags are lettered (A-F) in order of protease affinity, from strongest (highest degradation rate) to weakest. These parts have been made BioBrick compatible, codon-optimized for E. coli, and placed in between UNS sites, so that teams may easily insert them after their gene of choice using Gibson Assembly. They are also flanked by BsaI cut sites, for ease of use with Golden Gate Assembly, and contain sites other useful and well-characterized primer binding sites. This should allow future teams to efficiently adapt the speed-control system to their own parts and circuits, regardless of their preferred assembly method.
Part
Description
K2333401
Cloning Ready pdt A with Double Terminator
K2333402
Cloning Ready pdt B with Double Terminator
K2333403
Cloning Ready pdt C with Double Terminator
K2333404
Cloning Ready pdt D with Double Terminator
K2333405
Cloning Ready pdt E with Double Terminator
K2333406
Cloning Ready pdt F with Double Terminator
Inducible Tagged Reporters
We further include a collection of six aTc-inducible mScarlet-I reporter constructs tagged with each respective protein degradation tag (and a pdt-less inducible mScarlet-I control construct), all under the control of the pTet promoter. These parts, in combination with inducible mf-Lon protease constructs (listed below), allowed us to characterize the degradation properties of each protein degradation tag on a plasmid-based system. We successfully demonstrated distinct levels of protein degradation by each of the 6 pdt’s; see our full characterization here . This collection of parts was also used in our gene expression speed measurements, allowing us to control the initiation of reporter expression using the small molecule aTc. We used these constructs along with the IPTG-inducible mf-Lon protease (listed directly below) to demonstrate distinct levels of speed to steady state in reporter expression proportional to the relative strength of each pdt; the full results of these experiments are found here . We also took advantage of the inducible nature of these constructs by manipulating levels of aTc exposure in order to adjust final steady state values independently of speed control; the details of our readjustment experiments are found here . Once again, we also created analogous constructs for each of these parts, replacing mScarlet with sfGFP. These constructs are listed in the All Parts table at the bottom of this page.
Part
Description
K2333427
UNS pTet mScarlet-I
K2333428
UNS pTet mScarlet-I pdt A
K2333429
UNS pTet mScarlet-I pdt B
K2333430
UNS pTet mScarlet-I pdt C
K2333431
UNS pTet mScarlet-I pdt D
K2333432
UNS pTet mScarlet-I pdt E
K2333433
UNS pTet mScarlet-I pdt F
Constitutive Tagged Reporters
Our next collection of parts consist of a protein degradation tagged mScarlet reporter under the control of the strong constitutive promoter J23100. We also included a tagless control construct (J23100 mScarlet with no pdt) as a comparison. In order to demonstrate that our protein degradation tags operated similarly regardless of the tagged protein, we also built and characterized analogous constructs with an sfGFP reporter; these are listed below in the “All Parts” table. mScarlet and sfGFP reporters have been codon-optomized for E. coli and feature a double stop codon.
Part
Description
K2333413
J23100 mScarlet-I
K2333414
J23100 mScarlet-I pdt A
K2333415
J23100 mScarlet-I pdt B
K2333416
J23100 mScarlet-I pdt C
K2333417
J23100 mScarlet-I pdt D
K2333418
J23100 mScarlet-I pdt E
K2333419
J23100 mScarlet-I pdt F
Inducible mf-Lon Protease
Our gene expression speed control system features the E. coli-orthogonal mf-Lon protease originally characterized by Gur and Sauer in 2008 [2], which specifically targets the above protein degradation tags with varying affinities corresponding to varying degradation rates. We have modified the mf-Lon gene via codon-optimization for iGEM use and added a double terminator (for details see Bba_K2333011 ). We have submitted mf-Lon constructs that are inducible by IPTG and arabinose, respectively. We used the IPTG-inducible mf-Lon construct in tandem with the above aTc-inducible pdt reporter constructs to obtain gene expression speed measurements--these results can be found here ).
Part
Description
K2333434
pLac0-1 mf-Lon
K2333435
pBad mf-Lon
Degradation-tagged Copper Sensor
As part of our collaboration with the University of Maryland iGEM team, we built six additional constructs incorporating our protein degradation tags onto their CueR-based copper sensor (For reference, the copper biosensor by UMD without pdts is listed on the registry as Bba_K2477013 ). These parts allowed us to demonstrate persistence of speed-change effects for protein outputs beyond simple reporter proteins, and provide an example of a practical application of our speed-control system to improve biosensor output speed. The details of these experiments can be found here .
Part
Description
K2333437
Copper Sensor pdt A
K2333438
Copper Sensor pdt B
K2333439
Copper Sensor pdt C
K2333440
Copper Sensor pdt D
K2333441
Copper Sensor pdt E
K2333442
Copper Sensor pdt F
All Submitted Parts
Part
Description
K2333401
Cloning Ready pdt A with Double Terminator
K2333402
Cloning Ready pdt B with Double Terminator
K2333403
Cloning Ready pdt C with Double Terminator
K2333404
Cloning Ready pdt D with Double Terminator
K2333405
Cloning Ready pdt E with Double Terminator
K2333406
Cloning Ready pdt F with Double Terminator
K2333407
UNS J23100 sfGFP pdt A
K2333408
UNS J23100 sfGFP pdt B
K2333409
UNS J23100 sfGFP pdt C
K2333410
UNS J23100 sfGFP pdt D
K2333411
UNS J23100 sfGFP pdt E
K2333412
UNS J23100 sfGFP pdt F
K2333413
J23100 mScarlet-I
K2333414
J23100 mScarlet-I pdt A
K2333415
J23100 mScarlet-I pdt B
K2333416
J23100 mScarlet-I pdt C
K2333417
J23100 mScarlet-I pdt D
K2333418
J23100 mScarlet-I pdt E
K2333419
J23100 mScarlet-I pdt F
K2333420
UNS pTet sfGFP
K2333421
UNS pTet sfGFP pdt A
K2333422
UNS pTet sfGFP pdt B
K2333423
UNS pTet sfGFP pdt C
K2333424
UNS pTet sfGFP pdt D
K2333425
UNS pTet sfGFP pdt E
K2333426
UNS pTet sfGFP pdt F
K2333427
UNS pTet mScarlet-I
K2333428
UNS pTet mScarlet-I pdt A
K2333429
UNS pTet mScarlet-I pdt B
K2333430
UNS pTet mScarlet-I pdt C
K2333431
UNS pTet mScarlet-I pdt D
K2333432
UNS pTet mScarlet-I pdt E
K2333433
UNS pTet mScarlet-I pdt F
K2333434
pLac0-1 mf-Lon
K2333435
pBad mf-Lon
K2333437
Copper Sensor pdt A
K2333438
Copper Sensor pdt B
K2333439
Copper Sensor pdt C
K2333440
Copper Sensor pdt D
K2333441
Copper Sensor pdt E
K2333442
Copper Sensor pdt F
References
[1] D Ewen Cameron and James J Collins. Tunable protein degradation in bacteria. Nature biotechnology, 32(12):1276–1281, 2014.
[2] Eyal Gur and Robert T Sauer. Evolution of the ssra degradation tag in mycoplasma: specificity switch to a different protease. Proceedings of the National Academy of Sciences, 105(42):16113– 16118, 2008.
Top
Contact us at igem@wm.edu
2017 William & Mary iGEM
