Team:William and Mary/Results
Toggle navigation
Description
Project Overview
Demonstrate
Measurement
Results
Results Overview
Degradation
Speed Control
Readjustment
Dynamic Control
Proof of Concept
Enabling Teams
Modeling
Human Practices
Overview
Consult Community
Build Connections
Sustainable Impact
HP/Silver
Activities Booklet
Judging
Achievements
Medal Requirements
Parts
Composite Parts
Part Collection
Interlab
Team
Meet the Team
Attributions
Notebook
Protocols
Safety
Degradation Rates
When Cameron and Collins demonstrated the functionality of protein degradation tags (pdt) and the Mesoplasma florum Lon (mf-Lon) protease in E. coli in 2014, they did their work exclusively using genomically integrated constructs. However, since most iGEM teams work mainly with plasmid constructs, we first wanted to confirm and characterize the parts using iGEM backbones. To do this we assembled constitutive and aTc inducible constructs carrying the fast folding red fluorescent protein mScarlet-I, tagged with each of our six different pdts, or left untagged as a control. Further, to ensure that our project will work with a variety of different proteins, we made identical constructs encoding for superfolder GFP (sfGFP) and performed preliminary characterization (Figure 2).
Figure 1: Schematic of a generic reporter construct used to test degradation rates. Analogous constructs were built with sfGFP reporters.
Figure 2: Degradation rates measured in the above constructs. Each data point represents the population geometric mean of at least 10,000 cells of a distinct biological replicate. Degradation was calculated relative to the geometric mean fluorescence of the untagged control at the final point of a 200 minute time course.
Speed Control
Once we confirmed that degradation was working reliably, and that our tags had a variety of different strength tags, we next tested whether we had control over gene expression speed. Using the ATC inducible mScarlet-I constructs from the previous section, we confirmed that we could change the gene expression speed of our constructs (Figure 3a). Further, if you plot the gene expression speed of each construct against its degradation rate then the shape of the curve resembles that of a constant/degradation, just as we would expect from our model (Figure 3B). Together, this data represents the first experimental confirmation of the relationship between gene expression speed and degradation rate.
Figure 3: A. Measurements of gene expression normalized to steady state using our ATC inducible mScarlet-I constructs. Data is shown for each construct until steady state is reached (at least two consecutive subsequent data points do not increase fluorescence). Geometric mean of 10,000 cells each of three biological replicates. Shaded region represents one geometric standard deviation above and below the mean.
B. Comparison of calculated t1/2 vs degradation rate. Degradation rate was obtained as above, and t1/2 was defined as time at which each biological replicate's regression line reached half of steady state. The blue line represents an optical guide for the eye, and is not fitted.
Preserving Steady State Protein Concentration
While we have demonstrated a change in gene expression speed, recall that the steady state value for protein concentration is given as the production rate divided by the degradation rate. This means that as we increase the speed of gene expression, we are also decreasing the steady-state value. While some applications of genetic circuits may only be concerned with a gene’s expression as an on or off signal, we wanted our system to affect speed while maintaining the original steady state protein concentration.
According to our model, gene expression speed is only regulated by degradation. This implies that it should be possible to readjust our steady-state value back up to its original expression level by manipulating protein production rate, without affecting the associated speed change. Using pdt E as an example, we measured the time to steady state with and without mf-Lon at a given ATC induction level. We then showed that by increasing the ATC concentration (increasing production rate), we can return the steady state of the with-protease condition to that of the without-protease condition while maintaining the same speed change, exactly as our model predicts.
Figure 3: Measurements of the fluorescence (A) or the steady state normalized fluorescence (B) over time of BBa_K2333432 (pTet mScarlet-I pdt E), induced at 50ng/mL ATC with and without mf-Lon, and readjusted at 85ng/mL ATC with mf-Lon. Each data point represents the geometric mean of three biological replicates, with at least 10,000 cells collected for each replicate. Shaded region represents +/- geometric standard deviation. Reporter constructs were used on pSB1C3 while a pSB3K3 version of BBa_K2333434 (pLac mf-Lon) was used.
Achieving Dynamical Control
Our measurements have demonstrated that the pdt system can be used to predictably control the speed of a given gene’s expression. We then wanted to use this control over speed to obtain control over the temporal dynamics of a circuit. One of the simplest examples of a dynamical circuit is the incoherent feed forward loop (IFFL), which consists of three proteins X, Y, and Z which regulate each other such that X activates Y and Z, and Y represses Z. This circuit architecture can generate a pulsatile response upon activation of X (Figure 6).
Figure 6: Schematic of an Incoherent Feedforward Loop (IFFL) circuit. X, Y, and Z represent arbitrary transcription factors. At t=0, the production of X is activated. In region (1), the molecules of X activate the production of both Z and Y. However, the concentration of Y is insufficient to significantly repress the production of Z, so its concentration increases. In region (2), the concentration of Y has grown sufficiently large that it now exerts a significant repressive effect on the production of Z. This causes the concentration of Z to decrease. In region (3), the repressive effect from Y and the activating effect from X have balanced and the circuit has reached its steady state.
An important dynamical property of this circuit is the pulse sharpness. There are two ways to tune this property: (1) By increasing the strength of Y -| Z, you make the damping to the lower steady state occur faster, which means the pulse is narrower. (2) By increasing the speed of X->Z, you make the rise-time to the peak faster, which makes the pulse taller. Both properties lead to sharper pulses. We realized that we would be able to construct a minimal IFFL circuit by using Lon’s proteolytic degradation of a tagged protein as the inhibition step in the circuit.
While we can schematize the Lon-based circuit to classify as an IFFL, the fact that one of the nodes in the circuit is a small-molecule inducer and that the inhibition of mScarlet by Lon is a post-translational degradation process rather than an actual inhibition of transcription, it is not immediately intuitive that our Lon-based circuit would indeed function as a canonical IFFL and generate a pulse. In order to address this issue, we simulated an ODE model of the Lon-based circuit over a wide range of parameters, and determined that the system can indeed exhibit pulsatile behavior representative of a canonical IFFL circuit (Figure 7).
Figure 7: Some example timecourses from our ODE model of the Lon-based IFFL circuit. Our circuit is able to generate pulses of various sharpness, as well as more standard relaxation responses, all of which correspond to the behavior of a canonical IFFL. The simulations were performed by taking our protease complexing model [see Modeling section] and setting the initial conditions so that Lon and the reporter are activated simultaneously.
We next moved to experimentally validate whether our Lon-based IFFL can generate measurable pulses in experimental conditions. We measured mScarlet-I pdtA for a 200-minute timecourse, with the Lon either induced simultaneously with the mScarlet-I pdtA or induced beforehand, allowing its concentration to reach steady state before mScarlet-I pdtA becomes activated. We should expect to see that the simultaneous induction condition will exhibit a pulse-like response, while the pre-induced Lon condition will not. Indeed, we observe that this is the case (Figure 8).
Figure 8: Time course measurement of pTet mScarlet-I pdtA induction with Lon already at steady state or induced at the same time. Geometric mean of three biological replicates, shaded region represents one geometric standard deviation above and below the mean, at least 10,000 cells collected per time point.
Proof of Concept
After demonstrating the functionality of our system using basic reporter constructs, we wanted to exhibit an example of a practical application for speed control. To that end, we collaborated with the University of Maryland iGEM team by creating and characterizing the speed of pdt-tagged versions of their copper detecting circuit. We then sent the tagged constructs back to them for future use. Though we were unable to perform the full range of characterization that we wanted due to time concerns, we found that we were able to achieve an increase in speed similar to that of our simple reporter circuits. This serves as a practical proof of concept, as we were successfully able to show that our ready-cloning parts could be used to increase the speed of an arbitrary genetic circuit.
Figure 5:  Normalized fluorescence of three different pdt modified versions of copper parts, contrasted against a no Lon control. Each data point represents the geometric mean of 10,000+ cells from each of three biological replicates. Shaded region represents one geometric standard deviation above and below the mean. Reporter constructs were used on 1C3 while a 3K3 version of BBa_K2333434 (pLac mf-Lon) was used. Note that the no Lon control did not reach steady state, so its true expression speed may be even slower
Enabling Future iGEM Teams
Once we felt that we could understand and control gene expression speed, we next wanted to make our system more accessible to future iGEM teams. While our system is inherently easy to clone and implement, as it only consists of only a 27 amino acid residue pdt and the associated mf-Lon protease, we wanted to make it even easier to implement. With this in mind, we created a suite of ready-to-clone pdt constructs and added them to the registry. Each part contains one of our six different strength E. coli-optimized protein degradation tags with a double stop codon and a double terminator. Combining all these parts together into one construct prevents extra cloning steps, saving time, money and aggravation. In addition to the functional elements above, each construct also contains two BsaI restriction sites for Golden Gate Assembly, two Universal Nucleotide Sequences for Gibson Assembly, as well as a number of well-tested primer sequences that can be used for any other type of cloning. We also made it easy to swap and design large libraries of constructs with different speeds, by making sure that the only difference between each ready-cloning construct was a small unique region in the pdt. That means there is no need to switch primers to use a different strength pdt. Alongside our well-characterized construct Bba_K2333434 (pLac mf-Lon), these ready-to-clone parts should make it cheap and easy for future teams to test their constructs with a wide variety of different gene expression speeds, either by changing the pdt or the concentration of mf-Lon.
Figure 4: Schematic of generic cloning ready part. UNS sites can be used for easy cloning and backbone transfers, while BsaI sites enable Golden Gate Assembly
Top
Contact us at igem@wm.edu
2017 William & Mary iGEM
