<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNDS_China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNDS China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu"><DIV class="menu_item submenu" id="team"><A class="mt">Team</A><UL class="sublist"><LI id="tm"><A href="https://2017.igem.org/Team:BNDS_China/Team">Team</A></LI><LI id="co" style="border:none"><A href="https://2017.igem.org/Team:BNDS_China/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="menu_item submenu" id="project"><A class="mt">Project</A><UL class="sublist"><LI id="dc"><A href="https://2017.igem.org/Team:BNDS_China/Description">Description</A></LI><LI id="ds"><A href="https://2017.igem.org/Team:BNDS_China/Design">Design</A></LI><LI id="ex"><A href="https://2017.igem.org/Team:BNDS_China/Experiments">Experiment</A></LI><LI id="il"><A href="https://2017.igem.org/Team:BNDS_China/InterLab">InterLab</A></LI><LI id="dm"><A href="https://2017.igem.org/Team:BNDS_China/Demonstrate">Demonstrate</A></LI><LI id="at" style="border:none"><A href="https://2017.igem.org/Team:BNDS_China/Attributions">Attributions</A></LI></UL></DIV><DIV class="menu_item submenu" id="parts"><A href="https://2017.igem.org/Team:BNDS_China/Parts" class="mt">Parts</A></DIV><DIV class="menu_item nosubmenu" id="safety"><A href="https://2017.igem.org/Team:BNDS_China/Safety" class="mt">Safety</A></DIV><DIV class="menu_item submenu" id="hprac"><A class="mt">H.Prac</A><UL class="sublist"><LI id="sh"><A href="https://2017.igem.org/Team:BNDS_China/HP/Silver">Silver HP</A></LI><LI id="ig"><A href="https://2017.igem.org/Team:BNDS_China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI id="pe" style="border:none"><A href="https://2017.igem.org/Team:BNDS_China/Engagement">Public Engagement</A></LI></UL></DIV><DIV class="menu_item submenu" id="awards"><A href="https://2017.igem.org/Team:BNDS_China/Model" class="mt">Awards</A></DIV><DIV class="menu_item nosubmenu" id="judge"><A href="https://igem.org/2017_Judging_Form?team=BNDS_China" class="mt">Judge</A></DIV></DIV><DIV class="wrapper"><DIV class="block_white"><H3 style="padding-top: 6%;">Experiments</H3></DIV><DIV class="block_grey" style="margin-top: 0px; height: 8500px;"><H3>Polymerase Chain Reaction (PCR)</H3><P>We used Q5 DNA polymerase purchased form NEB for PCR. Below are the different groups and primers we used for amplifying <I>gadR, gadC, gadA</I>, and GFP insert sequences.</P><TABLE border="1"><TBODY><TR><TH><H5>Bacterial Strains</H5></TH></TR><TR><TH>Species</TH><TH>Strains</TH><TH>Description</TH></TR><TR><TD><I>Escherichia coli</I></TD><TD>DH5α</TD><TD>Cloning host</TD></TR><TR><TD><I>Escherichia coli</I></TD><TD>Nissle</TD><TD>Expression host</TD></TR><TR><TH><H5>Plasmids</H5></TH></TR><TR><TH>ID</TH><TH>Antibiotic resistance</TH><TH>Description</TH></TR><TR><TD>Repressor Generator (RPG)</TD><TD>Ampicillin</TD><TD>Cloning &amp; Expression vector</TD></TR><TR><TD>RPG-LacI-Ptac-RBS-<I>gadR</I>-RBS-<I>gadC</I>-RBS-<I>gadA</I>-terminator</TD><TD>Ampicillin</TD><TD>Expression plasmid carrying <I>LacI, gadR, gadC, gadA</I> from <I>Lactobacillus brevis</I> NCL912 and pTac from original RPG</TD></TR><TR><TD>RPG-LacI-Ptac-B0034-gadA-B0034-E0040-terminator</TD><TD>Ampicillin</TD><TD>Expression plasmid carrying LacI, gadA from <I>Lactobacillus brevis</I> NCL912 and pTac from original RPG. With super ribosomal binding site before <I>gadA</I> and GFP (Green Fluorescent Protein), respectively.</TD></TR><TR><TD>RPG-LacI-Ptac-B0034-<I>gadA</I>-Linker-Histag-terminator</TD><TD>Ampicillin</TD><TD>Expression plasmid with Overexpression of <I>gadA</I> with super ribosomal binding side under the control of a lactose inducible promoter from <I>Lactobacillus brevis</I> NCL912 and intending to exam expression of <I>gadA</I> using integrated polyhistidine-tag.</TD></TR><TR><TD>LacI-Ptac-B0034-<I>gadA</I>-Linker-E0040-terminator</TD><TD>Ampicillin</TD><TD>Expression plasmid with Overexpression of <I>gadA</I> with super ribosomal bonding side controlling with lactose inducible promoter and intending to exam expression of <I>gadA</I> using integrated fluorescent protein.</TD></TR></TBODY></TABLE><TABLE border="1"><TBODY><TR><TH><H5>Primers</H5></TH></TR><TR><TH>Primer ID</TH><TH>Sequences (5’ to 3’)</TH><TH>Amplification  Size (bp)</TH><TH>Target Genes</TH></TR><TR><TD>ecRBS-1-<I>gadR</I>-F</TD><TD>ACCATAGATAAGAGTTTAATCGCAGatggaatcaagaacgaaggacattct</TD><TD>644</TD><TD><I>gadR</I></TD></TR><TR><TD>ecRBS-1-<I>gadR</I>-R</TD><TD>ccacTCTCTAATGTCCTCGCCGTGACGTttagacagataatacttgttgtttaaataa</TD></TR><TR><TD>ecRBS-1-<I>gadC</I>-F</TD><TD>ACGTCACGGCGAGGACATTAGAGAgtggatgaaaataagtctggacagcagattgatac</TD><TD>1554</TD><TD><I>gadC</I></TD></TR><TR><TD>ecRBS-1-<I>gadC</I>-R</TD><TD>TACCTGTTACCGTCGTAATGTGCGttacttagtttccttttccaatgattcatccg</TD></TR><TR><TD>ecRBS-13-<I>gadA</I>-F</TD><TD>CGCACATTACGACGGTAACAGGTAatgactaaacaggatcaggaaacgca</TD><TD>1486</TD><TD><I>gadA</I></TD></TR><TR><TD>RPG-<I>gadA</I>-R</TD><TD>ccattcgccattcaggttagttgtgaactgtcgtcttgttatcagtatccg</TD></TR><TR><TD>RPG-<I>gadA</I>-F</TD><TD>gttcacaactaacctgaatggcgaatggtaaGGTCTC</TD><TD>4882</TD><TD>RPG vector for <I>gadRCA</I></TD></TR><TR><TD>ecRBS-1-RPG-R</TD><TD>ccatCTGCGATTAAACTCTTATCTATGGTATTATTTGTAGAGGCTGTTTCGTCCTCAC</TD></TR><TR><TD>ptac-<I>gadA</I>-F</TD><TD>acaattaaagaggagaaatactagatgactaaacaggatcaggaaacgcag</TD><TD>1481</TD><TD><I>gadA</I></TD></TR><TR><TD><I>gadA</I>-GFP-R</TD><TD>CCACCGCCACCttagttgtgaactgtcgtcttgttatcagtatccg</TD></TR><TR><TD><I>gadA</I>-GFP-F</TD><TD>cacaactaaGGTGGCGGTGGCtctatgcgtaaaggagaagaacttttcact</TD><TD>1043</TD><TD>GFP</TD></TR><TR><TD>GFP-RPG-R</TD><TD>ttgatgcctggACTAGTAGttattatttgtatagttcatccatgccatgtgtaatcc</TD></TR><TR><TD>RPG-GFP-F</TD><TD>taaCTACTAGTccaggcatcaaataaaacgaaaggctca</TD><TD>4847</TD><TD>RPG for old GFP construction</TD></TR><TR><TD>ptac-<I>gadA</I>-R</TD><TD>ctagtatttctcctctttaattgtgagcgctcacaattccacaca</TD></TR><TR><TD><I>gadA</I>-RPG-F</TD><TD>ctCATCATCATCATCATCACtaaCTACTAGTccaggcatcaaataaaacgaaaggctca</TD><TD>4791</TD><TD>RPG for His-tag construction</TD></TR><TR><TD>ptac-<I>gadA</I>-R</TD><TD>ctagtatttctcctctttaattgtgagcgctcacaattccacaca</TD></TR><TR><TD>ptac-<I>gadA</I>-F</TD><TD>acaattaaagaggagaaatactagatgactaaacaggatcaggaaacgcag</TD><TD>1500</TD><TD><I>gadA</I></TD></TR><TR><TD><I>gadA</I>-RPG-R</TD><TD>gTGATGATGATGATGATGagaGCCACCGCCACCgttgtgaactgtcgtcttgttatcagtatccg</TD></TR></TBODY></TABLE><H5>Q5 PCR Reaction system (50 μL) for <I>gadR/gadC/gadA</I>/GFP:</H5><OL type="A"><LI>Q5 Reaction Buffer (5x): 10 μL</LI><LI>dNTP: 1 μL</LI><LI>Forward Primer/ Reverse Primer :2.5 μL/2.5 μL</LI><LI>Template: 45ng</LI><LI>Q5 DNA Polymerase: 0.5 μL</LI><LI>ddh2O：33 μL</LI></OL><H5>Q5 PCR Reaction system (50 μL) for the vector RPG</H5><OL type="A"><LI>Q5 Reaction Buffer (5x): 10 μL</LI><LI>dNTP: 3 μL</LI><LI>Forward Primer/ Reverse Primer :2.5 μL/2.5 μL</LI><LI>Template (RPG): 45ng</LI><LI>Q5 DNA Polymerase: 0.5 μL</LI><LI>ddh2O：31 μL</LI></OL><H3>Gibson Assembly</H3><P>Since we had many segments, the success rate for Gibson assembly of 4 segments directly would be relatively low. Hence we chose to run Gibson Assembly twice. First, we conducted Gibson Assembly toward two groups of two segments. Then, we mixed the two Gibson Assembly reaction systems together.</P><TABLE border="1"><TBODY><TR><TH>Group</TH><TH>Assembly 1</TH><TH>Assembly 2</TH></TR><TR><TD>RPG-<I>gadR-gadC-gadA</I></TD><TD>RPG for RCA and gadR</TD><TD><I>gadC</I> and <I>gadA</I></TD></TR><TR><TD>RPG-A-GFP (old)</TD><TD>RPG for GFP (old)</TD><TD><I>gadA</I> and GFP</TD></TR><TR><TD>RPG-A-linker-His tag</TD><TD>RPG and <I>gadA</I> with His-tag</TD><TD>N/A</TD></TR><TR><TD>RPG-A-linker-GFP</TD><TD>RPG-1 and GFP</TD><TD>RPG-2 and <I>gadA</I></TD></TR></TBODY></TABLE><H3>Gibson Products Transformation</H3><P>We transformed our Gibson products into competent E. coli DH5α cells.</P><H5>Transformation system:</H5><P>Positive control: 25 μL competent cell + 1 μL RPG</P><P>Negative control: 25 μL competent cell</P><P>Experiment group (with Gibson products): 50 μL competent cells + 2 μL Gibson products</P><H3>Colony PCR</H3><P>Gibson Assembly might cause a relatively high probability of false positive. In order to check whether the colony incorporated the targeted plasmid we expected, we ran colony PCR and used UV gel imaging to observe PCR product bands and then ran sequencing.</P><OL type="A"><LI>Pick 6-10 colonies.</LI><LI>Add 20 μL Amp+ liquid LB broth into 0.2 mL microfuge tube and add colonies, respectively. Then use 37°C shaker (with rotate speed 270r/min) to revive the bacteria for 1 hour.</LI></OL><H5>Colony PCR Reaction System:</H5><OL type="A"><LI>Taq Mix: 12.5 μL</LI><LI>Forward Primer/ Reverse Primer: 1 μL/1 μL</LI><LI>Bacteria LB solution: 2 μL</LI><LI>ddh2O: 10.5 μL</LI></OL><H3>Plasmid Extraction</H3><P>As the sequencing results were confirmed after comparing with the constructed plasmid, we in vitro cultured <I>E. coli</I> for 12 hours, then used the protocol of Axygen Mini-prepare Plasmid Extraction Kit to extract the RPG-<I>gadR-gadC-gadA</I> plasmid.</P><H3>Plasmid Transformation</H3><P>Since <I>E. coli</I> Nissle 1917 was not a conventionally used strain for synthetic biology, we decided to transform the desired plasmid instead of Gibson products into <I>E. coli</I> Nissle.</P><H3>IPTG Induction for the First and Second Construction</H3><P>We used the overnight (12 hours) incubated bacteria LB solution, diluted them 200x into Amp+ LB/M9 with 0.5 mM and 1mM IPTG, MSG (our substrate: sodium glutamate, 6 mL system), then we incubated them for 3 hours. After the incubation of 3 hours, <I>E. coli</I> Nissle 1917 would reach the early point of its exponential growth period. We then diluted the bacteria solution again 200x into Amp+ LB/M9 with 0.5 mM and 1mM IPTG, MSG(6 mL system) respectively and incubated 12 hours.</P><H3>IPTG Induction for the Third Construction</H3><P>The system and procedures were similar to that of IPTG Induction for the First and Second construction, instead we used only a concentration of 1mM IPTG to induce bacteria. And since we planned to do the reaction between MSG and GAD outside our bacteria, we did not add MSG through the induction process.  We used incubation shaker to incubate two 5mL bacteria LB solution for 8 hours and we transform the solution to 1L A+ LB solution. Then we incubated the bacteria for another 3 hours in order to allow bacteria entering exponential growth stage. Eventually, we added IPTG with the final concentration of 1mM and induced overnight.</P><H3>Plate Reader Measurement for GFP Constructions</H3><P>We measured the concentration of GFP using a Plate Reader to estimate the expression of <I>gadA</I>.</P><H3>Protein treatment for the first construction</H3><P>We first added lysis buffer (pH=7.8) and PMSF (Phenylmethanesulfonyl fluoride) protease inhibitor to avoid enzymes in lysosome degrading our target protein - GAD enzyme. Then we used ultrasound cell disrupter to lyse our bacteria. After that, we ran SDS-PAGE to observe generally if there were correct protein bands at 55 kDa (The length of GAD is 54.8).</P><H3>Protein treatment for the second construction</H3><P>We did not treat the induced bacteria solution, and we measured GABA directly. (See details in “GABA Measurement”)</P><H3>Protein Purification for the third construction 1.1 (His-tag)</H3><P>We first added lysis buffer (0.2M Sodium acetate-Acetic acid Buffer, pH=4.6) and PMSF (Phenylmethanesulfonyl fluoride) protease inhibitor to avoid enzymes in lysosome degrading our target protein --- GAD enzyme. Then we used ultrasound cell disrupter to lyse our bacteria. After that, we suspend the bacteria solution on the nickel column, which would allow proteins to suspend on the column but not other molecules  due to the 6x His tag we have added after gad A. Then we used imidazole to undergo competitive elution in order to elute our protein. After that, we used G250 (A type of Coomassie brilliant blue) to test whether the protein had been suspended on the column. At last, we ran SDS-PAGE to observe generally if there were correct protein bands at 55 kDa (The length of GAD is 54.8).</P><H3>GAD enzyme Extraction for the third construction 1.2 (His-tag)</H3><P>We have found another cofactor of the reaction of MSG and GAD --- Vitamin B, and direct protein purification process will eliminate not only gab T (see wiki-demonstration and project description) but also the cofactor Vitamin B. Meanwhile, the direct enzyme extraction will stop TCA cycle and the function of gab T which might influence the existence of GABA. So, we first added PMSF and then we used ultrasound (20 min, P=200W) to lyse bacteria directly and use the enzyme extraction solution to react with MSG directly.</P><H3>The Reaction of the Third Construction 1.1</H3><P>According to the result derived from our mathematic model, we prepared 3 acetic acid-sodium acetate buffers with pH=4.2, 4.4, 4.6 respectively. Then we added 2% MSG into the buffer and 800 μg GAD solution into the reaction system (20mL). After that, we incubated the reaction system with 100r/min and 37°C shaker for 3 hours.</P><H3>GABA Measurement for the first and second construction</H3><P>We measured the induced bacteria LB solution directly via AAA (Amino Acid Analyzer). Before the measurement, we ran the GABA standard curve with pure GABA (concentration of 50, 100ppm and 200ppm, filtered before adding samples). After that, we used the ultrasound with ice to lyse the bacteria, then centrifuged products for 10 minutes with 4 °C. We diluted samples and transferred the solutions into the vial afterward.</P><H3>GABA Measurement for the third construction</H3><P>We measured the induced bacteria LB solution directly via AAA (Amino Acid Analyzer). Before the measurement, we ran the GABA standard curve with pure GABA (concentration of 50, 100ppm and 200ppm, filtered before adding samples). Meanwhile, we diluted all our samples 12x with ddh2O and filter with 0.2 nm filter membrane.</P></DIV></DIV><FOOTER><P style="margin-left: 20px; padding-top:15px; font-size: 20px;">© 2017 BNDS_China All Rights Reserved.</P></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>