Team:BNDS China/Experiments
Team
Team
Collaborations
Project
Description
Design
Experiment
InterLab
Demonstrate
Attributions
Parts
Safety
H.Prac
Silver HP
Integrated and Gold
Public Engagement
Awards
Judge
Experiments
Polymerase Chain Reaction (PCR)
We used Q5 DNA polymerase purchased form NEB for PCR. Below are the different groups and primers we used for amplifying gadR, gadC, gadA, and GFP insert sequences.
Bacterial Strains
Species
Strains
Description
Escherichia coli
DH5α
Cloning host
Escherichia coli
Nissle
Expression host
Plasmids
ID
Antibiotic resistance
Description
Repressor Generator (RPG)
Ampicillin
Cloning & Expression vector
RPG-LacI-Ptac-RBS-gadR-RBS-gadC-RBS-gadA-terminator
Ampicillin
Expression plasmid carrying LacI, gadR, gadC, gadA from Lactobacillus brevis NCL912 and pTac from original RPG
RPG-LacI-Ptac-B0034-gadA-B0034-E0040-terminator
Ampicillin
Expression plasmid carrying LacI, gadA from Lactobacillus brevis NCL912 and pTac from original RPG. With super ribosomal binding site before gadA and GFP (Green Fluorescent Protein), respectively.
RPG-LacI-Ptac-B0034-gadA-Linker-Histag-terminator
Ampicillin
Expression plasmid with Overexpression of gadA with super ribosomal binding side under the control of a lactose inducible promoter from Lactobacillus brevis NCL912 and intending to exam expression of gadA using integrated polyhistidine-tag.
LacI-Ptac-B0034-gadA-Linker-E0040-terminator
Ampicillin
Expression plasmid with Overexpression of gadA with super ribosomal bonding side controlling with lactose inducible promoter and intending to exam expression of gadA using integrated fluorescent protein.
Primers
Primer ID
Sequences (5’ to 3’)
Amplification  Size (bp)
Target Genes
ecRBS-1-gadR-F
ACCATAGATAAGAGTTTAATCGCAGatggaatcaagaacgaaggacattct
644
gadR
ecRBS-1-gadR-R
ccacTCTCTAATGTCCTCGCCGTGACGTttagacagataatacttgttgtttaaataa
ecRBS-1-gadC-F
ACGTCACGGCGAGGACATTAGAGAgtggatgaaaataagtctggacagcagattgatac
1554
gadC
ecRBS-1-gadC-R
TACCTGTTACCGTCGTAATGTGCGttacttagtttccttttccaatgattcatccg
ecRBS-13-gadA-F
CGCACATTACGACGGTAACAGGTAatgactaaacaggatcaggaaacgca
1486
gadA
RPG-gadA-R
ccattcgccattcaggttagttgtgaactgtcgtcttgttatcagtatccg
RPG-gadA-F
gttcacaactaacctgaatggcgaatggtaaGGTCTC
4882
RPG vector for gadRCA
ecRBS-1-RPG-R
ccatCTGCGATTAAACTCTTATCTATGGTATTATTTGTAGAGGCTGTTTCGTCCTCAC
ptac-gadA-F
acaattaaagaggagaaatactagatgactaaacaggatcaggaaacgcag
1481
gadA
gadA-GFP-R
CCACCGCCACCttagttgtgaactgtcgtcttgttatcagtatccg
gadA-GFP-F
cacaactaaGGTGGCGGTGGCtctatgcgtaaaggagaagaacttttcact
1043
GFP
GFP-RPG-R
ttgatgcctggACTAGTAGttattatttgtatagttcatccatgccatgtgtaatcc
RPG-GFP-F
taaCTACTAGTccaggcatcaaataaaacgaaaggctca
4847
RPG for old GFP construction
ptac-gadA-R
ctagtatttctcctctttaattgtgagcgctcacaattccacaca
gadA-RPG-F
ctCATCATCATCATCATCACtaaCTACTAGTccaggcatcaaataaaacgaaaggctca
4791
RPG for His-tag construction
ptac-gadA-R
ctagtatttctcctctttaattgtgagcgctcacaattccacaca
ptac-gadA-F
acaattaaagaggagaaatactagatgactaaacaggatcaggaaacgcag
1500
gadA
gadA-RPG-R
gTGATGATGATGATGATGagaGCCACCGCCACCgttgtgaactgtcgtcttgttatcagtatccg
Q5 PCR Reaction system (50 μL) for gadR/gadC/gadA/GFP:
Q5 Reaction Buffer (5x): 10 μL
dNTP: 1 μL
Forward Primer/ Reverse Primer :2.5 μL/2.5 μL
Template: 45ng
Q5 DNA Polymerase: 0.5 μL
ddh2O：33 μL
Q5 PCR Reaction system (50 μL) for the vector RPG
Q5 Reaction Buffer (5x): 10 μL
dNTP: 3 μL
Forward Primer/ Reverse Primer :2.5 μL/2.5 μL
Template (RPG): 45ng
Q5 DNA Polymerase: 0.5 μL
ddh2O：31 μL
Gibson Assembly
Since we had many segments, the success rate for Gibson assembly of 4 segments directly would be relatively low. Hence we chose to run Gibson Assembly twice. First, we conducted Gibson Assembly toward two groups of two segments. Then, we mixed the two Gibson Assembly reaction systems together.
Group
Assembly 1
Assembly 2
RPG-gadR-gadC-gadA
RPG for RCA and gadR
gadC and gadA
RPG-A-GFP (old)
RPG for GFP (old)
gadA and GFP
RPG-A-linker-His tag
RPG and gadA with His-tag
N/A
RPG-A-linker-GFP
RPG-1 and GFP
RPG-2 and gadA
Gibson Products Transformation
We transformed our Gibson products into competent E. coli DH5α cells.
Transformation system:
Positive control: 25 μL competent cell + 1 μL RPG
Negative control: 25 μL competent cell
Experiment group (with Gibson products): 50 μL competent cells + 2 μL Gibson products
Colony PCR
Gibson Assembly might cause a relatively high probability of false positive. In order to check whether the colony incorporated the targeted plasmid we expected, we ran colony PCR and used UV gel imaging to observe PCR product bands and then ran sequencing.
Pick 6-10 colonies.
Add 20 μL Amp+ liquid LB broth into 0.2 mL microfuge tube and add colonies, respectively. Then use 37°C shaker (with rotate speed 270r/min) to revive the bacteria for 1 hour.
Colony PCR Reaction System:
Taq Mix: 12.5 μL
Forward Primer/ Reverse Primer: 1 μL/1 μL
Bacteria LB solution: 2 μL
ddh2O: 10.5 μL
Plasmid Extraction
As the sequencing results were confirmed after comparing with the constructed plasmid, we in vitro cultured E. coli for 12 hours, then used the protocol of Axygen Mini-prepare Plasmid Extraction Kit to extract the RPG-gadR-gadC-gadA plasmid.
Plasmid Transformation
Since E. coli Nissle 1917 was not a conventionally used strain for synthetic biology, we decided to transform the desired plasmid instead of Gibson products into E. coli Nissle.
IPTG Induction for the First and Second Construction
We used the overnight (12 hours) incubated bacteria LB solution, diluted them 200x into Amp+ LB/M9 with 0.5 mM and 1mM IPTG, MSG (our substrate: sodium glutamate, 6 mL system), then we incubated them for 3 hours. After the incubation of 3 hours, E. coli Nissle 1917 would reach the early point of its exponential growth period. We then diluted the bacteria solution again 200x into Amp+ LB/M9 with 0.5 mM and 1mM IPTG, MSG(6 mL system) respectively and incubated 12 hours.
IPTG Induction for the Third Construction
The system and procedures were similar to that of IPTG Induction for the First and Second construction, instead we used only a concentration of 1mM IPTG to induce bacteria. And since we planned to do the reaction between MSG and GAD outside our bacteria, we did not add MSG through the induction process.  We used incubation shaker to incubate two 5mL bacteria LB solution for 8 hours and we transform the solution to 1L A+ LB solution. Then we incubated the bacteria for another 3 hours in order to allow bacteria entering exponential growth stage. Eventually, we added IPTG with the final concentration of 1mM and induced overnight.
Plate Reader Measurement for GFP Constructions
We measured the concentration of GFP using a Plate Reader to estimate the expression of gadA.
Protein treatment for the first construction
We first added lysis buffer (pH=7.8) and PMSF (Phenylmethanesulfonyl fluoride) protease inhibitor to avoid enzymes in lysosome degrading our target protein - GAD enzyme. Then we used ultrasound cell disrupter to lyse our bacteria. After that, we ran SDS-PAGE to observe generally if there were correct protein bands at 55 kDa (The length of GAD is 54.8).
Protein treatment for the second construction
We did not treat the induced bacteria solution, and we measured GABA directly. (See details in “GABA Measurement”)
Protein Purification for the third construction 1.1 (His-tag)
We first added lysis buffer (0.2M Sodium acetate-Acetic acid Buffer, pH=4.6) and PMSF (Phenylmethanesulfonyl fluoride) protease inhibitor to avoid enzymes in lysosome degrading our target protein --- GAD enzyme. Then we used ultrasound cell disrupter to lyse our bacteria. After that, we suspend the bacteria solution on the nickel column, which would allow proteins to suspend on the column but not other molecules  due to the 6x His tag we have added after gad A. Then we used imidazole to undergo competitive elution in order to elute our protein. After that, we used G250 (A type of Coomassie brilliant blue) to test whether the protein had been suspended on the column. At last, we ran SDS-PAGE to observe generally if there were correct protein bands at 55 kDa (The length of GAD is 54.8).
GAD enzyme Extraction for the third construction 1.2 (His-tag)
We have found another cofactor of the reaction of MSG and GAD --- Vitamin B, and direct protein purification process will eliminate not only gab T (see wiki-demonstration and project description) but also the cofactor Vitamin B. Meanwhile, the direct enzyme extraction will stop TCA cycle and the function of gab T which might influence the existence of GABA. So, we first added PMSF and then we used ultrasound (20 min, P=200W) to lyse bacteria directly and use the enzyme extraction solution to react with MSG directly.
The Reaction of the Third Construction 1.1
According to the result derived from our mathematic model, we prepared 3 acetic acid-sodium acetate buffers with pH=4.2, 4.4, 4.6 respectively. Then we added 2% MSG into the buffer and 800 μg GAD solution into the reaction system (20mL). After that, we incubated the reaction system with 100r/min and 37°C shaker for 3 hours.
GABA Measurement for the first and second construction
We measured the induced bacteria LB solution directly via AAA (Amino Acid Analyzer). Before the measurement, we ran the GABA standard curve with pure GABA (concentration of 50, 100ppm and 200ppm, filtered before adding samples). After that, we used the ultrasound with ice to lyse the bacteria, then centrifuged products for 10 minutes with 4 °C. We diluted samples and transferred the solutions into the vial afterward.
GABA Measurement for the third construction
We measured the induced bacteria LB solution directly via AAA (Amino Acid Analyzer). Before the measurement, we ran the GABA standard curve with pure GABA (concentration of 50, 100ppm and 200ppm, filtered before adding samples). Meanwhile, we diluted all our samples 12x with ddh2O and filter with 0.2 nm filter membrane.
© 2017 BNDS_China All Rights Reserved.
