<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BNDS_China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BNDS China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu"><DIV class="menu_item submenu" id="team"><A class="mt">Team</A><UL class="sublist"><LI id="tm"><A href="https://2017.igem.org/Team:BNDS_China/Team">Team</A></LI><LI id="co" style="border:none"><A href="https://2017.igem.org/Team:BNDS_China/Collaborations">Collaborations</A></LI></UL></DIV><DIV class="menu_item submenu" id="project"><A class="mt">Project</A><UL class="sublist"><LI id="dc"><A href="https://2017.igem.org/Team:BNDS_China/Description">Description</A></LI><LI id="ds"><A href="https://2017.igem.org/Team:BNDS_China/Design">Design</A></LI><LI id="ex"><A href="https://2017.igem.org/Team:BNDS_China/Experiments">Experiment</A></LI><LI id="il"><A href="https://2017.igem.org/Team:BNDS_China/InterLab">InterLab</A></LI><LI id="dm"><A href="https://2017.igem.org/Team:BNDS_China/Demonstrate">Demonstrate</A></LI><LI id="at" style="border:none"><A href="https://2017.igem.org/Team:BNDS_China/Attributions">Attributions</A></LI></UL></DIV><DIV class="menu_item submenu" id="parts"><A href="https://2017.igem.org/Team:BNDS_China/Parts" class="mt">Parts</A></DIV><DIV class="menu_item nosubmenu" id="safety"><A href="https://2017.igem.org/Team:BNDS_China/Safety" class="mt">Safety</A></DIV><DIV class="menu_item submenu" id="hprac"><A class="mt">H.Prac</A><UL class="sublist"><LI id="sh"><A href="https://2017.igem.org/Team:BNDS_China/HP/Silver">Silver HP</A></LI><LI id="ig"><A href="https://2017.igem.org/Team:BNDS_China/HP/Gold_Integrated">Integrated and Gold</A></LI><LI id="pe" style="border:none"><A href="https://2017.igem.org/Team:BNDS_China/Engagement">Public Engagement</A></LI></UL></DIV><DIV class="menu_item submenu" id="awards"><A href="https://2017.igem.org/Team:BNDS_China/Model" class="mt">Awards</A></DIV><DIV class="menu_item nosubmenu" id="judge"><A href="https://igem.org/2017_Judging_Form?team=BNDS_China" class="mt">Judge</A></DIV></DIV><DIV class="wrapper"><DIV class="block_white"><H3 style="padding-top: 6%;">Demonstrate</H3></DIV><DIV class="block_grey" style="margin-top: 0px; height: 15000px;"><H3>LacI-pTac-<I>gadR-gadC-gadA</I> (<A href="http://parts.igem.org/Part:BBa_K2326003">BBa_K2326003</A>)</H3><H5>Cloning</H5><P>The construct was assembled with the vector Repressor Generator (RPG) and was transformed to <I>Escherichia coli</I> Nissle 1917 (<I>E. coli</I> Nissle) to test expression. Fig. 1 shows the result of sequencing of colony PCR. It shows that the targeted plasmid was assembled correctly.</P><P>Fig. 1 Sequencing result of construction of RPG- BBa_K2326003. Derived from Snapegene.</P><H5>SDS-PAGE</H5><P>We test the expression of glutamate gamma-aminobutyrate antiporter and glutamate decarboxylase (GAD), which were coded by <I>gadC</I> and <I>gadA</I> respectively. Fig. 2 shows the result of SDS-PAGE. The protein size of the antiporter should be 55.22 kDa, and for the GAD enzyme should be 52.91 kDa. Protein bands in the correct molecular weight range were visualized in the area around 50 kDa. Comparing the control group, which used endogenous, wild-type <I>E. coli</I> Nissle, the protein bands presented in the supernatant of <I>E. coli</I> Nissle liquid culture with constructed plasmid were clearer, indicating that <I>gadC</I> and <I>gadA</I> are expressed.</P><P>Fig. 2 SDS-PAGE result of the construction Bba_K2326000.</P><H5>Amino Acid Analyzer</H5><P>We also tried to measure the yield of γ－aminobutyric acid (GABA) by using amino acid analyzer (AAA). Firstly, analytically pure GABA sample solution was tested and the result is shown in Fig. 3. The peak of GABA should have rention time of 48 min to 49 min, according to the standard control run.</P><P>Fig. 3. Amino acid analyzer result of pure GABA sample</P><P>Since the promoter we used is pTac, inducible by Isopropyl β-D-Thiogalactoside (IPTG), The IPTG inducing concentration was 1.0 mM. 1% of monosodium glutamate (MSG) was added as substrate. After 12 hours’ induction, the bacterial supernatant was processed and tested. Fig. 4 shows the result of the GABA yield by <I>E. coli</I> Nissle with the constructed plasmid. However, there was no peak during 48 min to 49 min, indicating that GABA was not produced.</P><P>Fig. 4 yield of GABA of <I>E. coli</I> Nissle with part Bba_K2326003</P><H5>Problems</H5><P>There were three problems in our construction causing the inability to produce GABA. First, since we were unable to know which genes the transcriptional regulator (<I>gadR</I>) regulates in <I>Lactobacillus brevis</I> NCL912, <I>gadR</I> might not function well in <I>E. coli</I> Nissle because of the absence of genes it regulates. Also, we could not locate the glutamate gamma-aminobutyrate antiporter (<I>gadC</I>) in <I>E. coli</I> Nissle cell membrane, so <I>gadC</I> might not function. There were two genes between the pTac promoter and <I>gadA</I>, so<I> gadA</I> might not be fully expressed due to increased plasmid and gene lenth. Based on the analysis of these problems, we constructed the part Bba_K2326004.</P><H3>LacI-pTac-gadA-GFP (<A href="http://parts.igem.org/Part:BBa_K2326004">Bba_K2326004</A>)</H3><H5>Cloning</H5><P>Same as the first approach, we constructed the part on RPG and transformed the plasmid to <I>E. coli</I> Nissle. Fig. 5 shows the sequencing result of colony PCR. It shows that the targeted plasmid was assembled correctly.</P><P>Fig. 5 Sequencing result of construction of RPG- BBa_K2326004. Derived from Snapegene.</P><H5>Plate Reader</H5><P>The part contains green fluoresce protein (GFP). We measured the relative concentration of GFP in bacterial suspension to determine the expression of <I>gadA</I>. The concentrations of GFP in M9 broth are represented by `&quot;Relative Fluorescence Intensity&quot;=log⁡(&quot;Fluorescence Intensity&quot;)/(&quot;OD&quot;_600)` in Fig. 6. Positive control was tested on <I>E. coli</I> Nissle with PSB1C3-GFP plasmid, and the negative control was tested on the endogenous, wild-type <I>E. coli</I> Nissle. The same induction process was used.</P><P>Results suggest the concentrations of GFP expressed by <I>E. coli</I> Nissle with the constructed plasmid was higher than the positive control and negative control. Since <I>gadA</I> was closer to the promoter compared to GFP, it was expected to express more than GFP.</P><H5>Amino Acid Analyzer</H5><P>Fig. 8 yield of GABA of E. coli Nissle with part Bba_K2326004 (M9 broth)</P><P>The same experimental condition was used in the amino acid analyzing process. Fig. 8 shows the yield of GABA by <I>E. coli</I> Nissle with the constructed plasmid in M9. However, there were still no peaks during 48 min to 49 min in the two figures, indicating that GABA was not produced.</P><H5>Problems</H5><P>There were two possible reasons of our failure to produce GABA. First, there might be a strong ribosomal binding side presented on the upstream of GFP, making most of the ribosomes bind with GFP but not <I>gadA</I>. Thus, <I>gadA</I> was not fully expressed. Second, evidence showed that GABA permease, 4-aminobutyrate aminotransferase, and 4-aminobutyrate aminotransferase presented in <I>E. coli</I> can degrade GABA to glutamate and flow back into TCA cycle, so GABA maybe have been degraded prior to analysis (Vo &amp; Hong).</P><P>Our culture conditions during induction had deficiencies as well. According to Lü et.al., the yield of GABA of <I>L. sakei</I> B2-16 after incubating for 4 days in MRS medium containing 3% MSG was 165 mM. Since we only incubated the bacterial culture for 12 hours and the concentration of MSG was 1%, <I>E. coli</I> Nissle might not have enough time and enough substrate to produce GABA. Moreover, the GAD activity of <I>Lactobacillus brevis</I> NCL912 is the highest when pH of MSG medium ranges from 4.0 – 5.0 <CITE>(Huang et.al.)</CITE>. However, the pH of LB medium is about 7, so GAD might not function well in such a basic environment. Therefore, the yield of GABA may have been low.</P><P>We developed Bba_K2326005 and following experiment to eliminate these problems.</P><H3>LacI-pTac-<I>gadA</I>-Linker-GFP &amp; LacI-pTac-<I>gadA</I>-His-tag (<A href="http://parts.igem.org/Part:BBa_K2326005">BBa_K2326005</A>&amp;<A href="http://parts.igem.org/Part:BBa_K2326006">BBa_K2326006</A>)</H3><H5>Cloning</H5><P>The 2 constructs were assembled with RPG and were transformed to <I>E. coli&lt;.i&gt; Nissle. Fig. 8 and Fig 9 shows the sequencing results of the 2 constructions. They show that the targeted plasmid was assembled correctly.</I></P><P>Fig. 8 Sequencing result of construction RPG- BBa_K2326005. Derived from Snapegene.</P><P>Fig. 9 Sequencing result of construction RPG- BBa_K2326006. Derived from Snapegene.</P><H5>Plate Reader</H5><P>We measured the relative concentration of GFP in recombinant <I>E. coli</I> Nissle (RPG- BBa_K2326005) suspension. The IPTG inducing concentration was 1.0 mM, and 2% of MSG was added as substrate. The induction process lasted for12 hours. Negative control was tested with endogenous, wild-type <I>E. coli</I> Nissle. The relative concentration of GFP is represented by relative fluorescence intensity in Fig. 10.</P><P>The concentration of GFP of recombinant <I>E. coli</I> Nissle (RPG- BBa_K2326005) was higher than the negative control, indicating that the fusion protein of GAD and GFP was expressed. Thus, GAD could be expressed in<I> E. coli</I> Nissle (RPG-BBa_K2326006).</P><H5>Amino Acid Analyzer Result After Protein Purification</H5><P>We purified GAD in <I>E. coli</I> Nissle and did the reaction (see <A href="https://2017.igem.org/Team:BNDS_China/Experiments">experiments</A>). Fig. 11 – Fig. 13 show the result of reaction products. Since there were still no peaks in the range of 48 – 50 min, GABA was still not produced.</P><P>Fig. 11 Yield of GABA of <I>E. coli</I> Nissle (RPG- BBa_K2326006) with reaction pH 4.2 (protein purification)</P><P>Fig. 12 Yield of GABA of <I>E. coli</I> Nissle (RPG- BBa_K2326006) with reaction pH 4.4 (protein purification)</P><P>Fig. 13 Yield of GABA of <I>E. coli</I> Nissle (RPG- BBa_K2326006) with reaction pH 4.6 (protein purification)</P><P>When we analyzed the experiment, we found that we ignored the effect of cofactors. Pyridoxal phosphate is the cofactor of GAD enzyme, which is derived from vitamin B6 (Purves &amp; Williams). In our reaction system, vitamin B6 was absent, so GAD might have low activity. We developed the next step based on this problem.</P><H5>The Final Threshold – Amino Acid Analyzer Result After Crude Enzyme Extraction</H5><P>Since vitamin B6 presents originally in wild-type <I>E. coli</I> Nissle (Klijn), we decided to use crude enzyme extraction (see <A href="https://2017.igem.org/Team:BNDS_China/Experiments">experiments</A>), so the cofactor of GAD was present in the reaction system. The same procedure was used to produce GABA. Then, the yield of GABA was measured by amino acid analyzer. Fig. 14 – Fig. 16 show the yield of GABA.</P><P>Fig. 14 Yield of GABA of <I>E. coli</I> Nissle (RPG- BBa_K2326006) with reaction pH 4.2 (crude extraction)</P><P>Fig. 15 Yield of GABA of <I>E. coli</I> Nissle (RPG- BBa_K2326006) with reaction pH 4.4 (crude extraction)</P><P>Fig. 16 Yield of GABA of <I>E. coli</I> Nissle (RPG- BBa_K2326006) with reaction pH 4.6 (crude extraction)</P><P>The prescense of peaks corresponding to 48 to 50 min suggests GABA was produced. The yield of GABA is calculated by:</P><P>`&quot;Peak area of GABA yielded by E.coli Nissle&quot;/&quot;Peak area of pure GABA sample&quot;**&quot;Concentration of pure GABA sample&quot;**&quot;dilution factor&quot;`. Peak areas were derived from the system report of amino acid analyzer. Concentration of pure GABA sample in this experiment was 500 mg/L, and the dilution factor was 60. Fig. 17 shows yield of GABA at 3 pH values.</P><P>Fig. 17 Yield of GABA (after crude enzyme extraction)</P><P>The maximum GABA yield was 2976.79 mg/L, produced at pH 4.4. The modelling result suggests that the optimum reaction pH is 4.28, which is close to the optimum pH value in experiment. The result demonstrates that the optimum working temperature for GAD ranges from 4.2 to 4.6.</P><H5>Future Plan</H5><P>To make <I>E. coli</I>&amp;<I>Lactobacillus</I> produce GABA stably and to avoid using an antibiotic selecting mark, we can insert <I>gadA</I> to the chromosome of <I>E. coli</I> and <I>Lactobacillus</I>. Also, to enhance the yield of GABA, modifying <I>gadC</I> to locate the glutamate gamma-aminobutyrate antiporter in <I>E. coli</I> cell membrane can help to transport more MSG into the cell, which may increase the production of GABA as well. GABA permease, 4-aminobutyrate aminotransferase, and 4-aminobutyrate aminotransferase, which are expressed by <I>gabP</I>, <I>gabT</I>, and <I>puuE</I>, can degrade GABA in metabolic pathway (Li). It will be a stronger approach to knock out these genes in <I>E. coli</I>’s chromosome, so GABA will not be degraded. Scaffold proteins play an important role in regulating the signaling pathways, and can help the protein kinase and phosphatase to react with their specific substrates (Ferrell). Thus, if we can synthesize scaffold proteins to regulate the GABA synthesis pathway, the yield of GABA will be improved.</P><H5>References</H5><CITE><P>Vo, T. D., &amp; Hong, S. (2012). Optimization of gamma-Aminobutyric Acid Bioconversion 	by Recombinant Escherichia coli. KSBB Journal, 27(2), 127-130. 	doi:10.7841/ksbbj.2012.27.2.127</P><P>Lü, Y. G., Zhang, H., &amp; Yao, H. Y. (2012). Enzymatic Production of γ-Aminobutyric 	Acid Using Rice Bran. Advanced Materials Research, 586, 85-91.	doi:10.4028/www.scientific.net/amr.586.85</P><P>Ferrell, J. E. (2000). What Do Scaffold Proteins Really Do? Sciences STKE, 2000(52). 	doi:10.1126/stke.2000.52.pe1</P><P>Huang, G., Li, C., &amp; Cao, Y. (2011). Effect of Sodium L-glutamate on Growth and 	Survival of Lactobacillus brevis NCL912 at Different Acidic pH. Annals of 	Microbiology, 62(1), 351-355. doi:10.1007/s13213-011-0269-7</P><P>Li, Peng. (n.d.). Metabolic Engineering of Escherichia coli for Gamma-Aminobutyric Acid 	Production. Retrieved October 19, 2017, from www.doc88.com/p-	5844923971361.html</P><P>Purves, D., &amp; Williams, S. M. (2001). Neuroscience. 2nd edition. Sinauer Associates.</P><P>Klijn, J. (2012). Ongoing 2b/3a inhibition In Myocardial infarction 	Evaluation. Http://isrctn.org/&gt;. doi:10.1186/isrctn06195297</P></CITE></DIV></DIV><FOOTER><P style="margin-left: 20px; padding-top:15px; font-size: 20px;">© 2017 BNDS_China All Rights Reserved.</P></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>