<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Ruia-Mumbai_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Ruia-Mumbai/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><NAV class="navbar navbar-expand-sm navbar-light fixed-top bg-light "><DIV class="collapse navbar-collapse" id="collapsibleNavbar"><UL class="navbar-nav  nav-center" style="margin-top: 20px;"><LI class="nav-item spacing dropdown"><A class="nav-link dropdown-toggle" href="#" id="aboutdrop" data-toggle="dropdown">
                ABOUT
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Team">Team</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/sponsors">Sponsors</A></DIV></LI><LI class="nav-item dropdown spacing"><A class="nav-link dropdown-toggle" href="#" id="projectdrop" data-toggle="dropdown">
                PROJECT
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Description">Description</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Design">Design</A><A class="dropdown-item" href=" https://2018.igem.org/Team:Ruia-Mumbai/Improve">Improve </A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Attributions">Attributions</A></DIV></LI><LI class="nav-item spacing dropdown"><A class="nav-link dropdown-toggle" href="#" id="partsdrop" data-toggle="dropdown">
               INTERLAB
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/InterLab#Objective">Objective</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/InterLab#Experience">Experience</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/InterLab#platereader">Plate Reader</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/InterLab#flowcytometry">Flow cytometry</A></DIV></LI><LI class="nav-item spacing dropdown"><A class="nav-link dropdown-toggle" href="#" id="partsdrop" data-toggle="dropdown">
                PARTS
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Parts">Parts Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Basic_Part">Basic Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Composite_Part">Composite Parts</A></DIV></LI><LI class="nav-item spacing dropdown"><A class="nav-link dropdown-toggle" href="#" id="partsdrop" data-toggle="dropdown">
               NOTEBOOK
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Notebook">Daily Notes</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Experiments">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Special:Search/Team:Ruia-Mumbai/Bibliography">Bibliography</A></DIV></LI><LI class="nav-item  spacing dropdown"><A class="nav-link dropdown-toggle" href="#" id="pracdrop" data-toggle="dropdown">
               SAFETY
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Safety"> Lab Safety</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Safety#ProductSafety"> Product Safety</A></DIV></LI><LI class="nav-item  spacing dropdown"><A class="nav-link dropdown-toggle" href="https://2018.igem.org/Team:Ruia-Mumbai/Human_Practices" id="pracdrop" data-toggle="dropdown">
               HUMAN PRACTICES
              </A><DIV class="dropdown-menu"><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Human_Practices#Overview">Overview </A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Human_Practices#Community">Community</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Human_Practices#Engagement">Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Human_Practices#IHP">Integated human practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:Ruia-Mumbai/Human_Practices#sustainability">Sustainibility</A></DIV></LI><LI class="nav-item  spacing"><A class="nav-link" href="https://2018.igem.org/Team:Ruia-Mumbai/achievements" id="pracdrop">
               ACHIEVEMENTS
              </A></LI></UL></DIV></NAV></DIV><DIV class="page" style=""><DIV class="row"> </DIV><H1 style="color:#d80505;  text-align : center; margin : 20px;"> RESULTS </H1><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px"><B><H2>Experiment 1:</H2></B><DIV class="row"> </DIV><DIV class="row"><DIV class="col-md-7"><B><H2>Catechu standardisation</H2></B><P style="text-align: justify;">
                                A successful attempt in standardising the ratio of catechu:slaked lime required for media formulation. 
                        </P><BUTTON class="fadeandscale1_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="row"><DIV class="card card-body " style="" id="fadeandscale1">
                                    After interactive sessions with various paanwalas we learnt that the amount of catechu applied to paan is high whereas the amount of slaked lime applied is quite low keeping in mind its inflammatory properties.
        
                                    With the above information we tried out various ratios of catechu:slaked lime so as to get the maximum intensity of the distinct red colour which is characteristic to the paan stains prevalent across the country.
                                    <DIV style="overflow-x: auto;"><TABLE style=""><TBODY><TR><TH style="border-collapse: collapse;border: 1px solid black;">
                                                Ratio (Catechu:slaked lime)
                                            </TH><TH style="border-collapse: collapse;border: 1px solid black;">Amount of catechu(gms)</TH><TH style="border-collapse: collapse;border: 1px solid black;">Boiling water
                                                (ml)
                                                </TH><TH style="border-collapse: collapse;border: 1px solid black;">
                                                header not given
                                            </TH><TH style="border-collapse: collapse;border: 1px solid black;">
                                                Slaked lime
                                                    (gms)

                                            </TH><TH style="border-collapse: collapse;border: 1px solid black;">

                                                Absorbance
                                                at λmax= 254nm

                                            </TH></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">1:0.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">100</TD><TD style="border-collapse: collapse;border: 1px solid black;">Cool the</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.01</TD><TD style="border-collapse: collapse;border: 1px solid black;">1.921</TD></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">1:0.2</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">100</TD><TD style="border-collapse: collapse;border: 1px solid black;">Solution</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.02</TD><TD style="border-collapse: collapse;border: 1px solid black;">1.871</TD></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">1:0.3</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">100</TD><TD style="border-collapse: collapse;border: 1px solid black;">for 15-20</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.03</TD><TD style="border-collapse: collapse;border: 1px solid black;">1.754</TD></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">1:0.4</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">100</TD><TD style="border-collapse: collapse;border: 1px solid black;">mins</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.04</TD><TD style="border-collapse: collapse;border: 1px solid black;">1.206</TD></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">1:0.5</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">100</TD><TD style="border-collapse: collapse;border: 1px solid black;">0.05</TD><TD style="border-collapse: collapse;border: 1px solid black;">1.319</TD></TR></TBODY></TABLE></DIV>
From the above observations it was concluded that the maximum intensity of red colour obtained  from the solution was from the sample with 1:0.1 Catechu : slaked lime concentration.
Hence, the same ratio was used for the preparation of the stock with 10g Catechu and 1g slaked lime of which 1% was used for media preparation.
<BUTTON class="fadeandscale1_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 2:</H2></B><B><H4>Media Formulation and Enrichment</H4></B><DIV class="row"><DIV class="col-md-7"><P style="text-align: justify;">
                                    Various media formulations for enriching natural samples with the aim of finding catechu degrading isolates.



                            </P><BUTTON class="fadeandscale2_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale2">
                                We narrowed down to four variants of the minimal medium, M9, for processing samples collected from different sources. The four different media used were:
                                <OL><LI>
                                    M9
                                </LI><LI>
                                    M9+Glucose
                                </LI><LI>
                                    M9+CS (catechu+slaked lime stock)
                                </LI><LI>
                                   M9+CS (catechu+slaked lime stock)
                                   </LI></OL>
                                
                                The samples were processed (passed through bacterial proof filters of 0.22µ) before inoculation so as to eliminate chances of growth due to presence of existent carbon source in the collected sample.
Hence, a flask with M9 medium was also inoculated and checked for growth so as to ensure that the bacteria are not growing due to the carbon source in the sample but are using the carbon source provided in the medium.The choice of minimal medium was also made keeping in mind the single carbon source utilisation by the organisms.
Growth was observed in all the flasks, except the flasks containing only M9 medium without any carbon source, in less than 72 hours of shaking incubation at room temperature.
No growth/turbidity in the flasks containing only M9 medium proves that the carbon source from sample was successfully filtered &amp; thus has proven to be an efficient negative control.
Moreover, reduction in the intensity of the red colour (due to catechu extract in the medium) was observed after 24 hours. 

                             <P style="text-align: center; margin: 10px;">Media formulation for Sample 1 viz. water from Ulhas river</P><BUTTON class="fadeandscale2_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 3:</H2></B><DIV class="row"><DIV class="col-md-7"><B><H4>Isolation of organisms and identification by 16S rRNA sequencing</H4></B><P style="text-align: justify;">
                                        Isolation on solid media after enrichment of samples and identification of the isolates by 16s rRNA sequencing.


    
                                </P><BUTTON class="fadeandscale3_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale3">
                                     Isolation was carried out by taking inocula from the enrichment broths on various combinations (same as used for enrichment of samples) of solid media. Growth was seen on all the plates except on just M9 minimal medium plates.
Isolates which were showing prominent degradation of colour on the plates were sent for identification by 16s rRNA sequencing.

Results from National Centre for Microbial Resource (NCMR) for 16s rRNA sequencing:
<B>Bacterial Strains: 
</B><DIV style="overflow-x: auto;"><TABLE style=""><TBODY><TR><TH style="border-collapse: collapse;border: 1px solid black;">
                                                Strain no.                                            </TH><TH style="border-collapse: collapse;border: 1px solid black;">Closest neighbour</TH><TH style="border-collapse: collapse;border: 1px solid black;">
                                                Accession No.
                                                </TH><TH style="border-collapse: collapse;border: 1px solid black;">
                                                % similarity
                                            </TH></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">S2C1</TD><TD style="border-collapse: collapse;border: 1px solid black;">Klebsiella pneumoniae subsp.
pneumonia DSM 30104(T)
</TD><TD style="border-collapse: collapse;border: 1px solid black;">AJJI01000018</TD><TD style="border-collapse: collapse;border: 1px solid black;">99.66</TD></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">S2C2</TD><TD style="border-collapse: collapse;border: 1px solid black;">Klebsiella quasipneumoniae subsp.quasipneumoniae 01A030(T)</TD><TD style="border-collapse: collapse;border: 1px solid black;">HG933296</TD><TD style="border-collapse: collapse;border: 1px solid black;">99.73</TD></TR></TBODY></TABLE></DIV><B>Fungal Strains: 
</B><TABLE style="border-collapse: collapse;border: 1px solid black;"><TBODY><TR><TH style="border-collapse: collapse;border: 1px solid black;">
                                                Strain no.                                            </TH><TH style="border-collapse: collapse;border: 1px solid black;">Closest neighbour</TH><TH style="border-collapse: collapse;border: 1px solid black;">
                                                Accession No.
                                                </TH><TH style="border-collapse: collapse;border: 1px solid black;">
                                                % similarity
                                            </TH></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">S1</TD><TD style="border-collapse: collapse;border: 1px solid black;">Aspergillus niger strain ATCC 16888
</TD><TD style="border-collapse: collapse;border: 1px solid black;">AY373852.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">99</TD></TR><TR><TD style="border-collapse: collapse;border: 1px solid black;">A.niger</TD><TD style="border-collapse: collapse;border: 1px solid black;">Aspergillus niger strain ATCC 16888</TD><TD style="border-collapse: collapse;border: 1px solid black;">AY373852.1</TD><TD style="border-collapse: collapse;border: 1px solid black;">99</TD></TR></TBODY></TABLE><BUTTON class="fadeandscale3_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 4:</H2></B><DIV class="row"><DIV class="col-md-7"><B><H4>In search of potential enzymes that degrade catechu!</H4></B><P style="text-align: justify;">                                           
                                     Literature survey and screening of various enzymes that break bonds in molecules having similar structures as that of components of catechu!


                                    </P><BUTTON class=" fadeandscale4_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style=" " id="fadeandscale4">
                                        After having the sequencing results in hand we did a thorough literature survey to find out the potential enzymes present in the isolates which could degrade components of catechu. We used softwares like BRENDA, PUBMED, UNIPROT to obtain relevant information. 
The major candidates from our literature survey were tannases and catechol dioxygenases.

On these grounds we also looked at various team wikis from past years in search for similar projects. We found that Team Paris-Bettencourt has worked with catechol dioxygenases. We got in touch with them and they sent us their cloned enzymes [catechol-1,2-dioxygenase (CatA) and catechol-2,3-dioxygenase(XylE)] which we could successfully transform into DH5alpha cells. We tested these transformants on catechu and slaked lime media and the results were flabbergasting. 

A distinct zone of clearance was observed around the streaks and spots of the growing culture on the catechu and slaked lime solid medium. Hence, we decided to proceed with these cultures as well our natural isolates.
<BUTTON class="fadeandscale4_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 5:</H2></B><DIV class="row"><DIV class="col-md-6"><B><H4>DEGRADATION - Chewing Catechu</H4></B><P style="text-align: justify;">
                                                
                                        Degradation of catechu by potential isolates and selected enzyme XylE in M9+CS broth medium.
                                        </P><BUTTON class="fadeandscale5_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale5">                                     
                                      Other than plates we checked the degradation activity of potential isolates and enzyme XylE in broth medium.
The distinct colour difference between the control tube and potential isolates which includes Klebsiella pneumoniae, Aspergillus niger (S1 fungus and A.niger), S4 fungus can be observed.
For quantifying degradation activity of DH5α containing enzyme XylE in M9+CS broth medium we used ELISA plate reader. 10 % of volume inoculum of DH5α was inoculated in M9+CS broth of catechu stock 0.1%, 0.5% and 1% readings was taken at 415 nm on interval of 48 hrs.
                                     
                                        <BUTTON class="fadeandscale5_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 6:</H2></B><DIV class="row"><DIV class="col-md-7"><B><H4>Cloning - Challenge Accepted!</H4></B><P style="text-align: justify;">
                                                    Molecular cloning of the gblock fragments in pSB1C3 backbone (digestion, ligation, transformation) and their confirmation by diagnostic digest agarose gel run.     
                
                                            </P><BUTTON class=" fadeandscale6_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale6">
                                              As all cloning strategies, there were few trial and error scenarios wrt to the ratios of fragments to be used for a perfectly assembled DNA. 
Ligation math for all the fragments were done and the process of digestion, ligation and transformation was carried out successfully to yield 2 well assembled constructs. The plasmid was again extracted from the selected clones on the plates and run an agarose gel electrophoresis after a single and dual cut diagnostic digest with EcoR1 and Pst1.
In the above Gel-Red stained Agarose gel, in which construct (pBAD_ompT_xylE) and construct (pBAD_xylE) are the cloned modules which have the bands at the expected size of 3278 bp or 3.3 kb and 3218 bp or 3.2 kb respectively after single EcoR1 digest.
Also faint bands of the backbone at 2 kb were obtained but the fragment DNA wasn’t well stained (maybe due to low amounts) and very faint bands of fragments of size 1.2 kb for (pBAD_ompT_xylE) as expected of 1208 bp/1.2 kb and ~1.2 kb for (pBAD_xylE) as expected 1148 bp/1.15 kb were observed.
This diagnostic digest confirms that the cloned fragments are of the desired assembly and the clones of these DNA constructs are used for further analysis by protein characterisation. 


<BUTTON class="fadeandscale6_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 7:</H2></B><DIV class="row"><DIV class="col-md-7"><B><H4>SDS-PAGE Protein Characterisation</H4></B><P style="text-align: justify;">
After analysing a successful cloning, the plasmid DNA was then transferred into DH5a cells to have growing transformants for characterising protein expression. 


                                                       
                    
                                                </P><BUTTON class="fadeandscale7_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale7">
                                                   From the above SDS-PAGE Coomassie Brilliant Blue stained gel it is interpreted that-
                           <UL><LI>
The dark protein bands are observed slightly above the 33 kDa ladder which is equal to the expected size of the enzyme catechol-2,3-dioxygenase of 35 kDa. </LI><LI>
DH5a pBAD_xylE transformants grown in Glucose do not show any protein bands at this size and hence can be verified as negative control.
While DH5a pBAD_xylE transformants grown in Arabinose show the bands at the expected size.</LI><LI>
In presence of Glucose in the substrate, the enzyme is not synthesised, while in presence of Arabinose, the enzyme is synthesised, which is true to the fact that the construct having promoter pBAD is being induced only in presence of arabinose to synthesise the enzyme catechol-2,3-dioxygenase.</LI><LI>
The above gel confirms that the constructs are well assembled and successfully synthesize the desired enzyme catechol-2,3-dioxygenase which is used for catechu degradation and further tested for its ability to degrade various dye substrates.</LI><BUTTON class="fadeandscale7_close" style="margin :15px auto 0 auto">Close</BUTTON><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 8:</H2></B><DIV class="row"><DIV class="col-md-7"><B><H4>Assay for detection of catechol-2,3-dioxygenase</H4></B><P style="text-align: justify;">
                                                           An assay which will detect the presence of the enzyme catechol-2,3-dioxygenase which is a translational product of our cloned gene-XylE.


                                                    </P><BUTTON class=" fadeandscale8_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale8">
                                                        This assay gives a direct readout of the presence of the enzyme catechol-2,3-dioxygenase. The colourless substrate catechol is broken down to a yellow coloured product 2-hydroxymuconic semialdehyde (2-HMS) and this reaction is catalysed by the enzyme catechol-2,3-dioxygenase.
                                                        
                                                        Intensity of the generated yellow colour was detected in the form of absorbance values.The absorbance readouts were taken at 415 nm at an interval of 5 minutes and graphs were prepared to interpret the generated data. 
The XylE gene in our construct is under the regulation of the pBAD promoter which is arabinose responsive. From the generated data we could conclude that with advancement in time, the absorbance of the system increases when the cells are grown in the presence of arabinose but remains fairly constant when grown in glucose. 
                    <BUTTON class="fadeandscale8_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row" style="border: solid #efefef; padding: 10px 30px 30px; border-radius: 10px;"><B><H2>Experiment 9:</H2></B><DIV class="row"><DIV class="col-md-7"><B><H4>Versatility - Testing on dyes</H4></B><P style="text-align: justify;">
                                                                Demonstrating versatility of construct/model to degrade common lab dyes other than paan stains.Clearance around the colony observed on plates containing several dyes.


                                                        </P><BUTTON class=" fadeandscale9_open btn btn-danger learn_more">Learn More</BUTTON></DIV></DIV></DIV><DIV class="card card-body" style="" id="fadeandscale9">
                                                            After gaining insight of versatility point by Ajay Bio-Tech company, we tested our construct on chemically related laboratory dyes.
We inoculated our positive clones on luria bertani medium plates of 0.025% concentration of dyes which includes congo red,methylene blue and safranin. Clearance around the colony indicating degradation was distinctly observed after 48 hrs of incubation. This concludes the activity of enzyme/ action of catechewing coli on other stains/dyes giving us future prospective to work on project. <BUTTON class="fadeandscale9_close" style="margin :15px auto 0 auto">Close</BUTTON></DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="row"> </DIV><DIV class="container-fluid"><DIV class="row"><DIV class="col-lg-3" style="padding: 10px;"><H4 style="text-align:center;"><A href="https://goo.gl/maps/zSNK2yykxW62"> Our co-ordinates:</A></H4><P style="color:#59462b; text-align:center;">L .Nappo Road,

              Matunga

              Dadar East,Mumbai,

              Maharashtra 400019,

              India</P><P style="font-weight: bold; text-align : center;"> catechewingcoli@gmail.com</P></DIV><DIV class="col-lg-3" style="padding: 10px;"><H4 color="#55372b">FOLLOW US ON:</H4></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>