Team:Ruia-Mumbai/Results
ABOUT
Team Collaborations Sponsors
PROJECT
Description Design Improve Results Attributions
INTERLAB
Objective Experience Plate Reader Flow cytometry
PARTS
Parts Overview Basic Parts Composite Parts
NOTEBOOK
Daily Notes Protocols Bibliography
SAFETY
Lab Safety Product Safety
HUMAN PRACTICES
Overview Community Engagement Integated human practices Sustainibility
ACHIEVEMENTS
 
RESULTS
 
Experiment 1:
 
Catechu standardisation
A successful attempt in standardising the ratio of catechu:slaked lime required for media formulation.
Learn More
After interactive sessions with various paanwalas we learnt that the amount of catechu applied to paan is high whereas the amount of slaked lime applied is quite low keeping in mind its inflammatory properties.
With the above information we tried out various ratios of catechu:slaked lime so as to get the maximum intensity of the distinct red colour which is characteristic to the paan stains prevalent across the country.
Ratio (Catechu:slaked lime)
Amount of catechu(gms)
Boiling water                                                 (ml)
header not given
Slaked lime                                                     (gms)
Absorbance                                                 at λmax= 254nm
1:0.1
0.1
100
Cool the
0.01
1.921
1:0.2
0.1
100
Solution
0.02
1.871
1:0.3
0.1
100
for 15-20
0.03
1.754
1:0.4
0.1
100
mins
0.04
1.206
1:0.5
0.1
100
0.05
1.319
From the above observations it was concluded that the maximum intensity of red colour obtained  from the solution was from the sample with 1:0.1 Catechu : slaked lime concentration. Hence, the same ratio was used for the preparation of the stock with 10g Catechu and 1g slaked lime of which 1% was used for media preparation.
Close
 
 
 
Experiment 2:
Media Formulation and Enrichment
Various media formulations for enriching natural samples with the aim of finding catechu degrading isolates.
Learn More
We narrowed down to four variants of the minimal medium, M9, for processing samples collected from different sources. The four different media used were:
M9
M9+Glucose
M9+CS (catechu+slaked lime stock)
M9+CS (catechu+slaked lime stock)
The samples were processed (passed through bacterial proof filters of 0.22µ) before inoculation so as to eliminate chances of growth due to presence of existent carbon source in the collected sample.
Hence, a flask with M9 medium was also inoculated and checked for growth so as to ensure that the bacteria are not growing due to the carbon source in the sample but are using the carbon source provided in the medium.The choice of minimal medium was also made keeping in mind the single carbon source utilisation by the organisms.
Growth was observed in all the flasks, except the flasks containing only M9 medium without any carbon source, in less than 72 hours of shaking incubation at room temperature.
No growth/turbidity in the flasks containing only M9 medium proves that the carbon source from sample was successfully filtered & thus has proven to be an efficient negative control.
Moreover, reduction in the intensity of the red colour (due to catechu extract in the medium) was observed after 24 hours.
Media formulation for Sample 1 viz. water from Ulhas river
Close
 
 
 
Experiment 3:
Isolation of organisms and identification by 16S rRNA sequencing
Isolation on solid media after enrichment of samples and identification of the isolates by 16s rRNA sequencing.
Learn More
Isolation was carried out by taking inocula from the enrichment broths on various combinations (same as used for enrichment of samples) of solid media. Growth was seen on all the plates except on just M9 minimal medium plates.
Isolates which were showing prominent degradation of colour on the plates were sent for identification by 16s rRNA sequencing.
Results from National Centre for Microbial Resource (NCMR) for 16s rRNA sequencing:
Bacterial Strains:
Strain no.
Closest neighbour
Accession No.
% similarity
S2C1
Klebsiella pneumoniae subsp. pneumonia DSM 30104(T)
AJJI01000018
99.66
S2C2
Klebsiella quasipneumoniae subsp.quasipneumoniae 01A030(T)
HG933296
99.73
Fungal Strains:
Strain no.
Closest neighbour
Accession No.
% similarity
S1
Aspergillus niger strain ATCC 16888
AY373852.1
99
A.niger
Aspergillus niger strain ATCC 16888
AY373852.1
99
Close
 
 
 
Experiment 4:
In search of potential enzymes that degrade catechu!
Literature survey and screening of various enzymes that break bonds in molecules having similar structures as that of components of catechu!
Learn More
After having the sequencing results in hand we did a thorough literature survey to find out the potential enzymes present in the isolates which could degrade components of catechu. We used softwares like BRENDA, PUBMED, UNIPROT to obtain relevant information.  The major candidates from our literature survey were tannases and catechol dioxygenases.
On these grounds we also looked at various team wikis from past years in search for similar projects. We found that Team Paris-Bettencourt has worked with catechol dioxygenases. We got in touch with them and they sent us their cloned enzymes [catechol-1,2-dioxygenase (CatA) and catechol-2,3-dioxygenase(XylE)] which we could successfully transform into DH5alpha cells. We tested these transformants on catechu and slaked lime media and the results were flabbergasting.
A distinct zone of clearance was observed around the streaks and spots of the growing culture on the catechu and slaked lime solid medium. Hence, we decided to proceed with these cultures as well our natural isolates.
Close
 
 
 
Experiment 5:
DEGRADATION - Chewing Catechu
Degradation of catechu by potential isolates and selected enzyme XylE in M9+CS broth medium.
Learn More
Other than plates we checked the degradation activity of potential isolates and enzyme XylE in broth medium.
The distinct colour difference between the control tube and potential isolates which includes Klebsiella pneumoniae, Aspergillus niger (S1 fungus and A.niger), S4 fungus can be observed.
For quantifying degradation activity of DH5α containing enzyme XylE in M9+CS broth medium we used ELISA plate reader. 10 % of volume inoculum of DH5α was inoculated in M9+CS broth of catechu stock 0.1%, 0.5% and 1% readings was taken at 415 nm on interval of 48 hrs.
Close
 
 
 
Experiment 6:
Cloning - Challenge Accepted!
Molecular cloning of the gblock fragments in pSB1C3 backbone (digestion, ligation, transformation) and their confirmation by diagnostic digest agarose gel run.
Learn More
As all cloning strategies, there were few trial and error scenarios wrt to the ratios of fragments to be used for a perfectly assembled DNA.
Ligation math for all the fragments were done and the process of digestion, ligation and transformation was carried out successfully to yield 2 well assembled constructs. The plasmid was again extracted from the selected clones on the plates and run an agarose gel electrophoresis after a single and dual cut diagnostic digest with EcoR1 and Pst1.
In the above Gel-Red stained Agarose gel, in which construct (pBAD_ompT_xylE) and construct (pBAD_xylE) are the cloned modules which have the bands at the expected size of 3278 bp or 3.3 kb and 3218 bp or 3.2 kb respectively after single EcoR1 digest.
Also faint bands of the backbone at 2 kb were obtained but the fragment DNA wasn’t well stained (maybe due to low amounts) and very faint bands of fragments of size 1.2 kb for (pBAD_ompT_xylE) as expected of 1208 bp/1.2 kb and ~1.2 kb for (pBAD_xylE) as expected 1148 bp/1.15 kb were observed.
This diagnostic digest confirms that the cloned fragments are of the desired assembly and the clones of these DNA constructs are used for further analysis by protein characterisation.
Close
 
 
 
Experiment 7:
SDS-PAGE Protein Characterisation
After analysing a successful cloning, the plasmid DNA was then transferred into DH5a cells to have growing transformants for characterising protein expression.
Learn More
From the above SDS-PAGE Coomassie Brilliant Blue stained gel it is interpreted that-
The dark protein bands are observed slightly above the 33 kDa ladder which is equal to the expected size of the enzyme catechol-2,3-dioxygenase of 35 kDa.
DH5a pBAD_xylE transformants grown in Glucose do not show any protein bands at this size and hence can be verified as negative control. While DH5a pBAD_xylE transformants grown in Arabinose show the bands at the expected size.
In presence of Glucose in the substrate, the enzyme is not synthesised, while in presence of Arabinose, the enzyme is synthesised, which is true to the fact that the construct having promoter pBAD is being induced only in presence of arabinose to synthesise the enzyme catechol-2,3-dioxygenase.
The above gel confirms that the constructs are well assembled and successfully synthesize the desired enzyme catechol-2,3-dioxygenase which is used for catechu degradation and further tested for its ability to degrade various dye substrates.
Close
 
 
 
Experiment 8:
Assay for detection of catechol-2,3-dioxygenase
An assay which will detect the presence of the enzyme catechol-2,3-dioxygenase which is a translational product of our cloned gene-XylE.
Learn More
This assay gives a direct readout of the presence of the enzyme catechol-2,3-dioxygenase. The colourless substrate catechol is broken down to a yellow coloured product 2-hydroxymuconic semialdehyde (2-HMS) and this reaction is catalysed by the enzyme catechol-2,3-dioxygenase.
Intensity of the generated yellow colour was detected in the form of absorbance values.The absorbance readouts were taken at 415 nm at an interval of 5 minutes and graphs were prepared to interpret the generated data.
The XylE gene in our construct is under the regulation of the pBAD promoter which is arabinose responsive. From the generated data we could conclude that with advancement in time, the absorbance of the system increases when the cells are grown in the presence of arabinose but remains fairly constant when grown in glucose.
Close
 
 
 
Experiment 9:
Versatility - Testing on dyes
Demonstrating versatility of construct/model to degrade common lab dyes other than paan stains.Clearance around the colony observed on plates containing several dyes.
Learn More
After gaining insight of versatility point by Ajay Bio-Tech company, we tested our construct on chemically related laboratory dyes.
We inoculated our positive clones on luria bertani medium plates of 0.025% concentration of dyes which includes congo red,methylene blue and safranin. Clearance around the colony indicating degradation was distinctly observed after 48 hrs of incubation. This concludes the activity of enzyme/ action of catechewing coli on other stains/dyes giving us future prospective to work on project.
Close
 
 
 
Our co-ordinates:
L .Nappo Road,                Matunga
Dadar East,Mumbai,
Maharashtra 400019,
India
catechewingcoli@gmail.com
FOLLOW US ON:
