Team:Ruia-Mumbai/Bibliography
ABOUT
Team Collaborations Sponsors
PROJECT
Description Design Improve Results Attributions
INTERLAB
Objective Experience Plate Reader Flow cytometry
PARTS
Parts Overview Basic Parts Composite Parts
NOTEBOOK
Daily Notes Protocols Bibliography
SAFETY
Lab Safety Product Safety
HUMAN PRACTICES
Overview Community Engagement Integated human practices Sustainibility
ACHIEVEMENTS
BIBLIOGRAPHY
http://www.academicjournals.org/article/article1421835732_Atomssa%20and%20Gholap.pdf
Characterization and determination of catechins in green tea leaves using UV-visible spectrometer
T. Atomssa* and A. V. Gholap
.Journal of Engineering and Technology Research Vol 7 (1), pp. 21-31, January 2015
https://openi.nlm.nih.gov/detailedresult.php?img=PMC4500849_12010_2015_1613_Fig1_HTML&req=4
Cytotoxic Activity of Highly Purified Silver Nanoparticles Sol Against Cells of Human Immune System.
Barbasz A, Oćwieja M, Barbasz J
Appl. Biochem. Biotechnol. (2015)
https://www.ncbi.nlm.nih.gov/pubmed/20578715
Absorption spectrum, mass spectrometric properties, and electronic structure of 1,2-benzoquinone
Albarran G1, Boggess W, Rassolov V, Schuler RH.
J Phys Chem A. 2010 Jul 22;114(28):7470-8. doi: 10.1021/jp101723s
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.462.6793&rep=rep1&type=pdf
Biodegradation of Catechin Biodegradation of Catechin
M ARUNACHALAM, M MOHAN RAJ, N MOHAN and A  MAHADEVAN
Proc. Indian natn Sci Acad. B69  B69  B69  B69  B69  No. 4 pp  353-370 (2003)
5.https://www.sciencedirect.com/science/article/pii/S1978301916302467
Fractionation and Characterization of Tannin Acyl Hydrolase from Aspergillus niger
YUNITA ARIAN SANIANWAR1I MADEARTIKA2HASIMDANURI
HAYATI Journal of Biosciences, September 2009, p 95-99 Vol. 16, No. 3 ISSN: 1978-3019
https://en.wikipedia.org/wiki/Paan
https://sites.google.com/site/knowledgebookworld/home/paan
https://www.brenda-enzymes.org/search_result.php?quicksearch=1&noOfResults=10&a=21&W[2]=Aspergillus&T[2]=2
https://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.59
https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.1
https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.24
https://www.brenda-enzymes.org/enzyme.php?ecno=1.10.3.1
https://www.brenda-enzymes.org/enzyme.php?ecno=3.1.1.20
https://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.46
http://www.brenda-enzymes.org/enzyme.php?ecno=4.1.1.63
https://www.brenda-enzymes.org/enzyme.php?ecno=1.1.5.9
https://www.brenda-enzymes.org/enzyme.php?ecno=1.13.11.2
https://www.brenda-enzymes.org/structure.php?show=reaction&id=29376&type=I&displayType=marvin
https://www.brenda-enzymes.org/search_result.php?quicksearch=1&noOfResults=10&a=21&W[2]=Aspergillus&T[2]=2
Characterization of the Erwinia carotovora pelB Gene and Its Product Pectate Lyase
SHAU-PING LEI,12 HUN-CHI LIN,12 SHAN-SHAN WANG,2 JAMES CALLAWAY,2 AND GARY WILCOX'*
Department of Microbiology, University of California
Journal of Bacteriology, September 1987, p. 4379-4383
XylE breaks down catechol to 2-HMS (2-hydroxymuconic semialdehyde) while Tannases break down tannis to gallic acid/ pyrogallol etc.
Found an already existing xylE (Imperial college 2010 wiki).
Imperial XylE http://parts.igem.org/Part:BBa_K316003
Bettencourt XylE http://parts.igem.org/Part:BBa_K2043004
DNA sequence is very different (because they "codon optimized" for E.coli so the wobble base is different) but
the protein sequence is identical to Bettencourt XylE protein except no His tag in the 2010 part.
XylE is active only as a tetramer and needs the N-term free for this association.
https://2010.igem.org/Team:Imperial_College_London/Modules/Fast_Response
Catechol (2,3) dioxygenase (C23O) is the protein product of the XylE gene. It originates from Pseudomonas putida
and the active protein is made up of a homotetramer of C2,3O monomers. The enzyme catalyses the conversion of a
colourless substrate (catechol or substituted catechols) into a bright yellow product (2-hydroxymuconic semialdehyde)
within seconds of substrate addition.
Rapid and tunable post-translational coupling of genetic circuits
Arthur Prindle1*, Jangir Selimkhanov1*, Howard Li1, Ivan Razinkov1, Lev S. Tsimring2 & Jeff Hasty1,2,3
Nature Publications Vol. 508, April 17 2014
http://www.im.microbios.org/0803/0803213.pdf
https://2013.igem.org/Team:Valencia_Biocampus
A promoter that responds to aromatic hydrocarbons
https://2013.igem.org/Team:Peking/Project/BioSensors/XylS
http://parts.igem.org/Part:BBa_I0500:Experience
An archetypical extradiol-cleaving catecholic dioxygenase: the crystal structure of  catechol 2,3-dioxygenase
(metapyrocatechase) from Pseudomonas putida mt-2 Akiko Kita1, Shin-ichi Kita1,2, Ikuhide Fujisawa1,
Koji Inaka2, Tetsuo Ishida3,Kihachiro Horiike3, Mitsuhiro
Nozaki3 and Kunio Miki1*
Structure, January 1999, Vol 7 No 1
Structure and reaction mechanism of catechol 2,3-dioxygenase (metapyrocatechase)
Tetsuo Ishida, Akiko Kita, Kunio Miki, Mitsuhiro Nozaki, Kihachiro Horiike
International Congress Series 1233 (2002) 213 – 220
https://www.ijraset.com/fileserve.php?FID=1949
Comparative Studies on Methods of Tannase Assay
Dhruvil Brahmbhatt1, H. A. Modi2
International Journal for Research in Applied Science & Engineering Technology (IJRASET)
Volume 3 Issue III, March 2015
A Spectrophotometric Method for Assay of Tannase Using Rhodanine
Shweta Sharma,* T. K. Bhat,†,1 and R. K. Dawra†
Analytical Biochemistry Vol. 279, 85-89 (2000)
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.870.1465&rep=rep1&type=pdf
Characterization of Bioimprinted Tannase and Its Kinetic and Thermodynamics Properties in Synthesis of
Propyl Gallate by Transesterification in Anhydrous Medium
Guangjun Nie & Zhiming Zheng & Guohong Gong & Genhai Zhao & Yan Liu & Junying Song & Jun Dai
Appl Biochem Biotechnol (2012) 167:2305–2317
Our co-ordinates:
L .Nappo Road,                Matunga
Dadar East,Mumbai,
Maharashtra 400019,
India
catechewingcoli@gmail.com
FOLLOW US ON:
