"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU-Denmark/Methods/Timeline
From 2014.igem.org
Technical University of Denmark
Home
Overview
Background
Experimental Design
Interlab Study
Policy and Practices
Ethics
Methods
Timeline
Protocols
Safety
Achievements
Lab Results
Modelling
Parts
Interlab Study
Policy and Practices
BioBrick Workshop
Medal Fulfillments
Team
Team Members
Official Team Page
Sponsors
Attributions
At the Giant Jamboree, we received a gold medal and an Interlab Study Award. We also won the Overgrad Best Parts Collection.
Timeline
Experimental planning
Proof of concept
Construct strains
In vitro transcription
Measurement
Construct strains
Measurement
Target group identification
and strategy
Workshop development and documentary
Workshops
Promoter Activity
Experimental Planning
After brainstorming ideas we concluded that we wanted to work with the characterisation of promoter activities. We heard about the Spinach technology and were intrigued about the potential to develop an assay for promoters activity in Polymerases Per Second (PoPS). PoPS is a measure of the amount of polymerases that passes a certain point on the DNA string. The Spinach technology involves an RNA-aptamer (“Spinach”) that binds to a ligand called DFHBI-1T, creating a fluorophore. The fluorescence from this fluorophore can be measured and will correlate with the concentration of Spinach RNA in the cell. After these considerations we decided to work with the Spinach technology and synthesised the sequence for Spinach2 (an improved version of Spinach), flanked by stabilising tRNA sequences as well as the BioBrick prefix and suffix. We modified the Spinach2 sequence to remove an internal SpeI site, resulting in a sequence we called Spinach2.1 . Unless otherwise specified, the term “Spinach” will in the following refer to Spinach2.1.
We derived a model describing the relationship between promoter activity and Spinach concentration. According to the model we needed to
Measure the degradation rate of Spinach in the cells
Measure the fluorescence from cells expressing Spinach
Create a standard series to correlate fluorescence with Spinach concentration.
We therefore designed experiments to achieve these goals. We wanted to obtain the standard curve by measuring RNA synthesised in vitro. This would allow us to add the ligand DFHBI-1T in known concentrations with excess RNA and measure the fluorescence signal. To determine the degradation rate of Spinach we wanted to grow the cells to exponential phase, halt transcription with rifampicin and measure the decline of fluorescent signal from the cells.
Finally we needed to create strains that expressed Spinach2.1 from various promoters, which we could then measure.
Proof of Concept
The Spinach2 RNA-aptamer has previously only been tested with T7-phage promoters and strong rRNA promoters. These promoters have a far greater activity than most of the promoters used in synthetic biology systems, e.g. the Anderson promoter library. Therefore we needed to do a proof of concept, proving that the fluorescence generated by Spinach2 expressed from an Anderson promoter was detectable. This proof of concept would also allow us to verify that our newly designed Spinach2.1 was working as well as Spinach2. Spinach2 and Spinach2.1 were synthesised as gBlocks from IDT. We planned to express Spinach from a fairly powerful promoter from the Anderson library and insert the construct into the iGEM registry standard plasmid backbone, pSB1C3 . The high copy number of this plasmid would ensure a high number of genes in the cell which in combination with the strong promoter would generate a high concentration of RNA in the cell. We chose the promoter BBa_J23101 and ligated this with Spinach and pSB1C3 and transformed the product into DH5α. After verifying that the construct was correct, by restriction analysis , the new strain was grown O/N at 37 degrees Celsius and the signal was measured with a fluorometer and observed fluorescence was confirmed by fluorescence microscopy . When fluorescence was confirmed from Spinach2 as well as Spinach2.1 we wanted to make a study to compare the fluorescence signal of our new version versus the original Spinach2 aptamer. We therefore did in vitro transcription of both sequences. The transcripts were then mixed with DFHBI-1T and fluorescence was measured using a fluorometer .
Strain Construction
Encouraged by our results in proof of concept we began constructing strains with 15 of the 20 Anderson promoters. Spinach was amplified from gBlocks using PCR and each BioBrick plasmid containing an Anderson promoter in pSB1C3 was cut with SpeI and PstI. The Spinach gene was ligated with each of the cut plasmids. After the ligated plasmids had been transformed into DH5α, they were purified with Zyppy miniprep kit from Zymo. The plasmids were then verified by restriction analysis. If the restriction analysis showed the expected band lengths, the plasmids were sequenced by Macrogen EZ-Seq. Strains containing plasmids verified by sequencing were added to our glycerol stock for use in future measurements.
In Vitro Transcription
We needed to establish a standard curve for fluorescence of Spinach. We therefore ordered a primer with a T7-promoter tail so we could do PCR amplification of Spinach2 and Spinach2.1. The products were purified with Qiagen Qiaquick PCR purification kit. The purified PCR product was used as template for in vitro transcription . The RNA product concentration was measured on a NanoDrop, and stored at -80 degrees celsius.
Measurement
Three types of measurements were done:
Measurement of standard curve using in vitro RNA.
Measurement of fluorescence signal from cell culture.
Measurement of Spinach degradation rate in vivo.
For measuring the standard curve, excess of in vitro transcription product was mixed with increasing known concentrations of DFHBI-1T and fluorescence was measured. A fluorescence buffer was used. This enabled us to derive a correlation between fluorescence and concentration of Spinach-DFHBI-1T complex, as well as compare the fluorescence of Spinach2 and Spinach2.1.
To measure the fluorescence signal from a cell culture containing Spinach, 10 ml LB media was inoculated with 1 ml overnight culture. When the culture reached an OD600 of approximately 1, the culture was centrifuged and the pellet was washed twice and resuspended in 1 ml 0.9% NaCl and kept on ice. The OD600 of the sample was measured and DFHBI-1T was added to a final concentration of 200μM and the samples were kept on ice for 1 hour before measurements were made in a fluorometer .
The half life of the Spinach2.1 RNA was measured by inoculating 100 ml of LB with 2 ml overnight culture. Spectrophotometric measurements were made during the exponential growth phase in order to determine the growth rate. At approximately OD600 = 1, rifampicin was added to terminate transcription in the culture, and a T0 sample was taken. Every hour a sample of 10 ml was taken and measured .
Interlab Study
Strain Construction
Being a part of the measurement track requires participation in the Interlab Study. In the Interlab Study, GFP is used as a reporter protein to measure promoters of the Anderson library. Using standard assembly BioBrick BBa_E0240 was ligated with the 15 promoters from the Anderson promoter library, which had been provided in pSB1C3 with the distribution kit. The plasmids were purified with Zyppy miniprep kit by Zymo and verified by restriction analysis . The verified plasmids were sequenced using EZ-Seq by Macrogen. Strains containing plasmids verified by sequencing were added to our glycerol stock for use in future measurements.
The reason for only applying 15 of the Anderson promoters, was that the remaining 5 was not available in the standard iGEM backbone pSB1C3 with chloramphenicol resistance. To ease future use of the Anderson library we intended to introduce the remaining 5 promoters into the iGEM standard backbone pSB1C3, see parts .
Measurement
Two measurements were performed as part of the Interlab Study:
Culture measurements
Single cell measurements with a flow cytometer
Cultures of the constructed strains, were grown and measured in a BioLector . The BioLector incubates the cultures with shake and temperature control and measures OD600 and fluorescence from each well with defined time intervals.
Measurements at single cell level was performed using flow cytometry . Cultures were made by inoculating fresh LB media with 10 μl overnight culture. The cultures were incubated at 37 °C for 3 hours and measurements were made by diluting the culture until approximately 1000 events per second were achieved.
Policy and Practices
Target Group Identification and Strategy
We wanted our public outreach to have a streamlined profile, and therefore we wanted a clear target group and objective. As synthetic biology is a fairly complex field young students are not introduced to the subject until very late in their education. This limits their opportunity to make an informed decision as to whether it should be their chosen career path. We therefore wanted to target younger students in their final years of primary education or in the secondary educational system. At this level of education the students have not yet obtained the necessary biological knowledge to fully understand synthetic biology. We therefore chose to focus our strategy on inspiration. We argued that inspiring the students is an essential step towards getting them interested in the field of synthetic biology.
Workshop Development and Documentary
To achieve our goal of inspiring younger students we decided to create a brainstorming workshop. The idea was to present high school students with a number of local and global problems and give them a “synthetic biology toolbox” which could be used to solve these problems. The toolbox was a list comprising a number of organisms, cell types, biological components (e.g. proteins, enzymes and receptors) and examples of functions of these cells and biological components. See our achievement page for further information. As another initiative we took part in documentary on biomimicry aimed at students in the final stage of primary education. In this documentary we talked about how synthetic biologists are inspired by nature to combine natural parts in new ways, to create new systems.
Workshops
We used the above mentioned toolbox to host two workshops at summer camps for talented high school students. The workshops started with us giving a small introduction to the world of synthetic biology followed by a presentation of previous iGEM projects that gives an idea of the unlimited possibilities within synthetic biology. The students were then presented with the toolbox and a list of suggested problems and asked to think of ways to solve these (or other) problems using the cells and biological components from the toolbox. The workshops ended with a follow up where the students presented their ideas. We emphasised that no answers were wrong, in order to keep the students’ minds as open as possible.
Technical University of Denmark Department of Systems Biology Søltofts Plads 221 2800 Kgs. Lyngby Denmark
+45 45 25 25 25
dtuigem14@gmail.com  .
Retrieved from " http://2014.igem.org/Team:DTU-Denmark/Methods/Timeline "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
