Team:McMasterU
McMasterU MGEM Team
McMasterU
Home
Team
Team
Collaborations
Project
Description
Experiments
Results
Notebook
Parts
Parts
Attributions
Attributions
Human Practices
Human Practices:
Overview
Human Practices:
Comparisons
Community
Engagement
Awards
Software
Model
Project Description
GI tract cancers are an enormous public health issue, and together are responsible for more deaths than any other form of cancer. One of the major difficulties with diagnosing and treating these cancers, especially when localized to the intestine, is access to the tumour - patients often do not exhibit obvious symptoms until the later stages, when treatment options are limited.
We looked into a few different ways of tackling this issue, from improving existing diagnostic methods to developing new avenues for treatment. One of the consistent themes that came up in our brainstorming process was the potential for us to use the host's cell-mediated immune response to fight cancer. Cancer immunotherapy is a rapidly growing field, and clinical trials for certain forms of leukemia are already underway.
Thus, this year, McMaster iGEM sought to augment the power of the host immune system to fight against GI tract cancers, using a specially engineered strain of commensal lactobacillus bacteria. When completed, our bacteria will be able to sense the presence of tumours in the gut, bind to specific receptors on tumour cells, and begin secreting pro-inflammatory cytokines in this tumour microenvironment. This will recruit T cells to the site of the tumour and elicit an anti-cancer response, effectively stopping the cancerous growth using the body's own toolkits. This summer, we aim to develop a proof-of-concept of our idea, and create a bacterial strain that can secrete IL-2 under tightly controlled, tumour-specific conditions
This project was worked on by many undergraduate students in the McMasterU iGEM 2016 Team, with backgrounds ranging from engineering to health sciences to biotechnology to integrated sciences. We have listed our sub-teams and our members alphabetically below:
Wet Lab Project Design, DNA Constructs Design, and Benchwork: Chirayu Bhatt, Karanbir Brar, Tony Chen, Yosef Ellenbogen, Melodie (Na-yoon) Kim, Vivian Lau, Dhanyasari Maddiboina, Maxwell Ng, Yu Fei Xia
Website, Graphics Design, and Public Relations: Tony Chen, Farhaan Kanji, Jinny Lee, Nicolas Lehman, Maxwell Ng, Candy Niu, Yu Fei Xia, Kaijie Zhang
Computational and Software Lab: Ian Fare, Nicolas Lehman, Lauren Liu, Maxwell Ng, Christine Wang, Kaijie Zhang
Human Practices: Mohammed Ahmed, Karanbir brar, Yosef Ellenbogen, Melodie (Na-yoon) Kim, Jinny lee, Clara Long, Hwa Young (Haley) Yun
Community Outreach: Mohammed Ahmed, Chirayu Bhatt, Tony Chen, Yosef Ellenbogen, Vivian Lau, Clara Long, Ramneet Mann, Maxwell Ng, Rui Sun, Hwa Young (Haley) Yun
Administration and External Liaisons: Mohammed Ahmed, Beenish Ajmal, Karanbir Brar, Farhaan Kanji, Samaher Ramzan, Michael Xie
We have been supported by our Primary and Secondary PIs Dr. Rosa da Silva and Ms. Alison Cowie, who we would like to thank for guiding us and giving us the platform to investigate our research questions and conduct our project. We have also been advised greatly by Dr. Surette and the Farncombe Institute, as well as Alastair Tracey from  the McMaster University Biology Department. It took an extraordinary amount of collaboration and support and we could not have done it without them.
Understanding that we cannot change the Wiki between years, we have created an additional website where we announce events and blog posts, as well as update our membership each year. You can check it out here at mcmastergem.com
In the past May of 2016, mGEM collaborated with Hamilton Health Sciences to present Discovery Day at McMaster. This constitutes a day filled with workshops where secondary school students are able to explore a wide variety of careers in medicine and health sciences. One of the workshops lead by mGEM allowed students to apply their basic knowledge of cellular structures to a common laboratory practice where they were able to stain and visualize the cells. The second workshop reviewed the basic structure of DNA approached in the science curriculum, and further applied this to characteristically separate DNA in PCR, a common laboratory practice. This collaboration between mGEM and Hamilton Health Sciences further strengthened our efforts to educate other students on synthetic biology and its scientific relevance in the working world today. The reputability of Hamilton Health Sciences was effective in attracting secondary school students for the workshops prepared by mGEM.
Gastrointestinal (GI) cancers have one of the highest cancer mortality rates in Canada due to difficulties in tumour diagnosis and treatment1. Common anti-cancer therapies used today can lack specificity and have off-target effects, damaging the body in the process2. Our goal is to develop the foundation for a novel form of GI cancer immunotherapy that exploits commensal gut bacteria to specifically target GI tumour cells for destruction by the immune system. Our bacteria of choice is Lactobacillus brevis, which has been designed to bind and aggregate towards Her2+ GI tumours through the use of a cell-wall anchored, anti-Her2 DARPin. Like antibodies, DARPins constitute high-specificity interactions that have the potential to mitigate many off-target effects3. With sufficient bacterial density, a quorum sensing mechanism (Lux and Las) triggers the production of a vital T cell activating cytokine, interleukin-2 (IL-2), to recruit tumour-specific T cells for an anti-cancer response. Future steps will include simulations of the bacteria in a possible tumour environment, which will help drive quorum-sensing design (e.g. promoter sensitivity).
Ethical concerns were addressed through research into the feasibility of such a treatment, particularly into methods to minimize health risks, make the treatment financially accessible, and abide relevant legislation.
References: (click to view)
Canadian Cancer Society’s Advisory Committee on Cancer Statistics. Canadian Cancer Statistics 2015. Toronto, ON: Canadian Cancer Society; 2015.
Dy, G. K.; Adjei, A. A. CA. Cancer J. Clin. 2013, 63 (4), 249–279.
Stumpp, M. T.; Binz, H. K.; Amstutz, P. Drug Discov. Today 2008, 13 (15), 695–701.
Experimental overview:
Thus far, our project has developed a successful PCR amplification protocol for both the Lux and Las quorum sensing systems. Both parts were de novo synthesized from Integrated DNA Technologies via funding from iGEM. Future experimentation will serve to validate the function of these two quorum sensing systems through the concomitant expression of the mCherry red fluorescent protein (BBa_J06504) upon activation of the quorum sensing pathway.
A PCR protocol was also successfully developed for the amplification of a de novo synthesized IL-2 gene. The gene was modified to be biobrick compatible, however the expression of the IL-2 within a bacterial system is pending characterization. Characterization will be completed through the insertion of the gene into a pET26b expression vector that contains both an inducible T7 lac promoter, and a His(6) tag prior to the termination sequence1. This would allow for inducible expression of the cytokine in compatible bacterial system using Isopropyl β-D-1-thiogalactopyranoside (IPTG), and verification using an anti-His Western Blot2. The IL-2 gene was synthesized to be compatible with our L.brevis compatible plasmid, pIB184. This would allow for expression of the cytokine in our model bacterium.
Our L.brevis compatible plasmid, pIB184, was obtained through the Mahony Lab located in St. Joseph’s Healthcare Hamilton. The Kim (2005) protocol was used from http://openwetware.org/ to stimulate our L. brevis cells to become electrocompetent, allowing for their transformation with pIB184, which was done through an electroporation protocol3-4.
pIB184 contains a gene that confers erythromycin resistance to an unknown degree5. We assayed several concentrations of erythromycin in testing for the degree of bacterial growth inhibition, whereby we determined that (insert concentration) of erythromycin allowed for growth of pIB184 transformed L. brevis and inhibited the growth of competing bacteria.
Lastly, we designed the gene sequence of a Gram-positive cell wall anchoring protein that was fused with an anti-Her2 DARPin (CWA-Her2-DARPin), which would be expressed at the L. brevis cell surface to allow for binding to Her2 positive cancer cells.
These successes are only the preliminary steps in the grand scheme of our project. We hope to continue with characterizing our IL-2 expression, and Lux and Las quorum sensing systems. These would be transformed into our L. brevis cells along with the gene for CWA-Her2-DARPin. We will finally test our bacterial system in a human cancer tissue culture and assay for T cell recruitment.
Protocols:
PCR Amplification of de novo synthesized genes from IDT
Lux and Las amplification, and IL-2 amplification, modifying to be compatible with both pET26b and pIB184
For the Lux and Las quorum sensing systems, each system was ordered as two gblocks (sequence verified, double-stranded genomic blocks) from IDT. The goal was to amplify the two gblocks (which comprised the whole system) individually. The gblocks were designed to have an overlap region at one end. The gblocks would then be allowed to anneal and extend, forming a complete Las or Lux circuit.
Provide gene sequences, primer sequences, melt temps, G+C content
Provide PCR temperatures - melting, annealing, extending - number of cycles
Designing DARPin-CWA fusion protein
Designing this and modifying it to be compatible with both pET26b and pIB184
Provide gene sequence
L. brevis electrocompetent cells
First, the following concentrated stock solutions were made. Concentrations, dry molecular weight, and the total mass added per given volume are listed.
1. NaH2PO4 = 200mM (1.2g in 50mL)
v = 50mL
MW = 119.98g/mol
m = 1.1998g = 1.2g
2. MgCl2 = 50mM (0.2380g in 50mL)
v = 50mL
MW = 95.211g/mol
m = 0.2380g
3. Sucrose = 2M (34.2296g in 50mL)
solubility at room temp: 2g/mL = 0.006mol/mL = 6M
v = 50mL
MW = 342.2965g/mol
m = 34.2296g
4. Glycine = 20% (10g in 50mL)
Based on Kim (2005) protocol, optimized for L. brevis:
5. Concentrated Wash (10x): 100mL
50mM NaH2PO4 —> Use 25mL of Stock
10mM MgCl2 —> Use 20mL of Stock
Add 55mL ddH2O
6. Large Volume of Stock Buffer (1x, use 5mL at a time): 50mL
1M Sucrose —> Use 25mL of Stock
3mM MgCl2 —> Use 3mL of Stock
Add 22mL ddH2O
After stock solutions are made, the following protocol was used to create the electrocompetent L. brevis cells.
Reagents:
L.brevis 884 overnights
Wash Buffer (50mM NaH2PO4, 10mM MgCl2), on ice
Electroporation Buffer (1M Sucrose, 3mM MgCl2), on ice
MRS media (1% glycine added)
Dry Ice
Procedure
400mL MRS media was inoculated with 8mL L.brevis overnight culture and grown at 30C with 75rpm shaking until OD600 = 0.2 (~2.5h)
A sample of the culture was gram stained to verify presence of L. brevis
Culture was split into 10 aliquots of 40mL and cooled on ice for 10 mins
Aliquots were centrifuged at 4C at 2900xg for 5 mins
Supernatant was discarded and pellets were resuspended in 40mL each of wash buffer by vortex at low speed
Cells were centrifuged to pellet as before
Supernatant discarded; resuspension in 20mL each of wash buffer
Cells were centrifuged to pellet as before
Supernatant discarded; resuspension in 10mL each of wash buffer
Aliquots were combined into 2 tubes of 50mL each
Cells were centrifuged to pellet as before
Supernatant discarded; resuspension in 2mL each of electroporation buffer
40 aliquots of 100uL were made into prechilled cryotubes and flash frozen on dry ice
Cells were stored at -80C
Verification of tests and contamination
MacConkey (Mac) Plate: To check for Gram-negative bacteria growth   Procedure: Added growth solution to Mac Plate; used several (aim for use of 5) glass beads as spreaders; dried, then dumped glass beads in ethanol for sterilization; flipped and marked the plate, and placed in 37 degree incubator. Growth of Gram-negative bacteria invalidates experiment, as L. brevis is Gram-positive.
pIB184 electroporation into L. brevis
Use two aliquots of L. brevis chemically competent cells
Add 3uL of pIB184 to one aliquot
Add 3uL of nuclease free water to another aliquot as a control
Allow to stand on ice for 10 minutes
Place two electro-cuvette in their wrappers into ice (ensure that the cuvettes do not touch the ice as this could cause sparking)
Mark one cuvette for DNA and one for Control, and add the cells respectively
Before electroporating, use a kimwipe to be sure the cuvettes are completely dry on all sides and bottom
Electroporate at 12.5 kV/cm (pulse number = 10, pulse interval = 500 ms)
Add 1mL of MRS broth to each cuvette for recovery, pipette up and down gently to mix
Transfer the recovery solutions into two separate 1.5mL microcentrifuge tubes
Place the tubes in 37 degrees for 3 hours
Plate the cells on MRS plates with 8ug/mL erythromycin
Determining erythromycin concentration to allow for growth of L. brevis transformed with pIB184, and inhibit competing bacterial growth
In order to determine the appropriate erythromycin (antibiotic) concentration required for optimal growth of L. brevis, MRS agar plates of a range of concentrations were created. MRS agar favours the growth of Lactobacilli.
A 7% solution was made using MRS agar powder and dH2O
The solution was heated (medium heat) and stirred until homogenous.
The solution was autoclaved for 15 min.
After the solution cooled down (was only slightly warm) erythromycin powder was added to create MRS- erythromycin of concentrations 4 ug/ml, 8 ug/ml, and 12 ug/ml.
Solution was poured into plates.
Two 100uL aliquots of electrocompetent L. brevis cells were recovered (no electroporation) into 5mL of MRS broth.
After 3hr incubated at 30C, it was concentrated (centrifugation and removal of supernatant) and then plate onto the 3 different Ery
concentrations (one plate each).
All plates incubated at 33C for overnight.
References: (click to view)
pET-26b(+) - Addgene Vector Database (Plasmids, Expression Vectors, etc) https://www.addgene.org/vector-database/2563/ .
Dubendorf, J. W.; Studier, F. W. J. Mol. Biol. 1991, 219 (1), 45–59.
Electro-transformation of Lactobacillus spp. OpenWetWare. http://openwetware.org/wiki/Electro-transformation_of_Lactobacillus_spp.#Kim_.282005.29 .
Kim, Y. H.; Han, K. S.; Oh, S.; You, S.; Kim, S. H. J. Appl. Microbiol. 2005, 99 (1), 167–174.
Porcellato, D.; Frantzen, C.; Rangberg, A.; Umu, O. C.; Gabrielsen, C.; Nes, I. F.; Amdam, G. V.; Diep, D. B. Genome Announc. 2015, 3 (2), e00144-15.
Results
We determined the optimal erythromycin concentration for E. coli and L. Brevis containing the PIB184 plasmid with erythromycin resistance. For E. coli, a concentration of 8 ug/ml ended up being optimal. For L. brevis, 0.25 ug/ml was determined to be the best.
Initially, Lux and Las systems were transformed from the 2016 kit. However, the number of ligations required to build each system was too high. We instead designed gblocks (synthesized DNA pieces with our desired sequence) for the Lux and Las system. Our first set of primers proved to be faulty and amplification of the gblocks did not work.
New primers were designed, and amplification of the Lux and Las constituent gblocks was successful. However, all attempts at stitching the gblocks together in a stitch PCR protocol failed.
This shows the successful Las1 gblock amplification
An ELISA was attempted to characterize IL-2 production in E. coli first. The results proved inconclusive.
Future Plans
Our future plans involve successfully stitching the gblocks together to create a complete Las and Lux system. These will be characterized by using mCherry as the reporter. Optimizing the ELISA assay for detecting IL-2 is pertinent.
Gel results of PCR amplifications for IL-2 and Las/Lux systems
3 gels (one for each, unless two successful results are shown on a single gel)
Discuss why the result is valid based on band size
We could work on using pics of plates with different concentrations of Ery and see where transformed L. brevis grew for ery concentration - tie this in to confirmation of successful electroporation
These pictures could be taken tomorrow if they are still in the fridge?   Otherwise we could just discuss results in text
For failures we can discuss our failed PCR attempts, Failed CWA submission, Failed electroporation attempts
BBa_K1929300
Interleukin-2 (IL-2) with designed RBS
Interleukin-2 (IL-2) protein coding sequence. A designed RBS with spacer DNA was added preceding the IL-2 for protein expression. This biobrick can be inserted after promoter of choice. IL-2 will be ultimately incorporated into the Las and Lux quorum sensing systems. With sufficient stimulation of the quorum sensing mechanism by an aggregation of bacteria, IL-2 production will occur and be released by the bacteria into the extracellular environment. Il-2 is a signalling cytokine and stimulates the production and differentiation of T cells (adaptive immunity). There is a silent point mutation at base pair insert 272 (A>T) to make the gene BioBrick compatible (XbaI site was removed).
This project was worked on by many undergraduate students in the McMasterU iGEM 2016 Team, listed here by sub-team and alphabetically:
Wet Lab Project Design, DNA Constructs Design, and Benchwork: Chirayu Bhatt, Karanbir Brar, Tony Chen, Yosef Ellenbogen, Melodie (Na-yoon) Kim, Vivian Lau, Dhanyasari Maddiboina, Maxwell Ng,  Yu Fei Xia
Website, Graphics Design, and Public Relations: Tony Chen, Farhaan Kanji, Jinny Lee, Nicolas Lehman, Maxwell Ng, Candy Niu, Yu Fei Xia, Kaijie Zhang
Computational and Software Lab: Ian Fare, Nicolas Lehman, Lauren Liu, Maxwell Ng, Christine Wang, Kaijie Zhang
Human Practices: Mohammed Ahmed, Karanbir brar, Yosef Ellenbogen, Melodie (Na-yoon) Kim, Jinny lee, Clara Long, Hwa Young (Haley) Yun
Community Outreach: Mohammed Ahmed, Chirayu Bhatt, Tony Chen, Yosef Ellenbogen, Vivian Lau, Clara Long, Ramneet Mann, Maxwell Ng, Rui Sun, Hwa Young (Haley) Yun
Administration and External Liaisons: Mohammed Ahmed, Beenish Ajmal, Karanbir Brar, Farhaan Kanji, Samaher Ramzan, Michael Xie
We would like to thank all our sponsors and mentors for guiding us and giving us the platform to investigate our research questions and conduct our project. It took an extraordinary amount of collaboration and support and we could not have done it without you.
Dr. Rosa da Silva, Ms. Alison Cowie, Mr. Alastair Tracey, and the McMaster University Biology Department, for their support, guidance, provision of reagents, and lab space.
Dr. Michael Surette and the Farncombe Institute for providing us with lab space, L. brevis strain and reagents to work with.
Dr. Dawn Bowdish and the McMaster Immunology Research Centre, for providing us with lab space and tissue culture reagents.
Dr. Indranil Biswas from the University of Kansas for providing the pIB184-Km plasmid.
Hamilton Health Sciences
Integrated DNA Technologies
McMaster Student Union USIF
Faculty of Science
Faculty of Health Sciences, BHSc Program
Department of Biochemistry and Biomedical Sciences
Biotechnology Program
Introduction
As we develop our IL-2 delivery mechanism, we have considered the ethical, practical and legal implications of implementing such treatment in the current Canadian oncology institutions. How reasonable is it to mass produce for regular usage? Who will be paying? How will it be administered? Who will regulate it? How secure is our product?
First, we introduce our prospective product: the lactobacilli in a pill form, similar to a probiotic pill.
Product Description:
The pill will contain our engineered bacteria condensed within a microcapsule to protect them from the upper GI tract and help localize to the colon. Specifically, we are aiming to use resistant starch which cannot be metabolized by pancreatic enzymes but is fermented by gut microflora in the colon, releasing our bacteria.1 In addition, starch is non-toxic and a normal component of human diet, making it less likely to cause adverse reactions. Additionally, we know from past work that lactic acid bacteria that is protected by the potato starch granules can survive for 6 months in room temperature, and 18 months when frozen.1 This indicates relatively moderate shelf life if our product is to be mass produced and distributed over the country, in a variety of environmental conditions.1
Cost in Mass Production?
To offset the high cost of growing a large amount of bacteria for medical applications, it would be beneficial to purchase effluent waste products from agricultural processing of grains and vegetables as cultivation media for our bacteria. These waste products contain high levels of proteins, carbohydrates, sugars, and vitamins, and thus can be used as low-cost growth media for lactic acid bacteria, including the species of lactobacillus that will act as our treatment.2 This reduction in production cost will assist in keeping the treatment burden low for the patients.
Dosage deliberations:
Determining an appropriate dosage of the bacteria that provides an optimal level of IL-2 production in GI tract cancer patients is an important consideration, one that will require several rounds of animal trials and phase 1 clinical trials in order to perfect. Prior work using lactobacilli in the treatment of IBD has used a dosage of two tablets a day containing 1010 colony-forming units (CFU), for a period of six to nine months.3, 4 As our treatment would not require substantial colonization of the gut as in IBD trials, we predict a lower dosage, ideally at one tablet a day. However, as bacteria have never before been used in cancer treatment, dosing experiments on animal models are required before any reliable approximations of human dose can be made.
Concerns on Safety:
Lactobacilli as delivery:
Lactobacilli, including Lactobacillus brevis, are “generally regarded as safe” worldwide.  Support for this stems from its long history of use in food products and from clinical studies examining the safety of many lactobacilli strains.5 In fact, this safety status is considered one of the major benefits of using lactobacilli as a vector for the therapeutic targeting of GI pathologies.6
IL-2 as therapy:
IL-2 is a regulatory cytokine, and too much of it in the body can have severe adverse effects. The most common manifestation of IL-2 toxicity is capillary leak syndrome, when fluids within the vascular system leaks into the tissue outside the bloodstream.7 This results in low blood pressure and poor blood flow to the internal organs. Capillary leak syndrome is characterized by the presence of 2 or more of the following 3 symptoms; low blood pressure, swelling, and low levels of protein in the blood.7 During the treatment phase, it is vital that patients are monitored and regularly screened for symptoms that is common to IL-2 toxicity.
Who should be excluded?
The therapy requires the body to able to respond to IL-2. Therefore, patients with IL-2 receptor deficiency would be unaffected by this treatment. Additionally, patients undergoing total body irradiation as part of a bone marrow transplant would be unable to respond to this treatment. This treatment is limited to tumours present in the intestinal lumen, and not within the intestinal tissues. Furthermore, consideration must be made to individuals with any type of immunodeficiency disorder as they might not respond optimally. Deliberations must be made case-by-case.
Frequency of treatment:
After entering the gut, lactobacillus brevis leaves 24 hours later.8 Therefore this pill will have to be administered daily to maintain constant level of therapy.
Monitoring therapy:
This pill is designed so that the patients would be able to self-administer, without the need of entering the hospital. However, we highly recommend strict regular check-ups for early detection of adverse effects and progression of disease.
To check for the presence of our bacteria in the gut, we will collect and culture patient stools during weekly follow-up visits after commencing the treatment.3 In addition, blood assays testing for the level of colon cancer tumor markers will be carried out both prior to and at monthly intervals during treatment to assess its effectiveness9. Magnetic resonance imaging (MRI) will also be used at regular intervals to assess tumor size and remission. Patient will be asked series of questions that screen for symptoms of IL-2 toxicity, and blood assays will test for the localization of IL-2 release, to be isolated from the rest of the body.
Ethical Considerations:
Our product must follow the Canadian Medical Association’s code of ethics (last reviewed 2015) as radical therapeutic to a vulnerable population group.10 This therapeutic approach will be artificially modifying the levels of cytokine IL-2 in patients, many with affected immune system. Although safety mechanisms are designed in the bacteria to limit the release of IL-2 to specific cancer regions (see wet-lab), multiple follow-up trials must be conducted before this drug can be put into large-scale practice. These protocols must be clearly defined and standardized. Doctor - patient communication is an important factor to consider when introducing any novel approach to cancer therapy. Before the treatment takes place, the patient must be fully informed of the risks and possible side-effects before obtaining consent.
Outside of practice, this product as a commercialized object is a point of discussion. This therapeutic approach is devised as a pill form that is taken outside of a hospital. Therefore, the cost falls on the patients, not on the government’s health coverage, OHIP (in the province of Ontario, Canada).11 As a result, this potential therapy may be inaccessible to patients with little finances. Regulations or subsidies on behalf of the government could elevate this issue.
Regulations:
Health Canada, the governing body that regulates probiotic use in Canada, has permitted the use of lactobacilli strains, including Lactobacillus brevis, in many products.12
Probiotics used for therapeutic purpose are classified as either natural health products (NHP) sold over the counter under the Natural Health Products Regulations, or as prescription drug under the Food and Drugs Act.13,14 Depending on the intended use, delivery and composition of the product, different regulations apply15.
Health Canada, specifically the Health Products and Food Branch, is also responsible for the regulation of products derived from living sources including cytokines through the Biologics Program. Depending on the type of the biopharmaceutical, different centres carry part of the responsibilities of the Biologics Program. Specifically for cytokines, the Centre for Evaluation of Radiopharmaceuticals and Biotherapeutics reviews and evaluates the products and documents to assure safety and quality, and performs further testing for research.16 The Centre establishes  regulations and standard operating procedures of such products, and ensures that the clinical trials conducted were properly designed and had no serious adverse reactions. They also conduct pre-market review of the products and their labels, authorizing for sale in Canada. Even after the products have been released into the market, Health Canada assesses complaints, adverse events, and perform Health Risk Assessments, as part of the Lot Release Program.17 They provide correct information about the product to health care practitioners, consumers, and the general public. Furthermore, they visit product manufacturing facilities for evaluations. Lastly, they conduct further research on the methods and the science of the products through academic collaboration nationally and internationally.18
Security:
In itself, Lactobacilli is considered to be non-pathogenic.5 Furthermore, because of the acidity of the stomach and enzymes present, it is likely that multiple dose is required for any effect to be seen. Therefore, the potential for this product to be an effective weapon is extremely low. However, in the sight of biosecurity, measures must be taken so that the probiotic is kept in a designated, safe place during research settings. Precautions should be taken to ensure it cannot be used for malicious purposes.
References: (click to show)
Anal AK, Singh H. Recent advances in microencapsulation of probiotics for industrial applications and targeted delivery. Trends in Food Science & Technology. 2007 May 31;18(5):240-51.
Figueroa‐González I, Quijano G, Ramirez G, Cruz‐Guerrero A. Probiotics and prebiotics—perspectives and challenges. Journal of the Science of Food and Agriculture. 2011 Jun 1;91(8):1341-8.
Gupta P, Andrew H, Kirschner BS, Guandalini S. Is Lactobacillus GG helpful in children with Crohn’s disease? Results of a preliminary, open-label study. Journal of pediatric gastroenterology and nutrition. 2000 Oct 1;31(4):453-7.
Gionchetti P, Rizzello F, Venturi A, Brigidi P, Matteuzzi D, Bazzocchi G, Poggioli G, Miglioli M, Campieri M. Oral bacteriotherapy as maintenance treatment in patients with chronic pouchitis: a double-blind, placebo-controlled trial. Gastroenterology. 2000 Aug 31;119(2):305-9.
Snydman DR. The safety of probiotics. Clinical infectious diseases. 2008 Feb 1;46(Supplement 2):S104-11.
Bermúdez-Humarán LG, Aubry C, Motta JP, Deraison C, Steidler L, Vergnolle N, Chatel JM, Langella P. Engineering lactococci and lactobacilli for human health. Current opinion in microbiology. 2013 Jun 30;16(3):278-83.
IL-2 - Drug Information [Internet]. Chemocare.com. 2016 [cited 7 July 2016]. Available from: http://chemocare.com/chemotherapy/drug-info/il-2.aspx
Delgado S, Flórez AB, Mayo B. Antibiotic susceptibility of Lactobacillus and Bifidobacterium species from the human gastrointestinal tract. Current microbiology. 2005 Apr 1;50(4):202-7.
Patient Guide to Tumor Markers [Internet]. Oncolink.org. 2016 [cited 8 July 2016]. Available from: http://www.oncolink.org/treatment/article.cfm?id=296
Canadian Medical Association. CMA code of ethics (update 2004). Ottawa, ON: Canadian Medical Association; 2004.
Taylor, Paul. 2015. "I Live In Ontario And Can’T Afford My Cancer Drug. What Can I Do?". Healthydebate.Ca. http://healthydebate.ca/personal-health-navigator/cancer-drug-Ontario.
Sanders ME, Akkermans LM, Haller D, Hammerman C, Heimbach JT, Hörmannsperger G, Huys G. Safety assessment of probiotics for human use. Gut microbes. 2010 May 1;1(3):164-85.
Questions and Answers on Probiotics - Food Labelling [Internet]. Health Canada Hc-sc.gc.ca. 2016 [cited 12 July 2016]. Available from: http://www.hc-sc.gc.ca/fn-an/label-etiquet/claims-reclam/probiotics_qa-qr_probiotiques-eng.php
Natural and Non-prescription Health Products - Drugs and Health Products [Internet]. Health Canada Hc-sc.gc.ca. 2016 [cited 16 August 2016]. Available from: http://www.hc-sc.gc.ca/dhp-mps/prodnatur/index-eng.php
Natural and Non-prescription Health Products - Drugs and Health Products [Internet]. Health Canada Hc-sc.gc.ca. 2016 [cited 12 August 2016]. Available from: http://www.hc-sc.gc.ca/dhp-mps/prodnatur/index-eng.php
Evaluation of the Biologics Program 1999-2000 to 2012-2013 [Internet]. Health Canada Hc-sc.gc.ca. 2016 [cited 16 July 2016]. Available from: http://www.hc-sc.gc.ca/ahc-asc/performance/eval/biologics-biologiques-eng.php
Guidance for Sponsors: Lot Release Program for Schedule D (Biologic) Drugs [Health Canada, 2005] [Internet]. Hc-sc.gc.ca. 2016 [cited 10 May 2016]. Available from: http://www.hc-sc.gc.ca/dhp-mps/brgtherap/applic-demande/guides/lot/gui_sponsors-dir_promoteurs_lot_program-eng.php#a1_0
Preclinical Safety Evaluation of Biotechnology-Derived Pharmaceuticals [Internet]. Health Care Hc-sc.gc.ca. 2016 [cited 10 July 2016]. Available from: http://www.hc-sc.gc.ca/dhp-mps/prodpharma/applic-demande/guide-ld/ich/securit/s6(r1)-step4etape-eng.php
Introduction
The following are project studies (“cases”) noted by the members of the Human Practices subcommittee. Each “Case” will examine studies that has some relevance to certain aspects of Human Practices that we recognize within our own project. We have treated this side study as training exercises - as the means of brainstorming additional points that must be considered for the implementation of our product in the real world, based on the current projects ongoing in the synthetic biology field.
CASE 1: PHEnominal - patent criteria
In 2013, a group of researchers (Deming et al) filed a claim for a genetically modified probiotic, PHEnominal, to be used in the treatment of phenylketonuria (PKU) disease. PHEnominal is genetically modified to express phenylalanine ammonia-lyase, an enzyme that degrades phenylalanine, and thus can be used to treat pku patients who are unable to digest this amino acid themselves. They published their patent just six months later, in 2014.1 While these researchers are based in the United States, they had also decided to undergo the Canadian patent process. This process, overseen by the Canadian Intellectual Property Office, is experienced by many researchers, and it may apply to us depending on the outcomes of our project. Patents offer researchers the opportunity to protect and profit from their inventions, and they have the broader implication of promoting scientific advancement for the benefit of society.2
PHEnominal, as a genetically modified probiotic therapy, can easily be compared to our project. Thus, we can learn a lot from the experiences of these researchers. In order to publish their patent, Deming et al. would have had to submit a patent application and pay the appropriate fees. The application must include experimental results, drawings and sketches, and an explanation of the product. Deming et al have included their pHENOMM vector as a representative drawing in their application. Due to the nature of our project, we would likely do the same. It is also necessary to write a clear claim, which defines the legal boundaries of the invention.2 PHEnominal claims as a composition to treat PKU, comprising of Lactobacillus reuteri and a gene expressing an enzyme to degrade phenylalanine. The claim also specifies about the pHENOMM vector, gene sequence, and the drug administration.1
In the description of their product, the researchers establish the patent criteria for novelty, utility, and inventiveness.2 They discuss advantages over conventional treatments for PKU, possible administration, and construction of the probiotic. It has been proposed that PHEnominal can be administered orally in different forms, depending on patient circumstance. In comparison to synthetic foods, the proposed classification of this product as a “drug” may also make it more accepted by insurance companies, thus contributing to affordability. These are all factors we should consider in our project.
Government of Canada IC. Canadian Patent Database / Base de données sur les brevets canadiens [Internet]. 2015 [cited 2016 Oct 5]. Available from: http://brevets-patents.ic.gc.ca/opic-cipo/cpd/eng/patent/2880620/summary.html
Government of Canada I. Tutorial — Your patent application [Internet]. [cited 2016 Oct 5]. Available from: https://www.ic.gc.ca/eic/site/cipointernet-internetopic.nsf/eng/wr01398.html
CASE 2: Laws and regulations
In 2015, a group of researchers engineered commensal bacteria Lactobacillus gasseri to secrete glucagon-like peptides-1 (GLP-1), which reprogram the intestinal cells into cells that secrete insulin in response to glucose as a possible treatment for hyperglycemia. They looked at the effects of feeding the bacteria to diabetic rats and found increased levels of insulin and greater tolerance to glucose, indicative of the bacteria’s potential therapeutic effects against diabetes.2 In addition, researchers were also able to genetically modify commensal bacteria to secrete keratinocyte growth factor-2 (KGF-2) as a new drug delivery system to treat colitis. KGF-2 has been shown as possible therapeutic agent for inflammation in the past. The bacteria were fed to mice with colitis, and the results showed a therapeutic effect in reducing the symptoms, such as weight loss and rectal bleeding.5
The laws that we have considered for our project would apply to other research of similar types. Our project used Lactobacillus brevis, which is a generally-recognized-as-safe (GRAS) species, which have a long history of safe use and hence, exempted from the typical regulations by Health Canada. Research that uses an uncommon species of bacteria, such as Bacteroides ovatus, are classified as a type of a pathogen, and hence would have to be approved by the Minister of Health Canada that the species is not unsafe for use.1 This is the process that we would have followed if we have not chosen to use a GRAS species such as Lactobacilli.
Our project aims to genetically modify probiotic bacteria to treat gastrointestinal cancer. Any other research that engineers probiotic bacteria would follow the regulations that we have considered for our project. Most research with engineered probiotic bacteria observes the effects of administering the bacteria orally, as probiotic bacteria should enter the digestive tract. For example, researchers engineered commensal bacteria Lactobacillus gasseri to secrete glucagon-like peptides-1 (GLP-1) and looked at the treatment’s effect on diabetic mice by feeding them with the bacteria.2 Similarly, our project would be in the form of orally-administered pills if it was to become a product. The final products of such research or projects would be governed by the same laws, as they would both be considered as drugs or therapeutic products under the Canadian Food and Drugs Act. Unlike pure probiotics in the form of pills, which are considered as natural health products and regulated under the Natural Health Product Regulations, genetically-engineered bacteria with therapeutic properties, are considered as drugs.3 The Food and Drugs Act of Canada is different from the similarly-named Food and Drug Administration of the United States of America.
Another aspect that we have considered for our project is the laws and regulations of the secreted biological molecules, cytokines. We found that all products, including therapeutics, that are derived from living organisms are regulated through the Biologics Program, which includes cytokines.4 This law would then apply to many of the possible products that may arise from engineered bacteria that secrete therapeutic substances for the treatment of diseases. Many of the substances are biological and hence, derived from organisms. For example, researchers have genetically modified commensal bacteria to secrete keratinocyte growth factor-2 (KGF-2) for treatment of colitis. If these bacteria were to be developed into a product, it would be controlled under the Biologics Program, similarly to our project.
Public Health Agency of Canada. Pathogen Safety Data Sheets and Risk Assessment [Internet]. 2011 [cited 2016 Aug 15]. Available from: http://www.phac-aspc.gc.ca/lab-bio/res/psds-ftss/index-eng.php
Duan FF, Liu JH, March JC. Engineered Commensal Bacteria Reprogram Intestinal Cells Into Glucose-Responsive Insulin-Secreting Cells for the Treatment of Diabetes. Diabetes [Internet]. 2015 May;64(5):1794–803. Available from: http://diabetes.diabetesjournals.org/lookup/doi/10.2337/db14-0635
Government of Canada. Food and Drugs Act [Internet]. 2016 [cited 2016 Aug 15]. Available from: http://laws-lois.justice.gc.ca/eng/acts/F-27/page-1.html
Health Canada and the Public Health Agency of Canada. Evaluation of the Biologics Program. 2014;(May). Available from: http://www.hc-sc.gc.ca/ahc-asc/alt_formats/pdf/performance/eval/biologics-biologiques-eng.pdf
Hamady ZZR, Scott N, Farrar MD, Lodge JPA, Holland KT, Whitehead T, et al. Xylan-regulated delivery of human keratinocyte growth factor-2 to the inflamed colon by the human anaerobic commensal bacterium Bacteroides ovatus. Gut [Internet]. 2010 Apr 1;59(4):461–9. Available from: http://www.researchgate.net/publication/26795250_Xylan-regulated_delivery_of_human_keratinocyte_growth_factor-2_to_the_inflamed_colon_by_the_human_anaerobic_commensal_bacterium_Bacteroides_ovatus
CASE 3: Biomass using wheat stillage and sugar beet molasses - mass production
There is much established work in the area of mass-producing bacteria for medical purposes, primarily focused on probiotics. The large-scale growth of lactic acid bacteria is commonplace, which provides established protocols that we can follow to produce our IL-2 secreting lactobacilli. Lactic acid bacteria (LAB) like lactobacilli require a large amount of nutrients for growth, and sourcing high-quality growth medium is very expensive. This problem can be overcome by utilizing effluent waste products from agricultural processes as growth media for our lactobacili. This approach involves obtaining waste produced as a byproduct of corn, wheat, and other grain processing and using it as growth media, as it is rich in the sugars, proteins, and other macronutrients important for bacterial growth. A study showed that using effluent waste results in bacterial yields equal to those obtained through traditional growth medium, demonstrating the efficacy of this technique.1 This approach would be a cost-effective solution to producing our bacteria, and consequently would reduce the downstream drug costs to patients, hospitals, and other healthcare stakeholders (i.e. the Government of Ontario subsidizing healthcare costs). This makes our treatment more likely to be a cost-efficient alternative to current cancer treatment methods.
Krzywonos M, Eberhard T. High density process to cultivate Lactobacillus plantarum biomass using wheat stillage and sugar beet molasses. Electronic Journal of Biotechnology. 2011 Mar;14(2):6-.
CASE 4: IL-2 therapy: Safety and risk evaluations
Interleukin-2 (IL-2) is a cytokine signaling molecule within the immune system. It is responsible for regulating the activity of leukocytes. This keeps the immune system protected against microbial infections by distinguishing between foreign and resident cells. It allows for the prevention of autoimmune diseases by promoting the differentiation of immature T cells into regulatory cells. This allows for the body to fight off infections. Due to this, it is marketed as a protein therapeutic, Proleukin, for the treatment of cancer. High-dose interleukin-2 in fact resulted in objective clinical regression of metastatic cancer in 15% to 17% of patients.1
This was the cornerstone around which the research of McMaster iGEM’s (mGEM) wet lab focused on. mGEM’s research is for a non invasive and highly selective method for treating gastrointestinal tumour cells. IL-2 is a key component of this since it is possible to engineer it to be only expressed in the presence of tumors. The highly selective nature of this makes it an ideal candidate to be modified using quorum sensing in the gut microenvironment.
Since the application of IL-2 is often done in a large, continuous dose, it is important to minimize the impact from side effects. The toxic nature of IL-2 therapy is attributed mainly to capillary/vascular leak syndrome (C/VLS).2 This side effect manifests itself as peripheral edema and weight gain, ascites (collection of fluid in peritoneal cavity), pleural effusions or pulmonary edema. Patients with previous history of hypotension, oliguria (having less urine) and respiratory failure are often more susceptible to VLS symptoms. Protocols approved by the Clinical Research Committee of the National Institutes of Health (NIH) and by the Food and Drug Administration (FDA) were modeled to be selective of test demographics. Patients eligible for high dose IL-2 therapy include 18 years or older individuals with a minimum platelet count of 100,000/mm,3 a serum creatine level of <2mg%, a bilirubin value of <1.5mg%, and a performance score on the Karnovsky scale of 80% or greater.3 Other exclusion criteria include patients with autoimmune disorders, renal disease or those consuming immunosuppressive drugs. By following through with these criteria, it ensures a minimal risk background for sample patients.
Other methods of controlling IL-2 toxicity were also considered. High IL-2 doses included gastrointestinal side effects such as transient nausea, vomiting, diarrhea and anorexia.4 Prophylactic (intending to prevent disease) antiemetics (a drug used to prevent nausea/vomiting) given before and throughout the first IL-2 dose, were observed to minimize nausea and vomiting. This type of prophylactic was also seen to not increase the risk of hypotension, unlike similar medications.5 Even with all these options considered, it is crucial that IL-2 dosages remain low and are preferably administered by trained medical personnel.
Schwartzentruber DJ. Guidelines for the safe administration of high-dose interleukin-2. Journal of immunotherapy. 2001 Jul 1;24(4):287-93.
Siegel JP, Puri RK. Interleukin-2 toxicity. Journal of Clinical Oncology. 1991 Apr 1;9(4):694-704.
Lotze MT, Matory YL, Rayner AA, Ettinghausen SE, Vetto JT, Seipp CA, Rosenberg SA. Clinical effects and toxicity of interleukin‐2 in patients with cancer. Cancer. 1986 Dec 15;58(12):2764-72.
Schwartz RN, Stover L, Dutcher J. Managing toxicities of high-dose interleukin-2. Oncology (Williston Park, NY). 2002 Nov;16(11 Suppl 13):11-20.
Kim H, Rosenberg SA, Steinberg SM, Cole DJ, Weber JS. A randomized double-blinded comparison of the antiemetic efficacy of ondansetron and droperidol in patients receiving high-dose interleukin-2. Journal of immunotherapy. 1994 Jul 1;16(1):60-5.
In March we hosted our Synbio Speaker Series, an event where professionals discussed their research pertinent to synthetic biology for university students. We continuously create blog and social media posts to share information and our research. In May, Discovery Day in Health Sciences was held at McMaster University, which constitutes a day filled with workshops where secondary school students are able to explore a wide variety of careers in medicine and health sciences. Two of the workshops led by McMasterU allowed students to apply knowledge of cellular structures to laboratory practice where they stained and visualized the cells. Two other workshops we held reviewed the basic structure of DNA approached in the science curriculum, and further applied this to characteristically analyze DNA by gel electrophoresis. These events allow students to elaborate on concepts presented within class, and apply this knowledge to novel and relevant situations in the working world.
The McMaster iGEM blog was initially created in 2015 with the goal of sharing knowledge and research on synthetic biology in a friendly accessible format with the general public. It is geared toward both students interested in research and the general Hamilton community. This year our blog ran a series of infographics in order to help readers gain a better understanding of our project, as well as learn about the science involved. We aimed to make the series engaging and educational with modern graphics and a wide range of topics including DARPINs, electroporation, and quorum sensing. With a growing audience and supporters like PLOS Synbio on social media platforms, our blog has been able to reach the Hamilton community and beyond.
The McMaster iGEM blog is also an extension of our public engagement efforts, and we make sure to post about all events we host or participate in. As we introduce new posts and blog series over the years, we hope that the McMaster iGEM blog will become a repository of basic knowledge on genetic engineering and synthetic biology, as well as a collection of memories on all of our fun activities with the community. Head to the blog now to see our posts!
Through discussion and collaboration with the wet lab research team, the computational research team aimed to produce a tool that would meet the needs of, and aid in automating repetitive procedures done by the wet lab. Survey results from these discussions suggested that plasmid design was tedious and time-consuming, but a necessary portion of the research process. Since plasmid design relies heavily on pattern matching, the computational research team proposed a suitable solution to automate this process, by creating an application that would accelerate iteration through various parameter combinations to find compatible designs. The application takes input parameter sequences for the plasmid backbone, promoter, terminator, and protein of interest, and checks their compatibility to produce a recombinant plasmid. Following this analysis, the program provides the user with a list of potential restriction enzyme cut sites, and based on user selection, the program outputs the final plasmid design sequence.
Download the python app! (change the .txt ending to .py to run
Trial Values that work
The model and results: a summary
The agent-based model shows communication between agents through quorum sensing, which leads to the production of a factor (say, IL-2) in regions with high bacteria density, with no production in regions with low bacteria density. We ran the model on a 2D plane, with two distinct regions of different bacteria densities. The higher-density region exhibited production of the factor, while the lower-density region did not. Initially there was no factor production at all, but the signal concentration accumulated in the high-density region. Starting at step 57,000, the signal concentration in the high-density region passed the threshold for factor production, so the bacteria started producing the factor. The factor diffused through the medium, approaching equilibrium by at most step 100,000. Through the entire 100,000 steps, there was no factor production in the low-density region.
Here is how it works. A rectangular map is populated with bacteria in two populations, separated by some space. One is dense (a grid of bacteria separated by 3 distance units from neighbours) and one is sparse (a grid of bacteria separated by 15 distance units from neighbours). In each (again, unitless) time step, each bacterium does the following (parameters are in square brackets):
If the bacterium is active, produce [passive_signal_production] amount of signal
If the bacterium is active, produce [active_signal_production] amount of signal
If the bacterium is passive, and the signal at the bacterium’s location is above [signal_threshold], then switch to active state
If the bacterium is active, and the signal at the bacterium’s location is below [signal_threshold], then switch to passive state
If the bacterium is active, produce [factor_production] amount of factor
Then, diffusion proceeds, and the next step starts.
The results:
Bacterium locaions
Animated: steps 0 - 500,000
What the model is not
This model is not a simulation. It is not the computation of numerically known relationships, etc.. This is because we do not know, on a mathematical level, how the bacteria produce and handle signal and factor molecules. No-one has said exactly how much of the signal or factor molecules a single one of our bacteria produces while active or inactive. No-one has determined the diffusion and extinction coefficients of the signal or factor molecules used in our project. So this model is not the computation of things that we know will happen in real life, the same way a physics simulation of a block falling is such (because we know g = 9.81m/s, and we know the laws of kinematics).
This model does not explore how bacteria attach to tumour cells. That’s just outside the scope of what we programmed. The model only deals with stationary bacteria with locations defined manually beforehand. So the high-density region is the tumour where the bacteria accumulate - but that must all be accounted for before our model starts running.
What the model is
This model does three things:
Takes assumptions given to it by humans, i.e. us.
Takes parameters given to it by humans (or by a program that gives it parameters from a list)
Computes what the system does, following the assumptions, plugging in the parameters.
So it may or may not resemble real phenomena, depending on whether the assumptions and parameters are realistic.
Two of the model’s three assumptions are either valid physical laws or assumptions made by other researchers publishing agent-based models in peer-reviewed literature:
Signals and factors both diffuse following the 2D diffusion equation.
That is really the only reasonable assumption. With express permission from the author, we used the Python implementation of a numerical approximation of the solution to the 2D diffusion equation from the Brick in the Sky
Bacteria produce signals at a lower rate when they are inactive; upon activation, when the signal concentration at their location reaches a threshold, they switch to a higher rate.  Signal production is thus a step function of signal concentration.
This is the the assumption made by Netotea et al. in their agent-based model of quorum sensing, modelling bacterial swarming (Netotea et al., 2009)
One assumption was just taken from conversations with wet lab members:
Bacteria produce factors only when the signal concentration at their location is above a threshold (not necessarily the same threshold as for signal concentration)
Then, we fiddled around with parameters, using trial and error until they showed the behaviour we were looking for - that is, that dense bacterial populations would produce the factor while sparse ones would produce none.
The parameters are all unitless. Unitless parameters are often used in models (such as this one) which aim to simply demonstrate an effect, without necessarily simulating real, physical behaviour.
Since we used unitless parameters, found through trial and error, our model does not necessarily closely mirror any phenomenon that might be seen in a lab. It wasn’t meant to; while its parameters do correspond with real-life properties of bacteria and the chemicals they produce and detect, those properties have unknown real-life values, too. Instead, the model and its results just tell us that there exists a set of parameters that give us what we want - knowing the values of those parameters, relative to each other, we can learn about how to make our quorum sensing work the way we want it to in the lab.
What we can learn
The important part of this model is what we can learn from it to guide our future research in the wet lab. In tweaking the parameters to display this behaviour - which we’ll also be looking for in the wet lab - we developed a few basic insights. The important parts are bolded:
Assuming we have no control over the diffusion and extinction coefficients of our chemicals, we only have control over passive and active signal production rates, as well as the signal threshold, in order to make our bacteria exhibit our desired effect.
These parameters will likely have to be extensively experimented with - just as they were in this model - to find a set that works.
An active signal production that is 10x the passive signal production produces good results. Too much lower, and there’s not enough of a difference between active and passive bacteria; too much higher, and a signal active bacterium can activate others (again, we want populations to work together, not just one influential individual). If the wet lab can achieve solid passive and active signal production rates at around this level, then it may be possible to reduce the free parameters to just the signal threshold.
Reducing the number of free parameters in this way dramatically increases the speed of testing. Leaving too many parameters free to experiment with results in an impossibly large number of trials to test. Finding just one parameter to adjust, keeping others constant, is really, really important; by investigating too many parameters, it is very easy to create an experiment that will take longer than the life of the universe to carry out in a wet lab.
In the wet lab, if we can indeed resort to only adjusting the signal threshold, we will adjust it according to how dense the bacteria are on the tumour. The denser they are on the tumour, relative to their density elsewhere, the higher the signal threshold can be. The signal threshold must be above the signal concentration in non-tumour regions, so that no factor (IL-2) is produced there.
As the project continues, we’ll continue to play around with this model, learning new insights to guide the wet lab! The code can be found in our Github repository .
Literature Cited
Leocorte, 2013. Maths With Python 3: Diffusion Equation. [online] the Brick In the Sky. Available at:
[Accessed 4 Sep. 2016].
Netotea, S., Bertani, I., Steindler, L., Kerényi, Á., Venturi, V. and Pongor, S., 2009. A simple model for the early events of quorum sensing in Pseudomonas aeruginosa: modeling bacterial swarming as the movement of an" activation zone". Biology direct, 4(1), p.1.
