<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCLA_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCLA/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header-bar"><DIV id="header-menu-container"><UL id="header-menu"><LI class="dropdown"><A><H2>Projects</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Protein_Cages">Protein Cages</A><A href="https://2016.igem.org/Team:UCLA/Super_Soldier_Bacteria">Super Soldier Systems</A><A href="https://2016.igem.org/Team:UCLA/InterLab">Interlab Study</A></DIV></LI><LI class="dropdown"><A href="https://2016.igem.org/Team:UCLA/Parts"><H2>Parts</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Basic_Part">Basic Parts</A><A href="https://2016.igem.org/Team:UCLA/Composite_Part">Composite Parts</A><A href="https://2016.igem.org/Team:UCLA/Part_Collection">Part Collection</A></DIV></LI><LI class="dropdown"><A><H2>Outreach</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/HP/Silver">Human Practices</A><A href="https://2016.igem.org/Team:UCLA/Collaborations">Collaborations</A></DIV></LI><LI class="dropdown"><A><H2>Judging</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Medal_Requirements">Medal Requirements</A><A href="https://2016.igem.org/Team:UCLA/Safety">Safety</A></DIV></LI><LI class="dropdown"><A><H2>Team</H2></A><DIV class="dropdown-content"><A href="https://2016.igem.org/Team:UCLA/Team">Meet the Team</A><A href="https://2016.igem.org/Team:UCLA/Attributions">Attributions</A></DIV></LI></UL></DIV></DIV><DIV class="row hero"><H1>EXPERIMENTS</H1></DIV><DIV class="row team-select"><DIV class="team-select-header">P R O J E C T S</DIV><DIV class="columns small-6 team team-active" id="team-cdi"><H3>CDI</H3></DIV><DIV class="columns small-6 team team-inactive" id="team-pcage"><H3>Protein Cages</H3></DIV></DIV><DIV class="row"><DIV class="columns small-9"><DIV class="row journal" id="journal"><DIV class="row" id="cdi-team-journal"><H2><STRONG>Development of the Project</STRONG></H2><P> </P><P>In order to test the expression of foreign CDI systems in DH5-α and <EM>Enterobacter aerogenes</EM>, we need to express each CDI system on a plasmid. We first received two recombinant <EM>E. coli</EM> strains from the Hayes lab located at the University of California, Santa Barbara: EPI100 with  pDAL930, containing the CDI system from EC869, and X90 with pDAL660, containing the CDI system from EC93. Our next objective was to transform electrocompetent DH5-α with pDAL930 and pDAL660 to see if we could express foreign CDI systems in DH5-α. We performed competition and aggregation assays on the transformed DH5-α, using the strains obtained from the Hayes lab as positive controls and ampicillin resistant DH5-α as a negative control. Assuming this is successful, we would move on to expressing two CDI systems derived from <EM>Enterobacter aerogenes </EM>in DH5-α individually and performing the same set of assays. These CDI systems must be isolated from genomic DNA and constructed into plasmids using Gibson assembly and overlap PCR. We plan on fusing the CdiA and CdiB genes from <EM>Enterobacter aerogenes </EM>with the CdiA-CT and CdiI genes of EC93, and inserting this chimeric construct into a pSC101-derived pSK33 backbone. Successful colonies are screened using CPCR and sequencing.</P><P> </P><H2><STRONG>Protocols</STRONG></H2><P> </P><H3>Competition Assay</H3><P><U>Purpose:</U> verify inhibitor strain is properly expressing CDI system</P><P>Inhibitory strain (CDI+): bacterial strain transformed with designated CDI system</P><P>Target Strain (CDI-): native bacteria without immunity or bacteria containing appropriate receptor</P><P>Positive Control: strain confirmed to properly express the CDI system and can properly inhibit the target</P><P>Negative Control: the inhibitory strain lacking any CDI system</P><OL style="list-style-type: upper-roman;"><LI>At each mark, plate ten-fold serial dilutions in a M9 salt solution or SOC from 1:1 to 1:1,000,000
</LI><LI>Not all dilutions may be needed; determine experimentally</LI><LI>Plate 100 µL of the culture on each plate</LI><LI>Count CFUs per mL
</LI><LI>Count up the total number of colonies found on a plate
</LI><LI>Pick plates that are easily countable and do not exhibit lawn growth (30-300 colonies)</LI></OL><LI>Divide the total number of colonies by the dilution factor</LI><LI>Incubate at 37 C at 225 rpm while plating at the 0 hour and 4 hour mark</LI><LI>Grow all cells to correct growth phase in 50 mL Luria-Bertani (LB) at 37 C in shaking incubator
</LI><LI>Positive control to logarithmic phase (OD<SUB>600</SUB> = ~ 0.35)</LI><LI>Inhibitor to OD<SUB>600</SUB> = 0.35</LI><LI>Negative control to OD<SUB>600</SUB> = 0.35</LI><LI>Target to either logarithmic phase or stationary phase</LI><LI>Mix inhibitor (or control) and target cell at a ratio of 10:1 (determined through concentration relation listed below) in LB without any antibiotic
</LI><LI>Multiply the Mass Attenuation Constant for your bacteria with the OD<SUB>600</SUB> reading to get the cellular density of your culture. Use this to determine the 10:1 ratio of inhibitor to target cultures.</LI><LI>Divide that value by the number of mLs plated to get CFUs/mL</LI><P> </P><H3>Gibson Assembly</H3><P><U>Purpose</U>: Assemble a plasmid with DNA segments that originate from different sources</P><OL style="list-style-type: upper-roman;"><LI>Design forward and reverse primers for each fragment with homology regions between adjacent fragments (look at table 1 below) </LI><LI>PCR amplify all the fragments using the primers you designed</LI><LI>Run the fragments on a gel to verify that they are the correct products and gel extract</LI><LI>Determine the concentration of each fragment</LI><LI>Mix the fragments in equimolar ratios starting with the largest fragment (look at table 2 below) into a PCR tube</LI><LI>Use the following formula to determine the amount of the subsequent fragments to add
</LI><LI>(x) pmol = 1.55 * ( y ng)/(z bp)</LI></OL><LI>Fill up to 5 µL with dH<SUB>2</SUB>O</LI><LI>Add 5 µL of Gibson Master Mix</LI><LI>Incubate at 50 C for 1 h</LI><P> </P><TABLE style="width: 563px;"><TBODY><TR style="height: 35px;"><TD style="height: 35px; width: 180px;"><P>Table 1</P></TD><TD style="height: 35px; width: 180px;"> </TD><TD style="height: 35px; width: 181px;"> </TD></TR><TR style="height: 35px;"><TD style="height: 35px; width: 180px;"><P>Number of fragments</P></TD><TD style="height: 35px; width: 180px;"><P>Fragment Size</P></TD><TD style="height: 35px; width: 181px;"><P>Length of overlap regions</P></TD></TR><TR style="height: 35px;"><TD style="height: 70px; width: 180px;" rowspan="2"><P>1 – 2</P></TD><TD style="height: 35px; width: 180px;"><P>≤ 8 kb</P></TD><TD style="height: 35px; width: 181px;"><P>20 – 40 bp</P></TD></TR><TR style="height: 35px;"><TD style="height: 35px; width: 180px;"><P>8 – 32 kb</P></TD><TD style="height: 35px; width: 181px;"><P>25 – 40 bp</P></TD></TR><TR style="height: 35px;"><TD style="height: 70.1875px; width: 180px;" rowspan="2"><P>3 – 5</P></TD><TD style="height: 35px; width: 180px;"><P>≤ 8 kb</P></TD><TD style="height: 35px; width: 181px;"><P>40 bp</P></TD></TR><TR style="height: 35.1875px;"><TD style="height: 35.1875px; width: 180px;"><P>8 – 32 kb</P></TD><TD style="height: 35.1875px; width: 181px;"><P>40 – 100 bp</P></TD></TR></TBODY></TABLE><P> </P><P> </P><TABLE><TBODY><TR><TD style="width: 213px;"><P>Table 2</P></TD><TD style="width: 213px;"> </TD><TD style="width: 213px;"> </TD></TR><TR><TD style="width: 213px;"><P>Fragment size</P></TD><TD style="width: 213px;"><P>Amount</P></TD><TD style="width: 213px;"><P>Pmols</P></TD></TR><TR><TD style="width: 213px;"><P>≤ 1 kb</P></TD><TD style="width: 213px;"><P>20 – 40 ng</P></TD><TD style="width: 213px;"><P>0.04</P></TD></TR><TR><TD style="width: 213px;"><P>1 – 5 kb</P></TD><TD style="width: 213px;"><P>10 – 25 ng</P></TD><TD style="width: 213px;"><P>0.008 – 0.04</P></TD></TR><TR><TD style="width: 213px;"><P>5 – 8 kb</P></TD><TD style="width: 213px;"><P>25 ng</P></TD><TD style="width: 213px;"><P>0.005 – 0.008</P></TD></TR><TR><TD style="width: 213px;"><P>8 – 20 kb</P></TD><TD style="width: 213px;"><P>25 – 100 ng</P></TD><TD style="width: 213px;"><P>0.005 – 0.008</P></TD></TR><TR><TD style="width: 213px;"><P>20 – 32 kb</P></TD><TD style="width: 213px;"><P>100 ng</P></TD><TD style="width: 213px;"><P>0.005</P></TD></TR></TBODY></TABLE><H3> </H3><H3>Overlap PCR</H3><P><U>Purpose</U>: fuse two or more fragments of DNA using PCR (less efficient)</P><OL style="list-style-type: upper-roman;"><LI>PCR amplify the fragments</LI><LI>Design PCR primers with 20 base pairs of homology</LI><LI>Run product on a gel to confirm that you have the correct products
</LI><LI>Clean up the product</LI></OL><LI>Use the PCR products as template in a PCR reaction where the overlap regions will anneal on their own
</LI><LI>Do not use any primers in this step</LI><LI>Run 15 cycles</LI><LI>PCR amplify the fused fragments with the end primers that encapsulate both fragments</LI><P> </P><H3>Colony PCR</H3><P><U>Purpose</U>: Determine if cells took up construct after transformation</P><OL style="list-style-type: upper-roman;"><LI>Take “x” number of colonies from a plate of bacteria and suspend them separately in 100 µL of dH<SUB>2</SUB>O</LI><LI>Design primers that amplify a region contained in the target construct (surrounding homology region ideally)
</LI><LI>Regions of 200 – 500 bp are ideal</LI></OL><LI>Create a mastermix of forward primers, reverse primers, APEX mastermix, and dH<SUB>2</SUB>O (look at table 3 below)
</LI><LI>The mastermix should be made for “x” number of reactions without the colony suspension added</LI><LI>Add 1 µL of the colony suspension to 24 µL of the mastermix in a PCR tube and run the samples in a thermocycler</LI><LI>Run the PCR products on a gel to determine if you have the correct construct</LI><P> </P><TABLE><TBODY><TR><TD style="width: 319px;"><P>Table 3</P></TD><TD style="width: 319px;"> </TD></TR><TR><TD style="width: 319px;"><P>Item</P></TD><TD style="width: 319px;"><P>Amount</P></TD></TR><TR><TD style="width: 319px;"><P>Forward Primer (10 µM)</P></TD><TD style="width: 319px;"><P>0.5 µL</P></TD></TR><TR><TD style="width: 319px;"><P>Reverse Primer (10 µM)</P></TD><TD style="width: 319px;"><P>0.5 µL</P></TD></TR><TR><TD style="width: 319px;"><P>Colony Suspension</P></TD><TD style="width: 319px;"><P>1.0 µL</P></TD></TR><TR><TD style="width: 319px;"><P>APEX mastermix</P></TD><TD style="width: 319px;"><P>12.5 µL</P></TD></TR><TR><TD style="width: 319px;"><P>dH<SUB>2</SUB>O</P></TD><TD style="width: 319px;"><P>Up to 25 µL</P></TD></TR></TBODY></TABLE><H3> </H3><H3> Electroporation of plasmids</H3><OL style="list-style-type: upper-roman;"><LI>Load cuvette into electroporation apparatus and pulse
</LI><LI>Make sure arc time result is appropriate considering the cuvette used</LI></OL><LI>Transfer enough cells to cuvette (depends on the cuvette)
</LI><LI>Wipe condensation off the outside of the cuvette and tap the bottom of the housing on something solid to bring the cells to the bottom of the cuvette</LI><LI>Add 1 µL of plasmid insert to cells
</LI><LI>Mix gently by flicking the bottom of the tube</LI><LI>Start incubating 1 mL of SOC</LI><LI>Take electro-competent cells and put them on ice for ten minutes (or until they have partially thawed)</LI><LI>Put electroporation cuvettes on ice</LI><LI>Add 970 µL of warm SOC to cuvette and pipet slowly up and down twice</LI><LI>Load solution into a 1.5 mL micro-centrifuge tube and shake at 37 C and 800 rpm for 1 h</LI><LI>Place plates with appropriate antibiotic into an incubator at 37 C</LI><LI>After the hour is up, spin the cells down at 6000 rpm for 4 min
</LI><LI>Discard supernatant and suspend in 100 µL of SOC</LI><LI>Plate using serial dilutions and incubate for desired length</LI><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P><P> </P></DIV><DIV class="row" id="pcage-team-journal"><H3><STRONG>PCquad and O3-33 Expression and Purification</STRONG></H3><P>Growth and Pelleting:</P><OL><LI>Inoculate 5 ml of cells in 37 C overnight</LI><LI>Incubate cells in larger flask(s) (1 - 4 L per batch) in 37 C until OD reaches 0.8</LI><LI>Add 0.5 mM IPTG (1 mL of 0.5 M stock per liter of cells)</LI><LI>Incubate cells in 30 C for 5 hrs</LI><LI>Aliquot cells into 50 mL Falcon tubes</LI><LI>Wash and pellet cells</LI><OL><LI>Centrifuge cells 4000g for 10 min</LI><LI>Dump out supernatant</LI><LI>Resuspend cells in 10 ml of cold water</LI><LI>Consolidate suspended pellets into fewer falcon tubes</LI><LI>Centrifuge… Repeat until all cells washed into a single tube</LI></OL></OL><P>Lysis and filtration:</P><OL><LI>Resuspend cell pellet in lysis buffer (1 : 2 cell weight : ml of lysis buffer)</LI><OL><LI>PCQuad: 20mM sodium phosphate (ph=8.0), 300 mM sodium chloride, 10 mM imidazole</LI><LI>O3-33: 50 mM TRIS (ph=8.0), 250 mM NaCl, 20mM imidazole</LI></OL><LI>Chill cell suspension for 10 min</LI><LI>Add 10 uL 100 mM PMSF for every mL of cell suspension right before sonication</LI><LI>Sonicate cell suspension:for 10 sec (10 cycles; 30 sec pause between each cycle)</LI><LI>Centrifuge lysates for 30 min at 13000 rpm</LI><LI>Filter supernatant through a 22 um Millipore filter</LI></OL><P>Purification:</P><OL><LI>Used Hispur column purification protocol</LI><OL><LI>Add appropriate amount of resin bed to purification column</LI><LI>Equilibriate column with two resin bed volumes of equilibration buffer</LI><LI>Add equal volume of equilibration buffer through filtered protein sample</LI><LI>Add sample through column. Collect flow through and reapply through column. Save flow through for downstream analysis if desired</LI><LI>Wash resin with two resin bed volumes of wash buffer with a gradient of imidazole concentrations until protein A280 reaches baseline Save flow through for downstream analysis if desired</LI><LI>Elute protein with two resin bed volumes of elution buffer. Repeat elution. Save flow through for downstream analysis if desired</LI><LI>Imidazole concentrations used:</LI><OL><LI>PCQuad</LI><OL><LI>Equilibration: 20 mM</LI><LI>W1: 25 mM, W2:40 mM, W3: 150 mM</LI><LI>E1: 400 mM, E2: 400 mM</LI></OL><LI>O3-33</LI><OL><LI>Equilibration: 20 mM</LI><LI>W1: 30 mM, W2: 40 mM, W3: 50 mM</LI><LI>E1: 400 mM, E2: 400 mM</LI></OL></OL></OL><LI>Run SDS-PAGE on equilibration, three washes and two elutions</LI><OL><LI>Mix samples with an equal volume of sample buffer (prepped with BME)</LI><LI>Heat samples at 70 C for 10 min before loading into gel</LI><LI>Use 1x SDS MOPS Running Buffer</LI></OL><LI>Run Native-PAGE gel on best elutions</LI><OL><LI>Use MOPS Running Buffer</LI></OL><LI>Syringe filter 1 mL of best elutions through 0.22 um filter and use 30 uL for DLS</LI></OL><H3><STRONG>Buffer Preparation Protocols</STRONG></H3><P>O3-33:</P><UL><LI>Lysis buffer: 50 mM TRIS (ph=8.0), 250 mM NaCl, 20mM imidazole supplemented with 1 mM phenylmethanesulfonyl fluoride</LI><UL><LI>50 ml solution</LI><UL><LI>2.5 mL TRIS</LI><LI>0.7305 g NaCl</LI><LI>0.068 g imidazole</LI><LI>0.0087 g phenylmethanesulfonyl fluoride</LI><LI>Up to 50 ml water</LI></UL></UL></UL><P>PCQuad:</P><UL><LI>Lysis buffer: 20mM sodium phosphate (ph=8.0), 300 mM sodium chloride, 10 mM imidazole</LI><UL><LI>50 ml solution</LI><UL><LI>.164 g sodium phosphate</LI><LI>.877 g NaCl</LI><LI>.034 g imidazole</LI><LI>Up to 50 ml water</LI></UL></UL><LI>Lysis buffer supplemented with 500 mM imidazole(<STRONG>Elution)</STRONG></LI><UL><LI>50 ml solution</LI><UL><LI>1.702 g imidazole</LI><LI>Up to 50 ml lysis buffer</LI></UL></UL></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>