Team:UCLA/Experiments
Projects
Protein Cages Super Soldier Systems Interlab Study
Parts
Basic Parts Composite Parts Part Collection
Outreach
Human Practices Collaborations
Judging
Medal Requirements Safety
Team
Meet the Team Attributions
EXPERIMENTS
P R O J E C T S
CDI
Protein Cages
Development of the Project
 
In order to test the expression of foreign CDI systems in DH5-α and Enterobacter aerogenes, we need to express each CDI system on a plasmid. We first received two recombinant E. coli strains from the Hayes lab located at the University of California, Santa Barbara: EPI100 with  pDAL930, containing the CDI system from EC869, and X90 with pDAL660, containing the CDI system from EC93. Our next objective was to transform electrocompetent DH5-α with pDAL930 and pDAL660 to see if we could express foreign CDI systems in DH5-α. We performed competition and aggregation assays on the transformed DH5-α, using the strains obtained from the Hayes lab as positive controls and ampicillin resistant DH5-α as a negative control. Assuming this is successful, we would move on to expressing two CDI systems derived from Enterobacter aerogenes in DH5-α individually and performing the same set of assays. These CDI systems must be isolated from genomic DNA and constructed into plasmids using Gibson assembly and overlap PCR. We plan on fusing the CdiA and CdiB genes from Enterobacter aerogenes with the CdiA-CT and CdiI genes of EC93, and inserting this chimeric construct into a pSC101-derived pSK33 backbone. Successful colonies are screened using CPCR and sequencing.
 
Protocols
 
Competition Assay
Purpose: verify inhibitor strain is properly expressing CDI system
Inhibitory strain (CDI+): bacterial strain transformed with designated CDI system
Target Strain (CDI-): native bacteria without immunity or bacteria containing appropriate receptor
Positive Control: strain confirmed to properly express the CDI system and can properly inhibit the target
Negative Control: the inhibitory strain lacking any CDI system
At each mark, plate ten-fold serial dilutions in a M9 salt solution or SOC from 1:1 to 1:1,000,000
Not all dilutions may be needed; determine experimentally
Plate 100 µL of the culture on each plate
Count CFUs per mL
Count up the total number of colonies found on a plate
Pick plates that are easily countable and do not exhibit lawn growth (30-300 colonies)
Divide the total number of colonies by the dilution factor
Incubate at 37 C at 225 rpm while plating at the 0 hour and 4 hour mark
Grow all cells to correct growth phase in 50 mL Luria-Bertani (LB) at 37 C in shaking incubator
Positive control to logarithmic phase (OD600 = ~ 0.35)
Inhibitor to OD600 = 0.35
Negative control to OD600 = 0.35
Target to either logarithmic phase or stationary phase
Mix inhibitor (or control) and target cell at a ratio of 10:1 (determined through concentration relation listed below) in LB without any antibiotic
Multiply the Mass Attenuation Constant for your bacteria with the OD600 reading to get the cellular density of your culture. Use this to determine the 10:1 ratio of inhibitor to target cultures.
Divide that value by the number of mLs plated to get CFUs/mL
 
Gibson Assembly
Purpose: Assemble a plasmid with DNA segments that originate from different sources
Design forward and reverse primers for each fragment with homology regions between adjacent fragments (look at table 1 below) 
PCR amplify all the fragments using the primers you designed
Run the fragments on a gel to verify that they are the correct products and gel extract
Determine the concentration of each fragment
Mix the fragments in equimolar ratios starting with the largest fragment (look at table 2 below) into a PCR tube
Use the following formula to determine the amount of the subsequent fragments to add
(x) pmol = 1.55 * ( y ng)/(z bp)
Fill up to 5 µL with dH2O
Add 5 µL of Gibson Master Mix
Incubate at 50 C for 1 h
 
Table 1
 
 
Number of fragments
Fragment Size
Length of overlap regions
1 – 2
≤ 8 kb
20 – 40 bp
8 – 32 kb
25 – 40 bp
3 – 5
≤ 8 kb
40 bp
8 – 32 kb
40 – 100 bp
 
 
Table 2
 
 
Fragment size
Amount
Pmols
≤ 1 kb
20 – 40 ng
0.04
1 – 5 kb
10 – 25 ng
0.008 – 0.04
5 – 8 kb
25 ng
0.005 – 0.008
8 – 20 kb
25 – 100 ng
0.005 – 0.008
20 – 32 kb
100 ng
0.005
 
Overlap PCR
Purpose: fuse two or more fragments of DNA using PCR (less efficient)
PCR amplify the fragments
Design PCR primers with 20 base pairs of homology
Run product on a gel to confirm that you have the correct products
Clean up the product
Use the PCR products as template in a PCR reaction where the overlap regions will anneal on their own
Do not use any primers in this step
Run 15 cycles
PCR amplify the fused fragments with the end primers that encapsulate both fragments
 
Colony PCR
Purpose: Determine if cells took up construct after transformation
Take “x” number of colonies from a plate of bacteria and suspend them separately in 100 µL of dH2O
Design primers that amplify a region contained in the target construct (surrounding homology region ideally)
Regions of 200 – 500 bp are ideal
Create a mastermix of forward primers, reverse primers, APEX mastermix, and dH2O (look at table 3 below)
The mastermix should be made for “x” number of reactions without the colony suspension added
Add 1 µL of the colony suspension to 24 µL of the mastermix in a PCR tube and run the samples in a thermocycler
Run the PCR products on a gel to determine if you have the correct construct
 
Table 3
 
Item
Amount
Forward Primer (10 µM)
0.5 µL
Reverse Primer (10 µM)
0.5 µL
Colony Suspension
1.0 µL
APEX mastermix
12.5 µL
dH2O
Up to 25 µL
 
 Electroporation of plasmids
Load cuvette into electroporation apparatus and pulse
Make sure arc time result is appropriate considering the cuvette used
Transfer enough cells to cuvette (depends on the cuvette)
Wipe condensation off the outside of the cuvette and tap the bottom of the housing on something solid to bring the cells to the bottom of the cuvette
Add 1 µL of plasmid insert to cells
Mix gently by flicking the bottom of the tube
Start incubating 1 mL of SOC
Take electro-competent cells and put them on ice for ten minutes (or until they have partially thawed)
Put electroporation cuvettes on ice
Add 970 µL of warm SOC to cuvette and pipet slowly up and down twice
Load solution into a 1.5 mL micro-centrifuge tube and shake at 37 C and 800 rpm for 1 h
Place plates with appropriate antibiotic into an incubator at 37 C
After the hour is up, spin the cells down at 6000 rpm for 4 min
Discard supernatant and suspend in 100 µL of SOC
Plate using serial dilutions and incubate for desired length
 
 
 
 
 
 
 
PCquad and O3-33 Expression and Purification
Growth and Pelleting:
Inoculate 5 ml of cells in 37 C overnight
Incubate cells in larger flask(s) (1 - 4 L per batch) in 37 C until OD reaches 0.8
Add 0.5 mM IPTG (1 mL of 0.5 M stock per liter of cells)
Incubate cells in 30 C for 5 hrs
Aliquot cells into 50 mL Falcon tubes
Wash and pellet cells
Centrifuge cells 4000g for 10 min
Dump out supernatant
Resuspend cells in 10 ml of cold water
Consolidate suspended pellets into fewer falcon tubes
Centrifuge… Repeat until all cells washed into a single tube
Lysis and filtration:
Resuspend cell pellet in lysis buffer (1 : 2 cell weight : ml of lysis buffer)
PCQuad: 20mM sodium phosphate (ph=8.0), 300 mM sodium chloride, 10 mM imidazole
O3-33: 50 mM TRIS (ph=8.0), 250 mM NaCl, 20mM imidazole
Chill cell suspension for 10 min
Add 10 uL 100 mM PMSF for every mL of cell suspension right before sonication
Sonicate cell suspension:for 10 sec (10 cycles; 30 sec pause between each cycle)
Centrifuge lysates for 30 min at 13000 rpm
Filter supernatant through a 22 um Millipore filter
Purification:
Used Hispur column purification protocol
Add appropriate amount of resin bed to purification column
Equilibriate column with two resin bed volumes of equilibration buffer
Add equal volume of equilibration buffer through filtered protein sample
Add sample through column. Collect flow through and reapply through column. Save flow through for downstream analysis if desired
Wash resin with two resin bed volumes of wash buffer with a gradient of imidazole concentrations until protein A280 reaches baseline Save flow through for downstream analysis if desired
Elute protein with two resin bed volumes of elution buffer. Repeat elution. Save flow through for downstream analysis if desired
Imidazole concentrations used:
PCQuad
Equilibration: 20 mM
W1: 25 mM, W2:40 mM, W3: 150 mM
E1: 400 mM, E2: 400 mM
O3-33
Equilibration: 20 mM
W1: 30 mM, W2: 40 mM, W3: 50 mM
E1: 400 mM, E2: 400 mM
Run SDS-PAGE on equilibration, three washes and two elutions
Mix samples with an equal volume of sample buffer (prepped with BME)
Heat samples at 70 C for 10 min before loading into gel
Use 1x SDS MOPS Running Buffer
Run Native-PAGE gel on best elutions
Use MOPS Running Buffer
Syringe filter 1 mL of best elutions through 0.22 um filter and use 30 uL for DLS
Buffer Preparation Protocols
O3-33:
Lysis buffer: 50 mM TRIS (ph=8.0), 250 mM NaCl, 20mM imidazole supplemented with 1 mM phenylmethanesulfonyl fluoride
50 ml solution
2.5 mL TRIS
0.7305 g NaCl
0.068 g imidazole
0.0087 g phenylmethanesulfonyl fluoride
Up to 50 ml water
PCQuad:
Lysis buffer: 20mM sodium phosphate (ph=8.0), 300 mM sodium chloride, 10 mM imidazole
50 ml solution
.164 g sodium phosphate
.877 g NaCl
.034 g imidazole
Up to 50 ml water
Lysis buffer supplemented with 500 mM imidazole(Elution)
50 ml solution
1.702 g imidazole
Up to 50 ml lysis buffer
