<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Concordia_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Concordia/Parts</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="cssmenu"><UL><LI><A href="#">OUR TEAM</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Team">Team Members</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Contact">Contact Us</A></LI></UL><LI><A href="#">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Lactococcus_Lactis">Lactococcus Lactis</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Scaffold">Scaffold</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/FoodIntolerance">Food Intolerances</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Pathway">Prototype Pathway</A></LI><LI><A href="#">RESULTS</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/ResultsCloning">Results of Cloning</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Results">Results of Testing</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/FutureWork">Future Work</A></LI><LI><A href="#">WET LAB</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Notebook">Notebook</A></LI><LI><A href="#">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Parts">Team Parts</A></LI><LI><A href="#">POLICIES</A></LI><LI><A href="/Team:Concordia/Practices">Human Practices</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Outreach">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:Concordia/Safety">Safety</A></LI></DIV><DIV id="mainContainer"><DIV class="container"><DIV class="jumbotron" style="background-color:#F38630"><H2 style="color:white; font-size:46px">Team Parts</H2></DIV><TABLE class="table table-bordered" style="background-color:white"><THEAD><TR><TH>#</TH><TH>Part Name</TH><TH>Registry #</TH><TH>Short Description</TH><TH>Long Description</TH><TH>Reference</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>pET15BB</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830000">BBa_K1830000</A></TD><TD>IPTG-inducible E. Coli Expression vector with RCF25-compatible MCS</TD><TD>pET15BB is a vector designed for expression and purification of proteins of interest from E. Coli. It is derived from the iGEM standard pSB1C3 backbone and so includes a Chloramphenicol resistance selection marker. The LacI operon was added to the backbone for inducible expression as well as a multiple cloning site designed to welcome any part displaying a 5’ NgoMIV and 3’ SpeI cut sites, in accordance with the RCF25 compatibility</TD><TD>Husser, M., Narcross, L., Sundarakrishnan, A. iGEM Concordia 2015</TD></TR><TR><TD>2</TD><TD>PnisA</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830001">BBa_K1830001</A></TD><TD>Nisin-inducible promoter and ribosome binding site</TD><TD>This promoter requires complementation from the Lactococcal nisin operon genes (nisZBTCIPRKFEB). Lactococcus lactis is inherently resistant to the antimicrobial peptide nisin. These two operons are responsible for both the immunity of L. lactis and nisin production. NisA is in fact the gene coding for the nisin precursor and its promoter (PnisA) will express any gene under its control when induced by nisin.</TD><TD>De Ruyter, P. G., Kuipers, O. P., &amp; De Vos, W. M. (1996). Controlled gene expression systems for Lactococcus lactis with the food-grade inducer nisin. Applied and environmental microbiology, 62(10), 3662-3667.</TD></TR><TR><TD>3</TD><TD>SPusp45</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830002">BBa_K1830002</A></TD><TD>Signal peptide of USP45 for direct extracellular localization by Lactococcus lactis</TD><TD>USP45 is a secreted extracellular protein of L. lactis with no known function and an N-terminal 22 amino acid signal peptide. Use of the signal peptide allows for direct extracellular secretion of any protein of interest.</TD><TD>Le Loir, Y., Nouaille, S., Commissaire, J., Bretigny, L., Gruss, A., &amp; Langella, P. (2001). Signal peptide and propeptide optimization for heterologous protein secretion in Lactococcus lactis. Applied and Environmental Microbiology, 67(9), 4119-4127.</TD></TR><TR><TD>4</TD><TD>CBD</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830003">BBa_K1830003</A></TD><TD>Cellulose Binding Domain</TD><TD>This domain is part of the Cellulosome Integrating Protein A (CipA), which forms part of the cellulosome of Clostridium thermocellum. The cellulosome is an extracellular protein scaffold attached to the bacterial cell wall and displays cellulose-degrading enzymes (cellulases). For increased cellulose degradation, the cellulosome of C. thermocellum includes a Cellulose Binding Domain (CBD) named CBM3a that brings enzymes and substrate together. This domain is also thought to increase stability of the secreted scaffold.</TD><TD>Gerngross, U. T., Romaniec, M. P. M., Kobayashi, T., Huskisson, N. S., &amp; Demain, A. L. (1993). Sequencing of a Clostridium thermocellum gene (cipA) encoding the cellulosomal SL‐protein reveals an unusual degree of internal homology. Molecular microbiology, 8(2), 325-334.
Wieczorek, A. S., &amp; Martin, V. J. (2010). Engineering the cell surface display of cohesins for assembly of cellulosome-inspired enzyme complexes on Lactococcus lactis. Microbial cell factories, 9(1), 69.

GenBank accession number: Q06851 (protein) / L08665.1 (gene)
</TD></TR><TR><TD>5</TD><TD>coh1C3</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830004">BBa_K1830004</A></TD><TD>Type I cohesin domain</TD><TD>coh1C3 is the third type I cohesin domain of the Cellulosome Integrating Protein (CipA), part of the Clostridium thermocellum cellulosome. The cellulosome is an extracellular protein scaffold that displays cellulose-degrading enzymes. These cellulases are fused to a dockerin domain that attaches to the cellulosome through cohesin/dockerin binding interactions. These interactions are calcium ion dependant and binding is specific, therefore a type I cohesin will only bind to a type I dockerin.</TD><TD>Wieczorek, A. S., &amp; Martin, V. J. (2010). Engineering the cell surface display of cohesins for assembly of cellulosome-inspired enzyme complexes on Lactococcus lactis. Microbial cell factories, 9(1), 69.

GenBank accession number : Q06851 (protein) / L08665.1 (gene)
</TD></TR><TR><TD>6</TD><TD>coh2S1</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830005">BBa_K1830005</A></TD><TD>Type II cohesin domain</TD><TD> coh2S1 is the type II cohesin domain of the Scaffolding Dockerin Binding protein A (SdbA), part of the Clostridium thermocellum cellulosome. The cellulosome is an extracellular protein scaffold that displays cellulose-degrading enzymes. The scaffold’s components are assembled by the interaction between their cohesin and dockerin domains. These interactions are calcium ion dependant and binding is specific, therefore a type II cohesin will only bind to a type II dockerin.</TD><TD>Leibovitz, E., &amp; Beguin, P. (1996). A new type of cohesin domain that specifically binds the dockerin domain of the Clostridium thermocellum cellulosome-integrating protein CipA. Journal of bacteriology, 178(11), 3077-3084.

GenBank accession number : U49980.1
</TD></TR><TR><TD>7</TD><TD>CWAm6</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830006">BBa_K1830006</A></TD><TD>Bacterial cell wall anchor for Gram+ bacteria</TD><TD>The C-terminal cell wall anchor from the Streptococcus pyogenes M6 protein contains a conserved LPXTG motif recognised by sortase enzymes for attachment to the cell wall of Lactic acid bacteria.

</TD><TD>Piard, J. C., Hautefort, I., Fischetti, V. A., Ehrlich, S. D., Fons, M., &amp; Gruss, A. (1997). Cell wall anchoring of the Streptococcus pyogenes M6 protein in various lactic acid bacteria. Journal of bacteriology, 179(9), 3068-3072.

GenBank accession number : M11338.1
</TD></TR><TR><TD>8</TD><TD>L. shuttle</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830008">BBa_K1830008</A></TD><TD>Lactococcus lactis shuttle vector part/td&gt;
        </TD><TD>The Lactococcus lactis shuttle vector contains all the components required for transformation, replication and expression of a vector in L. lactis. For this purpose, it has to be integrated into the vector that you want to transform L. lactis with.</TD><TD>Courtesy of A. Wieczorek</TD></TR><TR><TD>9</TD><TD>Dock1</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830009">BBa_K1830009</A></TD><TD>Type I dockerin domain</TD><TD>Dock1 is the type I dockerin domain of the Cellulase S (CelS), part of the Clostridium thermocellum cellulosome. The cellulosome is an extracellular protein scaffold that displays cellulose-degrading enzymes. These cellulases contain a dockerin domain that attaches to the cellulosome through cohesin/dockerin binding interactions. The interactions are calcium ion dependant and binding is specific, therefore a type I cohesin will only bind to a type I dockerin.</TD><TD>Choi, S. K., &amp; Ljungdahl, L. G. (1996). Structural role of calcium for the organization of the cellulosome of Clostridium thermocellum. Biochemistry,35(15), 4906-4910.
	Wieczorek, A. S., &amp; Martin, V. J. (2010). Engineering the cell surface display of cohesins for assembly of cellulosome-inspired enzyme complexes on Lactococcus lactis. Microbial cell factories, 9(1), 69.

GenBank accession number : L06942.1
</TD></TR><TR><TD>10</TD><TD>XDock2</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830010">BBa_K1830010</A></TD><TD>Type II dockerin domain and its 5' X module</TD><TD>XDock2 is the type II dockerin domain of the Cellulosome Integrating Protein A (CipA), part of the Clostridium thermocellum cellulosome. The cellulosome is an extracellular protein scaffold that displays cellulose-degrading enzymes. The scaffold’s components are assembled by the interaction between their cohesin and dockerin domains. The interactions are calcium ion dependant and binding is specific, therefore a type II cohesin will only bind to a type II dockerin. « X module » is the name given to the 300bp region located 5’ to the dockerin II domain in the native CipA protein. It favors proper folding of the dockerin domain and improves the ability for the cohesin to bind to the dockerin II domain.

</TD><TD>Leibovitz, E., &amp; Beguin, P. (1996). A new type of cohesin domain that specifically binds the dockerin domain of the Clostridium thermocellum cellulosome-integrating protein CipA. Journal of bacteriology, 178(11), 3077-3084.

GenBank accession number: L06942.1
</TD></TR><TR><TD>11</TD><TD>ADH2 alcohol dehydrogenase</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830013">BBa_K1830013</A></TD><TD>Saccharomyces cerevisiae S288c alcohol dehydrogenase coding sequence</TD><TD>The Saccharomyces cerevisiae alcohol dehydrogenase ADH2 is the enzyme that catalyzes the interconversion between alcohols and aldehydes or ketones. The reaction requires NAD+ as a cofactor. 

</TD><TD>GenBank accession number: NM_001182812.1</TD></TR><TR><TD>12</TD><TD>ALD2 aldehyde dehydrogenase</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830014">BBa_K1830014</A></TD><TD>Saccharomyces cerevisiae S288c aldehyde dehydrogenase coding sequence</TD><TD>The Saccharomyces cerevisiae aldehyde dehydrogenase ALD2 is the enzyme that catalyzes the oxydation of aldehydes. This reaction requires NAD+ as a cofactor. </TD><TD>GenBank accession number: NM_001182674.1</TD></TR><TR><TD>13</TD><TD>SacC sucrase</TD><TD><A href="http://parts.igem.org/Part:BBa_K1830015">BBa_K1830015</A></TD><TD>SacC extracellular sucrase</TD><TD>Extracellular sucrase from Zymomonas mobilis submitted by the iGEM team Tec-Monterrey 2011 under the registry number BBa_K633003. It has been modified to be RCF25 compatible by removal of an AgeI cut site.</TD><TD>iGEM registry, BBa_K633003</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>