<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Notebook_Test"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Notebook/Test">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Notebook/Test&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/Test&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/Test&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Notebook/Test" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Notebook/Test</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><P><ASIDE><HEADER>
  Notebook
 </HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI></UL></ASIDE></P><H1>Weeks</H1><TABLE align="center"><THEAD><TR><TH align="center">Month</TH><TH align="center" colspan="5">Week</TH></TR></THEAD><TBODY><TR><TH class="red"> Jun. </TH><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w1" title="17th June to 23rd June">1</A></TD><TD colspan="4"><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w2" title="24th June to 30th June">2</A></TD></TR><TR><TH class="yellow"> Jul.</TH><TD><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w3" title="1st July to 7th July">3</A></TD><TD><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w4" title="8th July to 14th July">4</A></TD><TD><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w5" title="15th July to 21st July">5</A></TD><TD colspan="2"><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w6" title="22nd July to 28th July">6</A></TD></TR><TR><TH class="red"> Aug.</TH><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w7" title="29th July to 4th August">7</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w8" title="5th August to 11th August">8</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w9" title="12th August to 18th August">9</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w10" title="19th August to 25th August">10</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w11" title="26th August to 1st September">11</A></TD></TR><TR><TH class="yellow"> Sep.</TH><TD><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w12" title="2nd September to 8th September">12</A></TD><TD><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w13" title="9th September to 15th September">13</A></TD><TD><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w14" title="16th September to 22nd September">14</A></TD><TD colspan="2"><A class="yellow" href="https://2013.igem.org/Team:Evry/Notebook/w15" title="23rd September to 29th September">15</A></TD></TR><TR><TH class="red"> Oct. </TH><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w16" title="30th September to 6th October">16</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w17" title="7th October to 13th October">17</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w18" title="14th October to 20th October">18</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w19" title="21st October to 27th October">19</A></TD><TD><A class="red" href="https://2013.igem.org/Team:Evry/Notebook/w20" title="28th October to 3rd November">20</A></TD></TR></TBODY></TABLE><DIV id="maincontainer_evry2013"><DIV id="mainTextcontainer_notebook"><H1 id="week1">Week 1: 17<SUP>th</SUP> June - 23<SUP>rd</SUP> June</H1><H2>Tuesday, 19th June</H2><P>As a start for the labwork, we first decided to make competent cells. We grew from glycerol samples three E. coli strains in 2 mL of LB medium.
The chemically competent strains chosen are as follows:
BL21
DH5α
TOP10</P><DIV align="center">Protocol for pre-culture preparation</DIV>

In a 15 mL tube, add 2 mL of LB medium and inoculate cells from glycerol<H2>Thursday, 20th June</H2><P>We started to make 200 mL of BL21 and DH5α E. coli strains and 400 mL of TOP10 to make competent cells.</P><B><DIV align="center">Protocol for competent cells</DIV><P>
First prepare the following solutions required to make competent cells:
Solution for 1M Cacl2:
Add 14,30g of CaCl2 into 100 ml desalted water

Solution for 0,1M Cacl2:
Add 50 mL of CaCl2 1M solution into 450 ml of desalted water

Solution for 0,1M Cacl2 + 15% glycerol:
Add 50 mL of CaCl2 1M solution and 75 mL of glycerol 100% into 450 ml of desalted water</P><P>For 200 ml LB medium, add 400 µL of strain sample.
Let the bacteria grow until it reaches an OD between 0,3 and 0,35.
Once it reached the right OD, put the medium on ice for 30 minutes to slow down growth.
Split the 200 mL into 4x50 mL tubes then centrifuge at 3000 rpm for 5 minutes at 4°C and suppress supernatant afterwards.
Resuspend the cells with 5 mL of Cacl2 at 0,1M for each 50 mL tube.
Again, put the medium on ice for 30 minutes.
Centrifuge at 3000 rpm for 5 minutes at 4°C then suppress supernatant.
Resuspend the cells with 1 mL of Cacl2 at 0,1M + 15% glycerol for each 50 mL tube.
Split the total 4 mL into 40 tubes containing each 100 µL of concentrated cell solution. This step should be executed fast enough and on ice.</P><H2>Friday, 21st June</H2><P>To check the quality of our work, each strain has been plated on LB medium with ampicillin, kanamycin or chloramphenicol in order to evaluate if it contaminated or not. Furthermore, to evaluate wether our strains are competent or not, we also transformed our bacteria with a pSB1A3 plasmid (red colonies) and plated them on LB medium with ampicillin only.</P><DIV align="center">Protocol for transformation</DIV><P>For 100 µL of chemical competent cells, add 1 µL of plasmidic DNA.
Let 30 minutes on ice.
Let the cells at 42°C for exactly 50 seconds.
Let 5 minutes on ice.
Resuspend the cells with 1 mL of LB spread on a petri dish then let it at 37°C.</P><H1 id="week2">Week 2: 24<SUP>th</SUP> June - 30<SUP>th</SUP> June</H1><H2> Monday, June 24th </H2><P> Our competents cells made on friday the 21st were plated and incubated the whole week-end at 30°C. Today, we analyzed the plates and came to the conclusion that the medium was contaminated due to poor protocol execution. As a consequence, we decided to make new LB medium and recalculate the antibiotic concentrations.</P><B><DIV align="center">Protocol for LB medium preparation</DIV>
Add 20 g of LB broth into 1000 mL of desalted water and autoclave.<B><DIV align="center">Protocol for antibiotic solutions preparation</DIV><P>Kanamycin is used as an effective concentration of 10-50 µg/ml and at a storage concentration of 10 mg/ml
Carbenicillin is used as an effective concentration of 20-60 µg/ml and at a storage concentration of 50 mg/ml
Chloramphenicol is used as an effective concentration of 25-170 µg/ml and at a storage concentration of 34 mg/ml (ethanol)
Tetracyclin is used as an effective concentration of 10-50 and at a storage concentration of 5 mg/ml (ethanol)</P><H2>Tuesday, June 25th</H2><P>New competent cells have been made with the same bacterial strains (BL21, DH5α and TOP10). We thought that the contamination of our cells was due to the CaCl2 solution that may not have been sterile and/or incorrect manipulated. As a consequence, we prepared new CaCl2 solutions made sure to autoclave them before usage. We ran the same tests as monday the 24th to evaluate the quality of our work and check for potential contamination.</P><H2>Wednesday, June 26th</H2><P>The results of the competent cells protocol is as follows:
DH5α strain was not contaminated but competent
TOP10 strain was contaminated only on the Kanamycin plate but competent
BL21 strain was completely contaminated and, thus, not usable.</P><H2> Thurdsay, June 27th </H2><P>We finished designing are first two plasmids and constructed the primers to extract the natural promoters from the genomic DNA from E. coli and the different parts for Golden Gate assembly.</P><H2> Friday, June 28th </H2><P>We want to obtain reasonnable competent cells for our oncoming transformation for newt week. Thus, we decided to plate the three strains on LB-Agar medium in order to isolate one and only one colony for monday when we'll start over the whole competent process.</P><H1 id="week3">Week 3: 1<SUP>st</SUP> July - 7<SUP>th</SUP> July</H1><H2> Tuesday, July 2nd </H2><P>We prepared the BL21 and BG1655 strains to be competent.</P><H2> Wednesday, July 3rd </H2><P>The two strains have been contaminated  and are not usable.</P><H2> Thursday, July 4th </H2><P>Genomic DNA extraction of the BL21 and BG1655 strains have been done using two different methods. 
We received the primers we have ordered. They have been prepared to do a PCR. In order to get our genes of interest we did a PCR using as the DNA template the genomic DNA extracted from BL21 and BG1655.</P><H2>Friday, July 5th</H2><P>To ensure the amplification of our genes by PCR, we did a agarose gel electrophoresis. To do so, we first prepared a TAE 50X stock solution. By diluting it, we use a 1X TAE solution to do a 1% agarose gel. We obtain the following gel. That showed that only two of our genes have been amplificated by PCR. A new PCR will be made.</P><H1 id="week4">Week 4: 8<SUP>th</SUP> July - 14<SUP>th</SUP> July</H1><H2> Monday, July 8th </H2><P>We prepared the BL21 and BG1655 strains to be competent.</P><H2> Tuesday, July 9th </H2></B></B></B><TABLE id="team" cellspacing="10" align="center"><TBODY><TR><TD><P> 
        Both strain are not contaminated.
      </P></TD><TD><P>
        BL21 are highly competent, BG1655 are little competent.
      </P></TD></TR></TBODY></TABLE><H2> Wednesday, July 10th </H2><H3>PCR procedure optimization</H3><P><B><U>Aim:</U> Define the number of matrix of optimal DNA to amplify a gene by PCR.</B><B><U>Data:</U></B></P><UL><LI> [primer] = --- ng/µl</LI><LI> size(primer) = --- nt</LI></UL><P>
For PCR we chose to use … ng of primer (V=…µL). As we  know the size of our primers, we could define the number of primers that we have to use to begin the reaction.
We define the number of recquired matrix DNA in terms of concentration of sample with the following formula:


Analysis of Agarose gel's results:</P><H3> Extraction of natural promoter sequence </H3><P><B><U>Aim:</U></B>  We want to have sequences genes' promoters' sequences under the control of FUR (Ferric Uptake Regulation)transcription factor. 
We remake PCR samples that failed the previous week.</P><UL><LI> Fec A
    </LI><LI> Ent C
    </LI><LI> Fec C
    </LI><LI> Ace B
  </LI></UL><P>
PCR products are placed on gel and purificate. After, PCR products are tested with nanodrop.</P><TABLE id="Dosage PCR" cellspacing="10" align="center"><TBODY><TR><TD><P><B>Fec A</B> (BG1655)
        [c] = 23.7 ng/µl
        260/280 = 1.90</P></TD><TD><P><B>Fec A</B> (BL21)
        [c] = 53.1 ng/µl
        260/280 = 1.73</P></TD><TD><P><B>Ent C</B> (BG1655)
        [c] = 52.4 ng/µl
        260/280 = 1.71</P></TD><TD><P><B>Ent C</B> (BL21)
        [c] = 50.0 ng/µl
        260/280 = 1.94</P></TD><TD><P><B>Ace B</B> (BG1655)
        [c] = 46.5 ng/µl
        260/280 = 1.89</P></TD><TD><P><B>Fec C</B> (BG1655)
        [c] = 34.2 ng/µl
        260/280 = 1.91</P></TD></TR></TBODY></TABLE><H3>Tris-HCl solution preparation(1M) : Stock solution 100X</H3><P><B><U>Aim:</U></B> Tris-HCl solution will be use in a final concentration of 10 mM to resuspend our primers.</P><U>Preparation of the stock solution (1 M, pH 7.5) for a volume of 50 mL:</U><OL><LI> Disolve 4,6g of Trisbase in 30 mL of distilled water.
     </LI><LI> Adjust pH with concentrated HCl(~4 mL) until pH=7,5.
     </LI><LI>Add distilled water until 50 mL then put in autoclave.
  </LI></OL><P><B><U>Note: </U> If the solution has a yellow coloration, do the preparation again with better Trisbase.</B><U>Preparation of the solution (10mM, pH 7.5) for a volume of 50 mL.</U>
Take 50µL of the stock solution at 1M and add 49.5 mL of water for the dilution.
</P><H3>Kanamycin preparation</H3><P>
Preparation of 3 tubes of 1.5 mL of Kanamycin.
</P><H2>Thursday, July 11</H2><H3>Preparation of primers solution:</H3><OL><LI>Centrifugate tubes at 8000 rpm, begin 30 seconds.
    </LI><LI>Add 250 µL of Tris HCl (10 mM); [Primer] = 100µM
  </LI></OL><P>
We prepare a diluted solution at 5 µM from to the stock solution, for PCR reactions. 
Dilution at 1/20: We take 5 µL of the stock solution (100 µM) and we diluate in 95 µL of tris-HCl (10 mM).

</P><H3>Transformation of BL21:</H3><P>
Chemical transformation of BL21 with Cyrille's samples:

  <UL><LI> Terminator (T)
    </LI><LI> Promotor (P)
    </LI><LI> Plasmid 1K3
    </LI><LI> Plasmid 1C3
    </LI><LI> sfGFP
  </LI></UL></P><P>
These transformations allow us to do glycerols of these constrctions.
</P><H3>PCR on E.coli genom (BG1655 strain)</H3><UL><LI> Fep A (Primers P021 and P022)
    </LI><LI> Fes (Primers P023 and P024)
    </LI><LI> sdh C (Primers P025 and  P026)
    </LI><LI> ybi L (Primers P027 and P028)
    </LI><LI> ync E (Primers P029 and P030)
  </LI></UL><H2>Friday, July 12</H2><H3>Transformations test</H3><UL><LI> Terminator (T) = OK
    </LI><LI> Promotor (P) = OK
    </LI><LI> Plasmid 1K3 = OK
    </LI><LI> Plasmid 1C3 = OK
    </LI><LI> sfGFP = OK
  </LI></UL><P><B><U>Note :</U> The negative control was suspect.</B>

Petri dishes are le on the bench at room temperature during the week-end, colonies will be reisolate next week. </P><H3>Migration of samples of PCR extraction.</H3><P>


The promoter sequence of genes Fep A, Fes, Sdh C, ybi L and ync E were extracted successfully.
Samples were purified and, after, tested with nanodrop.</P><TABLE id="Dosage PCR" cellspacing="10" align="center"><TBODY><TR><TD><P><B>Fep A</B> (BG1655)
        [c] = 38.3 ng/µl
        260/280 = 1.93</P></TD><TD><P><B>Fes</B> (BG1655)
        [c] = 45.5 ng/µl
        260/280 = 1.89</P></TD><TD><P><B>Sdh C</B> (BG1655)
        [c] = 28.5 ng/µl
        260/280 = 1.86</P></TD><TD><P><B>ybi L</B> (BG1655)
        [c] = 31.0 ng/µl
        260/280 = 2.09</P></TD><TD><P><B>ync E</B> (BG1655)
        [c] = 25.1 ng/µl
        260/280 = 1.93</P></TD></TR></TBODY></TABLE><H1 id="week5">Week 5: 15<SUP>th</SUP> July - 21<SUP>st</SUP> July</H1><H2> Tuesday, July 16th </H2><H2> Wednesday, July 17th </H2><P>We prepared the TOP10 to be competent.</P><P>We launch our first Golden Gates.</P><H1 id="week6">Week 6: 22<SUP>nd</SUP> July - 28<SUP>th</SUP> July</H1><H2> Monday, July 22nd </H2><P>
We did a PCR on the seven products of golden gate 1 using the VR and VF2 primers. We mixed :
</P><UL><LI>10 uL of One Taq Buffer 10X
</LI><LI>1 uL of 10 mM dNTPs
</LI><LI>1 uL of each VF2 and VR primers
</LI><LI>35,5 uL of H2O
</LI><LI>1 uL of the One Taq enzyme
</LI><LI>1 uL of the golden gate products
</LI></UL><P>
We used the following PCR program :
</P><P><OL><LI> 95°C for 3 min
</LI><LI> 95°C for 15 sec
</LI><LI> 55°C for 30 sec
</LI><LI> 68°C for 2 min
</LI><LI> 68°C for 10 min
</LI></OL></P><P><B><U>Note:</U> We repeated step 2, 3 and 4 29 times.</B></P><P>
The gel migration didn't reveal any band. We supposed that it hadn't worked due to the annealing temperature we used.
Indeed, with the help of the New England BioLabs Tm Calculator website, we have found that the optimal annealing temperature for those primers was 53°C.      
</P><H2> Tuesday, July 23rd </H2><P>
In order to test the best annealing temperature for our following PCRs, we did a PCR on our 5 first golden gate products with two different annealing temperature : 53 or 51°C. We also prepared two positiv controls for each conditions with either bacteria transformed by empty PSB1A3 plasmid or 1 uL of the miniprep plasmid. This was realised in order to verify if the problem of our first PCR was due to a bad lysis of our bacteria. What is more, we prepared one negativ control with an annealing temperature of 55°C.</P><P>According to our results, we can assume that our first PCR was badly realised due to a handling error and not because of the annealing temperature we used.</P><P>In the meantime, we did the Golden Gate 1 again in order to obtain better ligation results. We transformed TOP 10 E. coli with the golden gate products and plated them on LB-Agar with carbenicillin antibiotic. The petri dishes have been let overnight at 37°C.
</P><H2> Wednesday, July 24th </H2><P> We did a colony PCR from colony obtained on the petri dishes plated with golden gate 2 products transformed bacteria. We chose one white colony of each of the first ten different FUR BS constructions. The positiv control was prepared by using the PSB1A3 plasmid. The same PCR program was used but we were out of One Taq so we utilize Dream Taq. </P><P>We re isolate colony of bacteria transformed with our golden gate 1 products.</P><H2> Thursday, July 25th </H2><P> We did a 1% gel in order to migrate our PCR products. It revealed that our PCR hadn't work. We then realised that we hadn't changed the elongation temperature of our PCR program. Indeed, the dream taq functions at 72°C.             

</P><P> We did a PCR colony with the white colony obtained after we plated our golden gate 1 products transformed bacteria.
</P><P>
We used the following PCR program :
</P><P><OL><LI> 95°C for 3 min
</LI><LI> 95°C for 15 sec
</LI><LI> 55°C for 30 sec
</LI><LI> 72°C for 2 min
</LI><LI> 72°C for 10 min
</LI></OL></P><P> Considering our gel, we can reasonably think that our golden gate had worked properly. We then did miniprep. 

</P><H2> Friday, July 26th </H2><P>We used nanodrop to measure the concentration of plasmid of our minipreps. The concentrations obtained were extremely low. We decided to do it again, but we got the same low results.</P><H1 id="week7">Week 7: 29<SUP>th</SUP> July - 4<SUP>th</SUP> August</H1><H2>Construction of plasmid N°1</H2><P>We make an electrophoresis with 5 µL of plamsid to check the plasmid purification made on the last friday.</P><P>There is not the 3 bandes that we sould see, so to check another time, we make a digestion with Pst I and EcoR I:
  <UL><LI> NEB Buffer 10X  5 µL  </LI><LI> BSA 10X  5 µL  </LI><LI> DNA  2 µg  </LI><LI> PstI  2 µL </LI><LI> EcoRI  2 µL </LI><LI> Water  50 µL </LI></UL>

After another electrophoresis, we obtain no band.</P><H3>Sequencing preparation</H3><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="4" align="center">
     AceB Fur Binding Site
    (with sfGFP)</TD><TD align="center">
    1
   </TD><TD align="center">
    166.1 ng/µL
   </TD><TD align="center">
    1.86
   </TD><TD align="center">
    1.76
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    192.0 ng/µL
   </TD><TD align="center">
    1.62
   </TD><TD align="center">
    1.02
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    123.5 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.75
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    128.0 ng/µL
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.66
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="4" align="center">
     FepA Fur Binding Site
    (with sfGFP)</TD><TD align="center">
    1
   </TD><TD align="center">
    21.1 ng/µL
   </TD><TD align="center">
    2.06
   </TD><TD align="center">
    -
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    22.4 ng/µL
   </TD><TD align="center">
    1.96
   </TD><TD align="center">
    -
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    55.4 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.58
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    68.5 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.67
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="4" align="center">
     Fes Fur Binding Site
    (with sfGFP)</TD><TD align="center">
    1
   </TD><TD align="center">
    39.9 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.80
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    40.6 ng/µL
   </TD><TD align="center">
    1.74
   </TD><TD align="center">
    -
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    68.4 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.63
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    20.8 ng/µL
   </TD><TD align="center">
    2.05
   </TD><TD align="center">
    -
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD rowspan="4" align="center">
     ybiL Fur Binding Site
    (with sfGFP)</TD><TD align="center">
    1
   </TD><TD align="center">
    58.2 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.73
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    16.3 ng/µL
   </TD><TD align="center">
    1.95
   </TD><TD align="center">
    -
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    18.8 ng/µL
   </TD><TD align="center">
    1.95
   </TD><TD align="center">
    -
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    45.5 ng/µL
   </TD><TD align="center">
    1.90
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="4" align="center">
     FecA Fur Binding Site
    (with sfGFP)</TD><TD align="center">
    1
   </TD><TD align="center">
    20.0 ng/µL
   </TD><TD align="center">
    1.96
   </TD><TD align="center">
    -
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    27.5 ng/µL
   </TD><TD align="center">
    1.77
   </TD><TD align="center">
    -
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    31.5 ng/µL
   </TD><TD align="center">
    1.94
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    57.0 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.64
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="3" align="center">
     yncE Fur Binding Site
    (with sfGFP)</TD><TD align="center">
    1
   </TD><TD align="center">
    92.4 ng/µL
   </TD><TD align="center">
    1.90
   </TD><TD align="center">
    1.75
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    64.7 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.66
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    38.6 ng/µL
   </TD><TD align="center">
    1.91
   </TD><TD align="center">
    1.80
   </TD><TD align="center">
    Good
   </TD></TR></TBODY></TABLE><H2>Construction of plasmid N°2</H2><P>
Our plasmid N°2 is building with a synthetic promote sequence which is composed of:
  <UL><LI> Andersen's promotor </LI><LI> Fur Binding Site (15 different)</LI><LI> RBS </LI><LI> sfGFP </LI><LI> Terminator </LI></UL></P><H3>Golden Gate</H3><P>
In order to associate these sequences we performed a golden gate, using for each sample :
  <UL><LI> 80 ng of Andersen's promotor
    </LI><LI> 80 ng of Fur Binding Site
    </LI><LI> 80 ng of RBS
    </LI><LI> 80 ng of sfGFP
    </LI><LI> 80 ng of Terminator
    </LI><LI> 1.5 µL of T4 buffer (10X)
    </LI><LI> 15 Unit of T4 ligase
    </LI><LI> 2.5 Unit of Bsa I
  </LI></UL></P><P>
The Golden Gate products are chemically transformed into E. coli Top 10 strains. After the transformation process, bacteria are plated into LB medium with carbenicillin and icubated overnigth at 37°C.
</P><P>
Golden Gates plates are composed at 98% of red colonies.
The problem was probably on the equimolarity ratio between our different part which were not totally respected.
We isolated white colonies (4 maximum by each sample) on LB medium with carbenicillin and we incubated them overnight at 37°c.
</P><P>
We made pre culture (V = 10 mL), using bacterial colonies isolated from our plates of Golden Gate transformation. Then minipreps and glycerol conservation were made using these pre cultures. 
</P><TABLE width="100%" cellpadding="10" align="center" border="1"><TBODY><TR><TH align="center"><B>Fur Binding Site</B></TH><TH align="center"><B>Sequence</B></TH><TH align="center"><B>Clone</B></TH><TH align="center"><B>Concentration</B></TH></TR><TR><TD rowspan="3" align="center"><P> 
        Fur BS 1
      </P></TD><TD rowspan="3" align="center"><P> 
        ATTATTGATAACTATTTG
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        80.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        83.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°4
      </P></TD><TD align="center"><P> 
        90.3 ng/µL
      </P></TD></TR><TR><TD rowspan="3" align="center"><P> 
        Fur BS 2
      </P></TD><TD rowspan="3" align="center"><P> 
        GATAACTATTTGCATTTGCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        68.6 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        77.4 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°3
      </P></TD><TD align="center"><P> 
        68.0 ng/µL
      </P></TD></TR><TR><TD rowspan="3" align="center"><P> 
        Fur BS 3
      </P></TD><TD rowspan="3" align="center"><P> 
        CAAATGCAAATAGTTATCCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        79.4 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        110.5 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°3
      </P></TD><TD align="center"><P> 
        457.4 ng/µL
      </P></TD></TR><TR><TD rowspan="2" align="center"><P> 
        Fur BS 4
      </P></TD><TD rowspan="2" align="center"><P> 
        CAAATAGTTATCAATAATCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        73.2 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        66.2 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 5
      </P></TD><TD align="center"><P> 
        GTAAATTAATATTATTTACA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        57.2 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 6
      </P></TD><TD align="center"><P> 
        AATAATGCTTCTCATTTTCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        63.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 7
      </P></TD><TD align="center"><P> 
        ATAAATGATAATCATTATCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        70.6 ng/µL
      </P></TD></TR><TR><TD rowspan="4" align="center"><P> 
        Fur BS 8
      </P></TD><TD rowspan="4" align="center"><P> 
        GAAAATAATTCTTATTTCCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        75.6 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        87.2 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°3
      </P></TD><TD align="center"><P> 
        48.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P>
        Clone n°4
      </P></TD><TD align="center"><P>
        57.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 9
      </P></TD><TD align="center"><P> 
        AGCACTTATTATTATTTTCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        58.6 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 10
      </P></TD><TD align="center"><P> 
        GATAATTGTTATCGTTTGCA
      </P></TD><TD align="center"><P> 
        No Clone
      </P></TD><TD align="center"><P> 
        -
      </P></TD></TR><TR><TD rowspan="3" align="center"><P> 
        Fur BS 11
      </P></TD><TD rowspan="3" align="center"><P> 
        GATAATGCTTATCAAAATCA
      </P></TD><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        75.6 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°3
      </P></TD><TD align="center"><P> 
        398.0 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°4
      </P></TD><TD align="center"><P> 
        57.4 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 12
      </P></TD><TD align="center"><P> 
        CAAAATTATTATCACTTTCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        50.0 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Fur BS 13
      </P></TD><TD align="center"><P> 
        AATAATGATTACCATTCCCA
      </P></TD><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        70.7 ng/µL
      </P></TD></TR><TR><TD rowspan="4" align="center"><P> 
        Fur BS 14
      </P></TD><TD rowspan="4" align="center"><P> 
        GAAATTGTTTTTGATTTTCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        42.7 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        32.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°3
      </P></TD><TD align="center"><P> 
        37.8 ng/µL
      </P></TD></TR><TR><TD align="center"><P>
        Clone n°4
      </P></TD><TD align="center"><P>
        45.7 ng/µL
      </P></TD></TR><TR><TD rowspan="3" align="center"><P> 
        Fur BS 15
      </P></TD><TD rowspan="3" align="center"><P> 
        GTTAATTGTAATGATTTTCA
      </P></TD><TD align="center"><P> 
        Clone n°1
      </P></TD><TD align="center"><P> 
        37.0 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°2
      </P></TD><TD align="center"><P> 
        53.2 ng/µL
      </P></TD></TR><TR><TD align="center"><P> 
        Clone n°3
      </P></TD><TD align="center"><P> 
        196.0 ng/µL
      </P></TD></TR></TBODY></TABLE><H2>Construction of plasmid N°3</H2><H3>29/07/13</H3><P>
We received the primers ordered on friday the 26th of July and started to dilute them into TrisCL at 10 mM. Secondly, we diluted the previous stock solution at a rate of 1/20th to obtain a final concentration of 5 µM for our intermediate solution.</P><P>
Then, for our plasmid three construction, we need to first extract the 6 enterobactin gene (EntA, EntB, EntC, EntD, EntE and EntF). The ordered primers from friday will theoretically extract them in the appropriate Golden Gate format with their own RBS upstream (designed from Salis RBS). This will allow us to construct our two N°3 plasmids, one containing EntA, EntD and EntF, and the other one EntB, EntC and EntE. The genes are spread like this to obtain two equivalent plasmids xxx</P><P>
We proceeded to a genomic extraction of the 6 genes of interest and migrated to PCR products on a 1% gel. We successfully extracted 5 out of the 6.</P><H3>30/07/13</H3><P>
We annealled the oligonucleotides of the PL-LacO part in the golden gate format. Also, we extracted the sfGFP with the RBS upstream which will allow us to construct the control positive plasmid 3 for future TECAN experiments.</P><H3>31/07/13</H3><P>
We started by purifying our 6 succesful PCR extraction from the 29/07 and 30/07. So we managed to extract EntA, EntB, EntC, EntD and EntF from E. coli's genomic DNA and sfGFP in the golden gate format with a RBS (plasmid 3 construction) from plasmidic DNA.</P><P>
Additionnaly, we optimized our PCR to extract our missing gene, EntE. We obtained a smear +/- a double band. As a consequence, we adjusted the annealing temperature with a range from 54°C to 66°C. This will allow us to try extracting the missing gene (EntE) in the appropriate conditions.</P><H3>01/08/13</H3><P>
We made a PCR for the EntE gene with a gradient of temperature to know at which temperature the PCR result were the best. 
We made 8 (ou mettre 6 ?) tubes with the same composition:
  <UL><LI> Water </LI><LI> Taq Buffer 5 </LI><LI> dNTPs </LI><LI> Rev EntE 2,5 </LI><LI> For EntE 2,5 </LI><LI> DNA 0,5 </LI><LI> Taq polymerase 0,5 </LI></UL>
We also made positive and negative controles (with Rev EntC and For EntC instead of EntE).
After electrophoresis, we obtain the best band at .. °C.
However our experiment have been made with Taq polymerase (and Taq Buffer) instead of Q5 (and Q5 buffer) that we use usually. Then, we have to repeat this experiment with Q5 and Q5 buffer. 
The did a gradient PCR and migrated on a gel our samples. On the gel, we notice that both the smear and the double band disappear when the annealing temperature increases. So to conclude, we run tonight a PCR with Q5 polymerase with increasing temperature starting from 56 degrees. The absolute annealing temperature will be determined for the Q5 (not extrapolable from a OneTaq PCR).</P><P>
Also, we made a golden gate for the construction of plasmid 3. For analysis purposes, only a GFP was added downstream of the PL LacO promoter. Also, the EntE gene is still to be extracted from the genome. Also, we runned 3 extras golden gate for the construction of plasmid 2. In fact, we corrected our calculations because we initially we used mass concentrations instead of molar concentrations.</P><H3>02/08/13</H3><P>
The transformations of the golden gate 2 and 3 are successful and we managed to invert the rate of red colonies versus white colonies (red = false positive with mRFP, white = positive), thus meaning we have a lot of white ones! We conserved the plates for future isolation on coming monday.</P><P>
Additionnaly, we migrated the PCR products of 01/08. The gel is empty, but the positive controls are there, meaning we made a mistake in the primers (the master mix is the one used for the positive control).</P><H1 id="week8">Week 8: 5<SUP>th</SUP> August - 11<SUP>th</SUP> August</H1><H2>Plasmid 3:</H2><P>
05/08/13
We started the week by doing the golden gates over again, meaning the GG 3 and GG 2 (2.1 with FurBS 1, 2.2 with FurBS2 and 2.3 with FurBS3). In fact, our previous control positive had some unwanted white spots (01/08/13), thus suggesting some contaminations during the transformation step. curretnmy waiting
</P><P>
06/08/13
We performed 6 golden gates, 3 for the second construction, 2 for the third construction and one for the controle plasmid.
We prepared a mix for the 6+1 tubes :
  <UL><LI> 7x 1,27 = 8,89 µl of 1A3 plasmide 
    </LI><LI> 7x 1,74 = 12,18 µL of terminator
    </LI><LI> 7x 1,5 = 10,5 µL of T4 Buffer 
    </LI><LI> 7 x 0,5 = 3,5 µL of BSA
    </LI><LI> 7 x 0,5 = 3,5 µL of T4 ligase
  </LI></UL></P><P>

We then added:

Tube 1. = 
  <UL><LI>  0,76 µl of Andersen's promoter 
    </LI><LI>  0,76 µL of RBS
    </LI><LI>  0,76 µl OF Fur BS 
    </LI><LI> 2,93 µL of sfGFP
  </LI></UL></P>

Tube 2. =
  <UL><LI>  0,76 µl of Andersen's promoter 
    </LI><LI>  0,76 µL of RBS
    </LI><LI>  0,76 µl OF Fur BS 
    </LI><LI> 2,93 µL of sfGFP
  </LI></UL>

Tube 3. =
  <UL><LI>  0,76 µl of Andersen's promoter 
    </LI><LI>  0,76 µL of RBS
    </LI><LI>  0,76 µl OF Fur BS 
    </LI><LI> 2,93 µL of sfGFP
   </LI><LI>   4,28 µL of water
  </LI></UL>

Tube 4. =
  <UL><LI>  1,25 µl of pLac O 
    </LI><LI>   µL of RBS-sfGFP
   </LI><LI>   3,4 µL of water
  </LI></UL>

Tube 5. =
  <UL><LI>  1,25 µl of pLac O 
    </LI><LI>  2,05 µl of EntA 
    </LI><LI>  3,37 µL of EntD
    </LI><LI>  2,07 µl of EntF 
    </LI><LI> 0,5 µL of water
  </LI></UL>

Tube 6. = Controle Plasmid
  <UL><LI>  0,76 µl of Andersen's promoter 
    </LI><LI>  0,76 µL of RBS
    </LI><LI>  2,93 µl of sfGFP 
    </LI><LI> 4,28 µL of water
  </LI></UL>

On the afternoon, we launch another golden gate for EntB/C/E

Tube 7. = 
  <UL><LI>  1,27 µl of plasmid 1A3 
    </LI><LI>  1,5 µL of pLacO
    </LI><LI>  0,49 µl of EntB 
    </LI><LI> 0,65 µL of EntC
    </LI><LI>  1,74 µl of terminator 
    </LI><LI>  1,5 µL of Buffer T4 ligase
    </LI><LI>  0,5 µl of T4 ligase 
    </LI><LI> 0,5 µL of Bsa 1
    </LI><LI> 5,88 µL of water
  </LI></UL><P>
Additionnally, we did the optimization of our PCR over with the temperature gradient for the annealing step. current waiting
</P><P>
09/08
We make 38 tube of Top 10
</P><H1 id="week9">Week 9: 12<SUP>th</SUP> August - 18<SUP>th</SUP> August</H1><H2>TECAN</H2><H3>Medium preparation</H3><P>
For the Tecan analysis we must use M9 medium and not a classical LB medium because turbidity and fluorescence of our sample are measured. Then, in order to obtain good results, the medium must not emit a side signal, that is why we use the M9 medium.
</P><P><U>Composition for 50 mL of:</U></P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Reagent
   </TH><TH align="center">
    M9 medium (without iron)
   </TH><TH align="center">
    M9 medium (with iron)
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    M9 salt (5X)
   </TD><TD colspan="2" align="center">
    10 mL
   </TD></TR><TR><TD align="center">
    CaCl<SUB>2</SUB> (1M)
   </TD><TD colspan="2" align="center">
    5 µL
   </TD></TR><TR><TD align="center">
    MgSO<SUB>4</SUB> (1M)
   </TD><TD colspan="2" align="center">
    100 µL
   </TD></TR><TR><TD align="center">
    Glycerol (50%)
   </TD><TD colspan="2" align="center">
    800 µL
   </TD></TR><TR><TD align="center">
    Thiamine
   </TD><TD colspan="2" align="center">
    5 µL
   </TD></TR><TR><TD align="center">
    NaOH (pH 7.4)
   </TD><TD colspan="2" align="center">
    12.5 µL
   </TD></TR><TR><TD align="center">
    H<SUB>2</SUB>O
   </TD><TD align="center">
    40 mL
   </TD><TD align="center">
    39 mL
   </TD></TR><TR><TD align="center">
    FeSO<SUB>4</SUB> (10mM)
   </TD><TD align="center">
    -
   </TD><TD align="center">
    50 µL
   </TD></TR><TR><TD align="center">
    Casamino acids (0.2%)
   </TD><TD align="center">
    -
   </TD><TD align="center">
    1 mL
   </TD></TR></TBODY></TABLE><P>
Once the mixture is prepared, the medium must be filtered to be sterilised using 0.22 µm filter.
</P><H3>TECAN analysis</H3><P>
Using the TECAN analysis, we are measuring the capacity of repression of the natural binding site that we extract from the <I>E.coli</I>'s genome.
</P><P>
Preculture with M9 medium (with carbenicillin) have been launched for BL21 transformed with our 1<SUP>st</SUP> construction:
</P><UL><LI>Natural Fur Binding Site of Fec A + sfGFP (clone 1, 2, 3)
</LI><LI>Natural Fur Binding Site of Fep A + sfGFP (clone 1, 2, 3)
</LI><LI>Natural Fur Binding Site of Ace B + sfGFP (clone 1, 2, 3)
</LI></UL><P><B>Note:</B> Preculture have been made in M9 medium with iron in oder to inhibit the expression of sfGFP.
</P><P>
After one night of culture, time the precultures have been refreshed by diluting them 200 times in M9 medium (with iron and carbenicillin).
</P><P>
After 8 hours of culture, the 96 wells plate has been prepare (see following scheme).
</P><H2>1<SUP>st</SUP> construction</H2><H3>Sequencing preparation</H3><P>
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our 1<SUP>st</SUP> construction, then minipreps have been realized using the miniprep kit from Machrey Nagel.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="4" align="center"><P>AceB Fur Binding Site
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    66.4 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.92
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center"> 2
   </TD><TD align="center">
    53.8 ng/µL
   </TD><TD align="center">
    1.85
   </TD><TD align="center">
    1.61
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    64.9 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    71.8 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.66
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="4" align="center"><P>ybiL Fur Binding Site
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    49.1 ng/µL
   </TD><TD align="center">
    1.94
   </TD><TD align="center">
    1.80
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    58.6 ng/µL
   </TD><TD align="center">
    1.99
   </TD><TD align="center">
    2.32
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    41.2 ng/µL
   </TD><TD align="center">
    1.97
   </TD><TD align="center">
    2.44
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    31.8 ng/µL
   </TD><TD align="center">
    1.93
   </TD><TD align="center">
    2.16
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="4" align="center"><P>yncE Fur Binding Site
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    43.7 ng/µL
   </TD><TD align="center">
    1.95
   </TD><TD align="center">
    2.17
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    45.6 ng/µL
   </TD><TD align="center">
    1.97
   </TD><TD align="center">
    2.22
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    34.8 ng/µL
   </TD><TD align="center">
    2.00
   </TD><TD align="center">
    2.41
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    42.5 ng/µL
   </TD><TD align="center">
    2.00
   </TD><TD align="center">
    2.28
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fes Fur Binding Site
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    39.4 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    36.6 ng/µL
   </TD><TD align="center">
    -
   </TD><TD align="center">
    -
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    35.3 ng/µL
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    1.13
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD rowspan="2" align="center"><P>FepA Fur Binding Site
    (with LacI-LVA)</P></TD><TD align="center">
    2
   </TD><TD align="center">
    50.7 ng/µL
   </TD><TD align="center">
    1.90
   </TD><TD align="center">
    1.61
   </TD><TD align="center">
    Good
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    57.4 ng/µL
   </TD><TD align="center">
    1.92
   </TD><TD align="center">
    1.88
   </TD><TD align="center">
    Good
   </TD></TR></TBODY></TABLE><H2>2<SUP>nd</SUP> construction</H2><H3>Bacterial isolation</H3><P>
4 clones of a Golden Gate plate are isolated for each construction. It allows us to save the clone on a plate.
</P><DIV align="center"><I>Top 10 transformed with the synthetic Fur BindingSite n°1, 2 and 3 isolated on a plate.</I></DIV><H3>Sequencing preparation</H3><P>
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our 2<SUP>nd</SUP> construction, then minipreps have been realized using the miniprep kit from Machrey Nagel.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°1
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    50.7 ng/µL
   </TD><TD align="center">
    1.62
   </TD><TD align="center">
    0.68
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    21.1 ng/µL
   </TD><TD align="center">
    1.69
   </TD><TD align="center">
    0.44
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    21.6 ng/µL
   </TD><TD align="center">
    1.20
   </TD><TD align="center">
    0.13
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    70.8 ng/µL
   </TD><TD align="center">
    1.66
   </TD><TD align="center">
    0.72
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°2
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    39.1 ng/µL
   </TD><TD align="center">
    1.23
   </TD><TD align="center">
    0.40
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    25.1 ng/µL
   </TD><TD align="center">
    1.37
   </TD><TD align="center">
    0.29
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    61.1 ng/µL
   </TD><TD align="center">
    1.57
   </TD><TD align="center">
    0.71
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    54.7 ng/µL
   </TD><TD align="center">
    1.62
   </TD><TD align="center">
    0.68
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fur Binding Site n°3
    (with sfGFP)</P></TD><TD align="center">
    2
   </TD><TD align="center">
    38.4 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    0.78
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    114.6 ng/µL
   </TD><TD align="center">
    1.66
   </TD><TD align="center">
    0.78
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    39.2 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.21
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fur Binding Site n°5
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    77.7 ng/µL
   </TD><TD align="center">
    1.76
   </TD><TD align="center">
    0.99
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    78.4 ng/µL
   </TD><TD align="center">
    1.61
   </TD><TD align="center">
    0.81
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    72.7 ng/µL
   </TD><TD align="center">
    1.85
   </TD><TD align="center">
    4.23
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°6
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    70.4 ng/µL
   </TD><TD align="center">
    1.76
   </TD><TD align="center">
    2.16
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    91.5 ng/µL
   </TD><TD align="center">
    1.55
   </TD><TD align="center">
    1.21
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    65.4 ng/µL
   </TD><TD align="center">
    1.70
   </TD><TD align="center">
    1.52
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    85.6 ng/µL
   </TD><TD align="center">
    1.72
   </TD><TD align="center">
    1.65
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fur Binding Site n°7
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    53.5 ng/µL
   </TD><TD align="center">
    1.76
   </TD><TD align="center">
    1.93
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    77.6 ng/µL
   </TD><TD align="center">
    1.69
   </TD><TD align="center">
    1.22
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    72.3 ng/µL
   </TD><TD align="center">
    1.68
   </TD><TD align="center">
    1.22
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°8
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    73.9 ng/µL
   </TD><TD align="center">
    1.58
   </TD><TD align="center">
    0.84
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    51.3 ng/µL
   </TD><TD align="center">
    1.75
   </TD><TD align="center">
    1.10
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    45.9 ng/µL
   </TD><TD align="center">
    1.85
   </TD><TD align="center">
    1.43
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    46.1 ng/µL
   </TD><TD align="center">
    1.88
   </TD><TD align="center">
    1.61
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°9
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    108.3 ng/µL
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    1.37
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    67.0 ng/µL
   </TD><TD align="center">
    1.88
   </TD><TD align="center">
    1.61
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    103.5 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.79
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    28.2 ng/µL
   </TD><TD align="center">
    2.06
   </TD><TD align="center">
    1.76
   </TD><TD align="center">
    No sequencing
  </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°10
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    29.5 ng/µL
   </TD><TD align="center">
    1.95
   </TD><TD align="center">
    1.22
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    70.0 ng/µL
   </TD><TD align="center">
    1.90
   </TD><TD align="center">
    1.67
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    46.0 ng/µL
   </TD><TD align="center">
    1.89
   </TD><TD align="center">
    1.62
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    55.8 ng/µL
   </TD><TD align="center">
    1.88
   </TD><TD align="center">
    1.44
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°11
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    55.3 ng/µL
   </TD><TD align="center">
    1.85
   </TD><TD align="center">
    1.49
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    87.1 ng/µL
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    1.37
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    57.7 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.44
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    21.7 ng/µL
   </TD><TD align="center">
    2.06
   </TD><TD align="center">
    1.20
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fur Binding Site n°15
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    16.2 ng/µL
   </TD><TD align="center">
    2.13
   </TD><TD align="center">
    1.20
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    47.4 ng/µL
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    1.16
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    63.2 ng/µL
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    1.16
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°1
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    99.7 ng/µL
   </TD><TD align="center">
    1.73
   </TD><TD align="center">
    0.99
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    46.9 ng/µL
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    0.90
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    55.8 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.75
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    35.3 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    0.99
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fur Binding Site n°2
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    43.8 ng/µL
   </TD><TD align="center">
    1.83
   </TD><TD align="center">
    1.01
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    47.0 ng/µL
   </TD><TD align="center">
    1.77
   </TD><TD align="center">
    0.92
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    32.4 ng/µL
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    1.09
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°3
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    57.8 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.45
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    51.3 ng/µL
   </TD><TD align="center">
    1.79
   </TD><TD align="center">
    1.04
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    41.6 ng/µL
   </TD><TD align="center">
    1.88
   </TD><TD align="center">
    1.75
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    112.6 ng/µL
   </TD><TD align="center">
    1.86
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    Bad
   </TD></TR></TBODY></TABLE><P><B>Note:</B> Fur Binding Sites n°4, 12, 13 and 14 are missing.
</P><H2>3<SUP>rd</SUP> construction</H2><H3>Sequencing preparation</H3><P>
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our plasmid which contains the sfGFP under the control of Lac O promoter. After one night of culture minipreps of 3 different clones have been realized using the miniprep kit from Machrey Nagel.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="3" align="center"><P>Testing construction
    (Lac O with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    25.7 ng/µL
   </TD><TD align="center">
    2.02
   </TD><TD align="center">
    2.29
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    27.0 ng/µL
   </TD><TD align="center">
    2.09
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    79.7 ng/µL
   </TD><TD align="center">
    1.86
   </TD><TD align="center">
    1.60
   </TD><TD align="center">
    Good
   </TD></TR></TBODY></TABLE><H3>Golden Gate</H3><P>
A new Golden Gate of our 3<SUP>rd</SUP> construction as been made for the two plasmids containing:
</P><OL><LI>Enterobactin biosynthesis genes (Ent A, D and F)
</LI><LI>Enterobactin biosynthesis genes (Ent B, C and E)
</LI></OL><P>
5 µL of each Golden Gate product are transformed into Top10 competent cells. To controlled the quality of the transformation, a negative controle (transformation procedure without plasmid) and a positive controle (transformation procedure with plasmid 1A3) are made.
</P><P>
Unfortunatly, the Golden Gate reactions did not work.
</P><H2>Others</H2><H3>Miniprep of Plasmid 1A3 with RFP designed for Golden Gates</H3><P>
Top 10 transformed with the plasmid 1A3 designed for Golden Gate have been cultivated into 10 mL of LB medium (with carbenicillin). After an overnight culture, plasmid has been minipreped using the Machrey Nagel kit.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Sample
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    1
   </TD><TD align="center">
    265 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.78
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    237.8 ng/µL
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.67
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    240.8 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.81
   </TD></TR></TBODY></TABLE><P>
The plasmid has been diluted as a working solution with a concenration of 80 ng/µL. Then the plasmid has been aliquoted in 50 µL into 9 different tubes.
</P><H3>Competent cells</H3><P>
Top10 competent cells had been made last week and we tested them on different antibiotics (Carbenicillin, Kanamycin, Chloramphenicol). After incubation at 37°C, overnight, no contaminant had been observed.
</P><H1 id="week10">Week 10: 19<SUP>th</SUP> August - 25<SUP>th</SUP> August</H1><H2>2<SUP>nd</SUP> Construction</H2><H3>Primer hybridation</H3><P>
After sequencing analysis we observed that our Golden Gate results were not significant for the 2<SUP>nd</SUP> construction.
However, our sequences of natural promoter give us good results. Then we supposed that the problem of our Golden Gate come from the hybridation of our synthetic promoters.
We made new hybridations of:
</P><UL><LI>Fur Binding Site 1 to 15 (5 µM)
</LI><LI>Andersen promoter J23100 (5 µM)
</LI><LI>RBS 0034 (5 µM)
</LI></UL><H3>Golden Gate</H3><P>
A new Golden Gate of our 2<SUP>nd</SUP> construction are made with:
</P><UL><LI>Fur Binding Site 1, 2 or 3 + sfGFP
</LI><LI>Fur Binding Site 1, 2 or 3 + LacI-LVA
</LI></UL><P>
5 µL of each Golden Gate product are transformed into Top10 competent cells. To controlled the quality of the transformation, a negative controle (transformation procedure without plasmid) and a positive controle (transformation procedure with plasmid 1A3) are made.
</P><H3>Sequencing preparation</H3><P>
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our 2<SUP>nd</SUP> construction, then minipreps have been realized using the miniprep kit from Machrey Nagel.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD align="center"><P>Fur Binding Site n°1
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    20.2 ng/µL
   </TD><TD align="center">
    2.07
   </TD><TD align="center">
    1.29
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°1
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    28.5 ng/µL
   </TD><TD align="center">
    2.06
   </TD><TD align="center">
    1.65
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    25.5 ng/µL
   </TD><TD align="center">
    2.01
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    45.6 ng/µL
   </TD><TD align="center">
    1.91
   </TD><TD align="center">
    1.90
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
  </TD><TD align="center">
    54.7 ng/µL
   </TD><TD align="center">
    1.83
   </TD><TD align="center">
    1.27
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°2
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    29.4 ng/µL
   </TD><TD align="center">
    2.01
   </TD><TD align="center">
    1.46
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    25.5 ng/µL
   </TD><TD align="center">
    2.07
   </TD><TD align="center">
    2.00
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    15.0 ng/µL
   </TD><TD align="center">
    2.21
   </TD><TD align="center">
    1.72
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    47.1 ng/µL
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.53
   </TD><TD align="center">
    -
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°2
    (with LacI-LVA)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    25.4 ng/µL
   </TD><TD align="center">
    1.72
   </TD><TD align="center">
    0.60
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    27.7 ng/µL
   </TD><TD align="center">
    1.93
   </TD><TD align="center">
    1.67
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    41.8 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.69
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    31.1 ng/µL
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.26
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fur Binding Site n°3
    (with sfGFP)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    37.8 ng/µL
   </TD><TD align="center">
    1.85
   </TD><TD align="center">
    1.40
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    78.3 ng/µL
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.85
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    53.4 ng/µL
   </TD><TD align="center">
    1.72
   </TD><TD align="center">
    0.98
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    23.3 ng/µL
   </TD><TD align="center">
    2.06
   </TD><TD align="center">
    2.53
   </TD><TD align="center">
    No sequencing
   </TD></TR><TR><TD rowspan="2" align="center"><P>Fur Binding Site n°3
    (with LacI-LVA)</P></TD><TD align="center">
    2
   </TD><TD align="center">
    43.8 ng/µL
   </TD><TD align="center">
    1.78
   </TD><TD align="center">
    1.21
   </TD><TD align="center">
    Bad
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    28.2 ng/µL
   </TD><TD align="center">
    2.15
   </TD><TD align="center">
    2.31
   </TD><TD align="center">
    -
   </TD></TR></TBODY></TABLE><H2>3<SUP>rd</SUP> construction</H2><H3>Sequencing preparation</H3><P>
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our plasmid which contains the sfGFP under the control of Lac O promoter. After one night of culture minipreps of 3 different clones have been realized using the miniprep kit from Machrey Nagel.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="4" align="center"><P>Enterobactin construction
    (Lac O with EntA, EntD, EntF)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    62.7 ng/µL
   </TD><TD align="center">
    2.02
   </TD><TD align="center">
    0.84
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    86.8 ng/µL
   </TD><TD align="center">
    1.97
   </TD><TD align="center">
    0.99
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    3
   </TD><TD align="center">
    112.2 ng/µL
   </TD><TD align="center">
    1.98
   </TD><TD align="center">
    1.23
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    78.7 ng/µL
   </TD><TD align="center">
    1.97
   </TD><TD align="center">
    1.17
   </TD><TD align="center">
    -
   </TD></TR><TR><TD rowspan="2" align="center"><P>Enterobactin construction
    (Lac O with EntB, EntC, EntE)</P></TD><TD align="center">
    1
   </TD><TD align="center">
    80.1 ng/µL
   </TD><TD align="center">
    1.92
   </TD><TD align="center">
    1.01
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    4
   </TD><TD align="center">
    330.3 ng/µL
   </TD><TD align="center">
    1.91
   </TD><TD align="center">
    1.20
   </TD><TD align="center">
    -
   </TD></TR></TBODY></TABLE><H2>Other</H2><P>
We prepared 20 tubes of BL21 competent cells.
</P><H1 id="week11">Week 11: 26<SUP>th</SUP> August - 1<SUP>st</SUP> September</H1><H2>Creation of a E. Coli ΔFur</H2><H3>Strain preparation</H3><P>
Chemically competent Top 10 <I>E. Coli</I> was transformed with the plasmid PTK which contained λ Red factor. We plated our bacteria on LB Agar with spectinomycin and let them grow overnight at 30°C. 
</P><P>
One colony had grown on our plate. We started an overnight culture of this colony in 2 mL of LB with spectinomycin and still at 30°C. 
</P><P>
We then diluted 1 mL of our culture in 100 mL of LB with spectinomycin and IPTG so the λ Red factor would start to be expressed. We let bacteria grew to a OD of 0.5 and made them electro competent. The protocol used can be found in our Protocol pages.   
</P><H3>Homologous Recombination</H3><P>
We realised a PCR on a plasmid with a kanamycin cassette resistance and with our two designed primers P119 and P120. The PCR product had been then purified and migrated on 1% agarose gel. We obtained the 1 kb band expected which confirmed that we got the right amplificat.
</P><P>
We transformed by electroporation 50 uL of our strain with 100 ng of PCR product. We had a time constant of 5,6. After a 2 hours recovery in 2 mL of LB with IPTG, we added kanamycin and let our bacteria grow overnight at 30°C.
</P><P>
Next day, our culture was very turbid. Our bacteria got a kanamycin resistance. We plated 50 uL of our culture on four kanamycin-agar petri dishes and grew overnight at 30°C.
</P><H1 id="week12">Week 12: 2<SUP>nd</SUP> September - 8<SUP>th</SUP> September</H1><H2>Creation of a E. Coli ΔFur</H2><P><B>02/09/13</B><B>04/09/13</B>
We picked colonies from our four plates, that we grew overnight at 42°C, to plate each of them on one plate with kanamycin 

</P><H2>TECAN</H2><P><B>02/09/13</B>
To further improve the interpretability of the plate reading, we need to transfer our construction from a BL21 to a TOP10 strain. As a start, we transformed the following constructions:</P><DIV style="padding-left:10%;"><P>- AceB with LacI</P></DIV><DIV style="padding-left:10%;"><P>- FecA with GFP</P></DIV><DIV style="padding-left:10%;"><P>- FepA with GFP</P></DIV><DIV style="padding-left:10%;">- FepA with LacI</DIV><DIV style="padding-left:10%;">- Fes with GFP</DIV><DIV style="padding-left:10%;">- Fes with LacI</DIV><DIV style="padding-left:10%;">- YbiL with GFP</DIV><DIV style="padding-left:10%;">- YbiL with LacI</DIV><DIV style="padding-left:10%;">- YncE with GFP</DIV><DIV style="padding-left:10%;">- YncE with LacI</DIV><DIV style="padding-left:10%;">- control positive with pSB1A3</DIV><DIV style="padding-left:10%;">- control negative</DIV><P>
We did not have anymore AceB with GFP plasmid available to realize the transformation. We pre-cultured a remaining glycerol stock at 37°C overnight:</P><DIV style="padding-left:10%;">10 mL LB</DIV><DIV style="padding-left:10%;">10 µL Carbenicillin 1000X</DIV><DIV style="padding-left:10%;">50 µL cells from glycerol stock</DIV><B>03/09/13</B><P>
The transformations succeeded and made liquid cultures for glycerol conservation.
Also, we isolated the plasmid AceB-GFP from the pre-culture of the 03/09/13. As a consequence, we transformed our last plasmid in a TOP10 strain.
</P><B>04/09/13</B><P>
To anticipate the TECAN for the evening, we started a pre-culture of the following constructions we want to characterize:
</P><DIV style="padding-left:10%;">TOP10 (growth control)</DIV><DIV style="padding-left:10%;">TOP10 PL-LacO (GFP control)</DIV><DIV style="padding-left:10%;">TOP10 YbiL-GFP</DIV><DIV style="padding-left:10%;">TOP10 YncE-GFP</DIV>
Also, we prepared the iron solutions from 10^-3 to 10^-10 M that will be used to prepare all the different media and completed the plate.
TECAN GRID<B>05/09/13</B><P>
To anticipate the TECAN for the evening, we started a pre-culture of the following constructions we want to characterize:
</P><DIV style="padding-left:10%;">TOP10 (growth control)</DIV><DIV style="padding-left:10%;">TOP10 PL-LacO (GFP control)</DIV><DIV style="padding-left:10%;">TOP10 AceB-GFP</DIV><DIV style="padding-left:10%;">TOP10 Fes-GFP</DIV>
TECAN GRID<H2>Plasmid 3</H2>

We launch another PC to get Enterobactins A, B, C, D, E and F genes.
Add the recquired primers in each tube:
<OL><LI> EntA: primers 065 and 066
</LI><LI> EntB: primers 067 and 068
</LI><LI> EntC: primers 069 and 070
</LI><LI> EntD: primers 071 and 072
</LI><LI> EntE: primers 073 and 074
</LI><LI> EntF: primers 075 and 076
</LI><LI> Positive controle: primers 009 and 010
</LI><LI> Negative controle: primers 009 and 010 </LI></OL><I>We use pEntC as our controle.</I>
For more details about our primers, see the <A href="https://2013.igem.org/Team:Evry/Primers" target="_blank">corresponding page</A>.
 
We then prepared a master mix 1 with:
<UL><LI> 9 x 27,5 = 247,5 µL of distilled water
</LI><LI> 9 x 1 = 9 µL of dNTPs
</LI><LI> 9 x 10 = 90 µL of Q<INF>5</INF> Buffer
</LI><LI> 9 x 0,5 = 4,5 µL of Q<INF>5</INF>

For the negative controle, we added 1 µL of distilled water in tube 8 and 39 µL of master mix 1.
For other tubes (6 genes and positive controle), we added 8 µL of genomic DNA in the master mix 1. Then add 40 µL of master mix 1' in each tube.<H1 id="week13">Week 13: 9<SUP>th</SUP> September - 15<SUP>th</SUP> September</H1><H2>Design of the capsule</H2><B>09/09/13</B><P>
To chelate the iron in the duodenum and the initial portion of the jejunum, it was pretty obvious that a capsule or a tablet was required. The best galenic formulation for our purposes is the capsule, meaning a delivery of our bacteria in the jejunum. The first reason is that a tablet requires not only a heavy and dry compression, but also the second reason is that it requires lyophilized bacteria, which consists of extreme conditions (from very high to very low temperature and high pressure). As a consequence, the capsule offers the possibility to be easily coated for resistance against stomachal acidity and presents a softer environment for the bacteria to transport.
</P><P>
The first step in the design of the capsule is to choose the right diluent, meaning the appropriate powder to 'dissolve' the bacteria in. We experimented both <I>maltodextrin</I> and <I>colloidal silica</I> and tried to disperse as much LB medium as possible with hand mortar and pestle (figure 1 and 2). For 10g of <I>maltodextrin</I>, we managed to dissolve 400 µL of LB medium, and for 3g of <I>colloidal silica</I> 4 mL of LB medium. We choose <I>colloidal silica</I> for 

- Réflexion le choix des bio-adhésifs: ici on utilisera les HPMC.
- Réflexion sur la qualité des bio-adhésif, notamment dans la relation viscosité/bio-adhésif. En effet, les HPMC vont gonfler en contact avec l'eau, adhérant ainsi les bactéries à la paroi. Il faut que la viscosité soit minimale pour pas que le bol alimentaire vienne arracher la fixation de nos bactéries, mais qu'en même temps l'eau ait une pénétrance suffisante.
- Réflexion sur l'enrobage (enteric coating). Il faut que la gélule se délite le plus haut possible dans l'intestin (ici le duodénum), mais qu'il soit suffisamment résistant à l'acidité de l'estomac en amont.
</P><P>
Choice of excipients (reference: Handbook of Pharmaceutical Excipients, sixth edition):
Hypromellose (Hydroxypropyl methylcellulose, HPMC): is used as a bioadhesive material for a controlled release at pH 5-8 (duodenum and jejunum).
Methacrylic acid L100-55 (Eudragit): is used as an enteric coating for resistance against stomachal pH, thus allowing the release of the bacteria in the duodenum at pH values of 5,5.
Colloidal silicon dioxide: is used to dry pellet of bacteria and allow an homogenous mixture with the other components.</P><P>
TOP10 and TOP10 transformed with pSB1A3 stayed the whole weekend at 4°C. I resuspended the cell in 10 mL LB + antibiotic.
</P><B>10/09/13</B><P>
I tested two different powders to dilute our medium with bacteria. As a start, i began to dilute LB medium, without bacteria in Maltodextrin (Glucidex) or Colloidal silica (Aerosil). After dispersing the liquid in the powder, the goal was to put as much as possible in it, but the powder should not agglomerate and should keep its powder form. As such, I managed to dilute 200 µL LB in 10 g of Maltodextrin but 3825 µL in 3 g of colloidal silica. As a consequence, we chose for colloidal silica to dilute the LB, and thus, the bacteria.
</P><P>
The second step is to obtain a final volume of 35g of powder. After I diluted 3825 µL of LB in colloidal silica, I obtained a total volume of 16 mL. I added a sufficient quantity of HPMC (Hydroxyprpylmethylcellulose, Methocel K100) to obtain a final volume of 35 mL. The two components are mixed together and I made sure the product is dry.
</P><P>
With a 'gelulier', I equally dispersed the whole volume in 50 capsules. After weight measurement, the capsules are 342 +/- 4 mg.
</P><P>
To anticipate the dispersion tests at pH = 2 and at pH = 6,5, I prepared a PBS buffer and an acid medium that mimics the gastric context and its acidty.
The capsule should not be dispersed after two hours in the medium of pH = 2. Also, the capsule, afterwards, should disperse in between 1 hour in PBS buffer. If the capsule satisfies these conditions, the bacteria will be releases right at the end of the duodenum, thus allowing the growth in the jejunum.
</P><P>
PBS preparation for a volume of 5L:
</P><DIV style="padding-left:10%;"><P>NaCl: 40g</P></DIV><DIV style="padding-left:10%;"><P>KCl: 1g</P></DIV><DIV style="padding-left:10%;"><P>Na2HPO4: 7,2g</P></DIV><DIV style="padding-left:10%;"><P>KH2PO4: 1,2g</P></DIV><P>
The first attempt to make capsules from LB is repeated, but this time with LB and bacteria. After I diluted 3000 µL of LB and bactera in colloidal silica, I obtained a total volume of 20mL. I added a sufficient quantity of HPMC (Hydroxyprpylmethylcellulose, Methocel K100) to obtain a final volume of 35 mL. The two components were mixed together. However, since the bacteria are dry in the powder, they may be very volatile when mixed to the colloidal silica. As an additional precaution, I wore a mask to prevent the inhale of these potenially contaminated particles.
</P><P>
I refreshed my cultures in 3ml LB (x2) + an additional control negative (only 3ml LB) to make sure that I am not in fact working with the wrong bacteria or any contamination. This control negative was actually very important. In this lab, I don't have the same sterilized pipet tips or 15ml tube. Also, the only sterile environnement is in the presence of a very short flame.
</P><P>
At the end of the day, I managed to create 50 capsules containing only dry LB and 50 capsules with dry LB and bacteria. Additionnaly, I prepared the alcoholic solution of 12,5% Eudragit L100-55 to create the gastro-enteric resistant coating around my capsule. The receipe is as follows:
50ml of pure ethanol (98%)
6,25g of eudragit
Contrary to the litterature, the Eudragit was not that 'extremely' soluble in pure ethanol and did not take 20mn to dissolve. As a consequence, I sonicated for 45 minutes to solubilize the Eudragit.
</P><P>
Finally, I tried to make the coating on some capsules. I came to the conclusion that the drying lasts a very long time. To accelerate the drying process, I used the hairdryer for 30 seconds at maximum temperature at average speed, then 1mn at lowest temperature at highest speed. This way i managed to dry the fastest way the coating of my capsules, without being to sticky at the end. Also, every single capsule had to be done twice, once on the upper side and another on the downside. The not only allows a better dring, but also double-covers the edge of the capsule where both sides close (in the middle) and gives a double protection at the weakest point of the capsule. Finally, keep in mind that the coating process is doubled, thus spending on average 8 to 10 minutes for each capsule.
</P><B>11/09/13</B><P>
I double-coated 6 capsules containing the dry bacteria and put them for 2 hours in the acid to test their resistance to low pH (pH = 2). They did not break during this stage.
After 20 minutes of shacking in the PBS buffer, the water starts to become turbid, thus proving the degradation of the gelatin of the capsid and thus releasing the HPMC. I extracted only one sample in liquid culture to see if the bacteria survived both the tests. Two otherw were kept as control negative (in fact, the main bottle of medium seems to be turbid). We'll now tomorrow.
</P><B>12/09/13</B><P>
&quot;Test des bactéries fraiches&quot;.
1 confection de s gelules
2 enrobage
3 test a l'acidité
4 test  au pbs
After the dissolution in acid medium, the capsule were still complete. Then, they were transfered in the PBS medium where they have been shaking for 1h. A sample (+photo) has been taken at 20, 30, 40, 50 and 60 minutes to follow the progress of dissolution of the capsules. We can easily observe, due to the yellow coloring of the capsule, that they dissolve quickly. The samples at each moment are put in 8ml LB medium + streptomycin. Tomorrow we'll knwo if the bacteria survived the manufacture of the capsule and the dissolving for 3 hours. One blank with only 8ml medium + 8 µL Streptomycin is kept to avoid any fasle interpretation of potential growth.
DEMANDER RECETTE DU MILIEU MIMANT LE SUC GASTRIQUE.
</P><H2>Plasmid 3</H2><P><B>09/09/13</B>
We made miniprep of the culture we had prepared the previous week.</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
(ng/µL)
   </TH><TH align="center">  260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Digestion
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="10" align="center"><P>Plasmid 3
(with EntA,D and F)
    </P></TD><TD align="center">
    1
    </TD><TD align="center">
   127.1
    </TD><TD align="center">
    1.88
    </TD><TD align="center">
    1.94
    </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    92
   </TD><TD align="center">
1.87
   </TD><TD align="center">
1.67
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
114.2
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.78
   </TD></TR><TR><TD align="center">
4
   </TD><TD align="center">
137.1
   </TD><TD align="center">
1.86
   </TD><TD align="center">
1.66
   </TD></TR><TR><TD align="center">
5
   </TD><TD align="center">
139.2
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.88
   </TD></TR><TR><TD align="center">
6
   </TD><TD align="center">
76.9
   </TD><TD align="center">
1.89
   </TD><TD align="center">
1.69
   </TD></TR><TR><TD align="center">
7
   </TD><TD align="center">
65.9
   </TD><TD align="center">
1.95
   </TD><TD align="center">
2.34
   </TD></TR><TR><TD align="center">
8
   </TD><TD align="center">
49.0
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.60
   </TD></TR><TR><TD align="center">
9
   </TD><TD align="center">
35.5
   </TD><TD align="center">
1.87
   </TD><TD align="center">
1.60
   </TD></TR><TR><TD align="center">
10
   </TD><TD align="center">
29.1
   </TD><TD align="center">
1.98
   </TD><TD align="center">
1.32
   </TD></TR><TR><TD rowspan="10" align="center"><P>Plasmid 3
(with EntB,C and E)
   </P></TD><TD align="center">
    1
   </TD><TD align="center">
107.9
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.83
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
45.7
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.95
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
78.6
   </TD><TD align="center">
1.85
   </TD><TD align="center">
1.68
   </TD></TR><TR><TD align="center">
4
   </TD><TD align="center">
95.4
   </TD><TD align="center">
1.85
   </TD><TD align="center">
1.72
   </TD></TR><TR><TD align="center">
5
   </TD><TD align="center">
134.9
   </TD><TD align="center">
1.84
   </TD><TD align="center">
1.82
   </TD></TR><TR><TD align="center">
6
   </TD><TD align="center">
108.9
   </TD><TD align="center">
1.81
   </TD><TD align="center">
1.55
   </TD></TR><TR><TD align="center">
7
   </TD><TD align="center">
75
   </TD><TD align="center">
1.84
   </TD><TD align="center">
1.55
   </TD></TR><TR><TD align="center">
8
   </TD><TD align="center">
133.0
   </TD><TD align="center">
1.82
   </TD><TD align="center">
1.80
   </TD></TR><TR><TD align="center">
9
   </TD><TD align="center">
101.2
   </TD><TD align="center">
1.82
   </TD><TD align="center">
1.63
   </TD></TR><TR><TD align="center">
10
   </TD><TD align="center">
106.4
   </TD><TD align="center">
1.85
   </TD><TD align="center">
1.75
   </TD></TR></TBODY></TABLE><P>
To determin which clones have our constructions, we make a digestion with PstI, EcoRI and BsaBI; and then an electrophoresis with the digestion products.
We compared our gel with the theroical profil made on Geneious.</P><H2>Biobricking</H2><P>
In order to send our constructions to the Registry we have to change their prefix and subclone them into a chloramphenicol backbone pSB1C3.
</P><H3>Modification of the sequence using PCR</H3><P>
PCR is performed using the high fidelity Q5 DNA polymerase.
</P><P>
Mix preparation:
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Component
   </TH><TH align="center">
    50 µl Reaction
   </TH><TH align="center">
    Volume for 13 reactions
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    Q5 buffer (5X)
   </TD><TD align="center">
    10 µL
   </TD><TD align="center">
    130 µL
   </TD></TR><TR><TD align="center">
    dNTPs (10 mM)
   </TD><TD align="center">
    1 µL
   </TD><TD align="center">
    13 µL
   </TD></TR><TR><TD align="center">
    Forward primer (5 µM)
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    Reverse primer (5 µM)
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    DNA template
   </TD><TD align="center">
    1 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    Q5 DNA polymerase
   </TD><TD align="center">
    0.5 µL
   </TD><TD align="center">
    6.5 µL
   </TD></TR><TR><TD align="center">
    H<SUB>2</SUB>O (qsp 50 µL)
   </TD><TD align="center">
    27.5 µL
   </TD><TD align="center">
    357.5 µL
   </TD></TR></TBODY></TABLE><P>
Primers used were different for each construction.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Construction
   </TH><TH align="center">
    Forward Primer
   </TH><TH align="center">
    Reverse Primer
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    AceB promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P87
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    FecA promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P88
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    FepA promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P89
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    Fes promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P90
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    ybiL promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P91
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    AceB promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P92
   </TD><TD align="center">
    P86
   </TD></TR></TBODY></TABLE><P>
After the PCR fragment are verrified by gel migration and each PCR product is purified using PCR clean up kit from Machrey Nagel.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Construction
   </TH><TH align="center">
    Concentration(ng/µL)
   </TH><TH align="center">
    260/280
   </TH><TH>
    260/230
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    AceB promoter + sfGFP
   </TD><TD align="center">
    131.0
   </TD><TD align="center">
    1.79
   </TD><TD align="center">
    0.97
   </TD></TR><TR><TD align="center">
    AceB promoter + LacI-LVA
   </TD><TD align="center">
    74.5
   </TD><TD align="center">
    1.64
   </TD><TD align="center">
    1.05
   </TD></TR><TR><TD align="center">
    FecA promoter + sfGFP
   </TD><TD align="center">
    342.0
   </TD><TD align="center">
    1.56
   </TD><TD align="center">
    0.69
   </TD></TR><TR><TD align="center">
    FepA promoter + sfGFP
   </TD><TD align="center">
    237.8
   </TD><TD align="center">
    1.60
   </TD><TD align="center">
    0.78
   </TD></TR><TR><TD align="center">
    FepA promoter + LacI-LVA
   </TD><TD align="center">
    268.7
   </TD><TD align="center">
    1.57
   </TD><TD align="center">
    0.78
   </TD></TR><TR><TD align="center">
    Fes promoter + sfGFP
   </TD><TD align="center">
    82.5
   </TD><TD align="center">
    1.91
   </TD><TD align="center">
    1.07
   </TD></TR><TR><TD align="center">
    Fes promoter + LacI-LVA
   </TD><TD align="center">
    338.6
   </TD><TD align="center">
    1.54
   </TD><TD align="center">
    0.82
   </TD></TR><TR><TD align="center">
    ybiL promoter + sfGFP
   </TD><TD align="center">
    54.5
   </TD><TD align="center">
    1.93
   </TD><TD align="center">
    1.08
   </TD></TR><TR><TD align="center">
    ybiL promoter + LacI-LVA
   </TD><TD align="center">
    247.9
   </TD><TD align="center">
    1.65
   </TD><TD align="center">
    0.62
   </TD></TR><TR><TD align="center">
    yncE promoter + sfGFP
   </TD><TD align="center">
    124.2
   </TD><TD align="center">
    1.80
   </TD><TD align="center">
    1.23
   </TD></TR><TR><TD align="center">
    yncE promoter + LacI-LVA
   </TD><TD align="center">
    210.3
   </TD><TD align="center">
    1.62
   </TD><TD align="center">
    0.78
   </TD></TR></TBODY></TABLE><H3>pSB1C3 plasmid Minipreping</H3><P>
In order to subclone our construction into the plasmid pSB1C3, we must miniprep it. Then, we launched a 20 mL overnight culture of <I>E. coli</I> transformed with pSB1C3 and we purified it using a Machrey Nagel miniprep purification kit. 
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Plasmid
   </TH><TH>
    Replicate
   </TH><TH align="center">
    Concentration(ng/µL)
   </TH><TH align="center">
    260/280
   </TH><TH>
    260/230
   </TH></TR></THEAD><TBODY><TR><TD rowspan="6" align="center">
    pSB1C3
   </TD><TD align="center">
    n°1
   </TD><TD align="center">
    87.1
   </TD><TD align="center">
    1.86
   </TD><TD align="center">
    1.55
   </TD></TR><TR><TD align="center">
    n°2
   </TD><TD align="center">
    91.8
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.38
   </TD></TR><TR><TD align="center">
    n°3
   </TD><TD align="center">
    77.9
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    1.21
   </TD></TR><TR><TD align="center">
    n°4
   </TD><TD align="center">
    98.9
   </TD><TD align="center">
    1.74
   </TD><TD align="center">
    1.00
   </TD></TR><TR><TD align="center">
    n°5
   </TD><TD align="center">
    78.4
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.33
   </TD></TR><TR><TD align="center">
    n°6
   </TD><TD align="center">
    91.8
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.31
   </TD></TR></TBODY></TABLE><H3>Digestion using EcoRI and PstI</H3><P>
Our construction are subcloned into pSB1C3 using the restriction enzymes EcoRI and PstI (FastDigest from Fermentas)
</P><P>
Mix preparation for 1 µg of DNA (15 µL reaction - 2.5 µL/tube):
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Component
   </TH><TH align="center">
    15 µl Reaction
   </TH><TH align="center">
    Volume for 13 reactions
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    Fast Digest Buffer
   </TD><TD align="center">
    1.5 µL
   </TD><TD align="center">
    18 µL
   </TD></TR><TR><TD align="center">
    EcoRI
   </TD><TD align="center">
    0.5 µL
   </TD><TD align="center">
    6 µL
   </TD></TR><TR><TD align="center">
    PstI
   </TD><TD align="center">
    0.5 µL
   </TD><TD align="center">
    6 µL
   </TD></TR></TBODY></TABLE><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Construction
   </TH><TH align="center">
    15 µl Reaction
   </TH><TH align="center">
    H<SUB>2</SUB>O (qsp 15 µL)
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    AceB promoter + sfGFP
   </TD><TD align="center">
    7.6 µL
   </TD><TD align="center">
    4.9 µL
   </TD></TR><TR><TD align="center">
    AceB promoter + LacI-LVA
   </TD><TD align="center">
    12.5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    FecA promoter + sfGFP
   </TD><TD align="center">
    4 µL
   </TD><TD align="center">
    8.5 µL
   </TD></TR><TR><TD align="center">
    FepA promoter + sfGFP
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    7.5 µL
   </TD></TR><TR><TD align="center">
    FepA promoter + LacI-LVA
   </TD><TD align="center">
    4 µL
   </TD><TD align="center">
    8.5 µL
   </TD></TR><TR><TD align="center">
    Fes promoter + sfGFP
   </TD><TD align="center">
    12.5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    Fes promoter + LacI-LVA
   </TD><TD align="center">
    4 µL
   </TD><TD align="center">
    8.5 µL
   </TD></TR><TR><TD align="center">
    ybiL promoter + sfGFP
   </TD><TD align="center">
    12.5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    ybiL promoter + LacI-LVA
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    7.5 µL
   </TD></TR><TR><TD align="center">
    yncE promoter + sfGFP
   </TD><TD align="center">
    8 µL
   </TD><TD align="center">
    4.5 µL
   </TD></TR><TR><TD align="center">
    yncE promoter + LacI-LVA
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    7.5 µL
   </TD></TR></TBODY></TABLE><P>
We prepared a special mix for the plasmid because 5 µg have been digested.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Component
   </TH><TH align="center">
    Volume needed for the reaction
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    pSB1C3 (85 ng/µL)
   </TD><TD align="center">
    55 µL
   </TD></TR><TR><TD align="center">
    Buffer (10X)
   </TD><TD align="center">
    6.5 µL
   </TD></TR><TR><TD align="center">
    EcoRI
   </TD><TD align="center">
    2.5 µL
   </TD></TR><TR><TD align="center">
    PstI
   </TD><TD align="center">
    2.5 µL
  </TD></TR></TBODY></TABLE><P>
Tube are placed 1h at 37°C. Products are purified using PCR clean up kit from Machrey Nagel.
</P><H3>Ligation</H3><P>
To perform the ligation, we use 100 ng of vector for 200 ng of insert.
Volume of reaction = 15 µL
</P></LI><LI>T4 buffer = 1.5 µL
</LI><LI>PEG 4000 = 1 µL
</LI><LI>T4 ligase = 0.5 µL
</LI><LI>pSB1C3 = 1 µL
</LI></UL><H1 id="week14">Week 14: 16<SUP>th</SUP> September - 22<SUP>nd</SUP> September</H1><H2>Plasmid 1</H2><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
(ng/µL)
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH></TR></THEAD><TBODY><TR><TD rowspan="3" align="center"><P>AceB
with GFP
    </P></TD><TD align="center">
    1
    </TD><TD align="center">
63.0
    </TD><TD align="center">
1.83
    </TD><TD align="center">
1.23
    </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
84.3
   </TD><TD align="center">
1.95
   </TD><TD align="center">
1.50
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
60.8
   </TD><TD align="center">
1.73
   </TD><TD align="center">
1.42
   </TD></TR><TR><TD rowspan="3" align="center"><P>AceB
with LacI
    </P></TD><TD align="center">
1
   </TD><TD align="center">
78.3
   </TD><TD align="center">
1.75
   </TD><TD align="center">
1.05
   </TD></TR><TR><TD align="center">
2
   </TD><TD align="center">
61.6
   </TD><TD align="center">
1.84
   </TD><TD align="center">
1.36
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
61.1
   </TD><TD align="center">
1.83
   </TD><TD align="center">
1.27
   </TD></TR><TR><TD rowspan="4" align="center"><P>Fes
with GFP
    </P></TD></TR><TR><TD align="center">
1
   </TD><TD align="center">
51.8
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.47
   </TD></TR><TR><TD align="center">
2
   </TD><TD align="center">
55.1
   </TD><TD align="center">
1.81
   </TD><TD align="center">
1.07
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
49.2
   </TD><TD align="center">
1.71
   </TD><TD align="center">
0.97
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fes
with LacI
    </P></TD><TD align="center">
1
   </TD><TD align="center">
67.5
   </TD><TD align="center">
1.75
   </TD><TD align="center">
1.16
   </TD></TR><TR><TD align="center">
2
   </TD><TD align="center">
137.7
   </TD><TD align="center">
1.82
   </TD><TD align="center">
1.38
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
147.2
   </TD><TD align="center">
1.89
   </TD><TD align="center">
1.80
   </TD></TR><TR><TD rowspan="3" align="center"><P>Fep A
with GFP
    </P></TD><TD align="center">
    1
    </TD><TD align="center">
74.4
    </TD><TD align="center">
1.60
    </TD><TD align="center">
0.67
    </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
76.8
   </TD><TD align="center">
1.56
   </TD><TD align="center">
0.61
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
97.6
   </TD><TD align="center">
1.59
   </TD><TD align="center">
0.68
   </TD></TR><TR><TD rowspan="3" align="center"><P>FepA
with LacI
    </P></TD><TD align="center">
1
   </TD><TD align="center">
17.0
   </TD><TD align="center">
1.72
   </TD><TD align="center">
1.01
   </TD></TR><TR><TD align="center">
2
   </TD><TD align="center">
76.8
   </TD><TD align="center">
1.63
   </TD><TD align="center">
0.75
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
28.9
   </TD><TD align="center">
1.70
   </TD><TD align="center">
0.70
   </TD></TR><TR><TD rowspan="4" align="center"><P>yncE
with GFP
    </P></TD></TR><TR><TD align="center">
1
   </TD><TD align="center">
86.6
   </TD><TD align="center">
1.63
   </TD><TD align="center">
0.81
   </TD></TR><TR><TD align="center">
2
   </TD><TD align="center">
72.1
   </TD><TD align="center">
1.62
   </TD><TD align="center">
0.77
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
70.8
   </TD><TD align="center">
1.66
   </TD><TD align="center">
0.83
   </TD></TR><TR><TD rowspan="3" align="center"><P>yncE
with LacI
    </P></TD><TD align="center">
1
   </TD><TD align="center">
114.1
   </TD><TD align="center">
1.61
   </TD><TD align="center">
0.73
   </TD></TR><TR><TD align="center">
2
   </TD><TD align="center">
109.6
   </TD><TD align="center">
1.56
   </TD><TD align="center">
0.60
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
100.8
   </TD><TD align="center">
1.56
   </TD><TD align="center">
0.59
   </TD></TR></TBODY></TABLE><H1 id="week15">Week 15: 23<SUP>rd</SUP> September - 29<SUP>th</SUP> September</H1><P>
We transformed our pAceB-LacI construction with an other plasmid containing PL-lacO-RFP. Then the system has been caracterized using TECAN analysis. <A href="https://2013.igem.org/Team:Evry/Results">See our results</A></P><H1 id="week16">Week 16: 30<SUP>th</SUP> September - 6<SUP>th</SUP> October</H1><H2>Microfluidic experiment</H2><P><B>30/09/13</B>
Chip's channel were full of bacteria, even after 2 hours of washing <I>(with free iron M9)</I> because we let the bacteria growth over the week-end <I>(in LB medium)</I>. Hence, we prepared another chip to launch another experiment on the next morning.
As our bacteria have a plasmid AceB-sfGFP and there is no iron in our M9, we still check if bacteria expressed sfGFP protein.
The first results appear promising, but we need to confirm them with a proper experiment.


</P><DIV align="center"><DIV class="captionedPicture" style="width:40%;float:center;"><DIV class="caption"><B>Figure 1:</B><I>Escherichia coli</I> (TOP 10 strain) expressing sfGFP.
                     At the top of the picture, we can see that the channel is full of bacteria.
   </DIV></DIV></DIV><H2 align="center"> 4<SUP>th</SUP> October </H2><H1 id="week17">Week 17: 7<SUP>th</SUP> October - 13<SUP>th</SUP> October</H1><H1 id="week18">Week 18: 14<SUP>th</SUP> October - 20<SUP>th</SUP> October</H1><H1 id="week19">Week 19: 21<SUP>st</SUP> October - 27<SUP>th</SUP> October</H1><H1 id="week20">Week 20: 28<SUP>th</SUP> October - 3<SUP>rd</SUP> November</H1></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/Notebook/Test">http://2013.igem.org/Team:Evry/Notebook/Test</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Notebook/Test" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Notebook/Test" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Notebook/Test&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Notebook/Test&amp;oldid=313459" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>