<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Model3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Model3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Model3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Model3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Model3&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Model3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Model3</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><P><ASIDE><HEADER>
  Model
 </HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/Modeling">Overview</A></LI></UL><UL><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modeltr1">Disease model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modeltr2">Treatment model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin Production:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta1">Sensor model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta2">Inverter model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta3">Enterobactin model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/interactions">Interaction Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/supply">Supply Model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI></UL><HEADER>Tools</HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic Functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI></UL></ASIDE></P><DIV id="maincontainer_evry2013"><DIV id="mainTextcontainer"><H1>Metabolic model</H1><H2>Overview</H2><P>
This model aims to simulate the entire pathway of the enterobactin production in our Iron Coli. It includes the synthetic sensing system, and the chemical reactions leading to the enterobactin.
This model has two goals:
<UL><LI>To the delay needed to produce siderophores starting from the iron-sensing</LI><LI>To determine the ideal plasmid configuration</LI></UL>
The enterobactin production regulation is based on two consecutives inhibitions, which, in the end, is an activator with a certain delay. The model will follow this principle.
</P><H2>Assumptions</H2><UL><LI>The only limiting reagant in the pathway is the chorismic acid, which we consider being in excess amount</LI><LI>The FUR is also considered in excess amount</LI><LI>All the enzymes (and the mRNA involved) needed are assimilated to a single variable, since they all play equivalent roles</LI></UL><H2>Model Description </H2><P>
Variables:<UL><LI>[Fe] : Iron concentration inside the bacteria</LI><LI>[Fur] : FUR concentration inside the bacteria</LI><LI>[FeFur] : Iron-FUR complex concentration inside the bacteria</LI><LI>LacI : Number of inhibited LacI</LI><LI>LacO : Number of non-inhibited LacO</LI><LI>[mRNA]: mRNA (from LacO) concentration</LI><LI>[Enz] : Enzyme concentration :  EntA,-B,-C,-D,-E,-F</LI></UL>
All those concentrations are expressed in mmol/L
</P><P><U>Fe, FUR and FeFUR:</U>
The iron-FUR complex is simply formed that way: 
We reduced this equation to:
Which is not annoying, since we just have to divide our [FeFur] by to to get the real complex concentration.
We can easily write down both the formation (v) and the dissociation (v') speed:
We chose to model the iron input in the bacteria using a linear function of the external iron concentration <I>Ferext</I>, the factor <I>p</I> being the cell-wall permeability for iron.
The FUR on the other hand is produced by the bacteria. Its evolution can also be considered as linear, using a mean production rate <I>Fur0</I>.
In this model, we only track the free Fe-FUR complex and not those attached to a FUR Binding Site. As <I>LacI</I> is the number of inhibited LacI, we can use this number to express how much Fe-FUR does bind to a FBS per unit of time.</P><P><U>LacI-LacO system:</U>
To simulate the inhibition phenomenon, we chose to use our <A href="https://2013.igem.org/Team:Evry/LogisticFunctions">logistic function</A> under its differential form. Since it is the Fe-FUR that represses it, the LacI can be expressed as a logistic fuction of the Fe-FUR:
Ki1 is a non-dimensional parameter that repesents the inhibition power, and Kf is the fixation rate of the Fe-FUR on the FBS. Finally, Nbrpla1 is the number of pasmids containing the LacI.
In the same way, LacO is modeled with a logistic funtion. LacO is repressed by LacI:
Ki2 is the inhibition power and Nbrpla2 is the number of plasmimds containing LacO.
LacI and LacO are both ruled by a normal logistic function. If we were to track the number of expressed LacI/LacO, we would be using two inverted logistic fuctions to model a double inverter. The thing is, since <I>LacI</I> represents the number of <B>repressed</B> genes and <I>LacO</I> the number of <B>expressed</B> genes, the double inverter is still there, but the calculations are easier.</P><P><U>mRNA and Enzymes:</U>
The [mRNA] and [Enz] equations are alike. The prodction rates are Kr for the mRNA and Kp for the enzymmes, and both variables have a negative degadation term:</P><P><U>Finally:</U></P><H2>Parameters</H2><TABLE width="100%" border="1"><TBODY><TR width="100%"><TH>Name</TH><TH>Value</TH><TH>Unite</TH><TH>Description</TH></TR><TR width="100%"><TD>p</TD><TD>0.1</TD><TD>min^-1</TD><TD>Permeability of cell wall</TD></TR><TR width="100%"><TD>KfeFUR</TD><TD>0.01</TD><TD>M^-1.s^-1</TD><TD>Formation constant of FeFur complex</TD></TR><TR width="100%"><TD>Dff</TD><TD>0.001</TD><TD>min^-1</TD><TD>-</TD></TR><TR width="100%"><TD>Kp</TD><TD>0.5</TD><TD>min^-1</TD><TD>translation rate</TD></TR><TR width="100%"><TD>KT</TD><TD>2.0</TD><TD>M.min^-1</TD><TD>transcription rate</TD></TR><TR width="100%"><TD>milliNa</TD><TD>6.02.10^20</TD><TD>mol^-1</TD><TD>Avogadro's constant</TD></TR><TR width="100%"><TD>V</TD><TD>6.5.10^-16</TD><TD>m^3</TD><TD>Volume of a call</TD></TR><TR width="100%"><TD>Dmrna</TD><TD>0.001</TD><TD>min^-1</TD><TD>mRNA degradation rate</TD></TR><TR width="100%"><TD>Denz</TD><TD>0.001</TD><TD>min^-1</TD><TD>Enzyme degradation rate</TD></TR><TR width="100%"><TD>Kf</TD><TD>10^-4</TD><TD>min^-1</TD><TD>fixation rate of FeFUR</TD></TR><TR width="100%"><TD>Fur0</TD><TD>0.01</TD><TD>mM.min^⁻1</TD><TD>Fur Production</TD></TR><TR width="100%"><TD>vE1</TD><TD>0.01</TD><TD>mM.min^⁻1</TD><TD>Chorismate production</TD></TR></TBODY></TABLE><U>Enzymatic Parameters:</U><TABLE width="100%" border="1"><TBODY><TR width="100%"><TD>KcatA</TD><TD>5550</TD><TD>min^-1</TD><TD>PMID:2144454</TD></TR><TR width="100%"><TD>KcatB</TD><TD>600</TD><TD>min^-1</TD><TD>PMID:2139796</TD></TR><TR width="100%"><TD>KcatC</TD><TD>173</TD><TD>min^-1</TD></TR><TR width="100%"><TD>KmA</TD><TD>300</TD><TD>μM</TD><TD>-</TD></TR><TR width="100%"><TD>KmB</TD><TD>14.7</TD><TD>μM</TD><TD>PMID:2139796</TD></TR><TR width="100%"><TD>KmC</TD><TD>14</TD><TD>μM</TD><TD>-</TD></TR></TBODY></TABLE><P>
Note that we don't have any informations about the KI1 and KI2 parameters, since they are purely artificial. We thus had to run simulations to study the effects of different sets of values on the model.
</P><H2>Results</H2><P><U>High copy?</U>
Here are 9 gaphs pointing out the relevance of different plasmid configurations. The goal of this simulation is to try answering a biological question: &quot;should we increase the number of plLacO plasmids or the LacI plasmids?&quot;The horizontal axes are the time (minutes) and the iron concentration (logarithmic scale). The vertical axis is the Enzyme production (mmol/L).</P>
The graph is clear: the most important plasmid is the one containing the LacI (the direct iron sensor).
<P><U>About the KI1/KI2:</U>
Here are 9 other graphs with a set of different values for KI1 and KI2.
</P><P><U>Temporal answers</U>


This curve integrates the chemical pathway developped <A href="">here</A>.
</P>
There is a chemical delay in the enterobactin production, which is about the same length as the bacterial delay.
<P id="model_galenic"><U>Impact on the galenic formulation</U>
The delay of siderophore production is predicted to be way to long if you choose for the flush strategy. As a consequence, these result directly impacted our chelating strategy. Thus, our final decision is to try to implant the bacteria in the intestins (here in the jejunum) to statically enhance its growth and let it enough time to recover for a dessicated environment.
</P><A href="https://static.igem.org/mediawiki/2013/b/be/Metabolicmodel.zip">Download the model</A></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/Model3">http://2013.igem.org/Team:Evry/Model3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Model3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Model3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Model3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Model3&amp;oldid=324730" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>