<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Notebook_w13"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Notebook/w13">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Notebook/w13&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/w13&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Notebook/w13&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Notebook/w13" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Notebook/w13</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><DIV class="onglet-left"><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Protocols#Protocol's_overview">Protocols: BioBricks' Conception</A></LI><DIV class="hide"><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Protocol's_overview" title="Protocol's overview">Protocols' overview</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Media" title="Solutions and Media">Solutions and Media</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#DNA_genomic_extraction" title="DNA genomic extraction">DNA genomic extraction</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Polymerase_Chain_Reaction" title="PCR">PCR</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Golden_Gate" title="Golden Gate">Golden Gate</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Competent_cells" title="Competent cells">Competent cells</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Transformation" title="Transformation">Transformation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Plasmid_purification" title="Plasmid purification ">Plasmid purification </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Electrophoresis" title="Ectrophoresis">Ectrophoresis</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#PCR_Purification" title="PCR Purification ">PCR Purification </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#BioBrick_Assembly" title="BioBrick Assembly ">BioBrick Assembly </A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Protocols#Characterization_overview">Protocols: BioBricks' Characterization</A></LI><DIV class="hide"><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Characterization_overview" title="Protocols' overview">Protocols' overview</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#TECAN_Analysis" title="TECAN Analysis">TECAN Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Microfluidic_experiment " title="Microfluidic experiment ">Microfluidic experiment </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Homologous_recombination " title="Homologous recombination ">Homologous recombination </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Siderophore_detection " title="Siderophore detection ">Siderophore detection </A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI></UL></DIV><DIV class="onglet"><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week1">JUNE</A></LI><DIV class="hide"><LI id="w1"><A href="https://2013.igem.org/Team:Evry/Notebook#week1" title="17th June - 23rd June">Week 1</A></LI><LI id="w2"><A href="https://2013.igem.org/Team:Evry/Notebook#week2" title="24th June - 30th June">Week 2</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week3">JULY</A></LI><DIV class="hide"><LI id="w3"><A href="https://2013.igem.org/Team:Evry/Notebook#week3" title="1st July - 7th July">Week 3</A></LI><LI id="w4"><A href="https://2013.igem.org/Team:Evry/Notebook#week4" title="8th July - 14th July">Week 4</A></LI><LI id="w5"><A href="https://2013.igem.org/Team:Evry/Notebook#week5" title="15th July - 21st July">Week 5</A></LI><LI id="w6"><A href="https://2013.igem.org/Team:Evry/Notebook#week6" title="22nd July - 28th July">Week 6</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook#/week7">AUGUST</A></LI><DIV class="hide"><LI id="w7"><A href="https://2013.igem.org/Team:Evry/Notebook#week7" title="29th July - 4th August">Week 7</A></LI><LI id="w8"><A href="https://2013.igem.org/Team:Evry/Notebook#week8" title="5th August - 11th August">Week 8</A></LI><LI id="w9"><A href="https://2013.igem.org/Team:Evry/Notebook#week9" title="12th August - 18th August">Week 9</A></LI><LI id="w10"><A href="https://2013.igem.org/Team:Evry/Notebook#week10" title="19th August - 25th August">Week 10</A></LI><LI id="w11"><A href="https://2013.igem.org/Team:Evry/Notebook#week11" title="26th August - 1st September">Week 11</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week12">SEPTEMBER</A></LI><DIV class="hide"><LI id="w12"><A href="https://2013.igem.org/Team:Evry/Notebook#week12" title="2nd September - 8th September">Week 12</A></LI><LI id="w13"><A href="https://2013.igem.org/Team:Evry/Notebook#week13" title="9th September - 15th September">Week 13</A></LI><LI id="w14"><A href="https://2013.igem.org/Team:Evry/Notebook#week14" title="16th September - 22nd September">Week 14</A></LI><LI id="w15"><A href="https://2013.igem.org/Team:Evry/Notebook#week15" title="23rd September - 29th September">Week 15</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week16">OCTOBER</A></LI><DIV class="hide"><LI id="w16"><A href="https://2013.igem.org/Team:Evry/Notebook#week16" title="30th September - 6th October">Week 16</A></LI><LI id="w17"><A href="https://2013.igem.org/Team:Evry/Notebook#week17" title="7th October - 13th October">Week 17</A></LI><LI id="w18"><A href="https://2013.igem.org/Team:Evry/Notebook#week18" title="14th October - 20th October">Week 18</A></LI><LI id="w19"><A href="https://2013.igem.org/Team:Evry/Notebook#week19" title="21st October - 27th October">Week 19</A></LI><LI id="w20"><A href="https://2013.igem.org/Team:Evry/Notebook#week20" title="28th October - 3rd November">Week 20</A></LI></DIV></UL></DIV><DIV id="maincontainer_evry2013notebook"><DIV id="mainTextcontainer"><H1>Week 13: 9th September - 15th September</H1><H2>Design of the capsule</H2><B>09/09/13</B><P>
To chelate the iron in the duodenum and the initial portion of the jejunum, it was pretty obvious that a capsule or a tablet was required. The best galenic formulation for our purposes is the capsule, meaning a delivery of our bacteria in the jejunum. The first reason is that a tablet requires not only a heavy and dry compression, but also the second reason is that it requires lyophilized bacteria, which consists of extreme conditions (from very high to very low temperature and high pressure). As a consequence, the capsule offers the possibility to be easily coated for resistance against stomachal acidity and presents a softer environment for the bacteria to transport.
</P><P>
The first step in the design of the capsule is to choose the right diluent, meaning the appropriate powder to 'dissolve' the bacteria in. We experimented both <I>maltodextrin</I> and <I>colloidal silica</I> and tried to disperse as much LB medium as possible with hand mortar and pestle (figure 1 and 2). For 10g of <I>maltodextrin</I>, we managed to dissolve 400 µL of LB medium, and for 3g of <I>colloidal silica</I> 4 mL of LB medium. We choose <I>colloidal silica</I> for 

- Réflexion le choix des bio-adhésifs: ici on utilisera les HPMC.
- Réflexion sur la qualité des bio-adhésif, notamment dans la relation viscosité/bio-adhésif. En effet, les HPMC vont gonfler en contact avec l'eau, adhérant ainsi les bactéries à la paroi. Il faut que la viscosité soit minimale pour pas que le bol alimentaire vienne arracher la fixation de nos bactéries, mais qu'en même temps l'eau ait une pénétrance suffisante.
- Réflexion sur l'enrobage (enteric coating). Il faut que la gélule se délite le plus haut possible dans l'intestin (ici le duodénum), mais qu'il soit suffisamment résistant à l'acidité de l'estomac en amont.
</P><P>
Choice of excipients (reference: Handbook of Pharmaceutical Excipients, sixth edition):
Hypromellose (Hydroxypropyl methylcellulose, HPMC): is used as a bioadhesive material for a controlled release at pH 5-8 (duodenum and jejunum).
Methacrylic acid L100-55 (Eudragit): is used as an enteric coating for resistance against stomachal pH, thus allowing the release of the bacteria in the duodenum at pH values of 5,5.
Colloidal silicon dioxide: is used to dry pellet of bacteria and allow an homogenous mixture with the other components.</P><P>
TOP10 and TOP10 transformed with pSB1A3 stayed the whole weekend at 4°C. I resuspended the cell in 10 mL LB + antibiotic.
</P><B>10/09/13</B><P>
I tested two different powders to dilute our medium with bacteria. As a start, i began to dilute LB medium, without bacteria in Maltodextrin (Glucidex) or Colloidal silica (Aerosil). After dispersing the liquid in the powder, the goal was to put as much as possible in it, but the powder should not agglomerate and should keep its powder form. As such, I managed to dilute 200 µL LB in 10 g of Maltodextrin but 3825 µL in 3 g of colloidal silica. As a consequence, we chose for colloidal silica to dilute the LB, and thus, the bacteria.
</P><P>
The second step is to obtain a final volume of 35g of powder. After I diluted 3825 µL of LB in colloidal silica, I obtained a total volume of 16 mL. I added a sufficient quantity of HPMC (Hydroxyprpylmethylcellulose, Methocel K100) to obtain a final volume of 35 mL. The two components are mixed together and I made sure the product is dry.
</P><P>
With a 'gelulier', I equally dispersed the whole volume in 50 capsules. After weight measurement, the capsules are 342 +/- 4 mg.
</P><P>
To anticipate the dispersion tests at pH = 2 and at pH = 6,5, I prepared a PBS buffer and an acid medium that mimics the gastric context and its acidty.
The capsule should not be dispersed after two hours in the medium of pH = 2. Also, the capsule, afterwards, should disperse in between 1 hour in PBS buffer. If the capsule satisfies these conditions, the bacteria will be releases right at the end of the duodenum, thus allowing the growth in the jejunum.
</P><P>
PBS preparation for a volume of 5L:
</P><DIV style="padding-left:10%;"><P>NaCl: 40g</P></DIV><DIV style="padding-left:10%;"><P>KCl: 1g</P></DIV><DIV style="padding-left:10%;"><P>Na2HPO4: 7,2g</P></DIV><DIV style="padding-left:10%;"><P>KH2PO4: 1,2g</P></DIV><P>
The first attempt to make capsules from LB is repeated, but this time with LB and bacteria. After I diluted 3000 µL of LB and bactera in colloidal silica, I obtained a total volume of 20mL. I added a sufficient quantity of HPMC (Hydroxyprpylmethylcellulose, Methocel K100) to obtain a final volume of 35 mL. The two components were mixed together. However, since the bacteria are dry in the powder, they may be very volatile when mixed to the colloidal silica. As an additional precaution, I wore a mask to prevent the inhale of these potenially contaminated particles.
</P><P>
I refreshed my cultures in 3ml LB (x2) + an additional control negative (only 3ml LB) to make sure that I am not in fact working with the wrong bacteria or any contamination. This control negative was actually very important. In this lab, I don't have the same sterilized pipet tips or 15ml tube. Also, the only sterile environnement is in the presence of a very short flame.
</P><P>
At the end of the day, I managed to create 50 capsules containing only dry LB and 50 capsules with dry LB and bacteria. Additionnaly, I prepared the alcoholic solution of 12,5% Eudragit L100-55 to create the gastro-enteric resistant coating around my capsule. The receipe is as follows:
50ml of pure ethanol (98%)
6,25g of eudragit
Contrary to the litterature, the Eudragit was not that 'extremely' soluble in pure ethanol and did not take 20mn to dissolve. As a consequence, I sonicated for 45 minutes to solubilize the Eudragit.
</P><P>
Finally, I tried to make the coating on some capsules. I came to the conclusion that the drying lasts a very long time. To accelerate the drying process, I used the hairdryer for 30 seconds at maximum temperature at average speed, then 1mn at lowest temperature at highest speed. This way i managed to dry the fastest way the coating of my capsules, without being to sticky at the end. Also, every single capsule had to be done twice, once on the upper side and another on the downside. The not only allows a better dring, but also double-covers the edge of the capsule where both sides close (in the middle) and gives a double protection at the weakest point of the capsule. Finally, keep in mind that the coating process is doubled, thus spending on average 8 to 10 minutes for each capsule.
</P><B>11/09/13</B><P>
I double-coated 6 capsules containing the dry bacteria and put them for 2 hours in the acid to test their resistance to low pH (pH = 2). They did not break during this stage.
After 20 minutes of shacking in the PBS buffer, the water starts to become turbid, thus proving the degradation of the gelatin of the capsid and thus releasing the HPMC. I extracted only one sample in liquid culture to see if the bacteria survived both the tests. Two otherw were kept as control negative (in fact, the main bottle of medium seems to be turbid). We'll now tomorrow.
</P><B>12/09/13</B><P>
&quot;Test des bactéries fraiches&quot;.
1 confection de s gelules
2 enrobage
3 test a l'acidité
4 test  au pbs
After the dissolution in acid medium, the capsule were still complete. Then, they were transfered in the PBS medium where they have been shaking for 1h. A sample (+photo) has been taken at 20, 30, 40, 50 and 60 minutes to follow the progress of dissolution of the capsules. We can easily observe, due to the yellow coloring of the capsule, that they dissolve quickly. The samples at each moment are put in 8ml LB medium + streptomycin. Tomorrow we'll knwo if the bacteria survived the manufacture of the capsule and the dissolving for 3 hours. One blank with only 8ml medium + 8 µL Streptomycin is kept to avoid any fasle interpretation of potential growth.
DEMANDER RECETTE DU MILIEU MIMANT LE SUC GASTRIQUE.
</P><H2>Plasmid 3</H2><P><B>09/09/13</B>
We made miniprep of the culture we had prepared the previous week.</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Name
   </TH><TH align="center">
    Clone
   </TH><TH align="center">
    Concentration
(ng/µL)
   </TH><TH align="center">
    260/280
   </TH><TH align="center">
    260/230
   </TH><TH align="center">
    Digestion
   </TH><TH align="center">
    Sequencing
   </TH></TR></THEAD><TBODY><TR><TD rowspan="10" align="center"><P>Plasmid 3
(with EntA,D and F)
    </P></TD><TD align="center">
    1
    </TD><TD align="center">
   127.1
    </TD><TD align="center">
    1.88
    </TD><TD align="center">
    1.94
    </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
    92
   </TD><TD align="center">
1.87
   </TD><TD align="center">
1.67
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
114.2
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.78
   </TD></TR><TR><TD align="center">
4
   </TD><TD align="center">
137.1
   </TD><TD align="center">
1.86
   </TD><TD align="center">
1.66
   </TD></TR><TR><TD align="center">
5
   </TD><TD align="center">
139.2
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.88
   </TD></TR><TR><TD align="center">
6
   </TD><TD align="center">
76.9
   </TD><TD align="center">
1.89
   </TD><TD align="center">
1.69
   </TD></TR><TR><TD align="center">
7
   </TD><TD align="center">
65.9
   </TD><TD align="center">
1.95
   </TD><TD align="center">
2.34
   </TD></TR><TR><TD align="center">
8
   </TD><TD align="center">
49.0
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.60
   </TD></TR><TR><TD align="center">
9
   </TD><TD align="center">
35.5
   </TD><TD align="center">
1.87
   </TD><TD align="center">
1.60
   </TD></TR><TR><TD align="center">
10
   </TD><TD align="center">
29.1
   </TD><TD align="center">
1.98
   </TD><TD align="center">
1.32
   </TD></TR><TR><TD rowspan="10" align="center"><P>Plasmid 3
(with EntB,C and E)
   </P></TD><TD align="center">
    1
   </TD><TD align="center">
107.9
   </TD><TD align="center">
    1.87
   </TD><TD align="center">
    1.83
   </TD></TR><TR><TD align="center">
    2
   </TD><TD align="center">
45.7
   </TD><TD align="center">
1.88
   </TD><TD align="center">
1.95
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
3
   </TD><TD align="center">
78.6
   </TD><TD align="center">
1.85
   </TD><TD align="center">
1.68
   </TD></TR><TR><TD align="center">
4
   </TD><TD align="center">
95.4
   </TD><TD align="center">
1.85
   </TD><TD align="center">
1.72
   </TD></TR><TR><TD align="center">
5
   </TD><TD align="center">
134.9
   </TD><TD align="center">
1.84
   </TD><TD align="center">
1.82
   </TD></TR><TR><TD align="center">
6
   </TD><TD align="center">
108.9
   </TD><TD align="center">
1.81
   </TD><TD align="center">
1.55
   </TD></TR><TR><TD align="center">
7
   </TD><TD align="center">
75
   </TD><TD align="center">
1.84
   </TD><TD align="center">
1.55
   </TD></TR><TR><TD align="center">
8
   </TD><TD align="center">
133.0
   </TD><TD align="center">
1.82
   </TD><TD align="center">
1.80
   </TD></TR><TR><TD align="center">
9
   </TD><TD align="center">
101.2
   </TD><TD align="center">
1.82
   </TD><TD align="center">
1.63
   </TD></TR><TR><TD align="center">
10
   </TD><TD align="center">
106.4
   </TD><TD align="center">
1.85
   </TD><TD align="center">
1.75
   </TD></TR></TBODY></TABLE><P>
To determin which clones have our constructions, we make a digestion with PstI, EcoRI and BsaBI; and then an electrophoresis with the digestion products.
We compared our gel with the theroical profil made on Geneious.</P><H2>Biobricking</H2><P>
In order to send our constructions to the Registry we have to change their prefix and subclone them into a chloramphenicol backbone pSB1C3.
</P><H3>Modification of the sequence using PCR</H3><P>
PCR is performed using the high fidelity Q5 DNA polymerase.
</P><P>
Mix preparation:
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Component
   </TH><TH align="center">
    50 µl Reaction
   </TH><TH align="center">
    Volume for 13 reactions
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    Q5 buffer (5X)
   </TD><TD align="center">
    10 µL
   </TD><TD align="center">
    130 µL
   </TD></TR><TR><TD align="center">
    dNTPs (10 mM)
   </TD><TD align="center">
    1 µL
   </TD><TD align="center">
    13 µL
   </TD></TR><TR><TD align="center">
    Forward primer (5 µM)
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    Reverse primer (5 µM)
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    DNA template
   </TD><TD align="center">
    1 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    Q5 DNA polymerase
   </TD><TD align="center">
    0.5 µL
   </TD><TD align="center">
    6.5 µL
   </TD></TR><TR><TD align="center">
    H<SUB>2</SUB>O (qsp 50 µL)
   </TD><TD align="center">
    27.5 µL
   </TD><TD align="center">
    357.5 µL
   </TD></TR></TBODY></TABLE><P>
Primers used were different for each construction.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Construction
   </TH><TH align="center">
    Forward Primer
   </TH><TH align="center">
    Reverse Primer
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    AceB promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P87
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    FecA promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P88
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    FepA promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P89
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    Fes promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P90
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    ybiL promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P91
   </TD><TD align="center">
    P86
   </TD></TR><TR><TD align="center">
    AceB promoter + (sfGFP/LacI-LVA)
   </TD><TD align="center">
    P92
   </TD><TD align="center">
    P86
   </TD></TR></TBODY></TABLE><P>
After the PCR fragment are verrified by gel migration and each PCR product is purified using PCR clean up kit from Machrey Nagel.
</P><H1>AJOUTER PHOTO LEGENDEE DU GEL</H1><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Construction
   </TH><TH align="center">
    Concentration(ng/µL)
   </TH><TH align="center">
    260/280
   </TH><TH>
    260/230
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    AceB promoter + sfGFP
   </TD><TD align="center">
    131.0
   </TD><TD align="center">
    1.79
   </TD><TD align="center">
    0.97
   </TD></TR><TR><TD align="center">
    AceB promoter + LacI-LVA
   </TD><TD align="center">
    74.5
   </TD><TD align="center">
    1.64
   </TD><TD align="center">
    1.05
   </TD></TR><TR><TD align="center">
    FecA promoter + sfGFP
   </TD><TD align="center">
    342.0
   </TD><TD align="center">
    1.56
   </TD><TD align="center">
    0.69
   </TD></TR><TR><TD align="center">
    FepA promoter + sfGFP
   </TD><TD align="center">
    237.8
   </TD><TD align="center">
    1.60
   </TD><TD align="center">
    0.78
   </TD></TR><TR><TD align="center">
    FepA promoter + LacI-LVA
   </TD><TD align="center">
    268.7
   </TD><TD align="center">
    1.57
   </TD><TD align="center">
    0.78
   </TD></TR><TR><TD align="center">
    Fes promoter + sfGFP
   </TD><TD align="center">
    82.5
   </TD><TD align="center">
    1.91
   </TD><TD align="center">
    1.07
   </TD></TR><TR><TD align="center">
    Fes promoter + LacI-LVA
   </TD><TD align="center">338.6
   </TD><TD align="center">
    1.54
   </TD><TD align="center">
    0.82
   </TD></TR><TR><TD align="center">
    ybiL promoter + sfGFP
   </TD><TD align="center">
    54.5
   </TD><TD align="center">
    1.93
   </TD><TD align="center">
    1.08
   </TD></TR><TR><TD align="center">
    ybiL promoter + LacI-LVA
   </TD><TD align="center">
    247.9
   </TD><TD align="center">
    1.65
   </TD><TD align="center">
    0.62
   </TD></TR><TR><TD align="center">
    yncE promoter + sfGFP
   </TD><TD align="center">
    124.2
   </TD><TD align="center">
    1.80
   </TD><TD align="center">
    1.23
   </TD></TR><TR><TD align="center">
    yncE promoter + LacI-LVA
   </TD><TD align="center">
    210.3
   </TD><TD align="center">
    1.62
   </TD><TD align="center">
    0.78
   </TD></TR></TBODY></TABLE><H3>pSB1C3 plasmid Minipreping</H3><P>
In order to subclone our construction into the plasmid pSB1C3, we must miniprep it. Then, we launched a 20 mL overnight culture of <I>E. coli</I> transformed with pSB1C3 and we purified it using a Machrey Nagel miniprep purification kit. 
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Plasmid
   </TH><TH>
    Replicate
   </TH><TH align="center">
    Concentration(ng/µL)
   </TH><TH align="center">
    260/280
   </TH><TH>
    260/230
   </TH></TR></THEAD><TBODY><TR><TD rowspan="6" align="center">
    pSB1C3
   </TD><TD align="center">
    n°1
   </TD><TD align="center">
    87.1
   </TD><TD align="center">
    1.86
   </TD><TD align="center">
    1.55
   </TD></TR><TR><TD align="center">
    n°2
   </TD><TD align="center">
    91.8
   </TD><TD align="center">
    1.84
   </TD><TD align="center">
    1.38
   </TD></TR><TR><TD align="center">
    n°3
   </TD><TD align="center">
    77.9
   </TD><TD align="center">
    1.81
   </TD><TD align="center">
    1.21
   </TD></TR><TR><TD align="center">
    n°4
   </TD><TD align="center">
    98.9
   </TD><TD align="center">
    1.74
   </TD><TD align="center">
    1.00
   </TD></TR><TR><TD align="center">
    n°5
   </TD><TD align="center">
    78.4
   </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.33
   </TD></TR><TR><TD align="center">
    n°6
   </TD><TD align="center">
    91.8
  </TD><TD align="center">
    1.82
   </TD><TD align="center">
    1.31
   </TD></TR></TBODY></TABLE><H3>Digestion using EcoRI and PstI</H3><P>
Our construction are subcloned into pSB1C3 using the restriction enzymes EcoRI and PstI (FastDigest from Fermentas)
</P><P>
Mix preparation for 1 µg of DNA (15 µL reaction - 2.5 µL/tube):
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Component
   </TH><TH align="center">
    15 µl Reaction
   </TH><TH align="center">
    Volume for 13 reactions
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    Fast Digest Buffer
   </TD><TD align="center">
    1.5 µL
   </TD><TD align="center">
    18 µL
   </TD></TR><TR><TD align="center">
    EcoRI
   </TD><TD align="center">
    0.5 µL
   </TD><TD align="center">
    6 µL
   </TD></TR><TR><TD align="center">
    PstI
   </TD><TD align="center">
    0.5 µL
   </TD><TD align="center">
    6 µL
   </TD></TR></TBODY></TABLE><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Construction
   </TH><TH align="center">
    15 µl Reaction
   </TH><TH align="center">
    H<SUB>2</SUB>O (qsp 15 µL)
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    AceB promoter + sfGFP
   </TD><TD align="center">
    7.6 µL
   </TD><TD align="center">
    4.9 µL
   </TD></TR><TR><TD align="center">
    AceB promoter + LacI-LVA
   </TD><TD align="center">
    12.5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    FecA promoter + sfGFP
   </TD><TD align="center">
    4 µL
   </TD><TD align="center">
    8.5 µL
   </TD></TR><TR><TD align="center">
    FepA promoter + sfGFP
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    7.5 µL
   </TD></TR><TR><TD align="center">
    FepA promoter + LacI-LVA
   </TD><TD align="center">
    4 µL
   </TD><TD align="center">
    8.5 µL
   </TD></TR><TR><TD align="center">
    Fes promoter + sfGFP
   </TD><TD align="center">
    12.5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    Fes promoter + LacI-LVA
   </TD><TD align="center">
    4 µL
   </TD><TD align="center">
    8.5 µL
   </TD></TR><TR><TD align="center">
    ybiL promoter + sfGFP
   </TD><TD align="center"> 12.5 µL
   </TD><TD align="center">
    -
   </TD></TR><TR><TD align="center">
    ybiL promoter + LacI-LVA
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    7.5 µL
   </TD></TR><TR><TD align="center">
    yncE promoter + sfGFP
   </TD><TD align="center">
    8 µL
   </TD><TD align="center">
    4.5 µL
   </TD></TR><TR><TD align="center">
    yncE promoter + LacI-LVA
   </TD><TD align="center">
    5 µL
   </TD><TD align="center">
    7.5 µL
   </TD></TR></TBODY></TABLE><P>
We prepared a special mix for the plasmid because 5 µg have been digested.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Component
   </TH><TH align="center">
    Volume needed for the reaction
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    pSB1C3 (85 ng/µL)
   </TD><TD align="center">
    55 µL
   </TD></TR><TR><TD align="center">
    Buffer (10X)
   </TD><TD align="center">
    6.5 µL
   </TD></TR><TR><TD align="center">
    EcoRI
   </TD><TD align="center">
    2.5 µL
   </TD></TR><TR><TD align="center">
    PstI
   </TD><TD align="center">
    2.5 µL
   </TD></TR></TBODY></TABLE><P>
Tube are placed 1h at 37°C. Products are purified using PCR clean up kit from Machrey Nagel.
</P><H1>Ajouter les résultats de dosage des échantillons</H1><H3>Ligation</H3><P>
To perform the ligation, we use 100 ng of vector for 200 ng of insert.
Volume of reaction = 15 µL
</P><UL><LI>T4 buffer = 1.5 µL
</LI><LI>PEG 4000 = 1 µL
</LI><LI>T4 ligase = 0.5 µL
</LI><LI>pSB1C3 = 1 µL
</LI></UL></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/Notebook/w13">http://2013.igem.org/Team:Evry/Notebook/w13</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Notebook/w13" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Notebook/w13" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Notebook/w13&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Notebook/w13&amp;oldid=258798" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>