"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Notebook/w7
From 2013.igem.org
Iron coli project
Home
Project
Iron metabolism
Related diseases
Iron chelators
Achievements
References
Team
iGEM Profile
Attributions
Collaborations
Acknowledgement
Press
Photo gallery
Model
Flush treatment
Enterobactin production
Genome scale
Population scale
Logistic functions
Chemical reasoning
Plate Reader
Biology
Parts
Sensor
Inverter
Chelator
Results
Human Practice
Consultation
Survey
Seminar
Philosophy
Safety
Economy
Art
Capsule
Notebook
Protocols
Primers
Sponsors
Protocols: BioBricks' Conception
Protocols' overview
Solutions and Media
DNA genomic extraction
PCR
Golden Gate
Competent cells
Transformation
Plasmid purification
Ectrophoresis
PCR Purification
BioBrick Assembly
Protocols: BioBricks' Characterization
Protocols' overview
TECAN Analysis
Microfluidic experiment
Homologous recombination
Siderophore detection
Primers
JUNE
Week 1
Week 2
JULY
Week 3
Week 4
Week 5
Week 6
AUGUST
Week 7
Week 8
Week 9
Week 10
Week 11
SEPTEMBER
Week 12
Week 13
Week 14
Week 15
OCTOBER
Week 16
Week 17
Week 18
Week 19
Week 20
Week 7: 29th July - 4th August
Construction of plasmid N°1
We make an electrophoresis with 5 µL of plamsid to check the plasmid purification made on the last friday.
Mettre l'image légendée ici
There is not the 3 bandes that we sould see, so to check another time, we make a digestion with Pst I and EcoR I:
NEB Buffer 10X  5 µL
BSA 10X  5 µL
DNA  2 µg
PstI  2 µL
EcoRI  2 µL
Water  50 µL
After another electrophoresis, we obtain no band.
Sequencing preparation
Name
Clone
Concentration
260/280
260/230
Sequencing
AceB Fur Binding Site
(with sfGFP)
1
166.1 ng/µL
1.86
1.76
Good
2
192.0 ng/µL
1.62
1.02
Good
3
123.5 ng/µL
1.87
1.75
Good
4
128.0 ng/µL
1.82
1.66
Good
FepA Fur Binding Site
(with sfGFP)
1
21.1 ng/µL
2.06
-
Good
2
22.4 ng/µL
1.96
-
Good
3
55.4 ng/µL
1.87
1.58
Good
4
68.5 ng/µL
1.89
1.67
Good
Fes Fur Binding Site
(with sfGFP)
1
39.9 ng/µL
1.87
1.80
Good
2
40.6 ng/µL
1.74
-
No sequencing
3
68.4 ng/µL
1.87
1.63
Good
4
20.8 ng/µL
2.05
-
No sequencing
ybiL Fur Binding Site
(with sfGFP)
1
58.2 ng/µL
1.87
1.73
Good
2
16.3 ng/µL
1.95
-
No sequencing
3
18.8 ng/µL
1.95
-
No sequencing
4
45.5 ng/µL
1.90
1.81
Good
FecA Fur Binding Site
(with sfGFP)
1
20.0 ng/µL
1.96
-
Good
2
27.5 ng/µL
1.77
-
No sequencing
3
31.5 ng/µL
1.94
1.81
Good
4
57.0 ng/µL
1.89
1.64
Good
yncE Fur Binding Site
(with sfGFP)
1
92.4 ng/µL
1.90
1.75
Good
2
64.7 ng/µL
1.89
1.66
Good
3
38.6 ng/µL
1.91
1.80
Good
Construction of plasmid N°2
Our plasmid N°2 is building with a synthetic promote sequence which is composed of:
Andersen's promotor
Fur Binding Site (15 different)
RBS
sfGFP
Terminator
Golden Gate
In order to associate these sequences we performed a golden gate, using for each sample :
80 ng of Andersen's promotor
80 ng of Fur Binding Site
80 ng of RBS
80 ng of sfGFP
80 ng of Terminator
1.5 µL of T4 buffer (10X)
15 Unit of T4 ligase
2.5 Unit of Bsa I
The Golden Gate products are chemically transformed into E. coli Top 10 strains. After the transformation process, bacteria are plated into LB medium with carbenicillin and icubated overnigth at 37°C.
Golden Gates plates are composed at 98% of red colonies. The problem was probably on the equimolarity ratio between our different part which were not totally respected. We isolated white colonies (4 maximum by each sample) on LB medium with carbenicillin and we incubated them overnight at 37°c.
We made pre culture (V = 10 mL), using bacterial colonies isolated from our plates of Golden Gate transformation. Then minipreps and glycerol conservation were made using these pre cultures.
Fur Binding Site
Sequence
Clone
Concentration
Fur BS 1
ATTATTGATAACTATTTG
Clone n°1
80.8 ng/µL
Clone n°2
83.8 ng/µL
Clone n°4
90.3 ng/µL
Fur BS 2
GATAACTATTTGCATTTGCA
Clone n°1
68.6 ng/µL
Clone n°2
77.4 ng/µL
Clone n°3
68.0 ng/µL
Fur BS 3
CAAATGCAAATAGTTATCCA
Clone n°1
79.4 ng/µL
Clone n°2
110.5 ng/µL
Clone n°3
457.4 ng/µL
Fur BS 4
CAAATAGTTATCAATAATCA
Clone n°1
73.2 ng/µL
Clone n°2
66.2 ng/µL
Fur BS 5
GTAAATTAATATTATTTACA
Clone n°1
57.2 ng/µL
Fur BS 6
AATAATGCTTCTCATTTTCA
Clone n°1
63.8 ng/µL
Fur BS 7
ATAAATGATAATCATTATCA
Clone n°1
70.6 ng/µL
Fur BS 8
GAAAATAATTCTTATTTCCA
Clone n°1
75.6 ng/µL
Clone n°2
87.2 ng/µL
Clone n°3
48.8 ng/µL
Clone n°4
57.8 ng/µL
Fur BS 9
AGCACTTATTATTATTTTCA
Clone n°1
58.6 ng/µL
Fur BS 10
GATAATTGTTATCGTTTGCA
No Clone
-
Fur BS 11
GATAATGCTTATCAAAATCA
Clone n°2
75.6 ng/µL
Clone n°3
398.0 ng/µL
Clone n°4
57.4 ng/µL
Fur BS 12
CAAAATTATTATCACTTTCA
Clone n°1
50.0 ng/µL
Fur BS 13
AATAATGATTACCATTCCCA
Clone n°2
70.7 ng/µL
Fur BS 14
GAAATTGTTTTTGATTTTCA
Clone n°1
42.7 ng/µL
Clone n°2
32.8 ng/µL
Clone n°3
37.8 ng/µL
Clone n°4
45.7 ng/µL
Fur BS 15
GTTAATTGTAATGATTTTCA
Clone n°1
37.0 ng/µL
Clone n°2
53.2 ng/µL
Clone n°3
196.0 ng/µL
Construction of plasmid N°3
29/07/13 We received the primers ordered on friday the 26th of July and started to dilute them into TrisCL at 10 mM. Secondly, we diluted the previous stock solution at a rate of 1/20th to obtain a final concentration of 5 µM for our intermediate solution. Then, for our plasmid three construction, we need to first extract the 6 enterobactin gene (EntA, EntB, EntC, EntD, EntE and EntF). The ordered primers from friday will theoretically extract them in the appropriate Golden Gate format with their own RBS upstream (designed from Salis RBS). This will allow us to construct our two N°3 plasmids, one containing EntA, EntD and EntF, and the other one EntB, EntC and EntE. The genes are spread like this to obtain two equivalent plasmids xxx We proceeded to a genomic extraction of the 6 genes of interest and migrated to PCR products on a 1% gel. We successfully extracted 5 out of the 6. 30/07/13 We annealled the oligonucleotides of the PL-LacO part in the golden gate format. Also, we extracted the sfGFP with the RBS upstream which will allow us to construct the control positive plasmid 3 for future TECAN experiments. 31/07/13 We started by purifying our 6 succesful PCR extraction from the 29/07 and 30/07. So we managed to extract EntA, EntB, EntC, EntD and EntF from E. coli's genomic DNA and sfGFP in the golden gate format with a RBS (plasmid 3 construction) from plasmidic DNA. Additionnaly, we optimized our PCR to extract our missing gene, EntE. We obtained a smear +/- a double band. As a consequence, we adjusted the annealing temperature with a range from 54°C to 66°C. This will allow us to try extracting the missing gene (EntE) in the appropriate conditions. 01/08/13
We made a PCR for the EntE gene with a gradient of temperature to know at which temperature the PCR result were the best.
We made 8 (ou mettre 6 ?) tubes with the same composition:
Water
Taq Buffer 5
dNTPs
Rev EntE 2,5
For EntE 2,5
DNA 0,5
Taq polymerase 0,5
We also made positive and negative controles (with Rev EntC and For EntC instead of EntE).
After electrophoresis, we obtain the best band at .. °C.
However our experiment have been made with Taq polymerase (and Taq Buffer) instead of Q5 (and Q5 buffer) that we use usually. Then, we have to repeat this experiment with Q5 and Q5 buffer.
The did a gradient PCR and migrated on a gel our samples. On the gel, we notice that both the smear and the double band disappear when the annealing temperature increases. So to conclude, we run tonight a PCR with Q5 polymerase with increasing temperature starting from 56 degrees. The absolute annealing temperature will be determined for the Q5 (not extrapolable from a OneTaq PCR). Also, we made a golden gate for the construction of plasmid 3. For analysis purposes, only a GFP was added downstream of the PL LacO promoter. Also, the EntE gene is still to be extracted from the genome. Also, we runned 3 extras golden gate for the construction of plasmid 2. In fact, we corrected our calculations because we initially we used mass concentrations instead of molar concentrations.
02/08/13
The transformations of the golden gate 2 and 3 are successful and we managed to invert the rate of red colonies versus white colonies (red = false positive with mRFP, white = positive), thus meaning we have a lot of white ones! We conserved the plates for future isolation on coming monday. Additionnaly, we migrated the PCR products of 01/08. The gel is empty, but the positive controls are there, meaning we made a mistake in the primers (the master mix is the one used for the positive control).
Contact: igemevry@gmail.com
Site developped by Gabriel Guillocheau
Retrieved from " http://2013.igem.org/Team:Evry/Notebook/w7 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
