"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Model3
From 2013.igem.org
Iron coli project
Home
Project
Iron metabolism
Related diseases
Iron chelators
Achievements
References
Team
iGEM Profile
Attributions
Collaborations
Acknowledgement
Press
Photo gallery
Model
Flush treatment
Enterobactin production
Genome scale
Population scale
Logistic functions
Chemical reasoning
Plate Reader
Biology
Parts
Sensor
Inverter
Chelator
Results
Human Practice
Consultation
Survey
Seminar
Philosophy
Safety
Economy
Art
Capsule
Notebook
Protocols
Primers
Sponsors
Model
Overview
Flush treatment:
Disease model
Treatment model
Enterobactin Production:
Sensor model
Inverter model
Enterobactin model
Genome scale:
Interaction Model
Supply Model
Population scale
Tools
Logistic Functions
Chemical reasoning
Plate Reader
Metabolic model
Overview
This model aims to simulate the entire pathway of the enterobactin production in our Iron Coli. It includes the synthetic sensing system, and the chemical reactions leading to the enterobactin.
This model has two goals:
To the delay needed to produce siderophores starting from the iron-sensing
To determine the ideal plasmid configuration
The enterobactin production regulation is based on two consecutives inhibitions, which, in the end, is an activator with a certain delay. The model will follow this principle.
Assumptions
The only limiting reagant in the pathway is the chorismic acid, which we consider being in excess amount
The FUR is also considered in excess amount
All the enzymes (and the mRNA involved) needed are assimilated to a single variable, since they all play equivalent roles
Model Description
Variables:
[Fe] : Iron concentration inside the bacteria
[Fur] : FUR concentration inside the bacteria
[FeFur] : Iron-FUR complex concentration inside the bacteria
LacI : Number of inhibited LacI
LacO : Number of non-inhibited LacO
[mRNA]: mRNA (from LacO) concentration
[Enz] : Enzyme concentration :  EntA,-B,-C,-D,-E,-F
All those concentrations are expressed in mmol/L
Fe, FUR and FeFUR:
The iron-FUR complex is simply formed that way:
 
We reduced this equation to:
Which is not annoying, since we just have to divide our [FeFur] by to to get the real complex concentration.
We can easily write down both the formation (v) and the dissociation (v') speed:
We chose to model the iron input in the bacteria using a linear function of the external iron concentration Ferext, the factor p being the cell-wall permeability for iron.
The FUR on the other hand is produced by the bacteria. Its evolution can also be considered as linear, using a mean production rate Fur0.
In this model, we only track the free Fe-FUR complex and not those attached to a FUR Binding Site. As LacI is the number of inhibited LacI, we can use this number to express how much Fe-FUR does bind to a FBS per unit of time.
LacI-LacO system:
To simulate the inhibition phenomenon, we chose to use our logistic function under its differential form. Since it is the Fe-FUR that represses it, the LacI can be expressed as a logistic fuction of the Fe-FUR:
Ki1 is a non-dimensional parameter that repesents the inhibition power, and Kf is the fixation rate of the Fe-FUR on the FBS. Finally, Nbrpla1 is the number of pasmids containing the LacI.
In the same way, LacO is modeled with a logistic funtion. LacO is repressed by LacI:
Ki2 is the inhibition power and Nbrpla2 is the number of plasmimds containing LacO.
LacI and LacO are both ruled by a normal logistic function. If we were to track the number of expressed LacI/LacO, we would be using two inverted logistic fuctions to model a double inverter. The thing is, since LacI represents the number of repressed genes and LacO the number of expressed genes, the double inverter is still there, but the calculations are easier.
mRNA and Enzymes:
The [mRNA] and [Enz] equations are alike. The prodction rates are Kr for the mRNA and Kp for the enzymmes, and both variables have a negative degadation term:
Finally:
Parameters
Name
Value
Unite
Description
p
0.1
min^-1
Permeability of cell wall
KfeFUR
0.01
M^-1.s^-1
Formation constant of FeFur complex
Dff
0.001
min^-1
-
Kp
0.5
min^-1
translation rate
KT
2.0
M.min^-1
transcription rate
milliNa
6.02.10^20
mol^-1
Avogadro's constant
V
6.5.10^-16
m^3
Volume of a call
Dmrna
0.001
min^-1
mRNA degradation rate
Denz
0.001
min^-1
Enzyme degradation rate
Kf
10^-4
min^-1
fixation rate of FeFUR
Fur0
0.01
mM.min^⁻1
Fur Production
vE1
0.01
mM.min^⁻1
Chorismate production
Enzymatic Parameters:
KcatA
5550
min^-1
PMID:2144454
KcatB
600
min^-1
PMID:2139796
KcatC
173
min^-1
KmA
300
μM
-
KmB
14.7
μM
PMID:2139796
KmC
14
μM
-
Note that we don't have any informations about the KI1 and KI2 parameters, since they are purely artificial. We thus had to run simulations to study the effects of different sets of values on the model.
Results
High copy?
Here are 9 gaphs pointing out the relevance of different plasmid configurations. The goal of this simulation is to try answering a biological question: "should we increase the number of plLacO plasmids or the LacI plasmids?"
The horizontal axes are the time (minutes) and the iron concentration (logarithmic scale). The vertical axis is the Enzyme production (mmol/L).
The graph is clear: the most important plasmid is the one containing the LacI (the direct iron sensor).
About the KI1/KI2:
Here are 9 other graphs with a set of different values for KI1 and KI2.
Temporal answers
This curve integrates the chemical pathway developped here .
There is a chemical delay in the enterobactin production, which is about the same length as the bacterial delay.
Impact on the galenic formulation
The delay of siderophore production is predicted to be way to long if you choose for the flush strategy. As a consequence, these result directly impacted our chelating strategy. Thus, our final decision is to try to implant the bacteria in the intestins (here in the jejunum) to statically enhance its growth and let it enough time to recover for a dessicated environment.
Download the model
Contact: igemevry@gmail.com
Site developped by Gabriel Guillocheau
Retrieved from " http://2013.igem.org/Team:Evry/Model3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
