<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_interactions"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/interactions">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/interactions&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/interactions&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/interactions&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/interactions" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/interactions</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><P><ASIDE><HEADER>
  Model
 </HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/Modeling">Overview</A></LI></UL><UL><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modeltr1">Disease model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modeltr2">Treatment model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin Production:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta1">Sensor model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta2">Inverter model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta3">Enterobactin model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/interactions">Interaction Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/supply">Supply Model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI></UL><HEADER>Tools</HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic Functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI></UL></ASIDE></P><DIV id="maincontainer_evry2013"><DIV id="mainTextcontainer"><H1>Studying the metabolic perturbations</H1><H2>Question</H2>

Are there any interractions between our synthetic construction <I>E. coli</I> central metabolism ?

<H2>Observation</H2><P>
If the ENTx genes present on our constructions are expressed, it will increase the quantity of the corresponding enzymes in the bacteria. More enzymes mean more consumption pressure on all the metabolites of the ENTEROBACTIN production pathway. As seen on <A href="#Fig1">Figure 1</A>, two metabolites could be subject to this effect : CHORISMATE and ISOCHORISMATE.
</P><CENTER><DIV class="captionedPicture" style="float:left;"><DIV class="caption"><B>Figure 1:</B><P>The ENTEROBACTIN production pathway (cyan) and the other metabolic reactions consuming the same metabolites (red). circles : enzymes name.</P></DIV></DIV></CENTER><H3>The ISOCHORISMATE case</H3><P>
The ISOCHORISMATE metabolite is only consumed by one other metabolic reaction catalyzed by the MenD enzyme. The <A href="#Fig2">Figure 2</A> presents the metabolic map conaining the reaction catalyzed by MEND.
</P><CENTER><DIV class="captionedPicture" style="float:left;"><DIV class="caption"><B>Figure 2:</B><P>The metabolic map containing the pathway menD belongs to.</P></DIV></DIV></CENTER><P>
As one can see, the menD enzyme is used upstream of the VITAMIN K cycle. It is interesting because vitamins are always produced in small quantities as they are toxic for the cell. With this observation we can deduce that menD should be present in small quantities inside the cell and thus should not use too much of the produced ISOCHORISMATE.
</P><H3>The CHORISMATE case</H3><P>
There are different pathways consuming this metabolite, some could be neglected in the same manner as the menD pathway presented above. But others are part of the central metabolism of <I>E. coli</I>. The enzymes which associated reaction consume chorismate are the following:

<UL><LI>TrpE : which belongs to <A href="http://www.genome.jp/kegg-bin/show_pathway?ec00400+4.1.3.27">tryptophan biosynthesis pathway</A>, and thus to <I>E. coli</I> central metabolism. This reaction should have a non-null flux when doing FBA simulations.</LI><LI>UbiC : which belongs to the <A href="http://www.genome.jp/kegg-bin/show_pathway?ec00130+4.1.3.40">ubiquinone biosynthesis pathway</A>, a coenzyme used in the aerobic cellular respiration. This reaction may have a null flux in the FBA simulation as it does not directly participate to the metabolic function of the cell.</LI><LI>PabA/B : which belongs to the <A href="http://www.ecocyc.org/ECOLI/NEW-IMAGE?type=PATHWAY&amp;object=PWY-6543">4-AMINOBENZOATE biosynthesis pathway</A>, which is used in the production of the folic acid, which is a vitamin. This reaction may have a null flux in the FBA simulation also.
  </LI><LI>PheA/B : which belongs to the tyrosine and <A href="http://www.ecocyc.org/ECOLI/NEW-IMAGE?type=ENZYME&amp;object=CHORISMUTPREPHENDEHYDRAT-MONOMER">phenylalanine biosynthesis pathways</A> which belongs to <I>E. coli</I> central metabolism. This flux should not be null in the FBA simulation.
</LI></UL></P><H2>Setup</H2><P><EM>Model</EM> : <I>E. coli</I> iJR904 with addition of ENTSYNTH and ENTOUT reactions.

In order to test whether the system interacts with the metabolism of the bacteria we applied the FBA with a modification to the objective function:
</P><P>
By varying i between 0 and 1 we can vary the contributions of enterobacting production and biomass growth as shown in the picture below.
</P><P>
The above figure shows the role of the i variable in the objective function. The higher i (between 0 and 1), the more CHORISMATE goes into the ENTEROBACTIN production pathway and the less in the <I>E. coli</I> central metabolism (TrpE and PheA/B).</P><H3>Biological Motivationfor the objective function</H3><P>
By adding the synthetic construct inside the cell, more Ent enzymes will be present thus consuming more CHORISMATE(and all other precursors). The value i can be seen as the ratio :
</P><P>
This proportion models the following biological dynamics: the more EntC enzymes are present in the cell, the more CHORISMATE will go through its corresponding reaction. Hence the i will tend towards 0 and the flux through the EntC reaction will tend towards its maximal value.
</P><P>
Note that we can neglect the MenF concentration as it should not change during the experiment.
</P><H2>Simulation</H2><P>
The first simulation consist in observing the different distributions of fluxes as a function of i. The <A href="#Fig3">Figure 3</A> presents the results of this simulation.
</P><CENTER><DIV class="captionedPicture" style="float:left;"><DIV class="caption"><B>Figure 3:</B><P>Fluxes as function of the i coefficient of the modified objective function. Dotted lines : Important fluxes; Solid lines : Reactions belonging to the ENTEROBACTIN biosynthesis pathway; Dashed lines : Other reactions consuming important metabolites.</P></DIV></DIV></CENTER><P><UL><LI>Setup : i varied from 0 to 1 with a step of 0.01.</LI><LI>The code for this model can be found at the bottom of the page</LI><LI>The EntB and EntC curves are superposed</LI></UL></P><H2>Interpretation</H2>

The UbiC and MenD curves are always zero, this may be due as explained earlier because they do not belong to central metabolic pathways.

<P>
The graph is divided in two parts :
</P><H3>i &lt; 0.303</H3><P>
In this part, no ENTOUT flux is present, hence there is no production of ENTEROBACTIN. In the other hand there are two non-zero constant fluxes, the PheA (at 0.28 mmol/gDW/h), TrpE and PabA/B (both at 0.04 mmol/gDW/h). The objective function is linearly decreasing with i meaning that the system does not change in this part.
</P><H3>i = 0.303</H3><P>
For this value of i there is a brutal change in the different fluxes :

<UL><LI>PheA flux goes to 0 mmol/gDW/h</LI><LI>TrpE and PabA/B flux goes to 0 mmol/gDW/h</LI><LI>EntC, EntB and ENTOUT fluxes goes to their maximal values 2.1 mmol/gDW/h</LI></UL>

The objective function f stops decreasing.
</P><H3>i &gt; 0.303</H3><P>
After this value of i the system stays the same as proved by the linear increase of the objective function with i. This effect is only due to the change in i because no other fluxes are modified and is thus an artifact of the method.

</P><H2>Conclusion</H2><P>

The production of enterobactin modifies the flux distribution for only three reactions out of 5 (the others have null fluxes) : PabA, TrpE and PheA.Thus <EM>there is an interaction between the enterobactin production system and the <I>E. coli</I> metabolism</EM> that happens at the level of these reactions. The analysis of these reactions is done in the next model on <A href="https://2013.igem.org/Team:Evry/supply">this page</A>.
</P><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/interactions">http://2013.igem.org/Team:Evry/interactions</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/interactions" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/interactions" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/interactions&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/interactions&amp;oldid=354968" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>