<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Protocols/">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Protocols/&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Protocols/&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Protocols/&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Protocols/" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Protocols/</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><DIV class="onglet-left"><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Protocols#Protocol's_overview">Protocols: BioBricks' Conception</A></LI><DIV class="hide"><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Protocol's_overview" title="Protocol's overview">Protocols' overview</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Media" title="Solutions and Media">Solutions and Media</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#DNA_genomic_extraction" title="DNA genomic extraction">DNA genomic extraction</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Polymerase_Chain_Reaction" title="PCR">PCR</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Golden_Gate" title="Golden Gate">Golden Gate</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Competent_cells" title="Competent cells">Competent cells</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Transformation" title="Transformation">Transformation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Plasmid_purification" title="Plasmid purification ">Plasmid purification </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Electrophoresis" title="Ectrophoresis">Ectrophoresis</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#PCR_Purification" title="PCR Purification ">PCR Purification </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#BioBrick_Assembly" title="BioBrick Assembly ">BioBrick Assembly </A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Protocols#Characterization_overview">Protocols: BioBricks' Characterization</A></LI><DIV class="hide"><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Characterization_overview" title="Protocols' overview">Protocols' overview</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#TECAN_Analysis" title="TECAN Analysis">TECAN Analysis</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Microfluidic_experiment " title="Microfluidic experiment ">Microfluidic experiment </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Homologous_recombination " title="Homologous recombination ">Homologous recombination </A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols#Siderophore_detection " title="Siderophore detection ">Siderophore detection </A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI></UL></DIV><DIV class="onglet"><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week1">JUNE</A></LI><DIV class="hide"><LI id="w1"><A href="https://2013.igem.org/Team:Evry/Notebook#week1" title="17th June - 23rd June">Week 1</A></LI><LI id="w2"><A href="https://2013.igem.org/Team:Evry/Notebook#week2" title="24th June - 30th June">Week 2</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week3">JULY</A></LI><DIV class="hide"><LI id="w3"><A href="https://2013.igem.org/Team:Evry/Notebook#week3" title="1st July - 7th July">Week 3</A></LI><LI id="w4"><A href="https://2013.igem.org/Team:Evry/Notebook#week4" title="8th July - 14th July">Week 4</A></LI><LI id="w5"><A href="https://2013.igem.org/Team:Evry/Notebook#week5" title="15th July - 21st July">Week 5</A></LI><LI id="w6"><A href="https://2013.igem.org/Team:Evry/Notebook#week6" title="22nd July - 28th July">Week 6</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook#/week7">AUGUST</A></LI><DIV class="hide"><LI id="w7"><A href="https://2013.igem.org/Team:Evry/Notebook#week7" title="29th July - 4th August">Week 7</A></LI><LI id="w8"><A href="https://2013.igem.org/Team:Evry/Notebook#week8" title="5th August - 11th August">Week 8</A></LI><LI id="w9"><A href="https://2013.igem.org/Team:Evry/Notebook#week9" title="12th August - 18th August">Week 9</A></LI><LI id="w10"><A href="https://2013.igem.org/Team:Evry/Notebook#week10" title="19th August - 25th August">Week 10</A></LI><LI id="w11"><A href="https://2013.igem.org/Team:Evry/Notebook#week11" title="26th August - 1st September">Week 11</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week12">SEPTEMBER</A></LI><DIV class="hide"><LI id="w12"><A href="https://2013.igem.org/Team:Evry/Notebook#week12" title="2nd September - 8th September">Week 12</A></LI><LI id="w13"><A href="https://2013.igem.org/Team:Evry/Notebook#week13" title="9th September - 15th September">Week 13</A></LI><LI id="w14"><A href="https://2013.igem.org/Team:Evry/Notebook#week14" title="16th September - 22nd September">Week 14</A></LI><LI id="w15"><A href="https://2013.igem.org/Team:Evry/Notebook#week15" title="23rd September - 29th September">Week 15</A></LI></DIV></UL><UL><LI class="vis"><A href="https://2013.igem.org/Team:Evry/Notebook/#week16">OCTOBER</A></LI><DIV class="hide"><LI id="w16"><A href="https://2013.igem.org/Team:Evry/Notebook#week16" title="30th September - 6th October">Week 16</A></LI><LI id="w17"><A href="https://2013.igem.org/Team:Evry/Notebook#week17" title="7th October - 13th October">Week 17</A></LI><LI id="w18"><A href="https://2013.igem.org/Team:Evry/Notebook#week18" title="14th October - 20th October">Week 18</A></LI><LI id="w19"><A href="https://2013.igem.org/Team:Evry/Notebook#week19" title="21st October - 27th October">Week 19</A></LI><LI id="w20"><A href="https://2013.igem.org/Team:Evry/Notebook#week20" title="28th October - 3rd November">Week 20</A></LI></DIV></UL></DIV><DIV id="maincontainer_evry2013notebook"><DIV id="mainTextcontainer_notebook"><H1 id="Protocol's_overview">BioBricks conception overview </H1><H1> Solutions, Media and Petri Dishes </H1><H2> Antibiotics </H2><P><B>Carbenicillin:</B> Put 150 mg of Carbenicilline in 15 mL of distilled water (10 mg per mL).
Filter with a 0,22 µm millipore,divide in 2 mL tubes and store at -4 °C.
Use 1 µL of Carbenicillin for 1 mL of solution.</P><P><B>Chloramphenicol:</B> Put 525 mg of Kanamycin in 15 mL of ethanol (35 mg per mL).
Filter with a 0,22 µm millipore,divide in 2 mL tubes and store at -4 °C.
Use 25 µL of Chloramphenicol for 1 mL of solution.</P><P><B>Kanamycin:</B> Put 750 mg of Kanamycin in 15 mL of distilled water (50 mg per mL).
Filter with a 0,22 µm millipore,divide in 2 mL tubes and store at -4 °C.
Use 1 µL of Kanamycin for 1 mL of solution.</P><P><B>Spectinomycin:</B> Put  900 mg of Spectinomycin in 15 mL of  ( 60 mg per mL).
Filter with a 0,22 µm millipore,divide in 2 mL tubes and store at -4°C.
Use 1 µL of Spectinomycin for 1 mL of solution.</P><I>Carbenicillin inhibits cell wall synthesis, kanamycin and spectinomycin interfer with mRNA traduction and chloramphenicol inhibits ribosomial peptidyl transferase activity.
Thoses antibiotics are used to select bacteria with a plasmid including an interest construction and the corresponding antibiotic resistance gene.</I><H2> LB and LB Agar </H2><P>Mix 20 g of LB Broth  in 1L of distilled water in a bottle of 1L, then put in autoclave. 
Mix 17,5 g of LB Agar in 500 mL of distilled water in a bottle of 1L, then put in autoclave.
</P><H2> M9 Medium </H2><P><U>Composition for 50 mL of:</U></P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Reagent
   </TH><TH align="center">
    M9 medium (without iron)
   </TH><TH align="center">
    M9 medium (with iron)
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    M9 salt (5X)
   </TD><TD colspan="2" align="center">
    10 mL
   </TD></TR><TR><TD align="center">
    CaCl<SMALL><SUB>2</SUB></SMALL> (1M)
   </TD><TD colspan="2" align="center">
    5 µL
   </TD></TR><TR><TD align="center">
    MgSO<SMALL><SUB>4</SUB></SMALL> (1M)
   </TD><TD colspan="2" align="center">
    100 µL
   </TD></TR><TR><TD align="center">
    Glycerol (50%)
   </TD><TD colspan="2" align="center">
    800 µL
   </TD></TR><TR><TD align="center">
    Thiamine
   </TD><TD colspan="2" align="center">
    5 µL
   </TD></TR><TR><TD align="center">
    NaOH (pH 7.4)
   </TD><TD colspan="2" align="center">
    12.5 µL
   </TD></TR><TR><TD align="center">
    H<SMALL><SUB>2</SUB></SMALL>O
   </TD><TD align="center">
    40 mL
   </TD><TD align="center">
    39 mL
   </TD></TR><TR><TD align="center">
    FeSO<SMALL><SUB>4</SUB></SMALL> (10mM)
   </TD><TD align="center">
    -
   </TD><TD align="center">
    50 µL
   </TD></TR><TR><TD align="center">
    Casamino acids (0.2%)
   </TD><TD align="center">
    -
   </TD><TD align="center">
    1 mL
   </TD></TR></TBODY></TABLE><P>
Once the mixture is prepared, the medium must be filtered to be sterilised using 0.22 µm filter.
</P><H2> Petri dish </H2><P>Melt the LB Agar in a microwave, and split it in 50 mL tubes.
When the tubes have cooled, put the necessary amount of antibiotics in it (50 µL for Carbenicillin and Kanamycin, 1250 µL for Chloramphenicol) and mix thoroughly.
Put 25 mL of the solution in a Petri dish.</P><H2> CaCl<SMALL><SUB>2</SUB></SMALL> and CaCl<SMALL><SUB>2</SUB></SMALL>+glycerol </H2><P><B>Solution for 1M Cacl<SMALL><SUB>2</SUB></SMALL>:</B>
Add 14,30g of CaCl<SMALL><SUB>2</SUB></SMALL> into 100 ml desalted water</P><B><P>Solution for 0,1M Cacl<SMALL><SUB>2</SUB></SMALL>:</P></B>
Add 50 mL of CaCl<SMALL><SUB>2</SUB></SMALL> 1M solution into 450 ml of desalted water<P><B>Solution for 0,1M Cacl<SMALL><SUB>2</SUB></SMALL> + 15% glycerol:</B>
Add 50 mL of CaCl<SMALL><SUB>2</SUB></SMALL> 1M solution and 75 mL of glycerol 100% into 450 ml of desalted water</P><H2> Tris-Acetate-EDTA (TAE) Buffer solutions </H2><P><B>NaOH solution 5M 100 mL:</B>
Add 19,99 g of NaOH in 100 mL of distilled water.</P><P><B>EthyleneDiamineTetraacetic Acid (EDTA) solution 0,5M 300 mL:</B>
Add 43,8 g of EDTA in 250 mL of distilled water.
Add NaOH 5M until EDTA is solubilized and pH~8.
Add distilled water until you reach 300 mL.</P><P><B>TAE Stock solution 50X 300 mL:</B>
Add 60,5 g of TAE in 187,5 mL of distilled water. 
Add 14,27 mL of acetate solution (previously put at 4°C). 
Add 25 mL of EDTA solution previously prepared.
Add distilled water until you reach 250 mL. </P><P><B>TAE solution 1X 100 mL:</B>
Add 5 mL of TAE Stock solution 50X in 100 mL of distilled water. </P><H2> Tris HCl </H2><P>Disolve 4,6g of Trisbase in 30 mL of distilled water.
Adjust pH with concentrated HCl(~4 mL) until pH=7,5.
Add distilled water until 50 mL then put in autoclave.
If the solution has a yellow coloration, do the preparation again with better Trisbase.<I>Tris HCl is used to store DNA.</I></P><H1>DNA genomic extraction</H1><H2> Goal </H2><P>The aim of the extraction step is to recover <I>Escherichia coli</I> genomic DNA.
The different step are use to throw other bacterial components off (proteins, cell wall, plasmids,etc).
Once we have <I>E.coli</I> genomic DNA</P> and specific primers, we will obtain sequences we need, doing a <A href="https://2013.igem.org/Team:Evry/Protocols/Test#Polymerase_Chain_Reaction" target="_blank">PCR</A>.

<H2> Preparation </H2><I>Protocol adapted from Thermo Scientific Genomic extraction notebook</I><B><P>1. Cell culture</P></B>
Cultivate cells in LB medium overnight.<P><B>2. Cell harvesting</B>
Set saturated E.coli LM culture into 2 mL tubes. Centrifuge at 8 000 x g for 5 minutes. Discard as much as supernatant as possible.</P><P><B>3. Cell lysis</B>
Add 180 μL of Digestion Solution and 20 μL of Proteinase K Solution. Resuspend the cells thoroughly with a vortex or a pipette.

Incubate the tubes at 56°C while vortexing occassionally until the cells are completely lysed(~30 minutes).

Add 20 μL of RNase A solution, mix by vortexing and incubate the tubes for 10 minutes at room temperature.
Add 200 μL of Lysis Solution to the sample. Mix thoroughly by vortexing until a homogeneous mixture is obtained. (~15 secondes)
Add 400 μL of 50% ethanol and mix with a vortex or a pipette.</P><P><B>4. DNA Binding</B>
Transfer the lysate to a GeneJET Genomic DNA Purification Column inserted in a collection tube. Centrifuge the column at 12 000 x g for 1 minute. Discard the collection tube containing flow-through solution and place the column into a new 2 mL tube.</P><P><B>5. Membrane washing</B>
Add 500 μL of Wash Buffer I (previously added with ethanol). Centrifuge at 1 600 x g. Discard the flow-through and place the purification column back into the collection tube.

Add 500 μL of Wash Buffer II (previously added with ethanol)to the purification column. 
</P><P><B>6. Dry membrane</B>
Centrifuge at 16 000 x g for 3 minutes.

</P><P><B>7. DNA Elution</B>
Add 200 μL of Elution Buffer to the center of the purification column membrane to elute genomic DNA. Incubate for 2 minutes at room temperature and centrifuge at 8 000 x g for 1 minute.
Discard the purification column, test the concentration with a Nanodrop.
Store the purified DNA in TrisHCl at -20°C or use it immediatly.</P><H1> Polymerase Chain Reaction  </H1><H2>Principle</H2><DIV class="captionedPicture" style="width:30%;float:right;"><DIV class="caption"><B>Figure 1:</B> PCR principle
   </DIV></DIV><P>Polymerase Chain Reaction (PCR) is a molecular biology method used to amplify a small amount of genetic material (DNA or RNA), using specific primers of a target sequence. 
PCR is divided into 5 steps:
<OL><LI>First denaturation:
</LI><LI>Denaturation step
</LI><LI>Annealing step
</LI><LI>Elongation step
</LI><LI>Final step
</LI></OL>
The 2nd, 3th and 4th steps are repeated 20-40 cycles.</P><P><B>Denaturation step</B> occure between 94 and 98°C.The heat breaks the hydrogen bonds and then causes the separation of the double strand of DNA into two single strands. <B>Annealing step</B> occure between 50 and 65 °C. Primers anneals to DNA simple strand by complementarity of the nitrogenized bases. <B>Elongation step</B> occure between 70 and 80°C, depending on the DNA polymerase used. The polymerase synthesizes a new DNA strand complementary to the DNA template</P><H2> Preparation</H2><H3>Mixture preparation</H3><P>We use two differents polymerase for our PCR: Taq Polymerase and Q<SMALL><SUB>5</SUB></SMALL>.
As Q<SMALL><SUB>5</SUB></SMALL> is a hight fidelity polymerase, we use it to obtain sequences without mutation (for a total of 50 μL of mixture); otherwise we use Taq Polymerase instead (for a total of 25 μL of mixture).
Afterwards, we will describe the protocol for a PCR with Q<SMALL><SUB>5</SUB></SMALL> (total amount: 50 μL).<I><B><SMALL>Note: Proportion are the same with Taq Polymerase, for 25 μL or 50 μL: 1:10 of each primer, 1:5 of Buffer, 1:50 of dNTPs, 1:50 of genomic DNA and 1:100 of polymerase</SMALL></B></I>
Make tubes with 5 μL (1:10) of Forward primer and 5 μL (1:10) of Reverse primer for each desidered sequence.
For positive and negative controle, use primers that worked previously in the same condition of PCR.<I>We use primers (Primers 009 and 010) for pEntC for ou controles.</I>
For more details about our primers, see the <A href="https://2013.igem.org/Team:Evry/Primers" target="_blank">corresponding page</A>.

For n tubes, prepare a master mix with the following solutions:
<UL><LI> (n+1) x 10 μL of Q<SMALL><SUB>5</SUB></SMALL> Buffer
</LI><LI>(n+1) x 1 μL of dNTPs
</LI><LI> (n+1) x 27,5 μL of distilled water.
</LI><LI> (n+1) x 0,5 μL of Q<SMALL><SUB>5</SUB></SMALL></LI></UL></P><P><I><SMALL><B>Note: Take the enzyme out of -20°C just before utilisation.</B></SMALL></I>
Split 39 μL of the master mix in each tube.
Add 1 μL of genomic DNA in each tube, except in your negative controle tube. In this one, put 1 μL of distilled water instead.</P>
Put your tubes into Thermo Cycler.
<H3>PCR Cycles</H3><P>Set the following program on your Thermo Cycler:</P><OL><LI>First denaturation: 30 seconds at 98°C
</LI><LI>Denaturation step 10 seconds at 98°C
</LI><LI>Annealing step 20 seconds at 56°C
</LI><LI>Elongation step (n+1) minutes at 72°C (<I>as n is the number of kB of your sequence</I>)
</LI><LI>Final step 5 minutes at 72°C
</LI></OL><P>Steps 2, 3 and 4 are repeated 29 times.
Then let the mixture cool down at 4°C.<I><B><SMALL>Note: Annealing temperature depend on your primers and elongation temperature on the polymerase used.</SMALL></B></I></P><H1> Golden Gate</H1><H2> Goal </H2>

The aim of Golden Gate assembly is to assemble in one step various fragment of DNA.


<DIV align="center"><DIV class="captionedPicture" style="width:60%;float:center;"><DIV class="caption"><B>Figure 1:</B> Golden Gates Assembly principle
   </DIV></DIV></DIV><H2> Preparation</H2><H1> Competent cells </H1><H2> Goal </H2><P>
Bacteria can integrate DNA fragments from the environment. <I>Escherichia coli</I>, contrary to <I>Bacillus subtillis</I> another bacteria frequently used in molecular and synthetic biology, is not naturally in a state of competence (able to integrate DNA fragments during <A href="https://2013.igem.org/Team:Evry/Protocols/Test#Transformation" target="_blank">transformation</A>), that is why we have to prepare them.
Two methods can be used:
  -Chimiocompetent cells
  -Electrocompetent cells


We choose to use chimiocompetent cells for usual transformation and electrocompetent cells just for ΔFur strain preparation by <A href="https://2013.igem.org/Team:Evry/Protocols/Test#Homologous_recombination" target="_blank">homologous recombination</A>.</P><H2> Preparation </H2><P>Grow overnight pre-culture of the strain you want to make competent (chimio or electro) in 2 mL of LB.</P><H3>Chimiocompetent cells</H3><H4>Solutions preparation</H4><P><B>Solution for 1M Cacl2:</B>
Add 14,30g of CaCl2 into 100 ml distilledwater</P><B><P>Solution for 0,1M Cacl2:</P></B>
Add 50 mL of CaCl2 1M solution into 450 ml of distilled water<P><B>Solution for 0,1M Cacl2 + 15% glycerol:</B>
Add 50 mL of CaCl2 1M solution and 75 mL of glycerol 100% into 450 ml of dissilted water</P><H4>Competent cells preparation</H4><P>For 200 ml LB medium, add 400 µL of strain sample.
Let the bacteria grow until it reaches an Optical Density (OD) between 0,3 and 0,35. Chimiocompetent bacteria are more able to transform when they are in exponential phase.
Once it reached the right OD, put the medium on ice for 30 minutes to slow down growth.
Split the 200 mL into 4x50 mL tubes then centrifuge at 3000 rpm for 5 minutes at 4°C and suppress supernatant afterwards.
Resuspend the pellet cells with 5 mL of Cacl2 at 0,1M for each 50 mL tube.
Again, put the medium on ice for 30 minutes.
Centrifuge at 3000 rpm for 5 minutes at 4°C then suppress supernatant.
Resuspend the cells with 1 mL of Cacl2 at 0,1M + 15% glycerol for each 50 mL tube.
Split the total 4 mL into 40 tubes containing each 100 µL of concentrated cell solution. This step should be executed fast enough and on ice.
Store at - 80°C.</P><H3 id="Electro">Electrocompetent cells</H3><P>For 200 ml LB medium, add 200 µL of strain sample.
Let the bacteria grow until it reaches an OD between 0,5 and 0,7.
Once it reached the right OD, put the medium on ice for 20 minutes to slow down growth.
Split the 200 mL into 4x50 mL tubes then centrifuge at 4000 rpm for 15 minutes at 4°C and suppress supernatant afterwards.
Resuspend the pellet cells with 50 mL of ice-cold 10% glycerol for each 50 mL tube.
Centrifuge at 4000 rpm for 15 minutes at 4°C then suppress supernatant.
Resuspend the cells with 25 mL of ice-cold 10% glycerol for each 50 mL tube.
Centrifuge at 4000 rpm for 15 minutes at 4°C then suppress supernatant.
Resuspend the cells with 2 mL of ice-cold 10% glycerol for each 50 mL tube.
Centrifuge at 4000 rpm for 15 minutes at 4°C then suppress supernatant.
Resuspend the cells with 2 mL of ice-cold 10% glycerol for each 50 mL tube.
Split the total 4 mL into 40 tubes containing each 100 µL of concentrated cell solution. This step should be executed fast enough and on ice.
Store at - 80°C.</P><H2>Contamination and competence tests</H2><P>Plate each strain on LB medium with Ampicillin, Kanamycin or Chloramphenicol in order to evaluate if it contaminated or not. 
Transform each strain with a pSB1A3 plasmid (red colonies) and plated them on LB medium with Ampicillin only to evaluate wether our strains are competent or not.
If the bacteria is competent, it will incorporate the plasmid that code for a red protein and thus have a red phenotype.(see figure below)</P><DIV align="center"><DIV class="captionedPicture" style="width:40%;float:center;"><DIV class="caption"><B>Figure 1:</B><I>Escherichia coli</I> (MG1655 strain) on the left are highly competent and expressed RFP, bacteria on the right are little competent.
   </DIV></DIV></DIV><H1> Transformation </H1><H2> Goal </H2><P>In this step, the heat shock (between -80°C up to 42°C) induces chimiocompetent bacteria to be in state of competence. DNA from the environment (in this case: plasmid) can then integrate the cell.</P><H2> Preparation </H2><H3>Chimiocompetent cells</H3><P>For 100 µL of chemical competent cells, add 1 µL of plasmidic DNA. If the ligation protocol is a
<A href="https://2013.igem.org/Team:Evry/Protocols/Test#Golden_Gate" target="_blank">Golden Gate</A> , add 5 µL instead.
Let 30 minutes on ice.
Let the cells at 42°C for exactly 50 seconds.
Let 5 minutes on ice.
Resuspend the cells with 1 mL of LB spread on a petri dish then let it at 37°C.</P><H2> Controles</H2><P>To check if the transformation have work, make a positive and a negative controle.<LI>Positive controle: Use plasmid pSB1A3
If the transformation have worked, colonies must have a red phenotype.</LI><LI>Negative controle: Ø 
Without plasmid with antibiotic resistance, no colonies must grow. </LI></P><H1> Plasmid purification </H1><H2> Goal </H2><P>The aim of the plasmid purification step is to recover the plasmid produced by the bacteria (cloning strain).
The different step are use to throw other bacterial components off (proteins, cell wall, genomic DNA, etc).</P><H2> Preparation </H2><I>Protocol adapted from Macherey-Nagel plasmid purification notebook</I><B><P>1. Cell culture</P></B>
Cultivate cells in LB medium overnight.<P><B>2. Cell harvesting</B>
Set saturated E.coli LM culture into 2 mL tubes. Centrifuge at 11 000 x g for 30 secondes. Discard as much as supernatant as possible.</P><P><B>3. Cell lysis</B>
Add 250 μL of Buffer A1 (resuspension buffer). Resuspend the cells with a vortex or a pipette.

Add 250 μL of Buffer A2 (lysis buffer). Mix gently by inverting the tube 6 - 8 times. Incubate at room temperature until lysate appears clear.
Add 300 μL of Buffer A3 (neutralisation buffer). Mix thoroughly by inverting the tube 6 - 8 times .</P><P><B>4. Lysate clarification</B>
Centrifuge at 11 000 x g for 5 minutes . Repeat this step until supernatant is not clear.</P><P><B>5. DNA Binding</B>
Place a NucleoSpin Plasmid Column in a Collection Tube of 2 mL and set the supernatant from the last step. Centrifuge at 11 000 x g for 1 minute. Discard flow-through and place the column back into the collection tube.</P><P><B>6. Membrane washing</B>
Add 600 μL of Buffer A4 (wash buffer) previously supplemented with ethanol. Centrifuge ar 11 000 x g for 1 minute. Discard flow-through and place the column back into an empty collection tube.</P><P><B>7. Dry membrane</B>
Centrifuge at 11 000 x g for 2 minutes and discard the collection tube.</P><P><B>8. DNA Elution</B>
Place the column in a 1,5 mL and add 50 μL de Buffer AE (elution buffer). Incubate at room temperature and centrifuge at 11 000 x g for 1 minute.</P><H2> Test </H2><P>Before the storage of the tubes at -20°C, measure the concentration with nanodrop.</P><DIV class="captionedPicture" style="width:50%;float:left;"><DIV class="caption"><B>Figure 1:</B> Figure 1: Nanodrop test after a plasmid purification</DIV></DIV><P>
If the concentration is between 30 and 50 ng/μL, concentrate your DNA with SpeedVac concentrator or any other method.
If the concentration is below 30 ng/μL or if 260/280 ratio or 260/230 ratio is not correct, make another purification.<I>260/280 ratio and 260/230 indicate the purity of DNA (or RNA). 
Nucleic acids absorb at 260 nm while proteins and phenols absorb at 280 nm and carbohydrates at 230 and other contaminents at 230 nm.
For DNA, a 260/280 ratio around 1,8 is concider to be pure and ange 260/230 ratio must be between 2,0 and 2,2.</I></P><H1> Gel electrophoresis analysis</H1><H2> Principle</H2><P>Electrophoresis is a method using electrical field to separate DNA or RNA sequence by size. Smaller the fragment is, the more it migrates on the gel. 
Using a DNA ladder, we can know the size of DNA sequence and then check if we have the sequence we wanted.
</P><H2> Preparation</H2><P>Put 0.5 g of Agar in 50 mL of TAE 1X and heat until the mix is melted.
Once the agar have dissolved and the mixture have cooled down, put 0.3 mL of EtBr in it.
Pour the mixture on the mould, put the comb and wait until the gel polymerized.
Put the gel in the electrophoresis unit (with TAE 1X).
Put your DNA sample (with loading dye) in the PCR well, do not forget the kB ladder.
After the migration (<I>~30 to 45 minutes depanding to your fragment size</I>), observe the gel with UV light.</P><H1> PCR Purification </H1><H2> Goal </H2><P>The aim of the PCR purification step is to recover DNA from <A href="#Polymerase_Chain_Reaction" target="_blank">PCR</A> and other enzymatic reaction mixtures.</P><H2> Preparation </H2><I>Protocol adapted from Thermo Scientific PCR Purification notebook</I><B><P>1. Mixture preparation</P></B>
Add 1:1 volume of Binding Buffer to completed PCR mixture (e.g. if you have 100 µL of reaction mixture, add 100 µL of Binding Buffer).
Mix thoroughly and check the solution's color: a yellow color color indicates an optimal pH for DNA binding.
If the color of the solution is orange or violet, add 10 µL of 3 M sodium acetate, pH 5,2 and mix. The color will become yellow.
If the DNA fragment is superior to 500 bp, add a 1:2 volume of 100% isopropanol (e.g. if you have 100 µL of PCR mixture combined with 100 µL <I>(total amount of 200 µL)</I>, add 100 µL isopropanol). Mix thoroughly.<P><B>2. Binding DNA</B>Transfer up 800 µL of the solution from step previous step to the GeneJET purification column.
Centrifuge for 30-60 seconds, then discard the flow-through.
</P><P><B>3. Membrane washing</B>
Add 700 µL of Wash Buffer (previously dilted with ethanol) to the purification column. 
Centrifuge for 30-60 seconds, then discard the flow-through and place the purification column back into the collection tube?
</P><P><B>4. Dry membrane</B>
Centrifuge the empty GeneJET purification for 1 minute to completely remove any residual wash buffer</P><P><B>5. DNA Elution</B>
Transfer the GeneJET purification column to a clean 1,5 mL tube.
Add 50 µL of Elution Buffer to the center of the GeneJET purification column membrane and centrifuge for 1 minute.

Discard the GeneJET purification column, test with the Nanodrop and store the purified DNA at -20°C.</P><H1> BioBrick Assembly </H1><H2> Goal </H2><P>Once you have your constructions on your plasmids, you need to biobrick them before sending them to the registry.</P><H2> Preparation </H2><H3>Restriction</H3><P>For restriction you need: distilled water, Bovine Serum Albumin (BSA, <I>used to stabilize restriction enzymes</I>), Restriction Buffer 1,2,3 or 4 (buffer 2 works for all combinations of EcoRI-HF, XbaI, PstI, SpeI), two restriction enzymes per sample. 

Determine which restrictions enzymes and Buffer you need using openwetware.org<SUP><A href=" http://openwetware.org/wiki/Enzyme_selection_for_BioBricks_digest" target="_blank">1</A></SUP>.
Determine the DNA-concentration of the vector and the insert. 
Prepare restriction mix for all restrictions,for each reaction add:
<UL><LI>12,5 μl od distilled water
</LI><LI>5,0 μl of Buffer 1,2,3 or 4 
</LI><LI>0,5 μl of BSA 
</LI><LI>1,0 μl of restriction restriction enzyme 1
</LI><LI>1,0 μl of restriction restriction enzyme 2
</LI></UL></P><P>Then, for each reaction, add: 
<UL><LI>20 μl restriction mix previously prepared to a PCR-tube
</LI><LI>1 μg of DNA. 
</LI><LI>Distilled water, until your reach a total volume of 50 μl
</LI></UL>

Incubate for 2 hours at 37°C, then put your tubes  20 minutes at 80°C to inactivate the restriction enzymes.</P><H3>PCR Purification</H3><P>Purify the restriction digest, and then determine the concentration of the purified DNA with Nanodrop.
For more information, see the <A href="https://2013.igem.org/Team:Evry/Protocols/Test#PCR_Purification" target="_blank">PCR purification</A> protocol.</P><H3>Ligation</H3><P>
Prepare ligation mix.
Prepare two extra reactions, “No insert” and “No ligase”  that will be used as negative controls.
</P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Per vector/insert
   </TH><TH align="center">
    No insert
   </TH><TH align="center">
    No ligase
   </TH></TR></THEAD><TBODY><TR><TD align="center">
Ligase
   </TD><TD align="center">
0,5 μL
   </TD><TD align="center">
0,5 μL
   </TD><TD align="center">
0
   </TD></TR><TR><TD align="center">
Vector
   </TD></TR><TR><TD align="center">
Insert
   </TD><TD align="center">
0 μL
   </TD></TR><TR><TD align="center">
Distilled water
   </TD></TR><TR><TD align="center">
10X Buffer
   </TD><TD align="center">
2 μL
   </TD><TD align="center">
2 μL
   </TD><TD align="center">
2 μL
   </TD></TR></TBODY></TABLE><P>
For each reaction,add: <UL><LI>50 ng of vector-DNA.
</LI><LI>Insert in three times molar excess, then add 3·50·(<I>length of insert/length of vector</I>) ng of insert-DNA.
</LI><LI>Distilled water, until your reach a total volume of 20 μl
</LI></UL><SMALL><B><I>Note: For “No insert” omit the insert-DNA, for “No ligase” omit the ligase and replace with H2O.</I></B></SMALL>
Then, incubate all your tubes for 30 minutes at 16°C.</P><H2>Transformation Test </H2><A href="https://2013.igem.org/Team:Evry/Protocols/Test#Transformation" target="_blank">Transform</A> the ligation product with <A href="https://2013.igem.org/Team:Evry/Protocols/Test#Competent_cells" target="_blank">competent cells</A>, and also &quot;No insert&quot; and &quot;No ligase&quot; controles.<UL><LI>If no colonies form in both controle, the restriction was perfect for both the restriction enzymes.</LI><LI>If colonies form in both cases, neither of the restriction enzymes worked well. Most of the transformants will contain the vector only.</LI><LI>If the “No insert” control resulted in many colonies but “No ligase” did not, one of the enzymes worked well and the other did not. Most of colonies will carry the self-ligated vector.</LI></UL><P>Re-isolate colonies.
Check that colonies have been successfully transformed (e.g. by PCR).
Inoculate those colonies in overnight culture for further use.
Send your part to iGEM registry !</P><DIV id="citation_box"><P id="references">Reference:</P><OL><LI> http://openwetware.org/wiki/Enzyme_selection_for_BioBricks_digest </LI></OL></DIV><H1>BioBricks Characterization overview</H1><H1>TECAN Analysis</H1><H2> Goal </H2><P>
The Tecan is a plate reader that is able to characterize our bacteria of interest by measuring, for example, growth rate (OD) and/or protein production (GFP). The software saves the measurements over time, thus allowing to study the kinetics of the studied phenomena. With a 96 wells plate (8x12), we are able to test various conditions.
</P><H2> Steps </H2><P>
The hardest part when you use a plate reader to characterize your parts is to carefully choose your controls and anticipate the timings of your bacterial pre-cultures.
We tested various controls during this summer:<UL><LI>TOP10 growth control</LI><LI>BL21 growth controlFor GFP expression: GFP expression under the control of a pLac promoteLac operator in a delta-LacI strain and GFP expression under the control of a Fur operator in a Δ-Fur strain.
</LI></UL></P><H2> Preparation </H2><H3>Medium preparation</H3><P>
LB medium emit a side signal. As turbidity (OD) and fluorescence of our sample are measured, M9 medium is use instead.


</P><P><U>Composition for 50 mL of:</U></P><TABLE cellpadding="10" cellspacing="0" align="center" border="1"><THEAD><TR><TH align="center">
    Reagent
   </TH><TH align="center">
    M9 medium (without iron)
   </TH><TH align="center">
    M9 medium (with iron)
   </TH></TR></THEAD><TBODY><TR><TD align="center">
    M9 salt (5X)
   </TD><TD colspan="2" align="center">
    10 mL
   </TD></TR><TR><TD align="center">
    CaCl<SUB>2</SUB> (1M)
   </TD><TD colspan="2" align="center">
    5 µL
   </TD></TR><TR><TD align="center">
    MgSO<SUB>4</SUB> (1M)
   </TD><TD colspan="2" align="center">
    100 µL
   </TD></TR><TR><TD align="center">
    Glycerol (50%)
   </TD><TD colspan="2" align="center">
    800 µL
   </TD></TR><TR><TD align="center">
    Thiamine
   </TD><TD colspan="2" align="center">
    5 µL
   </TD></TR><TR><TD align="center">
    NaOH (pH 7.4)
   </TD><TD colspan="2" align="center">
    12.5 µL
   </TD></TR><TR><TD align="center">
    H<SUB>2</SUB>O
   </TD><TD align="center">
    40 mL
   </TD><TD align="center">
    39 mL
   </TD></TR><TR><TD align="center">
    FeSO<SUB>4</SUB> (10mM)
   </TD><TD align="center">
    -
   </TD><TD align="center">
    50 µL
   </TD></TR><TR><TD align="center">
    Casamino acids (0.2%)
   </TD><TD align="center">
    -
   </TD><TD align="center">
    1 mL
   </TD></TR></TBODY></TABLE><P>
Once the mixture is prepared, the medium must be filtered to be sterilised using 0.22 µm filter.
</P><H3>Pre-culture preparation</H3><P>
In a 15 mL tube, add 2 mL of M9 medium and inoculate BL21 cells (expression strain) from glycerol
To inhibit the expression of sfGPF, use M9 with iron, instead of classical M9.

After one night of culture, refresh the precultures by diluting them 200 times in M9 medium (with iron and carbenicillin).
After 8 hours of culture, prepare your wells plate according to the following scheme:</P><P><I>
Technical triplicate are made to see the variation induced by the manipulator.
Biological duplicate are made to determine the natural variation of the process observed.</I></P><H1>Microfluidic experiment </H1><H2> Goal </H2><P>The aim of microfluidic analysis is to trap bacteria in a chip, with a continuous medium. Hence, the behaviour (e.g. GFP production) of bacteria in exponnential phase can be analyse with microscope during long acquisition (&gt;24h).</P><H2> Chip preparation </H2><DIV class="captionedPicture" style="width:25%;float:right;"><DIV class="caption"><B>Figure 1:</B> Microfluidic ship mould<I>(designed by Boris Kirov)</I></DIV></DIV><P>Mix gently 36 g of Polydiméthylsiloxane (PDMS,condensed formula: (C<INF><SMALL>2</SMALL></INF>H<INF><SMALL>6</SMALL></INF>OSi)<INF><SMALL>n</SMALL></INF> ) with 3.6 g of curring agent (which make the PDMS polymerize).
Put in a 50 mL tube and centifugate it at 13 000 for 1 minute to remove bubbles from the mixture.
Pour the mixture on the chip mould and put it under a vacuum pump. 




Once there is no more bubbles that go out of the mixture, put it in the  oven at 80°C for 2 hours.
Separate the different chips and make the holes of the input and outpout of channels with a punch.
Remove all the dust on chips surfaces. Then put chips and glass slides on plasma oven.
After that, put quickly the chip on glass  slide and put them in the oven at 80°C for 2 hours.</P><H2> Bacteria preparation </H2><P>Make 30 mL of preculture of your strain of interest in LB medium with the recquired antibiotics to avoid contamination. 
Let it growth overnight.
Centrifugate and resuspend the pellet in 3 mL of LB medium with the recquired antibiotics and mild detergent
Inject bacteria in the chip with needle and syringe.


</P><DIV align="center"><DIV class="captionedPicture" style="width:85%;float:center;"><DIV class="caption"><B>Figure 2:</B> Microfluidic system acquisition from microfluidic ship (A: two ship), microscop (B) and injecting system (C)
   </DIV></DIV></DIV>


Prepare 2x25 mL of LB medium with antibiotic and mild detergent and inject it in the chip.
Check your preparation with the microscope and let the bacteria growth overnight.<DIV align="center"><DIV class="captionedPicture" style="width:40%;float:center;"><DIV class="caption"><B>Figure 3:</B> A <I>Escherichia coli</I> (TOP 10 strain) in a microfluidic trap.
                     At the top and bottom of the picture, the flow of medium go throught the 2 channels.
   </DIV></DIV></DIV><H2>Acquisition</H2><P>Wash the chip with your recquired medium (in our case: M9 with or without iron).</P><H1>Homologous recombination </H1><H2> Goal </H2><H2> Preparation </H2><FONT color="#ff0000"><B>Be careful, all recombination protocols should be done at 30°C unless indicated otherwise !</B></FONT><H3>Strain Preparation</H3><P>
Transform the PTKred Plasmid<A href="http://www.addgene.org/41062/" target="_blank"><SMALL><SUP>1</SUP></SMALL></A> with the strain of interest.

Start an over night culture with the strain transformed with PTKred at 30°C.

Dilute the overnight culture in a ratio of 1 mL/100 mL and add IPTG (isopropyl β-D-1-thiogalactopyranoside; final concentration of 2 mM) and Spectinomycin (60 mg/mL).

Grow up to OD600: 0.5 then make <A href="https://2013.igem.org/Team:Evry/Protocols/Test#Electro" target="_blank">electrocompetent cells</A> . 
</P><H3>Integration</H3><P>
Prepare the PCR fragment to be integrated flanked by 50 bp genomic sequences.

<A href="#PCR_Purification" target="_blank">Gel-purify</A> the product and transform with the electrocompetent PTKred strain.

Recover in LB for 2 hours with IPTG.

After 2 hours incubation add the selective drug which should be inserted in the genom (for example kanamycin if you have kan R cassette inserted) and incubate overnight in liquid culture at 30°C.

Next day, culture should be very turbid or the recombination did not work. Plate streak 50 μL of the overnight culture on a plate containing selection drug which in this case is kanamycin and grow overnight at 30°C.

Next day pick a colony and make glycerol stocks 
</P><H3>Elimination of PTKred plasmid</H3><P>
Inoculate the strain that you have made by Homologous Recombination in LB overnight with spectinomycin at 30°C.
<I>Adding spectinomycin in this step ensures that the plasmid will not integrate in the genome in a rare occurrence.</I>Dilute 1/100 and grow at <B>42°C</B> for 4 hours and then plate sreak 50 μL of the culture on LB plate at <B>42°C</B>.  
Next day, pick colonies and grid plate on LB then spectinomycin and grow at <B>37°C</B>.
</P><H3>Flip Recombinase and Anti-biotic Resistance Gene</H3><P><A href="https://2013.igem.org/Team:Evry/Protocols/Test#Transformation" target="_blank">Transform</A> a cured strain with PCP20 plasmid 

Grow overnight in LB with Cam at 30°C overnight.

Dilute 1/100 and grow for 3 hours at <B>37°C</B>.

Streak plate on LB plates and grow over night at <B>42°C</B>.  

Repeat the grid.
</P><DIV id="citation_box"><P id="references">Reference:</P><OL><LI>http://www.addgene.org/41062/</LI></OL></DIV><H1> Siderophore detection </H1><H2> Goal </H2><P>Once our bacteria is transformed with the plasmid with the Fur Binding Site and Lac I and the two plasmids with Enterobactins genes, we need to check if our system really work. The production of siderophores can be detected visualy with Blue Agar Chrome Azurol S (CAS). Without siderophore in the medium, CAS and Hexadecyltrimethylammonium bromide  (HDTMA) complexes with ferric iron, producing a blue color. When a bacteria strain produce siderophore, the medium color change from blue to orange. 
</P><H2> Preparation </H2><I>Protocol adapted from Louden, B.C., Haarmann, D., and Lynne, A. (2011). Use of Blue Agar CAS Assay for Siderophore Detection. 
<SUP>1</SUP></I><H3> Blue Dye</H3><B>Solution 1</B>
Dissolve 0,06 g of CAS in 50 mL of distilled water.<P><B>Solution 2</B>
Dissolve 0,0027 g of FeCl<SUB><SMALL>3</SMALL></SUB>-6H<SUB><SMALL>2</SMALL></SUB>0 in 10 mL of 10 mM HCl.</P><P><B>Solution 3</B>
Dissolve 0,073 g of HDTMA in 40 mL of distilled water.</P><P><B>Mix</B>
Mix solution 1 with 9 mL of solution 2. Then mix with solution 3.
Solution should have a blue color. 
Autoclave and store in a bottle. </P><H3> Mixture solution</H3><P><B>Minimal Media 9 (MM9) Salt Solution Stock</B>
Dissolve 15 g of  KH<SUB><SMALL>2</SMALL></SUB>PO<SUB><SMALL>4</SMALL></SUB>, 25 g of NaCl and 50 g of NH<SUB><SMALL>4</SMALL></SUB>Cl in 500 mL of distilled water.</P><P><B>NaOH Stock</B>
Dissolve 25 g of NaOH in 150 mL of distilled water.
pH should be around 12.</P><P><B>20% Glucose Stock</B>
Dissolve 20 g of glucose in 100 mL of distilled water.</P><P><B>Casamino Acid Solution</B>
Dissolve 3 g of Casamino acid in 27 mL of distilled water.
Extract with 3% 8-hydroxyquinoline in chloroform to remove iron.
Filter with a 0,22 µm millipore.</P><H3> CAS agar preparation</H3><P>Add 100 mL of MM9 salt solution to 750 mL of distilled water.
Bring pH up to 6 and dissolve 32,24 g of piperazine-N,N'-bis(2-ethanesulfonic acid) (PIPES); PIPES will not dissolve between pH of 5.
Add 15 g of Bacto Agar.
Autoclave and the cool to 50°C.
Add 30 mL of sterile Casamino acid solution and 10 mL of sterile 20% glucose solution to MM9/PIPES mixture.
Slowly add 100 mL of Blue Dye solution along the glass wall while mixing thoroughly.</P><H3> Bacteria </H3><DIV align="center"><DIV class="captionedPicture" style="width:40%;float:center;"><DIV class="caption"><B>Figure 1:</B> CAS plate with four different bacteria plated. Moving clockwise from top left: <I>Micrococcus luteus</I>, <I>Staphylococcus aureus</I>, <I>Enterobacter aerogenes</I>, <I>Escherichia coli</I></DIV></DIV></DIV><DIV id="citation_box"><P id="references">References:</P><OL><LI>Louden, B.C., Haarmann, D., and Lynne, A. (2011). Use of Blue Agar CAS Assay for Siderophore Detection. Journal of Microbiology &amp; Biology Education 12,.

</LI></OL></DIV></DIV></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/Protocols/">http://2013.igem.org/Team:Evry/Protocols/</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Protocols/" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Protocols/" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Protocols/&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Protocols/&amp;oldid=314811" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>