"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Notebook
From 2013.igem.org
Iron coli project
Home
Project
Iron metabolism
Related diseases
Iron chelators
Achievements
References
Team
iGEM Profile
Attributions
Collaborations
Acknowledgement
Press
Photo gallery
Model
Flush treatment
Enterobactin production
Genome scale
Population scale
Logistic functions
Chemical reasoning
Plate Reader
Biology
Parts
Sensor
Inverter
Chelator
Results
Human Practice
Consultation
Survey
Seminar
Philosophy
Safety
Economy
Art
Capsule
Notebook
Protocols
Primers
Sponsors
Protocols: BioBricks' Conception
Protocols' overview
Solutions and Media
DNA genomic extraction
PCR
Golden Gate
Competent cells
Transformation
Plasmid purification
Ectrophoresis
PCR Purification
BioBrick Assembly
Protocols: BioBricks' Characterization
Protocols' overview
TECAN Analysis
Microfluidic experiment
Homologous recombination
Siderophore detection
Primers
JUNE
Week 1
Week 2
JULY
Week 3
Week 4
Week 5
Week 6
AUGUST
Week 7
Week 8
Week 9
Week 10
Week 11
SEPTEMBER
Week 12
Week 13
Week 14
Week 15
OCTOBER
Week 16
Week 17
Week 18
Week 19
Week 20
Week 1: 17th June - 23rd June
Tuesday, 19th June
As a start for the labwork, we first decided to make competent cells. We grew from glycerol samples three E. coli strains in 2 mL of LB medium.
The chemically competent strains chosen are as follows:
BL21
DH5α
TOP10
Protocol for pre-culture preparation
In a 15 mL tube, add 2 mL of LB medium and inoculate cells from glycerol
Thursday, 20th June
We started to make 200 mL of BL21 and DH5α E. coli strains and 400 mL of TOP10 to make competent cells.
Protocol for competent cells
First prepare the following solutions required to make competent cells:
Solution for 1M Cacl2:
Add 14,30g of CaCl2 into 100 ml desalted water
Solution for 0,1M Cacl2:
Add 50 mL of CaCl2 1M solution into 450 ml of desalted water
Solution for 0,1M Cacl2 + 15% glycerol:
Add 50 mL of CaCl2 1M solution and 75 mL of glycerol 100% into 450 ml of desalted water
For 200 ml LB medium, add 400 µL of strain sample.
Let the bacteria grow until it reaches an OD between 0,3 and 0,35.
Once it reached the right OD, put the medium on ice for 30 minutes to slow down growth.
Split the 200 mL into 4x50 mL tubes then centrifuge at 3000 rpm for 5 minutes at 4°C and suppress supernatant afterwards.
Resuspend the cells with 5 mL of Cacl2 at 0,1M for each 50 mL tube.
Again, put the medium on ice for 30 minutes.
Centrifuge at 3000 rpm for 5 minutes at 4°C then suppress supernatant.
Resuspend the cells with 1 mL of Cacl2 at 0,1M + 15% glycerol for each 50 mL tube.
Split the total 4 mL into 40 tubes containing each 100 µL of concentrated cell solution. This step should be executed fast enough and on ice.
Friday, 21st June
To check the quality of our work, each strain has been plated on LB medium with ampicillin, kanamycin or chloramphenicol in order to evaluate if it contaminated or not. Furthermore, to evaluate wether our strains are competent or not, we also transformed our bacteria with a pSB1A3 plasmid (red colonies) and plated them on LB medium with ampicillin only.
Protocol for transformation
For 100 µL of chemical competent cells, add 1 µL of plasmidic DNA.
Let 30 minutes on ice.
Let the cells at 42°C for exactly 50 seconds.
Let 5 minutes on ice.
Resuspend the cells with 1 mL of LB spread on a petri dish then let it at 37°C.
Week 2: 24th June - 30th June
Monday, June 24th
Our competents cells made on friday the 21st were plated and incubated the whole week-end at 30°C. Today, we analyzed the plates and came to the conclusion that the medium was contaminated due to poor protocol execution. As a consequence, we decided to make new LB medium and recalculate the antibiotic concentrations.
Protocol for LB medium preparation
Add 20 g of LB broth into 1000 mL of desalted water and autoclave.
Protocol for antibiotic solutions preparation
Kanamycin is used as an effective concentration of 10-50 µg/ml and at a storage concentration of 10 mg/ml
Carbenicillin is used as an effective concentration of 20-60 µg/ml and at a storage concentration of 50 mg/ml
Chloramphenicol is used as an effective concentration of 25-170 µg/ml and at a storage concentration of 34 mg/ml (ethanol)
Tetracyclin is used as an effective concentration of 10-50 and at a storage concentration of 5 mg/ml (ethanol)
Tuesday, June 25th
New competent cells have been made with the same bacterial strains (BL21, DH5α and TOP10). We thought that the contamination of our cells was due to the CaCl2 solution that may not have been sterile and/or incorrect manipulated. As a consequence, we prepared new CaCl2 solutions made sure to autoclave them before usage. We ran the same tests as monday the 24th to evaluate the quality of our work and check for potential contamination.
Wednesday, June 26th
The results of the competent cells protocol is as follows:
DH5α strain was not contaminated but competent
TOP10 strain was contaminated only on the Kanamycin plate but competent
BL21 strain was completely contaminated and, thus, not usable.
Thurdsay, June 27th
We finished designing are first two plasmids and constructed the primers to extract the natural promoters from the genomic DNA from E. coli and the different parts for Golden Gate assembly.
Friday, June 28th
We want to obtain reasonnable competent cells for our oncoming transformation for newt week. Thus, we decided to plate the three strains on LB-Agar medium in order to isolate one and only one colony for monday when we'll start over the whole competent process.
Week 3: 1st July - 7th July
Tuesday, July 2nd
We prepared the BL21 and BG1655 strains to be competent.
Wednesday, July 3rd
The two strains have been contaminated  and are not usable.
Thursday, July 4th
Genomic DNA extraction of the BL21 and BG1655 strains have been done using two different methods.  We received the primers we have ordered. They have been prepared to do a PCR. In order to get our genes of interest we did a PCR using as the DNA template the genomic DNA extracted from BL21 and BG1655.
Friday, July 5th
To ensure the amplification of our genes by PCR, we did a agarose gel electrophoresis. To do so, we first prepared a TAE 50X stock solution. By diluting it, we use a 1X TAE solution to do a 1% agarose gel. We obtain the following gel. That showed that only two of our genes have been amplificated by PCR. A new PCR will be made.
Week 4: 8th July - 14th July
Monday, July 8th
We prepared the BL21 and BG1655 strains to be competent.
Tuesday, July 9th
Both strain are not contaminated.
BL21 are highly competent, BG1655 are little competent.
Wednesday, July 10th
PCR procedure optimization
Aim: Define the number of matrix of optimal DNA to amplify a gene by PCR.
Data:
[primer] = --- ng/µl
size(primer) = --- nt
For PCR we chose to use … ng of primer (V=…µL). As we  know the size of our primers, we could define the number of primers that we have to use to begin the reaction. We define the number of recquired matrix DNA in terms of concentration of sample with the following formula:
Analysis of Agarose gel's results:
Extraction of natural promoter sequence
Aim: We want to have sequences genes' promoters' sequences under the control of FUR (Ferric Uptake Regulation)transcription factor.
We remake PCR samples that failed the previous week.
Fec A
Ent C
Fec C
Ace B
PCR products are placed on gel and purificate. After, PCR products are tested with nanodrop.
Fec A (BG1655)
[c] = 23.7 ng/µl
260/280 = 1.90
Fec A (BL21)
[c] = 53.1 ng/µl
260/280 = 1.73
Ent C (BG1655)
[c] = 52.4 ng/µl
260/280 = 1.71
Ent C (BL21)
[c] = 50.0 ng/µl
260/280 = 1.94
Ace B (BG1655)
[c] = 46.5 ng/µl
260/280 = 1.89
Fec C (BG1655)
[c] = 34.2 ng/µl
260/280 = 1.91
Tris-HCl solution preparation(1M) : Stock solution 100X
Aim: Tris-HCl solution will be use in a final concentration of 10 mM to resuspend our primers.
Preparation of the stock solution (1 M, pH 7.5) for a volume of 50 mL:
Disolve 4,6g of Trisbase in 30 mL of distilled water.
Adjust pH with concentrated HCl(~4 mL) until pH=7,5.
Add distilled water until 50 mL then put in autoclave.
Note: If the solution has a yellow coloration, do the preparation again with better Trisbase.
Preparation of the solution (10mM, pH 7.5) for a volume of 50 mL.
Take 50µL of the stock solution at 1M and add 49.5 mL of water for the dilution.
Kanamycin preparation
Preparation of 3 tubes of 1.5 mL of Kanamycin.
Thursday, July 11
Preparation of primers solution:
Centrifugate tubes at 8000 rpm, begin 30 seconds.
Add 250 µL of Tris HCl (10 mM); [Primer] = 100µM
We prepare a diluted solution at 5 µM from to the stock solution, for PCR reactions.
Dilution at 1/20: We take 5 µL of the stock solution (100 µM) and we diluate in 95 µL of tris-HCl (10 mM).
Transformation of BL21:
Chemical transformation of BL21 with Cyrille's samples:
Terminator (T)
Promotor (P)
Plasmid 1K3
Plasmid 1C3
sfGFP
These transformations allow us to do glycerols of these constrctions.
PCR on E.coli genom (BG1655 strain)
Fep A (Primers P021 and P022)
Fes (Primers P023 and P024)
sdh C (Primers P025 and  P026)
ybi L (Primers P027 and P028)
ync E (Primers P029 and P030)
Friday, July 12
Transformations test
Terminator (T) = OK
Promotor (P) = OK
Plasmid 1K3 = OK
Plasmid 1C3 = OK
sfGFP = OK
Note : The negative control was suspect.
Petri dishes are le on the bench at room temperature during the week-end, colonies will be reisolate next week.
Migration of samples of PCR extraction.
The promoter sequence of genes Fep A, Fes, Sdh C, ybi L and ync E were extracted successfully.
Samples were purified and, after, tested with nanodrop.
Fep A (BG1655)
[c] = 38.3 ng/µl
260/280 = 1.93
Fes (BG1655)
[c] = 45.5 ng/µl
260/280 = 1.89
Sdh C (BG1655)
[c] = 28.5 ng/µl
260/280 = 1.86
ybi L (BG1655)
[c] = 31.0 ng/µl
260/280 = 2.09
ync E (BG1655)
[c] = 25.1 ng/µl
260/280 = 1.93
Week 5: 15th July - 21st July
Tuesday, July 16th
Wednesday, July 17th
We prepared the TOP10 to be competent.
We launch our first Golden Gates.
Week 6: 22nd July - 28th July
Monday, July 22nd
We did a PCR on the seven products of golden gate 1 using the VR and VF2 primers. We mixed :
10 uL of One Taq Buffer 10X
1 uL of 10 mM dNTPs
1 uL of each VF2 and VR primers
35,5 uL of H2O
1 uL of the One Taq enzyme
1 uL of the golden gate products
We used the following PCR program :
95°C for 3 min
95°C for 15 sec
55°C for 30 sec
68°C for 2 min
68°C for 10 min
Note: We repeated step 2, 3 and 4 29 times.
The gel migration didn't reveal any band. We supposed that it hadn't worked due to the annealing temperature we used. Indeed, with the help of the New England BioLabs Tm Calculator website, we have found that the optimal annealing temperature for those primers was 53°C.
Tuesday, July 23rd
In order to test the best annealing temperature for our following PCRs, we did a PCR on our 5 first golden gate products with two different annealing temperature : 53 or 51°C. We also prepared two positiv controls for each conditions with either bacteria transformed by empty PSB1A3 plasmid or 1 uL of the miniprep plasmid. This was realised in order to verify if the problem of our first PCR was due to a bad lysis of our bacteria. What is more, we prepared one negativ control with an annealing temperature of 55°C.
According to our results, we can assume that our first PCR was badly realised due to a handling error and not because of the annealing temperature we used.
In the meantime, we did the Golden Gate 1 again in order to obtain better ligation results. We transformed TOP 10 E. coli with the golden gate products and plated them on LB-Agar with carbenicillin antibiotic. The petri dishes have been let overnight at 37°C.
Wednesday, July 24th
We did a colony PCR from colony obtained on the petri dishes plated with golden gate 2 products transformed bacteria. We chose one white colony of each of the first ten different FUR BS constructions. The positiv control was prepared by using the PSB1A3 plasmid. The same PCR program was used but we were out of One Taq so we utilize Dream Taq.
We re isolate colony of bacteria transformed with our golden gate 1 products.
Thursday, July 25th
We did a 1% gel in order to migrate our PCR products. It revealed that our PCR hadn't work. We then realised that we hadn't changed the elongation temperature of our PCR program. Indeed, the dream taq functions at 72°C.
We did a PCR colony with the white colony obtained after we plated our golden gate 1 products transformed bacteria.
We used the following PCR program :
95°C for 3 min
95°C for 15 sec
55°C for 30 sec
72°C for 2 min
72°C for 10 min
Considering our gel, we can reasonably think that our golden gate had worked properly. We then did miniprep.
Friday, July 26th
We used nanodrop to measure the concentration of plasmid of our minipreps. The concentrations obtained were extremely low. We decided to do it again, but we got the same low results.
Week 7: 29th July - 4th August
Construction of plasmid N°1
We make an electrophoresis with 5 µL of plamsid to check the plasmid purification made on the last friday.
There is not the 3 bandes that we sould see, so to check another time, we make a digestion with Pst I and EcoR I:
NEB Buffer 10X  5 µL
BSA 10X  5 µL
DNA  2 µg
PstI  2 µL
EcoRI  2 µL
Water  50 µL
After another electrophoresis, we obtain no band.
Sequencing preparation
Name
Clone
Concentration
260/280
260/230
Sequencing
AceB Fur Binding Site
(with sfGFP)
1
166.1 ng/µL
1.86
1.76
Good
2
192.0 ng/µL
1.62
1.02
Good
3
123.5 ng/µL
1.87
1.75
Good
4
128.0 ng/µL
1.82
1.66
Good
FepA Fur Binding Site
(with sfGFP)
1
21.1 ng/µL
2.06
-
Good
2
22.4 ng/µL
1.96
-
Good
3
55.4 ng/µL
1.87
1.58
Good
4
68.5 ng/µL
1.89
1.67
Good
Fes Fur Binding Site
(with sfGFP)
1
39.9 ng/µL
1.87
1.80
Good
2
40.6 ng/µL
1.74
-
No sequencing
3
68.4 ng/µL
1.87
1.63
Good
4
20.8 ng/µL
2.05
-
No sequencing
ybiL Fur Binding Site
(with sfGFP)
1
58.2 ng/µL
1.87
1.73
Good
2
16.3 ng/µL
1.95
-
No sequencing
3
18.8 ng/µL
1.95
-
No sequencing
4
45.5 ng/µL
1.90
1.81
Good
FecA Fur Binding Site
(with sfGFP)
1
20.0 ng/µL
1.96
-
Good
2
27.5 ng/µL
1.77
-
No sequencing
3
31.5 ng/µL
1.94
1.81
Good
4
57.0 ng/µL
1.89
1.64
Good
yncE Fur Binding Site
(with sfGFP)
1
92.4 ng/µL
1.90
1.75
Good
2
64.7 ng/µL
1.89
1.66
Good
3
38.6 ng/µL
1.91
1.80
Good
Construction of plasmid N°2
Our plasmid N°2 is building with a synthetic promote sequence which is composed of:
Andersen's promotor
Fur Binding Site (15 different)
RBS
sfGFP
Terminator
Golden Gate
In order to associate these sequences we performed a golden gate, using for each sample :
80 ng of Andersen's promotor
80 ng of Fur Binding Site
80 ng of RBS
80 ng of sfGFP
80 ng of Terminator
1.5 µL of T4 buffer (10X)
15 Unit of T4 ligase
2.5 Unit of Bsa I
The Golden Gate products are chemically transformed into E. coli Top 10 strains. After the transformation process, bacteria are plated into LB medium with carbenicillin and icubated overnigth at 37°C.
Golden Gates plates are composed at 98% of red colonies. The problem was probably on the equimolarity ratio between our different part which were not totally respected. We isolated white colonies (4 maximum by each sample) on LB medium with carbenicillin and we incubated them overnight at 37°c.
We made pre culture (V = 10 mL), using bacterial colonies isolated from our plates of Golden Gate transformation. Then minipreps and glycerol conservation were made using these pre cultures.
Fur Binding Site
Sequence
Clone
Concentration
Fur BS 1
ATTATTGATAACTATTTG
Clone n°1
80.8 ng/µL
Clone n°2
83.8 ng/µL
Clone n°4
90.3 ng/µL
Fur BS 2
GATAACTATTTGCATTTGCA
Clone n°1
68.6 ng/µL
Clone n°2
77.4 ng/µL
Clone n°3
68.0 ng/µL
Fur BS 3
CAAATGCAAATAGTTATCCA
Clone n°1
79.4 ng/µL
Clone n°2
110.5 ng/µL
Clone n°3
457.4 ng/µL
Fur BS 4
CAAATAGTTATCAATAATCA
Clone n°1
73.2 ng/µL
Clone n°2
66.2 ng/µL
Fur BS 5
GTAAATTAATATTATTTACA
Clone n°1
57.2 ng/µL
Fur BS 6
AATAATGCTTCTCATTTTCA
Clone n°1
63.8 ng/µL
Fur BS 7
ATAAATGATAATCATTATCA
Clone n°1
70.6 ng/µL
Fur BS 8
GAAAATAATTCTTATTTCCA
Clone n°1
75.6 ng/µL
Clone n°2
87.2 ng/µL
Clone n°3
48.8 ng/µL
Clone n°4
57.8 ng/µL
Fur BS 9
AGCACTTATTATTATTTTCA
Clone n°1
58.6 ng/µL
Fur BS 10
GATAATTGTTATCGTTTGCA
No Clone
-
Fur BS 11
GATAATGCTTATCAAAATCA
Clone n°2
75.6 ng/µL
Clone n°3
398.0 ng/µL
Clone n°4
57.4 ng/µL
Fur BS 12
CAAAATTATTATCACTTTCA
Clone n°1
50.0 ng/µL
Fur BS 13
AATAATGATTACCATTCCCA
Clone n°2
70.7 ng/µL
Fur BS 14
GAAATTGTTTTTGATTTTCA
Clone n°1
42.7 ng/µL
Clone n°2
32.8 ng/µL
Clone n°3
37.8 ng/µL
Clone n°4
45.7 ng/µL
Fur BS 15
GTTAATTGTAATGATTTTCA
Clone n°1
37.0 ng/µL
Clone n°2
53.2 ng/µL
Clone n°3
196.0 ng/µL
Construction of plasmid N°3
29/07/13
We received the primers ordered on friday the 26th of July and started to dilute them into TrisCL at 10 mM. Secondly, we diluted the previous stock solution at a rate of 1/20th to obtain a final concentration of 5 µM for our intermediate solution.
Then, for our plasmid three construction, we need to first extract the 6 enterobactin gene (EntA, EntB, EntC, EntD, EntE and EntF). The ordered primers from friday will theoretically extract them in the appropriate Golden Gate format with their own RBS upstream (designed from Salis RBS). This will allow us to construct our two N°3 plasmids, one containing EntA, EntD and EntF, and the other one EntB, EntC and EntE. The genes are spread like this to obtain two equivalent plasmids xxx
We proceeded to a genomic extraction of the 6 genes of interest and migrated to PCR products on a 1% gel. We successfully extracted 5 out of the 6.
30/07/13
We annealled the oligonucleotides of the PL-LacO part in the golden gate format. Also, we extracted the sfGFP with the RBS upstream which will allow us to construct the control positive plasmid 3 for future TECAN experiments.
31/07/13
We started by purifying our 6 succesful PCR extraction from the 29/07 and 30/07. So we managed to extract EntA, EntB, EntC, EntD and EntF from E. coli's genomic DNA and sfGFP in the golden gate format with a RBS from plasmidic DNA.
Additionnaly, we optimized our PCR to extract our missing gene, EntE. We obtained a smear +/- a double band. As a consequence, we adjusted the annealing temperature with a range from 54°C to 66°C. This will allow us to try extracting the missing gene (EntE) in the appropriate conditions.
01/08/13
We made a PCR for the EntE gene with a gradient of temperature to know at which temperature the PCR result were the best.
We made 8 tubes with the same composition:
Water
Taq Buffer 5
dNTPs
Rev EntE 2,5
For EntE 2,5
DNA 0,5
Taq polymerase 0,5
We also made positive and negative controles (with Rev EntC and For EntC instead of EntE).
After electrophoresis, we obtain the best band at .. °C.
However our experiment have been made with Taq polymerase (and Taq Buffer) instead of Q5 (and Q5 buffer) that we use usually. Then, we have to repeat this experiment with Q5 and Q5 buffer.
The did a gradient PCR and migrated on a gel our samples. On the gel, we notice that both the smear and the double band disappear when the annealing temperature increases. So to conclude, we run tonight a PCR with Q5 polymerase with increasing temperature starting from 56 degrees. The absolute annealing temperature will be determined for the Q5 (not extrapolable from a OneTaq PCR).
Also, we made a golden gate for the construction of plasmid 3 (chelator system). For analysis purposes, only a GFP was added downstream of the PL LacO promoter. Also, the EntE gene is still to be extracted from the genome. Also, we runned 3 extras golden gate for the construction of plasmid 2 (syntethic sensor system) (syntethic sensor system). In fact, we corrected our calculations because we initially we used mass concentrations instead of molar concentrations.
02/08/13
The transformations of the golden gate 2 and 3 are successful and we managed to invert the rate of red colonies versus white colonies (red = false positive with mRFP, white = positive), thus meaning we have a lot of white ones! We conserved the plates for future isolation on coming monday.
Additionnaly, we migrated the PCR products of 01/08. The gel is empty, but the positive controls are there, meaning we made a mistake in the primers (the master mix is the one used for the positive control).
Week 8: 5th August - 11th August
Plasmid 3 (chelator system):
05/08/13 We started the week by doing the golden gates over again, meaning the GG 3 and GG 2 (2.1 with FurBS 1, 2.2 with FurBS2 and 2.3 with FurBS3). In fact, our previous control positive had some unwanted white spots (01/08/13), thus suggesting some contaminations during the transformation step. curretnmy waiting
06/08/13 We performed 6 golden gates, 3 for the second construction, 2 for the third construction and one for the controle plasmid. We prepared a mix for the 6+1 tubes :
7x 1,27 = 8,89 µl of 1A3 plasmide
7x 1,74 = 12,18 µL of terminator
7x 1,5 = 10,5 µL of T4 Buffer
7 x 0,5 = 3,5 µL of BSA
7 x 0,5 = 3,5 µL of T4 ligase
We then added:
Tube 1. =
0,76 µl of Andersen's promoter
0,76 µL of RBS
0,76 µl OF Fur BS
2,93 µL of sfGFP
Tube 2. =
0,76 µl of Andersen's promoter
0,76 µL of RBS
0,76 µl OF Fur BS
2,93 µL of sfGFP
Tube 3. =
0,76 µl of Andersen's promoter
0,76 µL of RBS
0,76 µl OF Fur BS
2,93 µL of sfGFP
4,28 µL of water
Tube 4. =
1,25 µl of pLac O
µL of RBS-sfGFP
3,4 µL of water
Tube 5. =
1,25 µl of pLac O
2,05 µl of EntA
3,37 µL of EntD
2,07 µl of EntF
0,5 µL of water
Tube 6. = Controle Plasmid
0,76 µl of Andersen's promoter
0,76 µL of RBS
2,93 µl of sfGFP
4,28 µL of water
On the afternoon, we launch another golden gate for EntB/C/E  Tube 7. =
1,27 µl of plasmid 1 A3
1,5 µL of pLacO
0,49 µl of EntB
0,65 µL of EntC
1,74 µl of terminator
1,5 µL of Buffer T4 ligase
0,5 µl of T4 ligase
0,5 µL of Bsa 1
5,88 µL of water
Additionnally, we did the optimization of our PCR over with the temperature gradient for the annealing step. current waiting
09/08 We make 38 tube of Top 10
Week 9: 12th August - 18th August
TECAN
Medium preparation
For the Tecan analysis we must use M9 medium and not a classical LB medium because turbidity and fluorescence of our sample are measured. Then, in order to obtain good results, the medium must not emit a side signal, that is why we use the M9 medium.
Composition for 50 mL of:
Reagent
M9 medium (without iron)
M9 medium (with iron)
M9 salt (5X)
10 mL
CaCl2 (1M)
5 µL
MgSO4 (1M)
100 µL
Glycerol (50%)
800 µL
Thiamine
5 µL
NaOH (pH 7.4)
12.5 µL
H2O
40 mL
39 mL
FeSO4 (10mM)
-
50 µL
Casamino acids (0.2%)
-
1 mL
Once the mixture is prepared, the medium must be filtered to be sterilised using 0.22 µm filter.
TECAN analysis
Using the TECAN analysis, we are measuring the capacity of repression of the natural binding site that we extract from the E.coli's genome.
Preculture with M9 medium (with carbenicillin) have been launched for BL21 transformed with our 1st construction:
Natural Fur Binding Site of Fec A + sfGFP (clone 1, 2, 3)
Natural Fur Binding Site of Fep A + sfGFP (clone 1, 2, 3)
Natural Fur Binding Site of Ace B + sfGFP (clone 1, 2, 3)
Note: Preculture have been made in M9 medium with iron in oder to inhibit the expression of sfGFP.
After one night of culture, time the precultures have been refreshed by diluting them 200 times in M9 medium (with iron and carbenicillin).
After 8 hours of culture, the 96 wells plate has been prepare (see following scheme).
1st construction
Sequencing preparation
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our 1st construction, then minipreps have been realized using the miniprep kit from Machrey Nagel.
Name
Clone
Concentration
260/280
260/230
Sequencing
AceB Fur Binding Site
(with LacI-LVA)
1
66.4 ng/µL
1.89
1.92
Good
2
53.8 ng/µL
1.85
1.61
Good
3
64.9 ng/µL
1.89
1.87
Bad
4
71.8 ng/µL
1.84
1.66
Good
ybiL Fur Binding Site
(with LacI-LVA)
1
49.1 ng/µL
1.94
1.80
Good
2
58.6 ng/µL
1.99
2.32
Good
3
41.2 ng/µL
1.97
2.44
Good
4
31.8 ng/µL
1.93
2.16
Good
yncE Fur Binding Site
(with LacI-LVA)
1
43.7 ng/µL
1.95
2.17
Good
2
45.6 ng/µL
1.97
2.22
Good
3
34.8 ng/µL
2.00
2.41
Good
4
42.5 ng/µL
2.00
2.28
Good
Fes Fur Binding Site
(with LacI-LVA)
1
39.4 ng/µL
1.89
1.82
Good
2
36.6 ng/µL
-
-
Good
4
35.3 ng/µL
1.81
1.13
Good
FepA Fur Binding Site
(with LacI-LVA)
2
50.7 ng/µL
1.90
1.61
Good
4
57.4 ng/µL
1.92
1.88
Good
2nd construction
Bacterial isolation
4 clones of a Golden Gate plate are isolated for each construction. It allows us to save the clone on a plate.
Top 10 transformed with the synthetic Fur Binding
Site n°1, 2 and 3 isolated on a plate.
Sequencing preparation
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our 2nd construction, then minipreps have been realized using the miniprep kit from Machrey Nagel.
Name
Clone
Concentration
260/280
260/230
Sequencing
Fur Binding Site n°1
(with sfGFP)
1
50.7 ng/µL
1.62
0.68
Bad
2
21.1 ng/µL
1.69
0.44
No sequencing
3
21.6 ng/µL
1.20
0.13
No sequencing
4
70.8 ng/µL
1.66
0.72
Bad
Fur Binding Site n°2
(with sfGFP)
1
39.1 ng/µL
1.23
0.40
No sequencing
2
25.1 ng/µL
1.37
0.29
No sequencing
3
61.1 ng/µL
1.57
0.71
Bad
4
54.7 ng/µL
1.62
0.68
Bad
Fur Binding Site n°3
(with sfGFP)
2
38.4 ng/µL
1.84
0.78
No sequencing
3
114.6 ng/µL
1.66
0.78
Bad
4
39.2 ng/µL
1.87
1.21
No sequencing
Fur Binding Site n°5
(with sfGFP)
1
77.7 ng/µL
1.76
0.99
Bad
3
78.4 ng/µL
1.61
0.81
Bad
4
72.7 ng/µL
1.85
4.23
Bad
Fur Binding Site n°6
(with sfGFP)
1
70.4 ng/µL
1.76
2.16
Bad
2
91.5 ng/µL
1.55
1.21
Bad
3
65.4 ng/µL
1.70
1.52
Bad
4
85.6 ng/µL
1.72
1.65
Bad
Fur Binding Site n°7
(with sfGFP)
1
53.5 ng/µL
1.76
1.93
Bad
2
77.6 ng/µL
1.69
1.22
Bad
3
72.3 ng/µL
1.68
1.22
Bad
Fur Binding Site n°8
(with sfGFP)
1
73.9 ng/µL
1.58
0.84
No sequencing
2
51.3 ng/µL
1.75
1.10
Bad
3
45.9 ng/µL
1.85
1.43
Bad
4
46.1 ng/µL
1.88
1.61
Bad
Fur Binding Site n°9
(with sfGFP)
1
108.3 ng/µL
1.78
1.37
Bad
2
67.0 ng/µL
1.88
1.61
Bad
3
103.5 ng/µL
1.87
1.79
Bad
4
28.2 ng/µL
2.06
1.76
No sequencing
Fur Binding Site n°10
(with sfGFP)
1
29.5 ng/µL
1.95
1.22
No sequencing
2
70.0 ng/µL
1.90
1.67
Bad
3
46.0 ng/µL
1.89
1.62
Bad
4
55.8 ng/µL
1.88
1.44
Bad
Fur Binding Site n°11
(with sfGFP)
1
55.3 ng/µL
1.85
1.49
Bad
2
87.1 ng/µL
1.81
1.37
Bad
3
57.7 ng/µL
1.84
1.44
Bad
4
21.7 ng/µL
2.06
1.20
No sequencing
Fur Binding Site n°15
(with sfGFP)
1
16.2 ng/µL
2.13
1.20
No sequencing
2
47.4 ng/µL
1.78
1.16
Bad
4
63.2 ng/µL
1.78
1.16
Bad
Fur Binding Site n°1
(with LacI-LVA)
1
99.7 ng/µL
1.73
0.99
Bad
2
46.9 ng/µL
1.78
0.90
Bad
3
55.8 ng/µL
1.87
1.75
Bad
4
35.3 ng/µL
1.84
0.99
Bad
Fur Binding Site n°2
(with LacI-LVA)
1
43.8 ng/µL
1.83
1.01
Bad
2
47.0 ng/µL
1.77
0.92
Bad
3
32.4 ng/µL
1.78
1.09
Bad
Fur Binding Site n°3
(with LacI-LVA)
1
57.8 ng/µL
1.84
1.45
Bad
2
51.3 ng/µL
1.79
1.04
Bad
3
41.6 ng/µL
1.88
1.75
Bad
4
112.6 ng/µL
1.86
1.81
Bad
Note: Fur Binding Sites n°4, 12, 13 and 14 are missing.
3rd construction
Sequencing preparation
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our plasmid which contains the sfGFP under the control of Lac O promoter. After one night of culture minipreps of 3 different clones have been realized using the miniprep kit from Machrey Nagel.
Name
Clone
Concentration
260/280
260/230
Sequencing
Testing construction
(Lac O with sfGFP)
1
25.7 ng/µL
2.02
2.29
No sequencing
2
27.0 ng/µL
2.09
1.78
No sequencing
3
79.7 ng/µL
1.86
1.60
Good
Golden Gate
A new Golden Gate of our 3rd construction as been made for the two plasmids containing:
Enterobactin biosynthesis genes (Ent A, D and F)
Enterobactin biosynthesis genes (Ent B, C and E)
5 µL of each Golden Gate product are transformed into Top10 competent cells. To controlled the quality of the transformation, a negative controle (transformation procedure without plasmid) and a positive controle (transformation procedure with plasmid 1 A3) are made.
Unfortunatly, the Golden Gate reactions did not work.
Others
Miniprep of Plasmid 1A3 with RFP designed for Golden Gates
Top 10 transformed with the plasmid 1A3 designed for Golden Gate have been cultivated into 10 mL of LB medium (with carbenicillin). After an overnight culture, plasmid has been minipreped using the Machrey Nagel kit.
Sample
Concentration
260/280
260/230
1
265 ng/µL
1.84
1.78
2
237.8 ng/µL
1.82
1.67
3
240.8 ng/µL
1.84
1.81
The plasmid has been diluted as a working solution with a concenration of 80 ng/µL. Then the plasmid has been aliquoted in 50 µL into 9 different tubes.
Competent cells
Top10 competent cells had been made last week and we tested them on different antibiotics (Carbenicillin, Kanamycin, Chloramphenicol). After incubation at 37°C, overnight, no contaminant had been observed.
Week 10: 19th August - 25th August
2nd Construction
Primer hybridation
After sequencing analysis we observed that our Golden Gate results were not significant for the 2nd construction.
However, our sequences of natural promoter give us good results. Then we supposed that the problem of our Golden Gate come from the hybridation of our synthetic promoters.
We made new hybridations of:
Fur Binding Site 1 to 15 (5 µM)
Andersen promoter J23100 (5 µM)
RBS 0034 (5 µM)
Golden Gate
A new Golden Gate of our 2nd construction are made with:
Fur Binding Site 1, 2 or 3 + sfGFP
Fur Binding Site 1, 2 or 3 + LacI-LVA
5 µL of each Golden Gate product are transformed into Top10 competent cells. To controlled the quality of the transformation, a negative controle (transformation procedure without plasmid) and a positive controle (transformation procedure with plasmid 1 A3) are made.
Sequencing preparation
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our 2nd construction, then minipreps have been realized using the miniprep kit from Machrey Nagel.
Name
Clone
Concentration
260/280
260/230
Sequencing
Fur Binding Site n°1
(with sfGFP)
1
20.2 ng/µL
2.07
1.29
Bad
Fur Binding Site n°1
(with LacI-LVA)
1
28.5 ng/µL
2.06
1.65
-
2
25.5 ng/µL
2.01
1.84
No sequencing
3
45.6 ng/µL
1.91
1.90
Bad
4
54.7 ng/µL
1.83
1.27
Bad
Fur Binding Site n°2
(with sfGFP)
1
29.4 ng/µL
2.01
1.46
Bad
2
25.5 ng/µL
2.07
2.00
Bad
3
15.0 ng/µL
2.21
1.72
No sequencing
4
47.1 ng/µL
1.84
1.53
-
Fur Binding Site n°2
(with LacI-LVA)
1
25.4 ng/µL
1.72
0.60
-
2
27.7 ng/µL
1.93
1.67
No sequencing
3
41.8 ng/µL
1.87
1.69
Bad
4
31.1 ng/µL
1.82
1.26
Bad
Fur Binding Site n°3
(with sfGFP)
1
37.8 ng/µL
1.85
1.40
Bad
2
78.3 ng/µL
1.87
1.85
Bad
3
53.4 ng/µL
1.72
0.98
Bad
4
23.3 ng/µL
2.06
2.53
No sequencing
Fur Binding Site n°3
(with LacI-LVA)
2
43.8 ng/µL
1.78
1.21
Bad
4
28.2 ng/µL
2.15
2.31
-
3rd construction
Sequencing preparation
Preculture with LB medium (with carbenicillin) have been launched for Top10 transformed with our plasmid which contains the sfGFP under the control of Lac O promoter. After one night of culture minipreps of 3 different clones have been realized using the miniprep kit from Machrey Nagel.
Name
Clone
Concentration
260/280
260/230
Sequencing
Enterobactin construction
(Lac O with EntA, EntD, EntF)
1
62.7 ng/µL
2.02
0.84
-
2
86.8 ng/µL
1.97
0.99
-
3
112.2 ng/µL
1.98
1.23
-
4
78.7 ng/µL
1.97
1.17
-
Enterobactin construction
(Lac O with EntB, EntC, EntE)
1
80.1 ng/µL
1.92
1.01
-
4
330.3 ng/µL
1.91
1.20
-
Other
We prepared 20 tubes of BL21 competent cells.
Week 11: 26th August - 1st September
Creation of a E. Coli ΔFur
Strain preparation
Chemically competent Top 10 E. Coli was transformed with the plasmid PTK which contained λ Red factor. We plated our bacteria on LB Agar with spectinomycin and let them grow overnight at 30°C.
One colony had grown on our plate. We started an overnight culture of this colony in 2 mL of LB with spectinomycin and still at 30°C.
We then diluted 1 mL of our culture in 100 mL of LB with spectinomycin and IPTG so the λ Red factor would start to be expressed. We let bacteria grew to a OD of 0.5 and made them electro competent. The protocol used can be found in our Protocol pages.
Homologous Recombination
We realised a PCR on a plasmid with a kanamycin cassette resistance and with our two designed primers P119 and P120. The PCR product had been then purified and migrated on 1% agarose gel. We obtained the 1 kb band expected which confirmed that we got the right amplificat.
We transformed by electroporation 50 uL of our strain with 100 ng of PCR product. We had a time constant of 5,6. After a 2 hours recovery in 2 mL of LB with IPTG, we added kanamycin and let our bacteria grow overnight at 30°C.
Next day, our culture was very turbid. Our bacteria got a kanamycin resistance. We plated 50 uL of our culture on four kanamycin-agar petri dishes and grew overnight at 30°C.
Week 12: 2nd September - 8th September
Creation of a E. Coli ΔFur
02/09/13 04/09/13 We picked colonies from our four plates, that we grew overnight at 42°C, to plate each of them on one plate with kanamycin
TECAN
02/09/13 To further improve the interpretability of the plate reading, we need to transfer our construction from a BL21 to a TOP10 strain. As a start, we transformed the following constructions:
- AceB with LacI
- FecA with GFP
- FepA with GFP
- FepA with LacI
- Fes with GFP
- Fes with LacI
- YbiL with GFP
- YbiL with LacI
- YncE with GFP
- YncE with LacI
- control positive with pSB1A3
- control negative
We did not have anymore AceB with GFP plasmid available to realize the transformation. We pre-cultured a remaining glycerol stock at 37°C overnight:
10 mL LB
10 µL Carbenicillin 1000X
50 µL cells from glycerol stock
03/09/13
The transformations succeeded and made liquid cultures for glycerol conservation. Also, we isolated the plasmid AceB-GFP from the pre-culture of the 03/09/13. As a consequence, we transformed our last plasmid in a TOP10 strain.
04/09/13
To anticipate the TECAN for the evening, we started a pre-culture of the following constructions we want to characterize:
TOP10 (growth control)
TOP10 PL-LacO (GFP control)
TOP10 YbiL-GFP
TOP10 YncE-GFP
Also, we prepared the iron solutions from 10^-3 to 10^-10 M that will be used to prepare all the different media and completed the plate. TECAN GRID
05/09/13
To anticipate the TECAN for the evening, we started a pre-culture of the following constructions we want to characterize:
TOP10 (growth control)
TOP10 PL-LacO (GFP control)
TOP10 AceB-GFP
TOP10 Fes-GFP
TECAN GRID
Plasmid 3 (chelator system)
We launch another PC to get Enterobactins A, B, C, D, E and F genes. Add the recquired primers in each tube:
EntA: primers 065 and 066
EntB: primers 067 and 068
EntC: primers 069 and 070
EntD: primers 071 and 072
EntE: primers 073 and 074
EntF: primers 075 and 076
Positive controle: primers 009 and 010
Negative controle: primers 009 and 010
We use pEntC as our controle.
For more details about our primers, see the corresponding page .
We then prepared a master mix 1 with:
9 x 27,5 = 247,5 µL of distilled water
9 x 1 = 9 µL of dNTPs
9 x 10 = 90 µL of Q
5
Buffer
9 x 0,5 = 4,5 µL of Q
5
For the negative controle, we added 1 µL of distilled water in tube 8 and 39 µL of master mix 1.
For other tubes (6 genes and positive controle), we added 8 µL of genomic DNA in the master mix 1. Then add 40 µL of master mix 1' in each tube.
Week 13: 9th September - 15th September
Design of the capsule
09/09/13
To chelate the iron in the duodenum and the initial portion of the jejunum, it was pretty obvious that a capsule or a tablet was required. The best galenic formulation for our purposes is the capsule, meaning a delivery of our bacteria in the jejunum. The first reason is that a tablet requires not only a heavy and dry compression, but also the second reason is that it requires lyophilized bacteria, which consists of extreme conditions (from very high to very low temperature and high pressure). As a consequence, the capsule offers the possibility to be easily coated for resistance against stomachal acidity and presents a softer environment for the bacteria to transport.
The first step in the design of the capsule is to choose the right diluent, meaning the appropriate powder to 'dissolve' the bacteria in. We experimented both maltodextrin and colloidal silica and tried to disperse as much LB medium as possible with hand mortar and pestle (figure 1 and 2). For 10g of maltodextrin, we managed to dissolve 400 µL of LB medium, and for 3g of colloidal silica 4 mL of LB medium. We choose colloidal silica for   - Réflexion le choix des bio-adhésifs: ici on utilisera les HPMC.
- Réflexion sur la qualité des bio-adhésif, notamment dans la relation viscosité/bio-adhésif. En effet, les HPMC vont gonfler en contact avec l'eau, adhérant ainsi les bactéries à la paroi. Il faut que la viscosité soit minimale pour pas que le bol alimentaire vienne arracher la fixation de nos bactéries, mais qu'en même temps l'eau ait une pénétrance suffisante.
- Réflexion sur l'enrobage (enteric coating). Il faut que la gélule se délite le plus haut possible dans l'intestin (ici le duodénum), mais qu'il soit suffisamment résistant à l'acidité de l'estomac en amont.
Choice of excipients (reference: Handbook of Pharmaceutical Excipients, sixth edition): Hypromellose (Hydroxypropyl methylcellulose, HPMC): is used as a bioadhesive material for a controlled release at pH 5-8 (duodenum and jejunum).
Methacrylic acid L100-55 (Eudragit): is used as an enteric coating for resistance against stomachal pH, thus allowing the release of the bacteria in the duodenum at pH values of 5,5.
Colloidal silicon dioxide: is used to dry pellet of bacteria and allow an homogenous mixture with the other components.
TOP10 and TOP10 transformed with pSB1A3 stayed the whole weekend at 4°C. I resuspended the cell in 10 mL LB + antibiotic.
10/09/13
I tested two different powders to dilute our medium with bacteria. As a start, i began to dilute LB medium, without bacteria in Maltodextrin (Glucidex) or Colloidal silica (Aerosil). After dispersing the liquid in the powder, the goal was to put as much as possible in it, but the powder should not agglomerate and should keep its powder form. As such, I managed to dilute 200 µL LB in 10 g of Maltodextrin but 3825 µL in 3 g of colloidal silica. As a consequence, we chose for colloidal silica to dilute the LB, and thus, the bacteria.
The second step is to obtain a final volume of 35g of powder. After I diluted 3825 µL of LB in colloidal silica, I obtained a total volume of 16 mL. I added a sufficient quantity of HPMC (Hydroxyprpylmethylcellulose, Methocel K100) to obtain a final volume of 35 mL. The two components are mixed together and I made sure the product is dry.
With a 'gelulier', I equally dispersed the whole volume in 50 capsules. After weight measurement, the capsules are 342 +/- 4 mg.
To anticipate the dispersion tests at pH = 2 and at pH = 6,5, I prepared a PBS buffer and an acid medium that mimics the gastric context and its acidty.
The capsule should not be dispersed after two hours in the medium of pH = 2. Also, the capsule, afterwards, should disperse in between 1 hour in PBS buffer. If the capsule satisfies these conditions, the bacteria will be releases right at the end of the duodenum, thus allowing the growth in the jejunum.
PBS preparation for a volume of 5L:
NaCl: 40g
KCl: 1g
Na2HPO4: 7,2g
KH2PO4: 1,2g
The first attempt to make capsules from LB is repeated, but this time with LB and bacteria. After I diluted 3000 µL of LB and bactera in colloidal silica, I obtained a total volume of 20mL. I added a sufficient quantity of HPMC (Hydroxyprpylmethylcellulose, Methocel K100) to obtain a final volume of 35 mL. The two components were mixed together. However, since the bacteria are dry in the powder, they may be very volatile when mixed to the colloidal silica. As an additional precaution, I wore a mask to prevent the inhale of these potenially contaminated particles.
I refreshed my cultures in 3ml LB (x2) + an additional control negative (only 3ml LB) to make sure that I am not in fact working with the wrong bacteria or any contamination. This control negative was actually very important. In this lab, I don't have the same sterilized pipet tips or 15ml tube. Also, the only sterile environnement is in the presence of a very short flame.
At the end of the day, I managed to create 50 capsules containing only dry LB and 50 capsules with dry LB and bacteria. Additionnaly, I prepared the alcoholic solution of 12,5% Eudragit L100-55 to create the gastro-enteric resistant coating around my capsule. The receipe is as follows: 50ml of pure ethanol (98%) 6,25g of eudragit Contrary to the litterature, the Eudragit was not that 'extremely' soluble in pure ethanol and did not take 20mn to dissolve. As a consequence, I sonicated for 45 minutes to solubilize the Eudragit.
Finally, I tried to make the coating on some capsules. I came to the conclusion that the drying lasts a very long time. To accelerate the drying process, I used the hairdryer for 30 seconds at maximum temperature at average speed, then 1mn at lowest temperature at highest speed. This way i managed to dry the fastest way the coating of my capsules, without being to sticky at the end. Also, every single capsule had to be done twice, once on the upper side and another on the downside. The not only allows a better dring, but also double-covers the edge of the capsule where both sides close (in the middle) and gives a double protection at the weakest point of the capsule. Finally, keep in mind that the coating process is doubled, thus spending on average 8 to 10 minutes for each capsule.
11/09/13
I double-coated 6 capsules containing the dry bacteria and put them for 2 hours in the acid to test their resistance to low pH (pH = 2). They did not break during this stage.
After 20 minutes of shacking in the PBS buffer, the water starts to become turbid, thus proving the degradation of the gelatin of the capsid and thus releasing the HPMC. I extracted only one sample in liquid culture to see if the bacteria survived both the tests. Two otherw were kept as control negative (in fact, the main bottle of medium seems to be turbid). We'll now tomorrow.
12/09/13
"Fresh bacteria test"
Pills conception
enrobage
pH test
test  au pbs
After the dissolution in acid medium, the capsule were still complete. Then, they were transfered in the PBS medium where they have been shaking for 1h. A sample (+photo) has been taken at 20, 30, 40, 50 and 60 minutes to follow the progress of dissolution of the capsules. We can easily observe, due to the yellow coloring of the capsule, that they dissolve quickly. The samples at each moment are put in 8ml LB medium + streptomycin. Tomorrow we'll knwo if the bacteria survived the manufacture of the capsule and the dissolving for 3 hours. One blank with only 8ml medium + 8 µL Streptomycin is kept to avoid any fasle interpretation of potential growth.
Plasmid 3 (chelator system)
09/09/13
We made miniprep of the culture we had prepared the previous week.
Name
Clone
Concentration
(ng/µL)
260/280
260/230
Digestion
Sequencing
Plasmid 3 (chelator system) (with EntA,D and F)
1
127.1
1.88
1.94
2
92
1.87
1.67
-
3
114.2
1.88
1.78
4
137.1
1.86
1.66
5
139.2
1.88
1.88
6
76.9
1.89
1.69
7
65.9
1.95
2.34
8
49.0
1.88
1.60
9
35.5
1.87
1.60
10
29.1
1.98
1.32
Plasmid 3 (chelator system)
(with EntB,C and E)
1
107.9
1.87
1.83
2
45.7
1.88
1.95
-
3
78.6
1.85
1.68
4
95.4
1.85
1.72
5
134.9
1.84
1.82
6
108.9
1.81
1.55
7
75
1.84
1.55
8
133.0
1.82
1.80
9
101.2
1.82
1.63
10
106.4
1.85
1.75
To determin which clones have our constructions, we make a digestion with PstI, EcoRI and BsaBI; and then an electrophoresis with the digestion products.
We compared our gel with the theroical profil made on Geneious.
Biobricking
In order to send our constructions to the Registry we have to change their prefix and subclone them into a chloramphenicol backbone pSB1C3.
Modification of the sequence using PCR
PCR is performed using the high fidelity Q5 DNA polymerase.
Mix preparation:
Component
50 µl Reaction
Volume for 13 reactions
Q5 buffer (5X)
10 µL
130 µL
dNTPs (10 mM)
1 µL
13 µL
Forward primer (5 µM)
5 µL
-
Reverse primer (5 µM)
5 µL
-
DNA template
1 µL
-
Q5 DNA polymerase
0.5 µL
6.5 µL
H2O (qsp 50 µL)
27.5 µL
357.5 µL
Primers used were different for each construction.
Construction
Forward Primer
Reverse Primer
AceB promoter + (sfGFP/LacI-LVA)
P87
P86
FecA promoter + (sfGFP/LacI-LVA)
P88
P86
FepA promoter + (sfGFP/LacI-LVA)
P89
P86
Fes promoter + (sfGFP/LacI-LVA)
P90
P86
ybiL promoter + (sfGFP/LacI-LVA)
P91
P86
AceB promoter + (sfGFP/LacI-LVA)
P92
P86
After the PCR fragment are verrified by gel migration and each PCR product is purified using PCR clean up kit from Machrey Nagel.
Construction
Concentration
(ng/µL)
260/280
260/230
AceB promoter + sfGFP
131.0
1.79
0.97
AceB promoter + LacI-LVA
74.5
1.64
1.05
FecA promoter + sfGFP
342.0
1.56
0.69
FepA promoter + sfGFP
237.8
1.60
0.78
FepA promoter + LacI-LVA
268.7
1.57
0.78
Fes promoter + sfGFP
82.5
1.91
1.07
Fes promoter + LacI-LVA
338.6
1.54
0.82
ybiL promoter + sfGFP
54.5
1.93
1.08
ybiL promoter + LacI-LVA
247.9
1.65
0.62
yncE promoter + sfGFP
124.2
1.80
1.23
yncE promoter + LacI-LVA
210.3
1.62
0.78
pSB1C3 plasmid Minipreping
In order to subclone our construction into the plasmid pSB1C3, we must miniprep it. Then, we launched a 20 mL overnight culture of E. coli transformed with pSB1C3 and we purified it using a Machrey Nagel miniprep purification kit.
Plasmid
Replicate
Concentration
(ng/µL)
260/280
260/230
pSB1C3
n°1
87.1
1.86
1.55
n°2
91.8
1.84
1.38
n°3
77.9
1.81
1.21
n°4
98.9
1.74
1.00
n°5
78.4
1.82
1.33
n°6
91.8
1.82
1.31
Digestion using EcoRI and PstI
Our construction are subcloned into pSB1C3 using the restriction enzymes EcoRI and PstI (FastDigest from Fermentas)
Mix preparation for 1 µg of DNA (15 µL reaction - 2.5 µL/tube):
Component
15 µl Reaction
Volume for 13 reactions
Fast Digest Buffer
1.5 µL
18 µL
EcoRI
0.5 µL
6 µL
PstI
0.5 µL
6 µL
Construction
15 µl Reaction
H2O (qsp 15 µL)
AceB promoter + sfGFP
7.6 µL
4.9 µL
AceB promoter + LacI-LVA
12.5 µL
-
FecA promoter + sfGFP
4 µL
8.5 µL
FepA promoter + sfGFP
5 µL
7.5 µL
FepA promoter + LacI-LVA
4 µL
8.5 µL
Fes promoter + sfGFP
12.5 µL
-
Fes promoter + LacI-LVA
4 µL
8.5 µL
ybiL promoter + sfGFP
12.5 µL
-
ybiL promoter + LacI-LVA
5 µL
7.5 µL
yncE promoter + sfGFP
8 µL
4.5 µL
yncE promoter + LacI-LVA
5 µL
7.5 µL
We prepared a special mix for the plasmid because 5 µg have been digested.
Component
Volume needed for the reaction
pSB1C3 (85 ng/µL)
55 µL
Buffer (10X)
6.5 µL
EcoRI
2.5 µL
PstI
2.5 µL
Tube are placed 1h at 37°C. Products are purified using PCR clean up kit from Machrey Nagel.
Ligation
To perform the ligation, we use 100 ng of vector for 200 ng of insert.
Volume of reaction = 15 µL
T4 buffer = 1.5 µL
PEG 4000 = 1 µL
T4 ligase = 0.5 µL
pSB1C3 = 1 µL
Week 14: 16th September - 22nd September
Plasmid 1 (natural sensor system)
Name
Clone
Concentration
(ng/µL)
260/280
260/230
AceB with GFP
1
63.0
1.83
1.23
2
84.3
1.95
1.50
3
60.8
1.73
1.42
AceB with LacI
1
78.3
1.75
1.05
2
61.6
1.84
1.36
3
61.1
1.83
1.27
Fes
with GFP
1
51.8
1.88
1.47
2
55.1
1.81
1.07
3
49.2
1.71
0.97
Fes with LacI
1
67.5
1.75
1.16
2
137.7
1.82
1.38
3
147.2
1.89
1.80
Fep A with GFP
1
74.4
1.60
0.67
2
76.8
1.56
0.61
3
97.6
1.59
0.68
FepA with LacI
1
17.0
1.72
1.01
2
76.8
1.63
0.75
3
28.9
1.70
0.70
yncE
with GFP
1
86.6
1.63
0.81
2
72.1
1.62
0.77
3
70.8
1.66
0.83
yncE with LacI
1
114.1
1.61
0.73
2
109.6
1.56
0.60
3
100.8
1.56
0.59
Week 15: 23rd September - 29th September
We transformed our pAceB-LacI construction with an other plasmid containing PL-lacO-RFP. Then the system has been caracterized using TECAN analysis. See our results
Microfluidic experiment
We prepare the microfluidic ship. (See microfluidic protocol)
Week 16: 30th September - 6th October
Microfluidic experiment
30/09/13
Chip's channel were full of bacteria, even after 2 hours of washing (with free iron M9) because we let the bacteria growth over the week-end (in LB medium). Hence, we prepared another chip to launch another experiment on the next morning.
As our bacteria have a plasmid AceB-sfGFP and there is no iron in our M9, we still check if bacteria expressed sfGFP protein.
The first results appear promising, but we need to confirm them with a proper experiment.
Figure 1: Escherichia coli (TOP 10 strain) expressing sfGFP.
At the top of the picture, we can see that the channel is full of bacteria.
4th October
Week 17: 7th October - 13th October
Week 18: 14th October - 20th October
Week 19: 21st October - 27th October
Monday, 21th October
Pre-cultures of :
AceB-GFP + pLacO-RFP version 1 inverter
AceB-LacI + pLacO-RFP version 1 inverter
AceB-GFP + pLacO-RFP version 2 inverter
AceB-LacI + pLacO-RFP version 2 inverter
AceB-LacI
AceB-GFP
pGFP (high expression)
Tuesday, 22th October
In the morning, we ran a TECAN to characterize AceB again, but this time with a strong promoter expressing GFP as a control.
Additionnally, we digested both our inverters to figure out if they have the right plasmids in it. The v1 seem to have the right one, the v2 is wrong. So we further need to pick new colonies and digest again until we find a good strain.
PCR with the ordered primers :
pLac for Crossing-Over PCR (pSB1A3 template)
FbpA for Crossing-Over PCR (genomic Neisseria gonorrheoae)
pLac for further golden gate assembly (pSB1A3 template)
FbpA for further golden gate assembly (genomic Neisseria gonorrheoae)
EntA for further golden gate assembly (genomic E. coli)
EntC for further golden gate assembly (genomic E. coli)
AceB adjustment
gBlock adjustment
EntA for cloning in pSB1C3 (genomic E. coli)
EntB for cloning in pSB1C3 (genomic E. coli)
EntC for cloning in pSB1C3 (genomic E. coli)
EntD for cloning in pSB1C3 (genomic E. coli)
EntE for cloning in pSB1C3 (genomic E. coli)
EntF for cloning in pSB1C3 (genomic E. coli)
PCR purification.
Wednesday, 23th October
Gel on purified PCR products.
Sizes are as follows:
pLac for Crossing-Over PCR : 250 nt
FbpA for Crossing-Over PCR :  nt
pLac for further golden gate assembly :  nt
FbpA for further golden gate assembly :  nt
EntA for further golden gate assembly :  nt
EntC for further golden gate assembly :  nt
AceB adjustment  : 340 nt
gBlock adjustment :  nt
EntA for cloning in pSB1C3 : 790 nt
EntB for cloning in pSB1C3 : 900 nt
EntC for cloning in pSB1C3 : 1200 nt
EntD for cloning in pSB1C3 : 670 nt
EntE for cloning in pSB1C3 : 1650 nt
EntF for cloning in pSB1C3 : 3920 nt
The following PCR's worked :
pLac for Crossing-Over PCR : 250 nt
FbpA for further golden gate assembly : 1000 nt
EntA for further golden gate assembly : 800 nt
EntC for further golden gate assembly : 1200 nt
AceB adjustment  : 340 nt
gBlock adjustment :  nt
We adjusted the protocol of the 8 remaining genes to extract :
We did this by using both Q5 and Herculase at two different annealing temperatures: 50 and 55°C. As a consequence, we have 4 different conditions.
Thursday, 24th October
PCR purification, Nanodrop and calculation for the proportions of each element required for Golden Gate.
The following ligations by Golden Gate have been done:
1C3 + pLacO + FbpA
1C3 + pLacO + EntA
1C3 + pLacO + EntC
1C3 + AceB + sfGFP + Term
All of the previous constructions are transformed in E. coli MG1655. The presence of LacI in this strain will inhibit the potential the expression of the downstream gene, thus preventing a potential toxicity.
Additionnally, we did the last remaining PCR step for the cross-over PCR and migrated on gel. According to both our control, the band was indeed the size of 1300 bp, so it worked!
Friday, 25th October
PCR purification of the cross-over PCR product and pSB1C3, followed by a digestion by EcorI and PstI and then ligation.
MG1655 have been transformed by pLacO-FbpA (A)
Saturday, 26th October
Transformation of the rest of the strategy by Golden Gate ligation
Sunday, 27th October
Pre-cultures of :
6 pLacO-Fbpa (coPCR A) (Cm)
1 control pSB1C3 (Cm)
5 pLacO-EntA (GG C) (K)
5 pLacO-EntC (GG C) (K)
1 pLaco-FbpA (GG C) (K)
5 AceB-LacI (GG C) (K)
1 control pSB1K3 (K)
Week 20: 28th October - 3rd November
Monday, 28th October
Miniprep, digestion and migration of all the transformation to check for the right construction: all of them failed.
Contact: igemevry@gmail.com
Site developped by Gabriel Guillocheau
Retrieved from " http://2013.igem.org/Team:Evry/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
