<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Metabolism_model"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Metabolism_model">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Metabolism_model&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Metabolism_model&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Metabolism_model&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Metabolism_model" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Metabolism model</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><P><ASIDE><HEADER>
  Model
 </HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/Modeling">Overview</A></LI></UL><UL><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modeltr1">Disease model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modeltr2">Treatment model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin Production:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta1">Sensor model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta2">Inverter model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta3">Enterobactin model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/interactions">Interaction Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/supply">Supply Model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI></UL><HEADER>Tools</HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic Functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI></UL></ASIDE></P><DIV id="maincontainer_evry2013"><DIV id="mainTextcontainer"><H1>Flux model</H1><H2>Introduction</H2><P>
In this part of our modeling work we focus on genome scale analysis of the enterobactin production pathway. A major concern about our system is its non-orthogonality with the natural metabolic network of the cells : <I>E. coli</I> already possesses the genes for producing enterobactins. Therefore we wanted to assess the possible interactions between our system and the bacterial metabolism.
</P><H2>Observations</H2><P>
This model stems from the following observations :

<UL><LI>Enterobactin production pathway is a metabolic process;</LI><LI>Any of the involved metabolites may limit the rate of the reactions.</LI></UL>

The <A href="#Fig1">Figure 1</A> presents the biochemical pathway for producing the enterobactin compound in <I>E. coli</I>.
</P><DIV class="captionedPicture" style="float:left;"><DIV class="caption"><B>Figure 1:</B><P>The ENTEROBACTIN biosynthesis pathway. Arrows : reactions, circles : enzymes.</P></DIV></DIV><P>
As can be seen in the <A href="#Fig1">Figure 1</A> the pathway is 4 steps long and composed of six different enzymes. Hence there exists 4 possible metabolites which concentration may be limiting :

<UL><LI>Chorismate</LI><LI>Isochorismate</LI><LI>2,3-dihydroxy-2,3-dihydrobenzoate</LI><LI>2,3-dihydroxybenzoat</LI></UL></P><H2>Goals</H2><P>
We highlighted two main kind of interactions between the bacteria and our system :

<OL><LI>The synthetic system interacts with the bacterial metabolism. Leading to a depletion of the metabolites involved in the pathway for the other (possibly essential) metabolic reactions of the cell.</LI><LI>The other way round, the metabolic reactions could prevent our synthetic system to work as expected by limiting the quantity of the involved metabolites available.</LI></OL>

From these assumptions we formulated the following questions :

<UL><LI><EM>Is the metabolic model of <I>E. coli</I> able to provide any information about the possible interactions between our system and the metabolism?</EM></LI><LI><EM>Is the concentration of any metabolite limiting ?</EM></LI><LI><EM>In the latter case, what is the quantity of precursor that should be added in the medium?</EM></LI></UL></P><H2>Materials and Methods</H2><H3>Model</H3><P>
We used the metabolic model <I>E. coli</I> iJR904 downloaded from the <A href="http://bigg.ucsd.edu/bigg/home.pl">BiGG model database</A><A href="#ref1">[1]</A>. We chose this model because it contains all the metabolites involved in the enterobactin production pathway.
</P><P>
This model contains 4037 reactions and 625 metabolites but lacks the <A href="http://www.genome.jp/dbget-bin/www_bget?ec:6.3.2.14">enterobactin synthase</A> (the last reaction of the enterobactin production pathway). We thus extended the model by adding three new reactions :
</P><CENTER><TABLE><TBODY><TR><TH>Name</TH><TH>Formula</TH></TR><TR><TD>ENTSYNTH</TD><TD>6 ATP + 3 2,3-DIHYDROXYBENZOATE + 3 L-SERINE =&gt; ENTEROBACTIN + 6 AMP + 6 DIPHOSPHATE</TD></TR><TR><TD>ENTOUT</TD><TD>  =&gt; -1 ENTEROBACTIN</TD></TR><TR><TD>CHORIN</TD><TD>  =&gt; +1 CHORISMATE</TD></TR></TBODY></TABLE></CENTER><P>

The last two reactions are respectively output flux of ENTOUT and intput flux of CHORIN that are artificial reactions used to measure and interact with our system.

The modified <I>E. coli</I> iJR904 model containing these two new reactions can be found in the download section at the bottom of the page.
</P><H3>Network Reduction</H3><P>
The ENTEROBACTIN production pathway in kegg starts with the CHORISMATE (or CHORISMIC ACID) compound. Hence, we restricted our study to this specific sub-network, presented in <A href="#Fig2">Figure 2</A> of the whole metabolic network. We did not directly investigated the previous reactions leading to CHORISMATE production. They are taken into accound in the whole metabolic network model but we did not try to act on these. It is also worth noting that the CHORISMATE compound can be bought from any chemical compound supplier (for example <A href="http://www.sigmaaldrich.com/catalog/product/aldrich/c1761?lang=fr&amp;region=FR">sigma</A>).
</P><P>
We also found that  it would also be possible to by the 2-3-DIHYDROXYBENZOATE compound from the <A href="http://www.sigmaaldrich.com/catalog/product/aldrich/126209?lang=fr&amp;region=FR">same provider</A>. This compound may be very interesting to test our constructions later as the last precursor of the pathway. For the same reason, and because it is not consumed by any other metabolic reaction present in the current model.
</P><CENTER><DIV class="captionedPicture" style="float:left;"><DIV class="caption"><B>Figure 2:</B><P>The considered subnetwork of metabolic reactions of <I>E. coli</I>. Red arrows are reaction consuming compounds used in the ENTEROBACTIN production pathway. Arrows : reactions, circles : enzymes.</P></DIV></DIV></CENTER><H2>Flux Balance Analysis</H2><P>
The metabolic model of <I>E. Coli</I> is based on a stoichiometry matrix S representing the metabolic network. Our modified version has the size 4039 (reactions) * 625 (compounds). The unknown is the flux distribution  vector v, a column-vector of size 4039 representing the flow of matter (mmol/gDW/h) going through each reaction.
</P><P>
The Flux Balance Analysis method<A href="#ref2">[2]</A> is about finding this flux repartition vector v given an objective function to optimize (usually the growth rate) and a set of constraints on the fluxes values.
</P><H3>Assumptions of the model</H3><P>
The two assumptions at the heart of the method are the following :

<OL><LI><EM>steady state</EM>: The fluxes are considered to have attained a static equilibrium value and do not change through time.</LI><LI><EM>No enzyme saturation</EM>: The enzymes are supposed to be not saturated, the number of enzymes is always greater than the number of the corresponding reactions happening.</LI></OL></P><H3>Formalism</H3><P>
The FBA method uses a representation of the metabolic reaction network in the form of a stoichiometry matrix S where :

<UL><LI>Each row corresponds to a reaction R_i</LI><LI>Each column corresponds to a metabolite C_j</LI></UL>

The definition of S is :
</P><P>
The FBA problem is then formulated as a maximisation problem under some constraints:
</P><P>
where :

<UL><LI>v is the vector of unknown reaction fluxes</LI><LI>c is a vector of constants defining the objective function</LI><LI>S is the stoichiometry matrix</LI><LI>lowerbound and upperbound are vector of constraints (minimal and maximal flux values for each reactions)</LI></UL>

The values for the lower and upper bounds on the flux of each reactions are either deduced from experiments or put to a very high value when unknown (most of the time):
</P><P>
Tunning these boundaries allows to represent different experimental conditions, for example reducing an upper bound to a low value may represent a loss of reaction flux due to the scarcity of a certain compound.
</P><P>
Such optimization is then realized through a linear programming algorithm finding an approximated solution of the real optimal flux distribution (the distribution maximizing the objective function).
</P><H3>Software used</H3><P>
To perform the simulations we decided to use the <A href="http://opencobra.sourceforge.net/openCOBRA/Welcome.html">cobrapy software</A><A href="#ref3">[3]</A> in conjunction with the <A href="http://www.gnu.org/software/glpk/">GLPK</A> (GNU Linear Progamming Kit) linear programming solver.

</P><P>
All the simulations and model modifications were made through python scripts, available at the end of the document.
</P><H2>Results</H2>

With this formalism we were able to answer to the questions presented at the begining of the page. The results are presented on the two following pages:

<OL><LI>Metabolic Interactions on <A href="https://2013.igem.org/Team:Evry/interactions">this</A> page.</LI><LI>The effects of supplying chorismate on <A href="https://2013.igem.org/Team:Evry/supply">this</A> page.</LI></OL><H2>Conclusion</H2><P>
The metabolic models presented in this sections have been used to answer the general questions we had about our system.
</P><P>
First, we wanted to know if adding new synthetic constructs would perturb the metabolism of <I>E. coli</I>. This was answered by a first model plotting a modified objective function optimizing both ENTEROBACTIN production and growth rate. The model showed that the <EM>enterobactin pathway perturbs the CHORISMATE MUTASE and DEOXYCHORISMATE SYNTHASE reactions</EM> that belong to <I>E. coli</I> central metabolism.
</P><P>
In two other models, we showed that the <EM>CHORISMATE MUTASE AND DEOXYCHORISMATE SYNTHASE were very crucial for the bacteria</EM>, as a reduction of their maximal upper flux lead to a direct (linear) decrease of the growth rate function. No flux going through any of these reactions result in no predicted growth of the bacteria.
</P><P>
Finally, we analysed how we could find a workaround to this problem by <EM>adding some CHORISMATE to the medium</EM>. This situation was presented in a third model with a CHORISMATE input flux that we plotted against the modified objective function. This model clearly showed that the ENTEROBACTIN flux attained a maximal stable region for any value of the objective function starting from <EM>17 mmol/gDW/h of CHORISMATE</EM>. Finally, the dynamic of the ENTOUT flux is being smoother as more CHORISMATE is added.
</P><P>
In the end, this FBA model shows that it is possible to counter-balance the effects of our synthetic construction on the metabolism of <I>E. coli</I> by adding some CHORISMATE to the medium.

On the modeling point of view, these results are in agreement with one of the central assumption made in the other model: no metabolites of the pathway is limiting.  
</P><H2>Models and Scripts</H2><H3>Metabolic Models</H3><P>
The two metabolic models used are :

<OL><LI>Wild-Type <I>E. coli</I> iJR904</LI><LI>ENTOUT(enterobactin outflux) + ENTSYNTH(enterobactin synthase) <I>E. coli</I> iJR904</LI></OL>
These models can be found in <A href="https://static.igem.org/mediawiki/2013/b/bc/Evry_ecoli_iJR904_models.zip">this tarball</A>.
</P><H3>Scripts</H3><P>
The scripts used to generates the curves on this page can be found on <A href="https://static.igem.org/mediawiki/2013/f/f6/Evry_ecoli_fba_models.zip">this archive</A>. They are Python scripts with distinct names corresponding to the simulation they represent.
</P><DIV id="citation_box"><P id="references"><B>References:</B></P><OL><LI>Schellenberger, J., Park, J. O., Conrad, T. C., and Palsson, B. Ø., &quot;BiGG: a Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions&quot;, BMC Bioinformatics, 11:213, (2010).</LI><LI>Orth, Jeffrey D., Ines Thiele, and Bernhard Ø. Palsson. &quot;What is flux balance analysis?.&quot; Nature biotechnology 28.3 (2010): 245-248.</LI><LI>Ebrahim A, Lerman JA, Palsson BO, Hyduke DR. 2013  COBRApy: COnstraints-Based Reconstruction and Analysis for Python. BMC Syst Bio 7:74.</LI></OL></DIV></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/Metabolism_model">http://2013.igem.org/Team:Evry/Metabolism_model</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Metabolism_model" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Metabolism_model" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Metabolism_model&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Metabolism_model&amp;oldid=350861" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>