"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Modelmeta1
From 2013.igem.org
Iron coli project
Home
Project
Iron metabolism
Related diseases
Iron chelators
Achievements
References
Team
iGEM Profile
Attributions
Collaborations
Acknowledgement
Press
Photo gallery
Model
Flush treatment
Enterobactin production
Genome scale
Population scale
Logistic functions
Chemical reasoning
Plate Reader
Biology
Parts
Sensor
Inverter
Chelator
Results
Human Practice
Consultation
Survey
Seminar
Philosophy
Safety
Economy
Art
Capsule
Notebook
Protocols
Primers
Sponsors
Model
Overview
Flush treatment:
Disease model
Treatment model
Enterobactin Production:
Sensor model
Inverter model
Enterobactin model
Genome scale:
Interaction Model
Supply Model
Population scale
Tools
Logistic Functions
Chemical reasoning
Plate Reader
Sensor Model
Introduction
In order to determine our Iron Coli's enterobactin production rate, we first have to know how much time our bacteria will take to sense the ambient iron. So, this first part of the Enterobactin production model focuses on the synthetic sensing system our team implemented in the bacteria.
Observations
Naturally, when there is a large amount of iron in the environment, 2 iron ions complex with 2 FUR proteins and bind to the FUR Binding Site (FBS). This interaction regulate negativelly the genes downstream.
As shown on the Figure 1 , our sensing system relies on a FBS. In order to easily model the sensing delay and analyse its results, the first construction is composed of a GFP placed right after the FBS.
Figure 1: Our first construction, our Iron sensor system
See our sensor page for more details
Goals
Our goal in this part of the model is to create a generic FBS-related sensing model so that:
We can determine the iron-sensing delay of our bacteria
The model can be reused by other projects that include a FBS sensor
Materials and methods
From Iron to FBS:
The iron-FUR complex is simply formed that way:
 
We reduced this equation to:
Which is not annoying, since we just have to divide our [FeFur] by two to get the real complex concentration.
We can easily write down both the formation (v) and the dissociation (v') speeds:
We chose to model the iron input in the bacteria using a linear function of the external iron concentration Ferext, the factor p being the cell-wall permeability for iron.
The FUR on the other hand is produced by the bacteria. Its evolution can also be considered as linear, using a mean production rate of Fur0.
In this model, we only track the free Fe-FUR complex and not those attached to a FUR Binding Site. FBS is the number of inhibited Fur Binding Sites.
GFP expression:
To simulate the inhibition phenomenon, we chose to use our Logistic function under its differential form. Since it is the Fe-FUR that represses it, the LacI can be expressed as a logistic fuction of the Fe-FUR:
Ki1 is a non-dimensional parameter that repesents the inhibition power, and Kf is the fixation rate of the Fe-FUR on the FBS. Finally, Npla1 is the number of pasmids containing the GFP.
GFP Production:
The [mRNA] and [GFP] equations are alike. The prodction rates are Kr for the mRNA and Kp for the GFP. Since FBS represents the number of inhibited Fur Binding Sites, we have to substract it from Npla1.
Both variables also have a negative degadation term:
Tuning the model
The only parameter we couldn't find in the litterature is Ki1. Nevertheless, thanks to our construction and the experimental results we obtained with a plate reader, we can tune this parameter to obtain a realistic sensing model!
Figure 2: GFP/OD(t) for different iron concentrations
The Figure 2 is a set of experimental results showing the sensing efficiency for different iron concentrations. We can notice a substantial behaviour change between 10-4M and 10-5M of iron : the activity of our bacteria leaps at 10-5M.
Therefore, we ran simulations with different Ki1 values to try to determine the most fitting Ki1.
Figure 3: Maximum GFP for different Ki1 values
The Figure 3 shows that our model can lead to 3 different leaps, depending on the Ki1 value. The leap we are interested in is the 10-4M  10-5M leap.
We thus zoomed in, and performed the simulations with a more adapted range of Ki1.
Figure 4: Maximum GFP for different Ki1 values
The Figure 4 shows a more precise graph. It is with those simulation values that we calculated the most realistic Ki1, by maximizing the leap it creates. We also figured that the best fitting case requires the GFP reaching a plateau right after 10-5M of iron.
Finally, we set Ki1 at 6,3.10-5.
Results
The following results were computed using those parameters:
Name
Value
Unit
Description
p
0.1
min^-1
Permeability of cell wall
KFeFUR
0.01
M-1.s-1
Formation constant of FeFur complex
Dff
0.001
min-1
FeFUR degradation rate
Kp
0.5
min-1
translation rate
NA
6.02*1020
mol-1
Avogadro's constant
V
6.5*10-16
m3
Volume of a bacterium
DmRNA
0.001
min-1
mRNA degradation rate
Kf
10-4
min-1
fixation rate of FeFUR
Fur0
0.01
mM.min-1
Fur Production rate
Ki1
6,3.10-5
-
Sensor efficiency
Figure 5: GFP(t) for different iron concentrations
As shown in the Figure 5 , there is a significant change in GFP production after 1000 seconds between the curve at 10-4M of iron and the one at 10-5M. The bacteria can thus sense low concentrations of iron.
Conclusion
The first construction of our team is fully modeled : we made a generic iron sensing model (with the FBS system).
Thanks to the experimental results we obtained with a plate reader, we were able to tune an unknown parameter to obtain a more realistic model.
This sensing model allowed us to continue the Enterobactin production model, the next step being the Inverter model .
Models and scripts
This model was made using the Python language. You can download the python script here .
Contact: igemevry@gmail.com
Site developped by Gabriel Guillocheau
Retrieved from " http://2013.igem.org/Team:Evry/Modelmeta1 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
