"
Page
Discussion
View source
History
teams
Log in
 
Team:Evry/Project FUR
From 2013.igem.org
Iron coli project
Home
Project
Iron metabolism
Related diseases
Iron chelators
Achievements
References
Team
iGEM Profile
Attributions
Collaborations
Acknowledgement
Press
Photo gallery
Model
Flush treatment
Enterobactin production
Genome scale
Population scale
Logistic functions
Chemical reasoning
Plate Reader
Biology
Parts
Sensor
Inverter
Chelator
Results
Human Practice
Consultation
Survey
Seminar
Philosophy
Safety
Economy
Art
Capsule
Notebook
Protocols
Primers
Sponsors
Project
Overview
Iron metabolism
Related diseases
FUR system
Achievements
References
Iron chelators
FUR system
Iron is an essential element in the development of E. coli, but it can also be toxic and E. coli can be killed, if iron is absorbed in high quantity. Using the ferric-uptake regulator protein (Fur), bacteria developed an advanced system to regulate their iron homeostasis.
FUR protein (Ferric Uptake Regulator)
The Fur protein acts a transcriptional repressor for more than 90 genes involved, in majority, in iron homeostasis1,2. It plays an important role in the control of the intracellular concentration of iron in E. coli.
Fur acts as an active repressor in presence of ferrous ion (Fe2+), its co-repressor. Then, Fe2+ binds to the Fur protein (one ferrous ion per subunit of Fur). This interaction leads to a structural modification and induces the dimerization of Fur and Fe2+. Then the homodimeric Fur-Fe2+ complex binds to the DNA on a Fur binding site and inhibits the mRNA transcription. In absence of Fe2+, a disinhibiting effect occurs and mRNA can be transcribed.
Figure 1: Regulation by the Fur transcriptional factor. In absence of iron mRNA can be transcribed. In presence of iron, Fur binds the DNA, on the Fur binding site and mRNA expression is inhibited.
Fur is found at a level of 5000 copies in E. coli during exponential growth and it can reach up to 10,000 copies in stationary phase. Note that this high number of Fur protein is essential to E. coli during its development. As explained previously, Fur regulates the expression of genes involved in iron absorption, and it avoids an over absorption of iron which could be really toxic and kill E. coli.
FUR binding site architecture
Figure 2: Consensus palindromic sequence of Fur binding site.
The Fur Binding site also named “Fur box” or “iron box” is localized between -35 and -10 site at the promoters of Fur-repressed genes. As shown in the Figure 2, the Fur binding site is composed of 19-base pair that are organized as a palindromic consensus sequence. It must be noted that this consensus sequence is not an exact sequence. Others sequences of Fur binding sites can be found into E. coli’s genome.
Often, Fur binding site is repeated (as shown in the Figures 3 and 4). It means that several homodimer of Fur-Fe2+ can bind among the DNA. Then Fur binding sites can be localized in a larger region than between -35 and -10 of the promoter region (defined previously). The affinity of these Fur binding site to Fur-Fe2+ is different. At first the complex Fur-Fe2+ binds to the 1st Fur binding site, which presents a higher affinity for its complex. Then, others Fur-Fe2+ bind to the following Fur binding sites which present a weaker affinity. A polymer of Fur protein is formed among the DNA which allows a stronger inhibition of the expression of the gene downstream these Fur binding sites.
Figure 3: Repeated Fur binding site sequences.
Figure 4: Repeated Fur binding site sequences on a double strand DNA.
Natural inverter system
It had been shown that Fur could be involved in a positive regulation of several genes of E. coli (fntA, bfr, acnA, fumA, sdh operon, sodB). In fact E. coli uses an inverter system mediated by a small RNA named RhyB.
RhyB is a 90 nucleotids long sRNA which is regulated by the Fur transcriptional factor, then it is expressed at low concentration of iron3.
Figure 5: Regulation by the Fur transcriptional and the RhyB sRNA, acting as an inverter. In absence of iron mRNA is binded by the RhyB and then it is degradated. In presence of iron, Fur binds the DNA, the RhyB expression is inhibited and the mRNA can be transcribed.
As shown in Figure 5, and if we apply it to the sodB gene expression system. In iron starvation, Fur-Fe2+ cannot be formed and RyhB transcription is disinhibited. RyhB is expressed in the intracellular environment and it can binds to sodB mRNA which contains RyhB’s target sequence. Once RyhB binds to SodB mRNA, RNA degrading enzymes, such as RNaseE and RNase III, are recruited and the new formed complex is digested.
However, if the iron concentration is high enough, Fur-Fe2+ complex is formed and it can inhibit the RyhB transcription. SodB is not repressed and can be synthesized3. That is why Fur is described as an activator transcriptional factor in such a case.
Siderophores
During iron starvation, bacteria produce siderophores, small molecules that can catch any atom of iron in the environment to feed the bacteria. There are different types of siderophores, depending on their structure and their interaction with iron:
Catecholate,
phenolate,
hydroxymate,
carboxylate types.
We focus more specifically on enterobactin which is a catecholate siderophore (composed of catechol = bi-phenol) produced by E. coli. Enterobactin has a very high affinity for ferric iron K = 1052 M-14. This number is greater than the affinity of heme B and EDTA metal chelator for iron, respectively K = 1039 M−1 and 1025 M−1. De facto, enterobactin is one of the most powerfull siderophores known: it can extract iron from hemic source and even from air!
Figure 7: Structure of Enterobactin molecule.
Ferric iron Fe3+ or Ferrous iron Fe2+ established hydrogen bound with the alcohol groupment of enterobactin.
Figure 8: 3D structure of enterobaction without (A) or with (B) Ferric Iron5.
Genes of enterobactin pathway are under the regulation of the Fur protein in E.coli. It means that genes of enterobactin pathway are expressed only in lack of iron.
Figure 9: Pathway of the synthese of the enterobactin.
In our project, we want to produce enterobactin. Then we focus on the 6 enzymes (EntA, EntB, EntC, EntD, EntE and EntF) of it biosynthesis started from chorismic acid. However, we want to produce enterobactin in presence of iron, in order to reduce an excess of iron.
An alternative iron chelator
At the Regional Jamboree, we discussed with Edinburgh team which also work on iron chelation but with another chelator: the ferric ion-binding protein A (FbpA)
6
.
This protein from Neisseria gonorrhoeae is able to chelate various ion in solution as Fe
3+
, Ti
4+
,Cu
2+
, Zr
4+
(Zirconium ion)  or Hf
4+
(Hafnium ion)
7
. Its affinity for ferrous iron is 10
18
, less than our enterobactins system, but it remains a good alternative for our treatment.
Constructions
Figure 10: Enterobactine production using the Fur-inverter system.
Our goal is to decrease the iron absorption from the intestines to the blood by using an iron chelating bacteria, Iron Coli.
We have design a system that could produce enterobactin from chorimsic acid, under the regulation of an iron sensor with an inverter . When the iron is present in the environment (ie intestine) in high concentration, our Iron Coli will produce siderophores to chelate the iron and hence reduce the amount of iron that could be absorbed by the patient.
We also model enterobactin production of our Iron Coli and the potential efficiency of our treatment to cure iron overload.
References:
Andrews, S. C., Robinson, A. K. & Rodriguez-Quiñones, F. Bacterial iron homeostasis. FEMS Microbiology Reviews 27, 215–237 (2003).
Klaus Hantke, Iron and metal regulation in bacteria, Elsevier,2001, vol. 4, no2, pp. 172-177
Amanda G., Iron responsive Bacterial small RNAs: variation on a theme, Metallomics, January 2013
Zheng, T., Siderophore-Mediated Cargo Delivery to the Cytoplasm of Escherichia coli and Pseudomonas aeruginosa : Syntheses of Monofunctionalized Enterobactin Scaffolds and Evaluation of Enterobactin–Cargo Conjugate Uptake, J Am Chem Soc., 2012 Nov 7.
Raymond, K. N., Enterobactin: An archetype for microbial iron transport, PNAS, vol.100, n°7, April 2003.
Ferreiros, C., Criado, M. and Gomez, J. (1999) The Neisserial 37kDa ferric binding protein (FbpA). Comparative Biochemistry and Physiology Part B. 123. 1-7.
Alexeev, D., Zhu, H., Guo, M., Zhong, W., Hunter, D., Yang, W., Campopiano, D. and Sadler, P. (2003) A novel protein-mineral interface. Nature Structural Biology. 10. 297-302.
Contact: igemevry@gmail.com
Site developped by Gabriel Guillocheau
Retrieved from " http://2013.igem.org/Team:Evry/Project_FUR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
