<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Evry_Modelmeta1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Evry/Modelmeta1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Evry/Modelmeta1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Modelmeta1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Evry/Modelmeta1&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Evry/Modelmeta1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Evry/Modelmeta1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><TITLE>Iron coli project</TITLE></P><DIV id="header"><UL id="menu"><LI><A id="home-link" href="https://2013.igem.org/Team:Evry">Home</A></LI><LI><A id="project-link" href="https://2013.igem.org/Team:Evry/Project" title="Overview">Project</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_metabolism">Iron metabolism</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_diseases">Related diseases</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Project_FUR">Iron chelators</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Evry/References">References</A></LI></UL><LI><A id="team-link" href="https://2013.igem.org/Team:Evry/Team" title="Our team">Team</A></LI><LI><A href="https://igem.org/Team.cgi?id=1163" target="_blank">iGEM Profile</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Acknowledgement">Acknowledgement</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Press">Press</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Gallery">Photo gallery</A></LI><LI><A id="model-link" href="https://2013.igem.org/Team:Evry/Modeling" title="Overview">Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin production</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI><LI><A id="biology-link" href="https://2013.igem.org/Team:Evry/Biology" title="Overview">Biology</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Sensor">Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Inverter">Inverter</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Chelator">Chelator</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice" title="Overview">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Expert_consultation">Consultation</A></LI><LI><A href="https://2013.igem.org/Team:Evry/HumanPractice/Patients_perceptions">Survey</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Seminar">Seminar</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Philosophy">Philosophy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Economy">Economy</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Art">Art</A></LI><LI><A id="ref-link" href="https://2013.igem.org/Team:Evry/Pill_design">Capsule</A></LI><LI><A id="notebook-link" href="https://2013.igem.org/Team:Evry/Notebook" title="Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Primers">Primers</A></LI><LI><A id="spon-link" href="https://2013.igem.org/Team:Evry/Sponsors">Sponsors</A></LI></DIV><P><ASIDE><HEADER>
  Model
 </HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/Modeling">Overview</A></LI></UL><UL><LI><A href="https://2013.igem.org/Team:Evry/flush_model">Flush treatment:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modeltr1">Disease model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modeltr2">Treatment model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/ent_prod">Enterobactin Production:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta1">Sensor model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta2">Inverter model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/Modelmeta3">Enterobactin model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/Metabolism_model">Genome scale:</A></LI><UL class="second"><LI><A href="https://2013.igem.org/Team:Evry/interactions">Interaction Model</A></LI><LI><A href="https://2013.igem.org/Team:Evry/supply">Supply Model</A></LI></UL><LI><A href="https://2013.igem.org/Team:Evry/pop_scale">Population scale</A></LI></UL><HEADER>Tools</HEADER><UL><LI><A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic Functions</A></LI><LI><A href="https://2013.igem.org/Team:Evry/ChemicalTools">Chemical reasoning</A></LI><LI><A href="https://2013.igem.org/Team:Evry/data_analysis">Plate Reader</A></LI></UL></ASIDE></P><DIV id="maincontainer_evry2013"><DIV id="mainTextcontainer"><H1>Sensor Model</H1><H2>Introduction</H2><P>
In order to determine our <B><SPAN style="color:#bb8900">Iron</SPAN><SPAN style="color:#7B0000"> Coli</SPAN></B>'s enterobactin production rate, we first have to know how much time our bacteria will take to sense the ambient iron. So, this first part of the <I>Enterobactin production model</I> focuses on the synthetic <A href="https://2013.igem.org/Team:Evry/Sensor">sensing system</A> our team implemented in the bacteria.
</P><H2>Observations</H2><P>
Naturally, when there is a large amount of iron in the environment, 2 iron ions complex with 2 <A href="https://2013.igem.org/Team:Evry/FUR_Project">FUR</A> proteins and bind to the FUR Binding Site (FBS). This interaction regulate negativelly the genes downstream. 
As shown on the <A href="#Fig1">Figure 1</A>, our sensing system relies on a FBS. In order to easily model the sensing delay and analyse its results, the first construction is composed of a GFP placed right after the FBS.

</P><DIV align="center"><DIV class="captionedPicture" style="width:60%;"><DIV class="caption"><B>Figure 1:</B> Our first construction, our Iron sensor system<P>
See <A href="https://2013.igem.org/Team:Evry/Sensor" target="_blank">our sensor page</A> for more details</P></DIV></DIV></DIV><H2>Goals</H2><P>
Our goal in this part of the model is to create a generic FBS-related sensing model so that:
<UL><LI>We can determine the iron-sensing delay of our bacteria</LI><LI>The model can be reused by other projects that include a FBS sensor</LI></UL></P><H2>Materials and methods</H2><P><U><B>From Iron to FBS:</B></U>
The iron-FUR complex is simply formed that way: 
We reduced this equation to:
Which is not annoying, since we just have to divide our [FeFur] by two to get the real complex concentration.
We can easily write down both the formation (v) and the dissociation (v') speeds:
We chose to model the iron input in the bacteria using a linear function of the external iron concentration <I>Ferext</I>, the factor <I>p</I> being the cell-wall permeability for iron.
The FUR on the other hand is produced by the bacteria. Its evolution can also be considered as linear, using a mean production rate of <I>Fur0</I>.</P><P>
In this model, we only track the free Fe-FUR complex and not those attached to a FUR Binding Site. <I>FBS</I> is the number of inhibited Fur Binding Sites.</P><P><U><B>GFP expression:</B></U>
To simulate the inhibition phenomenon, we chose to use our <A href="https://2013.igem.org/Team:Evry/LogisticFunctions">Logistic function</A> under its differential form. Since it is the Fe-FUR that represses it, the LacI can be expressed as a logistic fuction of the Fe-FUR:
K<SUB>i1</SUB> is a non-dimensional parameter that repesents the inhibition power, and K<SUB>f</SUB> is the fixation rate of the Fe-FUR on the FBS. Finally, N<SUB>pla1</SUB> is the number of pasmids containing the GFP.</P><P><U><B>GFP Production:</B></U>
The <I>[mRNA]</I> and <I>[GFP]</I> equations are alike. The prodction rates are K<SUB>r</SUB> for the mRNA and K<SUB>p</SUB> for the GFP. Since <I>FBS</I> represents the number of inhibited Fur Binding Sites, we have to substract it from N<SUB>pla1</SUB>. 
Both variables also have a negative degadation term:</P><H2>Tuning the model</H2><P>
The only parameter we couldn't find in the litterature is K<SUB>i1</SUB>. Nevertheless, thanks to our construction and the experimental results we obtained with a plate reader, we can tune this parameter to obtain a <B>realistic sensing model</B>!
</P><DIV align="center"><DIV class="captionedPicture" style="width:60%;"><DIV class="caption"><B>Figure 2:</B> GFP/OD(t) for different iron concentrations
</DIV></DIV></DIV><P>
The <A href="#Fig2">Figure 2</A> is a set of experimental results showing the sensing efficiency for different iron concentrations. We can notice a substantial behaviour change between 10<SUP>-4</SUP>M and 10<SUP>-5</SUP>M of iron : the activity of our bacteria leaps at 10<SUP>-5</SUP>M.
Therefore, we ran simulations with different K<SUB>i1</SUB> values to try to determine the most fitting K<SUB>i1</SUB>.
</P><DIV align="center"><DIV class="captionedPicture" style="width:60%;"><DIV class="caption"><B>Figure 3:</B> Maximum GFP for different K<SUB>i1</SUB> values
</DIV></DIV></DIV><P>
The <A href="#Fig3">Figure 3</A> shows that our model can lead to 3 different leaps, depending on the K<SUB>i1</SUB> value. The leap we are interested in is the 10<SUP>-4</SUP>M  10<SUP>-5</SUP>M leap.
We thus zoomed in, and performed the simulations with a more adapted range of K<SUB>i1</SUB>.
</P><DIV align="center"><DIV class="captionedPicture" style="width:60%;"><DIV class="caption"><B>Figure 4:</B> Maximum GFP for different K<SUB>i1</SUB> values
</DIV></DIV></DIV><P>
The <A href="#Fig4">Figure 4</A> shows a more precise graph. It is with those simulation values that we calculated the most realistic K<SUB>i1</SUB>, by maximizing the leap it creates. We also figured that the best fitting case requires the GFP reaching a plateau right after 10<SUP>-5</SUP>M of iron.
Finally, we set <B>K<SUB>i1</SUB></B> at <B>6,3.10<SUP>-5</SUP></B>.
</P><H2>Results</H2><P>
The following results were computed using those parameters:<TABLE width="100%" border="1" style="border-collapse:collapse;"><TBODY><TR width="100%"><TH>Name</TH><TH>Value</TH><TH>Unit</TH><TH>Description</TH></TR><TR width="100%"><TD>p</TD><TD>0.1</TD><TD>min^-1</TD><TD>Permeability of cell wall</TD></TR><TR width="100%"><TD>K<SUB>FeFUR</SUB></TD><TD>0.01</TD><TD>M<SUP>-1</SUP>.s<SUB>-1</SUB></TD><TD>Formation constant of FeFur complex</TD></TR><TR width="100%"><TD>D<SUB>ff</SUB></TD><TD>0.001</TD><TD>min<SUP>-1</SUP></TD><TD>FeFUR degradation rate</TD></TR><TR width="100%"><TD>K<SUB>p</SUB></TD><TD>0.5</TD><TD>min<SUP>-1</SUP></TD><TD>translation rate</TD></TR><TR width="100%"><TD>N<SUB>A</SUB></TD><TD>6.02*10<SUP>20</SUP></TD><TD>mol<SUP>-1</SUP></TD><TD>Avogadro's constant</TD></TR><TR width="100%"><TD>V</TD><TD>6.5*10<SUP>-16</SUP></TD><TD>m<SUP>3</SUP></TD><TD>Volume of a bacterium</TD></TR><TR width="100%"><TD>D<SUB>mRNA</SUB></TD><TD>0.001</TD><TD>min<SUP>-1</SUP></TD><TD>mRNA degradation rate</TD></TR><TR width="100%"><TD>K<SUB>f</SUB></TD><TD>10<SUP>-4</SUP></TD><TD>min<SUP>-1</SUP></TD><TD>fixation rate of FeFUR</TD></TR><TR width="100%"><TD>Fur<SUB>0</SUB></TD><TD>0.01</TD><TD>mM.min<SUP>-1</SUP></TD><TD>Fur Production rate</TD></TR><TR width="100%"><TD>K<SUB>i1</SUB></TD><TD>6,3.10<SUP>-5</SUP></TD><TD>-</TD><TD>Sensor efficiency</TD></TR></TBODY></TABLE></P><DIV align="center"><DIV class="captionedPicture" style="width:60%;"><DIV class="caption"><B>Figure 5:</B> GFP(t) for different iron concentrations
</DIV></DIV></DIV><P>
As shown in the <A href="#Fig5">Figure 5</A>, there is a significant change in GFP production after 1000 seconds between the curve at 10<SUP>-4</SUP>M of iron and the one at 10<SUP>-5</SUP>M. The bacteria can thus sense low concentrations of iron.
</P><H2>Conclusion</H2><P>
The first construction of our team is fully modeled : we made a <B>generic iron sensing model</B> (with the FBS system).
Thanks to the <B>experimental results</B> we obtained with a plate reader, we were able to <B>tune an unknown parameter</B> to obtain a more realistic model.
This sensing model allowed us to continue the Enterobactin production model, the next step being the <A href="https://2013.igem.org/Team:Evry/Modelmeta2">Inverter model</A>. 
</P><H2>Models and scripts</H2><P>
This model was made using the Python language. You can <A href="https://static.igem.org/mediawiki/2013/1/1b/Sensormodel.zip"> download the python script here</A>.
</P></DIV></DIV><DIV id="footer-wrapper"><DIV id="footer"><DIV>Contact: <A href="mailto:igemevry@gmail.com">igemevry@gmail.com</A></DIV></DIV><DIV align="center" style="font-size:60%">Site developped by Gabriel Guillocheau</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Evry/Modelmeta1">http://2013.igem.org/Team:Evry/Modelmeta1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Evry/Modelmeta1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Evry/Modelmeta1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Evry/Modelmeta1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Evry/Modelmeta1&amp;oldid=360815" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>