<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NYU_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NYU/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NYU/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NYU/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NYU/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NYU/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NYU/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="allcontent"><DIV id="headerlinks"><UL id="nav"><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:NYU">Home</A></LI><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:NYU/Project">Project</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Project">Overview</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Experiments">Experiments</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Assembly">Overlap Assembly</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Notebook">Notebook</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Parts">Biobricks</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Programming">Programming</A></LI></UL><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:NYU/Team">Team</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Team">NYU</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/CornellMed">Cornell Med</A></LI><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:NYU/Sponsors">Sponsors</A></LI><LI><A class="bannerlinks" href="https://2010.igem.org/Team:NYU/Sponsors">Sponsors</A></LI><LI><A class="bannerlinks" href="http://www.sciencehouse.com">ScienceHouse</A></LI><LI><A class="bannerlinks" href="http://www.nysynbio.org">NY synbio</A></LI><LI><A class="bannertoplinks" href="https://2010.igem.org/Team:NYU/Contact">Contact Us</A></LI></DIV><P>
In order to keep everyone (including you) up to date on the team's progress over the course of the summer and fall we've decided to keep an ongoing lab notebook on our wiki. In that light, here are the thoughts, opinions, trials and tribulations of the 2010 NYU iGEM team:
</P><P>Check out our OpenWetware Lab Notebook to see the protocols optimized over the summer : <A href="http://openwetware.org/wiki/IGEM:New_York_University/2009/Notebook/ImmunoYeast_2010" class="external text" rel="nofollow">ImmunoYeast2010</A></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Notebook"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Notebook</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#7.2F20.2F10"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">7/20/10</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#7.2F21.2F10"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">7/21/10</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#7.2F22.2F10"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">7/22/10</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#7.2F25.2F10"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">7/25/10</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#7.2F26.2F10"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">7/26/10</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#7.2F28.2F10"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">7/28/10</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#7.2F29.2F10"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">7/29/10</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#7.2F30.2F10"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">7/30/10</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#8.2F02.2F10"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">8/02/10</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#8.2F03.2F10"><SPAN class="tocnumber">1.10</SPAN><SPAN class="toctext">8/03/10</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#8.2F09.2F10"><SPAN class="tocnumber">1.11</SPAN><SPAN class="toctext">8/09/10</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#8.2F10.2F10.2F"><SPAN class="tocnumber">1.12</SPAN><SPAN class="toctext">8/10/10/</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#8.2F16.2F10"><SPAN class="tocnumber">1.13</SPAN><SPAN class="toctext">8/16/10</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#8.2F30.2F10"><SPAN class="tocnumber">1.14</SPAN><SPAN class="toctext">8/30/10</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#9.2F1.2F10"><SPAN class="tocnumber">1.15</SPAN><SPAN class="toctext">9/1/10</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Notebook">Notebook</SPAN></H2><H5><SPAN class="mw-headline" id="7.2F20.2F10">7/20/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>Welcome to the NYU iGEM Lab Notebook! Today we initiate writing in this notebook in order for each of us on the team to know the trials and tribulations everyone else is going through, and for our advisers to know what's going on with the team without having to be in the lab all the time.  While it may not have the formatting of some of the more computer-literate teams, it will certainly work for us!
</P><UL><LI>John:
</LI></UL><P>Performed digestion of 32 plasmids using either EcoRI, PstI, or both.  These plasmids included RBS, Flag, E1, F1, sectag (multiple), NUB, and Strep.  This was performed to see if our digestion enzymes were working correctly.  After digestion, these solutions were run in 4 1.5 gels with large sized wells to verify if the enzymes were cutting.
</P><H5><SPAN class="mw-headline" id="7.2F21.2F10">7/21/10</SPAN></H5><UL><LI>John:
</LI></UL><P>Ran 1.5 % agarose gel to verify whether primers worked on the Nub-Fusion construct.  Previously, we PCRed NUB-Fusion plasmid using verification primers.  After, we used this product as a template for PCR with the NUB specific primers that we designed. Sec-tag was also PCRed and run on the gel.  Additionally, old PCR cleanup NUB and Sectag were run on the gel to see if they can be used or disposed.
</P><UL><LI>Harris:
</LI></UL><P>Transformations from the day before of ADH1-Aga2Linker and ECFP-Terminator in the Biobrick Chloramphenicol plasmid failed, although the retransformation of Aga2 (in the Amp plasmid) alongside them worked. Ligations left overnight so today I am retransforming them alongside BBa_K098994, which is 5B from Plate 3 of the initial distribution, and is in a Chloramphenicol resistance plasmid. This should grow up fine, and if it does, then we know that the problem is with the ligations, but if it doesn’t, then there is a problem with the Chloramphenicol plates.
</P><H5><SPAN class="mw-headline" id="7.2F22.2F10">7/22/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>PCR results from 7/20 were disappointing but not unforeseen. Previous sec tag PCRs show band of appropriate size, but biobricking attempts have been unsuccessful with cause unknown. Will perform one more biobricking procedure on the sec tag under careful scrutiny before troubleshooting for larger problems. 
</P><P>Transformed the N-terminal and C-terminal GFP biobricks from the initial distribution. These will be used to show that the test scFv and test antigen are, in fact, complexing in the cytoplasm (and maybe on the surface). 
</P><P>Miniprepped the Gal4 and VP16 biobricks to use as a Gal1-10 promoter activator for the response system.
</P><H5><SPAN class="mw-headline" id="7.2F25.2F10">7/25/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>Harris’ transformations of pADH1:Aga2-Linker and ECFP:Terminator assembly were successful. I inoculated 4 colonies from each assembly product in LB+Chloramphenicol and performed colony PCRs. 
</P><P>Unfortunately I REALIZED THERE IS NO RBS BETWEEN THE pADH1 PROMOTER AND AGA2! This simple mistake prevented the MIT team last year from having a successful system so we need to watch out for this. We may want to ligate an RBS onto each of the promoters we want to use and use each of those instead so we definitely do not make this mistake.
</P><P>Retransformation of the secretion tag biobricking ligation was also successful. I inoculated 6 colonies from the two plates and performed colony PCR. 
</P><P>Transformation of the split-GFP biobricks (N-terminal and C-terminal halves of GFP) from the iGEM initial distribution was successful – inoculated 3 colonies of each biobrick but had already started thermal cycler, so colony PCR will be done later. 
</P><P>Ordered second set of Nub primers as well as biobrick-compatible primers for manipulating the backbone of pCTCON.
</P><H5><SPAN class="mw-headline" id="7.2F26.2F10">7/26/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>Got sequencing results from Genewiz for both of the supposed N-ub fusions we received from iGEM – neither sequence contained Nub, apparently both plasmids are void. 
</P><P>Instead of excising from a plasmid, we get to build the part via overlapping PCR (which, luckily, I’ve been wanting to try for awhile). So I broke the ~140bp Nub gene down into 3 overlapping sequences and ordered the appropriate oligos for the PCR.
</P><H5><SPAN class="mw-headline" id="7.2F28.2F10">7/28/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>Received both orders from IDT. Performed two overlap PCRs with different concentrations of oligos. Will run on a gel tomorrow with some Biobrick verification PCRs. 
</P><UL><LI>John:
</LI></UL><P>Research potential providers of scFv and antigen parts.  Made transformation media and SOB.
</P><H5><SPAN class="mw-headline" id="7.2F29.2F10">7/29/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>Constructed the pCTCON plasmid backbone with BIOBRICK ends through PCR amplification. [extremely long cycle - 5:30 ext. time]. Also verified biobrick Aga2:Linker:LoxP via VerF/R2 primers and Aga2F/VerR2.
</P><UL><LI>John:
</LI></UL><P>Email out requests for potential scFv and antigen parts.
</P><H5><SPAN class="mw-headline" id="7.2F30.2F10">7/30/10</SPAN></H5><UL><LI>Russell:
</LI></UL><P>I am trying to develop a couple of new (at least for me) protocols to use with biobricks. 
</P><P>The first is overlap PCR of the N-ubiquitin biobrick. This biobrick is only 140 bases long, so it is not viable to synthesize it via GeneArt (even though we still have all our basebucks) because of the much larger minimum order requirement. Instead, I broke the part down into three overlapping oligos and ordered them from IDT. The first oligo is just the first 60bp of the CDS. The second and third are the next 60bps (with 10bp overlap) but they are taken from the reverse complement of the sequence. I am currently writing a perl script to do this automatically and will post it as soon as I have it. 
</P><P>The second protocol I’m developing will help with maintaining stocks of biobricks. We are in full swing at the moment and keeping our stocks of soluble biobrick DNA is troublesome. Instead of using a good amount of each of our biobrick minipreps in an assembly, it may be possible to use a PCR product of the VerF/R2 primers and a biobrick template. That way we can just amplify the biobrick region from biobrick template and use the PCR product (with or without purification) directly in our assembly digestions and ligations. I just started the PCR cycle and will post results when I have them.  
</P><H5><SPAN class="mw-headline" id="8.2F02.2F10">8/02/10</SPAN></H5><UL><LI>John:
</LI></UL><P>Emailed Dr. Marasco about anti-Tat ScFv, potential for this ScFv to work in the reducing environment of the cytoplasm.  Also, researched potential sources for other “intrabodies,” stumbling on “Single Domain intracellular Antibodies: A Minimal Fragment for Direct In Vivo Selction of Antigen-specific Intrabodies”:  Further thought required on the applicability of intracellular single variable domain (IDab).
</P><UL><LI>Russell:
</LI></UL><P>N-ubiquitin biobricking transformation resulted in only one colony on the plate with 200uL of transformant. I inoculated it, but such a low efficiency may mean that it is a self-ligated plasmid (although self-ligation control resulted in no colonies). 
</P><P>pTEF:RBS assemblies from PCR amplicons resulted in many colonies. I inoculated 4 colonies and performed VerF/R PCR test to see if inserts are the expected size. Alongside this PCR I am making another 50uL stock of pCTCON backbone, so the cycle will be overnight and will run the gel tomorrow. 
</P><P>Literature research has turned a new leaf. Research into intrabody scFv has yielded a lot of great information and previous studies similar to the functional aspect of our system. Previous studies, however, do not use the split ubiquitin system nor encourage transcription of nutritional markers for directed evolution toward antibody binding (rather, they merely mutate via error-prone PCR and scan for binding antibodies through a system more like the two-hybrid). John is writing to researchers requesting scFv intrabodies and antigens, as well as a library of scFvs mutagenized from the original intrabody scaffold. Hopefully these requests will result in useful materials and advice!
</P><UL><LI>Harris:
</LI></UL><P>Miniprepped pTEF-RBS from ligation of PCR product digested pieces, along with minipreps of the FLAG tag grown up and transformed from the initial distribution. 
</P><H5><SPAN class="mw-headline" id="8.2F03.2F10">8/03/10</SPAN></H5><UL><LI>Harris
</LI></UL><P>Ran gel for pTEF-RBS and Aga2Linker-ECFPTerminator, and found that pTEF-RBS isn’t correct, probably only RBS was cut out. Innoculated pTEF-RBS from plate from last week, where the ligation was performed from a Miniprepped digest, rather than from a PCR (from which the current minipreps of pTEF-RBS are). Aga2Linker-ECFPTerm had a band at the correct size (1.3 kb) but also had a smaller band around 400-500 bp, need to think about this and make sure the insert is correct. 
</P><H5><SPAN class="mw-headline" id="8.2F09.2F10">8/09/10</SPAN></H5><UL><LI>Russell
</LI></UL><P>Today I transformed biobricks necessary for the second half of our system construction. Over the weekend I've been brainstorming ways to construct the response system circuits and have decided that we'll be using the LexA and LacI DNA binding proteins. The Initial Distribution this year came with most of the parts we'll need including some very useful composite reporter devices. For a time I also organized our growing collection of liquid and glycerol biobricks to maintain smaller stocks, something that will probably be an ongoing battle in the war against disorganization. 
</P><P>We are having some trouble assembling very small parts onto larger ones, case in point: the RBS onto a large promoter. Harris believes that the dichotomy of sizes of these parts makes it hard to relatively represent each in reaction solutions. We thought that the PCR product assembly method would work for this but have been disappointed thus far. For the sake of time I am going to order oligos and PCR-ligate the RBS onto pGAL and pTEF, as well as the RBS onto LacO onto pTEF for our reporter constructs. Once this is finished we'll hopefully be able to work smoothly through the assembly procedure. At this point I would probably give up a kidney for the ability to synthesize all of these constructs. 
</P><P>Also sent ECFP:Terminator I and II to sequencing to be 100% certain they are of the correct sequence. 
</P><H5><SPAN class="mw-headline" id="8.2F10.2F10.2F">8/10/10/</SPAN></H5><UL><LI>Russell
</LI></UL><P>I ordered the rest of the primers we will need for the construction of our system (except for genomic integration, which I will order when I know what loci we want to use). Because we are having so much trouble assembling two parts of great size differences (the RBS – 18bp and a promoter – 1000bp) we are going to biobrick the RBS onto the promoter using PCR assembly. For that method I ordered two primer constructs for an inducible Gal promoter and the constitutive Tef promoter. 
</P><P>Although the Biobrick Verification primers that the registry uses work for sequencing and larger parts, PCR gel verification of biobrick assemblies or basic parts would be easier if we could amplify just the insert and not the ~200bp outside the insert. In that light, I constructed primers based on the biobrick prefix and suffix, including 6bp 5’ GCG clamps so that, if we so choose, we can use the PCR products to do assemblies.
</P><H5><SPAN class="mw-headline" id="8.2F16.2F10">8/16/10</SPAN></H5><UL><LI>Russell
</LI></UL><P>Weekend trip to LA’s over – let’s dive right in:
</P><P>In order to circumvent the difficulty of assembling biobricks of very different size (namely a ribosome binding site, 18bp, and a promoter, ~1000bp) we PCRed and biobricked the inducible pGAL and the consititutive pTEF promoters with RBSs on the reverse primers. Today we actually did the main portion of this biobricking. We’ll submit these promoters to the Registry and hopefully this will save some time and stress in future assemblies. 
</P><P>In order to use the backbone of the pCTCON plasmid (which has a trp1 selectable marker) we need a strain of yeast that does not have the trp1 gene. After talking with Dr. Gresham we’ve decided to make our own yeast knockout strain – something I’m excited about because I’ll get to learn something new! We’re going to use the FY4 wild-type strain because the entire genomic sequence is available online, making it very easy to manipulate the genome thanks to yeast DSB recombination pathways. To this end I designed and ordered primers for the KanMX gene (an antibiotic-selectable gene) with trp1 extensions flanking the KanMX coding region. After PCRing the KanMX CDS, we can transform the PCR product and the yeast will take it from there, inserting the KanMX CDS into the trp1 locus. Because this will konckout trp1 function, we’ll have to grow this strain with tryptophan as well as G148 to select for KanMX transcription. 
</P><P>I also ordered primers for genomic integration at the Ura3 locus of any biobrick part. Using sequences complementary to the biobrick prefix and suffix, these oligos will make it possible to insert any fully-assembled biobrick construct into this locus. We will post these sequences to the Registry after we get a successful integration. 
</P><UL><LI>Harris
</LI></UL><P>Biobricking PCR assemblies of pTEF-RBS and pGAL-RBS, along with Ura3 from PCR product clean up as well, all into Amp plasmid. Assembling ECFP-Terminator into Tet plasmid, also from PCR product clean ups. Digested more Amp plasmid for this purpose, 50 uL digest at 10 ng/uL.
</P><H5><SPAN class="mw-headline" id="8.2F30.2F10">8/30/10</SPAN></H5><UL><LI>Russell
</LI></UL><P>We’ve had some success with the PCR amplification of biobrick parts for assembly. Instead of growing up large quantities of a 2.5kb plasmid and using ~100bp, we’ve decided to amplify the part via PCR and then use the PCR product in our assemblies. This has helped us manage stocks of biobrick parts because each team member can have their own PCR product to do their assemblies while working off of the Biobrick reservoir that I maintain. Because each person has their own PCR product from the original stock PCR-induced mutations are less likely to occur, but they are still a possibility. Because of this we are maintaining a rigorous (and costly) Quality Control program by sequencing every assembly.
</P><P>In an effort to conserve funds we have moved from Qiagen spin columns to those offered by Epoch Life Sciences. We have begun to make our own buffers (or use those leftover from Qiagen kits) to do our purifications. We chose this tack because it is cheaper and easier to do large quantities of minipreps, but instead we could recycle our spin columns to be a bit more environmentally friendly [like Harvard’s team this year].
</P><H5><SPAN class="mw-headline" id="9.2F1.2F10">9/1/10</SPAN></H5><UL><LI>Russell
</LI></UL><P>Successful assembly of the Aga2….linker…..eCFP! Stay tuned for pictures of cyan fluorescing yeast cell surfaces.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:NYU/Notebook">http://2010.igem.org/Team:NYU/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NYU/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NYU/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NYU/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NYU/Notebook&amp;oldid=177502" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>