<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sheffield_project_science_furreporter skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sheffield/project/science/furreporter</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>A template page</TITLE></P><DIV class="container"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown" id="li-home"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>HOME</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>INTRO</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>PROJECT</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>TEAM</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>NOTES</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>JUDGING</P></TD></TR></TBODY></TABLE></CENTER></LI><DIV id="shield"><DIV class="container"><DIV class="heading"><TABLE><TBODY><TR><TH><DIV class="heading-title"><H1 style="padding:0px">REPORTERS</H1></DIV></TH></TR></TBODY></TABLE></DIV><TABLE id="banner-section"><TBODY><TR><TH><DIV class="cont button1"><DIV class="banner"><P>HEMERYTHRIN</P></DIV></DIV></TH></TR></TBODY></TABLE><TABLE class="banner-inside"><TBODY><TR><TH><DIV class="cont button2"><DIV class="banner"><P>FUR/RYHB</P></DIV></DIV></TH></TR></TBODY></TABLE><DIV class="box1"><DIV class="jumbotron"><H2>Introduction</H2><P>Hemerythrin is a protein mainly found in marine invertebrates such as lobsters, crabs, shrimps etc. This protein is capable of binding iron without the need to possess a heme group. Its active site is characterised by the presence of five histidines and two carboxylate ligands which coordinate two iron atoms<SUP>1</SUP> (Fig. 1). Upon the incorporation of iron, hemerythrin binds oxygen and change its colour from the colourless state into violet-pink. Hemerythrin is a relatively small protein (13.5-16 kDa) with a strongly conserved sequence and secondary structure. Using hemerythrin, we hypothesised that we would be able to detect an increase in intracellular iron levels through observing the colour change of expressed hemerythrin.
</P><P><SMALL><B>Figure 1. Active site of hemerythrins comprise of five histidine residues and two glutamate residues.</B> One of the incorporated irons can bind oxygen and change the colour of the protein from colourless to violet/pink.</SMALL></P><P>The hemerythrin we have initially chosen for our project comes from marine organism Themistedyscritum<SUP>2</SUP> (TdHr). Although it is an attractive protein to use in our experiment, marine invertebrates are eukaryotic organisms and expression of these hemerythrins in bacterial cells is somewhat challenging. We were able to find hemerythrin-like bacterial proteins, which share similarities in sequence and function, by searching the literature. These proteins can be found in organisms such as methane-oxidising <I>Methylococcus capsulatus</I> (McHr)<SUP>3</SUP> and sulfate-reducing <I>Desulfovibrio vulgarisi</I> (Dcr-Hr)<SUP>4</SUP>.</P><P>Sequence comparison among our three chosen hemerythrins showed conservation of 5 histidine residues required for iron binding, as well as some other crucial domains. All of the proteins were previously documented to show a colour change upon oxidation and can therefore be considered suitable for the project. However, visibility, speed of the colour change and dependence on iron concentrations can differ among these proteins, which would need to be tested in our experiments. 
 </P><H2>Results</H2><P>All hemerythrin genes were successfully cloned under control of a constitutive (BBa_J23100 biobrick) or an IPTG-inducible promoter to aid us with characterisation of our proteins. For the constitutive system the gene was first cloned between the XbaI and EcoRI sites (leaving RFP in) and then the constitutive promoter was cloned between the PstI and XbaI sites afterwards. Genes under the control of the inducible promoter were cloned into pET28a between the NdeI and XhoI sites. Presence of the cloned genes was confirmed by restriction digest of isolated plasmids (Figs. 2, 3 &amp; 4A).  
</P><P><SMALL><B>Figure 2. Cloning of Dcr and Td hemerythrins under control of strong J23110 promoter.</B><B>Lane 1:</B> 2-Log Gene ladder; <B>2</B>: pSB1C3 digested with PstI and EcoRI, visible bands: plasmid backbone (~2 kb) and RFP+Dcr (promoterless) (~1.5 kb); <B>3-7</B>: : pSB1C3 digest after promoter was cloned in (restriction with PstI and EcoRI), visible bands: plasmid backbone and Dcr + promoter (511 bp); <B>8-12</B>: : pSB1C3 digest after promoter was cloned in, visible bands: plasmid backbone and Td + promoter (442 bp).</SMALL></P><P>All of our proteins have been shown to be expressed from the inducible plasmid upon induction with IPTG at 37°C for 3 hours (Fig. 4B). However, no expression can be observed from the constitutive promoter. Due to time constraints, expressed proteins have not been tested for the colour change between the uninduced sample and the induced sample. However, future work could investigate a change of cell culture colour upon IPTG induction would be measured using a spectrophotometer, either by measuring absorbance at specified wavelengths<SUP>1,3,4</SUP> or by measuring absorbance for all wavelengths of visible spectrum; thus performing a more detailed analysis. Such an experiment could elucidate possible applications of hemerythrins in the detection of iron levels or presence of oxygen within a short amount of time. 
</P><P><SMALL><B>Figure 3. Cloning of Mc hemerythrin under control of strong J23110 promoter.</B><B>Lane 1</B>: 2-Log Gene ladder; <B>2</B>: pSB1C3 digested with EcoRI and XbaI, visible bands: plasmid backbone (~2 kb) + RFP and promoterless Mc (~0.4 kb); <B>3</B>: pSB1C3 digested with PstI and EcoRI, visible bands: plasmid backbone and RFP+Mc (promoterless) (~1.5 kb); <B>4-6</B>: pSB1C3 digest after promoter was cloned in (restriction with PstI and EcoRI), visible bands: plasmid backbone and Mc + promoter (493 bp).</SMALL></P><P><SMALL><B>Figure 4. A: Cloning of Dcr, Mc and Td into IPTG inducible vector pET28a.</B><B>Lane 1</B>: 2-Log Gene ladder; <B>2</B>: pET28a-Dcr digested with NdeI and XhoI, visible bands: plasmid backbone (~5.3 kb) and Dcr (414 bp); <B>3</B> pET28a-Mc, visible bands: plasmid backbone (~5.3 kb) and Mc (396 bp); <B>4</B>: pET28a-Td, visible bands: plasmid backbone (~5.3 kb) and Td (345 bp). <B>B</B>: Expression of Dcr, Mc and Td in cells before and after induction with IPTG. For each protein 2 different biological repeats were first grown overnight and then induced with IPTG, L- ladder, U – uninduced, I – induced. For Dcr (16 kDa) and Td (13.5 kDa) we can see clear induction of expression after adding IPTG. Expression of Mc (14.7 kDa) is less clear and should be investigated further. </SMALL></P><H2>References</H2><OL><LI><SMALL>Stenkamp RE, Sieker LC, Jensen LH, McCallum JD, Sanders-Loehr J. <I>Active site structures of deoxyhemerythrin and oxyhemerythrin.</I> Proc Natl Acad Sci U S A. 1985;82(February):713–6. </SMALL></LI><LI>Loehr JS, Lammers PJ, Brimhall B, Hermodson MA. <I>Amino acid sequence of hemerythrin from Themiste dyscritum.</I> J Biol Chem. American Society for Biochemistry and Molecular Biology; 1978 Aug 25;253(16):5726–31. </LI><LI>Kao WC, Wang VCC, Huang YC, Yu SSF, Chang TC, Chan SI. <I>Isolation, purification and characterization of hemerythrin from Methylococcus capsulatus</I> (Bath). J Inorg Biochem. 2008;102(8):1607–14. </LI><LI>Xiong J, Kurtz DM, Ai J, Sanders-Loehr J. <I>A hemerythrin-like domain in a bacterial chemotaxis protein. </I>Biochemistry. American Chemical Society; 2000;39(17):5117–25. </LI></OL></DIV></DIV><DIV class="box2"><DIV class="jumbotron"><H2>The Theory</H2><P>Our Fur repressor/RyhB reporter system is based upon the way that the cell normally detects, and then controls, its intracellular iron levels. This <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Homeostasis" data-content="The ability of the cell or organism to regulate its own conditions to a particular equilibrium">homeostasis</BUTTON> is necessary as, although iron is vital for the cell to function, it is also potentially dangerous due to the Fenton reaction, which produces dangerous <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Reactive Oxygen Species" data-content="Chemically reactive chemicals that contain oxygen, produced as a by-product from many biological reactions. These are able to damage DNA by reacting with the nucleotides, causing mutations">reactive oxygen species (ROS)</BUTTON> under <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Aerobic conditions" data-content="An environment that has free diatomic oxygen (O&lt;sub&gt;2&lt;/sub&gt;)">aerobic conditions</BUTTON><SUP>1</SUP>. </P><P>To combat this the cell <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Downregulates" data-content="A regulatory process wherein the cell to reduces the concentration of a certain cellular component">downregulates</BUTTON> genes needed for import of iron, such as those that produce and uptake <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Siderophores" data-content="Small organic molecules produced by some bacteria and fungi capable of binding iron with extremely high affinity, these can then be taken up by the organism where they release their bound iron">siderophores</BUTTON>, with the Ferric uptake regulator (Fur) repressor. When bound to <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Fe&lt;sup&gt;2+&lt;/sup&gt;" data-content="Charged form of iron (iron ion)">Fe<SUP>2+</SUP></BUTTON> Fur is able to bind a region of DNA known as the Fur box, repressing the genes downstream. However, in low iron conditions Fur is no longer able to bind to Fe<SUP>2+</SUP>, which causes a change in structure that renders it unable to bind to DNA, de-repressing the genes under its control and hence increasing iron uptake<SUP>1</SUP>. </P><P>Fur has also been noted to positively regulate certain genes under high iron conditions. These genes include proteins that require iron, those that store it, and some that help to deal with its negative effects. One of these genes is <I>sodB</I>, which encodes a <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Superoxide Dismutase" data-content="An antioxidant enzyme that helps to reduce concentrations of reactive oxygen species">superoxide dismutase</BUTTON> that is able to detoxify the ROS produced by iron via the Fenton reaction under aerobic conditions<SUP>1</SUP>.</P><P>This positive regulation is actually done indirectly, via a <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="small regulatory RNA" data-content="Small, non-coding, RNA molecules that inhibit the expression of specific genes, by preventing RNA translation">small regulatory RNA (srRNA)</BUTTON> known as RyhB. This is able to bind to complementary sequences found on the <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Messenger RNA (mRNA)" data-content="RNA that is transcribed from DNA, that codes for the production of a protein">mRNA</BUTTON> of certain genes, such as <I>sodB</I>, and inhibit their <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Translation" data-content="The process whereby a peptide polymer is produced by a ribosome, using an mRNA sequence as a guide">translation</BUTTON>. <I>ryhB</I> is located directly downstream of a Fur box, meaning that it is downregulated in high iron conditions, causing genes that are negatively regulated by RyhB to be effectively <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Upregulated" data-content="A regulatory process wherein the cell to increases the concentration of a certain cellular component">upregulated</BUTTON> in high iron conditions when <I>ryhB</I> is repressed<SUP>2</SUP>.</P><P>In order to exploit this, we have included the RyhB binding sequence from <I>sodB</I> in the <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="5’ untranslated region (5’ UTR)" data-content="A sequence found upstream of the initiation codon on mRNA that is not translated. It is often important in the regulation of translation">5’ untranslated region (5’ UTR)</BUTTON> of <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Superfolder Green Fluorescent Protein (sfGFP)" data-content="A form of green fluorescent protein that folds more readily">superfolder green fluorescent protein (sfGFP)</BUTTON>. We hypothesised that when RyhB was expressed then translation of <I>sfGFP</I> would be inhibited and hence, sfGFP would be downregulated in low iron conditions when <I>ryhB</I> was not being repressed. Ultimately this would lead to increased fluorescence at high intracellular iron concentrations, but decreased fluorescence when these concentrations were lower.  </P><H2>Cloning</H2><P>Our reporter system contains a RyhB binding sequence in the 5’ UTR of <I>sfgfp</I> and is present under the control of two promoters of different strengths, a medium and a strong. These constructs were cloned into both pBluescript KS(+) and pSB1C3 <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Plasmid" data-content="A piece of bacterial DNA which is separate from the chromosomal DNA, that can be transferred between bacteria by conjugation">plasmids</BUTTON> and then transformed into our enterobactin production mutant, <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="JC28" data-content="A mutant &lt;i&gt;E. coli&lt;/i&gt; strain with the genotype: W3110 &lt;i&gt;∆fecABCDE ∆zupT ∆mntH ∆entC ∆feoABC&lt;/i&gt;. Deficient in enterobactin production.">JC28</BUTTON>, along with our wild type <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="W3110" data-content="A common lab strain of &lt;i&gt;E. coli&lt;/i&gt; K12. Parent strain of the JC28 mutant type.">W3110</BUTTON> for use as a control (Fig. 1).</P><P><SMALL><B>Figure 1. Cloning of RyhB-GFP into pSB1C3 and pBluescript KS(+) plasmids.</B> 
This figure shows a map of pSB1C3 plasmid containing RyhB-GFP insert (top-left) and a map of pBluescript KS(+) plasmid (top-right) containing RyhB-GFP insert (C and D) </SMALL><BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Agarose gel electrophoresis" data-content="A technique used for separating DNA fragments according to their size by driving the molecules through a gel with an electric field"><SMALL>Agarose gel electrophoresis</SMALL></BUTTON> image of a <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Restriction Digest" data-content="The act of cutting DNA into smaller pieces using restriction enzymes. Products can be run on an agarose gel to check for plasmid insertions of the correct size">restriction digest</BUTTON> with SpeI and EcoRI <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="Restriction Enzyme" data-content="Enzymes that are able to cleave DNA sequences at or near a specific DNA sequence">restriction enzymes</BUTTON> of either our insert-containing pSB1C3 (bottom-left) or pBluescript KS(+) (bottom-right) stock.These plasmids were then used to transform our JC28 mutant and our W3110 wild type control.</P><H2>Change in GFP expression during growth at different iron concentrations</H2><P>As an initial test, we wanted to see if fluorescence changed in response to growth in different iron concentrations. We grew our mutant and wild type strains containing pBluescript KS(+)-RyhB-GFP construct in defined medium with either 5 μM or 100 μM FeCl<SUB>3</SUB> for low or high iron conditions respectively. We then measured the <BUTTON class="btn btn-lg btn-danger" data-placement="top" data-toggle="popover" title="OD&lt;sub&gt;600&lt;/sub&gt;" data-content="The optical density of a sample, which is indicative of the concentration of cells. This is determined by measuring absorbance at 600 nm using a spectrophotometer">OD<SUB>600</SUB></BUTTON> and the fluorescence of our cells at 535 nm, which we adjusted according to each samples respective OD (Fig. 2). We were unable to detect any significant correlation between fluorescence of the sample and iron concentration in the growth medium, suggesting that our RyhB-GFP reporter system is not affected by changes in extracellular iron concentrations.   </P><P><SMALL><B>Figure 2. Fluorescence of JC28 mutants or W3110 wild types transformed with RyhB-GFP constructs under the control of medium (MedGFP) or strong promoters (StrGFP).</B> Measurements were taken at a single time point after growth overnight in either low iron containing medium (5 μM FeCl<SUB>3</SUB>) or high iron containing medium (100 μM FeCl<SUB>3</SUB>). An unpaired t-test was performed showing no significant differences (&lt;0.05) between low and high iron media. </SMALL></P><H2>Change in GFP expression in response to addition of iron </H2><P>In order to see the effect of an increase of extracellular iron concentration on fluorescence, JC28 and W3110 strains containing RyhB-GFP constructs under the control of medium and strong promoters were first grown on low iron containing medium (5 μM FeCl<SUB>3</SUB>). Samples were taken and FeCl<SUB>3</SUB> was added to half, these samples were then placed in a plate reader. Fluorescence was measured at 485 nm excitation and 535 nm emission with cell density being estimated using OD<SUB>600</SUB>. These measurements took place every 3.6 minutes for 86 minutes total (Fig. 3). This demonstrates a significant increase in fluorescence following addition of iron. We believe that this is due to a decrease in RyhB levels due to repression by Fur, and not due to growth of the bacteria, as we have adjusted for this increase based upon the OD<SUB>600</SUB> measurements.</P><P><SMALL><B>Figure 3. Change in GFP expression after 90 minutes following addition of FeCl<SUB>3</SUB> to a final concentration of 100 μM (A) or without addition of FeCl<SUB>3</SUB> (B)</B>. There is a significant increase in fluorescence upon addition of iron over the course of 90 minutes (A), however this was not seen over the same time period without addition of iron (B). Significance was assessed using a paired t-test and is conveyed with * &lt;0.05, **&lt;0.01, *** &lt;0.001, **** &lt;0.0001.</SMALL></P><H2>References</H2><OL><LI><SMALL>Andrews SC, Robinson AK, Rodrı́guez-Quiñones F. <I>Bacterial iron homeostasis.</I> FEMS Microbiol Rev. 2003;27(2):215–37.</SMALL></LI><LI>Massé E, Gottesman S. <I>A small RNA regulates the expression of genes involved in iron metabolism in Escherichia coli.</I> Proc Natl Acad Sci U S A. 2002 Apr 2;99(7):4620–5.</LI></OL></DIV></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>