<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Sheffield_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Sheffield/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>A template page</TITLE></P><DIV class="container"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI class="dropdown" id="li-home"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>HOME</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>INTRO</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>PROJECT</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>TEAM</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>NOTES</P></TD></TR></TBODY></TABLE></CENTER></LI><LI class="dropdown"><CENTER><TABLE id="navtable"><TBODY><TR><TD><P>JUDGING</P></TD></TR></TBODY></TABLE></CENTER></LI><DIV id="shield"><DIV class="container"><DIV class="heading"><TABLE><TBODY><TR><TH><DIV class="heading-title"><H1 style="padding:0px">PROTOCOLS</H1></DIV></TH></TR></TBODY></TABLE></DIV><DIV class="jumbotron"><P><FONT size="30" style="text-align:left"> A. WET LAB </FONT></P><H2>I.Recipes  </H2><P><B><U> I.a Agarose gel (50ml of 1% gel)</U></B>
1.	Weight 0.5 g of agarose 
2.	Dissolve in 50 ml of 1x TAE buffer 
3.	Microwave until completely dissolved 
4.	Cool down to ~50<SUP>o</SUP>C (doesn't burn when you hold it in your hand) 
5.	Add 5ul of SybrSafe 
6.	Pour the gel into the casting stand 
7.	Cover the casting stand with foil to protect it from light (SybrSafe is sensitive to light)
8.	Wait for the gel to solidify 
9.	Load the samples (5-10ul of sample + 1-2ul of Loading Dye)
10.	Load 7-10ul of 2-log ladder 
11.	Run the gel at 110 V for ~45 min 
12.	Use lower gel percentage for larger samples (plasmids etc.) and higher gel percentage for smaller samples (~200-400 bp PCR products)</P><P><B><U>I.b. Buffers </U></B></P><P><U><B>I.b.1. 50X TAE Buffer (1 liter)</B></U> 
1.	242 g Tris-base 
2.	18.61 g EDTA 
3.	57.1 ml glacial acetic acid 
4.	 Water top up to 1L 
</P><P><U><B>I.b.2. 1X TAE Buffer (1 liter)</B></U> 
1.	Measure 20 ml of 50X TAE buffer 
2.	Add 980 ml dH2O </P><P><B><U>  I.c. Growth media </U></B></P><P><B><U> I.c.1. Liquid Broth (LB) Medium – 250ml </U></B> 
1. The following ingredients have been added to a 250ml bottle:  
-    2.5 g Tryptone 
-    1.25 g Yeast Extract
-    2.5 g NaCl
-    250ml dH2O 
2. Loose the lid of the bottle 
3. Secure the lid with autoclave tape
4. Autoclave
5. Store the media at 4ᴼC (i.e. in the fridge)
</P><P><B><U> I.c.2. LB agar (250 ml ~ 10 plates) </U></B> 
1. Add the following ingredients in a 250ml bottle:  
-    2.5 g Tryptone  
-    1.25 g Yeast Extract  
-    2.5 g NaCl  
-    3.75 g agar  
-    250ml dH2O  
2. Loose the lid of the bottle  
3. Secure the lid with autoclave tape  
4. Autoclave  
5. Pour plates next to a Bunsen burner (if add antibiotics to the LB agar, antibiotics should be added only when the temperature of the flask becomes below 60ᴼC) 
6. Wrap plates in foil and store them at 4ᴼC (i.e. in the fridge)
</P><P><B><U> I.c.3. Chrome azurol (CAS) plates agar </U></B> 
CAS agar was prepared with a mixture of 90 ml Y minimal agar and 10 ml CAS mix which were prepared and autoclaved separately (Schwyn and Neilands, 1987).  Following cooling to ~45 °C, the mixtures were mixed slowly ( with a pipette) and swirled until homogenous. <U>Y minimal agar</U> 
1.The agar was prepared by dissolving the following ingredients in 50 ml pure water: 
-    0.169 g of sodium glutamate (Sigma)
-    0.3 g Tris base (Fisher Scientific)
-    0.1 ml of MgSO<SUB>4</SUB>.7H<SUB>2</SUB>0 (10 % w/v)
-    0.1 ml of CaCl<SUB>2</SUB>.6H<SUB>2</SUB>O (22 % w/v) 
-    0.1 ml of K<SUB>2</SUB>HPO<SUB>4</SUB>.3H<SUB>2</SUB>O (22 % w/v) 
2. pH was adjusted to 6.8 
3. 1.5 g agar was then added to the medium and autoclaved <U>CAS mix</U> 
For preparation of 100 ml CAS mix, 10 ml of 1 mM FeCl<SUB>3</SUB>.6H2O (dissolved in 10 mM HCl) and 60.5 mg of CAS powder were dissolved in 50 ml of pure water giving a final volume of 60 ml. 72.9 mg of hexadecyltrimethylammonium bromide (HDTMA) was dissolved in 40 ml pure water and both solutions were mixed with constant stirring giving a total of 100 ml CAS mix solution. This dark purple solution was autoclaved and stored at room temperature in the dark. Mixing 10 μl of this solution with 90 μl Y minimal agar would give a final concentration of 10 μM FeCl<SUB>3</SUB>. To prepare a high iron solution, 10 ml of 6 mM FeCl<SUB>3</SUB>.6H2O (dissolved in 10 mM HCl) was used making a 60 μM FeCl<SUB>3</SUB> deep blue CAS mix solution. 
</P><P><B><U> I.c.6. SOC medium (200ml) </U></B> 
1.   	Add 1g yeast extract 
2.   	Add 4 g tryptone 
3.   	Add 2ml of 1M NaCl 
4.   	Add 0.4ml of 1M KCl 
5.   	Add 4ml of 0.5M Mg SO<SUB>4</SUB>
6.   	Add 2ml of 1M MgCl<SUB>2</SUB>
7.   	Top up with dH2O -up to 200ml 
Adjust the pH to 7.5 
8.   	Autoclave+ cool down
9.   	Add 4ml of 20% glucose solution 
</P><P><B><U> I.c.7. M9 + agar </U></B> 
1.   	Add 180ml dH2O 
2.   	Add 20 ml M9 salts 
3.   	Add 3g agar 
4.       Autoclave samples + cool down 
5.   	Add 0.4 ml 1M MgSO<SUB>4</SUB>
6.   	Add 0.4 ml 100mM CaCl<SUB>2 </SUB>
7.   	Add 2 ml 50% glucose 
8.   	Add 0.1 ml thiamine (1% stock) 
9.   	Add 0.2 ml chloramphenicol (50mg/ml)
10.   Pour plates 
</P><P><B><U> I.c.8. Defined medium </U></B><U>Autoclavable ingredients:</U> 
-MOPS-8.3625g
-Tricine-0.7125g
-NH<SUB>4</SUB>Cl-0.5g
-K<SUB>2</SUB>SO<SUB>4</SUB>-550µl (0.5M stock solution)
-CaCl<SUB>2</SUB>-5µl (100mM stock solution)
-MgCl<SUB>2</SUB>-525µl (1M stock solution)
-NaCl-5ml (1M stock solution)
-K<SUB>2</SUB>HPO<SUB>4</SUB>-2.6375ml (0.5 M stock solution)
-Casamino Acids-2g
-Trace metals-1ml (1000x stock solution)
-Distilled Water-981ml<U>Optional ingredients:</U> 
-FeCl3	600µl (10mM stock solution) <U>Non-autoclavable ingredients:</U> 
-Sterile Glucose	8ml (25% solution)
-Sterile Thiamine	1ml (1% solution)
</P><H2>  II.Protocols for cloning </H2><P><B><U> 
II.a. Introduction - strains used </U></B><B><I>Escherichia coli</I> Top10 </B>
-naturally competent 
          	-has streptomycin resistance<B><I>Escherichia coli</I> DH5α </B>
-naturally competent 
-have several mutations that eliminate restriction endonucleases that might  
degrade plasmids <B><I>Escherichia coli</I> BL21 </B>
-strain used for protein overexpression 
-T7 expression system <B><I>Escherichia coli</I> W3110 </B>
- wild-type strain (wild-type corresponding to the mutant JC28 strain)<B><I>Escherichia coli</I> JC28 </B>
- mutant strain- lacks entC gene
-does not produce enterobactin </P><P><B><U> II.b. Introduction - plasmids used</U></B><U>pSB1C3 </U>
-iGEM plasmid for constitutive gene expression
 -has RFP cloned into its multiple-cloning site
-selectable marker- chloramphenicol<U>pBSKII</U>
-plasmid for constitutive gene expression
-selectable marker- ampicillin  
pUC18
-high copy number plasmid 
-selectable marker-ampicillin <U>pET15b</U>
 -plasmid used for gene overexpression 
-selectable marker- ampicillin 
-plasmid has a His Tag<U>pET28a </U>
-plasmid used for gene overexpression 
-selectable marker-kanamycin </P><P><B><U> III.c. Working concentrations for the antibiotics used</U></B> 
-Ampicillin (Amp)-100 µg/ml
-Chloramphenicol (Cm)-30 µg/ml
-Kanamycin (Kan)-50 µg/ml
-Streptomycin (Str)25 µg/ml</P><P><B><U> II.d. Making competent cells</U></B><B>Prepare:</B>
*Cool in a freezer: Falcon tubes, CalC<SUB>2</SUB>, CaCl<SUB>2</SUB>-15% glycerol 
*Cool in –80C: 15x Eppendorf tube for each strain 

1. Inoculate single colony of each strain in 5-10 ml of LB (+Antibiotic) in 50 ml Falcon tube; Grow   overnight at 37°C 
2. Use 0.5 ml to inoculate in 50 ml LB (pH 7.2). Grow 2-4h to OD ~0.4-0.5  
3. Transfer cells to ice cold 50ml Falcon  
4. Cool the cells down on ice for 20 min and spin down at 4C, 5000 rpm for 10 min 
5. Resuspend gently in 10 ml ice-cold 100mM CaCl<SUB>2</SUB>  
6. Incubate on ice for 20 min 
7. Spin down as in step 4 
8. Discard supernatant and resuspend cells in 5 ml ice-cold CaCl<SUB>2</SUB>, cool for 10 min, spin down 
9. Discard supernatant and resuspend cells in 1.5 ml ice-cold CaCl<SUB>2</SUB>-15% glycerol 
10. Aliquot 100ul into 1.5ml ice-cold Eppendorf tubes 
11. Snap freeze and store in -80°C </P><P><B><U> II.e. Plasmid minipreps</U></B>
 
0.Centrifuge 5-10 ml of overnight culture at maximum speed for 10 minutes 
*** Overnight cultures for plasmid minipreps are set up by transferring a colony of bacteria in 10ml LB containing the corresponding antibiotic 
1.	Resuspend pellet in 250 µl resuspension buffer (P1 buffer)
2.	 Mix resuspension with 250µl lysis buffer (P2 buffer) and invert tubes 5 -  7 times 
3.	Add 350µl neutralization buffer (N3 buffer) and invert the tubes 5 -  7 times 
4.	Centrifuge samples for 5 min at 13000 rpm 
5.	 Transfer supernatant to the provided spin column (take care not to disturb the pellet) 
6.	Centrifuge the column for 1 min at 12000 rpm 
7.	Discard the liquid that has run through the column (liquid waste) 
8.	Add 500 µl wash buffer (PE buffer) 
9.	Centrifuge samples for 1 min at 13000 rpm 
10.	Discard the liquid that has run through the column (liquid waste) 
11.	Repeat again steps 9, 10 and 11 
12.	Perform one additional centrifugation step (without adding any buffer) for removing residual ethanol 
13.	Transferred the dried column to a fresh Eppendorf tube 
14.	 Add with 35µl  elution  buffer  in the center of the column membrane 
15.	Incubate the columns with elution buffer for 2 min at room temperature 
16.	Elute plasmids by centrifugation for 2 min at 13000 rpm 
17.	Store samples in the freezer (-20°C) </P><P><B><U> II.f. Transformation </U></B>
1.	Thaw 100ul of competent cells on ice 
2.	Add 0.5-1 ul of desired plasmid (conc. 130-30 ng/ul)
3.	 Mix gently by flicking the tube 
4.	Leave on ice for 30 min  
5.	Heat shock in 42°C for 60s  
6.	Transfer directly onto ice and leave for 5 min 
7.	Add 200ul SOC media into each tube 
8.	Grow in a shaker for 1h, 37<SUP>o</SUP>C, 200 rpm 
9.	Plate 50-100ul of cells onto agar plates containing the appropriate antibiotic
10.	Grow overnight at 37°C </P><P><B><U> II.i. II.j. Restriction digestion</U></B>

1. The following reagents have been added to each reaction tube: 
15 MiliQ H2O 
-2ul 10X restriction digestion buffer (10Xcut smart or 10X Fast digest)
-1ul Enzyme 1 
-1ul Enzyme 2 
-2ul DNA (up to 1 ug)
2.Incubate samples at 37°C for 20 min 
3.Inactivate restriction enzymes- incubate samples at 80<SUP>o</SUP>C for 20 min 
4.Store samples in the –20°C freezer </P><P><B><U> II.h. Ligation </U></B>
***T4 DNA ligase and buffers from ThermoFischer have been used 

1. The following reagents have been added to each tube:
-14ul MiliQ H2O 
-2ul T4 ligase buffer 
-2ul plasmid prep (~0.04 pmol DNA) 
-1ul construct suspension (~0.12 pmol DNA) 
-1ul T4 ligase 
2. Incubate samples at room temperature for 2h 
3. Store ligated products in the -20°C freezer
</P><P><B><U> II.i. Genomic DNA extraction</U></B><B>II.l.1. Using a boiling procedure </B>
1.	Resuspend a colony from a plate or spin down overnight cultures and resuspend in ~50-100ul water 
2.	Boil the cells for 15 min in 100°C heating block 
3.	Spin down the cell debris for ~3 min at 13,000 rpm 
4.	Take out ~2ul of supernatant and use as a PCR template<B>II.l.2. Using DNeasy kit </B>
 
Specifications provided in the DNeasy kit (QUIAGEN) Handbook have been followed </P><P><B><U> II.j. PCR </U></B><B>All reactions have been set up as following: </B>
1.Phusion Master Mix -25µl  
2. 10uM FWD primer -2.5µl  
3. 10uM REV primer -2.5µl  
4.Template DNA 	~2µl  
5.Nuclease free water -up to 50µl  
6.DMSO (optional) 3% -1.5 µl 
 *** DMSO can be used during PCR reactions where the template DNA is GC rich or is known to form secondary structures  <B>Cycling conditions used: </B>
Initial denaturation -98°C; 30s 
30 cycles: 
1) denaturation -98°C; 10s 
2) annealing -58°C; 30s 
3) extension -72°C; 30s 
Final extension -72°C; 7 min 
HOLD -4°C </P><P><B><U> II.k. PCR clean-up (DNA Purification by Centrifugation)</U></B>
Specifications provided in the Wizard SV Gel and PCR Clean-Up System (Promega) have been followed.
</P><H2>     III. Analysing gene expression </H2><P><B><U> III.a. Cell lysis (sonication)</U></B>
***Cells have been cultured overnight in 5ml of appropriate growth media <U>Controls: </U>
1.500ul were taken out of the sample and put into separate Eppendorf tubes (non-lysed controls- soluble + insoluble fractions) 
2.Samples were centrifuged for 10 min at 4000rpm 
3.Supernatant was discarded 
4.Pellet was resuspended in 50ul MiliQ <U> Lysis protocol: </U>
1.The remaining 4500ul were centrifuged for 10min at 4000rpm 
2.Supernatant was disacarded 
3.Cells were resuspended in 1ml MiliQ 
4.Cells were sonicated for 3 min on ice (5s sonication-30s incubation on ice and so on) 
5.Samples were spun down for 8min at 13,000 rpm 
6.50ul of supernatant have been transferred to a new tube 
</P><P><B><U> III.b. Running of an SDS Page Gel</U></B><U>III.b.1 Sample preparation: </U>
1.Laemmli buffer has been prepared (1200ul total- 60ul BME and 1140ul lBME buffer) 
2.50ul Laemmli buffer have been added to the 50ul sample 
3.Samples were boiled at 95oC for 10 min 
4.Samples were spun at 13,000rpm for 10 min 
5.10ul sample were loaded into the gel<U> III.c.2. Running the gel: </U>
LifeScience precast gels have been used together with the buffers provided. Gels have been run at 200 V for 1h and 30min.<U>III.c. Staining of SDS Page Gels (Cromassie Blue)</U>
1.Gels have been placed in Cromassie stain and have been microwaved for 30s, followed by a 3h incubation with the stain
2. The Cromassie stain has been removed 
3. Gels have been placed in de-stain solution (H<SUB>2</SUB>O, methanol and acetic acid in a 50:40:10 volume ratio) and microwaved for 30s followed by a 2h incubation 
4.Destain solution has been removed and steps 3-4 have been repeated until the bands became clear (usually 2-3 times)</P><H2>  IV. Ferene assay  </H2><P> 

1.	Pelleting and resuspending cells in fresh solution <B>a.	For lower concentrations of cells: </B>
i.	Transfer 1ml of overnight culture to a microcentrifuge tube and spin at 13000 in the microcentrifuge for 3-5 minutes 
ii.	Discard the supernatant (first pour off majority then remove the remaining supernatant with P200)
iii.	Resuspend pellet in desired mix of ascorbic acid and double distilled water (Milli Q) up to 1ml e.g. 300ul of 10mM ascorbic acid and 700ul of water <B> b.	For higher concentrations of cells: </B>
i.	Spin down 5-10ml of overnight culture in the falcon tubes in the larger centrifuges for 10 minutes
ii.	Discard the supernatant (first pour off majority then remove the remaining supernatant with P200)
iii.	Resuspend pellet in desired mix of ascorbic acid and double distilled water (Milli Q) up to 1ml e.g. 300ul of 10mM ascorbic acid and 700ul of water, then transfer to a microcentrifuge tube
2.	Take a bucket of ice and your samples to the sonicator
3.	Clean the nozzle of the sonicator with ethanol and attach, ensuring it is tightened with a wrench 
4.	Put one pair of ear defenders on the handle of the outside of the door and ensure anyone in the room is wearing ear defenders when sonicating
5.	Place nozzle inside sample and sonicate at amplitude 16 for 5 seconds 
6.	Put samples on ice for 30 seconds then sonicate sample for a further 5 seconds
7.	Repeat steps 5 and 6 three more times (four cycles total)
8.	Clean nozzle with ethanol between samples
9.	Place on the hot block at 80˚C for 1 hour to denature protein, when removing samples from hot block use tongs
10.	Spin samples for 5 minutes at 13000rpm in the microcentrifuge
11.	Take 800ul of supernatant and mix with 200ul of 10mM ferene solution in a micro cuvette
12.	Incubate for 10-20 minutes
13.	Measure absorbance at 593nm in the spectrophotometer</P><H2>  V. Growth curves </H2><P><B><U>The day before:</U></B> 
1.Autoclave enough 250ml conical flasks
2.Make up enough liquid media 
3.Pour enough agar plates <B><U>On the day; In the flow hood:</U></B>
4.Innoculate 50ml liquid media in each 250ml conical flask 
5.Take 1ml and measure OD600 for time zero, record 
6.Take 1ml, do serial dilutions and spot 3 10µl spots on the right plate    segment
7.Put conical flasks in the shaker, leave for an amount of time 
8.Leave plate to dry for 30 min, then put in the fridge upside-down 
9.Repeat the above steps for each time-point 
10.Put plates from the fridge into the 37°C room at the end of the day<B><U>On the day after:</U></B>
11.	Take plates from the 37°C room, count colonies </P><H2>  VI. Testing <I>gfp</I> expression </H2><P><B><U>Day 1:</U></B>
1.	Set up overnights in defined medium (5 ml) + appropriate antibiotic (Ampicillin) + 5uM Fe; of following colonies: 
o	3x W3110 
o	3x JC28 
o	3x W3110 Str-GFP 
o	3x Jc28 Str-GFP 
o	3x W3110 Med-GFP 
o	3x JC28 Med-GFP 
o	1x blank control 
2.	Allow growth for 16-18h in 37C shaking at 220 rpm <B><U>Day 2:</U></B>
3.	Set up 96 well plates using your overnight colonies as following: (where 1-3 are technical repeats of the same biological sample). 100 ul of overnights were added to each well. Iron (final conc. 100uM) was added at the end of preparation using multichannel pipette to increase the efficiency and speed of the process 

4.	Repeat experiment 3 times, 1 time for each biological repeat 
5.	To read the fluorescence use excitation of 485 nm and emission of 535 nm 
6.	Samples were measured every 2 minutes (plus reading time), measuring both fuorescence, as well as OD600 to check for any growth of the cells (this is to ensure that increade in fluorescence comes from increased gfp expression rather than from cell growth)
7.	Data was normalised for the OD600 value and mean of technical repeats for each sample was calculated. Values were imported into GraphPad and multiple paired t-tests were performed to show any statistical differences between the samples (with or without iron addition)</P><P><FONT size="30" style="text-align:left"> B. DRY LAB  </FONT><B><U> I.PDMS Microfluidic Chip Protocol</U></B><B><U>I.a. Making PDMS </U></B>
1.	Weigh 20g of the base silicone elastomer and 2g of the curing agent in a weighing dish. This amount of base and curing agent produces 1 petri dish of cured PDMS. A 10:1 ratio is the recommended ratio for use in cell culture 
2.	Mix the base and curing agent for approximately 5 minutes until a thin mixture is obtained

3.	Place the weighing dish in a vacuum desiccator and close the lid

4.	Make sure the pipe from the desiccator is attached to the correct outlet of the pneumatic pump before turning on the pneumatic pump. Check that the lid is closed by lifting it. The lid should not come off the desiccator

5.	Leave to de-gas for 30 minutes
6.	Turn the pneumatic pump off and remove the pipe from the desiccator
7.	Allow the bubbles formed on the surface of the uncured PDMS to pop. If they are not popping, tap the weighing dish against the table to make them pop<B><U>I.b.Making the Mould </U></B>
1.	Design the channels using Inkscape. The entire design should fit onto a glass slide of dimensions 75mm by 25mm 
2.	Save the files as both an svg and a dxf file 
3.	Load the dxf file into the robocutter program 
4.	Use the robocutter to cut out the stickers (50 μm thick). Be sure to cut out multiple stickers of the same design to account for robocutter inaccuracy and stickers being broken while peeling 
5.	Peel the sticker off using tweezers and stick it at the bottom a petri dish to form the mould <B><U>I.c. Making the Chip</U></B>
1.	When both the mould and the PDMS are made, pour the PDMS into the mould to a depth of around 3mm 
2.	Place the petri dish into the vacuum desiccator again and leave to de-gas for 10 minutes to remove air bubbles 
3.	Take the petri dish out of the desiccator and allow the air bubbles formed to pop 
4.	Cover the petri dish and leave it to cure fully for 2 days 
5.	Alternatively, cure it for 1 day, peel the PDMS from the petri dish and place it in an oven (100 degrees Celsius) for 15 minutes to cure fully 
6.	Once the PDMS has fully cured, cut out the designs using a scalpel 
7.	Punch holes in the PDMS for the inlet and outlet chambers and where necessary 
8.	Press the PDMS firmly onto a glass slide making sure that the roof of the channels is not in contact with the glass slide. For best results, plasma treatment is recommended prior to this step. See below for details on plasma treatment <B><U>I.d.Plasma Treatment</U></B>
1.	Place a glass slide and the PDMS in a plasma machine, making sure that the side to be bonded (i.e. the side with the channels) is facing up 
2.	Follow instructions on using the plasma machine 
3.	Reduce pressure to around 0.2atm and then increase pressure to 1atm before switching on plasma generation  for 30 seconds. Once 30 seconds are up, the machine will automatically turn off 
4.	Remove the PDMS and glass slide from the plasma machine 
5.	Press the PDMS and glass slide firmly together but make sure that the roof of the channels is not in contact with the glass slide <B><U>I.e.Testing</U></B>
1.	Use a micropipette to add food colouring into the inlet. Recommended volume is 5 μL for a 2 mm hole. Do not push liquid into the inlet. Instead, form a drop of liquid at the micropipette tip and gently tap the drop onto the inlet 
2.	Allow the food colouring to flow within the channels (average time is around 20 minutes) and record the results observed </P></DIV></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>