Team:Sheffield/Notebook
A template page
HOME
INTRO
PROJECT
TEAM
NOTES
JUDGING
NOTEBOOK
CLONING OF HEMERYTHRIN
Week 1:
-Literature research
-Decided an experimental outline
-In silico design of hemerythrin constructs
-Synthetic genes and primers required for cloning have been ordered
Week 2:
-Make competent cells for the E.coli strains that we used during our project- TOP10, JC28 and W3110
-Tested the efficiency of transformation in these strains
-Prepared buffers, growth media and pouring agar plates for the experiments we were planning to carry out during the following weeks
-Several experiments have been performed in order to monitor the growth of the wild type (W3110) and siderophore-deficient mutant (JC28)
Week 3:
-Ensured we had sufficient amounts of plasmid for further cloning experiments:
transformed the pSB1C3, pUC18 and pBSKII plasmids into TOP10 competent cells
made plasmid mini-preps from these transformed cells
-Characterised the plasmid stocks obtained (Nano Drop and agarose gel)
-Characterised the genotype of wild-type (W3110) and mutant (JC28) strains:
-genomic DNA extractions have been performed
- entC gene has been amplified- PCR
-PCR products have been analysed- agarose gel
Constitutive hemerythrin expression
Week 4:
-Synthetic hemerythrin genes as well as primers required for amplifying these genes have arrived
-Synthetic genes have been amplified (PCR) and the success of the PCR reactions has been tested (PCR and Nano Drop)
-Synthetic genes as well as plasmid backbones (pSB1C3 and pBSKII) have been digested using appropriate restriction enzymes
-Ligation reactions have been set up
-Ligated plasmids have been transformed in TOP10 competent cells
Week 5:
-Plasmid mini-preps have been made from the TOP10 cells transformed with constitutively expressed hemerythrin genes (both pSB1C3+hemerythrin and pBSKII+hemerythrin)
-Small fractions of harvested pSB1C3+hemerythrin plasmids have been digested with appropriate restriction enzymes; restriction products have been run through an agarose gel and screened for the presence of desired inserts
-Plasmids containing desired inserts have been transformed in W3110 (wild-type) and JC28 (siderophore-deficient) E.coli strains
-Plasmid mini-preps have been made again from transformed W3110 and JC28 strains
Week 6:
-Plasmid mini-preps (from W3110 and JC28 strains transformed with pSB1C3+hemerythrin) have been digested with appropriate restriction enzymes
- Small fractions of harvested pSB1C3+hemerythrin plasmids have been digested with appropriate restriction enzymes; restriction products have been run through an agarose gel and screened for the presence of desired inserts
-Plasmids containing the appropriate inserts have been sent for sequencing
-Sequences have been analysed
Week 7:
- Measured absorbance at 500nm in overnight cultures of Dcr and Mc in JC28 and W3110 strains and Td in W3110 strain in chloramphenicol LB
-No change in colour has been detected between hemerythrin encoding mutant, wild-type strains and negative controls (no hemerythrin expression) suggesting that protin was not expressed
-Tried to figure out what was going on– realised that there were several methylation sites at adjacent positions to the promoters used
-Decided that we should replace the promoters and remove the methylation sites
-New promoters have been ordered
Week 10:
-Promoters have been cut out from the linear hemerythrin constructs
-Promoterless hemerythrin genes have been cloned into pSB1C3
-pSB1C3+promoterless hemerythrin plasmids have been cloned into TOP10
-Very few colonies of transformant TOP10 have been observed on every plate
Week 11:
-Cloning of promoterless hemerythrin genes into pSB1C3 has been carried out again, allocating longer incubation times for digestion and ligation reactions to take place
-Ligated plasmids have been transformed in TOP10
-Plasmids have been re-harvested from TOP10 and screened for the presence of the insert (samples were digested with appropriate restriction enzymes and run through an agarose gel)
-The new promoters have been cloned into the pSB1c3+promoterless hemerythrin genes containing desired inserts; these plasmids have been transformed into DH5α cells
Week 12:
-pSB1C3+new promoter+promoterless hemerythrin genes plasmids have been harvested from DH5α cells and screened for the presence of desired inserts
-Whole cell lysates of the transformed cells have been made and the soluble and insoluble protein fractions have been separated
-Soluble and insoluble protein fractions have been run through an SDS-PAGE gel
-Gel was stained with Cromassie Blue
-SDS-PAGE gels shown that hemerythrin genes have not been successfully expressed
Weeks 13-17:
-Cloning and submitting iGEM BioBricks
Overexpression of hemerythrin
Week 4:
-Synthetic hemerythrin genes as well as primers required for amplifying these genes have arrived
-Synthetic genes have been amplified (PCR) and the success of the PCR reactions has been tested (PCR and Nano Drop)
Week 6:
-PCR amplifications of the hemerythrin genes – experiment failed due to issues with the primers
-in silico design of new primers
Week 7:
-All 3 hemerythrin genes (Dcr, Td and Mc) have been amplified and inserted into each of the 2 overexpression plasmids- pET15b and pET28a
-Ligated plasmids have been transformed in TOP10 competent cells, but unexpected cell morphologies have been observed, therefore the cloning procedure has been repeated again (week 8)
Week 8:
-Hemerythrin genes have been cloned again in pET15b and pET28a
-Ligated plasmids have been transformed in TOP10
-Overexpression plasmids have been re-harvested from transformant TOP10 cells
Week 9:
-Harvested plasmids have been checked for the presence of the desired inserts (samples have been digested with appropriate restriction enzymes and run through an agarose gel)
-Plasmids containing the desired insert have been transformed in BL21 cells
Week 11:
-Confirming that the transformed BL21 cells contain plasmids with the appropriate inserts (samples were digested with appropriate restriction enzymes and run through an agarose gel)
-IPTG induction of hemerythrin genes
-Characterisation of protein expression profile (SDS-PAGE)
CLONING OF RyhB-GFP
Week 6:
-in silico design of the GFP-RyhB construct; synthetic genes as well as primers required for cloning have been ordered
Week 8:
-Primers and synthetic GFP-RyhB constructs have arievd
-PCR amplification of the constructs
-pSB1C3 and amplified GFP-RyhB constructs have been digested with appropriate restriction enzymes
-Ligated plasmids have been transformed in TOP1
-Following antibiotic selection no bacterial growth was observed so all experiments had to be repeated (week 9)
Week 9:
-Attempts to clone RyhB-GFP constructs into pSB1C3, but cloning experiments did not work due to issues with the forward primers
-Designed new primers for amplifying these constructs
Week 10:
-New genes and primers ordered have arrived
-PCR of the new RyhB-GFP constructs
Week 11:
-PCR amplified GFP-RyhB constructs have been cloned into both pSB1C3 and pBSKII
-Ligated plasmids have been transformed into TOP10
-Transformed TOP10 cells have been screened for the expression of GFP
-Plasmids from transformed cells have been harvested, checked for the presence of desired inserts and transformed into W3110 and JC28
Week 12:
-Fluorescence –based assays have been carried out in order to analyse the intensity of GFP when W3110 and JC28 are incubated with low and high iron concentrations
Weeks 13-17:
-Further characterisation of the RyhB-GFP constructs
-Cloning and submitting iGEM BioBricks
IRON MEASUREMENTS
Week 1:
-Assessing sensitivity range of ferene for measurement of iron concentrations
Week 2:
-Assessing cell lysis and iron reduction/dissolution techniques for intracellular iron concentration measurement
-Ascorbic acid selected as reducing agent, effect of pH on ferene absorbance investigated
-Different cell lysis methods were performed on E.coli cells and absorbance compared. Sonication found to be more effective than lysis with SDS
Week 3:
-Assessing cell lysis and iron reduction/dissolution techniques for intracellular iron concentration measurement
-Attempted to optimise pH to produce higher absorbances
-Using ferene assay on cells grown in different media, produced low absorbances
Week 4:
-Assessing cell lysis and iron reduction/dissolution techniques for intracellular iron concentration measurement
-Attempted to optimise pH to produce higher absorbances
-Produced accurate calibration curve
-Acetate buffer investigated and found to be ineffective
Week 6:
-Literature survey of alternative intracellular iron measurement techniques
Week 7:
-Developed ICP sample preparation protocol
Week 8:
-Attempted ICP sample preparation, issues with overnight cultures and CFU count
Week 9:
-Attempted ICP sample preparation, issues with overnight cultures and CFU counts
Week 11:
-Investigation of using concentration step in ferene assay. Some issues with precipitates forming
-Investigated adding tonB supernatant to LB cultures to observe any effect of siderophores that may be present
Week 12:
-ICP samples prepared, measurement performed by university analytical chemistry service
CAS PLATES
Week 1:
-Siderophore production in wild-type (W3110, TOP10) and mutant E. coli has been tested using CAS plates- the initially tested protocol did not work
Week 6:
-CAS plates from week 1 were tested again; instead of adding WT E.coli cells, EDTA iron chelators have been added as a positive control
-Presence of yellow halos suggested that our plates could change colour in the presence of iron chelators, but in the case of E.coli cells the approach was not successful as cells probably require additional carbon sources  -Multiple other protocols have been tested
Week 10:
- CAS plates have been poured again; a new protocol has been followed
-inoculated overnight cultures of W3110, JC28, TOP10 and TonB on the CAS plates
-following a couple of days, yellow have been observed, suggesting that this approach was successful
GROWTH CURVES
Week 3:
-Growth curves of JC28 and W3110 have been carried out; cells have been grown in LB media
Week 6:
-Growth curves of JC28 and W3110 have been carried out; cells have been grown in defined media as well as M9 media
MODELLING
Week 1:
-Read up on modelling techniques used to model biological systems: differential and stochastic methods
Week 2:
-Compiled a list of objectives more modelling - the results of these objectives would potentially aid in building a set of specifications for the device and inform the project about the efficacy of our project
-Modelled the rate of enterobactin binding to iron using equations found by reading up on  single-ligand-single-receptor binding kinetics
Week 3:
-Divided the objectives for modelling into smaller sections that can be individually modelled, i.e. built a block diagram for modelling
-Started modelling Iron Uptake of bacteria via the FepA mechanism using the Hill equation
Week 4:
-Finished modelling Iron Uptake
-Created a spreadsheet for all the parameters that we would potentially need for modelling
Week 5:
-Spoke with our supervisors Coca and Visakan about how we could improve the modelling - they advised on the importance of linking all the different parts of the model together
Week 6:
-There has been a change in our reporter system from Hemerythrin to Fur, so the model was modified to reflect this
-Started working on modelling the binding of ferric enterobactin to lipocalin by looking parameters like dissociation constants (FeEnt Binding)
Week 7:
-Found an error in the code for iron uptake and corrected it
-Tried, but was unsuccessful in, converting the matlab script into a python script
-Linked the code for FeEnt Binding to the rest of the model
Week 8:
-The stock solution has 2 receptors binding to the same ligand, so we spoke to Craven to gain some advice on how to model that
Week 9:
-Coded a function that calculates ferric enterobactin concentration from given concentrations of ferric ions, enterobactin and EDTA chelator (Prep. Stock)
-Wrote a summary of all the things that have been done by modelling thus far
Week 10:
-Discussed with an advisor on how to structure the modelling section of the Wiki, created a block diagram to aid with the potential explanation for the Wiki
-Read up on how to model the Fur-dependent RyhB-mediated RNAi
-Cleaned up codes to make them more efficient
Week 11:
-Read up on how to model RNAi
Week 12:
-Read up on how to model RNAi
-Started writing up a draft for modelling
Week 13-17:
-Modelled RNAi and modified Iron Uptake before linking all the separate parts of the model together
-Validated the model using experimental data from the wet lab
-Tested the model using different promoters, found the optimum enterobactin concentration to use and found how long the system would take to produce results
-Finished writing up for modelling, including creating the appropriate figures and uploaded it all onto the Wiki
DESIGNING THE DEVICE
Week 1:
-Read up on past iGEM teams’ device and brainstormed about what kind of device is most appropriate for our project
Week 2:
-Compiled a list of questions that need to be answered in order to build a list of specifications for the device
Week 3:
-Microfluidics - spoke to Dr Cecile Perrault about a potential microfluidics design for our system
Week 4:
-Microfluidics - went for a health-and-safety induction in the microfluidics lab; learnt how to build a chip, built and tested our first batch of chips
-Larger Device - Moved on to looking at making a bigger device in case microfluidics didn’t turn out right. Device should have a small capsule inserted that contains all the ingredients
Week 5:
-Microfluidics - designed chips for testing length and width, built and tested how length and width affected flow rate
-Larger device - Made the first cardboard model of the device complete with two chambers
Week 6:
-Microfluidics - read up of the designs used for mixing and came up with a few mixing designs
-Larger device - Designs for  automated device were drawn up
Week 7:
-Microfluidics - analysed the data from testing length and width, drew up mixing design 2 in Inkscape, built mixing design 2 in the lab and tested them
-Larger device - No progress made
Week 8:
-Microfluidics - finished testing mixing design 2, took some pictures of design 2, wrote up the microfluidics protocol
-Larger device - Designs for  automated device scrapped due to the complexity of the design and the level of control required. Instead, using a capsule that does the main steps and then a simple chamber that measures the GFP levels
Week 10:
-Microfluidics - Discussed the structure of the microfluidics page of the Wiki
-Larger device - Drafted up the designs for the basic building of the measurement chamber. Put up the designs up on 3D modelling software
Week 11:
-Microfluidics - wrote a draft of the content for the Wiki
-Larger device - Made some changes to the designs of the parts, assembled together virtually to check everything fitted
Week 12:
-Microfluidics - added pictures and diagrams into the draft
-Larger device - No progress made
Weeks 13-17:
-Microfluidics - designed, built and tested mixing design 3; finished writing up content for the Wiki; uploaded content and figures onto the Wiki page
POLICES AND PRACTICES
Week 2:
-Interviewed RAPID panel
Week 5:
-Interviewed Professor Milton Wainwright
-Interviewed Simon Rushton
Week 7:
-Interviewed Andrew Rycroft
Week 8:
-Interviewed Sarah Thompson
-Interviewed David Oglesby
Weeks 13-17:
-22/09/16 - STEM event
-14/10/16 - iGEM edu day
WIKI AND MEDIA DESIGN
Week 1:
-The poster for the iGEM European Meeting in Paris has been designed
-The iGEM Sheffield 2016 logo has been designed
Week 2:
-The leaflets for the iGEM European Meeting in Paris have been designed
-Learnt about bootstrap, much progress was made after
Weeks 4-7:
-Establishing a template for the WiKi and deciding on the main sections
-Decided on using the main black and gold after the university sports colours
-Worked out how to upload css and javascript pages to the website
Week 8:
-The poster for the iGEM UK Meetup (London)  has been designed
Week 9:
-London visit so not much progress made. Pushed out the first home page of the website with text about us
Week 10:
-Deciding which team member is writing each section
-Played around with a second version of how to display the content. Decided to organise the work by adding sub menus when clicking each icon
-Edited navbar so that it now works on mobile
-Writing-up WiKi content
Week 11:
-Organising the the content and making sure that we have all the stuff for it. Redesigned some icons for the website and made a template for the content sections with intractable banners
Week 12:
-Before we broke up, all the parts of the website that needed writing were distributed to people so that on our break from the lab we could write it up without worrying about experiments
-New (epic) home page made for the website
-All templates for standard content and sub menu’s uploaded and fixed on the website
Weeks 13-17:
-Started gathering up everybody's work (and forcing people to write stuff that hadn’t done what they were supposed to)
-Uploaded the content
-There was a lot crying, uploading, more crying, stuff not working, some progress, even more crying and suffering, then the Wiki was finally done.  Our team Wiki leader was never the same again…..
