Team:Imperial College/Results
Project
Description
Results
Proof
Basic Parts
Composite Parts
Modelling
Overview
Single Cell Model
Population Model
Software Tools
Notebook
Lab book
Protocols
Medal Criteria
Brainstorm
Human Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
Team
Team
Attributions
Results
Click on the circuit to discover our results for each section of the G.E.A.R or click on the cell culture below to see our cell growth characterization experiments!
Communication Module
Quorum is one of the key modules of our G.E.A.R. system. It allows the different cells within a population and across populations to communicate with each other. The quorum module provides a way for the cells to detect the density of each population, which is essential to allow the RNA-based logic to operate downstream. We have examined four quorum sensing systems, which have all been fully modelled and two of the four quorum systems have been fully characterised, so that future iGEM teams can use and expand on communication modules.
Key Achievements
Characterisation of the Las quorum sensing system, including crosstalk characterisation.
Characterisation of the Rhl quorum sensing system, including crosstalk characterisation.
Generated transfer curves for all characterisation experiments.
Creation of a group of characterised and orthogonal quorum parts that other groups can use to implement communication modules into their projects.
Overview
In order to create a circuit that maintains a fixed ratio of two bacterial populations, we set out to engineer a system for bidirectional communication: our two populations of E. coli needed to detect both their own density and the density of the other population. Quorum sensing is a naturally occurring mechanism that certain strains of bacteria use to regulate gene expression in response to their population density. These bacteria secrete signalling molecules such as N-acyl homoserine lactones, or AHLs, which bind to transcription factors to alter gene expression. Because we were growing two distinct E. coli populations in co-culture, we need two unique quorum sensing systems, so that each population density can be reflected in the concentration of a different AHL.
Figure 1: Quorum response proteins and their associated AHLs.
Our Approach
We chose to examine 4 candidate quorum sensing systems: Cin, Las, Lux, and Rhl. Our aim was to identify two quorum sensing systems with a high degree of orthogonality that could be used for bidirectional bacterial communication. Additionally, we hope to provide future iGEM teams with a set of characterised constructs for each of the four systems, so that they have multiple options when it comes to bacterial communication systems. The Cin transcriptional activator, CinR, is activated by 3O-C14 AHL. LasR, LuxR, and RhlR are activated by  3O-C12 AHL, 3O-C6 AHL, and C4 AHL respectively (See Figure 1). These AHLs are produced by different inducer proteins. For each of the transcriptional activators for the four systems, we created two new composite parts (See Figure 2). The first part type is the AHL response system, which includes the transcriptional activator under the control of a constitutive promoter along with the appropriate quorum promoter. In these constructs, a gene inserted downstream would be expressed in the presence of the AHL ( BBa_1893000 , BBa_1893002 , BBa_1893004 , and BBa_1893006 ). The second part type is a reporter construct which includes the transcriptional activator under the control of a constitutive promoter with the appropriate quorum promoter upstream of a GFP expression construct ( BBa_1893001 , BBa_1893003 , BBa_1893005 , and BBa_1893007 ). This part allows us to characterise the response of our construct in the presence of various concentrations of AHL.
Figure 2: A schematic representation of one of our basic plasmid types, the response construct. It includes the transcriptional activator (Response) and the quorum promoter (pQ). This construct allows genes placed downstream of the quorum promoter to be transcribed when the cells are induced with the appropriate AHL.
Figure 3: This is the characterisation construct, which we used to generate transfer curves relating AHL concentration to fluorescence of GFP.
Experimental Design
Experiments were carried out by measuring the fluorescence output from each of the constructed devices by inducing cell cultures with various concentrations of AHL molecules. Top 10 cells harbouring plasmids of constructed resporter devices were cultured to the exponential phase and then transferred to 96-well microplates where they were induced with appropriate AHL concentrations. The induced cell cultures were grown in the microplates and the fluorescence signal and absorbance value (OD 600) were monitored over time using a microplate reader. The reported values for the normalised fluorescence represent the recorded values 180 minutes after AHL induction. The normalised fluorescence was calculated by dividing fluorescence values by absorbance values and correcting for LB autofluorescence.
We needed to characterise the response of the construct to different concentrations of AHL so that we could use the data in our model to predict how the system could function. If one transcription activator were to be sensitive to a much lower concentration of AHL, we could use the model to inform what adjustments should be made to balance the system, such as changing the strength of the constitutive promoter upstream the transcriptional activator.
It is important that the two quorum sensing systems in the circuit are as orthogonal as possible, as crosstalk would result in the incorrect AHL activating growth suppression in the wrong population. As a result, we chose to characterise the crosstalk in 4 candidate systems. We measured the absorbance and fluorescence of the cells after treating them with different concentrations of each of our 4 AHLs, to determine if the AHLs were activating the response protein and resulting in transcription of GFP. The characterisation for Las and Rhl has been completed, and we are continuing our work on Cin and Lux.
Results
The concentration ranges of AHLs required for activation in each quorum sensing system were calculated to be 100nM-100uM for Rhl and 100pM-10uM for Las. The activation ranges were compared between the quorum sensing systems in order to determine if they were a suitable pair to be used together. Las and Rhl were found to differ by a 1,000-fold difference. These differences fed back into the modelling in order to predict what promoter strength ahead of the AHL synthesiser genes would balance the circuit.
Our crosstalk characterisation data shows that LasR can be activated by 3O-C14, indicating that Las and Cin lack the orthogonality required for our circuit. RhlR can be activated by 3O-C6 AHL, and exhibits only slight activity at high concentrations of 3O-C12 AHL. This suggests that Rhl and Lux are not sufficiently orthogonal either. Based on their lack of significant crosstalk, Rhl and Las appear to be the most promising quorum sensing systems for our project. Cin and Rhl, as well as Las and Lux, could also potentially work together, but crosstalk experiments for CinR and LuxR must first be completed and analysed.  Furthermore, Cin and Rhl have been previously been reported to work as orthogonal quorum sensing systems (Chen et al, 2015).
Figure 3: Characterisation of the Rhl response device ( BBa_K1893003 ). (A) Transfer function curve of normalised fluorescence against cognate inducer C4-AHL concentrations. (B) Heat map of normalised fluorescence of RhlR-GFP system over a range of AHL concentrations: (i) Binding of RhlR-GFP to its cognate AHL (C4 AHL). (ii) Binding of RhlR-GFP to 3 non-cognate AHLs (3O-C6 AHL, 3O-C12 AHL, 3OH-C14 AHL). (C) Transfer function curves of normalised fluorescence against non-cognate inducer AHL (C4 AHL) concentrations to investigate inducer AHL crosstalk: (i) C6-AHL (3O-C6 AHL) of the Lux system (ii) C12-AHL (3O-C12 AHL) of the Las system (iii) C14-AHL (3O-C14 AHL) of the Cin system. Experiments were performed in E. coli Top10 cell strain cultured at 37°C. Normalised fluorescence was calculated by dividing fluorescent signal by cell density (OD600). Fluorescence measurements were recorded at 180 minutes. Reported values represent the mean normalised fluorescence value from 3 technical repeats and error bars represent standard deviation of these.
Figure 4: Characterisation of the Las response device ( BBa_K1893001 ). (A) Transfer function curve of normalised fluorescence against cognate inducer C12-AHL (3O-C12 AHL) concentrations. (B) Heat map of normalised fluorescence of RhlR-GFP system over a range of AHL concentrations: (i) Binding of RhlR-GFP to its cognate AHL (C4 AHL). (ii) Binding of RhlR-GFP to 3 non-cognate AHLs (3O-C6 AHL, 3O-C12 AHL, 3OH-C14 AHL). (C) Transfer function curves of normalised fluorescence against non-cognate inducer AHL (3O-C12 AHL) concentrations to investigate inducer AHL crosstalk: (i) C4-AHL of the Rhl system (ii) C6-AHL (3O-C6 AHL) of the Lux system (iii) C14-AHL (3O-C14 AHL) of the Cin system. Experiments were performed in E. coli Top10 cell strain cultured at 37°C. Normalised fluorescence was calculated by dividing fluorescent signal by cell density (OD600). Fluorescence measurements were recorded at 180 minutes. Reported values represent the mean normalised fluorescence value from 3 technical repeats and error bars represent standard deviation of these.
What’s Next?
Complete characterisation of CinR.
Complete characterisation of LuxR.
Assembly of inducer constructs.
Genome integration of AHL synthesiser genes in order to minimise plasmid load.
Experience
The construct assembly for Las and Rhl was completed quickly, so the characterisation data was acquired fairly early. Cin and Lux, on the other hand, took quite a lot more effort. We had CinR synthesised, as the Biobrick part contained an LVA tag and was not available. Due to the size of the part, synthesis took quite a while, and we did not receive the plasmid until late September, pushing assembly and characterisation back. The DNA distribution included a construct containing LuxR, but it was downstream of the pTet promoter rather than an Anderson promoter. For this part, we used PCR to change the promoter, which took a few attempts to get right. Afterwards, we attempted to insert GFP for the characterisation of the part. However, the ligation failed repeatedly, and this system too was delayed in its characterisation. Fortunately, we had Las and Rhl characterised well enough to assemble them into our circuit.
Key Information Communication Module
Characterisation of the Las quorum sensing system, including crosstalk characterisation.
Characterisation of the Rhl quorum sensing system, including crosstalk characterisation.
Generated transfer curves for all characterisation experiments.
Creation of a group of characterised quorum parts that other groups can use to implement communication modules into their projects.
Comparator Module
The comparator module is the central processing unit of our Genetically Engineered Artificial Ratio (G.E.A.R.) system. We are utilising STAR (Small Transcriptional-Activating RNA) a novel RNA-logic technology that allows for rapid and robust regulation of gene transcription (Chappell et al, 2015). We have submitted it as our best basic part and it is a completely new addition to the BioBrick Registry. RNA-based logic systems are a rapidly expanding field, and we believe that expanding this library of tools is essential for advancing synthetic biology. We hope that our design will encourage future iGEM teams to implement STAR in their circuits as a versatile, highly orthogonal, and effective method of gene regulation.
Key Achievements
Characterisation of STAR-mediated target gene activation in various conditions.
Designing of a new anti-sense RNA sequence complementary to STAR, Anti-STAR.
Characterisation of quorum-sensing regulated STAR.
Expansion of the BioBrick Registry library by addition of a new RNA-logic toolset.
Overview
Our engineered cells need a way to compare the population sizes of the different cell species in a co-culture. Therefore, we designed a comparator module that compares the amount of each type of quorum signal (AHL), which is proportional to each population size, and responds by regulation of a growth-inhibition gene in the growth module.
STAR is is the first instance of small bacterial RNAs being used to directly activate transcription. STAR binds a sense target RNA sequence, the pAD1 plasmid attenuator sequence, which is fused upstream of the gene of interest. In our circuit, it is upstream of a growth inhibiting gene. The pAD1 plasmid attenuator sequence forms an intrinsic hairpin structure, preventing RNA polymerase from transcribing the gene and thus acting as a terminator of transcription. However, when STAR is produced, it binds to the target sequence and interferes with the hairpin formation, allowing transcription of the downstream gene.
To regulate STAR activity, we designed an inhibitory RNA sequence (Anti-STAR). The Anti-STAR sequence is identical to a portion of the pAD1 attenuator sequence, and is therefore also complementary to STAR. Anti-STAR has a higher affinity for STAR than the pAD1 attenuator does, so, when transcribed, Anti-STAR sequesters STAR and prevents it from interacting with the target sequence and activating transcription.
Our STAR approach offers significant advantages over traditional protein-based regulatory systems, including:
→ Low metabolic burden: the RNA is functional after transcription, avoiding the resource-intensive translation steps.
→ Portability: trans-acting RNA based regulation is found throughout the bacterial kingdom (as well as other kingdoms) and is not host-specific.
→ Tight Regulation: In the absence of STAR there is very low baseline expression of the downstream gene, while in the presence of STAR the expression is activated  96-fold.
→ Ease of design: Watson-Crick base pairing avoids the need for structural protein-protein interactions, which are difficult to predict and design.
→ Reduced lag in circuits: RNA degradation is rapid, reducing the lag in the comparison function, and offering short time response for our comparator circuit
Our Approach
We chose to place one quorum-regulated promoter upstream of the gene for the STAR sequence and a different quorum-regulated promoter upstream the gene for the Anti-STAR sequence. Consequently, the amount of transcription of the two sequences is dependent on the concentration of each quorum sensing molecule, respectively. The ratio of STAR and Anti-STAR is therefore proportional to the ratio of the two types of quorum sensing molecules.
If both population sizes are the same,  equal amounts of STAR and Anti-STAR should be transcribed in both cells and, therefore,  STAR and Anti-STAR should sequester one another. As a result, STAR is unable to activate the transcription of the growth regulatory protein and both populations should grow as normal (i.e. no growth inhibition).
When population sizes are imbalanced, in the cells whose population is too large, more STAR should be transcribed than Anti-STAR, so that   STAR can activate transcription of the  growth inhibition gene. The expression of this gene slows down the growth of the faster growing population, allowing the slower population to catch up in size.
We used our model to predict what ratio of the two quorum sensing molecules(AHLs) would result in equal levels of STAR and Anti-STAR, since different quorum sensing systems have different activation ranges. This information is used to determine what promoters should be placed upstream of the AHL synthesiser genes. By setting the efficiency of synthesiser transcription, we can set the amount of AHL produced per cell such that when population sizes are equal, the ratio of AHL concentrations allows for equal expression of STAR and Anti-STAR. The same mechanism can be tuned to define different population ratios in situations where equal population numbers are not desired.
Experimental Design
We generated a two-plasmid system for characterisation experiments. The first plasmid contains the STAR sequence downstream of a constitutive Anderson promoter and followed by the t500 transcriptional terminator on a high-copy plasmid (Figure 1). The second plasmid is a reporter plasmid that contains  the superfolder GFP (SFGFP) gene with a ribosome binding site immediately downstream of the STAR-target (pAD1 plasmid attenuator) sequence (Figure 2). The SFGFP coding sequence is under the control of a constitutive Anderson promoter and also has its own TrrnB terminator.
Figure 1: A schematic representation of the first plasmid. This has the basic STAR part (STAR-t500) under control of the constitutive promoter j23119.
Figure 2: A schematic representation of the reporter plasmid with the coding sequence of superfolder GFP (SFGFP).
We conducted a series of STAR characterisation experiments.  First, we characterised STAR activity 30 and 37 degrees Celsius by using a plate reader to record the fluorescence of SFGFP over 200 minutes in cells with both plasmids, as well as in cells with just the reporter plasmid to determine to what degree STAR was able to activate SFGFP transcription. We used the same design to characterise the activation range of STAR under the control of the quorum promoters pLas and pRhl. Finally, we characterised the deactivation range of STAR when both STAR and Anti-STAR are induced by varying concentrations of Las and Rhl AHLs.
Initially, we characterised the STAR system in terms of fold activation of GFP expression from the reporter plasmid in the absence and presence of STAR molecules. For this experiment, Top 10 E. coli cell lines were co-transformed with either the reporter plasmid and a plasmid with the J23119 promoter (no STAR) or with the reporter plasmid and the J23119-STAR plasmid. Cell cultures of these two cell lines were grown in microplates and well fluorescence was monitored over time using a microplate reader. The fluorescence signal from each well was normalised by dividing with the O.D. 600 value of that well. This gave the normalised fluorescence value for that cell line (FI/Abs). To account for cell autofluorescence, DH10B cells (similar to Top10) were used to determine background normalised fluorescence value.
Next, an experiment was carried out to investigate STAR system functionality at different temperatures. RNA elements functionality can be strongly depended on temperature and thermodynamics. In the case that we needed to use the STAR system under different circumstances (co-culture with B. subtilis that grows better at 30 degrees Celcius) we characterised the STAR system at 30 degrees Celsius cell culture condition. Cell cultures of the previously mentioned cell lines were grown for 5 hours in exponential phase. Then, the cell cultures were normalised at O.D. 600 = 0.4 and the fluorescent signal for each condition was recorded.
Results
The results displayed in Figure 3A show that the expression of SFGFP is greatly increased in the presence of STAR over the course of 200 minutes of culturing. The graph from Figure 3B represents the normalised fluorescence once the growth data (optical density) has been taken into account. The data from this graph shows that SFGFP expression is increased more than 17-fold at 100 minutes of culturing, when STAR is present. This indicates that STAR is successful in preventing the pAD1 plasmid attenuator from interfering with SFGFP transcription. The data suggests that STAR is a promising tool for the regulation of the growth repressing gene in our circuit.
Figure 3: Characterisation of STAR system in TOP10 E. coli cells.  (A) Normalised fluorescence monitored over time for cell lines incorporating the STAR system in the absence or presence of transcribed STAR molecules  (B) Normalised endpoint fluorescence (100 minutes) for cell lines in the absence or presence of STAR molecules. We used the two-plasmid system described in the Experimental Design for characterisation experiments involving STAR. For the absence of STAR condition, the plasmid did not include STAR sequence but just the J23119 promoter. Normalised fluorescence was calculated by dividing fluorescent signal by the O.D.600 value of the culture. Background was determined by the use of DH10B cells with no plasmid transformed. Error bars represent standard deviation from 3 technical repeats.
Experimental results showed that STAR system activated transcription at various levels at different temperatures. Figure 4A shows that SFGFP expression at 30°C increases significantly in the presence of STAR molecules as determined by the cell culture fluorescence. Figure 4B shows that SFGFP expression at 37°C increases significantly in the presence of STAR molecules as determined by the cell culture fluorescence, in agreement with our previous results. Figure 4C shows that the activation SFGFP expression attainable by the use of the STAR system was much greater at 37°C (26.7 fold) rather than at 30°C (5.5 fold). The results showcase important considerations that need to be taken into account when the STAR system is used under different culture conditions, and can be incorporated into the models of the system to inform design.
Figure 4: Characterisation of STAR system in TOP10 E. coli cells  at different temperatures. (A) Cell culture fluorescence at 30°C (B) Cell culture fluorescence assay at 37°C. (C) Fold activation SFGFP expression in presence of STAR. We used the two-plasmid system described in the Experimental Design for characterisation experiments involving STAR. For the absence of STAR condition, the plasmid did not include STAR sequence but just the J23119 promoter. The autofluorescence background control used is E. coli Top 10 cells with no reporter plasmid. Error bars represent standard deviation from 3 technical repeats.
What’s Next?
Characterise quorum-sensing regulated STAR and Anti-STAR.
Characterise the deactivation curve of STAR in a construct where STAR and Anti-STAR are controlled by two different promoters that respond to different AHL.
Run all the characterisation experiments by using Las and Cin as the two bi-directional and orthogonal quorum sensing systems.
Experience
Even though they are very small sequences, STAR and Anti-STAR with their associated terminator were both easy to amplify by PCR. Thus, the assembly of STAR and Anti-STAR with the quorum-regulated promoters did not cause any problems.  However, when we attempted to ligate STAR and Anti-STAR, both under the control of the J23119 promoter, into the same plasmid, using the BioBrick assembly method, we were unsuccessful even though we tried a number of times to optimise various steps in the protocol. We think that perhaps a recombination event was occurring in the cells as the sequence of the promoter is duplicated. We did not experience any major difficulties when assembling STAR and Anti-STAR under the control of the quorum-regulated promoters.
Key Information Comparator Module
Characterisation of STAR-mediated target gene activation in various conditions.
Designing of a new anti-sense RNA sequence complementary to STAR, the Anti-STAR.
Characterisation of quorum-sensing regulated STAR and Anti-STAR.
Expansion of the BioBrick Registry library by addition of a new RNA-logic toolset.
Growth Regulator Module
The growth module is the last of the three modules constituting our Genetically Engineered Artificial Ratio (G.E.A.R.) system. If the STAR system determines that one population's quorum signal is too high, it needs to activate transcription of a protein that arrests the growth of that population. We investigated a range of growth-regulation targets, including the established techniques of antibiotics and auxotrophy. We decided on a novel growth regulatory system, utilising the phage gene Gp2, and the growth regulation achieved is rapid, effective and reversible, and has many advantages over existing growth control methods.
Key Achievements
Characterised a novel, effective and reversible growth control method for E. coli.
Improved the pBad-AraC construct by adding a reverse terminator.
Characterised our pBAD-AraC construct in top10 cells.
Overview
The final part of our circuit is the growth-regulation system. This needs to be a protein that inhibits the growth of the cell when its expression is activated by STAR. Our target gene had to have a number of features in order to prevent the system from losing balance and to minimise lag time. It had to inhibit growth rapidly without killing the cell, and allow the cell to recover normal growth conditions after the STAR system no longer activates the circuit.
Our Approach
We settled on four target genes:
Cat, a gene encoding the enzyme Chloramphenicol acetyltransferase, which is an enzyme that confers resistance to the antibiotic chloramphenicol. We decided to use chloramphenicol because it is a bacteriostatic rather than a bactericidal antibiotic. When using Cat, chloramphenicol would be added to the growth medium.
LeuB - a gene that codes for an enzyme in the biosynthetic pathway of leucine in E. coli- the gene has been used before as a control method in co-culture. When using LeuB, we would need to use a strain that is auxotrophic for leucine and a growth medium lacking leucine.
Under the conditions described above, expression of LeuB and Cat would both be required for normal growth, and so in the context of our system, growth inhibition would be achieved by STAR triggering production of a protein that inhibits their expression, e.g. using an inverter.
Gp2 is a gene from the E. coli bacteriophage T7 phage, which slows down cell growth by binding reversibly to the E. coli RNA polymerase complex, thus inhibiting transcription. T7 phage infection is characterised by the arrest of bacterial growth, and Gp2 has been mooted as a potential antimicrobial agent.
Gp0.4 is another T7 phage gene which inhibits growth by  binding to the FtsZ ring during mitosis, preventing cytokinesis (the final stage of cell division where the two daughter cells separate).
To characterise these genes, we placed them under the control of the arabinose-inducible pBAD promoter. This was done because constitutive expression of growth-inhibiting proteins would have made characterisation extremely difficult. After several attempts at cloning, we were able to obtain pBAD-Gp2 and pBAD leuB constructs. Our characterisation efforts then focussed on Gp2 because it allows direct inhibition of growth without the need for an inverter.
Gp2 is also an ideal protein for our purposes because:
It has a small coding sequence (>200bp) and therefore it is transcribed faster than LeuB or Cat, which both have a coding sequence of about 1kb.
Gp2 does not require the use of an inverter, which would slow down the circuit significantly. Therefore Gp2 is faster than both LeuB and Cat, as they require the implementation of an inverter to fit in our G.E.A.R. system.
Auxotrophy systems, such as the LeuB system, require the creation of knockout strains. Gp2 does not depend on any auxotrophy system and it can therefore be used directly on the species of interest. This could potentially simplify the building step and minimize the costs of iGEM teams projects.
Using chloramphenicol resistance reduces the amount of plasmids the cell can be transformed with, and also causes issues with regards to antimicrobial resistance. On the contrary, Gp2 does not cause any of those problems.
Cat and LeuB both require special media conditions, reducing the versatility of the system. Gp2-based system does not need any special medium to function properly, thus making it easier to work with.
These reasons, identified during our reflexivity process, lead us to choose Gp2 as our growth regulating protein in our G.E.A.R. system. Reflexivity is the core aspect of our integrated human practices, that has helped us identify problems and solutions through our project. For more information on our reflexivity process, please visit our integrated human practices page!
Experimental Design
We characterised the activity of the araBAD operon by placing a GFP reporter under control of the pBAD promoter, and quantifying the amount of fluorescence produced by varying concentrations of arabinose. Once we had obtained an activation range, we moved on to characterising the activity of Gp2.
For all characterisation experiments, transformed cells were grown from stocks overnight, and subcultured in the morning. After three to four hours, (when the cultures were in exponential phase) the cultures were diluted to 0.05 O.D. and aliquoted into a 96 wells plate, then treated with varying concentrations of arabinose. Repression experiments involved treating the culture with arabinose for two hours, before aliquoting into a plate and adding a range of concentrations of glucose.
Results
Our results show that Gp2 effectively inhibits E. coli growth in a reversible manner (Figure 3). At the highest level of induction on the high copy psB1A2 plasmid, the cell can recover from the growth inhibition on its own after approximately eight hours, but this was not observed at lower induction strengths. Our results also indicate that once Gp2 production is switched off, the cells can recover back to a normal growth rate within an hour.
Figure 1: Characterisation of the pBAD-GFP construct with varying concentrations of L-arabinose. ( BBa_K1893017 ).Experiments were performed in E. coli Top10 cell strain cultured at 37°C, which were diluted to 0.05 O.D. and inoculated with L-arabinose at the 0 minute timepoint. Normalised fluorescence was calculated by dividing fluorescent signal by cell density (O.D. 600). Reported values represent the mean normalised fluorescence value from 3 technical repeats and error bars represent standard deviation
Figure 2: Transfer function of normalised GFP fluorescence against concentration of L-arabinose
Figure 3: Growth inhibition of Top10 cells by induction of Gp2 by L-arabinose. Experiments were performed in E. coli Top10 cell strain cultured at 37°C, which were diluted to 0.05 O.D. and inoculated with L-arabinose at the 0 minute timepoint. O.D. was recorded at 600nm. Reported values represent the mean normalised O.D. for three repeats, with error bars representing standard deviation.
Figure 4: Recovery of growth by Top10 cells after arabinose-induced pBAD operon was switched off by glucose-mediated catabolite repression. 100uM of L-arabinose was added to the culture 2 hours before the 0 minute timepoint and D-glucose was added at the 0 minute timepoint. . pBAD-GFP controls indicate that the pBAD operon was switched off at approximately after approximately 250 minutes.  Experiments were performed in E. coli Top10 cell strain cultured at 37°C, which were diluted to 0.05 O.D., which was recorded at 600nm. Reported values represent the mean normalised O.D. for three repeats, with error bars representing standard deviation.
What’s Next?
Investigate how long  it takes for E. coli to escape the growth repression by Gp2.
Investigate how to prevent cheater cells (mutants who inactivate the growth control circuit), potentially by creating an  operon containing Gp2 and an essential metabolic enzyme.
Gp0.4 was abandoned because of ligation issues, but further work would include an attempt to characterize it again. Gp0.4 has an advantage over Gp2 in that it doesn't affect protein production, and so will not unbalance the circuit (Gp2 will also prevent production of proteins that are necessary for the circuit to function). However, it does appear that Gp2 plays a potential role in damping the ratio oscillations in our circuit (modelling link).
Experience
It took several attempts to clone genes into the pBAD construct, and we were unable to successfully insert two of four of our genes into the pBAD construct. We had some success with increasing the insert:vector ratio (NEB). Gp2 was extremely easy to work with, and worked as expected first time.
Key Information Growth Regulating Module
Characterised a novel, effective and reversible growth control method for E. coli.
Improved the pBad-AraC construct by adding a reverse terminator.
Characterised our pBAD-AraC construct in TOP10 cells.
A.L.I.C.E Growth Experiments
Overview
Another issue with co-culture beyond that of ratio control that we identified was that it was difficult to know under what conditions to culture the two microbial strains together. To rectify this, we have created a central repository to store information about optimal media conditions for co-culture. You can read about the A.L.I.C.E. database on our software page. We wanted to characterize growth data for different species of microorganisms in monoculture -  with a view to understanding how well the microbe will grow in a co-culture experiment. We also characterised optimum conditions for several co-cultures.  Finally, we were generously provided with monoculture data by other iGEM teams, which you can read about on our collaboration page.
Our Approach
We designed a high throughput method of characterising monoculture data using an automated robotic system, in collaboration with the DNA Synthesis and Construction Foundry, managed by SynbiCITE.
We used this system to characterise 7 common microbial cultures used in synthetic biology:
E.coli (BL21, MG1665, Top 10, Turbo)
B. subtilis (168)
Yeast ( S.Cerevisiae BY4741, P. Pastoris X33)
The experiments were run using a range of media compositions in order to find compositions that were amenable to multiple cell types. This was done with the aim of identifying the optimal media composition for co-cultures.
Experimental Design
For monoculture growth characterisation, cultures were grown overnight using standard growth media, and then diluted to 0.1 OD in mixed media. We then loaded the robot in the foundry and ran it over 12 hours. This was done multiple times and the data was analysed to obtain growth curves.
For co-culture growth characterization, cultures were grown overnight using standard growth media, and then diluted to 0.1OD in mixed media. The co culture combinations were then inoculated and a 96 well plate was set up for the flow cytometer by diluting the co-cultures 100 fold in PBS. The co culture was then incubated and flow cytometry readings were performed. These measurements were repeated every hour to obtain growth data for the co-culture.
Results
Full characterisation for the 7 species selected were obtained for monoculture growth characterisation. Our results suggested that the E.coli species grew better at 37°C than 30°C, B. Subtilis grew better at 37°C   and both species of Yeast didn’t grow at 37°C (infection is the likely reason for growth in the BY4741 sample at 37°C.).
Figure 1. Monoculture growth characterization at 30°C and 37°C
Co-culture growth characterization is ongoing for Yeast - E.coli, Yeast - B.subtilis and E.coli - B.subtilis. This will be uploaded onto A.L.I.C.E and will feature at the Jamboree!
Key Information ALICE Growth Experiments
Characterised 7 different strains under various temperature and media conditions in monoculture.
Used flow cytometry to provide data on optimal conditions for  three variants of co-culture. Full characterisation will be uploaded to A.L.I.C.E.
References
Chappell, J., Takahashi, M. & Lucks, J. (2015) Creating small transcription activating RNAs. Nature Chemical Biology. [Online] 11 (3), 214-220.
Chen, Y., Kim, J., Hirning, A., Josi, K. and Bennett, M. (2015). Emergent genetic oscillations in a synthetic microbial consortium. Science, 349(6251), pp.986-989.
Email: igem.imperial2016@gmail.com
