<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Imperial_College_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Imperial College/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Description">Project </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Composite_Part">Composite Parts</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Model">Modelling </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/SingleCell">Single Cell Model</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/GRO">Population Model</A></LI><LI><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Software">Software Tools </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Notebook">Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Notebook">Lab book</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Experiments">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Medals">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/BrainStorm">Brainstorm</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Human_Practices">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Team">Team </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI></DIV></DIV><DIV class="bg-primary"><SECTION><DIV class="col-lg-10 col-centered container"><P> This year our team created a mathematical representation of our Genetically Engineered Artificial Ratio (G.E.A.R.) system. This representation, or model, informed the wet lab on the timescales of the three modules of GEAR: communication, comparator and growth regulator. It also helped develop assembly strategies thanks to a thorough analysis of parameter scans and sensitivities. After a series of experiments in the wet lab, we were able to finesse our parameters making them more accurate. This process, presented below, can be seen as a continuous feedback between the wet lab and the dry lab of Ecolibrium.</P><P>
We developed our models with two main goals in mind:
<OL style="font-size:18px;"><LI>We wanted our model to be able to aid the team in the wetlab. Especially for optimizing the assembly process and balancing the circuit.</LI><LI>We wanted to develop an in silico version of our circuit at single cell and population level. This allowed us to test the viability of the system in both normal and abnormal conditions. It also allows us to plan future experiments and used for the system.</LI></OL></P><P>
The following pages will show how we implemented modelling approaches to achieve our goals.
<SPECIALH4><A href="https://2016.igem.org/Team:Imperial_College/SingleCell">The Single Cell Model</A></SPECIALH4><UL style="font-size:18px;"><LI>Communication Module - We constructed the four quorum systems that we considered viable choices for our system (cin, rhl, lux and las)to allow us to directly compare the expected behaviour and plan our growth module experiments accordingly.
</LI><LI>Comparator Module - We used RNAstruct developed by Matthews Lab to help aid the development of the ANTISTAR. We modeled STAR and ANTISTAR behaviour in silico 
</LI><LI>Growth Regulator Module - We modelled 4 different growth regulator systems in silico in order to assess the speed and effectiveness of each case. 
</LI></UL></P><SPECIALH4><A href="https://2016.igem.org/Team:Imperial_College/GRO">
Population Model </A></SPECIALH4><UL style="font-size:18px;"><LI> Matlab population model with GP2 growth regulation in two population 
</LI><LI>We used the GRO programming language to model a simplified version of our circuit with GP0.4 growth regulation into two populations of E.Coli.  
</LI></UL><DIV class="col-lg-10 col-centered"><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="SpeAcc"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#SpeAcc-collapse" aria-expanded="false" aria-controls="SpeAcc-collapse"><DIV><DIV class="col-md-11">Species</DIV></DIV></A></H4></DIV><DIV id="SpeAcc-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="SpeAcc"><DIV class="panel-body"><SPECIALH4> Table 1: Universal Species Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Species Name</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD>C4</TD><TD>AHLs produced by RhlI synthase </TD></TR><TR><TD>C14</TD><TD>AHLs produced by CinI synthase</TD></TR><TR><TD>g_C4R</TD><TD>Copy number for plasmid backbone housing C4R </TD></TR><TR><TD>mC4R</TD><TD>mRNA for RhlR</TD></TR><TR><TD>C4R</TD><TD>RhlR transcriptional regulator  </TD></TR><TR><TD>C4Comp</TD><TD>Complex by C4 AHL and RhlR transcriptional regulator </TD></TR><TR><TD>g_STARC4Comp</TD><TD>Complex formed between plasmid backbone for STAR (pRhl) and c4 Comp  </TD></TR><TR><TD>g_C14R</TD><TD>Copy number for plasmid backbone housing C14R </TD></TR><TR><TD>mC14R</TD><TD>mRNA for CinR </TD></TR><TR><TD>C14R</TD><TD>CinR transcriptional regulator</TD></TR><TR><TD>C14Comp</TD><TD>Complex by C14 AHL and CinR transcriptional regulator</TD></TR><TR><TD>g_ANTISTARC14Comp</TD><TD>Complex formed between plasmid backbone for ANTISTAR (pCin) and C14 Comp</TD></TR><TR><TD>g_STAR</TD><TD>Copy number for plasmid backbone housing STAR</TD></TR><TR><TD>STAR</TD><TD>Short transcription activating RNA (STAR)</TD></TR><TR><TD>g_ANTISTAR</TD><TD>Copy number for plasmid backbone housing ANTISTAR</TD></TR><TR><TD>ANTISTAR</TD><TD>Anti Short transcription activating RNA (ANTISTAR) </TD></TR><TR><TD>STAR:ANTISTAR</TD><TD>STAR:ANTISTAR Complex</TD></TR><TR><TD>STAR_Target</TD><TD>Star Target plasmid (AD1 terminator)</TD></TR></TBODY></TABLE><SPECIALH4> Table 2: Unique Species for GP0.4 model Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Species Name</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD>g_GP0.4</TD><TD>Plasmid hosting STAR Target and growth regulating protein GP2</TD></TR><TR><TD>mGP0.4</TD><TD>mRNA for GP0.4</TD></TR><TR><TD>pre_GP0.4</TD><TD>Unfolded GP0.4</TD></TR><TR><TD>GP0.4</TD><TD>GP0.4 (Gene Product 0.4) sequesters FtsZ and stops cell division.</TD></TR><TR><TD>FtsZ</TD><TD>FtsZ is a protein that influences cell division</TD></TR><TR><TD>GP0.4:FtsZ</TD><TD>GP0.4:FtsZ Complex</TD></TR></TBODY></TABLE><SPECIALH4> Table 3: Unique Species for GP2 model Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Species Name</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD>g_GP2</TD><TD>Plasmid hosting STAR Target and growth regulating protein GP2</TD></TR><TR><TD>mGP2</TD><TD>mRNA for GP2</TD></TR><TR><TD>pre_GP2</TD><TD>Unfolded GP2</TD></TR><TR><TD>GP2</TD><TD>GP2 (Gene Product 2) sequesters RNA Polymerase </TD></TR><TR><TD>RNAP</TD><TD>RNA Polymerase</TD></TR><TR><TD>GP2:RNAP</TD><TD>GP2:RNAP Complex</TD></TR></TBODY></TABLE><SPECIALH4> Table 4: Unique Species for CAT and LeuB </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Species Name</TH><TH>Description</TH></TR></THEAD><TBODY><TR><TD>mTet</TD><TD>mRNA for Tetracycline</TD></TR><TR><TD>Tet</TD><TD>Tetracycline </TD></TR><TR><TD>pTet</TD><TD>Promoter that is repressed by Tetracycline</TD></TR><TR><TD>Tet:pTet</TD><TD>Tetracycline:pTet complex </TD></TR><TR><TD>mLeuB</TD><TD>mRNA for Leucine B</TD></TR><TR><TD>LeuB</TD><TD>Leucine B</TD></TR><TR><TD>mCAT</TD><TD>mRNA for Chloramphenicol Acetyltransferase</TD></TR><TR><TD>CAT</TD><TD>Chloramphenicol Acetyltransferase</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="ParaAcc"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#ParaAcc-collapse" aria-expanded="false" aria-controls="ParaAcc-collapse"><DIV><DIV class="col-md-11">Parameters</DIV></DIV></A></H4></DIV><DIV id="ParaAcc-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="ParaAcc"><DIV class="panel-body"><SPECIALH4> Table 5: General Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>Cell Volume</TD><TD>6.70E<SUP>-16</SUP></TD><TD>L</TD><TD>Neidhardt F.C. Escherichia coli and Salmonella: Cellular and Molecular Biology. Vol 1. pp. 15, ASM Press 1996.</TD></TR><TR><TD>Transcription rate</TD><TD>28 Nucleotides/s was used</TD><TD>28-89</TD><TD>Nucleotides/s</TD><TD>Vogel U, Jensen KF. The RNA chain elongation rate in Escherichia coli depends on the growth rate. J Bacteriol. 1994 May176(10):2807-13 p.2811 table 2</TD></TR><TR><TD>Translation rate</TD><TD>12 Amino acid/s was used</TD><TD>12-21</TD><TD>Amino acids/s</TD><TD>Bremer, H., Dennis, P. P. (1996) Modulation of chemical composition and other parameters of the cell by growth rate. Neidhardt, et al. eds. Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, p. 1559, Table 3</TD></TR><TR><TD>d_prot</TD><TD>Protein degradation</TD><TD>1.39E<SUP>-05</SUP></TD><TD>1/min</TD><TD>Biomolecular Systems by Del Vecchio</TD></TR><TR><TD>k_mat</TD><TD>Protein maturation rate</TD><TD>0.16</TD><TD>1/min</TD><TD>Megerle JA, Fritz G, Gerland U, Jung K, Rädler JO. Timing and dynamics of single cell gene expression in the arabinose utilization system. Biophys J. 2008 Aug95(4):2103-15. p.2106 right column bottom of paragraph</TD></TR><TR><TD>D</TD><TD>Dilution rate</TD><TD>0.048</TD><TD>1/min</TD><TD>A synthetic Escherichia coli predator–prey ecosystem</TD></TR><TR><TD>d_mRNA</TD><TD>mRNA degradation rate constant</TD><TD>0.139</TD><TD>1/min</TD><TD>Taken from 5 min mRNA degradation time</TD></TR><TR><TD>Copy number for 3K3 and 3C3</TD><TD>Copy number</TD><TD>20-30</TD><TD>molecules</TD><TD>http://parts.igem.org/Part:pSB3K3</TD></TR><TR><TD>Copy number for 1K3 and 1C3</TD><TD>Copy number</TD><TD>100-300</TD><TD>molecules</TD><TD>http://parts.igem.org/Part:pSB1C3</TD></TR></TBODY></TABLE><SPECIALH4> Table 6: Quorum Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k_C4</TD><TD>Rhl/C4 Production Rate Constant</TD><TD><UL><LI>8.00E<SUP>-07</SUP></LI><LI>5.36E<SUP>-13</SUP></LI><LI>5.36E<SUP>-22</SUP></LI><LI>3.22E<SUP>02</SUP></LI></UL></TD><TD><UL><LI>M/min</LI><LI>nMol/min</LI><LI>mol/min</LI><LI>molecules/min</LI></UL></TD><TD>Optimal tuning of bacterial sensing potential - Anand Pai &amp; Lingchong You</TD></TR><TR><TD>k_C14</TD><TD>Cin/C14 Production Rate Constant</TD><TD><UL><LI>5.00E<SUP>-08</SUP></LI><LI>3.35E<SUP>-14</SUP></LI><LI>3.35E<SUP>-23</SUP></LI><LI>2.00E<SUP>01</SUP></LI></UL></TD><TD><UL><LI>M/min</LI><LI>nMol/min</LI><LI>mol/min</LI><LI>molecules/min</LI></UL></TD><TD>Optimal tuning of bacterial sensing potential - Anand Pai &amp; Lingchong You</TD></TR><TR><TD>k_mC4R</TD><TD>Transcription rate of C4R</TD><TD><UL><LI>2.23</LI><LI>5.51E<SUP>-09</SUP></LI><LI>3.69E<SUP>-24</SUP></LI></UL></TD><TD><UL><LI>mRNA/min</LI><LI>nMol/min</LI><LI>Mol/min</LI></UL></TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_mC14R</TD><TD>Transcription rate of C14R</TD><TD><UL><LI>2.23</LI><LI>3.69E<SUP>-15</SUP></LI><LI>3.69E<SUP>-24</SUP></LI></UL></TD><TD><UL><LI>mRNA/min</LI><LI>nMol/min</LI><LI>Mol/min</LI></UL></TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_C4R</TD><TD>Translation rate of C4R</TD><TD><UL><LI>2.99</LI><LI>4.96E<SUP>-24</SUP></LI></UL></TD><TD><UL><LI>Protein/min</LI><LI>Mol/min</LI></UL></TD><TD>Calculated from Bionumbers average translation rate : Bremer et al</TD></TR><TR><TD>k_C14R</TD><TD>Translation rate of C14R</TD><TD><UL><LI>2.99</LI><LI>4.96E<SUP>-24</SUP></LI></UL></TD><TD><UL><LI>Protein/min</LI><LI>Mol/min</LI></UL></TD><TD>Calculated from Bionumbers average translation rate : Bremer et al</TD></TR><TR><TD>kf_C4Comp</TD><TD>C4 - C4R Complex formation</TD><TD><UL><LI>1.00E<SUP>08</SUP></LI><LI>2.50E<SUP>-01</SUP></LI></UL></TD><TD><UL><LI>1/M.min</LI><LI>1/complexes.min</LI></UL></TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kr_C4Comp</TD><TD>C4:C4R Complex dissassociation</TD><TD>10</TD><TD>1/min</TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kf_C14Comp</TD><TD>C4 - C4R Complex formation</TD><TD><UL><LI>1.00E<SUP>08&gt;/sup&gt;</SUP></LI><LI>2.50E<SUP>-01</SUP></LI></UL></TD><TD><UL><LI>1/M.min</LI><LI>1/complexes.min</LI></UL></TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kr_C14Comp</TD><TD>C4:C4R Complex dissassociation</TD><TD>10</TD><TD>1/min</TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kf_DimC4</TD><TD>C4:C4R complex dimerization</TD><TD>5.00E<SUP>07</SUP></TD><TD>M/min</TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kr_DimC4</TD><TD>C4:C4R complex reverse dimerization</TD><TD>1</TD><TD>1/min</TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kf_DimC14</TD><TD>C14:C14R complex dimerization</TD><TD>5.00E<SUP>07</SUP></TD><TD>M/min</TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>kr_DimC14</TD><TD>C14:C14R complex reverse dimerization</TD><TD>1</TD><TD>1/min</TD><TD>Dynamics of quorum sensing switch - Weber</TD></TR><TR><TD>d_C4</TD><TD>C4 degradation rate constant</TD><TD>2.21E<SUP>-04</SUP></TD><TD>1/min</TD><TD>Optimal tuning of bacterial sensing potential - Anand Pai &amp; Lingchong You</TD></TR><TR><TD>d_C14</TD><TD>C14 degradation rate constant</TD><TD>2.83E<SUP>-04</SUP></TD><TD>1/min</TD><TD>Optimal tuning of bacterial sensing potential - Anand Pai &amp; Lingchong You</TD></TR><TR><TD>d_C4R</TD><TD>C4R degradation rate constant</TD><TD>0.002</TD><TD>1/min</TD><TD>Optimal tuning of bacterial sensing potential - Anand Pai &amp; Lingchong You</TD></TR><TR><TD>d_C14R</TD><TD>C14R degradation rate constant</TD><TD>0.002</TD><TD>1/min</TD><TD>Optimal tuning of bacterial sensing potential - Anand Pai &amp; Lingchong You</TD></TR></TBODY></TABLE><SPECIALH4> Table 7: STAR Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>kf_gSTARC4Comp</TD><TD>gSTAR-C4 comple association</TD><TD>0.25</TD><TD>1/molecules.min</TD><TD>Assumed same as quorum complex formation</TD></TR><TR><TD>kr_gSTARC4Comp</TD><TD>gSTAR-C4 complex disassociation</TD><TD>10</TD><TD>1/min</TD><TD>Assumed same as quorum complex formation</TD></TR><TR><TD>kf_gANTISTARC14Comp</TD><TD>gANTISTAR-C14 complex association</TD><TD>0.25</TD><TD>1/molecules.min</TD><TD>Assumed same as quorum complex formation</TD></TR><TR><TD>kr_gANTISTARC14Comp</TD><TD>gANTISTAR - C14 complex disassociation</TD><TD>10</TD><TD>1/min</TD><TD>Assumed same as quorum complex formation</TD></TR><TR><TD>k_STAR</TD><TD>Basal Rate of STAR production (without C4:C4R induction)</TD><TD><UL><LI>1.91E<SUP>-01</SUP></LI><LI>3.08E<SUP>-23</SUP></LI></UL></TD><TD><UL><LI>mRNA/min</LI><LI>mol/min</LI></UL></TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_iSTAR</TD><TD>Rate of STAR production (after induction of C4:C4R)</TD><TD><UL><LI>18.57</LI><LI>3.08E<SUP>-23</SUP></LI></UL></TD><TD><UL><LI>mRNA/min</LI><LI>mol/min</LI></UL></TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_ANTISTAR</TD><TD>Basal Rate of ANTISTAR production (without C14:C14R induction)</TD><TD>1.91E<SUP>-01</SUP></TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_iANTISTAR</TD><TD>Rate of ANTI STAR production (after induction of C14:C14R)</TD><TD>18.57</TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>kf_STARANTISTAR</TD><TD>STAR:ANTISTAR complex formation</TD><TD>62</TD><TD>complexes/min</TD><TD>Calculated using DNA strand displacement (Winfree et al)</TD></TR><TR><TD>kr_STARANTISTAR</TD><TD>STAR:ANTISTAR complex dissociation</TD><TD>2</TD><TD>1/min</TD><TD>Calculated using DNA strand displacement (Winfree et al)</TD></TR><TR><TD>kf_STARAD1</TD><TD>STAR:AD1 Complex formation</TD><TD>62</TD><TD>complexes/min</TD><TD>Calculated using DNA strand displacement (Winfree et al)</TD></TR><TR><TD>kr_STARAD1</TD><TD>STAR:AD1 Complex dissociation</TD><TD>2</TD><TD>1/min</TD><TD>Calculated using DNA strand displacement (Winfree et al)</TD></TR></TBODY></TABLE><SPECIALH4> Table 8: Gp0.4 Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k_mGP0.4</TD><TD>Transcription of mGP0.4 before STAR induction</TD><TD>0.086</TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_imGP0.4</TD><TD>Transcription of mGP0.4 after STAR induction</TD><TD>8.08</TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>kf_GP0.4Ftsz</TD><TD>GP0.4:Ftsz complex formation</TD><TD>9.44E<SUP>03</SUP></TD><TD>1/molecules.min</TD></TR><TR><TD>kr_GP0.4Ftsz</TD><TD>GP0.4:Ftsz complex dissociation</TD><TD>30</TD><TD>1/min</TD></TR><TR><TD>k_GP0.4</TD><TD>Translation of GP0.4</TD><TD>13.86</TD><TD>Protein/min</TD><TD>Calculated from Bionumbers average translation rate : Bremer et al</TD></TR></TBODY></TABLE><SPECIALH4> Table 9: Tet Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k_mTET</TD><TD>Basal transcription of TET</TD><TD>0.028</TD><TD>mRNA/min</TD><TD>Calculated from experimental results for STAR activation (94 fold activation)</TD></TR><TR><TD>k_imTET</TD><TD>Induced transcription of TET</TD><TD>2.7</TD><TD>mRNA/min</TD></TR><TR><TD>k_TET:pTET</TD><TD>TET:pTET complex formation</TD><TD>120</TD><TD>1/molecules.min</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD></TR><TR><TD>k_-TET:pTET</TD><TD>TET:pTET complex dissassociation</TD><TD>8.40E<SUP>05</SUP></TD><TD>1/min</TD><TD><A href="https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1">https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1</A></TD></TR></TBODY></TABLE><SPECIALH4> Table 10: LeuB Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k_mLeuB</TD><TD>Basal transcription of LeuB transcription</TD><TD>1.538</TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_imLeuB</TD><TD>Induced transcription of LeuB transcription</TD><TD>0.015</TD><TD>mRNA/min</TD><TD>99% Repression from tet was assumed</TD></TR><TR><TD>k_LeuB</TD><TD>Translation of LeuB</TD><TD>0.0218</TD><TD>molecule/min</TD><TD><A href="http://biocyc.org/gene?orgid=ECOLI&amp;id=3-ISOPROPYLMALDEHYDROG-MONOMER#tab=FTRS">http://biocyc.org/gene?orgid=ECOLI&amp;id=3-ISOPROPYLMALDEHYDROG-MONOMER#tab=FTRS</A></TD></TR></TBODY></TABLE><SPECIALH4> Table 11: CAT Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k_mCAT</TD><TD>Basal transcription of CAT transcription</TD><TD>1.875</TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_imCAT</TD><TD>Induced transcription of CAT transcription</TD><TD>0.018</TD><TD>mRNA/min</TD><TD>99% Repression from CAT was assumed</TD></TR><TR><TD>k_CAT</TD><TD>Translation of CAT</TD><TD>0.03568</TD><TD>molecule/min</TD><TD><A href="http://www.uniprot.org/uniprot/P62577">http://www.uniprot.org/uniprot/P62577</A></TD></TR></TBODY></TABLE><SPECIALH4> Table 12: GP2 Module Numbers Table </SPECIALH4><TABLE class="table table-bordered table-hover"><THEAD><TR><TH>Parameter name</TH><TH>Description</TH><TH>Value</TH><TH>Unit</TH><TH>Source</TH></TR></THEAD><TBODY><TR><TD>k_mGP2</TD><TD>Transcription of mGP2 before STAR induction - Basal</TD><TD>0.07</TD><TD>mRNA/min</TD><TD>Calculated using 94 fold activation </TD></TR><TR><TD>k_imGP2</TD><TD>Transcription of mGP2 after STAR induction (7.2 kDa)</TD><TD>6.56</TD><TD>mRNA/min</TD><TD>Calculated from Bionumbers average transcription rate - RNA polymerase transcription</TD></TR><TR><TD>k_GP2RNAP</TD><TD>GP2:RNAP complex formation</TD><TD>1/Mole.min</TD><TD>Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7</TD></TR><TR><TD>k_-GP2RNAP</TD><TD>GP2:RNAP complex dissociation</TD><TD>1/min</TD><TD>Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7</TD></TR><TR><TD>k_GP2</TD><TD>Translation of GP2</TD><TD>11.2</TD><TD>Protein/min</TD><TD>Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7</TD></TR><TR><TD>RNA</TD><TD>Total RNA pol</TD><TD>1800</TD><TD>molecules/cell</TD><TD>Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7</TD></TR><TR><TD>k_mRNAP</TD><TD>Transcription of RNA pol</TD><TD>2400</TD><TD>nucleotides/min.RNAP</TD><TD>Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER class="footer-distributed"><DIV class="footer-right"><P> Email: igem.imperial2016@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>