Team:Imperial College/Experiments
Project
Description
Results
Proof
Basic Parts
Composite Parts
Modelling
Overview
Single Cell Model
Population Model
Software Tools
Notebook
Lab book
Protocols
Medal Criteria
Brainstorm
Human Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
Team
Team
Attributions
On this page, the ecolibrium team would like to share with you the protocols that we have been using over the summer. This library of protocols has been developed alongside our supervisors for the purpose of this study. Here you can find the exact methods we use to generate our data and results. We share them in the interest of reproducibility.
Chemical Transformation
Materials:
LB broth
Ice
Selection plates
Methods:
Thaw 50µL competent E. coli cells on ice for 10 minutes
Add:
5-10 µl DNA from a ligation reaction mix or
10-100ng DNA of a known plasmid
Carefully flick the tube 4-5 times to mix cells and DNA. Do not vortex.
Place the mixture on ice for 30 minutes. Do not mix.
Heat shock at exactly 42°C for exactly 30 seconds. Do not mix.
Place on ice for 5 minutes. Do not mix.
Pipette 950 µl of room temperature SOC or LB media into the mixture.
Incubate at 37°C and 200-250 rpm for 60 minutes.
Mix the cells thoroughly by flicking the tube and inverting.
Spread:
For ligation reaction DNA: 100µl of each transformation reaction onto a selection plate. For the rest of 900 µL:
Pellet cells at 8000rpm for 3 minutes
Remove and dispense 600 µL of supernatant
Re-suspend cells by light vortexing
Plate resuspended cells as above
For known plasmid: 10 & 100 µL of each transformation reaction onto a selection plate. For the rest of 890 µL:
Pellet cells at 8000rpm for 3 minutes
Remove and dispense 600 µL of supernatant
Re-suspend cells by light vortexing
Plate resuspended cells as above
Incubate overnight at 37°C with plates upside down.
Growing Overnight Cultures
Materials:
5 ml LB broth
5 μl antibiotic
Loops
12 ml culture tube
Methods:
Overnight cultures were prepared under sterile conditions using a Bunsen burner
Add 5 ml liquid LB media into 12 ml culture tubes
Add 5 μl of appropriate antibiotic into the broth
Using the loop, pick a single colony and inoculate the cultures by dipping the loop into the LB broth
Seal the tubes and incubate overnight at 37°C shaking at 200-250 rpm
PCR From Plasmid DNA Template
Materials:
2x Phusion Mastermix
10 µM forward primer
10 µM forward primer
PCR tube
Sterile water
Plasmid DNA
Methods:
For a 25 µL reaction
In a PCR tube on ice, combine  1-10 ng of plasmid DNA, 1.25 µL of 10 µM forward primer, 1.25 µL of 10 µM reverse primer to a PCR tube on ice, 12.5 µL of 2x Phusion Mastermix, and sterile water up to 25 µL.
Note: It is important to add Phusion Master Mix last in order to prevent primer degradation  caused by the 3 ́→ 5 ́ exonuclease activity
Gently mix the reaction
If necessary, collect the liquid to the bottom of the PCR tube by spinning briefly
Transfer the PCR tube from ice to a PCR machine to begin thermocycling
For a 50 µL reaction
In a PCR tube on ice, combine  1-10 ng of plasmid DNA, 2.50 µL of 10 µM forward primer, 2.50 µL of 10 µM reverse primer to a PCR tube on ice, 25 µL of 2x Phusion Mastermix, and sterile water up to 50 µL.
Note: It is important to add Phusion Master Mix last in order to prevent primer degradation  caused by the 3 ́→ 5 ́ exonuclease activity
Gently mix the reaction
If necessary, collect the liquid to the bottom of the PCR tube by spinning briefly
Transfer the PCR tube from ice to a PCR machine preheated to 98°C to begin thermocycling
Thermocycling
The PCR machine should be set to run the following steps:
Step
Temperature (°C)
Time
Initial denaturation
98
30 seconds
25-35 cycles
98 (denaturation)
45-72 (annealing) see Note 1
72 (extension)
5-10 seconds
10-30 seconds
15-30 seconds per kb
Final extension
72
2-5 minutes
Hold
4
Indefinitely
Note 1: Use the NEB Tm calculator should be used to determine the annealing temperature when using Phusion: http://tmcalculator.neb.com/#!/
Colony PCR
Materials:
Sterile Water
25 µL RedTaq mastermix
1 E. coli colony
2.5 µL of 10 µM forward primer
2.5 µL of 10 µM reverse primer
Methods:
Add a single colony of cells to 50 µL of water. Incubate at 95C for a minute to lyse the cells.
Combine 1 µL cell lysate, 25 µL RedTaq mastermix, 2.5 µL of 10 µM forward primer, 2.5 µL of 10 µM reverse primer, and sterile water up to 50 µL.
Note: It is important to add RedTaq Master Mix last in order to prevent primer degradation caused by the 3 ́→ 5 ́ exonuclease activity
Incubate in the thermocycler -  Taq has a lower optimum temperature than Phusion.
Thermocycling
The PCR machine should be set to run the following steps:
Step
Temperature (°C)
Time
Initial denaturation
98
30 seconds
25-35 cycles
98 (denaturation)
45-72 (annealing) see Note 1
68 (extension)
5-10 seconds
10-30 seconds
15-30 seconds per kb
Final extension
72
5-10 minutes
Hold
4
Indefinitely
Note: If loading on a gel, the RedTaq mix contains loading dye, so don’t add anything else.
Preparation of LB Broth, Agar, and Glycerol Stocks
LB Broth
Materials:
25 g LB broth (powder)
1 Litre Purified Water
Methods:
Add 25g LB broth to 1 litre purified water
Autoclave
LB Agar
Materials:
37 g LB Agar (powder)
1 Litre Purified Water
Methods:
Add 37g LB Agar to 1 litre purified water
Autoclave
Glycerol Stocks
Materials:
500µl glycerol (80%)
500µl overnight culture in LB
Methods:
Add 500µl glycerol (80%) to 1.5ml eppendorf tube
Add 500µl overnight culture in LB
Store at -80°C
DNA Gel Electrophoresis
Materials:
Agarose Powder
TAE buffer
Gel mould
5-10 µL SybrSafe
Gel Tank
8-10 µL DNA ladder
DNA loading dye
Methods:
Prepare 8% w/v solution of agarose powder in 1/10 TAE buffer (e.g. 0.8g agarose powder in 100 mL buffer) using a conical flask
Heat the mixture until agarose is completely dissolved. Do not let the solution boil.
Pour the solution into a gel mould
Add 5-10 µL SybrSafe® to the solution. Make sure there are no bubbles in the solution.
Allows the solution to set (approx 15-20 minutes)
Transfer the agarose gel to a tank, remove the comb and apply:
8-10 µL of the DNA ladder
DNA samples with the corresponding amount of DNA loading dye (6X)
Run the gel for 45-60 minutes at 100V
Ligation
Materials:
Microcentrifuge tube
Ice
1 µL T4 DNA Ligase
2 µL 10X T4 DNA ligase buffer
50ng Vector Plasmid
Insert DNA
Sterile water
Methods:
Calculate volumes of vector and insert DNA required using NEBioCalculator ( http://nebiocalculator.neb.com/#!/ligation - required molar ratio of 1:3::vector:insert)
Add vector plasmid, insert DNA, T4 DNA ligase and T4 DNA ligase buffer to the microcentrifuge tube on ice (add T4 DNA ligase last)
Make reaction up to 20 µL using sterile water
Incubate at room temperature for 30 - 60 min for sticky ends or  1-2 hours for blunt ends
Restriction Digestion (Test Digest and Assembly Digest)
Materials:
Restriction Enzyme: NEB enzyme finder used to see which restriction enzymes are required
10X buffer: NEB double digest finder used to see which buffers are required for the particular restriction enzymes
Plasmid DNA
Sterile water
Methods:
Component
Test digest
Double digestion (20 µL, for construct analysis)
Assemble digest
Double digestion (20-50 µL, for gene assembly)
Sterile water
to 20 µL
to 50 µL
10X buffer
2 µL
2-5 µL
Plasmid DNA
~200 ng for test digest,  (DNA mass is variable dependent on insert size.
Smallest digestion fragment mass should be > 50ng)
1,000 -2,000 ng
Restriction enzyme
0.5 µL + 0.5 µL
1 µL + 1 µL
Set up the reaction following the instruction in the table above , depending on whether test digest or assembly digest is being performed.
Incubate digestion reaction at 37°C:
30 min for Test digest
2-3 hours or overnight for Assembly digest
Perform heat deactivation at 80°C for 20 minutes, if not running on a gel at the end of incubation.
Growth Characterisation
Materials:
Microcentrifuge tube
Media of choice
Overnight Culture of Cells
96 well plate
Plate reader
Methods:
Dilute overnight cultures to 0.1 OD. Don’t forget the media control.
Seed your cells in the 96 well plate at 100μL per well.
Make reaction up to 20 µL using sterile water
Set the plate reader for 600OD and run it for 12 hours for bacteria or 24 hours plus for yeast
Cell Sorting for Co-Culture Characterisation
Materials:
Microcentrifuge tube
Media of choice
Overnight Culture of Cells
96 well plate
Flow Cytometer
PBS
Methods:
Use Flowjo to predefine the gates for the Flow cytometer. This allows the cells within the co culture to be counted based on size.
Dilute the Cultures to 1 OD (don’t forget the media control) and culture together at desired inoculation ratio
Set up samples in triplicate form on a 96 well plate. Dilute 10 fold and 100 fold in PBS.
Incubate Cells and take samples at desired time points to obtain growth curves
Quorum Characterisation Experiments
Materials:
Replica plate or glycerol stock of cells
Appropriate antibiotic
LB broth
96 well plate
AHLs (at concentrations of 10pM, 100pM, 1nM, 10nM, 100nM, 1µM, 10µM and 100µM)
Methods:
Grow overnight culture of cells, either from replica plate or glycerol stock (see overnight culture protocol)
In the morning, add 100 µL of the cell suspension to 10 mL LB and 10µL of antibiotic
Grow cells in incubator for 3-4 hours at 200rpm and 37°C
Blank spectrophotometer with 1mL LB and measure OD of cells (dilute 10x in LB)
Make up suspension of cells at OD 0.05
Add 100µL of cells to each well in the 96-well plate using the block-filler machine
Using the multi-channel pipette add 2µL of intended AHL to each well, add samples in triplicate form
Using the multi-channel pipette add 2µL of intended AHL to each well, add samples in triplicate form
Run on plate-reader for 12 hours, with fluorescence and absorbance measurements taken every 15 minutes (set plate reader to 37°C and 500rpm shaking)
PCR Purification, Gel Extraction & Miniprep
PCR purification was performed according to the QIAquick PCR Purification Kit
Gel extraction was performed according to the QIAquick Gel Extraction Kit
Minipreps were carried out according to the QIAprep Spin Miniprep Kit
Email: igem.imperial2016@gmail.com
