<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Imperial_College_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Imperial College/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Description">Project </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Composite_Part">Composite Parts</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Model">Modelling </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/SingleCell">Single Cell Model</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/GRO">Population Model</A></LI><LI><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Software">Software Tools </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Notebook">Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Notebook">Lab book</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Experiments">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Medals">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/BrainStorm">Brainstorm</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Human_Practices">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Team">Team </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI></DIV></DIV><DIV class="bg-primary"><SECTION><DIV class="col-lg-10 col-centered"><P>On this page, the ecolibrium team would like to share with you the protocols that we have been using over the summer. This library of protocols has been developed alongside our supervisors for the purpose of this study. Here you can find the exact methods we use to generate our data and results. We share them in the interest of reproducibility.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P1"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P1-collapse" aria-expanded="false" aria-controls="P1-collapse"><DIV><DIV class="col-md-11">Chemical Transformation</DIV></DIV></A></H4></DIV><DIV id="P1-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P1"><DIV class="panel-body"><P><B>Materials:</B>
LB broth
Ice
Selection plates<B>Methods:</B><OL style="font-size:16px;"><LI>Thaw 50µL competent E. coli cells on ice for 10 minutes</LI><LI>Add:
</LI><LI>5-10 µl DNA from a ligation reaction mix or </LI><LI>10-100ng DNA of a known plasmid </LI><LI>Carefully flick the tube 4-5 times to mix cells and DNA. <B>Do not vortex.</B></LI><LI>Place the mixture on ice for 30 minutes. <B>Do not mix.</B></LI><LI>Heat shock at exactly 42°C for exactly 30 seconds. <B>Do not mix.</B></LI><LI>Place on ice for 5 minutes. <B>Do not mix.</B></LI><LI>Pipette 950 µl of room temperature SOC or LB media into the mixture.</LI><LI>Incubate at 37°C and 200-250 rpm for 60 minutes.</LI><LI>Mix the cells thoroughly by flicking the tube and inverting.</LI><LI>Spread:
</LI><LI>For ligation reaction DNA: 100µl of each transformation reaction onto a selection plate. For the rest of 900 µL:
</LI><LI>Pellet cells at 8000rpm for 3 minutes</LI><LI>Remove and dispense 600 µL of supernatant </LI><LI>Re-suspend cells by light vortexing</LI><LI>Plate resuspended cells as above</LI></OL><LI>For known plasmid: 10 &amp; 100 µL of each transformation reaction onto a selection plate. For the rest of 890 µL:
</LI><LI>Pellet cells at 8000rpm for 3 minutes</LI><LI>Remove and dispense 600 µL of supernatant </LI><LI>Re-suspend cells by light vortexing</LI><LI>Plate resuspended cells as above</LI><LI>Incubate overnight at 37°C with plates upside down.</LI></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P2"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P2-collapse" aria-expanded="false" aria-controls="P2-collapse"><DIV><DIV class="col-md-11">Growing Overnight Cultures</DIV></DIV></A></H4></DIV><DIV id="P2-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P2"><DIV class="panel-body"><P><B>Materials:</B>
5 ml LB broth
5 μl antibiotic
Loops
12 ml culture tube<B>Methods:</B>
Overnight cultures were prepared under sterile conditions using a Bunsen burner<OL style="font-size:16px;"><LI>Add 5 ml liquid LB media into 12 ml culture tubes</LI><LI>Add 5 μl of appropriate antibiotic into the broth</LI><LI>Using the loop, pick a single colony and inoculate the cultures by dipping the loop into the LB broth</LI><LI>Seal the tubes and incubate overnight at 37°C shaking at 200-250 rpm</LI></OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P3"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P3-collapse" aria-expanded="false" aria-controls="P3-collapse"><DIV><DIV class="col-md-11">PCR From Plasmid DNA Template</DIV></DIV></A></H4></DIV><DIV id="P3-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P3"><DIV class="panel-body"><P><B>Materials:</B>
2x Phusion Mastermix
10 µM forward primer
10 µM forward primer
PCR tube
Sterile water
Plasmid DNA<B>Methods:</B>
For a 25 µL reaction<OL style="font-size:16px;"><LI>In a PCR tube on ice, combine  1-10 ng of plasmid DNA, 1.25 µL of 10 µM forward primer, 1.25 µL of 10 µM reverse primer to a PCR tube on ice, 12.5 µL of 2x Phusion Mastermix, and sterile water up to 25 µL.
<I>Note: It is important to add Phusion Master Mix last in order to prevent primer degradation 
caused by the 3 ́→ 5 ́ exonuclease activity</I></LI><LI>Gently mix the reaction</LI><LI>If necessary, collect the liquid to the bottom of the PCR tube by spinning briefly</LI><LI>Transfer the PCR tube from ice to a PCR machine to begin thermocycling</LI></OL></P><P>For a 50 µL reaction</P><OL style="font-size:16px;"><LI>In a PCR tube on ice, combine  1-10 ng of plasmid DNA, 2.50 µL of 10 µM forward primer, 2.50 µL of 10 µM reverse primer to a PCR tube on ice, 25 µL of 2x Phusion Mastermix, and sterile water up to 50 µL.
<I>Note: It is important to add Phusion Master Mix last in order to prevent primer degradation 
caused by the 3 ́→ 5 ́ exonuclease activity</I></LI><LI>Gently mix the reaction</LI><LI>If necessary, collect the liquid to the bottom of the PCR tube by spinning briefly</LI><LI>Transfer the PCR tube from ice to a PCR machine preheated to 98°C to begin thermocycling</LI></OL><P><B>Thermocycling</B>
The PCR machine should be set to run the following steps: </P><TABLE class="table table-bordered table-striped"><THEAD><TR><TH>Step</TH><TH>Temperature (°C)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30 seconds</TD></TR><TR><TD>25-35 cycles</TD><TD>98 (denaturation)
                45-72 (annealing) <A href="#Note1">see Note 1</A>
                72 (extension)</TD><TD>5-10 seconds 
                10-30 seconds
                15-30 seconds per kb</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>2-5 minutes</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>Indefinitely</TD></TR></TBODY></TABLE><P id="Note1">Note 1: Use the NEB Tm calculator should be used to determine the annealing temperature when using Phusion: <A target="_blank" href="http://tmcalculator.neb.com/#!/">http://tmcalculator.neb.com/#!/</A></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P4"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P4-collapse" aria-expanded="false" aria-controls="P4-collapse"><DIV><DIV class="col-md-11">Colony PCR</DIV></DIV></A></H4></DIV><DIV id="P4-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P4"><DIV class="panel-body"><P><B>Materials:</B>
Sterile Water
25 µL RedTaq mastermix
1 E. coli colony
2.5 µL of 10 µM forward primer
2.5 µL of 10 µM reverse primer<B>Methods:</B><OL style="font-size:16px;"><LI>Add a single colony of cells to 50 µL of water. Incubate at 95C for a minute to lyse the cells.</LI><LI>Combine 1 µL cell lysate, 25 µL RedTaq mastermix, 2.5 µL of 10 µM forward primer, 2.5 µL of 10 µM reverse primer, and sterile water up to 50 µL. <I>Note: It is important to add RedTaq Master Mix last in order to prevent primer
degradation caused by the 3 ́→ 5 ́ exonuclease activity</I></LI><LI>Incubate in the thermocycler -  Taq has a lower optimum temperature than Phusion.</LI></OL></P><P><B>Thermocycling</B>
The PCR machine should be set to run the following steps: </P><TABLE class="table table-bordered table-striped"><THEAD><TR><TH>Step</TH><TH>Temperature (°C)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>Initial denaturation</TD><TD>98</TD><TD>30 seconds</TD></TR><TR><TD>25-35 cycles</TD><TD>98 (denaturation)
                45-72 (annealing) <A href="#Note1">see Note 1</A>
                68 (extension)</TD><TD>5-10 seconds 
                10-30 seconds
                15-30 seconds per kb</TD></TR><TR><TD>Final extension</TD><TD>72</TD><TD>5-10 minutes</TD></TR><TR><TD>Hold</TD><TD>4</TD><TD>Indefinitely</TD></TR></TBODY></TABLE><P>Note: If loading on a gel, the RedTaq mix contains loading dye, so don’t add anything else.</P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P5"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P5-collapse" aria-expanded="false" aria-controls="P5-collapse"><DIV><DIV class="col-md-11">Preparation of LB Broth, Agar, and Glycerol Stocks</DIV></DIV></A></H4></DIV><DIV id="P5-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P5"><DIV class="panel-body"><P><SPECIALH3>LB Broth</SPECIALH3><B>Materials:</B>
25 g LB broth (powder)
1 Litre Purified Water<B>Methods:</B><OL style="font-size:16px;"><LI>Add 25g LB broth to 1 litre purified water</LI><LI>Autoclave</LI></OL></P><P><SPECIALH3>LB Agar</SPECIALH3><B>Materials:</B>
37 g LB Agar (powder)
1 Litre Purified Water<B>Methods:</B><OL style="font-size:16px;"><LI>Add 37g LB Agar to 1 litre purified water</LI><LI>Autoclave</LI></OL></P><P><SPECIALH3>Glycerol Stocks</SPECIALH3><B>Materials:</B>
500µl glycerol (80%)
500µl overnight culture in LB<B>Methods:</B><OL style="font-size:16px;"><LI>Add 500µl glycerol (80%) to 1.5ml eppendorf tube</LI><LI>Add 500µl overnight culture in LB</LI><LI>Store at -80°C</LI></OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P6"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P6-collapse" aria-expanded="false" aria-controls="P6-collapse"><DIV><DIV class="col-md-11">DNA Gel Electrophoresis</DIV></DIV></A></H4></DIV><DIV id="P6-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P6"><DIV class="panel-body"><P><B>Materials:</B>
Agarose Powder
TAE buffer
Gel mould
5-10 µL SybrSafe
Gel Tank
8-10 µL DNA ladder
DNA loading dye<B>Methods:</B><OL style="font-size:16px;"><LI>Prepare 8% w/v solution of agarose powder in 1/10 TAE buffer (e.g. 0.8g agarose powder in 100 mL buffer) using a conical flask</LI><LI>Heat the mixture until agarose is completely dissolved. Do not let the solution boil.</LI><LI>Pour the solution into a gel mould</LI><LI>Add 5-10 µL SybrSafe® to the solution. Make sure there are no bubbles in the solution.</LI><LI>Allows the solution to set (approx 15-20 minutes)</LI><LI>Transfer the agarose gel to a tank, remove the comb and apply:
</LI><LI>8-10 µL of the DNA ladder </LI><LI>DNA samples with the corresponding amount of DNA loading dye (6X)</LI><LI>Run the gel for 45-60 minutes at 100V</LI></OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P7"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P7-collapse" aria-expanded="false" aria-controls="P7-collapse"><DIV><DIV class="col-md-11">Ligation</DIV></DIV></A></H4></DIV><DIV id="P7-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P7"><DIV class="panel-body"><P><B>Materials:</B>
Microcentrifuge tube
Ice
1 µL T4 DNA Ligase
2 µL 10X T4 DNA ligase buffer
50ng Vector Plasmid
Insert DNA
Sterile water<B>Methods:</B><OL style="font-size:16px;"><LI>Calculate volumes of vector and insert DNA required using NEBioCalculator (<A href="http://nebiocalculator.neb.com/#!/ligation"> http://nebiocalculator.neb.com/#!/ligation</A> - required molar ratio of 1:3::vector:insert)</LI><LI>Add vector plasmid, insert DNA, T4 DNA ligase and T4 DNA ligase buffer to the microcentrifuge tube on ice (add T4 DNA ligase last)</LI><LI>Make reaction up to 20 µL using sterile water</LI><LI>Incubate at room temperature for 30 - 60 min for sticky ends or  1-2 hours for blunt ends</LI></OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P8"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P8-collapse" aria-expanded="false" aria-controls="P8-collapse"><DIV><DIV class="col-md-11">Restriction Digestion (Test Digest and Assembly Digest)</DIV></DIV></A></H4></DIV><DIV id="P8-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P8"><DIV class="panel-body"><P><B>Materials:</B>
Restriction Enzyme: 
<A href="">NEB enzyme finder</A> used to see which restriction enzymes are required 
10X buffer: <A href="https://www.neb.com/tools-and-resources/interactive-tools/double-digest-finder">NEB double digest finder</A> used to see which buffers are required for the particular restriction enzymes 
Plasmid DNA
Sterile water<B>Methods:</B><TABLE class="table table-bordered table-striped"><THEAD><TR><TH>Component</TH><TH>Test digest
Double digestion (20 µL, for construct analysis)
</TH><TH>Assemble digest
Double digestion (20-50 µL, for gene assembly)</TH></TR></THEAD><TBODY><TR><TD>Sterile water</TD><TD>to 20 µL</TD><TD>to 50 µL</TD></TR><TR><TD>10X buffer</TD><TD>2 µL</TD><TD>2-5 µL</TD></TR><TR><TD>Plasmid DNA</TD><TD>~200 ng for test digest, 
(DNA mass is variable dependent on insert size. Smallest digestion fragment mass should be &gt; 50ng)</TD><TD>1,000 -2,000 ng</TD></TR><TR><TD>Restriction enzyme</TD><TD>0.5 µL + 0.5 µL</TD><TD>1 µL + 1 µL</TD></TR></TBODY></TABLE><OL style="font-size:16px;"><LI>Set up the reaction following the instruction in the table above , depending on whether test digest or assembly digest is being performed.</LI><LI> Incubate digestion reaction at 37°C:
    </LI><LI>30 min for Test digest</LI><LI>2-3 hours or overnight for Assembly digest</LI></OL><LI>Perform heat deactivation at 80°C for 20 minutes, if not running on a gel at the end of incubation. </LI></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P9"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P9-collapse" aria-expanded="false" aria-controls="P9-collapse"><DIV><DIV class="col-md-11">Growth Characterisation</DIV></DIV></A></H4></DIV><DIV id="P9-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P9"><DIV class="panel-body"><P><B>Materials:</B>
Microcentrifuge tube
Media of choice
Overnight Culture of Cells
96 well plate
Plate reader<B>Methods:</B><OL style="font-size:16px;"><LI>Dilute overnight cultures to 0.1 OD. <B>Don’t forget the media control.</B></LI><LI>Seed your cells in the 96 well plate at 100μL per well.</LI><LI>Make reaction up to 20 µL using sterile water</LI><LI>Set the plate reader for 600OD and run it for 12 hours for bacteria or 24 hours plus for yeast
</LI></OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P10"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P10-collapse" aria-expanded="false" aria-controls="P10-collapse"><DIV><DIV class="col-md-11">Cell Sorting for Co-Culture Characterisation</DIV></DIV></A></H4></DIV><DIV id="P10-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P10"><DIV class="panel-body"><P><B>Materials:</B>
Microcentrifuge tube
Media of choice
Overnight Culture of Cells
96 well plate
Flow Cytometer
PBS<B>Methods:</B><OL style="font-size:16px;"><LI>Use Flowjo to predefine the gates for the Flow cytometer. This allows the cells within the co culture to be counted based on size.</LI><LI>Dilute the Cultures to 1 OD (don’t forget the media control) and culture together at desired inoculation ratio</LI><LI>Set up samples in triplicate form on a 96 well plate. Dilute 10 fold and 100 fold in PBS.</LI><LI>Incubate Cells and take samples at desired time points to obtain growth curves</LI></OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P11"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P11-collapse" aria-expanded="false" aria-controls="P11-collapse"><DIV><DIV class="col-md-11">Quorum Characterisation Experiments</DIV></DIV></A></H4></DIV><DIV id="P11-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P11"><DIV class="panel-body"><P><B>Materials:</B>
Replica plate or glycerol stock of cells
Appropriate antibiotic
LB broth
96 well plate
AHLs (at concentrations of 10pM, 100pM, 1nM, 10nM, 100nM, 1µM, 10µM and 100µM)<B>Methods:</B><OL style="font-size:16px;"><LI>Grow overnight culture of cells, either from replica plate or glycerol stock (see overnight culture protocol)
</LI><LI>In the morning, add 100 µL of the cell suspension to 10 mL LB and 10µL of antibiotic</LI><LI>Grow cells in incubator for 3-4 hours at 200rpm and 37°C</LI><LI>Blank spectrophotometer with 1mL LB and measure OD of cells (dilute 10x in LB)</LI><LI>Make up suspension of cells at OD 0.05</LI><LI>Add 100µL of cells to each well in the 96-well plate using the block-filler machine</LI><LI>Using the multi-channel pipette add 2µL of intended AHL to each well, add samples in triplicate form
</LI><LI>Using the multi-channel pipette add 2µL of intended AHL to each well, add samples in triplicate form
</LI>Run on plate-reader for 12 hours, with fluorescence and absorbance measurements taken every 15 minutes (set plate reader to 37°C and 500rpm shaking)
</OL></P></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><DIV class="panel-heading" role="tab" id="P12"><H4 class="panel-title"><A role="button" data-toggle="collapse" data-parent="#accordion" href="#P12-collapse" aria-expanded="false" aria-controls="P12-collapse"><DIV><DIV class="col-md-11">PCR Purification, Gel Extraction &amp; Miniprep</DIV></DIV></A></H4></DIV><DIV id="P12-collapse" class="panel-collapse collapse" role="tabpanel" aria-labelledby="P12"><DIV class="panel-body"><P><B>PCR purification</B> was performed according to the QIAquick PCR Purification Kit<B>Gel extraction</B> was performed according to the QIAquick Gel Extraction Kit<B>Minipreps</B> were carried out according to the QIAprep Spin Miniprep Kit</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION><FOOTER class="footer-distributed"><DIV class="footer-right"><P> Email: igem.imperial2016@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>