Team:Imperial College/Model
Project
Description
Results
Proof
Basic Parts
Composite Parts
Modelling
Overview
Single Cell Model
Population Model
Software Tools
Notebook
Lab book
Protocols
Medal Criteria
Brainstorm
Human Practices
Human Practices
Integrated Practices
Public Engagement
Collaborations
Team
Team
Attributions
This year our team created a mathematical representation of our Genetically Engineered Artificial Ratio (G.E.A.R.) system. This representation, or model, informed the wet lab on the timescales of the three modules of GEAR: communication, comparator and growth regulator. It also helped develop assembly strategies thanks to a thorough analysis of parameter scans and sensitivities. After a series of experiments in the wet lab, we were able to finesse our parameters making them more accurate. This process, presented below, can be seen as a continuous feedback between the wet lab and the dry lab of Ecolibrium.
We developed our models with two main goals in mind:
We wanted our model to be able to aid the team in the wetlab. Especially for optimizing the assembly process and balancing the circuit.
We wanted to develop an in silico version of our circuit at single cell and population level. This allowed us to test the viability of the system in both normal and abnormal conditions. It also allows us to plan future experiments and used for the system.
The following pages will show how we implemented modelling approaches to achieve our goals.
The Single Cell Model
Communication Module - We constructed the four quorum systems that we considered viable choices for our system (cin, rhl, lux and las)to allow us to directly compare the expected behaviour and plan our growth module experiments accordingly.
Comparator Module - We used RNAstruct developed by Matthews Lab to help aid the development of the ANTISTAR. We modeled STAR and ANTISTAR behaviour in silico
Growth Regulator Module - We modelled 4 different growth regulator systems in silico in order to assess the speed and effectiveness of each case.
Population Model
Matlab population model with GP2 growth regulation in two population
We used the GRO programming language to model a simplified version of our circuit with GP0.4 growth regulation into two populations of E.Coli.
Species
Table 1: Universal Species Table
Species Name
Description
C4
AHLs produced by RhlI synthase
C14
AHLs produced by CinI synthase
g_C4R
Copy number for plasmid backbone housing C4R
mC4R
mRNA for RhlR
C4R
RhlR transcriptional regulator
C4Comp
Complex by C4 AHL and RhlR transcriptional regulator
g_STARC4Comp
Complex formed between plasmid backbone for STAR (pRhl) and c4 Comp
g_C14R
Copy number for plasmid backbone housing C14R
mC14R
mRNA for CinR
C14R
CinR transcriptional regulator
C14Comp
Complex by C14 AHL and CinR transcriptional regulator
g_ANTISTARC14Comp
Complex formed between plasmid backbone for ANTISTAR (pCin) and C14 Comp
g_STAR
Copy number for plasmid backbone housing STAR
STAR
Short transcription activating RNA (STAR)
g_ANTISTAR
Copy number for plasmid backbone housing ANTISTAR
ANTISTAR
Anti Short transcription activating RNA (ANTISTAR)
STAR:ANTISTAR
STAR:ANTISTAR Complex
STAR_Target
Star Target plasmid (AD1 terminator)
Table 2: Unique Species for GP0.4 model Table
Species Name
Description
g_GP0.4
Plasmid hosting STAR Target and growth regulating protein GP2
mGP0.4
mRNA for GP0.4
pre_GP0.4
Unfolded GP0.4
GP0.4
GP0.4 (Gene Product 0.4) sequesters FtsZ and stops cell division.
FtsZ
FtsZ is a protein that influences cell division
GP0.4:FtsZ
GP0.4:FtsZ Complex
Table 3: Unique Species for GP2 model Table
Species Name
Description
g_GP2
Plasmid hosting STAR Target and growth regulating protein GP2
mGP2
mRNA for GP2
pre_GP2
Unfolded GP2
GP2
GP2 (Gene Product 2) sequesters RNA Polymerase
RNAP
RNA Polymerase
GP2:RNAP
GP2:RNAP Complex
Table 4: Unique Species for CAT and LeuB
Species Name
Description
mTet
mRNA for Tetracycline
Tet
Tetracycline
pTet
Promoter that is repressed by Tetracycline
Tet:pTet
Tetracycline:pTet complex
mLeuB
mRNA for Leucine B
LeuB
Leucine B
mCAT
mRNA for Chloramphenicol Acetyltransferase
CAT
Chloramphenicol Acetyltransferase
Parameters
Table 5: General Numbers Table
Parameter name
Description
Value
Unit
Source
Cell Volume
6.70E-16
L
Neidhardt F.C. Escherichia coli and Salmonella: Cellular and Molecular Biology. Vol 1. pp. 15, ASM Press 1996.
Transcription rate
28 Nucleotides/s was used
28-89
Nucleotides/s
Vogel U, Jensen KF. The RNA chain elongation rate in Escherichia coli depends on the growth rate. J Bacteriol. 1994 May176(10):2807-13 p.2811 table 2
Translation rate
12 Amino acid/s was used
12-21
Amino acids/s
Bremer, H., Dennis, P. P. (1996) Modulation of chemical composition and other parameters of the cell by growth rate. Neidhardt, et al. eds. Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology, 2nd ed. chapter 97, p. 1559, Table 3
d_prot
Protein degradation
1.39E-05
1/min
Biomolecular Systems by Del Vecchio
k_mat
Protein maturation rate
0.16
1/min
Megerle JA, Fritz G, Gerland U, Jung K, Rädler JO. Timing and dynamics of single cell gene expression in the arabinose utilization system. Biophys J. 2008 Aug95(4):2103-15. p.2106 right column bottom of paragraph
D
Dilution rate
0.048
1/min
A synthetic Escherichia coli predator–prey ecosystem
d_mRNA
mRNA degradation rate constant
0.139
1/min
Taken from 5 min mRNA degradation time
Copy number for 3K3 and 3C3
Copy number
20-30
molecules
http://parts.igem.org/Part:pSB3K3
Copy number for 1K3 and 1C3
Copy number
100-300
molecules
http://parts.igem.org/Part:pSB1C3
Table 6: Quorum Module Numbers Table
Parameter name
Description
Value
Unit
Source
k_C4
Rhl/C4 Production Rate Constant
8.00E-07
5.36E-13
5.36E-22
3.22E02
M/min
nMol/min
mol/min
molecules/min
Optimal tuning of bacterial sensing potential - Anand Pai & Lingchong You
k_C14
Cin/C14 Production Rate Constant
5.00E-08
3.35E-14
3.35E-23
2.00E01
M/min
nMol/min
mol/min
molecules/min
Optimal tuning of bacterial sensing potential - Anand Pai & Lingchong You
k_mC4R
Transcription rate of C4R
2.23
5.51E-09
3.69E-24
mRNA/min
nMol/min
Mol/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_mC14R
Transcription rate of C14R
2.23
3.69E-15
3.69E-24
mRNA/min
nMol/min
Mol/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_C4R
Translation rate of C4R
2.99
4.96E-24
Protein/min
Mol/min
Calculated from Bionumbers average translation rate : Bremer et al
k_C14R
Translation rate of C14R
2.99
4.96E-24
Protein/min
Mol/min
Calculated from Bionumbers average translation rate : Bremer et al
kf_C4Comp
C4 - C4R Complex formation
1.00E08
2.50E-01
1/M.min
1/complexes.min
Dynamics of quorum sensing switch - Weber
kr_C4Comp
C4:C4R Complex dissassociation
10
1/min
Dynamics of quorum sensing switch - Weber
kf_C14Comp
C4 - C4R Complex formation
1.00E08>/sup>
2.50E-01
1/M.min
1/complexes.min
Dynamics of quorum sensing switch - Weber
kr_C14Comp
C4:C4R Complex dissassociation
10
1/min
Dynamics of quorum sensing switch - Weber
kf_DimC4
C4:C4R complex dimerization
5.00E07
M/min
Dynamics of quorum sensing switch - Weber
kr_DimC4
C4:C4R complex reverse dimerization
1
1/min
Dynamics of quorum sensing switch - Weber
kf_DimC14
C14:C14R complex dimerization
5.00E07
M/min
Dynamics of quorum sensing switch - Weber
kr_DimC14
C14:C14R complex reverse dimerization
1
1/min
Dynamics of quorum sensing switch - Weber
d_C4
C4 degradation rate constant
2.21E-04
1/min
Optimal tuning of bacterial sensing potential - Anand Pai & Lingchong You
d_C14
C14 degradation rate constant
2.83E-04
1/min
Optimal tuning of bacterial sensing potential - Anand Pai & Lingchong You
d_C4R
C4R degradation rate constant
0.002
1/min
Optimal tuning of bacterial sensing potential - Anand Pai & Lingchong You
d_C14R
C14R degradation rate constant
0.002
1/min
Optimal tuning of bacterial sensing potential - Anand Pai & Lingchong You
Table 7: STAR Module Numbers Table
Parameter name
Description
Value
Unit
Source
kf_gSTARC4Comp
gSTAR-C4 comple association
0.25
1/molecules.min
Assumed same as quorum complex formation
kr_gSTARC4Comp
gSTAR-C4 complex disassociation
10
1/min
Assumed same as quorum complex formation
kf_gANTISTARC14Comp
gANTISTAR-C14 complex association
0.25
1/molecules.min
Assumed same as quorum complex formation
kr_gANTISTARC14Comp
gANTISTAR - C14 complex disassociation
10
1/min
Assumed same as quorum complex formation
k_STAR
Basal Rate of STAR production (without C4:C4R induction)
1.91E-01
3.08E-23
mRNA/min
mol/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_iSTAR
Rate of STAR production (after induction of C4:C4R)
18.57
3.08E-23
mRNA/min
mol/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_ANTISTAR
Basal Rate of ANTISTAR production (without C14:C14R induction)
1.91E-01
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_iANTISTAR
Rate of ANTI STAR production (after induction of C14:C14R)
18.57
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
kf_STARANTISTAR
STAR:ANTISTAR complex formation
62
complexes/min
Calculated using DNA strand displacement (Winfree et al)
kr_STARANTISTAR
STAR:ANTISTAR complex dissociation
2
1/min
Calculated using DNA strand displacement (Winfree et al)
kf_STARAD1
STAR:AD1 Complex formation
62
complexes/min
Calculated using DNA strand displacement (Winfree et al)
kr_STARAD1
STAR:AD1 Complex dissociation
2
1/min
Calculated using DNA strand displacement (Winfree et al)
Table 8: Gp0.4 Module Numbers Table
Parameter name
Description
Value
Unit
Source
k_mGP0.4
Transcription of mGP0.4 before STAR induction
0.086
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_imGP0.4
Transcription of mGP0.4 after STAR induction
8.08
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
kf_GP0.4Ftsz
GP0.4:Ftsz complex formation
9.44E03
1/molecules.min
kr_GP0.4Ftsz
GP0.4:Ftsz complex dissociation
30
1/min
k_GP0.4
Translation of GP0.4
13.86
Protein/min
Calculated from Bionumbers average translation rate : Bremer et al
Table 9: Tet Module Numbers Table
Parameter name
Description
Value
Unit
Source
k_mTET
Basal transcription of TET
0.028
mRNA/min
Calculated from experimental results for STAR activation (94 fold activation)
k_imTET
Induced transcription of TET
2.7
mRNA/min
k_TET:pTET
TET:pTET complex formation
120
1/molecules.min
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
k_-TET:pTET
TET:pTET complex dissassociation
8.40E05
1/min
https://2009.igem.org/Team:Aberdeen_Scotland/parameters/invest_1
Table 10: LeuB Module Numbers Table
Parameter name
Description
Value
Unit
Source
k_mLeuB
Basal transcription of LeuB transcription
1.538
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_imLeuB
Induced transcription of LeuB transcription
0.015
mRNA/min
99% Repression from tet was assumed
k_LeuB
Translation of LeuB
0.0218
molecule/min
http://biocyc.org/gene?orgid=ECOLI&id=3-ISOPROPYLMALDEHYDROG-MONOMER#tab=FTRS
Table 11: CAT Module Numbers Table
Parameter name
Description
Value
Unit
Source
k_mCAT
Basal transcription of CAT transcription
1.875
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_imCAT
Induced transcription of CAT transcription
0.018
mRNA/min
99% Repression from CAT was assumed
k_CAT
Translation of CAT
0.03568
molecule/min
http://www.uniprot.org/uniprot/P62577
Table 12: GP2 Module Numbers Table
Parameter name
Description
Value
Unit
Source
k_mGP2
Transcription of mGP2 before STAR induction - Basal
0.07
mRNA/min
Calculated using 94 fold activation
k_imGP2
Transcription of mGP2 after STAR induction (7.2 kDa)
6.56
mRNA/min
Calculated from Bionumbers average transcription rate - RNA polymerase transcription
k_GP2RNAP
GP2:RNAP complex formation
1/Mole.min
Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
k_-GP2RNAP
GP2:RNAP complex dissociation
1/min
Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
k_GP2
Translation of GP2
11.2
Protein/min
Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
RNA
Total RNA pol
1800
molecules/cell
Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
k_mRNAP
Transcription of RNA pol
2400
nucleotides/min.RNAP
Intracellular Kinetics of a Growing Virus: A Genetically Structured Simulation for Bacteriophage T7
Email: igem.imperial2016@gmail.com
