<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Imperial_College_Proof skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Imperial College/Proof</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV id="navbar" class="navbar-collapse collapse"><UL class="nav navbar-nav navbar-right" style="margin-top: 15px;"><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Description">Project </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Results">Results</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Proof">Proof</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Basic_Part">Basic Parts</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Composite_Part">Composite Parts</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Model">Modelling </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Model">Overview</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/SingleCell">Single Cell Model</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/GRO">Population Model</A></LI><LI><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Software">Software Tools </A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Notebook">Notebook </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Notebook">Lab book</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Experiments">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Medals">Medal Criteria</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/BrainStorm">Brainstorm</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Human_Practices">Human Practices </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Human_Practices">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Collaborations">Collaborations</A></LI><LI class="dropdown"><A class="dropdown-toggle active" href="https://2016.igem.org/Team:Imperial_College/Team">Team </A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:Imperial_College/Attributions">Attributions</A></LI></DIV></DIV><DIV class="bg-primary"><SECTION><DIV class="col-lg-10 col-centered"><SPECIALH3>Colour proof</SPECIALH3><P>
In order to demonstrate one of the many possible applications of our ratio control circuit, we decided to use chromoproteins as visual proof of concept. We were able to create a multitude of colours by mixing different ratios of <I>E. coli</I> cultures expressing the chromoproteins.  We hope that this provided a glimpse of the power of co-culture in the production of composite biomaterials. </P><SPECIALH3>Key Achievements</SPECIALH3><UL style="font-size:18px;"><LI>Built constructs for expressing 7 different chromoproteins </LI><LI>Produced a construct linking Gp2 expression to chromoprotein expression </LI><LI>Showed that Gp2 can be used to control the growth of cells and consequently the production of proteins</LI></UL><P><SPECIALH3>Overview</SPECIALH3>
Chromoproteins are brightly coloured when expressed, and the colour is clearly visible to the naked eye, unlike fluorescent proteins.  We want to use these proteins to produce coloured cells to demonstrate how co-cultures can be used to create varying shades of colour, providing a visual reference for the ratio of two cells.  Chromoproteins are of great interest to the synthetic biology community for used as reporters and also as a new form of non-toxic biological pigments.  Colours provide a simple way of demonstrating how different ratio and composition of materials can generate an array of products with different properties. <SPECIALH3>Our Approach</SPECIALH3>
We hoped to work with multiple chromoproteins for our demonstration, so we selected 7 gene sequences from the 2016 iGEM distribution kit. These are: spisPink, amajLime, amilGFP, fwYellow, eforRed, gfasPurple and cjBlue. We assembled these coding sequences with an RBS part with a built-in Anderson promoter and a terminator.  We next transformed these constructs into Top10 cells for characterization. <SPECIALH3>Experimental Design</SPECIALH3>
We first aimed to simply demonstrate that multiple colors could be produced by mixing coloured cells. We then attempted to recreate these colors by growing cells in co-culture, as color production by co-culture is more efficient and more scalable.

We measured the growth rate of cells producing different chromoproteins by using a plate reader to measure the optical density of isolated populations over 10 hours In order to determine if different types of chromoprotein expression were influencing the growth rate differently.

We want to show that growth control can be used to produce a stable co-culture that can maintain its ratio overtime.  Therefore we combined the arabinose-inducible Gp2 construct with a construct for the chromoprotein eforRed. We then induced the cells with arabinose to observe the effect of Gp2 on the population stability of the colored cells.<SPECIALH3>Results</SPECIALH3>
In our cell mixing experiment, we were able to produce over 70 different colors from the 7 base chromoproteins, indicating that a wide range of biological colours can potentially be produced by chromoprotein mixing. </P><CENTER><P><B>Figure 1:</B> Picture of the different colours obtained by manually mixing different ratios of colored cells. </P></CENTER><P>Our data revealed that cells expressing different chromoproteins tended to have different growth rates, despite the fact that they all were similar in size. This aligns with the theory that co-cultures tend to fail due to one populationâ€™s growth rate exceeding that of the other. This affirms the need for a genetic circuit to stabilise co-cultures.


</P><P><B>Figure 2:</B> Plot of Growth rate versus time for <I> Escherichia coli</I> Top 10 cells expressing various color constructs. </P><P>The cells transformed with the efoRed+Gp2 construct showed a decrease in growth rate when induced with arabinose, suggesting that our circuit can be a suitable system for controlling the growth of colored cells.</P><CENTER><P><B>Figure 3:</B> Plot of Growth rate versus time for the eforRed with growth control Gp2 construct <I> Escherichia coli</I> Top 10 cells. </P></CENTER><P><SPECIALH3>What's Next?</SPECIALH3>
We aim to demonstrate different chromoprotein expressing cells growing together in co-culture. To do this, we plan on inoculating the two populations at a 1:1 ratio and recording the peak absorption over time. This would allow us to observe if the population composition is changing, as it would be reflected in a change in color and therefore in absorption properties. We could then determine if the slower growing population, as determined from our earlier growth rate experiments, is outcompeted by the other population as expected.

We also hope to grow the cells transformed with the efoRed+Gp2 under the control of the arabinose promoter alongside cells expressing the chromoprotein amajLime. We could then monitor the absorption of the sample after induction with different amounts of arabinose to see if varying levels of induction changed the colour of the co-culture.

Once these preliminary experiments are successfully completed, we hope to integrate different chromoprotein genes into the genomes of two populations of <I> E. coli</I> and transform our circuit plasmids into the cells. We would then induce the co-culture with different ratios of AHLs to set various population ratios, which would be visible as different colours. <SPECIALH3>Experience</SPECIALH3>

Chromoproteins allow easy identification of successfully transformed cells. However different chromoproteins have different maturation time and provide different colour intensity.  This further complicate the mixing process as more intense colours are better at being mixed that the less intense ones. </P><SPECIALH3>Future work</SPECIALH3><P>However, a more comprehensive  proof-of-concept requires putting all three modules together - i.e. linking the quorum communication system to the production of STAR, and linking the production of STAR to growth control.
 
We are currently attempting to create constructs for several experiments to link the modules together. We hope to have some results for this by the jamboree.</P><SPECIALH3>Proof-of-concept 1: Linking the STAR system to quorum communication</SPECIALH3><P>To show that the STAR system can compare the size of two quorum inputs, we are placing the STAR and anti-STAR under the control of the pLas and pRhl promoters, respectively. This construct will be co-transformed with the STAR reporter construct, and we hope to show that different ratios of exogeneous C12 (las) and C4 (rhl) signals will result in differing levels of fluorescence.</P><SPECIALH3>Proof-of-concept 2: Linking the STAR system to the growth control module</SPECIALH3><P>One final result is to try and link all three modules together. We will take the construct created for the proof-of-concept 1 experiment, and co-transform it along with the STAR reporter plasmid, with the SFGFP replaced by the Gp2 gene.
 
Using the same protocol as in proof-of-concept one, we would show that our system can compare the size of two quorum signals, and arrest its own growth.</P><SPECIALH3>Further work</SPECIALH3><P>After Boston, we have plans to refine our circuit further.  We have detailed some of our future plans below:</P><SPECIALH3>Completion of the circuit as designed</SPECIALH3><P>There are a few features of the circuit that we were not able to complete because of time restraints. First, we need to check that the circuit does indeed work for a co-culture. This will be achieved by creating two populations that both have the circuit, but with STAR and Anti-STAR[PKM2]  under control of the opposite quorum promoters, and each population containing a different reporter gene. We would inoculate the cultures with a ratio of two different quorum signals, and measure how the ratio of the two reporters changes.
 
Secondly, we also need to insert the quorum synthesiser genes (one in each population) so the circuit can be autonomous and not require exogenous application of the quorum signals. </P><SPECIALH3>Tuning the circuit</SPECIALH3><P>Another facet to our future work needs to be how to balance everything, to make sure that the modules work together in harmony. We are using our model to forward-engineer these experiments.  Our sensitivity analysis  has shown us which elements of the circuit we should alter to tune the behaviour. 
 
Lastly, our circuit is two-plasmid system, which requires the production of four proteins in each cell. This means that the load is fairly heavy, and any researcher that wants to further Therefore, we have considered trying to integrate the circuit into the E. coli genome, which will reduce the metabolic burden on the cell, and allow researchers to use any plasmid they choose. This will of course affect the balance of our circuit (genetic elements would be reduced down to a single copy), although modelling has shown that the circuit should still work correctly.</P></DIV></SECTION><FOOTER class="footer-distributed"><DIV class="footer-right"><P> Email: igem.imperial2016@gmail.com </P></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>