<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Harvard_Dailybook_Week1_Chemical_and_Light"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Harvard/Dailybook/Week1/Chemical_and_Light">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Harvard/Dailybook/Week1/Chemical_and_Light&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Dailybook/Week1/Chemical_and_Light&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Harvard/Dailybook/Week1/Chemical_and_Light&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Harvard/Dailybook/Week1/Chemical_and_Light" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Harvard/Dailybook/Week1/Chemical and Light</H1><DIV id="bodyContent"><H3 id="siteSub">From 2008.igem.org</H3><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Chemical"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Chemical</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Results_from_Friday_.286.2F20.29_Electroporation_and_Transformation_in_S1"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Results from Friday (6/20) Electroporation and Transformation in S1</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#6.2F23:"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">6/23:</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-4"><A href="#Growing_cultures_of_E._coli_and_S1_with_BioBrick_Parts_or_Duet_Vectors_in_them"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Growing cultures of E. coli and S1 with BioBrick Parts or Duet Vectors in them</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#6.2F23:_2"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">6/23:</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#6.2F24:"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">6/24:</SPAN></A></LI><LI class="toclevel-4 tocsection-7"><A href="#Concentrations_of_DNA_from_minipreps:"><SPAN class="tocnumber">1.2.2.1</SPAN><SPAN class="toctext">Concentrations of DNA from minipreps:</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Transformation_of_P5_and_P6_into_E3"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation of P5 and P6 into E3</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#6.2F24:_2"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">6/24:</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#16S_PCR_to_Identify_S1_and_E._Coli"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">16S PCR to Identify S1 and E. Coli</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#6.2F24:_3"><SPAN class="tocnumber">1.4.1</SPAN><SPAN class="toctext">6/24:</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#6.2F25"><SPAN class="tocnumber">1.4.2</SPAN><SPAN class="toctext">6/25</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Remy.27s_pZ_vector_system_Plasmids"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Remy's pZ vector system Plasmids</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Re-Transformation_of_BioBrick_Parts_in_E1_and_E3"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Re-Transformation of BioBrick Parts in E1 and E3</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#6.2F26:"><SPAN class="tocnumber">1.6.1</SPAN><SPAN class="toctext">6/26:</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#6.2F27:"><SPAN class="tocnumber">1.6.2</SPAN><SPAN class="toctext">6/27:</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#McConkey_Plates_to_Distinguish_E._coli_from_Shewanella"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">McConkey Plates to Distinguish E. coli from Shewanella</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#6.2F24"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">6/24</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#6.2F25:"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">6/25:</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Growing_Shewie_anaerobically"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Growing Shewie anaerobically</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#6.2F25:_2"><SPAN class="tocnumber">1.8.1</SPAN><SPAN class="toctext">6/25:</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#6.2F26:_2"><SPAN class="tocnumber">1.8.2</SPAN><SPAN class="toctext">6/26:</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Questions.2F_Concerns"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Questions/ Concerns</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Light"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Light</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Transformations_of_P11.2C_P15-20"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Transformations of P11, P15-20</SPAN></A></LI><LI class="toclevel-3 tocsection-26"><A href="#Colony_counts"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Colony counts</SPAN></A></LI><LI class="toclevel-3 tocsection-27"><A href="#Restreaking"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Restreaking</SPAN></A></LI><LI class="toclevel-3 tocsection-28"><A href="#Overnight_cultures"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Overnight cultures</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Miniprep_of_P17"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Miniprep of P17</SPAN></A></LI><LI class="toclevel-3 tocsection-30"><A href="#Restriction_Digest"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Restriction Digest</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Transformations_of_P22-24"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Transformations of P22-24</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Transformations_of_P17.2C_20.2C_25.2C_26"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Transformations of P17, 20, 25, 26</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#New_transformations:_P11.2C_P15-20.2C_P22-26"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">New transformations: P11, P15-20, P22-26</SPAN></A></LI><LI class="toclevel-3 tocsection-34"><A href="#Transformation_results"><SPAN class="tocnumber">2.5.1</SPAN><SPAN class="toctext">Transformation results</SPAN></A></LI><LI class="toclevel-3 tocsection-35"><A href="#Minipreps_and_digests"><SPAN class="tocnumber">2.5.2</SPAN><SPAN class="toctext">Minipreps and digests</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#RE_digest"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">RE digest</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Chemical">Chemical</SPAN></H1><H2><SPAN class="mw-headline" id="Results_from_Friday_.286.2F20.29_Electroporation_and_Transformation_in_S1">Results from Friday (6/20) Electroporation and Transformation in S1</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F23:">6/23:</SPAN></H3><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Registry Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Description</B></TD><TD align="center" style="background:#eeeeee;"><B>Origin</B></TD><TD align="center" style="background:#eeeeee;"><B>Marker</B></TD><TD align="center" style="background:#eeeeee;"><B>Observations</B></TD></TR><TR><TD> P1a </TD><TD><A href="http://partsregistry.org/Part:BBa_J23113" class="external text" rel="nofollow">J23113</A></TD><TD>Low promoter GFP tester</TD><TD>pMB1</TD><TD>Kan</TD><TD>16 pink colonies of approx 7mm diameter
</TD></TR><TR><TD> P2a </TD><TD><A href="http://partsregistry.org/Part:BBa_J23113" class="external text" rel="nofollow">J23150</A></TD><TD>Medium promoter GFP tester</TD><TD>pMB1</TD><TD>Kan</TD><TD>no growth
</TD></TR><TR><TD> P3a </TD><TD><A href="http://partsregistry.org/Part:BBa_J23113" class="external text" rel="nofollow">J23151</A></TD><TD>High promoter GFP tester</TD><TD>pMB1</TD><TD>Kan</TD><TD>8 pink colonies of approx 7mm diameter
</TD></TR><TR><TD> P12 </TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pETDuet-1</A></TD><TD>Low copy plasmid</TD><TD>pBR322-derived ColE1</TD><TD>Amp</TD><TD>non-specific Lawn; restreaks did not grow
</TD></TR><TR><TD> P13 </TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pCDFDuet-1</A></TD><TD>Low copy plasmid</TD><TD>CloDF13</TD><TD>Sm</TD><TD>41 pink colonies and some small yellow ones that may be E. coli
</TD></TR><TR><TD> P14 </TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pCOLADuet-1</A></TD><TD>Low copy plasmid</TD><TD>ColA</TD><TD>Kan</TD><TD>No growth
</TD></TR></TBODY></TABLE></DD></DL><P>S1-P1A:
</P><P>
S1-P3A:
</P><P>S1-P13:
</P><H2><SPAN class="mw-headline" id="Growing_cultures_of_E._coli_and_S1_with_BioBrick_Parts_or_Duet_Vectors_in_them">Growing cultures of E. coli and S1 with BioBrick Parts or Duet Vectors in them</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F23:_2">6/23:</SPAN></H3><P>Picked colonies of E. coli and S1 and grew overnight at 37 and 30 degrees, respectively.
</P><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Strain</B></TD><TD align="center" style="background:#eeeeee;"><B>Plasmid Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Registry Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Plate Information</B></TD><TD align="center" style="background:#eeeeee;"><B>Marker</B></TD><TD align="center" style="background:#eeeeee;"><B>Colonies Picked</B></TD></TR><TR><TD> E. coli </TD><TD> P15</TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">A1</A></TD><TD>6/18 electrocompetent</TD><TD>Cm</TD><TD>2
</TD></TR><TR><TD> TOP10 </TD><TD> P4</TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pACYC duet</A></TD><TD>6/19 restreak electrocomp</TD><TD>Cm</TD><TD>2
</TD></TR><TR><TD> TOP10 </TD><TD> P7</TD><TD><A href="http://partsregistry.org/Part:pSB1A2" class="external text" rel="nofollow">pSB1A2</A></TD><TD>6/19 restreak electrocomp big and small</TD><TD>Cm</TD><TD>2 (1 from big, 1 from small)
</TD></TR><TR><TD> TOP10 </TD><TD> P8</TD><TD><A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A></TD><TD>6/19 restreak electrocomp big and small</TD><TD>Amp</TD><TD>2 (1 from big, 1 from small)
</TD></TR><TR><TD> TOP10 </TD><TD> P9</TD><TD><A href="http://partsregistry.org/Part:BBa_J04430" class="external text" rel="nofollow">BBa_J04430</A></TD><TD>6/19 restreak electrocomp big and small</TD><TD>Amp</TD><TD>2 (1 from big, 1 from small)
</TD></TR><TR><TD> TOP10 </TD><TD> P10</TD><TD><A href="http://partsregistry.org/Part:BBa_I715038" class="external text" rel="nofollow">BBa_I715038</A></TD><TD>6/19 restreak electrocomp big and small</TD><TD>Amp</TD><TD>2 (1 from big, 1 from small)
</TD></TR><TR><TD> S1 </TD><TD> P4</TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pACYC duet</A></TD><TD>6/18 electrocomp</TD><TD>Cm</TD><TD>2
</TD></TR><TR><TD> S1 </TD><TD> P21</TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">A1</A></TD><TD>6/19 electrocomp</TD><TD>Cm</TD><TD>2
</TD></TR><TR><TD> S1 </TD><TD> P13</TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pCDFDuet-1</A></TD><TD>6/20 electrocomp</TD><TD>Sm</TD><TD>2
</TD></TR><TR><TD> S1 </TD><TD> P1a</TD><TD><A href="http://partsregistry.org/Part:BBa_J23113" class="external text" rel="nofollow">J23113</A></TD><TD>6/20 electrocomp</TD><TD>Kan</TD><TD>3
</TD></TR><TR><TD> S1 </TD><TD> P3a</TD><TD><A href="http://partsregistry.org/Part:BBa_J23113" class="external text" rel="nofollow">J23151</A></TD><TD>6/20 electrocomp</TD><TD>Kan</TD><TD>3
</TD></TR></TBODY></TABLE></DD></DL><P>Total number of cultures: 24
</P><H3><SPAN class="mw-headline" id="6.2F24:">6/24:</SPAN></H3><P>None of the samples in the LB amp media grew.  All the others grew and then were miniprepped and stored in glycerol.
</P><UL><LI>Glycerol stocks: 250 uL of culture in 500 uL of 20% LB glycerol.
</LI></UL><H4><SPAN class="mw-headline" id="Concentrations_of_DNA_from_minipreps:">Concentrations of DNA from minipreps:</SPAN></H4><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Plasmid</B></TD><TD align="center" style="background:#eeeeee;"><B>Organism</B></TD><TD align="center" style="background:#eeeeee;"><B>Concentration (ng/uL)</B></TD></TR><TR><TD> P1A a </TD><TD>S1</TD><TD>92.54
</TD></TR><TR><TD> P1A b </TD><TD>S1</TD><TD>55.34
</TD></TR><TR><TD> P1A c </TD><TD>S1</TD><TD>58.21
</TD></TR><TR><TD> P3A a </TD><TD>S1</TD><TD>62.43
</TD></TR><TR><TD> P3A b </TD><TD>S1</TD><TD>168.41
</TD></TR><TR><TD> P3A c </TD><TD>S1</TD><TD>151.49
</TD></TR><TR><TD> P4 </TD><TD>S1</TD><TD>150.32
</TD></TR><TR><TD> P13 </TD><TD>S1</TD><TD>435.39
</TD></TR><TR><TD> P21 </TD><TD>S1</TD><TD>219.33
</TD></TR><TR><TD> P4A a  </TD><TD>TOP10</TD><TD>49.81
</TD></TR><TR><TD> P4A b </TD><TD>TOP10</TD><TD>47.22
</TD></TR><TR><TD> P21A a </TD><TD>TOP10</TD><TD>61.16
</TD></TR><TR><TD> P21A b </TD><TD>TOP10</TD><TD>55.71
</TD></TR></TBODY></TABLE></DD></DL><P>6/26: Amp cultures still did not grow in liquid media, neither at 1:1000 or a 1:2000 dilution of amp
</P><H2><SPAN class="mw-headline" id="Transformation_of_P5_and_P6_into_E3">Transformation of P5 and P6 into E3</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F24:_2">6/24:</SPAN></H3><P>P5 and P6 were punched from the book and transformed in E3 (the death resistant strain)
</P><P>6/25: No Growth
</P><H2><SPAN class="mw-headline" id="16S_PCR_to_Identify_S1_and_E._Coli">16S PCR to Identify S1 and E. Coli</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F24:_3">6/24:</SPAN></H3><P>Performed PCR as per <A href="http://openwetware.org/wiki/IGEM:Harvard/2008/Lab_Notebooks/GenProtocols#Colony_PCR_protocol" class="external text" rel="nofollow">protocol</A> listed in General Protocols.  
</P><P>Primer Sets
</P><UL><LI>1 = Shewanella F and Shewanella R
</LI><LI>2 = E. coli F and E. coli R
</LI><LI>3 = Common F and Common R
</LI></UL><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Tube #</B></TD><TD align="center" style="background:#eeeeee;"><B>Strain/ Colony of Template DNA</B></TD><TD align="center" style="background:#eeeeee;"><B>Primer Set Used</B></TD><TD align="center" style="background:#eeeeee;"><B>Volume of Template DNA</B></TD><TD align="center" style="background:#eeeeee;"><B>Volume of Primer</B></TD><TD align="center" style="background:#eeeeee;"><B>Volume of Supermix</B></TD><TD align="center" style="background:#eeeeee;"><B>Volume of Water</B></TD></TR><TR><TD> 1</TD><TD> S1 P1a</TD><TD>1 (SF &amp; SR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 2</TD><TD> S1 P1a</TD><TD>1 (SF &amp; SR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 3</TD><TD> S1 P1a</TD><TD>2 (EF &amp; ER)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 4</TD><TD> S1 P1a</TD><TD>2 (EF &amp; ER)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 5</TD><TD> S1 P1a</TD><TD>3 (CF &amp; CR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 6</TD><TD> S1 P1a</TD><TD>3 (CF &amp; CR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 7</TD><TD> S1 P3a</TD><TD>1 (SF &amp; SR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 8</TD><TD> S1 P3a</TD><TD>1 (SF &amp; SR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 9</TD><TD> S1 P3a</TD><TD>2 (EF &amp; ER)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 10</TD><TD> S1 P3a</TD><TD>2 (EF &amp; ER)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 11</TD><TD> S1 P3a</TD><TD>3 (CF &amp; CR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 12</TD><TD> S1 P3a</TD><TD>3 (CF &amp; CR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 13</TD><TD> S1 MR-1</TD><TD>1 (SF &amp; SR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 14</TD><TD> S1 MR-1</TD><TD>1 (SF &amp; SR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 15</TD><TD> S1 MR-1</TD><TD>2 (EF &amp; ER)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 16</TD><TD> S1 MR-1</TD><TD>2 (EF &amp; ER)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 17</TD><TD> S1 MR-1</TD><TD>3 (CF &amp; CR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 18</TD><TD> S1 MR-1</TD><TD>3 (CF &amp; CR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 19</TD><TD> E. coli P1a</TD><TD>1 (SF &amp; SR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 20</TD><TD> E. coli P1a</TD><TD>1 (SF &amp; SR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 21</TD><TD> E. coli P1a</TD><TD>2 (EF &amp; ER)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 22</TD><TD> E. coli P1a</TD><TD>2 (EF &amp; ER)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 23</TD><TD> E. coli P1a</TD><TD>3 (CF &amp; CR)</TD><TD>1uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR><TR><TD> 24</TD><TD> E. coli P1a</TD><TD>3 (CF &amp; CR)</TD><TD>3uL</TD><TD>1uL (each)</TD><TD>45uL</TD><TD>2uL
</TD></TR></TBODY></TABLE></DD></DL><H3><SPAN class="mw-headline" id="6.2F25">6/25</SPAN></H3><P>Gel results from PCR:
</P><P>Key:
</P><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Well #</B></TD><TD align="center" style="background:#eeeeee;"><B>Strain/ Colony</B></TD><TD align="center" style="background:#eeeeee;"><B>Primer Set Used</B></TD><TD align="center" style="background:#eeeeee;"><B>Volume of Template DNA</B></TD></TR><TR><TD> 1</TD><TD> 1 Kb ladder</TD><TD>N/A</TD><TD>N/A
</TD></TR><TR><TD> 2</TD><TD> S1 P3a</TD><TD>1 (SF &amp; SR)</TD><TD>3uL
</TD></TR><TR><TD> 3</TD><TD> S1 P3a</TD><TD>1 (SF &amp; SR)</TD><TD>1uL
</TD></TR><TR><TD> 4</TD><TD> S1 P3a</TD><TD>2* (EF &amp; ER)</TD><TD>1uL
</TD></TR><TR><TD> 5</TD><TD> S1 P3a</TD><TD>3 (CF &amp; CR)</TD><TD>1uL
</TD></TR><TR><TD> 6</TD><TD> S1 P3a</TD><TD>3 (CF&amp; CR)</TD><TD>3uL
</TD></TR><TR><TD> 7</TD><TD> S1 P1a</TD><TD>1 (SF &amp; SR)</TD><TD>3uL
</TD></TR><TR><TD> 8</TD><TD> S1 P1a</TD><TD>1 (SF &amp; SR)</TD><TD>1uL
</TD></TR><TR><TD> 9</TD><TD> S1 P1a</TD><TD>2 (EF &amp; ER)</TD><TD>1uL
</TD></TR><TR><TD> 10</TD><TD> S1 P1a</TD><TD>2 (EF &amp; ER)</TD><TD>3uL
</TD></TR><TR><TD> 11</TD><TD> S1 P1a</TD><TD>3 (CF &amp; CR)</TD><TD>3uL
</TD></TR><TR><TD> 12</TD><TD> S1 P1a</TD><TD>3 (CF &amp; CR)</TD><TD>1uL
</TD></TR></TBODY></TABLE></DD></DL><P>Note: S1 P3a With Primer Set 2 and 3 uL of template DNA evaporated during PCR so was not included in the gel.
</P><P>Key:
</P><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Well #</B></TD><TD align="center" style="background:#eeeeee;"><B>Strain/ Colony</B></TD><TD align="center" style="background:#eeeeee;"><B>Primer Set Used</B></TD><TD align="center" style="background:#eeeeee;"><B>Volume of Template DNA</B></TD></TR><TR><TD> 1</TD><TD> 1 Kb ladder</TD><TD>N/A</TD><TD>N/A
</TD></TR><TR><TD> 2</TD><TD> S1 MR-1</TD><TD>1 (SF &amp; SR)</TD><TD>1uL
</TD></TR><TR><TD> 3</TD><TD> S1 MR-1</TD><TD>2 (EF &amp; ER)</TD><TD>1uL
</TD></TR><TR><TD> 4</TD><TD> S1 MR-1</TD><TD>3 (CF &amp; CR)</TD><TD>1uL
</TD></TR><TR><TD> 5</TD><TD> 100 bp Ladder</TD><TD>N/A</TD><TD>N/A
</TD></TR><TR><TD> 6</TD><TD> E.coli P1a</TD><TD>1 (SF &amp; SR)</TD><TD>1uL
</TD></TR><TR><TD> 7</TD><TD> E.coli P1a</TD><TD>1 (SF &amp; SR)</TD><TD>3uL
</TD></TR><TR><TD> 8</TD><TD> E.coli P1a</TD><TD>2 (EF &amp; ER)</TD><TD>1uL
</TD></TR><TR><TD> 9</TD><TD> E.coli P1a</TD><TD>2 (EF &amp; ER)</TD><TD>3uL
</TD></TR><TR><TD> 10*</TD><TD> E.coli P1a</TD><TD>3 (CF &amp; CR)</TD><TD>1uL
</TD></TR><TR><TD> 11</TD><TD> E.coli P1a</TD><TD>3 (CF &amp; CR)</TD><TD>3uL
</TD></TR><TR><TD> 12</TD><TD> 1 Kb Ladder</TD><TD>N/A</TD><TD>N/A
</TD></TR></TBODY></TABLE></DD></DL><P>Note: There was a problem with the gel in Well 10, which most likely accounts for the lighter band.
</P><H2><SPAN class="mw-headline" id="Remy.27s_pZ_vector_system_Plasmids">Remy's pZ vector system Plasmids</SPAN></H2><P><A href="/wiki/images/d/dd/Bujard_lutz.pdf" class="internal" title="Bujard lutz.pdf"> Independent and tight regulation of transcriptional units in Escherichia coli via the LacR/O, the TetR/O and AraC/I1-I2 regulatory elements</A></P><P>6/25: transfer colonies to liquid media
</P><P>6/26: mini prep plasmids
</P><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Registry Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Description</B></TD><TD align="center" style="background:#eeeeee;"><B>Origin</B></TD><TD align="center" style="background:#eeeeee;"><B>Marker</B></TD><TD align="center" style="background:#eeeeee;"><B>Regulatory unit</B></TD><TD align="center" style="background:#eeeeee;"><B>Concentration ng/uL</B></TD></TR><TR><TD> P27 </TD><TD>pZS4int2</TD><TD>repressor plasmid for chromosomal integration</TD><TD>pSC101</TD><TD>Sm</TD><TD>Constitutive expression of LacI and TetR</TD></TR><TR><TD> P28 </TD><TD>pZE21</TD><TD>GFP</TD><TD>ColE1</TD><TD>Kan</TD><TD>PLtctO-1</TD></TR><TR><TD> P29 </TD><TD>pZS*2R</TD><TD>Venus (mutant YFP)</TD><TD>pSC101*</TD><TD>Kan</TD><TD>PL (Plambda)</TD></TR><TR><TD> P30 </TD><TD>pZA32</TD><TD>YFP</TD><TD>p15A</TD><TD>Cm</TD><TD>PLlacO-1</TD></TR><TR><TD> P31 </TD><TD>pZS*1R</TD><TD>Venus</TD><TD>pSC101*</TD><TD>amp</TD><TD>PL (Plambda)</TD></TR><TR><TD> P32 </TD><TD>pZA32</TD><TD>Venus</TD><TD>p15A</TD><TD>Cm</TD><TD>PLlacO-1</TD></TR><TR><TD> P33 </TD><TD>pZA31</TD><TD>Venus</TD><TD>p15A</TD><TD>Cm</TD><TD>PLtctO-1
</TD></TR><TR><TD> P34 </TD><TD>pZE12</TD><TD>GFP</TD><TD>ColE1</TD><TD>Amp</TD><TD>PLlacO-1</TD></TR><TR><TD> P35 </TD><TD>pLDR8</TD><TD>Accessory for chromosomal integration</TD><TD>??</TD><TD>???</TD><TD>??</TD></TR></TBODY></TABLE></DD></DL><P>Also two strains with integrated copies of repressors, low level constitutive promoters
</P><UL><LI>MC4100 Z1 - repressor lac and tet (Sm resistant)
</LI><LI>DH10B Z2 - repressor lac and tet (Sm resistant)
</LI></UL><H2><SPAN class="mw-headline" id="Re-Transformation_of_BioBrick_Parts_in_E1_and_E3">Re-Transformation of BioBrick Parts in E1 and E3</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F26:">6/26:</SPAN></H3><P>Re-transformed BioBrick parts in E1 because amp plates weren't working.  Also re-transformed P5 and P6 in E3.
</P><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Registry Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Description</B></TD><TD align="center" style="background:#eeeeee;"><B>Origin</B></TD><TD align="center" style="background:#eeeeee;"><B>Marker</B></TD><TD align="center" style="background:#eeeeee;"><B>Transformed? (6/26)</B></TD><TD align="center" style="background:#eeeeee;"><B>Picked colonies?</B></TD><TD align="center" style="background:#eeeeee;"><B>Miniprepped?</B></TD><TD align="center" style="background:#eeeeee;"><B>Made glycerol stocks?</B></TD></TR><TR><TD> P7 </TD><TD><A href="http://partsregistry.org/Part:pSB1A3" class="external text" rel="nofollow">pSB1A3</A>*</TD><TD>High copy plasmid with Amp resistance</TD><TD>ColE1</TD><TD>Amp</TD><TD>Failed in E1</TD></TR><TR><TD> P8 </TD><TD><A href="http://partsregistry.org/Part:BBa_J04450" class="external text" rel="nofollow">BBa_J04450</A></TD><TD>RFP with LacI promoter</TD><TD>pMB1</TD><TD>Amp</TD><TD>Failed in E1</TD></TR><TR><TD> P9 </TD><TD><A href="http://partsregistry.org/Part:BBa_J04430" class="external text" rel="nofollow">BBa_J04430</A></TD><TD>GFP with LacI promoter</TD><TD>pMB1</TD><TD>Amp</TD><TD>Failed in E1</TD></TR><TR><TD> P10 </TD><TD><A href="http://partsregistry.org/Part:BBa_I715038" class="external text" rel="nofollow">BBa_I715038</A></TD><TD>T7 Polymerase with LacI promoter</TD><TD>pMB1</TD><TD>Kan/Amp</TD><TD>Failed in E1</TD></TR><TR><TD> P36 </TD><TD><A href="http://partsregistry.org/Part:BBa_I763004" class="external text" rel="nofollow">BBa_I763004</A></TD><TD>GFP + IPTG + LVA</TD><TD>Kan/Amp</TD><TD>Failed in E1</TD></TR><TR><TD> P5 (2007) </TD><TD><A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3</A>*</TD><TD>Low-Medium copy vector (w/ death gene)</TD><TD>p15A</TD><TD>Kan</TD><TD>Yes, in E3</TD><TD>2 (A and B spilled)</TD></TR><TR><TD> P6 (2007) </TD><TD><A href="http://partsregistry.org/Part:BBa-E1010" class="external text" rel="nofollow">BBa-E1010</A></TD><TD>RFP only (w/ death gene)</TD><TD>pMB1</TD><TD>Kan</TD><TD>Failed in E3</TD></TR><TR><TD> P5 (2008) </TD><TD><A href="http://partsregistry.org/Part:pSB3K3" class="external text" rel="nofollow">pSB3K3</A>*</TD><TD>Low-Medium copy vector (w/ death gene)</TD><TD>p15A</TD><TD>Kan</TD><TD>Failed in E3</TD></TR></TBODY></TABLE></DD></DL><P>Notes: 
</P><UL><LI>Control was also done with untransformed E3 on LB plates.
</LI><LI>pSB1A3 was used instead of pSB1A2 because the Parts Registry lists pSB1A3 as the replacement for pSB1A2.</LI><LI>P5 and P6 were taken from Spring 2007 plates, and iGEM 2008 P5 was punched out and transformed again.
</LI></UL><H3><SPAN class="mw-headline" id="6.2F27:">6/27:</SPAN></H3><P>P5 (2007) plate was put in incubator for longer, and will be picked on Sunday along with the other transformations done today.
</P><DL><DD><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Registry Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Description</B></TD><TD align="center" style="background:#eeeeee;"><B>Origin</B></TD><TD align="center" style="background:#eeeeee;"><B>Marker</B></TD><TD align="center" style="background:#eeeeee;"><B>Transformed? (6/27)</B></TD><TD align="center" style="background:#eeeeee;"><B>Picked colonies? (6/29)</B></TD><TD align="center" style="background:#eeeeee;"><B>Miniprepped?</B></TD><TD align="center" style="background:#eeeeee;"><B>Made glycerol stocks?</B></TD></TR><TR><TD> P12 </TD><TD><A href="http://www.merckbiosciences.co.uk/html/NVG/duettable.html" class="external text" rel="nofollow">pETDuet-1</A></TD><TD>Low copy plasmid</TD><TD>pBR322-derived ColE1</TD><TD>Amp</TD><TD>in E1</TD><TD>2</TD></TR><TR><TD> P37 </TD><TD><A href="http://partsregistry.org/wiki/index.php/Part:BBa_I722007" class="external text" rel="nofollow">BBa_I722007</A></TD><TD>Constructive expression LacI w/ pTetR promoter</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (A and B didn't work)</TD></TR><TR><TD> P38 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23114" class="external text" rel="nofollow">BBa_J23114</A></TD><TD>High constitutive promoter</TD><TD>Amp</TD><TD>in E1</TD><TD> 2</TD></TR><TR><TD> P38 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23114" class="external text" rel="nofollow">BBa_J23114</A></TD><TD>High constitutive promoter</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (A and B didn't work)</TD></TR><TR><TD> P39 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23113" class="external text" rel="nofollow">BBa_J23113</A></TD><TD>Low constitutive promoter</TD><TD>Amp</TD><TD>in E1</TD><TD>2 (B didn't work) </TD></TR><TR><TD> P39 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_J23113" class="external text" rel="nofollow">BBa_J23113</A></TD><TD>Low constitutive promoter</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (B did not work)</TD></TR><TR><TD> P40 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0032" class="external text" rel="nofollow">BBa_B0032</A></TD><TD>Ribosome Binding Site</TD><TD>Amp</TD><TD>in E1</TD><TD>2 (B didn't work) </TD></TR><TR><TD> P40 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0032" class="external text" rel="nofollow">BBa_B0032</A></TD><TD>Ribosome Binding Site</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (A did not work)</TD></TR><TR><TD> P41 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0010" class="external text" rel="nofollow">BBa_B0010</A></TD><TD>Transcriptional Terminator</TD><TD>Amp</TD><TD>in E1</TD><TD>2 (A and B didn't work) </TD></TR><TR><TD> P41 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_B0010" class="external text" rel="nofollow">BBa_B0010</A></TD><TD>Transcriptional Terminator</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (Neither worked)</TD></TR><TR><TD> P42 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_C0012" class="external text" rel="nofollow">BBa_C0012</A></TD><TD>LacI coding region</TD><TD>Amp</TD><TD>in E1</TD><TD>2 (A and B didn't work) </TD></TR><TR><TD> P42 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_C0012" class="external text" rel="nofollow">BBa_C0012</A></TD><TD>LacI coding region</TD><TD>Amp</TD><TD>in E1</TD><TD> 1 (didn't work)</TD></TR><TR><TD> P43 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_C0040" class="external autonumber" rel="nofollow">[1]</A></TD><TD>TetR coding region</TD><TD>Amp</TD><TD>in E1</TD><TD>2 (A and B didn't work) </TD></TR><TR><TD> P43 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_C0040" class="external autonumber" rel="nofollow">[2]</A></TD><TD>TetR coding region</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (B didn't work)</TD></TR><TR><TD> P44 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_C0051" class="external text" rel="nofollow">BBa_C0051</A></TD><TD>cI lambda</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (A and B didn't work)</TD></TR><TR><TD> P44 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_C0051" class="external text" rel="nofollow">BBa_C0051</A></TD><TD>cI lambda</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (B didn't work)</TD></TR><TR><TD> P45 </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_E0240" class="external text" rel="nofollow">BBa_E0240</A></TD><TD>GFP only w/ RBS &amp; terminator</TD><TD>Amp</TD><TD>in E1</TD><TD>2 (A and B didn't work)</TD></TR><TR><TD> P45 (2007) </TD><TD><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_E0240" class="external text" rel="nofollow">BBa_E0240</A></TD><TD>GFP only w/ RBS &amp; terminator</TD><TD>Amp</TD><TD>in E1</TD><TD> 2</TD></TR><TR><TD> P46 </TD><TD><A href="http://partsregistry.org/Part:BBa_I51020" class="external text" rel="nofollow">BBa_I51020</A></TD><TD>Base Vector</TD><TD>Amp</TD><TD>in E3</TD><TD>2 </TD></TR><TR><TD> P47 </TD><TD><A href="http://partsregistry.org/Part:BBa_P1003" class="external text" rel="nofollow">BBa_P1003</A></TD><TD>Kan Resistance Cassette</TD><TD>Kan</TD><TD>No in E1</TD><TD> 0</TD></TR><TR><TD> P47 (2007) </TD><TD><A href="http://partsregistry.org/Part:BBa_P1003" class="external text" rel="nofollow">BBa_P1003</A></TD><TD>Kan Resistance Cassette</TD><TD>Kan</TD><TD>in E1</TD><TD> 2 (A and B spilled)</TD></TR><TR><TD> P48 </TD><TD><A href="http://partsregistry.org/Part:BBa_P1004" class="external text" rel="nofollow">BBa_P1004</A></TD><TD>Cm Resistance Cassette</TD><TD>Cm</TD><TD>No in E1</TD><TD> 0</TD></TR><TR><TD> P48 (2007) </TD><TD><A href="http://partsregistry.org/Part:BBa_P1004" class="external text" rel="nofollow">BBa_P1004</A></TD><TD>Cm Resistance Cassette</TD><TD>Cm</TD><TD>in E1</TD><TD>2 </TD></TR><TR><TD> P49 </TD><TD><A href="http://partsregistry.org/Part:BBa_P1002" class="external text" rel="nofollow">BBa_P1002</A></TD><TD>Amp Resistance Cassette</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (B didn't work) </TD></TR><TR><TD> P49 (2007) </TD><TD><A href="http://partsregistry.org/Part:BBa_P1002" class="external text" rel="nofollow">BBa_P1002</A></TD><TD>Amp Resistance Cassette</TD><TD>Amp</TD><TD>in E1</TD><TD> 2</TD></TR><TR><TD> P50 </TD><TD><A href="http://partsregistry.org/Part:BBa_P1005" class="external text" rel="nofollow">BBa_P1005</A></TD><TD>Tet Resistance Cassette</TD><TD>Amp</TD><TD>in E1</TD><TD> 2 (A and B didn't work) </TD></TR><TR><TD> P50 (2007) </TD><TD><A href="http://partsregistry.org/Part:BBa_P1005" class="external text" rel="nofollow">BBa_P1005</A></TD><TD>Tet Resistance Cassette</TD><TD>Amp</TD><TD>in E1</TD><TD> 2</TD></TR></TBODY></TABLE></DD></DL><H2><SPAN class="mw-headline" id="McConkey_Plates_to_Distinguish_E._coli_from_Shewanella">McConkey Plates to Distinguish E. coli from Shewanella</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F24">6/24</SPAN></H3><P>Plates streaked and put in incubator.
</P><H3><SPAN class="mw-headline" id="6.2F25:">6/25:</SPAN></H3><P>
Top Left: E1
Bottom Left: E1-P1a
Top Right: S1
Bottom Right: S1-P1a</P><P>
Top Left: chemically competent E. coli
Bottom Left: E1-P3
Top Right: WT S1
Bottom Right: S1-P3</P><P>
Top Left: E1
Bottom Left: E1-A1 (P21)
Top Right: S1  
Bottom Right: S1-A1 (P21)</P><H2><SPAN class="mw-headline" id="Growing_Shewie_anaerobically">Growing Shewie anaerobically</SPAN></H2><H3><SPAN class="mw-headline" id="6.2F25:_2">6/25:</SPAN></H3><P>Added lactate and birnesite to LB in a vial, bubbled N2 gas to degas it and then added Shewie at concentrations of 100 uL and 500 uL, while keeping a control with no Shewie added. Vials were placed in 30 degree shaking incubator overnight.
</P><H3><SPAN class="mw-headline" id="6.2F26:_2">6/26:</SPAN></H3><P>All vials, including the control, turned light yellow (the color of the LB).  This was most likely due to contamination, which was partly confirmed by looking at the vials through the microscope.  This was not unexpected because the vials we used had not been autoclaved, and it was very difficult to maintain sterile conditions and precise concentrations of birnesite/ lactate.
</P><H2><SPAN class="mw-headline" id="Questions.2F_Concerns">Questions/ Concerns</SPAN></H2><P>Regarding transformations of BioBrick Parts:
</P><OL><LI>Given the parts that we've succeeded in transforming in either E. coli, or Shewanella, or both, what is our plan for designing vectors?  How many plasmids will we need?
</LI><LI>How should we incorporate Remy's plasmids into those designs?
</LI><LI>To what extent can we take parts from or use the Duet vectors?
</LI><LI>Can we confirm that pMB1 does work in Shewanella, contrary to the 1997 paper that suggests otherwise?
</LI><LI>If we have successful transformations with both the 2007 and 2008 versions of a BioBrick part, should we prefer one over the other?
</LI></OL><P>Regarding anaerobic Shewie growth: (Many of these questions may be the same as those posed by the Widgetry group.)
</P><OL><LI>What are the optimal concentrations of lactate and birnesite that we should use?
</LI><LI>Tips on maintaining sterility/ preventing contamination?
</LI><LI>Orianna's paper lists three different media that she used for various purposes-- what does our basic media need?
</LI><LI>How much Shewie should we be using?  Should we be adding them at a certain growth phase or OD?
</LI></OL><H1><SPAN class="mw-headline" id="Light">Light</SPAN></H1><H2><SPAN class="mw-headline" id="Transformations_of_P11.2C_P15-20">Transformations of P11, P15-20</SPAN></H2><P><B>ENDED IN FAILURE: SEE NEW TRANSFORMATION BELOW</B></P><H3><SPAN class="mw-headline" id="Colony_counts">Colony counts</SPAN></H3><P>6/23:
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Name</B></TD><TD align="center" style="background:#eeeeee;"><B>Description</B></TD><TD align="center" style="background:#eeeeee;"><B>Origin</B></TD><TD align="center" style="background:#eeeeee;"><B>Size</B></TD><TD align="center" style="background:#eeeeee;"><B>Marker</B></TD><TD align="center" style="background:#eeeeee;"><B>Number of Colonies</B></TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_M30109" class="external text" rel="nofollow">P11</A></TD><TD>Light responsive system, dual regulation</TD><TD>pUC19-derived pMB1</TD><TD>4333</TD><TD>Amp/Cm</TD><TD>224
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_I13522" class="external text" rel="nofollow">P15</A></TD><TD>GFP with Tet promoter</TD><TD>ColE1</TD><TD>937</TD><TD>Amp</TD><TD>360
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_R0040" class="external text" rel="nofollow">P16</A></TD><TD>Tet repressible promoter</TD><TD>pMB1</TD><TD>54</TD><TD>Amp</TD><TD>264
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_Q04400" class="external text" rel="nofollow">P17</A></TD><TD>Inverter (TetR w/o promoter and Tet promoter)</TD><TD>OriS, P1 lytic, F1</TD><TD>902</TD><TD>Kan</TD><TD>2
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_R0051" class="external text" rel="nofollow">P18</A></TD><TD>lambda promoter (cI regulated)</TD><TD>pMB1</TD><TD>49</TD><TD>Amp</TD><TD>112
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_I763007" class="external text" rel="nofollow">P19</A></TD><TD>RFP with lambda promoter</TD><TD>pMB1</TD><TD>918</TD><TD>Amp</TD><TD>104
</TD></TR><TR><TD><A href="http://partsregistry.org/Part:BBa_J04431" class="external text" rel="nofollow">P20</A></TD><TD>GFP (LVA tagged) with Lac promoter</TD><TD>pMB1</TD><TD>1122</TD><TD>Amp, Kan</TD><TD>136
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Restreaking">Restreaking</SPAN></H3><P>06/23: P15, which has GFP under a constitutive Tet promoter did not appear to fluoresce (none of the fluorescent constructs did), so we restreaked all the of the plates (with the same antibiotic on which they were originally grown). They were left in the 37 degree incubator along with a blank Kan and a blank Amp plate (- controls).
</P><P>06/24: All of the restreaked plates grew and were streaked to individual colonies. Both the Kan and Amp negative controls (no cells) were blank. However, p15 (GFP under pTet) did not appear to glow.
</P><H3><SPAN class="mw-headline" id="Overnight_cultures">Overnight cultures</SPAN></H3><P>06/23: The above was repeated in 5mL LB liquid cultures instead of plates.
</P><DL><DD>06/24: None of the cultures in LB Amp grew (E1P17 in LB Kan did grow). We recultured the other samples in LB Amp and made a master plate with patches from the same colonies that were cultured. Since others seemed to get similar problems (no growth in liquid culture but growth on plates for Amp), we made fresh LB Amp (from freshly made Amp stock) and repeated the cultures.
</DD></DL><DL><DD>06/25: These LB-Amp cultures did not grow in the new medium either. We plated them on new LB amp plates (new plates 1X amp or 1/1000) and cultured them in low concentration LB amp (0.5X or 1/2000 amp).
</DD></DL><DL><DD>06/26: None of the cultures grew on either of the liquid media (1X and .5X LB-Amp). Additionally, none of the master plate colonies restreaked onto the new LB-Amp plates that we poured grew. This suggests to us that the original LB-Amp plates were not selective enough (perhaps Amp broke down). As a control that our bacteria were capable of growth, we inoculated a plain LB liquid culture with each of the 6 colonies from the master plate. This culture did grow (cloudy). A plain LB liquid culture left as a negative control did not.
</DD></DL><H2><SPAN class="mw-headline" id="Miniprep_of_P17">Miniprep of P17</SPAN></H2><P><B>ENDED IN FAILURE: SEE NEW TRANSFORMATION BELOW</B></P><P>06/24: Miniprep of E1P17. The DNA is in the -20 Â°C iGEM freezer. The DNA concentration is 54.8 ng/Î¼L.
</P><H3><SPAN class="mw-headline" id="Restriction_Digest">Restriction Digest</SPAN></H3><P>06/24: Digest of P17 DNA with Xba1 and Spe1. The entire plasmid is 4425 bp and the short fragment (containing only the BioBrick) is 902 bp.
</P><P>Digest mixture (incubated overnight at 37 Â°C): 
</P><P>15 Î¼l DNA
</P><P>6.25 Î¼l water
</P><P>2.5 Î¼l NEB Buffer 2
</P><P>0.25 Î¼l 100X BSA
</P><P>1 Î¼l Xba1
</P><P>1 Î¼l Spe1
</P><P>Gel of digest products (6/25)
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR><TR><TD>2</TD><TD>Digest. Note that these are not what would be expected for the correct plasmid (902 band for part and 4.5kb for vector). Even if there are cutting issues, the sum of the band sizes is around 2kb too low.
</TD></TR><TR><TD>3</TD><TD>Undigested plasmid
</TD></TR><TR><TD>4</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR></TBODY></TABLE><P>06/25: Since the previous digest was unsuccessful, we digested the same plasmid with different combinations of restriction enzymes. We also tested two other plasmids from the chemical group (P4 and P21). The digest mixtures were the same as above (6/24) with the appropriate enzyme swapped in. For the single digests, the volume of water was 6.75 Î¼l and all else remained constant. Again, incubation is overnight at 37 Â°C.
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR><TR><TD>2</TD><TD>P17 with XbaI, SpeI
</TD></TR><TR><TD>3</TD><TD>P17 with EcoRI, SpeI
</TD></TR><TR><TD>4</TD><TD>P17 with EcoRI, PstI
</TD></TR><TR><TD>5</TD><TD>P17 with XbaI, PstI
</TD></TR><TR><TD>6</TD><TD>P17 with XbaI
</TD></TR><TR><TD>7</TD><TD>P17 with SpeI
</TD></TR><TR><TD>8</TD><TD>P4 with XbaI, SpeI
</TD></TR><TR><TD>9</TD><TD>P4 with EcoRI, SpeI
</TD></TR><TR><TD>10</TD><TD>P21 with XbaI, PstI
</TD></TR><TR><TD>11</TD><TD>P21 with EcoRI, PstI
</TD></TR><TR><TD>12</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR></TBODY></TABLE><P>From the lanes with P17, which again indicate that the plasmid we have from the miniprep is not the right size, we conclude that the P17 transformation failed. It is being repeated (see below).
</P><H2><SPAN class="mw-headline" id="Transformations_of_P22-24">Transformations of P22-24</SPAN></H2><P><B>ENDED IN FAILURE: SEE NEW TRANSFORMATION BELOW</B></P><P>06/24: Transformations occurred in duplicate (TOP10 and DH5Î±) with Amp selection.
</P><P>06/25: All plates had colonies. We inoculated liquid cultures of cells with each of these plasmids in low (0.5X or 1/2000 amp) and high (1X or 1/1000 amp) concentration LB amp.
</P><P>E1P22: 376 colonies
E1P23: 352 colonies
E1P24: 392 colonies
E2P22, E2P23, and E2P24 had too many colonies to count.
</P><DL><DD>06/26: None of the liquid cultures (both high and low Amp) had any growth. Additionally, we patched the colonies we were growing on a master plate that was of the new LB-Amp plates that we poured. No colonies grew on this plate either. This again seems to indicate that the initial transformation plates did not have enough selection (perhaps Amp was degraded).
</DD></DL><H2><SPAN class="mw-headline" id="Transformations_of_P17.2C_20.2C_25.2C_26">Transformations of P17, 20, 25, 26</SPAN></H2><P><B>SEE NEW TRANSFORMATION BELOW</B></P><P>06/25: Transformed P17, P20, P25, and P26 in DH5Î± cells. P17 and P20 have Amp and Kan resistance, but since AmpR cells don't seem to grow in the liquid media, we're repeating the process by plating the cells on Kan (and eventually grow them on LB-Kan liquid medium). P25 and P26 are new and have Amp resistance.
</P><P><B>06/26</B>: P20 (kan plate) had 4 colonies and P25 (amp plate) had 1 colony. The other plates (with P17, P20, and P26) did not have any colonies. The kan and amp negative control plates (DH5Î± cells with no DNA) also had no colonies.
</P><H2><SPAN class="mw-headline" id="New_transformations:_P11.2C_P15-20.2C_P22-26">New transformations: P11, P15-20, P22-26</SPAN></H2><P>06/26
</P><P>We transformed TOP10 cells using parts from both the 2008 and 2007 registries, apart from P11 and P19 which were not available in the 2007 registry. 
</P><P>The protocol for getting DNA from the 2007 registry is as follows:
</P><P>1. Puncture a hole in the foil with a pipet.
</P><P>2. Add 15 Î¼l H<SUB>2</SUB>O and wait for about 5 minutes.
</P><P>3. Add 1 Î¼l DNA to 50 Î¼l bacteria.
</P><P>P17 and P20 were plated on Kan plates. All other samples were on Amp plates that we made (at 1X or 1/1000 amp).
</P><H3><SPAN class="mw-headline" id="Transformation_results">Transformation results</SPAN></H3><P>Both negative plate controls (Amp and Kan) did not have any colonies. The &quot;mock transformations&quot; (no DNA) were also negative. The positive plate control had a sample of pEZ12 GFP from Remy and this plasmid also carried Amp resistance. There was a lawn of bacteria on this plate. We also did a positive transformation control that had 1 Î¼g pUC19 DNA for 50 Î¼l TOP10 cells. This DNA came with the TOP10 cells from Invitrogen and there were 26 colonies on the plate. 
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#eeeeee;"><B>Plasmid</B></TD><TD align="center" style="background:#eeeeee;"><B>Number of Colonies</B></TD></TR><TR><TD>P15 (2007)</TD><TD>112
</TD></TR><TR><TD>P16 (2008)</TD><TD>2
</TD></TR><TR><TD>P18 (2007)</TD><TD>22
</TD></TR><TR><TD>P20 (2007)</TD><TD>96
</TD></TR><TR><TD>P23 (2007)</TD><TD>40
</TD></TR><TR><TD>P24 (2007)</TD><TD>4
</TD></TR><TR><TD>P26 (2007)</TD><TD>14
</TD></TR></TBODY></TABLE><P>6/10 of the 2007 parts appear to have transformed (colonies grew) and 1/12 of the 2008 parts appear to have transformed successfully.
</P><P>Remy's positive control with GFP was fluorescent, along with P15 (GFP with Tet promoter) and P20 (LVA-tagged GFP with Lac promoter). This indicates that the transformations of P15 and P20 were successful.
</P><P>Set up liquid cultures (with master plates) for all of the plasmids that had colonies.
</P><H3><SPAN class="mw-headline" id="Minipreps_and_digests">Minipreps and digests</SPAN></H3><P>6/28: The overnight cultures grew successfully and were mini-prepped. Note that there are replicates (e.g. 15A and 15B are different tubes from the overnight cultures. * and *-less denotes replicates during minipreppring). Unfortunately, I forgot to keep culture for glycerol stocks. I performed a NotI digest (see recipe under &quot;RE Digest&quot;)- it's the standard single digest recipe, but halfed. The digest ran for the evening, and the results are below:
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR><TR><TD>2</TD><TD>P15A* digested with NotI (2079, 937 bp expected)
</TD></TR><TR><TD>3</TD><TD>P15A* uncut (3016 bp expected)
</TD></TR><TR><TD>4</TD><TD>P15A digested with NotI (2079, 937 bp expected)
</TD></TR><TR><TD>5</TD><TD>P15A uncut (3016 bp expected)
</TD></TR><TR><TD>6</TD><TD>P15B* digested with NotI (2079, 937 bp expected)
</TD></TR><TR><TD>7</TD><TD>P15B* uncut (3016 bp expected)
</TD></TR><TR><TD>8</TD><TD>P15B digested with NotI (2079, 937 bp expected)
</TD></TR><TR><TD>9</TD><TD>P15B uncut (3016 bp expected)
</TD></TR><TR><TD>10</TD><TD>P16* digested with NotI (2079, 54 bp expected)
</TD></TR><TR><TD>11</TD><TD>P16* uncut (2133 bp expected)
</TD></TR><TR><TD>12</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR></TBODY></TABLE><P>All of the P15 samples appear to have worked properly. P16 is questionable.
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR><TR><TD>2</TD><TD>P16 digested with NotI (2079, 54 bp expected)
</TD></TR><TR><TD>3</TD><TD>P16 uncut (2133 bp expected)
</TD></TR><TR><TD>4</TD><TD>P18A* digested with NotI (2079, 49 bp expected)
</TD></TR><TR><TD>5</TD><TD>P18A* uncut (2128 bp expected)
</TD></TR><TR><TD>6</TD><TD>P18A digested with NotI (2079, 49 bp expected)
</TD></TR><TR><TD>7</TD><TD>P18A uncut (2128 bp expected)
</TD></TR><TR><TD>8</TD><TD>P18B* digested with NotI (2079, 49 bp expected)
</TD></TR><TR><TD>9</TD><TD>P18B* uncut (2128 bp expected)
</TD></TR><TR><TD>10</TD><TD>P18B digested with NotI (2079, 49 bp expected)
</TD></TR><TR><TD>11</TD><TD>P18B uncut (2128 bp expected)
</TD></TR><TR><TD>12</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR><TR><TD>2</TD><TD>P20A* digested with NotI (1122, 3189 bp expected)
</TD></TR><TR><TD>3</TD><TD>P20A* uncut (4311 bp expected)
</TD></TR><TR><TD>4</TD><TD>P20A digested with NotI (1122, 3189 bp expected)
</TD></TR><TR><TD>5</TD><TD>P20A uncut (4311 bp expected)
</TD></TR><TR><TD>6</TD><TD>P23* digested with NotI
</TD></TR><TR><TD>7</TD><TD>P23* uncut
</TD></TR><TR><TD>8</TD><TD>P23 digested with NotI
</TD></TR><TR><TD>9</TD><TD>P23 uncut
</TD></TR><TR><TD>10</TD><TD>P23B* digested with NotI
</TD></TR><TR><TD>11</TD><TD>P23B* uncut
</TD></TR><TR><TD>12</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR><TR><TD>2</TD><TD>P23B digested with NotI
</TD></TR><TR><TD>3</TD><TD>P23B uncut
</TD></TR><TR><TD>4</TD><TD>P24A* digested with NotI
</TD></TR><TR><TD>5</TD><TD>P24A* uncut
</TD></TR><TR><TD>6</TD><TD>P24A digested with NotI
</TD></TR><TR><TD>7</TD><TD>P24A uncut
</TD></TR><TR><TD>8</TD><TD>P24B* digested with NotI
</TD></TR><TR><TD>9</TD><TD>P24B* uncut
</TD></TR><TR><TD>10</TD><TD>P24B digested with NotI
</TD></TR><TR><TD>11</TD><TD>P24B uncut
</TD></TR><TR><TD>12</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TH colspan="2" align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TH></TR><TR><TD align="center" style="background:#f0f0f0;"><B>Lane</B></TD><TD align="center" style="background:#f0f0f0;"><B>Contents</B></TD></TR><TR><TD>1</TD><TD><A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR><TR><TD>2</TD><TD>P26A* digested with NotI
</TD></TR><TR><TD>3</TD><TD>P26A* uncut
</TD></TR><TR><TD>4</TD><TD>P26A digested with NotI
</TD></TR><TR><TD>5</TD><TD>P26A uncut
</TD></TR><TR><TD>6</TD><TD>P26B* digested with NotI
</TD></TR><TR><TD>7</TD><TD>P26B* uncut
</TD></TR><TR><TD>8</TD><TD>P26B digested with NotI
</TD></TR><TR><TD>9</TD><TD>P26B uncut
</TD></TR><TR><TD>10</TD><TD><A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 KB ladder</A></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="RE_digest">RE digest</SPAN></H2><P>06/27
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TD></TR><TR><TD>Lane 3: <A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 kb ladder</A></TD></TR><TR><TD>Lane 4: P17 uncut (should be 4425 bp)
</TD></TR><TR><TD>Lane 5: P17 cut with NotI (should be 3503 and 902 bp)
</TD></TR><TR><TD>Lane 6: P3 uncut (should be 2750 bp)???
</TD></TR><TR><TD>Lane 7: P3 cut with NotI (should be 1831 and 919 bp)???
</TD></TR><TR><TD>Lane 8: P1 uncut (should be 3002 bp)???
</TD></TR><TR><TD>Lane 9: P1 cut with NotI (should be 2083)???
</TD></TR><TR><TD>Lane 10: <A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR></TBODY></TABLE><P>These digests were meant to test the enzymes and digest conditions. P3 and P1 were supposed to be controls. However they were Biobricks miniprepped from Shewie and it doesn't look like the transformation into Shewie worked as even the uncut DNA is not the right size. We will redo the digest with DNA from E. coli.
</P><P>I set up another overnight digest but doubled the overall volume so that there was 1 Î¼l of the restriction enzyme (NotI).
</P><P>30 Î¼l DNA
</P><P>13.5 Î¼l H<SUB>2</SUB>O
</P><P>5 Î¼l 10X NEB buffer 3
</P><P>0.5 Î¼l 100X BSA
</P><P>1 Î¼l NotI
</P><TABLE border="1" cellpadding="4" cellspacing="0" style="border:#c9c9c9 1px solid; margin: 1em 1em 1em 0; border-collapse: collapse;"><TBODY><TR><TD align="center" style="background:#f0f0f0;"><B>1.2% agarose E-gel run for 30 min and visualized using EtBr/UV</B></TD></TR><TR><TD>Lane 1: <A href="http://tools.invitrogen.com/content/sfs/manuals/15615016.pdf" class="external text" rel="nofollow">1 kb ladder</A></TD></TR><TR><TD>Lane 2: P17 cut with NotI (should be 3503 and 902 bp)
</TD></TR><TR><TD>Lane 3: P17 uncut (should be 4425 bp)
</TD></TR><TR><TD>Lane 4: P1 cut with NotI (2750, 919 bp expected)
</TD></TR><TR><TD>Lane 5: P1 uncut (3669 bp expected)
</TD></TR><TR><TD>Lane 6: P2 cut with NotI (2750, 919 bp expected)
</TD></TR><TR><TD>Lane 7: P2 uncut (3669 bp expected)
</TD></TR><TR><TD>Lane 8: P3 cut with NotI (2750, 919 bp expected)
</TD></TR><TR><TD>Lane 9: P3 uncut (3669 bp expected)
</TD></TR><TR><TD>Lane 10:<A href="http://tools.invitrogen.com/Content/SFS/Gallery/high/1368.jpg" class="external text" rel="nofollow">100 bp ladder</A></TD></TR></TBODY></TABLE><P>The bands in the cut P1-3 appear to be of the expected size. However, running a sample of the uncut miniprep gives more than one band (is this b/c an unclean miniprep?) P17 continues to digest as if it is a biobrick. However, the bands continue to be of unexpected sizes.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2008.igem.org/Team:Harvard/Dailybook/Week1/Chemical_and_Light">http://2008.igem.org/Team:Harvard/Dailybook/Week1/Chemical_and_Light</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Harvard/Dailybook/Week1/Chemical_and_Light" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Harvard/Dailybook/Week1/Chemical_and_Light" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Harvard/Dailybook/Week1/Chemical_and_Light&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Harvard/Dailybook/Week1/Chemical_and_Light&amp;oldid=69050" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2008.igem.org:Privacy_policy" title="2008.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2008.igem.org:General_disclaimer" title="2008.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>