Team:Lubbock TTU/Results
Toggle navigation
Lubbock_TTU
Home
Team
Meet the Team
Collaborations
Project
Description
Experiments
Proof of Concept
Results
Notebook
Parts
Safety
Attributions
Human Practices
Silver
Gold
Engagement
Awards
Hardware
Top
Using Lambda Red Recombinase System for Gene Knockout
PFU PCR Linear DNA
The LacI and ArgI with 280 bp homology arm gene were amplify using PFU PCR.
The size of the LacI template with spectinomycin resistance inserted between the homology arms is 1995 bp.
The size of the ArgI template with spectinomycin resistance inserted between the homology arms is 1990 bp
LacI and ArgI Transformation into E. Coli W3100
Amplified PCR LacI and ArgI products were electroporated into E. Coli W3110 harboring pREDKI plasmid. The transformed cells were plated into a gradient of different spectinomycin concertation; starting at 60 ug/ml, 70 ug/ml and 80 ug/ml and 100 ug/ml.
ArgI had grown in 80 µg/mL Spec Plates. There colonies were evaluated and determined to be false positives for recombination.
LacI had grown in 70 µg/mL Spec plates. There colonies were evaluated and determined to be false positives for recombination. However, the negative control had grown as well.
Genomic PCR of the W3110 Lambda Red Modified Cells
A genomic PCR was done to confirm the recombination.
ArgI Genomic PCR did not work in terms of Lambda Red Recombination because the gel still shows the native gene size of 1549bp not the ArgI Homology arms with the spectinomycin template which would be closer to 1995bp, but the gel reflects the size of just above 1500bp.
LacI Genomic PCR did not work in terms of Lambda Red Recombination because the gel still shows the native gene size of 1616bp not the LacI Homology arms with the spectinomycin template which would be closer to 1990bp, but the gel reflects the size of just above 1500bp.
If the recombination worked an expect band size near the 2000bp mark on the ladder. Since the band size are the native gene size it is safe to assume that the cells didn't accept the linear gBlock for the recombination event.
The positive control was pREDKI which has a size of 1050 bp using the Exo Forward and Exo reverse primer sequences.
Problems Faced
Linear PCR fragments with the antibiotic template was frozen at -20C which could have caused issues because PCR products have nicks in the backbone.
The antibiotic concentration for selection of the genomic recombination was not clear and so gradient of antibiotic was used and this lead to some false positive colonies on these plates.
Concentration of arabinose for the pREDKI plasmid was not at the appropriate concentration for the most efficient expression of the lambda red recombination proteins.
Future Plans
Optimize the pREDKI recombination protocol for higher efficiency.
Knock out nonessential genes to metabolically engineer E. coli cells.
Potentially knock in new genes to build a better metabolic pathway using other more efficient enzymes for putrescine production.
Using pSIM plasmid as the biofunctional helper plasmid for the recombination
Bio Bricks
Amplification of Truncated ArgI sequence using PFU PCR protocol
The ArgI gene was amplify by PFU PCR using M13 primers.
The size of the Truncated ArgI template is 995 bp
Positive Control
The LacI gene was amplify by PFU PCR using T7 primers.
The size of the Truncated ArgI template is 835 bp
Positive Control
Digestion/Ligation of LacI and ArgI into pSB1C3
The truncated LacI and ArgI PCR product were digested with EcoRI-PstI and ligated with T4 ligase into pSB1C3.  The ligation product was plated on 3.7 ug/ml chloramphenicol plates. Colonies were confirmed by direct colony PCR.
Lubbock_TTU · Texas Tech University
