<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Results_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Results/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Results/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Results/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Results/Modeling&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Results/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Results/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="wrapper-outer"><DIV class="wrapper"><DIV class="header-banner"><DIV class="header-right"><A href="https://2014.igem.org/Team:Freiburg/Project"><P class="project-title">The Ac<SPAN>CELL</SPAN>erator</P></A><NAV><UL><LI><A href="/Team:Freiburg">Home</A></LI><LI><A href="/Team:Freiburg/Project/Overview">Project</A></LI><LI><A href="/Team:Freiburg/Project/Overview">Overview</A></LI><LI><A href="/Team:Freiburg/Project/The_light_system">The Light System</A></LI><LI><A href="/Team:Freiburg/Project/Receptor">The Receptor</A></LI><LI><A href="/Team:Freiburg/Project/The_viral_vector">The Viral Vector</A></LI><LI><A href="/Team:Freiburg/Project/Vision">Mammalian Techniques for iGEM</A></LI><LI><A href="/Team:Freiburg/Project/Outlook">Outlook</A></LI></UL><LI><A href="/Team:Freiburg/Results">Results</A></LI><LI><A href="/Team:Freiburg/Results/Summary">Summary</A></LI><LI><A href="/Team:Freiburg/Results/Vector">The Viral Vector</A></LI><LI><A href="/Team:Freiburg/Results/Receptor">The Receptor</A></LI><LI><A href="/Team:Freiburg/Results/Light_system">The Light System</A></LI><LI><A href="/Team:Freiburg/Results/The combination">The Combination</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick">BioBricks</A></LI><LI><A href="/Team:Freiburg/Results/Modeling">Modeling</A></LI><LI><A href="/Team:Freiburg/Results/Light Boxes">Light Boxes</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices ">Policy and Safety</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices">Link-It</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/Saturdayballoonfever">Saturday Balloon Fever</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/survey">Online Survey</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/artificialcell">Ethical Essay</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/Interview">Interview with Dr. Boldt</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/Safety/Symposium">Symposium of Biosafety</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/Safety">Safety Introduction</A></LI><LI><A href="/Team:Freiburg/HumanPracticeAndSafety/Safety/viralvector">Viral Vector Safety</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/Safety/SafetySheet">Safety-Sheet</A></LI><LI><A href="/Team:Freiburg/PolicyAndPractices/Safety/BiosafetyLvl2">BSL 2 Regulation</A></LI><LI><A href="/Team:Freiburg/Notebook">Notebook</A></LI><LI><A href="/Team:Freiburg/Notebook/Labjournal">Labjournal</A></LI><LI><A href="/Team:Freiburg/Notebook/Cloning">Cloning</A></LI><LI><A href="/Team:Freiburg/Notebook/Methods">Methods</A></LI><LI><A href="/Team:Freiburg/Team/Members">Team</A></LI><LI><A href="/Team:Freiburg/Team/Members">Members</A></LI><LI><A href="/Team:Freiburg/Team/Collaboration">Collaborations</A></LI><LI><A href="/Team:Freiburg/Team/Judging">Judging</A></LI><LI><A href="/Team:Freiburg/Team/FUN-Part">FUN-Part</A></LI><LI><A href="/Team:Freiburg/Attributions">Attributions</A></LI><LI><A href="/Team:Freiburg/Attributions/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Freiburg/Attributions/Sponsors">Sponsors</A></LI></NAV></DIV></DIV></DIV></DIV><DIV class="wrapper-outer"><DIV class="wrapper"><DIV class="content-wrapper inner-page"><DIV class="content"><P><TITLE>The AcCELLerator</TITLE></P><DIV class="row category-row"><DIV class="col-sm-6"><DIV class="container-fluid" style="float: left"><DIV style="position: relative; float: right; margin-top: 4px;"><A href="https://2014.igem.org/Team:Freiburg/Results/BioBrick">Go back to The Biobricks</A></DIV></DIV></DIV><DIV class="col-sm-6"><DIV class="container-fluid" style="float: right"><DIV style="position: relative; float: left; margin-top: 4px;"><A href="https://2014.igem.org/Team:Freiburg/Results/Light_Boxes">See our self-made Light Boxes</A></DIV></DIV></DIV></DIV><H1><BIG>Modeling</BIG></H1><SECTION id="Results-Modeling-Introduction"><H1>Introduction</H1></SECTION><SECTION><P>The AcCELLerator has two major components: the light-regulated gene expression and the retroviral gene delivery. These two system are interconnected by the mCAT1, which acts as the receptor for MLV vectors and whose expression is regulated by the light system. The goal of modeling is describing the systems using mathematical language and trying to predict their behavior under any conditions. Since light-regulated gene expression and the retroviral gene delivery have been well studied and characterized, findings from previous research was adopted and modified to fit our experiments.</P><P>Our modeling was based on the combination of different differential equations. All functions were solved either analytically or numerically using MATLAB.</P></SECTION><SECTION id="Results-Modeling-MLV-Infection"><H1>MLV Infection</H1></SECTION><SECTION><H2 id="Results-Modeling-Model-MLV-Infection-Formulation">Model Formulation</H2><P>In our project, gene delivery was achieved by infecting the cells with the recombinant murine leukemia virus (MLV). As a typical retrovirus, its life cycle has been well characterized. Usually, this process can be divided artificially into several steps, including adsorption, internalization, integration, replication, assembly and release. However, our recombinant MLVs lacked the genes which are essentially for the replication and the virus assembly. Thus only the gene of interest (GOI), which is packed inside the viral RNA, could be integrated into the genome and expressed. The process from adsorption to integration can be again subdivided into seven steps (see Fig. 1), so that each step can be described with a simple mathematical model.</P><FIGURE><FIGCAPTION><P class="header">Figure 1. Schematic diagram of MLV infection </P><P class="desc">(1) The infection is initiated by the adsorption of the virions onto the cell surface, which is covered by the specific receptor mCAT1. (2) Once the virion binds with the receptor, fusion of viral and cell membrane is triggered and the viral core is internalized into the cytoplasm. (3) Using viral RNA as template, double stranded DNA is produced by reverse transcriptase. (4) The viral DNA is then transported along the microtubuli to the microtubule organizing center near the nucleus. (5) During the transcription and transport, the viral molecules could be degraded by cellular factors. (6) During cell division, the nuclear envelope dissolves and the viral DNA is imported into the nucleus. (7) Viral DNA is integrated into the chromosome by integrase. (modified from [1])</P></FIGCAPTION></FIGURE><P>The whole infection process was described by a set of differential equations, which was adopted from previous works [1,2] with  modifications.</P><P>First, a viral suspension of depth h was added to a layer of adherent cells to infect them (see Fig. 2). The concentration of the virion (V<SUB>m</SUB>) is a function of both time and depth. It is influenced by four different factors: diffusion, sedimentation, degradation and binding.</P><FIGURE><FIGCAPTION><P class="header">Figure 2. Schematic diagram of the transduction procedure. </P><P class="desc">The virions (shown in yellow) build a gradient on the z-axis. Flux of virions is determined by four factors: diffusion along the concentration gradient, sedimentation due to gravity, degradation and binding to the receptor (shown in violett) on the adherent cells (shown in green).</P></FIGCAPTION></FIGURE><P>The diffusion of the virions follows the fick’s law of diffusion. <EM class="highlight-kursiv">u</EM> is the velocity of sedimentation due to gravity. Since the virions are instable at 37°C, they decay in the medium with a constant decay rate <EM class="highlight-kursiv">k<SUB>d_vm</SUB></EM>. Thus this additional term was added to the partial differential equation (PDE).</P><P>The diffusion coefficient <EM class="highlight-kursiv">D<SUB>v</SUB></EM> can be calculated according to the Stokes–Einstein equation, </P><P>and <EM class="highlight-kursiv">u</EM> was estimated as the Stokes settling velocity.</P><P>To solve the PDE, boundary conditions are required at the top and the bottom of the medium. At the top (<EM class="highlight-kursiv">z=h</EM>), net flux of virions has to be zero.</P><P>At the bottom (<EM class="highlight-kursiv">z=0</EM>), virions bind irreversibly onto the cell surface and are subsequently internalized. The binding rate is proportional to local virus concentration, cell density and density of mCAT1 receptor on the cell surface. Moreover, the binding is saturating at high receptor density, so a Michaelis-constant for receptor density was added to the equation. </P><P>Second, the MLV undergoes several steps (see Fig. 1 (3)-(7)) after the internalization to be integrated into the genome. However, it is difficult to determine the intracellular concentration of each viral component. The only output of our experiments was the transduction efficiency determined by flow cytometry. Thus it makes more sense to consider the dynamics between infected and non-infected cells.</P><P>According to the location of intracellular viral particles, the cells were divided into three populations (see Fig. 3): virus-free cells (<EM class="highlight-kursiv">C<SUB>f</SUB></EM>), cells that carry MLV in the cytoplasm (<EM class="highlight-kursiv">C<SUB>c</SUB></EM>) and cells with viral DNA integrated in the genome (<EM class="highlight-kursiv">C<SUB>i</SUB></EM>). Similarly, intracellular viral particles were divided into cytoplasmic viruses (<EM class="highlight-kursiv">V<SUB>c</SUB></EM>) and integrated viruses (<EM class="highlight-kursiv">V<SUB>i</SUB></EM>). Particularly, a subpopulation of cytoplasmic virus inside the virus-carrying cells only (<EM class="highlight-kursiv">V<SUB>f</SUB></EM>) was also considered for further calculations. </P><FIGURE><FIGCAPTION><P class="header">Figure 3. Cell and virus populations and their density.</P>
			Cell populations (C) are shown in green and viruses (V) in yellow. The variables are defined as the density of each population (in 1/cm²) at the bottom of the petri dish.</FIGCAPTION></FIGURE><P>The dynamics between different cell and virus populations was influenced by several factors (see Fig. 4), including binding of new virions, intracellular degradation of viral particles with rate <EM class="highlight-kursiv">k<SUB>d_vc</SUB></EM>, integration of viral DNA with integration rate <EM class="highlight-kursiv">k<SUB>in</SUB></EM>, and cell division with a constant growth rate <EM class="highlight-kursiv">μ</EM>. </P><FIGURE><FIGCAPTION><P class="header">Figure 4. Dynamics of cell and virus populations.</P><P class="desc">Arrows towards and away from the rectangles describe the increase/decrease of the population with a specific rate indicated on the arrows, respectively.</P></FIGCAPTION></FIGURE><P>Thus the differential equations were set up as follows. </P><P><EM class="highlight-kursiv">δ<SUB>1</SUB></EM> is the possibility of a virus-carrying cell that becomes virus-free owing to the degradation of intracellular virus. Since multiple cytoplasmic viruses are unlikely to degrade at the same time, the possibility equals the fraction of virus-carrying cells with only one virus. Since the distribution of the multiplicities of infection (MOI) follows the poisson distribution[3], <EM class="highlight-kursiv">δ<SUB>1</SUB></EM> can be calculated using the following equation, </P><P>where the distribution is slightly modified to consider the virus-carrying cells only and <EM class="highlight-kursiv">n<SUB>v</SUB></EM> is the average number of virus per cell inside these cells.</P><P><EM class="highlight-kursiv">δ<SUB>2</SUB></EM> is the possibility of a virus-carrying cell that becomes virus-free during cell division. Because virus particles enter two daughter cells in a random process, one of them may lose all particles and became virus-free again. This possibility derived from the binomial distribution for a certain virus number. <EM class="highlight-kursiv">δ<SUB>2</SUB></EM> is the then the sum of products of poisson and binomial possibility for each MOI. </P><P>Besides, terms for the virus integration are calculated at time <EM class="highlight-kursiv">τ</EM> prior to the current time point <EM class="highlight-kursiv">t</EM>. <EM class="highlight-kursiv">τ</EM> is the mean trafficking time of the virus inside the cytoplasm and includes time for reverse transcription and viral transport (see Fig. 1 (3)-(4)). These terms turn these differential equations into a delay differential equation system (DDE), which was solved here by numerical methods. </P><P>The experimental output, the transduction efficiency, is defined as the fraction of virus-integrated cells. </P></SECTION><SECTION><H2 id="Results-Modeling-Data-MLV-Infection-Analysis">Data Analysis</H2><H3>Extracellular virus degradation</H3><P>The extracellular degradation rate of MLV (<EM class="highlight-kursiv">k<SUB>d_vm</SUB></EM>) was first investigated. MLV suspension was incubated for different durations. The viral concentration should follow a single exponential decay function. </P><P>In our experiment, the virus suspension was transferred to the cells after the incubation and transduction efficiency was measured. The results were fitted to an exponential function to determine the degradation rate (see Fig. 5). </P><FIGURE><FIGCAPTION><P class="header">Figure 5.  Extracellular degradation of MLV. </P><P class="desc">MLV vectors were incubated in cell-free medium at 37 °C for different durations and then infected NIH-3T3 cells for 72h. Transduction efficiency was measured by FACS. A single exponential function was fitted to the data (shown in line).</P></FIGCAPTION></FIGURE><P>The extracellular degradation rate was about 0.124 h<SUP>-1</SUP> and the half-life of MLV equals 5.6 h. This result is consistent with that obtained by Tayi et al. and shows the instability of the retroviral vector at physiological temperature in extracellular conditions. </P><H3>Estimation of viral titer</H3><P>Due to limited time, we did not carry out all experiments to determine every parameter. Thus the remaining parameters including binding, intracellular degradation, integration rate and trafficking time were adopted from Tayi’s paper. By means of this model with the parameter, we estimated the viral titer of our MLV suspension. </P><P>In this case, mouse cells were infected by MLV for different durations and then incubated in virus-free medium for a certain time. The equations with an unknown virus concentration were solved and fitted to the experimental data using least square estimation (see Fig. 6). </P><FIGURE><FIGCAPTION><P class="header">Figure 6.  Estimation of virus titer. </P><P class="desc">Mouse cells were incubated with MLV for different durations and then washed with medium. After 72 h of growth, cells were analyzed by FACS. The mathematical model with an unknown viral titer was fitted to the data by least square estimation (shown in line).</P></FIGCAPTION></FIGURE><P>The estimated virus titer is 1.2∙10<SUP>7</SUP> vectors/ml. This value is typical for standard protocols with helper-free retroviral production.[4]</P><H3>Verification of virus titer</H3><P>The estimated virus titer was verified by predicting the transduction efficiency with virus dilution and comparing it with the experimental data. MLV stock was diluted with a factor 0.1 to 0.5 and incubated with mouse cells. Transduction efficiency measured at the end of the experiments was shown in Fig. 7 as well as the prediction from our model. </P><FIGURE><FIGCAPTION><P class="header">Figure 7.  Verification of virus titer. </P><P class="desc">MLV suspension was diluted by different factors and added to the NIH-3T3 cells and incubated for 48 h. The viral titre estimated in the last experiments was used to predict the transduction efficiency (shown in line). </P></FIGCAPTION></FIGURE><P>The prediction was consistent with the data. The result demonstrates that our model is a good simulation of the infection process and can be applied for further experiments. </P></SECTION><SECTION id="Results-Modeling-Light-System"><H1>Light System</H1></SECTION><SECTION><H2 id="Results-Modeling-Light-System-Model-Formulation">Model Formulation</H2><P>The light system regulates the expression of the mCAT1, enabling the spatio-temporal control of virus-targeting. Modeling of the light system is based on ordinary differential equations (ODE). Fig. 8 shows the three major components of the system that are essential for the model: the activator ePDZ-VP16, the mRNA of mCAT1 and the mCAT1 protein on the plasma membrane (R: receptor).</P><FIGURE><FIGCAPTION><P class="header">Figure 8. The blue light system </P><P class="desc">The dashed lines indicate regulation by another component and the solid lines indicate changes with a certain rate. l is defined as the light condition and equals 1/0 when light is on/off, respectively.</P></FIGCAPTION></FIGURE><P>First of all, genes for the light system are introduced into the cells by transfection. The plasmid is not able to self-replicate inside mammalian cells, so the average plasmid concentration decreases during cell division. It is necessary to consider this dilution factor, the gene dosis (GD), which reduces exponentially with the growth rate of the cells. </P><P>The key element in the blue light system is the ePDZ-VP16 fusion protein, which interacts with the light-sensitive LOV2 protein and activates the transcription. After the transfection, expression and degradation of ePDZ-VP16 reaches an equilibrium. The concentration of ePDZ-VP16 and thus the rate of activation is proportional to the plasmid concentration. In addition, the mCAT1 gene has a basic transcription rate which is also proportional to the plasmid concentration. Rate of transcription is their sum if the light is on, i.e. if the light condition <EM class="highlight-kursiv">l=1</EM>. Thus the transcription of mCAT1 RNA can be described as follows,</P><P>The mRNA is then translated into mCAT1 protein.</P></SECTION><SECTION><H2 id="Results-Modeling-Light-System-Data-Analysis">Data Analysis</H2><P>The ODE of the blue light system were solved analytically under different conditions with arbitrary parameter values to simulate the properties of light-regulation. mRNA concentration and receptor density of mCAT1 were calculated with the transfected cells kept in dark or illuminated for different durations (see Fig. 9 and 10).</P><FIGURE><FIGCAPTION><P class="header">Figure 9.  Simulation of mRNA concentration at different conditions. </P><P class="desc">Arbitrary parameter values were used for simulation. Transfected cells were illuminated with blue light for 0 (dark), 1, 2.5 or 5 h and incubated 24 h. </P></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><P class="header">Figure 10.  Simulation of receptor density at different conditions. </P><P class="desc">Arbitrary parameter values were used for simulation. Transfected cells were illuminated with blue light for 0 (dark), 1, 2.5 or 5 h and incubated 24 h. </P></FIGCAPTION></FIGURE><P>For comparison, the blue light system with the SEAP gene was transfected into HEK-293T cells and the production of SEAP was assayed at different illumination conditions (see Fig. 11). </P><FIGURE><FIGCAPTION><P class="header">Figure 11.  SEAP production after different times of blue light illumination. </P><P class="desc">Cells were transfected with the blue light system including the SEAP reporter, illuminated with blue light for 0 (dark), 1, 2.5 or 5 h and incubated for 24 h. SEAP concentration was measured. </P></FIGCAPTION></FIGURE><P>The data was insufficient for model fitting and determination of the parameters. However, it indicates a similar trend with the simulation: longer illumination time leads to a larger amount of GOI expression. But longer illumination with the energy-rich blue light may be harmful for the cells. The toxicity of light should be considered in the model for future improvements. Further experiments are required to determine the parameters and the optimal illumination time.</P></SECTION><SECTION><H2 id="Results-Modeling-Ref">References</H2><OL class="two-columns small"><LI>Tayi, V. S., Bowen, B. D., &amp; Piret, J. M. (2010). Mathematical model of the rate‐limiting steps for retrovirus‐mediated gene transfer into mammalian cells. Biotechnology and bioengineering, 105(1), 195-209.</LI><LI>Kwon, Y. J., Hung, G., Anderson, W. F., Peng, C. A., &amp; Yu, H. (2003). Determination of infectious retrovirus concentration from colony-forming assay with quantitative analysis. Journal of virology, 77(10), 5712-5720.</LI><LI>Fields, B. N., Knipe, D. M., &amp; Howley, P. M. (2007). Fields virology. 5th.</LI><LI>Pear W S, Nolan G P, Scott M L, et al. Production of high-titer helper-free retroviruses by transient transfection[J]. Proceedings of the National Academy of Sciences, 1993, 90(18): 8392-8396.</LI></OL></SECTION><DIV class="row category-row"><DIV class="col-sm-6"><DIV class="container-fluid" style="float: left"><DIV style="position: relative; float: right; margin-top: 4px;"><A href="https://2014.igem.org/Team:Freiburg/Results/BioBrick">Go back to The Biobricks</A></DIV></DIV></DIV><DIV class="col-sm-6"><DIV class="container-fluid" style="float: right"><DIV style="position: relative; float: left; margin-top: 4px;"><A href="https://2014.igem.org/Team:Freiburg/Results/Light_Boxes">See our self-made Light Boxes</A></DIV></DIV></DIV></DIV></DIV><ASIDE><DIV id="content-nav"><UL class="nav"><LI><A href="/Team:Freiburg/Results/Summary">Summary</A></LI><LI><A href="/Team:Freiburg/Results/Vector">The Viral Vector</A></LI><LI><A href="/Team:Freiburg/Results/Vector#Vector-Introduction">The Viral Vector</A></LI><LI><A href="/Team:Freiburg/Results/Vector#Vector-Stable-Integration">Stable Integration</A></LI><LI><A href="/Team:Freiburg/Results/Vector#Vector-High-Efficiency">High Efficiency</A></LI><LI><A href="/Team:Freiburg/Results/Vector#Vector-Specificity-and-Safety">Specificity and Safety of MuLV</A></LI></UL><LI><A href="/Team:Freiburg/Results/Receptor">The Receptor</A></LI><LI><A href="/Team:Freiburg/Results/Receptor#Receptor">The Receptor</A></LI><LI><A href="/Team:Freiburg/Results/Receptor#Results-Receptor-LocalisationReceptor">Localisation</A></LI><LI><A href="/Team:Freiburg/Results/Receptor#Results-Receptor-ReceptorFunctionality">Functionality</A></LI><LI><A href="/Team:Freiburg/Results/Receptor#Results-Receptor-ReceptorExpression">Results</A></LI><LI><A href="/Team:Freiburg/Results/Light_system">The Light System</A></LI><LI><A href="/Team:Freiburg/Results/Light_system#Light-system">The Light System</A></LI><LI><A href="/Team:Freiburg/Results/Light_system#Results-LightSystem-Function">Function</A></LI><LI><A href="/Team:Freiburg/Results/Light_system#Results-LightSystem-SEAP-as-Reporter">SEAP as Reporter</A></LI><LI><A href="/Team:Freiburg/Results/The_combination">The Combination</A></LI><LI><A href="/Team:Freiburg/Results/The_combination#Combination">The Combination</A></LI><LI><A href="/Team:Freiburg/Results/The_combination#Light-induced-receptor">Light Induced Receptor</A></LI><LI><A href="/Team:Freiburg/Results/The_combination#Results-LightSystem-MuLVSEAP">Pattern Generation</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick">BioBricks</A></LI><LI class="listcat"><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks">BioBricks</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-mCAT-1">mCAT-1</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-PAC">PAC</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-Gal4BD">Gal4BD</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-WPRE">WPRE</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-SEAP">SEAP</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-ePDZ">ePDZ</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-pMIG">pMIG</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks-Cas9-Nickase">Cas9-Nickase</A></LI><LI class="listcat"><A href="/Team:Freiburg/Results/BioBrick#Plasmid-BioBricks">Plasmids</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-mCAT-1">mCAT-1</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-Light-Systems">Light System</A></LI><LI><A href="/Team:Freiburg/Results/BioBrick#Plasmid-Viral-Vector">Viral Vector</A></LI><LI><A href="/Team:Freiburg/Results/Modeling">Modeling</A></LI><LI><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-Introduction">Introduction</A></LI><LI class="listcat"><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-MLV-Infection">MLV-Infection</A></LI><LI><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-Model-MLV-Infection-Formulation">Model Formulation</A></LI><LI><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-Data-MLV-Infection-Analysis">Data Analysis</A></LI><LI class="listcat"><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-Light-System">Light System</A></LI><LI><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-Light-System-Model-Formulation">Model Formulation</A></LI><LI><A href="/Team:Freiburg/Results/Modeling#Results-Modeling-Light-System-Data-Analysis">Data Analysis</A></LI><LI><A href="/Team:Freiburg/Results/Modeling#Results%20Modeling%20Ref">References</A></LI><LI><A href="/Team:Freiburg/Results/Light%20Boxes#Results%20Lightboxes">Light Boxes</A></LI></DIV></ASIDE></DIV><FOOTER><UL><LI><A href="https://2014.igem.org/Team:Freiburg/Team/Members">iGEM Team Freiburg 2014</A></LI></UL></FOOTER></DIV></DIV><BUTTON class="invertbutton" onclick="invertColors();">Inv.</BUTTON><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Freiburg/Results/Modeling">http://2014.igem.org/Team:Freiburg/Results/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Results/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Results/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Results/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Results/Modeling&amp;oldid=372562" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>