<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Content_Notebook_Methods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Content/Notebook/Methods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Content/Notebook/Methods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Methods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Methods&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Content/Notebook/Methods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Content/Notebook/Methods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>The AcCELLerator</TITLE></P><H1>Methods</H1><SECTION><H2 id="Notebook-Methods-Cloning">Cloning</H2></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Gel-Elektrophoresis">Agarose Gel Electrophoresis</H3><P>
Towards PCR and enzymatic digest analyses, agarose gels were prepaired with 0,5x TAE and Agarose concentrations between 0.7 % and 0.9 % (w/v). At 65 °C, 1 µl of 10x GelRed was added to the Gel before an one hours polymerisation step. As a marker, 1 µl of GeneRuler 1 kb DNA ladder was loaded. Gel runs were commonly performed with voltage ranges between 80 V and 130 V for 45 minutes.</P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Plasmid-Isolation">Plasmid Isolation</H3><P>
To purify plasmid DNA from agar plates, either Roti-Prep Plasmid Mini Kit from Carl Roth or High Pure Plasmid Isolation Kit from Roche were used for Minipreps. For high yields of supercoiled, endotoxin-less DNA, Genomed‘s Jetstar Plasmid Purification MIDI Kit or Genopure plasmid MIDI Kit from Roche were used with 150 mL of transformed E. coli o/n culture. No deviances from the manufacturers‘ guidelines were incorporated by performing the isolation steps. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Gel-Extraction">Gel Extraction</H3><P>
When extracting PCR bands or digested fragments from an agarose gel, the QIAquick Gel extraction Kit from Qiagen was used. Steps were performed in accord with provided user‘s manuals, including three deviances: centrifugation steps were always performed with or with less than 17,900 g, columns were rotated by 180° after washing and last, DNA-binding columns were heated to 50 °C for two minutes before the final elution step. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Gibson-Assembly">Gibson Assembly</H3><P>To purify plasmid DNA we used several kits that were provided by our sponsors. For Minipreps we used Peqlab, Roche, qiagen, bioRon, Genetics, Roth. For Midipreps we used Qiagen, Jetstar, Peqlab. Beforehand o/n cultures of transformed <EM class="highlight-kursiv">E.coli</EM> were prepared. No deviances from the manufacturer's guidelines were incorporated by performing the isolation steps. 
</P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Ligation">Ligation</H3><P>
In order to avoid low ligation efficiencies or undesired byproduct vectors, fragment molarities and lengths were considererd as parameters for calculational schemes. Smaller fragments were mostly added excessively within between 4-fold and 8-fold molar amounts, compared to larger backbone fragments. Volume constraint often arose low Gel Extraction yields of particular fragments, so that total DNA quantities were fixed to values at around 25 ng. 5‘ dephosphorylation of single fragments was scarcely performed for reactions with high proportions of backbone religation, using Antarctic Phosphatase. Typical ligation approaches contained 2 µl 10x T4 Buffer, 1-7 µl of each fragment, 1 µl of T4 Ligase and a water fill-up to 20 µl. A 30-minute incubation step was performed at RT, preceding subsequent transformation. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Oligo-Annealing">Oligo Annealing</H3><P>
5 µl of 100 µM both Forward and Reverse primer solutions were diluted in 80 µl of water. 10 µl of NEB Buffer No. 2 were added. Samples were heated up to 95 °C for two minutes, whereupon the heating block was switched off. After a 2-hour step of gradual cooling down to RT, the annealed oligos were stored at -20 °C. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Plasmid-Isolation2">Plasmid Isolation</H3><P>First step of Gibson Assemblies was the rigidly calculation of molar amounts of the fragments. We afterwards joined the fragments and filled them up to 5 µL. These 5 µL were added to 15 µL ice chilled Gibson-Mastermix (Gibson et al., 2009), containing 5x ISO-Buffer, Taq-Ligase and T5 Exonuclease. Except Phusion Polymerase (Gibson et al., 2009) we used the same amount of Q5 Polymerase for the Mastermix. The samples were immediately heated to 50°C for one hour. Afterwards, the samples were cooled for 3 minutes at room temperature and 3 minutes on ice, before they were transformed into competent <EM class="highlight-kursiv">E.coli</EM>.
</P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-PCR">Polymerase Chain Reaction (PCR)</H3><P>
In order to amplify different DNA-templates, preferably from plasmids, different PCR approaches were used - with the following component amounts for 50 µl of a total volume. 31.5 µl of water, 10 µl of 5x Q5 Reaction Buffer, 4 µl of 2.5 mM dNTP solution, 1 µl of DNA template (200 ng), 1 µl of 10 µM Forward and Reverse Primer, 1 µl of DMSO and 0.5 µl of Q5 High-Fidelity DNA Polymerase. Apart from Touch-down variants and annealing temperature gradient analyses, thermocycler programs consisted of the guideline annotated below. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Assembly-PCR">Assembly PCR</H3><P>
in case of Gibson Assemblies subsequent to fragment amplification, a reduction of fragment quantities to less than five was soon considered appropriate for more efficient Cloning. Assembly PCRs were established for halving the amounts, by using a consecutive double PCR strategy. Expect for Forward and Reverse Primers, all components of a Standard PCR were joined as mentioned above. A first PCR was done with an annealing temperature derived from the sequences of overlapping template regions. Elongation temperature was estimated by considering the fragment length of the larger DNA template (1 kBP per 30 seconds). Five cycles were performed. 2.5 µl of 10 µM Forward and Reverse Primers, both binding to the new 5‘ of both strands were subsequently added to the first PCR mix. A second PCR was performed, with an elongation temperature calculated on the basis of the assembled fragment‘s size. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Colony-PCR">Colony PCR</H3><P>
In order to screen for positive bacterial clones after transformation experiments, Colony PCRs with available primers were done as follows. One single plate colony was picked and dissolved in 17.8 µl of water in a PCR tube, subsequently 2.5 µl of 10x Standard Taq  Reaction Buffer, 1 µl of 10 µM Forward and Reverse Primer, 1 µl 2.5 mM dNTP solution and 0.125 µl Taq Polymerase were added from a previously prepared Mastermix. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Preparative-Enzyme-Digest">Preparative Enzymatic Digest</H3><P>
Buffers for two or more combinatoral enzymes were selected by NEB Double Digest Finder. In order to gain high fragment concentrations in subsequent Gel Extractions, a total volume of 50 µl was mostly chosen. Typical Preparative Digests constisted of 2-3 µg vector DNA, 5 µl 10x NEB Buffer, 0.5 µl of 100x BSA, 1 µl per enzyme and were filled up to 50µl with water. Digests were commonly performed at 37 °C for two hours. </P></SECTION><SECTION><H3 id="Notebook-Methods-Cloning-Transformation">Transformation</H3><P>
3-4 µl of assembled plasmids, derived from either Gibson Assemblies or classical cloning steps, were added to a 25 µl aliquot of chemically competent Top10 <I>E. coli</I> cells on ice. After an incubation period of 30 minutes, a 42 °C heat shock was performed for exactly 45 seconds. Subsequently, a second incubational step on ice was conducted for 2 minutes. 500 µl of previously autoclaved LB medium were added to each aliquot of transformation mix. Samples were incubated at 37 °C and 300 rpm for one hour. Transformations were finalized with plating of 500 µl of transformed <I>E. coli</I> solution on an Ampicillin or Chloramphenicol containing agar plate. Subsequently, plates were stored at 37 °C over night.</P></SECTION><SECTION id="Notebook-Methods-Cell-Culture"><H2>Cell Culture</H2></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Conditions">Growing Conditions</H3><P>Cells were cultured in DMEM complemented with 10% FCS, glutamin (2 mM), 100 units/ml Penicillin and 100 ug/ml Streptomycin. Cells were grown in humid conditions at 37°C and 7.5% CO<SUB>2</SUB>.</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Splitting">Cell Splitting</H3><P>DMEM, DPBS and Trypsin solutions were prewarmed to 37 °C. Old medium was aspirated and cells were washed with 5 ml of DPBS, which was subsequently aspirated as well. After an addition of 1 ml Trypsin, an incubation step at 37 °C was performed - until all cells showed signs of detachment. 10 ml of fresh DMEM were then added to recollect trypsinated cells from the dish. Cells were further diluted in DMEM and seeded into new 10 cm, 6-well or 24-well dishes and cells stored at 37 °C. Confluency was usually checked the day after.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Seeding">Cell Seeding</H3><P>For a 24-well plate, 0.5 ml cell suspension was commonly utilized per well. Accordingly, 2 ml were added to 6-wells and 10 ml per 10 cm petridish. The amount of cells depended on the respective experiment, but usually ranged from 65,000 in 24 wells to 300,000 in 6-wells and 1,800,000 per 10 cm dishes.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Transfection">PEI-Transfection</H3><P><B>10 cm dish</B>24 µl of PEI-solution were mixed with ~ 600 µl of Opti-MEM. 8 µg of the DNA of interest was added to the solution, then the tube was immediately vortex thoroughly and had to incubate at room temperature for exactly 15 min. Finally, the solution was gently resuspended and drop-wisely spread to the cells in the dish. This was in turn swung in an 8-form.<B>6-well plates</B>
		7.5 µl of PEI-solution, 200 µl of Opti-MEM and 3 µg of interest DNA were used.<B>24-well plates</B>
		1.5 µl of PEI-solution, 50 µl of Opti-MEM and 0.5 µg of interest DNA were used.</P><P>As another possibility cells were transfected in suspension. Therefore cells were trypsinated as described before. 300,000 cells per ml were incubated with 100 ml transfection mix for 5 hours at 37°C in a Falkon tube. Cells were centrifuged at 900 RPM for 2 min afterwards and resuspended in prewarmed completed DMEM. Cells were seeded as described before.
</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Lysis">Cell Lysis</H3><P>Cells of 24-well plates were washed with 500 µl of cooled DPBS. Then, cells were lysed in 100 µl of chilled RIPA Buffer, followed by a 10-minute incubation on ice. Next, cells were dissolved with a cell scrapper or inverted pipet tip and the developing lysate transferred to a 1.5 ml tube. Samples were vortexed thoroughly, preceding a second 10-minute incubation on ice. A 5 minute centrifugation step was done at 10,000 g. 60 µl of the supernatant were mixed with 20 µl of 4x Laemmli protein dye. Samples were heated to 95 °C for 5 minutes and afterwards either frozen at -20 °C or directly used for subsequent Western Blotting.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Virus-Production">Production Of Viral Particles</H3><P>MuLV was produced in Phoenix cells in 100mm dishes. Cells were splitted several times before virus production for separation and transfected with PEI at a density of 70% with pMIG. After 24 hours the medium was removed and replaced with 4 ml fresh completed DMEM. 24 hours later the supernatant was collected and replaced by 4 ml fresh DMEM. This step was repeated once, i.e. having a final volume of 8 ml per dish. Viral supernatants were stored at 4°C.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Virus-Concentration">Concentration Of Viral Particles</H3><P>The viral supernatant was concentrated using chondroitin sulfate and polybrene. Chondroitin sulfate was added to a concentration of 80 µg/ml and incubated for 10 min at 37°C. Polybrene was added to the same concentration and the mixture was again incubated at 37°C for 10 min. The mixture was centrifuged for 15 min at 10,000 x g. The supernatant was carefully aspirated and fresh DMEM was added to a desired concentration.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Transduction">Viral Transduction</H3><P>For transduction medium was removed from the cells and replaced with half the working volume of fresh completed growth medium. Polybrene was added to 10 µg/ml (final volume). Viral supernatant was added with the same volume as growth medium. Viral supernatants were not filtered before. Medium was changed between 4 and 24 hours.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-SEAP-Assay">SEAP Assay</H3><P>200 µl of cell supernatant was transferred from a 24-well plate into one 96-well plate (round bottom), followed by a heat incubation at 65°C for 30 min. Centrifugation was done for 1 min at 1250xg. 100 µl of 2x SEAP buffer (20 mM homoarginine, 1 mM MgCl<SUB>2</SUB>, 21 %(v/v) diethanolamine, pH 9.8) were filled in the wells of a 96 well plate (flat bottom). Addition of 80 µl supernatant from the round bottom plate to each well with SEAP buffer. Addition of 20 µl pNPP (120 mM para-nitrophenyl phosphate in H<SUB>2</SUB>O) and bubbles carefully removed. Immediately after this, the 96-well plate was placed into a plate reader. Spectroscopic measurement was taken every minute for 120 times (2h) at a wavelength of 405 nm. 
	</P><P>For generation of patterns in 96W or 384W plates the SEAP assay was completely performed in the original plate without transfer into another 96W plate. All the other steps were performed as described before.
	</P></SECTION><SECTION><H3 id="Notebook-Methods-Cell-Culture-Fixation">Fixation Of Cells With PFA</H3><P>For fixation of cells on cover slips cells were washed with PBS and incubated with PFA (4%)for 10 min on ice. Cells were incubated for at least 10 min at room temperature afterwards. 8 µl mowiol prepared with DABCO was placed on an object slide. Cells on cover slips were stained for 10 s in 5 ml DAPI solution (1 µl in 5 ml ddH2O) and washed in ddH2O. After getting rid of the water droplets slides were placed with the cells facing down onto the mowiol droplet. Slides were dried for 30 min at 37°C and stored at  4°C in the darkness afterwards. 
		</P></SECTION><SECTION id="Notebook-Methods-Light-Experiments"><H2>Light Experiments</H2></SECTION><SECTION><H3 id="Notebook-Methods-Light-Experiments-Red-Light">Red-Light Experiments</H3><P>24 h post transfection, the medium was exchanged thereby adding 15µM phycocyanobilin (PCB). The cells were incubated for 1h in the CO2- incubator to allow the formation of holo-Phytochrome B. Subsequently the cells were illuminated under controlled conditions using light sources emitting distinct amounts of light: 20µE for 48h, 660nm or 740nm wavelength. The control-plate was kept in darkness.
		</P></SECTION><SECTION><H3 id="Notebook-Methods-Light-Experiments-Blue-Light">Blue-Light Experiments</H3><P>24 h post transfection plates were illuminated with blue light for 1-5 hours (465 nm). The control-plate was kept in darkness.
			</P></SECTION><SECTION id="Notebook-Methods-Western-Blot"><H2>Western Blotting</H2></SECTION><SECTION><H3 id="Notebook-Methods-Western-Blot-Blot-And-SDS-PAGE">Western Blot And SDS-PAGE</H3><P>
For efficient detection of proteins in our studies, especially dCas9 and it‘s targets, cells were transfected and after 48 h cell lysis followed. 

SDS-Gels (10 % SDS) were performed in Invitrogen-cassettes or Biorad-systems. Therefore, resolving- and stacking-gels were poured as listed in the according table.
The bags were loaded with 20 µl sample or 10 µl marker (cell lysates were boiled at 95 °C in 1x Loading Buffer for 5 minutes before). Runs were performed at 80 volt until the samples had reached the resoltution gel. Then, the voltage was increased to 120 volt. Gels run until the blue frontline had passed the gel completely.
Afterwards, blotting in a wet tank or semi-dry blotting followed. therefore PVDF-membranes were activated for 5 minutes in methanole. Then, they were passed to transfer-buffer just as the Whatman-Paper.  
wet-blotting:
The blotting sandwiches were prepared as follows, started from the bottom: 
<UL><LI> one fiber pad</LI><LI> 2 pieces of whatman-paper</LI><LI> SDS-gel</LI><LI> PVDF-membrane</LI><LI> one piece of whatman-paper</LI><LI> one fiber pad</LI></UL>

Then, the sandwich was inserted into the wet tank. Blotting was performed for 1.5 h at 4 °C. For one membrane 200 mA was set (two membranes: 400 mA). 
semi-dry-blotting:
For semi-dry blotting the apparatuses of AG Weber were used.
<UL><LI> one piece of whatman-paper</LI><LI> PVDF-membrane</LI><LI> SDS-gel</LI><LI> one piece of whatman-paper</LI></UL>
Blotting was performed for 1.5 h. For one membrane 350 mA was set (two membranes: 700 mA). 
After blotting, the membrane was blocked in 10 ml of 4 % milkpouder-TBST solution for 1 h. The solution was aspirated before primary antibody in 2 % milkpouder-TBST solution was added (10 ml) and incubated over night on a shaker at 4 °C. In general primary antibodies were: anti-HA (dilution: 1:2500), anti B-Actin (dilution: 1:2500), anti-GAPDH (dilution: 1:5000) 
The next day, three washing steps with TBST and dH20 followed, each 5 minutes (primary antibody solution was stored and used 4 times). Secondary antibody solution (also in 2 % milkpouder-TBST solution) was added for 1 h at RT and decanted afterwards. In general secondary antibodies were: anti-mouse-HRP (1:5000) or anti-rabbit-HRP (1:5000) 
Another 3 washing steps with TBST followed. Proteins were detected by respectively adding 0.5 mL ECL solution 1 and 2 (Promega) to the membrane.


<TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Resolution Gel (1x)	</TH><TH>	Stacking Gel (1x)	</TH></TR><TR><TD>	1.34 ml H20	</TD><TD>	1.9 ml H2O	</TD></TR><TR><TD>	1.2 ml 50% Succrose	</TD></TR><TR><TD>	2.82 ml Tris/HCl pH 8,8	</TD><TD>	3.2 ml Tris/HCl pH 6,8	</TD></TR><TR><TD>	1.89ml 38% Acrylamid	</TD><TD>	0.33 ml 38% Acrylamid	</TD></TR><TR><TD>	75µl 10% SDS	</TD><TD>	31.5 µl 10% SDS	</TD></TR><TR><TD>	2µl TEMED	</TD><TD>	1.7 µl TEMED	</TD></TR><TR><TD>	94µl APS	</TD><TD>	83 µl APS	</TD></TR></TBODY></TABLE><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Running Buffer (250 mM; 1,92 M Glycin; 1% SDS)	</TH></TR><TR><TD>	30 g Tris	</TD></TR><TR><TD>	144 g Glycin	</TD></TR><TR><TD>	100 ml 10% SDS	</TD></TR><TR><TD>	fill up to 1 L with dH20	</TD></TR></TBODY></TABLE><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	Transferpuffer (48 mM Tris; 39 mM Glycin; 20% Methanol; pH 9,2; 0,1% SDS)	</TH></TR><TR><TD>	5,82 g Tris	</TD></TR><TR><TD>	2, 93 g Glycin 	</TD></TR><TR><TD>	200 ml MeOH	</TD></TR><TR><TD>	10 ml 10% SDS 	</TD></TR><TR><TD>	fill up to 1 L with dH20	</TD></TR></TBODY></TABLE><TABLE id="tabelle"><TABLE border="3" frame="box"><TBODY><TR><TH>	TBST	</TH></TR><TR><TD>	100 ml 10x TBS	</TD></TR><TR><TD>	900 ml Milipore H2O	</TD></TR><TR><TD>	 2 ml Tween-20 (1:4 solution)	</TD></TR></TBODY></TABLE></TABLE></TABLE></TABLE></TABLE></P></SECTION><SECTION id="Notebook-Methods-Confocal"><H2>Confocal Microscopy</H2></SECTION><SECTION><P>For confocal microscopy, cells were seeded in 12/24-well plates on the top of 18/10-mm cover slips and incubated with standard protocols.</P><P>Prior to microscopy, cells on the cover slips were fixed for 7 min with 4% PFA/PBS, washed with PBS, and in some case counterstained for 5 min with 5 mg/ml DAPI/PBS. Then the cover slips were washed again with PBS, taken out from the plate with tweezers and dipped once in distilled water. About 15 ml mowiol mounting medium was dropped onto a clean microscope slide. The cover slip was dried carefully with a kimtech towel on its side and mounted slowly and upside down on the microscope slide. The slides were then kept in dark till the mounting medium was dry.</P><P>All microscope slides were imaged with All samples the Nikon Confocal C2 microscope with four excitation lasers: 405 nm, 488 nm, 561 nm and 640 nm. Images were captured at 60x magnification using the Nikon N Plan Apo l 60x/1.40 Oil objective.</P></SECTION><SECTION id="Notebook-Methods-Fluorescence"><H2>Fluorescence Microscopy</H2></SECTION><SECTION><P>Cells expressing a fluorescence marker were analyzed by fluorescence microscopy using 4x, 10x and 20x objectives on an Olympus IX71 inverted microscope. Filter cubes for green light (EGFP) and red light were used (mCherry, mKO, mKate). mKO fluorecence was changed to orange on a computer afterwards.</P></SECTION><SECTION id="Notebook-Methods-FACS"><H2>Flow cytometer</H2></SECTION><SECTION><P>For quantify cells that expressed mCherry or EGFP a Gallios Flow Cytometer was used. Cells were washed an resuspended in PBS containing FCS before measurement. In the cytometer each cell was analysed for expression of a distinct fluorophore by different lasers.</P></SECTION><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Freiburg/Content/Notebook/Methods">http://2014.igem.org/Team:Freiburg/Content/Notebook/Methods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Content/Notebook/Methods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Content/Notebook/Methods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Methods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Methods&amp;oldid=401502" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>