<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Freiburg_Content_Notebook_Cloning"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Freiburg/Content/Notebook/Cloning">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Freiburg/Content/Notebook/Cloning&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Cloning&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Cloning&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Freiburg/Content/Notebook/Cloning" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Freiburg/Content/Notebook/Cloning</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>The AcCELLerator</TITLE></P><H1>Cloning</H1><P>The protocols for our methods can be found <A href="https://2014.igem.org/Team:Freiburg/Notebook/Methods">here</A>.</P><P>During our experiments, we conducted a lot of PCRs with different Primers. In order to stay on top of things, we numbered Primers, PCRs and constructs. You can download both lists with detailed information about Primer composition and about PCRs in the following documents:</P><H2>p14rz_002</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR: o14rz_005, o14_rz006; Template pQXCIN-SLC7A1</LI><LI>Digest: pKM006, EcoRI + NotI; PCR-product</LI><LI>Testdigest with PstI in Cutsmart, expected bands: 5,9; 0,7</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_003</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 3</LI><LI>Digest 3, 4</LI><LI>Testdigest with PstI + SpeI in Cutsmart, expected bands: 4,1;2,1;1,8</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_004</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 21,22</LI><LI>Digest -</LI><LI>Testdigest with PstI in Cutsmart, expected bands: 4,1;3,2</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_005</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 4,5</LI><LI>Digest 5</LI><LI>Testdigest with AgeI+HindIII in Cutsmart, expected bands: 6,6;1,3</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_006</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 17,18</LI><LI>Digest 5</LI><LI>Testdigest with AgeI+HindIII in Cutsmart, expected bands: 6,6;1,4</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_007</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 37,38</LI><LI>Digest 21</LI><LI>Testdigest with PstI in Cutsmart, expected bands: 4,1;2,2;1,7</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_008</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR blue light receptor; Template: pQXCIN-SLC7A1</LI><LI>Digest pKM006 EcoRI, NotI</LI><LI>Testdigest with SmaI in Cutsmart, expected bands: 5,7</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_009</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 39,40</LI><LI>Digest 5</LI><LI>Testdigest with BamHI in Cutsmart, expected bands: 3,0;2,7</LI></UL></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14rz_010</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 23,24</LI><LI>Digest 12</LI><LI>Testdigest with PstI in Cutsmart, expected bands:3,1;2,3;0,56</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14ls_003</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 7,9,10</LI><LI>Digest -</LI><LI>Testdigest with KpnI in Cutsmart, expected bands: 4,9; 1,3; 0,4</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_001</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR Primers vv003,vv004; Template: pMIG</LI><LI>Digest pMIG EcoRI,NotI; C1-eGFP EcoRI,NotI</LI><LI>Testdigest with NgoMIV in Cutsmart, expected bands: 5,7; 0,6</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_002</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR -</LI><LI>Digest 8,11</LI><LI>Testdigest with PstI in Cutsmart, expected bands: 3,2;2,1;1,4</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_003</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR pMIG with primer vv003+vv004-</LI><LI>Digest xx</LI><LI>Testdigest with KpnI in Cutsmart, expected bands: 3,5;1,5</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_006</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 26</LI><LI>Digest 13</LI><LI>Testdigest with NgoMIV in Cutsmart, expected bands: 3,5;1,9;0,95,0,6</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_007</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 27</LI><LI>Digest 14</LI><LI>Testdigest with NgoMIV in Cutsmart, expected bands: 3,5; 1,3; 0,97; 0,66</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_008</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 43</LI><LI>Digest 22</LI><LI>Testdigest with EcoRV in Cutsmart, expected bands: 3,4; 1,4 0,75</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_009</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 44</LI><LI>Digest 22</LI><LI>Testdigest with EcoRV+MluI in Cutsmart, expected bands: 3,4; 1,43; 0,75</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_010</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 45</LI><LI>Digest 22</LI><LI>Testdigest with EcoRV in Cutsmart, expected bands: 3,4; 2,2</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_012</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 46</LI><LI>Digest 22</LI><LI>Testdigest with EcoRV in Cutsmart, expected bands: 4,7;3,4</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_013</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 47,48,49</LI><LI>Digest 22</LI><LI>Testdigest with NgoMIV in Cutsmart, expected bands: 3,4;2,0;1,3;0,2</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14vv_014</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 50,51</LI><LI>Digest 22</LI><LI>Testdigest with NgoMIV in Cutsmart, expected bands: 4,1; 1,56; 0,66</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14zl_010</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 1,2</LI><LI>Digest 1</LI><LI>Testdigest with NgOMIV in Cutsmart, expected bands: 4,7;2,4;1,1</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14zl_011</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 19,20</LI><LI>Digest 1</LI><LI>Testdigest with NgOMIV in Cutsmart, expected bands: 4,7;2,0;1,3;0,43</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14zl_016</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 52,53</LI><LI>Digest 22</LI><LI>Testdigest with HindIII in Cutsmart, expected bands: 5,7;2,3;1,7</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14zl_017</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 64,65</LI><LI>Digest 24</LI><LI>testdigest with NgoMIV in Cutsmart, expected bands: 5,4;2,9</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR_digest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><H2>p14zl_020</H2><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><UL><LI>PCR 59,60</LI><LI>Digest 24</LI><LI>testdigest with BamHI-HF, NgoMIVin Cutsmart, expected bands: 4,7; 2,4; 1,8</LI></UL></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">PCR</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION><DIV class="row category-row"><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Digest</P></FIGCAPTION></FIGURE></DIV><DIV class="col-sm-6"><FIGURE><FIGCAPTION><P class="header">Testdigest</P></FIGCAPTION></FIGURE></DIV></DIV></SECTION><SECTION id="Standardization"><H1>Standardization</H1><H2 id="Standardization-August">Standardization - August</H2><H4>PCR mix for site-directed mutagenisis (25 µl)</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD>template</TD><TD>1 µl</TD></TR><TR><TD>buffer (5x)</TD><TD>5 µl</TD></TR><TR><TD>Primer</TD><TD>1 µl each</TD></TR><TR><TD>polymerase</TD><TD>0.5 µL</TD></TR><TR><TD>dNTPs</TD><TD>1 µl</TD></TR><TR><TD>DMSO</TD><TD>0.5 µl</TD></TR><TR><TD>water</TD><TD>15 µl</TD></TR></TBODY></TABLE><P>start your PCR mix with just one primer and let reaction run for ten cycles including end elongation. shortly after that add 1 µl dNTP and 1 µl of primer 2. start second PCR under same conditions for 15 cycles. Don't spend too much time with adding dNTPs and primer 2. </P><P>DNA amplifications such for ligation PCR approaches contained 50 µl.</P><H4>protocoll site-directed mutagenisis</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD> 98 °C </TD><TD> 5 min </TD><TD> denaturation </TD></TR><TR><TD> 98 °C </TD><TD> 30 s </TD><TD> denaturation </TD></TR><TR><TD> 60 °C </TD><TD> 30 s </TD><TD> annealing </TD></TR><TR><TD> 72 °C </TD><TD> 30 s/kb </TD><TD> elongation </TD></TR><TR><TD> 72 °C </TD><TD> 10 min </TD><TD> end elongation </TD></TR></TBODY></TABLE><P>To avoid high background activities by not-mutated plasmids, the PCR mixture was digested by DpnI. It cuts methylated DNA only, leading to remove the DNA template.</P><P>Therefore transformation is done with unmethylated PCR product, where plasmids contain the desired mutation.</P><H4>DpnI digest</H4><P>add 3 µl Cutsmart, 0.5 µl water and 0.5 µl DpnI. Incubate at 37 °C for 90 minutes, followed by an inactivation step at 80 °C for 20 minutes.</P><H4>test digest</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR>2 µl buffer</TR><TR>about 500 - 1000 ng DNA</TR><TR>0.5 µl enzyme</TR><TR>up to 20 µl water</TR></TBODY></TABLE><P>PCR products or test digests of plasmids where separated gelelectrophoresis, containing 1 % agarose. Small gels were run at 95 V for 90 minutes, large gels at 105 V for 90 minutes.</P><H3>2014.08.21.</H3><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD>PCR #</TD><TD>template</TD><TD>primer 1</TD><TD>primer 2</TD><TD>polymerase</TD><TD>product</TD><TD>size</TD><TD>annealing temperature</TD><TD>elongation time</TD></TR><TR><TD>1</TD><TD>pKM297</TD><TD>o14_sd_003</TD><TD>o14_sd_004</TD><TD>Q5</TD><TD>ePDZb_G1448C_PstI</TD><TD>3.8 kb</TD><TD>60 °C</TD><TD>01:55 min</TD></TR><TR><TD>2</TD><TD>pKM292</TD><TD>o14_sd_011</TD><TD>o14_sd_012</TD><TD>Q5</TD><TD>GAL4_binding_domain_RFC_25</TD><TD>0.5 kb</TD><TD>60 °C</TD><TD>00:16 min</TD></TR><TR><TD>3</TD><TD>pKM292</TD><TD>o14_sd_019</TD><TD>o14_sd_020</TD><TD>Q5</TD><TD>LOV_C3423T_PstI</TD><TD>3.7 kb</TD><TD>60</TD><TD>01:53 min</TD></TR><TR><TD>4</TD><TD>zl_003</TD><TD>o14_sd_023</TD><TD>o14_sd_024</TD><TD>Q5</TD><TD>P2A_RFC_10</TD><TD>0.1 kb</TD><TD>60</TD><TD>00:04 min</TD></TR><TR><TD>5</TD><TD>zl_003</TD><TD>o14_sd_027</TD><TD>o14_sd_028</TD><TD>Q5</TD><TD>P2A_C9374G_NgoMIV</TD><TD>11.7 kb</TD><TD>60</TD><TD>05:50 min</TD></TR></TBODY></TABLE><P>Each PCR was done in triplicates and digested with DpnI. Each replicate was used for transformation with competent bacteria on a seperated Ampicillin plate and incubated at 37 °C. bacteria containing zl_003 were incubated at 32 °C.</P><H3>2014.08.22.</H3><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD>PCR #</TD><TD>template</TD><TD>primer 1</TD><TD>primer 2</TD><TD>polymerase</TD><TD>product</TD><TD>size</TD><TD>annealing temperature</TD><TD>elongation time</TD></TR><TR><TD>6</TD><TD>zl_oo3</TD><TD>o14_sd_029</TD><TD>o14_sd_030</TD><TD>Phusion</TD><TD>puromycin_resistance_gene_RFC_25</TD><TD>0.6 kb</TD><TD>60 °C</TD><TD>00:20 min</TD></TR><TR><TD>7</TD><TD>rz_008</TD><TD>o14_sd_008</TD><TD>o14_sd_009</TD><TD>Phusion</TD><TD>CAT-1_C1168T_PstI</TD><TD>6.4 kb</TD><TD>60 °C</TD><TD>03:15 min</TD></TR><TR><TD>8</TD><TD>pKM084</TD><TD>o14_sd_047</TD><TD>o14_sd_048</TD><TD>Phusion</TD><TD>SEAP_G1419C_PstI</TD><TD>5.4 kb</TD><TD>60</TD><TD>02:45 min</TD></TR></TBODY></TABLE><P>All PCRs were digested with DpnI and transformed on Ampicillin containing plates at 37 °C.</P><P>Transformation from PCR 1.1/1.2/1.3, 3.1/3.2/3.3, 5.1/5.3 were successful, but entirely PCR 2 &amp; 4 not (repeated with inkubation at 32 °C). Three colonies were picked from each plate and used for over night culures (ONC).</P><P>Advice from instructor: an elongation time of 30 s for each DNA fragment smaller than 1 kb will be used.</P><H3>2014.08.23.</H3><P>A mini prep kit from Quiagen (250 reactions) were used for DNA isolation:</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD>template</TD><TD>concentration [ng/µl]</TD><TD>template</TD><TD>concentration [ng/µl]</TD></TR><TR><TD>1.1 I</TD><TD>155.2 ng/µl</TD><TD>1.2 I</TD><TD>174.0 ng/µl</TD></TR><TR><TD>1.1 II</TD><TD>153.1 ng/µl</TD><TD>1.2 II</TD><TD>727.5 ng/µl</TD></TR><TR><TD>1.1 III</TD><TD>566.6 ng/µl</TD><TD>1.2 III</TD><TD>562.5 ng/µl</TD></TR><TR><TD>1.3 I</TD><TD>265.8 ng/µl</TD><TD>3.1 I</TD><TD>174.1 ng/µl</TD></TR><TR><TD>1.3 II</TD><TD>226.4 ng/µl</TD><TD>3.1 II</TD><TD>144.5 ng/µl</TD></TR><TR><TD>1.3 III</TD><TD>187.7 ng/µl</TD><TD>3.1 III</TD><TD>144.4 ng/µl</TD></TR><TR><TD>3.2 I</TD><TD>197.0 ng/µl</TD><TD>3.3 I</TD><TD>163.1 ng/µl</TD></TR><TR><TD>3.3 II</TD><TD>163.5 ng/µl</TD><TD>5.1 I</TD><TD>150.0 ng/µl</TD></TR><TR><TD>5.3 I</TD><TD>448.7 ng/µl</TD><TD>5.1 II</TD><TD>442.2 ng/µl</TD></TR><TR><TD>5.3 II</TD><TD>230.1 ng/µl</TD><TD>5.1 III</TD><TD>295.7 ng/µl</TD></TR><TR><TD>5.3 III</TD><TD>559.4 ng/µl</TD></TR></TBODY></TABLE><P>test digest with:</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD>PCR</TD><TD>enzyme</TD><TD>buffer</TD><TD>fragment size</TD></TR><TR><TD>1</TD><TD>PstI/ClaI</TD><TD>CutSmart</TD><TD>mutated: 2.4 kb, 1.5 kb, not mutated: 2.4 kb, 0.9 kb, 0.6 kb</TD></TR><TR><TD>3</TD><TD>PstI/BbsI</TD><TD>CutSmart</TD><TD>mutated: 2.4 kb, 0.65 kb, 0.53 kb; not mutated: 2.4 kb, 0.53 kb, .04 kb, .026 kb, 0.23  kb, 0.06 kb</TD></TR><TR><TD>5</TD><TD>AgeI/NgoMIV</TD><TD>NEB 1.1</TD><TD>mutated: 5.1 kb, 4.7 kb, 1.8 kb; not mutated: 4.7 kb, 4.1 kb, 2.8 kb</TD></TR></TBODY></TABLE><P>0.625 µl BbsI was added, due to its decreased activity (75%) in NEB 1.1. Test digests were done over night.</P><P>Transformations of PCR 8 and PCR 5 didn't work. Repeated in duplicates: SEAP was incubated at 37 °C, P2A at 32 °C.</P><P>5 ONC of PCR 7 (CAT-1) were done.</P><H3>2014.08.24.</H3><P>results of gel electrophoresis:</P><FIGURE><FIGCAPTION><P class="desc">gel electrophoresis of PCR 1 &amp; 5.</P></FIGCAPTION></FIGURE><P>PCR 1:         PCR 1.1 I and PCR 1.1 II showed perfect fragment sizes (2.3 kb &amp; 1.4 kb). àCandidates for sequencing.</P><P>PCR 5:         PCR 5.1 &amp; PCR 5.3 were successful. DNA fragments at 5.1 kb and 4.7 kb. Fragment at 2 kb invisible, but if DNA wasn’t mutated, there would not be a fragment at 5.1 kb. à Candidates for sequencing.</P><FIGURE><FIGCAPTION><P class="desc">gel electrophoresis of PCR 3.</P></FIGCAPTION></FIGURE><P>PCR 3:         PCR 3.1 III looked fine. Fragments as expected.à Candidate for sequencing.</P><P>Despite missing sequencing results, PCR 5.1 I was used for a transformation.</P><H4>DNA extraction of CAT-1 (5x)</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Concentration [ng/µl]</P></TD></TR><TR><TD><P>7 I</P></TD><TD><P>489.2</P></TD></TR><TR><TD><P>7 II</P></TD><TD><P>316.5</P></TD></TR><TR><TD><P>7 III</P></TD><TD><P>385.0</P></TD></TR><TR><TD><P>7 IV</P></TD><TD><P>451.8</P></TD></TR><TR><TD><P>7 V</P></TD><TD><P>550.0</P></TD></TR></TBODY></TABLE><P> </P><P>Test digest with:</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>7</P></TD><TD><P>AgeI/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated:4.2 kb, 1.6 kb</P><P>not mutated: 4.2 kb, 1.3 kb, 0.3 kb</P></TD></TR></TBODY></TABLE><P>test digest was done overnight.</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>10</P></TD><TD><P>pKM292</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Phusion</P></TD><TD><P>GAL4_RFC_25</P></TD><TD><P>0.5 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>11</P></TD><TD><P>zl_003</P></TD><TD><P>o14sd_29</P></TD><TD><P>014sd_030</P></TD><TD><P>Phusion</P></TD><TD><P>Puromycin_resistance_gene_RFC_25</P></TD><TD><P>0.6 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P> </P><H4>2014.08.25</H4><P> </P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>12</P></TD><TD><P>pKM084</P></TD><TD><P>o14sd_047</P></TD><TD><P>014sd_048</P></TD><TD><P>Phusion</P></TD><TD><P>SEAP_G1419C_PstI</P></TD><TD><P>5.4 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>13</P></TD><TD><P>zl_003</P></TD><TD><P>o14sd_055</P></TD><TD><P>014sd_056</P></TD><TD><P>Phusion</P></TD><TD><P>WPRE_C530A_NgoMIV</P></TD><TD><P>11.7 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>14</P></TD><TD><P>LOV PCR Product</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Phusion</P></TD><TD><P>GAL4_ RFC_25</P></TD><TD><P>0.5 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P> </P><H2>Result gel electrophoresis (digest CAT-1)</H2><FIGCAPTION><P class="desc">gel electrophoresis of PCR 7.</P></FIGCAPTION></SECTION><P>Result of colony 1 looked fine. à Candidate for sequencing.</P><P>Transformation of PCR 6 &amp; 8 did not work.</P><H3>2014.08.26</H3><H4>DNA extraction from PCR 5.1 (5x)</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Concentration [ng/µl]</P></TD></TR><TR><TD><P>5.1 I</P></TD><TD><P>213.0</P></TD></TR><TR><TD><P>5.1 II</P></TD><TD><P>283.2</P></TD></TR><TR><TD><P>5.1 III</P></TD><TD><P>254.1</P></TD></TR><TR><TD><P>5.1 IV</P></TD><TD><P>198.4</P></TD></TR><TR><TD><P>5.1 V</P></TD><TD><P>232.2</P></TD></TR></TBODY></TABLE><P> </P><P>Test digests according to 2014.08.23 overnight.</P><H3>2014.08.27</H3><H4>Gel electrophoresis from test digest PCR 5.1</H4><FIGCAPTION><P class="desc">gel electrophoresis of PCR 5.1.</P></FIGCAPTION><P>Test digests looked fine. Fragments as expected at 5.1 kb, 4.7 kb and about 2.0 kb. PCR 5.1 I was used as a template to amplify P2A with RFC 25 prefix and suffix.</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>15</P></TD><TD><P>PCR 5.1 I</P></TD><TD><P>o14sd_025</P></TD><TD><P>014sd_026</P></TD><TD><P>Phusion</P></TD><TD><P>P2A_RFC_25</P></TD><TD><P>0.1 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>PCR 15 was done in 50 µl approach and triplicates.</P><H4>Gel electrophoresis from PCR 15</H4><FIGCAPTION><P class="desc">gel electrophoresis of PCR 15.</P></FIGCAPTION><P>Gel contains 2 % agarose. Marker is 100 bp from Promega. Fragments lay between 100 and 200 bp. Theoretical size 136 bp. àFragments cut out and prepared for gel extraction. All three fragments were run over one column.</P><P>PCR 14 repeated as PCR 16 due to contaminated water.</P><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>16</P></TD><TD><P>LOV PCR Product</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Phusion</P></TD><TD><P>GAL4_ RFC_25</P></TD><TD><P>0.5 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P> </P><H4>Gel electrophoresis from PCR 16</H4><FIGCAPTION><P class="desc">gel electrophoresis of PCR 16.</P></FIGCAPTION><P>Gel contains 2 % agarose. Marker is 100 bp from Promega. Fragments lay at 500 bp, expected size: 517 bp. Fragments were cut out and prepared for gel extraction. All three fragments were run over one column.</P><H4>Gel extraction from P2A RFC 25 and GAL4 RFC 25</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>name</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>GAL4 RFC 25</P></TD><TD><P>38.5</P></TD></TR><TR><TD><P>P2A RFC 25</P></TD><TD><P>34.4</P></TD></TR></TBODY></TABLE><P> </P><H4>Ligation of P2A RFC 25 and GAL4 RFC 25 in pSB1C3</H4><P>Preparative digest according to test digest protocol, but incubated for four hours. Inserts and backbone were cut with EcoRI-HF/PstI-HF in CutSmart.</P><P>Used ratio backbone:insert = 1:3</P><H4>Used volumes of insert and backbone</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>GAL4 RFC 25</P></TD><TD><P>1.72 µl</P></TD><TD><P>pSB1C3</P></TD><TD><P>0.48 µl</P></TD></TR><TR><TD><P>P2A RFC 25</P></TD><TD><P>1.7 µl</P></TD><TD><P>pSB1C3</P></TD><TD><P>0.5 µl</P></TD></TR><TR><TD><P>control (water)</P></TD><TD><P>1.7 µl</P></TD><TD><P>pSB1C3</P></TD><TD><P>0.5 µl</P></TD></TR></TBODY></TABLE><P>Ligation was performed with T4 ligase (40.000 U/ml) from NEB</P><P>After transformed with DNA bacteria culture were incubated at 37 °C for one hour. Afterwards cultures were streaked out onto agar plates containing Chloramphenicol.</P><P>Despite missing sequencing results, ONC were made from PCR 1.1 I &amp; PCR 3.1 III (5x each).</P><H3>2014.08.28</H3><P>Sequencing results: P2A not mutated, CAT-1: fail. nucleotide length 1 nt., LOV: not been uploaded.</P><H4>DNA extraction from ONC PCR 1.1 I (ePDZb) &amp; PCR 3.1 III (LOV)</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>LOV I</P></TD><TD><P>103.7</P></TD><TD><P>ePDZb I</P></TD><TD><P>136.8</P></TD></TR><TR><TD><P>LOV II</P></TD><TD><P>91.4</P></TD><TD><P>ePDZb II</P></TD><TD><P>121.4</P></TD></TR><TR><TD><P>LOV III</P></TD><TD><P>119.4</P></TD><TD><P>ePDZb II</P></TD><TD><P>137.0</P></TD></TR><TR><TD><P>LOV IV</P></TD><TD><P>113.4</P></TD><TD><P>ePDZb IV</P></TD><TD><P>102.2</P></TD></TR><TR><TD><P>LOV V</P></TD><TD><P>129.9</P></TD><TD><P>ePDZb V</P></TD><TD><P>155.2</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digests PCR 1.1 I &amp; PCR 3.1 III:</H4><TABLE border="1" cellspacing="0" cellpadding="0"><TBODY><TR><TD><P>template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>LOV</P></TD><TD><P>XhoI/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated:2.7 kb, 0.7 kb</P><P>not mutated: 2.7 kb, 0.4 kb, 0.3 kb</P></TD></TR><TR><TD><P>ePDZb</P></TD><TD><P>ClaI/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated:2.3 kb, 1.5 kb</P><P>not mutated: 2.3 kb, 0.9 kb, 0.6 kb</P></TD></TR></TBODY></TABLE><P> </P><FIGCAPTION><P class="desc">gel electrophoresis of LOV &amp; ePDZb.</P></FIGCAPTION><P>LOV didn’t work -&gt; no fragment at 0.7 kb. ePDZb looked fine. Visible fragment at 1.5 kb as expected.</P><P>Ligation of P2A RFC 25 and GAL4 RFC 25 didn’t work. Next time preparative digest will be desalted by PCR purification kit.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>17</P></TD><TD><P>zl_003</P></TD><TD><P>o14sd_023</P></TD><TD><P>014sd_024</P></TD><TD><P>Q5</P></TD><TD><P>P2A_RFC_10</P></TD><TD><P>0.1 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>18</P></TD><TD><P>zl_003</P></TD><TD><P>o14sd_057</P></TD><TD><P>014sd_058</P></TD><TD><P>Q5</P></TD><TD><P>WPRE_RFC_10</P></TD><TD><P>0.6 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>19</P></TD><TD><P>PCR 5.1 II</P></TD><TD><P>o14sd_039</P></TD><TD><P>014sd_040</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1_RFC_10</P></TD><TD><P>1.9 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>20</P></TD><TD><P>ePDZb I</P></TD><TD><P>o14sd_005</P></TD><TD><P>014sd_006</P></TD><TD><P>Q5</P></TD><TD><P>ePDZb_A1775G_EcoRI</P></TD><TD><P>3.8 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>21</P></TD><TD><P>pKM084</P></TD><TD><P>o14sd_047</P></TD><TD><P>014sd_048</P></TD><TD><P>Q5</P></TD><TD><P>SEAP_G1419C_PstI</P></TD><TD><P>5.4 kb</P></TD><TD><P>50 °C</P></TD></TR></TBODY></TABLE><P>PCR 20 &amp; 21 digested by DpnI and afterwards transformed onto Ampicillin plates and incubates at 37 °C</P><P>PCR 17 -19 were directly purified preparedly digested overnight with EcoRI-HF/PstI-HF in Cutsmart.</P><P>Apporach:          28 µl eluate</P><P>                               1 µl enzyme each</P><P>                               4 µl CutSmart</P><P>                               6 µl water</P><H3>2014.08.30</H3><H4>ONC from GAL4 RFC 25 in pSB1C3</H4><P>A Chloramphenicol plate from GAL4 RFC 25 ligation (2014.08.27.) was found in the 37 °C incubator. One colony was visible. à Three ONC were picked from this colony.</P><H4>ONC from PCR 20 &amp; 21</H4><FIGCAPTION><P class="desc">gel electrophoresis of PCR 17 &amp; 18.</P></FIGCAPTION><P>Plates were settled with bacteria. Three ONC were picked from each plate.</P><P>P2A looked fine. The expected size was 134 bp. Fragments were above the100 bp lane and cut out for gel extraction. WPRE showed expected lane sizes (646 bp.). Cut out for gel extraction.</P><FIGCAPTION><P class="desc">gel electrophoresis of PCR 19.</P></FIGCAPTION><P>Instead of 2-logfrom NEB, Promega’s 100 bp ladder was used. Expected size for CAT-1 RFC 10 were 1577 bp. The highest lane of DNA ladder is 1.5 kb -&gt; PCR products were cut out for gel extraction.</P><P>All triplicates of one sample were run over one column.</P><H4>Gel extraction from preparative digests (2014.08.29.)</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>name</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>P2A RFC 10 (PCR 17)</P></TD><TD><P>22.9</P></TD></TR><TR><TD><P>WPRE RFC 10 (PCR 18)</P></TD><TD><P>53.9</P></TD></TR><TR><TD><P>CAT-1 RFC 10 (PCR 19)</P></TD><TD><P>17.4</P></TD></TR></TBODY></TABLE><P> </P><H4>Ligation of WPRE RFC 10, CAT-1 RFC 10, P2A RFC 10 in pSB1C3</H4><P>ratio backbone:insert = 1:3</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P> </P></TD><TD><P>Volume [µl]</P></TD><TD><P>Volume backbone [µl]</P></TD></TR><TR><TD><P>P2A</P></TD><TD><P>1.68</P></TD><TD><P>0.52</P></TD></TR><TR><TD><P>CAT-1</P></TD><TD><P>1.64</P></TD><TD><P>0.56</P></TD></TR><TR><TD><P>WPRE</P></TD><TD><P>1.83</P></TD><TD><P>0.37</P></TD></TR><TR><TD><P>control (water)</P></TD><TD><P>1.72</P></TD><TD><P>0.48</P></TD></TR></TBODY></TABLE><P> </P><P>We got a new ligation buffer from our instructor à Quick ligation buffer. Used buffer volume: 2.6 µl, 2.4 ligation volume and 0.2 T4 ligase (400.000 U/ml) from NEB.</P><P>5 µl ligation approaches were used for transformation. 200 µl antibiotic free LB media were added and the reaction tube was incubated at 37 °C with 400 rpm for one hour. Afterwards whole cultures were streaked out onto plates containing Chloramphenicol.</P><H3>2014.08.31</H3><H4>DNA extraction from GAL4 RFC 25 ONC (1x), PCR 20 (3x), PCR 21 (3x)</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>GAL4 RFC 25 in pSB1C3</P></TD><TD><P>154.2</P></TD><TD><P>(PCR 21) SEAP I</P></TD><TD><P>188.3</P></TD></TR><TR><TD><P>(PCR 20) ePDZb I</P></TD><TD><P>91.4</P></TD><TD><P>(PCR 21) SEAP II</P></TD><TD><P>228.3</P></TD></TR><TR><TD><P>(PCR 20) ePDZb II</P></TD><TD><P>119.4</P></TD><TD><P>(PCR 21) SEAP III</P></TD><TD><P>209.6</P></TD></TR><TR><TD><P>(PCR 20) ePDZb III</P></TD><TD><P>113.4</P></TD><TD colspan="2" width="307"><P> </P></TD></TR></TBODY></TABLE><P> </P><H4>Test digests</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>GAL4 RFC 25 in pSB1C3</P></TD><TD><P>EcoRI-HF/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>2.0 kb, 0.5 kb</P></TD></TR><TR><TD><P>ePDZb</P></TD><TD><P>EcoRI-HF/PstI-HF/NotI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated:1.0 kb</P><P>not mutated: 0.6 kb, 0.4 kb</P></TD></TR><TR><TD><P>SEAP</P></TD><TD><P>PstI-HF/BbsI</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 4.1 kb, 1.3 kb</P><P>not mutated: 4.1 kb, 1 kb, 0.3 kb</P></TD></TR></TBODY></TABLE><P>Digests pipetted according to protocol.</P><FIGCAPTION><P class="desc">gel electrophoresis of PCR 21.</P></FIGCAPTION><P>Digests of GAL4 and ePDZb were run over 2 % agarose gel. Promega 100 bp was used as DNA ladder. SEAP was run over normal gel, with standard DNA ladder.</P><P>Fragment sizes weren’t as expected. Five new colonies for ONC were picked from same plate.</P><H2 id="Standardization-September">Standardization - September</H2><H3>2014.09.01</H3><P>No ligation worked. They were repeated with a different protocol.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>WPRE RFC 10</P></TD><TD><P>0.83 µl</P></TD></TR><TR><TD><P>SLC7A1 RFC 10</P></TD><TD><P>7.98 µl</P></TD></TR><TR><TD><P>P2A RFC 10</P></TD><TD><P>0.34 µl</P></TD></TR><TR><TD><P>pSB1C3</P></TD><TD><P>always 2.75 µl</P></TD></TR></TBODY></TABLE><P> </P><P>Protocol:             2 µl 10x T4 buffer</P><P>                               0.2 µl T4 ligase (4e5 U/ml)</P><P>                               add water up to 20 µl</P><P>                               incubate at room temperature for 15 minutes</P><P> </P><P>Transformed bacteria were streaked out onto plates containing Chloramphenicol and incubated at 37 °C</P><P> </P><H4>DNA extraction from SEAP ONC (5x)</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>SEAP I</P></TD><TD><P>215.7</P></TD></TR><TR><TD><P>SEAP II</P></TD><TD><P>257.1</P></TD></TR><TR><TD><P>SEAP III</P></TD><TD><P>285.0</P></TD></TR><TR><TD><P>SEAP IV</P></TD><TD><P>139.9</P></TD></TR><TR><TD><P>SEAP V</P></TD><TD><P>367.6</P></TD></TR></TBODY></TABLE><P>Afterwards DNA was digested according to protocol.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>SEAP</P></TD><TD><P>PstI-HF/NotI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 3.7 kb, 1.7 kb</P><P>not mutated: 3.7 kb, 1.4 kb, 0.3 kb</P></TD></TR></TBODY></TABLE><P> </P><FIGCAPTION><P class="desc">test digest of overnight culture. Took from agar plate  containig PCR 21.</P></FIGCAPTION><P>Gel looked fine. SEAP III and SEAP IV will be sequenced. DNA Mini of CAT-1 (2014.08.22.) will be sequenced too, using another sequencing primer.</P><H3>2014.09.02</H3><P>Sequencing results of CAT-1 showed that it didn’t contain any PstI site anymore.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>22</P></TD><TD><P>ePDZb_mutated</P></TD><TD><P>o14sd_010</P></TD><TD><P>014sd_020</P></TD><TD><P>Phusion</P></TD><TD><P>ePDZb_RFC_25_fwd -&gt; AgeI site</P></TD><TD><P>0.5 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>23</P></TD><TD><P>ePDZb_mutated</P></TD><TD><P>o14sd_002</P></TD><TD><P>014sd_008</P></TD><TD><P>Phusion</P></TD><TD><P>ePDZb_AgeI -&gt; ePDZb_RFC_25_rev</P></TD><TD><P>0.2 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>24</P></TD><TD><P>SEAP III</P></TD><TD><P>o14sd_049</P></TD><TD><P>014sd_050</P></TD><TD><P>Phusion</P></TD><TD><P>SEAP_RFC_10</P></TD><TD><P>1.6 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>25</P></TD><TD><P>SEAP III</P></TD><TD><P>o14sd_045</P></TD><TD><P>014sd_046</P></TD><TD><P>Phusion</P></TD><TD><P>SEAP_C2784G_NgoMIV</P></TD><TD><P>5.4 kb</P></TD><TD><P>50 °C</P></TD></TR></TBODY></TABLE><P> </P><FIGCAPTION><P class="desc">test digest of PCR 22 &amp; 23.</P></FIGCAPTION><P>PCR 22 + PCR 23 were done in duplicates, PCR 24 in triplicates. Gel electrophoresis was done in 2 % gel.</P><P>All PCR but PCR 23 didn’t show expected size. PCR 23’s DNA fragments were cut out and prepared for gel extraction.</P><P>Ligation of CAT-1 and P2A in pSB1C3 didn’t work, but plate with bacteria containing WPRE RFC 10 in pSB1C3 shows eight colonies. They were picked for ONC.</P><P>Ligation of CAT-1 and P2a were repeated with a modified yesterday’s protocol. 2 µl of T4 ligase (40,000 U/ml) instead 0.2 µl T4 ligase (400,000 U/ml) were used.</P><H3>2014.09.03</H3><P> </P><H4>DNA extraction of WPRE ONC (8x)</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>WPRE RFC 10 I</P></TD><TD><P>132.9</P></TD><TD><P>WPRE RFC 10 V</P></TD><TD><P>130.0</P></TD></TR><TR><TD><P>WPRE RFC 10 II</P></TD><TD><P>148.7</P></TD><TD><P>WPRE RFC 10 VI</P></TD><TD><P>106.7</P></TD></TR><TR><TD><P>WPRE RFC 10 III</P></TD><TD><P>91.9</P></TD><TD><P>WPRE RFC 10 VII</P></TD><TD><P>89.0</P></TD></TR><TR><TD><P>WPRE RFC 10 IV</P></TD><TD><P>148.8</P></TD><TD><P>WPRE RFC 10 VIII</P></TD><TD><P>106.7</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digest of WPRE RFC 10</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>WPRE</P></TD><TD><P>PstI-HF/EcoRI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 2.0 kb, 0.6 kb</P></TD></TR></TBODY></TABLE><P> </P><FIGCAPTION><P class="desc">test digest of ligated WPRE RFC 10 into pSB1C3.</P></FIGCAPTION><P>Digests were run over 1 % agarose gel</P><P>All eight test digests looked fine. WPRE RFC V will be sequenced. Other ligations didn’t work perhaps of damaged DNA.</P><P>Bacteria were transformed on Ampicillin plate with PCR 25.</P><H3>2014.09.04</H3><P>Sequencing results showed that WPRE was correctly ligated into pSB1C3.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>26</P></TD><TD><P>CAT-1 mutated</P></TD><TD><P>o14sd_031</P></TD><TD><P>014sd_032</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1_C1184A_NgoMIV</P></TD><TD><P>5.8 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>27</P></TD><TD><P>WPRE RFC 10 V</P></TD><TD><P>o14sd_055</P></TD><TD><P>014sd_056</P></TD><TD><P>Q5</P></TD><TD><P>WPRE_C530A_NgoMIV</P></TD><TD><P>2.5 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>28</P></TD><TD><P>LOV_mutated</P></TD><TD><P>o14sd_021</P></TD><TD><P>014sd_022</P></TD><TD><P>Q5</P></TD><TD><P>LOV_RFC_25</P></TD><TD><P>3.8 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>29</P></TD><TD><P>ePDZb_EcoRI_out</P></TD><TD><P>o14sd_021</P></TD><TD><P>014sd_022</P></TD><TD><P>Q5</P></TD><TD><P>ePDZb_RFC_10</P></TD><TD><P>3.8 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>30</P></TD><TD><P>pKM292</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Q5</P></TD><TD><P>GAL4 RFC 25</P></TD><TD><P>0.6 kb</P></TD><TD><P>50 °C</P></TD></TR></TBODY></TABLE><P>All PCRs were used to transform bacteria (Ampicillin, but PCR 27 Chloramphenicol) and afterwards incubated at 37 °C.</P><P>Triplicates of PCR 30 were made (50 µl) and preparative digest (EcoRI-HF/PstI-HF) was done directly in PCR tube. and afterwards purified. Concentration: 130 ng/µl.</P><P>Eight colonies were picked from plate containing PCR 15.</P><H3>2014.09.05</H3><H3>DNA extraction of ONC PCR 15 (8x) and PCR 25 (5x)</H3><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>P2A I</P></TD><TD><P>275.4</P></TD><TD><P>SEAP I</P></TD><TD><P>248.1</P></TD></TR><TR><TD><P>P2A II</P></TD><TD><P>192.4</P></TD><TD><P>SEAP II</P></TD><TD><P>127.1</P></TD></TR><TR><TD><P>P2A III</P></TD><TD><P>176.3</P></TD><TD><P>SEAP III</P></TD><TD><P>218.2</P></TD></TR><TR><TD><P>P2A IV</P></TD><TD><P>339.3</P></TD><TD><P>SEAP IV</P></TD><TD><P>145.9</P></TD></TR><TR><TD><P>P2A V</P></TD><TD><P>373.3</P></TD><TD><P>SEAP V</P></TD><TD><P>167.4</P></TD></TR><TR><TD><P>P2A VI</P></TD><TD><P>304.2</P></TD></TR><TR><TD><P>P2A VII</P></TD><TD><P>167.7</P></TD></TR><TR><TD><P>P2A VIII</P></TD><TD><P>305.7</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digests PCR 15 and PCR 25</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>P2A</P></TD><TD><P>NgoMIV</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 7.0 kb, 4.7 kb</P><P>not mutated: 4.7 kb, 4.1 kb, 2.8 kb</P></TD></TR><TR><TD><P>SEAP</P></TD><TD><P>NgoMIV</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 5,4 kb</P><P>not mutated: 4.4 kb, 1 kb</P></TD></TR></TBODY></TABLE><P> </P><FIGCAPTION><P class="desc">test digest of PCR 15.</P></FIGCAPTION><P>P2A and SEAP separated by gel electrophoresis.</P><FIGCAPTION><P class="desc">test digest of PCR 25.</P></FIGCAPTION><P>Test digests of SEAP didn’t work. P2A V + VIII showed fragments with correct size.</P><H4>Transformations and ligations</H4><P>No transformation worked.</P><P>Following transformations were made:</P><P>PCR 26 -29, on Ampicillin plates and incubated at 37 °C</P><P>Following ligations were made:</P><P>CAT-1 RFC 10, P2A RFC 10, GAL4 RFC 25 in pSB1C3.</P><H3>2014.09.06</H3><P>No ligation worked.</P><H3>2014.09.07</H3><P> </P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>31</P></TD><TD><P>SEAP_PstI_out</P></TD><TD><P>o14sd_049</P></TD><TD><P>014sd_050</P></TD><TD><P>Q5</P></TD><TD><P>SEAP RFC 10</P></TD><TD><P>1.6 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>32</P></TD><TD><P>CAT-1 mutated V</P></TD><TD><P>o14sd_031</P></TD><TD><P>014sd_032</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1 RFC 10</P></TD><TD><P>1.9 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>33</P></TD><TD><P>P2A_mutated_V</P></TD><TD><P>o14sd_025</P></TD><TD><P>014sd_026</P></TD><TD><P>Q5</P></TD><TD><P>P2A RFC 25</P></TD><TD><P>0.1 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>34</P></TD><TD><P>GAL4_binding_domain</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Q5</P></TD><TD><P>GAL4 RFC 25</P></TD><TD><P>0.6 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>35</P></TD><TD><P>WPRE RFC 10</P></TD><TD><P>o14sd_055</P></TD><TD><P>014sd_056</P></TD><TD><P>Q5</P></TD><TD><P>WPRE RFC 25</P></TD><TD><P>0.6 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>PCR 31-34 were made in 50 µl, PCR 35 in 25 µl.</P><FIGCAPTION><P class="desc">test digest of PCR 31 &amp; 32.</P></FIGCAPTION><P>SEAP looked fine, but CAT-1 showed unspecific PCR products.</P><H4>Gel extraction of GAL4 binding domain, SEAP, P2A</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>GAL 4 binding domain RFC 25</P></TD><TD><P>60.8</P></TD></TR><TR><TD><P>SEAP RFC 10</P></TD><TD><P>27.9</P></TD></TR><TR><TD><P>P2A RFC 10</P></TD><TD><P>11.9</P></TD></TR></TBODY></TABLE><P> </P><P>Ligation protocol</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>plasmid</P></TD><TD><P>backbone</P></TD><TD><P>P2A</P></TD><TD><P>SEAP</P></TD><TD><P>GAL 4</P></TD></TR><TR><TD><P>length (bp)</P></TD><TD><P>2070</P></TD><TD><P>136</P></TD><TD><P>1577</P></TD><TD><P>507</P></TD></TR><TR><TD><P>concentration [ng/µl]</P></TD><TD><P>27.3</P></TD><TD><P>11.9</P></TD><TD><P>27.9</P></TD><TD><P>60.8</P></TD></TR><TR><TD><P>volume [µl]</P></TD><TD><P>1.83</P></TD><TD><P>0.83</P></TD><TD><P>4.10</P></TD><TD><P>0.60</P></TD></TR><TR><TD><P>water</P></TD><TD><P> </P></TD><TD><P>14.83</P></TD><TD><P>11.57</P></TD><TD><P>15.06</P></TD></TR><TR><TD colspan="5" width="619"><P>plus 2 µl 1x T4 buffer and 0.5 µl T4 ligase (400,000 U/ml) each</P></TD></TR></TBODY></TABLE><P>Approaches were incubated at room temperature for 30 minutes.</P><P>Transformation: lent cells from our instructor and added 10 µl of ligation approach and 3 µl of PCR 38 respectively.</P><P>Transformations were made in duplicates.</P><H3>2014.09.08</H3><P>All ligations worked. Three ONC were made from GAL4 RFC 25, P2A RFC 25, SEAP RFC 10, six from PCR 25, eight from WPRE RFC 25.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>36</P></TD><TD><P>CAT-1_PstI_out</P></TD><TD><P>o14sd_039</P></TD><TD><P>014sd_040</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1 RFC 10</P></TD><TD><P>1.9 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>37</P></TD><TD><P>ePDZb II (145.3 ng/µl)</P></TD><TD><P>o14sd_005</P></TD><TD><P>014sd_006</P></TD><TD><P>Q5</P></TD><TD><P>ePDZB_A1775G_EcoRI</P></TD><TD><P>3.8 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>PCR according to protocol, afterwards PCRs were digested by DpnI and transformed onto agar plates containing Ampicilin.</P><H3>2014.09.09</H3><P>DNA extractions failed due to old buffers. Agar plates were overgrown. Bacteria were diluted onto another plate. Transformation of PCR 36 failed. Repeated it with different primers</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>38</P></TD><TD><P>CAT-1_PstI_out</P></TD><TD><P>o14sd_039</P></TD><TD><P>014sd_040</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1 RFC 10</P></TD><TD><P>1.9 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>39</P></TD><TD><P>CAT-1_PstI_out</P></TD><TD><P>o14sd_039</P></TD><TD><P>014sd_040</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1 RFC 10</P></TD><TD><P>1.9 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P> </P><H3>2014.09.10</H3><FIGCAPTION><P class="desc">test digest of PCR 38 &amp; 39.</P></FIGCAPTION><P>Amplifications of CAT-1 RFC 10 were successful. The amount of PCR product at 55 °C was much higher than at 50 °C. Both lanes were cut out used for ligation into pSB1C3.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>40</P></TD><TD><P>CAT-1_PstI_out</P></TD><TD><P>o14sd_031</P></TD><TD><P>014sd_032</P></TD><TD><P>Q5</P></TD><TD><P>CAT-1_C1184A_NgoMIV</P></TD><TD><P>5.8 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>PCR according to protocol, afterwards PCR was digested by DpnI and transformed onto agar plates containing Ampicilin.</P><H4>Gel extraction of CAT-1</H4><P>Concentration: 21.4 ng/µl</P><P>Ligation approach:</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Plasmid</P></TD><TD><P>backbone</P></TD><TD><P>CAT-1 RFC 10</P></TD></TR><TR><TD><P>length (bp)</P></TD><TD><P>2070</P></TD><TD><P>1925</P></TD></TR><TR><TD><P>concentration [ng/µl]</P></TD><TD><P>5.1</P></TD><TD><P>21.4</P></TD></TR><TR><TD><P>volume [µl]</P></TD><TD><P>9.8</P></TD><TD><P>6.33</P></TD></TR><TR><TD><P>Water</P></TD><TD><P>11.2</P></TD><TD><P>4.37</P></TD></TR><TR><TD><P>plus 2 µl 1x T4 buffer and 0.5 µl T4 ligase (400,000 U/ml) each</P></TD></TR></TBODY></TABLE><P>Control and ligation were incubated at room temperature for 30 minutes. Transformation was done with 10 µl each. Bacteria cultures were incubated at 37 °C and 600 rpm for 60 minutes.</P><P>Transformation was done with 100, 50, 20, 10 µl to discover the optimal transformation volume.</P><P>Several ONC were made (PCR 31 – 35 (3x each), PCR 37 (10x)).</P><H3>2014.09.11</H3><H4>DNA extraction from ONC</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>ePDZB_A1775G_EcoRI I</P></TD><TD><P>98.5</P></TD><TD><P>ePDZB_A1775G_EcoRI VI</P></TD><TD><P>142.1</P></TD></TR><TR><TD><P>ePDZB_A1775G_EcoRI II</P></TD><TD><P>91.5</P></TD><TD><P>ePDZB_A1775G_EcoRI VII</P></TD><TD><P>78.1</P></TD></TR><TR><TD><P>ePDZB_A1775G_EcoRI III</P></TD><TD><P>95.5</P></TD><TD><P>ePDZB_A1775G_EcoRI VIII</P></TD><TD><P>64.0</P></TD></TR><TR><TD><P>ePDZB_A1775G_EcoRI IV</P></TD><TD><P>93.8</P></TD><TD><P>ePDZB_A1775G_EcoRI IX</P></TD><TD><P>80.3</P></TD></TR><TR><TD><P>ePDZB_A1775G_EcoRI V</P></TD><TD><P>78.0</P></TD><TD><P>ePDZB_A1775G_EcoRI X</P></TD><TD><P>71.1</P></TD></TR><TR><TD><P>GAL4 RFC 25 I</P></TD><TD><P>105.9</P></TD><TD><P>WPRE_C530A_NgoMIV I</P></TD><TD><P>109.3</P></TD></TR><TR><TD><P>GAL4 RFC 25 II</P></TD><TD><P>123.8</P></TD><TD><P>WPRE_C530A_NgoMIV II</P></TD><TD><P>119.0</P></TD></TR><TR><TD><P>GAL4 RFC 25 III</P></TD><TD><P>117.6</P></TD><TD><P>WPRE_C530A_NgoMIV III</P></TD><TD><P>209.5</P></TD></TR><TR><TD><P>P2A RFC 10 I</P></TD><TD><P>107.3</P></TD><TD><P>SEAP RFC 10 I</P></TD><TD><P>117.5</P></TD></TR><TR><TD><P>P2A RFC 10 II</P></TD><TD><P>117.8</P></TD><TD><P>SEAP RFC 10 II</P></TD><TD><P>123.4</P></TD></TR><TR><TD><P>P2A RFC 10 III</P></TD><TD><P>113.1</P></TD><TD><P>SEAP RFC 10 III</P></TD><TD><P>119.6</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digests</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>P2A RFC 10</P></TD><TD><P>EcoRI-HF/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>2.0 kb,0.1 kb kb</P></TD></TR><TR><TD><P>GAL4 RFC 25</P></TD><TD><P>EcoRI-HF/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>2.0 kb, 0.5 kb</P></TD></TR><TR><TD><P>SEAP RFC 10</P></TD><TD><P>EcoRI-HF/PstI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>2.0 kb, 1.5 kb</P></TD></TR><TR><TD><P>ePDZb</P></TD><TD><P>EcoRI-HF/NcoI</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 3.8 kb</P><P>not mutated: 3.2 kb, 0.6 kb</P></TD></TR></TBODY></TABLE><P> </P><H4>Gel result</H4><FIGCAPTION><P class="desc">test digest of PCR 31 &amp; 32.</P></FIGCAPTION><P>It looked like ePDZb hadn’t been totally digested. The second lane lay short under 1 kb lane and not at 600 bp. WPRE RFC 10, SEAP RFC 10 and P2A didn’t show expected results. Especially it seemed that SEAP RFC 10 was actually identical with GAL 4 RFC 25 due to the lanes at 500 bp, which were expected for GAL4 RFC 25.</P><P>As a result GAL4 RFC I will be sequenced</P><P>Eight colonies from containing PCR 40 were picked for ONC.</P><H3>2014.09.12</H3><P>Despite to bad results on gel electrophoresis, P2A I and SEAP RFC 10 I will be sequenced.</P><H4>DNA extraction from ONC</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>CAT-1_C1184A_NgoMIV I</P></TD><TD><P>473.8</P></TD><TD><P>CAT-1_C1184A_NgoMIV V</P></TD><TD><P>269.1</P></TD></TR><TR><TD><P>CAT-1_C1184A_NgoMIV II</P></TD><TD><P>461.6</P></TD><TD><P>CAT-1_C1184A_NgoMIV VI</P></TD><TD><P>329.3</P></TD></TR><TR><TD><P>CAT-1_C1184A_NgoMIV III</P></TD><TD><P>373.3</P></TD><TD><P>CAT-1_C1184A_NgoMIV VII</P></TD><TD><P>452.2</P></TD></TR><TR><TD><P>CAT-1_C1184A_NgoMIV IV</P></TD><TD><P>469.1</P></TD><TD><P>CAT-1_C1184A_NgoMIV VIII</P></TD><TD><P>398.4</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digest</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>CAT-1_C1184A_NgoMIV</P></TD><TD><P>XbaI/NgoMIV</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 4.2, 1.2 kb,0.3 kb kb</P><P>not mutated: 4.2, 1.2 kb,165 bp, 142 bp.</P></TD></TR></TBODY></TABLE><P> </P><H4>Gel electrophoresis</H4><FIGCAPTION><P class="desc">test digest of PCR 38 &amp; 39.</P></FIGCAPTION><P>Samples were run in a 2 % agarose gel. Marker is 100 bp from Promega. Due to missing lane at 300 bp, the restriction site was still in CAT-1.</P><P>Ligation of SEAP RFC 10 and CAT-1 were repeated and digested with EcoRI-HF/PstI-HF in CutSmart buffer. Afterwards samples were purified by PCR purification Kit. Concentrations:</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Plasmid</P></TD><TD><P>backbone</P></TD><TD><P>SEAP</P></TD><TD colspan="2" width="115"><P>CAT-1</P></TD></TR><TR><TD><P>length (bp)</P></TD><TD><P>2070</P></TD><TD><P>1577</P></TD><TD colspan="2" width="115"><P>1869</P></TD></TR><TR><TD><P>concentration [ng/µl]</P></TD><TD><P>16.2</P></TD><TD><P>39.5</P></TD><TD colspan="2" width="115"><P>21.4</P></TD></TR><TR><TD><P>volume [µl]</P></TD><TD><P>1.83</P></TD><TD><P>2.79</P></TD><TD colspan="2" width="115"><P>6.33</P></TD></TR><TR><TD><P>Water</P></TD><TD><P> </P></TD><TD><P>11.20</P></TD><TD colspan="2" width="115"><P>7.59</P></TD></TR><TR><TD colspan="4" width="495"><P>plus 2 µl 1x T4 buffer and 0.5 µl T4 ligase (400,000 U/ml) each</P></TD><TD><P> </P></TD></TR><TR><TD> </TD><TD> </TD><TD> </TD><TD> </TD><TD> </TD></TR></TBODY></TABLE><P>Transformed and incubated in Chloramphenicol containing LB-medium. 10, 50 and 100 µl of cluture were streaked out onto agar plates.</P><H3>2014.09.13</H3><P>Only few colonies were grown. Wait another day before using them for ONC.</P><P>Our instructor recommended us to use Pfu-Polymerase instead of Q5 or Phusion for site-directed mutagenisis PCRs.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>40</P></TD><TD><P>SEAP_PstI_out (09.01.)</P></TD><TD><P>o14sd_045</P></TD><TD><P>014sd_046</P></TD><TD><P>Pfu</P></TD><TD><P>SEAP_G2780C_NgoMIV</P></TD><TD><P>5.8 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>41</P></TD><TD><P>LOV_09_10</P></TD><TD><P>o14sd_039</P></TD><TD><P>014sd_040</P></TD><TD><P>Pfu</P></TD><TD><P>LOV_without_PstI</P></TD><TD><P>3.6 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>42</P></TD><TD><P>ePDZb (145.3 ng/µl)</P></TD><TD><P>o14sd_005</P></TD><TD><P>014sd_006</P></TD><TD><P>Pfu</P></TD><TD><P>ePDZb_Eco_out</P></TD><TD><P>3.4 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>43</P></TD><TD><P>CAT-1 mutated</P></TD><TD><P>o14sd_037</P></TD><TD><P>014sd_038</P></TD><TD><P>Pfu</P></TD><TD><P>CAT-1_AgeI_out</P></TD><TD><P>5.8 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>44</P></TD><TD><P>WPRE RFC 10</P></TD><TD><P>o14sd_059</P></TD><TD><P>014sd_060</P></TD><TD><P>Pfu</P></TD><TD><P>WPRE_NgoMIV_out</P></TD><TD><P>2.6 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>45</P></TD><TD><P>P2A RFC 10 (107.3 ng/µl)</P></TD><TD><P>o14sd_027</P></TD><TD><P>014sd_028</P></TD><TD><P>Pfu</P></TD><TD><P>P2A_NgoMIV_out</P></TD><TD><P>2.2 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P> </P><P>Pfu protocol:     basically just a normal PCR program:      1x           95 °C      30 seconds</P><P>                                                                                                              15x:       95 °C      30 seconds</P><P>                                                                                                                             55 °C      1 minute</P><P>                                                                                                                             68 °C      2 minutes/kb</P><P>                                                                                                              1x:          68 °C      10 minutes</P><P>Afterwards, PCRs weren’t digested by DpnI, but 5 µl were taken for transformation on either Ampicillin or Chloramphenicol agar plates.</P><H3>2014.09.14</H3><P>3 transformations worked (PCR 40, 42, 44). Colonies of PCR 42 were small, so they were left an additional day in the incubator.</P><H3>2014.09.15</H3><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>46</P></TD><TD><P>GAL4</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Q5</P></TD><TD><P>GAL4 RFC 25</P></TD><TD><P>0.5 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>PCR was made in 50 µl and in triplicates.</P><H3>2014.09.16</H3><P>Triplicates of PCR 46 were digested in CutSmart by EcoRI-HF/PstI-HF for ligation into pSB1C3. Ligation of PCR 46 was made as last time reported.</P><P>ONC were made from PCR 40 (2x), 42 (6x), 44 (2x).</P><H3>2014.09.17</H3><H4>DNA extraction of ONCs.</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>ePDZb_Eco_out I</P></TD><TD><P>135.6</P></TD><TD><P>ePDZb_Eco_out VI</P></TD><TD><P>313.9</P></TD></TR><TR><TD><P>ePDZb_Eco_out II</P></TD><TD><P>185.2</P></TD><TD><P>SEAP_NgoMIV_out I</P></TD><TD><P>151.0</P></TD></TR><TR><TD><P>ePDZb_Eco_out III</P></TD><TD><P>62.0</P></TD><TD><P>SEAP_NgoMIV_out II</P></TD><TD><P>200.0</P></TD></TR><TR><TD><P>ePDZb_Eco_out IV</P></TD><TD><P>117.0</P></TD><TD><P>WPRE_NgoMIV_out I</P></TD><TD><P>84.0</P></TD></TR><TR><TD><P>ePDZb_Eco_out V</P></TD><TD><P>63.9</P></TD><TD><P>WPRE_NgoMIV_out II</P></TD><TD><P>115.8</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digests</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>WPRE_NgoMIV_out</P></TD><TD><P>XhoI/NgoMIV</P></TD><TD><P>CutSmart</P></TD><TD><P>????</P></TD></TR><TR><TD><P>SEAP_NgoMIV_out</P></TD><TD><P>NgoMIV_BamHI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>Mutated: 3 kb, 1.7 kb</P><P>Not mutated: 3 kb, 0,9 kb, 0.8 kb</P></TD></TR><TR><TD><P>ePDZb</P></TD><TD><P>EcoRI-HF/XbaI/NheI</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 2.4 kb, 1.4 kb</P><P>not mutated: 2.4 kb, 0.9 kb. 0.5 kb</P></TD></TR></TBODY></TABLE><P> </P><P>Ligation of GAL 4 RFC 25 didn’t work.</P><H3>2014.09.18</H3><P>Every test digest failed.</P><P>Ligation of GAL4 RFC 25 repeated, but with Quick ligation buffer.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Plasmid</P></TD><TD><P>backbone</P></TD><TD><P>GAL4 RFC 25</P></TD></TR><TR><TD><P>length (bp)</P></TD><TD><P>2070</P></TD><TD><P>441</P></TD></TR><TR><TD><P>concentration [ng/µl]</P></TD><TD><P>64.1</P></TD><TD><P>87.9</P></TD></TR><TR><TD><P>volume [µl]</P></TD><TD><P>0.78</P></TD><TD><P>0.36</P></TD></TR><TR><TD><P>Water</P></TD><TD><P> </P></TD><TD><P>1.20</P></TD></TR><TR><TD colspan="3" width="382"><P>plus 2.6 µl 1x T4 buffer and 0.2 µl T4 ligase (400,000 U/ml)</P></TD></TR></TBODY></TABLE><P>We used for a 2<SUP>nd</SUP> approach instead of molare ratio of 1:3 just thrice volume of insert (0.73 µl pSB1C3 + 1.47 µl GAL 4 RFC 25).</P><P>Ligations were incubated at room temperature for 15 minutes. 5 µl were taken for transformation and afterwards plates were incubated at 37 °C.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>47</P></TD><TD><P>pKM292 re-transformation</P></TD><TD><P>o14sd_011</P></TD><TD><P>014sd_012</P></TD><TD><P>Q5</P></TD><TD><P>GAL 4 RFC 25</P></TD><TD><P>0.6 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>48</P></TD><TD><P>p14zl_003</P></TD><TD><P>o14sd_029</P></TD><TD><P>014sd_030</P></TD><TD><P>Q5</P></TD><TD><P>Puromycin acetyl transferase</P></TD><TD><P>3.6 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>49</P></TD><TD><P>WPRE RFC 10 seq.</P></TD><TD><P>o14sd_059</P></TD><TD><P>014sd_060</P></TD><TD><P>Q5</P></TD><TD><P>WPRE RFC 10 with RFC 25 overhangs</P></TD><TD><P>0.6 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>Each PCR was made in 50 µl and triplicates.</P><P>Gel electrophoresis showed fine results. Lanes were cut out and ligations into pSB1C3 were made at 21.09.</P><H3>2014.09.21</H3><P> </P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>50</P></TD><TD><P>CAT-1 mutated</P></TD><TD><P>o14sd_041</P></TD><TD><P>014sd_042</P></TD><TD><P>Pfu</P></TD><TD><P>CAT-1 RFC 10 with RFC 25 overhangs</P></TD><TD><P>1.9 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>51</P></TD><TD><P>SEAP_PstI_out</P></TD><TD><P>o14sd_051</P></TD><TD><P>014sd_052</P></TD><TD><P>Pfu</P></TD><TD><P>SEAP RFC 10 with RFC 25 overhangs</P></TD><TD><P>1.6 kb</P></TD><TD><P>50 °C</P></TD></TR></TBODY></TABLE><P>Each PCR was made in 50 µl and triplicates.</P><P>ONC were made from ligations.</P><P> </P><H3>2014.09.22</H3><P> </P><P>No PCR product visible. PCR 50 &amp; 51 will be repeated with Phusion instead of Pfu with a common annealing temperature of 50 °C.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>polymerase</P></TD><TD><P>Product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>54</P></TD><TD><P>PCR 1.1 II</P></TD><TD><P>o14sd_005</P></TD><TD><P>014sd_006</P></TD><TD><P>Phusion</P></TD><TD><P>ePDZb_EcoRI_out</P></TD><TD><P>1.9 kb</P></TD><TD><P>55 °C</P></TD></TR></TBODY></TABLE><P>PCR was digested by DpnI and afterwards 3 µl were taken for transformation. The bacteria culture was streaked out onto a Ampicilin plate.</P><P> </P><H4>DNA extraction of ONC</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>WPRE RFC 10 (PCR 49) I</P></TD><TD><P>105.0</P></TD><TD><P>Puromycin acetyltransferase RFC 25 I</P></TD><TD><P>131.4</P></TD></TR><TR><TD><P>WPRE RFC 10 (PCR 49) II</P></TD><TD><P>146.0</P></TD><TD><P>Puromycin acetyltransferase RFC 25 II</P></TD><TD><P>161.2</P></TD></TR><TR><TD><P>WPRE RFC 10 (PCR 49) III</P></TD><TD><P>131.0</P></TD><TD><P>SEAP_NgoMIV_out II Puromycin acetyltransferase RFC 25 III</P></TD><TD><P>70.0</P></TD></TR><TR><TD><P>WPRE RFC 10 (PCR 49) IV</P></TD><TD><P>267.0</P></TD><TD><P>Puromycin acetyltransferase RFC 25 IV</P></TD><TD><P>124.0</P></TD></TR><TR><TD><P>WPRE RFC 10 (PCR 49) V</P></TD><TD><P>140.0</P></TD><TD><P>Puromycin acetyltransferase RFC 25 V</P></TD><TD><P>230.0</P></TD></TR><TR><TD><P>GAL 4 RFC 25 I</P></TD><TD><P>156.1</P></TD></TR><TR><TD><P>GAL 4 RFC 25 II</P></TD><TD><P>247.8</P></TD></TR><TR><TD><P>GAL 4 RFC 25 III</P></TD><TD><P>134.4</P></TD></TR><TR><TD><P>GAL 4 RFC 25 IV</P></TD><TD><P>135.7</P></TD></TR><TR><TD><P>GAL 4 RFC 25 V</P></TD><TD><P>130.4</P></TD></TR></TBODY></TABLE><H4>Test digsts of ONC</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>WPRE RFC 25</P></TD><TD><P>NcoI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>1.5 kb, 1.2 kb</P></TD></TR><TR><TD><P>GAL 4 RFC 25</P></TD><TD><P>XhoI</P></TD><TD><P>CutSmart</P></TD><TD><P>1.2 kb, 0.9 kb, 0.3 kb</P></TD></TR><TR><TD><P>Puromycin acetyltransferase</P></TD><TD><P>XhoI</P></TD><TD><P>CutSmart</P></TD><TD><P>1.8 kb, 0.9 kb</P></TD></TR></TBODY></TABLE><P> </P><P>All digests showed lanes with the expected sizes.</P><H3>2014.09.23</H3><P> </P><P>Transformation with PCR 54 worked. Three colonies were picked for ONC.</P><H4>Gel electrophoresis of PCR 50 &amp; PCR 51</H4><FIGCAPTION><P class="desc">test digest of PCR 50 &amp; 51.</P></FIGCAPTION><H4>Gel electrophoresis of PCR 50 &amp; PCR 51</H4>

img src=&quot;https://static.igem.org/mediawiki/2014/2/2a/Freiburg2014_2014-09-23_testverdauPCR50_PCR51_neu.jpg&quot; alt=&quot;Description of Image&quot;&gt;
						<FIGCAPTION><P class="desc">test digest of PCR 50 &amp; 51.</P></FIGCAPTION><H3>2014.09.24</H3><P> </P><H4>DNA extraction of ONC</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>ePDZb_Eco_out I</P></TD><TD><P>49.6</P></TD></TR><TR><TD><P>ePDZb_Eco_out II</P></TD><TD><P>67.7</P></TD></TR><TR><TD><P>ePDZb_Eco_out III</P></TD><TD><P>52.7</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digest</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>ePDZb_EcoRI_out</P></TD><TD><P>EcoRI-HF/ClaI/HindHIII-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>Mutated: 2.3 kb, 1.5 kb</P><P>Not mutated: 2.3 kb, 0.9 kb, 0.6 kb</P></TD></TR></TBODY></TABLE><P> </P><P>Result: all 3 samples showed expected DNA lanes. Colony 2 will be sequenced.</P><P>Colony 1 of WPRE RFC 25, GAL 4 RFC 25 and Puromycin acetyltransferase RFC 25 will be sequenced.</P><P> </P><H3>2014.09.25</H3><P> </P><P>All biobricks showed expected DNA sequence and ePDZb lost the EcoRI site.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>Polymerase</P></TD><TD><P>Product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>55</P></TD><TD><P>ePDZb_EcoRI_out</P></TD><TD><P>o14sd_001</P></TD><TD><P>014sd_002</P></TD><TD><P>Phusion</P></TD><TD><P>ePDZb_AgeI_out</P></TD><TD><P>3.8 kb</P></TD><TD><P>55 °C</P></TD></TR><TR><TD><P>56</P></TD><TD><P>SEAP_PstI_out (367.6 ng/µl)</P></TD><TD><P>o14sd_045</P></TD><TD><P>014sd_046</P></TD><TD><P>Phusion</P></TD><TD><P>SEAP_NgoMIV_end</P></TD><TD><P>5.8 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>57</P></TD><TD><P>SEAP_PstI_out (367.6 ng/µl)</P></TD><TD><P>o14sd_049</P></TD><TD><P>014sd_050</P></TD><TD><P>Phusion</P></TD><TD><P>SEAP RFC 10</P></TD><TD><P>1.6 kb</P></TD><TD><P>50 °C</P></TD></TR><TR><TD><P>58</P></TD><TD><P>WPRE RFC 10 (25.09.)</P></TD><TD><P>o14sd_055</P></TD><TD><P>014sd_056</P></TD><TD><P>Phusion</P></TD><TD><P>WPRE RFC 25</P></TD><TD><P>1.6 kb</P></TD><TD><P>50.5 °C</P></TD></TR></TBODY></TABLE><P> </P><P>Gel of PCR 57 showed a strong lane. The lane was cut out and prepared for digest/ligation. Digest was done with EcoRI-HF/Pst-HF in CutSmart.</P><H4>Ligation of PCR 50 and PCR 57</H4><P> </P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Plasmid</P></TD><TD><P>backbone</P></TD><TD><P>CAT-1 RFC 25 overhangs</P></TD><TD><P>SEAP RFC 10</P></TD></TR><TR><TD><P>length (bp)</P></TD><TD><P>2070</P></TD><TD><P>1945</P></TD><TD><P>1577</P></TD></TR><TR><TD><P>concentration [ng/µl]</P></TD><TD><P>64.1</P></TD><TD><P>68.7</P></TD><TD><P>80.4</P></TD></TR><TR><TD><P>volume [µl]</P></TD><TD><P>0.38</P></TD><TD><P>1.84</P></TD><TD><P>1.71</P></TD></TR><TR><TD><P>Water</P></TD><TD><P> </P></TD><TD><P>1.20</P></TD><TD><P> </P></TD></TR><TR><TD colspan="4" width="506"><P>plus 2.6 µl 1x T4 buffer and 0.2 µl T4 ligase (400,000 U/ml)</P></TD></TR></TBODY></TABLE><P>Additional approach: just use a triple volume of insert each (0.6 µl backbone, 1.8 µl insert).</P><H3>2014.09.26</H3><P>Transformation of PCR 58 didn’t work but PCR 55 &amp; 56 showed colonies on plates. 5 colonies were picked for ONC.</P><P>Ligations were partly successful. There for both SEAP RFC 10 and CAT-1 with RFC 25 overhangs colonies. 5 of them were picked for ONC each.</P><H3>2014.09.27</H3><P> </P><H4>DNA extraction from OVC</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>PCR 56 I</P></TD><TD><P>160.4</P></TD><TD><P>PCR 55 I</P></TD><TD><P>81.1</P></TD></TR><TR><TD><P>PCR 56 II</P></TD><TD><P>122.9</P></TD><TD><P>PCR 55 II</P></TD><TD><P>71.4</P></TD></TR><TR><TD><P>PCR 56 III</P></TD><TD><P>109.9</P></TD><TD><P>PCR 55 III</P></TD><TD><P>46.8</P></TD></TR><TR><TD><P>PCR 56 IV</P></TD><TD><P>105.2</P></TD><TD><P>PCR 55 IV</P></TD><TD><P>43.1</P></TD></TR><TR><TD><P>PCR 56 V</P></TD><TD><P>199.3</P></TD><TD><P>PCR 55 V</P></TD><TD><P>81.0</P></TD></TR><TR><TD><P>SEAP RFC 10 I</P></TD><TD><P>182.2</P></TD><TD><P>PCR 50 I</P></TD><TD><P>130.2</P></TD></TR><TR><TD><P>SEAP RFC 10 II</P></TD><TD><P>100.9</P></TD><TD><P>PCR 50 II</P></TD><TD><P>120.2</P></TD></TR><TR><TD><P>SEAP RFC 10 III</P></TD><TD><P>122.7</P></TD><TD><P>PCR 50 III</P></TD><TD><P>17.2</P></TD></TR><TR><TD><P>SEAP RFC 10 IV</P></TD><TD><P>239.0</P></TD><TD><P>PCR 50 IV</P></TD><TD><P>128.2</P></TD></TR><TR><TD><P>SEAP RF 10 V</P></TD><TD><P>201.6</P></TD><TD><P>PCR 50 V</P></TD><TD><P>170.3</P></TD></TR></TBODY></TABLE><P> </P><H4>Test digests</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>PCR 50</P></TD><TD><P>EcoRV-HF/KpnI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>2.2 kb, 1.3 kb</P></TD></TR><TR><TD><P>PCR 55</P></TD><TD><P>AgeI-HF/XbaI/NheI</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 2.4 kb, 1.4 kb</P><P>not mutated: 2.4 kb, 0.9 kb, 0.5 kb</P></TD></TR><TR><TD><P>PCR 56</P></TD><TD><P>NgoMIV/BamHI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>mutated: 3.0 kb, 1.7 kb</P><P>not mutated: 3.0 kb, 1 kb, 0.7 kb</P></TD></TR><TR><TD><P>PCR 57</P></TD><TD><P>NcoI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>1.8 kb, 1 kb, 0.7 kb</P></TD></TR></TBODY></TABLE><P> </P><P>Gel electrophoresis</P><FIGCAPTION><P class="desc">test digest of PCR 50, 55-57.</P></FIGCAPTION><P>PCR 50, 56 and 57 failed. PCR 55 looked like the plasmid had been cut once. Sequence analysis showed that XbaI site was overlapped with a methylation site, thus restriction site was blocked and couldn’t recognized anymore. Therefore PCR 55 I would be sequenced.</P><H3>2014.09.28</H3><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>Polymerase</P></TD><TD><P>Product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>59</P></TD><TD><P>WPRE RFC 10 -NgoMIV</P></TD><TD><P>o14sd_055</P></TD><TD><P>014sd_056</P></TD><TD><P>Phusion</P></TD><TD><P>WPRE RFC 25</P></TD><TD><P>0.6 kb</P></TD><TD><P>57 °C</P></TD></TR><TR><TD><P>60</P></TD><TD><P>CAT-1 PstI out</P></TD><TD><P>o14sd_039</P></TD><TD><P>014sd_040</P></TD><TD><P>Phusion</P></TD><TD><P>CAT-1 RFC 10</P></TD><TD><P>1.9 kb</P></TD><TD><P>56 °C</P></TD></TR></TBODY></TABLE><P>PCR 59 was done in 50 µl approaches and in triplicates and was separated in a 2 % gel.</P><H4>Gel electrophoresis</H4><P> </P><FIGCAPTION><P class="desc">test digest of WPRE RFC 25 ligation into pSB1C3.</P></FIGCAPTION><P>Results looked fine. The PCR product was cut out with EcoRI-HF/PstI-HF and used for ligation into pSB1C3.</P><FIGCAPTION><P class="desc">test digest of PCR 60.</P></FIGCAPTION><P>Results looked fine. The PCR product was cut out with EcoRI-HF/PstI-HF and used for ligation into pSB1C3.</P><P> </P><H4>Gel extraction</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>WPRE RFC 25</P></TD><TD><P>76.2</P></TD></TR><TR><TD><P>CAT-1 RFC 10</P></TD><TD><P>289.6</P></TD></TR><TR><TD><P>SEAP RFC 10</P></TD><TD><P>like PCR 57</P></TD></TR></TBODY></TABLE><P> </P><H3>2014.09.30</H3><P>Sequencing results confirmed that ePDZb is AgeI-free.</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>PCR #</P></TD><TD><P>Template</P></TD><TD><P>primer 1</P></TD><TD><P>primer 2</P></TD><TD><P>Polymerase</P></TD><TD><P>Product</P></TD><TD><P>size</P></TD><TD><P>annealing temperature</P></TD></TR><TR><TD><P>61</P></TD><TD><P>PCR 55 I</P></TD><TD><P>o14sd_007</P></TD><TD><P>014sd_008</P></TD><TD><P>Phusion</P></TD><TD><P>ePDZb RFC 10</P></TD><TD><P>0.6 kb</P></TD><TD><P>56.5°C</P></TD></TR><TR><TD><P>62</P></TD><TD><P>SEAP PstI out</P></TD><TD><P>o14sd_045</P></TD><TD><P>014sd_046</P></TD><TD><P>Phusion</P></TD><TD><P>SEAP_C2784G_NgoMIV</P></TD><TD><P>5.4 kb</P></TD><TD><P>56 °C</P></TD></TR></TBODY></TABLE><P>PCR Product was purified by gel extraction and cut with EcoRI-HF/PstI-HF and used for ligation into pSB1C3.</P><P>PCR 62 was digested by DpnI and transformed on agar plate containing Ampicillin.</P><H4>ligation</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P> </P></TD><TD><P>WPRE RFC 25 (09/29)</P></TD><TD><P>CAT-1 RFC 10 (09/29)</P></TD><TD><P>SEAP RFC 10 (09/25)</P></TD></TR><TR><TD><P>pSB1C3</P></TD><TD><P>1.46 µl</P></TD><TD><P>1 µl</P></TD><TD><P>1 µl</P></TD></TR><TR><TD><P>insert</P></TD><TD><P>0.74 µl</P></TD><TD><P>0.4 µl</P></TD><TD><P>0.4 µl</P></TD></TR><TR><TD><P>water</P></TD><TD><P>/</P></TD><TD><P>0.8</P></TD><TD><P>/</P></TD></TR><TR><TD colspan="4" width="614"><P>plus 2.6 µl Quick ligation buffer, 0.2 µl T4 ligase (400,000 U/ml)</P></TD></TR></TBODY></TABLE><P> </P><H3>2014.10.01</H3><P>All transformed plates had colonies.</P><H4>DNA extraction</H4><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>WPRE RFC 25 I</P></TD><TD><P>130.0</P></TD></TR><TR><TD><P>WPRE RFC 25 II</P></TD><TD><P>109.0</P></TD></TR><TR><TD><P>WPRE RFC 25 III</P></TD><TD><P>115.0</P></TD></TR><TR><TD><P>WPRE RFC 25 IV</P></TD><TD><P>140.0</P></TD></TR><TR><TD><P>PCR 62 I</P></TD><TD><P>448.0</P></TD></TR><TR><TD><P>PCR 62 II</P></TD><TD><P>458.0</P></TD></TR><TR><TD><P>PCR 62 III</P></TD><TD><P>490.0</P></TD></TR><TR><TD><P>PCR 62 IV</P></TD><TD><P>413.0</P></TD></TR></TBODY></TABLE><P> </P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD><TD><P>Template</P></TD><TD><P>concentration [ng/µl]</P></TD></TR><TR><TD><P>SEAP RFC 10 I</P></TD><TD><P>216.9</P></TD><TD><P>CAT-1 RFC 10 I</P></TD><TD><P>265.9</P></TD></TR><TR><TD><P>SEAP RFC 10 II</P></TD><TD><P>192.9</P></TD><TD><P>CAT-1 RFC 10 II</P></TD><TD><P>396.3</P></TD></TR><TR><TD><P>SEAP RFC 10 III</P></TD><TD><P>193.3</P></TD><TD><P>CAT-1 RFC 10 III</P></TD><TD><P>223.2</P></TD></TR><TR><TD><P>SEAP RFC 10 IV</P></TD><TD><P>176.5</P></TD><TD><P>CAT-1 RFC 10 IV</P></TD><TD><P>272.2</P></TD></TR><TR><TD><P>SEAP RFC 10 V</P></TD><TD><P>167.5</P></TD><TD><P>CAT-1 RFC 10 V</P></TD><TD><P>209.0</P></TD></TR><TR><TD><P>SEAP RFC 10 VI</P></TD><TD><P>193.4</P></TD><TD><P>CAT-1 RFC 10 VI</P></TD><TD><P>188.6</P></TD></TR><TR><TD><P>ePDZb RFC 10 I</P></TD><TD><P>125.1</P></TD></TR><TR><TD><P>ePDZb RFC 10 II</P></TD><TD><P>144.4</P></TD></TR><TR><TD><P>ePDZb RFC 10 III</P></TD><TD><P>117.5</P></TD></TR><TR><TD><P>ePDZb RFC 10 IV</P></TD><TD><P>125.0</P></TD></TR><TR><TD><P>ePDZb RFC 10 V</P></TD><TD><P>118.8</P></TD></TR><TR><TD><P>ePDZb RFC 10 VI</P></TD><TD><P>109.5</P></TD></TR></TBODY></TABLE><P>Test digests</P><TABLE border="„1“" cellspacing="„0“" cellpadding="„0“"><TBODY><TR><TD><P>Template</P></TD><TD><P>enzyme</P></TD><TD><P>buffer</P></TD><TD><P>fragment size</P></TD></TR><TR><TD><P>PCR 59</P></TD><TD><P>EcoRV-HF/NgoMIV</P></TD><TD><P>CutSmart</P></TD><TD><P>1.7 kb, 0.9 kb</P></TD></TR><TR><TD><P>PCR 60</P></TD><TD><P>EcoRV-HF/KpnI-HF</P></TD><TD><P>CutSmart</P></TD><TD><P>2.2 kb, 1.4 kb, 0.4 kb</P></TD></TR><TR><TD><P>PCR 61</P></TD><TD><P>XhoI</P></TD><TD><P>CutSmart</P></TD><TD><P>1.8 kb, 0.9 kb</P></TD></TR><TR><TD><P>PCR 62</P></TD><TD><P>BamHI-HF/NgoMIV</P></TD><TD><P>CutSmart</P></TD><TD><P>2.3 kb, 1 kb, 0.3 kb</P></TD></TR></TBODY></TABLE><H4>Gel electrophoresis</H4><FIGCAPTION><P class="desc">test digest of PCR 62 and ligation of WPRE RFC 25 into pSB1C3.</P></FIGCAPTION><P>All results looked fine. It seemed that SEAP contained a NgoMIV site less and the WPRE digests proved that WPRE RFC 25 is in pSB1C3.</P><FIGCAPTION><P class="desc">test digest of following ligations: ePDZb RFC 10, SEAP RFC 10, CAT-1 RFC 10.</P></FIGCAPTION><P>Results looked fine. Every sample showed expected lane size.</P><P>Colony one of each plasmid was sent to sequencing.</P><H2 id="Standardization-October">Standardization - October</H2><H3>2014.10.02</H3><P>All four ligations contain expected DNA sequence.</P><P> </P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Freiburg/Content/Notebook/Cloning">http://2014.igem.org/Team:Freiburg/Content/Notebook/Cloning</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Freiburg/Content/Notebook/Cloning" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Freiburg/Content/Notebook/Cloning" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Cloning&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Freiburg/Content/Notebook/Cloning&amp;oldid=394596" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>