"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Notebook/Cloning
From 2014.igem.org
The AcCELLerator
Home
Project
Overview
The Light System
The Receptor
The Viral Vector
Mammalian Techniques for iGEM
Outlook
Results
Summary
The Viral Vector
The Receptor
The Light System
The Combination
BioBricks
Modeling
Light Boxes
Policy and Safety
Link-It
Saturday Balloon Fever
Online Survey
Ethical Essay
Interview with Dr. Boldt
Symposium of Biosafety
Safety Introduction
Viral Vector Safety
Safety-Sheet
BSL 2 Regulation
Notebook
Labjournal
Cloning
Methods
Team
Members
Collaborations
Judging
FUN-Part
Attributions
Acknowledgement
Sponsors
The AcCELLerator
Cloning
The protocols for our methods can be found here .
During our experiments, we conducted a lot of PCRs with different Primers. In order to stay on top of things, we numbered Primers, PCRs and constructs. You can download both lists with detailed information about Primer composition and about PCRs in the following documents:
p14rz_002
PCR: o14rz_005, o14_rz006; Template pQXCIN-SLC7A1
Digest: pKM006, EcoRI + NotI; PCR-product
Testdigest with PstI in Cutsmart, expected bands: 5,9; 0,7
PCR
Testdigest
p14rz_003
PCR 3
Digest 3, 4
Testdigest with PstI + SpeI in Cutsmart, expected bands: 4,1;2,1;1,8
PCR
Digest
Testdigest
p14rz_004
PCR 21,22
Digest -
Testdigest with PstI in Cutsmart, expected bands: 4,1;3,2
PCR
PCR
Testdigest
p14rz_005
PCR 4,5
Digest 5
Testdigest with AgeI+HindIII in Cutsmart, expected bands: 6,6;1,3
PCR
Digest
Testdigest
p14rz_006
PCR 17,18
Digest 5
Testdigest with AgeI+HindIII in Cutsmart, expected bands: 6,6;1,4
PCR
Digest
Testdigest
p14rz_007
PCR 37,38
Digest 21
Testdigest with PstI in Cutsmart, expected bands: 4,1;2,2;1,7
PCR
Digest
Testdigest
p14rz_008
PCR blue light receptor; Template: pQXCIN-SLC7A1
Digest pKM006 EcoRI, NotI
Testdigest with SmaI in Cutsmart, expected bands: 5,7
PCR
Testdigest
p14rz_009
PCR 39,40
Digest 5
Testdigest with BamHI in Cutsmart, expected bands: 3,0;2,7
Digest
Testdigest
p14rz_010
PCR 23,24
Digest 12
Testdigest with PstI in Cutsmart, expected bands:3,1;2,3;0,56
PCR
PCR
Testdigest
p14ls_003
PCR 7,9,10
Digest -
Testdigest with KpnI in Cutsmart, expected bands: 4,9; 1,3; 0,4
PCR
Digest
Testdigest
p14vv_001
PCR Primers vv003,vv004; Template: pMIG
Digest pMIG EcoRI,NotI; C1-eGFP EcoRI,NotI
Testdigest with NgoMIV in Cutsmart, expected bands: 5,7; 0,6
Digest
Digest
Testdigest
p14vv_002
PCR -
Digest 8,11
Testdigest with PstI in Cutsmart, expected bands: 3,2;2,1;1,4
PCR
Digest
Testdigest
p14vv_003
PCR pMIG with primer vv003+vv004-
Digest xx
Testdigest with KpnI in Cutsmart, expected bands: 3,5;1,5
PCR
Testdigest
p14vv_006
PCR 26
Digest 13
Testdigest with NgoMIV in Cutsmart, expected bands: 3,5;1,9;0,95,0,6
PCR
Testdigest
p14vv_007
PCR 27
Digest 14
Testdigest with NgoMIV in Cutsmart, expected bands: 3,5; 1,3; 0,97; 0,66
PCR
Testdigest
p14vv_008
PCR 43
Digest 22
Testdigest with EcoRV in Cutsmart, expected bands: 3,4; 1,4 0,75
PCR
Digest
Testdigest
p14vv_009
PCR 44
Digest 22
Testdigest with EcoRV+MluI in Cutsmart, expected bands: 3,4; 1,43; 0,75
PCR
Digest
Testdigest
p14vv_010
PCR 45
Digest 22
Testdigest with EcoRV in Cutsmart, expected bands: 3,4; 2,2
PCR
Digest
Testdigest
p14vv_012
PCR 46
Digest 22
Testdigest with EcoRV in Cutsmart, expected bands: 4,7;3,4
PCR
Digest
Testdigest
p14vv_013
PCR 47,48,49
Digest 22
Testdigest with NgoMIV in Cutsmart, expected bands: 3,4;2,0;1,3;0,2
PCR
Digest
Testdigest
p14vv_014
PCR 50,51
Digest 22
Testdigest with NgoMIV in Cutsmart, expected bands: 4,1; 1,56; 0,66
PCR
Digest
Testdigest
p14zl_010
PCR 1,2
Digest 1
Testdigest with NgOMIV in Cutsmart, expected bands: 4,7;2,4;1,1
PCR
Digest
Testdigest
p14zl_011
PCR 19,20
Digest 1
Testdigest with NgOMIV in Cutsmart, expected bands: 4,7;2,0;1,3;0,43
PCR
Digest
Testdigest
p14zl_016
PCR 52,53
Digest 22
Testdigest with HindIII in Cutsmart, expected bands: 5,7;2,3;1,7
PCR
Digest
Testdigest
p14zl_017
PCR 64,65
Digest 24
testdigest with NgoMIV in Cutsmart, expected bands: 5,4;2,9
PCR_digest
Testdigest
p14zl_020
PCR 59,60
Digest 24
testdigest with BamHI-HF, NgoMIVin Cutsmart, expected bands: 4,7; 2,4; 1,8
PCR
Digest
Testdigest
Standardization
Standardization - August
PCR mix for site-directed mutagenisis (25 µl)
template
1 µl
buffer (5x)
5 µl
Primer
1 µl each
polymerase
0.5 µL
dNTPs
1 µl
DMSO
0.5 µl
water
15 µl
start your PCR mix with just one primer and let reaction run for ten cycles including end elongation. shortly after that add 1 µl dNTP and 1 µl of primer 2. start second PCR under same conditions for 15 cycles. Don't spend too much time with adding dNTPs and primer 2.
DNA amplifications such for ligation PCR approaches contained 50 µl.
protocoll site-directed mutagenisis
98 °C
5 min
denaturation
98 °C
30 s
denaturation
60 °C
30 s
annealing
72 °C
30 s/kb
elongation
72 °C
10 min
end elongation
To avoid high background activities by not-mutated plasmids, the PCR mixture was digested by DpnI. It cuts methylated DNA only, leading to remove the DNA template.
Therefore transformation is done with unmethylated PCR product, where plasmids contain the desired mutation.
DpnI digest
add 3 µl Cutsmart, 0.5 µl water and 0.5 µl DpnI. Incubate at 37 °C for 90 minutes, followed by an inactivation step at 80 °C for 20 minutes.
test digest
2 µl buffer
about 500 - 1000 ng DNA
0.5 µl enzyme
up to 20 µl water
PCR products or test digests of plasmids where separated gelelectrophoresis, containing 1 % agarose. Small gels were run at 95 V for 90 minutes, large gels at 105 V for 90 minutes.
2014.08.21.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
elongation time
1
pKM297
o14_sd_003
o14_sd_004
Q5
ePDZb_G1448C_PstI
3.8 kb
60 °C
01:55 min
2
pKM292
o14_sd_011
o14_sd_012
Q5
GAL4_binding_domain_RFC_25
0.5 kb
60 °C
00:16 min
3
pKM292
o14_sd_019
o14_sd_020
Q5
LOV_C3423T_PstI
3.7 kb
60
01:53 min
4
zl_003
o14_sd_023
o14_sd_024
Q5
P2A_RFC_10
0.1 kb
60
00:04 min
5
zl_003
o14_sd_027
o14_sd_028
Q5
P2A_C9374G_NgoMIV
11.7 kb
60
05:50 min
Each PCR was done in triplicates and digested with DpnI. Each replicate was used for transformation with competent bacteria on a seperated Ampicillin plate and incubated at 37 °C. bacteria containing zl_003 were incubated at 32 °C.
2014.08.22.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
elongation time
6
zl_oo3
o14_sd_029
o14_sd_030
Phusion
puromycin_resistance_gene_RFC_25
0.6 kb
60 °C
00:20 min
7
rz_008
o14_sd_008
o14_sd_009
Phusion
CAT-1_C1168T_PstI
6.4 kb
60 °C
03:15 min
8
pKM084
o14_sd_047
o14_sd_048
Phusion
SEAP_G1419C_PstI
5.4 kb
60
02:45 min
All PCRs were digested with DpnI and transformed on Ampicillin containing plates at 37 °C.
Transformation from PCR 1.1/1.2/1.3, 3.1/3.2/3.3, 5.1/5.3 were successful, but entirely PCR 2 & 4 not (repeated with inkubation at 32 °C). Three colonies were picked from each plate and used for over night culures (ONC).
Advice from instructor: an elongation time of 30 s for each DNA fragment smaller than 1 kb will be used.
2014.08.23.
A mini prep kit from Quiagen (250 reactions) were used for DNA isolation:
template
concentration [ng/µl]
template
concentration [ng/µl]
1.1 I
155.2 ng/µl
1.2 I
174.0 ng/µl
1.1 II
153.1 ng/µl
1.2 II
727.5 ng/µl
1.1 III
566.6 ng/µl
1.2 III
562.5 ng/µl
1.3 I
265.8 ng/µl
3.1 I
174.1 ng/µl
1.3 II
226.4 ng/µl
3.1 II
144.5 ng/µl
1.3 III
187.7 ng/µl
3.1 III
144.4 ng/µl
3.2 I
197.0 ng/µl
3.3 I
163.1 ng/µl
3.3 II
163.5 ng/µl
5.1 I
150.0 ng/µl
5.3 I
448.7 ng/µl
5.1 II
442.2 ng/µl
5.3 II
230.1 ng/µl
5.1 III
295.7 ng/µl
5.3 III
559.4 ng/µl
test digest with:
PCR
enzyme
buffer
fragment size
1
PstI/ClaI
CutSmart
mutated: 2.4 kb, 1.5 kb, not mutated: 2.4 kb, 0.9 kb, 0.6 kb
3
PstI/BbsI
CutSmart
mutated: 2.4 kb, 0.65 kb, 0.53 kb; not mutated: 2.4 kb, 0.53 kb, .04 kb, .026 kb, 0.23  kb, 0.06 kb
5
AgeI/NgoMIV
NEB 1.1
mutated: 5.1 kb, 4.7 kb, 1.8 kb; not mutated: 4.7 kb, 4.1 kb, 2.8 kb
0.625 µl BbsI was added, due to its decreased activity (75%) in NEB 1.1. Test digests were done over night.
Transformations of PCR 8 and PCR 5 didn't work. Repeated in duplicates: SEAP was incubated at 37 °C, P2A at 32 °C.
5 ONC of PCR 7 (CAT-1) were done.
2014.08.24.
results of gel electrophoresis:
gel electrophoresis of PCR 1 & 5.
PCR 1:         PCR 1.1 I and PCR 1.1 II showed perfect fragment sizes (2.3 kb & 1.4 kb). àCandidates for sequencing.
PCR 5:         PCR 5.1 & PCR 5.3 were successful. DNA fragments at 5.1 kb and 4.7 kb. Fragment at 2 kb invisible, but if DNA wasn’t mutated, there would not be a fragment at 5.1 kb. à Candidates for sequencing.
gel electrophoresis of PCR 3.
PCR 3:         PCR 3.1 III looked fine. Fragments as expected.à Candidate for sequencing.
Despite missing sequencing results, PCR 5.1 I was used for a transformation.
DNA extraction of CAT-1 (5x)
PCR #
Concentration [ng/µl]
7 I
489.2
7 II
316.5
7 III
385.0
7 IV
451.8
7 V
550.0
 
Test digest with:
PCR #
enzyme
buffer
fragment size
7
AgeI/PstI-HF
CutSmart
mutated:4.2 kb, 1.6 kb
not mutated: 4.2 kb, 1.3 kb, 0.3 kb
test digest was done overnight.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
10
pKM292
o14sd_011
014sd_012
Phusion
GAL4_RFC_25
0.5 kb
55 °C
11
zl_003
o14sd_29
014sd_030
Phusion
Puromycin_resistance_gene_RFC_25
0.6 kb
55 °C
 
2014.08.25
 
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
12
pKM084
o14sd_047
014sd_048
Phusion
SEAP_G1419C_PstI
5.4 kb
55 °C
13
zl_003
o14sd_055
014sd_056
Phusion
WPRE_C530A_NgoMIV
11.7 kb
55 °C
14
LOV PCR Product
o14sd_011
014sd_012
Phusion
GAL4_ RFC_25
0.5 kb
55 °C
 
Result gel electrophoresis (digest CAT-1)
gel electrophoresis of PCR 7.
Result of colony 1 looked fine. à Candidate for sequencing.
Transformation of PCR 6 & 8 did not work.
2014.08.26
DNA extraction from PCR 5.1 (5x)
PCR #
Concentration [ng/µl]
5.1 I
213.0
5.1 II
283.2
5.1 III
254.1
5.1 IV
198.4
5.1 V
232.2
 
Test digests according to 2014.08.23 overnight.
2014.08.27
Gel electrophoresis from test digest PCR 5.1
gel electrophoresis of PCR 5.1.
Test digests looked fine. Fragments as expected at 5.1 kb, 4.7 kb and about 2.0 kb. PCR 5.1 I was used as a template to amplify P2A with RFC 25 prefix and suffix.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
15
PCR 5.1 I
o14sd_025
014sd_026
Phusion
P2A_RFC_25
0.1 kb
55 °C
PCR 15 was done in 50 µl approach and triplicates.
Gel electrophoresis from PCR 15
gel electrophoresis of PCR 15.
Gel contains 2 % agarose. Marker is 100 bp from Promega. Fragments lay between 100 and 200 bp. Theoretical size 136 bp. àFragments cut out and prepared for gel extraction. All three fragments were run over one column.
PCR 14 repeated as PCR 16 due to contaminated water.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
16
LOV PCR Product
o14sd_011
014sd_012
Phusion
GAL4_ RFC_25
0.5 kb
55 °C
 
Gel electrophoresis from PCR 16
gel electrophoresis of PCR 16.
Gel contains 2 % agarose. Marker is 100 bp from Promega. Fragments lay at 500 bp, expected size: 517 bp. Fragments were cut out and prepared for gel extraction. All three fragments were run over one column.
Gel extraction from P2A RFC 25 and GAL4 RFC 25
name
concentration [ng/µl]
GAL4 RFC 25
38.5
P2A RFC 25
34.4
 
Ligation of P2A RFC 25 and GAL4 RFC 25 in pSB1C3
Preparative digest according to test digest protocol, but incubated for four hours. Inserts and backbone were cut with EcoRI-HF/PstI-HF in CutSmart.
Used ratio backbone:insert = 1:3
Used volumes of insert and backbone
GAL4 RFC 25
1.72 µl
pSB1C3
0.48 µl
P2A RFC 25
1.7 µl
pSB1C3
0.5 µl
control (water)
1.7 µl
pSB1C3
0.5 µl
Ligation was performed with T4 ligase (40.000 U/ml) from NEB
After transformed with DNA bacteria culture were incubated at 37 °C for one hour. Afterwards cultures were streaked out onto agar plates containing Chloramphenicol.
Despite missing sequencing results, ONC were made from PCR 1.1 I & PCR 3.1 III (5x each).
2014.08.28
Sequencing results: P2A not mutated, CAT-1: fail. nucleotide length 1 nt., LOV: not been uploaded.
DNA extraction from ONC PCR 1.1 I (ePDZb) & PCR 3.1 III (LOV)
template
concentration [ng/µl]
template
concentration [ng/µl]
LOV I
103.7
ePDZb I
136.8
LOV II
91.4
ePDZb II
121.4
LOV III
119.4
ePDZb II
137.0
LOV IV
113.4
ePDZb IV
102.2
LOV V
129.9
ePDZb V
155.2
 
Test digests PCR 1.1 I & PCR 3.1 III:
template
enzyme
buffer
fragment size
LOV
XhoI/PstI-HF
CutSmart
mutated:2.7 kb, 0.7 kb
not mutated: 2.7 kb, 0.4 kb, 0.3 kb
ePDZb
ClaI/PstI-HF
CutSmart
mutated:2.3 kb, 1.5 kb
not mutated: 2.3 kb, 0.9 kb, 0.6 kb
 
gel electrophoresis of LOV & ePDZb.
LOV didn’t work -> no fragment at 0.7 kb. ePDZb looked fine. Visible fragment at 1.5 kb as expected.
Ligation of P2A RFC 25 and GAL4 RFC 25 didn’t work. Next time preparative digest will be desalted by PCR purification kit.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
17
zl_003
o14sd_023
014sd_024
Q5
P2A_RFC_10
0.1 kb
55 °C
18
zl_003
o14sd_057
014sd_058
Q5
WPRE_RFC_10
0.6 kb
55 °C
19
PCR 5.1 II
o14sd_039
014sd_040
Q5
CAT-1_RFC_10
1.9 kb
50 °C
20
ePDZb I
o14sd_005
014sd_006
Q5
ePDZb_A1775G_EcoRI
3.8 kb
50 °C
21
pKM084
o14sd_047
014sd_048
Q5
SEAP_G1419C_PstI
5.4 kb
50 °C
PCR 20 & 21 digested by DpnI and afterwards transformed onto Ampicillin plates and incubates at 37 °C
PCR 17 -19 were directly purified preparedly digested overnight with EcoRI-HF/PstI-HF in Cutsmart.
Apporach:          28 µl eluate
                               1 µl enzyme each
                               4 µl CutSmart
                               6 µl water
2014.08.30
ONC from GAL4 RFC 25 in pSB1C3
A Chloramphenicol plate from GAL4 RFC 25 ligation (2014.08.27.) was found in the 37 °C incubator. One colony was visible. à Three ONC were picked from this colony.
ONC from PCR 20 & 21
gel electrophoresis of PCR 17 & 18.
Plates were settled with bacteria. Three ONC were picked from each plate.
P2A looked fine. The expected size was 134 bp. Fragments were above the100 bp lane and cut out for gel extraction. WPRE showed expected lane sizes (646 bp.). Cut out for gel extraction.
gel electrophoresis of PCR 19.
Instead of 2-logfrom NEB, Promega’s 100 bp ladder was used. Expected size for CAT-1 RFC 10 were 1577 bp. The highest lane of DNA ladder is 1.5 kb -> PCR products were cut out for gel extraction.
All triplicates of one sample were run over one column.
Gel extraction from preparative digests (2014.08.29.)
name
concentration [ng/µl]
P2A RFC 10 (PCR 17)
22.9
WPRE RFC 10 (PCR 18)
53.9
CAT-1 RFC 10 (PCR 19)
17.4
 
Ligation of WPRE RFC 10, CAT-1 RFC 10, P2A RFC 10 in pSB1C3
ratio backbone:insert = 1:3
 
Volume [µl]
Volume backbone [µl]
P2A
1.68
0.52
CAT-1
1.64
0.56
WPRE
1.83
0.37
control (water)
1.72
0.48
 
We got a new ligation buffer from our instructor à Quick ligation buffer. Used buffer volume: 2.6 µl, 2.4 ligation volume and 0.2 T4 ligase (400.000 U/ml) from NEB.
5 µl ligation approaches were used for transformation. 200 µl antibiotic free LB media were added and the reaction tube was incubated at 37 °C with 400 rpm for one hour. Afterwards whole cultures were streaked out onto plates containing Chloramphenicol.
2014.08.31
DNA extraction from GAL4 RFC 25 ONC (1x), PCR 20 (3x), PCR 21 (3x)
template
concentration [ng/µl]
template
concentration [ng/µl]
GAL4 RFC 25 in pSB1C3
154.2
(PCR 21) SEAP I
188.3
(PCR 20) ePDZb I
91.4
(PCR 21) SEAP II
228.3
(PCR 20) ePDZb II
119.4
(PCR 21) SEAP III
209.6
(PCR 20) ePDZb III
113.4
 
 
Test digests
template
enzyme
buffer
fragment size
GAL4 RFC 25 in pSB1C3
EcoRI-HF/PstI-HF
CutSmart
2.0 kb, 0.5 kb
ePDZb
EcoRI-HF/PstI-HF/NotI-HF
CutSmart
mutated:1.0 kb
not mutated: 0.6 kb, 0.4 kb
SEAP
PstI-HF/BbsI
CutSmart
mutated: 4.1 kb, 1.3 kb
not mutated: 4.1 kb, 1 kb, 0.3 kb
Digests pipetted according to protocol.
gel electrophoresis of PCR 21.
Digests of GAL4 and ePDZb were run over 2 % agarose gel. Promega 100 bp was used as DNA ladder. SEAP was run over normal gel, with standard DNA ladder.
Fragment sizes weren’t as expected. Five new colonies for ONC were picked from same plate.
Standardization - September
2014.09.01
No ligation worked. They were repeated with a different protocol.
WPRE RFC 10
0.83 µl
SLC7A1 RFC 10
7.98 µl
P2A RFC 10
0.34 µl
pSB1C3
always 2.75 µl
 
Protocol:             2 µl 10x T4 buffer
                               0.2 µl T4 ligase (4e5 U/ml)
                               add water up to 20 µl
                               incubate at room temperature for 15 minutes
 
Transformed bacteria were streaked out onto plates containing Chloramphenicol and incubated at 37 °C
 
DNA extraction from SEAP ONC (5x)
template
concentration [ng/µl]
SEAP I
215.7
SEAP II
257.1
SEAP III
285.0
SEAP IV
139.9
SEAP V
367.6
Afterwards DNA was digested according to protocol.
template
enzyme
buffer
fragment size
SEAP
PstI-HF/NotI-HF
CutSmart
mutated: 3.7 kb, 1.7 kb
not mutated: 3.7 kb, 1.4 kb, 0.3 kb
 
test digest of overnight culture. Took from agar plate  containig PCR 21.
Gel looked fine. SEAP III and SEAP IV will be sequenced. DNA Mini of CAT-1 (2014.08.22.) will be sequenced too, using another sequencing primer.
2014.09.02
Sequencing results of CAT-1 showed that it didn’t contain any PstI site anymore.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
22
ePDZb_mutated
o14sd_010
014sd_020
Phusion
ePDZb_RFC_25_fwd -> AgeI site
0.5 kb
55 °C
23
ePDZb_mutated
o14sd_002
014sd_008
Phusion
ePDZb_AgeI -> ePDZb_RFC_25_rev
0.2 kb
55 °C
24
SEAP III
o14sd_049
014sd_050
Phusion
SEAP_RFC_10
1.6 kb
50 °C
25
SEAP III
o14sd_045
014sd_046
Phusion
SEAP_C2784G_NgoMIV
5.4 kb
50 °C
 
test digest of PCR 22 & 23.
PCR 22 + PCR 23 were done in duplicates, PCR 24 in triplicates. Gel electrophoresis was done in 2 % gel.
All PCR but PCR 23 didn’t show expected size. PCR 23’s DNA fragments were cut out and prepared for gel extraction.
Ligation of CAT-1 and P2A in pSB1C3 didn’t work, but plate with bacteria containing WPRE RFC 10 in pSB1C3 shows eight colonies. They were picked for ONC.
Ligation of CAT-1 and P2a were repeated with a modified yesterday’s protocol. 2 µl of T4 ligase (40,000 U/ml) instead 0.2 µl T4 ligase (400,000 U/ml) were used.
2014.09.03
 
DNA extraction of WPRE ONC (8x)
template
concentration [ng/µl]
template
concentration [ng/µl]
WPRE RFC 10 I
132.9
WPRE RFC 10 V
130.0
WPRE RFC 10 II
148.7
WPRE RFC 10 VI
106.7
WPRE RFC 10 III
91.9
WPRE RFC 10 VII
89.0
WPRE RFC 10 IV
148.8
WPRE RFC 10 VIII
106.7
 
Test digest of WPRE RFC 10
template
enzyme
buffer
fragment size
WPRE
PstI-HF/EcoRI-HF
CutSmart
mutated: 2.0 kb, 0.6 kb
 
test digest of ligated WPRE RFC 10 into pSB1C3.
Digests were run over 1 % agarose gel
All eight test digests looked fine. WPRE RFC V will be sequenced. Other ligations didn’t work perhaps of damaged DNA.
Bacteria were transformed on Ampicillin plate with PCR 25.
2014.09.04
Sequencing results showed that WPRE was correctly ligated into pSB1C3.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
26
CAT-1 mutated
o14sd_031
014sd_032
Q5
CAT-1_C1184A_NgoMIV
5.8 kb
55 °C
27
WPRE RFC 10 V
o14sd_055
014sd_056
Q5
WPRE_C530A_NgoMIV
2.5 kb
55 °C
28
LOV_mutated
o14sd_021
014sd_022
Q5
LOV_RFC_25
3.8 kb
55 °C
29
ePDZb_EcoRI_out
o14sd_021
014sd_022
Q5
ePDZb_RFC_10
3.8 kb
55 °C
30
pKM292
o14sd_011
014sd_012
Q5
GAL4 RFC 25
0.6 kb
50 °C
All PCRs were used to transform bacteria (Ampicillin, but PCR 27 Chloramphenicol) and afterwards incubated at 37 °C.
Triplicates of PCR 30 were made (50 µl) and preparative digest (EcoRI-HF/PstI-HF) was done directly in PCR tube. and afterwards purified. Concentration: 130 ng/µl.
Eight colonies were picked from plate containing PCR 15.
2014.09.05
DNA extraction of ONC PCR 15 (8x) and PCR 25 (5x)
template
concentration [ng/µl]
template
concentration [ng/µl]
P2A I
275.4
SEAP I
248.1
P2A II
192.4
SEAP II
127.1
P2A III
176.3
SEAP III
218.2
P2A IV
339.3
SEAP IV
145.9
P2A V
373.3
SEAP V
167.4
P2A VI
304.2
P2A VII
167.7
P2A VIII
305.7
 
Test digests PCR 15 and PCR 25
template
enzyme
buffer
fragment size
P2A
NgoMIV
CutSmart
mutated: 7.0 kb, 4.7 kb
not mutated: 4.7 kb, 4.1 kb, 2.8 kb
SEAP
NgoMIV
CutSmart
mutated: 5,4 kb
not mutated: 4.4 kb, 1 kb
 
test digest of PCR 15.
P2A and SEAP separated by gel electrophoresis.
test digest of PCR 25.
Test digests of SEAP didn’t work. P2A V + VIII showed fragments with correct size.
Transformations and ligations
No transformation worked.
Following transformations were made:
PCR 26 -29, on Ampicillin plates and incubated at 37 °C
Following ligations were made:
CAT-1 RFC 10, P2A RFC 10, GAL4 RFC 25 in pSB1C3.
2014.09.06
No ligation worked.
2014.09.07
 
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
31
SEAP_PstI_out
o14sd_049
014sd_050
Q5
SEAP RFC 10
1.6 kb
50 °C
32
CAT-1 mutated V
o14sd_031
014sd_032
Q5
CAT-1 RFC 10
1.9 kb
50 °C
33
P2A_mutated_V
o14sd_025
014sd_026
Q5
P2A RFC 25
0.1 kb
55 °C
34
GAL4_binding_domain
o14sd_011
014sd_012
Q5
GAL4 RFC 25
0.6 kb
55 °C
35
WPRE RFC 10
o14sd_055
014sd_056
Q5
WPRE RFC 25
0.6 kb
55 °C
PCR 31-34 were made in 50 µl, PCR 35 in 25 µl.
test digest of PCR 31 & 32.
SEAP looked fine, but CAT-1 showed unspecific PCR products.
Gel extraction of GAL4 binding domain, SEAP, P2A
template
concentration [ng/µl]
GAL 4 binding domain RFC 25
60.8
SEAP RFC 10
27.9
P2A RFC 10
11.9
 
Ligation protocol
plasmid
backbone
P2A
SEAP
GAL 4
length (bp)
2070
136
1577
507
concentration [ng/µl]
27.3
11.9
27.9
60.8
volume [µl]
1.83
0.83
4.10
0.60
water
 
14.83
11.57
15.06
plus 2 µl 1x T4 buffer and 0.5 µl T4 ligase (400,000 U/ml) each
Approaches were incubated at room temperature for 30 minutes.
Transformation: lent cells from our instructor and added 10 µl of ligation approach and 3 µl of PCR 38 respectively.
Transformations were made in duplicates.
2014.09.08
All ligations worked. Three ONC were made from GAL4 RFC 25, P2A RFC 25, SEAP RFC 10, six from PCR 25, eight from WPRE RFC 25.
PCR #
template
primer 1
primer 2
polymerase
product
size
annealing temperature
36
CAT-1_PstI_out
o14sd_039
014sd_040
Q5
CAT-1 RFC 10
1.9 kb
55 °C
37
ePDZb II (145.3 ng/µl)
o14sd_005
014sd_006
Q5
ePDZB_A1775G_EcoRI
3.8 kb
55 °C
PCR according to protocol, afterwards PCRs were digested by DpnI and transformed onto agar plates containing Ampicilin.
2014.09.09
DNA extractions failed due to old buffers. Agar plates were overgrown. Bacteria were diluted onto another plate. Transformation of PCR 36 failed. Repeated it with different primers
PCR #
Template
primer 1
primer 2
polymerase
product
size
annealing temperature
38
CAT-1_PstI_out
o14sd_039
014sd_040
Q5
CAT-1 RFC 10
1.9 kb
50 °C
39
CAT-1_PstI_out
o14sd_039
014sd_040
Q5
CAT-1 RFC 10
1.9 kb
55 °C
 
2014.09.10
test digest of PCR 38 & 39.
Amplifications of CAT-1 RFC 10 were successful. The amount of PCR product at 55 °C was much higher than at 50 °C. Both lanes were cut out used for ligation into pSB1C3.
PCR #
Template
primer 1
primer 2
polymerase
product
size
annealing temperature
40
CAT-1_PstI_out
o14sd_031
014sd_032
Q5
CAT-1_C1184A_NgoMIV
5.8 kb
55 °C
PCR according to protocol, afterwards PCR was digested by DpnI and transformed onto agar plates containing Ampicilin.
Gel extraction of CAT-1
Concentration: 21.4 ng/µl
Ligation approach:
Plasmid
backbone
CAT-1 RFC 10
length (bp)
2070
1925
concentration [ng/µl]
5.1
21.4
volume [µl]
9.8
6.33
Water
11.2
4.37
plus 2 µl 1x T4 buffer and 0.5 µl T4 ligase (400,000 U/ml) each
Control and ligation were incubated at room temperature for 30 minutes. Transformation was done with 10 µl each. Bacteria cultures were incubated at 37 °C and 600 rpm for 60 minutes.
Transformation was done with 100, 50, 20, 10 µl to discover the optimal transformation volume.
Several ONC were made (PCR 31 – 35 (3x each), PCR 37 (10x)).
2014.09.11
DNA extraction from ONC
Template
concentration [ng/µl]
template
concentration [ng/µl]
ePDZB_A1775G_EcoRI I
98.5
ePDZB_A1775G_EcoRI VI
142.1
ePDZB_A1775G_EcoRI II
91.5
ePDZB_A1775G_EcoRI VII
78.1
ePDZB_A1775G_EcoRI III
95.5
ePDZB_A1775G_EcoRI VIII
64.0
ePDZB_A1775G_EcoRI IV
93.8
ePDZB_A1775G_EcoRI IX
80.3
ePDZB_A1775G_EcoRI V
78.0
ePDZB_A1775G_EcoRI X
71.1
GAL4 RFC 25 I
105.9
WPRE_C530A_NgoMIV I
109.3
GAL4 RFC 25 II
123.8
WPRE_C530A_NgoMIV II
119.0
GAL4 RFC 25 III
117.6
WPRE_C530A_NgoMIV III
209.5
P2A RFC 10 I
107.3
SEAP RFC 10 I
117.5
P2A RFC 10 II
117.8
SEAP RFC 10 II
123.4
P2A RFC 10 III
113.1
SEAP RFC 10 III
119.6
 
Test digests
Template
enzyme
buffer
fragment size
P2A RFC 10
EcoRI-HF/PstI-HF
CutSmart
2.0 kb,0.1 kb kb
GAL4 RFC 25
EcoRI-HF/PstI-HF
CutSmart
2.0 kb, 0.5 kb
SEAP RFC 10
EcoRI-HF/PstI-HF
CutSmart
2.0 kb, 1.5 kb
ePDZb
EcoRI-HF/NcoI
CutSmart
mutated: 3.8 kb
not mutated: 3.2 kb, 0.6 kb
 
Gel result
test digest of PCR 31 & 32.
It looked like ePDZb hadn’t been totally digested. The second lane lay short under 1 kb lane and not at 600 bp. WPRE RFC 10, SEAP RFC 10 and P2A didn’t show expected results. Especially it seemed that SEAP RFC 10 was actually identical with GAL 4 RFC 25 due to the lanes at 500 bp, which were expected for GAL4 RFC 25.
As a result GAL4 RFC I will be sequenced
Eight colonies from containing PCR 40 were picked for ONC.
2014.09.12
Despite to bad results on gel electrophoresis, P2A I and SEAP RFC 10 I will be sequenced.
DNA extraction from ONC
Template
concentration [ng/µl]
template
concentration [ng/µl]
CAT-1_C1184A_NgoMIV I
473.8
CAT-1_C1184A_NgoMIV V
269.1
CAT-1_C1184A_NgoMIV II
461.6
CAT-1_C1184A_NgoMIV VI
329.3
CAT-1_C1184A_NgoMIV III
373.3
CAT-1_C1184A_NgoMIV VII
452.2
CAT-1_C1184A_NgoMIV IV
469.1
CAT-1_C1184A_NgoMIV VIII
398.4
 
Test digest
Template
enzyme
buffer
fragment size
CAT-1_C1184A_NgoMIV
XbaI/NgoMIV
CutSmart
mutated: 4.2, 1.2 kb,0.3 kb kb
not mutated: 4.2, 1.2 kb,165 bp, 142 bp.
 
Gel electrophoresis
test digest of PCR 38 & 39.
Samples were run in a 2 % agarose gel. Marker is 100 bp from Promega. Due to missing lane at 300 bp, the restriction site was still in CAT-1.
Ligation of SEAP RFC 10 and CAT-1 were repeated and digested with EcoRI-HF/PstI-HF in CutSmart buffer. Afterwards samples were purified by PCR purification Kit. Concentrations:
Plasmid
backbone
SEAP
CAT-1
length (bp)
2070
1577
1869
concentration [ng/µl]
16.2
39.5
21.4
volume [µl]
1.83
2.79
6.33
Water
 
11.20
7.59
plus 2 µl 1x T4 buffer and 0.5 µl T4 ligase (400,000 U/ml) each
 
 
 
 
 
 
Transformed and incubated in Chloramphenicol containing LB-medium. 10, 50 and 100 µl of cluture were streaked out onto agar plates.
2014.09.13
Only few colonies were grown. Wait another day before using them for ONC.
Our instructor recommended us to use Pfu-Polymerase instead of Q5 or Phusion for site-directed mutagenisis PCRs.
PCR #
Template
primer 1
primer 2
polymerase
product
size
annealing temperature
40
SEAP_PstI_out (09.01.)
o14sd_045
014sd_046
Pfu
SEAP_G2780C_NgoMIV
5.8 kb
55 °C
41
LOV_09_10
o14sd_039
014sd_040
Pfu
LOV_without_PstI
3.6 kb
55 °C
42
ePDZb (145.3 ng/µl)
o14sd_005
014sd_006
Pfu
ePDZb_Eco_out
3.4 kb
55 °C
43
CAT-1 mutated
o14sd_037
014sd_038
Pfu
CAT-1_AgeI_out
5.8 kb
55 °C
44
WPRE RFC 10
o14sd_059
014sd_060
Pfu
WPRE_NgoMIV_out
2.6 kb
55 °C
45
P2A RFC 10 (107.3 ng/µl)
o14sd_027
014sd_028
Pfu
P2A_NgoMIV_out
2.2 kb
55 °C
 
Pfu protocol:     basically just a normal PCR program:      1x           95 °C      30 seconds
                                                                                                              15x:       95 °C      30 seconds
                                                                                                                             55 °C      1 minute
                                                                                                                             68 °C      2 minutes/kb
                                                                                                              1x:          68 °C      10 minutes
Afterwards, PCRs weren’t digested by DpnI, but 5 µl were taken for transformation on either Ampicillin or Chloramphenicol agar plates.
2014.09.14
3 transformations worked (PCR 40, 42, 44). Colonies of PCR 42 were small, so they were left an additional day in the incubator.
2014.09.15
PCR #
Template
primer 1
primer 2
polymerase
product
size
annealing temperature
46
GAL4
o14sd_011
014sd_012
Q5
GAL4 RFC 25
0.5 kb
55 °C
PCR was made in 50 µl and in triplicates.
2014.09.16
Triplicates of PCR 46 were digested in CutSmart by EcoRI-HF/PstI-HF for ligation into pSB1C3. Ligation of PCR 46 was made as last time reported.
ONC were made from PCR 40 (2x), 42 (6x), 44 (2x).
2014.09.17
DNA extraction of ONCs.
Template
concentration [ng/µl]
Template
concentration [ng/µl]
ePDZb_Eco_out I
135.6
ePDZb_Eco_out VI
313.9
ePDZb_Eco_out II
185.2
SEAP_NgoMIV_out I
151.0
ePDZb_Eco_out III
62.0
SEAP_NgoMIV_out II
200.0
ePDZb_Eco_out IV
117.0
WPRE_NgoMIV_out I
84.0
ePDZb_Eco_out V
63.9
WPRE_NgoMIV_out II
115.8
 
Test digests
Template
enzyme
buffer
fragment size
WPRE_NgoMIV_out
XhoI/NgoMIV
CutSmart
????
SEAP_NgoMIV_out
NgoMIV_BamHI-HF
CutSmart
Mutated: 3 kb, 1.7 kb
Not mutated: 3 kb, 0,9 kb, 0.8 kb
ePDZb
EcoRI-HF/XbaI/NheI
CutSmart
mutated: 2.4 kb, 1.4 kb
not mutated: 2.4 kb, 0.9 kb. 0.5 kb
 
Ligation of GAL 4 RFC 25 didn’t work.
2014.09.18
Every test digest failed.
Ligation of GAL4 RFC 25 repeated, but with Quick ligation buffer.
Plasmid
backbone
GAL4 RFC 25
length (bp)
2070
441
concentration [ng/µl]
64.1
87.9
volume [µl]
0.78
0.36
Water
 
1.20
plus 2.6 µl 1x T4 buffer and 0.2 µl T4 ligase (400,000 U/ml)
We used for a 2nd approach instead of molare ratio of 1:3 just thrice volume of insert (0.73 µl pSB1C3 + 1.47 µl GAL 4 RFC 25).
Ligations were incubated at room temperature for 15 minutes. 5 µl were taken for transformation and afterwards plates were incubated at 37 °C.
PCR #
Template
primer 1
primer 2
polymerase
product
size
annealing temperature
47
pKM292 re-transformation
o14sd_011
014sd_012
Q5
GAL 4 RFC 25
0.6 kb
55 °C
48
p14zl_003
o14sd_029
014sd_030
Q5
Puromycin acetyl transferase
3.6 kb
55 °C
49
WPRE RFC 10 seq.
o14sd_059
014sd_060
Q5
WPRE RFC 10 with RFC 25 overhangs
0.6 kb
55 °C
Each PCR was made in 50 µl and triplicates.
Gel electrophoresis showed fine results. Lanes were cut out and ligations into pSB1C3 were made at 21.09.
2014.09.21
 
PCR #
Template
primer 1
primer 2
polymerase
product
size
annealing temperature
50
CAT-1 mutated
o14sd_041
014sd_042
Pfu
CAT-1 RFC 10 with RFC 25 overhangs
1.9 kb
55 °C
51
SEAP_PstI_out
o14sd_051
014sd_052
Pfu
SEAP RFC 10 with RFC 25 overhangs
1.6 kb
50 °C
Each PCR was made in 50 µl and triplicates.
ONC were made from ligations.
 
2014.09.22
 
No PCR product visible. PCR 50 & 51 will be repeated with Phusion instead of Pfu with a common annealing temperature of 50 °C.
PCR #
Template
primer 1
primer 2
polymerase
Product
size
annealing temperature
54
PCR 1.1 II
o14sd_005
014sd_006
Phusion
ePDZb_EcoRI_out
1.9 kb
55 °C
PCR was digested by DpnI and afterwards 3 µl were taken for transformation. The bacteria culture was streaked out onto a Ampicilin plate.
 
DNA extraction of ONC
Template
concentration [ng/µl]
Template
concentration [ng/µl]
WPRE RFC 10 (PCR 49) I
105.0
Puromycin acetyltransferase RFC 25 I
131.4
WPRE RFC 10 (PCR 49) II
146.0
Puromycin acetyltransferase RFC 25 II
161.2
WPRE RFC 10 (PCR 49) III
131.0
SEAP_NgoMIV_out II Puromycin acetyltransferase RFC 25 III
70.0
WPRE RFC 10 (PCR 49) IV
267.0
Puromycin acetyltransferase RFC 25 IV
124.0
WPRE RFC 10 (PCR 49) V
140.0
Puromycin acetyltransferase RFC 25 V
230.0
GAL 4 RFC 25 I
156.1
GAL 4 RFC 25 II
247.8
GAL 4 RFC 25 III
134.4
GAL 4 RFC 25 IV
135.7
GAL 4 RFC 25 V
130.4
Test digsts of ONC
Template
enzyme
buffer
fragment size
WPRE RFC 25
NcoI-HF
CutSmart
1.5 kb, 1.2 kb
GAL 4 RFC 25
XhoI
CutSmart
1.2 kb, 0.9 kb, 0.3 kb
Puromycin acetyltransferase
XhoI
CutSmart
1.8 kb, 0.9 kb
 
All digests showed lanes with the expected sizes.
2014.09.23
 
Transformation with PCR 54 worked. Three colonies were picked for ONC.
Gel electrophoresis of PCR 50 & PCR 51
test digest of PCR 50 & 51.
Gel electrophoresis of PCR 50 & PCR 51
img src="https://static.igem.org/mediawiki/2014/2/2a/Freiburg2014_2014-09-23_testverdauPCR50_PCR51_neu.jpg" alt="Description of Image">
test digest of PCR 50 & 51.
2014.09.24
 
DNA extraction of ONC
Template
concentration [ng/µl]
ePDZb_Eco_out I
49.6
ePDZb_Eco_out II
67.7
ePDZb_Eco_out III
52.7
 
Test digest
Template
enzyme
buffer
fragment size
ePDZb_EcoRI_out
EcoRI-HF/ClaI/HindHIII-HF
CutSmart
Mutated: 2.3 kb, 1.5 kb
Not mutated: 2.3 kb, 0.9 kb, 0.6 kb
 
Result: all 3 samples showed expected DNA lanes. Colony 2 will be sequenced.
Colony 1 of WPRE RFC 25, GAL 4 RFC 25 and Puromycin acetyltransferase RFC 25 will be sequenced.
 
2014.09.25
 
All biobricks showed expected DNA sequence and ePDZb lost the EcoRI site.
PCR #
Template
primer 1
primer 2
Polymerase
Product
size
annealing temperature
55
ePDZb_EcoRI_out
o14sd_001
014sd_002
Phusion
ePDZb_AgeI_out
3.8 kb
55 °C
56
SEAP_PstI_out (367.6 ng/µl)
o14sd_045
014sd_046
Phusion
SEAP_NgoMIV_end
5.8 kb
50 °C
57
SEAP_PstI_out (367.6 ng/µl)
o14sd_049
014sd_050
Phusion
SEAP RFC 10
1.6 kb
50 °C
58
WPRE RFC 10 (25.09.)
o14sd_055
014sd_056
Phusion
WPRE RFC 25
1.6 kb
50.5 °C
 
Gel of PCR 57 showed a strong lane. The lane was cut out and prepared for digest/ligation. Digest was done with EcoRI-HF/Pst-HF in CutSmart.
Ligation of PCR 50 and PCR 57
 
Plasmid
backbone
CAT-1 RFC 25 overhangs
SEAP RFC 10
length (bp)
2070
1945
1577
concentration [ng/µl]
64.1
68.7
80.4
volume [µl]
0.38
1.84
1.71
Water
 
1.20
 
plus 2.6 µl 1x T4 buffer and 0.2 µl T4 ligase (400,000 U/ml)
Additional approach: just use a triple volume of insert each (0.6 µl backbone, 1.8 µl insert).
2014.09.26
Transformation of PCR 58 didn’t work but PCR 55 & 56 showed colonies on plates. 5 colonies were picked for ONC.
Ligations were partly successful. There for both SEAP RFC 10 and CAT-1 with RFC 25 overhangs colonies. 5 of them were picked for ONC each.
2014.09.27
 
DNA extraction from OVC
Template
concentration [ng/µl]
Template
concentration [ng/µl]
PCR 56 I
160.4
PCR 55 I
81.1
PCR 56 II
122.9
PCR 55 II
71.4
PCR 56 III
109.9
PCR 55 III
46.8
PCR 56 IV
105.2
PCR 55 IV
43.1
PCR 56 V
199.3
PCR 55 V
81.0
SEAP RFC 10 I
182.2
PCR 50 I
130.2
SEAP RFC 10 II
100.9
PCR 50 II
120.2
SEAP RFC 10 III
122.7
PCR 50 III
17.2
SEAP RFC 10 IV
239.0
PCR 50 IV
128.2
SEAP RF 10 V
201.6
PCR 50 V
170.3
 
Test digests
Template
enzyme
buffer
fragment size
PCR 50
EcoRV-HF/KpnI-HF
CutSmart
2.2 kb, 1.3 kb
PCR 55
AgeI-HF/XbaI/NheI
CutSmart
mutated: 2.4 kb, 1.4 kb
not mutated: 2.4 kb, 0.9 kb, 0.5 kb
PCR 56
NgoMIV/BamHI-HF
CutSmart
mutated: 3.0 kb, 1.7 kb
not mutated: 3.0 kb, 1 kb, 0.7 kb
PCR 57
NcoI-HF
CutSmart
1.8 kb, 1 kb, 0.7 kb
 
Gel electrophoresis
test digest of PCR 50, 55-57.
PCR 50, 56 and 57 failed. PCR 55 looked like the plasmid had been cut once. Sequence analysis showed that XbaI site was overlapped with a methylation site, thus restriction site was blocked and couldn’t recognized anymore. Therefore PCR 55 I would be sequenced.
2014.09.28
PCR #
Template
primer 1
primer 2
Polymerase
Product
size
annealing temperature
59
WPRE RFC 10 -NgoMIV
o14sd_055
014sd_056
Phusion
WPRE RFC 25
0.6 kb
57 °C
60
CAT-1 PstI out
o14sd_039
014sd_040
Phusion
CAT-1 RFC 10
1.9 kb
56 °C
PCR 59 was done in 50 µl approaches and in triplicates and was separated in a 2 % gel.
Gel electrophoresis
 
test digest of WPRE RFC 25 ligation into pSB1C3.
Results looked fine. The PCR product was cut out with EcoRI-HF/PstI-HF and used for ligation into pSB1C3.
test digest of PCR 60.
Results looked fine. The PCR product was cut out with EcoRI-HF/PstI-HF and used for ligation into pSB1C3.
 
Gel extraction
Template
concentration [ng/µl]
WPRE RFC 25
76.2
CAT-1 RFC 10
289.6
SEAP RFC 10
like PCR 57
 
2014.09.30
Sequencing results confirmed that ePDZb is AgeI-free.
PCR #
Template
primer 1
primer 2
Polymerase
Product
size
annealing temperature
61
PCR 55 I
o14sd_007
014sd_008
Phusion
ePDZb RFC 10
0.6 kb
56.5°C
62
SEAP PstI out
o14sd_045
014sd_046
Phusion
SEAP_C2784G_NgoMIV
5.4 kb
56 °C
PCR Product was purified by gel extraction and cut with EcoRI-HF/PstI-HF and used for ligation into pSB1C3.
PCR 62 was digested by DpnI and transformed on agar plate containing Ampicillin.
ligation
 
WPRE RFC 25 (09/29)
CAT-1 RFC 10 (09/29)
SEAP RFC 10 (09/25)
pSB1C3
1.46 µl
1 µl
1 µl
insert
0.74 µl
0.4 µl
0.4 µl
water
/
0.8
/
plus 2.6 µl Quick ligation buffer, 0.2 µl T4 ligase (400,000 U/ml)
 
2014.10.01
All transformed plates had colonies.
DNA extraction
Template
concentration [ng/µl]
WPRE RFC 25 I
130.0
WPRE RFC 25 II
109.0
WPRE RFC 25 III
115.0
WPRE RFC 25 IV
140.0
PCR 62 I
448.0
PCR 62 II
458.0
PCR 62 III
490.0
PCR 62 IV
413.0
 
Template
concentration [ng/µl]
Template
concentration [ng/µl]
SEAP RFC 10 I
216.9
CAT-1 RFC 10 I
265.9
SEAP RFC 10 II
192.9
CAT-1 RFC 10 II
396.3
SEAP RFC 10 III
193.3
CAT-1 RFC 10 III
223.2
SEAP RFC 10 IV
176.5
CAT-1 RFC 10 IV
272.2
SEAP RFC 10 V
167.5
CAT-1 RFC 10 V
209.0
SEAP RFC 10 VI
193.4
CAT-1 RFC 10 VI
188.6
ePDZb RFC 10 I
125.1
ePDZb RFC 10 II
144.4
ePDZb RFC 10 III
117.5
ePDZb RFC 10 IV
125.0
ePDZb RFC 10 V
118.8
ePDZb RFC 10 VI
109.5
Test digests
Template
enzyme
buffer
fragment size
PCR 59
EcoRV-HF/NgoMIV
CutSmart
1.7 kb, 0.9 kb
PCR 60
EcoRV-HF/KpnI-HF
CutSmart
2.2 kb, 1.4 kb, 0.4 kb
PCR 61
XhoI
CutSmart
1.8 kb, 0.9 kb
PCR 62
BamHI-HF/NgoMIV
CutSmart
2.3 kb, 1 kb, 0.3 kb
Gel electrophoresis
test digest of PCR 62 and ligation of WPRE RFC 25 into pSB1C3.
All results looked fine. It seemed that SEAP contained a NgoMIV site less and the WPRE digests proved that WPRE RFC 25 is in pSB1C3.
test digest of following ligations: ePDZb RFC 10, SEAP RFC 10, CAT-1 RFC 10.
Results looked fine. Every sample showed expected lane size.
Colony one of each plasmid was sent to sequencing.
Standardization - October
2014.10.02
All four ligations contain expected DNA sequence.
 
Cloning
Standardization
August
September
October
iGEM Team Freiburg 2014
Inv.
Retrieved from " http://2014.igem.org/Team:Freiburg/Notebook/Cloning "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
