<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCSF_Notebook_StanleeQuach_html skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCSF/Notebook/StanleeQuach.html</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Stanlee's Lab Notebook 2015</TITLE></P><H1>Stanlee's Lab Notebook</H1><H2>6/8/15</H2><P>First day of iGem. Took notes on introductory lectures and set up PCRs for Constitutive Promoters.</P><H3>[]Lectures</H3><PRE><CODE>-Lecture on Molecular Biology and Cloning
-Lecture on Synthetic Biology
-Introduced to &quot;A Plasmid Editor&quot; for planing plasmids and inserts
</CODE></PRE><H3>[]pTEF1 and Bar1 PCR Reaction</H3><PRE><CODE>-pTEF1 PCR Reaction:
    -10uM FW Primer (+ApaI)     2.5uL
    -10uM RV Primer (+XhoI)     2.5uL
    -pTEF1 Template DNA         0.5uL
    -2x Phusion Master Mix      25.0uL
    -Water                      19.5uL
                    Total:      50.0uL

-Bar1 PCR Reaction
    -10uM Primer #92 (+XhoI)    2.5uL
    -10uM Primer #93 (+NotI)    2.5uL
    -Bar1 Template DNA          1.0uL
    -2x Phusion Master Mix      25.0uL
    -Water                      19.0uL
                    Total:      50.0uL

-Thermocycler Protocol
    -Initial Denaturation       98'C    30s
    -35 Cycles
        -Denaturation           98'C    10s
        -Annealing              55'C    20s
        -Extension              72'C    1m
    -Final Extension            72'C    5m
    -Hold                       4'C     Forever
</CODE></PRE><H3>[]In Stanlee's Freezer Box</H3><PRE><CODE>-pTEF1 FW Primer (+ApaI)
-Bar1 Primer #92 (+XhoI)
-Bar1 Primer #93 (NotI)
</CODE></PRE><H2>6/9/15</H2><P>Ran gels for yesterday's PCR reactions. Took notes on introductory lectures. Performed digest for pTEF1, Bar1, and pJW608 plasmid.</P><H3>[]Lectures</H3><PRE><CODE>-Lecture on How to Read a Scientific Paper
-Lecture on Presentation Skills
-Lecture on Cell Signaling
</CODE></PRE><H3>[]pTEF1 and Bar1 PCR Reactions Gel</H3><PRE><CODE>-Transferred 5.0uL of both pTEF1 and Bar1 PCR Reactions to new PCR tubes
-Added 1.0uL of Purple Loading Dye (6x) to each tube
-Loaded 5.0uL of each tube to a 1% Agarose Gel
</CODE></PRE><H3>[]pTEF1, Bar1, and pJW608 Digestion</H3><PRE><CODE>-pTEF1 Digest
    -Added 5.0uL CutSmart Buffer
    -Added 0.5uL ApaI. Vortexed
    -Incubated at room temperature for at least 1 hour
    -Added 0.5uL XhoI. Vortexed
    -Incubated at 37'C for at least 1 hour
-Bar1 Digest
    -Added 5.0uL CutSmart Buffer
    -Added 0.5uL XhoI and 0.5uL NotI. Vortexed
    -Incubated at 37'C for at least 1 hour
-pJW608 Digest
    -Transferred 10.0uL of plasmid from stock into a new tube
    -Added 3.0uL CutSmart Buffer and 16uL water
    -Added 0.5uL ApaI. Vortexed
    -Incubated at room temperature for at least 1 hour
    -Added 0.5uL NotI. Vortexed
    -Incubated at 37'C for at least 1 hour
</CODE></PRE><H2>6/10/15</H2><P>Ran gels for our digests. Took notes on introductory lecture. Extracted DNA from gel. Performed ligation for plasmid and inserts. Performed transformation of E. coli cells. Plated transformed cells.</P><H3>[]Lectures</H3><PRE><CODE>-Lecture on Flow Cytometry
</CODE></PRE><H3>[]pTEF1, Bar1, and pJW608 Digestion Gel</H3><PRE><CODE>-Added 10.0uL of Purple Loading Dye (6x) to pTEF1 and Bar1 Digest Reactions.
-Added 6uL Purple Loading Dye (6x) to pJW608 Digest Reaction
-Transferred 60uL of pTEF1 and Bar1 Digest Reactions to gel
-Transferred 36uL of pJW608 Digest Reaction to gel
</CODE></PRE><H3>[]pTEF1, Bar1, and pJW608 Gel Extraction</H3><PRE><CODE>-Excised gel bands 2 and 3 and transferred to 1.5mL tube
-Excised gel bands 4 and 5 and transferred to 1.5mL tube
-Excised gel band 6 and transferred to 1.5mL tube
-Weighted each tube
    -pTEF1 Tube:    201.9mg
    -Bar1 Tube:     355.1mg
    -pJW608 Tube:   198.1mg
-Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
    -pTEF1 Tube:    +600uL
    -Bar1 Tube:     +1050uL
    -pJW608 Tube:   +600uL
-Incubated at 50'C for 10 minutes
-Added isopropanol in a 1:1 ratio
    -pTEF1 Tube:    +200uL
    -Bar1 Tube:     +355uL
    -pJW608 Tube:   +198uL
-Transferred each sample to a QIAquick column in a 2mL collection tube
-Centrifuged for 1 minute
-Discarded flow-through, returned column to collection tube
-Added 0.5mL of Buffer QG to column and centrifuged for 1 minute
-Discarded flow-through, returned column to collection tube
-Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
-Discarded flow-through, returned column to collection tube
-Centrifuged column empty for 1 minute
-Transferred column to new 1.5mL tube
-Added 50uL of water to the column to elute DNA
-Centrifuged for 1 minute
</CODE></PRE><H3>[]pTEF1, Bar1, and pJW608 Nanodrop</H3><PRE><CODE>-Used Nanodrop to measure 1.5uL of each sample for concentration
    -pTEF1:     -XXXng/uL
    -Bar1:      -XXXng/uL
    -pJW608:    25.0ng/uL
        -ERROR
            -pTEF1 and Bar1 were reported with negative concentrations of ng/uL
            -Possibly due to residual ethanol from Buffer PE
</CODE></PRE><H3>[]pTEF1, Bar1, and pJW608 Ligation</H3><PRE><CODE>-Since Bar1 was reported with a negative concentration, we used Erik Lamp's Sample (15.44ng/uL) as a substitution
-Added the following to a 1.5mL tube:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -pJW608 DNA (50ng):         2.0uL
    -Bar1 DNA (75ng):           5.0uL
    -pTEF1 DNA (???):           10.0uL
    -Nuclease-Free Water:       0.0uL
    -T4DNA Ligase:              1.0uL
                    Total:      20.0uL
-Mixed reaction by pipetting up and down
-Incubated at room temperature for 1 hour
</CODE></PRE><H3>[]In Erika's Freezer Box:</H3><PRE><CODE>-pTEF1 (+ApaI + XhoI) Digest
-pJW608 (+XhoI + NotI) Digest
</CODE></PRE><H3>[]pTEF1, Bar1, and pJW608 Transformation</H3><PRE><CODE>-Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
-Added 5uL of ligased pJW608 to tube. Flicked tube 5 times to mix
-Placed mixture on ice for 30 minutes
-Heat Shocked at 42'C for 30 seconds
-Placed on ice for 5 minutes
-Added 950uL of SOC into mixture
-Incubated at 37'C for 60 minutes. Shaken at 250rpm
-Warmed two plates (LB + Carb) to 37'C
-Mixed cells by inverting
-Spread 100uL cells onto one plate
-Centrifuged remaining mixture to concentrate cells
-Discarded excess liquid and resuspended cells
-Spread 100uL (8x) cells onto second plate
-Incubated both plates overnight at 37'C
</CODE></PRE><H2>6/11/15</H2><P>PCRed a colony sample of our E.coli to see if plasmid was properly ligated together with inserts and transformed into cells. Took notes on introductory lectures and project introduction. Ran gel of our colony PCR.</P><H3>[]Lectures</H3><PRE><CODE>-Lecture on Working with E.coli and Yeast
-Lecture on The Immune System
-Lecture on iGem Project Introduction
</CODE></PRE><H3>[]pTEF1, Bar1, pJW608 E.coli Colony PCR</H3><PRE><CODE>-Added 25uL of water to 6 PCR tubes
-Transferred a single colony from the plate to each PCR tube
-Transferred 5uL of each cell/water mixture into a new PCR tube
-Created a GoTaq Green Master Mix for 7 reactions:
    -2x GoTaq Green PCR Mix:            70.0uL
    -10uM pTEF1 FW Primer (+ApaI):      7.0uL
    -10uM Bar1 RV Primer #93 (+NotI):   7.0uL
    -Water:                             21.0uL
                            Total:      105.0uL
        -ERROR
            -Did not keep master mix on ice
-Added 15uL of GoTaq Green Master Mix to each PCR tube
-Vortexed and centrifuged tubes
-PCR tubes placed in thermocycler:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation           95'C    45s
            -Annealing              55'C    30s
            -Extension              72'C    1m
    -Final Extension            72'C    10m
</CODE></PRE><H3>[]pTEF1, Bar1, pJW608 E.coli Colony PCR Gel</H3><PRE><CODE>-Added 10uL of PCR reaction to 1% Agarose Gel
</CODE></PRE><H3>[]In Stanlee's Freezer Box</H3><PRE><CODE>-pTEF1 FW Primer (+ApaI)
-Bar1 Primer #92 (+XhoI)
-Bar1 Primer #93 (NotI)
-GoTaq Green Master Mix
</CODE></PRE><H2>6/12/15</H2><P>Purified plasmid DNA with Miniprep. Sent sample of DNA to be sequenced. Observed yeast transformation lab performed. Field trip to NASA Wind Tunnel and meeting with Stanford/Brown iGem Team.</P><H3>[]pTEF1, Bar1, pJW608 E.coli Miniprep</H3><PRE><CODE>-Resuspended cells in 250uL Buffer P1
-Transferred mixture to 1.5mL tube
-Added 250uL Buffer P2. Mixed by inverting 6 times
-Added 350uL Buffer N3. Mixed by inverting 6 times
-Centrifuged for 10 minutes
-Poured supernatant into a QIAprep spin column
-Centrifuged for 30 seconds. Discarded flow-through
-Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
-Discarded flow-through. Centrifuged empty for 1 minute
-Placed column in a new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
-Centrifuged for 1 minute.
</CODE></PRE><H2>6/15/15</H2><P>Presented iGEM competition information. Assigned team members role for overall competition. Outlined out 2015 iGem project concept. Assigned team members role for project. Took notes on lecture.</P><H3>[]Lecture</H3><PRE><CODE>-Lecture on Introduction to Genetic Circuits
</CODE></PRE><H3>[]Assignments</H3><PRE><CODE>-Assigned to work on Team Wiki
-Assigned to work on Positive Feedback Aspect
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Ask about inserting Gel Photos
-Discuss Positive Feedback plan with Samantha and Erik
-Compare lab data in Erika's Notebook
-Review Secrete and Sense Article
-Brainstorm for Policy and Practices idea
</CODE></PRE><H2>6/16/15</H2><P>Didn't do much today. Started learning HTML programming and took notes on a lecture on computational modeling.</P><H3>[]Lecture</H3><PRE><CODE>-Lecture on Computational Modeling
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Discuss Positive Feedback plan with Samantha and Erik
-Read other science journals
</CODE></PRE><H2>6/17/15</H2><P>Didn't do much today. Continued learning HTML programming and started work on MATLAB. Also went over with Kara about the Positive Feedback circuit plan outline.</P><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Design plasmid for positive feedback
-Read other science journals
</CODE></PRE><H2>6/18/15</H2><P>Continued lessons on MATLAB. Attended Journal Club, reviewing Wendell's Secrete and Sense paper. Reviewed what to present for next week's meeting. Made primers for LexAOps promoter.</P><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Read new journal
-PCR LexAOps promoter
-Work on presentation for next week's meeting
</CODE></PRE><H2>6/19/15</H2><P>Attended lecture on immune system. Recapped on policies and practices ideas. Finished group meeting presentation slides. Finished lessons on MATLAB. Participated in social hour with Lim Lab.</P><H3>[]Lecture</H3><PRE><CODE>-Lecture on Adaptive Immune System Overview
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Read new journal
-Phospho-flow Research
-PCR LexAOps promoter
-Digest LexAOps and pNH605 with ApaI and XhoI
-Ligate LexAOps and pNH605 together
-Transform E.coli cells
-Grow colonies
-Purify DNA
</CODE></PRE><H2>6/22/15</H2><P>Presented on last week's progress in group meeting. PCRed  LexAOps promoter with forward and reverse primers. Ran gel to check for correct band size.</P><H3>[]LexAOps PCR Reaction</H3><PRE><CODE>-Reconstitute Primers:
    -LexAOps (+ApaI) FW-129:
        -35.2 nmoles x 10 = 352uL
    -LexAOps (+XhoI) RV-130:
        -29.9 nmoles x 10 = 299uL
-Dilution:
    -10uL primers + 90uL water = 10uM stock solution / 100uL
-PCR Reaction:
    -10uM FW Primer-129 (+ApaI)     2.5uL
    -10uM RV Primer-130 (+XhoI)     2.5uL
    -T-64 M-64 Template DNA         0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Thermocycler Protocol:
    -Initial Denaturation       98'C    30s
    -35 Cycles
        -Denaturation           98'C    10s
        -Annealing              55'C    20s
        -Extension              72'C    30s
    -Final Extension            72'C    5m
    -Hold                       4'C     Forever
-Gel Loading:
    -1% Agarose Gel         35mL
    -SyberSafe              3.5uL

    -2-log DNA Ladder           10uL    Lane #1
    -LexAOps Sample             6.0uL   Lane #2
        -LexAOps PCR Sample     5.0uL
        -6x Purple Loading Dye  1.0uL
    -*pTEF (Josh)               6.0uL   Lane #4
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps (+ApaI + XhoI) PCR&quot;

-ERROR:
    -Bands were not visible on the first gel.
    -Repeated entire lab protocol with the following changes:
        -2uL of old PCR reaction was used instead of T-64 M-64 as our template DNA
        -Used 25uL of 1% Agarose Gel and 2.5uL if SyberSafe

-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps (+ApaI + XhoI) PCR #2 30 mins&quot;
        -2-Log DNA Ladder       10uL    Lane #1
        -LexAOps Sample #2      6.0uL   Lane #2
        -LexA (HC,JS,NS)        6.0uL   Lane #5
        -pFig2c (HC,JS,NS)      6.0uL   Lane #6
        -2-Log DNA Ladder       10uL    Lane #7
    -See Gel Folder for gel labeled &quot;LexAOps (+ApaI + XhoI) PCR #2 45 mins&quot;
        -2-Log DNA Ladder       10uL    Lane #1
        -LexAOps Sample #2      6.0uL   Lane #2
        -LexA (HC,JS,NS)        6.0uL   Lane #5
        -pFig2c (HC,JS,NS)      6.0uL   Lane #6
        -2-Log DNA Ladder       10uL    Lane #7
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Read new journal
-Phospho-flow Research
-Digest LexAOps and pNH605 with ApaI and XhoI
-Ligate LexAOps and pNH605 together
-Transform E.coli cells
-Grow colonies
-Purify DNA
</CODE></PRE><H2>6/23/15</H2><P>Researched Phospho-Flow Cytometry. Performed restriction digests on LexAOps promoter and plasmid. Ran gel for digest. Made more agarose gel and TAE Buffer. Extracted DNA from gel. Repeated PCR for LexAOps promoter again. Transformed Hy86E3 and pGEM38 plasmids into E.coli cells.</P><H3>[]LexAOps + Hy86E3 Digest</H3><PRE><CODE>-LexAOPs Promoter:
    -Added 5.0uL of CutSmart Buffer to LexAOps PCR tube
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C for 1 hour
-Hy86E3 Plasmid:
    -Transferred 10.0uL of plasmid into a new PCR tube
    -Added 16.0uL of water
    -Added 3.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL of XhoI
    -Vortexed
    -Incubated at 37'C for 1 hour
</CODE></PRE><H3>[]LexAOps + Hy86E3 Gel</H3><PRE><CODE>-Gel Loading:
    -1% Agarose Gel     25mL
    -SyberSafe          2.5uL

    -2-Log DNA Ladder       10.0uL  Lane #1
    -Hy86E3 Digest #1       30.0uL  Lane #2
    -Hy86E3 Digest #2       10.0uL  Lane #3
        -ERROR: Added too much loading dye to plasmid sample, 10.0uL instead of 6.0uL
    -LexAOps Digest #1      30.0uL  Lane #4
    -LexAOps Digest #2      30.0uL  Lane #5
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + Hy86E3 Digestion&quot;
    -Bands visible for Hy86E3 plasmid
-ERROR:
    -No bands for LexAOps promoter -&gt; Retry
    -Repeated PCR protocol with the following changes:
        -#1 (no DMSO)
            -10uM FW Primer-129 (+ApaI)     2.5uL
            -10uM RV Primer-130 (+XhoI)     2.5uL
            -T-64 M-64 Template DNA         0.5uL       
            -2x GoTaq Green Master Mix      25.0uL
            -Water                          19.5uL
                                Total:      50.0uL
        -#2 (w/ DMSO)
            -10uM FW Primer-129 (+ApaI)     2.5uL
            -10uM RV Primer-130 (+XhoI)     2.5uL
            -T-64 M-64 Template DNA         0.5uL       
            -2x GoTaq Green Master Mix      25.0uL
            -DMSO                           1.5uL
            -Water                          18.0uL
                                Total:      50.0uL
        -Thermocycler Protocol:
            -Initial Denaturation       95'C    30s
                -5 Cycles
                    -Denaturation           95'C    30s
                    -Annealing              45'C    30s
                    -Extension              72'C    1m
                -30 Cycles
                    -Denaturation           95'C    30s
                    -Annealing              55'C    30s
                    -Extension              72'C    1m
                -Final Extension            72'C    5m
                -Hold                       4'C     Forever
-Gel Extraction:
    -Excised gel bands 2 and 3 and transferred to 1.5mL tube
    -Weighted tube
        -Hy86E3 Tube:   107.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -Hy86E3 Tube:   +320uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -Hy86E3 Tube:   +107uL
    -Transferred each sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to new 1.5mL tube
    -Added 30uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of Hy86E3 plasimd DNA
    -Results:
        -9.8ng/uL
</CODE></PRE><H3>[]Hy86E3 + pGEM38 Transformation</H3><PRE><CODE>-Hy86E3 + pGEM38 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of plasmid Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 10 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Added 450uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Spread 100uL cells onto each plate
    -Incubated both plates overnight at 37'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Read new journal
-Digest LexAOps with ApaI and XhoI
-Ligate LexAOps and pNH605 together
-Transform E.coli cells
-Grow colonies
-Purify DNA
-Check colonies
</CODE></PRE><H2>6/24/15</H2><P>Ran gel for LexAOps Promoter PCR #3. Set up liquid culture tubes for Hy86E3 and pGEM38 colonies. Set up PCR reaction for LexAOps Promoter PCR #4. Attended Journal Club and presented information on Phospho-Flow Cytometry.</P><H3>[]LexAOps PCR #3 Gel</H3><PRE><CODE>-Gel Loading:
    -1% Agarose Gel         25mL
    -SyberSafe              2.5uL

    -2-Log DNA Ladder       10.0uL  Lane #1
    -LexAOps PCR (no DMSO)  5.0uL   Lane #2
    -LexAOps PCR (w/ DMSO)  5.0uL   Lane #3
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps PCR #3&quot;
    -Faint bands in Lanes #2 and #3
        -~3 kb, likely template DNA
    -Retry for a fourth time
</CODE></PRE><H3>[]Hy86E3 + pGEM38 Liquid Culture</H3><PRE><CODE>-Hy86E3 Liquid Culture
    -Labeled culturing tube
    -Turned on bunson burner
    -Added 5mL of LB + Carb Media to tube
    -Used a pick to get one colony
    -Mixed into tube liquid
    -Incubated overnight at 37'C with shaking at 250rpm
-pGEM38 Liquid Culture
    -Labeled culturing tube
    -Turned on bunson burner
    -Added 5mL of LB + Carb Media to tube
    -Used a pick to get one colony
    -Mixed into tube liquid
    -Incubated overnight at 37'C with shaking at 250rpm
</CODE></PRE><H3>[]LexAOps PCR #4</H3><PRE><CODE>-LexAOps Promoter (Phusion):
    -10uM FW Primer-129 (+ApaI)     2.5uL
    -10uM RV Primer-130 (+XhoI)     2.5uL
    -T-64 (tdTomato) Template DNA   0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-LexAOps Promoter (GoTaq):
    -10uM FW Primer-129 (+ApaI)     2.5uL
    -10uM RV Primer-130 (+XhoI)     2.5uL
    -T-64 (tdTomato) Template DNA   0.5uL       
    -2x GoTaq Green Master Mix      25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Thermocycler Protocol:
    -TouchDownPhu
        -Initial Denaturation       98'C    30s
            -10 Cycles
                -Denaturation       98'C    10s
                -Annealing          55'C    20s
                -(-1'C Per Cycle)
                -Extension          72'C    30s
            -30 Cycles
                -Denaturation       98'C    10s
                -Annealing          55'C    20s
                -Extension          72'C    30s
            -Final Extension        72'C    5m
            -Hold                   4'C     Forever
    -TouchDown_Go
        -Initial Denaturation       95'C    2m
            -10 Cycles
                -Denaturation       95'C    30s
                -Annealing          55'C    30s
                -(-1'C Per Cycle)
                -Extension          72'C    1m
            -30 Cycles
                -Denaturation       95'C    30s
                -Annealing          55'C    30s
                -Extension          72'C    1m
            -Final Extension        72'C    5m
            -Hold                   4'C     Forever
-Gel Loading:
    -1% Agarose Gel     25mL
    -SyberSafe          2.5uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps Promoter (Phusion)     6.0uL   Lane #2
    -LexAOps Promoter (GoTaq)       5.0uL   Lane #3
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps PCR #4&quot;
    -Band at around 600bp for Phusion sample
    -No Band for Taq sample
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Digest LexAOps with ApaI and XhoI
-Ligate LexAOps and pNH605 together
-Transform E.coli cells
-Grow colonies
-Purify DNA
-Check colonies
-Make Weekly Meeting Slides
</CODE></PRE><H2>6/25/15</H2><P>Digested LexAOps Promoter PCR (Phusion) with ApaI and XhoI. Minipreped Hy86E3 and pGem38 plasmids from liquid culture. Ran gel for LexAOps digest. Extracted LexAOps DNA from gel. Ligated LexAOps with Hy86E3 plasmid. Transformed ligation into E.coli cells.</P><H3>[]LexAOps Promoter PCR (Phusion) Digestion</H3><PRE><CODE>-LexAOps Digestion
    -Added 5.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C for 1 hour 
-Gel Loading:
    -1.5% Agarose Gel   35mL
    -SyberSafe          3.5uL

    -2-Log DNA Ladder                       10.0uL  Lane #1
    -LexAOps Promoter Digest (Phusion) #1   30.0uL  Lane #2
    -LexAOps Promoter Digest (Phusion) #2   30.0uL  Lane #3
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps Digest #2&quot;
    -Nice Clear Bands
</CODE></PRE><H3>[]LexAOps Promoter Digestion Gel Extraction</H3><PRE><CODE>-LexAOps Promoter Digestion Gel Extration:
    -Excised gel bands in lanes 2 and 3 and transferred to 1.5mL tube
    -Weighted tube
        -LexAOps Tube:  152.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -LexAOps Tube:  +460uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -LexAOps Tube:  +107uL
    -Transferred each sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to new 1.5mL tube
    -Added 30uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of LexAOps promoter DNA
    -Results:
        -7.665ng/uL
</CODE></PRE><H3>[]Hy86E3 + pGEM38 Miniprep</H3><PRE><CODE>-Hy86E3 Miniprep:
    -Centrifuged liquid cultures for 10 mins
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixture to 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into a QIAprep spin column
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed column in a new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
    -Nanodrop
</CODE></PRE><H3>[]LexAOps + Hy86E3 Ligation</H3><PRE><CODE>-LexAOps + Hy86E3 Ligation (Experiment):
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         6.0uL
    -LexAOps DNA (15ng):        2.0uL
    -Nuclease-Free Water:       9.0uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
-Negative Control:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         6.0uL
    -Nuclease-Free Water:       11.0uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
</CODE></PRE><H3>[]LexAOps + Hy86E3 Transformation</H3><PRE><CODE>-LexAOps + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight at 37'C
-Negative Control Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of negative control sample to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight at 37'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Grow colonies
-Purify DNA
-Check colonies
-Make Weekly Meeting Slides
</CODE></PRE><H2>6/26/15</H2><P>Finished up meeting slides. Took colonies from plates and ran a colony PCR.</P><H3>[]LexAOps + Hy86E3 Colony PCR</H3><PRE><CODE>-LexAOps + Hy86E3 Colony PCR:
    -Filled 8 PCR tubes with 25uL of water
    -Created PCR Master Mix:
        -2x GoTaq Green PCR Master Mix  130.0uL
        -10uM FW Primer-129 (+ApaI)     13.0uL
        -10uM RV Primer-130 (+XhoI)     13.0uL
        -Water                          39.0uL
    -Transferred Colonies:
        -Transferred 1 colony each to 6 PCR tubes from 8x LexAOps/Hy86E3 experiment plate and mixed by pipetting up and down
        - Transferred 1 colony each to 2 PCR tubes from 8x (-) Control plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       95'C    5m
            -10 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -(-1'C Per Cycle)
                -Extension          72'C    1m
            -30 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -Extension          72'C    1m
            -Final Extension        72'C    10m
            -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel               35mL
    -SyberSafe                      3.5uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -Experiment Culture #1          5.0uL   Lane #2
    -Experiment Culture #2          5.0uL   Lane #3
    -Experiment Culture #3          5.0uL   Lane #4
    -Experiment Culture #4          5.0uL   Lane #5
    -Experiment Culture #5          5.0uL   Lane #6
    -Experiment Culture #6          5.0uL   Lane #7
    -(-) Control #1                 5.0uL   Lane #8
    -(-) Control #2                 5.0uL   Lane #9
    -Empty                          0.0uL   Lane #10
    -Josh (C)                       6.0uL   Lane #11
    -Josh (D)                       6.0uL   Lane #12
    -Josh (E)                       6.0uL   Lane #13
    -Empty                          0.0uL   Lane #14
    -2-Log DNA Ladder               10.0uL  Lane #15

    -Ran at 100 volts for 37 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + Hy86E3 Colony PCR&quot;
    -Nice Clear Bands
    -ERRORS:
        -Negative control gave bands when it shouldn't have
        -Bands for Experiment samples seems larger than 665bp
        -Lane 7 didn't have similar bands to the other experiment samples
</CODE></PRE><H3>[]LexAOps + Hy86E3 Liquid Culture</H3><PRE><CODE>-LexAOps + Hy86E3 Liquid Culture:
    -Labeled 5 culturing tube
    -Turned on bunson burner
    -Added 5mL of LB + Carb Media to each tube
    -transferred 20uL from colony PCR tubes #1-5 to culturing tubes
    -Mixed into tube liquid
    -Incubated on Friday and Saturday overnight in 4'C refrigerator
    -Incubated on Sunday overnight in 37'C with shaking at 250rpm 
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Purify DNA
-Finalize Weekly Meeting Slides
</CODE></PRE><H2>6/29/15</H2><P>Placed Liquid Cultures in 37'C heat room shaker for incubation overnight. Presented slides on last week's progress. Discussed Wiki ideas and concepts.</P><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Purify DNA
-Sequence DNA
</CODE></PRE><H2>6/30/15</H2><P>Minipreped LexAOps and Hy86E3 plasmid liquid cultures. Sent purified DNA to sequence.</P><H3>[]Miniprep LexAOps + Hy86E3 Plasmid</H3><PRE><CODE>-Miniprep LexAOps + Hy86E3 Plasmids (#1-#5):
    -Centrifuged liquid cultures for 10 mins
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to 1.5mL tubes
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into a QIAprep spin column
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed column in a new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of each tube
        -LexAOps + Hy86E3 Experiment #1     517.6ng/uL
        -LexAOps + Hy86E3 Experiment #2     746.5ng/uL
        -LexAOps + Hy86E3 Experiment #3     252.8ng/uL
        -LexAOps + Hy86E3 Experiment #4     204.7ng/uL
        -LexAOps + Hy86E3 Experiment #5     172.9ng/uL
-DNA Sequencing:
    -Sent Experiment samples #3-#5 to Quintara for DNA sequencing.
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL?
</CODE></PRE><H2>7/1/15</H2><P>DNA Sequencing reactions failed. Decided to restart experiment from the beginning. Performed PCR and digestion.</P><H3>[]LexAOps Promoter PCR #5</H3><PRE><CODE>-LexAOps Promoter PCR #5 (Td-Tomato):
    -10uM FW Primer-129 (+ApaI)     2.5uL
    -10uM RV Primer-130 (+XhoI)     2.5uL
    -T-64 (tdTomato) Template DNA   0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-LexAOps Promoter PCR #5 (T-64 M-64):
    -10uM FW Primer-129 (+ApaI)     2.5uL
    -10uM RV Primer-130 (+XhoI)     2.5uL
    -T-64 M-64 Template DNA         0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Thermocycler Protocol:
    -TouchDownPhu
        -Initial Denaturation       98'C    30s
            -10 Cycles
                -Denaturation       98'C    10s
                -Annealing          55'C    20s
                -(-1'C Per Cycle)
                -Extension          72'C    30s
            -30 Cycles
                -Denaturation       98'C    10s
                -Annealing          55'C    20s
                -Extension          72'C    30s
            -Final Extension        72'C    5m
            -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel   35mL
    -SyberSafe          3.5uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps Promoter (Td-Tomato)   6.0uL   Lane #2
    -LexAOps Promoter (T-64 M-64)   6.0uL   Lane #3
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps PCR #5 30/40mins&quot;
    -Bands visible, but appears closer to 600bp rather than expected 665bp
</CODE></PRE><H3>[]LexAOps + Hy86E3 + Hy111E2 Digestion</H3><PRE><CODE>-LexAOps (Td-Tomato) Digestion:
    -Added 5.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
-LexAOps (T-64 M-64) Digestion:
    -Added 5.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
-Hy86E3 Digestion:
    -Transferred 10.0uL of plasmid into a new PCR tube
    -Added 16.0uL of water
    -Added 3.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL of XhoI
    -Vortexed
    -Incubated at 37'C overnight
-Hy111E2 Digestion:
    -Transferred 10.0uL of plasmid into a new PCR tube
    -Added 16.0uL of water
    -Added 3.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL of XhoI
    -Vortexed
    -Incubated at 37'C overnight
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Run digestion on gel
-Finish up meeting slides for 7/6/15
</CODE></PRE><H2>7/2/15</H2><P>Ran yesterday's digestion on a gel. PCRed mFa and Ste2 genes. Performed gel extraction for digestion. Ligated LexAOps with plasmid and transformed E.coli.</P><H3>[]LexAOps + Hy86E3 + Hy111E2 Digestion Part 2</H3><PRE><CODE>-Gel Loading:
    -1% Agarose Gel     35mL
    -SyberSafe          3.5uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps Promoter (Td-Tomato)   30.0uL  Lane #2
    -LexAOps Promoter (Td-Tomato)   30.0uL  Lane #3
    -LexAOps Promoter (T-64 M-64)   30.0uL  Lane #4
    -LexAOps Promoter (T-64 M-64)   30.0uL  Lane #5
    -Hy111E2 Plasmid                36.0uL  Lane #6
    -Hy86E3 Plasmid                 36.0uL  Lane #7
    -2-Log DNA Ladder               10.0uL  Lane #8
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps (Td-Tomato + M-64) + Hy111E2 + Hy86E3 30/40mins&quot;
    -Bands visible, but no bands for Hy111E2 plasmid
-LexAOps (Td + M-64) + Hy86E3 Digestion Gel Extration:
    --Excised gel bands in lanes 2, 3, 4, 5, and 7 and transferred to 1.5mL tube
    -Weighted tube
        -LexAOps (Td-Tomato):   153.0mg
        -LexAOps (T-64 M-64):   140.0mg
        -Hy86E3 Plasmid:        158.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -LexAOps (Td-Tomato):   +480.0uL
        -LexAOps (T-64 M-64):   +420.0uL
        -Hy86E3 Plasmid:        +470.0uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -LexAOps (Td-Tomato):   +153.0uL
        -LexAOps (T-64 M-64):   +140.0uL
        -Hy86E3 Plasmid:        +158.0uL
    -Transferred each sample to QIAquick column 2mL collection tubes
    -Centrifuged for 1 minute
    -Discarded flow-through, returned columns to collection tubes
    -Added 0.75mL of Buffer PE to columns and centrifuged for 1 minute
    -Discarded flow-through, returned columns to collection tubes
    -Centrifuged columns empty for 1 minute
    -Transferred columns to new 1.5mL tubes
    -Added 30uL of water to the columns to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL each sample
    -Results:
        -LexAOps (Td-Tomato):   6.259ng/uL
        -LexAOps (T-64 M-64):   8.964ng/uL
        -Hy86E3 Plasmid:        46.22ng/uL
</CODE></PRE><H3>[]LexAOps (Td + M-64) + Hy86E3 Ligation</H3><PRE><CODE>-LexAOps (Td-Tomato) + Hy86E3 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -LexAOps DNA (15ng):        2.4uL
    -Nuclease-Free Water:       13.4uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
-LexAOps (T-64 M-64) + Hy86E3 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -LexAOps DNA (15ng):        1.8uL
    -Nuclease-Free Water:       14.0uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
-Negative Control:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -Nuclease-Free Water:       15.8uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
</CODE></PRE><H3>[]LexAOps (Td + M-64) + Hy86E3 Transformation</H3><PRE><CODE>-LexAOps (Td-Tomato) + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (Td-Tomato) + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over-weekend at room temperature
-LexAOps (T-64 M-64) + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (T-64 M-64) + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over-weekend at room temperature
-Negative Control Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of negative control sample to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over-weekend at room temperature
</CODE></PRE><H3>[]mFa + Ste2 PCR</H3><PRE><CODE>--Reconstitute Primers:
    -mFa (+LexAOps) FW-131:
        -31.0 nmoles x 10 = 310uL
    -mFa (+pNH605) RV-132:
        -31.3 nmoles x 10 = 313uL
    -Ste2 (+LexAOps) FW-133:
        -32.6 nmoles x 10 = 326uL
    -Ste2 (+pNH605) RV-134:
        -28.2 nmoles x 10 = 282uL
-Dilution:
    -10uL primers + 90uL water = 10uM stock solution / 100uL
-mFa PCR:
    -10uM FW Primer-131 (+LexAOps)  2.5uL
    -10uM RV Primer-132 (+pNH605)   2.5uL
    -pTS133 Template DNA            0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Ste2 PCR:
    -10uM FW Primer-133 (+LexAOps)  2.5uL
    -10uM RV Primer-134 (+pNH605)   2.5uL
    -*Ste2(?) Template DNA          0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
    -*Unknown whether contained Ste2 gene or not
-Thermocycler Protocol:
    -Initial Denaturation       98'C    30s
    -35 Cycles
        -Denaturation           98'C    10s
        -Annealing              55'C    20s
        -Extension              72'C    45s
    -Final Extension            72'C    5m
    -Hold                       4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel       35mL
    -SyberSafe              3.5uL

    -2-log DNA Ladder           10uL    Lane #1
    -mFa PCR                    6.0uL   Lane #2
    -Ste2 PCR                   6.0uL   Lane #3
-Gel Results:
    -See Gel Folder for gel labeled &quot;mFa + Ste2 PCR&quot;
    -Good Ste2 Band
    -mFa Band faint. Attempt to reamplify with second PCR.
</CODE></PRE><H3>[]To Do:</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Perform Colony PCR
-Make Liquid Culture
-Sequence DNA
-Re-PCR mFa
-Digest mFa + Ste2
-Finish up meeting slides for 7/6/15
</CODE></PRE><H2>7/6/15</H2><P>Performed a colony PCR of our LexAOps + Hy86E3 transformed cells. Phosphotased digested Hy86E3 plasmid. Created liquid cultures. Re-PCRed mFa. Ligated and transformed third batch of E.coli cells.</P><H3>[]Colony PCR LexAOps + Hy86E3 #2</H3><PRE><CODE>-Colony PCR LexAOps + Hy86E3 #2
    -Filled 14 PCR tubes with 25uL of water
    -Created PCR Master Mix:
        -2x GoTaq Green PCR Master Mix  190.0uL
        -10uM FW Primer-129 (+ApaI)     19.0uL
        -10uM RV Primer-130 (+XhoI)     19.0uL
        -Water                          57.0uL
    -Transferred Colonies:
        -Transferred 1 colony each to 6 PCR tubes from 8x LexAOps/Hy86E3  (TD) experiment plates and mixed by pipetting up and down
        -Transferred 1 colony each to 6 PCR tubes from 8x LexAOps/Hy86E3  (M-64) experiment plates and mixed by pipetting up and down
        - Transferred 1 colony each to 2 PCR tubes from 8x (-) Control plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       95'C    5m
            -10 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -(-1'C Per Cycle)
                -Extension          72'C    1m
            -30 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -Extension          72'C    1m
            -Final Extension        72'C    10m
            -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel               100mL
    -SyberSafe                      10.0uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps + Hy86E3 (M-64) #1     5.0uL   Lane #2
    -LexAOps + Hy86E3 (M-64) #2     5.0uL   Lane #3
    -LexAOps + Hy86E3 (M-64) #3     5.0uL   Lane #4
    -LexAOps + Hy86E3 (M-64) #4     5.0uL   Lane #5
    -LexAOps + Hy86E3 (M-64) #5     5.0uL   Lane #6
    -LexAOps + Hy86E3 (M-64) #6     5.0uL   Lane #7
    -2-Log DNA Ladder               10.0uL  Lane #8
    -LexAOps + Hy86E3 (Td) #1       5.0uL   Lane #9
    -LexAOps + Hy86E3 (Td) #2       5.0uL   Lane #10
    -LexAOps + Hy86E3 (Td) #3       5.0uL   Lane #11
    -LexAOps + Hy86E3 (Td) #4       5.0uL   Lane #12
    -LexAOps + Hy86E3 (Td) #5       5.0uL   Lane #13
    -LexAOps + Hy86E3 (Td) #6       5.0uL   Lane #14
    -2-Log DNA Ladder               10.0uL  Lane #15
    -(-) Control #1                 5.0uL   Lane #16
    -(-) Control #2                 5.0uL   Lane #17

    -Ran at 100 volts for 35 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + Hy86E3 Colony PCR #2&quot;
    -No bands
</CODE></PRE><H3>[]LexAOps + Hy86E3 Liquid Culture #2</H3><PRE><CODE>-LexAOps + Hy86E3 Liquid Culture:
    -Labeled 8 culturing tube
    -Turned on bunson burner
    -Picked 4 colonies from both LexAOps + Hy86E3 (Td) 8x plate and LexAOps + Hy86E3 (M-64) 8x plate
    -Mixed 1 colony into each tube's liquid
    -Incubated overnight
</CODE></PRE><H3>[]mFa PCR #2</H3><PRE><CODE>-mFa PCR #2
    -10uM FW Primer-131 (+LexAOps)  2.5uL
    -10uM RV Primer-132 (+pNH605)   2.5uL
    -mFa PCR #1 Sample              2.0uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-mFa PCR Redo
    -10uM FW Primer-131 (+LexAOps)  2.5uL
    -10uM RV Primer-132 (+pNH605)   2.5uL
    -pTS133 Template DNA            0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Thermocycler Protocol:
    -Initial Denaturation       98'C    30s
    -35 Cycles
        -Denaturation           98'C    10s
        -Annealing              55'C    20s
        -Extension              72'C    45s
    -Final Extension            72'C    5m
    -Hold                       4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel       35mL
    -SyberSafe              3.5uL

    -2-log DNA Ladder           10uL    Lane #1
    -Empty                      0.0uL   Lane #2
    -mFa PCR Redo               6.0uL   Lane #3
    -mFa PCR #2                 6.0uL   Lane #4
-Gel Results:
    -See Gel Folder for gel labeled &quot;mFa PCR #2&quot;
    -Bands appear much smaller than anticipated
</CODE></PRE><H3>[]Phosphotase Treatment</H3><PRE><CODE>-Hy86E3 Plasmid Digest:
    -Added 1.0uL to digest tube
    -Incubated at 37'C for 1 hour
    -Incubated at 65'C for 10 mins
</CODE></PRE><H3>[]LexAOps + Hy86E3 Ligation:</H3><PRE><CODE>-LexAOps (Td-Tomato) + Hy86E3 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -LexAOps DNA (15ng):        2.4uL
    -Nuclease-Free Water:       13.4uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
-LexAOps (T-64 M-64) + Hy86E3 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -LexAOps DNA (15ng):        1.8uL
    -Nuclease-Free Water:       14.0uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
-LexAOps (Old Digest 6/25/15) + Hy86E3 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -LexAOps DNA (15ng):        2.0uL
    -Nuclease-Free Water:       13.8uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
-Negative Control:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         1.2uL
    -Nuclease-Free Water:       15.8uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Mixed reaction by pipetting up and down
    -Incubated at room temperature for 1 hour
</CODE></PRE><H3>[]LexAOps + Hy86E3 Transformation</H3><PRE><CODE>-LexAOps (Td-Tomato) + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (Td-Tomato) + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
-LexAOps (T-64 M-64) + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (T-64 M-64) + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
-LexAOps (Old Digest) + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (Old Digest) + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
-Negative Control Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of negative control sample to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
</CODE></PRE><H3>[]To Do:</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Miniprep
-Sequence DNA
-Re-PCR mFa
</CODE></PRE><H2>7/7/15</H2><P>Re-ran mFa on a gel. Minipreped liquid cultures from yesterday. PCRed miniprep samples. Colony PCRed transformation #3.</P><H3>[]mFa PCR</H3><PRE><CODE>-Gel Loading:
    -1.5% Agarose Gel       35mL
    -SyberSafe              3.5uL

    -2-log DNA Ladder           10uL    Lane #1
    -mFa PCR #2                 6.0uL   Lane #2
    -mFa PCR Redo               6.0uL   Lane #3
-Gel Results:
    -See Gel Folder for gel labeled &quot;mFa PCR #2_1&quot;
    -Bands appear much smaller than anticipated
</CODE></PRE><H3>[]LexAOps + Hy86E3 Miniprep</H3><PRE><CODE>-Error:
    -All LexAOps + Hy86E3 (T-64 M-64) samples and LexAOps + Hy86E3 (Td-Tomato) #2) had no culture growth.
-LexAOps + Hy86E3 Miniprep:
    -Centrifuged liquid cultures for 10 mins
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to 1.5mL tubes
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into a QIAprep spin column
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed column in a new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of each tube
        -LexAOps + Hy86E3 (Td-Tomato) #1        786.3ng/uL
        -LexAOps + Hy86E3 (Td-Tomato) #3        554.1ng/uL
        -LexAOps + Hy86E3 (Td-Tomato) #4        744.9ng/uL
</CODE></PRE><H3>[]Miniprep PCR</H3><PRE><CODE>-LexAOps + Hy86E3 (Td-Tomato) #1 PCR:
    -10uM FW Primer-129 (+ApaI)                 2.5uL
    -10uM RV Primer-130 (+XhoI)                 2.5uL
    -LexAOps + Hy86E3 (Td-Tomato) Miniprep DNA  0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-LexAOps + Hy86E3 (Td-Tomato) #3 PCR:
    -10uM FW Primer-129 (+ApaI)                 2.5uL
    -10uM RV Primer-130 (+XhoI)                 2.5uL
    -LexAOps + Hy86E3 (Td-Tomato) Miniprep DNA  0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-LexAOps + Hy86E3 (Td-Tomato) #4 PCR:
    -10uM FW Primer-129 (+ApaI)                 2.5uL
    -10uM RV Primer-130 (+XhoI)                 2.5uL
    -LexAOps + Hy86E3 (Td-Tomato) Miniprep DNA  0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-Gel Loading:
    -Loaded on &quot;LexAOps + Hy86E3 Colony PCR #3&quot; gel and &quot;Td Colony PCR + mFa #4&quot;
-Gel Results:
    -See respective gels for results
</CODE></PRE><H3>[]LexAOps + Hy86E3 Colony PCR #3</H3><PRE><CODE>-LexAOps + Hy86E3 Colony PCR #3:
    -Filled 20 PCR tubes with 25uL of water
    -Created PCR Master Mix:
        -2x GoTaq Green PCR Master Mix  210.0uL
        -10uM FW Primer-129 (+ApaI)     21.0uL
        -10uM RV Primer-130 (+XhoI)     21.0uL
        -Water                          63.0uL
    -Transferred Colonies:
        -Transferred 1 colony each to 6 PCR tubes from 8x LexAOps/Hy86E3  (Td-Tomato) experiment plate and mixed by pipetting up and down
        -Transferred 1 colony each to 6 PCR tubes from 8x LexAOps/Hy86E3  (T-64 M-64) experiment plate and mixed by pipetting up and down
        -Transferred 1 colony each to 6 PCR tubes from 8x LexAOps/Hy86E3  (Old Digest) experiment plate and mixed by pipetting up and down
        - Transferred 1 colony each to 2 PCR tubes from 8x (-) Control plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       95'C    5m
            -10 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -(-1'C Per Cycle)
                -Extension          72'C    1m
            -30 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -Extension          72'C    1m
            -Final Extension        72'C    10m
            -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel               100mL
    -SyberSafe                      10.0uL

    -DNA Loading Dye (Error)                    10.0uL  Lane #1
    -2-Log DNA Ladder                           10.0uL  Lane #2
    -Miniprep LexAOps/Hy86E3  (Td-Tomato) #1    6.0uL   Lane #3
    -Miniprep LexAOps/Hy86E3  (Td-Tomato) #3    6.0uL   Lane #4
    -Miniprep LexAOps/Hy86E3  (Td-Tomato) #4    6.0uL   Lane #5
    -LexAOps/Hy86E3  (T-64 M-64) #1             6.0uL   Lane #6
    -LexAOps/Hy86E3  (T-64 M-64) #2             6.0uL   Lane #7
    -LexAOps/Hy86E3  (T-64 M-64) #3             6.0uL   Lane #8
    -LexAOps/Hy86E3  (T-64 M-64) #4             6.0uL   Lane #9
    -LexAOps/Hy86E3  (T-64 M-64) #5             6.0uL   Lane #10
    -LexAOps/Hy86E3  (T-64 M-64) #6             6.0uL   Lane #11
    -(-) Control #1                             6.0uL   Lane #12
    -(-) Control #2                             6.0uL   Lane #13
    -LexAOps/Hy86E3  (Old Digest) #1            6.0uL   Lane #14
    -LexAOps/Hy86E3  (Old Digest) #2            6.0uL   Lane #15
    -LexAOps/Hy86E3  (Old Digest) #3            6.0uL   Lane #16
    -LexAOps/Hy86E3  (Old Digest) #4            6.0uL   Lane #17
    -LexAOps/Hy86E3  (Old Digest) #5            6.0uL   Lane #18
    -Empty                                      0.0uL   Lane #19
    -2-Log DNA Ladder                           10.0uL  Lane #20

    -Ran at 100 volts for 35 mins
    -Errors:
        -Accidently loaded DNA Loading Dye into lane #1
        -LexAOps/Hy86E3 (Old Digest) #6 disappeared in the tube.
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + Hy86E3 Colony PCR #3&quot; 
    -Bands visible, but uncertain if correct size.
    -Negative control had bands.
</CODE></PRE><H3>[]DNA Sequencing</H3><PRE><CODE>-Sent samples: &quot;Miniprep LexAOps/Hy86E3  (Td-Tomato) #1,3,4&quot; to DNA sequencing.
</CODE></PRE><H3>[]LexAOps + Hy86E3 Liquid Culture</H3><PRE><CODE>-LexAOps + Hy86E3 (T-64 M-64 #3) Liquid Culture:
    -Labeled a culturing tube
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tube
    -Transfered 20uL of cell solution from colony PCR to culturing tube
    -Incubated overnight
-LexAOps + Hy86E3 (Old Digest #1-2) Liquid Culture:
    -Labeled 2 culturing tubes
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to each tube
    -Transfered 20uL of cell solutions from colony PCR to culturing tubes
    -Incubated overnight
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-PCR mFa with pGEM41
-Miniprep
-DNA Sequence
</CODE></PRE><H2>7/8/15</H2><P>PCRed mFa again. Minipreped liquid cultures from yesterday. Ran a gel with minipreped DNA and mFa. Sent minipreped DNA to sequencing. </P><H3>[]mFa PCR #3</H3><PRE><CODE>-mFa PCR #3
    -10uM FW Primer-131 (+LexAOps)  2.5uL
    -10uM RV Primer-132 (+pNH605)   2.5uL
    -pGEM41 Template DNA            0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Thermocycler Protocol:
    -Initial Denaturation       98'C    30s
    -35 Cycles
        -Denaturation           98'C    10s
        -Annealing              55'C    20s
        -Extension              72'C    45s
    -Final Extension            72'C    5m
    -Hold                       4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel       35mL
    -SyberSafe              3.5uL

    -2-log DNA Ladder           10.0uL  Lane #1
    -2-log DNA Ladder           10.0uL  Lane #2
    -Td-Tomato Colony PCR #1    6.0uL   Lane #3
    -Td-Tomato Colony PCR #2    6.0uL   Lane #4
    -Td-Tomato Colony PCR #3    6.0uL   Lane #5
    -Td-Tomato Colony PCR #4    6.0uL   Lane #6
    -Td-Tomato Colony PCR #5    6.0uL   Lane #7
    -Td-Tomato Colony PCR #6    6.0uL   Lane #8
    -mFa PCR #3                 6.0uL   Lane #9
    -2-log DNA Ladder           10.0uL  Lane #10
-Gel Results:
    -See Gel Folder for gel labeled &quot;Td Colony PCR + mFa #3&quot;
    -Bands for Colony PCR seem ok
    -mFa bands appears much smaller than expected
</CODE></PRE><H3>[]LexAOps (M-64 + Old Digest) Liquid Cultures</H3><PRE><CODE>-LexAOps (M-64 + Old Digest) Liquid Cultures:
    -Centrifuged liquid cultures for 10 mins
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to 1.5mL tubes
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into a QIAprep spin column
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed column in a new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of each tube
        -LexAOps + Hy86E3 (Old Digest) #1       31.30ng/uL
        -LexAOps + Hy86E3 (Old Digest) #2       20.67ng/uL
        -LexAOps + Hy86E3 (T-64 M-64) #3        12.13ng/uL
-DNA Sequencing:
    -Send DNA to Quintara
    -See Folder &quot;LexAOps + Hy86E3 Plasmid #3&quot;</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Toss old primers and dilute new working stock
-PCR LexAOps Again
-Digest LexAOps + Hy111E2 + Hy86E3 
</CODE></PRE><H2>7/9/15</H2><P>Restarted LexAOps PCR with fresh reagents. Digested new LexAOps PCR and Hy111E2 and Hy86E3 plasmids.</P><H3>[]LexAOps PCR #6</H3><PRE><CODE>-Diluted New Primers for Working Stock
    -Tossed old Primers
    -Transferred 90uL water to two 1.5mL tubes
    -Transferred 10uL concentrate stock to tubes
        -129 FW
        -130 RV
-LexAOps PCR #6
    -10uM FW Primer-129 (+ApaI)                 2.5uL
    -10uM RV Primer-130 (+XhoI)                 2.5uL
    -T-64 (tdTomato) Template DNA               0.5uL       
    -2x GoTaq Master Mix                        25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
    -Made 8 samples
-Thermocycler Protocol:
    -TouchDown_Go
        -Initial Denaturation       95'C    2m
            -10 Cycles
                -Denaturation       95'C    30s
                -Annealing          55'C    30s
                -(-1'C Per Cycle)
                -Extension          72'C    1m
            -30 Cycles
                -Denaturation       95'C    30s
                -Annealing          55'C    30s
                -Extension          72'C    1m
            -Final Extension        72'C    5m
            -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel       72mL
    -SyberSafe              7.5uL

    -2-log DNA Ladder           10.0uL  Lane #1
    -LexAOps PCR #6 (#1)        10.0uL  Lane #2
    -LexAOps PCR #6 (#2)        5.0uL   Lane #3
    -LexAOps PCR #6 (#3)        5.0uL   Lane #4
    -LexAOps PCR #6 (#4)        5.0uL   Lane #5
    -LexAOps PCR #6 (#5)        5.0uL   Lane #6
    -LexAOps PCR #6 (#6)        5.0uL   Lane #7
    -LexAOps PCR #6 (#7)        5.0uL   Lane #8
    -LexAOps PCR #6 (#8)        5.0uL   Lane #9
    -2-log DNA Ladder           10.0uL  Lane #10
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps PCR #6 45mins&quot;
    -Bands appear to be right size
</CODE></PRE><H3>[]LexAOps + Hy111E2 + Hy86E3 Digestion #4 Part 1</H3><PRE><CODE>-LexAOps Digestion:
    -Added 5.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
-Hy111E2 Digestion:
    -Added 10.0uL of Hy111E2 Plasmid
    -Added 3.0uL of CutSmart Buffer
    -Added 16.0uL of Water
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
-Hy86E3 Digestion:
    -Added 10.0uL of Hy86E3 Plasmid
    -Added 3.0uL of CutSmart Buffer
    -Added 16.0uL of Water
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
-ERROR:
    -LexAOps #1 Digestion got 5uL of ApaI instead of 0.5uL
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Run Digest on a gel
-Gel Extract
-Ligation
</CODE></PRE><H2>7/10/15</H2><P>Ran yesterday's digest on a gel. Extract digest from gel. Worked on slides for Monday's meeting.</P><H3>[]LexAOps + Hy111E2 + Hy86E3 Digestion #4 Part 2</H3><PRE><CODE>-Gel Loading #1:
    -1.5% Agarose Gel   100mL
    -SyberSafe          10.0uL

    -2-Log DNA Ladder                       10.0uL  Lane #1
    -LexAOps Promoter Digest (Td) #1        30.0uL  Lane #2
    -LexAOps Promoter Digest (Td) #1        19.0uL  Lane #3
    -LexAOps Promoter Digest (Td) #2        30.0uL  Lane #4
    -LexAOps Promoter Digest (Td) #2        19.0uL  Lane #5
    -LexAOps Promoter Digest (Td) #3        30.0uL  Lane #6
    -LexAOps Promoter Digest (Td) #3        20.0uL  Lane #7
    -LexAOps Promoter Digest (Td) #4        30.0uL  Lane #8
    -LexAOps Promoter Digest (Td) #4        20.0uL  Lane #9
    -LexAOps Promoter Digest (Td) #5        30.0uL  Lane #10
    -LexAOps Promoter Digest (Td) #5        20.0uL  Lane #11
    -LexAOps Promoter Digest (Td) #6        30.0uL  Lane #12
    -LexAOps Promoter Digest (Td) #6        20.0uL  Lane #13
    -2-log DNA Ladder                       10.0uL  Lane #14
    -Jeffery's Sample                       35.0uL  Lane #15
-Gel Results #1:
    -See Gel Folder for gel labeled &quot;LexAOps Digestion #4&quot;
    -Some bands are bright, some are faint
    -Slight middle gel curve effect
--Gel Loading #2:
    -1.5% Agarose Gel   50mL
    -SyberSafe          5.0uL

    -2-Log DNA Ladder                       10.0uL  Lane #1
    -Hy86E3 Digestion                       36.0uL  Lane #2
    -Hy111E2 Digestion                      32.0uL  Lane #3
    -LexAOps Promoter Digest (Td) #7        17.0uL  Lane #4
    -LexAOps Promoter Digest (Td) #8        30.0uL  Lane #5
    -LexAOps Promoter Digest (Td) #8        13.6uL  Lane #6
    -2-log DNA Ladder                       10.0uL  Lane #7
-Gel Results #2:
    -See Gel Folder for gel labeled &quot;LexAOps Digestion #4&quot;
    -Promoter bands are fairly bright
    -Hy111E2 band extremely bright
    -pTET band appears, but is faint
        -&gt;Re-Digest plasmids
</CODE></PRE><H3>[]LexAOps + Hy86E3 + Hy111E2 Gel Extraction</H3><PRE><CODE>-Gel #1:
    -Excised gel bands in lanes 6-7 and 8-9 and transferred each to 1.5mL tubes
    -Weighted tubes
        -LexAOps Extract #3:    113.0mg
        -LexAOps Extract #4:    133.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -LexAOps Extract #3:    +340uL
        -LexAOps Extract #4:    +400uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -LexAOps Extract #3:    +113uL
        -LexAOps Extract #4:    +133uL
    -Transferred each sample to a QIAquick column in 2mL collection tubes
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tubes
    -Added 0.75mL of Buffer PE to columns and centrifuged for 1 minute
    -Discarded flow-through, returned columns to collection tubes
    -Centrifuged columns empty for 1 minute
    -Transferred columns to new 1.5mL tubes
    -Added 30uL of water to the columns to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -LexAOps Extract #3:    17.28ng/uL
        -LexAOps Extract #4:    17.51ng/uL
-Gel #2:
    -Excised gel bands in lanes 1 and 2 and transferred each to 1.5mL tubes
    -Weighted tubes
        -Hy86E3 Extract:    34.0mg
        -Hy111E2 Extract:   75.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -Hy86E3 Extract:    +100uL
        -Hy111E2 Extract:   +230uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -Hy86E3 Extract:    +34uL
        -Hy111E2 Extract:   +75uL
    -Transferred each sample to a QIAquick column in 2mL collection tubes
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tubes
    -Added 0.75mL of Buffer PE to columns and centrifuged for 1 minute
    -Discarded flow-through, returned columns to collection tubes
    -Centrifuged columns empty for 1 minute
    -Transferred columns to new 1.5mL tubes
    -Added 30uL of water to the columns to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -Hy86E3 Extract:    19.18ng/uL
        -Hy111E2 Extract:   71.52ng/uL
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Re-Digest Plasmids
-Gel Extract Again
-Ligation
</CODE></PRE><H2>7/13/15</H2><P>Presented week's progress in group meeting. Re-digested plasmids.</P><H3>[]Hy86E3 + Hy111E2 Re-Digestion Part 1</H3><PRE><CODE>-Hy86E3 Digestion:
    -Added 6.0uL CutSmart Buffer
    -Added 1.0uL ApaI
    -Vortexed
    -Incubated at room temperature for 2 hours
    -Added 1.0uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
-Hy111E2 Digestion:
    -Added 6.0uL CutSmart Buffer
    -Added 1.0uL ApaI
    -Vortexed
    -Incubated at room temperature for 2 hours
    -Added 1.0uL XhoI
    -Vortexed
    -Incubated at 37'C overnight
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Load Digest on gel
-Gel Extract
-Ligation
-Transformation
</CODE></PRE><H2>7/14/15</H2><P>Ran yesterday's digest on a gel and extracted. Phosphotase treated digested plasmids and then ligated plasmid with insert. Transformed E.coli and plated.</P><H3>[]Hy86E3 + Hy111E2 Re-Digestion Part 2</H3><PRE><CODE>-Gel Loading:
    -1.5% Agarose Gel   50mL
    -SyberSafe          5.0uL

    -2-Log DNA Ladder                       10.0uL  Lane #1
    -Empty                                  0.0uL   Lane #2
    -Hy86E3 Re-Digestion #1                 35.0uL  Lane #3
    -Hy86E3 Re-Digestion #2                 31.0uL  Lane #4
    -Empty                                  0.0uL   Lane #5
    -Hy111E2 Re-Digestion #1                35.0uL  Lane #6
    -Hy111E2 Re-Digestion #2                30.0uL  Lane #7
    -2-log DNA Ladder                       10.0uL  Lane #8

    -Ran at 75V for 40 minutes
-Gel Results #2:
    -See Gel Folder for gel labeled &quot;Hy86E3 + Hy111E2 Re-Digest&quot;
    -No visible bands for pTET promoter
        -Potentially still did not properly digest
    -Faint bands for Hy86E3
</CODE></PRE><H3>[]Hy111E2 Gel Extraction</H3><PRE><CODE>-Hy111E2 Gel Extraction:
    -Excised gel bands in lanes 6-7 and transferred to a 1.5mL tube
    -Weighted tube
        -Hy111E2 Extract:       194.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -Hy111E2 Extract:       +580uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -Hy111E2 Extract:       +194uL
    -Transferred sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to a new 1.5mL tube
    -Added 30uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -Hy111E2 Extract:   20.84ng/uL
</CODE></PRE><H3>[]Hy111E2 Phosphotase Treat</H3><PRE><CODE>-Added 1uL of Antarctic Phosphatase to purified tube
-Incubated at 37'C for 1 hour
-Heat Inactivated at 70'C for 10 mins
</CODE></PRE><H3>[]LexAOps + Hy111E2 Ligation</H3><PRE><CODE>-LexAOps #3 + Hy111E2 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         2.6uL
    -LexAOps #3 DNA (15ng):     1.0uL
    -Nuclease-Free Water:       13.4uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL
-LexAOps #4 + Hy111E2 Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         2.6uL
    -LexAOps #4 DNA (15ng):     1.0uL
    -Nuclease-Free Water:       13.4uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL
-(-) Control Ligation:
    -10x T4 DNA Ligase Buffer:  2.0uL
    -Hy86E3 DNA (55ng):         2.6uL
    -Nuclease-Free Water:       14.4uL
    -T4 DNA Ligase:             1.0uL
                    Total:      20.0uL

    -Incubated at room temperature for 1 hour
</CODE></PRE><H3>[]LexAOps + Hy111E2 Transformation</H3><PRE><CODE>-LexAOps (#3) + Hy111E2 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (#3) + Hy111E2 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
-LexAOps (#4) + Hy111E2 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of ligased LexAOps (#4) + Hy111E2 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
-Negative Control Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 5uL of negative control sample to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates overnight
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Colony PCR
</CODE></PRE><H2>7/15/15</H2><P>Colony PCRed our plates. Liquid cultured colonies. PCRed new LexAOps promoters with gibson overhangs. Re-PCRed mFa using a gradient thermocycler protocol. Liquid cultured colony PCRs.</P><H3>[]LexAOps #6 Colony PCR</H3><PRE><CODE>-LexAOps #6 Colony PCR
    -Filled 5 PCR tubes with 25uL of water
    -Created PCR Master Mix:
        -2x GoTaq Green PCR Master Mix  60.0uL
        -10uM FW Primer-129 (+ApaI)     6.0uL
        -10uM RV Primer-130 (+XhoI)     6.0uL
        -Water                          18.0uL
    -Transferred Colonies:
        -Transferred 1 colony each to 3 PCR tubes from 8x LexAOps/Hy111E2 #3 experiment plate and mixed by pipetting up and down
        -Transferred 1 colony to 1 PCR tubes from 8x LexAOps/Hy111E2 #4 experiment plate and mixed by pipetting up and down
        - Transferred 1 colony to 1 PCR tubes from 8x (-) Control plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       98'C    30m
            -10 Cycles
                -Denaturation       98'C    10s
                -Annealing          55'C    20s
                -(-1'C Per Cycle)
                -Extension          72'C    30s
            -30 Cycles
                -Denaturation       98'C    10s
                -Annealing          55'C    20s
                -Extension          72'C    30s
            -Final Extension        72'C    5m
            -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel               100mL
    -SyberSafe                      10.0uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps Colony PCR #1          10.0uL  Lane #2
    -LexAOps Colony PCR #2          10.0uL  Lane #3
    -LexAOps Colony PCR #3          10.0uL  Lane #4
    -LexAOps Colony PCR #4          10.0uL  Lane #5
    -Empty                          10.0uL  Lane #6
    -(-) Control                    10.0uL  Lane #7
    -2-log DNA Ladder               10.0uL  Lane #8

    -Ran at 90 volts for 45 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps #6 Colony PCR&quot; 
    -Bands visible and clear
    -Negative control had faint bands.
</CODE></PRE><H3>[]LexAOps #6 Liquid Culture</H3><PRE><CODE>-LexAOps #6 Liquid Cultures:
    -Labeled 4 culturing tubes
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tubes
    -Transfered 20uL of cell solution from colony PCRs to culturing tubes
    -Incubated overnight
</CODE></PRE><H3>[]LexAOps PCR Gibson</H3><PRE><CODE>-Reconstitute Primers:
    -LexAOps (+GFP) RV #147: 27.5 nmoles x 10 = 275uL
    -LexAOps (+mFa) RV #148: 27.8 nmoles x 10 = 278uL
    -LexAOps (+Ste2) RV #149: 27.2 nmoles x 10 = 272uL
    -LexAOps (+pNH605) FW #150: 28.4 nmoles x 10 = 284uL
-Dilution for Working Stock:
    -Transferred 10uL of each primer to a new tube and added 90uL of water
-LexAOps (+GFP) PCR:
    -10uM FW Primer-150 (+pNH605)               2.5uL
    -10uM RV Primer-147 (+GFP)                  2.5uL
    -T-64 (tdTomato) Template DNA               0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-LexAOps (+mFa) PCR:
    -10uM FW Primer-150 (+pNH605)               2.5uL
    -10uM RV Primer-148 (+mFa)                  2.5uL
    -T-64 (tdTomato) Template DNA               0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-LexAOps (+Ste2) PCR:
    -10uM FW Primer-150 (+pNH605)               2.5uL
    -10uM RV Primer-149 (+Ste2)                 2.5uL
    -T-64 (tdTomato) Template DNA               0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-Thermocycler Protocol:
    -Initial Denaturation       98'C    30m
        -10 Cycles
            -Denaturation       98'C    10s
            -Annealing          55'C    20s
            -(-1'C Per Cycle)
            -Extension          72'C    30s
        -30 Cycles
            -Denaturation       98'C    10s
            -Annealing          55'C    20s
            -Extension          72'C    30s
        -Final Extension        72'C    5m
        -Hold                   4'C     Forever
</CODE></PRE><H3>[]mFa PCR #4</H3><PRE><CODE>-mFa PCR #4 (8X):
    -10uM FW Primer-131 (+LexAOps)  2.5uL
    -10uM RV Primer-132 (+pNH605)   2.5uL
    -pGEM41 Template DNA            0.5uL       
    -2x Phusion Master Mix          25.0uL
    -Water                          19.5uL
                        Total:      50.0uL
-Thermocycler Protocol (PhuGradient):
    -Initial Denaturation       98'C    30s
    -35 Cycles
        -Denaturation           98'C    10s
        -Annealing              55-65'C 20s
        -Extension              72'C    45s
    -Final Extension            72'C    5m
    -Hold                       4'C     Forever
</CODE></PRE><H3>[]To Do:</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Run gel for LexAOps Gibson samples
-Run gel for mFa PCR #4 samples
-Miniprep LexAOps PCR #6 samples
</CODE></PRE><H2>7/16/15</H2><P>Ran gels for LexAOps Gibson and mFa PCR #4 samples. Minipreped LexAOps #6. Sent samples for DNA samples. Re-PCRed LexAOps (+Ste2) and re-ran mFa on a gel.</P><H3>[]LexAOps Gibson + mFa PCR #4 Gel</H3><PRE><CODE>-Gel Loading
    -1.5% Agarose Gel               100mL
    -SyberSafe                      10.0uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps (+GFP)                 6.0uL   Lane #2
    -LexAOps (+mFa)                 6.0uL   Lane #3
    -LexAOps (+Ste2)                6.0uL   Lane #4
    -Empty                          0.0uL   Lane #5
    -mFa #1                         6.0uL   Lane #6
    -mFa #2                         6.0uL   Lane #7
    -mFa #3                         6.0uL   Lane #8
    -mFa #4                         6.0uL   Lane #9
    -mFa #5                         6.0uL   Lane #10
    -mFa #6                         6.0uL   Lane #11
    -mFa #7                         6.0uL   Lane #12
    -Empty                          0.0uL   Lane #13
    -mFa #8                         6.0uL   Lane #14
    -Empty                          0.0uL   Lane #15
    -2-log DNA Ladder               10.0uL  Lane #16

    -Ran at 100 volts for 40 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps Gibson + mFa PCR #4 40mins&quot; 
    -Bands are a bit faint
    -LexAOps (+Ste2) band is very faint
        -Re-PCR
    -mFa bands #1-3 appear to have the brightest bands at 500bp
        -Re-Run on new gel and extract
</CODE></PRE><H3>[]LexAOps (+Ste2) PCR #2</H3><PRE><CODE>-LexAOps (+Ste2) PCR #2
    -10uM FW Primer-150 (+pNH605)               2.5uL
    -10uM RV Primer-149 (+Ste2)                 2.5uL
    -LexAOps Digest #3 (7-13-15)                0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-Thermocycler Protocol:
    -Initial Denaturation       98'C    30m
        -10 Cycles
            -Denaturation       98'C    10s
            -Annealing          55'C    20s
            -(-1'C Per Cycle)
            -Extension          72'C    30s
        -30 Cycles
            -Denaturation       98'C    10s
            -Annealing          55'C    20s
            -Extension          72'C    30s
        -Final Extension        72'C    5m
        -Hold                   4'C     Forever
</CODE></PRE><H3>[]mFa PCR #4 Gel Loading #2</H3><PRE><CODE>-Gel Loading
    -4% Agarose Gel                 30mL
    -SyberSafe                      3.0uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -mFa #1 (1)                     27.0uL  Lane #2
    -mFa #1 (2)                     14.0uL  Lane #3
    -mFa #2 (1)                     20.0uL  Lane #4
    -mFa #2 (2)                     20.0uL  Lane #5
    -mFa #3 (1)                     20.0uL  Lane #6
    -mFa #3 (2)                     20.0uL  Lane #7
    -2-Log DNA Ladder               10.0uL  Lane #8

    -Ran at 100 volts for 45 mins
    -Ran an extra 30 mins at 120 volts
-Gel Results:
    -See Gel Folder for gel labeled &quot;mFa PCR #4 Gel #2&quot;</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Run LexAOps (+Ste2) on a gel
-Extract mFa
</CODE></PRE><H2>7/17/15</H2><P>Ran a gel for LexAOps (+Ste2). Extracted mFa from gel. Digested Hy86E3 plasmids. PCR purified samples. Performed Gibson Assembly. Transformed cells and plated.</P><H3>[]LexAOps (+Ste2) Gel</H3><PRE><CODE>-Gel Loading:
    -1.5% Agarose Gel               35mL
    -SyberSafe                      5.0uL

    -2-Log DNA Ladder               10.0uL  Lane #1
    -LexAOps (+Ste2)                6.0uL   Lane #2
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps (+Ste2) PCR #2&quot;
    -Good bright band
</CODE></PRE><H3>[]mFa Gel Extraction</H3><PRE><CODE>-mFa Gel Extraction:
    -Excised gel bands in lanes 4-7 and transferred to a 1.5mL tube
    -Weighted tube
        -mFa Extract:       242.0mg
    -Added Buffer QG in a 6:1 ratio (100mg ~ 100uL)
        -mFa Extract:       +1452uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -mFa Extract:       +242uL
    -Transferred sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to a new 1.5mL tube
    -Added 30uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -mFa Extract:   42.48ng/uL
</CODE></PRE><H3>[]Hy86E3 Plasmid Digestion</H3><PRE><CODE>-Hy86E3 Plasmids Digestion:
    -Transferred 10.0uL of plasmid into a new PCR tube
    -Added 16.0uL of water
    -Added 3.0uL of CutSmart Buffer
    -Added 0.5uL of ApaI
    -Vortexed
    -Incubated at room temperature for 1 hour
    -Added 0.5uL of NotI
    -Vortexed
    -Incubated at 37'C for 1 hour
-Gel Loading:
    -1% Agarose Gel         35mL
    -SyberSafe              3.5uL

    -2-Log DNA Ladder       10.0uL  Lane #2
    -Empty                  0.0uL   Lane #3
    -Hy86E3 Digestion       36.0uL  Lane #4
-Gel Results:
    -See Gel Folder for gel labeled &quot;Hy86E3 Gibson Digest&quot;
    -Good plasmid band
    -Excised pTET and GFP band is visible
-Gel Extraction:
    -Excised gel bands in lane 4 and transferred to a 1.5mL tube
    -Weighted tube
        -Hy86E3 Extract:        193.0mg
    -Added Buffer QG in a 3:1 ratio (100mg ~ 100uL)
        -Hy86E3 Extract:        +580uL
    -Incubated at 50'C for 10 minutes
    -Added isopropanol in a 1:1 ratio
        -Hy86E3 Extract:        +193uL
    -Transferred sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to a new 1.5mL tube
    -Added 30uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -mFa Extract:   47.32ng/uL
</CODE></PRE><H3>[]PCR Purification:</H3><PRE><CODE>-PCR Purification [LexAOps (+mFa), LexAOps (+Ste2), LexAOps (+GFP), Ste2 (+LexAOps)]
    -Added Buffer PB in a 5:1 ratio and mix
        -LexAOps (+mFa):        +450uL
        -LexAOps (+Ste2):       +750uL*
        -LexAOps (+GFP):        +750uL*
        -Ste2 (+LexAOps):       +750uL*

        -*ERROR
    -Transferred sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to a new 1.5mL tube
    -Added 50uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -LexAOps (+mFa):        49.27ng/uL
        -LexAOps (+Ste2):       114.8ng/uL
        -LexAOps (+GFP):        30.62ng/uL
        -Ste2 (+LexAOps):       101.5ng/uL
</CODE></PRE><H3>[]Gibson Assembly LexAOps + (mFa/Ste2/GFP) + pNH605</H3><PRE><CODE>-Protocol:
    -LexAOps + mFa + Hy86E3
        -Vector DNA (50ng):     1.06uL
        -LexAOps DNA (15ng):    0.30uL
        -mFa DNA (10ng):        0.24uL
        -2x Gibson Master Mix:  5.00uL
        -Water:                 3.40uL
                    Total:      10.0uL
    -LexAOps + Ste2 + Hy86E3
        -Vector DNA (50ng):     1.06uL
        -LexAOps DNA (15ng):    0.13uL
        -Ste2 DNA (10ng):       0.30uL
        -2x Gibson Master Mix:  5.00uL
        -Water:                 3.51uL
                    Total:      10.0uL
    -LexAOps + GFP + Hy111E2
        -Vector DNA (50ng):     2.40uL
        -LexAOps DNA (15ng):    0.50uL
        -2x Gibson Master Mix:  5.00uL
        -Water:                 2.10uL
                    Total:      10.0uL
    -(-) Control
        -Vector DNA (50ng):     1.10uL
        -2x Gibson Master Mix:  5.00uL
        -Water:                 3.90uL
                    Total:      10.0uL
    -(+) Control
        -Positive Control Mix:  5.00uL
        -x Gibson Master Mix:   5.00uL
                    Total:      10.0uL
-Incubation:
    -Incubated in thermocycler at 50'C for 60 mins
    -Hold at 4'C forever
</CODE></PRE><H3>[]LexAOps + (mFa/Ste2/GFP) + pNH605 Transformation</H3><PRE><CODE>-LexAOps + mFa + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned LexAOps + mFa + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
-LexAOps + Ste2 + Hy86E3 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned LexAOps + Ste2 + Hy86E3 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
-LexAOps + GFP + Hy111E2 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned LexAOps + GFP + Hy111E2 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
-(-) Control Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned (-) Control to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
-(+) Control Transformation
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned (+) Control to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Colony PCR plates
-Liquid Culture
</CODE></PRE><H2>7/20/15</H2><P>We Colony PCRed our plates and ran them on a gel. We also liquid cultured the one sample that appeared to be correct. We then performed a second colony PCR for the samples that didn't work.</P><H3>[]LexAOps + mFa/Ste2/GFP + Hy86E3/Hy111E2 Colony PCR</H3><PRE><CODE>-LexAOps + mFa/Ste2/GFP + Hy86E3/Hy111E2 Colony PCR:
    -Filled 18 PCR tubes with 25uL of water
    -Created PCR Master Mixes:
        -LexAOps + mFa:
            -2x GoTaq Green PCR Master Mix              70.0uL
            -10uM FW Primer-150 [LexAOps (+pNH605)]     7.0uL
            -10uM RV Primer-132 [mFa (+pNH605)]         7.0uL
            -Water                                      21.0uL
        -LexAOps + Ste2:
            -2x GoTaq Green PCR Master Mix              70.0uL
            -10uM FW Primer-150 [LexAOps (+pNH605)]     7.0uL
            -10uM RV Primer-134 [Ste2 (+pNH605)]        7.0uL
            -Water                                      21.0uL
        -LexAOps (+GFP):
            -2x GoTaq Green PCR Master Mix              70.0uL
            -10uM FW Primer-150 [LexAOps (+pNH605)]     7.0uL
            -10uM RV Primer-147 [LexAOps (+GFP)]        7.0uL
            -Water                                      21.0uL
    -Transferred Colonies:
        -Transferred 1 colony each to 6 PCR tubes from 1x LexAOps+mFa+Hy86E3 experiment plate and mixed by pipetting up and down
        -Transferred 1 colony to 6 PCR tubes from 1x LexAOps+Ste2+Hy86E3 experiment plate and mixed by pipetting up and down
        - Transferred 1 colony to 6 PCR tubes from 1x LexAOps+GFP+Hy111E2 experiment plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       95'C    5m
            -30 Cycles
                -Denaturation       95'C    45s
                -Annealing          65'C    30s
                -Extension          72'C    2m
            -Final Extension        72'C    10m
            -Hold                   4'C     Forever
-Gel Loading:
    -1% Agarose Gel             100mL
    -SyberSafe                  10.0uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -LexAOps + mFa #1           5.0uL   Lane #2
    -LexAOps + mFa #2           5.0uL   Lane #3
    -LexAOps + mFa #3           5.0uL   Lane #4
    -LexAOps + mFa #4           5.0uL   Lane #5
    -LexAOps + mFa #5           5.0uL   Lane #6
    -LexAOps + mFa #6           5.0uL   Lane #7
    -LexAOps + Ste2 #1          5.0uL   Lane #8
    -LexAOps + Ste2 #2          5.0uL   Lane #9
    -LexAOps + Ste2 #3          5.0uL   Lane #10
    -LexAOps + Ste2 #4          5.0uL   Lane #11
    -LexAOps + Ste2 #5          5.0uL   Lane #12
    -LexAOps + Ste2 #6          5.0uL   Lane #13
    -LexAOps (+GFP) #1          5.0uL   Lane #14
    -LexAOps (+GFP) #2          5.0uL   Lane #15
    -LexAOps (+GFP) #3          5.0uL   Lane #16
    -LexAOps (+GFP) #4          5.0uL   Lane #17
    -LexAOps (+GFP) #5          5.0uL   Lane #18
    -LexAOps (+GFP) #6          5.0uL   Lane #19
    -2-log DNA Ladder           10.0uL  Lane #20

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + mFa,Ste2,GFP Colony PCR&quot; 
    -Several bands for LexAOps + mFa
    -Lane 7 Appeared to have the correct size band
    -LexAOps + Ste2 bands too small
    -No bands for LexAOps (+GFP)
</CODE></PRE><H3>[]LexAOps + mFa + Hy86E3 Liquid Culture</H3><PRE><CODE>-LexAOps + mFa + Hy86E3 Liquid Culture:
    -Labeled a culturing tube
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tube
    -Transfered 20uL of cell solution LexAOps + mFa #6 from colony PCR to culturing tube
    -Incubated overnight
</CODE></PRE><H3>[]LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Colony PCR #2 Part 1</H3><PRE><CODE>-LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Colony PCR #2:
    -Filled 16 PCR tubes with 25uL of water
    -Created PCR Master Mixes:
        -LexAOps + Ste2:
            -2x GoTaq Green PCR Master Mix              90.0uL
            -10uM FW Primer-150 [LexAOps (+pNH605)]     9.0uL
            -10uM RV Primer-134 [Ste2 (+pNH605)]        9.0uL
            -Water                                      27.0uL
        -LexAOps (+GFP):
            -2x GoTaq Green PCR Master Mix              90.0uL
            -10uM FW Primer-150 [LexAOps (+pNH605)]     9.0uL
            -10uM RV Primer-147 [LexAOps (+GFP)]        9.0uL
            -Water                                      27.0uL
    -Transferred Colonies:
        -Transferred 1 colony to 8 PCR tubes from 1x LexAOps+Ste2+Hy86E3 experiment plate and mixed by pipetting up and down
        - Transferred 1 colony to 8 PCR tubes from 1x LexAOps+GFP+Hy111E2 experiment plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       95'C    5m
            -30 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -Extension          72'C    2m
            -Final Extension        72'C    10m
            -Hold                   4'C     Forever
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Run Colony PCR #2 on a gel
-Miniprep Liquid Culture
-Sent to DNA Sequencing
-Culture new correct samples
</CODE></PRE><H2>7/21/15</H2><P>Ran yesterday's PCR on a gel. Minipreped LexAOps + mFa + Hy86E3 and sent for DNA sequencing. Liquid Cultured correct samples of LexAOps + Ste2 and LexAOps (+GFP).</P><H3>[]LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Colony PCR #2 Part 2</H3><PRE><CODE>-Gel Loading:
    -1% Agarose Gel             100mL
    -SyberSafe                  10.0uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -LexAOps + Ste2 #1          5.0uL   Lane #2
    -LexAOps + Ste2 #2          5.0uL   Lane #3
    -LexAOps + Ste2 #3          5.0uL   Lane #4
    -LexAOps + Ste2 #4          5.0uL   Lane #5
    -LexAOps + Ste2 #5          5.0uL   Lane #6
    -LexAOps + Ste2 #6          5.0uL   Lane #7
    -LexAOps + Ste2 #7          5.0uL   Lane #8
    -LexAOps + Ste2 #8          5.0uL   Lane #9
    -2-Log DNA Ladder           10.0uL  Lane #10
    -LexAOps (+GFP) #1          5.0uL   Lane #11
    -LexAOps (+GFP) #2          5.0uL   Lane #12
    -LexAOps (+GFP) #3          5.0uL   Lane #13
    -LexAOps (+GFP) #4          5.0uL   Lane #14
    -LexAOps (+GFP) #5          5.0uL   Lane #15
    -LexAOps (+GFP) #6          5.0uL   Lane #16
    -LexAOps (+GFP) #7          5.0uL   Lane #17
    -LexAOps (+GFP) #8          5.0uL   Lane #18
    -2-Log DNA Ladder           10.0uL  Lane #19

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + Ste2,GFP PCR Colony #2&quot; 
    -Several bands for Ste2
    -Lanes 4,7, and 9 appear to be the correct size
    -Only one band for LexAOps (+GFP) which appears correct
</CODE></PRE><H3>[]LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Liquid Culture</H3><PRE><CODE>-LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Liquid Culture:
    -Labeled 4 culturing tubes
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tubes
    -Transfered 20uL of cell solution from each correct colony PCR [(LexAOps + Ste2:#3,6,8) (LexAOps (+GFP):#6)] to culturing tubes
    -Incubated overnight
</CODE></PRE><H3>[]LexAOps + mFa + Hy86E3 Miniprep</H3><PRE><CODE>-LexAOps + mFa + Hy86E3 Miniprep:
    -Centrifuged liquid cultures for 10 mins
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixture to a 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into a QIAprep spin column
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed column in a new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of the tube
        -LexAOps + mFa + Hy86E3     645.2ng/uL
-DNA Sequencing:
    -Sent 5uL to Quintara for DNA sequencing
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Miniprep Liquid Cultures
-Sent for DNA Sequencing
-Prepare for Gibson for LexAOps + mFa + Ste2
</CODE></PRE><H2>7/22/15</H2><P>Minipreped yesterday's liquid cultures. Sent sample for DNA Sequencing. Digested LexAOps + mFa + Hy86E3.</P><H3>[]LexAOps + mFa + Hy86E3 DNA Sequencing Results</H3><PRE><CODE>-DNA was sequenced correctly and showed evidence of both the LexAOps promoter and mFa gene.
-LexAOps + mFa + Hy86E3 -&gt; **pGEM47**
</CODE></PRE><H3>[]LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Miniprep</H3><PRE><CODE>-LexAOps + Ste2/GFP + Hy86E3/Hy111E2 Miniprep
    -Centrifuged liquid cultures for 10 mins
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to a 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into QIAprep spin columns
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to columns to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed columns into new 1.5mL tubes. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of the tube
        -LexAOps + Ste2 + Hy86E3 #3     685.4ng/uL
        -LexAOps + Ste2 + Hy86E3 #6     667.6ng/uL
        -LexAOps + Ste2 + Hy86E3 #8     480.4ng/uL
        -LexAOps + GFP + Hy111E2 #6     596.5ng/uL
-DNA Sequencing:
    -Sent 5uL of each sample to Quintara for DNA sequencing
</CODE></PRE><H3>[]pGEM47 Digestion</H3><PRE><CODE>-pGEM47 Digestion:
    -Transferred 5.0uL of plasmid into a new 1.5mL tube
    -Added 12.5uL of water
    -Added 2.0uL of CutSmart Buffer
    -Added 0.5uL of NotI
    -Vortexed
    -Incubated at 37'C for 2 hour
</CODE></PRE><H3>[]pGEM47 Purification</H3><PRE><CODE>-pGEM Purification:
    -Added Buffer PB in a 5:1 ratio and mix
        -pGEM47:        +100uL
    -Transferred sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to a new 1.5mL tube
    -Added 50uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -pGEM47:        97.84ng/uL
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-PCR Ste2 with correct overhangs
-Gibson pGEM47 with terminator sequence and Ste2
-Transform cells
</CODE></PRE><H2>7/23/15</H2><P>Check our DNA Sequencing results. Worked on our group meeting powerpoint.</P><H3>[]LexAOps + Ste2/GFP + Hy86E3/Hy111E2 DNA Sequencing</H3><PRE><CODE>-LexAOps + Ste2 + Hy86E3 DNA Sequencing
    -Sample #8 Sequenced Correctly
-**pGEM49**
-LexAOps + GFP + Hy111E2 DNA Sequencing
    -Sample Sequenced Correctly
-**pGEM50**
</CODE></PRE><H3>[]LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) PCR Part 1</H3><PRE><CODE>-Reconstitute Primers:
    -LexAOps (+ApaI/tCyc1) FW #155: 31.5 nmoles x 10 = 315uL
    -Ste2 (+Sbf1/tAdh1) RV #156: 36.0 nmoles x 10 = 360uL
-Dilution for Working Stock:
    -Transferred 10uL of each primer to a new tube and added 90uL of water
-LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) PCR:
    -10uM FW Primer-155 (+ApaI/tCyc1)           2.5uL
    -10uM RV Primer-156 (+Sbf1/tAdh1)           2.5uL
    -pGEM49 Template DNA                        0.5uL       
    -2x Phusion Master Mix                      25.0uL
    -Water                                      19.5uL
                                    Total:      50.0uL
-Thermocycler Protocol
    -Initial Denaturation       98'C    30m
        -10 Cycles
            -Denaturation       98'C    10s
            -Annealing          55'C    20s
            -(-1'C Per Cycle)
            -Extension          72'C    2m
        -30 Cycles
            -Denaturation       98'C    10s
            -Annealing          55'C    20s
            -Extension          72'C    2m
        -Final Extension        72'C    5m
        -Hold                   4'C     Forever

    -ERROR*: Left top loose for a few mins during start
</CODE></PRE><H3>[]To Do:</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Run PCR on a gel
-PCR Purify
-Gibson Assembly
-Transform Cells
</CODE></PRE><H2>7/24/15</H2><P>Ran yesterday's PCR on a gel. PCR purified. Gibson Assembly of LexAOps + mFa + tCyc1 + Ste2. Transformed and plated cells.</P><H3>[]LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) PCR Part 2</H3><PRE><CODE>-Gel Loading:
    -1% Agarose Gel             35mL
    -SyberSafe                  3.5uL

    -2-Log DNA Ladder           10.0uL  Lane #2
    -LexAOps + Ste2             6.0uL   Lane #4

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LexAOps + Ste2 (OH)&quot; 
    -Band appeared to be slightly smaller than expected
</CODE></PRE><H3>[]LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) PCR Purification</H3><PRE><CODE>-LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) PCR Purification
    -Added Buffer PB in a 5:1 ratio and mix
        -LexAOps + Ste2:        +225uL
    -Transferred sample to a QIAquick column in a 2mL collection tube
    -Centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Added 0.75mL of Buffer PE to column and centrifuged for 1 minute
    -Discarded flow-through, returned column to collection tube
    -Centrifuged column empty for 1 minute
    -Transferred column to a new 1.5mL tube
    -Added 50uL of water to the column to elute DNA
    -Centrifuged for 1 minute
-Nano-Drop:
    -Tested 1.5uL of each sample
        -LexAOps + Ste2:        104.9ng/uL
</CODE></PRE><H3>[]Gibson Assembly LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) + tCyc1 (+mFa) + pGEM47</H3><PRE><CODE>-Protocol:
    -LexAOps + mFa + tCyc1 + Ste2
        -Vector DNA [pGEM47] (50ng):    0.55uL
        -tCyc1 DNA (15ng):              0.10uL
        -LexAOps + Ste2 DNA (10ng):     0.35uL
        -2x Gibson Master Mix:          5.00uL
        -Water:                         4.00uL
                            Total:      10.0uL
    -(-) Control
        -Vector DNA [pGEM47] (50ng):    0.55uL
        -2x Gibson Master Mix:          5.00uL
        -Water:                         4.45uL
                            Total:      10.0uL
    -(+) Control
        -Positive Control Mix:          5.00uL
        -2x Gibson Master Mix:          5.00uL
                            Total:      10.0uL
-Incubation:
    -Incubated in thermocycler at 50'C for 60 mins
    -Hold at 4'C forever
</CODE></PRE><H3>[]LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) + tCyc1 (+mFa) + pGEM47 Transformation</H3><PRE><CODE>-LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) + tCyc1 (+mFa) + pGEM47 Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) + tCyc1 (+mFa) + pGEM47 to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
-(-) Control Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned (-) Control to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
-(+) Control Transformation
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 2uL of Gibsoned (+) Control to tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 30 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Turned on Bunson Burner
    -Added 950uL of SOC into mixture
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed two plates (LB + Carb) to 37'C
    -Mixed cells by flicking
    -Spread 100uL cells onto one plate
    -Centrifuged remaining mixture to concentrate cells
    -Discarded excess liquid and vortexed resuspended cells
    -Spread 100uL (8x) cells onto second plate
    -Incubated both plates over weekend at room temperature
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Colony PCR plates
-Liquid Culture
</CODE></PRE><H2>7/27/15</H2><P>We colony PCRed our transformed plates from Friday. We made a liquid culture of one correct sample. We created new transformed plates of our pGEM47, 49, and 50. We prepared overnight cultures of yeast cells.</P><H3>[]LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) + tCyc1 (+mFa) + pGEM47 Colony PCR</H3><PRE><CODE>-LexAOps (+ApaI/tCyc1) + Ste2 (+Sbf1/tAdh1) + tCyc1 (+mFa) + pGEM47 Colony PCR:
    -Little to no colonies on (-) Control Plates
    -Some colonies on (+) Control Plates
    -Lots of Colonies on LmtS plates

    -Filled 8 PCR tubes with 25uL of water
    -Created PCR Master Mixes:
        -LexAOps + Ste2:
            -2x GoTaq Green PCR Master Mix              90.0uL
            -10uM FW Primer-131 [mFa (+LexAOps)]        9.0uL
            -10uM RV Primer-156 [Ste2 (+pNH605)]        9.0uL
            -Water                                      27.0uL
    -Transferred Colonies:
        -Transferred 1 colony to 6 PCR tubes from 1x LmtS experiment plate and mixed by pipetting up and down
        - Transferred 1 colony to 2 PCR tubes from (-) Control plate and mixed by pipetting up and down
        -Transferred 5uL from each PCR tube to its associated clean tube
        -Vortexed and centrifuged
    -Thermocycler Protocol:
        -Initial Denaturation       95'C    5m
            -30 Cycles
                -Denaturation       95'C    45s
                -Annealing          55'C    30s
                -Extension          72'C    3.5m
            -Final Extension        72'C    10m
            -Hold                   4'C     Forever
-Gel Loading:
    -1% Agarose Gel             100mL
    -SyberSafe                  10.0uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -LmtS #1                    5.0uL   Lane #2
    -LmtS #2                    5.0uL   Lane #3
    -LmtS #3                    5.0uL   Lane #4
    -LmtS #4                    5.0uL   Lane #5
    -LmtS #5                    5.0uL   Lane #6
    -LmtS #6                    5.0uL   Lane #7
    -Empty                      0.0uL   Lane #8
    -(-) Control #1             5.0uL   Lane #9
    -(-) Control #2             5.0uL   Lane #10
    -2-Log DNA Ladder           10.0uL  Lane #11
    -Jeffrey's Sample #1        5.0uL   Lane #12
    -Jeffrey's Sample #2        5.0uL   Lane #13
    -Jeffrey's Sample #3        5.0uL   Lane #14
    -Jeffrey's Sample #4        5.0uL   Lane #15
    -Jeffrey's Sample #5        5.0uL   Lane #16
    -Jeffrey's Sample #6        5.0uL   Lane #17
    -Jeffrey's Sample #7        5.0uL   Lane #18
    -Jeffrey's Sample #8        5.0uL   Lane #19
    -2-Log DNA Ladder           10.0uL  Lane #20

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;LmtS Colony PCR&quot; 
    -One correct band for sample #3 in lane #4
</CODE></PRE><H3>[]LmtS Liquid Culture</H3><PRE><CODE>-LmtS Liquid Culture:
    -Labeled a culturing tube
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tube
    -Transfered 20uL of cell solution from LmtS colony PCR Sample #3 to culturing tubes
    -Incubated overnight
</CODE></PRE><H3>[]pGEM47, 49, and 50 Transformations</H3><PRE><CODE>-pGEM47, 49, and 50 Transformations:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into 3 1.5mL tubes. Placed on ice
    -Added 0.5uL of plasmid pGEM47, 49, and 50 to each tube. Flicked tubes 5 times to mix
    -Placed mixture on ice for 10 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Added 450uL of SOC into tubes
    -Incubated at 37'C for 60 minutes. Shaken at 250rpm
    -Warmed three plates (LB + Carb) to 37'C
    -Spread 100uL cells onto each plate
    -Incubated plates overnight at 37'C
</CODE></PRE><H3>[]Yeast Transformation Part 1</H3><PRE><CODE>-Yeast Liquid Culture
    -Added 5mL YPD to two glass culturing tubes
    -Using a toothpick, selected a sample of yeast from streak plates CB008DB and yGEM
    -Mixed cells into tubes
    -Placed in 30'C and spun overnight
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Miniprep Liquid Culture
-Create Liquid Cultures of pGEM's
-DNA Sequencing
-Transform Yeast Cells
</CODE></PRE><H2>7/28/15</H2><P>We minipreped yesterday's liquid culture. We sent a sample for DNA Sequencing. We streaked our transformation plates due to excessive colonies. We transformed our pGEM50 into yeast cells.</P><H3>[]LmtS Miniprep</H3><PRE><CODE>-LmtS Miniprep:
    -Centrifuged liquid cultures for 10 mins 3000rpm
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to a 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into QIAprep spin columns
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to columns to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed columns into new 1.5mL tubes. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of the tube
        -LmtS               234.8ng/uL
-DNA Sequencing:
    -Prepared 3 separate tubes for sequencing:
        -1:
            -4uL LmtS DNA
            -2.5uL Primer #136
            -8.5uL Water
        -2:
            -4uL LmtS DNA
            -2.5uL Primer #137
            -8.5uL Water
        -3:
            -2.5uL LmtS DNA
            -2.5uL Water
</CODE></PRE><H3>[]pGEM47, 49, and 50 Plates Streak</H3><PRE><CODE>-pGEM47, 49, and 50 Plates Streak:
    -Using a toothpick, gather a line of colonies
    -Spread onto a new warmed LB+Carb plate into a small area
    -Using a new toothpick, spread across plate with one line and then pie-wedges
    -Using a third toothpick, spread across plate again with one line and then pie-wedges
    -Incubated overnight at 37'C
</CODE></PRE><H3>[]Yeast Transformation Part 2</H3><PRE><CODE>-pGEM50 Digestion:
    -Added 2000ng of pGEM50 ~ 3.4uL
    -Added 2uL CutSmart Buffer
    -Added 14.1uL Water
    -Added 0.5uL PmeI
    -Vortexed
    -Incubated at 37'C for 2 hours
-O/N Culture Dilution:
    -Dilution of 1:20 in YPD
    -Incubated at 30'C with spinning for 3 hours
    -*Performed by Jeffrey
-ssDNA Boiling:
    -Boil 10uL of 10mg/uL (per rxn) ssDNA in Thermocycler at 99'C for 10mins
    -Immediately place on ice once finished
    -*Performed by Jeffrey
-Yeast Transformation Protocol:
    -Transfer mixture from O/N Culture Dilution to new culturing tubes
    -Centrifuge at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 1mL 0.1M LiOAc
    -Transfer over to 1.5mL tubes
    -Centrifuge cells at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 100uL 0.1M LiOAc per 2.5mL Culture
        -If Necessary, split into 100uL tubes for each transformation
    -Add 100ug (10uL) ssDNA to each 100uL sample
    -Add 5uL of target DNA
        -1:
            -pGEM50
        -2:
            -pGEM50
            -pGEM48
    -Add in order:
        -480uL 50% PEG 3350
        -60uL 10X TE
        -60uL 1M LiOAc
        -75uL DMSO
    -Vortex
    -Incubate ar 42'C for 20 mins
    -Warm Selection Plates
        -1:
            -LEU2
        -2:
            -TRP + LEU2
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard Supernatent by pipetting
    -Resuspend in 500uL YPD
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard most of the Supernatent, leaving ~ 50uL
    -Resuspend cells in residual YPD
    -Plate on selective media
    -Incubate for 1-3 days at 30'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Make Liquid Cultures of Streaked plates
-Check DNA Sequencing
</CODE></PRE><H2>7/29/15</H2><P>Checked DNA Sequencing results. Made liquid cultures of streaked plates. Transformed new cells with LmtS for amplification of plasmid.</P><H3>[]LmtS DNA Sequencing Results</H3><PRE><CODE>-LmtS DNA Sequencing Results:
    -LexAOps #1:
        -Everything appeared to be sequenced correctly
    -mFa:
        -There was a single basepair mismatch. Thankfully, regardless of whether it was a true mutation or sequencing error, it would still code for the same amino acid.
    -tCyc1:
        -Everything appeared to be sequenced correctly
    -LexAOps #2:
        -Some mismatches were present in the promoter, but hopefully it shouldn't have any significant consequence.
    -Ste2:
        -Everything appeared to be sequenced correctly

-LexAOps + mFa + tCyc1 + LexAOps + Ste2 -&gt; **pGEM53**
</CODE></PRE><H3>[]pGEM47, 49, and 50 Liquid Culture</H3><PRE><CODE>-pGEM47, 49, and 50 Liquid Culture:
    -Labeled 3 culturing tubes
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tubes
    -Transfered 1 colony with a toothpick from the plates to each culturing tubes
    -Incubated overnight
</CODE></PRE><H3>[]pGEM53 Amplification Transformation</H3><PRE><CODE>-pGEM53 Amplification Transformation:
    -Transferred 25uL of NEB 5-alpha Competent E. coli cells into a 1.5mL tube. Placed on ice
    -Added 0.5uL of plasmid pGEM53 to the tube. Flicked tube 5 times to mix
    -Placed mixture on ice for 10 minutes
    -Heat Shocked at 42'C for 30 seconds
    -Placed on ice for 5 minutes
    -Added 450uL of SOC into tubes
    -Incubated at 37'C for 20 minutes. Shaken at 250rpm
    -Warmed a plate (LB + Carb) to 37'C
    -Spread 75uL cells onto the plate
    -Incubated plate overnight at 37'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Make Liquid Cultures pGEM53 plate
-Miniprep Liquid Cultures
-Check Yeasts
</CODE></PRE><H2>7/30/15</H2><P>Liquid cultured a colony from pGEM53 plate. Minipreped pGEM47, 49, and 50 liquid cultures. Helped miniprep Jeffrey's samples.</P><H3>[]pGEM53 Amplification Liquid Culture</H3><PRE><CODE>-pGEM53 Amplification Liquid Culture:
    -Labeled a culturing tube
    -Turned on bunson burner
    -Added 5mL of LB+Carb media to tube
    -Transfered 1 colony with a toothpick from the plate to the culturing tube
    -Incubated overnight
</CODE></PRE><H3>[]pGEM47, 49, and 50 Miniprep</H3><PRE><CODE>-pGEM47, 49, and 50 Miniprep:
    -Centrifuged liquid cultures for 10 mins 3000rpm
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to a 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into QIAprep spin columns
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to columns to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed columns into new 1.5mL tubes. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of the tube
        -pGEM47             594.1ng/uL
        -pGEM49             388.9ng/uL
        -pGEM50             597.8ng/uL
</CODE></PRE><H3>[]Miniprep for Jeffrey</H3><PRE><CODE>-Miniprep for Jeffrey:
    -Samples:
        -Hwp1 (Sbf1) #1
        -Hwp1 (Sbf1) #4
        -Mgfp5 (NotI) #1
        -Mgfp5 (NotI) #5
        -Mgfp5 (Sbf1) #1
        -Mgfp5 (Sbf1) #5
    -Centrifuged liquid cultures for 10 mins 3000rpm
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixtures to a 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into QIAprep spin columns
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to columns to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed columns into new 1.5mL tubes. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of the tube
        -Hwp1 (Sbf1) #1     311.9ng/uL
        -Hwp1 (Sbf1) #4     386.8ng/uL
        -Mgfp5 (NotI) #1    772.4ng/uL
        -Mgfp5 (NotI) #5    689.7ng/uL
        -Mgfp5 (Sbf1) #1    566.9ng/uL
        -Mgfp5 (Sbf1) #5    551.6ng/uL
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Miniprep pGEM53 Liquid Culture
-Check Yeasts
</CODE></PRE><H2>7/31/15</H2><P>Minipreped pGEM53. Re-Transformed Yeast cells. Remade Lithium Acetate solutions and TE Buffer. Check Current Yeast plates -&gt; No Cells. Left in incubate over weekend.</P><H3>[]pGEM53 Miniprep</H3><PRE><CODE>-pGEM53 Miniprep
    -Centrifuged liquid culture for 10 mins 3000rpm
    -Resuspended cells in 250uL Buffer P1
    -Transferred mixture to a 1.5mL tube
    -Added 250uL Buffer P2. Mixed by inverting 6 times
    -Added 350uL Buffer N3. Mixed by inverting 6 times
    -Centrifuged for 10 minutes
    -Poured supernatant into QIAprep spin column
    -Centrifuged for 30 seconds. Discarded flow-through
    -Added 0.75mL Buffer PE to column to wash. Centrifuged for 30 seconds
    -Discarded flow-through. Centrifuged empty for 1 minute
    -Placed column into new 1.5mL tube. Added 50uL water to column. Let stand for 1 minute
    -Centrifuged for 1 minute.
-Nanodrop:
    -Measured 1.5uL of the tube
        -pGEM53             246.3ng/uL
</CODE></PRE><H3>[]Yeast Transformation #2</H3><PRE><CODE>-New LiOAc and TE Buffer
    -1M LiOAc
        -5.101g Lithium Acetate
        -50mL Milli-Q Water
        -Shake to Mix
        -Vaccum Filtered
    -10X TE Buffer
        -.288g Trizma Hydrochloric Acid
        -.18612g EDTA
        -50mL Milli-Q Water
        -Shake to Mix
        -Added ~1.2mL 1M NaOH to bring pH to 7.5
        -Shake to Mix
        -Vaccum Filtered
    -0.1M LiOAc
        -40mL Milli-Q Water
        -5mL 1M LiOAc
        -5mL 10X Te Buffer
        -Shake to Mix
-pGEM50 Digestion:
    -Added 2000ng of pGEM50 ~ 3.4uL
    -Added 2uL CutSmart Buffer
    -Added 13.6uL Water
    -Added 1.0uL PmeI
    -Vortexed
    -Incubated at 37'C for 2 hours
-O/N Culture Dilution:
    -Dilution of 1:20 in YPD
        -500uL of O/N Culture
        -9.5mL YPD
    -Incubated at 30'C with spinning for 3 hours
-ssDNA Boiling:
    -Boil 10uL of 10mg/uL (per rxn) ssDNA in Thermocycler at 99'C for 10mins
    -Immediately place on ice once finished for 10 mins
-Yeast Transformation Protocol:
    -Transfer mixture from O/N Culture Dilution to new culturing tubes
    -Centrifuge at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 1mL 0.1M LiOAc
    -Transfer over to 1.5mL tubes
    -Centrifuge cells at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 100uL 0.1M LiOAc per 2.5mL Culture
        -If Necessary, split into 100uL tubes for each transformation
    -Add 100ug (10uL) ssDNA to each 100uL sample
    -Add 5uL of target DNA
        -1:
            -pGEM50
        -2:
            -pGEM50
            -pGEM48
    -Add in order:
        -480uL 50% PEG 3350
        -60uL 10X TE
        -60uL 1M LiOAc
        -75uL DMSO
    -Vortex
    -Incubate ar 42'C for 20 mins
    -Warm Selection Plates
        -1:
            -LEU2
        -2:
            -TRP + LEU2
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard Supernatent by pipetting
    -Resuspend in 500uL YPD
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard most of the Supernatent, leaving ~ 50uL
    -Resuspend cells in residual YPD
    -Plate on selective media
    -Incubate for 1-3 days at 30'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Check Yeast Plates
</CODE></PRE><H2>8/3/15</H2><P>Performed yeast colony PCR. Ran it on a gel.</P><H3>[]Yeast Colony PCR</H3><PRE><CODE>-Primer Dilution:
    -#161 FW pNH605
    -#87 RV 600 Series
-Yeast Colony PCR:
    -Some colonies on LexAOps + GFP plate
    -No Colonies on pFig2C + LexADBD/ LexAOps + GFP

    -Filled 6 PCR tubes with 50uL of Zymolyase
    -Transferred Colonies:
        -Selected one colony with a toothpick
        -Spread onto a new LEU plate within a drawn box
        -Twirled same stick into respective PCR tube
        -Repeated with new colonies for each new tube.
    -Incubated at 37'C for 30mins, then 95'C for 10mins
    -Created PCR Master Mix:
            -2x GoTaq Green PCR Master Mix          70.0uL
            -10uM FW Primer-161 [pNH605]            7.0uL
            -10uM RV Primer-87 [600 Series]         7.0uL
            -Water                                  21.0uL
    -Added 15uL of Master Mix to 6 new PCR Tubes
    -Transferred 5uL from boiled Yeast cells PCR tubes to its associated tube
    -Vortexed and centrifuged
-Thermocycler Protocol:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation       95'C    45s
            -Annealing          55'C    30s
            -Extension          72'C    1m
        -Final Extension        72'C    10m
        -Hold                   4'C     Forever
-Gel Loading:
    -1% Agarose Gel             100mL
    -SyberSafe                  10.0uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -Yeast Colony PCR #1        5.0uL   Lane #2
    -Yeast Colony PCR #2        5.0uL   Lane #3
    -Yeast Colony PCR #3        5.0uL   Lane #4
    -Yeast Colony PCR #4        5.0uL   Lane #5
    -Yeast Colony PCR #5        5.0uL   Lane #6
    -Yeast Colony PCR #6        5.0uL   Lane #7
    -Empty                      0.0uL   Lane #8
    -2-Log DNA Ladder           5.0uL   Lane #9
    -Empty                      0.0uL   Lane #10
    -Erika + Josh's Sample #1   10.0uL  Lane #11
    -Erika + Josh's Sample #2   10.0uL  Lane #12
    -Erika + Josh's Sample #3   10.0uL  Lane #13
    -Erika + Josh's Sample #4   10.0uL  Lane #14
    -Erika + Josh's Sample #5   10.0uL  Lane #15
    -Empty                      0.0uL   Lane #16
    -2-Log DNA Ladder           10.0uL  Lane #17

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR LexAOps+GFP&quot; 
    -Two bands in lanes 6 and 7~600bp Expected size
    -Faint band in lane 5~600bp Expected size
-**yGEM131**
</CODE></PRE><H3>[]yGEM131 Liquid Culture</H3><PRE><CODE>-yGEM131 Liquid Culture:
    -Labeled two yeast culturing tubes
    -Added 5mL YPD to each tube
    -Using a stick, transferred some colonies from initial plate (#5,6) to tubes
    -Incubated at 30'C overnight
</CODE></PRE><H3>[]To Do:</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Make Glycerol Stock
-Make Streak Plate with patch plate
-Transform Yeast cells with pGEM48
</CODE></PRE><H2>8/4/15</H2><P>Made glycerol stocks of yGEM131. Made streak plates of our yGEM131. Transformed yGEM131 with pGEM48.</P><H3>[]yGEM131 Glycerol Stock</H3><PRE><CODE>-yGEM131 Glycerol Stock
    -Labeled a cyrovial
    -Added 350uL of 60% glycerol
    -Added 350uL of O/N culture (yGEM131 #6)
    -Vortexed for 2 mins
    -Freeze with dry ice for 2 mins
    -Store in -80'C freezer
</CODE></PRE><H3>[]yGEM131 Streak Plate</H3><PRE><CODE>-yGEM131 Streak Plate
    -Using a stick, gathered cells from #6 from yGEM131 patch plate
    -Streaked over a new LEU plate
    -Left to incubate overnight
</CODE></PRE><H3>[]yGEM131 + pGEM48 Transformation</H3><PRE><CODE>-pGEM48 Digestion:
    -Added 2000ng of pGEM48 ~ 3.5uL
    -Added 1uL CutSmart Buffer
    -Added 4.5uL Water
    -Added 1.0uL PmeI
    -Vortexed
    -Incubated at 37'C for 2 hours
-O/N Culture Dilution:
    -Dilution of 1:20 in YPD
        -250uL of O/N Culture
        -4.75mL YPD
    -Incubated at 30'C with spinning for 3 hours
-ssDNA Boiling:
    -Boil 10uL of 10mg/uL (per rxn) ssDNA in Thermocycler at 99'C for 10mins
    -Immediately place on ice once finished for 10 mins
-Yeast Transformation Protocol:
    -Transfer mixture from O/N Culture Dilution to new culturing tubes
    -Centrifuge at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 1mL 0.1M LiOAc
    -Transfer over to 1.5mL tubes
    -Centrifuge cells at 6000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 100uL 0.1M LiOAc per 2.5mL Culture
        -If Necessary, split into 100uL tubes for each transformation
    -Add 100ug (10uL) ssDNA to each 100uL sample
    -Add 5uL of target DNA
    -Add in order:
        -480uL 50% PEG 3350
        -60uL 10X TE
        -60uL 1M LiOAc
        -75uL DMSO
    -Vortex
    -Incubate ar 42'C for 30 mins
    -Warm Selection Plates
        -TRP
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard Supernatent by pipetting
    -Resuspend in 500uL YPD
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard most of the Supernatent, leaving ~ 50uL
    -Resuspend cells in residual YPD
    -Plate on selective media
    -Incubate for 1-3 days at 30'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Check Yeast Plates
</CODE></PRE><H2>8/5/15</H2><P>Checked Yeast plate, still need to grow. Fixed UCSF iGEM logo design and created short gif.</P><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Check Yeast Plates
</CODE></PRE><H2>8/6/15</H2><P>Checked Yeast plate, still need to grow, no colonies yet. Helped with prep-work for flow cytometry. Ran flow cytometer for alpha-factor (DB,34,128,132, &amp; 34+128) and doxycycline (DB,129, &amp; 130.</P><H3>[]Flow Cytometry</H3><PRE><CODE>-Day Before:
    -Make Overnight cultures of strain to be tested in 5-10mL SD-Comp
-Flow Cytometry Prep Work
    -Dilute overnight cultures 1:100 in 96 well shaker plate
        -1000uL SD-Comp
        -XuL Cell Culture
            -See Spectrophotometer
    -Spectrophotometer:
        -900uL SD-Comp per Cuvett
        -100uL O/N Culture
            -1:10 Dilution
        -Read at 600nm
        -Vortex all cuvetts, including blank
        -Zero with blank first, then read other samples
        -Calculate necessary sample volume
            - (Read * 10) / Desired OD (0.3) = Dilution Factor
            - Volume of Well (1000uL) / Dil-Factor = XuL per well for Sample
    -Cover with breathable film
    -Place in Incubator shaker for 3 hours
-Flow Cytometry Induce:
    -30mins prior to end of incubation, thaw necessary induction factor
    -See &quot;Flow&quot; in google docs for dilution recipe
    -Add 10uL of desire concentration of induction factor to each well
    -Cover with breathable film
    -Place in Incubator Shaker for 90mins - 2 hours
-Flow Cytometry Run:
    -Transfer 250uL from each well to new 96 well V-Bottom Plate
    -Add 10uL of cycloheximid to arrest cells
    -Run on Flow Cytometer
    -Analyze Data
-**Running the Flow Cytometer**
    -Check that machine is on standby and not run.
    -Run a clean plate. (A1-A4 bleach, B1-B4 water)
    -Check fluid buffer box and storage tanks.
    -Make sure you have a positive and negative control so that you can set the parameters.
    -Open up FACSDiva. Sign into iGEM account. 
    -Create new FACS experiment. Highlight needed wells and use the blue button to create wells for use. Create specimens among the wells and rename them. 
    -Open up inspector to check parameters. 
    -Run well. (Do not click 'Run Plate'.)
    -Once finished, export data onto a USB.
    -Run a clean plate.
</CODE></PRE><H3>[]Yeast Liquid Culture</H3><PRE><CODE>-Made liquid cultures of yGEM129,yGEM130, and CB008DB
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Check Yeast Plates
-Transform yGEM129 and yGEM130
</CODE></PRE><H2>8/7/15</H2><P>Cleaned up lab area and old tubes with bleach. Transformed yGEM129 and yGEM130 with pGEM53.</P><H3>[]Cleaning</H3><PRE><CODE>-Use 10% bleach in all tubes
-Let sit for 30 minutes
-Dump out liquid at bleach station
-Rinse with water
-Place at autoclave station
</CODE></PRE><H3>[]Yeast Transformation with pGEM53 to yGEM129 &amp; 130</H3><PRE><CODE>-pGEM53 Digestion:
    -Added 4000ng of pGEM50 ~ 17.0uL
    -Added 2uL CutSmart Buffer
    -Added 1.0uL PmeI
    -Vortexed
    -Incubated at 37'C for 2 hours
-O/N Culture Dilution:
    -Dilution of 1:20 in YPD
        -250uL of O/N Culture
        -4.75mL YPD
    -Incubated at 30'C with spinning for 3 hours
-ssDNA Boiling:
    -Boil 10uL of 10mg/uL (per rxn) ssDNA in Thermocycler at 99'C for 10mins
    -Immediately place on ice once finished for 10 mins
    -*Slightly less salmon sperm for yGEM129 transformation due to it boiling away
-Yeast Transformation Protocol:
    -Transfer mixture from O/N Culture Dilution to new culturing tubes
    -Centrifuge at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 1mL 0.1M LiOAc
    -Transfer over to 1.5mL tubes
    -Centrifuge cells at 6000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 100uL 0.1M LiOAc per 2.5mL Culture
        -If Necessary, split into 100uL tubes for each transformation
    -Add 100ug (10uL) ssDNA to each 100uL sample
    -Add 5uL of target DNA
    -Add in order:
        -480uL 50% PEG 3350
        -60uL 10X TE
        -60uL 1M LiOAc
        -75uL DMSO
    -Vortex
    -Incubate ar 42'C for 30 mins
    -Warm Selection Plates
        -LEU
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard Supernatent by pipetting
    -Resuspend in 500uL YPD
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard most of the Supernatent, leaving ~ 50uL
    -Resuspend cells in residual YPD
    -Plate on selective media
    -Incubate for 1-3 days at 30'C
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Check Yeast Plates
-Colony PCR
</CODE></PRE><H2>8/10/15</H2><P>Helped make some new PEG. Colony PCRed our yeast plates. Ran on a gel. Made patch plates.</P><H3>[]Making PEG</H3><PRE><CODE>-Warm water to 42'C
-Add 25g of PEG powder to 50mL tube
-Fill tube with water till 50mL mark
-Shake vigorously for 30 mins
-Vacuum Filtrate
</CODE></PRE><H3>[]Colony PCR Yeast Plates 129/130</H3><PRE><CODE>-Yeast Colony PCR:
    -Some colonies on 129 and 130 plates
    -Filled 12 PCR tubes with 50uL of Zymolyase
    -Filled 12 PCR tubes with 20uL of Water
    -Transferred Colonies:
        -Selected one colony with a toothpick
        -Spread onto a new LEU plate within a drawn box
        -Twirled same stick into respective PCR tube (water)
        -Transferred 5uL to respective Zymolyase tube
        -Repeated with new colonies for each new tube
    -Incubated at 37'C for 30mins, then 95'C for 10mins
    -Created PCR Master Mix:
            -2x GoTaq Green PCR Master Mix          130.0uL
            -10uM FW Primer-161 [pNH605]            13.0uL
            -10uM RV Primer-87 [600 Series]         13.0uL
            -Water                                  39.0uL
    -Added 15uL of Master Mix to 6 new PCR Tubes
    -Transferred 5uL from boiled Yeast cells PCR tubes to its associated tube
    -Vortexed and centrifuged
-Thermocycler Protocol:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation       95'C    45s
            -Annealing          55'C    30s
            -Extension          72'C    1m
        -Final Extension        72'C    10m
        -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel           100mL
    -SyberSafe                  10.0uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -Empty                      0.0uL   Lane #2
    -yGEM129 + pGEM53 #1        5.0uL   Lane #3
    -yGEM129 + pGEM53 #2        5.0uL   Lane #4
    -yGEM129 + pGEM53 #3        5.0uL   Lane #5
    -yGEM129 + pGEM53 #4        5.0uL   Lane #6
    -yGEM129 + pGEM53 #5        5.0uL   Lane #7
    -Empty *Error               0.0uL   Lane #8
    -Empty                      0.0uL   Lane #9
    -2-Log DNA Ladder           10.0uL  Lane #10
    -Empty                      0.0uL   Lane #11
    -yGEM130 + pGEM53 #1        5.0uL   Lane #12
    -yGEM130 + pGEM53 #2        5.0uL   Lane #13
    -yGEM130 + pGEM53 #3        5.0uL   Lane #14
    -yGEM130 + pGEM53 #4        5.0uL   Lane #15
    -yGEM130 + pGEM53 #5        5.0uL   Lane #16
    -yGEM130 + pGEM53 #6        5.0uL   Lane #17
    -Empty                      0.0uL   Lane #18
    -2-Log DNA Ladder           10.0uL  Lane #19

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR yGEM129+130 + pGEM53 #1&quot; 
    -No Bands
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Re-Do Colony PCR
</CODE></PRE><H2>8/11/15</H2><P>Re-did yeast colony pcr for yGEM129/130 + pGEM53. Performed yeast colony PCR for yGEM131 + pGEM48.</P><H3>[]Yeast Colony PCR yGEM129/130 + pGEM53</H3><PRE><CODE>-Yeast Colony PCR yGEM129/130 + pGEM53:
    -Many Colonies on both yGEM129 + pGEM53 and yGEM130 + pGEM53
    -Filled 12 PCR tubes with 50uL of Zymolyase
    -Transferred Colonies:
        -Selected one colony with a toothpick
        -Spread onto a new LEU plate within a drawn box
        -Twirled same stick into respective PCR tube
    -Incubated at 37'C for 30mins, then 95'C for 10mins
    -Created PCR Master Mix:
            -2x GoTaq Green PCR Master Mix          130.0uL
            -10uM FW Primer-161 [pNH605]            13.0uL
            -10uM RV Primer-87 [600 Series]         13.0uL
            -Water                                  39.0uL
    -Added 15uL of Master Mix to 6 new PCR Tubes
    -Transferred 5uL from boiled Yeast cells PCR tubes to its associated tube
    -Vortexed and centrifuged
-Thermocycler Protocol:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation       95'C    45s
            -Annealing          55'C    30s
            -Extension          72'C    1m
        -Final Extension        72'C    10m
        -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel           85mL
    -SyberSafe                  8.5uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -yGEM129 + pGEM53 #1        5.0uL   Lane #2
    -yGEM129 + pGEM53 #2        5.0uL   Lane #3
    -yGEM129 + pGEM53 #3        5.0uL   Lane #4
    -yGEM129 + pGEM53 #4        5.0uL   Lane #5
    -yGEM129 + pGEM53 #5        5.0uL   Lane #6
    -yGEM129 + pGEM53 #6        5.0uL   Lane #7
    -Empty                      0.0uL   Lane #8
    -2-Log DNA Ladder           10.0uL  Lane #9
    -Empty                      0.0uL   Lane #10
    -yGEM130 + pGEM53 #1        5.0uL   Lane #11
    -yGEM130 + pGEM53 #2        5.0uL   Lane #12
    -yGEM130 + pGEM53 #3        5.0uL   Lane #13
    -yGEM130 + pGEM53 #4        5.0uL   Lane #14
    -yGEM130 + pGEM53 #5        5.0uL   Lane #15
    -yGEM130 + pGEM53 #6        5.0uL   Lane #16
    -Empty                      0.0uL   Lane #17
    -2-Log DNA Ladder           10.0uL  Lane #18

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR yGEM129+130 + pGEM53 #2&quot;
    -Bands for all samples
    -Liquid Culture Samples yGEM129 + pGEM53 #3 and yGEM130 + pGEM53 #6
</CODE></PRE><H3>[]Yeast Colony PCR yGEM131 + pGEM48</H3><PRE><CODE>-Yeast Colony PCR yGEM131 + pGEM48:
    -Few colonies on yGEM131 + pGEM48 plates
    -Filled 8 PCR tubes with 50uL of Zymolyase
    -Transferred Colonies:
        -Selected one colony with a toothpick
        -Spread onto a new TRP plate within a drawn box
        -Twirled same stick into respective PCR tube
    -Incubated at 37'C for 30mins, then 95'C for 10mins
    -Created PCR Master Mix:
            -2x GoTaq Green PCR Master Mix          90.0uL
            -10uM FW Primer-160 [pNH604]            9.0uL
            -10uM RV Primer-87 [600 Series]         9.0uL
            -Water                                  27.0uL
    -Added 15uL of Master Mix to 6 new PCR Tubes
    -Transferred 5uL from boiled Yeast cells PCR tubes to its associated tube
    -Vortexed and centrifuged
-Thermocycler Protocol:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation       95'C    45s
            -Annealing          55'C    30s
            -Extension          72'C    1m
        -Final Extension        72'C    10m
        -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel           35mL
    -SyberSafe                  3.5uL

    -Empty                      0.0uL   Lane #1
    -2-Log DNA Ladder           10.0uL  Lane #2
    -Empty                      0.0uL   Lane #3
    -yGEM131 + pGEM48 #1        5.0uL   Lane #4
    -yGEM131 + pGEM48 #2        5.0uL   Lane #5
    -yGEM131 + pGEM48 #3        5.0uL   Lane #6
    -yGEM131 + pGEM48 #4        5.0uL   Lane #7
    -yGEM131 + pGEM48 #5        5.0uL   Lane #8
    -yGEM131 + pGEM48 #6        5.0uL   Lane #9
    -yGEM131 + pGEM48 #7        5.0uL   Lane #10
    -yGEM131 + pGEM48 #8        5.0uL   Lane #11
    -Empty                      0.0uL   Lane #12
    -2-Log DNA Ladder           10.0uL  Lane #13

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR yGEM131 + pGEM48&quot;
    -Few bands that are correct
    -Liquid Culture Samples yGEM131 + pGEM48 #8
</CODE></PRE><H3>[]NEW yGEMS</H3><PRE><CODE>-**yGEM136** (yGEM129 + pGEM53)
-**yGEM137** (yGEM130 + pGEM53)
-**yGEM138** (yGEM131 + pGEM48)
</CODE></PRE><H3>[]Yeast Liquid Culture</H3><PRE><CODE>-Made liquid cultures of yGEM129 + pGEM53, yGEM130 + pGEM53, yGEM131 + pGEM48
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Glycerol Stock new yGEMs
-Transform yGEM136 and yGEM137 with pGEM48
-Run yGEM138 on Flow
</CODE></PRE><H2>8/12/15</H2><P>Made Glycerol Stocks of yGEM136, 137, and 138. Made streak plates for yGEM136, 137, and 138. Transformed yGEM133, 136, and 137 with pGEM48. Helped with colony PCR for yGEM129 + pGEM48 for Jeffrey. Made new liquid cultures.</P><H3>[]yGEM Glycerol Stocks</H3><PRE><CODE>-yGEM136, 137, and 138 Glycerol Stocks
    -Labeled a cyrovial
    -Added 350uL of 60% glycerol
    -Added 350uL of O/N culture
    -Vortexed for 2 mins
    -Freeze with dry ice for 2 mins
    -Store in -80'C freezer
</CODE></PRE><H3>[]Yeast Transformations yGEM133, 136, and 137</H3><PRE><CODE>-pGEM48 Digestion #1:
    -Added 2000ng of pGEM48 ~ 3.5uL
    -Added 1uL CutSmart Buffer
    -Added 1.0uL PmeI
    -Added 4.5uL Water
    -Vortexed
    -Incubated at 37'C for 2 hours
-pGEM48 Digestion #2:
    -Added 1000ng of pGEM48 ~ 1.8uL
    -Added 1uL CutSmart Buffer
    -Added 1.0uL PmeI
    -Added 6.2uL Water
    -Vortexed
    -Incubated at 37'C for 2 hours
-O/N Culture Dilution:
    -Dilution of 1:20 in YPD
        -250uL of O/N Culture
        -4.75mL YPD
    -Incubated at 30'C with spinning for 3 hours
-ssDNA Boiling:
    -Boil 10uL of 10mg/uL (per rxn) ssDNA in Thermocycler at 99'C for 10mins
    -Immediately place on ice once finished for 10 mins
-Yeast Transformation Protocol:
    -Transfer mixture from O/N Culture Dilution to new culturing tubes
    -Centrifuge at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 1mL 0.1M LiOAc
    -Transfer over to 1.5mL tubes
    -Centrifuge cells at 6000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 100uL 0.1M LiOAc per 2.5mL Culture
        -If Necessary, split into 100uL tubes for each transformation
    -Add 100ug (10uL) ssDNA to each 100uL sample
    -Add 5uL of target DNA
    -Add in order:
        -480uL 50% PEG 3350
        -60uL 10X TE
        -60uL 1M LiOAc
        -75uL DMSO
    -Vortex
    -Incubate ar 42'C for 30 mins
    -Warm Selection Plates
        -TRP
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard Supernatent by pipetting
    -Resuspend in 500uL YPD
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard most of the Supernatent, leaving ~ 50uL
    -Resuspend cells in residual YPD
    -Plate on selective media
    -Incubate for 1-3 days at 30'C

    -*Little amount of cells for all yGEMs, transformation success probability probably low.
        -&gt;Re-do Tommorrow
</CODE></PRE><H3>[]Yeast Liquid Cultures</H3><PRE><CODE>-Made liquid cultures for yGEM136, 137, 138, and CB008DB
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Transform yGEM136 and yGEM137 again
-Run yGEM138 on Flow
</CODE></PRE><H2>8/13/15</H2><P>Re-did yeast transformations for yGEM136 and 137. Ran Flow Cytometer for yGEM138.</P><H3>[]Yeast Transformations yGEM136, and 137</H3><PRE><CODE>-pGEM48 Digestion:
    -Added 2000ng of pGEM48 ~ 4uL
    -Added 1uL CutSmart Buffer
    -Added 1.0uL PmeI
    -Added 4.0uL Water
    -Vortexed
    -Incubated at 37'C for 2 hours
-O/N Culture Dilution:
    -Dilution of 1:20 in YPD
        -250uL of O/N Cultures
        -4.75mL YPD
    -Incubated at 30'C with spinning for 3 hours
-ssDNA Boiling:
    -Boil 10uL of 10mg/uL (per rxn) ssDNA in Thermocycler at 99'C for 10mins
    -Immediately place on ice once finished for 10 mins
-Yeast Transformation Protocol:
    -Transfer mixture from O/N Culture Dilution to new culturing tubes
    -Centrifuge at 3000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 1mL 0.1M LiOAc
    -Transfer over to 1.5mL tubes
    -Centrifuge cells at 6000 for 4 mins
    -Discard Supernatent
    -Resuspend cells with 100uL 0.1M LiOAc per 2.5mL Culture
        -If Necessary, split into 100uL tubes for each transformation
    -Add 100ug (10uL) ssDNA to each 100uL sample
    -Add 5uL of target DNA
        -*ERROR One tube may have not recieved any DNA while the other recieved 2000ng
    -Add in order:
        -480uL 50% PEG 3350
        -60uL 10X TE
        -60uL 1M LiOAc
        -75uL DMSO
    -Vortex
    -Incubate ar 42'C for 30 mins
    -Warm Selection Plates
        -TRP
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard Supernatent by pipetting
    -Resuspend in 500uL YPD
    -Centrifuge tubes at 6000rpm for 2 mins
    -Discard most of the Supernatent, leaving ~ 50uL
    -Resuspend cells in residual YPD
    -Plate on selective media
    -Incubate for 1-3 days at 30'C
</CODE></PRE><H3>[]Flow Cytometery yGEM138</H3><PRE><CODE>-Flow Cytometry Prep Work
    -Dilute overnight cultures 1:100 in 96 well shaker plate
        -1000uL SD-Comp
        -XuL Cell Culture
            -See Spectrophotometer
    -Spectrophotometer:
        -900uL SD-Comp per Cuvett
        -100uL O/N Culture
            -1:10 Dilution
        -Read at 600nm
        -Vortex all cuvetts, including blank
        -Zero with blank first, then read other samples
        -Calculate necessary sample volume
            - (Read * 10) / Desired OD (0.3) = Dilution Factor
            - Volume of Well (1000uL) / Dil-Factor = XuL per well for Sample
    -Cover with breathable film
    -Place in Incubator shaker for 3 hours
-Flow Cytometry Induce:
    -30mins prior to end of incubation, thaw necessary induction factor
    -See &quot;Flow&quot; in google docs for dilution recipe
    -Add 10uL of desire concentration of induction factor to each well
    -Cover with breathable film
    -Place in Incubator Shaker for 90mins - 2 hours
-Flow Cytometry Run:
    -Transfer 250uL from each well to new 96 well V-Bottom Plate
    -Add 10uL of cycloheximid to arrest cells
    -Run on Flow Cytometer
    -Analyze Data
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
</CODE></PRE><H2>8/14/15</H2><P>Focused efforts on Wiki and Graphic Design. Helped a little with Flow Cytometry Work with Jeffrey and Nick</P><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Check Yeast Plates
</CODE></PRE><H2>8/17/15</H2><P>Colony PCRed yeast strains. Made overnight cultures of new yGEMs.</P><H3>[]Yeast Colony PCR yGEM136/137/133 + pGEM48</H3><PRE><CODE>-Yeast Colony PCR yGEM136/137/133 + pGEM48:
    -Few colonies on yGEM133 + pGEM48 and yGEM137 + pGEM48 plates
    -Lots of colonies on yGEM136 + pGEM48 plate
    -Filled 7 PCR tubes with 50uL of Zymolyase
    -Transferred Colonies:
        -Selected one colony with a toothpick
        -Spread onto a new TRP plate within a drawn box
        -Twirled same stick into respective PCR tube
    -Incubated at 37'C for 30mins, then 95'C for 10mins
    -Created PCR Master Mix:
            -2x GoTaq Green PCR Master Mix          120.0uL
            -10uM FW Primer-160 [pNH604]            12.0uL
            -10uM RV Primer-87 [600 Series]         12.0uL
            -Water                                  36.0uL
    -Added 15uL of Master Mix to 11 new PCR Tubes
    -Transferred 5uL from boiled Yeast cells PCR tubes to its associated tube
    -Vortexed and centrifuged
-Thermocycler Protocol:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation       95'C    45s
            -Annealing          55'C    30s
            -Extension          72'C    1m
        -Final Extension        72'C    10m
        -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel           35mL
    -SyberSafe                  3.5uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -yGEM136 + pGEM48 #1        5.0uL   Lane #2
    -yGEM136 + pGEM48 #2        5.0uL   Lane #3
    -yGEM136 + pGEM48 #3        5.0uL   Lane #4
    -yGEM136 + pGEM48 #4        5.0uL   Lane #5
    -yGEM136 + pGEM48 #5        5.0uL   Lane #6
    -yGEM136 + pGEM48 #6        5.0uL   Lane #7
    -2-Log DNA Ladder           10.0uL  Lane #8
    -yGEM133 + pGEM48 #1        5.0uL   Lane #9
    -yGEM133 + pGEM48 #2        5.0uL   Lane #10
    -yGEM133 + pGEM48 #3        5.0uL   Lane #11
    -yGEM137 + pGEM48 #4        5.0uL   Lane #12
    -yGEM137 + pGEM48 #5        5.0uL   Lane #13
    -2-Log DNA Ladder           10.0uL  Lane #14

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR yGEM136 &amp; 133 &amp; 137 + pGEM48&quot;
    -Some bands are correct
    -Need new colony PCR for yGEM137 + pGEM48
</CODE></PRE><H3>[]Yeast Colony PCR yGEM137 + pGEM48 #2</H3><PRE><CODE>-Yeast Colony PCR yGEM137 + pGEM48 #2:
    -3 colonies for yGEM137 + pGEM48 plate #2
    -Filled 3 PCR tubes with 50uL of Zymolyase
    -Transferred Colonies:
        -Selected one colony with a toothpick
        -Spread onto a new TRP plate within a drawn box
        -Twirled same stick into respective PCR tube
    -Incubated at 37'C for 30mins, then 95'C for 10mins
    -Created PCR Master Mix:
            -2x GoTaq Green PCR Master Mix          400.0uL
            -10uM FW Primer-160 [pNH604]            4.0uL
            -10uM RV Primer-87 [600 Series]         4.0uL
            -Water                                  12.0uL
    -Added 15uL of Master Mix to 3 new PCR Tubes
    -Transferred 5uL from boiled Yeast cells PCR tubes to its associated tube
    -Vortexed and centrifuged
-Thermocycler Protocol:
    -Initial Denaturation       95'C    5m
        -30 Cycles
            -Denaturation       95'C    45s
            -Annealing          55'C    30s
            -Extension          72'C    1m
        -Final Extension        72'C    10m
        -Hold                   4'C     Forever
-Gel Loading:
    -1.5% Agarose Gel           35mL
    -SyberSafe                  3.5uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -yGEM137 + pGEM48 #1        5.0uL   Lane #2
    -yGEM137 + pGEM48 #2        5.0uL   Lane #3
    -yGEM137 + pGEM48 #3        5.0uL   Lane #4
    -2-Log DNA Ladder           10.0uL  Lane #5

    -Ran at 100 volts for 30 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR yGEM137 + pGEM48&quot;
    -2 bands are correct
</CODE></PRE><H3>[]NEW yGEMS</H3><PRE><CODE>-**yGEM155** (yGEM137 + pGEM48)
</CODE></PRE><H3>[]Yeast Liquid Culture</H3><PRE><CODE>-Made liquid cultures of yGEM147, 148, and 155
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Glycerol Stock new yGEMs
-Streak Plates
-Run yGEM147 + 155 on Flow
</CODE></PRE><H2>8/18/15</H2><P>Streaked plates for yGEM147, 148, and 155. Made Glycerol Stocks of yGEM147, 148, and 155. Helped with Flow Cytometry for DB, yGEM147, and yGEM155 induced with Dox.</P><H3>[]To Do</H3><PRE><CODE>-Learn HTML
-Learn WebGL
-Learn JavaScript
-Create Wiki Abstract Animation
</CODE></PRE><H2>8/19/15</H2><P>Made O/N liquid cultures of DB and yGEM138 for flow tomorrow.</P><H3>[]To Do</H3><PRE><CODE>-Flow for yGEM138
-Add Glow Effects to Wiki Abstract Video
</CODE></PRE><H2>8/20/15</H2><P>Ran flow cytometer for yGEM138. Worked on Wiki Abstract Video. Discussed presentation plans.</P><H3>[]To Do</H3><PRE><CODE>-Finish adding effects to Wiki Abstract Video
-Start Creating Presentation Slides
</CODE></PRE><H2>8/21/15</H2><P>Worked on Wiki Abstract Video all day.</P><H3>[]To Do</H3><PRE><CODE>-Finish adding effects to Wiki Abstract Video
-Start Creating Presentation Slides
</CODE></PRE><H2>8/24/15</H2><P>Colony PCRed yeast plates. Transformed yGEM146 with pGEM63. Made O/N liquid cultures for flow tomorrow.</P><H3>[]Yeast Colony PCR</H3><PRE><CODE>-Plates:
    -yGEM147 + m7-Bar1
    -yGEM153 + m7-Bar1
    -yGEM154 + m7-Bar1
    -yGEM155 + m7-Bar1
-Master Mix (17x):
    -2x GoTaq Green PCR Master Mix          170.0uL
    -10uM FW Primer-91 [NatMX]              17.0uL
    -10uM RV Primer-87 [600 Series]         17.0uL
    -Water                                  51.0uL
-Gel:
    -1.5% Agarose Gel           100mL
    -SyberSafe                  10.0uL

    -2-Log DNA Ladder           10.0uL  Lane #1
    -yGEM147 + m7-Bar1 #1       5.0uL   Lane #2
    -yGEM147 + m7-Bar1 #2       5.0uL   Lane #3
    -yGEM147 + m7-Bar1 #3       5.0uL   Lane #4
    -yGEM147 + m7-Bar1 #4       5.0uL   Lane #5
    -2-Log DNA Ladder           10.0uL  Lane #6
    -yGEM153 + m7-Bar1 #1       5.0uL   Lane #7
    -yGEM153 + m7-Bar1 #2       5.0uL   Lane #8
    -yGEM153 + m7-Bar1 #3       5.0uL   Lane #9
    -yGEM153 + m7-Bar1 #4       5.0uL   Lane #10
    -2-Log DNA Ladder           10.0uL  Lane #11
    -yGEM154 + m7-Bar1 #1       5.0uL   Lane #12
    -yGEM154 + m7-Bar1 #2       5.0uL   Lane #13
    -yGEM154 + m7-Bar1 #3       5.0uL   Lane #14
    -yGEM154 + m7-Bar1 #4       5.0uL   Lane #15
    -2-Log DNA Ladder           10.0uL  Lane #16
    -yGEM155 + m7-Bar1 #1       5.0uL   Lane #17
    -yGEM155 + m7-Bar1 #2       5.0uL   Lane #18
    -yGEM155 + m7-Bar1 #3       5.0uL   Lane #19
    -yGEM155 + m7-Bar1 #4       5.0uL   Lane #20

    -Ran at 100 volts for 35 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Yeast Colony PCR yGEM147 153 154 155 +m7-Bar1&quot;
    -All lanes appear to have correct size band (945bp)
    -Selected #2 for each sample
-New yGEMs:
-**yGEM160**
    -yGEM147 + m7-Bar1
-**yGEM161**
    -yGEM153 + m7-Bar1
-**yGEM162**
    -yGEM154 + m7-Bar1
-**yGEM163**
    -yGEM155 + m7-Bar1
</CODE></PRE><H3>[]Yeast Transformation</H3><PRE><CODE>-Transformed yGEM146 with pGEM63
</CODE></PRE><H3>[]O/N Liquid Cultures</H3><PRE><CODE>-CB008DB
-yGEM138
-yGEM160
-yGEM163
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Run yeast strains on Flow
-Make YPD O/N Cultures for yGEM160-163
-Finish up Wiki Abstract Video
-Work on Presentation Slides
-Create Shorter Wiki Flicks Animation Video
</CODE></PRE><H2>8/25/15</H2><P>Ran flow cytometer. Worked on Wiki Flicks video and presentation slides for Reno.</P><H3>To Do</H3><PRE><CODE>-Create new plasmid with pTET and mFa
-Test current yeast strains for mFa production
</CODE></PRE><H2>8/26/15</H2><P>PCRed pTET + mFa and gel extracted. Digested pJW607 and gel extracted.</P><H3>[]PCR pTET + mFa</H3><PRE><CODE>-PCR Reaction:
    -FW primer  #173
    -RW primer  #132
    -Template   11E3128
-Thermocycler:
    -Initial Denaturation       98'C    30s
        -35 Cycles
            -Denaturation       98'C    10s
            -Annealing          65'C    20s
            -Extension          72'C    30s
        -Final Extension        72'C    5m
        -Hold                   4'C     Forever
-Gel:
    -1.5% Agarose

    -2-Log DNA Ladder       10uL    Lane #1
    -pTET + mFa PCR #1      30uL    Lane #2
    -pTET + mFa PCR #2      30uL    Lane #3

    -Ran at 100V for 50 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;PCR pTET + mFa&quot;
    -Both lanes appear to have the correct band size ~1200bp
    -Extracted both lanes
</CODE></PRE><H3>[]pJW607 Digestion</H3><PRE><CODE>-Digestion:
    -Digested with ApaI and NotI
-Gel:
    -1% Agarose

    -2-Log DNA Ladder       10uL    Lane #1
    -pJW607 DIgestion       36uL    Lane #2
-Gel Results:
    -See Gel Folder for gel labeled &quot;pJW607 Digestion&quot;
    -Bright Clear band for cut plasmid and second band below suggesting plasmid was properly cut.
    -Extracted band at ~7kb
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Run flow cytometer for yeast strains
-Seamless for pJW607 with pTET+mFa
-Yeast colony PCR yGEM146 + pGEM63
</CODE></PRE><H2>8/27/15</H2><P>Tested current strains for alpha production by inducing with dox and the testing supernatent with alpha responsive strains. Seamless of pJW607 with pTET+mFa. Transformed E.coli and plated.</P><H3>[]To Do</H3><PRE><CODE>-Yeast Colony PCR for yGEM146 + pGEM63
</CODE></PRE><H2>8/28/15</H2><P>Colony PCRed yeast colonies for yGEM146 + pGEM63. Streaked plates for yGEM160, 161, 162, and 163.</P><H3>[]Yeast Colony PCR yGEM146 + pGEM63</H3><PRE><CODE>-Used FW Primer #160 for TRP site and RV Primer #87 for 600 series.
-Ran a gel
-Band for sample #1 in lane #2 appeared correct.
-**yGEM164**
    -yGEM146 + pGEM63
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Make Glycerol stocks
-Colony PCR E.coli with pJW607 + pTET-mFa
</CODE></PRE><H2>8/31/15</H2><P>Colony PCRed pJW607 + pTET-mFa. Ran on a gel. </P><H3>[]Colony PCR pJW607 + pTET-mFa</H3><PRE><CODE>-PCR Reaction:
    -Used FW Primer #173
    -Used RV Primer #132
-Gel:
    -1% Agarose Gel

    -2-Log DNA Ladder       10uL    Lane #1
    -pJW607 + pTET-mFa #1   5uL     Lane #2
    -pJW607 + pTET-mFa #2   5uL     Lane #3
    -pJW607 + pTET-mFa #3   5uL     Lane #4
    -pJW607 + pTET-mFa #4   5uL     Lane #5
    -pJW607 + pTET-mFa #5   5uL     Lane #6
    -pJW607 + pTET-mFa #6   5uL     Lane #7
    -pJW607 + pTET-mFa #7   5uL     Lane #8
    -pJW607 + pTET-mFa #8   5uL     Lane #9
    -2-Log DNA Ladder       10uL    Lane #10

    -Ran at 100V for 50 mins
-Gel Results:
    -See Gel Folder for gel labeled &quot;Colony PCR pTET + mFa&quot;
    -Lanes appear to have varying band lengths
    -Samples #1, 3, 4, and 5 were liquid cultured.
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Miniprep Liquid Cultures
-Send to DNA Sequencing
</CODE></PRE><H2>9/1/15</H2><P>Miniprepped O/N Liquid Cultures. Sent to DNA Sequencing. Ran Flow cytometer for yGEM165 and 166.</P><H3>[]To Do</H3><PRE><CODE>-Transform yeasts with Bar1 plasmids
-Transform Bar1 yeasts with BFP
</CODE></PRE><H2>9/2/15</H2><P>Finished up designs for T-Shirt and Hoodies. Transformed Bar1 Yeast Strains with BFP. Got tubes for yGEM165 and 166 mixed up, so have to re-transform tomorrow.</P><H3>[]Yeast Transformations</H3><PRE><CODE>-yGEM129 + pGEM64
-yGEM130 + pGEM64
-yGEM143 + pGEM63
-yGEM144 + pGEM63
-yGEM145 + pGEM63
-yGEM146 + pGEM63
</CODE></PRE><H3>[]To Do</H3><PRE><CODE>-Re-Transform Yeasts
-Test yGEM165 and 166 strains mixed with Bar1 Strains on Flow
</CODE></PRE><H2>9/3/15</H2></DIV></DIV></DIV></DIV></BODY></HTML>