"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins/Project/VitC
From 2011.igem.org
VitaYeast - Johns Hopkins University, iGEM 2011
Team
Members
Acknowledgements
Advisors
Sponsors
Attributions
Gallery
Vitamins
Background
Overview
Vitamin A
Vitamin C
Measurements
Applications
Results
Future Plans
Yeast Toolkit
Background
Overview
Promoters and UTRs
Violacein
Yeast Vector Library
Results
Future Plans
Human Practices
Overview
IRB Approval
Data
Modeling
Platform
LBS Models
Optimization
Sensitivity
Parameter Fitting
Notebook
Protocols
Vitamin Experiments
Yeast Toolkit Experiments
Safety
Related Links:
On This Page:
Relevant Parts
Vitamins:
Background
Overview
Vitamin A
Measurements
Applications
Results
Future Plans
Vitamin C
The genes for this pathway were taken from the genome of Arabidopsis thaliana and were codon optimized for yeast. The restriction sites from the BioBrick prefix and suffix were removed to make them biobrick compatible.  These in silico sequences were then constructed de novo using templateless PCR and overlap extension.
The plan for the construction of the Vitamin C synthesis pathway in yeast consisted of three steps.
First we synthesized three open reading frames (ORF) corresponding to the three genes for producing GDP-mannose-3,5-epimerase, GDP-L-galactose phosphorylase, and L-Galactose 1-phosphate phosphatase. This was done using overlap extension to create two or more ‘building blocks’, each corresponding to a chunk of the ORF. Each building block had a 40 base pair overlap with its adjacent building block and with the vector (pRS416) in to which we inserted the ORF.
After this, the building blocks were assembled into the complete ORF in the vector using the CPEC assembly method. The ORFs were subsequently sequenced to make sure we constructed the correct sequences and then BioBricked using primers with the prefix and suffix flanking an overlap with the ORF.
GDP-L-galactose phosphorylase expression vector
GDP-mannose-3,5-epimerase expression vector
L-Galactose 1-phosphate phosphatase expression vector
In the end, the completed ORFs were assembled into expression cassettes under the influence of strong promoter and terminator combinations (TDH3, RPS8b, and RPL24a) again using the CPEC method of assembly.
http://j5.jbei.org/j5manual/pages/22.html
We will later characterize our system by growing liquid cultures and assaying samples for ascorbic acid at incremental time intervals.  We will then use this data to establish rate constants as well as quantify how much Vitamin C (ascorbic acid) our system can produce.
Once we have curves for Vitamin C production versus time, we will run a comparative study on how yeast producing Vitamin C grows on YPD plates (a lab standard) versus how it grows on our desired substrate, bread. To test its growth on bread we will create plates of dough-like media.
To optimize the production of ascorbic acid on the new substrate we will apply a strategy of directed evolution. We will make a combinatorial library of expression cassettes using Golden Gate Assembly for every synthesized ORF in the Vitamin C pathway. This will be made by using promoters and terminators of varying strengths from across the genome. Once constructed, this library will be plated on dough media plates with a calculated amount of peroxide in the agar. The presence of the peroxide will apply oxidative stress on the cells and, as Vitamin C confers oxidative stress resistance, this behaves as a selection for the cells that produce more Vitamin C. By adjusting the amount of peroxide on the plate, we can create a threshold of ascorbic acid production below which all the cells will die.  To establish which one of the library members that passed through the selection is the best, we will perform a quantitative screen on the survivors. This will be done by growing up the cells in liquid culture, extracting the ascorbic acid, and determining its concentration through spectrophotometry.
Pathway
The following is the diagram for the Vitamin C pathway.
Sauer, M., Branduardi, P., Valli, M., & Porro, D. (2004). Production of L-ascorbic acid by metabolically engineered Saccharomyces cerevisiae and Zygosaccharomyces bailii. Applied and environmental microbiology, 70(10), 6086-91. doi: 10.1128/AEM.70.10.6086-6091.2004.
Relevant Parts
GLGP
GME
LGPP
Retrieved from " http://2011.igem.org/Team:Johns_Hopkins/Project/VitC "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
