<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Notebook_Vio"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Notebook/Vio">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Notebook/Vio&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Vio&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Vio&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Notebook/Vio" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Notebook/Vio</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>VitaYeast - Johns Hopkins University, iGEM 2011</TITLE></P><DIV id="mydroplinemenu" class="droplinebar"><UL><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Members">Members</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Johns_Hopkins/Team/Gallery">Gallery</A></LI></UL><LI><A href="#">Vitamins</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitA">Vitamin A</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitC">Vitamin C</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant">Measurements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Baking">Applications</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Future">Future Plans</A></LI><LI><A href="#">Yeast Toolkit</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR">Promoters and UTRs</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Violacein">Violacein</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Vector">Yeast Vector Library</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Future">Future Plans</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/HumanPrac">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/IRBProcess">IRB Approval</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Data">Data</A></LI><LI><A href="#">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Platforms">Platform</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/LBSMod">LBS Models</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Opt">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Sensitivity">Sensitivity</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/ParaFit">Parameter Fitting</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments">Vitamin Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YTExperiments">Yeast Toolkit Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></DIV><DIV id="secondarycontentgreen"><DIV id="boxheading">
			Related Links:</DIV><DIV id="boxcontent"><DL><DIV class="heading">Notebook: Violacein:</DIV><DL><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Vio#Violacein_loxPsym_Integration">Violacein loxPsym Integration</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Vio#Violacein_Biosynthesis_in_Yeast">Violacein Biosynthesis in Yeast</A></DD></DL><DL><DIV class="heading">Notebook: Other Yeast Toolkit:</DIV><DL><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YTExperiments">Promoters and UTRs</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YVL">Yeast Vector Library</A></DD></DL><DIV id="primarycontent"><H6><SPAN class="mw-headline" id="Violacein_loxPsym_Integration">Violacein loxPsym Integration</SPAN></H6><H5><SPAN class="mw-headline" id="6.2F11.2F2011">6/11/2011</SPAN></H5><P>Verify that the promoters and 3'UTRs are present in the plasmid library made by Yuxuan. The inserts are ligated into pCR 2.1-TOPO and contain BsaI sites, so following digestion with BsaI they should be visible on a gel.
</P><P>Note: Due to a lack of good sequences the promoters and 3'UTRs for RPL8A, FCY2, and KRE9 were not available.
</P><P>Minipreps were prepared by Ashan.
</P><P>20µl digest
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> Plasmid DNA
</TD><TD> 5
</TD></TR><TR><TD> 10x NEBuffer 4
</TD><TD> 2
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 12.5
</TD></TR><TR><TD> BsaI (10U/l)
</TD><TD> .5
</TD></TR><TR><TD> Total
</TD><TD> 20
</TD></TR></TBODY></TABLE><P>~2.5 hour incubation @ 50°C.
</P><P>The 3'UTRs were expected to be small (most ~142bp), so they were run on a 2% MetaPhor agarose gel.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Vio UTRs 6/11/11</DIV></DIV></DIV><P><B>Results</B>:
</P><P>None of the expected bands were observed. The bands at ~1kb are pieces of the vector (pCR2.1-TOPO), which contains a BsaI site in its sequence.
</P><H5><SPAN class="mw-headline" id="6.2F12.2F2011">6/12/2011</SPAN></H5><P>Gel electrophoresis of promoters from 6/11 restriction digest to verify presence in provided plasmids.
</P><P><B>Results</B>:
</P><P>Some bands are visible around 500bp, but they are not consistent with the expected size of the promoters. The ladder is also very weak.
</P><H5><SPAN class="mw-headline" id="6.2F20.2F2011">6/20/2011</SPAN></H5><P>The 6/11 digest was run with BsaI at 50°C based on Dr. Boeke's NEB book, but the NEB website recommends 37°C. I will run a digest on lambda DNA provided by Dr. Boeke to test which temperature yields better results.
</P><P>Was given 10µl of DNA, 1µg/µl. Made two master mixes in order to make a titration of BsaI, enough for 7 digests each.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Reaction Mix 1
</TD><TD> x7
</TD><TD> Master Mix 1
</TD><TD> Reaction Mix 2
</TD><TD> x7
</TD><TD> Master Mix 2
</TD></TR><TR><TD> .67µl DNA
</TD><TD> 4.67µl DNA
</TD><TD> .67µl DNA
</TD><TD> 4.67µl DNA
</TD></TR><TR><TD> 17.13µl H<SUB>2</SUB>O
</TD><TD> 119.91µl H<SUB>2</SUB>O
</TD><TD> 17.33µl H<SUB>2</SUB>O
</TD><TD> 121.31µl H<SUB>2</SUB>O
</TD></TR><TR><TD> 2µl 10x B4
</TD><TD> 14µl 10x B4
</TD><TD> 2µl 10x B4
</TD><TD> 14µl 10x B4
</TD></TR><TR><TD> .2µl BsaI
</TD><TD> 1.4µl BsaI
</TD></TR><TR><TD> 20µl total, 3 units Bsa1/DNA
</TD><TD> 140µl total
</TD><TD> 20µl total, 0 units BsaI/DNA
</TD><TD> 140µl total
</TD></TR></TBODY></TABLE><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD>  Titration
</TD></TR><TR><TD> 3x
</TD><TD> 2x
</TD><TD> 1x
</TD><TD> 1/3x
</TD><TD> 0x (-control)
</TD></TR><TR><TD> 60µl MM1
</TD><TD> 40µl MM1
</TD><TD> 20µl MM1
</TD><TD> 6.67µl MM1
</TD></TR><TR><TD> 20µl MM2
</TD><TD> 40µl MM2
</TD><TD> 53.33µl MM2
</TD><TD> 60µl MM2
</TD></TR></TBODY></TABLE>
The master mixes and titration were prepared on ice. 20µl of each reaction mix was distributed into 2 groups of 5 microcentrifuge tubes. One group was transferred to a 37°C incubator,<DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">BsaI 37°C vs. 50°C 6/20/11</DIV></DIV></DIV> the other to a 50°C heat block. Both digests were run for two hours.
<P>Digestion at 50°C yielded slightly clearer bands. This means that the promoter/UTR digestion was performed at the correct temperature, and the digestion temperature does not explain the unexpected results. Additionally, it seems that the 2-log ladder with CS red that we had been using (the leftmost column) is showing up very faintly, so we will stop using it.
</P><H5><SPAN class="mw-headline" id="6.2F22.2F2011">6/22/2011</SPAN></H5><P>The unexpected digestion patterns might be explained if the plate with the plasmids sent out for sequencing was flipped. I will do colony PCR on the sequencing plates to test this hypothesis. If this fails, I will PCR the promoters and UTRs from genomic DNA.
</P><P>I chose 6 promoters of varying sizes to test. The promoters are generally larger than the UTRs so they should be more visible on a gel.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Label
</TD><TD> Promoter
</TD><TD> Length (bp)
</TD><TD> position
</TD><TD> flipped position
</TD><TD> Label
</TD><TD> Fragment
</TD><TD> Length (bp)
</TD></TR><TR><TD> 1
</TD><TD> THD3
</TD><TD> 534
</TD><TD> A2
</TD><TD> H11
</TD><TD> 1F
</TD><TD> BAP2utr
</TD><TD> 542
</TD></TR><TR><TD> 2
</TD><TD> RPL24A
</TD><TD> 664
</TD><TD> A9
</TD><TD> H4
</TD><TD> 2F
</TD><TD> HHO1utr
</TD><TD> 354
</TD></TR><TR><TD> 3
</TD><TD> RPS8B
</TD><TD> 834
</TD><TD> B7
</TD><TD> G5
</TD><TD> 3F
</TD><TD> KRE9utr w/mutation
</TD><TD> 263
</TD></TR><TR><TD> 4
</TD><TD> STM1
</TD><TD> 534
</TD><TD> D7
</TD><TD> E6
</TD><TD> 4F
</TD><TD> FCY2utr w/mutation
</TD><TD> 142
</TD></TR><TR><TD> 5
</TD><TD> ARD1
</TD><TD> 535
</TD><TD> F5
</TD><TD> E9
</TD><TD>5F
</TD><TD> RPS2utr
</TD><TD> 142
</TD></TR><TR><TD> 6
</TD><TD> BAP2
</TD><TD> 834
</TD><TD> H5
</TD><TD> F9
</TD><TD> 6F
</TD><TD> TDH3utr
</TD><TD> 142
</TD></TR></TBODY></TABLE><P>
The <I>E. coli</I> containing these plasmids were streaked out on LB and incubated at 37°C, along with the strains containing STM1, ARD1, and BAP2 the original restriction digests were performed on.
</P><H5><SPAN class="mw-headline" id="6.2F24.2F2011">6/24/2011</SPAN></H5><P>The bacteria were left in the incubator for two days, but I decided to proceed since the PCR did not need to select for single colonies.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Promoter, flipped position, and initial strain colony PCR 6/24/11</DIV></DIV></DIV>Followed <A href="/wiki/images/a/a4/Yeast_colony_pcr_protocol.pdf" class="internal" title="Yeast colony pcr protocol.pdf">this</A> protocol. Substituted bacteria, and changed the volume of primer to .01µl since the primers provided I were what I thought was 5mM versus the 20µM in the protocol. 
<P>Did four reactions for each promoter, flipped sequence, and original sequence. 
</P><P>No bands were observed. Too many cells may have been added to the reaction, inhibiting the PCR. Also, the primers were 5µM rather than the 5mM written on the plate, so not enough primer was used.
</P><H5><SPAN class="mw-headline" id="6.2F27.2F2011">6/27/2011</SPAN></H5><P>The 6/24 colony PCR failed, so I will repeat the PCR, adding fewer cells to the reaction, using the correct concentration of primer, and extending the initial denaturation step.
</P><P>Followed same protocol as 6/24, trying not to pick too many cells, using .15µl of the 5µM primer mix, and extending the initial denaturation to 10 minutes. PCR product was stored @ 4°C.
</P><H5><SPAN class="mw-headline" id="6.2F30.2F2011">6/30/2011</SPAN></H5><P>Ran the 6/27 PCR product.
</P><P>No bands observed. It was probably a bad idea modifying a yeast protocol for bacteria, will use a more suitable protocol and try again.
</P><H5><SPAN class="mw-headline" id="7.2F1.2F2011">7/1/2011</SPAN></H5><P>Still trying to get a colony PCR to work in order to verify the promoters and UTRs are present, this time with a different protocol. Tested 6 promoters and their corresponding 'flipped' positions on the sequencing place in order to test the hypothesis that the sequencing plate was flipped.
</P><P>Followed Marty's <A href="/wiki/images/3/3a/Colony_pcr_standard_protocol.pdf" class="internal" title="Colony pcr standard protocol.pdf">protocol for colony PCR</A> using GoTaq Green.
</P><P>Still nothing observed besides primers.
</P><H5><SPAN class="mw-headline" id="7.2F5.2F2011">7/5/2011</SPAN></H5><P>One more attempt at the colony PCR. Same as 7/1 procedure.
</P><P>Same pattern of only primers/primer dimers/partial products observed 7/1. Move to PCR the promoters and 3'UTRs from genomic DNA instead.
</P><H5><SPAN class="mw-headline" id="7.2F7.2F2011">7/7/2011</SPAN></H5><P>Test whether the gDNA PCR will work, using a small set of the promoters and 3'UTRs.
</P><P>Received two sets of wild-type genomic DNA from Zheng, one fully processed and one semi-processed, along with a positive control primer pair (12336/9613) and a negative control primer pair (12310/9613). Because the control primers were 20µM and the promoter/UTR primers were 5µM, made two reaction mixes.
</P><P>Promoter/UTR gDNA PCR Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> gDNA
</TD><TD> 2 µl
</TD></TR><TR><TD> 10x ExTaq buffer
</TD><TD> 2 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD> 1 µl
</TD></TR><TR><TD> F/R Primer mix (5µM)
</TD><TD> 3.2 µl
</TD></TR><TR><TD> ExTaq
</TD><TD> .1 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 11.7 µl
</TD></TR><TR><TD> Total
</TD><TD> 20 µl
</TD></TR></TBODY></TABLE><P>Control Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> gDNA
</TD><TD> 2 µl
</TD></TR><TR><TD> 10x ExTaq buffer
</TD><TD> 2 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD> 1 µl
</TD></TR><TR><TD> F primer (20µM)
</TD><TD> .4 µl
</TD></TR><TR><TD> R primer (20µM)
</TD><TD> .4 µl
</TD></TR><TR><TD> ExTaq
</TD><TD> .1 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 14.1 µl
</TD></TR><TR><TD> Total
</TD><TD> 20 µl
</TD></TR></TBODY></TABLE><P>PCR Program: 5 min @ 94°C, 30x(30s @ 94°C, 30s @ 55°C, 30s @ 72°C), 5 min @ 72°C, hold @ 4°C
</P><P>All the bands showed up as expected with the fully processed gDNA.
</P><H5><SPAN class="mw-headline" id="7.2F8.2F2011">7/8/2011</SPAN></H5><P>Should have nanodropped the gDNA to see how much to put in 7/7 reaction, so did that.
</P><P>gDNA: 63.2 ng/µl
</P><H5><SPAN class="mw-headline" id="7.2F12.2F2011">7/12/2011</SPAN></H5><P>PCR out all of the promoters/3'UTRs w/BsaI compatible primers for later ligation with ORFs and loxPsym site.
</P><P>50µl reaction volume and Herculase polymerase. Got two positive primers from Zheng (ZK096/097, ZK098/099).
</P><P>Promoter/UTR gDNA PCR Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (63ng/µl)
</TD><TD>    2 µl
</TD></TR><TR><TD> F/R Primer mix (5µM)
</TD><TD>    5 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>  27.5 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>Control Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (63ng/µl)
</TD><TD>    2 µl
</TD></TR><TR><TD> F Primer (20µM)
</TD><TD>  .625 µl
</TD></TR><TR><TD> R Primer (20µM)
</TD><TD>  .625 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>  31.25 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>PCR Program: 2 min @ 95°C, 30x(20s @ 95°C, 20s @ 56°C, 30s @72°C), 3 min @ 72°C, hold at 4°C.
</P><P>Took 10µl for gel to check for product.
</P><P>Will PCR purify.
</P><H5><SPAN class="mw-headline" id="7.2F13.2F2011">7/13/2011</SPAN></H5><P>PCR purification of 7/12 PCR product using <A href="/wiki/images/6/61/Qiaquick_spin_handbook-1.pdf" class="internal" title="Qiaquick spin handbook-1.pdf">Qiagen kit</A>. Ran 10µl to check products.
</P><H5><SPAN class="mw-headline" id="7.2F15.2F2011">7/15/2011</SPAN></H5><P>Based on the 7/13 PCR purification, I need to gel extract, because there is lots of junk at around 100bp for many of the reactions. I will do a new PCR with reduced primer concentration.
</P><P>Got gDNA from Patrick, 138ng/µl. 50µl reaction using Herculase.
</P><P>Promoter/UTR Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (138ng/µl)
</TD><TD>    1 µl
</TD></TR><TR><TD> F/R Primer mix (5µM)
</TD><TD>    1 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>  32.5 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>Control Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (63ng/µl)
</TD><TD>    1 µl
</TD></TR><TR><TD> F Primer (20µM)
</TD><TD>  .125 µl
</TD></TR><TR><TD> R Primer (20µM)
</TD><TD>  .125 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>  33.25 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>PCR Program: 2 min @ 95°C, 30x(20s @ 95°C, 20s @ 56°C, 30s @72°C), 3 min @ 72°C, hold at 4°C.
</P><P>Not enough DNA for extraction, will adjust primer concentration again.
</P><H5><SPAN class="mw-headline" id="7.2F18.2F2011">7/18/2011</SPAN></H5><P>Promoter/UTR gDNA PCR attempt 3, now with an intermediate primer concentration.
</P><P><B>Procedure</B>:
</P><P>Promoter/UTR Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (138ng/µl)
</TD><TD>    1 µl
</TD></TR><TR><TD> F/R Primer mix (5µM)
</TD><TD>  2.5 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>   31 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>Control Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (63ng/µl)
</TD><TD>    1 µl
</TD></TR><TR><TD> F Primer (20µM)
</TD><TD>  .3125 µl
</TD></TR><TR><TD> R Primer (20µM)
</TD><TD>  .3125 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>  32.875 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>PCR Program: 2 min @ 95°C, 30x(20s @ 95°C, 20s @ 56°C, 30s @72°C), 3 min @ 72°C, hold at 4°C.
</P><H5><SPAN class="mw-headline" id="7.2F19.2F2011">7/19/2011</SPAN></H5><P>Purified DNA from 7/18 gel extraction according to <A href="https://static.igem.org/mediawiki/2011/6/61/Qiaquick_spin_handbook-1.pdf|Qiagen" class="external text" rel="nofollow">gel extraction protocol</A>, then nanodropped to find concentration. Used Invitrogen spin columns I found in the columns box.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD>  Sequence
</TD><TD> Concentration (ng/µl)
</TD><TD> Sequence
</TD><TD> Concentration (ng/µl)
</TD></TR><TR><TD> TDH3p
</TD><TD> 31.6
</TD><TD> FCY2p
</TD><TD> 31.7
</TD></TR><TR><TD> TDH3utr
</TD><TD> 13.7
</TD><TD> FCY2utr
</TD><TD> 22.9
</TD></TR><TR><TD> RPL24Ap
</TD><TD> 37.9
</TD><TD> PRY1p
</TD><TD> 4.6
</TD></TR><TR><TD> RPL24Autr
</TD><TD> 12.6
</TD><TD> PRY1utr
</TD><TD> 21.9
</TD></TR><TR><TD> RPS8Bp
</TD><TD> 22.2
</TD><TD> ARD1p
</TD><TD> 3.0
</TD></TR><TR><TD> RPS8Butr
</TD><TD> 5.1
</TD><TD> ARD1utr
</TD><TD> 9.7
</TD></TR><TR><TD> RPS2p
</TD><TD> 2.9
</TD><TD> KRE9p
</TD><TD> 42.1
</TD></TR><TR><TD> RPS2utr
</TD><TD> 3.7
</TD><TD> KRE9utr
</TD><TD> 41.0
</TD></TR><TR><TD> RPL8Ap
</TD><TD> 3.0
</TD><TD> HHO1p
</TD><TD> 49.4
</TD></TR><TR><TD> RPL8Autr
</TD><TD> 18.6
</TD><TD> HHO1utr
</TD><TD> 14.6
</TD></TR><TR><TD> STM1p
</TD><TD> 31.5
</TD><TD> BAP2p
</TD><TD> 9.3
</TD></TR><TR><TD> STM1utr
</TD><TD> 17.9
</TD><TD> BAP2utr
</TD><TD> 27.9
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="7.2F21.2F2011">7/21/2011</SPAN></H5><P>Loaded 5µl of product on a 1% gel to verify 7/19 gel purification products.
</P><H5><SPAN class="mw-headline" id="7.2F25.2F11">7/25/11</SPAN></H5><P>Colony purified ß-carotene yeast strains in preparation for gDNA PCR experiment investigating white/super orange colonies.
Inoculated YPD and dough media plates w/Rob for ß-carotene assay.
Prepared WT gDNA for Dan W.
</P><H5><SPAN class="mw-headline" id="7.2F26.2F11">7/26/11</SPAN></H5><P>Try to PCR missing promoter and 3'UTRs from gDNA using adjusted annealing temperature.
</P><P>Promoter/UTR Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (138ng/µl)
</TD><TD>    1 µl
</TD></TR><TR><TD> F/R Primer mix (5µM)
</TD><TD>    5 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>   28.5 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>Control Reaction Mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 5x Herc. buffer
</TD><TD>   10 µl
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD>    5 µl
</TD></TR><TR><TD> gDNA (63ng/µl)
</TD><TD>    1 µl
</TD></TR><TR><TD> F Primer (20µM)
</TD><TD>  .625 µl
</TD></TR><TR><TD> R Primer (20µM)
</TD><TD>  .625 µl
</TD></TR><TR><TD> Herculase Polymerase
</TD><TD>   .5 µl
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD>  32.25 µl
</TD></TR><TR><TD> Total
</TD><TD>   50 µl
</TD></TR></TBODY></TABLE><P>PCR Program: 4 min @ 95°C, 30x(30s @ 95°C, 30s @ 58°C, 30s @72°C), 7 min @ 72°C, hold at 4°C. 
</P><P>Also ran 3 timepoints of ß-carotene assay w/Rob.
</P><H5><SPAN class="mw-headline" id="7.2F27.2F2011">7/27/2011</SPAN></H5><P>Gel extraction of 7/26 PCR products. Cut gels and took mass for purification, stored in 4°C overnight.
</P><H5><SPAN class="mw-headline" id="7.2F28.2F2011">7/28/2011</SPAN></H5><P>Finish purification of 7/26 PCR product according to <A href="https://static.igem.org/mediawiki/2011/6/61/Qiaquick_spin_handbook-1.pdf|Qiagen" class="external text" rel="nofollow">gel extraction with microcentrifuge</A>, ran 5µl on gel to check.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Sequence
</TD><TD> Concentration (ng/µl)
</TD><TD> Sequence
</TD><TD> Concentration (ng/µl)
</TD></TR><TR><TD> TDH3p
</TD><TD> 38.8
</TD><TD> FCY2p
</TD><TD> 41.8
</TD></TR><TR><TD> TDH3utr
</TD><TD> 21.3
</TD><TD> FCY2utr
</TD><TD> 26.1
</TD></TR><TR><TD> RPL24Ap
</TD><TD> 53.1
</TD><TD> PRY1p
</TD><TD> 14.9
</TD></TR><TR><TD> RPL24Autr
</TD><TD> 26.4
</TD><TD> PRY1utr
</TD><TD> 41.2
</TD></TR><TR><TD> RPS8Bp
</TD><TD> 40.9
</TD><TD> ARD1p
</TD><TD> 4.4
</TD></TR><TR><TD> RPS8Butr
</TD><TD> 9.7
</TD><TD> ARD1utr
</TD><TD> 21.1
</TD></TR><TR><TD> RPS2p
</TD><TD> 3.4
</TD><TD> KRE9p
</TD><TD> 66.3
</TD></TR><TR><TD> RPS2utr
</TD><TD> 16.5
</TD><TD> KRE9utr
</TD><TD> 54.3
</TD></TR><TR><TD> RPL8Ap
</TD><TD> 3.2
</TD><TD> HHO1p
</TD><TD> 48.5
</TD></TR><TR><TD> RPL8Autr
</TD><TD> 21.8
</TD><TD> HHO1utr
</TD><TD> 15.2
</TD></TR><TR><TD> STM1p
</TD><TD> 43.0
</TD><TD> BAP2p
</TD><TD> 8.4
</TD></TR><TR><TD> STM1utr
</TD><TD> 31.3
</TD><TD> BAP2utr
</TD><TD> 34.3
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="7.2F29.2F2011">7/29/2011</SPAN></H5><P>Linearize vector pTZ19u w/SmaI in order to blunt-end ligate the BsaI containing fragments in for cloning and attaching URA3 to UTRs.
</P><P>20µl digest
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> ptz19u (60ng/µl)
</TD><TD> 20
</TD></TR><TR><TD> NEBuffer 4 (10x)
</TD><TD> 5
</TD></TR><TR><TD> BSA (100x)
</TD><TD> .5
</TD></TR><TR><TD> SmaI
</TD><TD> 1
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> 23.5
</TD></TR><TR><TD> Total
</TD><TD> 50
</TD></TR></TBODY></TABLE><P>Left to digest overnight at room temperature (~23°C), rather than suggested temperature 25°C.
</P><H5><SPAN class="mw-headline" id="7.2F30.2F2011">7/30/2011</SPAN></H5><P>Heat-killed SmaI (20 min @ 65°C).
</P><P>Diluted pTZ19u 10x to 156.4 ng/µl (50µl total).
</P><P>Ran gel of digested and undigested pTZ19u.
</P><P>SmaI digest looks good; will go ahead with ligation.
</P><P>Ligation Mix (20µl)
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume
</TD></TR><TR><TD> 10x T4 Buffer
</TD><TD> 2µl
</TD></TR><TR><TD> SmaI linearized pTZ19u (58ng/µl)
</TD><TD> 1µl
</TD></TR><TR><TD> T4 ligase (120,000 U/ml)
</TD><TD> 1µl
</TD></TR><TR><TD> insert
</TD><TD> 6:1 molar ratio insert:vector
</TD></TR><TR><TD> H<SUB>2</SUB>O
</TD><TD> to 20µl
</TD></TR></TBODY></TABLE><P>Left in 16°C overnight.
</P><H5><SPAN class="mw-headline" id="8.2F1.2F2011">8/1/2011</SPAN></H5><P>Streaked out TOP10 e. coli on LB+carb for making competent cells.
</P><H5><SPAN class="mw-headline" id="8.2F2.2F2011">8/2/2011</SPAN></H5><P>The TOP10 plates were accidentally placed in 30°C incubator instead of 37°C incubator overnight, there was no growth at all. Also I think you have to streak out on plain LB or the carb will kill the cells as they come out of -80°C.
</P><P>Streaked out two new plates of TOP10 cells, one on LB, one on LB+carb, put in 37°C incubator along with the 8/1 plate.
</P><P>Came back in evening, some growth on LB only plate, but no colonies for inoculation yet, so will have to delay one day.
</P><H5><SPAN class="mw-headline" id="8.2F3.2F2011">8/3/2011</SPAN></H5><P>Inoculated 25ml LB with TOP10 E. coli, put in 37°C w/shaking.
</P><P>Made gDNA from orange yeasts according to this protocol <B>upload and link to protocol here</B>. Accidentally forgot to spin and discard supernatant before adding glass beads, so spun 5 min @ 14000 rpm after adding beads and then discarded supernatant. Got pretty low concentrations:
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Strain
</TD><TD> gDNA concentration (ng/µl)
</TD></TR><TR><TD> 039
</TD><TD> 54.6
</TD></TR><TR><TD> 179
</TD><TD> 85.9
</TD></TR><TR><TD> 201
</TD><TD> 85.1
</TD></TR><TR><TD> 128
</TD><TD> 33.4
</TD></TR><TR><TD> 128 Super Orange
</TD><TD> 81.1
</TD></TR><TR><TD> 128 White
</TD><TD> 63.1
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="8.2F4.2F2011">8/4/2011</SPAN></H5><P>Prepared competent cells according to <A href="/Protocols#Making_Competent_E._coli" title="Protocols">competent E. coli preparation protocol</A>.
</P><P>There were no LB Carb plates for transformations of the 7/30 ligation, so spread 90µl of an X-Gal/IPTG/Carb mix on plain LB plates.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> X-Gal (1000x)
</TD><TD> 25
</TD></TR><TR><TD> 100mM IPTG
</TD><TD> 40
</TD></TR><TR><TD> Carbenicillin (1000x)
</TD><TD> 25
</TD></TR><TR><TD> Total
</TD><TD> 90
</TD></TR></TBODY></TABLE><P>Followed <A href="/wiki/images/6/65/Oneshottop10_man_%281%29.pdf" class="internal" title="Oneshottop10 man (1).pdf">Top10 transformation protocol</A> for all promoters and UTRs except for RPS2utr (from 7/30 ligation). Spread one set of 50µl cells/ 35µl LB and one set of 200µl cells/ 85µl LB. Placed in 37°C overnight.
</P><H5><SPAN class="mw-headline" id="8.2F5.2F2011">8/5/2011</SPAN></H5><P>Blue/white colonies observed for both 50µl and 200µl plates, but were very small, so incubated further. Spread 25µl 1000x Carbenicillin on plain LB plates to streak out colonies. Picked one white colony for each promoter/UTR and restreaked; put in 37°C incubator. Blue/white plates moved to iGEM 4°C space.
</P><H5><SPAN class="mw-headline" id="8.2F6.2F2011">8/6/2011</SPAN></H5><P>Inoculated promoter/UTR E. coli into 5ml LB. Moved 8/5 plates to iGEM 4°C space.
</P><H5><SPAN class="mw-headline" id="8.2F8.2F2011">8/8/2011</SPAN></H5><P>All overnight cultures looked ok except for PRY1p, which was yellowish. Perhaps contamination?
</P><P>Miniprepped all promoter/UTR overnights according to <B>protocol</B>.
</P><P>Performed restriction digest with BsaI. If the ligation/transformation was successful, the promoter/UTR should be cut out of the vector.
</P><P>Made 15% glycerol stocks of E. coli w/correct fragments present, put in iGEM -80°C space.
</P><H5><SPAN class="mw-headline" id="8.2F9.2F2011">8/9/2011</SPAN></H5><P>PCR of URA3 to get product and add NotI site to add to 3' UTR.
</P><P>Diluted ura3_fwd and ura3_rev primers to 10µM (50µl 60µM primer, 250µl TE). 50µl reaction using Herculase II Polymerase.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> 5x Herculase Buffer
</TD><TD> 10
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD> 5
</TD></TR><TR><TD> gDNA (148.8ng/µl)
</TD><TD> .67 (100ng), 1.34 (200ng), 2.01 (300ng)
</TD></TR><TR><TD> forward primer (10µM)
</TD><TD> 1.25
</TD></TR><TR><TD> reverse primer (10µM)
</TD><TD> 1.25
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> to 50
</TD></TR><TR><TD> Total
</TD><TD> 50
</TD></TR></TBODY></TABLE><P>Control primers were 20µM (ZK096/097), used .625µl ea.
</P><P>PCR Program: 2 min @ 95°C, 30x (20s @95°C, 20s @64°C, 34s @72°C), 3 min @ 72°C
</P><P>Ran gel under long wave in preparation for extraction, only saw faint product for 300ng band, decided not enough DNA to cut.
</P><H5><SPAN class="mw-headline" id="8.2F10.2F2011">8/10/2011</SPAN></H5><P>Looking at 8/8 gels, need to redo ligation and transformation for TDH3utr, RPS8Butr, RPS2utr, RPL8Ap, RPL8Autr, STM1utr, FCY2utr, PRY1p, PRY1utr, ARD1utr, KRE9utr, HH01p, HH01utr, BAP2utr.
</P><P>Linearized pTZ19u w/SmaI.
</P><P>50µl reaction
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> pTZ19U (156.4 ng/µl)
</TD><TD> 7.67
</TD></TR><TR><TD> 10x NEBuffer 4
</TD><TD> 5
</TD></TR><TR><TD> BSA (100x)
</TD><TD> .5
</TD></TR><TR><TD> SmaI (20U/µl)
</TD><TD> 1
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 35.83
</TD></TR><TR><TD> Total
</TD><TD> 50
</TD></TR></TBODY></TABLE><P>2 hour digest, heat kill SmaI 65°C 20 min.
</P><P>20µl ligation mix
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> 10x T4 buffer
</TD><TD> 2
</TD></TR><TR><TD> T4 ligase (Enzymatics: 120U/µl)
</TD><TD> 1
</TD></TR><TR><TD> linearized pTZ19u (58ng/µl)
</TD><TD> 1
</TD></TR><TR><TD> insert
</TD><TD> 6x molar ratio insert/vector
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> to 20
</TD></TR><TR><TD> Total
</TD><TD> 20
</TD></TR></TBODY></TABLE><P>Moved to 25°C for overnight.
</P><P>Redid URA3 PCR. Trying primers suspended in H<SUB>2</SUB>O as well as in TE, and lowering the annealing temperature to 61°C vs 64°C.
</P><P>Both primer sets yielded a good product this time. Gel extracted using <A href="https://static.igem.org/mediawiki/2011/6/61/Qiaquick_spin_handbook-1.pdf" class="external text" rel="nofollow">Qiagen protocol with microcentrifuge</A>.
</P><H5><SPAN class="mw-headline" id="8.2F11.2F2011">8/11/2011</SPAN></H5><P>Heat killed ligase 65°C 20 min.
</P><P>Discovered that TOP10 cells do not require IPTG for blue/white screen.
Spread 25µl 1000x X-Gal/ 175µl H<SUB>2</SUB>O onto LB-carb plates, put in 37°C for 1 hour.
Followed <A href="/wiki/images/6/65/Oneshottop10_man_%281%29.pdf" class="internal" title="Oneshottop10 man (1).pdf">TOP10 transformation protocol (heat shock)</A>, spread one set 50µl cells/150µl LB, one set 200µl cells/100µl LB, put plates in 37°C overnight.
</P><H5><SPAN class="mw-headline" id="8.2F12.2F2011">8/12/2011</SPAN></H5><P>Blue/white colonies observed for all, except STMutr, which had no growth for both 50µl and 200µl plates.
Picked white colonies and restreaked on LB+carb, put in 37°C.
</P><P>PCR of ß-Carotene producing yeasts to determine which strain is which. The ORFs all have the same promoter and 3'UTR so after PCR with one set of primers I expect to see bands corresponding to each gene.
</P><P>50µl reaction
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> 2x GoTaq Green Master Mix
</TD><TD> 25
</TD></TR><TR><TD> Forward primer (10µM)
</TD><TD> 5
</TD></TR><TR><TD> Reverse primer (10µM)
</TD><TD> 5
</TD></TR><TR><TD> gDNA template
</TD><TD> 200ng
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> to 50
</TD></TR><TR><TD> Total
</TD><TD> 50
</TD></TR></TBODY></TABLE><P>PCR Program: 2 min @ 95°C, 30x(30s @ 95°C, 30s @ 52°C, 2 min @ 72°C), 5 min @ 72°C
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> ORF
</TD><TD> Expected Size (bp)
</TD></TR><TR><TD> crtE
</TD><TD> 1131
</TD></TR><TR><TD> crtYB
</TD><TD> 2234
</TD></TR><TR><TD> crtI
</TD><TD> 2011
</TD></TR><TR><TD> BTS1
</TD><TD> 1008
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="8.2F13.2F2011">8/13/2011</SPAN></H5><P>Inoculated 5ml LB w/streaked out white colonies with remaining promoters/UTRs, put in 37°C drum.
</P><P>Inoculated 25ml LB w/ TOP10 for competent cell preparation.
</P><H5><SPAN class="mw-headline" id="8.2F14.2F2011">8/14/2011</SPAN></H5><P>Began preparation of competent cells, but did not have time to finish, so abandoned this.
</P><P>Miniprepped 8/13 remaining promoter/UTRs.
</P><H5><SPAN class="mw-headline" id="8.2F15.2F2011">8/15/2011</SPAN></H5><P>BsaI restriction digest of 8/14 minipreps to check if promoters/UTRs ligated correctly.
</P><P>40µl reaction
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> 10x NEBuffer 4
</TD><TD> 4
</TD></TR><TR><TD> miniprep product
</TD><TD> 20
</TD></TR><TR><TD> BSA (100x)
</TD><TD> .75
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 15.25
</TD></TR><TR><TD> Total
</TD><TD> 40
</TD></TR></TBODY></TABLE><P>Incubated @ 50°C 2 hours.
</P><P>All fragments looked correct, so made 15% glycerol stocks of the E. coli they came from and stored in iGEM -80°C space.
</P><P>Picked another white colony and inoculated 5ml LB with E. coli containing pTZ19u with STM1utr SmaI ligated in.
</P><H5><SPAN class="mw-headline" id="8.2F16.2F2011">8/16/2011</SPAN></H5><P>Miniprepped and BsaI digested STM1utr, same protocol as 8/15.
</P><P>
No fragment observed.
</P><H5><SPAN class="mw-headline" id="8.2F17.2F2011">8/17/2011</SPAN></H5><P>Picked all white colonies (5) from 1st transformation for STM1utr, streaked out on LB+carb, put in 37°C incubator.
</P><P>The primers for RPS2utr arrived: Did 50µl reaction according to Herculase II protocol with 20µM control primers ZK096/097.
</P><P>PCR program: 2 min @ 95°C, 30x(20s @ 95°C, 20s @ 58°C, 30s @72°C), 3 min @ 72°C. PCR product to -20°C.
</P><H5><SPAN class="mw-headline" id="8.2F20.2F2011">8/20/2011</SPAN></H5><P>3/5 white colonies for STM1utr grew, but were too overgrown to pick single colonies, so were restreaked on LB+carb and put in 37°C.
</P><P>Ran gel of RPS2utr PCR product from 8/16, extracted.
</P><H5><SPAN class="mw-headline" id="8.2F22.2F2011">8/22/2011</SPAN></H5><P>Inoculated 5 ml LB w/STM1utr (3 colonies), put in 37°C drum.
</P><P>Gel purified RPS2utr according to Qiagen gel extraction protocol.
</P><P>Nanodrop of possible RPS2utr product: 8.0 ng/µl.
</P><P>Loaded 12.5µl (~100ng) in 2% MetaPhor agarose gel to verify product.
</P><P>Product is too small.
</P><H5><SPAN class="mw-headline" id="8.2F23.2F2011">8/23/2011</SPAN></H5><P>Streaked out ß-carotene yeasts on YPD, put in 30°C incubator for enzyme assays.
</P><P>Miniprepped 3 STM1utr samples.
</P><P>Did overnight BsaI digest @ 50°C according to 8/15 protocol.
</P><H5><SPAN class="mw-headline" id="8.2F24.2F2011">8/24/2011</SPAN></H5><P>Ran digested STM1utr:
</P><P>Got one good colony, made 15% glycerol perm of liquid culture (500µl 30% glycerol/500µl cells), put in -80°C w/other promoter/UTR <I>E. coli</I>.
</P><H5><SPAN class="mw-headline" id="8.2F25.2F2011">8/25/2011</SPAN></H5><P>Another PCR attempt for RPS2utr.
</P><P>50µl reaction using Herculase II protocol along with 20µM control primers (ZK096/097).
</P><P>PCR program: 2 min @95°C, (20s @95°C, 20s @45°C, 30s @72°C), 3 min @ 72°C
</P><P>Ran possible RPS2utr product on gel, extracted.
</P><P>Gel purified according to Qiagen Gel Extraction protocol, nanodropped cleaned product: 8.9ng/µl. Loaded 11.24µl (100ng) on gel to verify product.
</P><P>Product looks correct.
</P><H5><SPAN class="mw-headline" id="8.2F29.2F2011">8/29/2011</SPAN></H5><P>Ligation of RPS2utr PCR product into SmaI-digested pTZ19u.
</P><P>20µl reaction w/6 molar insert to vector ratio:
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> 10x T4 buffer
</TD><TD> 2
</TD></TR><TR><TD> SmaI linearized pTZ19u (25.2 ng/µl)
</TD><TD> 1
</TD></TR><TR><TD> T4 ligase (Enzymatics, 120U/µl)
</TD><TD> 1
</TD></TR><TR><TD> RPS2utr (8.9ngµl)
</TD><TD> .66
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 15.34
</TD></TR><TR><TD> Total
</TD><TD> 20
</TD></TR></TBODY></TABLE><P>Ligation at 25°C overnight.
</P><P>Streaked out 23 other promoter/3'UTR containing E. coli from -80°C glycerol stocks on LB+Carb, put in 37°C incubator for inoculation of liquid cultures for miniprepping to make stocks of fragments.
</P><H5><SPAN class="mw-headline" id="8.2F30.2F2011">8/30/2011</SPAN></H5><P>Transformed RPS2utr/pTZ19u ligation into TOP10 cells according to <A href="/wiki/images/6/65/Oneshottop10_man_%281%29.pdf" class="internal" title="Oneshottop10 man (1).pdf"> TOP10 transformation protocol</A>. Spread 50µl in 150µl LB and 200µl in 300µl LB on LB+Carb plates w/40µl 40mg/ml X-Gal spread previously in 160µl LB. To 37°C incubator.
</P><H5><SPAN class="mw-headline" id="9.2F2.2F2011">9/2/2011</SPAN></H5><P>Couldn't come in to take transformation plates out, plates are too overgrown to be any good.
</P><P>Miniprepped 23 promoters/3'UTRs according to E. Davis protocol.
</P><P>40µl BsaI digest of minipreps.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> 10x NEBuffer 4
</TD><TD> 4
</TD></TR><TR><TD> miniprep
</TD><TD> 18
</TD></TR><TR><TD> BsaI (10U/µl)
</TD><TD> 1
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 14.5
</TD></TR><TR><TD> Total
</TD><TD> 37.5
</TD></TR></TBODY></TABLE><P>50°C overnight incubation.
</P><H5><SPAN class="mw-headline" id="9.2F3.2F2011">9/3/2011</SPAN></H5><P>Heat kill BsaI 65°C 20 min.
Gel extraction of all parts.
</P><H5><SPAN class="mw-headline" id="9.2F9.2F2011">9/9/2011</SPAN></H5><P>Gel purification of all parts according to <A href="https://static.igem.org/mediawiki/2011/6/61/Qiaquick_spin_handbook-1.pdf" class="external text" rel="nofollow">Qiagen gel extraction protocol</A>.
</P><P>Nanodrop concentrations of products (ng/µl)
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Fragment
</TD><TD> concentration (ng/µl)
</TD><TD> Fragment
</TD><TD> concentration (ng/µl)
</TD></TR><TR><TD> TDH3p
</TD><TD> 14.9
</TD><TD> FCY2p
</TD><TD> 22.0
</TD></TR><TR><TD> TDH3utr
</TD><TD> 6.2
</TD><TD> FCY2utr
</TD><TD> 6.0
</TD></TR><TR><TD> RPL24Ap
</TD><TD> 9.0
</TD><TD> PRY1p
</TD><TD> 13.4
</TD></TR><TR><TD> RPL24Autr
</TD><TD> 4.5
</TD><TD> PRY1utr
</TD><TD> 8.1
</TD></TR><TR><TD> RPS8Bp
</TD><TD> 22.0
</TD><TD> ARD1p
</TD><TD> 8.0
</TD></TR><TR><TD> RPS8Butr
</TD><TD> 5.3
</TD><TD> ARD1utr
</TD><TD> 6.5
</TD></TR><TR><TD> RPS2p
</TD><TD> 18.8
</TD><TD> KRE9p
</TD><TD> 6.9
</TD></TR><TR><TD> RPS2utr
</TD><TD> ----
</TD><TD> KRE9utr
</TD><TD> 8.4
</TD></TR><TR><TD> RPL8Ap
</TD><TD> 11.1
</TD><TD> HHO1p
</TD><TD> 7.8
</TD></TR><TR><TD> RPL8Autr
</TD><TD> 13.6
</TD><TD> HHO1utr
</TD><TD> 7.5
</TD></TR><TR><TD> STM1p
</TD><TD> 15.2
</TD><TD> BAP2p
</TD><TD> 15.9
</TD></TR><TR><TD> STM1utr
</TD><TD> 5.9
</TD><TD> BAP2utr
</TD><TD> 19.2
</TD></TR></TBODY></TABLE><P>Ran 100ng each to verify product.
</P><P>DNA is not very visible, will do another round of minipreps. 
</P><P>Streaked out another round of 23 promoters and 3'UTRs from -80°C on LB+Carb, put in 37°C.
</P><H5><SPAN class="mw-headline" id="9.2F10.2F2011">9/10/2011</SPAN></H5><P>Inoculated 5ml LB w/colonies from overnight plates. Plates to cold room, liquid cultures to 37°C drum.
</P><P>Hybridized loxPsym oligos: Resuspended ssDNAs to 100µM, took 10µl of each in one tube, put @ 94°C 10 minutes, moved to bench and then to cold room for slow cooling.
</P><P>Nanodropped ss + strand, ss - strand, and ds loxPsym linker.
+ strand- 1360ng/µl
- strand- 1345ng/µl
double stranded- 2096.7ng/µl
</P><P>Ran 1µg of each on 2% MetaPhor agarose gel.
</P><H5><SPAN class="mw-headline" id="9.2F11.2F2011">9/11/2011</SPAN></H5><P>Miniprepped all 23 overnight cultures according to Marty Taylor's <A href="/wiki/images/4/4d/Miniprep_without_columns_mst.pdf" class="internal" title="Miniprep without columns mst.pdf">protocol</A>, put liquid cultures in cold room.
</P><H5><SPAN class="mw-headline" id="9.2F12.2F2011">9/12/2011</SPAN></H5><P>Nanodrop all minipreps from 9/11.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Fragment
</TD><TD> Concentration (ng/µl)
</TD><TD> Fragment
</TD><TD> Concentration (ng/µl)
</TD></TR><TR><TD> TDH3p
</TD><TD> 292.1
</TD><TD> FCY2p
</TD><TD> 116.4
</TD></TR><TR><TD> TDH3utr
</TD><TD> 288.2
</TD><TD> FCY2utr
</TD><TD> 298.4
</TD></TR><TR><TD> RPL24Ap
</TD><TD> 84.0
</TD><TD> PRY1p
</TD><TD> 253.1
</TD></TR><TR><TD> RPL24Autr
</TD><TD> 385.0
</TD><TD> PRY1utr
</TD><TD> 308.3
</TD></TR><TR><TD> RPS8Bp
</TD><TD> 317.6
</TD><TD> ARD1p
</TD><TD> 54.1
</TD></TR><TR><TD> RPS8Butr
</TD><TD> 235.8
</TD><TD> ARD1utr
</TD><TD> 299.2
</TD></TR><TR><TD> RPS2p
</TD><TD> 338.1
</TD><TD> KRE9p
</TD><TD> 290.7
</TD></TR><TR><TD> ------
</TD><TD> KRE9utr
</TD><TD> 566.7
</TD></TR><TR><TD> RPL8Ap
</TD><TD> 307.8
</TD><TD> HHO1p
</TD><TD> 292.9
</TD></TR><TR><TD> RPL8Autr
</TD><TD> 280.5
</TD><TD> HHO1utr
</TD><TD> 175.4
</TD></TR><TR><TD> STM1p
</TD><TD> 27.3
</TD><TD> BAP2p
</TD><TD> 262.7
</TD></TR><TR><TD> STM1utr
</TD><TD> 354.2
</TD><TD> BAP2utr
</TD><TD> 337.8
</TD></TR></TBODY></TABLE><P>Digested all promoters with BsaI, split 3'UTRs into two aliquots, digested one with NotI, then realized that in order to cut the entire 3'UTR and URA3 out I would have to digest with BsaI, which works in buffer 4, while NotI works in NEBuffer 3. Digested the other UTR aliquot with NotI-HF, which works in NEBuffer 4.
</P><H5><SPAN class="mw-headline" id="9.2F16.2F2011">9/16/2011</SPAN></H5><P>PCR of URA3 w/Herculase. 8 reactions in parallel.
</P><P>50µl reaction
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> gDNA template (156 ng/µl)
</TD><TD> 1
</TD></TR><TR><TD> Herculase II polymerase
</TD><TD> 1
</TD></TR><TR><TD> Herc. II 5x Buffer
</TD><TD> 10
</TD></TR><TR><TD> URA3 fwd primer (10µM)
</TD><TD> 1.25
</TD></TR><TR><TD> URA3 rev primer (10µM)
</TD><TD> 1.25
</TD></TR><TR><TD> dNTP mix (2.5mM ea.)
</TD><TD> 5
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 30.5
</TD></TR><TR><TD> Total
</TD><TD> 50
</TD></TR></TBODY></TABLE><P>PCR program: 2min @95°C, 30x(20s @ 95°C, 20s @42°C, 34s @ 72°C), 3 min @ 72°C.
</P><P>Got 8 very bright bands at correct size (1134bp), extracted all and purified according to <A href="https://static.igem.org/mediawiki/2011/6/61/Qiaquick_spin_handbook-1.pdf" class="external text" rel="nofollow">Qiagen gel extraction protocol</A></P><P>Nanodropped 8 URA3 gel purification products:
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> URA3 fragment
</TD><TD> Concentration (ng/µl)
</TD></TR><TR><TD> 1
</TD><TD> 28.8
</TD></TR><TR><TD> 2
</TD><TD> 21.6
</TD></TR><TR><TD> 3
</TD><TD> 21.9
</TD></TR><TR><TD> 4
</TD><TD> 23.2
</TD></TR><TR><TD> 5
</TD><TD> 28.5
</TD></TR><TR><TD> 6
</TD><TD> 33.7
</TD></TR><TR><TD> 7
</TD><TD> 33.8
</TD></TR><TR><TD> 8
</TD><TD> 23.8
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="9.2F17.2F2011">9/17/2011</SPAN></H5><P>NotI-HF digests of all 8 URA3 PCR products.
</P><P>55µl digest
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (ng/µl)
</TD></TR><TR><TD> URA3 PCR product
</TD><TD> 48
</TD></TR><TR><TD> 10x NEBuffer 4
</TD><TD> 5.5
</TD></TR><TR><TD> BSA (100x)
</TD><TD> .55
</TD></TR><TR><TD> NotI-HF (20U/µl)
</TD><TD> .3
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> .65
</TD></TR><TR><TD> Total
</TD><TD> 55
</TD></TR></TBODY></TABLE><P>2 hour digest @ 37°C, 20 min heat kill @ 65°C.
</P><P>Ligation of URA3 into 3'UTRs, also a ligation of plain RPS2utr.
20µl ligation (same protocol as 7/30. 1 hour at 25°C.
Transformed into DH5α competent cells (<A href="/wiki/images/6/65/Oneshottop10_man_%281%29.pdf" class="internal" title="Oneshottop10 man (1).pdf">protocol</A>). Spread two volumes of cells, 50µl cells in 250µl LB and 200µl cells in 300µl LB.
</P><H5><SPAN class="mw-headline" id="9.2F18.2F2011">9/18/2011</SPAN></H5><P>Transformations showed mixed results. Many plates showed only white colonies. Picked 7 colonies from good plate and 2 colonies from bad plates for miniprep and digestion to check for URA3 ligation. Inoculated in 5ml LB+carb.
</P><H5><SPAN class="mw-headline" id="9.2F19.2F2011">9/19/2011</SPAN></H5><P>Miniprepped 9/18 liquid cultures [<A href="https://static.igem.org/mediawiki/2011/6/61/Qiaquick_spin_handbook-1.pdf|protocol" class="external autonumber" rel="nofollow">[1]</A>].
</P><H5><SPAN class="mw-headline" id="9.2F20.2F2011">9/20/2011</SPAN></H5><P>BsaI digests of URA3 3'UTR ligations.
</P><P>50µl digests
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (µl)
</TD></TR><TR><TD> Miniprep DNA (~1.2µg/µl)
</TD><TD> 1
</TD></TR><TR><TD> 10x NEBuffer 4
</TD><TD> 5
</TD></TR><TR><TD> BSA (100x)
</TD><TD> .5
</TD></TR><TR><TD> BsaI (10U/µl)
</TD><TD> 1
</TD></TR><TR><TD> dH<SUB>2</SUB>O
</TD><TD> 42.5
</TD></TR><TR><TD> Total
</TD><TD> 50
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="9.2F23.2F2011">9/23/2011</SPAN></H5><P>Ran gels of 9/20 digests.
</P><P>Made SC-his-leu-ura plates for Vitamin C transformations.
</P><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Component
</TD><TD> Volume (ml)
</TD></TR><TR><TD> 2x SC--8
</TD><TD> 250
</TD></TR><TR><TD> 20% galactose
</TD><TD> 50
</TD></TR><TR><TD> 40mM trp
</TD><TD> 5
</TD></TR><TR><TD> 16mM ade
</TD><TD> 5
</TD></TR><TR><TD> 100mM lys
</TD><TD> 5
</TD></TR><TR><TD> 50mM met
</TD><TD> 6
</TD></TR><TR><TD> 4% agar
</TD><TD> 250
</TD></TR></TBODY></TABLE><H5><SPAN class="mw-headline" id="9.2F30.2F2011">9/30/2011</SPAN></H5><P>Finally had time to look at the 9/23 gels. Turns out there is an internal BsaI site in the URA3 sequence, so will have to find a BsaI/BsmBI free URA3 sequence and redo the 3'UTR tagging.
</P><H5><SPAN class="mw-headline" id="10.2F1.2F2011">10/1/2011</SPAN></H5><P>Ligation of RPS2utr into pTZ19u.
</P><H5><SPAN class="mw-headline" id="10.2F2.2F2011">10/2/2011</SPAN></H5><P>Picked 5 colonies from each transformation (3), and inoculated 5ml LB+carb.
</P><H5><SPAN class="mw-headline" id="10.2F3.2F2011">10/3/2011</SPAN></H5><P>Made 15% glycerol stocks of the possible RPS2utr bacteria. (No time to miniprep)
</P><H5><SPAN class="mw-headline" id="10.2F7.2F2011">10/7/2011</SPAN></H5><P>Restreaked the 15 possible RPS2utr strains onto LB plates.
</P><H5><SPAN class="mw-headline" id="10.2F8.2F2011">10/8/2011</SPAN></H5><P>Inoculated 5ml LB+carb with RPS2utr candidates.
</P><H5><SPAN class="mw-headline" id="10.2F9.2F2011">10/9/2011</SPAN></H5><P>Didn't have time to miniprep today, will redo next week.
</P><H5><SPAN class="mw-headline" id="10.2F16.2F2011">10/16/2011</SPAN></H5><P>Miniprepped the RPS2utr candidates, put into -20°C.
</P><H5><SPAN class="mw-headline" id="10.2F23.2F2011">10/23/2011</SPAN></H5><P>Digested yeast shuttle vectors with 4 igem sites to verify single cuts.
</P><H5><SPAN class="mw-headline" id="10.2F28.2F2011">10/28/2011</SPAN></H5><P>Inoculated VitC constructs with Rob to miniprep and make expression cassettes.
</P><H6><SPAN class="mw-headline" id="Violacein_Biosynthesis_in_Yeast">Violacein Biosynthesis in Yeast</SPAN></H6><H5><SPAN class="mw-headline" id="June_7th.2C_2011">June 7th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><OL><LI> Make 4 copies of violacein building blocks and select promoters and 3’UTRs on 96 well plates.  2 were made previously by Dr. Boeke's Lab Tech and I (not entered into wiki).  Therefore I am only making 2 new copies today.
</LI><LI> Begin growing colonies on petri dishes (to prepare for miniprep).
</LI></OL><P><B>Procedure</B>:
</P><UL><LI> Made 100ml LB+Carbenicillin Solution (ampicillin resistance gene also resists Carbenicillin) using 100uL 1000x Carbenicillin (thawed from -20 degrees Celsius) and 100mL LB Broth.
</LI><LI> Labelled plates and Petri Dishes
</LI><LI> Except for one petri dish (forgot to add a label), each label says: “iGEM: Boeke promoter, 3’ end &amp; Vio BB’s from YW (pseudonym); F2.”
</LI></UL><LI> Transferred LB+Carb solution to a ‘sterile reagent reservoir for multi-channel pipetting. 
</LI><LI> Took out a previous copy of the genes (arrayed on a 96 well plate) from -80 degree freezer, placed in a centrifuge with a balance.
</LI><LI> Centrifuged until speed reached 1000g (short spin).
</LI><LI> Used a 12-lane multichannel micropipette to transfer 100uL of LB+Carb to rows A-D on new 96 well plates.
</LI><LI> Placed air-pore tape on each new 96 well plate (now inoculated) and incubated them at 37 degrees Celsius.  
</LI><LI> Incubated petri dishes at 37 degrees.
</LI><P><B>Problems</B>:
</P><UL><LI> I added LB+carb to every well in rows a-d.  The original array only filled row A/C, half of row B and 11/12 wells in Row D.  
</LI><LI> Always remember how original array was organized and always stick to it.
</LI></UL><LI> Original plate (the one being copied) was left out for around 50min.  Not good.
</LI><LI> Next time, only take out plate from -80 after each new 96 well plate/petri dish has been prepared for inoculation (e.g. LB+Carb should be added to wells before original plate is taken out to thaw).
</LI><LI> I am still learning where everything in the lab is.  This will take time.
</LI><LI> When pipetting, I would scrape the tip onto the side of the well to get rid of the final drops.  This motion resulted in me almost mixing the original cultures.  
</LI><LI> Remember: Do not worry about “pipetting the last drop.”
</LI><LI> When inoculating petri dishes, the liquid sprayed out of the tip instead of dripping out.
</LI><LI> Don’t let the liquid spray out of the tip, the different cultures may mix.
</LI><H5><SPAN class="mw-headline" id="June_9th.2C_2011">June 9th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Inoculated test tubes (filled with ~ 5mL LB+Carb)with each part (Vio ORF Building Blocks (ie ORF fragments), Promoters and UTR's)
</LI></UL><P><B>Procedure</B>:
</P><UL><LI>  For each part, took wooden stick, placed flat end onto a single colony on the agar plate, dipped stick into the appropriate text tube (inoculating the test tube) and threw out the stick.
</LI><LI> Incubated them overnight at 37 degrees on a shaker (a slanted spinning wheel with slots for test tubes).
</LI></UL><H5><SPAN class="mw-headline" id="June_10th.2C_2011">June 10th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Mini-Prep of parts (Vio ORF Building Blocks, Promoters, UTR's)
</LI><LI> Double Cut Restriction Digest of Mini Prep Samples (to run an analytical gel to check if the correct inserts (parts) are present).
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> See <A href="/File:Miniprep_stet_prep_pm_.pdf" title="File:Miniprep stet prep pm .pdf">File:Miniprep stet prep pm .pdf</A> for boiling mini prep.
</LI><LI><A href="/wiki/images/3/34/June_10_restiction_digest_rxn_mixes.jpg" class="internal" title="June 10 restiction digest rxn mixes.jpg">See Double Digest Reaction Mixes here</A></LI><LI> All other parts (promoters and UTR's) double digested with XmaI and SphI)
</LI></UL><LI> See neb.com for correct digestion procedures with each enzyme.  
</LI><LI> After digestions completed, stored samples at -20 degrees Celsius.
</LI><P><B>Results</B>:
</P><P><B>Results</B> show high concentrations of plasmid, too high.  In reality there is RNA in the solution too and nanodrop detects nucleic acid without distinction for DNA or RNA.  According to Dr. Boeke’s Lab Tech, the readings still indicate that the DNA concentration is probably high.
</P><P><B>Problems</B>:
</P><DL><DD><B>Mini Prep</B>:  
</DD></DL><UL><LI> Instead of boiling all 29 samples at once, I boiled 15 and then 14.  For the first 15, I boiled for 2min 45sec and not the required 3 minutes.  The reason: the water was not boiling when we entered the samples.  At the last minute, we had to refill the bath with hot water and place the samples in.  We set the timer for 4 minutes and the water started to boil with 2minutes 45seconds left.  
</LI><LI> Sometimes during mini prep, I did not change the tip for each sample while aspirating supernatant.
</LI></UL><H5><SPAN class="mw-headline" id="June_11th.2C_2011">June 11th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Gel Electrophoresis of Yesterday's Digests
</LI></UL><P><B>Procedure</B>:
</P><UL><LI>   Added 4uL CS Red Dye (with added RNAse) to each sample and 2 log ladder.
</LI></UL><P><B>Problems</B>:
</P><UL><LI> Added 7uL Dye to VioA2
</LI><LI> Ladder Didn't show up on the gel (must redo tomorrow).
</LI></UL><H5><SPAN class="mw-headline" id="June_13th.2C_2011">June 13th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Redo Digest (No image uploaded), the ladders on the gel did not show up.  Will use 2 log ladder with blue Juice this time and not CS Red. Will add a second digest condition for VioE, using a BsaI single cut.
</LI><LI> Note:  Although the ladder couldn't be seen properly, we agreed that the patterns/digests were correct and matched what we expected.  We believe there is nothing wrong with our restriction enzymes or our starting material (parts). 
</LI></UL><P><B>Procedure</B>:
</P><UL><LI><A href="/wiki/images/8/82/June_13_double_digest_rxn_mix.jpg" class="internal" title="June 13 double digest rxn mix.jpg">See reaction here</A></LI><LI> See neb.com for correct digestion procedures.
</LI></UL><P><B>Problems</B>:
</P><UL><LI> Accidentally heated VioE2 tube with the others at 37 degrees.  After 37 degree heating, I Placed it in 50 degree water bath for around 1hour 55minutes.  Made a PCR tube of new VioE2 solution (according to above table) and used thermocycler (PCR machine) to heat it for 2hours at 65 degrees and refrigerate it (4 degrees) afterwards).  Still not the right temperature.  I made yet another PCR tube of VioE2 (VioE2=VioE cut with BsaI) and 'PCR'd' at 50 degrees (I refer to this tube now as 'VioE BsaI cut (3).' 
</LI><LI> &quot;R&quot; from Zhu lab took VioD BB’s and VioE2 out for me and placed the samples in my minus 20 freezer space.  I left the lab when I put them to digest.
</LI></UL><H5><SPAN class="mw-headline" id="June_14th.2C_2011">June 14th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Run a preparative gel (a gel for gel extraction/purification of Vio ORF Building Blocks).
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Made 200uL of 1% agarose gel with 1x Ethidium Bromide (diluted from 20000x) added.
</LI><LI> 25 thick lanes indented into gel.
</LI></UL><LI> Planned Lanes:
</LI><UL><LI> Note: Lane 25 for this run was for VioE BsaI Cut (3) (see June 13th's 'Problems' section).
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel looked fine.  Got expected band patterns.  See todays 'Problems' section
</LI></UL><P><B>Problems</B>:
</P><UL><LI> It's not good practice to take a picture of a gel before gel extraction, UV exposure can damage the DNA.  Hand held UV light can go to long wave, unlike the 'picture taker, which uses only short wave UV light'  Long form is less harmful to the DNA.  This gel must be re-done.
</LI><LI> Note: New practice is to allow pictures before gel extraction, but exposure to short wave UV light must be as short as possible (less than a split second is good).
</LI></UL><LI> See 'Blooper' section.  As we exposed the gel (today's image on the wiki) for a short time to short UV, we could have used it for gel extraction.  Unfortunately, it fell and shattered.
</LI><LI> I redid the digests according to the June 13th protocol to run a future analytic gel (to definitively see if the patterns are what we expect).
</LI><LI> Note: I also ran another exact set of digests in parallel for a future preparative gel for gel extraction.  
</LI><H5><SPAN class="mw-headline" id="June_15th.2C_2011">June 15th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Run an analytical gel to determine if the digests worked before we do a preparative gel.  
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Made 2x 200mL 1% gels with EB, exactly like the June 14th Gel.
</LI><LI> Made a 100mL gel with 25 thick, narrow lanes.  We decided that we should use a smaller gel for an analytical gel.  This way (out of 1 tube of 25uL digest mixture + CS Red) we can load 3uL for the analytical gel and the rest for the preparative gel.  I originally made a copy of each digest; one for analytical, one for preparative.  We decided to use one copy for both (the second set of digests I made remain unused).
</LI><LI> 2 log ladder in lane 2, 100bp ladder in lane 24.
</LI></UL><P><B>Data</B>:
</P><P><B>Problem</B>:
</P><UL><LI> Accidentally switched the 2nd copy Vio B BB1 (which should have remained unused) with the first copy of VioB BB2 (which should have been loaded).  It was a mis-labelling error.  There are 2 vio B BB1 samples in the analytical gel (lanes 5 and 7), and no VioB BB2.  We decided we could proceed to the preparative gel step as I found the 1st copy VioB BB2.  
</LI></UL><H5><SPAN class="mw-headline" id="June_19th.2C_2011">June 19th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Plan with Zheng how to do a ligation reaction (I planned today to digest the vector  in preparation for the gel extraction and future ligation of ORF buidling blocks).
</LI></UL><UL><LI> Final Planned Procedure for Ligation:
</LI></UL><UL><LI> Note: If using enzymatics reagents, ligate at room temp for an hour.  If NEB reagents, 16 degrees Celsius (from now on, all temperature measurements are in Celsius) overnight. 
</LI></UL><H5><SPAN class="mw-headline" id="June_20th.2C_2011">June 20th, 2011</SPAN></H5><P><B>Purpose</B>: Gel Extraction
</P><P><B>Procedure</B>: See Qiagen Gel Purification protocol, June 13th Gel Specifics
</P><UL><LI> Only 16uL VioE 2x RE Cut loaded into the gel.
</LI><LI> 120V 15min, 10V 15min, 130V 10 Min (all to try and separate 2 bands, only 1 of which we wanted to extract.
</LI><LI> Left for home after QG, NaAcetate and 100% Isopropanol were added to the excised gel fragments (experiment not finished yet).  A post-doc said this is OK to do.
</LI></UL><H5><SPAN class="mw-headline" id="June_21st.2C_2011">June 21st, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Finish gel purification
</LI></UL><P><B>Results</B>:
</P><UL><LI> DNA concentrations are small, but workable.  I digested 30uL of each ORF with 1uL of each enzyme and added dye just in case.
</LI></UL><H5><SPAN class="mw-headline" id="June_22nd.2C_2011">June 22nd, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Just came in to add dye to the VioD digests (they need to be digested at 2 temps, which is why I didn't add dye to them with the others).
</LI></UL><H5><SPAN class="mw-headline" id="June_30th.2C_2011">June 30th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Replica Plate transformed colonies from LB+Carb plates to LB+Carb+XGal+IPTG Plates.
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Mixed 3ml LB with 500uL 1000x IPTG and 500uL 1000x XGal.
</LI><LI> Spread 19X200uL aliquots of this solution onto 19 LB+Carb Plates using sterile glass beads.  
</LI><LI> Note: I selected 15 out of 31 plates for replica plating and made 4 extra Xgal plates just in case.
</LI></UL><LI> Dr. Boeke said to let transformed colonies grow more at 37 degrees.  
</LI><LI> Left the 15 selected plates in the 37 degree incubator, left the new xgal plates to soak on my bench, placed all other plates in the 4 degree fridge.
</LI><P><B>Problems</B>:
</P><UL><LI> Wasted lots of time thawing the xgal because its solvent is DMSO and I left it on ice to thaw.  Note:  Never leave DMSO on ice to thaw, it will not melt. Do not even leave thawed DMSO on ice, it will refreeze quickly.
</LI><LI> Be more careful when pouring beads, wasted a couple.
</LI></UL><H5><SPAN class="mw-headline" id="July_1st.2C_2011">July 1st, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Replica Plate transformed colonies from LB+Carb plates to LB+Carb+XGal+IPTG Plates (made yesterday).
</LI><LI> Inoculate glass tubes for mini prep for each white colony on the xgal destination plates.
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Drew a line on original plate and destination plate.  Used it to make sure destination plate colonies had the same orientation as original plate.  Looked for white colonies in the evening.
</LI><LI> Pressed original plate onto a sterile velvet, pressed empty destination plate onto the newly 'dirty' velvet to pick up the colonies.   
</LI><LI> Filled glass tubes with 5mL LB+5uL Carb, labeled them according to ligation condition (16 degree or room temp incubation), transformation volume (e.g. 180uL transformation mixture added to the plate) &amp; sample name (e.g. VioA, VioB,...VioE).
</LI><LI> For all white colonies on the destination plates, I found the colony location on the destination plate and picked it with a wooden stick(pressed the flat end onto the colony, avoiding small satellite colonies).  I then shook the picked colony in the LB mixture vigorously, threw the toothpick, and placed the tubes in a 37 degree shaking incubator overnight.
</LI></UL><H5><SPAN class="mw-headline" id="July_2nd.2C_2011">July 2nd, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Miniprep from yesterday's inoculated tubes.
</LI><LI> Digest 
</LI></UL><P><B>Procedure</B>: 
</P><H5><SPAN class="mw-headline" id="July_3rd.2C_2011">July 3rd, 2011</SPAN></H5><P><B>Purpose</B>: 
</P><UL><LI> Run an analytical get to see if the correct inserts are present.
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Prepare a 200mL 1% agarose gel with EB and 25 think lanes.
</LI><LI> Add 10uL of digest and CS red dye.
</LI><LI> Run at 150V and take picture after 40min.
</LI></UL><P><B>Results</B>:
</P><UL><LI> C, D, E ligated well, but E-90-R did not ligate.
</LI><LI> Dr. Boeke suggests making replica plates of all other A, B plates and searching for a white colony.
</LI></UL><H5><SPAN class="mw-headline" id="July_5th.2C_2011">July 5th, 2011</SPAN></H5><P><B>Purpose</B>: 
</P><UL><LI> Make 14 replica plates of A/B
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Used 25mL of agar, 25uL of IPTG and Xgal, and 150mL LB
</LI><LI> Mix consists of 375uL of Xgal, 375uL of IPTG, and 2250uL of LB for a total volume of 3000uL.
</LI><LI> 30, 90, and R 30uL plates held at 16 degrees Celsius.
</LI></UL><P><B>Results</B>:
</P><UL><LI> VioA and VioE 30uL RT had no colonies.
</LI><LI> Because the original colonies were very small, the 90uL B was not re-plated to allow it to grow more.
</LI><LI> All colonies incubated at 30 degrees Celsius to try to lower the amount of Halos.
</LI><LI> Note: Halos: Original colony secretes resistance protein for other colonies to leech off of. Only good colonies survive in liquid suspension, as Halos can't effectively leech anymore. Streaking on a new plate kills most Halos too.
</LI></UL><H5><SPAN class="mw-headline" id="July_6th.2C_2011">July 6th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Find whites on replica plate B-90 1b and inoculate the white on the A XGal plate
</LI><LI> June 27th ligation: aliquot 5uL and save, leave the rest overnight.
</LI><LI> Transformation (June 27th protocol)
</LI><LI> Protocol:
</LI><LI> Transform dilute PTE 19 after first A/B tubes heat shocked.
</LI><LI> Put ligation tubes back to 16 degrees Celsius after heat shock of vector and negative control.
</LI></UL><P><B>Results</B>:
</P><UL><LI> One plate of A-30-RT has a white-ish colony (Placed in 37 degrees Celsius to see if it turns blue
</LI><LI> All useless inoculated plates from yesterday thrown out (except the whitish A, which was put in 4 degrees Celsius). 
</LI><LI> No DNA control full of bacteria
</LI><LI> PTE19 plate empty
</LI><LI> PTE-LB plate chalk full
</LI><LI> Conclusion: No DNA plate and PTE19 plate (LB-Carb) switched. No DNA bacteria went to PTE19 plate and vice-versa. PTE cannot be toxic as it grows on LB.
</LI></UL><H5><SPAN class="mw-headline" id="July_7th.2C_2011">July 7th, 2011</SPAN></H5><UL><LI> Protocol:
</LI><LI> Inoculate from July 5th VioB (85uL, 45uL at 16 degrees Celsius) and July 30th VioA (180uL at 16 degrees Celsius)
</LI><LI> Replica plate July 6th VioA (30-90-180 at 16 degrees Celsius)
</LI><LI> PTE19 plate in 4 degrees Celsius and negative DNA and PTE LB (proof)
</LI></UL><P><B>Results</B>:
</P><UL><LI> &quot;All whitish A&quot; from yesterday was blue...not using it.
</LI><LI> July 5th 15uL VioB had ambiguous large colonies, want to let it grow
</LI></UL><H5><SPAN class="mw-headline" id="July_8th.2C_2011">July 8th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Miniprep B-T 16-285 a, B-T 16-245 &amp; A 16 180 (Take 3mL of each and split into 2 1.5mL tubes)
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Aided STET suspension by heating at 37 degrees Celsius for 5min
</LI><LI> After double digest, run 100mL 1% agarose gel with EB for 25min
</LI><LI> Inoculate relica whites if gel doesn't work
</LI><LI> Vortexed B-T456 and A minipreps before adding to digest mix
</LI><LI> Run 50mL 1% agarose gel with EB for 25min at 150V
</LI></UL><P>- Lanes: Ladder, A, B-45, B-85, Ladder
</P><P><B>Results</B>:
</P><UL><LI> Inoculated B-T 16-285 b and c were clear, with no bacteria
</LI><LI> 15uL VioB had no colonies
</LI><LI> VioA 185 and B 95 had one white colony each
</LI><LI> Gel broke, but probably still worked
</LI><LI> Dr. Boeke suggests digesting with the middle restriction enzyme (for the unique restriction site that stitched the BBs together)to make sure to make competent cells, design primers to add BSAI sites and unique sequences for each Vio expression cassette objective, and stitch all cassettes to the neochromosome in one go. 
</LI><LI> Suggestions: Whenever measuring pieces, always digest the vector alone and run it alongside the others. Also, run gels longer, add EB to TE.
</LI></UL><H5><SPAN class="mw-headline" id="July_9th.2C_2011">July 9th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Make sure no internal BsaI or BsnbI sites are in any designs.
</LI><LI> Double digest within regions (no E required, at least 3 Bs)
</LI><LI> Single digest with BsaI
</LI><LI> Choose the more plentiful Eu out of XmaI and SphI
</LI><LI> Design primers according to the incoming Boeke diagram
</LI></UL><H5><SPAN class="mw-headline" id="July_10th.2C_2011">July 10th, 2011</SPAN></H5><P><B>Procedure</B>:
</P><UL><LI> Run gel at 150V for 1 hour (500mL, 1% agarose)
</LI><LI> Lanes: Ladder, A B C D (BsaI), A B B2 B3 C D, Ladder
</LI><LI> Add 400mL TE and and EB
</LI></UL><P><B>Results</B>:
</P><UL><LI> Digest went too long, need to redo
</LI><LI> BsaI and DraII at 37 degrees Celsius for 50min
</LI><LI> XbaI is gone - finished, will stick with a DraIII, BspHI 28cut
</LI><LI> Make mixes
</LI></UL><H5><SPAN class="mw-headline" id="July_11th.2C_2011">July 11th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Make 500uL of CS red dye with 0.1ug/uL of RNAse
</LI><LI> Digests
</LI></UL><H5><SPAN class="mw-headline" id="July_12th.2C_2011">July 12th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Run gel at 150V for 75min (1% agarose with EB in gel and buffer, 5uL CS red dye per sample)
</LI><LI> design Boeke oligos
</LI><LI> Protocol:
</LI><LI> Lanes: Ladder, BsaI A B B B, Ladder, BsaI C C D D, Ladder
</LI></UL><P><B>Results</B>:
</P><H5><SPAN class="mw-headline" id="July_13th.2C_2011">July 13th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Run 50mL 1% agarose gel with EB (300mL TTE &amp; 15uL EB for 45min at 150V)
</LI></UL><P><B>Results</B>:
</P><UL><LI> VioB is not here :(  Transform overnight ligations, miniprep and try again
</LI></UL><H5><SPAN class="mw-headline" id="July_19th.2C_2011">July 19th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Digest and ligate B 1, 2, 3 to PTZ19u tomorrow
</LI><LI> Transform overnight ligations
</LI><LI> Design oligos
</LI></UL><H5><SPAN class="mw-headline" id="July_23th.2C_2011">July 23th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Ligations
</LI><LI> Finish oligos with Boeke
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Cooked vector at 65 degrees Celsius for 20min
</LI></UL><P><B>Results</B>:
</P><UL><LI> Will aliquot 6uL for transformations. Leave rest overnight
</LI><LI> Make sure melting temps for yeast DNA regions are nearly equal +/- 1-2 degrees best (so that lengths are nearly equal)
</LI></UL><H5><SPAN class="mw-headline" id="July_26th.2C_2011">July 26th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Cook all my ligation rxns for 10min at 65 degrees Celsius
</LI><LI> Digest available ORFs with BsaI in preparation for gel purification
</LI><LI> Strategy: Take bare minimum (may need to redo)
</LI></UL><H5><SPAN class="mw-headline" id="July_27th.2C_2011">July 27th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Gel purify ORFs
</LI><LI> Digest PTZ14u with BsaI to load with other 4 samples
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Gel lanes: Ladder, A C D E PTZ19u
</LI></UL><P><B>Results</B>:
</P><UL><LI> A is suspect, may be contaminated with upper band
</LI><LI> C is inseparable on this gel
</LI><LI> Mixed D and E, must redo and repeat July 26 digest
</LI><LI> Run bigger and longer gel next time
</LI></UL><H5><SPAN class="mw-headline" id="July_28th.2C_2011">July 28th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Gel extractions on long plate (300uL TTE and 3g agarose and 15uL EB)
</LI><LI> Procudure:
</LI><LI> Run for 1.5hr at 150V
</LI><LI> Lanes: Ladder, A C, Ladder, D E, Ladder
</LI><LI> Make new wide, thin gel (1.5% to run at 100-120V). Tape combs together and add 12.5uL per well.
</LI></UL><P><B>Results</B>:
</P><UL><LI> Blue band at ~300bp, gel was slow
</LI></UL><H5><SPAN class="mw-headline" id="July_29th.2C_2011">July 29th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Run gel for 2hrs
</LI><LI> Purify what you can. If time permits, set up a double digest.
</LI></UL><P><B>Results</B>:
</P><UL><LI> DNA did not diffuse from the refrigeration. (Don't try to do too much in a day)
</LI><LI> A and D melting gel at 48 degrees Celsius, closest block available
</LI><LI> A: 3.2ug/uL
</LI><LI> D: 9.1ug/uL
</LI><LI> C: highly suspect of contamination, 10.8ug/uL
</LI></UL><H5><SPAN class="mw-headline" id="August_1st.2C_2011">August 1st, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Transform B1, 2, 3
</LI><LI> Make 6x XGal plates (1 for 2hrs, one for overnight)
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> 2hr: 50 and 500 uL
</LI><LI> Overnight: 50 and 500uL (no DNA, just vector only) 
</LI></UL><H5><SPAN class="mw-headline" id="August_2nd.2C_2011">August 2nd, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Transform B 1, 2, 3
</LI><LI> Digest promoters with BsaI
</LI><LI> ligate to appropriate ORFs
</LI></UL><H5><SPAN class="mw-headline" id="August_4th.2C_2011">August 4th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Digest vector with XbaI and XmaI at 37 degrees Celsius for 0.5hr.
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Cook digest, run 50mL 1% agarose gel with EB of all BBs (ligation mix, individual parts, ladder, and uncut vector (all&gt;10ug))
</LI><LI> Transform with 1ug uncut vector
</LI><LI> Larger than 30uL...insert digest
</LI><LI> After 2hrs, check on gel and leave on for 3hrs.
</LI></UL><P><B>Results</B>:
</P><UL><LI> some wells too shallow
</LI></UL><H5><SPAN class="mw-headline" id="August_5th.2C_2011">August 5th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Ligation and transformation
</LI></UL><P><B>Results</B>:
</P><UL><LI> Did not digest vector first
</LI><LI> added wrongly-digest vector
</LI></UL><H5><SPAN class="mw-headline" id="August_6th.2C_2011">August 6th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Prep PTZ19u for ligation 
</LI><LI> Prep ligations to add digest vector
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> 200uL PB, vortex, column, spin for 1min, discard, add 750uL of TE, spin for 1min, discard, spin for 1min @1300rpm, discard, add 25uL EB, spin 1min (into new tube), nanodrop
</LI><LI> Add 25uL of water, 3uL T4, 3uL ligase. Split to three tubes, each containing 20uL. (Label 1:3, 1:5, 1:10)
</LI><LI> Add 1, 2, 3 to right tubes, add 1 to vector tubes.
</LI><LI> Ligate 2hrs overnight, transform 1uL, ligate
</LI></UL><H5><SPAN class="mw-headline" id="August_9th.2C_2011">August 9th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> Ran a gel to verify size/presence of VioB Building Blocks in previous gel purifications. 
</LI><LI> It’s not on my notes, but according to the brightness of the bands, I think I ran 1ul for the Elu product and 3 ul for B1 B2 B3 B4
</LI></UL><PRE> * ladder  B1Elu  B1  B2Elu  B2  B3Elu  B3  B4Elu  B4
</PRE><UL><LI> Redo Ligation: Purpose is to get VioB by ligation
</LI><LI> Rxn1: B1:3ul     B2:3ul     B3:3ul    vector:1ul     T4 ligase:1ul     T4 10xBuffer:2ul     H2O:7ul     Total=20ul
</LI><LI> Rxn1Control1: B1:3ul     B2:3ul     B3:3ul    XmaI XbaI vector:1ul     T4 10xBuffer:2ul     H2O:8ul     Total=20ul
</LI><LI> Rxn1Control2: XmaI XbaI vector:1ul     T4 ligase:1ul      T4 10xBuffer:2ul     H2O:16ul     Total=20ul
</LI><LI> Rxn2: B1:3ul     B2:3ul     B3:3ul    B4:3ul     vector:1ul     T4 ligase:1ul     T4 10xBuffer:2ul     H2O:4ul     * Total=20ul
</LI><LI> Rxn2Control1: B1:3ul     B2:3ul     B3:3ul    B4:3ul     XmaI SphI vector:1ul     T4 10xBuffer:2ul     H2O:5ul      * Total=20ul
</LI></UL><UL><LI> Rxn2Control2: XmaI SphI vector:1ul     T4 ligase:1ul      T4 10xBuffer:2ul     H2O:16ul     Total=20ul
</LI></UL><PRE>Ligate for 2 hours
</PRE><UL><LI> Then use LA Taq to do PCR.  
</LI></UL><P>cocktail and DNA:
</P><UL><LI> Ligation DNA 1ul
</LI></UL><UL><LI><I>PCR Rxn Mix</I></LI></UL><P>Primer1: 1ul
Primer2:1ul
La Taq: 0.5ul
10x Buffer: 5ul
dNTP: 8ul
H2O:33.5ul
Total: 50 ul
</P><UL><LI> PCR condition:  
</LI></UL><P>94 degree 1 min
30 cycles:
	94 degree 30 seconds
	55 degree 30 seconds
	72 degree 1 min
72 degree 10 min
4 degree infinitely
</P><UL><LI> PCR Gel:
</LI></UL><UL><LI> It’s hard to tell if there’s any band below the first well.  I inverted, and adjusted many times on the picture taking machine and saw a super light band which can’t be shown on the picture.  
</LI><LI> It’s not on my notes, but I sense this is a picture of gel which contain 2 product, the new VioB is the PCR version of my ligation product, the old VioB may be yours ligation product which I might also PCRed. 
</LI></UL><UL><LI> In the following picture, only 2 wells are for iGEM.  Those 2 wells proved that puc19 vectors were purified.  Other wells are for my personal projects in the Boeke Lab.  
</LI></UL><UL><LI> Lastly, transformation: 4 controls and 2 rxns and 1 no DNA control
</LI></UL><H5><SPAN class="mw-headline" id="August_10th.2C_2011">August 10th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> “We’re waiting for the transformation from 2 a.m.  Transformation efficiency isn’t very high.  Rxn 2 with B1 B2 B3 and B4 only has one colony.  We did some troubleshooting with Qing (Member of the <A href="http://www.bs.jhmi.edu/MBG/boekelab/members.html" class="external text" rel="nofollow">Boeke Lab</A>).  She said that our vector band is really bright, so its concentration is probably very high and our pieces of VioB [are not very] concentrated.  Although we used 3:1 volume ratio of B1 B2 B3 B4 and the vectors, it’s not good enough.”  
</LI></UL><UL><LI> Plates Images: (probably saw colonies around 4 p.  I think I incubated them at 37deg.)  
</LI></UL><H5><SPAN class="mw-headline" id="August_11th.2C_2011">August 11th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> Addition to August 10th—picked a white colony for liquid culture in the 37 degree shaker.  I then used Patrick Cai’s Miniprep protocol and stopped at the first step.
</LI></UL><UL><LI> I transformed 5ul ligation product by using Patrick’s competent cell which are super hot as Qing suggested.  And put them into 37 deg overnight.
</LI></UL><UL><LI> And we used herculase enzyme and protocol to do another PCR of VioB.  We got a super bright band:
</LI></UL><UL><LI> The upper one is around 3kb.  There is a double band situation.  May be need further explanation.  
</LI></UL><UL><LI> Then I transformed this PCR version VioB into E.coli, I believe it’s Patrick’s competent cell.  He tested them they’re super hot.  Make them overnight at 37 degree.  We get them next morning I believe.
</LI></UL><UL><LI> Here’s the plates:
</LI></UL><H5><SPAN class="mw-headline" id="August_12th.2C_2011">August 12th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> I continued miniprep of VioB’s first time transformation on 8/9. (The time with only one white colony of B1 B2 B3 B4 + vector)
</LI></UL><UL><LI> Then I digested the minipreped white colonies from first time transformation of VioB.  
</LI></UL><P>I checked with the <A href="http://biologylabs.utah.edu/jorgensen/wayned/ape/" class="external text" rel="nofollow">ApE(Universal)computer program</A> of how to digest.
</P><UL><LI> Digest Rxn Mix:
</LI></UL><PRE>         1 x Rxn   10 x Rxn
</PRE><P>DNA	  2 ul          /
EcoRI	  0.25 ul	2.5 ul
Buffer 4  2 ul	        20 ul
10 x BSA  2 ul	        20 ul
H2O	  13.75 ul	137.5 ul
</P><UL><LI> Aliquot 18 ul per tube.  Incubate @ 37 degree for an hour. 
</LI></UL><UL><LI> Run gels of the miniprep: (Rxn11 means colony 1 which has been circled and numbered on the plate of Rxn 1, Rxn12 13 14 15 mean the same thing; R1C2dot?(Rxn1 Control2 dot question mark))
</LI></UL><UL><LI> Gel lane order:
</LI></UL><P>ladder Rxn11 Rxn12 Rxn13 Rxn14 Rxn15 Rxn21 R1C2dot? R2C1dot?1 R2C1dot?2 
</P><UL><LI> Gel:
</LI></UL><UL><LI> The plates I made on 8/11, the ones with 5 ul ligation product transformed.  I believe what I used on 8/9 is 1ul ligation and 200 ul competent cells.  This time is five times volume of ligation product than the first time transformation.  It’s the same ligation from 8/9.
</LI></UL><UL><LI> Plate Pictures:
</LI></UL><UL><LI> I dip some colonies into a growth plate.  100ul LB/Carb in sterile 96 well plates.  Overnight at 37 degree.
</LI></UL><PRE> * Plate Schematic:
</PRE><UL><LI> I picked 24 colonies from VioB PCR 2nd time, 24 colonies from VioB Rxn1 2nd time, 12 colonies from VioB Rxn 2 2nd time.  
</LI></UL><UL><LI> I did overhang PCR of ACD, because primers for E didn’t arrive with them.  I diluted primers which are in Patrick’s primer box.  There’s a concentration number on the commercial tube, for example, 25.1 nM, multiply by 20, 502ul is the volume of the water I added to the commercial tube.  Then I made another stock to eppendorf tubes which concentration I can’t remember.  Ask Patrick for them.  
</LI></UL><UL><LI> Checked the size of ACD
</LI></UL><P>A: 1295+500
C: 1328+500
D: 1160+500
</P><UL><LI> Used herculase and protocol:
</LI></UL><UL><LI> 50uL rxn mix:
</LI></UL><P>1uL Exp. Cassette DNA
1.25uL forward primer
1.25uL reverse primer
10uL 5x Herculase buffer
0.5uL Herculase
4uL dNTP
35.5uL H2O
</P><UL><LI> Rxn:
</LI></UL><P>94 degrees-2min
</P><P>30X Cycle
94 degrees-20sec
55 degrees-20sec
72 degrees-20sec
</P><P>72 degrees-3min
4 degrees-Infinite
</P><UL><LI> After PCR:
</LI></UL><UL><LI> We can tell the only white colony worked, it’s 3kb. (Rxn2)
</LI></UL><H5><SPAN class="mw-headline" id="August_13th.2C_2011">August 13th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> Gel purified VioA C D
</LI></UL><P>Weight:
A:0.35g
C:0.37g
D:0.42g
</P><H5><SPAN class="mw-headline" id="August_15th.2C_2011">August 15th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> I wanted to verify the VioB I got so that I can be more sure.  Digested internal restriction sites in the VioB ORF Sequence.
</LI></UL><UL><LI> Double Digest Reaction Mix:
</LI></UL><P>DNA 2ul
Enzyme 0.25+0.25 ul
Buffer 2 ul
10xBSA 2 ul
H2O 13.50 ul
</P><UL><LI> Only one exception, when I digested with BspHI and XbaI:
</LI></UL><P>DNA 5ul
BspHI 0.5ul
XbaI 0.5ul
NEB Buffer 4 2ul
BSA 2ul
Sterile Water 10ul
</P><UL><LI> I did this first then I changed to the condition above for other enzymes.
</LI></UL><UL><LI> BspHI XbaI  Digest Gel:
</LI></UL><UL><LI> The other gel:
</LI></UL><PRE>* order:
</PRE><P>ladder XmaI BspHI XmaI XbaI XbaI SphI BspHI SphI
</P><P>VioB internal site digestion Gel:
</P><UL><LI> They are not perfect match, but Patrick said they’re fine.
</LI></UL><UL><LI> Then I dilute primers for VioE to do overhang PCR.
</LI></UL><P>24.3x20=486
22.1x20=442
then get 10 ul our of each of them, add 40 ul water to make 10uM stock.
</P><UL><LI> Overhang PCR: 
</LI></UL><P>Dilute Template PCR 1:20 (1ul VioE+19ul H2O)
cocktail: 
</P><P>5x Herculase Buffer 10ul
Herculase Enzyme 0.5ul
dNTPs 4ul
H2O 35.5ul
</P><P>Diluted DNA 1ul
primer1 1.25ul
Primer2 1.25ul
cocktail 47 ul
total 50ul
</P><UL><LI> PCR condition: 
</LI></UL><P>94 degrees-2min
</P><P>30X Cycle
94 degrees-20sec
55 degrees-20sec
72 degrees-20sec
</P><P>72 degrees-3min
4 degrees-Infinite
</P><UL><LI> Then ran the gel, pretty ladder.  VioE may be contaminated so didn’t show anything:
</LI></UL><UL><LI> I shake VioB PCR and ligation-transformation product from growth plate to this big tube I left you.  Because Dr. Boeke advised that I not do csPCR with them.  They were shaking in 37 degree overnight.
</LI></UL><H5><SPAN class="mw-headline" id="August_16th.2C_2011">August 16th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> Tubes are ready for Miniprep.  But I only did 6 for miniprep, they’re from VioB Rxn2 2nd time plate which contains B1 B2 B3 &amp; B4.
</LI></UL><UL><LI> And there’s a correction for the previous day's entry, I said there are 12 colonies I dip from VioB Rxn2 2nd time plate, but actually there were only 6.
</LI></UL><UL><LI> I dip some colonies into a growth plate.  100ul LB/Carb in sterile 96 well plates.  Overnight at 37degree.I dip some colonies into a growth plate.  100ul LB/Carb in sterile 96 well plates.  Overnight at 37degree.
</LI></UL><UL><LI> 96 Well Schematic:
</LI></UL><UL><LI> I picked 24 colonies from VioB PCR 2nd time, 24 colonies from VioB Rxn1 2nd time, 6 colonies from VioB Rxn 2 2nd time.  
</LI></UL><UL><LI> I ran a gel for VioA C and D which already purified:
</LI></UL><PRE> *  Order:ladder A C D, 3 ul DNA+3 ul loading dye
</PRE><H5><SPAN class="mw-headline" id="August_17th.2C_2011">August 17th, 2011</SPAN></H5><P>Experiments Performed by Xin Yuan (see <A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements" class="external text" rel="nofollow"><I>Acknowledgments </I></A>).  These entries are transcribed from her lab notebook.
</P><UL><LI> I nanodropped all the miniprep product, the concentrations are on the tubes.  And ran a gel of them:
</LI></UL><UL><LI> Forgot to digest before running the gel and order is ladder VioB no PCR 1 2 3 4 5 6 Qing’s stuff ladder.  Lane 3 is wired.
</LI></UL><H5><SPAN class="mw-headline" id="August_22nd.2C_2011">August 22nd, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Draw up offcut ligation for RFC assembly standard
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Digest 30uL E180 16-1 (7/2/2011) and run it by two purified ORFs
</LI><LI> Run 20-well gel (thin comb, B2-20, 200mL 1% agarose with EB). Load 50ug
</LI></UL><P><B>Results</B>:
</P><UL><LI> Purify E, use rest of digest product
</LI></UL><H5><SPAN class="mw-headline" id="August_23rd.2C_2011">August 23rd, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> End Goal: 58ng scale ligation of VioE to its TDM3p and RPL8a utr
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Gel purify what's left of digest (took 10uL out of 40uL solution)
</LI><LI> Run 7x8cm gel (100uL, 1% agarose, and 5uL EB)
</LI><LI> Digest 10uL of each promoter/utr
</LI></UL><P><B>Results</B>:
</P><UL><LI> Prep Gel Pic: 
</LI></UL><H5><SPAN class="mw-headline" id="August_24th.2C_2011">August 24th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Overhang PCR
</LI></UL><H5><SPAN class="mw-headline" id="August_25th.2C_2011">August 25th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Analytical gel of VioE cassette PCR product
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel Pic: 
</LI></UL><H5><SPAN class="mw-headline" id="August_26th.2C_2011">August 26th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> End Goal: Ligate VioB ORF to RPS2p &amp; utr (5ug for each part)
</LI><LI> Gel purify ORF after BsaI digest
</LI><LI> Digest p &amp; utr
</LI></UL><P><B>Results</B>:
</P><UL><LI> Used wrong tube
</LI><LI> Gel Pic: 
</LI></UL><H5><SPAN class="mw-headline" id="August_29th.2C_2011">August 29th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Digest VioB DNA with BsaI and BsaI/XbaI
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Run 7x8 1% agarose gel for 35min at 150V
</LI></UL><P><B>Results</B>:
</P><UL><LI> Pre-Gel Extraction: 
</LI></UL><UL><LI> Post-Gel Extraction: 
</LI></UL><H5><SPAN class="mw-headline" id="August_30th.2C_2011">August 30th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Overhang PCR of VioB expression cassette
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Recommended extension time is 2min at 72 degree Celsius
</LI><LI> Run 6-well 50mL 7x8 1% agarose gel at 150V (add 2.5uL EB)
</LI><LI> Lanes: Ladder, 0.5 1 2, Ladder
</LI></UL><P><B>Results</B>:
</P><UL><LI> Bands turned out correct
</LI><LI> Gel Pic: 
</LI></UL><H5><SPAN class="mw-headline" id="September_1st.2C_2011">September 1st, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> End Goal: Design, Digest, and Ligation of all 5 Vio genes
</LI><LI> Nanodrop all products
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Ligated according to Aug 23rd protocol. Left at room temp.
</LI></UL><P><B>Results</B>:
</P><UL><LI> Lost VioE, re-PCR and gel.
</LI></UL><H5><SPAN class="mw-headline" id="September_2nd.2C_2011">September 2nd, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> PCR cleanup (Invitrogen)
</LI><LI> Design ligation, find PUC19 to digest and do it
</LI><LI> Get reagents for yeast transformation
</LI><LI> Grow culture (inoculate 10mL YPD with 2uL cells for tomorrow, measure OD and dilute based on murkiness on Sunday)
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Run 7x8 50mL 1% agarose gel + 2.5uL EB at 150V for 35min
</LI><LI> Dilute YPD culture, multiply dilution factors by the absorbance reading
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel Pic: 
</LI></UL><H5><SPAN class="mw-headline" id="September_6th.2C_2011">September 6th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Final Plan: Select Auxotroph specific vector, get sequence
</LI><LI> Re-PCR expression cassettes and digest internal sites
</LI></UL><H5><SPAN class="mw-headline" id="September_7th.2C_2011">September 7th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Get PRS416 DNA, digest with EcoRI and HindIII
</LI><LI> Find smallest possible bar and comb
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Run 150mL 1% agarose gel with 7.5uL EB at 160V
</LI><LI> Lanes: 300ug 200ug, Ladder, 100ug 50ug 25ug 10ug 5ug 1ug
</LI><LI> Lanes: 300 200 100(1) 100(10) 50 25 10 5 1(1) 1(10)
</LI><LI> Lanes: 10 50 300 200 100
</LI></UL><P><B>Results</B></P><UL><LI> Water makes ladder float, not sink
</LI></UL><H5><SPAN class="mw-headline" id="September_8th.2C_2011">September 8th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> PCR cleanup products (amplify them)
</LI><LI> Gel purify PRS416
</LI><LI> PCR Purify again
</LI><LI> Gel purify
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Run 7x8cm gel(50mL, 1% agarose, 2.5uL EB)
</LI><LI> Lanes: Ladder, A B C D E
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel Prep Pic:
</LI></UL><UL><LI> Analytical Gel Pic: 
</LI></UL><UL><LI> A, B, C, D good, E is too small (don't purify)
</LI><LI> Redo E, likely primers were wrong
</LI><LI> Use 1uL PRS as positive control in transformation
</LI><LI> Redo PCR (cleanup took out final product)
</LI></UL><H5><SPAN class="mw-headline" id="September_9th.2C_2011">September 9th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Transform ligation misses and no DNA or vector control
</LI><LI> Redo PCR for VioE
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Add 5uL positive and negative ligation and 1uL PRS416
</LI><LI> 2min on ice
</LI><LI> Heat shock for 45s
</LI><LI> Ice for 2min
</LI><LI> Added 450mL SOC to each
</LI><LI> 1hr on shaking incubator
</LI><LI> 1 plate (50uL), next plate (centrifuge 7000rpm for 25s)
</LI><LI> Take 350mL supernatent out, plate rest after resuspension with pipet
</LI></UL><P><B>Results</B>:
</P><UL><LI> Can't tell which tube was PRS416, and no DNA.
</LI></UL><H5><SPAN class="mw-headline" id="September_10th.2C_2011">September 10th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Miniprep colonies and digest them
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Run 7x8 50mL 1% agarose gel with 2.5uL EB
</LI><LI> Lanes: 5uL Ladder, 10uL each of E(new) and E(old)
</LI><LI> Redid VioE PCR from ligation mix
</LI><LI> PCR cleanup with Qiagen
</LI><LI> Gel purify of undigested PCR clean product
</LI><LI> Run 12x14 150mL 1% agarose gel with 7.5 EB
</LI><LI> Lanes: A, Ladder, B X, Ladder, C X D, Ladder, E X X
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel Pics: 
</LI></UL><UL><LI> Amplified the wrong fragment?
</LI></UL><H5><SPAN class="mw-headline" id="September_12th.2C_2011">September 12th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Colony PCR design
</LI></UL><H5><SPAN class="mw-headline" id="September_13th.2C_2011">September 13th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Colony PCR
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel Pics: 
</LI></UL><H5><SPAN class="mw-headline" id="September_15th.2C_2011">September 15th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Religate in case yeast transformation didn't work
</LI><LI> Colony PCR
</LI></UL><H5><SPAN class="mw-headline" id="September_16th.2C_2011">September 16th, 2011</SPAN></H5><P><B>Purpose</B>:
</P><UL><LI> Colony PCR gel (each 96-well plate section of 3 rows occupies one gel row)
</LI><LI> Redigest PRS416 and test plasmid with EcoRI and HindIII
</LI></UL><P><B>Procedure</B>:
</P><UL><LI> Run 7x8cm 50mL 1% agarose with 2.5uL EB gel
</LI><LI> Lanes: PRS X Ladder, PAR...6wells
</LI></UL><P><B>Results</B>:
</P><UL><LI> Gel Pics: 
</LI></UL><UL><LI> Did it work?
</LI><LI> 4th colony section the best
</LI></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Johns_Hopkins/Notebook/Vio">http://2011.igem.org/Team:Johns_Hopkins/Notebook/Vio</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Notebook/Vio" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Notebook/Vio" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Vio&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Vio&amp;oldid=256864" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>