<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Notebook_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Notebook/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Notebook/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Notebook/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Notebook/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>VitaYeast - Johns Hopkins University, iGEM 2011</TITLE></P><DIV id="mydroplinemenu" class="droplinebar"><UL><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Members">Members</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Johns_Hopkins/Team/Gallery">Gallery</A></LI></UL><LI><A href="#">Vitamins</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitA">Vitamin A</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitC">Vitamin C</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant">Measurements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Baking">Applications</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Future">Future Plans</A></LI><LI><A href="#">Yeast Toolkit</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR">Promoters and UTRs</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Violacein">Violacein</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Vector">Yeast Vector Library</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Future">Future Plans</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/HumanPrac">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/IRBProcess">IRB Approval</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Data">Data</A></LI><LI><A href="#">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Platforms">Platform</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/LBSMod">LBS Models</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Opt">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Sensitivity">Sensitivity</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/ParaFit">Parameter Fitting</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments">Vitamin Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YTExperiments">Yeast Toolkit Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></DIV><DIV id="secondarycontentgreen"><DIV id="boxheading">
			Related Links:</DIV><DIV id="boxcontent"><DL><DIV class="heading">Other Protocols:</DIV><DL><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/DNAAssayProtocol">DNA Protocols Page 2</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/VitProtocol">Vitamin Assays</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/BBProtocol">BioBrick</A></DD></DL><CENTER><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#DNA_Synthesis_and_Assay_Protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">DNA Synthesis and Assay Protocols</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Gel_Extraction_.28QIAquick.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Gel Extraction (QIAquick)</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Mini-Prep_.28QIA.29"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Mini-Prep (QIA)</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#PCR_Reactions_on_Small_DNA_Pieces"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">PCR Reactions on Small DNA Pieces</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#DNA_Digestion"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">DNA Digestion</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#PCR_1.0.25_Agarose_Gel"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">PCR 1.0% Agarose Gel</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#QuickChange_PCR"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">QuickChange PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Synthesis"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Synthesis</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Overlap_Extension"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Overlap Extension</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#CPEC"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">CPEC</SPAN></A></LI></UL></TD></TR></TBODY></TABLE></CENTER><DIV id="primarycontent"><H6><SPAN class="mw-headline" id="DNA_Synthesis_and_Assay_Protocols">DNA Synthesis and Assay Protocols</SPAN></H6><H5><SPAN class="mw-headline" id="Gel_Extraction_.28QIAquick.29">Gel Extraction (QIAquick)</SPAN></H5><OL><LI> Excise DNA fragment from agarose gel with clean, sharp scalpel.
</LI><LI> Weigh gel slice in colorless tube. Add 3 volumes of Buffer QG to 1 volume of gel (100mg ~ 100µl)
</LI><LI> Incubate @ 50C for 10 min.
</LI><LI> SOLUBILIZE AGAROSE COMPLETELY.
</LI><LI> To help dissolve gel, vortex tube every 2-3 min during incubation.
</LI><LI>  After the gel has dissolved, check that the color of the mixture is yellow. If not, add 10µl 3M NaOAc. Or just add the NaOAc anyway.
</LI><LI> Add 1 gel volume of isopropanol to the sample and mix
</LI><LI> Do only if &lt;500bp or &gt;4kb
</LI><LI> Place spin column in 2mL collection tube.
</LI><LI> To bind DNA, apply sample to column. Centrifuge 1 min.
</LI><LI> Discard flowthrough and place column back in same collection tube.
</LI><LI> Recommended: Add 0.5mL of Buffer QG to column and centrifuge for 1 min.
</LI><LI> To wash, add 0.75mL of Buffer PE to column and centrifuge for 1 min.
</LI><LI> Discard flowthrough and centrifuge for an additional 1 min @ 17900 x g (13000rpm).
</LI><LI> Place column into a clean 1.5mL microcentrifuge tube.
</LI><LI> To elute DNA, add 30µL Buffer EB to center of membrane, let column stand for 1 min, then centrifuge for 1 min.
</LI></OL><H5><SPAN class="mw-headline" id="Mini-Prep_.28QIA.29">Mini-Prep (QIA)</SPAN></H5><OL><LI> Resuspend pelleted bacterial cells in 250 µL Buffer P1 and transfer to a microcentrifuge tube.
</LI><LI> Add 250 µL Buffer P2 and mix thoroughly by inverting the tube 4-6 times.
</LI><LI> Add 350 µL Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times. 
</LI><LI> Centrifuge for 10 min at 13,000 rpm in a table-top microcentrifuge.
</LI><LI> Apply the supernatants from step 4 to the QIAprep spin column by decanting or pipetting.
</LI><LI> Centrifuge for 30-60s. Discard the flow-through.
</LI><LI> Recommended: Wash the QIAprep spin column by adding 0.5 mL Buffer PB and centrifuging for 30-60s. Discard flow-through.
</LI><LI> Wash QIAprep spin column by adding 0.5 mL Buffer PB and centrifuging 30-60s.
</LI><LI> Discard flow-through and centrifuge for an additional minute to remove residual wash bufffer. **IMPORTANT: Residual wash buffer will not be completly removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE may inhibit subsequent enzyme reactions.**
</LI><LI> Ordered List ItemPlace QIAprep column in a clean 1.5 mL microcentrifuge tube. To elute DNA, add 50 µL Buffer EB (10mM Tris-CL, pH 8.5) or water to the center of each QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.
</LI></OL><H5><SPAN class="mw-headline" id="PCR_Reactions_on_Small_DNA_Pieces"> PCR Reactions on Small DNA Pieces </SPAN></H5><P>Procedure:
</P><OL><LI> Combine the following reagents in one tube:
</LI><LI> 18uL PCR Platinum Supermix
</LI><LI> .24uL Forward Primer
</LI><LI> .24uL Reverse Primer
</LI><LI> .4uL Target DNA
</LI><LI> 1.12uL Nuclease Free Water
</LI><LI> Place tube into Thermal Cycler
</LI><LI> Program Thermal Cycler with desired Denaturing, Annealing and Extension Temperatures.
</LI><LI> Example: 
</LI></OL><PRE>    94C for 2 minutes
    94C for 30 seconds (Denaturing)
    55C for 30 seconds (Annealing)
    72C for 1min/kb (Extension)
    Repeat Denaturing, Annealing and Extension at least 24 times.
    Hold at 4C
</PRE><H5><SPAN class="mw-headline" id="DNA_Digestion">DNA Digestion</SPAN></H5><P>DNA Digestion Mixture:
</P><UL><LI> 25 μl DNA
</LI><LI> 3 μl Buffer 2
</LI><LI> 1 μl EcoRI-HF
</LI><LI> 1 μl PST1
</LI><LI> 3 μl BSA1
</LI></UL><H5><SPAN class="mw-headline" id="PCR_1.0.25_Agarose_Gel">PCR 1.0% Agarose Gel</SPAN></H5><OL><LI> Add .3 grams Agarose to 30mL TAE Buffer in volumetric flask.
</LI><LI> Stopper volumetric flask with Kim Wipes.
</LI><LI> Microwave mixture for 1 minute to allow Agarose to dissolve.
</LI><LI> Prepare BioRad Gel Machine:
</LI><LI> Place gel stand.
</LI><LI> Add both gel blocks.
</LI><LI> Insert comb.
</LI><LI> Allow mixture to cool down to ~40C.
</LI><LI> Use micropipetter to seal edges of gel stand with the mixture.
</LI><LI> Add 3uL Ethidium Bromide to remaining mixture.
</LI><LI> Mix well.
</LI><LI> Pour mixture onto gel stand. 
</LI><LI> Cover with Paper Towels and allow to harden.
</LI><LI> Prepare PCR Reaction tubes:
</LI><LI> For every 10uL PCR product add 1uL Blue Juice.
</LI><LI> Mix well.
</LI><LI> Once gel hardens remove gel blocks and comb.
</LI><LI> Caution: Remove comb vertically so as not to damage wells.
</LI><LI> Fill gel machine with TAE buffer till liquid reaches above gel.
</LI><LI> Load 5uL DNA Ladder.
</LI><LI> Load 5-10uL samples.
</LI><LI> Run at 100 Volts for 20-25 minutes.
</LI><LI> Turn machine off and transfer gel to imaging machine.
</LI><LI> Position gel and take picture under UV.
</LI></OL><H5><SPAN class="mw-headline" id="QuickChange_PCR">QuickChange PCR</SPAN></H5><P>From Marty Taylor of the Boeke Lab: 
</P><P>Based on two stage protocol from - W. Wang, B. A. Malcolm, Biotechniques 26, 680 (Apr, 1999).
This two-stage protocol allows formation of some of a “hybrid” between WT and mutant, reducing competition 
between primer and its GC
</P><P>Design primer to span region to be changed. Depending on the degree of change (single point mutation vs large deletion/insertion) I usually have 15-40nt on either side with perfect match. Order primer and its reverse complement. We have had good luck using Ultramers from IDT up to ~150nt long, haven’t tried anything longer. We do not PAGE purify primers for this application.
</P><P>For each primer, setup a 25uL reaction (for older PCR machines with 0.5mL tubes, double everything and you may want to extend the annealing/extension times). I usually do a number of these at a time from the same template, so I make a master mix containing everything but primers, aliquot, and then add primers. 
</P><OL><LI> Template dNA (~25ng) [1 uL from miniprep]
</LI><LI> 12.5pMol primer [1.25uL] of 10uM stock – be sure to use only one primer each!
</LI><LI> 5x Buffer [5 uL]
</LI><LI> dNTPs (2.5mM each) [2 uL]
</LI><LI> Herculase II [0.5 uL] 
</LI><LI> H2O → 25uL
</LI></OL><P><I>Control reaction</I> – this is your background! Add 50uL master mix (without primers) to PCR tube and put it on the cycler, identical to the other tubes. DpnI digest transform like the others. 
</P><P><I>PCR Protocol</I>:
</P><OL><LI> Stage 1: setup 2 tubes as above – one for primer, one for its complement – run 2 or 3 cycles
</LI><LI> Stage 2: combine the two reactions into one, mix well, run for 18 more cycles 
</LI><LI> 95C, 5min 
</LI><LI> [95C, 30sec / 55C, 30sec / 72C (1min/kB total vector+insert)] – 2-3x in stage 1, 18x in stage 2
</LI><LI> 72C, 10min
</LI><LI> Extension time – with the new faster enzymes, 30s/kB may be enough, but have had better luck with 1min
</LI></OL><P><I>DpnI digest</I> – add 0.5uL DpnI (10U/uL) to each 50uL reaction, vortex, incubate at 37°C for 1-3hrs (longer is better if time permits). Be sure to DpnI digest the control and transform and plate it equally. If you think the reaction is a hard one, you can extend the Dpn digestion to reduce background further (or add more enzyme).
</P><P><I>Agarose Gel</I> – WASTE OF TIME! This technique often fails to produce enough DNA to see on a gel and still works. So, basically, this step tells you nothing, since you are going to do the transformation whether or not the gel shows you something. I would do a positive control on the transformation long before I run a gel here.
</P><P><I>PCR Cleanup</I> – Herculase II reaction mix is incompatible with our standard competent cells (made using Mn/Ca), reducing the efficiency of transformation by at least 3 logs (perhaps due to detergent in the buffer). A quick PCR cleanup solves this problem.
</P><P>Transform 5uL into a 50uL aliquot of competent cells. We make our own competent cells; for very difficult reactions, we have had success with supercompetent cells, but I haven’t used these since switching to PFU-Ultra. After heat shock, I grow the cells for 1hr in 500uL SOC and plate both 50uL and 450uL. Sometimes the 450uL plate is a lawn; sometimes there are only 10-50 colonies.
</P><P>Screening – if your control plate is clean (no colonies), I recommend sequencing (or otherwise screening – if your design allows) 3 clones. Usually at least 2 are positive, but sometimes only 1.  Sometimes odd things happen, such as single nucleotide insertions or deletions, errors in the primer sequence itself, or part of the primer annealing with the wrong region of the template, but these results are rarely in all clones sequenced. If there are a lot of colonies on the control (25-50% of the experimental plates), you can sometimes get lucky. Consider longer DpnI digest or repeating. 
</P><H5><SPAN class="mw-headline" id="Synthesis">Synthesis</SPAN></H5><H4><SPAN class="mw-headline" id="Overlap_Extension">Overlap Extension</SPAN></H4><P>In order to assemble our Vitamin C genes, we used overlap extension PCR. Oligos of up to 60 bp were ordered from IDT, sequentially, in building blocks of up to 800 bp. The first oligo had a 40 base pair overlap with the next, and so on, until the end of that particular chunk of the gene, called a building block. GDP L-Galactose Phosphatase and GDP Mannose-3,5-Epimerase both are made up of two building blocks, and L-Galactose 1-phosphate phosphatase is composed of only one building block. This process is known as templateless PCR. Following this, the PCR product (which will include both incomplete building blocks and a small amount of final product) is PCR amplified using the first and last oligos. This step is called finishing PCR. These building blocks are then purified using a zymogen DNA purification column and then assembled into the final construct in the vector via a CPEC reaction.
</P><H4><SPAN class="mw-headline" id="CPEC">CPEC</SPAN></H4><P>(Quan 2009)
</P><OL><LI> Measure the DNA concentration of each assembly piece
</LI><LI> Assay 100ng of the linearized vector backbone and equimolar amounts of the other assembly pieces to a 25ul total volume assembly reaction mi#xture accordingly:
</LI><LI>100 ng of vector backbone
</LI><LI> equimolar ammounts of each assembly piece
</LI><LI> 5ul 5X HF Phusion Buffer
</LI><LI> 1ul 10mM dNTPs
</LI><LI> 0.75ul DMSO
</LI><LI> 0.5ul 2U/ul Phusion Polymerase
</LI><LI> H2O to 25ul
</LI><LI> Perform the assembly reaction in a thermocycler as follows:
</LI></OL><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:40px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> Temperature
</TD><TD> Time
</TD><TD> Cycles
</TD></TR><TR><TD> 98C
</TD><TD> 3 min
</TD><TD> 1
</TD></TR><TR><TD> 98C
</TD><TD> 30 sec
</TD><TD> 15 *
</TD></TR><TR><TD> 55C
</TD><TD> 30 sec
</TD><TD> 15 *
</TD></TR><TR><TD> 72C
</TD><TD> total length(kb) * 15 sec
</TD><TD> 15 *
</TD></TR><TR><TD> 72C
</TD><TD> 10 min
</TD><TD> 1
</TD></TR></TBODY></TABLE><UL><LI>Note: the number of repeated cycles should exceed the number of assembly pieces
</LI></UL><DL><DD> 4. Transform 5ul of the assembly reaction into 100ul of competent E. coli and/or run a diagnostic agarose gel to check for successful assembly.
</DD></DL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols">http://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols</A>&quot;</DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Notebook/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Notebook/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/Protocols&amp;oldid=196303" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>