<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Project_MeasureQuant"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Project/MeasureQuant">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Project/MeasureQuant&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/MeasureQuant&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/MeasureQuant&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Project/MeasureQuant" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Project/MeasureQuant</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>VitaYeast - Johns Hopkins University, iGEM 2011</TITLE></P><DIV id="mydroplinemenu" class="droplinebar"><UL><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Members">Members</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Johns_Hopkins/Team/Gallery">Gallery</A></LI></UL><LI><A href="#">Vitamins</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitA">Vitamin A</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitC">Vitamin C</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant">Measurements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Baking">Applications</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Future">Future Plans</A></LI><LI><A href="#">Yeast Toolkit</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR">Promoters and UTRs</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Violacein">Violacein</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Vector">Yeast Vector Library</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Future">Future Plans</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/HumanPrac">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/IRBProcess">IRB Approval</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Data">Data</A></LI><LI><A href="#">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Platforms">Platform</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/LBSMod">LBS Models</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Opt">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Sensitivity">Sensitivity</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/ParaFit">Parameter Fitting</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments">Vitamin Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YTExperiments">Yeast Toolkit Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></DIV><DIV id="secondarycontentgreen"><DIV id="boxheading">
			Related Links:</DIV><DIV id="boxcontent"><DL><DIV class="heading">Vitamins:</DIV><DL><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Bg">Background</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Over">Overview</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitA">Vitamin A</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitC">Vitamin C</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Baking">Applications</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Results">Results</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Future">Future Plans</A></DD></DL><DIV id="primarycontent"><H6><SPAN class="mw-headline" id="Measurements_and_Quantification">Measurements and Quantification</SPAN></H6><P>In order to assess the efficiency and effectiveness of our yeast strains, we performed a series of quantitative measurements which would allow us to determine protein expression levels, amount of products and bi-products synthesized, and kinetic rates of the enzymes in our system.  The results of these experiments also provide important initial conditions for the modeling aspect of our project, as it provides us with reliable and realistic <I>in vivo</I> gene expression and catalytic activity of the enzymes specific to our system. The following experiments will allow us to quantitatively determine these desired expression/synthesis in our system.
</P><H5><SPAN class="mw-headline" id="Extraction_of_.CE.B2-carotene_from_Yeast">Extraction of β-carotene from Yeast</SPAN></H5><P>β-carotene was extracted from yeast for quantification.  After the addition of pyrogallol in methanol, cells were saponified with potassium hydroxide.  The samples were incubated at 75 degrees, and β-carotene was then extracted with hexane.  Samples were dried in a vacuum desiccator for future analysis.
</P><H5><SPAN class="mw-headline" id="Spectroscopy_and_High_Pressure_Liquid_Chromatography_.28HPLC.29_for_Measuring_Product_Synthesis">Spectroscopy and High Pressure Liquid Chromatography (HPLC) for Measuring Product Synthesis</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Shimadzu HPLC system</DIV></DIV></DIV><P>HPLC was performed in order to characterize the products synthesized and measure the amount of β-carotene and produced by our yeast strains. This strategy was chosen for its high degree of accuracy and potential to seperate products extracted from cell lysate. Based on a standard curve, <I>in vivo</I> measurements of β-carotene produced per yeast cell was determined. HPLC analysis was performed on a C18 Varian 5 micron 5x250mm column to verify the presence of β-carotene.  The UV detector was set at 450nm, and the retention time of beta carotene is around 16.6 minutes.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">UV Spectrophotometer</DIV></DIV></DIV><P>Spectroscopy was utilized in conjunction with HPLC as an alternative, and offered a very practical method for product measurement, as carotenoids are colored pigments. Absorbance of β-carotene was measured with a UV spectrophotometer at 449nm.
</P><H5><SPAN class="mw-headline" id="Protein_Purification_for_Enzyme_Assay">Protein Purification for Enzyme Assay</SPAN></H5><P>In order to perform enzyme kinetics measurements for the enzymes in the β-carotene pathway, the enzymes had to be purified first.  The enzymes being purified were geranylgeranyl pyrophosphate synthase, phytoene desaturase, and  phytoene synthase and lycopene cyclase produced from the genes crtE, crtI, and crtYB, respectively.  By isolating our enzymes of interest, we would greatly increase the accuracy of the kinetics experiments by eliminating overlapping background interference of absorption spectra, removing any competitive inhibitors of our enzymes, and ensuring a controlled measurement of our enzymatic rates were being measured. A procedure was designed which would allow for high amounts of product purified from extract, as well as high purity of our purification product. Using a TEV-his6X-mCherry-KAN tag, we would be able to purify our enzyme with only one high efficiency purification step (Nickel Column Purification using His-tag) and characterize product with great accuracy through the use of an mCherry (RFP) component of the tag. The following procedure is an outline of sequential experiments necessary to isolate and confirm the isolation of our enzymes.  
</P><P>To purify the expressed protein of interest, the steps are as follows:
</P><OL><LI> Growth of yeast strains.
</LI><LI> PCR gene of interest-TEV-his6X-linker-mCherry-KAN.  Tag is on C-terminus of the protein.
</LI><LI> Transform PCR products into yeast strains.
</LI><LI> Microscopy to show red fluorescence.
</LI><LI> Genomic prep from transformed cells and testing by PCR for integration of the tag at the c-terminus of the genes.
</LI><LI> Make a cell lysate and run through a nickel column for purification.
</LI><LI> Perform a Western Blot with DSred antibody or run a coommassie gel to assess purification (both size and quantity).
</LI></OL><H5><SPAN class="mw-headline" id="Enzyme_Assay">Enzyme Assay</SPAN></H5><P>The set of enzyme kinetics experiments that we plan to perform for both β-carotene and L-ascorbic Acid will allow us to calculate the catalytic rates and efficiency of the enzymes in our pathways, and will allow us to optimize the system as we see fit based on environmental conditions, substrate levels, and stress created by over-stressing our pathway.  Assuming Michaelis Menten kinetics for our enzymes, we will be able to calculate kinetic values such as Km, kcat, and Vmax of the enzymes.  In preparation for the measurements, cellular conditions will be modeled as close as possible for this <I>in vitro</I> experiment.  An <I>in vivo</I> enzyme kinetics experiment was hypothesized, but due to the complexity of our pathway, it was decided that the interference would be too great to determine our enzymes signal. Details of the protocols are still being evaluated. The use of a fluorescent co-factor, NADPH, which is necessary for reactions catalyzed by phytoene desaturase, will prove useful as a fluorescent indicator of a kinetic rate. However many of the substrates and products of the beta-carotene reactions have similar absorption spectra, or do not require a fluorescent co-factor, therefore spectroscopy alone would prove to be inaccurate. As an alternative, one option would be to quench the reaction at regular time intervals and isolate the product with HPLC, thus measuring product concentration over time, which is directly related to the reaction rate.     
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant">http://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Project/MeasureQuant" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Project/MeasureQuant" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/MeasureQuant&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/MeasureQuant&amp;oldid=256028" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>