<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Project_Vector"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Project/Vector">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Project/Vector&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/Vector&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/Vector&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Project/Vector" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Project/Vector</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>VitaYeast - Johns Hopkins University, iGEM 2011</TITLE></P><DIV id="mydroplinemenu" class="droplinebar"><UL><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Members">Members</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Johns_Hopkins/Team/Gallery">Gallery</A></LI></UL><LI><A href="#">Vitamins</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitA">Vitamin A</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitC">Vitamin C</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant">Measurements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Baking">Applications</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Future">Future Plans</A></LI><LI><A href="#">Yeast Toolkit</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR">Promoters and UTRs</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Violacein">Violacein</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Vector">Yeast Vector Library</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Future">Future Plans</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/HumanPrac">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/IRBProcess">IRB Approval</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Data">Data</A></LI><LI><A href="#">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Platforms">Platform</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/LBSMod">LBS Models</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Opt">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Sensitivity">Sensitivity</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/ParaFit">Parameter Fitting</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments">Vitamin Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YTExperiments">Yeast Toolkit Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></DIV><DIV id="secondarycontentpurple"><DIV id="boxheading">
			Related Links:</DIV><DIV id="boxcontent"><DL><DIV class="heading">Yeast Toolkit:</DIV><DL><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Bg">Background</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Over">Overview</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR">Promoters and UTRs</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Violacein">Violacein</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Results">Results</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Future">Future Plans</A></DD></DL><DIV id="primarycontent"><H6><SPAN class="mw-headline" id="Yeast_Vector_Library">Yeast Vector Library</SPAN></H6><H5><SPAN class="mw-headline" id="Overview">Overview</SPAN></H5><P>Due to <I>Saccharomyces cerevisiae'</I>s amenability to genetic manipulation as well other benefits associated with a eukaryotic host, there has recently been an increase in interest regarding yeast as a microbial workhorse. With this comes a need for synthetic biological tools to manipulate yeast quickly and easily. The goal of this sub-project is to create a library of vectors that both efficiently manipulates DNA in <I>S. cerevisiae</I> as well as conforms to the BioBrick Standard Assembly protocol. In designing these vectors, we hope to utilize the speed of propagation of <I>E. coli</I> along with the production capabilities of <I>S. cerevisiae</I> to create the first easy-to-use BioBrick-standard vector series.
</P><H5><SPAN class="mw-headline" id="pRS_Series_Vectors">pRS Series Vectors</SPAN></H5><P>The library will be a slight modification of the pRS series vectors created by Sikorski et al.<SUP><A href="#Foot1">[1]</A></SUP> and Christianson et al.<SUP><A href="#Foot2">[2]</A></SUP>. They are designed to be &quot;shuttle vectors&quot;, which can be easily propagated in Escherichia coli as well as transfected into the final chassis, <I>S. cerevisiae</I>. They are a hybrid of a yeast vector and the <I>E. coli</I> phagemid vector pBluescript SK+, which contains a lacZ gene with a polylinker as well as an ampicillin resistance (bla) gene. They contain an <I>E. coli</I> origin of replication (the pBluescript SK+ ori) and for the episomal plasmids, a yeast origin (either CEN/ARS or 2µ). Sikorski et al. and Christianson et al. designed three types of vectors: Integrating, CEN/ARS and 2μ (2micron) vectors:
</P><OL><LI><B>Integrating Plasmids</B>: A combination of the YIp (Yeast Integrating Plasmid) type vector with the multipurpose E. coli pBluescript II SK+ phagemid. Chromosomally integrates the plasmid.</LI><LI><B>CEN/ARS Plasmids</B>: Endogenous yeast plasmids based on the YCp (Yeast Centromere plasmid) type plasmid as well as the previously mentioned pBluescript II SK+. Medium copy number plasmid.</LI><LI><B>2µ Plasmids</B>: YEp (Yeast Episomal plasmid) type plasmid. On the backbone of pBluescript II SK+ phagemid. High copy number plasmid of about 20 per cell.
</LI></OL><P>Within each vector type, individual vectors differ by four selectable markers genes: HIS3 (essential for histidine biosynthesis), TRP1 (tryptophan), LEU2 (leucine), and URA3 (uracil). Each individual vector (12 total) requires a specific mutant strain of yeast with the respective biosynthesis gene knocked out. For <I>E. coli</I> colony selection, all the vectors contain LacZ genes for the white/blue ß-galactosidase assay.
</P><H5><SPAN class="mw-headline" id="pRSbb_Series_Vectors">pRSbb Series Vectors</SPAN></H5><P>The new BioBrick Standard vectors (pRSbb) contain the necessary restriction enzymes in the correct order in the place of the original pRS-series multiple cloning site. These vectors will be ready to accept any part via the BioBrick Standard Assembly protocol:
</P><P>BioBrick Standard Assembly. E = EcoRI, X = XbaI, S = SpeI, P = PstI
</P><H5><SPAN class="mw-headline" id="Final_Vectors">Final Vectors</SPAN></H5><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Example: Vector schematic for pRSbb# 416</DIV></DIV></DIV><TABLE border="1" cellpadding="5" cellspacing="0" style="margin-left:30px;"><TBODY><TR style="background-color:#EBEBE9;"><TD> pRSbb #
</TD><TD> Type
</TD><TD> Marker type
</TD></TR><TR><TD> 403
</TD><TD> Integrating
</TD><TD> HIS3
</TD></TR><TR><TD> 404
</TD><TD> Integrating
</TD><TD> TRP1
</TD></TR><TR><TD> 405
</TD><TD> Integrating
</TD><TD> LEU2
</TD></TR><TR><TD> 406
</TD><TD> Integrating
</TD><TD> URA3
</TD></TR><TR><TD> 413
</TD><TD> CEN/ARS
</TD><TD> HIS3
</TD></TR><TR><TD> 414
</TD><TD> CEN/ARS
</TD><TD> TRP1
</TD></TR><TR><TD> 415
</TD><TD> CEN/ARS
</TD><TD> LEU2
</TD></TR><TR><TD> 416
</TD><TD> CEN/ARS
</TD><TD> URA3
</TD></TR><TR><TD> 423
</TD><TD> 2µ
</TD><TD> HIS3
</TD></TR><TR><TD> 424
</TD><TD> 2µ
</TD><TD> TRP1
</TD></TR><TR><TD> 425
</TD><TD> 2µ
</TD><TD> LEU2
</TD></TR><TR><TD> 426
</TD><TD> 2µ
</TD><TD> URA3
</TD></TR></TBODY></TABLE><H6><SPAN class="mw-headline" id="Workflow">Workflow</SPAN></H6><H3><SPAN class="mw-headline" id="Removal_of_restriction_enzyme_sites_from_vector">Removal of restriction enzyme sites from vector</SPAN></H3><P>Any instances of restriction enzymes used in the BioBrick protocol must be removed from the vector.
</P><OL><LI> Find the occurrence of EcoRI, XbaI, PstI, and SpeI sites in all of the pRS400-series vectors.
</LI><LI> Create a spreadsheet with these values, compare them to Dr. Boeke’s numbers
</LI><LI> Map out all of the vectors including ORFs, RE sites and origins of replication and annotate their respective sequences in ApE files
</LI><LI> Comb through all of these annotated sequences and find one vector for each type of plasmid with the least number of changes necessary.
</LI><LI> Criteria for removal of RE sites (Quick Change):
</LI><LI> RE site is in ORF, ori, or other impt. Feature
</LI><LI> Set up QuikChange  protocol for the 3 initial vectors
</LI><LI> Notes:
</LI><LI> Look at translation map to determine which two (or three) codons the RE sites translates to
</LI><LI> Change the 3rd “wobble base” of a codon
</LI><LI> Make a conservative change to a codon of similar bias to the changed codon or most used codon.
</LI><LI> Finalize list of changes. To include:
</LI><LI> What to change
</LI><LI> Where change occurs
</LI><LI> Change to what
</LI><LI> How to change (primer info, etc.)
</LI><LI> Order primers
</LI><LI> Perform <A href="http://kirschner.med.harvard.edu/files/protocols/Stratagene_quickchangepdf.pd" class="external text" rel="nofollow">QuikChange® PCR</A></LI><LI> Record and store all data on a backup
</LI><LI> Gel verification
</LI><LI> Transform into E. coli
</LI><LI> Make minipreps; cut with enzyme of choice 
</LI></OL><H5><SPAN class="mw-headline" id="Conversion_from_existing_to_BioBrick_compatible_multiple_cloning_site">Conversion from existing to BioBrick compatible multiple cloning site</SPAN></H5><OL><LI> Determine correct prefix/suffix 
</LI><LI> Design primers to remove MCS and replace with BioBrick Standard
</LI><LI> Order primers
</LI><LI> PCR
</LI><LI> Gel verification
</LI><LI> Transform
</LI><LI> Miniprep and cut
</LI><LI> Send winners from above to sequencing
</LI><LI> Make Perms (freeze cells in 15% glycrol, stored at -80C
</LI><LI> Conclude, Clean up, Write report and Aggregate Data
</LI></OL><H6><SPAN class="mw-headline" id="Primer_Design">Primer Design</SPAN></H6><H5><SPAN class="mw-headline" id="Multiple_cloning_site_removal_and_insertion">Multiple cloning site removal and insertion</SPAN></H5><P>PCR primers were designed to flank the regions of the old pRS multiple cloning site and replace it with the BioBrick prefix and suffix. 
</P><P>The initial pRS 416 with primers flanking the multiple cloning site to remove the old MCS and insert the new BioBrick MCS.
</P><P>The final product after PCR.
</P><H5><SPAN class="mw-headline" id="Site_directed_mutagenesis">Site directed mutagenesis</SPAN></H5><P>In order to ensure the vector will be cut in only the correct places by specific restriction enzymes, instances of certain restriction enzyme sequences must be removed from the vector. The restriction enzymes required for BioBrick Standard Assembly are:
</P><OL><LI><B>EcoRI</B>: GAATTC
</LI><LI><B>XbaI</B>: TCTAGA
</LI><LI><B>SpeI</B>: ACTAGT
</LI><LI><B>PstI</B>: CTGCAG
</LI></OL><P>If a restriction enzyme sequence that needed to be changed was in an open reading frame, the third (&quot;wobble&quot;) base of an in-frame codon within the sequence was changed to ensure a silent mutation. This is done using site directed mutagenesis. This involves a primer with the desired mutation in the middle, with regions of homology on either side:
</P><P>Above is an example of a primer to remove a PstI site from pRS416. Purple denotes regions of homology, red is the desired base change, and yellow is the original restriction enzyme site. Two primers are used, one being the reverse complement of the other, with the <A href="http://kirschner.med.harvard.edu/files/protocols/Stratagene_quickchangepdf.pdf" class="external text" rel="nofollow">QuikChange® Site-Directed Mutagenesis</A> protocol in order to substitute the desired base in.
</P><H5><SPAN class="mw-headline" id="References">References</SPAN></H5><P><SPAN id="Foot1"><SUP>[1]</SUP>Sikorski, R.S.; Hieter, P. (1989) <A href="http://www.ncbi.nlm.nih.gov/pubmed/2659436" class="external text" rel="nofollow">&quot;A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in Saccharomyces cerevisiae.&quot;.</A> Genetics. 1989 May; 122(1):19-27.</SPAN></P><P><SPAN id="Foot2"><SUP>[2]</SUP> Christianson, T.W.; Sikorski, R.S.; Dante, M.; Shero, J.H.; Hieter, P. (1992) <A href="http://www.ncbi.nlm.nih.gov/pubmed/1544568" class="external text" rel="nofollow">&quot;Multifunctional yeast high-copy-number shuttle vectors&quot;.</A> Gene. 1992 Jan 2; 110(1):119-22.</SPAN></P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Johns_Hopkins/Project/Vector">http://2011.igem.org/Team:Johns_Hopkins/Project/Vector</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Project/Vector" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Project/Vector" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/Vector&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Project/Vector&amp;oldid=195030" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>