"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins/Notebook/DNAAssayProtocol
From 2011.igem.org
VitaYeast - Johns Hopkins University, iGEM 2011
Team
Members
Acknowledgements
Advisors
Sponsors
Attributions
Gallery
Vitamins
Background
Overview
Vitamin A
Vitamin C
Measurements
Applications
Results
Future Plans
Yeast Toolkit
Background
Overview
Promoters and UTRs
Violacein
Yeast Vector Library
Results
Future Plans
Human Practices
Overview
IRB Approval
Data
Modeling
Platform
LBS Models
Optimization
Sensitivity
Parameter Fitting
Notebook
Protocols
Vitamin Experiments
Yeast Toolkit Experiments
Safety
Related Links:
Other Protocols:
DNA Protocols Page 1
Vitamin Assays
BioBrick
Contents
1 DNA Synthesis and Assay Protocols Page 2
2 Making Competent E. coli
3 Yeast Transformation 1
4 Yeast Transformation 2
5 Yeast Transformation 3
6 E. coli Transformation
7 Reaction to PCR Beta-Carotene Genes out of Cells
8 Gene Tagging
9 Yeast Genomic Preparation
DNA Synthesis and Assay Protocols Page 2
Making Competent E. coli
(From Pam Meluh)
Streak out TOP10 (or other strain) from freezer on plain LB (or other appropriate plate).
Inoculate 25 ml LB in a 250 ml flask; grow at 37°C in the air shaker with good aeration.
Next day, subculture O/N into a large volume of LB, diluting 1:100 (e.g. 5 ml O/N into 500ml). Make sure to use a large flask, so that the cultures will be well aerated (e.g. for 500 ml, use at least a 2 liter flask).
Incubate with rapid shaking at 37°C for 2-3 hours until the OD600 reaches ~0.4. If OD600 passes 0.5 start over! (Check OD600 after 1.5hr; when OD ~0.2, check OD again after 20 min to see if it's at ~0.4)
Chill centrifuge and appropriate rotor to 4°C and prepare cold 0.1M CaCl2
Put culture on ice for 10 minutes. Transfer to bottles and harvest by centrifugation at 4°C (try 10 min. at 4000 rpm for SL-250 rotor).
Loosen cell pellets by vortexing the bottles, gently resuspend each pellet in 10-20ml 0.1 M CaCl2, and consolidate cells in one bottle. Use pre-chilled glass pipets (or pipet up+down w/ cold CaCl2) to transfer cells and be very gentle with the cells from now on. NO MORE VORTEXING ONCE CaCl2 IS ADDED. Keep on ice at all times!
Let cells sit packed on ice for several hours (at least two hours). Harvest cells by centrifugation.
Gently resuspend cells in 1/20 (relative to the starting culture--e.g. 25 ml for a 500 ml culture) of ice cold 0.1M CaCl2 containing 15% glycerol. You can let the cells sit on ice here for a while as well.
Aliquot to sterile microfuge tubes (usually .5 and 1 ml aliquots). Flash freeze in liquid nitrogen and store at -80°C until use.
Thaw cells as needed for bacterial transformations.
Good TOP10 cells should give at least 3x106 transformants per microgram of super-coiled DNA (i.e. 3000 transformants per nanogram)
Yeast Transformation 1
The day before the transformation, there should be yeast on agar plate as colonies or in liquid culture.
Inoculate a colony or a small amount of liquid culture (depending on the OD of the culture; usually 250-500 µL) into 5mL YPD liquid culture
Place the liquid culture on a rotating drum at 30oC overnight (prepare two cultures at a time to balance the drum.
On the next day check the OD of the inoculate.
Re-inoculate the cultures to a final volume of 20 mL, with OD600 of 0.125
Each 20 mL should be good for 2 individual transformations
Separate the diluted culture into doubles, and incubate on rotating drum at 30oC for 3 to 5 hours.
Spin the culture at 2000rpm for 5 minutes to pellet the cells. Discard supernatant.
Wash by resuspending the pellets in 10 mL of water. Spin again at 2000rpm. Discard supernatant.
Wash again by resuspending the pellets in 10mL of 100mM lithium acetate (LiOAc). Spin again at 2000 rpm. Discard supernatant.
Resuspend the pellets in the residual LiOAc solution in the culture tubes.
In a test tube, mix the following solutions for each transformation, vortex to mix:
Reagents
Volume (µL)
50% polyethylene glycol (PEG-3350, Sigma)
240
1 M LiOAc
36
Sheared, heat-denatured herring sperm DNA (10 mg/ml)
20
DNA to be transformed (0.5-1 µg/µL)
14
Total
310
Add the suspend cells (cells from 10mL of culture from step 5) into the PEG mixture specified in the table above. Vortex to mix
Inoculate at 30oC for 30 minutes.
Add 36 µL of DMSO into each preparation.
Heat shock the preparations at 42oC for 15 minutes.
Pellet the cells at 2000rpm for one minute. Discard supernatant carefully by pipetting.
Resuspend cells in 400µL of 5mM CaCl2 buffer.
Incubate at room temperature for 10 minutes.
Plate cells directly onto YPD plates with necessary selection criteria (URA-, etc.). Spread with glass beads.
Incubate the plates at 30oC for 2 days.
Yeast Transformation 2
High Efficiency Yeast transformation  LM 8-23-2011 Modified from here
Grow a 20 ml culture of cells to be transformed to mid-log phase (OD600 1.2-2.0).
Pellet cells at RT at 2,000 x g for 3 minutes.  Decant supernatant.
Wash cells with 10 ml of dH2O.
Repeat step 2.
Resuspend cells in 750 µl of 100mM lithium acetate (LiOAc).
Transfer to 75µl to each tube.
During cell preparation, boil sheared salmon sperm carrier DNA for 5 minutes and then place on ice for at least 2 minutes.
Pellet cells in microfuge for 15 seconds.  Decant supernatant.
Add IN THIS ORDER to one tube:
240 µl for 50% PEG (mol. wt. 3350)
36 µl of 1.0 M LiOAc
25 µl of sheared salmon sperm DNA (10 mg/ml stock solution)
45 µl of cassette (from PCR above)
Mix.  Place in 30˚Crotator and leave for 30 minutes.
Heat shock in 42˚C water bath for 20-25 minutes.
Pellet at 6,000 x g for 1 minute. (or fast pulse for 15sec)
Resuspend cell in 500 µl of dH2O.
Plate mix on 2 YPD plates (100uL and 400uL) and incubate at 30˚C overnight.
Replica plate to the appropriate selective medium (YPD+G418) the following day.
After 2 days at 30˚C, pick large colonies and streak for single colonies on selective medium.
Note: If transformation efficiency is low, try doubling the amount of carrier DNA or replacing your stock. In our experience, the carrier DNA is crucial for efficient transformation.
In this case, confirm integration of tag by (1) PCR; (2) fluorescence microscopy; (3) Western.
Yeast Transformation 3
High Efficiency Yeast Transformation
Ref: Pan et al. Methods in Cell Cycle Research. 2007. 41(2): 206-21.
For each transformation:
6.25 OD600 nm (1 OD600 nm = ∼2 × 107 yeast cells) of the haploid-convertible heterozygous diploid YKO pool is inoculated into 50 ml YPD liquid (starting at 0.125 OD600 nm/ml in a 250 ml Erlenmeyer flask) and shaken at 200 rpm at 30 °C for 5.5 h to a cell density of ∼0.5 OD600 nm/ml.
The culture is harvested by spinning at 5000 rpm in a Sorvall RT6000B centrifuge for 2 min at 4 °C or room temperature, washed once in 10 ml of water and subsequently in 10 ml of 0.1 M lithium acetate (LiOAc), and finally resuspended in residual 0.1 M LiOAc in a total volume of 100 μl in a 1.5 ml Eppendorf tube.
A transformation mixture is freshly made according to Table 2 and added to the 100 μl of yeast competent cells. The transformation reaction is immediately mixed well by pipetting with a P1000 pipetter followed by vortexing (VMR, Vortexer 2) at top speed for 5–10 s, then incubated at 30 °C for 30 min.
72 microliters of dimethyl sulfoxide (DMSO; Qbiogene DMSO0001, Molecular Biology Grade) is subsequently added to the transformation reaction and immediately mixed thoroughly by vortexing at top speed for 5–10 s. DMSO is intrinsically sterile and no further sterilization is needed. It is also fairly stable when stored at room temperature and can be repeatedly opened and used for years.
The mixture is then incubated in a 42 °C water bath for 13 min.
After the heat-shock, cells are spun down at 3600 rpm in an Eppendorf centrifuge (Model 5417C) for 30 s and after aspirating the supernatant, the cells are resuspended and incubated in 400 μl of 5 mM CaCl2 for 5–15 min at room temperature. Incubation of cells in 5 mM CaCl2 for a short period of time improves transformation efficiency by 2 to 3-fold; however, prolonged incubation (>30 min) of cells in CaCl2 will reduce transformation efficiency.
For transformation with a yfgΔ::URA3 cassette, 1/2000 of the transformation is spread on a regular (100 × 15 mm) SC−Ura plate for calculating transformation yields. The remaining cells from the transformation are plated on a single 150 × 25 mm Petri dish containing SC−Ura for URA3 selection. ∼40 autoclaved glass beads (Fisher Scientific, Cat. # 11-312A, diameter 3 mm) are used to evenly spread the cells; these should be left inside the plate, which is inverted and incubated at 30 °C for 2 days. The spreading procedure should be gentle in the beginning to avoid splashing of cell suspension. A good transformation will give rise to 5–20 × 105 transformants, which form a confluent lawn of cells.
A recipe for the high efficiency yeast transformation mixture
Ingredient
Volume (μl)
50% polyethylene glycol (PEG-3350, Sigma)
480
1 M LiOAc
72
Sheared, heat-denatured herring sperm DNA (10 mg/ml)
40
Query construct DNA (∼0.5–1 μg/μl)
28
Total
620
Note: To prepare the herring sperm DNA stock solution, 1 g of dry sample (Sigma, D6898) is first dissolved in 100 ml of 1× TE buffer by stirring overnight in a cold room. The DNA is then sheared by sonication (3 × 10 s with 1-min interval sitting in ice), aliquoted (0.5–1 ml/aliquot), and stored at −20 °C. Incubate 500 μl aliquots of 10 mg/mL herring sperm DNA at 100 °C in a heat block for 5 min and cool on ice just before use. For best transformation efficiency, an aliquot of herring sperm DNA should not be repeatedly heat-denatured and frozen; always use herring sperm DNA from a new tube.
For transformation with an yfgΔ::natMX cassette, the 400 μl CaCl2 suspended cells should be transferred to a 250 ml Erlenmeyer flask containing 50 ml fresh YPD and shaken at 30 °C for 2–3 h to allow expression of the clonNAT-resistance gene. Cells are then spun down and resuspended in residual YPD in a total volume of 400 μl and plated on a single 150 × 25 mm plate containing YPD plus 50 μg/ml clonNAT. This plate is incubated at 30 °C for 2 days and replica-plated to another YPD plus clonNAT plate to reduce the background of untransformed cells. The new plate is incubated at 30 °C for another day. Transformation yield can also be monitored by plating a small aliquot on a regular Petri dish containing clonNAT as outlined above.
If multiple dSLAM screens are performed simultaneously, one should grow a single larger culture of the haploid-convertible heterozygote pool (e.g., 500 ml in a 2-liter flask for 10 screens) to prepare competent cells. A scaled-up master transformation mixture (omitting query construct DNA) for multiple transformations is also preferred. Each transformation should aim for >5 × 105 independent stable transformants to prevent random drop out of YKO mutants from the pool. The yield of a transformation is determined by a number of factors but is largely proportional to the amount of query DNA put into the transformation reaction. The number (∼5 × 108 or ∼25 OD600 nm cells) and the growth stage (after two doublings) of the competent cells also greatly affect transformation yield. In addition, bigger ORFs are typically more difficult to disrupt than smaller ORFs [9]. The exact reagents and experimental procedures used can also affect transformant yield but we recommend using those described here.
E. coli Transformation
Make sure heat block is holding at 42oC.
Prepare plasmids sample by dilution by adding 3µL sterile H2O to 1µL of plasmid. Dilution can be ignored if cells will be plated at diluted concentration, or if the plasmids concentrations are low.
Thaw one tube of E. coli competent cells (50µL) on ice. One tube is enough for two transformations.
Move 25µL of competent cells into new, chilled 1.5mL Eppendorf tube.
Keep the competent cells on ice at all time unless otherwise specified.
Add 1µL of plasmids into an aliquot of competent cells. Mix by gently flicking. Do not pipet up and down.
Incubate the preparation on ice for 20 minutes.
Heat shock the preparations at 42oC for 45 seconds.
Immediately place the preparation on ice for 2 minutes.
Add 125µL of LB medium to the preparation and incubate at 30oC for 1 hour.
Resuspend cells by gentle pipetting. The plate 100µL of the preparation on LB plates (with any other necessary selection criterion) and spread by glass beads.
Incubate the plates overnight at 37oC
Reaction to PCR Beta-Carotene Genes out of Cells
Reaction volume: 20 µL Reagents
dNTP(10mM): 0.4
primer A (10µM): 1µL
primer B (10µM): 1µL
Phusion Polymerase: 0.2 µL
H2O: 13.4 µL
TAE Gel:
2µL loading dye
5µL DNA
5µL H2O
tPCR
Templateless primer mix
Dilute oligos from 60µM (which is what they are delivered at) by a factor of 10.
Use diluted oligo stock to further dilute oligos by a factor of 20.
Add 10 µL of each oligo into an eppendorf tube.
Outer primer mix
Add 25 µL of each the first and last primer to an eppendorf tube. Mix thoroughly.
tPCR Reactions
Master Mix:
dNTP (10mM): 0.5 µL
Phusion Buffer: 5µL
Phusion Polymerase: 0.25µL
H2O: 14.75µL
Reaction tube:
Master Mix: 22.4 µL
2.5µL of templateless primer mix
Reaction conditions:
98ºC, 30 s
98ºC, 10 s
54ºC, 30 s
72ºC, 30 s
Repeat steps 2 to 4 50 times
72ºC, 10 min
4ºC, hold
fPCR
Dilute 5µL of tPCR product by a factor of 5.
Master Mix:
dNTP (10mM): 0.5 µL
Phusion Buffer: 5µL
Phusion Polymerase: 0.25µL
H2O: 12.75µL
Reaction tube:
Master mix: 20.5µL
Outer primer mix: 2 µL
diluted tPCR: 2.5 µL
Reaction conditions:
94ºC, 3 min
55ºC, 30 s
72ºC, 63 s
94ºC, 30 s
55ºC, 30 s
72ºC, 1 min
Repeat steps 4 to 6 25 times
72ºC, 3 min
4ºC, hold
Run the products on a gel to confirm correct band size
Steps to Assemble:
Overlap extension
PCR vector pRS416 with primers to linearize
Use CPEC to assemble the piece
Gene Tagging
PCR Round 1: GOAL: create PCR produce including HIS-TEV-mCherry-KAN-CYC1tt  Expected product size: ~2460bp
1x50uL Rxn.
10uL 5X Herculase II Reation Buffer
1 uL mCherry-KAN plasmid (pAR733)
1uL Herculase II fusion DNA polymerase
0.5uL DMSO
1.25uL F primer  (F-in-TEV-HIS-mCherry-AR+linker)
1.25uL R primer (R-cyc-tt-mCherry)
5uL dNTP to 50uL with H2O
PCR conditions:
5min 95C
30sec 95C
30sec 55C
75sec 72C (30sec/kb)
repeat 2-4 30 times total
7min 72C
forever 4C
Agarose gel to check amplification (load 2uL).  If successful, PCR cleanup using Invitrogen Kit; USE binding buffer B2 agarose gel again to verify product is still there (load 2uL).
PCR Round 2: Goal: To introduce 40bp of gene specific sequence at 5’ end of PCR product from round 1. Expected product size: ~2500bp
4x50uL Rxn Master Mix. (LEAVE OUT F primer)
40uL 5X Herculase II Reation Buffer
4 uL PCR product from PCR round 1
4uL Herculase II fusion DNA polymerase
2uL DMSO
5uL R primer (R-cyc-tt-mCherry)
20uL dNTP to 200uL with H2O
Vortex, pulse in centrifuge to collect at bottom of Eppendorf. Aliquot 4x48.75uL to PCR tubes Add 1.25uL of gene specific Forward primer to correct tube (F-out-crtE-TevHis, F-out-crtI-TevHIS, F-out-crtYB-TevHIS, F-out-BTS1-TevHIS).
PCR conditions:
5min 95C
30sec 95C
30sec 50C
75sec 72C (30sec/kb)
repeat 2-4 30 times total
7min 72C
forever 4C
Yeast Genomic Preparation
Centrifuge 1mL of each culture to be prepped (2000rpm, 2 min. RT).
Aspirate supernatant.
Add 200μL of breaking buffer, resuspend pellet by mixing with pipette tip.
Add glass beads up to the top of the mixture.
Add 200μL of phenol:chloroform:isoamyl alcohol (in hume hood).
Cap tubes tightly.
Vortex for 4 minutes at top speed.
Centrifuge for 10 minutes at top speed.
In hume hood transfer 75μL of aqueous layer into labeled tube containing 1mL 100% EtOH.
Invert 5 times to mix.
Centrifuge 20 minutes at RT top speed.
Aspirate supernatant.
Add 500μL of 70% EtOH.
Invert to mix.
Centrifuge for 5 minutes at RT top speed.
Aspirate supernatant.
Air dry at RT 10 minutes.
Resuspend pellet in 50μL of 10mM Tris 7.4pH.
Store at -20C.
Use 0.5L of each genomic prep in a 25μL standard PCR reaction.
Retrieved from " http://2011.igem.org/Team:Johns_Hopkins/Notebook/DNAAssayProtocol "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
