<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins_Notebook_VitExperiments"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Johns_Hopkins/Notebook/VitExperiments">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins/Notebook/VitExperiments&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/VitExperiments&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/VitExperiments&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins/Notebook/VitExperiments" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins/Notebook/VitExperiments</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>VitaYeast - Johns Hopkins University, iGEM 2011</TITLE></P><DIV id="mydroplinemenu" class="droplinebar"><UL><LI><A href="#">Team</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Members">Members</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Advisors">Advisors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Sponsors">Sponsors</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Team/Attributions">Attributions</A></LI><LI><A href="/Team:Johns_Hopkins/Team/Gallery">Gallery</A></LI></UL><LI><A href="#">Vitamins</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitA">Vitamin A</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/VitC">Vitamin C</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/MeasureQuant">Measurements</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Baking">Applications</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Vit/Future">Future Plans</A></LI><LI><A href="#">Yeast Toolkit</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Bg">Background</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Over">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR">Promoters and UTRs</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Violacein">Violacein</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Project/Vector">Yeast Vector Library</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/YT/Future">Future Plans</A></LI><LI><A href="#">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/HumanPrac">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/IRBProcess">IRB Approval</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Data">Data</A></LI><LI><A href="#">Modeling</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Platforms">Platform</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/LBSMod">LBS Models</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Opt">Optimization</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/Sensitivity">Sensitivity</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Modeling/ParaFit">Parameter Fitting</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments">Vitamin Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/YTExperiments">Yeast Toolkit Experiments</A></LI><LI><A href="https://2011.igem.org/Team:Johns_Hopkins/Safety">Safety</A></LI></DIV><DIV id="secondarycontentgreen"><DIV id="boxheading">
			Related Links:</DIV><DIV id="boxcontent"><DL><DIV class="heading">Vitamin Notebook:</DIV><DL><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/BCAssay">Beta-Carotene Assay</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/DoughMed">Dough Media</A></DD><DD><A href="https://2011.igem.org/Team:Johns_Hopkins/Notebook/Baking">Baking</A></DD></DL><DIV id="primarycontent"><H6><SPAN class="mw-headline" id="Vitamin_C_Biosynthesis:_Gene_Synthesis">Vitamin C Biosynthesis: Gene Synthesis</SPAN></H6><H5><SPAN class="mw-headline" id="8.2F12.2F11">8/12/11</SPAN></H5><P>Our order from IDT got mixed up, so we were initially missing either the first or last oligos of GDP-mannose 3,5-epimerase and GDP-L-galactose phosphatase, respectively. Instead of working on them initially, we decided to digest our vector, which was necessary based on the desired overlaps with our genes.
</P><P>We digested around the BamHI and HindIII sites of pRS416, and used sequential digests because the digestion buffers are incompatible. First BamHI and then HindIII. We ran the results of our cut (1st lane) and uncut (2nd lane) in a gel for comparison.
</P><P>The single band in the first lane represents the digestion of the vector into a linearized piece of DNA. The laddering in the uncut piece is common due to differing amounts of coiling in the circular DNA.
</P><H5><SPAN class="mw-headline" id="8.2F14.2F11">8/14/11</SPAN></H5><P>We had received the missing oligos from IDT. After having gone through several failed attempts, with gels being run with no bands of the correct size, we finally achieved a set of good results. We believe that the incorrect bands were a result of not adding in the all of the appropriate oligos: even missing one can drastically  We assembled them using our templateless PCR method, then ran the results of the GLGP (1st and 2nd) building blocks, and the LGPP gene on a gel.
</P><P>Our GLGP building blocks are of the right size, however our LGPP still seems unsuccessful. We expected to have trouble with this, because we are assembling the entire part at once (as opposed to in separate building blocks), and it is around 840 base pairs including the added overlaps.
</P><H5><SPAN class="mw-headline" id="8.2F17.2F11">8/17/11</SPAN></H5><P>We have GLGP, so we will strive to make our GME and LGPP building blocks. Using the same tPCR method, we run the results on another gel (ladder, LGPP, GME 1 and 2)
</P><P>The bands on every one of the genes is the appropriate size. There is some laddering on the GME 1 building block, however we do not think this is a problem. During a CPEC reaction, the only band that will be able to put the plasmid together is 
</P><H6><SPAN class="mw-headline" id="Sequencing">Sequencing</SPAN></H6><H5><SPAN class="mw-headline" id="8.2F21.2F11">8/21/11</SPAN></H5><P>The past week had been spent researching and designing the proper reactions for putting everything together with CPEC. We have experimented since then in the lab with optimizing the protocol, until we have created one that successfully creates the desired plasmid.
</P><P>We used CPEC to insert each gene into a vector (pRS 416), and then transformed these vectors via electroporation into <I>E. coli</I>. pRS 416 has an expressed gene for ampicillin resistance, so we can grow up the <I>E.coli</I> on ampicillin plates. We grew up the cells overnight, selected a single colony, and then grew that colony up in liquid media. We obtained plasmid DNA from these cultures using the QIAgen miniprep protocol, and sent these in to be sequenced.
</P><H5><SPAN class="mw-headline" id="8.2F23.2F11">8/23/11</SPAN></H5><P>Unfortunately, we received no correct sequences. We realize now that it is not the PCR reactions that introduce the most errors, but rather the errors present in the oligo mixtures to begin with. On top of that, the LGPP reactions did not work at all.
</P><P>The lack of reactions working led us to try running a colony PCR. For colony PCR, we took 16 colonies from each plate and then performed a PCR reaction with the first and last oligos. The products were ran on 0.8% agarose gels. The first image shows GLGP (top) and GME (bottom).
</P><P>It is clear that these inserts are present in the colonies we've plated. However, the LGPP gel is more discouraging, returning no positive results. 
</P><H5><SPAN class="mw-headline" id="8.2F26.2F11">8/26/11</SPAN></H5><P>Because of the lack of colonies on our LGPP, we looked at our fPCR products for the building blocks. We had made several trials of LGPP building blocks, and narrowed it down to two potential candidates. In addition, we decided to take 2 more colonies from the  and used the same reaction. We also included 2 colonies in this next gel in the last two columns:
</P><P>One of our LGPP fPCR products is of the correct size, the other is far too small, and different colonies still yield nothing. It is entirely possible that we used an incorrect PCR product, however we do not know why the plasmids were taken up even though they should have remained linear.
</P><H5><SPAN class="mw-headline" id="8.2F29.2F11">8/29/11</SPAN></H5><P>We have decided to make new CPEC reactions for each genes as well as LGPP, and then send 16 different colonies as opposed to only one. These colonies were grown up overnight in liquid media (with ampicillin resistance to ensure only <I>E. coli</I> with our  (and replica plated, so we would have more colonies should they become necessary), mini-prepped, and then an aliquot of the mini-prepped DNA was sent with custom sequencing primers.
</P><H5><SPAN class="mw-headline" id="9.2F1.2F11">9/1/11</SPAN></H5><P>We obtained no positive results. In addition, we also had a high rate of the sequencing reactions failing. We believe the cause was degradation/other problems with our custom sequencing primers.
</P><P>We examined the vector's sequence, and found that the T7 promoter primer and the M13Reverse primer (both universal sequencing primers) should give us the forward and reverse sequences of our genes in the vector. We also realized that instead of spending a large amount of time growing and mini-prepping many different colonies, we could instead plate the same colony on two separate plates, keeping one and sending the other in to be sequenced.
</P><P>The plates would be labeled in a grid, and have each colony streaked in the appropriate section of the grid. This way, the gene-sequencing service would be able to perform the sequencing reaction with a bacterial sample rather than a DNA sample, and once the sequencing reactions got back, we would be able to selectively mini-prep the winners we streaked out on replica plates.
</P><P>Since GLGP is the longest reaction gene, we decided to use our middle sequencing primer to ensure that we got the whole gene, and because our second custom primer should cover the rest of the gene, we will not request an additional M13Reverse primer reaction
</P><H5><SPAN class="mw-headline" id="9.2F5.2F11">9/5/11</SPAN></H5><P>Upon getting the results back, we found that only LGPP had given us a positive result. We would need to send in more colonies of GME. For GLGP, only one sequence returned no errors using the T7 primer (the first 700 base pairs) unfortunately, ALL of our custom primer reactions didn't work. However, now that we have a certain result for a perfect LGPP gene, we will focus our attention on GLGP and GME.
</P><P>We took 16 more single colonies of each and streaked them out on two separate plates.
</P><H5><SPAN class="mw-headline" id="9.2F7.2F11">9/7/11</SPAN></H5><P>Today we received our sequencing reactions back, and have determined that we have all of our genes properly formed.
</P><H6><SPAN class="mw-headline" id="Synthesizing_Vitamin_C_Yeast">Synthesizing Vitamin C Yeast</SPAN></H6><H5><SPAN class="mw-headline" id="9.2F10.2F11">9/10/11</SPAN></H5><P>In order to create a strain of yeast that has all three of the genes in it, we cannot use the same vector for all 3 genes, as we have been in the past. The reason for this is that we cannot use the same selectable marker to choose the yeast cells which have all 3 plasmids, since they would all have the same marker. In other words, we could not tell which yeast colonies grew up with one, two, or all three plasmids.
</P><P>In order to counteract this problem, we have decided to put the genes into 3 separate vectors: we will use pRS413 and pRS415 in addition to pRS 416. These vectors correspond to his-, leu-, and ura- respectively. This way we can make a plate lacking these three metabolites, and any cells that grow on them will have to have all three plasmids.
</P><P>But this will only get the genes inside of the yeast, it will not necessarily express them. Therefore, we need to make a proper cassette, with a promoter region in front of the gene and a 3'UTR trailing it.
</P><P>The three genes will be in the following vectors flanked by the following promoters and 3'UTRs (found in yeast)
</P><OL><LI> GLGP - pRS413, flanked by TDH3 promoter and 3'UTR
</LI><LI> GME - pRS415, flanked by RPS8B promoter and 3'UTR
</LI><LI> LGPP - prs416, flanked by RPL24A promoter and 3'UTR
</LI></OL><P>We wish to use a CPEC reaction to put all of our pieces together, but first we have to add appropriate common overhangs to corresponding pieces. To this end, we have designed primers that should be able to amplify our pieces from plasmids received from our teammates with the appropriate promoter or 3'UTR region, as well as the vectors and genes.
</P><H5><SPAN class="mw-headline" id="9.2F11.2F11">9/11/11</SPAN></H5><P>The vectors have been successfully linearized, along with addition of the appropriate overlaps. The primers, however, didn't work on the promoter and 3'UTRs. Because of that, we have designed new ones that will instead both take the promoter and UTR regions directly out of genomic DNA and add the necessary overlaps in one reaction.
</P><H5><SPAN class="mw-headline" id="9.2F15.2F11">9/15/11</SPAN></H5><P>The primers have arrived, and we used them on genomic DNA to extract out promoters and UTR's, in addition performing PCR on our genes. The results have been run on a gel.
</P><P>After initially having trouble getting the reactions to work correctly, we have successfully created products of the correct size. We still are having trouble with the RPS8B 3'utr, and have since designed and ordered new primers to extract larger pieces from the genomic DNA (since we assume that junk DNA after the 3'UTR should not affect gene expression.
</P><H5><SPAN class="mw-headline" id="9.2F20.2F11">9/20/11</SPAN></H5><P>We have received the second set of primers for the RPS8B 3'UTR, however it is still giving us trouble. After attempting yet another PCR reaction, we believe that the annealing temperature for the PCR pieces is too low for good binding. We will have to order yet another set of primers.
</P><H5><SPAN class="mw-headline" id="9.2F23.2F11">9/23/11</SPAN></H5><P>The second set of primers also didn't work. We are going to attempt a gradient PCR in order to see whether our reaction works at any annealing temperature at all. We have, however, put both GLGP and LGPP into their respective CPEC vectors and plan on growing them up in <I>E. coli</I> before transferring them to yeast.
</P><H6><SPAN class="mw-headline" id="Tagging_Genes.2C_Transformation.2C_Fluorescence_Microscopy.2C_and_Protein_Purification">Tagging Genes, Transformation, Fluorescence Microscopy, and Protein Purification</SPAN></H6><H5><SPAN class="mw-headline" id="9.2F03-04.2F11">9/03-04/11</SPAN></H5><P>Performed 1st round PCR to create a PCR product that contains the tag sequence.  There was some DNA streaking as a result of a non-optimal annealing temperature creating partial product.  Performed PCR clean up and ran gel electrophoresis to confirm the success of the PCR. 
</P><H5><SPAN class="mw-headline" id="9.2F10.2F11_2">9/10/11</SPAN></H5><P>Performed 2nd round PCR to create a PCR product that contains the tag sequence with enzymatic genes.  There was minimal DNA streaking as the annealing temperature was dropped from 55C to 50C.  Performed PCR clean up and ran gel electrophoresis to confirm the success of the PCR. 
</P><H5><SPAN class="mw-headline" id="9.2F16-17.2F11">9/16-17/11</SPAN></H5><P>Performed Transformation using Transformation Protocol 2 and replica plated the next day.  Colonies grew in a thick lawn after the first overnight incubation due to a high OD to increase the amount of competent cells.  Colonies grew on each of the different strain plates.
</P><H5><SPAN class="mw-headline" id="9.2F21.2F11">9/21/11</SPAN></H5><P>Fluorescence Microscopy was performed on the 4 strains of transformed yeast cells. All controls showed no red fluorescence, whereas all strains showed red fluorescence.  Green fluorescence was noted as beta carotene in cells.  DIC microscopy showed beta carotene in lipid droplets.
</P><H5><SPAN class="mw-headline" id="9.2F22.2F11">9/22/11</SPAN></H5><P>Genomic Prep of the DNA tagged genes from cells was performed for protein purification assay.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments">http://2011.igem.org/Team:Johns_Hopkins/Notebook/VitExperiments</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins/Notebook/VitExperiments" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins/Notebook/VitExperiments" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/VitExperiments&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins/Notebook/VitExperiments&amp;oldid=199462" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>