"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins/Project/PromUTR
From 2011.igem.org
VitaYeast - Johns Hopkins University, iGEM 2011
Team
Members
Acknowledgements
Advisors
Sponsors
Attributions
Gallery
Vitamins
Background
Overview
Vitamin A
Vitamin C
Measurements
Applications
Results
Future Plans
Yeast Toolkit
Background
Overview
Promoters and UTRs
Violacein
Yeast Vector Library
Results
Future Plans
Human Practices
Overview
IRB Approval
Data
Modeling
Platform
LBS Models
Optimization
Sensitivity
Parameter Fitting
Notebook
Protocols
Vitamin Experiments
Yeast Toolkit Experiments
Safety
Related Links:
Yeast Toolkit:
Background
Overview
Violacein
Yeast Vector Library
Results
Future Plans
Promoters and Terminators
The iGEM registry is lacking in promoter and terminators sequences for yeast, especially when compared with the number available for use in E. coli. In order to address this problem and increase the use of yeast in synthetic biology we have decided to provide a library of promoter and terminator sequences from 12 yeast genes: 4 causing high expression, 4 causing medium expression, and 4 causing low expression. We hope these BioBrick compatible parts will make it easier for future iGEM teams to work with Saccharomyces cerevisiae.
Characterizing the Promoters
In order to provide useful parts with better ease of use, we have decided to not only provide a library of useful yeast promoters to the part registry, but also quantitatively characterize the strength of our promoters by using reference promoters. In a nut shell, we compared the ability of our promoters to express the GFP gene to that of other well characterized promoters in identical laboratory strain. This way, iGEM teams in the future can accurately predict the functionality of our promoters in their strains of interests by referencing the functionality of the reference promoters. A diagram and a brief step-by-step explanation of the process is provided below:
Plasmids with the reference promoter (in white), GFP, ampicilin (AMP) resistance, and Ura3 production genes were provided by the Tom Ellis lab in Imperial College London.  Three different versions of the plasmids with different reference promoters (ADH1p, CYC1p, PFY1p) were provided.  The AMP resistance gene allows for selection of bacterial transformation, while the URA3 production gene allows for selection of yeast transformation.
The given plasmids were first transformed into ‘E. coli’ then miniprepped to retrieve the plasmids.  Some of the miniprep product undergoes StuI digest (see step 4), while the rest undergoes EcoRI (E) and SpeI (S) digest. The E and S digest cleaves the reference promoter out of the plasmids.
The E + S digested plasmids were then ligated with our promoters of interest, which have also been digested with E + S to have compatible overhang. The ligation results in a complete plasmids with integrated promoters of interest
The complete plasmids then underwent StuI digest. The StuI digestion site is in the Ura3 gene. This step results in a linear DNA with parts of Ura3 gene on both ends.
The linear DNA from step 3 can then be integrated into the yeast genome due to the homologous Ura3 domains flanking the DNA.
The linear DNA pieces were then transformed into YPH500, a yeast strain that has a mutation in its Ura production gene. The transformed yeasts were then plated on Ura deficient YPD plates. Only cells with successful integration of linear DNA are able to grow.
Colonies were then picked and grown overnight. The GFP production of the cells was then analyzed using flow cytometry. The GFP production of strain with our promoters of interests was then compared to the GFP production of strains with the reference promoters.  The relative strength of our promoters was quantified.
Promoters
Promoter sequences that have been BioBricked:
RPS8Bp
BAP2p
FCY2p
Gal 1/10p
HHO1p
KRE9p
PRY1p
STM1p
TDH3p
RPS2p
Terminators
3'UTR sequences that have been BioBricked:
ARD1utr
BAP2utr
FCY2utr
HHo1utr
KRE9utr
PRY1utr
RPL8Autr
RPL24Autr
RPS2utr
RPS8Butr
TDH3utr
Promoters and 3'UTRs for Golden Gate Assembly
We also have a second promoter/3'UTR library consisting of the same sequences with custom overhangs with BsaI sites rather than the BioBrick sites. These are being used in our two violacein projects , where they will allow Golden Gate assembly of violacein expression cassettes.
Retrieved from " http://2011.igem.org/Team:Johns_Hopkins/Project/PromUTR "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
