"
Page
Discussion
View source
History
teams
Log in
 
Team:Technion/Project/Reporter
From 2012.igem.org
Home
Team
Team
Gallery
Team Profile
Project
Project description
Overview
YES gates
Riboswitch (RS)
RNA polymerases
Reporter system standard
The phage
Modeling
Parts
Notebook
Safety
Attributions & Achievements
Human Practices
Articles about us
High School lecture
Sponsors & Thanks
Contents
1Objective
2Work plan
3Results
3.1The Assay Genes
3.2The Promoter Plasmids
3.3Restriction,  Ligation and Transformation
3.3.1phoA Colony PCR
3.3.2xylE colony PCR
3.4Controls
3.4.1Positive Controls
3.4.1.1Restriction, Ligation and Transformation
3.4.2Negative Controls
3.5Characterization of the Assays
3.5.1Alkaline Phosphatase
3.5.2xylE
4Conclusions and summary
5Best BioBrick Measurement Approach:
6References
Objective
Figure 1: Assay  Plasmids' cassettes.
There are  two parts in our system, the Phage and the E. coli. Before we can test  them together, we have to make sure that our parts work individually. For that  purpose, we allocated two team members to create assays that can test the  function of our E. coli parts, the inducible RNA polymerases, and  characterize them.
The  cassettes that are used in the plasmids are shown in Figure 1.
Work plan
We intend to  create two colored assays that can indicate and quantify the expression of the  different polymerases from our plasmids. Each assay gene will be cloned after  each of the RNA polymerase promoters, so that the expression of the polymerases  will be linked to the color developed. Before using them with the designated  RNA polymerase plasmid, we intend to characterize them by using different  inducer concentrations and measuring the absorbance.
Main Parts:
Alkaline Phosphatase Assay: In E.  coli, alkaline phosphatase (ALP) is encoded by the phoA gene. The wild  type phoA gene from Citrobacter has a signal sequence directing export  of alkaline phosphatase into the periplasm, where it is active. In this  protocol, the colorless substrate p-nitrophenyl phosphate (PNPP) is hydrolyzed  by ALP, producing nitrophenol, a yellow color product, which is quantified by  an assay at a maximum absorbance 420 nm.
xylE Assay: The xylE  gene from Pseudomonas putida TOL  pWW0. This gene encodes the enzyme catechol-2,3-dioxygenase  (metapyrocatechase), which converts catechol to the bright yellow product  2-hydroxy-cis,cis-muconic semialdehyde. This is a useful reporter gene;  colonies or broths expressing active XylE, in the presence of oxygen, will  rapidly convert catechol, a cheap colorless substrate, to a bright yellow compound  with an absorbance maximum around 380 nm.
Promoters: In order to  check as many different polymerases to see with which set of enzymes we achieve  minimum cross-reactivity, we worked with 5 polymerases. Both the polymerases  and their matching promoters were acquired from Christopher A. Voigt1.  He sent us the four RNA polymerases promoters: pT7*,p T3, pK1F, and pN4 with a  T7 terminator at the end.These four plasmids will also function as our  backbone plasmids, and to them we'll clone the assay genes along with their  RBS. The Sp6 promoter was added via PCR over the T7 promoter.
Controls:
Positive- In order to see whether our assay's genes, phoA and xylE, work properly and generate a colored output, we created a  positive control. The positive control will consist of a Tetracycline  repressible promoter (pTetO) before each of the assay's genes. In the absence of an inducer, TetR binds to tetO  and prevents transcription. It can be turned on when an inducer, such as  tetracycline, binds to TetR and causes a conformation change that prevents TetR  from remaining bound to the operator.
Figure 2: an illustration of the pET system as designed originally [A] and our own  pET as used in our experiment [B]
When the operator site is not bound, the  activity of the promoter is restored. Unfortunately,  we don't have a bacterium strain with the repressor we plan to use the pTetO as  a constitutive promoter.
Negative: We have two kinds of negative controls: The  first one is to see whether we get any absorbance without the presence of the  assay parts, promoter + assay gene. The second one is to measure the leakage of  the polymerase promoters. For that reason, we will measure the absorbance of  the promoter plasmids without the polymerase genes. 
pET Expression System: In order to  characterize our assay, we intend to use the pET expression system. The host  cell for the pET expression system is a bacteria which has been genetically  engineered to incorporate the gene for T7 RNA polymerase, the lac promoter and  the lac operator in its genome (BL21 strain). When lactose or a molecule  similar to lactose is present inside the cell, transcription of the T7 RNA  polymerase is activated.Therefore, we have a similar system to the T7  RNA polymerase plasmid that we intended to create in the first place. By  transformation of our pT7 test plasmid to the engineered bacteria with the  endogenous T7 polymerase gene, we can characterize our pT7 test plasmid. By adding  rising concentrations of the inducer, IPTG, and measuring the absorbance for  each one, we can deduce whether we have a rising concentration of our assay  genes, phoA and xylE. However, this only works in bulk or culture  assay and not single cell assays, since in single cells the lac system is  on/off. Meaning that at low concentrations of IPTG some cells will turn on,  while the rest will be off. An illustration of the normal pET system and our "made" system is shown in Figure 2.
Results
The Assay Genes
Figure 3: gel electrophoresis for the PCR results of all the promoter backbone  plasmids. Expected size: 3400 bp.  The  left lane is a standard 1 Kb ladder (NEB). A- Only the T7 promoter  yielded positive results. B- Only the K1F promoter yielded positive  results. C- the remaining promoters- T3,N4 and SP6 yielded positive  results.
In order to  have our assay ready we ordered the Alkaline Phosphatase reporter gene, phoA,  from the registry (BBa_J61032). We already had the xylE reporter gene in  the distribution kit (BBa_J33204). We added XhoI and XmaI restriction sites in  both ends of the parts via PCR, for the phoA gene we also added an RBS  (BBa_B0064). Since the phoA sequence was reported to be inconsistent we sequenced it and found that  there was only one silent mutation in the phoA CDS.
The Promoter Plasmids
Due to the  fact that the promoter plasmids had an mRFP gene between the promoter and the  terminator, we ran another PCR in order to delete it, in which the primers were  at both ends of the mRFP facing outwards. In order to have a fifth plasmid with  the SP6 promoter2 we performed another PCR, only with different  primers: we engineered primers that had the Sp6 promoter in them while the  template was an pT7 plasmid. This way, the T7 promoter was replaced with the  Sp6 promoter. In all of the reactions described above, we added XhoI and XmaI  restriction sites in both ends of the plasmids.
As  shown in Figure 3 we managed to get all the backbones in the  right size. The first run we had a positive result only with the pT7 backbone,  the second run we had only the K1F backbone and in the third run, after raising  the Tm by 10 degrees, we had the rest of the plasmids backbones- T3, N4 and  SP6.
Restriction,  Ligation and Transformation
Figure 4: gel electrophoresis for the Colony PCR results  of all the plasmids with phoA as an insert. Expected size: 1750 bp . The left  lane is a standard 1 Kb ladder (NEB).  A- Four positive colonies for the pT7  backbone can be seen; control stands for PCR mix without a colony. B- Two positive colonies for the pN4 and pT3 backbones can be seen; the control is  a ligation reaction without an insert. C- One positive colony for the  pSp6 backbone can be seen; control is the PCR mix without a colony.
Figure 5: gel electrophoresis picture for the colony PCR  results of all the plasmids with xylE as an insert. Expected size: 1200 bp. The  left lane is a standard 1 Kb ladder (NEB).  A- One positive colony for the pT7  backbone can be seen; control stands for PCR mix without a colony. B- At  least one positive colony for the pT3, pN4, pK1F and pSP6 backbones can be  seen; the pT3 and pSP6 controls stands for a ligation reaction without an  insert, the "control- no colony" stands for the PCR mix without a  colony.
We performed  restriction with XhoI and XmaI restriction enzymes on both the plasmids and the  inserts (phoA and xylE) and ligated them together. In order to  check whether the ligation was successful and we got ourselves new BioBricks we  did a colony PCR with one primer for the insert and one for the plasmid.
phoA Colony PCR
As  shown in Figure 4, we managed to ligate the phoA insert  with most of our backbones, except for the K1F promoter backbone. This means  that we got 4 new BioBricks :)
xylE colony PCR
As shown in Figure  5, we managed to ligate the xylE insert with all of our  backbones. This means that we got 5 new BioBricks :)
Controls
Positive Controls
Figure 6:  gel electrophoresis for the PCR results of the  positive control backbone (triplicate). Expected size: 2300 bp. The left lane is a standard 1 Kb ladder (NEB).
As mentioned  before, after we made our new BioBricks we wanted to know whether the genes are  expressed and the assays actually work. So we ligated our inserts- phoA and xylE, to a backbone from the distribution kit (BBa_I13600), which  contained the pTetO promoter along with an engineered CFP (eCFP) gene which we  removed by PCR. We also added XhoI and XmaI restriction sites in both ends of  the plasmid. 
As shown in Figure  6, the experiment yielded a positive result.
Figure 7: gel electrophoresis for the colony PCR results  of the positive control plasmids with each insert. Expected size for phoA: 3750  bp, and for xylE: 3250 bp. The left lane is a standard 1 Kb ladder (NEB).
Restriction, Ligation and Transformation
We performed  restriction with XhoI and XmaI restriction enzymes on both the plasmids and the  inserts (phoA and xylE) and ligated them together. In order to  check whether the ligation was successful and we got ourselves new BioBricks we  did a colony PCR with one primer for the insert and one for the plasmid.  uniform
As shown in Figure  7, we had positive colonies only for the xylE and not for the phoA.
As shown in Figure 8, a distinct yellowish hue in the xylE assay was  detected after the substrate was added, which means our xylE gene works  as planned.
Figure 8: Presents a starter of the BL21 bacteria with the pTetO + xylE plasmid. On the left is a tube with the addition of catechol, on the right is the one without an addition of catechol.
Negative Controls
The first negative control is a  bacteria without the assay genes, which means the bacteria contains only the  PET system. The results of the absorbance After induction with IPTG and  performing the assays protocols are shown in Table 1:
Table 1: Absorbance results for the first negative  control, after blank reduction, for xylE and phoA assays after  induction with IPTG
 
Absorbance
xylE assay(380 nm)
0.064
phoA assay (420 nm)
0.000*
* Because of the blank reduction  the absorbance was negative, which is not a valid absorbance.
As seen in the results, the absorbance acquired  from the assays was very low. This indicates that there hasn't been enough  enzyme in the bacteria that could have caused the yellow color.
The second negative control is  a bacteria containing only the assay part- T7 promoter+ xylE\phoA genes, without any RNAP. The results of the absorbance after performing the  assays protocols are shown in Table 2:
Table 2: Absorbance  results for second negative control after blank reduction for xylE and phoA
 
Absorbance
xylE (380 nm)
1.200
phoA (420 nm)
0.171
As  seen in the results, the absorbance yielded from the assays was low in comparison  to the assay results from the pET experiment, [0.5 to 1.5] for the phoA and [10 to 60] for the xylE. This indicates that the T7 promoter has a  very low leakage level.
Characterization of the Assays
As mentioned  before, the characterization of the parts BBa_K784000- pT7+phoA and BBa_K784001-pT7+xylE will be done by adding different concentrations of  the inducer, IPTG, while the saturation time and amount of substrate for the  assay are identical.
Alkaline Phosphatase
The graph  below presents the results of the experiment described. The absorbance at 420nm  for each of the different concentrations of the inducer in a range of 0.5 μΜ to 1000 μΜ on a logarithmic scale. The absorbance was calculated via  different dilutions of the samples: dilutions by 0.5, by 0.25, and by 0.1. The  error bars represent the processing of the data collected. The line at the  bottom of the graph represents the basal level according to the control result-  no IPTG induction.
Figure  9: absorbance at 420nm against concentration of inducer
As can be seen, the graph shows a clear positive  tendency- the higher the concentration the higher the absorption, as expected. Starting  from a concentration of  40 μM and above, there are only small deviations from  the absorption value of  1.2, probably  due to the fact that saturation has been achieved.
xylE
The graph  below presents the results of the experiment described. The absorbance at 380nm  for each of the different concentrations of the inducer in a range of 0.5 μM to 500 μM on a  logarithmic scale. The absorbance was calculated via different dilutions of the  samples: dilutions by 0.5, by 0.25, and by 0.1. The error bars represent the  processing of the data collected. The line at the bottom of the graph  represents the basal level according to the control result- no IPTG induction.
Figure  10: absorbance at 380nm against concentration of inducer.
As can be seen, the graph shows a small tendency  to rise, but it's not clear enough. There are several explanations, one is that saturation has been achieved  from lowest concentration measured, which shows both that the assay is highly  sensitive and that a small amount of polymerase is enough to cause translation.  The experiment should be repeated with either lower concentrations of IPTG or  Catechol. The line at the bottom of the graph represents the basal level  according to the control result- no IPTG induction
Conclusions and summary
We have produced two separate assays for testing the presence of the RNA polymerases that is used for the  induction of the phage.
We have tested two parts BBa_K784000- pT7+phoA and BBa_K784001-pT7+xylE and found it fully functional.
Since both of our assays show  very low leakage the possibility of getting a false positive result is not  high.
Since in both cases very small  amounts of IPTG showed positive results, we can deduce that even a small amount  of the RNA polymerase can activate the promoters. In terms of the Trojan Phage,  that can create a situation that even if there are no activators present in the  cell, the leakage of the polymerase promoters can create enough polymerases to  translate the phage's proteins and cause lysis.
Best BioBrick Measurement Approach:
We have developed an assay that can be used to test the activity of the T7 polymerase, or any polymerase, by using any of the similar plasmids that we have made or by simply replacing the promoter using Xbal and XhoI restriction enzymes.  However, it can also act as a reporter system for the production of any part by a simple cloning step: first extracting the RBS+xylE / RBS+ALP part via XhoI and XmaI and inserting it into any plasmid that holds a desired gene. By performing the assay, one can measure the amount of the part translated according to the intensity of the color. It can be used instead of a fluorescent gene or as an addition to one.
Moreover, combined with the pET system, or one of our polymerase plasmids, the systems can be used as an inducible production systems for any Biobrick, that can also measure  the amount produced. In order to use it in this manner, one has to insert the required part with an RBS after the T7 promoter via XhoI, transfer it to a bacteria with the standard pET system, and add the desired amount of inducer, IPTG, according to the graph presented in the experience part in the registry. The result is an inducible system in which the user can choose the production amount of any chosen Biobrick and verify it by the colored assay.
If by simply adding another measurement option, as an alternative to fluorescent genes, or by creating an inducible measurable production system, we have created an important, easy- to- use Biobricks  that can be used as measurement systems in a wide range of applications.
References
Temme K., et al. 2012. Modular control of multiple pathways using  engineered orthogonal T7 polymerases. Nucleic Acids Research (advance access):  1-9.
Shin I., Kang C. 2003. Mutational analysis and structure of the  phage SP6 promoter. Methods in Enzymology 370: 658-668.
Retrieved from " http://2012.igem.org/Team:Technion/Project/Reporter "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
