"
Page
Discussion
View source
History
teams
Log in
 
Team:Technion/Project/RNAPs
From 2012.igem.org
Home
Team
Team
Gallery
Team Profile
Project
Project description
Overview
YES gates
Riboswitch (RS)
RNA polymerases
Reporter system standard
The phage
Modeling
Parts
Notebook
Safety
Attributions & Achievements
Human Practices
Articles about us
High School lecture
Sponsors & Thanks
Contents
1Overview
2The different sources for our RNA polymerases
3WT polymerases
4Chris' Engineered polymerases
5Workplan
6Results
6.1Notes
7References
Overview
The purpose of different RNAPs used in our project is to control expression from suitable promoters inserted into phage genome. Addition of inducers to the medium will trigger transcription of polymerases needed to promote transcription of phage’s genes, thus allowing the phage to complete its lytic cycle. At first we thought to use three native polymerases to control our AND gates – T7, T3 and SP6 RNAPs. Those polymerases arise from T7, T3 and SP6 bacteriophages, respectively and belong to the same family of polymerases. They were chosen to be used in our project for 2 main reasons  {1}:
Those polymerases have high specificity recognizing their own promoters, but not native E.coli‘s or other polymerases’ promoters, those minimizing dangers of unwanted cross-reactivity.
pT7, pSP6 and pT3 are very strong promoters, so their activity is very tightly regulated by inducers that can't be found in a specific E.coli we used in our project. Also, using these promoters prevents possible leakiness.
However, we had discovered that phage polymerases can exhibit toxicity to host cells, so we began to search after less toxic variants. Luckily, Ilya have found an article published by Chris Voight et al. Chris’ research focused around creating a set of orthogonal polymerases based on T7 wild type (WT) RNA polymerase (RNAP) that will be less toxic to the host cells and will recognize different promoters.
Chris had agreed to share his work with our team, so in total during our project we had worked with 7 different polymerases: T7, T3 and SP6 WT RNAPS and T7*, T7*(T3), T7*(N4), T7*(K1F) mutants produced by Chris and his team (‘e’ before RNAP’s name means engineered polymerase).
The different sources for our RNA polymerases
Some of the polymerases used during our project were donated to us. We want to thank the researchers that helped us out:
Christopher A. Voight – for donating his synthetic engineered polymerases with suitable promoters: T7*, T7*(T3), T7*(N4), T7*(K1F).
Ann K. Ganesan – for donating a plasmid with WT T7 RNA polymerase.
Changwon Kang - for donating a plasmid with the SP6 RNAP.
WT polymerases
During our project we have worked with 3 wild type polymerases produced by phages: T7 RNAP, SP6 RNAP and T3 RNAP.
T7 RNAP is transcripted from gene 1 of T7 bacteriophage. It consists of single polypeptide chain with weight of 98 kDa. {2}
SP6 RNAP is produced by SP6 bacteriophage that grows on Salmonella typhimurium LT2. It consists of single polypeptide chain with weight of 96 kDa. {2,3}
T3 RNAP is produced by T3 bacteriophage (T7-like virus that infects E. coli). It consists of single polypeptide chain with weight of 100 kDa. {4,5}
T7, T3 and SP6 polymerases belong to same polymerase family and share some common features {2,4}:
They require Mg+2 as a cofactor and their activity can be stimulated by addition of BSA or spermidine.
The RNAPS possess very stringent promoters meaning that there is almost none cross-talking between them.
Unlike bacterial polymerases they are not inhibited by rifampicin antibiotic.
In molecular biology those RNAPS are used for in vitro/in vivo specific RNA synthesis.
Chris' Engineered polymerases
The aim of Chris Voight’s research {6} was to broaden the arsenal of polymerases available for creating synthetic genetic constructs. Chris and his team had created 4 orthogonal polymerases based on T7 WT RNAP with affinities to different promoters and with reduced toxicity to host cells {6}.
To achieve reduced levels of toxicity Chris and his team had added N-terminal degradation tag to the T7 WT polymerase to reduce its concentration in host E.coli cell and controlled the RNAP's expression using a weak ribosome binding site and 'GTG' start codon. Besides, during the cloning of the polymerase a spontaneous mutation in the polymerase active site (R632S) arose. This mutation had reduced polymerase toxicity without noticeable effects on the polymerase activity {6}.
After achieving reduced toxicity, Chris and his research group had done mutations inside the polymerase region known as specificity loop in order to create variants of the polymerase that recognize different promoters. The final result was a set of 4 orthogonal polymerases and given them the following names: T7* – recognizes pT7, T7*(T3) – recognizes pT3, T7*(K1F) – recognizes pK1F and T7*(N4) - recognizes pN4 {6}.
Workplan
Different RNAPs were used in our project in order to create different YES gates. In order to create a gate, the RNAPs were amplified via PCR reaction and when cloned downstream to inducible promoter and reporter fluorescent protein. More information about the logical gates and their creation can be found in YES gates section of our wiki.
Results
The results of cloning the RNAPs under different inducible promoters are shown in the table below. (V) means successful attempt of Yes gate creation, (X) - failed attempt. For more information see notes below the table.
Table 1: Results of RNAPs' cloning
RNA    polymerase
YES    gate #1 (PTetO)1
Yes    gate #2 (PLac/Ara)
Yes    gate #3
(PLux+RS)2
Polymerase    compatibility with BioBrick Standard3
Remarks
T7
X
To create the gate the first step was the    addition of a terminator along    with additional restriction sites for the cloning of different RNAPs. Due to    the failure of this step no cloning has been done to the plasmid.
V
V
-
T3
X
X
V
Was already a BioBrick - BBa_K346000
SP6
X
X
X
See note #3
T7*
X
X
V
Has been cloned upstream to BBa_B0015 in pSB1AK34
T7*(T3)
X
V
V
-
T7*(N4)
X
V
V
Has been cloned upstream to BBa_B0015 in pSB1AK34
T7*(K1F)
X
V
V
Has been cloned upstream to BBa_B0015 in pSB1AK34
Notes
YES gate #1:
We will continue our work after wiki freeze and till the day of departure. We hope to create the gate with T7* (K1F), T7*(N4) and T7* RNAPs till when.
YES gate #3:
We have managed to successfully fuse T7, T7*(T3), T7*(K1F) and T7*(N4) with riboswitch and put the resulting construct under control of a tac pormoter, due to lack of time we haven't transferred RS+RNAP constructs under control of pLux as planned.
Compatibility with BioBrick standard:
SP6 RNAP gene contains many restriction sites that are similar to ones found in the standard prefix and suffix, such as EcoRI and SpeI sites, so it can't be used as part of standard BioBrick.
T7*, T7*(N4) and T7*(K1F) were cloned into pSB1AK3 upstream to BBa_B0015 . However, due to lack of time, they were not incorporated into any of the YES gates.
References
1. Lodge J., Lund P., Minchin S. (2007) "Gene Cloninng: Principles and Applications", Taylor & Francis Group, New York, NY. p. 258.
2. Dr. Aehle W. (ed.) (2007) "Enzymes in Industry", WILEY-VCH Verlag GmbH & Co. KGaA, Weinheim. pp. 370, 372
3. Butler E. T., Chamberlin M. J. (1982) "Bacteriophage SP6-specific RNA Polymerase", The Journal of Biological Chemistry. Vol. 257 (10), pp. 5772-5778.
4. Brown T. A. (1998) "Molecular Biology Labfax: Recombinant DNA", Academic Press, San Diego, California. p. 202.
5. The UniProt Consortium, < Bacteriophage T3 > (Visited on 23/09/12)
6. Temme K., Hill R., Segall-Shapiro T. H., Moser F. Voight C. A. (2012) "Modular control of multiple pathways using engineered orthogonal T7 polymerases", Nucleic Acid Research, 1–9 doi:10.1093/nar/gks597.
Retrieved from " http://2012.igem.org/Team:Technion/Project/RNAPs "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
