<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Technion_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Technion/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Technion/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Technion/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Technion/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Technion/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Technion/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><UL id="navbar"><LI><A href="https://2012.igem.org/Team:Technion">Home</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Team Gallery">Gallery</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Technion">Team Profile</A></LI></UL><LI><A href="https://2012.igem.org/Team:Technion/Project/Description">Project</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/Description">Project description</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/Overview">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/YES_gates">YES gates</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/RS">Riboswitch (RS)</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/RNAPs">RNA polymerases</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/Reporter">Reporter system standard</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Project/Phage">The phage</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Notebook">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Attributions">Attributions &amp; Achievements  </A></LI><LI><A href="https://2012.igem.org/Team:Technion/Human Practices/Articles">Human Practices</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Human Practices/Articles">Articles about us</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Human Practices/High School Lecture">High School lecture</A></LI><LI><A href="https://2012.igem.org/Team:Technion/Sponsors">Sponsors &amp; Thanks</A></LI>

Most of our modelling efforts were done for the riboswitch system. The modelling helped us in the debugging process and in some planing stages.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Back_of_the_envelope_calculations"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Back of the envelope calculations</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Secondary_structures"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Secondary structures</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#RBS_strength_calculations"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">RBS strength calculations</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Conclusion"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Back_of_the_envelope_calculations">Back of the envelope calculations</SPAN></H2><P>In order to assess the ration of theophylline molecules to riboswitch  containing RNA molecules, some &quot;back of the envelope&quot; calculations  were done. These calculations were under the following assumptions:</P><UL><LI>1 nm of theophylline = 1  molecule per cell =&gt; 1mM = 106 molecules per cell. This  assumption is based on the BioNumbers database {1,  2}.</LI><LI>The RNA half life in a cell  is approximately 5 minutes {2}.  Therefore, the degradation rate is ~0.14(1/min).</LI><LI>The RNA synthesis rate is  ~1(molecule/min) considering the mRNA length for the RS+mCherry construct.</LI></UL><P dir="ltr">Assuming a steady state, the amount of RNA molecules in a cell would be  the synthesis rate divided by the degradation rate. This would produce ~7 RNA  molecules per DNA template.
  Next, the number of DNA templates was approximated using <A href="http://endmemo.com/bio/dnacopynum.php" class="external text" rel="nofollow">this calculator</A>. The amount of DNA from pSB1C3 minipreps was ~5000ng for a  plasmid+insert size of 2888bp. The amount of DNA from pSB3C5 minipreps was  ~12500ng for a plasmid+insert size of 3556bp. Using the calculator, the copy  number of the plasmids was ~160 and ~325 for pSB1C3 and pSB3C5 respectively.
  Assuming that not all the copies of the plasmid are transcribed  simultaneously (therefore dividing the copy number by 3), the number of RNA  molecules per cell was 366 and 745 for pSB1C3 and pSB3C5 respectively.  Therefore, the ratio between the theophylline molecules and the mRNA molecules  at 1mM theophylline is ~2730 and ~1340 for pSB1C3 and pSB3C5 respectively.
  This means that there are a lot of theophylline molecules in the cells  to interact with the riboswitch. The level of mRNA molecules that are actually  translated would depend on the riboswitch-theophylline Kd.</P><H2><SPAN class="mw-headline" id="Secondary_structures">Secondary structures</SPAN></H2><P>In order to understand the behavior of our RS+mCherry construct, we  examined the secondary structure of our system and compared it to systems that  have been shown to work with the riboswitch. The secondary structures were  calculated using <A href="http://www.nupack.org/home/references" class="external text" rel="nofollow">NUPACK</A>{3}.
The comparison between different secondary structures is presented in <STRONG><EM>Figure  1</EM></STRONG>.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><STRONG><EM>Figure 1: </EM></STRONG><EM>a comparison between secondary structures of  theophylline riboswitch fused to different proteins. A and B show structures  that have been shown to work with luciferase and IS10+lacZ respectively{4}. C shows the structure for the riboswitch  and mCherry. D shows the structure for the positive control, RBS+mCherry.</EM></DIV></DIV></DIV></DIV><P>It can be seen that there are only minor differences in the secondary  structure as predicted by the model. However, this does not mean that there  isn't some other effect of the mRNA folding on the riboswitch function.  Moreover, for the positive control (D) it can be seen that all the interactions  are at low probability. Therefore, this RBS should be free for interaction with  the ribosome most of the time.</P><H2><SPAN class="mw-headline" id="RBS_strength_calculations">RBS strength calculations</SPAN></H2><P>To further investigate the behavior of the RS+mCherry construct and its  positive control (RBS+mCherry) we submitted the mRNA sequences to the <A href="https://salis.psu.edu/software/" class="external text" rel="nofollow">Salis lab RBS calculator</A>. <STRONG><I>Table 1</I></STRONG> summarizes the translation initiation rates at the expected  start codon for each construct.</P><DIV align="center" dir="ltr"><EM><STRONG>Table 1:</STRONG> translation  initiation rates for different proteins fused to the theophylline riboswitch</EM><TABLE id="table1"><TBODY><TR><TD>Construct </TD><TD><P dir="ltr">Translation initiation    rate</P></TD></TR><TR><TD><P dir="ltr">RS+luciferase</P></TD><TD><P dir="ltr">319</P></TD></TR><TR><TD><P dir="ltr">RS+IS10+lacZ</P></TD><TD><P dir="ltr">139</P></TD></TR><TR><TD><P dir="ltr">RS+mCherry</P></TD><TD><P dir="ltr">56.7</P></TD></TR><TR><TD><P dir="ltr">RBS+mCherry</P></TD><TD><P dir="ltr">199</P></TD></TR></TBODY></TABLE></DIV><P dir="ltr">It can be seen that the model gives low translation initiation rates for  all the constructs. For the riboswitch constructs this is due to the secondary  structures and the fact the calculation is done in the absence of theophylline.  For the RBS+mCherry construct it is visible that the translation initiation  rate is ~3.5 higher. Therefore, according to the model, the fluorescence from  this construct should be higher than the one measured for the riboswitch in the  absence of theophylline.  However, this  is in contrast to some of our <A href="https://2012.igem.org/Team:Technion/Project/RS#Second_set_of_experiments" class="external text" rel="nofollow">experimental results</A>.</P><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>We tried to use computational analysis tools such as <A href="http://www.nupack.org/home/references" class="external text" rel="nofollow">NUPACK</A> and the <A href="https://salis.psu.edu/software/" class="external text" rel="nofollow">Salis lab RBS calculator</A> to explain and predict our experimental results.  However, we saw that these tools did not provide us with a clear explanation.  Therefore, we conclude that these tools should be used with caution and are  good for general estimations.</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI><STRONG>Suess  B., et al.</STRONG> 2004. A theophylline responsive riboswitch based on helix  slipping controls gene expression in vivo. Nucleic Acids Research <STRONG>32</STRONG>(4): 1610-1614.</LI><LI><STRONG>Milo  R., et al.</STRONG> 2009. BioNumbers--the database of key numbers in molecular  and cell biology. Nucleic Acids Research <STRONG>38</STRONG>(Database):  D750-D753.</LI><LI><STRONG>Zadeh  J. N., et al.</STRONG> 2011. NUPACK: Analysis and design of nucleic acid systems.  Journal of Computational Chemistry <STRONG>32</STRONG>(1):  170-173.</LI><LI><STRONG>Lynch  S. A., Gallivan J. P.</STRONG> 2009. A flow cytometry-based screen for synthetic  riboswitches. Nucleic Acids Research <STRONG>37</STRONG>(1):  184-192.</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Technion/Modeling">http://2012.igem.org/Team:Technion/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Technion/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Technion/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Technion/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Technion/Modeling&amp;oldid=219982" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>