"
Page
Discussion
View source
History
teams
Log in
 
Team:WHU-China/Footages/notebook
From 2014.igem.org
Notebook
Methods
Function Test of E7 Lysis system
The construction of E7-pUHE21 contained strain w3110
BBa_K117000 which contains E7 lysis gene is found out from iGEM 2014 registry and primers are design to PCR the target gene(the sequence of the primers are shown below) BBa_B0034 is contained in the upstream primer, which is a strong RBS.The plasmid pUHE21 is a commercialized one that can be extracted from a modified strain of E.coli(part of its sequence is shown below). EcoRI and PstI are used to cut both the PCR product and the pUHE21 backbone. After purification and ligation with T4 ligase, the reconstructed E7-pUHE21 plasmid is transformed into competent E.coli strain w3110. The positive clone with right DNA sequence inserted is screened out and referred to as strain E7
Sequence of upstream primer with strong RBS (BBa_B0034): CGGAATTCTTCTAGAGAAAGAGGAGAAATACTAGATGAAAAAAATAACAGGGATTAT
Sequence of downstream primer: AACTGCAGGCGGCCGCTACTAGTATTACTGCGTTTCCACTCCG
The construction of pUHE21 contained strain w3110
In order to make a negative control of this experiment, pUHE21 is transformed into competent E.coli strain w3110 and the positive clone is screened out, referred to as strain BB(short for backbone).
IPTG induction and OD measurement
Inoculate 3 Erlenmeyer flasks (referred to as E7-0, E7-1 and E7-2 separately) containing 100ml LB medium with 1ml of saturated overnight strain E7 culture, and at the same time inoculate 2 Erlenmeyer flasks (referred to as BB-0 and BB-1 separately) containing 100ml LB medium with 1ml of saturated overnight strain BB culture. All of the broth are cultured to OD=0.5, then IPTG is added to E7-1, E7-2 and BB-1 to make their IPTG concentration reach 1.0mM, 2.0mM and 1.0mM, respectively. After induction those 5 Erlenmeyer flasks are cultured for another 8 hours, during which 3 samples are taken into a 96-well plates from each flask every hours and their OD are measured with microplate reader.
Western blot to further prove the lysis effect of E7
Besides OD measurement, we also do Western blot. Taken from each Erlenmeyer flasks at 0, 1, 2, 4, 6 hours after induction, those samples are centrifuged ,the supernatant and sediment are used to run a SDS-PAGE, then the antibody of DnaK(which is an endogenous protein expressed constitutively in E.coli) is used to conduct a western blot.
Function Test of The Enzymes
To test whether formaldehyde dehydrogenase (PADH) work or not, we developed this system. In this system, when PADH oxidize formaldehyde, electrons lost by formaldehyde are transferred to nicotinamide-adenine dinucleotide (NAD+). Gaining electrons, the NAD+ are reduced to reduced form of nicotinamide-adenine dinucleotide (NADH). By measuring the absorbance of NADH under light of 340nm, we can know the concentration of NADH, and finally know the activity of PADH.
Culture
Experimental group (E.coli with PADH gene) and control group (E.coli without PADH gene) are both cultured under 37&#8451, overnight
Lysis
Collect cells by centrifugation, and wash them with Tris-HCl (PH 7.5) twice
Resuspend cells with Tris-HCl (PH 7.5)
Lyse them by ultrasonic cell disrupter: 500w, work 5s, interval 7s, repeat dozens of times until the solute becomes clear.
Collect supernatant by cryogenic centrifugal (4℃, 12 000 r/min , 20min)
Enzyme activity assay
Prepare the reaction system in colorimetric tube: 1.5ml Tris-HCl (PH 8.0), 0.5ml NAD+ (6mM), 0.5ml formaldehyde solution (12mM), 2.5ml total
Incubate under 50℃ for 10min, measure the absorbance from time to time until stabilized
Add 0.5ml supernatant (experimental group) or Tris-HCl buffer (control group) to the reaction system, and continue incubating
Measure the absorbance under 340nm wavelength of light, after 5min, 10min, 15min, 20min respectively
Timeline
2013
Nov. - Dec., Team building
2014
Jan. - Feb., Knowledge preparing. Looked into iGEM and previous teams’ wiki.
Mar    Brainstorming
1st,   Put forward our idea to reduce formaldehyde
8th,   the idea was made our project purpose. Decided to start with acid promoter
15th,   acid promoter plan failed. Start to focus on formaldehyde promoter.
19th,   first campus weekly thesis defense.
Apr.,  Idea building.
10th,    last Campus Thesis Defense.
20th, internal adjustment. Established the group framework. Group members increased to 30.
25th,    with primers, managed to generate qualified promoters
May,    Instructor assessment and suggestions.
3rd,    lab cleaning. “Assembly Line” mode was introduced. Timing of preliminary stage.
4th,    the result of the identification of the single colony which was positive.
Jun. ~ Oct.,  Experiment and Human Practice
Jun. 30th,  Group meeting. T-easy plasmid was introduced.
Aug. 15th,  Group meeting.
Aug. 18th,  Group members set off to HZAU for investigation and study.
Aug. 20th,  Debug of Human Practice. Team networking.
Aug. 26th,  Attended CCiC held in Huazhong Agricultural University.
Sept. 4th,      Decided group Logo. First sketch of our poster.
Sept. 6th,      Got first result to prove formaldehyde promoter functional.
Sept. 10th,  Mathematical Modeling preparation.
Sept. 20th,     PowerPoint materials preparation.
Sept. 28th,  Biobrick shipping.
Sept. 30th,  Another biobrick shipping.
Oct. 1st,      Assisted HUST-Innovators in constructing a biobrick
Oct. 4th,      Prepare for mathematical modeling and establishment of website
Multi-Assembly-Line Journal:
Promoter
Jun. ~ Sept.,  Pre-experiment for the construction and function test of hxlR operon.
Aug. 20th,  hxlR-GFP completed.
Sept.,   Got a passable result with a small amount of leakage.
Lux Systerm
Sept. 26th,  Construction complete.
Oct. 1st,   Function test showed a positive result
.
FDH, PADH
Jul. ~ Aug.,  Enzyme DNA extracting and processing.
E7
Aug. 20th,  Lysing function test. Western blot processing and OD600 observing.
Retrieved from " http://2014.igem.org/Team:WHU-China/Footages/notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
