"
Page
Discussion
View source
History
teams
Log in
 
Team:WHU-China/Footages/results
From 2014.igem.org
Results
1 Coloration system
We have successfully amplified the P-formaldehyde gene (595bp) from Bacillus subtilis 168 using different annealing temperature(60℃, 58℃, 56℃ corresponding to products 1, 2, 3 respectively).
[1]
Figure 1 The PCR product of formaldehyde promoter.
Then we ligate P-formaldehyde with the pSB1C3 backbone using standard assembly to make BBa_K1334002 .
Figure 2 Formaldehyde promoter in the pSB1C3 backbone (BBa_K1334002).
In order to prove the function of P-formaldehyde in the system described above, we add a GFP reporter gene to the downstream of it. Thus by detecting relative fluorescence intensity we get to know whether the P-formaldehyde works.
[1][2][3]
Figure 3 A GFP reporter is added downstream of the formaldehyde promoter
We transform P-formaldehyde+GFP plasmid into E.coli strain DH5α. The positive clone is screened out and used as experimental group. After cultured to OD600 reaching 1.0 (mid-log phase of E.coli growth curve), formaldehyde is added to broth to 1mM. bacteria samples are collected at different time and their fluorescence intensity and absorbance are measured by microplate reader.
From the following two figures, we can see significant increase in relative fluorescence intensity after formaldehyde stimulation, indicating that our coloration system works well. In addition,the growth condition curve shows that the difference of the fluorescence/absorbance is not caused by the amount of bacteria.
Figure 4 Formaldehyde promoter function chart.
Figure 5 The growth condition of E.coli strain DH5a after formaldehyde stimulation.
2 Street Cleaner System
We construct biobricks: Part: BBa_K1334016, BBa_K1334015 and BBa_K1334017. And we construct a plasmid as following:
Figure 6 The Formate dehydrogenase generator.
Figure 7 The Formaldehyde dehydrogenase generator.
And we transform the plasmids into the expression strain of E.coli. we incubate the E.coli and use the bacteria lysate to run a SDS-PAGE. The FDH from E.coli is about 80kD while we can see a 25kD fragment and the protein PADH is 42.5kD while there is a fragment between 15 and 25kD region. So we need to modify our biobrick to express the two enzymes.
[4][5][6]
Figure 8 SDS-PAGE for the FDH generator.
Figure 9 SDS-PAGE for the PADH generator.
3 lysis system
(1) Lux system
As we have mentioned in our design, we construct a biobrick to test whether the lux system works or not.
Figure 10 A signal complex to test whether the lux system works ( BBa_K1334020 ).
We transform this plasmid into the E.coli strain DH5a. After exogenous AHL (3OC6HSL ,final concentration:100nM)stimulation, we detect the relative fluorescence intensity of RFP. The following figures shows that the lux system works well.
[7]
Figure 11 The function test of our part BBa_K1334020 in which the relative fluorescence reflects that the device works well.
Figure 12 A picture which shows the color change of the bacteria after AHL (3OC6HSL) stimulation.
(2) E7 protein
We ligate e7 gene with a commercialized plasmid backbone called pUHE21, which expresses E7 protein when there is IPTG induction.
W3110 is a wild type strain of E.coli that can secret colicin. According to some research, E7 lysis protein can cause the lysis of w3110 strain membrane so that they can release colicin to the extracellular space to kill other E.coli strain (w3110 itself can resist colicin because of the expression of immunoprotein).
[8][9]
E.coli strain W3110 with or without E7 lysis gene integrated in plasmid pUHE21 are cultured to OD=0.5 and then induced with or without IPTG. We collected bacteria samples every hour from the fermented liquid and the Optical Density are measured by microplate reader. The results are shown below.
Figure 13   The growth condition after IPTG inducing E7 expression which shows E7 does lyse the cells.
Figure 14   The growth condition of the blank after IPTG stimulation which shows IPTG stimulation does not influence OD600 a lot.
From Figure 13 we get to know that the expression of E7 lysis protein can induce the lysis of colicin-secreted E.coli. Although at the beginning of the induction, the OD of those 3 groups are slightly different with each other because of accidental error, after 2 hours of induction, the differences in OD are too obvious to be considered as the consequence of the error, and after 8 hours induction, the difference even reaches as great as 1 fold.
Besides, Figure 14 can prove that IPTG is not the main cause of the decrease growth rate of E.coli. we know that IPTG itself may do some harm to E.coli, on the other hand, the induction with IPTG greatly increases the expression level of genes downstream pLac ,which may bring additional burden to E.coli as well. So in order to make sure the difference showed in Figure 14 is mainly caused by the lysing function of E7 lysis protein other than IPTG, w3110 without E7 lysis gene on its plasmid (pUHE21, also referred to as backbone) is induced with same concentration of IPTG. The result shows that IPTG can affect the growth rate of E.coli to some degree, but its influence is not strong enough to cause 1 fold difference. Thus we can say that E7 lysis protein is functional to cause the lysing of E.coli and decrease its growth rate.
We also use the collected bacteria and the supernatant to do western-blot,using anti-DnaK antibody. (DnaK, or Heat Shock Protein 70 is coded by a house-keeping gene and it cannot go through the cell membrane freely.) We find DnaK in the supernatant largely increases when E7 expresses, which means that the E7 protein do lyse the bacteria.
Figure 15  Western-blot using anti-DnaK. The minus symbol means without IPTG induction. In the stimulation groups, the DnaK level increase in the supernatant shows that the E7 lyse the cell continuously.
Reference
[1] Yurimoto H, Hirai R, Matsuno N, et al. HxlR, a member of the DUF24 protein family, is a DNA-binding protein that acts as a positive regulator of the formaldehyde-inducible hxlAB operon in Bacillus subtilis[J]. Molecular microbiology, 2005, 57(2): 511-519.
[2] Huyen N T T, Eiamphungporn W, Mäder U, et al. Genome-wide responses to carbonyl electrophiles in Bacillus subtilis: control of the thiol-dependent formaldehyde dehydrogenase AdhA and cysteine proteinase YraA by the MerR-family regulator YraB (AdhR)[J]. Molecular microbiology, 2009, 71(4): 876-894.
[3] Yasueda H, Kawahara Y, Sugimoto S. Bacillus subtilis yckG andyckF Encode Two Key Enzymes of the Ribulose Monophosphate Pathway Used by Methylotrophs, and yckH Is Required for Their Expression[J]. Journal of bacteriology, 1999, 181(23): 7154-7160.
[4] Sheng B, Zheng Z, Lv M, et al. Efficient Production of (R)-2-Hydroxy-4-Phenylbputyric Acid by Using a Coupled Reconstructed d-Lactate Dehydrogenase and Formate Dehydrogenase System[J]. PloS one, 2014, 9(8): e104204.
[5] Khangulov S V, Gladyshev V N, Dismukes G C, et al. Selenium-containing formate dehydrogenase H from Escherichia coli: a molybdopterin enzyme that catalyzes formate oxidation without oxygen transfer[J]. Biochemistry, 1998, 37(10): 3518-3528.
[6] Zinoni F, Birkmann A, Stadtman T C, et al. Nucleotide sequence and expression of the selenocysteine-containing polypeptide of formate dehydrogenase (formate-hydrogen-lyase-linked) from Escherichia coli[J]. Proceedings of the National Academy of Sciences, 1986, 83(13): 4650-4654.
[7] Winson M K, Camara M, Latifi A, et al. Multiple N-acyl-L-homoserine lactone signal molecules regulate production of virulence determinants and secondary metabolites in Pseudomonas aeruginosa[J]. Proceedings of the National Academy of Sciences, 1995, 92(20): 9427-9431.
[8]Lin L J R, Liao C C, Chen Y R, et al. Induction of membrane permeability in Escherichia coli mediated by lysis protein of the ColE7 operon[J]. FEMS microbiology letters, 2009, 298(1): 85-92.
[9] Chak K F, Kuo W S, James R. Cloning and characterization of the ColE7 plasmid[J]. Journal of general microbiology, 1991, 137(1): 91-100.
Retrieved from " http://2014.igem.org/Team:WHU-China/Footages/results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
