Team:SDU-Denmark/Experiments
Menu
Home
Introduction
Team
Attributions
Collaborations
Achievements
Project
Description
Experiments
Demonstration & Results
Proof of Concept
Scientific Reproduction
Parts & Procedures
Parts
Notebook
Safety
Practices & Prospects
Bioethics
Integrated Practices
Outreach
Prospects
Perspectives
Bacto-Aid
Experiments
Overview
Generel Verification
Verification of Stepwise Experiments: pPCR, cPCR and Fast Digest by Gel Electroforesis
Working with Bacteriocins
IMPACT Method
Bacteriocin Extraction
Determining Bacteriocin Concentration and Effect
Performning Minimum Inhibitory Concentration (MIC) Test
Creating Silk Monomers and Hybrid-Silk
Iterative Capped Assembly (ICA) Method
Assembling Silk Gene Fragments
Working with PHB
PHB Production and Extraction
Digestion with Hypochlorite
Solvent Extraction
Evaluation of Extraction Methods
Developing a Secretion System
Work with S. aureus Pantothenate Kinase II
Analysis of E. coli Proteome in panK's Expressing Cells
Determination of PHB Production using Flow Cytometry
Filament Making - and 3D Printing with PHB
Protocols
SOP's
Verification by Gel Electrophoresis ( Top )
To verify our stepwise experiments, pPCR , cPCR , and Fast Digest , we used gel electrophoresis. A successful transformation were confirmed by a colony PCR with specially designed primers for each gene. The results were confirmed according to the theoretical expected length of the gene.
If satisfied with the results, plasmid purification were made from ON cultures from the respective cPCR templates.
Bacteriocins
Bacteriocins are antimicrobial peptide compounds that effect through adhesion to specific receptors. These are present on the external surface of the target bacteria, where they induce metabolic changes by pore formation. The bacteriocins were selected according to the three listed criteria:
They elicited an effect on pathogens, most frequent detected in open wounds, causing wound infection.
They targeted the cell wall of the target bacteria in order to work when fused with silk.
They were non-lethal to E.coli, in order to use E.coli as the producing strain.
Strains that produce bacteriocins, which target the respective bacteria present in open wounds, were examined. Genes encoding the bacteriocins were found and codon optimized, in order to create a BioBrick part containing only the bacteriocin gene. We also created hybrid bacteriocins in order to test if the hybrid showed similar, inhibited or better effect when combined. The use of bacteriocins as new antimicrobial components are of great interest when used in local treatment. In addition it seems like resistance is rarely developed towards bacteriocins. This idea is supported by professor Frank Møller Arrestrup .
Table 1 shows a brief overview of the chosen bacteriocins we worked with in our experiments. To learn more about the specific bacteriocins click on the part numbers in the table.
Part reg.
Name
Length (bp)
Host Organism
Target Organism
K2018010
Laterosporulin
352
Brevibacillus
Staphylococcus aureus & Pseudonomas aeruginosa
K2018011
ThuricinS
259
Bacillus thuringiensis
Pseudonomas aeruginosa, Enterobacter Cloacae
K2018012
LacticinQ
361
Lactococcus lactis QU5
Staphylococcus aureus
K2018014
Laterusporulin-ThuricinS
415
Brevibacillus, Bacillus thuringiensis
-
K2018015
LacticinQ-LacticinZ
529
Lactococcus lactis QU5, Lactococcus lactis
-
K2018019
PyocinS5
1696
Pseudonomas aeruginosa
Staphylococcus aureus & Pseudonomas aeruginosa
The Purification of Bacteriocins
To purify our bacteriocins, we used the IMPACT method (Intein Mediated Purification with an Affinity Chitin-binding Tag). The IMPACT method has the ability to purify native proteins in a single chromatographic step without the use of proteases. We use the IMPACT vector pTXB1 in order to fuse our bacteriocin gene with an intein tag embedded in the pTXB1 plasmid that allow affinity purification. We add IPTG (Isopropyl β-D-1-thiogalactopyranoside) to induce the expression of our intein-bacteriocin fusion. The intein-bacteriocin protein is then extracted by use of French press that disrupt the bacterial cell membrane. The cell lysates are poured onto prepared chitin beads that allow affinity chitin-binding of the intein-bacteriocin fusion. By use of a cleavage buffer containing a thiol-reagent, we induce cleavage between the intein tag and the bacteriocin protein. This results in native bacteriocin elutes.
IMPACT Method ( Top )
X
Figure 1 illustrates the removal of the hybrid bacteriocin Laterosporulin-ThuricinS from the pSB1C3 vector into the pTXB1 vector. This is schematically illustrated via SnapGene History.
Figure 1 illustrates the removal of the hybrid bacteriocin Laterosporulin-ThuricinS from the pSB1C3 vector into the pTXB1 vector. This is schematically illustrated via SnapGene History.
For the IMPACT procedure, the vector pTXB1 was used. The vector pTXB1 contains a mini-intein (Mxe GyrA intein) that has been modified to undergo thiol-induced cleavage at its N-terminus. At its C-terminus the intein tag contains a chitin binding domain (CBD) that allows affinity purification on a chitin column (IMPACT Kit) . The target protein - in this case our bacteriocins - is fused at its C-terminus to the N-terminus of the intein tag marked by the MCS (Multiple cloning site), shown in Figure 1. With the use of a cleavage buffer that contains dithiothreitol (DTT), the vector allows protein purification of a target protein, by thiol-induced cleavage, at the site between the intein tag and the target protein. The results were obtained through cloning into NdeI and SapI sites, which allow the target proteins to be cleaved without any residual amino acids. In order to clone the bacteriocins into the pTXB1 vector from pSB1C3, specific primers were designed to add compatible restriction sites that included the restriction sites for NdeI and SapI.
Expression of Bacteriocins by the IMPACT Plasmid pTXB1 and the E. coli Strain ER2566
X
Figure 2 illustrates the effect of IPTG induction on DNA level. The effect of IPTG induction on the ER2566 genome are shown on the left and the effect on the pTXB1 are shown on the right. Thus illustrating how the ER2566 genome and pTXB1 act in concert when induced with IPTG.
Figure 2 illustrates the effect of IPTG induction on DNA level. The effect of IPTG induction on the ER2566 genome are shown on the left and the effect on the pTXB1 are shown on the right. Thus illustrating how the ER2566 genome and pTXB1 act in concert when induced with IPTG.
The E. coli strain ER2566 were used for the expression of the target gene cloned into pTXB1. ER2566 carries a copy of the T7 RNA polymerase gene inserted into the LacZ gene. The T7 RNA polymerase is thereby under the control of the lac promoter. The pTXB1 contains a T7 promoter in front of our target gene and is therefore dependent on the presence of T7 RNA polymerase. To induce protein expression in ER2566, IPTG is added. IPTG binds the lacl repressor located on the pTXB1 plasmid and alter its conformation allosterically. The event prevents the repression of the β-galactosidase, coding the LacZ gene in ER2566 and thus induces the transcription of T7 RNA polymerase (Figure 2). By addition of IPTG we allow E. coli to transcribe the T7 RNA polymerase through the LacZ gene, thereby indirectly induces controlled expression of our target gene, through binding of the T7 promoter.
In addition IPTG is not hydrolyzed by β-galactosidase encoded by the LacZ gene as it contains a sulfur atom in its structure, thereby diverges from allolactose, which is the natural inducer of the lac operon and substrate to hydrolysis by β-galactosidase. The diverge structure of IPTG  prevents the cell from degrading the IPTG. The concentration of the inducer remains constant in the experiment, which allow the expression of lac-controlled genes not to be inhibited Juers, D. H., Matthews, B. W., & Huber, R. E. (2012). LacZ β-galactosidase: Structure and function of an enzyme of historical and molecular biological importance. Protein Science : A Publication of the Protein Society, 21(12), 1792–1807. .
Bacteriocin Extraction ( Top )
X
Figure 3 shows that we extract our bacteriocins from ER2566 by using French press followed by thiol induced cleavage in order to purify the bacteriocins.
Figure 3 shows that we extract our bacteriocins from ER2566 by using French press followed by thiol induced cleavage in order to purify the bacteriocins.
To extract our protein-Intein from the ER2566 cells after induction with IPTG, we used a French press to destroy the bacterial cell membrane. The lysate were then poured onto prepared chitin beads. Cleavage buffer containing DTT was added, thus causing thiol-induced cleavage at the fusion between the intein and our target protein (Figure 3).
Determining Bacteriocin Concentration and Effect ( Top )
X
Figure 4 shows a graph of Bradford Standard Protein assay performed with BSA. The concentration [µg/mL] of BSA are plotted on the x-axis and measured OD values are plotted on the y-axis. The equation has been made by linear regression of the plotted data.
Figure 4 shows a graph of Bradford Standard Protein assay performed with BSA. The concentration [µg/mL] of BSA are plotted on the x-axis and measured OD values are plotted on the y-axis. The equation has been made by linear regression of the plotted data.
The concentration of our bacteriocins were determined according to a Bradford Standard protein assay (Figure 4). We created a Bradford protein standard curve of which the protein concentration were calculated according to the measured absorbance. To view final protein concentrations go to Demonstration & Results .
Minimum Inhibitory Concentration (MIC) Test ( Top )
MIC tests are essential in order to establish the bacteriocins viability as an antimicrobial compound. The MIC test studies the in vitro susceptibility of a bacterial strain towards a compound with an antimicrobial effect. This is measured by the optical density of an overnight culture after 28 hours incubated at 37°C, with a specific concentration of a bacteriostatic and/or bactericidal compound. We wanted to perform a MIC test towards P. aeruginosa and MRSA strains. This was inspired from Hans Jørn Kolmos (professor in clinical microbiology), who suggests that bacteriocins could be potent to treat MRSA carriers. In collaboration with the iGEM Stockholm team 2016 , MIC tests were performed in order to compare the effect of our bacteriocins. The iGEM Stockholm team 2016 performed a MIC test on cell lysates, whereas we performed a MIC on our purified proteins. By collaborating with the iGEM Stockholm team 2016 we thus got an indication of which of our bacteriocins elicited a significant effect towards chosen target strains. The MIC plates were incubated for 28 hours. To see the results of MIC test, go to Demonstration & Results .
Summary
Design of primers compatible to bacteriocin/pSB1C3 in order to add restriction sites for Nde1 and Sap1
↓
Phusion PCR with designed Forward and Reverse primers for the respective devices (K20180XX/pSB1C3)
↓
Fast digest of pPCR products + vector pTXB1 with Nde1 and Sap1
↓
Ligation with Fast Digest products + vector pTXB1
↓
Transformation into E. coli TOP10
↓
Colony PCR with Impact_forward and Impact_reverse primers  
↓
Plasmid purification of ON cultures from cPCR templates
↓
Transformation of purified bacteriocin/pTXB1 plasmid into E. coli ER2566
↓
ON of cultures → 1 L culture
↓
IPTG protein induce
↓
French Press
↓
Chitin beads and cleavage buffer containing DTT
↓
Concentration determination through Bradford Protein Standard Assay (Verification)
↓
MIC test of the purified bacteriocins effect on different strains
Creating Silk Monomers and Hybrid-Silk by Iterative Capped Assembly (ICA)
The ICA method ( Top )
The ICA method was described as a solid surface-based sequential ligation, which adds DNA constructs with matching overhangs individually in series Briggs, A. W., et al. (2012). "Iterative capped assembly: rapid and scalable synthesis of repeat-module DNA such as TAL effectors from individual monomers." Nucleic Acids Res 40(15) . This makes it possible to custom design genes, which can later be amplified through PCR. The method is especially effective when making repetitive DNA constructs. Traditionally DNA is added to a gene through primer overhangs in a PCR, but when working with repetitive genes, the primer could recognize multiple sites and thereby cause multiple variations of the gene. The stepwise ligation method utilizes the strong chemical binding between streptavidin and biotin. The concept is to use magnetic beads coated in streptavidin, while the initiator oligo contains a biotinylated end at the 5’. This allows the magnetic beads to be continuously isolated from the solution with a magnetic rack and the supernatant can be removed. This controls a stepwise ligation construct that ends with a terminator sequence with one sticky-end that eliminates future ligations. Correct ligations will be controlled by the use of cap-sequences, which bind to the previous step's non-ligated constructs and end the possibility of further ligations at the failed ligations. In the end, the gene construct will consist of an initiator sequence, a coding region and a terminator. The initiator and terminator have specific primer sites, which makes it possible for pPCR of the whole synthesized gene sequence. The ICA method was used by the iGEM UCLA team 2015 to synthesize silk. During our experiment the used initiator contained a standard iGEM prefix and the terminator contained a standard iGEM suffix. These sites made it possible for the gene sequence to be inserted into desired vectors.
Assembling Silk Gene Fragments ( Top )
Silk from Nephila clavipes consists of the genes MaSp1 and MaSp2, which individually contains repetitive gene sequences. The genes have the same restriction sites in both ends so that the overhangs are consistent throughout the construct. This step by step ligation of the individual genes (e.g. MaSp1 AB + MaSp1 BC + MaSp1 CA), allows for custom designs of the silk gene both in length and in variation (e.g. 2xMaSp1:1xMaSp2). For a graphical illustration of the ICA Method, check out the video above.
The ICA method gives way for alternative constructs, where other genes can be incorporated in between the repetitive silk genes. This freedom of construction creates the opportunity for incorporation of bacteriocin genes into the silk construct, using the ICA method. Coloplast are already aware of the great qualities of recombinant spider silk and bacteriocins, however the idea of combining the two elements are completely new to them and they see a great oppertunity in it.
Summary
pPCR of the ordered geneblocks of MaSp1 and MaSp2 (AB/BC/CA)
↓
FastDigest of pPCR products with Eco31I  
↓
ICA of the digested products
↓
PCR of the ICA-product
////
We tried our best but we did not succeed in creating a silk construct.
Working with PHB
In the lab we wanted to start out by 1) creating a library of promoter hybrids to determine an optimal construct for the production of PHB, 2) determine the ideal purifaction method, 3) increase the production of PHB by using panK and characterize the effect of panK on the proteome, 4) have the cells secrete PHB to ease purification efforts, and 5) 3D print the recovered PHB.
PHB Production and Extraction ( Top )
Retrieving a pure sample of PHB from the cellular cytoplasm has proven to be an expensive task. We have tested four methods of extraction in order to determine, which extraction method is optimal for large scale production of PHB, considering pricing, environmental strain, required equipment, toxicity, yield and purity.
Digestion with Hypochlorite ( Top )
A method for extracting biodegradable plastic, is simply digesting the cellular components with hypochlorite , where it dissolves the cellular components and leaves the polyester behind. The advantages of using hypochlorite as opposed to digestion with surfactants (e.g. sodium dodecyl sulfate (SDS)), is that it dissolves the membrane. Thus a single treatment can provide high purity and yield of PHB. After treatment with PHB the samples were centrifuged with 4000xG. A small side effect of hypochlorite is that it causes degradation of the PHB granules that greatly reduce the molecular weight of the molecule and thereby losing some PHB Jacquel, N., et al. (2008). "Isolation and purification of bacterial poly(3-hydroxyalkanoates)." Biochemical Engineering Journal 39(1): 15-27. , Heinrich, D., et al. (2012). "Large scale extraction of poly(3-hydroxybutyrate) from Ralstonia eutropha H16 using sodium hypochlorite." AMB Express 2(1): 59 .
Solvent Extraction ( Top )
Another method of  PHB extraction is treating lyophilized biomass with a solvent in which PHB is soluble. Removing the remaining biomass by filtration and then precipitating the PHB will provide a sample with high purity. We have tested two different methods of solvent extraction with chlorinated agents.
Chloroform is the most toxic agent we have used for extraction, but it provides up to 99 % pure PHB Ramsay, J. A., et al. (1994). "Extraction of poly-3-hydroxybutyrate using chlorinated solvents." Biotechnology Techniques 8(8): 589-594. .
The Ethyl acetate extration method involves the least toxic substances and requires no pretreatment, but fairly large amounts of chemicals are needed for the extraction, which provides a purity of 97-99 % Aramvash, A., et al. (2015). "An Environmentally Friendly and Efficient Method for Extraction of PHB Biopolymer with Non-Halogenated Solvents." J Microbiol Biotechnol 25(11): 1936-1943. .
Evaluation of Extraction Methods ( Top )
We use Proton Nuclear Magnetic Resonance (HNMR) in order to determine the purity of PHB extracted with the different methods. HNMR is a method in which the structure of a molecule is analyzed in a magnetic spectrum. The spin of different hydrogen atoms allows us to determine the amount of and study each distinct type of hydrogen atoms. By analyzing such a spectrum, the content of the sample can be determined.  For details on how we performed HMNR analysis, see protocol HNMR - Measurement of plast purity .
Developing a Secretion System ( Top )
X
Figure 5 shows an illustration of our modified type I secretion system. The figure illustrates how PHB are secreted due to the recognition of HlyA.
Figure 5 shows an illustration of our modified type I secretion system. The figure illustrates how PHB are secreted due to the recognition of HlyA.
In order to avoid the high costs of plastic extraction we are utilizing a type I secretion pathway for the toxin, hemolysin, to make the bacteria secrete the plastic itself. The secretion system requires only two transport proteins, hemolysin B and hemolysin D. Together these proteins are responsible for the secretion of hemolysin in pathogenic strains of E. coli. Since PHB is not a protein compound, applying the system directly, is not possible. Instead we bio-fused a domain of the toxin hemolysin A with a protein called phasin. Phasin is a protein used in R. eutropha that binds to PHB granules in order to regulate their size. Hemolysin A is the exotoxin secreted by hemolysin B and hemolysin D. By using only a short signal domain of the protein, we can tag the bio-fused protein for secretion without making it toxic. The fused phasin-hemolysin protein will keep its ability to attach to the surface of PHB granulates, through non-covalent interaction and thereby target the PHB for secretion RAHMAN, A., LINTON, E., HATCH, A. D., SIMS, R. C. & MILLER, C. D. 2013. Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach. Journal of Biological Engineering, 7. . The event will reduce the use of toxic and environmentally straining chemicals and eventually it could lead to a continuous system. In addition phasin has low binding specificity, thus the system can be used for other PHA’s (polyhydroxyalkanoates) Hanisch, J., et al. (2006). "The Ralstonia eutropha H16 phasin PhaP1 is targeted to intracellular triacylglycerol inclusions in Rhodococcus opacus PD630 and Mycobacterium smegmatis mc2155, and provides an anchor to target other proteins." Microbiology 152(Pt 11): 3271-3280. . The idea of a secretion system has already been explored by the iGEM Utah Team 2009 and have been processed in earlier studies from which we were inspired to develop our secretion system RAHMAN, A., LINTON, E., HATCH, A. D., SIMS, R. C. & MILLER, C. D. 2013. Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach. Journal of Biological Engineering, 7. . Plastic companies are very interested in use of PHB as the material in their products, but are not doing it currently because of the price of the PHB production. Using our secretion system it would be possible optimize the production of PHB, making it more attractive for the respective companies. For further details check out the assembly of the secretion system .
Pantothenate Kinase II (panK) ( Top )
PanK is a pantothenate kinase that activates the first protein in the anabolic pathway for coenzyme A. E. coli has this protein as part of its core genome, but the kinase from E. coli is strongly regulated by a negative feedback mechanism.  However, an increase in coenzyme A and its thioesters will inhibit the activity of pantothenate kinase. Staphylococcus aureus requires a higher amount of coenzyme A as it lacks glutathione and thus relies on a coenzyme A based reductase system to repair oxidative damage Leonardi, R., et al. (2005). "A pantothenate kinase from Staphylococcus aureus refractory to feedback regulation by coenzyme A." J Biol Chem 280(5): 3314-3322. . As a results the staphylococcal panK is not feedback regulated, and we have thereby taken advantage of this by adding the specific panK to our construct. The BioBrick ( K1692020 ) containing staphylococcal panK was created by the iGEM Standford Brown team 2015 .
Analysis of E.coli Proteome in panK's Expressing Cells ( Top )
X
Figure 6 shows a short overview of the steps involved in iTRAQ analysis. This includes protein extraction, digestion, labeling, Titanium dioxid purification, HILIC fractionation and Mass spec analysis. The figure is derived from a protocol created by the Protein Research Group at the Southern University Denmark.
Figure 6 shows a short overview of the steps involved in iTRAQ analysis. This includes protein extraction, digestion, labeling, Titanium dioxid purification, HILIC fractionation and Mass spec analysis. The figure is derived from a protocol created by the Protein Research Group at the Southern University Denmark.
A protein from one organism do not necessarily recognize all its native target proteins in another organism. Additionally, it could be able to phosphorylate proteins, which it did not phosphorylate before. We have therefore performed an iTRAQ analysis of staphylococcal panK (pantothenate kinase II) in E. coli. For an iTRAQ analysis , the entire proteome of the cells are extracted and digested. The peptides from each sample are then tagged with one of four different tags that binds to the free N-terminal present on tryptic peptides. The peptides are then mixed in an equal ratio, securing that the same amount of peptides are present from each sample. The phosphorylated peptides that are especially interesting for the analysis of a kinase, are isolated and purified by a TiO2 purification. Lastly the non-modified peptides and the phosphopeptides are analysed separately by mass spectrometry (Figure 6).
In the mass spectrometer the different iTRAQ tags will fragmentize differently. Therefore while analysing a single spectrum, one can compare the amounts of the proteins from the different samples, simply by comparing the intensity of the reporter ions from the iTRAQ tag. The computer can thus analyse the entire proteome of four bacterial strains at once, identifying interesting differences caused by pantothenate kinase II.
Determination of PHB Production using Flow Cytometry ( Top )
In order to determine the effectiveness of our different PHB producing strains we used nile red staining combined with flow cytometry . Nile red binds to PHB, and as the substance is fluorescent, it can be used as a quantitative measure for the amount of plastic present in a cell Leonardi, R., et al. (2005). "A pantothenate kinase from Staphylococcus aureus refractory to feedback regulation by coenzyme A." J Biol Chem 280(5): 3314-3322. . By combining nile red staining with flow cytometry, the average amount of plastic inside every cell from different strains can be compared and thus generate a relative measure ability of the respective strains to produce plastic.
Filament making - and 3D Printing with PHB ( Top )
X
Figure 7 shows the FilaStruder that made our filaments.
Figure 7 shows the FilaStruder that made our filaments.
X
Figure 8 shows the 3D printer Hyrel System 30M, which we used to create a piece of a jaw implant from our extracted PHB.
Figure 8 shows the 3D printer Hyrel System 30M, which we used to create a piece of a jaw implant from our extracted PHB.
The HYREL SYSTEM 30M 3D printer requires filaments with a diameter of 1.75mm. In order to 3D print our produced PHB, we needed to create filaments made by a filament maker . The Filastruder was assembled and tested with PLA prior to use with bacterially produced PHB.
All of our 3D printing was performed on a HYREL SYSTEM 30M . The 3D model was created as an STL file and tested with PLA filaments. PHB was ordered to compare constructs containing 100% pure PHB and constructs containing our purified plastic. Morten Østergaard Andersen suggested that PHB could be used as an implant itself, in cases where e.g. the bone needs a scaffold until it has healed. This further support the idea of trying to 3D print PHB.
Summary
Creating a library of PHB with hybrid promoters
↓
Comparison of PHB producing BioBricks
↓
Extraction of PHB
↓
Comparison of extraction methods
↓
Assembly of secretion system
↓
Addition of pantothenate kinase II BioBrick to the most efficient PHB producing BioBrick
↓
Addition of the secretion system the the most efficient PHB producing BioBrick
↓
Comparison of new BioBricks with previous BioBricks
↓
Characterization of pantothenate kinase with iTRAQ
↓
Determination of bacterial PHB content with gas chromatography
↓
Analysis of purified material with HNMR
↓
Mass production of plastic
↓
3D printing
Protocols
Bacteriocins
Cloning composite part K2018010 into pSB1C3
Cloning composite part K2018011 into pSB1C3
Cloning composite part K2018012 into pSB1C3
Cloning composite part K2018014 into pSB1C3
Cloning composite part K2018015 into pSB1C3
Cloning composite part K2018019 into pSB1C3
IMPACT Purification of K2018010
IMPACT Purification of K2018011
IMPACT Purification of K2018012
IMPACT Purification of K2018014
IMPACT Purification of K2018015
Silk
Cloning ThuricinS with DA silk-overhangs into pSB1C3
Cloning ThuricinS with DE silk-overhangs into pSB1C3
Cloning ThuricinS with EA silk-overhangs into pSB1C3
Cloning ThuricinS with ED silk-overhangs into pSB1C3
Cloning Basic part K1763002 into pSB1C3
Cloning Basic part K1763003 into pSB1C3
Cloning Basic part K1763004 into pSB1C3
Cloning Basic part K1763009 into  pSB1C3
Cloning Basic part K1763010 into pSB1C3
Cloning Basic part K1763011 into pSB1C3
Cloning Basic part K1763012 into pSB1C3
Cloning Basic part K2018043 into pSB1C3
Cloning Basic part K2018044 into pSB1C3
Plastic
Gas Chromatography - Characterizing and optimizing PHB production
HNMR - Measurement of plast purity
Creating a Hybrid promoter PHB library
iTRAQ analysis - Characterizing and optimizing PHB production in E.coli
Creating BioBricks containing the secretion system
Comparing media for growth of plast producing bacterias
Fluorescence activated cell sorting (FACS) - Comparison of cells
Extraction of PHB with different methods
PHB Extraction following 3D printing
Standard Operational Procedures
SOP's
SOP0001_v01 ON culture of E.coli
SOP0005_v02 Freeze Stock
SOP0007_v01 LA plates with antibiotics
SOP0009_v01 TSB Transformation
SOP0010_v01 Phusion PCR
SOP0015_v01 Ligation
SOP0017_v01 FastDigest
SOP0019_v01 MiniPrep
SOP0021_v01 Colony PCR with My Tag
SOP0022_v01 Competent cell freeze stock
SOP0023_v01 Ca2+ Transformation
SOP0024_v01 SDS-page
SOP0025_v01 Impact Purification
SOP0026_v01 Iterative Capped Assembly
SOP0027_v01 MIC test
SOP0028_v01 Bradford Protein Concentration Determination
SOP0029_v01 DISC diffusion assay
SOP0030_v01 2xYT media
SOP0031_v01 TB Media
SOP0032_v01 Ethylacetate PHB recovery
SOP0033_v01 PHB extraction with Triton x-100 and hypochlorite
SOP0034_v01 Chloroform PHB recovery
SOP0035_v01 Fluorescence microscopy
SOP0036_v01 Nile red agar plates
SOP0037_v01 iTRAQ sample preparation
SOP0038_v01 Qubit®Protein Assay Kits
SOP0039_v1 Purification with C8 and C18 columns
SOP0040_v01 iTRAQ labelling
SOP0041_v01 TiO2 purification
SOP0042_v01 Daily use of FACS Aria II
SOP0043_v01 Sorting setup FACS Aria II
SOP0044_v01 Hypochlorite PHB recovery
SOP0045_v01 Gas Chromatography
Manual for Filastruder
Menu
Home
Introduction
Team
Attributions
Collaborations
Achievements
Project
Description
Experiments
Demonstration & Results
Proof of Concept
Scientific Reproduction
Parts & Procedures
Parts
Notebook
Safety
Practices & Prospects
Bioethics
Integrated Practices
Outreach
Prospects
Perspectives
