<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SDU-Denmark_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SDU-Denmark/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="row"><DIV id="wikimenu" class="menu_wrapper hidden-xl hidden-lg col-sm-12 hidden-md" aria-expanded="true"><DIV class="collapsable_menu_control">Menu</DIV><UL id="accordion" class="accordion"><LI class="menu_item" id="1"><A id="one" href="https://2016.igem.org/Team:SDU-Denmark">Home</A></LI><LI class="menu_item" id="2"><A id="two" href="https://2016.igem.org/Team:SDU-Denmark/Introduction">Introduction</A></LI><LI><A id="team" href=" https://2016.igem.org/Team:SDU-Denmark/Team">Team</A></LI><LI><A id="attributions" href="https://2016.igem.org/Team:SDU-Denmark/Attributions"> 
                                Attributions </A></LI><LI><A id="collaborations" href="https://2016.igem.org/Team:SDU-Denmark/Collaborations">Collaborations</A></LI><LI><A id="achievements" href="https://2016.igem.org/Team:SDU-Denmark/Achievements">Achievements</A></LI></UL><LI class="menu_item" id="3"><A id="three" href="https://2016.igem.org/Team:SDU-Denmark/Project">Project</A></LI><LI><A id="description" href="https://2016.igem.org/Team:SDU-Denmark/Description">Description 
				</A></LI><LI><A id="experiments" href="https://2016.igem.org/Team:SDU-Denmark/Experiments">Experiments 
				</A></LI><LI><A id="demonstration" href="https://2016.igem.org/Team:SDU-Denmark/Demonstrate">  
				Demonstration &amp; Results</A></LI><LI><A id="proof" href="https://2016.igem.org/Team:SDU-Denmark/Proof">Proof of Concept</A></LI><LI><A id="reproducibility" href="https://2016.igem.org/Team:SDU-Denmark/Reproducibility">Scientific Reproduction</A></LI><LI class="menu_item" id="4"><A id="four" href="https://2016.igem.org/Team:SDU-Denmark/Parts_procedures">Parts &amp; Procedures</A></LI><LI><A id="parts" href="https://2016.igem.org/Team:SDU-Denmark/Parts">Parts</A></LI><LI><A id="notebook" href="https://2016.igem.org/Team:SDU-Denmark/Notebook">Notebook </A></LI><LI><A id="safety" href="https://2016.igem.org/Team:SDU-Denmark/Safety">Safety </A></LI><LI class="menu_item" id="5"><A id="five" href="https://2016.igem.org/Team:SDU-Denmark/Practices">Practices &amp; Prospects</A></LI><LI><A id="bioethics" href="https://2016.igem.org/Team:SDU-Denmark/Bioethics"> Bioethics</A></LI><LI><A id="integrated" href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices">Integrated Practices </A></LI><LI><A id="outreach" href="https://2016.igem.org/Team:SDU-Denmark/Outreach">Outreach</A></LI><LI><A id="prospects" href="https://2016.igem.org/Team:SDU-Denmark/Prospects">Prospects</A></LI><LI><A id="perspectives" href="https://2016.igem.org/Team:SDU-Denmark/Perspectives">Perspectives</A></LI></DIV></DIV></DIV><DIV id="header" class="container-fluid"><DIV class="row"><DIV class="header col-lg-8 col-lg-push-2 col-md-10"><DIV class="brand pull-left"><A href="https://2016.igem.org/Team:SDU-Denmark"><H1 id="bodyContent">Bacto-Aid</H1></A></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV id="page" class="col-lg-8 col-lg-push-2 col-md-10"><DIV class="row"><DIV class="col-sm-11 col-sm-push-1"><H2>Demonstration &amp; Results</H2></DIV></DIV><DIV class="row"><DIV class="col-sm-11 col-sm-push-1"><DIV id="overview" style="background-color:#dddddd; padding:0px 10px 10px 20px;"><H3 id="top">Overview</H3><H5>Verification of bacteriocins as antimicrobials</H5><UL class="list"><LI><I><A href="#purify">Purification of the bacteriocins by the IMPACT method</A></I></LI><LI><I><A href="#determine">Determination of bacteriocin protein concentration using Bradford Protein Assay</A></I></LI><LI><I><A href="#effect">Growth inhibition of S. aureus MRSA strains and P. aeruginosa detected by MIC</A></I></LI><LI><I><A href="#synergy">Investigation of synergistic effect of hybrid bacteriocins measured by MIC</A></I></LI></UL><H5>Construction of silk fibers</H5><UL class="list"><LI><I><A href="#insert">Insertion of the MaSp1 and MaSp2 gene fragments into pSB1C3</A></I></LI><LI><I><A href="#ligate">Ligation test of silk gene fragments</A></I></LI><LI><I><A href="#prepare">Assembly of silk fragments by the ICA method</A></I></LI><LI><I><A href="#dream">Ligation test of a full lenght silk construct </A></I></LI></UL><H5>Optimization of plastic production using bacteria</H5><UL class="list"><LI><I><A href="#hybrids">Determination of PHB production efficiency</A></I></LI><LI><I><A href="#optimization">Comparisson of PHB extraction methods</A></I></LI><LI><I><A href="#characterization">Quantitative proteome analysis of panK expressing cells </A></I></LI><LI><I><A href="#secretion">The properties of the secretion system </A></I></LI></UL></DIV><H5>Bacteriocin purification (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="demonstrate2" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#close" title="Close" class="close">X</A><P>Figure 1 shows results of a cPCR originating from respective colonies from transformation of K2018011/pTXB1/E.coli Top10 (ThuricinS), K2018012/pTXB1/E.coli Top10 (LacticinQ) and K2018014/pTXB1/E.coli Top10 (Laterusporulin-ThuricinS). From right; Well 1-3 + 5-7 + 13-14= <A href="http://parts.igem.org/Part:BBa_K2018011" target="_blank">K2018011</A>, Well 4+8 = <A href="http://parts.igem.org/Part:BBa_K2018014" target="_blank">K2018014</A>, Well 9-12 = <A href="http://parts.igem.org/Part:BBa_K2018012" target="_blank">K2018012</A>. The expected length of ThuricinS is 304 bp. <A href="https://static.igem.org/mediawiki/2016/e/ea/T--SDU-Denmark--ThermoFisher_Generuler_DNA_ladder.jpeg" target="_blank">GeneRulerTM 100 bp DNA ladder </A> were used <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.thermofisher.com/dk/en/home/brands/thermo-scientific/molecular-biology/thermo-scientific-nucleic-acid-electrophoresis-purification/dna-electrophoresis-thermo-scientific/dna-ladders-thermo-scientific/generuler-dna-ladders.html#" target="_blank">ThermoFisher scientific. </A></SPAN></SPAN>.</P></DIV></DIV><P class="figuretext"><EM>Figure 1 shows the result of a cPCR of positive clones from the transformation of Top10/pTXB1-<A href="http://parts.igem.org/Part:BBa_K2018011" target="_blank">K2018011</A> (ThuricinS lane 1-3 +5-7 + 13-14), Top10/pTXB1-<A href="http://parts.igem.org/Part:BBa_K2018012" target="_blank">K2018012</A> (LacticinQ lane 9-12) and Top10/pTXB1-<A href="http://parts.igem.org/Part:BBa_K2018014" target="_blank">K2018014</A> (Laterusporulin-ThuricinS lane 4+8). The expected length of ThuricinS, LacticinQ and Laterusporulin-ThuricinS is 304 bp, 406 bp, and 459 bp, respectively. The red box marks ThuricinS clones. <A href="https://static.igem.org/mediawiki/2016/e/ea/T--SDU-Denmark--ThermoFisher_Generuler_DNA_ladder.jpeg" target="_blank">GeneRulerTM 100 bp DNA ladder </A> was used <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.thermofisher.com/dk/en/home/brands/thermo-scientific/molecular-biology/thermo-scientific-nucleic-acid-electrophoresis-purification/dna-electrophoresis-thermo-scientific/dna-ladders-thermo-scientific/generuler-dna-ladders.html#" target="_blank">ThermoFisher scientific. </A></SPAN></SPAN>.</EM></P></DIV><P>Successful cloning of the bacteriocins into the IMPACT vector pTXB1 was a key step in purifying the bacteriocins. <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#IMPACT" target="_blank">Cloning into the IMPACT vector</A> pTXB1 was validated by gel electrophoresis (Figure 1). A successful cloning was verified from the results of a cPCR performed with specifically designed primers according to the theoretical expected length (304 bp) of <A href="https://static.igem.org/mediawiki/2016/0/03/T--SDU-Denmark--IMPACTPurificationofK2018011.pdf" target="_blank">ThuricinS with IMPACT overhangs</A>. The clones were verified by sequencing. The results in the red box (Figure 1) show bands of similar lengths, thus showing a successful ligation of ThuricinS into pTXB1.</P></DIV><H5>Determination of bacteriocin concentration (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><P class="figuretext">Table 1 shows the respective concentrations of the purified bacteriocins. The concentrations are calculated according to the equation  y = 0.0011x + 0.0202 derived from linear regression of the <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#bactoconc" target="_blank">Bradford Standard Protein Assay</A>. The protein concentration (x) is calculated as x = (y - 0.0202)/0.0011. </P><TABLE class="bacto"><TBODY><TR><TH>BACTERIOCINS</TH><TH>OD  (595 nm)</TH><TH>µg/mL</TH></TR><TR><TD>IMPACT ELUATE (Laterosporulin-ThuricinS)</TD><TD>0.078</TD><TD>52.55</TD></TR><TR><TD>IMPACT ELUATE (LacticinQ)</TD><TD>0.069</TD><TD>44.36</TD></TR><TR><TD>IMPACT ELUATE (Laterosporulin)</TD><TD>0.094</TD><TD>67.09</TD></TR><TR><TD>IMPACT ELUATE (ThuricinS)</TD><TD>0.057</TD><TD>33.45</TD></TR><TR><TD>IMPACT ELUATE (LacticinQ-LacticinZ)</TD><TD>0.054</TD><TD>30.36</TD></TR></TBODY></TABLE></DIV><P> We purified the bacteriocins using the <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#IMPACT" target="_blank">IMPACT Method</A>. To assay the concentration of the purified bacteriocins, we used a Bradford standard protein assay. Known concentrations of BSA (Bovine serum albumine) was used to compare measured absorbance of the bacteriocin to the relative absorbance of known BSA concentrations. Measured concentrations are illustrated in Table 1. </P></DIV><H5>Bacteriocidal effect on<EM> S. aureus</EM> MRSA and <EM>P. aeruginosa</EM> (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="demonstrate1" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 2: The graphs shows the results of MIC tests performed with cell lysates of strains expressing respective bacteriocins. The graphs are plotted as OD/hours where SN = Supernatant. The color code indicates the concentrations used [µg/mL]. SDU11 = ThuricinS <A href="http://parts.igem.org/Part:BBa_K2018011" target="_blank">K2018011</A> tested on E. Cloacae, SDU14 = Laterusporulin-ThuricinS <A href="http://parts.igem.org/Part:BBa_K2018014" target="_blank">K2018014</A> tested on P. aeruginosa.</P></DIV></DIV><P class="figuretext"><EM>Figure 2: The graphs shows the results of MIC tests performed with cell lysates of strains expressing respective bacteriocins. The graphs are plotted as OD/hours where SN = Supernatant. The color code indicates the concentrations used [µg/mL]. SDU11 = ThuricinS <A href="http://parts.igem.org/Part:BBa_K2018011" target="_blank">K2018011</A> tested on E. cloacae, SDU14 = Laterusporulin-ThuricinS <A href="http://parts.igem.org/Part:BBa_K2018014" target="_blank">K2018014</A> tested on P. aeruginosa.</EM></P></DIV><P>From the collaboration with the <A href="https://2016.igem.org/Team:Stockholm" target="_blank">iGEM Stockholm team 2016</A> we got an indication of the effect of our bacteriocins. The team performed MIC tests using cell lysates that contained our bacteriocins, towards the strains <EM>S. aureus, P. aeruginosa</EM> and <EM>E. cloacae. </EM> Part of the MIC results from iGEM Stockholm team is illustrated in Figure 2. Click here to view <A href="https://static.igem.org/mediawiki/2016/6/64/T--SDU-Denmark--Stockholm_graphcollect_wiki_bacto.pdf" target="_blank">full report </A>.</P><P>The red/brown line (Figure 2) indicates gentamycin as a positive control which is shown to inhibit the growth of <EM>Enterobacter cloacae</EM>. <EM>E. cloacae</EM> is considered an important cause of nosocomial infections <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC113501/" target="blank">Keller, R., Pedroso, M. Z., Ritchmann, R., &amp; Silva, R. M. (1998). Occurrence of Virulence-Associated Properties in Enterobacter cloacae. Infection and Immunity, 66(2), 645–649. </A></SPAN></SPAN>. The cell lysate containing ThuricinS (SDU11-SN) with the highest concentration elicit a similar and better effect towards <EM>E. cloacae</EM> compared to the traditional antibiotic, gentamycin. The lysate containing ThuricinS is still effective after 40 hours of incubation.  SDU14-SN represents a MIC test performed on <EM>P. aeruginosa</EM> with cell lysate containing our hybrid bacteriocin Laterusporulin-ThuricinS. The cell lysate show similar effect on <EM>P. aeruginosa</EM> compared to gentamycin. The results from <A href="https://2016.igem.org/Team:Stockholm" target="_blank">iGEM Stockholm team 2016</A> thus suggest that ThuricinS and the hybrid Laterusporulin-ThuricinS are potent inhibitors of the growth of <EM>E. cloacae</EM> and <EM>P. aeruginosa</EM>, respectively. However as the MIC tests were performed using cell lysates a clear conclusion cannot be drawn. </P><P>We purified our bacteriocins in order to specify the effect of our bacteriocins as single acting proteins. To assay the effect of our purified bacteriocins we performed a <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#mictest" target="_blank">MIC test</A>. The bacteriocins were dissolved in water from added to wells A-G, having the largest concentration possible at column A. The maximal concentration used is the half of the calculated concentration of the bacteriocin (Tabel 1). We used different MRSA strains to test the effect of the bacteriocins in order to investigate the possibility for the bacteriocins as substitutes for traditional antibiotic of which the bacteria have evolved resistance. The following strains where tested:</P><UL class="list"><LI><EM>MRSA:CC398</EM>– a methicillin and tetracyclin resistant <EM>S. aureus </EM>isolated from a patient, which have gained it from a pig. The MRSA:CC398 is called pig-MRSA, which is found in many pigs in Scandinavia. It is a huge contributor to the resistance development.</LI><LI><EM>Hetero - VISA</EM>– a <EM>S. aureus</EM> strain. The strain is capable of growing at vancomycin concentration of &gt;4 mg/L and have thereby gained relative vancomycin resistance. This is due to thickened walls of the strain, which is suggested to catch the vancomycin before entry <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2094890/." target="blank">Conly, J. M., &amp; Johnston, B. L. (2002). VISA, hetero-VISA and VRSA: The end of the vancomycin era? The Canadian Journal of Infectious Diseases, 13(5), 282–284. </A></SPAN></SPAN>.</LI><LI><EM>MRSA:USA300</EM>– the <EM>S. aureus</EM> strain is a community associated MRSA and is a common cause of skin infections <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="http://mbio.asm.org/content/6/2/e00054-15" target="blank"> Alam, M. T., et al. (2015). &quot;Transmission and microevolution of USA300 MRSA in U.S. households: evidence from whole-genome sequencing.&quot; MBio 6(2): e00054.</A></SPAN></SPAN>.</LI><LI><EM>P. aeruginosa:PAO1</EM>– the strain is commonly spread at hospitals thus given rise to re-hospitalization of people with open wound infection, which we would like our bacteriocins to face.</LI></UL></DIV><H5>MIC test (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:50%;padding-left:15px;"><DIV id="demonstrate3" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 3 shows visual MIC test results of <A href="http://parts.igem.org/Part:BBa_K2018010" target="_blank">K2018010</A>/Laterusporulin. Well A(1-10) contain 33.5 µg/mL as final maximal concentration. A two-fold dilution is performed in well B-G(1-10). Well H(1-10) does not contain bacteriocin, but contain bacterial culture. Well A-H(11-12) is BLANK.</P></DIV></DIV><P class="figuretext"><EM>Figure 3 shows visual MIC test results of <A href="http://parts.igem.org/Part:BBa_K2018010" target="_blank">K2018010</A>/Laterusporulin. Well A(1-10) contain 33.5 µg/mL as final maximal concentration. A two-fold dilution is performed in well B-G(1-10). Well H(1-10) does not contain bacteriocin, but contain bacterial culture. Well A-H(11-12) is blank.</EM></P></DIV><P>The MIC plates were performed with a two-fold dilution of the respective concentrations shown in Table 1. The concentrations in Table 1 indicate final concentrations i.e. the concentration of the bacteriocin in the well after adding bacterial culture. An example of the bacteriocin <A href="http://parts.igem.org/Part:BBa_K2018010" target="_blank">K2018010</A>/Laterosporulin plated in the MIC wells is shown in Figure 3. To validate growth or inhibition we used the absorbance measured in wells with no visual growth. A mean of absorbance measured in the wells without growth, was calculated for each MIC plate. The mean value was used as a reference value for the estimation of growth or inhibition. In Figure 3 a visual inhibition of growth of the listed strains by Laterosporulin is shown.</P></DIV><DIV><P class="figuretext"><EM>Table 2 and 3 shows the respective MIC values according to the tested strains and the bacteriocins/traditional antibiotic they were exposed to.</EM></P><TABLE class="bacto" style="width:100%;"><TBODY><TR><TH>Bacteriocin/MIC [µg/mL]</TH><TH>hVISA</TH><TH>USA300</TH><TH>CC398</TH><TH>PAO1</TH></TR><TR><TD>ThuricinS</TD><TD>16.7</TD><TD>16.7</TD><TD>16.7</TD><TD>16.7</TD></TR><TR><TD>LacticinQ</TD><TD>22.2</TD><TD>22.2</TD><TD>22.2</TD><TD> &gt;22.2</TD></TR><TR><TD>Laterosporulin</TD><TD>33.6</TD><TD>16.8</TD><TD>16.8</TD><TD>33.6</TD></TR><TR><TD>LacticinQ-LacticinZ</TD><TD>7.6</TD><TD>7.6</TD><TD>7.6</TD><TD>15.2</TD></TR><TR><TD>Laterosporulin-ThuricinS</TD><TD>13.1</TD><TD>13.1</TD><TD>6.6</TD><TD> &gt;13.1</TD></TR></TBODY></TABLE><TABLE class="bacto" style="width:100%;"><TBODY><TR><TD><B>Traditional Antibiotic/MIC<B>[µg/mL]</B></B></TD><TD><B>hVISA</B></TD><TD><B>USA300</B></TD><TD><B>CC398</B></TD><TD><B>PAO1</B></TD></TR><TR><TD><B>Ampicilin</B></TD><TD>&lt;16.0</TD><TD>&lt;16.0</TD><TD>1000.0</TD><TD> &gt;1000.0</TD></TR><TR><TD><B>Chloramphenicol</B></TD><TD>78.0</TD><TD> &gt;1000.0</TD><TD> &gt;1000.0</TD><TD>39.0</TD></TR></TBODY></TABLE></DIV><H5> Effect of bacteriocins (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="demonstrate4" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 4 visualizes MIC values for each bacteriocin towards each tested strain. The respective strains are listed in the x-axis and the MIC values are plotted logarithmic(2) on the y-axis. The lowest bar indicates the bacteriocin eliciting the strongest effect.</P></DIV></DIV><P class="figuretext"><EM>Figure 4 visualizes MIC values for each bacteriocin towards each tested strain. The respective strains are listed in the x-axis and the MIC values are plotted logarithmic(2) on the y-axis. The lowest bar indicates the bacteriocin eliciting the strongest effect.</EM></P></DIV><P>MIC values are visualized in the bar-chard in Figure 4. The bacteriocin eliciting the strongest effect is indicated by the lowest bar – thus the lowest MIC value. Compared to MIC values of traditional antibiotics (Table 3) i.e. ampicillin and chloramphenicol, the bacteriocins show similar effect. <STRONG>Importantly</STRONG>, the bacteriocins also show better effect due to inhibition of growth of strains, which the traditional antibiotics does not inhibit. </P></DIV><P>In Figure 4 it is seen that ThuricinS affect all of the strains at a concentration of 16.6 µg/mL. LacticinQ affects only <EM>S. aureus</EM> strains at a MIC value of 22.2 µg/mL. Laterosporulin inhibits growth of all strains with the lowest MIC being towards the MRSA strains, USA300 and CC398, at MIC 16.8 µg/mL. Even though the laterosporulin as a single bacteriocin affect <EM> P. aeruginosa</EM>, the Laterosporulin-ThuricinS hybrid does not effect <EM>P. aeruginosa.</EM> However, the MIC values towards hVISA, USA300 and CC398 (13.1 µg/mL, 13.1 µg/mL, 6.6 µg/mL respectively) are lower than the MIC value of their single bacteriocin form (ThuricinS MIC 16.7 µg/mL and Laterosporulin MIC 16.8 µg/mL). This indicates a synergistic effect of hybrid bacteriocins.</P><P>LacticinQ does not inhibit <EM>P. aeruginosa</EM> in the tested range of concentrations. The hybrid LacticinQ-LacticinZ inhibits growth of all the tested strains with a MIC of 7.6 µg/mL against the <EM>S. aureus</EM> strains and a MIC corresponding to 15.2 µg/mL for <EM> P. aeruginosa</EM> (Figure 4). Moreover, LacticinQ-LacticinZ MIC values are lower compared to the MIC estimated for LacticinQ alone (22.2 µg/mL). This could be due to the presence of LacticinZ. However, LacticinZ is not tested as a single bacteriocin and a direct comparison cannot state a decrease in MIC for LacticinZ but only a decrease in MIC for LacticinQ. Two bacteriocins working in concert (a hybrid) seem to contribute to a decrease in MIC and thus a stronger effect, indicating synergy. It should thus be noted that the hybrid Laterosporulin-ThuricinS “looses” its effect towards <EM>P. aeruginosa</EM>, compared to their effect as single proteins.</P><P>In conclusion, we cloned and purified the bacteriocins. We tested the bacteriocins and showed that in some cases the hybrid bacteriocins are more effective then as single proteins. Most importantly we showed the bacteriocins inhibit growth of multi resistant strains most often present in open wounds.</P><BUTTON class="accordion"><B>Replicating MIC</B></BUTTON><DIV id="foo" class="panel"><P>The MIC plates were incubated for 28 hours with equal OD values for each bacterial strain i.e. equal amount of bacteria were added. For replicating the MIC test, the respective generation times of the bacterial strains tested should be considered. The generation times can diverge between the bacterial strains thus representing diverse conditions, which is reflected in the OD measured. Therefore, the OD values between the strains cannot be compared directly, but the MIC values can. Furthermore, instead of using duplicates, multiple replications could have been made for specification of the OD value. Thereby we could determine the OD value due to calculating a median, instead of calculating a mean value and thereby be able compare more OD values. In addition, by using duplicates, pipette errors cannot be neglected. A new MIC test could therefore be performed with multiplicates thus eliminating some of the stated source of error.</P></DIV><H3>Silk assembly</H3><H5>Insertion of MaSp gene fragments into pSB1C3 (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><P class="figuretext"><EM>Figure 5 shows the cPCR products of MaSp gene fragments with overhangs. <A href="https://static.igem.org/mediawiki/2016/e/ea/T--SDU-Denmark--ThermoFisher_Generuler_DNA_ladder.jpeg" target="_blank">GeneRulerTM 100 bp DNA ladder </A> were used. </EM></P></DIV><P>The MaSp gene fragments were synthesized by IDT. In order to achieve a renewable source of the genes a key step was to ligate each gene fragment into their separate pSB1C3 vectors. Recall that the MaSp genes contain 3 gene fragments, which together make up one monomer and thus a functional MaSp protein. The individual ligated gene fragments in pSB1C3 were transformed into <EM>E. coli</EM>. Successful ligations were verified by colony PCR (Figure 5). The expected length from cPCR is ca. 400 bp, which corresponds to one MaSp gene fragment with cPCR overhangs. The ligations were verified by sequencing. </P></DIV><P>The <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#ICA" target="_blank">ICA technique</A> allows us to ligate different pieces of DNA together stepwise due to matching overhangs. Eco31I recognizes four nucleotides, but cuts four randomly base pairs at the location + 1bp upstream from the recognition site. By making specific primers that add restriction sites, we were able to create compatible overhangs between silk fragments. This approach was inspired from the <A href="https://2015.igem.org/Team:UCLA" target="_blank">iGEM UCLA team 2015</A>.</P><H5>Ligation test (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><P class="figuretext"><EM>Figure 6 shows a ligation test. Bands at 106 bp are the unligated silk genes. Bands at 212 bp are the ligated product.</EM></P></DIV><P>We tested the ability of the gene fragments to ligate by performing a ligation test. The digested gene is 106 bp. The bonds at 212 bp verified that the gene fragments could be ligated together (Figure 6).</P></DIV><H5>Preparing silk by the ICA method (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><P class="figuretext"><EM>Figure 7 shows a monomer of MaSp1 and MaSp2 made by ICA.</EM></P></DIV><P>It was possible to prove the concept of preparing silk from the <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#ICA" target="_blank">ICA method</A>. One monomer of MaSp1 and MaSp2 was made by the ICA method (Figure 7). The monomer is expected to have a length of 406 bp as one monomer of a MaSp gene consists of the gene fragments AB, BC and CA with the respective lengths of 106 bp. Additional base pairs originates from primer overhangs.</P><P>We tried to make constructs containing 4 MaSp gene monomers. However, we did not manage to make a longer construct. The closest result we achieved was to make a gene containing 700 bp. In the end we found that our magnetic beads had little to no streptavidin bound, alas it was not possible to use the ICA protocol for consistent results. There is however a great potential in the protocol, but one should test each individual component before attempting the ICA.</P></DIV><H5>The dream of creating a silk-bacteriocin hybrid (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><P class="figuretext"><EM>Figure 8 shows the result of a fusion PCR. The result indicates that we succeeded getting <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments" target="_blank">silk-overhangs</A> on ThuricinS. </EM></P></DIV><P>We wanted to create the hybrid silk of a bacteriocin fused with one or multiple monomers of MaSp1, MaSp2 with different proportions of the gene fragments at the DNA level. This was tested using recombinant DNA technology. Two methods were tested: <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#ICA" target="_blank">ICA,</A> and restriction cloning</P><P>We managed to prepare a bacteriocin with specific overhangs, matching the overhangs present on the MaSp genes. Lane no. 5 (Figure 8) shows the bacteriocin ThuricinS with DA-overhangs. The band is expected to be 217 bp. ThermoFisher scientific <A href="https://static.igem.org/mediawiki/2016/e/ea/T--SDU-Denmark--ThermoFisher_Generuler_DNA_ladder.jpeg" target="_blank">GeneRuler 50 kb ladder </A> was used. 
</P><P>Despite various attempts we never succeeded in cutting and ligating ThuricinS together with a silk monomer. The bacteriocin with specific silk overhangs are therefore left to further research.</P><P>In conclusion, we created standard parts containing single silk gene fragments. Each silk gene fragment was ligated to its respective counterpart. A monomer was created using the ICA protocol. We added silk-overhangs to the bacteriocins allowing for incorporation into silk gene fragments. </P></DIV><BUTTON class="accordion"><B>What went wrong - what to do?</B></BUTTON><DIV id="foo" class="panel"><P><STRONG>ICA troubleshooting</STRONG></P><P>Due to the multiple ligation steps in ICA, individual ligation test were made in order to establish the viability of the constructs. Individual ligation tests were time consuming but a necessity since a singular failed ligation will create negative results. As the ICA protocol is heavily reliant on everything to work as planned, alternative methods were tried. In addition, it must be noted that the small overhangs are seldom fit to ligate at room temperature. Likewise, redesigning the gene towards a larger overhang, might create easier ligations. Furthermore during ligation, constant turbation of the fluids must be applied, as the beads subside in the bottom of the tube. Thereby creating unfavorable conditions for ligations.</P><P><STRONG>Alternative method</STRONG></P><P>We tried ligating constructs, which did not have overlapping overhangs eg. Initiator-AB-BC, CA-AB and BC-CA-Terminator. This were obtained by ligating in series at 16 degrees celsius overnight. The lower temperature is suggested to increase the possibility of ligation. However the lower temperature also decrease the activity of T7 ligase. Thus we added a small amount of 40 mM ATP to the sample at each ligation step hoping to create a silk construct. Unfortunately we ran out of time due to expired streptavidin beads. The experiment are thus left for further trial.</P><P><STRONG>Missing amplification of ThuricinS with silk overhangs </STRONG></P><P>We noticed that using the standard primers from iGEM, <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_G00100" target="_blank">VF2</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_G00101" target="_blank">VR</A>, our pPCR on plasmids containing the bacteriocin/silk-overhangs, the following digestion leaded to two bands that only differed by 1 bp from each other. We therefore decided to amplify our bacteriocin with silk-overhangs. However, after changing primers we experienced that the pPCR gave unexplainable results which included a lot of smear. Various pPCR programs were tested but we never ended up with a product that could be purified and used in the ICA method. We also tried to digest the pSB1C3 plasmid and the ThuricinS/silk-overhang pPCR product to perform a ligation into the pSB1C3 directly. This result was verified by gel electrophoresis. However, it was not possible to digest the plasmid containing the ThuricinS with silk overhangs and a purification could not be performed. Ligation between the ThuricinS and a silk monomer were therefore not achieved. For future testing it would be interesting to digest with restrictions enzymes for longer periods, or design new primers for the pPCR, necessary for ICA method.</P></DIV><H3>Production of Polyhydroxybutyrate (PHB)</H3><P>The following section contains our results for optimizing and characterizing the production of PHB in <EM>E. coli</EM>.</P><H5>PHB producing cells (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><P class="figuretext"><EM>Table 4 shows the BioBrick composition of our PHB producing libary. For the extra promoter and the extra RBS the background color gradient indicates the strength of the extra promotor/RBS. The darkest color is the strongest and visa versa. Strength is defines by iGEM teams Berkeley 2006 and Warsaw 2010. </EM></P></DIV><P>In 2013 iGEM team Imperial college London increased PHB production 11-fold by adding an extra promotor and ribosomal binding site in front of the phaCAP genes. We created a library of different promotor hybrids in order to investigate elements of importance and possibly increase PHB production further. In order to <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#flow" target="_blank">determine the effectiveness</A> of our library of PHB producing cells, we used flow cytometry. By staining the cell cultures with nile red, we created a relative measure on PHB levels inside each cell. The following results were obtained with 4 biologic replicates of each <EM>E.coli</EM> strain (Table 4). The <EM>E.coli</EM> strains were incubated for 28 hours before the measurements were made.</P></DIV><H5>Measurement of PHB production (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="demonstrate5" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 9 display the number of normalized events on the y-axis and the corresponding intensities on the x-axis. Cells of the different strains are displayed in different colors. Each event represents a cell and the intensity represents the amount of PHB in the respective cell.</P></DIV></DIV><P class="figuretext"><EM>Figure 9 display the number of normalized events on the y-axis and the corresponding intensities on the x-axis. Cells of the different strains are displayed in different colors. Each event represents a cell and the intensity represents the amount of PHB in the respective cell.</EM></P></DIV><P>Figure 9 shows a clear separation of the different strains. This we would expect for strains with different promoter and ribosomal binding sites in front of the phaCAB genes. We included the BioBrick created by the <A href="https://2009.igem.org/Team:Tokyo_Tech" target="_blank">iGEM Tokyo Tech team 2009 </A><A href="http://parts.igem.org/Part:BBa_K934001" target="_blank">(K934001)</A> and one by the <A href="https://2013.igem.org/Team:Imperial_College" target="_blank">iGEM Imperial College team 2013 </A><A href="http://parts.igem.org/Part:BBa_K1149051" target="_blank">(K1149051)</A> as references for our own constructs. The strain containing the reference Imperial College team construct is displayed in green in Figure 9. However, as seen in Figure 9, the events with the largest intensities, originates from the <EM>E. coli</EM> strain containing our construct <A href="http://parts.igem.org/Part:BBa_K2018036" target="_blank">(K2018036)</A>. This indicates that our construct generates more plastic, than any other hybrid promoter phaCAB tested in the figure.</P><P>Unlike our initial expectations, the results of our <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#flow" target="_blank">flow cytometry analysis</A> strongly suggest that <B> the strongest additional promoter does not produce the largest amount of plastic</B>(see below). However, without further qualitative data, we can only speculate as to why, this is the case. </P></DIV><H5> Additional RBS - largest increase in PHB production (<A href="#overview">Top</A>) </H5><DIV><DIV style="float:right;width:40%;padding-left:15px;"><DIV id="demonstrate6" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 10 display the different <EM>E. coli</EM>strains on the x-axis. The promoter and RBS are marked with corresponding affinity: “s” = strong, “W” = weak and “MS”=medium. The y-axis displays the average intensity on the flow cytometer detected by red fluorescence. The intensity of the red fluorescence is calculated due to a mean value of the intensities detected in Figure 9.</P></DIV></DIV><P class="figuretext"><EM>Figure 10 display the different <EM>E.coli</EM>strains on the x-axis. The promoter and RBS are marked with corresponding affinity: “s” = strong, “W” = weak and “MS”=medium. The y-axis displays the average intensity on the flow cytometer detected by red fluorescence. The intensity of the red fluorescence is calculated due to a mean value of the intensities detected in Figure 9. </EM></P></DIV><P>Equally surprising is the fact that the additional RBS is responsible for the largest increase in PHB (Figure 10). The additional RBS for the <I>phaCAB</I> genes is not followed by a start-codon and as a result, ribosomal attachment to the specific RBS will not be able to initiate transcription. We suggest that a <EM>strong RBS</EM> will bind the ribosome with high affinity and thus it will not detach readily.  For a <EM>medium RBS</EM>, the affinity will be high enough for ribosomes to be attached to the RNA frequently, but due to the lower affinity, they will also detach more frequently. The event results in an increase in the local concentration of ribosomes. For the <EM>weak RBS</EM> the affinity will be too low for the local concentration of ribosomes to be effected.</P><P>The strong and weak RBS thereby generates less plastic, relative to the medium strong RBS. We speculate that this tendency is caused by changes to the local concentration of ribosomes, due to the presence of an additional RBS, upstream from the translational RBS.</P><P>We used the medium strong RBS for further analysis. </P></DIV><H3>Examining methods of PHB extraction</H3><DIV><DIV style="float:right;width:40%;padding-left:15px;"><DIV id="demonstrate7" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 11 illustrates the amount of plastic extracted pr. biomass for the different methods of extraction.</P></DIV></DIV><P class="figuretext"><EM>Figure 11 Illustrates the amount of plastic extracted pr. biomass for the different methods of extraction.</EM></P></DIV><P>The method of PHB extraction is an important part of the production that should be taken into account, as it has a large impact on the purity and yield of PHB. We have chosen four <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#solvent" target="_blank">methods of extraction</A>; Hypochlorite digestion, Hypochlorite digestion with Triton X-100 pre-treatment, Chloroform solvent extraction and Ethyl acetate solvent extraction.</P><P>When we compare the amount of purified material from the different extraction methods, hypochlorite provides the most material, indicating that it has the highest yield. The amount of material alone is not enough to determine which method provides the largest yield of PHB, as the recovered material might not be plastic. In order to get a better view of the actual amount of plastic each method provides, purity needs to be taken into account.</P></DIV><H5> H NMR as qualitative measure of purity for extration methods (<A href="#overview">Top</A>)</H5><DIV><P> In order to determine whether the different methods of PHB extraction we used were effective, we analyzed the purified material with H NMR. 
We determined the peaks caused by PHB by analyzing a sample of pure PHB as a reference. The sample of pure PHB (Figure 12) can be held against the spectra from extracted PHB (Figure 13). Since this spectrum is based on pure PHB, all the peaks present are not considered contaminations in the other spectra observed in Figure 13.</P><DIV class="row"><DIV class="col-sm-6"><DIV id="demonstrate8" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 12 illustrate the results of HNMR. Each peak is caused by hydrogen atoms. The area under each curve is proportional to the number of chemical equivalent hydrogen atoms causing the peak.</P></DIV></DIV><P class="figuretext"><EM>Figure 12 illustrate the results of HNMR. Each peak is caused by hydrogen atoms. The area under each curve is proportional to the number of chemical equivalent hydrogen atoms causing the peak.</EM></P></DIV><DIV class="col-sm-6"><DIV id="demonstrate9" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 13 illustrate the spectra for the methods of PHB extraction of which we have tested. Top left is for chloroform extraction, top right is for Ethyl acetate extraction, bottom left is for Hypochlorite extraction and bottom right is for Hypochlorite extraction with Triton X-100 pre-treatment.</P></DIV></DIV><P class="figuretext"><EM>Figure 13 illustrate the spectra for the methods of PHB extraction of which we have tested. Top left is for chloroform extraction, top right is for Ethyl acetate extraction, bottom left is for Hypochlorite extraction and bottom right is for Hypochlorite extraction with Triton X-100 pre-treatment.</EM></P></DIV></DIV><P><B>Extraction with Chloroform</B></P><P>The spectrum that shows chloroform extraction has very clearly defined PHB tops and no indications of impurities. The spectrum indicates that the specific method of extraction, provides PHB with a high level of purity.</P><P><B>Extraction with Ethyl acetate </B></P><P>The spectrum that shows ethyl acetate extraction has well defined PHB peaks, but an unknown compound in the sample cause a peak at approximately 2.17 ppm. This indicates that the PHB purity is lowered with this extraction method, when compared to chloroform extraction.</P><P><B>Extraction with Hypochlorite</B></P><P>In the spectrum showing hypochlorite extraction the PHB peaks are clear, but a single impurity is registered in the spectrum at 2.3 ppm. The area under the two peaks near the impurity are almost doubled compared to the peak at 5.2 ppm. This inequality, however, is not significant as it is the impurity in the sample, that contribute to the additional area under the peaks.</P><P><B>Extraction with hypochlorite and Triton X-100</B></P><P>The peaks representing PHB peaks are present but the peak for water is absent. There is no indication of impurities, although these could have evaporated along with the water. This specific method is similar to the Hypochlorite extraction method. The Triton X-100 pre-treatment is performed to increase the digestion of cellular content, which could explain the absence of impurity. </P><P>For the hypochlorite PHB extraction with Triton X-100 and the ethyl acetate purification, the entire content of the purification, was not soluble in chloroform. This indicates that large impurities present in the sample were not visible in the H NMR spectra.</P></DIV><H5>Determination of intracellular PHB concentration(<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:40%;padding-left:15px;"><DIV id="demonstrate10" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 14 shows the mirrored spectrum from a gas chromatography analysis of 20 mg cells containing <A href="http://parts.igem.org/Part:BBa_K2018036" target="_blank">(K2018036)</A> (blue) and a 2 mg pure PHB standard (red). PHB causes a top at 1.150 min. and a top at 2.135 min. The top at 2.135 min is an internal methyl benzoate standard.</P></DIV></DIV><P class="figuretext"><I>Figure 14 shows the mirrored spectrum from a gas chromatography analysis of 20 mg cells containing <A href="http://parts.igem.org/Part:BBa_K2018036" target="_blank">(K2018036)</A> (blue) and a 2 mg pure PHB standard (red). PHB causes a top at 1.150 min. and a top at 2.135 min. The top at 2.135 min is an internal methyl benzoate standard.</I></P></DIV><P>With gas chomatography the amount of PHB inside the cells can be determined. In order to do this, PHB standards containing 2 mg of pure PHB were created to compare and determine the amount of PHB in the tested cell samples. Cell samples of 20 mg were analysed and quadruplets of both standards and samples were made.</P><P>The area under the curve illustrates the amount of analyzed  material at the respective time. The amount of PHB in the figure is significantly lower in the tested cell sample than observed for the standard sample containing 2 mg pure PHB. The observation indicates that there is less than 2 mg PHB in the tested cell sample. When the data was normalized according to the methyl benzoate standard, it showed that there were 0.7 mg PHB in 20 mg cell sample. This corresponds to 3.5% of the cell mass being PHB. With this value we can determine the yield for the <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#solvent" target="_blank">methods of extraction</A> (Table 4). </P></DIV><DIV><P class="figuretext">Table 4 shows the determined properties of the different PHB extraction methods.</P><TABLE class="plast"><TBODY><TR><TH>Extraction method</TH><TH>yield</TH><TH>HNMR assessment</TH><TH>Soluble in chloroform</TH></TR><TR><TD>Hypochlorite</TD><TD>25.4%</TD><TD>unpure</TD><TD>yes</TD></TR><TR><TD>Ethyl acetate</TD><TD>25.3%</TD><TD>unpure</TD><TD>no</TD></TR><TR><TD>Triton x-100</TD><TD>19.3%</TD><TD>pure</TD><TD>no</TD></TR><TR><TD>Chloroform</TD><TD>11.2%</TD><TD>pure</TD><TD>yes</TD></TR></TBODY></TABLE></DIV><P>We were capable of making in house large scale PHB purification. Based on data in table 4 and the fact that we could produce in house large scale PHB purification, we continued with the hypochlorite extraction protocol. </P><H3>Quantitative proteome analysis of <I>panK</I> expressing cells - iTRAQ analysis (<A href="#overview">Top</A>)</H3><P>In 2015 Stanford Brown increased PHB production 23% by introducing staphylococcal pantothenate kinase II to PHB producing <I>E. coli</I>. The purpose is to increase intracellular levels of acetyl-CoA used as substrate for PHB production. We investigated the proteomic effect of expressing the panK in <I>E. coli</I> cells to characterize the BioBrick. To do so, we used <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#iTRAQ" target="_blank">iTRAQ analysis</A> which allowed us to compare the proteomes of <I>E. coli</I> and <I>E. coli</I> expressing panK cells. The <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#flow" target="_blank">iTRAQ labelling</A> was only partial, resulting in compromised data. However, the up- and down regulation of several proteins, was identified in the cells where pantothenate kinase II was expressed. In the MS/MS analysis a total of 2402 proteins was identified in our bacteria. Out of these we determined 21 were upregulated and 11 were down regulated (Figure 15). </P><H5>String analysis </H5><DIV style="float:right;width:40%;padding-left:15px;"><DIV id="openDemonstrate" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 15 illustrates all the proteins that are upregulated due to the presence of pantothenate kinase II. Proteins that are associated with each other are linked and proteins involved in metabolism are highlighted in red.</P></DIV></DIV><P class="figuretext"><EM>Figure 15 illustrates all the proteins that are upregulated due to the presence of pantothenate kinase II. Proteins that are associated with each other are linked and proteins involved in metabolism are highlighted in red.</EM></P></DIV><P>We performed <A href="http://string-db.org" target="_blank">String analysis </A> on the proteins that were upregulated in order to determine the effect of <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#panK" target="_blank">pantothenate kinase II</A> on the proteome of <I>E. coli</I>. In Figure 15 the proteins involved in metabolism are highlighted. It is not surprising that a large amount of the proteins are involved in metabolism, as pantothenate kinase itself is a metabolic protein. The three central proteins in the figure; tpiA (triose phosphate isomerase), PGK (phosphoglycerate kinase) and pykF (puryvate kinase) are all involved in glycolysis. NuoG (NADH dehydrogenase subunit G) is involved in the electron transport chain, another mechanism by which the cell generates energy. The rest of the upregulated proteins did not have as clear a correlation. The increased amount of glycolytic enzymes indicates an increased energy consumption in the cells, due to the presence of pantothenate kinase II. The fact that coenzyme A is utilized in the oxidation of pyruvate, fits with the increase in glycolytic enzymes.</P><H5>Identification of phosphorylations (<A href="#overview">Top</A>)</H5><DIV><DIV style="float:right;width:40%;padding-left:15px;"><DIV id="demonstrate11" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 16: Illustrates the result of a masspec analysis of a peptide derived from 2,3-bisphosphoglycerate-independent phosphoglycerate mutase. The green fragment in the spectrum indicates the detection of a phosphoryl modification.</P></DIV></DIV><P class="figuretext"><I>Figure 16: Illustrates the result of a masspec analysis of a peptide derived from 2,3-bisphosphoglycerate-independent phosphoglycerate mutase. The green fragment in the spectrum indicates the detection of a phosphoryl modification. </I></P></DIV><P>The titanium dioxide purification allowed us to analyze the phosphorylated proteins in the sample. It is however, important to note that this purification is meant for analysis of eukaryotic cells and therefore, the amount of identified proteins will be low. There were no identified phosphorylations that were more (or less) abundant in cells that expressed pantothenate kinase II. Since none of the upregulated proteins where protein kinases this result is not surprising. Although the phophoprotein analysis did not provide any data relevant for our project, a phosphorylation on serine 5 of <A href="http://www.uniprot.org/uniprot/A1AHE9#ptm_processing" target="_blank">2,3-bisphosphoglycerate-independent phosphoglycerate mutase</A> that is not registered on uniprot, was identified in all of our samples.</P></DIV><H3>Secretion system (<A href="#overview">Top</A>)</H3><DIV class="row"><DIV class="col-sm-6"><DIV id="demonstrate13" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 17: The flask on the left contains E. coli with <A href="http://parts.igem.org/Part:BBa_K1149051" target="_blank"> BioBrick K1149051</A>(minus sec, minus panK) in pSB1C3 and the flask on the right contains E. coli with <A href="http://parts.igem.org/Part:BBa_K2018050" target="_blank">BioBrick K2018050</A> (plus sec, plus panK). The two flasks have been incubated for approximately 48 hours each at the same temperature. PHB aggregates are visible in the right flask.</P></DIV></DIV><P class="figuretext"><EM>Figure 17: The flask on the left contains E. coli with <A href="http://parts.igem.org/Part:BBa_K1149051" target="_blank"> BioBrick K1149051</A>(minus sec, minus panK) in pSB1C3 and the flask on the right contains E. coli with <A href="http://parts.igem.org/Part:BBa_K2018050" target="_blank">BioBrick K2018050</A> (plus sec, plus panK). The two flasks have been incubated for approximately 48 hours each at the same temperature. PHB aggregates are visible in the right flask.</EM></P></DIV><DIV class="col-sm-6"><DIV id="demonstrate14" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 18: Illustrates the amount of biomaterial extracted from different strains of bacteria pr. OD measured, with the hypochlorite extraction method. The experiment was performed with duplicates and the pilars illustrate the average PHB weight. The extractions were performed 24 hours (blue), 48 hours (brown) &amp; 72 hours (grey) after incubation.</P></DIV></DIV><P class="figuretext"><EM> Figure 18: Illustrates the amount of biomaterial extracted from different strains of bacteria pr. OD measured, with the hypochlorite extraction method. The experiment was performed with duplicates and the pilars illustrate the average PHB weight. The extractions were performed 24 hours (blue), 48 hours (brown) &amp; 72 hours (grey) after incubation. </EM></P></DIV><P>In order to improve the platform for PHB production we implemented a secretion system for PHB. The effect of the <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#secretion" target="_blank">secretion system</A> was quickly confirmed, as visible aggregates formed in the media of PHB secreting cultures. This phenomenon has only been observed in cultures with bacteria containing the secretion system. We tried to provide a <A href="https://2016.igem.org/Team:SDU-Denmark/Experiments#flow" target="_blank">quantitative measure</A> for the amount of PHB the secretion system provides, but due to the large PHB aggregates flow cytometry was not an option. Instead we compared the amount of extracted material from PHB secreting strains with other PHB producing strains (mg pr. OD) (Figure 18). </P><P>The data was too inconsistent to form any strong conclusions, but <I>E. coli</I> containing the <A href="http://parts.igem.org/Part:BBa_K2018050" target="_blank">BioBrick K2018050</A>(plus panK, plus sec) grew significantly slower than other strains. This problem did not occur for the cells containing the <A href="http://parts.igem.org/Part:BBa_K2018049" target="_blank">BioBrick K2018049</A> (plus sec, minus panK). The samples extracted at the same amount of time after incubation are comparable with each other, as they were incubated under the same conditions.</P><H5>Choosing the optimal media for PHB production</H5><DIV><DIV style="float:right;width:40%;padding-left:15px;"><DIV id="demonstrate20" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P> Figure 19: Shows the logarithmic value of OD600 plotted against time in minutes for growth curves of Top10/pSB1C3-panK+sec (dashed lines) and Top10 (filled lines). The red lines show bacteria grown in 2xYT media, blue lines show bacteria grown in LB media and green lines show bacteria grown in TB media.</P></DIV></DIV><P class="figuretext"><EM> Figure 19: Shows the logarithmic value of OD600 plotted against time in minutes for growth curves of Top10/pSB1C3-panK+sec (dashed lines) and Top10 (filled lines). The red lines show bacteria grown in 2xYT media, blue lines show bacteria grown in LB media and green lines show bacteria grown in TB media. </EM></P></DIV><P>To increase the yield of PHB we examined the growth of Top10 and Top10/pSB1C3-panK+sec in three different types of media. Figure 19 gives a quick overview over the growth curves for the two bacterial strains in Luria-Bertani (LB), Teriffic broth (TB) &amp; 2x Yeast extract and tryptone (2xYT) media. The growth curves in Figure 19 was used to determine the growth rate expressed as doubling time and final OD.</P></DIV><DIV><DIV style="float:right;clear:right; width:40%;padding-left:15px;"><DIV id="demonstrate21" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 20: Shows the doubling time of the two bacterial strains: Top10 (white bars) and Top10/pSB1C3-panK+sec (black bars). Each bacterial strain was cultivated in LB, 2xYT and TB. </P></DIV></DIV><P class="figuretext"><EM> Figure 20: Shows the doubling time of the two bacterial strains: Top10 (white bars) and Top10/pSB1C3-panK+sec (black bars). Each bacterial strain was cultivated in LB, 2xYT and TB. </EM></P></DIV><P>In Figure 20 it can be seen that Top10/pSB1C3 grows slower than Top10 in all of the three tested media. Students t-test with 95% confidence determines p values lower than 0.0002 for the difference in doubling time in each of the three types of media. Growth of the same strains in TB and LB had similar doubling times. The maximum growth rate was twice as large in 2xYT as compared with the other media.</P></DIV><DIV><DIV style="float:right;clear:right; width:40%;padding-left:15px;"><DIV id="demonstrate22" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 20: Shows the doubling time of the two bacterial strains: Top10 (white bars) and Top10/pSB1C3-panK+sec (black bars). Each bacterial strain was cultivated in LB, 2xYT and TB. </P></DIV></DIV><P class="figuretext"><EM> Figure 21: Shows OD_600 of Top10/pSB1C3-panK+sec after 20 hours of growth in the three different types of media. </EM></P></DIV><P>As seen in Figure 21 Top10/pSB1C3-panK+sec reached an OD that is twice as high when grown in 2xYT as compared to growth in LB. For growth in TB Top10/pSB1C3-panK+sec reached an OD that is four times as high, as compared to growth in LB. Based on students t-test these results are also significant with p values lower than 0.0001. Despite the faster growth in 2xYT the final amount of cells is significantly larger for growth in TB media, indicating that this media is more suitable for production of PHB.</P><P>To sum it up 2xYT is optimal for rapid growth of bacteria but the bacteria also reaches steady state faster suggesting that the bacteria depletes the media and therefore stops replicating, whereas LB media and TB media has slower doubling time. In both figure 19 and figure 21 it is apparent that TB provides the highest concentration of cells, which is desirable for production of PHB. Although TB media is the media where the bacteria grow at the slowest rate, it is also the media that produces most bacteria over time and therefore it is expected that this is the media that would be able to give the highest yield of PHB. which is why TB media was chosen for further experiments.</P></DIV><P>In conclusion, we created a PHB promoter/RBS libary, identifying the BioBrick producing the highest level of PHB. We analyzed different purification methods with regards to purity, yield and ease of large scale production and in addition we found the optimal media for high cell density of PHB producing cells. Finally we improved the characterization of a BioBrick previously proven to increase PHB production, quantitatively at a proteomic level. </P></DIV></DIV></DIV></DIV><DIV id="sidewikimenu" class="menu_wrapper col-lg-2 col-lg-push-2 col-md-2 hidden-xs hidden-sm"><DIV class="collapsable_menu_control hidden-xs hidden-sm hidden-md hidden-lg hidden-xl">Menu</DIV><UL id="sideaccordion" class="accordion"><LI class="menu_item" id="side1"><A id="sideone" href="https://2016.igem.org/Team:SDU-Denmark">Home</A></LI><LI class="menu_item" id="side2"><A id="sidetwo" href="https://2016.igem.org/Team:SDU-Denmark/Introduction">Introduction</A></LI><LI><A id="team" href=" https://2016.igem.org/Team:SDU-Denmark/Team" hijax="">Team</A></LI><LI><A id="attributions" href="https://2016.igem.org/Team:SDU-Denmark/Attributions"> 
                                Attributions </A></LI><LI><A id="collaborations" href="https://2016.igem.org/Team:SDU-Denmark/Collaborations">Collaborations</A></LI><LI><A id="achievements" href="https://2016.igem.org/Team:SDU-Denmark/Achievements">Achievements</A></LI></UL><LI class="menu_item" id="side3"><A id="sidethree" href="https://2016.igem.org/Team:SDU-Denmark/Project">Project</A></LI><LI><A id="description" href="https://2016.igem.org/Team:SDU-Denmark/Description">Description 
				</A></LI><LI><A id="experiments" href="https://2016.igem.org/Team:SDU-Denmark/Experiments">Experiments 
				</A></LI><LI><A id="demonstration" href="https://2016.igem.org/Team:SDU-Denmark/Demonstrate">  
				Demonstration &amp; Results</A></LI><LI><A id="proof" href="https://2016.igem.org/Team:SDU-Denmark/Proof">Proof of Concept</A></LI><LI><A id="reproducibility" href="https://2016.igem.org/Team:SDU-Denmark/Reproducibility">Scientific Reproduction</A></LI><LI class="menu_item" id="side4"><A id="sidefour" href="https://2016.igem.org/Team:SDU-Denmark/Parts_procedures">Parts &amp; Procedures</A></LI><LI><A id="parts" href="https://2016.igem.org/Team:SDU-Denmark/Parts">Parts </A></LI><LI><A id="notebook" href="https://2016.igem.org/Team:SDU-Denmark/Notebook">Notebook </A></LI><LI><A id="safety" href="https://2016.igem.org/Team:SDU-Denmark/Safety">Safety </A></LI><LI class="menu_item" id="side5"><A id="sidefive" href="https://2016.igem.org/Team:SDU-Denmark/Practices">Practices &amp; Prospects</A></LI><LI><A id="bioethics" href="https://2016.igem.org/Team:SDU-Denmark/Bioethics">Bioethics</A></LI><LI><A id="integrated" href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices"> Integrated Practices </A></LI><LI><A id="outreach" href="https://2016.igem.org/Team:SDU-Denmark/Outreach">Outreach</A></LI><LI><A id="prospects" href="https://2016.igem.org/Team:SDU-Denmark/Prospects">Prospects</A></LI><LI><A id="perspectives" href="https://2016.igem.org/Team:SDU-Denmark/Perspectives">Perspectives</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>