<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SDU-Denmark_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SDU-Denmark/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container-fluid"><DIV class="row"><DIV id="wikimenu" class="menu_wrapper hidden-xl hidden-lg col-sm-12 hidden-md" aria-expanded="true"><DIV class="collapsable_menu_control">Menu</DIV><UL id="accordion" class="accordion"><LI class="menu_item" id="1"><A id="one" href="https://2016.igem.org/Team:SDU-Denmark">Home</A></LI><LI class="menu_item" id="2"><A id="two" href="https://2016.igem.org/Team:SDU-Denmark/Introduction">Introduction</A></LI><LI><A id="team" href=" https://2016.igem.org/Team:SDU-Denmark/Team">Team</A></LI><LI><A id="attributions" href="https://2016.igem.org/Team:SDU-Denmark/Attributions"> 
                                Attributions </A></LI><LI><A id="collaborations" href="https://2016.igem.org/Team:SDU-Denmark/Collaborations">Collaborations</A></LI><LI><A id="achievements" href="https://2016.igem.org/Team:SDU-Denmark/Achievements">Achievements</A></LI></UL><LI class="menu_item" id="3"><A id="three" href="https://2016.igem.org/Team:SDU-Denmark/Project">Project</A></LI><LI><A id="description" href="https://2016.igem.org/Team:SDU-Denmark/Description">Description 
				</A></LI><LI><A id="experiments" href="https://2016.igem.org/Team:SDU-Denmark/Experiments">Experiments 
				</A></LI><LI><A id="demonstration" href="https://2016.igem.org/Team:SDU-Denmark/Demonstrate">  
				Demonstration &amp; Results</A></LI><LI><A id="proof" href="https://2016.igem.org/Team:SDU-Denmark/Proof">Proof of Concept</A></LI><LI><A id="reproducibility" href="https://2016.igem.org/Team:SDU-Denmark/Reproducibility">Scientific Reproduction</A></LI><LI class="menu_item" id="4"><A id="four" href="https://2016.igem.org/Team:SDU-Denmark/Parts_procedures">Parts &amp; Procedures</A></LI><LI><A id="parts" href="https://2016.igem.org/Team:SDU-Denmark/Parts">Parts</A></LI><LI><A id="notebook" href="https://2016.igem.org/Team:SDU-Denmark/Notebook">Notebook </A></LI><LI><A id="safety" href="https://2016.igem.org/Team:SDU-Denmark/Safety">Safety </A></LI><LI class="menu_item" id="5"><A id="five" href="https://2016.igem.org/Team:SDU-Denmark/Practices">Practices &amp; Prospects</A></LI><LI><A id="bioethics" href="https://2016.igem.org/Team:SDU-Denmark/Bioethics"> Bioethics</A></LI><LI><A id="integrated" href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices">Integrated Practices </A></LI><LI><A id="outreach" href="https://2016.igem.org/Team:SDU-Denmark/Outreach">Outreach</A></LI><LI><A id="prospects" href="https://2016.igem.org/Team:SDU-Denmark/Prospects">Prospects</A></LI><LI><A id="perspectives" href="https://2016.igem.org/Team:SDU-Denmark/Perspectives">Perspectives</A></LI></DIV></DIV></DIV><DIV id="header" class="container-fluid"><DIV class="row"><DIV class="header col-lg-8 col-lg-push-2 col-md-10"><DIV class="brand pull-left"><A href="https://2016.igem.org/Team:SDU-Denmark"><H1 id="bodyContent">Bacto-Aid</H1></A></DIV></DIV></DIV></DIV><DIV class="container-fluid"><DIV class="row"><DIV id="page" class="col-lg-8 col-lg-push-2 col-md-10"><DIV class="row"><DIV class="col-sm-11 col-sm-push-1"><H2>Experiments</H2></DIV></DIV><DIV class="row"><DIV class="col-sm-11 col-sm-push-1"><DIV id="overview1" style="background-color:#dddddd; padding:0px 10px 10px 20px;"><H3 id="top">Overview</H3><H5>Generel Verification</H5><UL class="list"><LI><A href="#verify"><I>Verification of Stepwise Experiments: pPCR, cPCR and Fast Digest by Gel Electroforesis </I></A></LI></UL><H5>Working with Bacteriocins</H5><UL class="list"><LI><A href="#IMPACT"><I>IMPACT Method</I></A></LI><LI><A href="#extract"><I>Bacteriocin Extraction</I></A></LI><LI><A href="#bactoconc"><I>Determining Bacteriocin Concentration and Effect</I></A></LI><LI><A href="#mictest"><I>Performning Minimum Inhibitory Concentration (MIC) Test</I></A></LI></UL><H5>Creating Silk Monomers and Hybrid-Silk</H5><UL class="list"><LI><A href="#ICA"><I>Iterative Capped Assembly (ICA) Method</I></A></LI><LI><A href="#assembly"><I>Assembling Silk Gene Fragments</I></A></LI></UL><H5>Working with PHB</H5><UL class="list"><LI><A href="#extraction"><I>PHB Production and Extraction</I></A></LI><LI><A href="#digest"><I>Digestion with Hypochlorite</I></A></LI><LI><A href="#solvent"><I>Solvent Extraction</I></A></LI><LI><A href="#evaluate"><I>Evaluation of Extraction Methods</I></A></LI><LI><A href="#secretion"><I>Developing a Secretion System</I></A></LI><LI><I><A href="#panK">Work with <EM>S. aureus</EM> Pantothenate Kinase II</A></I></LI><LI><I><A href="#iTRAQ">Analysis of <I>E. coli</I> Proteome in panK's Expressing Cells</A></I></LI><LI><A href="#flow"><I>Determination of PHB Production using Flow Cytometry</I></A></LI><LI><A href="#filament"><I>Filament Making - and 3D Printing with PHB</I></A></LI></UL><H5><A href="#protocols">Protocols</A></H5><H5><A href="#SOPs">SOP's</A></H5></DIV><H5>Verification by Gel Electrophoresis (<A href="#overview1">Top</A>) </H5><P>To verify our stepwise experiments, <A href="https://static.igem.org/mediawiki/2016/3/37/T--SDU-Denmark--SOP0010-PhusionPCR.pdf" target="_blank">pPCR</A>, <A href="https://static.igem.org/mediawiki/2016/f/f9/T--SDU-DENMARK--SOP0021-ColonyPCRwithMyTag.pdf" target="_blank">cPCR</A>, and <A href="https://static.igem.org/mediawiki/2016/b/b8/T--SDU-Denmark--SOP0017-FastDigest.pdf" target="_blank">Fast Digest</A>, we used gel electrophoresis. A successful transformation were confirmed by a colony PCR with specially designed primers for each gene. The results were confirmed according to the theoretical expected length of the gene. </P><P>If satisfied with the results, <A href="https://static.igem.org/mediawiki/2016/3/3a/T--SDU-Denmark--SOP0019-MiniPrep.pdf" target="_blank">plasmid purification</A> were made from <A href="https://static.igem.org/mediawiki/2016/e/ec/T--SDU-Denmark--SOP0001-ONcultureofE.coli.pdf" target="_blank">ON cultures </A> from the respective cPCR templates.</P><H3>Bacteriocins </H3><P>Bacteriocins are antimicrobial peptide compounds that effect through adhesion to specific receptors. These are present on the external surface of the target bacteria, where they induce metabolic changes by pore formation. The bacteriocins were selected according to the three listed criteria: </P><P><UL class="list"><LI><I>They elicited an effect on pathogens, most frequent detected in open wounds, causing wound infection.</I></LI><LI>They targeted the cell wall of the target bacteria in order to work when fused with silk. </LI><LI><I>They were non-lethal to E.coli, in order to use E.coli as the producing strain.</I></LI></UL></P><P>Strains that produce bacteriocins, which target the respective bacteria present in open wounds, were examined. Genes encoding the bacteriocins were found and codon optimized, in order to create a BioBrick part containing only the bacteriocin gene. We also created hybrid bacteriocins in order to test if the hybrid showed similar, inhibited or better effect when combined. The use of bacteriocins as new antimicrobial components are of great interest when used in local treatment. In addition it seems like resistance is rarely developed towards bacteriocins. This idea is supported by professor
<A href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices#Frank" target="_blank">Frank Møller Arrestrup</A>.
</P><DIV><DIV><P class="figuretext">Table 1 shows a brief overview of the chosen bacteriocins we worked with in our experiments. To learn more about the specific bacteriocins click on the part numbers in the table.</P></DIV><TABLE><TBODY><TR><TH>Part reg.</TH><TH>Name</TH><TH>Length (bp)</TH><TH>Host Organism</TH><TH>Target Organism</TH></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2018010" target="_blank">K2018010</A></TD><TD>Laterosporulin</TD><TD>352</TD><TD><I>Brevibacillus</I></TD><TD><I>Staphylococcus aureus &amp; Pseudonomas aeruginosa</I></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2018011" target="blank">K2018011</A></TD><TD>ThuricinS</TD><TD>259</TD><TD><I>Bacillus thuringiensis</I></TD><TD><I>Pseudonomas aeruginosa, Enterobacter Cloacae</I></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2018012" target="blank">K2018012</A></TD><TD>LacticinQ</TD><TD>361</TD><TD><I>Lactococcus lactis QU5</I></TD><TD><I>Staphylococcus aureus</I></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2018014" target="_blank">K2018014</A></TD><TD>Laterusporulin-ThuricinS</TD><TD>415</TD><TD><I>Brevibacillus, Bacillus thuringiensis</I></TD><TD><I>-</I></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2018015" target="blank">K2018015</A></TD><TD>LacticinQ-LacticinZ</TD><TD>529</TD><TD><I>Lactococcus lactis QU5, Lactococcus lactis</I></TD><TD><I>-</I></TD></TR><TR><TD><A href="http://parts.igem.org/Part:BBa_K2018019" target="blank">K2018019</A></TD><TD>PyocinS5</TD><TD>1696</TD><TD><I>Pseudonomas aeruginosa</I></TD><TD><I>Staphylococcus aureus &amp; Pseudonomas aeruginosa</I></TD></TR></TBODY></TABLE></DIV><H5>The Purification of Bacteriocins </H5><P>To purify our bacteriocins, we used the <A href="https://static.igem.org/mediawiki/2016/d/d9/T--SDU-Denmark--iGEM2016SOP0025_v01-Impact_Purification.pdf" target="_blank">IMPACT method</A> (Intein Mediated Purification with an Affinity Chitin-binding Tag). The IMPACT method has the ability to purify native proteins in a single chromatographic step without the use of proteases. We use the IMPACT vector pTXB1 in order to fuse our bacteriocin gene with an intein tag embedded in the pTXB1 plasmid that allow affinity purification. We add IPTG (Isopropyl β-D-1-thiogalactopyranoside) to induce the expression of our intein-bacteriocin fusion. The intein-bacteriocin protein is then extracted by use of French press that disrupt the bacterial cell membrane. The cell lysates are poured onto prepared chitin beads that allow affinity chitin-binding of the intein-bacteriocin fusion. By use of a cleavage buffer containing a thiol-reagent, we induce cleavage between the intein tag and the bacteriocin protein. This results in native bacteriocin elutes.</P><H5>IMPACT Method (<A href="#overview1">Top</A>)</H5><DIV><DIV style="float:right;width:50%;padding-left:15px;"><DIV id="openModal" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 1 illustrates the removal of the hybrid bacteriocin Laterosporulin-ThuricinS from the pSB1C3 vector into the pTXB1 vector. This is schematically illustrated via SnapGene History.</P></DIV></DIV><P class="figuretext"><I>Figure 1 illustrates the removal of the hybrid bacteriocin Laterosporulin-ThuricinS from the pSB1C3 vector into the pTXB1 vector. This is schematically illustrated via SnapGene History.</I></P></DIV><P>For the IMPACT procedure, the vector pTXB1 was used. The vector pTXB1 contains a mini-intein (Mxe GyrA intein) that has been modified to undergo thiol-induced cleavage at its N-terminus. At its C-terminus the intein tag contains a chitin binding domain (CBD) that allows affinity purification on a chitin column 
<A href="https://www.neb.com/~/media/Catalog/All-Products/21A73B351DD24F94BC584FAED2A83A0F/Datacards%20or%20Manuals/manualE6901.pdf" target="_blank">(IMPACT Kit)</A>. The target protein - in this case our bacteriocins - is fused at its C-terminus to the N-terminus of the intein tag marked by the MCS (Multiple cloning site), shown in Figure 1. With the use of a cleavage buffer that contains dithiothreitol (DTT), the vector allows protein purification of a target protein, by thiol-induced cleavage, at the site between the intein tag and the target protein. The results were obtained through cloning into NdeI and SapI sites, which allow the target proteins to be cleaved without any residual amino acids. In order to clone the bacteriocins into the pTXB1 vector from pSB1C3, specific primers were designed to add compatible restriction sites that included the restriction sites for NdeI and SapI.</P></DIV><H5>Expression of Bacteriocins by the IMPACT Plasmid pTXB1 and the <EM>E. coli</EM> Strain ER2566 </H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="openModal2" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 2 illustrates the effect of IPTG induction on DNA level. The effect of IPTG induction on the ER2566 genome are shown on the left and the effect on the pTXB1 are shown on the right. Thus illustrating how the ER2566 genome and pTXB1 act in concert when induced with IPTG.</P></DIV></DIV><P class="figuretext"><I>Figure 2 illustrates the effect of IPTG induction on DNA level. The effect of IPTG induction on the ER2566 genome are shown on the left and the effect on the pTXB1 are shown on the right. Thus illustrating how the ER2566 genome and pTXB1 act in concert when induced with IPTG. </I></P></DIV><P>The <I>E. coli</I> strain ER2566 were used for the expression of the target gene cloned into pTXB1. ER2566 carries a copy of the T7 RNA polymerase gene inserted into the <I>LacZ</I> gene. The T7 RNA polymerase is thereby under the control of the lac promoter. The pTXB1 contains a T7 promoter in front of our target gene and is therefore dependent on the presence of T7 RNA polymerase. To induce protein expression in ER2566, IPTG is added. IPTG binds the <I>lacl</I> repressor located on the pTXB1 plasmid and alter its conformation allosterically. The event prevents the repression of the β-galactosidase, coding the <I>LacZ</I> gene in ER2566 and thus induces the transcription of T7 RNA polymerase (Figure 2). By addition of IPTG we allow <I>E. coli</I> to transcribe the T7 RNA polymerase through the <I>LacZ gene</I>, thereby indirectly induces controlled expression of our target gene, through binding of the T7 promoter.</P><P>In addition IPTG is not hydrolyzed by β-galactosidase encoded by the <I>LacZ</I> gene as it contains a 
<A href="http://www.sigmaaldrich.com/catalog/product/sial/i6758?lang=en&amp;region=DK" target="_blank">sulfur atom</A> in its structure, thereby diverges from allolactose, which is the natural inducer of the lac operon and substrate to hydrolysis by β-galactosidase. The diverge structure of IPTG  prevents the cell from degrading the IPTG. The concentration of the inducer remains constant in the experiment, which allow the expression of lac-controlled genes not to be inhibited <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3575911/" target="_blank">Juers, D. H., Matthews, B. W., &amp; Huber, R. E. (2012). LacZ β-galactosidase: Structure and function of an enzyme of historical and molecular biological importance. Protein Science : A Publication of the Protein Society, 21(12), 1792–1807.</A></SPAN></SPAN>.</P></DIV><H5>Bacteriocin Extraction (<A href="#overview1">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="openModal3" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 3 shows that we extract our bacteriocins from ER2566 by using French press followed by thiol induced cleavage in order to purify the bacteriocins.</P></DIV></DIV><P class="figuretext"><I>Figure 3 shows that we extract our bacteriocins from ER2566 by using French press followed by thiol induced cleavage in order to purify the bacteriocins. </I></P></DIV><P>To extract our protein-Intein from the ER2566 cells after induction with IPTG, we used a <A href="http://www.thermo.com.cn/Resources/200802/productPDF_17636.pdf" target="_blank">French press</A> to destroy the bacterial cell membrane. The lysate were then poured onto prepared chitin beads. Cleavage buffer containing DTT was added, thus causing thiol-induced cleavage at the fusion between the intein and our target protein (Figure 3).</P></DIV><H5>Determining Bacteriocin Concentration and Effect (<A href="#overview1">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="openModal4" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 4 shows a graph of Bradford Standard Protein assay performed with BSA. The concentration [µg/mL] of BSA are plotted on the x-axis and measured OD values are plotted on the y-axis. The equation has been made by linear regression of the plotted data.</P></DIV></DIV><P class="figuretext"><I>Figure 4 shows a graph of Bradford Standard Protein assay performed with BSA. The concentration [µg/mL] of BSA are plotted on the x-axis and measured OD values are plotted on the y-axis. The equation has been made by linear regression of the plotted data. </I></P></DIV><P>The concentration of our bacteriocins were determined according to a <A href="https://static.igem.org/mediawiki/2016/c/cb/T--SDU-Denmark--iGEM2016SOP0028_v01-BradfordProteinConcentrationDetermination.pdf" target="_blank">Bradford Standard protein assay</A> (Figure 4). We created a Bradford protein standard curve of which the protein concentration were calculated according to the measured absorbance. To view final protein concentrations go to 
<A href="https://2016.igem.org/Team:SDU-Denmark/Demonstrate#determine" target="_blank">Demonstration &amp; Results</A>.</P></DIV><H5>Minimum Inhibitory Concentration (MIC) Test (<A href="#overview1">Top</A>)</H5><P><A href="https://static.igem.org/mediawiki/2016/a/a8/T--SDU-Denmark--iGEM2016SOP0027_v01-MIC.pdf" target="_blank">MIC tests</A> are essential in order to establish the bacteriocins viability as an antimicrobial compound. The MIC test studies the in vitro susceptibility of a bacterial strain towards a compound with an antimicrobial effect. This is measured by the optical density of an overnight culture after 28 hours incubated at 37°C, with a specific concentration of a bacteriostatic and/or bactericidal compound. We wanted to perform a MIC test towards <I>P. aeruginosa</I> and MRSA strains. This was inspired from 
<A href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices#Hans" target="_blank">Hans Jørn Kolmos</A> (professor in clinical microbiology), who suggests that bacteriocins could be potent to treat MRSA carriers. In collaboration with the 
<A href="https://2016.igem.org/Team:Stockholm" target="_blank">iGEM Stockholm team 2016</A>
, MIC tests were performed in order to compare the effect of our bacteriocins. The 
<A href="https://2016.igem.org/Team:Stockholm" target="_blank">iGEM Stockholm team 2016</A> 
performed a MIC test on cell lysates, whereas we performed a MIC on our purified proteins. By collaborating with the 
<A href="https://2016.igem.org/Team:Stockholm" target="_blank">iGEM Stockholm team 2016</A> 
we thus got an indication of which of our bacteriocins elicited a significant effect towards chosen target strains. The MIC plates were incubated for 28 hours. To see the results of MIC test, go to 
<A href="https://2016.igem.org/Team:SDU-Denmark/Demonstrate#effect" target="_blank">Demonstration &amp; Results</A>.</P><BUTTON class="accordion"><B>Summary</B></BUTTON><DIV id="foo" class="panel"><P style="text-align: center;">Design of primers compatible to bacteriocin/pSB1C3 in order to add restriction sites for Nde1 and Sap1</P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;">Phusion PCR with designed Forward and Reverse primers for the respective devices (K20180XX/pSB1C3)</P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;">Fast digest of pPCR products + vector pTXB1 with Nde1 and Sap1</P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;">Ligation with Fast Digest products + vector pTXB1</P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;">Transformation into <I>E. coli</I> TOP10</P><P style="text-align: center;">↓</P><P style="text-align: center;">Colony PCR with Impact_forward and Impact_reverse primers  </P><P style="text-align: center;">↓</P><P style="text-align: center;">Plasmid purification of ON cultures from cPCR templates</P><P style="text-align: center;">↓</P><P style="text-align: center;">Transformation of purified bacteriocin/pTXB1 plasmid into <I>E. coli</I> ER2566</P><P style="text-align: center;">↓</P><P style="text-align: center;">ON of cultures → 1 L culture</P><P style="text-align: center;">↓</P><P style="text-align: center;">IPTG protein induce</P><P style="text-align: center;">↓</P><P style="text-align: center;">French Press</P><P style="text-align: center;">↓</P><P style="text-align: center;">Chitin beads and cleavage buffer containing DTT</P><P style="text-align: center;">↓</P><P style="text-align: center;">Concentration determination through Bradford Protein Standard Assay (Verification)</P><P style="text-align: center;">↓</P><P style="text-align: center;">MIC test of the purified bacteriocins effect on different strains</P></DIV><H3>Creating Silk Monomers and Hybrid-Silk by Iterative Capped Assembly (ICA)</H3><H5>The ICA method (<A href="#overview1">Top</A>)</H5><P>The <A href="https://static.igem.org/mediawiki/2016/b/b3/T--SDU-Denmark--SOP0026_v01-ICA.pdf" target="_blank">ICA method</A> was described as a solid surface-based sequential ligation, which adds DNA constructs with matching overhangs individually in series <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="http://www.ncbi.nlm.nih.gov/pubmed/22740649" target="_blank">Briggs, A. W., et al. (2012). &quot;Iterative capped assembly: rapid and scalable synthesis of repeat-module DNA such as TAL effectors from individual monomers.&quot; Nucleic Acids Res 40(15)</A></SPAN></SPAN>. This makes it possible to custom design genes, which can later be amplified through PCR. The method is especially effective when making repetitive DNA constructs. Traditionally DNA is added to a gene through primer overhangs in a PCR, but when working with repetitive genes, the primer could recognize multiple sites and thereby cause multiple variations of the gene. The stepwise ligation method utilizes the strong chemical binding between streptavidin and biotin. The concept is to use magnetic beads coated in streptavidin, while the initiator oligo contains a biotinylated end at the 5’. This allows the magnetic beads to be continuously isolated from the solution with a magnetic rack and the supernatant can be removed. This controls a stepwise ligation construct that ends with a terminator sequence with one sticky-end that eliminates future ligations. Correct ligations will be controlled by the use of cap-sequences, which bind to the previous step's non-ligated constructs and end the possibility of further ligations at the failed ligations. In the end, the gene construct will consist of an initiator sequence, a coding region and a terminator. The initiator and terminator have specific primer sites, which makes it possible for pPCR of the whole synthesized gene sequence. The ICA method was used by the <A href="https://2015.igem.org/Team:UCLA/Project/Functionalizing_Silk" target="_blank">iGEM UCLA team 2015</A> to synthesize silk. During our experiment the used initiator contained a standard iGEM prefix and the terminator contained a standard iGEM suffix. These sites made it possible for the gene sequence to be inserted into desired vectors. </P><VIDEO autoplay="" controls="" loop="" muted="" style="margin-left:10%;width:80%;"><H5>Assembling Silk Gene Fragments (<A href="#overview1">Top</A>)</H5><P>Silk from <I>Nephila clavipes</I> consists of the genes MaSp1 and MaSp2, which individually contains repetitive gene sequences. The genes have the same restriction sites in both ends so that the overhangs are consistent throughout the construct. This step by step ligation of the individual genes (e.g. MaSp1 AB + MaSp1 BC + MaSp1 CA), allows for custom designs of the silk gene both in length and in variation (e.g. 2xMaSp1:1xMaSp2). For a graphical illustration of the ICA Method, check out the video above. </P><P>The ICA method gives way for alternative constructs, where other genes can be incorporated in between the repetitive silk genes. This freedom of construction creates the opportunity for incorporation of bacteriocin genes into the silk construct, using the ICA method. <A href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices#Coloplast" target="_blank">Coloplast</A> are already aware of the great qualities of recombinant spider silk and bacteriocins, however the idea of combining the two elements are completely new to them and they see a great oppertunity in it.</P><BUTTON class="accordion"><B>Summary</B></BUTTON><DIV id="foo" class="panel"><P style="text-align: center;"><SPAN style="font-weight: 400;">pPCR of the ordered geneblocks of MaSp1 and MaSp2 (AB/BC/CA)</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">FastDigest of pPCR products with Eco31I  </SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">ICA of the digested products</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">PCR of the ICA-product</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">////</SPAN></P><P style="text-align: center;"><STRONG><SPAN style="font-weight: 400;">We tried our best but we did not succeed in creating a silk construct. </SPAN></STRONG></P></DIV><H3> Working with PHB</H3><P>In the lab we wanted to start out by 1) creating a library of promoter hybrids to determine an optimal construct for the production of PHB, 2) determine the ideal purifaction method, 3) increase the production of PHB by using panK and characterize the effect of panK on the proteome, 4) have the cells secrete PHB to ease purification efforts, and 5) 3D print the recovered PHB. </P><H5>PHB Production and Extraction (<A href="#overview1">Top</A>)</H5><P>Retrieving a pure sample of PHB from the cellular cytoplasm has proven to be an expensive task. We have tested four methods of extraction in order to determine, which extraction method is optimal for large scale production of PHB, considering pricing, environmental strain, required equipment, toxicity, yield and purity. </P><H5>Digestion with Hypochlorite (<A href="#overview1">Top</A>)</H5><P>A method for extracting biodegradable plastic, is simply <A href="https://static.igem.org/mediawiki/2016/8/88/T--SDU-Denmark--SOP0044_v01_Hypochlorite_PHB_recovery.pdf" target="_blank">digesting the cellular components with hypochlorite</A>, where it dissolves the cellular components and leaves the polyester behind. The advantages of using hypochlorite as opposed to digestion with surfactants (e.g. sodium dodecyl sulfate (SDS)), is that it dissolves the membrane. Thus a single treatment can provide high purity and yield of PHB. After treatment with PHB the samples were centrifuged with 4000xG. A small side effect of hypochlorite is that it causes degradation of the PHB granules that greatly reduce the molecular weight of the molecule and thereby losing some PHB <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="http://ul2nb3kn6e.search.serialssolutions.com.proxy1-bib.sdu.dk:2048/?url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.atitle=Isolation%20and%20purification%20of%20bacterial%20poly%20%283-hydroxyalkanoates%29&amp;rft.aufirst=Nicolas&amp;rft.aulast=Jacquel&amp;rft.date=2008&amp;rft.epage=27&amp;rft.genre=article&amp;rft.issn=1369-703X&amp;rft.issue=1&amp;rft.jtitle=BIOCHEMICAL%20ENGINEERING%20JOURNAL&amp;rft.pages=15-27&amp;rft.spage=15&amp;rft.stitle=BIOCHEM%20ENG%20J&amp;rft.volume=39&amp;rfr_id=info:sid/www.isinet.com:WoK:WOS&amp;rft.au=Lo%2C%20Chi-Wei&amp;rft.au=Wei%2C%20Yu-Hong&amp;rft.au=Wu%2C%20Ho-Shing&amp;rft.au=Wang%2C%20Shaw%20S%2E&amp;rft_id=info:doi/10%2E1016%2Fj%2Ebej%2E2007%2E11%2E029" target="_blank">Jacquel, N., et al. (2008). &quot;Isolation and purification of bacterial poly(3-hydroxyalkanoates).&quot; Biochemical Engineering Journal 39(1): 15-27.</A></SPAN></SPAN>,<SPAN class="tooltip"><SPAN class="tooltiptext"><A href="http://amb-express.springeropen.com/articles/10.1186/2191-0855-2-59" target="_blank">Heinrich, D., et al. (2012). &quot;Large scale extraction of poly(3-hydroxybutyrate) from Ralstonia eutropha H16 using sodium hypochlorite.&quot; AMB Express 2(1): 59</A></SPAN></SPAN>.</P><H5>Solvent Extraction (<A href="#overview1">Top</A>)</H5><P>Another method of  PHB extraction is treating lyophilized biomass with a solvent in which PHB is soluble. Removing the remaining biomass by filtration and then precipitating the PHB will provide a sample with high purity. We have tested two different methods of solvent extraction with chlorinated agents.</P><UL class="list"><LI><A href="https://static.igem.org/mediawiki/2016/2/26/T--SDU-Denmark--iGEM_2016_SOP0034_v1_Chloroform_PHB_recovery.pdf" target="_blank">Chloroform</A> is the most toxic agent we have used for extraction, but it provides up to 99 % pure PHB <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="http://ul2nb3kn6e.search.serialssolutions.com.proxy1-bib.sdu.dk:2048/?url_ver=Z39.88-2004&amp;url_ctx_fmt=info:ofi/fmt:kev:mtx:ctx&amp;rft_val_fmt=info:ofi/fmt:kev:mtx:journal&amp;rft.atitle=EXTRACTION%20OF%20POLY-3-HYDROXYBUTYRATE%20USING%20CHLORINATED%20SOLVENTS&amp;rft.aufirst=JA&amp;rft.aulast=RAMSAY&amp;rft.date=1994&amp;rft.epage=594&amp;rft.genre=article&amp;rft.issn=0951-208X&amp;rft.issue=8&amp;rft.jtitle=BIOTECHNOLOGY%20TECHNIQUES&amp;rft.pages=589-594&amp;rft.spage=589&amp;rft.stitle=BIOTECHNOL%20TECH&amp;rft.volume=8&amp;rfr_id=info:sid/www.isinet.com:WoK:WOS&amp;rft.au=BERGER%2C%20E&amp;rft.au=VOYER%2C%20R&amp;rft.au=CHAVARIE%2C%20C&amp;rft.au=RAMSAY%2C%20BA&amp;rft_id=info:doi/10%2E1007%2FBF00152152" target="_blank">Ramsay, J. A., et al. (1994). &quot;Extraction of poly-3-hydroxybutyrate using chlorinated solvents.&quot; Biotechnology Techniques 8(8): 589-594.</A></SPAN></SPAN>.</LI><LI>The <A href="https://static.igem.org/mediawiki/2016/2/22/T--SDU-Denmark--iGEM_2016_SOP0032_v1_-_Ethylacetate_PHB_recovery.pdf" target="_blank">Ethyl acetate</A> extration method involves the least toxic substances and requires no pretreatment, but fairly large amounts of chemicals are needed for the extraction, which provides a purity of 97-99 % <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pubmed/26198125" target="_blank">Aramvash, A., et al. (2015). &quot;An Environmentally Friendly and Efficient Method for Extraction of PHB Biopolymer with Non-Halogenated Solvents.&quot; J Microbiol Biotechnol 25(11): 1936-1943.</A></SPAN></SPAN>.</LI></UL><H5>Evaluation of Extraction Methods (<A href="#overview1">Top</A>) </H5><P>We use Proton Nuclear Magnetic Resonance (HNMR) in order to determine the purity of PHB extracted with the different methods. HNMR is a method in which the structure of a molecule is analyzed in a magnetic spectrum. The spin of different hydrogen atoms allows us to determine the amount of and study each distinct type of hydrogen atoms. By analyzing such a spectrum, the content of the sample can be determined. 
For details on how we performed HMNR analysis, see protocol <A href="https://static.igem.org/mediawiki/2016/f/ff/T--SDU-Denmark--iGEM2016_01_HNMR.pdf" target="_blank">HNMR - Measurement of plast purity</A>.</P><H5>Developing a Secretion System (<A href="#overview1">Top</A>)</H5><DIV><DIV style="float:right;width:60%;padding-left:15px;"><DIV id="openModal5" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 5 shows an illustration of our modified type I secretion system. The figure illustrates how PHB are secreted due to the recognition of HlyA.</P></DIV></DIV><P class="figuretext"><I>Figure 5 shows an illustration of our modified type I secretion system. The figure illustrates how PHB are secreted due to the recognition of HlyA. </I></P></DIV><P style="clear:left">In order to avoid the high costs of plastic extraction we are utilizing a type I secretion pathway for the toxin, hemolysin, to make the bacteria secrete the plastic itself. The secretion system requires only two transport proteins, hemolysin B and hemolysin D. Together these proteins are responsible for the secretion of hemolysin in pathogenic strains of <I>E. coli</I>. Since PHB is not a protein compound, applying the system directly, is not possible. Instead we bio-fused a domain of the toxin hemolysin A with a protein called phasin. Phasin is a protein used in <I>R. eutropha</I> that binds to PHB granules in order to regulate their size. Hemolysin A is the exotoxin secreted by hemolysin B and hemolysin D. By using only a short signal domain of the protein, we can tag the bio-fused protein for secretion without making it toxic. The fused phasin-hemolysin protein will keep its ability to attach to the surface of PHB granulates, through non-covalent interaction and thereby target the PHB for secretion <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pubmed/24139229" target="_blank">RAHMAN, A., LINTON, E., HATCH, A. D., SIMS, R. C. &amp; MILLER, C. D. 2013. Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach. Journal of Biological Engineering, 7.</A></SPAN></SPAN>. The event will reduce the use of toxic and environmentally straining chemicals and eventually it could lead to a continuous system. In addition phasin has low binding specificity, thus the system can be used for other PHA’s (polyhydroxyalkanoates) <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pubmed/17074898">Hanisch, J., et al. (2006). &quot;The Ralstonia eutropha H16 phasin PhaP1 is targeted to intracellular triacylglycerol inclusions in Rhodococcus opacus PD630 and Mycobacterium smegmatis mc2155, and provides an anchor to target other proteins.&quot; Microbiology 152(Pt 11): 3271-3280.</A></SPAN></SPAN>. The idea of a secretion system has already been explored by the <A href="https://2009.igem.org/Team:Utah_State" target="_blank">iGEM Utah Team 2009</A> and have been processed in earlier studies from which we were inspired to develop our secretion system <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pubmed/24139229" target="_blank">RAHMAN, A., LINTON, E., HATCH, A. D., SIMS, R. C. &amp; MILLER, C. D. 2013. Secretion of polyhydroxybutyrate in Escherichia coli using a synthetic biological engineering approach. Journal of Biological Engineering, 7.</A></SPAN></SPAN>. <A href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices#Plastic" target="_blank">Plastic companies</A> are very interested in use of PHB as the material in their products, but are not doing it currently because of the price of the PHB production. Using our secretion system it would be possible optimize the production of PHB, making it more attractive for the respective companies.
For further details check out the <A href="https://static.igem.org/mediawiki/2016/6/63/T--SDU-Denmark--iGEM2016_01_K2018050.pdf" target="_blank">assembly of the secretion system</A>.</P></DIV><H5>Pantothenate Kinase II (panK) (<A href="#overview1">Top</A>)</H5><P>PanK is a pantothenate kinase that activates the first protein in the anabolic pathway for coenzyme A. <I>E. coli</I> has this protein as part of its core genome, but the kinase from <EM>E. coli</EM> is strongly regulated by a negative feedback mechanism.  However, an increase in coenzyme A and its thioesters will inhibit the activity of pantothenate kinase. <I>Staphylococcus aureus</I>  requires a higher amount of coenzyme A as it lacks glutathione and thus relies on a coenzyme A based reductase system to repair oxidative damage <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="https://www.ncbi.nlm.nih.gov/pubmed/15548531" target="_blank">Leonardi, R., et al. (2005). &quot;A pantothenate kinase from Staphylococcus aureus refractory to feedback regulation by coenzyme A.&quot; J Biol Chem 280(5): 3314-3322.</A></SPAN></SPAN>.
As a results the staphylococcal panK is not feedback regulated, and we have thereby taken advantage of this by adding the specific panK to our construct. The BioBrick (<A href="http://parts.igem.org/Part:BBa_K1692020" target="_blank">K1692020</A>) containing staphylococcal panK was created by the <A href="https://2015.igem.org/Team:Stanford-Brown" target="_blank">iGEM Standford Brown team 2015</A>.</P><H5>Analysis of <I>E.coli</I> Proteome in panK's Expressing Cells (<A href="#overview1">Top</A>)</H5><DIV style="float:right;width:50%;padding-left:15px;"><DIV id="openModal6" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 6 shows a short overview of the steps involved in iTRAQ analysis. This includes protein extraction, digestion, labeling, Titanium dioxid purification, HILIC fractionation and Mass spec analysis. The figure is derived from a protocol created by the Protein Research Group at the Southern University Denmark. </P></DIV></DIV><P class="figuretext"><I>Figure 6 shows a short overview of the steps involved in iTRAQ analysis. This includes protein extraction, digestion, labeling, Titanium dioxid purification, HILIC fractionation and Mass spec analysis. The figure is derived from a protocol created by the Protein Research Group at the Southern University Denmark. </I></P></DIV><P>A protein from one organism do not necessarily recognize all its native target proteins in another organism. Additionally, it could be able to phosphorylate proteins, which it did not phosphorylate before. We have therefore performed an <A href="https://static.igem.org/mediawiki/2016/6/6f/T--SDU-Denmark--iGEM_2016_SOP0037_v1_-_iTRAQ_sample_preparation.pdf">iTRAQ analysis</A> of staphylococcal panK (pantothenate kinase II) in <I>E. coli</I>. For an <A href="https://static.igem.org/mediawiki/2016/6/6f/T--SDU-Denmark--iGEM_2016_SOP0037_v1_-_iTRAQ_sample_preparation.pdf">iTRAQ analysis</A>, the entire proteome of the cells are extracted and digested. The peptides from each sample are then tagged with one of four different tags that binds to the free N-terminal present on tryptic peptides. The peptides are then mixed in an equal ratio, securing that the same amount of peptides are present from each sample. The phosphorylated peptides that are especially interesting for the analysis of a kinase, are isolated and purified by a TiO<SUB>2</SUB> purification. Lastly the non-modified peptides and the phosphopeptides are analysed separately by mass spectrometry (Figure 6). </P><P>In the mass spectrometer the different iTRAQ tags will fragmentize differently. Therefore while analysing a single spectrum, one can compare the amounts of the proteins from the different samples, simply by comparing the intensity of the reporter ions from the iTRAQ tag. The computer can thus analyse the entire proteome of four bacterial strains at once, identifying interesting differences caused by pantothenate kinase II.</P><H5>Determination of PHB Production using Flow Cytometry (<A href="#overview1">Top</A>)</H5><P>In order to determine the effectiveness of our different PHB producing strains we used <A href="https://static.igem.org/mediawiki/2016/4/46/T--SDU-Denmark--iGEM2016_01_FACS_comparison_of_cells.pdf" target="_blank">nile red staining combined with flow cytometry</A>. Nile red binds to PHB, and as the substance is fluorescent, it can be used as a quantitative measure for the amount of plastic present in a cell <SPAN class="tooltip"><SPAN class="tooltiptext"><A href="http://aem.asm.org.proxy1-bib.sdu.dk:2048/content/72/5/3412.full.pdf+html" target="_blank">Leonardi, R., et al. (2005). &quot;A pantothenate kinase from Staphylococcus aureus refractory to feedback regulation by coenzyme A.&quot; J Biol Chem 280(5): 3314-3322.</A></SPAN></SPAN>. By combining nile red staining with flow cytometry, the average amount of plastic inside every cell from different strains can be compared and thus generate a relative measure ability of the respective strains to produce plastic.</P><H5>Filament making - and 3D Printing with PHB (<A href="#overview1">Top</A>)</H5><DIV class="row"><DIV class="col-sm-9"><DIV id="experiment1" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 7 shows the FilaStruder that made our filaments.</P></DIV></DIV><P class="figuretext"><EM>Figure 7 shows the FilaStruder that made our filaments. </EM></P></DIV><DIV class="col-sm-3"><DIV id="experiment2" class="modalDialog" style="overflow: auto;max-height: 100vh;"><DIV><A href="#closepop" title="Close" class="closepop">X</A><P>Figure 8 shows the 3D printer Hyrel System 30M, which we used to create a piece of a jaw implant from our extracted PHB. </P></DIV></DIV><P class="figuretext"><EM> Figure 8 shows the 3D printer Hyrel System 30M, which we used to create a piece of a jaw implant from our extracted PHB. </EM></P></DIV><P style="clear:left;">The <A href="http://www.hyrel3d.com" target="_blank">HYREL SYSTEM 30M</A> 3D printer requires filaments with a diameter of 1.75mm. In order to 3D print our produced PHB, we needed to create filaments made by a <A href="https://static.igem.org/mediawiki/2016/9/99/T--SDU-Denmark--Manual_for_Filastruder_ENGLISH.pdf" target="_blank">filament maker</A>. The Filastruder was <A href="https://www.youtube.com/watch?v=OHft4c4SxWQ" target="_blank">assembled</A> and tested with PLA prior to use with bacterially produced PHB. </P><P>All of our 3D printing was performed on a <A href="http://www.hyrel3d.com" target="_blank">HYREL SYSTEM 30M</A>. The 3D model was created as an STL file and tested with PLA filaments. PHB was ordered to compare constructs containing 100% pure PHB and constructs containing our purified plastic. <A href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices#Morten" target="_blank">Morten Østergaard Andersen</A> suggested that PHB could be used as an implant itself, in cases where e.g. the bone needs a scaffold until it has healed. This further support the idea of trying to 3D print PHB. </P></DIV><BUTTON class="accordion"><B>Summary</B></BUTTON><DIV id="foo" class="panel"><P style="text-align: center;">Creating a library of PHB with hybrid promoters</P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Comparison of PHB producing BioBricks</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Extraction of PHB</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Comparison of extraction methods</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Assembly of secretion system</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Addition of pantothenate kinase II BioBrick to the most efficient PHB producing BioBrick</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Addition of the secretion system the the most efficient PHB producing BioBrick</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">↓</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;"> Comparison of new BioBricks with previous BioBricks</SPAN></P><P style="text-align: center;"><STRONG>↓</STRONG></P><P style="text-align: center;"><SPAN style="font-weight: 400;"> Characterization of pantothenate kinase with iTRAQ</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">↓</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Determination of bacterial PHB content with gas chromatography</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">↓</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Analysis of purified material</SPAN><SPAN style="font-weight: 400;"> with HNMR</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">↓</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">Mass production of plastic </SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">↓</SPAN></P><P style="text-align: center;"><SPAN style="font-weight: 400;">3D printing</SPAN></P></DIV><H3>Protocols</H3><BUTTON class="accordion">Bacteriocins</BUTTON><DIV id="foo" class="panel"><UL class="list"><LI><A href="https://static.igem.org/mediawiki/2016/9/9c/T--SDU-Denmark--K2018010-CloningcompositepartintoiGEMstandardplasmidpSB1C3.pdf" target="_blank"> Cloning composite part K2018010 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/ea/T--SDU-Denmark--K2018011clonedintopSB1C3.pdf" target="_blank">Cloning composite part K2018011 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/09/T--SDU-Denmark--K2018012-CloningcompositepartintoiGEMstandardplasmid.pdf" target="_blank">Cloning composite part K2018012 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/bf/T--SDU-Denmark--K2018014-CloningcompositepartintoiGEMstandardplasmid.pdf" target="_blank">Cloning composite part K2018014 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/cc/T--SDU-Denmark--K2018015-CloningcompositepartintoiGEMstandardplasmid.pdf" target="_blank">Cloning composite part K2018015 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/5/5a/T--SDU-Denmark--K2018019-CloningcompositepartintoiGEMstandardplasmid.pdf" target="_blank">Cloning composite part K2018019 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/a6/T--SDU-Denmark--IMPACTPurificationofK2018010.pdf" target="_blank">IMPACT Purification of K2018010</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/03/T--SDU-Denmark--IMPACTPurificationofK2018011.pdf" target="_blank">IMPACT Purification of K2018011</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/a0/T--SDU-Denmark--IMPACTPurificationofK2018012.pdf" target="_blank">IMPACT Purification of K2018012</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/f/f6/T--SDU-Denmark--IMPACTPurificationofK2018014.pdf" target="_blank">IMPACT Purification of K2018014</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/66/T--SDU-Denmark--IMPACTPurificationofK2018015.pdf" target="_blank">IMPACT Purification of K2018015</A></LI></UL></DIV><BUTTON class="accordion">Silk</BUTTON><DIV id="foo" class="panel"><UL class="list"><LI><A href="https://static.igem.org/mediawiki/2016/1/12/T--SDU-Denmark--CloningThuricinwithDAsilk-overhangsintopSB1C3.pdf" target="_blank">Cloning ThuricinS with DA silk-overhangs into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/0d/T--SDU-Denmark--CloningThuricinwithDEsilk-overhangsintopSB1C3.pdf" target="_blank">Cloning ThuricinS with DE silk-overhangs into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/5/52/T--SDU-Denmark--CloningThuricinwithEAsilk-overhangsintopSB1C3.pdf" target="_blank">Cloning ThuricinS with EA silk-overhangs into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b1/T--SDU-Denmark--CloningThuricinwithEDsilk-overhangsintopSB1C3.pdf" target="_blank">Cloning ThuricinS with ED silk-overhangs into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/2/27/T--SDU-Denmark--K1763002-CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763002 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/c3/T--SDU-Denmark--K1763003-CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763003 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/e5/T--SDU-Denmark--K1763004_CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763004 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/4/44/T--SDU-Denmark--K1763009_CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763009 into  pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b9/T--SDU-Denmark--K1763010-CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763010 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/4/42/T--SDU-Denmark--K1763011-CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763011 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/8/80/T--SDU-Denmark--K1763012-CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K1763012 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/3/3f/T--SDU-Denmark--K2018043-CloningBasicpartintoiGEMstandardplasmidpSB1C3.pdf" target="_blank">Cloning Basic part K2018043 into pSB1C3</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/1/19/T--SDU-Denmark--K2018044-CloningBasicpartintoiGEMstandardplasmidpsB1C3.pdf" target="_blank">Cloning Basic part K2018044 into pSB1C3</A></LI></UL></DIV><BUTTON class="accordion">Plastic</BUTTON><DIV id="foo" class="panel"><LI><A href="https://static.igem.org/mediawiki/2016/0/05/T--SDU-Denmark--iGEM2016_01_Gas_chromatography.pdf" target="_blank">Gas Chromatography - Characterizing and optimizing PHB production</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/f/ff/T--SDU-Denmark--iGEM2016_01_HNMR.pdf" target="_blank">HNMR - Measurement of plast purity</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/9b/T--SDU-Denmark--iGEM2016_01_Hybrid_promoter_PHB_library.pdf" target="_blank">Creating a Hybrid promoter PHB library</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/e7/T--SDU-Denmark--iGEM2016_01_iTRAQ_analysis.pdf" target="_blank">iTRAQ analysis - Characterizing and optimizing PHB production in <EM>E.coli</EM></A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/63/T--SDU-Denmark--iGEM2016_01_K2018050.pdf" target="_blank">Creating BioBricks containing the secretion system</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/9d/T--SDU-Denmark--iGEM2016_01_Media_comparison.pdf" target="_blank">Comparing media for growth of plast producing bacterias</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/4/46/T--SDU-Denmark--iGEM2016_01_FACS_comparison_of_cells.pdf" target="_blank">Fluorescence activated cell sorting (FACS) - Comparison of cells</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/99/T--SDU-Denmark--Extraction_of_PHB_with_different_methods.pdf" target="_blank">Extraction of PHB with different methods</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/05/T--SDU-Denmark--Protokol_PHB_extraction_3Dprint.pdf" target="_blank">PHB Extraction following 3D printing</A></LI></DIV><H3>Standard Operational Procedures</H3><BUTTON class="accordion">SOP's</BUTTON><DIV id="foo" class="panel"><UL class="list"><LI><A href="https://static.igem.org/mediawiki/2016/e/ec/T--SDU-Denmark--SOP0001-ONcultureofE.coli.pdf" target="_blank">SOP0001_v01 ON culture of <EM>E.coli</EM></A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/af/T--SDU-Denmark--iGEM2013SOP0005_v02-FreezeStock.pdf" target="_blank">SOP0005_v02 Freeze Stock </A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/c8/T--SDU-Denmark--SOP0007-LAplateswithantibiotics.pdf" target="_blank">SOP0007_v01 LA plates with antibiotics</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/1/10/T--SDU-Denmark--SOP0009-TSBTransformation.pdf" target="_blank">SOP0009_v01 TSB Transformation</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/3/37/T--SDU-Denmark--SOP0010-PhusionPCR.pdf" target="_blank">SOP0010_v01 Phusion PCR </A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/c6/T--SDU-Denmark--SOP0015-Ligation.pdf" target="_blank">SOP0015_v01 Ligation </A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b8/T--SDU-Denmark--SOP0017-FastDigest.pdf" target="_blank">SOP0017_v01 FastDigest</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/3/3a/T--SDU-Denmark--SOP0019-MiniPrep.pdf" target="_blank">SOP0019_v01 MiniPrep</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/f/f9/T--SDU-DENMARK--SOP0021-ColonyPCRwithMyTag.pdf" target="_blank">SOP0021_v01 Colony PCR with My Tag</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/9a/T--SDU-Denmark--iGEM2016SOP0022_v01-Competentcell-freeze-stock.pdf" target="_blank">SOP0022_v01 Competent cell freeze stock</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/ca/T--SDU-Denmark--iGEM2016SOP0023-CaTransformation.pdf" target="_blank">SOP0023_v01 Ca2+ Transformation </A></LI><LI><A href="https://static.igem.org/mediawiki/2016/8/8c/T--SDU-Denmark--iGEM2016SOP0024_v01-SDS-page.pdf" target="_blank">SOP0024_v01 SDS-page </A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d9/T--SDU-Denmark--iGEM2016SOP0025_v01-Impact_Purification.pdf" target="_blank">SOP0025_v01 Impact Purification</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b3/T--SDU-Denmark--SOP0026_v01-ICA.pdf" target="_blank">SOP0026_v01 Iterative Capped Assembly</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/a8/T--SDU-Denmark--iGEM2016SOP0027_v01-MIC.pdf" target="_blank">SOP0027_v01 MIC test</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/c/cb/T--SDU-Denmark--iGEM2016SOP0028_v01-BradfordProteinConcentrationDetermination.pdf" target="_blank">SOP0028_v01 Bradford Protein Concentration Determination</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/7/7b/T--SDU-Denmark--iGEM_2016_SOP0029_v01_-_DISC_diffusion_assay.pdf" target="_blank">SOP0029_v01 DISC diffusion assay</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/3/38/T--SDU-Denmark--iGEM_2016_SOP0030_v1_-_2xYT_media.pdf" target="_blank">SOP0030_v01 2xYT media</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/0/0d/T--SDU-Denmark--iGEM_2016_SOP0031_v1_-_TB_Media.pdf" target="_blank">SOP0031_v01 TB Media </A></LI><LI><A href="https://static.igem.org/mediawiki/2016/2/22/T--SDU-Denmark--iGEM_2016_SOP0032_v1_-_Ethylacetate_PHB_recovery.pdf" target="_blank">SOP0032_v01 Ethylacetate PHB recovery</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/e/ec/T--SDU-Denmark--iGEM_2016_SOP0032_v1_-_Ethylacetate_PHB_recovery.pdfiGEM_2016_SOP0033_v1_-_triton_x-100_and_Hypochlorite_PHB_recovery.pdf" target="_blank">SOP0033_v01 PHB extraction with Triton x-100 and hypochlorite</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/2/26/T--SDU-Denmark--iGEM_2016_SOP0034_v1_Chloroform_PHB_recovery.pdf" target="_blank">SOP0034_v01 Chloroform PHB recovery</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/97/T--SDU-Denmark--iGEM_2016_SOP0035_v1_-_Fluorescence_microscopy.pdf" target="_blank">SOP0035_v01 Fluorescence microscopy</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/b/b5/T--SDU-Denmark--iGEM_2016_SOP0036_v1_–_Nile_red_agar_plates.pdf" target="_blank">SOP0036_v01 Nile red agar plates</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/6f/T--SDU-Denmark--iGEM_2016_SOP0037_v1_-_iTRAQ_sample_preparation.pdf" target="_blank">SOP0037_v01 iTRAQ sample preparation</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d8/T--SDU-Denmark--iGEM_2016_SOP0038_v1_-_Qubit®_Protein_Assay_Kits.pdf" target="_blank">SOP0038_v01 Qubit®Protein Assay Kits</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/61/T--SDU-Denmark--iGEM_2016_SOP0039_v1_-_C8_%26_C18_columns.pdf" target="_blank">SOP0039_v1 Purification with C8 and C18 columns</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/5/54/T--SDU-Denmark--iGEM_2016_SOP0040_v1_-_iTRAQ_labelling.pdf" target="_blank">SOP0040_v01 iTRAQ labelling</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/ac/T--SDU-Denmark--iGEM_2016_SOP0041_v1_-_TiO2_purification.pdf" target="_blank">SOP0041_v01 TiO2 purification</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/6/6b/T--SDU-Denmark--SOP0042_v01_Daily_use_of_FACS_Aria_II.pdf" target="_blank">SOP0042_v01 Daily use of FACS Aria II</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/d/d1/T--SDU-Denmark--SOP0043_v01_Sorting_setup_FACS_Aria_II.pdf" target="_blank">SOP0043_v01 Sorting setup FACS Aria II</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/8/88/T--SDU-Denmark--SOP0044_v01_Hypochlorite_PHB_recovery.pdf" target="_blank">SOP0044_v01 Hypochlorite PHB recovery</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/a/ae/T--SDU-Denmark--iGEM_2016_SOP0045_v1_-_Gas_Chromatography.pdf" target="_blank">SOP0045_v01 Gas Chromatography</A></LI><LI><A href="https://static.igem.org/mediawiki/2016/9/99/T--SDU-Denmark--Manual_for_Filastruder_ENGLISH.pdf" target="_blank">Manual for Filastruder</A></LI></UL></DIV></VIDEO></DIV></DIV></DIV><DIV id="sidewikimenu" class="menu_wrapper col-lg-2 col-lg-push-2 col-md-2 hidden-xs hidden-sm"><DIV class="collapsable_menu_control hidden-xs hidden-sm hidden-md hidden-lg hidden-xl">Menu</DIV><UL id="sideaccordion" class="accordion"><LI class="menu_item" id="side1"><A id="sideone" href="https://2016.igem.org/Team:SDU-Denmark">Home</A></LI><LI class="menu_item" id="side2"><A id="sidetwo" href="https://2016.igem.org/Team:SDU-Denmark/Introduction">Introduction</A></LI><LI><A id="team" href=" https://2016.igem.org/Team:SDU-Denmark/Team" hijax="">Team</A></LI><LI><A id="attributions" href="https://2016.igem.org/Team:SDU-Denmark/Attributions"> 
                                Attributions </A></LI><LI><A id="collaborations" href="https://2016.igem.org/Team:SDU-Denmark/Collaborations">Collaborations</A></LI><LI><A id="achievements" href="https://2016.igem.org/Team:SDU-Denmark/Achievements">Achievements</A></LI></UL><LI class="menu_item" id="side3"><A id="sidethree" href="https://2016.igem.org/Team:SDU-Denmark/Project">Project</A></LI><LI><A id="description" href="https://2016.igem.org/Team:SDU-Denmark/Description">Description 
				</A></LI><LI><A id="experiments" href="https://2016.igem.org/Team:SDU-Denmark/Experiments">Experiments 
				</A></LI><LI><A id="demonstration" href="https://2016.igem.org/Team:SDU-Denmark/Demonstrate">  
				Demonstration &amp; Results</A></LI><LI><A id="proof" href="https://2016.igem.org/Team:SDU-Denmark/Proof">Proof of Concept</A></LI><LI><A id="reproducibility" href="https://2016.igem.org/Team:SDU-Denmark/Reproducibility">Scientific Reproduction</A></LI><LI class="menu_item" id="side4"><A id="sidefour" href="https://2016.igem.org/Team:SDU-Denmark/Parts_procedures">Parts &amp; Procedures</A></LI><LI><A id="parts" href="https://2016.igem.org/Team:SDU-Denmark/Parts">Parts </A></LI><LI><A id="notebook" href="https://2016.igem.org/Team:SDU-Denmark/Notebook">Notebook </A></LI><LI><A id="safety" href="https://2016.igem.org/Team:SDU-Denmark/Safety">Safety </A></LI><LI class="menu_item" id="side5"><A id="sidefive" href="https://2016.igem.org/Team:SDU-Denmark/Practices">Practices &amp; Prospects</A></LI><LI><A id="bioethics" href="https://2016.igem.org/Team:SDU-Denmark/Bioethics">Bioethics</A></LI><LI><A id="integrated" href="https://2016.igem.org/Team:SDU-Denmark/Integrated_Practices"> Integrated Practices </A></LI><LI><A id="outreach" href="https://2016.igem.org/Team:SDU-Denmark/Outreach">Outreach</A></LI><LI><A id="prospects" href="https://2016.igem.org/Team:SDU-Denmark/Prospects">Prospects</A></LI><LI><A id="perspectives" href="https://2016.igem.org/Team:SDU-Denmark/Perspectives">Perspectives</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>