Team:SDU-Denmark/Notebook
Menu
Home
Introduction
Team
Attributions
Collaborations
Achievements
Project
Description
Experiments
Demonstration & Results
Proof of Concept
Scientific Reproduction
Parts & Procedures
Parts
Notebook
Safety
Practices & Prospects
Bioethics
Integrated Practices
Outreach
Prospects
Perspectives
Bacto-Aid
Notebook
Week 10
The team meets for the first time. Date 16.03.09
The team meets for the first time.
Idea generation weekend with the team, in a weekend cottage far, far away from the university. Date 16.03.11 to 16.03.13
Week 13
The project is chosen. Date 16.03.28
The project is chosen - Bacto-Aid!.
The first project description is submitted to the advisors. Date 16.04.01
The first project description is submitted to the advisors.
Week 14
The project is presented for the advisors. Date 16.04.04
The project is presented for the advisors.
Presentation for the danish paticipants in BioOL. Date 16.04.07
Week 15
Laboratory course Date 16.04.14
We learned to orientate in the "new" lab settings, and tried to perform basic experiments as a rehearsal. On the photo is Joel "learning" how to pipet.
Successful transformation of K608002 in to E. coli. Date 16.04.15
The transformation is validated by cPCR.
The following program is used.
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
>15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
The protocol .
Ligation of K608002 and K934001. Date 16.04.19
Ratio: 1:0, 1:2 and 1:5
The protocol .
Week 16
K2018000 is successful transformed to E. coli Top 10. Date 16.04.21
The transformation is validated by cPCR.
The following program is used.
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
>15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
The protocol .
Week 17
Research day at our University. Date 16.04.30
This year the faculty of Biochemistry and Molecular Biology had a big workshop area with the theme “The Good, the Bad and the Ugly” : the probiotic bacteria, the pathogenic bacteria and the multiresistant bacteria.
SB transformation on K2018038 and K2018039, ON cultures of K2018039 and K2018038, miniprep purification and Ca2+ transformation of K2018038 into E. Coli. 16.04.30-16.04.07
See Protocol .
Week 18
Danish Meetup at Danish Technical University. Date 16.05.06-16.05-08
Week 24
Nordic Meetup in Stockholm. Date 16.06.16-16.06-19
Week 26
Received gBlock, K201843, and resuspend. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we have done with the BioBrick K2018043 can be read in the protocol: Cloning Basic part K2018043 into pSB1C3
Received gBlock, K201844, and resuspend. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we have done with the BioBrick K2018044 can be read in the protocol: Cloning Basic part K2018044 into pSB1C3
Received gBlock, K1763002, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763002 can be read in the protocol: Cloning Basic part K1763002 into pSB1C3
Received gBlock, K1763003, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763003 can be read in the protocol: Cloning Basic part K1763003 into pSB1C3
Received gBlock, K1763004, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763004 can be read in the protocol: Cloning Basic part K1763004 into pSB1C3
Received gBlock, K1763009, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763009 can be read in the protocol: Cloning Basic part K1763009 into  pSB1C3
Received gBlock, K1763010, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763010 can be read in the protocol: Cloning Basic part K1763010 into pSB1C3
Received gBlock, K1763011, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763011 can be read in the protocol: Cloning Basic part K1763011 into pSB1C3
Received gBlock, K1763012, and resuspended. Date: 16.06.27
The brick was ordered from IDT and received: The received gblock had a weight of 200 ng. and was resuspended in dH2O.  5 µl diluted gblock and 95 µl ddH2O to make a 20X dilution.
What we had done with the BioBrick K1763012 can be read in the protocol: Cloning Basic part K1763012 into pSB1C3
Successful transformation of J23104 (K2018038) and J23106 (K2018039) in to E. coli Top 10. Date 16.06.27
The BioBricks are from the iGEM kit.
The protocol .
Successful transformation of J23104 in to E. coli Top 10. Date 16.06.30
The protocol .
Ligation of J23104:K934001 and J23106:K93400. Date 16.07.03
The ratio: 1:0, 1:2 and 1:5
J23104:K934001 (K2018038)  and J23106:K934001 (K2018039)
The protocol .
Week 27
Received plasmid, K2018011, and transformation into Top10 E. coli. Date: 16.07.05
The BioBrick was ordered from IDT in a plasmid pUCIDT-AMP and after receiving it, it was resuspended to a final concentration of 0.1 µg/µl.
After the resuspension, the plasmid was transformed into Top10 E. coli by following SOP0023_v01 Ca2+ Transformation , with the following remarks:
2.5 µl plasmid was used instead of the normal 1 µl.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Successful transformation of K2018038 into E. coli Top 10. Date 16.07.07
The BioBrick is in an ampicilin resistant plasmid.
The protocol .
Thursday concert in Odense. Date 16.07.07
Thursday concert in Odense.
Week 28
Received plasmid, K2018010, and transformation into Top10 E. coli. Date: 16.07.14
The BioBrick was ordered from IDT in a plasmid pUCIDT-AMP and after receiving it, it was resuspended to a final concentration of 0.1 µg/µl.
After the resuspension, the plasmid was transformed into Top10 E. coli by following SOP0023_v01 Ca2+ Transformation , with the following remarks:
2.5 µl plasmid was used instead of the normal 1 µl.
What we had done with the BioBrick K2018010 can be read in the protocols: Cloning composite part K2018010 into pSB1C3 and IMPACT Purification of K2018010
Verification of K2018011 in standard iGEM plasmid (pSB1C3). Date: 16.07.14
The BioBrick containing the gene K2018011 was digested with PstI and XbaI overnight by using SOP0017 .
After digesting, the sample was ligated by following SOP0015 with the standard iGEM plasmid, pSB1C3, overnight, and transformed into E. coli Top10.
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
>15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
After a colony PCR, SOP0021 , was run and verified on a gel, the following result was given:   Our part is 259 bp and primer overhangs are 150 X 2 bp, which is, in total, 559 bp: A gel electrophoresis from cPCR showed a band around 600 bp, so we assumed that we had the correct  part inserted into the iGEM standard plasmid.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Received plasmid, K2018014, and transformation into Top10 E. coli. Date: 16.07.14
The BioBrick was ordered from IDT in a plasmid pUCIDT-AMP and after receiving it, it was resuspended to a final concentration of 0.1 µg/µl.
After the resuspension, the plasmid was transformed into Top10 E. coli by following SOP0023_v01 Ca2+ Transformation , with the following remarks:
2.5 µl plasmid was used instead of the normal 1 µl.
What we had done with the BioBrick K2018014 can be read in the protocols: Cloning composite part K2018014 into pSB1C3 and IMPACT Purification of K2018014
Received plasmid, K2018019, and transformation into Top10 E. coli. Date: 16.07.14
The BioBrick was ordered from IDT in a plasmid pUCIDT-AMP and after receiving it, it was resuspended to a final concentration of 0.1 µg/µl.
After the resuspension, the plasmid was transformed into Top10 E. coli by following SOP0023_v01 Ca2+ Transformation , with the following remarks:
2.5 µl plasmid was used instead of the normal 1 µl.
What we had done with the BioBrick K2018019 can be read in the protocol: Cloning composite part K2018019 into pSB1C3
Successful transformation of K2018039 into E. coli Top 10. Date 16.07.15
The BioBrick is in an ampicilin resistant plasmid.
The protocol .
Week 29
Party Date 16.07.23
Verification of K1763002, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG47, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG47
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be observed .
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band, which was desired.
The second gel photo shows cutting of SB71 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763002 can be read in the protocol: Cloning Basic part K1763002 into pSB1C3
Verification of K1763003, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , and thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG48, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG48
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be observed .
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band.
The second gel photo shows cutting of SB73 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763003 can be read in the protocol: Cloning Basic part K1763003 into pSB1C3
Verification of K1763004, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , and thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG49, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG49
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be seen.
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band.
The second gel photo shows cutting of SB75 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763004 can be read in the protocol: Cloning Basic part K1763004 into pSB1C3
Verification of K1763009, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , and thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
>30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG56, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG56
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be observed .
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band.
The second gel photo shows cutting of SB77 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763009 can be read in the protocol: Cloning Basic part K1763009 into  pSB1C3
Verification of K1763010, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , and thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
>30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG43, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG43
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be observed.
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band.
The second gel photo shows cutting of SB63 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763010 can be read in the protocol: Cloning Basic part K1763010 into pSB1C3
Verification of K1763011, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , and thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
>30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG44, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG44
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be observed .
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band.
The second gel photo shows cutting of SB65 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763011 can be read in the protocol: Cloning Basic part K1763011 into pSB1C3
Verification of K1763012, in standard iGEM plasmid (pSB1C3). Date: 16.07.24
The BioBrick was first amplified by phusion PCR, SOP0010 , and thereafter digested with restrictions enzymes, following SOP0017 . The restriction enzymes were: PstI for 2 h and EcoRI for 30 min. After digesting, the sample was verified by running a gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
>30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step, the vector was ligated with the insert, SG45, overnight by following SOP0015 , and thereafter it was transformed, following SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG45
2 µl
5 µl
SG58
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp which gives a total of DNA length of 419 bp.  From this gel, the ligation of the gen incorporated into psB1C3 can be observed .
Picture of cPCR of different colonies on the same plate (each gene had its own plate). All the colony PCR shows a band near the 4-500 bp band.
The second gel photo shows cutting of SB67 with Pst1 at 2 hours with 37 °C and EcoR1 at 30 minutes. Every second well had added restriction enzymes.If a band of approximately 160 bp could be seen on the gel, it was concluded as being successful and sent to sequencing.
What we had done with the BioBrick K1763012 can be read in the protocol: Cloning Basic part K1763012 into pSB1C3
Week 30
Verification of K2018010 in standard iGEM plasmid (pSB1C3). Date: 16.07.25
The BioBrick containing the gene K2018010 was digested with PstI overnight and with EcoRI the day after by using SOP0017 .
After digesting, the sample was verified by gel electrophoresis. The result gave the following: The Gel Photo below shows BG51 (At the photo named; 51). The composite part is 395 bp long. It is possible from the Gel photo to assume that the DNA is cut correctly. From this gel, is the BG51 band of approximately 400 bp purified and thereafter ligated into the standard iGEM plasmid.
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and transformed into E. coli Top10.
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
Hereafter a colony PCR was runned and verified on a gel, the result is given below:
The Gel Photo below shows to the left the GeneRulerTM 1 kb DNA ladder were used ThermoFisher scientific. . The following bands are BB54, 1, 2 and 4 which are the ligation of the vector PR17 and the insert BG51. From the photo, it can be seen that sample 54.1 and 54.3 are the vector with the insert, and 54.2 is a religation.
What we had done with the BioBrick K2018010 can be read in the protocols: Cloning composite part K2018010 into pSB1C3 and IMPACT Purification of K2018010
Verification of K2018014 in standard iGEM plasmid (pSB1C3). Date: 16.07.25
The BioBrick containing the gene K2018014 was digested with PstI overnight and with EcoRI the day after by using SOP0017 .
After digesting, the sample was verified by gel electrophoresis. The result gave the following: The Gel Photo below shows BG52 (At the photo named; 52).The composite part is 458 bp long. It is possible, from the Gel photo, to assume that the DNA is cut correctly. From this gel, the BG52 band of approximately 400 bp purified and thereafter ligated into the standard iGEM plasmid.
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and transformed into E. coli Top10.
Ratio
1:2
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
BG75
2 µl
5 µl
10 µl
BG17
1 µl
1 µl
1 µl
ddH2O
14 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
Hereafter a colony PCR was runned and verified on a gel, the result is given below:
The Gel Photo below shows to the left, the GeneRulerTM 1 kb DNA ladder were used ThermoFisher scientific. . The following bands are BB55, 1, 2 and 4 are the ligation of the vector PR17 and the insert BG52. From the photo, it can be assumed that all 3 samples in BB55 are the vector with insert.
What we had done with the BioBrick K2018014 can be read in the protocols: Cloning composite part K2018014 into pSB1C3 and IMPACT Purification of K2018014
Verification of K2018019 in standard iGEM plasmid (pSB1C3). Date: 16.07.25
The BioBrick containing the gene K2018019 was digested with PstI for 2 hours and EcoRI for 30 min. by using SOP0017 .
After digesting, the sample was verified by gel electrophoresis. The result gave the following: The two Gel Photos below show the results for BG59 (At the photo named; 59). The composite part is 1739 bp long. It is possible from the Gel photo to assume that the DNA is cut correctly. From this gel, is the BG59 band of around 2000 bp purified and thereafter ligated into the standard iGEM plasmid.
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed, SOP009_v01 ,  into E. coli Top10.
The ratio of the ligation is shown hereunder:
Ratio
1:0
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
BG59
0 µl
5 µl
10 µl
BG60
1 µl
1 µl
1 µl
ddH2O
16 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
Hereafter a colony PCR was runned and verified on a gel. The result is given in the following section: The Gel Photo below shows to the left the GeneRulerTM 1 kb DNA ladder were used ThermoFisher scientific. .. The following bands are BB59, 1 to 14, which represent the ligation of the vector BR60 with the insert BG59. From the photo it can be assumed that all the samples are perfectly ligated. One of the colonies can therefore, be used to make ON for the plasmid purification and Freeze stock.
What we had done with the BioBrick K2018019 can be read in the protocol: Cloning composite part K2018019 into pSB1C3
Verification of K2018012 in standard iGEM plasmid (pSB1C3). Date: 16.07.26
The BioBrick was ordered from IDT in a plasmid pUCIDT-AMP. When received, it was resuspended to a final concentration at 0.1 µg/µl.
After the resuspension, the plasmid is transformed into Top10 E. coli by following SOP0023_v01 Ca2+ Transformation with the following remarks:
2.5 µl plasmid were used instead of the normal 1 µl.
What we had done with the BioBrick K2018012 can be read in the protocols: Cloning composite part K2018012 into pSB1C3 and IMPACT Purification of K2018012
Verification of K2018015 in standard iGEM plasmid (pSB1C3). Date: 16.07.26
The BioBrick was ordered from IDT in a plasmid pUCIDT-AMP. When received, it was resuspended to a final concentration at 0.1 µg/µl.
After the resuspension, the plasmid is transformed into Top10 E. coli by following SOP0023_v01 Ca2+ Transformation with the following remarks:
2.5 µl plasmid were used instead of the normal 1 µl.
What we had done with the BioBrick K2018015 can be read in the protocols: < Cloning composite part K2018015 into pSB1C3 and IMPACT Purification of K2018015
Successful transformation of K2018038 and K2018039 into E. coli Top 10. Date 16.07.28
First are the BioBricks digested by restrictions enzymes, and after ligated into pSB1C3, and then transformed to E. coli Top 10.
The protocol .
Changes the promoter region on K2018015, and transformation into Top10 E. coli. Date: 16.07.29
The BioBrick, that was ordered from IDT had a defected promoter, and therefore we replaced the promoter region with a functional one. This was done by a phusion PCR, SOP0010 . The primer was;k2018015_IDT.
ddH2O
32.5 µl
5 x HFbuffter
10 µl
dNTPs
1 µl
Primer; K2018015_IDT
2.5 µl
Suffix-primer
2.5 µl
Polymerase
0.5 µl
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
2 min
2
34 cycles
95 °C
10 sec
 
 
72 °C
16 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
10 min
4
Keep the sample cold
20 °C
HOLD
The Gel photo below shows the result from the pPCR. This was done to replace a defective promoter, with the correct one. This is done by using the Primer; k2018015_IDT. As it can be seen from the photo, is first the GeneRulerTM 1 kb DNA ladder were used ThermoFisher scientific. . and then sample BR74. The gene is 572 bp long. and it is therefore possible from the gel to assume that the pPCR is succeed.
Note: There is loaded 5 µl of each sample
What we had done with the BioBrick K2018015 can be read in the protocols: < Cloning composite part K2018015 into pSB1C3 and IMPACT Purification of K2018015
Week 31
Verification of K2018012 in standard iGEM plasmid (pSB1C3). Date: 16.08.02
The BioBrick containing the gene K2018012 was digested with PstI and XbaI overnight by using SOP0017 .
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed, SOP009_v01 ,  into E. coli Top10.
The ratio of the ligation is shown hereunder:
Ratio
1:0
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
BG75
0 µl
5 µl
10 µl
BG17
1 µl
1 µl
1 µl
ddH2O
16 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
Hereafter a colony PCR, SOP0021 , was runned and verified on a gel, The result is given below:
The following gel picture shows that we successfully cloned our part K2018012 into the standard iGEM plasmid, pSB1C3.
What we had done with the BioBrick K2018012 can be read in the protocols: Cloning composite part K2018012 into pSB1C3 and IMPACT Purification of K2018012
Verification of K2018015 in standard iGEM plasmid (pSB1C3). Date: 16.08.02
The BioBrick containing the gene K2018015 was digested with PstI and XbaI overnight by using SOP0017 .
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed, SOP009_v01 , into E. coli Top10.
The ratio of the ligation is shown hereunder:
Ratio
1:0
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
BG78
0 µl
5 µl
10 µl
BG17
1 µl
1 µl
1 µl
ddH2O
16 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
10 sec
 
 
72 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
12 °C
HOLD
After a colony PCR, SOP0021 , was runned and verified on a gel, the results were the following:
The Gel Photos below show the first Gel , to the left the GeneRulerTM 1 kb DNA ladder were used can be seen ThermoFisher scientific. . The following bands are BB79, 1 to 14. On the other gel can be seen first  the GeneRulerTM 1 kb DNA ladder were used, and then BB79 sample 15 to 23 ThermoFisher scientific. . BB79 is the ligation of the vector BG17 and the insert BG78. From the photo it can be assumed that all the samples, except; 2 and 6, are perfectly ligated. One of the colonies are therefore used to make ON for plasmid purification and Freeze stock.
What we had done with the BioBrick K2018015 can be read in the protocols: < Cloning composite part K2018015 into pSB1C3 and IMPACT Purification of K2018015
iGEM meets Marburg – Meetup in Marburg Germany. Date 16.08.04-16.08.07
Week 32
Math Danish Meetup. Date 16.08.10
Math Danish Meetup. It was awesome!
Week 33
Successful transformation of PHB parts K2018024, K2018027 and  K2018029. Date 16.08.15
This BioBricks are in an amp resistance plasmid.  They contain respectively, hlyA, hlyB or hlyD.
This three elements are digested and then ligated together.
The protocol .
Correct Overhang, D-forward and A-reverse on K2018011. Date: 16.08.19
K2018011 in standard iGEM plasmid (pSB1C3) was used in a phusion PCR, SOP0010 . The primes have an overhang, which makes the gene compatible with the silk overhang, and thereby enabling us to make a hybrid silk fiber.
pPCR program:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
30 sec
 
 
70 °C
30 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
20 °C
HOLD
The result is verified by Gel electrophoresis. The result is shown here:
Lane no. 5 is ThuricinS with DA-overhangs and is expected to be 217 bp long. We have used GeneRulerTM 50 bp DNA ladder were used, where the fourth band represents 200 bp ThermoFisher scientific. . Based on that, it can be assumed that we succeeded at getting silk-overhangs on ThuricinS.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Correct Overhang, D-forward and E-reverse on K2018011. Date: 16.08.19
K2018011 in standard iGEM plasmid (pSB1C3) was used in a phusion PCR, SOP0010 . The primes have an overhang, which makes the gene compatible with the silk overhang, and thereby enabling us to make a hybrid silk fiber.
pPCR program:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
30 sec
 
 
70 °C
30 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
20 °C
HOLD
The result is verified by Gel electrophoresis. The result is shown here:
The lane marked no. 2 is ThuricinS with DE-overhangs and it is assumed to be 217 bp long. We have used GeneRulerTM 50 bp DNA ladder were used, where the fourth band is 200 bp long ThermoFisher scientific. . Based on that, it can be assumed that we succeeded at getting silk-overhangs on ThuricinS.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Correct Overhang, E-forward and A-reverse on K2018011. Date: 16.08.19
K2018011 in standard iGEM plasmid (pSB1C3) was used in a phusion PCR, SOP0010 . The primes have an overhang, which makes the gene compatible with the silk overhang, and thereby enabling us to make a hybrid silk fiber.
pPCR program:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
30 sec
 
 
70 °C
30 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
20 °C
HOLD
The result is verified by Gel electrophoresis. The result is shown here:
The lane marked no. 7 and 8 are ThuricinS with EA-overhangs and it are expected to be 217 bp long. We have used GeneRulerTM 50 bp DNA ladder were used, where the fourth band is 200 bp long ThermoFisher scientific. . Based on that, it can be expected that we succeeded at getting silk-overhangs on ThuricinS.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Correct Overhang, E-forward and D-reverse on K2018011. Date: 16.08.19
K2018011 in standard iGEM plasmid (pSB1C3) was used in a phusion PCR, SOP0010 . The primes have an overhang, which makes the gene compatible with the silk overhang, and thereby enabling us to make a hybrid silk fiber.
pPCR program:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
2 min
2
34 cycles
98 °C
30 sec
 
 
70 °C
30 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
5 min
4
Keep the sample cold
20 °C
HOLD
The result is verified by Gel electrophoresis. The result is shown here:
The lane marked no. 4 and 8 are ThuricinS with ED-overhangs and is expected to be 217 bp long. We have used GeneRulerTM 50 bp DNA ladder were used, where the fourth band is 200 bp ThermoFisher scientific. . Based on that it  can be assumed that we succeeded at getting silk-overhangs on ThuricinS.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011 .
Week 34
Transformation of K2018030. Date 16.08.23
Transformation of ligated hlyABD (K2018030). Bacteria with amp resistance, were plated directly after heat shock Bacteria with chloramphenicol resistance were give 40- 60 min. of phenotypical expression.
The protocol .
Presentation on KOLD college. Date 16.08.23
We prepared an entire theme day for the biotech student on KOLD College where we had "designed" a experiment in the lab with some of our own BioBricks. The student got to perform a FastDigest following gel electrophoresis to verify their results. Thus we helped the students in the laboratory and entertained them in between with details of our project.
Correct Overhang, to IMPACT on K2018010. Date: 16.08.25
For using the IMPACT method for purification of bacteriocins, some specific restrictions sites are necessary. Therefore a Phusion PCR was made by following SOP0010 .
pPCR program:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
30 sec
2
34 cycles
98 °C
5 sec
 
 
69 °C
20 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
10 min
4
Keep the sample cold
4 °C
HOLD
What we had done with the BioBrick K2018010 can be read in the protocols: Cloning composite part K2018010 into pSB1C3 and IMPACT Purification of K2018010 .
Correct Overhang, to IMPACT on K2018011. Date: 16.08.25
For using the IMPACT method for purification of bacteriocins, some specific restrictions sites are necessary. Therefore a Phusion PCR was made by following SOP0010 .
pPCR program:
Segment
Step
Temperature
Duration
1
Initial denaturation
98 °C
30 sec
2
34 cycles
98 °C
5 sec
 
 
69 °C
20 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
10 min
4
Keep the sample cold
4 °C
HOLD
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Weeding with Birka and Laura. Date 16.08.27
It was an AMAZING wedding where one of our team member got married! Congratulations Birka!
Successful transformation of PHB parts K2018030. Date 16.08.28
The BioBrick is in an amp resistance plasmid, which should contain both hlyABD, but when it was verified the gel showed that not all colonies had all the gene, most of the assembled DNA did not include the meddle part (hlyB). Therefore we ligated two fragment together, to get the corrected gene.
The protocol .
Week 35
Successful transformation of K608003, K608004 K081005 and K880005 in to E. coli. Date 16.08.30
This BioBricks are in an amp resistance plasmid.  They contain respectively, hlyA, hlyB or hlyD.
This three elements are digested and then ligated together.
The protocol .
Verification of K2018011 with D-forward and A-reverse overhang, in standard iGEM plasmid (pSB1C3). Date: 16.09.01
The BioBrick was digested with with SpeI for 2h  and EcoRI for 30 min, by following SOP0017 .
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed into E. coli Top10, by following SOP0009 . The ratio of the ligation is shown hereunder:
Ratio
1:3
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
11
3 µl
5 µl
10 µl
BR123
1 µl
1 µl
1 µl
ddH2O
13 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
2 min
2
34 cycles
95 °C
15 sec
 
 
60 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
2.5 min
4
Keep the sample cold
4 °C
HOLD
The samples 11-17 are from colonies containing plasmids with ThuricinS with DA overhang
ThuricinS with silk overhangs, together with primer overhangs (VF2 and VR), is 517 bp long. GeneRulerTM 1 kb DNA ladder were used was used, so from these pictures it can verified that ThuricinS with DA overhangs have the right length ThermoFisher scientific. .
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Verification of K2018011 with D-forward and E-reverse overhang, in standard iGEM plasmid (pSB1C3). Date: 16.09.01
The BioBrick was digested with with SpeI for 2h  and EcoRI for 30 min, by following SOP0017 .
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed into E. coli Top10, by following SOP0009 . The ratio of the ligation is shown hereunder:
Ratio
1:3
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
13
3 µl
5 µl
10 µl
BR123
1 µl
1 µl
1 µl
ddH2O
13 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
2 min
2
34 cycles
95 °C
15 sec
 
 
60 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
2.5 min
4
Keep the sample cold
4 °C
HOLD
The samples 25- 28 are from colonies containing plasmids with ThuricinS with DE overhang
ThuricinS with silk overhangs, together with primer overhangs (VF2 and VR), is 517 bp long. GeneRulerTM 1 kb DNA ladder were used was used, so from these pictures it can verified that ThuricinS with DE overhangs have the right length ThermoFisher scientific. .
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Verification of K2018011 with E-forward and A-reverse overhang, in standard iGEM plasmid (pSB1C3). Date: 16.09.01
The BioBrick was digested with with SpeI for 2h  and EcoRI for 30 min, by following SOP0017 .
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed into E. coli Top10, by following SOP0009 . The ratio of the ligation is shown hereunder:
Ratio
1:3
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
12
3 µl
5 µl
10 µl
BR123
1 µl
1 µl
1 µl
ddH2O
13 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
2 min
2
34 cycles
95 °C
15 sec
 
 
60 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
2.5 min
4
Keep the sample cold
4 °C
HOLD
The samples 18-24 are from colonies containing plasmids with ThuricinS with EA overhang
ThuricinS with silk overhangs, together with primer overhangs (VF2 and VR), is 517 bp long. GeneRulerTM 1 kb DNA ladder were used was used, so from these pictures it can verified that ThuricinS with EA overhangs have the right length ThermoFisher scientific. .
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Verification of K2018011 with E-forward and D-reverse overhang, in standard iGEM plasmid (pSB1C3). Date: 16.09.01
The BioBrick was digested with with SpeI for 2h  and EcoRI for 30 min, by following SOP0017 .
After digesting, the sample was ligated by following SOP0015 , with the standard iGEM plasmid, pSB1C3, overnight, and thereafter transformed into E. coli Top10, by following SOP0009 . The ratio of the ligation is shown hereunder:
Ratio
1:3
1:5
1:10
Ligase Buffer
2 µl
2 µl
2 µl
Ligase
1 µl
1 µl
1µl
10
3 µl
5 µl
10 µl
BR123
1 µl
1 µl
1 µl
ddH2O
13 µl
11 µl
6 µl
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
2 min
2
34 cycles
95 °C
15 sec
 
 
60 °C
15 sec
 
 
72 °C
15 sec
3
Final extension
72 °C
2.5 min
4
Keep the sample cold
4 °C
HOLD
The samples 1-10 are from colonies containing plasmids with ThuricinS with ED overhang
ThuricinS with silk overhangs, together with primer overhangs (VF2 and VR), is 517 bp long. GeneRulerTM 1 kb DNA ladder were used was used, so from these pictures it can verified that ThuricinS with ED overhangs have the right length ThermoFisher scientific. .
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Ligation to make: K2018033, K2018034, K2018035 & K2018036>. Date 16.09.02
K608003:k934001 (K2018035) K608004:K934001 (K2018036) K081005:K934001 (k2018034) k880005:k934001 (k2018033)
This three elements are digested and then ligated together.
The protocol .
Successful transformation of K2018033, K2018034, K2018035 and K2018036 in to E. coli. Date 16.09.03
This BioBricks are in an amp resistance plasmid.  They contain respectively, hlyA, hlyB or hlyD.
This three elements are digested and then ligated together.
The protocol .
Week 36
Presentation for the Academy of Talented Youth. Date 16.09.06
We presented our project in front of the Academy of Talented Youth and also worked as their instructors in a brief lab course.
Successful transformation to the IMPACT vector.  K2018010. Date: 16.09.08
Before purifying the bacteriocin by IMPACT, the bacteriocin must first be transformed into the IMPACT vector bacteria, E. Coli:ER2566.  To do this, we  incorporated  the  pPCR product into the IMPACT plasmid, pTXB1. Thereafter  the products are digested by following the SOP0025. step. 8.1.1. and afterwards ligated by following step 8.1.3 → 8.1.4 in the same SOP.
The following gel image were used to verify that K2018010 were successfully cloned into pTXB1 (the IMPACT vector). Lane 4-6 is K2018010/pTXB1. Lane 4 and 6 showed the correct sized band. Lane 5 is a negative.
What we had done with the BioBrick K2018010 can be read in the protocols: Cloning composite part K2018010 into pSB1C3 and IMPACT Purification of K2018010
Successful transformation to the IMPACT vector.  K2018011. Date: 16.09.08-
Before purifying the bacteriocin by IMPACT, the bacteriocin must first be transformed into the IMPACT vector bacteria, E. Coli:ER2566.  To do this, we  incorporated  the  pPCR product into the IMPACT plasmid, pTXB1. Thereafter  the products are digested by following the SOP0025. step. 8.1.1. and afterwards ligated by following step 8.1.3 → 8.1.4 in the same SOP.
The following gel image were used to verify that K2018011 were successfully cloned into pTXB1 (the IMPACT vector). Lane 4-6 is K2018010/pTXB1. Lane 1-3  is K2018011/pTXB1 and all showed the correct sized band.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Successful transformation to the IMPACT vector.  K2018012. Date: 16.09.08
Before purifying the bacteriocin by IMPACT, the bacteriocin must first be transformed into the IMPACT vector bacteria, E. Coli:ER2566.  To do this, we  incorporated  the  pPCR product into the IMPACT plasmid, pTXB1. Thereafter  the products are digested by following the SOP0025. step. 8.1.1. and afterwards ligated by following step 8.1.3 → 8.1.4 in the same SOP.
The following gel image were used to verify that K2018012 were successfully cloned into pTXB1 (the IMPACT vector). Lane 9-12 is K2018012/pTXB1 and they all showed the correct sized band.
What we had done with the BioBrick K2018012 can be read in the protocols: Cloning composite part K2018012 into pSB1C3 and IMPACT Purification of K2018012
Successful transformation to the IMPACT vector.  K2018014. Date: 16.09.08
Before purifying the bacteriocin by IMPACT, the bacteriocin must first be transformed into the IMPACT vector bacteria, E. Coli:ER2566.  To do this, we  incorporated  the  pPCR product into the IMPACT plasmid, pTXB1. Thereafter  the products are digested by following the SOP0025. step. 8.1.1. and afterwards ligated by following step 8.1.3 → 8.1.4 in the same SOP.
The following gel image were used to verify that K2018014 were successfully cloned into pTXB1 (the IMPACT vector). The three first lanes is K2018014/pTXB1, where lane 1 and 3 showed the correct sized band but lane 2 was negative..
What we had done with the BioBrick K2018014 can be read in the protocols: Cloning composite part K2018014 into pSB1C3 and IMPACT Purification of K2018014
Successful transformation to the IMPACT vector.  K2018015. Date: 16.09.08
Before purifying the bacteriocin by IMPACT, the bacteriocin must first be transformed into the IMPACT vector bacteria, E. Coli:ER2566.  To do this, we  incorporated  the  pPCR product into the IMPACT plasmid, pTXB1. Thereafter  the products are digested by following the SOP0025. step. 8.1.1. and afterwards ligated by following step 8.1.3 → 8.1.4 in the same SOP.
The following gel image were used to verify that K2018015 were successfully cloned into pTXB1 (the IMPACT vector). We used three different primers to ‘cross amplify’ the gene and a part of the vector plasmid, getting 2 different DNA bands. All of the given results were, as expected, at 330 bp and 494 bp.
What we had done with the BioBrick K2018015 can be read in the protocols: < Cloning composite part K2018015 into pSB1C3 and IMPACT Purification of K2018015
Verification of K2018043, in standard iGEM plasmid (pSB1C3). Date: 16.09.10
The BioBrick was first digested with restrictions enzymes by following SOP0017 . The restriction enzymes were: PstI and XbaI for 2h. After digesting, the sample was verified by Gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step the vector was ligated with the insert overnight by following SOP0015 , and thereafter it was transformed, SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG74
2 µl
5 µl
SG26
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp, which gives a total DNA length of 419 bp.  From this gel the ligation  of the gen incorporated into psB1C3 can be observed.
This is the picture of the colony PCR where 1.2 to 1.6 are different colonies of the same plate for MaSp1 CD. 1.3 was used to purify plasmids.
The second gel photo shows cutting of SB100 with Xbal and PstI. Every second well had added restriction enzymes. If a band of 145 bp long was observed on the gel of ligation and transformation of SB100, it was concluded successful and sent to sequencing.
What we have done with the BioBrick K2018043 can be read in the protocol: > Cloning Basic part K2018043 into pSB1C3
Verification of K2018044, in standard iGEM plasmid (pSB1C3). Date: 16.09.10
The BioBrick was first digested with restrictions enzymes by following SOP0017 . The restriction enzymes were: PstI and XbaI for 2 h. After digesting, the sample was verified by Gel electrophoresis.
pPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
In the next step the vector was ligated with the insert overnight by following SOP0015 , and thereafter it was transformed, SOP0009 , into E. coli Top10. The ratio of the ligation is shown hereunder:
Ratio
1:2
1:5
Ligase Buffer
2 µl
2 µl
Ligase
1 µl
1 µl
SG75
2 µl
5 µl
SG26
1 µl
1 µl
ddH2O
14 µl
11 µl
To verify that the transformation and ligation succeeded, a colony PCR was run, SOP0021 .
cPCR program used:
Segment
Step
Temperature
Duration
1
Initial denaturation
95 °C
3 min
2
34 cycles
95 °C
25 sec
 
 
58 °C
25 sec
 
 
72 °C
30 sec
3
Final extension
72 °C
1 min
4
Keep the sample cold
20 °C
HOLD
The Gel Photo below shows the cPCR results. VR and VF2 primers adds a total of 300 bp, the gene is 119 bp long, which gives a total DNA length of 419 bp.  From this gel the ligation  of the gen incorporated into psB1C3 can be indicated .
This is the picture of the colony PCR where 2.1 to 2.3 are different colonies of the same plate for MaSp2 CD. 2.1 was used to purify plasmids
The second gel photo shows cutting of SB101 with Xbal and PstI. Every second well had added restriction enzymes. From this gel, the ligation and transformation of SB101, a band around 145 bp after digestion, was concluded as being successful and sent to sequencing.
What we have done with the BioBrick K2018044 can be read in the protocol: Cloning Basic part K2018044 into pSB1C3
Week 37
Presentation at Odense Technical High School (OTG) .
Week 38
Extraction of PHB with different methods are started. Date 16.10.22
The protocol .
Successful purification by IMPACT.  K2018010. Date: 16.09.25
The IMPACT purification is done by following the SOP0025 step.  8.1.6→ 8.4.3. and the purity is determined by Bradford Protein Concentration determination.
In the Bradford assay, we measured OD595= 0.09400, and together with the BSA standard curve (see appendix) we calculated the final protein concentration as being 67.1 µg/ml.
What we had done with the BioBrick K2018010 can be read in the protocols: Cloning composite part K2018010 into pSB1C3 and IMPACT Purification of K2018010
Successful purification by IMPACT.  K2018011. Date: 16.09.25
The IMPACT purification is done by following the SOP0025 step.  8.1.6→ 8.4.3. and the purity is determined by Bradford Protein Concentration determination.
In the Bradford assay we measured OD595= 0.05700, and together with the BSA standard curve (see appendix) we calculated a final protein concentration as being 33.5 µg/ml.
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Successful purification by IMPACT.  K2018012. Date: 16.09.25
The IMPACT purification is done by following the SOP0025 step.  8.1.6→ 8.4.3. and the purity is determined by Bradford Protein Concentration determination.
In the Bradford assy, we measured an OD595= 0.06900, and together with the BSA standard curve (see appendix) we calculated a final protein concentration as being 44.4 µg/ml.
What we had done with the BioBrick K2018012 can be read in the protocols: Cloning composite part K2018012 into pSB1C3 and IMPACT Purification of K2018012
Successful purification by IMPACT.  K2018014. Date: 16.09.25
The IMPACT purification is done by following the SOP0025 step.  8.1.6→ 8.4.3. and the purity is determined by Bradford Protein Concentration determination.
In the Bradford assay we measured an OD595= 0.05360, and together with the BSA standard curve (see appendix) we calculated a final protein concentration as being 30.4 µg/ml.
What we had done with the BioBrick K2018014 can be read in the protocols: Cloning composite part K2018014 into pSB1C3 and IMPACT Purification of K2018014
Successful purification by IMPACT.  K2018015. Date: 16.09.25
The IMPACT purification is done by following the SOP0025 step.  8.1.6→ 8.4.3. and the purity is determined by Bradford Protein Concentration determination.
In the Bradford assay we measured OD595= 0.07800, and together with the BSA standard curve (see appendix) we calculated a final protein concentration as being 52.5 µg/ml.
What we had done with the BioBrick K2018015 can be read in the protocols: < Cloning composite part K2018015 into pSB1C3 and IMPACT Purification of K2018015
Week 39
The group set up for the HNMR experiment with K2018000. Date 16.09.27
The protocol .
Validation by MIC-test for K2018010. Date: 16.09.28
The effect of the bacteriocins is validated by a MIC-test, which is made by following the SOP0027 step.  7.2→ 7.9.
The MIC assay gave the following MIC values:
Strain
MIC (µg/ml)
S. aureus:CC398
33.6
S. aureus:USA300
16.8
S. aureus:hVISA
16.8
P. aeruginosa:PAO1
33.6
What we had done with the BioBrick K2018010 can be read in the protocols: Cloning composite part K2018010 into pSB1C3 and IMPACT Purification of K2018010
Validation by MIC-test for  K2018011. Date: 16.09.28
The effect of the bacteriocins is validated by a MIC-test, which is made  by following the SOP0027 step.  7.2→ 7.9.
The MIC assay gave the following MIC values:
Strain
MIC (µg/ml)
S. aureus:CC398
16.7
S. aureus:USA300
16.7
S. aureus:hVISA
16.7
P. aeruginosa:PAO1
16.7
What we had done with the BioBrick K2018011 can be read in the protocols: Cloning composite part K2018011 into pSB1C3 , Cloning ThuricinS with DA silk-overhangs into pSB1C3 , Cloning ThuricinS with DE silk-overhangs into pSB1C3 , Cloning ThuricinS with EA silk-overhangs into pSB1C3 , Cloning ThuricinS with ED silk-overhangs into pSB1C3 and IMPACT Purification of K2018011
Validation by MIC-test for  K2018012. Date: 16.09.28
The effect of the bacteriocins is validated by a MIC-test, which is made  by following the SOP0027 step.  7.2→ 7.9.
The MIC assay gave the following MIC values:
Strain
MIC (µg/ml)
S. aureus:CC398
22.2
S. aureus:USA300
22.2
S. aureus:hVISA
22.2
P. aeruginosa:PAO1
>22.2
What we had done with the BioBrick K2018012 can be read in the protocols: Cloning composite part K2018012 into pSB1C3 and IMPACT Purification of K2018012
Validation by MIC-test for  K2018014. Date: 16.09.28
The effect of the bacteriocins is validated by a MIC-test, which is made  by following the SOP0027 step.  7.2→ 7.9.
The MIC assay gave the following MIC values:
Strain
MIC (µg/ml)
S. aureus:CC398
13.1
S. aureus:USA300
12.1
S. aureus:hVISA
6.6
P. aeruginosa:PAO1
>13.1
What we had done with the BioBrick K2018014 can be read in the protocols: Cloning composite part K2018014 into pSB1C3 and IMPACT Purification of K2018014
Validation by MIC-test for  K2018015. Date: 16.09.28
The effect of the bacteriocins is validated by a MIC-test, which is made  by following the SOP0027 step.  7.2→ 7.9.
The MIC assay gave the following MIC values:
Strain
MIC (µg/ml)
S. aureus:CC398
7.6
S. aureus:USA300
7.6
S. aureus:hVISA
7.6
P. aeruginosa:PAO1
>15.2
What we had done with the BioBrick K2018015 can be read in the protocols: < Cloning composite part K2018015 into pSB1C3 and IMPACT Purification of K2018015
Analyzing PHB by Fluorescence activated cell sorting (FACS). Date 16.09.29
The figure below display the number of events on the y-axis and the corresponding intensities on the x-axis. Cells of the different strains are displayed in different colors. Each event represents a cell and the intensity represents the amount of PHB in the respective cell.
The figure below display the different E. coli strains on the x-axis. The promoter and RBS are marked with corresponding affinity: “s” = strong, “W” = weak and “MS”=medium. The y-axis displays the average intensity on the flow cytometer detected by red fluorescence.
The protocol .
BMB totaldag. Date 16.09.30
BMB Totaldag WOOP
Purification by Chloroform of PHB. Date 16.09.30
The protocol .
Purification by Ethyl acetat of PHB. Date 16.09.30
The protocol .
Week 40
Successful transformation of PHB parts K2018048 and K2018050. Date 16.10.05
This BioBricks are created by ligation between K2018036, K2018030 & K2018021 (K2018050), between K2018036 & K2018021 (K2018048) and between K2018036 & K2018030 (K2018049). All this BioBricks are now in an amp. Resistance plasmid.
The protocol .
Successful transformation of PHB parts K2018049. Date 16.10.08
The protocol .
Samples for iTRAQ are prepared. Date 16.10.05
The protocol .
Qubit® Protein Assay Kits is used to determine concentration. Date 16.10.05
From the following table the determined concentrations are showed.
Sample
Koncentration
1(panK)
4.38 mg/ml
2(panK)
4.24 mg/ml
3(control)
5.2 mg/ml
4(control)
5.48 mg/ml
The protocol .
Purification by Triton x- 100 and hypochlorite of PHB. Date 16.10.05
The protocol .
Purification by hypochlorite of PHB. Date 16.10.05
The protocol .
Samples from Qubit® Protein Assay Kits where runned through a Columm 16.10.06
The samples where runned on C2 columm and then on a C3 columm.
The protocol .
Samples from Qubit® Protein Assay Kits where runned through a Columm 16.10.06
The samples where runned on C2 columm and then on a C3 columm.
The protocol .
iTRAQ labelling. Date 16.10.06
As there was no 4plex available, we used 8plex chemicals instead. However, they are of a different compositions and are should be suspended in isopropanol not ethanol. We didn’t know this and this has effected the labelling. Therefore, instead of pooling the entirety of each sample, 5 µl of each sample were kept separately.
The protocol .
Purification by TiO_2. Date 16.10.07
The purification was performed on the pooled peptides.
The protocol .
C2 and C3 columns on PHB. Date 16.10.07
Afterwards LC-MS/MS were performed on the samples in order to retrieve data on the bacterial proteome. The purification was repeated, due to visible impurities in the dried down sample.
The protocol .
Purification by TiO_2. Date 16.10.08
The purification was performed on each of the remaining 5 µl that had not been pooled.
The protocol .
Week 41
Successful transformation of PHB parts K2018049. Date 16.10.08
Now K2018049 is ligated in to pSB1C3 and transformed into top 10 E. coli.
The protocol .
Successful transformation of PHB parts K2018030. Date 16.10.11
Now K2018030 is ligated in to pSB1C3 and transformed into top 10 E. coli.
The protocol .
Production of PHB to 3D printing. Date 16.10.12
Creating 25 L of TB media, and growth BioBrick K2018050
The protocol .
Gas chomotography on PHB. Date 16.10.13
The injection split ratio was 120:1. The injection port and the detector temperatures were set at 180°C and 200°C, respectively. The initial oven temperature was constant for one minute at 90°C. Then increased at 3.8 ℃/ 𝑚𝑖𝑛𝑢𝑡𝑒 to a final temperature of 150°C. The temperature was maintained for 5 min. The flow rate of the helium carrier gas was 1 mL/minute.
The protocol .
Purified PHB by using Chloroform. Date 16.10.13
The protocol .
HNMR on the purified PHB is made. Date 16.10.15
The protocol .
Extracts the 25 L of TB media with Hypochlorite. Date 16.10.16
The protocol .
Week 42
Making PHB filament by using Filastruder. Date 16.10.17
After the PHB had dried we measured the mass of PHB which was around 100 g. The dried mass was added to the filastruder. We made 3 meters of filament of PHB that almost had a constant diameter.
The protocol .
Testing of three different types of growth media. Date 16.10.18
We had tested which media that is best for plastic production. The three medas are, 2xYT, TB and LB.
The medias were tested by OD measuring approximate made every half hour in 7 hours.
The 2xYT media waas better for fast growth of bacteria. Both Top10 and Top10/pSB1C3-PanK+sec had a lower generation time in 2xYT media.TB media however, was better for producing a lot of bacteria. A higher OD was reached when growing bacteria in this media.
The protocol.
Printing PHB by 3D printing. Date 16.10.18
The 3D print were done in several attempts because the filaments of PHB were not constant in diameter. In the last try the machine managed to make a nice model of the implant (jawstump).
The protocol.
WIKI FREEZE
See you in Boston soon, we are ready
Menu
Home
Introduction
Team
Attributions
Collaborations
Achievements
Project
Description
Experiments
Demonstration & Results
Proof of Concept
Scientific Reproduction
Parts & Procedures
Parts
Notebook
Safety
Practices & Prospects
Bioethics
Integrated Practices
Outreach
Prospects
Perspectives
