Team:Lethbridge/Results
iGEM
iGEM
RNAiCare
Project
Description
Design
Results
Production
Judging
Parts
Basic Parts
Composite Parts
Measurement
Practices
Risks
Stakeholders
Current Problems
Notebook
July
August
September
Team
Members
Collaborations
Attributions
Sponsors
Project Results
THEOPHYLLINE APTAZYME IN VITRO:
The theophylline-responsive aptazyme used in our project was previously engineered to function in vivo in the yeast species Saccharomyces cerevisiae (Win and Smolke 2007). The device was designed to modulate the cleavage of the host RNA via addition of exogenous theophylline (theophylline is not normally present in yeast cells). Binding of theophylline to the aptazyme is predicted to induce a conformational change in the RNA that promotes a cleavage competent structure. Initially, we wanted to determine if the device was viable under in vitro conditions. To test this, we generated PCR products of the full length aptazyme and used this as a template for in vitro transcription by T7 RNA polymerase (Fig. 1).
Figure 1. Theophylline Aptazyme in vitro transcription. September 1, 2015.
We readily detected large amounts of aptazyme in vitro transcript (Fig 1), however we noted that a significant amount of the transcript was truncated and of a length consistent with cleaved aptazyme, even in the absence of theophylline. We hypothesized that the premature cleavage of the aptazyme may have resulted from the temperature conditions in which our in vitro transcription was performed; non-optimal temperatures may impair proper folding of the structure resulting in premature cleavage during in vitro transcription. We therefore tried performing the in vitro transcription at room temperature and 37°C (Fig. 2).
Figure 2. Theophylline Aptazyme in vitro transcription at 24°C and 37°C time course. September 2, 2015
Interestingly, in the in vitro transcriptions that were carried out at room temperature for one hour, approximately half of the transcripts were full-length, while longer incubations at room temperature or at 37°C showed mostly cleaved aptazyme transcripts. This is the first time the performance of this device has been exhibited in vitro. In an attempt to further improve the yield of full-length aptazyme transcript, we tested a wide variety of temperatures, incubation times, and buffer conditions for in vitro transcription (Fig. 3).
Figure 3. Theophylline Aptazyme in vitro transcription temperature and magnesium testing. September 9, 2015
Unfortunately, none of the in vitro transcription conditions tested were able to generate additional full-length transcript. Next, we wanted to see if addition of theophylline might still result in further cleavage of the aptazyme. Thus, we performed cleavage assays of the partially full-length in vitro transcripts using theophylline concentrations ranging from 100nM to 1mM, and analyzed the products on denaturing polyacrylamide gels (Fig. 4).
Figure 4. Theophylline Aptazyme Cleavage Assay. Purified in vitro. September 8, 2015.
Unexpectedly, after incubation at room temperature for 30 minutes, aptazyme transcripts were all cleaved, irrespective of theophylline concentration. This indicates that further optimization of the cleavage conditions must be undertaken to perfect the performance of the theophylline aptazyme in vitro.
GENERATION OF RNA USING RIBOZYME AFFINITY PURIFICATION:
We next attempted to purify RNA utilizing our Ribozyme Affinity Purification strategy. Two separate constructs were employed; one construct expresses amino-terminally 6xHistidine-tagged MS2 coat-protein under the control of a T7 promoter and standard ribosome binding site (RBS) (termed High RBS), while the second construct expresses the tagged MS2 under control of a T7 promoter and weak RBS (Low RBS). In this way, we hoped to determine the optimal amount of MS2 expression required for effective purification of our desired RNA species by modulating the translation efficiency of MS2 coat protein. Both of the constructs also contain a theophylline aptazyme coding module that is also under control of a T7 promoter.
Initially, our constructs were cloned into the pSB1C3 backbone vector and then transformed into BL21(DE3) Escherichia coli cells. Induction of protein and target RNA expression was initiated simultaneously by addition of 1 mM isopropyl β-D-1-thiogalactopyranoside (IPTG) that results in overexpression of T7 RNA polymerase.
Figure 5. Low RBS RAP SDS-PAGE (left) and Standard RBS RAP SDS-PAGE (right).
As expected, induction of the High RBS construct resulted in strong expression of MS2 coat-protein, whilst the Low RBS construct showed markedly less expression with MS2 not being easily distinguishable against the E. coli protein background, in the latter (Fig. 5A). Nonetheless, upon performing Ni2+-affinity chromatography followed by SDS-PAGE analysis of samples, both High RBS and Low RBS showed detectable amounts of purified MS2 protein (Fig 5B). Interestingly however, the yield of purified MS2 protein from the Low RBS construct was similar to that of the High RBS construct. Indeed, we observed a considerable amount of the MS2 protein in the insoluble fraction of the High RBS construct purification and we predict that this loss resulted in the reduced yields of MS2. In the future, other expression conditions and/or MS2 isoforms will be tested to improve the yield of soluble protein.
Next, we wanted to determine if we had successfully co-purified the aptazyme transcript during purification of MS2 protein. Consequently, we performed phenolic extractions on one tenth of each protein purification sample and then examined them for the presence of RNA on denaturing urea polyacrylamide gels. Interestingly, while the Low RBS samples did not show detectable aptazyme transcripts (230 to 250 nucleotides, nt), elusions from the High RBS purification showed a clear enrichment of RNAs with expected length for the aptazyme transcript (Fig. 6). Most strikingly, the predicted ratio of the full-length to cleaved aptazyme appears to be improved over our in vitro transcribed aptazymes (~70% full-length versus ~30% cleaved). We take these results to suggest that cellular conditions during expression in vivo and/or differences in RNA polymerase incorporation rate affect the folding of the aptazyme and consequently the frequency of cleavage. It is also interesting to note that we detected a small amount of the small RNA cleavage product after RNA purification. Because the small RNA fragment does not contain the MS2 binding region, we take this to suggest that cleavage events giving rise to the fragments occurred sometime during or after purification. If this is true, then the actual amount of full-length transcript from the in vivo expression may be even higher than the predicted ~70%.
Figure 6. Low RBS RAP Urea PAGE (left) and Standard RBS RAP Urea PAGE (right).
Finally, we wanted to determine whether the full-length in vivo purified aptazyme transcripts could be induced to cleave via addition of theophylline. Purified RNAs were incubated with theophylline concentrations from 1μM to 1mM and analyzed by denaturing PAGE. In contrast to the previous cleavage assays using in vitro transcribed aptazyme, we did not detect any significant cleavage of the in vivo purified transcript (Fig. 7). Because conditions for aptazyme cleavage may require different buffer conditions (such as pH, salt and magnesium), in the future we will attempt to induce cleavage using a variety of reaction conditions.
TESTING OF dsRNA TARGET SEQUENCES (TS) 1 – 5 ON FUSARIUM GRAMINEARUM (FG) STRAIN G23639:
In this test we focused our efforts on TS2* as it showed the most pronounced decrease in pigment production compared to the other target sequences. We produced spores by incubating FG in liquid carboxymethylcellulose (CMC) media containing different concentrations of dsRNA for 7 days, mimicking a study by Khatri, M. and Rajam V. (2007). Next, we measured how many spores were produced in each sample using a hemocytometer and plated ~3x105 spores per plate onto Potato Dextrose Agar (PDA) plates. We then allowed the spores to grow for 4 days and analyzed them to see if we could detect a change in pigment levels.
*In our original design this was termed TS3 however we changed it to TS2 as this target is the second target sequence from the 5' end of the gene
Day 0:
Triplicate of FG spores incubated with target seqeunce two dsRNA plated on PDA agar.
Triplicate of FG spores incubated with water (control) plated on PDA agar.
Day 3:
Triplicate of FG spores incubated with target seqeunce two dsRNA plated on PDA agar.
Triplicate of FG spores incubated with water (control) plated on PDA agar.
We observed a marked decrease in pigment production in our test samples incubated with dsRNA TS2. Having tested our dsRNA while incubating it with Fusarium during spore production, we wanted to test the effects of dsRNA when it was pipetted onto Fusarium. In this way we hope to closely mimic the real world conditions that we want our dsRNA pesticide to be used in.
In order to test how efficiently the Fusarium graminearum uptakes our dsRNA target sequences we plated a small piece of FG in the middle of a PDA plate. After 3 days of growth at 27°C, we applied 500μL of 20nM dsRNA onto our test plates and 500μL of water onto our control plates. We then incubated the plates for 3 days at 27°C.
Day 3:
Day 6:
Day 9:
We noticed a marked decrease in pigment production compared to wild type in the plate that had TS1 dsRNA applied to it. Where you see a near complete loss of red pigment production.This was a surprising result for us as we had expected TS2 to result in the most pronounced decrease in pigment production.
