<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Software_CELLO skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Software CELLO</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2016</TITLE></P><DIV class="container"><DIV class="col-md-2"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse text-center" id="bs-example-navbar-collapse-1"><DIV id="nav-menu" class="col-md-10 col-xs-12"><UL class="nav navbar-nav navbar-right"><LI class="active page-scroll"><A href="https://2016.igem.org/Team:EPFL">Home</A></LI><LI class="dropdown page-scroll"><A href="#project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Results">Results</A></LI></UL><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Parts">Parts</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="dropdown page-scroll"><A href="#lab" class="dropdown-toggle" data-toggle="dropdown">Lab</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Medals_Criteria">Medal Criteria</A></LI><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="dropdown page-scroll"><A href="#hp" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Engagement">Engagement</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_CELLO">Cello</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_DATABASE">Database</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model_DESKGEN">DESKGEN</A></LI></DIV></DIV></DIV><DIV class="simple-page"><SECTION class=""><DIV class="container"><DIV class="col-md-10 col-md-offset-1 text-center"><H2 class="lead animate-box">Cello</H2><P class="sub-lead text-justify animate-box">
                            Over the course of this summer, we developed a suite of tools intended to help users 
                            design their own biological circuits for their target systems. Our journey into the 
                            realm of bioinformatics began when we discovered Cello. Published on April 1<SUP>st</SUP>, 2016 
                            in <A href="http://science.sciencemag.org/content/352/6281/aac7341">
                                Science</A>, Cello takes user inputs in the form of a circuit described using 
                                <A href="https://en.wikipedia.org/wiki/Verilog">Verilog</A> 
                            code, and a user constraint file (UCF) that contains biological data pertaining to the 
                            system chosen by the user and a list of gates that the user can put in that system. By 
                            combining these two inputs with a series of algorithms, the program returns one or more
                            plasmids that, once put in a host cell, will recreate the circuit the user has defined.
                            By choosing the input promoters, the user can decide what molecules or transcription 
                            factors to use as inputs to the logical circuit. This circuit can allow the cell to make
                            novel decisions and calculations based on its environment, metabolism, and health. 

                        </P><P class="sub-lead text-justify">
                            Cello is immensely powerful, as well as accurate. In its original round of testing, 
                            the authors found that 75% of the 60 circuits texted acted exactly as expected for all 
                            input combinations. Cello is modular, which makes it adaptable to user requirements. 
                            These requirements are passed to it using a User Constraint File, which also contains 
                            information specific to the biological system that is being used, such as gates that are 
                            available to use. 
                        </P><P class="sub-lead text-justify">
                        Cello does not work with all biochemistries, unfortunately. Although it works well with CRISPRi,
                        it does not accept newer dCas9-based gate structures and technologies. We felt that we could make
                        Cello even more powerful by introducing these new biochemistries, and scRNA specifically (link 
                        Zalatan). This is because dCas9 allows for the creation of “programmable” transcription factors. 
                        Finding actual transcription factors that work well in the host, but also don’t interfere with its
                        genome is a real challenge. With dCas9, guide RNAs can be used to guide the dCas9 to specific 
                        23-nucleotide long targets. In addition, transcriptional effectors can be attached to the dCas9 
                        directly, or attached to the RNA directly using an architecture first described by 
                        <A href="https://www.ncbi.nlm.nih.gov/pubmed/25533786">Zalatan et al. 
                            (2015)</A>. See our <A href="https://2016.igem.org/Team:EPFL/Description">Project Description</A> for more details! Using this structure, we can have 
                        multiple effectors in the same synthetic system, with varying degrees of repression and activation, 
                        effectively fine-tuning the transcriptional response to each promoter. This system is made even more 
                        powerful by the discovery of “programmable” promoters, which have certain known internal regions in 
                        which the code can be changed without affecting the strength of the promoter. By modifying the code 
                        in these regions, it is possible to give each promoter its own ‘barcode’, which can be targeted by 
                        a unique guide RNA. <A href="https://2016.igem.org/Team:EPFL/Model_DESKGEN">Click here to see the barcodes we designed.</A></P><P class="sub-lead text-justify">
                            The current organization of information in the program could also be improved, 
                            to the benefit of the program’s usability. The current implementation of the UCF 
                            has certain limitation. In particular, the only publically available UCFs are 
                            defined in <EM>E. coli</EM>, meaning that it is not immediately usable in other systems. 
                            This brings up an even more important point: UCFs are treated as completely local 
                            files, meaning that for one researcher to use information obtained by another, 
                            that information has to be passed directly from one to the other. This is inefficient
                            since it is likely that at any given point more gate structures would have been 
                            invented than someone would be aware of, since those structures might be locked away
                            in another person’s local file. Finally, using Cello requires a basic understanding
                            of how Verilog code works, meaning that understanding how to use a coding language
                            is necessary before you can start to use the program. 
                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page"><SECTION class=""><DIV class="container"><DIV class="col-md-10 col-md-offset-1 text-center"><H2 class="lead">Our modifications</H2><H3>Graphical User Interface</H3><P class="sub-lead text-justify">
                            The first modification we made was to eliminate the need to understand how to 
                            code in Verilog to be able to use Cello. Verilog is not a difficult programming
                            language, but learning any programming language can be challenging for beginners, 
                            and we wanted to open up Cello to people who had no other programming experience, 
                            and eliminate the initial learning curve due to this technological barrier. The 
                            user interface we developed is based on an intuitive drag-and-drop interface. This 
                            interface is based on a template by <A href="https://github.com/edwardball/academo.org">
                            edwardball</A>, and uses the jsPlump library. Some users may feel more at home
                            using Verilog code, so the original input forms are still available on their original page. 
                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page"><SECTION class=""><DIV class="container"><DIV class="col-md-10 col-md-offset-1 text-center"><H3>Accepting External Inputs</H3><P class="sub-lead text-justify">
                        In order to add the new biochemistries to Cello, we first had to delve into its 
                        inner workings. A large part of our time was spent trying to understand what each 
                        part of the program did, and how it interacted with its libraries. It was during
                        this point that we identified two bugs in the program which we reported to the 
                        development team of Cello, through Prashant Vaidyanathan at Boston University. 
                        This led to an updated version of the NetSynth library being made to deal with 
                        gates that were not NOT, NOR, or AND. 
                        </P><P class="sub-lead text-justify">
                            We felt that the best way to integrate the aforementioned biochemistries and 
                            to also make the program more open-sourced would be to move the storage of 
                            information about dCas9-based gates to the web. Since we did not find an extant database
                            for parts with this biochemistry that returns data that is usable by Cello, 
                            we decided to make an open and free database for these parts.
                            <A href="https://2016.igem.org/Team:EPFL/Software_DATABASE">You can find out more about 
                                our databases here!</A> Cello had to be modified, however, to accept inputs from 
                            this third source. As the UCF is written in <A href="http://www.json.org/">JSON</A> (JavaScript Object Notation), 
                            we decided that we would also pass information to Cello through JSON files. 
                            These files also allow dCas9-based gates to work correctly in the target system, by appreciating
                            the particularities specific to dCas9-based gates. For example, all these gates would require 
                            dCas9 to be expressed in the host cells to function, and many gates would have overlapping effector
                            molecules. Recopying these &quot;associated sequences&quot; multiple times into the plasmids would constitute a massive waste of
                            space, so these sequences are copied <EM>just once</EM> into a separate plasmid.  
                        </P><FIGURE><FIGCAPTION id="associated-plasmid-caption">The &quot;associated sequences&quot; are visible in Cello's output as part of a separate plasmid.</FIGCAPTION></FIGURE><P class="sub-lead text-justify">
                            To know how to use these new gates, Cello requires their response functions. 
                            Since many of these gates would be created at the moment they are needed, 
                            they might not have experimentally confirmed response functions, but it would
                            still be important to know how they react. We worked on a model to help 
                            explain their behavior, which you can find <A href="https://2016.igem.org/Team:EPFL/Model">here</A>.
                        </P><P class="sub-lead text-justify">
                            Cello serves to make the design of synthetic biological genetic circuits easier than ever; our 
                            modifications to Cello continue in the same vein, by making the software more usable, and the 
                            information it uses more transferable. Although the impementation of these new features is not 
                            perfect yet – the integration of new genetic material from json files is subject to bugs, and 
                            Cello is not yet connected with intelligene.plus – we are working diligently past wiki freeze 
                            to give the synthetic biology community the best tools possible! 
                        </P></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>