<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Model_DESKGEN skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Model DESKGEN</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2016</TITLE></P><DIV class="container"><DIV class="col-md-2"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse text-center" id="bs-example-navbar-collapse-1"><DIV id="nav-menu" class="col-md-10 col-xs-12"><UL class="nav navbar-nav navbar-right"><LI class="active page-scroll"><A href="https://2016.igem.org/Team:EPFL">Home</A></LI><LI class="dropdown page-scroll"><A href="#project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Results">Results</A></LI></UL><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Parts">Parts</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="dropdown page-scroll"><A href="#lab" class="dropdown-toggle" data-toggle="dropdown">Lab</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Medals_Criteria">Medal Criteria</A></LI><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="dropdown page-scroll"><A href="#hp" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Engagement">Engagement</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_CELLO">Cello</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_DATABASE">Database</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model_DESKGEN">DESKGEN</A></LI></DIV></DIV></DIV><DIV class="simple-page"><SECTION class=""><DIV class="container"><DIV class="col-md-10 col-md-offset-1 text-center"><H2 class="lead animate-box">DESKGEN</H2><P class="sub-lead text-justify animate-box">
                            A fundamental part in the gate design based on dCas9 are the guide RNAs (gRNA) Although different
                            biochemistries exist for these RNAs (scRNA, sgRNA, crRNA, etc.) thier unifying characteristic is a 
                            a 20 nucleotide sequence that will bind to a given target. This binding occurs with varying levels of 
                            specificity, and is sequence dependent. In the case of our project, the target is always in a
                            promoter sequence. It was thus vital to have guides that 
                            would bind both strongly and specifically to the target sequence with very little off-target 
                            activity. Designing the perfect gRNAs is not an easy task, as a simple BLAST will not be enough
                            to ensure specificity. The change in binding energy resulting from mismatches varies depending on the identity
                            of the  mismatch, and the pition in the target. When we saw that Twist Bioscience and Desktop Genetics - a company
                            specialized in sgRNA design - were offering training in guide design to iGEM teams, and were sponsoring the synthesis of these guides, 
                            we sent our application letter. They contacted us a few days later to inform us that they were interested in our project 
                            and that we were part of the 5 lucky teams selected for the sponsoring. After this great news, 
                            our guide design adventure was ready to start.
                        </P><H3 class="dg_section_titles">First Steps</H3><P class="sub-lead text-justify"><A href="https://www.deskgen.com/landing/">DESKGEN©</A>, is Desktop Genetics's experiment planning platform. One of 
                            it's major functions is to find sgRNAs that will target your sequence of interest and perform knock-in 
                            and knock-out experiments, which could help scientists in the understanding of the function of various genes. 
                            Our needs were a little bit different than those of most since we worked with the CYC minimal promoter. 
                            We did not want just want to target a promoter, but we wanted to target it <EM>multiple </EM> times. Interestingly, 
                            this promoter has regions which are known to be changeable - without modifying the function of the promoter. We 
                            therefore decided that it would be interesting to use a CYC multiple times throughout a certain circuit, each one 
                            with unique &quot;barcode&quot; sequences. This would allow each CYC promoter to be regulated independently of the others. 
                            We therefore required various, different sgRNAs with no cross-talk between them, so that we could build a lot 
                            of different versions of the CYC promoter with their corresponding target and be able to design any complex biological circuit.
                            With the help of Eugenia Petrova, an intern at Desktop Genetics, we found a way to use their platform in order 
                            to exploit all their implemented algorithms to find guides suited to our needs. 
                        </P><H3 class="dg_section_titles">Guide Design</H3><P class="sub-lead text-justify">
                            Since we did not wanted and off-target binding in the <EM>S. cerevisiae</EM> or human genomes, any other genome 
                            excluding those two might constitute good “mine” where we could search for our guides. We decided to 
                            use the <A href="http://www.candidagenome.org/"><EM>C. albicans</EM> genome</A> for this purpose. Our mining work 
                            started by uploading portions of the FASTA file of the <EM>C. albicans</EM> genome in the “Sequence Only” section 
                            of the knock-in mode experiment design of DESKGEN.
                        </P><P class="sub-lead text-justify">
                            Next we had to set the correct parameters: specifying the organism in which we were working, S. 
                            Cervisiae and the dCas9 we were working with, SpCas9. It was necessary as the SpCas9 can only bind to a 
                            sequence and target a sgRNA at it if a specific PAM sequence, NGG, is lying next to it.
                        </P><P class="sub-lead text-justify">
                            After complex calculations, the platform shows all the possible sgRNAs and their targets on the uploaded 
                            sequence. Accodring to DESKGEN, an on-target activity score of 50/100 is sufficient to  
                            have  well performing guides, but we decided to selected only those that had a score of at least 65/100. This score 
                            is calculated using the algorithm described by <A href="https://www.ncbi.nlm.nih.gov/pubmed/26780180">Doench al. (2016)</A>.
                            For the sgRNAs with high enough on-target activity scores, we used DESKGEN to calculate the off-target activity, according to
                            an algorithm first described by <A href="http://www.nature.com/nbt/journal/v31/n9/full/nbt.2647.html">Hsu et al. (2013)</A>. 
                            A higher score means higher specificity and minimal binding to the genome of the host cell. We selected only the sgRNAs with a 100 off-target 
                            score in S.cerevisiae. In order to verify the safety of these guides, and to allow the same guides to be used in 
                            possible forthcoming  experiments in human cells, we also checked that the few sgRNAs that met the previous 
                            conditions had also almost no off-target in humans. Taking into account the enormous size of the human 
                            genome, it is nearly impossible to have no off-target activity, for probabilistic reasons. It is possible, however,
                            to ensure the safety of these guides by making sure that any binding behavior they exhibit occurs in a non-coding 
                            region of the genome. In addition, we only selected guides that had two more mismatches or more with targets in these
                            non-coding regions, thereby reducing their binding capabilities. Theoretically, all these guides could be used safely 
                            even in human cells. 
                        </P><H3 class="dg_section_titles">Guide Implementation</H3><P class="sub-lead text-justify">
                            After all of these screenings, the number of possible sgRNAs decreased drastically. To find a decent number of sgRNAs, we had to analyze 
                            almost three entire chromosomes of <EM>C. albicans</EM>. Below is the final list of targets and their complements that 
                            we were able to find that met our stringent criteria. 
                        </P><TABLE class="table table-striped tg"><TBODY><TR><TH class="tg-yw4l">Target Sequence</TH><TH class="tg-yw4l">Guide Sequence</TH></TR><TR><TD class="tg-yw4l">GGTGTACATACAGTAGGATCCTA</TD><TD class="tg-yw4l">CCACATGTATGTCATCCTAGGAT</TD></TR><TR><TD class="tg-yw4l">GGAGTACTGTATGTACATACAGT</TD><TD class="tg-yw4l">CCTCATGACATACATGTATGTCA</TD></TR><TR><TD class="tg-yw4l">GGATGCAAAGGTCCTAATGTATA</TD><TD class="tg-yw4l">CCTACGTTTCCAGGATTACATAT</TD></TR><TR><TD class="tg-yw4l">GGATATAGTAATTTATGCTGCAA</TD><TD class="tg-yw4l">CCTATATCATTAAATACGACGTT</TD></TR><TR><TD class="tg-yw4l">GGTGTATCGTTGCAACTACCCGG</TD><TD class="tg-yw4l">CCACATAGCAACGTTGATGGGCC</TD></TR><TR><TD class="tg-yw4l">GGAAGTGGCACCTAACTTGAACG</TD><TD class="tg-yw4l">CCTTCACCGTGGATTGAACTTGC</TD></TR><TR><TD class="tg-yw4l">GGTGCATTCGAGTAATAATCTTG</TD><TD class="tg-yw4l">CCACGTAAGCTCATTATTAGAAC</TD></TR><TR><TD class="tg-yw4l">GGAGGTCACTCTATACTTTCCGT</TD><TD class="tg-yw4l">CCTCCAGTGAGATATGAAAGGCA</TD></TR><TR><TD class="tg-yw4l">GGTGCAACGATACGATGGCCGAG</TD><TD class="tg-yw4l">CCACGTTGCTATGCTACCGGCTC</TD></TR><TR><TD class="tg-yw4l">GGTACTCGGAGTTGATTATATCG</TD><TD class="tg-yw4l">CCATGAGCCTCAACTAATATAGC</TD></TR><TR><TD class="tg-yw4l">GGAGGTTAACAAACTATACAACG</TD><TD class="tg-yw4l">CCTCCAATTGTTTGATATGTTGC</TD></TR><TR><TD class="tg-yw4l">GGTAACGAGACACGTCATAACGG</TD><TD class="tg-yw4l">CCATTGCTCTGTGCAGTATTGCC</TD></TR><TR><TD class="tg-yw4l">GGACGATCTCTACGGCAGCACCA</TD><TD class="tg-yw4l">CCTGCTAGAGATGCCGTCGTGGT</TD></TR><TR><TD class="tg-yw4l">GGTATAGTATTCCCCTTGCTCGG</TD><TD class="tg-yw4l">CCATATCATAAGGGGAACGAGCC</TD></TR></TBODY></TABLE></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>