<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Description</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2016</TITLE></P><DIV class="container"><DIV class="col-md-2"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse text-center" id="bs-example-navbar-collapse-1"><DIV id="nav-menu" class="col-md-10 col-xs-12"><UL class="nav navbar-nav navbar-right"><LI class="active page-scroll"><A href="https://2016.igem.org/Team:EPFL">Home</A></LI><LI class="dropdown page-scroll"><A href="#project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Results">Results</A></LI></UL><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Parts">Parts</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="dropdown page-scroll"><A href="#lab" class="dropdown-toggle" data-toggle="dropdown">Lab</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Medals_Criteria">Medal Criteria</A></LI><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="dropdown page-scroll"><A href="#hp" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Engagement">Engagement</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_CELLO">Cello</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_DATABASE">Database</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model_DESKGEN">DESKGEN</A></LI></DIV></DIV></DIV><DIV class="simple-page"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2 class="lead animate-box text-center">intelligene</H2><P class="sub-lead text-center animate-box">
                            A new way to design biological circuits
                        </P><P class="sub-lead text-justify animate-box">
                          In the world of synthetic biology, it is difficult to not be surrounded by examples of the applications of genetic engineering. After all, anything a cell
                           does can be traced back to its DNA. Many teams are genetically modifying cells to do amazing things, but rather than work on biosensors or cancer, we decided
                           we wanted to make a tool that could help everybody modify their cells faster, more predictably, and more powerfully than ever. As a community, we have become
                           pretty good at modifying cells to incorporate new genes precisely into their genomes, but what if we don’t want to make our cells just produce new proteins or
                           RNAs? What if we want to make cells that are capable of making new decisions, ones they have never made before? The idea is not so outlandish. After all cells
                            make decisions based on their surroundings, their health, and their metabolism all the time as signaling cascades modify the expression of their genes. By inserting plasmids with interconnected genetic elements, we can already make these “decision-making” circuits! Currently, these circuits are built using transcription factors, but finding transcription factors appropriate for this application can prove to be quite a challenge. This is because they must work in the host to target promoters in the new circuit, but also be completely orthogonal to the host genome, to not interfere with the cell’s survival.
                        </P><P class="sub-lead text-justify animate-box">
                          dCas9 has already been used by a variety of authors to act as a sort of “programmable” transcription factor (“Programmable repression and activation of bacterial gene expression using an engineered CRISPR-Cas system”, Nucl. Acids Res. 2013). Since dCas9 uses a guide RNA to find sequences on the DNA, it can be used to guide transcriptional activators or repressors to their targets. As you will see, this can be used to intuitively make complex transcriptional networks. This summer, we harnessed the power of dCas9 and worked to create a suite of tools to make the creation of synthetic genetic networks easier than ever, saving scientists valuable time and reducing the cost of project development.
                        </P><P class="sub-lead text-justify animate-box">
                          To increase the ease of development of new synthetic genetic circuits we also worked on computational tools that aid in their design. Notably, we modified Cello – a program published earlier this year in Science, which automates synthetic genetic network design – to help make it compatible with emerging dCas9-based technologies.
                        </P><P class="sub-lead text-justify animate-box">
                          Intelligene is best thought of as a suite of tools, combining aspects of computational biology and synthetic biology to help scientists with new dCas9-based genetic circuits, from their design all the way through to their implementation.
                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2>General idea of the project</H2></DIV><DIV class="col-md-10 col-md-offset-1"><P class="sub-lead">
                          We based our project on CRISPR/Cas9, a system largely used for genome engineering. Taking the catalytically dead dCas9 from this system allows the targeted binding of almost any DNA sequence. dCas9 is a mutant of the original Cas9 protein that lost its ability to cut DNA, but can still bind to DNA sequences using a specific guide RNA (gRNA). The only limitation for the target is that it has to lie next to a 3-base-pairs “PAM” sequence, NGG, N being any nucleotide. At the molecular level, the gRNA encodes the target sequence which is then ‘loaded’ into the dCas9, which, in turn then, ‘opens up’ the two DNA strands at the target sequence. Changing the sequence of the gRNA means changing which DNA sequence is targeted. In this way, it is possible to target different regions of a promoter. Moreover, dCas9 can be fused with activating transcriptional effectors to increase the expression of the promoter it binds to. This allows for the activation of the downstream gene that is associated to the promoter region.
                        </P><P class="sub-lead">              The main disadvantage of this system was that it only relied on allosteric repression to turn genes off by not allowing the polymerase to bind to the targeted promoter. This type of repression is not entirely reliable and is not performed by a “real” repressor, making it less efficient
                        </P><DIV class="col-md-7 col-md-offset-3 text-center caption"><P class="text-center">
                            Figure 1: Two ways to act on the expression of a promoter through CRISPR/dCas9 classical system. Figure adapted from Zalatan et al. 2015.
                          </P></DIV><P class="sub-lead">
                            With these limitations in mind, we set out to develop a system that was more intuitive, precise, and dynamic.
                        </P><P class="sub-lead">
                          This is exactly what we found in the study of Zalatan et al. (Engineering Complex Synthetic Transcriptional Programs with CRISPR RNA Scaffolds, Cell 2015) as it described a system in which a small hairpin is added to the end of the gRNA which is able to recruit either activators or repressors. RNA binding molecules can then bind to the RNA hairpin and depending on which molecule is recruited, activate or repress the expression of a downstream gene. In this way, a ‘repressing’ gRNA will assure the repression of a gene, and an ‘activating’ gRNA will activate gene expression. This makes the system more intuitive than previous studies that use an ‘activating’ system to repress gene expression.
                        </P><DIV class="col-md-7 col-md-offset-3 text-center caption"><P class="text-center">
                            Figure 2: CRISPR/dCas9 new scaffold system. The gRNA is supplemented with an additional hairpin to be able to recruit transcriptional effectors and becomes a scRNA. In this way, the scRNA does not only determine the sequence of the promoter targeted, but also the effect imposed on the promoter.
                          </P></DIV></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2>How to obtain a <B>modular</B> system?</H2><P class="sub-lead">
                          Modularity is the ability of a system to be composed of simple pieces that can then be recombined to form another one. It is also the key to obtaining an easy-to-use and understandable tool. Something that was unavailable until recently, was a truly modular way to achieve complex biological circuits composed of many different gates. The choice of the promoter was pivotal as the ability to use only one promoter in entire circuits would significantly simplify the system many times over. We had to find a promoter with distinct regions that could be targeted by different gRNAs: some for gene activation and other for repression. The sequences of those regions would also have to be modifiable to avoid cross-talk between all the gRNAs composing a given biological circuit.
                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2 class="">
                            Making it work when you want to,
                            means making it <B>inducible</B></H2><P class="sub-lead">
                          As we now solved the issue of modularity of our system, it would be a lost opportunity to only end up with a tool that is not able to adapt to its surrounding. Adapting to its surrounding means requiring inducible promoters that do not interfere with the vital functions of the cell but still detect the molecule of interest. These promoters would allow researchers to achieve functional biosensors for many applications by triggering distinct responses in the cell according to the signals they detect.

                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2 class="">Finding the right organism to work with</H2><P class="sub-lead">
                          We choose to work in yeasts for two main reasons: firstly, the promising results regarding transcription activation of genes using the new CRISPR/dCas9 system from Zalatan et al. 2015 were obtained in yeast, so we knew that our activation experiments on our modular promoter should function accordingly.
                        </P><P class="sub-lead">
                          Second, yeast is a eukaryotic system which means that the results can be predictably extrapolated to mammalian cells, without their high cost and long replication time. This makes it appealing for many other researchers as they may take advantage of our system to develop new ways to detect and cure diseases in humans for example.
                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2 class="">How to <B>automate</B> the process? </H2><P class="sub-lead">
                          In order to help with the intelligent and intuitive design of these circuits, we decided to modify a recently-published program called Cello (Nielsen et al., Science 2016). Cello works by combining information about the circuit the user wants to create, and biological information about the system the user is building the circuit for. After putting this information through a series of algorithms, it produces plasmids which contain an optimized biological form of that circuit. While we loved the concept and its powerful design, we thought that certain aspects could be built upon to increase user-friendliness and embellish its open-source nature. To this end, we created a new, simple, graphical user interface, and connected Cello with databases of dCas9-based parts we created, which make the information it uses public and easily transferrable between users. These databases help to expand Cello’s capabilities by allowing it to handle emerging dCas9-based technologies. <A href="https://2016.igem.org/Team:EPFL/Software_CELLO">Click here to learn more about our tools!</A></P></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2 class="">Going further with activation and repression: <B>proof of concept</B></H2><P class="sub-lead">
                          As a proof of concept, we chose to bring together all the preceding steps to build a simple logic gate: the NOT gate. This NOT gate will be inducible by galactose. The final step of our project should prove that all the foregoing work done functions accordingly and can simply be integrated in the automatically biological circuits design. The NOT gate is also interesting because in electrical circuits, one can build any possible gate with a NOT and a XOR gate.  From all of this, reaching the final goal of creating even more complex biological circuits in living cells is only a few steps away.
                        </P></DIV></DIV></SECTION></DIV><DIV class="simple-page animate-box"><SECTION><DIV class="container"><DIV class="col-md-10 col-md-offset-1"><H2 class="">Cool! But is it useful?</H2><P class="sub-lead">
                          Sure it is! Designing and constructing gates is one of the trickiest steps in the synthetic biology. And with this project, it just became easier! Once your gates are designed and ready, you can make a lot of things such as strong biosensors, or even detection methods for diseases or substances. And these are only few examples of the large applications of biological circuits.
                        </P></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>