<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_EPFL_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:EPFL/Protocols</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM EPFL 2016</TITLE></P><DIV class="container"><DIV class="col-md-2"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse text-center" id="bs-example-navbar-collapse-1"><DIV id="nav-menu" class="col-md-10 col-xs-12"><UL class="nav navbar-nav navbar-right"><LI class="active page-scroll"><A href="https://2016.igem.org/Team:EPFL">Home</A></LI><LI class="dropdown page-scroll"><A href="#project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Proof">Proof of Concept</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Results">Results</A></LI></UL><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Parts">Parts</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Team">Team</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Collaborations">Collaborations</A></LI><LI class="dropdown page-scroll"><A href="#lab" class="dropdown-toggle" data-toggle="dropdown">Lab</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Medals_Criteria">Medal Criteria</A></LI><LI class="page-scroll"><A href="https://2016.igem.org/Team:EPFL/Attributions">Attributions</A></LI><LI class="dropdown page-scroll"><A href="#hp" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Engagement">Engagement</A></LI><LI class="dropdown page-scroll"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software">Software</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_CELLO">Cello</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Software_DATABASE">Database</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:EPFL/Model_DESKGEN">DESKGEN</A></LI></DIV></DIV></DIV><DIV class="simple-page"><SECTION class=""><DIV class="container"><DIV class="row"><DIV class="col-md-3"><LI><A href="#singlecolonystreak">Single Colony Streaks</A></LI><LI><A href="#spreadingcells">Spreading cells</A></LI><LI><A href="#pcr">PCR</A></LI><LI><A href="#pcrpurification">PCR purification</A></LI><LI><A href="#pcrcolony">PCR colony</A></LI><LI><A href="#gibsonassembly">Gibson Assembly</A></LI><LI><A href="#transformationintocompetentcells">Transformation into competent cells</A></LI><LI><A href="#lbmedium">LB medium</A></LI><LI><A href="#lbagarplate">LB agar plates</A></LI><LI><A href="#plasmidpurification">Plasmid Purification</A></LI><LI><A href="#restrictionenzymes">Restriction Enzymes</A></LI><LI><A href="#yeastsintegration">Yeast Integration</A></LI><LI><A href="#facs">FACS Measurement</A></LI></DIV><DIV class="col-md-9"><H2 class="lead text-center">Protocols</H2><H3 id="singlecolonystreak">Single Colony Streaks</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> Glycerol stock or culture of the desired bacteria </LI><LI> Bunsen burner </LI><LI> Inoculation loop </LI><LI> Plate with correct selection </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Put on the Bunsen burner and always work in the sterile environment it provides. </LI><LI> Heat the loop to red hot and cool it down on the edge of the plate until it doesn‚Äôt fizz or melt the agar anymore. </LI><LI> Dip the loop into the glycerol stock or the culture. </LI><LI> Draw 4 to 6 lines on the top of the plate. </LI><LI> Sterilize the loop by putting it into the flame until red hot. Don‚Äôt dip the loop in the glycerol stock or culture again! </LI><LI> Start from the end of the first lines and draw 4 to 6 additional lines in the same manner as step 4. </LI><LI> Repeat the operation 3 times until you form a shape similar to a pentagon on the plate. Be careful to always flame between the different streaks! </LI><LI> Incubate the plates overnight at 37¬∞C. </LI></OL></P><H3 id="spreadingcells">Spreading cells</H3><H5>Materials</H5><P class="sub-lead"><UL><LI>culture of cells </LI><LI> Bunsen burner </LI><LI> Spreader or autoclaved glass beads </LI><LI> Plate with correct selection </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Put on the Bunsen burner and always work in the sterile environment it provides. </LI><LI> Place the relevant volume of culture in the center of the plates, usually 200 ul for bacteria. </LI><LI> If you are working with a  spreader, remove it from alcohol and let it drain briefly. </LI><LI> Pass the spreader through the pilot flame. </LI><LI> Open the plate and check that the spreader is not too hot by pressing it against the agar on the edge of the plate and waiting until it doesn‚Äôt fizz or melt the agar anymore. </LI><LI> Spread the liquid all over the plate while avoiding that all the liquid end up at the edge of the plate. </LI><LI> Replace the lid and replace the spreader in alcohol. </LI><LI> If you are working with glass beads, don‚Äôt take into consideration steps 2 to 4 : pour the clean beads on the plate, replace the lid and spread the liquid by shaking the plate in a circular motion. </LI><LI> Put the glass beads back in a bottle and send for autoclaving. </LI><LI> Incubate the plates upside down to avoid condensation to fall on the growing cells at the appropriate temperature for as long as required by the organism. </LI></OL></P><H3 id="pcr">PCR</H3><H5>Materials</H5><P class="sub-lead"><UL><LI>Thermocycler </LI><LI> DNA polymerase Q5 (NEB) </LI><LI> 5X Q5 Reaction Buffer (NEB) </LI><LI> 10 uM forward and reverse primers </LI><LI> 10 mM dNTPs </LI><LI> Template DNA</LI><LI> Nuclease-free water </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> In 0,2 ml PCR tubes on ice, add 5 ul Q5 reaction buffer, 0.5 ul dNTPs, 1.25 ul of each primer, 1 ng of template DNA and fill the tube up to 24.75 ul with nuclease-free water. For negative control, add everything to the tube apart from the template DNA. </LI><LI> Before adding the polymerase, make sure to have an appropriate PCR program already set up on the thermocycler. If not, follow these guidelines:
                                    </LI><LI> Initial denaturation is performed at 98¬∞C for 30 sec, and denaturation during the cycles is also at 98¬∞C for about 10 sec. </LI><LI> Annealing temperature depends on the primer sequence and should be determined using NEB Tm calculator and choosing the right polymerase (Q5), buffer (Q5) and primer concentration (500 nM). Annealing time is around 10 to 30 sec, 10 sec if the annealing temperature is low, and 30 if it is high</LI><LI> Extension time is performed at 72¬∞C and is around 30 sec per kb. To be sure extension is complete, increase a bit the final extension time. Final extension is also at 72¬∞C for 5 min.</LI><LI>Number of cycles is normally 25 to 35. </LI></OL><LI> Add 0.25 ul of the Q5 polymerase, flick and spin down shortly the tubes. </LI><LI> Put the tubes in the thermocycler previously pre-heated and start the program. </LI><LI>When the program is finished, run a gel with the PCR products to check that the PCR worked and store at 4¬∞C. </LI></P><H3 id="pcrpurification">PCR purification</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> PCR product </LI><LI> QIAquick spin column </LI><LI> Microcentrifuge </LI><LI> PB buffer </LI><LI> PE buffer (ethanol added) </LI><LI> EB buffer</LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Add 5 volume of PB buffer to 1 volume of PCR product and mix well. </LI><LI> If pH indicator was added to the PB buffer, the solution should be yellow. If the solution is orange or violet, add 10 ul 3M sodium acetate pH 5 and mix. The color will turn yellow.</LI><LI> Apply the sample to the QIAquick spin column and centrifuge for 60 sec at 13‚Äô000 rpm. DIscard the flow-through and put the column back in the collection tube. </LI><LI> Wash the column by adding 750 ul PE buffer to the column and centrifuge again for 60 sec. Discard the flow through and put the column back in the collection tube. </LI><LI>To remove all wash buffer, centrifuge for 1 more min. </LI><LI> Place the column in a clean 1.5 ml centrifuge tube. </LI><LI> For elution of bound DNA, apply 50 ul EB buffer to the center of the membrane of the column. Centrifuge for 1 min. </LI><LI> Run a gel to check that the purification was successful and determine the concentration of your sample with a Nanodrop for further </LI></OL></P><H3 id="pcrcolony">PCR Colony</H3><H5>Materials</H5><P class="sub-lead"><UL><LI>Thermocycler </LI><LI> Taa polymerase  </LI><LI> 10X Thermopol reaction buffer (NEB) </LI><LI> 10 uM forward and reverse primers </LI><LI> 10 mM dNTPs </LI><LI> Plates with colony of interest</LI><LI> Nuclease-free water </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> If no previous colony PCR program has been set, follow these guidelines to create one before starting:
                                    </LI><LI> Initial denaturation is performed at 95¬∞C for 7 min, and denaturation during the cycles is also at 95¬∞C for about 30 sec.</LI><LI> Annealing temperature depends on the primer sequence and should be determined using NEB Tm calculator and choosing the right polymerase (Taq), buffer (Thermopol) and primer concentration (200 nM). Annealing time is about 1 min. </LI><LI> Extension time is performed at 68¬∞C and is around 1 min per kb. To be sure extension is complete, increase a bit the final extension time. Final extension is also at 68¬∞C for 5 min.</LI><LI> Number of cycles is normally 35.</LI></OL><LI> Working on ice, prepare a MasterMix containing for each PCR reaction :
                                    </LI><LI> 2.5 ul of reaction buffer </LI><LI> 0.5 ul of each primer</LI><LI> 0.125 of polymerase</LI><LI>20.875 ul water </LI><LI> Distribute 24 ul MasterMix in each tube and label them from 1 to the number of reaction performed. </LI><LI> Number every colony that will be picked. </LI><LI>Under the flame, pick half of one colony and put it in the tube with the same number. Repeat the operation until every colony has been picked. </LI><LI> Flick and spin down shortly the samples before putting them into the thermocycler previously pre-heated. Start the corresponding program.</LI><LI> Run a gel with PCR products to check that the PCR colony went right and that the plasmid is present in the bacteria.</LI></P><H3 id="gibsonassembly">Gibson Assembly</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> Gibson master mix from NEB 2X </LI><LI> DNA fragments </LI><LI> Deionized Water </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Take 100 ng of vector plasmid and add a ratio between 1:3 and 1:6 of insert in pmol. To do so, first calculate the number of pmol that corresponds to 100 ng of vector with the following formula : 100 (mass of vector) * 1000 / (650 * number of nt of the vector). Then multiply by the ratio to have the number of pmol of the insert and convert this number back in ng with this formula : pmol of insert * 650 * number of nt of the insert / 1000. </LI><LI>Add 10 ul of gibson assembly master mix and complete the reaction with water until 20 ul.</LI><LI> Put the samples for one hour at 50¬∞C. </LI><LI> Integrate directly in competent bacteria or store at -20¬∞C until doing so. </LI></OL></P><H3 id="transformationintocompetentcells">Transformation into competent E.coli cells (Dh5Œ±)</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> Chemically competent E.coli cells (DH5Œ±) </LI><LI> Plasmid to integrate </LI><LI> SOC medium (2% Tryptone, 0.5% Yeast extract, 10mM NaCl, 2.5mM KCl, 10 mM MgCl2, 10 mM MgSO4, 20 mM glucose) </LI><LI> LB ‚Äì Ampicillin plates (1.0% Tryptone, 0.5% Yeast Extract, 1.0% NaCl pH 7.0; 100 Œºg/ml ampicillin; 15 g/L agar)</LI><LI> Heating block </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Thraw competent cells on ice for 10 minutes. </LI><LI> Add 5 uL DNA to the competent cells, mix by pipetting up and down or flicking the tube 4-5 times.</LI><LI> Incubate on ice for 30 min. </LI><LI> Heat-shock the cells for 35 sec at 42¬∞C. Immediately transfer the tube back on ice for 5 min.</LI><LI> Add 800 uL of SOC medium at room temperature.</LI><LI> Incubate the samples for 45 min at 37¬∞C with shaking (225 rpm). </LI><LI><A href="#spreadingcells">Spread </A> 200 uL on selection plates (warm plates to 37¬∞C prior to this step for increased efficiency). </LI><LI> Incubate overnight at 37¬∞C. </LI></OL></P><H3 id="lbmedium">LB Medium</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> 4g NaCl </LI><LI> 4g Tryptone </LI><LI> 2g yeast extract </LI><LI> 400 ml dH2O</LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Weigh out NaCl, Tryptone and yeast extract and add to a 500 ml bottle. </LI><LI> Add part of the water and mix well.</LI><LI> Add the rest of the water. If the powder is not completely dissolved, use a magnetic stirrer along with a magnetic stirring rod to achieve an homogenous solution. </LI><LI> Autoclave the medium.</LI></OL></P><H5>Notes</H5><P class="sub-lead">
                                Always open the bottle of medium under the flame to avoid contamination.
                                To reduce the risk of contamination by decreasing the number of times the bottle is opened, it is also possible to aliquot the medium in smaller bottles.
                            </P><H3 id="lbagarplate">LB agar ampicillin plates</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> 5g NaCl </LI><LI> 5g Tryptone </LI><LI> 2.5g yeast extract </LI><LI> 7.5g Agar</LI><LI> 500 ml dH2O </LI><LI> ampicilin </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Weigh out NaCl, Tryptone, yeast extract and agar and add to a 500 ml bottle. </LI><LI> Add part of the water and mix well.</LI><LI> Add the rest of the water. If the powder is not completely dissolved, use a magnetic stirrer along with a magnetic stirring rod to achieve an homogenous solution. </LI><LI> Autoclave the medium.</LI><LI> After autoclaving, put the bottle at 55¬∞C in a water bath to cool.</LI><LI> When the solution has reached the desired temperature, add ampicillin at a concentration of 100 ug/ml, corresponding to 50 mg for 500ml. </LI><LI> Under the flame, pour 20 ml of medium in a Petri Dish while avoiding bubbles. Repeat the operation until there is no medium left : it should make approximately 25 plates. </LI><LI> Let the plates cool for 30 minutes until they are solidified, then invert them and label them. </LI><LI> To store the plates, seal them with parafilm and put them in a plastic bag labelled with the date at 4¬∞C. </LI></OL></P><H3 id="plasmidpurification">Plasmid Purification (Miniprep)</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> Bacterials pellet from overnight culture </LI><LI> QIAprep Spin Miniprep Kit components:
                                    </LI><LI> Resuspension buffer P1 + RNase A </LI><LI>Lysis buffer P2</LI><LI>Neutralizing buffer N3</LI><LI>Wash buffer PB</LI><LI>Wash buffer PE + ethanol (96-100%)</LI><LI>Elution buffer EB (10 mM Tris¬∑Cl, pH 8.5) </LI><LI>  QIAprep spin column</LI><LI>  Collection tube</LI><LI>  1.5ml microfuge tube</LI><LI> Microcentrifuge </LI>
                            Plasmid purification is performed according to the QIAprep Spin Miniprep kit protocol (QIAGEN).
                            </UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Transfer 1.5ml bacterial overnight culture into microfuge tube and centrifuge at 13‚Äô000 rpm for 3 min at room temperature. A pellet will form. Remove the supernatant, add 1.5ml bacterial culture and centrifuge again. </LI><LI> Resuspend pelleted bacterial cells in in 250 Œºl Buffer P1. Vortex until solution become homogenous.</LI><LI> Add 250 Œºl Buffer P2 and mix thoroughly by inverting the tube 4-6 times. Allow the lysis reaction to proceed for 5 min (no longer). </LI><LI> Add 350 Œºl Buffer N3 and mix immediately and thoroughly by inverting the tube 4‚Äì6 times.</LI><LI> Centrifuge for 10 min at 13‚Äô000 rpm.</LI><LI> Apply the supernatant to the QIAprep spin column.</LI><LI> Centrifuge for 30s, discard the flow-through and put column back into collection tube. </LI><LI> Wash the QIAprep spin column by adding 500Œºl Buffer PB and centrifuge for 30s at 13‚Äô000rpm. Discard the flow-through and put column back into collection tube. </LI><LI> Wash the QIAprep spin column with 750Œºl Buffer PE and centrifuge for 30s at 13‚Äô000 rpm. Discare the flow-through and put column back into collection tube. </LI><LI> Centrifuge for 1 min to remove residual wash buffer.</LI><LI> Place the QIAprep column in a clean 1.5 ml microcentrifuge tube. To elute DNA, add 50 Œºl Buffer EB to the center of the QIAprep spin column, let stand for 1 min, and centrifuge for 1 min.</LI></OL></P><H3 id="restrictionenzymes">Restriction Enzymes Digestion</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> Restriction enzyme </LI><LI> 10X enzyme buffer</LI><LI> DNA of interest </LI><LI>Water</LI><LI>Heating block</LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Take the appropriate amount of DNA of interest, depending on what it will be used for. For a regular restriction enzyme analysis, take 1 ug. For integration in yeasts, take between 1 and 4 ug. </LI><LI> Add 5 ul of enzyme buffer.</LI><LI> Calculate how much of restriction enzyme is required, knowing that 1U of enzyme is needed to digest 1 ug of DNA in 1 hour at 37¬∞C. </LI><LI> Fill the reaction to 50 ul minus the amount of enzyme needed.</LI><LI> Heat the heating block to 37¬∞C (or the temperature required by the enzyme).</LI><LI> When the heating block is ready, add the enzyme to the reaction and let  incubate at the appropriate temperature for 15 min (if enzyme is Time-Saver qualified by NEB) or 1 hour.</LI><LI> Put on ice to stop the digestion and run a gel with the digested sample next to the undigested one to check that the digestion worked. </LI></OL></P><H3 id="yeastsintegration">Yeast Integration</H3><H4> Step 1: Making Competent yeasts</H4><H5>Materials</H5><P class="sub-lead"><UL><LI> Appropriate medium for yeasts (selective or non-selective, depending on the strain) </LI><LI> Plated yeast strain</LI><LI> Glucose 40% </LI><LI>dH2O</LI><LI>10X TE buffer (100mM Tris, 10mM EDTA, pH 8.0)</LI><LI> 1M LiAc </LI><LI> Salmon sperm DNA (ssDNA)</LI><LI> DMSO </LI><LI> Spectrophotometer</LI><LI> Centrifuge</LI></UL></P><H5>Procedure</H5><H4>Day 1</H4><P class="sub-lead"><OL><LI> At 15:00 approximately, start pre-culture by making a solution of yeast medium and glucose 40 % : put 2.5 mL of glucose in an erlenmeyer and add 47.5 mL of medium with one big colony of the plated yeasts. </LI><LI> Let the preculture grow overnight at 30¬∞C on a shaker.</LI></OL></P><H4>Day 2</H4><P class="sub-lead"><OL><LI> At 11:00, check the OD of preculture by measuring absorbance at 600nm with a spectrophotometer. </LI><LI> If the absorbance is between 1.6 and 1.8, seed yeast in medium and add glucose as such : 450 mL YEP  + 25 mL 40 % glucose + 25 mL of pre-culture. If not, seed enough preculture in the new medium to have a final OD around 0.15-0.2 and a total volume of 500 mL.</LI><LI> Grow at 30¬∞C with a shake for about 3h until OD of the culture is 0.6-0.8.</LI><LI> Meanwhile, if using a non-wild type strain, prepare the selective plates on which the yeasts will be plated according to their specific marker.</LI><LI> Spin yeasts down for 5‚Äô at 1800 rpm by splitting the culture into 50 mL tubes. Pour off medium and check that the medium is clear. If not, you may have a bacterial contamination.</LI><LI> Wash yeast with 25 mL of dH2O. Resuspend and pool the two tubes together.</LI><LI> Spin yeast down 5‚Äô at 1800 rpm.</LI><LI> Wash yeasts with 10 mL TE/LiAc. To make TE/LiAc, add 1 part of LiAc to 1 part of TE and 8 parts water.</LI><LI> Spin yeasts down 5‚Äô at 1800 rpm.</LI><LI>Resuspend yeast in TE/LiAc solution according to the following formula : <B>OD600 x culture volume /60 = volume</B> in which to resuspend in ml. When resuspending the cells, be extracareful and never suck up the cell pellet. This step is critical for the success of the integration.
                                </LI><LI> Add 10%vol of ssDNA, previously boiled at 100¬∞C for 5‚Äô and cooled on ice.</LI><LI> To freeze yeast, add 10% of the final volume of DMSO and put at -80¬∞C. Aliquot by 250ùúáL and put paper around each tube in a styrofoam rack.</LI></OL></P><H4> Step 2: Plasmid linearization by restriction digestion accesskey</H4><H5>Materials </H5><P class="sub-lead"><UL><LI> Plasmid of interest</LI><LI> Restriction enzyme</LI><LI> RE buffer (according to the enzyme)</LI><LI> Heating block</LI></UL></P><H5>Procedure </H5><P class="sub-lead"><OL><LI> In an eppendorf tube mix 4ùúág of the plasmid you want to digest, 4ùúál of enzyme, 10ùúál of buffer and water until reaching a final volume of 100ùúál.
                                    If you want to perform a double digestion as a further control, mix 0.5ùúág of plasmid, 0.5ùúál of each enzyme, 5ùúál of buffer and water until reaching a final volume of 50ùúál. Note: check that both enzymes work in the same buffer. Otherwise, choose the best compromise for both enzymes.</LI><LI> Incubate at 37¬∞C for 1 hour.</LI><LI>Run a gel to make sure that the plsmid linearized correctly. As a negative control, load also the undigested plasmid.</LI></OL></P><H4> Step 3: Yeast genomic integration</H4><H5>Materials </H5><P class="sub-lead"><UL><LI> Competent yeast cells</LI><LI> Linearized plasmid</LI><LI> 10X TE buffer (100mM Tris, 10mM EDTA, pH 8.0)</LI><LI> Lithium acetate 10X</LI><LI> PEG </LI><LI> Sterile water GIBCO </LI><LI> Selective plates</LI><LI> Heating block </LI><LI> Microcentrifuge</LI></UL></P><H5>Procedure </H5><P class="sub-lead"><OL><LI> Take 200 ùúál of yeast culture per integration. If frozen, quickly  thaw at 42¬∞C. Don‚Äôt heat yeasts for more than 30s: the tube must stay cool.</LI><LI> Add the 4 ùúág of linearized vector to the yeasts.</LI><LI>Make a solution of 1ml of TE, 1 ml of lithium acetate and 8ml of PEG.</LI><LI> Add 1ml of the solution and resuspend by pipetting up and down very carefully. This step is critical for the success of the integration.</LI><LI> Incubate 30‚Äô at 30¬∞C at 200 rpm.</LI><LI> Heat shock the cells at 42¬∞C for 20‚Äô exactly in a water bath.</LI><LI> Centrifuge quickly (about 15s)  the tubes at full speed at room temperature. Remove supernatant.</LI><LI> Resuspend the cells by pipetting up and down in 100 ùúál of sterile water (GIBCO).<B> Do not vortex. </B><A href="#spreadingcells">Plate</A> 50 ùúál on appropriate selective media plates. Before doing so, if amino acids were added, be sure the plate is totally dry and all the amino acid(s) solution has been absorbed. Incubate the plates for 3 days at 30¬∞C.</LI></OL></P><H3 id="facs">FACS(Fluorescence-activated cell sorting)</H3><H5>Materials</H5><P class="sub-lead"><UL><LI> Yeast selective media 2% glucose </LI><LI> Yeast selective media 2% galactose for induction </LI><LI> Shaking incubator 30¬∞C </LI><LI> Centrifug e</LI><LI> PBS </LI><LI> 5ml FACS tubes </LI></UL></P><H5>Procedure</H5><P class="sub-lead"><OL><LI> Grow cells from a single colony overnight in 3ml selective media. For GAL1 promoter induction, use the media supplemented with 2% galactose.</LI><LI> Spin down the cells at 1800rpm for 5‚Äô the and remove the supernatant.</LI><LI> Wash with 1ml of PBS, spin down again and remove the supernatant. </LI><LI> Resuspend the pellet with 1ml of PBS and transfer in a 5ml FACS tubes.</LI></OL></P></DIV></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>