"
Page
Discussion
View source
History
teams
Log in
 
Team:Dundee/Project
From 2011.igem.org
The University of Dundee
iGem 2011
Project
Team
Modelling
Notebook
Data
Safety
Sponsors
DNA App
Software
Human Practices
Project Description:
Sphereactor: a synthetic bacterial microcompartment
It's a box, not a bug!
In the Beginning…
Over millennia, eukaryotic cells have evolved sophisticated organelles, which enabled them to partition their cytoplasmic contents into functional sectors (e.g. the nucleus for storage of genetic material). Such compartmentalisation allows greater efficiency of cellular processes, where each organelle is allocated a set of specific metabolic tasks. Some prokaryotes have also developed a method of forming intracellular subdivisions called bacterial microcompartments (BMCs) by expressing a set of proteins that ‘cage in’ a reaction pathway to make it more efficient. A wide range of metabolic pathways are isolated by BMCs, from carbon dioxide fixation pathways to organic compound degradation. Despite these varying functions, the proteinaceous BMC shells show considerable homology as a result of a highly conserved protein domains. [1]
One such set of proteins is expressed from the propanediol utilisation (Pdu) operon in Salmonella enterica, which is involved in the coenzyme B(12)-dependent breakdown of the organic compound 1,2-propanediol. S. enterica synthesises its microcompartments during aerobic or anaerobic growth on 1,2-propanediol, but does not form them when grown on other carbon sources. Enzymes vital for the catabolic pathway are thought to be enclosed within the BMC. [2]
Figure 1: Gene organization in the Pdu Operon and Pdu metabolic pathway. [1]
The Pdu operon is composed of 21 genes, which encode proteins involved in the Pdu microcompartment (Figure 1) either in formation of the shell or in the metabolic pathway contained inside. Of these 21 genes, seven have been identified as having a role in BMC shell assembly. These genes code for the proteins PduA, -B, -J, -K, -N, -T, and -U plus a truncated version of PduB termed PduB’. [3]
It is thought the Pdu BMC shell adopts an icosahedral shape, with multiple flat triangular faces and 12 vertices. With the exception of PduN, the above Pdu shell proteins form a mosaic of hexagonal building blocks that associate into flat sheets to form the facets of the icosahedron. While the single-BMC-domain proteins PduA, -J, -K, and -U (Figure 2) form protein hexamers, PduB, -B’, and -T have tandem BMC domains so instead form hexagonal protein trimers. [4]
Figure 2: Crystal Structure of the PduU Shell Protein. [5]
Small pores, approximately 10Å in diameter, run through the centre of some hexagonal units and are thought to allow transport of molecules across the protein shell.[3] PduN forms pentamers that occupy the 12 vertices of the icosahedron and is therefore considerably less abundant than the other Pdu shell proteins which contribute hundreds of copies to the faces of the BMC. [4]
Figure 3: Icosahedral Bacterial Microcompartment [6]
New Biobricks and Improving Old Biobricks
The Grand Plan: A Synthetic Microcompartment - ''The Sphereactor''
To begin with, our team aims to produce a BMC comprised of the eight Salmonella Pdu shell proteins (Pdu-A, -B, -B’, -J, -K, -N, -T, and -U) in Escherichia coli, which itself is a close relative of Salmonella. This involves expressing seven Pdu genes, as PduB’ is expressed from the PduB gene sequence. Our intention is to add a poly-histidine affinity tag onto the C-terminus of several compartment proteins, which will aid in determining whether the proteins have been expressed and may allow us to isolate complete compartments from the cell by affinity purification.
In order to achieve expression of the BMC in E. coli, we will construct a biobrick that contains all seven genes for the Pdu shell proteins, including sequences encoding the poly-histidine affinity tags. In order to create our biobrick, we will employ a progressive strategy involving construction of individual Pdu gene biobricks and sequential addition of these biobricks to form our complete synthetic operon (Figure 4). As the majority of the Pdu shell genes lie adjacent to one other shell gene in the Pdu operon of Salmonella, it is possible to clone these genes as pairs. Therefore our initial individual biobricks will consist of PduAB, PduJK, PduN and PduTU.
Figure 4: Construction of Biobricks.
Targeting to the Sphereactor
Once the empty BMC is expressed and characterised, we would like to take the project in a number of different directions, as we feel the microcompartment has broad potential. One issue which was addressed early on was how other proteins could be localised to the interior of the BMC. For example, if the BMC was to be utilised as a microbial factory to manufacture a useful product, how could enzymes and substrate molecules be directed into the empty microcompartment? Studies of the Pdu operon have shown that some proteins, such as PduD, are recruited to the interior of the BMC with the help of an N-terminal signal sequence. Targeting of other proteins to the inside of the microcompartment can be achieved by incorporation of this N-terminal sequence into the N-terminus of the target protein7. We aim to construct biobricks of two PduD N-terminal sequences: The first, PduD40, will encode 40 amino acids from PduD’s N-terminus, and the second, Pdu20, will code for only the first 20 amino acids.
PduD20: MEINEKLLRQIIEDVLRDMK
PduD40: MEINEKLLRQIIEDVLRDMKGSDKPVSFNAPAASTAPQTA
Individually fusing PduD20 and PduD40 to a reporter gene, for example GFP, will hopefully allow us to show whether the essential sequence for targeting proteins to the microcompartment is in the first 20 amino acids or the first 40 amino acids.
Figure 5: PduD signal sequence.
Packing the Sphereactor
One idea we would like to explore is that of making the compartments magnetic by targeting iron-rich proteins into them. This would allow us to collect the bacteria and/or compartments using a magnet after they have carried out their task, which could be a very useful attribute (e.g. in bioremediation).  Ferritin is a large globular protein that stores iron and releases it in a controlled manner; HybA is another such protein. By fusing the PduD targeting sequence to Ferritin and HybA we will hopefully succeed in packing the BMC with the iron-rich proteins, potentially resulting in a magnetic microcompartment. We will additionally try to target a heat-activated protease (DegP) into the compartment, which should degrade the ferretin and leave behind only the metal particles.
Figure 6: Ferritin multimer (24mer) (left) and Ferritin monomer (right). [8]
Another idea we are interested in is targeting the fluorescent proteins GFP and mCherry into our microcompartment. GFP and mCherry expression will allow us to visualise our microcompartments and test the efficiency of our PduD targeting sequence as a localisation signal. In order to compare GFP/mCherry expression in the cytoplasm to expression in the microcompartments we will use an SSRA signal. Fusion of this SSRA signal to GFP will target it to CLIP proteases in the cytoplasm resulting in degradation of the fluorescent protein. Therefore only GFP molecules protected within the BMC will be expressed and so any fluorescence detected in the cell will confirm expression of the microcompartment. However this is dependent on GFP reaching the microcompartment before it is degraded.
In conjunction with biobricks made by other iGEM groups in the past, we aim to test the versatility of our microcompartment and to expand our BMC approach into a wide variety of new applications. For example, we hope to use part BBa_K190019 from the registry with our own PduD part to target fMT into our microcompartment. fMT is an arsenic-binding protein which could play an important role in the purification of water polluted with heavy metals, an idea effectively tackled by the Groningen team in 2009. If we could target fMT to the inside of our BMCs then remove them from the E. coli cells using the poly-histidine affinity tag, we will have created a water purifying microcompartment. Isolating functional microcompartments from the cells might avoid many of the complications that accompany introduction of genetically engineered bacteria into the environment.
Other parts we hope to manipulate include BBa_I742111 and BBa_C0083, which code for limonene and aspartate ammonia-lyase, respectively. These parts will be used to create bacterial lemonade, although the human guinea pigs that will have to drink it have not been nominated yet. We are combining the lemon smell (limonene) with a sweet taste (either “diet” aspartame or “full-fat” glucose isomerise) and targeting the two to the microcompartment. The idea is to concentrate the flavours in the microcompartments and then purify them so that the bacterial metabolites do not end up in the final drink.
In vitro applications: it's a box, not a bug!
One final idea we would like to explore is to remove entirely from the equation any concerns about releasing genetically-modified organisms (GMOs) into the environment. We will design our sphereactor so that it can be isolated and used for cell-free applications. For example, a single sphereactor could be filled with any number of enzymes - a complete multi-step pathway perhaps - but as a proteinaceous complex would be unable to modify or recombine with natural organisms.
Relevant, modern and cool software for all
In addition, through collaboration with expert applied computing scientists, we have developed software, which will help in the design and construction of our biobricks. They are also the ones responsible for the design of this superb website and two brilliant smartphone apps!
Find us on Facebook
Follow us on Twitter
Retrieved from " http://2011.igem.org/Team:Dundee/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
