<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Dundee_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Dundee/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Dundee/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Dundee/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Dundee/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Dundee/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="wrapper"><DIV id="header"><H1><A href="/Team:Dundee">The University of Dundee</A></H1><DIV class="text1"><H2><SPAN>iGem 2011</SPAN></H2></DIV></DIV><DIV id="menu"><UL><LI><A href="/Team:Dundee/Project" title="Project">Project</A></LI><LI><A href="/Team:Dundee/Team" title="Team">Team</A></LI><LI class="current_page_item"><A href="/Team:Dundee/Modelling" title="Modelling">Modelling</A></LI><LI><A href="/Team:Dundee/Notebook" title="Notebook">Notebook</A></LI><LI><A href="/Team:Dundee/Results" title="Data">Data</A></LI><LI><A href="/Team:Dundee/Safety" title="Safety">Safety</A></LI><LI><A href="/Team:Dundee/Sponsors" title="Sponsors">Sponsors</A></LI><LI><A href="/Team:Dundee/dnaapp" title="DNA App">DNA App</A></LI><LI><A href="/Team:Dundee/Software" title="Software">Software</A></LI><LI><A href="/Team:Dundee/HumanPractices" title="Human Practices">Human Practices</A></LI></UL></DIV><DIV id="container"><DIV id="content"><DIV class="largepost2"><H2>Modelling the Sphereactor</H2><H3>Structure of a Pdu Microcompartment</H3><P>One approach to modelling we decided to take, was to investigate the structure of a Pdu microcompartment and its size relative to an E.coli chassis. Pdu microcompartments are thought to be very similar structurally to carboxysomes, the microcompartments involved in carbon fixation in autotrophic cyanobacteria and some chemoautotrophic bacteria. Comparisons of shell proteins of carboxysomes and Pdu microcompartments have shown considerable homology, and it is extremely likely that they assemble to form a similar structure. Studies of carboxysomes have shown that they are regular icosahedral structures and it is thought that Pdu microcompartments also assume roughly this shape, with approximate diameter 120nm. <A href="https://2011.igem.org/Team:Dundee/References"><B>[3]</B></A></P><P>A regular icosahedron is a polyhedron with twenty equilateral triangular faces, 30 edges, and 12 vertices.<A href="https://2011.igem.org/Team:Dundee/References"><B>[14]</B></A></P><P>Figure 1: A regular icosahedron and its net. <A href="https://2011.igem.org/Team:Dundee/References"><B>[14]</B></A></P><P>For a regular icosahedron, it is possible to calculate values for the circumradius, (Rc, radius of a sphere touching all of an icosahedron’s vertices), inradius (Ri, radius of a sphere inscribed in an icosahedral that touches all of its faces) and midradius (Rm, radius that touches the middle of each edge) as follows: </P><P>where ‘α’ is the edge length of an icosahedron.</P><P>Say the radius = 60nm. Then using the above formula for the midradius, edge length can be calculated as α  = 74.1641nm. </P><P>The total external surface area of an icosahedron, At, can be found by calculating the area of one equilateral triangular face, Ae, then multiplying by the number of faces.</P><P>An icosahedron is composed of 20 pyramids, each with a height Ri and a base of area Ae. The total volume of the icosahedron, Vt, can be found by calculating the volume of one of these pyramids, Vp, then multiplying by the number of pyramids. </P><P>The size of the sphereactor relative to its E. coli chassis was then investigated. The approximate length, L, and diameter, D, of an E. coli cell are 2μm and 0.8μm respectively [15]. If we assume an E. coli cell has a roughly cylindrical structure with two hemispherical ends, the approximate volume of our chassis can be calculated. </P><P>As the total volume of our sphereactor, Vt, is approximately 889,970nm3 or 8.8897x10-4μm3, we can therefore say that it is roughly 1/979 the size of its E. coli chassis. </P><P>Next, the proportions of the individual proteins that assemble into the shell of the BMC were investigated. It is thought that five molecules of the protein pduN form pentamers at each of the icosahedron’s twelve vertices. Therefore, theoretically there should be 60 pduN protein molecules present in a single microcompartment. 
PduA, -B, -B’, -J, -K, -T and –U all form hexagonal building blocks which associate into flat sheets to form the triangular faces of the icosahedron. While the single-BMC-domain proteins PduA, -J, -K, and -U form protein hexamers, PduB, -B’, and -T have tandem BMC domains so instead form hexagonal protein trimers. <A href="https://2011.igem.org/Team:Dundee/References"><B>[4]</B></A> 
The centre-to-centre length of two hexagonal sub-units varies but is on average 68Å or 6.8nm. <A href="https://2011.igem.org/Team:Dundee/References"><B>[3]</B></A> Using this value, the surface area of the external face of an individual hexagonal building block, Ah, can be calculated. The total external surface area of the icosahedron, At, can then be used to give the approximate number of hexagonal sub-units, Ht, present in the BMC shell. 
</P><P>We already know there are 12 PduN pentamers at the vertices of the icosahedron, so assuming that the total area of a pduN pentamer is similar to that of a hexagonal protein sub-unit, we can say that the total number of pduA, -B, -B’, -J, -K, -T and –U hexagonal sub-units is approximately 1190 – 12 = 1178. The molar ratios of these Pdu proteins in a single BMC are pduA:pduB:pduB’:pduJ:pduK:pduT:pduU = 19.5:11.5:12.5:30.5:2.5:2:3. <A href="https://2011.igem.org/Team:Dundee/References"><B>[16]</B></A> Taking into account the fact that some Pdu shell proteins form trimers while some form hexamers, the number of molecules of each protein present in the shell of our Sphereactor can be calculated.  </P><H3>Synthesis of Pdu Proteins</H3><P>After exploring the structure of our Sphereactor, we next investigated the synthesis of the individual Pdu shell proteins from our synthetic PduABTUNJK operon. In order to express these proteins, the DNA sequence of our PduABTUNJK biobrick is first transcribed into a polycistronic mRNA molecule. This mRNA molecule is then translated into all eight shell proteins, PduA, -B, -B’, -T, -U, -N, -J, and –K by ribosomes.</P><P>One hypothesis for the mechanism of translation of our biobrick by ribosomes, is that the translation of a Pdu protein from its sequence on the polycistronic mRNA strand is dependent on the translation of the Pdu protein preceding it on the mRNA. For example, the first ribosome binds at a ribosome binding site (RBS) at the start of the mRNA strand, then moves along the mRNA strand while synthesising the first protein molecule. Only once this first protein molecule has been produced, can translation of the next protein begin, and so on. This might occur in the case where translation of all 8 proteins is carried out by a single ribosome which binds once at the first RBS at the beginning of the mRNA strand and only dissociates upon reaching the end of the last Pdu gene. Alternatively, a ribosome reaching the end of a Pdu gene sequence may somehow activate the RBS of the next Pdu gene, allowing a second ribosome to bind and start translating while the first ribosome dissociates from the mRNA strand.</P><P>This diagram is simplistic in that it shows only one ribosome bound to the mRNA strand. In reality, it is likely that once the first ribosome has vacated the ribosome binding site, a second would bind, and so on. A series of ordinary differential equations (ODEs) consistent with the above diagram can be written to show how the concentrations of the Pdu proteins change with time. First, an ODE was written to represent the change in mRNA concentration with respect to time. For this model, it is assumed that mRNA production, α, is in steady state, and that mRNA has a rate of degradation, kdm. For simplicity, it is also assumed that the production and degradation of mRNA is balanced. The concentration of mRNA therefore remains constant and so its change with respect to time is equal to zero. </P><P>Seven ODEs representing the change in concentration with respect to time for each Pdu protein are shown below. As PduB and PduB’ are translated from the same Pdu gene, they are treated as one protein represented by a single ODE for simplicity in this model.</P><P>We have chosen our parameters, kt (rate of transcription), kd (rate of protein degradation), kdm (rate of mRNA degradation), α (source of mRNA) and K (Michaelis constant), so that the model behaves in a realistic manner. The orders of our parameters fit with approximate rates discussed in literature <B><A href="https://2011.igem.org/Team:Dundee/References">[17]</A></B> and with the order of parameters used in other studies of protein translation<B><A href="https://2011.igem.org/Team:Dundee/References"> [18]</A></B>.When the concentration of pduA is in steady state, d[A]/dt = 0. Therefore when [A] is in steady state, [A]ss.</P><P>Similarly, when [B/B’] is in steady state, [B/B’]ss</P><P>Continuing in this fashion for every Pdu protein in steady state confirms that in order for all of the Pdu proteins to have the same final steady state value, the parameter Ks must be:</P><P>Ks is included so that every protein is synthesised to the same concentration in steady state, as would be expected. Ks is the same for all of the ODEs because kt and kd have also been assumed to be the same for all of the Pdu proteins. </P><P>In E.coli, translation of mRNA occurs at a rate of ~20aa/s which corresponds to ~60nuc/s. <B><A href="https://2011.igem.org/Team:Dundee/References">[17]</A></B> The total length of the mRNA transcribed from our PduABTUNJK operon is ~3200 nucleotides. Therefore, the approximate length of time for translation of one mRNA strand to occur is ≈ 3200/60 ≈ 53.3s. We can then say it takes ~53.3/7 ≈ 7.6 seconds to translate one Pdu protein. </P><P>Therefore the length of time to produce kt/kd protein molecules ≈ 7.6 x kt/kd secs
If we then choose kt = 4 min-1 and kd = 0.4 min-1, we have the length of time to produce kt/kd protein molecules ≈ 760s ≈ 12.7mins. Therefore, we expect the Pdu proteins in our model to reach steady state at time ≈ 12.7mins.</P><P>This system of ODEs was solved in MATLAB using the ode45solver.</P><P>The above graph shows how the concentration of each Pdu protein changes with respect to time.</P><P>This is only one hypothesis for the mechanism of translation of our PduABTUNJK biobrick. Future work could focus on other hypotheses, for example that a ribosome translates 2 or 3 genes before dissociating.</P><P>We also wanted to address the question of how many Sphereactors we might expect to see in our E. coli cells. By adding a term to the end of each of the above ODEs we were able to account for the assembly of Pdu protein into microcompartments. This assembly term took into account the different proportions of each Pdu protein present in a Sphereactor and affected the final steady states for each protein. We were therefore able to predict that after 15 minutes, there will be approximately 90 Sphereactors in a cell. Our model is simplistic in that it does not account for cell growth but instead assumes the cell to be a static structure with a constant volume.</P></DIV></DIV><DIV id="footer"><DIV class="text2"><H2><SPAN><A href="http://www.facebook.com/pages/Dundee-iGem-Team-2011/165447726856145">Find us on Facebook</A></SPAN></H2></DIV><DIV class="text3"><H2><SPAN><A href="http://twitter.com/#!/DundeeiGEMTeam">Follow us on Twitter</A></SPAN></H2></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Dundee/Modelling">http://2011.igem.org/Team:Dundee/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Dundee/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Dundee/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Dundee/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Dundee/Modelling&amp;oldid=254470" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>