<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Michigan_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Michigan/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Michigan/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Michigan/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Michigan/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Michigan/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">¬†</FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Michigan/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Xayona Website Template</TITLE></P><DIV class="page"><DIV class="header"><DIV id="navigation"><UL><LI class="selected"><A>Project</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Project">Project Description</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Results">Results</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Future">Future Direction</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Modeling">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Parts">Completed Parts</A></LI></UL><LI><A>Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Notebook">Construction</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Construction">Characterization</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Sponsors">Sponsors</A></LI><LI><A href="https://2012.igem.org/Team:Michigan/Outreach">Outreach</A></LI></DIV></DIV></DIV><DIV class="body"><DIV id="basic"><H3>
Introduction
</H3>

In order to test the effectiveness of the recombinases used in our system, we developed two assays -- an ‚Äúasymmetrical digest‚Äù reporter and a fluorescence reporter.  The former is used to directly test the inversion of the target DNA region, while the latter is used to demonstrate inversion induced protein expression. 

<H2>
The Asymmetrical Digest Reporters
</H2>

The two asymmetrical digest reporters in their unflipped state (invertible by FimE) are shown in figures 2 and 3. These reporters rely on an off-center AgeI restriction site which will yield fragments of varying size depending on the orientation of the inversion region. Table 1 shows the expected sizes of the fragments in each orientation. The differences between the reporters are discussed below. 

<DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 1. A diagram of the asymmetrical reporter K880001</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Figure 2. A diagram of the asymmetrical reporter K880003</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Table 1. Expected outcomes for the asymmetrical digest assay of the reporters K880001 and K880003.</DIV></DIV></DIV></DIV><H2>
The Fluorescent Reporters
</H2>
The fluorescent reporters, show in in figures 3 and 4, have a constitutive tetR promoter within the inversion site, allowing for the expression of GFP in the flipped orientation. 

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:299px;"><DIV class="thumbcaption">Fig 3. A diagram of the fluorescent reporter K137058</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:301px;"><DIV class="thumbcaption">Fig 4. A diagram of the fluorescent reporter K137058</DIV></DIV></DIV></DIV><H3>
Results
</H3><H2>
Negative Fluorescent Reporter (BBa_K137058) Data 
</H2>

To initially test the FimE recombinase, we utilized a reporter created by the CalTech ‚Äò08 iGEM team (BBa_K137058) for the same purpose. The K137058 reporter in the vector pSB3C5 was co-transformed in 10-beta <I>E. coli</I> (NEB) with strong, constitutively expressed FimE (K880005-K137007) in the vector pSB1A2.  Despite several co-transformation trials, no fluorescence was observed in contrast to CalTech ‚Äò08‚Äôs results. 
<H2>
Negative Asymmetrically Digestible Reporter (BBa_K880003) Data
</H2>
An asymmetrical digest reporter (K880003) in the vector pSB3C5 based on the inversion region found in K137058 was co-transformed in 10-beta <I>E. coli</I> (NEB) with strong, constitutively expressed FimE (K880005-K137007) in the vector pSB1A2.   Colonies as well as broth subcultures of co-transformants were analyzed for inversion using the asymmetrical digest assay<A href="https://static.igem.org/mediawiki/2012/a/aa/MichAsm.pdf" class="external autonumber" rel="nofollow">[1]</A>.  The asymmetrical digest reporter and <I>fimE</I> are present within the co-transformants; however, the asymmetrical digest reporter remains in the uninverted state (Fig 5, lane 3).  
<DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig 5.  A representative electrophoresis gel of the asymmetric digest assay of K880003 co-transformed with FimE.  Lane 1 is a 100bp Ladder (NEB), lane 2 is K880003 transformed alone, lane 3 is K88003 co-transformed with FimE, and lane 4 is FimE transformed alone.  Uninverted K880003 yields a 357bp band as seen in lane 2 and 3, while inversion induced by FimE is expected to result in a 255 and 238bp band.</DIV></DIV></DIV><H2>
Observation of IRR and IRL Orientation:
</H2>

Scrutiny of the DNA sequence provided a possible explanation of the negative results. 
As constructed, verified by sequencing, the orientations of IRL(K137010) and IRR (K137008) within the inversion region of K137058 were orientated 180¬∫ when compared to the natural <I>fimS</I> region (Gally et al., 1996) and a previously engineered recombinase systems (Ham et al., 2006).  This resulted in the IRL and IRR being oriented such that the external half-sites are localized adjacent to the inversion region, while the internal half-sites are located externally (Fig 6).  We hypothesize that this 180¬∫ orientation of the IRL and IRR is the determinative factor to the inversion negative results.

<DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Fig 6. Comparison of the natural <I>fimS</I> IRR and IRL orientation to that of K137058.  The orientation of IRL (Blue-Yellow-Orange) and IRR (Purple-Yellow-Green) are both exhibiting 180¬∫ orientations in K137058 (B) when compared to the natural <I>fimS</I> region (A).</DIV></DIV></DIV><H2>
Modified Asymmetrically Digestible Reporter (BBa_K880001) Data:
</H2>

To test the hypothesis that the IRL and IRR orientations were the determinative factor to the inversion negative results observed, we created a second asymmetrical reporter (K880001).  K880001 was engineered such that the IRL is situated on the right of the inverted region and the IRR is situated on the left.  This IRL/IRR orientation restores the orientation as found in the natural <I>fimS</I> region (Gally et al., 1996) and a previously engineered recombinase systems (Ham et al., 2006 ).  


The new asymmetrical digest reporter (K880001) in the vector pSB3C5 was co-transformed in 10-beta <I>E. coli</I> (NEB) with strong, constitutively expressed FimE (K880005-K137007) in the vector pSB1A2.  Co-transformants colonies were analyzed for inversion using the asymmetrical digest assay<A href="https://static.igem.org/mediawiki/2012/a/aa/MichAsm.pdf" class="external autonumber" rel="nofollow">[2]</A>.  The asymmetrical digest reporter and <I>fimE</I> (700bp band Fig7, Lanes 2-4) are both present within the co-transformants.  The asymmetrical digest reporter was observed in a partially flipped state exhibiting both the 353bp band as well as the 255/238bp bands (Fig 7, Lane 2-4).  It has been noted that fim inversions induced by FimE in a previously engineered recombinase systems exhibited low efficiency resulting in a mixed population of flipped and unflipped inversion regions (Ham et al., 2008).


To further characterize the new asymmetrical digest reporter (K880001) in the vector pSB3C5 was transformed into <I>E. coli</I> DH5Œ±, a strain with endogenous FimE.  In accordance with the co-transformation experiment, the asymmetrical digest reporter was observed in a partially flipped state (Fig 8, Lane 2).

<DIV class="thumb tleft"><DIV class="thumbinner" style="width:196px;"><DIV class="thumbcaption">10-beta <I>E. coli</I> with FimE.  Lane 1 is a 100bp Ladder (NEB), lane 2-4 are K880001 co-transformed with FimE showing a mixed state indicated the presence the uninverted K880001 yields a 357bp band, while inversion induced by FimE is expected to result in a 255/238bp band.</DIV></DIV></DIV><DIV class="thumb tnone"><DIV class="thumbinner" style="width:118px;"><DIV class="thumbcaption">Fig 8.  An electrophoresis gel of the asymmetric digest assay of K880001 in <I>E. coli</I> DH5ùõº with endogenous FimE.  Lane 1 is a 100bp Ladder (NEB), lane 2 is K880001 showing a mixed state indicated by the presence the uninverted K880001 yielding the 357bp band, and the inverted K880001 resulting in a 255 and 238bp band.</DIV></DIV></DIV><H2>
Negative Modified Fluorescent Reporter (BBa_K880002) Data:
</H2>

To additionally test our hypothesis we made a fluorescent reporter with the IRR and IRL orientation, analogous to K880001. Unexpectedly, we did not observe fluorescence possibly attributable to partially flipped states observed in the K880001 co-transformation assays (Fig 7 and 8 above)

<H2>
Observation of HbiF induced fimbriation in DH5Œ±</H2>

We were hoping to test HbiF on the FimE-flipped reporters however due to only obtaining partially flipped states we were unable to isolate a flipped plasmid to test this.  Upon transformation of <I>E. coli</I> DH5Œ± with strong, constitutively expressed HbiF (K880005-K880000), we noticed cell aggregates in liquid cultures that were difficult to resuspend upon pelleting. Transmission electron microscopy (TEM) of the culture showed that HbiF expressing cells were fimbriated (Fig 9, A and B), suggesting that HbiF is expressed and actively working on the natural <I>fimS</I> region flipping from ‚ÄúOFF‚Äù to ‚ÄúON‚Äù (Xie et al., 2006).   
  
<DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">TEM micrographs of <I>E. coli</I> DH5ùõº cells transformed with HbiF in pSB1A2 (A, B) versus untransformed <I>E. coli</I> DH5ùõº cells (C, D).  Fimbriae are clearly visible in the cells expressing HbiF (A, B), while absent in the control cells (C, D). Bar length indicates 500 nm.</DIV></DIV></DIV><H3>Conclusion</H3>
We have designed a protein expression system utilizing unidirectional recombinases HbiF and FimE, and have tested for the activities of both recombinases by creating two asymmetric digest assays. The effect of recombinase activity on the reporter was observed more consistently than what was shown by the FimE reporter (K137058) as submitted by Caltech 2008, which we did not observe to have an inversion. We created two asymmetrical digest reporters in order to obtain semi-quantitative data on recombinase activity at the DNA level.  Using the digestible reporters, we determined that FimE has a low inversion activity. We demonstrated that <I>hbiF</I> as submitted to the Registry is functional.  DH5Œ± transformed with <I>hbiF</I> has a fimbriated phenotype, suggesting HbiF activity on the native <I>fimS</I>.

Native recombinase activity in <I>E. coli</I> is augmented by DNA-bending proteins. As inversion is accomplished through a Holliday junction intermediate, the invertible regions must at some point be in close proximity with each other. The simplicity of our system likely contributes to the lower efficiency of the recombinases. Co-expression of the recombinase system with DNA-bending proteins, such as leucine-responsive regulatory proteins and integration host factor protein, will increase the probability that the invertible regions are at a distance preferred for higher recombinase activity.

Production and degradation rates of each recombinase within the system can be further optimized by investigating combinations of promoters and RBSs, and the use of degradation tags. Imprecise regulation of recombinase expression can also be compensated by preventing the leaky gene expression from forming proteins, or counteracting the effects of leaky protein expression. Although the system is more complicated than what we envisioned, it continues to be a viable and interesting way of manipulating genetic information, and have the potential to become increasingly complex and robust with additional components.
<H2>References</H2>
Ham et al., Biotechnol. Bioeng. (2006), Vol. 94(1) 

Ham et al., PLoS ONE (2008), 3(7)


Gally et al., Molecular Microbiology (1996) 21(4)

</DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Michigan/Results">http://2012.igem.org/Team:Michigan/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Michigan/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Michigan/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Michigan/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Michigan/Results&amp;oldid=269465" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>