<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Calgary_Project_OurSensor_Reporter_BetaLactamase"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Calgary/Project/OurSensor/Reporter/BetaLactamase">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Calgary/Project/OurSensor/Reporter/BetaLactamase&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/OurSensor/Reporter/BetaLactamase&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Calgary/Project/OurSensor/Reporter/BetaLactamase&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Calgary/Project/OurSensor/Reporter/BetaLactamase" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Calgary/Project/OurSensor/Reporter/BetaLactamase</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="Banner"><H1>β-Lactamase</H1></DIV><SECTION id="Page"><HEADER id="TopBar"><UL id="TopLvlNav"><LI><A href="https://2013.igem.org/Team:Calgary">Home</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project">Project</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor">Our Sensor</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Detector">Detector</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Reporter">Reporter</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Reporter/PrussianBlueFerritin">PrussianBlue Ferritin</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Reporter/BetaLactamase">β-Lactamase</A></LI></UL><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Linker">Linker</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Modelling">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Modelling/SpatialModelling">SpatialModelling</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Modelling/QuantitativeModelling">QuantitativeModelling</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Prototype">Prototype</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/HumanPractices/Platform">Our Platform</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/HumanPractices">Human Practices</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/HumanPractices/InformedDesign">InformedDesign</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/HumanPractices/Safety">Safety</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/Collaboration">Collaboration</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/Collaboration/BiosensorDatabase">SensiGEMDatabase</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/DataPage">Data Page</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Project/PostRegionals">FinalSystem</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/OurTeam">Our Team</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/OurTeam/TeamMembers">Team Members</A></LI><LI><A href="https://igem.org/Team.cgi?id=1189">Our Profile</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/OurTeam/TheUniversity">TheUniversity</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/OurTeam/Attributions">Attributions</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/OurTeam/Sponsors">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook">Notebook</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Journal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Journal/Detector">Detector</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Journal/Reporter">Reporter</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Journal/Linker">Linker</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Journal/Modelling">Modelling</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Notebook/References">References</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Outreach">Outreach</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Outreach/TelusSpark">TelusSpark</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Outreach/ConsortAlberta">ConsortAlberta</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Outreach/Beakerhead">Beakerhead</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Outreach/GenomeAlberta">GenomeAlberta</A></LI><LI><A href="https://2013.igem.org/Team:Calgary/Outreach/ScienceCafe">ScienceCafé</A></LI><LI><A href="https://2013.igem.org">iGEM</A></LI></HEADER><SECTION id="Content"><H1>β-Lactamase</H1><H2>What is β-lactamase?</H2><P>β-lactamase is an enzyme encoded by the ampicillin resistance gene (<I>amp</I>R) frequently present in plasmids for selection. Structurally, β-lactamase is a 29 kDa monomeric enzyme (Figure 1). Its enzymatic activity provides resistance to β-lactam antibiotics such as carbapenems, penicillin and ampicillin through hydrolysis of the β-lactam ring, a structure shared by the β-lactam class of antibiotics (Qureshi, 2007).</P><FIGURE><FIGCAPTION><P><B>Figure 1.</B>3D structure of β-lactamase made using the Embedded Python Molecular Viewer and Maya.  We used spatial modeling to better understand what our proteins such as β-lactamase look like and how they interact in our system. To learn more about how we used spatial modeling to inform systems our design, click <A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Modeling">here</A>.
</P></FIGCAPTION></FIGURE><P>Many advantages come from working with β-lactamase. It shows high catalytic efficiency and simple kinetics. Also, no orthologs of <I>amp</I>R are known to be encoded by eukaryotic cells and no toxicity was identified making this protein very useful in studies involved eukaryotes (Qureshi, 2007). β-lactamase has been used to track pathogens in infected murine models (Kong <I>et al.</I>, 2010). However, in addition to its application in eukaryotic cells,β-lactamase has been found to have an alternative application in synthetic proteins as well. β-lactamase is able to preserve its activity when fused to other proteins, meaning  it can viably be used in fusion proteins (Moore <I>et al.</I>, 1997). This feature makes β-lactamase a potentially valuable tool for assembly of synthetic constructs.</P><H2>How is β-lactamase used as a Reporter?</H2><P>β-lactamase, in the presence of different substrates, can give various outputs. It can produce a fluorogenic output in the presence of a cephalosporin derivative (CCF2/AM), which can then subsequently be measured using a fluorometer (Remy <I>et al</I>., 2007). Additionally, β-lactamase can also be used to obtain colourimetric outputs by breaking down synthetic compounds such as nitrocefin (Figure 2). The result of nitrocefin hydrolysis is a colour change from yellow to red (Remy <I>et al</I>., 2007). A third output that β-lactamase can give out is through pH. One example is the hydrolysis of benzylpenicillin by β-lactamase, converting the substrate to an acid and lowering pH. This can then be seen through the use of pH indicators such as phenol red to give an observable output (Li <I>et al</I>., 2008). The multiple ways this enzyme can be used shows the versatillity of it, as it is capable of three different outputs, fluorescent, colourimetric, and pH.</P><FIGURE><FIGCAPTION><P><B>Figure 2.</B> Hydrolysis of nitrocefin catalyzed by β-lactamase. This causes a colour change from yellow to red that can be seen.</P></FIGCAPTION><P>β-lactamase can also be split apart in to two halves for protein complementation assays, where each half is linked to one of the two proteins being tested. If the two proteins interact the two halves are able to fold into their correct structure and give an output (Wehrman <I>et al.</I>, 2002).</P><P>Therefore, this enzyme gives a lot of flexibility, both in how it can be attached to proteins as well as the various outputs it can give, making it a useful reporter to characterize and add to the Parts Registry.</P><H2>How does β-lactamase fit in our Biosensor?</H2><P>β-lactamase serves as another reporter we explored for our system in parallel to our <A href="https://2013.igem.org/Team:Calgary/Project/OurSensor/Reporter/PrussianBlueFerritin">
Prussian blue ferritin reporter</A>. But unlike the Prussian Blue ferritin system,  β-lactamase can also be used for a pH output as well. This would add more versatility to our system, as β-lactamase can add a different type of output which would mean that our system is not only limited to the colourimetric outputs. Additionally, the pH output can be paired with different pH indicators to give a variety of colourimetric outputs. We have characterized this ability with two separate pH indicators, phenol red and bromothymol blue, and were able to demonstrate two completely different colourimetric output. In addition, the output of our system can be scaled by altering the number of fused β-lactamase proteins by exploiting the ferritin nanoparticle. This can be achieved through modifying the number of β-lactamase molecules attached to ferritin, ranging from 24 or 12 depending on whether our ferritin nanoparticle consists of the 12 heavy-light subunit fusions (<A href="http://parts.igem.org/Part:BBa_K157018">BBa_K157018</A>),or 24  individual subunits, composed of separate light and heavy subunits. The result is a system that can be scaled by utilizing 24 or 12 β-lactamase proteins, or only 1 Prussian blue ferritin core, or one β-lactamase protein fused to a TALE as an alternate. This adds the elements of versatility, flexibility, and most importantly, modularity to our biosensor and serves a vital basis for our platform.
</P><H2>Constructs</H2><P>We retrieved the <I>amp</I>R gene from the backbone of the <A href="http://parts.igem.org/Part:pSB1A3">
pSB1A3
</A> plasmid. We added a a His-tag  was added to the N-terminus of it using a flexible glycine linker (<A href="http://parts.igem.org/Part:BBa_K157013">BBa_K157013</A>), allowing purification through Ni-NTA protein purification, as well as the lacI promoter for expression (Figure 3). Additionaly, we modified this gene to make it a more useful part for the registry, such as the removal of a BsaI cut-site, making it viable for Golden Gate assembly (Figure 4). We also fused the <I>amp</I>R gene to on our TALE A to bind to our target sequence (Figure 5). This could be used in conjunction with another TALE to act in our strip assay. These modifications resulted in the products shown below:</P><FIGURE><FIGCAPTION><P><B>Figure 3.</B> On the left, part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189009">
BBa_K1189009</A> which is β-lactamase with a his 6 tag that facilitated purification. On the right, part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189007"> BBa_K1189007</A>. In addition to the His-tag, PLacI + RBS were added upstream of the β-lactamase gene so we can express and characterize our part.</P></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><P><B>Figure 4.</B> Part <A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189008">
BBa_K1189008</A>. To enable the construction of β-lactamase with Golden Gate Assembly we removed a BsaI cut site.</P></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><P><B>Figure 5.</B> Part <A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>. This construct works as the mobile detector in our biosensor. TALE A is linked to β-lactamase and if the <I>stx2</I> gene is present in the strip, our mobile is retained on the strip so β-lactamase can give a colour output in the presence of a substrate.</P></FIGCAPTION></FIGURE><H2>Results</H2><P> As a preliminary test to confirm proper protein expression, we tested purified β-lactamase with benzylpenicillin, a substrate that gives a colourimetric and a pH output. First, we wanted to demonstrate that our bacteria carrying the <I>amp</I>R gene was expressing functional β-lactamase. <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189007">BBa_K1189007</A>. In order to do so, we performed an <A href="https://2013.igem.org/Team:Calgary/Notebook/Protocols/AmpicillinSurvivalAssay1">
ampicillin survival assay
</A> using <I>E. coli</I> transformed with a plasmid encoding the <I>amp</I>R gene. This assay would involve culturing the bacteria and then exposing them to ampcillin, and survival was then measured by OD. This then allowed us to determine whether the β-lactamase was produced and whether it is functional. Only the bacteria producing functional β-lactamase enzymes were able to survive in the presence of ampicillin resulting in an increase in OD. Whereas bacteria lacking the abililty to produce functional β-lactamase enzyme were unable to survive, seen by a decrease in OD. (Figure 6). Therefore, we are able to produce functional β-lactamase enzyme (Figure 6).</P><FIGURE><FIGCAPTION><P><B>Figure 6. </B>Absorbance values at 600nm for each tube at four different time points: 0, 30, 60 and 120min. The cultures that expressed β-lactamase (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189007">BBa_K1189007</A>) showed higher absorbance levels, showing that the cells were able to grow in the presence of ampicillin.</P></FIGCAPTION><P>After confirming protein expresison, we wanted to demonstrate that we can also purify our proteins. Figure 7 demonstrates that we were able to purify both our β-lactamase (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189007">BBa_K1189007</A>) and our TALE-A-β-lactamase protein (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) (Figure 7). After purification, we  wanted to demonstrate whether our purified TALE A - β-lactamase fusion protein retained its enzymatic activity. This was tested by a variation of the ampicillin survival assay where we pretreated, LB containing ampicillin and chloramphenicol, with our purified TALE A linked to β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>). We then cultured bacteria in the treated LB carrying the psB1C3 (<A href="http://parts.igem.org/Part:pSB1C3">pSB1C3</A>), conveying resistance to chloramphenicol. In order for the bacteria to survive our isolated protein needed to retain its enzymatic abilities. We can show that the bacteria susceptible to ampicillin were able to grow in the presence of our purified proteins (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>), which means that we are expressing and purifying functional fusion protein which is degrading the ampicillin (Figures 8). Both graphs show an increase in OD for cultures pre-treated with our protein demonstrating our protein is functional.</P><FIGURE><FIGCAPTION><P><B>Figure 7. </B>On the left crude lysate of β-lactamase + His (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189007">BBa_K1189007</A>) from different lysis protocols, <A href="https://2013.igem.org/Team:Calgary/Notebook/Protocols/GlassBeadsCellLysisProtocolforProteinSamples">

beat beating

</A> and <A href="https://2013.igem.org/Team:Calgary/Notebook/Protocols/OsmoticShock">sucrose osmotic shock</A> respectively. On the right, western blot of <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K782004">TALE A+linker+β-lactamase (</A><A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) showing that we were able to express and purify our fusion protein.
</P></FIGCAPTION><FIGURE><FIGCAPTION><P><B>Figure 8. </B>Absorbance values at 600nm after 24h. Amount of protein added ranged from 0.1µg to 20µg of TALE A-link-β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) were sufficient to degrade the ampicillin in the media allowing bacteria susceptible to ampicillin to grow.</P></FIGCAPTION><FIGURE><FIGCAPTION><P><B>Figure 9. </B>Absorbance values at 600nm in different time points. Amounts from 1.0µg to 10µg of TALE A-link-β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) were sufficient to degrade the ampicillin in the media allowing bacteria susceptible to ampicillin to grow.</P></FIGCAPTION><P>After verifying that <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K782004">TALE A</A>-linker-β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">
BBa_K1189031</A>) retained enzymatic activity and was able to degrade ampicillin, we performed a <A href="https://2013.igem.org/Team:Calgary/Notebook/Protocols/BenzylpenicillianAssay">
pH assay</A> using benzylpenicillin as our substrate to demonstrate that our fusion TALE A - β-lactamase protein can act as a reporter. β-lactamase hydrolyzes benzylpenicillin to penicillinoic acid, which changes the pH of the solution from alkaline to acidic. We tested it out with two separate pH indicators, phenol red and bromothymol blue, and we were able to show two different colourimetric outputs. We were able to show successful reporter acitivy with both indicators. </P><P>
We were able to see a colour change due to the presence of phenol red, a pH indicator with a transition pH of 6.8-8.2, turning red at lower pH.  This pH change causes the phenol red to change from red to yellow. Our negative controls, to which benzylpenicillin was not added, remained red. We can also see the colour change coincides with the amount of purified TALE A-β-lactamase present in each sample (Figure 10). </P><FIGURE><FIGCAPTION><P><B>Figure 10. </B>Benzylpenicillin assay. On the top, the wells only had <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K782004">
TALE A</A>-linker-β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>). Benzylpenicillin was added and after a 10-minute incubation at room temperature, we were able to observe a colour output from red to yellow (bottom row) while the control wells remained red.</P></FIGCAPTION></FIGURE><P>Additionally, we have demonstrated the same pH change of benzylpenicillin to penicillinoic acid by the TALE A β-lactamase fusion (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>)  with bromothymol blue. The pH change causes the bromothymol blue to change in colour from blue to yellow as it gets more acidic (Figure 11 &amp; 12).  In the presence of TALE A β-lactamase fusion we see the colour change to yellow whereas the negatives not containing TALE A β-lactamase fusion remains blue (Figure 11). We did a kinetic analysis at 616 nm every 30 seconds and as the blue colour disappears, the absorbance at 616 nm decreases (Figure 12). Therefore, the lower the decrease the better the TALE A β-lactamase fusion reporter activity. Our kinetic activity shows that the biggest decrease is in our positive recombinant β-lactamase followed by the TALE A β-lactamase fusion at 10 micrograms. This decrease is lessened as we decrease the amount of TALE A β-lactamase fusion. We can also show that in our negatives with no TALE A β-lactamase fusion we do not have a decrease in absorbance.  We have demonstrated the reporter activity both qualitatively (Figure 10 &amp; 11) and quantitatively (Figure 12).  </P><FIGURE><FIGCAPTION><P><B>Figure 11.</B>Change in pH catalyzed by TALE A linked to β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) using benzylpenicillin. Bromothymol blue was used to keep track of this colour change. Absorbance readings were taken at 616 nm every 30 seconds. Different amounts of TALE A linked to β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) were tested. Commercial β-lactamase was used as a positive control. Negative controls included were bovine serum albumin, β-lactamase without the substrate and the substrate by itself.</P></FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><P><B>Figure 12.</B>Change in pH catalyzed by TALE A linked to β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) using benzylpenicillin. Bromothymol blue was used to keep track of this colour change. Absorbance readings were taken at 616 nm every 30 seconds. Different amounts of TALE A linked to β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) were tested. Commercial β-lactamase was used as a positive control. Negative controls included were bovine serum albumin, β-lactamase without the substrate and the substrate by itself.</P></FIGCAPTION></FIGURE><P> In order to demonstrate that we can successfully capture target DNA with two TALEs we did a capture TALE assay (Figure 13). TALE B was incubated with DNA containing target sites for TALE A and TALE B and blotted on nitrocellulose. After blocking and washing, TALE A β-lactamase fusion (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) was added to the nitrocellulose strips. The strips were transferred into a 96 well plate to which a benzylpenicillin substrate solution with phenol red was added. If the TALE A β-lactamase fusion bound to the target site for TALE A then the solution will change colour from pink to clear. If TALE A β-lactamase fusion   was not present, the solution will remain pink.  We can show that the first four samples which have TALE B with DNA for TALE A and TALE B show a colour change indicating that we are successfully capturing the target DNA and reporting it. Furthermore we can also show that when we add non-specific DNA we do not see a colour change demonstrating that we can successfully capture only specific DNA and report its presence with an easy visual colourimetric output. </P><FIGURE><FIGCAPTION><P><B>Figure 13: </B> TALE capture assay was done with TALE B  (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189001"><B>)
BBa_K1189001
</B></A>)and TALE A B-lac fusion (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031"><B>
BBa_K1189031
</B></A>) and DNA (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189006"><B>
BBa_K1189006
</B></A>)
with both target sequences. If capture is successful, the TALE A β-lactamase is present in the well giving a colour change from pink to yellow when subjected to benzylpenicillin substrate solution within 20 minutes. The only wells that change colour are the first four wells which contain TALE B, specific DNA, and TALE A β-lactamase fusion and our positive control wells which are the controls for our fusion TALE A β-lactamase protein and our positive recombinant β-lactamase. All our other controls including our test using a non-specific sequence of DNA remained pink . This preliminary characterization data demonstrates that the TALEs are able to bind to DNA with specificity. Additionally it also shows that our system of capturing DNA with two detector TALEs and then subsequent reporting of the DNA’s presence works.
</P></FIGCAPTION></FIGURE><P>This assay shows that we can <SPAN class="Yellow"><B> capture our target DNA </B></SPAN> with two detector TALEs with <SPAN class="Yellow"><B>specificity </B></SPAN>. Additionally, <SPAN class="Yellow"><B>we can report whether that DNA has been captured</B></SPAN> and is present in the sample, which is a very important concept for our sensor system. </P><H2> Conclusion </H2><P>To conclude, we have demonstrated that we have built, expressed, purified, and submitted parts containing β-lactamase both on its own and linked to TALE A. We then expressed, purified and demonstrated the final purified products have retained their enzymatic activity. We can show activity for our mobile TALE A linked to β-lactamase (<A href=" http://parts.igem.org/wiki/index.php?title=Part:BBa_K1189031">BBa_K1189031</A>) for our sensor in two different ways, through pH output (with two pH indicators) and cell growth assays. We have also demonstrated that we can use the TALE A β-lactamase fusion as a <SPAN class="Yellow"><B> biobrick that has been characterized to show both its ability to be a good reporter and its ability to be able to bind to DNA with specificity. </B></SPAN></P><H2>Future Directions</H2><P> We would like to further characterize β-lactamase in conjunction with our ferritin nanoparticle to test the modularity of our system in our ability to scale reporter activity in response to the various sensitivities that might be required as per our platform. </P></FIGURE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Calgary/Project/OurSensor/Reporter/BetaLactamase">http://2013.igem.org/Team:Calgary/Project/OurSensor/Reporter/BetaLactamase</A>&quot;</DIV></FIGURE></FIGURE></FIGURE></FIGURE></SECTION></SECTION></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Calgary/Project/OurSensor/Reporter/BetaLactamase" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Calgary/Project/OurSensor/Reporter/BetaLactamase" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Calgary/Project/OurSensor/Reporter/BetaLactamase&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Calgary/Project/OurSensor/Reporter/BetaLactamase&amp;oldid=361165" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>