"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Sandbox/Notebook/Journal/Detector
From 2013.igem.org
(Redirected from Team:Calgary/Sandbox/Notebook/Journal/TALEsJournal )
Home
Project
Our Sensor
Detector
Reporter
Prussian
Blue Ferritin
β-Lactamase
Linker
Modelling
Spatial
Modelling
Quantitative
Modelling
Prototype
Our Platform
Human Practices
Informed
Design
Safety
Collaboration
SensiGEM
Database
Data Page
Achievements
Final
System
Our Team
Team Members
Our Profile
The
University
Attributions
Sponsors
Notebook
Journal
Detector
Reporter
Linker
Modelling
Protocols
Parts
References
Outreach
Telus
Spark
Consort
Alberta
Beakerhead
Genome
Alberta
Science
Café
iGEM
Detector Journal
Week 1: May 1 - May 3
This week we attended courses required by our University to work in the lab. Besides that, the undergraduate supervisors presented about the principles of Genetics and Synthetic Biology to the new team members.
Week 2: May 6 - May 10
This week we attended a workshop based on general molecular biology techniques.
Week 3: May 13 - May 17
We continued the molecular biology workshop. Also, the team members were assigned in different parts of the project such as TALEs as our detector, Ferritin as our reporter, Linker, Human Practices, Modelling and Business.
Week 4: May 20 - May 24
We did our literature search for optimization of TALE binding affinity. There were conflicting results for the optimal truncation of the N and C terminus. On N terminus, 152 (Miller et al., 2011), 147 (Mussolini et al., 2011), 186 (Slovenia’s TALEs), and 158 or 186 (Mercer et al., 2012) residues were retained. On C terminus, 28 or 63 (Miller et al., 2011), and 76 (Slovenia’s TALEs) residues were retained.
We learned about three very important TALE characteristics:  Firstly, TALEs demonstrate polarity. In other words, they interact more strongly with DNA at the N terminus compared to the C terminus (Meckler et al., 2013). Secondly, the Thymidine nucleotide at the zero repeat is very important to the binding affinity of the TALE (Meckler et al., 2013). Thirdly, the binding affinity of TALE varies based on the types of the nucleotides present in its target site. The relative affinities are: NG(1)>NN (0.18)!HD (0.16) " NI (0.0016)>NK (0.00016) (Meckler et al., 2013). All these points must be carefully considered while designing the TALE to maximize its binding affinity and specificity.
The four TALEs designed by Slovenia 2012 iGEM team, were ordered from the registry (TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ) and TALE D ( BBa_K782005 )). Primers were designed to incorporate the the target sequences of TALE A ( BBa_K782004 ) and TALE B ( BBa_K782006 ) ([A] and [B]) in the RFP generator. This plasmid will be used as a target sequence. Primers were also designed that incorporate selected point mutations of [A] and [B] into a biobrick. Once these plasmids are made they will be used to determine how the TALEs affinity is altered by mutations. In addition, two other TALEs were ordered from the the authors of the Meckler et al. 2013 paper.
Week 5: May 27 - May 31
We continued our literature search for a unique and stable pathogenic E. coli marker for the TALEs to bind to. Potential pathogenic O157:H7 E. coli markers: stx1, stx2 (Yoshitomi et al., 2012), uidA (Yoshitomi et al., 2003 and Feng et al., 1994), eaeA, hly, rbfE (Fortin et al., 2000), and Z3276 (Li & Chen, 2012). However, BLAST searches determined that the use of only one 18 bp sequence is not sufficiently specific and sensitive to detect pathogenic markers. Should we continue to search for only one marker or more than one or expand the detection to any toxic Shiga producing strains?
In addition to searching for a pathogenic E. coli marker, we also started working on the TALE sequence and its construction. The TALE sequence chosen has a truncation in both N- and C-terminus. The truncated N-terminus has 111 residues and the C-terminus has 42 residues (Meckler et al., 2013). The sequence was codon optimized for E.coli. NEBCutter 2.0 was used to screen restriction sites for PstI, EcoRI, NotI, XbaI, SpeI, NgoMIV and AgeI, all present in the RFC 25. Cut sites for BsaI and Esp3I were also removed to pave the road for Golden Gate Assembly method. The original TALE sequence has 11 cut sites: BsaI (3), PstI (1), EcoRI (1), Esp3I (4), AgeI (2). The restriction sites were then eliminated with silent mutations, based on a E. coli codon usage table (San Diego State University, 2012 modified from Maloy, Stewart & Taylor, 1996).
The large size of the TALE prevents simple cloning, therefore we looked to Golden Gate Assembly. Golden Gate Assembly is a subcloning strategy that uses type IIs restriction enzymes, which cut outside of their recognition sequence allowing ligation into a final product that lacks any specific scar sites (Engler, Kandzia & Marillonnet, 2008).
Week 6: June 3 - June 7
This week also mostly spent on literature research.
The next step after optimization of TALEs and their target sequence is to locate this system’s niche in the beef industry. Many of the current high fidelity methods for pathogenic E. coli detection require cell cultures, which increases turnaround time up to 4 days (Blais et al., 2012 & Gill et al., 2013).
A 2013 paper by Jeon et al. explores the genetic and physiological effects of the cattle on the amount of the O157 it excretes. According to this paper, 90% of the bacteria excreted from the cattle comes from the super-shedders. Super-shedders are cattle that excrete more than more than 10 000 CFU/g and have long term recto anal junction colonization. Higher excretion is detected in bulls, pen raised cattle, and certain breeds with a seasonal peak between late spring and early fall. Preharvest detection of pathogenic E. coli decreases cross contamination during meat processing.
Shiga toxin 1 and 2 have 55% homology and are composed of 5 identical highly conserved B subunits that bind to the Gb3 host receptor cells and a single A subunit which enters the host target cell and cleaves rRNA. Cattle do not have Gb3, consequently, O157:H7 infected cattle are asymptomatic and difficult to detect. More than 200 serotypes of E.coli contain Stx, of these, some have LEE (locus of enterocyte) which does not directly indicate pathogenicity. Other strains of pathogenic shiga toxin producing E. coli recognized by the USDA as harmful are: O26:H11, O103:H2, O111:H8, O118:H16, O121:H19, and O145:H28 (USDA  Laboratory Guidebook, 2012). These strains are also known as the Big Six. Currently, the USDA targets both the eae and stx genes in the PCR. When these genes are being used in identification, if the sample tests positive for both eae and stx, the sample undergoes further testing. If it only has one or neither of the genes, the sample is considered safe and testing stops (USDA  Laboratory Guidebook, 2012).
Regarding our TALE construct, we compared 2 standard construction strategies for multiple DNA fragments: Golden Gate and Gibson Assembly. Gibson Assembly is a one-step isothermal reaction involving an exonuclease, a DNA polymerase and a DNA ligase. All enzymes are put together in one reaction along with DNA fragments and the vector and incubated at 50°C for 15 to 60 min. (Gibson et al., 2009)
Week 7: June 10 - June 14
This week, we had the opportunity to tour Cargill Meat Solutions’ High River beef plant and experience firsthand the stringency and efficiency of the meat processing facility. Global prevalence of O157:H7 in cattle farms is 20% (Chase-Topping et al., 2008) and the presence of super shedders is influenced by the E. coli strain and host dependent factors (breed, sex, reproductivity, stress, movement) independent of the external farm environment (Chase-Topping et al., 2007). These few super shedders are important factors to pre-harvest contamination as the source of 80% of O157:H7 strains isolated from beef carcasses are from the high cattle density and rapid turnover environment of the processing plant and transport trailers, not the original feedlot (Arthur, 2009).
We also met with and discussed E. coli markers with Dr. Glen Armstrong, professor and head of the Department of Microbiology, Immunology & Infectious Diseases at the University of Calgary. Dr. Armstrong suggested that we implement multiple TALEs to target different markers to identify E. coli serotypes.
Primers containing the target sequences of TALE A ( BBa_K782004 ), TALE B( BBa_K782006 ), and variations of the target sequences (with point mutations in different positions) were used to perform a PCR on RFP generator ( BBa_J04450 ); the product of the PCR would be the target sequence of the TALE and a part of the RFP gene which is 660 base paris (figures 1, 2 and 3). The PCR products were purified and digested with XbaI and PstI and inserted into  a vector; RFP generator was also digested with XbaI and PstI to prepare the vector.
IPTG inducible promoter with RBS ( BBa_J04500 ) was transformed with competent cells and plated on Amp and Chlor.
Figure 1.PCR products of mutated TALE B target sequences, which were inserted in the RFP generator gene. Bands were expected at 660bp.
Figure 2.PCR products of intact and mutated TALE B target sequences, which were inserted in the RFP generator gene. Bands were expected at 660bp.
Figure 3.PCR products of mutated TALE B target sequence and intact TALE A target sequence, which were inserted in the RFP generator gene. Bands were expected at 660bp.
The parts received from the registry - TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ), TALE D ( BBa_K782005 ), NicTAL 12 ( BBa_K782007 ) and GFP in RFC 25 Standard ( BBa_K648013 )- were transformed. Subsequently, a overnight cultures of the colonies were made and a plasmid purification was performed. Digestion of the purified plasmids with NotI and running the products on the gel produced the expected results (figure 4).  TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ) and TALE D ( BBa_K782005 )were sent for sequencing.
Figure 4.Verification digest of TALEs A ( BBa_K782004 ), B  ( BBa_K782006 ) and D ( BBa_K782005 ) and GFP ( BBa_K648013 ) with NotI.
Week 8: June 17 - June 21
While attending the University of Calgary Veterinary Medicine Beef Cattle conference, we learned a lot more about the beef processing industry. Our project was well received by ranchers and members of the cattle industry who found it a practical and applicable option.
In the lab, we ran a colony PCR on the TALE target sequences and ran agar gels (figures 5, 6 and 7).
Figure 5.Colony PCR on TALE target sequences.
Figure 6.Colony PCR on TALE target sequences.
Figure 7.Colony PCR on TALE target sequences.
The PCR products were purified and digested with AflII and PstI and then ran on a gel, producing two bands as expected at ~500 and ~2100 bp. BBa_J04500 only produced one band as expected since it did not have the AflII cut site (figures 8 and 9).
Figure 8.Verification digest of TALE target sequences and BBa_J04500 with AflII and PstI.
Figure 9.Verification digest of TALE target sequences with AflII and PstI.
We also got the sequencing results for TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ) and TALE D ( BBa_K782005 ). TALE A ( BBa_K782004 ) and TALE D ( BBa_K782005 ) sequences were fine. However, TALE B ( BBa_K782006 ) had a mutation in the RVD region of the second repeat making it specific for a cytosine instead of a thymine. Because of that, we will have to design new primers to add the new TALE B target sequence to the RPF generator.
University closed June 21 due to flooding.
Week 9: June 24 - June 28
University closed June 24 and 25 due to flooding.
Literature search to cleave His tags from proteins determined TAGZyme would be an expensive option.
In the lab, we did a restriction digest of BBa_J04500 , the vector containing promoter and ribosome binding site, with SpeI and PstI. Restriction digest of TALE A ( BBa_K782004 ) and TALE B ( BBa_K782006 ), TALE D ( BBa_K782005 ) as separate inserts with XbaI and PstI. Ligated and transformed BBa_J04500 + TALE A and BBa_J04500 + TALE B .
Week 10: July 1 - July 5
University closed July 1 due to Canada Day.
Repeated restriction digest of vector BBa_J04500 with SpeI and PstI and of TALE A and TALE B as separate inserts with XbaI and PstI. We ran restriction digest products of TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ), and BBa_J04500 on gel for gel extraction. Nanodrop concentrations  were lower than4ng/uL and had poor purity; products were discarded. Repeated ligation and transformation of BBa_J04500 + TALE A and BBa_J04500 + TALE B . Products were plated and run in a colony PCR. For BBa_J04500 + TALE A , only three of the eight  lanes had bands at ~3000bp, while all eight of BBa_J04500 + TALE B had unclean but expected results (figure 10).
Figure 10.Colony PCR on BBa_J04500 + TALE A and BBa_J04500 + TALE B .
We also received carbenicillin resistant TALEs from University of California (Meckler et al., 2013): IINNp, IIINGp, IINNp Luciferase, and IIINGp Luciferase. Transformed TALEs were plated on Amp.
We did overnight cultures of BBa_J04500 + TALE A , BBa_J04500 + TALE B , and University of California TALEs. Subsequent miniprep and Nanodrop of the products yielded generally low concentrations ranging from 9.4 - 63.1 ng/uL with mediocre purity. Verification digest of BBa_J04500 + TALE A and BBa_J04500 had bands at 3000bp and 3300bp, not at the expected 2100bp for backbone and 2700bp for insert.
Week 11: July 8 - July 12
This week, we did miniprep on sequence verified BBa_J04500 . We also ran PCR products of the first Kapa HiFi PCR of TALE A and TALE B primers on gel, no bands visible. Subsequent higher temperature gradient Kapa HiFi PCR produced bands around 2500bp and lots of non specific bands (figure 11). Gel extraction had very low yields (<20ng/uL) and poor purity, and repeated extractions produced similar results.
Figure 11.Gradient Kapa Hifi PCR products of BBa_J04500 + TALE A and BBa_J04500 + TALE B .
We have received 5 primers for TALE B target sequences and did a Kapa HiFi PCR. Gel of PCR product produced expected results with bands ~600bp (figure 12).
Figure 12.Gradient Kapa Hifi PCR products of TALE target sequences.
A Kapa HiFi PCR of K coil primers was performed and showed expected results but also had lots of primer dimers. Subsequent gel extractions had poor purity and yields (figure 13).
Figure 13.Gradient Kapa Hifi PCR products of K coil.
Week 12: July 15 - July 19
This week we completed ligation, transformation, overnight culture, and miniprep of 5 TALE target sequences (figures 14 and 15). The verification digest gel had strong bands at ~2000bp and weak bands at ~550bp, and plasmids were sent for sequence verification (figures 16 and 17).
Figure 14.Colony PCR on new TALE B target sequences.
Figure 15.Colony PCR on new TALE B target sequences.
Figure 16.Verification digest on new TALE B target sequences.
Figure 17.Verification digest on new TALE B target sequences.
The results of the BBa_J04500 + TALE A and BBa_J04500 + TALE B sequencing were back. However, we realized that TALEs have a Kozak sequence at the beginning, which increases the distance between the promoter and the start codon of the gene. Therefore, a primer (TALseqF) was designed  to take out TALE A and TALE B without the Kozak sequence. The PCR performed with forward primer TALseqF and reverse primer BBK R on TALEs was successful.
BsaI enzyme finally arrived. So the constructions of BBa_J04500 +His+ TALE A +linker+Kcoil and BBa_J04500 +His+ TALE B +linker+Kcoil in Golden Gate reaction were performed. The ligation reactions were run on the gel and expected bands were observed. However, the transformation was unsuccessful because of the new competent cells.
The PCR products of TALE A and TALE B without the Kozak sequence were run on a 1% gel and contained a lot of non specific amplification.
Week 13: July 22 - July 26
The 5 TALE B target sequences were sequence verified and are good.
We repeated PCR of TALE A and TALE B without the Kozak sequence and it worked (figure 18), so the PCR products were ligated with BBa_J04500 and successfully transformed. Subsequent colony PCR showed some positive results (figures 19 and 20).
Figure 18.PCR products of TALEs without the Kozak sequence.
Figure 19.Colony PCR on constructs 12 and 11.
Figure 20.Colony PCR on constructs 12 and 11.
Tour of the Chinook Feedlot.
Week 14: July 29 - August 2
For the TALE constructs, we did a colony PCR, plasmid prep, verification digest and sequencing preparation of the BBa_J04500 + TALE A and BBa_J04500 + TALE B , both without the Kozak sequence (figure 21). Glycerol stocks and streak plates of sequence verified BBa_J04500 + TALE A colony 12 and BBa_J04500 + TALE B colony 7 were made.
Figure 21.Colony PCR on construct 12 without the Kozak sequence.
We also made glycerol stocks and streak plates of the sequence verified BBa_J04500 + TALE A and BBa_J04500 + TALE B with the Kozak sequence, TALE A (colonies 5 and 6) and the intact and mutated (B3M3, B5M0, B5M1, B5M7) TALE B target sequences.
For the Golden Gate constructs, we did a PCR of Beta-lactamase with primers 16 and 17 and PCR purified the products with concentrations around 40ng/uL.
A colony PCR of constructs 2 ( pSB1C3 - BBa_J04500 -His-Ecoil-link-hFTN-lFTN) and construct 9 pSB1C3 - BBa_J04500 -His- TALE B -K coil) did not show bands in the expected size.
Week 15: August 5 - August 9
University closed August 5 due to Civic Holiday.
This week we worked on the following constructs:
a) construct 11*: pSB1C3 - BBa_J04500 - TALE A -link-Blac;
b) construct 11: pSB1C3 - BBa_J04500 -His- TALE A -link-Blac;
c) construct 10*: pSBIC3 - BBa_J04500 - TALE A -link-Kcoil.
The gel for construct 10* and 11* showed bands in the expected size. After gel purification and DpNI treatment, they were transformed in Top10 cells. Colony PCR showed  right-sized bands for constructs 11*.
We did PCR on BBa_J04500 -His- TALE A with primers 1 and 8 to use it as our vector in Golden Gate Assembly of construct 11. The gel showed right-sized bands but PCR purification had low yields (20.4ng/uL) and low purity. After Antarctic Phosphatase treatment, we set up Golden Gate Assembly of construct 11. The gel showed no bands.We repeated the procedure, and PCR had nonspecific amplification. Gel purification also had low yields and low purity. After that, we did PCR with less amount of DNA (10ng/reaction) which showed no bands. Another attempt was performed with different DNA concentration (5 to 45ng/reaction). Some of them had no bands and some of them had non-specific amplification.
Week 16: August 12 - August 16
This week, we did minipreps of construct 11* ( pSBIC3 - BBa_J04500 - TALE A -link-Blac) of the colonies that showed correct bands in the colony PCR. Following the successful verification digest (figure 22), the samples were sent for sequencing. We did a Kapa PCR of BBa_J04500 + TALE A , but the gel of the PCR products showed a band size that was approximately 1000bp less than expected. This PCR was done so we could use it for other constructions. Instead we ordered new primers to try a different approach to our remaining constructs.
Figure 22.Verification digest on construct 11*.
Now that we have our TALEs we need to express them. We are experimenting using different time points and IPTG amounts to determine how to best express our proteins. We will be running our samples on SDS PAGE gels next week.
Week 17: August 19 - August 23
This week we repeated overnight cultures, minipreps, and sequencing of BBa_J04500 + TALE A and construct 11* ( pSBIC3 - BBa_J04500 - TALE A -link-Blac), as previous sequencing results were inconclusive.
For TALE protein characterization and expression we did a trial run to determine the optimum time point and IPTG concentration.
We subcultured our overnight cultures of TALE A , TALE B , the positive control RFP and the negative control BBa_J04500 . For each pellet and supernatant, we had 3 time points 3 hours, 6.5 hours, and 24 hours and 3 IPTG concentrations of 0.1mM, 0.4mM, and 0.8mM for a total of 72 samples on 8 SDS page gels. Our results indicate that the optimum IPTG concentration is 4mM and 8mM IPTG and induction time of 3 hours and 6.5 hours.
A subsequent SDS page gel was run with TALE A , TALE B , Beta lactamase, BBa_J04500 , and RFP .
We also received new primers for construct 10 and 11. We set up a PCR of the construct 10 primers and BBa_J04500 + TALE A template DNA. The gel of the PCR products showed 1800bp bands, which were too small. We repeated the PCR with an increased extension time and the gel showed bands of the expected size. The PCR product was purified and cut with BsaI, ligated together and transformed.
We set up a PCR of the construct 11 primers and BBa_J04500 + TALE A and BBa_J04500 + TALE B template DNA. The gel of the PCR products showed bands about ~2900bp in length.
Because the PCR product was shorter than expected for the first Kapa PCR (construct 10), we also did a PCR of the BBa_J04500 + TALE A and BBa_J04500 + TALE B with BioBrick primers and with specially designed TALE primers to confirm the length of our TALEs. The results were good (figure 23).
Figure 23.Kapa HiFi PCR products of construct 9 and 10, BBa_J04500 + TALE A and BBa_J04500 + TALE B .
We did a Kapa PCR of  TALE target sequences and the gel of the PCR products had bands of the expected size (figure 24). After PCR purification, the product was cut with EcoRI and SpeI. We also digested RFP with EcoRI and SpeI. We used the backbone from RFP as the vector for our PCR product and ligated them together.
Figure 24.Kapa HiFi PCR products of TALE target sequences.
Regarding our TALE expression and characterization, we did mini expression of TALE A , TALE B , Beta lactamase, BBa_J04500 , and RFP .
Week 18: August 26 - August 30
We did a colony PCR of construct 9 ( pSBIC3 - BBa_J04500 -His- TALE B -link-Kcoil), construct 10 ( pSBIC3 - BBa_J04500 -His- TALE A -link-Kcoil) and the TALE target sequences which were transformed over the weekend. The PCR products were run on a gel, the colonies with the bands of the right size were made into overnight cultures and miniprepped. The gel of the verification digests did not have any bands and the overnight cultures were repeated with good results. Construct 9, construct 10 and the TALE target sequences were sent for sequencing.
The PCR product of construct 11 ( pSBIC3 - BBa_J04500 -His- TALE A -link-Blac) was purified and cut with EcoRI and AgeI and ligated with the vector, linker and Beta lactamase, which was cut with EcoRI and NgoMIV. The products were transformed and the gel of the colony PCR had faint bands at ~3500bp. These 4 colonies with the expected size bands were made into overnight cultures and miniprepped, however the gel of the verification digests showed 3 bands at approximately 2000 bp, 2900 bp, and 5000 bp.
A Kapa PCR of construct 11 was repeated, along with digestion, ligation and transformation.
Week 19: September 2 - September 6
This week, we got the sequencing results for construct 9, 10 and the TALE target sequences. The TALE target sequences (with old and new [B] + [A]) worked. Construct 9 sequencing did not have a good reading with one of the primers, so we send it for sequencing again. Construct 10 didn’t work, so we set up digestion, ligation and transformation again. Colony PCR revealed right-sized  bands for 5 colonies. We step up overnights and mini prepped those over the weekend.
We did colony PCR of construct 11, but the bands were not at the expected size, so we transformed more Top10 cells.
We also received constructs 9, 10 and 11 from GenScript and did a transformation with them. The plates had colonies, but now we have to plasmid switch them to pSB1C3 , because they came in PUC57 backbone. We did colony PCR and mini-prepped the colonies that worked.
Regarding our protein work, we have transformed BBa_J04500 + TALE A and BBa_J04500 + TALE B into E. coli ER2566 and into E. coli BL21, strains commonly used for protein expression. We also tried a different lysis protocol, with glass beads, which had good yields for protein concentration through Bradford assay. We did Western Blot and verified that we have our protein expressed (figure 25).
Figure 25.Western Blot of BBa_J04500 + TALE A and BBa_J04500 + TALE B .
Week 20: September 9 - September 13
This week, we did verification digest of constructs 10 ( pSBIC3 - BBa_J04500 -His- TALE A -link-Kcoil) and 11 ( pSB1C3 - BBa_J04500 -His- TALE A -link-Blac). One of the colonies for construct 11 showed the correct band size, so it was sent for sequencing, but the part was 100bp of the Blac gene. We transformed constructs 9, 10 and 11 from GenScript in pSB1C3 again. Colony PCR showed some positive results for construct 9 and 11.
We also did a gel shift assay to test the binding of TALE A to its target site. We tried different concentrations for TALE A + [A] and we picked two of those to run TALE A + [B]. However, the concentrations of protein were too high. We will repeat the assay next week if lower concentrations of TALE.
Week 21: September 16 - September 20
This week, we did  plasmid prep and verification digest of constructs 9 ( pSBIC3 - BBa_J04500 -His- TALE B -link-Kcoil) and 11 ( pSB1C3 - BBa_J04500 -His- TALE A -link-Blac). We have some good results for construct 11, so we did overnight of those, plasmid purification and submitted it to Parts Registry along with our other constructs.
We also repeated the gel shift assay, this time with TALE A and TALE B . The concentration range was from 0.1ng/uL of protein to 40ng/uL. Because Red Safe, which we used for staining the DNA, is not very sensitive we loaded higher amounts of DNA recommended and the gel revealed nonspecific binding by the TALES. We repeated the assay and this time we added chromosomal DNA to prevent nonspecific binding. We also added the recommend amount of DNA (55pM) and stained the gel after it ran with SYBR Green. This time, we were not able to see any bands at all.
We transformed construct 11 into E. coli BL21, cultured the colonies, induced them with IPTG and purified the protein. We performed Bradford Assay on our samples and the concentration range was between 0.20 and 1.06mg/mL. We ran these samples on a 7.5% SDS page gel.
Because construct 11 has Blac on it, we cultured the cells in ampicillin too as beta-lactamase provides resistance to beta-lactam antibiotics The medium density increased suggesting that beta-lactamase was functional in our construct.
Week 22: September 23 - September 27
Enter Text Here
Week 23: September 30 - October 4
Practice, practice, practice for our Regional Jamboree presentation!
Retrieved from " http://2013.igem.org/Team:Calgary/Sandbox/Notebook/Journal/Detector "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
