"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Sandbox/Project/Detector/Background
From 2013.igem.org
Home
Project
Our Sensor
Detector
Reporter
Prussian
Blue Ferritin
β-Lactamase
Linker
Modelling
Spatial
Modelling
Quantitative
Modelling
Prototype
Our Platform
Human Practices
Informed
Design
Safety
Collaboration
SensiGEM
Database
Data Page
Achievements
Final
System
Our Team
Team Members
Our Profile
The
University
Attributions
Sponsors
Notebook
Journal
Detector
Reporter
Linker
Modelling
Protocols
Parts
References
Outreach
Telus
Spark
Consort
Alberta
Beakerhead
Genome
Alberta
Science
Café
iGEM
Background
TALEs: a powerful tool in synthetic biology
Transcriptor activator-like effectors (TALEs) are proteins produced by bacteria of the genus Xanthomonas and secreted into plant cells. These naturally occurring TALEs play a key role in bacterial infection, as they are responsible for upregulation of the host genes required for pathogenic growth and expansion (Mussolino & Cathomen, 2012). Recently, it was reported that another plant pathogen, Ralstonia solanacearum, produces type III effectors, which have a sequence similar to TALEs from Xanthomonas spp. These proteins were therefore named Ralstonia injected protein TALEs or RipTALEs (De Lange et al., 2013).
TALEs share some common features, such as an N-terminal type III secretion signal, which allows the proteins to be translocated from the bacterium and into the plant cell. They also present nuclear localization signals (NLS) and an acidic activation domain (AAD) in the C-terminus. The central region, also termed repeat region, mediates DNA recognition through tandem repeats of 33 to 35 amino acids residues each (Bogdanove et al., 2010). The binding domain usually comprises 15.5 to 19.5 single repeats (Figure 1). The last repeat, close to the C-terminus, is called “half-repeat” because it is only around 20 amino acids in length. Although the modules have conserved sequences, polymorphisms are found in residues 12 and 13, the “repeat-variable di-residue” (RVD). RVDs are specific for a single nucleotide; therefore, 19.5 repeat units target a specific 20-nucleotide sequence in the DNA (Mussolino & Cathomen, 2012).
Figure 1.(A) Schematic representation of a TAL effector with the DNA binding domain in red. (B) 3D structure of TALEs obtained from our team’s work in Autodesk Maya. To learn more about our modeling, click here .
When in contact with the DNA, the TALE aligns the N-terminal to C-terminal with the DNA 5’ to 3’ direction. Each repeat has a RVD loop, which is a two alpha helices structure connected by three residues, two of them being RVDs. Although both amino acids 12 and 13 are responsible for base specificity, the TALE-DNA interaction happens through intermolecular bonds between residue 13 and the target base in the major groove. Residue 12 plays a role in stabilizing the RVD loop (Meckler et al., 2013).
Over 20 different RVDs have been identified in TAL effectors. However, four of them appear in 75% of the repeats: HD, NG, NI and NN (Bogdanove et al., 2010). Quantitative analysis of DNA-TALE interactions by Meckler et al.. (2013) revealed that the binding affinity is affected by the RVDs in the following order: NG > HD ~ NN >> NI > NK. NG, specific to thymine, and HD, specific to cytosine, are strong RVDs. NN binds both guanine and adenine, but it prefers guanine. NK also interacts with guanine, but with 103-fold lower affinity. NI is specific for adenine, but it has low affinity when compared to strong RVDs such as NG and HD (Meckler et al., 2013). Although less common, another naturally occurring RVD, NH, was described to bind strongly to guanine (Cong et al., 2013). NS binds to any of the four bases and it is present in naturally occurring TALEs such as AvrBs3 from Xanthomonas campestris (Boch et al., 2009).
In addition to RVDs, the DNA binding affinity is also subject to polarity effects. Point mutations at the 5’ end of the target sequence affect TALE-DNA recognition more than the ones at the 3’ end (Meckler et al., 2013). Taking this in consideration, recommendations for TALE design include incorporation of strong RVDs close to the N-terminus (Streubel et al., 2012).
Because TAL effectors can be engineered to bind virtually any DNA sequence, they represent a powerful tool in synthetic biology. They have been extensively used in gene modulation by fusing an activator or a repressor to their C-terminus. Slovenia 2012 iGEM team designed and created repressor TAL effectors by adding KRAB repressor domains and activator TALEs through fusion VP16 activation domain.
Besides gene regulation, TALEs can be fused with DNA cleavage domains of endonucleases and serve as restriction enzymes (Beurdeley et al., 2013). These engineered proteins are termed TALENs or Transcription Activator-Like Effector Nucleases. TALENs can also be used in gene knockout as they are able to promote gene disruption (Bogdanove & Voytas, 2011).
Our team, however, proposes an innovative application for TAL effectors: detection of pathogenic E. coli (EHEC) and other entero-haemorrhagic bacteria in feces of super-shedders in cattle populations. As sensors, TALEs can bind to specific regions of the Shiga Toxin II gene (Stx2) and capture the DNA of interest from a feces sample, making it available for a second TALE, whose binding domain is specific for another region of Stx2. This second TALE is connected to a reporter, which turns the TALE-DNA interaction visible in a short period of time. To find out more about our EHEC TALEs, click here .
Retrieved from " http://2013.igem.org/Team:Calgary/Sandbox/Project/Detector/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
