"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Notebook/Journal/Detector
From 2013.igem.org
Detector Journal
Home
Project
Our Sensor
Detector
Reporter
Prussian
Blue Ferritin
β-Lactamase
Linker
Modelling
Spatial
Modelling
Quantitative
Modelling
Prototype
Our Platform
Human Practices
Informed
Design
Safety
Collaboration
SensiGEM
Database
Data Page
Achievements
Final
System
Our Team
Team Members
Our Profile
The
University
Attributions
Sponsors
Notebook
Journal
Detector
Reporter
Linker
Modelling
Protocols
Parts
References
Outreach
Telus
Spark
Consort
Alberta
Beakerhead
Genome
Alberta
Science
Café
iGEM
Detector Journal
Week 1: May 1 - May 3
This week we attended courses required by our University to work in the lab. Besides that, the undergraduate supervisors presented about the principles of Genetics and Synthetic Biology to the new team members.
Week 2: May 6 - May 10
This week we attended a workshop based on general molecular biology techniques.
Week 3: May 13 - May 17
We continued the molecular biology workshop. Also, the team members were assigned in different parts of the project such as the detector TALEs, the reporter Ferritin, Linker, Human Practices, Modelling, and Business.
Week 4: May 20 - May 24
We did our literature search for optimization of TALE binding affinity. There were conflicting results for the optimal truncation of the N and C terminus. On N terminus, 152 (Miller et al., 2011), 147 (Mussolini et al., 2011), 186 (Slovenia’s TALEs), and 158 or 186 (Mercer et al., 2012) residues were retained. On the C terminus, 28 or 63 (Miller et al., 2011), and 76 (Slovenia’s TALEs) residues were retained.
We learned about three very important TALE characteristics:  Firstly, TALEs demonstrate polarity. In other words, they interact more strongly with DNA at the N terminus compared to the C terminus (Meckler et al., 2013). Secondly, the thymidine nucleotide at the zero repeat is very important to the binding affinity of the TALE (Meckler et al., 2013). Thirdly, the binding affinity of TALE varies based on the types of the nucleotides present in its target site. The relative affinities are: NG (1) > NN (0.18) > HD (0.16) > NI (0.0016) > NK (0.00016) (Meckler et al., 2013). All of these points must be carefully considered while designing the TALE to maximize its binding affinity and specificity.
The three TALEs designed by Slovenia 2012 iGEM team, were ordered from the registry (TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ) and TALE D ( BBa_K782005 )). Primers were designed to incorporate the the target sequences of TALE A and TALE B ([A] and [B]) in the RFP generator. This plasmid will be used as a target sequence. Primers were also designed that incorporate selected point mutations of [A] ( BBa_K1189004 ) and [B] ( BBa_K1189005 ) into a biobrick. Once these plasmids are made they will be used to determine how the TALEs affinity is altered by mutations. In addition, two other TALEs were ordered from the the authors of the Meckler et al. (2013) paper.
Week 5: May 27 - May 31
We continued our literature search for a unique and stable pathogenic E. coli marker for the TALEs to bind to. Potential pathogenic O157:H7 E. coli markers: stx1, stx2 (Yoshitomi et al., 2012), uidA (Yoshitomi et al., 2003 and Feng et al., 1994), eaeA, hly, rbfE (Fortin et al., 2001), and Z3276 (Li & Chen, 2012). However, BLAST searches determined that the use of only one 18 bp sequence is not sufficiently specific and sensitive to detect pathogenic markers. Should we continue to search for only one marker or more than one or expand the detection to any toxic Shiga producing strains?
In addition to searching for a pathogenic E. coli marker, we also started working on the TALE sequence and its construction. The TALE sequence chosen has a truncation in both N and C terminus. The truncated N terminus has 111 residues and the C terminus has 42 residues (Meckler et al., 2013). The sequence was codon optimized for E.coli. NEBCutter 2.0 was used to screen restriction sites for PstI, EcoRI, NotI, XbaI, SpeI, NgoMIV and AgeI, all present in the RFC 25. Cut sites for BsaI and Esp3I were also removed to pave the road for Golden Gate Assembly method. The original TALE sequence has 11 cut sites: BsaI (3), PstI (1), EcoRI (1), Esp3I (4), and AgeI (2). The restriction sites were then eliminated with silent mutations, based on a E. coli codon usage table (San Diego State University, 2012 modified from Maloy et al., 1996).
The large size of the TALE prevents simple cloning, therefore we looked to Golden Gate Assembly. Golden Gate Assembly is a subcloning strategy that uses type II restriction enzymes, which cut outside of their recognition sequence allowing ligation into a final product that lacks any specific scar sites (Engler et al., 2008).
Week 6: June 3 - June 7
This week was also mostly spent on literature research.
The next step after optimization of TALEs and their target sequence is to locate this system’s niche in the beef industry. Many of the current high fidelity methods for pathogenic E. coli detection require cell cultures, which increases turnaround time up to 4 days (Blais et al., 2012 & Gill et al., 2013).
A paper by Jeon et al. (2013) explores the genetic and physiological effects of the cattle on the amount of the O157:H7 E. coli it excretes. According to this paper, 90% of the bacteria excreted from the cattle comes from the super-shedders. Super-shedders are cattle that excrete more than 10 000 CFU/g and have long term recto anal junction colonization. Higher excretion is detected in bulls, pen raised cattle, and certain breeds with a seasonal peak between late spring and early fall. Preharvest detection of pathogenic E. coli decreases cross contamination during meat processing.
Shiga toxin 1 and 2 have 55% homology and are composed of 5 identical highly conserved B subunits that bind to the Gb3 host receptor cells and a single A subunit which enters the host target cell and cleaves rRNA. Cattle do not have Gb3, consequently, O157:H7 infected cattle are asymptomatic and difficult to detect. More than 200 serotypes of E.coli contain shiga toxin, of these, some have LEE (locus of enterocyte) which does not directly indicate pathogenicity. Other strains of pathogenic shiga toxin producing E. coli recognized by the USDA as harmful are: O26:H11, O103:H2, O111:H8, O118:H16, O121:H19, and O145:H28 (USDA  Laboratory Guidebook, 2012). These strains are also known as the Big Six. Currently, the USDA targets both the eae and stx genes using PCR. When these genes are being used in identification, if the sample tests positive for both eae and stx, the sample undergoes further testing. If it only has one or neither of the genes, the sample is considered safe and testing stops (USDA  Laboratory Guidebook, 2012).
Regarding our TALE construct, we compared 2 standard construction strategies for multiple DNA fragments: Golden Gate and Gibson Assembly. Gibson Assembly is a one-step isothermal reaction involving an exonuclease, a DNA polymerase and a DNA ligase. All enzymes are put together in one reaction along with DNA fragments and the vector and incubated at 50°C for 15 to 60 min. (Gibson et al., 2009)
Week 7: June 10 - June 14
This week, we had the opportunity to tour Cargill Meat Solutions’ High River beef plant and experience firsthand the stringency and efficiency of the meat processing facility. Global prevalence of O157:H7 in cattle farms is 20% (Chase-Topping et al., 2008) and the presence of super shedders is influenced by the E. coli strain and host dependent factors (breed, sex, reproductivity, stress, movement) independent of the external farm environment (Chase-Topping et al., 2008). These few super shedders are important factors to pre-harvest contamination as the source of 80% of O157:H7 strains isolated from beef carcasses are from the high cattle density and rapid turnover environment of the processing plant and transport trailers, not the original feedlot (Arthur et al., 2009).
We also met with and discussed E. coli markers with Dr. Glen Armstrong, professor and head of the Department of Microbiology, Immunology & Infectious Diseases at the University of Calgary. Dr. Armstrong suggested that we implement multiple TALEs to target different markers to identify E. coli serotypes.
Primers containing the target sequences of TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ), and variations of the target sequences (with point mutations in different positions) were used to perform a PCR on RFP generator ( BBa_J04450 ); the product of the PCR would be the target sequence of the TALE and a 660base pair part of the RFP gene (Figures 1, 2 and 3). The PCR products were purified and digested with XbaI and PstI and inserted into  a vector; RFP generator was also digested with XbaI and PstI to prepare the vector.
IPTG inducible promoter with RBS ( BBa_J04500 ) was transformed with competent cells and plated on Amp and Chlor.
Figure 1. Kapa HiFi PCR using custom primers to extract mutated TALE B and non mutated TALE A and TALE B target sequences from RFP generator gene. This gel shows [B]5M1 and [B]5M7 which are the mutated first and seventh nucleotide from the 5' end of the TALE B target sequence, respectively. Every lane except for the tenth lane of shows bands at 650bp, the target sequence band is expected at 660bp. Some unexpected amplification is seen in all the lanes. Contamination is observed in the tenth lane. The gel was 1% agarose and was run at 100 V for an hour.
Figure 2. Kapa HiFi PCR using custom primers to extract mutated TALE B and non mutated TALE A and TALE B target sequences from RFP generator gene. This gel shows [B] and [B]5M0 which are the non mutated and mutated zeroth nucleotide from the 5' end of the TALE B target sequence, respectively. Every lane shows bands at 650bp, the target sequence band is expected at 660bp. Some unexpected amplification is seen in all the lanes except for the empty tenth lane. The gel was 1% agarose and was run at 100 V for an hour.
Figure 3. Kapa HiFi PCR using custom primers to extract mutated TALE B and non mutated TALE A and TALE B target sequences from RFP generator gene. This gel shows [B]3M3 and [A] which are the mutated third nucleotide from the 3' end of the TALE B target sequence and the non mutated TALE A target sequence, respectively. All of the [B]3M3 lanes and the third and fifth to eight lanes of [A] shows bands at 650bp, the target sequence band is expected at 660bp. Some unexpected amplification is seen in all the lanes except for the empty tenth lane. The gel was 1% agarose and was run at 100 V for an hour.
The parts received from the registry - TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ), TALE D ( BBa_K782005 ), NicTAL 12 ( BBa_K782007 ) and GFP in RFC 25 Standard ( BBa_K648013 )- were transformed. Subsequently, an overnight cultures of the colonies were made and a plasmid purification was performed. Digestion of the purified plasmids with NotI and running the products on the gel produced the expected results (Figure 4).  TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ) and TALE D ( BBa_K782005 ) were sent for sequencing.
Figure 4.Verification digest of TALEs A ( BBa_K782004 ), B  ( BBa_K782006 ) and D ( BBa_K782005 ) and GFP ( BBa_K648013 ) cut with NotI. All of the TALE lanes show bands at 2100bp and 2400bp, the backbone and TALE band are expected at 2070bp and 2442bp. All of the green fluorescent protein (GFP) show bands at 750bp and 2100bp, the backbone and GFP band are expected at 772bp and 2070bp. No unexpected sized bands are seen. The gel was 1% agarose and was run at 100 V for an hour.
Week 8: June 17 - June 21
While attending the University of Calgary Veterinary Medicine Beef Cattle conference, we learned a lot more about the beef processing industry. Our project was well received by ranchers and members of the cattle industry who found it a practical and applicable option.
In the lab, we ran a colony PCR on the TALE target sequences and ran agar gels (Figures 5, 6 and 7).
Figure 5. Colony PCR of transformed mutated TALE B and non mutated TALE A and TALE B target sequences in Top10 E. coli cells. This gel shows [A], [B], [B]5M0 and [B]3M3 which are the non mutated TALE A, non mutated TALE B, and mutated fifth nucleotide from the 5' end and mutated third nucleotide from the 3' end of the TALE B target sequence, respectively. All of the lanes except for the fifth lane shows bands at 700bp, the target sequence band is expected at 701bp. Some unexpected amplification is seen in all the lanes except for the empty fifth lane. The gel was 1% agarose and was run at 100 V for an hour.
Figure 6. Colony PCR of transformed mutated TALE B and non mutated TALE A and TALE B target sequences in Top10 E. coli cells. This gel shows [B]3M3, [B]5M7, [B]5M8 and [B]5M1 which are the mutated TALE B target sequences. All of the lanes except for the seventh lane shows bands at 700bp, the target sequence band is expected at 701bp. Some unexpected amplification is seen in all the lanes except for the empty seventh lane. The gel was 1% agarose and was run at 100 V for an hour.
Figure 7. Colony PCR of transformed mutated TALE B and non mutated TALE A and TALE B target sequences in Top10 E. coli cells. This gel shows [B]3M3, [B]5M7, [B]5M8 and [B]5M1 which are the mutated TALE B target sequences. All of the lanes except for the seventh lane shows bands at 700bp, the target sequence band is expected at 701bp. Some unexpected amplification is seen in all the lanes except for the empty seventh lane. The gel was 1% agarose and was run at 100 V for an hour.
The PCR products were purified and digested with AflII and PstI and then ran on a gel, producing two bands as expected at ~500 and ~2100 bp. BBa_J04500 only produced one band as expected since it did not have the AflII cut site (Figures 8 and 9).
Figure 8.Verification digest of transformed mutated TALE B and non mutated TALE A and TALE B target sequences and BBa_J04500 cut with AflII and PstI. All of the TALE target sequences lanes show bands at 500bp and 2100bp, the TALE target sequences and backbone are expected at 501bp and 2070bp. Both of the BBa_J04500 lanes show a single band at 2400bp because it did not have the AflII cut site, the band is expected at 2290bp. No unexpected sized bands are seen. The gel was 1% agarose and was run at 100 V for an hour.
Figure 9. Verification digest of minipreps of transformed mutated TALE B and non mutated TALE A and TALE B target sequences and BBa_J04500 cut with AflII and PstI. All of the TALE target sequences lanes except the twelfth lanes show bands at 500bp and 2100bp, the TALE target sequences and backbone are expected at 501bp and 2070bp. No contamination is observed in the negative control without DNA. No unexpected sized bands are seen. The gel was 1% agarose and was run at 100 V for an hour.
We also got the sequencing results for TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ) and TALE D ( BBa_K782005 ). TALE A ( BBa_K782004 ) and TALE D ( BBa_K782005 ) sequences were fine. However, TALE B ( BBa_K782006 ) had a mutation in the RVD region of the second repeat making it specific for a cytosine instead of a thymine. Because of that, we will have to design new primers to add the new TALE B target sequence to the RPF generator.
University closed June 21 due to flooding.
Week 9: June 24 - June 28
University closed June 24 and 25 due to flooding.
Literature search to cleave His tags from proteins determined TAGZyme would be an expensive option.
In the lab, we did a restriction digest of BBa_J04500 , the vector containing promoter and ribosome binding site, with SpeI and PstI. Restriction digest of TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ), and TALE D ( BBa_K782005 ) as separate inserts with XbaI and PstI. Ligated and transformed BBa_J04500 + TALE A and BBa_J04500 + TALE B .
Week 10: July 1 - July 5
University closed July 1 for Canada Day.
Repeated restriction digest of vector BBa_J04500 with SpeI and PstI and of TALE A and TALE B as separate inserts with XbaI and PstI. We ran restriction digest products of TALE A ( BBa_K782004 ), TALE B ( BBa_K782006 ), and BBa_J04500 on gel for gel extraction. Nanodrop concentrations  were lower than 4ng/uL and had poor purity; products were discarded. Repeated ligation and transformation of BBa_J04500 + TALE A and BBa_J04500 + TALE B . Products were plated and run in a colony PCR. For BBa_J04500 + TALE A , only three of the eight  lanes had bands at ~3000bp, while all eight of BBa_J04500 + TALE B had unclean but expected results (Figure 10).
Figure 10. Colony PCR of transformed BBa_J04500 + TALE A and BBa_J04500 + TALE B in Top10 E. coli cells.  Only three of the eight lanes for BBa_J04500 + TALE A and all of the eight lanes for BBa_J04500 + TALE B show bands at 2500bp, the band is expected at 2662bp. No contamination is observed in the negative control without DNA. Some unexpected amplification is seen in the lanes. The gel was 1% agarose and was run at 100 V for an hour.
We also received carbenicillin resistant TALEs from University of California (Meckler et al., 2013): IINNp, IIINGp, IINNp Luciferase, and IIINGp Luciferase. Transformed TALEs were plated on Amp.
We did overnight cultures of BBa_J04500 + TALE A , BBa_J04500 + TALE B , and University of California TALEs. Subsequent miniprep and Nanodrop of the products yielded generally low concentrations ranging from 9.4 - 63.1 ng/uL with mediocre purity. Verification digest of BBa_J04500 + TALE A and BBa_J04500 had bands at 3000bp and 3300bp, not at the expected 2100bp for backbone and 2700bp for insert.
Week 11: July 8 - July 12
This week, we did miniprep on sequence verified BBa_J04500 . We also ran PCR products of the first Kapa HiFi PCR of TALE A and TALE B primers on a 1% agarose gel, however no bands were visible. Subsequent higher temperature gradient Kapa HiFi PCR produced bands around 2500bp and lots of non specific bands (Figure 11). Gel extraction had very low yields (<20ng/uL) and poor purity, and repeated extractions produced similar results.
Figure 11.Gradient Kapa HiFi PCR used to extract construct BBa_J04500 + TALE A and BBa_J04500 + TALE B .  Every lane shows bands at 2500 bp and was expected at about 2442bp. Some unexpected amplification is seen for this PCR. No contamination is observed in the negative control without template DNA. The gel was 1% agarose and was run at 100 V for an hour.
We have received 5 primers for TALE B target sequences and did a Kapa HiFi PCR. Gel of PCR product produced expected results with bands ~600bp (Figure 12).
Figure 12.Gradient Kapa Hifi PCR products of mutated and non mutated TALE B target sequences extracted from RFP generator gene. Every lane except the fifteenth lane shows bands at 650bp, the target sequence band is expected at 660bp. Some unexpected amplification is seen in all the lanes. The gel was 1% agarose and was run at 100 V for an hour.
A Kapa HiFi PCR of K coil primers was performed and showed expected results but also had lots of primer dimers. Subsequent gel extractions had poor purity and yields (Figure 13).
Figure 13. Gradient Kapa Hifi PCR products of K coil. Every lane shows bands at 100bp, the band is expected at 103bp. Some unexpected amplification is seen in the lanes. The gel was 1% agarose and was run at 100 V for an hour.
Week 12: July 15 - July 19
This week we completed ligation, transformation, overnight culture, and miniprep of 5 TALE target sequences (Figures 14 and 15). The verification digest gel had strong bands at ~2000bp and weak bands at ~550bp, and plasmids were sent for sequence verification (Figures 16 and 17).
Figure 14.Colony PCR of transformed mutated and non mutated new TALE B target sequences in Top10 E. coli cells. Thirteen of the sixteen lanes of TALE target sequences show bands at 700bp, the target sequence band is expected at 701bp. Some unexpected amplification is seen in all the lanes. No contamination is observed in the negative control without DNA, the positive control shows bands at 1500, the bands are expected at 1500bp. The gel was 1% agarose and was run at 100 V for an hour.
Figure 15.Colony PCR of transformed mutated and non mutated new TALE B target sequences in Top10 E. coli cells. Both lanes of TALE target sequences show bands at 700bp, the target sequence band is expected at 701bp. No unexpected amplification is seen in the lanes. The positive control shows bands at 1500, the bands are expected at 1500bp. The gel was 1% agarose and was run at 100 V for an hour.
Figure 16.Verification digest of transformed mutated and non mutated new TALE B target sequences in Top10 E. coli cells. All of the TALE target sequences lanes show bands at 500bp and 2100bp, the TALE target sequences and backbone are expected at 501bp and 2070bp. No contamination is observed in the negative control without DNA  No unexpected sized bands are seen. The gel was 1% agarose and was run at 100 V for an hour.
Figure 17.Verification digest of transformed mutated and non mutated new TALE B target sequences in Top10 E. coli cells. All of the TALE target sequences lanes show bands at 500bp and 2100bp, the TALE target sequences and backbone are expected at 501bp and 2070bp. No contamination is observed in the negative control without DNA, the positive control shows bands at 1100bp, 2100bp and 3100bp; the RFP gene, backbone, and uncut plasmid are expected at 1100bp, 2070bp and 3170bp, respectively. No unexpected sized bands are seen. The gel was 1% agarose and was run at 100 V for an hour.
The results of the BBa_J04500 + TALE A and BBa_J04500 + TALE B sequencing were back. However, we realized that TALEs have a Kozak sequence at the beginning, which increases the distance between the promoter and the start codon of the gene. Therefore, a primer (TALseqF) was designed  to take out TALE A and TALE B without the Kozak sequence. The PCR performed with forward primer TALseqF and reverse primer BBK R on TALEs was successful.
BsaI enzyme finally arrived. So the constructions of construct 10 ( BBa_J04500 +His+ TALE A +linker+K coil) and construct 9 ( BBa_J04500 +His+ TALE B +linker+K coil) in Golden Gate reaction were performed. The ligation reactions were run on the gel and expected bands were observed. However, the transformation was unsuccessful because of the new competent cells which were not properly made.
The PCR products of TALE A and TALE B without the Kozak sequence were run on a 1% gel and contained a lot of non specific amplification.
Week 13: July 22 - July 26
The 5 TALE B target sequences were sequence verified and are good.
We repeated PCR of TALE A and TALE B without the Kozak sequence and it worked (Figure 18), so the PCR products were ligated with BBa_J04500 and successfully transformed. Subsequent colony PCR showed some positive results (Figures 19 and 20).
Figure 18.Kapa Hifi PCR product of both TALE A and TALE B without the Kozak sequences. Every lane shows bands at 2400bp, the target sequence band is expected at 2442bp. No contamination is observed in the negative control without DNA. Some unexpected amplification is seen in all the lanes. The gel was 1% agarose and was run at 100 V for an hour.
Figure 19.Colony PCR of transformed BBa_J04500 + TALE A and BBa_J04500 + TALE B in Top10 E. coli cells.  Four of the six lanes show bands at 2500bp, the bands are expected at 2662bp. Some unexpected amplification is seen in all the lanes. The gel was 1% agarose and was run at 100 V for an hour.
Figure 20.Colony PCR of transformed BBa_J04500 + TALE A +Link+K coil in Top10 E. coli cells. One of the seven lanes show bands at 2500bp, the bands are expected at 2662bp. No contamination is observed in the negative control without DNA, the positive control shows bands at 1500, the bands are expected at 1500bp. Some unexpected amplification is seen in all the lanes. The gel was 1% agarose and was run at 100 V for an hour.
Tour of the Chinook Feedlot.
Week 14: July 29 - August 2
For the TALE constructs, we did a colony PCR, plasmid prep, verification digest and sequencing preparation of the BBa_J04500 + TALE A and BBa_J04500 + TALE B , both without the Kozak sequence (Figure 21). Glycerol stocks and streak plates of sequence verified BBa_J04500 + TALE A colony 12 and BBa_J04500 + TALE B colony 7 were made.
Figure 21.Colony PCR of transformed construct 12 without the Kozak sequence in Top10 E. coli cells. Five of the eighteen lanes show bands at 2500bp, the bands are expected at 2662bp. Some unexpected amplification is seen in all the lanes. The gel was 1% agarose and was run at 100 V for an hour.
We also made glycerol stocks and streak plates of the sequence verified BBa_J04500 + TALE A and BBa_J04500 + TALE B with the Kozak sequence, TALE A (colonies 5 and 6) and the intact and mutated (B3M3, B5M0, B5M1, B5M7) TALE B target sequences.
For the Golden Gate constructs, we did a PCR of β-lactamase with primers 16 and 17 and PCR purified the products with concentrations around 40ng/uL.
A colony PCR of construct 2 ( pSB1C3 + BBa_J04500 +His+E coil+Link+hFTN+lFTN) and construct 9 ( pSB1C3 + BBa_J04500 +His+ TALE B +K coil) did not show bands in the expected size.
Week 15: August 5 - August 9
University closed August 5 due to Civic Holiday.
This week we worked on the following constructs:
a) construct 11*: pSB1C3 + BBa_J04500 + TALE A +Link+βlac;
b) construct 11: pSB1C3 - BBa_J04500 +His+ TALE A +Link+βlac;
c) construct 10*: pSBIC3 - BBa_J04500 + TALE A +Link+K coil.
The gel for construct 10* and 11* showed bands in the expected size. After gel purification and DpNI treatment, they were transformed in competent Top10 cells. Colony PCR showed bands of the correct size for construct 11*.
We did a PCR on BBa_J04500 -His- TALE A with primers 1 and 8 to use it as our vector in Golden Gate Assembly of construct 11. The gel showed bands of the correct size but PCR purification had low yields (20.4ng/uL) and low purity. After Antarctic Phosphatase treatment, we set up Golden Gate Assembly of construct 11. The gel showed no bands. We repeated the procedure, and PCR had nonspecific amplification. Gel purification also had low yields and low purity. After that, we did a PCR with less amount of DNA (10ng/reaction) which showed no bands. Another attempt was performed with different DNA concentration (5 to 45ng/reaction). Some of them had no bands and some of them had non-specific amplification.
Week 16: August 12 - August 16
This week, we did minipreps of construct 11* ( pSBIC3 - BBa_J04500 + TALE A +Link+βlac) of the colonies that showed correct bands in the colony PCR. Following the successful verification digest (Figure 22), the samples were sent for sequencing. We did a Kapa PCR of BBa_J04500 + TALE A , but the gel of the PCR products showed a band size that was approximately 1000bp less than expected. This PCR was done so we could use it for other constructions. Instead we ordered new primers to try a different approach to our remaining constructs.
Figure 22.Verification digest of transformed construct 11 without His tags in Top10 E. coli cells. Six of the thirteen lanes show bands at 2100bp, 2800bp and 5000bp, the backbone, construct and uncut plasmid are expected at 2070bp, 3546bp and 5616bp. No contamination is observed in the negative control without DNA, the positive control shows no bands however the BBa_J04500 lane shows bands at 2100bp and 2300bp, the backbone and uncut plasmid are expected at 2070bp and 2290bp. The gel was 1% agarose and was run at 100 V for an hour.
Now that we have our TALEs we need to express them. We are experimenting using different time points and IPTG amounts to determine how to best express our proteins. We will be running our samples on SDS PAGE gels next week.
Week 17: August 19 - August 23
This week we repeated overnight cultures, minipreps, and sequencing of BBa_J04500 + TALE A and construct 11* ( pSBIC3 + BBa_J04500 + TALE A +Link+βlac), as previous sequencing results were inconclusive.
We also received new primers for construct 10 ( pSBIC3 - BBa_J04500 +His+ TALE A +Link+K coil) and construct 11 ( pSB1C3 + BBa_J04500 +His+ TALE A +Link+βlac).   We set up a PCR of the construct 11 primers with BBa_J04500 + TALE A and BBa_J04500 + TALE B template DNA. The gel of the PCR products showed bands about ~2900bp in length.  We also set up a PCR of the construct 10 primers and BBa_J04500 + TALE A template DNA. The gel of the PCR products showed 1800bp bands, which were too small. We repeated the PCR with an increased extension time and the gel showed bands of the expected size. The PCR product was purified and cut with BsaI, ligated together and transformed.
Because the PCR product was shorter than expected for the first Kapa PCR (construct 10), we also did a PCR of the BBa_J04500 + TALE A and BBa_J04500 + TALE B with BioBrick primers and with specially designed TALE primers to confirm the length of our TALEs. The results were good (Figure 23).
Figure 23.Kapa Hifi PCR product of both construct 9 and 10, BBa_J04500 + TALE A and BBa_J04500 + TALE B .The first eight lanes of the whole constructs with Biobrick primers show bands at 5000bp, the bands are expected at 4932bp. The next four lanes of the TALEs with biobrick primers show bands at 3000bp, the bands are expected at 2862bp. The next four lanes of the TALEs with biobrick primers show bands at 2500bp, the bands are expected at 2442bp. No contamination is observed in the negative control without DNA. Some unexpected amplification is seen in all the lanes. The gel was 1% agarose and was run at 100 V for an hour.
We did a Kapa PCR of  TALE target sequences and the gel of the PCR products had bands of the expected size (Figure 24). The PCR purified product and RFP vector were cut with EcoRI and SpeI and then ligated together.
Figure 24.Kapa HiFi PCR using custom primers to extract construct of combined TALE target sequences [A] and [B]. The construct of the two target sequences contained about 200bp of junk DNA between the two target sites and was flanked by the biobrick cut sites on both side. Every lane shows bands at 500 bp and was expected at about 507bp. Some unexpected amplification is seen for this PCR. No contamination is observed in the negative control without template DNA. The gel was 1% agarose and was run at 100 V for an hour.
Regarding our TALE expression and characterization, we did a mini expression of TALE A , TALE B , β-lactamase, BBa_J04500 , and RFP .  We also did a trial run to determine the optimum time point and IPTG concentration.  We subcultured our overnight cultures of TALE A , TALE B , the positive control RFP and the negative control BBa_J04500 . For each pellet and supernatant, we had 3 time points of 3 hours, 6.5 hours, and 24 hours and 3 IPTG concentrations of 0.1mM, 0.4mM, and 0.8mM for a total of 72 samples on 8 SDS page gels. Our results indicate that the optimum IPTG concentration is 4mM and 8mM IPTG and induction time of 3 hours and 6.5 hours.  A subsequent SDS page gel was run with TALE A , TALE B , β-lactamase, BBa_J04500 , and RFP .
Figure 25. Lab adventure: running 8 SDS Page Gels!
Week 18: August 26 - August 30
We did a colony PCR of construct 9 ( pSBIC3 + BBa_J04500 +His+ TALE B +Link+K coil), construct 10 ( pSBIC3 + BBa_J04500 +His+ TALE A +Link+K coil), and the TALE target sequences which were transformed over the weekend. The PCR products were run on a gel, the colonies with the bands of the right size were made into overnight cultures and miniprepped. The gel of the verification digests did not have any bands and the overnight cultures were repeated with good results (Figure 26). Construct 9, construct 10, and the TALE target sequences were sent for sequencing.
The PCR product of construct 11 ( pSBIC3 + BBa_J04500 +His+ TALE A +Link+βlac) was purified and cut with EcoRI and AgeI and ligated with the vector, linker and β-lactamase, which was cut with EcoRI and NgoMIV. The products were transformed and the gel of the colony PCR had faint bands at ~3500bp. These 4 colonies with the expected size bands were made into overnight cultures and miniprepped, however the gel of the verification digests showed 3 bands at approximately 2000 bp, 2900 bp, and 5000 bp.
A Kapa PCR of construct 11 was repeated, along with digestion, ligation, and transformation.
Figure 26.Verification digest of transformed construct 10 in Top10 E. coli cells. Three of the four lanes show bands at 2100bp, 2700bp and 5000bp, the backbone, construct and uncut plasmid are expected at 2070bp, 2642bp and 4712bp. No contamination is observed in the negative control without DNA. The gel was 1% agarose and was run at 100 V for an hour.
Week 19: September 2 - September 6
This week, we got the sequencing results for construct 9 ( pSBIC3 + BBa_J04500 +His+ TALE B +Link+K coil), construct 10 ( pSBIC3 + BBa_J04500 +His+ TALE A +Link+K coil) and the TALE target sequences. The TALE target sequences (with old and new [B] + [A]) worked. One of the primers for construct 9 sequencing had poor results, so it was resent for sequencing. Construct 10 didn’t work, so we set up digestion, ligation and transformation again. Colony PCR revealed right-sized  bands for 4 colonies (Figure 27). We set up overnight cultures and mini prepped those over the weekend.
Figure 27. Colony PCR of transformed construct 10 in Top10 E. coli cells. Four of the twelve lanes show bands at 3000bp, the bands are expected at 2862bp. No contamination is observed in the negative control without DNA. Some unexpected amplification is seen in the lanes.The gel was 1% agarose and was run at 100 V for an hour.
We did colony PCR of construct 11 ( pSB1C3 + BBa_J04500 +His+ TALE A +Link+βlac), but the bands were not at the expected size, so we repeated transformations with more competent Top10 cells.
We also received new sequences of construct 9, construct 10, and construct 11 from GenScript and did a transformation with them. The plates had colonies, now the next step is to plasmid switch them to pSB1C3 , because they came in PUC57 backbone. We did a colony PCR and mini-prepped the colonies that worked.
Regarding our protein work, we have transformed BBa_J04500 + TALE A and BBa_J04500 + TALE B into E. coli ER2566 and into E. coli BL21, strains commonly used for protein expression. We also tried a different lysis protocol, with glass beads, which had good yields for protein concentration as determined by the Bradford assay. We did a Western Blot and verified that we have expressed the proteins (Figure 28).
Figure 28.Western Blot of BBa_J04500 + TALE A and BBa_J04500 + TALE B .
Week 20: September 9 - September 13
This week, we did a verification digest of construct 10 ( pSBIC3 - BBa_J04500 +His+ TALE A +Link+K coil) and construct 11 ( pSB1C3 + BBa_J04500 +His+ TALE A +Link+βlac). One of the colonies for construct 11 showed the correct band size, so it was sent for sequencing, however it was missing 100bp of the βlac gene. We transformed construct 9 ( pSBIC3 + BBa_J04500 +His+ TALE B +Link+K coil), construct 10, and construct 11 from GenScript in pSB1C3 again. Colony PCR showed some positive results for construct 9 and construct 11.
In addition, we also did plasmid switches of construct 2, construct 20, construct 21, and construct 22 from PUC57 backbone into a PS1C3 backbone. Both the constructs and vector were cut with EcoRI and PstI, and the RFP generator vector also had antarctic phosphatase treatment. The ligations for each construct had 2 different vector amounts, 25ng and 50ng, as well as different insert to vector ratios, 3:1 and 6:1. Construct 2, construct 20, construct 21, and construct 22 in PS1C3 were miniprepped and the verification digests  were good for some of construct 20, construct 21, and construct 22.
Construct 7 and construct 11 sequences received from GenScript were transformed into competent Top10 cells and plated on ampicillin. Overnight cultures and minipreps were made of the successful colonies.
To test our proteins, we needed to use purified versions of them. Therefore, we purified TALE A and TALE B and ran them on an SDS PAGE gel (shown below). The gel showed that we were successful in our purification.
Figure 29.SDS Page Gel of Purified TALE A ( BBa_K782004 ) and TALE B ( BBa_K782006 ).
We also did a gel shift assay to test the binding of TALE A to its target site. We tried different concentrations for TALE A + [A] and we picked two of those to run TALE A + [B]. However, the concentrations of protein were too high. We will repeat the assay next week with lower concentrations of TALE.
Week 21: September 16 - September 20
This week, we did plasmid prep and verification digest of construct 9 ( pSBIC3 + BBa_J04500 +His+ TALE B +link+K coil), construct 11 ( pSB1C3 - BBa_J04500 +His+ TALE A +Link+βlac), and construct 23. We had some good results for construct 11, so we did overnight cultures and mini preps and submitted it to Parts Registry for the DNA submission along with our other constructs.
We also repeated the gel shift assay, this time with TALE A and TALE B . The concentration range was from 0.1ng/uL of protein to 40ng/uL. Because Red Safe, which we used for staining the DNA, is not very sensitive we loaded higher amounts of DNA than recommended and the gel revealed nonspecific binding by the TALES.    We repeated the assay and added chromosomal DNA to prevent nonspecific binding, added the recommend amount of DNA (55pM), and stained the gel after it ran with SYBR Green, unfortunately, we were not able to see any bands at all.
We transformed construct 11 into E. coli BL21, cultured the colonies, induced them with IPTG and purified the protein. We performed Bradford Assay on our samples and the concentration range was between 0.20mg/mL and 1.06mg/mL. We ran these samples on a 7.5% SDS page gel (Figure 30).
Figure 30.SDS Page Gel of Construct 11 ( pSB1C3 - BBa_J04500 +His+ TALE A +link+βlac).
Because construct 11 has βlac on it, we cultured the cells in ampicillin too as β-lactamase provides resistance to β-lactam antibiotics. The medium density increased suggesting that β-lactamase was functional in our construct.
Week 22: September 23 - September 27
This week, we did an ampicillin survival assay to verify if the expressed construct 11 ( pSB1C3 + BBa_J04500 +His+ TALE A +Link+βlac) was able to degrade ampicillin, allowing non β-lactamase producing bacteria to survive in LB and ampicillin.
Regarding protein expression and characterization, we transformed construct 9 ( pSBIC3 + BBa_J04500 +His+ TALE B +Link+K coil), construct 10 ( pSBIC3 - BBa_J04500 -His- TALE A +link+K coil), and construct 11 in E. coli BL21.
A Bradford assay, protein assay, and Western blot were done of construct 2, construct 6, construct 7, construct 9, construct 10, construct 11, construct 23, BBa_J04500 . BBa_J04500 + TALE A and BBa_J04500 + TALE B (Figures 30 and 31).
Figure 31.SDS Page Gel of Construct 10 ( pSBIC3 - BBa_J04500 +His+ TALE A +Link+K coil. Bands were expected at 82kDa.
Figure 32.SDS Page Gel of TALE B . Bands were expected at 82kDa.
We performed a Western Blot on TALE B B ( BBa_K782006 ) with and without the Kozak sequence. The TALE B without the Kozak sequence did not work, indicating that it cannot be expressed in E. coli, while our TALE with the removed Kozak sequence can.
Figure 33. Western Blot of crude samples of TALE B with and without the Kozak sequence
We also performed Western Blots on construct 7 ( BBa_K1189021 ) and construct 11 ( BBa_K1189031 ). The gels showed that we have successfully expressed our protein.
Figure 34. Western Blot of Construct 7 ( BBa_K1189021 ) showing that our protein construct has been expressed in E. coli
Figure 35. Western Blot of Construct 11 ( BBa_K1189031 ) showing that our protein construct has been expressed in E. coli
Week 23: September 30 - October 4
Practice, practice, practice for our Regional Jamboree presentation!
Week 24: October 7 - October 11
More of the plasmid containing the construct of the combined [A] and [B] ( BBa_K1189006 ) was prepared for future assays.
We also did a gel shift assay in order to test the ability of TALE B to bind to its intended target site [B] as well as the non intended target site [A] as a control (Figure 36). The gel shift failed, most likely due to an excessive concentration of protein in the loaded samples. This concentration was amended and optimized for following gel shift assays.
Figure 36. Gel shift assay of TALE B and the intended [B] and non intended [A] target sites. Five of the lanes contain varying concentrations of the intended [B] target site and two of the lanes contain varying concentrations of the non intended [A] target site. No contamination is observed in the controls with only the target sites. The gel shift failed, most likely due to an  excessive concentration of TALE B protein in the loaded samples. The gel was 1.3% agarose and was run at 50 V for 40 minutes.
Week 25: October 14 - October 18
This week we continued optimizing our protocol for the gel shift assay. After observing the electrophoretic mobility shift assay (EMSA) or gel shift assay results (Figure 37), we can clearly see that the loaded samples of protein and target sites cannot leave the well. Some of the wells such as wells 8 and 9 clearly exhibit this problem. The protein remains stuck in the wells and cannot go through the pores of the native gel. In order to correct this, we decided to modify the percentage of the native gels and increase the acrylamide to bisacrylamide ratio.
Figure 37. Gel shift assay of TALE A+K coil and the intended [A] and non intended [B] target sites. Thirteen of the lanes contain varying concentrations of the intended [A] target site and four of the lanes contain varying concentrations of the non intended [B] target site. No contamination is observed in the negative controls and the controls with only the target sites. The gel was 1.3% agarose and was run at 50 V for 40 minutes.
In addition, we started cloning the EHEC target sequence into a vector that already contains the target sequence for TALE B.
Week 26: October 21 - October 25
Repeated gel shift assay with TALEA+K coil ( BBa_K1189029 ) on a FAM labeled [A] on polyacrylamide gel. This time a 70:1 acrylamide to bisacrylamide ratio was used to make the native gel. Gels of varying percentages were made: in the 3.75% gel, no shift was observed; in the 5.5% gel, no clear distinct shifted band was observed.
After the unsuccessful gel shift assays with native gels, we attempted to run our samples on agarose gel instead. FAM labeled DNA of [A] was run on a 1.3% agarose gel with increasing dilutions (Figure 38). Unfortunately the DNA did not run on the gel, therefore a new method for testing the binding of the TALEs to the DNA was developed.
Figure 38. This gel shows FAM labeled [A] TALE A target site DNA of increasing dilutions. The ratios on the top show the dilution factors of a 50mM DNA solution. The gel was 1.3% agarose and was run at 50 V for an 40 minutes.
We did a colony PCR of our construct of combined EHEC target sequence and [B] TALE B target sequence (Figure 39).
Figure 39. Colony PCR of transformed construct of combined TALE target sequences [EHEC] and [B] in Top10 E. coli cells. All of the lanes show bands at 500bp, the bands are expected at 507bp. Some unexpected amplification is seen in the lanes. The gel was 1% agarose and was run at 100 V for an hour.
We ordered 60mer FAM labeled [A] TALE A target sequence and FAM labeled [B] TALE B target sequence oligos and hybridized them with their reverse complement oligo to make double stranded pieces of DNA containing the target sequence of our TALEs. Using these target sequences and following the TALE Nitrocellulose Functionality Assay , we showed that TALEs bind to their target sequence. We incubated Ferritin fused to an E coil to TALE fused to a K coil to make the FerriTALE complex. The complex was then blotted on strips of nitrocellulose paper. The strips were then blocked with milk and soaked in the appropriate DNA solution. Finally, the strips were washed and imaged. We showed not only that TALEs bind DNA (Figure 40 and 41), they are also specific to their own target site (Figure 42).
Figure 40. (A) Dot blot of FerriTALE A exposed to FAM labeled DNA containing the [A] TALE A target sequence ( protocol ). 1.5µg of TALEA+K coil and 1µg of ferritin with E coil were incubated for 1 hour to make the FerriTALE complex and the complex was blotted on a strip. The blots were then exposed to 1.66 mM FAM labeled [A] TALE A target site from 1 to 90 minutes as indicated on the strips.  "x" is a FerriTALE that was exposed to FAM labeled DNA prior to being blotted onto the nitrocellulose. The kinetics from the densitometry is shown in section B of the figure. The Kd from this plot was determined to be 293nM.
Figure 41. (A) Dot blot of FerriTALE B exposed to FAM labeled DNA containing the [B] TALE B target sequence ( protocol ). 1µg of ferritin fused to E coil was incubated with 2µg of TALE B fused to K coil for 1 hour to make the FerriTALE B complex. Subsequently the complex was blotted on the nitrocellulose strip. The blots were then exposed to 1.66 mM FAM labeled DNA from 1 to 90 minutes as indicated on the strips. The controls are to the right, with "Ftn" being ferritin only, "np" being no protein, and "D-" being no DNA exposure. The kinetics from the densitometry is shown in section B of the figure. The Kd from this plot was determined to be 66nM.
Figure 42. (A) A Dot blot of TALE A on nitrocellulose paper ( protocol ). A6 is TALE A soaked in 1.66mM FAM labeled [B] TALE B target sequence. A7 is TALE A soaked in 1.66mM FAM labeled [A] TALE A target sequence. A2 is TALE A soaked in 1mM FAM labeled [B] TALE B target sequence. A3 is TALE A soaked in 1mM FAM labeled [A] TALE A target sequence. On A- strip no protein was blotted and it was soaked in 1.66mM [A]. All strips were soaked in DNA solution for 90 minutes. (B) 1µL of the DNA solutions used for soaking were blotted on nitrocellulose and a picture was taken instantly, to indicate that both [A] and [B] fluoresce to the same extent.
Week 27: October 28 - November 1
It's our favourite time of the year; Wiki Freeze! See you in Boston.
Retrieved from " http://2013.igem.org/Team:Calgary/Notebook/Journal/Detector "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
