"
Page
Discussion
View source
History
teams
Log in
 
Team:Calgary/Sandbox/Project/Detector/ProofofConcept
From 2013.igem.org
Home
Project
Our Sensor
Detector
Reporter
Prussian
Blue Ferritin
β-Lactamase
Linker
Modelling
Spatial
Modelling
Quantitative
Modelling
Prototype
Our Platform
Human Practices
Informed
Design
Safety
Collaboration
SensiGEM
Database
Data Page
Achievements
Final
System
Our Team
Team Members
Our Profile
The
University
Attributions
Sponsors
Notebook
Journal
Detector
Reporter
Linker
Modelling
Protocols
Parts
References
Outreach
Telus
Spark
Consort
Alberta
Beakerhead
Genome
Alberta
Science
Café
iGEM
Proof of Concept
The TALEs are a very fundamental part of our project this year. In order to have a functioning system for E. coli detection, we need to have proteins that will successfully recognize and bind our DNA. However, before we can actually create and use TALEs to bind to E. coli DNA, we need to have a proof of concept. On the iGEM parts registry we came across TALEs that the Slovenian team synthesized for a previous project. TALEs are very large proteins and take a long time to design and synthesize. Therefore, we decided to use these TALEs to test our system. We also saw this as an opportunity to use and build upon parts made by former iGEM teams. Thus, we ordered their three TALEs: TALA ( BBa_K782004 ), TALB ( BBa_K782006 ), and TALD  ( BBa_K782005 ).
We used TALA( BBa_K782004 ) and TALB ( BBa_K782006 ) to build our constructs and test our system, as it requires the use of two different TALE proteins. To test our TALEs, we had to synthesize the target sequences that they would recognize and bind to. We constructed a series of target sequences in order to test the binding affinity. Some sequences were directly corresponding with the TALE protein sequence, while some had certain base pair alterations. These mutations will allow us to test the binding affinity of the TALE when it encounters a non-exact target sequence. In this way, we can determine how specific these proteins really are and how they might respond in our test to DNA not belonging to enterohemorrhagic (EHEC) E. coli. This will help us define how specific we expect our final system to be.
When we sequenced TALB ( BBa_K782006 ), we discovered that it had a small mutated segment. We expected a sequence of AGCAATGGG in the repeat variable di-residue of the second repeat. However, the sequence was actually TCCCACGAC. This means that the required target sequence at this position is a C, and not a T, as the parts registry web page shows.  The two TALEs also had a kozak sequence at the front of the sequence. Instead of changing the TALE, we decided to create a new series of target sequences for the mutated TAL B ( BBa_K782006 ) to bind to. We also used PCR to remove the kozak sequence at the front of each TALE, so it would not inhibit the expression.
Having target sequences created for both TALA( BBa_K782004 ) and TALB ( BBa_K782006 ), we assembled a target sequence construct ( BBa_K1189006 ), to be used in our system. Special primers were synthesized and used to create a linear PCR product, containing both the A and B target sequences. We then put it into a pSB1C3 backbone by switching out RFP.
With the TALEs ordered from the registry and our correct TALE target sequences, we have been able to demonstrate a proof of concept by binding TALE proteins with DNA. Next, we will use our designed TALEs to bind to the DNA of EHEC E. coli .
Retrieved from " http://2013.igem.org/Team:Calgary/Sandbox/Project/Detector/ProofofConcept "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
