"
Page
Discussion
View source
History
teams
Log in
 
Team:Duke/Project/Notebook
From 2010.igem.org
  ड्यूक i G E M
home
project
about us
Details
Notebook
Biobricks
Protocols
Judging/Safety
Duke iGEM 2010 Notebook
Contents
1 October
2 September
3 August
4 July
5 June
October
10/26/10:
Biobricks K429000 and K429001 submitted to registry.
10/25/10:
Samples of BioBrick submissions K429000 and K429001 were successfully miniprepped and were packaged for shipping.
CPEC reaction from 10/24/10 showed only faint banding, so the extension cycle duration was increased and the reaction was performed again.
10/24/10:
BioBricks from 10/23/10 were transformed into GC5 and grown in liquid medium.
First attempt at CPEC was performed.
10/23/10:
K429000 and K429001 samples from 10/22/10 were miniprepped and cut with EcoRI and PstI, the ligated into the precut linearized pSB1C3 plasmids supplied by the registry.
PCR of plasmid DNA from 10/22/10 produced insufficient yield of the fully extended plasmid, so another run of PCR was performed on the sample with varied annealing temperatures to find an optimal annealing temperature.
10/22/10:
Q04400, Q04510 and I13504 were first miniprepped, then digested and ligated following the standard assembly process. Gel electrophoresis was performed on the samples and they appeared to have correctly ligated, so they were then transformed into GC5 colonies and grown in a liquid medium.
PCR assembly of [J23100 + P0412 + B0015 + pSB1C3 overlap] appeared to be successful; the product was extracted from the gel and further extended by the PCA of [pSB1C3 overlap + pLacI +RBS], so that the pSB1C3 plasmid would be ready for CPEC.
10/18/10:
PCR assembly of [J23100 + P0412 + B0015 + pSB1C3 overlap] was unsuccessful; annealing temperatures were changed and the reactions were run again. All other PCRs from 10/17/10 were successful, and upon gel extraction the samples were prepared for the CPEC. Noting that problems arising with CPEC would mean that we would be unable to submit BioBricks by the deadline, parts Q04400, Q04510 and I13504 were transformed into GC5 and grown in liquid cultures for extraction.
10/17/10:
PCA of [A-fos + B0015],  [J23100 + RBS + c-fos] were successful, and were followed by the PCA of the new segments. Each PCA product was run through a 1% agarose gel; banding was very faint for the final product, and upon gel extraction only .056ug of the product DNA were yielded. This product was put through another round of PCR prior to the second assembly step, and upon gel extraction the product yield was of .218ug. The PCR assemblies for [c-jun + B0015 + custom promoter], [RBS + tetR + B0015 + RBS + GFP + terminator], [J23100 + P0412 + B0015 + pSB1C3 overlap] and [pSB1C3 overlap + R0010 +RBS] were started.
10/15/10:
Annealing temperatures for PCR reactions from 10/8/10 were modified, and the reactions were performed again; Yields were significantly higher than those from 10/11/10, and DNA bands appeared clearly at the appropriate lengths in the gel electrophoresis. PCA of [A-fos + B0015],  [J23100 + RBS + c-fos]  was performed.
10/11/10:
Gel electrophoresis was performed on samples from 10/8/10, the result being very low concentrations of extended DNA. Gel extraction was performed, but reported yields of negligible concentration.
10/8/10:
New primers for BioBrick parts arrived, and PCA reactions were started for Q04400, I13504, J23100 and P0412. Gel extraction of PCR extensions of samples from 10/4/10 was performed, with yields between .15 and .2 ng.
10/4/10:
Electrophoresis revealed that only A-Fos, c-Jun and J23100 had extended properly. The extended samples were extracted, changing annealing temperatures will likely show extension for OH-Fos and B0015.
10/1/10:
Ran PCA for samples of A-Fos, c-Jun, OH-Fos, custom promoter, J23100, and B0015
September
9/27/10:
Having had problems with BioBrick assembly we determined that the alternative method of PCA would likely produce better results. New primers were designed and ordered to remove the necessity for BioBrick Standard Assembly.
Reactions can be spatially separated in assembly process to reduce mis-dimerization possibilities as follows:
1  pSB1C3 + pLacI +RBS
2  A-fos +B0015 + J23100 +RBS +c-fos
3  c-jun + B0015 + pDIMER
4  RBS + tetR + B0015 + pTet + RBS + GFP + terminator
5  J23100 + RBS + lacI + terminator + overlap with pSB1C3
9/24/10:
Either 9/17/10 ligation or 9/13/10 digest were unsuccessful, as the control for transformation had formed bacterial lawn on plates with the same antibiotic concentrations.
9/21/10:
Transformation failed completely, no growth was visible on antibiotic plates.
9/20/10:
Ligated biobricks were transformed into GC5 cultures and plated on agar with the requisite antibiotics.
9/17/10:
Reattempt ligation. Ligation determined to be successful- bands of the requisite lengths appeared when the product was electrophoresed.
9/13/10:
Digestion reaction was performed again, and appeared to be successful, the products were run through a 1% agarose gel and bands of approximately the lengths of the BioBrick parts were present as well as bands the length of the plasmid DNA; Samples were extracted from the gel.
9/9/10:
Ligation was unsuccessful, multiple bands were visible in the gel electrophoresis of the sample and none of them were of the requisite length.
9/6/10:
With over .2 ug of DNA for each BioBrick the ligation was reattempted.
9/3/10:
The DNA yields from 8/27/10 were deemed to be to low to be ligated and then transformed, so another digestion reaction was performed in order to collect enough digested BioBrick DNA to ligate together successfully.
August
8/27/10:
Retried the biobrick digest protocol from 8/20/10, with a digest time decreased to 1 hour. Faint bands showed when the samples were run through a 1% gel. Samples were extracted from gel, and nanodrop analysis revealed very low concentrations of DNA, no yield above .1 ug, with the lowest being  Q04400 at .082ug.
8/20/10:
Miniprep of Biobrick cultures from 8/16/10. Yield was lower than previously, on average .5 ug plus or minus .060 ug.   Biobrick enzyme double digest from standard assembly performed on Q04400, Q04510, I13504. Digested samples were run through a 1% agarose gel, and no bands appeared, although significant smearing was evident. It is likely that the digestion ran for too long, and nonspecific cutting resulted.
8/16/10:
*start of classes    - time available for iGEM work greatly reduced.  Miniprep of 8/15/10 leucine zipper DNA succesful, nanodrop showed DNA yield to be ~.6 ug plus or minus .050 ug. Transformation from 8/15/10 was successful. Started liquid cultures for Q04400, Q04510, I13504, R0010, B0015, J23100 minipreps.
8/15/10 :
Transformed iGEM parts Q04400, Q04510, R0010, B0015, J23100; Started liquid cultures for A-Fos, c-Jun, OH-Fos, 4H-Fos miniprep as we ensured we would have sufficient DNA for the assembly process.
July
7/19/10:
More team members from 2009- Sahil and Ang, were contacted for feedback/help. Sahil was unavailable but Ang made a series of suggestions pertaining to the assembly process that were later adopted, and suggested RT-PCR as a more reliable determinant of transistor functionality than a fluorescence assay.
Primer design for transistor assembly and CPEC finalized.
7/16/10:
New BioBrick parts were ordered to serve as a testing chasis for the leucine zipper repression pathway.
7/15/10:
Having been given approval to continue, we determined how we would test and assemble each of the transistors that we had designed. CPEC was proposed as a method for the part assembly, and appeared to be the best option as we would be combining biobrick and nonbiobrick parts into a  biobrick. We then designed primers for leucine zipper PCR extension.
7/13/10:
Project idea was presented for review to Dr. Tian, who, having spoken to Dr. Buchler previously, aproved of the idea.
7/7/10:
Sequencing order was repeated, one sample was contaminated (A-Fos) and no usable data was gathered
June
6/29/10:
Samples of A-Fos, c-Jun, and OH-Fos were miniprepped and the resulting DNA was sent in for sequencing.
6/19/10:
Received a donation of A-Fos, c-Jun, OH-Fos plasmids from Dr. Buchler, who had initially modeled dimer repressed pathways with leucine zippers as being potentially ultrasensitive. The samples we received were transformed and plated, then grown in liquid cultures for plasmid DNA extraction.
6/18/10:
Idea to use fligA and sigma28 factor repression was discarded due to the realization that altering levels of sigma factors already present in bacteria would likely not cause isolated changes that would be easily observable, but would have lead to drastic changes in the functioning of the bacterial cells. Alternative pathways were explored using leucine zippers, which, not being produced naturally in E coli, should not have met any interference and could therefore be studied in a controlled manner.
6/17/10:
Designed assembly for sigma protein repression
6/16/10:
Given our limited labour resources this year, the team decided to opt for a simpler project- the design of a transistor/amplifier circuit for gene expression.
http://www.ncbi.nlm.nih.gov/pubmed/19003438
http://www.ncbi.nlm.nih.gov/pubmed/19455136
http://www.genome.duke.edu/people/faculty/buchler/publications/
http://www.sciencedirect.com/science?_ob=ArticleURL&_udi=B6WK7-4TN0M2Y- 1&_user=38557&_coverDate=12%2F31%2F2008&_rdoc=1&_fmt=high&_orig=search&_sort=d&_docanchor=&view=c&_searchStrId=1371893 022&_rerunOrigin=google&_acct=C000004358&_version=1&_urlVersion=0&_userid=38557&md5=081ad43e29f1e89810bf973c1e421506
6/10/10:
Alterior topic proposals were made, including the following:
1
Human waste → Hydrogen
Nitrogenase
Alcanivorax borkumensis enzymes for aromatic hydrocarbon degradation
gas vescicles for hydrogen collection? (if so, include lysis mechanism)
Concept:
This bacterium would turn common organic water contaminants into a usable and renewable fuel, collected into gas vescicles for easy retrieval from any natural setting, or alternatively for water treatment plants.
2
Bacterial bandaid:
Chitosen and platelet production
Ampercillin production and immunity for disinfectant/ predator prey mechanism
Concept:
What this would attempt to accomplish would be to produce clotting at a wound site while simultaneously disinfecting the site, thereby sterilizing the wound and preventing blood loss.
3
Bone Marrow Replacement/ pluripotent bacteria:
Imitates megakaryocyte response to thrombopoietin
Releases platelets in response to TPO
Concept:
Compartmentalization of DNA could make it possible to have a bacterium with multible possible differentiation outlets, whose daughter cells would be very fundamentally different from the parent cell in response to stimulus (TPO). Purpose would be to supply readily available artificial bone marrow transplants, also just as proof of concept for a pluripotent bacterium.
4
Bacterial alarm clock
Ammonium carbonate production
Oscillatory system
Concept:
Simply circadian oscillatory system or light response system, leading to a pathway for release or production of ammonium carbonate/smelling salts. Efficient olfactory alarm clock, potentially impractical due to irritation from long exposure to ammonium carbonate.
5
In vivo adrenaline pump
Adrenaline production
Pressure response for blood level
Concept:
Tachycardia results in irregular pressure changes, release of adrenaline or epinephrine in response to change in blood pressure levels or heartbeat irregularities could allow a bacterium to function as an on site adrenaline pump.
6/9/10:
Preliminary research was conducted, and ideas to continue the Duke iGEM 2009 project were proposed. Proposed project options included the codon optimization of the bioplastic synthesis process.
http://igem.bme.duke.edu/doku.php?id=attacking_the_plastic_waste_problem-_a_two-pronged_approach#regulation_of_the_synthesis_of_poly_3-hydroxybutyrate-co-4-hydroxybutryate_-_experimental
http://en.wikipedia.org/wiki/Citric_acid_cycle#Major_metabolic_pathways_converging_on_the_TCA_cycle
http://www.jbc.org/content/277/34/30409.full
http://www.jbc.org/content/264/23/13430.full.pdf
Retrieved from " http://2010.igem.org/Team:Duke/Project/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
