"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Protocols
From 2011.igem.org
Team BYU Provo
Home
Team
Project
Results
Protocols
Parts Submitted
Modeling
Safety
Attributions
Notebooks:
Team OxyR:
The Beginning
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
Week 13
Week 14
Week 15
Week 16
Week 17
Week 18
Week 19
Week 20
Week 21
Week 22
Week 23
Week 24
Week 25
Week 26
Week 27
Week 28
Team Thermosensor:
The Beginning
Week 1
Week 2
Week 3
Week 4
Week 5
Week 6
Week 7
Week 8
Week 9
Week 10
Week 11
Week 12
Week 13
Week 14
Week 15
Week 16
Week 17
Week 18
Week 19
Week 20
Week 21
Week 22
Week 23
Week 24
Indiana Trip
Outreach
 
Contents
1Protocols
2PCR
3PCR (Phusion Polymerase - 50 μl reaction)
4PCR (Taq Polymerase - 50 μl reaction)
5Colony PCR (25 μl reaction)
6Standard Agarose Gel
7Restriction Digest of insert and/or vector (50 μl reaction)
8Low-melt gel purification
9Ligation (15 μl reaction)
10Transformation
11Sequencing (submitting to BYU DNA sequencing center)
12Freezing strains in -80˚C freezer
13Electroporation
14Mutagenic PCR (from paper by RC Cadwell and GF Joyce in Genome Research, 1994)
15"Plate Reader Experiment" Protocol
16Beta-Galactosidase Assay
17ZS Buffer
Protocols
These are the marvelous protocols our team used this summer to create BYU's E. colinoscopy.
PCR
There are two types of PCR polymerases used in the lab, Taq and high fidelity polymerases (such as Phusion). Taq has an error rate of about 1 bp changer per 1 kb, while many high fidelity polymerases have an error rate that is 60 times lower. Taq is cheaper and will give an abundance of product, so use when accuracy is not an issue.
PCR (Phusion Polymerase - 50 μl reaction)
~35 μl ddH2O
10 μl 5x Phusion Buffer
1.5 μl 10 mM dNTP's
1 μl of each primer
1 μl appropriate diluted template DNA
0.5 μl Phusion Polymerase
PCR (Taq Polymerase - 50 μl reaction)
40 μl ddH2O
5 μl 10x Thermopol Buffer
1.5 μl 10 mM dNTP's
1 μl of each primer
1 μl appropriate diluted template DNA
0.5 μl Taq Polymerase
Colony PCR (25 μl reaction)
19 μl H2O
2.5 μl 10x reaction buffer
0.5 μl 10 mM dNTP's
0.5 μl each primer
0.5 μl Taq DNA polymerase (*add last to master mix)
add 2 μl of boiled colony sample
Mix (vortex or flick) tubes well before adding to the reaction. Set up reactions on ice and keep them on ice until placing them on the PCR machine which has been pre-warmed to 94˚C (this is called a "hot start"). Extension times vary according to target size. Taq  requires 1 min per kb of  product, and Phusion requires 2 min per kb of product. Annealing temperature depends on primer Tm values. For typical primers, 55˚C is a good guess. Sample program for amplifying a 1kb target with Taq polymerase:
94˚C, 2:00
94˚C,  :30
55˚C,  :30
72˚C, 1:00
Repeate (steps 2-4) 35 times
72˚C, 2:00
4˚C forever
Standard Agarose Gel
For a standard 1% agarose gel, use 50 mL of 1xTAE and 0.5 grams of agarose (regular agarose, NOT low-melt). Microwave for about 1 minute or until the agarose is completely disolved. Pulsing the microwave may be necessary to avoid boiling over. Add 8 μl ethidium bromide and swirl to mix. BE SURE TO WEAR GLOVES AND TO HANDLE ETHIDIUM BROMIDE CAREFULLY ... IT IS A CARCINOGEN.
Allow the flask to cool so that the glass feels warm/hot not burn/hot. Pour the liquid into the gel bed and let it cool. Insert the appropriate sample comb.
To run the gel:
Add loading dye to each sample. Move the gel into the proper orientation in the gel box and cover with 1x TAE buffer. Load all your samples into the wells (4-5 μl into each well should be sufficient). ADD DNA LADDER AS REFERENCE. Put lid on, and set power supply to between 130 and 170 volts. It will take between 15 and 30 minutes to run, depending on the desired resolution.
Visualize gel on imager, print of results and paste into your notebook.
Restriction Digest of insert and/or vector (50 μl reaction)
~14 μl H2O
5 μl 10x NEB buffer (check online for double enzyme digest chart) [1]
0.5 μl 100x BSA
30 μl DNA sample
1-2 μl of each restriction enzyme
Always mix reagents well before adding enzyme as the final reagent
(incubate this reaction at 37˚C for at least 2.5 hours)
Note: If a non-directional ligation will be performed (single restriction site), then you must dephosphorylate the vector by adding 0.5 μl of calf intestinal phosphatase (CIP) to the restriction digest for the last hour prior to low-melt gel purification.
Low-melt gel purification
Low-melt agarose gel electrophoresis is similar to regular gel, with some exceptions. The low-melt agarose dissolves faster, and boles over more easily during microwaving, so microwave in brief pulses. Add the normal amount of ethidium bromide (~160 ng/ml) to the molten agarose before casting. Cast your gel in the fridge to accelerate solidification. Use a large-tooth comb to form the wells that will accomidate ~40-50 μl of sample. Use TAE buffer to make and run the gel. Run gel at 90 volts. It will take about 45 - 60 minutes to run.
When the gel has run, use a portable UV lamp and carefully remove the DNA bands from the gel with a razor blade. Place the DNA samples in clearly labeled 1.5 ml tubes. Also, cut out a slice of gel that has no DNA for use as a vector-only control sample (see ligation section).
Ligation (15 μl reaction)
6.5 μl H2O
1.5 μl 10x ligase buffer (includes ATP)
1 μl T4 DNA ligase
3 μl vector
3 μl insert
(incubate this reaction at room temperature for at least 30 minutes)
Compatible sticky ends will bring vector and insert together only transiently. DNA ligase forms phosphodiester bonds between vector and insert. Set up two reactions: vector + insert and the vector-only control. The gel slices should be heated to 65˚C to melt them. Once melted, the samples will remain melted for a few minutes.
Transformation
Typically, E. coli strain DH5α is made chemically competent. Thaw DH5α chemically competent cells on ice. Meanwhile, melt the ligations for a few minutes at 65˚C. Also, be sure you have a 42˚C heating block ready, as well as LB-agar plates with the proper antibiotic.
When the DH5α is thawed, quickly add 10 μl of the ligation mix to ~25 μl of competent cells. Flick or vortex briefly and put them back in the ice for 5 - 15 minute (it is important that the DH5α cells be kept as cold as possible during this process).
Heat shock at 42˚C for 60 seconds. Immediately place the tubes back on ice.
Add 500 μl of plain LB to the reactions and incubate at 37˚C for 30-60 minutes (30 minutes if the selection is ampicillin, 60 minutes for everything else).
Plate 100 μl of cells and incubate at 37˚C overnight.
Sequencing (submitting to BYU DNA sequencing center)
5 μl H2O
5 μl template
1.5 μl primer (ONLY ONE PRIMER)
Submit sample online and label tubes with sample number. Take to sequencing center and place on "cycle sequencing ready" shelf.
Freezing strains in -80˚C freezer
200 μl DMSO 1.3 mL overnight culture Place in sterile 1.5 ml tube. Vortex and quickly freeze at -80˚C.
Electroporation
Cell preparation:
Fill a 1.5 mL tube with 500 μl ddH2O
Add approximately 3 mg cells (3 generous swipes across a patch ... don't include any agar since it inhibits transformation)
Centrifuge at 14000 rmp for 1 minute. Remove supernatant and suspend cells in 500 μl ddH2O.
Centrifuge again at 14000 for 1 minute. Remove supernatant and suspend cells in 40 μl ddH2O. Keep on ice.
Cells are now ready for electroporation.
Electroporation:
Add 5 - 50 ng of plasmid DNA and mix gently to ensure homogenous suspension.
Transfer DNA/cell suspensions to electroporation cuvettes and keep on ice.
After pulsing, immediately add ice-cold LB to each cuvette.
Transfer to culture tubes and incubate for 1 hour at 37˚C.
Spread on selective plates and incubate overnight at 37˚C.
Mutagenic PCR (from paper by RC Cadwell and GF Joyce in Genome Research, 1994)
Prepare a 10x mutagenix PCR buffer containing 70 nM MgCl2, 500 mM KCl, 100 mM tris (pH 8.3 at 25˚C), and 0.1% (wt/vol) gelatin.
Prepare a 10x dNTP mix containing 2 mM dGTP, 2 mM dATP, 10 mM dCTP, and 10 mM TTP.
Prepare a solution of 5 mM MnCl2. DO NOT combine with the 10x PCR buffer, which would result in formation of a precipitate that disrupts PCR amplification.
Combine 10 μl of 10x mutagenic PCR buffer, 10 μl of 10x dNTP mix, 30 pmoles of each primer, 20 fmoles of input DNA, and an amount of H2O that brings the total volume to 88 μl. Mix well.
Add 10 μl of 5 mM MnCl2. Mix well and confirm that a precipitate has not formed.
Add 5 units (2 μl) of Taq polymerase, bringing the final volume to 100 μl. Mix gently. Cover with mineral oil or a wax bead if desired.
Incubate for 30 cycles of 94˚C for 1 minute, 45˚C for 1 minute, and 72˚C for 1 minute. Do not employ a "hot start" procedure or a prolonged extension time at the end of the last cycle.
Purify the reaction products and run a small portion on an agarose gel.
"Plate Reader Experiment" Protocol
Start overnight cultures on the day before the experiment. Use appropriate antibiotic in the media, and label tubes well. Place on shaker at 37˚C overnight, on shaker.
Measure 1/10 optical density (OD) of each overnight culture by diluting 100 μl of saturated overnight culture into 900 μl of plain LB. Place the resulting mL of diluted culture into a cuvette and measure 1/10 OD. Multiple resulting measurement by 10 for actual OD of culture.
Dilute each culture to an OD of 0.025 into overnight culture tubes. An example calculation:
1/10 OD = 0.62
Actual OD = 6.2
6.2 / 0.025 = 248x (The overnight culture is 248 times more concentrated than our desired  dilution)
2000 μl / 248 = 8.06 μl
Thus we must add ~8 μl to 2 mL of plain LB to reach our desired OD of 0.025.
Perform this calculation for each culture, and make the dilutions.
Incubate at 37˚C for 1.5 hours, on shaker.
Add H2O2 (hydrogen peroxide) to the 2 mL dilutions so that desired concentrations of H2O2 are in the desired range (similar concentration calculations as in step 3).
Incubate at 37˚C for 3 hours, on shaker.
Add 100 μl of each H2O2-induced culture, in order, into the wells of a black plate, and identically into a clear plate. The clear plate is for measuring the OD of each sample, and the black plate for measuring fluorescence.
Use iGEM protocols in lab machine. For OD, use 600 nm, and for fluorescence, use excitation 485 nm and emission 528 nm. Set fluorescence reader sensitivity to 45.
Export results to spreadsheet. Prepare data by dividing fluorescence readings by corresponding OD measurement. Graph or analyze appropriately.
Beta-Galactosidase Assay
Grow cells overnight in minimal media.
Inoculate 3 - 1.5 mL of assay media with 20 uL fresh overnight culture.  Grow at 30˚C, 35˚C and 37˚C until culture reaches OD600 of 0.2 to 0.7.
Chill culture on ice for 20 minutes.  Record cell density by measure OD600.
Add 500 uL cells to 550 uL ZS-buffer.
Add 100 uL chloroform to the tubes.  Incubate for 2 minutes at 30˚C.
Add 200 uL 4mg/mL o-NPG to start the reaction. (o-NPG made fresh daily 56mg o-NPG in 14 mL sterile water).  Note the time of addition precisely.
Incubate the reaction at 37˚C until sufficient yellow color has developed.
Stop the reaction by addition of 500 uL 1M Sodium Carbonate.  Incubate for 5 minutes at 30˚C.  Note the time of addition precisely.
Centrifuge to precipitate cell debris and transfer 1 mL of supernatant to a cuvette.
Record OD405 and OD550 for each tube.  OD550 is for cell debris control and should be low.
Calculate Miller Units with the equation: (1000)*[OD405-1.75*OD550]/[t*v*OD600] where t is reaction time in minutes and v is volume of culture used in mL.
ZS Buffer
16.1 g Na2HPO4.7H2O
5.5 g NaH2PO4.H2O
10.0 mL 1 M KCL
1.0 mL 1 M MgSO4
2.7 mL beta-Mercaptoethanol
985 mL water
100 mL 0.1% SDS
Mix all ingredients together except the SDS first, then add in the SDS and gently mix to prevent it from bubbling.
Retrieved from " http://2011.igem.org/Team:BYU_Provo/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
