Team:Nanjing-China/Parts
Main Page Research Result Modeling Biosafety Human Practice Team NoteBook
Parts
Recombination
Adsorption
Adhesion
Application
Achievements
Parts
As we have mentioned above, we need to construct 16 kits, which proved to be successful. The following picture shows some results of electrophoresis.(Figure 1)
Fig 1. Some results of electrophoresis during part construction.
1.Parts that we used
During the experiment, we utilized a constitutive promoter Pveg, a biobrick part coding for eGFP and two terminators. We not only benefited a lot from these parts, but identified the characteristics of them as well.
Name
Type
Description
Length
BBa_K193202
Coding
Coding for eGFP
797bp
BBa_K143053
Promoter
A constitutive promoter in B. subtilis
117bp
BBa_B0010
Terminator
Terminator
80bp
BBa_B0012
Terminator
Terminator
41bp
2.Parts that we created
We also contributed 6 parts to the part registry. There are two promoters ( PtasA and PcotC), a kind of B. subtilis biofilm protein ( Tas A ) and three kinds of metallo-proteins ( PbrR, GolB and SUP ) that can adsorb lead, gold and uranyl respectively.
Name
Type
Description
Length
BBa_K1701002
Promoter
Promoter of cotC gene in the genome of B. subtilis.
201bp
BBa_K1701003
Promoter
Promoter of tasA gene in the genome of B. subtilis.
63bp
BBa_K1701006
Coding
Endospore coat protein CotC.
101bp
BBa_K1701000
Coding
Gold-specific binding protein.
195bp
BBa_K1701004
Coding
Uranyl-specific binding protein.
267bp
Homologous recombination
After we constructed the 16 kits in E.coli, we inserted them into the shuttle plasmid pDG1730 which can transfer from E.coli to B. subtilis. We insert our fusion genes between two sequences of the amylase genes on the plasmid. By homologous recombination, our fusion genes will replace the amylase gene on the genome of B. subtilis. In order to test and verify whether homologous recombination is successful, we built the following three steps. Antibiotic selection, bacterial PCR analysis and amylase activity analysis. After a series of test and verification experiments, we have successfully inserted the 16 kits into the genome of B. subtilis. The following figure shows the result of the amylase activity analysis. The picture on the right is the control group while the pictures on the left are the experiment groups. When we applied iodine on the plate with starch, there would be hydrolysis cycles on the plate because the α-amylase gene wasn’t replaced. However, there wouldn’t be hydrolysis cycles on the plate if homologous recombination was successful.(Figure 2)
Fig 2. Result of homologous recombination
B. subtilis cells carrying fusion proteins (TasA-EGFP and CotC-EGFP) were fixed for visualization under a fluorescence microscope. The same batch of B. subtilis cells without fusion proteins was used as s control. Strong fluorescence was observed when the B. subtilis cells expressed the fusion proteins (TasA-EGFP and CotC-EGFP), whereas no fluorescence was detected for the control group. Taken together, these results confirmed the fusion proteins can express well on the surface of B. subtilis. The following pictures show the results of fluorescence measurements.(Figure 3)
Fig 3. Fluorescence measurement of B. subtilis cells containing the fusion proteins. a) Fluorescence intensity of the fusion proteins Pveg-TasA-EGFP and Ptas-TasA-EGFP. b) Fluorescence intensity of the fusion proteins Pveg-CotC-EGFP and Pcot-CotC-EGFP. c) A visible photograph of a) and b).
Bio-adsorption
As shown in Fig. 4a, B. subtilis bacteria with the TasA-dependent surface displayed PbrR with promoter Pveg were able to adsorb lead ions with a capacity of about 81.2 μmol g-1cells, which is 3-fold higher than un-displayed samples. While Bacteria with promoter Ptas were able to adsorb lead ions with a capacity of about 69.5 μmol g-1cells, which is 2-fold higher than un-displayed samples.
Fig 4. a) Adsorption of lead ions by B. subtilis TasA-dependent surface-displayed PbrR protein.
As shown in Fig. 4b, B. subtilis bacteria with the TasA-dependent surface displayed GolB with promoter Pveg were able to adsorb gold ions with a capacity of about 5.2 μmol g-1cells, which is 26-fold higher than un-displayed samples. While bacteria with promoter Ptas were able to adsorb gold ions with a capacity of about 3.5 μmol g-1cells, which is 18-fold higher than un-displayed samples.
Fig 4. b) Adsorption of gold ions by B. subtilis TasA-dependent surface-displayed GolB protein.
As shown in Fig. 4c, B. subtilis bacteria with the TasA-dependent surface displayed SUP with promoter Pveg were able to adsorb uranyl ions with a capacity of about 8.7 μmol g-1cells, which is 4-fold higher than un-displayed samples. While bacteria with promoter Ptas were able to adsorb uranyl ions with a capacity of about 6.8 μmol g-1cells, which is 3-fold higher than un-displayed samples.
Fig 4. c) Adsorption of uranyl ions by B. subtilis TasA-dependent surface-displayed SUP protein.
As shown in Fig. 4d, B. subtilis endospores with the CotC-dependent surface displayed PbrR with promoter Pveg were able to adsorb lead ions with a capacity of about 227.6 μmol g-1cells, which is 2-fold higher than un-displayed samples. While bacteria with promoter Pcot were able to adsorb lead ions with a capacity of about 225.1 μmol g-1cells, which is 2-fold higher than un-displayed samples.
Fig 4. d) Adsorption of lead ions by B. subtilis endospore CotC dependent surface-displayed PbrR protein.
As shown in Fig. 4e, B. subtilis endospores with the CotC-dependent surface displayed GolB with promoter Pveg were able to adsorb gold ions with a capacity of about 4.4 μmol g-1cells, which is 22-fold higher than un-displayed samples. While bacteria with promoter Pcot were able to adsorb gold ions with a capacity of about 3.85 μmol g-1cells, which is 19-fold higher than un-displayed samples.
Fig 4. e) Adsorption of gold ions by B. subtilis endospore CotC dependent surface-displayed GolB protein.
As shown in Fig. 4f, B. subtilis endospores with the CotC-dependent surface displayed SUP with promoter Pveg were able to adsorb uranyl ions with a capacity of about 16.5 μmol g-1cells, which is 2-fold higher than un-displayed samples. While bacteria with promoter Pcot were able to adsorb uranyl ions with a capacity of about 16.3 μmol g-1cells, which is 2-fold higher than un-displayed samples.
Fig 4. f) Adsorption of uranyl ions by B. subtilis endospore CotC dependent surface-displayed SUP protein.
In conclusion, the engineered B. subtilis as well as the endospores worked well in heavy metal adsorption. Adsorption of gold ions is the most efficient among adsorption of the three kinds of heavy metal. Besides, it is proved by us that Pveg is a more efficient promoter than Ptas and Pcot, so maybe we can utilize promoter Pveg when engineering B. sublitis.
Bacteria adhesion
As we have mentioned above, we fused the biofilm protein gene tasA with the metalloprotein genes and inserted them into the genome of B. subtilis. The engineered B. subtilis was co-cultured with the stuffing materials afterwards and we have found that biofilm formed on the stuffing materials, which is proved in the pictures below. It proves that bacterial adhesion is highly efficient on the second generation plastic pellet.
Fig 5. Bacteria adhesion on the materials that we used.
Fig 6. Bacteria adhesion on the materials that we used.
Application
Fig 7. Our devices
We placed the bacteria adhesion materials into devices designed by us, which is shown in the picture. Water with high concentration of heavy metal will enter the device and go through many cycles of adsorption and purification. Then the water flowing out is of low concentration of heavy metals. Besides, as we have mentioned before, our project also aimed at heavy metal recovery. Therefore, we tested the recovery rate of our device. We repeated for three times for each of the three kinds of heavy metal. The results are shown below.
As is shown, the ratio of recovery reached 97% with a concentration of 50μM/L lead ions while we recovered 90% of gold ions when the concentration of it was 10μM/L. As for uranyl, the ratio of recovery reached 76% with a concentration of 13nM/L uranyl ions. Although the recovery rate is relatively high, we are still working on it. We are still testing the longevity of the bacteria and exploring whether our engineered bacteria can function in the sediments of rivers and lakes because the concentration of heavy metal in sediments is the highest in the aquatic environment. We do hope that our device can pragmatically contribute to the development of mechanical engineering and environmental protection some day.
Fig 8. Ratio of recovery with a concentration of 50μM/L lead ions.
Fig 9. Ratio of recovery with a concentration of 10μM/L gold ions.
Fig 10. Ratio of recovery with a concentration of 13nM/L uranyl ions.
Achievements
1.Requirements for a Bronze Medal:
Register the team, have a great summer, and plan to have fun at the Giant Jamboree.
✓
 
Successfully complete and submit this iGEM 2015 Judging form.
✓
 
Create and share a Description of the team's project using the iGEM wiki and the team's parts using the Registry of Standard Biological Parts.
✓
 
Plan to present a Poster and Talk at the iGEM Jamboree.
✓
 
The description of each project must clearly attribute work done by the students and distinguish it from work done by others, including host labs, advisors, instructors, sponsors, professional website designers, artists, and commercial services.
https://2015.igem.org/Team:Nanjing-China/Attributions/
✓
 
Document at least one new standard BioBrick Part or Device used in your project/central to your project and submit this part to the iGEM Registry (submissions must adhere to the iGEM Registry guidelines). Please note you must submit this new part to the iGEM Parts Registry. Please see the Registry help page on adding new parts. A new application and/or outstanding documentation (quantitative data showing the Part's/ Device's function) of a previously existing BioBrick part also counts. Please see the Registry help page on how to document your contributions. To fulfill this criteria, you will also need to submit the part with its original part name to the Registry, following the submission guidelines.
Part numbers:
BBa_K1701006
✓
2.Additional Requirements for a Silver Medal:
Experimentally validate that at least one new BioBrick Part or Device of your own design and construction works as expected.
Part Numbers:
BBa_K1701002
BBa_K1701003
BBa_K1701000
BBa_K1701004
✓
 
Document the characterization of this part in the Main Page section of that Part's/Device's Registry entry.
✓
 
Submit this new part to the iGEM Parts Registry (submissions must adhere to the iGEM Registry guidelines)
Part numbers:
BBa_K1701002
BBa_K1701003
BBa_K1701000
BBa_K1701004
✓
 
iGEM projects involve important questions beyond the bench, for example relating to (but not limited to) ethics, sustainability, social justice, safety, security, or intellectual property rights. Articulate at least one question encountered by your team, and describe how your team considered the(se) question(s) within your project. Include attributions to all experts and stakeholders consulted.
https://2015.igem.org/Team:Nanjing-China/Practices/
✓
3.Additional Requirements for a Gold Medal:
Choose one of these two options: (1) Expand on your silver medal Human Practices activity by demonstrating how you have integrated the investigated issues into the design and/or execution of your project. OR (2) Demonstrate an innovative Human Practices activity that relates to your project (this typically involves educational, public engagement, and/or public perception activities; see the Human Practices Hub for information and examples of innovative activities from previous teams).
https://2015.igem.org/Team:Nanjing-China/Practices/
✓
 
Help any registered iGEM team from a high-school, different track, another university, or institution in a significant way by, for example, mentoring a new team, characterizing a part, debugging a construct, modeling/simulating their system or helping validate a software/hardware solution to a synbio problem.
https://2015.igem.org/Team:Nanjing-China/Collaborations/
✓
