Team:SHSBNU China/Protocol
SHSBNU-China iGEM 2017 Home Page
Project | Model | HP | Safety | Parts | Team | Achievement
Project
Project | Experiment | Improve | Demonstrate | Applied Design | Protocol | InterLab
Procotol
I.    Polymerase Chain Reaction
PCR Reaction System
※ Addition order of all components: ddH2O → Primers → Template → Q5/Taq Mix
※ The reaction system should be gently mixed, and can be gently centrifuged if necessary.
PCR Reaction Conditions (Q5® High-Fidelity 2× Master Mix)
PCR Reaction Conditions (2×Taq PCR Star Mix with Loading Dye)
※ Q5® High-Fidelity 2× Master Mix was used for the amplification of gene fragments
※ 2×Taq PCR Star Mix was used for colony PCR and verification PCR
II.   Agarose Gel DNA Extraction
1. Agarose Gel Electrophoresis
（1） Fasten the gel tray into the holder on a level surface and insert the comb into the gel tray at ~1 cm from one end. Make sure that the bottom edge of the comb maintains a 0.5~1 mm gap from the bottom of the gel tray.
（2） For a 1% 50 mL agarose gel, weigh 0.5 g agarose into a conical flask, add 50 mL of TAE buffer. Microwave the mixture for a few minutes until all the agarose is dissolved completely.
（3） Cool down the solution to a temperature that can be handled with gloved hands (60-70 °C). Add an appropriate amount of SYBR® safe DNA Gel Stain (10000×) to a final concentration of 1×. Mix the solution gently and pour it into the gel tray. Remove any air bubbles using a pipette tip.
（4） After 30~60 min, the gel will solidify at room temperature. Remove the comb carefully so that cleanly separated sample wells are formed.
（5） Release the gel tray from the holder, put it into the electrophoresis chamber, and add TAE buffer until the level reaches 2-3 mm above the gel surface.
（6） Take aliquots of the DNA samples and mix with loading dye. Load the samples into the wells.
※ DNA Marker should be loaded into a separate well as reference.
※ Taq Mix includes a loading dye; PCR products generated with Taq Mix can be loaded into the wells directly.
（7） Put the lid onto the electrophoresis chamber and connect it to the power supply, making sure to put it in the right direction since the DNA runs from the negative electrode (black) towards the positive electrode (red).
（8） Run the gel at 120V for about 15~30 min, stop the electrophoresis when the loading gel dyes move to about 2/3 of the gel.
（9） Move the gel from the gel tray to a UV table to visualize the gel bands and take a picture of the gel.
※ Use protective glasses.
※ If sufficient separation was not achieved, put the gel back into the buffer chamber and run it for a longer time.
2. DNA Gel Extraction
（1） Carefully excise the target DNA fragment using a clean scalpel and put it into a clean 1.5-mL microfuge tube.
（2） Weigh the gel slice to determine the volume of gel dissolution buffer to be added. For each 0.1 g of gel, add 100 μL gel-dissolution buffer. Incubate the mixture at 50 °C for a few minutes until the gel has completely dissolved. Mix by shaking or inverting the tube every 2~3 min.
（3） Put the adsorption column into a collection tube, add 500 μL of equilibration buffer into it, centrifuge in the tabletop centrifuge at 12 000 rpm for 1 min, and discard the waste liquid.
（4） Cool down the gel solution to room temperature and transfer it to the adsorption column, incubate at room temperature for 2 min, centrifuge at 12 000 rpm for 30~60 s, and discard the waste liquid.
※ For gel solution volumes greater than the maximum volume of the adsorption column, load the same column and centrifuge multiple times.
（5）Add 600 μL of wash buffer diluted with absolute ethanol , centrifuge at 12 000 rpm for 30~60 s, discard the waste liquid; do the procedure a total of 2 times.
（6） Reusing the collection tube, centrifuge the empty column at 12 000 rpm for 2 min to dry the column matrix.
（7） Place the column into a clean 1.5-mL microfuge tube, add 50~100 μL ddH2O (sterilized，pre-heated to 50 °C ) directly into the center of the column, incubate for 2 min at 50 °C, and centrifuge at 12 000 rpm for 1 min to elute the DNA.
（8） The concentration and quality of the DNA were determined using a NanoDrop2000 micro-spectrophotometer.
III.  3.    Gibson Assembly
Gibson Assembly Reaction System
※ Addition order of all components: Gibson Assembly Mix → ddH2O → Backbone → Fragment
※ The reaction system should be gently mixed, and can be gently centrifuged if necessary.
Gibson Assembly Reaction Conditions
IV.   4.    Transformation
（1） Thaw a tube with 50 μL of competent cells on ice for 5~15 min.
（2） Add the target DNA (ligation reaction mixture, plasmids or controls) to the competent cells, and incubate for 30 min on ice.
（3） Heat shock for 60 s at 42 °C , then incubate for 5 min on ice immediately. Do not shake the tube at this step.
（4） Add 150 μL of LB medium without antibiotics.
（5） Incubate for 60 min at 37 °C .
（6） Resuspend the cells and spread the suspension on an agar plate containing the appropriate antibiotic.
（7）Incubate for 16-18 h at 37 °C.
V.     Flow Cytometry
（1） Transfer bacteria harboring parts/circuits of interest from plates into LB medium, incubate for 8−12 h at 37°C under vigorous shaking at 1000 rpm.
（2） Dilute 10 μL of each culture sequentially with 130 μL of fresh M9+Glycerol medium twice, yielding a total dilution of 1:196.
（3） Incubate the diluted cultures for 3 h (1000 rpm, 37 °C) to reach the exponential phase, and dilute 700-fold with fresh M9+Glycerol medium (supplemented with different concentrations of the inducer: sodium tetrathionate or sodium thiosulfate (both from Sigma-Aldrich).
※ The dilution process was as follows: 10 μL of cell culture is added to 130 μL of M9 medium, which is followed by diluting 3 μL of the resulting suspension with 147 μL of fresh medium.
（4） Continue the cultivation (1000 rpm, 37 °C) to the specific time points, then transfer a 15 μL aliquot of each culture to a new plate containing 185 μL of PBS (2 mg/mL kanamycin pre-added to terminate protein expression).
※ M9 + glycerol Medium：1× M9 salts, 0.4% v/v glycerol, 0.2% casamino acids, 2 mM MgSO4, and 100 μM CaCl2
※ Flat-bottom 96-well plates and sealing film were used throughout the experiment.
VI.    Chemically Transformation of Bacillus Subtilis
This protocol is provided and optimized by Zhao Xueming Lab, Tianjing University
Step1, Cultivate Bacillus subtilis in 5ml GM1 medium, 200rpm, 37℃, overnight culture.
Step2, propagate 500μl bacteria culture (from step1) into new 5ml GM1 medium, 200rpm, 37℃, 4.5h culture. Harvest cell from the end of log-phase growth.
Step3, propagate 750μl bacteria culture (from step2) into new 5ml GM2 medium, 200rpm, 37℃, 1.5h culture. Harvest competent cell from this culture.
Step4, Use 500μl bacteria culture (from step3), add 10μg (at most) target DNA, 37℃, stationary culture for 1h, then another 1h culture, 200rpm, 37℃
Reference
Zhang, H. M., Chen, S., Shi, H., Ji, W., Zong, Y., & Ouyang, Q. (2016). Measurements of gene expression at steady state improve the predictability of part assembly.Acs Synthetic Biology, 5(3), 269.
iGEM 2017
Made by Azir.
The Second High School Attached Beijing Normal University International Division
