<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Utah_State_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Utah_State/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Utah_State/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Utah_State/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Utah State/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>Demo Menu - PSDGraphics.com</TITLE><TITLE>USU iGEM 2012</TITLE></P><DIV style="width: 1210px; height: 5000 px; margin-left: auto; margin-right: auto;"><DIV class="wrapOverall"><DIV class="logo"><DIV class="navigation"><UL id="navigation" class="nav-main"><LI><A href="https://2012.igem.org/Team:Utah_State">Home</A></LI><LI class="list"><A href="https://2012.igem.org/Team:Utah_State/Team">Team</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Team#AboutUSU">About Utah State</A></LI></UL><LI class="list"><A href="https://2012.igem.org/Team:Utah_State/Project">Project</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Project#Background">Background</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Project#Design">Design</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Project#Experimental">Experimental</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Project#Applications">Applications</A></LI><LI class="list"><A href="https://2012.igem.org/Team:Utah_State/Results">Results</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Results#Cloning">Cloning</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Results#GFPExpression">GFP Expression</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Results#ProteinExpression">Protein Expression</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Results#SilkProduction">Silk Production</A></LI><LI class="list"><A href="https://2012.igem.org/Team:Utah_State/Notebook">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Notebook#SpiderSilkProtocols">Spider Silk</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Notebook#MolecularBiologyProtocols">Molecular Biology</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Parts">BioBricks</A></LI><LI class="list"><A href="https://2012.igem.org/Team:Utah_State/Attributions">Attributions</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Attributions#Collaboration">Collaboration</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Attributions#HighSchoolOutreach">High School Outreach</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Attributions#Sponsors">Sponsors</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Safety#Safety">General Safety</A></LI><LI><A href="https://2012.igem.org/Team:Utah_State/Safety#HumanPractices">Human Practices</A></LI></DIV></DIV><A name="Background"><H1 style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #000; text-decoration: none;" class="table-main">Background</H1></A><P class="textProfile">
Spider silk from most orb weavers has been noted for its extraordinary properties since ancient times. The unusual mechanical properties are the key features attracting researches to spider’s silks. High protein sequence conservation has been maintained over the 125 million years since the various species diverged from each other (Lewis R.V., 2006).  Unlike many man-made materials, it is constantly uniform in its basic protein structure and yet extremely complex and repetitive.  These structures lead to the mechanical properties of the total material.  There are several types of silk each with different properties which depend upon their composition of amino acids, mainly alanine, proline, and glycine, which in turn arises from specific genetic coding. This trend is seen across a broad spectrum of known spider species and is robustly conserved with only slight rearrangements of the amino acid sequences (Hayashi et al., 1999). 


These amino acid structures lead to specific secondary structures which interact with each other and give rise to the protein's tertiary structure.  The varying types of silk are then formed by these varying secondary elements and their composition within the silk.  Flagelliform silk is mostly composed of ß-spirals and helices.  Both of which replicate spring like properties and thus give the silk high elasticity and elongation.  Major ampullate silk, in comparison, is composed of mostly ß-spirals and ß-sheets.  The spirals exhibit elastic properties, but the sheets are crystalline in structure giving strong rigid properties to the fiber. When the mechanical properties of these varying types of fibers are observed, it becomes clear how the amino acid sequences and arrangements affect the fibers (Hayashi et al., 1999).

</P><P style="font-family: Helvetica; font-weight: normal; font-size: 12px;" align="center">Beta-Sheet, Beta-Spiral and Beta-Helix Protein Formations</P><P class="textProfile">
Spiders have evolved the ability to produce as many as six different silk fibers that have differing tensile strengths and elasticities (Lewis R.V., 2006).   Each of these silks has a different form and thus a different function. When being extruded in native form by spiders, the percentages of silks can be varied for any particular situation.
</P><P style="font-family: Helvetica; font-weight: normal; font-size: 12px;" align="center">Spider silk glands and silk uses (Lewis R.V., 2006)</P><P class="textProfile">
We have chosen dragline silk from the major ampullate gland in order to produce it in E. coli because of the unique combination of tensile strength and elasticity that it exhibits. Dragline silk is a unique biomaterial, it will absorb more energy prior to breaking than nearly any commonly used material. It is nearly as strong as several of the current synthetic fibers, but can outperform them in many applications requiring total energy absorption (Lewis R.V., 2006).  


Another unique feature of major ampullate silks is that they represent the only known example of supercontraction when exposed to water. Depending on the spider species and other factors, these silks will contract to 50% or less of their original length in water (Lewis R.V., 2006).   Also, it has been shown that natural spider silks do not induce an immune response whether implanted subcutaneously or intramuscularly in rats, mice, or pigs (Lewis R.V., 2006).   These varying properties make spider silk a very attractive material for an extremely wide range of applications.


Several groups have used synthetic genes based on spider silk to produce proteins in bacteria (Lewis R.V., 2006). We are basing our synthetic gene on the sequence of the Major Ampullate Silk Protein 2 (MaSp2) of the <I>Argiope aurantia</I> spider. By simple genetic manipulation synthetic constructs of spider silks can be formed (Brooks et al., 2008). However, unlike native silk these constructs can be altered to shorter or longer lengths, along with increased repetition or blending of several types of silks into one.  Furthermore, the nature of bacteria makes it easy to manipulate and control when developing and when producing (Brooks et al., 2008).
</P><P style="font-family: Helvetica; font-weight: normal; font-size: 12px;" align="center">
Synthetic silk sequences derived from Argiope spider. Protein molecular weights are shown without the presence of the N-terminal and C-terminal 6 histidine tags (Brooks et al., 2008).</P><P class="textProfile">
Nevertheless, due to the size and repetitiveness of spider silk, the internal functions of the bacteria can become overloaded or stalled. This is mainly due to the vast number of amino acids that are required. A solution to this problem is to simply engineer the bacteria with a tRNA producing gene in order to elevate the quantity of tRNAs available for protein synthesis (Xia et al., 2010). This insures that the entire protein is produced instead of laddering. Without this gene, synthesis is incomplete and can ultimately lead to reduced yield or silk with undesirable properties. This extra resource allows for a more desirable and economic approach to the production of the selected spider silk.


In conclusion, it has been determined that spider silk is a glorious biomaterial with endless possibilities due to its extravagant properties. It can almost always outpace any current synthetic material and many organic ones as well. Its repetitive nature allows for easy manipulation either at the secondary structures or in the percentages of silks present in the final product. By using genetically engineered E. coli which is well understood, versatile silk can be produced and altered to desired properties by altering the coding sequences.  Also, unlike spiders, goats, or any other animals, bacteria require less work and whole industries already exist to support the use of them. The repetitive nature of the spider silk can often stall bacteria due to the high volume of amino acids that are required for synthesis. This can be solved though by further altering the code of the bacteria to produce tRNAs as needed. After expression and processing of the silk proteins have occurred, the possibilities and applications of them are extraordinary.
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Overview of spider silk properties and origin
</P><P align="center" class="textProfile">
Properties of spider silk compared with other similar materials (Foo et al., 2002).
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Orb-Weaver Spiders
</P><UL><LI>Most common group of builders of spiral wheel-shaped webs.
    </LI><LI>Include over 10,000 species and make up about 25% of spider diversity.
    </LI><LI>Can produce up to six high-performance silk fibers and a specialized glue.
    </LI><LI>Major ampullate dragline silk, one of the six solid fibers used as safety line and web frame, is a molecular composite of two main proteins: MaSp1 and MaSp2.
    </LI><P style="font-family: Helvetica; font-weight: normal; font-size: 12px; margin-right: 150px;" align="right">Orb-weaver spider's web</P></UL><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main"><I>Argiope aurantia</I></P><UL><LI>Commonly known as the black and yellow garden spider, writing spider, corn spider, or banana spider.
    </LI><LI>Have distinctive yellow and black markings on their abdomens and a mostly white cephalothorax.
    </LI><LI>Common to the contiguous United States, Hawaii, southern Canada, Mexico, and Central America.
    </LI><P style="font-family: Helvetica; font-weight: normal; font-size: 12px; margin-right: 180px;" align="right"><I>Argiope aurantia</I></P></UL><A name="Design"><H1 style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #000; text-decoration: none;" class="table-main">Design of Spider Silk Sequences and Silk Overexpression Strategies in <I>E. coli</I>.</H1></A><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Introduction
</P><P class="textProfile">
Attempts at overproduction of spider silk in <I>E. coli</I>have been hampered by the highly repetitive nature of the spider silk protein, and the small number of amino acids that it is comprised of. The spider silk sequences designed for this project are derived from the 2E silk construct from Brooks et al. (2008). The original 2E sequence used in this paper is based on the native sequence of the MaSp2 silk gene in <I>Argiope aurantia</I>. The 2E silk subunit contains codons for only six amino acids in the following proportions: glycine (44.1%), glutamine (17.6%), proline (14.7%), alanine (11.8%), serine (7.4%), and tyrosine (4.4%). The native <I>Argiope aurantia</I> codon usage profile is significantly different from <I>E. coli</I> and the GC content of the DNA molecule itself (75%) is significantly different from the average GC content of native <I>E. coli</I> genes (51%). These features can make the native form of the gene difficult and slow to produce for the ribosomes in the cell as the necessary tRNA molecules are present in lower concentrations in <I>E. coli</I> and the genetic machinery of <I>E. coli</I> is not adapted to opening DNA of high GC content when making mRNA, and thus lower amounts of spider silk mRNA are produced. 


In this study we have altered the DNA sequence of the spider silk gene to reduce its GC content to a level more closely matching a wild type <I>E. coli</I>, and altered its codon usage profile to use a limited set of tRNA molecules, which we will increase the intracellular concentration through gene addition to remove the limiting factor on ribosomal production of silk protein. We will also supplement the cell with additional glycine amino acid by the introduction of additional copies of serine hydroxymethyltransferase genes, which produces glycine from serine. These sequence alterations and gene supplementations should act to increase the amount of spider silk that can be produced in an E. coli cell.
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Design of Spider Silk Sequences
</P><P class="textProfile">
The spider silk sequences designed for this project are derived from the 2E silk subunit from Brooks et al. (2008), which is based on the native sequence of the MaSp2 silk gene in Argiope aurantia. The 2E DNA sequence was entered into the Gene Designer software from DNA 2.0 and translated. Codon optimization was performed, not using the Gene Designer software algorithm, but by hand.


First, the list of tRNA genes in E. coli K-12 (which is very closely related to the lab strains of DH5a and BL21 that were used in this study) was obtained from the <A href="http://gtrnadb.ucsc.edu/Esch_coli_K12/eschColi_K12-tRNAs.fa">Genomic tRNA Database</A>, and the list of codons was generated that were recognized by these tRNA anti-codons for each of the six amino acids present in the 2E construct. New codons were selected so that they contained the minimum possible GC content for that amino acid and were included on the list of recognized anti-codons by known E. coli tRNA genes. 


The “F” construct was designed using the fewest possible codons, only one for each amino acid while the “B” construct utilizes two codons for glycine, proline, and serine as these were the only amino acids that had multiple codons with low GC%. Alanine also possessed two codons with low GC% that had known E. coli tRNA genes, but inclusion of both codons would have created PstI restriction sites in the silk subunit which conflicts with <A href="http://partsregistry.org/Assembly_standard_23">Assembly Standard #23</A>, so it was left with a single codon in the “B” construct. The two codons for the glycine, proline, and serine amino acids in the “B” construct were distributed roughly evenly across the gene, while avoiding the creation of any restriction sites in the sequence that would interfere with cloning procedures.
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Design of tRNA Promoter and Terminator
</P><P class="textProfile">
All of the promoters for the tRNA genes used in the overexpression constructs are identical. The sequence used is based on the ZH14 sequence from Bauer et al. (1993). This sequence was obtained in that study from mutations made to the leuV tRNA promoter from E. coli, and has approximately normal growth-rate dependent regulation but 12.0x normal promoter strength in rich medium.


The tRNA terminator seqeuence used for each tRNA gene in the overexpression constructs is based around the downstream sequence of the E. coli Ala(GCC) tRNA. It contains the region of 3-6 T’s that is characteristic of tRNA terminators and an additional 10 bp of native sequence. This additional region of native sequence is to ensure RNA termination and to act as a small spacer region between tRNA genes. 
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Design of tRNA Sequences for overexpression
</P><P class="textProfile"> 
The tRNA sequences for use in overexpression were selected from the Genomic tRNA Database for E. coli K-12, a strain of E. coli closely related to the strains used in this study DH5a and BL21, and whose genome is the most extensively annotated. tRNA sequences with anti-codons recognizing the codons used in the “B” and “F” subunits were selected, and their native sequences used.


Two sets of tRNA genes were selected for overexpression constructs. The six tRNAs corresponding to the codons used in the “F” construct were made into a single construct with each gene proceeded by its own tRNA promoter and followed by its own terminator sequence. The additional tRNAs for the glycine, proline, and serine amino acids were included in the second construct, in addition to the first six tRNAs, as well as two additional tRNAs that had anticodons matching the threonine and arginine amino acids that would be generated in the scar sites from use of the BioBrick <A href="http://partsregistry.org/Assembly_standard_23">Assembly Standard #23</A> cloning method.
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
Design of SHMT for Overexpression
</P><P class="textProfile">
Serine hydroxymethyltransferase (SHMT) was utilized in the study to increase intracellular levels of glycine in order to yield increased spider silk production (Xia et al., 2010). The SHMT protein catalyzes the conversion of the amino acid serine into the amino acid glycine using the recyclable co-factor tetrahydrofolate (THF). Since glycine is the most used amino in spider silk (44.1% of total amino acids), increased intracellular levels of glycine will allow the cell more material to build additional spider silk proteins with. We amplified the gene with its native gene sequence from plasmid pet19-SX provided by the Lewis lab at Utah State University. 
</P><P class="textProfile">
Reaction Diagram of Serine Hydroxymethyltransferase converting serine into glycine using tetrahydrofolate as a recyclable cofactor.
</P><A name="Experimental"><H1 style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #000000; text-decoration: none;" class="table-main">Experimental Plan</H1></A><P class="textProfile">
Part of the experimental work in this iGEM project was to create functioning spider silk BioBricks that could be assembled together with the standard assembly method. The figure below shows that using the standard assembly method different sized spider silk units could be built with different level of spider silk subunit repeats. Promoter and ribosome bind systems could be dropped in upstream of these repeating units and Histag downstream to aid in purification. The repeating units are not limited to what is seen in this figure, i.e. you could poteinally have high number of repeating subunits. 


By increasing the number of silk subunits in a silk protein, we expect to see the physical properties (elasticity, breaking strength, etc) of the silk thread to increase. We also expect that after a certain size threshold, the production of the silk will decrease as the intracellular tRNA pool is exhausted and the metabolic cost per protein increases. 
</P><P class="textProfile">
Spider silk proteins expression units of increasing size (increased number of repeatitive subunits) with 10x-Histidine tags.
</P>

The sets of tRNA genes discussed above would be transferred into BioBrick plasmid pSB3K3, which contains a different origin of replication from the pSB1C3 plasmid that contains the spider silk gene construct. This allows the two plasmids to be co-transformed in the same cell line, allowing expression of both the tRNA genes and the spider silk protein. This co-expression is expected to help relieve some of the stress on the tRNA pool in the cell, and allow for increased spider silk production, even at large silk protein sizes. The diagram below illustrates what this cell line would look like.
<P class="textProfile"><I>E. coli</I> cell co-transformed with plamsids for spider silk production and tRNA overexpression.
</P><A name="Applications"><H1 style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #000000; text-decoration: none;" class="table-main">Spider Silk Applications</H1></A><P class="textProfile">Spider Silk has many potential applications.  Its unique properties of strength, elasticity, as well as being lightweight, allow it to be an excellent biomaterial.  Imagine the day when artificial tendons and ligaments or bridge cables are constructed with spider silk.  This biomaterial could be used to keep people safe in ways such as in air bags, seat belts, body armor, rock climbing ropes or cords for parachutes.  These are just a few of the ways spider silk could be used.  Scroll over the photos for applications.</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
References
</P><P class="textProfile">
Bauer B.F., Elford R.M., and Holmes W.M. 1993. Mutagenesis and functional analysis of the Escherichia coli tRNA (1Leu) promoter. Mol microbio. 7:265–273.

Blackledge T.A., Hayashi C.Y. Silken. 2006.  Toolkits: biomechanics of silk fibers spun by the orb web spider Argiope argentata (Fabricius 1775). J. Exp. Biol.  209:2452–2461.

Brooks, A.E., Stricker, S.M., Joshi, S.B., Kamerzell, T.J., Middaugh, C.R. and Lewis, R.V. 2008. Properties of synthetic spider silk fibers based on Argiope aurantia MaSp2. Biomacromolecules. 9, 1506-1510.

Foo C.W.P. and Kaplan D.L. 2002.Genetic engineering of fibrous proteins: spider dragline silk and collagen. Adv Drug Deliv Rev.54:1131-1143.

Hayashi C.Y., Shipley, N. H., Lewis, R.V. 1999. Hypotheses that correlate the sequence, structure, and mechanical properties of spider silk proteins. Int. J. of Biol. Macromol. 24, 271.

Lewis R.V. 2006. Spider silk: ancient ideas for new biomaterials. Chem. Rev. 106(9):3762–3774.

Xia, X.X., Qian, Z.G., Ki, C.S., Park, Y.H., Kaplan, D.L., and Lee, S.Y. 2010. Native-sized recombinant spider silk protein produced in metabolically engineered Escherichia coli results in a strong fiber. PNAS. 107:14059-14063.
</P><P style="width: 100%; margin-left:0 px; margin-right: 20px; text-align: left; font-family:Arial, Helvetica, sans-serif; font-weight: normal; border-bottom: 2px solid #C0C0C0; text-decoration: none;" class="table-main">
URL Credits for Images
</P><P class="textProfile"><OL><LI>Orb-weaver web: http://scienceillustrated.com.au/blog/science/news/the-safest-pattern
</LI><LI><I>Argiope aurantia</I>: http://www.designswan.com/archives/something-about-the-spiders.html
</LI><LI>Bridge Cables: http://www.winwallpapers.net/w1/2011/03/Golden-Gate-Bridge.jpg
</LI><LI>Airbags: http://4.bp.blogspot.com/-W__6FeTrx74/TqiVI7MDamI/AAAAAAAAAHg/YRq2m6lSYUY/s1600/airbags.jpg/
</LI><LI>Sutures: http://www.lotus-surgicals.com/subcategoryimages/20090310022724_products1.jpg
</LI><LI>Parachute: http://sblazak.files.wordpress.com/2011/08/parachute1.jpg
</LI><LI>Web: http://lecyberpunk.deviantart.com/art/spider-web-vector-157436131
</LI><LI>Body Armor: http://www.safeguardarmour.co.uk/media/catalog/product/cache/1/image/9df78eab33525d08d6e5fb8d27136e95/g/o/goretexreal_7.jpg
</LI><LI>Performance Wear: http://www.usoutdoor.com/spyder/spyder-silver-dip-dry-web-tneck/
</LI><LI>Tendons: http://www.eorthopod.com/images/ContentImages/foot/foot_achilles/foot_achilles_tendon_anatomy01a.jpg
</LI><LI>Seatbelts: http://www.sunwarrior.com/news/wp-content/uploads/2012/08/seatbelt-testing-sxc.jpg
</LI></OL></P></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Utah_State/Project">http://2012.igem.org/Team:Utah_State/Project</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Utah_State/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Utah_State/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Utah_State/Project&amp;oldid=281592" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>