"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w14
From 2013.igem.org
06th
Summary
1. Inoculation/Streak-out: DarR reporter system A C1/ DarRrev-TermrevC3 in pSB1C3
2. Control PCR
Inoculation/Streak-out: DarR reporter system A C1/ DarRrev-TermrevC3 in pSB1C3 
-          cultures inoculated yesterday and forgotten in fridge were incubated at 37 °Cà in the afternoon, cultures taken out and put to autoclave…
-          streak-out of both clones using plates from 23.8.13 and masterplate, respectively, was repeated and plate incubated at 37 °C (plate wasn’t dry, so clones might be mixed by water drops running over the plate…)→  plate taken out in afternoon and put into fridge (again inoculation for CryoStocks next week)
DNAse digest
2500 ng RNA
respectively
DNAse
2,5 µl
10x buffer
2,5 µl
water
rest
Total
25 µl
stored in heat block 30 min at 37°C
then EDTA was added 2,5 µl per probe and put into thermo cycler at 65°C for 10 min
 
Control PCR
-          final reaction composition:
 Component
Volume
Template (RNA or gDNA)
1 µl
Primer fwd 1:20 iGEM 102
1 µl
Primer rev 1:20
iGEM 103
1 µl
PfuS
0.5µl
dNTPs
1 µl
5x HF buffer
5 µl
water
15,5µl
total
25 µl
in thermo cycler protocol: Ivonne -> PhuS
 
Gel doc:
wells:Part1C2exp, part2C2exp, part3C2exp, part4C2exp, part8C2exp, Part1C2stat, part2C2stat, part3C2stat, part4C2stat, part8C2stat; Part1C2ON, part2C2ON, part3C2ON, part4C2ON, part8C2ON, gDNA
Nanodrop measurements:
Fold ↑
05th
Summary
1. Transformation of RiboA C5.8 and CFP-Religate control C1 into freshly competent BL-21 (in which the DAC and Empty vectors already are)
2. Platereader experiment (fluorescence characterization of RiboA) Reproduction
3.Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2
4. Preparation of Parts for the Registry
5. Inoculation/Streak-out
Transformation of RiboA C5.8 and CFP-Religate control C1 into freshly competent BL-21 (in which the DAC and Empty vectors already are)
3 µl of RiboA C5.8 were used for the transformation. 100 µl of the competent cells were used for the transformation.
4µl of the religate control.
Transformation and negative control were plated onto Cm plates (2 plates each, 2 step streak out)
Put into 37°C around 3:30 pm
Platereader experiment (fluorescence characterization of RiboA) Reproduction
The RiboA C5.8.1 and C5.8.2 did not grow in the synchronization culture. The Synch cultures were inocculated from an ON culture (OD>3) to OD=0,5 and did not grow during the following 3 hours.
Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2
-        harvested again cells of over night culture of an OD around between 2-3
-        defreezing of cells for RNA extraction 
RNA extraction from E. coli
with kit: Qiagen RNeasy Mini
-        clean bench and pipets with ethanol
-        prepare RLT buffer with 2-mercaptoethanole (1 ml RLT buffer →  10 µl 2-mercaptoethanol); here: for 15 samples, 17 ml RLT buffer were supplied with 170 µl 2-mercaptoethanol (in other Bacillus-lab in the hood)
-        prepare lysis buffer: 1xTE buffer + 0.5 mg/ml lysozyme in RNase free water (here: 340 µl 10x TE (sterile) + 85 µl 20 mg/ml lysozyme + 2975 ul RNase free waterà 3.4 ml lysis buffer)
-        set one heating block to 70 °C and pre-heat RNase free water for elution (100 µl for elution of 1 sample)
-        set another heating block to 37 °C for lysis
a) Lysis: add 200 µl lysis buffer to E. coli pellet, resuspend properly and incubate for 10 min at 37 °C;add 1 ml RLT supplied with 2-mercaptoethanol to the suspension and centrifuge for 5 min at 13 000 rpm, RT
b) removal of gDNA: load supernatant in 700 µl steps on violet columns and centrifuge at 13 000 rpm, RT, 1 min →  collect FLOW-THROUG in 2 ml epi tube (column binds gDNA, but NOT RNA →  RNA is in FLOW-THROUG!!!)
c) binding of RNA: add more or less equal volume of 70% ethanol (RNA) (here: 1 ml) to flow-through, mix by pipetting and vortexing, then load mixture on pink columns (pink columns bind RNA)
d) washing: →  no, one has to discard the flow-through…
d1) load 700 µl RW1 buffer onto column, centrifuge at 13000 rpm, RT, 1 min
d2) load 500 µl RPE1 buffer onto column, centrifuge at 13000 rpm, RT, 1 min
d3) load again 500 µl RPE1 buffer onto column, centrifuge at 13000 rpm, RT, 1 min
e) drying of column: centrifuge column in emptied collection tube 1 min, 13000 rpm, RT
f) elution: place column in fresh epi tube (RNase-free) elute RNA with 50 µl pre-heated (70°C) RNase-free water, incubate tubes at 70 °C for 5 min, then spin down at 13000rpm for 1 min, RT (here: sample “exp. 8” was eluted with 100 µl by mistake…)
g) measure concentration with NanoDrop (set it to RNA!); Blank: HPLC water (no difference to RNase free water…)
NanoDrop concentrations from today
Preparation of Parts for the Registry
preparation of the following samples: 
sample no.
part no.
sample name
2
BBa_K1045001
DarR reverse
5
BBa_K1045004
Terminator reverse
6
BBa_K1045005
RBS reverse
8
BBa_K1045007
composite part Promoter 3 with DarR operator
9
BBa_K1045008
composite part Promoter 3-DarR operator - Bba_E0240
10
BBa_K1045009
composite part Promoter 1 reverse-Promoter 3-DarR operator - Bba_E0240
12
BBa_K1045011
composite part Terminator reverse-DarR reverse
13
BBa_K1045012
composite part Terminator reverse-DarR reverse - RBS reverse-Promoter 1 reverse-Promoter 3-DarR operator - Bba_E0240 (DarR reporter system, weak  promoter)
-        pipetting scheme:
(concentration and date of sample are given; followed by the exact volume of the sample that has to be used for 250 ng plasmid. This value is followed by the blue number indicating the volume that was added to the tube and a black number indicating the amount of HPLC water added to the tube) 
at least 250 ng in 10 µl HPLC water were added to a PCR tube and stored in blue “parts box”, stored at -20 °C, 3rd freezer box of small freezer
Inoculation/Streak-out 
-          inoculation and streak-out of DarR reporter system A C1
-          inoculation and streak-out of DarRrev-TermrevC3 in pSB1C3 (from plate from 23.8.13…) (CryoStock was gone…)
-          in 4 ml LBCm
-          incubation ON, 37 °C (broth cultures forgotten in fridge, only plates were incubated…)
Fold ↑
04th
Summary
1.Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2
2. No-template control test PCR (Promoter characterization – primer test PCR)
3. Sequencing results G2L
4. Promoter characterization: qRT-PCR for RNA isolated from clones 1
5. pictures of 2nd round of drop experiment with RiboA C5.8.1 and CFP Religate C1 plates.
Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2
-        preparation of cryostocks
-        measuring of OD600nm of ON cultures in 1:20 dilution of LB and calculation of actual OD600nm→  OD600nm in ON cultures varied from 4,06 to 4,54 nmà inoculation of synchronization pre-culture: 4 ml LBAmp + 50 μl ON culture, incubation at 37 °C at ca. 200 rpm for > 2 h (the OD600nm was measured and the cells kept on bench)
-        inoculation of an over day culture 15 ml LB Amp + around 1,9 ml from synchronization culture
-        measuring of OD600nm in 1:10 dilution of LB and calculation of actual OD600nm after 30 min, 1h, 1,15h, 1,30h, 1,45h, 2h, 2,15h, 2,30h, 2,30h, 2,45h, 3h, 3,30h, 4h, 4,30h, 5h
-        (Exel table of values to be done tomorrow)
-        harvested cells of OD around 0,6 and between 2-3 stored in green cryo boy at -80°C
-        (an additional harvest at OD 0,6 was taken in order to prepare microscope single cell pictzures – have to be saved in jpg format and added tomorrow)
 
a) OD600nm ON culture
Strain and Clone
OD600nm (1:20 dilution in LB)
OD600nm (1:1 calculated)
exact ON culture volume for synchronization culture in µl*
P1 C2
0,216
4,32
46
P2 C2
0,203
4,06
49
P3 C2
0,227
4,54
49
P4 C2
0,216
4,32
46
P8 C2
0,219
4,38
46
 
b) measurement of synchronization culture before main culture and titer plate preparation
Strain and Clone
OD600nm (1:10 dilution in LB)
OD600nm (1:1 calculated)
synchronization culture volume for main cultures (15 ml) in ml *
P1 C2
0,044
0,44
1,70
P2 C2
0,043
0,43
1,74
P3 C2
0,037
0,37
2,03
P4 C2
0,039
0,39
1,9
P8 C2
0,049
0,49
1,5
No-template control test PCR (Promoter characterization – primer test PCR)
-        neg. controls of primer test PCR for promoter characterization (test of primers iGEM_94 to 103) were not negative. It could be that one of the components e.g. the dNTP mix is contaminated with all our plasmids or E. coli DNA. So, dNTPs (and RNase free water, HPLC water, dH2O and HF buffer) will be tested in a PCR. For this initial test, only the RFP primers will be used since they led to the strongest band for the negative control reactions with plasmids as templates in the test PCR from 2.9.13. This test PCR is only based on the asSummaryption that only 1 component is contaminated. In case more components are contaminated, one cannot conclude which components are the contaminated ones.
-        the following PCR reaction (total volume: 25 µl) were pipetted individually:
-        gel run:
-        2x 1% agarose-1xTAE gel
-        loading of 3 µl 2 log ladder as a marker
-        loading of 1 µl 5xLD + 4 µl PCR reaction (between two samples, the lane was left unloaded, so that settling of one sample in the neighbouring well  by mistake won’t influence the band pattern of the sample loaded next to it …)
-        run at 100 V
-        EtBr staining + destaining in water
-        UV detection 
Gel:
Loading: Marker/Reaction 1/---/Reaction2/---/Reaction 3/---/Reaction 4/---Reaction5/---/Reaction6/Marker
expected bands: a band of 280 bp is expected for reaction 6, and for all other reactions in case one or more of the components are contaminated with (plasmid) DNA
A strong 280 bp band is visible for positive control. Weak 280 bp-bands are observed for all reactions except for Reaction 3. So, it could be that the buffer is contaminated… buffer thrown away
Sequencing results G2L
samples:
→  Repeat cloning of Promoter1rev and RBSrev in part 6.3 A.
→  Forget about cloning of DarR under control of strong promoter. Might not be so good for E. coli growth if DarR is strongly expressed.
Promoter characterization: qRT-PCR for RNA isolated from clones 1
-        qRT-PCR with Biorad kit
-        Final reaction (total volume: 20 µl) should contain 100 ng RNA, 10 µl 2x buffer, 0.4 µl enzyme and 1.2 µl of each primer (1:20 diluted in RNase free water)
-        Mastermixes for each RNA sample and a neg. control were prepared. They consisted of2x buffer (light-sensitive – contains SYBRgreen!), RNA/RNase free water for no-template-controls, RNase free water, and enzyme for 3.5 reactions.
-        Mastermixes consisting of 1.2 µl Primer fwd and 1.2 µl Primer reverse per reaction were prepared and then added to 2.4 µl primer mix prepared in the wells of the qRT-PCR plate (in the beginning, I made a mistake and prepared only 10x mastermixes though I needed 20x mastermixes. I noticed this, when rrsB primer mastermix was empty after the half of the wells of the plate. Therefore, for rrsB primers, I had to make two 10x mastermixes, while I added again 12 µl of each primer to the 10x mastermixes prepared for rpoA and rfp primers to yield 20x mastermixes…)
-        17.6 µl of the buffer/template/water/enzyme mastermix were added to the wells (many bubbles on surface of solution…; I made another mistake: I added the sample no. 16 mastermix to well that already contained sample 10 mastermix and rfp primers. Therefore, I had to pipet both reactions individually again…)
-        qRT-PCR run using standard protocol:
 
Protocol:
            Cycle 1: (1X)
            Step 1:                              50,0 °C                            for 10:00.
            Cycle 2: (1X)
            Step 1:                              95,0 °C                            for 05:00.
            Cycle 3: (45X)
            Step 1:                              95,0 °C                            for 00:10.
            Step 2:                              60,0 °C                            for 00:30.
            Data collection and real-time analysis enabled.
            Cycle 4: (1X)
            Step 1:                              95,0 °C                            for 01:00.
            Cycle 5: (1X)
            Step 1:                              55,0 °C                            for 01:00.
            Cycle 6: (81X)
            Step 1:                              55,0 °C-95,0 °C                      for 00:10.
            Increase set point temperature after cycle 2 by 0,5 °C
            Melt curve data collection and analysis enabled.
Calculations*:
The values in the brackets indicate the amount used for 3.5x Mastermix.
The other values are for 1x reaction
*numbers and abbreviations: see pipetting scheme DNase I digest, 2.9.13 and RNA concentrations 28.9.13
Pipetting scheme*
*numbers and abbreviations: see pipetting scheme DNase I digest, 2.9.13 and RNA concentrations 28.9.13
Results:
I made a mistake and named well B07 “P1 C1 ON rrsB”, though it contained sample “P1 C1 ON rpoA”! Therefore in the original qRT-PCR data it’s named B07 P1 C1 ON rrsB
[file:Melt Curve Analysis Spreadsheet Data for SYBR]
→  A peak at a certain temperature indicates the formation of a product that is molten to 50 % at this temperature. During a PCR, only one specific product should be formed by one specific primer pair. Therefore, for all PCR reactions with one specific primer pair, there should be only one peak at a certain temperature.
Here, it looked as if for rpoA or rrsB primer pair only one specific product was formed during PCR (rrsB is marked in red, TM of specific PCR product = 84.50 °C; rpoA is marked in black; TM of specific PCR product = 85.50°C).
For rfp primers, for most reactions, a PCR product of a TM of 83.50 °C wasobserved for most reactions.
For P2 C1 and P8 C1,several different products seemed to have been formed when rfp primers were used (Could be because boths clones might/should not contain RFP).
Only for reactionsB10: P1 C1 ON rfp andD10: P2 C2 ON rfp and for all P4 C1 rfp (F08 – F10) samples as well as for P3 C1 ON rfp (F11), a PCR product of a different TM was detected (TM = 83.00 °C). This product might also be present in reaction G10: P8 C1 ON rfp. Hence, the rfp primers could bind slightly unspecifically to residual plasmid/chrom. DNA or other E. coli transcripts
The CT values for most clones could be analyzed further. The analysis was also done for the problematic clones with non-specific product formation by rfp primers. If these values prove to be stable will be seen in future experiments…
-        The NTC for rpoA had also a CT though it should have N/A. It might be slightly contaminated with E.coli chrom.DNA/RNA, because the rpoA product was formed in very low amounts. This might however be neglectible since the CT value is 37 and therefore much higher than the highest CT value obtained for the actual reactions with rpoA primers (D7 = P2 C2 ON rpoA; CT = 26.03 →  difference is ca. 10 cycles →  template in NTC (and possibly also in other wells…?) is >1000x diluted compared to actual reactions)
-        For rrsB, the CT values were <10. CT values <10 might be inexactly because of detection limitations. Hence, rrsB CT values were not considered for further data analysis. For rrsB, there seemed to be a slight contamination with template in the NTC, as well. However, the melting point of this product differed from that of all other rrsB samples, indicating that it might be non-specific.
-        For the rfp NTC, no CT value was measured, but several products could have been formed as indicated by melting point analysis. However their amounts might have been too low to be detected.
→  rpoA was used as housekeeping gene for normalization
Calculations:
The formulas used are indicated in the tables. The values are given with far too many digits after the decimal point. I don’t know how many of them are significant. This we will find out when we have our first replicates and when we calculate the mean and the standard deviations
Discussion:
Only the strenght of the promoters relative to the weakest promoter for each phase can be considered. It is not possible to calculate the relative strenght of a specific promoter (e. g. promoter 3) relative to the different phases, since the rpoA mRNA levels indicate a different intesity of transcription in each phase.
During exponential phase, Promoter 1 was the weakest, Promoter 2 was 3x stronger. Promoter 3 was the strongest (ca. 98x stronger than Promoter 1). Promoter 4 was weaker than Promoter 3 (only ca. 37x stronger than Promoter). P2 C1 seemed to contain no promoter/RFP as for P8, the empty vector control.
During stationary phase, Promoter 1 was still the weakest, while Promoter 2 was ca. 6x stronger. Promoter 3 was ca. 19x stronger. Interestingly, Promoter 4 was the strongest and slightly stronger than Promoter 3 (vs. exponential phase). No significant rfp mRNA levels were observed for P2 C2.
After ON incubation, Promoter 1 was still weak. Promoter 2 showed ca. 13x strength. Promoter 3 was (as in exponential phase) the strongest promoter with ca. 32x strength compared to Promoter 1. Promoter 4 was slightly weaker (ca. 29x as strong as Promoter 1). P2 C1 did not show significant rfp transcript amounts. 
Be careful: All values obtained for “After ON incubation” are likely to be wrong since the melting point of the PCR product is different from that of other samples. This might also be true for all P4 values.
ordering of nuclease free water downstairs
pictures of 2nd round of drop experiment with RiboA C5.8.1 and CFP Religate C1 plates:
[pic7]
Discussion:
RiboA C5.8.1 and DAC on the same AB- IPTG+/IPTG- Plates
Pictures:
Cells mixed before plating. –IPTG left, +IPTG right
Discussion:
Inocculation of cultures for platereader experiment (fluorescence characterization of RiboA) Reproduction
Cryostocks RiboA 5.8.1, 5.8.2 and CFP C1 and CFP C2 were used
4ml LB + Cm each. 4 µl of the cryostock per inocculation
Fold ↑
03rd
Summary: 1. Gel run: DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization) 2. NanoDrop concentration measurement of RNA samples after DNase I digest 3. Sequencing results DarR reporter system 4. 2nd round “Drop”-experiment with RiboA C5.8 and CFP Religate C1 plates 5. Evaluation of the Plate reader experiment yesterday (to be done) (fluorescence characterization of RiboA) 6. Trafo of RiboA C5.8 plasmid ino the newly made competent cells
Gel run: DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization) 
-        2x 1% agarose-1xTAE gel
-        loading of 3 µl 2 log ladder as a marker
-        loading of 1 µl 5xLD + 4 µl PCR reaction
-        run at 100 V
-        EtBr staining + destaining in water
-        UV detection
Gel 1:
Loading: Marker/neg. control/pos. control/1/2/3/4/5/6/7/8/9/10/11/12/Marker*
 
Gel 2:
Loading: Marker/13/14/15/16/17/18/pos. control/neg. control/Marker*
-        expected bands: only for pos. control, a band at 280 bp should be visible
-        neg. control has slight band at ca. 280 bp… again not negative…
-        for all RNA samples very weak bands at ca. 280 bp could be detected
-        positive control as expected
*numbers: see pipetting scheme DNase I digest, 2.9.13
NanoDrop concentration measurement of RNA samples after DNase I digest
Click to view full resolution
This time, for all samples, a peak at ca. 220 to 225 nm was noticed.
Samples stored at -20 °C, 3rd freezerbox in yellow-tip-box rag
Sequencing results DarR reporter system
DarR reporter system A C1: the whole sequence is ok
DarR reporter system B C1: the sequence is ok, except for
-        scar between RBSrev and Promoter 3 rev
-        Promoter 3rev corresponds to Promoter1rev with an additional A at pos. 5 of promoter
-        sequence between the two promoters is not sequenced (xxxxxx…)
-        operator could contain a C instead of an A at pos. 6 (but the sequences from VR and KG307 stop shortly after operator →  therefore, this could also be a sequencing error
→ only work with reporter system A, but forget about B…
Plates: Promoter Parts Clones
-        plates taken out and stored at 4 °C, big fridge
2nd round “Drop”-experiment with RiboA C5.8 and CFP Religate C1 plates
 
8 plates were prepared:
2x RiboA C5.8 without IPTG
2x RiboA C5.8 with IPTG
2x CFP Religate C1 without IPTG
2x CFP Religate C1 with IPTG
As follows:
25ml sterile water were filled into four 50ml falcons. Where needed, 50µl IPTG 1M (end concentration 1mM) were added. Also the 300µl Glycerin stocks prepared earlier were added to the water in the dedicated falcons. The falcons were inverted several times. 2x LB medium was prepared by Katrin Gunka and melted in the microwave. It was added to the 25ml water rather hot to the total volume of 50ml. The falcons were inverted a few times and then two 25ml plates were poured from each falcon.
 
On each plate 3 drops of DAC or empty control:
1x undiluted overnight culture
1x 1:10 diluted overnight culture
1x undiluted overnight culture smeared over one half of the plate
Plates incubated at 37°C overnight
Evaluation of the Plate reader experiment yesterday (to be done) (fluorescence characterization of RiboA)
Plated out RiboA C5.8.1 and DAC on the same AB- IPTG+/IPTG- Plates. 
4 plates: 2xIPTG-
                  2xIPTG+
On one of each IPTG+/- plates, DAC and RiboA were plated separately on the right and the left, but were mixed in the middle of the plate
On the other two, the Cells were mixed prior to the plating.
This Experiment is to easily show possible interactions of the clones
Preparation of competent BL-21 cells 
ON culture of BL-21 (inoculated by Joni) was used to inoculate 10 ml LB with a start OD 600of 0,11. After 1h at 37°C, the culture reached an OD of 0,34.
The MgCl2 treatment was performed as described in the method collection.
Trafo of RiboA C5.8 plasmid ino the newly made competent cells
3 µl of RiboA C5.8 were used for the transformation. 100 µl of the competent cells were used for the transformation.
Transformation and negative control were plated onto Cm plates (2 plates each, 2 step streak out)
Put into 37°C around 3:30 pm
To be repeated on the 5. Colonies on negative control!! ON culture inoculated on the 4.9 at 5pm
Shopping downstairs
Pooring of 4 LB plates with (2) and without (2) (25 µl) IPDG, but without any further resistance
Innoculation of re-trafo of part 1,2,3,4, 8 in LB-medium with amp
Fold ↑
02nd
Summary:
1. Fluorescence Assay of our RiboswitchA CFP C5.8.1+2 and the CFP Religate C1+2 as control (Fluorescence characterization of RiboA)
2. Minipreps of RiboA Retrafo Clones 5.8.1-5 and the CFP Religate Controls Clones 1+2
3. Test PCR for qRT-PCR primers iGEM_94 – 103
4. Sequencing of DarR reporter system clones by SeqLab
5. DNase I digest of RNA samples (Promoter characterization)
6. DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization)
Fluorescence Assay of our RiboswitchA CFP C5.8.1+2 and the CFP Religate C1+2 as control (Fluorescence characterization of RiboA)
For the assay, the CFP Religate is the non fluorescent control. Two clones were used for a biological replicate.
→ Yesterday´s overnight cultures were inoculated in 4mL cultures with Cm at 0,5 to synchronize them
→ After 1h the cells had not grown at all
Culture
CFP Rel. C1
CFP Rel. C2
RiboA 5.8.1
RiboA 5.8.2
OD600
0.5
0.43
0.5
0.48
→ After 2h three of four cultures had grown
Culture
CFP Rel. C1
CFP Rel. C2
RiboA 5.8.1
RiboA 5.8.2
OD600
1
0.9
0.4
0.7
→ All cultures (also the one that didn’t grow) were diluted in a 1mL eppi to OD=0.05 and used for the Fluorescence assay in the plate reader in technical triplets
→ The platereader will breed the cells at 37°C and read the plate every 15min. for 21 hours. Excitation 436nm and Emission 480nm
Minipreps of RiboA Retrafo Clones 5.8.1-5 and the CFP Religate Controls Clones 1+2 
Nanodrop measurements on the eppis (btw. 70 and 500ng/µl)
→  The CFP Religate clones need to be sequenced on friday!
Inocculated DAC and empty cells for the “Drop” Experiment tomorrow
Inocculated RiboA C5.8.1 and CFP Religate C1 for possible experiment ideas
Inocculated BL-21 T4 (not Blue) cells to be made competent tomorrow. These cells are for a transformation of RiboA into this stem for comparability with the DAC and empty cells.
MiniPlasmid Preparation of DarR reporter system clones
-        samples: cultures inoculated yesterday evening by Katrin G.
-        MiniPrep with Nucleospin kit from Macherery-Nagel
-        elution in 30 µl HPLC-water (pre-warmed, incubation at 50 °C, 2 min)
-        NanoDrop concentration measurement:
→  enough DNA for sequencing by SeqLab!!!^^
Test PCR for qRT-PCR primers iGEM_94 – 103
-        preparation of 1:20 dilution of primer stock solutions iGEM_94 to iGEM_103 in RNase free water, now stored in orange RNA box in 3rd freezer box
-        Test PCR:
-        final reaction composition:
Component
Volume
Template
1 µl
Primer fwd 1:20
1 µl
Primer rev 1:20
1 µl
PfuS
1µl
dNTPs
1 µl
5x HF buffer
5 µl
water
15µl
total
25 µl
 -        preparation of template and primers in PCR tubes:
template
plasmid part 6 C1
(GFP)
plasmid DarRrev in pSB1C3
(DarRrev)
plasmid part 1 C1
(RFP)
E.coli chrom. DNA
(rrsB)
E.coli chrom. DNA
(rpoA)
primer combination
iGEM_94/95
iGEM_96/97
iGEM_98/99
iGEM_100/101
iGEM_102/103
 
for the plasmids, dilutions containing ca. 10 ng/µl were prepared and 1 µl of this dilution used as template
(DarRrev in pSB1C3 C4 plasmid, 16.8.13, 282.5 ng/µl →  1:28 dilution;
part 6 C1 plasmid, 7.6.13, 88.9 ng/µl →  1:9 dilution
part 1 C1 plasmid, 254.8 ng/µl →  1:25 dilution)
preparation of neg. control with water as a template for each primer combination, as well →  in total, 10 PCR reactions were prepared
-        preparation of mastermix for 12 reactions consisting of buffer, water, PfuS and dNTP mix, then addition of 22 µl mastermix to each reaction
THE WATER USED FOR THE WHOLE PCR (except for the primers, of course) WAS FRESH HPLC WATER! (to prevent “positive” negative controls….) 
-        PCR protocol:
Step
temperature
time
initial denaturation
98.5 °C
5 min
denaturation
98.5 °C
30 s
annealing
52 °C
35 s
elongation
72 °C
1 min
final elongation
72 °C
10 min
hold
15 °C
∞
 →  30 cycles
 
-        agarose gel run for PCR: 
-        1%-agarose-1xTAE gel
-        addition of 5 µl 5xLD to each PCR sample
-        loading of 5 µl of each PCR sample
-        loading of 3 µl 2 log ladder as a marker
-        run at 100 V
-        EtBr staining + destaining in water
-        UV detection
Loading: Marker/ Neg. control iGEM_94+95/PCR iGEM_94+95/ Neg. control iGEM_96+97/PCR iGEM_96+97/ Neg. control iGEM_98+99/PCR iGEM_98+99/ Neg. control iGEM_100+101/PCR iGEM_100+101/ Neg. control iGEM_102+103/PCR iGEM_102+103/Marker/---/---/---/---
-        expected product size: 280 bp; neg. controls: no product (in case of reactions with plasmid as a template, the bands of the plasmid could be visible)
→  for all neg. controls weak to very weak bands at 280 bp could be detected →  for samples with primers iGEM_100/101 and iGEM_102/103, this could be because of a pipetting error (it could be that, after pipetting the mastermix into PCR iGEM_100/101, I forgot to exchange the pipet tip. Then, I added the mastermix to samples neg. control iGEM_100/101, PCR iGEM_102/103 and neg. control iGEM_102/103); an alternative explanation would be that I used the PfuS purified from E. coli in this department. It could contain residual E. coli chrom. DNA); in general, all primers generate under the given conditions very high amounts of the expected PCR product. It could therefore be that the weak bands in the neg. control lanes result from PCR product that went from the PCR reaction lane into the neg. control lane…
→  Annealing temperature is ok., primers lead to PCR product of expected size/ all primers lead to PCR product of same size; →  repeat PCR with Phusion® for rrsB and rpoA, to see, if bands result from contamination with E. coli chrom. DNA
Re-streak of clones 
-        re-streak of Re-trafo clones C2 and C3 for parts 1, 2, 3, 4 and 8 from plates from 30.9.13 on fresh LBAmp (100 µg/ml)
-        re-streak of initial promoter clones C1 – C3 of parts 1 – 4 on fresh LBAmp (100 µg/ml) plates (it could be that the plates borrowed from DAC team, contained only 50 µg/ml Ampicillin →  repetition of experiment with plates for fotos…)
-        streak-out was done in the morning, therefore plates were kept in fridge til evening and then put to 37 °C for ON incubation
Sequencing of DarR reporter system clones by SeqLab
-        600 ng plasmid DNA in 3 µl + 4 µl primer 1:20 diluted
reactions:
1 – DarR reporter system A C1 + VF2
2 – DarR reporter s ystem A C1 + VR
3 – DarR reporter system A C1 + KG307
4 – DarR reporter system B  C1 + VF2
5 – DarR reporter system B  C1 + VR
6 – DarR reporter system B  C1 + KG307
Test PCR for qRT-PCR primers repeated (for rrsB and rpoA) 
-        PCR done exactly as described before, but this time, Phusion ® was used instead of PfuS and only the PCR reactions for rrsB and rpoA including their neg. controls were prepared (with help of 5x master mix (HF buffer 5x, Phusion®, HPLC water, dNTP mix) which was added to primer-template mixtures)
-        PCR protocol was the same as before
-        gel run (together with Katrin G.’s samples), by Katrin G.:
1%-agarose-1xTAE gel, 5 µl 5xLD added to PCR sample and loaded on gel, EtBr staining + destaining in water, UV detection
Loading: ----Marker/Neg. control iGEM_100+101/---/PCR iGEM_100+101/ Neg. control iGEM_102+103/PCR iGEM_102+103/
→  neg. control for iGEM_100/101 still weak band at ca. 280 bp →  no explanation! It can’t be that samples ran over into neighbouring lanes, since lane that was left unloaded did not show bands…
→  iGEM_102/103 led to additional bands above and below 280 bp and to very strong primer cloud… or a very short unspecific product…?
→  do DNase digest control PCR with iGEM_102/103, because for this neg. control, no band at 280 bp was detected
→  neg. control for iGEM_100/101 still weak band at ca. 280 bp →  no explanation! It can’t be that samples ran over into neighbouring lanes, since lane that was left unloaded did not show bands…
→  iGEM_102/103 led to additional bands above and below 280 bp and to very strong primer cloud… or a very short unspecific product…?
→  do DNase digest control PCR with iGEM_102/103, because for this neg. control, no band at 280 bp was detected
DNase I digest of RNA samples (Promoter characterization)
-        samples: from 29.8.13
-        final reaction should contain:
RNA
2500 ng (2.5 µg)
DNase I (Fermentas)
2.5 µl
10x DNase I buffer
2.5 µl
RNase free water
Filled up to 25 µl total volume
 -        RNA and RNase free water were prepared in RNA-PCR tubes (autoclaved twice) (total volume: 10.5 µl), then 14. 5 µl of the following mastermix (for 20 reactions) were added:
Component of 20x mastermix
Volume
DNase I (Fermentas)
50 µl
10x DNase I buffer
50 µl
RNase free water
190 µl
The RNA-RNase free water preparations were (table from 29.8.13 expanded and partially re-written for better deciphering…):
1. sample name and number; 2. RNA concentration, 3. A260nm/A280nm; 4. A260nm/A230nm, 5. Volume of RNA solution used for RNA-RNase free water preparation (“µl RNA”); 6. Volume of RNase free water used for RNA-RNase free water preparation (“µl dH2O”)
Click to view full resolution
-        Incubation for 30 min at 37 °C in PCR cycler (program: DNase1)
-        Addition of 2.5 µl EDTA (50 mM stock solution, Fermentas), incubation for 10 min at 65 °C in PCR cycler (program: DNase2)
-        Samples stored at -20°C in yellow-tip-box rag (freezer box 3)
DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization)
Principle: DNA polymerase only amplifies DNA, but not RNA; if residual DNA from E. coli is present in RNA after DNase I digest, one can test this by PCR amplification of an E. coli housekeeping gene
Samples: RNA after today’s DNase I digest
Primers: qRT-PCR primers for rpoA (iGEM_102/103) →  these primers gave a big primer cloud at bottom of lane in qRT-PCR primer test PCR, but no band in neg. control at 280 bp
Note: since DNA matters in this case, but not the RNA, one can use normal water for the PCR.
-        Final PCR reaction should contain:
Component
Volume
dNTPs
1 µl
Primer iGEM_102
1 µl
Primer iGEM_103
1 µl
PfuS
1 µl (mistake: one should use only 0.5 μl to save material!)
Template (RNA after NaseI-treament, water or chrom. DNA)
1 µl
HF buffer 5x
5 µl
HPLC water
15 µl (use 15.5 μl if only 0.5 μl PfuS!)
total
25 µl
-       A mastermix consisting of all components except for the template was prepared (in the beginning a 20 x master mix was made, but this was just enough for all reactions with RNA after DNaseI digest as a template. Since nothing was left for the (optional) neg. control (HPLC water as template) and the (obligate) positive control (E. coli chrom. DNA), an additional 2x mastermix was made and used for the controls). In all cases, 24 µl mastermix were added to 1 µl of template prepared in normal PCR tubes.
-        PCR protocol: same as for qRT-PCR primer test PCR, run ON
Fold ↑
Low and high phospahte medium for RT-PCR
Low and high phospahte medium for   RT-PCR
 
MW (g/mol)
Stock
V (1000 ml)
V (50 ml)
50 mM TRIS pH7.0
121.14
1 M
50 ml
2.5 ml
3.03 mM (NH4)2 SO4
132.14
20% = 1.51 M
2 ml
100 µl
6.8 mM Na-Citrate
294.1
mit NaOH pH 7.0
1 M
6.8 ml
340 µl
3.04 mM FeCl3
270.3
1 M
3 ml
150 µl
1 mM MnCl2
197.91
1 M
1 ml
50 µl
3.5 mM MgSO4
246.48
1 M
3.5 ml
175 µl
0.01 mM ZnCl2
136.28
1 mM
10 ml
500 µl
0.5% Glukose
 
50%
10 ml
500 µl
0.05%     CAA
 
10%
17.5 ml
875 µl
10 mM L-Arg
174.2
10% = 0.57 M
10 ml
500 µl
Tryptophan
 
5 mg/ml
10 ml
500 µl
3.5 mM KH2PO4
High
136.09 mit KOH pH     7.0
500 mM
7 ml
350 µl
0.065 mM KH2PO4
Low
136.09
500 mM
130 µl
6.5 µl
Wasser
 
 
 
 
·         The   same strains were used as before
·         Procedure   to do the RNA was also the same
The   digestion was redone but I forgot to save the picture…extremely clever…
Results:
The   idea was that in cells with low amounts of c-di-AMP there is a higher   expression of ydaO because the riboswitch is not present. In cells   with much c-di-AMP the riboswitch should be present and therefore there   should be less expression of the ydaO gene.
Reminder:
·         44   means GP 1344 à hyperactive strain =   much c-di-AMP production
·         46   means GP 1346 à knock out mutant =   little c-di-AMP production
Note:
·         the   wt in high phosphate has the same c-di-AMP amount like the 1344 in low   phosphate
·         Furthermore:   c-di-AMP levels in low phosphate are very similar to c-di-AMP levels in CSE   0,5% Glc medium (see 09.08.2013)
 
 
 
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w14 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
