"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w3
From 2013.igem.org
21st
Transformation   from yesterday worked!
Transformation   from yesterday worked! 6 clones! And the   Without-Insert-Control lacks clones!
è     Plates stored at 4 °C
20th
Digestion of PCR product (DarR Operator) (10 cycles) and Plasmid backbone (pSB1A3)
Concentration   of stored PCR products from
19.6.13
No. of PCR cycles
c(DNA) [ng/µl]
A260/A280
A260/A230
PCR product
10 cycles
1068.7
1.63
1.07
PCR product
20 cycles
975.7
1.66
1.10
PCR product
30 cycles
964.9
1.65
1.11
 
Digestion   of PCR product (AND WHATS IN THE PCR?) (10   cycles) and Plasmid backbone (pSB1A3 (with AmpR))
According   to: http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones
 
Ezyme   master mix:
5 µl Buffer 0
0.5 µl EcoRI
0.5 µl PstI
18 µl dH2O
 
Digestion of backbone resp. insert:
4 µl linearized plasmid backbone (25 ng/µl   for 100 ng total)
4 µl Enzyme master mix
resp.:
5 µl insert
4 µl Enzyme master mix
resp.:
10 µl insert
4 µl Enzyme master mix
 
digest 37°C/30 min heat kill 80°C/20 min
 
Ligation   of backbone and insert:
According   to: http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones
 
2 µl digested plasmid backbone
10 µl EcoRI and PstI digested fragment
1.5 µl T4 DNA Ligase   buffer
0.5 µl T4 DNA Ligase
1 µl H2O
ligate 30°C/30 min heat kill 80°C/min
 
Transformation   of ligated parts
According to method book
Digestion of amplified DAC-domains (S.aureus and S.   pneumonia) from 17.06.
Digestion of amplified DAC-domains (S.aureus and S.   pneumonia) from 17.06.
 
Components:
Volume
 
plasmid     DNA
 
16µl
PstI
6µl
EcoRI
6µl
orange     buffer (10x)
8µl
H2O
44µl
è incubation for 30min at 37°C
è heat inactivation for 20min at 65°C
è store at -20°C
 
Digestion of prepared Strep-Tag from 17.06.
 
Components:
Volume
Strep-Tag
16µl
PstI
6µl
EcoRI
6µl
orange     buffer (10x)
8µl
H2O
44µl
 
è incubation for 30min at 37°C
è heat inactivation for 20min at 65°C
è store at -20°C
 
è Preparation of   Strap-Tag worked
è Preparation of DAC   domain of S.aureus worked
è Preparation of DAC   domain of S- pneumonia failed = EcoRI restriction site in fragment
 
Fold ↑
19th
PCR using primer iGEM 36,37 for construction of DarR operator biobrick,DarR sequencing
PCR using primer iGEM 36,37 for construction of DarR operator biobrick.
Same recipe as yesterdays PCR, rest the   protocol,
98.5
5min
 
 
 
98.5
30s
10 cycle
20cycle
30cycle
61.0
30s
72.0
2min
72.0
10min
 
 
 
 Ctrl: no enzyme
Increase the gel concentration to 4%.
PCR worked, Gel here:
Lane 1-8: ladder, iGEM36, iGEM37, ctrl(no Polyermase), 1 cycle, 10cycle, 20cycle, 30cycle.
Gel concentration : 4%
1cycle PCR product from yesterday.
So,   use 4% gel, 10cycle PCR is enough
DarR   sequencing results
in “our” genomic DNA of M. smegmatis, there are two additional   codons inserted right after the GTG start codon:
“our” gDNA: GTG  AGC  GCG  AGC  GCC à Ser-Ala-Ser-Ala
Paper gDNA:   GTG  _ _ _  _ _ _  AGC  GCC à Ser-Ala
Additionally, there are 5 point   mutations:
120 C à G
132 A à G
210 T à C
228 C à T
263 T à C
All are silent, except for one: aa 87 Val à Ala   (both small, hydrophobic aa)
Primer   Design
Design of new DarR biobrick fwd primer   (since 2 additional codons inserted and old primer designed for DarR gene   without these codons)
18th
Test restriction of Part 2 C2,Primer design,Construction of DarR operator biobrick(PCR)
Plate reader assay
-          Plate thrown away after ON measurement
-          Results:
 
Concentration of plasmid purified from   Clone 2 of Part 2
Part no.
c(DNA)
[ng/µl]
A260/A280
A260/A230
2 C2
120.6
1.87
2.24
 
Test restriction of Part 2 Clone 2
Component
Uncut control
SpeI single     digestion
PstI single     digestion
SpeI/PstI     double digestion
Plasmid DNA     (150 ng)
1.2     µl
1.2     µl
1.2     µl
1.2     µl
10x buffer
1     µl (Tango)
1     µl (Tango)
1     µl (buffer 0)
1     µl (Tango)
SpeI
-
1     µl
-
1     µl
PstI
-
-
1     µl
2     µl
dH2O
7.8     µl
6.8     µl
6.8     µl
4.8     µl
Total
10 µl
10 µl
10 µl
10 µl
-          Incubation for c
a. 1.5
h at 37   °C (incubator)
-          Addition of 2 µl of 5x DNA   loading dye and run on 1 % 1xTAE-agarose gel (marker: 100 bp ladder)
-          EtBr staining and destaining   in H2O, UV detection
è     Gel: See ppt-file “dropbox > iGEM > Reporter-Team > Geldoc“
Conclusions:
è     Clone 2 contains the expected   plasmid harboring part 2 (impure stock?)
 
Primer design and ordering
Design   of primer iGEM_43
Ordering   of primers iGEM_38 to iGEM_43 from Sigma-Aldrich
 
PCR:
PCR using primer iGEM32,33 for   construction of DarR biobrick.
Because of those two primers have high   tendency to form 2nd structure, 1x 2x 4x primers are applied, to   see the recipe, see dropbox > Reporter Team > PCR_iGEM32_33.xls
PCR was successful, better use 2x   primer. Protocol as follows:
98.5
5min
 
98.5
30s
30 cycles
67.0
35s
72.0
2min
72.0
10min
 
 
è     Gel: See ppt-file “dropbox > iGEM > Reporter-Team > Geldoc“
PCR using primer iGEM 36,37 for   construction of DarR operator biobrick. To see   the recipe, see dropbox > Reporter Team > PCR_iGEM32_33.xls
Protocol as follows:
98.5
5min
Only one cycle
98.5
30s
61.0
30s
72.0
12min
 
è     Gel: See ppt-file “dropbox > iGEM > Reporter-Team > Geldoc“
 
2% gel cannot separate the primers and   PCR product:
iGEM36:36nt
iGEM37:35nt
PCR product should be 57bp
Gel1: forget the primer as ctrl
Gel2: load the wrong primer
 
17th
Plate reader assay for the pink strains, Plasmid mini prep for Part C2
Cryo-Stocks of E. coli transfomant (2, C2)
Stored   in -70 °C (red box)
 
Plasmid Mini-Preparation of part 2 C2
-          harvesting of ca. 5 ml pellets in 2 ml Epis for future Plasmid   Mini-Preparation à stored at – 20 °C in red box
-          harvesting of remaining culture for today’s prep
è     Performed as on
7.6.13
 
Plate reader assay
-          Measurment of OD of ON cultures in 1:10 dilution in LB; LB as blank   (see table)
-          Preparation of E.coli cultures of an OD600nm = 0.05 in LB (without antibiotics) in   96-well microtiterplate;
final volume: 180 µl (see table) à triplicates for each clone
Calculation:
V(well culture) * OD600nm (wanted)/ OD600nm (ON-culture) = V(ON culture needed for   preparation of well cultures)
180 µl * 0.05/OD600nm (ON-culture) = V(ON culture needed for preparation of well cultures) [µ]
 
Values   and amounts for well cultures:
Clone
OD600nm
(ON culture)
1:10
OD600nm
(ON culture)
1:1
V(ON culture for preparation of well culture)[µ]
DHα
0.433
4.33
2.1
Part 1
C1
0.426
4.26
2.1
C2
0.450
4.50
2
Part 2
C1
0.416
4.16
2.2
C2
0.436
4.36
2.1
C3
0.434
4.34
2.1
Part 3
C1
0.467
4.67
1.9
C2
0.475
4.75
1.9
C3
0.473
4.73
1.9
Part 4
C1
0.457
4.57
2
C 2
0.466
4.66
1.9
C3
0.437
4.37
2.1
Part 6
C1
0.450
4.50
2
C2
0.431
4.31
2.1
C3
0.448
4.48
2
è     Pipetting of ON-culture volume into   each well, then addition of 180 µl LB without antibiotics using multi-channel   pipett (be careful: tips must be fixed properly!)
è     For scheme: see folder
-          Incubation of plate at 37 °C with vortexing in plate reader,   fluorescence of GFP and RFP, as well as OD600nm measured every 15   min for approx. 3 h
 
Fluorescence excited and measured at:
 
Absorption
Emission
RFP
555 nm
583 nm
GFP
489 nm
509 nm
-          Storage of plate at 4 °C
-          Measurement ON under same conditions
Preperation of Strep-Tag for Biobrick assembly
Preperation of Strep-Tag for Biobrick assembly
 
Components:
Volume (µl)
Primer     iGEM_11 (5µM)
5µl
Primer     iGEM_12 (5µM)
5µl
dNTPs
1µl
buffer     (5x)
10µl
H2O
28µl
 
è incubation at 90°C for 5min, cool   down at room temperature and add 1µl of PfuS polymerase
è incubation at 72°C for 15min
 
Amplification of S. aureus and S. pneumonia DAC domains   for Biobrick assembly
 
Components:
Volume
Primer     iGEM_13 (5µM)
2µl
Primer     iGEM_14 (5µM)
2µl
Primer     iGEM_15 (5µM)
2µl
Primer     iGEM_16 (5µM)
2µl
genomic     DNA
2µl
dNTPs
2µl
buffer     (5x)
10µl
H2O
33µl
PfuS     polymerase
1µl
 
Fold ↑
16th
Innoculation of Part 3 C2
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
