"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w11
From 2013.igem.org
16th
Transformation of ligations part6.2 + Promoter1rev or Promoter3rev and pSB1C3 + Promoter1rev , Promoter3rev or RBSrev, Colony PCR of clones from transformation of ligations part6.2 + Promoter1rev or Promoter3rev and pSB1C3 + Promoter1rev , Promoter3rev or RBSrev, Mini Plasmid Prep of clones inoculated on 15.6.13 and preparation of some cryostocks, Test Restriction Digest of DarRrev and Termrev clones, Preparation of samples for sequencing by SeqLab.
Loading of yesterday´s colony PCR of Ribo A-D onto Gel
Gel:
M|Ribo A clone 1-10|RiboB clone 1-5
 
M|Ribo C clone 1-5|RiboD clone 1-5
Minprep of inoculated cultures from RiboA
(wrong eluted -> has to be done next week again)
for R.D. and ligation
Miniprep of inoculated cultures from
Ribo B clones 1 and 2; Ribo C clones 2 and 4 , Ribo D clones 2 and 5
(stored in To do box) 
Nanodrop:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Ribo B clone 1
11.3
1.71
1.60
Ribo B clone 2
60.6
1.85
1.76
Ribo C clone 2
31.8
1.83
1.33
Ribo C clone 4
58.4
1.81
1.91
Ribo D clone 2
40.5
1.96
1.99
Ribo D clone 5
81.4
1.87
1.66
Transformation of ligations part6.2 + Promoter1rev or Promoter3rev and pSB1C3 + Promoter1rev , Promoter3rev or RBSrev 
- negative control: no colonies, neither on 50 μl-plate nor on rest-plate
- part 6.2 w/o insert: no colonies on 50 μl-plate, but many clones on rest-plate (different sizes: big, small, medium)
- part 6.2 + Promoter1rev: no colonies on 50 μl-plate, but 8 clones on rest-plate (different sizes: big, small, medium)
- part 6.2 + Promoter3rev: 1 colony on 50 μl-plate (big), and 10 clones on rest-plate (different sizes: big, small, medium)
- pSB1C3 w/o insert: no colonies on 50 μl-plate, 6 clones on rest-plate (small to medium in size)
- pSB1C3 + Promoter1rev: no colonies on 50 μl-plate, no clones on rest-plate
- pSB1C3 + Promoter3rev: no colonies on 50 μl-plate, no clones on rest-plate
- pSB1C3 + RBSrev: no colonies on 50 μl-plate, 5 clones on rest-plate (small, medium to big in size)
-> colony PCR to see, if any of the clones are positive…
-> plates stored at 4°C, big fridge
Colony PCR of clones from transformation of ligations part6.2 + Promoter1rev or Promoter3rev and pSB1C3 + Promoter1rev , Promoter3rev or RBSrev 
- part 6.2 w/o insert: 1 medium – small clone for re-ligand control picked (orange tube, R)
- part 6.2 + Promoter1rev: all clones picked (termed C1 – C8) (blue tubes, 1 – 8)
- part 6.2 + Promoter3rev: all clones picked (termed C1 – C11) (green tubes, 1 – 11)
- contol: plasmid part 6.2 C1 (from 9.8.13, 261.5 ng/μl, dilution 1:20 --> ca. 13 ng/μl) (white tube, p6.2)
- pSB1C3 w/o insert: 1 small clone picked for re-ligand control (termed R4) (pink tube, R)
- pSB1C3 + RBSrev: all clones picked (violet tubes, 1 – 5)
- control: plasmid part 7 C1 (dilution from 15.8.13 with ca. 12 ng/μl used) (white tube, p7)
- colony PCR was done as described previously (10.7.13)
- 30x MasterMix prepared, distribution of 15 μl to 28 tubes, then addition of 1 μl plasmid dilution or inoculation with clones after streak-out on master plates --> master plates incubated over day at 37 °C
- protocol as usual (4 min elongation time should be enough for ca. 1500 bp in case of part6.2 plasmids)
Mini Plasmid Prep of clones inoculated on 15.6.13 and preparation of some cryostocks
- preparation of DMSO cryostocks of CFP C1, YFP C1 and part6.2 C1 Re-trafo clone 1 (done as described before)
- MiniPrep of remaining culture and Terminatorrev and DarRrev clones:
- kit: Nucleospin, Macherey-Nagel (while working, DarRrev C6 column fell on bench --> contaminated?)
- 1st elution with 30 μl pre-warmed HPLC water, incubating at 50 °C for 2 min
- 2nd elution with 22 μl pre-warmed HPLC water, incubating at 50 °C for 2 min
- concentration determination with NanoDrop:
Test Restriction Digest of DarRrev and Termrev clones
- Composition of final reactions: 1 μl PstI FD, 1 μl EcoRI FD, 1 μl 10x FD Green buffer, 2 μl plasmid mix (ca. 200 – 300 ng plasmid), 5 μl dH2O --> in total: 10 μl
- since DarRrev plasmid solutions had such a high concentration, a 1:3 dilution was prepared (1 μl plasmid + 2 μl water) and 2 μl of this dilution used for the RD
- for Termrev plasmids, 2 μl were used, since the concentration was that low
- a MasterMix for 9 reactions was prepared, consisting of the enzymes, the buffer and the water
- the plasmid mix was prepared in the tubes, then 8 μl of the MasterMix were added to the plasmid mix
- the reactions were incubated at 37 °C for 1 h
Gel run for Test Restriction Digest of DarRrev and Termrev clones and Colony PCR of clones from transformation of ligations part6.2 + Promoter1rev or Promoter3rev and pSB1C3 + Promoter1rev , Promoter3rev or RBSrev 
- 4x 1 %-agarose-1xTAE gels poured
- loading of 3 μl 2 log ladder as a marker
- addition of 5 μl 5x LD to the PCR reactions, loading of 5 μl of each reaction
- loading of 5 μl of the RD reactions
- loading of 1 μl uncut plasmid of the RD reactions + 3 μl dH2O + 1 μl LD
- loading of 1 μl of plasmid purified from part6.2 C1 Re-trafo C1 and YFP C1 and CFP C1
- run at 65 – 100 V (the voltage decreased during the run…)
- EtBr staining and destaining in water
- UV detection
 
Gel 1:
Loading: Marker/part 7 plasmid/ pSB1C3+RBSrev Re-ligand R4/pSB1C3+RBSrev C1/ pSB1C3+RBSrev C2/ pSB1C3+RBSrev C3/ pSB1C3+RBSrev C4/ pSB1C3+RBSrev C5/part6.2 C1 plasmid/ part 6.2 Re-ligand/ part 6.2 + Promoter1rev C1/ part 6.2 + Promoter1rev C2/ part 6.2 + Promoter1rev C3/ part 6.2 + Promoter1rev C4/ part 6.2 + Promoter1rev C5/Marker
- expected product sizes (approximately):
part 7 plasmid and RBSrev Re-ligand: 310 bp (VF2, VR) + 130 bp (Terminator) = 440 bp --> band could be detected
RBSrev in pSB1C3: 12 bp + 310 bp (VF2, VR) = 322 bp --> band could be detected for all clones, but there is also the band of the terminator seen --> purify plasmids from C1, C2 and C3 and perform a PCR with the plasmids to see if terminator band is also visible 
part 6.2 C1 plasmid and part 6.2 Re-ligand: 310 bp (VF2, VR) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1251 --> ca. 1250 bp --> band could be detected
part 6.2 C1 plasmid and Promoter1rev: 310 bp (VF2, VR) + 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1371 --> ca. 1370 bp --> C1 to C5 are apparently positive --> plasmid prep of C1, C2 and C3 and test RD
 
 
 
Gel 2:
Loading: Marker/part6.2 C1 plasmid/ part 6.2 Re-ligand/ part 6.2 + Promoter1rev C6/ part 6.2 + Promoter1rev C7/ part 6.2 + Promoter1rev C8/ part 6.2 + Promoter3rev C1/ part 6.2 + Promoter3rev C2/ part 6.2 + Promoter3rev C3/ part 6.2 + Promoter3rev C4/ part 6.2 + Promoter3rev C5/ part 6.2 + Promoter3rev C6/ part 6.2 + Promoter3rev C7/ part 6.2 + Promoter3rev C8/ part 6.2 + Promoter3rev C9/Marker
- expected product sizes (approximately)
part 6.2 C1 plasmid and part 6.2 Re-ligand: 310 bp (VF2, VR) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1251 --> ca. 1250 bp --> band could be detected
part 6.2 C1 plasmid and Promoter1rev/Promoter3rev: 310 bp (VF2, VR) + 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1371 --> ca. 1370 bp --> band is seen for all clones except both C7 (they seem to correspond to re-ligand) --> MiniPrep of C1, C2, C3 and test RD 
 
Gel 3:
Loading: Marker/ part6.2 C1 plasmid/ part 6.2 Re-ligand/ part 6.2 + Promoter3rev C10/ part 6.2 + Promoter3rev C11/ DarRrev C1 uncut plasmid/DarRrev C1 RD/ DarRrev C4 uncut plasmid/DarRrev C4 RD/ DarRrev C6 uncut plasmid/DarRrev C6 RD/ DarRrev C12 uncut plasmid/DarRrev C12 RD/ DarRrev C13 uncut plasmid/DarRrev C13 RD/Marker
- expected product sizes (approximately)
part 6.2 C1 plasmid and part 6.2 Re-ligand: 310 bp (VF2, VR) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1251 --> ca. 1250 bp --> band could be detected
part 6.2 C1 plasmid and Promoter1rev/Promoter3rev: 310 bp (VF2, VR) + 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1371 --> ca. 1370 bp --> band is seen for C10, but not for C11 (C11 seems to correspond to re-ligand)
- test restriction of DarRrev plasmids – expected product sizes (approximately) for positive clone:
2070 bp of pSB1C3 and ca. 690 bp for DarRrev with Pre-/Suffix sequences --> bands are seen for all clones --> prepare C1 and C4 for sequencing 
 
Gel 4:
Loading: Marker/ Termrev C1 uncut plasmid/Termrev C1 RD/ Termrev C2 uncut plasmid/Termrev C2 RD/ Termrev C3 uncut plasmid/Termrev C3 RD/part6.2 C1 Re-trafo C1 plasmid/YFP C1 plasmid/ CFP C1 plasmid/ Marker
- test restriction of Termrev plasmids – expected product sizes (approximately) for positive clone:
2070 bp of pSB1C3 and ca. 190 bp for Termrev with Pre-/Suffix sequences --> bands are seen for all clones --> prepare C1 and C3 for sequencing 
- the plasmids of part 6.2 C1 re-trafo C1, CFP C1 and YFP C1 after MiniPrep look normal
Preparation of samples for sequencing by SeqLab
Since sequences from last week were so good, more or less the same plasmid amount should be used for this sequencing.
- 1 - Termrev C1 + VF2 (6 µl plasmid + 1 µl iGEM_38 1:5)
- 2 – Termrev C3 + VF2 (6 µl plasmid + 1 µl iGEM_38 1:5)
- 3 – DarRrev C1 + VF2 (4 µl plasmid + 2 µl HPLC water + 1 µl iGEM_38 1:5)
- 4 – DarRrev C1 + VR (4 µl plasmid + 2 µl HPLC water + 1 µl iGEM_39 1:5)
- 5 – DarRrev C4 + VF2 (5 µl plasmid + 1 µl HPLC water + 1 µl iGEM_38 1:5)
- 6 – DarRrev C4 + VR (5 µl plasmid + 1 µl HPLC water + 1 µl iGEM_39 1:5)
Fold ↑
Growing of cells used for RNA prep
Growing of cells used for RNA prep
Same   procedure as for the cells used for the array in Groningen (see 25.07.13)
Fold ↑
15th
Terminatorrev transformation, DarRrev transformation, Re-transformation of part 6.2. C1, Transformation of pSB1C3 + Promoter1rev, Promoter3rev or RBSrev, Clones picked from Riboswitch A-D plates from 13.08.13 (Riboswitch in pSB1C3).
Terminatorrev transformation 
- no colonies on NC plates (50 μl and rest-plate)
- w/o insert control plates: no colonies on 50 μl-plate, but some on rest-plate (different sizes)
- Terminatorrev plates: no colonies on 50 μl-plate, but several on rest-plate (different sizes)
- Colony PCR --> use primer combination that indicates, if Terminator is oriented in the desired way in plasmid: 
- 1x w/o insert control plate clone (re-ligand), size: medium – big; primers iGEM_38 (VF2) and iGEM_71 (primer fwd Termrev; PstI and SpeI restriction sites of suffix) --> should give no PCR product, since both primers read into the same direction
- 12 x Terminatorrev plate clones (termed C1 – C12); size: small, medium or big; , primers iGEM_38 (VF2) and iGEM_71 (primer fwd Termrev; PstI and SpeI restriction sites of suffix)
--> should give PCR product if the terminator is oriented reverse (the desired way) of size of terminator with 20 bp of Prefix and 120 additional bp of pSB1C3 backbone caused by VF2;
--> should give no PCR product, if Terminator is oriented in the normal way
- 1x control with part 7 C1 plasmid (from 11.7.13, 120.5 ng/μl --> fresh 1:10 dilution);  primers iGEM_38 (VF2) and iGEM_72 (primer rev Termrev; EcoRI and XbaI restriction sites of prefix) --> should give PCR product of size of terminator (ca. 130 bp) with 20 bp of Prefix and 120 additional bp of pSB1C3 backbone caused by VF2
- procedure: 
part 7 C1 plasmid control was pipetted individually; 1 μl of 1:10 dilution (i.e. ca. 12 ng plasmid) used, therefore, only 17.5 μl dH2O were added
for the reactions containing E. coli cells, a 15x MasterMix was prepared and distributed as usual (see Journal I, for protocol) to 13 epis, which were subsequently inoculated
preparation of a MasterPlate + incubation of plate at 37 °C over day
PCR protocol was the same as usual (iGEM_71 has a TM = 60°C and iGEM_72 has a TM = 59.1°C which are similar to that of iGEM_38 and iGEM_39 with 60 °C --> TA is the same as always (54°C)
- Gel run:
addition of 5 μl of 5x LD to each PCR reaction
pouring of 1%-agarose-1xTAE gel
loading of 3 μl 2 log ladder as a marker
loading of 5 μl of each PCR reaction
run at 100 V
EtBr staining + destaining in water (EtBr bath was not made fresh this weak, so staining was poor; after preparation of new EtBr bath, again few min staining)
UV detection
loading: Marker/part 7/re-ligand/C1/C2/C3/C4/C5/C6/C7/C8/C9/C10/C11/C12/Marker
- for part 7 plasmid template, a band at ca. 270 bp could be observed --> height of terminator PCR product
- for re-ligand, no PCR product of this size is seen
- for C1 – C3, C5 – C12, the expected band is seen --> positive clones --> inoculation of C1, C2 and C3 from Backup plate in 4 ml LBCm for MiniPrep, incubation ON at 37 °C
- backup plate put to 4 °C, big fridge
DarRrev transformation 
- though the transformation was done only yesterday, the plates showed already several colonies:
- Neg. control plates: no colonies, neither on 50 μl-plate nor on rest-plate
- w/o insert plate: no colonies, neither on 50 μl-plate nor on rest-plate
- DarRrev plate: 1 colonie on 50 μl-plate, and many colonies on rest-plate (different sizes; small or medium-big; but clones in general smaller compared to Termrev clones, probably because DarRrev clones grew just overnight)
Colony PCR --> primer combination does not matter, since only 1 orientation of DarR should be possible; additionally, DarR primers make problems during PCR and have a TM diverging from that of VR and VF2) 
- 1x control with part 7 C1 plasmid (from 11.7.13, 120.5 ng/μl --> fresh 1:10 dilution);  primers iGEM_38 (VF2) and iGEM_39  --> should give PCR product of size of terminator (ca. 130 bp) with 2x20 bp of Prefix and Suffix and 120 additional bp of pSB1C3 backbone caused by VF2 and 150 additional bp of pSB1C3 backbone caused by VR (= 440 bp)
- 13x DarRrev clones (termed C1 – C13) primers iGEM_38 (VF2) and iGEM_39  --> should give PCR product of size of DarR (ca. 630 bp) with 2x20 bp of Prefix and Suffix and 120 additional bp of pSB1C3 backbone caused by VF2 and 150 additional bp of pSB1C3 backbone caused by VR (= 940 bp)
 
- procedure: 
15x MasterMix was prepared and distributed as usual (see Journal I, for protocol) to 14 epis,
part 7 C1 plasmid control: To 1 μl plasmid dilution, 25 μl MasterMix were added
25 μl PCR reaction were inoculated with E. coli cells
preparation of a MasterPlate + incubation of plate at 37 °C over day
PCR protocol was the same as usual
- Gel run:
addition of 5 μl of 5x LD to each PCR reaction
pouring of 1%-agarose-1xTAE gel
loading of 3 μl 2 log ladder as a marker
loading of 5 μl of each PCR reaction
run at 100 V
EtBr staining + destaining in water (EtBr bath was not made fresh this weak, so staining was poor; after preparation of new EtBr bath, again few min staining)
UV detection
loading: Marker/part 7/C1/C2/C3/C4/C5/C6/C7/C8/C9/C10/C11/C12/C13/Marker
- for part 7 plasmid template, a band at ca. 440 bp could be observed --> height of terminator PCR product
- for all other clones, a band at ca. 250 bp could be seen, this does not correspond to the expected band of ca. 940 bp (250 bp band is only weak for C 8)
- but a band at ca. 900 bp is observed for C1, C4, C6, C12, and C13
- in addition to that band, several other bands, sometimes stronger are seen for several different clones
-> it could be, that the short primers VF2 and VR bind non-specifically within DarR leading to several other products shorter than the expected one (alignment of the 6 3’ end bases of VR shows that it could bind to the start codon plus the first codon of DarR… this would however only give rise to a product of ca. 140 bp)
-> colony PCR seems to be fruitless --> inoculation of C1, C4, C6, C12 and C13 from backup plates in 4 ml LBCm for MiniPrep, incubation ON at 37 °C
-> backup plate put to 4 °C, big fridge
Re-transformation of part 6.2. C1
- part 6.2. C1 clones grew (few on 50 μl plate; many on rest plate)
- Neg. control plates showed no growth
- 3 clones were picked and streaked out on plate for making a Backup-plate --> incubation ON at 37 °C
- Clone 1 was inoculated in 4ml LBCm and incubated ON at 37 °C for MiniPrep
Transformation of part6.2 + Promoter1rev or Promoter 3rev 
- Neg. control plates showed no growth
- w/o insert control: no colonies on 50 μl plate, several clones on rest plate
- Promoter1rev: no clones on 50 μl plate, 3 clones observed on rest plate
- Promoter3rev: 1 clone on 50 μl plate, 1 clone on rest plate
-> further incubation
Transformation of pSB1C3 + Promoter1rev, Promoter3rev or RBSrev 
- Neg. control plates showed no growth
- w/o insert control: no colonies, neither on 50 μl-plate nor on rest-plate
- Promoter1rev: no colonies, neither on 50 μl-plate nor on rest-plate
- Promoter3rev: no colonies, neither on 50 μl-plate nor on rest-plate
- RBS3rev: no colonies, neither on 50 μl-plate nor on rest-plate 
-> further incubation
Preparation of new LBCm plates 
plates prepared from 2x 500 ml
Preparation of new Cm (35 mg/ml) 
354 mg Cm dissolved in 10.114 ml EtOH 70% dissolved, filter-sterilized, stored in our freezer box – 20 °C
 (calculation of ethanol that has to be added to a certain amount of Cm:
354 mg Cm/x ml EtOH 70 % = 35 mg/ml ó 354 mg/35 mg = x ml EtOH 70%)
Clones picked from Riboswitch A-D plates from 13.08.13 (Riboswitch in pSB1C3)
Ribo A: 10 Clones (Native Promoter and RBS, our most important construct to be cloned tomorrow)
Ribo B-D: 5 Clones
They were inocculated into 4ml Cm Cultures for incubation at 37°C on the shaker.
A Backup plate was plated with all clones
For all clones, a colony PCR was set after the normal protocol:
2.5µl Taq buffer
1µl Taq
1µl NTPs
1µl VF2 Primer (38)
1µl VR Primer (39)
18.5µl H2O
25µl reaction
 Program:
94°C 5 min.
94°C 45s
54°C 40s
72°C 4 min.
72°C 10 min.
15°C HOLD
Fold ↑
Harvest cells
Harvest cells
·         Cells   were harvested with an OD600 = 1
o   wt: OD = 1,06
o   1344: OD = 0,94
o   1346:   OD = 1,06
Fold ↑
14th
Test PCR for specificity of DarRrev Primer/contamination of DarRrev PCR reactions, Transformation of ligation reactions from 13.8.13 and re-trafo of part 6.2 C1 plasmid , Plasmid MiniPrep of part 8 C1, Tidying up our – 20 °C freezer boxes…
Test PCR for specificity of DarRrev Primer/contamination of DarRrev PCR reactions
 
- Neg. control was not negative --> probably, primers or water are contaminated with M. smegmatis chrom.DNA (most likely because of order of pipetting the reactions) or they are non-specific generating another PCR product of the same size as DarRrev (seems very unlikely)
- Test PCR:
- 3 reactions, all reactions were pipetted individually (no mastermix)
- Old water from yesterday and old primer dilutions (1:20) from yesterday used (reaction A)
- Old water from yesterday and new primer dilutions (1:20) from today used (reaction B)
- New water from today and new primer dilutions 1:20 from today used (reaction C)
- All 3 reactions had the same composition as the neg. control reaction yesterday
- PCR program was the same as yesterday
- Gel run:
1%-agarose-1xTAE gel
Loading of 3 µl 2 log ladder as a marker
Loading of 4 µl PCR reaction (A, B, C or neg. control from yesterday’s PCR) + 1 µl 5xLD
Loading of 3 µl purified DarRrev PCR product (from yesterday) + 1 µl dH2O + 1 µl 5xLD
Run at 100 V
EtBr staining + destaining with water
UV detection
Loading: 2 log ladder/reaction A/reaction B/reaction C/neg. control from yesterday/purified DarRrev PCR product from yesterday/ 2 log ladder/ reaction A/reaction B/reaction C/neg. control from yesterday/purified DarR PCR product from yesterday/ 2 log ladder
-> The PCR product (ca. 630 bp + 2x 30bp = 690 bp) is only seen, when all old components are used, but not if new primers and old water are used --> old primer dilutions are contaminated --> dilutions thrown away
-> Since DarRrev-no template control was apparently positive because of not exchanging the pipet tip before adding the primers, the insert obtained from yesterday’s PCR will be further used for cloning…
Transformation of ligation reactions from 13.8.13 and re-trafo of part 6.2 C1 plasmid 
- E.coli XL1-Blue comp. cells used
- According to methods folder
- Neg. control contained 20 µl sterile dH2O
- For re-trafo of part 6.2 C1 plasmid, 1 µl of plasmid from purification on 9.8.13 was used
- For transformation of ligations, the whole ligation reactions were pipetted to the comp. cells
- After centrifugation, 500 µl supernatant were removed; strangely, the epi with pSB1C3+Prom1rev contained less than 500 µl solution (but cell pellet was at the bottom of the tube and the colour corresponded to that of LB + comp. cells seen in all other tubes… possibly less comp. cells?)
- Plating of all reactions on LBCm
- Incubation at 37 °C
Plasmid MiniPrep of part 8 C1
 
- Culture at > 4 ml…
- Elution
1st: 30 µl HPLC-water (pre-warmed) while incubating at 50 °C for 2 min
2nd: 22 µl HPLC-water (pre-warmed) while incubating at 50 °C for 2 min
Both elutions were collected in the same tube
- Concentration (NanoDrop)
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Part 8 C1 plasmid
152.2
1.78
1.35
Tidying up our – 20 °C freezer boxes…
 
- I found a tube where the date and the concentration where difficult to decipher. It was part 7 C1 plasmid. It could be the plasmid purified by myself on 12.7.13 with a concentration of 179.3 ng/µl. But since I wasn’t sure about that I measured the concentration (NanoDrop) again:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Part 7 C1 plasmid
144.1
1.82
1.37
 - We still had the senseless RBS-GFP-Terminator clones; I threw them away, as well…
Fold ↑
13th
P3op + pSB1C3 C1: Cryo-Stock and MiniPrep, DarRrev PCR repeated – with Phusion® and GC buffer, EcoRI and PstI double digest of purified DarRrev PCR product , XbaI-digest of part6.2. C1 linearized with EcoRI , PCR clean-up of XbaI- and EcoRI-digested part6.2 C1 plasmid , Hybridization of oligos for Promoter3rev, Promoter1rev and RBSrev
P3op + pSB1C3 C1: Cryo-Stock and MiniPrep
- preparation of DMSO cryo-stock (900 μl ON culture + 100 μl DMSO 100%, vortexing, put directly to – 80 °C in our box)
- Mini Plasmid Preparation of remaining ON culture using Nucleospin kit from Macherey-Nagel (I accidently used 750 μl A4 instead of 600 μl for washing of DNA on column); elution 1x with 30 μl HPLC-water (pre-warmed), incubation at 50 °C for 1 – 2 min)
- NanoDrop concentration measurement
Sample
ng/μl
A260nm/A280nm
A260nm/A230nm
P3op in pSB1C3 C1 plasmid
151.0
1.92
1.79
 - stored in to-do-Box
DarRrev PCR – Gel run
- 1st try: loading of gel was difficult because of deformed wells --> some sample was spilled --> but this should be a semi-quantitative gel --> gel thrown away and new gel was poured and loaded
- 2nd try:
1% Agarose-1xTAE gel
loading of 3 μl 2 log ladder
loading of 4 μl of each PCR reaction + 1 μl 5xLD
run at 100 V
EtBr staining + destaining in water
UV detection
 Loading: Marker/Negative control/ 2 μl primer reaction/ 4 μl primer reaction/ 8 μl primer reaction/---
-> for 2 μl primer reaction and 4 μl primer reaction, a very weak band at ca 700 bp (approx. bp of DarRrev PCR product: 630 bp + 2x 30 bp = 690 bp) was detected
-> strength of this band seemed to decrease with increasing primer amounts
-> PCR did not work properly: M. smegmatis has high GC content, but normal HF buffer and PfuS were used --> try comercial Phusion® and GC buffer, only testing of 2 and 4 μl primer amount (lower concentration seem to be more optimal…
 
DarRrev PCR repeated – with Phusion® and GC buffer 
- Reactions pipetted as before (see 12.8.13, DarRrev PCR with normal PfuS and HF buffer)
- Volume 50 μl
- 4x Master Mix consiting of 5x GC buffer, water, Phusion® and dNTPs
- preparation of template (dH2O for neg. control, M. smegmatis chrom.DNA for other reactions), primers (iGEM_83, iGEM_84 and a part of water for 2 μl primer and neg. control reaction); then, addition of required volume of mastermix
- Testing of 2 and 4 μl primer; neg. control (dH2O as template and 2 μl as primers)
- PCR protocol: the same as for PfuS in HF buffer
1% Agarose-1xTAE gel
loading of 3 μl 2 log ladder
loading of 4 μl of each PCR reaction + 1 μl 5xLD
run at 100 V
EtBr staining + destaining in water
UV detection
Loading: Marker/Negative control/ 2 μl primer reaction/ 4 μl primer reaction
-> wells of gel were somehow damaged: this could explain weak band observed for neg. control (some of 2 μl sample ran over well into well of neg. control…) --> same gel as used for control of part6.2 C1 XbaI digest (see below and to check first DarRrev PCR using PfuS and HF buffer)
-> bands of expected product size for both reactions, 2 μl primers and 4 μl primers. But one cannot say, if 4 μl reaction contains really more product than 2 μl reactions since wells were so strange…
-> pool and purifiy 2 μl primer and 4 μl primer reaction, digest the PCR product and run it again with the neg. control on a proper gel
 
PCR clean-up of DarRrev 2 μl primers and 4 μl primer reactions 
- both reactions were pooled (ca. 100 μl)
- addition of 500 μl PB buffer
- elution with 1st 30 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min; 2nd 22 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min
- NanoDrop concentration
Sample
ng/μl
A260nm/A280nm
A260nm/A230nm
DarRrev PCR product
10.6
2.20
1.50
 - Stored in DarR reporter system-Box
 
EcoRI and PstI double digest of purified DarRrev PCR product 
- 30 μl DarRrev PCR product (purified today) + 3 μl EcoRI FD + 3 μl PstI FD + 4 μl 10x FD buffer = 40 μl digest reaction
- digest at 37 °C (heating block showed sometimes 37.7 or 37.8 °C though it was set to 37 °C) for 1 h
 
Gel run
- 1% Agarose-1xTAE gel
- loading of 3 μl 2 log ladder
- loading of 4 μl neg. control PCR reaction + 1 μl 5xLD
- loading of 3 μl of RD reaction + 1 μl 5xLD + 1 μl dH2O
- loading of 3 μl of uncut PCR product + 1 μl 5xLD + 1 μl dH2O
- (loading of samples for XbaI digest of part6.2 C1, that was already digested with EcoRI, as well – see below)
- run at 100 V
- EtBr staining + destaining in water
- UV detection
 
Loading: Marker/neg.control from PCR/uncut purified PCR product/ RD reaction of PCR product/ uncut plasmid part 6.2 C1/ RD reaction plasmid part 6.2 C1 E+ P digest, purified/---
-> Negative control is not negative (repeat neg. control PCR tomorrow, to see, if it’s really not negative, but purifiy and ligate DarRrev insert)
-> After R.D., PCR product is still seen (no over-digestion)
 
PCR clean-up of DarRrev PCR product
- With Qiagen PCR purification kit
- 500 µl PB buffer used
- Elution with 30 µl pre-warmed HPLC water, incubating sample for 2 min at 50 °C
- NanoDrop concentration measurement:
Sample
ng/μl
A260nm/A280nm
A260nm/A230nm
DarRrev insert
6.0
1.50
1.67
- Stored in DarR reporter system-Box
XbaI-digest of part6.2. C1 linearized with EcoRI 
- ca. 27 μl (mostly) linearized part6.2 C1 from 12.8.13
- addition of 4 μl FD buffer 10x, 4 μl XbaI FD, 5 μl dH2O
- in total: 40 μl
- digest for 2 h at 37 °C (after 1 h, it has been incomplete up till now…)
Gel run
- 1% Agarose-1xTAE gel
- loading of 3 μl 2 log ladder
- loading of 3 μl of RD reaction + 1 μl 5xLD + 1 μl dH2O
- loading of 1 μl of uncut plasmid + 1 μl 5xLD + 3 μl dH2O
- run at 100 V
- EtBr staining + destaining in water
- UV detection
Loading: Marker/uncut plasmid/ RD reaction
-> digest is still incomplete (linearized plasimd at ca. 3 kb), even after 2 h XbaI digest. Band of uncut supercoil plasmid very weak (runs at ca. 2 kb)--> purifiy vector for ligation
-> repeat gel run on proper gel (wells were apparently damaged; see DarRrev PCR gel run) --> same gel as used for control of DarRrev PCR reactions (see above; PCR using PfuS and HF buffer, but also the PCR reaction with Phusion® and GC buffer) --> apparently, the last part of the gel had damaged wells (fault of comb? Or because of multiple runs?)
 
PCR clean-up of XbaI- and EcoRI-digested part6.2 C1 plasmid 
- addition of 500 μl PB buffer
- elution with 1st 30 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min; 2nd 22 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min
- NanoDrop concentration
Sample
ng/μl
A260nm/A280nm
A260nm/A230nm
part6.2 C1 vector EcoRI + XbaI digested
15.6
1.75
2.07
  
Repeat of gel run (see above)
- 1% Agarose-1xTAE gel
- loading of 3 μl 2 log ladder
- loading of 3 μl of RD reaction + 1 μl 5xLD + 1 μl dH2O
- loading of 1 μl of uncut plasmid + 1 μl 5xLD + 3 μl dH2O
- (loading of samples for DarRrev PCR, as well (see above))
- run at 100 V
- EtBr staining + destaining in water
- UV detection
 
Loading: Marker/neg.control from PCR/uncut purified PCR product/ RD reaction of PCR product/ uncut plasmid part 6.2 C1/ RD reaction plasmid part 6.2 C1 E+ P digest, purified/---
-> Same result as before (see above), but at least a bit nicer (though still smeary…) 
- Stored in DarR reporter system-Box
Hybridization of oligos for Promoter3rev, Promoter1rev and RBSrev 
- Oligos iGEM_73, iGEM_78 (RBSrev)
- Oligos iGEM_74; iGEM_79 (Promoter1rev)
- Oligos iGEM_76; iGEM_81 (Promoter3rev)
1. Oligos dissolved in required amount of HPLC water
2. 10 µl of each oligo of one hybridization pair were mixed
3. incubation for 10 min at 80 °C (for RBSrev oligos --> TM is below 80 °C) or at 98 °C (for Promoter oligos --> TM is above 80 °C)
4. after 10 min, heating blocks were switched off; the mixtures were left in the heating block until it was cooled down to ca. RT.
--> can be used directly for ligation
- Hybridizations stored in DarR reporter system-Box
- Oligos stored in Primer-Box
Ligation of
a) pSB1C3 E+P with DarR insert E + P
b) pSB1C3 E+S with Promoter1rev, Promoter3rev, or RBSrev
c) part6.2 C1 E+X vector with Promoter1rev or Promoter3rev 
Total volume of ligations: 20 µl
Ratio for DarRrev ligation: vector:insert = 1:3 --> for 50 ng vector of 2070 bp, use 50 ng insert of 690 bp (online ligation calculator)
Ligation a)
 
w/o insert
DarRrev
Vector (pSB1C3 EcoRI- and PstI-digested, 56.6 ng/µl; 31.7.13)
1 µl
1 µl
Insert (DarRrev from today) or dH2O for w/o insert control
9 µl
9 µl
T4 ligase (Thermo Scientific)
2 µl
2 µl
10x T4 buffer (Thermo Scientific)
2 µl
2 µl
dH2O
 
6 µl
6 µl
Total
20 µl
20 µl
 
Ligation b)
 
w/o insert
Promoter1rev
Promoter3rev
RBSrev
Vector (pSB1C3 EcoRI- and SpeI-digested, 6.4 ng/µl; 30.7.13)
8 µl
8 µl
8 µl
8 µl
Insert (Hybridization oligos from today) or dH2O for w/o insert control
8 µl
8 µl
8 µl
8 µl
T4 ligase (Thermo Scientific)
2 µl
2 µl
2 µl
2 µl
10x T4 buffer (Thermo Scientific)
2 µl
2 µl
2 µl
2 µl
dH2O
-
-
-
-
Total
20 µl
20 µl
20 µl
20 µl
 
Ligation c)
 
w/o insert
Promoter1rev
Promoter3rev
Vector (part6.2 C1 EcoRI- and XbaI-digested, 15.6 ng/µl; from today)
3.5 µl
3.5 µl
3.5 µl
Insert (Hybridization oligos from today) or dH2O for w/o insert control
10 µl
10 µl
10 µl
T4 ligase (Thermo Scientific)
2 µl
2 µl
2 µl
10x T4 buffer (Thermo Scientific)
2 µl
2 µl
2 µl
dH2O
2.5 µl
2.5 µl
2.5 µl
Total
20 µl
20 µl
20 µl
- All reactions were pipetted individually, no mastermix
- Incubation ON at 16 °C (cold room)
Transformation of ligation reaction from yesterday (pSB1C3 + Terminatorrev) 
- According to methods folder, but instead of removing 400 µl after plating 50 µl and centrifugation, 500 µl supernatant were removed
- Negative control: addition of 10 µl sterile dH2O
- Plating on LBCm  plates (35 µg/ml)
- Incubation at 37 °C
Preparation of LB Agar for Chloramphenicol plates
shopping and ordering in cellar
Trafo of Ribo A-D from yesterday´s ligation
protocol followed except for streak out
streak out in 3 steps: first 50 μl, spin down, than 500 μl to waste, resuspend and streak out again in 50 μl and the rest (ca. 150 μl ) on another plate
3 plates per transformation
6 transformation (4 Ribos (A-D), 1 religation, 1 negativ control with 20 μl H2O)
Fold ↑
12th
Terminatorrev PCR – Gel ex, Terminatorrev PCR – Repeated, Restriction of Part 6.2 C1 plasmid with EcoRI to generate vector for ligation with hybridization oligos of promoters, Restriction of Terminatorrev PCR product obtained after gel ex with EcoRI and PstI (for ligation with pSB1C3), Ligation of pSB1C3 (E+P-digested) with Terminatorrev insert (E+P digested),
Gel and Nanodrop measurements of R.D. from friday (Jonathan)
Gel:
M|CFP|CFPr.d.|YFP|YFPr.d.|Part1|Part1r.d.|Part2|Part2r.d.|Part3|Part3r.d.|Part4|Part4r.d.|RiboA|RiboAr.d.
M|RiboD|RiboDr.d.|RiboC|RiboCr.d.
àRiboB was forgotten
Nanodrop:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
part8 E+X purified 9.8.
14.1
1.84
2.13
CFP E+X purified 9.8
27.1
1.68
1.48
YFP E+X purified 9.8
19.6
2.03
2.22
part1 E+X purified 9.8
26.7
1.69
1.49
part2 E+X purified 9.8
22.6
1.89
1.95
part3 E+X purified 9.8
30.0
1.89
1.99
part4 E+X purified 9.8
39.2
1.89
1.89
Rino A E+X purified 9.8
31.0
1.89
1.93
Ribo D E+X purified 9.8
32.7
1.98
2.27
Ribo C E+X purified 9.8
29.1
1.92
1.95
Ribo B E+X purified 9.8.
30.9
1.51
2.79
 
 
New inoculation of part 8 in Lb Amp
 
Inocculation of GFP clones (part 6.2 clones 1-7)
for cryostocks and a masterplate, which was forgotten previously
Terminatorrev PCR – Gel ex
 -> Terminatorrev PCR from 7.8.13 looks a bit strange (band in neg. control at height of expected PCR product); it could be the primers, however, since the plasmid used as a template is still present in the sample, PCR clean up (as it was done) might not be enough to purifiy PCR product from template. Thus, gel extraction needs to be performed.
- Addition of <7.5 μl to each of the purified samples
- Loading of entire samples on 1 %-agarose-1xTAE-gel
- Loading of 2 log ladder (3 μl)
- Loading of part 7 C1 plasmid as a control (1 μl of 120.5 ng/μl + 1 µl 5xLD + 3 µl water)
- Run at 85 V
 - Gel extraction + purification using Qiagen PCR clean-up/gel extraction kit
- since Terminatorrev PCR product is ca. 190 bp, 1 volume isopropanol was added after the gel pieces were dissolved
- 2 columns used (loading of sample amounts corresponding to max. 400 mg gel in total per column); when loading one column, I pipetted some of the sample onto the TARA-tube by mistake… so this part of the sample was lost…
- elution from both columns with 30 μl HPLC water each into the same tube
- concentration (NanoDrop):
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Termrev PCR product gel ex
16.5
1.60
0.26
 
Terminatorrev PCR – Repeated
 - since PCR from 7.8.13 had a strange negative control, the PCR was repeated as on 7.8.13, but with some modifications:
- with only 2 μl primers (iGEM_71/72)
- 5 reactions in total:
a) neg. control 1: no primers --> which bands are caused by primers?
b) neg. control 2: no template --> which bands are caused by template?
c - e) 3 reactions with template and primers
- template: part 7 C1 plasmid, dilution from 7.8.13 used
- 6x MasterMix containing PfuS, 5x HF buffer, water and dNTPs
- after PCR: 3 reactions with template and primers pooled into 1 tube
- 1%-agarose-1xTAE gel (from Nina)
- loading of 3 μl 2 log ladder
- loading of 4 μl of each PCR reaction + 1 μl 5xLD
- Run at 100 V
- EtBr staining + destaining in water
- UV detection
- samples ran almost out --> no conclusion possible --> gel run repeated:
 
- gel run:
- 1%-agarose-1xTAE gel
- loading of 3 μl 2 log ladder
- loading of 4 μl of each PCR reaction + 1 μl 5xLD
- (Loading of Terminatorrev PCR product digest, as well – see below; loading of digest of part 6.2 C1 plasmid with EcoRI, too – see below)
- Run at 100 V
- EtBr staining + destaining in water
- UV detection
-> band observed in neg. control results indeed from primers --> one can work with the old PCR product
-> green PCR tube with pooled PCR reactions stored in 15 ml Falcon tube at – 20°C for purification
 
Restriction of Part 6.2 C1 plasmid with EcoRI to generate vector for ligation with hybridization oligos of promoters
1500 ng plasmid (6 μl of 261.5 ng/μl sample; purified on 9.8.13)
4 μl EcoRI FD
4 μl 10x FD buffer
26 μl water
in total 40 μl
 - digest for 1 h at 37 °C
 
- gel run:
- 1 %-agarose-1xTAE gel
- Loading of 3 μl 2 log ladder
- Loading of 1 μl plasmid (uncut) + 3 μl water + 1 μl 5x LD
- Loading of 3 μl RD reaction + 1 μl water + 1 μl 5x LD
- Run at 100 V
- EtBr staining + destaining in water
- UV detection
-> digest is still incomplete
-> addition of 1 μl EcoRI FD, 1 μl 10x FD buffer, 8 μl water
-> digest for another 1 h at 37 °C
 
- gel run:
- 1%-agarose-1xTAE gel
- Loading of 3 μl 2 log ladder
- Loading of 1 μl uncut plasmid + 3 μl water + 1 μl 5x LD
- Loading of 4 μl RD reaction + 1 μl 5x LD
- (Loading of Terminatorrev PCR product digest, as well – see below; repeat of gel run of new Terminatorrev PCR, as well – see above)
- Run at 100 V
- EtBr staining + destaining in water
- UV detection
-> Digest only slightly incomplete --> tolerable --> purify vector for further digest with XbaI with PCR clean-up kit
 
Purification:
 - Qiagen PCR purification kit
- 600 µl PB buffer used (by mistake…)
- Elution 1x with 30 µl HPLC-water (pre-warmed) with incubation at 50 °C for 2 min
- NanoDrop: concentration
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Part6.2 C1 vector + EcoRI (after PCR clean-up)
34.9
1.86
2.14
- Samples stored in To-do-Box
Restriction of Terminatorrev PCR product obtained after gel ex with EcoRI and PstI (for ligation with pSB1C3)
 
30 μl PCR product
3 μl EcoRI FD
3 μl PstI FD
4 μl 10x FD buffer
40 μl in total
- digest for 1 h at 37 °C
- gel run:
- 1%-agarose-1xTAE gel
- Loading of 3 μl 2 log ladder
- Loading of 4 μl uncut PCR product + 1 μl 5x LD
- Loading of 4 μl RD reaction + 1 μl 5x LD
- (Loading of part6.2 C1 plasmid EcoRI 2 h digest, as well – see above; repeat of gel run of new Terminatorrev PCR, as well – see above)
- Run at 100 V
- EtBr staining + destaining in water
- UV detection
-> PCR product still present/ no over-digest --> PCR clean-up to purify insert for ligation
 
Purification: 
- Qiagen PCR purification kit
- 500 µl PB buffer used
- Elution 1x with 30 µl HPLC-water (pre-warmed) with incubation at 50 °C for 2 min
- NanoDrop: concentration
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Termrev insert (after PCR clean-up)
15.0
1.91
2.17
- Samples stored in To-do-Box
DarRrev PCR: Test-PCR for optimal primer concentration 
- DarRrev primers (iGEM_83 and iGEM_84) arrived --> dissolved in HPLC-water and preparation of 1:20 dilution in HPLC water
- iGEM_83 and iGEM_84 tend to form strong secondary structures --> test for optimal primer concentration in PCR
- Test- PCR done as on 24.6.13: 5x MasterMix containing water, 5x HF buffer, PfuS, dNTPs; 2 μl, 4 μl and 8 μl primer tested, neg. control w/o template (M.smegmatis chrom.DNA); Annealing temperature of both primers is the same as for iGEM_33 and iGEM_44, since only the prefix and suffix sequences were switched)
- PCR taken out and stored at 4 °C (small fridge) in yellow-tip-box-rag
Sequencing results of Promoter3op cloning in front of part 6 and into shipping vector 
-> Plasmids part 6.2. C1 and part 6.2. C2 contain the Promoter3op insert at the desired site in the vector w/o any mutations --> part 6.2 C1 used for further cloning
->Plasmid Promoter3op in pSB1C3 C1 (see: Colony PCR from 8.8.13) contains the Promoter3op insert at the desired site in the vector w/o any mutations --> inoculation of C1 in 4 ml LBCm for MiniPrep and CryoStock, incubation ON at 37 °C, ca. 200 – 210 rpm
Ligation of pSB1C3 (E+P-digested) with Terminatorrev insert (E+P digested)
 
- Ligation calculator: for ratio vector: insert= 1:3 and 50 ng vector (2070 bp), ca. 14 ng insert (190 bp) are required 
Component
Ligation
Termrev insert/dH2O (w/o insert control) from today
1 µl (15.0 ng/µl)
T4 Ligase (ThermoScientific)
1 µl
T4 ligation buffer 10x
1 µl
pSB1C3 vectorE+P digested from 31.7.13
1 µl (56.6 ng/µl)
dH2O
6 µl
Total
10 µl
 
-> Ligation reactions incubated ON at 16 °C (cold room)
Preparation of LBCm plates 
- From 500 ml medium
- 35 µg/ml Cm in plates
Ligation of Riboswitch Constructs A-D into pSB1C3(from part 7)
 
For that, ~45ng Vector were ligated to ~150ng Inserts as follows:
                  1.5µl pSB1C3
                  4.5µl Insert (Ribo A-D)
                  2µl T4 Buffer
                  2µl T4 Ligase
                  10µl H20
         = 20µl reaction
Incubation overnight at 16°C
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w11 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
