"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w9
From 2013.igem.org
02nd
Colony PCR from 1.8.13 (pBluescript cloning of DarR and Riboswitches): Gel run + MiniPrep,   Transformation (pSB1C3 cloning) from 31.7.13 and Comp.Cells-Test, Ligation of pSB1C3 with Riboswitch inserts.
Colony PCR from 1.8.13 (pBluescript cloning of DarR and Riboswitches): Gel run + MiniPrep
-          Backup plate for pBluescript clones 1 – 3 and Master Plate for Colony PCR stored at 4 °C (big fridge)
-          1 % agarose-1xTAE gel (3 gels)
-          Addition of 5 µl 5x LD to 25 µl PCR reaction
-          Loading of 5 µl of each reaction on agarose gel
-          Loading of 3 µl of 2 log ladder
-          Run at (70 - ) 100 V
-          EtBr staining + destaining in water
-          UV detection
-          Reactions stored at 4 °C in yellow-tip-box rag (small fridge)
 
Gel 1
-> pBluescript plasmid gave band at ca. 200 bp -> a band at the PCR product sizes of DarR and riboswitches + 300 bp is expected, if DarR and riboswitches are ligated into pBluescript
-> pBluescript C1 seems to contain pBluescript plasmid (band at ca. 200 bp)
-> No inserts in all Riboswitch clones or DarR clones, except for DarR C1 and C4. However, the PCR products are between 2 and 3 kb. This is far too large for the expected product size of > 1 kb -> probably 1 or more inserts ligated into vector?
-> Some of the clones obtained from the transformation of the ligation reactions seem to contain normal pBluescript -> possibly, they were light blue or not yet blue when they were picked (band at ca. 200 bp; e.g. for C3 of w/o insert control) (when looking at all pBluescript plates again, I noticed, that we counted several blue ones as white clones… maybe it’s better, to put the plates for 1 ON to 4 °C before counting and picking…
-> There were many, many blue clones on the plates indicating that pBluescript vector was not digested properly (DAC team did RD of pBluescript possibly in a double digest…) -> repeat digest in two rounds of single digest: 1. EcoRI; 2. PstI. This order is important, since pBluescript contains two PstI restriction sites. One is ca. 340 bp upstream of the EcoRI restriction site, the other one just 2 bp. Thus, in a double digest, the enzymes might inhibit each other at this second PstI site (EcoRI would not cut properly). Additionally, if one would digest first using PstI, there would be just a 2 bp overhang left at the EcoRI site. But restriction enzymes usually need a longer overhang for proper binding and cleaving of the DNA.
 
-> Cloning did not work again.
 
-> All clones harvested for the MiniPrep were thrown away, except for pBluescript C1 and DarR C1 and C4 (we want to know what’s inside…)
 
-> When discussing the gels, Katrin and I noticed that there might be a too short overhang after the EcoRI and PstI restriction sites in the prefix and suffix sites for the enzymes to cut -> Insert is not restricted at the ends -> no ligation can occur with the properly digested vector. We checked it, and, indeed, the restriction sites were directly at the ends of the primers used for amplification of the inserts (no overhang at all!). On the iGEM web page, there are the actual Prefix and Suffix sequences with the overhangs preferred by EcoRI and PstI. But they were not entered into the “Primers List”. Instead, the pure Suffix and Prefix sites were entered -> This means, that all our primers contain the wrong prefix and suffix sites. So, we have order new primers containing the overhangs or to try amplify the inserts with Taq (this enzyme adds polyA to the ends of the PCR products, thereby generating an overhang).
 
Transformation (pSB1C3 cloning) from 31.7.13 and Comp.Cells-Test
 
Transformation of pSB1C3 cloning:
-          colonies only on some plates
-          However, they look more like a contamination than like E. coli
-          Negative control: no contamination or E. coli colonies
 
-> Plates thrown away
 
Comp. Cells test:
-          LBAmp , LBCm , LBAmp+X-Gal : No colonies of E. coli on plates, but contamination on one of the plates (possibly Amp plate?) -> untransformed E. coli cannot grow on the plates
-          Rest plate of pBluescript transformation: Colonies became more blue, white big colonies in between indicating contamination (looks like contamination on antibiotic resistance test plate…)
-> Competent cells are fine.
Contamination source:
-          The contamination on the antibiotics plate was only observed today (but not yesterday, at least on the antibiotics test plate (for pBluescript rest, it could be that there was already yesterday something…).
-          It was likewise not seen in other transformations done before with the same comp. cells, except for the transformation done one week ago by Jonathan, Dominik and Bingyao.
-          It was also not found on the 50 µl plate of the pBluescript transformation or in the negative controls.
-          The contamination is also seen on the pSB1C3 transformation, but only on few plates.
-> Hence, it could be that the contamination derives from some other source than the comp. cells themselves i.e. unsterile pipette tips and glass pipettes or even the paper box in which the plates were incubated. Thus, the whole sterile bench was cleaned and all tips and glass pipettes removed.
Taq polymerase PCR of Riboswitches
-          Used primer combinations and programs: 42/43 (Ribo1 Taq); 42/41 (Ribo2 Taq); 40/41 (Ribo3 Taq); 40/43 (Ribo4 Taq)
-          Standard Taq Polymerase PCR Protocol used, except for double amount of primer for primer 43
-          Template: gDNA of B. subitilis
-          each setup 5 times (1 MM for each setup)
-          Colorcodes of the PCR tubes:
o    Ribo 1: WHITE
o    Ribo2: PURPLE
o    Ribo3: ORANGE
o    Ribo4: YELLOW
 
PolyA overhang of DarR
-          Setup: 150ng of template
-          0,2 mM dATP
-          1x Taq-Buffer
-          1 U Taq polymerase
-          30 min in Cycler at 70°C
 
-          Template: DarR PCR Purified 2 24.7 by Joni (17ng/ µl)
-          9 µl Template
-          1 µl 0,2 mM dATP
-          1 µl 10x Taq Buffer
-          1 µl Taq Pol
 
pooled and put into freezer by Katrin G..
 
Miniprep of DarR C1, DarR C4 and pBluescript plasmids
NanoDrop values:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
pblue
342,0
1,89
2,26
DarR 1
402,1
1,87
2,24
DarR 4
401,3
1,87
2,24
 
Restriction digestion of pbluescript
1500ng (4,4 µl) plasmid
4 µl EcoRI
4 µl Buffer
28µl dH2O
digested for 1h at 37°C
Test restriction digestion of DarR C1 and DarR C4 (respectively)
300ng (1µl) plasmid
4 µl EcoRI
4 µl PstI
4 µl Buffer
27 µl dH2O
digested for 1h at 37°C
 
Gel doc:
pblue, DarR1, DarR4
pblue is not completely linearized -> has to be further digested on monday (is stored in to do box)
 
-> Actually, one cannot conclude anything from this gel, since one does not know at which height the different forms of the uncut plasmid are running. Remember: ALWAYS load 30 ng or more (since after MiniPrep we usually have 100 ng or more of plasmid, it’s enough to load 1 µl. But make first sure, if this is really the case!) of uncut vector as a control.
Repeat of gel run for pBluescript and DarR C1 and C4 plasmid RD and gel for riboswitch PCR
-          1.5% agarose-1xTAE-gel (1.5 g agarose in 100 ml 1xTAE)
-          Loading of 1 µl pBluescript plasmid/DarR C1/DarR C4 plasmid + 3 µl dH₂O + 1 µl 5xLD
-          Loading of 4 µl RD reaction/PCR reaction (after pooling them) + 1 µl 5xLD
-          Loading of 3 µl 2 log ladder
-          Run at 90 V
-          EtBr staining + destaining in water
-          UV detection
 
Gel:
<
->  pBluescript digest is incomplete -> further digest
->  DarR C1 and C4 plasmids seem to be fully digested -> C1 insert is slightly smaller than the vector: no self-ligation, but possibly multiple inserts?; C4 insert has same bands as pBluescript RD, though bands of uncut C4 plasmid has different bands: No explanation…? Samples switched???
->  PCR of riboswitches: the expected bands were more or less (maybe slightly too high?) observed -> purification and digest to generate inserts
 
Purification and Digest (EcoRI and PstI) of riboswitch PCR products to generate inserts
-          Purification using Qiagen PCR clean-up kit (ca. 250 µl of PCR reaction -> addition of 1.250 ml PB buffer)
-          Elution with pre-warmed HPLC water (1st elution: 40 µl, 2nd elution (eluted into same tube already containing the 40 µl 1st elution!): 30 µl)
-          Concentration (NanoDrop)
Purified PCR…
ng/µl
A260nm/A280nm
A260nm/A230nm
Riboswitch iGEM_40/41
38.2
1.81
1.99
Riboswitch iGEM_40/43
66.7
1.85
2.23
Riboswitch iGEM_42/41
40.2
1.78
2.03
Riboswitch iGEM_42/43
58.6
1.83
2.15
 
 
-          Digest (E+P)
All reactions were pipetted independently (no MasterMix) 
Component
Amount
PCR product
30 µl
EcoRI FD
3 µl
PstI FD
3 µl
10x FD buffer
4 µl
Total
40 µl
 
->  1 h, 37 °C
-     Gel run (1 % agarose-1xTAE)
Loading of 3 µl 2 log ladder
Loading of 1 µl uncut purified PCR product + 3 µl dH₂O + 1 µl 5xLD
Loading of 4 µl RD reaction + 1 µl 5xLD
Run at 100 V
EtBr staining + destaining in water
UV detection
 
Gel:
->  Inserts still present at expected bp -> purification and ligation with pSB1C3 (E+P digested)
 
-          Purification of reactions using Qiagen PCR clean-up kit (500 µl PB buffer used; elution with 2x 22 µl HPLC H₂O into the same tube)
-          Concentration (NanoDrop)
Insert…
ng/µl
A260nm/A280nm
A260nm/A230nm
Riboswitch iGEM_40/41
20.9
1.91
1.82
Riboswitch iGEM_40/43
36.1
1.81
2.05
Riboswitch iGEM_42/41
23.6
1.83
1.63
Riboswitch iGEM_42/43
32.5
1.80
2.02
 
Ligation of pSB1C3 with Riboswitch inserts
-          Ratio Vector : insert = 1:3 (calculated with Uni D’dorf online ligation calculator)
50 ng vector, 2070 bo
Insert…
bp used for calculation
Amount for ligation with 50 ng pSB1C3 vector
ng/µl
Riboswitch iGEM_40/41
213
15 ng
20.9
Riboswitch iGEM_40/43
403
29 ng
36.1
Riboswitch iGEM_42/41
370
27 ng
23.6
Riboswitch iGEM_42/43
466
34 ng
32.5
 
->  For each reaction, 1 µl insert and 1 µl pSB1C3 E+P vector (56.6 ng/µl) were used: 
-          Reaction
Component
1x
MasterMix 6x
pSB1C3
1µl
6 µl
T4 ligase (ThermoScientific)
1 µl
6 µl
T4 ligation buffer 10x
1 µl
6 µl
dH2O
6 µl
36 µl
Insert
1 µl
-
Total
10 µl
54 µl
 
->  1 µl of insert or dH₂O (for w/o insert control) + 9 µl MasterMix
->  Incubation ON at 16 °C
Fold ↑
01st
Restriction digestion of these riboswitch PCR products:, Competent Cells-Test, Transformation of pBluescript and pSB1C3 ligations from 31.7.13, Colony PCR for pBluescript cloning on 31.7.13, Transformation of parts BBa_E0030 and BBa_E0020.
Pooling of the PCR products from yesterdays riboswitch and new NanoDrop measurement
 
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
42/41
8.9
1.46
0.90
42/43
15.1
1.52
1.23
40/41
11.0
1.51
1.72
40/43
14.0
1.92
1.86
 
Gel doc:
42/41, 42/43, 40/41, 40/43
 
Restriction digestion of these riboswitch PCR products:
EcoRI +PstI
for 42/43, 40/41, 40/43
30µl PCR product
3µl EcoRI
3µl PstI
4µl FD Buffer
7µl H2O
 
for 42/41:
37µl PCR product
3µl EcoRI
3µl PstI
4µl FD Buffer
7µl H2O
 
EcoRI +SpeI
for 42/43, 40/41, 40/43
30µl PCR product
3µl EcoRI
3µl PstI
4µl FD Buffer
7µl H2O
 
for 42/41:
37µl PCR product
3µl EcoRI
3µl PstI
4µl FD Buffer
7µl H2O
digested for 30min at 37° (or 1h?)
 
Gel run:
Pouring of 1% agarose gel
Loading of 4 µl supplied with1 µl 5x LD of each digest
Loading of < 4 µl of the remaining uncut purified PCR products supplied with 1 µl 5xLD (just 40/41, 40/43 and 43/42 left…)
Loading of 3 µl log ladder
Run at 100 V
EtBr staining and de-staining
UV detection
Gel:
->  Strangely, the relationship between the length of the different inserts fits on their own. However, they are running at a much lower height than expected (expected 200 – 500; observed: 100 – 200…) -> something strange happened during the PCR, because already on the gel to see if the PCR worked, the bands are too low… -> Wait for Colony PCR (see below); if all clones negative, repeat PCR
 
->  Reactions stored at – 20 °C in one of our pink boxes
 
 
Preparation of LB plates
1x 500 ml LBChl
 
Competent Cells-Test
-          No colonies on LBAmp+X-Gal, LBAmp, LBCm
-          > 50 colonies on 50 µl plate of 1 ng pBluescript transformation (expected contamination was not observed); on Rest-plate: many more colonies; colonies appeared blue, but some seemed to be white (these colonies could have produced too low amounts of indigo dye – there are always differencens in the strength of the blue colour of different colonies; sometimes it’s hard to distinguish if these colonies are white or just light blue -> one can put the plates to 4 °C for some time, then the blue colour of the really blue colonies gets more intense), or they contain a somehow mutated plasmid…)
-          Neg. control (see pBluescrpit cloning) showed no colonies
-> 3 blue clones from 50 µl plate of pBluescript transformation were picked and streaked out on LBAmp; Clone 1 was inoculated in 4 ml LBAmp for MiniPrep and used as a control for Colony PCR (see below)
 
Transformation of pBluescript and pSB1C3 ligations from 31.7.13
-          No colonies on neg. control plates
-          Blue and white colonies on all other plates
-          Counting of the white colonies on 50 µl plates for w/o-insert-control, DarR and Riboswitch transformations
 
Plate
White colonies (approximate values)
w/o insert control
40
DarR
80
Ribo 40/41
80
Ribo 40/43
80
Ribo 41/42
160 -> many light blue colonies, counting was quite hard, just one half of the plate was counted
Ribo 42/43
160-> many light blue colonies, counting was quite hard, just one half of the plate was counted
 
-> There were more white clones on the ligation plates than on w/o insert control (white clones might be derived from relegated and mutated vector (eg. Frame shift))
 
-> Colony PCR incl. Master Plate and inoculation of 4 ml LBAmp cultures for MiniPrep (3 white clones from w/o insert control, 5 white clones from each of the other plates)
 
pSB1C3:
-          No colonies on neg. control plates and on ligation plates
-> Further incubation of all plates
 
 
Colony PCR for pBluescript cloning on 31.7.13
 
-          Reactions:
-                    1x pBluescript plasmid 1:10 diluted (white epi A)
-                    1x C1 from transformation of 1 ng pBluescript plasmid (see above) (white epi B)
-                    (white) C1, C2 and C3 from w/o insert control plate (white epis C, D and E)
-                    (white) C1 – C5 from DarR plate (white epis 1 – 5)
-                    (white) C1 – C5 from Ribo 40/41 plate (yellow epis 1 – 5)
-                    (white) C1 – C5 from Ribo 40/43 plate (orange epis 1 – 5)
-                    (white) C1 – C5 from Ribo 42/41 plate (violet epis 1 – 5)
-                    (white) C1 – C5 from Ribo 42/43 plate (blue epis 1 – 5)
 
-          Colony PCR was performed according to protocol from 10.7.13 with some modifications:
a) Inoculation of liquid culture for MiniPrep, as well
b) Primers M13-PUC-fwd and M13-PUC-rev diluted 1:20 in HPLC water and used for Colony PCR (now as well in 1:20 dilution in our box with all other primers) -> I changed the Annealing Temperature in one of the cyclers in which the protocols are saved!!! It’s 52 °C, not 54°C as required for VF2/VR PCR.
 
Inoculation/Plating was done in the following order:
1. Picking of clone and streak-out on Master Plate
2. Inoculation of 4 ml LBAmp  liquid culture ON for MiniPrep
3. Inoculation of PCR reaction
 
-> Incubation of liquid cultures and plates ON at 37 °C
-> After PCR, the PCR reactions were stored at -20°C in yellow-tip-box-rags sealed with parafilm
Transformation of parts BBa_E0030 and BBa_E0020
 
-          Solved parts in 10 µl HPLC water
-          Transformation of E. coli DH5α according to methods folder
-          Plating on LBCm plates + Incubation ON at 37 °C
 
Part
What?
Backbone and Resistance
Length
Emission
Extinction
Well
BBa_E0030
EYFP; no RBS, no terminator
pSB1C3 -> CmR
723 bp
527
514
Plate 3 16D
BBa_E0020
ECFP; no RBS, no terminator
pSB1C3 -> CmR
723 bp
476
439
Plate 3 3M
 
->  For some of the Riboswitches, we need a Reporter without RBS. But The GFP of part 6 already contains a terminator and an RBS. So, I searched in the registry for a GFP reporter (or something similar) without RBS (and maybe with terminator). However, the transformation of these parts is just a try, because previous transformations of distribution kit plasmids did not work at all (including the GFP in part 6 without any regulatory sequences). If the transformation fails again, we will order the forward primer for amplifying GFP from part 6 (see below)
 
Primer Design
 
-          Design of forward primer for amplification of GFP from Part 6 (as a reverse primer, we can use VR. Then we directly amplify the terminator with the GFP)
-          I introduced a point mutation in the primer for the third base of the second codon after the ATG (AAA -> AAG; both encodes for Lys), because then, the primer won’t form secondary structures (without this mutation, it forms strong secondary structures)… but now, I’m unsure, if we can do that. Because it should actually be the same sequence we clone into our other plasmids…
Primer not yet ordered
Fold ↑
July
31st
Restriction digestion of Part1,3,4 with SpeI and PstI:, New Riboswitch PCR, Transformation of ON ligations from 30.7.13 and Test of DH5α competent cells.
Restriction digestion of Part1,3,4 with SpeI and PstI:
Gel doc:
part M/1/ 1/3/3/4/4
-> only partial restriction -> we’ll do an overnight restriction
 
New Riboswitch PCR
(as performed on 24.6.)
 
 
Primers
T       A = 0.5 * [Tm(primer1) + Tm(Primer2)] – 6 °C
       PCR product size
 Name of Protocol in cycler
Riboswitch with native Promoter and RBS
iGEM_42
iGEM_43
51 °C
 
466 bp
Ribo1
Riboswitch with native Promoter
iGEM_42
iGEM_41
55.2°C
370 bp
Ribo2
Riboswitch only
iGEM_40
iGEM_41
59.7°C
213 bp
Ribo3
Riboswitch with native RBS
iGEM_40
iGEM_43
55.5 °C
403 bp
Ribo4
 
Tm (iGEM_40) = 65.7 °C
Tm (iGEM_41) = 65. 7 °C
Tm (iGEM_42) = 56.7 °C
Tm (iGEM_43) = 57.3 °C
 
PCR reaction and protocol as usual (50µl) (template: B. subtilis chrom.DNA)
Concentration of primer iGEM_43 doubled (4µl per reaction)
Reactions stored at -20°C in Falcon Tube
 
Gel doc:
iGEM_42/43, 42/43, 42/43; 42/41, 42/41, 42/41; 40/41, 40/41, 40/41; 40/43, 40/43, 40/43
<
after PCR purification following values with Nanodrop:
 
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
42/43
11.9
1.81
2.24
42/43
14.1
1.85
2.36
42/43
16.2
1.73
1.11
42/41
9.5
1.34
1.27
42/41
8.2
1.71
1.81
42/41
8.0
2.23
2.83
40/41
9.9
1.81
1.80
40/41
12.4
1.58
1.14
40/41
13.0
1.54
1.19
40/43
13.1
1.96
2.31
40/43
13.0
1.64
2.17
40/43
12.3
1.78
1.77
 
Restriction digestion of Part 7 with EcoR1 and PstI
-          2x 40 µl digestion setup with ~1,5 µg template
-          1h at 37°C
-          Run on gel for gelextraction
 
Gelextraction of Part 1,3,4,7
(planned ON digest of parts 1,3,4 was not done…???)
-          Whole volume of the digestion setups run on gel, 85 V for ~45 min
-          stained in EtBr for 15 min, washed in H20 for 10 min
<
 
-          1 LogLadder; 2-6 Part 1; 7-14 Part 3;   1 LogLadder; 2-7 Part 4; 8-16 Part 7
-          Thick band in the middle was cut out, same templates pooled. For part 7, upper band cut out and pooled in the gel extraction process
-          Standard protocol for gel extraction was used. Eluation in HPLC H2O, 2 Steps with 15 µl each (prewarmed 80°C)
Nanodrop measurements:
-          Part1: 6,7 ng/ µl
-          Part3: 14,4 ng/ µl
-          Part4: 15,8 ng/ µl
Part7: 56,6 ng/ µl
Transformation of ON ligations from 30.7.13 and Test of DH5α competent cells
-          Strain: DH5α
Transformation A1: pBluescript cloning
-          ON ligation reactions for cloning of DarR and Riboswitches 40/41, 40/43, 41/42, 42/43 into pBluescript -> addition of whole ligation reactions to comp. cells-> transformation -> plated on LBAmp+X-Gal plates
-          neg. control: addition of 10 µl dH2O-> transformation -> plated on LBAmp+X-Gal plates
Transformation A2: competent cells-testing
-     pBluescript (272 ng/µl) -> preparation of 1:272 dilution in dH2O (= 1 ng/µl) -> addition of 1 µl of the dilution to comp. cells -> transformation -> plated on LBAmp+X-Gal plates
-     1 tube with comp. cells was left untransformed (no DNA, no water) ->cells had to endure same transformation procedure as all other cells -> 150 µl were plated on LBAmp, LBCm or LBAmp+X-Gal plates (no rest plates, no 50 µl plates)
-    The negative control with water from Transformation A1 is considered to be a negative control for comp.cell-test, as well
Transformation B: pSB1C3 cloning
-    ON ligation reactions for cloning of DarR and Riboswitches 40/41, 41/42, 42/43 into pSB1C3-> transformation -> plated on LBCmplates
-     neg. control: addition of 10 µl dH2O-> transformation -> plated on LBCmplates
 
->  transformation was performed according to the methods folder with the following silly mistakes
1. heat shock took longer than 90 sec., probably 100 sec?
2. while all reactions where heat-shocked, the pSB1C3 neg. control was still on ice -> heat-shock done with slight time shift to all other samples…; heat shock for pSB1C3-NC took 100 sec precisely
3. I contaminated at least 2 plates during plating…
4. Since X-Gal is light-sensitive, the plates were incubated in a paper box at 37 °C
 
->  Conclusion: This transformation will work. Because I DON’T want it to work.
Preparation of Plates
1x 500 ml LBCm
1x 250 ml LBAmp+X-Gal
-> plates started already to solidify -> clumps were in the agar, while I poured them… I’m not sure, if we should use them…
Fold ↑
30th
RD of Parts 1-4 and Part7 (see yesterday), Ligation of DarR, the Riboswitch constructs (40/41, 42/41, 42/43) ,Ligation of DarR, the Riboswitch constructs (40/41, 40/43, 42/41, 42/43) with pBluescript.
RD of Parts 1-4 and Part7 (see yesterday)
 a 1%gel was run to determine the quality of the digest
Gel:
M|Part1|Part2|Part3|Part4|Part7|M|Oligo Hybridization iGEM63+64 (see yesterday)
Parts 1, 3 and 4 seem to be pretty much completely digested. Part 2 shows strange bands, the tube was discarded. The Primer hybridization shows no telling bands but a normal primer cloud. Hopefully it can be ligated.
 
->Parts 1, 3 and 3 were purified using the PCR clean up kit and eluted with 50µl pre-warmed H20 (mistake)
->2nd round of the RD with PstI (NOT FD because empty new stuff ordered). Due to the 50µl elutions, I distributed 25µl into a new vial and then pippetted two digestion reactions per construct. This should have the beneficial side effect of an even more complete RD.
 
           2x        4µl PstI
                  4µl Buffer 0
                  25µl DNA (everything)
                  7µl H2O
                  40µl reaction
 
->Incubation at 37°C for 6,5h
->frozen away, gel and clean-up tomorrow
DarR and Riboswitch Clonation from last week
 
The plates used for DarR in pSB1C3, Riboswitch in pSB1C3 and E0840 turned out to be of the wrong antibiotic recepee (5µg/ml instead of the right one: 35µg/ml).
->The inoculated cultures from yesterdayof course didn´t grow therefore
->The plates from last week were washed with LB media and plated on new, right 35µg/ml plates to see if they contain correctly ligated clones.
Gel extraction of Part 7
-          Gel Ex using the standard protocol. Eluation in 2 steps: 40 µl and 10 µl, prewarmed H2O
-          2 epis 50 µl each, stored in -20°C
-          Nanodrop meassurment: 6,4 and 6,3 ng/ µl. Also written on the tube
Ligation of DarR, the Riboswitch constructs (40/41, 42/41, 42/43)
 
Ligation of DarR and Riboswitch inserts with pSB1C3 backbone purified from part 7
-          Reaction of 20 µl total volume (1 µl ThermoScientific T4 Ligase + 2 µl 10x buffer + 17 µl insert-vector-water mix)
-          Vector: gel ex + purification of pSB1C3 backbone from part 7 by Dominik and Jonathan -> there was not enough vector left for the cloning of all riboswitches -> therefore, Riboswitch iGEM_40/43 was left out
-          Inserts: from 15.7.13
-          Calculation of approximate insert amounts for vector:insert ratio = 1:3 using online ligation calculator:
Size of vector ca. 2070 bp
Size of DarR and Riboswitch inserts -> uncut PCR product size used
PCR product
Size of PCR product
Approx. amount of insert required for ligation with 20 ng vector
DarR
ca. 600 – 700 bp
17 – 20 ng
Riboswitch iGEM_40/41
213 bp
6 ng
Riboswitch iGEM_42/43
466 bp
14 ng
Riboswitch iGEM_42/41
370 bp
11 ng
 
Reactions
-          Preparation of Master Mix for 6 reactions in total (Actually, it’s more clever to put the vector and 1 µl water/reaction more into the master mix, but I prepared the DNA mixtures first. Because of a pipetting error, I had to do the master mix this way. Otherwise the buffer and ligase concentration between the reactions would have varied… So, don’t wonder if this protocol appears a bit… linky?...)
Component
For 1 reaction
For 6 reactions
 
dH2O
3 µl
18 µl
10x ligation buffer
2 µl
12 µl
T4 ligase
1µl
6 µl
Total Volume
6 µl
36 µl
 
-          Preparation of DNA mixtures (14 µl in total; ca. 20 ng of vector for cloning of riboswitches and w/o insert control)
Component
Riboswitch iGEM_40/41
(7.5 ng/µl)
Riboswitch iGEM_41/42
(3 ng/µl)
Riboswitch iGEM_42/43
(3 ng/µl)
DarR
(2 ng/µl)
w/o insert
vector DNA (5.3 ng/µl)
4 µl
4 µl
4 µl
4 µl
4 µl
Insert DNA
1 µl
4 µl
5 µl
9 µl
-
dH2O
9 µl
6 µl
5 µl
1 µl
10 µl
Total Volume
14 µl
14 µl
14 µl
14 µl
14 µl
-          Addition of 6 µl Master Mix (it could be, that – when I was pipetting the riboswitch reactions – I did not change the tip… but I’m not sure about that. It was just a sudden thougth…)
-          Incubation on at 16 °C
Ligation of DarR, the Riboswitch constructs (40/41, 40/43, 42/41, 42/43) with pBluescript
Ligation of DarR and Riboswitch inserts with pBluescript vector digested with PstI and EcoRI
-          Reaction of 10 µl total volume (1 µl ThermoScientific T4 Ligase + 1 µl 10x buffer + 8 µl insert-water mix)
-          Vector: digested and purified by DAC team; Amp resistance + MCS in lacZ gene -> plate transformation reactions on LBAmp+X-Gal
-          Inserts: from 15.7.13
-          Calculation of approximate inserts amounts for vector:insert ratio = 1:3 using online ligation calculator:
Size of digested vector ca. 2616 bp
Size of DarR and Riboswitch inserts -> uncut PCR product size used
PCR product
    Size of PCR product
  Approx. amount of insert required for ligation with 20 ng vector
DarR
ca. 600 – 700 bp
14 - 16 ng
Riboswitch iGEM_40/41
213 bp
5 ng
Riboswitch iGEM_40/43
403 bp
9 ng
Riboswitch iGEM_42/43
466 bp
11 ng
Riboswitch iGEM_42/41
370 bp
8 ng
Reactions
-          Preparation of Master Mix for 7 reactions in total (ca. 20 ng of vector for cloning of riboswitches and w/o insert control)
Component
For 1 reaction
For 7 reactions
 
Vector (11.3 ng/µl)
2 µl
14 µl
dH2O
1 µl
7 µl
10x ligation buffer
1 µl
7 µl
T4 ligase
1µl
7µl
Total Volume
5 µl
35 µl
-          Preparation of DNA mixtures
Component
Riboswitch iGEM_40/41
(7.5 ng/µl)
Riboswitch iGEM_40/43
(5.5 ng/µl)
Riboswitch iGEM_41/42
(ca. 3 ng/µl; 2nd elution)
Riboswitch iGEM_42/43
(3 ng/µl)
DarR
(2.9 ng/µ; 2nd elutionl)
w/o insert
Insert DNA
1 µl
2 µl
3 µl
4 µl
5 µl
-
dH2O
4 µl
3 µl
2 µl
1
-
5 µl
Total Volume
5µl
5 µl
5 µl
5 µl
5 µl
5 µl
-          Addition of 5 µl Master Mix
-          Incubation on at 16 °C
Attention: DarR and Riboswitch iGEM_41/42 inserts are almost empty! I marked the tubes with a green dot. We have to prepare new ones! -> Whenever Spe and Pst FD arrive:
Fold ↑
29th
Inoculation of 8 Clones from each, Riboswitch, DarR and E0840 (were put in the fridge on friday, plated 23. and 24.7.),Hybridization of iGEM_63 and iGEM_64 to generate the Promoter(part3)-Operator construct
Inoculation of 8 Clones from each, Riboswitch, DarR and E0840 (were put in the fridge on friday, plated 23. and 24.7.)
 ->E0840 heavily contaminated with something that looks like fungus. Plates discarded.
->This leaves DarR and the Riboswitch for inoculation in 4ml Cam cultures for a miniprep tomorrow.
->in addition, a masterplate was plated.
Hybridization of iGEM_63 and iGEM_64 to generate the Promoter(part3)-Operator construct
The primers were first diluted in HPLC-H2O according to the QC-sheet (100µM). Then a 1:20 dilution stock was pipetted. For this experiment, the 1:20 dilution stocks were used.
Protocol:
10µl iGEM_63
+         10µl iGEM_64
           20µl reaction
->80°C for 10 minutes (heatblock)
->after 10 minutes, turn the heatblock off and let the sample slowly cool down in it
 
-> The ends of the oligos are “digested” with EcoRI and SpeI and phosphorylated. The product of this experiment can directly be ligated in front of the GFP-part!
See the gel tomorrow
Restriction digestion of Part 1-4 and Part 7
-          Part 1-4 digested with SpeI FD
-          Part 7 digested with EcoR1+SpeI FD + Phosphatase treatment (last 20 minutes of digestion)
-          40 µl setups, standard assay.
->for part 1-4, restriction with PstI (not FD, because empty) on 30.7.
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w9 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
