"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w12
From 2013.igem.org
23rd
Transformation of Promoter1rev/Promoter3rev + pSB1C3 , Colony PCR for Transformation of Promoter1rev/Promoter3rev + pSB1C3, Preparation of new Backup plates from yesterday’s ColonyPCRs/Inoculation...
Transformation of Promoter1rev/Promoter3rev + pSB1C3 
-          neg. control plates still negative
-          pSB1C3 w/o insert control: 3 big clones on 50 μl-plate, many clones on rest-plate (small, big, medium in size)
-          pSB1C3 + Promoter1rev: no clones on 50 μl-plate, 10 clones on rest-plate (small or medium size)
-          pSB1C3 + Promoter3rev: no clones on 50 μl-plate, 6 clones on rest-plate (small or medium size)
→ Colony PCR to test for positive clones
→ plates stored at 4 °C, big fridge
Colony PCR for Transformation of Promoter1rev/Promoter3rev + pSB1C3
-          as described previously (10.7.13)
-          1x RBSrev in pSB1C3 plasmid as additional control (plasmid prepped on 19.8.13, 211.6 ng/μl →  1:20 dilution →  ca. 10 ng/μl), orange tube, termed RBS
-          1x medium size clone from w/o insert control plate picked (purple tube, termed R)
-          10x clones of Promoter1rev + pSB1C3 transformation picked (termed C1 – C10), blue tubes
-          6x clones of Promoter3rev + pSB1C3 transformation picked (termed C1 – C6), pink tubes
-          preparation of mastermix for 20 reactions, primers: iGEM_38 (VF2) and iGEM_39 (VR)
-          preparation of master plate (LBCm) →  incubation over day at 37 °C
-          25 μl mastermix + 1 μl plasmid dilution or cells
-          PCR protocol: same as on  10.7.13
-          plates stored at 4 °C, big fridge
Preparation of new Backup plates from yesterday’s ColonyPCRs/Inoculation
-          I wasn’t sure anymore, if I inoculated DarRrev-Termrev C1 – C5, part 6.4 B C1 – C5 and part 6.4A C1 – C8 correctly in the labeled tubes →  streak-out of all clones for DarRrev-Termrev C1 – C5, part 6.4 B C1 – C5 and part 6.4A C1 – C8 on new LBCm plates
-          in case of DarRrev-Termrev and part 6.4 B, the remaining clones on the master plates (that were not inoculated yesterday), were streaked out as well on the new master plate
-          incubation over day at 37 °C, then storage at 4 °C, big fridge
MiniPlasmidPreparation of clones inoculated yesterday
-          samples: DarRrev-Termrev C1 – C5, part 6.4 B C1 – C5 and part 6.4A C1 – C8
-          with Nucleospin kit (Macherery-Nagel)
-          1st elution: 30 μl HPLC water (pre-warmed), incubation for 2 min at 50 °C
2nd elution: 22 μl HPLC water (pre-warmed), incubation for 2 min at 50 °C
-          NanoDrop concentration measurement:
Test Restriction Digest of DarRrev-Termrev C1 – C5 
-          10 μl in total: 1 μl EcoRI FD + 1 μl PstI FD + 1 μl 10x FD Green buffer + 4 μl dH2O + 3 μl plasmid mix (ca. 200 – 300 ng plasmid)
-          preparation of 6x Mastermix consisting of enzymes, buffer and water
-          preparation of 3 μl plasmid-water Mix (plasmids prepped today, see above)
DarRrev-Termrev C1 →  3 μl plasmid (88.6 ng/μl)
DarRrev-Termrev C2 →  2 μl plasmid (160.2 ng/μl) + 1 μl dH2O
DarRrev-Termrev C1 →  1 μl plasmid (229.7 ng/μl) + 2 μl dH2O
DarRrev-Termrev C1 →  3 μl plasmid (88.5 ng/μl)
DarRrev-Termrev C1 → 2μl plasmid (107.3 ng/μl) + 1 μl dH2O
-          addition of 7 μl Mastermix
-          incubation at 37 °C for 1 h
Gel run for Test RD of DarRrev-Termrev C1 – C5, plasmids from today’s MiniPrep and Colony PCR for Transformation of Promoter1rev/Promoter3rev + pSB1C3
-        3x 1%-agarose-1xTAE gel
-        addition of 5 μl 5xLD to 25 μl Colony PCR reaction and loading of 5 μl PCR reaction supplied with 5xLD (during loading, some sample Colony PCR Promoter3rev+pSB1C3 C4 was spilled over the well…)
-        loading of 1 μl uncut DarRrev-Termrev C1 – C5 plasmid + 1 μl 5x LD + 3 μl dH2O; loading of 5 μl RD reaction mix
-        loading of 1 μl uncut part 6.4 B C1 – C5 and part 6.4A C1 – C8 plasmid + 1 μl 5x LD + 3 μl dH2O
-        loading of 3 μl 2 log ladder as a marker
-        run at 100 V
-        EtBr staining + destaining in water
-        UV detection
 
Gel 1:
Loading: Marker/uncut DarRrev-Termrev C1/ RD DarRrev-Termrev C1/ uncut DarRrev-Termrev C2/ RD DarRrev-Termrev C2/ uncut DarRrev-Termrev C3/ RD DarRrev-Termrev C3/ uncut DarRrev-Termrev C4/ RD DarRrev-Termrev C4/ uncut DarRrev-Termrev C5/ RD DarRrev-Termrev C5/part 6.4 B C1 plasmid/part 6.4 B C2 plasmid/part 6.4 B C3 plasmid/part 6.4 B C4 plasmid/Marker
expected sizes for DarRrev-Termrev plasmids: 630 bp (DarRrev) + 130 bp (Terminatorrev) = 760 bp and 2070 bp from pSB1C3 backbone
→ all DarRrev-Termrev clones are positive →  sequencing of C2 and C3 by SeqLab (because these clones have highest plasmid concentration)
→ all purified part 6.4 B plasmids look normal except for plasmid part 6.4 B C1 and C3 →  sequencing of C2 and C5 by SeqLab (because these clones have highest plasmid concentration and because their plasmids seem to be ok)
 
Gel 2:
Loading: Marker/part 6.4 B C5 plasmid/ part 6.4A C1 plasmid/ part 6.4A C2 plasmid/ part 6.4A C3 plasmid/ part 6.4A C4 plasmid/ part 6.4A C5 plasmid/ part 6.4A C6 plasmid/ part 6.4A C7 plasmid/ part 6.4A C8 plasmid/Colony PCR RBSrev+pSB1C3 plasmid/ Colony PCR Re-ligand/Colony PCR Promoter1rev+pSB1C3 C1/ Colony PCR Promoter1rev+pSB1C3 C2/ Colony PCR Promoter1rev+pSB1C3 C3/Marker
expected sizes for Promoter1rev and Promoter3rev Colony PCR products: 310 bp (VR, VF2) + 112 bp Promoter1/3rev = 422 bp
expected sizes for RBSrev+pSB1C3 and Re-ligand: 310 bp (VR, VF2) + 12 bp (RBSrev) = 322 bp  
→  all purified part 6.4 B/A plasmids look normal →  sequencing of part 6.4 A C2, C3, C5 and C8 by SeqLab (because these clones have highest plasmid concentration)
→  Colony PCR: for RBSrev+pSB1C3 and Re-ligand, the expected band was observed, band at same bp for Promoter1rev+pSB1C3 C1 – C3 (negative clones)
 
Gel3:
Loading: Marker/ Colony PCR RBSrev+pSB1C3 plasmid/ Colony PCR Promoter1rev+pSB1C3 C4/ Colony PCR Promoter1rev+pSB1C3 C5/ Colony PCR Promoter1rev+pSB1C3 C6/ Colony PCR Promoter1rev+pSB1C3 C7/ Colony PCR Promoter1rev+pSB1C3 C8/ Colony PCR Promoter1rev+pSB1C3 C9/ Colony PCR Promoter1rev+pSB1C3 C10/ Colony PCR Promoter3rev+pSB1C3 C1/ Colony PCR Promoter3rev+pSB1C3 C2/ Colony PCR Promoter3rev+pSB1C3 C3/ Colony PCR Promoter3rev+pSB1C3 C4/ Colony PCR Promoter3rev+pSB1C3 C5/ Colony PCR Promoter3rev+pSB1C3 C6/Marker
expected sizes for Promoter1rev and Promoter3rev Colony PCR products: 310 bp (VR, VF2) + 112 bp Promoter1/3rev = 422 bp
expected sizes for RBSrev+pSB1C3 and Re-ligand: 310 bp (VR, VF2) + 12 bp (RBSrev) = 322 bp
→  Colony PCR: for RBSrev+pSB1C3, the expected band was observed, band at same bp for Promoter1rev+pSB1C3 C5 and for Promoter3rev+pSB1C3 C1 – C4 and C6 (negative clones); band at ca. 400 bp for romoter1rev+pSB1C3 C4 and Promoter3rev+pSB1C3 C5 (= positive clones)→  on Monday: inoculation of these clones for MiniPrep, test RD and sequencing
Preparations of samples for sequencing by SeqLab
9 – DarRrev-Termrev C2 + VF2
10 – DarRrev-Termrev C2 + VR
11 – DarRrev-Termrev C3 + VF2
12  - DarRrev-Termrev C3 + VR
13 – part6.4 B C2 + VF2
14 – part 6.4 B C5 + VF2
15 – part 6.4 A C2 + VF2
16 – part 6.4 A C3 + VF2
17- part 6.4 A C5 + VF2
18 – part 6.4 A C8 + VF2
 
-          for all samples: 6 μl plasmid + 1 μl iGEM_38 1:5 or iGEM_39 1:5
Cryostocks and Minipreps of Ribo A C4 and C5 inocculated by Nina yesterday
Following clones were frozen away prepped: 
Samples
Procedures done
C4 3
Cryostock and Mini
C4 4
Cryostock and Mini
C4 6
Cryostock and Mini
C4 7
Cryostock
C4 8
Cryostock and Mini
C5 2
Cryostock and Mini
C5 3
Cryostock and Mini
C5 6
Cryostock and Mini
C5 8
Cryostock and Mini
-     C4 7 was dropped during the miniprep
 
Nanodrop:
All ca. 400-500ng/µl , good ratio, exact measurements on the vials
 
Sequencing by SeqLab:
1: Ribo A C4 6 VF
2: Ribo A C4 6 VR
3: Ribo A C4 8 VF
4: Ribo A C4 8 VR
5: Ribo A C5 6 VF
6: Ribo A C5 6 VR
7: Ribo A C5 8 VF
8: Ribo A C5 8 VR
Inoculation of 10ml cultures of Ribo A C4 7 and 8, the DAC cells and an empty vector as control (names to be added later) for an experiment with plates containing either the reporter system or the DAC producing cells in the agar. Clones were inoculated by Katrin G. yesterday.
Our Ribo A clones didn´t grow at all, the other two cultures were harvested at OD600=2, washed and frozen away at -80°C in Glycerin as follows:
→ harvest 4x 2ml of the culture in 2ml eppis
→ centrifuge down
→ discard supernatant
→ resuspend cells in 100µl LB medium
→ transfer the resuspended cells into a 2ml screw-cap-eppi
→ add 200µl Glycerin
→ Vortex and freeze at -80°C
Fold ↑
SDS Page of   dialysis and elution samples
SDS Page of   dialysis and elution samples
-       Mix 15µl of samples  with 5µl Pab   and 5µl buffer W
-       Incubation for 10min at 93°C
-       Mix each sample with 4µl protein marker
-       Load whole volume on 15% SDS-gel
-       Run for 5min at 90V and 1h at 120V
 
Fold ↑
22nd
Colony PCR for DarRrev-Termrev transformation , Colony PCR for part6.3 B + RBSrev transformation, Gel run for Colony PCR for DarRrev-Termrev transformation, Sequencing of part6.3 B C1, C2 and C3 plasmids by G2L, Plates of part6.3 A + RBSrev transformation...
Transformation from 21.8.13
-          neg. control: no clones on 50 μl-plate, no clones on rest-plate
-          Termrev vector w/o insert: no clones on 50 μl-plate, few clones on rest-plate
-          Termrev vector + DarRrev insert: few clones on 50 μl-plate, many clones on rest-plate
-          part 6.3 A vector w/o insert: no clones on 50 μl-plate, 1 clone on rest-plate
-          part 6.3 A vector + RBSrev insert: no clones on 50 μl-plate, few clones on rest-plate
-          part 6.3 B vector w/o insert: no clones on 50 μl-plate, no clones on rest-plate
-          part 6.3 B vector + RBSrev insert: no clones on 50 μl-plate, some clones on rest-plate
-          pSB1C3 vector w/o insert: 1 or 2 clones on 50 μl-plate, several clones on rest-plate
-          pSB1C3 vector + Promoter1rev insert: no clones on 50 μl-plate, no clones on rest-plate
-          pSB1C3 vector + Promoter3rev insert: no clones on 50 μl-plate, few clones on rest-plate
→ colony PCR for DarRrev-Termrev transformation
→ further incubation of all other plates, until clones are bigger/grown
Colony PCR for DarRrev-Termrev transformation 
-          as described previously (10.7.13)
-          1x Termrev C1 plasmid as additional control (plasmid prepped on 16.8.13, 102.8 ng/μl →  1:10 dilution →  ca. 10 ng/μl), blue tube (termed T)
-          1x Re-ligand (termed R), pink tube
-          12x DarRrev-Termrev clones (termed C1 – C12), purple tubes
-          preparation of mastermix for 15 reactions, primers: iGEM_38 (VF2) and iGEM_39 (VR)
-          preparation of master plate (LBCm) →  incubation over day at 37 °C
-          25 μl mastermix + 1 μl plasmid dilution or cells
-          PCR protocol: same as on  10.7.13
-          plates stored at 4 °C, big fridge
Plates for part 1 – 4 C1 – C3 
-          plates taken out, since all clones grew
-          plates put to 4 °C, big fridge
Colony PCR for part6.3 B + RBSrev transformation 
-          after some additional incubation time
-          as described previously (10.7.13)
-          1x part6.3 B C2 plasmid as additional control (plasmid prepped on 19.8.13, 220.3 ng/μl →  1:20 dilution →  ca. 11 ng/μl), purple tube, termed 6.3 B
-          neg. control plates still without colonies
-          still no religands →  performing dephosphorylation longer than actually necessary seems to be helpful (see 20.8.13)
-          1 or 2 clones on 50 μl plate, many clones on rest-plate: 13x clones picked (termed C1 – C13), green tubes
-          preparation of mastermix for 15 reactions, primers: iGEM_38 (VF2) and iGEM_39 (VR)
-          preparation of master plate (LBCm) →  incubation over day at 37 °C
-          25 μl mastermix + 1 μl plasmid dilution or cells
-          PCR protocol: same as on  10.7.13
-          plates stored at 4 °C, big fridge
→ 12 bp (size of RBSrev) difference between positive and negative clones, one would never see in a gel… PCR thrown away…
Gel run for Colony PCR for DarRrev-Termrev transformation
-          1%-agarose-1xTAE gel
-          addition of 5 μl 5xLD to 25 μl PCR reaction
-          loading of 3 μl 2 log ladder as a marker
-          loading of 5 μl PCR reaction supplied with 5xLD
-          run at 100 V
-          EtBr staining + destaining in water
-          UV detection
 
Loading: Marker/Termrev plasmid/Re-ligand/ DarRrev-Termrev C1/ DarRrev-Termrev C2/ DarRrev-Termrev C3/ DarRrev-Termrev C4/ DarRrev-Termrev C5/ DarRrev-Termrev C6/ DarRrev-Termrev C7/ DarRrev-Termrev C8/ DarRrev-Termrev C9/ DarRrev-Termrev C10/ DarRrev-Termrev C11/ DarRrev-Termrev C12/Marker
-          expected product sizes
negative clone: 310 bp (VR, VF2) + 130 bp (Terminatorrev) = 440 bp
positive clone: 310 bp (VR, VF2) + 130 bp (Terminatorrev) + 630 bp (DarRrev) = 1070 bp
→ Religand plasmid corresponds to Termrev plasmid, as expected
→ all DarRrev-Termrev clones appear to be positive →  inoculate 3 clones for MiniPrep tomorrow
-          PCR reactions stored at 4 °C, small fridge
Sequencing of part6.3 B C1, C2 and C3 plasmids by G2L
sequencing by SeqLab failed; for both clones, sequence stopped at same region: possible reason: secondary structures formed during sequencing
5 μl total reaction: ca. 300 ng plasmid + 10 % DMSO (to prevent secondary structures) + 1 μl primer iGEM_38, 1:20 diluted
 
Preparation of the following samples
-          ppra_1: 2 μl plasmid prepped on 19.8.13 (268.4 ng/μl) + 1 μl DMSO (50 % in HPLC water) + 1 μl iGEM_38 1:20 + 1 μl HPLC water
-          ppra_2: 2 μl plasmid prepped on 19.8.13 (220.3 ng/μl) + 1 μl DMSO (50 % in HPLC water) + 1 μl iGEM_38 1:20 + 1 μl HPLC water
-          ppra_3: 2 μl plasmid prepped on 19.8.13 (247.2 ng/μl) + 1 μl DMSO (50 % in HPLC water) + 1 μl iGEM_38 1:20 + 1 μl HPLC water
-          ppra_4: 1 μl plasmid prepped on 21.8.13 (305.4 ng/μl) + 1 μl DMSO (50 % in HPLC water) + 1 μl iGEM_38 1:20 + 2 μl HPLC water
-          ppra_5: 1 μl plasmid prepped on 21.8.13 (332.1 ng/μl) + 1 μl DMSO (50 % in HPLC water) + 1 μl iGEM_38 1:20 + 2 μl HPLC water
-          ppra_6: 1 μl plasmid prepped on 21.8.13 (328.8 ng/μl) + 1 μl DMSO (50 % in HPLC water) + 1 μl iGEM_38 1:20 + 2 μl HPLC water
prior to sequencing, the samples have to be denatured →  remark was entered into excel-sheet
Plates of part6.3 A + RBSrev transformation
-          still no clones on neg. control
-          3 clones on w/o insert plate
-          8 clones on w/ insert plate
Inoculation of DarRrev-Termrev clones, part6.3 B + RBSrev(= part 6.4 B) and part 6.3 A + RBSrev(= part 6.4 A) clones for MiniPrep
-          inoculation of
DarRrev-Termrev clones C1 – C5→  Test restriction digest
 B clones C1 – C5 (no re-ligands)→  sequencing
part6.4 A clones C1 – C8 (3 re-ligands and 8 clones on WITH-insert plate→  sequencing
-          in 4 ml LBCm
-          incubation at 37 °C ON
Run the colony PCR from yesterday on a gel
Gel doc:
Wells: M/Ribo A C4 clone 1/2/3/4/5/6/7/8/9/10/blank/religant from control plate w/o insert
Wells: M/Ribo A C5 clone 1/2/3/4/5/6/7/8/9/10/blank/religant from control plate w/o insert
→ all clones with the high band should contain the Ribo A insert
 
Following clones were choosen to prepare a miniprep:
Ribo A C4 clones 3/4/6/7/8 (all others didn’t grew or showed wrong bands)
Ribo A C5 clones 2/3/6/8 (all others didn’t grew or showed wrong bands)
 
Miniprep of the DAC team Lm cryostock colony
Nanodrop:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
C4 3
15.8
0.82
0.18
C4 4
105.8
1.92
0.89
C4 6
46.8
1.44
0.41
C4 7
223.6
1.69
0.93
C4 8
85.2
1.53
0.73
C5 2
23.3
1.74
2.27
C5 3
56.0
1.97
0.87
C5 6
154.9
1.56
0.71
C5 8
-0.1
-0.06
-0.05
DAC Lm
36.5
1.75
1.91
Second elute C5 2
17.5
1.46
0.65
Second elute DAC Lm
51.1
1.83
2.41
Second measurement C5 8
3.2
1.37
0.37
Stored in to do box
 
Following plasmids were choosen for sequencing:
Seq number
Plasmid + primer
1
C4 clone 7 for
2
C4 clone 7 rev
3
C4 clone 6 for
4
C4 clone 6 rev
5
C5 clone 3 for
6
C5 clone 3 rev
7
C5 clone 6 for
8
C5 clone 6 rev
 
Again inocculation of all preped Ribo A C4 and C5 clones that were already prepped, for preparation of Cryostocks and a new miniprep, as the old one showed smears
 
Katrin Gunka
Preparation of plates with Lb medium also containing the DAC team Lm stain. One plate just like this, another one also containing
Fold ↑
RT-PCR
RT-PCR
·         Another   gel to check the digestion from the day before and to check the newly ordered   primers for ydaO and ktrA.
·         The   gel looked fine, so the RT-PCR can get started
Concentrations digested RNA
·         digestion   20.08.13
o   wt: 76 ng/µl
o   44: 73 ng/µl
o   46: 83 ng/µl
o   wt*: 87 ng/µl
o   44*: 73 ng/µl
o   46*: 69 ng/µl
·         digestion   21.08.13
o   wt: 63 ng/µl
o   44: 64,5 ng/µl
o   46:   72 ng/µl
Results RT-PCR
·         no   significant differences. All genes seem to be downregulated in both strains   (1344 = much c-di-AM; 1346 = little c-di-AMP)
Ø  for   better differences a new RT-PCR with differrent conditions are tested   (medium: low and high phosphate medium)
These results show us, that there is no real difference between   the strains, because all genes are downregulated.ptsH, hrcA and groEL, served   as control. We are mainly interested in ydaO and ktrA. YdaO is the gene behind the riboswitch, ktrA is a phosphate channel.   Due to the fact that there is no difference between 1344 and 1346 compared to   the wildtype in CSE 0,5% Glucose medium, we did the exact same setup only in   low & high phosphate medium (see below).
Fold ↑
21st
Plasmid Mini Prep,  Transformation of ligations prepared yesterday, observation of clones of Ribo A C4 and C5 CFP under UV light...
CryoStocks 
-          preparation of DMSO cryostocks of all inoculated clones (according to protocol described previously)
Plasmid Mini Prep
-          MiniPrep was done using the culture volume that remained after preparing the cryostocks (see above)
-          kit: Nucleospin, Macherey-Nagel
-          cells were harvested by centrifuging at 13000 g, 1 min
-          elution: 1x 30 μl HPLC water (pre-warmed), incubation for 3 min at 50 °C
-          NanoDrop concentration measurement:
Transformation of ligations prepared yesterday
-          according to protocol in methods folder
-          with XL1-Blue comp. cells
-          neg. control was done
-          when adding the Promoter1rev + pSB1C3 ligation mix to the comp. cells, a drop of the comp. cells was spilled…
-          500 μl were removed after centrifuging
-          plating on LBCm plates
-          incubation at 37 °C
Gel run after MiniPrep to check if plasmids are clean
-          1 %-agarose-1xTAE gel from yesterday (prepared by Jan)
-          loading of 3 μl 2 log ladder
-          loading of 1 μl plasmid + 1 μl 5xLD + 3 μl dH2O
-          run at 100 V
-          EtBr staining + destaining in water
-          UV detection
loading: Marker/Termrev C1/DarRrev C4/ RBSrev C1/part6.3 A C1/part6.3 B C1/ part6.3 B C2/ part6.3 B C3
→ all plasmids are purified from chrom.DNA and look normal
→ samples stored in DarR reporter system boxRe-streak of part 1 – 4 Clones 1 – 3 on LBAmp 
-          plates from 6.6.13 (stored in fridge…. at 4 °C….)
-          bacteria were quite dry, so I scratched some cell material off and streaked it out on new plates
-          incubation ON at 37 °C
-          let’s hope they grow…
Preparation of new LBCm plates 
-          2x500 ml
-          35 μg/ml Cm
-          stored at 4 °C, big fridge
observation of clones of Ribo A C4 and C5 CFP under UV light
(all clones with YFP don’t glow under UV light) 
Pictures:
Ribo A C4 + CFP UV light
Ribo A C4 + CFP transmitted light
Ribo A C5 + CFP UV light
Ribo A C5 + CFP transmiited light
Control w/o insert UV light
Control w/o insert transmitted light
Preparation of a Masterplate of 10 clones from plate with C4 and C5 CFP
Inocculation of 10 clones from plate with C4 and C5 CFP in LB medium with Cm
Colony PCR of the inoculated clones and a control w/o insert CFP
Component
Volume
Taq
1μl
10x Taq buffer
2,8μl
dNTP mix
1μl
iGEM_38 (VF2) 1:20
1μl
iGEM_39 (VR) 1:20
1μl
dH2O
18,8μl
Total
25μl
 
Fold ↑
DNAse I digestion
DNAse I digestion
·         this   digestion is necessary to get rid of DNA contamination
·         use   PCR tubes
25 µl   sample
·         2.5 µg RNA
·         2.5   µl 10 x buffer
·         5 µl   DNAse I
·         x µl   RNAse free water
30   min 37°C in the PCR cycler
Stop   the reaction with 2.5 µl EDTA for 10 min at 65°C (also in the   cycler)
Control PCR on digestion
PCR   Scheme (50 µl)
·         10 µl HF buffer
·         2 µl   dNTPs Mix
·         4 µl   Primer Mix (here rpsE)
·         1 µl   RNA
·         0,5   µl PhuS
·         32 µl   RNAse free H20.
PCR   Programm
·         98°C   2 min
·         98°C   35 sec
·         52°C   35 sec
·         72°C   45 sec
·         72°C   10 min
·         15°C   hold
 
·         run a   gel to check if everything went correctly
In   this case we did this digestion with the wt, GP1344 and GP1346. Additionally   we used a NTC and cDNA control.
Ø  The   digestion worked fine. No DNA detectable in our samples, only in the cDNA
Ø  Determine   RNA concentration
Ø  This   failed
We   did the digestion again. This time we used 6 samples in total, two samples of   each strain. IN the first three samples we used 5 µl DNAse I   according to protocol, in the other three samples we only used 2.5 µl DNAse   I.
Stored   the samples over night at -20°C.
 
Fold ↑
20th
Gel extraction of DarRrev insert , Restriction digest of Termrev with PstI FD, Restriction digest of RBSrev C1 and part6.3 A and B (C2 each) with EcoRI FD, AP treatment of Termrev vector for ligation with DarRrev insert , Restriction Digest of all vectors digested with EcoRI FD today, ligation.....
Transformation of yesterday´s ligation
Gel extraction of DarRrev insert 
-          with Qiagen PCR gel/PCR purification kit
-          since DarRrev insert has > 600 bp, no isopropanol was added/only QG buffer used
-          only 1 column used since <400 mg gel
-          elution 1x with 30 μl of pre-heated HPLC water, incubation at 50 °C for 2 min
-          NanoDrop concentration measurement
sample
ng/μl
A260nm/A280nm
A260nm/A230nm
DarRrev insert X+P
14.8
1.53
0.12
Restriction digest of Termrev with PstI FD
-          DNA: Termrev C1 plasmid restricted with SpeI and purified yesterday
-          digest: 
Component
Volume
plasmid
30μl (volume determined with pipet)
10x FD Green buffer
4μl
PstI FD
4μl
dH2O
2μl
Total
40μl
 
-          digest for 2 h at 37 °C
Restriction digest of RBSrev C1 and part6.3 A and B (C2 each) with EcoRI FD
-          DNA: plasmids purified yesterday
-          digest for all reactions (pipette individually, no mastermix):
Component
Volume
plasmid
8μl (since each plasmid had > 200 ng/μl)
10x FD Green buffer
4μl
Eco
I FD
4μl
dH2O
24μl
Total
40μ
 -          digest for 2 h at 37 °C
Sequencing
 results RBSrev C1 and part6.3 A and B (C1 and C2 each)
-          part 6.3 A C1 (sample 3) – sequence as expected (no mutations)
-          part 6.3 A C2 (sample 4) – a G at position 52 of Hybridization oligo is missing →  digest sample (see above) thrown away, new digest (exactly as described above) using clone 1
-          part 6.3 B C1 (sample 5) – sequencing did not work, only first part of oligo in vector sequenced →  was ok
-          part 6.3 B C2 (sample 6) – sequencing did not work, only first part of oligo in vector sequenced →  was ok →  further digest
-          RBSrev in pSB1C3 C1 (sample 7) – sequence as expected
Loading: 
Marker/uncut Termrev C1/RD Termrev C1/uncut RBSrev C1/RD RBSrev C1/uncut part6.3 A C1/RD part6.3 A C1/uncut part6.3 B C2/RD part6.3 B C2/Marker
-          all digests are only partial, but most of plasmid is cut  →  purify digests
New sizes of Riboswitch PCR products
 
I checked again the alignments of the riboswitch PCR products, and the sizes I entered into the journal on 24.6.13 are partially wrong.
 
The correct sizes are: 
 
Primers
approx. PCR product size (w/o prefix/suffix)
approx. PCR product size (w/ prefix/suffix)
Riboswitch with native Promoter and RBS
iGEM_42
iGEM_43
460 bp
500 bp
Riboswitch with native Promoter
iGEM_42
iGEM_41
315 bp
355 bp
Riboswitch only
iGEM_40
iGEM_41
230 bp
270 bp
Riboswitch with native RBS
iGEM_40
iGEM_43
375 bp
415 bp
 
Inoculation of clones for CryoStocks
inoculation of
-          RBSrev + pSB1C3 C1
-          part 6.3 A C1
-          part 6.3 B C1, C2, C3
-          Termrev + pSB1C3 C1
-          DarRrev + pSB1C3 C4
 → from master plates, inoculation in 4 ml LBCm
incubation ON at 37 °C, 200 – 210 rpm
Purification of vectors digested today 
-          samples: RBSrev + pSB1C3 RD, part 6.3 A C1 and part 6.3 B C2 RD, Termrev + pSB1C3 RD
-          with Qiagen PCR clean-up kit
-          addition of 500 μl PB buffer to each reaction
-          elution with 30 μl pre-warmend HPLC water, incubation at 50 °C for 2 min
AP treatment of Termrev vector for ligation with DarRrev insert 
-          this is silly… I could have done it before purification, then I would have lost less vector…
-          addition of 1 μl AP, 4 μl AP buffer 10x, 8 μl dH2O to 27 μl purified vector
-          incubation for 1 h at 37 °C (PstI makes 3’ overhang)
Restriction Digest of all vectors digested with EcoRI FD today
-          samples: RBSrev + pSB1C3 RD, part 6.3 A C1 and part 6.3 B C2 RD
-          all samples had a volume of 30 μl after clean-up, so addition of 2 μl dH2O and 4 μl 10x FD buffer
-          addition of 4 μl SpeI FD to RBSrev + pSB1C3 RD sample, and 4μl XbaI FD to part 6.3 A C1 and part 6.3 B C2 RD samples
-          for all reactions: total volume of 40 μl
-          part 6.3 A C1 and part 6.3 B C2 RD digests incubated at 37 °C for 1 h (then dephosphorylation)
-          RBSrev + pSB1C3 RD digest incubated at 37 °C for 2 h (no dephosphorylation, since Promoter hybridization oligos are dephosphorylated!)
AP treatment of part 6.3 A C1 and B C2
-          40 μl reaction + 5 μl 10x AP buffer + 2 μl AP + 3 μl dH2O
-          incubation for ca. 50 min at 37 °C (although only 15 min needed, since XbaI and EcoRI generate 5’ overhangs. But dephosphorylation often incomplete…)
Gel run of digested and AP treated part 6.3 A C1 and BC2 and RBSrev digested with SpeI and EcoRI
-          1%-agarose-1xTAE gel
-          loading of 3 μl  2 log ladder
-          loading of 4 μl of dephosphorylated reactions + 1 μl 5xLD
-          loading of 3 μl RBSrev digest + 1 μl dH2O + 1μl 5xLD
-          run at 100 V
-          EtBr staining + destaining in water
-          UV detection
 
Loading:
Marker/ uncut RBSrev C1/RD RBSrev C1/uncut part6.3 A C1/RD + AP part6.3 A C1/uncut part6.3 B C2/RD + AP part6.3 B C2/Marker
-          no over-digest; digests still incomplete, but since AP treatment of at least part 6.3 A and B vector, this should not matter; in case of RBSrev ligation, re-ligands could occur
Purification of digested and dephosphorylated samples
-          with Qiagen PCR purification kit
-          addition of 500 μl PB buffer to each sample
-          elution 1x with 30 μl HPLC water (pre-warmed), incubation for 2 min at 50 °C
-          NanoDrop concentration measurement
 
Ligation
Ligation of
a) DarRrev insert + Termrev vector →  Termrev-DarRrev in pSB1C3
b) RBSrev hybridization oligo + part6.3 A/part 6.3 B →  part 6.4 A/part 6.4 B (RBSrev-Promoter1/3rev in part 6.2)
c) Promoter1/3rev + pSB1C3 derived from RBSrev in pSB1C3 digest →  Promoter1/3rev in pSB1C3
 
Pipetting scheme
a) DarRrev insert + Termrev vector →  Termrev-DarRrev in pSB1C3
Uni D’dorf ligation calculator: for a ratio vector:insert = 1:3, one has to use 50 ng of a 2200 bp (= 2070 of pSB1C3 + 130 bp of terminator) vector and 44 ng insert of 650 bp →  approximate amounts used in this ligation.
Component
With DarRrev insert
Without insert control
Termrev vector (purified today, 19.9 ng/µl)
3 µl
3 µl
DarRrev insert (purified today, 14.8 ng/µl) or dH2O for w/o insert control
3 µl
3 µl
T4 ligase (ThermoScientific)
1 µl
1 µl
10x T4 ligase buffer (ThermoScientific)
1 µl
1 µl
dH2O
2 µl
2 µl
Total
10 µl
10 µl
 
b) RBSrev hybridization oligo + part6.3 A/part 6.3 B →  part 6.4 A/part 6.4 B (RBSrev-Promoter1/3rev in part 6.2)
Component
With RBSrev insert
Without insert control
Part 6.3 A vector (purified today, 30.5 ng/µl)
2 µl
2 µl
RBSrev hybridization oligo (prepared on 19.8.13) or dH2O for w/o insert control
8 µl
8 µl
T4 ligase (ThermoScientific)
2 µl
2 µl
10x T4 ligase buffer (ThermoScientific)
2 µl
2 µl
dH2O
6 µl
6 µl
Total
20 µl
20 µl
 
Component
With RBSrev insert
Without insert control
Part 6.3 B vector (purified today, 33.6 ng/µl)
2 µl
2 µl
RBSrev hybridization oligo (prepared on 19.8.13) or dH2O for w/o insert control
8 µl
8 µl
T4 ligase (ThermoScientific)
2 µl
2 µl
10x T4 ligase buffer (ThermoScientific)
2 µl
2 µl
dH2O
6 µl
6 µl
Total
20 µl
20 µl
 
c) Promoter1/3rev + pSB1C3 derived from RBSrev in pSB1C3 digest →  Promoter1/3rev in pSB1C3
Component
With Promoter1rev insert
With Promoter3rev insert
Without insert control
psB1C3 vector (derived from RBSrev C1 plasmid, purified today, 29.9 ng/µl)
2 µl
2 µl
2 µl
Promoter1/3rev hybridization oligo (prepared on 19.8.13) or dH2O for w/o insert control
10 µl
10 µl
10 µl
T4 ligase (ThermoScientific)
2 µl
2 µl
2 µl
10x T4 ligase buffer (ThermoScientific)
2 µl
2 µl
2 µl
dH2O
4 µl
4 µl
4 µl
Total
20 µl
20 µl
20 µl
 
-          All reaction were pipetted individually/no mastermix
-          Incubation ON at 16 °C (cold room heat block)
Fold ↑
DNAse I digestion
DNAse I digestion
·         this   digestion is necessary to get rid of DNA contamination
·         use   PCR tubes
25 µl   sample
·         2.5 µg RNA
·         2.5   µl 10 x buffer
·         5 µl   DNAse I
·         x µl   RNAse free water
30   min 37°C in the PCR cycler
Stop   the reaction with 2.5 µl EDTA for 10 min at 65°C (also in the   cycler)
Control PCR on digestion
PCR   Scheme (50 µl)
·         10 µl HF buffer
·         2 µl   dNTPs Mix
·         4 µl   Primer Mix (here rpsE)
·         1 µl   RNA
·         0,5   µl PhuS
·         32 µl   RNAse free H20.
PCR   Programm
·         98°C   2 min
·         98°C   35 sec
·         52°C   35 sec
·         72°C   45 sec
·         72°C   10 min
·         15°C   hold
 
·         run a   gel to check if everything went correctly
In   this case we did this digestion with the wt, GP1344 and GP1346. Additionally   we used a NTC and cDNA control.
Ø  The   digestion worked fine. No DNA detectable in our samples, only in the cDNA
Ø  Determine   RNA concentration
Ø  This   failed
We   did the digestion again. This time we used 6 samples in total, two samples of   each strain. IN the first three samples we used 5 µl DNAse I   according to protocol, in the other three samples we only used 2.5 µl DNAse   I.
Stored   the samples over night at -20°C.
 
Fold ↑
19th
Mini-Preps of RiboA Clones 1, 2, 4 and 5 (see gel from 12.08.),  RD of Ribo 2,4,5, Ligations of the Riboswitches into next vectors, MiniPrep of RBSrev + pSB1C3, Prom1rev + part6.2, Prom3rev + part 6.2 (C1 – C3 each) and PCR test, PCR to test, if RBSrev C1 – C3 are positive or negative, Test restriction digest of Prom1rev + part6.2, Prom3rev + part 6.2 (C1 – C3 each)....
Cloning of Riboswitch A
Mini-Preps of RiboA Clones 1, 2, 4 and 5 (see gel from 12.08.) using the plasmid purification kit
Nanodrop measurements:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Ribo A clone 1
213.9
1.89
1.86
Ribo A clone 2
152.5
1.95
2.00
Ribo A clone 4
396.6
1.90
2.11
Ribo A clone 5
429.4
1.89
2.18
 
Restriction Digestion of
Ribo A clones 4 +5, (R.D. with E + S)
Ribo B clone 2, (R.D. with X + P)
Ribo C clone 4, (R.D. with E + S)
Ribo D clone 5 (R.D. with X + P)
 
1500 ng DNA
3µl enzyme FD
3µl enzyme FD
4µl Buffer FD
40 µl reaction (filled up with H2O)
for 1.5 h at 37°C
 
Gel:
M/plasmid DNA Ribo A clone 4/ R.D. Ribo A clone 4/ plasmid DNA Ribo A clone 5/ R.D. Ribo A clone 5/ plasmid DNA Ribo B clone 2/ R.D. Ribo B clone 2/ plasmid DNA Ribo C clone 4/ R.D. Ribo C clone 4/ plasmid DNA Ribo D clone 5/ R.D. Ribo D clone 5
→ loading of all R.D. Ribos A-D onto Gel for gel extraction
→ Gelextraxtion using the PCR purification kit, nanodrop measurements on the eppis (3-5ng/µl)
Ligations of the Riboswitches into next vectors
Ribo A clones 4 +5, (R.D. with E + S) with CFP and YFP (R.D. with E+X) (see 12.08.)
Ribo B clone 2, (R.D. with X + P) with Part1 and Part3 (R.D. with S+P) (see 12.08.)
Ribo C clone 4, (R.D. with E + S) with Part8 (R.D. with E+X) (see 12.08.)
Ribo D clone 5 (R.D. with X + P) with Part1 and Part3 (R.D. with S+P) (see 12.08.)
For that, all of the inserts (60ng) was used for ligation ino the Vectors (20ng)
0,75-1µl Vectors
12µl Insert (Ribo A-D)
2µl T4 Buffer
2µl T4 Ligase
3-3,25µl H20
20µl reaction
Incubation overnight at 16°C
MiniPrep of RBSrev + pSB1C3, Prom1rev + part6.2, Prom3rev + part 6.2 (C1 – C3 each)
-          kit: Nucleospin, Macherey-Nagel
-          1st elution: 30 μl HPLC-water (pre-warmed), incubation for 2 min at 50 °C
-          2nd elution: 22 μl HPLC-water (pre-warmed), incubation for 2 min at 50 °C
-          no concentration measurement, since NanoDrop is dirty and measures 6 – 7 ng DNA in HPLC water used as a blank before…
-          samples stored in “DarR reporter system” - box
PCR to test, if RBSrev C1 – C3 are positive or negative 
-          RBSrev insert has only 12 bp →  one would not see it in a gel
-          in Colony PCR, there was for all clones a band at height of terminator band (re-ligand)
-          perform PCR to test, if terminator band is really there/ if clones are positive
-          PCR (similar to colony PCR, but with plasmids as a template):
Component
Volume
Taq
1 μl
10x Taq buffer
2.5 μl
dNTP mix
1 μl
iGEM_38 (VF2) 1:20
1 μl
iGEM_39 (VR) 1:20
1 μl
dH2O
17.5 μl
template
1 μl
Total
25 μl
 
 
-          of the plasmids prepped today (see MiniPrep), a 1:20 dilution was prepared, of this dilution, 1 μl was used a template (1 reaction for each clone)
-          as an additional control, part 7 C1 plasmid (dilution from 15.6.13, 1:10) was used
-          Neg. control: 1 μl water as a template
-          preparation of a 6x MasterMix:
 
Component
Volum
Taq
6μl
10x Taq buffe
15μl
dNTP mix
6μl
iGEM_38 (V2) 1:20
6μl
iGEM_39 (VR) 1:20
6μl
dH2O
105μl
Total
144μl
→ addition of 24 μl MasterMix to each sample
-          PCR protocol: same as for colony-PCR
Test restriction digest of Prom1rev + part6.2, Prom3rev + part 6.2 (C1 – C3 each) 
-          of the plasmids prepped today, 2 μl were used for the digest, since the concentration could not be measured with NanoDrop
-          one reaction contained
Component
Volume
plasmid
2μl
10x FD Green buffer
1μl
EcoRI FD
1μ
PstI FD
1μl
dH2O
5μl
Total
10μl
 
-          the plasmid was added to the tubes, then addition of 8 μl MasterMix; the MasterMix consisted of…
Component
Volume
10x FD Green buffer
7μl
EcoRI FD
7μl
PstI FD
7μl
dH2O
35μl
Total
56μl
 
 
-          incubation at 37 °C for 1 h
 
Gel run:
 
-          1 % agarose-1xTAE gel
-          loading of 3 μl 2 log ladder
-          loading of 1 μl of uncut plasmid + 3 μl dH2O + 1 μl 5xLD
-          loading of 5 μl of RD reaction
-          run at 100 V
-          EtBr staining + destaining
-          UV detection
 Loading: Marker/Prom1rev + part6.2 C1 uncut/ Prom1rev + part6.2 C1 RD/ Prom1rev + part6.2 C2 uncut/ Prom1rev + part6.2 C2 RD/ Prom1rev + part6.2 C3 uncut/ Prom1rev + part6.2 C3 RD/ Prom3rev + part6.2 C1 uncut/ Prom3rev + part6.2 C1 RD/ Prom3rev + part6.2 C2 uncut/ Prom3rev + part6.2 C2 RD/ Prom3rev + part6.2 C3 uncut/ Prom3rev + part6.2 C3 RD/Marker
part 6.2 Re-ligand: 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 941 → ca. 940 bp should be visible in addition to pSB1C3 (2070 bp)
part 6.2 C1 plasmid and Promoter1rev/Promoter3rev: 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1061 → ca. 1060 bp should be visible in addition to pSB1C3 (2070 bp)
 
Conclusion:
All clones seemed to be positive, since fragments have slightly more than 1 kb and ca. 2 kb. Sequencing of C1 and C2 of each.
Sequencing results from 16.8.13
Terminatorrev + pSB1C3 C1 and C3: both clones contain the correct sequence of the terminator, but in reverse direction
DarRrev + pSB1C3 C1:
-          between NotI and PstI restriction site, an additional G is inserted
-          2nd base of 10th codon: A is missing
-          → don’t work with this clone! 
DarRrev + pSB1C3 C4:
-          all mutations from the forward primer are present at correct site
-          no other mutations observed
-          →  work with this clone
Restriction of Terminatorrev + pSB1C3 C1 for generating vector for ligation with DarRrev 
15 μl (ca. 1500 ng) plasmid Termrev C1 (from 16.8.13, 102.8 ng/μl)
4 μl SpeI FD
4 μl FD buffer 10x
17 μl dH2O
in total 40 μl
-         digest: 2 h at 37 °C
Restriction of DarRrev + pSB1C3 C4 for generating the insert for ligation with Termrev C1 vector
6 μl (ca. 1500 ng) plasmid Termrev C1 (from 16.8.13, 282.5 ng/μl)
3μl XbaI FD
3 μl PstI FD
4 μl FD buffer 10x
24μl dH2O
in total 40 μl
-         digest: 2 h at 37 °C
Gel run for DarRrev + pSB1C3 C4 and Terminatorrev + pSB1C3 C1 RDs and RBSrev Test PCRs (all from today, see above) 
-          1 % agarose-1xTAE gel
-          loading of 3 μl 2 log ladder
-          loading of 1 μl of uncut plasmid + 3 μl dH2O + 1 μl 5xLD
-          loading of 5 μl of RD reaction
-          loading of 5 μl PCR reaction (addition of 5 μl of 5xLD to each PCR sample)
-          run at 100 V
-          EtBr staining + destaining
-          UV detection
Loading: Marker/Terminatorrev C1 uncut/Terminatorrev + SpeI RD/DarRrev C4 uncut/DarRrev C4 + XbaI and PstI/RBSrev PCR: neg. control/RBSrev PCR: part 7 C1 plasmid/RBSrev PCR: RBSrev C1/RBSrev PCR: RBSrev C2/RBSrev PCR: RBSrev C3/Marker
-          Termrev C1 is digested incompletely, but since the DarRrev insert won’t be dephosphorylated, one can dephosphorylate the incompletely digested vector →  purifiy vector and digest with PstI
-          DarRrev C4 plasmid is digested completely →  purify DarRrev fragment with gel extraction
-          PCR: the plasmids part 7 C1 and the RBSrev C1, C2 and C3 showed the expected products, but the negative control had a band at ca. 200 – 300 bp (I’ve no explanation for that: the water I used was the same for all other reactions! And the other material as well. Could be Bacillus spores, unclean PCR tubes, or my DNA…? XD →  forget negative control (some people even don’t do no-template controls…), since expected products dominate/bands on neg. control seem not to be in lanes of all other samples →  prepare C1 for sequencing
Hybridization of RBSrev, Promoter1rev and Promoter3rev repeated  
-          RBSrev must be  cloned into Prom1rev/Prom3rev + part6.2 (from now on termed part 6.3A/B)
-          Promoter1rev and Promoter3rev must be cloned into shipping vector
-          hybridization performed as on 13.08.2013
-          samples stored at – 20 °C in DarR reporter system-box
Concentration measurement (NanoDrop worked again!) of plasmids purified today
Preparation of samples for sequencing by SeqLab
-          sample no. 3: Promoter1rev + part6.2 C1 (part 6.3 A C1) + VF2 (6 μl plasmid, purified today + 1 μl primer iGEM_38 1:5)
-          sample no. 4: Promoter1rev + part6.2 C2 (part 6.3 A C1) + VF2 (6 μl plasmid, purified today + 1 μl primer iGEM_38 1:5)
-          sample no. 5: Promoter3rev + part6.2 C1 (part 6.3 B C1) + VF2 (6 μl plasmid, purified today + 1 μl primer iGEM_38 1:5)
-          sample no. 6: Promoter3rev + part6.2 C2 (part 6.3 B C1) + VF2 (6 μl plasmid, purified today + 1 μl primer iGEM_38 1:5)
-          sample no. 7: RBSrev + pSB1C3 C1 + VF2 (6 μl plasmid, purified today + 1 μl primer iGEM_38 1:5)
PCR clean up of Termrev C1 + SpeI digest
-          with Qiagen PCR purification kit
-          500 μl PB buffer used
-          elution 1x with 30 μl HPLC water (pre-warmed), incubation at ca. 57°C for 2 min
sample stored at – 20 °C in DarR reporter system box, marked with green
Gel ex of DarRrev insert 
-          1%-agarose-1xTAE gel
-          addition of 9 μl 5x LD to 37 μl reaction
-          loading of whole reaction on gel
-          loading of 3 μl marker
-          loading of 1 μl uncut DarRrev C4 plasmid + 3 μl dH2O + 1 μl 5xLD
-          run at 85 V
-          5 min staining with EtBr, short destaining in water, UV detection briefly, then gel ex under low UV light, tried to do gel ex, but signal was too weak, so again brief staining and destaining, then gel ex was possible
Gel after gel ex:
-          a bit was lost, but that’s ok…
-          gel pieces weighed and stored in DarR reporter system box at – 20 °C
Fold ↑
Protein expression for crystallization of L. monocytogenes DAC domain
Protein   expression for crystallization of L. monocytogenes DAC domain
-       Preparation of 10l LB medium
-       Inoculation of 2x 50ml LB medium   containing ampicillin with 1013 cryo stock
-       Incubation o/n at 37°C with agitation
-           Measurement of   OD600
-       Inoculation of 10x 1l LB medium   containing ampicillin with culture having OD600=0,1
-       Incubation at 37°C until OD600=0,5-0,7
-       Add each 1ml of 1M IPTG to each 1l of   culture
-       Incubation for 12-14h at 16°C with   agitation
-       Measurement of OD600 and   determine protein concentration and save samples for SDS-PAGE
-       Transfer each 500ml of cultures into   centrifugation tubes and centrifuge for 10min at 5000rpm at 4°C
-       Resuspend pellet in 20ml buffer W
-       Pool solution of 2 pellets  and   transfer protein solution into falcon tube
-       Centrifugation for 15min at 5000rpm at   4°C
-       Discard supernatant   and store pellet at -20°C
 
Fold ↑
18th
Riboswitch A Clones 1, 2, 4 and 5 inocculated into 4ml Cm cultures for minipreps tomorrow
Fold ↑
Protein expression for crystallization of L. monocytogenes DAC domain
Protein   expression for crystallization of L. monocytogenes DAC domain
-       Preparation of 10l LB medium
-       Inoculation of 2x 50ml LB medium   containing ampicillin with 1013 cryo stock
-       Incubation o/n at 37°C with agitation
-           Measurement of   OD600
-       Inoculation of 10x 1l LB medium   containing ampicillin with culture having OD600=0,1
-       Incubation at 37°C until OD600=0,5-0,7
-       Add each 1ml of 1M IPTG to each 1l of   culture
-       Incubation for 12-14h at 16°C with   agitation
-       Measurement of OD600 and   determine protein concentration and save samples for SDS-PAGE
-       Transfer each 500ml of cultures into   centrifugation tubes and centrifuge for 10min at 5000rpm at 4°C
-       Resuspend pellet in 20ml buffer W
-       Pool solution of 2 pellets  and   transfer protein solution into falcon tube
-       Centrifugation for 15min at 5000rpm at   4°C
-       Discard supernatant   and store pellet at -20°C
 
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w12 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
