<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_NoteBook_w11"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/NoteBook_w11">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/NoteBook_w11&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w11&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w11&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/NoteBook_w11" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/NoteBook w11</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="tlob" id="tl_0816"><SPAN class="date">16th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Transformation of ligations part6.2 + Promoter1<SUB>rev</SUB> or Promoter3<SUB>rev</SUB> and pSB1C3 + Promoter1<SUB>rev</SUB> , Promoter3<SUB>rev</SUB> or RBS<SUB>rev</SUB>, Colony PCR of clones from transformation of ligations part6.2 + Promoter1<SUB>rev</SUB> or Promoter3<SUB>rev</SUB> and pSB1C3 + Promoter1<SUB>rev</SUB> , Promoter3<SUB>rev</SUB> or RBS<SUB>rev</SUB>, Mini Plasmid Prep of clones inoculated on 15.6.13 and preparation of some cryostocks, Test Restriction Digest of DarR<SUB>rev</SUB> and Term<SUB>rev</SUB> clones, Preparation of samples for sequencing by SeqLab.  </P><DIV class="timeline-cont"><P class="c11-0"><SPAN class="c11-5">Loading of yesterday´s colony PCR of Ribo A-D onto Gel </SPAN></P><P class="c11-0"><SPAN class="c11-5">Gel:</SPAN></P><P class="c11-0"><SPAN class="c11-5">M|Ribo A clone 1-10|RiboB clone 1-5</SPAN></P><P class="c11-0"><SPAN class="c11-5"> </SPAN></P><P class="c11-0"><SPAN class="c11-5">M|Ribo C clone 1-5|RiboD clone 1-5</SPAN></P><P class="c11-0"><SPAN class="c11-5">Minprep of inoculated cultures from RiboA</SPAN></P><P class="c11-0"><SPAN>(wrong eluted -&gt; has to be done next week again)</SPAN></P><P class="c11-0"><SPAN>for R.D. and ligation </SPAN></P><P class="c11-0"><SPAN class="c11-5">Miniprep of inoculated cultures from</SPAN></P><P class="c11-0"><SPAN class="c11-5">Ribo B clones 1 and 2; Ribo C clones 2 and 4 , Ribo D clones 2 and 5</SPAN></P><P class="c11-0"><SPAN>(stored in To do box)</SPAN><SPAN class="c11-5"> </SPAN></P><P class="c11-0"><SPAN>Nanodrop: </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-33"><P class="c11-6"><SPAN class="c11-5">Sample</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN class="c11-5">ng/µl</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN class="c11-5">A</SPAN><SPAN class="c11-5 c11-4">260nm</SPAN><SPAN class="c11-5">/A</SPAN><SPAN class="c11-5 c11-4">280nm</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN class="c11-5">A</SPAN><SPAN class="c11-5 c11-4">260nm</SPAN><SPAN class="c11-5">/A</SPAN><SPAN class="c11-5 c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-33"><P class="c11-6"><SPAN>Ribo B clone 1</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN>11.3</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.71</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.60</SPAN></P></TD></TR><TR><TD class="c11-33"><P class="c11-6"><SPAN>Ribo B clone 2</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN>60.6</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.85</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.76</SPAN></P></TD></TR><TR><TD class="c11-33"><P class="c11-6"><SPAN>Ribo C clone 2</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN>31.8</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.83</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.33</SPAN></P></TD></TR><TR><TD class="c11-33"><P class="c11-6"><SPAN>Ribo C clone 4</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN>58.4</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.81</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.91</SPAN></P></TD></TR><TR><TD class="c11-33"><P class="c11-6"><SPAN>Ribo D clone 2</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN>40.5</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.96</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.99</SPAN></P></TD></TR><TR><TD class="c11-33"><P class="c11-6"><SPAN>Ribo D clone 5</SPAN></P></TD><TD class="c11-45"><P class="c11-6"><SPAN>81.4</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.87</SPAN></P></TD><TD class="c11-32"><P class="c11-6"><SPAN>1.66</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0"><SPAN class="c11-5">Transformation of ligations part6.2 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> and pSB1C3 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> , Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or RBS</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- negative control: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN></P><P class="c11-0 c11-8"><SPAN>- part 6.2 w/o insert: no colonies on 50 μl-plate, but many clones on rest-plate (different sizes: big, small, medium)</SPAN></P><P class="c11-0 c11-8"><SPAN>- part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies on 50 μl-plate, but 8 clones on rest-plate (different sizes: big, small, medium)</SPAN></P><P class="c11-0 c11-8"><SPAN>- part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: 1 colony on 50 μl-plate (big), and 10 clones on rest-plate (different sizes: big, small, medium)</SPAN></P><P class="c11-0 c11-8"><SPAN>- pSB1C3 w/o insert: no colonies on 50 μl-plate, 6 clones on rest-plate (small to medium in size)</SPAN></P><P class="c11-0 c11-8"><SPAN>- pSB1C3 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies on 50 μl-plate, no clones on rest-plate</SPAN></P><P class="c11-0 c11-8"><SPAN>- pSB1C3 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies on 50 μl-plate, no clones on rest-plate</SPAN></P><P class="c11-0 c11-8"><SPAN>- pSB1C3 + RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies on 50 μl-plate, 5 clones on rest-plate (small, medium to big in size) </SPAN></P><P class="c11-0 c11-20"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>colony PCR to see, if any of the clones are positive…</SPAN></P><P class="c11-0 c11-20"><SPAN>-&gt; plates stored at 4°C, big fridge</SPAN></P><P class="c11-0"><SPAN class="c11-5">Colony PCR of clones from transformation of ligations part6.2 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> and pSB1C3 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> , Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or RBS</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- part 6.2 w/o insert: 1 medium – small clone for re-ligand control picked (orange tube, R)</SPAN></P><P class="c11-0 c11-8"><SPAN>- part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: all clones picked (termed C1 – C8) (blue tubes, 1 – 8)</SPAN></P><P class="c11-0 c11-8"><SPAN>- part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: all clones picked (termed C1 – C11) (green tubes, 1 – 11)</SPAN></P><P class="c11-0 c11-8"><SPAN>- contol: plasmid part 6.2 C1 (from 9.8.13, 261.5 ng/μl, dilution 1:20 --&gt; ca. 13 ng/μl) (white tube, p6.2) </SPAN></P><P class="c11-0 c11-8"><SPAN>- pSB1C3 w/o insert: 1 small clone picked for re-ligand control (termed R4) (pink tube, R)</SPAN></P><P class="c11-0 c11-8"><SPAN>- pSB1C3 + RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: all clones picked (violet tubes, 1 – 5)</SPAN></P><P class="c11-0 c11-8"><SPAN>- control: plasmid part 7 C1 (dilution from 15.8.13 with ca. 12 ng/μl used) (white tube, p7) </SPAN></P><P class="c11-0 c11-8"><SPAN>- colony PCR was done as described previously (10.7.13)</SPAN></P><P class="c11-0 c11-8"><SPAN>- 30x MasterMix prepared, distribution of 15 μl to 28 tubes, then addition of 1 μl plasmid dilution or inoculation with clones after streak-out on master plates --&gt; master plates incubated over day at 37 °C</SPAN></P><P class="c11-0 c11-8"><SPAN>- protocol as usual (4 min elongation time should be enough for ca. 1500 bp in case of part6.2 plasmids)</SPAN></P><P class="c11-0"><SPAN class="c11-5">Mini Plasmid Prep of clones inoculated on 15.6.13 and preparation of some cryostocks </SPAN></P><P class="c11-0 c11-8"><SPAN>- preparation of DMSO cryostocks of CFP C1, YFP C1 and part6.2 C1 Re-trafo clone 1 (done as described before)</SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-29">MiniPrep of remaining culture and Terminator</SPAN><SPAN class="c11-4 c11-29">rev</SPAN><SPAN class="c11-29"> and DarR</SPAN><SPAN class="c11-4 c11-29">rev</SPAN><SPAN class="c11-29"> clones:</SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN>kit: Nucleospin, Macherey-Nagel (while working, DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C6 column fell on bench --&gt; contaminated?)</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1</SPAN><SPAN class="c11-19">st</SPAN><SPAN> elution with 30 μl pre-warmed HPLC water, incubating at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-8"><SPAN>- 2</SPAN><SPAN class="c11-19">nd</SPAN><SPAN> elution with 22 μl pre-warmed HPLC water, incubating at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-8"><SPAN>- concentration determination with NanoDrop:</SPAN></P><P class="c11-0"><SPAN class="c11-5">Test Restriction Digest of DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> and Term</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> clones </SPAN></P><P class="c11-0 c11-1"><SPAN>- Composition of final reactions: 1 μl PstI FD, 1 μl EcoRI FD, 1 μl 10x FD Green buffer, 2 μl plasmid mix (ca. 200 – 300 ng plasmid), 5 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O --&gt; in total: 10 μl</SPAN></P><P class="c11-0 c11-1"><SPAN>- since DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plasmid solutions had such a high concentration, a 1:3 dilution was prepared (1 μl plasmid + 2 μl water) and 2 μl of this dilution used for the RD</SPAN></P><P class="c11-0 c11-1"><SPAN>- for Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plasmids, 2 μl were used, since the concentration was that low</SPAN></P><P class="c11-0 c11-1"><SPAN>- a MasterMix for 9 reactions was prepared, consisting of the enzymes, the buffer and the water</SPAN></P><P class="c11-0 c11-1"><SPAN>- the plasmid mix was prepared in the tubes, then 8 μl of the MasterMix were added to the plasmid mix</SPAN></P><P class="c11-0 c11-1"><SPAN>- the reactions were incubated at 37 °C for 1 h</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Gel run for Test Restriction Digest of DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> and Term</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> clones and Colony PCR of clones from transformation of ligations part6.2 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> and pSB1C3 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> , Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or RBS</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>- 4x 1 %-agarose-1xTAE gels poured</SPAN></P><P class="c11-0 c11-1"><SPAN>- loading of 3 μl 2 log ladder as a marker</SPAN></P><P class="c11-0 c11-1"><SPAN>- addition of 5 μl 5x LD to the PCR reactions, loading of 5 μl of each reaction</SPAN></P><P class="c11-0 c11-1"><SPAN>- loading of 5 μl of the RD reactions</SPAN></P><P class="c11-0 c11-1"><SPAN>- loading of 1 μl uncut plasmid of the RD reactions + 3 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O + 1 μl LD</SPAN></P><P class="c11-0 c11-1"><SPAN>- loading of 1 μl of plasmid purified from part6.2 C1 Re-trafo C1 and YFP C1 and CFP C1</SPAN></P><P class="c11-0 c11-1"><SPAN>- run at 65 – 100 V (the voltage decreased during the run…)</SPAN></P><P class="c11-0 c11-1"><SPAN>- EtBr staining and destaining in water</SPAN></P><P class="c11-0 c11-1"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>Gel 1: </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/part 7 plasmid/ pSB1C3+RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> Re-ligand R4/pSB1C3+RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1/ pSB1C3+RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C2/ pSB1C3+RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C3/ pSB1C3+RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4/ pSB1C3+RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C5/part6.2 C1 plasmid/ part 6.2 Re-ligand/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C2/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C3/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C5/Marker</SPAN></P><P class="c11-0 c11-1"><SPAN>- expected product sizes (approximately):</SPAN></P><P class="c11-0 c11-1"><SPAN>part 7 plasmid and RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> Re-ligand: 310 bp (VF2, VR) + 130 bp (Terminator) = 440 bp </SPAN><SPAN class="c11-17">--&gt; band could be detected</SPAN></P><P class="c11-0 c11-1"><SPAN>RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> in pSB1C3: 12 bp + 310 bp (VF2, VR) = 322 bp </SPAN><SPAN class="c11-17">--&gt; band could be detected for all clones, but there is also the band of the terminator seen --&gt; purify plasmids from C1, C2 and C3 and perform a PCR with the plasmids to see if terminator band is also visible</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>part 6.2 C1 plasmid and part 6.2 Re-ligand: 310 bp (VF2, VR) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1251 --&gt; ca. 1250 bp </SPAN><SPAN class="c11-17">--&gt; band could be detected</SPAN></P><P class="c11-0 c11-1"><SPAN>part 6.2 C1 plasmid and Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: 310 bp (VF2, VR) + 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1371 --&gt; ca. 1370 bp </SPAN><SPAN class="c11-17">--&gt; C1 to C5 are apparently positive --&gt; plasmid prep of C1, C2 and C3 and test RD</SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>Gel 2: </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/part6.2 C1 plasmid/ part 6.2 Re-ligand/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C6/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C7/ part 6.2 + Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C8/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C2/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C3/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C5/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C6/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C7/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C8/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C9/Marker</SPAN></P><P class="c11-0 c11-1"><SPAN>- expected product sizes (approximately)</SPAN></P><P class="c11-0 c11-1"><SPAN>part 6.2 C1 plasmid and part 6.2 Re-ligand: 310 bp (VF2, VR) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1251 --&gt; ca. 1250 bp </SPAN><SPAN class="c11-17">--&gt; band could be detected</SPAN></P><P class="c11-0 c11-1"><SPAN>part 6.2 C1 plasmid and Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>/Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: 310 bp (VF2, VR) + 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1371 --&gt; ca. 1370 bp </SPAN><SPAN class="c11-17">--&gt; band is seen for all clones except both C7 (they seem to correspond to re-ligand) --&gt; MiniPrep of C1, C2, C3 and test RD</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>Gel 3: </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/ part6.2 C1 plasmid/ part 6.2 Re-ligand/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C10/ part 6.2 + Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C11/ DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 uncut plasmid/DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 RD/ DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4 uncut plasmid/DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4 RD/ DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C6 uncut plasmid/DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C6 RD/ DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C12 uncut plasmid/DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C12 RD/ DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C13 uncut plasmid/DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C13 RD/Marker</SPAN></P><P class="c11-0 c11-1"><SPAN>- expected product sizes (approximately)</SPAN></P><P class="c11-0 c11-1"><SPAN>part 6.2 C1 plasmid and part 6.2 Re-ligand: 310 bp (VF2, VR) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1251 --&gt; ca. 1250 bp </SPAN><SPAN class="c11-17">--&gt; band could be detected</SPAN></P><P class="c11-0 c11-1"><SPAN>part 6.2 C1 plasmid and Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>/Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: 310 bp (VF2, VR) + 112 bp (rev. Promoter) + 8 bp (Scar between rev. promoter and promoter 3) + 35 bp (Promoter 3) + 8 bp (Scar between promoter and DarR operator) + 14 bp (operator) + 8 bp (Scar between operator and part 6 GFP) + 876 bp (part 6 GFP) = 1371 --&gt; ca. 1370 bp </SPAN><SPAN class="c11-17">--&gt; band is seen for C10, but not for C11 (C11 seems to correspond to re-ligand)</SPAN></P><P class="c11-0 c11-1"><SPAN>- test restriction of DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plasmids – expected product sizes (approximately) for positive clone:</SPAN></P><P class="c11-0 c11-1"><SPAN>2070 bp of pSB1C3 and ca. 690 bp for DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> with Pre-/Suffix sequences </SPAN><SPAN class="c11-17">--&gt; bands are seen for all clones --&gt; prepare C1 and C4 for sequencing</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>Gel 4: </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/ Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 uncut plasmid/Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 RD/ Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C2 uncut plasmid/Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C2 RD/ Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C3 uncut plasmid/Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C3 RD/part6.2 C1 Re-trafo C1 plasmid/YFP C1 plasmid/ CFP C1 plasmid/ Marker</SPAN></P><P class="c11-0 c11-1"><SPAN>- test restriction of Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plasmids – expected product sizes (approximately) for positive clone:</SPAN></P><P class="c11-0 c11-1"><SPAN>2070 bp of pSB1C3 and ca. 190 bp for Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> with Pre-/Suffix sequences </SPAN><SPAN class="c11-17">--&gt; bands are seen for all clones --&gt; prepare C1 and C3 for sequencing</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>- the plasmids of part 6.2 C1 re-trafo C1, CFP C1 and YFP C1 after MiniPrep look normal</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Preparation of samples for sequencing by SeqLab </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Since sequences from last week were so good, more or less the same plasmid amount should be used for this sequencing. </SPAN></P><P class="c11-0 c11-1"><SPAN>- 1 - Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 + VF2 (6 µl plasmid + 1 µl iGEM_38 1:5)</SPAN></P><P class="c11-0 c11-1"><SPAN>- 2 – Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C3 + VF2 (6 µl plasmid + 1 µl iGEM_38 1:5)</SPAN></P><P class="c11-0 c11-1"><SPAN>- 3 – DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 + VF2 (4 µl plasmid + 2 µl HPLC water + 1 µl iGEM_38 1:5)</SPAN></P><P class="c11-0 c11-1"><SPAN>- 4 – DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C1 + VR (4 µl plasmid + 2 µl HPLC water + 1 µl iGEM_39 1:5)</SPAN></P><P class="c11-0 c11-1"><SPAN>- 5 – DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4 + VF2 (5 µl plasmid + 1 µl HPLC water + 1 µl iGEM_38 1:5)</SPAN></P><P class="c11-0 c11-1"><SPAN>- 6 – DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> C4 + VR (5 µl plasmid + 1 µl HPLC water + 1 µl iGEM_39 1:5)</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV><P class="timeline-title goe-array">Growing of cells used for RNA prep</P><DIV class="timeline-cont"><H2><SPAN lang="EN-GB" style="font-size:11.0pt;line-height:115%;font-family:
  &quot;Arial&quot;,&quot;sans-serif&quot;;text-decoration:none">Growing of cells used for RNA prep</SPAN></H2><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Same
  procedure as for the cells used for the array in Groningen (see 25.07.13)</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0815"><SPAN class="date">15th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Terminator<SUB>rev</SUB> transformation, DarR<SUB>rev</SUB> transformation, Re-transformation of part 6.2. C1, Transformation of pSB1C3 + Promoter1<SUB>rev</SUB>, Promoter3<SUB>rev</SUB> or RBS<SUB>rev</SUB>, Clones picked from Riboswitch A-D plates from 13.08.13 (Riboswitch in pSB1C3). </P><DIV class="timeline-cont"><P class="c11-0"><SPAN class="c11-5">Terminator</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> transformation</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- no colonies on NC plates (50 μl and rest-plate)</SPAN></P><P class="c11-0 c11-8"><SPAN>- w/o insert control plates: no colonies on 50 μl-plate, but some on rest-plate (different sizes)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plates: no colonies on 50 μl-plate, but several on rest-plate (different sizes) </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-5">Colony PCR --&gt; use primer combination that indicates, if Terminator is oriented in the desired way in plasmid:</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- 1x w/o insert control plate clone (re-ligand), size: medium – big; primers iGEM_38 (VF2) and iGEM_71 (primer fwd Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>; PstI and SpeI restriction sites of suffix) --&gt; should give no PCR product, since both primers read into the same direction </SPAN></P><P class="c11-0 c11-8"><SPAN>- 12 x Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plate clones (termed C1 – C12); size: small, medium or big; , primers iGEM_38 (VF2) and iGEM_71 (primer fwd Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>; PstI and SpeI restriction sites of suffix) </SPAN></P><P class="c11-0 c11-20"><SPAN>--&gt; should give PCR product if the terminator is oriented reverse (the desired way) of size of terminator with 20 bp of Prefix and 120 additional bp of pSB1C3 backbone caused by VF2;</SPAN></P><P class="c11-0 c11-20"><SPAN>--&gt; should give no PCR product, if Terminator is oriented in the normal way </SPAN></P><P class="c11-0 c11-8"><SPAN>- 1x control with part 7 C1 plasmid (from 11.7.13, 120.5 ng/μl --&gt; fresh 1:10 dilution);  primers iGEM_38 (VF2) and iGEM_72 (primer rev Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>; EcoRI and XbaI restriction sites of prefix) --&gt; should give PCR product of size of terminator (ca. 130 bp) with 20 bp of Prefix and 120 additional bp of pSB1C3 backbone caused by VF2 </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-29">procedure:</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-20"><SPAN>part 7 C1 plasmid control was pipetted individually; 1 μl of 1:10 dilution (i.e. ca. 12 ng plasmid) used, therefore, only 17.5 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O were added </SPAN></P><P class="c11-0 c11-20"><SPAN>for the reactions containing </SPAN><SPAN class="c11-17">E. coli</SPAN><SPAN> cells, a 15x MasterMix was prepared and distributed as usual (see Journal I, for protocol) to 13 epis, which were subsequently inoculated </SPAN></P><P class="c11-0 c11-20"><SPAN>preparation of a MasterPlate + incubation of plate at 37 °C over day </SPAN></P><P class="c11-0 c11-20"><SPAN>PCR protocol was the same as usual (iGEM_71 has a T</SPAN><SPAN class="c11-4">M</SPAN><SPAN> = 60°C and iGEM_72 has a T</SPAN><SPAN class="c11-4">M</SPAN><SPAN> = 59.1°C which are similar to that of iGEM_38 and iGEM_39 with 60 °C --&gt; T</SPAN><SPAN class="c11-4">A</SPAN><SPAN> is the same as always (54°C) </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-29">Gel run:</SPAN></P><P class="c11-0 c11-20"><SPAN>addition of 5 μl of 5x LD to each PCR reaction</SPAN></P><P class="c11-0 c11-20"><SPAN>pouring of 1%-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-20"><SPAN>loading of 3 μl 2 log ladder as a marker</SPAN></P><P class="c11-0 c11-20"><SPAN>loading of 5 μl of each PCR reaction</SPAN></P><P class="c11-0 c11-20"><SPAN>run at 100 V</SPAN></P><P class="c11-0 c11-20"><SPAN>EtBr staining + destaining in water (EtBr bath was not made fresh this weak, so staining was poor; after preparation of new EtBr bath, again few min staining)</SPAN></P><P class="c11-0 c11-20"><SPAN>UV detection </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">loading</SPAN><SPAN>: Marker/part 7/re-ligand/C1/C2/C3/C4/C5/C6/C7/C8/C9/C10/C11/C12/Marker</SPAN></P><P class="c11-0 c11-8"><SPAN>- for part 7 plasmid template, a band at ca. 270 bp could be observed --&gt; height of terminator PCR product</SPAN></P><P class="c11-0 c11-8"><SPAN>- for re-ligand, no PCR product of this size is seen</SPAN></P><P class="c11-0 c11-8"><SPAN>- for C1 – C3, C5 – C12, the expected band is seen --&gt; positive clones --&gt; inoculation of C1, C2 and C3 from Backup plate in 4 ml LB</SPAN><SPAN class="c11-19">Cm</SPAN><SPAN> for MiniPrep, incubation ON at 37 °C</SPAN></P><P class="c11-0 c11-8"><SPAN>- backup plate put to 4 °C, big fridge</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> transformation</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>- though the transformation was done only yesterday, the plates showed already several colonies:</SPAN></P><P class="c11-0 c11-1"><SPAN>- Neg. control plates: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- w/o insert plate: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> plate: 1 colonie on 50 μl-plate, and many colonies on rest-plate (different sizes; small or medium-big; but clones in general smaller compared to Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> clones, probably because DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> clones grew just overnight) </SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Colony PCR --&gt; primer combination does not matter, since only 1 orientation of DarR should be possible; additionally, DarR primers make problems during PCR and have a T</SPAN><SPAN class="c11-5 c11-4">M</SPAN><SPAN class="c11-5"> diverging from that of VR and VF2)</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>- 1x control with part 7 C1 plasmid (from 11.7.13, 120.5 ng/μl --&gt; fresh 1:10 dilution);  primers iGEM_38 (VF2) and iGEM_39  --&gt; should give PCR product of size of terminator (ca. 130 bp) with 2x20 bp of Prefix and Suffix and 120 additional bp of pSB1C3 backbone caused by VF2 and 150 additional bp of pSB1C3 backbone caused by VR (= 440 bp) </SPAN></P><P class="c11-0 c11-1"><SPAN>- 13x DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> clones (termed C1 – C13) primers iGEM_38 (VF2) and iGEM_39  --&gt; should give PCR product of size of DarR (ca. 630 bp) with 2x20 bp of Prefix and Suffix and 120 additional bp of pSB1C3 backbone caused by VF2 and 150 additional bp of pSB1C3 backbone caused by VR (= 940 bp)</SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-1"><SPAN>- </SPAN><SPAN class="c11-29">procedure:</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>15x MasterMix was prepared and distributed as usual (see Journal I, for protocol) to 14 epis, </SPAN></P><P class="c11-0 c11-8"><SPAN>part 7 C1 plasmid control: To 1 μl plasmid dilution, 25 μl MasterMix were added</SPAN></P><P class="c11-0 c11-8"><SPAN>25 μl PCR reaction were inoculated with </SPAN><SPAN class="c11-17">E. coli</SPAN><SPAN> cells </SPAN></P><P class="c11-0 c11-8"><SPAN>preparation of a MasterPlate + incubation of plate at 37 °C over day </SPAN></P><P class="c11-0 c11-8"><SPAN>PCR protocol was the same as usual </SPAN></P><P class="c11-0 c11-1"><SPAN>- </SPAN><SPAN class="c11-29">Gel run:</SPAN></P><P class="c11-0 c11-8"><SPAN>addition of 5 μl of 5x LD to each PCR reaction</SPAN></P><P class="c11-0 c11-8"><SPAN>pouring of 1%-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>loading of 3 μl 2 log ladder as a marker</SPAN></P><P class="c11-0 c11-8"><SPAN>loading of 5 μl of each PCR reaction</SPAN></P><P class="c11-0 c11-8"><SPAN>run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>EtBr staining + destaining in water (EtBr bath was not made fresh this weak, so staining was poor; after preparation of new EtBr bath, again few min staining)</SPAN></P><P class="c11-0 c11-8"><SPAN>UV detection </SPAN></P><P class="c11-0"><SPAN class="c11-5">loading</SPAN><SPAN>: Marker/part 7/C1/C2/C3/C4/C5/C6/C7/C8/C9/C10/C11/C12/C13/Marker</SPAN></P><P class="c11-0 c11-1"><SPAN>- for part 7 plasmid template, a band at ca. 440 bp could be observed --&gt; height of terminator PCR product</SPAN></P><P class="c11-0 c11-1"><SPAN>- for all other clones, a band at ca. 250 bp could be seen, this does not correspond to the expected band of ca. 940 bp (250 bp band is only weak for C 8)</SPAN></P><P class="c11-0 c11-1"><SPAN>- but a band at ca. 900 bp is observed for C1, C4, C6, C12, and C13</SPAN></P><P class="c11-0 c11-1"><SPAN>- in addition to that band, several other bands, sometimes stronger are seen for several different clones</SPAN></P><P class="c11-0 c11-1"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>it could be, that the short primers VF2 and VR bind non-specifically within DarR leading to several other products shorter than the expected one (alignment of the 6 3’ end bases of VR shows that it could bind to the start codon plus the first codon of DarR… this would however only give rise to a product of ca. 140 bp)</SPAN></P><P class="c11-0 c11-1"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>colony PCR seems to be fruitless --&gt; inoculation of C1, C4, C6, C12 and C13 from backup plates in 4 ml LB</SPAN><SPAN class="c11-19">Cm</SPAN><SPAN> for MiniPrep, incubation ON at 37 °C</SPAN></P><P class="c11-0 c11-1"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>backup plate put to 4 °C, big fridge</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Re-transformation of part 6.2. C1 </SPAN></P><P class="c11-0 c11-1"><SPAN>- part 6.2. C1 clones grew (few on 50 μl plate; many on rest plate)</SPAN></P><P class="c11-0 c11-1"><SPAN>- Neg. control plates showed no growth </SPAN></P><P class="c11-0 c11-1"><SPAN>- 3 clones were picked and streaked out on plate for making a Backup-plate --&gt; incubation ON at 37 °C</SPAN></P><P class="c11-0 c11-1"><SPAN>- Clone 1 was inoculated in 4ml LB</SPAN><SPAN class="c11-19">Cm</SPAN><SPAN> and incubated ON at 37 °C for MiniPrep</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Transformation of part6.2 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or Promoter 3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> </SPAN></P><P class="c11-0 c11-1"><SPAN>- Neg. control plates showed no growth</SPAN></P><P class="c11-0 c11-1"><SPAN>- w/o insert control: no colonies on 50 μl plate, several clones on rest plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no clones on 50 μl plate, 3 clones observed on rest plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: 1 clone on 50 μl plate, 1 clone on rest plate </SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>further incubation</SPAN></P><P class="c11-0"><SPAN class="c11-5">Transformation of pSB1C3 + Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5">, Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or RBS</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> </SPAN></P><P class="c11-0 c11-1"><SPAN>- Neg. control plates showed no growth</SPAN></P><P class="c11-0 c11-1"><SPAN>- w/o insert control: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN></P><P class="c11-0 c11-1"><SPAN>- RBS3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>: no colonies, neither on 50 μl-plate nor on rest-plate</SPAN><SPAN class="c11-5"> </SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>further incubation</SPAN></P><P class="c11-0"><SPAN class="c11-5">Preparation of new LB</SPAN><SPAN class="c11-5 c11-19">Cm</SPAN><SPAN class="c11-5"> plates</SPAN><SPAN> </SPAN></P><P class="c11-0"><SPAN>plates prepared from 2x 500 ml </SPAN></P><P class="c11-0"><SPAN class="c11-5">Preparation of new Cm (35 mg/ml)</SPAN><SPAN> </SPAN></P><P class="c11-0"><SPAN>354 mg Cm dissolved in 10.114 ml EtOH 70% dissolved, filter-sterilized, stored in our freezer box – 20 °C</SPAN></P><P class="c11-0"><SPAN> (calculation of ethanol that has to be added to a certain amount of Cm:</SPAN></P><P class="c11-0"><SPAN>354 mg Cm/x ml EtOH 70 % = 35 mg/ml ó 354 mg/35 mg = x ml EtOH 70%)</SPAN></P><P class="c11-0"><SPAN class="c11-5">Clones picked from Riboswitch A-D plates from 13.08.13 (Riboswitch in pSB1C3) </SPAN></P><P class="c11-0"><SPAN>Ribo A: 10 Clones (Native Promoter and RBS, our most important construct to be cloned tomorrow)</SPAN></P><P class="c11-0"><SPAN>Ribo B-D: 5 Clones </SPAN></P><P class="c11-0"><SPAN>They were inocculated into 4ml Cm Cultures for incubation at 37°C on the shaker.</SPAN></P><P class="c11-0"><SPAN>A Backup plate was plated with all clones </SPAN></P><P class="c11-0"><SPAN>For all clones, a colony PCR was set after the normal protocol: </SPAN></P><P class="c11-0 c11-8"><SPAN>2.5µl Taq buffer</SPAN></P><P class="c11-0 c11-8"><SPAN>1µl Taq</SPAN></P><P class="c11-0 c11-8"><SPAN>1µl NTPs</SPAN></P><P class="c11-0 c11-8"><SPAN>1µl VF2 Primer (38)</SPAN></P><P class="c11-0 c11-8"><SPAN>1µl VR Primer (39)</SPAN></P><P class="c11-0 c11-8"><SPAN>18.5µl H2O</SPAN></P><P class="c11-0 c11-8"><SPAN>25µl reaction</SPAN></P><P class="c11-0"><SPAN> Program: </SPAN></P><P class="c11-0 c11-8"><SPAN>94°C 5 min.</SPAN></P><P class="c11-0 c11-8"><SPAN>94°C 45s</SPAN></P><P class="c11-0 c11-8"><SPAN>54°C 40s</SPAN></P><P class="c11-0 c11-8"><SPAN>72°C 4 min.</SPAN></P><P class="c11-0 c11-8"><SPAN>72°C 10 min.</SPAN></P><P class="c11-0 c11-8"><SPAN>15°C HOLD</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV><P class="timeline-title goe-array">Harvest cells</P><DIV class="timeline-cont"><H2><SPAN lang="EN-GB" style="font-size:11.0pt;line-height:115%;font-family:
  &quot;Arial&quot;,&quot;sans-serif&quot;;text-decoration:none">Harvest cells</SPAN></H2><P class="MsoListParagraphCxSpFirst" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Cells
  were harvested with an OD<SUB>600</SUB> = 1</SPAN></P><P class="MsoListParagraphCxSpMiddle" style="margin-left:72.0pt;text-indent:
  -18.0pt"><SPAN lang="EN-GB" style="font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">wt: OD = 1,06</SPAN></P><P class="MsoListParagraphCxSpMiddle" style="margin-left:72.0pt;text-indent:
  -18.0pt"><SPAN lang="EN-GB" style="font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1344: OD = 0,94</SPAN></P><P class="MsoListParagraphCxSpLast" style="margin-left:72.0pt;"><SPAN lang="EN-US" style="font-family:&quot;Courier New&quot;">o<SPAN style="font:7.0pt &quot;Times New Roman&quot;">  </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1346:
  OD = 1,06</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0814"><SPAN class="date">14th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Test PCR for specificity of DarRrev Primer/contamination of DarRrev PCR reactions, Transformation of ligation reactions from 13.8.13 and re-trafo of part 6.2 C1 plasmid , Plasmid MiniPrep of part 8 C1, Tidying up our – 20 °C freezer boxes…</P><DIV class="timeline-cont"><P class="c11-0 c11-1"><SPAN class="c11-5">Test PCR for specificity of DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> Primer/contamination of DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR reactions</SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Neg. control was not negative --&gt; probably, primers or water are contaminated with </SPAN><SPAN class="c11-17">M. smegmatis</SPAN><SPAN> chrom.DNA (most likely because of order of pipetting the reactions) or they are non-specific generating another PCR product of the same size as DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> (seems very unlikely)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Test PCR:</SPAN></P><P class="c11-0 c11-8"><SPAN>- 3 reactions, all reactions were pipetted individually (no mastermix)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Old water from yesterday and old primer dilutions (1:20) from yesterday used (reaction A)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Old water from yesterday and new primer dilutions (1:20) from today used (reaction B)</SPAN></P><P class="c11-0 c11-8"><SPAN>- New water from today and new primer dilutions 1:20 from today used (reaction C)</SPAN></P><P class="c11-0 c11-8"><SPAN>- All 3 reactions had the same composition as the neg. control reaction yesterday</SPAN></P><P class="c11-0 c11-8"><SPAN>- PCR program was the same as yesterday</SPAN></P><P class="c11-0 c11-8"><SPAN>- Gel run:</SPAN></P><P class="c11-0 c11-20"><SPAN>1%-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-20"><SPAN>Loading of 3 µl 2 log ladder as a marker</SPAN></P><P class="c11-0 c11-20"><SPAN>Loading of 4 µl PCR reaction (A, B, C or neg. control from yesterday’s PCR) + 1 µl 5xLD</SPAN></P><P class="c11-0 c11-20"><SPAN>Loading of 3 µl purified DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product (from yesterday) + 1 µl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O + 1 µl 5xLD</SPAN></P><P class="c11-0 c11-20"><SPAN>Run at 100 V</SPAN></P><P class="c11-0 c11-20"><SPAN>EtBr staining + destaining with water</SPAN></P><P class="c11-0 c11-20"><SPAN>UV detection</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Loading:</SPAN><SPAN> 2 log ladder/reaction A/reaction B/reaction C/neg. control from yesterday/purified DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product from yesterday/ 2 log ladder/ reaction A/reaction B/reaction C/neg. control from yesterday/purified DarR PCR product from yesterday/ 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>The PCR product (ca. 630 bp + 2x 30bp = 690 bp) is only seen, when all old components are used, but not if new primers and old water are used --&gt; old primer dilutions are contaminated --&gt; dilutions thrown away</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Since DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>-no template control was apparently positive because of not exchanging the pipet tip before adding the primers, the insert obtained from yesterday’s PCR will be further used for cloning…</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Transformation of ligation reactions from 13.8.13 and re-trafo of part 6.2 C1 plasmid</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-17">E.coli</SPAN><SPAN> XL1-Blue comp. cells used</SPAN></P><P class="c11-0 c11-8"><SPAN>- According to methods folder</SPAN></P><P class="c11-0 c11-8"><SPAN>- Neg. control contained 20 µl sterile dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- For re-trafo of part 6.2 C1 plasmid, 1 µl of plasmid from purification on 9.8.13 was used</SPAN></P><P class="c11-0 c11-8"><SPAN>- For transformation of ligations, the whole ligation reactions were pipetted to the comp. cells</SPAN></P><P class="c11-0 c11-8"><SPAN>- After centrifugation, 500 µl supernatant were removed; strangely, the epi with pSB1C3+Prom1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> contained less than 500 µl solution (but cell pellet was at the bottom of the tube and the colour corresponded to that of LB + comp. cells seen in all other tubes… possibly less comp. cells?)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Plating of all reactions on LB</SPAN><SPAN class="c11-19">Cm</SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN>Incubation at 37 °C</SPAN></P><P class="c11-0"><SPAN class="c11-5">Plasmid MiniPrep of part 8 C1</SPAN></P><P class="c11-0 c11-8"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Culture at &gt; 4 ml…</SPAN></P><P class="c11-0 c11-8"><SPAN>- Elution</SPAN></P><P class="c11-0 c11-20"><SPAN>1</SPAN><SPAN class="c11-19">st</SPAN><SPAN>: 30 µl HPLC-water (pre-warmed) while incubating at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-20"><SPAN>2</SPAN><SPAN class="c11-19">nd</SPAN><SPAN>: 22 µl HPLC-water (pre-warmed) while incubating at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-20"><SPAN>Both elutions were collected in the same tube</SPAN></P><P class="c11-0 c11-8"><SPAN>- Concentration (NanoDrop)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-37"><P class="c11-10 c11-9"><SPAN class="c11-5">Sample</SPAN></P></TD><TD class="c11-37"><P class="c11-10 c11-9"><SPAN class="c11-5">ng/µl</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN class="c11-5">A</SPAN><SPAN class="c11-5 c11-4">260nm</SPAN><SPAN class="c11-5">/A</SPAN><SPAN class="c11-5 c11-4">280nm</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN class="c11-5">A</SPAN><SPAN class="c11-5 c11-4">260nm</SPAN><SPAN class="c11-5">/A</SPAN><SPAN class="c11-5 c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-37"><P class="c11-10 c11-9"><SPAN class="c11-5">Part 8 C1 plasmid</SPAN></P></TD><TD class="c11-37"><P class="c11-10 c11-9"><SPAN>152.2</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN>1.78</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN>1.35</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-1"><SPAN class="c11-5">Tidying up our – 20 °C freezer boxes…</SPAN></P><P class="c11-0 c11-1"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- I found a tube where the date and the concentration where difficult to decipher. It was part 7 C1 plasmid. It could be the plasmid purified by myself on 12.7.13 with a concentration of 179.3 ng/µl. But since I wasn’t sure about that I measured the concentration (NanoDrop) again: </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-37"><P class="c11-9 c11-10"><SPAN class="c11-5">Sample</SPAN></P></TD><TD class="c11-37"><P class="c11-10 c11-9"><SPAN class="c11-5">ng/µl</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN class="c11-5">A</SPAN><SPAN class="c11-5 c11-4">260nm</SPAN><SPAN class="c11-5">/A</SPAN><SPAN class="c11-5 c11-4">280nm</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN class="c11-5">A</SPAN><SPAN class="c11-5 c11-4">260nm</SPAN><SPAN class="c11-5">/A</SPAN><SPAN class="c11-5 c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-37"><P class="c11-10 c11-9"><SPAN class="c11-5">Part 7 C1 plasmid</SPAN></P></TD><TD class="c11-37"><P class="c11-10 c11-9"><SPAN>144.1</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN>1.82</SPAN></P></TD><TD class="c11-31"><P class="c11-10 c11-9"><SPAN>1.37</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN> - We still had the senseless RBS-GFP-Terminator clones; I threw them away, as well…</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0813"><SPAN class="date">13th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">P3op + pSB1C3 C1: Cryo-Stock and MiniPrep, DarRrev PCR repeated – with Phusion® and GC buffer, EcoRI and PstI double digest of purified DarRrev PCR product , XbaI-digest of part6.2. C1 linearized with EcoRI , PCR clean-up of XbaI- and EcoRI-digested part6.2 C1 plasmid , Hybridization of oligos for Promoter3rev, Promoter1<SUB>rev</SUB> and RBS<SUB>rev</SUB></P><DIV class="timeline-cont"><P class="c11-0"><SPAN class="c11-5">P3op + pSB1C3 C1: Cryo-Stock and MiniPrep</SPAN></P><P class="c11-0 c11-8"><SPAN>- preparation of DMSO cryo-stock (900 μl ON culture + 100 μl DMSO 100%, vortexing, put directly to – 80 °C in our box)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Mini Plasmid Preparation of remaining ON culture using Nucleospin kit from Macherey-Nagel (I accidently used 750 μl A4 instead of 600 μl for washing of DNA on column); elution 1x with 30 μl HPLC-water (pre-warmed), incubation at 50 °C for 1 – 2 min)</SPAN></P><P class="c11-0 c11-8"><SPAN>- NanoDrop concentration measurement </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-44"><P class="c11-7 c11-41"><SPAN>Sample</SPAN></P></TD><TD class="c11-16"><P class="c11-7 c11-41"><SPAN>ng/μl</SPAN></P></TD><TD class="c11-36"><P class="c11-7 c11-41"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-36"><P class="c11-7 c11-41"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-44"><P class="c11-7 c11-41"><SPAN>P3op in pSB1C3 C1 plasmid</SPAN></P></TD><TD class="c11-16"><P class="c11-7 c11-41"><SPAN>151.0</SPAN></P></TD><TD class="c11-36"><P class="c11-7 c11-41"><SPAN>1.92</SPAN></P></TD><TD class="c11-36"><P class="c11-7 c11-41"><SPAN>1.79</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN> - stored in to-do-Box</SPAN></P><P class="c11-0"><SPAN class="c11-5">DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR – Gel run</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1st try: loading of gel was difficult because of deformed wells --&gt; some sample was spilled --&gt; but this should be a semi-quantitative gel --&gt; gel thrown away and new gel was poured and loaded</SPAN></P><P class="c11-0 c11-8"><SPAN>- 2nd try:</SPAN></P><P class="c11-0 c11-20"><SPAN>1% Agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-20"><SPAN>loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-20"><SPAN>loading of 4 μl of each PCR reaction + 1 μl 5xLD</SPAN></P><P class="c11-0 c11-20"><SPAN>run at 100 V</SPAN></P><P class="c11-0 c11-20"><SPAN>EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-20"><SPAN>UV detection</SPAN></P><P class="c11-0"><SPAN> </SPAN><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/Negative control/ 2 μl primer reaction/ 4 μl primer reaction/ 8 μl primer reaction/---</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>for 2 μl primer reaction and 4 μl primer reaction, a very weak band at ca 700 bp (approx. bp of DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product: 630 bp + 2x 30 bp = 690 bp) was detected</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>strength of this band seemed to decrease with increasing primer amounts</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>PCR did not work properly: </SPAN><SPAN class="c11-17">M. smegmatis</SPAN><SPAN> has high GC content, but normal HF buffer and PfuS were used --&gt; try comercial Phusion® and GC buffer, only testing of 2 and 4 μl primer amount (lower concentration seem to be more optimal…</SPAN></P><P class="c11-0 c11-8"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR repeated – with Phusion® and GC buffer</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Reactions pipetted as before (see 12.8.13, DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR with normal PfuS and HF buffer)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Volume 50 μl</SPAN></P><P class="c11-0 c11-8"><SPAN>- 4x Master Mix consiting of 5x GC buffer, water, Phusion® and dNTPs</SPAN></P><P class="c11-0 c11-8"><SPAN>- preparation of template (dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O for neg. control, </SPAN><SPAN class="c11-17">M. smegmatis</SPAN><SPAN> chrom.DNA for other reactions), primers (iGEM_83, iGEM_84 and a part of water for 2 μl primer and neg. control reaction); then, addition of required volume of mastermix</SPAN></P><P class="c11-0 c11-8"><SPAN>- Testing of 2 and 4 μl primer; neg. control (dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O as template and 2 μl as primers)</SPAN></P><P class="c11-0 c11-8"><SPAN>- PCR protocol: the same as for PfuS in HF buffer </SPAN></P><P class="c11-0 c11-20"><SPAN>1% Agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-20"><SPAN>loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-20"><SPAN>loading of 4 μl of each PCR reaction + 1 μl 5xLD</SPAN></P><P class="c11-0 c11-20"><SPAN>run at 100 V</SPAN></P><P class="c11-0 c11-20"><SPAN>EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-20"><SPAN>UV detection </SPAN></P><P class="c11-0"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/Negative control/ 2 μl primer reaction/ 4 μl primer reaction</SPAN></P><P class="c11-47 c11-0"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>wells of gel were somehow damaged: this could explain weak band observed for neg. control (some of 2 μl sample ran over well into well of neg. control…) --&gt; same gel as used for control of part6.2 C1 XbaI digest (see below and to check first DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR using PfuS and HF buffer)</SPAN></P><P class="c11-0 c11-47"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>bands of expected product size for both reactions, 2 μl primers and 4 μl primers. But one cannot say, if 4 μl reaction contains really more product than 2 μl reactions since wells were so strange…</SPAN></P><P class="c11-47 c11-0"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>pool and purifiy 2 μl primer and 4 μl primer reaction, digest the PCR product and run it again with the neg. control on a proper gel</SPAN></P><P class="c11-47 c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">PCR clean-up of DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> 2 μl primers and 4 μl primer reactions</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- both reactions were pooled (ca. 100 μl)</SPAN></P><P class="c11-0 c11-8"><SPAN>- addition of 500 μl PB buffer</SPAN></P><P class="c11-0 c11-8"><SPAN>- elution with 1st 30 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min; 2nd 22 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-8"><SPAN>- NanoDrop concentration </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-40"><P class="c11-7 c11-1"><SPAN>Sample</SPAN></P></TD><TD class="c11-33"><P class="c11-7 c11-1"><SPAN>ng/μl</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-40"><P class="c11-7 c11-1"><SPAN>DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product</SPAN></P></TD><TD class="c11-33"><P class="c11-7 c11-1"><SPAN>10.6</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>2.20</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>1.50</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN> - Stored in DarR reporter system-Box</SPAN></P><P class="c11-0"><SPAN class="c11-5"> </SPAN></P><P class="c11-0"><SPAN class="c11-5">EcoRI and PstI double digest of purified DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR product</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- 30 μl DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product (purified today) + 3 μl EcoRI FD + 3 μl PstI FD + 4 μl 10x FD buffer = 40 μl digest reaction</SPAN></P><P class="c11-0 c11-8"><SPAN>- digest at 37 °C (heating block showed sometimes 37.7 or 37.8 °C though it was set to 37 °C) for 1 h</SPAN></P><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">Gel run</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1% Agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 4 μl neg. control PCR reaction + 1 μl 5xLD</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl of RD reaction + 1 μl 5xLD + 1 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl of uncut PCR product + 1 μl 5xLD + 1 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- (loading of samples for XbaI digest of part6.2 C1, that was already digested with EcoRI, as well – see below)</SPAN></P><P class="c11-0 c11-8"><SPAN>- run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-8"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/neg.control from PCR/uncut purified PCR product/ RD reaction of PCR product/ uncut plasmid part 6.2 C1/ RD reaction plasmid part 6.2 C1 E+ P digest, purified/---</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Negative control is not negative (repeat neg. control PCR tomorrow, to see, if it’s really not negative, but purifiy and ligate DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> insert)</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>After R.D., PCR product is still seen (no over-digestion)</SPAN></P><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">PCR clean-up of DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR product</SPAN></P><P class="c11-0 c11-8"><SPAN>- With Qiagen PCR purification kit</SPAN></P><P class="c11-0 c11-8"><SPAN>- 500 µl PB buffer used</SPAN></P><P class="c11-0 c11-8"><SPAN>- Elution with 30 µl pre-warmed HPLC water, incubating sample for 2 min at 50 °C</SPAN></P><P class="c11-0 c11-8"><SPAN>- NanoDrop concentration measurement:</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-48"><P class="c11-7 c11-1"><SPAN>Sample</SPAN></P></TD><TD class="c11-25"><P class="c11-7 c11-1"><SPAN>ng/μl</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-48"><P class="c11-7 c11-1"><SPAN>DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> insert</SPAN></P></TD><TD class="c11-25"><P class="c11-7 c11-1"><SPAN>6.0</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>1.50</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>1.67</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN>- Stored in DarR reporter system-Box</SPAN></P><P class="c11-0"><SPAN class="c11-5">XbaI-digest of part6.2. C1 linearized with EcoRI</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- ca. 27 μl (mostly) linearized part6.2 C1 from 12.8.13</SPAN></P><P class="c11-0 c11-8"><SPAN>- addition of 4 μl FD buffer 10x, 4 μl XbaI FD, 5 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- in total: 40 μl</SPAN></P><P class="c11-0 c11-8"><SPAN>- digest for 2 h at 37 °C (after 1 h, it has been incomplete up till now…) </SPAN></P><P class="c11-0 c11-8"><SPAN class="c11-5">Gel run</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1% Agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl of RD reaction + 1 μl 5xLD + 1 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 1 μl of uncut plasmid + 1 μl 5xLD + 3 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection </SPAN></P><P class="c11-0"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/uncut plasmid/ RD reaction</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>digest is still incomplete (linearized plasimd at ca. 3 kb), even after 2 h XbaI digest. Band of uncut supercoil plasmid very weak (runs at ca. 2 kb)--&gt; purifiy vector for ligation</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>repeat gel run on proper gel (wells were apparently damaged; see DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR gel run) --&gt; same gel as used for control of DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR reactions (see above; PCR using PfuS and HF buffer, but also the PCR reaction with Phusion® and GC buffer) --&gt; apparently, the last part of the gel had damaged wells (fault of comb? Or because of multiple runs?) </SPAN></P><P class="c11-0 c11-50"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">PCR clean-up of XbaI- and EcoRI-digested part6.2 C1 plasmid</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- addition of 500 μl PB buffer</SPAN></P><P class="c11-0 c11-8"><SPAN>- elution with 1st 30 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min; 2nd 22 μl HPLC water (pre-warmed) while incubating at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-8"><SPAN>- NanoDrop concentration </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-40"><P class="c11-7 c11-1"><SPAN>Sample</SPAN></P></TD><TD class="c11-33"><P class="c11-7 c11-1"><SPAN>ng/μl</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-40"><P class="c11-7 c11-1"><SPAN>part6.2 C1 vector EcoRI + XbaI digested</SPAN></P></TD><TD class="c11-33"><P class="c11-7 c11-1"><SPAN>15.6</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>1.75</SPAN></P></TD><TD class="c11-22"><P class="c11-7 c11-1"><SPAN>2.07</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0"><SPAN> </SPAN><SPAN class="c11-5"> </SPAN></P><P class="c11-0"><SPAN class="c11-5">Repeat of gel run (see above)</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1% Agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl of RD reaction + 1 μl 5xLD + 1 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 1 μl of uncut plasmid + 1 μl 5xLD + 3 μl dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- (loading of samples for DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR, as well (see above))</SPAN></P><P class="c11-0 c11-8"><SPAN>- run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">Loading:</SPAN><SPAN> Marker/neg.control from PCR/uncut purified PCR product/ RD reaction of PCR product/ uncut plasmid part 6.2 C1/ RD reaction plasmid part 6.2 C1 E+ P digest, purified/---</SPAN></P><P class="c11-0 c11-20"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Same result as before (see above), but at least a bit nicer (though still smeary…)</SPAN><SPAN class="c11-5"> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Stored in DarR reporter system-Box</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Hybridization of oligos for Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5">, Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> and RBS</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Oligos iGEM_73, iGEM_78 (RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Oligos iGEM_74; iGEM_79 (Promoter1</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Oligos iGEM_76; iGEM_81 (Promoter3</SPAN><SPAN class="c11-4">rev</SPAN><SPAN>) </SPAN></P><P class="c11-0 c11-1"><SPAN>1. Oligos dissolved in required amount of HPLC water</SPAN></P><P class="c11-0 c11-1"><SPAN>2. 10 µl of each oligo of one hybridization pair were mixed</SPAN></P><P class="c11-0 c11-1"><SPAN>3. incubation for 10 min at 80 °C (for RBS</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> oligos --&gt; T</SPAN><SPAN class="c11-4">M</SPAN><SPAN> is below 80 °C) or at 98 °C (for Promoter oligos --&gt; T</SPAN><SPAN class="c11-4">M</SPAN><SPAN> is above 80 °C)</SPAN></P><P class="c11-0 c11-1"><SPAN>4. after 10 min, heating blocks were switched off; the mixtures were left in the heating block until it was cooled down to ca. RT.</SPAN></P><P class="c11-0 c11-1"><SPAN>--&gt; can be used directly for ligation </SPAN></P><P class="c11-0 c11-8"><SPAN>- Hybridizations stored in DarR reporter system-Box</SPAN></P><P class="c11-0 c11-8"><SPAN>- Oligos stored in Primer-Box</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Ligation of</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">a) pSB1C3 E+P with DarR insert E + P</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">b) pSB1C3 E+S with Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5">, Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5">, or RBS</SPAN><SPAN class="c11-5 c11-4">rev</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">c) part6.2 C1 E+X vector with Promoter1</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> or Promoter3</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>Total volume of ligations: 20 µl</SPAN></P><P class="c11-0 c11-8"><SPAN>Ratio for DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> ligation: vector:insert = 1:3 --&gt; for 50 ng vector of 2070 bp, use 50 ng insert of 690 bp (online ligation calculator)</SPAN></P><P class="c11-0"><SPAN>Ligation a)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN> </SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>w/o insert</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>DarR</SPAN><SPAN class="c11-4">rev</SPAN></P></TD></TR><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN>Vector (pSB1C3 EcoRI- and PstI-digested, 56.6 ng/µl; 31.7.13)</SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>1 µl</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>1 µl</SPAN></P></TD></TR><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN>Insert (DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> from today) or dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O for w/o insert control</SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>9 µl</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>9 µl</SPAN></P></TD></TR><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN>T4 ligase (Thermo Scientific)</SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD></TR><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN>10x T4 buffer (Thermo Scientific)</SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD></TR><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN>dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-11"><SPAN> </SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>6 µl</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>6 µl</SPAN></P></TD></TR><TR><TD class="c11-46"><P class="c11-0 c11-11"><SPAN>Total</SPAN></P></TD><TD class="c11-39"><P class="c11-30 c11-11 c11-9"><SPAN>20 µl</SPAN></P></TD><TD class="c11-13"><P class="c11-30 c11-11 c11-9"><SPAN>20 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-38"><SPAN> </SPAN></P><P class="c11-0"><SPAN>Ligation b)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN> </SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>w/o insert</SPAN></P></TD><TD class="c11-23"><P class="c11-30 c11-11 c11-9"><SPAN>Promoter1</SPAN><SPAN class="c11-4">rev</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>Promoter3</SPAN><SPAN class="c11-4">rev</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>RBS</SPAN><SPAN class="c11-4">rev</SPAN></P></TD></TR><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN>Vector (pSB1C3 EcoRI- and SpeI-digested, 6.4 ng/µl; 30.7.13)</SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD><TD class="c11-23"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD></TR><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN>Insert (Hybridization oligos from today) or dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O for w/o insert control</SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD><TD class="c11-23"><P class="c11-11 c11-9 c11-30"><SPAN>8 µl</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>8 µl</SPAN></P></TD></TR><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN>T4 ligase (Thermo Scientific)</SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-23"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD></TR><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN>10x T4 buffer (Thermo Scientific)</SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-23"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>2 µl</SPAN></P></TD></TR><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN>dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>-</SPAN></P></TD><TD class="c11-23"><P class="c11-30 c11-11 c11-9"><SPAN>-</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>-</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>-</SPAN></P></TD></TR><TR><TD class="c11-42"><P class="c11-0 c11-11"><SPAN>Total</SPAN></P></TD><TD class="c11-14"><P class="c11-30 c11-11 c11-9"><SPAN>20 µl</SPAN></P></TD><TD class="c11-23"><P class="c11-30 c11-11 c11-9"><SPAN>20 µl</SPAN></P></TD><TD class="c11-31"><P class="c11-30 c11-11 c11-9"><SPAN>20 µl</SPAN></P></TD><TD class="c11-26"><P class="c11-30 c11-11 c11-9"><SPAN>20 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-38"><SPAN> </SPAN></P><P class="c11-0"><SPAN>Ligation c)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN> </SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>w/o insert</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>Promoter1</SPAN><SPAN class="c11-4">rev</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>Promoter3</SPAN><SPAN class="c11-4">rev</SPAN></P></TD></TR><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN>Vector (part6.2 C1 EcoRI- and XbaI-digested, 15.6 ng/µl; from today)</SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>3.5 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>3.5 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>3.5 µl</SPAN></P></TD></TR><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN>Insert (Hybridization oligos from today) or dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O for w/o insert control</SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>10 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>10 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>10 µl</SPAN></P></TD></TR><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN>T4 ligase (Thermo Scientific)</SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>2 µl</SPAN></P></TD></TR><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN>10x T4 buffer (Thermo Scientific)</SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>2 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>2 µl</SPAN></P></TD></TR><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN>dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>2.5 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>2.5 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>2.5 µl</SPAN></P></TD></TR><TR><TD class="c11-24"><P class="c11-0 c11-18"><SPAN>Total</SPAN></P></TD><TD class="c11-3"><P class="c11-18 c11-30 c11-9"><SPAN>20 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>20 µl</SPAN></P></TD><TD class="c11-24"><P class="c11-18 c11-30 c11-9"><SPAN>20 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN>- All reactions were pipetted individually, no mastermix</SPAN></P><P class="c11-0 c11-8"><SPAN>- Incubation ON at 16 °C (cold room)</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Transformation of ligation reaction from yesterday (pSB1C3 + Terminator</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5">)</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- According to methods folder, but instead of removing 400 µl after plating 50 µl and centrifugation, 500 µl supernatant were removed</SPAN></P><P class="c11-0 c11-8"><SPAN>- Negative control: addition of 10 µl sterile dH</SPAN><SPAN class="c11-4">2</SPAN><SPAN>O</SPAN></P><P class="c11-0 c11-8"><SPAN>- Plating on LB</SPAN><SPAN class="c11-19">Cm </SPAN><SPAN> plates (35 µg/ml)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Incubation at 37 °C</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Preparation of LB Agar for Chloramphenicol plates</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">shopping and ordering in cellar</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">Trafo of Ribo A-D from yesterday´s ligation </SPAN></P><P class="c11-0 c11-1"><SPAN>protocol followed except for streak out</SPAN></P><P class="c11-0 c11-1"><SPAN>streak out in 3 steps: first 50 μl, spin down, than 500 μl to waste, resuspend and streak out again in 50 μl and the rest (ca. 150 μl ) on another plate</SPAN></P><P class="c11-0 c11-1"><SPAN>3 plates per transformation</SPAN></P><P class="c11-0 c11-1"><SPAN>6 transformation (4 Ribos (A-D), 1 religation, 1 negativ control with 20 μl H2O) </SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0812"><SPAN class="date">12th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Terminatorrev PCR – Gel ex, Terminatorrev PCR – Repeated, Restriction of Part 6.2 C1 plasmid with EcoRI to generate vector for ligation with hybridization oligos of promoters, Restriction of Terminatorrev PCR product obtained after gel ex with EcoRI and PstI (for ligation with pSB1C3), Ligation of pSB1C3 (E+P-digested) with Terminatorrev insert (E+P digested), </P><DIV class="timeline-cont"><P class="c11-9"><SPAN class="c11-5">Gel and Nanodrop measurements of R.D. from friday (Jonathan)</SPAN></P><P class="c11-9"><SPAN>Gel:</SPAN></P><P class="c11-9"><SPAN>M|CFP|CFPr.d.|YFP|YFPr.d.|Part1|Part1r.d.|Part2|Part2r.d.|Part3|Part3r.d.|Part4|Part4r.d.|RiboA|RiboAr.d.</SPAN></P><P class="c11-9"><SPAN>M|RiboD|RiboDr.d.|RiboC|RiboCr.d.</SPAN></P><P class="c11-9"><SPAN class="c11-5">àRiboB was forgotten</SPAN></P><P class="c11-9"><SPAN class="c11-5">Nanodrop:</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-25"><P class="c11-2"><SPAN>Sample</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>ng/µl</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>part8 E+X purified 9.8.</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>14.1</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.84</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>2.13</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>CFP E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>27.1</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.68</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.48</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>YFP E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>19.6</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>2.03</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>2.22</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>part1 E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>26.7</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.69</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.49</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>part2 E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>22.6</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.89</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.95</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>part3 E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>30.0</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.89</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.99</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>part4 E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>39.2</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.89</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.89</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>Rino A E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>31.0</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.89</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.93</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>Ribo D E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>32.7</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.98</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>2.27</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>Ribo C E+X purified 9.8</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>29.1</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.92</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.95</SPAN></P></TD></TR><TR><TD class="c11-25"><P class="c11-2"><SPAN>Ribo B E+X purified 9.8.</SPAN></P></TD><TD class="c11-35"><P class="c11-2"><SPAN>30.9</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>1.51</SPAN></P></TD><TD class="c11-12"><P class="c11-2"><SPAN>2.79</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-9"><SPAN> </SPAN></P><P class="c11-9"><SPAN> </SPAN></P><P class="c11-9"><SPAN class="c11-5">New inoculation of part 8 in Lb Amp</SPAN></P><P class="c11-9"><SPAN class="c11-5"> </SPAN></P><P class="c11-9"><SPAN class="c11-5">Inocculation of GFP clones (part 6.2 clones 1-7)</SPAN></P><P class="c11-9"><SPAN>for cryostocks and a masterplate, which was forgotten previously</SPAN></P><P class="c11-9"><SPAN class="c11-5">Terminator</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR – Gel ex</SPAN></P><P class="c11-9"><SPAN> -&gt;</SPAN><SPAN> </SPAN><SPAN>Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR from 7.8.13 looks a bit strange (band in neg. control at height of expected PCR product); it could be the primers, however, since the plasmid used as a template is still present in the sample, PCR clean up (as it was done) might not be enough to purifiy PCR product from template. Thus, gel extraction needs to be performed.</SPAN></P><P class="c11-0 c11-8"><SPAN>- Addition of &lt;7.5 μl to each of the purified samples</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of entire samples on 1 %-agarose-1xTAE-gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 2 log ladder (3 μl)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of part 7 C1 plasmid as a control (1 μl of 120.5 ng/μl + 1 µl 5xLD + 3 µl water)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Run at 85 V</SPAN></P><P class="c11-8 c11-9"><SPAN> - Gel extraction + purification using Qiagen PCR clean-up/gel extraction kit</SPAN></P><P class="c11-0 c11-8"><SPAN>- since Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product is ca. 190 bp, 1 volume isopropanol was added after the gel pieces were dissolved</SPAN></P><P class="c11-0 c11-8"><SPAN>- 2 columns used (loading of sample amounts corresponding to max. 400 mg gel in total per column); when loading one column, I pipetted some of the sample onto the TARA-tube by mistake… so this part of the sample was lost…</SPAN></P><P class="c11-0 c11-8"><SPAN>- elution from both columns with 30 μl HPLC water each into the same tube</SPAN></P><P class="c11-0 c11-8"><SPAN>- concentration (NanoDrop): </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-23"><P class="c11-7"><SPAN>Sample</SPAN></P></TD><TD class="c11-21"><P class="c11-7"><SPAN>ng/µl</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-23"><P class="c11-7"><SPAN>Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product gel ex</SPAN></P></TD><TD class="c11-21"><P class="c11-7"><SPAN>16.5</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>1.60</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>0.26</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-34 c11-9"><SPAN> </SPAN></P><P class="c11-9"><SPAN class="c11-5">Terminator</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR – Repeated</SPAN></P><P class="c11-8 c11-9"><SPAN> - since PCR from 7.8.13 had a strange negative control, the PCR was repeated as on 7.8.13, but with some modifications:</SPAN></P><P class="c11-0 c11-8"><SPAN>- with only 2 μl primers (iGEM_71/72)</SPAN></P><P class="c11-0 c11-8"><SPAN>- 5 reactions in total:</SPAN></P><P class="c11-0 c11-20"><SPAN>a) neg. control 1: no primers --&gt; which bands are caused by primers?</SPAN></P><P class="c11-0 c11-20"><SPAN>b) neg. control 2: no template --&gt; which bands are caused by template?</SPAN></P><P class="c11-0 c11-20"><SPAN>c - e) 3 reactions with template and primers</SPAN></P><P class="c11-0 c11-8"><SPAN>- template: part 7 C1 plasmid, dilution from 7.8.13 used</SPAN></P><P class="c11-0 c11-8"><SPAN>- 6x MasterMix containing PfuS, 5x HF buffer, water and dNTPs</SPAN></P><P class="c11-0 c11-8"><SPAN>- after PCR: 3 reactions with template and primers pooled into 1 tube</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1%-agarose-1xTAE gel (from Nina)</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 4 μl of each PCR reaction + 1 μl 5xLD</SPAN></P><P class="c11-0 c11-8"><SPAN>- Run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-8"><SPAN>- samples ran almost out --&gt; no conclusion possible --&gt; gel run repeated:</SPAN></P><P class="c11-0 c11-8"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-29">gel run:</SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN>1%-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- loading of 4 μl of each PCR reaction + 1 μl 5xLD</SPAN></P><P class="c11-0 c11-8"><SPAN>- (Loading of Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product digest, as well – see below; loading of digest of part 6.2 C1 plasmid with EcoRI, too – see below)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>band observed in neg. control results indeed from primers --&gt; one can work with the old PCR product</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>green PCR tube with pooled PCR reactions stored in 15 ml Falcon tube at – 20°C for purification</SPAN></P><P class="c11-0 c11-34"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-5">Restriction of Part 6.2 C1 plasmid with EcoRI to generate vector for ligation with hybridization oligos of promoters</SPAN></P><P class="c11-0"><SPAN>1500 ng plasmid (6 μl of 261.5 ng/μl sample; purified on 9.8.13)</SPAN></P><P class="c11-0"><SPAN>4 μl EcoRI FD</SPAN></P><P class="c11-0"><SPAN>4 μl 10x FD buffer</SPAN></P><P class="c11-0"><SPAN>26 μl water</SPAN></P><P class="c11-0"><SPAN>in total 40 μl</SPAN></P><P class="c11-0 c11-8"><SPAN> - digest for 1 h at 37 °C</SPAN></P><P class="c11-0 c11-8"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-29">gel run:</SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN>1 %-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 1 μl plasmid (uncut) + 3 μl water + 1 μl 5x LD</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 3 μl RD reaction + 1 μl water + 1 μl 5x LD</SPAN></P><P class="c11-0 c11-8"><SPAN>- Run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>digest is still incomplete</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>addition of 1 μl EcoRI FD, 1 μl 10x FD buffer, 8 μl water</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>digest for another 1 h at 37 °C</SPAN></P><P class="c11-0 c11-8"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN class="c11-29">gel run:</SPAN></P><P class="c11-0 c11-8"><SPAN>- </SPAN><SPAN>1%-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 1 μl uncut plasmid + 3 μl water + 1 μl 5x LD</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 4 μl RD reaction + 1 μl 5x LD</SPAN></P><P class="c11-0 c11-8"><SPAN>- (Loading of Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR product digest, as well – see below; repeat of gel run of new Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR, as well – see above)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-34 c11-49"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Digest only slightly incomplete --&gt; tolerable --&gt; purify vector for further digest with XbaI with PCR clean-up kit</SPAN></P><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-29">Purification:</SPAN></P><P class="c11-0 c11-8"><SPAN> - Qiagen PCR purification kit</SPAN></P><P class="c11-0 c11-8"><SPAN>- 600 µl PB buffer used (by mistake…)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Elution 1x with 30 µl HPLC-water (pre-warmed) with incubation at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-8"><SPAN>- NanoDrop: concentration </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-23"><P class="c11-7"><SPAN>Sample</SPAN></P></TD><TD class="c11-21"><P class="c11-7"><SPAN>ng/µl</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-23"><P class="c11-7"><SPAN>Part6.2 C1 vector + EcoRI (after PCR clean-up)</SPAN></P></TD><TD class="c11-21"><P class="c11-7"><SPAN>34.9</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>1.86</SPAN></P></TD><TD class="c11-27"><P class="c11-7"><SPAN>2.14</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN>- Samples stored in To-do-Box</SPAN></P><P class="c11-0"><SPAN class="c11-5">Restriction of Terminator</SPAN><SPAN class="c11-5 c11-4">rev </SPAN><SPAN class="c11-5">PCR product obtained after gel ex with EcoRI and PstI (for ligation with pSB1C3)</SPAN></P><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN>30 μl PCR product</SPAN></P><P class="c11-0"><SPAN>3 μl EcoRI FD</SPAN></P><P class="c11-0"><SPAN>3 μl PstI FD</SPAN></P><P class="c11-0"><SPAN>4 μl 10x FD buffer</SPAN></P><P class="c11-0"><SPAN>40 μl in total </SPAN></P><P class="c11-0 c11-8"><SPAN>- digest for 1 h at 37 °C </SPAN></P><P class="c11-0 c11-8"><SPAN class="c11-29">- gel run:</SPAN></P><P class="c11-0 c11-8"><SPAN>- 1%-agarose-1xTAE gel</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 3 μl 2 log ladder</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 4 μl uncut PCR product + 1 μl 5x LD</SPAN></P><P class="c11-0 c11-8"><SPAN>- Loading of 4 μl RD reaction + 1 μl 5x LD</SPAN></P><P class="c11-0 c11-8"><SPAN>- (Loading of part6.2 C1 plasmid EcoRI 2 h digest, as well – see above; repeat of gel run of new Terminator</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> PCR, as well – see above)</SPAN></P><P class="c11-0 c11-8"><SPAN>- Run at 100 V</SPAN></P><P class="c11-0 c11-8"><SPAN>- EtBr staining + destaining in water</SPAN></P><P class="c11-0 c11-8"><SPAN>- UV detection</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>PCR product still present/ no over-digest --&gt; PCR clean-up to purify insert for ligation</SPAN></P><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0"><SPAN class="c11-29">Purification:</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Qiagen PCR purification kit</SPAN></P><P class="c11-0 c11-8"><SPAN>- 500 µl PB buffer used</SPAN></P><P class="c11-0 c11-8"><SPAN>- Elution 1x with 30 µl HPLC-water (pre-warmed) with incubation at 50 °C for 2 min</SPAN></P><P class="c11-0 c11-8"><SPAN>- NanoDrop: concentration </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-23"><P class="c11-2"><SPAN>Sample</SPAN></P></TD><TD class="c11-21"><P class="c11-2"><SPAN>ng/µl</SPAN></P></TD><TD class="c11-27"><P class="c11-2"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">280nm</SPAN></P></TD><TD class="c11-27"><P class="c11-2"><SPAN>A</SPAN><SPAN class="c11-4">260nm</SPAN><SPAN>/A</SPAN><SPAN class="c11-4">230nm</SPAN></P></TD></TR><TR><TD class="c11-23"><P class="c11-2"><SPAN>Term</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> insert (after PCR clean-up)</SPAN></P></TD><TD class="c11-21"><P class="c11-2"><SPAN>15.0</SPAN></P></TD><TD class="c11-27"><P class="c11-2"><SPAN>1.91</SPAN></P></TD><TD class="c11-27"><P class="c11-2"><SPAN>2.17</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0 c11-8"><SPAN>- Samples stored in To-do-Box</SPAN></P><P class="c11-0 c11-1"><SPAN class="c11-5">DarR</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> PCR: Test-PCR for optimal primer concentration</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- DarR</SPAN><SPAN class="c11-4">rev</SPAN><SPAN> primers (iGEM_83 and iGEM_84) arrived --&gt; dissolved in HPLC-water and preparation of 1:20 dilution in HPLC water</SPAN></P><P class="c11-0 c11-8"><SPAN>- iGEM_83 and iGEM_84 tend to form strong secondary structures --&gt; test for optimal primer concentration in PCR</SPAN></P><P class="c11-0 c11-8"><SPAN>- Test- PCR done as on 24.6.13: 5x MasterMix containing water, 5x HF buffer, PfuS, dNTPs; 2 μl, 4 μl and 8 μl primer tested, neg. control w/o template (</SPAN><SPAN class="c11-17">M.smegmatis</SPAN><SPAN> chrom.DNA); Annealing temperature of both primers is the same as for iGEM_33 and iGEM_44, since only the prefix and suffix sequences were switched)</SPAN></P><P class="c11-0 c11-8"><SPAN>- PCR taken out and stored at 4 °C (small fridge) in yellow-tip-box-rag</SPAN></P><P class="c11-0"><SPAN class="c11-5">Sequencing results of Promoter3op cloning in front of part 6 and into shipping vector</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-52"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Plasmids part 6.2. C1 and part 6.2. C2 contain the Promoter3op insert at the desired site in the vector w/o any mutations --&gt; part 6.2 C1 used for further cloning</SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;Plasmid Promoter3op in pSB1C3 C1 (see: Colony PCR from 8.8.13) contains the Promoter3op insert at the desired site in the vector w/o any mutations --&gt; inoculation of C1 in 4 ml LB</SPAN><SPAN class="c11-19">Cm</SPAN><SPAN> for MiniPrep and CryoStock, incubation ON at 37 °C, ca. 200 – 210 rpm</SPAN></P><P class="c11-0"><SPAN class="c11-5">Ligation of pSB1C3 (E+P-digested) with Terminator</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> insert (E+P digested)</SPAN></P><P class="c11-0"><SPAN class="c11-5"> </SPAN></P><P class="c11-0 c11-8"><SPAN>- Ligation calculator: for ratio vector: insert= 1:3 and 50 ng vector (2070 bp), ca. 14 ng insert (190 bp) are required</SPAN><SPAN class="c11-5"> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c11-29"><TBODY><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">Component</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN class="c11-5">Ligation</SPAN></P></TD></TR><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">Term</SPAN><SPAN class="c11-5 c11-4">rev</SPAN><SPAN class="c11-5"> insert/dH</SPAN><SPAN class="c11-5 c11-4">2</SPAN><SPAN class="c11-5">O (w/o insert control) from today</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN>1 µl (15.0 ng/µl)</SPAN></P></TD></TR><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">T4 Ligase (ThermoScientific)</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN>1 µl</SPAN></P></TD></TR><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">T4 ligation buffer 10x</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN>1 µl</SPAN></P></TD></TR><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">pSB1C3 vectorE+P digested from 31.7.13</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN>1 µl (56.6 ng/µl)</SPAN></P></TD></TR><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">dH</SPAN><SPAN class="c11-5 c11-4">2</SPAN><SPAN class="c11-5">O</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN>6 µl</SPAN></P></TD></TR><TR><TD class="c11-43"><P class="c11-30 c11-9"><SPAN class="c11-5">Total</SPAN></P></TD><TD class="c11-16"><P class="c11-30 c11-9"><SPAN>10 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c11-0"><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Ligation reactions incubated ON at 16 °C (cold room)</SPAN></P><P class="c11-0"><SPAN class="c11-5">Preparation of LB</SPAN><SPAN class="c11-5 c11-19">Cm</SPAN><SPAN class="c11-5"> plates</SPAN><SPAN> </SPAN></P><P class="c11-0 c11-8"><SPAN>- From 500 ml medium</SPAN></P><P class="c11-0 c11-8"><SPAN>- 35 µg/ml Cm in plates</SPAN></P><P class="c11-0"><SPAN class="c11-5">Ligation of Riboswitch Constructs A-D into pSB1C3(from part 7)</SPAN></P><P class="c11-0"><SPAN class="c11-5"> </SPAN></P><P class="c11-0"><SPAN class="c11-5">For that, ~45ng Vector were ligated to ~150ng Inserts as follows: </SPAN></P><P class="c11-0"><SPAN class="c11-5">             1.5µl pSB1C3</SPAN></P><P class="c11-0"><SPAN class="c11-5">             4.5µl Insert (Ribo A-D)</SPAN></P><P class="c11-0"><SPAN class="c11-5">             2µl T4 Buffer</SPAN></P><P class="c11-0"><SPAN class="c11-5">             2µl T4 Ligase</SPAN></P><P class="c11-0"><SPAN class="c11-5">             10µl H20</SPAN></P><P class="c11-0"><SPAN class="c11-5">     = 20µl reaction </SPAN></P><P class="c11-0"><SPAN class="c11-5">Incubation overnight at 16°C</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Goettingen/NoteBook_w11">http://2013.igem.org/Team:Goettingen/NoteBook_w11</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/NoteBook_w11" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/NoteBook_w11" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w11&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w11&amp;oldid=256426" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>