<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_NoteBook_w14"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/NoteBook_w14">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/NoteBook_w14&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w14&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w14&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/NoteBook_w14" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/NoteBook w14</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="tlob" id="tl_0906"><SPAN class="date">06th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">
Summary
1. Inoculation/Streak-out: DarR reporter system A C1/ DarR<SPAN class="c0 c15">rev</SPAN><SPAN class="c0">-Term</SPAN><SPAN class="c0 c15">rev</SPAN>C3 in pSB1C3
2. Control PCR</P><DIV class="timeline-cont"><P class="c18 c1"><SPAN class="c0 c2">Inoculation/Streak-out: </SPAN><SPAN class="c0">DarR reporter system A C1/ DarR</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0">-Term</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0">C3 in pSB1C3</SPAN><SPAN class="c0 c2"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-      cultures inoculated yesterday and forgotten in fridge were incubated at 37 °Cà in the afternoon, cultures taken out and put to autoclave…</SPAN></P><P class="c20 c1"><SPAN class="c0">-      streak-out of both clones using plates from 23.8.13 and masterplate, respectively, was repeated and plate incubated at 37 °C (plate wasn’t dry, so clones might be mixed by water drops running over the plate…)→  plate taken out in afternoon and put into fridge (again inoculation for CryoStocks next week)</SPAN></P><P class="c30 c1"><SPAN class="c0 c2">DNAse digest</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR><TD class="c63"><P class="c18 c1 c27"><SPAN class="c0">2500 ng RNA</SPAN></P></TD><TD class="c80"><P class="c18 c1 c27"><SPAN class="c0">respectively</SPAN></P></TD></TR><TR><TD class="c63"><P class="c18 c1 c27"><SPAN class="c0">DNAse</SPAN></P></TD><TD class="c80"><P class="c18 c1 c27"><SPAN class="c0">2,5 µl</SPAN></P></TD></TR><TR><TD class="c63"><P class="c18 c1 c27"><SPAN class="c0">10x buffer</SPAN></P></TD><TD class="c80"><P class="c18 c1 c27"><SPAN class="c0">2,5 µl</SPAN></P></TD></TR><TR><TD class="c63"><P class="c18 c1 c27"><SPAN class="c0">water</SPAN></P></TD><TD class="c80"><P class="c18 c1 c27"><SPAN class="c0">rest</SPAN></P></TD></TR><TR><TD class="c63"><P class="c18 c1 c27"><SPAN class="c0 c2">Total</SPAN></P></TD><TD class="c80"><P class="c18 c1 c27"><SPAN class="c0 c2">25 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c18 c1"><SPAN class="c0">stored in heat block 30 min at 37°C </SPAN></P><P class="c18 c1"><SPAN class="c0">then EDTA was added 2,5 µl per probe and put into thermo cycler at 65°C for 10 min</SPAN></P><P class="c18 c1"><SPAN class="c0 c2"> </SPAN></P><P class="c18 c1"><SPAN class="c0 c2">Control PCR </SPAN></P><P class="c20 c1"><SPAN class="c0">-      final reaction composition:</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR><TD class="c152"><P class="c41 c1 c27"><SPAN class="c0"> </SPAN><SPAN class="c0 c2">Component</SPAN></P></TD><TD class="c61"><P class="c1 c27 c41"><SPAN class="c0 c2">Volume</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">Template (RNA or gDNA)</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">Primer fwd 1:20 iGEM 102</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">Primer rev 1:20</SPAN></P><P class="c41 c1 c27"><SPAN class="c0 c2">iGEM 103</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">PfuS</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">0.5µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">dNTPs</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">5x HF buffer</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">5 µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">water</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0">15,5µl</SPAN></P></TD></TR><TR><TD class="c93"><P class="c41 c1 c27"><SPAN class="c0 c2">total</SPAN></P></TD><TD class="c61"><P class="c41 c1 c27"><SPAN class="c0 c2">25 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c18 c1"><SPAN class="c0">in thermo cycler protocol: Ivonne -&gt; PhuS</SPAN></P><P class="c18 c1"><SPAN class="c0"> </SPAN></P><P class="c18 c1"><SPAN class="c0">Gel doc:</SPAN></P><P class="c18 c1"><SPAN class="c0">wells:Part1C2exp, part2C2exp, part3C2exp, part4C2exp, part8C2exp, Part1C2stat, part2C2stat, part3C2stat, part4C2stat, part8C2stat; Part1C2ON, part2C2ON, part3C2ON, part4C2ON, part8C2ON, gDNA</SPAN></P><P class="c18 c1"><SPAN class="c0">Nanodrop measurements:</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0905"><SPAN class="date">05th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">
Summary
1. Transformation of RiboA C5.8 and CFP-Religate control C1 into freshly competent BL-21 (in which the DAC and Empty vectors already are)
2. Platereader experiment (fluorescence characterization of RiboA) Reproduction
3.Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2
4. Preparation of Parts for the Registry
5. Inoculation/Streak-out
</P><DIV class="timeline-cont"><P class="c1"><SPAN class="c0 c2">Transformation of RiboA C5.8 and CFP-Religate control C1 into freshly competent BL-21 (in which the DAC and Empty vectors already are)</SPAN></P><P class="c1"><SPAN class="c0">3 µl of RiboA C5.8 were used for the transformation. 100 µl of the competent cells were used for the transformation.</SPAN></P><P class="c1"><SPAN class="c0">4µl of the religate control.</SPAN></P><P class="c1"><SPAN class="c0">Transformation and negative control were plated onto Cm plates (2 plates each, 2 step streak out)</SPAN></P><P class="c1"><SPAN class="c0">Put into 37°C around 3:30 pm</SPAN></P><P class="c1"><SPAN class="c0 c2">Platereader experiment (fluorescence characterization of RiboA) Reproduction</SPAN></P><P class="c1"><SPAN class="c0">The RiboA C5.8.1 and C5.8.2 did not grow in the synchronization culture. The Synch cultures were inocculated from an ON culture (OD&gt;3) to OD=0,5 and did not grow during the following 3 hours.</SPAN></P><P class="c1"><SPAN class="c0 c2">Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2</SPAN></P><P class="c20 c1"><SPAN class="c0">-     harvested again cells of over night culture of an OD around between 2-3</SPAN></P><P class="c20 c1"><SPAN class="c0">-     defreezing of cells for RNA extraction</SPAN><SPAN class="c0 c2"> </SPAN></P><P class="c20 c1"><SPAN class="c0 c2">RNA extraction from E. coli</SPAN></P><P class="c20 c1"><SPAN class="c0">with kit: Qiagen RNeasy Mini </SPAN></P><P class="c20 c1"><SPAN class="c0">-     clean bench and pipets with ethanol</SPAN></P><P class="c20 c1"><SPAN class="c0">-     prepare RLT buffer with 2-mercaptoethanole (1 ml RLT buffer →  10 µl 2-mercaptoethanol); </SPAN><SPAN class="c0 c7">here: for 15 samples, 17 ml RLT buffer were supplied with 170 µl 2-mercaptoethanol (in other Bacillus-lab in the hood)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     prepare lysis buffer: 1xTE buffer + 0.5 mg/ml lysozyme in RNase free water (here: 340 µl 10x TE (sterile) + 85 µl 20 mg/ml lysozyme + 2975 ul RNase free waterà 3.4 ml lysis buffer)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     set one heating block to 70 °C and pre-heat RNase free water for elution (100 µl for elution of 1 sample)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     set another heating block to 37 °C for lysis </SPAN></P><P class="c26 c1"><SPAN class="c0">a) Lysis: add 200 µl lysis buffer to </SPAN><SPAN class="c0 c7">E. coli</SPAN><SPAN class="c0"> pellet, resuspend properly and incubate for 10 min at 37 °C;add 1 ml RLT supplied with 2-mercaptoethanol to the suspension and centrifuge for 5 min at 13 000 rpm, RT</SPAN></P><P class="c26 c1"><SPAN class="c0">b) removal of gDNA: load supernatant in 700 µl steps on </SPAN><SPAN class="c0 c55">violet columns</SPAN><SPAN class="c0"> and centrifuge at 13 000 rpm, RT, 1 min →  collect FLOW-THROUG in 2 ml epi tube (column binds gDNA, but NOT RNA </SPAN><SPAN class="c0 c55">→  RNA is in FLOW-THROUG!!!</SPAN><SPAN class="c0">)</SPAN></P><P class="c26 c1"><SPAN class="c0">c) binding of RNA: add more or less equal volume of 70% ethanol (RNA) (here: 1 ml) to flow-through, mix by pipetting and vortexing, then load mixture on pink columns (</SPAN><SPAN class="c0 c82">pink columns bind RNA</SPAN><SPAN class="c0">)</SPAN></P><P class="c26 c1"><SPAN class="c0">d) washing: →  no, one has to discard the flow-through…</SPAN></P><P class="c1 c66"><SPAN class="c0">d1) load 700 µl RW1 buffer onto column, centrifuge at 13000 rpm, RT, 1 min</SPAN></P><P class="c66 c1"><SPAN class="c0">d2) load 500 µl RPE1 buffer onto column, centrifuge at 13000 rpm, RT, 1 min</SPAN></P><P class="c66 c1"><SPAN class="c0">d3) load again 500 µl RPE1 buffer onto column, centrifuge at 13000 rpm, RT, 1 min</SPAN></P><P class="c1 c26"><SPAN class="c0">e) drying of column: centrifuge column in emptied collection tube 1 min, 13000 rpm, RT</SPAN></P><P class="c26 c1"><SPAN class="c0">f) elution: place column in fresh epi tube (RNase-free) elute RNA with 50 µl pre-heated (70°C) RNase-free water, incubate tubes at 70 °C for 5 min, then spin down at 13000rpm for 1 min, RT (here: sample “exp. 8” was eluted with 100 µl by mistake…)</SPAN></P><P class="c26 c1"><SPAN class="c0">g) measure concentration with NanoDrop </SPAN><SPAN class="c0 c82">(set it to RNA!)</SPAN><SPAN class="c0">; Blank: HPLC water (no difference to RNase free water…) </SPAN></P><P class="c20 c1"><SPAN class="c0 c109">NanoDrop concentrations from today</SPAN></P><P class="c18 c1"><SPAN class="c0 c2">Preparation of Parts for the Registry </SPAN></P><P class="c18 c1"><SPAN class="c0">preparation of the following samples:</SPAN><SPAN class="c0 c2"> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR><TD class="c87"><P class="c36 c1 c27 c86"><SPAN class="c0 c2">sample no.</SPAN></P></TD><TD class="c40"><P class="c1 c36"><SPAN class="c0 c2">part no.</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0 c2">sample name</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">2</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045001</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">DarR reverse</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">5</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045004</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">Terminator reverse</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">6</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045005</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">RBS reverse</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">8</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045007</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">composite part Promoter 3 with DarR operator</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">9</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045008</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">composite part Promoter 3-DarR operator - Bba_E0240</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">10</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045009</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">composite part Promoter 1 reverse-Promoter 3-DarR operator - Bba_E0240</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">12</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045011</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">composite part Terminator reverse-DarR reverse</SPAN></P></TD></TR><TR><TD class="c87"><P class="c36 c1 c27"><SPAN class="c0">13</SPAN></P></TD><TD class="c40"><P class="c36 c1"><SPAN class="c0">BBa_K1045012</SPAN></P></TD><TD class="c96"><P class="c36 c1"><SPAN class="c0">composite part Terminator reverse-DarR reverse - RBS reverse-Promoter 1 reverse-Promoter 3-DarR operator - Bba_E0240 (DarR reporter system, weak  promoter)</SPAN></P></TD></TR></TBODY></TABLE><P class="c20 c1"><SPAN class="c0 c2">-     pipetting scheme:</SPAN></P><P class="c1"><SPAN class="c0 c7 c2">(concentration and date of sample are given; followed by the exact volume of the sample that has to be used for 250 ng plasmid. This value is followed by the blue number indicating the volume that was added to the tube and a black number indicating the amount of HPLC water added to the tube)</SPAN><SPAN class="c0 c2"> </SPAN></P><P class="c1"><SPAN class="c0 c2">at least 250 ng in 10 µl HPLC water were added to a PCR tube and stored in blue “parts box”, stored at -20 °C, 3</SPAN><SPAN class="c0 c2 c74">rd</SPAN><SPAN class="c0 c2"> freezer box of small freezer</SPAN></P><P class="c18 c1"><SPAN class="c0 c2">Inoculation/Streak-out</SPAN><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-</SPAN><SPAN class="c10">     </SPAN><SPAN class="c0">inoculation and streak-out of DarR reporter system A C1</SPAN></P><P class="c20 c1"><SPAN class="c0">-</SPAN><SPAN class="c10">     </SPAN><SPAN class="c0">inoculation and streak-out of DarR</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0">-Term</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0">C3 in pSB1C3 (from plate from 23.8.13…) (CryoStock was gone…)</SPAN></P><P class="c20 c1"><SPAN class="c0">-</SPAN><SPAN class="c10">     </SPAN><SPAN class="c0">in 4 ml LB</SPAN><SPAN class="c0 c74">Cm</SPAN></P><P class="c20 c1"><SPAN class="c0">-</SPAN><SPAN class="c10">     </SPAN><SPAN class="c0">incubation ON, 37 °C (broth cultures forgotten in fridge, only plates were incubated…)</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0904"><SPAN class="date">04th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">
Summary
1.Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2
2. No-template control test PCR (Promoter characterization – primer test PCR)
3. Sequencing results G2L
4. Promoter characterization: qRT-PCR for RNA isolated from clones 1
5. pictures of 2<SPAN class="c0 c2 c74">nd </SPAN>round of drop experiment with RiboA C5.8.1 and CFP Religate C1 plates.
</P><DIV class="timeline-cont"><P class="c1"><SPAN class="c0 c2">Harvesting of cells for qRT-PCR analysis and plate reader assay (promoter clones) of re-trafos from part 1,2,3,4,8 respectively clone 2 </SPAN></P><P class="c20 c1"><SPAN class="c0">-     preparation of cryostocks</SPAN></P><P class="c20 c1"><SPAN class="c0">-     measuring of OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0"> of ON cultures in 1:20 dilution of LB and calculation of actual OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0">→  OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0"> in ON cultures varied from 4,06 to 4,54 nmà inoculation of synchronization pre-culture: 4 ml LB</SPAN><SPAN class="c0 c74">Amp</SPAN><SPAN class="c0"> + 50 μl ON culture, incubation at 37 °C at ca. 200 rpm for &gt; 2 h (the OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0"> was measured and the cells kept on bench)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     inoculation of an over day culture 15 ml LB </SPAN><SPAN class="c0 c74">Amp </SPAN><SPAN class="c0">+ around 1,9 ml from synchronization culture</SPAN></P><P class="c20 c1"><SPAN class="c0">-     measuring of OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0"> in 1:10 dilution of LB and calculation of actual OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0"> after 30 min, 1h, 1,15h, 1,30h, 1,45h, 2h, 2,15h, 2,30h, 2,30h, 2,45h, 3h, 3,30h, 4h, 4,30h, 5h</SPAN></P><P class="c20 c1"><SPAN class="c0">-     (Exel table of values to be done tomorrow)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     harvested cells of OD around 0,6 and between 2-3 stored in green cryo boy at -80°C</SPAN></P><P class="c20 c1"><SPAN class="c0">-     (an additional harvest at OD 0,6 was taken in order to prepare microscope single cell pictzures – have to be saved in jpg format and added tomorrow)</SPAN></P><P class="c1"><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">a) OD</SPAN><SPAN class="c0 c15">600nm</SPAN><SPAN class="c0"> ON culture </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c54"><P class="c25 c1"><SPAN class="c0 c2">Strain and Clone</SPAN></P></TD><TD class="c62"><P class="c25 c1"><SPAN class="c0 c2">OD</SPAN><SPAN class="c0 c2 c15">600nm</SPAN><SPAN class="c0 c2"> (1:20 dilution in LB)</SPAN></P></TD><TD class="c117"><P class="c25 c1"><SPAN class="c0 c2">OD</SPAN><SPAN class="c0 c2 c15">600nm</SPAN><SPAN class="c0 c2"> (1:1 calculated)</SPAN></P></TD><TD class="c136"><P class="c25 c1"><SPAN class="c0 c2">exact ON culture volume for synchronization culture in µl*</SPAN></P></TD></TR><TR class="c108"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P1 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,216</SPAN></P></TD><TD class="c102"><P class="c25 c1"><SPAN class="c0">4,32</SPAN></P></TD><TD class="c129"><P class="c25 c1"><SPAN class="c0">46</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P2 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,203</SPAN></P></TD><TD class="c102"><P class="c25 c1"><SPAN class="c0">4,06</SPAN></P></TD><TD class="c129"><P class="c25 c1"><SPAN class="c0">49</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P3 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,227</SPAN></P></TD><TD class="c102"><P class="c25 c1"><SPAN class="c0">4,54</SPAN></P></TD><TD class="c129"><P class="c25 c1"><SPAN class="c0">49</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P4 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,216</SPAN></P></TD><TD class="c102"><P class="c25 c1"><SPAN class="c0">4,32</SPAN></P></TD><TD class="c129"><P class="c25 c1"><SPAN class="c0">46</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P8 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,219</SPAN></P></TD><TD class="c102"><P class="c25 c1"><SPAN class="c0">4,38</SPAN></P></TD><TD class="c129"><P class="c25 c1"><SPAN class="c0">46</SPAN></P></TD></TR></TBODY></TABLE><P class="c1"><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">b) measurement of synchronization culture before main culture and titer plate preparation </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c54"><P class="c25 c1"><SPAN class="c0 c2">Strain and Clone</SPAN></P></TD><TD class="c62"><P class="c25 c1"><SPAN class="c0 c2">OD</SPAN><SPAN class="c0 c2 c15">600nm</SPAN><SPAN class="c0 c2"> (1:10 dilution in LB)</SPAN></P></TD><TD class="c79"><P class="c25 c1"><SPAN class="c0 c2">OD</SPAN><SPAN class="c0 c2 c15">600nm</SPAN><SPAN class="c0 c2"> (1:1 calculated)</SPAN></P></TD><TD class="c153"><P class="c25 c1"><SPAN class="c0 c2">synchronization culture volume for main cultures (15 ml) in ml *</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P1 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,044</SPAN></P></TD><TD class="c56"><P class="c25 c1"><SPAN class="c0">0,44</SPAN></P></TD><TD class="c125"><P class="c25 c1"><SPAN class="c0">1,70</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P2 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,043</SPAN></P></TD><TD class="c56"><P class="c25 c1"><SPAN class="c0">0,43</SPAN></P></TD><TD class="c125"><P class="c25 c1"><SPAN class="c0">1,74</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P3 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,037</SPAN></P></TD><TD class="c56"><P class="c25 c1"><SPAN class="c0">0,37</SPAN></P></TD><TD class="c125"><P class="c25 c1"><SPAN class="c0">2,03</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P4 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,039</SPAN></P></TD><TD class="c56"><P class="c25 c1"><SPAN class="c0">0,39</SPAN></P></TD><TD class="c125"><P class="c25 c1"><SPAN class="c0">1,9</SPAN></P></TD></TR><TR class="c33"><TD class="c46"><P class="c25 c1"><SPAN class="c0">P8 C2</SPAN></P></TD><TD class="c94"><P class="c25 c1"><SPAN class="c0">0,049</SPAN></P></TD><TD class="c56"><P class="c25 c1"><SPAN class="c0">0,49</SPAN></P></TD><TD class="c125"><P class="c25 c1"><SPAN class="c0">1,5</SPAN></P></TD></TR></TBODY></TABLE><P class="c1"><SPAN class="c0 c2">No-template control test PCR (Promoter characterization – primer test PCR) </SPAN></P><P class="c20 c1"><SPAN class="c0">-     neg. controls of primer test PCR for promoter characterization (test of primers iGEM_94 to 103) were not negative. It could be that one of the components e.g. the dNTP mix is contaminated with all our plasmids or E. coli DNA. So, dNTPs (and RNase free water, HPLC water, dH</SPAN><SPAN class="c0 c15">2</SPAN><SPAN class="c0">O and HF buffer) will be tested in a PCR. For this initial test, only the RFP primers will be used since they led to the strongest band for the negative control reactions with plasmids as templates in the test PCR from 2.9.13. This test PCR is only based on the asSummaryption that only 1 component is contaminated. In case more components are contaminated, one cannot conclude which components are the contaminated ones. </SPAN></P><P class="c20 c1"><SPAN class="c0">-     the following PCR reaction (total volume: 25 µl) were pipetted individually:</SPAN></P><P class="c20 c1"><SPAN class="c0">-     </SPAN><SPAN class="c0 c2">gel run:</SPAN></P><P class="c20 c1"><SPAN class="c0">-     2x 1% agarose-1xTAE gel</SPAN></P><P class="c20 c1"><SPAN class="c0">-     loading of 3 µl 2 log ladder as a marker</SPAN></P><P class="c20 c1"><SPAN class="c0">-     loading of 1 µl 5xLD + 4 µl PCR reaction (between two samples, the lane was left unloaded, so that settling of one sample in the neighbouring well  by mistake won’t influence the band pattern of the sample loaded next to it …)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     run at 100 V</SPAN></P><P class="c20 c1"><SPAN class="c0">-     EtBr staining + destaining in water</SPAN></P><P class="c20 c1"><SPAN class="c0">-     UV detection</SPAN><SPAN class="c0 c2"> </SPAN></P><P class="c30 c1"><SPAN class="c0 c2">Gel: </SPAN></P><P class="c30 c1"><SPAN class="c0 c109">Loading: </SPAN><SPAN class="c0">Marker/Reaction 1/---/Reaction2/---/Reaction 3/---/Reaction 4/---Reaction5/---/Reaction6/Marker</SPAN></P><P class="c30 c1"><SPAN class="c0 c2">expected bands: a band of 280 bp is expected for reaction 6, and for all other reactions in case one or more of the components are contaminated with (plasmid) DNA</SPAN></P><P class="c30 c1"><SPAN class="c0">A strong 280 bp band is visible for positive control. Weak 280 bp-bands are observed for all reactions except for Reaction 3. So, it could be that the buffer is contaminated… buffer thrown away</SPAN></P><P class="c1"><SPAN class="c0 c2">Sequencing results G2L</SPAN></P><P class="c1"><SPAN class="c0 c2">samples:</SPAN></P><P class="c13 c1"><SPAN class="c0">→  </SPAN><SPAN class="c0 c2">Repeat cloning of Promoter1</SPAN><SPAN class="c0 c2 c15">rev</SPAN><SPAN class="c0 c2"> and RBS</SPAN><SPAN class="c0 c2 c15">rev</SPAN><SPAN class="c0 c2"> in part 6.3 A.</SPAN></P><P class="c1 c13"><SPAN class="c0">→  </SPAN><SPAN class="c0 c2">Forget about cloning of DarR under control of strong promoter. Might not be so good for</SPAN><SPAN class="c0 c7 c2"> E. coli</SPAN><SPAN class="c0 c2"> growth if DarR is strongly expressed.</SPAN></P><P class="c1"><SPAN class="c0 c2">Promoter characterization: qRT-PCR for RNA isolated from clones 1 </SPAN></P><P class="c20 c1"><SPAN class="c0">-     qRT-PCR with Biorad kit</SPAN></P><P class="c20 c1"><SPAN class="c0">-     Final reaction (total volume: 20 µl) should contain 100 ng RNA, 10 µl 2x buffer, 0.4 µl enzyme and 1.2 µl of each primer (1:20 diluted in RNase free water)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     Mastermixes for each RNA sample and a neg. control were prepared. They consisted of2x buffer (light-sensitive – contains SYBRgreen!), RNA/RNase free water for no-template-controls, RNase free water, and enzyme for 3.5 reactions.</SPAN></P><P class="c20 c1"><SPAN class="c0">-     Mastermixes consisting of 1.2 µl Primer fwd and 1.2 µl Primer reverse per reaction were prepared and then added to 2.4 µl primer mix prepared in the wells of the qRT-PCR plate (in the beginning, I made a mistake and prepared only 10x mastermixes though I needed 20x mastermixes. I noticed this, when rrsB primer mastermix was empty after the half of the wells of the plate. Therefore, for rrsB primers, I had to make two 10x mastermixes, while I added again 12 µl of each primer to the 10x mastermixes prepared for rpoA and rfp primers to yield 20x mastermixes…)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     17.6 µl of the buffer/template/water/enzyme mastermix were added to the wells (many bubbles on surface of solution…; I made another mistake: I added the sample no. 16 mastermix to well that already contained sample 10 mastermix and rfp primers. Therefore, I had to pipet both reactions individually again…)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     qRT-PCR run using standard protocol:</SPAN></P><P class="c1"><SPAN class="c0 c2"> </SPAN></P><P class="c1"><SPAN class="c0 c2">Protocol:</SPAN></P><P class="c1"><SPAN class="c0">          Cycle 1: (1X)</SPAN></P><P class="c1"><SPAN class="c0">          Step 1:                    50,0 °C                   for 10:00.</SPAN></P><P class="c1"><SPAN class="c0">          Cycle 2: (1X)</SPAN></P><P class="c1"><SPAN class="c0">          Step 1:                    95,0 °C                   for 05:00.</SPAN></P><P class="c1"><SPAN class="c0">          Cycle 3: (45X)</SPAN></P><P class="c1"><SPAN class="c0">          Step 1:                    95,0 °C                   for 00:10.</SPAN></P><P class="c1"><SPAN class="c0">          Step 2:                    60,0 °C                   for 00:30.</SPAN></P><P class="c1"><SPAN class="c0">          Data collection and real-time analysis enabled.</SPAN></P><P class="c1"><SPAN class="c0">          Cycle 4: (1X)</SPAN></P><P class="c1"><SPAN class="c0">          Step 1:                    95,0 °C                   for 01:00.</SPAN></P><P class="c1"><SPAN class="c0">          Cycle 5: (1X)</SPAN></P><P class="c1"><SPAN class="c0">          Step 1:                    55,0 °C                   for 01:00.</SPAN></P><P class="c1"><SPAN class="c0">          Cycle 6: (81X)</SPAN></P><P class="c1"><SPAN class="c0">          Step 1:                    55,0 °C-95,0 °C                for 00:10.</SPAN></P><P class="c1"><SPAN class="c0">          Increase set point temperature after cycle 2 by 0,5 °C</SPAN></P><P class="c1"><SPAN class="c0">          Melt curve data collection and analysis enabled. </SPAN></P><P class="c1"><SPAN class="c0">Calculations*:</SPAN></P><P class="c1"><SPAN class="c0">The values in the brackets indicate the amount used for 3.5x Mastermix.</SPAN></P><P class="c1"><SPAN class="c0">The other values are for 1x reaction</SPAN></P><P class="c1"><SPAN class="c0">*</SPAN><SPAN class="c0 c7">numbers and abbreviations: see pipetting scheme DNase I digest, 2.9.13 and RNA concentrations 28.9.13</SPAN></P><P class="c1"><SPAN class="c0">Pipetting scheme*</SPAN></P><P class="c1"><SPAN class="c0">*</SPAN><SPAN class="c0 c7">numbers and abbreviations: see pipetting scheme DNase I digest, 2.9.13 and RNA concentrations 28.9.13</SPAN></P><P class="c1"><SPAN class="c0">Results:</SPAN></P><P class="c1"><SPAN class="c0 c88">I made a mistake and named well B07 “P1 C1 ON rrsB”, though it contained sample “P1 C1 ON rpoA”! Therefore in the original qRT-PCR data it’s named B07 P1 C1 ON rrsB</SPAN></P><P class="c1"><SPAN class="c0">[file:Melt Curve Analysis Spreadsheet Data for SYBR]</SPAN></P><P class="c13 c1"><SPAN class="c0">→  A peak at a certain temperature indicates the formation of a product that is molten to 50 % at this temperature. During a PCR, only one specific product should be formed by one specific primer pair. Therefore, for all PCR reactions with one specific primer pair, there should be only one peak at a certain temperature. </SPAN></P><P class="c1"><SPAN class="c0">Here, it looked as if for </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A or </SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B primer pair only one specific product was formed during PCR (</SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B is marked in red, T</SPAN><SPAN class="c0 c15">M </SPAN><SPAN class="c0">of specific PCR product = 84.50 °C; </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A is marked in black; T</SPAN><SPAN class="c0 c15">M </SPAN><SPAN class="c0">of specific PCR product = 85.50°C). </SPAN></P><P class="c1"><SPAN class="c0">For rfp primers, for most reactions, a PCR product of a T</SPAN><SPAN class="c0 c15">M </SPAN><SPAN class="c0">of 83.50 °C wasobserved for most reactions. </SPAN></P><P class="c1"><SPAN class="c0">For </SPAN><SPAN class="c0 c16">P2 C1</SPAN><SPAN class="c0"> and P8 C1,several different products seemed to have been formed when </SPAN><SPAN class="c0 c7">rfp</SPAN><SPAN class="c0"> primers were used (Could be because boths clones might/should not contain RFP). </SPAN></P><P class="c1"><SPAN class="c0">Only for reactions</SPAN><SPAN class="c55 c19">B10: P1 C1 ON rfp </SPAN><SPAN class="c19">and</SPAN><SPAN class="c19 c55">D10: P2 C2 ON rfp </SPAN><SPAN class="c19">and for all</SPAN><SPAN class="c55 c19"> P4 C1 rfp (F08 – F10) samples </SPAN><SPAN class="c19">as well as for</SPAN><SPAN class="c55 c19"> P3 C1 ON rfp (F11)</SPAN><SPAN class="c19">, a PCR product of a different T</SPAN><SPAN class="c19 c15">M </SPAN><SPAN class="c19">was detected (T</SPAN><SPAN class="c19 c15">M</SPAN><SPAN class="c19"> = 83.00 °C). This product might also be present in reaction </SPAN><SPAN class="c55 c19">G10: P8 C1 ON rfp</SPAN><SPAN class="c19">. Hence, the </SPAN><SPAN class="c7 c19">rfp</SPAN><SPAN class="c19"> primers could bind slightly unspecifically to residual plasmid/chrom. DNA or other </SPAN><SPAN class="c7 c19">E. coli</SPAN><SPAN class="c19"> transcripts</SPAN></P><P class="c1"><SPAN class="c0 c2">The C</SPAN><SPAN class="c0 c2 c15">T</SPAN><SPAN class="c0 c2"> values for most clones could be analyzed further. The analysis was also done for the problematic clones with non-specific product formation by </SPAN><SPAN class="c0 c7 c2">rfp</SPAN><SPAN class="c0 c2"> primers. If these values prove to be stable will be seen in future experiments…</SPAN></P><P class="c20 c1"><SPAN class="c0">-     The NTC for </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A had also a C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> though it should have N/A. It might be slightly contaminated with </SPAN><SPAN class="c0 c7">E.coli</SPAN><SPAN class="c0"> chrom.DNA/RNA, because the </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A product was formed in very low amounts. This might however be neglectible since the C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> value is 37 and therefore much higher than the highest C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> value obtained for the actual reactions with rpoA primers (D7 = P2 C2 ON rpoA; C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> = 26.03 →  difference is ca. 10 cycles →  template in NTC (and possibly also in other wells…?) is &gt;1000x diluted compared to actual reactions)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     For </SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B, the C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> values were &lt;10. C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> values &lt;10 might be inexactly because of detection limitations. Hence, </SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> values were not considered for further data analysis. For rrsB, there seemed to be a slight contamination with template in the NTC, as well. However, the melting point of this product differed from that of all other </SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B samples, indicating that it might be non-specific.</SPAN></P><P class="c20 c1"><SPAN class="c0">-     For the </SPAN><SPAN class="c0 c7">rfp</SPAN><SPAN class="c0"> NTC, no C</SPAN><SPAN class="c0 c15">T</SPAN><SPAN class="c0"> value was measured, but several products could have been formed as indicated by melting point analysis. However their amounts might have been too low to be detected.</SPAN></P><P class="c13 c1"><SPAN class="c0">→  </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A was used as housekeeping gene for normalization</SPAN></P><P class="c1"><SPAN class="c0 c2 c109">Calculations:</SPAN></P><P class="c1"><SPAN class="c0">The formulas used are indicated in the tables. The values are given with </SPAN><SPAN class="c0 c2">far too many</SPAN><SPAN class="c0"> digits after the decimal point. I don’t know how many of them are significant. This we will find out when we have our first replicates and when we calculate the mean and the standard deviations</SPAN></P><P class="c1"><SPAN class="c0 c2">Discussion: </SPAN></P><P class="c1"><SPAN class="c0">Only the strenght of the promoters relative to the weakest promoter for each phase can be considered. It is not possible to calculate the relative strenght of a specific promoter (e. g. promoter 3) relative to the different phases, since the </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A mRNA levels indicate a different intesity of transcription in each phase. </SPAN></P><P class="c1"><SPAN class="c0">During </SPAN><SPAN class="c0 c109">exponential phase</SPAN><SPAN class="c0">, Promoter 1 was the weakest, Promoter 2 was 3x stronger. Promoter 3 was the strongest (ca. 98x stronger than Promoter 1). Promoter 4 was weaker than Promoter 3 (only ca. 37x stronger than Promoter). P2 C1 seemed to contain no promoter/RFP as for P8, the empty vector control. </SPAN></P><P class="c1"><SPAN class="c0">During </SPAN><SPAN class="c0 c109">stationary pha</SPAN><SPAN class="c0">se, Promoter 1 was still the weakest, while Promoter 2 was ca. 6x stronger. Promoter 3 was ca. 19x stronger. Interestingly, Promoter 4 was the strongest and slightly stronger than Promoter 3 (vs. exponential phase). No significant </SPAN><SPAN class="c0 c7">rfp</SPAN><SPAN class="c0"> mRNA levels were observed for P2 C2. </SPAN></P><P class="c1"><SPAN class="c0">After </SPAN><SPAN class="c0 c109">ON incubation</SPAN><SPAN class="c0">, Promoter 1 was still weak. Promoter 2 showed ca. 13x strength. Promoter 3 was (as in exponential phase) the strongest promoter with ca. 32x strength compared to Promoter 1. Promoter 4 was slightly weaker (ca. 29x as strong as Promoter 1). P2 C1 did not show significant </SPAN><SPAN class="c0 c7">rfp</SPAN><SPAN class="c0"> transcript amounts.</SPAN><SPAN class="c0 c55"> </SPAN></P><P class="c1"><SPAN class="c0 c55 c2">Be careful: All values obtained for “After ON incubation” are likely to be wrong since the melting point of the PCR product is different from that of other samples. This might also be true for all P4 values.</SPAN></P><P class="c1"><SPAN class="c0 c2">ordering of nuclease free water downstairs</SPAN></P><P class="c1"><SPAN class="c0 c2">pictures of 2</SPAN><SPAN class="c0 c2 c74">nd</SPAN><SPAN class="c0 c2"> round of drop experiment with RiboA C5.8.1 and CFP Religate C1 plates:</SPAN></P><P class="c1"><SPAN class="c0 c2">[pic7]</SPAN></P><P class="c1"><SPAN class="c0 c2">Discussion: </SPAN></P><P class="c1"><SPAN class="c0">RiboA C5.8.1 and DAC on the same AB- IPTG+/IPTG- Plates </SPAN></P><P class="c1"><SPAN class="c0">Pictures:</SPAN></P><P class="c1"><SPAN class="c0">Cells mixed before plating. –IPTG left, +IPTG right</SPAN></P><P class="c1"><SPAN class="c0 c2">Discussion:</SPAN></P><P class="c1"><SPAN class="c0 c2">Inocculation of cultures for platereader experiment (fluorescence characterization of RiboA) Reproduction</SPAN></P><P class="c1"><SPAN class="c0">Cryostocks RiboA 5.8.1, 5.8.2 and CFP C1 and CFP C2 were used</SPAN></P><P class="c1"><SPAN class="c0">4ml LB + Cm each. 4 µl of the cryostock per inocculation</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0903"><SPAN class="date">03rd</SPAN><DIV class="cont"><P class="timeline-title goe-rt">
Summary:
1. Gel run: DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization)
2. NanoDrop concentration measurement of RNA samples after DNase I digest
3. Sequencing results DarR reporter system 
4. 2<SPAN class="c0 c74">nd </SPAN>round “Drop”-experiment with RiboA C5.8 and CFP Religate C1 plates
5. Evaluation of the Plate reader experiment yesterday (to be done) (fluorescence characterization of RiboA)
6. Trafo of RiboA C5.8 plasmid ino the newly made competent cells
</P><DIV class="timeline-cont"><P class="c1"><SPAN class="c0 c2">Gel run: DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization)</SPAN><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-     2x 1% agarose-1xTAE gel</SPAN></P><P class="c20 c1"><SPAN class="c0">-     loading of 3 µl 2 log ladder as a marker</SPAN></P><P class="c20 c1"><SPAN class="c0">-     loading of 1 µl 5xLD + 4 µl PCR reaction</SPAN></P><P class="c20 c1"><SPAN class="c0">-     run at 100 V</SPAN></P><P class="c20 c1"><SPAN class="c0">-     EtBr staining + destaining in water</SPAN></P><P class="c20 c1"><SPAN class="c0">-     UV detection </SPAN></P><P class="c1"><SPAN class="c0">Gel 1:</SPAN></P><P class="c1"><SPAN class="c0">Loading: Marker/neg. control/pos. control/1/2/3/4/5/6/7/8/9/10/11/12/Marker*</SPAN></P><P class="c1"><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">Gel 2:</SPAN></P><P class="c1"><SPAN class="c0">Loading: Marker/13/14/15/16/17/18/pos. control/neg. control/Marker*</SPAN></P><P class="c30 c1 c106"><SPAN class="c0">-     expected bands: only for pos. control, a band at 280 bp should be visible </SPAN></P><P class="c30 c1 c106"><SPAN class="c0">-     neg. control has slight band at ca. 280 bp… again not negative…</SPAN></P><P class="c30 c1 c106"><SPAN class="c0">-     for all RNA samples very weak bands at ca. 280 bp could be detected</SPAN></P><P class="c30 c1 c106"><SPAN class="c0">-     positive control as expected </SPAN></P><P class="c1"><SPAN class="c0">*</SPAN><SPAN class="c0 c7">numbers: see pipetting scheme DNase I digest, 2.9.13</SPAN></P><P class="c1"><SPAN class="c0 c2">NanoDrop concentration measurement of RNA samples after DNase I digest</SPAN></P><A href="https://2013.igem.org/File:Goe-03.09.13-RT-3.JPG" target="_blank"><P class="c0 c55">Click to view full resolution</P></A><P class="c1"><SPAN class="c0">This time, for all samples, a peak at ca. 220 to 225 nm was noticed.</SPAN></P><P class="c1"><SPAN class="c0">Samples stored at -20 °C, 3</SPAN><SPAN class="c0 c74">rd</SPAN><SPAN class="c0"> freezerbox in yellow-tip-box rag</SPAN></P><P class="c1"><SPAN class="c0 c2">Sequencing results DarR reporter system </SPAN></P><P class="c1"><SPAN class="c0">DarR reporter system A C1: the whole sequence is ok</SPAN></P><P class="c1"><SPAN class="c0">DarR reporter system B C1: the sequence is ok, except for</SPAN></P><P class="c20 c1"><SPAN class="c0">-     scar between RBS</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0"> and Promoter 3 </SPAN><SPAN class="c0 c15">rev</SPAN></P><P class="c20 c1"><SPAN class="c0">-    </SPAN><SPAN class="c0"> Promoter 3</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0"> corresponds to Promoter1</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0"> with an additional A at pos. 5 of promoter</SPAN></P><P class="c20 c1"><SPAN class="c0">-     sequence between the two promoters is not sequenced (xxxxxx…)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     operator could contain a C instead of an A at pos. 6 (but the sequences from VR and KG307 stop shortly after operator →  therefore, this could also be a sequencing error </SPAN></P><P class="c13 c1"><SPAN class="c0">→ only work with reporter system A, but forget about B…</SPAN></P><P class="c1"><SPAN class="c0 c2">Plates: Promoter Parts Clones </SPAN></P><P class="c20 c1"><SPAN class="c0">-     plates taken out and stored at 4 °C, big fridge</SPAN></P><P class="c1"><SPAN class="c0 c2">2</SPAN><SPAN class="c0 c2 c74">nd</SPAN><SPAN class="c0 c2"> round “Drop”-experiment with RiboA C5.8 and CFP Religate C1 plates</SPAN></P><P class="c1"><SPAN class="c0 c2"> </SPAN></P><P class="c1"><SPAN class="c0 c2">8 plates were prepared: </SPAN></P><P class="c1"><SPAN class="c0">2x RiboA C5.8 without IPTG</SPAN></P><P class="c1"><SPAN class="c0">2x RiboA C5.8 with IPTG</SPAN></P><P class="c1"><SPAN class="c0">2x CFP Religate C1 without IPTG</SPAN></P><P class="c1"><SPAN class="c0">2x CFP Religate C1 with IPTG </SPAN></P><P class="c1"><SPAN class="c0">As follows: </SPAN></P><P class="c1"><SPAN class="c0">25ml sterile water were filled into four 50ml falcons. Where needed, 50µl IPTG 1M (end concentration 1mM) were added. Also the 300µl Glycerin stocks prepared earlier were added to the water in the dedicated falcons. The falcons were inverted several times. 2x LB medium was prepared by Katrin Gunka and melted in the microwave. It was added to the 25ml water rather hot to the total volume of 50ml. The falcons were inverted a few times and then two 25ml plates were poured from each falcon.</SPAN></P><P class="c1"><SPAN class="c0 c2"> </SPAN></P><P class="c1"><SPAN class="c0 c2">On each plate 3 drops of DAC or empty control: </SPAN></P><P class="c1"><SPAN class="c0">1x undiluted overnight culture</SPAN></P><P class="c1"><SPAN class="c0">1x 1:10 diluted overnight culture</SPAN></P><P class="c1"><SPAN class="c0">1x undiluted overnight culture smeared over one half of the plate </SPAN></P><P class="c1"><SPAN class="c0">Plates incubated at 37°C overnight</SPAN></P><P class="c1"><SPAN class="c0 c2">Evaluation of the Plate reader experiment yesterday (to be done) (fluorescence characterization of RiboA)</SPAN></P><P class="c1"><SPAN class="c0 c2">Plated out RiboA C5.8.1 and DAC on the same AB- IPTG+/IPTG- Plates.</SPAN><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">4 plates: 2xIPTG-</SPAN></P><P class="c1"><SPAN class="c0">             2xIPTG+</SPAN></P><P class="c1"><SPAN class="c0">On one of each IPTG+/- plates, DAC and RiboA were plated separately on the right and the left, but were mixed in the middle of the plate</SPAN></P><P class="c1"><SPAN class="c0">On the other two, the Cells were mixed prior to the plating. </SPAN></P><P class="c1"><SPAN class="c0">This Experiment is to easily show possible interactions of the clones</SPAN></P><P class="c1"><SPAN class="c0 c2">Preparation of competent BL-21 cells</SPAN><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">ON culture of BL-21 (inoculated by Joni) was used to inoculate 10 ml LB with a start OD </SPAN><SPAN class="c0 c15">600</SPAN><SPAN class="c0">of 0,11. After 1h at 37°C, the culture reached an OD of 0,34.</SPAN></P><P class="c1"><SPAN class="c0">The MgCl</SPAN><SPAN class="c0 c15">2</SPAN><SPAN class="c0"> treatment was performed as described in the method collection.</SPAN></P><P class="c1"><SPAN class="c0 c2">Trafo of RiboA C5.8 plasmid ino the newly made competent cells </SPAN></P><P class="c1"><SPAN class="c0">3 µl of RiboA C5.8 were used for the transformation. 100 µl of the competent cells were used for the transformation.</SPAN></P><P class="c1"><SPAN class="c0">Transformation and negative control were plated onto Cm plates (2 plates each, 2 step streak out)</SPAN></P><P class="c1"><SPAN class="c0">Put into 37°C around 3:30 pm </SPAN></P><P class="c1"><SPAN class="c0">To be repeated on the 5. Colonies on negative control!! ON culture inoculated on the 4.9 at 5pm</SPAN></P><P class="c1"><SPAN class="c0 c2">Shopping downstairs</SPAN></P><P class="c1"><SPAN class="c0 c2">Pooring of 4 LB plates with (2) and without (2) (25</SPAN><SPAN class="c0"> µl)</SPAN><SPAN class="c0 c2"> IPDG, but without any further resistance</SPAN></P><P class="c1"><SPAN class="c0 c2">Innoculation of re-trafo of part 1,2,3,4, 8 in LB-medium with amp</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0902"><SPAN class="date" style="left:-5px">02nd</SPAN><DIV class="cont" style="left:-10px"><P class="timeline-title goe-rt">
Summary:
1. Fluorescence Assay of our RiboswitchA CFP C5.8.1+2 and the CFP Religate C1+2 as control (Fluorescence characterization of RiboA) 
2. Minipreps of RiboA Retrafo Clones 5.8.1-5 and the CFP Religate Controls Clones 1+2
3. Test PCR for qRT-PCR primers iGEM_94 – 103 
4. Sequencing of DarR reporter system clones by SeqLab 
5. DNase I digest of RNA samples (Promoter characterization) 
6. DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization)
</P><DIV class="timeline-cont"><P class="c1"><SPAN class="c0 c2">Fluorescence Assay of our RiboswitchA CFP C5.8.1+2 and the CFP Religate C1+2 as control (Fluorescence characterization of RiboA) </SPAN></P><P class="c1"><SPAN class="c0">For the assay, the CFP Religate is the non fluorescent control. Two clones were used for a biological replicate. </SPAN></P><P class="c112 c1 c131"><SPAN class="c19">→ </SPAN><SPAN class="c0">Yesterday´s overnight cultures were inoculated in 4mL cultures with Cm at 0,5 to synchronize them</SPAN></P><P class="c112 c1 c131"><SPAN class="c19">→ </SPAN><SPAN class="c0">After 1h the cells had not grown at all</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c80"><P class="c1"><SPAN class="c0">Culture</SPAN></P></TD><TD class="c123"><P class="c1"><SPAN class="c0">CFP Rel. C1</SPAN></P></TD><TD class="c101"><P class="c1"><SPAN class="c0">CFP Rel. C2</SPAN></P></TD><TD class="c60"><P class="c1"><SPAN class="c0">RiboA 5.8.1</SPAN></P></TD><TD class="c68"><P class="c1"><SPAN class="c0">RiboA 5.8.2</SPAN></P></TD></TR><TR class="c33"><TD class="c80"><P class="c1"><SPAN class="c0">OD600</SPAN></P></TD><TD class="c123"><P class="c1"><SPAN class="c0">0.5</SPAN></P></TD><TD class="c101"><P class="c1"><SPAN class="c0">0.43</SPAN></P></TD><TD class="c60"><P class="c1"><SPAN class="c0">0.5</SPAN></P></TD><TD class="c68"><P class="c1"><SPAN class="c0">0.48</SPAN></P></TD></TR></TBODY></TABLE><P class="c112 c1 c131"><SPAN class="c19">→ </SPAN><SPAN class="c0">After 2h three of four cultures had grown</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c80"><P class="c1"><SPAN class="c0">Culture</SPAN></P></TD><TD class="c123"><P class="c1"><SPAN class="c0">CFP Rel. C1</SPAN></P></TD><TD class="c101"><P class="c1"><SPAN class="c0">CFP Rel. C2</SPAN></P></TD><TD class="c60"><P class="c1"><SPAN class="c0">RiboA 5.8.1</SPAN></P></TD><TD class="c68"><P class="c1"><SPAN class="c0">RiboA 5.8.2</SPAN></P></TD></TR><TR class="c33"><TD class="c80"><P class="c1"><SPAN class="c0">OD600</SPAN></P></TD><TD class="c123"><P class="c1"><SPAN class="c0">1</SPAN></P></TD><TD class="c101"><P class="c1"><SPAN class="c0">0.9</SPAN></P></TD><TD class="c60"><P class="c1"><SPAN class="c0">0.4</SPAN></P></TD><TD class="c68"><P class="c1"><SPAN class="c0">0.7</SPAN></P></TD></TR></TBODY></TABLE><P class="c112 c1 c131"><SPAN class="c19">→ </SPAN><SPAN class="c0">All cultures (also the one that didn’t grow) were diluted in a 1mL eppi to OD=0.05 and used for the Fluorescence assay in the plate reader in technical triplets</SPAN></P><P class="c112 c1 c131"><SPAN class="c19">→ </SPAN><SPAN class="c0">The platereader will breed the cells at 37°C and read the plate every 15min. for 21 hours. Excitation 436nm and Emission 480nm</SPAN></P><P class="c1"><SPAN class="c0 c2">Minipreps of RiboA Retrafo Clones 5.8.1-5 and the CFP Religate Controls Clones 1+2</SPAN><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">Nanodrop measurements on the eppis (btw. 70 and 500ng/µl)</SPAN></P><P class="c13 c1"><SPAN class="c19">→ </SPAN><SPAN class="c0"> </SPAN><SPAN class="c0 c2">The CFP Religate clones need to be sequenced on friday!</SPAN></P><P class="c1"><SPAN class="c0 c2">Inocculated DAC and empty cells for the “Drop” Experiment tomorrow</SPAN></P><P class="c1"><SPAN class="c0 c2">Inocculated RiboA C5.8.1 and CFP Religate C1 for possible experiment ideas</SPAN></P><P class="c1"><SPAN class="c0 c2">Inocculated BL-21 T4 (not Blue) cells to be made competent tomorrow. These cells are for a transformation of RiboA into this stem for comparability with the DAC and empty cells.</SPAN></P><P class="c1"><SPAN class="c0 c2">MiniPlasmid Preparation of DarR reporter system clones </SPAN></P><P class="c20 c1"><SPAN class="c19">-    </SPAN><SPAN class="c0"> samples: cultures inoculated yesterday evening by Katrin G.</SPAN></P><P class="c20 c1"><SPAN class="c19">-    </SPAN><SPAN class="c0"> MiniPrep with Nucleospin kit from Macherery-Nagel</SPAN></P><P class="c20 c1"><SPAN class="c19">-    </SPAN><SPAN class="c0"> elution in 30 µl HPLC-water (pre-warmed, incubation at 50 °C, 2 min)</SPAN></P><P class="c20 c1"><SPAN class="c19">-    </SPAN><SPAN class="c0"> NanoDrop concentration measurement:</SPAN></P><P class="c13 c1"><SPAN class="c0">→  enough DNA for sequencing by SeqLab!!!^^</SPAN></P><P class="c1"><SPAN class="c0 c2">Test PCR for qRT-PCR primers iGEM_94 – 103 </SPAN></P><P class="c20 c1"><SPAN class="c0">-    </SPAN><SPAN class="c0"> preparation of 1:20 dilution of primer stock solutions iGEM_94 to iGEM_103 in RNase free water, now stored in orange RNA box in 3</SPAN><SPAN class="c0 c74">rd</SPAN><SPAN class="c0"> freezer box</SPAN></P><P class="c20 c1"><SPAN class="c0">-     Test PCR:</SPAN></P><P class="c20 c1"><SPAN class="c0">-     final reaction composition:</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">Component</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0 c2">Volume</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">Template</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">Primer fwd 1:20</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">Primer rev 1:20</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">PfuS</SPAN></P></TD><TD class="c104"><P class="c1 c14"><SPAN class="c0">1µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">dNTPs</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">5x HF buffer</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0">5 µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">water</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0">15µl</SPAN></P></TD></TR><TR class="c33"><TD class="c50"><P class="c14 c1"><SPAN class="c0 c2">total</SPAN></P></TD><TD class="c104"><P class="c14 c1"><SPAN class="c0 c2">25 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c20 c1"><SPAN class="c0"> -     preparation of template and primers in PCR tubes: </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c134"><P class="c58 c1 c27"><SPAN class="c0">template</SPAN></P></TD><TD class="c43"><P class="c58 c1 c27"><SPAN class="c0">plasmid part 6 C1</SPAN></P><P class="c58 c1 c27"><SPAN class="c0">(GFP)</SPAN></P></TD><TD class="c83"><P class="c58 c1 c27"><SPAN class="c0">plasmid DarR</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0"> in pSB1C3</SPAN></P><P class="c58 c1 c27"><SPAN class="c0">(DarR</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0">)</SPAN></P></TD><TD class="c133"><P class="c58 c1 c27"><SPAN class="c0">plasmid part 1 C1</SPAN></P><P class="c58 c1 c27"><SPAN class="c0">(RFP)</SPAN></P></TD><TD class="c72"><P class="c58 c1 c27"><SPAN class="c0 c7">E.coli</SPAN><SPAN class="c0"> chrom. DNA</SPAN></P><P class="c58 c1 c27"><SPAN class="c0">(rrsB)</SPAN></P></TD><TD class="c147"><P class="c58 c1 c27"><SPAN class="c0 c7">E.coli</SPAN><SPAN class="c0"> chrom. DNA</SPAN></P><P class="c1 c27 c58"><SPAN class="c0">(rpoA)</SPAN></P></TD></TR><TR class="c33"><TD class="c134"><P class="c58 c1 c27"><SPAN class="c0">primer combination</SPAN></P></TD><TD class="c43"><P class="c58 c1 c27"><SPAN class="c0">iGEM_94/95</SPAN></P></TD><TD class="c83"><P class="c58 c1 c27"><SPAN class="c0">iGEM_96/97</SPAN></P></TD><TD class="c133"><P class="c58 c1 c27"><SPAN class="c0">iGEM_98/99</SPAN></P></TD><TD class="c72"><P class="c58 c1 c27"><SPAN class="c0">iGEM_100/101</SPAN></P></TD><TD class="c147"><P class="c58 c1 c27"><SPAN class="c0">iGEM_102/103</SPAN></P></TD></TR></TBODY></TABLE><P class="c1"><SPAN class="c0"> </SPAN></P><P class="c1"><SPAN class="c0">for the plasmids, dilutions containing ca. 10 ng/µl were prepared and 1 µl of this dilution used as template</SPAN></P><P class="c112 c1"><SPAN class="c0">(DarR</SPAN><SPAN class="c0 c15">rev</SPAN><SPAN class="c0"> in pSB1C3 C4 plasmid, 16.8.13, 282.5 ng/µl →  1:28 dilution;</SPAN></P><P class="c112 c1"><SPAN class="c0">part 6 C1 plasmid, 7.6.13, 88.9 ng/µl →  1:9 dilution</SPAN></P><P class="c112 c1"><SPAN class="c0">part 1 C1 plasmid, 254.8 ng/µl →  1:25 dilution) </SPAN></P><P class="c1"><SPAN class="c0">preparation of neg. control with water as a template for each primer combination, as well →  in total, 10 PCR reactions were prepared </SPAN></P><P class="c20 c1"><SPAN class="c0">-     preparation of mastermix for 12 reactions consisting of buffer, water, PfuS and dNTP mix, then addition of 22 µl mastermix to each reaction </SPAN></P><P class="c1"><SPAN class="c0 c107">THE WATER USED FOR THE WHOLE PCR (except for the primers, of course) WAS FRESH HPLC WATER! (to prevent “positive” negative controls….)</SPAN><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-     PCR protocol:</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR class="c33"><TD class="c28"><P class="c21 c1"><SPAN class="c0 c2">Step</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0 c2">temperature</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0 c2">time</SPAN></P></TD></TR><TR class="c33"><TD class="c28"><P class="c21 c1"><SPAN class="c0 c2">initial denaturation</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0">98.5 °C</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0">5 min</SPAN></P></TD></TR><TR class="c33"><TD class="c28"><P class="c21 c1"><SPAN class="c0 c2">denaturation</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0">98.5 °C</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0">30 s</SPAN></P></TD></TR><TR class="c33"><TD class="c28"><P class="c21 c1"><SPAN class="c0 c2">annealing</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0">52 °C</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0">35 s</SPAN></P></TD></TR><TR class="c33"><TD class="c28"><P class="c1 c21"><SPAN class="c0 c2">elongation</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0">72 °C</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0">1 min</SPAN></P></TD></TR><TR class="c33"><TD class="c28"><P class="c21 c1"><SPAN class="c0 c2">final elongation</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0">72 °C</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0">10 min</SPAN></P></TD></TR><TR class="c33"><TD class="c28"><P class="c21 c1"><SPAN class="c0 c2">hold</SPAN></P></TD><TD class="c57"><P class="c21 c1"><SPAN class="c0">15 °C</SPAN></P></TD><TD class="c23"><P class="c21 c1"><SPAN class="c0">∞</SPAN></P></TD></TR></TBODY></TABLE><P class="c30 c1"><SPAN class="c0"> →  30 cycles </SPAN></P><P class="c1 c120"><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-     </SPAN><SPAN class="c0 c109">agarose gel run for PCR:</SPAN><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-     1%-agarose-1xTAE gel</SPAN></P><P class="c20 c1"><SPAN class="c0">-     addition of 5 µl 5xLD to each PCR sample</SPAN></P><P class="c20 c1"><SPAN class="c0">-     loading of 5 µl of each PCR sample</SPAN></P><P class="c20 c1"><SPAN class="c0">-     loading of 3 µl 2 log ladder as a marker</SPAN></P><P class="c20 c1"><SPAN class="c0">-     run at 100 V</SPAN></P><P class="c20 c1"><SPAN class="c0">-     EtBr staining + destaining in water</SPAN></P><P class="c20 c1"><SPAN class="c0">-     UV detection </SPAN></P><P class="c1 c112"><SPAN class="c0 c109">Loading:</SPAN><SPAN class="c0"> Marker/ Neg. control iGEM_94+95/PCR iGEM_94+95/ Neg. control iGEM_96+97/PCR iGEM_96+97/ Neg. control iGEM_98+99/PCR iGEM_98+99/ Neg. control iGEM_100+101/PCR iGEM_100+101/ Neg. control iGEM_102+103/PCR iGEM_102+103/Marker/---/---/---/---</SPAN></P><P class="c20 c1"><SPAN class="c0">-     expected product size: 280 bp; neg. controls: no product (in case of reactions with plasmid as a template, the bands of the plasmid could be visible)</SPAN></P><P class="c84 c1"><SPAN class="c0">→  for all neg. controls weak to very weak bands at 280 bp could be detected →  for samples with primers iGEM_100/101 and iGEM_102/103, this could be because of a pipetting error (it could be that, after pipetting the mastermix into PCR iGEM_100/101, I forgot to exchange the pipet tip. Then, I added the mastermix to samples neg. control iGEM_100/101, PCR iGEM_102/103 and neg. control iGEM_102/103); an alternative explanation would be that I used the PfuS purified from </SPAN><SPAN class="c0 c7">E. coli</SPAN><SPAN class="c0"> in this department. It could contain residual </SPAN><SPAN class="c0 c7">E. coli</SPAN><SPAN class="c0"> chrom. DNA); in general, all primers generate under the given conditions very high amounts of the expected PCR product. It could therefore be that the weak bands in the neg. control lanes result from PCR product that went from the PCR reaction lane into the neg. control lane…</SPAN></P><P class="c1 c84"><SPAN class="c0">→  Annealing temperature is ok., primers lead to PCR product of expected size/ all primers lead to PCR product of same size; →  repeat PCR with Phusion® for </SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B and </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A, to see, if bands result from contamination with </SPAN><SPAN class="c0 c7">E. coli </SPAN><SPAN class="c0">chrom. DNA</SPAN></P><P class="c18 c1"><SPAN class="c0 c2">Re-streak of clones</SPAN><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-     re-streak of Re-trafo clones C2 and C3 for parts 1, 2, 3, 4 and 8 from plates from 30.9.13 on fresh LB</SPAN><SPAN class="c0 c74">Amp (100 µg/ml)</SPAN></P><P class="c20 c1"><SPAN class="c0">-    </SPAN><SPAN class="c0"> re-streak of initial promoter clones C1 – C3 of parts 1 – 4 on fresh LB</SPAN><SPAN class="c0 c74">Amp (100 µg/ml)</SPAN><SPAN class="c0"> plates (it could be that the plates borrowed from DAC team, contained only 50 µg/ml Ampicillin →  repetition of experiment with plates for fotos…)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     streak-out was done in the morning, therefore plates were kept in fridge til evening and then put to 37 °C for ON incubation</SPAN></P><P class="c18 c1"><SPAN class="c0 c2">Sequencing of DarR reporter system clones by SeqLab </SPAN></P><P class="c20 c1"><SPAN class="c0">-     600 ng plasmid DNA in 3 µl + 4 µl primer 1:20 diluted </SPAN></P><P class="c59 c1"><SPAN class="c0">reactions: </SPAN></P><P class="c59 c1"><SPAN class="c0">1 – DarR reporter system A C1 + VF2</SPAN></P><P class="c59 c1"><SPAN class="c0">2 – DarR reporter s ystem A C1 + VR</SPAN></P><P class="c59 c1"><SPAN class="c0">3 – DarR reporter system A C1 + KG307</SPAN></P><P class="c59 c1"><SPAN class="c0">4 – DarR reporter system B  C1 + VF2</SPAN></P><P class="c59 c1"><SPAN class="c0">5 – DarR reporter system B  C1 + VR</SPAN></P><P class="c59 c1"><SPAN class="c0">6 – DarR reporter system B  C1 + KG307</SPAN></P><P class="c18 c1"><SPAN class="c0 c2">Test PCR for qRT-PCR primers repeated (for rrsB and rpoA)</SPAN><SPAN class="c0"> </SPAN></P><P class="c20 c1"><SPAN class="c0">-     PCR done exactly as described before, but this time, Phusion ® was used instead of PfuS and only the PCR reactions for </SPAN><SPAN class="c0 c7">rrs</SPAN><SPAN class="c0">B and </SPAN><SPAN class="c0 c7">rpo</SPAN><SPAN class="c0">A including their neg. controls were prepared (with help of 5x master mix (HF buffer 5x, Phusion®, HPLC water, dNTP mix) which was added to primer-template mixtures)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     PCR protocol was the same as before</SPAN></P><P class="c20 c1"><SPAN class="c0">-     gel run (together with Katrin G.’s samples), by Katrin G.:</SPAN></P><P class="c30 c1"><SPAN class="c0">1%-agarose-1xTAE gel, 5 µl 5xLD added to PCR sample and loaded on gel, EtBr staining + destaining in water, UV detection</SPAN></P><P class="c30 c1"><SPAN class="c0 c2">Loading: ----</SPAN><SPAN class="c0">Marker/Neg. control iGEM_100+101/---/PCR iGEM_100+101/ Neg. control iGEM_102+103/PCR iGEM_102+103/</SPAN></P><P class="c105 c1"><SPAN class="c0">→  neg. control for iGEM_100/101 still weak band at ca. 280 bp →  no explanation! It can’t be that samples ran over into neighbouring lanes, since lane that was left unloaded did not show bands…</SPAN></P><P class="c1 c105"><SPAN class="c0">→  iGEM_102/103 led to additional bands above and below 280 bp and to very strong primer cloud… or a very short unspecific product…?</SPAN></P><P class="c105 c1"><SPAN class="c0">→  do DNase digest control PCR with iGEM_102/103, because for this neg. control, no band at 280 bp was detected</SPAN></P><P class="c71 c1"><SPAN class="c0">→  neg. control for iGEM_100/101 still weak band at ca. 280 bp →  no explanation! It can’t be that samples ran over into neighbouring lanes, since lane that was left unloaded did not show bands…</SPAN></P><P class="c71 c1"><SPAN class="c0">→  iGEM_102/103 led to additional bands above and below 280 bp and to very strong primer cloud… or a very short unspecific product…?</SPAN></P><P class="c71 c1"><SPAN class="c0">→  do DNase digest control PCR with iGEM_102/103, because for this neg. control, no band at 280 bp was detected</SPAN></P><P class="c1"><SPAN class="c0 c2">DNase I digest of RNA samples (Promoter characterization) </SPAN></P><P class="c20 c1"><SPAN class="c0">-    </SPAN><SPAN class="c0"> samples: from 29.8.13</SPAN></P><P class="c20 c1"><SPAN class="c0">-     final reaction should contain: </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR><TD class="c28"><P class="c1"><SPAN class="c0">RNA</SPAN></P></TD><TD class="c132"><P class="c1"><SPAN class="c0">2500 ng (2.5 µg)</SPAN></P></TD></TR><TR><TD class="c28"><P class="c1"><SPAN class="c0">DNase I (Fermentas)</SPAN></P></TD><TD class="c132"><P class="c1"><SPAN class="c0">2.5 µl</SPAN></P></TD></TR><TR><TD class="c28"><P class="c1"><SPAN class="c0">10x DNase I buffer</SPAN></P></TD><TD class="c132"><P class="c1"><SPAN class="c0">2.5 µl</SPAN></P></TD></TR><TR><TD class="c28"><P class="c1"><SPAN class="c0">RNase free water</SPAN></P></TD><TD class="c132"><P class="c1"><SPAN class="c0">Filled up to 25 µl total volume</SPAN></P></TD></TR></TBODY></TABLE><P class="c20 c1"><SPAN class="c0"> -     RNA and RNase free water were prepared in RNA-PCR tubes (autoclaved twice) (total volume: 10.5 µl), then 14. 5 µl of the following mastermix (for 20 reactions) were added: </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR><TD class="c6"><P class="c90 c1"><SPAN class="c0 c2">Component of 20x mastermix</SPAN></P></TD><TD class="c73"><P class="c90 c1"><SPAN class="c0 c2">Volume</SPAN></P></TD></TR><TR><TD class="c6"><P class="c90 c1"><SPAN class="c0">DNase I (Fermentas)</SPAN></P></TD><TD class="c73"><P class="c90 c1"><SPAN class="c0">50 µl</SPAN></P></TD></TR><TR><TD class="c6"><P class="c90 c1"><SPAN class="c0">10x DNase I buffer</SPAN></P></TD><TD class="c73"><P class="c90 c1"><SPAN class="c0">50 µl</SPAN></P></TD></TR><TR><TD class="c6"><P class="c90 c1"><SPAN class="c0">RNase free water</SPAN></P></TD><TD class="c73"><P class="c1 c90"><SPAN class="c0">190 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c1"><SPAN class="c0">The RNA-RNase free water preparations were (table from 29.8.13 expanded and partially re-written for better deciphering…):</SPAN></P><P class="c1"><SPAN class="c0">1. sample name and number; 2. RNA concentration, 3. A</SPAN><SPAN class="c0 c15">260nm</SPAN><SPAN class="c0">/A</SPAN><SPAN class="c0 c15">280nm</SPAN><SPAN class="c0">; 4. A</SPAN><SPAN class="c0 c15">260nm</SPAN><SPAN class="c0">/A</SPAN><SPAN class="c0 c15">230nm</SPAN><SPAN class="c0">, 5. Volume of RNA solution used for RNA-RNase free water preparation (“µl RNA”); 6. Volume of RNase free water used for RNA-RNase free water preparation (“µl dH</SPAN><SPAN class="c0 c15">2</SPAN><SPAN class="c0">O”)</SPAN></P><A href="https://2013.igem.org/File:Goe-02.09.13-RT-4.JPG" target="_blank"><P class="c0 c55">Click to view full resolution</P></A><P class="c20 c1"><SPAN class="c0">-     Incubation for 30 min at 37 °C in PCR cycler (program: DNase1)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     Addition of 2.5 µl EDTA (50 mM stock solution, Fermentas), incubation for 10 min at 65 °C in PCR cycler (program: DNase2)</SPAN></P><P class="c20 c1"><SPAN class="c0">-     Samples stored at -20°C in yellow-tip-box rag (freezer box 3)</SPAN></P><P class="c1"><SPAN class="c0 c2">DNase I digest – Control PCR if all DNA was digested by DNase I (Promoter characterization) </SPAN></P><P class="c1"><SPAN class="c0 c7">Principle: DNA polymerase only amplifies DNA, but not RNA; if residual DNA from E. coli is present in RNA after DNase I digest, one can test this by PCR amplification of an E. coli housekeeping gene</SPAN></P><P class="c1"><SPAN class="c0">Samples: RNA after today’s DNase I digest</SPAN></P><P class="c1"><SPAN class="c0">Primers: qRT-PCR primers for rpoA (iGEM_102/103) →  these primers gave a big primer cloud at bottom of lane in qRT-PCR primer test PCR, but no band in neg. control at 280 bp</SPAN></P><P class="c1"><SPAN class="c0">Note: since DNA matters in this case, but not the RNA, one can use normal water for the PCR. </SPAN></P><P class="c20 c1"><SPAN class="c0">-     Final PCR reaction should contain: </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c34"><TBODY><TR><TD class="c78"><P class="c1"><SPAN class="c0 c2">Component</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0 c2">Volume</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">dNTPs</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">Primer iGEM_102</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">Primer iGEM_103</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">PfuS</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">1 µl (mistake: one should use only 0.5 μl to save material!)</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">Template (RNA after NaseI-treament, water or chrom. DNA)</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">1 µl</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">HF buffer 5x</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">5 µl</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0">HPLC water</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0">15 µl (use 15.5 μl if only 0.5 μl PfuS!)</SPAN></P></TD></TR><TR><TD class="c78"><P class="c1"><SPAN class="c0 c2">total</SPAN></P></TD><TD class="c42"><P class="c1 c27"><SPAN class="c0 c2">25 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c20 c1"><SPAN class="c0">-     A mastermix consisting of all components except for the template was prepared (in the beginning a 20 x master mix was made, but this was just enough for all reactions with RNA after DNaseI digest as a template. Since nothing was left for the (optional) neg. control (HPLC water as template) and the (obligate) positive control (</SPAN><SPAN class="c0 c7">E. coli</SPAN><SPAN class="c0"> chrom. DNA), an additional 2x mastermix was made and used for the controls). In all cases, 24 µl mastermix were added to 1 µl of template prepared in normal PCR tubes.</SPAN></P><P class="c20 c1"><SPAN class="c0">-     PCR protocol: same as for qRT-PCR primer test PCR, run ON</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV><P class="timeline-title goe-array">Low and high phospahte medium for RT-PCR</P><DIV class="timeline-cont"><H2><SPAN lang="EN-GB" style="font-size:11.0pt;line-height:115%;font-family:
  &quot;Arial&quot;,&quot;sans-serif&quot;;text-decoration:none">Low and high phospahte medium for
  RT-PCR</SPAN></H2><DIV align="center"><TABLE class="MsoNormalTable" border="1" cellspacing="0" cellpadding="0" style="border-collapse:collapse;border:none"><TBODY><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="121" style="width:91.0pt;border:solid windowtext 1.0pt;border-left:
    none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">MW (g/mol)</SPAN></P></TD><TD width="123" style="width:92.4pt;border:solid windowtext 1.0pt;border-left:
    none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Stock</SPAN></P></TD><TD width="124" style="width:92.8pt;border:solid windowtext 1.0pt;border-left:
    none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">V (1000 ml)</SPAN></P></TD><TD width="114" style="width:85.75pt;border:solid windowtext 1.0pt;
    border-left:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">V (50 ml)</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">50 mM TRIS pH7.0</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">121.14</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">50 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">2.5 ml</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">3.03 mM (NH<SUB>4</SUB>)<SUB>2</SUB>
    SO<SUB>4</SUB></SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">132.14</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">20% = 1.51 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">2 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">100 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">6.8 mM Na-Citrate</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">294.1</SPAN></P><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">mit NaOH pH 7.0</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">6.8 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">340 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">3.04 mM FeCl<SUB>3</SUB></SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">270.3</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">3 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">150 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 mM MnCl<SUB>2</SUB></SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">197.91</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">50 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">3.5 mM MgSO<SUB>4</SUB></SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">246.48</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">3.5 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">175 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">0.01 mM ZnCl<SUB>2</SUB></SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">136.28</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">1 mM</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">500 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">0.5% Glukose</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">50%</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">500 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center;line-height:150%"><SPAN lang="EN-GB" style="font-size:11.0pt;line-height:150%;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">0.05%
    CAA</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10%</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">17.5 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">875 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10 mM L-Arg</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">174.2</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10% = 0.57 M</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">500 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Tryptophan</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">5 mg/ml</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">10 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">500 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">3.5 mM KH<SUB>2</SUB>PO<SUB>4</SUB></SPAN></P><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;;background:red">High</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">136.09 mit KOH pH
    7.0</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">500 mM</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">7 ml</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">350 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">0.065 mM KH<SUB>2</SUB>PO<SUB>4</SUB></SPAN></P><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;;color:white;
    background:navy">Low</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">136.09</SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">500 mM</SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">130 µl</SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">6.5 µl</SPAN></P></TD></TR><TR style="height:36.85pt"><TD width="134" style="width:100.15pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Wasser</SPAN></P></TD><TD width="121" style="width:91.0pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="123" style="width:92.4pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="124" style="width:92.8pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD><TD width="114" style="width:85.75pt;border-top:none;border-left:none;
    border-bottom:solid windowtext 1.0pt;border-right:solid windowtext 1.0pt;
    padding:0cm 5.4pt 0cm 5.4pt;height:36.85pt"><P class="MsoNormal" align="center" style="text-align:center"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P></TD></TR></TBODY></TABLE></DIV><P class="MsoListParagraphCxSpFirst" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">The
  same strains were used as before</SPAN></P><P class="MsoListParagraphCxSpLast" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Procedure
  to do the RNA was also the same</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">The
  digestion was redone but I forgot to save the picture…extremely clever…</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Results:</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">The
  idea was that in cells with low amounts of c-di-AMP there is a higher
  expression of <I>ydaO</I> because the riboswitch is not present. In cells
  with much c-di-AMP the riboswitch should be present and therefore there
  should be less expression of the <I>ydaO</I> gene.</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Reminder:</SPAN></P><P class="MsoListParagraphCxSpFirst" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">44
  means GP 1344 </SPAN><SPAN lang="EN-GB" style="font-family:Wingdings">à</SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> hyperactive strain =
  much c-di-AMP production</SPAN></P><P class="MsoListParagraphCxSpLast" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">46
  means GP 1346 </SPAN><SPAN lang="EN-GB" style="font-family:Wingdings">à</SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> knock out mutant =
  little c-di-AMP production</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Note:</SPAN></P><P class="MsoListParagraphCxSpFirst" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">the
  wt in high phosphate has the same c-di-AMP amount like the 1344 in low
  phosphate</SPAN></P><P class="MsoListParagraphCxSpLast" style=""><SPAN lang="EN-GB" style="font-family:Symbol">·<SPAN style="font:7.0pt &quot;Times New Roman&quot;">        </SPAN></SPAN><SPAN lang="EN-GB" style="font-family:&quot;Arial&quot;,&quot;sans-serif&quot;">Furthermore:
  c-di-AMP levels in low phosphate are very similar to c-di-AMP levels in CSE
  0,5% Glc medium (see 09.08.2013)</SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P><P class="MsoNormal"><SPAN lang="EN-GB" style="font-size:11.0pt;font-family:&quot;Arial&quot;,&quot;sans-serif&quot;"> </SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Goettingen/NoteBook_w14">http://2013.igem.org/Team:Goettingen/NoteBook_w14</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/NoteBook_w14" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/NoteBook_w14" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w14&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w14&amp;oldid=256411" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>