"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w10
From 2013.igem.org
09th
Miniprep of green clones, 2nd round RD from yesterday, Colony PCR from 8.8.13 – Gel run, MiniPrep of P3op clones, Preparation of P3op + pSB1C3 and P3op + part 6 clones for sequencing, PCR purification of test Riboswitch PCR reactions from 7.8.13
test of newly competent cells XL1 Blue is ok
many clones on Amp-Xgal plate, no on all control plates
Miniprep of Part1 (nothing grew in the inoculated culture of part8 -> because we used the wrong resistance)
new inoccutaion of part8 from cryo stock in LB-Amp has to be done on monday
Miniprep of green clones:
part6,1 Clone1, Clone 2, Clone 3, clone 4, clone 5, clone 6, clone 7
Nanodrop:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
part 6,2 clone 1
261.5
1.88
2.77
part 6,2 clone 2
226.5
1.85
2.18
part 6,2 clone 3
212.9
1.84
2.11
part 6,2 clone 4
232.0
1.85
2.24
part 6,2 clone 5
246.4
1.87
2.15
part 6,2 clone 6
226.7
1.87
2.30
part 6,2 clone 7
222.1
1.90
2.27
part 1
254.8
1.86
2.19
2nd round RD from yesterday
Parts 1-4: SpeI FD
YFP, CFP and Part 8: XbaI FD
 
Katrin Gunkas Überprotokoll:
                  40µl DNA (all the elution)
                  5µl Enzyme
                  5µl Buffer FD
         = 50µl reaction
-->incubation at 37°C for 1.5h
-->cleanup using the PCR cleanup kit, elution in 30+10µl prewarmed H2O
-->Nanodrop and Gel monday!
Colony PCR from 8.8.13 – Gel run
 
-  4x 1%-agarose-1xTAE gels
-  Loading of 3 µl 2 log ladder as a marker
-  Loading of 5 µl of Colony-PCR samples
-  Run at 60 – 100 V
-  EtBr staining + destaining with water
-  UV detection
Gel 1:
Gel2:
Gel3:
Gel4:
-> w/o-insert-control clones correspond to part 7 plasmid (Terminator insert; PCR product 129 bp of Terminator + 140 bp (additional bases because of VF2 primer binding site) + 170 bp (additional bases because of VR primer binding site) = 439 bp
-> P3op + pSB1C3 – clones 1, 3 and 4: expected band at ca. 360 bp (ca. 50 bp P3op + 310 bp VR/VF2)
-> Riboswitches + pSB1C3 – clones: expected bands not seen, only bands at lower bp (SeqMan alignment of Riboswitch sequences and VR and VF2 --> the primers were not aligned to the riboswitch sequence --> no unspecific binding of primers somewhere in riboswitch sequence that could explain product shorter than the expected riboswitch products or the terminator product)
(expected bands:
iGEM_40/41: 231/213 bp + 310 bp = 541/523 bp ;
iGEM_40/43: 376/403 bp+ 310 bp = 686/713 bp ;
iGEM_41/42: 318/370 bp + 310 bp = 628/680 bp ;
iGEM_42/43: 463/466 bp + 310 bp = 773/767 bp)
-> DarR + pSB1C3 – clones: expected bands not seen (ca. 630 bp + 310 bp = 940 bp)
-> Cloning with Taq-amplified PCR products (Riboswitches) or PolyA overhang added by Taq after PCR (DarR) seemed not to allow restriction of ends with EcoRI and PstI
-> Cloning using hybridization oligos seemed to work quite well (3 positives out of 4 clones)
-> Repeat cloning of Riboswitches with new primers harboring the overhangs
-> MiniPrep of P3op clones 1, 3, 4 and their re-ligation clone C
MiniPrep of P3op clones
-  P3op clones 1, 3, 4 and re-ligation clone C
-  With Nucleospin kit from Macherey-Nagel
-  Elution with 30 µl HPLC water (pre-warmed)
-  NanoDrop:
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
Clone C
131.8
1.78
1.26
P3op C1
175.7
1.88
1.70
P3op C3
162.1
1.85
1.58
P3op C4
168.8
1.91
1.88
 
Preparation of P3op + pSB1C3 and P3op + part 6 clones for sequencing
-  Sequencing by SeqLab: in general 600 – 700 ng plasmid DNA in 6 µl + 1 µl Primer in 1:5 dilution of stock (tip: if plasmid concentration is not too high – i.e. 300 or 400 ng/µl, one can simply use 6 µl of plasmid solution with 100 – 200 ng/µl)
-  Part 6.2 C1: 6 µl of 261.5 ng/µl-plasmid solution + 1 µl iGEM 38 (VF2) 1:5 --> tube 1
-  Part 6.2 C2: 6 µl of 226.5 ng/µl-plasmid solution + 1 µl iGEM 38 (VF2) 1:5 --> tube 2
-  Sequencing of P3op C1: 6 µl of 175.7 ng/µl-plasmid solution + 1 µl iGEM 38 (VF2) 1:5 --> tube 3
-  Sequencing of P3op C1: 6 µl of 175.7 ng/µl-plasmid solution + 1 µl iGEM 39 (VR) 1:5 --> tube 4
PCR purification of test Riboswitch PCR reactions from 7.8.13
-  Qiagen PCR purification kit
-  Purification of iGEM_67/70: 4 µl and 8 µl reactions
-  Purification of iGEM_69/68: 4 µl and 8 µl reactions
-  Purification of iGEM_69/70: 2 µl, 4 µl and 8 µl reactions
-  Purification of iGEM_67/68: 4 µl and 8 µl reactions
-  Elution with 30 µl HPLC water (pre-warmed)
-  Concentration (NanoDrop):
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
iGEM_67/70
115.6
1.77
2.77
iGEM_69/68
84.0
1.67
3.65
iGEM_69/70
105.5
1.82
2.01
iGEM_67/68
122.8
1.76
2.83
Fold ↑
Staining of samples
Staining of samples
·         According   to protocoll of the AGS (p.   131ff)
Cyclic-di-AMP   concentration from the strains used fort he microarray in Groningen.
With the microarray we wanted to analyse different expression levels in the wildtype, GP 1344   (supposed to have much c-di-AMP) and GP1346 (supposed to have little c-di-AMP   amounts).
The c-di-AMP amounts were confirmed via c-di-AMP measurements (see figure   above; the measurements were done at the MHH in Hannover).
The results from the microarray analysis were useless since only one dye   worked. Therefore we did RT-PCR studies (see below).
qRT-PCR
Fold ↑
08th
Preparation of CFP and YFP Samples from Katrins Miniprep yesterday for sequencing by SeqLab, First Round RD of the parts for new cloning strategy, Colony PCR for Trasnformations from 6.8.13, Transformation from 6.8.13, Primer design for DarRrev for DarR reporter system
Preparation of CFP and YFP Samples from Katrins Miniprep yesterday for sequencing by SeqLab
 Pipetted ~700ng of DNA + 0.25µl undiluted primer + Water to 7µl as follows:
 1: CFP C1 VF2
2: CFP C2 VR
3: YFP C2 VF2
4: YFP C1 VR
Gel doc of PCR from Katrin yesterday test PCR part7 C1
M/2/4/8/NC
bands are ok so PCR purification was performed
-> marked as rev. terminator (in to do box)
newly prepared competent cells from yesterday (XL1 Blue) are tested with a 1 ng/ µl pbluescript Trafo and a trafo with just water and one without (see journal above)
put to 37°C at 13:00
inoculation of GFP positive cells
inoculation of part 8 and part1 from cryo stocks
First Round RD of the parts for new cloning strategy
Parts 1-4: PstI FD
YFP, CFP and Part 8: EcoRI FD
 
Katrin Gunkas Überprotokoll:
                  1.5µg DNA
                  4µl Enzyme
                  4µl Buffer FD
                  xµl H2O
         =40µl reaction
-->incubation at 37°C for 1.5h
-->cleanup using the PCR cleanup kit, elution in 30+10µl (40 overall) prewarmed H2O
-->Nanodrop results on eppis (all around 30ng), about 1.2µg DNA retrieved in all samples (Which is highly unlikely and once again should make us think about nanodrop measurements)
Tomorrow 2nd round! Parts 1-4 with SpeI and the others with XbaI!
Transformation from 6.8.13
-   plates: no colonies on 50 µl-plates observed, but on rest-plates
-  Neg. control plates: no colonies
-  w/o DarR insert plate: few colonies
-  w/o Riboswitch insert plate: few colonies
-  w/o Promoter3operator insert plate: few colonies
-  Promoter3operator + pSB1C3: few colonies
-  Riboswitch 40/41 + pSB1C3: several colonies
-  Riboswitch 40/43 + pSB1C3: several colonies
-  Riboswitch 41/42 + pSB1C3: several colonies
-  Riboswitch 42/43 + pSB1C3: many colonies
-  DarR + pSB1C3: many colonies
-  Promoter3operator + part 6: some colonies --> see above (fluorescence microscopy)
-  Plates stored in big fridge at 4 °C
 
Colony PCR for Trasnformations from 6.8.13
-  according to protocol from ….; preparation of MasterMix for 55 reactions --> 25 µl
-  PCR tube Colour/Number coding of clones:
o   w/o DarR insert plate: 1 colony picked (A; white)
o   w/o Riboswitch insert plate: 1 colony picked (B; white)
o   w/o Promoter3operator insert plate: 1 colony picked (C; white)
o   Promoter3operator + pSB1C3: 4 colonies picked (1, 2, 3, 4; yellow)
o   Riboswitch 40/41 + pSB1C3: 8 colonies (5, 6, 7, 8, 9, 10, 11, 12; orange)
o   Riboswitch 40/43 + pSB1C3: 9 colonies (13, 14, 15, 16, 17, 18, 19, 20, 21; pink)
o   Riboswitch 41/42 + pSB1C3: 7 colonies (22, 23, 24, 25, 26, 27, 28; violet)
o   Riboswitch 42/43 + pSB1C3: 10 colonies (29, 30, 31, 32, 33, 34, 35, 36, 37, 38; blue)
o  DarR + pSB1C3: 10 colonies, but only first 8 inoculated for MiniPrep (not enough medium prepared for 48 and 47); (39, 40, 41, 42, 43, 44, 45, 46, 47, 48; green)
-  PCR taken out, addition of 5 µl 5xLD to each reaction; reactions stored at 4 °C ON --> gel run tomorrow
-  Inoculation of clones 48 and 47 in 4 ml LBCm and incubation ON at 37 °C, ca. 200 - 210 rpm
Primer design for DarRrev for DarR reporter system
-  Sequence of DarR fwd primer + suffix with overhang
-  Sequence of DarR rev primer + prefix with overhang
-> Ordered by Katrin G.
Fold ↑
c-di-AMP extraction
c-di-AMP extraction
·         the   same strains as used in Groningen for the array.
·         According   to protocoll of AGS (p. 131ff.)
 
Fold ↑
07th
PCR for amplification of different riboswitch inserts: Test PCR for optimal primer concentration, Gel run: Riboswitch Test PCR and Test RD of plasmids from YFP and CFP clones, Primers for generating reverse terminator insert
PCR for amplification of different riboswitch inserts: Test PCR for optimal primer concentration
-   Primers iGEM_67/68/69/70
1x reaction (50 µl)
Component
Volume
Negative control
Volume
1x primer concentration
Volume
2x primer concentration
Volume
4x primer concentration
5x HF buffer
10 µl
10 µl
10 µl
10 µl
dNTP mix (12.5 mM each)
2 µl
2 µl
2 µl
2 µl
Primer fwd (5 pmol)
2 µl
2 µl
4 µl
8 µl
Primer rev (5 pmol)
2 µl
2 µl
4 µl
8 µl
Chromosomal DNA B. subtilis 168 (or water for neg. control)
1 µl
1 µl
1 µl
1 µl
PfuS
1 µl
1 µl
1 µl
1 µl
dH2O
32 µl
(12 µl)
32 µl
(12 µl)
28 µl
(8 µl)
20 µl
(0 µl)
         - preparation of master mix for 20 reactions containing
 
Component
Volume
dNTP mix (12.5 mM each)
40 µl
5x HF buffer
200 µl
PfuS
20 µl
dH2O
400 µl
Total
660 µl
 
- addition of template (chrom. DNA/dH2O), primers and a part of the water (indicated in parenthesis in table for composition of each reaction) individually, then addition of 33 µl master mix
 
Step
Temperature
Time
Initial denaturation
98.5 °C
5 min
Denaturation
98.5 °C
30 s
Annealing
Different; see table below…
(TA = TM (≈66 °C) – 6 °C)
35 s
Elongation
72 °C
2 min
Final elongation
72 °C
10 min
Hold
15 °C
∞
 
 
 
Primers
TA = 0.5 * [Tm(primer1) + Tm(Primer2)] – 6 °C
insert size (no overhangs from primers)
Name of Protocol in cycler
Colour code of epi tube
Riboswitch
with native Promoter and RBS
iGEM_69
iGEM_70
51 °C
 
466 bp
Ribo1
Orange
Riboswitch with native Promoter
iGEM_69
iGEM_68
55.2°C
370 bp
Ribo2
Violett
Riboswitch only
iGEM_67
iGEM_68
59.7°C
213 bp
Ribo3
Yellow
Riboswitch with native RBS
iGEM_70
iGEM_67
55.5 °C
403 bp
Ribo4
Blue
 
Gel run: Riboswitch Test PCR and Test RD of plasmids from YFP and CFP clones
-   1.5 % Agarose-1xTAE gel
-   4 µl PCR reaction + 1 µl 5x DNA Loading Dye
-   5 µl of Test RDs; 1 µl uncut plasmid + 1 µl 5x DNA Loading Dye + 3 µl dH₂O
-   3 µl 2 log ladder
-   Run at ca. 100 V in 1xTAE buffer
-   Staining in EtBr and destaining in water
-    UV detection
Gel 1:
Gel 2:
-> RD: Expected band ca. 750 bp + ca. 2 kb for all digests. The expected bands were obtained for all RDs --> all clones contain plasmids --> prepare the plasmids for sequencing by G2L
-> PCR: for expected products: add ca. 2x 20 bp to sizes indicated above. The expected bands were obtained. However, for iGEM_67/68, iGEM_69/68 and iGEM_67/70, a slight band at approx. the bp of 2x expected was observed for 2 µl reaction (“product dimmers?”), this seemed to decrease with increasing primer amount. Hig iGEM_69/70 primer amounts seemed to interfere with PCR. The negative control only shows primer clouds at bottom of lanes. The optimal primer concentrations for the different PCR reactions seem to be:
 
 
Primers
Primer amount (1:20 dilution)
Riboswitch with native Promoter and RBS
iGEM_69
iGEM_70
2 µl
Riboswitch with native Promoter
iGEM_69
iGEM_68
4 µl
Riboswitch only
iGEM_67
iGEM_68
4 µl
Riboswitch with native RBS
iGEM_70
iGEM_67
4 µl
 
Primers for generating reverse terminator insert
    -> Primers arrived: dissolved in HPLC water and diluted 1:20
-> Primers tend to form secondary structures --> Test PCR for optimal primer concentration
-> Annealing temperature: TA = 0.5 * [Tm(iGEM_71) + Tm(iGEM_72)] – 6 °C = 0.5 *[60 °C + 59.1] – 6 °C =53.55 °C
 
TEST PCR
1x reaction (50 µl) using part 7 C1 plasmid as template
Plasmid concentration when used as a template: 10 – 20 ng/ reaction
Part 7 C1 purified on 11.7.13; concentration = 120.5 ng/µl --> 1:10 dilution in water --> ca. 12 ng/µl
Component
Volume
Negative control
Volume
1x primer concentration
Volume
2x primer concentration
Volume
4x primer concentration
5x HF buffer
10 µl
10 µl
10 µl
10 µl
dNTP mix (12.5 mM each)
2 µl
2 µl
2 µl
2 µl
Primer fwd (5 pmol)
2 µl
2 µl
4 µl
8 µl
Primer rev (5 pmol)
2 µl
2 µl
4 µl
8 µl
Part 7 C1 plasmid (ca. 12 ng/µl) or water for negative control
1 µl
1 µl
1 µl
1 µl
PfuS
1 µl
1 µl
1 µl
1 µl
dH2O
32 µl
(12 µl)
32 µl
(12 µl)
28 µl
(8 µl)
20 µl
(0 µl)
 
-  preparation of master mix for 5 reactions containing
 
Component
Volume
dNTP mix (12.5 mM each)
10 µl
5x HF buffer
50 µl
PfuS
5 µl
dH2O
100 µl
Total
165 µl
 
-   addition of template (chrom. DNA/dH2O), primers and a part of the water (indicated in parenthesis in table for composition of each reaction) individually, then addition of 33 µl master mix
 
PCR protocol
Step
Temperature
Time
Initial denaturation
98.5 °C
5 min
Denaturation
98.5 °C
30 s
Annealing
53.5 °C
35 s
Elongation
72 °C
2 min
Final elongation
72 °C
10 min
Hold
15 °C
∞
-> run ON
Fold ↑
06th
Restriction digest of DarR (polyadenylated PCR product), Purification of pBluescript(RD with EcoRI) from gel, Ligation of DarR/hybridization oligo inserts with pSB1C3/part6, Gel run of DarR insert and pBluescript+EcoRI, Design of oligos for self-hybridization, Transformation of parts from distribution kit 2013 (CFP, YFP)
Restriction digest of DarR (polyadenylated PCR product)
-> Volume: 12 µl of unpurified DarR polyA reaction
- Preparation of the digest:
 
DarR polyA reaction mix
12 µl
EcoRI FD
1.5 µl
PstI FD
1.5 µl
10xFD buffer
2 µl
dH₂O
3 µl
Total
20 µl
 
-> 1 h at 37 °C in ThermoCycler
-  Purification of RD reaction using PCR clean up kit (Qiagen): 500 µl PB buffer, 1x elution with 22 µl pre-warmed HPLC water (incubation for ca. 4 min at RT, ca. 1 min at 50 °C)
-  Concentration (NanoDrop)
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
DarR insert E+P
6.1
1.34
1.15
Purification of pBluescript(RD with EcoRI) from gel
-  1.1088 g (2 ml epi with gel) – 1.0376 g (another empty 2ml epi) = 0.0712 g --> ca. 70 mg gel --> 210 µl QG buffer were used
-  Elution with 2x 22 µl pre-warmed HPLC water
-  Concentration (NanoDrop)
 
Sample
ng/µl
A260nm/A280nm
A260nm/A230nm
pBluescript + EcoRI
6.6
1.92
0.09
 
Ligation of DarR/hybridization oligo inserts with pSB1C3/part6
-  Ligation reactions
A) DarR insert (from polyA product) + pSB1C3 E+P (from part 7; 31.7.13)
B) hybridization oligo Promoter 3-DarRoperator + pSB1C3 E+S (from part 7)
C) hybridization oligo Promoter 3-DarRoperator + part 6 E+ X
For DarR ligation: ratio 1:3 was kept --> ca. 50 ng vector (2070 bp) + ca. 43 – 51 ng insert (600-700 bp)
For Hyp oligo: 20- 30 ng vector + half amount of reaction mix from
Component
DarR
Hyb oligo + pSB1C3
Hyb oligo + part 6
Insert/dH₂O (w/o insert control)
8 µl (6.1 ng/µl)
8.05 µl
8.05 µl
T4 Ligase (ThermoScientific)
2 µl
2 µl
2 µl
T4 ligation buffer 10x
2 µl
2 µl
2 µl
Vector
1 µl (56.6 ng/µl)
5 µl (6.3 ng/µl)
3 µl (9.8 ng/µl)
dH₂O
7 µl
4.95 µl
2.95 µl
Total
20 µl
20 µl
20 µl
-> Incubation at 16 °C for several hours (started at 11:30)
Gel run of DarR insert and pBluescript+EcoRI
-   1 % agarose-1xTAE gel
-    Loading of 3 µl 2 log ladder
-    Loading of 1 µl pBluescript (2.8.13; 342.0 ng/µl) + 3 µl dH₂O + 1 µl 5xLD
-    Loading of 4 µl pBluescript purified EcoRI RD + 1 µl 5xLD
-    Run at 100 V
-    EtBr staining + destaining in water
-    UV detection
Gel:
>
-> EcoRI-linearzied pBluescript seems to be pure. Still, shortly below the linearized-plasmid-band, some additional weak bands were seen --> possibly unspecific side products? But the strong band of the supercoil-plasmid is gone --> further digest with PstI
-> For DarR insert, an additional band at 0.1 kb was obtained --> polyA’s that were cut off by the enzymes??? Or a non-specific side product from PCR? --> if this fragment can be cloned, we will notice it when the transformants are tested in Colony PCR…
Digest of pBluescript-EcoRI with PstI
Ca. 33 µl pBluescript left
-> Addition of 4 µl FD buffer and 4 µl PstI FD; incubation for 1 h at 37 °C
Design of oligos for self-hybridization
-> These oligos contain the reverse complement sequence of the parts indicated below, since they are designed for reverse integration into pSB1C3 (DarR transcription unit)
-> Oligos hybridization leads to EcoRI and SpeI overhangs
-  For strong RBS (part 8)
-  For promoters parts 1,2,3,4 (an additional sequence of ca. 60 bp (reverse translation of “cyclicdiampacterim”) was added in front of the promoter so that binding of RNA-Pol does not interfere with binding of RNA-Pol to promoter of GFP transcription unit)
Transformation of parts from distribution kit 2013 (CFP, YFP)
Inoculation of 3 clones of each transformation in LBcm for miniprep
Preparation of back-up plates
-> Forgotten in fridge (4 °C) on 5.8.13
-> Put to 37 °C by Dominik today
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w10 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
