"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/Project/OurProject
From 2013.igem.org
The beast and its Achilles heel:
 A novel target to fight multi-resistant pathogenic bacteria
Home Project ▼
   Overview       Reporter Team       Array Team       DAC Team    Outlook    Organization Team Collaborations Parts Safety
Team ▼
   Instructors    Students Gallery Acknowledgement Sponsors
Timeline Human Practice▼
Panel discussion BMBF Congress Interviews Online-Qustionnaire Survey Results
Outreach Achievements
Project Overview:
Background
The threat
c-di-AMP
Our Project
Reporter systems
Microarray
Diadenylate cyclase
Contents
1 Navigation:
2 Background
3 The threat
4 c-di-AMP
4.1 Our Project
5 Reporter systems
6 Microarray
7 Diadenylate cyclase
7.1  
Navigation:
Our Project
Overview
Reporter Team
Array Team
DAC Team
Outlook
Organization Team
Collaborations
Parts
Safety
Background
The threat
The discovery of penicillin by Alexander Fleming in 1928 and the following broad application of antibiotics have marked a major victory of mankind in the battle against infectious diseases. The intake of the wrong dosage or even the unnecessary use of antibiotics in medicine and agriculture have led to an enhanced appearance of resistant bacteria. Subsequently, we have reached the point where bacteria have evolved resistances against all common antibiotics. The drugs of last resort are rapidly being exhausted in cases like the methicillin-resistant Staphylococcus aureus (MRSA) or the vancomycin-resistant Enterococcus (VRE). According to the US Threat Report, 2 million persons are infected by resistant bacteria annually  whereof  23,000 die.
Further misuse will lead us back to medieval conditions not only for postoperative patients but even small infections will be able to cause severe problems and may cost many lives. Tragically, only four new classes of antibiotics made it to the market in the last 40 years as research in the field of antibiotics is getting less profitable due to high research and development costs and the poor chance of success to actually launch a new antibiotic (Cooper and Shlaes, 2011).
Therefore, we should better control the use of antibiotics in medicine as well as agriculture. Meanwhile, we need to develop new antibiotics, which can sufficiently eliminate the pathogenic bacteria without affecting beneficial and harmless bacteria such as the well-known gut bacterium Escherichia coli (Witte et al., 2008).
c-di-AMP
What a good antibiotic should target? The answer has been given by Dr. Sven Halbedel. From his point of view, a good antibacterial target should:
● be essential for growth/ virulence of pathogenic bacteria
● have no homolog in humans
● be pleiotropic
● have no suppressors
● be conserved among pathogenic bacteria
Of all those possible targets for novel antibiotics, c-di-AMP is the "shining star". Though very recently discovered (in the year 2008), it has already caught a lot of attention. It is the only known essential signaling nucleotide. It exists and plays a vital role in a wide range of Gram-positive bacteria like Bacillus, Listeria, Streptococcus and Staphylococcus. Meanwhile its trace has never been found in known beneficial Gram-negative bacteria nor human beings. With all these reasons, c-di-AMP and its regulatory compartments become the most heated targets of interest in the development of new antibacterial substances.
Therefore the iGEM Team Göttingen addresses the problem of multi-resistant bacteria by targeting c-di-AMP,the achilles heel of our enemy.
References
1. Rebecca M. Corrigan & Angelika Gründling, (2013) Cyclic di-AMP: another second messenger enters the fray. Nature Reviews Microbiology11, 513–52'
2. Cooper MA and Shlaes D (2011) Fix the antibiotics pipeline. Nature. 472: 32
3. Witte G, Hartung S, Büttner K, Hopfner KP. (2008) Structural biochemistry of a bacterial checkpoint protein reveals diadenylate cyclase activity regulated by DNA recombination intermediates. Mol Cell. 30(2):167-178
Our Project
Our project is aimed at finding a way to fight against multi-resistant bacteria by targeting c-di-AMP. We made three different approaches.
We built two reporter systems which enable us to visualize the level of c-di-AMP (accomplished by Reporter Team ).
We also searched for the genes in Bacillus subtilis, whose expression level is affected by the level of c-di-AMP. We found ydaO and identified a Riboswitch upstream of its open reading frame, which responds to c-di-AMP. We used the ydaO Riboswitch directly in our second reporter system (accomplished by Array Team ).
Last but not least, we characterized the diadenylate cyclase (DAC) fromListeria monocytogenes. We successfully expressed tagged truncated DAC (catalytic domain) in E.coli and purified it.We analyzed its enzymatic kinetics and crystallized it. In the end, we are able to determine the structure (accomplished by DAC Team )
Reporter systems
For the Reporter Team, our final goal is to build a screening system, which allows quick identification and characterization of substances which are able to disturb c-di-AMP homeostasis in pathogenic bacteria. We believe the accomplished  screening system will be a great help for pharmaceutical industry worldwide in finding new and more effective antibiotics against Gram-positive pathogens, for which c-di-AMP homeostasis is crucial. To accomplish that goal, we first need a reporter system that can visualize different levels of c-di-AMP.
We used a few existing BioBricks and also created new ones to build up our system. At first, we attempted to construct a reporter system which is controlled by DarR, a transcriptional inhibitor identified in Mycobacterium smegmatis. This reporter system consists of three parts, namely a constitutively active promoter, the operator sequence DarR binds and a reporter gene cassette. c-di-AMP is able to stimulate the binding of DarR and its operator, acting as a co-inhibitor. Therefore the level of c-di-AMP can be visualized by the fluorescence of GFP: the higher the c-di-AMP level is, the lower the GFP fluorescence becomes. The reporter system is transformed into E.coli, which produces no endogens c-di-AMP. Therefore, we are able to test the system by providing the E.coli with c-di-AMP.
The second reporter system is based on the result of our Array Team. They found the gene ydaO, whose expression level is affected by the level of c-di-AMP. When the c-di-AMP level is low, the ydaO expression is up-regulated. We are able to identify a Riboswitch upstream of the ydaO open reading frame and used it in our second reporter system. The ydaO Riboswitch has two states: "ON" and "OFF". The switch between the two states depends on the presence of c-di-AMP: basically, when c-di-AMP is there, the Riboswitch is "OFF" and when there is no c-di-AMP, the Riboswitch is "ON". We cloned a reporter gene cassette CFP downstream of the native promoter + ydaO Riboswitch. This reporter system should act similarly to our first reporter system: when there is c-di-AMP, no signal is expected, but when there is no c-di-AMP, there will be a fluorescence signal.
We modified, improved and created several BioBricks during the construct of our reporter system. To know more, please go to our subteam page and parts page .
Microarray
We focused mainly on the genes, whose expression level is regulated by c-di-AMP. Therefore, we compared the transcriptomic data of wild type with those of a hyperactive strain of Bacillus, which produces more c-di-AMP.
By doing so, we are able to: first, find other compartments which also respond to c-di-AMP and therefore can be used to construct our reporter system, second we can shed light on the signaling web of c-di-AMP.
In the collaboration with iGEM Team Groningen, we accomplished the microarray analysis, which has given us a first glimpse on all genes which could be regulated by c-di-AMP. Among those genes, ydaO caught most our attention. ydaO motif is reported to be a c-di-ATP-sensing Riboswitch in Bacillus substilis (unpublished data). We confirmed the array data with qRT-PCR. And the results consisted with array data. Therefore we believe we identified another genetic element that responds to c-di-AMP and we directly used it to build our second reporter system.
To know more about our work, please visit subteam page and parts page .
Diadenylate cyclase
As the homeostasis of c-di-AMP is very important for Listeria monocytogenes (Witte et al., 2013), a Gram-positive human pathogen, we are convinced that the DAC is a very promising target for the development of highly specific antibiotic substances which exclusively act on Gram-positive bacteria and are not harmful to Gram-negative ones, including the gut bacterium Escherichia coli. Therefore, we aimed to reveal the protein structure of a not yet characterized diadenylate cyclase domain.
First we tried to express the complete CdaA protein of L. monocytogenes in E.coli. Unfortunately, we couldn't get any clone after several attempts. The full length protein might be toxic expressed in E.coli, so we focused on a truncated version of the whole protein excluding the trans-membrane domains. The 173 amino acid protein fragment (100 – 273 of CdaA) harbors exclusively the catalytic domain of CdaA, also referred to as DacA. We attached an N-terminal Strep-tag to the cyclase domain and expressed the protein driven by a T7-promoter. HPLC-MS/MS measurements determined the presence of c-di-AMP and, thus, the activity of the catalytic domain in also in the Gram-negative bacterium E.coli in vivo.
Excluding the trans-membrane domains, the protein localizes to the cytoplasm and can be purified using streptavidin purification columns. After we purified the DAC protein, we wanted to prove the functionality also in vitro. The results showed that the purified catalytic domain of CdaA is still functional in vitro in proper conditions. Thus, the truncated diadenylate cyclase domain is qualified to construct a screening system with the ectopic production of c-di-AMP in E.coli, as a further goal of our whole project. Also our experiments went on to our final step, crystallization and, finally, the determination of the 3D protein structure.
In collaboration with Dr. Achim Dickmanns , we successfully obtained protein crystals and with the help of Dr. Piotr Neumann , we managed to determine the 3D structure from the X-ray diffraction pattern.
We believe that the 3D structure of the diadenylate cyclase domain of Listeria can help pharmaceutical industry in developing novel antibiotics that interfere with DAC function.
In our progress we also created a new BioBrick, to know more, please go to our subteam page and parts page .
Mr.Green Coli with his full Bio Gear
 
Previous Next
Retrieved from " http://2013.igem.org/Team:Goettingen/Project/OurProject "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
