"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w2
From 2013.igem.org
14th
Observation of Back-up plates and preparation of media
Observation of Back-up plates
-          Part 1, C1 and C2 probably very light pink
-          Part 2, C2 and C3 became light pink, but C1 stayed white (compare with   restriction analysis
13.6.13
)
 
Preparation of media for inoculation   on Sunday (by Katrin Gunka)
-          For Plate Reader assay: 4 ml LB + 4 μl Cm or Amp for C1, C2 and C3 of parts 1 – 4 and part 6,   negative control DH5α in 4 ml LB
-          For Plasmid Mini-Preparation: 10 ml LB + 10 μl Amp for inoculation of C2   Part 2
Fold ↑
13th
Test restriction of plasmids containing parts 1 – 8
Test restriction of plasmids   containing parts 1 – 8
-          Reactions/Master Mixes: see Excel-sheet “dropbox > iGEM > Reporter-Team > digestion system 2hour”
-          Plasmids: parts 1 – 6 (plasmids purified on
7.6.13
)   and part 7, 8 (plasmids purified on 11.6.13)
-          1x reaction with single enzyme:
Component
Volume
Enzyme
0.5 µl
Buffer 10x
1 µl
Plasmid DNA
200 ng
Total volume
10 µl
è     For double digestion:
           0.5 µl of each enzyme in case of EcoRI and PstI;
          for SpeI and PstI, ratio 1:2 is recommended à 0.5 µl SpeI and 1 µl PstI used
-          Incubation at 37°C for1.5 hours
-          Additon of 5xDNA-Loading Dye to whole reaction
-          Loading of 8 µl on agarose gel (~1.5 %, 1xTAE), QuickLoad 1 kb ladder   as marker
-          Run at 200 V for 1.5 h in 1xTAE
-          EtBr staining ca. 45 min, destaining in water ca. 30 min
-          UV detection
 
Expected Fragments:
Part no.
Fragments for PstI and SpeI (bp)
Plasmid size (linearization with PstI or SpeI)(bp)
1
~900 + 2000
2948
2
3
4
Part no.
Fragments for PstI and EcoRI(bp)
Plasmid size (linearization with PstI or EcoRI)(bp)
5
???
(if RFP gene à same pattern as expected for parts 1 –     4; if no RPF gene à same pattern as expected for     part 8)
6
917
2029
2946
7
140
2050
2199
8
53
2038
2091
 
Gel: See ppt-file “dropbox > iGEM > Reporter-Team > Geldoc“
Conclusions:
è     Partial digestion (next time: longer incubation time, less plasmid…)
è     For parts 1, 3, 4, 6, 7 and 8,   the expected bands were observed (ok)
è     Band pattern of part 5 resembles   that expected for parts 1 – 4 à this plasmid contains RFP   gene
è     In case of part 2, SpeI was   unable to cut the plasmid à no cleavage for SpeI single   digest + linearization for SpeI/PstI double digest à plasmid contains probably something else   (SpeI/XbaI scar at actual SpeI site?)
è     For cloning: terminator (part 7) and strong RBS (part 8) might be   difficult to extract from the gel, since they are very short à think of other cloning strategy without gel extraction (e.g.   “play” with resistances of plasmid backbones to digest backbone after cutting   out part…)
 
Preparation of DarR PCR samples for   sequencing
-          4 µl of DarR seq PCR product of reaction 1 + 1 µl of iGEM_34 (fwd) or   iGEM_35 (rev)
Tubes:
kgun_1 à iGEM_34
kgun_2 à iGEM_35
-          Sequencing at G2L
Fold ↑
SDS-PAGE of overexpressing cells (#811, 814, 1013) containing pGP172 + cdaA (from #81) and pGP172 + DAC (from #101)
SDS-PAGE of overexpressing cells (#811,   814, 1013) containing pGP172 + cdaA (from #81) and pGP172 + DAC (from #101)
 - take pelletized cells - add 5 μl of 5x SDS loading   dye
 - add 10 μl ofZAP buffer - incubate   for 10 min at 96°C
 → load samples on SDS PAGE and run   gels at first at 80V, then turn to 100V
 - after running: cover gels with   Coomassie Brilliant Blue staining solution and incubae for 30 min
 - wash the gels afterwards twiche   with deionized H2O
 - destain gels overnight in 10%   acetic acid with agitation
Gel:
first pre-experiment Marker (10 - 170 kDa   ladder) | control (0 h) | control (1 h) | control (2 h) | 811 (0 h) | 811 (1   h) | 811 (2 h) | 814 (0 h) | 814 (1 h) | 814 (2 h)
Gel: first pre-experiment 1013 (0 h) |   1013 (1 h) | Marker (10 – 170 kDa ladder) | 1013 (2 h) | 1013 (4.5 h)
The T7 polymerase has a molecular weight   of approximately 99 kDa. The full length cdaA protein from L. monocytogenes   has a molecular weight of approximately 30.5 kDa, fused to a Strep-tag it has   31.5 kDa. The DAC domain of L. monocytogenes has a molecular weight of   approximately 18.8 kDa, fused to a Strep-tag it has 19.8 kDa.
For the first 2 h after gene induction by   IPTG no gene expression could be detected!
 
Gel: repeat pre-experiment control (0 h)   | control (1 h) | 811 (0 h) | 811 (14 h) | 811 (2 h) | 814 (0 h) | 814 (14 h)   | 1013 ( 0 h) | 1013 (14 h) | Marker (10 – 170 kDa ladder)
After 14 h could be gene expression for   both T7 polymerase and DAC domain detected. For cdaA gene expression no   signal could be detected. Therefore in further experiments cells should be   incubated at least for 14 h to obtain high expression levels!
 
Fold ↑
12th
Continue: Repeat: Pre-experiment:   Overexpression of cdaA and DAC domain by IPTG induction -measure OD600 of induced cultures
Continue: Repeat: Pre-experiment:   Overexpression of cdaA and DAC domain by IPTG induction -measure OD600 of induced cultures
Culture
 
OD600
Saved cells [μl] at t0
#811
0.69
145
#814
0.79
127
#1013
1.82
55
control
2.20
46
- pelletize cells and store at - 20?C
 
Fold ↑
11th
Miniprep of Part7,8 and DarR sequencing PCR clean-up
Cryo-Stocks of E. coli transfomants (8, 7 C1)
Stored   in -70 °C (red box)
 
Plasmid Mini-Preparation of parts 8, 7   C1
-          harvesting of ca. 5 ml pellets in 2 ml Epis for future Plasmid   Mini-Preparation à stored at – 20 °C in red box
-          harvesting of remaining culture for today’s prep
è     Performed as on
7.6.13
NanoDrop   – Plasmid concentrations
Part no.
c(DNA)
[ng/µl]
A260/A280
A260/A230
7
67.0
1.98
2.25
8
62.7
1.89
2.08
 
DarR seq PCR purification
-          for DarR seq PCR reactions 1 - 4
-          with QIAquick PCR purification Kit (Qiagen), according to quick start   manual
-          ca. 50 µl of PCR reaction + 250 µl of PB buffer
reactions 2 and 4 turned violet à addition of 10 µl NaAc 3.3 M, pH = 5.0
-          elution: 30 µl pre-warmed HPLC water directly applied to membrane,   incubation for 2 min at RT, centrifugation
-          purified PCR products stored at – 20 °C in red box
Fold ↑
Preparation of c-di-AMP supernatant   without AB
Preparation of c-di-AMP supernatant   without AB
·         168   innoculation of 1L CSE to an OD600 = 0.1 at 9.00h (used preculture   that grew over night [50 ml CSE at ])
11.00h: 0.384
12.00h: 0.69
12.20h: 0.82
12.50h: 1.035
·         centrifuge   (5000 rpm, 10 min), cook for 30 min (preheat in microwave), centrifuge (5000   rpm, 5 min), sterile filtration
·         store   in fridge
Microarray in Groningen
Fold ↑
Repeat: Pre-experiment: Overexpression of   cdaA and DAC domain by IPTG induction
Repeat: Pre-experiment: Overexpression of   cdaA and DAC domain by IPTG induction
- take over night cultures of #811, 814,   1011, 1012 and 1013 and measure OD600:
-inoculate afterwards 4 ml liquid media   to an OD600 = 0.1
- add IPTG [1mM] to a final concentration   of 1 μM to the liquid cultures
- incubate at 16°C for 12 – 14 h
 
Fold ↑
10th
Cloning DarR ORF from g-DNA
Colonies on the plates: parts 8, 7 C1
10 ml   LB medium with 10 µl antibiotics, overnight culture   for mini-prep
backup   plates: C1, C2, C3 for part 8
 
Colonies on the plates: parts 8, 7 C1
Preparation   of 250 ml LB broth media (stored on shelf above bench)
10 ml   LB medium with 10 µl antibiotics, overnight culture   for mini-prep
backup   plates: C1, C2, C3 for part 8
Re-streak of E.coli clones from prtas 6 and 7 on new LBChloram plates
Preparation of new LBChloram plates (500 ml) and Preparation of 250 ml LB broth media (stored on shelf   above bench)
 
PCR with DarR primers with different   enzymes and different buffers
-          dilution of primer stocks 1:20 (100 pmol à 5 pmol):
95 µl HPLC water + 5 µl primer   100 pmol
-          preparation of dNTP mix à dilute stocks 1:8 (100 mM à 12.5 mM):
50 µl of dATP 100 mM
+ 50 µl of dGTP 100 mM
+ 50 µl of dTTP 100 mM
+ 50 µl of dCTP 100 mM
+ 200 µl dH2O
è     diluted primers and dNTP mix   stored in red box at -20 °C
 
1x   reaction (50 µl)
Component
Volume
Buffer (5x GC buffer or 5x HF buffer)
10 µl
dNTP mix (12.5 mM each)
2 µl
Primer fwd iGEM_34 (5 pmol)
2 µl
Primer rev iGEM_35 (5 pmol)
2 µl
Chromosomal DNA M. smegmatis
2 µl
DNA-Polymerase (Phusion or PhuS) or dH2O for water control
1 µl
dH2O
31 µl
 
-          preparation of master mix for 6 reactions containing
Component
Volume
dNTP mix (12.5 mM each)
12 µl
Primer fwd iGEM_34 (5     pmol)
12 µl
Primer rev iGEM_35 (5     pmol)
12 µl
dH2O
186 µl
-          addition of template (chrom. DNA/dH2O), polymerase and   buffer individually, then addition of 37 µl master mix
è     For tested combinations of buffer and Pol: see table for gel   loading
 
-          PCR protocol (cycler 7; folder “Katrin” > “iGEM” > “DarR seq”)
Step
Temperature
Time
Initial denaturation
98.5 °C
5 min
Denaturation
98.5 °C
30 s
Annealing
60 °C
(TA = TM (≈66 °C) – 6 °C)
35 s
Elongation
72 °C
2 min (Phu
 needs more time than Phusion!)
Final elongation
72 °C
10 min
Hold
15 °C
∞
-          1 % Agarose-1xTAE gel
-          4 µl PCR reaction + 1 µl 5x DNA Loading Dye
-          3 µl 1 kb ladder (Quick Load)
-          Run at 100 V in 1xTAE buffer
-          Staining in EtBr and destaining in water
-          UV detection
M
1
2
3
4
5
M
1 kb ladder
(QuickLoad)
HF
GC
HF
GC
Water controlHF
1 kb ladder
(QuickLoad)
 
Phusion
PhuS
 
è     Reactions stored at - 20°C in 50   ml Falcon
Fold ↑
Pre-experiment: Overexpression of cdaA and DAC domain by IPTG induction
Pre-experiment: Overexpression of cdaA   and DAC domain by IPTG induction
- take over night cultures of #811, 814,   1011, 1012 and 1013 and measure OD600:
-inoculate afterwards 4 ml liquid media   to an OD600 = 0.1
Culture
OD600
Used μl for inoculation
#811
0.67
600
#814
1.39
290
#1011
1.29
310
#1012
1.18
340
#1013
0.95
420
control
1.54
220
- add IPTG [1mM] to a final concentration   of 1 μM to the liquid cultures for
induction - incubate at 16°C for 0 – 2 h   - measure OD600 of induced cultures → take equivalent amount of cells after 0   h, 1 h and 2 h: 100 [μl] /OD600
Culture
OD600
Saved cells [μl] at t0
#811
0.60
167
#814
0.64
167
#1013
0.87
115
-pelletize cells and store at -20?C
 
 
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w2 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
