"
Page
Discussion
View source
History
teams
Log in
 
Team:Goettingen/NoteBook w0
From 2013.igem.org
31st
Selection of clones from transformation of pGP172 + cdaA (L. monocytogenes) and pGP172 + DAC domain (L.monocytogenes)
Selection of clones from transformation   of pGP172 + cdaA (L. monocytogenes) and pGP172 + DAC domain (L.   monocytogenes)
pGP172 + cdaA
1 colony on 100 μl plate 14 clones on   concentrated plate
→ it may be presummed that the ful length   protein is toxic!
→ 10 clones were selected named: #81 -   #90
pGP172 + DAC domain
many clones on 100 μl plate many clones   on concentrated plate
→ re-streak them on LB Amp100 plates and   incubate for 5 h at 37°C; store them for later use at 4°C
→ inoculate 4 ml LB Amp100 liquid media   over night at 37°C and spread them afterwards on LB Amp100 plates
→ 10 clones were selected and named:   #101- #110
 
Cololny PCR of selected clones for pGP172   + cdaA (L. monocytogenes) and pGP172 + DAC domain (L. monocytogenes)
The selected clones were analyzed by   colony PCR
Components
Volume(μl)
Forward primer (T7 promoter) [5μM]
1
Reverse primer (iGEM 9) [5μM]
2
dNTPs [12.5 μM]
1
10x Taq buffer
2.5
Taq polymerase
0.5
dd H2O
18
Total
25
 
Fold ↑
30th
Transformation of pGP172 + cdaA (L.monocytogenes) and pGP172 + DAC domain (L. monocytogenes)
Transformation of pGP172 + cdaA (L.   monocytogenes) and pGP172 + DAC domain (L. monocytogenes)
Protocol:
- thaw 200 μl of competent E. coli cells   on ice
- add ligation mixture to the cells and   mix carefully
- incubate the transformation mixture for   30 min on ice
- transfer the samples for 90 sec to 42°C   (heat shock)
- replace samples for 5 min to ice and ad   800 μl liquid LB medium incubate transformation mixture for 1 h at 37°C and   agitation
-add 100 μl of the mixure onto LB Amp100   media plates
- centrifuge the samples for 1 min at   13,000 rpm, discard the upper supernatant ans resuspend cells in the   remaining 100 μl
-plate concentrated cells as well on LB   Amp100 medium plates
- incubate plates overnight at 37°C -   store plates afterwards at 4°C
 
Fold ↑
29th
Prepare of agar plates.
Transformation of GP 991 with chrom. DNA. (cdaA::cat)
 
 
Creation of Agar plates
-          Inoculation   of 2 x 250 ml CSE – Glc medium with the over night culture to an OD600 = 0.1
-          Harvest   at OD600 = 1 (takes about 4 hours)
(centrifuge for 10 min at 5000 rpm)
-          Keep   the supernatant (contains c-di-AMP)
-          1 x   250 ml
cook for 30 min on Bunsen burner in falcon tubes, centrifuge,  subsequent sterile filtration
-          1 x   250 ml
sterile filtration
-          Adding   of 25 ml supernatant and 25 ml 2 x LB into a 50 ml Falcon
(4 Falcons for the cooked and filtrated medium; 4 Falcons for the filtrated   medium)
-          Start   by preparing each Falcon with 25 ml of the appropriate supernatant and the   appropriate antibiotics
(in this case we need E/L, tet and cat; keep in mind   that you need to supply your supernatant with 2 x antibiotic, because it gets   diluted with LB later)
-          Now   add 25 ml 2 x LB to each falcon. Mix the complete medium quickly and poor the   plates!
(50 ml medium is needed to create 2 Agar Plates)
-          There   should be 6 plates containing the cooked and filtrated supernatant and 6   plates containing only the filtrated supernatant (and 2 extra plates each)
-          Furthermore,   prepare 2 plates, which contain no supernatant.
these plates will be needed for the pos. and neg. control
Transformation of GP 991 with   chrom. DNA (cdaA::cat)
-          Inoculation   of MNGE Medium (+ antibiotics) with the over night culture of GP 991 to an OD600 = 0.1
-          Proceeded according to protocol of the AG Stülke
o   Pos. control: another cat   resistance cassette (BP121)
o   Neg. control: H2O
o   Actual experiment: GP 991   + GP 997
§  500 ng/μl used
-          plated   as shown below:
 
Fold ↑
Gel electrophoresis of digested cdaA (L.   monocytogenes) and DAC domain (L. monocytogenes) (from 28.05.13)
Gel electrophoresis of digested cdaA (L.   monocytogenes) and DAC domain (L. monocytogenes) (from 28.05.13)
Gel: Marker (100 bp) | undigested cdaA |   double digested cdaA | undigested DAC domain | digested DAC domain | Marker   (1 kb ladder) The bands of the digested sequences seem to have the expected   sizes as there is no difference between the digested and the undigested DNA.
Ligation of cdaA (L. monocytogenes)   and DAC domain (L. monocytogenes) into pGP172
Components
pGP172 + cdaA
pGP172 + DAC
Control
pGP172
1.7
1.7
1.7
cdaA
1.1
-
-
DAC domain 
-
1.1
-
T4 liase buffer 
1
1
1
T4 ligase
1
1
1
HPLC H2O 
5.2
5.2
6.3
Total
10
10
10
 Ligation was performed overnight at   16.
 
Fold ↑
28th
Antibiotics and media preperation.
Preparation of cultures and medium for c-di-AMP uptake experiment.
Antibiotics
 Antibiotic
Concentration
 solve in:
Kanamycin
10 μg/ml
H2O
Lincomycin
25 μg/ml
H2O
Spectinomycin
150 μg/ml
H2O
Erythromycin
2 μg/ml
70% EtOH
Tetracyclin
needs to be stored in the dark
12.5 μg/ml
70% EtOH
Chloramphenicol
5 μg/ml
70% EtOH
Media
1 x CSE - 0.5% Glc
20 ml
 5 x C-Salts
1 ml
 Tryptophan (5 mg/ml)
1 ml
 CAF (2.2 mg/ml)
1 ml
 III'-Salts
2 ml
 Kaliumglutamat (40%)
2 ml
 Nattriumsuccinat (30%)
1 ml
 Glucose (50%)
ad 100 ml
dH2O
1 x LB
10 g
Trypton
5 g
Yeastextract
10 g
NaCl
1,5 %
Agar
ad 1000 ml
H2O
 
 
 
Preparation of cultures and medium
-          Bacillus   subtilis wt 168 inoculation in 4 ml LB over day (28 °C and 200 rpm)
-          Over   day 168 LB culture transferred to 2 x 50 ml CSE – 0.5% Glc medium
(50 μl of   over day culture used; 30 °C and 200 rpm over night)
-          This   strain is used for the production of c-di-AMP
-          Prepare   2 x 250 ml CSE – Glc medium and store these over night at 37 °C
-          Inoculation   of GP 991 in 4 ml LB – L/B, tet
o   This   strain is used for transformation
 
Fold ↑
PCR for amplification of cdaA full length proteind and DAC domain from L. monocytogenes and Gel Extraction and purification of PCR products
Digestion of cdaA from L. monocytogenes and DAC domain from L .monocytogenes and preparation for sequencing
PCR for amplification of cdaA full length proteind and   DAC domain from L. monocytogenes and Gel Extraction and purification of PCR   products
PCR   of full length protein:                           PCR of DAC domain:
Components
Volume (μl)
Components
Volume (μl)
forward primer (iGEM 8) [5 μM]
4
forward primer (iGEM 8) [5 μM]
4
reverse primer (iGEM 9) [5 μM]
4
reverse primer (iGEM 9) [5 μM]
4
DNTPs [12.5 mM]
2
DNTPs [12.5 mM]
2
DNA
2
DNA
2
5x HF buffer
10
5x HF buffer
10
HPLC water
27
HPLC water
27
Phusion polymerase
1
Phusion polymerase
1
Total
50
Total
50
4   μl of each primer were used instead of 2 μl to avoid the high selfannealing
rate!
Melting   temperature: 60°C
Annealing   Temperature: 54°C
Extension   time: 2:30 min
The   obtained PCR products were analyszed on 1% agarose gel.
Gel:
Marker   (100 bp ladder) | full length cdaA | full length cdaA | DAC domain |
DAC   domain | supposed control
Expected size for the PCR product of the full length     protein: ~ 900 bp
Expected size for the PCR product of the DAC domain:     ~ 600 bp
The supposed control should only contain the primers,     but as a band could
be detected, it has to contain as well genomic DNA     from L.
monocytogenes!
→ The     gel pieces for the bands of the full length protein and the DAC domain were     excised and the PCR products extracted and purified by adding the 3x volume     of QG buffer and following the Quiagen PCR purification kit according to     the manufacturer's protocoll.
→ The     concentration was measured by NanoDrop:
cdaA (L.     monocytogenes): 81.0 ng/μl
DAC domain (L.     monocytogenes): 111.4 ng/μl
 
 
Digestion of cdaA from L. monocytogenes and DAC domain   from L .monocytogenes and preparation for sequencing
Components
Volume (μl)
1500 ng insert
18.5 /13.5
SacI
4
BamHI
4
10x Fast Digest Buuffer
4
HPLC water
9.5/14/5
Total
40
Incubation   for 30 min at 37°C
Preparation   for Sequencing: Volume (μl)
Components
Sau_iGEM4
Sau_iGEM5
Spneu_iGEM 6
Spneu_iGEM 7
DNA
10
10
10
10
Forward primer
4
-
4
-
Reverse primer
-
4
-
4
Total
14
14
14
14
Sent   for Sequencing!
→   sequencing was successful!
 
Fold ↑
27th
Restriction digestion of pGP172 with BamHI to check   BamHI activity
Gel extraction and purification of pGP172 digested with   BamHI and SacI
PCR for amplification of cdaA from S. aureus and S. pneumoniae
Restriction digestion of pGP172 with BamHI to check   BamHI activity
Gel extraction and purification of pGP172 digested with   BamHI and SacI
concentration   of pGP172: 26.6 ng/μl
Components
Volume (μl)
Plasmid
20
BamHI
4
Fast Digest ? buffer 4
4
HPLC water
12
Total
40
The   obtained PCR products were analyszed on 1% agarose gel.
Gel:
Marker (100 bp ladder) |
undigested plasmid |
BamHI digested plasmid |
BamHI + SacI digested plasmid
 
 
For   the undigested plasmid could be several bands detetced that represent the   different states of the plasmid: linear, circular and super coiled. For the   only BamHI digested plamid could be several bands but one main band be   detected and for the double digested plasmid could be one band detecetd.
→   BamHI shows high activity, therefore the samples should be digested for 30   min
→   If you have a circular plasmid and need a BamHI and a SacI digestion, you   should start with SacI to get a linear plasmid, then heat inactivate the   enzyme and continue then with the BamHI digestion. This order is very   important, because BamHI tends to cut non-specific regions in circular   plasmids.
→   The gel piece for the band of the double digestes pGP172 was excised and the   plasmid extracted and purified by adding the 3x volume of QG buffer and   following the Quiagen PCR purification kit according to the manufacturer's   protocoll.
→   The concentration was measured by NanoDrop: 17.8 ng/μl
 
PCR for amplification of cdaA from S. aureus and S. pneumoniae
S. aureus:
Components
Volume (μl)
Melting temperature: 58°C
Annealing Temperature: 52°C
Extension time: 2:30 min
Expected product size: ~ 1.3 kb
 
forward primer (iGEM 4) [5 μM]
2
reverse primer (iGEM5)[5 μM]
2
DNTPs [12.5 mM]
2
DNA
2
5x HF buffer
10
HPLC water
31
Phusion polymerase
1
Total
50
S. pneumoniae:
Components
Volume (μl)
Melting temperature: 58°C
Annealing Temperature: 52°C
Extension time: 2:30 min
Expected product size: ~ 1.3 kb
 
forward primer (iGEM 6) [5 μM]
2
reverse primer (iGEM 7) [5 μM]
2
DNTPs [12.5 mM]
2
DNA
2
5x HF buffer
10
HPLC water
31
Phusion polymerase
1
Total
50
 
Expected   product size: ~ 1.3 kb
The   obtained PCR products were analyszed on 1% agarose gel.
Gel:   PCR product S.a. | PCR product S.a. | PCR product S.p. | PCR product S.p. |   control (both primers only) | Marker (1 kb ladder)
Expected sizes for cdA of S. aureus und cdaA of S.     pneumoniae: ~ 1.3 kb
→ For     both PCR products the expected bands of about 1.3 kb could be observedand     could be used for further experiments.
→ The     gel pieces for the bands of the cdaA of S. aureus and S.pneumoniae were excised and the PCR product extracted and purified by adding the 3x     volume of QG buffer and following the Quiagen PCR purification kit     according to the manufacturer's protocoll.
→ The concentration was measured by NanoDrop:
cdaA (S. aureus):     138.4 ng/μl
cdaA (S.     pneumoniae): 138.0 ng/μl
 
Fold ↑
Retrieved from " http://2013.igem.org/Team:Goettingen/NoteBook_w0 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
