<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Goettingen_NoteBook_w9"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Goettingen/NoteBook_w9">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Goettingen/NoteBook_w9&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w9&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w9&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Goettingen/NoteBook_w9" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Goettingen/NoteBook w9</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="tlob" id="tl_0802"><SPAN class="date" style="left:-5px">02nd</SPAN><DIV class="cont" style="left:-10px"><P class="timeline-title goe-rt">Colony PCR from 1.8.13 (pBluescript cloning of DarR and Riboswitches): Gel run + MiniPrep,  
Transformation (pSB1C3 cloning) from 31.7.13 and Comp.Cells-Test, Ligation of pSB1C3 with Riboswitch inserts.</P><DIV class="timeline-cont"><P class="c9-2"><SPAN class="c9-5">Colony PCR from 1.8.13 (pBluescript cloning of DarR and Riboswitches): Gel run + MiniPrep</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Backup plate for pBluescript clones 1 – 3 and Master Plate for Colony PCR stored at 4 °C (big fridge)</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1 % agarose-1xTAE gel (3 gels)</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Addition of 5 µl 5x LD to 25 µl PCR reaction</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Loading of 5 µl of each reaction on agarose gel</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Loading of 3 µl of 2 log ladder</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Run at (70 - ) 100 V</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>EtBr staining + destaining in water</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>UV detection</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Reactions stored at 4 °C in yellow-tip-box rag (small fridge)</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>Gel 1</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>pBluescript plasmid gave band at ca. 200 bp -&gt; a band at the PCR product sizes of DarR and riboswitches + 300 bp is expected, if DarR and riboswitches are ligated into pBluescript</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>pBluescript C1 seems to contain pBluescript plasmid (band at ca. 200 bp)</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>No inserts in all Riboswitch clones or DarR clones, except for DarR C1 and C4. However, the PCR products are between 2 and 3 kb. This is far too large for the expected product size of &gt; 1 kb -&gt; probably 1 or more inserts ligated into vector?</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Some of the clones obtained from the transformation of the ligation reactions seem to contain normal pBluescript -&gt; possibly, they were light blue or not yet blue when they were picked (band at ca. 200 bp; e.g. for C3 of w/o insert control) (when looking at all pBluescript plates again, I noticed, that we counted several blue ones as white clones… maybe it’s better, to put the plates for 1 ON to 4 °C before counting and picking…</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>There were many, many blue clones on the plates indicating that pBluescript vector was not digested properly (DAC team did RD of pBluescript possibly in a double digest…) -&gt; repeat digest in </SPAN><SPAN class="c9-5 c9-38">two rounds of single digest: 1. EcoRI; 2. PstI.</SPAN><SPAN class="c9-38"> This order is important, since pBluescript contains two PstI restriction sites. One is ca. 340 bp upstream of the EcoRI restriction site, the other one just 2 bp. Thus, in a double digest, the enzymes might inhibit each other at this second PstI site (EcoRI would not cut properly). Additionally, if one would digest first using PstI, there would be just a 2 bp overhang left at the EcoRI site. But restriction enzymes usually need a longer overhang for proper binding and cleaving of the DNA.</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN>Cloning did not work again.</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>All clones harvested for the MiniPrep were thrown away, except for pBluescript C1 and DarR C1 and C4 (we want to know what’s inside…)</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>When discussing the gels, Katrin and I noticed that there might be a too short overhang after the </SPAN><SPAN class="c9-45">Eco</SPAN><SPAN>RI and </SPAN><SPAN class="c9-45">Pst</SPAN><SPAN>I restriction sites in the prefix and suffix sites for the enzymes to cut -&gt; Insert is not restricted at the ends -&gt; no ligation can occur with the properly digested vector. We checked it, and, indeed, the restriction sites were directly at the ends of the primers used for amplification of the inserts (no overhang at all!). On the iGEM web page, there are the actual Prefix and Suffix sequences with the overhangs preferred by </SPAN><SPAN class="c9-45">Eco</SPAN><SPAN>RI and </SPAN><SPAN class="c9-45">Pst</SPAN><SPAN>I. But they were not entered into the “Primers List”. Instead, the pure Suffix and Prefix sites were entered -&gt; This means, that all our primers contain the wrong prefix and suffix sites. So, we have order new primers containing the overhangs or to try amplify the inserts with </SPAN><SPAN class="c9-45">Taq</SPAN><SPAN> (this enzyme adds polyA to the ends of the PCR products, thereby generating an overhang).</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Transformation (pSB1C3 cloning) from 31.7.13 and Comp.Cells-Test</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Transformation of pSB1C3 cloning:</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>colonies only on some plates</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>However, they look more like a contamination than like </SPAN><SPAN class="c9-45">E. coli</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Negative control: no contamination or </SPAN><SPAN class="c9-45">E. coli</SPAN><SPAN> colonies</SPAN></P><P class="c9-1"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN>Plates thrown away</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Comp. Cells test:</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>LB</SPAN><SPAN class="c9-7">Amp</SPAN><SPAN> , LB</SPAN><SPAN class="c9-7">Cm</SPAN><SPAN> , LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN><SPAN> : No colonies of </SPAN><SPAN class="c9-45">E. coli</SPAN><SPAN> on plates, but contamination on one of the plates (possibly Amp plate?) -&gt; untransformed </SPAN><SPAN class="c9-45">E. coli</SPAN><SPAN> cannot grow on the plates</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Rest plate of pBluescript transformation: Colonies became more blue, white big colonies in between indicating contamination (looks like contamination on antibiotic resistance test plate…)</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Competent cells are fine.</SPAN></P><P class="c9-2"><SPAN class="c9-5">Contamination source:</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>The contamination on the antibiotics plate was only observed today (but not yesterday, at least on the antibiotics test plate (for pBluescript rest, it could be that there was already yesterday something…).</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>It was likewise not seen in other transformations done before with the same comp. cells, except for the transformation done one week ago by Jonathan, Dominik and Bingyao.</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>It was also not found on the 50 µl plate of the pBluescript transformation or in the negative controls.</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>The contamination is also seen on the pSB1C3 transformation, but only on few plates.</SPAN></P><P class="c9-2"><SPAN>-&gt; Hence, it could be that the contamination derives from some other source than the comp. cells themselves i.e. unsterile pipette tips and glass pipettes or even the paper box in which the plates were incubated. Thus, the whole sterile bench was cleaned and all tips and glass pipettes removed.</SPAN></P><P class="c9-2"><SPAN class="c9-5">Taq polymerase PCR of Riboswitches</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Used primer combinations and programs: 42/43 (Ribo1 Taq); 42/41 (Ribo2 Taq); 40/41 (Ribo3 Taq); 40/43 (Ribo4 Taq)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Standard Taq Polymerase PCR Protocol used, except for double amount of primer for primer 43</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Template: gDNA of B. subitilis</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>each setup 5 times (1 MM for each setup)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Colorcodes of the PCR tubes:</SPAN></P><P class="c9-2 c9-37"><SPAN>o</SPAN><SPAN>  </SPAN><SPAN>Ribo 1: WHITE</SPAN></P><P class="c9-2 c9-37"><SPAN>o</SPAN><SPAN>  </SPAN><SPAN>Ribo2: PURPLE</SPAN></P><P class="c9-2 c9-37"><SPAN>o</SPAN><SPAN>  </SPAN><SPAN>Ribo3: ORANGE</SPAN></P><P class="c9-2 c9-37"><SPAN>o</SPAN><SPAN>  </SPAN><SPAN>Ribo4: YELLOW</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">PolyA overhang of DarR</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Setup: 150ng of template</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>0,2 mM dATP</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1x Taq-Buffer</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1 U Taq polymerase</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>30 min in Cycler at 70°C</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Template: DarR PCR Purified 2 24.7 by Joni (17ng/ µl)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>9 µl Template</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1 µl 0,2 mM dATP</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1 µl 10x Taq Buffer</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1 µl Taq Pol</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>pooled and put into freezer by Katrin G..</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Miniprep of DarR C1, DarR C4 and pBluescript plasmids</SPAN></P><P class="c9-2"><SPAN class="c9-5">NanoDrop values:</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN class="c9-5">Sample</SPAN></P></TD><TD class="c9-70"><P class="c9-2 c9-8"><SPAN class="c9-5">ng/µl</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">280nm</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">230nm</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>pblue</SPAN></P></TD><TD class="c9-70"><P class="c9-2 c9-8"><SPAN>342,0</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN>1,89</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN>2,26</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>DarR 1</SPAN></P></TD><TD class="c9-70"><P class="c9-2 c9-8"><SPAN>402,1</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN>1,87</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN>2,24</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>DarR 4</SPAN></P></TD><TD class="c9-70"><P class="c9-2 c9-8"><SPAN>401,3</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN>1,87</SPAN></P></TD><TD class="c9-0"><P class="c9-2 c9-8"><SPAN>2,24</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Restriction digestion of pbluescript</SPAN></P><P class="c9-2"><SPAN>1500ng (4,4 µl) plasmid</SPAN></P><P class="c9-2"><SPAN>4 µl EcoRI</SPAN></P><P class="c9-2"><SPAN>4 µl Buffer</SPAN></P><P class="c9-2"><SPAN>28µl dH</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O</SPAN></P><P class="c9-2"><SPAN>digested for 1h at 37°C</SPAN></P><P class="c9-2"><SPAN class="c9-5">Test restriction digestion of DarR C1 and DarR C4 (respectively)</SPAN></P><P class="c9-2"><SPAN>300ng (1µl) plasmid</SPAN></P><P class="c9-2"><SPAN>4 µl EcoRI</SPAN></P><P class="c9-2"><SPAN>4 µl PstI</SPAN></P><P class="c9-2"><SPAN>4 µl Buffer</SPAN></P><P class="c9-2"><SPAN>27 µl dH</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O</SPAN></P><P class="c9-2"><SPAN>digested for 1h at 37°C</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN>Gel doc:</SPAN></P><P class="c9-2"><SPAN>pblue, DarR1, DarR4</SPAN></P><P class="c9-20 c9-21"><SPAN>pblue is not completely linearized -&gt; has to be further digested on monday (is stored in to do box)</SPAN></P><P class="c9-20 c9-21"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Actually, one cannot conclude anything from this gel, since one does not know at which height the different forms of the uncut plasmid are running. </SPAN><SPAN class="c9-39">Remember: ALWAYS load </SPAN><SPAN class="c9-60 c9-39">30 ng or more </SPAN><SPAN class="c9-39">(since after MiniPrep we usually have 100 ng or more of plasmid, it’s enough to load 1 µl. But make first sure, if this is really the case!) of </SPAN><SPAN class="c9-60 c9-39">uncut vector as a control.</SPAN></P><P class="c9-2"><SPAN class="c9-5">Repeat of gel run for pBluescript and DarR C1 and C4 plasmid RD and gel for riboswitch PCR </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1.5% agarose-1xTAE-gel (1.5 g agarose in 100 ml 1xTAE)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Loading of 1 µl pBluescript plasmid/DarR C1/DarR C4 plasmid + 3 µl dH</SPAN><SPAN>₂</SPAN><SPAN>O + 1 µl 5xLD</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Loading of 4 µl RD reaction/PCR reaction (after pooling them) + 1 µl 5xLD</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Loading of 3 µl 2 log ladder</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Run at 90 V</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>EtBr staining + destaining in water</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>UV detection</SPAN></P><P class="c9-20 c9-21"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Gel:</SPAN></P>&lt;<P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>pBluescript digest is incomplete -&gt; further digest</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>DarR C1 and C4 plasmids seem to be fully digested -&gt; C1 insert is slightly smaller than the vector: no self-ligation, but possibly multiple inserts?; C4 insert has same bands as pBluescript RD, though bands of uncut C4 plasmid has different bands: No explanation…? Samples switched???</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>PCR of riboswitches: the expected bands were more or less (maybe slightly too high?) observed -&gt; purification and digest to generate inserts</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Purification and Digest (EcoRI and PstI) of riboswitch PCR products to generate inserts </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Purification using Qiagen PCR clean-up kit (ca. 250 µl of PCR reaction -&gt; addition of 1.250 ml PB buffer)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Elution with pre-warmed HPLC water (1</SPAN><SPAN class="c9-7">st</SPAN><SPAN> elution: 40 µl, 2</SPAN><SPAN class="c9-7">nd</SPAN><SPAN> elution (eluted into </SPAN><SPAN class="c9-60">same</SPAN><SPAN> tube already containing the 40 µl 1</SPAN><SPAN class="c9-7">st</SPAN><SPAN> elution!): 30 µl)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Concentration (NanoDrop) </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN class="c9-5">Purified PCR…</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN class="c9-5">ng/µl</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">280nm</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">230nm</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_40/41</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>38.2</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.81</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>1.99</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_40/43</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>66.7</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.85</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>2.23</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_42/41</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40.2</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.78</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>2.03</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_42/43</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>58.6</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.83</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>2.15</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-1"><SPAN> </SPAN></P><P class="c9-1"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Digest (E+P)</SPAN></P><P class="c9-20 c9-21"><SPAN>All reactions were pipetted independently (no MasterMix)</SPAN><SPAN class="c9-5"> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-25"><P class="c9-20 c9-56"><SPAN class="c9-5">Component</SPAN></P></TD><TD class="c9-27"><P class="c9-3"><SPAN class="c9-5">Amount</SPAN></P></TD></TR><TR><TD class="c9-25"><P class="c9-20 c9-56"><SPAN class="c9-5">PCR product</SPAN></P></TD><TD class="c9-27"><P class="c9-3"><SPAN>30 µl</SPAN></P></TD></TR><TR><TD class="c9-25"><P class="c9-20 c9-56"><SPAN class="c9-5">EcoRI FD</SPAN></P></TD><TD class="c9-27"><P class="c9-3"><SPAN>3 µl</SPAN></P></TD></TR><TR><TD class="c9-25"><P class="c9-20 c9-56"><SPAN class="c9-5">PstI FD</SPAN></P></TD><TD class="c9-27"><P class="c9-3"><SPAN>3 µl</SPAN></P></TD></TR><TR><TD class="c9-25"><P class="c9-20 c9-56"><SPAN class="c9-5">10x FD buffer</SPAN></P></TD><TD class="c9-27"><P class="c9-3"><SPAN>4 µl</SPAN></P></TD></TR><TR><TD class="c9-25"><P class="c9-20 c9-56"><SPAN class="c9-5">Total</SPAN></P></TD><TD class="c9-27"><P class="c9-3"><SPAN>40 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-20 c9-21"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>1 h, 37 °C</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>   </SPAN><SPAN>Gel run (1 % agarose-1xTAE)</SPAN></P><P class="c9-20 c9-21"><SPAN>Loading of 3 µl 2 log ladder</SPAN></P><P class="c9-20 c9-21"><SPAN>Loading of 1 µl uncut purified PCR product + 3 µl dH</SPAN><SPAN>₂</SPAN><SPAN>O + 1 µl 5xLD</SPAN></P><P class="c9-20 c9-21"><SPAN>Loading of 4 µl RD reaction + 1 µl 5xLD</SPAN></P><P class="c9-20 c9-21"><SPAN>Run at 100 V</SPAN></P><P class="c9-20 c9-21"><SPAN>EtBr staining + destaining in water</SPAN></P><P class="c9-20 c9-21"><SPAN>UV detection</SPAN></P><P class="c9-20 c9-21"><SPAN> </SPAN></P><P class="c9-20 c9-21"><SPAN>Gel:</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Inserts still present at expected bp -&gt; purification and ligation with pSB1C3 (E+P digested)</SPAN></P><P class="c9-1"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Purification of reactions using Qiagen PCR clean-up kit (500 µl PB buffer used; elution with 2x 22 µl HPLC H</SPAN><SPAN>₂</SPAN><SPAN>O into the same tube)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Concentration (NanoDrop) </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN class="c9-5">Insert…</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN class="c9-5">ng/µl</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">280nm</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">230nm</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_40/41</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>20.9</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.91</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>1.82</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_40/43</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>36.1</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.81</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>2.05</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_42/41</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>23.6</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.83</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>1.63</SPAN></P></TD></TR><TR><TD class="c9-44"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_42/43</SPAN></P></TD><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>32.5</SPAN></P></TD><TD class="c9-9"><P class="c9-2 c9-8"><SPAN>1.80</SPAN></P></TD><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>2.02</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-1"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Ligation of pSB1C3 with Riboswitch inserts </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Ratio Vector : insert = 1:3 (calculated with Uni D’dorf online ligation calculator)</SPAN></P><P class="c9-20 c9-21"><SPAN>50 ng vector, 2070 bo </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-28"><P class="c9-2 c9-8"><SPAN class="c9-5">Insert…</SPAN></P></TD><TD class="c9-67"><P class="c9-6 c9-8"><SPAN class="c9-5">bp used for calculation</SPAN></P></TD><TD class="c9-36"><P class="c9-16 c9-8"><SPAN class="c9-5">Amount for ligation with 50 ng pSB1C3 vector</SPAN></P></TD><TD class="c9-53"><P class="c9-2 c9-8"><SPAN class="c9-5">ng/µl</SPAN></P></TD></TR><TR><TD class="c9-28"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_40/41</SPAN></P></TD><TD class="c9-67"><P class="c9-6 c9-8"><SPAN>213</SPAN></P></TD><TD class="c9-36"><P class="c9-8 c9-16"><SPAN>15 ng</SPAN></P></TD><TD class="c9-53"><P class="c9-2 c9-8"><SPAN>20.9</SPAN></P></TD></TR><TR><TD class="c9-28"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_40/43</SPAN></P></TD><TD class="c9-67"><P class="c9-6 c9-8"><SPAN>403</SPAN></P></TD><TD class="c9-36"><P class="c9-16 c9-8"><SPAN>29 ng</SPAN></P></TD><TD class="c9-53"><P class="c9-2 c9-8"><SPAN>36.1</SPAN></P></TD></TR><TR><TD class="c9-28"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_42/41</SPAN></P></TD><TD class="c9-67"><P class="c9-6 c9-8"><SPAN>370</SPAN></P></TD><TD class="c9-36"><P class="c9-16 c9-8"><SPAN>27 ng</SPAN></P></TD><TD class="c9-53"><P class="c9-2 c9-8"><SPAN>23.6</SPAN></P></TD></TR><TR><TD class="c9-28"><P class="c9-2 c9-8"><SPAN>Riboswitch iGEM_42/43</SPAN></P></TD><TD class="c9-67"><P class="c9-6 c9-8"><SPAN>466</SPAN></P></TD><TD class="c9-36"><P class="c9-16 c9-8"><SPAN>34 ng</SPAN></P></TD><TD class="c9-53"><P class="c9-2 c9-8"><SPAN>32.5</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-20 c9-21"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN class="c9-5">For each reaction, 1 µl insert and 1 µl pSB1C3 E+P vector (56.6 ng/µl) were used:</SPAN><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Reaction</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">Component</SPAN></P></TD><TD class="c9-51"><P class="c9-35 c9-8 c9-20"><SPAN class="c9-5">1x</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN class="c9-5">MasterMix 6x</SPAN></P></TD></TR><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">pSB1C3</SPAN></P></TD><TD class="c9-51"><P class="c9-35 c9-8 c9-20"><SPAN>1µl</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN>6 µl</SPAN></P></TD></TR><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">T4 ligase (ThermoScientific)</SPAN></P></TD><TD class="c9-51"><P class="c9-8 c9-20 c9-35"><SPAN>1 µl</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN>6 µl</SPAN></P></TD></TR><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">T4 ligation buffer 10x</SPAN></P></TD><TD class="c9-51"><P class="c9-35 c9-8 c9-20"><SPAN>1 µl</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN>6 µl</SPAN></P></TD></TR><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">dH</SPAN><SPAN class="c9-5 c9-15">2</SPAN><SPAN class="c9-5">O</SPAN></P></TD><TD class="c9-51"><P class="c9-35 c9-8 c9-20"><SPAN>6 µl</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN>36 µl</SPAN></P></TD></TR><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">Insert</SPAN></P></TD><TD class="c9-51"><P class="c9-35 c9-8 c9-20"><SPAN>1 µl</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN>-</SPAN></P></TD></TR><TR><TD class="c9-32"><P class="c9-8 c9-20 c9-56"><SPAN class="c9-5">Total</SPAN></P></TD><TD class="c9-51"><P class="c9-35 c9-8 c9-20"><SPAN>10 µl</SPAN></P></TD><TD class="c9-31"><P class="c9-3"><SPAN>54 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>1 µl of insert or dH</SPAN><SPAN>₂</SPAN><SPAN>O (for w/o insert control) + 9 µl MasterMix</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Incubation ON at 16 °C</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0801"><SPAN class="date">01st</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Restriction digestion of these riboswitch PCR products:, Competent Cells-Test, Transformation of pBluescript and pSB1C3 ligations from 31.7.13, Colony PCR for pBluescript cloning on 31.7.13, Transformation of parts BBa_E0030 and BBa_E0020.</P><DIV class="timeline-cont"><P class="c9-2"><SPAN class="c9-5">Pooling of the PCR products from yesterdays riboswitch and new NanoDrop measurement</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>Sample</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN class="c9-5">ng/µl</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">280nm</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">230nm</SPAN></P></TD></TR><TR><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>42/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>8.9</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.46</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>0.90</SPAN></P></TD></TR><TR><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>42/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>15.1</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.52</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.23</SPAN></P></TD></TR><TR><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>40/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>11.0</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.51</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.72</SPAN></P></TD></TR><TR><TD class="c9-18"><P class="c9-2 c9-8"><SPAN>40/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>14.0</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.92</SPAN></P></TD><TD class="c9-19"><P class="c9-2 c9-8"><SPAN>1.86</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN>Gel doc:</SPAN></P><P class="c9-2"><SPAN>42/41, 42/43, 40/41, 40/43</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Restriction digestion of these riboswitch PCR products:</SPAN></P><P class="c9-2"><SPAN class="c9-5">EcoRI +PstI</SPAN></P><P class="c9-2"><SPAN>for 42/43, 40/41, 40/43</SPAN></P><P class="c9-2"><SPAN>30µl PCR product</SPAN></P><P class="c9-2"><SPAN>3µl EcoRI</SPAN></P><P class="c9-2"><SPAN>3µl PstI</SPAN></P><P class="c9-2"><SPAN>4µl FD Buffer</SPAN></P><P class="c9-2"><SPAN>7µl H</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN>for 42/41:</SPAN></P><P class="c9-2"><SPAN>37µl PCR product</SPAN></P><P class="c9-2"><SPAN>3µl EcoRI</SPAN></P><P class="c9-2"><SPAN>3µl PstI</SPAN></P><P class="c9-2"><SPAN>4µl FD Buffer</SPAN></P><P class="c9-2"><SPAN>7µl H</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">EcoRI +SpeI</SPAN></P><P class="c9-2"><SPAN>for 42/43, 40/41, 40/43</SPAN></P><P class="c9-2"><SPAN>30µl PCR product</SPAN></P><P class="c9-2"><SPAN>3µl EcoRI</SPAN></P><P class="c9-2"><SPAN>3µl PstI</SPAN></P><P class="c9-2"><SPAN>4µl FD Buffer</SPAN></P><P class="c9-2"><SPAN>7µl H</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>for 42/41:</SPAN></P><P class="c9-2"><SPAN>37µl PCR product</SPAN></P><P class="c9-2"><SPAN>3µl EcoRI</SPAN></P><P class="c9-2"><SPAN>3µl PstI</SPAN></P><P class="c9-2"><SPAN>4µl FD Buffer</SPAN></P><P class="c9-2"><SPAN>7µl H</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O</SPAN></P><P class="c9-2"><SPAN>digested for 30min at 37° </SPAN><SPAN class="c9-5 c9-39">(or 1h?)</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>Gel run:</SPAN></P><P class="c9-2"><SPAN>Pouring of 1% agarose gel</SPAN></P><P class="c9-2"><SPAN>Loading of 4 µl supplied with1 µl 5x LD of each digest</SPAN></P><P class="c9-2"><SPAN>Loading of &lt; 4 µl of the remaining uncut purified PCR products supplied with 1 µl 5xLD (just 40/41, 40/43 and 43/42 left…)</SPAN></P><P class="c9-2"><SPAN>Loading of 3 µl log ladder</SPAN></P><P class="c9-2"><SPAN>Run at 100 V</SPAN></P><P class="c9-2"><SPAN>EtBr staining and de-staining</SPAN></P><P class="c9-2"><SPAN>UV detection</SPAN></P><P class="c9-2"><SPAN>Gel:</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Strangely, the relationship between the length of the different inserts fits on their own. However, they are running at a much lower height than expected (expected 200 – 500; observed: 100 – 200…) -&gt; something strange happened during the PCR, because already on the gel to see if the PCR worked, the bands are too low… -&gt; Wait for Colony PCR (see below); if all clones negative, repeat PCR</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Reactions stored at – 20 °C in one of our pink boxes</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Preparation of LB plates</SPAN></P><P class="c9-2"><SPAN>1x 500 ml LB</SPAN><SPAN class="c9-7">Chl</SPAN></P><P class="c9-2"><SPAN class="c9-7"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Competent Cells-Test </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>No colonies on LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN><SPAN>, LB</SPAN><SPAN class="c9-7">Amp</SPAN><SPAN>, LB</SPAN><SPAN class="c9-7">Cm</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>&gt; 50 colonies on 50 µl plate of 1 ng pBluescript transformation (expected contamination was not observed); on Rest-plate: many more colonies; colonies appeared blue, but some seemed to be white (these colonies could have produced too low amounts of indigo dye – there are always differencens in the strength of the blue colour of different colonies; sometimes it’s hard to distinguish if these colonies are white or just light blue -&gt; one can put the plates to 4 °C for some time, then the blue colour of the really blue colonies gets more intense), or they contain a somehow mutated plasmid…)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Neg. control (see pBluescrpit cloning) showed no colonies</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>3 </SPAN><SPAN class="c9-38">blue</SPAN><SPAN> clones from 50 µl plate of pBluescript transformation were picked and streaked out on LB</SPAN><SPAN class="c9-7">Amp</SPAN><SPAN>; Clone 1 was inoculated in 4 ml LB</SPAN><SPAN class="c9-7">Amp</SPAN><SPAN> for MiniPrep and used as a control for Colony PCR (see below)</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Transformation of pBluescript and pSB1C3 ligations from 31.7.13</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>No colonies on neg. control plates</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Blue and white colonies on all other plates</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Counting of the white colonies on 50 µl plates for w/o-insert-control, DarR and Riboswitch transformations</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">Plate</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN class="c9-5">White colonies (approximate values)</SPAN></P></TD></TR><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">w/o insert control</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN>40</SPAN></P></TD></TR><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">DarR</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN>80</SPAN></P></TD></TR><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">Ribo 40/41</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN>80</SPAN></P></TD></TR><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">Ribo 40/43</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN>80</SPAN></P></TD></TR><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">Ribo 41/42</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN>160 -&gt; many light blue colonies, counting was quite hard, just one half of the plate was counted</SPAN></P></TD></TR><TR><TD class="c9-49"><P class="c9-6 c9-8"><SPAN class="c9-5">Ribo 42/43</SPAN></P></TD><TD class="c9-68"><P class="c9-6 c9-8"><SPAN>160-&gt; many light blue colonies, counting was quite hard, just one half of the plate was counted</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-6"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>There were more white clones on the ligation plates than on w/o insert control (white clones might be derived from relegated and mutated vector (eg. Frame shift))</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Colony PCR incl. Master Plate and inoculation of 4 ml LB</SPAN><SPAN class="c9-7">Amp</SPAN><SPAN> cultures for MiniPrep (3 white clones from w/o insert control, 5 white clones from each of the other plates)</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">pSB1C3:</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>No colonies on neg. control plates and on ligation plates</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Further incubation of all plates</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Colony PCR for pBluescript cloning on 31.7.13</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN class="c9-5">Reactions:</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> 1x pBluescript plasmid 1:10 diluted (white epi A)</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> 1x C1 from transformation of 1 ng pBluescript plasmid (see above) (white epi B)</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> (white) C1, C2 and C3 from w/o insert control plate (white epis C, D and E)</SPAN></P><P class="c9-20 c63"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> (white) C1 – C5 from DarR plate (white epis 1 – 5)</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> (white) C1 – C5 from Ribo 40/41 plate (yellow epis 1 – 5)</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> (white) C1 – C5 from Ribo 40/43 plate (orange epis 1 – 5)</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> (white) C1 – C5 from Ribo 42/41 plate (violet epis 1 – 5)</SPAN></P><P class="c63 c9-20"><SPAN>-</SPAN><SPAN>          </SPAN><SPAN> (white) C1 – C5 from Ribo 42/43 plate (blue epis 1 – 5)</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN class="c9-5">Colony PCR was performed according to protocol from 10.7.13 with some modifications:</SPAN></P><P class="c9-2"><SPAN>a) Inoculation of liquid culture for MiniPrep, as well</SPAN></P><P class="c9-20 c9-21"><SPAN>b) Primers M13-PUC-fwd and M13-PUC-rev diluted 1:20 in HPLC water and used for Colony PCR (now as well in 1:20 dilution in our box with all other primers) -&gt;</SPAN><SPAN class="c9-39">I changed the Annealing Temperature in one of the cyclers in which the protocols are saved!!! It’s 52 °C, not 54°C as required for VF2/VR PCR.</SPAN></P><P class="c9-20 c9-21"><SPAN class="c9-5"> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Inoculation/Plating was done in the following order</SPAN><SPAN>:</SPAN></P><P class="c9-2"><SPAN>1. Picking of clone and streak-out on Master Plate</SPAN></P><P class="c9-2"><SPAN>2. Inoculation of 4 ml LB</SPAN><SPAN class="c9-7">Amp </SPAN><SPAN> liquid culture ON for MiniPrep</SPAN></P><P class="c9-2"><SPAN>3. Inoculation of PCR reaction</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>Incubation of liquid cultures and plates ON at 37 °C</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>After PCR, the PCR reactions were stored at -20°C in yellow-tip-box-rags sealed with parafilm</SPAN></P><P class="c9-2"><SPAN class="c9-5">Transformation of parts BBa_E0030 and BBa_E0020</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Solved parts in 10 µl HPLC water</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Transformation of </SPAN><SPAN class="c9-45">E. coli</SPAN><SPAN> DH5</SPAN><SPAN>α</SPAN><SPAN> according to methods folder</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Plating on LB</SPAN><SPAN class="c9-7">Cm</SPAN><SPAN> plates + Incubation ON at 37 °C</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-59"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">Part</SPAN></P></TD><TD class="c9-69"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">What?</SPAN></P></TD><TD class="c9-10"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">Backbone and Resistance</SPAN></P></TD><TD class="c9-52"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">Length</SPAN></P></TD><TD class="c9-14"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">Emission</SPAN></P></TD><TD class="c9-58"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">Extinction</SPAN></P></TD><TD class="c9-61"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">Well</SPAN></P></TD></TR><TR><TD class="c9-59"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">BBa_E0030</SPAN></P></TD><TD class="c9-69"><P class="c9-8 c9-20 c48"><SPAN>EYFP; no RBS, no terminator</SPAN></P></TD><TD class="c9-10"><P class="c9-8 c9-20 c48"><SPAN>pSB1C3 -&gt; CmR</SPAN></P></TD><TD class="c9-52"><P class="c9-8 c9-20 c48"><SPAN>723 bp</SPAN></P></TD><TD class="c9-14"><P class="c9-8 c9-20 c48"><SPAN>527</SPAN></P></TD><TD class="c9-58"><P class="c9-8 c9-20 c48"><SPAN>514</SPAN></P></TD><TD class="c9-61"><P class="c9-8 c9-20 c48"><SPAN>Plate 3 16D</SPAN></P></TD></TR><TR><TD class="c9-59"><P class="c9-8 c9-20 c48"><SPAN class="c9-5">BBa_E0020</SPAN></P></TD><TD class="c9-69"><P class="c9-8 c9-20 c48"><SPAN>ECFP; no RBS, no terminator</SPAN></P></TD><TD class="c9-10"><P class="c9-8 c9-20 c48"><SPAN>pSB1C3 -&gt; CmR</SPAN></P></TD><TD class="c9-52"><P class="c9-8 c9-20 c48"><SPAN>723 bp</SPAN></P></TD><TD class="c9-14"><P class="c9-8 c9-20 c48"><SPAN>476</SPAN></P></TD><TD class="c9-58"><P class="c9-8 c9-20 c48"><SPAN>439</SPAN></P></TD><TD class="c9-61"><P class="c9-8 c9-20 c48"><SPAN>Plate 3 3M</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>For some of the Riboswitches, we need a Reporter without RBS. But The GFP of part 6 already contains a terminator and an RBS. So, I searched in the registry for a GFP reporter (or something similar) without RBS (and maybe with terminator). However, the transformation of these parts is just a try, because previous transformations of distribution kit plasmids did not work at all (including the GFP in part 6 without any regulatory sequences). If the transformation fails again, we will order the forward primer for amplifying GFP from part 6 (see below)</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Primer Design</SPAN></P><P class="c9-2"><SPAN class="c9-5"> </SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Design of forward primer for amplification of GFP from Part 6 (as a reverse primer, we can use VR. Then we directly amplify the terminator with the GFP)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>I introduced a point mutation in the primer for the third base of the second codon after the ATG (AAA -&gt; AAG; both encodes for Lys), because then, the primer won’t form secondary structures (without this mutation, it forms strong secondary structures)… but now, I’m unsure, if we can do that. Because it should actually be the same sequence we clone into our other plasmids…</SPAN></P><P class="c9-2"><SPAN>Primer not yet ordered</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="monat">July</DIV><DIV class="tlob" id="tl_0731 "><SPAN class="date">31st</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Restriction digestion of Part1,3,4 with SpeI and PstI:, New Riboswitch PCR, Transformation of ON ligations from 30.7.13 and Test of DH5α competent cells.</P><DIV class="timeline-cont"><P class="c9-2"><SPAN class="c9-5">Restriction digestion of Part1,3,4 with SpeI and PstI:</SPAN></P><P class="c9-2"><SPAN>Gel doc:</SPAN></P><P class="c9-2"><SPAN>part M/1/ 1/3/3/4/4</SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>only partial restriction -&gt; we’ll do an overnight restriction</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">New Riboswitch PCR</SPAN></P><P class="c9-2"><SPAN>(as performed on 24.6.)</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-17"><P class="c9-6 c9-8"><SPAN class="c9-5"> </SPAN></P></TD><TD class="c9-65"><P class="c9-6 c9-50"><SPAN class="c9-5">Primers</SPAN></P></TD><TD class="c9-32"><P class="c9-6 c9-8"><SPAN class="c9-5">T    </SPAN><SPAN class="c9-5 c9-15">A</SPAN><SPAN class="c9-5"> = 0.5 * </SPAN><SPAN class="c9-5">[</SPAN><SPAN class="c9-5">T</SPAN><SPAN class="c9-5 c9-15">m</SPAN><SPAN class="c9-5 c9-7">(primer1)</SPAN><SPAN class="c9-5"> + T</SPAN><SPAN class="c9-5 c9-15">m</SPAN><SPAN class="c9-5 c9-7">(Primer2)</SPAN><SPAN class="c9-5">] – 6 °C</SPAN></P></TD><TD class="c9-24"><P class="c9-6"><SPAN class="c9-5">    PCR product size</SPAN></P></TD><TD class="c9-57"><P class="c9-6"><SPAN class="c9-5"> Name of Protocol in cycler</SPAN></P></TD></TR><TR><TD class="c9-17"><P class="c9-6 c9-8"><SPAN class="c9-5">Riboswitch with native Promoter and RBS</SPAN></P></TD><TD class="c9-65"><P class="c9-6 c9-50"><SPAN>iGEM_42</SPAN></P><P class="c9-6 c9-50"><SPAN>iGEM_43</SPAN></P></TD><TD class="c9-32"><P class="c9-6 c9-8"><SPAN>51 °C</SPAN></P><P class="c9-6 c9-8"><SPAN> </SPAN></P></TD><TD class="c9-24"><P class="c9-6"><SPAN>466 bp</SPAN></P></TD><TD class="c9-57"><P class="c9-6"><SPAN>Ribo1</SPAN></P></TD></TR><TR><TD class="c9-17"><P class="c9-6 c9-8"><SPAN class="c9-5">Riboswitch with native Promoter</SPAN></P></TD><TD class="c9-65"><P class="c9-6 c9-50"><SPAN>iGEM_42</SPAN></P><P class="c9-6 c9-50"><SPAN>iGEM_41</SPAN></P></TD><TD class="c9-32"><P class="c9-6 c9-8"><SPAN>55.2°C</SPAN></P></TD><TD class="c9-24"><P class="c9-6"><SPAN>370 bp</SPAN></P></TD><TD class="c9-57"><P class="c9-6"><SPAN>Ribo2</SPAN></P></TD></TR><TR><TD class="c9-17"><P class="c9-6 c9-8"><SPAN class="c9-5">Riboswitch only</SPAN></P></TD><TD class="c9-65"><P class="c9-6 c9-50"><SPAN>iGEM_40</SPAN></P><P class="c9-6 c9-50"><SPAN>iGEM_41</SPAN></P></TD><TD class="c9-32"><P class="c9-6 c9-8"><SPAN>59.7°C</SPAN></P></TD><TD class="c9-24"><P class="c9-6"><SPAN>213 bp</SPAN></P></TD><TD class="c9-57"><P class="c9-6"><SPAN>Ribo3</SPAN></P></TD></TR><TR><TD class="c9-17"><P class="c9-6 c9-8"><SPAN class="c9-5">Riboswitch with native RBS</SPAN></P></TD><TD class="c9-65"><P class="c9-6 c9-50"><SPAN>iGEM_40</SPAN></P><P class="c9-6 c9-50"><SPAN>iGEM_43</SPAN></P></TD><TD class="c9-32"><P class="c9-6 c9-8"><SPAN>55.5 °C</SPAN></P></TD><TD class="c9-24"><P class="c9-6"><SPAN>403 bp</SPAN></P></TD><TD class="c9-57"><P class="c9-6"><SPAN>Ribo4</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-6 c9-8"><SPAN> </SPAN></P><P class="c9-2"><SPAN>T</SPAN><SPAN class="c9-15">m</SPAN><SPAN> (iGEM_40) = 65.7 °C</SPAN></P><P class="c9-2"><SPAN>T</SPAN><SPAN class="c9-15">m</SPAN><SPAN> (iGEM_41) = 65. 7 °C</SPAN></P><P class="c9-2"><SPAN>T</SPAN><SPAN class="c9-15">m</SPAN><SPAN> (iGEM_42) = 56.7 °C</SPAN></P><P class="c9-2"><SPAN>T</SPAN><SPAN class="c9-15">m</SPAN><SPAN> (iGEM_43) = 57.3 °C</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>PCR reaction and protocol as usual (50µl) (template: </SPAN><SPAN class="c9-45">B. subtilis</SPAN><SPAN> chrom.DNA)</SPAN></P><P class="c9-2"><SPAN>Concentration of primer iGEM_43 doubled (4µl per reaction)</SPAN></P><P class="c9-2"><SPAN>Reactions stored at -20°C in Falcon Tube</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN>Gel doc:</SPAN></P><P class="c9-2"><SPAN>iGEM_42/43, 42/43, 42/43; 42/41, 42/41, 42/41; 40/41, 40/41, 40/41; 40/43, 40/43, 40/43</SPAN></P>&lt;<P class="c9-2"><SPAN>after PCR purification following values with Nanodrop:</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>Sample</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN class="c9-5">ng/µl</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">280nm</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN class="c9-5">A</SPAN><SPAN class="c9-5 c9-15">260nm</SPAN><SPAN class="c9-5">/A</SPAN><SPAN class="c9-5 c9-15">230nm</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>42/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>11.9</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.81</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>2.24</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>42/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>14.1</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.85</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>2.36</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>42/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>16.2</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.73</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.11</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>42/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>9.5</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.34</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.27</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>42/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>8.2</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.71</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.81</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>42/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>8.0</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>2.23</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>2.83</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>9.9</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.81</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.80</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>12.4</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.58</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.14</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40/41</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>13.0</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.54</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.19</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>13.1</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.96</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>2.31</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>13.0</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.64</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>2.17</SPAN></P></TD></TR><TR><TD class="c9-12"><P class="c9-2 c9-8"><SPAN>40/43</SPAN></P></TD><TD class="c9-13"><P class="c9-2 c9-8"><SPAN>12.3</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.78</SPAN></P></TD><TD class="c9-47"><P class="c9-2 c9-8"><SPAN>1.77</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Restriction digestion of Part 7 with EcoR1 and PstI</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>2x 40 µl digestion setup with ~1,5 µg template</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1h at 37°C</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Run on gel for gelextraction</SPAN></P><P class="c9-2"><SPAN> </SPAN></P><P class="c9-2"><SPAN class="c9-5">Gelextraction of Part 1,3,4,7</SPAN></P><P class="c9-2"><SPAN class="c9-39">(planned ON digest of parts 1,3,4 was not done…???)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Whole volume of the digestion setups run on gel, 85 V for ~45 min</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>stained in EtBr for 15 min, washed in H20 for 10 min</SPAN></P>&lt; <P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>1 LogLadder; 2-6 Part 1; 7-14 Part 3;   1 LogLadder; 2-7 Part 4; 8-16 Part 7</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Thick band in the middle was cut out, same templates pooled. For part 7, upper band cut out and pooled in the gel extraction process</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Standard protocol for gel extraction was used. Eluation in HPLC H2O, 2 Steps with 15 µl each (prewarmed 80°C)</SPAN></P><P class="c9-20"><SPAN class="c9-5">Nanodrop measurements:</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Part1: 6,7 ng/ µl</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Part3: 14,4 ng/ µl</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Part4: 15,8 ng/ µl</SPAN></P><P class="c9-20"><SPAN>Part7: 56,6 ng/ µl </SPAN></P><P class="c66 c9-20"><SPAN class="c9-5">Transformation of ON ligations from 30.7.13 and Test of DH5α competent cells</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Strain: DH5α</SPAN></P><P class="c66 c9-20"><SPAN class="c9-5">Transformation A1: pBluescript cloning</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>ON ligation reactions for cloning of DarR and Riboswitches 40/41, 40/43, 41/42, 42/43 into pBluescript -&gt; addition of whole ligation reactions to comp. cells-&gt; transformation -&gt; plated on LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN><SPAN> plates</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>neg. control: addition of 10 µl dH</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O-&gt; transformation -&gt; plated on LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN><SPAN> plates</SPAN></P><P class="c9-20 c66"><SPAN class="c9-5">Transformation A2: competent cells-testing</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>   </SPAN><SPAN>pBluescript (272 ng/µl) -&gt; preparation of 1:272 dilution in dH</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O (= 1 ng/µl) -&gt; addition of 1 µl of the dilution to comp. cells -&gt; transformation -&gt; plated on LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN><SPAN> plates</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>   </SPAN><SPAN>1 tube with comp. cells was left untransformed (no DNA, no water) -&gt;cells had to endure same transformation procedure as all other cells -&gt; 150 µl were plated on LB</SPAN><SPAN class="c9-7">Amp</SPAN><SPAN>, LB</SPAN><SPAN class="c9-7">Cm</SPAN><SPAN> or LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN><SPAN> plates (no rest plates, no 50 µl plates)</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>  </SPAN><SPAN>The negative control with water from Transformation A1 is considered to be a negative control for comp.cell-test, as well</SPAN></P><P class="c66 c9-20"><SPAN class="c9-5">Transformation B: pSB1C3 cloning</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>  </SPAN><SPAN>ON ligation reactions for cloning of DarR and Riboswitches 40/41, 41/42, 42/43 into pSB1C3-&gt; transformation -&gt; plated on LB</SPAN><SPAN class="c9-7">Cm</SPAN><SPAN>plates</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>   </SPAN><SPAN>neg. control: addition of 10 µl dH</SPAN><SPAN class="c9-15">2</SPAN><SPAN>O-&gt; transformation -&gt; plated on LB</SPAN><SPAN class="c9-7">Cm</SPAN><SPAN>plates</SPAN></P><P class="c9-20"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>transformation was performed according to the methods folder with the following silly mistakes</SPAN></P><P class="c9-20"><SPAN>1. heat shock took longer than 90 sec., probably 100 sec?</SPAN></P><P class="c9-20"><SPAN>2. while all reactions where heat-shocked, the pSB1C3 neg. control was still on ice -&gt; heat-shock done with slight time shift to all other samples…; heat shock for pSB1C3-NC took 100 sec precisely</SPAN></P><P class="c9-20"><SPAN>3. I contaminated at least 2 plates during plating…</SPAN></P><P class="c9-20"><SPAN>4. Since </SPAN><SPAN class="c9-5 c9-38">X-Gal is light-sensitive</SPAN><SPAN>, the plates were incubated in a paper box at 37 °C</SPAN></P><P class="c9-20"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN class="c9-45">Conclusion: This transformation will work. Because I DON’T want it to work.</SPAN></P><P class="c9-2"><SPAN class="c9-5">Preparation of Plates</SPAN></P><P class="c9-2"><SPAN>1x 500 ml LB</SPAN><SPAN class="c9-7">Cm</SPAN></P><P class="c9-2"><SPAN>1x 250 ml LB</SPAN><SPAN class="c9-7">Amp+X-Gal</SPAN></P><P class="c9-2"><SPAN>-&gt;</SPAN><SPAN> plates started already to solidify -&gt; clumps were in the agar, while I poured them… I’m not sure, if we should use them…</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0730"><SPAN class="date">30th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">RD of Parts 1-4 and Part7 (see yesterday), Ligation of DarR, the Riboswitch constructs (40/41, 42/41, 42/43) ,Ligation of DarR, the Riboswitch constructs (40/41, 40/43, 42/41, 42/43) with pBluescript.</P><DIV class="timeline-cont"><P class="c9-20"><SPAN class="c9-5">RD of Parts 1-4 and Part7 (see yesterday)</SPAN></P><P class="c9-20"><SPAN class="c9-5"> </SPAN><SPAN>a 1%gel was run to determine the quality of the digest</SPAN></P><P class="c9-20"><SPAN class="c9-5">Gel:</SPAN></P><P class="c9-20"><SPAN class="c9-5">M|Part1|Part2|Part3|Part4|Part7|M|Oligo Hybridization iGEM63+64 (see yesterday)</SPAN></P><P class="c9-20"><SPAN>Parts 1, 3 and 4 seem to be pretty much completely digested. Part 2 shows strange bands, the tube was discarded. The Primer hybridization shows no telling bands but a normal primer cloud. Hopefully it can be ligated.</SPAN></P><P class="c9-20"><SPAN> </SPAN></P><P class="c9-20"><SPAN>-&gt;Parts 1, 3 and 3 were purified using the PCR clean up kit and eluted with 50µl pre-warmed H20 (mistake)</SPAN></P><P class="c9-20"><SPAN>-&gt;2</SPAN><SPAN class="c9-7">nd</SPAN><SPAN> round of the RD with PstI (NOT FD because empty new stuff ordered). Due to the 50µl elutions, I distributed 25µl into a new vial and then pippetted two digestion reactions per construct. This should have the beneficial side effect of an even more complete RD.</SPAN></P><P class="c9-20"><SPAN> </SPAN></P><P class="c9-20"><SPAN>          2x        4µl PstI</SPAN></P><P class="c9-20"><SPAN>             4µl Buffer 0</SPAN></P><P class="c9-20"><SPAN>             25µl DNA (everything)</SPAN></P><P class="c9-20"><SPAN>             7µl H2O</SPAN></P><P class="c9-20"><SPAN>             40µl reaction</SPAN></P><P class="c9-20"><SPAN> </SPAN></P><P class="c9-20"><SPAN>-&gt;Incubation at 37°C for 6,5h</SPAN></P><P class="c9-20"><SPAN>-&gt;frozen away, gel and clean-up tomorrow</SPAN></P><P class="c9-20"><SPAN class="c9-5">DarR and Riboswitch Clonation from last week</SPAN></P><P class="c9-20"><SPAN class="c9-5"> </SPAN></P><P class="c9-20"><SPAN class="c9-5">The plates used for DarR in pSB1C3, Riboswitch in pSB1C3 and E0840 turned out to be of the wrong antibiotic recepee (5µg/ml instead of the right one: 35µg/ml).</SPAN></P><P class="c9-20"><SPAN class="c9-5">-&gt;The inoculated cultures from yesterdayof course didn´t grow therefore</SPAN></P><P class="c9-20"><SPAN class="c9-5">-&gt;The plates from last week were washed with LB media and plated on new, right 35µg/ml plates to see if they contain correctly ligated clones.</SPAN></P><P class="c9-20"><SPAN class="c9-5">Gel extraction of Part 7</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Gel Ex using the standard protocol. Eluation in 2 steps: 40 µl and 10 µl, prewarmed H2O</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>2 epis 50 µl each, stored in -20°C</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Nanodrop meassurment: 6,4 and 6,3 ng/ µl. Also written on the tube</SPAN></P><P class="c9-20"><SPAN class="c9-5">Ligation of DarR, the Riboswitch constructs (40/41, 42/41, 42/43)</SPAN></P><P class="c9-20"><SPAN class="c9-5"> </SPAN></P><P class="c9-20"><SPAN class="c9-5">Ligation of DarR and Riboswitch inserts with pSB1C3 backbone purified from part 7</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Reaction of 20 µl total volume (1 µl ThermoScientific T4 Ligase + 2 µl 10x buffer + 17 µl insert-vector-water mix)</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Vector: gel ex + purification of pSB1C3 backbone from part 7 by Dominik and Jonathan -&gt; there was not enough vector left for the cloning of all riboswitches -&gt; therefore, Riboswitch iGEM_40/43 was left out</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Inserts: from 15.7.13</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Calculation of approximate insert amounts for vector:insert ratio = 1:3 using online ligation calculator:</SPAN></P><P class="c9-20 c9-46"><SPAN>Size of vector ca. 2070 bp</SPAN></P><P class="c9-46 c9-20"><SPAN>Size of DarR and Riboswitch inserts -&gt; uncut PCR product size used</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-22"><P class="c9-8 c9-20"><SPAN class="c9-5">PCR product</SPAN></P></TD><TD class="c9-22"><P class="c9-8 c9-20"><SPAN class="c9-5">Size of PCR product</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN class="c9-5">Approx. amount of insert required for ligation with 20 ng vector</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-8 c9-20"><SPAN class="c9-5">DarR</SPAN></P></TD><TD class="c9-22"><P class="c9-8 c9-20"><SPAN>ca. 600 – 700 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>17 – 20 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_40/41</SPAN></P></TD><TD class="c9-22"><P class="c9-8 c9-20"><SPAN>213 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>6 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_42/43</SPAN></P></TD><TD class="c9-22"><P class="c9-8 c9-20"><SPAN>466 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>14 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_42/41</SPAN></P></TD><TD class="c9-22"><P class="c9-8 c9-20"><SPAN>370 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>11 ng</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-20"><SPAN> </SPAN></P><P class="c9-20"><SPAN class="c9-5 c9-60">Reactions</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Preparation of Master Mix for 6 reactions in total (Actually, it’s more clever to put the vector and 1 µl water/reaction more into the master mix, but I prepared the DNA mixtures first. Because of a pipetting error, I had to do the master mix this way. Otherwise the buffer and ligase concentration between the reactions would have varied… So, don’t wonder if this protocol appears a bit… linky?...)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">Component</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN class="c9-5">For 1 reaction</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN class="c9-5">For 6 reactions</SPAN></P><P class="c9-8 c9-20"><SPAN class="c9-5"> </SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">dH</SPAN><SPAN class="c9-5 c9-15">2</SPAN><SPAN class="c9-5">O</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>3 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>18 µl</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">10x ligation buffer</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>2 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>12 µl</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">T4 ligase</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>1µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>6 µl</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">Total Volume</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>6 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20"><SPAN>36 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-20"><SPAN> </SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Preparation of DNA mixtures (14 µl in total; ca. 20 ng of vector for cloning of riboswitches and w/o insert control)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">Component</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_40/41</SPAN></P><P class="c9-8 c9-20"><SPAN class="c9-5">(7.5 ng/µl)</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_41/42</SPAN></P><P class="c9-8 c9-20"><SPAN class="c9-5">(3 ng/µl)</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_42/43</SPAN></P><P class="c9-8 c9-20"><SPAN class="c9-5">(3 ng/µl)</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN class="c9-5">DarR</SPAN></P><P class="c9-8 c9-20"><SPAN class="c9-5">(2 ng/µl)</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN class="c9-5">w/o insert</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">vector DNA (5.3 ng/µl)</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>4 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>4 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>4 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>4 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>4 µl</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">Insert DNA</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>1 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>4 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>5 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>9 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>-</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">dH</SPAN><SPAN class="c9-5 c9-15">2</SPAN><SPAN class="c9-5">O</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>9 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>6 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>5 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>1 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>10 µl</SPAN></P></TD></TR><TR><TD class="c9-11"><P class="c9-8 c9-20"><SPAN class="c9-5">Total Volume</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>14 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>14 µl</SPAN></P></TD><TD class="c9-40"><P class="c9-8 c9-20"><SPAN>14 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>14 µl</SPAN></P></TD><TD class="c9-64"><P class="c9-8 c9-20"><SPAN>14 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Addition of 6 µl Master Mix (it could be, that – when I was pipetting the riboswitch reactions – I did not change the tip… but I’m not sure about that. It was just a sudden thougth…)</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Incubation on at 16 °C</SPAN></P><P class="c9-2"><SPAN class="c9-5">Ligation of DarR, the Riboswitch constructs (40/41, 40/43, 42/41, 42/43) with</SPAN><SPAN class="c9-5"> </SPAN><SPAN class="c9-5 c9-38">pBluescript</SPAN></P><P class="c9-2"><SPAN class="c9-5">Ligation of DarR and Riboswitch inserts with pBluescript vector digested with PstI and EcoRI</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Reaction of 10 µl total volume (1 µl ThermoScientific T4 Ligase + 1 µl 10x buffer + 8 µl insert-water mix)</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Vector: digested and purified by DAC team; Amp resistance + MCS in </SPAN><SPAN class="c9-45">lac</SPAN><SPAN>Z gene </SPAN><SPAN class="c9-5 c9-72 c9-60">-&gt; plate transformation reactions on LB</SPAN><SPAN class="c9-5 c9-7 c9-60 c9-72">Amp+X-Gal</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Inserts: from 15.7.13</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Calculation of approximate inserts amounts for vector:insert ratio = 1:3 using online ligation calculator:</SPAN></P><P class="c9-46 c9-20"><SPAN>Size of digested vector ca. 2616 bp</SPAN></P><P class="c9-46 c9-20"><SPAN>Size of DarR and Riboswitch inserts -&gt; uncut PCR product size used</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-22"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">PCR product</SPAN></P></TD><TD class="c9-22"><P class="c9-2 c9-8"><SPAN class="c9-5">   Size of PCR product</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN class="c9-5">  Approx. amount of insert required for ligation with 20 ng vector</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">DarR</SPAN></P></TD><TD class="c9-22"><P class="c9-2 c9-8"><SPAN>ca. 600 – 700 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>14 - 16 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_40/41</SPAN></P></TD><TD class="c9-22"><P class="c9-2 c9-8"><SPAN>213 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>5 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_40/43</SPAN></P></TD><TD class="c9-22"><P class="c9-2 c9-8"><SPAN>403 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>9 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_42/43</SPAN></P></TD><TD class="c9-22"><P class="c9-2 c9-8"><SPAN>466 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>11 ng</SPAN></P></TD></TR><TR><TD class="c9-22"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Riboswitch iGEM_42/41</SPAN></P></TD><TD class="c9-22"><P class="c9-2 c9-8"><SPAN>370 bp</SPAN></P></TD><TD class="c9-30"><P class="c9-8 c9-20"><SPAN>8 ng</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN class="c9-5 c9-60">Reactions</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Preparation of Master Mix for 7 reactions in total (ca. 20 ng of vector for cloning of riboswitches and w/o insert control)</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-54"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Component</SPAN></P></TD><TD class="c9-24"><P class="c9-8 c23 c9-20"><SPAN class="c9-5">For 1 reaction</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c23 c9-20"><SPAN class="c9-5">For 7 reactions</SPAN></P><P class="c9-8 c23 c9-20"><SPAN class="c9-5"> </SPAN></P></TD></TR><TR><TD class="c9-54"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Vector (11.3 ng/µl)</SPAN></P></TD><TD class="c9-24"><P class="c9-8 c23 c9-20"><SPAN>2 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c23 c9-20"><SPAN>14 µl</SPAN></P></TD></TR><TR><TD class="c9-54"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">dH</SPAN><SPAN class="c9-5 c9-15">2</SPAN><SPAN class="c9-5">O</SPAN></P></TD><TD class="c9-24"><P class="c9-8 c23 c9-20"><SPAN>1 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c23 c9-20"><SPAN>7 µl</SPAN></P></TD></TR><TR><TD class="c9-54"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">10x ligation buffer</SPAN></P></TD><TD class="c9-24"><P class="c9-8 c23 c9-20"><SPAN>1 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c9-20 c23"><SPAN>7 µl</SPAN></P></TD></TR><TR><TD class="c9-54"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">T4 ligase</SPAN></P></TD><TD class="c9-24"><P class="c9-8 c23 c9-20"><SPAN>1µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c23 c9-20"><SPAN>7µl</SPAN></P></TD></TR><TR><TD class="c9-54"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Total Volume</SPAN></P></TD><TD class="c9-24"><P class="c9-8 c23 c9-20"><SPAN>5 µl</SPAN></P></TD><TD class="c9-33"><P class="c9-8 c23 c9-20"><SPAN>35 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Preparation of DNA mixtures</SPAN></P><TABLE cellpadding="0" cellspacing="0" class="c9-42"><TBODY><TR><TD class="c9-43"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Component</SPAN></P></TD><TD class="c9-41"><P class="c9-6 c9-8"><SPAN class="c9-5">Riboswitch iGEM_40/41</SPAN></P><P class="c9-6 c9-8"><SPAN class="c9-5">(7.5 ng/µl)</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-20 c26"><SPAN class="c9-5">Riboswitch iGEM_40/43</SPAN></P><P class="c9-8 c9-20 c26"><SPAN class="c9-5">(5.5 ng/µl)</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-55 c9-20"><SPAN class="c9-5">Riboswitch iGEM_41/42</SPAN></P><P class="c9-8 c9-20 c9-55"><SPAN class="c9-5">(ca. 3 ng/µl; 2</SPAN><SPAN class="c9-5 c9-7">nd</SPAN><SPAN class="c9-5"> elution)</SPAN></P></TD><TD class="c9-41"><P class="c9-2 c9-8"><SPAN class="c9-5">Riboswitch iGEM_42/43</SPAN></P><P class="c9-2 c9-8"><SPAN class="c9-5">(3 ng/µl)</SPAN></P></TD><TD class="c9-43"><P class="c9-8 c9-20 c9-29"><SPAN class="c9-5">DarR</SPAN></P><P class="c9-8 c9-20 c9-29"><SPAN class="c9-5">(2.9 ng/µ; 2</SPAN><SPAN class="c9-5 c9-7">nd</SPAN><SPAN class="c9-5"> elutionl)</SPAN></P></TD><TD class="c9-62"><P class="c9-8 c9-71 c9-20"><SPAN class="c9-5">w/o insert</SPAN></P></TD></TR><TR><TD class="c9-43"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">Insert DNA</SPAN></P></TD><TD class="c9-41"><P class="c9-6 c9-8"><SPAN>1 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-20 c26"><SPAN>2 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-55 c9-20"><SPAN>3 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-2 c9-8"><SPAN>4 µl</SPAN></P></TD><TD class="c9-43"><P class="c9-8 c9-20 c9-29"><SPAN>5 µl</SPAN></P></TD><TD class="c9-62"><P class="c9-8 c9-71 c9-20"><SPAN>-</SPAN></P></TD></TR><TR><TD class="c9-43"><P class="c9-34 c9-8 c9-20"><SPAN class="c9-5">dH</SPAN><SPAN class="c9-5 c9-15">2</SPAN><SPAN class="c9-5">O</SPAN></P></TD><TD class="c9-41"><P class="c9-6 c9-8"><SPAN>4 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-20 c26"><SPAN>3 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-55 c9-20"><SPAN>2 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-2 c9-8"><SPAN>1</SPAN></P></TD><TD class="c9-43"><P class="c9-8 c9-20 c9-29"><SPAN>-</SPAN></P></TD><TD class="c9-62"><P class="c9-8 c9-20 c9-71"><SPAN>5 µl</SPAN></P></TD></TR><TR><TD class="c9-43"><P class="c9-8 c9-20 c9-34"><SPAN class="c9-5">Total Volume</SPAN></P></TD><TD class="c9-41"><P class="c9-6 c9-8"><SPAN>5µl</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-20 c26"><SPAN>5</SPAN><SPAN class="c9-5"> </SPAN><SPAN>µl</SPAN></P></TD><TD class="c9-41"><P class="c9-8 c9-55 c9-20"><SPAN>5 µl</SPAN></P></TD><TD class="c9-41"><P class="c9-2 c9-8"><SPAN>5 µl</SPAN></P></TD><TD class="c9-43"><P class="c9-8 c9-20 c9-29"><SPAN>5 µl</SPAN></P></TD><TD class="c9-62"><P class="c9-8 c9-71 c9-20"><SPAN>5 µl</SPAN></P></TD></TR></TBODY></TABLE><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Addition of 5 µl Master Mix</SPAN></P><P class="c9-2"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Incubation on at 16 °C</SPAN></P><P class="c9-2"><SPAN class="c9-73 c9-5">Attention: DarR and Riboswitch iGEM_41/42 inserts are almost empty! I marked the tubes with a green dot. We have to prepare new ones! -&gt; Whenever Spe and Pst FD arrive:</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="tlob" id="tl_0729"><SPAN class="date">29th</SPAN><DIV class="cont"><P class="timeline-title goe-rt">Inoculation of 8 Clones from each, Riboswitch, DarR and E0840 (were put in the fridge on friday, plated 23. and 24.7.),Hybridization of iGEM_63 and iGEM_64 to generate the Promoter(part3)-Operator construct</P><DIV class="timeline-cont"><P class="c9-20"><SPAN class="c9-5">Inoculation of 8 Clones from each, Riboswitch, DarR and E0840 (were put in the fridge on friday, plated 23. and 24.7.)</SPAN></P><P class="c9-20"><SPAN class="c9-5"> </SPAN><SPAN>-&gt;</SPAN><SPAN>E0840 heavily contaminated with something that looks like fungus. Plates discarded.</SPAN></P><P class="c9-20"><SPAN>-&gt;This leaves DarR and the Riboswitch for inoculation in 4ml Cam cultures for a miniprep tomorrow.</SPAN></P><P class="c9-20"><SPAN>-&gt;</SPAN><SPAN>in addition, a masterplate was plated.</SPAN></P><P class="c9-20"><SPAN class="c9-5">Hybridization of iGEM_63 and iGEM_64 to generate the Promoter(part3)-Operator construct</SPAN></P><P class="c9-20"><SPAN>The primers were first diluted in HPLC-H2O according to the QC-sheet (100µM). Then a 1:20 dilution stock was pipetted. For this experiment, the 1:20 dilution stocks were used.</SPAN></P><P class="c9-20"><SPAN>Protocol:</SPAN></P><P class="c9-20 c9-75"><SPAN>10µl iGEM_63</SPAN></P><P class="c9-20"><SPAN>+         10µl iGEM_64</SPAN></P><P class="c9-20"><SPAN>          20µl reaction </SPAN></P><P class="c9-20"><SPAN>-&gt;80°C for 10 minutes (heatblock)</SPAN></P><P class="c9-20"><SPAN>-&gt;after 10 minutes, turn the heatblock off and let the sample slowly cool down in it</SPAN></P><P class="c9-20"><SPAN> </SPAN></P><P class="c9-2 c9-21"><SPAN>-&gt;</SPAN><SPAN> </SPAN><SPAN>The ends of the oligos are “digested” with EcoRI and SpeI and phosphorylated. The product of this experiment can directly be ligated in front of the GFP-part!</SPAN></P><P class="c9-20"><SPAN>See the gel tomorrow</SPAN></P><P class="c9-20"><SPAN class="c9-5">Restriction digestion of Part 1-4 and Part 7</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Part 1-4 digested with SpeI FD</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>Part 7 digested with EcoR1+SpeI FD + Phosphatase treatment (last 20 minutes of digestion)</SPAN></P><P class="c9-2 c9-21"><SPAN>-</SPAN><SPAN>     </SPAN><SPAN>40 µl setups, standard assay.</SPAN></P><P class="c9-20"><SPAN>-&gt;for part 1-4, restriction with PstI (not FD, because empty) on 30.7.</SPAN></P><DIV class="fbutton">Fold ↑</DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Goettingen/NoteBook_w9">http://2013.igem.org/Team:Goettingen/NoteBook_w9</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Goettingen/NoteBook_w9" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Goettingen/NoteBook_w9" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w9&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Goettingen/NoteBook_w9&amp;oldid=256384" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>