<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Korea_U_Seoul_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Korea_U_Seoul/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Korea_U_Seoul/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Korea_U_Seoul/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Korea_U_Seoul/Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Korea_U_Seoul/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Korea U Seoul/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>jQuery based 2depth menu-bar</TITLE></P><DIV id="header"><DIV id="nav"><UL><LI class="off"><A href="https://2011.igem.org/Team:Korea_U_Seoul">Home</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Team/Korea_University">Team</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Team/Korea_University">Korea University</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Team/Our_members">Our members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=Korea_U_Seoul">Official Team Profile</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Team/Attribution">Attribution</A></LI></UL><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Project/Abstract">Project</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Project/Abstract">Abstract</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Project/Design">Design</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Feb">Feb</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Mar">Mar</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Apr">Apr</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/May">May</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Jun">Jun</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Jul">Jul</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Aug">Aug</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Sep">Sep</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Notebook/Oct">Oct</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Safety/Safety_Questions">Safety</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Safety/Safety_Questions">Safety Questions</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Safety/Parts">Parts</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Human_Practice/Ownership_Innovation">Human Practice</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Human_Practice/Ownership_Innovation">Ownership &amp; Innovation</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Human_Practice/Sharing">Sharing</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/Human_Practice/Application">Application</A></LI><LI><A href="https://2011.igem.org/Team:Korea_U_Seoul/References">References</A></LI></DIV></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#2.2F11_-_Orientation.281st_meeting.29"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">2/11 - Orientation(1st meeting)</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#2.2F16_-_The_basics"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">2/16 - The basics</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#2.2F18_-_Parts_registry"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">2/18 - Parts registry</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#2.2F21_-_Q_.26_A_session"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">2/21 - Q&amp; A session</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#2.2F23_-_Groups_formed"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">2/23 - Groups formed</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#2.2F25_-_2010_iGEM_posters_.28Slovenia.2C_Peking.2C_Bristol.29"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">2/25 - 2010 iGEM posters (Slovenia, Peking, Bristol)</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#2.2F28_-_2010_iGEM_posters_.28Delft.2C_Imperial.2C_Cambridge.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">2/28 - 2010 iGEM posters (Delft, Imperial, Cambridge)</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#3.2F4_-_2009_iGEM_finalists.27_posters"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">3/4 - 2009 iGEM finalists' posters</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#3.2F18_-_Presentation_of_ideas"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">3/18 - Presentation of ideas</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#4.2F29_-_Division_of_labor"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">4/29 - Division of labor</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#5.2F30_-_Research_topic_selected"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">5/30 - Research topic selected</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#6.2F24_-_Plans_for_summer_break"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">6/24 - Plans for summer break</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#7.2F8_-_Advisor.27s_presentation"><SPAN class="tocnumber">13</SPAN><SPAN class="toctext">7/8 - Advisor's presentation</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#7.2F22_-_Presentation_on_experiment"><SPAN class="tocnumber">14</SPAN><SPAN class="toctext">7/22 - Presentation on experiment</SPAN></A></LI><LI class="toclevel-1 tocsection-15"><A href="#8.2F2_-_Wiki_team_meeting"><SPAN class="tocnumber">15</SPAN><SPAN class="toctext">8/2 - Wiki team meeting</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#8.2F5_-_Presentation_on_experiment_cont."><SPAN class="tocnumber">16</SPAN><SPAN class="toctext">8/5 - Presentation on experiment cont.</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#8.2F19_-_Advisor.27s_return"><SPAN class="tocnumber">17</SPAN><SPAN class="toctext">8/19 - Advisor's return</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#8.2F21_-_Logo_decided"><SPAN class="tocnumber">18</SPAN><SPAN class="toctext">8/21 - Logo decided</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#8.2F23_-_Lab_photos"><SPAN class="tocnumber">19</SPAN><SPAN class="toctext">8/23 - Lab photos</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#8.2F30_-_Team_Hoodie"><SPAN class="tocnumber">20</SPAN><SPAN class="toctext">8/30 - Team Hoodie</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#9.2F2_-_Plans_for_Asia_Jamboree"><SPAN class="tocnumber">21</SPAN><SPAN class="toctext">9/2 - Plans for Asia Jamboree</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#9.2F18_-_Briefing_on_experiment_results"><SPAN class="tocnumber">22</SPAN><SPAN class="toctext">9/18 - Briefing on experiment results</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#9.2F30_-_Almost_finished.."><SPAN class="tocnumber">23</SPAN><SPAN class="toctext">9/30 - Almost finished..</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="2.2F11_-_Orientation.281st_meeting.29"><B>2/11 - Orientation(1st meeting)</B></SPAN></H2><UL><LI>2010 KU iGEM team member's presentation
</LI><LI>Advisor's presentation
</LI><LI>2011 KU iGEM team member introduction
</LI></UL><H2><SPAN class="mw-headline" id="2.2F16_-_The_basics"><B>2/16 - The basics</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Presentation at 2nd meeting</DIV></DIV></DIV><P>We had a second meeting to study about iGEM registry.
</P><UL><LI><B>We learned about</B>:
</LI></UL><OL><LI> A reason for using low # in plasmid backbone's operation, and its merits and demerits.
</LI><LI> How to express parts of plasmid such as RBS, promoter, ...etc.
</LI><LI> Types of restriction enzymes
</LI><LI> Why a scar occurs and how to prevent it (Silver's method, application of Berkley's method, Lim standard)
</LI><LI> How to construct a backbone having desired origins and antibiotic resistance based on Reshma method.
</LI><LI> Practical usage in companies.
</LI><LI> Both parts and plasmids can be archived (recording, saving, storing) 
</LI></OL><UL><LI><B>What we didn't understand were</B>:
</LI></UL><OL><LI> Why scars occur, and preventing shift caused from it, and a relationship between mixed and scar.
</LI><LI> What the meaning of 'shuffling protein domain' and 'overhanging of 4 base pair' in Lims standard are.
</LI><LI> Relationship between restriction sites E, S, N.
</LI></OL><H2><SPAN class="mw-headline" id="2.2F18_-_Parts_registry"><B>2/18 - Parts registry</B></SPAN></H2><UL><LI>We continued studying about parts registry.
</LI></UL><DL><DD><A href="http://partsregistry.org/Part_Names" class="external free" rel="nofollow">http://partsregistry.org/Part_Names</A></DD></DL><DL><DD><A href="http://partsregistry.org/Help:BioBrick_Prefix_and_Suffix" class="external free" rel="nofollow">http://partsregistry.org/Help:BioBrick_Prefix_and_Suffix</A></DD></DL><DL><DD><A href="http://partsregistry.org/Assembly:Composable" class="external free" rel="nofollow">http://partsregistry.org/Assembly:Composable</A></DD></DL><DL><DD><A href="http://partsregistry.org/Help:BioBrick_Assembly" class="external free" rel="nofollow">http://partsregistry.org/Help:BioBrick_Assembly</A></DD></DL><DL><DD><A href="http://partsregistry.org/Assembly:Standard_assembly" class="external free" rel="nofollow">http://partsregistry.org/Assembly:Standard_assembly</A></DD></DL><DL><DD><A href="http://openwetware.org/wiki/Synthetic_Biology:BioBricks/3A_assembly" class="external free" rel="nofollow">http://openwetware.org/wiki/Synthetic_Biology:BioBricks/3A_assembly</A></DD></DL><DL><DD><A href="http://partsregistry.org/Assembly:Rolling_assembly" class="external free" rel="nofollow">http://partsregistry.org/Assembly:Rolling_assembly</A></DD></DL><DL><DD><A href="http://partsregistry.org/Assembly:RBS-CDS_issues" class="external free" rel="nofollow">http://partsregistry.org/Assembly:RBS-CDS_issues</A></DD></DL><DL><DD><A href="http://partsregistry.org/Help:Ribosome_Binding_Sites" class="external free" rel="nofollow">http://partsregistry.org/Help:Ribosome_Binding_Sites</A></DD></DL><DL><DD><A href="http://partsregistry.org/Ribosome_Binding_Sites/Catalog" class="external free" rel="nofollow">http://partsregistry.org/Ribosome_Binding_Sites/Catalog</A></DD></DL><DL><DD><A href="http://partsregistry.org/Ribosome_Binding_Sites/Further_Reading" class="external free" rel="nofollow">http://partsregistry.org/Ribosome_Binding_Sites/Further_Reading</A></DD></DL><DL><DD><A href="http://partsregistry.org/Ribosome_Binding_Sites/Glossary" class="external free" rel="nofollow">http://partsregistry.org/Ribosome_Binding_Sites/Glossary</A></DD></DL><DL><DD><A href="http://partsregistry.org/Help:Ribosome_Binding_Sites/Mechanism" class="external free" rel="nofollow">http://partsregistry.org/Help:Ribosome_Binding_Sites/Mechanism</A></DD></DL><DL><DD><A href="http://partsregistry.org/Ribosome_Binding_Sites/Design" class="external free" rel="nofollow">http://partsregistry.org/Ribosome_Binding_Sites/Design</A></DD></DL><DL><DD><A href="http://partsregistry.org/Ribosome_Binding_Sites/Construction" class="external free" rel="nofollow">http://partsregistry.org/Ribosome_Binding_Sites/Construction</A></DD></DL><DL><DD><A href="http://partsregistry.org/Help:Ribosome_Binding_Sites/Standard_Assembly" class="external free" rel="nofollow">http://partsregistry.org/Help:Ribosome_Binding_Sites/Standard_Assembly</A></DD></DL><DL><DD><A href="http://partsregistry.org/Help:Ribosome_Binding_Sites/Mechanism#The_Shine-Dalgarno_Sequence" class="external free" rel="nofollow">http://partsregistry.org/Help:Ribosome_Binding_Sites/Mechanism#The_Shine-Dalgarno_Sequence</A></DD></DL><UL><LI>We studied about part's prefix and suffix and how to assemble it. We went further to find out what an RBS is, and studied glossaries and composition of them. Then we asked concepts we didn't understand to advisors.
</LI></UL><H2><SPAN class="mw-headline" id="2.2F21_-_Q_.26_A_session"><B>2/21 - Q &amp; A session</B></SPAN></H2><P>Today’s meeting was canceled due to an unexpected problem with school’s Wi-Fi. We therefore had a question-and-answer session.
</P><UL><LI><B>The questions were</B>: 
</LI></UL><OL><LI> Aligned space.
</LI><LI> Orthogonal RBS
</LI><LI> Meaning of 'various purposes of origin of replication'.
</LI></OL><H2><SPAN class="mw-headline" id="2.2F23_-_Groups_formed"><B>2/23 - Groups formed</B></SPAN></H2><P>We finished studying the 'Help' section of registry.
</P><UL><LI><B>What we learned and did</B>:
</LI></UL><OL><LI>Termination, rho-dependent/independent pathway, types of RNA polymerase.
</LI><LI>Design, components, construction, efficiency of terminator.
</LI><LI>There are forward, reverse, and bidirectional terminators.
</LI><LI>How to differentiate between natural parts and synthesized parts using barcode.
</LI><LI>Parts symbols.
</LI><LI>Read dissertations related to the above topics.
</LI></OL><UL><LI><B>Questions</B></LI></UL><OL><LI>The relationship between poly (A) tract, poly (T) tract and bidirectional terminator
</LI><LI>(<A href="http://partsregistry.org/Help:Terminators/Design" class="external free" rel="nofollow">http://partsregistry.org/Help:Terminators/Design</A>)
</LI><LI>What is a reverse primer and why we need to design it in oligo terminator construction
</LI><LI>(<A href="http://partsregistry.org/Help:Terminators/Construction" class="external free" rel="nofollow">http://partsregistry.org/Help:Terminators/Construction</A>)
</LI><LI>The meaning of forward/bidirectional/reverse terminator
</LI><LI>The Existence of forward/bidirectional/reverse terminator in eukaryotic cell
</LI><LI>The meaning of barcode in forward/bidirectional/reverse terminator
</LI><LI>The meaning of stop/start/binding symbols
</LI></OL><UL><LI><B>Conference</B></LI></UL><DL><DD>- We decided to see previous years’ finalists' research for brain-storming.
</DD><DD>- Kyengwoo Jang will be editor-in-chief of wiki, and Min-jeong Lee will manage tweeter. Other members will also play a part in managing them.
</DD><DD>- We divided the team into three groups for effective drawing of ideas.
</DD><DD><UL><LI>Group 1 : MinGyu Kim, Seungwan Chae, Haerim Song, YunHyung Choi, Sooyoung Yoon, JinSick Choi.
</LI><LI>Group 2 : JeongWan Chae, Kyengwoo Jang, Ji-hyong Nam, Jong woo Hong, Min-jeong Lee, Songa Jung, Areum Go.
</LI><LI>Group 3 : Seong Min Pak, Yeji Chae, Seungbo Ryu, Hanon Lee, Beomgyo O, Heuiseop Jung.
</LI></UL></DD></DL><DL><DD>- Once first semester begins in March, we will have meetings once a week. 
</DD></DL><DL><DD>- Until the end of March, we need to gather/organize our ideas and make overall outline of our research procedure.
</DD></DL><H2><SPAN class="mw-headline" id="2.2F25_-_2010_iGEM_posters_.28Slovenia.2C_Peking.2C_Bristol.29"><B>2/25 - 2010 iGEM posters (Slovenia, Peking, Bristol)</B></SPAN></H2><P>We studied using 2010 iGEM teams’ posters. We focused on coming up with ideas and observing how the posters were made, rather than trying to understand the overall mechanism.
</P><OL><LI><B>Slovenia</B></LI><LI> Zinc finger; viewing it methodologically: SPR, EMSA, beta-galactosidase activity
</LI><LI> Explanation of oscillators based on ZNFs
</LI><LI><B>Peking</B></LI><LI>Heavy metal decontamination kit; detection of mercury and its coagulation.
</LI><LI><B>BCCS-Bristol</B></LI><LI>Measuring of nitrogen concentration using microorganism to measure soil fertility level.
</LI></OL><UL><LI><B>Conference</B></LI></UL><DL><DD>- We decided to receive additional applications to join the KU-iGEM team until 28/2/2011.
</DD></DL><DL><DD>- After first semester begins, meetings will be held every Friday at 7PM
</DD></DL><DL><DD>- Leader of each group
</DD><DD><UL><LI>Group 1 : MinGyu Kim
</LI><LI>Group 2 : JeongWan Chae
</LI><LI>Group 3 : Seong Min Pak
</LI></UL></DD></DL><DL><DD>- On 3/4, we will have a presentation on the 2009 iGEM finalists.
</DD></DL><DL><DD>- Each team should think of at least two ideas and present them by 18/3/2011
</DD></DL><H2><SPAN class="mw-headline" id="2.2F28_-_2010_iGEM_posters_.28Delft.2C_Imperial.2C_Cambridge.29"><B>2/28 - 2010 iGEM posters (Delft, Imperial, Cambridge)</B></SPAN></H2><P>Studied other teams’ posters
</P><DL><DD>- First presentation – Alkanivore (TU Delft)
</DD><DD><UL><LI>Microorganism that is capapble of removing oil, which contaminates oceans.
</LI></UL></DD></DL><DL><DD>- Second presentation – Parasight (Imperial College London)
</DD><DD><UL><LI>Schistosoma
</LI></UL></DD></DL><DL><DD>- Third presentation – Application of bioluminescence (Cambridge)
</DD><DD><UL><LI>Members brain-stormed about Gibson PCR to understand its mechanism
</LI></UL></DD></DL><UL><LI><B>Current plans</B></LI></UL><DL><DD>- Groups will get together separately and have meetings to come up with new ideas.
</DD></DL><DL><DD>- March 4th 7 O’clock
</DD><DD><UL><LI>2009 finalist team poster study
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="3.2F4_-_2009_iGEM_finalists.27_posters"><B>3/4 - 2009 iGEM finalists' posters</B></SPAN></H2><P>At Hong Kong University of Science and Technology 1st competition: 2011. 10/15~16
</P><P>At Massachusetts Institute of Technology 2nd competition: 2011. 11/5~7
</P><UL><LI><B>Brain-storming</B></LI></UL><DL><DD><UL><LI>Focus on
</LI></UL><OL><LI>Being original; come up with an ‘interesting’ research subject
</LI><LI>An already existing idea which failed in experiments
</LI></OL></DD><DD>(Experiment should involve bacteria)
</DD></DL><UL><LI><B>Today’s meeting</B> – 2009 finals team posters
</LI></UL><OL><LI>Valencia team
</LI><LI>Cambridge team
</LI><LI>Imperial College London team
</LI><LI>Heidelberg team
</LI></OL><H2><SPAN class="mw-headline" id="3.2F18_-_Presentation_of_ideas"><B>3/18 - Presentation of ideas</B></SPAN></H2><UL><LI><B>Group 1</B></LI></UL><OL><LI><B>Collection/removal of petroleum</B></LI><LI>Transportation of petroleum-containing micelle by magnetotaxis
</LI><LI>By using magnetotaxis, we can concentrate the microorganism at a certain place once the organism consumes petroleum (Formation of a micelle by using hydrophobic outer membrane is a difficult task, however.)
</LI><LI>Additionally, the possibility of drug-detecting <I>E.coli</I>. Fluorescing upon detection.
</LI><LI><B>Dealing with automobile exhaust</B></LI><LI>Incorporation of a gene that enables the microorganism to perform denitrification. (Removal of NO2, SO2)
</LI><LI>Formation of a semi-permeable membrane that only allows passage of gas.
</LI><LI>With this membrane in-between, exhaust and the liquid containing the microorganism will flow in opposite directions.
</LI><LI><B>Rare-earth-metal-detecting microorganism</B></LI><LI>Microorganism that can detect lanthanides.
</LI><LI>Formation of a porphyrin that can bind specifically with a desired ion size. Porphyrin will bind with lanthanides.
</LI><LI>After binding, recollect the microorganism and extract the metal.
</LI><LI><B>Bacteria for drought preparation</B></LI><LI>Expression of a polysaccharide with many OHs on bacteria’s outer membrane.
</LI><LI>These OHs can bind exceptionally well with water, enabling them to hold a relatively large amount of water.
</LI><LI>Once drought begins, bacteria will form an endospore, allowing them to survive the drought, and plants will be able to absorb water from them.
</LI></OL><UL><LI><B>Group 2</B></LI></UL><OL><LI><B>Self-adjusting window blind</B></LI><LI>Utilizes phototaxis
</LI><LI>Microorganism reacts to strong light concentration, and their movement activates a micromoter, which alters the degree in which blind is bent.
</LI><LI><B>Microorganism art</B></LI><LI>Utilizes microorganisms that differ in their preference of growth conditions: Oxygen concentration, temperature and atmospheric pressure, 
</LI><LI>Apply stimulus to lead the organisms to form a desired shape.
</LI><LI><B>Detection of alteration in food</B></LI><LI>In the case of ice-cream, refreezing it after it has melted decreases its flavor and quality.
</LI><LI>Because of the covering on food, it is difficult to detect any changes with bare eyes.
</LI><LI>Design an organism that changes gene expression when temperature rises abnormally.
</LI><LI>By incorporating sensors (made of the above mentioned microorganisms) on the covering, it is possible to detect changes.
</LI><LI><B>Microorganisms that detect many changes</B></LI><LI>Atmospheric pressure sensing microorganism that changes color depending on pressure: incorporating it into balloons may produce balloons that change color as they rise in altitude.
</LI><LI>Microorganism that can detect differences in chemical structure of ingredients in food: May be able to differentiate between good cheese and bad cheese. 
</LI><LI>Microorganism that can detect small vibrations.
</LI><LI><B>Odor-removing microorganism</B></LI><LI>Removes bacteria that proliferate and actively engage in metabolic activity on the skin of sweating humans.
</LI><LI>Alters chemical structure of the odor-causing substance, making the odor more favorable.
</LI><LI><B>Bio-pesticide</B></LI><LI>Fusion of pest-resistant microorganisms and endospore-forming microorganism.
</LI><LI><B>Prevention of petroleum spreading</B></LI><LI>There appears to be a substance that increases petroleum’s viscosity, decreasing the speed in which it spreads. 
</LI><LI>Design a microorganism that produces such substance.
</LI></OL><UL><LI><B>Group 3</B></LI></UL><OL><LI><B>Microorganism scanner</B></LI><LI>Light sensitive bacteria express certain genes depending on specific light stimulus.
</LI><LI>‘Translating’ gene expressions into electrical signal allows computer to express the image on screen.
</LI><LI><B>Microorganism key</B></LI><LI>Depending on temperature and pH, mutagen is formed, changing genetic structure. 
</LI><LI>Leads to various patterns.
</LI></OL><UL><LI><B>A word of advice from 2010 Korea University iGEM team members</B></LI></UL><DL><DD>- Think again about the meaning of Synthetic Biology. Stick to this concept and try to come up with an idea.
</DD><DD>- Come up with ideas that may potentially be useful and have a value as itself. 
</DD><DD>- Don’t be too held up with coming up with new ideas. You can always use existing ideas and be creative with them. Or, you can also try to prove that same result can be obtained through a different process.
</DD></DL><PRE>                -Simple detection is NOT original. Approach same subjects from different perspectives.
</PRE><H2><SPAN class="mw-headline" id="4.2F29_-_Division_of_labor"><B>4/29 - Division of labor</B></SPAN></H2><P>Because our research topic is not yet decided, we divided the iGEM team into small groups to prepare for the upcoming iGEM competition.
</P><OL><LI>Wiki
</LI><LI>Sponsor
</LI><LI>Brochure+poster+ppt
</LI><LI>Tweeter
</LI><LI>Experiment
</LI></OL><P>We plan on visiting our advisor as either a group or as an individual to explain our presentation topic on the ppt. Then we plan on deciding on which ones to include in our final presentation. 
On Friday, we expect to have our research topic decided.
</P><H2><SPAN class="mw-headline" id="5.2F30_-_Research_topic_selected"><B>5/30 - Research topic selected</B></SPAN></H2><P><B>Group 1</B></P><DL><DD>1. Hydrocarbon absorption
</DD><DD><UL><LI>Because it is hydrophobic, it will not be difficult for it to diffuse through the membrane and the cell wall.
</LI><LI>Other teams for example: used <I>AlnA</I> ptn to increase alkane solubility.
</LI><LI><I>AlkB2</I> and <I>ladA</I> were used as transporters. We can try to knock-out those genes that enable microorganisms to degrade alcohol to keep the alcohol stored within them.
</LI></UL></DD><DD>Suggestion 1) <I>HAE1</I> gene, <I>AcrAB</I> genes are an option
</DD><DD>Suggestion 2) Increasing absorption rate: increase metabolic rate to speed up the rate at which alkane is degraded.
</DD><DD>Suggestion 3) Instead of looking for a suitable transporter, focus on increasing efficiency in synthesizing and extracting wax.
</DD></DL><DL><DD>2. Determine the types of enzymes in <I>E.coli</I></DD><DD>3. Study about alkane monooxygenase
</DD><DD>4. Using and synthesizing of wax
</DD><DD><UL><LI>Directly injecting substrate is meaningless. Find out if we can use various substrates.
</LI><LI>Acyl-CoA reductase(<I>AcrM</I>) is the key enzyme.
</LI></UL></DD></DL><DL><DD>5. Increasing efficiency
</DD><DD><UL><LI>Overexpression of the gene <I>AcrM</I></LI><LI>Modification of inclusion body
</LI></UL></DD></DL><P><B>Group 2</B></P><DL><DD>1. Microorganism that synthesizes biosurfactants and degrades hydrocarbons
</DD><DD><UL><LI>Detection of soil contaminants; Because soil condition is difficult to determine through naked eyes, detection is important. (Removal of contaminant and thereby facilitating soil’s recovery)
</LI><LI>Absorbing the contaminant
</LI><LI>Surfactant expression: Green Fluorescent Protein; House-keeping gene expression: Red Fluorescent Protein
</LI><LI>Because many species of bacteria can synthesize biosurfactant, it may be possible to degrade various contaminants.
</LI><LI>Reason for TU Delft’s unsuccessful experiment: injecting too many genes all at once interferes with the regulation system, resulting in non-favorable results. 
</LI></UL></DD><DD>- Should we insert the genes all in one cell?
</DD></DL><UL><LI>Heterogeneous genes, if they are expressed at the same time, may cluster. Furthermore, the organism will not be able to sustain and survive a modification of its membrane.
</LI><LI>Facilitation mechanism: Incorporation of a regulation system into TU Delft's mechanism. Insertion of nitrogen fixation gene to bring about a more fertile soil. However, a downside to this method is that a plant must be nearby. Searching for a microorganism that is independent of plants may be an option.
</LI><LI>Nitrogen fixation is not easy. Hydrogenase is used in Nitrogen fixation, but its enzymatic activity is greatly diminished upon exposure to oxygen, thus decreasing efficiency.
</LI></UL><DL><DD>- Groups 1 and 2 should combine and study more about Delft.
</DD><DD>- We should not be satisfied with simple detection. We must proceed further.
</DD><DD>- Advisor will let us know of the possibilities of our experiment.
</DD><DD>- Find out why Delft used the genes they used.
</DD></DL><DL><DD>- Aiming for bioenergy production
</DD><DD><UL><LI>Substrate selection
</LI><LI> Fermentation by anaerobic bacteria of burnt waste matter from cities is too big of a scale
</LI><LI> Wax seems to be the ideal substrate.
</LI></UL></DD></DL><P><B>Group 3</B></P><DL><DD>1. Auto-inducible and auto-lytic system using quorum sensing
</DD><DD><UL><LI>Quorum sensing gene: amplified with PCR(a PCR that increases rate of mutation)
</LI><LI>Addition of Magnesium ion to a PCR reactant mixture(error-prone condition)
</LI><LI>As PCR proceeds, gene mutations occur and these genes are replicated. Affinity difference between inducer and promoter becomes clearer and the concentration at which transcription begins changes. By screening the replicated genes’ affinity, we can obtain a library, and by inserting a GFP gene near a promoter, degree of expression can be determined. Differentiate genes based on time of GFP expression, measure their affinity, and select desired bacterial species.
</LI><LI>Testing: Upon contacting the desired cell OD, is the target gene expressed? (Inducers differ in the concentrations they recognize.)
</LI></UL></DD></DL><DL><DD>2. Basic tool: wax synthesis
</DD><DD><UL><LI>Should the wax synthesis directed toward the cell interior, exterior, within the media? 
</LI></UL></DD><DD>- Directing it towards the surface of the media will be the most convenient method. Following synthesis, wax will float on the media which we simply will pick up. 
</DD></DL><UL><LI>Involvement of wax esterase and EtOH? 
</LI></UL><DL><DD>- The problem is, wax esterase is a dimer, making it difficult to display. On the other hand, making the microorganism secrete FA is too complex a task. We then came up with the idea of flipping the FA transporter.  If that’s difficult, we can try storing and stacking wax as layers within the organism, or try producing alcohol.
</DD></DL><UL><LI>Instead of supplying substrates manually, inducing the microorganism to synthesize its necessary compounds may be possible. In doing so, we should not interfere with the beta-oxidation pathway. Altering such crucial pathway may result in cell death.
</LI><LI>Using Atfat is an option. (Important to note that the type of esterase used depends on whether target FAs are short or long)
</LI></UL><DL><DD>3. FA absorption mechanism
</DD><DD><UL><LI>Anaerobic <I>Acinetobacter</I> growth rate is too slow. Desired quantities of products are not synthesized.
</LI><LI><I>Acinetobacter</I> transport components may be applied to <I>E.coli</I>. May be useful if possible.
</LI></UL></DD></DL><DL><DD>- <B>Storing of wax within cytoplasm (Selected)</B> (MICRO-DIESEL)
</DD></DL><H2><SPAN class="mw-headline" id="6.2F24_-_Plans_for_summer_break"><B>6/24 - Plans for summer break</B></SPAN></H2><P>We listened to our advisor’s presentation on the scope of biofuel chemistry.
</P><UL><LI>We divided the members into groups again so that they have specific things to do during the summer break. 
</LI></UL><DL><DD>- C.C.P - Mingyu, Jinsick, Yunhyung, Seungwan, Seungbo
</DD><DD>- Group 1: Verification of <I>E.coli</I> K27 strain - Jihyong, Minjeong, Hanon, Songa, Sanghoon
</DD><DD>- Group 2: Synthesis, codon optimization and expression tests of <I>car</I> and <I>fac</I> - Mingyu, Jeongwan, Seungwan, Heuiseop, Jongwoo, Areum
</DD><DD>- Group 3: Cloning method for <I>tesA</I> - Haerim, Hongjae, Yunhyung, Yeji, Sooyoung, Beomgyo
</DD><DD>- Group 4: Management of Wikipedia, poster, and fundraising - Hongjae, Hanon, Yeji, Beomgyo, Sooyoung, Haerim, Sanghoon, Gyeongwoo
</DD></DL><DL><DD>- There will be tasks for individual members such as reading dissertations and etc.
</DD><DD>- Due to our advisor’s personal matters, we decided to finish our experiment until late September.
</DD><DD>- We will have meetings on these days at 5 O’clock: 7/8, 7/22, 8/5, 8/19, 9/2
</DD></DL><UL><LI>Things we’ve done so far:
</LI></UL><DL><DD>0211 – First orientation
</DD><DD>0214 – First meeting
</DD><DD>0214~0304 – Studied biobricks, 2009, 2010 finalist posters; discussed our plans; formed several groups for more efficient brain-storming of ideas.
</DD><DD>0304 – Final team member list confirmed
</DD><DD>0318~0408 – Groups present their ideas and basic models; discussion on ideas presented and feedback
</DD><DD>0429~0520 – Formed new groups for iGEM competition preparation; idea presentation, modification
</DD><DD>0530 – Final idea presentation; experiment subject decided – biofuel
</DD><DD>0624 – Advisor’s briefing on biofuel; each group assigned different tasks: verification of <I>E.coli</I>/ordering, registering of parts/SA cloning
</DD></DL><H2><SPAN class="mw-headline" id="7.2F8_-_Advisor.27s_presentation"><B>7/8 - Advisor's presentation</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:398px;"><DIV class="thumbcaption">Plans for July~September</DIV></DIV></DIV><P><B>Advisor’s presentation</B></P><UL><LI>Mechanisms
</LI><LI>Plans
</LI><LI>Things needed to be ordered
</LI></UL><DL><DD>-<I>Vibrio harveyi</I>; media check; KACC number check
</DD></DL><UL><LI>Things needed to be done by individual members
</LI></UL><DL><DD>-Analyze trends of supporting data
</DD><DD>-Detection methods by TLC, GC, HPLC, and understand the mechanism
</DD><DD>-Understand reasons for codon optimization
</DD></DL><UL><LI>Things needed to be done as a group
</LI></UL><DL><DD>-Group 1: Search for promoter and vector type
</DD><DD>-Group 2: Search for C14’s usage; cost, etc.
</DD><DD>-Group 3: Search for medium conditions
</DD></DL><H2><SPAN class="mw-headline" id="7.2F22_-_Presentation_on_experiment"><B>7/22 - Presentation on experiment</B></SPAN></H2><P><B>Group 1</B></P><UL><LI>Presented about
</LI></UL><DL><DD>1. Plasmid in which the gene will be incorporated
</DD><DD>2. Promoter Tac = hybrid of E.coli's <I>trp</I> and <I>lac</I> operon
</DD><DD>- Strong promoter of gene expression
</DD></DL><DL><DD>3. Induction can be done by the addition of IPTG(isopropyl-beta-D-1-thiolgalactopyranoside)
</DD><DD>- IPTG is a derivative of lactose and is used as an inducer to operate the promoter.
</DD></DL><DL><DD>4. Repression can be done by the expression of <I>lac</I>1 protein, which is a type of repressor
</DD><DD>- <I>lac</I>1 protein is a DNA-binding protein involved in the metabolism of lactose.
</DD><DD>- Promoter mutation: intracellular concentration of <I>lac</I>1 repressor is increased, leading to stronger repression.
</DD></DL><H2><SPAN class="mw-headline" id="8.2F2_-_Wiki_team_meeting"><B>8/2 - Wiki team meeting</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Discussing wiki and logo design</DIV></DIV></DIV><UL><LI>Meeting
</LI></UL><DL><DD>- Discussed ideas for design.
</DD><DD>- Basic design for the Wiki page is decided. We are planning on using Flash to design our page.
</DD><DD>- Each member is responsible for coming up with a logo for our team.
</DD><DD>- Logo designed by each member must be posted on Korea Univ. iGEM facebook page. 
</DD><DD>- All members will vote for the logo they think is the best, and the logo with the most votes will be chosen and posted on the main page of our Wiki.
</DD></DL><H2><SPAN class="mw-headline" id="8.2F5_-_Presentation_on_experiment_cont."><B>8/5 - Presentation on experiment cont.</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Group 3 Beomgyo's presentation</DIV></DIV></DIV><P><B>Group 3</B></P><UL><LI>Presented about:
</LI></UL><DL><DD>- Methods that can be used to detect the synthesized compounds, which are fatty aldehydes, fatty acids, and hydrocarbons
</DD><DD>1. Immediate detection
</DD><DD>- Fatty acid detection by measuring changes in pH
</DD><DD>- Fatty aldehyde detection by silver mirror detection
</DD></DL><DL><DD>2. Qualitative analysis
</DD><DD>- Infrared spectroscopy (thin-layer chromatography or high-performance-liquid chromatography will be used prior to using infrared spectroscopy)
</DD><DD>- Mass spectrometry
</DD></DL><UL><LI>By using infrared spectroscopy, the functional groups of the unknown compound can be identified. By using mass spectrometry, molecular weight and chemical formula can be found. We are planning on combining the results we obtain from both to deduce the identity of the compound.
</LI></UL><H2><SPAN class="mw-headline" id="8.2F19_-_Advisor.27s_return"><B>8/19 - Advisor's return</B></SPAN></H2><UL><LI>Upon the advisor's return, we had a brief meeting to inform him about the progress we've made.
</LI></UL><DL><DD>- List of members going to Asia jamboree at Hong Kong this October.
</DD><DD>- Brief review of presentation session we had on July 22nd and August 5th.
</DD></DL><H2><SPAN class="mw-headline" id="8.2F21_-_Logo_decided"><B>8/21 - Logo decided</B></SPAN></H2><DL><DD>- KU iGEM 2011 logo has been decided!!! -
</DD></DL><H2><SPAN class="mw-headline" id="8.2F23_-_Lab_photos"><B>8/23 - Lab photos</B></SPAN></H2><H2><SPAN class="mw-headline" id="8.2F30_-_Team_Hoodie"><B>8/30 - Team Hoodie</B></SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">2011 iGEM KU Team Hoodie design</DIV></DIV></DIV><UL><LI>6 of our members met online and discussed team hoodie design.
</LI></UL><H2><SPAN class="mw-headline" id="9.2F2_-_Plans_for_Asia_Jamboree"><B>9/2 - Plans for Asia Jamboree</B></SPAN></H2><UL><LI>14 members have decided to participate in the upcoming Asia Jamboree held in Hong Kong.
</LI><LI>We decided that each member should book an airline ticket separately, because of the difficulty in booking seats for 14 people.
</LI><LI>We have reserved rooms for 14 people at Regal Riverside Hotel.
</LI></UL><H2><SPAN class="mw-headline" id="9.2F18_-_Briefing_on_experiment_results"><B>9/18 - Briefing on experiment results</B></SPAN></H2><UL><LI>Overall result is as described below.
</LI></UL><DL><DD>- It did not turn out very well as detection of hydrocarbon failed.
</DD><DD>- We are not certain of the reason for the failure, but it must be either the compound's concentration was too minute to be detected or it was simply not produced.
</DD><DD>- If detection failure is due to low concentration of the compound, we will have to concentrate the culture medium to increase the amount of hydrocarbon.
</DD><DD>- TLC will not be useful for this task, so we will have to use GC to detect.
</DD><DD>- The reason for using <I>E.coli</I> BW25113 is because of their promoter, <I>Pbad</I>. Bacteria that utilize <I>Pbad</I> utilizes arabinose as an inducer, and is not able to metabolize it.
</DD></DL><UL><LI>To Hong Kong, one of our advisors, Han-Sung Roh will accompany us.
</LI></UL><UL><LI>We formed teams for poster and powerpoint design, and organization of experimental results.
</LI></UL><H2><SPAN class="mw-headline" id="9.2F30_-_Almost_finished.."><B>9/30 - Almost finished..</B></SPAN></H2><UL><LI>Current situation
</LI></UL><DL><DD>- Codon optimization failed, resulting in low fluorescence
</DD><DD>- We thought about using triton x-100(a type of detergent) on <I>E.coli</I>, which theoretically will weaken its membrane, letting hydrocarbon diffuse more easily. Our advisor however did not approve of the option.
</DD><DD>- Other options available are
</DD><DD><OL><LI> Extraction from the media
</LI><LI> Extraction through cell lysis
</LI></OL></DD></DL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Korea_U_Seoul/Notebook">http://2011.igem.org/Team:Korea_U_Seoul/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Korea_U_Seoul/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Korea_U_Seoul/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Korea_U_Seoul/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Korea_U_Seoul/Notebook&amp;oldid=222160" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>