"
Page
Discussion
View source
History
teams
Log in
 
Team:Nanjing-China/Notebook
From 2012.igem.org
Home
Team
Profile
Project
Parts
Partner
Notebook
Safety
Attributions
Practise
The steps of the experiments
1. Design of amiRNA Sequences
The target gene we want to silence is PGC-1a which is related to metabolism in several tissues. After inputting the sequence of PGC-1a to the Web MicroRNA Designer, the software will give us several potential amiRNA sequences which have high possibilities to silence the target genes. Although the sequences are picked out based on several principles, we still need to select the sequences by ourselves in the rules:
It is preferable for all intended target genes to not have mismatches to the amiRNA at positions 2 to 12.
AmiRNA candidates with one or two mismatches at the 3¢ end of the amiRNA (positions 18 to 21) should be preferred, since it has been suggested that perfectly matching amiRNAs might trigger so-called transitive siRNA formation, where amplification of sequences adjacent to the binding site is primed by the miRNA. These sequences could in turn themselves serve as silencing triggers and affect other, unintended genes.
The absolute hybridization energy of the binding between amiRNA and the target sequence should be less than −30 kcal/ mole, and preferable be in the range between −35 and −40 kcal/ mole.
The amiRNA binding site should be located within the coding region of the target gene, since UTRs are more likely to be misannotated. At least two amiRNAs per target gene or group of genes should be selected for experimental work. If several are selected, the amiRNAs should bind the target mRNA at different locations, since secondary structure is suspected to influence miRNA efficacy.
Of course, there is still an important thing to do before we use the sequences to do experiments. That is we should to do BLAST research to check whether the sequences have   binding sites in other genes which will greatly affect the experiment.
After all these work we can get the amiRNA sequences which may work well during the experiments.
2. Construction of aMIRNA precursors by Site-Directed Mutagenesis
What we want to do is to make a miRNA precursor and make use of the Dicer in the plants to produce amiRNA which we design in the former step. MIRNA precursors fold back on themselves to form a hairpin structure, and it is important to preserve this structure for successful processing. Therefore, engineering of amiRNAs into MIRNA precursor templates not only requires the exchange of the miRNA by the amiRNA sequence, but also of the pairing region in the hairpin, called the (a)miRNA*, such that pairing positions as well as G:U pairs are retained. The WMD software (WMD-Oligo window) thus generates four oligonucleotides per amiRNA sequence input: I and II to engineer the actual amiRNA, and III and IV for the amiRNA* (with wobbles).
Endogenous MIRNA precursors that have been cloned into plasmids serve as templates for PCR reactions to exchange miRNA and miRNA*. These precursors include the hairpin and short pieces of flanking sequence on either side, which are known to be part of the longer endogenous MIRNA transcript.
Six PCR oligonucleotide primers are needed to produce an aMIRNA transgene. Four primer sequences are generated by WMD and are given in 5¢→3¢ orientation. They are 40 nucleotides long and specific for the intended amiRNA. The 5¢ most two and 3¢ most 17 nucleotides match the template MIRNA precursor, while the 21 nucleotides in between do not match and will generate the amiRNA and amiRNA* in the amplicon . An additional two general oligonucleotides (A and B) that match the harboring plasmid outside of the MIRNA precursor are also required. They have been placed such that the sizes of the resulting PCR products enable convenient purification and handling.
Using the six primers, the aMIRNA precursor is amplified in three pieces (a–c) as shown in the picture. The three pieces are subsequently fused to one amplicon (d) in a single PCR reaction which is called overlapping PCR. Then we can get the aMIRNA precursors.
3. Cloning
After getting the purified PCR fragment (d), we do blunt-end cloning by using linearized plasmids. The PCR products (d) are put into vectors, after that we do sequencing to ensure that the new plasmid is indeed transformed.
Once the result of sequencing is right, we do sub-cloning into binary plasmids. In this step we cut off the sequences from the sequencing plasmids and make it into binary plasmids. We transfect the new binary plasmids into the Agrobacterium.
Finally we pick out the Agrobacterium which has taken up the binary plasmids by using the Gentamycin and Specmycin to prepare for the infection in the next step.
4. Plant Transformation
We use the Agrobacterium strain delivering the above-described binary plasmid to generate transgenic plants. After that we pick out the transgenic plants by using Gentamycin. With several generations we can get some transgenic lettuce which can be put into soil.
5. Culture of lettuce
Clean the seeds with 10% NaClO for 15min and then clean the seeds with water until the water is limpid. Put the seeds in the 1/2 MS medium uniformly. Put the seeds in the culture box for two days without light and then for one day in light.
Trim the cotyledon through the vein and cut it down with scissors. Put them in a new 1/2 medium which has a filter paper on it for one day in light.
Use the 5% sucrose wash and re-suspend the Agrobacterium. Put the cotyledon in the sucrose with 0.1%AS for 15min. Put them back onto the filter paper. Put in the culture box for two days without light.
Wash the cotyledon with water which is mixed with Carb for 3 times. Put the cotyledon into MS medium which contain Gentamycin.
After several generations put the plantlet which is still alive to a medium which is used for inducing roots.
Put the plantlet which has roots into soil that experiences sterilization.
Recent changes
What links here
Related changes
Upload file
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
Retrieved from " http://2012.igem.org/Team:Nanjing-China/Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
