<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Mingdao_Part_Collection skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Mingdao/Part Collection</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Human Practice</TITLE></P><DIV class="w3-top" style="z-index: 4"><DIV class="w3-bar w3-black w3-card-2 w3-left-align w3-large"><A href="https://2017.igem.org/Team:Mingdao" class="w3-bar-item w3-button w3-padding-large w3-white">Home</A><A href="https://2017.igem.org/Team:Mingdao/Achievements" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-blue w3-hover-white" style="text-decoration: none; color: white">Achievements</A><A href="https://2017.igem.org/Team:Mingdao/Demonstrate" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white" style="text-decoration: none; color: white">Project</A><A href="https://2017.igem.org/Team:Mingdao/HP/Silver" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white" style="text-decoration: none; color: white">Human-Practice</A><A href="https://2017.igem.org/Team:Mingdao/Notebook" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white" style="text-decoration: none; color: white">Notebook</A><A href="https://2017.igem.org/Team:Mingdao/Attributions" class="w3-bar-item w3-button w3-hide-small w3-padding-large w3-hover-white" style="text-decoration: none; color: white;">Attributions</A></DIV></DIV><NAV class="w3-sidebar w3-collapse w3-animate-left" style="z-index:3;width:180px; background-color: #F8F9FA" id="mySidebar"><DIV class="w3-container"><H5>Dashboard</H5></DIV><DIV class="w3-bar-block"><FORM action="https://2017.igem.org/Team:Mingdao/Basic_Part" style="margin: 10px 15px -10px 0;"><BUTTON type="submit" class="btn btn-primary btn-block text-white" style="text-align: left; padding-left:1rem;font-size: 1.2rem;">  Basic</BUTTON></FORM><FORM action="https://2017.igem.org/Team:Mingdao/Composite_Part" style="margin: 0 15px -10px 0;"><BUTTON type="submit" class="btn btn-primary btn-block text-white" style="text-align: left; padding-left:1rem;font-size: 1.2rem"> Composite</BUTTON></FORM><FORM action="https://2017.igem.org/Team:Mingdao/Improve" style="margin: 0 15px -10px 0;"><BUTTON type="submit" class="btn btn-primary btn-block text-white" style="text-align: left; padding-left:1rem;font-size: 1.1rem"> Improvements</BUTTON></FORM><FORM action="https://2017.igem.org/Team:Mingdao/Part_Collection" style="margin: 0 15px -20px 0;"><BUTTON type="submit" class="btn btn-primary btn-block text-white" style="text-align: left; padding-left:1rem;font-size: 1.2rem"> Collection</BUTTON></FORM></DIV><DIV class="w3-bar-block"><A href="#" class="w3-bar-item w3-button w3-padding-16 w3-hide-large w3-dark-grey w3-hover-black" onclick="w3_close()" title="close menu">  Close Menu</A></DIV><NAV class="navbar navbar-light bg-light"><A class="navbar-brand btn" href="#" style="text-decoration: none;"><B style="letter-spacing: 0.2rem">Overview</B></A><A class="navbar-brand btn" href="#recomb-md" style="text-decoration: none">Recombination</A><A class="navbar-brand btn" href="#suicide-md" style="text-decoration: none">Suicide</A><A class="navbar-brand btn" href="#transport-md" style="text-decoration: none">Transporter</A></NAV></NAV><DIV style="margin-left: 180px; background-color: #f2f2f2"><SECTION id="intro" style="margin-bottom: 10rem"><DIV class="row"><DIV class="col-md-8 col-sm-10" style="display: block; margin: auto"><DIV class="card"><H4 class="card-header text-center" style="letter-spacing: 0.2rem; font-size:1.5rem; background-color: #004480; color: white">Overview</H4></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-sm-12" style="display: block; margin: auto"><DIV class="card"><DIV class="card-header text-center" style="color: white; letter-spacing: 0.3rem;font-size:1.8rem; background-color:#004480">
                        Background
                      </DIV><DIV class="card-body" style="background-image: url(https://static.igem.org/mediawiki/2017/6/65/MD-test-floater-bg-image.png); background-size: 100%;"><H4 class="card-title w3-padding-24">
                            To engineer <I>Lactobacillus acidophilus</I> by chromosome integration through homologous recombination, the selection of integration site is very important for successful recombination and not disturbing bacterial internal metabolism.
                        </H4><H4 class="card-title w3-padding-24">
                            Based on the method created by Grace L. Douglas and Todd R. Klaenhammer<SUP>1</SUP>, the region between <I>slpA</I> gene (LBA0169) stop codon and the terminator was chosen as the intergenic insertion location.  The gene of <I>slpA</I> encodes a surface-layer protein with a strong constitutive promoter activity, which can also drive the expression of the inserted gene.
                        </H4><DIV class="row"><DIV class="col-md-10 col-sm-12" style="display: block; margin: auto"><H4 style="padding-left: 3rem"><B>Fig 1. The schematic diagram of gene integration location (modified from Appl Environ Microbiol. 2011)</B></H4></DIV></DIV><BUTTON class="btn text-white" style="letter-spacing: 0.3rem;background-color: #004480"><SPAN class="badge badge-info">1</SPAN>   Recombination Vector – pLBA169
                          </BUTTON><H4 class="w3-padding-16">
                            To make pLBA169, the downstream region of LBA0169 and the upstream region of the terminator were amplified by PCR. Then, the two PCR-amplified DNA fragments were ligated to pSB1C3 by tripartite ligation.
                        </H4><H4 class="w3-padding-16">
                            Next, CP29-RBS-aeBlue was cut from Part: BBa_K1033280. And this gel isolated DNA fragment was further inserted between two recombination regions on pSB1C3. This process created the standard EcoRI-XbaI-PART-SpeI-PstI assembly position.
                        </H4><H4 class="w3-padding-16">
                            To delete the extra SpeI recognition site within the one of the recombination region, we performed site-directed mutagenesis to change one nucleotide of SpeI recognition sequence. The final product has been confirmed by DNA sequencing.
                        </H4><H4 class="w3-padding-16">
                            You can refer to the following flow chart of gene cloning of the recombination vector.
                        </H4><DIV class="col-md-12 col-sm-12"><DIV style="margin-left: 20px; margin-top: 3rem"><DIV style="margin-right: 50px"><UL class="list-group"><LI class="list-group-item w3-wide" style="text-align: center"><STRONG>- REFERENCE -</STRONG></LI><LI class="list-group-item" style="color: black;">1.     Directed chromosomal integration and expression of the reporter gene gusA3 in Lactobacillus acidophilus NCFM. Appl Environ Microbiol. 2011;77(20):7365-71.</LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="card" style="width: 100%; margin-top: 3rem; margin-bottom: 3rem"><DIV class="card-body"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230000" style="text-decoration:none; color: blue;">BBa_K2230000</A></H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Type:  </B>Composite part (device)</H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                Firstly, the elements of upstream and downstream recombination sites were replicated from gDNA of Lactobacillus acidophilus and put onto pSB1C3 as a L. acidophilus recombination vector, named pLBA169, at EcoRI and PstI locations. Secondly, to generate as a standard BioBrick part and assembly vector, the part of Promoter CP29-RBS-aeBlue (BBa_K1033280) was cut by EcoRI and PstI and assembled with the vector. Finally, because there’s a SpeI recognition sequence within the 169dn element, site-directed mutagenesis was performed to change a nucleotide to leave a single SpeI site in the BioBrick suffix region. (See the flow chart and data below).
                            </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                A vector can be used to transform Lactobacillus acidophilus by chromosomal homologous recombination at the downstream location of slpA (LBA0169), which encodes a surface layer protein A. The aeBlue gene, which is driven by the wide host range and high constitutive promoter CP29, acts as a reporter. The transformed L. acidophilus will express blue color proteins.
       </H4><BUTTON class="btn text-white" style="letter-spacing: 0.3rem;background-color: #004480; display: block; margin: auto; margin-bottom: 3rem; margin-top: 3rem"><SPAN style="padding: 0 8rem 0 8rem">Gene cloning for LBA169</SPAN></BUTTON><BUTTON class="btn bg-primary text-white" style="letter-spacing: 0.3rem; display: block; margin: auto; margin-bottom: 3rem; margin-top: 3rem"><SPAN style="padding: 0 2rem 0 2rem">Gene cloning for CP29-RBS-aeBlue/pLBA169</SPAN></BUTTON></DIV></DIV><DIV class="card" style="width: 100%; margin-top: 3rem; margin-bottom: 3rem"><DIV class="card-body"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230001" style="text-decoration:none; color: blue;">BBa_K2230001</A></H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Type:  </B>Basic part</H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                EmR (Erythromycin resistance gene) was amplified from pMG36e vector and cloned onto pSB1C3.
      </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230002" style="text-decoration:none; color: blue;">BBa_K2230002</A></H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Type:  </B>Composite part</H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                EmR (Erythromycin resistance gene) was amplified from pMG36e vector and cut by XbaI &amp; PstI. Then, the fragment was assembled with RBS/pSB1A2 (BBa_B0034) cut by SpeI &amp; PstI
                            </H4><BUTTON class="btn text-white" style="letter-spacing: 0.3rem;background-color: #004480; display: block; margin: auto; margin-bottom: 3rem; margin-top: 3rem"><SPAN style="padding: 0 1rem 0 1rem">Erythromycin Resistance Gene as a Selection Marker</SPAN></BUTTON><H4 class="w3-padding-16">
                                Erythromycin is a common selection marker for engineering lactic acid bacteria1. And teams <A href="https://2013.igem.org/Team:Uppsala" style="text-decoration:none; color: blue;">Uppsala</A> in 2013 and <A href="https://2016.igem.org/Team:TMMU_China" style="text-decoration:none; color: blue;">TMMU_China</A> in 2016 were using it to select the engineered Lactobacillus with erythromycin resistant strains.
                            </H4><H4 class="w3-padding-16">
                                However, the BioBrick parts containing the erythromycin resistance gene are currently unavailable. On the other hand, iGEM HQ is unable to service the request of plasmid carrying erythromycin resistance gene. Maybe iGEM HQ can’t manipulate erythromycin resistant bacteria so far.
                            </H4><H4 class="w3-padding-16">
                                Therefore, we decided to clone the erythromycin resistance gene (EmR) by ourselves. We searched the vector carrying EmR and found an iGEM team HKUST in 2010 working with it. We got the pMG36e vector from the team and cloned the EmR gene out onto pSB1C3 (EmR/pSB1C3, BBa_K2230001). We also made RBS-EmR/pSB1C3 (BBa_K2230002) and RBS-EmR-TT/pSB1C3 (BBa_K2230003) by assembling RBS (B0034) and double terminator (B0015) parts, respectively. These will provide iGEM team this resistance gene as selection marker in the future.
                            </H4><DIV class="col-md-12 col-sm-12"><DIV style="margin-left: 20px; margin-top: 3rem"><DIV style="margin-right: 50px"><UL class="list-group"><LI class="list-group-item w3-wide" style="text-align: center"><STRONG>- REFERENCE -</STRONG></LI><LI class="list-group-item" style="color: black;">1.     Genetic engineering techniques for lactic acid bacteria: construction of a stable shuttle vector and expression vector for β-glucuronidase. Biotechnol Lett. 2014;36(2):327-35.</LI></UL></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230003" style="text-decoration:none; color: blue;">BBa_K2230003</A></H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Type:  </B>Composite part</H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                 The fragment of RBS-EmR of RBS-EmR/pSB1C3 (BBa_K2230002) cut by EcoRI &amp; SpeI was gel eluted and assembled with TT/pSB1C3 (Double terminator, BBa_B0015) cut by EcoRI &amp; XbaI.
                             </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230004" style="text-decoration:none; color: blue;">BBa_K2230004</A></H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Type:  </B>Composite part (device)</H3><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4> RBS-EmR which was cut and gel-eluted from RBS-EmR/pSB1C3 (BBa_K2230002) was assembled with CP29-RBS-aeBlue/pLBA169 (BBa_K2230000)
                             </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function:  </B></H3><H4> A vector can be used to transform Lactobacillus acidophilus by chromosomal homologous recombination at the downstream location of slpA, which encodes a surface layer protein A. EmR (erythromycin resistance gene), which is driven by the high constitutive promoter of slpA, acts as a selection marker. And aeBlue gene, which is driven by the wide host range and high constitutive promoter CP29, acts as a reporter. The transformed L. acidophilus will be erythromycin resistant and express blue color proteins.
                             </H4></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-sm-10" style="display: block; margin: auto; margin-top: 3rem"><DIV class="card"><H4 class="card-header text-center" style="letter-spacing: 0.3rem; font-size:1.8rem; background-color: #004480; color: white">Background</H4><DIV class="card-body"><H4 class="card-title w3-padding-16">  Promoter Pcar [BBa_K861171] is a glucose responsive promoter created by WHU-China in 2012. Pcar promoter region was de novo  designed with overlapping of CRP and RNA polymerase binding site. The initiation of transcription by RNA polymerase may be hindered by the binding of CRP, which occurs at the formation of cAMP-CRP complex in the low concentration of glucose. In other words, when the amount of glucose is high enough, Pcar would be turned on after the leaving of CPR due to the low concentration of cAMP, and vice versa.
                            </H4><H4 class="card-title w3-padding-16">
                                PI promoter [BBa_K861170] is a modified version to Pcar with correction of -10 position in the promoter region, which gives the promoter stronger activity and a weaker CRP interaction. The PI promoter [BBa_K861170] was used as a major part in the work of team NCKU_Tainan in 2016. These two parts have been used in our work.
                            </H4><H4 class="card-title w3-padding-16">
                                PhlF repressor system contains the repressor PhlF [BBa_K1725041] and the PhlF repressible promoter [BBa_K1725001] created by Glasgow in 2015. PhlF could repress GFP fluorescence intensity by 83-fold according to the study of Glasgow’s work.
                            </H4><H4 class="card-title w3-padding-16">
                                We’ve innovated this year a novel glucose responsive repressor system (Pcar-wRBS-PhlF-T-Pr-sRBS-GFP/pSB1C3 [BBa_K2230012]) by connecting these two system and extend the function of them. Furthermore, based on this new system, we assembled lysis and nuclease genes to the device and created the suicide circuit controlled by the presence of glucose (Pcar-wRBS-PhlF-T-Pr-sRBS-GFP-sRBS-lysis-sRBS-NucA/pSB1C3 [BBa_K2230017]).
                            </H4><H4 class="card-title w3-padding-16">
                                Lysis gene [BBa_K117000] created by NTU-Singapore in 2008 encodes Lysis protein which could not only lyse bacterial cell membrane but also activate the endonuclease of Colicin E7 (ColE7). The lysis-colicin is one class of bacteriocins which are produced to response to worsening environmental conditions and outcompete other bacteria<SUP>1</SUP>.
                            </H4><H4 class="card-title w3-padding-16">
                                NucA [BBa_K1159105] created by TU-Munich in 2013 from Staphylococcus aureus produces a thermostable exo- and endo-nuclease that is able to degrade genomic DNAs2. NucA also has a role in the cleavage of extracellular DNAs and preventing biofilm formation.
                            </H4><H4 class="card-header text-center" style="letter-spacing: 0.3rem; font-size:1.8rem; background-color: #004480; color: white">The Parts</H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230005" style="text-decoration:none; color: blue;">BBa_K2230005</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                PI-RBS-RFP was amplified from RFP Coding Device (BBa_J04450) using a primer with PI-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
                            </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                RFP can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 &amp; NCKU_Tainan 2016, the promoter, PI, is glucose responsive.
                            </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230006" style="text-decoration:none; color: blue;">BBa_K2230006</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                Pcar-RBS-RFP was amplified from RFP Coding Device (BBa_J04450) using a primer with Pcar-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
                            </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                RFP can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 &amp; NCKU_Tainan 2016, the promoter, Pcar, is glucose responsive.
                            </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230007" style="text-decoration:none; color: blue;">BBa_K2230007</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                PI-RBS-aeBlue was amplified from CP29-RBS-aeBlue (BBa_K1033280) using a primer with PI-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
                            </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                aeBlue can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 &amp; NCKU_Tainan 2016, the promoter, PI, is glucose responsive.
                            </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230008" style="text-decoration:none; color: blue;">BBa_K2230008</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                 Pcar-RBS-aeBlue was amplified from CP29-RBS-aeBlue (BBa_K1033280) using a primer with Pcar-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
                            </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                aeBlue can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 &amp; NCKU_Tainan 2016, the promoter, Pcar, is glucose responsive.
                            </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230009" style="text-decoration:none; color: blue;">BBa_K2230009</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                 PI-RBS-PhlF-T was amplified from RBS + PhlF repressor + terminator (BBa_K1725041) using a primer with PI-RBS (B0032) sequence. The PCR product was ligated to pSB1C3 cut by XbaI and PstI.
                             </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                 Repressor PhlF can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 &amp; NCKU_Tainan 2016, the promoter, PI, is glucose responsive.
                             </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230010" style="text-decoration:none; color: blue;">BBa_K2230010</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                  Pcar-RBS-PhlF was amplified from RBS + PhlF repressor + terminator (BBa_K1725041) using a primer with Pcar-RBS (B0032) sequence. The PCR product was ligated to pSB1C3 cut by XbaI and PstI.
                             </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                 Repressor PhlF can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 &amp; NCKU_Tainan 2016, the promoter, Pcar, is glucose responsive.
                             </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230011" style="text-decoration:none; color: blue;">BBa_K2230011</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                  Pr-sRBS-GFP (BBa_K1725001) with strong RBS (B0034) was assembled with PI-RBS-PhlF-T/pSB1C3 (BBa_K2230009).
                              </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                  The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
                              </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230012" style="text-decoration:none; color: blue;">BBa_K2230012</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                  Pr-sRBS-GFP (BBa_K1725001) with strong RBS (B0034) was assembled with Pcar-RBS-PhlF-T/pSB1C3 (BBa_K2230010)
                              </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                  The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
                              </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230013" style="text-decoration:none; color: blue;">BBa_K2230013</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                  Pr-wRBS-GFP (BBa_K1725002) with weak RBS (B0032) was assembled with PI-RBS-PhlF-T/pSB1C3 (BBa_K2230009)
                              </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                  The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
                              </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230014" style="text-decoration:none; color: blue;">BBa_K2230014</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                  Pr-wRBS-GFP (BBa_K1725002) with weak RBS (B0032) was assembled with Pcar-RBS-PhlF-T/pSB1C3 (BBa_K2230010)
                              </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                  The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
                              </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230015" style="text-decoration:none; color: blue;">BBa_K2230015</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                   wRBS-lysis was amplified from Lysis gene (BBa_K117000) using a primer with weak RBS sequence (B0032) and assembled with Pcar-wRBS-PhlF-T-Pr-sRBS-GFP/pSB1C3 (BBa_K2230012)
                               </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                   Suicide genes of lysis was driven under the PhlF-repressed promoter (Pr). And the expression of the repressor (PhlF) would be induced upon glucose by a glucose responsive promoter (Pcar). That is, the bacteria can be killed by the suicide genes in the absence of (or running out of) glucose.
                               </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230016" style="text-decoration:none; color: blue;">BBa_K2230016</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                   sRBS-lysis was amplified from Lysis gene (BBa_K117000) using a primer with strong RBS sequence (B0034) and assembled with Pcar-wRBS-PhlF-T-Pr-sRBS-GFP/pSB1C3 (BBa_K2230012)
                               </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                   Suicide genes of lysis was driven under the PhlF-repressed promoter (Pr). And the expression of the repressor (PhlF) would be induced upon glucose by a glucose responsive promoter (Pcar). That is, the bacteria can be killed by the suicide genes in the absence of (or running out of) glucose.
                               </H4><DIV class="row"><DIV class="col-md-5 col-sm-5" style="padding-top: 3rem"><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Part No.  :</B><A href="http://parts.igem.org/Part:BBa_K2230017" style="text-decoration:none; color: blue;">BBa_K2230017</A></H3><H3 class="card-title" style="letter-spacing: 0.1rem"><B>Type:  </B>Composite part (device)</H3></DIV></DIV><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Cloning:  </B></H3><H4>
                                    sRBS-NucA was amplified from Mature Nuclease NucA from Staphylococcus aureus (BBa_K1159105) using a primer with strong RBS sequence (B0034) and assembled with Pcar-RSB-PhlF-T-Pr-sRBS-GFP-sRBS-lysis/pSB1C3
                                </H4><H3 class="card-title" style="letter-spacing: 0.2rem"><B>Function: </B></H3><H4>
                                    Suicide genes of lysis and nuclease (NucA) were driven under the PhlF-repressed promoter (Pr). And the expression of the repressor (PhlF) would be induced upon glucose by a glucose responsive promoter (Pcar). That is, the bacteria can be killed by the suicide genes in the absence of (or running out of) glucose.
                                </H4><H4 class="card-header text-center" style="letter-spacing: 0.3rem; font-size:1.8rem; background-color: #004480; color: white; margin-top: 5rem">Experiment and Results</H4><BUTTON class="btn bg-primary text-white" style="letter-spacing: 0.3rem; margin-top: 2rem; margin-bottom: 3rem; margin-left: 3rem;"><SPAN class="badge badge-info">1</SPAN>   Glucose responsive promoter activity
                                   </BUTTON><H4 style="padding-left: 3rem">The result shown in Fig. 1 indicated that PI promoter has significant activity in LB culture media. However, the activity of Pcar promoter is greater than negative control but much smaller than PI and positive control. It’s consistent with the properties of PI and Pcar promoters just mentioned previously in GENE CLONING section and described previously in Part Resgistry [Part: BBa_K861170] by team WHU-China in 2012 who designed the promoters.</H4><H4 style="padding-left: 3rem"> In our experiment as presented in Fig. 2, PI and Pcar promoters just responded to various concentrations of glucose with a  very slight dose-dependent increase. This phenomenon didn’t correspond to the data provided by team WHU-China in 2012 and team NCKU_Tainan in 2016. Maybe our measurement was not in an optimized condition. Or the reporter of RFP activity was not sensitive enough to respond this difference.</H4><DIV class="row"><DIV class="col-md-12 col-sm-12" style="display:block; margin: auto"><BUTTON class="btn bg-primary text-white" style="letter-spacing: 0.3rem; margin-left: 3rem; margin-top: 5rem; margin-bottom: 3rem; background-color:#004480"><SPAN class="badge badge-info">2</SPAN> Functional assay for Glucose Responsive Suppressor System
        								    </BUTTON><H4 style="padding-left: 5rem">
                                                   The bacteria carrying the indicated vector were cultured in LB media supplemented with 34 μg/ml of chloramphenicol (Cm) at 37°C overnight. The next day, OD<SUB>600</SUB> was measured and adjusted to 2.5 in M9 minimal media with various concentrations of glucose. The bacteria then were incubated for 4 hours at 37°C. 100 μl of the bacterial culture was put into one well of a black-walled, clear-bottom 96-well microplates (Thermo Fisher Scientific Inc.). The fluorescence intensity was measured using BioTek Synergy H1 Hybrid Multi-Mode Reader System at Ex/Em = 488nm/518nm  for GFP.
                                        </H4><H4 style="padding-left: 5rem">In the assay for repressor system, the data in Fig. 4 gave the similar results as team Glasgow did in 2015, in which the strong activity of the repressible promoter was significantly repressed in the presence of PhlF repressor.
                                        </H4><H4 style="padding-left: 5rem">
                                            In the assay for glucose responsive repressor system, we improved the Glasgow’s BioBrick device, in which the expression of PhlF repressor was driven by a glucose response promoter, Pcar. The result in Fig 5 clearly indicated that the GFP activity driven by the repressible promoter was gradually increased in response to the loss of glucose to 1.88 folds compared to the initial GFP activity at the beginning culture in M9 media, suggesting that the level of expression of PhlF was positively corresponding to the concentration of glucose.
                                        </H4></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><BUTTON class="btn bg-primary text-white" style="letter-spacing: 0.3rem; margin-top: 2rem; margin-bottom: 3rem; margin-left: 3rem;"><SPAN class="badge badge-info">3</SPAN>   Functional assay for glucose responsive suicide circuit
                 </BUTTON><H4 style="padding-left: 3rem">
                                                The bacteria carrying the indicated vector were cultured in LB media supplemented with 34 μg/ml of chloramphenicol (Cm) at 37°C overnight. The next day, OD<SUB>600</SUB> was measured and adjusted to 2.5 in M9 minimal media with various concentrations of glucose. The bacteria then were incubated for 4 hours at 37°C. 100 μl of the bacterial culture was put into one well of a black-walled, clear-bottom 96-well microplates (Thermo Fisher Scientific Inc.). OD<SUB>600</SUB> was measured using BioTek Synergy H1 Hybrid Multi-Mode Reader System. Further,  the culture media were taken out and diluted 106 times following by spreading onto LB Cm agar plate at 37°C overnight. The third day, the numbers of colonies were counted and bacterial viability was calculated.
                                         </H4><H4 style="padding-left: 3rem">
                                         As you can see in Fig 7., the OD value in response to the decreasing concentration of glucose was gradually reduced to 1.89 much less than average 2.71 in control group without suicide gene expression, implying that the suicide proteins killed the cells in the loss of glucose in the environment. Moreover, when the bacteria were grown in M9 media with 0.5mM glucose for 4 hours, the survival rate was decreased to 34% compared to 56% of bacteria without suicide genes (Fig. 8). And the cell numbers were reduced to 671 compared to 1120 of bacteria without suicide genes. Both data confirmed that this suicide device works well and indicated that killing process began when glucose in the media was running out.
                                        </H4></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-10 col-sm-10" style="display: block; margin: auto; margin-top: 3rem"><DIV class="card"><H4 class="card-header text-center" style="letter-spacing: 0.3rem; font-size:1.8rem; background-color: #004480; color: white">Background</H4><DIV class="card-body"><H4 class="card-title w3-padding-32"> Salmonella typhimurium LT2 has two glucose-specific transporter systems, PTS system and sodium/glucose cotransporter. PTS system contains two subunits IIA encoded by crr and IIBC by ptsG which are assembled to a high-affinity active transporter. The other is a Na+/glucose cotransporter encoded by STM1128 that contributes to facilitated transport with lower glucose affinity. Based on our research, the glucose transporter of Salmonella has a lower Km compared to human small intestine, Staphylococcus and E. coli (Table 1), indicating a higher efficiency for glucose uptake. In our project, we created the glucose transporter device and genetically engineer microbes with these two systems.</H4><H4 class="card-title w3-padding-32">
                                In order to express the genes in <I>E. coli</I> for demonstration and in probiotics for proof-of-concept in a real world. We chose promoter CP29 that is a strong constitutive promoter working well in both <I>E. coli</I> and Lactobacillus spp<SUP>1</SUP>. The biobrick part, CP29-RBS-aeBlue (BBa_K1033280) was used and to be assembled with the transporter genes.
                            </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230018" style="text-decoration:none; color: blue;">BBa_K2230018</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Basic Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The crr gene was amplified from gDNA of Salmonella typhimurium and cloned onto pSB1C3 </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230019" style="text-decoration:none; color: blue;">BBa_K2230019</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The crr gene was amplified from gDNA of Salmonella typhimurium using a primer with RBS sequence (B0034), and then cloned onto pSB1C3  </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230020" style="text-decoration:none; color: blue;">BBa_K2230020</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Basic Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The ptsG gene was amplified from gDNA of Salmonella typhimurium and cloned onto pSB1C3  </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230021" style="text-decoration:none; color: blue;">BBa_K2230021</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The ptsG gene was amplified from gDNA of Salmonella typhimurium using a primer with RBS sequence (B0034), and then cloned onto pSB1C3 </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230022" style="text-decoration:none; color: blue;">BBa_K2230022</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Basic Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The STM1128 gene was amplified from gDNA of Salmonella typhimurium and cloned onto pSB1C3 </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230023" style="text-decoration:none; color: blue;">BBa_K2230023</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The STM1128 gene was amplified from gDNA of Salmonella typhimurium using a primer with RBS sequence (B0034), and then cloned onto pSB1C3 </H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230024" style="text-decoration:none; color: blue;">BBa_K2230024</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The RBS-ptsG part (BBa_K2230021) was assembled with Double Terminator/pSB1C3 (BBa_B0015)</H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 3rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230025" style="text-decoration:none; color: blue;">BBa_K2230025</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The RBS-STM1128 part (BBa_K2230023) was assembled with Double Terminator/pSB1C3 (BBa_B0015)</H4><DIV class="row" style="margin-top: 3rem"><DIV class="col-md-6 col-sm-6" style="padding-top: 1rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230026" style="text-decoration:none; color: blue;">BBa_K2230026</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">The RBS-crr part (BBa_K2230019) was assembled with RBS-ptsG-TT/pSB1C3 (BBa_K2230024)</H4><DIV class="row"><DIV class="col-md-6 col-sm-6" style="padding-top: 1rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230027" style="text-decoration:none; color: blue;">BBa_K2230027</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part (device)</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">Promoter CP29-RBS-aeBlue (BBa_K1033280) was assembled with RBS-crr-RBS-ptsG-TT/pSB1C3 (BBa_K2230026) </H4><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Function:</B></H3><H4 style="padding-left: 5rem">An active high-affinity glucose transporter system. Promoter CP29 is a constitutive and strong promoter which works well in both E. coli and Lactobacillus spp. The blue protein encoded by aeBlue gene acts as indicator for the gene expression in the transgenic strain. This device can help E. coli and Lactobacillus spp. efficiently take up and retrieve glucose in the environment. </H4><DIV class="row" style="margin-top: 3rem"><DIV class="col-md-6 col-sm-6" style="padding-top: 1rem"><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Part No.:</B><A href="http://parts.igem.org/Part:BBa_K2230028" style="text-decoration:none; color: blue;">BBa_K2230028</A></H3><H3 style="padding-left: 3rem; letter-spacing: 0.2rem"><B>Type:</B> Composite Part (device)</H3></DIV></DIV><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Cloning:</B></H3><H4 style="padding-left: 5rem">Promoter CP29-RBS-aeBlue (BBa_K1033280) was assembled with RBS-STM1128-TT/pSB1C3 (BBa_K2230025) </H4><H3 style="padding-left: 5rem; letter-spacing: 0.2rem"><B>Function:</B></H3><H4 style="padding-left: 5rem">
                                A facilitated low-affinity glucose transporter system. Promoter CP29 is a constitutive and strong promoter which works well in both E. coli and Lactobacillus spp. The blue protein encoded by aeBlue gene acts as indicator for the gene expression in the transgenic strain. This device can help E. coli and Lactobacillus spp. efficiently take up and retrieve glucose in the environment.
                            </H4><H4 class="card-header text-center" style="letter-spacing: 0.3rem; font-size:1.8rem; background-color: #004480; color: white; margin-top: 5rem">Experiment and Results</H4><DIV id="test1019" style="padding: 3rem 3rem 3rem 3rem "><DIV class="box" style="background-color: rgba(255, 255, 255, 0); margin-right: 50px"><H4 style="font-color: #a6a6a6;">To measure glucose uptake by the engineered E. coli expressing PTS system or Na+/glucose cotransporter, the bacteria were culture in LB broth supplemented with 34μg/ml of chloramphenicol at 37°C overnight. The next day, the bacterial culture was adjusted to OD<SUB>600</SUB> = 3 and exchanged with M9 minimal media with 20mM of glucose for 4 hours or at different time points.</H4><DIV class="row"><DIV class="col-md-12 col-sm-12"><H4>Glucose concentration was analyzed with Glucose (HK) Assay Kit (Sigma-Aldrich) according to the manufacturer’s instruction. Briefly, glucose was phosphorylated (G6P) by hexokinase. Then G6P was further catalyzed by G6PDH and the reduced NAHD was formed from the oxidation of NAD, resulting in increasing in absorbance at 340 nm.</H4></DIV><DIV class="col-md-12 col-sm-12" style="margin-top: 5rem"><SPAN class="w3-center w3-padding-large w3-wide w3-xlarge w3-animate-opacity">Result</SPAN></DIV><DIV class="col-md-12 col-sm-12"><H4>As shown in Fig. 3, the growth of E. coli expressing the PTS reporter (i.e., crr and ptsG genes) was seriously retardant. As I mentioned earlier, the E. coli overexpressing glucose transporter may lose viability because of the toxic metabolites produced in glycolytic pathway. Not surprisingly, the PTS overexpression bacteria was almost unable to absorb glucose. </H4></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><SPAN style="padding-left: 6rem"><STRONG>Fig 3. The cell growth overnight and glucose uptake of E. coli expressing the PTS transporter in 4 hours </STRONG></SPAN></DIV></DIV><DIV class="col-md-12 col-sm-12" style="padding-top: 3rem; padding-bottom: 3rem"><H4>Fig. 4 represented that the cell growth of E. coli expressing the Na+/Glucose transporter was comparable and even slightly higher than the control group. The glucose began to be absorbed at the 3rd hour. The glucose uptake efficiency was achieved upto 97% in Na+/Glu group and greater than in control group with 1.2 times difference.</H4></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><STRONG style="padding-left: 6rem">Fig 4. The cell growth overnight and glucose uptake of E. coli expressing the Na+/Glu transporter at different time point</STRONG></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12 w3-padding" style="display: block; margin: auto"><DIV class="card mb-3 bg-primary"><DIV class="card-header text-center"><STRONG style="color: #fff; letter-spacing:0.5rem;">Discussion</STRONG></DIV><DIV class="card-body bg-light"><H4>We’ve successfully genetically engineered E. coli expressing high-affinity active PTS transporter system and low-affinity facilitated Na+/glucose cotransporter system by screening in MacConkey and glycerol agar plate with chloramphenicol, respectively. Our data is consistent with the previous study that overexpressing glucose transporter genes do really harm E. coli partly due to disturbing glycolytic pathways.</H4><H4>PTS expressing bacteria grew difficultly in LB broth but Na+/glucose transporter expressing bacteria grew comparably with general E. coli, suggesting that high-affinity active transporter did interact with sugar-related substances in LB culture media and harm the bacterial survival.</H4><H4>In our results, glucose absorption by Na+/glucose transporter expressing bacteria was achieved to 97% after 4 hours with 1.2 times enhanced efficiency compared to the normal E. coli. To further increase the rate of glucose uptake, one may think about the glucose metabolism or conversion to other materials when entering into the cell.</H4></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-12 col-sm-12"><DIV style="margin-left: 20px;"><DIV style="margin-right: 50px"><UL class="list-group"><LI class="list-group-item w3-wide" style="text-align: center"><STRONG>- REFERENCE -</STRONG></LI><LI class="list-group-item" style="color: black;">1.     Two mechanisms for growth inhibition by elevated transport of sugar phosphates in Escherichia coli. J Gen Microbiol. 1992;138(10):2007-14. </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></SECTION></DIV><TITLE>iGEM Human Practice</TITLE><DIV class="w3-container w3-black w3-padding-16" style="margin-left: 180px"><DIV class="w3-row"><DIV class="w3-container w3-third"><H3 class="w3-bottombar w3-border-blue w3-wide">
			Project
		</H3><H4 style="letter-spacing: 0.1rem; "><A href="https://2017.igem.org/Team:Mingdao/Demonstrate#research-md" style="text-decoration: none; color: white">Research</A></H4><H4 style="letter-spacing: 0.1rem;"><A href="https://2017.igem.org/Team:Mingdao/Demonstrate#design-md" style="text-decoration: none; color: white">Experiment</A></H4><H4 style="letter-spacing: 0.1rem; "><A href="https://2017.igem.org/Team:Mingdao/Model" style="text-decoration: none; color: white">Model</A></H4><H4 style="letter-spacing: 0.1rem;"><A href="https://2017.igem.org/Team:Mingdao/Applied_Design" style="text-decoration: none; color: white">Prototype</A></H4></DIV><DIV class="w3-container w3-third"><H3 class="w3-bottombar w3-border-white">  Human Practice
		</H3><H4 style="letter-spacing: 0.1rem; color: white;"><A href="https://2017.igem.org/Team:Mingdao/HP/Gold_Integrated" style="text-decoration: none; color: white">Gold</A></H4><H4 style="letter-spacing: 0.1rem;"><A href="https://2017.igem.org/Team:Mingdao/Silver" style="text-decoration: none; color: white">Silver</A></H4><H4 style="letter-spacing: 0.1rem;"><A href="https://2017.igem.org/Team:Mingdao/Collaborations" style="text-decoration: none; color: white">Collaborate</A></H4><H4 style="letter-spacing: 0.1rem;"><A href="https://2017.igem.org/Team:Mingdao/Engagement" style="text-decoration: none; color: white">Engagement</A></H4></DIV><DIV class="w3-container w3-third"><H3 class="w3-bottombar w3-border-blue w3-wide">
			Achievements
		</H3><DIV class="w3-container"><DIV class="w3-third"><H4 style="letter-spacing: 0.1rem; color: white;"><A href="https://2017.igem.org/Team:Mingdao/Achievements#intro" style="text-decoration: none; color: white">Medals</A></H4><H4 style="letter-spacing: 0.1rem; color: white;"><A href="https://2017.igem.org/Team:Mingdao/Achievements#special" style="text-decoration: none; color: white">Prizes</A></H4></DIV></DIV></DIV></DIV><DIV class="copyright" style="text-align: center">© iGEM Mingdao 2017.  Design: Kevin Li.  All rights reserved.
        </DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>