Team:Mingdao/Part Collection
iGEM Human Practice
Home Achievements Project Human-Practice Notebook Attributions
Dashboard
Basic
Composite
Improvements
Collection
  Close Menu
Overview Recombination Suicide Transporter
Overview
Background
To engineer Lactobacillus acidophilus by chromosome integration through homologous recombination, the selection of integration site is very important for successful recombination and not disturbing bacterial internal metabolism.
Based on the method created by Grace L. Douglas and Todd R. Klaenhammer1, the region between slpA gene (LBA0169) stop codon and the terminator was chosen as the intergenic insertion location.  The gene of slpA encodes a surface-layer protein with a strong constitutive promoter activity, which can also drive the expression of the inserted gene.
Fig 1. The schematic diagram of gene integration location (modified from Appl Environ Microbiol. 2011)
1    Recombination Vector – pLBA169
To make pLBA169, the downstream region of LBA0169 and the upstream region of the terminator were amplified by PCR. Then, the two PCR-amplified DNA fragments were ligated to pSB1C3 by tripartite ligation.
Next, CP29-RBS-aeBlue was cut from Part: BBa_K1033280. And this gel isolated DNA fragment was further inserted between two recombination regions on pSB1C3. This process created the standard EcoRI-XbaI-PART-SpeI-PstI assembly position.
To delete the extra SpeI recognition site within the one of the recombination region, we performed site-directed mutagenesis to change one nucleotide of SpeI recognition sequence. The final product has been confirmed by DNA sequencing.
You can refer to the following flow chart of gene cloning of the recombination vector.
- REFERENCE -
1.     Directed chromosomal integration and expression of the reporter gene gusA3 in Lactobacillus acidophilus NCFM. Appl Environ Microbiol. 2011;77(20):7365-71.
Part No.  : BBa_K2230000
Type: Composite part (device)
Cloning:
Firstly, the elements of upstream and downstream recombination sites were replicated from gDNA of Lactobacillus acidophilus and put onto pSB1C3 as a L. acidophilus recombination vector, named pLBA169, at EcoRI and PstI locations. Secondly, to generate as a standard BioBrick part and assembly vector, the part of Promoter CP29-RBS-aeBlue (BBa_K1033280) was cut by EcoRI and PstI and assembled with the vector. Finally, because there’s a SpeI recognition sequence within the 169dn element, site-directed mutagenesis was performed to change a nucleotide to leave a single SpeI site in the BioBrick suffix region. (See the flow chart and data below).
Function:
A vector can be used to transform Lactobacillus acidophilus by chromosomal homologous recombination at the downstream location of slpA (LBA0169), which encodes a surface layer protein A. The aeBlue gene, which is driven by the wide host range and high constitutive promoter CP29, acts as a reporter. The transformed L. acidophilus will express blue color proteins.
Gene cloning for LBA169
Gene cloning for CP29-RBS-aeBlue/pLBA169
Part No.  : BBa_K2230001
Type: Basic part
Cloning:
EmR (Erythromycin resistance gene) was amplified from pMG36e vector and cloned onto pSB1C3.
Part No.  : BBa_K2230002
Type: Composite part
Cloning:
EmR (Erythromycin resistance gene) was amplified from pMG36e vector and cut by XbaI & PstI. Then, the fragment was assembled with RBS/pSB1A2 (BBa_B0034) cut by SpeI & PstI
Erythromycin Resistance Gene as a Selection Marker
Erythromycin is a common selection marker for engineering lactic acid bacteria1. And teams Uppsala in 2013 and TMMU_China in 2016 were using it to select the engineered Lactobacillus with erythromycin resistant strains.
However, the BioBrick parts containing the erythromycin resistance gene are currently unavailable. On the other hand, iGEM HQ is unable to service the request of plasmid carrying erythromycin resistance gene. Maybe iGEM HQ can’t manipulate erythromycin resistant bacteria so far.
Therefore, we decided to clone the erythromycin resistance gene (EmR) by ourselves. We searched the vector carrying EmR and found an iGEM team HKUST in 2010 working with it. We got the pMG36e vector from the team and cloned the EmR gene out onto pSB1C3 (EmR/pSB1C3, BBa_K2230001). We also made RBS-EmR/pSB1C3 (BBa_K2230002) and RBS-EmR-TT/pSB1C3 (BBa_K2230003) by assembling RBS (B0034) and double terminator (B0015) parts, respectively. These will provide iGEM team this resistance gene as selection marker in the future.
- REFERENCE -
1.     Genetic engineering techniques for lactic acid bacteria: construction of a stable shuttle vector and expression vector for β-glucuronidase. Biotechnol Lett. 2014;36(2):327-35.
Part No.  : BBa_K2230003
Type: Composite part
Cloning:
The fragment of RBS-EmR of RBS-EmR/pSB1C3 (BBa_K2230002) cut by EcoRI & SpeI was gel eluted and assembled with TT/pSB1C3 (Double terminator, BBa_B0015) cut by EcoRI & XbaI.
Part No.  : BBa_K2230004
Type: Composite part (device)
Cloning:
RBS-EmR which was cut and gel-eluted from RBS-EmR/pSB1C3 (BBa_K2230002) was assembled with CP29-RBS-aeBlue/pLBA169 (BBa_K2230000)
Function:
A vector can be used to transform Lactobacillus acidophilus by chromosomal homologous recombination at the downstream location of slpA, which encodes a surface layer protein A. EmR (erythromycin resistance gene), which is driven by the high constitutive promoter of slpA, acts as a selection marker. And aeBlue gene, which is driven by the wide host range and high constitutive promoter CP29, acts as a reporter. The transformed L. acidophilus will be erythromycin resistant and express blue color proteins.
Background
Promoter Pcar [BBa_K861171] is a glucose responsive promoter created by WHU-China in 2012. Pcar promoter region was de novo  designed with overlapping of CRP and RNA polymerase binding site. The initiation of transcription by RNA polymerase may be hindered by the binding of CRP, which occurs at the formation of cAMP-CRP complex in the low concentration of glucose. In other words, when the amount of glucose is high enough, Pcar would be turned on after the leaving of CPR due to the low concentration of cAMP, and vice versa.
PI promoter [BBa_K861170] is a modified version to Pcar with correction of -10 position in the promoter region, which gives the promoter stronger activity and a weaker CRP interaction. The PI promoter [BBa_K861170] was used as a major part in the work of team NCKU_Tainan in 2016. These two parts have been used in our work.
PhlF repressor system contains the repressor PhlF [BBa_K1725041] and the PhlF repressible promoter [BBa_K1725001] created by Glasgow in 2015. PhlF could repress GFP fluorescence intensity by 83-fold according to the study of Glasgow’s work.
We’ve innovated this year a novel glucose responsive repressor system (Pcar-wRBS-PhlF-T-Pr-sRBS-GFP/pSB1C3 [BBa_K2230012]) by connecting these two system and extend the function of them. Furthermore, based on this new system, we assembled lysis and nuclease genes to the device and created the suicide circuit controlled by the presence of glucose (Pcar-wRBS-PhlF-T-Pr-sRBS-GFP-sRBS-lysis-sRBS-NucA/pSB1C3 [BBa_K2230017]).
Lysis gene [BBa_K117000] created by NTU-Singapore in 2008 encodes Lysis protein which could not only lyse bacterial cell membrane but also activate the endonuclease of Colicin E7 (ColE7). The lysis-colicin is one class of bacteriocins which are produced to response to worsening environmental conditions and outcompete other bacteria1.
NucA [BBa_K1159105] created by TU-Munich in 2013 from Staphylococcus aureus produces a thermostable exo- and endo-nuclease that is able to degrade genomic DNAs2. NucA also has a role in the cleavage of extracellular DNAs and preventing biofilm formation.
The Parts
Part No.  : BBa_K2230005
Type: Composite part (device)
Cloning:
PI-RBS-RFP was amplified from RFP Coding Device (BBa_J04450) using a primer with PI-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
Function:
RFP can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 & NCKU_Tainan 2016, the promoter, PI, is glucose responsive.
Part No.  : BBa_K2230006
Type: Composite part (device)
Cloning:
Pcar-RBS-RFP was amplified from RFP Coding Device (BBa_J04450) using a primer with Pcar-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
Function:
RFP can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 & NCKU_Tainan 2016, the promoter, Pcar, is glucose responsive.
Part No.  : BBa_K2230007
Type: Composite part (device)
Cloning:
PI-RBS-aeBlue was amplified from CP29-RBS-aeBlue (BBa_K1033280) using a primer with PI-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
Function:
aeBlue can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 & NCKU_Tainan 2016, the promoter, PI, is glucose responsive.
Part No.  : BBa_K2230008
Type: Composite part (device)
Cloning:
Pcar-RBS-aeBlue was amplified from CP29-RBS-aeBlue (BBa_K1033280) using a primer with Pcar-RBS (B0034) sequence. The PCR product was ligated to pSB1A3 cut by XbaI and PstI.
Function:
aeBlue can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 & NCKU_Tainan 2016, the promoter, Pcar, is glucose responsive.
Part No.  : BBa_K2230009
Type: Composite part (device)
Cloning:
PI-RBS-PhlF-T was amplified from RBS + PhlF repressor + terminator (BBa_K1725041) using a primer with PI-RBS (B0032) sequence. The PCR product was ligated to pSB1C3 cut by XbaI and PstI.
Function:
Repressor PhlF can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 & NCKU_Tainan 2016, the promoter, PI, is glucose responsive.
Part No.  : BBa_K2230010
Type: Composite part (device)
Cloning:
Pcar-RBS-PhlF was amplified from RBS + PhlF repressor + terminator (BBa_K1725041) using a primer with Pcar-RBS (B0032) sequence. The PCR product was ligated to pSB1C3 cut by XbaI and PstI.
Function:
Repressor PhlF can be dose-dependently expressed in an increasing concentrations of glucose. According to teams WHU-China 2012 & NCKU_Tainan 2016, the promoter, Pcar, is glucose responsive.
Part No.  : BBa_K2230011
Type: Composite part (device)
Cloning:
Pr-sRBS-GFP (BBa_K1725001) with strong RBS (B0034) was assembled with PI-RBS-PhlF-T/pSB1C3 (BBa_K2230009).
Function:
The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
Part No.  : BBa_K2230012
Type: Composite part (device)
Cloning:
Pr-sRBS-GFP (BBa_K1725001) with strong RBS (B0034) was assembled with Pcar-RBS-PhlF-T/pSB1C3 (BBa_K2230010)
Function:
The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
Part No.  : BBa_K2230013
Type: Composite part (device)
Cloning:
Pr-wRBS-GFP (BBa_K1725002) with weak RBS (B0032) was assembled with PI-RBS-PhlF-T/pSB1C3 (BBa_K2230009)
Function:
The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
Part No.  : BBa_K2230014
Type: Composite part (device)
Cloning:
Pr-wRBS-GFP (BBa_K1725002) with weak RBS (B0032) was assembled with Pcar-RBS-PhlF-T/pSB1C3 (BBa_K2230010)
Function:
The expression of GFP driven by PhlF-repressed promoter (named Pr here) was dose-dependently regulated by PhlF repressor and reduced upon an increasing concentrations of glucose. In other words, the intensity of GFP increased when glucose gradually ran out.
Part No.  : BBa_K2230015
Type: Composite part (device)
Cloning:
wRBS-lysis was amplified from Lysis gene (BBa_K117000) using a primer with weak RBS sequence (B0032) and assembled with Pcar-wRBS-PhlF-T-Pr-sRBS-GFP/pSB1C3 (BBa_K2230012)
Function:
Suicide genes of lysis was driven under the PhlF-repressed promoter (Pr). And the expression of the repressor (PhlF) would be induced upon glucose by a glucose responsive promoter (Pcar). That is, the bacteria can be killed by the suicide genes in the absence of (or running out of) glucose.
Part No.  : BBa_K2230016
Type: Composite part (device)
Cloning:
sRBS-lysis was amplified from Lysis gene (BBa_K117000) using a primer with strong RBS sequence (B0034) and assembled with Pcar-wRBS-PhlF-T-Pr-sRBS-GFP/pSB1C3 (BBa_K2230012)
Function:
Suicide genes of lysis was driven under the PhlF-repressed promoter (Pr). And the expression of the repressor (PhlF) would be induced upon glucose by a glucose responsive promoter (Pcar). That is, the bacteria can be killed by the suicide genes in the absence of (or running out of) glucose.
Part No.  : BBa_K2230017
Type: Composite part (device)
Cloning:
sRBS-NucA was amplified from Mature Nuclease NucA from Staphylococcus aureus (BBa_K1159105) using a primer with strong RBS sequence (B0034) and assembled with Pcar-RSB-PhlF-T-Pr-sRBS-GFP-sRBS-lysis/pSB1C3
Function:
Suicide genes of lysis and nuclease (NucA) were driven under the PhlF-repressed promoter (Pr). And the expression of the repressor (PhlF) would be induced upon glucose by a glucose responsive promoter (Pcar). That is, the bacteria can be killed by the suicide genes in the absence of (or running out of) glucose.
Experiment and Results
1    Glucose responsive promoter activity
The result shown in Fig. 1 indicated that PI promoter has significant activity in LB culture media. However, the activity of Pcar promoter is greater than negative control but much smaller than PI and positive control. It’s consistent with the properties of PI and Pcar promoters just mentioned previously in GENE CLONING section and described previously in Part Resgistry [Part: BBa_K861170] by team WHU-China in 2012 who designed the promoters.
In our experiment as presented in Fig. 2, PI and Pcar promoters just responded to various concentrations of glucose with a  very slight dose-dependent increase. This phenomenon didn’t correspond to the data provided by team WHU-China in 2012 and team NCKU_Tainan in 2016. Maybe our measurement was not in an optimized condition. Or the reporter of RFP activity was not sensitive enough to respond this difference.
2  Functional assay for Glucose Responsive Suppressor System
The bacteria carrying the indicated vector were cultured in LB media supplemented with 34 μg/ml of chloramphenicol (Cm) at 37°C overnight. The next day, OD600 was measured and adjusted to 2.5 in M9 minimal media with various concentrations of glucose. The bacteria then were incubated for 4 hours at 37°C. 100 μl of the bacterial culture was put into one well of a black-walled, clear-bottom 96-well microplates (Thermo Fisher Scientific Inc.). The fluorescence intensity was measured using BioTek Synergy H1 Hybrid Multi-Mode Reader System at Ex/Em = 488nm/518nm  for GFP.
In the assay for repressor system, the data in Fig. 4 gave the similar results as team Glasgow did in 2015, in which the strong activity of the repressible promoter was significantly repressed in the presence of PhlF repressor.
In the assay for glucose responsive repressor system, we improved the Glasgow’s BioBrick device, in which the expression of PhlF repressor was driven by a glucose response promoter, Pcar. The result in Fig 5 clearly indicated that the GFP activity driven by the repressible promoter was gradually increased in response to the loss of glucose to 1.88 folds compared to the initial GFP activity at the beginning culture in M9 media, suggesting that the level of expression of PhlF was positively corresponding to the concentration of glucose.
3    Functional assay for glucose responsive suicide circuit
The bacteria carrying the indicated vector were cultured in LB media supplemented with 34 μg/ml of chloramphenicol (Cm) at 37°C overnight. The next day, OD600 was measured and adjusted to 2.5 in M9 minimal media with various concentrations of glucose. The bacteria then were incubated for 4 hours at 37°C. 100 μl of the bacterial culture was put into one well of a black-walled, clear-bottom 96-well microplates (Thermo Fisher Scientific Inc.). OD600 was measured using BioTek Synergy H1 Hybrid Multi-Mode Reader System. Further,  the culture media were taken out and diluted 106 times following by spreading onto LB Cm agar plate at 37°C overnight. The third day, the numbers of colonies were counted and bacterial viability was calculated.
As you can see in Fig 7., the OD value in response to the decreasing concentration of glucose was gradually reduced to 1.89 much less than average 2.71 in control group without suicide gene expression, implying that the suicide proteins killed the cells in the loss of glucose in the environment. Moreover, when the bacteria were grown in M9 media with 0.5mM glucose for 4 hours, the survival rate was decreased to 34% compared to 56% of bacteria without suicide genes (Fig. 8). And the cell numbers were reduced to 671 compared to 1120 of bacteria without suicide genes. Both data confirmed that this suicide device works well and indicated that killing process began when glucose in the media was running out.
Background
Salmonella typhimurium LT2 has two glucose-specific transporter systems, PTS system and sodium/glucose cotransporter. PTS system contains two subunits IIA encoded by crr and IIBC by ptsG which are assembled to a high-affinity active transporter. The other is a Na+/glucose cotransporter encoded by STM1128 that contributes to facilitated transport with lower glucose affinity. Based on our research, the glucose transporter of Salmonella has a lower Km compared to human small intestine, Staphylococcus and E. coli (Table 1), indicating a higher efficiency for glucose uptake. In our project, we created the glucose transporter device and genetically engineer microbes with these two systems.
In order to express the genes in E. coli for demonstration and in probiotics for proof-of-concept in a real world. We chose promoter CP29 that is a strong constitutive promoter working well in both E. coli and Lactobacillus spp1. The biobrick part, CP29-RBS-aeBlue (BBa_K1033280) was used and to be assembled with the transporter genes.
Part No.: BBa_K2230018
Type: Basic Part
Cloning:
The crr gene was amplified from gDNA of Salmonella typhimurium and cloned onto pSB1C3
Part No.: BBa_K2230019
Type: Composite Part
Cloning:
The crr gene was amplified from gDNA of Salmonella typhimurium using a primer with RBS sequence (B0034), and then cloned onto pSB1C3
Part No.: BBa_K2230020
Type: Basic Part
Cloning:
The ptsG gene was amplified from gDNA of Salmonella typhimurium and cloned onto pSB1C3
Part No.: BBa_K2230021
Type: Composite Part
Cloning:
The ptsG gene was amplified from gDNA of Salmonella typhimurium using a primer with RBS sequence (B0034), and then cloned onto pSB1C3
Part No.: BBa_K2230022
Type: Basic Part
Cloning:
The STM1128 gene was amplified from gDNA of Salmonella typhimurium and cloned onto pSB1C3
Part No.: BBa_K2230023
Type: Composite Part
Cloning:
The STM1128 gene was amplified from gDNA of Salmonella typhimurium using a primer with RBS sequence (B0034), and then cloned onto pSB1C3
Part No.: BBa_K2230024
Type: Composite Part
Cloning:
The RBS-ptsG part (BBa_K2230021) was assembled with Double Terminator/pSB1C3 (BBa_B0015)
Part No.: BBa_K2230025
Type: Composite Part
Cloning:
The RBS-STM1128 part (BBa_K2230023) was assembled with Double Terminator/pSB1C3 (BBa_B0015)
Part No.: BBa_K2230026
Type: Composite Part
Cloning:
The RBS-crr part (BBa_K2230019) was assembled with RBS-ptsG-TT/pSB1C3 (BBa_K2230024)
Part No.: BBa_K2230027
Type: Composite Part (device)
Cloning:
Promoter CP29-RBS-aeBlue (BBa_K1033280) was assembled with RBS-crr-RBS-ptsG-TT/pSB1C3 (BBa_K2230026)
Function:
An active high-affinity glucose transporter system. Promoter CP29 is a constitutive and strong promoter which works well in both E. coli and Lactobacillus spp. The blue protein encoded by aeBlue gene acts as indicator for the gene expression in the transgenic strain. This device can help E. coli and Lactobacillus spp. efficiently take up and retrieve glucose in the environment.
Part No.: BBa_K2230028
Type: Composite Part (device)
Cloning:
Promoter CP29-RBS-aeBlue (BBa_K1033280) was assembled with RBS-STM1128-TT/pSB1C3 (BBa_K2230025)
Function:
A facilitated low-affinity glucose transporter system. Promoter CP29 is a constitutive and strong promoter which works well in both E. coli and Lactobacillus spp. The blue protein encoded by aeBlue gene acts as indicator for the gene expression in the transgenic strain. This device can help E. coli and Lactobacillus spp. efficiently take up and retrieve glucose in the environment.
Experiment and Results
To measure glucose uptake by the engineered E. coli expressing PTS system or Na+/glucose cotransporter, the bacteria were culture in LB broth supplemented with 34μg/ml of chloramphenicol at 37°C overnight. The next day, the bacterial culture was adjusted to OD600 = 3 and exchanged with M9 minimal media with 20mM of glucose for 4 hours or at different time points.
Glucose concentration was analyzed with Glucose (HK) Assay Kit (Sigma-Aldrich) according to the manufacturer’s instruction. Briefly, glucose was phosphorylated (G6P) by hexokinase. Then G6P was further catalyzed by G6PDH and the reduced NAHD was formed from the oxidation of NAD, resulting in increasing in absorbance at 340 nm.
Result
As shown in Fig. 3, the growth of E. coli expressing the PTS reporter (i.e., crr and ptsG genes) was seriously retardant. As I mentioned earlier, the E. coli overexpressing glucose transporter may lose viability because of the toxic metabolites produced in glycolytic pathway. Not surprisingly, the PTS overexpression bacteria was almost unable to absorb glucose.
Fig 3. The cell growth overnight and glucose uptake of E. coli expressing the PTS transporter in 4 hours
Fig. 4 represented that the cell growth of E. coli expressing the Na+/Glucose transporter was comparable and even slightly higher than the control group. The glucose began to be absorbed at the 3rd hour. The glucose uptake efficiency was achieved upto 97% in Na+/Glu group and greater than in control group with 1.2 times difference.
Fig 4. The cell growth overnight and glucose uptake of E. coli expressing the Na+/Glu transporter at different time point
Discussion
We’ve successfully genetically engineered E. coli expressing high-affinity active PTS transporter system and low-affinity facilitated Na+/glucose cotransporter system by screening in MacConkey and glycerol agar plate with chloramphenicol, respectively. Our data is consistent with the previous study that overexpressing glucose transporter genes do really harm E. coli partly due to disturbing glycolytic pathways.
PTS expressing bacteria grew difficultly in LB broth but Na+/glucose transporter expressing bacteria grew comparably with general E. coli, suggesting that high-affinity active transporter did interact with sugar-related substances in LB culture media and harm the bacterial survival.
In our results, glucose absorption by Na+/glucose transporter expressing bacteria was achieved to 97% after 4 hours with 1.2 times enhanced efficiency compared to the normal E. coli. To further increase the rate of glucose uptake, one may think about the glucose metabolism or conversion to other materials when entering into the cell.
- REFERENCE -
1.     Two mechanisms for growth inhibition by elevated transport of sugar phosphates in Escherichia coli. J Gen Microbiol. 1992;138(10):2007-14.
iGEM Human Practice
Project
Research
Experiment
Model
Prototype
  Human Practice
Gold
Silver
Collaborate
Engagement
Achievements
Medals
Prizes
© iGEM Mingdao 2017.  Design: Kevin Li.  All rights reserved.
