<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UESTC-software_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UESTC-software/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>三级页面</TITLE></P><DIV class="header-content"><NAV class="main-nav"><UL><LI><A href="https://2016.igem.org/Team:UESTC-software">HOME</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Project?id=0">PROJECT</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Description">Description</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Features">Features</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Model">Modeling</A></LI><LI class="three-nav"><A href="https://2016.igem.org/Team:UESTC-software/Proof">Proof</A></LI><LI class="three-nav"><A href="https://2016.igem.org/Team:UESTC-software/Demonstrate">Results</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Future">Future</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Parts">Parts</A></LI><LI class="three-nav"><A href="https://2016.igem.org/Team:UESTC-software/Extra_work">Extra Work—Bio2048</A></LI></UL><LI><A href="https://2016.igem.org/Team:UESTC-software/Judging?id=1">JUDGING</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Medal_requirements">Medal Requirements</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Safety?">Safety</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Team?id=2">TEAM</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Members?id=2&amp;index=0">Team</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Collaborations">Collaborations</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Notebooks">Notebooks</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/HP">HUMAN PRACTICES</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Integrated_Practices">Integrated Practices</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Engagement">Engagement</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Document">DOCUMENTS</A></LI><LI><A href="https://2016.igem.org/Team:UESTC-software/Attributions">ATTRIBUTIONS</A></LI></NAV></DIV><DIV class="content-top"><P class="title">Design</P></DIV><DIV class="detail-content"><P>The design of Bio101 involves two aspects, the algorithm for encoding information in DNA sequences and the software implementation of the algorithm.</P><H2 id="System Design">System Design</H2><STRONG>Workflow</STRONG><P>The overall workflow of <B>Bio101:DNA Information Storage System</B> is shown below (Fig. 1). Given a computer file, we <B>compress</B> it first, and then <B>encrypt</B> the file with the ISAAC algorithm <SUP>[1]</SUP>. The resulting binary file is <B>converted to a nucleotide (NT) stream</B> through a direct map between two bits and one base. The long NT stream is then fragmentized into short sequences and file <B>indexing and error checking NTs</B> are added to the ends of the sequences.</P><P class="img-p" style="font-size: 13px; "><B>Fig.1.</B>Workflow for encoding and decoding processes.</P><P>When the encoding process is finished, the resulting DNA sequences file can be delivered to a DNA synthesis company for the physical write-in.</P><P>The decoding process involves the physical read-out through sequencing the DNA sample and the reverse steps of the encoding steps.</P><STRONG>Compression is the First Step</STRONG><P>Before the file is transformed to DNA sequences, a compression step is applied in order to achieve higher information storage density <SUP>[2]</SUP>. Instead of designing a new compression algorithm of our own, we choose the well-known, bzip2, algorithm for this purpose to avoid reinventing the wheel <SUP>[3]</SUP>. Shorter file length corresponds to fewer DNA sequences which means less cost for synthesis and sequencing. </P><STRONG>Encryption is the Key</STRONG><P>The purpose of the encryption step is two-fold. </P><P>First, for a good information storage system, <B>data safety</B> is essential. Encryption ensures that one who obtains the physical DNA samples and knows the encoding protocol still cannot crack the information without the correct password. </P><P>Second, the encoded DNA sequences should be as <B>artificial and random</B> as possible. This means that the sequences should be devoid of long homopolymers, such as GGGGGG, or repeated base patterns, such as AGTCAGTCAGTC, which are difficult to synthesize and sequence, and the sequences <B>should not possess any biological functions</B> so that the DNA samples are safe to people and environment. </P><P>The ISAAC encryption algorithm <SUP>[1]</SUP> is chosen for this step. Pseudo-random bit sequences are generated while the original information is encrypted.  The possibility to produce homopolymers, repeated base patterns or fragments with biological functions  is extremely low. </P><STRONG>Bits to Nucleotides </STRONG><P>The randomized bits stream is converted into nucleotides (NT) stream using the straightforward map between two bits and one NT, 00 -&gt; A, 01 -&gt; C, 10 -&gt;G, 11 -&gt; T  <SUP>[4]</SUP>. </P><STRONG>Indexing is Essential for Data Integrity </STRONG><P>DNA in nature is a long one-dimensional molecule. However, the current technology only allows us to synthesize short oligonucleotides while the long DNA molecule with billions of nucleotides is still nature’s privilege.  In order to store a large piece of information, we have to fragmentize a long DNA sequence into oligonucleotides with a few hundreds of bases. However, simply fragmentizing the sequence would destroy all the data, since the order of these fragments is unknown.</P><P>To overcome this problem, indexing the fragments is necessary. We add sequence address code and error check code to the original fragments. The address code tells the <B>position</B> of one sequence in the file, while the check code acts as a <B>‘parity NT’</B><SUP>[5]</SUP> to check for errors in the encoded information, including length, address code, etc. Further scrambling is applied to the sequences to avoid homopolymers. </P><P>In order to address the possible errors introduced in the synthesis, storage or sequencing, four-fold redundancy is used.</P><H2 id="DNA File Editing">DNA File Editing</H2><STRONG>Workflow</STRONG><P><B>Random access</B>, which means reading and writing a piece of information at arbitrary position in a file, is an important requirement for DNA-based information processing and computing system.  Although much effort has been invested to ensure the information integrity in the above design scheme, it is not suitable for the file editing purpose: many DNA fragments need to be replaced even just one bit is changed in the middle of a file.  </P><P>To address this need, we design an alternative encoding scheme as shown below (Fig. 2). In the scheme, the original file is fragmentized first. Then the fragment bit sequences are encrypted and randomized using the <B>ISAAC</B> algorithm followed by the conversion into NT sequences and the indexing step. </P><P class="img-p" style="font-size: 13px;"><B>Fig.2.</B>Workflow for DNA File Editing.</P><STRONG>DNA File Editing Operation</STRONG><P>To edit a particular part of the file, we need to locate the DNA fragment where the to-be-modified information is located. To replace the information, two operations are needed. <B>(1)</B> The original fragment needs to be broken. <B>(2)</B> The new information needs to be encoded with the same procedure and compatible indexing information with the original sequence. </P><P>The second step is not an issue since each piece of information is encoded independently in this new scheme and only the index needs special care.</P><P>To degrade the original fragment, we propose to use the <B>CRISPR/Cas9</B> system which has general editing ability through a recognition step using the single-guide RNA (sgRNA). For this purpose, the GG nucleotide pairs are included in the encoded sequences to ensure that there is a PAM site. Bio101 additionally provides a <B>BioBrick part</B>,  a sgRNA expression cassette to work with the Cas9 system to guide the degradation of the targeted DNA sequences.</P><P class="img-p" style="font-size: 13px;"><B>Fig.3.</B>DNA file editing process.</P><H2 id="Front-end">Front-end</H2><STRONG>Web-based</STRONG><P>Due to the rapid development of Internet, web is now used extensively throughout the whole world. Meanwhile, a web application can be used without any installation hassle. More importantly, a web application can be accessed from many different devices from conventional computers to smart phones running with all kinds of operating systems. Therefore, we developed our software based on a web application model for a wider audience. All the computational work is done on our server while the results are provided as a hyperlink for users to download. </P><STRONG>User-friendly Interface</STRONG><P>The interface of our webpage is concise. There are five pages in total: one home page for the description on the software, one encoding page for a user to upload a file to encode it, one decoding page for a user to upload a DNA sequences file to extract the information, one about page to describe the background and algorithm of the software and one edit page to modify segment.  Users can get familiar with the web-app very quickly. </P><P>To develop a cross-platform software, HTML, CSS, bootstrap <SUP>[6]</SUP>,  Owl carousel <SUP>[7]</SUP> and jQuery are integrated into the web framework. The webpage is designed with a modern visualization standard and compatible with smart phones and other mobile devices, as well as quick in response.</P><P class="img-p" style="font-size: 13px;"><B>Fig.4.</B>The interface of Bio101.</P><STRONG>Clear operation flow</STRONG><P>Our website has a clear operation flow as the following figure (Fig.5). When a user starts to encode a file, he or she will submit a file and a code as token to encrypt the file. The file will be compressed and encrypted after stored in our server. After that, process goes to encode it and user will go to the ‘Download’ page after the encoding process. User can choose txt, fasta or SBOL-xml <SUP>[8]</SUP> format to download the final DNA sequences. As for decoding, user will submit a file with DNA sequences which are encoded by our software and a code which is set when encoding the file. Decoding will start after the file being stored in our server. Then, the file will be decrypted by token and decompressed. After that, website will skip to ‘Download’ page, so user can download the decoded file.</P><P class="img-p" style="font-size: 13px;"><B>Fig.5.</B>Design process of Bio101.</P><H2 id="Back-end">Back-end</H2><STRONG>Different file formats supported</STRONG><P>Our software supports the transforming of all formats of files, including jpg, pdf, mp3, etc. So users can store virtually all kinds of computer files in DNA. On the other hand, we provide different formats of DNA sequences for users to download, including Text, FASTA and SBOL. </P><STRONG>C and Python Combined</STRONG><P>C language has high execution efficiency, crossing platform application, etc. Whereas Python holds a great promise for conciseness, extensibility, abundant library, etc. So, the two are combined in Bio101 to form an ideal environment. The encryption and bit2nt parts are handled in C language while the rest is in Python, to guarantee the efficiency and the extensibility of the program. </P><P class="img-p" style="font-size: 13px;"><B>Fig.6.</B>Bio101 combines Python and C programming language.</P><H2 id="Web Framework">Web Framework</H2><STRONG>Web Programming with Django</STRONG><P>The front-end and back-end are separated, which are connected with Django web framework. Django is a high-level Python web framework that facilitates rapid development and clean, pragmatic design <SUP>[9]</SUP>. It is also free and open source. When users upload a file to the server-side interface, the back-end works, and then a DNA sequences file will be returned for the users to download. Developers can easily improve the codes in the back-end without worrying about any conflict with present front-end codes. </P><H2 id="References">References</H2><UL><LI style="font-size:13px;">[1] Robert J. Jenkins Jr. (1996) ISAAC: a fast cryptographic random number generator FSE 1996: 41-49. Available from:<A href="http://burtleburtle.net/bob/rand/isaacafa.html" style="color: #439ea3;">http://burtleburtle.net/bob/rand/isaacafa.html  </A></LI><LI style="font-size:13px;">[2] CUHK IGEM (2010) Bioencryption [online]. Available from: <A href="https://2010.igem.org/Team:Hong_Kong-CUHK" target="_blank" style="color: #439ea3;">https://2010.igem.org/Team:Hong_Kong-CUHK</A></LI><LI style="font-size:13px;">[3]bzip2 [online] Available from:<A href=" http://www.bzip.org/" target="_blank" style="color: #439ea3;">http://www.bzip.org/ </A></LI><LI style="font-size:13px;">[4] Yim AK-Y, Yu AC-S, LiJ-W, Wong AI-C, Loo JFC, Chan KM, Kong SK, Yip KY and Chan T-F (2014) The essential component in DNA-based information storage system: robust error-tolerating module. Front. Bioeng. Biotechnol. 2:49. doi: 10.3389/fbioe.2014.00049 </LI><LI style="font-size:13px;">[5] Tabatabaei Yazdi, S. M. H. et al (2015) A Rewritable, Random-Access DNA-BasedStorage System. Sci. Rep. 5, 14138; doi: 10.1038/srep14138 </LI><LI style="font-size:13px;">[6] Bootstrap is the most popular HTML, CSS, and JS framework for developing responsive, mobile first projects on the web. [online] Available from:
<A href=" http://getbootstrap.com/" target="_blank" style="color: #439ea3;">http://getbootstrap.com/</A></LI><LI style="font-size:13px;">[7] OWL Carousel Touch enabled jQuery plugin that lets you create beautiful responsive carousel slider. [online] Available from:
<A href="http://owlgraphic.com/owlcarousel/" target="_blank" style="color: #439ea3;">http://owlgraphic.com/owlcarousel/</A></LI><LI style="font-size:13px;">[8] Synthetic Biology Open Language (SBOL) [online] Available from:
<A href="http://sbolstandard.org/" target="_blank" style="color: #439ea3;">http://sbolstandard.org/</A></LI><LI style="font-size:13px;">[9] Django: The web framework for perfectionists with deadlines. [online] Available from:
<A href="https://www.djangoproject.com/" target="_blank" style="color: #439ea3;">https://www.djangoproject.com/</A></LI></UL></DIV><FOOTER class="footer"><DIV class="footer-top"><P>FOLLOW US：
            </P><P>UESTC-SOFTWARE</P></DIV><DIV class="footer-bottom">© 2016 University of Electronic Science and Technology of China</DIV></FOOTER><DIV class="catalog"><SPAN>CATALOGUE</SPAN></DIV><DIV class="catalog-area"><DIV class="title">CATALOGUE</DIV><UL><LI><A href="#System Design">
        System Design
        </A></LI><LI><A href="#DNA File Editing">
            DNA File Editing
        </A></LI><LI><A href="#Front-end">
            Front-end
        </A></LI><LI><A href="#Back-end">
            Back-end
        </A></LI><LI><A href="#Web Framework">
            Web Framework
        </A></LI><LI><A href="#References">
            References
        </A></LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>