Team:UESTC-software/Collaborations
三级页面
HOME
PROJECT
Description
Design
Features
Modeling
Proof
Results
Future
Parts
Extra Work—Bio2048
JUDGING
Medal Requirements
Safety
TEAM
Team
Collaborations
Notebooks
HUMAN PRACTICES
Silver
Gold
Integrated Practices
Engagement
DOCUMENTS
ATTRIBUTIONS
Collaborations
Collaboration and sharing, the key to step forward
As iGEMers, we all know sharing and collaboration are core values of iGEM. If a team seal themselves off, not only will they be trapped in projects, but also cannot stand in a higher position to grasp the overall situation. Instead of cooping ourselves up, we actively cooperated together for the betterment of Synthetic Biology as a whole. UESTC-Software had collaborated with other 5 teams in 6 different ways:
(1) Mentoring and supporting TMMU-China, which is a new team.
(2) Solving problems for UESTC-China, our brother team.
(3) Wet-lab help from UESTC-China, which helped us with validation.
(4) Software testing by AHTU-China, which was the first user of Bio101.
(5) Having meetup with SCU-China, TMMU-China, UESTC-China and SICAU.
(6) Participating meetup with Nanjing-China organized by TMMU_China.
Support for TMMU_China
Background
This year, TMMU_China as a young team needed a lot of support from other experienced teams. iGEM encourages help and collaborations among teams, especially the new teams. Our team built a friendly connection with other iGEM teams like TMMU_China, and we provided some useful supports for them in order to contribute to iGEM the competition itself, and even to Synthetic Biology.
During our iGEM trip, we helped TMMU_China to build Protein Calculator, a web app to predict experiment result based on their modeling.
This programming work is time-consuming but helpful, we believe our effort can cut enormous workload and free their time so that they can focus on their experiments and research.
Main work
After communication and discussion about the workflow and details, our engineer group began to build and test the web app since our Chongqing Trip. We used Django to manage the web and put all necessary contents into HTML files, with related models and views in Python files.
The main work
1. Setup the Web;
2. Fullfill all need in front end;
3. Implement file upload and download function;
4. Turn TMMU’s Matlab software into C to handle input;
5. Implement the parameter passing and result display.
Fig.1.The file tree.
The web app
This web app, Protein Calculator, needs three inputs:
1. Experiment parameters
2. Cnism value
3. Time
The first one is fulfilling our standard file and uploading the experiment parameters, and then TMMU_China’s algorithm can plot a result curve to simulate the growing situation.
Then, users type the specific cnism value and demand time, they can get a specific function curve at this point.
The whole workflow is the combination of 3 modeling:
1. Gene expression model;
2. Colony growth model;
3. Protein diffuse model;
The details of their modeling are in TMMU_China wiki . You can also browse this web app in their modeling part as a symbol of our cooperation.
Fig.2.Modeling Visualization web interface.
Future
Cooperation is always the main topic of iGEM. The interface and logrolling can really make a difference in our work progress and the friendship between different teams.
We hope this friendship can live from iGEM2016 to a far future, and hope the web app stemming from this friendship can really help researchers to achieve their purpose easily.
Development of Data-Processing Tool for UESTC-China
Background
This team’s goal was to degrade plastic and produce isobutanol. As a result of the ever increasing serious issues on plastic pollution, greenhouse effect and energy shortage, UESTC-China came up with an idea to try solving these three problems. That was to use genetically modified bacteria to achieve their goal. With their work going on, there was a stumbling block related to data processing simulation.
Fig.3. The experiment of UESTC-China.
Needs
In the course of experiment, there was a stage where team members should complete the drawing of the enzyme activity curve, through which to compare the work effect of each plasmid vector, so as to select the best one. A sea of data made it difficult to finish the data processing by manual method. Owing to the lack of programming ability, they had to use procedural means to solve the problem which took up lots of their time and energy.
So they sought for our support of programming expertise. We carried forward the spirit of collaboration to solve their problems relying on our skills.
Solutions
In the communication process, our team members joined the lab to have an idea of what we could contribute. Through the analysis of the process and the dynamic combination of the varied data number, we decided to design an algorithm for the fast simulation and solve the complex work with MATLAB.
1.  We batched import data collected in the experiment to MATLAB to generate a matrix, which can give the results they need quickly and accurately. Besides, it can generate EXCEL files automatically to help them do further analysis.
2.  Another function is that the system can find out large data errors and remove these bad data. In this way, they don’t have to manually check, greatly improving the efficiency even more.
3.  On this basis, we also provided a further function of curve fitting. With smooth curve instead of scattered data points, it makes the results more intuitive, easier to understand and analyze.
Some results are shown in Fig.4 and Fig.5:
Fig.4. Enzyme activity data simulation.
Fig.5. Enzyme activity data analysis.
The program we designed helped UESTC-China reduce the time of data processing, made them come to conclusions and start next step as soon as possible.
Wet-lab help from UESTC-China
We cooperated with the team UESTC-China to accomplish the validation of our experiments. We thank our collaborator for helping us with patience and making such strong contribution to our work.
We created a platform for information storage in synthesized DNA. This coding tool can create a convenient DNA storage information workflow, so researchers can choose any file they want to encode and focus on synthesizing DNA. And we wanted to demonstrate our tools’ capabilities under real-world condition. So UESTC-China mentored us on how to design the gene sequence which contains the suitable restriction site. They helped us amplify the genes and put the genes into the plasmids. UESTC-China also transformed the plasmids into the E. coli TOP10 and extracted them from E. coli for sequencing.
Fig.6.The DNA substance.
It’s hard for us to finish such a job. With their help, we finally validated the software we created can perform its intended function. Once again, we would like to thank UESTC-China.
Software testing by AHTU_China
In the dry-lab testing of our project, we invited team AHUT_China to help us accomplish the task together. As the developer, we tested our project to prove it had the intended function. Further, we needed someone who knew little about the development of Bio101 to do a more objective testing. Team AHUT_China tested the fault tolerance and randomness of Bio101 as our first user and offered us with a software testing report. Besides, they analyzed bzip2 compression algorithm we used in Bio101 through building mathematical model.
Fig.7.A group photo of video meeting with AHUT_China.
We also kept in touch to get timely feedback on the problems arising from the operation of Bio101. On September 16th, 2016, we had a video conference with AHUT_China. We both introduced our projects to increase interaction and understanding. Their project for this year is about BIO-COMPASS2.0, which is used to get optimal solution of path planning through bio navigation. Before putting into practice, extracting and storing information of a region in the map are needed, and our Bio101 can bring about convenience to their database storage and ensure their applications into reality, which is an exploration during our cooperation. Most importantly, we feel very grateful for what they have done for us.
iGEM Southwest China Union Meetup
We are developing and growing
Last year, UESTC iGEMers led the establishment of iGEM SouthWest Union of China (iSWU), which is a sustained platform for all the iGEMers in the southwest region of China. Nowadays, iSWU has a markedly increase in the number as well as the kind of teams.
This year, there were four universities and one organization interested in synthetic biology participated in the iSWU communication held in Sichuan University. The meetup focused on the theme, “To promote the communication and cooperation among iGEM teams in the southwest region of China” and the main activity forms were to discuss teams’ projects and to share experience. It gave all teams an opportunity to achieve a deeper understanding and exchanging.
Fig.8. A group photo of iSWU.
We are recognized and encouraged
In communication meeting, professors impressed the innovation awareness and excellent iGEM quality on us. Yun Zhao, the vice-president of SCU college of life sciences, stressed that there is no need to mention first time in science, all science to do is the first time. Nianhui Zhang, the coach of SCU, shared the experience of his iGEM team leading trip and affirmed we were valued. His word encouraged young scholars to research on innovation and fight for great achievements.
Fig.9.Our team’s presentation.
We are exchanging and improving
With excellent and lively presentation, all the teams were aroused in passionate discussions and got numerous responses from others, including suggestions on the thoughts of project, detailed Q&A on integrated design and reflection on application. As the unique software team, we concisely described the project we had completed and the improvement in the following work. Many members showed their interests in our project. For Bio101, some asked the DNA storage capacity and the superiority of the algorithm we used. One of the professor suggested us to take subsection coding strategy. Others played the game, Bio2048, and hoped us to help them complete the extra work such as biological game. These precious feedbacks we got from the meetup led us to what we should focus on.
Fig.10. We shared with other teams.
Modern technology makes information transmission more convenient, but there is no other way better than face-to-face communication. iSWU meetup helped the teams acquire reflections, inspirations and enhance efficiency to perfect the projects. Thus it can be seen, the communication will make a significant improvement for all attendees in the Giant Jamboree.
Meet-up with TMMU_China and Nanjing-China
Just a month away from the Giant Jamboree, TMMU-China invited UESTC-Software, UESTC-China and Nanjing-China to gather in Chongqing. They arranged a meet-up and aimed to prepare for the final presentation in Boston.
Fig.11. The group photo in TMMU-China.
The same experience, the same mood
As an experienced team, we were full of enthusiasm to share our abundant experience of competition and provide guidance for them. TMMU-China is in unclear condition about iGEM as we once were. We could empathize with them deeply. Therefore, we hoped to collaborate with them to improve their courage.
Fig.12.The presentation of TMMU-China.
The different styles, the different feels
This meet-up was completely consistent to the final competition process, with 15-20 minutes for presentation, question time and poster introduction. Although the scale was not as formal as the final event, it was really a good opportunity to feel different styles of presentation. We exchanged the final preparation for the competition, discussed the summary of the achievement, and found the problems existing in many aspects.
Fig.13. We communicated with the teachers from TMMU-China and Nanjing-China
The new collaboration, the new harvest
Significantly, it was our first time to contact with Nanjing-China. We learned from their indefatigable efforts. They were willing to cooperate with other teams and visit different universities to make improvement. They focused on public awareness about the spread of synthetic biology knowledge. We are making the same effort.
Fig.14. The introduction from Nanjing-China
The meet-up was held on the basis of project’s completion, so we inspired each other in all aspects. With such a good platform for collaboration, we can help others and improve ourselves at the same time, become less blind and less helpless. We got the inspiration from sharing, and we attained the courage from communicating. We would like to thank TMMU_China to design primer for the DNA sequences generated by Bio101. Their efforts provided basis for synthesizing DNA sequences.
What we harvest from the collaboration
Through all the collaborations we have taken part in, we learned a lot and shared happiness. We knew collaboration, competition and communication deeper and better. What we acquired was not only about the iGEM competition itself, we made friends and expanded our eyesight. We communicated face to face to understand what they did and let them know what we are doing, through which we improved our own abilities. And we encouraged and competed each other to hold on the scientific trip, which made us know about the importance of the competitors and collaborators to scientific workers. All these memories will be a unique part of iGEM experience as well as our scientific life experience.
FOLLOW US：
UESTC-SOFTWARE
© 2016 University of Electronic Science and Technology of China
CATALOGUE
CATALOGUE
Support for TMMU_China
Development of Data-Processing Tool for UESTC-China
Wet-lab help from UESTC-China
Software testing by AHTU_China
iGEM Southwest China Union Meetup
Meet-up with TMMU_China and Nanjing-China
What we harvest from the collaboration
