Team:UESTC-software/Parts
threeLevel
HOME
PROJECT
Description
Design
Features
Modeling
Proof
Results
Future
Parts
Extra Work—Bio2048
JUDGING
Medal Requirements
Safety
TEAM
Team
Collaborations
Notebooks
HUMAN PRACTICES
Silver
Gold
Integrated Practices
Engagement
DOCUMENTS
ATTRIBUTIONS
Parts
Background
In our project, the editing protocol of a DNA-based file involves the synthesis of a new DNA fragment encoding the new information and the breakage of the old DNA fragment encoding the original information. While the former can be realized easily through the chemical synthesis technique, the latter is not simple until the recent development of CRISPR genome editing method.
Restriction endonuclease can discern special DNA site and cut the particular DNA frag-ments with the endonuclease site, the constraint is that the DNA fragment which needs breakdown must contain the restriction endonuclease recognition site.  But for the DNA information storage system, a DNA fragment may contain arbitrary nucleotide sequences.
Cas9 (CRISPR associated protein 9) is a RNA-guided DNA endonuclease enzyme[2]. S. py-ogenes utilizes Cas9 to discern and cleave foreign DNA in its immunity system[3]. Cas9 binding site recruits Cas9 form sgRNA-Cas9 complex, the DNA recognition site interro-gates dsDNA specifically and inducts sgRNA-Cas9 complex combine with dsDNA so that generate sgRNA-Cas9-dsDNA complex. The combined Cas9 cleaves dsDNA and causes devastating damage to the dsDNA and lead to further hydrolyzation(Alec AK Nielsen & Christopher A Voigt*. Multi-input CRISPR/Cas genetic circuits that interface host regulatory networks).
Since Cas9 could cleave any DNA fragment in theory if there is a right sgRNA to guide the recognition process, it is the right tool for DNA information editing.
Fig.1.guid DNA transcribes sgRNA to guid Cas9[1].
Part Design
To degrade a particular DNA fragment, we need to design a part to express the sgRNA transcript to guide the Cas9 system to cleave the fragment.
nspired by Voigt’s study and the BioBrick parts (BBa_K1723002, BBa_K1723003 and BBa_K1723004) designed by the team of iGEM15_EPFL[4], we designed our sgRNA ex-pressing cassette based on the DNA sequence that needs to be modified.  The part in-cludes 275 bp and the sequence was biobricked with the pBAD promoter and the S. Py-ogenes terminator. The sgRNA sequences start right after the promoter and followed by the Cas9 handdle sequence which serves the binding purpose with the Cas9 protein.
Fig.2.Biobrick part design based on the DNA sequence that needs to be modified.
In the encoding algorithm, the NGG PAM site is inserted in the middle of the DNA frag-ment. The sgRNA sequence is generated from the upstream sequence of the PAM site which include the indexing information. Therefore, if the targeted DNA is recognized and cleaved by the Cas9 enzyme, the encoded information will get lost.
The Edit page of Bio101 will generate the part automatically based on the editing task. The part is provided as a SBOL format file for users to download.
Experience
The design scheme and parts remain to be experimentally validated in the future.
References
[1] Alec AK Nielsen & Christopher A Voigt*. Multi-input CRISPR/Cas genetic circuits that inter-face host regulatory networks. Mol Syst Biol. 2014 Nov 24;10:763. doi: 10.15252/msb.20145735.
[2] Cas9,wikipedia, [online], available from https://en.wikipedia.org/wiki/Cas9.
[3] Jinek M, Chylinski K, Fonfara I, Hauer M, Doudna JA, Charpentier E (Aug 2012). "A pro-grammable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity". Science. 337 (6096): 816–21.
[4]  iGEM15_EPFL, igem, [online], available from https://2015.igem.org/Team:EPF_Lausanne/Part_Collection.
FOLLOW US：
UESTC-SOFTWARE
© 2016 University of Electronic Technology and Science of China
CATALOGUE
CATALOGUE
Background
Part Design
Experience
References
