"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Wageningen UR/Backbone enzyme database
From 2013.igem.org
Home
Project
Parts
Achievements
Safety
Scientific Interactions
Team
Notebook
Why Aspergillus nigem?
Secondary metabolites
Toolbox
Host engineering
Summary
Achievements
Attributions
Safety introduction
General safety
Fungi-related safety
Biosafety Regulation
Safety Improvement Suggestions
Safety of the Application
Masterclass
Science cafe
iGEM Netherlands
Collaborations
Conference
Vitruvian man
Members
Profile
Thanks!
Lablog
Experimental protocols
Secondary Metabolite Enzyme Database
subtitle
Introduction
In general, the biosynthesis genes for fungal secondary metabolites are located in clusters. The fact that secondary metabolites are often synthesized as polymer backbones that are subsequently diversified greatly via the actions of tailoring enzymes sets the stage for combinatorial biochemistry because their biosynthesis is modular. In this combinatorial approach in which a modular system of domain shuffling can be used to generate a plethora of novel enzyme variants with new and improved functionalities. The number of permutations is extensive as there is a myriad of different domains that can be added, removed, reordered or exchanged. In order to facilitate mixing and matching of domains of these large enzymes a database for Aspergilli will be created.
Rationale
By recombining the different domains
Aim
Generating a secondary metabolites backbone enzyme database that contains the following information:
Gene sequence, gene size, introns, domains, illegal restriction sites and primary literature
Approach
Two recently published articles on secondary metabolites of Aspergilli species serve as the basis for this database. Analysis of the gene clusters that are related to secondary metabolite production allowed for identification of the backbone enzymes that stand at the basis of this process. Making a list of the backbone enzymes codes from the articles allows for extraction of additional information from other sources, such as NCBI and UniProt. Using a pattern match approach allows for identification of illegal restriction sites and other scripts can be used to automatically generate a database with this information. Finally, a graphical user interface will be created to allow easy access to the database for any user.
- D.O. Inglis et al., 2013 Comprehensive annotation of secondary metabolite biosynthetic genes and gene clusters of Aspergillus nidulans, A. fumigatus, A. niger and A. oryzae. BMC Microbiology Vol. 13 p. 1-23
- J.F. Sanchez et al., 2012. Advances in Aspergillus secondary metabolite research in the post-genomic era. Nat. Prod. Rep. Vol. 29, p. 351-371
Research Methods
Scripting will be done in python. The database will be contructed with the use of mySQL.
Retrieved from " http://2013.igem.org/Team:Wageningen_UR/Backbone_enzyme_database "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
