<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_Engineering_morphology"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/Engineering_morphology">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/Engineering_morphology&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/Engineering_morphology&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/Engineering_morphology&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/Engineering_morphology" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/Engineering morphology</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="topmenu"><UL class="menucenter"><LI><A href="/Team:Wageningen_UR" title="Team:Wageningen UR">Home</A></LI><LI class="project"><A href="/Team:Wageningen_UR/Why_Aspergillus_nigem" title="Team:Wageningen UR/Why Aspergillus nigem">Project</A></LI><LI class="parts"><A href="/Team:Wageningen_UR/Parts" title="Team:Wageningen UR/Parts">Parts</A></LI><LI class="achievements"><A href="/Team:Wageningen_UR/Achievements" title="Team:Wageningen UR/Achievements">Achievements</A></LI><LI class="safety"><A href="/Team:Wageningen_UR/Safety_introduction" title="Team:Wageningen UR/Safety introduction">Safety</A></LI><LI class="human"><A href="/Team:Wageningen_UR/Masterclass" title="Team:Wageningen UR/Masterclass">Scientific Interactions</A></LI><LI class="team"><A href="/Team:Wageningen_UR/Team" title="Team:Wageningen UR/Team">Team</A></LI><LI class="notebook"><A href="/Team:Wageningen_UR/Notebook" title="Team:Wageningen UR/Notebook">Notebook</A></LI></UL></DIV><UL class="sub-project subs"><LI><A href="/Team:Wageningen_UR/Why_Aspergillus_nigem" title="Team:Wageningen UR/Why Aspergillus nigem"><SPAN>Why Aspergillus nigem?</SPAN></A></LI><LI><A href="/Team:Wageningen_UR/Secondary_metabolites" title="Team:Wageningen UR/Secondary metabolites">Secondary metabolites</A></LI><LI><A href="/Team:Wageningen_UR/Biosensors" title="Team:Wageningen UR/Biosensors">Toolbox</A></LI><LI><A href="/Team:Wageningen_UR/Host_engineering" title="Team:Wageningen UR/Host engineering">Host engineering</A></LI><LI><A href="/Team:Wageningen_UR/Summary" title="Team:Wageningen UR/Summary">Summary</A></LI></UL><UL class="sub-achievements subs"><LI><A href="/Team:Wageningen_UR/Achievements" title="Team:Wageningen UR/Achievements">Achievements</A></LI><LI><A href="/Team:Wageningen_UR/Attributions" title="Team:Wageningen UR/Attributions">Attributions</A></LI></UL><UL class="sub-safety subs"><LI><A href="/Team:Wageningen_UR/Safety_introduction" title="Team:Wageningen UR/Safety introduction">Safety introduction</A></LI><LI><A href="/Team:Wageningen_UR/General_safety" title="Team:Wageningen UR/General safety">General safety</A></LI><LI><A href="/Team:Wageningen_UR/Fungi-related_safety" title="Team:Wageningen UR/Fungi-related safety">Fungi-related safety</A></LI><LI><A href="/Team:Wageningen_UR/Biosafety_Regulation" title="Team:Wageningen UR/Biosafety Regulation">Biosafety Regulation</A></LI><LI><A href="/Team:Wageningen_UR/Safety_Improvement_Suggestions" title="Team:Wageningen UR/Safety Improvement Suggestions">Safety Improvement Suggestions</A></LI><LI><A href="/Team:Wageningen_UR/Safety_of_the_Application" title="Team:Wageningen UR/Safety of the Application">Safety of the Application</A></LI></UL><UL class="sub-human-practices subs"><LI><A href="/Team:Wageningen_UR/Masterclass" title="Team:Wageningen UR/Masterclass">Masterclass</A></LI><LI><A href="/Team:Wageningen_UR/Science_cafe" title="Team:Wageningen UR/Science cafe">Science cafe</A></LI><LI><A href="/Team:Wageningen_UR/IgemNL" title="Team:Wageningen UR/IgemNL">iGEM Netherlands</A></LI><LI><A href="/Team:Wageningen_UR/Collaborations" title="Team:Wageningen UR/Collaborations">Collaborations</A></LI><LI><A href="/Team:Wageningen_UR/Conference" title="Team:Wageningen UR/Conference">Conference</A></LI><LI><A href="/Team:Wageningen_UR/Vitruvian_man" title="Team:Wageningen UR/Vitruvian man">Vitruvian man</A></LI></UL><UL class="sub-team subs"><LI><A href="/Team:Wageningen_UR/Team" title="Team:Wageningen UR/Team">Members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Wageningen_UR" target="_blank">Profile</A></LI><LI><A href="/Team:Wageningen_UR/Sponsors" title="Team:Wageningen UR/Sponsors">Thanks!</A></LI></UL><UL class="sub-notebook subs"><LI><A href="/Team:Wageningen_UR/Notebook" title="Team:Wageningen UR/Notebook">Lablog</A></LI><LI><A href="/Team:Wageningen_UR/Experimental_protocols" title="Team:Wageningen UR/Experimental protocols">Experimental protocols</A></LI></UL><DIV class="projectheader"><DIV class="content hostengineering"><H1>Host Engineering</H1><H2>Generating single cell factories</H2></DIV></DIV><DIV id="wrapper"><DIV id="tabs"><UL><LI id="li_tab1" onclick="tab('tab1')"><A>Introduction</A></LI><LI id="li_tab2" onclick="tab('tab2')"><A>Secondary metabolites</A></LI><LI id="li_tab3" onclick="tab('tab3')"><A>Toolbox</A></LI><LI id="li_tab4" onclick="tab('tab4')" class="active"><A>Host engineering</A></LI><LI id="li_tab5" onclick="tab('tab5')"><A>Summary</A></LI></UL></DIV></DIV><DIV id="hostengineering_content_area"><DIV id="tab1"><DIV class="tournav"><UL><LI class="firstbgsm fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Why_Aspergillus_nigem">Why <I>Aspergillus nigem</I>?</A></LI><LI class="fill">Secondary metabolites</LI><LI class="fill">Lovastatin</LI><LI class="fill">ATP Biosensor</LI><LI class="fill">pH Biosensor</LI><LI class="fill">Cytoskeleton and Septa</LI><LI class="fill">Chromoproteins</LI><LI class="smbgsm current">Host engineering</LI><LI class="last">Summary</LI></UL></DIV></DIV><DIV id="tab2"><DIV class="tournav"><UL><LI class="first fill">Why <I>Aspergillus nigem</I>?</LI><LI class="smbg fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites">Secondary metabolites</A></LI><LI class="bgbg fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Lovastatin">Lovastatin</A></LI><LI class="bgsm fill"><A href="https://2013.igem.org/Team:Wageningen_UR/ATP_biosensor">ATP Biosensor</A></LI><LI class="fill">pH Biosensor</LI><LI class="fill">Cytoskeleton and Septa</LI><LI class="fill">Chromoproteins</LI><LI class="smbgsm current">Host engineering</LI><LI class="last">Summary</LI></UL></DIV></DIV><DIV id="tab3"><DIV class="tournav"><UL><LI class="first fill">Why <I>Aspergillus nigem</I>?</LI><LI class="fill">Secondary metabolites</LI><LI class="fill">Lovastatin</LI><LI class="fill">ATP Biosensor</LI><LI class="smbg fill"><A href="https://2013.igem.org/Team:Wageningen_UR/pH_biosensor">pH Biosensor</A></LI><LI class="bgbg fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Cytoskeleton_and_septa">Cytoskeleton and Septa</A></LI><LI class="bgsm fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Chromoproteins">Chromoproteins</A></LI><LI class="smbgsm current">Host engineering</LI><LI class="last">Summary</LI></UL></DIV></DIV><DIV id="tab4"><DIV class="tournav"><UL><LI class="first fill">Why <I>Aspergillus nigem</I>?</LI><LI class="fill">Secondary metabolites</LI><LI class="fill">Lovastatin</LI><LI class="fill">ATP Biosensor</LI><LI class="fill">pH Biosensor</LI><LI class="fill">Cytoskeleton and Septa</LI><LI class="fill">Chromoproteins</LI><LI class="smbgsm current">Host engineering</LI><LI class="last">Summary</LI></UL></DIV></DIV><DIV id="tab5"><DIV class="tournav"><UL><LI class="first fill">Why <I>Aspergillus nigem</I>?</LI><LI class="fill">Secondary metabolites</LI><LI class="fill">Lovastatin</LI><LI class="fill">ATP Biosensor</LI><LI class="fill">pH Biosensor</LI><LI class="fill">Cytoskeleton and Septa</LI><LI class="fill">Chromoproteins</LI><LI class="smbgsm current">Host engineering</LI><LI class="last bg"><A href="https://2013.igem.org/Team:Wageningen_UR/Summary">Summary</A></LI></UL></DIV></DIV></DIV><DIV id="pagewrapper" class="hostengineering"><H2><SPAN class="mw-headline" id="Outline"> Outline </SPAN></H2><P>Synthetic biology doesn’t just stop at the level of molecular systems. To expand the scope of this project we have chosen for a multi-level approach in which we are working on biobricks, proteins, a pathway and the chassis. In order to achieve the latter, two strategies have been conceived of. In the first we have chosen to harness the power of directed evolution, a powerful tool that is not often used in this competition. In order to explore new territories we have chosen for a second, fully rational approach in which we analyze the transcriptome of two distinct fungal phenotypes; the mycelial and the single cellular. 
</P><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>The secretion capacity of <I>Aspergillus niger</I> is the feature mainly contributing to its status as an excellent industrial workhorse. However, when we investigate this process in more detail, we find that only the hyphal tips of the mycelium are actively secreting. The fungal filaments that are formed during cultivation increase the viscosity of the liquid broth, and in turn reduce oxygen and heat transfer. Since vegetative mycelium poses an issue for industrial applicability, generating metabolically active single cells holds the potential to alleviate this burden, as a unicellular strain could be cultured in a similar fashion as a yeast, e.g. <I>Saccharomyces cerevisiae</I>. This shows that  single cells are not only interesting from a molecular or bioinformatics perspective, but that also from a process-oriented point of view the potential of host engineering is interesting. The fact that this research intersects with a fundamental topic, the evolution of multicellularity, makes it even more intriguing.
</P><P class="caption">Figure 1) Schematic representation bioreactor</P><H2><SPAN class="mw-headline" id="Strategy_1:_Directed_evolution"> Strategy 1: Directed evolution </SPAN></H2><P>The power of experimental evolution with regard to complex adaptations has been demonstrated in recent research, allowing acquisition of multicellular <I>Saccharomyces cerevisiae</I>. The other way around one can think of evolutionary adaptation proceeding in the opposite direction in an environment where a unicellular phenotype is advantageous. 
</P><H3>A scientific paper on multicellular yeast by directed evolution</H3><P>Ratcliff, W. C., R. F. Denison, et al. (2012).<I> &quot;Experimental evolution of multicellularity.&quot; </I> Proc Natl Acad Sci U S A 109(5): 1595-1600.</P><P>Ratcliff <I>et al.</I> subjected the unicellular yeast <I> Saccharomyces cerevisiae </I> to an environment in which they expected multicellularity to be adaptive. Rapid evolution of clustering genotypes that display a novel multicellular life history characterized by reproduction via multicellular propagules was observed. Simple division of labor rapidly evolved among cells. Early multicellular strains were composed of physiologically similar cells, but these subsequently evolved higher rates of programmed cell death (apoptosis), an adaptation that increases propagule production. This shows that key aspects of multicellular complexity can readily evolve from unicellular eukaryotes. This made us ponder and led to the idea of generating a single cellular phenotype in a reverse approach.</P><H2><SPAN class="mw-headline" id="Aim"> Aim </SPAN></H2><P>Obtain a single cell phenotypic <I>Aspergillus niger</I> strain by directed evolution</P><H2><SPAN class="mw-headline" id="Approach"> Approach </SPAN></H2><P>In this approach we select for cells with a reduced mycelial cohesiveness by using filters with different pore sizes. The procedure is iterative; we grow the cells, vortex them, filter them and then culture the cells that were able to get through the filter for the next round.</P><P class="caption">Figure 2) The iterative procedure in this directed evolution approach.</P><H2><SPAN class="mw-headline" id="Research_Methods"> Research Methods </SPAN></H2><P>Mutagenised spores of <I>A. niger</I> N593 from which evolved strains were cultivated are stored at -80°C according to Appendix C for possible genomic comparison. Spores from <I>A. niger</I> N593 spores were mutagenized by exposure to a Philips TUV 30W lamp for 10 to 60 seconds.
In order to apply selective pressure for the desired phenotypic trait, filters with different pore sizes are used. For filter steps in the evolution experiments BD Falcon® Cell Strainers of 40, 70 and 100 µm pore size and SEFAR NITEX® 3-150/50 nylon filter gauze of 150 µm pore size were used. Stainless steel mesh ‘cups’ with pore size 20 µm from Anping Yuansheng Mesh Cooperation were also used.</P><H2><SPAN class="mw-headline" id="Strategy_2:_Comparative_transcriptomics"> Strategy 2: Comparative transcriptomics </SPAN></H2><P>RNA sequencing is a next generation sequencing technology that is rapidly replacing the conventional DNA microarrays. Because, unlike microarrays, RNA sequencing does not rely on probes or primers, there is a smaller bias.  It allows for more than analyzing differential gene expression, as it also allows for discovery of novel RNAs and analysis of isoforms of genes. Another advantage over of RNA sequencing over DNA microarrays is that data can be reanalyzed once more information on the transcriptome becomes available.
Combined with the existence of dimorphisms this allows for an interesting opportunity when it comes to the investigation of the genes involved in this phenotypic distinction. 
</P><H3>A scientific paper from 1971</H3><P>Anderson, J. G. and J. E. Smith (1972). <I>&quot;Effects of Elevated-Temperatures on Spore Swelling and Germination in Aspergillus Niger.&quot;</I> Canadian Journal of Microbiology 18 (3): 289-297.</P><P>Anderson and Smith found that at 44°C germ-tube formation was completely inhibited in Aspergillus niger, although spherical growth could occur over a prolonged period to produce large spherical cells. More generally, there are more dimorphic fungi that display such a distinctive phenotypic transition at elevated temperatures. By comparing transcriptional profiles of distinct phenotypical morphologies we can obtain insight in the genes causative to multicellularity.</P><H2><SPAN class="mw-headline" id="Aim_2"> Aim </SPAN></H2><P>
Finding sets of candidate genes causative to the single cell phenotype in <I> Aspergillus niger </I> by transcriptome analysis.
</P><H2><SPAN class="mw-headline" id="Approach_2"> Approach </SPAN></H2><P>
Spores of <I> A. niger </I> N400 are grown at three distinct conditions. Growing <I> A. niger </I> in liquid medium at 45°C will yield a single cell phenotype. When grown at the same temperature on a solid medium mycelium is formed. Mycelium is also formed when <I> A. niger </I> is grown at 30°C. Thus, by varying temperature or phase-state of the medium one can obtain either a multi- or single cellular phenotype. Transcriptome analysis allows identification of the transcripts that are uniquely present or absent in the single cell phenotype. This knowledge can be used to genetically modify <I> A. niger </I> to obtain a single cell phenotype at a broad range of conditions, such as at room temperature.</P><P class="caption">Figure 4)
</P><H2><SPAN class="mw-headline" id="Research_Methods_2"> Research Methods </SPAN></H2><P>After cultivation of the <I>Aspergilli</I> at different conditions, RNA needs to be extracted according to Appendix H. This RNA will be sent out to a sequencing company which will return us FASTQ files containing the reads. For the analysis of the data a transcriptomics pipeline needs to be constructed. Since there is a reference genome available to which all the constructs can be mapped we don’t need to perform a <I>de novo</I> assembly.</P><H3>TopHat</H3><P>RNA-Seq generates millions of short sequence reads and therefore the mapping of RNA-Seq is an intensive computational task. This process is performed with the use of specific software. One of these software packages is TopHat. Most mapping algorithms depend on the known splice junctions. TopHat on the other hand is designed to map RNA-Seq reads without relying on these known splice junctions. It detects splice sites <I>ab initio</I> by identifying reads that span exon junctions. The algorithm first maps the non-junction and short reads using Bowtie. Next the program creates a consensus of mapped reads. Then TopHat breaks up the initially unaligned reads into smaller pieces, constructing a seed table for tracking of possible junctions, enabling them to be matched onto the reference genome.</P><P class="caption">Figure 5) Mapping reads onto the <I>Aspergillus niger</I> reference genome allows for discovery of patterns in gene expression that are unique to the single cell phenotype.</P><P>
FastQ files containing paired-end reads are the primary data source. These files contain all reads and are different from a fasta format in that they contain an additional line for the quality score of the sequence reads. First TopHat is used to map the reads on the reference genome (genome.fa) with their respective annotation (genes.gtf). TopHat uses Bowtie for its efficient data structure to perform the read alignment. Then TopHat breaks up the initially unaligned reads into smaller pieces, enabling them to be matched onto the reference genome. Unlike other mapping algorithms which rely on known splice sites, TopHat identifies reads that span exon junctions to detect splice sites. By assuming that the reads that now do align are splice variants the intron length can be deduced. The mapped reads (accepted_hits.bam) are subsequently assembled into transcripts with Cufflinks, using a parsimonious approach to explain the data.</P><H3>Cufflinks</H3><P>Cufflinks was developed to investigate differential gene expression. This program is designed to calculate the abundances of transcripts constructing an overlap graph, or de Bruijn graph. This graph is used to calculate the maximum likelihood of transcripts resulting in Fragment Per Kilobase per Milion (FPKM) values. This value, which represents the quantity of mapped fragments relative to the length of the transcript, normalises for transcript length and machine yield such that expression of different transcripts can be compared. Expression levels of genes can simply be determined by summing the FPKM values of their respective isoforms.

Measurements of immature transcripts are excluded from the analysis on the basis of their low abundance. The fragments per kilobase of transcript per million mapped fragments (FPKM) normalizes the reads for transcript length and machine yield such that expression of different transcripts (transcripts.gtf) can be compared. 
</P><P class="caption">Figure) The first part of the RNA-seq pipeline upto the transcript assembly</P><P>Expression levels of genes can simply be determined by summing the FPKM values of their respective isoforms. On the basis of the average FPKM the highest and least expressed genes can be selected. The start and stop positions of exons in the genome, given in the .gtf file, can be used to calculate the exon length, which can then be subtracted from the gene length to obtain the intron length. The sequence data from the .gtf files can be formatted to fasta files containing nucleotide sequences of the genes. 
</P><P class="caption">Figure) The second part of the RNA-seq pipeline that generates fasta files </P><H3>Downstream analysis</H3><P>The statistical program ‘R’ will be used to visualise the final data output (see <A href="http://www.r-project.org" target="_blank">r-project.org</A>). Extraction of gene ID’s from the fasta files into a text file allows for manual input of this data into cytoscape (see <A href="http://www.cytoscape.org" target="_blank">cytoscape.org</A>). A hierarchical network can be inferred for all three Gene Ontology (GO) annotations for visualization of overrepresented terms in specific (sets of) nodes.</P><H2><SPAN class="mw-headline" id="Results:_growth_and_metabolic_activity"> Results: growth and metabolic activity </SPAN></H2><P>
One of the main questions that arose in our comparative transcriptomics approach is whether the so-called single cells that are formed at 45°C are metabolically active or are rather just crippled swollen cells. In our investigation we first monitor the growth of the single cells by a time-series analysis. Next, we assess the metabolic activity of the cells by several methods in which we attempt to examine division of cells, their nuclei and transcriptional and translation activity.
</P><H2>Spherical growth</H2><P>
Over a time interval of 32 hours, every four hours a series of photo's is taken to obtain insight in the development of the cells and their spherical growth. Most, if not all, of the cells survive this high temperature condition for a prolonged period of time. Even after 96 hours a lot of cells are still intact, however some have lysed and irregular structures within cells are also observed and gives the impression that the cells are not as healthy as they used to be.
</P><P class="caption">Figure) Growth of <I>A. niger</I> N593 at 44°C. From left to right, top to bottom: 0h, 4h, 8h, 12h, 16h, 20h, 24, 28h, 32h, 96h.</P><P>When the sample is moved to 30°C  after 24 hours, mycelial growth occurs as normal. This indicates that the cells are not irreversibly damaged, however more detailed investigation of cellular metabolism is required.
</P><H2>Change of Strain</H2><P>As described in literature, N593 formed giant cells after 24h at 44°C. Since transcriptome data on N400 from multiple stages in its life cycle is available, this strain is chosen such that this research complements the current RNA landscape profile of <I>A. niger</I> and allows for comparison of data from different life stages. However, unlike N593, N400 repeatedly formed mycelium at 44°C. To overcome this effect the temperature was increased to 45°C, at which a single cell phenotype was obtained for N400.</P><P class="caption">Figure) Distinct morphologies N400 and N593 grown at 44°C.</P><P>Besides from the occurrence or germination at 44°C, another noticeable different is the size. It appears that the mutations that N593 has obtained in comparison to N400, are causative to a reduction in size as well as a reduction in the maximum temperature at which germination can occur.</P><H2>Finding the right conditions</H2><P>
There are three conditions from which we wish to obtain samples; 30°C mycelium, 45°C single cells and 45°C mycelium. The first of these conditions is often encountered in the environment of <I>A. niger </I> and thus a familiar phenotype. The single cells can be obtained by cultivation of the spores as a liquid culture at 45°C. This leaves us with the challenge of obtaining a multicellular phenotype at 45°C. In order to obtain this phenotype we intend to grow <I>A. niger</I> N400 on a plate without shaking it at 250RPM.
Growing <I>A. niger</I> on plates with solid media, as well as with liquid media, causes rapid dehydration of the medium at this high temperature. We cannot seal the plates completely since exchange of air is required and even our best attempts caused plates to dry out. Therefore we decided to grow the <I>A. niger</I> in Falcon tubes without shaking them, hoping this would result in a multicellular phenotype, but unfortunately this was not the case. After having discussed the issue with one of our advisors, Peter Schaap, we decided to grow the fungus in Falcon tubes at 30°C for 8 hours, after which we will transfer them to 45°C for the remaining 16 of the 24 hours. Microscopic analysis after the first 8 hours and after the remaining 16 hours suggested that the mycelium remains to grow for some time after transfer to 45°C. This finding is interesting since it allows for a comparative transcriptomics approach in which the effects of an increased temperature can be filtered out.
</P><H2>Calcofluor Staining</H2><P>
In an attempt to find out whether the cells are actively dividing we conceived of the idea of staining the cells with calcofluor. Under the microscope clusters of cells are seen and we tried to assess whether some of these are connected via their cytoplasm, as in the the final stage of mitosis. Calcofluor is known to stain cellulose and chitin. In the case of our fungus chitin is found in the cell wall, thus if we find cells in this specific stage of mitosis, just before division is completed and when the cytoplasm is connected, we might be able to visualise this with the use of this staining method. Unfortunately, when we stained the cells we were not able to draw any conclusions. This is because even if the cells would be in this specific state, the staining of the cell wall that is surrounding their cytoplasmic connection is too intense.    
</P><P class="caption">Figure) Calcofluor staining of giant cells</P><H2>DAPI Staining</H2><P>
Since we were unable to assess whether the cells actively divide by staining them with calcofluor, we decided to see if we could determine whether the nuclei actively divide instead. There are multiple staining methods for nuclei, one of which is through the use of 4',6-diamidino-2-phenylindole (DAPI). A previous study has shown that 85% of the dormant spores contain 2 nuclei, whereas the remaining 15% contain one. When germination occurs these nuclei normally divide and the number of nuclei per cell increases. To assess whether this basic and vital function remain intact we have stained the giant cells after 24 hours of incubation at 45°C.

</P><P class="caption">Figure) DAPI staining indicated nuclei divide</P><P>
The images taken clearly show that the nuclei are actively dividing at this increased temperature. Thus, although we could not conclude whether the cells are capable of mitotic division by calcofluor staining, DAPI staining shows that meiosis does occur. Indicating that such a basic and vital function is still working is a very important indication of cellular activity.
-van Leeuwen, M.R. <I>et al.</I> (2012).<I> &quot;Germination of spores of Aspergillus niger is accompanied by major changes in RNA profiles.</I> Studies in mycology, Vol. 74, p.59-70.
</P><H2>expressing GFP</H2><P>
Now that we have established that the nuclei are actively dividing and that the meiosis is not impaired, we wish to go further in our analysis by examining metabolic activity of giant cells. We therefore use an <I> A. niger </I> strain that is transformed with GFP to see whether the giant cells are capable of proper transcription and translation of this protein. The GFP is placed behind the constitutive promotor pkiA (which we have biobricked;link there) and the results shows that after 24 hours of germination all the cells express GFP. This is yet another confirmation of the fact that the basic metabolism of the cells is still functional. 
</P><P class="caption">Figure) GFP transformed A. niger grown at 45°C.</P><H2><SPAN class="mw-headline" id="Results_directed_evolution"> Results directed evolution </SPAN></H2><P>
Evolution experiments can be viewed as a cycle of growth and selection, as depicted above in figure 2. Selection is the evolutionary pressure exerted on the population of cells, which in this case is based on size. The most important thing was to find the largest constant evolutionary pressure, and thus the smallest evolutionary bottleneck, without it leading to major extinction of the parallel evolving lines. To increase the efficiency of this process we have chosen to irradiate the spores first to generate a larger genotypic diversity.
</P><H2>UV mutagenesis</H2><P>
Small petri dishes were filled with 2 ml spore suspension of 10<SUP>8</SUP> spores/mL and exposed to UV radiation for 10, 20 and 40 seconds respectively.  Generating extra mutations gives the population a mutational ‘head start’ and may speed up the generation of beneficial mutations. On the basis of the survival rate, which was ~90% after 10 seconds, these spores were chosen for evolution experiments with 10mL liquid cultures. For the evolution experiment of 50mL liquid cultures diluted, larger batches of spores were used to enhance homogeneous distribution of UV radiation. A spore suspension was diluted to 5*10<SUP>6</SUP> spores/mL and batches of 10mL were exposed to UV in standard 94mm petri dishes for 10, 20, 40 and 60 seconds. The same approach was used to estimate survival rate and in this case the spores that had been radiated for 60 seconds showed a survival rate of 70% and were chosen.
</P><H2>Methodology</H2><P>
Since there is not a standardised protocol, the protocol that has been developed can itself be considered a result. Six parallel lines of 10mL CM liquid cultures were used to be able to note similar or divergent patterns in evolution. Multiple lines are needed to make sure the experiment can still continue when part of the lines are lost due to, for instance, severe infection with bacteria. To initiate the first round of selection, six liquid cultures of 10mL CM were inoculated with mutagenised spores to 10<SUP>6</SUP> spores/mL and incubated at 30°C and 250RPM. Selection was imposed and growth was allowed to resume under the same conditions for approximately 24 hours before the next round of selection could start. </P><H3>The iterative cycle:</H3><P><SPAN class="ref">1.</SPAN>Vortex the culture at max speed.<SPAN class="ref">2.</SPAN>Filter the culture.<SPAN class="ref">3.</SPAN>Spin down the filtrate (15 min @ 4800g).<SPAN class="ref">4.</SPAN>Decant the supernatant carefully.<SPAN class="ref">5.</SPAN>Resuspend the pellet in CM.<SPAN class="ref">6.</SPAN>Inoculate at least half of the resuspension in 10mL fresh CM.<SPAN class="ref">7.</SPAN>Use the other part to inoculate CM plates for spores or store it at 4°C as backup material.

Variables in this scheme are vortexing time, filter pore size and complete or incomplete decanting.
Vortexing was done holding the liquid culture tube upright, standardising shearing rate. More vortexing time would create larger shearing stress and thus a less strict selection on more easily separable viable mycelium, so less evolutionary pressure. 
Filtration creates a size threshold, excluding everything with a size above it. This threshold is mainly dependent on the filter pore size. Larger pore size would mean a less strict selection. The filtrate was collected and centrifuged in plastic 50 ml Greiner tubes. After centrifugation the samples were decanted to prevent factors affecting growth to be transferred from one round to the other as much as possible. However, complete decantation results in loss of cells because the pellet is not dense enough, thus the choice was made to let the last fraction of medium remain. Shaking the samples at 250RPM is important, because when cells get stuck to the glass they can sporulate. If this occurs the samples is rendered unsuitable for the next round of selection, since spores can easily permeate the filter which results in a loss of evolutionary pressure. The final protocol consists of the variables 10s of vortexing, filtering over 150 µm filter and incomplete decanting of the medium. A detailed version can be found under Appendix F. 
A protocol for an evolution experiment with 3 larger liquid cultures of 50 ml in 250 ml Erlenmeyers was based on this. Variables are the same, except for the vortexing step, which showed to be non-effective in a test run. Instead, a stirrer rod of 50 mm length was spun at 300 RPM to disrupt and disperse mycelium. In six rounds of selection, one extinction event occurred. A detailed protocol can be found under Appendix G. 

For the integrity of the parallel lines employed in the evolution experiments, mycelium-inoculated plates for harvest of spores should be kept of all lines and if necessary, lines should be recovered from spores of the same line.
</P><H2><SPAN class="mw-headline" id="Characterisation_evolved_strains"> Characterisation evolved strains </SPAN></H2><P>
Experimental evolution lines are coded ‘x-yTz’, where ‘x’ is the line number, ‘y’ is the number of selection rounds that have been applied, ‘T’ is short for transfer, the practical term for selection round, and ‘z’ is the filter pore size used. For evolved strains resulting from the experiment employing 50 ml liquid cultures, an ‘L’ was added for ‘Larger volume’. 
</P><H2>Morphology</H2><P>
When plated on CM and incubated at 30°C, evolved strains show the same morphology and growth rate as the parent strain N593. Also at 44°C no difference was found. However, when grown in liquid culture, a different macromorphology between the evolved and the parent strain became visible. All evolved strains, and 12T150 in particular, have developed a more pellet-like macromorphology. 12T150 shows the presence of pellets that appear to be of 2 distinct sizes. Microscopic analysis however showed no difference in micromorphology.
</P><P class="caption">Figure) Parent strain N593 (left) and evolved strain 12T150 (right) in liquid culture. The lower pictures show liquid cultures poured in a petri dish.</P><P>
Quantification of mycelial cohesiveness was done by analysing the amount of CFUs per amount of mycelium able to permeate a filter, or permeative ability. </P><P class="caption">Figure) Permeative ability of parent strain N593 and evolved strains by CFU count, including a check for ungerminated spores by plating filtered supernatant without prior disruption of the culture. Counts were made up to 200 CFUs per plate, causing a negative bias in the 1000µL plates, which suffered from that. </P><H2>Organic acid secretion</H2><P>
Oxalic acid production values calculated from UV measurements are given in figure 23. No significant decrease in production was found. 12T150 even shows a higher oxalic acid production. Data points even stronger in this direction when the final yield is calculated from the production values at 72h and the biomass measured for each liquid culture individually (figure 24). 12T150 shows a significant increase in yield of 57 % over N593 (P = 0.00456; two-tailed t-test, unequal variances).
</P><P class="caption">Figure) Oxalic acid production per amount of biomass after 72 hours of growth in 10 ml production medium.</P><H2><SPAN class="mw-headline" id="Results_comparative_transcriptomics"> Results comparative transcriptomics </SPAN></H2><P>The samples were immediately placed on ice after removing them from the 45°C incubator. Then they were centrifuged for 10 minutes at 4700RPM at 0°C. The liquid medium was decanted in 50mL bottles, which were subsequently autoclaved. The sugar concentration could then be measured to determine that no starvation occurred at any time. The biomass pellet was frozen in liquid nitrogen and extraction of the RNA was performed using protocol H.
</P><H2>RNA StdSens analysis</H2><P>RNA isolation proved troublesome. A minimum amount of 20ug is required with a RNA integrity number (RIN) value of 8.0. The first isolations yielded RNA concentrations that were too low. Overcoming this issue was rather easy compared to the problem of obtaining RNA with a sufficiently high RIN value. Even after having discarded all the solutions used and having made aliquots of each of the required solutions the results did not improve. When faith in obtaining samples of sufficient quality had nearly hit rock bottom, Tom Schonewille, a technician at the lab, suggested to clean the Experion machine thoroughly using the so-called 'Deep Cleaning' method. To our relieve this alleviated much of the problem, as now samples that were previously measured to be of insufficient quality showed to have the required RIN value of at least 8.0.
</P><P>The RNA extracted from the giant cells yielded a few good quality samples, as can be seen below. Compared to the mycelial samples the amount of RNA obtained from the single cells was lower, though the quality was good.</P><P class="caption">Figure) Output of the analysis of the RNA extracted from the giant cells by Experion automated electrophoresis. The measured RIN value is 9.1</P><P>In contrast to the latter the RNA samples obtained from the mycelium was disappointingly low. A remarkable feature that was observed was the complete absence of any peaks after 30 seconds.</P><P class="caption">Figure) Output of the analysis of the RNA extracted from the mycelium by Experion automated electrophoresis. The measured RIN value is 1.0</P><P>It appears that the RNA is nearly completely degraded. A good explanation is for this observation has still to be found.
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Wageningen_UR/Engineering_morphology">http://2013.igem.org/Team:Wageningen_UR/Engineering_morphology</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/Engineering_morphology" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/Engineering_morphology" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/Engineering_morphology&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/Engineering_morphology&amp;oldid=308361" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>