"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/pH biosensor
From 2013.igem.org
Home
Project
Parts
Achievements
Safety
Scientific Interactions
Team
Notebook
Why Aspergillus nigem?
Secondary metabolites
Toolbox
Host engineering
Summary
Achievements
Attributions
Safety introduction
General safety
Fungi-related safety
Biosafety Regulation
Safety Improvement Suggestions
Safety of the Application
Masterclass
Science cafe
iGEM Netherlands
Collaborations
Conference
Vitruvian man
Members
Profile
Thanks!
Lablog
Experimental protocols
How flush is the cell?
"Seeing through is rarely seeing into." - Elizabeth Bibesco
Introduction
Secondary metabolites
Toolbox
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
ATP Biosensor
pH Biosensor
Cytoskeleton and Septa
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
ATP Biosensor
pH Biosensor
Cytoskeleton and Septa
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
ATP Biosensor
pH Biosensor
Cytoskeleton and Septa
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
ATP Biosensor
pH Biosensor
Cytoskeleton and Septa
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
ATP Biosensor
pH Biosensor
Cytoskeleton and Septa
Chromoproteins
Host engineering
Summary
Outline
Bio-sensors are invaluable tools to easily visualize and track changes in the cell and its environment. Protein based bio-sensors have the advantage of being easily targeted to any part of the eukaryotic cell and . Thus they are important components of the Aspergillus arsenal, which is an array of parts and devices to make Aspergillus niger more amenable to work with. We envision the use of ATP and pH bio-sensors to monitor the micro-environment in the cell where a pathway of interest has been targeted, in our case, the lovastatin biosynthesis pathway.
Such bio-sensors can provide quantitative information that can prompt us to maybe direct pathways to proper cell compartments or change the growth conditions to increase the yields. The actual fluctuations of proton and ATP concentrations in live cells can be pursued corresponding to different growth conditions. Consequently, filling gaps in knowledge about the cell physiology which is also essential in the context of host engineering. For example, the ATP bio-sensor will be instrumental in exploring the metabolic capabilities of the single cell like phenotype of A. niger at 45'C.
Introduction
A great array of new and existing biological devices and circuits have been designed and streamlined for useful purposes. However, currently there is not enough emphasis on how these devices and circuits respond to changes in their physiological surroundings, such as the effect of changes in pH and the availability of ATP. Thus we tested bio-sensors that enable us to monitor the dynamic changes of pH and ATP in live cells. The ATP bio-sensor is based on the phenomenon of fluorescence resonance energy transfer (FRET) and the pH bio-sensor is based on changes in the ionic state of the recombinant red fluorescence protein (pHRed). By combining multicolour, pHRed (red fluorescence) and FRET (green fluorescence), we can simultaneously image intracellular pH and ATP, as changes in metabolism often correlate with changes in pH and ATP concentrations in the cell. This would be interesting to determine whether the pH and ATP levels, at a certain position within the cell,  are optimal for a certain reaction. This will allow us to direct and target a metabolic pathway into its appropriate compartments for optimal production. These bio-sensors are useful to non-invasively quantify specific measurements in living cells.
pHRed
A novel approach in measuring intracellular pH in a non-invasive way has been developed by Mathew Tantama et. al. 2011, by using a recombinant red fluorescent protein (RFP), termed mKeima, had been genetically engineered into a S213A mutant, called pHRed. This will allow it to exhibit dual excitation peaks at 440 nm and 585 nm and an emission peak at 610 nm that respond ratiometrically well to pH changes [<a href="#ref5">5</a>]. The advantages of using a ratiometric method is that the estimation of pH is not affected by photobleaching, changes in focus of the microscope, variations in laser intensity, optical path length and illumination intensity. The excitation peak at 585 nm is caused by the anionic chromophore in an acidic condition whereas the excitation peak at 440 nm is caused by the protonated neutral chromophore in an alkaline condition.
<img src="
"/>
Figure 1) How different pH levels affect the excitation peaks of pHRedA. A: pHRed in acidic condition has an excitation peak at 585 nm with an emission peak at 610 nm. B: pHRed in alkaline condition has an excitation peak at 440 nm with an emission peak 610 nm.
The pH can be calculated by measuring the fluorescence intensity ratio between F585 and F440 (F585/ F440); and compare it with a calibration curve, see FIG. pH responses to ratio intensity of F585/ F440 are not affected in different buffer concentration such as Na+, K+, Mg2+, Cl−, Ca2+, HCO3−, H2O2, DTT, and temperature (21°C to 37°C). Based on these findings, pHRed had proven to be useful in monitoring and analyzing the dynamic changes in pH.
<img src="
"/>
Figure 2)[<a href="#ref5">5</a>]
</html>
ATP FRET biosensor
Fluorescence resonance energy transfer (FRET) is a phenomenon widely exploited by bio-sensors to monitor concentrations and temporal fluctuations of metabolites and ions at cellular and sub-cellular level. FRET works by excitation of a fluorescent molecule (donor) by a light of particular wavelength, which consequently transfers this energy to an adjacent fluorescent molecule (acceptor) that in-turn emits light. This phenomenon is very sensitive to the distance between the donor and acceptor fluorophore groups. Thus, fusing fluorescent proteins with a sensing domain that undergoes big conformational changes upon binding of the sensory molecule confers the possibility for generating an assortment of custom-made genetically encoded biosensors. These are useful tools to non-invasively quantify metabolites in living cells.
Figure 1) Working of the ATP FRET sensor [ 6 ]
FRET based ATP bio-sensors, dubbed as Ateams, have been previously developed and implemented to quantify in vivo ATP levels in HeLa and yeast cells [ 6 ], [ 7 ] . The sensor consist of mseCFP (a variant of CFP), the ATP sensing domain is a subunit of the Bacillus subtilis FoF1-ATP synthase and mVenus (a YFP variant). The YFP/CFP emission ratio (527 nm/475 nm) gives an estimation of the ATP concentration. This sensor has been found to be ATP specific and pH independent, although it is temperature sensitive with a higher ATP binding capacity at lower temperatures. The Ateams have been requested from the Imamura lab in Japan for cloning and expression into A. niger and E. coli.
pH biosensor
Aim
Implement a pH-sensor that will measure pH between compartments in growing A. niger during different growth conditions, such as oxygen and glucose depletion; and during citric acid production.
Application
Regulation of intracellular pH is important for the metabolism of a functional cell as it is tightly regulated by the complex interactions between the consumption, production, transportation and buffering of H+ [3][4]. To maintain protein stability, enzyme and ion channel activity, growth and division, it is essential to retain pH within their physiological range of pH 7.5 [1]. By implementing a pH-sensor in Asp. niger, it would be interesting to investigate whether the production of organic acid has any influence of intracellular pH and how we can improve upon it.
Approach
The outline of the approach is as follows:  Codon optimized pHRed sensor (sequence obtained from Mathew Tantama et al. 2011) was cloned and expressed in Asp. niger by transforming it with both pHRed or mitochondrial signal peptide fused at the N-terminal of pHRed, allowing it to target both the cytoplasm or mitochondria, respectively [ 2 ][ 5 ].
In addition, pHRed gene will be cloned in a pET52b+ vector downstream of strep-tag sequence, which will allow for the purification of pHRed proteins.
1.Make 2 contructs for targeting pHRed to cytoplasmic and mitochondria.
2.Purify strepTagII-pHRed from sonificated E. coli. and make a calibration curve.
3.Transform Aspergilllus niger with cytoplasmic pHRed and mitochondrial pHRed
4.Measure in vivo pH during citric acid production by A. niger in time lapse using a fluorescence microscope
Research Methods
To utilize pHRed for in vivo measurements of pH inside a filamentous fungi during citric acid production, two constructs were made. With one targeting the cytoplasm and the other targeting the mitochondria by transforming fungi with a naked pHRed sensor and pHRed sensor fused with a mitochondrial signal peptide, respectively. The mitochondrial signal peptide has been proven to target the mitochondria of A. niger [ 2 ] . To measure pH, a calibration curve is established by purifying streptagII fused at N-terminus of  pHRed proteins from sonificated E. coli, and is observed under a fluorescence microscope.
Results
After transformation of DH5α with 1µl of mixture (gibson assembly with pJet1.2 blunt vector), 7 colonies were picked and  were cultured in 5 ml LB medium with ampicillin. Afterwards, plasmids were isolated using a miniprep kit (thermo scientific) according to their protocol using a homemade silicon column. Culture #1 till #3 were digested with pstI and notI; culture #4 till #7 were digested with nsiI and notI, see Figure 1.
Figure 1) lane 1, 3 and 4 contain pJet1.2 with the appropriate mit-pHRed (803 bp). Lane 5-7 contain the pJet1.2 with the appropriate pHRed (725 bp).
ATP biosensor
Aim
Implement a FRET based sensor that will measure ATP levels between compartments in growing A. niger during metabolic shifts.
Applications
An ATP bio-sensor can lead to visualization of the ATP levels in live Aspergillus cells. Inquiry into whether the single cell like phenotype of A. niger still retains its metabolic activity can be carried out. It would be interesting to quantify the ATP levels in different organelles and investigate where and why such ATP levels exist. Also, the differences in cellular ATP under various growth environments can be studied. The differences in ATP levels for cells at the hyphal tip and those in the inner mycelium can be validated.
Approach
Clone and express the Ateam bio-sensors in A. niger – 1. ATP sensitive and cytoplasmic localization (APT1); 2. ATP in-sensitive and cytoplasmic localization (APT2); 3. ATP sensitive and mitochondrial localization (APT3). The version 1 of the Ateam sensor was codon optimized for A. niger, illegal restriction sites were removed and a detachable N-terminal mitochondrial signal peptide was included. The mitochondrial signal peptide has been proven to target the mitochondria of A. niger [ 2 ]. Detachable here means removable from the gene by restriction digestion to target the sensor to the cytoplasm instead. This resulted in the design of new variants of Ateam - ATP4 (codon optimized for A. niger, ATP sensitive and cytoplasmic localization)and ATP5 (codon optimized for A. niger, ATP sensitive and mitochondrial localization)(Figure 4). These versions were ordered as synthetic construct blocks that were stitched together and incorporated into the pJET vector via Gibson Assembly.
Figure 9)Designed ATP4 and ATP5, codon optimized for A. niger
Observe the cells under a fluorescent microscope and excite the CFP at 435 nm and analyze emission wavelengths at 475 nm (CFP) and 527 nm (YFP). Use sorbic acid in the growth media of A. niger to perturb the ATP pool inside the cell for testing the functionality of the Ateam sensor [ 8 ].
Express sensor in E. coli and purify with the help of a Histidine tag for in-vitro characterization in a buffer mimicking A. niger physiological environment [ 9 ].
Research Methods
The initial target was to introduce the following versions of the ATP sensor into A. niger – 1. ATP sensitive and cytoplasmic localization; 2. ATP in-sensitive and cytoplasmic localization; 3. ATP sensitive and mitochondrial localization; 4. Codon optimized, ATP sensitive and cytoplasmic localization; 5. Codon optimized, ATP sensitive and mitochondrial localization.
Version 1, 2 and 3 of the Ateam were kindly donated by the Imamura lab. These constructs were amplified out by PCR, flanked with additional restriction sites (NdeI and NotI) for cloning into pal85, a shuttle vector with an established A.niger constitutive expression. The pal85 vector has a pkiA promoter and xlnD terminator with a multiple cloning site in between to insert a gene of interest. Also, it has a pyrA gene for supplementing uridine auxotrphy in A.niger N593 strain.
In the next step, the PCR amplicons were blunt-end ligated into the pJET cloning vector and the DNA sequenced to check for any errors in the gene. Then it was digested out of the pJET vector with restriction enzymes NdeI & NotI and ligated with NdeI & NotI digested pal85. The presence of the constructs in pal85 was verified by colony PCR and restriction analysis. Next, Aspergillus niger protoplasts were transformed with pal85 vector containing each of the 3 versions. Genomic DNA was isolated from pure single colonies of transformants, although the presence of the constructs in the genomic DNA was not established by PCR.
The transformed strains were grown in complete medium(CM) without Uridine and checked under the microscope for fluorescence. The excitation filter allowed excitation of CFP (435 nm) and the emission filter allowed light above 515 nm (YFP emission). Also, to test the functionality of the sensors sorbic acid was added to the medium and the pH of the medium was set to 4.0. Addition of sorbic acid is known to perturb the ATP pool inside the cell. The growth experiment setup was as follows:
Figure 4) Growth experiment set-up to test functionality of 3 expressed Ateams in A. niger. ATP1- ATP sensitive and cytoplasmic localization, ATP2- ATP in-sensitive and cytoplasmic localization & ATP3- ATP sensitive and mitochondrial localization; N593 - wild type control strain
The Procedure for the growth experiment:
1. Inoculated 4 strains as indicated in table above in CM with and without supplements at 350 spores/200uL media/well of a 12 well plate and grown overnight.
2. Observe and make pictures under fluorescence microscope. Excitation wavelength of CFP (donor) is 435 nm. Emission at 475 nm (CFP - donor) and 527 nm (YFP - acceptor).
3. Replace media with new CM containing sorbic acid with final concentraions of 0.5 mM and 1.0 mM.
4. After overnight incubation observe and make pictures under fluorescence microscope. Excitation wavelength of CFP (donor) is 435 nm. Emission at 475 nm (CFP - donor) and 527 nm (YFP - acceptor).
The version 1 of the Ateam sensor was codon optimized for A. niger, illegal restriction sites were removed and a detachable N-terminal mitochondrial signal peptide was included. The mitochondrial signal peptide has been proven to target the mitochondria of A. niger [ 2 ]. Detachable here means removable from the gene by restriction digestion to target the sensor to the cytoplasm instead. This resulted in the complete design of the version 4 and 5 of the Ateam sensor. These versions were ordered as synthetic construct blocks that were stitched together and incorporated into the pJET vector via Gibson Assembly. E. coli were transformed with the DNA and verified by colony-PCR. The isolated plasmid DNA was sequenced to verify the sequence of the constructs. Cloning the sensors into pal85 and introducing into A. niger for expression is currently under progress.
Also, it was aimed to express the ATP sensor in Escherchia coli for in-vitro characterization. The version 1 Ateam was amplified out by PCR with flanking BamHI and NotI restriction sites and ligated into digested pCDFDuet-1 E. coli expression vector. This vector has a N-terminal Histidine tag in front of the introduced construct (designed to be in the frame of translation). The expression is via a T7 polymerase promoter induced by IPTG in the medium. After confirmation by colony PCR and restriction analysis the pCDFDuet-Ateam vector was introduced into E.coli BL21 strain, that has the T7 polymerase encoding gene in its genome.
Two kinds of induction were performed - 1. Grow transformed E.coli at 37'C until it reaches an OD of 0.5, induce with ITPG addition at a final concentration of 1 mM and incubate for 4 hours at 37'C. 2. Grow transformed E.coli at 37'C until it reaches an OD of 0.3, induce with ITPG addition at a final concentration of 0.01 mM and incubate overnight at 20'C. Expression was checked under a fluorescence microscope. The excitation filter allowed excitation of CFP (435 nm) and the emission filter allowed light above 515 nm (YFP emission).
25 mL of the E. coli culture was pelleted and re-suspended in 5 mL followed by sonication to release the protein. This cell extract was separated by centrifugation at low temperatures to maintain the protein stability. The cell extract was tested by a fluorimeter to check for intact fluorescent CFP and YFP. Next, the cell extract was purified with a Batch purification Method or a continuous FPLC Nickel column method. Both the purification methods are based on the preferential binding property of the His-tagged sensor to the Nickel columns. The output of this was de-salted and the resulting purified protein was stored in 20% glycerol at -20'C. Further, the in-vitro characterization of the purified sensor was performed. This involves exciting the CFP with 435 nm light and measuring the YFP/CFP emission ratio (at 527/475 nm respectively) at different ATP concentrations. The characterization of the sensor was carried out at room temperature in a buffer mimicking the in-vivo conditions in A. niger.
Result
The version 1, 2 and 3 of the Ateams (here on-wards referred to as ATP1, ATP2 and ATP3) donated by the Imamura lab were amplified out by PCR, flanked with additional restriction sites (NdeI and NotI) for cloning into pal85, a shuttle vector with an established A.niger constitutive expression. In the next step, the PCR amplicons were blunt-end ligated into the pJET cloning vector and the DNA sequenced to check for any errors in the gene. A colony PCR and restriction analysis was performed to confirm the cloning into pJET. In Figure 4, lane 1 shows the right bands of size 1800 bp for ATP2 and 3000 bp for linearized pJET; lane4 showing bands including one 1800 bp band for ATP1; lane6 and lane8 show 2000 bp band for ATP3.
Figure 4) Restriction analysis of ATP1, ATP2 and ATP3 in pJET vector
Then it was digested out of the pJET vector with restriction enzymes NdeI & NotI and ligated with NdeI & NotI digested pal85. The presence of the constructs in pal85 was verified by colony PCR and restriction analysis. In Figure 5, lane 1 shows the right bands of size 1800 bp for ATP1 and 6000 bp for linearized pal85; lane4 and lane5 showing bands including one 2000 bp band for ATP3; lane7 and lane8 show a band around 1800 bp for ATP2.
Figure 5) Restriction analysis of ATP1, ATP2 and ATP3 in pal85 vector
Next, Aspergillus niger protoplasts were transformed with pal85 vector containing each of the 3 versions. Genomic DNA was isolated from pure single colonies of transformants, although the presence of the constructs in the genomic DNA was not established by PCR. The transformed strains were grown in complete medium(CM) without Uridine and checked under the microscope for fluorescence. The excitation filter allowed excitation of CFP (435 nm) and the emission filter allowed light above 515 nm (YFP emission). A. niger strains expressing all the three versions of the sensor were verified under the fluorescence microscope.
Figure 6) ATP1 expression in A.niger
The functionality of the sensors were tested by the addition of sorbic acid to the medium and lowering the pH of the medium to 4.0. Addition of sorbic acid is known to perturb the ATP pool inside the cell. The growth experiment was setup and pictures were taken under a confocal fluorescence microscope.
Figure 7) Changes in FRET for in vivo ATP3 in A. niger under different sorbic acid concentrations
Figure 8)FRET observed for A. niger containing ATP3
The version 1 of the Ateam sensor was codon optimized for A. niger, illegal restriction sites were removed and a detachable N-terminal mitochondrial signal peptide included. This resulted in the complete design of the version 4 and 5 of the Ateam sensor which was ordered as synthetic construct blocks that were stitched together and incorporated into the pJET vector via Gibson Assembly. This was verified by colony PCR (Figure 9).
Figure 9) Colony PCR to confirm Gibbson Assembly of ATP5
The isolated plasmid DNA, from a colony that showed positive  results for colony PCR, was sent for DNA sequencing to verify the sequence of ATP5. The sequence was found to match perfectly as per the design of ATP5. The ATP4 construct was amplified out from ATP5 by PCR with primers that had flanking "standard 10" restriction sites. This was then cloned into pSB1C3 and again sequence verified before submission to the parts registry (). Cloning ATP4 and ATP5 into pal85 and introducing into A. niger for expression is currently under progress.
Biobricked  Ateam in pCDFDuet-1  In-vivo characterization: fluorimetry
Also, it was aimed to express the ATP sensor in Escherchia coli for in-vitro characterization. The version 1 Ateam was amplified out by PCR with flanking BamHI and NotI restriction sites and ligated into digested pCDFDuet-1 E. coli expression vector. This vector has a N-terminal Histidine tag in front of the introduced construct (designed to be in the frame of translation). The expression is via a T7 polymerase promoter induced by IPTG in the medium. After confirmation by colony PCR and restriction analysis the pCDFDuet-Ateam vector was introduced into E.coli BL21 strain, that has the T7 polymerase encoding gene in its genome.
Two kinds of induction were performed - 1. Grow transformed E.coli at 37'C until it reaches an OD of 0.5, induce with ITPG addition at a final concentration of 1 mM and incubate for 4 hours at 37'C. 2. Grow transformed E.coli at 37'C until it reaches an OD of 0.3, induce with ITPG addition at a final concentration of 0.01 mM and incubate overnight at 20'C. Expression was checked under a fluorescence microscope. The excitation filter allowed excitation of CFP (435 nm) and the emission filter allowed light above 515 nm (YFP emission).
25 mL of the E. coli culture was pelleted and re-suspended in 5 mL followed by sonication to release the protein. This cell extract was separated by centrifugation at low temperatures to maintain the protein stability. The cell extract was tested by a fluorimeter to check for intact fluorescent CFP and YFP. Next, the cell extract was purified with a Batch purification Method or a continuous FPLC Nickel column method. Both the purification methods are based on the preferential binding property of the His-tagged sensor to the Nickel columns. The output of this was de-salted and the resulting purified protein was stored in 20% glycerol at -20'C. Further, the in-vitro characterization of the purified sensor was performed. This involves exciting the CFP with 435 nm light and measuring the YFP/CFP emission ratio (at 527/475 nm respectively) at different ATP concentrations. The characterization of the sensor was carried out at room temperature in a buffer mimicking the in-vivo conditions in A. niger.
Conclusions
References
1. Bagar, T., K. Altenbach, et al. (2009). "Live-Cell imaging and measurement of intracellular pH in filamentous fungi using a genetically encoded ratiometric probe." Eukaryot Cell 8(5): 703-712.
2. Blumhoff, M. L., M. G. Steiger, et al. (2013). "Targeting enzymes to the right compartment: Metabolic engineering for itaconic acid production by Aspergillus niger." Metabolic Engineering 19(0): 26-32.
3. Felle, H. H. (1996). "Control of cytoplasmic pH under anoxic conditions and its implication for plasma membrane proton transport in Medicago sativa root hairs." Journal of Experimental Botany 47(7): 967-973.
4. Madshus, I. H. (1988). "Regulation of intracellular pH in eukaryotic cells." Biochem J 250(1): 1-8.
5. Tantama, M., Y. P. Hung, et al. (2011). "Imaging intracellular pH in live cells with a genetically encoded red fluorescent protein sensor." J Am Chem Soc 133(26): 10034-10037.
6. Imamura, H., et al. (2009). "Visualization of ATP levels inside single living cells with fluorescence resonance energy transfer-based genetically encoded indicators." PNAS 106: 15651-15656.
7. Bermejo, C., et al. (2011). In vivo biochemistry: quantifying ion and metabolite levels in individual cells or cultures of yeast. Biochem. J 438 (1–10).
8. Plumridge, A., et al.(2004). "The weak acid preservative sorbic acid inhibits conidial germination and mycelial growth of Aspergillus niger through intracellular acidification" Appl. Environ. Microbiol. 70(6):3506.
9. Ruijter, G.J.G, et. al (1997). Overexpression of phosphofructokinase and pyruvate kinase in citric acid-producing Aspergillus niger, Biochimica et Biophysica Acta (BBA) - General Subjects, 1334(2–3): 317-326.
Retrieved from " http://2013.igem.org/Team:Wageningen_UR/pH_biosensor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
