"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/Chromoproteins
From 2013.igem.org
Home
Project
Parts
Achievements
Safety
Scientific Interactions
Team
Notebook
Why Aspergillus nigem?
Secondary metabolites
Toolbox
Host engineering
Summary
Achievements
Attributions
Safety introduction
General safety
Fungi-related safety
Biosafety Regulation
Safety Improvement Suggestions
Safety of the Application
Masterclass
Science cafe
iGEM Netherlands
Collaborations
Conference
Vitruvian man
Members
Profile
Thanks!
Lablog
Experimental protocols
Chromoproteins
Simple bioreporters in synthetic biology
Introduction
Secondary metabolites
Toolbox
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Outline
We aim to establish Aspergillus niger as a promising host in synthetic biology and in iGEM in particular. Therefore we introduce a range of simple bio-reporters such as pH, ATP sensors and chromoproteins, to increase the usability of the Aspergillus toolbox. Here, we focus on chromoproteins which are pigments obtained from corals. Chromoproteins have been characterized, codon-optimized and standardized in E. coli before by the Uppsala 2011 team . However, even though multiple different pigment coding genes can be found in the standard biological parts registry , none have been tested in filamentous fungi.
Introduction
Coral colorization occurs due to the presence of a range of fluorescent and non-fluorescent pigments. Fluorescent proteins (remotely homologous to the famous Green Fluorescent Protein superfamily from jellyfish[ 1 ], see Fig.1) are an interesting family of proteins. Coral fluorescent proteins are small, about 230 amino acid residues (~700bp) long. They acquired via evolution the ability to synthesize a chromophore from their own residues in a couple of autocatalytic reactions. Chromophore structure and kinetics are highly significant to the optics of colorizing proteins [ 2 ]. These proteins lead to myriad imaging techniques that capitalize their unique physical, biochemical and spectral properties. Non-fluorescent pigments are made up by chromoproteins, which absorb light effectively but hardly emit it. They have intrinsic colour which is visible to the naked eye. This feature, as well as their relatively small gene size, make chromoproteins simple but favorable bio-reporters in molecular biology [ 1 ]. Most chromoproteins possess single absorption maxima around 560-590 nm. However, small shifts in the absorption maxima can already lead to compelling changes in the perceived color; the pigment might even appear blue. Among GFP homologs, chromoproteins are quite unique in having the natural potential of far-red fluorescence (590-640 nm). Furthermore, far-red fluorescent proteins can also be generated from chromoproteins via mutagenesis.[ 3 ] This property could lead to an interesting biotechnological application, as fluorescent near-infrared (650+ nm) reporters offer new possibilities for in vivo studies on biological functioning due to their ability of penetrating mammalian tissue via near-infrared light.
In addition, mutagenesis or amino acid substitutions in chromoproteins can lead to a huge increase (several hundred-fold) in their quantum yield and can influence colourization [ 4 ]. Chromophores in chromoproteins are usually characterized by a non-planar trans-conformation [ 5 ]. However, the differences in chromoprotein colour are thought to occur due to interactions between their chromophore and environmental factors such as pH [ 6 ]. This finding is highlighted by results that show that a chromoprotein chromophore, though having the same amino acid sequence as a fluorescent protein, adopts a different conformation (deviation in chromophore ring planarity) under different environmental conditions [ 2 , 5 ]. Chromoprotein structures are found in different lineages and have three independent origins, suggesting functional convergence [ 3 ] (Fig.1).
Fig.1 Origins of chromoprotein structures
Aim
1.Design and validate the applicability of a set of chromoproteins originating from corals in Aspergillus niger.
2.Introduce the mitochondrial retention signal to chromoprotein coding genes for targeted localization.
3.Standardize the measurement of chromoprotein encoding gene induced colorization.
Approach
The XL1 Blue MRF’ E. coli strains which contain plasmids with chromoprotein encoding genes aeBlue , eforRed and amilGFP respectively and DH5α E. coli strain containing
mRFP were obtained from Braunschweig UR and Uppsala UR.   Then the palettes of those chromoprotein transcriptional units were tested for applicability in Aspergillus niger. Besides, we synthesized and then Gibson assembled a chromoprotein encoding gene (eforRed) which was codon-optimized for Aspergillus niger. A mitochondrial retention signal was added to this codon-optimized chromoprotein encoding gene for targeted organelle localization. Afterwards protoplasts were transformed with all chromoprotein constructs, normal and codon optimized, in order to be expressed in A. niger N593.
Results
The aeBlue, amilGFP, mRFP and eforRed chromoprotein genes were succesfully inserted into an in-house vector. After their sequence was confirmed by single read sequencing, protoplasts were transformed with these chromoprotein encoding genes. Then they were inoculated on complete medium plates without uridine supplement. We thereby created selective pressure since the A. niger N593 strain owns a uridine autroxophy gene.  However, unlike the E. coli transformants (Fig.2) the Aspergillus transformant colonies (Fig.3) did not show colour.
However, when performing PCR with the fungal genomic DNA as a template, the products appeared to be of the right size, giving the expected bands which were around 700bp after gel electrophoresis, see Fig 4.
The eforRed chromoprotein encoding gene was codon-optimized for A. niger and synthetic construct blocks were ordered and fused via Gibson Assembly. This synthetic construct contains a removable mitochondrial targeting sequence. The colony-PCR of E. coli transformed with this codon-optimized eforRed with mitochondrial retention sequence displayed the expected bands after gel electrophoresis, which were around 750 bp, see Fig.5. Afterwards, Aspergillus niger was transformed with the codon-optimized construct with mitochondrial retention signal. Today (05-10-2013) we noticed that the mycelium of one of the aforementioned transformed A. niger showed red colour (Fig.6)!!! The day before the wiki-freeze, we got the result we were longing for for so long!
We performed PCR on our construct in fungal DNA and on our construct in the biobrick backbone as a positive control this very evening before the wiki-freeze. It showed faint bands of the expected size (750 bp)(Fig.7)!
Lane 1 = marker, lane 2+3 = transformant fungal DNA, lane 5 = negative control, lane 7+8 = positive control (biobrick)
Fig. 7 PCR of transformant A. niger
Conclusions
The correct insertion of the non-coding optimized chromoprotein encoding genes in the fungus genomic DNA was confirmed by PCR. To further confirm whether the sequence of the chromoprotein encoding genes also remained unchanged, we sent the new constructs out for sequencing. Comparing the sequencing results to the original sequence from parts registry, for the aeBlue, mRFP and amilGFP chromoprotein gene, the identity was almost 100%. We thereby showed that the palettes of chromoprotein transcriptional units for A. niger were designed and transformed successfully. Additionally, we optimized the codon of eforRed chromoprotein encoding gene for A. niger and introduced mitochondrial retention signal for targeted localization. The transformation of A. niger with the optimized constructs was successfully validated via PCR and its mycelium showed the red colour as we expected. For the PCR however the bands for amplified transformant fungal DNA were not really intense compared to the bands for the amplified positive controls. The reason for this could be that our biobricking primers only partly annealed to our codon-optimized construct with signal sequence in the fungal DNA, while they fully anneal to our construct in biobrick vector.
To conclude, the codon optimized eforRed chromoprotein could be introduced into A. niger and applied as a simple selective marker or bioreporter. For the others whose mycelium didn’t show colour, we assume that it is either related to transcription or translation of the chromoprotein encoding gene in A. niger. Future analysis, such as RT-PCR is suggested to check if the mRNA is transcribed properly. A good alternative would be to add a His tag, purify the chromoprotein and thereby determine translation.
References
1.Alieva, N.O., et al., Diversity and evolution of coral fluorescent proteins. Plos One, 2008. 3(7): p. e2680.
2.Chan, M.C., et al., Structural characterization of a blue chromoprotein and its yellow mutant from the sea anemone Cnidopus japonicus. Journal of Biological Chemistry, 2006. 281(49): p. 37813-37819.
3.Shagin, D.A., et al., GFP-like proteins as ubiquitous metazoan superfamily: evolution of functional features and structural complexity. Molecular biology and evolution, 2004. 21(5): p. 841-850.
4.Prescott, M., et al., The 2.2 Å crystal structure of a pocilloporin pigment reveals a nonplanar chromophore conformation. Structure, 2003. 11(3): p. 275-284.
5.Shkrob, M., et al., Far-red fluorescent proteins evolved from a blue chromoprotein from Actinia equina. Biochem. J, 2005. 392: p. 649-654.
6.Chalfie, M. and S.R. Kain, Methods of Biochemical Analysis, Green Fluorescent Protein: Properties, Applications and Protocols2005: Wiley-Liss.
Retrieved from " http://2013.igem.org/Team:Wageningen_UR/Chromoproteins "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
