<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/Notebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="topmenu"><UL class="menucenter"><LI><A href="/Team:Wageningen_UR" title="Team:Wageningen UR">Home</A></LI><LI class="project"><A href="/Team:Wageningen_UR/Why_Aspergillus_nigem" title="Team:Wageningen UR/Why Aspergillus nigem">Project</A></LI><LI class="parts"><A href="/Team:Wageningen_UR/Parts" title="Team:Wageningen UR/Parts">Parts</A></LI><LI class="achievements"><A href="/Team:Wageningen_UR/Achievements" title="Team:Wageningen UR/Achievements">Achievements</A></LI><LI class="safety"><A href="/Team:Wageningen_UR/Safety_introduction" title="Team:Wageningen UR/Safety introduction">Safety</A></LI><LI class="human"><A href="/Team:Wageningen_UR/Masterclass" title="Team:Wageningen UR/Masterclass">Scientific Interactions</A></LI><LI class="team"><A href="/Team:Wageningen_UR/Team" title="Team:Wageningen UR/Team">Team</A></LI><LI class="notebook"><STRONG class="selflink">Notebook</STRONG></LI></UL></DIV><UL class="sub-project subs"><LI><A href="/Team:Wageningen_UR/Why_Aspergillus_nigem" title="Team:Wageningen UR/Why Aspergillus nigem"><SPAN>Why Aspergillus nigem?</SPAN></A></LI><LI><A href="/Team:Wageningen_UR/Secondary_metabolites" title="Team:Wageningen UR/Secondary metabolites">Secondary metabolites</A></LI><LI><A href="/Team:Wageningen_UR/Biosensors" title="Team:Wageningen UR/Biosensors">Toolbox</A></LI><LI><A href="/Team:Wageningen_UR/Host_engineering" title="Team:Wageningen UR/Host engineering">Host engineering</A></LI><LI><A href="/Team:Wageningen_UR/Summary" title="Team:Wageningen UR/Summary">Summary</A></LI></UL><UL class="sub-achievements subs"><LI><A href="/Team:Wageningen_UR/Achievements" title="Team:Wageningen UR/Achievements">Achievements</A></LI><LI><A href="/Team:Wageningen_UR/Attributions" title="Team:Wageningen UR/Attributions">Attributions</A></LI></UL><UL class="sub-safety subs"><LI><A href="/Team:Wageningen_UR/Safety_introduction" title="Team:Wageningen UR/Safety introduction">Safety introduction</A></LI><LI><A href="/Team:Wageningen_UR/General_safety" title="Team:Wageningen UR/General safety">General safety</A></LI><LI><A href="/Team:Wageningen_UR/Fungi-related_safety" title="Team:Wageningen UR/Fungi-related safety">Fungi-related safety</A></LI><LI><A href="/Team:Wageningen_UR/Biosafety_Regulation" title="Team:Wageningen UR/Biosafety Regulation">Biosafety Regulation</A></LI><LI><A href="/Team:Wageningen_UR/Safety_Improvement_Suggestions" title="Team:Wageningen UR/Safety Improvement Suggestions">Safety Improvement Suggestions</A></LI><LI><A href="/Team:Wageningen_UR/Safety_of_the_Application" title="Team:Wageningen UR/Safety of the Application">Safety of the Application</A></LI></UL><UL class="sub-human-practices subs"><LI><A href="/Team:Wageningen_UR/Masterclass" title="Team:Wageningen UR/Masterclass">Masterclass</A></LI><LI><A href="/Team:Wageningen_UR/Science_cafe" title="Team:Wageningen UR/Science cafe">Science cafe</A></LI><LI><A href="/Team:Wageningen_UR/IgemNL" title="Team:Wageningen UR/IgemNL">iGEM Netherlands</A></LI><LI><A href="/Team:Wageningen_UR/Collaborations" title="Team:Wageningen UR/Collaborations">Collaborations</A></LI><LI><A href="/Team:Wageningen_UR/Conference" title="Team:Wageningen UR/Conference">Conference</A></LI><LI><A href="/Team:Wageningen_UR/Vitruvian_man" title="Team:Wageningen UR/Vitruvian man">Vitruvian man</A></LI></UL><UL class="sub-team subs"><LI><A href="/Team:Wageningen_UR/Team" title="Team:Wageningen UR/Team">Members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Wageningen_UR" target="_blank">Profile</A></LI><LI><A href="/Team:Wageningen_UR/Sponsors" title="Team:Wageningen UR/Sponsors">Thanks!</A></LI></UL><UL class="sub-notebook subs"><LI><STRONG class="selflink">Lablog</STRONG></LI><LI><A href="/Team:Wageningen_UR/Experimental_protocols" title="Team:Wageningen UR/Experimental protocols">Experimental protocols</A></LI></UL><DIV id="filters"><DIV class="bysubject"><SPAN>Filter by subject</SPAN><DIV class="icons"><A class="wetlab">wet lab</A><A class="drylab">dry lab</A><A class="modeling">modeling</A><A class="human">human practices</A><A class="other">other</A></DIV></DIV><DIV class="bydate"><DIV class="iconsweek"><A href="#week17" class="week17"><SPAN>week 17</SPAN></A><A href="#week16" class="week16"><SPAN>week 16</SPAN></A><A href="#week15" class="week15"><SPAN>week 15</SPAN></A><A href="#week14" class="week14"><SPAN>week 14</SPAN></A><A href="#week13" class="week13"><SPAN>week 13</SPAN></A><A href="#week12" class="week12"><SPAN>week 12</SPAN></A><A href="#week11" class="week11"><SPAN>week 11</SPAN></A><A href="#week10" class="week10"><SPAN>week 10</SPAN></A><A href="#week9" class="week9"><SPAN>week 9</SPAN></A><A href="#week8" class="week8"><SPAN>week 8</SPAN></A><A href="#week7" class="week7"><SPAN>week 7</SPAN></A><A href="#week6" class="week6"><SPAN>week 6</SPAN></A><A href="#week5" class="week5"><SPAN>week 5</SPAN></A><A href="#week4" class="week4"><SPAN>week 4</SPAN></A><A href="#week3" class="week3"><SPAN>week 3</SPAN></A><A href="#week2" class="week2"><SPAN>week 2</SPAN></A><A href="#week1" class="week1"><SPAN>week 1</SPAN></A><A class="month may" href="#may"><SPAN>may</SPAN></A><A href="#april" class="april"><SPAN>april</SPAN></A><A href="#march" class="march"><SPAN>march</SPAN></A><A href="#february" class="february"><SPAN>february</SPAN></A><A href="#january" class="january"><SPAN>january</SPAN></A></DIV><DIV class="months"><P><SPAN class="spring">Spring</SPAN><SPAN class="june">June</SPAN><SPAN class="july">July</SPAN><SPAN class="aug">August</SPAN><SPAN class="sept">September</SPAN></P></DIV></DIV></DIV><DIV id="container"><H2><SPAN class="mw-headline" id="notebook_entries"> notebook entries </SPAN></H2><DIV class="item date" id="week18"><DIV class="midden"><H3>Week 18</H3><P>September (30.09 - 06.10)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Magic hand works again for ligation</H3><P class="date">02.10.2013</P><P> The ligation for KS domain in vector works finally. PS: This breaking news came out after 9PM. Hard working now return back!</P></DIV></DIV><DIV class="item date" id="week17"><DIV class="midden"><H3>Week 17</H3><P>September (23.09 - 29.09)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Transformation protoplast with shuttle vectors containing constructs from Cornell UR iGEM team</H3><P class="date">24.09.2013</P><P> As collaborating with Cornell iGEM team, they sent us the Cre recombinase, as well as hygromycin and geneticin resistances behind the <I>Aspergillus</I> promoter and a sample of geneticin. We transformed protoplast with shuttle vectors containing those constructs to test in our <I>A. Niger</I> host. </P></DIV></DIV><DIV class="item date" id="week16"><DIV class="midden"><H3>Week 16</H3><P>September (16.09 - 22.09)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> The invaluable painting Sunflower in our lab </H3><P class="date">20.09.2013</P><P> Marjan made a counterfeit of Van Gogh's sunflower using agar plates and some mysterious microorganism by her magic hand. What a masterpiece and makes our lab more colorful. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Transformation protoplast with codon optimized eforRed constructs </H3><P class="date">18.09.2013</P><P> We transformed protoplasts <I>Aspergillus niger</I> N593 with plasmids containing codon-optimized chromoprotein encoding genes. Since the in-house shuttle vectors contained the pyrA gene, allowing them to grow without uridine, we selected successful transformants via uridine-dependence. </P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Science Cafe Wageningen</H3><P class="date">16.09.2013</P><P>On 16th of September, there was an awesome gathering for all biologists in café Loburg. It was organized by iGEM team with some tipps and advices from the science café team. This edition was about “synthetic biology, towards the creation of life?” and the speakers were Prof dr. Arnold Driessen (University of Groningen) and dr. Dirk Stemerding (Rathenau institute) and as the side-kick Pieter van Boheemen , who is a DIY biohacker. It turned to be a really success!</P></DIV></DIV><DIV class="item date" id="week15"><DIV class="midden"><H3>Week 15</H3><P>September (09.09 - 15.09)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> New primers to clone GFP+Actin construct for parts registry </H3><P class="date">13.09.2013 – 15.09.2013</P><P> We ordered primers with standard 10 restriction sites for cloning our GFP+Actin construct into the recommended plasmid for parts registry. With these primers,  actin-GFP constructs were amplified by PCR. Then PCR products and parts registry plasmids were digest to obtain the compatible restriction sites and ligated to each other. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Alignment of Chromoprotein sequencing results </H3><P class="date">10.09.2013</P><P> We checked whether the sequence of the chromoprotein encoding genes had remained unchanged by sending it out for sequencing. Then the results were compared with the original sequence from parts registry. For aeBlue, mRFP and amilGFP  chromoprotein gene, the identity was almost 100%. </P></DIV></DIV><DIV class="item date" id="week14"><DIV class="midden"><H3>Week 14</H3><P>September (02.09 - 08.09)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Extraction of fungus genome DNA and validation insertion via PCR</H3><P class="date">06.09.2013</P><P>  The aeBlur, eforRed, mRFP and amilGFP transformants were culture in the CM- medium. After 2 days, their mycelium was harvested and used to extract fungus genome DNA. With the genome DNA as templates and corresponding chromoprotein primers, we did PCR to validate insertion of the chromoprotein encoding gene. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Sequencing result of QC-PCR products </H3><P class="date"> 05.09.2013</P><P> With Quick-change PCR,  actin gene was mutated to get rid of the xba1 restriction site. The site-directed mutagenesis was confirmed by sequencing result. Comparing with the original sequence, T was changed into G. Then it will be double checked by restriction analysis with double digestion “XbaI + NsiI”. </P></DIV></DIV><DIV class="item date" id="week13"><DIV class="midden"><H3>Week 13</H3><P>August (26.08 - 1.09)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Site-directed mutagenesis of actin gene </H3><P class="date">27.08.2013</P><P> We designed special primers that induced a point mutation to remove the xba1 restriction site. Quick Change Lightning kit was used to QC-PCR of p-Jet with actin gene. Then the PCR products was digested with Dpn1 to cut off methylated sites. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Work in progress</H3><P class="date">26.08.2013</P><P>Performed some more Gibson assemblies and tried to ligate DH and MT together in the same vector. Made chemically competent cells and the primers for our mini-genes arrived.</P></DIV></DIV><DIV class="item date" id="week12"><DIV class="midden"><H3>Week 12</H3><P>August (19.08 - 25.08)</P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>iGEM Netherlands</H3><P class="date">24.08.2013</P><P> We works really hard to participate and organize iGEM activities and now deserve a rainy BBQ as the end of a busy communication day. It is not only about eating, we are actually talking about science mainly.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Microscopic pictures of actin cytoskeleton</H3><P class="date">23.08.2013</P><P>From the actin GFP fusion pictures were made under a fluorescent microscope. Beforehand the <I>A.niger</I> transformantss were grown under different conditions in an 8-well plate specially designed for microscopy.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Gibson assembly</H3><P class="date">22.08.2013</P><P>The colony PCR’s of DH, MT, ACP and eforRed are successful. Performed another Gibson assembly with KS.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Ligation of proton-ATPase gene into in-house brick</H3><P class="date">22.08.2013</P><P>Finally we managed to ligate the proton-ATPase used for the visualization of the septa into an in-house brick containing an n-terminal GP fusion.</P></DIV></DIV><DIV class="item modeling"><DIV class="content"><H3>Improving the <I>A. terreus</I> model</H3><P class="date">21.08.2013</P><P>As it turns out, the lovastatin pathway in the published A. terreus model is not balanced and optimization for its production is therefore impossible. This is actually an interesting finding, since it implies that we can now improve the current model by adding the detailed lovastatin biosynthesis pathway that is balanced and allows for optimization of lovastatin production.
 </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Gibson assembly</H3><P class="date">21.08.2013</P><P>Today we did our first Gibson assemblies of DH, MT, ACP, eforRed and ATP. Including transformations.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Working on the Wiki</H3><P class="date">20.08.2013</P><P>The wiki is making good progress, everybody should be able to work with the system now. Objectives are to upload content, look for icons that can be used in the menu bar and to finish the team page</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Transformations</H3><P class="date">19.08.2013</P><P>We have two kinds of competent <I>E.coli</I> cells to do the transformation. Both of the Electro-competent cells and chemical-competent work.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>G-blocks from IDT arrived today</H3><P class="date">19.08.2013</P><P>Today the G-blocks we ordered finally arrived! This is great news, because now we can start on the gibson assembly and transformations.</P></DIV></DIV><DIV class="item date" id="week11"><DIV class="midden"><H3>Week 11</H3><P>August (12.08 - 18.08)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>iGem Olympics: Second round</H3><P class="date">18.08.2013</P><P>We played pool during the second round of the iGem Olympics. Also to celebrate Shreyans Birthday!</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Sequencing results arrived</H3><P class="date">15.08.2013</P><P>The sequencing results from the proton-ATPase arrived and it appeared that one of the two samples was correct and work could be continued. </P></DIV></DIV><DIV class="item modeling"><DIV class="content"><H3><I>A. terreus </I> model</H3><P class="date">14.08.2013</P><P>Recently an A. terreus model has been published. This is quite interesting, since A. terreus is the organisms that naturally produces lovastatin. A quick scan for metabolites shows that both dihydromolacolin L and lovastatin are included in the model. Altough reactions have been lumped, now we can also investigate the potential differences between the lovastatin pathway present and the one we constructed ourselves. 
- J. Lui et al., 2013. <I> Genome-scale reconstruction and in silico analysis of Aspergillus terreus metabolism. </I>. Molecular Biosystems, Vol. 9, p. 1939-1948.
 </P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Molecular Interactions Symposium Berlin</H3><P class="date">14.08.2013</P><P>We went to the Molecular Interactions symposium in Berlin to present a poster about our project! We were kind of surprised that there were no other iGem teams.
The interaction we had with the interested people lead to a couple of nice suggestions to the poster and the project. We even had a good discussion about Gibson Assembly.
The interesting speakers there, nice ambience and good dinner in the evenings these 2 days totally made up for the 2 times 7 hours of midnight driving!
</P></DIV></DIV><DIV class="item date" id="week10"><DIV class="midden"><H3>Week 10</H3><P>August (05.08 - 11.08)</P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Vitruvian man stamp</H3><P class="date">11.08.2013</P><P>We looked into using Vitruvian man metal stamps to make imprints in the agar.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>pH</H3><P class="date">10.08.2013</P><P>Finally gibson assembly worked for ph-sensor into pJet1.2. Here's a picture showing the digestion analysis using; nsiI and notI; pstI and notI</P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Workshop by Paulien Poelarends</H3><P class="date">09.08.2013</P><P>Paulien did her master thesis on last year’s iGEM teams and their way of communicating with the public and politicians. We have a wonderful afternoon with her, obtaining a lot by her workshop. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Protoplasts</H3><P class="date">08.08.2013</P><P>During this week we have prepared media and spores to make protoplasts. The protoplasts will be used for transformations. Until recently Novozym has been used to make protoplasts, this is not available anymore and some new protocols have to be used.</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Lovastatin</H3><P class="date">06.08.2013</P><P>The restriction enzymes that we ordered last week have arrived. After a few attempts the sequences have been optimized and the G-blocks can be ordered. Also some Lovastatin has been ordered to perform some experiments with <I>A. niger</I>. Lovastatin may be fatal to cell growth of <I>A. niger</I> therefore experiments have to indicate whether <I>A. niger</I> is resistant to Lovastatin. Otherwise <I>A. niger</I> has to be transformed with the resistance gene from <I>A. terreus</I>.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3> Plasmids extraction with Kit</H3><P class="date">06.08.2013</P><P> Using GeneJET Plasmid Miniprep Kit, we extracted the plasmids containing chromoprotein genes from DH5α <I>E.coli</I> and checked their concentration with Nanodrop 1000.</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Expanding the database</H3><P class="date">05.08.2013</P><P> In order to expand the database, more information on secondary metabolite backbone enzymes from Aspergilli needs to be added. Luckily there is a recent publication on secondary metabolites from A. nidulans, A. fumigatus, <I>A. niger</I> and <I>A. oryzae</I>. 
- D.O. Inglis et al., 2013. <I> Comprehensive annotation of secondary metabolite biosynthetic genes and gene clusters of <I>Aspergillus nidulans</I>, <I>A. fumigatus</I>, <I>A. niger</I> and <I>A. oryzae</I>. BMC Microbiology, Vol. 13, p. 1-23.</I></P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Chemical transformation </H3><P class="date">05.08.2013</P><P> The plasmids containing chromoprotein genes were transformed into DH5α <I>E.coli</I> competent cells and we set up the same background within the DH5α <I>E.coli</I> strains .</P></DIV></DIV><DIV class="item date" id="week9"><DIV class="midden"><H3>Week 9</H3><P>July (29.07 - 04.08)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>iGem BBQ</H3><P class="date">02.08.2013</P><P>We finally had our first group BBQ! Together with all people that are involved with our team! It was a really nice day.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Design the primers for Chromoprotein</H3><P class="date">02.08.2013</P><P>The primers for aeBlue, eforRed, amilGFP and mRFP chromoprotein genes were designed with Clone Manager.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>First transformation of <I>A. niger</I></H3><P class="date">01.08.2013</P><P>Finally it was time to transform <I>A. niger</I> with the first construct we got. This would be the actin GFP fusion. This was introduced into <I>A. niger</I> and transormation worked as it appeared 4 days later.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Shopping</H3><P class="date">30.07.2013</P><P> This week we ordered the restriction enzymes for the assembly strategy. Also other stuff should be ordered like pJET, lovastatin, <I>A. niger</I>&amp;<I>A. terrus</I> strain, Gibson assembly kit...
        </P></DIV></DIV><DIV class="item date" id="week8"><DIV class="midden"><H3>Week 8</H3><P>July (22.07 - 28.07)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Obtained <I>E.coli</I> expressing chromoproteins from Braunschweig</H3><P class="date">26.07.2013</P><P>We obtained the <I>E.coli</I> from Braunschweig that have plasmids with an chromoprotein encoding gene insert. Lets start introducing it into <I>Aspergillus Niger</I>!</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Actin gene was ligated in in-house vector</H3><P class="date">24.07.2013</P><P>It was time to ligate the actine gene into the in-house vector containing an n-terminal GFP fusion and transform <I>E. coli</I> with this vector.</P></DIV></DIV><DIV class="item modeling"><DIV class="content"><H3>Setting a standard</H3><P class="date">23.07.2013</P><P>Metanetx.org uses its own namespace to standardize the entities that are used within the models, such as metabolites and the reactions. Standardization makes working with different models from different origins much more feasible. However, Metanetx is a project in the making, which means that even though it facilitates the process, manual curation is required. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Metabolic activity: GFP</H3><P class="date">22.07.2013</P><P> In a similar way that the DAPI stained cells give an indication of metabolic activity by showing an increase in nuclei, GFP-transformed <I>A. niger</I> also give an indication of metabolic activity by giving an indication on whether transcription and translation occur.
            </P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Splitting up the domains</H3><P class="date">22.07.2013</P><P> Luckily, we got some help from Kal, a really nice guy who offered a lot of help. Several very useful websites he suggested could analyse the AA sequence and give some advice on where is the possible site to split the whole protein. We combined the analysis result with references, we finally got a clear idea (or at least we believe) about the boundary of the domains.
            </P></DIV></DIV><DIV class="item date" id="week7"><DIV class="midden"><H3>Week 7</H3><P>July (15.07 - 21.07)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Working on the Wiki</H3><P class="date">19.07.2013</P><P>The design of the wiki is making good progress. Basic layout has been determined and we created icon to indicate different parts of the project.</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Article searching</H3><P class="date">18.07.2013</P><P>Totally lost in how to divide each single domain in the lovB gene. By searching the articles, we could find information and experimental details about ACP, KS, MAT and CON domain. While for the others, still virgin land. KR seems to have two subdomains, what the hell~</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Database design</H3><P class="date">16.07.2013</P><P> MySQL the world's most widely used open-source relational database management system. Now we will also check the possibility to use it as our method.</P></DIV></DIV><DIV class="item date" id="week6"><DIV class="midden"><H3>Week 6</H3><P>July (08.07 - 14.07)</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Lovastatin strategy</H3><P class="date">12.07.2013</P><P>Now that we have the general idea the details need to be worked out. Therefore we came up with a strategy to assemble the modules using compatible restriction enzymes. The strategy will enable us to assemble several modules and to have a stop codon at the end of the assembled gene without a frame shift. Also the general planning of the project was worked out this week. </P></DIV></DIV><DIV class="item modeling"><DIV class="content"><H3>Expanding the scope</H3><P class="date">12.07.2013</P><P>To expand the scope of the modeling and create a larger overlap with the secondary metabolite backbone enzyme database we have chosen to expand the number of models. A. oryzae and A. nidulans have been added to the scope of this project. Expansion of the modeling with these three additional Aspergilli allows for a comparative approach in which results can be compared. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>DAPI staining</H3><P class="date">11.07.2013</P><P> It has been found that dormant conidia are predominantly bi-nucleate (85%), the remainder being uni-nucleate. Therefore, if one stains the giant cells with DAPI, which colours the nuclei, one can assess whether the nuclei within the cells are actively dividing. This appears to be the the case and thus indicates that the cells are not in a vegetative state.
            </P></DIV></DIV><DIV class="item date" id="week5"><DIV class="midden"><H3>Week 5</H3><P>July (01.07 - 07.07)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>BioKe meeting in Wageningen</H3><P class="date">05.07.2013</P><P>A BioKe spokesperson came over to Wageningen. We gave her a tour and presentation. We obtained the Q5 PCR kit!</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Lovastatin</H3><P class="date">05.07.2013</P><P>Finally the search has begun, this week we have started with literature research for the Lovastatin project. Some topics were for example what is Lovastatin, which genes are involved in Lovastatin production in <I>Aspergillus terreus</I>, and how to measure Lovastatin production. The first idea was to introduce the Lovastatin pathway of <I>A. terreus</I> into <I>A.niger</I>. During the research we found that one of the bigger genes involved is composed of several domains. Our new idea now is to synthesize these domains, making several modules that we can assemble as we wish. </P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Science Café</H3><P class="date">04.07.2013</P><P>We met with the Science Café organisation to discuss the possibilities towards our team organising an evening there with a Biohacker theme.</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Database planning</H3><P class="date">03.07.2013</P><P> A database of <I>Aspergillus niger</I> is planned to set. Thus we can get insight to comprehensive annotation of secondary metabolite biosynthetic genes and gene clusters of it.
            </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Primers for septa visualizition arrived</H3><P class="date">02.07.2013</P><P>After some waiting time the primers needed for the visualization of the septa using a proton-ATPase finally arrived and this part of the project could also get started by isolating the gene from genomic <I>A. niger</I> DNA.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Sequensing results arrived</H3><P class="date">01.07.2013</P><P>The sequencing results from the actin gene arrived and everything looked fine. We had the right gene without mistakes and could go on with the actin cytoskeleton project.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Actin gene in house internal brick system</H3><P class="date">01.07.2013</P><P>The actin gene for cytoskeleton visualization purposes was ligated into a house internal brick system with a n-terminal GFP fusion. This construct was then introduced into <I>E. coli</I> for amplification purposes.</P></DIV></DIV><DIV class="item date" id="week4"><DIV class="midden"><H3>Week 4</H3><P>June (24.06 - 30.06)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Complimentary Supplies from BIOKE</H3><P class="date">26.06.2013</P><P>Complimentary cloning kits arrived today from one of our sponsors, BIOKE.</P></DIV></DIV><DIV class="item modeling"><DIV class="content"><H3>Lovastatin Pathway</H3><P class="date">25.06.2013</P><P>The lovastatin pathway has been added to the metabolic model of <I>Aspergillus niger</I>. Because the medium composition of the model is not yet properly defined, the maximum flux towards lovastatin can be achieved. This however is not realistic and therefore the next thing is to redefined the medium composition.  
- B.D. Ames et al., 2011. <I> Crystal structure and biochemical studies of the trans-acting polyketide enoyl reductase LovC from lovastatin biosynthesis. </I> PNAS, doi/10.1073/pnas.1113029109
        </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Calcofluor staining</H3><P class="date">24.06.2013</P><P> Calcofluor is known to stain cellulose and chitin. In <I>A.niger</I>, chitin is found in the cell wall, thus if we find cells in this specific stage of mitosis, just before division is completed and when the cytoplasm is connected, we might be able to visualize this with the use of this staining method. </P></DIV></DIV><DIV class="item date" id="week3"><DIV class="midden"><H3>Week 3</H3><P>June (17.06 - 23.06)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Finding the right conditions for distinct phenotypes</H3><P class="date">22.06.2013</P><P> As described in literature, N593 formed giant cells after 24h at 44C. Since transcriptome data on N400 from multiple stages in its life cycle is available, this strain is chosen such that this research complements the current RNA landscape profile of <I>A. niger</I> and allows for comparison of data from different life stages. However, unlike N400, N593 repeatedly formed mycelium at 44C. To overcome this effect the temperature was increased to 45C, at which a single cell phenotype was obtained for N593.
 </P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Do It Yourself Biohacker Meeting 2</H3><P class="date">20.06.2013</P><P>We had a DIY Biohacker Meeting in the Waag, Amsterdam. The Waag had its grand opening and there was a exposition and party afterwards</P></DIV></DIV><DIV class="item human"><DIV class="content"><H3>Do It Yourself Biohacker Meeting 1</H3><P class="date">18.06.2013</P><P>We had a DIY Biohacker Meeting in the Waag, Amsterdam. Learned how to make your own potato-agar</P></DIV></DIV><DIV class="item date" id="week2"><DIV class="midden"><H3>Week 2</H3><P>June (10.06 - 16.06)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Funny group picture</H3><P class="date">15.06.2013</P><P>Opportunity was presented to take a few fun group pictures... we obviously took it</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Dinner and discussion</H3><P class="date">14.06.2013</P><P>We met at Mark’s place for dinner and evaluation of possibilities in using secondary metabolites for our project</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Host Engineering: Research plan</H3><P class="date">13.06.2013</P><P> Different environmental conditions induce changes in phenotypic cellularity of <I>Aspergillus niger</I>. Mapping reads onto the reference genome allows for discovery of patterns in gene expression that are unique to the single cell phenotype.
 </P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Last and 8th recruit</H3><P class="date">13.06.2013</P><P>Yeng Ding AKA Danny joind our team. We are now 8 members strong!</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>ThinkerCell &amp; CloneManager meeting</H3><P class="date">12.06.2013</P><P>The ones from our team who had learned to work with these software tools, during one of the iGem Netherlands courses, taught the others how use it. </P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Lab supplies</H3><P class="date">11.06.2013</P><P>We ordered reagents for cloning and PCR. </P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Collaboration</H3><P class="date">11.06.2013</P><P>We contacted the Cornell University iGem 2013 team to talk about possibilities for collaboration. They also work with fungus. </P></DIV></DIV><DIV class="item date" id="week1"><DIV class="midden"><H3>Week 1</H3><P> June (03.06 - 09.06)</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Actin primers arrived</H3><P class="date">06.06.2013</P><P>Finally the improved pair of actin primers arrived and after not getting good PCR results for some time I could finally get started and optimize PCR conditions for gene amplification of the actin gene.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>House internal brick system</H3><P class="date">05.06.2013</P><P>The house internal brick system with a n-terminal GFP fusion was proven to be ok by restriction digestion and was ready to be worked with.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Host Engineering: Research Question</H3><P class="date">04.06.2013</P><P>
Research question: 
Finding sets of candidate genes causative to the single cell phenotype in <I>Aspergillus niger</I> by transcriptome analysis. 
Subquestions: 
- is <I>A. niger</I> metabolically active at 44C? 
- does <I>A. niger</I> divide at 44C?
 </P></DIV></DIV><DIV class="item date" id="may"><DIV class="midden"><H3>May</H3></DIV></DIV><DIV class="item modeling"><DIV class="content"><H3>Metabolic model <I> A. niger </I></H3><P class="date">30.05.2013</P><P> The current metabolic model of <I>A. niger</I> needs to be expanded to include the lovastatin pathway as known from literature. First the model has to be checked: is it balanced? can we perform a FBA on the current model? 
- M.R. Anderson et al., 2008. <I> Metabolic model integration of the bibliome, genome, metabolome and reactome of Aspergillus niger. </I> Molecular Systems Biology, Vol. 4, Article number 178; doi:10.1038/msb.2008.12
        </P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Meeting room</H3><P class="date">26.05.2013</P><P>We reserved a room for all future Monday and Thursday lunch meetings in the Forum.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Visualization of the actin cytoskeleton</H3><P class="date">27.05.2013</P><P>Finally the labwork for the visualization of the septa and actin cytoskeleton started. And the first thing to do was order primers and isolate genomic DNA from <I>A. niger</I>.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Plasmid isolation and DNA quantification</H3><P class="date">24.05.2013</P><P>Plasmid DNA containing the ATP and pH constructs was isolated by midi preparation and quantified by Nanodrop measurement.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Created a Gmail account</H3><P class="date">23.05.2013</P><P>We created a Gmail account. Let the spam begin!</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Meeting iGem Uppsala</H3><P class="date">18.05.2013</P><P>We met with the iGem Uppsala 2013 team in Uppsala.</P></DIV></DIV><DIV class="item wetlab"><DIV class="content"><H3>Glycerol stocks made</H3><P class="date">16.05.2013</P><P>All the <I>E. coli</I> stabs with constructs from Addgene and the Imamura lab were revived and glycerol stocks were made and preserved at -80'C.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Created a Twitter account</H3><P class="date">15.05.2013</P><P>We created a Twitter account. Hello world! Meet iGem Wageningen 2013~</P></DIV></DIV><DIV class="item drylab"><DIV class="content"><H3>Chromoprotein sequencing</H3><P class="date">13.05.2013</P><P>Looked into sequencing options: codon optimization to <I>Aspergillus Niger</I>, suffix/prefix, (illegal) restrictionsites. Let’s order some g-blocks!</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>ATP,pH bio-sensor constructs delivered</H3><P class="date">13.05.2013</P><P>The constructs ordered from Addgene, Boston and from the Imamura lab, Japan reached us today.</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Lab space</H3><P class="date">08.05.2013</P><P>We arranged our very own lab space!</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Host Engineering: why?</H3><P class="date">05.05.2013</P><P> The single-cell phenotype Aspergillus has a higher surface to volume ratio and results in a lower viscosity of the liquid broth, offering perspective on substantially increasing process yields when used in liquid fermentations. 
Besides from an industrial point of view it is also interesting from an evolutionary perspective, which couples application-oriented research using a directed evolution approach to a more fundamental research topic: the evolution of multicellularity.
 </P></DIV></DIV><DIV class="item date" id="april"><DIV class="midden"><H3>April</H3></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Good Lab Practice</H3><P class="date">24.04.2013</P><P>We had a lecture on good lab practice and safety in the lab, awesome! ;)</P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Task division</H3><P class="date">18.04.2013</P><P>Co-ordinator – Michiel
Secretary – Emiel
Sponsoring - Emiel
Safety - Jingjing
Lab manager – Shreyans
Strategy - Marit
Wiki – Michiel
Human Practice - Marjan
  </P></DIV></DIV><DIV class="item other"><DIV class="content"><H3>Influence of temperature on germination</H3><P class="date">15.04.2013</P><P>A paper from 1970 shows that at 44C <I>Aspergillus niger</I> does not germinate, but rather forms giant cells. This dimorphism is not unfamiliar within the fungal world, where at lower temperatures the mycelium is formed and at higher temperatures the yeast-like form is found. 
Anderson, J. G. and J. E. Smith (1972). <I> &quot;Effects of Elevated-Temperatures on Spore Swelling and Germination in Aspergillus Niger.&quot; </I> Canadian Journal of Microbiology 18(3): 289-297.
  </P></DIV></DIV><DIV class="item date" id="march"><DIV class="midden"><H3>March</H3></DIV></DIV><H2><SPAN class="mw-headline" id="javascript"> javascript </SPAN></H2></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Wageningen_UR/Notebook">http://2013.igem.org/Team:Wageningen_UR/Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/Notebook&amp;oldid=312402" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>