<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Wageningen_UR_Biosensors"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Wageningen_UR/Biosensors">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Wageningen_UR/Biosensors&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/Biosensors&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Wageningen_UR/Biosensors&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Wageningen_UR/Biosensors" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Wageningen UR/Biosensors</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="topmenu"><UL class="menucenter"><LI><A href="/Team:Wageningen_UR" title="Team:Wageningen UR">Home</A></LI><LI class="project"><A href="/Team:Wageningen_UR/Why_Aspergillus_nigem" title="Team:Wageningen UR/Why Aspergillus nigem">Project</A></LI><LI class="parts"><A href="/Team:Wageningen_UR/Parts" title="Team:Wageningen UR/Parts">Parts</A></LI><LI class="achievements"><A href="/Team:Wageningen_UR/Achievements" title="Team:Wageningen UR/Achievements">Achievements</A></LI><LI class="safety"><A href="/Team:Wageningen_UR/Safety_introduction" title="Team:Wageningen UR/Safety introduction">Safety</A></LI><LI class="human"><A href="/Team:Wageningen_UR/Masterclass" title="Team:Wageningen UR/Masterclass">Scientific Interactions</A></LI><LI class="team"><A href="/Team:Wageningen_UR/Team" title="Team:Wageningen UR/Team">Team</A></LI><LI class="notebook"><A href="/Team:Wageningen_UR/Notebook" title="Team:Wageningen UR/Notebook">Notebook</A></LI></UL></DIV><UL class="sub-project subs"><LI><A href="/Team:Wageningen_UR/Why_Aspergillus_nigem" title="Team:Wageningen UR/Why Aspergillus nigem"><SPAN>Why Aspergillus nigem?</SPAN></A></LI><LI><A href="/Team:Wageningen_UR/Secondary_metabolites" title="Team:Wageningen UR/Secondary metabolites">Secondary metabolites</A></LI><LI><STRONG class="selflink">Toolbox</STRONG></LI><LI><A href="/Team:Wageningen_UR/Host_engineering" title="Team:Wageningen UR/Host engineering">Host engineering</A></LI><LI><A href="/Team:Wageningen_UR/Summary" title="Team:Wageningen UR/Summary">Summary</A></LI></UL><UL class="sub-achievements subs"><LI><A href="/Team:Wageningen_UR/Achievements" title="Team:Wageningen UR/Achievements">Achievements</A></LI><LI><A href="/Team:Wageningen_UR/Attributions" title="Team:Wageningen UR/Attributions">Attributions</A></LI></UL><UL class="sub-safety subs"><LI><A href="/Team:Wageningen_UR/Safety_introduction" title="Team:Wageningen UR/Safety introduction">Safety introduction</A></LI><LI><A href="/Team:Wageningen_UR/General_safety" title="Team:Wageningen UR/General safety">General safety</A></LI><LI><A href="/Team:Wageningen_UR/Fungi-related_safety" title="Team:Wageningen UR/Fungi-related safety">Fungi-related safety</A></LI><LI><A href="/Team:Wageningen_UR/Biosafety_Regulation" title="Team:Wageningen UR/Biosafety Regulation">Biosafety Regulation</A></LI><LI><A href="/Team:Wageningen_UR/Safety_Improvement_Suggestions" title="Team:Wageningen UR/Safety Improvement Suggestions">Safety Improvement Suggestions</A></LI><LI><A href="/Team:Wageningen_UR/Safety_of_the_Application" title="Team:Wageningen UR/Safety of the Application">Safety of the Application</A></LI></UL><UL class="sub-human-practices subs"><LI><A href="/Team:Wageningen_UR/Masterclass" title="Team:Wageningen UR/Masterclass">Masterclass</A></LI><LI><A href="/Team:Wageningen_UR/Science_cafe" title="Team:Wageningen UR/Science cafe">Science cafe</A></LI><LI><A href="/Team:Wageningen_UR/IgemNL" title="Team:Wageningen UR/IgemNL">iGEM Netherlands</A></LI><LI><A href="/Team:Wageningen_UR/Collaborations" title="Team:Wageningen UR/Collaborations">Collaborations</A></LI><LI><A href="/Team:Wageningen_UR/Conference" title="Team:Wageningen UR/Conference">Conference</A></LI><LI><A href="/Team:Wageningen_UR/Vitruvian_man" title="Team:Wageningen UR/Vitruvian man">Vitruvian man</A></LI></UL><UL class="sub-team subs"><LI><A href="/Team:Wageningen_UR/Team" title="Team:Wageningen UR/Team">Members</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Wageningen_UR" target="_blank">Profile</A></LI><LI><A href="/Team:Wageningen_UR/Sponsors" title="Team:Wageningen UR/Sponsors">Thanks!</A></LI></UL><UL class="sub-notebook subs"><LI><A href="/Team:Wageningen_UR/Notebook" title="Team:Wageningen UR/Notebook">Lablog</A></LI><LI><A href="/Team:Wageningen_UR/Experimental_protocols" title="Team:Wageningen UR/Experimental protocols">Experimental protocols</A></LI></UL><DIV class="projectheader"><DIV class="content toolbox"><H1>How flush is the cell?</H1><H2>&quot;Seeing through is rarely seeing into.&quot; - Elizabeth Bibesco</H2></DIV></DIV><DIV id="wrapper"><DIV id="tabs"><UL><LI id="li_tab1" onclick="tab('tab1')"><A>Introduction</A></LI><LI id="li_tab2" onclick="tab('tab2')"><A>Secondary metabolites</A></LI><LI id="li_tab3" onclick="tab('tab3')" class="active"><A>Toolbox</A></LI><LI id="li_tab4" onclick="tab('tab4')"><A>Host engineering</A></LI><LI id="li_tab5" onclick="tab('tab5')"><A>Summary</A></LI></UL></DIV></DIV><DIV id="toolbox_content_area"><DIV id="tab1"><DIV class="tournav"><UL><LI class="firstbgsm fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Why_Aspergillus_nigem">Why <I>Aspergillus nigem</I>?</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites">Secondary metabolites</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Lovastatin">Lovastatin</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Modeling">Modeling</A></LI><LI class="smbgsm current">Biosensors</LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Infrastructure">Infrastructure</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Host_engineering">Host engineering</A></LI><LI class="last"><A href="https://2013.igem.org/Team:Wageningen_UR/Summary">Summary</A></LI></UL></DIV></DIV><DIV id="tab2"><DIV class="tournav"><UL><LI class="first fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Why_Aspergillus_nigem">Why <I>Aspergillus nigem</I>?</A></LI><LI class="smbg fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites">Secondary metabolites</A></LI><LI class="bgbg fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Lovastatin">Lovastatin</A></LI><LI class="bgsm fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Modeling">Modeling</A></LI><LI class="smbgsm current">Biosensors</LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Infrastructure">Infrastructure</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Host_engineering">Host engineering</A></LI><LI class="last"><A href="https://2013.igem.org/Team:Wageningen_UR/Summary">Summary</A></LI></UL></DIV></DIV><DIV id="tab3"><DIV class="tournav"><UL><LI class="first fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Why_Aspergillus_nigem">Why <I>Aspergillus nigem</I>?</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites">Secondary metabolites</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Lovastatin">Lovastatin</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Modeling">Modeling</A></LI><LI class="smbg current">Biosensors</LI><LI class="bgbg"><A href="https://2013.igem.org/Team:Wageningen_UR/Infrastructure">Infrastructure</A></LI><LI class="bgsm"><A href="https://2013.igem.org/Team:Wageningen_UR/Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Host_engineering">Host engineering</A></LI><LI class="last"><A href="https://2013.igem.org/Team:Wageningen_UR/Summary">Summary</A></LI></UL></DIV></DIV><DIV id="tab4"><DIV class="tournav"><UL><LI class="first fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Why_Aspergillus_nigem">Why <I>Aspergillus nigem</I>?</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites">Secondary metabolites</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Lovastatin">Lovastatin</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Modeling">Modeling</A></LI><LI class="smbgsm current">Biosensors</LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Infrastructure">Infrastructure</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Chromoproteins">Chromoproteins</A></LI><LI class="smbgsm"><A href="https://2013.igem.org/Team:Wageningen_UR/Host_engineering">Host engineering</A></LI><LI class="last"><A href="https://2013.igem.org/Team:Wageningen_UR/Summary">Summary</A></LI></UL></DIV></DIV><DIV id="tab5"><DIV class="tournav"><UL><LI class="first fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Why_Aspergillus_nigem">Why <I>Aspergillus nigem</I>?</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites">Secondary metabolites</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Lovastatin">Lovastatin</A></LI><LI class="fill"><A href="https://2013.igem.org/Team:Wageningen_UR/Modeling">Modeling</A></LI><LI class="smbgsm current">Biosensors</LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Infrastructure">Infrastructure</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Chromoproteins">Chromoproteins</A></LI><LI><A href="https://2013.igem.org/Team:Wageningen_UR/Host_engineering">Host engineering</A></LI><LI class="last bg"><A href="https://2013.igem.org/Team:Wageningen_UR/Summary">Summary</A></LI></UL></DIV></DIV></DIV><DIV id="pagewrapper"><H2><SPAN class="mw-headline" id="Outline"> Outline </SPAN></H2><P>Bio-sensors are invaluable tools to easily visualize and track changes in the cell and its environment. Protein based bio-sensors have the advantage of being easily targeted to any part of the eukaryotic cell. Thus they are important components of the <I>Aspergillus</I> arsenal, which is an array of parts and devices to make <I>Aspergillus niger</I>  more amenable to work with. <I>Aspergillus niger</I> is an industrial workhorse that is a great producer of organic acids and can produce many heterologous eukaryotic proteins. We envision the use of ATP and pH bio-sensors to monitor the micro-environment in the cell where a pathway of interest has been targeted, in our case, the lovastatin biosynthesis pathway.
</P><H2><SPAN class="mw-headline" id="Introduction"> Introduction </SPAN></H2><P>A great array of new and existing biological devices and circuits have been designed and streamlined for useful purposes. However, currently there is not enough emphasis on how these devices and circuits respond to changes in their physiological surroundings, such as the effect of changes in pH and the availability of ATP. Thus we tested bio-sensors that enable us to monitor the dynamic changes of pH and ATP in live cells. The ATP bio-sensor is based on the phenomenon of fluorescence resonance energy transfer (FRET) and the pH bio-sensor is based on changes in the ionic state of the recombinant red fluorescence protein (pHRed). By combining multicolour, pHRed (red fluorescence) and FRET (green fluorescence), we can simultaneously image intracellular pH and ATP, as changes in metabolism often correlate with changes in pH and ATP concentrations in the cell. This would be interesting to determine whether the pH and ATP levels, at a certain position within the cell,  are optimal for a certain reaction. This will allow us to direct and target a metabolic pathway into its appropriate compartments for optimal production. These bio-sensors are useful to non-invasively quantify specific measurements in living cells. 
</P><B>pHRed</B><P>A novel approach in measuring intracellular pH in a non-invasive way has been developed by Mathew Tantama et. al. 2011, by using a recombinant red fluorescent protein (RFP), termed mKeima, had been genetically engineered into a S213A mutant, called pHRed. This will allow it to exhibit dual excitation peaks at 440 nm and 585 nm and an emission peak at 610 nm that respond ratiometrically well to pH changes [<A href="#ref5">5</A>]. The advantages of using a ratiometric method is that the estimation of pH is not affected by photobleaching, changes in focus of the microscope, variations in laser intensity, optical path length and illumination intensity. The excitation peak at 585 nm is caused by the anionic chromophore in an acidic condition whereas the excitation peak at 440 nm is caused by the protonated neutral chromophore in an alkaline condition.</P><P class="caption">Figure 1. How different pH levels affect the excitation peaks of pHRedA. A: pHRed in acidic condition has an excitation peak at 585 nm with an emission peak at 610 nm. B: pHRed in alkaline condition has an excitation peak at 440 nm with an emission peak 610 nm.</P><P>
The pH can be calculated by measuring the fluorescence intensity ratio between F585 and F440 (F585/ F440); and compare it with a calibration curve, see FIG. pH responses to ratio intensity of F585/ F440 are not affected in different buffer concentration such as Na+, K+, Mg2+, Cl−, Ca2+, HCO3−, H2O2, DTT, and temperature (21°C to 37°C). Based on these findings, pHRed had proven to be useful in monitoring and analyzing the dynamic changes in pH. 
</P><P class="caption">Figure 2. [<A href="#ref5">5</A>] </P><B>ATP FRET biosensor</B><P>
Fluorescence resonance energy transfer (FRET) is a phenomenon widely exploited by bio-sensors to monitor concentrations and temporal fluctuations of metabolites and ions at cellular and sub-cellular level. FRET works by excitation of a fluorescent molecule (donor) by a light of particular wavelength, which consequently transfers this energy to an adjacent fluorescent molecule (acceptor) that in-turn emits light. This phenomenon is very sensitive to the distance between the donor and acceptor fluorophore groups. Thus, fusing fluorescent proteins with a sensing domain that undergoes big conformational changes upon binding of the sensory molecule confers the possibility for generating an assortment of custom-made genetically encoded biosensors. These are useful tools to non-invasively quantify metabolites in living cells. 
</P><P class="caption">Figure 1) Working of the ATP FRET sensor [<A href="#ref6">6</A>] </P><P>
FRET based ATP bio-sensors, dubbed as Ateams, have been previously developed and implemented to quantify <I>in vivo</I> ATP levels in HeLa and yeast cells [<A href="#ref6">6</A>], [<A href="#ref7">7</A>] . The sensor consist of mseCFP (a variant of CFP), the ATP sensing domain is a subunit of the <I>Bacillus subtilis</I> FoF1-ATP synthase and mVenus (a YFP variant). The YFP/CFP emission ratio (527 nm/475 nm) gives an estimation of the ATP concentration. This sensor has been found to be ATP specific and pH independent, although it is temperature sensitive with a higher ATP binding capacity at lower temperatures. The Ateams have been requested from the Imamura lab in Japan for cloning and expression into <I>A. niger</I> and <I>E. coli</I>.
</P><H2><SPAN class="mw-headline" id="ATP_biosensor"> ATP biosensor </SPAN></H2><H2>Aim</H2><P>
Implement a FRET based sensor that will measure ATP levels between compartments in growing <I>A. niger</I> during metabolic shifts. 
</P><H2><SPAN class="mw-headline" id="Applications"> Applications </SPAN></H2><P>
An ATP bio-sensor can lead to visualization of the ATP levels in live Aspergillus cells. Inquiry into whether the single cell like phenotype of <I>A. niger</I> still retains its metabolic activity can be carried out. It would be interesting to quantify the ATP levels in different organelles and investigate where and why such ATP levels exist. Also, the differences in cellular ATP under various growth environments can be studied. The differences in ATP levels for cells at the hyphal tip and those in the inner mycelium can be validated.  
</P><H2><SPAN class="mw-headline" id="Approach"> Approach </SPAN></H2><P>
Clone and express the Ateam bio-sensors in <I>A. niger</I> – 1. ATP sensitive and cytoplasmic localization (APT1); 2. ATP in-sensitive and cytoplasmic localization (APT2); 3. ATP sensitive and mitochondrial localization (APT3). The version 1 of the Ateam sensor was codon optimized for <I>A. niger</I>, illegal restriction sites were removed and a detachable N-terminal mitochondrial signal peptide was included. The mitochondrial signal peptide has been proven to target the mitochondria of <I>A. niger</I> [<A href="#ref2">2</A>]. Detachable here means removable from the gene by restriction digestion to target the sensor to the cytoplasm instead. This resulted in the design of new variants of Ateam - ATP4 (codon optimized for <I>A. niger</I>, ATP sensitive and cytoplasmic localization)and ATP5 (codon optimized for <I>A. niger</I>, ATP sensitive and mitochondrial localization)(Figure 4). These versions were ordered as synthetic construct blocks that were stitched together and incorporated into the pJET vector via Gibson Assembly.</P><P class="caption">Figure 4. Designed ATP4 and ATP5, codon optimized for <I>A. niger</I></P><P>
Observe the cells under a fluorescent microscope and excite the CFP at 435 nm and analyze emission wavelengths at 475 nm (CFP) and 527 nm (YFP). Use sorbic acid in the growth media of <I>A. niger</I> to perturb the ATP pool inside the cell for testing the functionality of the Ateam sensor [<A href="#ref8">8</A>].
</P>

The growth experiment setup was as follows:
<P class="caption">Figure 4) Growth experiment set-up to test functionality of 3 expressed Ateams in <I>A. niger</I>. ATP1, ATP2 &amp; ATP3; N593 - wild type control strain</P><P>
The Procedure for the growth experiment:
</P><P><SPAN class="ref">1.</SPAN> Inoculated 4 strains as indicated in table above in CM with and without supplements at 350 spores/200uL media/well of a 12 well plate and grown overnight.<SPAN class="ref">2.</SPAN> Observe and make pictures under fluorescence microscope. Excitation wavelength of CFP (donor) is 435 nm. Emission at 475 nm (CFP - donor) and 527 nm (YFP - acceptor).<SPAN class="ref">3.</SPAN> Replace media with new CM containing sorbic acid with final concentraions of 0.5 mM and 1.0 mM.<SPAN class="ref">4.</SPAN> After overnight incubation observe and make pictures under fluorescence microscope. Excitation wavelength of CFP (donor) is 435 nm. Emission at 475 nm (CFP - donor) and 527 nm (YFP - acceptor).</P><P>
Express sensor in <I>E. coli</I> and purify with the help of a Histidine tag for <I>in-vitro</I> characterization in a buffer mimicking <I>A. niger</I> physiological environment [<A href="#ref9">9</A>].

</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>
The version 1, 2 and 3 of the Ateams (here on-wards referred to as ATP1, ATP2 and ATP3) donated by the Imamura lab were amplified out by PCR, flanked with additional restriction sites (NdeI and NotI) for cloning into pal85, a shuttle vector with an established <I>A.niger</I> constitutive expression. In the next step, the PCR amplicons were blunt-end ligated into the pJET cloning vector and the DNA sequenced to check for any errors in the gene. A colony PCR and restriction analysis was performed to confirm the cloning into pJET. In Figure 4, lane 1 shows the right bands of size 1800 bp for ATP2 and 3000 bp for linearized pJET; lane4 showing bands including one 1800 bp band for ATP1; lane6 and lane8 show 2000 bp band for ATP3. The DNA sequence of all the constructs were validated by DNA sequencing.       
</P><P class="caption"> Figure 4) Restriction analysis of ATP1, ATP2 and ATP3 in pJET vector</P><P>
Then it was digested out of the pJET vector with restriction enzymes NdeI &amp; NotI and ligated with NdeI &amp; NotI digested pal85. The presence of the constructs in pal85 was verified by colony PCR and restriction analysis. In Figure 5, lane 1 shows the right bands of size 1800 bp for ATP1 and 6000 bp for linearized pal85; lane4 and lane5 showing bands including one 2000 bp band for ATP3; lane7 and lane8 show a band around 1800 bp for ATP2. </P><P class="caption"> Figure 5) Restriction analysis of ATP1, ATP2 and ATP3 in pal85 vector</P><P>
Next, <I>Aspergillus niger</I> protoplasts were transformed with pal85 vector containing each of the 3 versions. Genomic DNA was isolated from pure single colonies of transformants, although the presence of the constructs in the genomic DNA was not established by PCR. The transformed strains were grown in complete medium(CM) without Uridine and checked under the microscope for fluorescence. The excitation filter allowed excitation of CFP (435 nm) and the emission filter allowed light above 515 nm (YFP emission). <I>A. niger</I> strains expressing all the three versions of the sensor were verified under the fluorescence microscope.</P><P class="caption"> Figure 6) ATP1 expression in <I>A.niger</I></P><P>
The functionality of the sensors were tested by the addition of sorbic acid to the medium and lowering the pH of the medium to 4.0. Addition of sorbic acid is known to perturb the ATP pool inside the cell. The growth experiment was setup and pictures were taken under a confocal fluorescence microscope. </P><P class="caption"> Figure 7) Changes in FRET for <I>in vivo</I> ATP3 in <I>A. niger</I> under different sorbic acid concentrations</P><P class="caption"> Figure 8)FRET observed for <I>A. niger</I> containing ATP3</P><P>
The CFP signal reduces due to increase in FRET upon addition of sorbic acid to the medium at pH 4.0. This is seen as a response to changing <I>in vivo</I> ATP levels because sorbic acid is known to perturb ATP pools in <I>A. niger</I>.
</P><P>
The version 1 of the Ateam sensor was codon optimized for <I>A. niger</I>, illegal restriction sites were removed and a detachable N-terminal mitochondrial signal peptide included. This resulted in the complete design of the version 4 and 5 of the Ateam sensor which was ordered as synthetic construct blocks that were stitched together and incorporated into the pJET vector via Gibson Assembly. This was verified by colony PCR (Figure 9).</P><P class="caption"> Figure 9) Colony PCR to confirm Gibbson Assembly of ATP5 </P><P>
The isolated plasmid DNA, from a colony that showed positive results for colony PCR, was sent for DNA sequencing to verify the sequence of ATP5. The sequence was found to match perfectly as per the design of ATP5. The ATP4 construct was amplified out from ATP5 by PCR with primers that had flanking &quot;standard 10&quot; restriction sites. This was then cloned into pSB1C3 and again sequence verified before submission to the parts registry (BBa_K1023003). Cloning ATP4 and ATP5 into pal85 and introducing into <I>A. niger</I> for expression is currently under progress. 
</P><P>
Also, it was aimed to express the ATP sensor in <I>Escherchia coli</I> for <I>in-vitro</I> characterization. The version 1 Ateam was amplified out by PCR with flanking BamHI and NotI restriction sites and ligated into digested pCDFDuet-1 <I>E. coli</I> expression vector. After confirmation by colony PCR and restriction analysis (figure 10) the pCDFDuet-ATP1 vector was introduced into E.coli BL21 strain.</P><P class="caption"> Figure 10. Restriction analysis to confirm ATP1 incorporation into pCDFDuet-1</P><P>
The <I>E. coli</I> culture was induced by 0.01 mM IPTG overnight and expression was observed under a fluorescence microscope. The excitation filter allowed excitation of CFP (435 nm) and the emission filter allowed light above 515 nm (YFP emission).</P><P class="caption">Figure 10. <I>E.coli</I> cells expressing the ATP sensor </P><P>
25 mL of the <I>E. coli</I> culture was pelleted and sonicated to release the protein. The cell extract was purified with a continuous Fast Protein Liquid Chromatography(FPLC) method using with a Nickel column for preferential binding of the His-tagged sensor to the Nickel. The output of this was de-salted and the resulting purified protein was stored in 20% glycerol at -20'C. Further, the <I>in-vitro</I> characterization of the purified sensor was carried out at room temperature in a buffer mimicking the <I>in-vivo</I> conditions in <I>A. niger</I>.</P><P class="caption">Figure 11. Fluorescence emission spectra of the sensor at room temperature </P><P class="caption">Figure 12. Response of the sensor to changing ATP concentrations </P><H2><SPAN class="mw-headline" id="pH_biosensor"> pH biosensor </SPAN></H2><H2> Aim </H2><P>
Implement a pH-sensor that will measure pH between compartments in growing A. niger during different growth conditions, such as oxygen and glucose depletion; and during citric acid production. 
</P><H2> Application </H2><P>
Regulation of intracellular pH is important for the metabolism of a functional cell as it is tightly regulated by the complex interactions between the consumption, production, transportation and buffering of H+ [3][4]. To maintain protein stability, enzyme and ion channel activity, growth and division, it is essential to retain pH within their physiological range of pH 7.5 [1]. By implementing a pH-sensor in <I>Asp. niger</I>, it would be interesting to investigate whether the production of organic acid has any influence of intracellular pH and how we can improve upon it. </P><H2>Approach</H2><P>
The outline of the approach is as follows: 
Codon optimized pHRed sensor (sequence obtained from Mathew Tantama <I>et al.</I> 2011) was cloned and expressed in <I>Asp. niger</I> by transforming it with both pHRed or mitochondrial signal peptide fused at the N-terminal of pHRed, allowing it to target both the cytoplasm or mitochondria, respectively [<A href="#ref2">2</A>][<A href="#ref5">5</A>]. 

In addition, pHRed gene will be cloned in a pET52b+ vector downstream of strep-tag sequence, which will allow for the purification of pHRed proteins. 
</P><P><SPAN class="ref">1.</SPAN>Make 2 contructs for targeting pHRed to cytoplasmic and mitochondria. <SPAN class="ref">2.</SPAN>Purify strepTagII-pHRed from sonificated E. coli. and make a calibration curve. <SPAN class="ref">3.</SPAN>Transform Aspergilllus niger with cytoplasmic pHRed and mitochondrial pHRed <SPAN class="ref">4.</SPAN>Measure in vivo pH during citric acid production by <I>A. niger</I> in time lapse using a fluorescence microscope 
</P><H2>Research Methods</H2><P>To utilize pHRed for in vivo measurements of pH inside a filamentous fungi during citric acid production, two constructs were made. With one targeting the cytoplasm and the other targeting the mitochondria by transforming fungi with a naked pHRed sensor and pHRed sensor fused with a mitochondrial signal peptide, respectively. The mitochondrial signal peptide has been proven to target the mitochondria of <I>A. niger</I> [<A href="#ref2">2</A>] . To measure pH, a calibration curve is established by purifying streptagII fused at N-terminus of  pHRed proteins from sonificated E. coli, and is observed under a fluorescence microscope.</P><H2><SPAN class="mw-headline" id="Results_2"> Results </SPAN></H2><P>After transformation of DH5α with 1µl of mixture (gibson assembly with pJet1.2 blunt vector), 7 colonies were picked and  were cultured in 5 ml LB medium with ampicillin. Afterwards, plasmids were isolated using a miniprep kit (thermo scientific) according to their protocol using a homemade silicon column. Culture #1 till #3 were digested with pstI and notI; culture #4 till #7 were digested with nsiI and notI, see Figure 1. </P><P>
The cloning step from ligating pHRed gene from pJet1.2 vector into a fungal expression vector was unsuccessful. Ligation has failed for about a month, and we are unable to provide any promising results. 
</P><P class="caption">Figure 1) lane 1, 3 and 4 contain pJet1.2 with the appropriate mit-pHRed (803 bp). Lane 5-7 contain the pJet1.2 with the appropriate pHRed (725 bp).  </P><P>
The cloning step from ligating pHRed gene from pJet1.2 vector into a fungal expression vector was unsuccessful. Ligation has failed for about a month, and we are unable to provide any promising results. 
</P></DIV><DIV id="pagewrapper"><H2><SPAN class="mw-headline" id="Conclusions"> Conclusions </SPAN></H2><P>
The Ateam bio-sensors donated by the Imamura lab – 1. ATP sensitive and cytoplasmic localization (APT1), 2. ATP in-sensitive and cytoplasmic localization (APT2) and 3. ATP sensitive and mitochondrial localization (APT3) were successfully cloned into pal85 and expressed in <I>A. niger</I>. The mitochondrial localization of ATP3 is still unsolved as we need to image under better micrscopic resolution. Furthermore, all the sensors showed response <I>in vivo</I> to addition of sorbic acid at pH 4.0 of the growth medium. Changes in FRET efficiency due to sorbic acid addition that is known to perturb the ATP pool inside the cell is proof that the sensors are functional <I>in vivo</I> in <I>A. niger</I>.
</P><P>
The newly designed variants of Ateam - ATP4 (codon optimized for A. niger, ATP sensitive and cytoplasmic localization)and ATP5 (codon optimized for A. niger, ATP sensitive and mitochondrial localization)were ordered as synthetic DNA blocks and stitched together by Gibbson Assembly. The construct is currently cloned in pJET and is awaiting cloning into pal85 for expression in <I>A. niger</I>. The ATP4 construct cloned into pSB1C3 and sequence verified before submission to the parts registry (BBa_K1023003). 
</P><P>
The ATP sensor version 1 was cloned and expressed in <I>E. coli</I>. The sensor was purified due to the N-terminal Histidine tag attached to it and characterized <I>in-vitro</I> at room temperature. The ratio (527 nm/ 475 nm) was found to increase with increase in ATP concentrations and it saturated after 6 mM ATP concentration in the buffer.
</P><P>
The pH sensor developed by Mathew Tantama <I>et. al</I> was codon optimized and a detachable N-terminal mitochondrial signal peptide added. This designed sensor was ordered as synthetic blocks and pieced together via Gibbson Assembly. The codon optimized pH sensor is in pJET and is awaiting cloning into an in-house vector for expression in <I>A. niger</I>.The codon optimized pH sensor without the signal peptide was cloned into pSB1C3 and sequence verified before submission to the parts registry (BBa_K1023004). 

Both the codon optimized ATP and pH sensor constructs were unable to be cloned into <I>A. niger</I> expression vectors as the ligation and transformations were not working for all our constructs a long time. It took us quite some time to get to the source of the problem which was incomplete digestion at the NsiI and NdeI sites. The solution was to separately digest with NsiI(/NdeI) and then NotI rather than double digestion as it resulted in incomplete restriction. The solution was applied to lovastatin domains and colonies were obtained on <I>E. coli</I> transformation plates, which were verified by colony PCR. So this solution was applied to both the ATP and pH sensor constructs but we have not yet confirmed the colonies observed on the <I>E. coli</I> transformation plates.
</P></DIV><DIV id="pagewrapper"><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><P><SPAN id="ref1" class="ref">1.</SPAN> Bagar, T., K. Altenbach, et al. (2009). &quot;Live-Cell imaging and measurement of intracellular pH in filamentous fungi using a genetically encoded ratiometric probe.&quot; Eukaryot Cell 8(5): 703-712. <SPAN id="ref2" class="ref">2.</SPAN> Blumhoff, M. L., M. G. Steiger, et al. (2013). &quot;Targeting enzymes to the right compartment: Metabolic engineering for itaconic acid production by Aspergillus niger.&quot; Metabolic Engineering 19(0): 26-32.<SPAN id="ref3" class="ref">3.</SPAN> Felle, H. H. (1996). &quot;Control of cytoplasmic pH under anoxic conditions and its implication for plasma membrane proton transport in Medicago sativa root hairs.&quot; Journal of Experimental Botany 47(7): 967-973. <SPAN id="ref4" class="ref">4.</SPAN> Madshus, I. H. (1988). &quot;Regulation of intracellular pH in eukaryotic cells.&quot; Biochem J 250(1): 1-8. <SPAN id="ref5" class="ref">5.</SPAN> Tantama, M., Y. P. Hung, et al. (2011). &quot;Imaging intracellular pH in live cells with a genetically encoded red fluorescent protein sensor.&quot; J Am Chem Soc 133(26): 10034-10037.<SPAN id="ref6" class="ref">6.</SPAN> Imamura, H., et al. (2009). &quot;Visualization of ATP levels inside single living cells with fluorescence resonance energy transfer-based genetically encoded indicators.&quot; PNAS 106: 15651-15656. <SPAN id="ref7" class="ref">7.</SPAN> Bermejo, C., et al. (2011). In vivo biochemistry: quantifying ion and metabolite levels in individual cells or cultures of yeast. Biochem. J 438 (1–10). <SPAN id="ref8" class="ref">8.</SPAN> Plumridge, A., et al.(2004). &quot;The weak acid preservative sorbic acid inhibits conidial germination and mycelial growth of Aspergillus niger through intracellular acidification&quot; Appl. Environ. Microbiol. 70(6):3506. <SPAN id="ref9" class="ref">9.</SPAN> Ruijter, G.J.G, <I>et. al</I> (1997). Overexpression of phosphofructokinase and pyruvate kinase in citric acid-producing Aspergillus niger, Biochimica et Biophysica Acta (BBA) - General Subjects, 1334(2–3): 317-326.
</P></DIV><DIV class="tournavleft"><A href="https://2013.igem.org/Team:Wageningen_UR/Secondary_metabolites" class="tourprev"><SPAN class="prev">Previous</SPAN><SPAN class="tourr">Tour Page</SPAN></A></DIV><DIV class="tournavright"><A href="https://2013.igem.org/Team:Wageningen_UR/Host_engineering" class="tournext"><SPAN class="prev">Next</SPAN><SPAN class="tourr">Tour Page</SPAN></A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Wageningen_UR/Biosensors">http://2013.igem.org/Team:Wageningen_UR/Biosensors</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Wageningen_UR/Biosensors" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Wageningen_UR/Biosensors" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Wageningen_UR/Biosensors&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Wageningen_UR/Biosensors&amp;oldid=317421" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>