"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/Experimental protocols
From 2013.igem.org
Home
Project
Parts
Achievements
Safety
Scientific Interactions
Team
Notebook
Why Aspergillus nigem?
Secondary metabolites
Toolbox
Host engineering
Summary
Achievements
Attributions
Safety introduction
General safety
Fungi-related safety
Biosafety Regulation
Safety Improvement Suggestions
Safety of the Application
Masterclass
Science cafe
iGEM Netherlands
Collaborations
Conference
Vitruvian man
Members
Profile
Thanks!
Lablog
Experimental protocols
Protocols
Materials
Standard lab procedures are explained in Appendix A, B, C and D. Novel protocols are mentioned and explained at the specific topic pages, since they cannot be explained without the mention of intermediate results and can be considered results. A detailed version of these protocols is also given here.
Cloning
G-blocks
To avoid all the illegal restriction found inside the different domains of lovb gene, G-blocks for the metabolic pathway of lovastatin  were ordered from IDT, which comes in with 200 ng of either 500 bp or 750 bp double stranded DNA. The G-blocks were designed in a way that allows it to assemble with a linearized pJet1.2 blunt end vector by adding overlap extended sequences with the pJet1.2 vector at both the 5’ and ‘3 end.
Gibson Assembly
Gibson Assembly master mix (NEB #2611) kit was used for all of our gibson assembly, as this master mix was suited for the assembling of more than 3 fragments. The protocols are as followed:
50 ng of insert fragments (G-blocks) + 25 ng of backbone vector (pJet1.2 blunt end), and fill the rest with Milli-Q H2O untill 10 μl. Afterwards, the mixture was added with 10 μl of Gibson Assembly master mix, ending up with a total volume of 20 μl. Subsequently, the mixture was incubated for 1 hour at 50°C and electroporated with 50 μl of DH5-α electro-competent cells (E.coli). After 1 hour of recovery time in SOC medium, the culture was then centrifuged (3000 rpm, 5 minutes) and 800 μl supernatant was discarded. The remaining 200 μl was resuspended and plated on agar plate with their corresponding antibiotic.
PCR
For the amplification of a gene, polymerase chain reaction were performed using different primer pairs. Generally, the PCR mix consisted of ±50 ng templates DNA, 0.5-1.0 μl  forward primer (10mM), 0.5-1.0 μl reverse primer (10mM), 4μl 5X buffer, 1.25μl MgCl2 (25mM), 0.25μl dNTPs (10mM) and filled the rest of the mix up until 19.5 μl with sterilized Milli-Q H2O. Before transferring the PCR mix to the thermocycler (applied biosystems 2720 thermal cycler), add either 0.25 μl of Q5® High-Fidelity DNA polymerase or 0.05 μl DreamTaq DNA polymerase (Thermo Scientific) into the PCR mix.
Q5 PCR
20 µl reaction mix contained 4 µl 5x Q5 reaction buffer, 1 µl dNTPs (10mM), 1 µl Forward primer (10µM), 1 µl Reverse primer (10µM), 200 ng  template DNA, 0.25 µl Q5® High-Fidelity polymerase and filled the rest with Milli-Q H2O.
Thermal cycle Q5-PCR
Colony PCR
20 µl reaction mix contained 2 µl 10x Dream Taq green buffer, 0.5 µl forward primers (10µM), 0.5 µl reverse primers (10µM), 0.25 µl dNTPs (10mM), 1.25 µl MgCL2 (25mM), 0.05 µl Dream Taq polymerase, 50 ng plasmid DNA, and filled the rest with Milli-Q H2O untill 20 µl.
Thermal cycle Colony-PCR
Digestion
Plasmid DNA are generally linearized by digesting with the selected restriction enzymes. Per 10 µl of digestion mixture, consisted of 1 µl of 10x appropiate restriction buffer, 0.5-1 μg DNA template, filled the rest with Milli-Q H2O. lastly, add 0.5 µl of the appropiate restriction enzyme and incubate it for 1-2 hours in a waterbed of 37°C. The mixture was loaded and run on gel for further analysis or purification.
Ligation
The 10 µl mixture consisted of 1 µl 10x ligation buffer, 1 µl T4 ligase, molar ratio of 1:3 of linearized vector and DNA insert, respectively, and the rest was filled with Milli-Q H2O untill 10 µl. Molar ratio was calculated with ligation calculator (Gibthon).  The ligation mix was incubated for 4 hours at room temperature for 1:30 hours. The ligase was inactivated by 10 minutes incubation at 65°C and then the ligation mix was run through a column for removing salt contamination. Finally, 1-2 µl could be used for the transformation of DH5-α competent cells.
Plasmid DNA purification with homemade columns
The making of purification columns were based on Tatiana et al. (2003) paper (Borodina, Lehrach et al. 2003) and the protocols and buffers were obtained from Thermo Scientific Miniprep Kit. Whatmann GF/F microfiber paper were inserted in a 0.5mL eppendorf tube that was punctured by a hot needle, that would be used as a column. After centrifuging the cell culture at 12.000 rpm for 1 minute, supernatant was removed, and the pellet was resuspended  with 150 µl of Resuspension buffer. Afterwards, gently mixed with 150 µl of Lysis buffer and then mixed with Neutralization. The mixtures were centrifuged at 12.000 rpm for 10 minutes, the supernatant are carefully transferred into the home made column, which was incubated for 15 minutes, then centrifuged at 12.000 rpm for 1 minute. Subsequently, the columns were washed once with 400 µl Washing buffer and the columns were then dried by centrifuging at 12.000 rpm for 2 minutes with 300 µl washing buffer. The columns were then transferred on to a clean 1.5 mL eppendorfs where 40 µl of Milli-Q H2O were added and incubated for 15 minutes. Lastly, plasmid DNA was then eluted by centrifuging at 12.000 rpm for 1 minute. Nanodrop was then used to determine to concentration of purified DNA in ng/µl.
DNA purification out of gel with homemade columns
The making of purification columns were based on Tatiana et al. (2003) paper (Borodina, Lehrach et al. 2003) and the protocols and buffers were obtained from Qiagen DNA gel extraction kit. The expected band was visualized by exposing it with UV light, which allowed for precise excision of DNA fragment from gel. After weighing the fragment, add 3x volumes of QC buffer and incubated at 50°C for 10 minutes. After complete dissolving, add 1x volumes of 100% isopropanol , mix and load on the homemade column. Wash the column with 400 µl and 300 µl washing buffer. Afterwards, elute with 30 µl of Milli-Q H2O.
Gel electrophoresis
Agarose gel were made by mixing 1% or 0.8% of agarose with 1x TAE buffer. The mixture was dissolved after heating it up in microwave. Afterwards, 10% (v/v) ethidium bromide was added and mixed gently, and poured them in gel tray for solidification. 10x loading buffer  was added to DNA sample before loading it on gel, and bands were compared GenerulerTM 1kb DNA ladder, see figure below. Generally, DNA  samples were run at constantly 100 volt for 20-30 minutes and were visualized by Syngene equipment.
Generuler 1kb DNA ladder
Transformation
Electroporation (E. coli transformation)
50 µl of aliquot of -80°C DH5-α electro-competent cells (E.coli) was first thawed on ice. Then 1-2 µl of purified plasmid solubilized in either Milli-Q H2O or TE buffer was added to the thawed electro-competent cells (E.coli), in which the mixture was electroporated with a shock of 200 Ω at 2500 volt inside an electroporation cuvette. After electroporation, 1mL of SOC medium without antibiotic was immediately added for recovery and was incubated for 1 hour at 37°C. Afterwards, 200 µl of the mixture was plated on an appropriate antibiotic LB agar plate, while the remaining 800 µl was concentrated by centrifuging (3000 rpm, 5 minutes) and 600 μl of supernatant was discarded. The remaining 200 μl was resuspended and plated on new LB agar plate. Colonies were formed after the plates were incubated overnight at 37°C and can be handpicked by a toothpick, and inoculate it in a 10 mL blue capped tube filled with 3mL LB medium with the appropriate antibiotic. After another overnight incubation at 37°C, isolation of DNA can be performed.
Heatshock (E. coli transformation)
50 µl of aliquot of -80°C DH5-α chemical-competent cells (E.coli) was first thawed on ice. 5 µl of purified plasmid solubilized in either Milli-Q H2O or TE buffer was added to the thawed chemical-competent cells (E.coli). The mixture was incubated in ice for 20 minutes and was subsequently incubated at exactly 42°C for 45 seconds. Afterwards, 500 µl of SOC medium was added and incubated at 37°C for 1:30 hours. After incubation, the culture was then centrifuged (3000 rpm, 5 minutes) and 800 μl supernatant was discarded. The remaining 200 μl was resuspended and plated on agar plate with their corresponding antibiotic.
Transformation of Aspergillus niger
Solutions TC (500 ml):     50 mM CaCl2 (1,86g)+10 mM Tris/HCl (0,61g), adjust it to pH=7.5
Autoclave
STC:
1,33 M sorbitol in TC
 Autoclave
PEG Buffer (25 % PEG-6000):
Weight 2,5 g PEG-6000 , add 7,5 ml TC and dissolve by heating to approx. 60oC in a microwave oven (when prepared fresh), or filter sterilize (0,2 um membrane)
store filter sterilized at 4C up to 1 week
1. Pipette 3 µg of selection gene plasmid DNA + 20 µg of co-transforming plasmid DNA that contained a selection marker(dissolved in a 10 - 20 µl TE) in a micro centrifuge tube.
- Add 200 µl of the protoplasts suspension in the micro-centrifuge tubes
- Add 50 µl of PEG buffer, mix gently by shaking and incubate at RT for 20 minutes. Include a positive and negative control, use 1 µg of selection gene DNA as a positive control and 20 µl of TE as a negative control.
2. Add 2 ml of PEG buffer, mix gently, incubate at RT for another 5 minutes
- Subsequently add 4 ml of STC, and mix gently (no vortex
3. Add transformation mixture into selective MMS-top agar so that the tube (50 ml) is almost completely filled. Cap the tube and mix by inverting the tube several times and pour the mixture onto selective MMS-bottom agar plate.
Competent Cells and Protoplast
Electro competent cells
Equipment
-80°C freezer
37°C incubator
Refrigerated centrifuge that accepts 225 mL culture tubes
Chemicals and reagents
~500 mL liquid LB medium without Sodium
~600 mL sterile deionized water chilled to 4°C
50 mL sterile 10% glycerol in deionized water chilled to 4°C
Ice bucket and ice
Dry ice (liquid nitrogen), ethanol bath or liquid nitrogen
Supplies
Many 1.5 mL plastic tubes chilled to -80 °C
14 mL culture tube for starter culture
2 L flask for culture
225 mL plastic tubes for centrifugation
Pipets
Procedure
1. Prechill all tubes and pipets at 4°C or -80°C as appropriate.
Also rinse all flasks with H2O prior to autoclaving in order to remove residual detergents that may remain on glassware from dishwashing. This step may increase competency. Autoclaving with water, which is then discarded, is even better.
2. Inoculate 5mL LB medium without salt and grow overnight at 37°C with rotation.
Use LB without sodiumchloride (10 g/l peptone or tryptone and 5 g/l yeast extract)
3. Add the 5mL overnight culture to 450mL LB medium and incubate at 37°C with vigorous shaking until the OD 600nm is between 0.5 and 1.0. It should take about 3 hours.
4. Fast cool the centrifuge with the correct rotor to 4°C
5. Pour the culture into two 225 mL centrifuge tubes.
6. Place the tubes on ice for 15 minutes.
This step can vary in incubation time between 15 minutes and 1 hr. Longer incubation times may lead to higher competency.
For the following steps it is important to keep cells cold and remove all the supernatant in each step to remove residual ions.
7. Centrifuge for 10 mins at 2000g at 4°C
8. Remove supernatant and gently resuspend pellets with 200mL cold sterile water.
Initially add 10-20 mL of water and resuspend by pipetting. Then add the rest of the water.
9. Centrifuge for 15 mins at 2000g at 4°C
10. Remove supernatant and gently resuspend pellets with 200mL cold sterile water.
Initially add 10-20 mL of water and resuspend by pipetting. Then add the rest of the water.
11. Hold on ice for 30 minutes
12. Centrifuge for 15 mins at 2000g at 4°C
13. Remove supernatant and gently resuspend pellets with 25mL cold 10% glycerol.
This can be optionally transferred to a 50 mL conical tube.
14. Hold on ice for 30 minutes
15. Centrifuge for 15 mins at 1500g at 4°C
16. Remove the supernatant and add 500 μl of 10% glycerol
17. Resuspend the cells in a final volume of approximately 1 ml
18. Aliquot 50 μL per tube (tubes on ice)
19. Shock freeze cell suspensions in a dry ice and ethanol bath.
One website recommended against using liquid nitrogen but did not justify this recommendation.
20. Store at -80°C
Chemically competent cells
Materials
K-Mes transformation buffer (100 ml):
6 ml 1 M CaCl2
0.5 ml 1 M MgCl2
2 ml 1 M K-Mes (pH 6 adjusted with KOH)
0.5 ml 1 M MnCl2
Storage buffer:
K-Mes transformation buffer + 15 % glycerol
Protocol:
1. Inoculate DH5 on a LB plate and incubate overnight at 37C
2. Innoculate100 ml LB, 10 mM MgSO4 (in 500 ml erlenmeyer flask) with 10 colonies from the plate.
3. Incubate until OD600 ~ 0.5
4. Cool the culture on ice and centrifuge 8 min at 3000 rpm using pre-cooled buckets and SLA-1500 rotor.
5. Resuspent the cells in 40 ml cold K-Mes transformation buffer and put on ice for 30 min
6. Centrifuge 8 min at 3000 rpm
7. Resuspent the cells in 10 ml K-Mes transformation buffer, 15% glycerol
8. Aliquot 200 ul portions of the competent cells in 1.5 ml eppendorf tubes and freeze immediately in liquid  nitrogen
9. Store at -70oC.
Protoplast (Aspergillus niger)
Materials
Sterile Büchner funnel with nylon gauze, sterile 10 ml glass tubes), Novozyme 234, sterile funnel with glass wool plug, shake incubator at 30 °C, bench centrifuge, sterile and dry universals (flat bottom 30 ml screw cap tubes), selector and co-transforming gene, sufficient MMS-plates (15 and 9 cm), and molten MMS top-agar at 48 °C.
Saline Tween (ST):
0.8 %  NaCl
0.005 % Tween-80 (1:100 dilution from 0,5 % (v/v) Tween-80 stock)
SMC:
1,33 M Sorbitol (242,3 g/L)
50 mM CaCl2 (7,35 g/L CaCl2 . 2 H2O)
20 mM MES buffer  pH 5,8
filter sterilize store @ 4C
TC (500 ml):
50 mM CaCl2   1,86g
10 mM Tris/HCl  pH 7,5   0,61g
Autoclave
STC:
1,33 M sorbitol in TC
Autoclave
PEG Buffer (25 % PEG-6000):
Weight 2,5 g PEG-6000 , add 7,5 ml TC and dissolve by heating to approx. 60oC in a microwave oven (when prepared fresh), or filter sterilize (0,2 um membrane)
store filter sterilized at 4C up to 1 week
TM: Transformation medium for Aspergillus (TM)  (1000 ml)
10x Minimal Medium salts (protocol 30)   100 ml
Vishniac solution (protocol 30)    1 ml
0.5% Yeast Extract     5 g
0.2 % Casamino acids (vitamins free)   2 g
Adjust pH to 6.0
Autoclave
add 2 % glucose after sterilization and appropriate supplements 20 g
Stabilized Minimal Medium (St. MM)(1000 ml)
10x Minimal Medium salts  100 ml
Vishniac solution   1 ml
Sucrose (0.95 M)    325.2 g
Adjust pH to 6.0
Add 1.2 % agar,    12 g
in case of top agar use 0.6 % agar.  6 g
 Autoclave
Used as selective/non-selective medium in transformations, depending on the strain used and the supplements added.
1. Inoculate 250 ml of transformation medium with 1*106 spores per ml and grow for 16 -18 hours at 30 °C and 250 rpm a New Brunswick shaker.
2. Harvest mycelium on nylon gauze using a Büchner funnel and mild suction, and wash once with SMC.
3. Dissolve 100 mg Novozyme 234 in 10 ml SMC and filter sterilize (0,2 µm membrane)
4. Resuspend the 1 g (wet weight) mycelium in the Novozyme solution by gently pipetting up and down a 10 ml Pipette.
You can fill multiple tubes as there is much more biomass available.
5. Incubate with gently shaking at 30°C (slow hand-shaking-like, 80-125 rpm). After 1 h the mycelium is carefully resuspended and a sample is taken to monitor protoplast formation (magnification 400X).
In case there are not enough protoplasts and the mycelium tips are not degraded incubate further and check every 30 min.
6. When sufficient protoplasts are present (more then 1*108) carefully resuspend the protoplasts and remove the mycelial debris by filtration over a sterile glass wool plug. Collect the filtrate in a 10 ml glass tube.
7. Pellet the protoplasts by 10 minute centrifugation at 2000 rpm (670 x g) for 10 min at 4°C in a bench centrifuge, carefully resuspend in 5 ml STC, pellet as above. Repeat this wash step twice and determine the protoplasts concentration before the last centrifugation step, using a heamocytometer (protocol 31a). Resuspend the protoplasts in STC at a density of 1*108 per ml
8. At this point, protoplast can be aliquoted and frozen in -80 °C in the STC-buffer.
Strains
A. niger N593
This strain is cspA and pyrA deficient. Media for N593 are supplemented with 5 mM uridine. Unlike citric acid producers, this strain produces mainly oxalic acid.
A. niger strains expressing GFP
Three strains have been applied that express Green Fluorescent Protein (GFP) in various constructs. GFP is widely used as a localisation tool. These strains have been constructed using strain AB4.1 (Vinck, A., M. Terlou, et al. (2005). "Hyphal differentiation in the exploring mycelium of Aspergillus niger." Molecular Microbiology 58(3): 693-699). The GFP constructs are made using the sGFP gene. This mutated GFP variant has an excitation peak at 488 nm and an emission peak at 510 nm. Information is given in table 1.
Table 1) Different Aspergillus niger strains expressing GFP
A. nidulans WG505
This is a pyr-deficient strain, so 5 mM uridine was added to the medium. Use of this strain is mentioned by Nyyssola (Nyyssola, A., R. Heshof, et al. (2012). "Methods for identifying lipoxygenase producing microorganisms on agar plates." AMB Express 2(1): 17). No further data available.
Media
The standard protocol for media for the SSB group is provided as Appendix D. Media and supplements used here are:
Uridine solution
Per 100 ml; 12.2 g uridine. Filter-sterilised using a 0.2 µm filter.
Vishniac solution
Per litre; 10 g EDTA; 4.4 g ZnSO4_7H2O; 1.0 g MnCl2_4H2O; 0.32 g CoCl2_6H2O; 0.32 g Cu¬SO4_5H2O; 0.22 g (NH4)6Mo7O24_¬4H2O; 1.47 g CaCl2_2H2O; 1.0 g FeSO4_7H2O. pH adjusted to 4. Filter-sterilised using a 0.2 µm filter.
Vitamins solution
Per 100 ml; 10 mg thiamine; 100 mg ribovlavine-5P; 10 mg p-aminobezoic acid; 100 mg nicotinamide; 50 mg pyridoxine-HCl; 10 mg panthothenic acid; 2 mg biotine. Filter-sterilised using a 0.2 µm filter.
Minimal medium salts (MMS)
For preparation of medium, 10x MMS stock solution is used. This contains per 1000 ml; 60 g NaNO3; 15 g KH2PO4; 5 g KCl; 5 g MgSO4_7 H2O and is adjusted to pH 5.45.
Minimal medium (MM)
Stock 10x MMS solution is diluted 10x, autoclaved for 20 minutes at 121°C and supplemented with 50 mM carbon source, appropriate supplements (e.g.  5 mM uridine) and 1 ml/l Vishniac trace element solution.
Complete medium (CM)
Stock 10x MMS solution is diluted 10x, supplemented per litre with 2 g meat peptone; 1 g yeast extract; 1 g Casamino acids; 0.3 g yeast ribonucleic acids and then autoclaved for 20 minutes at 121°C. After autoclaving it is supplemented with 50 mM carbon source, appropriate supplements (e.g.  5 mM uridine), 1 ml Vishniac solution and 2 ml vitamins solution.
Cultures
Antibiotic concentrations
An appropriate concentration for each antibiotic need to be added to either liquid LB medium or agar LB plate, see table below.
Antibiotic Concentrations
Microscopes
Appendices
A. Preparation spores suspension/inoculation cultures and making spore plates
B. Preparation and storage of Aspergillus glycerol stocks
C. Media for Aspergilli
D. HPLC, sample preparation
E. DAPI staining of A. niger
F. Calcofluor staining of A. niger
G. Experimental evolution protocol “Mycelium to single cell”
H. RNA extraction protocol
A. Preparation spores suspension/inoculation cultures and making spore plates
Materials:
Sterile Pipettes and Pipetting tips
Sterile saline-Tween solution (0.9 % NaCl + 0.005 % (v/v) Tween-80
Sterile saline solution (0.9 % NaCl)
Method for plates:
1. Inoculate on Thursday six 15 cm CM plates (protocol 30) with a total number of 4x105 spores (this is ~22 spores per mm2 on a 15 cm plate; for 9 cm plates, add total 1.4x105 spores). Make sure to pour the CM-agar while it is relatively cold, as this will enhance the spore scraping procedure. Do not add too many spores: with >50 spores per mm2 the plates will wrinkle and spore concentration will decline since it is more difficult to harvest the spores!
2. Place the plates for 4 days at 30 °C (until Monday).
3. Put the spore plates overnight in the cold room @ 4 °C for maturation (Monday overnight).
4. Pipette 10 ml saline-Tween solution on the spore-mat.
5. Scrape off the spores using a Drigalski spatula.
6. Transfer the spore suspension to a sterile 50 ml Greiner tube with a sterile 10 ml pipet.
7. Mix 30 seconds on Vortex and filter over funnel with glass wool. Centrifuge 10 minutes, 5000 rpm at room temperature.
8.  Pour off supernatant and resuspend in 40 ml saline solution, mix 30 seconds and repeat the wash once more.
9. Prepare a 50-fold dilution (20 µl spore suspension +  980 µl saline) and determine the concentration of spores by counting the diluted sample in a heamocytometer (see remarks).
10. Calculate the concentration of spores.
Neubauer improved heamocytometer:
depth: 0.1 mm
Minimal area (A): 1/400 mm2
Pipette spore dilution between coverslip and slide enough liquid until the space is flooded (~20 ul). NB If the coverslip is chipped or in any way damaged, it must be replaced- it is no longer functional.
Count the spores (magnification 400 X) in 16 fields.
Calculate the spore concentration in the spores suspension:
spores/ml = count x d x 2.5 x 105 (d = dilution)
B. Preparation and storage of Aspergillus glycerol stocks
1. Harvest spores from a plate.
2. Transfer the spore solution in ST into a sterile glass tube with aluminium cap
3. Spin down the spores at low speed (3000 RPM in table top centrifuge in rec lab)
4. Resuspend the spores in 1 ml sterile glycerol medium (1 part glycerol 87% : 2 parts 1% pepton).
5. Divide over two Eppendorff tubes and label well
6. Store immediately in the -70⁰C freezer and update the collection database
7. Sterilize all tubes used and fungal material left
Taking a strain from glycerol stock:
1. To take a strain from the stock use a sterile platinum needle.
2. Take an inoculum from the stock and transfer to a plate
3. Distribute the spores with a sterile Drigalski spatula
4. Return the stock to the freezer as quickly as possible
C. Media for Aspergilli
Minimal medium salts(MMS) for 1000 ml:
6.0 g NaNO3
1.5 g KH2PO4
0.5 g KCl
0.5 g MgSO4. 7 H2O
when 10xMMS is adjusted to pH=5.45, a 1x MM solution will have pH ~6.0
Minimal Medium for 1000 ml:
100 ml Minimal medium salts 10x
pH adjusted to 6.0
For solid medium 15 g agar is added
autoclave
Before use:
50 mM carbon source and appropriate supplements, 1 ml Vishniac trace-elements solution (lab fridge)
Complete Medium for 1000 ml:
100 ml Minimal medium salts 10x +
2 g meat peptone (pepton 100)
1 g yeast extract
1 g pepton 140 or Casamino acids (vitamins free)
0.3 g yeast ribonucleic acids (cold room shelf)
2 ml vitamins solution (fridge)
pH adjusted to 6.0
For solid medium 15 g agar is added
autoclave
Before use:
50 mM carbon source and appropriate supplements (added after sterilization) 1 ml Vishniac trace elements solution
Vishniac solution (Vishniac and Santer, 1957): 1 litre
10 g EDTA
4.4 g ZnSO4.7H2O
1.0 g MnCl2.4H2O
0.32 g CoCl2.6H2O
0.32 g CuSO4.5H2O
0.22 g (NH4)6Mo7O24.4H2O
1.47 g CaCl2.2H2O
1.0 g FeSO4.7H2O
pH adjusted to 4.0 and stored at 4°C.
Vitamins solution: 100 ml, store in protected from light
10 mg thiamine
100 mg ribovlavine-5P
10 mg p-aminobezoic acid
100 mg nicotinamide
50 mg pyridoxine-HCl
10 mg panthothenic acid
2 mg biotine
D. HPLC, sample preparation
Materials:
Samples and if necessary defrost them on time
HPLC vials and caps
micro centrifuge vials
Demi water
HPLC standards
Media
For Organic acids:
Crotonate Internal Standard:  30 mM crotonate in 0,3 N H2SO4
per liter: 2,58 g crotonate + 8,4 µl (0,96%) H2SO4
External Standards organic acids: (example) 100mM stock concentration
#1; oxalic acid, a-ketoglutaric acid, sorbitol, fumaric acid
#2; cis-aconitic acid, citric acid, xylose, , itaconic acid
#2; oxaloacetic acid, glucose, malic acid, succinic acid
#4; pyruvic acid, formic acid
External Standards sugar: (example) 100mM stock concentration
#1; sorbitol, Xylose (for Carbopac-MA1)
#2; sorbitol, arabinose, galactose, glucose, xylose, mannose (for Carbopac-PA20 max 100 µM)
Protocol
1. Take your samples, if necessary defrost them, centrifuge for 15 min max. speed
2. Label your HPLC vials or 96 well plate (ICS 5000)
3. Make a H2O HPLC vial
4. Prepare the external standards in the following concentrations in a micro centrifuge
Table 2)
5a.
For the HPLC vials take 800 µl standard dilution, add that to the HPLC vial + 200 µl Crotonate I.S.
Do this also with the samples.
Take sample/controls + Crotonate  I.S. in a 4:1 ratio,
The minimal total volume  in the HPLC vial should be 500 µl
5b.
For 96 well plate take 200 µl standard dilution, add that to the 96 well plate + 50 µl Crotonate I.S.
Do this also with the samples. When done cover the plate with a sheet
Take sample/controls + Crotonate  I.S. in a 4:1 ratio,
The minimal total volume  in the 96 well plate should be 250 µl
6. For HPLC-vials,  Cap the vials
7. Mix the HPLC vials well, not necessary for a 96 well plate
The samples are now ready for measurement on HPLC to measure organic acids.
E. DAPI staining of A. niger
Made using A. niger strain N400
Materials:
• Gloves
• 4',6-diamidino-2-phenylindole (DAPI) (Invitrogen D1036)
• pH meter
• Fluorescence microscope
Protocol
1. Use gloves! DAPI is carcinogenic. Prepare a 10x Hoechst stock solution (1,0 g/ml) in demi water and store protected from light at 4°C.
2. Sample a minimum of 200 µl of mycelium in an eppendorf.
3. Correct the pH to 7 (between pH 4  and 11 fluoresence remains unchanged)
4. Cultured cells are examined immediately after incubation with DAPI at 0.1g/mL in phosphate buffered saline at 30 °C for 15-30 minutes.
5. Use the fluorescence microscope with block A. Consider using Vectashield mounting medium if bleaching of the stain occurs too quickly.
F. Calcofluor staining
Calcofluor white (fluorescent brightener 28, Sigma; 100 μl) (0•3 g l−1) was added to 1 ml diluted sample in a 1•5 ml Eppendorf vial covered with tin foil. Samples were mixed and incubated at room temperature for 5 min. A few drops of the solution were placed on a glass slide and covered with a coverslip for analysis. The dye fluoresces when bound to chitin and glucans, and thus stains cell walls and septa. Images were obtained using fluorescence microscopy on a Leika microscope using a mercury burner and a filter block with 330–380 nm excitation wavelength and dichroic mirror with emission >420 nm.
G.   Experimental evolution protocol “Mycelium to single cell”
Daily transfers
Aim
Obtain genotype for single cell morphology from mycelium
Method:
1. Inoculate 10 ml CM to 1x106 spores/ml
2. Grow O/N @ 30⁰C
3. Disturb culture by 30s of vortexing
4. Filter culture and determine mass of filtrate (pellet) and retentate (dry)
5. Resuspend filtrate (in 1 ml) and inoculate fresh medium
6. Repeat step 2-6
NB: Take a more stringent method/smaller pore size when retentate ≤ filtrate
Requirements
• Filters with a range of pore sizes
• 30⁰C climate shaker
• Glass tubes + medium
When no transfers can be done during weekends, spore plates are inoculated on Friday and harvested on Monday according to protocol 31 to inoculate new liquid cultures on Monday for continuation of the experiment.
H.   RNA extraction protocol
• DEPC-treated H2O: Use DEPC (Diethylpyrocarbonate) to make the H2O RNase free. Add DEPC to water (final concentration 0.1%) and autoclave at least once. This DEPC-treated H2O can be used to prepare 75% EtOH: add 75 mL EtOH to 25 mL autoclaved DEPC H2O.
• Organic liquids (chloroform, phenol, isopropanol) do not have to be treated with DEPC,
• RNase free filter tips and Eppendorf tubes can be ordered. To make other material RNase free, soak it in 0.1% DEPC (in demi-H2O) in a beaker glass, cover it with foil, and leave O/N. Autoclave the next day and discard the liquid. After drying, the material can be used.
• Wear clean gloves all the time (human and water, also ice, are the main source of RNases)
Materials
-80°C freezer
Thermoblock
Beadbeater (MP FastPrep-24)
Liquid N2
Glass beads
RNase free filter tips
Screw cap vials 2.0 mL (Sigma Aldrich, Lot. 2215479)
1.5 mL microcentrifuge tubes RNase Free (Ambion, Lot. 1004034)
RNA extraction solution (peqGOLD TriFast, PeqLab, Lot 131012)
Chloroform
Isopropanol
DEPC-treated H2O
75% EtOH (RNase free)
Procedure:
1. Fill screw cap vial with glass beads
2. Add 1 mL peqGOLD TriFast. Flush tip to avoid dripping. Filter tip is not necessary, but you can use the tip for all samples.
Work in the fumehood, peqGOLD contains phenol. The peqGOLD should be protected from light as much as possible. If the extraction did not work, this might be because the peqGOLD was not stable anymore, replace it.
3. Add small piece of mycelium (not too much!) directly into the peqGOLD. Use tweezers which are pre-cooled in liquid N2.
Because of phenol, vials have to be tightly close, also take care of the rubber ring.
4. Place all vials in the same position in the Beadbeater and distribute them evenly over the rotor. Tighten well! Use Setting 6, 30 sec. (Press RUN 2x)
5. Keep the samples at RT, 5 min
Usually transport and unscrewing is long enough
6. Add 200 μL chloroform to each sample. Flush tip to avoid dripping. Filter tip is not necessary, but you can use the tip for all samples.
7. Vortex each sample exactly 15 sec (use a timer!)
8. Leave samples 3-10 min at RT
9. Centrifuge 5 min (12000×g, RT)
10. Transfer the aqueous (upper) phase to 1.5 mL tube using filter tips (app. 4-5 time 100 μL). Be careful not to contaminate the sample with DNA or proteins (in organic and interphase). If this accidently happens: Centrifuge again! Put samples on ice.
Organic and interphase should be discarded in phenol-wast. Also see manual, this can also be used for protein extraction.
11. Take samples from ice to add 0.5 mL isopropanol. Flush tip to avoid dripping. Filter tip is not necessary, but you can use the tip for all samples. Place samples on the ice again.
12. Leave samples 5-15 min on ice.
13. Centrifuge 10 min (12000×g, 4oC).
14. Remove the supernatant.
15. Wash the pellet with 0.7-1.0 mL 75% EtOH.
16. Vortex briefly.
17. Centrifuge 10 min (12000×g, 4oC).
18. Remove the supernatant
The easiest is to remove the supernatant in 2 rounds, once roughly with a big tip and afterwards with a small tip more carefully.
19. Air dry the samples.
20. Place samples 2-5 min in thermo-block at 65oC to remove the EtOH (leave caps open).
21. Dissolve the pellet in 50 μL DEPC-treated H2O by shaking the samples for app. 10 min at 65oC and vortexing once in a while.
22. Place samples back on ice.
23. When completely cooled down, vortex the samples, and spin them down.
24. Measure the RNA concentration.
25. Store the RNA at -80oC (Put samples directly in -80oC).
Retrieved from " http://2013.igem.org/Team:Wageningen_UR/Experimental_protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
