"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/Lovastatin
From 2013.igem.org
Home
Project
Parts
Achievements
Safety
Scientific Interactions
Team
Notebook
Why Aspergillus nigem?
Secondary metabolites
Toolbox
Host engineering
Summary
Achievements
Attributions
Safety introduction
General safety
Fungi-related safety
Biosafety Regulation
Safety Improvement Suggestions
Safety of the Application
Masterclass
Science cafe
iGEM Netherlands
Collaborations
Conference
Vitruvian man
Members
Profile
Thanks!
Lablog
Experimental protocols
Lovastatin
All you need is lov.
Introduction
Secondary metabolites
Toolbox
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Why Aspergillus nigem?
Secondary metabolites
Lovastatin
Modeling
Biosensors
Infrastructure
Chromoproteins
Host engineering
Summary
Outline
The discovery of secondary metabolites has had a profound influence on the development of human medicines. It was Alexander Fleming who in 1928 discovered the first antibiotic, penicillin, that would conquer some of mankind's most ancient scourges, including syphilis, gangrene and tuberculosis. The production of these secondary metabolites in most cases involves a large backbone enzyme containing multiple catalytic domains and whose genetics is rather complex. Modern cloning technology gives us the possibility to establish a modular system of domain shuffling to generate a plethora of novel enzymes with new and improved functionalities. The possibilities are endless as there are various different domains that can be added, removed, reordered, exchanged or even customized in this synthetic biology approach.
Introduction
As an inhibitor of the enzyme 3-hydroxy-3-methylglutaryl CoA reductase, lovastatin plays a significant role in cholesterol biosynthesis, and is therefore a medicinal compound used against cardiovascular diseases [ 1 ]. It is a naturally occurring drug which is found in foods such as oyster mushrooms and red yeast rice. In industrialized production of lovastatin, the natural producer Aspergillus terreus is used. Howere, toxins also occur during the production. Thus, a host with the potential to produce lovastatin more securely is needed. The perfect performance of Aspergillus niger in industrial production of organic acid producer makes it an inviting host for our project. Moreover, it seems to be a wise choice as there is a lot of industry experience about A. niger which has high similarity to A. terreus. In this project, the potential of A. niger to biosynthesis lovastatin are assessed by cloning and transferring the genes from A. terreus to the same species funges A. niger. Also the lovastatin resistance of A. niger is in consideration.
For the biosynthesis of lovastatin, A. terreus uses a range of enzymes, such as polyketide synthases (PKS), enoyl reductase, esterase and a cytochrome P450 oxygenase[ 2 ]. Among these, LovB (lovastatin nonaketide synthase), LovG (thioesterase) [ 3 ]. and LovC (enoyl reductase) together take charge of most of the production pathway by releasing the intermediate-dihydromonacolin L acid from nine malonyl-CoA units [ 3 ] after 35 reactions.
LovB contains β-ketosynthase (KS), acyl transferase (MAT), dehydratase (DH), methyl transferase (MT),  ketoreductase (KR), acyl carrier protein (ACP), nonribosomal-peptide synthase condensation (CON) domains and an inactive enoyl reductase (ER) domain, which is active in LovC [ 2 ][ 4 ]. The amino acid sequences of LovB and LovC were obtained of the predicted proteome of A. terreus and the functionality of some of these domains has previously been demonstrated by experiments [ 5 ].
Aim
We set out to prove that A. niger is an suitable candidate host for lovastatin production. By cloning the enzymes LovB (3038 Amino Acids (AA)), LovG (256 AA) and LovC (363 AA), which play significant roles in the lovastatin biosynthesis pathway, they can be expressed in the new genomic context. In addition, we decided to split  LovB into single domains during cloning, checking the possibilty to reassemble and fuse those domains as a new biosynthetic engineering strategy.
lov assembly method
For our project, we have developed a dedicated assembly method to allow us to fuse proteins in a modular approach by bio-bricking and standardizing them, NsiI-AgeI-gene-BspeI-TAATAG-NotI. The advantages of this design are the ability to fuse proteins without any frameshift or unwanted stop codons; the scars formed by BspeI and AgeI contain glycine and serine to minimize any steric hindrance; ligation of inserts are always in the correct orientation; allow the modularity of genes with or without a stop codon due two the flanking of two restriction sites (BspeI and NotI and because BspeI needs a overhang of 6 basepairs, double stop codons were introduced; and most importantly,  lov assembly method will enable us to work in parallel, e.g. a group of 3 persons, with each person working on 2 fusion proteins, and can be combined together in the end. Saving time is essential, especially within the context of iGEM, as you can only do 4 months of experiments. When more than one domains were planned to be inserted, the first one should be digested by NsiI and BspEI (from B to C) and the last one should be digested by AgeI and NotI (from A to E). The restriction enzymes AgeI and BspEI were used in pair when fusing domains together. A scar area of was formed, which cannot be digested by AgeI or BspEI anymore. Figure below illustrates on how our lov assembly method works.
Figure 1A. Module of gblocks. Shadow area of sequence A was restriction site for AgeI while B for NsiI, C for BspEI and E for NotI. Sequence D consists of two stop codons which ensure the inserted domain be expressed precisely.
Figure 1B. lov assembly method. A: single ligation of one gene B: double ligation of two genes.
Figure 1C. Illustrating that the assembly can ligate a third insert OR the ability to work in parallel by ligating a whole section of fused proteins
Domain separation
The minimal polyketide synthase domains of LovB as standalone proteins have never been clearly defined nor have their activities and substrate specicifities been systematically assayed. With the help of literature, Domcut and BLAST, we are able to define the boundaries of for each domain, while maintaining their individual functionality. By splitting enzymes into separate domains, it would be possible to rearrange domains from different sources and to design purposive multidomain enzymes producing novel products.
Research Methods
gBlocks and codon optimized
The nucleic acid chains for most domains are too big to be cloned using gBlocks, which are double-stranded, sequence-verified genomic blocks up to 750bp in length. The sizes are 448, 492, 321, 375, 815, 91 and 496 AA for KS, MAT, DH, MT, KR, ACP, CON domains respectively. Thus each domain should still be split into parts, making corresponding DNA fragments in a form of 500bp or 750bp which were produced by IDT (Integrated DNA Technologies). The design of those fragments also included restriction site attached to both ends of the domain and an overlap of 42bp for fragments belong to one domain.
DNA assembly
Gibson assembly allows for successful assembly of multiple DNA fragments, so we use it to fuse two or three DNA fragments together for a complete domain. A combination of 25 ng of backbone vector (pJet1.2 blunt end) with 50 ng of insert fragments (G-blocks) and adequate amount of MiliQ water (depends on the number of gBlocks inserted) to make 10µL in total, was incubated with 10µL 2X Gibson Assembly Master Mix in a final volume of 20µL. The samples were incubated at 50 °C temperature for one hour before transformed into competent cells.
Colony PCR
After ligation, competent cells were transformed and subsequently transferred to LB agar plates containing ampicillin as a selection  marker in a concentration of 0.1%. Successful transformants were identified using colony pcr. 8 single colonies for each assembled domain were chosen for further PCR analysis. 1% agarose gel was used to separate fragments of different size apart.
Preparation of plasmid
The reagents and protocol from a  Miniprep Kit of Thermo was used to isolate plasmid. After, the plasmid concentration was measured by Nano drop.
Digestion
Restriction enzymes from NEB (New England Biolabs) were used for digesting fragments. The digested fragments would be inserted into a vector for gene expression in fungi. According to the designed structure of gBlocks which was mentioned earlier, NotI and NsiI were used for inserting single domain into vector, while AgeI and BspEI were used to form scar area while more than one domain would be fused together. The protocol was obtained from double digestion manual of NEB.
Purification of DNA fragment
After digestion, a 1% agarose gel was used for DNA extraction. After the DNA fragments with different sizes were separated on the gel, the band with the expected size was cut out. The reagents and protocol from Gel extraction Kit of Thermo were used to purify the fragment, then the concentration of fragment was measured by Nano drop.
Results
Colony PCR confirmed the succesful assembly of Gblocks of each domain, shown in Figure 3. Correct assembly fragments could be obtained.
Figure 3. Colony PCR results for the domains.
The ligation of KS domain in vector is confirm by observing the band in the expected size, shown in Figure 4.
Figure 4. Successful ligation of KS domain.
To confirm the expected potential to be a lovastatin producer, the lovastatin resistance of A. niger was tested. The fungus was incubated on plates with different concentrations of lovastatin (Figure 5).
Figure 5. Lovastatin plate with A. niger on it.
Discussion
As ambitious as the Lovastatin project was, actually getting Lovastatin was always just a dream. Still it was a good opportunity to learn. An experience where we encountered problems but also successes. We learned to design our own domains, how to codon optimize them, remove illegal restriction sites and add our own restriction sites. Then once we had the G-blocks we were able to assemble the domains using Gibson assembly a technique never used on our lab. When we did have our domains however the ligation of the domains in our in-house vector needed some troubleshooting before it worked. So even though we did not get to produce Lovastatin it was a great learning experience.
References
1.Tobert, J. A. (2003). Lovastatin and beyond: The history of the HMG-CoA reductase inhibitors. Nature Reviews Drug Discovery, 2(7), 517-526.
2.Campbell, C. D., & Vederas, J. C. (2010). Biosynthesis of Lovastatin and Related Metabolites Formed by Fungal Iterative PKS Enzymes. Biopolymers, 93(9), 755-763.
3.Xu, W., Chooi, Y. H., Choi, J. W., Li, S., Vederas, J. C., Da Silva, N. A., & Tang, Y. (2013). LovG: The Thioesterase Required for DihydromonacolinL Release and Lovastatin Nonaketide Synthase Turnover in Lovastatin Biosynthesis. Angewandte Chemie-International Edition, 52(25), 6472-6475.
4.Ames, B. D., Nguyen, C., Bruegger, J., Smith, P., Xu, W., Ma, S., Tsai, S. C. (2012). Crystal structure and biochemical studies of the trans-acting polyketide enoyl reductase LovC from lovastatin biosynthesis. Proceedings of the National Academy of Sciences of the United States of America, 109(28), 11144-11149.
5.Ma, S. M., & Tang, Y. (2007). Biochemical characterization of the minimal polyketide synthase domains in the lovastatin nonaketide synthase LovB. Febs Journal, 274(11), 2854-2864.
PreviousTour Page
NextTour Page
Retrieved from " http://2013.igem.org/Team:Wageningen_UR/Lovastatin "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
