"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-BAG/Building Block synthesis
From 2009.igem.org
Moodle Synthetic Yeast Wiki Protocols References Acknowledgements Contact
Home
Team
Synthetic Yeast Genome
B-A-G course
Building Block synthesis
A New Standard
Software Tools
Building Block Synthesis
Our general method of building block synthesis based on overlap extension PCR is an efficient and inexpensive way to construct Building Blocks. Building Blocks are assembled by overlap assembly PCR from standard 5’-OH, 3’-OH 60-80 base pair oligonucleotides purchased from a DNA synthesis company. The overlap extension assembly process is broken into two polymerase chain reactions:
"Templateless" PCR- Oligos are mixed and standard overlap extension assembly PCR program is utilized. Mixed length products are formed.
"Finish" PCR - Outer two primers are mixed with the Templateless PCR product, such that a higher concentration of outer primer is present in the mixture, and a second round of PCR is completed to enrich for full length products
Cloning - Finish PCR product is cloned using PGEM-T easy vector, taking advantage of Taq polymerase single 3’ adenylation.
Sequence verification - Sequencing is completed by outside sources and sequence alignment using programs such as “CloneQC” aids with correct clone verification
Templateless/Finish PCR assembly trouble-shooting
PCR program conditions can be altered to optimize construction of Building Blocks. Varying the anneal and extension temperature MAY assist in the production f the Building Block
The Ligation Chain Reaction (LCR)
an alternative procedure that produces a more specific overlap between two strands which eliminates the gaps between DNA strands altogether. In other words, most overlapping oligonucleotides will now have a region of around 35bp in which they complement each other. By using ungapped strands of DNA, the Taq DNA Ligase enzyme will be used to string the DNA backbone together.
Like a normal PCR, the cycling process produces large quantities of feasible DNA Building Block. One important theoretical advantage is the extensive overlapping of complementary DNA that occurs during the annealing process. With the LCR, it is more likely for oligonucleotides to anneal correctly and result in a final product of the correct sequence. The LCR would also reduce the likelihood of problems involved with loxP sites and other anomalies found in the normal TPCR and FPCR protocols. However, the LCR should be emphasized as an alternative to the current methods due to the expensive enzymes and reagents involved in the process of phosphorylation of the oligonucleotides for use in the LCR cycling  program.
USER Reaction
The Uracil Specific Excision Reagent is composed of two enzymes, which are commercially available and robust:
Uracil DNA glycosylase (UDG)
Targets the pyrimidine and ribose components of the uracil base and cleaves them from the phosodiester backbone.
DNA glycosylase/lyase Endo VII
Cleaves the exposed phosphodiester backbone, which allows the release of the 3-11 bases upstream of the uracil base, thus creating a nonpalindromic sticky end.
This sticky end that is released is utilized for the construction of Composite Building Blocks; when these ends are designed so as to be complementary only to that of the adjacent Building Block, the Building Blocks can be ligated only in a single, predestined order and in a single, desired orientation.
Retrieved from " http://2009.igem.org/Team:Johns_Hopkins-BAG/Building_Block_synthesis "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
