<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins-BAG_Building_Block_synthesis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Johns_Hopkins-BAG/Building_Block_synthesis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins-BAG/Building_Block_synthesis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Building_Block_synthesis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Building_Block_synthesis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins-BAG/Building_Block_synthesis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins-BAG/Building Block synthesis</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="header"><DIV id="header_bar_link"><A href="http://baderlab.bme.jhu.edu:8888/moodle18/" class="external text" rel="nofollow">Moodle</A><A href="http://baderlab.bme.jhu.edu/syntheticyeast/wiki/index.php/Main_Page" class="external text" rel="nofollow">Synthetic Yeast Wiki</A><A href="/Team:Johns_Hopkins-BAG/Protocols" title="Team:Johns Hopkins-BAG/Protocols"> Protocols</A><A href="/Team:Johns_Hopkins-BAG/Papers" title="Team:Johns Hopkins-BAG/Papers"> References</A><A href="/Team:Johns_Hopkins-BAG/Acknowledgements" title="Team:Johns Hopkins-BAG/Acknowledgements"> Acknowledgements</A><A href="/Team:Johns_Hopkins-BAG/Contact" title="Team:Johns Hopkins-BAG/Contact"> Contact</A></DIV></DIV><DIV id="menu"><UL><LI><A href="/Team:Johns_Hopkins-BAG" title="Team:Johns Hopkins-BAG"> Home</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/Team" title="Team:Johns Hopkins-BAG/Team"> Team</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/Synthetic_Yeast_Genome" title="Team:Johns Hopkins-BAG/Synthetic Yeast Genome">Synthetic Yeast Genome </A></LI><LI><A href="/Team:Johns_Hopkins-BAG/B-A-G_course" title="Team:Johns Hopkins-BAG/B-A-G course"> B-A-G course</A></LI><LI><STRONG class="selflink"> Building Block synthesis</STRONG></LI><LI><A href="/Team:Johns_Hopkins-BAG/A_New_Standard" title="Team:Johns Hopkins-BAG/A New Standard"> A New Standard</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/Software_Tools" title="Team:Johns Hopkins-BAG/Software Tools"> Software Tools</A></LI></UL></DIV><DIV id="JHU_content_box"><DIV id="JHU_content"><H2><SPAN class="mw-headline" id="Building_Block_Synthesis">Building Block Synthesis</SPAN></H2><P>Our general method of building block synthesis based on overlap extension PCR is an efficient and inexpensive way to construct Building Blocks. Building Blocks are assembled by overlap assembly PCR from standard 5’-OH, 3’-OH 60-80 base pair oligonucleotides purchased from a DNA synthesis company. The overlap extension assembly process is broken into two polymerase chain reactions:
</P><UL><LI>&quot;Templateless&quot; PCR- Oligos are mixed and standard overlap extension assembly PCR program is utilized. Mixed length products are formed.
</LI><LI>&quot;Finish&quot; PCR - Outer two primers are mixed with the Templateless PCR product, such that a higher concentration of outer primer is present in the mixture, and a second round of PCR is completed to enrich for full length products
</LI><LI>Cloning - Finish PCR product is cloned using PGEM-T easy vector, taking advantage of Taq polymerase single 3’ adenylation.
</LI><LI>Sequence verification - Sequencing is completed by outside sources and sequence alignment using programs such as “CloneQC” aids with correct clone verification
</LI></UL><P>Templateless/Finish PCR assembly trouble-shooting
</P><UL><LI>PCR program conditions can be altered to optimize construction of Building Blocks. Varying the anneal and extension temperature MAY assist in the production f the Building Block
</LI></UL><UL><LI>The Ligation Chain Reaction (LCR)
</LI><LI>an alternative procedure that produces a more specific overlap between two strands which eliminates the gaps between DNA strands altogether. In other words, most overlapping oligonucleotides will now have a region of around 35bp in which they complement each other. By using ungapped strands of DNA, the Taq DNA Ligase enzyme will be used to string the DNA backbone together. 
</LI><LI>Like a normal PCR, the cycling process produces large quantities of feasible DNA Building Block. One important theoretical advantage is the extensive overlapping of complementary DNA that occurs during the annealing process. With the LCR, it is more likely for oligonucleotides to anneal correctly and result in a final product of the correct sequence. The LCR would also reduce the likelihood of problems involved with loxP sites and other anomalies found in the normal TPCR and FPCR protocols. However, the LCR should be emphasized as an alternative to the current methods due to the expensive enzymes and reagents involved in the process of phosphorylation of the oligonucleotides for use in the LCR cycling  program. 
</LI></UL><P>USER Reaction
</P><P>The Uracil Specific Excision Reagent is composed of two enzymes, which are commercially available and robust: 
</P><UL><LI>Uracil DNA glycosylase (UDG) 
</LI><LI>Targets the pyrimidine and ribose components of the uracil base and cleaves them from the phosodiester backbone.
</LI></UL><LI>DNA glycosylase/lyase Endo VII 
</LI><LI>Cleaves the exposed phosphodiester backbone, which allows the release of the 3-11 bases upstream of the uracil base, thus creating a nonpalindromic sticky end.
</LI><P>This sticky end that is released is utilized for the construction of Composite Building Blocks; when these ends are designed so as to be complementary only to that of the adjacent Building Block, the Building Blocks can be ligated only in a single, predestined order and in a single, desired orientation.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Johns_Hopkins-BAG/Building_Block_synthesis">http://2009.igem.org/Team:Johns_Hopkins-BAG/Building_Block_synthesis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins-BAG/Building_Block_synthesis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins-BAG/Building_Block_synthesis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Building_Block_synthesis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Building_Block_synthesis&amp;oldid=166875" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>