"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-BAG/A New Standard
From 2009.igem.org
Moodle Synthetic Yeast Wiki Protocols References Acknowledgements Contact
Home
Team
Synthetic Yeast Genome
B-A-G course
Building Block synthesis
A New Standard
Software Tools
Contents
1 Building Blocks: Large DNA/Genome assembly made easy!
1.1 Overview
1.2 What is a Building Block?
1.3 Standard Assembly
1.4 Interchangability
Building Blocks: Large DNA/Genome assembly made easy!
Overview
The Johns Hopkins University Build a Genome team has proposed a new standard, RFC 38 , for the construction and assembly of large DNA. The physical assembly of standard parts is currently a non-standard process, which can either come from direct genome PCR with restriction enzyme sites incorporated into the PCR primers, or overlap assembly PCR. Furthermore, current Biobrick construction standards [RFC10, RFC11,RFC12 etc.] rely heavily on restriction enzyme based methods, which can be sequence, cost and time restrictive. This is especially true of large DNA assembly, where 100% control of over DNA sequence may be mandatory, such as in whole genome assembly. We propose a novel standard, The Building Block Method, for both the construction of standard parts and their assembly
This Building Block standard can be either:
Interchangeable
Non-interchangeable format, based on the desired use of the part.
What is a Building Block?
A Standard Build Block is a ~750 base pair sequence of DNA that is composed of annotated biological parts. This particular size represents a “sweet spot” for a currently optimized set of efficient assembly techniques (e.g. high accuracy sequencing in a single run) but need not be fixed in stone. The Building Block, however, may not be composed entirely of single biological parts (as in BioBricks) and the junction with neighboring Building Blocks may be essential for biological for function.
A Composite Building Block is composed of two or more Building Blocks. Composite Building Blocks may be constructed from non-interchangeable Building Blocks or interchangeable Building Blocks.
Standard Assembly
Building blocks are assembled using (Uracil Specific Excision Reagent) USER reaction assembly, which relies on a uracil base within the end of the Building Block DNA. Both 5’ ends of Building blocks must conform to the extremely degenerate consensus sequence of A{Nx}T in which x= 3,5,7,9 or 11. There are no other sequence requirements. The general format for a building block can be seen in Figure 1.
Non-interchangeable Building blocks MAY have unique and different sequences for the {Nx} region. These sequences MAY be part of the Building Block functional contents
Interchangeable Building blocks MUST have A{Nx}T sequences that conform to a standard sequence agreed to by the community of researchers that decide to use it.
Standard Building Block Diagram
Here the Building block is diagrammed to conform to the sequence 5’ A{Nx}U … A{Nx}T3’. Building blocks can be synthesized either in this format (by including terminal oligonucleotides each containing a single strategically placed deoxyuracil residue) or the more conventional 5’ A{Nx}T … A{Nx}T 3’, in which the terminal oligonucleotides are composed of the four normal DNA bases.  Either set of primers can be used to amplify a Building Block, depending on the stage in the synthesis procedure.
Interchangability
An Interchangeable Building Block contains standard sequences on both 3’ and 5’ends
A non-interchangeable Building Block MAY contain non-standard 3’ or  5’ ends to aid in the construction of Composite Building Blocks.
Interchangeable Building Block Standard Ends
Retrieved from " http://2009.igem.org/Team:Johns_Hopkins-BAG/A_New_Standard "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
