"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-BAG/Synthetic Yeast Genome Project
From 2009.igem.org
Moodle Synthetic Yeast Wiki Protocols References Acknowledgements Contact
Home
Team
Synthetic Yeast Genome
B-A-G course
Building Block synthesis
A New Standard
Software Tools
Hello, interchangeable part:  Meet your new chassis
Yes, we have quite a collection of interchangeable biological parts for this and that.  And sure, they can light up, detect, act, react, and do all sorts of biological gymnastics that would make any biologist feel all warm and fuzzy.  However, we need a good chassis to put all these goodies in.  iGEM needs a sleek, efficient, and advanced host to showcase all the newly derived luxury, sport, and tow packages.  This is where JHU Build-A-Genome's custom Saccharomyces cerevisiae, Sc2.0 comes in.  Harnessing the power of undergrad labor, the genome is being synthesized from scratch.  A few liberties have been taken in removing junk DNA, and introns to see if these features are really needed.  Also, this strain of yeast has been redesigned from the ground up to act as a biological 'goal seek' to find all of the solutions to the fabled minimal genome.
Genome Destabilization (Recombineering)
The driving force for minimization is supplied by the addition of numerous loxPsym sites between genes throughout the genome.  LoxPsym sites act like couplers between train cars allowing additions, deletions, and general reordering of the yeast genes.  Cre recombinase acts on loxPsym sites and, depending on local geometry and more likely random chance, allows recombination to occur between any pair of sites.  We employ a very specialized engineered Cre-ER chemically regulated by the human sex hormone estradiol so the process can be controlled at will. (Those who attended iGEM2008 will know the JHU Team has a tradition of always inserting the word SEX into its presentations). Such genome reorganization allows for a staggering number of permutations.  Most of the genomes in this “swarm” will not survive, but the ones that do can be analyzed.  Commonalities and linkages can be drawn by what genes are commonly left behind, with which other genes, and a huge linkage map can be made.  This map can lead to the minimal genome as well as the discovery of genes related by their presence in a biological pathway.
An entire chromosome arm “9R” has been synthesized according to our mad scheme and successfully inserted into yeast. Moreover, the native chromosome 9R can be trashed and the cell continues to smile and happily reproduce ad infinitum. Initial testing of the Cre-ER system strongly suggests that recombination occurs when estradiol is added.  However, a more visual test is appropriate.  A FLO8 gene has been added; this gene allows yeast to grow and organize into “fuzzy” structures similar to mold, technically referred to as pseudohyphae.  We are doing experiments to initiate recombination in the presence of both FLO8 and the synthetic chromosome, which contains a number of genes required for fuzziness.  A control group of fuzzy yeast (with no synthetic chromosome) should not respond to the addition of hormone.  The experimental group should show a reduction in colony count, varying colony size, as well as a mix of fuzzy and non-fuzzy colonies.  This should show that recombination is occurring and can be controlled.
Genome Stabilization (and destabilization)
Another level of redesign was thre relocation of tRNA genes. tRNA genes have been as a documented source of genome instability in Saccharomyces cerevisiae v. 1.0. As a measure to combat genome instability, as well as examine the effects of a more rigidly constrained genome, the tRNA genes for Sc 2.0 are being relocated to a specialized chromosome, the tRNA array. This entirely synthetic chromosome thus far only consists of the tRNA genes from chromosomes III and IX of S. cerevisiae. The tRNA array of chromosomes III and IX of S. cerevisiae 2.0 consists of 17 tRNA genes; each gene consists of the coding region of the tRNA From S. cerevisiae and the upstream and downstream flanking regions from a close relative of S. cerevisiae, Ashbya gossypii. Use of the fusion tRNA genes was required to avoid incorporating the repetitive sequences often found upstream of tRNAs in S. cerevisiae, as another aspect of our redesign philosophy for Sc 2.0 is to minimize nonessential repetitive DNA.  Each of the genes is a representation of the same gene found on chromosome III and IX with all duplicates removed. Between each tRNA gene is a rox recombination site that will allow for subsequent induced recombination within and limited to the tRNA chromosome, when the Dre recombinase is expressed at the whim of the investigator, so that the importance of order and relative copy numbers of tRNA genes on the tRNA array chromosome can be examined. Also it will be possible to detect deletions and more complex rearrangements of genes using this system and their phenotypic effect.
Retrieved from " http://2009.igem.org/Team:Johns_Hopkins-BAG/Synthetic_Yeast_Genome_Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
