"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-BAG/Protocols
From 2009.igem.org
Moodle Synthetic Yeast Wiki Protocols References Acknowledgements Contact
Home
Team
Synthetic Yeast Genome
B-A-G course
Building Block synthesis
A New Standard
Software Tools
Contents
1 Protocols
1.1 Start to Finish Protocol
1.1.1 Overview
1.1.1.1 The Hierarchy of Build-A-Genome Synthetic Biology
1.1.1.2 The Nomenclature
1.1.1.3 The Workflow
1.1.2 Oligos
1.1.2.1 Diluting oligos to the proper concentration for future use
1.1.2.2 Templateless primer mix
1.1.2.3 Outer primer mix
1.1.3 Templateless PCR
1.1.3.1 Reaction Setup
1.1.3.2 Reaction Conditions
1.1.4 Finish PCR
1.1.4.1 Reaction Setup
1.1.4.2 Reaction Conditions
1.1.5 Gel Electrophoresis
1.1.6 Ligation
1.1.6.1 Reaction Setup
1.1.6.2 Reaction Conditions
1.1.7 Bacterial Transformation
1.1.7.1 Preparation
1.1.7.2 Reaction Conditions
1.2 The Ligase Chain Reaction Protocol
1.2.1 INTRODUCTION
1.2.2 THEORY
1.2.3 COMPLETE OVERLAP PROTOCOL
1.2.4 DILUTION PROCEDURE/ GENERAL OPTIMIZATION
1.2.5 2009 ACCOMPLISHMENTS
1.2.5.1 Analysis of Clone C
1.2.6 Results/Discussion
Protocols
Start to Finish Protocol
Overview
The Hierarchy of Build-A-Genome Synthetic Biology
You will be building segments of synthetic DNA from small oligonucleotides (around 60 nucleotides long).  These building blocks (BBs) will ultimately be incorporated into larger synthetic constructs, then integrated into the yeast genome.
From smallest to largest:                 Size    Oligos  (ordered from a commercial supplier)                    60 nucleotides   Building Blocks (constructed by Build-A-Genome students) 750 bp  Chunks (constructed from building blocks)            30 kb  Genome                      12 Mb
The Nomenclature
In all cases, the chromosome number and arm (left or right) is listed first, followed by   the design version, the chunk, the BB, then the oligo (which is prefaced with an "o" --   that's the letter "o" not a zero).For example, to refer to a...  ...chunk:  3L.3_23.B3  This is chunk B3 from the left arm of chromosome 3, design version 3_23.  ...BB:    3L.3_23.B3.1  This is BB 1 of chunk B3 from the left arm of chromosome 3, design version 3_23.  ...oligo:  3L.3_23.B3.1.o01  This is the first (most 5') oligo of BB1 of chunk B3, from the left arm of chromosome 3,   design version 3_23.
Oligo Plates 96-well plates of oligos are labeled as above, but designated "p" for plate. Multiple plates exist for each chunk, and are labeled p1, p2, p3, etc. For example:  3L.3_23.B3.p2    This is plate 2 of chunk B3, from the left arm of chromosome 3, design version 3_23.
Clones Applicable when sending clones (white colonies) off for sequencing. You will pick approximately 24 clones per BB for CSPCR, and send at least 12 off for sequencing. These 12 are labeled according to the convention laid out above, but have the designation "c" for clone. For example: 3L.3_23.B3.11.c05 This is clone 5 of BB 11 from chunk B3, from the left arm of chromosome 3, design version 3_23.
The Workflow
1. Oligo mixes – Oligos are distributed by the instructors in 96 well plates.  You will first make templateless PCR oligo mixes (TPMs) and outer primer mixes (OPMs) for use in the templateless PCR (TPCR) and finish PCR (FPCR), respectively.
2. Templateless PCR (TPCR) – The TPCR is performed to form a larger construct (the BB) from the oligo mixes.
3. Finish PCR (FPCR) – The FPCR is performed to amplify the full-length BB produced in small quantities during the TPCR.
4. Gel Electrophoresis I - You will gage the success of your FPCRs by running small samples of the TPCR and FPCR on a gel.
5. Cloning BBs – BBs which have been determined to be of the correct size will be cloned into a generic cloning vector and used to transform bacteria.
6. Bacterial Transformation – Bacteria will be transformed with the cloned construct to create a population of clones available for sequencing and future use.
7. Colony Screening PCR (CSPCR) – CSPCR will be used to determine which clones have the transformed plasmid.
8. Gel Electrophoresis II– You will analyze the results of the CSPCR by running a small volume of the CSPCR on a gel.
9. Sequencing – Clones which are shown to contain plasmids with an insert of the intended size will be sent for sequencing.  You will perform sequence analysis to identify “perfect” clones.
Oligos
Oligos will be dispensed in 96 well plates. Each plate will contain two or more complete BBs. You will be given a diagram of the oligos on each plate.
Diluting oligos to the proper concentration for future use
Oligos are delivered to us at a concentration of 60uM. We have diluted them 1/10 to give you a working stock of 6uM. You will need to create a template primer mix (TPM) and an outer primer mix (OPM).
Templateless primer mix
All primers must be present at a concentration of 300nM (the oligos must all be diluted by 1/20)  before use in the TPM. These dilutions should be done in a 1.7ml eppendorf tube, and primer mixes should be stored at -20ºC when not in use.
If you have up to 20 primers in a BB:
* Label your TPM tube with the BB number and oligo concentration (300 nM)    * Add 10 ul each primer    * If necessary (if under 20 primers), add H2O to a final volume of 200ul.    * Mix thoroughly
If you have >20 primers in a BB
* Label your TPM tube with the BB number and oligo concentration (see below)    * Add 10 ul each primer to an eppendorf tube.    * Do not add any water. Your final volume should be >200 ul, with the concentration       of each primer at ~250 - ~275 nM.    * Record the different concentration of this TPM in your lab notebook.
Outer primer mix
The outer primers (first and last: x.01 and x.18, x.20, x.22, or x.22) must be present at a concentration of 3uM (the oligos must both be diluted by 1/2) in the OPM. Again, these dilutions should be done in a 1.7ml eppendorf tube, and primer mixes should be stored at -20oC when not in use.
Add 25 ul each of first and last primers to a labeled eppendorf tube.
Mix thoroughly
Label your OPM tube with the concentration (3uM)
Templateless PCR
Reaction Setup
Before you set up reactions, make sure you sign up for a PCR machine.
Thaw dNTPs on ice and keep Taq on ice AT ALL TIMES!
When each reagent has thawed, vortex briefly before pipetting. Do not vortex Taq!
A positive control reaction using control B2 must be used with every set of PCR reactions.
Reagent
Stock Concentration
Final Concentration
Vol/25ul reaction
dNTPs
2.5mM
200uM
2.0ul
Buffer
10x
1x
2.5ul
Taq
-
-
0.25ul
H2O
-
-
17.75ul
1. A cocktail should be set up for all PCRs. Usually n+1 reactions is sufficient, where n is your number of PCRs (including the positive control). However, where you may be setting up >10 reactions, you can use n+2 for you cocktail, >50 reactions, n+3, etc.
2. Vortex briefly and aliquot 22.5 ul per tube.
3. To each tube, add either 2.5 ul of TPM  (300nM templateless primer mix) or positive control B2.
4. Record the PCR machine you are using, and the positions in which your tubes are placed.
Reaction Conditions
PCR Program: Method 9  Links programs 1, 2, 3, 4, 5, 6, 7, 8  94oC, 3 minutes………………Program 1 55oC, 30 seconds……………..Program 2 72oC, 1 minute………………..Program 3 5 cycles: 94oC, 30 seconds 69oC, 30 seconds 72oC, 1 minute..........…...Program 4  5 cycles: 94oC, 30 seconds 65oC, 30 seconds 72oC, 1 minute..........…...Program 5 20 cycles: 94oC, 30 seconds 61oC, 30 seconds 72oC, 1 minute..........…...Program 6  72oC, 3 minutes..................…..Program 7 10oC, forever......................…..Program 8
5. When your reactions are done, transfer the positive control reaction to a 1.7 ml eppendorf tube and hold on to it. You will run this Templateless PCR positive control on a gel along with your finish PCRs and your Finish PCR positive control.
Finish PCR
Reaction Setup
1. A cocktail should be set up for all PCRs. Usually n+1 reactions is sufficient, where n is your number of PCRs (plus the positive control). However, where you may be setting up >10 reactions, you can use n+2 for you cocktail, >50 reactions, n+3, etc.
Reagent    Stock Concentration    Final Concentration    Vol]/25ul reaction dNTPs              2.5mM               200uM            2.0ul Buffer               10x               1x            2.5ul Taq                -               -                    0.25ul H2O                -               -                    15.75ul
2. Vortex briefly and aliquot 20.5 ul of the reaction cocktail per tube.
3. Add 2.0 ul of OPM (3 uM outer primer mix) for your finish PCRs or 2.0 ul of water for the positive control reaction.
4. For the F-PCRs (not the positive control), add 2.5 ul of your templateless PCR which has been diluted 1:5. The templateless PCR product concentration is unknown, but should be estimated on a gel. The PCR should then be diluted 1/5 in H2O, and the dilution used in the finish PCR reaction. For the positive control, add 2.5 ul of positive control B2. Do not add any other TPM, templateless PCR or OPM to this tube.
5. Record the PCR machine you are using, and the positions in which your tubes are placed.
Reaction Conditions
PCR Program: Method 11
Links programs 1, 2, 3, 10, 7, 8 94oC, 3 minutes……………..Program 1 55oC, 30 seconds……………Program 2 72oC, 1 minute………………Program 3 25 cycles: 94oC, 30 seconds 55oC, 30 seconds 72oC, 1 minute..........…...Program 10 72oC, 3 minutes..................…..Program 7 10oC, forever......................…..Program 8  If your PCR did not work, an alternative protocol that works well is:
1. Touchdown templateless PCR (Program 9 as above) 2. Touchdown finish PCR (Program 9 again) 3. Dilute your samples 1:10 and use in a second round of touchdown finish PCR (Program 9)
Gel Electrophoresis
On an agarose gel, you will run 5 ul of each PCR product (combine with 4 ul water and 1 ul 10x loading dye). You should run:
Templateless PCR positive control
Finish PCR positive control
Finish PCR products
You may choose to run templateless PCR products, those these are typically less informative
Once you take a picture of your gel, upload it to the course website. Make sure you check what the expected size of your building blocks are-do not assume that because you have a band, it is the correct product!
Ligation
Reaction Setup
In a clearly labeled PCR tube, make a cocktail (for n+1 reactions) of::
2.5 ul 2x Ligase Buffer (allow to thaw at room temperature and vortex well)
0.5 ul ligase
0.5 ul vector (200ng/ul)
Pipet up and down 5-10 times to mix (do not vortex) and aliquot 3.5 ul per tube. Add 1.5 ul finish PCR product (after verification of successful PCR by gel electrophoresis).  Total reaction volume should be 5ul.
Note: this is a very small reaction volume. As such, use of PCR tubes gives you more control when manipulating the reagents in the tube. If you are only setting up one ligation, you must set up a 10 ul ligation. At no time should you be trying to pipet 0.5 ul as our pipets are highly inaccurate below 1 ul.
Reaction Conditions
Incubate at 4oC overnight.
Bacterial Transformation
Preparation
This is the easiest step to "mess up" in the entire workflow. Cells must be kept cold at all times! Chill labeled tubes, ligation mixes, etc., on ice prior to mixing of cells and DNA (ligation mix). Other hints are listed below.
Never thaw frozen cells in your hand; always plunge frozen cells in ice and allow to thaw there. As soon as they thaw, they start dying. The more time passes (up to the point where SOC is added) the more they die and the less transformants you'll get. Also, never heat the cells prematurely; make sure your timing for the heat shock step is precise.
Check to make sure the water bath is holding at 42°C. If it is not, but is just slightly under (still above 40°C), that's fine. If it is too hot add a small amount of ice to bring it down; notify TAs so temperature can be adjusted. Do not adjust the temperature yourself.
Get one LB/Carb/IPTG/Xgal plate per transformation from 4°C, label, and put on bench top.
Obtain SOC and make sure it's sterile (it should be totally transparent). This should be prepared for you by the TAs in advance, but if you are curious about what it contains:
Reagent Stock concentration Final concentration Vol/1ml MgSO4             1M             10mM          10ul MgCl2             1M             10mM          10ul Dextrose           20%             0.36%          18ul LB             1x               -                  962ul
Reaction Conditions
Transfer 5 ul of ligation mix into 1.7 ml eppendorf tubes. Put these ligation reactions onto ice, then ask an instructor/TA for competent cells. Tubes must be at 0 oC before adding bacteria. The instructor/TA will thaw the bacteria for you and aliquot 25 ul of JM109 competent cells per tube.
Incubate on ice 30 minutes
Heat shock at 42oC for 45 seconds. Timing is critical here-30 seconds or 1 minute is not OK!
Immediately (ie, bring your ice bucket to the water bath) place heat shocked cells back on ice for 2 minutes
Add 400 ul SOC, pipet up and down with P200 set at 200) and incubate at 37oC for 30 minutes
Plate cells/SOC onto LB/Carb/IPTG/Xgal plates
Dump out glass beads into trash
Incubate plates overnight at 37oC (Plates should be incubated no more than 18 hours.  If you are not ready to move to the next step, you may hold them at 4oC wrapped in parafilm.  Whenever you are incubating or storing plates, they should be stored upside down to prevent condensation from dripping onto your plates.)
The Ligase Chain Reaction Protocol
INTRODUCTION
The Ligase Chain Reaction Protocol (LCR) is designed to replace and improve the current Templateless Chain Reaction Protocol (TPCR). The LCR is thought to have many distinct advantages:
1.) Accuracy of BB sequence should be increased (theoretically).
2.) The actual construction of the BB should be considerably easier thanks to the overlapping feature of the oligonucleotides.
3.) Money should be saved throughout the entire workflow because the LCR only needs to be run once to produce workable results.
In order to make a working protocol we must first subject the given protocol to stress tests to test for robustness.   The second part of our experiments should be to test known working samples of the LCR trials in order to prove that the experimental procedure works correctly. (We did this because our oligonucleotides were built incorrectly) We used samples from Jennifer Tullman’s batch of 3L.3_23.A1 BB.  The third part of our testing will be to assemble the failed BB’s from the Intersession 2009 work that Mary and I worked on.
THEORY
The theoretical background for the efficiency and accuracy of the LCR is based on the un-gapped oligonucleotides and manner of construction of the BB.
Step 1
Build un-gapped oligonucleotides that are around 60bp each
Step 2a
Mix the oligonucleotides together.
Step 2b
Phosphorylate the 5’ ends of the oligonucleotides so that the Taq DNA Ligase (Thermostable) can work on ligating the spaces
Step 3 Use a chained reaction to allow the overlapping pieces to anneal and then use the ligase to join the DNA backbone.
This is the step where the efficiency and the accuracy are most obvious. The oligonucleotides have a very high specificity for their complementary strand. This is thanks to the ~30bp stretch of complementary DNA. There is also a very low possibility for loxP sites since a palindrome sequence is much harder to find as the stretch of complementary sites for annealing grow longer.
Step 4
Every step thereafter is the same as the old BAG protocol.
COMPLETE OVERLAP PROTOCOL
Reagent
Volume (μL)
6 μM    Oligo Mixture
10
10x PNK Buffer
2.0
T4 PNK
1.0
ATP (0.1 mM)
1.0
DTT (0.1 mM)
1.0
dH2O
5.0
TOTAL:
20.0
Phosphorylation - Thermocycler Program 41 37˚C    60 minutes 60˚C    10 minutes 94˚C    30 seconds
Protocols
Reagent
Volume (μL)
Phosphorylation    Mixture
10.0
Taq Ligase Buffer
2.0
dH2O
5.0
NAD (0.1 mM)
1.0
Taq Ligase
2.0
TOTAL
20.0
Ligation - Thermocycler Program 44 94˚C 4 minutes (Program 12) 30 Cycles: (Program 42) 94˚C 30 seconds 65˚C 5 minutes 45˚C indefinitely
FPCR
Reagent
Volume (μL)
dNTPs (2.5 mM)
2.0
Taq Buffer (10x)
2.5
Taq
0.25
dH20
15.75
OPM (3 μM)
2.0
Ligation Mixture    (1:5 dilution)
2.5
TOTAL
25
Ligation  for Bacterial Transformation
Reagent
Volume (μL)
2x Ligase Buffer
2.5
Ligase
0.5
Vector (200 ng/μL)
0.5
FPCR Product
1.5
TOTAL
5
Reaction Conditions Incubate at 4oC overnight.
DILUTION PROCEDURE/ GENERAL OPTIMIZATION
For the Ligation mixture, I did a study on whether or not a 1:5,1:7,1:10 dilution would be needed for the the FPCR step. The Results are shown below. The grouping on the left takes is comprised of 12 BBs of three groups of 4 BBs each subjected to the above dilutions. Only the 1:7 dilution produced a pass result for all 4 BBs. In the end we decided that having more (1:5 dilution) would be best since the LCR does not actually amplify the original amount of starting DNA. It only assembles what is already in the mix. The test on the right is for a 1:8 dilution of Taq DNA Ligase. Many dilutions of the Taq Ligase were performed but the original amount still works very well. 2µL for a 20µL reaction.
 
LENGTH
(bp)
PLATE for    sequencing
SPRING 2009
Sent for    Sequencing using modified FPCR technique.
Perfect Clone
From FPCR?
Going to LCR    protocol.
Perfect Clone    from LCR?
From Me:
 
 
 
 
 
 
3R.3_37.D1.03
764
 
 
 
Yes
N/A
3R.3_37.D1.05
746
Batch 1_1
Yes
No*
 
 
3R.3_37.D1.10
818
 
 
 
Yes
N/A
 
 
 
 
 
 
 
3R.3_37.D2.02
740
 
 
 
Yes
N/A
3R.3_37.D2.03
765
 
 
 
Yes
N/A
3R.3_37.D2.09
753
Batch 1_1
Yes
No*
 
 
3R.3_37.D2.12
760
Batch 1_1
Yes
No*
 
 
3R.3_37.D2.13
757
Batch 1_1
Yes
No*
 
 
 
 
 
 
 
 
 
3R.3_37.D3.02
794
Batch 1_1
Yes
No*
 
 
3R.3_37.D3.04
763
Batch 1_2
Yes
No*
 
 
3R.3_37.D3.06
798
 
 
 
Yes
N/A
3R.3_37.D3.09
854
 
 
 
Yes
N/A
3R.3_37.D3.12
881
 
 
 
Yes
N/A
 
 
 
 
 
 
 
3R.3_37.E1.01
809
 
 
 
Yes
N/A
3R.3_37.E1.02
778
 
 
 
Yes
No
3R.3_37.E1.03
835
 
 
 
Yes
No
3R.3_37.E1.04
766
Batch 1_2
Yes
No*
 
 
3R.3_37.E1.05
842
 
 
 
Yes
No
3R.3_37.E1.07
752
Batch 1_2
Yes
No*
 
 
3R.3_37.E1.09
740
Batch 1_2
Yes
No*
 
 
3R.3_37.E1.12
882
 
 
 
Yes
No
 
 
 
 
 
 
 
3R.3_37.E2.01
757
 
 
 
Yes
No
3R.3_37.E2.04
754
Batch 1_2
Yes
No*
 
 
3R.3_37.E2.06
772
 
 
 
Yes
No
3R.3_37.E2.08
758
 
 
 
Yes
No
3R.3_37.E2.10
739
Batch 1_2/1_3
Yes
No*
 
 
 
 
 
 
 
 
 
From Mary:
 
 
 
 
 
 
3R.3_37.F2.06
753
Batch 1_3
Yes
PASS
 
 
3R.3_37.F2.07
733
Batch 1_3
Yes
PASS
 
 
3R.3_37.F2.09
794
Batch 1_3
Yes
PASS
 
 
3R.3_37.F2.14
844
Batch 1_3
Yes
PASS
 
 
3R.3_37.F2.17
688
Batch 1_3
Yes
No
 
 
3R.3_37.G1.07
801
Batch 1_4
Yes
PASS
 
 
3R.3_37.G1.10
754
Batch 1_4
Yes
FIXABLE
 
 
3R.3_37.G2.10
747
Batch 1_4
Yes
CHECK
 
 
3R.3_37.G3.03
791
Batch 1_4
Yes
  PASS
 
 
3R.3_37.G3.05
787
Batch 1_4
Yes
PASS
 
 
3R.3_37.G3.07
799
Batch 1_4/1_5
Yes
FIXABLE
 
 
3R.3_37.G4.08
742
Batch 1_5
Yes
No
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
 
3L.3_37.D1.06
840
 
No
No
Yes
N/A
3L.3_37.D1.07
754
 
No
No
Yes
N/A
3L.3_37.D2.07
802
 
No
No
Yes
N/A
3L.3_37.D2.15
750
 
No
No
Yes
N/A
3L.3_37.D3.12
462
 
No
No
Yes
N/A
3R.3_37.F1.02
771
 
No
No
Yes
No
3R.3_37.F2.15
756
 
No
No
Yes
No
3R.3_37.F2.16
759
 
No
No
Yes
No
3R.3_37.G2.04
798
 
No
No
Yes
N/A
3R.3_37.G4.02
784
 
No
No
Yes
N/A
3R.3_37.G4.06
850
 
No
No
Yes
N/A
 
 
 
 
 
 
 
The modified FPCR just used a touchdown approach of different annealing temperatures.
Refer to analysis for Plates 1_1. The starred “No” means that they were contaminated.
The N/A means that these were doled out to other members: Chris/Jasper The N/A
2009 ACCOMPLISHMENTS
Analysis of Clone C
Plate 1_1 This is the Intersession BBs assigned and passed through FPCR. The problem is the conflict with BB 3L.3_23.C2.01  From this picture and from the sequencing results, we see that the BBs are really not the ones that were intended to be built. This means that the clones are on the plates are actually clones of 3L.3_23.C2.01. Somewhere along the way, a dirty SOC/ or LB Carb mix was introduced and caused the wrong BBs to be inserted into the plasmid. As for the few BBs that did get inserted correctly, none of them were perfect clones.
From the CSPCR plate 1:
From the CSPCR plate 2:
Results/Discussion
From our construction successes (< 95%) and our lack of perfect clones we must conclude a mixed result from our experiments. This method has many advantages and disadvantages that need to be accounted for in order to assess the ultimate efficiency and accuracy of the protocol.
First and foremost, this is a very costly protocol (refer to Jasper’s report) and even with one pass through the FPCR to CSPCR workflow, the price of the reagents cannot be reconciled with the lack of successes on our sequencing results. We use more BBs since it costs more to overlap every part of BB. This extra cost cannot be offset by the increased efficiency since my results suggests that even with the new protocol, a single perfect clone cannot be found out of 12 submitted clones. The increased accuracy was supposed to improve the parts where the overlapped regions met.
We must also take into account the relative difficulty that these BBs offer in their construction. They are all of the ones that have failed previous years. There are quite a few that are over 800bp in length. From our sequence data, transversions were the #1 type of mutations found in the LCR sequences followed by deletions which are then followed by transitions. These mutations occurred in very random places and never at the same place among same BBs. All the highlighted areas were based on the abi sequence files that had clear well-defined ratings for each mutation. All in all, these BBs were poor examples in terms of relative design and composition of BBs encountered by BAG students over the course of a semester.
The time that it takes the LCR to progress is staggering considering the workflow jams that might occur if more PCR machines are not introduced into the classroom. Testing for shortened annealing times in which 5 minutes instead of 15 minutes is used should be conducted. This would require a comparative study among the same BBs.
Next time a negative control is also needed for every step in order to prove that this LCR is created from contaminated reagents.
This protocol is excellent for pushing BBs through the FPCR stage. It’s ultimate accuracy is still up for debate as we have a very bad sample to draw any reasonable conclusions from. Ultimately, it may be that the commercial markets have not deemed the LCR reaction reagents as a low cost alternative to the Taq polymerase driven reactions of our TPCR.
Retrieved from " http://2009.igem.org/Team:Johns_Hopkins-BAG/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
