"
Page
Discussion
View source
History
teams
Log in
 
Team:Johns Hopkins-BAG/Synthetic Yeast Genome
From 2009.igem.org
Moodle Synthetic Yeast Wiki Protocols References Acknowledgements Contact
Home
Team
Synthetic Yeast Genome
B-A-G course
Building Block synthesis
A New Standard
Software Tools
Contents
1 Synthetic Yeast Genome
1.1 Genome Stabilization
1.1.1 tRNA Array
1.2 Genome Destabilization
1.3 What do we have so far?
Synthetic Yeast Genome
Our iGEM team originated from participants in the Synthetic Yeast Genome Project, which is an ongoing effort to redesign and construct the Saccharomyces cerevisiae genome. Many alternations to the genome have been made in the new design including: removal of transposable element, relocation of tRNA genes, removal of "unnecessary genes", and incorporation of site-specific recombination sites. More details about the project can be found on the Synthetic Yeast Genome Wiki
The team members on our team focus on the the parts of the project that invesgiates the role of genome stability in the yeast genome.
Genome Stabilization
tRNA Array
tRNA genes have been show to be hotspots for DNA instability due to several reasons:
Target of retrotransposon incorporation.
Transcribed heavily, which can lead to single strand breaks during DNA synthesis, when RNA polymerase III and DNA polymerase collide.
Homologous regions upstream of tRNA genes, due to retrotransposon insertions, can aid homologous recombination of chromosomal regions
Thus in the yeast 2.0 genome all tRNA genes will be moved to their own chromosome, so that their absence throughout the rest of the genome can be observed. All introns will also be removed from tDNA, as it is another goal of the project to observed the effect of a yeast genome with less introns. Duplicate genes will also be removed.
rox Recombination sites will be places between each tRNA gene, which will be used to observe if there is a preference for tRNA gene relative location and copy number
Cartoon of tRNA array of Chromosomes III and IX
Genome Destabilization
Hello, interchangeable part.  Meet your new chassis.
Yes, we have quite a collection of interchangeable biological parts for this and that.  And sure, they can light up, detect, act, react, and do all sorts of biological gymnastics that would make any biologist feel all warm and fuzzy.  However, we need a good chassis to put all these goodies in.  iGEM needs a sleek, efficient, and advanced host to showcase all the newly derived luxury, sport, and tow packages.  This is where JHU Build-A-Genome's custom Saccharomyces cerevisiae, Sc2.0 comes in.
A few liberties have been taken in removing junk DNA, and introns to see if these features are really needed.  Also, this strain of yeast has been redesigned from the ground up to act as a biological 'goal seek' to find all of the solutions to the fabled minimal genome.  The driving force for minimization is supplied by the addition of numerous loxPsym sites between genes throughout the genome.  LoxPsym sites act like couplers between train cars allowing additions, deletions, and general reordering of the yeast genes.  Cre recombinase acts on loxPsym sites and, depending on local geometry and more likely random chance, allows recombination to occur between any pair of sites.  We employ a very specialized engineered Cre-ER chemically regulated by the human sex hormone estradiol so the process can be controlled at will. (Those who attended iGEM2008 will know the JHU Team has a tradition of always inserting the word SEX into its presentations). Such genome reorganization allows for a staggering number of permutations.  Most of the genomes in this “swarm” will not survive, but the ones that do can be analyzed.  Commonalities and linkages can be drawn by what genes are commonly left behind, with which other genes, and a huge linkage map can be made.  This map can lead to the minimal genome as well as the discovery of genes related by their presence in a biological pathway.
This is an image of S. cerevisiae growing in a pseudohyphal state
An entire chromosome arm “9R” has been synthesized according to our mad scheme and successfully inserted into yeast. Moreover, the native chromosome 9R can be trashed and the cell continues to smile and happily reproduce ad infinitum. Initial testing of the Cre-ER system strongly suggests that recombination occurs when estradiol is added.  However, a more visual test is appropriate.  A FLO8 gene has been added; this gene allows yeast to grow and organize into “fuzzy” structures similar to mold, technically referred to as pseudohyphae.
We are doing experiments to initiate recombination in the presence of both FLO8 and the synthetic chromosome, which contains a number of genes required for fuzziness.  A control group of fuzzy yeast (with no synthetic chromosome) should not respond to the addition of hormone.  The experimental group should show a reduction in colony count, varying colony size, as well as a mix of fuzzy and non-fuzzy colonies.  This should show that recombination is occurring and can be controlled.
What do we have so far?
Most of the left arm of chromosome three, and all right arm of chromosome nine have been constructed and assembled. We submitted most of the left arm of chromosome three as can be seen in the diagram below:
Most of Chunks A1-D3 of the Left arm of Chromosome III have been submitted
Check out our sandbox for a list of parts that make up Chromosome III L and what is in them:
http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2009&group=Johns_Hopkins-BAG
The diagrams below represent the content of each chunk. The gene names correspond to Saccharomyces genome database names
chunk A1
chunk A2
chunk B1
chunk B2
chunk B3
chunk C1
chunk C2
chunk C3
chunk D1
chunk D2
chunk D3
Retrieved from " http://2009.igem.org/Team:Johns_Hopkins-BAG/Synthetic_Yeast_Genome "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
