<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Johns_Hopkins-BAG_Software_Tools"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Johns_Hopkins-BAG/Software_Tools">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Johns_Hopkins-BAG/Software_Tools&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Software_Tools&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Software_Tools&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Johns_Hopkins-BAG/Software_Tools" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Johns Hopkins-BAG/Software Tools</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="header"><DIV id="header_bar_link"><A href="http://baderlab.bme.jhu.edu:8888/moodle18/" class="external text" rel="nofollow">Moodle</A><A href="http://baderlab.bme.jhu.edu/syntheticyeast/wiki/index.php/Main_Page" class="external text" rel="nofollow">Synthetic Yeast Wiki</A><A href="/Team:Johns_Hopkins-BAG/Protocols" title="Team:Johns Hopkins-BAG/Protocols"> Protocols</A><A href="/Team:Johns_Hopkins-BAG/Papers" title="Team:Johns Hopkins-BAG/Papers"> References</A><A href="/Team:Johns_Hopkins-BAG/Acknowledgements" title="Team:Johns Hopkins-BAG/Acknowledgements"> Acknowledgements</A><A href="/Team:Johns_Hopkins-BAG/Contact" title="Team:Johns Hopkins-BAG/Contact"> Contact</A></DIV></DIV><DIV id="menu"><UL><LI><A href="/Team:Johns_Hopkins-BAG" title="Team:Johns Hopkins-BAG"> Home</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/Team" title="Team:Johns Hopkins-BAG/Team"> Team</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/Synthetic_Yeast_Genome" title="Team:Johns Hopkins-BAG/Synthetic Yeast Genome">Synthetic Yeast Genome </A></LI><LI><A href="/Team:Johns_Hopkins-BAG/B-A-G_course" title="Team:Johns Hopkins-BAG/B-A-G course"> B-A-G course</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/Building_Block_synthesis" title="Team:Johns Hopkins-BAG/Building Block synthesis"> Building Block synthesis</A></LI><LI><A href="/Team:Johns_Hopkins-BAG/A_New_Standard" title="Team:Johns Hopkins-BAG/A New Standard"> A New Standard</A></LI><LI><STRONG class="selflink"> Software Tools</STRONG></LI></UL></DIV><DIV id="JHU_content_box"><DIV id="JHU_content"><H2><SPAN class="mw-headline" id="Software_Tools">Software Tools</SPAN></H2><P>Both the Sc2.0 project and the Build-A-Genome course require a number of specialized software applications and editing environments,; each of these is briefly summarized here.  Part of the philosophy of Build-A-Genome is to use ONLY open source software.
</P><P>•	<A href="http://baderlab.bme.jhu.edu:8888" class="external text" rel="nofollow">Moodle</A> is an existing open source course management system. It was modified to function as an online lab notebook for all B-A-G students to keep track of voluminous data and monitor progress of Building Block synthesis. Things like gel images and progress reports on the Building Blocks were stored in the MOODLE, along with various resources like protocols, lectures, and articles.
</P><P>•	CloneQC is a brand new program designed to automate the analysis of thousands of tracefiles of sequenced DNA. The somewhat complex process involved in calling whether or not the generated Building Block is an exact match to the designed Building Block was automated. The program can successfully identify exact and non-exact matches, and in some cases where it isn’t sure, it will ask a human to determine the status.
</P><P>•	<A href="http://baderlab.bme.jhu.edu/baderlab/index.php/BioStudio" class="external text" rel="nofollow">BioStudio</A> is a web application developed by B-A-G Team members Sarah Richardson and Pablo Lee layered on top of an existing generic genome browser called GBROWSE. BioStudio allows collaborative creation and editing of genome-sized DNA sequences, using revision control systems and most importantly, a graphical user interface, which facilitates many genome editing tasks. It provides various useful tools to aid the editor in the design, like ways to visualize the sequence at various levels of detail, or to visualize restriction site positions and recombination sites, or the percent content of certain base pairs, and so on.
</P><P>•	<A href="http://www.genedesign.org" class="external text" rel="nofollow">GeneDesign</A> is an existing program written by B-A-G Team members Sarah Richardson and Pablo Lee that provides tools for more specific gene or Building Block level design. Features like codon substitutions, silent site removals, restriction site design, and more are available. Most relevant to B-A-G, after the design of a gene or Building Block, tiling oligo order sheets can be automatically generated. 
</P><P>A new database to tie all of these resources together is being put together to provide a  unified database, which will give a solid base for the creation of additional custom software and web applications. It will also allow for the different pieces of software to interact with each other, which in turn will allow for a seamless pipeline system to be generated. 
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Johns_Hopkins-BAG/Software_Tools">http://2009.igem.org/Team:Johns_Hopkins-BAG/Software_Tools</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Johns_Hopkins-BAG/Software_Tools" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Johns_Hopkins-BAG/Software_Tools" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Software_Tools&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Johns_Hopkins-BAG/Software_Tools&amp;oldid=165556" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>