"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Davis/Project Selection
From 2011.igem.org
Home
Team
Project
Safety
Notebook
Data
Attributions
iGEM
Our Sponsors
Start a Family
Got a favorite BioBrick? Check our our process for expanding basic parts into part families.
Criteria
View our judging criteria for iGEM 2011 here .
Project Selection
The addition of regional competitions to the international iGEM Jamboree this year reduced the amount of time allotted to teams to complete their summer projects. We framed our project goals with the limited timeline in mind so that we could produce tangible results for the American regional competition in October and refine them in the time before the World Championship in November.
We also wanted to select a project that would build foundations for future teams. iGEM is founded on the spirit of open collaboration and enabling the design of genetic machines. We wanted to reflect these ideals by furthering the practice of synthetic biology in some way.
With these ideas in mind, we decided to create a streamlined process for the creation of mutant libraries, and to use said process to produce mutant libraries of the repressible LacI, TetR, and λ cI promoters, as well as GFP.
Why make mutant libraries?
When designing novel genetic circuits, synthetic biologists are limited to using parts that are already available or that they can manufacture themselves. The creation of new parts can be incredibly time-consuming, since they must be extracted from existing natural genes and converted to a DNA part standard (like BioBricks ) or modified from existing parts.
The parts registry contains many useful parts, but in cases where fine control over the function of a part is required for a genetic circuit to function properly, the existing selection may not offer enough variation in expression strength, chemical response, or other characteristics.
Fine-tuning the control of a part can be achieved by random mutation of the bases in the DNA sequence of a part to change some characteristic -- say, transcription initiation, fluorescence intensity, or protein binding affinity -- whilst maintaining the basic functionality of the part. This is generally done in such away that many mutants are created at once which are then screened for their activity levels.
This allows the creation of large numbers of mutants which are viable (that is, maintaining the core function of the part) in a relatively short period of time. By performing this process on popular parts, one team can produce not only parts that are of a suitable activity for their circuits, but also parts that might fall outside of their usage range but be useful to other teams. Submitting these mutant libraries to the registry allows other scientists access to a broader range of parts with a broader activity range and familiar functionality, preventing wasted man-hours and reagents.
Why mutate repressible promoters?
Because promoters are integral to the function of all genetic circuits, it is especially important that they be available in a broad selection and that their activity is well-documented and predictable.
R0010, R0040 and R0051 (the LacI, TetR, and λ cI repressible promoters) are among the most commonly used promoters in the registry . They are useful because they allow control control over gene expression that can be regulated using their corresponding repressor proteins, C0012, C0040 and C0051. This permits the construction of complex circuits that take advantage of the ability to switch, oscillate, or otherwise control the expression of genes by combining these parts in different ways.
These sample images show some of the possible outcomes of the same oscillating circuit. They differ only in the sample values for the transcription rate of the LacI promoter. The values used could be easily generated through mutating promoters and the apparent change in visible outcome is significant.
J23101, another extremely popular promoter, is a member of a constitutive promoter mutant library based on the wild-type J23119. This family offers mutants with similar functionality at different expression activity levels, making it an ideal choice for designing circuits where fine control over gene expression is beneficial.  The creation of promoter mutant libraries for R0010, R0040 and R0051 makes the same kind of fine expression control possible within repressible systems. Extensive characterization of these parts allows synthetic biologists to pick and  choose mutants with an ideal level of repressor binding affinity or overall promoter activity, which is vital for the efficient design of complex circuits.
Why mutate GFP?
GFP looks really cool, and a plate full of mutant GFP transformants looks even cooler.
Unlike the promoter activity of our LacI promoter mutants, the fluorescence activity and color of our GFP mutants is a result of the structure of the translated folded protein. We wanted to quickly and easily asses our mutagenic PCR protocol's ability to produce mutants of protein-coding genes that retain some level of the activity of the original protein. Because GFP expression is visible to the naked eye (and even more visible under long-wave UV light exposure from a handheld lamp), mutations to the GFP gene that produce alterations in protein structure that produce changes in color or fluorescence level can be quickly assessed.
Why use error-prone PCR?
Error-prone PCR is a powerful and efficient way to add random (or at least pseudo-random) mutations to DNA.
First and foremost, it is extremely cheap, and nearly all labs performing synthetic biology will already have the reagents required to carry out our mutagenic PCR procedure .
Secondly, it is fast. "Mutator" strains of E. coli exist which will introduce random mutations to exogenous DNA under multiple generations of growth in liquid culture, but this can take days or weeks of repeated culturing to produce the intended results. Our PCR protocol, on the other hand, can be carried out multiple times in a single workday.
Thirdly, it is easy to prepare. Targetted mutagenesis is an excellent way to engineer specific alterations to DNA by designing custom primers, but this process takes time and requires knowing beforehand the sort of mutations that are most likely to cause changes in observed phenotype. Error-prone PCR followed by proper screening can rapidly produce mutants of varying phenotypes without any prior knowledge of the key deterministic features of your target DNA molecule.
Lastly, error-prone PCR can be generalized to work with any BioBrick parts. Because BioBricks are created with standard prefix and suffix tags for restriction enzyme degradation, they are already primed for PCR mutagenesis. The primers used in our reaction are the same as those used to sequence BioBrick parts or for normal PCR reactions.
The benefit of using our procedure is that it has been tuned to work well with BioBrick parts. The mix has been pre-calculated, the template amount per reaction stays the same, and everything can be prepared in a big batch and stored in a fridge for later use. The degree of mutagenesis (i.e., the number of mutations per kilobase of DNA) can be increased by performing successive rounds of error-prone PCR by diluting the PCR product and re-using it as template.
Retrieved from " http://2011.igem.org/Team:UC_Davis/Project_Selection "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
