"
Page
Discussion
View source
History
teams
Log in
 
Team:Wageningen UR/Journal/week19
From 2012.igem.org
Top
Home
Team
The Team
Team Registration
Project
1. Introduction
2. PnA System
3. Virus-Like Particles
CCMV
HepB
Polerovirus Natural BioBrick
4. Outside Modification
5. Inside Modification
6. Detection of VLPs
7. Applications
8. The Constructor
9. Final Overview
Human Practices
Human Practices Home
1. Mini Symposium
2. Visiting Secondary Schools
3. Discovery Festival
Communication Science
4. Stakeholders
5. Munich CAS Conference
Safety
Introduction
1. General safety
2. Virus-related safety
3. Regulations
4. Safety suggestions
5. Safety of applications
Modeling
Human Body Model
Tertiary folding prediction
VLP assembly model
Attributions
Parts
Notebook
Journal
Protocols
Last year's project: Synchronized Oscillatory System
iGEM Home
Contact Us
Our Sponsors:
week 19: 3 September - 9 September
Mark's hair
Mark will get married soon, so he had to spend the entire monday having his hair done. When he finally returned on tuesday, there was no difference.
CCMV
3rd September (Hugo)
Ligated purified products IPTG_CCMV_NEG, IPTG_CCMV_Delta26_His, IPTG_CCMV_His into the pSB1C3 backbone. Transformed ligation products into MachI cells. This transformation gave no colonies.
4th September (Hugo)
Isolated VLPs from one of the dialysed solutions using an ultracentrifuge.
Digested the pSB1C3 backbone with EcoRI and PstI, followed by PCR purification.
5th September (Hugo)
Ligated purified products IPTG_CCMV_NEG, IPTG_CCMV_Delta26_His, IPTG_CCMV_His into the pSB1C3 backbone, followed by transformation into MachI cells.
Gel purified the product of Jeroen’s 3rd CCMV_COIL PCR step.
6th September (Hugo)
Ran the fourth PCR step of Jeroen’s CCMV_COIL project.
Directly transformed PCR product 4 into a Pjet vector, this however didn’t work.
7th September (Hugo)
Isolated VLPs from the second dialysed solution using an ultracentrifuge.
PCR purified amplicon’s of both duplo’s of PCR 4 of CCMV_COIL.
Digested the following with EcoRI + PstI: CCMV_COIL amplicon, IPTG_CCMV_NEG plasmid,  IPTG_CCMV_Delta26,  IPTG_CCMV_Delta26_His,  IPTG_CCMV_His and pSB1C3 (RFP backbone)
PCR purified digestion products.
Ligated purified digestion products into the pSB1C3 backbone.
Digested BBA_J04500 (IPTG backbone) with SpeI + PstI.
Digested purified CCMV_COIL amplicon with XbaI + PstI
PCR purified digestion products.
Ligated purified digestion products into the pSB1C3 backbone.
Used ligation products to transform DH5A cells.
Polero
Mr "success" Marnix left with polero coat protein at the downstream of IPTG induced promoter in the chloramphenical backbone in DH5 alpha strain. Han and Wouter took over this part experiment and  continued. DH5 alpha strain was minipreped and the plasmids were transformed into JM-109, a E.coli strain used for protein expression. Colony PCR confirmed the transformation was successful with both sequencing primers and polero primers.
TuYV
TuYV constructs were ligated into pJET blunt vector and transformed into Mach I strain. Later we got many colonies on the plate.
Two colonies PCR with pJET sequencing primers were done to identify 4 different constructs.
We got enough colony containing the coat protein and coat protein with his-tag, but only got one colony containing either coat protein with readthrough or coat protein with readthrough and his-tag.
Hepatitis B general
4 September
Digestion of HepatitisB with a his tag (PCR fragment)once with Xba1 and Pst1 and once additionally with DPN1 to cut the template DNA from the PCR reaction and avoid transformation with this template construct
Ligation of the digested fragments into both Bba_J04500 as well as Bba_J04550
5 September
Transformation with Mach1
7 September
colony PCR of the transformations from 4. September
-> nothing on the gel – also no positive control!
Hepatitis B ouside modification
7 September (Kees)
• 5’phosphorylated primers
By amplifying the whole plasmid, including the whole desired insert, only one pcr step is needed which can be used right away to transform bacteria.
The Primers: FW-TEV: P-AAGATAGCGGCGTTGAAGGAGGAAAATCTTTATTTTCAAGGTCTTGCTGCTGCTGtcgacgctggtggaggt
Reverse-TEV: P-TTCTTTTAGTGCTGCGATTTTCTCCTTCAACGCCGCTATCTTAGCAGCAGCAAGCAGATTATTGCCGACCCAG
Legend:
Blue: HepBcAg gene
Arrows: primers
Black: Backbone
Red Star: 5’phosphorylation on primers and DNA
Figure 1: Schematic overview of the quick change method used
The primers used for whole plasmid pcr arrived and were diluted to 100µM.  The PCR mixture: component Volume (µL) Primers (FW and rev) 2 Enzyme (Phusion Thermo) 0.5 DMSO 100% 1 Buffer (HF Thermo) 10 Template (20ng/µL) 1 MQ water 34 dNTPs 1
PCR protocol: 1: 98˚C; 2min 2:  98˚C; 30s 3: 63˚C; 30s 4: 72˚C; 30s     15× from step 2 5: 4˚C;   ∞ After the PCR, 10µL was used to digest with DpnI, 10µL was mixed with T4 ligase and the rest left untreated. DpnI was heat inactivated after which all three different samples (Ligase, DpnI, normal) were used to transform DH5α.
8 September  (Kees)
• Colonies
All plates contained colonies, however there was only one on the DpnI-digested plate, which would suggested incomplete digestion rather than a wanted colony. Therefore, more of the sample was plated in order to obtain more colonies.
9 September  (Kees)
• Checked the plates
There were no new colonies at all on the two new plates.
Hepatitis B inside modification
4 September
Digestion of the step 4 PCR fragment once with Xba1 and Pst1 and once additionally with DPN1 to cut the template DNA from the PCR reaction and avoid transformation with this template construct
Ligation of the digested fragments into both Bba_J04500 as well as pSB1C3
5 September
Transformation with Mach1
7 September
colony PCR of the transformations from 4. September
-> nothing on the gel – also no positive control!
GFP modification
3rd September
2nd PCR check of the minipreps from 31th Aug (this time only the ones with a high DNA concentration) with sequencing primers as well as with GFP primers (from GFP-coil fusion step1)
-> still no bands on the gel
4th September
To ligate the PCR product of step II into a pSB1C3/ pSB1AK3 it was necessary to digest the IPTG backbone and the RFP backbone first. Digested PCR product was already available. PCR purification after digestion
ligation of 4 samples:
R1 – Bba_J04450 + GFP I
R2 – Bba_J04450 + GFP II
I1 – Bba_J04500 + GFP I
I2 – Bba_J04500 + GFP II
The ligation mixtures were incubated for 30 min at 22°C and 10 min heat inactivation of the ligase at 65°C. Followed by a transformation of E.coli strains MachI/ DH5α with the ligated plasmids and a 2h growth in SOC-medium. Transformed cell were later plated on agar plates with the corresponding antibiotic. Unfortunately we found no colonies on the after 24 h respectively 48 h of growth at 37°C.
6th September
Ligation of the 1st step of the PCR reaction into pJET
transformation with Mach1
-> very slow growth, only a few colonies of which some where green (probably the template Bba_I13522 (untagged GFP behind a constitutive promoter))
[ previous week ]           [ next week ]
Retrieved from " http://2012.igem.org/Team:Wageningen_UR/Journal/week19 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
