"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Wageningen UR/InSilico
From 2012.igem.org
Top
Home
Team
The Team
Team Registration
Project
1. Introduction
2. PnA System
3. Virus-Like Particles
CCMV
HepB
Polerovirus Natural BioBrick
4. Outside Modification
5. Inside Modification
6. Detection of VLPs
7. Applications
8. The Constructor
9. Final Overview
Human Practices
Human Practices Home
1. Mini Symposium
2. Visiting Secondary Schools
3. Discovery Festival
Communication Science
4. Stakeholders
5. Munich CAS Conference
Safety
Introduction
1. General safety
2. Virus-related safety
3. Regulations
4. Safety suggestions
5. Safety of applications
Modeling
Human Body Model
Tertiary folding prediction
VLP assembly model
Attributions
Parts
Notebook
Journal
Protocols
Last year's project: Synchronized Oscillatory System
iGEM Home
Contact Us
Our Sponsors:
Contents
1 In silico folding prediction
1.1 Introduction
1.2 Phyre2
2 Results
2.1 Hepatitis B outside modification
2.2 Hepatitis B inside modification
2.3 CCMV
In silico folding prediction
Introduction
We designed the construction of the subunits based on tertiary structure models. These models are generated by the web-based tool Phyre2 . This server bases its prediction on both known structures and interactions between the amino acids. We used the tool to investigate whether the insertions and modifications cause any conformational changes to the subunits, which will most likely inhibit VLP formation.
Phyre2
We used Phyre2 to predict the tertiary 3D structure of both the wild type and the modified subunits. Phyre2 first compares the sequence of the input with a large database of over 10 million sequences.  This creates an evolutionary comparison of the input data with known structures, which is placed in a Hidden Markov Model. After this first step, the same comparison is done with the whole database, but in an inverse way. Starting with the structure, the sequence in extracted after which a comparison is made with the same database, creating a new database of evolutionary fingerprints.
The Markov model of the input sequence is compared with this newly created database, creating a highly efficient comparison which only needs 15% homology to predict a new structure. A detailed explanation of this principle can be found here .
Results
Hepatitis B outside modification
After making the prediction with Phyre2, the .pdb files were analyzed with the programs PDB viewer and Chimera 1.6.1 in order to create a visual alignment of the wild type and modified subunits. In case the alignment was, except for the insert, nearly complete, the construct was selected for further wetlab work.  For the Hepatitis B core antigen it was already known that the outside loop would allow modifications. The rigidity of the coil however, could cause this structure to fall apart. Therefore, we assembled multiple amino acid sequences, all containing the same coil flanked by different linkers in order to maintain the structure and linearizing the coil. Figure 1 depicts the most simplistic of these investigations in which only the K-coil is added.
Figure 1. The wild-type subunit (brown) serves as a template for the structural comparison of the modified subunit (blue). There is a slight shift in the structure, especially in the helices forming the loop. The coil is clearly visible, extending out of the wild-type structure at the top of the loop.
Hepatitis B inside modification
Looking at 3D models of the Hepatitis B VLP it can be seen that the C-terminal is presented on the inside. We also know that deletions to this part of the VLP can be done without influencing the assembly [1]. Since we want to package something inside the capsid it would be beneficial to delete as much of the C-terminal as possible, thus creating more space. Figure 2 shows the part we were able to delete from the wild-type core protein.
Comparing different possibilities to fuse the K-coil on the Hepatitis B core protein (with and without deletion of a part of the C-terminal, with or without a flexible linker, with or without a His tag) it was decided that the version with a C-terminal deletion and a flexible linker instead looks most promising (see figure 3 and 4).
Figure 2: Wildtype Hepatitis B - the part that can be deleted is marked black .
Figure 3: Hepatitis B with a flexible linker and a K-coil fused to the C-terminal .
Figure 4: Comparison Hepatitis B with inside modification [green - with red K-coil] - Hepatitis B wild type (with C-terminal deletion)[blue]. This shows that the folding of the main part is hardly influenced by the modification.
CCMV
Making a modification to the outside of CCMV however turned out to be more difficult in design. Literature does not give a clear indication of which part to modify in order to get the modification exposed to the outside of the VLP.  We compared the structure of CCMV to that of HepB, in which we identified a loop on the outside of the VLP. Replacing part of this loop by adding a coil flanked by flexible linkers yielded the structure as compared in Figure 2. Although this idea seems promising, we did not continue because of indications in unpublished work from the Virology Department of Wageningen University. Besides, the coils seems to be in the area that interacts with neighbouring VLPs during assembly.
A second option for adding a modification to the outside of CCMV is modifying the N-terminus after adding a linker. This linker should give the coil the option to present itself at the outside of the VLP. Personal communication with Dr. richard Kormelink from the virology department did suggest that such a construct might leed to a coil on the outside of the VLP, as seen in Figure 3. Therefore we indicated this modification as the most promising in CCMV.
Figure 5: The wild-type CCMV (blue) is next to perfectly aligned with the modifyed CCMV (orange). The only two differences are the coil itself and the N-terminal part, which is not involved in VLP formation but in interaction with the RNA in the whole virus.
Figure 6: The K-coil (blue) that is attached to a CCMV subunit (green) clearly seems to tend to bend to a different location as the N-terminal part in the wild-type protein (orange).
Retrieved from " http://2012.igem.org/Team:Wageningen_UR/InSilico "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
