<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BOKU-Vienna_Protocol skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BOKU-Vienna/Protocol</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                Menu </BUTTON><A class="navbar-brand page-scroll" href="https://2017.igem.org/Team:BOKU-Vienna">
                D.I.V.E.R.T.
            </A></DIV><DIV class="collapse navbar-collapse navbar-right navbar-main-collapse"><UL class="nav navbar-nav"><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Team
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Team" class="dropdown-a">Team</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Collaborations" class="dropdown-a">Collaborations</A></LI></UL></DIV><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown3" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Project
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Description" class="dropdown-a">Theory</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Experiments" class="dropdown-a">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Discussion" class="dropdown-a">Discussion</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Notebook" class="dropdown-a">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Protocol" class="dropdown-a">Protocol</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Model" class="dropdown-a">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Demonstrate" class="dropdown-a">Demonstrate</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown4" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Parts
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Parts" class="dropdown-a">Overview</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Basic_Part" class="dropdown-a">Basic Part</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Part_Collection" class="dropdown-a">Part Collection</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Human Practices
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/HP/Silver" class="dropdown-a">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/HP/Gold_Integrated" class="dropdown-a">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Engagement" class="dropdown-a">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/HP/Newsletters" class="dropdown-a">Newsletters</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A onclick="window.location.href='https://2017.igem.org/Team:BOKU-Vienna/Safety'" class="dropdown-toggle" type="button" id="dropdown1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Safety
                      </A></DIV></LI><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown6" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Medal Criteria
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/MedalCriteria" class="dropdown-a">Overview</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Attributions" class="dropdown-a">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/InterLab" class="dropdown-a">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Improve" class="dropdown-a">Improve</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Model" class="dropdown-a">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Contribution" class="dropdown-a">Contribution</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A onclick="window.location.href='https://2017.igem.org/Team:BOKU-Vienna/Sponsors'" class="dropdown-toggle" type="button" id="dropdown7" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Sponsors
                      </A></DIV></LI></DIV></DIV><DIV id="page-top" data-spy="scroll" data-target=".navbar-fixed-top"><NAV class="navbar scrollnav-custom navbar-fixed-bottom" role="navigation" id="scrollnav"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                    Menu </BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right navbar-main-collapse"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#">Top</A></LI><LI><A class="page-scroll" href="#Protocol">Protocol</A></LI><LI><A class="page-scroll" href="#Materials">Materials</A></LI><LI><A class="page-scroll" href="#Methods">Methods</A></LI></UL></DIV></DIV></NAV><HEADER class="myintro"><DIV class="myintro-body"><DIV class="container"><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><H1 class="brand-heading">Protocol</H1></DIV></DIV></DIV></DIV></HEADER><SECTION id="Protocol" class="container content-section text-center Protocol"><DIV class="container text-center content-section Protocol"><DIV class="col-lg-8 col-lg-offset-2"><H2>Protocol</H2><P>Most of these protocols were not developed by ourselves. Please see our <A href="https://2017.igem.org/Team:BOKU-Vienna/Attributions">attributions page</A> for more information. </P><H3>Content</H3><UL class="mylist"><LI><A href="#Materials" class="page-scroll">MATERIALS</A></LI><LI class="small">• <A href="#Strains" class="page-scroll">Strains </A></LI></UL><UL><LI class="small">• <A href="#LBmedium" class="page-scroll">LB medium </A></LI></UL><UL><LI class="small">• <A href="#LBagar" class="page-scroll">LB agar</A></LI></UL><UL><LI class="small">• <A href="#YPmedium" class="page-scroll">YP medium</A></LI></UL><UL><LI class="small">• <A href="#YPagar" class="page-scroll">YP agar</A></LI></UL><UL><LI class="small">• <A href="#Agarose" class="page-scroll">Agarose gel </A></LI></UL><LI><A href="#Methods" class="page-scroll">METHODS</A></LI><LI class="small">• <A href="#CompetentColi" class="page-scroll">Preparation of chemically competent <I>E. coli</I></A></LI><UL><LI class="small">• <A href="#TransformationColi" class="page-scroll">Transformation of chemically competent <I>E. coli</I></A></LI></UL><UL><LI class="small">• <A href="#PreparationYeast" class="page-scroll">Preparation of electro competent <I>S. cerevisiae</I></A></LI></UL><UL><LI class="small">• <A href="#TransformationYeast" class="page-scroll">Transformation of electro competent <I>S. cerevisiae</I></A></LI></UL><UL><LI class="small">• <A href="#Miniprep" class="page-scroll">Plasmid DNA Extraction </A></LI></UL><UL><LI class="small">• <A href="#PCRcleanup" class="page-scroll">PCR Clean-Up/ Gel Extraction </A></LI></UL><UL><LI class="small">• <A href="#Q5" class="page-scroll">Q5 PCR</A></LI></UL><UL><LI class="small">• <A href="#OneTaq" class="page-scroll">OneTaq colony PCR</A></LI></UL><UL><LI class="small">• <A href="#GG" class="page-scroll">Golden Gate</A></LI></UL><UL><LI class="small">• <A href="#mRNA" class="page-scroll">mRNA extraction</A></LI></UL><UL><LI class="small">• <A href="#cDNA" class="page-scroll">cDNA synthesis</A></LI></UL><UL><LI class="small">• <A href="#Flow Cytometric Determination of GFP Expression in Yeast." class="page-scroll">Flow Cytometric Determination of GFP Expression in Yeast.</A></LI></UL><UL><LI class="small">• <A href="#CRISPR" class="page-scroll">Preparation of electro competent <I>E. coli </I> for CRISPR mediated integration via λ recombination</A></LI></UL><UL><LI class="small">• <A href="#TransformationEcompetentEcoli" class="page-scroll">Transformation of electrocompetent <I> E. coli</I></A></LI></UL><UL><LI class="small">• <A href="#RT" class="page-scroll">RT activity assay</A></LI></UL><UL><LI class="small">• <A href="#Temperature" class="page-scroll">Temperature stability test of plasmids</A></LI></UL><UL><LI class="small">• <A href="#Lethality" class="page-scroll">Lethality test of CRISPR-PPP </A></LI></UL></DIV></DIV></SECTION><SECTION id="Materials" class="container content-section text-center Protocol"><DIV class="container text-center content-section Protocol"><DIV class="col-lg-8 col-lg-offset-2"><H2>Materials</H2><H4 id="Strains" class="test">Strains</H4><P><I> E. coli </I>: Dh10b
<I> S. cerevisiae </I>: S288C
</P><H4 id="LBmedium" class="test">LB medium</H4><P> LB medium is a standard liquid medium for the cultivation of <I>E. coli. </I></P><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Peptone</TD><TD>10 g</TD></TR><TR><TD>Yeast Extract</TD><TD>5 g</TD></TR><TR><TD>NaCl</TD><TD>5 g</TD></TR><TR><TD>RO-water</TD><TD>fill up to 1000 mL</TD></TR></TBODY></TABLE><P>The LB medium is sterilized via autoclavation. After cooling down antibiotics may be added.</P><H4 id="LBagar" class="test">LB agar</H4><P> LB agar is a standard agar for the cultivation of <I>E. coli. </I></P><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Peptone</TD><TD>10 g</TD></TR><TR><TD>Yeast Extract</TD><TD>5 g</TD></TR><TR><TD>NaCl</TD><TD>5 g</TD></TR><TR><TD>Agar</TD><TD>14 g</TD></TR><TR><TD>RO water</TD><TD>fill up to 1000 mL</TD></TR></TBODY></TABLE><P>The LB agar is sterilized via autoclavation. After cooling down to approximately 60 °C antibiotics may be added. The LB agar is poured into disposable petri dishes.</P><H4 id="YPmedium" class="test">YP medium</H4><P> LB medium is a standard liquid medium for the cultivation of <I>S. cerevisiae. </I></P><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Soy Peptone</TD><TD>20 g</TD></TR><TR><TD>Yeast Extract</TD><TD>10 g</TD></TR><TR><TD>RO-water</TD><TD>fill up to 1000 mL</TD></TR></TBODY></TABLE><P>The YP medium is sterilized via autoclavation. After cooling down 50 mL of 10x carbon source (eg. 10x glucose) and antibiotics may be added.</P><H4 id="YPagar" class="test">YP agar</H4><P> LB agar is a standard agar for the cultivation of <I>S. cerevisiae. </I></P><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Soy Peptone</TD><TD>20 g</TD></TR><TR><TD>Yeast Extract</TD><TD>10 g</TD></TR><TR><TD>Agar</TD><TD>20 g</TD></TR><TR><TD>RO water</TD><TD>fill up to 1000 mL</TD></TR></TBODY></TABLE><P>The YP agar is sterilized via autoclavation. After cooling down 50 mL of 10x carbon source (eg. 10x glucose) and antibiotics may be added. The YP agar is poured into disposable petri dishes.</P><H4 id="Agarose" class="test">Agarose gel</H4><P> Agarose gels are used for a gel electrophoresis.</P><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Agarose</TD><TD>5.4 g</TD></TR><TR><TD>50x TAE buffer</TD><TD>7.2 g</TD></TR><TR><TD>RO water</TD><TD>fill up to 360 g</TD></TR></TBODY></TABLE><P><UL class="ulclass"><LI>• Heat 500 mL flask in microwave until liquid is completely clear.</LI><LI> • Add 8µL of Midori Green.</LI><LI> • Pour liquid into 3 prepared gel casts with 2 combs. </LI><LI> • Gel solidifies after approximately 30 mins.</LI></UL></P></DIV></DIV></SECTION><SECTION id="Methods" class="container content-section text-center Protocol"><DIV class="container text-center content-section Protocol"><DIV class="col-lg-8 col-lg-offset-2"><H2>Methods</H2><H4 id="CompetentColi" class="test">Preparation of chemical competent <I>E. coli</I></H4><P><UL class="ulclass"><LI>• <I>E. coli </I> strain DH10B is inoculated from a glycerol stock onto an LB plate; the inoculum is streaked on the plate using a loop so as to obtain individual colonies. The plate is incubated overnight at 37 °C. 
</LI><LI>• Inoculate 10 mL of LB medium with a single colony and incubate the flask overnight in a shaker-incubator (37°C, shaking 180 rpm). 
</LI><LI>• The following day, transfer 2 mL of this culture to a flask containing 200 mL LB medium and incubate for about 2h until OD<SUB>600</SUB> reaches 0.6. 
</LI><LI>• Cool down the cells on ice for 10 minutes. The cells are pelleted in a centrifuge for 5 min at 4,500 rpm (4,000 x g) at 4°C. The cells are resuspended in 0.4 volume of ice-cold TFB1*. 
</LI><LI> • Repeat the centrifugationstep. Resuspend the pellet in 1/25 volume of ice-cold TFB2**. 
</LI><LI>• the cells are aliquoted 100 µL per tube and stored at -80 °C. </LI></UL></P><P>* Solution TFB1: 30 mM potassium acetate, 10 mM CaCl<SUB>2</SUB>, 50 mM MnCl<SUB>2</SUB>, 100 mM RbCl, and 15% glycerol; adjust to pH 5.8 (with 1 M acetic acid), filter-sterilize, and store at 4 °C (ready to use) or at room temperature (cool down before use). 
 ** Solution TFB2: 100 mM MOPS (or  PIPES), 75 mM CaCl<SUB>2</SUB>, 10 mM RbCl, and 15% glycerol; adjust to pH 6.5 (with 1 M KOH), filter-sterilize, and store at 4 °C (ready to use) or at room temperature (cool down before use).</P><H4 id="TransformationColi" class="test">Transformation of chemical competent <I>E. coli</I></H4><P><UL class="ulclass"><LI> • Frozen chemically competent cells (100 µL per tube) are thawed on ice. 
</LI><LI> • The entire ligation (or other DNA sample to be transformed) is added to the cells, and the mix incubated on ice for 30 minutes. 
</LI><LI>• The cells and DNA mix is heat shocked for 90 s at 42 °C. 
</LI><LI>• Allow the cells to recover on ice for 5 minutes. 
</LI><LI> • Add 1 mL of LB medium to the cells and incubate the tube at 37 °C in a shaker-incubator (150 rpm) for 45 min to 1h. 
</LI><LI>• Plate the appropriate amount of cells on selective LB agar.</LI></UL></P><H4 id="PreparationYeast" class="test">Preparation of electro competent <I>S. cerevisiae</I></H4><P>Please make sure that the sorbitol (1M) for usage after pre-treatment is ice-cold before you start. Also, take care that your cells are kept ice-cold throughout the whole procedure, from pre-treatment until freezing the final aliquots at - 80 °C.
<U>Day 1 (evening)</U><UL class="ulclass"><LI>• Prepare the pre-culture: inoculate 5 - 10 mL selective YPD medium with a fresh colony (no older than 3 weeks) and incubate overnight (o/n; ~16 h) (shaker; 180 rpm; 25 °C).</LI></UL></P><P><U>Day 2 (evening)</U></P><UL class="ulclass"><LI>• Main culture: Inoculate 100 mL non-selective YPD medium with the preculture and incubate o/n (~16 h) (shaker; 180 rpm; 25 °C); the end-OD should be between 1.2 and 2.5. Calculate the volume of pre-culture needed for inoculation of the main culture.</LI></UL><P><U>Day 3</U></P><UL class="ulclass"><LI>
• Measure OD and split 90 mL of the culture into 2 Falcon-tubes (2 x 45 mL).
</LI><LI>• Harvest the cells by centrifuging (5 min; <NOBR>1500 xg</NOBR> and 4 °C), then discard the supernatant. 
</LI><LI>• Add 10 mL of pre-treating solution* (25 °C) to each of the pellets and resuspend the cells. 
</LI><LI>• Incubate the cells for 30 min (Shaker; 180 rpm; 25 °C).
</LI><LI>• Add 40 mL ice-cold sorbitol (1 M) to each Falcon-tube and harvest cells (5 min; <NOBR>1500 xg</NOBR>; 4 °C), then discard supernatant.
</LI><LI>• Combine pellets in one Falcon-tube and resuspend cells in 45 mL ice-cold sorbitol (1 M). 
</LI><LI>• Harvest the cells (5 min; <NOBR>1500 xg</NOBR>; 4 °C), then discard supernatant. 
</LI><LI>• Resuspend in cells in 200 µL ice-cold sorbitol (1 M); aliquot cells into pre-chilled Eppendorf tubes (á 100 µL) and keep them on ice until transformation (long term storage: - 80 °C). </LI></UL><P>* pre-treating solution (20 mL are needed for one 100mL cell batch): 7.4 mL AD; 12 mL of 1 M Sorbitol (for a final concentration of 0.6 M); 200 µL of 1 M Tris-HCl, pH 7.5 (for a final concentration of 10 mM); 200 µL of 1 M DTT (for a final concentration of 10 mM); 200 µL of 10 M Lithium acetate (for a final concentration of 100 mM)
</P><H4 id="TransformationYeast" class="test">Transformation of electro competent <I>S. cerevisiae</I></H4><P><UL class="ulclass"><LI>• An 100 µL aliquot of the electro-competent <I>S. cerevisiae</I> cells is mixed very gently with a small volume (use up to 30 µL eluate after column purification, see <A href="#PCRcleanup" class="page-scroll">PCR Clean-Up/ Gel Extraction</A>) of the linearized DNA. As a negative control, cells should be transformed with the elution solution or AD. The mixture should not be vortexed to avoid shearing of the DNA and cells.
</LI><LI>• The mixture is then transferred into a chilled electroporation cuvette (2 mm) and incubated on ice for 5 minutes. 
</LI><LI>• Electroporation is performed at the BioRad  MicroPulser<SUP>TM</SUP> using following parameters: 2000 V, 25 µF and 186 Ω. 
</LI><LI>• Immediately after transformation 1 mL of ice-cold YPD-media (or 1 M sorbitol for His-selection) is added to the electroporated cells in the cuvette, then the content of the cuvette is transferred carefully into a sterile microcentrifuge tube.
</LI><LI>• Regenerate the cells for at least 1.5 h to maximum3h at 28 °C before plating aliquots (eg 20 µL, 200 µL) on selective agar plates. Keep the rest at 4 °C. 
</LI><LI>• Incubate the plates at 28 °C until colonies appear (approx. 48 h). </LI></UL></P><H4 id="Miniprep" class="test">Plasmid DNA extraction</H4><P>Plasmid DNA extractions are performed using Hi Yield<SUP>®</SUP> Plasmid MINI kit according to the <A href="http://www.suedlabor.de/item/30-HYPD" target="_blank"> manufacturer's protocol </A>.
</P><H4 id="PCRcleanup" class="test">PCR Clean-up/ Gel Extraction</H4><P>PCR clean-ups and Gel extractions are performed using Hi Yield<SUP>®</SUP> PCR Clean-up/ Gel Extractions Kit according to the <A href="http://www.suedlabor.de/item/30-HYDF" target="_blank"> manufacturer's protocol </A>.</P><H4 id="Q5" class="test">Q5 PCR</H4><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Q5 2x MM <A href="https://www.neb.com/products/m0492-q5-high-fidelity-2x-master-mix" target="_blank">(NEB)</A></TD><TD>12.5 µL</TD></TR><TR><TD>AD</TD><TD>variable</TD></TR><TR><TD>Primer forward</TD><TD>1.25 µL</TD></TR><TR><TD>Primer reverse</TD><TD>1.25 µL</TD></TR><TR><TD>Template</TD><TD>1 ng</TD></TR><TR><TD>Total Volume</TD><TD>25 µL</TD></TR></TBODY></TABLE><P>Vortex and spin down PCR tube. Put tube in a thermocycler.</P><P><U> Thermocycler settings: </U></P><TABLE class="table"><THEAD><TR><TH>Step</TH><TH>Temperature in °C</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>98</TD><TD>30''</TD></TR><TR><TD>2</TD><TD>98</TD><TD>10''</TD></TR><TR><TD>3</TD><TD>variable - Primer annealing temperature</TD><TD>30''</TD></TR><TR><TD>4 → 2 30x</TD><TD>72</TD><TD>variable -
30'' /kb
</TD></TR><TR><TD>5</TD><TD>72</TD><TD>3'</TD></TR></TBODY></TABLE><P>Add 5 µL of 6x Loading Dye for gel electrophoresis. 
</P><H4 id="OneTaq" class="test">OneTaq colony PCR</H4><TABLE class="table"><THEAD><TR><TH>Name</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>oneTaq 2x MM <A href="https://www.neb.com/products/m0482-onetaq-2x-master-mix-with-standard-buffer" target="_blank">(NEB)</A></TD><TD>6 µL</TD></TR><TR><TD>AD</TD><TD>5 µL</TD></TR><TR><TD>Primer forward</TD><TD>0.5 µL</TD></TR><TR><TD>Primer reverse</TD><TD>0.5 µL</TD></TR><TR><TD>Template Colony</TD><TD>1 tip</TD></TR><TR><TD>Total Volume</TD><TD>12 µL</TD></TR></TBODY></TABLE><P>Vortex and spin down PCR tube. Put tube in a thermocycler.</P><P><U> Thermocycler settings: </U></P><TABLE class="table"><THEAD><TR><TH>Step</TH><TH>Temperature in °C</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>94</TD><TD>3'</TD></TR><TR><TD>2</TD><TD>94</TD><TD>30''</TD></TR><TR><TD>3</TD><TD>variable - Primer annealing temperature</TD><TD>40''</TD></TR><TR><TD>4 → 2 30x</TD><TD>68</TD><TD>variable -
1' /kb
</TD></TR><TR><TD>5</TD><TD>68</TD><TD>5'</TD></TR></TBODY></TABLE><P>PCR Mix is ready for electrophoresis. No loading dye is needed. 
</P><H4 id="GG" class="test">Golden Gate Assembly with BsaI/ BpiI</H4><P>For 20µL reaction volume pipette in a PCR Tube:</P><TABLE class="table"><THEAD><TR><TH>Substance</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>Empty Backbone 40 nM</TD><TD>1 µL</TD></TR><TR><TD>Backbone Insert(s) 40 nM</TD><TD>2 µL each</TD></TR><TR><TD>linear DNA Insert(s) 40 nM</TD><TD>4 µL each</TD></TR><TR><TD>Cutsmart 10x </TD><TD>2 µL</TD></TR><TR><TD>ATP</TD><TD>2 µL</TD></TR><TR><TD>BsaI or BpiI</TD><TD>1 µL</TD></TR><TR><TD>T4 ligase 1:10 diluted</TD><TD>1 µL</TD></TR><TR><TD>AD</TD><TD>variable</TD></TR><TR><TD>total volume</TD><TD>20 µL</TD></TR></TBODY></TABLE><P>Vortex and spin down PCR tube. Put tube in a thermocycler.</P><TABLE class="table"><THEAD><TR><TH>Step</TH><TH>Temperature in °C</TH><TH>Time in minutes</TH></TR></THEAD><TBODY><TR><TD>1</TD><TD>37</TD><TD>2</TD></TR><TR><TD>2 → 1 10x</TD><TD>16</TD><TD>5</TD></TR><TR><TD>3</TD><TD>37</TD><TD>10</TD></TR><TR><TD>4</TD><TD>55</TD><TD>30</TD></TR><TR><TD>5</TD><TD>80</TD><TD>10</TD></TR><TR><TD>6</TD><TD>23</TD><TD>10</TD></TR></TBODY></TABLE><P>Golden Gate Mix is ready for transformation. 
</P><H4 id="mRNA" class="test">mRNA extraction</H4><P><U>Sample preparation:</U><UL class="ulclass"><LI>
• Centrifuge 10 mL of <I>E. coli</I> overnight culture and discard the supernatant. 
</LI><LI>• Add 1 mL of TRI REAGENT (Ambion, AM9738, stored dark at 4°C) to  the <I>E. coli</I> pellet, then add approximately 500 µL glass-beads in a sealed tube. 
</LI><LI>• Ribolyze once at speed 5.5. m/s for 40 seconds. 
</LI><LI>• Allow samples to stand for 5 minutes at room temperature. 
</LI><LI>• Add 200 µL of chloroform per 1 mL of TRI REAGENT and shake vigorously for 15 seconds. 
</LI><LI>• Allow the samples to stand for 10 - 15 minutes at room temperature. Centrifuge at full speed for 10 - 15 minutes at 4 °C to separate phases [centrifugation separates the mixture into 3 phases: a red organic phase (containing protein), an interphase (containing DNA), and a colorless upper aqueous phase (containing RNA)]. </LI></UL></P><P><U>RNA isolation</U>RNA precipitation:</P>
• Transfer the aqueous phase to a fresh tube and add 0.5 mL of isopropanol per 1 mL of TRI REAGENT used for the initial homogenization. Take care not to transfer the interphase or the red DNA phase.
<LI>• Allow the sample to stand for 5 - 10 minutes at room temperature. 
</LI><LI>• Centrifuge at full speed for 10 minutes at 4'C. The RNA precipitate will form a pellet on the side and bottom of the tube. </LI><P>RNA wash:</P><UL class="ulclass"><LI>• Remove the supernatant and wash the RNA pellet by adding 1 mL (minimum) of 75 % ethanol per 1 mL of TRI REAGENT used for the initial homogenization.
</LI><LI>• Vortex the sample. 
</LI><LI>• Centrifuge at full speed for 5 minutes at 4 °C. </LI></UL><P>RNA solubilization: </P><UL class="ulclass"><LI>• Briefly dry the RNA pellet by air-drying. [Do not let the RNA pellet dry completely, as this will greatly decrease its solubility. Do not dry the RNA pellet by centrifugation under vacuum]. 
</LI><LI>• Add an appropriate volume (50 - 100 µL) of nuclease-free water.
</LI><LI>• To facilitate dissolution, mix by repeated pipetting with a micropipette and incubation at 65 °C for 10 - 15 minutes. </LI></UL><H4 id="cDNA" class="test">cDNA synthesis</H4><P>cDNA synthesis is performed using Biozym cDNA Synthesis kit according to the <A href="http://www.biozym.com/DesktopModules/WebShop/downloads/331470-170130%20Biozym%20cDNA%20Synthesis%20Kit.pdf" target="_blank"> manufacturer's protocol </A> with gene-specific primers.
</P><H4 id="Flow Cytometric Determination of GFP Expression in Yeast." class="test">Flow Cytometric Determination of GFP Expression in Yeast.</H4><P>The cultivations are performed in 24 deepwell plates <UL class="ulclass"><LI>
• Fill every used well with 1 mL YP media containing 2 % glucose and a selection marker, inoculated with clones of interest and the appropiate controls and incubated over night at 30°C, 280 rpm overnight. 
</LI><LI>
• On the next day, harvest the cells by centrifugation and remove the supernatant 
</LI><LI>
• Resuspend the cells in YP media without any additional carbon source to wash the cells and again harvest them by centrifugation. 
</LI><LI>
• Again resuspend the cells in 1 mL YP media. 
</LI><LI>
• For main culture, prepare a fresh 24 deepwell plate with 1,9 mL YP media with the respective concentration of inducer (e.g.:0.5%, 1% and 2% galactose). For inoculation add 100 µL of the precultures. Incubate the main cultures for around 5 hours at 30 °C and 280 rpm. 
</LI><LI>
• After cultivation, harvest 1 mL of the cultures by centrifugation and wash them once in PBS. Again harvest them by centrifugation und resuspend them in PBS. Afterwards dilute the samples with PBS to a final OD of 0.5. 
</LI><LI>
• Then the samples are ready to get measured in the flow cytometer. </LI></UL></P><H4 id="CRISPR" class="test">Preparation of electro competent <I>E. coli </I> for CRISPR mediated integration via λ recombination</H4><P><UL class="ulclass"><LI>• Inoculate an overnight culture of a bacterial strain containing CRISPR and Lambda Red elements into <NOBR>3 ‑ 5 mL </NOBR> LB media containing a selection marker and incubate it at 32 °C if the plasmid is temperature sensitive (otherwise 37 °C is fine). 
</LI><LI> 
• Add 350 µL to 35 mL selective LB media in a 250 mL baffled Erlenmeyer flask. Incubate the culture at 32 °C at 180 rpm until the cells reach a OD<SUB>600</SUB> of 0.4 – 0.6. </LI><LI>
• Place the flask in a 42 °C waterbath and shake it for 15 min for induction of the Lambda Red recombination elements. Afterwards place the culture in an ice- water slurry  for 10 min to  cool the cells down. 
</LI><LI>
• Transfer the culture to a centrifuge tube and centrifuge it at 4600∙g for 7 min at 4°C and remove the supernatant. 
</LI><LI>
• Resuspend the pellet in 30 mL of ice-cold AD and harvest the cells again by centrifugation like above. Promptly decant the supernatant. 
</LI><LI>
• Resuspend the cells in 1 mL ice cold AD and transfer the suspension to a pre-chilled 1.5 mL microcentrifuge and centrifuge them at 10000∙g for 30 seconds at 4°C. Carefully aspirate the supernatant and suspend cells in 1 ml of ice-cold H2O, centrifuge again and aspirate
supernatant carefully. 
</LI><LI>  
• Resuspend the cells in 200 µL sterile AD with 10 % glycerol. Cells are ready for transformation or could be frozen at -80°C until transformation. 
</LI><LI>
• Protocol is adapted from Sharan 2009
Sharan, S. K., Thomason, L. C., Kuznetsov, S. G., &amp; Court, D. L. (2009). Recombineering: a homologous recombination-based method of genetic engineering. Nature Protocols, 4(2), 206–223. <A href=" https://doi.org/10.1038/nprot.2008.227 "> https://doi.org/10.1038/nprot.2008.227</A></LI></UL></P><H4 id="TransformationEcompetentEcoli" class="test">Transformation of electrocompetent <I> E. coli</I></H4><P>This protocol was used for CRISPR mediated integration via λ recombination</P><P><UL class="ulclass"><LI>• Transfer 50 µL of el. comp. cells (prepared according the protocol: <A href="#CRISPR" class="page-scroll"> Preparation of electro competent <I>E. coli </I> for CRISPR mediated integration via λ recombination </A>into a 2 mm electroporationcuvette. 
</LI><LI>
• Add 100 – 300 ng of DNA to the cells. 
</LI><LI>
• Perform electroporation in a BioRad MicroPulser<SUP>TM</SUP> using the program EC2. Immediately after electroporation add 1 mL LB media to the cells and induce the CRISPR elements by adding arabinose (final concentration 1%) 
</LI><LI>
• Regenerate the cells for 6 hours at 30 °C at 400 rpm 
</LI><LI>
• Dilute the transformation mix 1:100 und plate 100 µL on LB plates with the appropriate selection marker and incubate them 30 °C. </LI></UL></P><H4 id="RT" class="test"> RT activity assay </H4><P>This assay is performed in 96 well deepwell plates<NOBR>• Inoculate 300 µL LB with the respective selection marker. Incubate the plate at 37 °C and 300 rpm overnight.</NOBR>
• Make 1:20 dilutions of the precultures in PBS and inoculate 360 µL LB +arabinose (0.5 %) with 40 µL of the dilution. 
• Incubate the plate at 37°C, 300 rpm for 6 hours. 
• After incubation dilute the sample in PBS to reach around 2000 events per second in the flow cytometer and store them on ice until the measurement in the FC.</P><H4 id="Temperature" class="test">Temperature stability test of plasmids</H4><P><NOBR>This protocol was used for investigation of the curability of temperature sensitive plasmids at 37 °C</NOBR></P><P>• Inoculate a preculture in selective LB and incubate it overnight at 30°C and 180 rpm. 
• On the next day, harvest the cells by centrifugation and resuspend them in LB media. 
• Split the cell suspension and dilute them 1:100 and incubate one culture at 37°C and the other as negative control at 30 °C for 8 hours at 180 rpm.
• After incubation dilute the cultures 1:10<SUP>6</SUP> and plate 100 µL on selective LB plates and incubate them overnight at 30 °C. 
• For evaluation compare the numbers of colonies of both cultures. </P><H4 id="Lethality" class="test"> Lethality test of CRISPR-PPP </H4><P>This protocol was used to determine the efficiency of our CRISPR set up.</P><P>• Inoculate a preculture in selective LB and incubate it overnight at 30°C and 180 rpm. 
• On the next day, harvest 2 mL of culture by centrifugation und resuspend them in 2 mL LB  containing the right selection marker.
• Split the cell suspension and induce one of them by adding 1% arabinose. The other one remained uninduced as negative control. 
• Incubate both culture in a thermomixer at 30 °C and 400 rpm for 6 hours.
• After incubation dilute the induced culture 1:100 and the negative control 1:10<SUP>6</SUP> and plate <NOBR> 100 µL</NOBR> of each dilution on selective LB plates and incubate them overnight at 30°C.
 • For evaluation compare the numbers of colonies of induced and uninduced one. </P></DIV></DIV></SECTION></DIV></BODY></HTML>