Team:BOKU-Vienna/Part Collection
Menu
D.I.V.E.R.T.
Team
Team
Collaborations
Project
Theory
Experiments
Discussion
Notebook
Protocol
Modeling
Demonstrate
Parts
Overview
Basic Part
Part Collection
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Newsletters
Safety
Medal Criteria
Overview
Attributions
Interlab
Improve
Modeling
Contribution
Sponsors
Menu
Top
Overview
BB1
BB2
BB3
CPPP
Part Collection
Part Collection overview
Most plasmids used in our project were created via Golden Gate assembly, which is based on type IIs restriction enzymes. These cut DNA outside of their recognition site and create DNA overhangs that consist of arbitrary nucleotides called fusion sites (shown in Figure 1). As a result, scarless junctions can be created by designing compatible overhangs. The restriction and ligation event of the desired fragments result in the loss of the original recognition sites. Therefore, it is possible to perform the restriction – ligation reactions in a single pot.
Figure 1: Recognition site (B) and cleavage site (f) of BsaI (8)
A hierarchical cloning system was used which contains three levels of complexity, which are shown in Figure 2. Level 1 (called BB1) consists of the basic donor plasmids containing genetic building blocks like promotors, coding sequences and terminators. At Level 2 (called BB2) a promotor, a gene of interest and a terminator are combined to form a transcription unit. Level 3 (called BB3) is the level of highest complexity in which the combination of up to 7 different transcription units yields a multigene construct.
Figure 2: Different levels of construct assembly (P: promotor, CDS: coding sequence, T: terminator)
Figure 3: The relation of all of our plasmids is depicted here.
BB1
• BB1_01
Cas9_NLS_FS_23
X
• BB1_02
cycT_Fs_34
X
• BB1_03
J23101_FS_12
X
• BB1_04
pTDH3_FS_34
X
• BB1_05
B1001_FS_34
X
• BB1_06
J23105_FS_12
X
• BB1_07
eGFP_FS_23
X
• BB1_08
pTEF1_FS_12
X
• BB1_09
FLPe_FS_23
X
• BB1_10
repA101ts_FS_23
X
• BB1_11
MMLV_RT_FS_23
X
• BB1_12
pBAD_RBS_FS_12
X
• BB1_13
GFP+PBS1_FS_23
X
• BB1_14
pGal-syn-v1_FS_12
X
• BB1_15
J23101_FS_12
X
• BB1_16
Ura3sgRNA_FS_23
X
• BB1_17
BB1_17_Leu2sgRNA_FS_23
X
• BB1_18
AmpR_frameshift_FS_23
X
• BB1_19
beta_gam_FS_23
X
• BB1_20
GFP+AlaBS_FS_23
X
• BB1_21
GFP+ArgBS_FS_23
X
• BB1_22
GFP+GluBS_FS_23
X
• BB1_23
GFP+GlyBS_FS_23
X
• BB1_24
GFP+IleBS_FS_23
X
• BB1_25
GFP+LeuBS_FS_23
X
• BB1_26
pGAL_syn-v2_kozak_FS_12
X
• BB1_27
pBAD+RBS+AraC_FS_12
X
• BB1_30
pGal_syn_kozak_igem_std_FS_23
X
• BB1_31
GFP_Improve
X
• BB1_32
GFPuv_FS_23
X
BB2
• BB2_01
J23101_GFP_B1001_AB
X
• BB2_02
pTEF1_FLPe_cycterm_EF
X
• BB2_03
pSNR52_Hefe-Primer1_sup4T_CD
X
• BB2_04
pSNR52_Hefe-Primer2_sup4T_DE
X
• BB2_05
J23101_GFP+PBS1_B1001_AB
X
• BB2_06
J23105_repA101ts_B1001_DE
X
• BB2_07
pBAD_Cas9_B1001_BC
X
• BB2_08
pBAD_RT_FLPe_B1001_BC
X
• BB2_09
Ura3-H1_AB
X
• BB2_10
Ura3-H2_FG
X
• BB2_11
Leu2-H1_AB
X
• BB2_12
Leu2-H2_CD
X
• BB2_13
pGAL-synv1_RT_cycT_BC
X
• BB2_14
J23105_GFP_B1001_AB
X
• BB2_15
J23105_GFP+PBS1_B1001_AB
X
• BB2_16
pGal-syn-v1_GFP_cyc_AB
X
• BB2_17
ori_pSC101(Ts)
X
• BB2_18
pBAD_RT_B1001_BC
X
• BB2_19
J23105_Primer1+HDV_B1001_CD
X
• BB2_20
J23105_AmpR_frameshift_T1001_AB
X
• BB2_21
pBAD_beta_gam_BC
X
• BB2_23
pTEF1_URAsgRNA_cycT
X
• BB2_24
pTEF1_LEUsgRNA_cycT
X
• BB2_25
J23105_GFP+AlaBS_B1001_AB
X
• BB2_26
J23105_GFP+ArgBS_B1001_AB
X
• BB2_27
J23105_GFP+GluBS_B1001_AB
X
• BB2_28
J23105_GFP+GlyBS_B1001_AB
X
• BB2_29
J23105_GFP+IleBS_B1001_AB
X
• BB2_30
J23105_GFP+LeuBS_B1001_AB
X
• BB2_31
PC+AraC_B1001_AB
X
• BB2_32
PC+AraC_B1001_CD
X
• BB2_33
PC+AraC_B1001_DE
X
• BB2_34
J23105_FRT_GFP_B1001_AB
X
• BB2_35
J23105_RNDM_GFP_B1001_AB
X
• BB2_36
pBad_RT+bet+gam_B1001_BC
X
• BB2_37
J23105_GFP+FRT+PBS1_B1001_AB
X
• BB2_38
pGAL-syn-v2_RT_cycT_BC
X
• BB2_39
pGAL-syn-v2_GFP_cycT_AB
X
• BB2_40
pTDH3_GFP_cycT_AB
X
• BB2_41
pBAD+RBS+AraC_Cas9_B1001_BC
X
• BB2_42
J23105_RNDM_GFP_B1001_AB
X
• BB2_43
J23105_ATG_FRT_GFP_B1001_AB
X
• BB2_44
J23105_Primer2+HDV_B1001_CD
X
• BB2_45
J23105_GFP+PBS2_B1001_AB
X
• BB2_46
J23105_GFP+AlaBSv2_B1001_AB
X
• BB2_47
J23105_GFP+GluBSv2_B1001_AB
X
• BB2_48
J23105_GFP+GluBSv2_B1001_AB
X
• BB2_49
J23105_GFP+GlyBSv2_B1001_AB
X
• BB2_50
J23105_GFP+IleBSv2_B1001_AB
X
• BB2_51
AraC+pBAD_GFP+AlaBSv2_B1001_AB
X
• BB2_52
AraC+pBAD_GFP+ArgBSv2_B1001_AB
X
• BB2_53
AraC+pBAD_GFP+GluBSv2_B1001_AB
X
• BB2_54
AraC+pBAD_GFP+GlyBSv2_B1001_AB
X
• BB2_55
AraC+pBAD_GFP+IleBSv2_B1001_AB
X
• BB2_56
J32105_RT_B1001_BC
X
• BB2_57
AraC+pBAD_RT_B1001_BC
X
• BB2_58
AraC+pBAD_GFP+PBS2_B1001_AB
X
• BB2_59
AraC+pBAD+GFP_B1001_AB
X
BB3
• BB3_01
coli helper v1
X
• BB3_01 Speziale
coli helper v1 (low copy ori)
X
• BB3_02
coli D.I.V.E.R.T._empty
X
• BB3_03
yeast helper
X
• BB3_04
J23105_ampR_B1001
X
• BB3_05
yeast D.I.V.E.R.T._empty
X
• BB3_06
J23105_flipped-ampR-with-selfsplicing-Ribozyme_B1001
X
• BB3_07
J23105-GFP_pBAD-RT_ J23105-Primer1-Kass._AraC
X
• BB3_08
pGAL-syn_GFP_cycT
X
• BB3_09
coli D.I.V.E.R.T._empty (weak promoter)
X
• BB3_10
J23105-AmpR_frameshift_pBAD-beta_gam_araC
X
• BB3_11
J23105-GFP+PBS1_pBAD-RT_J23105- Primer1-Kass._AraC
X
• BB3_12
coli D.I.V.E.R.T._AmpR
X
• BB3_13
URA3sgRNA_2µ
X
• BB3_14
LEU2sgRNA_2µ
X
• BB3_15
yeast D.I.V.E.R.T._URA3
X
• BB3_16
coli helper +beta, gam
X
• BB3_17
J23105-GFP_pBAD-RT_AraC
X
• BB3_18
J23105-GFP+AlaBS_pBAD-RT_Ara_C
X
• BB3_19
J23105-GFP+ArgBS_pBAD-RT_Ara_C
X
• BB3_20
J23105-GFP+GluBS_pBAD-RT_Ara_C
X
• BB3_21
J23105-GFP+GlyBS_pBAD-RT_Ara_C
X
• BB3_22
J23105-GFP+IleBS_pBAD-RT_Ara_C
X
• BB3_23
J23105-GFP+LeuBS_pBAD-RT_Ara_C
X
• BB3_24
coli D.I.V.E.R.T._ampR (without FRT sequence)
X
• BB3_25
yeast D.I.V.E.R.T._URA3 (without FRT sequence)
X
• BB3_26
coli D.I.V.E.R.T._empty + helper construct + AraC
X
• BB3_27
coli D.I.V.E.R.T._ampR + helper construct + AraC
X
• BB3_28
coli helper without Flpe
X
• BB3_29
J23105-AmpR-frameshift-B1001
X
• BB3_30
coli D.I.V.E.R.T._GFP
X
• BB3_31
pGALv2-GFP-cyct_2µ
X
• BB3_32
pTDH3-GFP-cyct_2µ
X
• BB3_33
yeast helper without Flpe
X
• BB3_34
coli Primer1, Primer2 cassette
X
• BB3_35
coli helper v2 (weaker promoter for primers)
X
• BB3_35 Speziale
coli helper v2 (weaker promoter for primers) (low copy ori)
X
• BB3_36
UraH1_pGALsyn-v2-GFP-cycT_URA-H2
X
• BB3_37
UraH1_pGALsyn-v1-GFP-cycT_URA-H2
X
• BB3_38
UraH1_pTDH3-GFP-cycT_URA-H2
X
• BB3_39
coli helper v3 (pBAD for primers)
X
• BB3_39 Speziale
coli helper v3 (pBAD for primers) (low copy ori)
X
• BB3_40
coli helper v3 (pBAD for primers) + AraC
X
• BB3_41
yeast helper v2 (gal. induced primers)
X
• BB3_42
coli helper v3 with beta, gam (pBAD for primers)
X
• BB3_43
coli D.I.V.E.R.T._AmpR(weak)
X
• BB3_45
AraC+pBAD-GFP-AlaBSv2_J23105-RT
X
• BB3_46
AraC+pBAD-GFP-ArgBSv2_J23105-RT
X
• BB3_47
AraC+pBAD-GFP-GluBSv2_J23105-RT
X
• BB3_48
AraC+pBAD-GFP-GlyBSv2_J23105-RT
X
• BB3_49
AraC+pBAD-GFP-IleBSv2_J23105-RT
X
• BB3_55
AraC+pBAD-GFP-PBS2_J23105-RT
X
• BB3_56
AraC+pBAD-GFP_J23105-RT
X
• BB3_57
AraC+pBAD-GFP_J23105-RT_Pr2-Kass.
X
• BB3_58
AraC+pBAD-GFP-PBS2_J23105-RT_Pr2-Kass.
X
CRISPR-PPP
CRISPR-PPP stands for CRISPR plug and play plasmid. Below only the empty plasmids without a target are listed. The targets were inserted using Golden Gate assembly.
• CPPP_01
X
• CPPP_04
+AraC
X
• CPPP_07
AraC + pBAD  repressible + sgRNA with pBAD and with RNAi with J23109
X
• CPPP_10
AraC + pBAD  repressible + sgRNA with pBAD and with RNAi with J23109 + λRed (from pSIM 5)
X
• CPPP_13
sgRNA with RNAi with J23103
X
• CPPP_16
sgRNA with RNAi with J23103 +λRed (from pSIM 9)
X
• CPPP_19
sgRNA inducible with RNAi with J23103 without HDV Ribozyme
X
• CPPP_22
sgRNA inducible with RNAi with J23103 without HDV Ribozyme +λRed (from pSIM 9)
X
• CPPP_25
sgRNA inducible without RNAi  without HDV Ribozyme
X
• CPPP_28
sgRNA inducible without RNAi without HDV Ribozyme +λRed (from pSIM 9)
X
• CPPP_31
sgRNA inducible without RNAi with HDV Ribozyme
X
• CPPP_34
sgRNA inducible without RNAi with HDV Ribozyme +λRed (from pSIM 9)
X
• CPPP_37
sgRNA constitutive Cas9 under control of pBAD with AraC (repressible)
X
