Team:BOKU-Vienna/Demonstrate
Menu
D.I.V.E.R.T.
Team
Team
Collaborations
Project
Theory
Experiments
Discussion
Notebook
Protocol
Modeling
Demonstrate
Parts
Overview
Basic Part
Part Collection
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Newsletters
Safety
Medal Criteria
Overview
Attributions
Interlab
Improve
Modeling
Contribution
Sponsors
Menu
Top
Demonstrate
Demonstrate
Demonstrate
Although not being lucky with our “shot in the dark” implementation of D.I.V.E.R.T. it still can be argued that our experiments laid a solid foundation for future work on (semi-)synthetic retroelement-based directed evolution. Our RT priming assay delivered strong evidence that triggering reverse transcription in vivo in E. coli using the retroviral RT from Moloney Murine Leukemia Virus (MMLV) is far easier than expected. If the results hold true in further investigations, this means that it will be possible to solely use native tRNAs to prime reverse transcription rendering the generation of exogenous primers unnecessary, hence eliminating one of the steps that had caused the most trouble in the concept we had had designed.
Similar things can be said about the discovery that the FRT sequence is a strong terminator, but only in one direction. Any possible implementation of D.I.V.E.R.T. that involves Flp/FRT-mediated recombination relies on RNA polymerase as well as reverse transcriptase being able to read through both FRT sites. Since the direction of extended FRT sites is irrelevant for their function in RMCE (as long as the sites that shall react with each other are identical)1,2 this information allows to freely choose their orientation trying to minimize the aborting effect.
If any researchers would build a nicely working version of D.I.V.E.R.T. in the future using these results, they still would have to prove and quantify its activity. Ideally they would use a “retrotransposition assay” delivering direct data on the frequency of completion of the D.I.V.E.R.T. cycle like Crook et al. did in their work3. Such an assay, however, relies on an intron disrupting a selectable marker in reverse orientation. So, to be able to do this in E. coli, they would have to find an adequate self-splicing ribozyme as well as a suitable selection marker featuring sequences fulfilling the necessary prerequisites, engineer the sequences of both and show that the generated construct efficiently splices itself at exactly the planned position to prevent any frameshifts. Well, actually they don’t since we have already done all of this for them.
After having optimized their system, to finally evolve a gene our future colleagues would need to integrate the D.I.V.E.R.T. cassette into the E. coli genome in order to make sure that it is present in only a single copy. Again, they can resort to our project since we have created an easy to use plug and play single plasmid CRISPR/Cas9 solution to select positive recombinants when integrating DNA using homologous recombination in E. coli. The next generation of our CRISPR plasmids would have made their future lives even easier as those would also have offered all lambda Red functions in addition to providing CRISPR for selection. Some of those would probably even allow cultivation of cells without glucose repression due to our trick involving background sgRNA levels being quenched by RNA interference. Unfortunately, we have not had enough time to complete all the plasmids of our library and troubleshoot the two we had already generated and which showed sufficient lethality but apparently did not promote recombination.
[1]: Turan S, Kuehle J, Schambach A, Baum C, Bode J. Multiplexing RMCE: Versatile Extensions of the Flp-Recombinase-Mediated Cassette-Exchange Technology. J Mol Biol. 2010;402(1):52-69. doi:10.1016/j.jmb.2010.07.015.
[2]: Turan S, Bode J. Site-specific recombinases: from tag-and-target- to tag-and-exchange-based genomic modifications. FASEB J. 2011;25(12):4088-4107. doi:10.1096/fj.11-186940.
[3]: Crook N, Abatemarco J, Sun J, Wagner JM, Schmitz A, Alper HS. In vivo continuous evolution of genes and pathways in yeast. Nat Commun. 2016;7:13051. doi:10.1038/ncomms13051.
