<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_BOKU-Vienna_HP_Gold_Integrated skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:BOKU-Vienna/HP/Gold Integrated</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                Menu </BUTTON><A class="navbar-brand page-scroll" href="https://2017.igem.org/Team:BOKU-Vienna">
                D.I.V.E.R.T.
            </A></DIV><DIV class="collapse navbar-collapse navbar-right navbar-main-collapse"><UL class="nav navbar-nav"><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown2" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Team
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Team" class="dropdown-a">Team</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Collaborations" class="dropdown-a">Collaborations</A></LI></UL></DIV><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown3" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Project
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Description" class="dropdown-a">Theory</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Experiments" class="dropdown-a">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Discussion" class="dropdown-a">Discussion</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Notebook" class="dropdown-a">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Protocol" class="dropdown-a">Protocol</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Model" class="dropdown-a">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Demonstrate" class="dropdown-a">Demonstrate</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown4" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Parts
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Parts" class="dropdown-a">Overview</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Basic_Part" class="dropdown-a">Basic Part</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Part_Collection" class="dropdown-a">Part Collection</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Human Practices
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/HP/Silver" class="dropdown-a">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/HP/Gold_Integrated" class="dropdown-a">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Engagement" class="dropdown-a">Public Engagement</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/HP/Newsletters" class="dropdown-a">Newsletters</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A onclick="window.location.href='https://2017.igem.org/Team:BOKU-Vienna/Safety'" class="dropdown-toggle" type="button" id="dropdown1" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Safety
                      </A></DIV></LI><LI class="dropdown-li"><DIV class="dropdown"><A class="dropdown-toggle" type="button" id="dropdown6" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Medal Criteria
                      </A></DIV></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/MedalCriteria" class="dropdown-a">Overview</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Attributions" class="dropdown-a">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/InterLab" class="dropdown-a">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Improve" class="dropdown-a">Improve</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Model" class="dropdown-a">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:BOKU-Vienna/Contribution" class="dropdown-a">Contribution</A></LI></DIV><LI class="dropdown-li"><DIV class="dropdown"><A onclick="window.location.href='https://2017.igem.org/Team:BOKU-Vienna/Sponsors'" class="dropdown-toggle" type="button" id="dropdown7" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
                        Sponsors
                      </A></DIV></LI></DIV></DIV><DIV id="page-top" data-spy="scroll" data-target=".navbar-fixed-top"><NAV class="navbar scrollnav-custom navbar-fixed-bottom" role="navigation" id="scrollnav"><DIV class="container"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-main-collapse">
                    Menu </BUTTON></DIV><DIV class="collapse navbar-collapse navbar-right navbar-main-collapse"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#">Top</A></LI><LI><A class="page-scroll" href="#Integrated">Integrated &amp; Gold</A></LI></UL></DIV></DIV></NAV><HEADER class="myintro"><DIV class="myintro-body"><DIV class="container"><DIV class="row"><DIV class="col-md-8 col-md-offset-2"><H1 class="brand-heading">Gold Medal &amp; Integrated Human Practices</H1></DIV></DIV></DIV></DIV></HEADER><SECTION id="Integrated" class="container content-section text-center"><DIV class="container text-center content-section"><DIV class="col-lg-8 col-lg-offset-2"><H2>Integrated Human Practices.</H2><P>One very simple way to ensure that your scientific work is applied in a responsible way, is to first find out how the public responds to your specific topic and analyze why they do so. With that knowledge in mind, one can then start designing his or her work in a way that, if it may change the world, it will be in a truly beneficial way. We went ahead and gathered information on the attitude of the European population towards genetically modified organisms (GMOs) and also on how it may be changed to a more positive one.</P><P>For example, Prof. Dr. Wolfgang Stroebe from the University of Groningen in Netherlands, held a <A href="http://www.ethikrat.org/dateien/pdf/tagung-14-02-2017-simultanmitschrift.pdf" target="_blank">speech</A> on the public’s attitude towards GMOs. He referred us to the work of <A href="https://publishup.uni-potsdam.de/opus4-ubp/frontdoor/deliver/index/docId/543/file/scholderer.pdf" target="_blank">Dr. Joachim Scholderer.</A></P><P>The view of the European population on the use of GMOs is very ambivalent. There is distrust of information supplied by big companies and a distinct desire to preserve the natural state of the environment among them. This approach is very sensible but in some cases can stand in the way of a reasonable discourse on applications of GMOs. We believe that since most companies use GMOs to slim their processes and increase their market value, the European population just does not see any utility for themselves. Therefore, to make our project useful not only in the lab, but as well to the society and the environment, it has been a major goal to design our work to have a positive effect for every person.</P><P>D.I.V.E.R.T. is a method to incorporate random mutations into a target gene without mutating the rest of the genome. This way, very big libraries of all possible random mutations can be assembled in little time. These libraries can then be screened for improved properties to pick out the best mutant for specific applications. If selection pressure is applied to the cells while D.I.V.E.R.T. is active, evolution should be sped up significantly and beneficial mutations are accumulated tuning the gene to enhance the desired property.</P><P>To showcase the power of this application in a way the public would engage with we tried to come up with a strategy to improve the activity of PETase to make the enzyme efficient enough for operative PET-biodegradation. The gene encoding PETase would be mutated randomly via the D.I.V.E.R.T. method and then screened for enzyme activity on a PET-film as substrate. Then, the best mutants would be chosen for another round of mutation. This cycle could be repeated until no more significant improvement would be detected. Since D.I.V.E.R.T. is intended to work rather fast, the screening method for finding the best PETase mutant should be very time-efficient as well. This imposes several additional problems, but we managed to gather enough information to come up with solutions for all the foreseeable challenges.</P><H3><I>Step 1 -</I> D.I.V.E.R.T.</H3><P>For the purpose of using D.I.V.E.R.T. it may be beneficial to employ <I>Escherichia coli</I> as chassis for the improvement of PETase. In order to apply selection pressure to the <I> E. coli </I> growing on the PET-film, they need to be able to utilize the product of PETase as carbon source. This way, in an otherwise carbon-deprived culture medium, the mutants expressing the best PETase should also be the fastest growers. This creates a situation where the fittest species will also make up the biggest portion of the population. However, PETase is secreted into the medium in order to degrade the plastic film ultimately causing less efficient mutants to benefit from more efficient mutants in their proximity drastically decreasing up screening stringency. However, as long as some form of selection pressure is upheld, the D.I.V.E.R.T. method should yield improved mutants.</P><P>PETase hydrolyzes PET into its monomer MHET (mono-(2-hydroxyethyl) terephthalate). MHET is then degraded into TPA (terephthalic acid) and ethylene glycol by the enzyme MHETase. <I> E. coli </I> is most likely not able to utilize MHET as carbon source, however, some mutant strains of <I> E. coli </I> are able to to grow on media containing ethylene glycol as their sole carbon source.<SUP>1</SUP></P><P>The <I> E. coli </I> strain may have to be modified further, to be able to effectively grow on and degrade the PET-film. <I> E. coli </I> can be made “sticky” to hydrophobic surfaces by having them display an adhesive catecholamine moiety on their cell surface. These proteins are inspired by mussel adhesive proteins (with mussels being able to stick to almost any surface in an aquatic environment) and have been developed by J. P. Park et al. in 2014<SUP>2</SUP></P>. One disadvantage in the application of these proteins is that they require the addition of tyrosinase to the media, in order to be adhesive.

<P>Accordingly, three genes have to be inserted into the <I> E. coli </I> chassis additionally to the helper construct : PETase, MHETase (both in D.I.V.E.R.T. cassettes) and the adhesive protein. Such a recombinant <I> E. coli </I> strain should be capable of growing on and degrading PET films.</P><H3><I>Step 2 -</I> Cell separation.</H3><P> In order to screen for the most efficient mutant, mutant cells need to be separated first. Separation can theoretically be done using a flow cytometric cell sorter. In Flow Cytometry, cells in suspension are passed single file across a laser interrogation point. In the case of our application, light scattering signals from the laser could be measured, which correlates with cell morphology, to identify single cells and divert them from the fluid stream.<SUP>3</SUP> This way, single cells can be sorted and assembled into microwell-plates with up to 100.000 wells.<SUP>4</SUP> These wells can then theoretically be directly used for the screening process. One big downside of simply singling out cells into microwell plates is that they cannot incorporate a whole library created with D.I.V.E.R.T. In other applications it may be possible to classify the cells during the cell sorting process, but this is hardly possible with catalytic enzymes.</P><H3><I>Step 3 -</I> Cell screening.</H3><P>In 2016 M. T. Zumstein<SUP>5</SUP> reported their high-throughput screening method for enzymatic hydrolysis of biodegradable polyesters utilizing the co-hydrolysis of a fluorogenic ester for real time activity measurement. This approach also seems suitable for screening of PETase variants in concert with D.I.V.E.R.T. The authors embedded a fluorogenic ester in thin polyester films. As the hydrolytic enzymes degrade the polyester matrix, they likewise hydrolyse the embedded fluorogenic compound releasing the fluorophore into the reaction solution. This means that, as the enzymes degrade the polyester-film, a proportional increase in fluorescence can be measured. What makes this method ideal, is that it can be conducted in microtiter plates, by coating the well with the modified polyester film.</P><P>The procedure, as it is established by Zumstein et al.<SUP>6</SUP>, could potentially be adapted to be fit for PETase screening using D.I.V.E.R.T. Microtiter plates coated with PET-film could be inoculated with single cells from aforementioned pre-cultures before determining the hydrolysis activity of PETase using a plate reader. As negative control and blank for quantification, <I> E. coli </I> carrying only adhesive mussel proteins, but no PETase, could be inoculated in parallel.
Finally, the most active clone can be picked from the microtiter plates and be subjected to the next round of mutation.</P><P>Mutation of a target gene could be conducted in a simple shaker-culture, induced with arabinose. According to S. Yoshida et al. (2016), <I>I. sakaiensis</I>, the organism originally carrying our gene of interest PETase,
 can be grown on PET film immersed in YSV medium and shaken at 300 strokes/ min at 30 °C.<SUP>7</SUP></P><A href="https://www.ncbi.nlm.nih.gov/pubmed/6336729" target="_blank"><P style="font-size:70%;">
 [1]: A. Boronat, E. Caballero, J. Aguilar. Experimental evolution of a metabolic pathway for ethylene glycol utilization by Escherichia coli. Journal of Bacteriology, 153(1): 134-9 (1983)
</P></A><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3911018/" target="_blank"><P style="font-size:70%;">
 [2]: J. P. Park, M. J. Choi, S. H. Kim, S. H. Lee, H. Lee. Preparation of Sticky Escherichia coli through Surface Display of an Adhesive Catecholamine Moiety, Applied Environmental Microbiology, 80(1): 43-53 (2014)
</P></A><A href="http://www.springer.com/de/book/9783540752622" target="_blank"><P style="font-size:70%;">
 [3]: A. Kumar, I. Y. Galaev, B. Mattiasson, Cell Separation: Fundamentals, Analytical and Preparative Methods. Advances in Biochemical Engineering/ Biotechnology 106, Springer-Verlag Berlin Heidelberg (2007) 
</P></A><A href="http://www.springer.com/de/book/9781617795664" target="_blank"><P style="font-size:70%;">
 [4]: S. Lindström, H. Andersson-Svahn. Single-Cell Analysis: Methods and Protocols. Methods in Molecular Biology, 853, Humana Press (2012)
</P></A><A href="http://pubs.acs.org/doi/abs/10.1021/acs.est.6b06060" target="_blank"><P style="font-size:70%;">
 [5]: M. T. Zumstein, H.P. E. Kohler, K. McNeill. M. Sander. High-Throughput Analysis of Enzymatic Hydrolysis of Biodegradeable Polyesters by Monitoring Cohydrolysis of a Polyester-Embedded Fluorogenic Probe. Environmental Science and Technology, 51 (8), 4358-4367 (2017)
</P></A><A href="http://pubs.acs.org/doi/abs/10.1021/acs.est.5b04103" target="_blank"><P style="font-size:70%;">
 [6]: M. T. Zumstein, H.P. E. Kohler, K. McNeill, M. Sander. Enzymatic Hydrolysis of Polyester Thin Films: Real-Time Analysis of Film Mass Changes and Dissipation Dynamics. Environmental Science and Technology, 50, 197-206 (2016)
</P></A><A href="http://science.sciencemag.org/content/sci/suppl/2016/03/09/351.6278.1196.DC1/aad6359-Yoshida-SM.corrected.pdf" target="_blank"><P style="font-size:70%;">
 [7]: S. Yoshida, K. Hiraga, T. Takehana, I. Taniguchi, H. Yamaji, Y. Maeda, K. Toyohara, K. Miyamoto, Y. Kimura, K. Oda. Supplementary Materials for A bacterium that degrades and assimilates poly(ethylene terephtalate). Science, 351, 1196 (2016)
</P></A></DIV></DIV></SECTION></DIV></BODY></HTML>