Team:BOKU-Vienna/Description
Menu
D.I.V.E.R.T.
Team
Team
Collaborations
Project
Theory
Experiments
Discussion
Notebook
Protocol
Modeling
Demonstrate
Parts
Overview
Basic Part
Part Collection
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Newsletters
Safety
Medal Criteria
Overview
Attributions
Interlab
Improve
Modeling
Contribution
Sponsors
Menu
Top
Directed Evolution
Concept
Theory
Directed Evolution – a (very) short overview
After decades of continuous methodological advancement and increasing understanding of biomolecule structure and functionality directed evolution nevertheless is still the most powerful method in protein or aptamer engineering. Classic in vitro strategies, however, require substantial effort in terms of lab work and time investment to perform several consecutive rounds of evolution1,2. In order to automate the laborious process scientists have tried to devise systems that traverse the four steps of Darwinian evolution (mutation, expression, selection, replication) in a continuous cycle in vivo (reviewed in 3). In the earliest approaches this was achieved by simply cloning the gene of interest into mutator E. coli strains4 showing reduced DNA replication fidelity or into E. coli strains carrying inducible mutator plasmids5. While these settings proved to be useful for the generation of complex multifactorial phenotypes like organic solvent tolerance5 they cannot provide the regional selectivity that is desired in single-gene protein evolution as globally enhanced mutagenesis leads to slow growth and reduced transformation efficiency6 in addition to obscured phenotypic expression due to unwanted off-target mutations7,8.
Therefore, the ideal system for in vivo directed evolution would avoid those side effects by subjecting the host organism to locally confined hypermutation. Such an arrangement would allow the researcher to rapidly mutate and evolve a defined single sequence or gene while leaving the rest of the genome unchanged. Several strategies to locally constrain enhanced mutation rates have been devised so far; including plasmids harboring regions with low replication fidelity9, elaborate phage-assisted systems confining the accumulation of mutations to the phage genome while keeping the overall mutational load in the cell population in a steady state10 or retrotransposon-based methods11. Although definitely representing a large step towards the right direction, still none of the designs mentioned allow the continuous mutation of a single copy of a single gene in vivo. With D.I.V.E.R.T. we want to build a system that does.
The D.I.V.E.R.T concept
In their work published in 2016 Crook et al. probably generated the very first retrotransposon-based system for in vivo directed evolution by inserting a gene of interest into a truncated version of the native Ty1 retrotransposon in S. cerevisiae (Figure 1)11. Thus, the GOI is subjected to the retrotransposon life cycle and continuously mutated due to the error-prone nature of Ty1 reverse transcriptase (low fidelity is inherent to most reverse transcriptases12). 
Figure 1: Scheme of the design used by Crook et al. (image taken from their publication 11, which is open access under the CC BY 4.0 license).
Although having delivered impressive results such as a mutation rate of 0.15 kb-1­ the concept still holds room for improvement as argued by Zheng et al. in their review of targeted mutagenesis13.  For example, Ty1 as a mobile genetic element can reintegrate anywhere in the genome, preferred upstream of genes transcribed by RNA polymerase III14 possibly reducing selection efficiency due to the presence of multiple copies of the heterologous gene in a single cell. Additionally, in this setting the reverse transcriptase likewise is continuously mutated increasing the risk of inactivation as time carries on.
Drawing inspiration from this retroelement-based approach and having its potential weaknesses in mind we started thinking about the benefits of a more universal hypermutation strategy relying on reverse transcription of the gene of interest and reintegration of the generated cDNA such as:
Site-specific reintegration would make sure to replace the original gene variant maintaining single-copy status for optimal selection behavior.
Expression of the required reverse transcriptase in trans rather than within the synthetic retroelement eliminates the risk of inactivating the enzyme by mutation.
In such a system only the GOI would be mutated and remain being present in just a single copy. The overall scheme of our D.I.V.E.R.T. (directed in vivo evolution via reverse transcription) concept is depicted in Figure 2.
Figure 2: General scheme of the D.I.V.E.R.T. cycle
Theoretical considerations
To sum up: we wanted to build a fully synthetic retrotransposon-like genetic element that would allow our gene of interest to continuously undergo the retrotransposon life cycle accumulating mutations over time. The main events that need to be functionally implemented for such a system to work would be the in vivo reverse transcription carried out by a heterologous reverse transcriptase as well as site-specific recombination. For both processes several options are available. The reasoning that goes into our choices as well as some other thoughts are briefly explained below.
Host range
Not being reliant on host-specific factors (like the Ty1 retrotransposon in yeast) but only on heterologous components (RT and recombinase) D.I.V.E.R.T. – at least in general – should be applicable in a broad range of organisms as we wanted to show by performing our proof of concept experiment in yeast as well as in E. coli. Some minor adjustments in regard to the host, however, need to be made. For example, the relevant proteins have to be tagged with a NLS in eukaryotic systems and ribosome vs. reverse transcriptase interactions might play a role in prokaryotes. More details can be found in the D.I.V.E.R.T. experiment section .
RT choice and priming conditions
In the early phase of project planning we spent quite some time on gathering relevant information about a variety of reverse transcriptases including processivity, mutation rate, mutational spectra and RNase H activity. Finally, we chose to use Moloney murine leukemia virus (MMLV) RT mainly due to it being the best characterized monomeric15 reverse transcriptase while the active forms of many other RTs (e.g. HIV16 and ASLV17 RTs) are heterodimers. Picking a monomeric enzyme allowed us to save some IDT DNA synthesis credits; a wise decision, since we managed to use up all the credits for gBlocks, primers and DNA oligos during lab work in the summer months. As an additional requirement MMLV RT also shows RNase H activity which might help to synthesize double-stranded cDNA by degrading the RNA template after the first cDNA strand has been generated and we knew that active enzyme could be expressed in E. coli18.
In its natural context MMLV RT uses a mouse tRNAPro for priming19,20. It has been shown, though, that MMLV RT shows not very stringent preferences and that MMLV can replicate using different tRNAs as long as the PBS is complementary to the 3’ end of the tRNA21. From in vitro studies (and cDNA synthesis kits for RT-PCRs) we furthermore know that MMLV RT can also initiate replication using DNA or RNA oligonucleotides with the general sequence of efficiency being DNA-oligo, tRNA, RNA-oligo22. Unfortunately, finding data on in vivo priming conditions for any heterologous reverse transcriptase in E. coli is difficult as apparently only one study has performed reverse transcription in E. coli so far23. In this work that focused on the generation of ssDNA for the creation of DNA nanostructures in vivo Elbaz et al. designed the 3’ end of their mRNA in a way so that it formed a distinct structure that on one hand acted as an transcription terminator while on the other hand it promoted efficient priming of reverse transcription by dimeric HIV RT.
So, after an extensive literature research, those were the priming conditions we found to be worthwhile to consider:
• RNA oligos transcribed in trans featuring a defined 3’ end
• DNA oligos generated using a native retroelement (e.g. retrons in E. coli; used in a similar fashion in 24)
• The tRNA corresponding to the heterologous RT (in case of MMLV tRNAPro from mouse; transcribed in trans with a defined 3’ end)
• A tRNA of the host organism (is already present in the cell, no need for worrying about primer production)
• Self-priming using a simple hairpin or a more sophisticated structure at the mRNA 3’ end like in 23.
For ease of implementation and broad host generality we opted for using RNA oligos in our “lucky shot” D.I.V.E.R.T. experiment . Nevertheless, being able to prime with a native tRNA would be most convenient. Hence, in our priming condition assay we tested 5 of the most abundant tRNAs in E. coli for initiating reverse transcription with MMLV RT.
Reintegration of the generated cDNA
Once the mRNA has been reverse transcribed and potentially mutated it has to replace the original copy of the gene of interest. Again, different methods are available:
• Site-specific recombinase-mediated recombination (analogous to recombinase-mediated cassette exchange)25,26
• Homologous recombination
For our D.I.V.E.R.T. experiment we decided to employ the FLP/FRT site-specific recombination system as described by Turan et al.25. However, extended FRT sites needed for efficient recombination are palindromic and form stable hairpins possibly acting as transcription (or reverse transcription) terminators. To evaluate this potential problem we determined the bidirectional termination efficiency of extended FRT sites using the classic terminator strength assay .
Does it work? Find a rigorous proof
In the early phase of development the mutation rates achieved might be very low as the full D.I.V.E.R.T. cycle will rarely be completed. Hence, using mutation frequency as an indicator or test on whether the system works might be problematic. Crook et al.11 solved this problem by using a very elegant and rigorous proof of concept assay (Figure 1, b) that was inspired by a test on recombination frequency of the Ty1 element27. Basically, they chose a selectable marker as cargo (i.e. GOI) that was inserted into the retrotransposon in reverse orientation. The GOI was disrupted by an intron which again was reversely oriented with respect to the cargo (i.e. oriented in the retrotransposon’s sense direction). Hence, upon transcription of the cargo the mRNA would contain the reverse complement of the intron which would not be spliced leading to nonfunctional protein. Upon transcription of the retroelement, however, the intron would end up in the right orientation and going to be spliced. After reverse transcription and reintegration the intron would not be present in the new version of the retroelement leading to functional protein and selectable cells.
For our D.I.V.E.R.T. experiments we used a similar concept, details can be found in the experiments section .
Single gene copy for optimal selection – we need to integrate!
As mentioned before, an efficient selection process relies on only one copy of the GOI being present in every cell. Hence, the D.I.V.E.R.T. construct needs to be genomically integrated into the host genome. Homologous recombination in E. coli (usually conferred by the phage-derived lambda Red system 28–30) is an inefficient process making the application of (mostly antibiotic) selection markers necessary (efficiencies lie between 10-3 and 10-4 30). Living in the age of CRISPR/Cas9 we planned on utilizing this technique by selecting recombinants as was done in a number of recent papers31–33. Unfortunately, only the Cas9 gene was in stock in the iGEM registry at the beginning of our planning phase and no ready-to-use Cas9 or sgRNA plasmids were available for us via other non-commercial sources. Thus, as a side project we decided to construct a plug and play CRISPR/Cas9 system carrying Cas9 and an sgRNA cassette on a single plasmid in which the target sequence of choice can be inserted in a single Golden Gate reaction.
[1]: Yuan L, Kurek I, English J, Keenan R. Laboratory-directed protein evolution. Microbiol Mol Biol Rev. 2005;69(3):373-392. doi:10.1128/MMBR.69.3.373-392.2005.
[2]: Voigt CA, Kauffman S, Wang ZG. Rational evolutionary design: the theory of in vitro protein evolution. Adv Protein Chem. 2000;55:79-160. doi:10.1016/S0065-3233(01)55003-2.
[3]: Badran AH, Liu DR. In Vivo Continuous Directed Evolution. Curr Opin Chem Biol. 2015;0:1-10. doi:10.1016/j.cbpa.2014.09.040.
[4]: Stefan A, Radeghieri A, Gonzalez Vara y Rodriguez A, Hochkoeppler A. Directed evolution of β-galactosidase from Escherichia coli by mutator strains defective in the 3′→5′ exonuclease activity of DNA polymerase III. FEBS Lett. 2001;493(2-3):139-143. doi:10.1016/S0014-5793(01)02293-1.
[5]: Selifonova O, Valle F, Schellenberger V. Rapid Evolution of Novel Traits in Microorganisms Rapid Evolution of Novel Traits in Microorganisms. Appl Environ Microbiol. 2001;67(8):3645-3649. doi:10.1128/AEM.67.8.3645.
[6]: Greener A, Callahan M, Jerpseth B. An efficient random mutagenesis technique using an E. coli mutator strain. Mol Biotechnol. 1997;7:189-195. doi:10.1007/BF02761755.
[7]: Long-McGie J, Liu AD, Schellenberger V. Rapid in vivo evolution of a β-lactamase using phagemids. Biotechnol Bioeng. 2000;68(1):121-125. doi:10.1002/(SICI)1097-0290(20000405)68:1<121::AID-BIT15>3.0.CO;2-X.
[8]: Negri MC, Lipsitch M, Blázquez J, Levin BR, Baquero F. Concentration-dependent selection of small phenotypic differences in TEM beta-lactamase-mediated antibiotic resistance. Antimicrob Agents Chemother. 2000;44(9):2485-2491. doi:10.1128/AAC.44.9.2485-2491.2000.
[9]: Fabret C, Poncet S, Danielsen S, Borchert T V, Ehrlich SD, Jannière L. Efficient gene targeted random mutagenesis in genetically stable Escherichia coli strains. 2000;28(21):8-12.
[10]: Esvelt KM, Carlson JC, Liu DR. A system for the continuous directed evolution of biomolecules. Nature. 2011;472(7344):499-503. doi:10.1038/nature09929.
[11]: Crook N, Abatemarco J, Sun J, Wagner JM, Schmitz A, Alper HS. In vivo continuous evolution of genes and pathways in yeast. Nat Commun. 2016;7:13051. doi:10.1038/ncomms13051.
[12]: Svarovskaia SE, Cheslock SR, Zhang W-H, Hu W-S, Pathak VK. RETROVIRAL MUTATION RATES AND REVERSE TRANSCRIPTASE FIDELITY. Front Biosci. 2003;1:117-134.
[13]: Zheng X, Xing X-H, Zhang C. Targeted mutagenesis: A sniper-like diversity generator in microbial engineering. Synth Syst Biotechnol. 2017;2(2):75-86. doi:10.1016/j.synbio.2017.07.001.
[14]: Devine SE, Boeke JD. Integration of the yeast retrotransposon Ty1 is targeted to regions upstream of genes transcribed by RNA polymerase III. Genes Dev. 1996;10(5):620-633. doi:10.1101/gad.10.5.620.
[15]: Roth MJ, Tanese N, Goff SP. Purification and characterization of murine retroviral reverse transcriptase expressed in Escherichia coli. J Biol Chem. 1985;260(16):9326-9335. doi:10.1111/j.1462-5822.2011.01744.x.
[16]: Fulvia Di Veronese, Terry D. Copeland, Anthony L. DeVico, Rukhsana Rahman, Stephen Oroszlan RCG and MGS. Characterization of Highly Immunogenic p66/p51 as the Reverse Transcriptase of HTLV- III/LAV. Science (80- ). 1986;231(4743):1289-1291.
[17]: Kacian DL, Watson KF, Spiegelman S. PURIFICATION OF THE DNA POLYMERASE OF AVIAN MYELOBLASTOSIS VIRUS. Biochim Biophys ACTA BBA. 1971;246:365-383.
[18]: Das D, Georgiadis MM. The crystal structure of the monomeric reverse transcriptase from moloney murine leukemia virus. Structure. 2004;12(5):819-829. doi:10.1016/j.str.2004.02.032.
[19]: Peters G, Harada F, Dahlberg JE, Panet  a, Haseltine W a, Baltimore D. Low-molecular-weight RNAs of Moloney murine leukemia virus: identification of the primer for RNA-directed DNA synthesis. J Virol. 1977;21(3):1031-1041.
[20]: Harada F, Peters GG, Dahlberg JE. The primer tRNA for Moloney murine leukemia virus DNA synthesis. Nucleotide sequence and aminoacylation of tRNAPro. J Biol Chem. 1979;254(21):10979-10985.
[21]: Lund AH, Duch M, Lovmand J, Jorgensen P, Pedersen FS. Mutated primer binding sites interacting with different tRNAs allow efficient murine leukemia virus replication. J Virol. 1993;67(12):7125-7130.
[22]: Isel C, Ehresmann C, Keith G, Ehresmann B, Marquet R. Two step synthesis of (-) strong-stop DNA by avian and murine reverse transcriptases in vitro. Nucleic Acids Res. 1997;25(3):545-552. doi:10.1093/nar/25.3.545.
[23]: Elbaz J, Yin P, Voigt CA. Genetic encoding of DNA nanostructures and their self-assembly in living bacteria. Nat Commun. 2016;7:11179. doi:10.1038/ncomms11179.
[24]: Farzadfard F, Lu TK. Genomically encoded analog memory with precise in vivo DNA writing in living cell populations. Science (80- ). 2014;346(6211):1256272-1256272. doi:10.1126/science.1256272.
[25]: Turan S, Kuehle J, Schambach A, Baum C, Bode J. Multiplexing RMCE: Versatile Extensions of the Flp-Recombinase-Mediated Cassette-Exchange Technology. J Mol Biol. 2010;402(1):52-69. doi:10.1016/j.jmb.2010.07.015.
[26]: Turan S, Bode J. Site-specific recombinases: from tag-and-target- to tag-and-exchange-based genomic modifications. FASEB J. 2011;25(12):4088-4107. doi:10.1096/fj.11-186940.
[27]: Curcio MJ, Garfinkel DJ. Single-step selection for Ty1 element retrotransposition. Proc Natl Acad Sci U S A. 1991;88(3):936-940. doi:10.1073/pnas.88.3.936.
[28]: Copeland NG, Jenkins N a, Court DL. Recombineering: a powerful new tool for mouse functional genomics. Nat Rev Genet. 2001;2(10):769-779. doi:10.1038/35093556.
[29]: Court DL, Sawitzke JA, Thomason LC. Genetic Engineering Using Homologous Recombination. Annu Rev Genet. 2002;36(1):361-388. doi:10.1146/annurev.genet.36.061102.093104.
[30]: Datta S, Costantino N, Court DL. A set of recombineering plasmids for gram-negative bacteria. Gene. 2006;379(1-2):109-115. doi:10.1016/j.gene.2006.04.018.
[31]: Bassalo MC, Garst AD, Halweg-Edwards AL, et al. Rapid and Efficient One-Step Metabolic Pathway Integration in E. coli. ACS Synth Biol. 2016;5(7):561-568. doi:10.1021/acssynbio.5b00187.
[32]: Zhao D, Yuan S, Xiong B, et al. Development of a fast and easy method for Escherichia coli genome editing with CRISPR/Cas9. Microb Cell Fact. 2016;15(1):205. doi:10.1186/s12934-016-0605-5.
[33]: Chung ME, Yeh IH, Sung LY, et al. Enhanced integration of large DNA into E. coli chromosome by CRISPR/Cas9. Biotechnol Bioeng. 2017;114(1):172-183. doi:10.1002/bit.26056.
