Team:BOKU-Vienna/Discussion
Menu
D.I.V.E.R.T.
Team
Team
Collaborations
Project
Theory
Experiments
Discussion
Notebook
Protocol
Modeling
Demonstrate
Parts
Overview
Basic Part
Part Collection
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Newsletters
Safety
Medal Criteria
Overview
Attributions
Interlab
Improve
Modeling
Contribution
Sponsors
Menu
Top
Discussion
Discussion
Discussion
For many of us the underlying principle of continuous in vivo directed evolution represents an intriguing idea. Drastically improving desired properties without having much information of the evolving system by just setting up an experiment, taking a step back and waiting for an optimum to be reached sounds almost too good to be true. But directed evolution also does something much more valuable than that: Repeating cycles of mutation and selection not only yields enhanced activity or new functionalities but also lets the researcher learn much more about the biomolecules they are chasing through sequence space on the never-ending hunt for fitness. A lot of the work done with phage-assisted directed evolution (PACE) gives an insight on how otherwise inaccessible information can be gained from continuous evolution experiments1–4.
To take the newly emerging strategy of retroelement-based in vivo directed evolution5,6 one step further, our iGEM team tried to generate a synthetic retrotransposon-like element that could be used to continuously hypermutate any specific sequence while leaving the rest of the genome unharmed. Unfortunately, we were not able to show that our implementation of the more general concept we had elaborated worked. Nevertheless, the results obtained in our side projects can be used in the future to find better approaches: We utilized the published promiscuity of MMLV RT in terms of primer tRNAs (7 and references reviewed therein) and found two tRNAs native to E. coli that appear to enable initiation of reverse transcription. Being able to prime with tRNAs would make the design of helper constructs easier as there would not be a need for the generation of exogenous primers. Also, the expression of higher levels of short RNAs connected to the HDV ribozyme seemed to have toxic effects on E. coli in giving us a hard time generating some backbones.
Additionally, we found that the FRT site employed in Flp/FRT-mediated recombination is not a strong bidirectional terminator giving a chance to work around the fact that RNA polymerase as well as reverse transcriptase need to read through these sequences when Flp/FRT-mediated RMCE is applied for reintegration of the cDNA generated during D.I.V.E.R.T. The FRT terminator strength could be further decreased by trying to engineer the FRT sites to form weaker secondary structures while still remaining functional.
To implement the retrotransposition assay used by Crook et al.5 in E. coli we engineered the Tetrahymena group I self-splicing ribozyme to splice itself from a beta lactamase gene providing another component for the development of future D.I.V.E.R.T. applications.
Since being present in a single copy is necessary for optimal selection behavior we planned and created several generations of plasmids for CRISPR-enhanced homologous recombination in E. coli out of which one yielded around 10 % positive transformants.
[1]: Esvelt KM, Carlson JC, Liu DR. A system for the continuous directed evolution of biomolecules. Nature. 2011;472(7344):499-503. doi:10.1038/nature09929.
[2]: Dickinson BC, Leconte AM, Allen B, Esvelt KM, Liu DR. Experimental interrogation of the path dependence and stochasticity of protein evolution using phage-assisted continuous evolution. Proc Natl Acad Sci. 2013;110(22):9007-9012. doi:10.1073/pnas.1220670110.
[3]: Dickinson BC, Packer MS, Badran AH, Liu DR. A system for the continuous directed evolution of proteases rapidly reveals drug-resistance mutations. Nat Commun. 2014;5:5352. doi:10.1038/ncomms6352.
[4]: Dickinson BC, Carlson JC, Badran AH, Guggiana-Nilo DA, Liu DR. Negative selection and stringency modulation in phage-assisted continuous evolution. Nat Chem Biol. 2014;10(3):216-222. doi:10.1038/nchembio.1453.
[5]: Crook N, Abatemarco J, Sun J, Wagner JM, Schmitz A, Alper HS. In vivo continuous evolution of genes and pathways in yeast. Nat Commun. 2016;7:13051. doi:10.1038/ncomms13051.
[6]: Zheng X, Xing X-H, Zhang C. Targeted mutagenesis: A sniper-like diversity generator in microbial engineering. Synth Syst Biotechnol. 2017;2(2):75-86. doi:10.1016/j.synbio.2017.07.001.
[7]: Mak J, Kleiman L. Primer tRNAs for reverse transcription. J Virol. 1997;71(11):8087-8095.
