Team:BOKU-Vienna/Experiments
Menu
D.I.V.E.R.T.
Team
Team
Collaborations
Project
Theory
Experiments
Discussion
Notebook
Protocol
Modeling
Demonstrate
Parts
Overview
Basic Part
Part Collection
Human Practices
Silver HP
Integrated and Gold
Public Engagement
Newsletters
Safety
Medal Criteria
Overview
Attributions
Interlab
Improve
Modeling
Contribution
Sponsors
Menu
Top
Overview
D.I.V.E.R.T.
CRISPR
Priming
FRTermination
Ribozyme
References
Experiments
Overview
Like most iGEM teams we picked a project maybe a little too ambitious to be executed in just 4 months' worth of lab work. Given enough time we could have built D.I.V.E.R.T. step by step: first finding a setting for efficient reverse transcription and reintegration prior to combining the optimized components and trying to translate the results to other host organisms in the end. Anyhow, we had to resort to a different strategy to get the most out of the time and resources we had to our hands by following both a scientific/systematic as well as a “hope for a lucky shot” approach. On the systematic side of things we wanted to lay a foundation for potential future work by trying to find decent priming conditions for reverse transcription with MMLV RT in E. coli (see here ). In terms of reintegration we are confident that the well-established FLP/FRT-mediated recombination system would do the job. However, as mentioned in the theory section , FRT sites are palindromic and form stable hairpins possibly impairing transcription (and reverse transcription). To assess the severity of this potential problem we conducted terminator strength measurements with an FRT site in sense and antisense orientation (see here ).
Besides this more systematic approach we still wanted to take our chances by building a D.I.V.E.R.T. cassette as well as a helper construct (carrying the RT, FLP and the primer cassettes) according to the best of our knowledge and see whether it worked. We attempted to do this for both S. cerevisiae and E. coli. (see here )
As described in the theory section , our proof of concept assay involves an intron oriented in antisense direction in perspective to the GOI and in sense direction in perspective to the retroelement that is only spliced out of the retroelement mRNA ultimately yielding selectable colonies. Since there is no splicing in prokaryotes we had to find and adopt a self-splicing ribozyme to implement this assay in E. coli (see here ).
The proof of concept assay requires the D.I.V.E.R.T. cassette to be bidirectional (as an additional advantage this also prevents the RT from bumping into ribosomes in E. coli) resulting in a somewhat more intricate genetic construct (see here , figure 1) that would have been troublesome to create via classic cloning techniques and better suited for gene synthesis. As most regulatory sequences (promoters and terminators) available for S. cerevisiae are rather long (~ 200 to 750 bp) we turned to relatively recently published short synthetic regulatory elements1–3 to keep the synthesized construct short. The synthetic galactose-inducible minimal promoter we used (sequence retrieved from 3) was characterized and sent to iGEM to expand the range of inducible yeast promoters available in the registry (see here ).
Details on the experiments conducted in the course of the generation of our CRISPR/Cas9 plug and play plasmids can be found here .
Our shot at D.I.v.e.r.t.
As argued in the theory section there are several options available for accomplishing the main processes necessary for completing the D.I.V.E.R.T. cycle (i.e. reverse transcription and recombination). Lacking the time to systematically optimize the different components we still wanted to try our luck by designing the system in a way that deemed most promising in the light of the information we had gathered during the planning phase. The experimental as well as genetic construct designs for yeast and E. coli are described below. For both hosts we chose to use RNA oligonucleotides transcribed in trans to prime reverse transcription and the FLP/FRT recombinase system for reintegration.
E. Coli
The plan
For our D.I.V.E.R.T. approach we planned on integrating 2 constructs into the E. coli chromosome: the D.I.V.E.R.T. cassette and a helper construct carrying the components required for reverse transcription and reintegration. Both integration cassettes were flanked with terminators to insulate them from nearby genomic transcription. As integration loci we picked IS6 from Bassalo et al.4 for the D.I.V.E.R.T. cassette and lacZ for the helper construct. Homologies measured 200 bp per arm outside of which we placed BsmBI recognition sites for scarless release of the fragment to be integrated from the Golden Gate backbone it was constructed in.
Building the D.I.V.E.R.T. cassette
Figure 1: Features of the E. coli D.I.V.E.R.T. cassette and proof of concept assay (T, P, PBS, RBS, FRT, ampR correspond to terminator, promoter, primer binding site, ribosomal binding site, FRT site, beta-lactamase)
The proof of concept D.I.V.E.R.T. cassette for use in E. coli contains a beta-lactamase gene (referred to as ampR) disrupted by a self-splicing intron in reverse orientation (Figure 1; details on intron design can be found here ). Upon transcription from promoter 1 (P1) the mRNA carries ampR in sense direction which leads to non‑functional protein as the ribozyme won’t splice due to being reversely oriented. Upon transcription from inducible promoter 2 (P2) the mRNA carries the beta-lactamase gene in antisense direction enabling the ribozyme to splice itself out. The spliced RNA should then be reverse transcribed and reintegrated into the genome replacing the original copy leading to functional beta-lactamase. Hence, only full completion of the D.I.V.E.R.T. cycle should yield ampicillin resistant cells. The D.I.V.E.R.T. cassette was synthesized as a single gBlock by IDT and inserted into a Backbone 3 of our Golden Gate standard ( BB3_02 ). Instead of a GOI the gBlock contains a spacer sequence that can be replaced by any CDS in another Golden Gate reaction (with BpiI). Since the cassette was intended to be integrated we picked a strong constitutive promoter as P1 (BBa_J23100) to maintain high expression levels although the GOI would be present only in a single copy. However, all the backbones of our Golden Gate standard featured the high-copy pUC origin of replication leading to substantial stress on the cells during cloning of constructs containing strong constitutive promoters. To accommodate to this challenge we changed P1 to weaker BBa_23108 ( BB3_09 ). Inserting the disrupted beta-lactamase gene with FRT sites at both ends (FRTwt upstream, FRT35 downstream) into BB3_09 gave BB3_43 . For P2 we selected a stronger version of the arabinose-inducible pBAD promoter (BBa_K206000).
Builing the helper construct
The helper construct for E. coli ( BB3_01 ) was to be created from 3 fragments:
• gBlock 1: upstream lacZ homology, insulating terminators, cassette for transcription of primer 1
• BB2_08 : pBAD-RT-FLP. RT and FLP were obtained via IDT gBlocks, cloned into BB1s and combined to a bicistronic cassette using PCRs and Golden Gate.
• gBlock 2: cassette for transcription of primer 2, insulating terminators, downstream lacZ homology
The design allowed for subsequent exchange of the primer cassettes using BpiI (primer 1) or AscI/AflII (primer 2). To ensure defined 3’ ends the primers were attached to the HDV self-cleaving ribozyme. The sequence was taken from Gao et al.6
To our surprise, we had a really hard time obtaining positive colonies in the final cloning step of BB3_01 . After having tried everything in our repertoire (lower regeneration/cultivation temepratures, pre-ligation of the fragments prior to insertion into the isolated empty backbone) without success, subsequent investigations revealed that the primer+HDV expression cassettes – which were featuring BBa_J23102, another rather strong constitutive promoter – were quite toxic for E. coli. While showing reduced growth carrying plasmids with only one primer cassette, the cells would not grow at all when both cassettes were present on the same plasmid. Again we had to resort to a weaker promoter (also BBa_J23105) to obtain positive clones ( BB3_35 ).
Ready for integration - but not ready to integrate
With plasmids carrying both constructs at our hands we were set to finally integrate. However, our plug and play CRISPR/Cas9 was not (details in the CRISPR section ). We had fatally underestimated the toxicity of leaky gene regulation when both the gene for Cas9 as well as for a functional sgRNA are present in the same cell. Hence, it took several generations of CRISPR plasmids to come up with a solution that was not too toxic for cells to grow in liquid culture but still lethal enough to screen for recombinants when induced.
So, we had to set aside the idea of integration and started to think about trying plasmid‑borne D.I.V.E.R.T. We already had everything we needed on 2 plasmids, but those shared the same origin of replication potentially leading to plasmid loss or other adverse events. To tackle this problem we amplified the origin of pSIM57 (the only origin from another compatibility group available to as at the time; details on pSIM5 can be found in the CRISPR section ) and used it to build another backbone that was employed in the “speziale” plasmid series.
The final test
An overnight culture of E. coli DH10B carrying BB3_35 Speziale and BB3_43 was diluted to an OD of 1 and induced with 0.2 % arabinose. 100 µL of cells were plated on LB Amp+ (100 µg/mL) after 6 and after 20 hours. Unfortunately, no colonies emerged. Our approach had failed.
X
X
X
X
X
X
X
Yeast - S. cerevisiae
The plan
For the implementation of our genuine D.I.V.E.R.T. approach in yeast (we chose S. cerevisiae SC288) we followed the same strategy as in E. coli. However, some adjustments needed to be made:
• Instead of an antibiotic resistance gene we decided to use URA3 (a yeast marker gene essential for growth on minimal media) in the proof of concept assay. We intended to delete the generic URA3 gene of SC288 via integration of the helper construct to generate the auxotroph. This way we could apply positive selection8 on the integration of the helper construct and create the auxotrophy we need for the D.I.V.E.R.T. assay, hence “killing two birds with one stone”. We disrupted our URA3 gene with an intron in reverse orientation, analogous to the self-splicing ribozyme in the D.I.V.E.R.T. assay for E. coli.
• To prevent the yeast cells from processing the primers like an mRNA we used the RNA polymerase III promoter from the snoRNA SNR52 gene and SUP4 terminator as described by DiCarlo et al. who applied those for transcription of their sgRNAs9.
• We used longer homology arms ranging from 500 to 800 bp out of which SmaI recognition sites were incorporated for linearization and release of the fragment up for integration. All homology arms were amplified from gDNA purified from SC288 and cloned into BB2s.
• Besides URA3 we picked LEU2 as second integration locus.
• RT and FLP are tagged with the NLS from the SV40 large T antigen10.
• We planned on integrating the DNA into the yeast genome using the CRISPR system (and very kind help) of Martin Altvater and Thomas Gaßler ( Attributions page ).
Building the D.I.V.E.R.T. cassette
To generate the D.I.V.E.R.T. cassette for yeast we combined 4 fragments into BB3_05 :
• LEU2 upstream homology ( BB2_11 )
• gBlock 1: insulating terminator, synthetic minimal constitutive promoter3, PBS1, Kozak sequence11, spacer sequence enclosed in BpiI recognition sites for replacement with GOI
• PBS2, galactose inducible synthetic minimal promoter3 (for details on this promoter see basic part page ), synthetic yeast minimal terminator (T8 from Curran et al.2)
• LEU2 downstream homology ( BB2_12 )
Insertion of the URA3 gene (with intron) into BB3_05 yielded BB3_15 . Since the URA3 gene in BB3_15 shows strong homology to the generic URA3 gene in SC288’s genome the helper fragment has to be integrated first to delete the original URA3 gene and avoid undesired recombination events.
Building a helper construct
6 fragments should have been combined into the helper construct for yeast ( BB3_03 ):
• URA3 upstream homology ( BB2_09 )
• RT under control of the pSynGal (= galactose inducible synthetic minimal promoter; BB2_13 )
• yeast primer cassette 1 ( BB2_03 ); derived from a gBlock obtained from IDT
• yeast primer cassette 2 ( BB2_04 ); derived from a gBlock obtained from IDT
• FLPe under control of the constitutive TEF1 promoter ( BB2_02 )
• URA3 downstream homology ( BB2_10 )
Just like with the original E. coli helper construct ( BB3_01 ) we were unable to obtain positive colonies in spite of trying different reaction conditions, regeneration temperatures etc. We speculated that this again might have something to do with the primer cassettes as we had experienced slow growth and low plasmid purification yields with BB2_03 and BB2_04 . Maybe the RNAP III promoter from SNR52 exhibits cryptic promoter activity in E. coli leading toxic effects of the transcribed RNAs. Remarkably, all constructs we had problems with expressed high levels of short RNAs carrying the HDV ribozyme. Anyhow, to deal with this problem we ordered new gBlocks featuring primer cassettes with pSynGal instead of pSNR52 but the first attempts also failed. We then realized that trying to resolve this mystery would not leave us with enough time to integrate twice and stopped working on our D.I.V.E.R.T. in yeast approach. Instead we concentrated our efforts on the generation of new CRISPR plasmids featuring the lambda Red system in addition to Cas9 and sgRNA making CRISPR enhanced integration in E. coli even easier (see here ).
X
X
X
X
X
X
X
X
X
X
X
X
CRISPR/Cas9 plug and play plasmids
for chromosomal integration in E. coli
Short recap of the theory section : the GOI needs to be present in a single copy per cell for optimal screening or selection behavior. Usually this is achieved by integrating the GOI into the host genome. Genomic integration in E. coli is rather inefficient and requires selection of positive recombinants. Nowadays markerless selection can be applied using CRISPR/Cas9, but there was no non-commercial plug and play solution available to us. So we decided to build our own CRISPR/Cas9 system.
Most publications on CRISPR-enhanced chromosomal integration in E. coli (for example 4,12–18) roughly follow the same procedure employing Cas9 and sgRNA on 2 different plasmids:
A recombination plasmid is transformed into the cells.
The plasmid carrying Cas9 is transformed into the cells.
The donor DNA as well as the plasmid carrying the sgRNA are electroporated into the cells.
While regenerating, expression of the inducible component of CRISPR is activated.
After regeneration cells are plated on selective agar for the gRNA vector.
Colonies are screened for positive recombinants using colony PCR.
We wondered whether it was possible to have sgRNA and Cas9 expression cassettes on the same plasmid to cut down from 3 to 2 transformations and decided to try. To provide convenient plasmid‑curing we used temperature sensitive replicons amplified from pSIM5 (pSC101, dependent on RepA) or pSIM9 (RK2/oriV, dependent on trfA) which we obtained from the Striedner group (see attributions page )7.
As those origins confer low copy numbers19–21 we at first reasoned that adding the regulatory protein to the plasmid might not be necessary and generated CPPP_01 (CRISPR plug and play plasmid) without araC in spite of using the (truncated) araBAD promoter to induce Cas9 (sgRNA was constitutively transcribed from BBa_J23100). We thought that leaky expression could not impose a problem as E. coli DH10B lacks AraC22 which is necessary for even lowest levels of expression (23 and the references therein) with the negative side effect that this would also render our plasmid useless for integration in an E. coli strain lacking araC. Anyhow, we were able to obtain colonies carrying CPPP_01 , but as soon as we inserted a target sequence into the sgRNA cassette the only positive clones that grew in liquid culture contained an indel in the target sequence indicating that cells with the correct target were killed by leaky expression.
CPPP_04 also contained araC, however not in conjunction with full-length pBAD like in situ in wild type E. coli but as separate parts due to the modular assembly of the plasmid. Again, we were not able to obtain functional plasmid carrying an unmutated target sequence.
To minimize background expression we exchanged the promoter of the sgRNA (BBa_J23100) with a shorter version of pBAD and additionally added a very weak constitutive promoter (BBa_J23109) directly after the sgRNA in reverse orientation to quench uninduced sgRNA levels with RNA interference yielding CPPP_07 . With this setup we finally obtained colonies carrying functional plasmids including the targeting sgRNAs. Using the RNAi approach we also created CPPP_10 additionally comprising all lambda Red functions offering the possibility to use one single plasmid for integration.
When trying to integrate ( protocol integration ) our E. coli D.I.V.E.R.T. cassette using cells carrying CPPP_09 (i.e. CPPP_07 targeting IS6) and pSIM9 or only CPPP_12 (i.e. CPPP_10 targeting IS6), however, yielded no positive recombinants out of 50 colonies tested. Apparently, CRISPR-mediated selection was not stringent enough which could be confirmed by lethality tests yielding unsatisfying ~ 10-2 surviving cells for both plasmids. Lambda Red mediated dsDNA integration gives about 10-4 positive recombinants per cell7 requiring CRISPR induced lethality to be at least in the same range for efficient selection.
With those results in mind we figured that arabinose induced sgRNA expression was too low to sufficiently overshoot RNA interference conferred by the antisense sgRNA transcribed from BBa_J23109. Moreover, we had been using a HDV ribozyme downstream of the sgRNA to provide a defined 3’ end which might not be necessary and could in fact even reduce efficiency as we did not incorporate flexible linker regions between the RNA structural elements (sgRNA scaffold, HDV ribozyme, terminator). Hence, we resolved to build a library of plasmids containing all meaningful combinations of
• with/ without RNAi (using the even weaker constitutive antisense promoter BBa_J23103)
• with/ without HDV ribozyme
• with/ without lambda Red functions
consisting of 8 plasmids ( CPPP_13 – CPPP_34 ).
Around that time we also started to specifically search for single-plasmid E. coli CRISPR systems and found Zhao et al.24 who reported that they similarly had had initial problems with leaky Cas9/sgRNA expression until they started using generic full-length pBAD + araC additionally applying further repression by growing cells on 1 % glucose. Adopting their strategy we created CPPP_37 and CPPP_39 (targeting IS6).
Unfortunately, in the time available we were not able to generate all 8 plasmids of the library but only CPPP_22 and CPPP_28 (corresponding to CPPP_24 and CPPP_30 targeting IS6). Both plasmids showed adequate lethality about 10-4. To facilitate less laborious screening we constructed a fragment containing the IS6 homologies enclosing eGFP under control of the BBa_J23100 promoter which was used for integration by means of CPPP_39 + pSIM9, CPPP_24 and CPPP_30.
No colonies obtained from the integration reactions showed fluorescence under blue light. We decided to nevertheless perform 40 colony PCRs (with primers binding on the chromosome just outside of the homology arms) per CRISPR plasmid since a single GFP copy might not produce enough protein to be detected with the naked eye. For CPPP_24 and CPPP_30 no positive clones were identified while CPPP_39 + pSIM9 yielded almost 10 % (3/40 colonies) recombinants
(Figure 2).
Figure 2: Colony PCRs of our integration experiments. When the insert is incorporated correctly the band appears at 1300 bp, wild type colonies exhibit bands at 500 bp. The 2-log DNA ladder was used as a standard.
So, we finally managed to integrate DNA in E. coli using one of our CRISPR/Cas9 plug and play plasmids. However, the plasmids featuring the genes necessary for homologous recombination in addition to the CRISPR components yielded no positive results.
All CPP plasmids were constructed using Golden Gate. Cas9 ( BB1_01 ) was obtained from Team Sauer. kanR was amplified from an empty BB1. Replicons and lambda Red functions were amplified from pSIM5 and pSIM9. For CPPPs including the lambda Red genes the whole pSIM plasmid was amplified and used as backbone in which the sgRNA and Cas9 cassettes were inserted (pSIM5 for CPPP_10, pSIM9 for the others) while all CPPPs missing lambda Red were constructed with the replicon amplified from pSIM5.
X
X
X
X
X
X
X
X
X
X
rt Priming
Finding priming conditions for reverse transcription with MMLV RT in E. coli
In addition to giving our D.I.V.E.R.T. concept a try we also wanted to follow a more systematic approach by testing whether we could prime reverse transcription with MMLV RT in E. coli employing E. coli’s native tRNAs or an exogenous RNA oligo primer. The design of our assay was inspired by a similar experiment done by Elbaz et al. in 25. We created 8 plasmids ( BB3_45 - BB3_49 , BB3_55 , BB3_57 , BB3_58 ), each of which carried an arabinose inducible GFP gene containing a spacer sequence and a tRNA binding site or PBS (primer binding site) upstream of the terminator. The constructs also featured the gene for MMLV RT expressed from a constitutive promoter of medium strength (BBa_J23105). BB3_57 and BB3_58 additionally included the primer 2 expression cassette from our D.I.V.E.R.T. experiment transcribing a 21 nt RNA oligonucleotide complementary to PBS2 on BB3_58 .
For tRNA priming 5 of the most abundant tRNAs according to Dong et al.26 were selected and corresponding sequences retrieved from the GtRNAdb data base27.
In principle, upon induction with arabinose GFP should be expressed from all constructs. In cells carrying adequate priming conditions, however, reverse transcription would be initiated and the MMLV RT traversing the mRNA from 3’ to 5’ would interfere with the ribosomes leading to reduced GFP production. Fluorescence was assessed using flow cytometry.
Construct details and priming sites:
• BB3_45 : Ala1B tRNA-BS (tRNA-Ala-TGC-1-1)
• BB3_46 : Arg2 tRNA-BS (tRNA-Arg-ACG-1-4)
• BB3_47 : Glu2 tRNA-BS (tRNA-Glu-TTC-1-2)
• BB3_48 : Gly3 tRNA-BS (tRNA-Gly-GCC-1-2)
• BB3_49 : Ile1 tRNA-BS (tRNA-Ile-GAT-1-1)
• BB3_55 : PBS2 (complementary to primer 2 which is only expressed in BB3_57 and BB3_38 . BB3_55 is the control for BB3_45 - BB3_49 )
• BB3_57 : no priming site, control for BB3_58
• BB3_58 : PBS2
In BB3_45 - BB3_49 the GFP gene is followed by 17 random base pairs and 18 nucleotides complementary to the 3’ end of the respective tRNA. In BB3_55 and BB3_58 the GFP gene is followed by 11 random base pairs and 21 nucleotides complementary to primer 2 (PBS2). BB3_57 does not feature any additional sequences between the GFP stop codon and terminator.
As can be seen in the representation of the flow cytometry results ( protocol RT activity assay ) in figure 3 and table 1 cells carrying BB3_45 and BB3_46 showed significantly reduced fluorescence compared to the control BB3_55 . The same can be said for BB3_58 exhibiting less than 20 % of the corresponding control’s ( BB3_57 ) fluorescence activity.
Figure 3: Results of the RT activity assay using: (A) a RNA oligonucleotide and (B) tRNAs for priming. Results are normalized to the respective controls and given as GFP ratio which is calculated by dividing the means of the samples by the means of the controls. Error bars represent the standard deviation of triplicates
Table 1: GFP expression ratios compared to the control of the tRNA priming experiment
These findings indicate that initiating reverse transcription with MMLV RT in E. coli could be most conveniently done by utilizing already present native tRNAs by simply including sequences complementary to their 3’ end in the respective genetic construct. This is supported by the fact that MMLV RT shows weaker specific interactions with its cognate tRNA primer than some other well described reverse transcriptases (28 and the references reviewed therein). Also, priming with an RNA oligo appears to be feasible. The next step in confirming these results would be the isolation of ssDNA generated by reverse transcription primed under the respective conditions.
X
X
X
X
X
X
X
X
FRT Terminaton strength
For measurement of the FRT-sequences' terminator strength we went with an adapted version of the protocol developed by Cassie Huang in Tom Knight's lab . From their experiments we took their plasmid for calibration, BBa_I13515 . This plasmid contains a promoter, followed by two consecutive genes for GFP and RFP expression and finally a terminator. If a terminator is inserted in between GFP and RFP, the amount of expressed RFP is decreased relative to the amount of expressed GFP. This can be measured via fluorescence and the terminator strength can be calculated with the following formula (1) from Jason Kelly in Drew Endy's lab :
In formula (1) RFPterm and GFPterm represent the number of fluorescence units from RFP and GFP respectively with the tested terminator in-between and RFPcontrol and GFPcontrol were taken from cultures containing the original plasmid BBa_I13514.
To test the terminator strength of our FRT-sequence, we created four different plasmids from BBa_I13415 as template (click to see the plasmid maps):
• FRT: With the FRT-sequence in between GFP and RFP
• TRF: With the inverted FRT-sequence in-between
• RNDM: With a random sequence the same size as the FRT-sequence*
• B1001: With the terminator BBa_B1001 in-between, as positive control of termination
X
X
X
X
X
The plasmids were created as follows. First, a BsmBI-recognition site inside the CmR gene was removed via site-directed mutagenesis. From that template, primers were designed to anneal alongside the space between the genes for GFP and RFP. Annexed to those primers were the respective halves of the target inserts, as well as BsmBI-fusion sites. These primers were used for PCR; the linear PCR-products were then digested with BsmBI and ligated with T4-ligase, resulting in circular plasmids containing our inserts at the desired position.
To test the terminator strength, aforementioned plasmids were transformed in E. coli DH10B and selected on agar plates containing chloramphenicol. From those plates, colonies were used to inoculate the pre-cultures for our determination - test tubes with 1 mL LB-medium and chloramphenicol, shaken overnight at 180 rpm and 37 °C. For main-cultures we used test tubes containing 1 mL LB medium with chloramphenicol, inoculated to an OD600 of 0.1 from the pre-cultures. After 2 h inoculation time, protein expression was induced with addition of arabinose to a concentration of 0.1% and after 10 h inoculation time, cells were harvested for fluorescence measurement.
Each plasmid was cultured in triplets. Measurements were done using a Tecan i-control infinite 200 plate reader device. Culture broth was transferred to Nunclon 96 Flat Bottom Black well plates. Fluorescence measurements were done at 488 nm excitation wavelength and 530 nm emission wavelength for GFP-quantification as well as 531 nm excitation wavelength and 650 nm emission wavelength for RFP-quantification. Gain was set to 60. Results can be seen in table 2 and figure 4
Table 2: Results from the fluorescence measurements and calculation of the terminator efficiency
Figure 4: Terminator efficiencies of our tested sequences.
Since the random sequence should not form any strong secondary structure as suggested by UNAFold we were quite suprised to detect terminator activity of around 0.5. This may be due to issues with mRNA stability or some other factor. However, the exact reason would have to be investigated further. What still can be seen from the results is, that the FRT sequence shows clear termination activity - but not in both directions.
[*]: The random sequence was tested as an additional control for the calibration. The sequence is: GGT CGC GAG TAC CTG AAC TAA GGC TCC GGA CAG GAC TAT ATA CTA AGG
self-splicing ribozyme
and the antibiotic resistance gene
Design
Selection of a self-splicing ribozyme:
For the proof of concept for D.I.V.E.R.T. an antibiotic resistance disrupted by a reversely oriented intron is required. While in yeast there are plenty of well-described introns to choose from, in E. coli information is rarer. However, scientists already discovered group I and group II introns working in prokaryotes as well. For example, the Tetrahymena group I intron, a self-splicing ribozyme, is proven to exhibit splicing activity in vivo in E. coli without the expression of additional factors29. In contrast to other self-splicing ribozymes the Tetrahymena group I intron is very well-described and can be engineered to cut itself scarlessly from any suitable sequence. For exact recognition of the splicing sites intron-exon pairing in the P1 and P10 (paired region 1 and 10) regions is necessary30. P1 determines the 5’ end of the self-splicing ribozyme and contains P1ex, which has considerable influence on splicing activity. P10 pairs with a sequence at the 5' end of the exon and therefore defines the 3’ end of the intron (
Figure 5
). Although P1 and P10 mark the opposite ends of the intron, they are both located in the same region and it is worth noticing that they can overlap.
Figure 5: Tetrahymena wt precursor mRNA. Exon bases are written in lowercase, the Tetrahymena group I intron bases in capital letters. A part of the P1 region is complementary to the 5’ exon sequence determining the 5’ end of the intron, whereas the P10 region is determining the 3’ end.
Selection of an antibiotic resistance:
For the Tetrahymena group I intron Guo et al. proposed in their publication31 that the 5’ adjacent exon sequence of the mRNA should consist of 5'-CUCUCU-3'. Considering this and the requirement of the intron being on the antisense strand we were looking for an antibiotic resistance gene featuring 5'-AGAGAG-3' (reverse complementary to 5'-CUCUCU-3') somewhere in the CDS. We finally found a β-lactamase gene with a 5'-AGAGAA-3' sequence approximately in the middle of the gene32. By introducing a silent mutation (A357G) the desired 5’ adjacent exon sequence was obtained. Furthermore, a BsaI recognition site was silently mutated to comply with our Golden Gate-cloning standard.
Design of the self-splicing ribozyme:
When engineering the Tetrahymena group I intron to splice itself from a foreign gene, the design of P1 and P10 is crucial to preserve self-splicing activity: The P10  region was exchanged for bases complementary to the 3’ adjacent exon sequence of the new β-lactamase gene. Because P1ex and P10 are overlapping, P1 was altered as well. Additionally, as Guo et al. suggest31, a non-complementary base pair was introduced in the P1ex sequence to promote splicing activity (
Figure 6
).
Figure 6: Design of the precursor mRNA of the β-lactamase gene with the engineered Tetrahymena intron . Exon bases are written in lowercase, intron bases in capital letters. The β-lactamase sequence is reverse complementary due to the proof of concept setup.  P1 as well as P10 were altered in the design process.
Experiments & Results
Proof of self-splicing:
To test the functionality of the engineered self-splicing Tetrahymena group I intron in the β-lactamase CDS the gene was cloned into an expression vector ( BB3_06 ) and transformed into E. coli. An overnight culture was then used for mRNA extraction ( protocol mRNA extraction ) and cDNA synthesis ( protocol cDNA synthesis ) was performed using a pair of gene specific primers. For enhancing the amount of detectable DNA another PCR using the cDNA as template was done afterwards. After running a gel DNA corresponding to the length of both, the spliced and the unspliced gene was detected (
Figure 7
).
Figure 7: The sample was loaded on an agarose gel. After 40 min at 130 V two bands were visible. One, 1248 bp long, represents the unspliced mRNA, the second, 835 bp long, represents the spliced mRNA.
The 835 bp band was cut out and cleaned up. The DNA was multiplied by PCR and subsequently sent for sequencing, which confirmed the self-splicing activity exactly as it was intended during the design.
Ampicillin resistance:
For the proof of concept a functional ampicillin resistance gene is required and, since the recombination is Flp/FRT mediated, FRT sequences are introduced directly after the start and stop codons. Therefore, 16 additional amino acids are expressed at the N-terminus of the ampicillin resistance gene. To test if the ampicillin resistance is still functional, this modified gene was cloned into an expression vector ( BB3_04 ). After subsequent transformation into E. Coli, cells were plated on LB low salt agar with 100 µg/mL ampicillin. After an overnight incubation ampicillin resistant colonies were visible. For verification a colony was picked and its plasmid was purified and the insert sequenced.
X
X
References
[1]: Guo Z, Sherman F. Signals sufficient for 3’-end formation of yeast mRNA. Mol Cell Biol. 1996;16(6):2772-2776.
[2]: Curran KA, Morse NJ, Markham KA, Wagman AM, Gupta A, Alper HS. Short Synthetic Terminators for Improved Heterologous Gene Expression in Yeast. ACS Synth Biol. 2015;4(7):824-832. doi:10.1021/sb5003357.
[3]: Redden H, Alper HS. The development and characterization of synthetic minimal yeast promoters. Nat Commun. 2015;6:7810. doi:10.1038/ncomms8810.
[4]: Bassalo MC, Garst AD, Halweg-Edwards AL, et al. Rapid and Efficient One-Step Metabolic Pathway Integration in E. coli. ACS Synth Biol. 2016;5(7):561-568. doi:10.1021/acssynbio.5b00187.
[5]: Turan S, Kuehle J, Schambach A, Baum C, Bode J. Multiplexing RMCE: Versatile Extensions of the Flp-Recombinase-Mediated Cassette-Exchange Technology. J Mol Biol. 2010;402(1):52-69. doi:10.1016/j.jmb.2010.07.015.
[6]: Gao Y, Zhao Y. Self-processing of ribozyme-flanked RNAs into guide RNAs in vitro and in vivo for CRISPR-mediated genome editing. J Integr Plant Biol. 2014;56(4):343-349. doi:10.1111/jipb.12152.
[7]: Datta S, Costantino N, Court DL. A set of recombineering plasmids for gram-negative bacteria. Gene. 2006;379(1-2):109-115. doi:10.1016/j.gene.2006.04.018.
[8]: Boeke JD, LaCroute F, Fink GR. A positive selection for mutants lacking orotidine- 5’-phosphate decarboxylase activity in yeast: 5-fluoro-orotic acid resistance. Mol Gen Genet. 1984;197(2):345-6.
[9]: Dicarlo JE, Norville JE, Mali P, Rios X, Aach J, Church GM. Genome engineering in Saccharomyces cerevisiae using CRISPR-Cas systems. Nucleic Acids Res. 2013;41(7):4336-4343. doi:10.1093/nar/gkt135.
[10]: Dingwall C, Laskey RA. Nuclear targeting sequences - a consensus? Amino Acids. 1991;16(December):478-481.
[11]: Hamilton R, Watanabe CK, de Boer HA. Compilation and comparison of the sequence context around the AUG startcodons in Saccharomyces cerevisiae mRNAs. Nucleic Acids Res. 1987;15(8):3581-3593. doi:10.1093/nar/15.8.3581.
[12]: Jiang W, Bikard D, Cox D, Zhang F, Marraffini LA. RNA-guided editing of bacterial genomes using CRISPR-Cas systems. Nat Biotechnol. 2013;31(3):233-239. doi:10.1038/nbt.2508.
[13]: Jiang Y, Chen B, Duan C, Sun B, Yang J, Yang S. Multigene editing in the Escherichia coli genome via the CRISPR-Cas9 system. Appl Environ Microbiol. 2015;81(7):2506-2514. doi:10.1128/AEM.04023-14.
[14]: Li Y, Lin Z, Huang C, et al. Metabolic engineering of Escherichia coli using CRISPR-Cas9 meditated genome editing. Metab Eng. 2015;31:13-21. doi:10.1016/j.ymben.2015.06.006.
[15]: Reisch CR, Prather KLJ. The no-SCAR (Scarless Cas9 Assisted Recombineering) system for genome editing in Escherichia coli. Sci Rep. 2015;5(June):15096. doi:10.1038/srep15096.
[16]: Pyne ME, Moo-Young M, Chung DA, Chou CP. Coupling the CRISPR/Cas9 system with lambda red recombineering enables simplified chromosomal gene replacement in Escherichia coli. Appl Environ Microbiol. 2015;81(15):5103-5114. doi:10.1128/AEM.01248-15.
[17]: Chung ME, Yeh IH, Sung LY, et al. Enhanced integration of large DNA into E. coli chromosome by CRISPR/Cas9. Biotechnol Bioeng. 2017;114(1):172-183. doi:10.1002/bit.26056.
[18]: Tang Q, Lou C, Liu S-J. Construction of an easy-to-use CRISPR-Cas9 system by patching a newly designed EXIT circuit. J Biol Eng. 2017;11(1):32. doi:10.1186/s13036-017-0072-5.
[19]: Hashimoto-Gotoh T, Franklin FCH, Nordheim A, Timmis KN. Specific-purpose plasmid cloning vectors I. Low copy number, temperature-sensitive, mobilization-defective pSC101-derived containment vectors. Gene. 1981;16(1-3):227-235. doi:10.1016/0378-1119(81)90079-2.
[20]: Figurski DH, Helinski DR. Replication of an origin-containing derivative of plasmid RK2 dependent on a plasmid function provided in trans. Proc Natl Acad Sci U S A. 1979;76(4):1648-1652. doi:10.1073/pnas.76.4.1648.
[21]: Grinter NJ. Replication control of IncP plasmids. Plasmid. 1984;11(1):74-81. doi:10.1016/0147-619X(84)90009-X.
[22]: Durfee T, Nelson R, Baldwin S, et al. The complete genome sequence of Escherichia coli DH10B: Insights into the biology of a laboratory workhorse. J Bacteriol. 2008;190(7):2597-2606. doi:10.1128/JB.01695-07.
[23]: Schleif R. AraC protein, regulation of the l-arabinose operon in Escherichia coli, and the light switch mechanism of AraC action. FEMS Microbiol Rev. 2010;34(5):779-796. doi:10.1111/j.1574-6976.2010.00226.x.
[24]: Zhao D, Yuan S, Xiong B, et al. Development of a fast and easy method for Escherichia coli genome editing with CRISPR/Cas9. Microb Cell Fact. 2016;15(1):205. doi:10.1186/s12934-016-0605-5.
[25]: Elbaz J, Yin P, Voigt CA. Genetic encoding of DNA nanostructures and their self-assembly in living bacteria. Nat Commun. 2016;7:11179. doi:10.1038/ncomms11179.
[26]: Dong H, Nilsson L, Kurland CG. Co-variation of tRNA Abundance and Codon Usage in Escherichia coli at Different Growth Rates. J Mol Biol. 1996;260(5):649-663. doi:10.1006/jmbi.1996.0428.
[27]: Chan PP, Lowe TM. GtRNAdb: A database of transfer RNA genes detected in genomic sequence. Nucleic Acids Res. 2009;37(SUPPL. 1):93-97. doi:10.1093/nar/gkn787.
[28]: Mak J, Kleiman L. Primer tRNAs for reverse transcription. J Virol. 1997;71(11):8087-8095.
[29]: Waring RB, Ray JA, Edwards SW, Scazzocchio C, Davies RW. The Tetrahymena rRNA intron self-splices in E. coli: in vivo evidence for the importance of key base-paired regions of RNA for RNA enzyme function. Cell. 1985 Feb;40(2):371-80.
[30]: Burke JM, Cech TR, Davies RW, Schweyen RJ, Shub DA, Szostak JW, Tabak HF. Structural conventions for group I introns. Nucleic Acids Res. 1987 Sep 25; 15(18): 7217–7221.
[31]:  Guo F, Cech TR. In vivo selection of better self-splicing introns in Escherichia coli: the role of the P1 extension helix of the Tetrahymena intron. RNA. 2002 May;8(5):647-58.
[32]: Zhou W, Wang Y, Lin J. Functional cloning and characterization of antibiotic resistance genes from the chicken gut microbiome.Appl Environ Microbiol. 2012 Apr;78(8):3028-32. doi: 10.1128/AEM.06920-11. Epub 2012 Jan 27.
