<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNebraska-Lincoln_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNebraska-Lincoln/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><H1 style="font-size:25px;padding:2px;font-family:'Roboto Condensed';">UNL 2018 Improving Early Detection of The Emerald Ash Borer</H1><DIV class="menu"><DIV class="menuDrop"><DIV id="teamButton" class="menuButton"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/TeamLanding">Team</A></DIV><DIV class="menuContent"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Team">Team</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Collaborations">Collaborations</A></DIV></DIV><DIV class="menuDrop"><DIV id="projectButton" class="menuButton"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/ProjectLanding">Project</A></DIV><DIV class="menuContent"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Description">Description</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Design">Design</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Notebook">Notebook</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/InterLab">InterLab</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Contribution">Contribution</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Model">Model</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Results">Results</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Attributions">Attributions</A></DIV></DIV><DIV class="menuDrop"><DIV id="partsButton" class="menuButton"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/PartsLanding">Parts</A></DIV><DIV class="menuContent"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Basic_Part">Basic Parts</A></DIV></DIV><DIV class="menuButton"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/Safety">Safety</A></DIV><DIV class="menuDrop"><DIV id="humanButton" class="menuButton"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/HumanLanding">Human Practices</A></DIV><DIV class="menuContent"><A href="https://2018.igem.org/Team:UNebraska-Lincoln/HP/Silver">Silver HP</A><A href="https://2018.igem.org/Team:UNebraska-Lincoln/HP/Gold_Integrated">Integrated and Gold</A></DIV></DIV></DIV><DIV class="reallyfull"><H1 class="topOfPage" style="font-family:'Roboto Condensed';padding-bottom=60px">Results</H1><H3 class="leap"><B>Overview: </B></H3><P>
We are excited to present the following results:
</P><P><UL><LI> Successfully cloned and fully sequenced codon-optimized TPS4-B73 gene (7-epi-sesquithujene synthase gene) into the vector p100169. 
		Also successfully cloned and fully sequenced non-codon optimized TPS4-B73 gene into the same vector.</LI><LI> Confirmed FPP production and tested our sesquiterpene production assay using ADS (amorpha-4,11-diene synthase).</LI><LI> Observed in vivo production of sesquiterpenoid, including target molecule 7-epi-sesquithujene, by codon-optimized TPS4-B73 and identified major products.</LI><LI> Quantified sesquiterpene production in E. coli culture expressing the codon-optimized TPS4-B73.</LI></UL></P><H3 class="leap">Cloning:</H3><P>
We first cut out the ADS gene from p100169. To clone TPS4 into the vector, we used Sequence and Ligation Independent Cloning (SLIC). 
On our third SLIC attempt, we isolated the plasmid from six colonies and performed a restriction digest to identify if any of the constructs were correct.
</P><H3 class="center">Fig 0.1: Gel electrophoresis of digested SLIC products</H3><P>
The fourth clone had the expected digest pattern of two bands, so we proceeded to fully sequence the plasmid around our part to confirm our construct pTPS4.
</P><P>
We used restriction cloning to clone the codon-optimized TPS4. There were many colonies on the control empty vector plate, so we chose six colonies to test the ligation product with a restrictive digest.
</P><H3 class="center">Fig 0.2: Gel electrophoresis of digested ligation products</H3><P>
 
Five of the colonies showed the correct digestive pattern. We fully sequenced the plasmid of the first colony around our part to confirm our construct pTPS4CO.

</P><H3 class="leap">Confirming FPP production: </H3><P>
We ran our sesquiterpene production assay 4 times, each time with different growth variables. 
The strains tested each had the FPP producing p100167 and a sesquiterpene synthase plasmid. 
For the first 3 assays, we tested p100169 (pADS), pTPS4, and pTPS4CO. 
ADS is the well-characterized amorpha-4,11-diene synthase, so we knew that if the pADS cultures produced amorpha-4,11-diene, 
then the mevalonate-dependent pathway was working and FPP was successfully being produced. This would show that our sesquiterpene production was not being bottlenecked by FPP production.
 We took OD600 measurements and took octane samples to measure the culture growth and sesquiterpene production. The octane samples were analyzed through gas chromatography-mass spectrometry.
</P><P>
In each assay, we observed the presence of amorpha-4,11-diene in our pADS cultures.
</P><H3 class="center">Fig 1: Gas chromatogram of the first assay 24 hour sample showing the presence of the amorpha-4,11-diene peak with a retention time of 7.684 minutes</H3><H3 class="center">Fig 2: Standard mass spectrum of amorpha-4,11-diene from the NIST library</H3><H3 class="center">Fig 3: Mass spectrum of produced amorpha-4,11-diene</H3><P>
The mass fragmentation pattern is unique to each molecule, like a molecular fingerprint. 
The mass detection range on the instrument we used had a lower limit of 50 m/z, so any characteristic peaks below 50 m/z will not appear on our samples. 
Comparison of the mass spectrum of our product to the standard identifies the product as the expected amorpha-4,11-diene. 
The presence of amorphadiene showed that the mevalonate-dependent pathway was functional and was producing sufficient amounts of FPP for sesquiterpene production.
</P><H3 class="leap">pTPS4CO product identification: </H3><P>
In the first assay, our pTPS4 and pTPS4CO cultures were tested, but only pTPS4CO produced sesquiterpenoid products. 
Since FPP was being produced under these conditions as shown in the pADS culture, the failure of pTPS4 was not a problem of FPP production. 
We were able to identify the major products of pTPS4CO to determine functionality.
</P><H3 class="center">Fig. 4: Gas chromatogram of the 120 hour sample of pTPS4CO sesquiterpene products. 7-epi-sesquithujene had a retention time of 6.689 minutes, and beta-bisabolene had a retention time of 7.989 minutes.</H3><H3 class="center">Fig. 5: Standard mass spectrum of 7-epi-sesquithujene from the NIST library</H3><H3 class="center">Fig. 6: Mass Spectrum of produced 7-epi-sesquithujene</H3><P>
The mass detection range on the instrument we used had a lower limit of 50 m/z, so any characteristic peaks below 50 m/z will not appear on our samples. Comparison of the mass spectrum with the standard identifies the product with retention time 6.689 minutes as 7-epi-sesquithujene, which means that our pathway and part were functional. 
We were also able to identify the other major product, with retention time 7.989 minutes, as beta-bisabolene.
</P><H3 class="center">Fig. 7: Standard mass spectrum of beta-bisabolene</H3><H3 class="center">Fig. 8: Mass spectrum of produced beta-bisabolene</H3><P>
The mass detection range on the instrument we used had a lower limit of 50 m/z, so any characteristic peaks below 50 m/z will not appear on our samples. Comparison of the mass spectrum to the standard identifies the product as beta-bisabolene. 
This was further evidence that our part was functioning correctly in vivo, since both 7-epi-sesquithujene and beta-bisabolene had been predicted as the major products in in vitro experiments (Kollner, 2004).
</P><H3 class="leap">Product Quantification:</H3><P>
After we had identified that our pathway was working and our part was producing the desired product, we developed a product quantification method. 
7-epi-sesquithujene is commercially unavailable, so we could not use it as an analytical quantification standard; instead, we used the sesquiterpene (-)-trans-caryophellene as an internal standard. 
It is not one of the potential products of 7-epi-sesquithujene synthase (Kollner et al, 2004), and it has a retention time of 7.210, which is separated from the products we wished to quantify.
 To calculate the concentrations of our products, we used the internal standard method developed by Martin et al in 2001. The concentrations of 7-epi-sesquithujene and beta-bisabolene 
 are reported as (-)-trans-caryophellene equivalents based on the relative abundance of the 204, 161, 119, and 105 ions to all other ions for each product. 
 These ions are the molecular ion and other distinctive ions to sesquiterpenes. The ratios were 10.9%, 20.1%, and 9.05% for (-)-trans-caryophellene, 7-epi-sesquithujene, and beta-bisabolene respectively.
</P><H3 class="center">Fig. 9: Selective ion search for the 204, 161, 119, and 105 ions</H3><P>
Using this method, we were able to quantify the amounts of products in each of our 4 assays and compare growing conditions to production.
</P><H3 class="leap">Assay Growth Conditions:</H3><P>
Our first assay had an initial OD600 of 0.02, and the cultures were induced at an OD600 ranging from 0.1-0.4 for the three cultures. 
Unfortunately, due to other students in the lab, the shaker speed was not kept constant at 150 rpm, but for hours 48-120 was set at 250 rpm. 
The assay went for 120 hours instead of 72 hours due to lab accessibility issues. For this and following assays, the cultures were induced and octane was added at the zero hour.
</P><P>
Our second assay had a higher initial OD600 of 0.07, and the cultures were induced at a much higher OD600 ranging from 1-4 for the three cultures. 
Again, due to other students in the lab, the shaker speed was not kept constant at 150 rpm and was instead set at 250 rpm for the first 24 hours, which caused culture growth to differ from how we had expected. 
They were grown for 48 hours.
</P><P>
Our third assay had an initial OD600 of 0.05 and an induction OD600 of 0.15. The shaker was set at 150 rpm throughout the experiment, and the cultures were grown for 72 hours.
</P><P>
Each of these three assays tested pADS, pTPS4, and pTPS4CO. Our fourth assay tested six biological replicates of pTPS4CO. 
They had an initial OD600 of 0.05 and an induction OD600 ranging from 0.15-0.53. They were grown for 49 hours.
</P><H3 class="leap">Assay Production Results: </H3><H3 class="center">Fig. 10: pADS growth and amorpha-4,11-diene production</H3><P>
Amorphadiene was produced under all three growing conditions, so FPP was being produced in all 3 experiment conditions. 
The results show that pADS favored the growing conditions with induction at the higher optical density.
</P><H3 class="center">Fig 11: pTPS4CO growth and production</H3><P>
The results show that 7-epi-sesquithujene was produced under all growing conditions. 
Unlike pADS, pTPS4CO but was not optimal when induced at a higher optical density. 
The results show that the production of 7-epi-sesquithujene can vary widely with different growing conditions.
</P><P>
Non-codon optimized pTPS4 was also tested in assays 1-3. It did not produce any sesquiterpenes in assay 1, but in assays 2 and 3 it did produce sesquiterpenes; 
however, we did not move forward in further testing due to time constraints. 
We chose to do further testing on the codon-optimized variant because it had consistently produced sesquiterpenes in each assay, albeit of varying amounts. 
We wished to test whether pTPS4CO production was consistent under identical growing conditions.
</P><P>
In our fourth assay, we tested six biological replicates of pTPS4CO. Unfortunately, an error was made in measuring the OD600 at the 29 hour time point for replicates 2,3, and 4.
</P><H3 class="center">Fig. 12: Growth and production of correctly measured TPS4CO replicates 1, 5, and 6</H3><H3 class="center">Fig. 13: Growth and production of biological replicates 2, 3, and 4 with missing OD600 data point at 29 hours</H3><P>
The biological replicates were grown under the same conditions; however, the resulting concentrations of 7-epi-sesquithujene varied more widely than our previous results testing growing conditions. 
Under these identical conditions, 7-epi-sesquithujene production ranged from 0.52 to 92.78 mg/L. 
We concluded that there must be a production variable that we have not discovered yet that is causing the variance.
</P><P>
We also quantified beta-bisabolene production in our biological replicates of pTPS4CO. Literature of in vivo experiments comparing the relative abundance of 7-epi-sesquithujene 
synthase reported that beta-bisabolene should be the major product, constituting 29.1% of total products, with 7-epi-sesquithujene as the second major product, constituting 24.4% of total products (Kollner, 2004). This produced a 7-epi-sesquithujene to beta-bisabolene ratio of 0.83.
</P><H3 class="center">Fig. 14: Comparison of 7-epi-sesquithujene production to beta-bisabolene production at the 49 hour time point of assay 4</H3><P>
These results show that 7-epi-sesquithujene was our major product instead of beta-bisabolene. Although the final concentrations varied in magnitude, 
each replicate produced slightly more 7-epi-sesquithujene than beta-bisabolene. The observed ratio of 7-epi-sesquithujene to beta-bisabolene was 1.05 with a standard deviation of 0.03. This was different than the literature had reported, 
where the ratio of 7-epi-sesquithujene to beta-bisabolene was 0.83 and beta-bisabolene was clearly the major product. This showed that either codon optimization, performing the experiment in vivo, 
or different growth conditions could change the product profile of 7-epi-sesquithujene synthase.
</P><H3 class="leap">Other experiments: </H3><P>
We attempted to do an SDS-PAGE several times to confirm the presence of 7-epi-sesquithujene synthase, but we had trouble with the procedure. 
New technical difficulties occurred each time we ran the procedure, so unfortunately we ran out of time to get conclusive results.
</P><P>
A preliminary purification experiment was done to attempt to purify amorpha-4,11-diene, but we ran out of time to get conclusive results and perform the procedure to purify 7-epi-sesquithujene.
</P><H3 class="leap">Other experiments: </H3><P><UL><LI> Purify 7-epi-sesquithujene to perform nuclear magnetic resonance spectroscopy (NMR) and infrared spectroscopy (IR) analysis to further confirm product identity</LI><LI> Repeat sesquiterpene production assays with a higher sample collection rate to identify production variables</LI><LI> Optimize the assay procedure to increase 7-epi-sesquithujene yields</LI></UL></P><H3 class="leap">Considerations for replicating experiments: </H3><P><UL><LI> In our experiments, pTPS4CO always produced some amount of sesquiterpene products. Replicate experiments should always produce products, but the yields may vary until further production variables are identified.</LI></UL></P><P>Works Cited</P><UL><LI>
Martin, V. J.; Yoshikuni, Y.; Keasling, J. D. The in vivo synthesis of plant sesquiterpenes by <I>Escherichia coli. Biotechnology and Bioengineering</I><B>2001</B>, 75(5), 497–503.
</LI><LI>
Köllner, T. G.; Schnee, C.; Gershenzon, J.; Degenhardt, J. The variability of sesquiterpenes emitted from two <I>Zea mays cultivars is controlled by allelic variation of two terpene synthase
 genes encoding stereoselective multiple product enzymes. <I>The Plant Cell</I><B>2004</B>, 16(5), 1115–1131.
</I></LI></UL></DIV><H1 class="topOfPage" style="font-family:'Roboto Condensed';text-align:initial;margin-left:20%;padding-bottom:30px">Thanks to Our Sponsors</H1></DIV></DIV></DIV></DIV></DIV></BODY></HTML>