"
Page
Discussion
View source
History
teams
Log in
 
Team:UNIPV-Pavia/Protocols
From 2011.igem.org
UNIPV TEAM 2011
Project
Background & Motivation
Solution
Modelling
Results
References
Lab
Materials & Methods
Protocols
Instruments
Measurements
Notebook
Calendar
Freezer Management
Biosafety
Parts
Submitted
Characterized
Data Page
About us
Team
Pavia
Gallery
Sponsors
Protocols
Contents
1 Media & Antibiotics
1.1 LB
1.2 LB Agar
1.3 SOB
1.4 SOC
1.5 M9 supplemented with glycerol (M9gly)
1.6 Antibiotics
2 E. coli transformation
2.1 Transforming home-made competent cells
2.2 Transforming commercial competent cells
3 E. coli competent cells preparation
3.1 H. Inoue et al. (1990), High efficiency transformation of Escherichia coli with plasmids, Gene 96 23-28.
3.2 J. Sambrook, E.F. Fritsch, T. Maniatis (1989)
4 E. coli strains (all in -80°C freezer)
4.1 TOP10
4.2 DH5alpha
4.3 MG1655 (seq)
4.4 MGZ1
5 Long term bacterial glycerol stocks
6 Plasmid digestion for BioBrick Standard Assembly
7 Ethanol precipitation with sodium acetate
8 Ligation
9 DNA resuspension from iGEM plates
10 PCR
11 Electrophoresis
11.1 1 kb Plus DNA Ladder preparation (Fermentas)
12 Glycerol stocks
Media & Antibiotics
LB
Add:
10 g/L NaCl
10 g/L Bacto-Tryptone
5 g/L Bacto-Yeast Extract
ddH2O
to a sterile pyrex bottle
autoclave
(add antibiotic when it reaches ~45°C)
store at +4°C
^top
LB Agar
Add:
10 g/L NaCl
10 g/L Bacto-Tryptone
5 g/L Bacto-Yeast Extract
15 g/L Bacto-Agar
ddH2O
to a sterile 1L flask
autoclave
(add antibiotic when the agar becomes visible in the liquid, at about 45°C; shake gently to avoid bubbles)
pour into Petri plates and close them immediately
let them polymerize for ~2-3h
invert plates and wrap them with aluminium foil and store at +4°C
^top
SOB
Add:
5 g/L Bacto-Yeast Extract
20 g/L Bacto-Tryptone
10mM NaCl
2.5mM KCl
10mM MgSO4
10mM MgCl2
to a sterile pyrex bottle
(optional: check that pH is ~6.8, otherwise adjust with NaOH)
autoclave
(add antibiotic when it reaches ~45°C)
store at +4°C
^top
SOC
Prepare SOB and add 20 ml of 1M glucose (prepare it dissolving 3.6 g of glucose in 20 ml of ddH2O).
^top
M9 supplemented with glycerol (M9gly)
For 1L of medium, add:
734 ml of autoclaved (and cooled to Tamb, N.B. consider evaporation during autoclaving) ddH2O with a magnetic stirrer inside the bottle
5 ml of autoclaved 80% glycerol as carbon source
100 ul of autoclaved or filtered (0.2um) CaCl2 1 M
20 ml of 10% autoclaved casamino acids (dissolve 50 g in 500 ml = 10% stock)
34 ml of filtered (0.2um) thiamine hydrochloride MW=337.27g/mol (340 mg in 34 ml) (keep in mind it is photosensitive)
2 ml of autoclaved MgSO4 1 M
200 ml of autoclaved M9 salts 5x (dissolve 56.4 g in 1 liter ddH2O = 5x stock)
shake the ddH2O with the magnetic stirrer and start adding the other solutions in sterility (each solution must be completely dissolved!) in the order listed above.
add antibiotic if needed
store at +4°C, protected from light
NOTE:
M9 salts 5x, 10% casamino acids, MgSO4 1 M and CaCl2 1 M can be stored at +4°C.
glycerol 80% can be stored at room temperature.
thiamine hydrochloride (LIGHT SENSITIVE) is one-shot and must be prepared each time (keep in mind you loose some volume during filtration)
^top
Antibiotics
Stocks at -20°C freezer:
Ampicillin 100 mg/ml (in water)
Kanamycin 50 mg/ml (in water)
Chloramphenicol 34 mg/ml (in 100% ethanol)
These stocks are 1000x for high copy number plasmids.   For low copy number plasmids, you should use these final concentrations in media:
Ampicillin 50 ug/ml
Kanamycin 20 ug/ml
Chloramphenicol 12.5 ug/ml
^top
E. coli transformation
Transforming home-made competent cells
heat ligation at 65°C to inactivate T4 ligase
thaw in ice a vial of TOP10 competent cells stored at -80°C
incubate a selective LB agar plate at 37°C
pipet 800ul of LB (without antibiotic) in a 15ml falcon tube and incubate it at 37°C
heat the water bath at 42°C
add 1 ul (~3ng of DNA vector) of ligation to 100ul of thawed TOP10
add parafilm and incubate in ice for 30 min
heat shock at 42°C for 1 min
incubate in ice for 2 min
transfer transformed bacteria to 800ul of pre-warmed LB
incubate at 37°C, 220 rpm for 1 h
centrifuge at 2000 rpm, 25°C for 10 min
take 750ul of supernatant and resuspend the pellet in the remaining LB (~150ul)
plate the entire culture and incubate the plate at 37°C overnight
Variants:
if you transform a miniprep, add less than 3 ng in order to have single colonies
if you use another home-made competent strain, the protocol is the same but you should consider the transformation efficiency to add a proper amount of DNA
if you use commercial Invitrogen TOP10 the protocol changes and it is reported below.
^top
Transforming commercial competent cells
(According to manufacturer's protocol)
heat ligation at 65°C to inactivate T4 ligase
thaw in ice a vial of TOP10 competent cells stored at -80°C (one vial contains 50ul of cells)
incubate a selective LB agar plate at 37°C
heat the water bath at 42°C
dilute the ligation 1:50 (or 1:100) in ddH2O, in order to have less than 100pg/ul
add 1 ul of ligation (or less than 100pg of miniprepped DNA) to 25 or 50ul of thawed TOP10
add parafilm and incubate in ice for 10 min
heat shock at 42°C for 1 min
incubate in ice for 2 min
add 250ul of SOC medium
incubate at 37°C, 220 rpm for 1 h
plate 150ul of the culture and incubate the plate at 37°C overnight
the remaining 150ul can be stored at +4°C
^top
E. coli competent cells preparation
H. Inoue et al. (1990), High efficiency transformation of Escherichia coli with plasmids, Gene 96 23-28.
DAY1
inoculum 5-8 ul from -80°C stock in 5 ml of LB (37°C, 220 rpm ON);
DAY2
dilution 1:1000 in SOB (flask, 18-25°C, 220 rpm ON);
DAY3
pre-chill centrifuge at 4°C;
prepare TB (prepare 50 ml every 125 ml of SOB):
15mM CaCl2
250mM KCl
10mM (3 g/L) Pipes
adjust pH at 6.7 with KOH
55mM (8.9 g/L) MnCl2
filter (0.2 um) the solution and chill in 50 ml
put the flask in ice when the culture reaches OD600=~0.05 (1mm pathlength – NanoDrop);
aliquot in pre-chilled 50 ml falcon tubes;
centrifuge at 2500g (4400rpm), 4°C, 10 min;
ICE: discard, resuspend in 40 ml of TB each 125 ml SOB, centrifuge as before;
ICE: discard, resuspend in 10 ml of TB each 125 ml SOB, add 700ul DMSO;
ICE: aliquot 100ul in pre-chilled 0.5ml tubes;
put in -80°C freezer;
ALWAYS TEST THE EFFICIENCY IN [CFU/ug] UNITS: transform 100ul of competent cells with 4ng of DNA and 100ul of competent cells without DNA (add 1ul of ddH2O), then plate on proper LB agar plates.
This protocol has shown to work with:
DH5alpha (10^8 with 100ul of cells);
TOP10 (5*10^7 with 100ul of cells);
BW20767 (10^3 with 100ul of cells);
DB3.1 (5*10^4 with 100ul of cells);
^top
J. Sambrook, E.F. Fritsch, T. Maniatis (1989)
DAY1
Inoculum 5-8 ul from -80°C stock in 5 ml of LB (37°C, 220 rpm ON).
DAY2
Dilution 1:500 in LB (flask, 30-37°C, 220 rpm), monitor OD600 until it reaches 0.04 (1mm pathlength – NanoDrop, it should take from 3 to 5 hours);
prepare the proper amount of 50 ml tubes in ice and pre-chill the centrifuge;
when the culture reaches the right OD600, aliquot the culture in the pre-chilled tubes;
centrifuge (4000rpm, 4°C, 10min) and discard the supernatant;
for each 50 ml of culture, add 30 ml of MgCl2-CaCl2 solution (Buffer1) and resuspend the pellet;
centrifuge (4000rpm, 4°C, 10min) and discard the supernatant;
for each 50 ml of the original culture, add 2 ml of CaCl2 solution (Buffer2) and resuspend the pellet;
aliquot in 0.5 ml tubes and store at -80°C.
ALWAYS TEST THE EFFICIENCY IN [CFU/ug] UNITS
Buffers preparation
Buffer1: 80mM MgCl2, 20mM CaCl2 (e.g.: mix 8 ml MgCl2 1M, 2 ml CaCl2 1M and 90 ml ddH2O):
put ddH2O into a flask or a bottle and autoclave it;
add MgCl2 previously filter-sterilized (0,2 um) and CaCl2 previously autoclaved or filter-sterilized (0,2 um).
Buffer2: 0.1 M CaCl2 and 15% of glycerol (e.g.: mix 100 mL of 1M CaCl2, 150 mL of 100% Glycerol and 750 mL of ddH2O):
put ddH2O and glycerol into a flask and autoclave it;
add CaCl2 previously autoclaved or filter-sterilized (0,2 um).
Keep Buffers cold.
^top
E. coli strains (all in -80°C freezer)
TOP10
F- mcrA Δ(mrr-hsdRMS-mcrBC) φ80lacZΔM15 ΔlacX74 nupG recA1 araD139 Δ(ara-leu)7697 galE15 galK16 rpsL(StrR) endA1 λ-
competent cells already prepared (5*10^7 CFU/ug with100ul of cells)
competent cells from Invitrogen available (10^9 CFU/ug with 50ul of cells)
commonly used for cloning and expression in our lab
they are equal to DH10B strain, whose genome is available from NCBI
NOTE: they have
lacI wt
cI of phi80 prophage (different from cI of lambda phage)
Streptomycin resistance
^top
DH5alpha
F- endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG Φ80dlacZΔM15 Δ(lacZYA-argF)U169, hsdR17(rK- mK+), λ–
competent cells already prepared (10^8 CFU/ug with100ul of cells)
commonly used for cloning
^top
MG1655 (seq)
F-, λ-, rph-1
CGSC#7740
Fully sequenced genome (GenBank: NC_000913)
^top
MGZ1
F-, λ-, rph-1 (identical to MG1655) with
Z1 cassette, containing constitutively expressed tetR and lacI integrated in the genome
^top
Long term bacterial glycerol stocks
Mix 750 ul of a culture (preferably in log-phase) with 250 ul of 80% glycerol, in a 1.5ml vial
label the vial with name, date and antibiotic resistance
leave at -20°C for one day
move to -80°C the day after
^top
Plasmid digestion for BioBrick Standard Assembly
To open vectors
a volume containing 1 ug of purified plasmid
2.5 ul of 10X buffer H
1 ul of first enzyme
1 ul of second enzyme
25 ul final volume
incubate at 37°C for 3 hours
To excide fragments
A volume containing 1-1.8 ug of purified plasmid
2.5 ul of buffer H
1 ul of first enzyme
1 ul of second enzyme
25 ul final volume
incubate at 37°C for 3 hours
NOTE: if you are performing a digestion for screening, 1 hour of incubation is sufficient.
^top
Ethanol precipitation with sodium acetate
Add 1/10 DNA solution volume of sodium acetate 3 M, pH 5.2
Add 2.5 DNA solution volume of absolute ethanol
Freeze at -80°C for 30 min
Centrifuge at 13000 rpm, 4°C for 20 min
Decant supernatant
Add 250 µl of 70% ethanol
Centrifuge at 13000 rpm, 4°C for 20 min
Remove all supernatant with a pipette
Air dry pellet until ethanol is totally removed
Elute with 5-10 µl of ddH2O
^top
Ligation
After the purification of two digested DNA fragments:
add a volume containing 20-50 ng of vector
add a volume containing:
File:Pv formula lig.png
("6" can be lowered to "2")
heat DNA mix at 65°C for 5 min for DNA denaturation
add 1 ul of T4 Ligase buffer (check if ATP is completely dissolved)
add 1 ul of T4 Ligase
10-20 µl final volume
incubate at 16°C overnight
inactivate the T4 Ligase heating at 65°C for 10 min
then, ligation can be conserved at 4°C or can be transformed
NOTE:   When the purified DNA of the insert also contains its native vector, you can perform the ligation anyway, but its antibiotic resistance must be different from the acceptor vector's resistance in order to select correct transformants on agar plates.   When doing this, you should modify the ligation protocol:
you should use "2" or "3" instead of "6" to compute the insert mass;
when you add the volume containing the insert mass, you must consider that the DNA quantification with NanoDrop refers to insert+NATIVE VECTOR. So, you must add:
File:Pv formula lig 2.jpeg
^top
DNA resuspension from iGEM plates
Find the right position of the DNA of interest in iGEM plates
Resuspend with 15 ul ddH2O and transfer it in 0,5 ml sterile Eppendorf tubes
Now you can store them at -20°C or transform in your favorite strain.
^top
PCR
For every DNA sample you want to amplify, put:
2 µl buffer
0.6 µl MgCl2
0.4 µl dNTPs
1 µl DNA (or ddH2O for blank sample). If you are performing a colony PCR, pick up the desired colony from a plate with a tip and dip it in the solution.
0.25 µl Taq Polymerase
0.5 ul VF2 primer (10 uM)
0.5 ul VR primer (10 uM)
A proper amount of ddH2O to have 20 µl of total reaction volume
into an eppendorf tube.
Put the Eppendorf tube in the thermal cycler and set this program:
95°C 10 min
CYCLE:
95°C 30 sec
60°C 1 min
72°C 1-3 min
for 35 cycles
72°C 7 min
16°C forever.
Now you can add a loading buffer to the solution and perform electrophoresis to check the amplified sequence length.
^top
Electrophoresis
Prepare agarose gel in 1x TBE buffer
Add ethidium bromide (using gloves and face mask for your safety):
1,5 µl in the small size agarose gel (70 ml)
3 µl in the middle size agarose gel (150 ml)
5 µl in the big size agarose gel (250 ml)
Cast the gel, insert the well-forming comb and let it polymerize
Add the loading buffer (10x Blue Juice, Invitrogen) to each sample
Load the samples and 8 µl of marker (when not specified: 1 kb Plus DNA Ladder, Fermentas)
Set to 70-100 volts and electrophorese for the required amount of time
Use UV-light to look at the bands (using gloves and protective glasses)
Take a picture of the gel, if needed (not when bands have to be cut!!!)
^top
1 kb Plus DNA Ladder preparation (Fermentas)
Mix gently:
1ul of DNA ladder (1 kb)
1ul of 6X DNA Loading Dye
4ul of Deionizied water
A final volume of 6ul to load
^top
Glycerol stocks
Mix 750 ul from the 5 ml sample of the incubated bacteria with 250 ul of 80% glycerol
Leave at -20°C for one day
Move to -80°C the day after
^top
Laboratory for Biomedical Informatics | Centre of Tissutal Engineering
University of Pavia
Retrieved from " http://2011.igem.org/Team:UNIPV-Pavia/Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
