<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UNIPV-Pavia_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UNIPV-Pavia/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UNIPV-Pavia/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UNIPV-Pavia/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UNIPV-Pavia/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>UNIPV TEAM 2011</TITLE></P><DIV id="art-main"><DIV class="art-sheet"><DIV class="art-sheet-body"><DIV class="art-nav"><DIV class="art-nav-outer"><UL class="art-hmenu"><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia"><SPAN class="t">Project</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Project/Motivation">Background &amp; Motivation</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Project/Solution">Solution</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Project/Modelling">Modelling</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Project/Results">Results</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Project/References">References</A></LI></UL><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia"><SPAN class="t">Lab</SPAN></A></LI><LI><A>Materials &amp; Methods</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Protocols">Protocols</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Instruments">Instruments</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Measurements">Measurements</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Calendar">Calendar</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Freezer">Freezer Management</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Biosafety">Biosafety</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia"><SPAN class="t">Parts</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Parts/Submitted">Submitted</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Parts/Characterized">Characterized</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Parts/DataPage">Data Page</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia"><SPAN class="t">About us</SPAN></A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Team">Team</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Pavia">Pavia</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Gallery">Gallery</A></LI><LI><A href="https://2011.igem.org/Team:UNIPV-Pavia/Sponsors">Sponsors</A></LI></DIV></DIV><DIV class="art-content-layout"><DIV class="art-content-layout-row"><DIV class="art-layout-cell art-content"><DIV class="art-post"><DIV class="art-post-body"><DIV class="art-post-inner art-article"><H2 class="art-postheader"> Protocols </H2><DIV class="art-postcontent"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Media_.26_Antibiotics"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Media &amp; Antibiotics</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#LB"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">LB</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#LB_Agar"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">LB Agar</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#SOB"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">SOB</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#SOC"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">SOC</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#M9_supplemented_with_glycerol_.28M9gly.29"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">M9 supplemented with glycerol (M9gly)</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Antibiotics"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Antibiotics</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#E._coli_transformation"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">E. coli transformation</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Transforming_home-made_competent_cells"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Transforming home-made competent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Transforming_commercial_competent_cells"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Transforming commercial competent cells</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#E._coli_competent_cells_preparation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">E. coli competent cells preparation</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#H._Inoue_et_al._.281990.29.2C_High_efficiency_transformation_of_Escherichia_coli_with_plasmids.2C_Gene_96_23-28."><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">H. Inoue et al. (1990), High efficiency transformation of Escherichia coli with plasmids, Gene 96 23-28.</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#J._Sambrook.2C_E.F._Fritsch.2C_T._Maniatis_.281989.29"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">J. Sambrook, E.F. Fritsch, T. Maniatis (1989)</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#E._coli_strains_.28all_in_-80.C2.B0C_freezer.29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">E. coli strains (all in -80°C freezer)</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#TOP10"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">TOP10</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#DH5alpha"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">DH5alpha</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#MG1655_.28seq.29"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">MG1655 (seq)</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#MG1655Z1"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">MGZ1</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Long_term_bacterial_glycerol_stocks"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Long term bacterial glycerol stocks</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Plasmid_digestion_for_BioBrick_Standard_Assembly"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Plasmid digestion for BioBrick Standard Assembly</SPAN></A></LI><LI class="toclevel-1 tocsection-21"><A href="#Ethanol_precipitation_with_sodium_acetate"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Ethanol precipitation with sodium acetate</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Ligation"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#DNA_resuspension_from_iGEM_plates"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">DNA resuspension from iGEM plates</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#PCR"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-25"><A href="#Electrophoresis"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">Electrophoresis</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#1_kb_Plus_DNA_Ladder_preparation_.28Fermentas.29"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">1 kb Plus DNA Ladder preparation (Fermentas)</SPAN></A></LI><LI class="toclevel-1 tocsection-27"><A href="#Glycerol_stocks"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">Glycerol stocks</SPAN></A></LI></TD></TR></TBODY></TABLE><DIV class="listcircle"><H1><SPAN class="mw-headline" id="Media_.26_Antibiotics">Media &amp; Antibiotics</SPAN></H1><H2><SPAN class="mw-headline" id="LB">LB</SPAN></H2><UL><LI> Add:
    </LI><LI> 10 g/L NaCl </LI><LI> 10 g/L Bacto-Tryptone </LI><LI> 5 g/L Bacto-Yeast Extract </LI><LI> ddH2O </LI></UL>
to a sterile pyrex bottle
  <UL><LI> autoclave </LI><LI> (add antibiotic when it reaches ~45°C) </LI><LI> store at +4°C </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="LB_Agar">LB Agar</SPAN></H2><UL><LI> Add:
    </LI><LI> 10 g/L NaCl </LI><LI> 10 g/L Bacto-Tryptone </LI><LI> 5 g/L Bacto-Yeast Extract </LI><LI> 15 g/L Bacto-Agar </LI><LI> ddH2O </LI></UL>
to a sterile 1L flask
  <UL><LI> autoclave </LI><LI> (add antibiotic when the agar becomes visible in the liquid, at about 45°C; shake gently to avoid bubbles) </LI><LI> pour into Petri plates and close them immediately </LI><LI> let them polymerize for ~2-3h </LI><LI> invert plates and wrap them with aluminium foil and store at +4°C </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="SOB">SOB</SPAN></H2><UL><LI> Add:
    </LI><LI> 5 g/L Bacto-Yeast Extract </LI><LI> 20 g/L Bacto-Tryptone </LI><LI> 10mM NaCl </LI><LI> 2.5mM KCl </LI><LI> 10mM MgSO4 </LI><LI> 10mM MgCl2 </LI></UL>
to a sterile pyrex bottle
  <UL><LI> (optional: check that pH is ~6.8, otherwise adjust with NaOH) </LI><LI> autoclave </LI><LI> (add antibiotic when it reaches ~45°C) </LI><LI> store at +4°C </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="SOC">SOC</SPAN></H2><UL><LI> Prepare SOB and add 20 ml of 1M glucose (prepare it dissolving 3.6 g of glucose in 20 ml of ddH2O). </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="M9_supplemented_with_glycerol_.28M9gly.29">M9 supplemented with glycerol (M9gly)</SPAN></H2><P>For 1L of medium, add: </P><UL><LI> 734 ml of autoclaved (and cooled to Tamb, N.B. consider evaporation during autoclaving) ddH2O with a magnetic stirrer inside the bottle </LI><LI> 5 ml of autoclaved 80% glycerol as carbon source </LI><LI> 100 ul of autoclaved or filtered (0.2um) CaCl2 1 M </LI><LI> 20 ml of 10% autoclaved casamino acids (dissolve 50 g in 500 ml = 10% stock) </LI><LI> 34 ml of filtered (0.2um) thiamine hydrochloride MW=337.27g/mol (340 mg in 34 ml) (keep in mind it is photosensitive) </LI><LI> 2 ml of autoclaved MgSO4 1 M </LI><LI> 200 ml of autoclaved M9 salts 5x (dissolve 56.4 g in 1 liter ddH2O = 5x stock) </LI><LI> shake the ddH2O with the magnetic stirrer and start adding the other solutions in sterility (each solution must be completely dissolved!) in the order listed above. </LI><LI> add antibiotic if needed </LI><LI> store at +4°C, protected from light </LI></UL><P>NOTE: </P><UL><LI> M9 salts 5x, 10% casamino acids, MgSO4 1 M and CaCl2 1 M can be stored at +4°C. </LI><LI> glycerol 80% can be stored at room temperature. </LI><LI> thiamine hydrochloride (LIGHT SENSITIVE) is one-shot and must be prepared each time (keep in mind you loose some volume during filtration) </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="Antibiotics">Antibiotics</SPAN></H2><P>Stocks at -20°C freezer: </P><UL><LI> Ampicillin 100 mg/ml (in water) </LI><LI> Kanamycin 50 mg/ml (in water) </LI><LI> Chloramphenicol 34 mg/ml (in 100% ethanol) </LI></UL><P>These stocks are 1000x for high copy number plasmids.
  For low copy number plasmids, you should use these final concentrations in media: </P><UL><LI> Ampicillin 50 ug/ml </LI><LI> Kanamycin 20 ug/ml </LI><LI> Chloramphenicol 12.5 ug/ml </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="E._coli_transformation">E. coli transformation</SPAN></H1><H2><SPAN class="mw-headline" id="Transforming_home-made_competent_cells">Transforming home-made competent cells</SPAN></H2><UL><LI> heat ligation at 65°C to inactivate T4 ligase </LI><LI> thaw in ice a vial of TOP10 competent cells stored at -80°C </LI><LI> incubate a selective LB agar plate at 37°C </LI><LI> pipet 800ul of LB (without antibiotic) in a 15ml falcon tube and incubate it at 37°C </LI><LI> heat the water bath at 42°C </LI></UL><UL><LI> add 1 ul (~3ng of DNA vector) of ligation to 100ul of thawed TOP10 </LI><LI> add parafilm and incubate in ice for 30 min </LI><LI> heat shock at 42°C for 1 min </LI><LI> incubate in ice for 2 min </LI><LI> transfer transformed bacteria to 800ul of pre-warmed LB </LI><LI> incubate at 37°C, 220 rpm for 1 h </LI><LI> centrifuge at 2000 rpm, 25°C for 10 min </LI><LI> take 750ul of supernatant and resuspend the pellet in the remaining LB (~150ul) </LI><LI> plate the entire culture and incubate the plate at 37°C overnight </LI></UL><P>Variants: </P><UL><LI> if you transform a miniprep, add less than 3 ng in order to have single colonies </LI><LI> if you use another home-made competent strain, the protocol is the same but you should consider the transformation efficiency to add a proper amount of DNA </LI><LI> if you use commercial Invitrogen TOP10 the protocol changes and it is reported below. </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="Transforming_commercial_competent_cells">Transforming commercial competent cells</SPAN></H2><P>(According to manufacturer's protocol) </P><UL><LI> heat ligation at 65°C to inactivate T4 ligase </LI><LI> thaw in ice a vial of TOP10 competent cells stored at -80°C (one vial contains 50ul of cells) </LI><LI> incubate a selective LB agar plate at 37°C </LI><LI> heat the water bath at 42°C </LI></UL><UL><LI> dilute the ligation 1:50 (or 1:100) in ddH2O, in order to have less than 100pg/ul </LI><LI> add 1 ul of ligation (or less than 100pg of miniprepped DNA) to 25 or 50ul of thawed TOP10 </LI><LI> add parafilm and incubate in ice for 10 min </LI><LI> heat shock at 42°C for 1 min </LI><LI> incubate in ice for 2 min </LI><LI> add 250ul of SOC medium </LI><LI> incubate at 37°C, 220 rpm for 1 h </LI><LI> plate 150ul of the culture and incubate the plate at 37°C overnight </LI><LI> the remaining 150ul can be stored at +4°C </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="E._coli_competent_cells_preparation">E. coli competent cells preparation</SPAN></H1><H2><SPAN class="mw-headline" id="H._Inoue_et_al._.281990.29.2C_High_efficiency_transformation_of_Escherichia_coli_with_plasmids.2C_Gene_96_23-28.">H. Inoue et al. (1990), High efficiency transformation of Escherichia coli with plasmids, Gene 96 23-28.</SPAN></H2><DL><DT> DAY1 </DT><DD> inoculum 5-8 ul from -80°C stock in 5 ml of LB (37°C, 220 rpm ON); </DD><DT>DAY2 </DT><DD> dilution 1:1000 in SOB (flask, 18-25°C, 220 rpm ON); </DD><DT> DAY3 </DT><DD> pre-chill centrifuge at 4°C; </DD><DD> prepare TB (prepare 50 ml every 125 ml of SOB):
    </DD><DD><UL><LI> 15mM CaCl2 </LI><LI> 250mM KCl </LI><LI> 10mM (3 g/L) Pipes </LI><LI> adjust pH at 6.7 with KOH </LI><LI> 55mM (8.9 g/L) MnCl2 </LI><LI> filter (0.2 um) the solution and chill in 50 ml </LI></UL></DD></DL><DL><DD> put the flask in ice when the culture reaches OD600=~0.05 (1mm pathlength – NanoDrop); </DD><DD> aliquot in pre-chilled 50 ml falcon tubes; </DD><DD> centrifuge at 2500g (4400rpm), 4°C, 10 min; </DD><DD> ICE: discard, resuspend in 40 ml of TB each 125 ml SOB, centrifuge as before; </DD><DD> ICE: discard, resuspend in 10 ml of TB each 125 ml SOB, add 700ul DMSO; </DD><DD> ICE: aliquot 100ul in pre-chilled 0.5ml tubes; </DD><DD> put in -80°C freezer; </DD></DL><P>ALWAYS TEST THE EFFICIENCY IN [CFU/ug] UNITS: transform 100ul of competent cells with 4ng of DNA and 100ul of competent cells without DNA (add 1ul of ddH2O), then plate on proper LB agar plates. </P><P>This protocol has shown to work with: </P><UL><LI> DH5alpha (10^8 with 100ul of cells); </LI><LI> TOP10 (5*10^7 with 100ul of cells); </LI><LI> BW20767 (10^3 with 100ul of cells); </LI><LI> DB3.1 (5*10^4 with 100ul of cells); </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="J._Sambrook.2C_E.F._Fritsch.2C_T._Maniatis_.281989.29">J. Sambrook, E.F. Fritsch, T. Maniatis (1989)</SPAN></H2><UL><LI>DAY1
    </LI><LI>Inoculum 5-8 ul from -80°C stock in 5 ml of LB (37°C, 220 rpm ON). </LI></UL><LI>DAY2
    </LI><LI>Dilution 1:500 in LB (flask, 30-37°C, 220 rpm), monitor OD600 until it reaches 0.04 (1mm pathlength – NanoDrop, it should take from 3 to 5 hours); </LI><LI>prepare the proper amount of 50 ml tubes in ice and pre-chill the centrifuge; </LI><LI>when the culture reaches the right OD600, aliquot the culture in the pre-chilled tubes; </LI><LI>centrifuge (4000rpm, 4°C, 10min) and discard the supernatant; </LI><LI>for each 50 ml of culture, add 30 ml of MgCl2-CaCl2 solution (Buffer1) and resuspend the pellet; </LI><LI>centrifuge (4000rpm, 4°C, 10min) and discard the supernatant; </LI><LI>for each 50 ml of the original culture, add 2 ml of CaCl2 solution (Buffer2) and resuspend the pellet; </LI><LI>aliquot in 0.5 ml tubes and store at -80°C. </LI><P>ALWAYS TEST THE EFFICIENCY IN [CFU/ug] UNITS </P><DL><DT>Buffers preparation </DT></DL><UL><LI>Buffer1: 80mM MgCl2, 20mM CaCl2 (e.g.: mix 8 ml MgCl2 1M, 2 ml CaCl2 1M and 90 ml ddH2O):
    </LI><LI>put ddH2O into a flask or a bottle and autoclave it; </LI><LI>add MgCl2 previously filter-sterilized (0,2 um) and CaCl2 previously autoclaved or filter-sterilized (0,2 um). </LI></UL><LI>Buffer2: 0.1 M CaCl2 and 15% of glycerol (e.g.: mix 100 mL of 1M CaCl2, 150 mL of 100% Glycerol and 750 mL of ddH2O):
    </LI><LI>put ddH2O and glycerol into a flask and autoclave it; </LI><LI>add CaCl2 previously autoclaved or filter-sterilized (0,2 um). </LI><P>Keep Buffers cold. </P><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="E._coli_strains_.28all_in_-80.C2.B0C_freezer.29">E. coli strains (all in -80°C freezer)</SPAN></H1><H2><SPAN class="mw-headline" id="TOP10">TOP10</SPAN></H2><P>F- mcrA Δ(mrr-hsdRMS-mcrBC) φ80lacZΔM15 ΔlacX74 nupG recA1 araD139 Δ(ara-leu)7697 galE15 galK16 rpsL(StrR) endA1 λ- </P><UL><LI> competent cells already prepared (5*10^7 CFU/ug with100ul of cells) </LI><LI> competent cells from Invitrogen available (10^9 CFU/ug with 50ul of cells) </LI><LI> commonly used for cloning and expression in our lab </LI><LI> they are equal to DH10B strain, whose genome is available from NCBI </LI></UL><P>NOTE: they have </P><UL><LI> lacI wt </LI><LI> cI of phi80 prophage (different from cI of lambda phage) </LI><LI> Streptomycin resistance </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="DH5alpha">DH5alpha</SPAN></H2><P>F- endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG Φ80dlacZΔM15 Δ(lacZYA-argF)U169, hsdR17(rK- mK+), λ– </P><UL><LI> competent cells already prepared (10^8 CFU/ug with100ul of cells) </LI><LI> commonly used for cloning </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="MG1655_.28seq.29">MG1655 (seq)</SPAN></H2><P>F-, λ-, rph-1 </P><UL><LI> CGSC#7740 </LI><LI> Fully sequenced genome (GenBank: NC_000913) </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="MG1655Z1">MGZ1</SPAN></H2><P>F-, λ-, rph-1 (identical to MG1655) with </P><UL><LI> Z1 cassette, containing constitutively expressed <EM>tetR</EM> and <EM>lacI</EM> integrated in the genome</LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="Long_term_bacterial_glycerol_stocks">Long term bacterial glycerol stocks</SPAN></H1><UL><LI> Mix 750 ul of a culture (preferably in log-phase) with 250 ul of 80% glycerol, in a 1.5ml vial </LI><LI> label the vial with name, date and antibiotic resistance </LI><LI> leave at -20°C for one day </LI><LI> move to -80°C the day after </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="Plasmid_digestion_for_BioBrick_Standard_Assembly">Plasmid digestion for BioBrick Standard Assembly</SPAN></H1><DL><DT> To open vectors</DT><DD><UL><LI> a volume containing 1 ug of purified plasmid </LI><LI> 2.5 ul of 10X buffer H </LI><LI> 1 ul of first enzyme </LI><LI> 1 ul of second enzyme </LI><LI> 25 ul final volume </LI><LI> incubate at 37°C for 3 hours </LI></UL></DD></DL><DL><DT> To excide fragments</DT><DD><UL><LI> A volume containing 1-1.8 ug of purified plasmid </LI><LI> 2.5 ul of buffer H </LI><LI> 1 ul of first enzyme </LI><LI> 1 ul of second enzyme </LI><LI> 25 ul final volume </LI><LI> incubate at 37°C for 3 hours </LI></UL></DD></DL><P>NOTE: if you are performing a digestion for screening, 1 hour of incubation is sufficient. </P><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="Ethanol_precipitation_with_sodium_acetate">Ethanol precipitation with sodium acetate</SPAN></H1><UL><LI> Add 1/10 DNA solution volume of sodium acetate 3 M, pH 5.2 </LI><LI> Add 2.5 DNA solution volume of absolute ethanol </LI><LI> Freeze at -80°C for 30 min </LI><LI> Centrifuge at 13000 rpm, 4°C for 20 min </LI><LI> Decant supernatant </LI><LI> Add 250 µl of 70% ethanol </LI><LI> Centrifuge at 13000 rpm, 4°C for 20 min </LI><LI> Remove all supernatant with a pipette </LI><LI> Air dry pellet until ethanol is totally removed </LI><LI> Elute with 5-10 µl of ddH2O </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H1><P>After the purification of two digested DNA fragments: </P><UL><LI> add a volume containing 20-50 ng of vector </LI><LI> add a volume containing: </LI></UL><P><A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Pv_formula_lig.png" class="new" title="File:Pv formula lig.png">File:Pv formula lig.png</A></P><P>(&quot;6&quot; can be lowered to &quot;2&quot;) </P><UL><LI> heat DNA mix at 65°C for 5 min for DNA denaturation </LI><LI> add 1 ul of T4 Ligase buffer (check if ATP is completely dissolved) </LI><LI> add 1 ul of T4 Ligase </LI><LI> 10-20 µl final volume </LI><LI> incubate at 16°C overnight </LI><LI> inactivate the T4 Ligase heating at 65°C for 10 min </LI><LI> then, ligation can be conserved at 4°C or can be transformed </LI></UL><P>NOTE:
  When the purified DNA of the insert also contains its native vector, you can perform the ligation anyway, but its antibiotic resistance must be different from the acceptor vector's resistance in order to select correct transformants on agar plates.
  When doing this, you should modify the ligation protocol: </P><UL><LI> you should use &quot;2&quot; or &quot;3&quot; instead of &quot;6&quot; to compute the insert mass; </LI><LI> when you add the volume containing the insert mass, you must consider that the DNA quantification with NanoDrop refers to insert+NATIVE VECTOR. So, you must add: </LI></UL><P><A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Pv_formula_lig_2.jpeg" class="new" title="File:Pv formula lig 2.jpeg">File:Pv formula lig 2.jpeg</A></P><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="DNA_resuspension_from_iGEM_plates">DNA resuspension from iGEM plates</SPAN></H1><UL><LI>Find the right position of the DNA of interest in iGEM plates </LI><LI>Resuspend with 15 ul ddH2O and transfer it in 0,5 ml sterile Eppendorf tubes </LI></UL><P>Now you can store them at -20°C or transform in your favorite strain. </P><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="PCR">PCR</SPAN></H1><UL><LI>For every DNA sample you want to amplify, put:
    </LI><LI>2 µl buffer </LI><LI>0.6 µl MgCl2 </LI><LI>0.4 µl dNTPs </LI><LI>1 µl DNA (or ddH2O for blank sample). If you are performing a colony PCR, pick up the desired colony from a plate with a tip and dip it in the solution. </LI><LI>0.25 µl Taq Polymerase </LI><LI>0.5 ul VF2 primer (10 uM) </LI><LI>0.5 ul VR primer (10 uM) </LI><LI>A proper amount of ddH2O to have 20 µl of total reaction volume </LI></UL><LI>into an eppendorf tube. </LI><LI>Put the Eppendorf tube in the thermal cycler and set this program:
    </LI><LI>95°C 10 min </LI><LI>CYCLE:
        </LI><LI>95°C 30 sec </LI><LI>60°C 1 min </LI><LI>72°C 1-3 min </LI><LI>for 35 cycles </LI><LI>72°C 7 min </LI><LI>16°C forever. </LI><LI>Now you can add a loading buffer to the solution and perform electrophoresis to check the amplified sequence length. </LI><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="Electrophoresis">Electrophoresis</SPAN></H1><TABLE><TBODY><TR><TD valign="top"><UL><LI>Prepare agarose gel in 1x TBE buffer </LI><LI>Add ethidium bromide (using gloves and face mask for your safety):
          </LI><LI>1,5 µl in the small size agarose gel (70 ml) </LI><LI>3 µl in the middle size agarose gel (150 ml) </LI><LI>5 µl in the big size agarose gel (250 ml) </LI></UL><LI>Cast the gel, insert the well-forming comb and let it polymerize </LI><LI>Add the loading buffer (10x Blue Juice, Invitrogen) to each sample </LI><LI>Load the samples and 8 µl of marker (when not specified: 1 kb Plus DNA Ladder, Fermentas) </LI><LI>Set to 70-100 volts and electrophorese for the required amount of time </LI><LI>Use UV-light to look at the bands (using gloves and protective glasses) </LI><LI>Take a picture of the gel, if needed (not when bands have to be cut!!!) </LI></TD></TR></TBODY></TABLE><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H2><SPAN class="mw-headline" id="1_kb_Plus_DNA_Ladder_preparation_.28Fermentas.29">1 kb Plus DNA Ladder preparation (Fermentas)</SPAN></H2><P>Mix gently: </P><UL><LI>1ul of DNA ladder (1 kb) </LI><LI>1ul of 6X DNA Loading Dye </LI><LI>4ul of Deionizied water </LI></UL><P>A final volume of 6ul to load </P><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV><H1><SPAN class="mw-headline" id="Glycerol_stocks">Glycerol stocks</SPAN></H1><UL><LI>Mix 750 ul from the 5 ml sample of the incubated bacteria with 250 ul of 80% glycerol </LI><LI>Leave at -20°C for one day </LI><LI>Move to -80°C the day after </LI></UL><DIV align="right"><SMALL><A href="#indice">^top</A></SMALL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="art-footer"><DIV class="art-footer-body"><DIV class="art-footer-text"><P><A href="http://www.labmedinfo.org" target="_blank">Laboratory for Biomedical Informatics</A> | <A href="http://cit.unipv.it/cit_it" target="_blank">Centre of Tissutal Engineering</A></P><A href="http://www.unipv.it" target="_blank">University of Pavia</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:UNIPV-Pavia/Protocols">http://2011.igem.org/Team:UNIPV-Pavia/Protocols</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UNIPV-Pavia/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UNIPV-Pavia/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UNIPV-Pavia/Protocols&amp;oldid=143985" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>