"
Page
Discussion
View source
History
teams
Log in
 
Team:UNIPV-Pavia/parts/characterized2.html
From 2011.igem.org
UNIPV TEAM 2011
Project
Background & Motivation
Solution
Modelling
Results
References
Lab
Materials & Methods
Protocols
Instruments
Measurements
Notebook
Calendar
Freezer Management
Biosafety
Parts
Submitted
Characterized
Data Page
About us
Team
Pavia
Gallery
Sponsors
Characterized Parts
Contents
1 New Parts
1.2 J23101 series with different RBSs
2 Existing parts: characterization
2.1 pTet - BBa_R0040
2.2 pLux - a 3OC6-HSL-in PoPs-out device, with less internal homology, has been characterized with mRFP
2.3 RBSs from the community collection
2.4 LuxI - BBa_C0061
2.5 AiiA - BBa_C0060
3 Existing parts: improvement
3.1 Rebuilt existing parts from the registry for DNA not available or sequence debugging
Our new parts
BBa_J23101
BBa_K516130 (wiki name: J101-E5 ) J101-RBS30-RFP-TT
BBa_K516131 (wiki name: J101-31 ) J101-RBS31-mRFP-TT
BBa_K516132 (wiki name: J101-E7 ) J101-RBS32-mRFP-TT
BBa_J23101 is the reference standard promoter for the computation of RPUs. As discussed in ' data analysis ' section, RPUs are relative units for the evaluation of promoter strength, based on a mathematical model of the transcription and the translation of a reporter gene.
The RPUs are supposed to be indepedent on the experimental setup, provided that the reference standard BBa_J23101 must be assayed in the same experimental condition of the studied promoter.
It means that if the studied promoter is in a low copy number plasmid and drives the expression of a reporter protein P, J23101 must be assembled in the same vector upstream of the same reporter P.
This approach is motivated by the assumption that the trascriptional strength of a promoter is independent on the encoded gene and on the experimental setup.
This approach is in accordance with the philosophy of synthetic biology, based on the concept of 'modularity' of the components. According to this approach, the assembly of basic well charcaterized modules to build complex circuits allow  whose behaviour can be predicted by knowledge on the basic parts.
-provide Scell with CI 95% (error bar?) and RPU?? with errorbar? -Provide RBS ranking -Discuss that RPUs can be used to compare different promoters in the same exp conditions (i.e: the same RBS), but are not reliable when comparing different RBSs with the same promoter because of the un-modularity of RBS (cit: RBS calculator Salis Nature Biot 2009, 'identical ribosome binding site sequences in different genetic contexts can result in different protein expression levels', 'It is likely that this absence of modularity is caused by the formation of strong secondary structures between the RBS-containing RNA sequence and one protein coding sequence but not another'). Highlight that we are not trying to show that RBSs are modular, since it has been shown to be false. We try to overcome the problem of modularity by considering the complex Promoter-RBS as a whole regulatory element, whose effect can't be split between the two components. Explain that here we provide a library of ready-to-use reference standard in order to easily compute RPUs of promoters with different RBSs. Discuss that it is not totally correct to compare RPUs coming from characterization with different RBSs?
Characterization of existing pars from the Registry
Do we say that we are charcterizing also F2620 or T9002? Or both?
Inducible and constitutive promoters were assembled upstream of different coding sequences containing an  RBS from the Community collection.
The assembled RBSs are:
BioBrick code
Estimated efficiency
BBa_B0030
0,6
BBa_B0031
0,07
BBa_B0032
0,3
BBa_B0034
1
For an inducible device, the RBS variation has the purpose to stretch the induction curve, thus modulating its PoPs-OUT range.
The complex RBS-promoter acts as a whole regulatory element and determines the amount of translated protein.  RBSs have been reported to have an un-modular behavior, since the translational efficiency is not independent on the coding sequences, but variates as an effect of different mRNA structure stability [Salis et al., Nat Biotec, 2009]. In addition, it is not possible to separate the effects of the sole promoter and of the sole RBS on the total amount of mRFP produced.
For this reason, every combination 'Promoter+RBS' was studied as a different regulatory element. Regulatory elements were characterized using mRFP reporter protein for different RBSs in terms of Synthesis rate per Cell (Scell) and R.P.U.s (Relative Promoter Units) as explained in measurements section.
pTet -
BBa_R0040
mRFP
BBa_K516230 (wiki name: E21 ) pTet-RBS30-mRFP-TT
BBa_K516231 (wiki name: E22 ) pTet-RBS31-mRFP-TT
BBa_K516232 (wiki name: E23 ) pTet-RBS32-mRFP-TT Rebuilt existing part from BBa_I20252 (DNA planning)
BBa_I13521 pTet-RBS34-mRFP-TT
AiiA
BBa_K516220 (wiki name: E24 ) pTet-RBS30-AiiA-TT
BBa_K516221 (wiki name: E25 ) pTet-RBS31-AiiA-TT
BBa_K516222 (wiki name: E26 ) pTet-RBS32-AiiA-TT Rebuilt existing part from BBa_ J22071 (2008 only, Bad sequencing)
BBa_K516224 (wiki name: E27 ) pTet-RBS34-AiiA-TT Rebuilt existing part from BBa_K077047 (Part deleted)
LuxI
BBa_K516210 (wiki name: E13 ) pTet-RBS30-LuxI
BBa_K516211 (wiki name: E14 ) pTet-RBS31-LuxI
BBa_K516212 (wiki name: E15 ) pTet-RBS32-LuxI
BBa_K516214 (wiki name: E16 ) pTet-RBS34-LuxI Rebuilt existing part from BBa_S03623  (DNA available, only 2008 kit, inconsistent)
The protocols for the characterization of pTet promoter are reported in the pTet measurement section .
This promoter is widely studied and characterized usually using the strong RBS BBa_B0034. Here we have characterized its transcriptional strength as a function of aTc induction (ng/ul) for different RBSs. Four different induction curves were obtained and are reported in figure: INSERIRE 4 CURVE pTET!! The data collected from the mRFP measurement systems were processed as described in data analysis section . The induction curves were obtained by fitting a Hill function as described in modelling section and the estimated parameters for pTet are reported in the table below.
RBS
αpTet
δpTet
ηpTet
kpTet
BBa_B0030
215
0.001
4.3
8.3
BBa_B0031
0.7
0
18.84
47.51
BBa_B0032
45.7
0.02
83
8
BBa_B0034
125
0.12
71.8
9.8
Their CV% (coefficient of variation):
RBS
αpTet
δpTet
ηpTet
kpTet
BBa_B0030
2.13
888
12
2.54
BBa_B0031
imm
1.47*10^11
imm
imm
BBa_B0032
10.88
411.3
676671474
393179
BBa_B0034
12.22
61.74
2705172
55888.6
While α parameter (representing the maximum trascriptional rate in the studied range of induction) varies as expected with the RBS variation , the K and η parameters (determining the switch-point of the induction curve) are quite constant among all the RBS variations.
This suggests that the RBS variation only modulates the amplitude of the induction curve, but doesn't affect the shape, i.e. the translational promoter activity.  These results are quite encouraging, because suggest that, given the non-modular behavior of RBS dpending on the encoded gene, the RBS has a modular behaviour respect to the promoter.
Do we need to insert a table with the most significant measurements for every pTet-RBSx (i.e.: min RPU, max RPU, switch point, saturation threshold?)
pLux
BBa_K516330 (wiki name: E17 ) pLambda-RBS30-LuxR-T-pLux-RBS30-mRFP-TT
BBa_K516331 (wiki name: E18 ) pLambda-RBS30-LuxR-T-pLux-RBS31-mRFP-TT
BBa_K516332 (wiki name: E19 ) pLambda-RBS30-LuxR-T-pLux-RBS32-mRFP-TT
BBa_K516334 (wiki name: E20 ) pLambda-RBS30-LuxR-T-pLux-RBS34-mRFP-TT
RBS
αpLux
δpLux
ηpLux
kpLux
BBa_B0030
438
0.05
2
1.88
BBa_B0031
9.8
0.11
1.2
1.5
BBa_B0032
206
0
1.36
1.87
BBa_B0034
1105
0.02
1.33
2.34
Their CV% (coefficient of variation):
RBS
αpLux
δpLux
ηpLux
kpLux
BBa_B0030
10.14
180
47.73
27.5
BBa_B0031
7.12
57.04
29.13
25.82
BBa_B0032
2.78
1318
9.75
8.46
BBa_B0034
5.81
187.2
19.3
17.86
RBSs
BBa_B0030
BBa_K516030 (wiki name: E5 ) RBS30-mRFP-TT Rebuilt existing part from BBa_S04180 (DNA planning)
BBa_K516230 (wiki name: E21 ) pTet-RBS30-mRFP-TT
BBa_K516330 (wiki name: E17 ) pLambda-RBS30-LuxR-T-pLux-RBS30-mRFP-TT
BBa_K516130 (wiki name: J101-E5 ) J101-RBS30-RFP-TT
BBa_K516020 (wiki name: E9 ) RBS30-AiiA-TT
BBa_K516220 (wiki name: E24 ) pTet-RBS30-AiiA-TT
BBa_K081008 (wiki name: E2 ) RBS30-LuxI
BBa_K516210 (wiki name: E13 ) pTet-RBS30-LuxI
BBa_B0031
BBa_K516031 (wiki name: E6 ) RBS31-mRFP-TT
BBa_K516231 (wiki name: E22 ) pTet-RBS31-mRFP-TT
BBa_K516331 (wiki name: E18 ) pLambda-RBS30-LuxR-T-pLux-RBS31-mRFP-TT
BBa_K516131 (wiki name: J101-31 ) J101-RBS31-RFP-TT
BBa_K516021 (wiki name: E10 ) RBS31-AiiA-TT Rebuilt existing part from BBa_I13914  (DNA planning)
BBa_K516221 (wiki name: E25 ) pTet-RBS31-AiiA-TT
BBa_K516011 (wiki name: E3 ) RBS31-LuxI
BBa_K516211 (wiki name: E14 ) pTet-RBS31-LuxI
BBa_B0032
BBa_K516032 (wiki name: E7 ) RBS32-mRFP-TT Rebuilt existing part from BBa_ J133000 (DNA planning)
BBa_K516232 (wiki name: E23 ) pTet-RBS32-mRFP-TT Rebuilt existing part from BBa_I20252 (DNA planning)
BBa_K516332 (wiki name: E19 ) pLambda-RBS30-LuxR-T-pLux-RBS32-mRFP-TT
BBa_K516132 (wiki name: J101-E7 ) J101-RBS32-RFP-TT
BBa_K516022 (wiki name: E11 ) RBS32-AiiA-TT Rebuilt existing part from BBa_I13912 (DNA planning)
BBa_K516222 (wiki name: E26 ) pTet-RBS32-AiiA-TT Rebuilt existing part from BBa_ J22071 (2008 only, Bad sequencing)
BBa_K516012 (wiki name: E4 ) RBS32-LuxI
BBa_K516212 (wiki name: E15 ) pTet-RBS32-LuxI
BBa_B0034
BBa_I13521 RBS34-mRFP-TT
BBa_I13521 pTet-RBS34-mRFP-TT
BBa_K516334 (wiki name: E20 ) pLambda-RBS30-LuxR-T-pLux-RBS34-mRFP-TT
BBa_J23101 J101-RBS34-RFP-TT
BBa_I0460 RBS34-AiiA-TT
BBa_K516224 (wiki name: E27 ) pTet-RBS34-AiiA-TT Rebuilt existing part from BBa_K077047 (Part deleted)
BBa_C0261 RBS34-LuxI
BBa_K516214 (wiki name: E16 ) pTet-RBS34-LuxI Rebuilt existing part from BBa_S03623  (DNA available, only 2008 kit, inconsistent)
Almost everything must be inserted in this section!! How do I evaluate the efficiency of RBSs in case of inducible promoters? We need a summary table with the estimated efficiency in the condition:
-Promoter - Gene
LuxI -
BBa_C0061
BBa_K081008 (wiki name: E2 ) RBS30-LuxI
BBa_K516210 (wiki name: E13 ) pTet-RBS30-LuxI
BBa_K516011 (wiki name: E3 ) RBS31-LuxI
BBa_K516211 (wiki name: E14 ) pTet-RBS31-LuxI
BBa_K516012 (wiki name: E4 ) RBS32-LuxI
BBa_K516212 (wiki name: E15 ) pTet-RBS32-LuxI
BBa_C0261 RBS34-LuxI
BBa_K516214 (wiki name: E16 ) pTet-RBS34-LuxI Rebuilt existing part from BBa_S03623  (DNA available, only 2008 kit, inconsistent)
LuxI has been characterized through the Biosensor T9002 (see modeling section). The HSL synthesis rate has been evaluated according to the model (Eq. 2 - dLuxI/dt, Eq. 3 dHSL/dt, Eq. 5 dN/dt). Explain that in all these evaluations we have estimated Nmax, mu, gamma_HSL from previous experiments - see measurement and modelling sections. When do we talk about HSL stability as a function of pH?
Provide parameters of HSL. How do we present these data? It would be nice also to say what's the amount of HSL produced by a liquid 5 ml culture at a given OD600 in M9 medium after tot hours starting from 1:1000 dilution of a saturated ON culture..
We nee a synthetic parameter to express LuxI activity as a function of PoPS in. I suggest to report the HSL vs pH analysis here AND in the AiiA section of registry and, for what concerns our wiki, to add an appendix to the 'measurement section' to wich link when explaining..
Decide figures!
AiiA -
BBa_C0060
BBa_K516020 (wiki name: E9 ) RBS30-AiiA-TT
BBa_K516220 (wiki name: E24 ) pTet-RBS30-AiiA-TT
BBa_K516021 (wiki name: E10 ) RBS31-AiiA-TT Rebuilt existing part from BBa_I13914  (DNA planning)
BBa_K516221 (wiki name: E25 ) pTet-RBS31-AiiA-TT
BBa_K516022 (wiki name: E11 ) RBS32-AiiA-TT Rebuilt existing part from BBa_I13912 (DNA planning)
BBa_K516222 (wiki name: E26 ) pTet-RBS32-AiiA-TT Rebuilt existing part from BBa_ J22071 (2008 only, Bad sequencing)
BBa_I0460 RBS34-AiiA-TT
BBa_K516224 (wiki name: E27 ) pTet-RBS34-AiiA-TT Rebuilt existing part from BBa_K077047 (Part deleted)
Tha same as LuxI. Here we report the AiiA equations. Even more fundamental the estimation of gamma_HSL, that is pH dependent, because of the necessity to depurate the observed degradation by the effect not depending on AiiA
Decide figures!
Improvement of existing pars from the Registry
Add the re-built existing parts with DNA planning!
Rebuilt existing parts
BBa_K516021 (wiki name: E10 ) RBS31-AiiA-TT Rebuilt existing part from BBa_I13914  (DNA planning)
BBa_K516022 (wiki name: E11 ) RBS32-AiiA-TT Rebuilt existing part from BBa_I13912 (DNA planning)
BBa_K516030 (wiki name: E5 ) RBS30-mRFP-TT Rebuilt existing part from BBa_S04180 (DNA planning)
BBa_K516032 (wiki name: E7 ) RBS32-mRFP-TT Rebuilt existing part from BBa_ J133000 (DNA planning)
BBa_K516214 (wiki name: E16 ) pTet-RBS34-LuxI Rebuilt existing part from BBa_S03623  (DNA available, only 2008 kit, inconsistent)
BBa_K516222 (wiki name: E26 ) pTet-RBS32-AiiA-TT Rebuilt existing part from BBa_ J22071 (2008 only, Bad sequencing)
BBa_K516224 (wiki name: E27 ) pTet-RBS34-AiiA-TT Rebuilt existing part from BBa_K077047 (Part deleted)
BBa_K516232 (wiki name: E23 ) pTet-RBS32-mRFP-TT Rebuilt existing part from BBa_I20252 (DNA planning)
Laboratory for Biomedical Informatics | Centre of Tissutal Engineering
University of Pavia
Retrieved from " http://2011.igem.org/Team:UNIPV-Pavia/parts/characterized2.html "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
