<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Project_overview"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Project_overview">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Project_overview&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Project_overview&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Project_overview&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Project_overview" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Project overview</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="eth_page"><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><STRONG class="selflink">Project</STRONG></LI><LI><STRONG class="selflink">Overview</STRONG></LI><LI><A href="/Team:ETH_Zurich/Motivation" title="Team:ETH Zurich/Motivation">Motivation</A></LI><LI><A href="/Team:ETH_Zurich/Applications" title="Team:ETH Zurich/Applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/Interplay" title="Team:ETH Zurich/Interplay">Lab &amp; Modelling</A></LI></UL><LI><A href="/Team:ETH_Zurich/Project" title="Team:ETH Zurich/Project">Parts</A></LI><LI><A href="/Team:ETH_Zurich/Project" title="Team:ETH Zurich/Project">Overview</A></LI><LI><A href="/Team:ETH_Zurich/UVR8" title="Team:ETH Zurich/UVR8">Direct: UVR8-TetR<SUB>DBD</SUB></A></LI><LI><A href="/Team:ETH_Zurich/Decoder" title="Team:ETH Zurich/Decoder">Indirect: Decoder</A></LI><LI><A href="/Team:ETH_Zurich/PABA" title="Team:ETH Zurich/PABA">Output: PABA</A></LI><LI><A href="/Team:ETH_Zurich/Parts" title="Team:ETH Zurich/Parts">Parts</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Photoinduction" title="Team:ETH Zurich/Modeling/Photoinduction">Photoinduction</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/UVR8" title="Team:ETH Zurich/Modeling/UVR8">UVR8-TetR<SUB>DBD</SUB> Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SPF_model" title="Team:ETH Zurich/Modeling/SPF model">Sun Protection Factor</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Construct2" title="Team:ETH Zurich/Modeling/Construct2">Decoder Circuit: LovTAP/Cph8</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters" title="Team:ETH Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Notebook</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Overview</A></LI><LI><A href="/Team:ETH_Zurich/MaterialMethods" title="Team:ETH Zurich/MaterialMethods">Material &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements_overall" title="Team:ETH Zurich/Achievements overall">Project Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/HumanPractice" title="Team:ETH Zurich/HumanPractice">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Team description</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Project_overview" class="external text" rel="nofollow">Login</A></LI><LI><A href="http://2012.igem.org/Team:ETH_Zurich/Project_overview" class="external text" rel="nofollow">WIKI</A></LI><LI><A href="http://2012.igem.org/Team:ETH_Zurich/Project_overview" class="external text" rel="nofollow">View page</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Team:ETH_Zurich/Project_overview&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Team:ETH_Zurich/Project_overview&amp;action=history" class="external text" rel="nofollow">History of this page</A></LI><LI><A href="http://2012.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#E.Colipse_-_intelligent_sunscreen"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">E.Colipse - intelligent sunscreen</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Who.27s_your_PABA.3F"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Who's your PABA?</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Direct_UV_detection:_a_novel_UV-B_light_receptor_in_E.coli"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Direct UV detection: a novel UV-B light receptor in E.coli</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Indirect_UV_detection:_decoding_of_sun_light_from_its_light_spectrum"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Indirect UV detection: decoding of sun light from its light spectrum</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="E.Colipse_-_intelligent_sunscreen"><I>E.Colipse</I> - intelligent sunscreen</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><I>Figure 1</I>: Spectrum of the sun. Dotted lines indicate the borders between different light types denoted with the labels at the top. 'A', 'B' and 'C' show the regions for UV-A/B/C radiation.</DIV></DIV></DIV><P>Sun UV radiation is unavoidable during warm sunny days. Despite the fact that suns light (Fig. 1) reaching Earth crust consists of only few percent of UV-A (320 - 400 nm) and UV-B (280 - 320 nm) radiation, it still poses serious risks and life destructive capabilities, e.g. DNA is one of the main targets which are hit by UV radiation, causing DNA breakage, mutations etc. <SPAN class="eth_reference">[Sinha2002]</SPAN> If unattended, this can lead to sun burns, skin damage or even cancer. Thus, a proper skin protection is usually a good idea in order to avoid unpleasant experience.
</P><P>However, conventional sunscreens provide relatively passive sun protection, leaving actual protection regulation to ones speculations when and how much of the sunscreen should be applied, which easily can result in inefficient layer of sunscreen and skin damage.
</P><P>Thus, we tried to <B>develop an intelligent sunscreen, which could adapt its protection factor to the exposed UV radiation and also give a warning signal if protection is insufficient.</B></P><H3><SPAN class="mw-headline" id="Who.27s_your_PABA.3F"> Who's your PABA?</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:102px;"><DIV class="thumbcaption"><I>Figure 2</I>: 4-para-aminobenzoic acid, natural occuring UV light absorbing molecule used in sunscreens</DIV></DIV></DIV><P>For our sunscreen we chose <I>E. coli</I> as our primer chassis, with a later idea to transfer a working system to a natural bacterial <A href="https://2012.igem.org/Team:ETH_Zurich/Applications" class="external text" rel="nofollow">flora of the skin or alginate beads</A>. As a UV-B light absorbing substance, <A href="https://2012.igem.org/Team:ETH_Zurich/PABA" class="external text" rel="nofollow">4-para-aminobenzoic acid</A> (PABA, Fig. 2) was chosen for the following reasons: i) PABA is already used in sunscreens for UV-B protection <SPAN class="eth_reference">[Sambandan2011]</SPAN>; ii) PABA is a natural occurring compound as an intermediate in folate synthesis pathway. Furthermore, overproduction of PABA requires overexpression of just three key enzymes: pabA, pabB, pabC, and several attempts were made to overproduce PABA in E.coli by other iGEM teams <SPAN class="eth_reference">[Wegkamp2007]</SPAN>. Thus we used pabC enzyme from the biobrick partregistry and also cloned both, <A href="https://2012.igem.org/Team:ETH_Zurich/PABA/Design" class="external text" rel="nofollow">pabA and pabB, into one operon</A> under the constitutive promoter aiming to <A href="https://2012.igem.org/Team:ETH_Zurich/PABA/Results" class="external text" rel="nofollow">increase the PABA concentration</A> in the cells. Furthermore, we are not limited to PABA and other natural light absorbance molecules, such as mycosporine-like amino acids, might be used for our final system <SPAN class="eth_reference">[Gao2011]</SPAN>. 
</P><H3><SPAN class="mw-headline" id="Direct_UV_detection:_a_novel_UV-B_light_receptor_in_E.coli"> Direct UV detection: a novel UV-B light receptor in <I>E.coli</I></SPAN></H3><P>To couple PABA synthesis to UV light we investigated <I>E.coli</I> response to UV radiation. However, in prokaryotes such response is activated indirectly by sensing DNA damage <SPAN class="eth_reference">[Goosen2008]</SPAN>. Such protection strategy would be erroneous to pursue, because induction of protection is initiated after DNA damage is already happened. Thus we used plant UV-B responsive protein <A href="https://2012.igem.org/Team:ETH_Zurich/UVR8" class="external text" rel="nofollow">UVR8</A> to design a novel UV-B receptor in <I>E. coli</I>. 
</P><P>In the dark state, <A href="https://2012.igem.org/Team:ETH_Zurich/UVR8" class="external text" rel="nofollow">UVR8</A> forms a dimer, which monomerizes upon UV-B irradiation and induces transcriptional changes necessary for plant UV response <SPAN class="eth_reference">[Heijde2012]</SPAN>. However, UVR8 is not able to bind DNA, thus we fused it with a DNA binding domain of tetracycline repressor protein (<A href="http://partsregistry.org/Part:BBa_K909007" class="external text" rel="nofollow">TetR<SUB>DBD</SUB></A>). TetR<SUB>DBD</SUB> does not contain dimerization domain required for efficient binding and repression of TetR responsive promoter P<SUB>tet</SUB>. However, external dimerization would in principle restore TetR<SUB>DBD</SUB> ability to repress P<SUB>tet</SUB>. Thus, our chimeric protein <A href="https://2012.igem.org/Team:ETH_Zurich/UVR8/Design" class="external text" rel="nofollow">UVR8-TetR<SUB>DBD</SUB></A> acts as an light inducible transcription repressor â at dark state, UVR8 dimer brings TetR<SUB>DBD</SUB> at close proximity to bind P<SUB>tet</SUB>, whereas UVR8 monomerization upon UV-B light releases UVR8-TetR<SUB>DBD</SUB> from DNA, thus inducing transcription of the downstream genes (Fig. 3). 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I>Figure 3</I>: UVR8-TetR<SUB>DBD</SUB> working mechanism: in dark state UVR8-TetR<SUB>DBD</SUB> is a dimer and represses P<SUB>tet</SUB>, while under exposure of UV-B light UVR8-TetR<SUB>DBD</SUB> releases DNA and allows transcription of a downstream genes.</DIV></DIV></DIV></DIV><P>
Our <A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/UVR8" class="external text" rel="nofollow">models</A> suggest that such system would efficiently induce production of PABA at the right time scale, giving broad possibilities for such novel UV-B switch. Also, <A href="https://2012.igem.org/Team:ETH_Zurich/UVR8/Results" class="external text" rel="nofollow">our data</A> shows that repression of P<SUB>tet</SUB> is indeed restored for TetR<SUB>DBD</SUB>. Nevertheless, in order not to fully rely on newly introduced plant protein to <I>E.coli</I>, a second approach of indirect UV sensing was also investigated. 
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:132px;"><DIV class="thumbcaption"><I>Figure 4</I>: NOR gates: Boolean NOR gate representation (upper) and genomic NOR gate - promoter regulated by two repressors (bottom)</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Indirect_UV_detection:_decoding_of_sun_light_from_its_light_spectrum"> Indirect UV detection: decoding of sun light from its light spectrum</SPAN></H3><P>Each light source has its characteristic light emission spectrum e.g. sun contains high red and blue light content compared to artificial light sources, such as tungsten light bulb have just high red and a very small fraction of blue light. Thus, we design <A href="https://2012.igem.org/Team:ETH_Zurich/Decoder" class="external text" rel="nofollow">a biological circuit</A> which can distinguish between four different light conditions: i) high red-high blue; ii) high red-low blue; iii) low red-high blue and iv) low red-low blue (Fig. 5). Such circuit consists of <A href="https://2012.igem.org/Team:ETH_Zurich/Decoder/Design" class="external text" rel="nofollow">three hybrid promoters</A> each acting as a NOR gate for two different transcriptional repressors i.e. transcription from such promoter is activated just when both repressors are absent. 
</P><P>We developed a novel <A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Photoinduction" class="external text" rel="nofollow">photoinduction model</A> to select for a suitable light receptors for the system inputs: <A href="https://2012.igem.org/Team:ETH_Zurich/LovTAP" class="external text" rel="nofollow">LovTAP</A> (blue light receptor) and <A href="https://2012.igem.org/Team:ETH_Zurich/Cph8" class="external text" rel="nofollow">Cph8</A> (Red light receptor). Later analysis of the circuit network revealed several <A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Construct2" class="external text" rel="nofollow">kinetic parameters constrains</A> upon which hybrid promoters were chosen and synthesized. Further testing of the promoters showed that as predicted, they act as a NOR gate, giving high potential of building a first full working light regulated biological decoder circuit in cells.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:702px;"><DIV class="thumbcaption"><I>Figure 5</I>: Boolean logic of Decoder.</DIV></DIV></DIV></DIV><DIV id="eth_references"><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><UL><LI id="Brown2009">Brown, B. a, Headland, L. R., &amp; Jenkins, G. I. (2009). UV-B action spectrum for UVR8-mediated HY5 transcript accumulation in Arabidopsis. Photochemistry and photobiology, 85(5), 1147â55.</LI><LI id="Christie1999">Christie, J. M., Salomon, M., Nozue, K., Wada, M., &amp; Briggs, W. R. (1999): LOV (light, oxygen, or voltage) domains of the blue-light photoreceptor phototropin (nph1): binding sites for the chromophore flavin mononucleotide. Proceedings of the National Academy of Sciences of the United States of America, 96(15), 8779â83.</LI><LI id="Christie2012">Christie, J. M., Arvai, A. S., Baxter, K. J., Heilmann, M., Pratt, A. J., OâHara, A., Kelly, S. M., et al. (2012). Plant UVR8 photoreceptor senses UV-B by tryptophan-mediated disruption of cross-dimer salt bridges. Science (New York, N.Y.), 335(6075), 1492â6.</LI><LI id="Cloix2008">Cloix, C., &amp; Jenkins, G. I. (2008). Interaction of the Arabidopsis UV-B-specific signaling component UVR8 with chromatin. Molecular plant, 1(1), 118â28.</LI><LI id="Cox2007">Cox, R. S., Surette, M. G., &amp; Elowitz, M. B. (2007). Programming gene expression with combinatorial promoters. Molecular systems biology, 3(145), 145. doi:10.1038/msb4100187</LI><LI id="Drepper2007">Drepper, T., Eggert, T., Circolone, F., Heck, A., Krauss, U., Guterl, J.-K., Wendorff, M., et al. (2007). Reporter proteins for in vivo fluorescence without oxygen. Nature biotechnology, 25(4), 443â5</LI><LI id="Drepper2011">Drepper, T., Krauss, U., &amp; Berstenhorst, S. M. zu. (2011). Lights on and action! Controlling microbial gene expression by light. Applied microbiology, 23â40.</LI><LI id="EC2006">EuropeanCommission (2006). SCIENTIFIC COMMITTEE ON CONSUMER PRODUCTS SCCP Opinion on Biological effects of ultraviolet radiation relevant to health with particular reference to sunbeds for cosmetic purposes.</LI><LI id="Elvidge2010">Elvidge, C. D., Keith, D. M., Tuttle, B. T., &amp; Baugh, K. E. (2010). Spectral identification of lighting type and character. Sensors (Basel, Switzerland), 10(4), 3961â88.</LI><LI id="GarciaOjalvo2004">GarciaOjalvo, J., Elowitz, M. B., &amp; Strogatz, S. H. (2004). Modeling a synthetic multicellular clock: repressilators coupled by quorum sensing. Proceedings of the National Academy of Sciences of the United States of America, 101(30), 10955â60.</LI><LI id="Gao2011">Gao Q, Garcia-Pichel F. (2011). Microbial ultraviolet sunscreens. Nat Rev Microbiol. 9(11):791-802. </LI><LI id="Goosen2008">Goosen N, Moolenaar GF. (2008) Repair of UV damage in bacteria. DNA Repair (Amst).7(3):353-79.</LI><LI id="Heijde2012">Heijde, M., &amp; Ulm, R. (2012). UV-B photoreceptor-mediated signalling in plants. Trends in plant science, 17(4), 230â7.</LI><LI id="Hirose2010">Hirose, Y., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2010). Cyanobacteriochrome CcaS regulates phycoerythrin accumulation in Nostoc punctiforme, a group II chromatic adapter. Proceedings of the National Academy of Sciences of the United States of America, 107(19), 8854â9.</LI><LI id="Hirose2008">Hirose, Y., Shimada, T., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2008). Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein. Proceedings of the National Academy of Sciences of the United States of America, 105(28), 9528â33.</LI><LI id="Kast1996"> Kast, Asif-Ullah &amp; Hilvert (1996) Tetrahedron Lett. 37, 2691 - 2694., Kast, Asif-Ullah, Jiang &amp; Hilvert (1996) Proc. Natl. Acad. Sci. USA 93, 5043  - 5048 </LI><LI id="Kiefer2010">Kiefer, J., Ebel, N., SchlÃ¼cker, E., &amp; Leipertz, A. (2010). Characterization of Escherichia coli suspensions using UV/Vis/NIR absorption spectroscopy. Analytical Methods, 9660. doi:10.1039/b9ay00185a</LI><LI id="Kinkhabwala2008">Kinkhabwala, A., &amp; Guet, C. C. (2008). Uncovering cis regulatory codes using synthetic promoter shuffling. PloS one, 3(4), e2030.</LI><LI id="RKI2010"> Krebs in Deutschland 2005/2006. HÃ¤ufigkeiten und Trends. 7. Auflage, 2010, Robert Koch-Institut (Hrsg) und die Gesellschaft der epidemiologischen Krebsregister in Deutschland e. V. (Hrsg). Berlin. </LI><LI id="Lamparter2002">Lamparter, T., Michael, N., Mittmann, F., &amp; Esteban, B. (2002). Phytochrome from Agrobacterium tumefaciens has unusual spectral properties and reveals an N-terminal chromophore attachment site. Proceedings of the National Academy of Sciences of the United States of America, 99(18), 11628â33.</LI><LI id="Levskaya2005">Levskaya, A. et al (2005). Engineering Escherichia coli to see light. Nature, 438(7067), 442.</LI><LI id="Mancinelli1986">Mancinelli, A. (1986). Comparison of spectral properties of phytochromes from different preparations. Plant physiology, 82(4), 956â61. </LI><LI id="Nakasone2007">Nakasone, Y., Ono, T., Ishii, A., Masuda, S., &amp; Terazima, M. (2007). Transient dimerization and conformational change of a BLUF protein: YcgF. Journal of the American Chemical Society, 129(22), 7028â35.</LI><LI id="Orth2000">Orth, P., &amp; Schnappinger, D. (2000). Structural basis of gene regulation by the tetracycline inducible Tet repressor-operator system. Nature structural biology, 215â219.</LI><LI id="Parkin2005"> Parkin, D.M., et al., Global cancer statistics, 2002. CA: a cancer journal for clinicians, 2005. 55(2): p. 74-108. </LI><LI id="Rajagopal2004">Rajagopal, S., Key, J. M., Purcell, E. B., Boerema, D. J., &amp; Moffat, K. (2004). Purification and initial characterization of a putative blue light-regulated phosphodiesterase from Escherichia coli. Photochemistry and photobiology, 80(3), 542â7.</LI><LI id="Rizzini2011">Rizzini, L., Favory, J.-J., Cloix, C., Faggionato, D., OâHara, A., Kaiserli, E., Baumeister, R., et al. (2011). Perception of UV-B by the Arabidopsis UVR8 protein. Science (New York, N.Y.), 332(6025), 103â6.</LI><LI id="Roux1992">Roux, B., &amp; Walsh, C. T. (1992). p-aminobenzoate synthesis in Escherichia coli: kinetic and mechanistic characterization of the amidotransferase PabA. Biochemistry, 31(30), 6904â10.</LI><LI id="Strickland2008">Strickland, D. (2008). Light-activated DNA binding in a designed allosteric protein. Proceedings of the National Academy of Sciences of the United States of America, 105(31), 10709â10714.</LI><LI id="Sinha2002"> Sinha RP, HÃ¤der DP. UV-induced DNA damage and repair: a review. Photochem Photobiol Sci. (2002). 1(4):225-36 </LI><LI id="Sambandan2011">Sambandan DR, Ratner D. (2011). Sunscreens: an overview and update. J Am Acad Dermatol. 2011 Apr;64(4):748-58.</LI><LI id="Tabor2011">Tabor, J. J., Levskaya, A., &amp; Voigt, C. A. (2011). Multichromatic Control of Gene Expression in Escherichia coli. Journal of Molecular Biology, 405(2), 315â324.</LI><LI id="Thibodeaux2009">Thibodeaux, G., &amp; Cowmeadow, R. (2009). A tetracycline repressor-based mammalian two-hybrid system to detect proteinâprotein interactions in vivo. Analytical biochemistry, 386(1), 129â131.</LI><LI id="Tschowri2009">Tschowri, N., &amp; Busse, S. (2009). The BLUF-EAL protein YcgF acts as a direct anti-repressor in a blue-light response of Escherichia coli. Genes &amp; development, 522â534.</LI><LI id="Tschowri2012">Tschowri, N., Lindenberg, S., &amp; Hengge, R. (2012). Molecular function and potential evolution of the biofilm-modulating blue light-signalling pathway of Escherichia coli. Molecular microbiology.</LI><LI id="Tyagi2009">Tyagi, A. (2009). Photodynamics of a flavin based blue-light regulated phosphodiesterase protein and its photoreceptor BLUF domain.</LI><LI id="Vainio2001">Vainio, H. &amp; Bianchini, F. (2001). IARC Handbooks of Cancer Prevention: Volume 5: Sunscreens. Oxford University Press, USA</LI><LI id="Quinlivan2003">Quinlivan, Eoin P &amp; Roje, Sanja &amp; Basset, Gilles &amp; Shachar-Hill, Yair &amp; Gregory, Jesse F &amp; Hanson, Andrew D. (2003). The folate precursor p-aminobenzoate is reversibly converted to its glucose ester in the plant cytosol. The Journal of biological chemistry, 278.</LI><LI id="VanThor2001">van Thor, J. J., Borucki, B., Crielaard, W., Otto, H., Lamparter, T., Hughes, J., Hellingwerf, K. J., et al. (2001). Light-induced proton release and proton uptake reactions in the cyanobacterial phytochrome Cph1. Biochemistry, 40(38), 11460â71.</LI><LI id="Wegkamp2007">Wegkamp A, van Oorschot W, de Vos WM, Smid EJ. (2007 )Characterization of the role of para-aminobenzoic acid biosynthesis in folate production by Lactococcus lactis. Appl Environ Microbiol. Apr;73(8):2673-81.</LI></UL></DIV></DIV><DIV id="eth_footer"><DIV id="eth_sponsors"><TABLE><CAPTION><B>We want to thank our sponsors:</B></CAPTION><TBODY><TR><TD><DIV class="center"><DIV class="floatnone"><A href="/File:ETH_sponsor_kgf.gif" title="File:ETH sponsor kgf.gif">180px</A></DIV></DIV></TD></TR><TR><TD><A href="/File:ETH_BSSE.gif" title="File:ETH BSSE.gif">100px</A></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:ETH_Zurich/Project_overview">http://2012.igem.org/Team:ETH_Zurich/Project_overview</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Project_overview" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Project_overview" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Project_overview&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Project_overview&amp;oldid=295747" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>