<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_MaterialMethods"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/MaterialMethods">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/MaterialMethods&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/MaterialMethods&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/MaterialMethods&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/MaterialMethods" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/MaterialMethods</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="eth_page"><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Project_overview" title="Team:ETH Zurich/Project overview">Project</A></LI><LI><A href="/Team:ETH_Zurich/Project_overview" title="Team:ETH Zurich/Project overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Motivation" title="Team:ETH Zurich/Motivation">Motivation</A></LI><LI><A href="/Team:ETH_Zurich/Applications" title="Team:ETH Zurich/Applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/Interplay" title="Team:ETH Zurich/Interplay">Lab &amp; Modelling</A></LI></UL><LI><A href="/Team:ETH_Zurich/Project" title="Team:ETH Zurich/Project">Parts</A></LI><LI><A href="/Team:ETH_Zurich/Project" title="Team:ETH Zurich/Project">Overview</A></LI><LI><A href="/Team:ETH_Zurich/UVR8" title="Team:ETH Zurich/UVR8">Direct: UVR8-TetR<SUB>DBD</SUB></A></LI><LI><A href="/Team:ETH_Zurich/Decoder" title="Team:ETH Zurich/Decoder">Indirect: Decoder</A></LI><LI><A href="/Team:ETH_Zurich/PABA" title="Team:ETH Zurich/PABA">Output: PABA</A></LI><LI><A href="/Team:ETH_Zurich/Parts" title="Team:ETH Zurich/Parts">Parts</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Photoinduction" title="Team:ETH Zurich/Modeling/Photoinduction">Photoinduction</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/UVR8" title="Team:ETH Zurich/Modeling/UVR8">UVR8-TetR<SUB>DBD</SUB> Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SPF_model" title="Team:ETH Zurich/Modeling/SPF model">Sun Protection Factor</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Construct2" title="Team:ETH Zurich/Modeling/Construct2">Decoder Circuit: LovTAP/Cph8</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters" title="Team:ETH Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Notebook</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Overview</A></LI><LI><STRONG class="selflink">Material &amp; Methods</STRONG></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements_overall" title="Team:ETH Zurich/Achievements overall">Project Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/HumanPractice" title="Team:ETH Zurich/HumanPractice">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Team description</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/MaterialMethods" class="external text" rel="nofollow">Login</A></LI><LI><A href="http://2012.igem.org/Team:ETH_Zurich/MaterialMethods" class="external text" rel="nofollow">WIKI</A></LI><LI><A href="http://2012.igem.org/Team:ETH_Zurich/MaterialMethods" class="external text" rel="nofollow">View page</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Team:ETH_Zurich/MaterialMethods&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Team:ETH_Zurich/MaterialMethods&amp;action=history" class="external text" rel="nofollow">History of this page</A></LI><LI><A href="http://2012.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Material_.26_Methods"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Material &amp; Methods</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Protocols"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-3 tocsection-3"><A href="#Ligation"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Ligation</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#Transformation"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#Glycerol_Stocks"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Glycerol Stocks</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#PCR"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#SDS-Page"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">SDS-Page</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Cell_Lysis"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">Cell Lysis</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Miller_Assay"><SPAN class="tocnumber">1.1.7</SPAN><SPAN class="toctext">Miller Assay</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#TECAN_plate_reader_Sample_preparation"><SPAN class="tocnumber">1.1.8</SPAN><SPAN class="toctext">TECAN plate reader Sample preparation</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Single_Cell_analysis_using_flow_cytometry"><SPAN class="tocnumber">1.1.9</SPAN><SPAN class="toctext">Single Cell analysis using flow cytometry</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#High-performance_liquid_chromatography_.28HPLC.29"><SPAN class="tocnumber">1.1.10</SPAN><SPAN class="toctext">High-performance liquid chromatography (HPLC)</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Purification_of_TetRDBD-UVR8-HisTag"><SPAN class="tocnumber">1.1.11</SPAN><SPAN class="toctext">Purification of TetR<SUB>DBD</SUB>-UVR8-HisTag</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-14"><A href="#Mediums"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Mediums</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Agar_plates"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Agar plates</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#LB_medium_.281L.29"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">LB medium (1L)</SPAN></A></LI><LI class="toclevel-3 tocsection-17"><A href="#LB_agar_.281L.29"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">LB agar (1L)</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Chemicals"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Chemicals</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Antibiotic_Stock_Solutions_.281000x.29"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Antibiotic Stock Solutions (1000x)</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#X-Gal"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">X-Gal</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#IPTG"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">IPTG</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#aTc"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">aTc</SPAN></A></LI><LI class="toclevel-2"><A href="#References"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Material_.26_Methods"> Material &amp; Methods </SPAN></H2><H3><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H3><H4><SPAN class="mw-headline" id="Ligation">Ligation</SPAN></H4><P>Ligation is performed with a ration of 1:5 plasmid backbone : Insert . The Ligation mix is incubated for at least 10 min at room temperature or for at least 1 hour at 16°C before the ligase is heat inactivated at 65°C for 20 min. 
</P><P>
Ligation mix: 
</P><TABLE><TBODY><TR><TD>DNA Mix
</TD><TD>x µL
</TD></TR><TR><TD>Ligase Buffer 10x
</TD><TD>2.5 µL
</TD></TR><TR><TD>Ligase
</TD><TD>0.5 µL
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>to 25 µL
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H4><UL><LI> Allow competent cells to thaw on ice
</LI><LI> Add ligation-mixture or 3-5 µl DNA to 50 µL cells
</LI><LI> Incubate on ice for 30 min 
</LI><LI> Heatshock 45 sec at 42 °C 
</LI><LI> Add 900 µL  LB medium and incubate in a shaker at 37 °C 60 min. 
</LI><LI> Spin cells down and remove supernatant 
</LI><LI> Resuspend cells in 50 µL medium and spread them onto a agar plate
</LI></UL><H4><SPAN class="mw-headline" id="Glycerol_Stocks">Glycerol Stocks</SPAN></H4><P>For longtime storrage 0.6 mL of cells (overnight culture) are mixed with 0.4 mL 100 % Glycerol and stored at -80°C. 
</P><H4><SPAN class="mw-headline" id="PCR">PCR</SPAN></H4><TABLE><TBODY><TR><TD>Template
</TD><TD>1 ng
</TD></TR><TR><TD>Phusion buffer (5x)
</TD><TD>10 µL
</TD></TR><TR><TD>Forward Primer (10µM)
</TD><TD>2.5 µL
</TD></TR><TR><TD>Reverse Primer (10µM)
</TD><TD>2.5 µL
</TD></TR><TR><TD>Phusion polymerase
</TD><TD> 0.2 µL
</TD></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD> to 50 µL
</TD></TR></TBODY></TABLE><P>
For colony PCR the colony is resuspended in 10 µL water and the PCR mix is adjusted with respect to the additional 10 µL. The initial denaturation step is extended up to 5 min.
PCR protocol: 
</P><UL><LI> Initial denaturation at 98°C for 30 sec
</LI><LI> 25-35 cycles: 
</LI><LI> Denaturation at 98°C for 5 sec
</LI><LI> Annealing for 20 sec (Temp. depends on Primer)
</LI><LI> Extension at 72°C for 30 sec per kb
</LI></UL><LI>Final extension at 72 °C for 5 min 
</LI><H4><SPAN class="mw-headline" id="SDS-Page">SDS-Page</SPAN></H4><TABLE><TBODY><TR><TH>Compound
</TH><TH> 12% Running gel
</TH><TH> 5% Stacking gel
</TH></TR><TR><TD>H<SUB>2</SUB>O
</TD><TD>6.67 mL
</TD><TD>3.54 mL
</TD></TR><TR><TD>1.5 M Tris-HCl, pH 8.8
</TD><TD>5 mL
</TD></TR><TR><TD>0.5M Tris-HCl, pH 6.8
</TD><TD>3 mL
</TD></TR><TR><TD>10% (w/v) SDS
</TD><TD>200 µL
</TD><TD>80µL
</TD></TR><TR><TD>Acrylamide/Bis-acrylamide (30%/0.8% w/v)
</TD><TD>8 mL
</TD><TD>1.32 mL
</TD></TR><TR><TD>10%(w/v) APS
</TD><TD>100 µL
</TD><TD>50 µL
</TD></TR><TR><TD>TEMED
</TD><TD>30 µL
</TD><TD>15 µL
</TD></TR><TR><TD>Total 4 gels:
</TD><TD> 20 mL
</TD><TD> 8mL
</TD></TR></TBODY></TABLE><P>As a marker for the SDS-Page  “PageRuler plus prestained protein marker” is used. Gel is stained with Coomassie Blue.
</P><P>Native-page acrylamide gels were made with the same composition, just lacking SDS and stacking layer. Also, native gels contained an acrylamide top to bottom gradient of 6 - 20 %.
</P><H4><SPAN class="mw-headline" id="Cell_Lysis">Cell Lysis</SPAN></H4><P>Lysis Buffer (Tris 50 mM, NaCl 150 mM, EDTA 5 mM, pH=7.4)
</P><UL><LI> Spin down 1 mL of liquid culture
</LI><LI> Resuspend in 1 mL of lysis buffer
</LI><LI> Add lysozyme to a concentration of 1 mg/mL
</LI><LI> Freeze cells in dry ice for 30 min
</LI><LI> Thaw the cells and centrifuge at 4°C
</LI><LI> Use supernatant for testing
</LI></UL><H4><SPAN class="mw-headline" id="Miller_Assay">Miller Assay</SPAN></H4><P>Z Buffer (NaH<SUB>2</SUB>PO<SUB>4</SUB> 40 mM, Na<SUB>2</SUB>HPO<SUB>4</SUB> 60 mM, KCl 10 mM, pH=7)
</P><P>Add 4mg/mL of ortho-Nitrophenyl-β-Glactoside (ONPG) just before useage.
</P><P>20 µL cell lysate is dissolved in 180 µL of Z-Buffer with ONPG. ONP activity is measured in a 96 wellplate at 420 nm every minute over a time period of 10min. β-Galactosidase activity determined based on the slope of the measurements. 
</P><H4><SPAN class="mw-headline" id="TECAN_plate_reader_Sample_preparation">TECAN plate reader Sample preparation</SPAN></H4><P>5 mL of fresh LB containing necessary antibiotic resistances was inoculated with 50 µL of overnight culture and grown at +37 °C while shaking vigorously. After cell density reaches OD<SUB>600</SUB> ~ 0.1-0.2, 1 mL was withdrawn and mixed with IPTG. Then, three 200 µL samples were taken and transferred into sterile 96 well plate (Thermofisher scientific, Denmark) and used for data analysis as a triplicate. Cells were later grown in TECAN Infinite 200Pro plate reader (Switzerland) at 37 °C, while shaking in orbital mode with 5 mm amplitude. Each 15 min fluorescence (excitation at 488±4.5 nm, emission at 530±10 nm; 25 flashes and integration over 20 µs) and cell density (absorbance at 600±4.5 nm; 15 flashes) measurements were taken. 
</P><H4><SPAN class="mw-headline" id="Single_Cell_analysis_using_flow_cytometry">Single Cell analysis using flow cytometry</SPAN></H4><P>For sample preparation the OD600 was measured to gauge the volume for sample-taking ensuring similar cell-concentrations for each measure. Cells are harvested and resuspended in an appropriate amount of Phosphate buffered saline (PBS). It is possible that the cells has to be diluted later because the flow is too high due to high concentration.
</P><P>Measurements were performed in the BD LSRFortessa<SUP>TM</SUP> using following lasers and filters:
</P><UL><LI> SSC and FSC: 488 nm - 488 nm/10 nm
</LI><LI> GFP: 488 - 530/30
</LI><LI> excitation: 488 nm
</LI><LI> emission: 530 ± 15 nm
</LI></UL><LI> mRFP and mCherry: 561 - 610/20
</LI><LI> YFP: 488 - 542/27
</LI><LI> eCFP: 445 - 473/10
</LI><H4><SPAN class="mw-headline" id="High-performance_liquid_chromatography_.28HPLC.29"> High-performance liquid chromatography (HPLC)</SPAN></H4><P>The HPLC was performed with a ... column. 
Sample preparation: Cells were lysed in acteonitril incubated for 15 min on ice and then centrifuged for 30 min at top speed. 
The supernatant was applied onto the column. 
</P><P>
HPLC Method:
</P><UL><LI> Column (preheated up to 40 °C) was equilibrated with 92% A and 8% B
</LI><LI> 50 µl per sample were applied to the column
</LI><LI> The proportion B was increased linearly to 50% in 7 min. The last 3 min it is increased to 100%. 
</LI></UL><P>Buffers:
</P><P>A: Water , 0.1% formic acid
</P><P>B: Methanol, 0.1% formic acid
</P><H4><SPAN class="mw-headline" id="Purification_of_TetRDBD-UVR8-HisTag"> Purification of TetR<SUB>DBD</SUB>-UVR8-HisTag </SPAN></H4><P>Cells, expresing TetR<SUB>DBD</SUB>-dUVR8 with 7xHis tag at the C-terminus under P<SUB>tac</SUB> promoter, were grown overnight in the 100 mL of LB media at 37 °C without IPTG induction. Culture were then centrifuged at 4000 rpm for 20 min at 4°, supernatant discarded and cells were resuspended at 10 mL of Lysis buffer (50 mM K-PO<SUB>4</SUB>, 500 mM NaCl, 10 mM Imidazol) and lysed with 1 mg/mL lysozyme at room temperature followed with rapid freezing in dry ice and kept frozen for 1h. Cells then were thawed and incubated for another hour with DNase and frozen as before. Cells debris were removed by centrifugation and supernatant was loaded on Ni ion affinity chromatography column, washed with lysis buffer containing 50 mM imidazol and eluted with 50 mM K-PO<SUB>4</SUB>, 500 mM NaCl, 300 mM imidazol. Elute was concentrated and diluted 1:1 with glycerol and stored at -20 °C.
</P><H3><SPAN class="mw-headline" id="Mediums">Mediums</SPAN></H3><H4><SPAN class="mw-headline" id="Agar_plates">Agar plates</SPAN></H4><UL><LI> 1 % gels in TAE buffer
</LI></UL><H4><SPAN class="mw-headline" id="LB_medium_.281L.29"> LB medium (1L)</SPAN></H4><UL><LI> 10 g Bacto-tryptone
</LI><LI> 5 g yeast extract
</LI><LI> 10g NaCl
</LI></UL><H4><SPAN class="mw-headline" id="LB_agar_.281L.29"> LB agar (1L) </SPAN></H4><UL><LI> LB medium
</LI><LI> 15 g Agar
</LI></UL><H3><SPAN class="mw-headline" id="Chemicals">Chemicals</SPAN></H3><H4><SPAN class="mw-headline" id="Antibiotic_Stock_Solutions_.281000x.29">Antibiotic Stock Solutions (1000x)</SPAN></H4><UL><LI> Kanamycin: 50 mg/mL 
</LI><LI> Ampicillin:  100 mg/mL
</LI><LI> Chloramphenicol: 34 mg/mL
</LI></UL><H4><SPAN class="mw-headline" id="X-Gal">X-Gal</SPAN></H4><UL><LI> 20 mg/mL in DMSO
</LI></UL><H4><SPAN class="mw-headline" id="IPTG">IPTG</SPAN></H4><UL><LI> 100 mM in water
</LI></UL><H4><SPAN class="mw-headline" id="aTc">aTc</SPAN></H4><UL><LI> 1 mg/mL in ethanol
</LI></UL><DIV id="eth_references"><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><UL><LI id="Brown2009">Brown, B. a, Headland, L. R., &amp; Jenkins, G. I. (2009). UV-B action spectrum for UVR8-mediated HY5 transcript accumulation in Arabidopsis. Photochemistry and photobiology, 85(5), 1147–55.</LI><LI id="Christie1999">Christie, J. M., Salomon, M., Nozue, K., Wada, M., &amp; Briggs, W. R. (1999): LOV (light, oxygen, or voltage) domains of the blue-light photoreceptor phototropin (nph1): binding sites for the chromophore flavin mononucleotide. Proceedings of the National Academy of Sciences of the United States of America, 96(15), 8779–83.</LI><LI id="Christie2012">Christie, J. M., Arvai, A. S., Baxter, K. J., Heilmann, M., Pratt, A. J., O’Hara, A., Kelly, S. M., et al. (2012). Plant UVR8 photoreceptor senses UV-B by tryptophan-mediated disruption of cross-dimer salt bridges. Science (New York, N.Y.), 335(6075), 1492–6.</LI><LI id="Cloix2008">Cloix, C., &amp; Jenkins, G. I. (2008). Interaction of the Arabidopsis UV-B-specific signaling component UVR8 with chromatin. Molecular plant, 1(1), 118–28.</LI><LI id="Cox2007">Cox, R. S., Surette, M. G., &amp; Elowitz, M. B. (2007). Programming gene expression with combinatorial promoters. Molecular systems biology, 3(145), 145. doi:10.1038/msb4100187</LI><LI id="Drepper2007">Drepper, T., Eggert, T., Circolone, F., Heck, A., Krauss, U., Guterl, J.-K., Wendorff, M., et al. (2007). Reporter proteins for in vivo fluorescence without oxygen. Nature biotechnology, 25(4), 443–5</LI><LI id="Drepper2011">Drepper, T., Krauss, U., &amp; Berstenhorst, S. M. zu. (2011). Lights on and action! Controlling microbial gene expression by light. Applied microbiology, 23–40.</LI><LI id="EC2006">EuropeanCommission (2006). SCIENTIFIC COMMITTEE ON CONSUMER PRODUCTS SCCP Opinion on Biological effects of ultraviolet radiation relevant to health with particular reference to sunbeds for cosmetic purposes.</LI><LI id="Elvidge2010">Elvidge, C. D., Keith, D. M., Tuttle, B. T., &amp; Baugh, K. E. (2010). Spectral identification of lighting type and character. Sensors (Basel, Switzerland), 10(4), 3961–88.</LI><LI id="GarciaOjalvo2004">GarciaOjalvo, J., Elowitz, M. B., &amp; Strogatz, S. H. (2004). Modeling a synthetic multicellular clock: repressilators coupled by quorum sensing. Proceedings of the National Academy of Sciences of the United States of America, 101(30), 10955–60.</LI><LI id="Gao2011">Gao Q, Garcia-Pichel F. (2011). Microbial ultraviolet sunscreens. Nat Rev Microbiol. 9(11):791-802. </LI><LI id="Goosen2008">Goosen N, Moolenaar GF. (2008) Repair of UV damage in bacteria. DNA Repair (Amst).7(3):353-79.</LI><LI id="Heijde2012">Heijde, M., &amp; Ulm, R. (2012). UV-B photoreceptor-mediated signalling in plants. Trends in plant science, 17(4), 230–7.</LI><LI id="Hirose2010">Hirose, Y., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2010). Cyanobacteriochrome CcaS regulates phycoerythrin accumulation in Nostoc punctiforme, a group II chromatic adapter. Proceedings of the National Academy of Sciences of the United States of America, 107(19), 8854–9.</LI><LI id="Hirose2008">Hirose, Y., Shimada, T., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2008). Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein. Proceedings of the National Academy of Sciences of the United States of America, 105(28), 9528–33.</LI><LI id="Kast1996"> Kast, Asif-Ullah &amp; Hilvert (1996) Tetrahedron Lett. 37, 2691 - 2694., Kast, Asif-Ullah, Jiang &amp; Hilvert (1996) Proc. Natl. Acad. Sci. USA 93, 5043  - 5048 </LI><LI id="Kiefer2010">Kiefer, J., Ebel, N., Schlücker, E., &amp; Leipertz, A. (2010). Characterization of Escherichia coli suspensions using UV/Vis/NIR absorption spectroscopy. Analytical Methods, 9660. doi:10.1039/b9ay00185a</LI><LI id="Kinkhabwala2008">Kinkhabwala, A., &amp; Guet, C. C. (2008). Uncovering cis regulatory codes using synthetic promoter shuffling. PloS one, 3(4), e2030.</LI><LI id="RKI2010"> Krebs in Deutschland 2005/2006. Häufigkeiten und Trends. 7. Auflage, 2010, Robert Koch-Institut (Hrsg) und die Gesellschaft der epidemiologischen Krebsregister in Deutschland e. V. (Hrsg). Berlin. </LI><LI id="Lamparter2002">Lamparter, T., Michael, N., Mittmann, F., &amp; Esteban, B. (2002). Phytochrome from Agrobacterium tumefaciens has unusual spectral properties and reveals an N-terminal chromophore attachment site. Proceedings of the National Academy of Sciences of the United States of America, 99(18), 11628–33.</LI><LI id="Levskaya2005">Levskaya, A. et al (2005). Engineering Escherichia coli to see light. Nature, 438(7067), 442.</LI><LI id="Mancinelli1986">Mancinelli, A. (1986). Comparison of spectral properties of phytochromes from different preparations. Plant physiology, 82(4), 956–61. </LI><LI id="Nakasone2007">Nakasone, Y., Ono, T., Ishii, A., Masuda, S., &amp; Terazima, M. (2007). Transient dimerization and conformational change of a BLUF protein: YcgF. Journal of the American Chemical Society, 129(22), 7028–35.</LI><LI id="Orth2000">Orth, P., &amp; Schnappinger, D. (2000). Structural basis of gene regulation by the tetracycline inducible Tet repressor-operator system. Nature structural biology, 215–219.</LI><LI id="Parkin2005"> Parkin, D.M., et al., Global cancer statistics, 2002. CA: a cancer journal for clinicians, 2005. 55(2): p. 74-108. </LI><LI id="Rajagopal2004">Rajagopal, S., Key, J. M., Purcell, E. B., Boerema, D. J., &amp; Moffat, K. (2004). Purification and initial characterization of a putative blue light-regulated phosphodiesterase from Escherichia coli. Photochemistry and photobiology, 80(3), 542–7.</LI><LI id="Rizzini2011">Rizzini, L., Favory, J.-J., Cloix, C., Faggionato, D., O’Hara, A., Kaiserli, E., Baumeister, R., et al. (2011). Perception of UV-B by the Arabidopsis UVR8 protein. Science (New York, N.Y.), 332(6025), 103–6.</LI><LI id="Roux1992">Roux, B., &amp; Walsh, C. T. (1992). p-aminobenzoate synthesis in Escherichia coli: kinetic and mechanistic characterization of the amidotransferase PabA. Biochemistry, 31(30), 6904–10.</LI><LI id="Strickland2008">Strickland, D. (2008). Light-activated DNA binding in a designed allosteric protein. Proceedings of the National Academy of Sciences of the United States of America, 105(31), 10709–10714.</LI><LI id="Sinha2002"> Sinha RP, Häder DP. UV-induced DNA damage and repair: a review. Photochem Photobiol Sci. (2002). 1(4):225-36 </LI><LI id="Sambandan2011">Sambandan DR, Ratner D. (2011). Sunscreens: an overview and update. J Am Acad Dermatol. 2011 Apr;64(4):748-58.</LI><LI id="Tabor2011">Tabor, J. J., Levskaya, A., &amp; Voigt, C. A. (2011). Multichromatic Control of Gene Expression in Escherichia coli. Journal of Molecular Biology, 405(2), 315–324.</LI><LI id="Thibodeaux2009">Thibodeaux, G., &amp; Cowmeadow, R. (2009). A tetracycline repressor-based mammalian two-hybrid system to detect protein–protein interactions in vivo. Analytical biochemistry, 386(1), 129–131.</LI><LI id="Tschowri2009">Tschowri, N., &amp; Busse, S. (2009). The BLUF-EAL protein YcgF acts as a direct anti-repressor in a blue-light response of Escherichia coli. Genes &amp; development, 522–534.</LI><LI id="Tschowri2012">Tschowri, N., Lindenberg, S., &amp; Hengge, R. (2012). Molecular function and potential evolution of the biofilm-modulating blue light-signalling pathway of Escherichia coli. Molecular microbiology.</LI><LI id="Tyagi2009">Tyagi, A. (2009). Photodynamics of a flavin based blue-light regulated phosphodiesterase protein and its photoreceptor BLUF domain.</LI><LI id="Vainio2001">Vainio, H. &amp; Bianchini, F. (2001). IARC Handbooks of Cancer Prevention: Volume 5: Sunscreens. Oxford University Press, USA</LI><LI id="Quinlivan2003">Quinlivan, Eoin P &amp; Roje, Sanja &amp; Basset, Gilles &amp; Shachar-Hill, Yair &amp; Gregory, Jesse F &amp; Hanson, Andrew D. (2003). The folate precursor p-aminobenzoate is reversibly converted to its glucose ester in the plant cytosol. The Journal of biological chemistry, 278.</LI><LI id="VanThor2001">van Thor, J. J., Borucki, B., Crielaard, W., Otto, H., Lamparter, T., Hughes, J., Hellingwerf, K. J., et al. (2001). Light-induced proton release and proton uptake reactions in the cyanobacterial phytochrome Cph1. Biochemistry, 40(38), 11460–71.</LI><LI id="Wegkamp2007">Wegkamp A, van Oorschot W, de Vos WM, Smid EJ. (2007 )Characterization of the role of para-aminobenzoic acid biosynthesis in folate production by Lactococcus lactis. Appl Environ Microbiol. Apr;73(8):2673-81.</LI></UL></DIV></DIV><DIV id="eth_footer"><DIV id="eth_sponsors"><TABLE><CAPTION><B>We want to thank our sponsors:</B></CAPTION><TBODY><TR><TD><DIV class="center"><DIV class="floatnone"><A href="/File:ETH_sponsor_kgf.gif" title="File:ETH sponsor kgf.gif">180px</A></DIV></DIV></TD></TR><TR><TD><A href="/File:ETH_BSSE.gif" title="File:ETH BSSE.gif">100px</A></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:ETH_Zurich/MaterialMethods">http://2012.igem.org/Team:ETH_Zurich/MaterialMethods</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/MaterialMethods" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/MaterialMethods" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/MaterialMethods&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/MaterialMethods&amp;oldid=296495" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>