<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_ETH_Zurich_Modeling_Construct1"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:ETH_Zurich/Modeling/Construct1">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:ETH_Zurich/Modeling/Construct1&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Construct1&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Construct1&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling/Construct1" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:ETH Zurich/Modeling/Construct1</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="eth_page"><DIV id="eth_menu"><DIV id="eth_nav"><UL><LI><A href="/Team:ETH_Zurich" title="Team:ETH Zurich">Home</A></LI><LI><A href="/Team:ETH_Zurich/Project_overview" title="Team:ETH Zurich/Project overview">Project</A></LI><LI><A href="/Team:ETH_Zurich/Project_overview" title="Team:ETH Zurich/Project overview">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Motivation" title="Team:ETH Zurich/Motivation">Motivation</A></LI><LI><A href="/Team:ETH_Zurich/Applications" title="Team:ETH Zurich/Applications">Applications</A></LI><LI><A href="/Team:ETH_Zurich/Interplay" title="Team:ETH Zurich/Interplay">Lab &amp; Modelling</A></LI></UL><LI><A href="/Team:ETH_Zurich/Project" title="Team:ETH Zurich/Project">Parts</A></LI><LI><A href="/Team:ETH_Zurich/Project" title="Team:ETH Zurich/Project">Overview</A></LI><LI><A href="/Team:ETH_Zurich/UVR8" title="Team:ETH Zurich/UVR8">Direct: UVR8-TetR<SUB>DBD</SUB></A></LI><LI><A href="/Team:ETH_Zurich/Decoder" title="Team:ETH Zurich/Decoder">Indirect: Decoder</A></LI><LI><A href="/Team:ETH_Zurich/PABA" title="Team:ETH Zurich/PABA">Output: PABA</A></LI><LI><A href="/Team:ETH_Zurich/Parts" title="Team:ETH Zurich/Parts">Parts</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Modeling</A></LI><LI><A href="/Team:ETH_Zurich/Modeling" title="Team:ETH Zurich/Modeling">Overview</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Photoinduction" title="Team:ETH Zurich/Modeling/Photoinduction">Photoinduction</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/UVR8" title="Team:ETH Zurich/Modeling/UVR8">UVR8-TetR<SUB>DBD</SUB> Circuit</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/SPF_model" title="Team:ETH Zurich/Modeling/SPF model">Sun Protection Factor</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Construct2" title="Team:ETH Zurich/Modeling/Construct2">Decoder Circuit: LovTAP/Cph8</A></LI><LI><A href="/Team:ETH_Zurich/Modeling/Parameters" title="Team:ETH Zurich/Modeling/Parameters">Parameters</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Notebook</A></LI><LI><A href="/Team:ETH_Zurich/Notebook" title="Team:ETH Zurich/Notebook">Overview</A></LI><LI><A href="/Team:ETH_Zurich/MaterialMethods" title="Team:ETH Zurich/MaterialMethods">Material &amp; Methods</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements" title="Team:ETH Zurich/Achievements">Medal Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Achievements_overall" title="Team:ETH Zurich/Achievements overall">Project Achievements</A></LI><LI><A href="/Team:ETH_Zurich/Safety" title="Team:ETH Zurich/Safety">Safety</A></LI><LI><A href="/Team:ETH_Zurich/HumanPractice" title="Team:ETH Zurich/HumanPractice">Human practice</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Team</A></LI><LI><A href="/Team:ETH_Zurich/Team" title="Team:ETH Zurich/Team">Team description</A></LI><LI><A href="/Team:ETH_Zurich/Attributions" title="Team:ETH Zurich/Attributions">Attributions &amp; Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=ETH_Zurich" class="external text" rel="nofollow">Official Team Profile</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:ETH_Zurich/Modeling/Construct1" class="external text" rel="nofollow">Login</A></LI><LI><A href="http://2012.igem.org/Team:ETH_Zurich/Modeling/Construct1" class="external text" rel="nofollow">WIKI</A></LI><LI><A href="http://2012.igem.org/Team:ETH_Zurich/Modeling/Construct1" class="external text" rel="nofollow">View page</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Team:ETH_Zurich/Modeling/Construct1&amp;action=edit" class="external text" rel="nofollow">Edit page</A></LI><LI><A href="http://2012.igem.org/wiki/index.php?title=Team:ETH_Zurich/Modeling/Construct1&amp;action=history" class="external text" rel="nofollow">History of this page</A></LI><LI><A href="http://2012.igem.org/Special:Upload" class="external text" rel="nofollow">Upload new file</A></LI></DIV></DIV><DIV id="eth_content"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#UVR8-TetRDBD.2FLovTAP"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">UVR8-TetR<SUB>DBD</SUB>/LovTAP</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Boolean_logic"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Boolean logic</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Circuit"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Circuit</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Modelling_assumptions"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Modelling assumptions</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#ODEs"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">ODEs</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Results"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Biological_implications"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">Biological implications</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Outlook"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Outlook</SPAN></A></LI><LI class="toclevel-2"><A href="#References"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">References</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="UVR8-TetRDBD.2FLovTAP"> UVR8-TetR<SUB>DBD</SUB>/LovTAP </SPAN></H2><P>In order to account for varying light intensity ratios, we employ a second photoreceptor to &quot;calibrate&quot; for light intensity. This prevents spurious activation of the downstream protection system. As UVR8-TetR<SUB>DBD</SUB> is our engineered protein that we wish to employ for UVB detection, we combine it with LovTAP in the form of a decoder. For this, we define a set of conditions beforehand and devise a model to differentiate between these light sources. The designed circuit includes a number of reporter genes for debugging. Once the circuit has been shown to work as a whole, pabAB is placed under control of the central operon regulated by UVR8 and LacI, as is depicted <A href="/Team:ETH_Zurich/Modeling/Construct1#Circuit" title="Team:ETH Zurich/Modeling/Construct1">here</A>.
</P><H3><SPAN class="mw-headline" id="Boolean_logic"> Boolean logic </SPAN></H3><TABLE align="center" border="1" cellpadding="8" cellspacing="0" style="background:#f8f8ff; font-weight:normal; text-align:center; width:85%"><CAPTION><B>Boolean response</B></CAPTION><TBODY><TR style="background:#e6e6ff"><TH style="width:13%"> UV-Light<SUB>input</SUB></TH><TH style="width:13%"> Blue-Light<SUB>input</SUB></TH><TH style="width:13%"> Output<SUB>Green</SUB></TH><TH style="width:13%"> Output<SUB>Red</SUB></TH><TH style="width:13%"> Output<SUB>Violet/pabAB</SUB></TH><TH style="width:30%"> Situation
</TH></TR><TR><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> 0 </TD><TD> darkness
</TD></TR><TR><TD> 0 </TD><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD> 0 </TD><TD> modern interior light source e.g. <A href="http://en.wikipedia.org/wiki/Cold_cathode" class="external text" rel="nofollow">CCFL</A></TD></TR><TR><TD> 1 </TD><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD> 0 </TD><TD> tanning bed - <B>known carcinogen</B>!
</TD></TR><TR><TD> 1 </TD><TD> 1 </TD><TD> 0 </TD><TD> 0 </TD><TD> 1 </TD><TD> sun exposure
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Circuit"> Circuit </SPAN></H3><P>The following scheme illustrates the system:
</P><H3><SPAN class="mw-headline" id="Modelling_assumptions"> Modelling assumptions </SPAN></H3><P>In order to analyse a number of properties of the system, the model needs to be tractable. For this, a number of assumptions and approximations have been made:
</P><UL><LI> UVR8-TetR<SUB>DBD</SUB> and LovTAP are conserved. This translates to a model not explicitly accounting for photoreceptor expression/degradation, as this will be the case for most systems in steady state anyway. Biological optimisation will instead focus on choosing the optimal ratio of promoter strength k<SUB>P</SUB> to degradation rate k<SUB>deg</SUB> of the photoreceptors that effectively equal the steady state concentration of UVR8-TetR<SUB>DBD</SUB>.
</LI><LI> All photoconversion processes are modelled as 2-state processes.
</LI><LI> Degradation rates are identical for all proteins. While this obviously does not resemble biological reality, setting a degradation rate for every protein species individually is practically impossible, as such rate parameters are not ubiquitous in literature.
</LI><LI> The negative feedback has not been modelled explicitly here, as it has been accounted for in the previous simpler <A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/UVR8" class="external text" rel="nofollow">UVR8-TetR<SUB>DBD</SUB></A> model. As no oscillations are possible, the negative feedback loop merely reduces the dynamic range of the switching.
</LI><LI> Basal expression as fraction of complete induction is the same for all promoters (= 15% in our case).
</LI></UL><H3><SPAN class="mw-headline" id="ODEs"> ODEs </SPAN></H3><P>With the previous assumptions, the coupled system boils down to:
</P><P>
Find the parameters we used on our <A href="https://2012.igem.org/Team:ETH_Zurich/Modeling/Parameters#UVR8-TetRDBD-LovTAP" class="external text" rel="nofollow">parameter page</A>.
</P><H3><SPAN class="mw-headline" id="Results"> Results </SPAN></H3><P>Notice the vertical dashed bars at 4 h, these indicate a switch of conditions from beginning in darkness to a physical situation highlighted by the black light sources in each row. The horizontal dotted lines indicate the response threshold, where values above such threshold indicate a positive (1) response and values below indicate a negative (0) response. Note that only the values in steady-state have a meaning - dynamics of the system might temporarily lift some pigment concentrations above the threshold that will drop below it in steady state again (as is the case for the red pigment in situation [no UV, no Blue] and [UV, Blue]). The separation between conditions is comparatively small - this is due to the intrinsic basal expression that we have modelled.
</P><H3><SPAN class="mw-headline" id="Biological_implications"> Biological implications </SPAN></H3><P>The system has been tuned manually to maximise the dynamic range between conditions. This has resulted in promoter expression parameters that constrain the model:
</P><P>This in effect requires the same promoter strengths to be employed in-vivo. While most promoters have a rough notion of expression strength attached to them, this will eventually boil down to a trial-and-error inspection of the correct promoter. From a informatics-heuristical point of view, we suggest working in a binary-logarithmic &quot;divide-and-conquer&quot; fashion, always choosing a new candidate promoter that is between 2 promoters having expressions levels too low and too high.
</P><H3><SPAN class="mw-headline" id="Outlook"> Outlook </SPAN></H3><P>For the constraints here we have optimised the parameters by hand using some intuition. In line with the general trend of computational optimisation of biological systems, we aim to optimise the dynamic range by mathematical optimisation of certain free parameters (such as the promoter strengths). This procedure is non-trivial and rather demanding, involving non-linear optimisation. This could increase the dynamic range further.
</P><DIV id="eth_references"><H3><SPAN class="mw-headline" id="References"> References </SPAN></H3><UL><LI id="Brown2009">Brown, B. a, Headland, L. R., &amp; Jenkins, G. I. (2009). UV-B action spectrum for UVR8-mediated HY5 transcript accumulation in Arabidopsis. Photochemistry and photobiology, 85(5), 1147–55.</LI><LI id="Christie1999">Christie, J. M., Salomon, M., Nozue, K., Wada, M., &amp; Briggs, W. R. (1999): LOV (light, oxygen, or voltage) domains of the blue-light photoreceptor phototropin (nph1): binding sites for the chromophore flavin mononucleotide. Proceedings of the National Academy of Sciences of the United States of America, 96(15), 8779–83.</LI><LI id="Christie2012">Christie, J. M., Arvai, A. S., Baxter, K. J., Heilmann, M., Pratt, A. J., O’Hara, A., Kelly, S. M., et al. (2012). Plant UVR8 photoreceptor senses UV-B by tryptophan-mediated disruption of cross-dimer salt bridges. Science (New York, N.Y.), 335(6075), 1492–6.</LI><LI id="Cloix2008">Cloix, C., &amp; Jenkins, G. I. (2008). Interaction of the Arabidopsis UV-B-specific signaling component UVR8 with chromatin. Molecular plant, 1(1), 118–28.</LI><LI id="Cox2007">Cox, R. S., Surette, M. G., &amp; Elowitz, M. B. (2007). Programming gene expression with combinatorial promoters. Molecular systems biology, 3(145), 145. doi:10.1038/msb4100187</LI><LI id="Drepper2007">Drepper, T., Eggert, T., Circolone, F., Heck, A., Krauss, U., Guterl, J.-K., Wendorff, M., et al. (2007). Reporter proteins for in vivo fluorescence without oxygen. Nature biotechnology, 25(4), 443–5</LI><LI id="Drepper2011">Drepper, T., Krauss, U., &amp; Berstenhorst, S. M. zu. (2011). Lights on and action! Controlling microbial gene expression by light. Applied microbiology, 23–40.</LI><LI id="EC2006">EuropeanCommission (2006). SCIENTIFIC COMMITTEE ON CONSUMER PRODUCTS SCCP Opinion on Biological effects of ultraviolet radiation relevant to health with particular reference to sunbeds for cosmetic purposes.</LI><LI id="Elvidge2010">Elvidge, C. D., Keith, D. M., Tuttle, B. T., &amp; Baugh, K. E. (2010). Spectral identification of lighting type and character. Sensors (Basel, Switzerland), 10(4), 3961–88.</LI><LI id="GarciaOjalvo2004">GarciaOjalvo, J., Elowitz, M. B., &amp; Strogatz, S. H. (2004). Modeling a synthetic multicellular clock: repressilators coupled by quorum sensing. Proceedings of the National Academy of Sciences of the United States of America, 101(30), 10955–60.</LI><LI id="Gao2011">Gao Q, Garcia-Pichel F. (2011). Microbial ultraviolet sunscreens. Nat Rev Microbiol. 9(11):791-802. </LI><LI id="Goosen2008">Goosen N, Moolenaar GF. (2008) Repair of UV damage in bacteria. DNA Repair (Amst).7(3):353-79.</LI><LI id="Heijde2012">Heijde, M., &amp; Ulm, R. (2012). UV-B photoreceptor-mediated signalling in plants. Trends in plant science, 17(4), 230–7.</LI><LI id="Hirose2010">Hirose, Y., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2010). Cyanobacteriochrome CcaS regulates phycoerythrin accumulation in Nostoc punctiforme, a group II chromatic adapter. Proceedings of the National Academy of Sciences of the United States of America, 107(19), 8854–9.</LI><LI id="Hirose2008">Hirose, Y., Shimada, T., Narikawa, R., Katayama, M., &amp; Ikeuchi, M. (2008). Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein. Proceedings of the National Academy of Sciences of the United States of America, 105(28), 9528–33.</LI><LI id="Kast1996"> Kast, Asif-Ullah &amp; Hilvert (1996) Tetrahedron Lett. 37, 2691 - 2694., Kast, Asif-Ullah, Jiang &amp; Hilvert (1996) Proc. Natl. Acad. Sci. USA 93, 5043  - 5048 </LI><LI id="Kiefer2010">Kiefer, J., Ebel, N., Schlücker, E., &amp; Leipertz, A. (2010). Characterization of Escherichia coli suspensions using UV/Vis/NIR absorption spectroscopy. Analytical Methods, 9660. doi:10.1039/b9ay00185a</LI><LI id="Kinkhabwala2008">Kinkhabwala, A., &amp; Guet, C. C. (2008). Uncovering cis regulatory codes using synthetic promoter shuffling. PloS one, 3(4), e2030.</LI><LI id="RKI2010"> Krebs in Deutschland 2005/2006. Häufigkeiten und Trends. 7. Auflage, 2010, Robert Koch-Institut (Hrsg) und die Gesellschaft der epidemiologischen Krebsregister in Deutschland e. V. (Hrsg). Berlin. </LI><LI id="Lamparter2002">Lamparter, T., Michael, N., Mittmann, F., &amp; Esteban, B. (2002). Phytochrome from Agrobacterium tumefaciens has unusual spectral properties and reveals an N-terminal chromophore attachment site. Proceedings of the National Academy of Sciences of the United States of America, 99(18), 11628–33.</LI><LI id="Levskaya2005">Levskaya, A. et al (2005). Engineering Escherichia coli to see light. Nature, 438(7067), 442.</LI><LI id="Mancinelli1986">Mancinelli, A. (1986). Comparison of spectral properties of phytochromes from different preparations. Plant physiology, 82(4), 956–61. </LI><LI id="Nakasone2007">Nakasone, Y., Ono, T., Ishii, A., Masuda, S., &amp; Terazima, M. (2007). Transient dimerization and conformational change of a BLUF protein: YcgF. Journal of the American Chemical Society, 129(22), 7028–35.</LI><LI id="Orth2000">Orth, P., &amp; Schnappinger, D. (2000). Structural basis of gene regulation by the tetracycline inducible Tet repressor-operator system. Nature structural biology, 215–219.</LI><LI id="Parkin2005"> Parkin, D.M., et al., Global cancer statistics, 2002. CA: a cancer journal for clinicians, 2005. 55(2): p. 74-108. </LI><LI id="Rajagopal2004">Rajagopal, S., Key, J. M., Purcell, E. B., Boerema, D. J., &amp; Moffat, K. (2004). Purification and initial characterization of a putative blue light-regulated phosphodiesterase from Escherichia coli. Photochemistry and photobiology, 80(3), 542–7.</LI><LI id="Rizzini2011">Rizzini, L., Favory, J.-J., Cloix, C., Faggionato, D., O’Hara, A., Kaiserli, E., Baumeister, R., et al. (2011). Perception of UV-B by the Arabidopsis UVR8 protein. Science (New York, N.Y.), 332(6025), 103–6.</LI><LI id="Roux1992">Roux, B., &amp; Walsh, C. T. (1992). p-aminobenzoate synthesis in Escherichia coli: kinetic and mechanistic characterization of the amidotransferase PabA. Biochemistry, 31(30), 6904–10.</LI><LI id="Strickland2008">Strickland, D. (2008). Light-activated DNA binding in a designed allosteric protein. Proceedings of the National Academy of Sciences of the United States of America, 105(31), 10709–10714.</LI><LI id="Sinha2002"> Sinha RP, Häder DP. UV-induced DNA damage and repair: a review. Photochem Photobiol Sci. (2002). 1(4):225-36 </LI><LI id="Sambandan2011">Sambandan DR, Ratner D. (2011). Sunscreens: an overview and update. J Am Acad Dermatol. 2011 Apr;64(4):748-58.</LI><LI id="Tabor2011">Tabor, J. J., Levskaya, A., &amp; Voigt, C. A. (2011). Multichromatic Control of Gene Expression in Escherichia coli. Journal of Molecular Biology, 405(2), 315–324.</LI><LI id="Thibodeaux2009">Thibodeaux, G., &amp; Cowmeadow, R. (2009). A tetracycline repressor-based mammalian two-hybrid system to detect protein–protein interactions in vivo. Analytical biochemistry, 386(1), 129–131.</LI><LI id="Tschowri2009">Tschowri, N., &amp; Busse, S. (2009). The BLUF-EAL protein YcgF acts as a direct anti-repressor in a blue-light response of Escherichia coli. Genes &amp; development, 522–534.</LI><LI id="Tschowri2012">Tschowri, N., Lindenberg, S., &amp; Hengge, R. (2012). Molecular function and potential evolution of the biofilm-modulating blue light-signalling pathway of Escherichia coli. Molecular microbiology.</LI><LI id="Tyagi2009">Tyagi, A. (2009). Photodynamics of a flavin based blue-light regulated phosphodiesterase protein and its photoreceptor BLUF domain.</LI><LI id="Vainio2001">Vainio, H. &amp; Bianchini, F. (2001). IARC Handbooks of Cancer Prevention: Volume 5: Sunscreens. Oxford University Press, USA</LI><LI id="Quinlivan2003">Quinlivan, Eoin P &amp; Roje, Sanja &amp; Basset, Gilles &amp; Shachar-Hill, Yair &amp; Gregory, Jesse F &amp; Hanson, Andrew D. (2003). The folate precursor p-aminobenzoate is reversibly converted to its glucose ester in the plant cytosol. The Journal of biological chemistry, 278.</LI><LI id="VanThor2001">van Thor, J. J., Borucki, B., Crielaard, W., Otto, H., Lamparter, T., Hughes, J., Hellingwerf, K. J., et al. (2001). Light-induced proton release and proton uptake reactions in the cyanobacterial phytochrome Cph1. Biochemistry, 40(38), 11460–71.</LI><LI id="Wegkamp2007">Wegkamp A, van Oorschot W, de Vos WM, Smid EJ. (2007 )Characterization of the role of para-aminobenzoic acid biosynthesis in folate production by Lactococcus lactis. Appl Environ Microbiol. Apr;73(8):2673-81.</LI></UL></DIV></DIV><DIV id="eth_footer"><DIV id="eth_sponsors"><TABLE><CAPTION><B>We want to thank our sponsors:</B></CAPTION><TBODY><TR><TD><DIV class="center"><DIV class="floatnone"><A href="/File:ETH_sponsor_kgf.gif" title="File:ETH sponsor kgf.gif">180px</A></DIV></DIV></TD></TR><TR><TD><A href="/File:ETH_BSSE.gif" title="File:ETH BSSE.gif">100px</A></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:ETH_Zurich/Modeling/Construct1">http://2012.igem.org/Team:ETH_Zurich/Modeling/Construct1</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:ETH_Zurich/Modeling/Construct1" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:ETH_Zurich/Modeling/Construct1" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Construct1&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:ETH_Zurich/Modeling/Construct1&amp;oldid=231054" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>