Team:AHUT China/Design
AHUT
Toggle navigation
Attribution
Project
Pathway&Design
Biobrick
protocols
Notebook
Lab Pictures
future work
Collaborations
Human Practice
Silver
Gold_Integrated
Engagement
Description
Proof
Demonstrate
1.Project
1.1Project Background
The project is based on a movie called The Maze Runner, in which the protagonist Thomas has to escape from a special maze. What makes this maze different from ordianary ones is that it changes constantly halfway with its complexity increasing correspondingly. To better solve the problem of this kind, we have conducted a series of researches with experiments.
It’s foreseeable that a moving maze can be abstracted as a problem of mathematical graph theory, i.e. finding a feasible pathway from the starting point to the terminal point to be more specific, which can be examined by methods like mathematical analysis or computer programs. We choose biology to analyze this theoretical model and make it more practical.
Using experiments of synthetic biology, we have acquired the starting point, terminal point and middle point by means of error-prone PCR and so on.  After this, we have worked out double strands containing a feasible pathway through experiments such as bridging. And then, the feasible pathway is confirmed through experiments such as sequencing.
1.2Project Description
In this project, the sequence of DNA template is seen as sides, front and back as points. As for the above mentioned movie—The Maze Runner—we can assume it a changeable and moving maze in this picture. What is fixed is an entrance and an exit while all pathways change according to certain rules. We simulate this maze and analyze and research all possible combination of pathways and points in order to find out the rules. We expect to establish a completely new biological guide system to make acquiring the only pathway faster and computing more convenient under the big data era, which we hope will become a small step towards breaking the bottleneck of breaking through the computing ability of electronic computers. This system is able to make automatic judgement quickly according to information about pathways and points so as to provide users with reliable pathways leading them out of a changeable maze.
2.Design
2.1Theoretical Analysis
DNA computing is to encode information and computing rules by means of DNA molecular structure and the principle of complementary base pairing, mapping the problem into a specific DNA segment, then to work out the problem through controllable biochemical reaction, and finally test and acquire computing results by using corresponding molecular biotechnology such as polymerase chain reaction (PCR), plasmid extraction, cloning, mutagenesis, electrophoresis, technology related to genetic engineering, DNA sequencing and so on.
2.2Designing of Points and Lines
Amplification of error-prone PCR:  with a prepared template (a known DNA sequence) as the template, the designed FNA and RNA primer is amplified to make different middle” points” (a DNA sequence that is 165bp long).Similarly, with a prepared template (a known DNA sequence) as the template, the designed PrefixF and RNA primer are amplified to make “head” ( a DNA sequence that is 195bp long) and the designed SuffixR and FNA primer  to make “tail” ( a DNA sequence that is 195bp long).
Agarose gel electrophoresis used to test amplification: the banding (compared with DNA marker) of electrophoresis can help to learn roughly the concentration and length of amplified segments.
AxyPrep DNA gel extraction kit is used to extract head, tail and points’ segments.
2.3 Experiment Procedures
2.3.1 Building head and tail:
the designed PCR template α is put in with RNA and FNA primer to amplify error-prone PCR and acquire different double-strand DNA as sides. And then prepared template is put in with RNA and PrefixF primer to do amplification to acquire templates with heads and with FNA and SuffixR to acquire templates with tails.
2.3.2 Building experiments:
extend extracted heads, tails, sides, three DNA segments and bridge segments used to connect. Due to multiple incisions of extended DNA, we are required to experiment on extracting and mending incisions.
2.3.3 Segment amplification:
Put above-mentioned DNA sample into PCR amplifier, add corresponding primer to amplify segments and extract DNA segments by electrophoresis. The expected result is that there are large quantities of DNA sequence with both a head and a tail.
2.3.4 Enzyme digestion and vector conjunction:
use the same endonuclease to incise built DNA segments and plasmid vectors, forming sticky ends. The incised segments and carriers are connected and amplified, acquiring recombined plasmid of a series of targeted gene segments.
2.3.5 Culturing and selecting in escherichia coli
2.3.6 Extracting and testing plasmids in escherichia coli
Attribution
Goto
Project
Pathway and Design
Biobrick
Protocols
Notebook
Lab picture
future work
Collaboration
Goto
Human
Practice
Silver
Gold
Engagement
Description
Goto
Demonstrate
Goto
Proof
Goto
Copyright 2017 © AHUT iGEM Team , All rights reserved .
