<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Utrecht_Tester skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Utrecht/Tester</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM UTRECHT Placeholder Page</TITLE></P><DIV style="text-align:center; box-sizing: content-box; width:100%; height:150px"><DIV style="width: 75%"><PAGE_TITLE style="vertical-align:middle; padding:20px"> Utrecht 2017</PAGE_TITLE><PROJECT_TITLE style="vertical-align:middle; horizontal-align:bottom">TESTER</PROJECT_TITLE></DIV></DIV><DIV style="text-align:center; color:#0000ff; font-size: 16px; box-sizing: border-box">Important note: This is a placeholder page and will be replaced in due time!</DIV><DIV style="text-align:center"><PROJECT_SUBTITLE>OUT-of-cell CRISPR Activated Sequence-specific Signal Transducer</PROJECT_SUBTITLE><DIV class="sec0">
		 	  This year, Utrecht University participates for the first time and we aim to build a biological DNA sensor that can recognise sequences depending of the user's choice. Further down, we propose some
applications but first, a description of how our sensor will work.
The OUTCASST sensor consist of two proteins, both expressed to the membrane of a mammalian HEK 293 cell.
			  The first protein connects dCas9, a catalytically dead Cas9 variant, to a transmembrane domain and an intracellular transcription factor.
			  The second protein connects dCpf1 to an intracellular protease. When provided with appropriate guide RNA's, the two proteins can bind to a DNA sequence.
			  When one sequence brings the two proteins together, the transcription factor is released into the cytoplasm, and can activate a reporter cascade.
			  See the scheme below for details.
		 </DIV></DIV><SECTION style="position: relative; width: 100%; margin-top:5px; margin-bottom:20px; padding: 0;"><DIV class="sec1"><DIV class="sectitle"><B>1. Guide RNA binding:</B></DIV>
		 	  The two proteins float around freely in the membrane. Two types of guide RNA (gRNA) can be added to the cells. 
			  Due to the different recognition sequences for the two proteins, the gRNA can bind specifically to the appropriate protein.
		 </DIV><DIV class="sec1"><DIV class="sectitle"><B>2. DNA sample addition:</B></DIV>
			  Now that the guide RNA's have been bound, both proteins are primed to bind to a specific sequence of a DNA strand.
			  A DNA sample from any source can be added and will then be bound by the protein with the guide RNA that is complementary to one of the DNA strands.
		 </DIV><DIV class="sec1"><DIV class="sectitle"><B>3. Sequence recognition:</B></DIV>
		 	  One of the proteins has bound a DNA stretch in the sample. 
			  If the other protein is bound to a guide RNA that is complementary to a sequence on the same stretch, it too will bind.
		 </DIV><DIV class="sec1"><DIV class="sectitle"><B>4. Co-localization &amp; cleavage:</B></DIV>
		 	  If the second protein also binds the DNA, close enough to the other protein, the protease at the end of one will be able to cleave the transcription factor from the other protein.
			  The transcription factor is then free to induce a reporter mechanism. In our case, this will be a fluorescent marker.
		 </DIV><DIV style="text-align:center"><PROJECT_SUBTITLE>Applications of the OUTCASST system:</PROJECT_SUBTITLE><DIV class="sec0">
		 	  So far, our team has not yet decided on a specific application of the system as there are many different fields wherein
			  sequence-specific detection is of use. Right now, most DNA detection is done by Polymerase Chain Reaction (PCR) and sequencing
			  techniques. These techniques rely on materials and machinery that are not available to everyone. The OUTCASST system, being 
			  cell-based, culturable and thus renewable, could provide a quick and easy detection tool that does not require an expensive 
			  lab-setup. It only requires medium to culture the cells and the gRNA that is specific to the sequence that you want to detect. 
			  Right now, we are interviewing potential end-users to figure out what type of applications they see for our concept and to 
			  assess what features they wish to see in our design. So far, we have identified four possible applications for our tool:
		 </DIV></DIV><SECTION style="position: relative; width: 100%; margin-top:5px; margin-bottom:20px; padding: 0;"><DIV class="sec2"><DIV class="sectitle"><B>Prenatal Genotyping:</B></DIV>
		 	  Small quantities of cell-free embryo DNA is present in the maternal peripheral blood, already early in pregnancy 
			  <A target="_tab" href="http://www.sciencedirect.com/science/article/pii/S0140673689919697">(Lo et al., 1989)</A>.
			  The detection of child-mutations by taking a serum sample from its mother reduces the risk to the unborn child. Such 
			  mutation calling is currently done by sequencing, which faces technological limitations when confronted with low sample
			  concentrations, and subsequent genotyping using bioinformatics pipelines, a process that takes days, if not weeks. We aim to create
			  the possibility to detect specific mutations in a matter of hours, cheaply. 
		 </DIV><DIV class="sec2"><DIV class="sectitle"><B>Pathogen Detection:</B></DIV>
			  Cell-free pathogen DNA can also be sampled from a patient's serum 
			  <A target="_tab" href="https://academic.oup.com/jid/article-abstract/170/2/436/893837">(Gan et al., 1994).</A> Again,
			  the aim of our design is to speed up the process of diagnosis. With one batch of our organism, multiple tests can be
			  done. By adding a small sample of our organism to each well in a 96-well plate and then adding a different subset of
			  gRNA sequences to each well, you could test for 96 separate pathogens. Such tests could speed up the diagnostics time
			  in medical labs. The sensor could even be shipped to general practitioners themselves, making more precise diagnoses 
			  in the doctor's office possible.
		 </DIV><DIV class="sec2"><DIV class="sectitle"><B>Cancer Screening</B></DIV>
		 	  Early and accurate detection of cancer is very important for proper treatment. Luckily, it too produces cell-free DNA that can be found in the
			  serum of a patient 
			  <A target="_tab" href="http://cancerres.aacrjournals.org/content/61/4/1659.short">(Schwarzenbach et al., 2011)</A>.
			  By designing gRNA, complementary to known oncogene mutations, the detection of these sequences in the serum sample thus means that the
			  harmful mutation is present in the patient. Our system can be modified such that signal intensity depends on the amount of dimerization and thus on the cell-free DNA concentration. Since the concentration of cell-free DNA depends on the growth
			  of the tumor, this concentration could be used for monitoring of the disease.
		 </DIV></SECTION></SECTION></DIV><DIV class="sec0" style="text-align:center">
		 	  This is a placeholder-page. In coming days, it will be replaced. For now, you can follow us on facebook and twitter:
			  </DIV></DIV></DIV></DIV></DIV></BODY></HTML>