<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_FAU_Erlangen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:FAU Erlangen/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Erlangen</TITLE></P><DIV id="model"><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container-fluid"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-collapse-main"><SPAN class="sr-only">Toggle navigation </SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-collapse-main"><UL class="nav navbar-nav navbar-right"><LI><A class="active" href="#">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team
                </A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Team">Team Members</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project
                </A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Model">Model</A></LI><LI><A href="#">Results</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Attributions">Attributions</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Parts
                </A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Parts">Parts Overview</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Safety">Safety</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices
                </A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Public_Engagement">Education &amp; Engagement</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Awards
                </A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Measurement">Measurement</A></LI><LI><A href="https://2018.igem.org/Team:FAU_Erlangen/Model">Model</A></LI><LI><A href="https://igem.org/2018_Judging_Form?team=FAU_Erlangen">Judging Form</A></LI></DIV></DIV></NAV><DIV class="content"><H1 class="model">Results</H1><H2 class="model">
Bio Lab
</H2><P class="model" style="padding-bottom:20px;">
The main idea of our iGEM project was paving the way for the implementation of S-Layer proteins in biocatalytically active membranes. These surface proteins are naturally occuring in different organisms and form diverse symmetries. We isolated SbsB from <I>Geobacillus stearothermophilus</I>, PS2 from <I>Corynebacterium glutamicum</I> and RsaA from <I>Caulobacter crescentus</I> forming p1, p2 and p3 symmetries respectively. Additionally, we created a fusion protein consisting of the p1 S-Layer SbsB and core-Streptatavidin. 
 
The principle behind the purification of SbsB, PS2 and RsaA is described below.
</P><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse-1"><P class="model-center"><B>Isolation of SbsB from <I>E. coli</I> (S-Layer protein of p1-symmetry)</B></P></A></H4></DIV><DIV id="collapse-1" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">
        SbsB, a S-Layer protein originating from <I>Geobacillus stearothermophilus</I> pV72/p2, was expressed in <I>E. coli</I> B834 (DE3). The coding sequence of SbSB, cloned into the expression vector cAb-Lys3-pHEN6, was given to us by the work group of Han Remaut. The sequence included an N-terminal His-tag for purification via Ni-NTA (Fig. 1). SbsB has an expected molecular weight of 98kDa and is represented on the gel as a large band. SbsB fragements or other proteins, which are able to bind to the Ni-NTA column are also visible. This purification is sufficient, because normal behaviour of the protein was observed in the chemistry lab.
        In a similar manner, a fusion protein consisting of SbsB lacking the first 30 amino acids and the minimum length Streptavidin needed for biotin binding (Core Streptavidin), was created. Core Streptavidin was linked to the N-terminus of SbsB via restriction assembly (for more detail see “Design” on our wiki). This sequence was cloned into the reading frame of pET28a(+) vector. The N-terminal His-tag was used for purification (Fig. 2). The Fusion protein’s estimated molecular weight is around 100kDa. On the gel, only a single protein band is visible running at around 85kDa. To verify our purified SbsB protein we conducted a Western Blot (see “Notebook” 04/25/2018).
        His-tag affinity chromatography was performed using one-way PE columns and Ni-NTA agarose from Qiagen. After column equilibration and sample application, the column was washed with binding buffer. Bound protein was eluted by applying a step-gradient with increasing imidazole-concentration (40mM in washing buffer, 125mM, 250mM, 375mM, 500mM). The eluate was fractionated into 1,5mL fractions and analyzed via SDS-PAGE (Fig.1, 2).</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 1. Purification of SbsB. </B>One-way columns containing 4ml of Ni-NTA agarose were used for purification. Percentages shown in lanes 4-10 describe concentration of imidazole used for elution (100% = 500mM imidazole). Eluate was taken in fractions of 1.5ml.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 2: Purification of Strav:SbsB. </B>One-way columns containing 4ml of Ni-NTA agarose were used for purification. Percentages shown in lanes 4-10 describe concentration of imidazole used for elution (100% = 500mM imidazole). Eluate was taken in fractions of 1.5ml.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse-2"><P class="model-center"><B>Isolation of PS2 from <I>Corynebacterium glutamicum</I> (S-Layer protein of p2-symmetry) </B></P></A></H4></DIV><DIV id="collapse-2" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">
        A 250ml overnight culture of <I>C. glutamicum</I> ATCC 14067 was grown for 24h at 30°C. For each sample 30ml of oD=1 were pelleted at 5000g for 5 min at room temperature. Pellets were resuspended in 1.5ml of 50mM TRIS-HCl (pH 6.8) or 50mM HEPES (pH 6.8). After incubation for 1h at room temperature the suspension was centrifuged (5000g, 5min, room temperature) and the supernatant containing the purified PS2 was transferred into fresh eppis and stored at 4°C. Purity was checked using SDS-PAGE (Fig. 3). The protein on the gel has an apparent molecular weight of 54kda which confirms the molecular weight of PS2.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 3. Isolation of PS2. </B>Isolation of PS2 was performed by lowering pH to 6.8 using Tris-HCl with 2% SDS.</P><P class="model">In the original isolation method for PS2 the buffer used for resuspension contained 2% SDS which caused problems in the chemistry lab. SDS causes micelles and artefacts under DLS leading to unclear results. Isolation without SDS showed smaller protein bands, however, fewer unspecific proteins were found (Fig. 4).</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 4. Isolation of PS2 using different SDS concentrations. </B>Different concentrations of SDS (0%-0.5%) in TRIS-HCl buffer (pH 6.8) were used to isolate PS2. </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse-3"><P class="model-center"><B>Isolation of RsaA from <I>Caulobacter crescentus</I> (S-Layer protein of p3-symmetry)</B></P></A></H4></DIV><DIV id="collapse-3" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">
RsaA, the p3 S-layer, was removed from the surface of <I>Caulobacter crescentus</I> using 5mL from an overnight culture (oD<SUB>600</SUB>=0,5). The cells were pelleted at room temperature at 10000g for 10min. The supernatant was discarded and the pellet was washed twice in 5mL 10mM HEPES buffer (pH7,2). The cells were pelleted, and the supernatant, containing RsaA, was collected. The pH of the supernatant was adjusted to 7 using 5M NaOH and later the supernatant was analyzed via SDS-PAGE (Fig 5). The gel shows a specific band at 98kDa, which we expected for RsaA.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 5. Isolation of RsaA</B>RsaA was isolated via lowering pH to 2.0 with HEPES buffer and pH was adjusted back to 7.0 using NaOH.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse-333"><P class="model-center"><B>Futher Experiments</B></P></A></H4></DIV><DIV id="collapse-333" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">We also tried to create fusion proteins of CFP/YFP and the S-Layer protein PS2 from <I>Corynebacterium glutamicum</I> (see “Design” page). These constructs could be used for FRET measurements to proof proximity affected by S-Layer self-assembly. We wanted to create fusion proteins capable of attachment to the cell wall of <I>C. glutamicum</I>. Therefore, neither N- nor C- terminal fusions were possible due to the distinct properties of those segments. The tertiary structure of PS2 is unknown, which made designing difficult. We tried to fuse CFP to three different positions in the coding sequence of the <I>PS2</I> gene. If the protein will be expressed in <I>E. coli</I>, a translation of a functionally N-terminal fusion proteins is possible. This assumption is made due to other already described S-Layer fusion proteins (e.g. Core Streptavidin and SbsB). Unfortunately, our team failed when we tried to assemble the different <I>PS2</I> fragments with <I>CFP</I>. </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse-4"><P class="model-center"><B>Outlook</B></P></A></H4></DIV><DIV id="collapse-4" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">
          With the mixture of different S-Layer proteins, we created the basis for new surface patterns, which can be used for biocatalysts. Another possible application of these surface pattern is the improvement of enzyme cascades by fusing enzymes to S-Layer proteins and bringing them into closer proximity. As a result, enzyme efficiency will be improved. A FRET assay with S-Layer proteins coupled to CFP and YFP will verify our concept. 
          For future perspective, we like to expand our SbsB:Streptavidin fusion protein to the third dimension to create inorganic nanotubes. Another application would be a fusion of biotinylated enzymes to the SbsB:Streptavidin fusion protein.
</P></DIV></DIV></DIV></DIV><H2 class="model">
Chemistry Lab
</H2><P class="model">The main goals of investigation was a thorough characterization of the three individual S-layer proteins in their buffer solution. Thereby, the size of the individual protein assemblies in solution as well as their mixtures could be determined with DLS experiments. The structure of PS2 (P2) S-Layer could be confirmed with atomic force microscopy imaging, without using fixation protocols and using the native protein solution.
Extinction coefficients of the individual proteins could be derived from the protein solution, which facilitates the determination of protein amounts in solution after purification with His-Tag via Ni-NTA column chromatography.
Further we could achieve an immobilization of SbsB on the surface of different Latex nanoparticles as Silica nanoparticles turned out to be not suited, which was tried by the
<A href="https://2011.igem.org/Team:Bielefeld-Germany">iGEM Team of Bielefeld in 2011</A>. The immobilization on the Latex nanoparticle surface could be attributed to the unpolar surface of latex, which is a better cell membrane mimic compared to silica. The size of the shell could be determined with DLS and TEM imaging as well as the covering could be demonstrated with steady state fluorescent spectroscopy. An equal result could be achieved for the streptavidin conjugates where, the interaction of streptavidin and the fluorescent label could be verified with steady state fluorescent spectroscopy as well as a native page. The interaction of biotin and streptavidin on the densely packed nanoparticle surface could be used for immobilization of biotinylated bio-catalysts.  </P><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse1"><P class="model-center"><B>Characterization methods</B></P></A></H4></DIV><DIV id="collapse1" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">For the light scattering experiment a CGS 3 goniometer (ALV Langen, Germany) with a 22 mW laser providing a wavelength of 632.8 nm and a ALV 5000 correlator with 320 channels were used. During the measurement the angles between 30° to 150° were covered in 10° steps. Data analysis of dynamic light scattering was taken out via transformation of the autocorrelation function of the scattered light into the autocorrelation function of the electric field using Siegert equation. The inverse Laplace transformation was carried out with the program CONTIN.  
UV/vis measurements were carried out with a Shimadzu UV-spectrometer (UV-1800). Quartz cuvettes with a pathlength of 1 cm and 1 mm purchased from Hellma/Müllheim were used respectively.
Fluorescence measurements were performed with a Horiba Yvon Jobin fluorescence spectrometer using a slit width of 2 nm and an integration time of 0.1 s. 

Transmission electron microscopy (TEM) images were obtained with a Zeiss EM 912 microscope, operated at 80 kV at magnifications from 20 000 to 250 000. The specimens were prepared by depositing 5 μL of the sample solution onto carbon-coated copper grids, 300 mesh, and air-dry the grids. The sizes of the particles were determined using the freely available software ImageJ. 

Spectroscopic ellipsometry data was collected from 400 to 1000 nm under a 70° incidence angle with an EL X-02 P Spec from DRE Dr Riss Ellipsometerbau GmbH. For fits, material files were used from the database which is provided with the instrument. 

Top-view scanning electron micrographs were taken on a Zeiss GeminiSEM 500 at different acceleration voltages, working distances and magnifications.
</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse2"><P class="model-center"><B>Extinction Coefficient</B></P></A></H4></DIV><DIV id="collapse2" class="panel-collapse collapse"><DIV class="panel-body"><DIV class="row"><DIV class="col-sm-4 text-center"><P class="model-center" style="margin-bottom: 30px;"><B>Figure 1.</B> Absorbance of SbsB @280 nm against concentration</P></DIV><DIV class="col-sm-4 text-right"><P class="model-center" style="margin-bottom: 30px;"><B>Figure 2.</B> Absorbance of PS2 @280 nm against concentration.</P></DIV><DIV class="col-sm-4 text-left"><P class="model-center" style="margin-bottom: 30px;"><B>Figure 3.</B>Absorbance of RsaA @280 nm against concentration</P></DIV></DIV><P class="model">To yield a reliable extinction coefficient for further at least ten soltuions with varying concentration were made, knowing the protein content from nanodrop analysis. For this purpose, SbsB was dialyzed against HEPES (0.01 M, pH 6.8) after purification via Ni-NTA column chromatography. The purity of the proteins was evaluated with SDS page, resulting in negligible amounts of side products. The absorbance of the protein at a wavelength of 280 nm was plotted against the concentration in mg mL-1. The proportionality of concentration and absorbance follows Lambert-Beer law, since the concentration is low and leads to a linear behavior. From the slope of the linear fit the extinction coefficients could be derived, which is 1.007 ml mg<SUP>-1</SUP> cm<SUP>-1</SUP> (SbsB, figure 1), 1.6 ml mg<SUP>-1</SUP> cm<SUP>-1</SUP> (PS2, figure 2) and 1.21 ml mg<SUP>-1</SUP> cm<SUP>-1</SUP> (RsaA, figure 3), respectively.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse3"><P class="model-center"><B>Stability of S-Layer Proteins and their Assemblies</B></P></A></H4></DIV><DIV id="collapse3" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">The organism <I>Geobacillus stearothermophilus</I> lives under elevated temperatures, which apparently has no impact on the formation of proper S-Layers on the cellular surface. Hence, we investigated the self-assembly process of the S-Layer protein SbsB at elevated temperatures. For this purpose, the protein samples were diluted to a concentration of 6.7x10-4 mmol/L and the samples were heated to 80°C, 90°C and 100°C, respectively.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 4.</B>Decay function of the electric field  (dots), distribution function of decaytime for SbsB @80°C (black), @90°C (red), @100°C (blue).</P><P class="model">The sizes of the aggregates turned out to be stable over the temperature range, resulting in hydrodynamic radii from R<SUB>H</SUB> = 20 nm (80°C) to RH = 29 nm (100°C) for the first decay time distributions. The peaks at higher decay times were almost maintained. However, with increasing temperature the amount of large clusters beyond R<SUB>H</SUB> = 1 µm seem to decrease indicated by the earlier decay in the decay function. </P><DIV class="row"><DIV class="col-sm-12 text-left"><P class="model-center" style="margin-bottom: 30px;"><B>Figure 5.</B>Decay function of the electric field  (dots), distribution function of decaytime for PS2 @80°C (black), @90°C (red), @100°C (blue); right Decay function of the electric field  (dots), distribution function of decaytime for PS2 control (black), @pH = 2.1 (red), @pH = 2.1 sonicated for 1h (blue).</P></DIV></DIV><P class="model">An equal experiment was carried out using PS2 resulting in constant values for all measurements between R<SUB>H</SUB> = 371 nm (PS2@80°C) and RH = 437 nm (PS2@90°C). The control measurement yielded a RH = 417 nm Beside thermal stability the proteins proved to be remarkably stable towards acid conditions at pH 2.1 and ultrasound, where the RH did not change significantly remaining at R<SUB>H</SUB>  = 495 nm and RH = 514 nm. An enhanced stability is essential for further coating experiments where the chemical precursors are highly reactive, which could potentially damage the protein structures. </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse4"><P class="model-center"><B>Impact of Molar S-Layer Protein ratio on Assembly Formation</B></P></A></H4></DIV><DIV id="collapse4" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">To create novel surface patterns the existing S-layer proteins were mixed systematically and the impact on the size of the assembly was detected with DLS measurements.
Measurement series were conducted by mixing two of the three mentioned S-Layer proteins, SbsB with 98 kDa, PS2 with 52.5 kDa and <I>RsaA</I> with 98 kDa in a, with a 0.01 M HEPES at pH = 6.8 buffered, solution. Throughout the measurements, the total protein concentration c = 6.67x10-4 mmol/L remained constant while the ratio of both proteins was varied in 0.1 steps.
</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 7.</B>The radius with deviation is plotted against
the ratio of S-Layer &quot;P2&quot; to S-Layer &quot;P3&quot;
</P><P class="model">The hydrodynamic radii of the SbsB/PS2 mixtures show a strong decrease of size for equimolar ratios. The radius of pristine SbsB aggregates deviates from this behavior with a RH = 245 nm. By adjustment to 9:1 (SbsB:PS2) ratio the particle size increases to RH = 553 nm. From this ratio the radii are approaching a minimum of RH = 153 nm (0.6 SbsB:PS2). From then the radii are increasing to the original value of PS2 assemblies. Hence, it can be assumed that the S-Layer proteins are mixing and the size determining protein has to be added in excess.  </P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 8.</B>The radius with deviation is plotted against
the ratio of S-Layer &quot;P2&quot; to S-Layer &quot;P3&quot;
</P><P class="model">(Fig.8) The objects in the solution containing only “P1” SbsB had a radius of around 900 nm and a radius of approximately 700 nm containing only “P3” RsaA. The radii stayed nearly constant throughout the series and only moved between 750 nm and 900 nm, except of one highly increased radius.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 9.</B>The radius with deviation is plotted against
the ratio of S-Layer &quot;P1&quot; to S-Layer &quot;P3&quot;
</P><P class="model">Because of the same protein size of 98 kDa, “P1” and “P3” have and the nearly constant radius we got mixing both proteins we can assume, that the size of the S-Layer proteins play a huge role on how big protein micelles get by self-assembly, but it can’t be by far the only attribute that matters. Regarding that mixing “P1” and “P2” or “P2” and “P3” leads to an opposite behavior in size change, intermolecular forces have to play a huge role as well.</P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse5"><P class="model-center"><B>Immobilization of S-Layer Proteins on Latex Nanobeads</B></P></A></H4></DIV><DIV id="collapse5" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">We deposited one type of S-Layer protein on latex beads to analyze if the protein attaches on the latex bead by hydrophobic interactions. Thereby, we used different sizes of Latex nano beads, namely 30 nm, 50 nm, 100 nm and 350 nm to investigate if the attachement depends on geometric issues of the protein, as smaller nanoparticles should not be covered completely. The size of the nanoparticles thereby reaches the size of the proteins, which makes a complete coverage unlikely and rises the chance of an integration into the system.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 9.</B>Decay function of the electric field  (dots), distribution function of decaytime (line) for PS2-Latex nanobead mixtures with 1:100 dilution of latex nanobeads (black), 1:1000 dilution of latex nanobeads (350 nm) (red), 1:10000 dilution of latex nanobeads (blue).
</P><P class="model">At first, a constant concentration of the S-Layer “P2” PS2 was applied on varying amounts of of the Latex beads. Thereby, the radius of the particles increases from almost the original radius of R<SUB> = 167 nm (1:100) to RH = 267 nm (1:10000). This is an indication that S-layer proteins attach to the latex by stacking on  multiple layers onto each other, which was demonstrated by the simulation. If the latex nanoparticles would simply be integrated into a hollow sphere of a larger PS2 assembly, the radius should be independent of the amount of latex and the original size of the “hollow sphere” should be found, which was determined previously at R<SUB>  = 405 nm. A measurement with smaller latex nano beads (100 nm diameter). In DLS the radius of 52.2 nm was measured whereas the mixture with “P2” PS2 increased the radius to 62.7nm.</SUB></SUB></P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 10. </B>Schemtaic drawing of S-Layer protein covered latex beads
</P><P class="model">We repeated the measurement with even smaller latex particles and the S-Layer “P1” SbsB which gave us a radius of 30.3 nm for the solution with only latex beads and a radius of 54.6 nm for the mixed solution with “P1” SbsB and latex beads (Figure 11 b). This supports the hypothesis of the proteins building layers, which should be demonstrated in Fig.10. To further undermine the hypothesis, fluorescent latex beads with a size of 50 nm was used and measured in presence and absence of SbsB via Steady-State Fluorescence Spectroscopy. Thereby, the particles not containing protein solution provided a 16 % lower emission than, particles including SbsB. The increase of emission might directly originate from shielding the fluorescent dye from the aqueous medium, which prevents other relaxation mechanisms including the solvent.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 11.</B>Steady State Fluorescent spectra of 50 nm latex beads (black) and 50 nm latex beads with SbsB (red), Decay function of the electric field  (dots), distribution function of decaytime (line) of 50 nm latex beads (black) and 50 nm latex beads with SbsB (red).
</P><P class="model">Furthermore, the same experiment was conducted with the RsaA S-layer protein and latex nanobeads with a diameter of 100 nm. Using RsaA lead to the same results as obtained for the other S-layer proteins, SbsB and PS2 (Figure 12 a) For the RsaA / latex mixture, latex beads of 100 nm were used. In DLS a radius of 64 nm was obtained for this mixture. This shows that the s-layer proteins, again, have to surround the latex beads as shown in figure 12.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 12. </B>a Decay function of the electric field  (dots), distribution function of decaytime (line) of 100 nm latex beads with RsaA (red) and b) corresponding TEM image.
</P><P class="model">In the TEM image (figure 12 b) a  slight shadow (1) surrounds parts of the particles which might be the organic protein matrix. However, other particles do not provide such an organic shadow (2) and provide the pristine size of the nano beads of 100 nm. </P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse6"><P class="model-center"><B>Immobilization of PS2 on Inorganic Surfaces</B></P></A></H4></DIV><DIV id="collapse6" class="panel-collapse collapse"><DIV class="panel-body"><P class="model-center" style="margin-bottom: 30px;"><B>Figure 13. </B>AFM images of PS2 S-Layer protein deposited on a Mica platelet.
        </P><P class="model">
For investigation of S-Layer structuring on the surface we investigated the structure of PS2 deposited on various grid materials. Thereby Silica and coated silica with a self assembled hydrophobic material did not lead to the desired results. PS2 at a concentration of 0.3 mg/mL deposited on a cleaned mica platelet resulted in a highly structured surface with average pore sizes of 9.7 nm. The size of the pores lies above the sizes of typical S-Layer covered surfaces (Sleytr et al. 2011). Further, the pores are irregular aligned in their position on the surface. This might be attributed to not using fixation agents, which is common for the preparation of S-Layer coated surfaces (Sleytr et al. 2011). Due to the direct coating the chemical integrity of the protein can be maintained, which is beneficial for further processing of the surface.
</P><P class="model" style="padding-top: 30px"><I>Reference:
  Sleytr, Uwe B.; Schuster, Bernhard; Egelseer, Eva M.; Pum, Dietmar; Horejs, Christine M.; Tscheliessnig, Rupert; Ilk, Nicola (2011): Nanobiotechnology with S-layer proteins as building blocks. In Progress in molecular biology and translational science 103, pp. 277–352. DOI: 10.1016/B978-0-12-415906-8.00003-0.
</I></P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse7"><P class="model-center"><B>Coating of S-Layer Protein Surfaces</B></P></A></H4></DIV><DIV id="collapse7" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">A thin layer of the PS2-protein has been deposited onto a silicon wafer as described above. SEM micrographs were taken at different magnifications and different accelerations voltages to find out if the protein-layer survives the measuring conditions inside an SEM.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 14. </B>Scanning electron microscopy images of the surface of one silicon wafer with deposited p2-protein layers. 2000x magnification, ETH at 1.00 kV, working distance at 4.6 mm. The bar is 20 μm.
</P><P class="model">Figure 14 shows the resulting surface at 2000x magnification. It shows irregular patterned patches with varying sizes, alongside sharp-edged pieces which are approximately 5 to 10 µm in diameter, which can be attributed to drying effects from the buffer salts. On (or under) most of these pieces, spheres with a diameter of approximately 2 µm are observed. The size and shape of these spheres could correspond to the data obtained by DLS, if the shape could be conserved during the drying procedure.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 15. </B>Magnified scanning electron microscope image of the same area as in Figure 14. 5000x magnification, ETH at 1.00 kV, working distance at 4.6 mm. The bar is 2 μm.
</P><P class="model">Figure 2 shows a 5000x magnified SEM image of the same area which is shown in Figure 1. Further magnifications resulted in beam-related contamination of the area. The contamination is made clearly visible by the grey rectangle in the middle of the image which forms after a few seconds at higher magnifications. Additionally, at higher magnifications, focusing or recording micrographs was nearly impossible due to local charging of the sample. Usually biological SEM samples require a metal or conductive coating to avoid this charging effect.<SUP>1</SUP>
In a second attempt, first, the p2-layer has been deposited onto a cleaned FTO glass as described above. Subsequently, a 5 nm thick gold layer has been deposited onto the protein-layer via DC sputter coating and the resulting substrate has been examined by SEM. The top-view of the surface is shown in the SEM micrograph in Figure 3.
</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 16. </B>SEM-micrograph of the top-view of an FTO glass substrate with a thin layer of p2-protein with a 5 nm gold layer on top. The deposited layers are ruptured. Magnification at 19 180x, EHT 5.00 kV, working distance 2.2 mm. The bar is 1 μm.
</P><P class="model">16 shows the surface of the substrate after approximately 5 seconds at a 19 180x magnification. The surface starts out flat but ruptures shortly after being hit by the focused electron beam of the SEM tip. It was not possible to record SEM micrographs of the flat surface due to the fast-occurring rupturing. This infers that the gold coating does not suffice to protect the layer under the thin Au layer. Electrons from the focused electron beam interact with the electrons surrounding the sample’s atomic nuclei which causes heating and/or structural damage – especially in organic materials. This results in breaking chemical bonds in the sample molecules and subsequently in a change of their shape and/or position.<SUP>2–5</SUP> This change seems to be enough to rupture the thin gold layer on top of the organic p2-layer.
Lastly, a PS2-layer was deposited onto a silicon wafer as described above. An approximately 15 nm thick layer of ZnS was deposited onto the substrate via atomic layer deposition. Again, the resulting surface was examined by SEM imaging. However, it was not possible to get an SEM micrograph with sufficient focus or magnification to present reasonable results.
With the data at hand, it cannot be confirmed nor confuted if the ordered assemblies of the S-layer proteins can be used as a matrix for thin film deposition methods. SEM does not prove as an appropriate method to investigate these matters. For further experiments, it is advised to change to different state of the art imaging techniques. For instance, AFM might allow the needed nanoscale imaging, under less harsh conditions.</P><P class="model" style="padding-top: 30px"><I>Reference:
  1	P. Echlin, Handbook of Sample Preparation for Scanning Electron Microscopy and X-Ray Microanalysis, Springer-Verlag US, Boston, MA, 2009. 
  2	D. T. Grubb, Radiation damage and electron microscopy of organic polymers, J Mater. Sci., 1974, 9, 1715–1736. DOI: 10.1007/BF00540772. 
  3	R. F. Egerton, Mechanisms of radiation damage in beam-sensitive specimens, for TEM accelerating voltages between 10 and 300 kV, Micr. res. tech., 2012, 75, 1550–1556. DOI: 10.1002/jemt.22099. 
  4	R. F. Egerton, P. Li and M. Malac, Radiation damage in the TEM and SEM, Micron, 2004, 35, 399–409. DOI: 10.1016/j.micron.2004.02.003. 
  5	B. Titze, Dissertation, Ruperto Carola University of Heidelberg, 2013. </I></P></DIV></DIV></DIV></DIV><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse8"><P class="model-center"><B>Experimental Setup ALD coating with ZnS</B></P></A></H4></DIV><DIV id="collapse8" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">Unless stated otherwise, all fluorine-doped tin oxide (FTO) conductive glasses were cleaned with soap and DI water. The substrates were then blown dry with N2 gas before they were cleaned ultrasonically in acetone, 2-propanol and DI water and finally blown dry again by N2 gas. 
Silicon wafer were first cleaned with 2-propanol followed by DI water and then blown dry by N<SUB>2</SUB> gas. 
40 μL of a PS2 (0.01 M HEPES, pH 6.8, from 02.08.18) was given onto the corresponding substrates and left for 10 minutes. Afterwards, excess solution was removed and the substrates were left for drying for one hour in air before being processed further.
Au thin films were deposited via DC sputtering at 0.6 Å s<SUP>-1</SUP> (power: 50 W; Argon gas flow: 30 sccm; chamber pressure: 10 x 10<SUP>-2</SUP> torr; model CRC 622 by Torr International Inc.). The thickness of the deposited films was controlled by a quartz crystal rate controller. 
Atomic layer deposition was carried out in a self-built hot-wall reactor equipped with DP-series pneumatic valves from Swagelok and with an MV10C pump from Vacuubrand. ZnS was deposited with Et2Zn and H<SUB>2</SUB>S (3vol% in N<SUB>2</SUB>). The reactor temperature was kept at 60 °C. The precursors were kept at room temperature. Et<SUB>2</SUB>Zn was pulsed into the reaction chamber for 0.2 s and left for 15 s. Afterwards, the reactor was purged for 25 s using N<SUB>2</SUB> gas. H<SUB>2</SUB>S was pulsed into the chamber for 0.2 s and held for 15 s before opening the pump valve and purging for 25 s. The chamber was equipped with a protein-free piece of silicon-wafer as a reference for the thickness of the layers deposited, which was then measured via spectroscopic ellipsometry.</P></DIV></DIV></DIV></DIV><H2 class="model">
Modelling
</H2><DIV class="panel-group" id="accordion"><DIV class="panel panel-default"><DIV class="panel-heading"><H4 class="panel-title"><A data-toggle="collapse" data-parent="#accordion" href="#collapse-53"><P class="model-center"><B>Discussion</B></P></A></H4></DIV><DIV id="collapse-53" class="panel-collapse collapse"><DIV class="panel-body"><P class="model">
          Unfortunately, no reliable statements can be made with the help of the results obtained with our simulation: This can be traced back to the pure complexity of the S-Layer proteins. The more constituents are involved in the system, the higher the required number of time steps that is needed in the simulation until a state of thermal equilibrium is reached, i.e. a configuration with minimized system energy.  
          
          In figure (1) and (2) we present the average of the proteins’ distance to the surface in dependence of the system temperature and the electromagnetic charge of the surface for a single S-layer type (p1) and two types (p1 and p3), respectively. The fluctuations of the configurations can clearly be seen in both cases, since the behavior, i.e. the average distance visualized by the brightness in the heat map, varies too much in the neighborhood of similar macroscopic parameters. The simulations were repeated for the same parameters multiple times and the resulting average distances also showed largely varying behavior.
          Further simulations with a quite higher number of time steps need to be conducted in the future.
</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 1</B>The average distance of the proteins with respect to the surface for a single S-layer type system (p1) as heat map: Bright values represent larger distances, darker values smaller distances.</P><P class="model-center" style="margin-bottom: 30px;"><B>Figure 2:</B>The average distance of the proteins with respect to the surface for a mixture S-layer type system (p1 and p3) as heat map: Bright values represent larger distances, darker values smaller distances.</P></DIV></DIV></DIV></DIV></DIV><FOOTER class="container-fluid text-center"><DIV class="row"><DIV class="col-sm-4"><H3> Contact Us</H3><H4>project-igem@fau.de</H4></DIV><DIV class="col-sm-4"><H3>Connect</H3></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>