Team:FAU Erlangen/Notebook
iGEM Erlangen
Toggle navigation
Home
Team
Team Members
Collaborations
Project
Description
Design
Experiments
Notebook
InterLab
Model
Results
Demonstrate
Attributions
Parts
Parts Overview
Basic Parts
Safety
Human Practices
Human Practices
Education & Engagement
Awards
Measurement
Model
Judging Form
Notebooks
Bio Lab
Lab Members
Week 1 (April 16 - April 22)
04/16/2018 (TM,NB)
Media production
1l of fluid CgXII medium and 1l BHI Agar
500ml solid medium (containing 6g Agar) + 500ml fluid media
1500ml solid medium (for cloning) + 500ml fluid medium
Plating
poured C. glutamicum DSM 20411 (=ATCC 14067) [03/21/2018] -> growth: 30°C
poured C. cresentus CB15 ATCC 19089 PYE [03/08/2018] -> growth: 30°C
04/17/2018 (TM)
Production of CgXII + N for inoculation
04/18/2018 (TM)
PS2 isolation (p2, from C. glutamicum)
Overnight culture oD600: 2,25 -> set on oD1: oD1/oD2,25= 0,44ml
3 samples for protocol (1) (iGEM protocol) & 3 samples for protocol (2) (microbiology)
[Instead of 1h at room temperature, intubate 5min at 100°C]
Centrifuge 5min at 11000 rpm room temperature=RT
Discard supernatant
Solve pellet in 50mM Tris + 2% SDS in 500μl
Incubation: (1) 1h 37°C, (2) 5min 100°C
Transfer supernatant in new 2ml Eppis
Store at 4°C
RsaA isolation (p3, from C. crescentus)
HEPES Buffer for RsaA Isolation
HEPES Buffer (Stock): 100ml 100mN pH 7,2
2,33g HEPES
Dissolve in 80ml H2O
Check pH (~5,0)
Adjust pH to 7,2 using NaOH pellets (~1)
Complete to 100ml with H2O
HEPES Buffer 10mM pH 7,2: dilute Stock 1:10 -> filter sterile
HEPES Buffer 10mM pH 2: dilute Stock 1:10/ adjust pH2 using HCl -> filter sterile
oD600= 0,5 (not enough)
centrifugation 8000g 10min
solve in 10ml: oD600 = 1,35
1,35 * x = 5ml * 0,6 -> x=2,2ml
4 samples, 4th with ~ 1,2ml
Centrifugation 10.000g for 10min
2x washing in 100mM HEPES pH = 7,2 and 10.000g 10min
Resuspend in 100mM 200μl HEPES pH = 2
10min incubation at 20°C
Pellet cells & adjust pH7 with 5N NaOH
04/18/2018 (TM)
Plating
SbsB WT2 (p1 S-layer; Ampicillin-restistence) & SbsB ΔN(AmpR) on LB plates containing Am -> overnight at 37°C
Nanodrop: Concentration measurement PS2
PS2 (C. glutamicum):
RsaA (C. crescentus):
Conclusion different inoculations during PS2 isolation
1h 37°C: Less yield, but protein structure remains
5min 100°C: higher amount of yield
04/19/2018 (FE,SB)
Isolation of PS2
Overnight culture: 10min, 5000rpm
resuspend pellet in 10ml
oD600 = 38,2 -> 0,025ml = 25μl in 1ml for oD600 = 1 -> oD600 = 1,15
5000g 5min, room temperature, Discard supernatant
3 dissolved in 250μl 50mM Tris + 2% SDS (3), other 3 dissolved in 500μl 50mM Tris + 2% SDS (4)
Incubate 1h at room temperature
5min 5000xg room temperature
On ice
Nanodrop: Concentration measurement PS2
-> Increased yield by dilution in less buffer (250µl 50mM Tris + 2% SDS)
SDS-Gele for protein analysis
Poured 2 SDS-Gels
SDS gele to check PS2 and RsaA isolation (see next day)
SbsB inoculation (Plasmids in E. coli)
SbsB WT2 & SBSS ΔN in 20ml LB fluid with 20μl Amp inoculated, shake overnight at 37°C. Next day: undying with 10% acetic acid (while shaking)
04/20/2018 (FE,SB)
Isolation of RsaA
oD600 = 0,53
6x 6ml 10min 10000g
Dilute HEPES Puffer 1/10 (= 0,01M)
Wash 2x in 10mM HEPES pH 7,2; 10000g 10min
Resuspend In 200μl 10mM HEPES pH 2,0
Incubate 10min RT (20°C)
Transfer 21,5ml Eppis (3x 200μl each)
10min at 13000rpm
supernatant in new 1,5ml Eppi
Adjust pH to 7,0 with 1M NaOH
Nanodrop: Concentration measurement SbsB WT/ ΔN and RsaA
Analysis of SDS-gel from 04/19/2018
SbsB Protein Extraction from E. coli (SBSB WT2, SBSB ΔN)
20ml overnight culture 4000g for 10min
Discard supernatant
Resuspend pellet in 1ml PBS buffer
Centrifugate 8min at 13000rpm at 4°C
Discard supernatant
Resuspend pellet in 500μl 1xPBS + EDTA (40Δl in 500μl 1xPBS)
Suspend in 2ml Cryotube with 300mg glass beads
in extraction machine 30s, 6,5m/s
repeat 4 times, in between: 2min on ice
centrifugate 1h at 4°C with 1300rpm
1h 5000rpm at 4°C
Transfer to new 1,5ml Eppi and store at -20°C
Week 2 (April 23 - April 29)
04/23/2018 (FE,MI)
SDS gel for Western Blot for SbsB WT2 and SbsB ΔN
Order scheme: Protein ladder prestained – free – SBSB WT2 – free – SBSB ΔN
SbsB WT2 & ΔN vector pHEN6 by IPTG
Let E. coli grow again with SbsB WT2/ ΔN
Store 20ml LB + 20Vl Amp + 20Δl IPTG 1M overnight at 35°C + control for His-Tag: E.coli DH5 & MR + GR + His in 20ml LB + 20Δl Cm25 + 20Δl IPTG
SDS-gel dying overnight (with Coomassie Blue)
Primer
ordered Hexokinas fw + rev, Invertase fw + rev; Streptavidin fw + rev Primer
04/24/2018 (FE,MI)
Westernplot
• Produced 1x transfer buffer (1l) and 60% Methanol (50ml)
Sample loading like on SDS gel (10μL sample + 10μl Ladepuffer PAP)
For ~1,5h at 120V
• Soak Filter paper (8 sheets for 1 gel) in transfer buffer
• Cut out PVDF membrane in size of the filter paper
• Soak membrane in 60% methanol for 5min
• Wash in water
• Wash PVDF membrane for 5min in 1x transfer bufferv • Gel sandwich: 3 paper sheets, PVDF membrane, SDS-Gel, 5 paper sheets (up to down)v • For 1h at 120mA (2 gels 230mA, 3 gels 300mA)
• Discard Gel and filter paper
• Wash membrane 3x in 1x TBST buffer for 15min
• 1x TBST buffer: 999ml 1x TBS + 1ml Tween 20 (0,1%)
• 10x TBS buffer: 0,198M Tris (23,98g), 1,5M NaCl (87,66g), pH = 7,6
• Membrane in Blocking buffer 1h at 4°C (30ml)
• Blocking buffer: 5g milk powder, 100ml TBST
• Incubate membrane with primary AK üN at 4°C 1:5000
• Wash membrane 4 times for 15min with TBS-T
• Shake membrane with secondary antibodies (1: 10000, 20ml blocking buffer + 2μl antibodies) for 1h at 4°C
• Wash 4 times for 15min with TBS-T at room temperature
• Pipette 1ml BCIP/ NBT one after another until the membrane is completely moistened (~3ml)
• BCIP/NBT => 10ml incubation buffer + 65μl BCIP + 65μl NBT (-> store in the dark)
• Incubate (in the dark) 1min -1h (often: ~30min)
• Stopping of the reaction with water
04/25/2018 (TM,MI)
Results of the first Western Blots
DH5α MR GR + His and SBSB WT2 shows a “lubricated bands”, but no specific band for the His-Tag. No detection at SBSB ΔN -> Repeat Western Blot with lower concentrations (20μg protein on gel)
Nanodrop: Concentration measurement:
1. Load SDS gels + prepare samples
• Clamp in gels
• Put gels in overflow chamber and fill between the gels with 1xCathode buffer and on the outside with 1xAnode buffer
• Remove comb and apply prepared samples
• Loading of gels with glass syringe
• Prepare Samples:
• Transfer each sample with loading dye = Lämmli buffer 1:1 (2x PAP)
• Incubate 5min at 100°C
• Standard: protein marker = Page Ruler Unstained ~5ml, protein marker for Western Blot = Page Ruler Prestained
-> Western Blot: ~1,5h at 120V
2. Blotting
1x Transfer buffer: 1l
100ml 10x Transfer buffer
200ml Methanol (99,99%)
700ml VE water
60% Methanol: 50ml
30ml Methanol (99,9%)
20ml VE water
• Adjust filter paper (=Whatmanpaper) and PVDF Membrane
• Soak PVDF membrane for 5min in 60& Methanol
• Wash PVDF membrane with water
• Wash membrane 5min in 1x Western Transfer buffer
• Soak filter paper in 1x Transfer buffer
• Building of gel”sandwich“ and removing of bubbles
• For 1h at 120mA (2 gels: 230mA, 3 gels: 300mA)
• Discard filter paper, perhaps coloring of gel
• Wash membrane 3x 15min in 1x TBST
3. Blocking
Incubate membrane 1h in Blocking buffer at 4°C (= 5% Skimmed milk powder in TBST, 5g Skimmed milk powder in 100ml TBST)
4. Incubation
• Incubate membrane with primary AK ü/N at 4°C
• Normally: 1:5000 (in blocking solution)
• This time: 1:3000 (10ml +4μl α-His)
• Wash membrane 4x 15min with TBST
• Shake membrane with secondary AK (1:10000, 20ml blocking buffer + 2μl AK) for 1h at 4°C
• Wash membrane 4x 15min with TBST
04/26/2018 (SS, TM)
5. Detection
• Prepare fresh: 5ml incubation buffer with 33μl NBT = nitro blue tetrazolium (50mg/50ml) and 16,5μl BCIP = 5-bromo-4-chloro-3-indolphsophatase (50mg/50ml) -> 25mg/ml = 33μl BCIP
• Insert ~3ml one after another until membrane is fully covered
• Incubate 1min-1h in the dark until coloring is sufficient (~30min)
• Stop reaction with water
• Dry membrane
Western Blot 1:
Westen Blot 2: repetition with 20μg protein, increased concentration of primary antibody (1:3000) and BCIP/NBT
Conclusion: Repeat SBSB ΔN extractiot
Primer
Primer 1-6 solved & diluted 1:10, primerbox in Freezer
• Stock solution: 100pmol/μl
• 1:10 dilutions 10pmol/μl  0,01nmol/μl
Preparation for the protein extraction of SBSB N:
20ml LB + 20μl Amp + 20μl IPTG
Preparation for protein purification of SBSB WT2
100ml LB + 100μl Amp + 100μl IPTG -> called WT 2.1 on the following pages
100ml LB + 100μl Amp without IPTG -> called WT 2.2 on the following pages
04/27/2018 (SS, SB)
Media production
• LB 2l
• BH1 1l
SbsB Protein Extraction:
Overexpression for the first purification
• 1l LB + 20ml SBSB with IPTG
• + 1ml 1M IPTG -> c= 1mMol
• + 1ml Ampicillin -> c= 1mMol
• 1l LB + 20ml SBSB without IPTG + 1ml Ampicillin
• oD600 measurements:
Media production
• Add 1ml IPTG to flask (containing 1l LB + 20ml SbsB without IPTG + Amp) after 1,5h
• Shake both flasks for 1,5h at 37°C
• Distribution of 1l LB + SBSB + Amp (+) IPTG (27.04.2018) WT2.2 and 1l LB + SBSB + Amp + IPTG (04/26/2018) WT2.1 on centrifuge bottles
• Centrifugate at 4000rpm for 10min at 4°C
• supernatant discarded -> store pellet overnight/weekend at -20°C
Week 3 (April 30 - Mai 6)
04/30/2018 (SS, DR)
SbsB Protein Extraction:
Protein purification with Ni-NTA
• Solve pellet in PBS (20ml) [Better: 16ml and on two columns]
• Pellet and solve in 8ml linking buffer
• + complete-Protease-inhibitor
• centrifuge cultures at 8500rpm for 30min
• Production of buffer A (1l): 50mM NaH2PO4 (6,9g) + 300mM NaCl (16,62g)
• 30mM Imidazole (of 1mM Stock)
• Pellets solved in 8ml buffer A
o + Roche complete (EDTA)
o +30μl Lysosyme (from staek)
• Sonificierung:
o 5x 30s
o 50% intensity
o 5min rest on ice
• Centrifugate 30min at 8500rpm
• 4ml Ni-NTA Agarose
• Prepared:
o 4ml Ni-NTA
o Wash gel columns 3 x with buffer A
• Gele column 3 times wash with buffer A -> in fridge
05/02/2018 (FE,DR)
Preparations
• Pour 3 gels
• Production of BHI-medium (2l): Brain Heart infusion (74g for 2l)
• Production of LB medium (2l)
SbsB Protein Extraction
Continuing Protein purification with Ni-NTA
• Wash columns 3x with washing buffer
• Elution with elution buffer: With dilutions
• 25% (fraction 1), 50% (fraction 2), 75% (fraction 3), from fraction 4 on to 100 use undiluted elution buffer
• Apply on SDS gel
Concentration measurement:
SDS- geles
SbsB WT 2.1:
SbsB WT 2.2:
Conclusion: SBSB WT2 protein (p1 S-layer) is 98kDa; His-Tag (1kDa) -> expected protein size: 99kDa
SDS cannot be classified exactly in the higher columns (200-100kDa)
Observable: protein has weight between 120kDa and 100kDa (expected protein: 99 kDa)
Inoculation:
250ml C. glutamicum in BH1-Medium -> overnight at 30°C
250ml C. crescentus in PYE -> overnight at 30°C
05/03/2018 (FE,SB)
PS2 isolation (so far best method)
• oD600 = 8,17 -> use 1,22ml overnight culture filled up to 10ml BHI medium (oD1) )
• 18x 10ml (oD1) [20ml 1mg/ml needed]
• Add 500μl Tris 50mM + 2% SDS for 1h at room temperature after resuspension
• Centrifugate for 5min at 5000g at room temperature
• Store at 4°C in the fridge
Nanodrop: )
Concentration measurement: PS2:
SDS gel for PS2 isolation:
• Prepare isolated samples of C. glutamicum for gel application (10μl protein + 10μl loading buffer -> 5min at 100°C in PCR machine
05/04/2018 (SB,NB)
RsaA isolation (C. crescentus):
Production of HEPES 0,1M = 100mM (= stock solution) and 10mM HEPES pH 7,2 and 2,0
• RsaA isolation from 200ml C. crescentus overnight culture (4x 50ml Falcon)
• Used oD: 0,9 (average of 3 values)
• For pH 7 at the end, 25-30μl 1M NaOH required
• Remaining steps as before
Bradford- test from SbsB (p1 S-layer protein from E.coli plasmid)
• Calibration curve with BSA
• BSA stock solution: 2mg/ml
• BSA for measurement: 0,2mg/ml
• Prepare everything 3 times & later form average value of calibration curve
• Vortex preparations, 20min at room temperature, vortex
• Identify oD595nm
• Reference is preparation without BSA (only 800μl buffer + 200μl Bradford)
• Additionally: oD595nm with elution buffer (contains 500mM Imidazol)
oD595nm with elution buffer (contains 500mM Imidazol):
oD595nm with washing buffer (contains 50mM Imidazol):
Week 4 (May 7 - May 13)
05/07/2018 (MAS, NB)
Nanodrop: concentrations of RsaA (C. crescentus):
Blank: 10mM HEPES pH 7,2
RsaA SDS-gele:
Inocultation: C. crescentus in 250ml PYE
05/08/2018 (MAS, NB)
SbsB (WT 2.2) concentrations:
pZ8-1 Digestion:
Dephosphorylation:
05/09/2018 (RV, FE)
RsaA isolation (C. crescentus)
• oD600 = 0,920
• centrifugate at 8000g for 10min (4x 50ml tubes)
• Resuspend pellet in 20ml PYE
• Generate 3 replicates à 5ml of each 20ml and centrifugate 10000xg for 10min
• Remaining steps like before
Retransformation of pZ8-1 in E. coli (DH5α)
•  100μl DHα + rest of the vector PZ 8-1
• 30min on ice
•  Heat shock: 1min at 42°C
•  5min on ice
•  Add 400μl LB (+control: without vector)
•  Shake 1h at 37°C and 700rpm in heating block
•  1300rpm, 1min -> tip over the supernatant, resuspend pellet in the rest
•  Plating out on LB +Kan 60μg/ml
•  Overnight at 37°C
Preparation:
Inoculation of C.glutamicum in 250ml BHI & incubation at 30°C and inoculation of 250ml of C.crescentus PYE (30°C for 48h).
05/11/2018 (RV, FE)
PS2-isolation from C. glutamicum
oD600=12.07 -> for oD600=1 (10ml): 0.83ml overnight culture + 9.17ml BHI (18 tubes). Remaining steps: see 03.05.18. We tried to solve the pellet in different buffers: 50mM Tris without SDS/ 50mM Tris + 0.2% SDS and 50mM Tris + 0.5% SDS (each treatment was performed with 6 falcons)
PS2 test gels (isolated with different SDS concentrations):
Preparation RsaA isolation (from C. crescentus):
Take out 20ml of the over-night culture of the 9th of May and switch it to another 500ml flask (additional 250ml PYE). Growing cultures in both flasks at 30°C over the weekend
Week 5 (May 14 - May 20)
05/14/2018 (DR, RV)
Preparation of pZ8-1 extraction:
Preparation for a midi-prep: inoculate 1 colony in 4ml LB (+ kanamycin) (-> 37°C). Inoculation of the starter culture in 100ml LB + kanamycin (100μl) and incubation at 37°C in the incubator
RsaA isolation (from C. crescentus):
oD600nm: ~1.3. Remaining steps according to protocol 20.04.18.
IDT sequences
Centrifugate DNA at 13.000 rpm for 5 sec., add showed volume mQ, vortex probe and incubate 20min at 50°C.
Cloning: Hxk1(cInv/Strav)
EcoRI digestion
-> Incubation for 1.5h at 37°C -> Heat-inactivation
Dephosphorylation of pZ8-1: 50μl from restriction reaction-mixture + 1μl CIP and 30min/37°C incubation. Loading on 1% agarose gel and clean-up & gel extraction (Nucleo Spin Kit), according to manufacturer instructions
Ligation: Hxk1/cInv/Strav in pZ8-1 (X= (3*5)/cInsert)
Transformation in E.coli: According to protocol 9.05.18 (plate on LB-kanamycin plates: 37°C). Additional to the three constructs a negative control (empty pZ8-1 was transformed).
05/15/2018 (DR, MI)
pZ8-1 midi-prep:
Measuring of overnight culture oD600: 2.8. Separate it in two tubes (50ml). Xtra Midi-Kit (according to manufacturer instructions).
-> very low concentration maybe due to the pellet loss during drying
05/16/2018 (FE, NB)
Plate preparation:
Cast 500ml LB + kanamycin plates. Picked 5 colonies for each construct (Hxk1/cInv/Strav) and inoculated in 4ml LB + kanamycin (37°C overnight)
05/17/2018 (SB, NB)
Mini-Prep (Hxk1/cInv/Strav in pZ8-1)
Isolation of plasmids (pZ8-1+Hxk1, pZ8-1 +cInv, pZ8-1 Strav) with NucleoSpin-Kit according to kit protocol. Annotations: 5th step: Washing of silica membrane with 500μl AW-Buffer @65°C in incubation-chamber (see: script „recommended “)
Conclusion: Hxk3, Dtrav 1 and 3 are probably positive.
Preparation for SDS-gels: 500ml Schrägger-Buffer (3x):
05/18/2018 (SB, NB)
Miniprep of cInv, Hxk1 and Strav constructs
According to kit protocol
Week 6 (May 21 - May 27)
05/22/2018 (DR, RV)
Repetition EcoRI-digestion and dephosphorylation
-> Incubation 1h 37°C
pZ-81 dephosphorylation: 1μl CIP added to 30μl restriction approach and digested for 30min at 37°C. Later: heat inactivation 100°C for 10min.
-> DNA degraded. Repeat necessary (by Renata’s protocol)
To prove that DNA was not degenerated ahead of the digest. Concentration of pZ8-1: 545.6ng/μ μl
Test gels for previous isolated RsaA and PS2 (different SDS concentrations):
Precipitated proteins were solved by inverting and checked by SDS-gels
05/23/2018 (DR, SB)
Repeat EcoRI digestion and dephosphorylation of pZ8-1 (with RAPID):
-> 90min, 37°C
-> 30 min 37 °C
Cloning Hxk/Inv/Strav in pZ8-1:
Formula: insert [ng] =[5*(vector[ng] * size insert(bp)]/ size vector(bp)
Ligation approach:
-> overnight at room temperature
Preparation:
Inoculate pZ8-1-in 200ml LB (+Kan) for Midi-Isolation (tomorrow) -> 37°C overnight
05/25/2018 (SB, SS)
Continuing cloning in E. coli (DH5α)
Ligation approaches (from yesterday) added to bacteria. Remaining steps carried out according to kit. Transformed constructs (Hxk1/cInv/Strav each in pZ8-1)
Mini-Prep. for the pZ8-1 Isolation
oD600 measurement (oD4.3) and separated 200ml in 4 falcons (each 40-45ml). Remaining steps carried out according to kit. Pellet was resolved in 50 ml TE-Buffer (4 °C over the weekend; concentration: 457ng/μl)
Week 7 (May 28 - June 03)
05/28/2018 (MAS, NB)
Buffer for SDS (Renatas alternative SDS Gels)
SDS-PAGE-BUFFER: pH 8.9:
• TRIS-HCL: pH 8.8
• TRIS-HCL: pH 6.8
• TRIS-BOR: pH 8.8
PCR Hxk1/cInv/Strav:
See PCR 14.05.18
Preparation
Inoculate 1 clone from Hxk-pZ8-1 construct in 4ml LB (+4μl Kan) and inoculate 1 clone from Strav-pZ8-1 construct in 4ml LB (+4μl Kan)
05/29/2018 (MAS, RV)
Miniprep:
Isolation according to Nucleo Spin Plasmid kit
Changed PCR (gradient PCR):
05/30/2018 (MAS, NB)
Double digestion of pZ8-1 and pZ8-1 (+Hxk1/cInv/Strav) + gele:
-> according to NEB 5-10min 37°C
06/01/2018 (ST, FE)
SDS-page PS2:
SDS gels: new method
PS2 isolated on 11.05.2018
Week 8 (June 04 - June 10)
06/04/2018 (DR, RV)
PCR HxK1 and Inv + gel extraction
See 05/14/2018: master mix: 4x
Cut out fragments and extracted DNA from the gel (instructions according to manufacturer). HxK:
0.1g solved in 200 μl NTI and cInv: 0.13g in 260 μl NTI.
Preparation:
Inoculate fluid culture for mini-prep: 3 cultures from Strav, HxK1, cInv clones (each 4 μl Kanamycin in 4ml LB) and inoculate 250ml C. glutamicum culture.
06/05/2018 (DR, RV)
PCR from the extracted inserts:
See 05/14/2018: master mix 13x
06/06/2018 (RV, DR)
06/06/2018 (RV, DR) PS2 isolation (improved method!)
The measured OD was OD(600)=7. An OD1 of 1 was adjusted (18x10ml cultures in 15ml falcons). Remaining steps: see protocol 05/02/2018 (resuspend pellet in 500 μl Tris pH 6.8 without SDS)
PCR: Hxk1 and cInv
Concentration of Inv (9.4 ng/μl) and HxK (3.13 ng/μl) were measured, but the wrong fragments were cut out. The DNA was discarded.
06/07/2018 (SB, SS)
Amplification of cut out DNA from 06.06.18 and original IDT sequence to check degradation. Reduced Annealing temperature from 10s to 5s and increased Elongation time from 1min to 1.5min.
06/08/2018 (SB, SS)
Cut out correct Hxk1 fragment and isolated it via gel-extraction “Nucleo Spin”. Isolation according to manufacturer information and the pellet was resuspended in 30μl NE. The concentrations of Hxk1 cut out fragment (18.2ng/μl) Hxk1 of the original sequence (224.5ng/μl) were measured. Later we noticed that the cut-out fragment had the wrong size, so we disposed it.
EcoRI digestion of Hxk1
The DNA was digested for 15 min at 37°C and purified via PCR clean up afterwards (according to kit protocol). Concentration after clean-up was 14ng/μl.
Week 9 (June 11 - June 17)
06/11/2018 (MS, MI)
06/12/2018 (MS, MI)
SbsB: concentration and test gel:
After dialyzing our samples, the chemist brought them back and we measured the concentration. For sample one a concentration of 0.7mg/ml and for sample two 0.92mg/ml (0.3) was measured.
06/13/2018 (MS, SS)
Preparation for SbsB extraction
Overnight culture for first purification: inoculate 100ml LB + Amp (100 μl) + E. coli (containing SbsB plasmid) overnight (37°C). Preparation of binding/washing buffer (buffer A: 50mM NaPO4, 500mM NaCl, 50mM Imidazol), elution buffer (buffer B: 50mM NaPO4, 500mM NaCl, 500mM Imidazol) and 1M Tris-HCl 6.8 pH.
06/14/2018 (SS, SB)
Continuing SbsB extraction (from E. coli):
Overexpression for first purification: 1L LB + 20ml overnight culture + 1ml Amp. Let it grow for 1.5h (oD 1.35) and induce with 1ml 1MIPTG for 4h. Centrifuge 10min/ 3000g/ 4°C and store pellet overnight at -20°C.
Test-PCR of IDT-Sequences (Hxk1, Inv, Strav.) with taq-Polymerase
Changes PCR program (increased cycle number: 12 -> 30 and increased Annealing time: 5 -> 10s, Elongation: 90min)
06/15/2018 (FE)
P1 (SbsB) Extraktion aus E.coli :
Lysis for purification:
resuspend the bacteria pellet in 8ml binding buffer (Puffer A with protease inhibiter (1 pill). Add 30ml of the Lysozyme solution. Sonicate 30x 9 circles. 5 times with 5min pause in between always keep on ice, with big sonde, 50% power. Centrifuge 30min at 8500rpm 4°C 1h 30min, add 30 ml DNAase I and incubate for 30 min.
Protein purification with Ni – NTA:
prepare columns: 4ml Ni-NTA per column (2 columns) in 4°C Room and discard supernatant from the columns. Wash Ni-NTA slurry in columns three times with binding buffer (Puffer A). Apply lysate supernatant to columns and incubate at 4°C for around 2h/over night
Week 10 (June 18 - June 24)
06/18/2018 (DR, MI)
SbsB Isolation – continued:
Equilibrate column with buffer A. Fill column with buffer a and let it flow through, (flowthrough was stored for analyzation). Protein elutes with elution buffer: gradient: 25%, 50%, 75%, 9x 100% (9 fractions per column)
06/20/2018 (SB)
Preparation:
10 clones (Nr. 2-11) of SbsB WT (CAb-Lys3-pHEN6 Vektor; AmpR) inoculated in LB (4ml LB + 4µl Amp); 37°C overnight.
06/21/2018 (DR, MI)
Plasmid (SbsB WT, AmpR): mini-prep (according to kit protocol) and DNA was solved in 50µl mQ.
Week 11 (June 25 - July 01)
06/26/2018 (TM, MI)
10XTBE: 0.372g 10mM EDTA-Na2+ 1.22g 100mM Tris and adjust pH
06/27/2018 (TM, MI)
PS2-isolation from C. glutamicum: OD=3.03
OD measured: OD600=3.03 and separated overnight culture in 6 falcons with 30ml culture (adjust OD=1). Remaining steps: see protocol 06/27/2018.
PS2 isolation: OD=6
Isolation was performed with an overnight culture oD600= 6.34. Separated overnight culture in 4 falcons with 30 ml culture ´(undiluted) and resuspend pellet in 3ml 50mM Tris pH 6.8. Remaining steps see protocol 06/27/2018. PS2 concentrations: 2.25 mg/ml (ratio 1.4); 2.21 mg/ml (ratio 1.4)
Preparation of chromosomal DNA (gDNA) – Corynebacteria
• Prepare an overnight in 20-30 ml BHI at 30°C
• 4500 g at 15mi
• Discard supernatant
• Resuspend pellet in 3 ml TE with pH 8 (1.5ml falcon)
• Add a spatula tip-full of lysozyme + 30 μl RnaseA (10μg/μl)
• Incubate at 37°C, 125 rpm shaker, 2h
• Add 210 µl 20% SDS (respectively 420 μl 10% SDS) + 30 µl proteinkinase K (20 μg/ml) (or 60μµl proteinkinase (10μg/ml))
• Invert sample
• Incubate for 2h in dryer at 65 °C
• Add 600 μl 5 M NaCl
• 30 min in dryer at 64°C
Next steps must be developed in a fume hood
• 4.5 ml chloroform: phenol: isoamylalkohol (25: 24: 1)
• Centrifuge for 15min, 8900g, 4°C
• Upper phase into a new falcon with 2ml isopropanol
• Invert many times until you see a cloud
• Fish the DNA – into prepared eppis with 500 µl 70% EtOH
• Centrifuge 2x 10 min 500 μl EtOH (70%)
• Dry in speedvac • Resuspend in 100 µl TE pH 8 (overnight, RT)
Concentration:
1. gDNA C. glut 879 ng/μl
2. gDNA C. glut 798 ng/μl
06/28/2018 (RV SS, SB)
Dissolve DNA from IDT (in 1x TE pH8)
500 ng Streptavidin (=Strav) powder was delivered and dissolved in 50 µl TE (1x pH8) (end concentration: 10 ng/μl). 2x20µl and 1x10µl aliquot Strav DNA.
Week 12 (July 02 - July 08)
07/02/2018 (TM)
PCRs
Primer P2 fw. = EcoRI PS2 sense
Primer P2 rev. = EcoRI PS2 antisense
PCR-program:
PS2 1-150 run out of the gel
07/03/2018 (MOS, MI, SB)
SDS-Page of PS2 from 06/27/2018
TM isolated PS2 (see protocol 06/27/2018)
DR and SB isolated (OD=6, see protocol 06/27/2018)
Preparation:20ml overnight culture (BHI medium) inoculated with C. glutamicum
07/04/2018 (MOS, SS)
Agarose gel of pZ8-1 and PS2 1-150
07/06/2018 (TM)
Assembly of the PS2-CFP-Fragments
Measurement of the concentration of the PCR products
Assembly protocol
Set up the following reaction on ice (see HiFi DNA Assembly Cloning Kit): 0.2-0.5 pmols of fragments
Incubate in Thermocycler at 50°C for 60 minutes.
Week 13 (July 09 - July 15)
07/09/2018 (TM)
pZ8-1 digestion with EcoRI and dephosphorylation:
UEnzyme = mplasmid [μg] x LRef [bp]/ Lplasmid [bp] x nplasmid/ nRef
L= length in bp and U= units
15min 37°C -> 5 minutes at 100°C
30 minutes at 37°C
PCR of the 3 Assembly constructs:
PCR program:
Conclusion: new PCR with less DNA (1µl) and positive (PS2) and negative (MQ) control
Repeated PCR gel. Used new TAE buffer and autoclaved agarose:
7/11/2018 (MOS)
Retransformation of pET28a(+) in E. coli DH5α
Transformed 2 μl pET28a (+) vector (KanR). Remaining steps: see protocol 05/09/2018
7/12/2018 (SS, TM)
EcoRI digestion of the fragments and pZ8-1
Added following material to each fragment: EcoRI (1μl), DNA (1μg), 10x CutSmart buffer (5μl) and fill up to 50μl with VE-H2O
15min at 37°C
Dephosphorylation: according to protocol from 07/09/2018.
Assembly of the P2-CFP- fragments in the pZ8-1 vector
Assembly protocol:
I. Set up the reaction: 2. construct
Positive control: 2.5 μl + 2.5 μl HiFi DNA Assembly Mix
Nanodrop measurement of the digested and dephosphorylated fragments:
II. Incubate samples on ice on thermocycler 50°C for 60 minutes
III. Transform competent cells with 2 μl of the assembly reaction
Chemical competent cell transformation:
Thaw competent cells on ice and add 2 μl of the chilled assembled product to the competent cells. Mix gently by pipetting up and down or by flicking the tube 4 – 5 times. Do not vortex. Place the mixture on ice for 30 minutes. Do not mix. Heat shock at 42°C for 30 seconds. Do not mix. Transfer tubes to ice for 2 minutes. Add 950µl of room temperature SOC-media to the tube. Incubate the tube at 37°C for 60 minutes 250rpm. Warm selection plates to 37°C. Centrifuge and spread 100µl of the cells onto the selection plates. Use LB plates with Ampicillin for the positive control sample. Incubate Overnight at 37°C
Midi-Preparation:
Inoculated 200 ml LB liquid media + 200 μl Kanamycin with vector pET28a (+) (37°C overnight)
07/13/2018 (TM, SS)
Interlab
Dissolved plasmids in 50 μl TE-buffer (plate 7)
Transformation in E. coli (sample 1-6 + positive and negative control):
Chemical competent cell transformation
1. Throw competent cells on ice.
2. Add 2 μl of the plasmid to the competent cells
3. Place the mixture on ice for 15 minutes. Do not mix.
4. Heat shock at 42°C for 30 seconds. Do not mix.
5. Transfer tubes to ice for 2 minutes.
6. Add 950µl of room temperature SOC-media to the tube.
7. Incubate the tube at 37°C for 60 minutes 250 rpm.
8. Warm selection plates to 37°C
9. Centrifuge and spread 100µl of the cells onto the selection plates with Chloramphenicol (Cm).
10. Incubate Overnight at 37°C
Week 14 (July 16 - July 22)
07/17/2018 (TM, AM; MI)
Midi preparation of 2. Assembly construct:
pET a 28+ (=low copy plasmid). V[ml] = 800/OD600 = 800/2.3 = 347.82 (150 ml). Remaining steps according to midi protocol (Nucleospin®Plasmid Kit). DNA dissolved in 50μl 1x TE buffer
Control digest of the 2. Assembly construct
Incubate 1h at 37°C
07/18/2018 (MOS, AM)
Prepare pET28a+ to sequence: Nanodrop measurement: 500.6 ng/μl; needed concentration: 20µl 100ng/μl (in TE buffer)
Interlab:
Inoculate 5 ml LB media with 25 ng/ml Chloramphenicol and pick 2 colonies per plate. Incubation at 37°C for 16- 18 hours.
07/19/2018 (TM, MI, AM)
Interlab
Calibration Protocols
CALIBRATION PROTOCOLS SHOULD BE COMPLETED BEFORE CELL MEASUREMENTS ARE TAKEN!
You will make three sets of unit calibration measurements: an OD600 reference point, a particle standard curve, and a fluorescein standard curve. Before beginning these protocols, please ensure that you are familiar with the measurement modes and settings of your instrument. For all of these calibration measurements, you must use the same plates and volumes that you will use in your cell-based assays. You must also use the same settings (e.g., filters or excitation and emission wavelengths) that you will use in your cell-based assays. If you do not use the same plates, volumes, and settings, the calibration will not be valid. Make sure to record all information about your instrument (checklist on page 1 of this protocol) as these will be required later when you document your experiment. If your instrument has variable temperature settings, the instrument temperature should be set to room temperature (approximately 20-25 C°) for all measurements.
> we use 620nm filter, because we don´t have a filter with 600nm
> T= 22.5°C
Calibration 1: OD 600 Reference point - LUDOX Protocol
Materials:
1ml LUDOX CL-X (provided in kit)
ddH20
96 well plate, with clear flat bottom preferred
Method:
❏ Add 100 μl LUDOX into wells A1, B1, C1, D1
❏ Add 100 μl of dd H2O into wells A2, B2, C2, D2
❏ Measure absorbance at 600 nm of all samples in the measurement mode you plan to use for cell measurements
❏ Record the data in the table below or in your notebook
❏ Import data into Excel sheet provided (OD600 reference point tab)
Calibration 2: Particle Standard Curve - Microsphere Protocol
Materials:
300μL Silica beads - Microsphere suspension (provided in kit, 4.7 x 10^8 microspheres) ddH20
96 well plate, black with clear flat bottom preferred
Method:
Prepare the Microsphere Stock Solution:
❏ Obtain the tube labeled “Silica Beads” from the InterLab test kit and vortex vigorously for 30 seconds. NOTE: Microspheres should NOT be stored at 0°C or below, as freezing affects the properties of the microspheres. If you believe your microspheres may have been frozen, please contact the iGEM Measurement Committee for a replacement (measurement at igem dot org).
❏ Immediately pipet 96 μL microspheres into a 1.5 mL eppendorf tube
❏ Add 904 μL of ddH2O to the microspheres
❏ Vortex well. This is your Microsphere Stock Solution.
Prepare the serial dilution of Microspheres:
Accurate pipetting is essential. Serial dilutions will be performed across columns 1-11. COLUMN 12
MUST CONTAIN ddH2O ONLY. Initially you will setup the plate with the microsphere stock solution in
column 1 and an equal volume of 1x ddH2O in columns 2 to 12. You will perform a serial dilution by
consecutively transferring 100 μl from column to column with good mixing.
❏ Add 100 μl of ddH 2O into wells A2, B2, C2, D2....A12, B12, C12, D12
❏ Vortex the tube containing the stock solution of microspheres vigorously for 10 seconds
❏ Immediately add 200 μl of microspheres stock solution into A1
❏ Transfer 100 μl of microsphere stock solution from A1 into A2.
❏ Mix A2 by pipetting up and down 3x and transfer 100 μl into A3 …
❏ Mix A3 by pipetting up and down 3x and transfer 100 μl into A4...
❏ Mix A4 by pipetting up and down 3x and transfer 100 μl into A5...
❏ Mix A5 by pipetting up and down 3x and transfer 100 μl into A6...
❏ Mix A6 by pipetting up and down 3x and transfer 100 μl into A7...
❏ Mix A7 by pipetting up and down 3x and transfer 100 μl into A8...
❏ Mix A8 by pipetting up and down 3x and transfer 100 μl into A9...
❏ Mix A9 by pipetting up and down 3x and transfer 100 μl into A10...
❏ Mix A10 by pipetting up and down 3x and transfer 100 μl into A11...
❏ Mix A11 by pipetting up and down 3x and transfer 100 μl into liquid waste
TAKE CARE NOT TO CONTINUE SERIAL DILUTION INTO COLUMN 12.
❏ Repeat dilution series for rows B, C, D
❏ IMPORTANT! Re-Mix (Pipette up and down) each row of your plate immediately before putting in the plate reader! (This is important because the beads begin to settle to the bottom of the wells within about 10 minutes, which will affect the measurements.) Take care to mix gently and avoid creating bubbles on the surface of the liquid.
❏ Measure Abs600 of all samples in instrument
❏ Record the data in your notebook
❏ Import data into Excel sheet provided (particle standard curve tab)
Calibration 3:  Fluorescence standard curve - Fluorescein Protocol
Plate readers report fluorescence values in arbitrary units that vary widely from instrument to instrument. Therefore, absolute fluorescence values cannot be directly compared from one instrument to another. In order to compare fluorescence output of test devices between teams, it is necessary for each team to create a standard fluorescence curve. Although distribution of a known concentration of GFP protein would be an ideal way to standardize the amount of GFP fluorescence in our E. coli cells, the stability of the protein and the high cost of its purification are problematic. We therefore use the small molecule fluorescein, which has similar excitation and emission properties to GFP, but is cost-effective and easy to prepare. (The version of GFP used in the devices, GFP mut3b, has an excitation maximum at 501 nm and an emission maximum at 511 nm; fluorescein has an excitation maximum at 494 nm and an emission maximum at 525nm).  You will prepare a dilution series of fluorescein in four replicates and measure the fluorescence in a 96 well plate in your plate reader. By measuring these in your plate reader, you will generate a standard curve of fluorescence for fluorescein concentration. You will be able to use this to convert your cell based readings to an equivalent fluorescein concentration. Before beginning this protocol, ensure that you are familiar with the GFP settings and measurement modes of your instrument. You will need to know what filters your instrument has for measuring GFP, including information about the bandpass width (530 nm / 30 nm bandpass, 25-30nm width is recommended), excitation (485 nm is recommended) and emission (520-530 nm is recommended) of this filter.
> GFP-filter: 485nm & 535nm
Materials:
Fluorescein (provided in kit)
10ml 1xPBS pH 7.4-7.6 (phosphate buffered saline;)
96 well plate, black with clear flat bottom
Method
Prepare the fluorescein stock solution:
❏ Spin down fluorescein kit tube to make sure pellet is at the bottom of tube.
❏ Prepare 10x fluorescein stock solution (100 μM) by resuspending fluorescein in 1 mL of 1xPBS. [Note: it is important that the fluorescein is properly dissolved. To check this, after the resuspension you should pipette up and down and examine the solution in the pipette tip – if any particulates are visible in the pipette tip continue to mix the solution until they disappear.]
❏ Dilute the 10x fluorescein stock solution with 1xPBS to make a 1x fluorescein solution with concentration 10 μM: 100 μL of 10x fluorescein stock into 900 μL 1x PBS
Prepare the serial dilutions of fluorescein:
Accurate pipetting is essential. Serial dilutions will be performed across columns 1-11. COLUMN 12 MUST CONTAIN PBS BUFFER ONLY. Initially you will setup the plate with the fluorescein stock in column 1 and an equal volume of 1xPBS in columns 2 to 12. You will perform a serial dilution by consecutively transferring 100 μl from column to column with good mixing.
❏ Add 100μl of PBS into wells A2, B2, C2, D2....A12, B12, C12, D12
❏ Add 200μμl of fluorescein 1x stock solution into A1, B1, C1, D1
❏ Transfer 100μl of fluorescein stock solution from A1 into A2.
❏ Mix A2 by pipetting up and down 3x and transfer 100μl into A3 …
❏ Mix A3 by pipetting up and down 3x and transfer 100μl into A4...
❏ Mix A4 by pipetting up and down 3x and transfer 100μl into A5...
❏ Mix A5 by pipetting up and down 3x and transfer 100μl into A6...
❏ Mix A6 by pipetting up and down 3x and transfer 100μl into A7...
❏ Mix A7 by pipetting up and down 3x and transfer 100μl into A8...
❏ Mix A8 by pipetting up and down 3x and transfer 100μl into A9...
❏ Mix A9 by pipetting up and down 3x and transfer 100μl into A10...
❏ Mix A10 by pipetting up and down 3x and transfer 100μl into A11...
❏ Mix A11 by pipetting up and down 3x and transfer 100μl into liquid waste TAKE CARE NOT TO CONTINUE SERIAL DILUTION INTO COLUMN 12.
❏ Repeat dilution series for rows B, C, D
❏ Measure fluorescence of all samples in instrument
❏ Record the data in your notebook
❏ Import data into Excel sheet provided (fluorescein standard curve tab)
Cell measurement protocol
Prior to performing the cell measurements, you should perform all three of the calibration measurements. Please do not proceed unless you have completed the three calibration protocols.
Completion of the calibrations will ensure that you understand the measurement process and that you can take the cell measurements under the same conditions. For the sake of consistency and reproducibility, we are requiring all teams to use E. coli K-12 DH5-alpha. If you do not have access to this strain, you can request streaks of the transformed devices from another team near you, and this can count as a collaboration as long as it is appropriately documented on both teams' wikis. If you are absolutely unable to obtain the DH5-alpha strain, you may still participate in the InterLab study by contacting the Measurement Committee (measurement at igem dot org) to discuss your situation.  For all of these cell measurements, you must use the same plates and volumes that you used in your calibration protocol. You must also use the same settings (e.g., filters or excitation and emission wavelengths) that you used in your calibration measurements. If you do not use the same plates, volumes, and settings, the measurements will not be valid.
Materials:
Competent cells (Escherichia coli strain DH5α)
LB (Luria Bertani) media
Chloramphenicol (stock concentration 25 mg/mL dissolved in EtOH)
50 ml Falcon tube (or equivalent, preferably amber or covered in foil to block light)
Incubator at 37°C
1.5 ml eppendorf tubes for sample storage
Ice bucket with ice
Micropipettes and tips
96 well plate, black with clear flat bottom preferred (provided by team)
Method:
Cell growth, sampling, and assay
◻ Make a 1:10 dilution of each overnight culture in LB+Chloramphenicol (0.5mL of culture into 4.5mL of LB+Chlor)
◻ Measure Abs600of these 1:10 diluted cultures
◻ Record the data in your notebook
◻ Dilute the cultures further to a target Abs600 of 0.02 in a final volume of 12 ml LB medium + Chloramphenicol in 50 mL falcon tube (amber or covered with foil to block light).
◻ Take 500µL samples of the diluted cultures at 0 hours into 1.5 ml eppendorf tubes, prior to incubation. (At each time point 0 hours and 6 hours, you will take a sample from each of the 8 devices, two colonies per device, for a total of 16 eppendorf tubes with 500µL samples per time point, 32 samples total). Place the samples on ice.
◻ Incubate the remainder of the cultures at 37°C and 220 rpm for 6 hours.
◻ Take 500μL samples of the cultures at 6 hours of incubation into 1.5 ml eppendorf tubes. Place samples on ice.
◻ At the end of sampling point you need to measure your samples (Abs600 and fluorescence measurement), see the below for details.
◻ Record data in your notebook
◻ Import data into Excel sheet provided (fluorescence measurement tab)
Measurement:
Samples should be laid out according to the plate diagram below. Pipette 100µl of each sample into each well. From 500µl samples in a 1.5 ml eppendorf tube, 4 replicate samples of colony #1 should be pipetted into wells in rows A, B, C and D. Replicate samples of colony #2 should be pipetted into wells in rows E, F, G and H.  Be sure to include 8 control wells containing 100μl each of only LB+ Chloramphenicol on each plate in column 9, as shown in the diagram below. Set the instrument settings as those that gave the best results in your calibration curves (no measurements off scale). If necessary, you can test more than one of the previously calibrated settings to get the best data (no measurements off scale). Instrument temperature should be set to room temperature (approximately 20-25 C) if your instrument has variable temperature settings.
Help Debugging:
● If you have measurements that are off scale (“OVERFLOW”), that data will not be usable. You need to adjust your settings so that the data will be in range and re-run your calibration.
● If your Abs600 measurements for your cell colonies are very close to that of your LB+Chlor, then your cells have probably not been transformed correctly or grown correctly.
● If your negative and positive control values are very close to each other, that probably means something has gone wrong in your protocol or measurement.
Layout for Abs600 and Fluorescence measurement:
At the end of the experiment, you should have two plates to read. Each plate should be set up as shown below. You will have one plate for each time point: 0 and 6 hours. On each plate you will read both fluorescence and absorbance.
Week 15 (July 23 - July 29)
07/23/2018 (DR, SB)
For interlab: Colony Forming Units per 0.1 oD600
Inoculated each 2 clones of negative and positive control (in 4ml LB + 4μl Chloramp.)
07/24/2018 (DR, SB)
PCR – Assembly (from 12.7.18):
4x Q5 Mastermix for PS2:
In each PCR tube 19μl + 1μl Template (positive control: gDNA, negative control: mQ and sample: Assembly (PS2 1-300-CFP))
-> no fragments visible  EcoR1 – Nde1 digestion from PET28a(+) + Strav:
• 1h at 37°C
• 10 min at 100°C
Continuing: Interlab
Used overnight culture of N.K (Negative Control) and P.K. (Positive Control) “old”to adjust an OD of 0.1
Step 2: Dilution Series Instructions
Do the following serial dilutions for your triplicate Starting Samples you prepared in Step 1.
You should have 12 total Startingv Samples - 6 for your Positive Controls and 6 for your Negative Controls.
For each Starting Sample (total for all 12 showed in italics in parenthesis):
• You will need 3 LB Agar + Cam plates (36 total).
• Prepare three 2.0mL tubes (36 total) with 1900μL of LB + Cam media for Dilutions 1, 2, and 3 (see figure below).
• Prepare two 1.5mL tubes (24 total) with 900μL of LB + Cam media for Dilutions 4 and 5 (see figure below)
• Label each tube according to the figure below (Dilution 1, etc.) for each Starting Sample.
• Pipet 100µL of Starting Culture into Dilution 1. Discard tip. Do NOT pipette up and down. Vortex tube for 5-10 secs.
• Repeat Step 5 for each dilution through to Dilution 5 as shown below.
• Aseptically spread plate 100μL on LB + Cam plates for Dilutions 3, 4, and 5.
• Incubate at 37°C overnight and count colonies after 18-20 hours of growth.
07/25/2018 (DR)
Repeating Assembly PCR:
Clean up of PET28a(+) + Strav after digestion
According to kit protocol:
Ligation of pET28a(+) and Strav
1 μl Strav = 34.3 ng (-> 6.7 μl vector: pET28a(+))
1h at RT incubated
Transformation of pET28a(+) : Strav in E.coli Dh5α
• Competent cells from -80°C (blue/red stripe)
• 1h cells on ice
• Add whole ligation to cells
• Incubate 15 min on ice
• Heatshock: 30 sec at 42 °C
• Back on ice
• Add 950µl SOC-Medium
• 1h at 37°C; 300 rpm on shaken
• Plating out and in 37°C overnight
• (a positive control with pET28a(+) was also performed)
Interlab:
All 36 plates plated:
• 4 clones
• 3 samples each
• 3 dilution each sample
07/26/2018 (MoS, FE)
Interlab:
Plates counting (Excel sheet)
Recalculate the dilution
Example: clon1.1:154x10x8x10-4
Inoculation of pET28a(+):Strav: Each 15 colonies in 4ml LB + 4 μl Kan (50mg/ml)
07/27/2018 (TM, FE)
Mini-Preparation of pET28a(+):Strav
with Nucleospin Plasmid Kit.
Digestion:
1h 37°C, 10 min 100°C
Conclusion: no fragments -> no plasmid
Week 16 (July 3 - August 05)
07/30/2018 (MOS; AM)
Ligation pET28a(+)::Strav
Negative control
1h RT
Transformation in E. coli DH5α
• Thaw competent cells on ice
• Add ligation
• Incubate on ice 15min
• 45s at 45°C
• Back on ice
• Add 750 μl SOC-Medium
• Incubate 1h at 37°C, 300rpm
• Plate on agar plate containing Ampicillin
• Incubate at 37°C overnight
Isolation of SbsB-Preculture
• 100ml (+Ampicillin to 100 µg/ml) of LB in 300ml shaking flask
• Inoculate with E. coli (SbsB WT2 AmpR)
• Shake overnight at 37°C, 180 rpm
07/31/2018 (MOS; AM)
Isolation of SbsB-main culture and IPTG induction
• Preculture grew to an OD600 of 5.47
• 500ml main culture with OD600 = 0.1 (added 9.1ml preculture to 500ml LB)
• Induction with 1mM IPTG after 2h (OD600=0.56). Continue incubation at 23°C, 150rpm for 4h
• Centrifuge at 4°C, 3000g, 10min
• Freeze pellets at -20°C
Continuing transformation pET28a(+)::Strav
No colonies on control. 12 colonies were picked for plasmid preparation (4ml LB with 100µg/ml Kanamycin). Incubation overnight at 37°C, 180 rpm
08/01/2018 (MOS; AM)
Plasmid preparation pET28a(+)::Strav via Nucleospin PlasmidKit (low copy plasmid). 1.5ml of culture was used. DNA was solved in 20 μl TE-buffer
Conclusion: #2 and #9 are probably positive. #1, #10, #11, #12 are probably negative.
Isolation of SbsB - Lysis of cells
Work on ice. Resuspend frozen pellet with buffer A. Add one pill of protease inhibitor and 30µl lysozyme solution. Sonification: pulse: 30s, 90%; power: 50%. 5 cycles with a cooldown time of 5min, samples were cooled with ice water. Centrifuge at 4°C, 8500rpm, 30 min. Add 30µl DNAseI, incubate for 30min.
Purification of SbsB – Ni NTA AC
Apply 8ml of Ni NTA slurry to column. Ethanol will flow through the column and the medium will settle. Wash three times with buffer A, close column and add cell lysate. Incubate at 4°C overnight.
Preparation: Inoculation of C. glutamicum in 250ml BHI medium (incubation at 30°C) Inoculation of C. crescentus in 250ml PYE medium (incubation at 30°C)
08/02/2018 (MOS; AM)
Isolation of C. glutamicum:
100ml overnight culture (OD600=1.5). Changed protocol 06/27/2018: resuspend pellet of 100ml overnight culture in 50ml50mM Tris pH6.8. Conclusion: it is better to resuspend in less volume (higher concentration).
Purification of SbsB – Elution:
Prepare 10ml of 25%, 50% and 75% elution buffer. Let the lysate flow through the column and wash with 3x10ml buffer A. Four fractions are produced: flowthrough, wash 1, wash 2, and wash 3. First elution step (25% elutionbuffer): Apply 1ml buffer to the column and collect the fraction of 1ml. Repeat 6 times. Elution steps for 50%, 75%, and 100% are carried out analogously. Lastly 10ml 10% elutionbuffer were added.
08/03/2018 (MOS; AM)
Isolation of C. crecentus:
According to protocol 04/02/2018
Preparations: Inoculated clone #2-9 (pET28a(+)::Strav) in 4ml LB (+Kan).
Week 17 (August 06 - August 12)
Mini Prep of pET28a(+)::Strav clones.
08/07/2018 (DR, SB)
Sequencing and sequence alignment of clone #4, #6
• #4 and #9 are identified as positive clones, #6 is negative due to having no insert • #4 and #9 are plated on agar plates containing Kanamycin
pET28a(+)::Strav
Digestion: 1h 37°C and PCR clean-up
08/08/2018 (DR, SB)
Ligation of Sbsb to pET28a (+)::Strav
Incubation 1h at RT.
Transformation: pET a (+)::Strav + SbsB in E.coli:
According to protocol from 07/30/2018.
08/09/2018 (DR, SB)
Preparation: Inoculate all clones from transformation (4 clones from plate #9.2 (pET28a::Strav) +SbsB and two clones from #4 (pET28a::Strav) +SbsB.
08/10/2018 (DR, SB)
Mini prep of pET28 a (+):: Strav + SbsB: according to kit protocol
Test gel: as a pET28a::Strav was used (without SbsB) negative control. Clone #9.2 and #9.4 seem to be positive (contain SbsB). Send them to sequence.
Week 18 (August 13 - August 19)
08/13/2018 (TM,MaS)
Inoculation
• C. glutamicum in 250ml BHI -> 30°C ON
• C. crescentus in 250ml CYE -> 30°C ON
Interlab
• Inoculation of colonies: 4 N.C., 2 P.C.
-> each inoculated with 5ml LB and 5µl Chloramphenicol
-> 16-18h, 37°C
08/14/2018 (TM, MaS)
Isolation of PS2 from C. glutamicum
Accoring to protocol from C. crescentus isolation from 04/04/2018
Interlab
• According to measurement protocol from 07/19/2018
• Adjusting to oD=0.02
• Measurement: for each colony 4 replicates
o Absorbance at 620nm
o Fluorescence with GFP filter
• Measurement scheme:
PS2 concentration
• Measurement Nanodrop:
C. glutamicum and C. crescentus on new plates
Prepare new HEPES and BHI medium
Inoculate pET28a (+) 9#2 and 9#4 for Mini prep
08/15/2018 (TM, MaS)
PS2 isolation from C. glutamicum
According to protocol from 05/03/2018
RsaA isolation from C. crescentus
According to protocol from 04/04/2018
Mini prep of pET28a (+) 9#2 and 9#4
According to NucleoSpin plasmid Kit
Concentration measurement:
Preparation of new SDS gels
GELBILDER RsaA + Miniprep
08/16/2018 (TM, MaS)
Clones 9#2 and 9#4 send to sequencing
PS2 isolation from C. glutamicum
According to protocol from C. crescentus isolation from 04/04/2018
Concentration measurement:
PS2 isolation from C. glutamicum
According to protocol from 05/03/2018
Concentration measurement:
-> 15μl sample with 5 μl 4xPAP
-> 100°C, 5min
-> Samples on SDS gels
08/17/2018 (TM, MaS)
Inoculation of C. crescentus in 250ml PYE
Week 19 (August 20 - August 26)
08/20/2018 (SB, MaS)
Preparation: 2l LB, 1l PYE,700ml 10mM HEPES pH7.2, 100ml 10mM HEPES pH 2.0
Inoculation of 9#2 and 9#4
SbsB isolation
• 2x100ml (+Ampicillin to 100 µg/ml) of LB in 300ml shaking flask
• Inoculate with E. coli (SbsB WT2 AmpR)
• Shake overnight at 37°C, 180 rpm
Preparation: 2 SDS gels
RsaA isolation from C. crescentus
According to protocol from 04/04/2018
• 3x 50ml falcon tubes
• Resuspended in 1ml, 1.5ml and 2ml 10mM HEPES pH 2
08/21/2018 (SB, MaS)
SbsB isolation
Isolation of SbsB-main culture and IPTG induction
• Main-cultures grew to an OD600 of 0.9/ 1.0
• Induction with 1.5ml 1mM IPTG/ 1l LB+Amp+SbsB. Continue incubation at 23°C, 130rpm for 4h.
• Centrifuge at 4°C, 3000g, 10min
• Freeze pellets at -20°C
Midi prep of pET28a (+)::Strav::SbsB 9#2 and 9#4
• With NucleoBond© Xtra Midi
• According to protocol of NucleoBond© Xtra Midi, but step 5: 10-15min incubation instead of 5min
• Concentration measurement:
• 9#2 inoculated again in 200ml LB (+200 μl Kan)
• Sequencing of 9#4: 20 μl ˜ 86.3ng/ μl
RsaA isolation from C. crescentus
According to protocol from 04/04/2018
• 3x 50ml falcons
• 50ml resuspended in 1ml HEPES pH2 instead of 2ml
• Concentration measurement
• Test 1% agarose gel RsaA isolation 08/20/2018 and 08/21/2018
Preparation: 1l 10x running buffer
08/22/2018 (SB,MaS)
Midi prep of pET28a (+)::Strav::SbsB 9#2
• With NucleoBond© Xtra Midi
• According to protocol for low copy plasmid
• Dissolved in 25 μl mQ
• Concentration measurement:
• 20 μl with 87ng/ μl 9#2 Plasmid-DNA for sequencing
Preparation: 2l LB medium, 2l 70% EtOH
Inoculation of C. crescentus in 200ml PYE for RsaA isolation (48h, 28°C)
Continue SbsB isolation
Isolation of SbsB - Lysis of cells
• Work on ice. Resuspend frozen pellet with buffer A. Add one pill of protease inhibitor and 30 μl lysozyme solution. Sonification: pulse: 30s, 90%; power: 50%. 5 cycles with a cooldown time of 5min, samples were cooled with ice water. Centrifuge at 4°C, 8500rpm, 30 min. Add 30 μl DNAseI, incubate for 30min.
• Centrifuge and store pellet at 4°C
08/23/2018 (SB, MaS)
Continue SbsB isolation
Purification of SbsB – Ni NTA AC
Apply 4ml of Ni NTA slurry to column. Ethanol will flow through the column and the medium will settle. Wash three times with buffer A, close column and add cell lysate. Let cell lysate flow through.
Purification of SbsB – Elution
Prepare 10ml of 25%, 50% and 75% elution buffer. Let the lysate flow through the column and wash with 3x10ml buffer A. Four fractions are produced: flowthrough, wash 1, wash 2, and wash 3. First elution step (25% elutionbuffer): Apply 1ml buffer to the column and collect the fraction of 1ml. Repeat 2 times. Elution steps for 50%, 75%, and 100% are carried out analogously. Lastly 10ml EtOH were added.
Apply samples on SDS gel.
1% agarose test gel of midi prep from 9#2 and 9#4
08/24/2018
RsaA isolation from C. crescentus
According to protocol from 04/04/2018
• 4x 50ml tubes
• Applied on SDS gel
GELBILD
Digestion pET28a (+)::Strav::SbsB 9#2 and 9#4
pET28a (+) digestion:
Digestion of pET28a (+)::Strav::SbsB 9#2
Digestion of pET28a (+)::Strav::SbsB 9#4
-> 1.5h at 37°C
-> 15-20min at 70°C
-> test gel 1% agarose
Preparation: 15ml 1M IPTG
Week 20 (August 27 - September 02)
08/27/2018 (DR)
RsaA isolation:
See protocol 04/20/2018. Measured OD of the overnight culture: OD=1.17 (2.5ml overnight culture + 2.5ml PYE medium)
pET28a::Strav and SbsB digestion:
PS2 isolation:
Overnight culture OD=2.5., adjusted OD=1 and used six times 30ml culture for the isolation. Remaining steps: see protocol 05/03/2018.
08/28/2018 (DR)
RsaA isolation:
Used again the overnight culture from yesterday (stored in the fridge since yesterday). Measured an OD of 0.99 and used 5ml directly from culture (12x). Remaining steps see protocol 04/20/2018. Changed: Incubation 20min instead of 10min at RT.
PS2 isolation:
Overnight culture OD=11.6., adjusted OD=1 and used 12x 30ml culture for the isolation. Remaining steps: see protocol 05/03/2018.
Agarose gel (of yesterday´s digestion):
08/29/2018
Mini-prep of pET28a(+):Strav (clone 4) 2x8ml of culture centrifuged in 15ml falcon (2min; 10000). Resuspend in 0.5ml A1 and add 0.5ml of A2. Incubate 5min at RT. Add 600 μl A3 and inverted the tube 6-8 times. Centrifuge for 10min at 11000rpm and transfer supernatant to column (750 μl). Centrifuge for 1min and add leftover cell lysate. Centrifuge for 1min at 11000rpm and add 600 μl A4. Centrifuge for 1min, discard flow-through and centrifuge for 2min. Add 50 μl pre-heated AE and incubate for 5min at 70°C. Centrifuge for 1min.
PCR of SbsB from SbsB:Phen6: 24 μl of MM per PCR tube + 1 μl template
24 μl of MM per PCR + 1 μl template. PCR program: same as 07/02/2018.
PS2 isolation:
Overnight culture OD=12; adjusted OD=1 and used 18x 30ml culture for the isolation. Remaining steps: see protocol 05/03/2018. Mixed supernatant of each three falcons’ ins one.
Digestion of pET28a(+):Strav
Test for Strav (digest with EcoRI + NdeI)
Agarosegel: PCR und digestion fehlt
-> PCR Clean-up: according to kit protocol.
Digestion for ligation of SbsB into pET28a(+)::Strav
• 2 μg DNA for higher DNA amount for ligation (-> 5 μl SbsB; 10 μl pET28a(+):Strav)
• 40µl of master mix +10µl DNA (5 μl vector + 5 μl insert)
• Incubate 37°C for 90min
• Inactivate for 10min at 75°C
08/30/2018
-> PCR Clean-up
Ligation:
• Inoculated 3h
• Transformation in E.coli (DH5α): see protocol 07/30/2018.
08/31/2018
Transformation result: many colonies on plate “without insert” and few on plate “without ligase” -> Colony PCR
Colony PCR:
• 5 colonies from pET28a(+):Strav+SbsB
• 5 colonies from “without insert” plate to check for wrongly added SbsB to ligation
• 1 SbsB positive control (from phen6:SbsB)
• 1 water control
Week 21 (September 03 - September 09)
Week 21 (September 03 - September 09)
Minipreparation of pET28a+: Strav:SbsB Clone 1,3,4,5
Use 8ml of culture for each clone. Remaining steps were performed according to kit protocol.
Measure the concentration on the nanodrop
Digestion of the pET28a+:Strav:SbsB constucts
Two different digestion (one with NdeI and another with XhoI)
Incubate 1h at 37°C and heat inactivation 10min at 75°C.
expected sizes
Isolation of PS2 from C. glutamicum
OD600: 5.4. Isolation with Tris pH 6.8 see 06/27/2018.
Measure the concentration on the nanodrop
1% Agarose gel picture
09/05/2018 (TM)
Isolation of PS2
OD600= 5,3
Isolation with 50mM HEPES pH 6.8 in 12 falcons with 30 ml culture OD600=1 (see 06/27/2018).
Measure the concentration on the nanodrop
Mini's from 09/04/2018 again inoculated with 8ml LB. and grow overnight at 37°C
09/06/2018 (MOS, RV, LS)
Mini prep # 1, # 3, # 4 and # 5:
Sent in sample #1 for sequencing.  RsaA isolation:
OD600=0.89. Remaining steps: see 04/20/2018
Mini prep # 1, # 3, # 4 and # 5
RsaA from 06.09. concentration measured with nanodrop
Inoculated # 1, # 3, # 4 and # 5 clones again in 8ml LB.
Week 22 (September 10 - September 16)
09/10/2018 (MOS, LS)
Transformation of pET28a:Strav:SbsB (# 1, # 3, # 4 and # 5 clones) in BL21:
Protocol: see 7/30/2018
SbsB isolation:
Inoculate SbsB (phen6 vector) in 100ml LB (+Amp) overnight at 37°C. See protocol 06/13/2018.
09/11/2018 (MOS, TM)
SbsB-isolation: main culture inoculated
• pre-culture: OD600=7.04
• main culture: 2x 500ml with 7.1ml for each from the main culture inoculated
• ampicillin concentration 100 μg/ml
• 2 hours at 37°C, 180 rpm > grow at OD600=0.97
• induce expression with 1mM IPTG: incubate for 4 hours at 150 rpm
Biological Parts for iGEM: Digestion of the backbone
Enzyme mastermix for plasmid backbone (for 5 reactions):
Add 4 μ linearized plasmid backbone pSB1C3 to 4 μl enzyme mastermix. Incubate 30 minutes at 37°C and 20 minutes at 80°C (for inactivation)
Digest of PS2-PCR-products 1, 2, 3 from 08/31/2018
Incubate 30 minutes at 37°C and 20 minutes at 80°C (for inactivation).
-> dilute: 25mg/μl
Ligation in pSB1C3
• add 2 μl of digested plasmid backbone (25ng)
• add equimolar amount of EcoRI-HF, PstI digested fragment (2 l)
• add 1 μl T4 DNA-ligase
• add water to 10 μl (> 5μl)
• incubate for 30 minutes at 16°C
• incubate for 20 minutes at 80°C
• ligation in pZ8-1 vector: see 09/12/2018 (tomorrow)
09/12/2018 (MOS, LS)
Diluted vector: 25ng/μl
4μl linearized plasmid + 4μl mastermix -> 3x in PCR tubes
Digestion for 30min at 37°C and heat inactivation 20min at 80°C.
Ligation of PS2 gene in pZ8-1
Ligation for 30min at 16°C and heat inactivation for 20min at 80°C.
Transformation pZ8-1:PS2 in DH5α:
See protocol 7/30/2018
09/13/2018 (MOS, LS)
RsaA isolation:
OD=1.34. Remaining steps: see 04/20/2018
Lysis for purification and protein purification with Ni – NTA:
See protocol: 06/15/2018. Incubation overnight.v  Inoculate pZ8-1:PS2 clone #1, #2 and #3
09/14/2018 (MOS, LS)
SbsB Isolation – continued:
See protocol 06/18/2018
Mini-prep pZ8-1:PS2 (in DH5α) clone #1, #2 and #3:
Eluted plasmid DNA in mQ.
Week 23 (September 17 - September 23)
09/17/2018 (ST, MoS)
Washing the PAGEs of Friday
Preparation: stocks and solutions for transformation of C. glutamicum
• 4% Isoniacide: filtered through 0,2μm; 20ml
• 10% Glycerol, 1L, autoclaved twice
• LB (1L)
• Stocks of chloramphenicol: 25mg/ml, 10mL in 70% ethanol
• 5ml Tween 80 filtered through 0,2μm
Inoculation of pSB1C3:PS2
Picking 3x DH5α-colonies with pSB1C3:PS2, into 4ml LB + 25mg/ml chloramphenicol (for miniprep)
09/18/2018 (ST, FE)
Miniprep of pSB1C3:PS2
According to the NucleoSpin plasmid kit protocol, elution in 50μl mQ
Concentrations measured via nanodrop
Production of 1 SDS-PAGE; reevaluation of p1 and p3 from last week
Preparation: 4l LB for Protein expression of Strav:SbSB in E.coli BL21(DE3)
Strav:SbsB isolation
4 overnight pre-cultures of BL21(DE3) 1-4 with pET28a (+):Strav:SbSB (150ml + 50μg/ml Kanamycin)
09/19/2018 (ST, FE)
Control digestion of pZ8-1:ps2 (14.9.18) and pSB1C3:PS2
• Mastermix x8
-> Dilute plasmid to 25ng/μL before digestion
• 4μl MM + 4μL plasmid in PCR-tubes, 30min at 37°C, 20min at 80°C
• 1% agarose gel 120V for 45 min
PCR with PS2 BioBrick
• 4x MM with non-precipitated Q5-Mix; +1x with precipitated MM (extra)
Strav:SbsB isolation
• Measurement of oD600 of the overnight pre-cultures; preparation of expression cultures (4x500ml LB; + 50μg/ml Kanamycin, +10mM MgCl2, to oD600=0,08)
• Induction of overexpression at oD600=0,8 with IPTG (1mM)
Preparation: 5ml IPTG 1M stock solution; sterile filtered
09/20/2018 (ST, FE)
PS2 BioBrick
• Agarose gel of PS2 BioBrick – 120V, 45min
• Concentration measurement of PS2 BioBrick amplificates
Digestion of pZ8-1, psB1C3, PS2#1, PS2#2 and PS2#4 (sample with precipitated Q5)
• 9x MM:
• 4μl MM + 4μl DNA (25ng/μl)
• 30min at 37°C, 20min at 80°C
• Remaining digest product in refridgerator for agarose gel on the next day
Strav:SbsB isolation
• oD600-Measurement of expression cultures
-> Flasks 1&3 (expression at RT): 6.6
-> Flasks 4&5 (expression at 20°C): 3.5
• Pelleting: 5000g, 4°C, 30min in the big centrifuge (MPP, in 500mL bottles)
• Harvesting of pellets into 50mL Falcons: ~11g
• Resuspension in 15ml Buffer A; addition of 1 tablet cOmplete each
• Sonification in 4°C-room with sonotrode on ice: 50% power, 25sx5cycles; 90s pauses to chill sample in between steps
• Pellet cell debris 1h at 8500rpm 4°C; filtered supernatant into 15mL tubes
Purification of Strav:SbSB sample #1:
• Loading of Ni-NTA-medium into column, let it sediment, let ethanol run through
• Wash column with Buffer A 3 times (10ml each); close column
• Add supernatant of sample (in this case: Strav:SbSB #1) onto column, incubate ON at 4°C
Preparation: 2l LB
Ligation of PS2#1,2 and #4 in pZ8-1 and pSB1C3
-> 30min at 16°C, 20min at 80°C
Prepared Samples:
Strav:SbsB isolation
Preparation: 25%, 50% and 75% Elution buffer (dilute in Buffer A)
• Let supernatant run through column, catch flow-through
• Wasch 3 times with Buffer A (10mL each), catch flow-through (W1/2/3)
• Elute with 7 times 1ml 25% EB -> 7 fractions 1ml
• Analog elution for 50% and 75% EB
• Elute once with 100% EB 10ml
• All steps are carried out at 4°C
Test digestion of pSB1C3:PS2
-> 2μl MM + 2μl sample; 30min at 37°C, 20min at 80°C
-> 1% agarose gel of digestion
Digestion of pSB1C3 & PS2#2 & PS2#4
-> 4μl Plasmid/PCR-product + 4μl MM
-> 30min at 37°C, 20min at 80°C
Week 24 (September 24 - September 30)
09/24/2018 (AM, FE)
Ligation of pSB1C3.1 with PS2 #2 & PS2 #4 & PSB1C.2 with PS2 #2 & PS2 #4
• 2µl digested plasmid + 2 µl digested fragment
• 1µl T4 DNA ligase
• Add mQ 10µl (-> 5µ mQ)
Attemps:
• pSB1C3.1:PS2#2
• pSB1C3.1:PS2#4
• pSB1C3.2:PS2#2
• pSB1C3.2:PS2#4
->16°C, 45 min
-> 80°C, 20 min
• 2 SDS-gels
Strav:SbsB isolation applied on SDS-gel
• 12µl probe + 4 µl PAP, 5 min 100°C
• At 100V for ˜ 1h in coomassie ON
Transformation pSB1C3.1:PS2 #2, pSB1C3.1:PS2 #4, pSB1C3.2:PS2#2 & pSB1C3.2:PS2#4 in E. coli DH5α
• Competent E. coli DH5α cells out of -80°C
• See p. 80 (instead of SOC-medium LB-medium was used)
• At 37°C ON incubated
09/25/2018 (AM, FE)
Inoculation of pSB1C3:PS2 constructs, pSB1C3 & pZ8-1
• In each 4 ml LB RG one colony
• For pSB1C3 constructs 4µl chloramphenicol (cm)
• For pZ8-1 4µl Kanamycin
• At 37°C ON shaking
2 SDS-gels poured
P1 probes of 09/13/2018: rest on SDS-gels applied + p3 of 09/13/2018
• 1h at 100V
• In coomassie ON
SDS-gels of SbsB:Strav isolation into 10%  acetic acid
• Heated in the microwave
• Discolored ON
09/26/2018 (AM, FE)
Plasmid-prep. of pZ8-1, pSB1C3, pSB1C3:pS2 constructs
• NucleoSpin PlasmidKit used
• High copy plasmid protocol
• Eluted in 50µl mQ
Measured via nanodrop:
Test digestion of pSB1C3:PS2 constructs Mastermix x5:
• 4µl plasmid prep (~25 ng/µl) + 4µl Mastermix in PCR-tubes
• 35 min at 37°C
• 20 min at 80°C
• On 1% agarose-gel applied
SDS-gel of P1-isolation of 09/13/2018 and P3 of 09/13/2018 discolored
• In 10% acetic acid
SDS-gels of SbsB:Strav isolation photographed
• No protein can be seen
• New SDS-gels poured and proteins applied
(10µl probe + 10µl PAP -> 5 min at 100°C
09/27/2018 (AM, FE)
Colony-PCR with pSB1C3:PS2 constructs
• Plates from 09/24/2018
Mastermix:
• Pick one colony and stir in the PCR-tube
• After that: colony in 4 ml LB
• At 37°C ON
PCR with His-Strav Biobrick & SbsB Biobrick
• Sequences solved like IDT-protocol
• Colony-PCR and His-Strav- & SbsB-PCR run with iGEM1 programm
-> 1% agarose-gel
SDS-gels of P1 + P3 isolation (09/13/2018) photographed
• On the SDS gel with the isolation of P3 (09/23/2018) no proteins can be seen
SDS-gels of Strav:SbsB-isolation discoloured
09/28/2018 (AM, FE)
Plasmidpreperation of pSB1C3:PS2#2
• Used constructs:
pSB1C3.1:PS2.1#2
pSB1C3.2:PS2.2#2.1
pSB1C3.2:PS2.2#2.2
• With NucleoSpin Plasmid Kit
-> high copy plasmid protocol, eluated in 50 µl mQ
• Concentration measured with nanodrop:
Control digestion of the pSB1C3:PS2 constructs
• Mastermix 5x:
• 4µl plasmid + 4µl MM in PCR tubes
• 30min at 37°C
• 20min at 80°C
-> 1% agarose gel
PCR with His-Strav % SbsB Biobrick Primer
• Run with iGEM_1 programm
• 1% agarose gel
SDS-gels of Strav:SbsB-isolation photographed
• No protein can be seen
-> new isolation of the Strav:SbsB construct
Week 25 (October 01 - October 07)
10/01/2018 (RV, AM)
EcoRI-HF/ Pst-HF digestion of Strav::SbsB constructs:
See 09/11/2018, but we did not digest with DpnI. Incubate 30 min at 37°C and 20 min at 80°C
Ligation Strav::SbsB constructs:
See ligation 09/11/2018.
Transformation Strav::SbsB constructs with E.coli:
• See protocol 7/30/2018
• Plates with Chloramphenicol
Inoculate new pre-culture with pET28a+ Strav::SbsB in BL21:
4x 150ml LB with 150µl Kanamycin with clone #1, #3, #4 or #5 and incubate at 37°C overnight.
10/02/2018 (RV, AM)
Make new main-culture of pET28a+ Strav::SbsB:
OD600 = 0.08 in 2l flasks with 500ml LB, let it grow to OD600 = 0.8
Preparation for protein-isolation:
• Inoculate C. glutamicum in 250ml BHI, incubate at 30°C over night
• Inoculate C. crescentus in 250ml PYE, incubate at 30°C over night
Isolation of SbsB-main culture and IPTG induction:
See protocol 09/11/2018
10/03/2018 (RV, AM)
Preparation for mini-prep.
Pick Strav::SbsB colonies and inoculate 4 ml LB + CAM
P2 isolation:
Isolation with 50mM HEPES pH 6.8 in 12 falcons with 30 ml culture OD
600=1 (see 06/27/2018).
10/04/2018 (RV, AM)
P3 isolation:
See protocol 04/20/2018.
Make new main-culture of pET28a+ Strav::SbsB:
• Mix Culture (OD600 = 0.115) with IPTG and incubate 4 hours (OD600 = 0.135)
• Centrifugate 5000g 30min
• Storage pellet on ice
Mini-prep. Strav::SbsB :
Use NucleoSpin Plasmid Kit
Colony PCR Strav::SbsB:
• Use plates from 02.10.
PCR-Program: see 07/02/2018.
Make new pre-culture of pET28a+ Strav::SbsB:
See protocol: 10/01/2018
10/05/2018 (RV, AM)
Induction of the main culture of pEt28a Strav:SbsB with IPTG:
• two main cultures with the same content
• they were diluted with LB-medium to a OD of 0.8
• IPTG was added (concentration at the end: 1mM)
•  Final volume main culture 1: 1.5l
•  Final volume main culture 2: 1l
• Incubation for 4h at RT (150rpm)
• Centrifugation at 5000g for 30min.
• Storage in the freezer in 50ml falcons
Completion of the parts for sending away
Week 26 (October 08 - October 14)
10/08/2018 (ST, MI)
Purification of Strav:SbsB:
Resuspension of Strav:SbsB pellets from 10/06/2018 in 5ml Buffer A per gram of pellet
Cell lysis via sonotrode: 5 cycles with 25 seconds each, at 50% power with 90s breaks for cooling on ice. Centrifuge at 4°C, 8500rpm for 1 hour and dispose of cell fragments. Sterile filtration of supernatant through 0.2 μm
Preparation of purification:
• prepare 14 ml of 25/50/75/100% elution buffer
• load two columns with Ni-NTA-agarose and let ethanol settle down and drain
• Rinse three times with 10 ml Buffer A
• Fill columns with 10 ml lysate
• Labeling of Eppis: 2*4*1.5 ml fractions for 25/50/75/100% EP each
Dispatching of part pSB103 p2
Tracking number; RR918 203 785DE
10/09/2018 (ST, MI)
Isolation of Strav:SbsB:
Purification via Ni-NTA-columns (one per main culture). See protocol 08/02/2018. Changes: 4*1.5 ml fractions per elutionbuffer step instead of 7*1 ml.
Received fractions (for main culture 1 and 2 each):
DF, W1 (10 ml), W2 (10ml), W3 (10ml), 25% EP*4 (1.5 ml), 50% EP*4 (1.5 ml), 75% EP*4 (1.5 ml), 100% EP*4 (1.5 ml) and 10 ml 100% EP (“fraction 5” of 100%)
MC1 Gel 1:
MC1 Gel 2:
MC2 Gel 1:
MC 2 Gel 2:
noculation of 100ml PTE with CB15A C.crescentus for RsaA-isolation at 28°C
10/10/2018 (ST, MI)
2*150 ml overnight preculture with Kan (50 µg/ml) was inoculated with BL21 DE3 pET 28a + Strav:SbsB at 37°C
10/11/2018 (ST, MI)
Documentation of Strav:SbsB – gels:
Measurement of oD of PC VK BL21DE3: oD600: 6.08 (PC 1 is used)
• 8.22 ml for 500 ml MC/6.58 ml for 400ml
• MC has an oD of 0.1
• 50 µg/ml Kanamycin and 10 mM MgCl2 was added
• Mc is shaken at room temperature and with 120 rpm
oD MC:
Induction with 1mM IPTG overnight
RsaA isolation of C.crescentus:
6*5 ml culture; rest in freezer. Combination of supernatants after isolation according to protocol (04/20/2018)
Native PAGE for Strav:SbsB:
Preparation of:
• 200ml 10x Native PAGE Running Buffer (1.92M glycine, 250 mM TRIS)
• 100ml TRIS-HCl pH 6.8
• 5ml native PAP (glycerol 50% v/v; bromophenol blue 1% w/v; TRIS HCl pH 6.8 300mM
10/12/2018 (ST, MI)
Harvesting pellets:
oD of main cultures 3.52. Transferring in 500 ml screw-cap bottle, 30 min at 4°C and 5000g in MPP.
Harvesting: 4g + 4.4g = 8.4 g. Pellet in freezer
Making 2 8% native PAGEs:
for 5 ml stacking gel: 1.6 ml 1M Tris-HCl ph 8.8 + 0.5 ml 40% acrylamide +50 µl 10% APS +5 µl TEMED and ad MQ 2.5 ml.
for 8% native seperating gel: 1.95 ml 40% acrylamide + 2.73 ml 1M Tris-HCl pH 8.8 +100 µl 10% APS + 10 µl TEMED and ad MQ 10 ml.
100V in cooling room for 1h 45min.
Pre – induction and post – induction probes are prepared in the cooler (PI/NI Strav:SbsB)
Week 27 (October 15 - October 17)
10/15/2018 (MAS, TM)
Native Page for Strav:SbsB:
See 10/12/2018
SbsB Isolation – continued:
See 08/01/2018
Chemistry Lab
Week 1 (June 27)
06/27/2018 (Christoph)
PS2 (Batch from 06/06/2018 Bio Lab) with HEPES (0.01 M) diluted to 10 μg*mL-1.
50 μg of Solution on each of four Silica pipetted.
In fume cupboard:
15 minutes exposed
30 minutes exposed
In Flow-Box:
15 minutes exposed
30 minutes exposed
Week 2 (July 4)
07/04/2018 (Christoph)
Repeated procedure of 06/27/2018
Week 3 (August 7 - August 10)
08/07/2018 (Christoph)
AFM: 50 µL PS2 (isolation 2, 02.08.2018)
Series of measurement:
• PS2: 0.44 mg/mL, 02.08.2018, M = 52.5 kg/mol, c = 8.38 x10-6 mol/L
• RsaA: 0.82 mg/mL, isolation 2, 03.08.2018, M = 98 kg/mol, c = 8.37 x10-6 mol/L
08/10/2018 (Christoph)
150 µL Polybead Polystyrene (2,6 % Solids-Latex; 350 nm) were diluted in 14,85 mL HEPES solution (0.01 M). This solution (Latex / HEPES 1:100) was used for the mixtures of the following measurements. Before every measurement the sample was heated to 50 °C for 10 min.
Used s-layer proteins:
• PS2, 0.44 mg/mL, isolation 2, 08/02/2018
• RsaA, 0.76 mg/mL, isolation 1, 08/03/2018
DLS:  Latex_1_10000_p2
->Latex / HEPES solution 1 : 10 000
Sample name: Latex_1_10000_p2
DLS:
Latex2_verd_p2
->Latex / HEPES solution 1 : 100 000
DLS:
Latex3_verd_p2
->Latex / HEPES solution 1 : 1 000 000
DLS:
p2_50grad
• 750 µL PS2 (0.44 mg/mL, 02.08.2018) + 750 µL HEPES (0.01 M)
->Solution was heated to 50 °C for 10 min
Week 4 (August 22)
08/22/2018 (Christoph)
SbsB (Batch from 08/15/2018); PS2 (Batch from 008/02/2018)
Total n(ges) = 1x10-9 mol per 1.5 mL (HEPES 0.01M, pH = 6.8).
Molar ratio (n/n) SbsB:PS2 (top to bottom): 1:0, 9:1, 8:2, 7:3, 6:4, 5:5, 4:6, 3:7, 2:8, 1:9, 0:1.
SbsB (Batch from 08/15/2018); RsaA (Batch from 08/16/2018)
Total n(ges) = 1x10--9 mol per 1.5 mL (HEPES 0.01M, pH = 6.8).
Molar ratio (n/n) SbsB:RsaA (top to bottom): 1:0, 9:1, 8:2, 7:3, 6:4, 5:5, 4:6, 3:7, 2:8, 1:9, 0:1.
Week 5 (August 28 - August 31)
08/28/2018 (Christoph)
PS2 (Batch from 8/02/2018); SbsB (Batch from 08/15/2018)
Total n(ges) = 1x10-9 mol per 1.5 mL (HEPES 0.01M, pH = 6.8). Heating time 1h.
08/29/2018 (Christoph)
PS2 (Batch from 08/02/2018); SbsB (Batch from 08/15/2018)
Total n(ges) = 1x10--9 mol per 1.5 mL (HEPES 0.01M pH = 2.1).
pH adjusted to 6.8 after addition of protein with 0.1M NaOH.
08/31/2018 (Christoph)
Repetition of measurements from 22.08.2018:
Solution was heated to 70 °C for 60 min
Results:
Week 6 (September 04 - September 06)
09/04/2018 (Andreas, Natalie W.)
PS2 (Batch from 08/02/2018); SbsB (Batch from 08/15/2018)
Total n(ges) = 1x10--9 mol per 1.5 mL (HEPES 0.01M, pH = 6.8).
Molar ratio (n/n) SbsB:PS2 (top to bottom): 1:0, 9:1, 8:2, 7:3, 6:4, 5:5, 4:6, 3:7, 2:8, 1:9, 0:1.
09/06/2018 (Andreas, Natalie W.)
PS2 (Batch from 08/02/2018); SbsB (Batch from 08/15/2018)
SbsB with n(ges) = 1x10--9 mol per 1.5 mL (HEPES 0.01M, pH 6.8). Heating time 1h.
Week 7 (September 10 - September 14)
09/10/2018 (Andreas, Natalie W.)
PS2 (Batch from 08/02/2018)
PS2 with n(ges) = 1x10--9 mol per 1.5 mL (HEPES 0.01M, pH 6.8). Heating time/sonication time 1h.
09/11/2018 (Andreas, Natalie W.)
PS2 (Batch from 08/02/2018))
PS2 with n(ges) = 5x10s-10- mol per 1.5 mL (HEPES 0.01M, pH 6.8).
Latex nano beads 350nm 2.6 wt% (Polyscience inc.) diluted in (HEPES 0.01M, pH 6.8) 1:100 (1), 1:1000 (2), 1:10000 (3).
09/14/2018 (Andreas, Natalie W.)
Series of measurement:
1) 750 µL PS2 (0,70 mg/ml, 04.09.2018) + 750 µL latex particles (350 nm, 1:1000 in HEPES solution); file: P2-LTX_1000
PS2-Tris (2.25 mg/ml) + 750 µL latex particles (350 nm, 1:1000 in HEPES solution); file: P2_Tris_Latex-1000
Series of dilution:
PS2 (4.21 mg/ml, isolation 2, 27.06.2018) in HEPES solution
Verd_P2_1500-0 => 1500 μL P2
Dilution 1 => 750 μL of dilution 0 + 750 μL HEPES
Dilution 2 => 750 μL of dilution 1 + 750 μL HEPES
Dilution 3 => 750 μL of dilution 2 + 750 μL HEPES
Dilution 4 => 750 μL of dilution 3 + 750 μL HEPES
Week 8 (September 18 - September 20)
09/18/2018 (Andreas, Natalie W.)
DLS:
750 μL PS2 (0.49 mg/mL, 08/02/2018) + 750 μL latex particles (100 nm, 1:1000 in HEPES solution)
DLS: PS2 (4.51 mg/mL, isolation 5, 06/27/2018) + latex particles (100 nm, 1:1000 in HEPES solution)
DLS:
75 µL PS2 (4.51 mg/mL, isolation 5, 06/27/2018) + 675 μL HEPES solution + 750 μL latex particles (100 nm, 1:1000 in HEPES solution)
TEM:
• SbsB (0.71 mg/mL, 50 %, 08/15/2018)
• PS2 (4.51 mg/mL, isolation 5, 07/27/2018) 1:20 in HEPES solution
• PS2 (4.51 mg/mL, isolation 5, 07/27/2018) 1:10 in HEPES solution + similar volume latex particles (100 nm, 1:1000 in HEPES solution)
• RsaA (06/03/2018)
09/19/2018 (Andreas, Natalie W.)
75 μL “P2 Isolation 27.06 Nr.5; 4,51mg/ml“ + 675 μL Hepes-Buffer + 750 μL “Latex 30nm 1:100 in Hepes”
750 μL Hepes-Buffer + 750 μL “Latex 30nm 1:100 in Hepes”
Analysis:
09/20/2018 (Andreas, Natalie W.)
Following samples were centrifuged:
Analysis:
Week 9 (October 1 - October 4)
10/01/2018 (Andreas, Natalie W.)
Analysis of the Cryo TEM images + comparison with DLS
P1:
E. Coli
- Size 1x3 μm (Literature)
Cryo TEM Image:
Particle with:
- Length 780nm
- Width 534nm
-> Particle in DLS ca 660nm < RH=330nm
DLS: 353,15K
Peak 1: 21nm (shell? Single S layer?)
Peak 2: 400nm (Particle)
P2:
Corynebacterium glutamicum
Size:
- Diameter:0,7 μm
- Length 1-1,5 μm
Cryo TEM Image: exel data
DLS: 343,15K
Peak 1: 95nm (small particle?)
Peak 2: 1,2 μm (large particle?, elongated particle?)
Peak 3: 2,5 μm (large particle?, elongated particle?)
P3:
Caulobacter crescentus:
Size:
- 1-2 μm
Cryo TEM Image: exel data
DLS: 353,15K
Peak 1: 32nm (circle)
Peak 2: 554nm (cell or image 1)
Peak 3: 2,3 μm (cell or image 1)
10/04/2018 (Andreas, Natalie W.)
centrifuged: 25min
PS2; 04.09; TRIS: white yellow solid
PS2; 6; HEPES; 5,52mg/mL; 16.08: yellow solid
PS2; 11; HEPES; pH6,8; 05.09.18; 1,28mg/mL: yellow solid
P2; 4; Isolation; 27.6; 4,86mg/mL: yellow solid
Analysis:
DLS: TEM_Latex_V3
750μL Latex
754μL HEPES
8μL Protein
DLS: TEM_Latex_V4
500μL Latex
1mL HEPES
1,5μL Protein
DLS: Trav_Protein
750μL HEPES
750μL Protein (EP 4)
DLS: Latex_100nm
750μL 1:1000 Latex 100nm
750μL Protein (EP 4)
Week 10 (October 10)
10/08/2018 (Andreas, Natalie W.)
Latex beads 50nm 1:1000 diluted with HEPES
UV-Vis
Solution emitted
DLS measured
Analysis:
Concentration of “12.9, 50%; SbSB; 50%; (1)/ (2); Eluationbuffer” aka P1 ε=1,007 ml/mgxcm
d=1 cm
UV Vis of Ltx 1:100 solved in HEPES
Waver coated with p1 and immediately deducted
TEM
750μL Latex Particle (fluoresces, 50nm)
750μL HR 2 Strav. SBSB 25% EP(3) from 10/8/2018
-> DLS: TEM_Latex
DLS: TEM_Latex_V2
760μL Latex as above
375μL HEPES 0,01M
375μL HR2 Strav. SBSB as above
Contact Us
project-igem@fau.de
Connect
