<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aachen_Notebook_Wetlab_October"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aachen/Notebook/Wetlab/October">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aachen/Notebook/Wetlab/October&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aachen/Notebook/Wetlab/October&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aachen/Notebook/Wetlab/October&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aachen/Notebook/Wetlab/October" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aachen/Notebook/Wetlab/October</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navDiv" style="top:14px; position:fixed; z-index: 1; border-bottom-style: solid; border-bottom-width: 3px; border-bottom-color: #bf1521;"><NAV style="width: 1250px;"><UL align="left" style=""><LI style="width:129px"><A href="https://2014.igem.org/Team:Aachen/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor">2D Biosensor</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter">The REACh Construct</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/Model">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/Measurement_Device"><I>WatsOn</I></A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/Project/Gal3">Galectin-3</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/OD/F device">OD/F Device</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Achievements">Achievements</A></LI></UL><LI style="width:216px"><A href="https://2014.igem.org/Team:Aachen/Interlab_Study">Interlab Study</A></LI><LI style="width:250px"><A href="https://2014.igem.org/Team:Aachen/PolicyPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Kaiser-Karls-Gymnasium">Kaiser-Karls-Gymnasium</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Neanderlab">NEAnderLab</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/Collaborations/MakerFaire">MakerFaire</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/PolicyPractices/Economics">Economical View</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/PolicyPractices/BioBrickIntellectualProperty">BBa Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Blog">Blog</A></LI><LI style="width:215px"><A href="https://2014.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Meetup">Aachen Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Bielefeld">Bielefeld</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Methane_Sensor">Braunschweig</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Freiburg">Freiburg</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Heidelberg">Heidelberg</A></LI><LI style="width:157px"><A href="https://2014.igem.org/Team:Aachen/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Engineering">Engineering</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Software">Software</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/LabEquipment">Equipment</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Index">Abbreviations</A></LI><LI style="width:109px"><A href="https://2014.igem.org/Team:Aachen/Attributions">Team</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#instructors">Instructors</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#partners">Partners</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#support">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/SecretHappenings"><I>Mr Holmes</I></A></LI></NAV></DIV><DIV id="contentdiv" style="margin-left: auto; margin-right: auto; width: 100%; position: absolute; bottom: 15px; top: 60px;"><TABLE class="transbg" width="100%" cellspacing="0"><TBODY><TR><TD class="contentcell" width="95%" align="justify"><P><CENTER><UL class="menusmall-grid"><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/April" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to April</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/May" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to May</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/June" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to June</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/July" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to July</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/August" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to August</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/September" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to September</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/October" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to October</DIV></DIV></A></LI></UL></CENTER></P><H1><SPAN class="mw-headline" id="October"> October </SPAN></H1><H2><SPAN class="mw-headline" id="1st"> 1st </SPAN></H2><UL><LI> Prepartations for sensor-chip production the following day (2014-10-02) was done accoringly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>:
</LI><LI> At 18:30 we prepared over-night cultures from K1319042, B0015 and K131026 by inoculating 250 ml Erlenmeyer flasks each containing 50 ml LB medium . The flasks were incubated for ~12 hours at 37°C on a shaker.
</LI></UL><H2><SPAN class="mw-headline" id="2nd"> 2nd </SPAN></H2><UL><LI> At 8:30, we did a plasmid prep of dublicate samples of K1319011 clone #1 and #6. We measured the DNA yield, and the higher concentrated sample of clone #1 and #6, respectively, were sent in for sequencing.
</LI><LI> made precultures and a master plate of 6 colonies of K3139008 in psB1C3 in NEB10β cells that had been plated at 5:30 this morning.
</LI></UL><UL><LI> Production of sensor-chips was done accordingly to <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>. Briefly:
</LI><LI> At 10:00 we prepared 150 ml 1.5% (w/v) LB+agarose solution. The LB+agarose solution was autoclaved and subsequently tempered to 45°C. Precultures (50 ml each) of K1319042, B0015 and K131026 were spun down at 3000 g for 10 min at 21°C and re-suspended in 1 ml pretempered (21°C) LB medium. The re-suspended cultures were mixed with 50 ml LB+agarose and poured onto three sensor-chip-templates (one template per culture). Sensor chips were cut out from the template and incubated at 37°C for 1 h.
</LI><LI> K1319042 and B0015 were induced with 0.2 µl IPTG (100 mM) subsequently to incubation and K131026 was induced with 0.2 µl homoserinlacton stock solution (500 µg/ml) 30 minutes after induction of the K1319042 and B0015. The induced sensor-chips were read out every 30 minutes for 180 minutes in total. An additional readout was conducted 285 minutes post induction. The readout was done at 450 nm and 480 nm wavelength. 
</LI></UL></TD></TR><TR><TD><B>Sensor Chips with B0015 in NEB in LB (480 nm): first try with final device</B>Sensor chips with B0015 in NEB in LB medium with 1,5% agarose, right chip induced. A) befor induction with 0.2 µl of 100 mM IPTG B) 0.5 h after induction C) 1 h after induction D) 1.5 h after induction E) 2 h after induction F) 2.5 h after induction
</TD></TR></TBODY></TABLE></DIV><UL><LI> Gibson assembly of K1319008
</LI><LI> template Backbone: I746909, Insert: K1319004
</LI><LI> transformation in <I>E.coli</I> NEB10β and BL21
</LI></UL><UL><LI> PCRs for Gibson assembly of K1319010 and K1319015
</LI><LI> template Backbone: I20260 for K1319010 and K1319015
</LI><LI> template Insert: K1319000 for K1319010 and K1319015 (but different primer)
</LI></UL><UL><LI> made precultures of K1319013 and K1319014 in pSB3K3
</LI></UL><UL><LI> K1319011 in pSB1C3 prepped for sequencing
</LI></UL><UL><LI> Gibson assembly of K1319017 (PCRs, Gibson assembly, restriction with DnpI, transformation into NEB10β)
</LI><LI> template Backbone: B0015
</LI><LI> template Insert 1: LasI synthesized gene
</LI><LI> template Insert 2: K660004
</LI></UL><H2><SPAN class="mw-headline" id="3rd">3rd</SPAN></H2><UL><LI> made master plates of K1319008 in NEB10β and BL21 and precultures
</LI></UL><UL><LI> check PCR for K1319008 to validate the Gibson assembly check for potential I746909 residues
</LI></UL><CENTER><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:800px;padding:0px;"><TABLE><TBODY><TR><TD><B>Check PCR K1319008</B>K1319008 was checked with the Primers K1319008_Check_R and I746909_Check_R (both with G00100 Alternative as froward primer) for presence of K1319008 or I746909. All tested clones were positive for K1319008 and negative for I746909.
</TD></TR></TBODY></TABLE></DIV></CENTER><UL><LI> did a plasmid prep and made cryo stocks of K1319008 in NEB (clones #1, #2, #3) and BL21 (clones #1, #2)
</LI></UL><UL><LI> plasmid prep of K1319013 and K1319014 in pSB3K3
</LI></UL><UL><LI> Gibson assembly for K1319010 and transformation into NEB10β
</LI></UL><UL><LI> made cryo stocks of K1319011 clone #6
</LI></UL><UL><LI> restriction of K1319012, k1319013 and K1319014 with EcoRI and PstI. Restriction of linearized plasmid backbone pSB1C3 with EcoRI and PstI. Ligation of K1319012, K1319013 and K1319014 into pSB1C3. Transformation into NEB10β. 
</LI></UL><UL><LI> Gibson assembly of K1319015 and transformation into NEB10β
</LI></UL><UL><LI> colony PCR of K1319017
</LI></UL><CENTER><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:800px;padding:0px;"><TABLE><TBODY><TR><TD><B>colony PCR K1319017</B>K1319017 was checked with a colony PCR for the right insert length. Clones #2 and #4 were correct and used forthwith.
</TD></TR></TBODY></TABLE></DIV></CENTER><UL><LI> did plasmid prep and cryo of clones #2 and #4 of K1319017
</LI></UL><UL><LI> new plasmid backbone of pSB1C3 was made using the <A href="http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones" class="external text" rel="nofollow">Standard Protocol</A>.
</LI></UL><UL><LI> transformation of K1319008 clone #1 (from BL21) and K1319013 into BL21 (two plasmids in one cell).
</LI></UL><UL><LI> transformation of K1319008 clone #1 (from BL21) and k1319014 into BL21 (two plasmids in one cell).
</LI></UL><UL><LI> OD measurements of three biological triplicates from <I>E. coli</I> BW21 113, <I>P. putida</I> and <I>S. cerevisiae</I>. Measurement as an analytic triplicate in the spectrophotometer (absorbance and transmission) and our own OD/F Device.
</LI></UL><UL><LI> Gibson assembly of K1319021 
</LI><LI> template Backbone: K1319008
</LI><LI> template Insert: LasI gene synthesis
</LI></UL><UL><LI> At 19:00 we measured OD (our OD-device), absorption (spectrophotometer) and transmission (spectrophotometer) for 19 diltuions in the range of 2.5-100% from yeast (<I>Saccharomyces cerevisiae</I>) and <I>P. putida</I> liquid cultures. Measurements were conducted in biological as well as technical triplicates. Aim of this experiment was the comparison of our OD-device to commonly used devices in terms of OD determination.
</LI></UL><UL><LI> Prepartations for sensor-chip production the following day (2014-10-04) were done accordingly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>: 
</LI><LI> At 22:00 we prepared over-night cultures from B0015, K1319017 and K131026 by inoculating 250 mL Erlenmeyer flasks each containing 50 mL LB medium for sensor-chip manufacturing the next day. The flasks were incubated for ~12 hours at 37°C on a shaker.
</LI></UL><UL><LI> Our BioBrick K1319021 enables expression of the TEV protease inducible by the autoinducers of <I>Pseumomonas aeruginosa</I>. In order to construct this BioBrick, a Gibson assembly of K1319008 (IPTG-inducible expression of TEV protease) and the synthezised composite HSL-promotor J23101.B0032.C0079.B0015.J64010.B0034 activated by the respective autoinducers (HomoSerineLactones) was perfomed. Prior to this, we used two PCRs to linearize pSB1C3-K1319008, serving as backbone for Gibson assembly, and cutting out J23101.B0032.C0079.B0015.J64010.B0034 as well as adding adequate overlapping sequences.
</LI></UL><CENTER><TABLE class="wikitable"><TBODY><TR><TD><DIV style="text-align: center;"><B>PCRs</B></DIV></TD><TD colspan="2"><DIV style="text-align: center;"><B>K1319008</B></DIV></TD><TD colspan="2"><DIV style="text-align: center;"><B>HSL-Promotor</B></DIV></TD></TR><TR><TH> step </TH><TH> time [mm:ss] </TH><TH> temperature [°C] </TH><TH> time [mm:ss] </TH><TH> temperature [°C] </TH></TR><TR><TD> Initial denaturation </TD><TD> 05:00 </TD><TD> 98 </TD><TD> 05:00 </TD><TD> 98 </TD></TR><TR><TD><B>Denaturation</B></TD><TD><B>00:30</B></TD><TD><B>98</B></TD><TD><B>00:30</B></TD><TD><B>98</B></TD><TD rowspan="3"><B>30 cycles</B></TD></TR><TR><TD><B>Annealing</B></TD><TD><B>00:30</B></TD><TD><B>55</B></TD><TD><B>00:30</B></TD><TD><B>51</B></TD></TR><TR><TD><B>Elongation</B></TD><TD><B>01:35</B></TD><TD><B>72</B></TD><TD><B>00:37</B></TD><TD><B>72</B></TD></TR><TR><TD> Final elongation </TD><TD> 05:00 </TD><TD> 72 </TD><TD> 05:00 </TD><TD> 72 </TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="4th">4th </SPAN></H2><UL><LI> colony PCR of K1319015 and Check PCR of K1319010
</LI><LI> K1319010: clone #1 was positive
</LI><LI> K1319015: in all clones the inserts were too short. 
</LI></UL><UL><LI> new digestion of Gibson master mix of K1319015 with DnpI
</LI></UL><UL><LI> transformation of new Gibson master mix into NEB 10β
</LI></UL><UL><LI> restriction of K1319010, K1319012, K1319013 and K1319014 (all in pSB3K3) with EcoRI and PstI, cutting of pSB1C3 with EcoRI and PstI, and then ligation.
</LI></UL><UL><LI> made master plates and precultures of the transformations of K1319008 in BL21, K1319013 + K1319008 in BL21 and K1319014 + K1319008 in BL21
</LI></UL><UL><LI> colony PCR of the master plates with the double plasmid constructs K1319013 + K1319008 and K1319014 + K1319008
</LI></UL><UL><LI> sent the first BioBricks to the iGEM headquarters:
</LI><LI> K1319000: RFC25 version of E0030
</LI><LI> K1319001: REACh1 (Quencher)
</LI><LI> K1319002: REACh2 (Quencher)
</LI><LI> K1319003: Galectin 3
</LI><LI> K1319004: TEV protease
</LI><LI> K1319008: IPTG inducible expression of TEV protease
</LI><LI> K1319011: J23101.B0032.K1319001.B0015
</LI><LI> K1319017: HSL inducible expression of iLOV
</LI><LI> K1319020: B0034.K1319009.B0015 in pSBX1A3
</LI><LI> K1319042: IPTG inducible expression of iLOV
</LI></UL><UL><LI> shake flask experiments with K1319008 (clone #1), K1319013 + K1319008 (clone #2) and K1319014 + K1319008 (clone #2) in LB (2 flasks each); inoculation with 50 µL preculture and inducing with iPTG at OD of 1.5.
</LI></UL><UL><LI> Production of sensor-chips was done accordingly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>: 
</LI><LI> At 13:30 we prepared sensor chips from pre-cultures of B0015, K1319017 and K131026.
</LI><LI> Subsequently to 1 h icubation at 37°C B0015, K1319017 and K131026 were induced with 0.2 µL homoserinlacton stock solution (500 µg/ml). The induced sensor-chips were read out every 30 minutes for 240 minutes in total. Readout was conducted at 450 nm and 480 nm wavelength. An additional readout was conducted after 12 hours.
</LI></UL><UL><LI> At 17:00 we prepared 4 liquid cultures from the K1319010_pSB3K3 master plate (clone#1) in 5 mL LB-medium, each. The liquid cultures were prepared in order to create cryo stocks from K1319010-pSB3K3. Kanamycin was added to the liquid cultures as antibiotic at an concentration of 1 µL/mL.
</LI></UL><UL><LI> At 18:00 we prepared a master plate (LB+C) and corresonding liquid cultures from 6 clones of <I>E.coli</I> NEB10B k1319021-psB1C3. Liquid cultures and master plate were incubated at 37°C.
</LI></UL><UL><LI> At 23:30 we prepared liquid cultures from K1319015-pSB3K3 clones #7, #8 and #9 in 5 mL LB-medium. Kanamycin was used as antibiotic and the cultures were incubated at 37°C. Purpose of the cultures was cryo stock preparation and plasmid prep.
</LI></UL><H2><SPAN class="mw-headline" id="5th">5th </SPAN></H2><UL><LI> At 11:45 we prepared cryo stocks from NEB K1319010-pSB3k3 #1, K1319015-pSB3K3 #7, K1319015-pSB3K3 #8 and K1319015-pSB3K3 #9 by mixing 750 µl lquid culture with 750 µl 50% (v/v) Glycerol-solution in 2 ml eppis.
</LI><LI>Plasmid prep was also done for the cultures mentioned above.
</LI></UL><UL><LI> At 12:00 we prepared liquid cultures from K139010-pSB3K3, K139011-pSB3K3, K139012-pSB3K3, K139013-pSB3K3, K139014-pSB3K3, K139015-pSB3K3 in 5 ml LB-medium each. Kanamycin was used as antibiotic. Purpose for the cultures was the characterization of constituitive expression and an additional plasmid prep of K139013-pSB3K3 and K139014-pSB3K3.
</LI></UL><UL><LI> Production of sensor-chips was done accordingly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>:
</LI><LI> At 13:00 we prepared sensor chips from shake flask pre-cultures of BL21 pSB1C3-K1319008+pSB3K3-K1319014, BL21 pSB1C3-K1319008+pSB3K3-K1319013 and NEB pSB1C3-B0015.
</LI><LI>Subsequently to 1 h incubation at 37°C, BL21 pSB1C3-K131900+pSB3K3-K1319014, BL21 pSB1C3-K1319008+pSB3K3-K1319013 and NEB pSB1C3-B0015 were induced with 0.2 µL IPTG. The induced sensor-chips were read out every 30 minutes for 360 minutes in total. K1319013 was induced earlier and thus measurements were taken for 450 min in total. Readout was conducted at  480 nm wavelength. An additional readout was conducted next day at 11:00.
</LI></UL><UL><LI> At 15:30 we prepared liquid cultures for the characterization of ITPG inducible expression:
</LI><LI> I746909 in pSB1C3
</LI><LI> I20260 in pSB3K3
</LI><LI> K731520 in pSB1C3
</LI><LI> K1319008 in pSB1C3
</LI><LI> K1319013 in pSB3K3
</LI><LI> K1319014 in pSB3K3
</LI><LI> B0015 in pSB1C3
</LI><LI> K1319013 in pSB3K3 + K1319008 in pSB1C3
</LI><LI> B0015 in pSB1C3
</LI><LI> K1319014 in pSB3K3 + K1319008 in pSB1C3
</LI></UL><UL><LI> prepared a colony PCR from K1319014-pSB1C3 clones #3, #4, #5 and#6. H20 and K1319014-pSB3K3 were used as controls.
</LI></UL><UL><LI> At 22:20 we prepared 1 L LB+C plates (2.5 g NaCl, 10 g Agar, 2.5 g yeast extract and 5 g Trypton). N-Z-amine (peptone from casein) was used instead of tryptone, because the tryptone stock was depleted.
</LI></UL><UL><LI> prepped multiple cultures K1319013 in pSB3K3 and K1319014 in pSB3K3. Now we have sufficient amount of both plasmids.
</LI></UL><UL><LI> Plates were made for the following constructs:
</LI></UL><CENTER><TABLE class="wikitable"><TBODY><TR><TH> part in vector </TH><TH> strain </TH><TH> resistance
</TH></TR><TR><TD> K1319008 in pSB1C3 </TD><TD> BL21(DE3) </TD><TD> C
</TD></TR><TR><TD> K1319010 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> K1319011 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> K1319012 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> K1319013 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> K1319014 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> K1319015 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> K1319017 in pSB1C3 </TD><TD> NEB10β </TD><TD> C
</TD></TR><TR><TD> K1319042 in pSB1C3 </TD><TD> BL21(DE3) </TD><TD> C
</TD></TR><TR><TD> K1319008 in pSB1C3 + K1319013 in pSB3K3 </TD><TD> BL21(DE3) </TD><TD> C+K
</TD></TR><TR><TD> K1319008 in pSB1C3 + K1319014 in pSB3K3 </TD><TD> BL21(DE3) </TD><TD> C+K
</TD></TR><TR><TD> I746909 in pSB1C3 </TD><TD> BL21(DE3) </TD><TD> C
</TD></TR><TR><TD> K731520 in pSB1C3 </TD><TD> DH5α </TD><TD> C
</TD></TR><TR><TD> I20260 in pSB3K3 </TD><TD> NEB10β </TD><TD> K
</TD></TR><TR><TD> B0015 in pSB1C3 </TD><TD> NEB10β </TD><TD> C
</TD></TR></TBODY></TABLE></CENTER><UL><LI> To determine whether the conducted Gibson assembly of K1319021 and subsequent transformation into <I>E. coli</I> NEB 10β were successful, a colony PCR on these cells was performed. Primers binding to each of the templates used for Gibson assembly were used: LasI_Insert_F binding in J23101.B0032.C0079.B0015.J64010.B0034 and K1319004_Check_R binding in pSB1C3-K1319008. Bands at 1341 bp would indicate the successful construction and transformation of K1319021. 
</LI></UL><CENTER><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:800px;padding:0px;"><TABLE><TBODY><TR><TD><B>Check PCR on K1319021</B>(Homoserinlactone-inducible expression of the TEV protease)
</TD></TR></TBODY></TABLE></DIV></CENTER><P>Since Bands &gt;2000 bp were observed, another PCR was performed by using primers binding upstream and downstream of the insert in pSB1C3: G00100_alternative and G00101_alternative. Here, bands with a length of 2210 bp would verify the correct length of K1319021.
</P><H2><SPAN class="mw-headline" id="6th">6th </SPAN></H2><UL><LI> made a SDS page of K1319010-15 and J23101.E0240
</LI></UL><UL><LI> made a master plate at 15:30 containing two clones of K1319015 plate from yesterday.
</LI></UL><UL><LI> At 22:00 we inocculated two 500 ml flasks containing 50 ml LB-medium with 1 ml pre-culture of K1319017, which had been prepared from a master-plate earlier this day. Chloramphinicol was used as antibiotic. The flasks were incubated at 37°C. One culture (50 ml) was induced with 25 µg homoserinlacton after an OD of 0.6 was reached and the induced culture as well as the control were used for fluorescence characterization. Fluorescene and OD were monitored once per hour. The OD of the induced culture stagnated at ~0.7 while the non induced culture grew further.
</LI></UL><H2><SPAN class="mw-headline" id="7th"> 7th </SPAN></H2><UL><LI> Two conducted colony PCRs confirmed that our double plasmid systems of K1319008 and K13190013/14 contain nothing but the desired BioBricks, which were used as positive controls. 
</LI></UL><CENTER><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:800px;padding:0px;"><TABLE><TBODY><TR><TD><B>colony PCR cells harboring the double plasmid system pSB3K3-K1319008 pSB1C3-K1319013/14</B>(IPTG-inducible expression of the TEV protease and constitutive expression of our GFP-Quencher-Constructs)
</TD></TR></TBODY></TABLE></DIV></CENTER><UL><LI>Characterisation of the biobrick K1319008 together with the biobricks K1319014 and K1319013.
</LI></UL><UL><LI>Therefore the following biobricks were cultivated in biological triplicates:
</LI></UL><CENTER><TABLE class="wikitable"><TBODY><TR><TH>biobrick </TH><TH> strain </TH><TH> plasmid </TH><TH> induced with iPTG
</TH></TR><TR><TD>K1319008 </TD><TD> BL21 </TD><TD> pSB1C3 </TD><TD> no
</TD></TR><TR><TD>K1319008 </TD><TD> BL21 </TD><TD> pSB1C3 </TD><TD> yes
</TD></TR><TR><TD>K1319013 + K1319008 </TD><TD> BL21 </TD><TD> pSB3K3 + pSB1C3 </TD><TD> no
</TD></TR><TR><TD>K1319013 + K1319008 </TD><TD> BL21 </TD><TD> pSB3K3 + pSB1C3 </TD><TD> yes
</TD></TR><TR><TD>K1319014 + K1319008 </TD><TD> BL21 </TD><TD> pSB3K3 + pSB1C3 </TD><TD> no
</TD></TR><TR><TD>K1319014 + K1319008 </TD><TD> BL21 </TD><TD> pSB3K3 + pSB1C3 </TD><TD> yes
</TD></TR><TR><TD>K731520 </TD><TD> BL21 </TD><TD> pSB1C3 </TD><TD> no
</TD></TR><TR><TD>K731520 </TD><TD> BL21 </TD><TD> pSB1C3 </TD><TD> yes
</TD></TR><TR><TD>I746909 </TD><TD> BL21 </TD><TD> pSB1C3 </TD><TD> no
</TD></TR><TR><TD>I746909 </TD><TD> BL21 </TD><TD> pSB1C3 </TD><TD> yes
</TD></TR><TR><TD>B0015 </TD><TD> NEB 10 Beta </TD><TD> pSB1C3 </TD><TD> no
</TD></TR><TR><TD>I20260 </TD><TD> BL21 </TD><TD> pSB3K3 </TD><TD> no
</TD></TR></TBODY></TABLE></CENTER><UL><LI> overnight culture of K1319013 + K1319008 # 1 and # 2, K1319014 + K1319008 # 1 and # 2 for chip production (<A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>).
</LI></UL><H2><SPAN class="mw-headline" id="8th"> 8th </SPAN></H2><UL><LI> We prepared sensor-chips with K1319013 + K1319008 # 1 and # 2, K1319014 + K1319008 # 1 and # 2 and B0015 accordingly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>. Images were made every 30 min with our own device.
</LI><LI> made precultures with K1319013 + K1319008 # 1 and # 2, K1319014 + K1319008 # 1 and # 2 and K731520. 
</LI><LI> cutting K1319013, K1319014 ans a pSB1A3 vector backbone with EcoRI and SpeI and K1319008 with XbaI and PstI
</LI><LI> Ligation of K1319013/K1319014 with K1319008 and then ligation into pSB1A3
</LI><LI>Transformation of the ligation product into BL21
</LI></UL><H2><SPAN class="mw-headline" id="9th"> 9th </SPAN></H2><UL><LI> SOC medium was added to the transformation before the heat shock had occurred by mistake.
</LI><LI> At 02:00 the precultures for the characterization experiment were transferred to 250 ml shake flasks (3 ml culture + 10 ml LB+antibiotics).
</LI><LI> new transformation of the ligation product was conducted into BL21 and DH5α
</LI><LI> At 23:00 a 500 ml flask containing 50 ml LB-medium was inocculated from a K131026 BL21 cryo stock for sensor-chip manufacturing the following day.
</LI></UL><H2><SPAN class="mw-headline" id="10th"> 10th </SPAN></H2><UL><LI> made master plates of the new transformation of the ligation product and overnight cultures
</LI><LI> made chips with K131026 in BL21. Images were taken automaticly every 5 min with our own device
</LI></UL><UL><LI> For sensor-chip manufacturing the following day, we inocculated 500 ml flasks containing 50 ml LB-medium with NEB10β K1319017.pSB1C3 clone #2 and with BL21 K1319042.pSB1C3 clone 2# accordingly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>. Inoccultion was done at 21:35.
</LI><LI>In preparation for an additional main charcterization experiment we inocculated 200 ml flasks or 300 ml flasks as available containing 25 ml LB-medium with seven of our own constructs listd below. Inoccuaation was done at 22:00.
</LI><LI>K1319010
</LI><LI>K1319011
</LI><LI>B0015
</LI><LI>I20260
</LI><LI>K1319015
</LI><LI>K1319014
</LI><LI>K1319012
</LI><LI>k1319013
</LI></UL><H2><SPAN class="mw-headline" id="11th">11th</SPAN></H2><UL><LI>At 21:00 we inocculted two 500 ml flasks containing 50 ml LB-medium with BL21 K1319042.pSB1C3 (from plate) and DH5α K1319026.pSB1C3 (from cryo), respectively. Chloramphenicol was used as antibiotik. Both cultures were required for chip manufacturing the next day (<A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>). The chips´ fluorescence was meant to be measured with a plate reader starting ~11:00 the next day. The chips were meant to be measured in parallel.
</LI></UL><UL><LI>At 20:30 eight cultures listed below were plated out for additional characterization experiments on monday.
</LI><LI>B0015 in pSB1C3
</LI><LI>I20260 in pSB3K3
</LI><LI>K731520 pSB1C3 #2
</LI><LI>I746909 in pSB1C3
</LI><LI>K1319042 i pSB1C3 #2
</LI><LI>K1319008 in pSB1C3 #1
</LI><LI>K1319008/13 in pSB1C3/3K3 #1
</LI><LI>K1319008/14 in pSB1C3/3K3 #2
</LI></UL><H2><SPAN class="mw-headline" id="12th">12th</SPAN></H2><UL><LI>At 9:00 sensor-chips were prepared from precultures of BL21 K1319042.pSB1C3 and DH5α K1319026pSB1C.pSB1C3 accordingly to the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">sensor-chip manufacturing protocol</A>. The cultures were induced at ~11:00 and fluorescence was measured using a plate-reader. K1319026 was additionally measured in our device for its fluorescence.
</LI><LI> Media with different antibiotics for the experiment for collaboration with Heidelberg were prepared. Also media for Robolektor were made.
</LI><LI> Cultures of I13507 in seven vectors listed below were solued in 0,9% NaCl for the OD measurment. Then all cultures and an additional positive control were inoculated in 96-wellsplate with resulted OD of 0,648. The cultures were required for calibration of the plate-reader used for fluorescence measurement, which was part of the Heidelberg chracaterization project. Inocculation was done at ~19:15.
</LI><LI>pSBX1A3
</LI><LI>pSBX4A5
</LI><LI>pSBX1C3
</LI><LI>pSBX4C5
</LI><LI>pSBX1K3
</LI><LI>pSBX4K5
</LI><LI>pSBX1T3
</LI></UL><UL><LI>At 22:15 a 300 ml flask containing 30 ml LB-medium was inocculated with I746909.pSB1C3. The culture was required for calibration of the robolector (Jülich FHZ). Cloramphenicol was used as antibiotic.
</LI></UL><H2><SPAN class="mw-headline" id="13th">13th</SPAN></H2><UL><LI> overnight culture of K131026 in BL21 for chips
</LI></UL><H2><SPAN class="mw-headline" id="14th">14th</SPAN></H2><UL><LI> made chips of K131026 in BL21 in LB. Images were taken nearly every 5 or 2 min with our own device. Chips were induced with 2 µl of <I>Pseudomonas aeruginosa</I> liquid culture or 500 µg/ml HSL
</LI><LI> overnight cultures of K1319013 + 8 and K1319014 + 8 in BL21 for chips
</LI></UL><H2><SPAN class="mw-headline" id="15th"> 15th </SPAN></H2><UL><LI> made chips of K1319013 + 8 and K1319014 + 8 (BL21) in LB. Images were taken every 2 min with our own device and every 4 min in the plate reader. Chips were induced with 2 µl of IPTG (100 mM).
</LI><LI> made overnight cultures of K1319013 + 8, K1319014 + 8, K731520 and I746909 for chips
</LI></UL><H2><SPAN class="mw-headline" id="16th"> 16th </SPAN></H2><UL><LI> made chips of K1319013 + 8, K1319014 + 8, K731520 and I746909 in LB. Images were taken every 2 min with our own device from K731520 and I746909 and every 4 min from all strains in the plate reader. Chips were induced with 2 µl of IPTG (100 mM).
</LI></UL><P><CENTER><UL class="menusmall-grid"><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/April" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to April</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/May" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to May</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/June" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to June</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/July" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to July</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/August" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to August</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/September" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to September</DIV></DIV></A></LI><LI style="width:106px;margin-left: 12px;margin-right: 12px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab/October" style="color:black"><DIV class="menusmall-item menusmall-info" style="height:100px; width: 100px;"><DIV class="menukachel" style="top: 30%; font-size: 16px;">Go to October</DIV></DIV></A></LI></UL></CENTER></P></DIV><DIV class="footerDiv" align="center" style="margin-bottom:-10px; background:#0069b3;"><CENTER><A class="footerCenter" href="https://2014.igem.org/Team:Aachen/Contact">Contact</A><A class="footerCenter" href="https://2014.igem.org/Team:Aachen/Disclaimer">Disclaimer</A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Aachen/Notebook/Wetlab/October">http://2014.igem.org/Team:Aachen/Notebook/Wetlab/October</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aachen/Notebook/Wetlab/October" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aachen/Notebook/Wetlab/October" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aachen/Notebook/Wetlab/October&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aachen/Notebook/Wetlab/October&amp;oldid=360075" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>