<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aachen_Project_2D_Biosensor"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aachen/Project/2D_Biosensor">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aachen/Project/2D_Biosensor&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aachen/Project/2D_Biosensor&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aachen/Project/2D_Biosensor&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aachen/Project/2D_Biosensor" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aachen/Project/2D Biosensor</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navDiv" style="top:14px; position:fixed; z-index: 1; border-bottom-style: solid; border-bottom-width: 3px; border-bottom-color: #bf1521;"><NAV style="width: 1250px;"><UL align="left" style=""><LI style="width:129px"><A href="https://2014.igem.org/Team:Aachen/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor">2D Biosensor</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter">The REACh Construct</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/Model">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/Measurement_Device"><I>WatsOn</I></A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/Project/Gal3">Galectin-3</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/OD/F device">OD/F Device</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Achievements">Achievements</A></LI></UL><LI style="width:216px"><A href="https://2014.igem.org/Team:Aachen/Interlab_Study">Interlab Study</A></LI><LI style="width:250px"><A href="https://2014.igem.org/Team:Aachen/PolicyPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Kaiser-Karls-Gymnasium">Kaiser-Karls-Gymnasium</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Neanderlab">NEAnderLab</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/Collaborations/MakerFaire">MakerFaire</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/PolicyPractices/Economics">Economical View</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/PolicyPractices/BioBrickIntellectualProperty">BBa Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Blog">Blog</A></LI><LI style="width:215px"><A href="https://2014.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Meetup">Aachen Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Bielefeld">Bielefeld</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Methane_Sensor">Braunschweig</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Freiburg">Freiburg</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Heidelberg">Heidelberg</A></LI><LI style="width:157px"><A href="https://2014.igem.org/Team:Aachen/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Engineering">Engineering</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Software">Software</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/LabEquipment">Equipment</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Index">Abbreviations</A></LI><LI style="width:109px"><A href="https://2014.igem.org/Team:Aachen/Attributions">Team</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#instructors">Instructors</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#partners">Partners</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#support">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/SecretHappenings"><I>Mr Holmes</I></A></LI></NAV></DIV><DIV id="contentdiv" style="margin-left: auto; margin-right: auto; width: 100%; position: absolute; bottom: 15px; top: 60px;"><TABLE class="transbg" width="100%" cellspacing="0"><TBODY><TR><TD class="contentcell" width="95%" align="justify"><H1><SPAN class="mw-headline" id="2D_Biosensor"> 2D Biosensor </SPAN></H1><P>With our 2D biosensor technology we are able to detect the pathogen <I>Pseudomonas aeruginosa</I> on solid surfaces. The sensor system is comprised of <B>two distinct but inseparable modules</B>, a biological and a technical part:
</P><UL><LI> Sensor chips containing <I><B>Cellocks</B></I>, our <B>engineered detective cells</B> that fluoresce in the presence of the pathogen, make up the biological part of <I>Cellock Holmes</I>.
</LI><LI> Our <B>measurement device <A href="https://2014.igem.org/Team:Aachen/Project/Measurement_Device" class="external text" rel="nofollow"><I>WatsOn</I></A></B> and the complementary <B>software <A href="https://2014.igem.org/Team:Aachen/Notebook/Software/Measurarty" class="external text" rel="nofollow"><I>Measurarty</I></A></B> complete our sensing technology on the technical side. 
</LI></UL><CENTER><P><UL class="team-grid" style="width:1064px;"><LI style="width:220px;margin-left: 23px;margin-right: 23px;margin-bottom: 23px;margin-top: 23px;"><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor#biosensorpoo" style="color:black"><DIV class="team-item team-info" style="width:214px;height:214px;"><DIV class="menukachel" style="top:32%;line-height:1.5em;">Principle of Operation</DIV></DIV></A></LI><LI style="width:220px;margin-left: 23px;margin-right: 23px;margin-bottom: 23px;margin-top: 23px;"><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor#biosensordevelopment" style="color:black"><DIV class="team-item team-info" style="width:214px;height:214px;"><DIV class="menukachel" style="top:32%;line-height:1.5em;">Development &amp; Optimization</DIV></DIV></A></LI><LI style="width:220px;margin-left: 23px;margin-right: 23px;margin-bottom: 23px;margin-top: 23px;"><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor#biosensorachievements" style="color:black"><DIV class="team-item team-info" style="width:214px;height:214px;"><DIV class="menukachel" style="top:40%">Achievements</DIV></DIV></A></LI><LI style="width:220px;margin-left: 23px;margin-right: 23px;margin-bottom: 23px;margin-top: 23px;"><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor#biosensoroutlook" style="color:black"><DIV class="team-item team-info" style="width:214px;height:214px;"><DIV class="menukachel" style="top:40%;">Outlook</DIV></DIV></A></LI></UL></P></CENTER></TD></TR></TBODY></TABLE><TABLE class="transbg" width="100%" cellspacing="0"><TBODY><TR><TD class="contentcell" width="95%" align="center"><H2><SPAN class="mw-headline" id="Principle_of_Operation"> Principle of Operation </SPAN></H2><P><I>Cellock Holmes</I> is designed upon a SynBio approach comprising a <B>two-dimensional biosensor and a measurement unit</B>. The two-dimensional biosensor is devised to recognize quorum sensing molecules secreted by the pathogen cells and to generate a distinct fluorescence signal; while the measurement device recognizes and analyzes the produced signal. On the molecular side, we use the <B><A href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter" class="external text" rel="nofollow">REACh construct</A></B> to transform regular <I>E. coli</I> cells into <I>Cellocks</I>.
</P></TD></TR><TR><TD><B>The principle of quorum sensing</B>Microorganisms can sense the presence of their own kind based on quorum sensing which is a form of chemical communication. Depending on their cell density, quorum sensing allows these cells to activate or deactivate certain gene expression cascades (Waters and Bassler, 2005) for a specific function.
</TD></TR></TBODY></TABLE></DIV><P>Our <B>sensor cells, <I>Cellocks</I>, are immobilized in agar chips</B>. To make the chips, we mix the <I>Cellocks</I> with liquid LB agar. 
In the course of our project, we designed a casting mold specifically for the production of our agar chips. When the agar has cooled down, the chips are cut out of the mold and are ready to use. Storage of the readily usable sensor chips is possible for up to two days at 4°C when using LB medium or up to five days if TB medium is used. A detailed description of the sensor chip manufacturing can be found in our <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols/detection" class="external text" rel="nofollow">Protocols</A> section.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>Assay to detect <I>P. aeruginosa</I> using <I>Cellock Holmes</I></B>This flow sheet shows the procedure to sample and detect <I>P. aeruginosa</I>: A sampling chip is briefly put onto the potentially contaminated surface, added onto one of our sensor chips and inserted into <I>WatsOn</I>.
</TD></TR></TBODY></TABLE></DIV><P>Using <I>Cellock Holmes</I>, we developed a simple assay to detect <I>P. aeruginosa</I>. Initially, a so called sampling chip is placed on a solid surface that is potentially contaminated with the pathogen. Subsequently, the sampling chip is removed from the surface and put onto one of our sensor chips. Theorectically, the sensor chips could be directly used for sampling, however, this was avoided in our project to <B>match  <A href="https://2014.igem.org/Team:Aachen/Safety" class="external text" rel="nofollow">biosafety regulations</A></B> and to prevent the spread of genetically modified organisms (GMOs) into the environment. The two layered chip-stack is then put into a petri dish which is inserted into our measurement device <A href="https://2014.igem.org/Team:Aachen/Project/Measurement_Device" class="external text" rel="nofollow"><I>WatsOn</I></A> for evalutation.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>Mode of action inside <I>WatsOn</I></B>Chips are incubated at 37°C to stimulate cell growth and then illuminated with blue light to excite fluorescence. A picture is taken and analyzed for fluorescence signals using the software <I>Measurarty</I>.
</TD></TR></TBODY></TABLE></DIV><P>Inside <I>WatsOn</I>, the chips are incubated at 37°C and the sampled populations of microorganisms attached on the sampling chip start to grow and multiply. During incubation the chips can be <B>illuminated with blue light</B> at any time, and a <B>photo of the chips</B> is taken. The <B>software <I>Measurarty</I></B> then analyzes any fluorescent signal. <I>P. aeruginosa</I> secrets an increasing number of quorum sensing molecules that are recognized by <I>Cellocks</I>, thereby producing a fluorescence signal. For detection of <I>P. aeruginosa</I>, we focused on a quorum sensing molecule called N-3-oxo-dodecanoyl-L-homoserine lactone (for short: 3-oxo-C<SUB>12</SUB>-HSL), which is involved in virulence regulation of <I>P. aeruginosa</I> (Jimenez, Koch, Thompson et al., 2012). The incorporation of the 3-oxo-C<SUB>12</SUB>-HSL detection system into the <I>Cellocks</I> is explained in detail in the <A href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter" class="external text" rel="nofollow">REACh Construct</A> section.
</P><H2><SPAN class="mw-headline" id="Development_.26_Optimization"> Development &amp; Optimization </SPAN></H2><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:500px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><B>iLOV and GFP in the Gel Doc<SUP>TM</SUP></B>Sensor cells producing iLOV (K1319042, A) and GFP (B) 1.5 h after induction.
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Equipment_and_medium_selection"> Equipment and medium selection </SPAN></H3><P>Our first approach (before developing our own device) was to use the Molecular Imager® Gel Doc™ XR+ from BIO-RAD in our lab to detect fluorescence. This device uses UV and white light illuminators. However, only two different filters were available for the excitation light wavelength, which resulted in very limited possibilities for the excitation of fluorescent molecules. For example, it was possible to detect the expression of iLOV (K1319042) in our sensor chips, but not the expression of GFP. Hence, the <B>Gel Doc™ was not suitable for our project</B>.
</P><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:500px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><B>Differend medium in the Gel Doc™</B>complex media exhibited high background fluorescence while less back- ground fluorescence was observed with the minimal media (HM, M9, NA).
</TD></TR></TBODY></TABLE></DIV><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:500px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><B>Testing our chips' shelf-life</B>Chips of <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> in NEB were stored five days at 4°C. The right chip was induced with 0.2 µL of 500 µg/mL HSL and an image was taken after 1.5 h.
</TD></TR></TBODY></TABLE></DIV><P>We tested different media (LB, TB, M9, NA and HM) for the preparation of our sensor chips. The medium compositions can be found in the <A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols" class="external text" rel="nofollow">Protocols</A> section. We screened for an optimized medium composition to minimize background fluorescence and to enhance cell growth. The results of the analysis are presented in the table below. Due to the low background fluorescence in <I>WatsOn</I> and the excellent cell growth, we <B>chose LB medium</B> over the other tested media for sensor chip manufacturing.
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TH> LB </TH><TH> TB </TH><TH> NA </TH><TH> M9 </TH><TH> HM
</TH></TR><TR><TD> Growth of <I>Cellock</I></TD><TD><DIV style="text-align: center;"><B>+</B></DIV></TD><TD><DIV style="text-align: center;"><B>+</B></DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD></TR><TR><TD> Background fluorescence in GelDoc </TD><TD><DIV style="text-align: center;"><B>+</B></DIV></TD><TD><DIV style="text-align: center;"><B>+</B></DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD></TR><TR><TD> Background fluorescence in <I>WatsOn</I></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;"><B>+</B></DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD><TD><DIV style="text-align: center;">-</DIV></TD></TR></TBODY></TABLE></CENTER><P>Another set of experiments were conducted to test the <B>long-time storage</B> of the sensor chips. We varied the glycerol content of the chips as well as the storage temperature. Storage at -20°C resulted in the loss of our sensor cells. Adding 5-10% (v/v) glycerol ensured survival of the sensor cells, but resulted in the loss of fluorescence ability. Hence, we concluded that long-time storage of the sensor chips at -20°C is not possible under the tested conditions. However, the 'ready-to-use' sensor chips can be kept at at 4°C for two days when using LB medium, and storage at this temperature for 5 days is possible with chips made from TB medium.
</P><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:500px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><B>Sensor chip manufacturing using the closed mold</B>When injecting the liquid agar into a closed mold we encounter problems due to frequent bubble formation.
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Optimal_Agarose_Concentration_for_Sensor_Chip_Manufacturing"> Optimal Agarose Concentration for Sensor Chip Manufacturing </SPAN></H3><P>For the sensor chip manufacturing, agarose was preferred over agar because of the uniform linkage between molecules that results in a better chip homogeneity. In addition, agarose reduced diffusion of the inducer molecules through the chip. A reduction in diffusion is essential for the formation of distinct fluorescent spots on the sensor chips.
</P><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:500px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><B>The finalized chip mold</B>An open casting mold was found to be optimal for sensor chip manufacturing, because this approach was fast, easy to handle and generated a reproducible chip quality.
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Optimal_Chip_Configuration"> Optimal Chip Configuration </SPAN></H3><P>Several approaches were tested for the production of agarose-based sensor chips with reproducible quality. The first approach was to cast every sensor chip individually. To achieve a plain chip surface, a requirement for high quality images, we casted the sensor chips between two microscope slides. However, this approach was not adequate because the agar was too liquid and leaked from the microscope slides. In a second approach, we designed a closed mold into which liquid agar is injected using a pipette, but we encountered a high number of bubbles in the resulting chips. Bubbles in the sensor chips interfered with fluorescence evaluation. Finally, we tried an open casting mold. Once solidified, we cut the agar along precast indentations in the casting mold to form the chips. An advantage of the open mold is the ability to simultaneously produce nine sensor chips while the surface tension of the liquid agar ensures a plane chip surface.
</P><H3><SPAN class="mw-headline" id="Induction_of_the_Sensor_Chips"> Induction of the Sensor Chips </SPAN></H3><P>To test our molecular constructs, we simulated the presence of <I>P. aeruginosa</I> by using IPTG or 3-oxo-C<SUB>12</SUB>-HSL. Initial experiments showed that diffusion of the inducers hinder the formation of distinct fluorescent spots. Through this set of experiments we determined that the best compromise between diffusion and spot intensity is an induction volume of 2.0 µL for IPTG and 0.2 µL for HSL. Furthermore, detection of growing <I>P. aeruginosa</I> based on secreted HSLs was possible using the <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> construct. The experiments for optimizing the induction of our sensor chips are described in more detail in the <A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor#biosensorachievements" class="external text" rel="nofollow">Achievements</A> section.
</P><H3><SPAN class="mw-headline" id="Negative_Control"> Negative Control </SPAN></H3><P>To ensure that the fluorescence signal resulted from the sensor construct and not from the medium or <I>E. coli</I> cells themselves, <A href="http://parts.igem.org/Part:BBa_B0015" class="external text" rel="nofollow">B0015</A> in NEB10β cells was used as negative control during sensor chip induction with IPTG, HSL and <I>P. aeruginosa</I>.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>B0015 in NEB10β was used as a negativ control</B>Induction with A) 0.2 µL of 100 mM IPTG, image taken after 2.5 h; B) 0.2 µL of 500 µg/mL 3-oxo-C<SUB>12</SUB>-HSL, image after 2.5 h; C) with five spots of <I>Pseudomonas aeruginosa</I> liquid culture on the left and one big spot on the right, image taken after 2 h.
</TD></TR></TBODY></TABLE></DIV><H2><SPAN class="mw-headline" id="Achievements">Achievements</SPAN></H2><P>We developed and optimized a 2D biosensor, which is able to detect IPTG, 3-oxo-C<SUB>12</SUB>-HSL and living <I>Pseudomonas aeruginosa</I> cells. 
</P><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:260px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><A href="/File:Aachen_K1319042_Platereader.gif" title="File:Aachen K1319042 Platereader.gif">260px</A></TD></TR><TR><TD><B>Testing K1319042 in our sensor chips</B>Sensor chips based on <A href="http://parts.igem.org/Part:BBa_K1319042" class="external text" rel="nofollow">K1319042</A> were investigated for fluorescence using a plate reader. Blue color indicates the absence of fluorescence, while red color indicates fluorescence. The upper chip was not induced, while the lower chip was induced with IPTG (2.0 µL, 100mM).
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Testing_our_Sensor_Chips_in_a_Plate_Reader"> Testing our Sensor Chips in a Plate Reader </SPAN></H3><P>To establish a prove-of-principle for our sensor chip design, we used our construct <A href="http://parts.igem.org/Part:BBa_K1319042" class="external text" rel="nofollow">K1319042</A>, an IPTG inducible iLOV. <I>E. coli</I> cells carrying the this construct were introduced into sensor chips and fluorescence was measured every 15 minutes after induction with 2 µL of 100 mM IPTG. The results are displayed on the left.
We observed a distinct difference in fluorescence between the non-induced chip (top) and the induced chip (bottom). The middle of the bottom chip started to exhibit a clear and visible fluorescence that increased over time and spread outwards. The top chip, however, also showed an increase in the measured fluorescence over time which was primarily due to the leaky promoter and background fluorescence.
</P><DIV class="figure" style="float:right; margin: 10px; border:0px solid #aaa;width:360px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><A href="/File:Aachen_K131026_Platereader.gif" title="File:Aachen K131026 Platereader.gif">360px</A></TD></TR><TR><TD><B>Testing K131026 in our sensor chips</B>Sensor chips based on <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> were investigated for fluorescence using a plate reader. Blue color indicates the absence of fluorescence, while red color indicates fluorescence. The lower chip was induced with with 3-oxo-C<SUB>12</SUB>-HSL (0.2 µL, 500 µg/mL).
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Detecting_3-oxo-C12-HSL_with_Sensor_Chips"> Detecting 3-oxo-C<SUB>12</SUB>-HSL with Sensor Chips </SPAN></H3><P>In an initial attempt to detect 3-oxo-C<SUB>12</SUB>-HSL, we incorporated the <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> construct generated by the 2008 iGEM Team Calgary in our sensor chips. This construct generates a fluorescent signal based on GFP in presence of 3-oxo-C<SUB>12</SUB>-HSL molecules produced by <I>P. aeruginosa</I> during quorum sensing (Jimenez, Koch, Thompson et al., 2012). First, we tested the construct by direct induction with 3-oxo-C<SUB>12</SUB>-HSL (0.2 µL, 500 µg/mL). The fluorescence measurement was taken every 15 minutes with an excitation wavelength of 496 nm and an emission wavelength of 516 nm. The results of this test are shown on the right.
</P><P>A distinct fluorescence signal was observed on the induced chip (bottom) compared to the non-induced chip (top). 
Fluorescence started in the middle of the chip, the point of induction, and then extended outwards while growing in intesnity. The basal level of fluorescence was attributed to leakiness of the promoter and general background fluorescence of growing <I>E. coli</I> cells. In the induced chip (bottom), the background fluorescence was lower than in the non-induced chip (top) because the signal masked the noise. The difference between the induced and non-induced chips indicates a clear response to the HSL and proofed the ability of our 2D sensor chip design to detect HSLs produced by <I>Pseudomonas aeruginosa</I>.
</P><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:480px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><A href="/File:Aachen_I746909_slower_reduced.gif" title="File:Aachen I746909 slower reduced.gif">480px</A></TD></TR><TR><TD><B>IPTG-inducible superfolder GFP (I746909) in sensor chips</B>Expression of superfolder GFP (<A href="http://parts.igem.org/Part:BBa_I746909" class="external text" rel="nofollow">I746909</A>) was induced by the addition of IPTG (2 µL, 100mM) on the right chip. The left chip was not induced.
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Detecting_IPTG_with_Sensor_Chips"> Detecting IPTG with Sensor Chips </SPAN></H3><P>The clip displayed on the left side shows the construct <A href="http://parts.igem.org/Part:BBa_I746909" class="external text" rel="nofollow">I746909</A> from the 2007 iGEM Team Cambridge which expresses super folder GFP under the control of a T7 promoter in combination with our 2D sensor chip technology. The <A href="http://parts.igem.org/Part:BBa_I746909" class="external text" rel="nofollow">I746909</A> construct was introduced into BL21(DE3) cells to make the expression IPTG-inducible since the genome of BL21(De3) contains the T7 RNA Polymerase under the control of a lacI promoter.
</P><P>While the left chip does not show visible fluorescence, the right chip exhibits a strong fluorescence signal. This proves the ability of our sensor chip technology to detect IPTG. The fluorescence response is also high enough to be detected and analyzed by our measurement device <I>WatsOn</I>.
</P><DIV class="figure" style="float:right; margin: 10px; border:0px solid #aaa;width:480px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><A href="/File:Aachen_K131026_HSLdetection_slow.gif" title="File:Aachen K131026 HSLdetection slow.gif">480px</A></TD></TR><TR><TD><B>Detection of 3-oxo-C<SUB>12</SUB>-HSL with K131026</B>0.2 µL of 3-oxo-C<SUB>12</SUB>-HSL was placed in the middle of a sensor chip based on <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> followed by incubation at 37°C in <I>WatsOn</I>.
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Detecting_the_3-oxo-C12-HSL_with_K131026_in_our_Sensor_Chips_using_WatsOn">Detecting the 3-oxo-C<SUB>12</SUB>-HSL with K131026 in our Sensor Chips using <I>WatsOn</I></SPAN></H3><P>The next step towards the final goal of detecting living cells of <I>Pseudomonas aeruginosa</I> was to reproduce the detection of 3-oxo-C<SUB>12</SUB>-HSL, already established in the plate reader, in our own <A href="https://2014.igem.org/Team:Aachen/Project/Measurement_Device" class="external text" rel="nofollow"><I>WatsOn</I></A> device. Therefore, we again used <I>E. coli</I> BL21(DE3) cells carrying <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> and induced with 0.2 µL 3-oxo-C<SUB>12</SUB>-HSL of a concentration of 500 µg/mL. In the clip displayed on the left, the right chip was induced and - as a negative control - the left chip was not induced. Pictures were taken every four minutes.
</P><P>The result was a clear replication of the success of the plate reader experiment. The induced chip showed a clear fluorescence response eminating from the center, where the induction with HSL took place. This demonstrated the ability of not only our sensor chip technology but also our measurement device <I>WatsOn to successfully</I> detect 3-oxo-C<SUB>12</SUB>-HSL.
</P><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:480px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><A href="/File:Aachen_K131026_Pseudomonas_aeruginosa_detection.gif" title="File:Aachen K131026 Pseudomonas aeruginosa detection.gif">480px</A></TD></TR><TR><TD><B>Detection of <I>Pseudomonas aeruginosa</I> with K131026</B>Direct detection of <I>Pseudomonas aeruginosa</I> on sensor chips based on <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A>.
</TD></TR></TBODY></TABLE></DIV><H3><SPAN class="mw-headline" id="Detecting_Pseudomonas.C2.A0aeruginosa_with_K131026_in_our_Sensor_Chip_with_WatsOn">Detecting <I>Pseudomonas aeruginosa</I> with K131026 in our Sensor Chip with <I>WatsOn</I></SPAN></H3><P>After establishing the successful detection of 3-oxo-C<SUB>12</SUB>-HSLs with our sensor chips the next step was to detect living cells of <I>Pseudomonas aeruginosa</I> with our measurement device <I>WatsOn</I>. Therefore sensor chips based on <A href="http://parts.igem.org/Part:BBa_K131026" class="external text" rel="nofollow">K131026</A> were prepared and the right chip was induced with 0.2 µL of a <I>Pseudomonas aeruginosa</I> culture while the left chip was not induced (Detection of 3-oxo-C12 HSL with K131026, displayed below). On the induced chip, a clear fluorescence signal was visible in response to <I>P. aeruginosa</I>. The fluorescence signal emerged outward from the induction point and showed a significant difference to the non-induced chip. The results clearly demonstrate the ability of our sensor chip technology and our measurement device <I>WatsOn</I> to detect <I>P. aeruginosa</I>!
</P><H2><SPAN class="mw-headline" id="Outlook"> Outlook </SPAN></H2><P>We are <B>committed to improve</B> our sensor chip platform. The current technique of using a simple agarose chip is not sufficient to collect all microorganisms from the sampled surface. Therefore, the aim is to improve the sampling chip by using a different, more adhesive material. 
</P><P>Furthermore, diffusion in the sensor chips will be reduced to <B>limit the spread of the fluorescence signal</B>. Currently, the fluorescence spot grows beyond the point of induction and makes it difficult to differentitate between multiple points of induction. By introducing diffusion barriers into our chips, the growth of the fluorescence spots might be reduced, thus enabling the detection of multiple sources of fluorescence lying close together. 
</P><P>Additionally, the application of our sensor chips in combination with our <I>WatsOn</I> device is currently being evaluated for the detection of signals other than fluorescence. <B>Detecting bio- and chemiluminescence</B> has been identified and will be investigated as an area of potential future application. 
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Waters, C. M., &amp; Bassler, B. L. (2005). QUORUM SENSING: Cell-to-Cell Communication In Bacteria. Annual Review of Cell and Developmental Biology, 21(1), 319-346. Available online at <A href="http://www.annualreviews.org/doi/full/10.1146/annurev.cellbio.21.012704.131001" class="external free" rel="nofollow">http://www.annualreviews.org/doi/full/10.1146/annurev.cellbio.21.012704.131001</A>.
</LI></UL><UL><LI> Jimenez, P. N., Koch, G., Thompson, J. A., Xavier, K. B., Cool, R. H., &amp; Quax, W. J. (2012). The Multiple Signaling Systems Regulating Virulence in <I>Pseudomonas aeruginosa</I>. Microbiology and Molecular Biology Reviews, 76(1), 46-65. Available online at <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294424/#B63" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294424/#B63</A>.
</LI></UL></DIV><DIV class="footerDiv" align="center" style="margin-bottom:-10px; background:#0069b3;"><CENTER><A class="footerCenter" href="https://2014.igem.org/Team:Aachen/Contact">Contact</A><A class="footerCenter" href="https://2014.igem.org/Team:Aachen/Disclaimer">Disclaimer</A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Aachen/Project/2D_Biosensor">http://2014.igem.org/Team:Aachen/Project/2D_Biosensor</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aachen/Project/2D_Biosensor" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aachen/Project/2D_Biosensor" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aachen/Project/2D_Biosensor&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aachen/Project/2D_Biosensor&amp;oldid=399573" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>