<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Aachen_Project_FRET_Reporter"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Aachen/Project/FRET_Reporter">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Aachen/Project/FRET_Reporter&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Aachen/Project/FRET_Reporter&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Aachen/Project/FRET_Reporter&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Aachen/Project/FRET_Reporter" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Aachen/Project/FRET Reporter</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="navDiv" style="top:14px; position:fixed; z-index: 1; border-bottom-style: solid; border-bottom-width: 3px; border-bottom-color: #bf1521;"><NAV style="width: 1250px;"><UL align="left" style=""><LI style="width:129px"><A href="https://2014.igem.org/Team:Aachen/Project">Project</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor">2D Biosensor</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter">The REACh Construct</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/Model">Modeling</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Project/Measurement_Device"><I>WatsOn</I></A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/Project/Gal3">Galectin-3</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/OD/F device">OD/F Device</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Parts">Parts</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Achievements">Achievements</A></LI></UL><LI style="width:216px"><A href="https://2014.igem.org/Team:Aachen/Interlab_Study">Interlab Study</A></LI><LI style="width:250px"><A href="https://2014.igem.org/Team:Aachen/PolicyPractices">Policy &amp; Practices</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Kaiser-Karls-Gymnasium">Kaiser-Karls-Gymnasium</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Neanderlab">NEAnderLab</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/Collaborations/MakerFaire">MakerFaire</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/PolicyPractices/Economics">Economical View</A></LI><LI style="border-bottom:2px solid; border-bottom-color: white;"><A href="https://2014.igem.org/Team:Aachen/PolicyPractices/BioBrickIntellectualProperty">BBa Intellectual Property</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Blog">Blog</A></LI><LI style="width:215px"><A href="https://2014.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Meetup">Aachen Meetup</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Bielefeld">Bielefeld</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Methane_Sensor">Braunschweig</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Freiburg">Freiburg</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Collaborations/Heidelberg">Heidelberg</A></LI><LI style="width:157px"><A href="https://2014.igem.org/Team:Aachen/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Wetlab">Wetlab</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Protocols">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Engineering">Engineering</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Software">Software</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/LabEquipment">Equipment</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Notebook/Index">Abbreviations</A></LI><LI style="width:109px"><A href="https://2014.igem.org/Team:Aachen/Attributions">Team</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#members">Members</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#instructors">Instructors</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#partners">Partners</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/Attributions#support">Attributions</A></LI><LI><A href="https://2014.igem.org/Team:Aachen/SecretHappenings"><I>Mr Holmes</I></A></LI></NAV></DIV><DIV id="contentdiv" style="margin-left: auto; margin-right: auto; width: 100%; position: absolute; bottom: 15px; top: 60px;"><TABLE class="transbg" width="100%" cellspacing="0"><TBODY><TR><TD class="contentcell" width="95%" align="justify"><H1><SPAN class="mw-headline" id="The_modified_REACh_Construct">The modified REACh Construct</SPAN></H1><P>On this page, we present our biosensor on a molecular level. Here you can explore the different parts of our genetic device:
</P><P><CENTER><UL class="menusmall-grid" style="width:inherit;"><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#fluorescence" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:32%; line-height:1.5em;">A Faster Answer</DIV></DIV></A></LI><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#fret" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:32%; line-height:1.5em;">The FRET System</DIV></DIV></A></LI><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#darkquencher" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:32%; line-height:1.5em;">REACh Quenchers</DIV></DIV></A></LI><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#gfp-reach" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:32%; line-height:1.5em;">The Fusion Protein</DIV></DIV></A></LI></UL></CENTER></P><P><CENTER><UL class="menusmall-grid" style="width:516px;"><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#tevprotease" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:32%; line-height:1.5em;">TEV Protease</DIV></DIV></A></LI><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#reachachievements" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:32%; line-height:1.5em;">Achieve-ments</DIV></DIV></A></LI><LI style="width: 156px;margin-left: 8px;margin-right: 8px;"><A class="menulink" href="https://2014.igem.org/Team:Aachen/Project/FRET_Reporter#reachoutlook" style="color:black"><DIV class="menusmall-item menusmall-info"><DIV class="menukachel" style="top:40%;">Outlook</DIV></DIV></A></LI></UL></CENTER></P></TD></TR></TBODY></TABLE><TABLE class="transbg" width="100%" cellspacing="0"><TBODY><TR><TD class="contentcell" width="95%" align="center"><H1><SPAN class="mw-headline" id="A_Fluorescence_Answer_Faster_than_Expression"> A Fluorescence Answer Faster than Expression </SPAN></H1><P>Biosensor systems are usually based on a reporter gene under the control of a promoter directly induced by the biomolecule of interest. In the case of our 2D biosensor for <I>Pseudomonas aeruginosa</I>, the expression of our fluorescing reporter protein is directly induced by the activity of the bacterium's quorum sensing molecules. However, transcription, translation, folding and post-translational modifications take their time. Since our goal is to detect the pathogen as fast as possible, we aimed to develop a system that provides a fluorescent answer faster than just expressing the fluorescent protein GFP.
</P></TD></TR><TR><TD><B>Schematic model of a traditional biosensor</B>In this model, the expression of GFP is directly controlled by a promoter whose activator binds to a molecule secreted by the pathogen.
</TD></TR></TBODY></TABLE></DIV><P>As an alternative to the traditional approach, we <B>constitutively express our reporter gene in a quenched form</B>. By fusing GFP to REACh, a mutated variant of EYFP, its original fluorescence is thoroughly suppressed. Thereby, a huge stock of temporary inactivated fluorescence proteins is already available prior to first contact with the pathogen. As a conseqence to the uptake of homoserine lactones of <I>P. aeruginosa</I> and binding to the LasI promoter in front of the protease gene, these <B>autoinducers activate the expression of the TEV protease</B>. This way, our biosensor provides a fast increase in fluorescence intensity by cleavage of GFP-REACh-constructs.
</P><CENTER><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>Schematic model of our novel biosensor</B>Expression of the TEV protease is induced by HSL. The protease cleaves the GFP-REACh fusion protein to elicit a fluorescence response.
</TD></TR></TBODY></TABLE></DIV></CENTER><P><B>Advantages of the novel biosensor:</B></P><UL><LI> When <I>P. aeruginosa</I> is detected by our cells, the reporter protein has already been expressed and only waits to be activated. The cleavage reaction catalyzed by the TEV protease is a faster process than expression and correct folding of GFP. Therefore, our sensor cells provide an exceptionally <B>early response</B>.
</LI></UL><UL><LI> While a certain concentration of homoserine lactone will produce the same number of gene read-outs and the TEV protease is about the same size as GFP (~27 kDa), one TEV protease can cleave many GFP-REACh constructs per time than fluorescent protein molecules could be expressed and go through fluorescence maturation. Through this additional reaction, we introduce an <B>amplification step</B> into our system. Hypothetically, the TEV protease will enable the production of <B>a much stronger signal</B> in a given time interval.
</LI></UL><P>In parallel, we worked on a <A href="https://2014.igem.org/Team:Aachen/Project/Model" class="external text" rel="nofollow">model</A> of the respective biosensor system, in order to confirm our hypothesis of a faster and stronger fluorescence response than a conventional biosensor would povide.
</P><H1><SPAN class="mw-headline" id="The_FRET_.28F.C3.B6rster_Resonance_Energy_Transfer.29_System">The FRET (Förster Resonance Energy Transfer) System</SPAN></H1><P>Förster resonance energy transfer (FRET), sometimes also called  fluorescence resonance energy transfer, is a physical process of energy transfer. In FRET, the energy of a donor chromophore, whose electrons are in an excited state, is passed to a second chromophore, the acceptor. The <B>energy is transferred without radiation</B> and is therefore not exchanged via emission and absorption of photons. The acceptor then releases the energy received from the donor, for example, as light of a longer wavelength. 
</P><P>In biochemistry and cell biology fluorescent dyes, which interact via FRET, are applied as &quot;optical nano metering rules&quot;, because the intensity of the transfer is dependent on the spacing between donor and acceptor. FRET can be observed over distances of up to 10 nm. This way, protein-protein interactions and conformational changes of a variety of tagged biomolecules can be observed. For an efficient FRET to occur, there must be a substantial overlap between the donor fluorescence emission spectrum and the acceptor fluorescence excitation (or absorption) spectrum (<A href="http://www.nature.com/nprot/journal/v8/n2/full/nprot.2012.147.html" class="external text" rel="nofollow">Broussard et al., 2013</A>), as described in the figure below.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>FRET betweeen donor and acceptor</B>On the left: Jablonski diagram showing the transfer of energy between donor and acceptor; on the right: For successful FRET, the emission spectrum of of the donor has to overlap with the absorption spectrum of the acceptor.
</TD></TR></TBODY></TABLE></DIV><P>However, <B>fluorescence is not an essential requirement for FRET</B>. This type of energy transfer can also be observed between donors that are capable of other forms of radiation, such as phosphorescence, bioluminescence or chemiluminescence. Acceptor chromophores do not necessarily emit the energy in form of light, and can lead to quenching instead. Thus, this kind of acceptors are also referred to as <B>dark quenchers</B>. In our project, we use a FRET system with a dark quencher, namely our <B>REACh construct</B>.
</P><H1><SPAN class="mw-headline" id="REACh_Proteins_-_Dark_Quenchers_of_GFP">REACh Proteins - Dark Quenchers of GFP</SPAN></H1><CENTER><DIV class="figure" style="float:left; margin: 10px; border:0px solid #aaa;width:256px;padding:2px 30px 2px 30px;"><TABLE><TBODY><TR><TD><A href="/File:Aachen_K1319000.gif" title="File:Aachen K1319000.gif">256px</A></TD></TR><TR><TD><B>Homology model of REACh.</B>This homology model was created with SWISS-MODEL. We used Chimera to prepare and export a scene that was then rendered into an animation with POV-Ray.
</TD></TR></TBODY></TABLE></DIV></CENTER><P>In 2006, <A href="http://www.pnas.org/content/103/11/4089.full" class="external text" rel="nofollow">Ganesan et al.</A> were the first to present a previously undescribed FRET acceptor, a non-fluorescent yellow fluorescent protein (YFP) mutant called <B>REACh (Resonance Energy-Accepting Chromoprotein)</B>. YFP can be used as a FRET acceptor in combination with GFP as the donor in FRET microscopy and miscellaneous assays in molecular biology. The ideal FRET couple should possess a large spectral overlap between donor emission and acceptor absorption - as illustrated in previous section - but have separated emission spectra to allow their selective imaging.
</P><P>To optimize the spectral overlap of this FRET pair, the group obtained <B>a genetically modified YFP acceptor</B>. Mutations of amino acid residues that stabilize the excited state of the chromophore in enhanced YFP (EYFP) resulted in a non-fluorescent chromoprotein. Two mutations, H148V and Y145W, reduced the fluorescence emission by 82 and 98%, respectively. Ganesan et al. chose the Y145W mutant and the Y145W/H148V double mutant as FRET acceptors, and named them REACh1 and REACh2, respectively. <B>Both REACh1 and REACh2 act as dark quenchers of GFP</B>.
</P><H1><SPAN class="mw-headline" id="Producing_a_GFP-REACh_Fusion_Protein">Producing a GFP-REACh Fusion Protein</SPAN></H1><P>In our project, we reproduced the REACh1 and REACh2 proteins by subjecting an RFC-25 compatible version of the BioBrick <A href="http://parts.igem.org/Part:BBa_E0030" class="external text" rel="nofollow">E0030</A> (EYFP) to a <B>QuikChange mutation</B>, creating the BioBricks <A href="http://parts.igem.org/Part:BBa_K1319001" class="external text" rel="nofollow">K1319001</A> and <A href="http://parts.igem.org/Part:BBa_K1319002" class="external text" rel="nofollow">K1319002</A>. Subsequently, we fused each REACh protein with <B>GFP (mut3b)</B> which is available as BioBrick <A href="http://parts.igem.org/Part:BBa_E0040" class="external text" rel="nofollow">E0040</A>. The protein complex was linked via a <B>protease cleavage site</B> (<A href="http://parts.igem.org/Part:BBa_K1319016" class="external text" rel="nofollow">K1319016</A>). As constitutive promoter we use <A href="http://parts.igem.org/Part:BBa_J23101" class="external text" rel="nofollow">J232101</A>. When GFP is connected to either REACh quencher, GFP will absorb light but the energy will be transferred to REACh via FRET and is then emitted in the form of heat; the fluorescence is quenched. Our cells also constitutively express the <B><A href="http://parts.igem.org/Part:BBa_C0179" class="external text" rel="nofollow">LasR</A> activator</B>. Together with the HSL molecules from <I>P. aeruginosa</I>, LasR binds to the <B><A href="http://parts.igem.org/Part:BBa_J64010" class="external text" rel="nofollow">LasI</A> promoter</B> that controls the expression of the TEV protease, which we made available as <A href="http://parts.igem.org/Part:BBa_K1319004" class="external text" rel="nofollow">K1319004</A>. When the fusion protein is cleaved by the TEV protease, REACh will be separated from GFP. The latter will then be able to absorb and emit light as usual.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>Composition of our biosensor</B>For our biosensor, we use a mix of already available and self-constructed BioBricks.
</TD></TR></TBODY></TABLE></DIV><P>The resulting fusion proteins can be expressed in <A href="http://parts.igem.org/Part:BBa_K1319013" class="external text" rel="nofollow">K1319013</A> (GFP fused with REACh1) and <A href="http://parts.igem.org/Part:BBa_K1319014" class="external text" rel="nofollow">K1319014</A> (GFP fused with REACh2). The linker between the proteins containing a TEV protease cleavage site is labelled as <A href="http://parts.igem.org/Part:BBa_K1319016" class="external text" rel="nofollow">K1319016</A>.
</P><P>Since we did not have enough time to build the complete system we tested our FRET reporter with an IPTG inducible TEV protease (<A href="http://parts.igem.org/Part:BBa_K1319008" class="external text" rel="nofollow">K1319008</A>). In this way we were able to establish a proof of concept of our reporter system including proper expression of the TEV protease as well as functionality of our GFP-REACh construct.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:900px;padding:0px;"><TABLE><TBODY><TR><TD><B>Composition of our biosensor for IPTG</B>To test the funtionality of our sensor concept, we expressed the TEV protease with a T7 promoter.
</TD></TR></TBODY></TABLE></DIV><H1><SPAN class="mw-headline" id="Cutting_the_Fusion_Protein_with_the_TEV_Protease">Cutting the Fusion Protein with the TEV Protease</SPAN></H1><P>To cleave the GFP-REACh fusion protein, we chose <B>Tobacco Etch Virus (TEV) protease</B>, a highly sequence-specific cysteine protease, that is frequently used for the controlled cleavage of fusion proteins <I>in vitro</I> and <I>in vivo</I>. The native protease also contains an internal self-cleavage site. This site is slowly cleaved to inactivate the enzyme. The physiological reason for the self-cleavage is unknown, however, undesired for our use. Therefore, our team uses a variant of the native TEV protease containing the mutation S219V which results in an alteration of the cleavage site so that self-inactivation is diminished.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:800px;padding:0px;"><TABLE><TBODY><TR><TD><B>TEV protease with a bound peptide</B>This picture shows the TEV protease with a peptide chain bound in the binding pocket ready to be cleaved. The recognition site of the bound peptide chain is located inside the binding pocket of the TEV protease. It was rendered with POV-Ray.
</TD></TR></TBODY></TABLE></DIV><P>Though quite popular in molecular biology, the TEV protease is not avaiable as a BioBrick yet. Hence, the team Aachen introduces the protease with anti-self cleavage mutation S219V and codon optimized for <I>E. coli</I><A href="http://parts.igem.org/Part:BBa_K1319004" class="external text" rel="nofollow"><B>to the Parts Registry this year.</B></A></P><H2><SPAN class="mw-headline" id="Achievements"> Achievements </SPAN></H2><H3><SPAN class="mw-headline" id="Characterization_of_GFP-REACh1_and_GFP-REACh2_fusion_proteins_in_combination_with_an_IPTG-inducible_TEV_Protease">Characterization of GFP-REACh1 and GFP-REACh2 fusion proteins in combination with an IPTG-inducible TEV Protease</SPAN></H3><P>The characterization of the TEV protease and the REACh1 and REACh2 dark quenchers was performed by introducing both simultaneously into <I>E. coli</I> Bl21 (DE3). The resulting double plasmid cells therefore contained <A href="http://parts.igem.org/Part:BBa_K1319013" class="external text" rel="nofollow">K1319013</A> (GFP-REACh1 fusion protein) or<A href="http://parts.igem.org/Part:BBa_K1319014" class="external text" rel="nofollow">K1319014</A> (GFP-REACh2 fusion protein) and <A href="http://parts.igem.org/Part:BBa_K1319008" class="external text" rel="nofollow">K1319008</A> (IPTG-inducible TEV protease). K1319013 and K1319014 were located on a pSB3K3 plasmid backbone and K1319008 on a pSB1C3 backbone, two standard iGEM plasmids with different oris allowing simultaneous use in one cell. 
</P><P><A href="http://parts.igem.org/Part:BBa_I20260" class="external text" rel="nofollow">I20260</A> was used as a positive control because I20260 contains the same promoter (<A href="http://parts.igem.org/Part:BBa_J23101" class="external text" rel="nofollow">J23101</A>), the same RBS (<A href="http://parts.igem.org/Part:BBa_B0032" class="external text" rel="nofollow">B0032</A>) and the same version of GFP (<A href="http://parts.igem.org/Part:BBa_E0040" class="external text" rel="nofollow">E0040</A>) and is located on the same plasmid backbone pSB3K3. Therefore, it is expected that when all fusion proteins are successfully cut by the TEV protease, the fluorescence level of the double plasmid constructs reaches the same level as the positive control of I20260. As a negative control <A href="http://parts.igem.org/Part:BBa_B0015" class="external text" rel="nofollow">B0015</A> was used, a coding sequence of a terminator which should not show any sign of fluorescence.
</P><P>To characterize our REACh1/2 constructs in combination with the TEV protease a growth experiment was conducted. Both of the double plasmid constructs, a constitutive expression of GFP (I20260) as positive control and B0015 as negative control were compared. For each expression IPTG-induced and non-induced cultures were grown in parallel. All measurements were done in a biological triplicate. 
</P><P>To better evaluate the fluorescence, the observed Optical desity (OD) was taken into account in order to achieve a fluorescence measurement independent of the amount of cells present. This way, the measurement represents the amount of fluorescence per cell only. 
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:700px;padding:0px;"><TABLE><TBODY><TR><TD><B>Comparison of K1319013 + K1319008, K1319014 + K1319008, I20260 (positive control) and B0015 (negative control)</B>Both double plasmid construct exhibit a clear fluorescence signal when induced.
</TD></TR></TBODY></TABLE></DIV><P>The negative control B0015 did not exhibit any significant fluorescence. The positive control I20260 showed a steady level of fluorescence as expected due to the constitutive expression of GFP. As expected, the production is also independent of addition of IPTG therefore the triplicates have been merged together.
</P><P>Both double plasmid constructs K1319013 + K1319008 and K1319014 + K1319008 did not exhibit a strong fluorescence before induction with IPTG. In the non-induced state, the fluorescence stays low and does not increase over time. It is significantly weaker than the fluorescence reached by the induced constructs or the positive control but was higher than the negative control. The higher base level of fluorescence in the not induced constructs is due to the imperfect quenching and the leakiness of the IPTG inducible promoter. 
</P><P>The induced double plasmid constructs exhibited a fast rise in fluorescence after induction. <B>The signal strenght increased  ~ 10-fold over the non-induced constructs.</B> K1319014 + K1319008 reached the the same level of fluorescence as I20260, indicating a complete cleavage of the fusion proteins by the TEV protease. K1319013 + K1319008 did not reach a level of fluorescence as high as K1319013 + K1319008, however, the nearly 10-fold increase in fluorescence after induction is a clear indicator for the TEV protease cutting the fusion protein K1319013. The weaker fluorescence signal is probably due to a lower expression level of K1319013 in the cells compared to the expression level of K1319014. 
</P><P>The fluorescence results of K1319014 + K1319008 was used to fit our <A href="https://2014.igem.org/Team:Aachen/Project/Model" class="external text" rel="nofollow">model</A> to experimental data.
</P><H4><SPAN class="mw-headline" id="Summary">Summary</SPAN></H4><P>The double plasmid systems of K1319013 + K1319008 and K1319014 + K1319008 demonstrate the quenching ability of the REACh1 and REACh2 proteins as well as the funcionality of the TEV protease.
The increase in fluorescence after induction with IPTG is based on a functional expression of the TEV protease which proceeds to cut the linker of the fusion protein already produced destroying the FRET system between GFP and its quencher and resulting in a strong fluorescence signal. Combined, this characterization is a <B>validation of the functionality of the REACh1 protein (<A href="http://parts.igem.org/Part:BBa_K1319001" class="external text" rel="nofollow">K1319001</A>), the REACh2 protein (<A href="http://parts.igem.org/Part:BBa_K1319002" class="external text" rel="nofollow">K1319002</A>) and the TEV protease (<A href="http://parts.igem.org/Part:BBa_K1319004" class="external text" rel="nofollow">K1319004</A>)</B>.
</P><H3><SPAN class="mw-headline" id="Determining_the_Quenching_ability_of_REACh1_and_REACh2">Determining the Quenching ability of REACh1 and REACh2</SPAN></H3><P>In order to further evaluate the quenching ability of the REACh1 and REACh2 constructs in the fusion proteins produced by K1319013 and K1319014, they were expressed alone without an IPTG inducible TEV protease. This eliminated the effect of a potential leakiness of the non induced promoter to reliably assess the quenching ability of the REACh1 and REACh2 proteins. 
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:700px;padding:0px;"><TABLE><TBODY><TR><TD><B>Comparison of K1319013 and K1319014 with I20260 and B0015</B>K1319013 and K1319014 show a severely reduced fluorescence compared to the positive control I20260.
</TD></TR></TBODY></TABLE></DIV><P>In the previous experiment it was established that the fusion proteins K1319013 and K1319014 are expressed funtionally. K1319014 reached the same level of fluorescence as the positive control after being cut by the TEV protease. Therefore the reduced fluorescence in this experiment is completely attributable to the quenching of REACh1. The quenching reduces the fluorescence of GFP by a factor ~ 25 which means a <B>quenching efficiency of ~ 96%!</B>. 
</P><P>K139013 was not able to reach the same fluorescence level as the positive control in the previous experiment. Therefore a worse rate of functional expression of K1319013 compared to K1319014 is assumed. incorporating this, the difference in fluorescence between induced and not induced is still a factor of ~ 30 resulting in a  <B>quenching efficiency of ~ 97%</B>.
</P><H4><SPAN class="mw-headline" id="Summary_2">Summary </SPAN></H4><P>The fusion proteins of GFP combined with REACh1 and REACh2 are not only fully functional but exhibit a great quenching efficiency of ~ 97% for REACh1 and ~ 96% for REACh2.Even though REACh1s quenching ability seems to be slighty superior, the expression level of the K1319014 fusion protein including REACh2 is nearly twice as high as the fusion protein K1319013 containing REACh1 and therefore shows a stronger fluorescence.  Ganesan et al. (2006) reported a reduction on emission of GFP of 82% for REACh1 and 98% for REACh2 but with a different Linker between the proteins and on a different vector backbone.
</P><H3><SPAN class="mw-headline" id="Comparing_fluorescence_kinetics_of_the_GFP-REACh_fusion_proteins_with_a_Standard_lacI-inducible_GFP_Expression">Comparing fluorescence kinetics of the GFP-REACh fusion proteins with a Standard lacI-inducible GFP Expression</SPAN></H3><P>To assess the kinetics of the fusion proteins K1319013 (GFP-REACh1) and K1319014 (GFP-REACh2), the double plasmid systems K1319013 + K1319008 and K1319014 + K1319008 were compared to a standard expression of GFP under the control of a lacI promoter in <A href="http://parts.igem.org/Part:BBa_K731520" class="external text" rel="nofollow">K731520</A>, a BioBrick made by the iGEM Team TRENTO in 2012. This tested the hypothesis of achieving a faster fluorescence response with the GFP-REACh fusion proteins compared to a standard expression.
</P><P>K731520 and the double plasmid constructs K1319013 + K1319008 and K1319014 + K1319008 were cultivated in <I>E. coli</I> BL21(DE3), and fluorescence and OD was measured.The fluorescence was adjusted for the OD to show a relative fluorescence on a per cell basis. The difference between the induced and non-induced state, the fluorescence quotient, serves as a better indicator for a system used as a sensor because the difference between an <I>on</I> and <I>off</I> state is more important for a clear and unmistakable signal than the overall fluorescence. Hence, the OD-adjusted fluorescence quotient for both double plasmid constructs and K731520 was obtained and plotted in the following graph.
</P><DIV class="figure" style="float:center; margin: 10px 10px 10px 40px; border:0px solid #aaa;width:700px;padding:0px;"><TABLE><TBODY><TR><TD>Fluorescence was normalized to the cell growth. The fluorescence of induced cells was additionally divided by the fluorescence of non-induced cells to obtain the fluorescence quotient.
</TD></TR></TBODY></TABLE></DIV><P>The graph clearly shows the faster response of the cut GFP-REACh fusion protein compared to a standard GFP expression. Both fluorescence signals of the double plasmid constructs achieve a higher difference in fluorescence signal between induced and non-induced state as well as at a faster rate. This proves the hypothesis made earlier about the kinetics of the GFP-REACh fusion protein combined with the TEV protease.
</P><H4><SPAN class="mw-headline" id="Summary_3">Summary</SPAN></H4><P>The kinetics of the fusion protein combined with the TEV protease exhibits the exact characteristics as predicted. The response is clearly faster than normal expression by accumulating a reservoir of fusion proteins which are not fluorescing due to the dark quencher attached to them. This reservoir is then activated by the induction of the TEV protease expression. Production of the protease results in the cleavage of the fusion protein, releasing GFP from the dark quencher and disturbing the interaction between the FRET pair. This results in the observed faster fluorescence reaction due to the amplificating effect of the TEV protease in which every one TEV protease can account for many fluorescence proteins being activated.
</P><H3><SPAN class="mw-headline" id="Characterizing_the_GFP-REACh_Constructs_in_Sensor_Chips">Characterizing the GFP-REACh Constructs in Sensor Chips</SPAN></H3><P>To further characterize the REACh construct, they were introduced into the sensor cells which were then induced with 2 µL IPTG with a concentration of 100 mM. Subsequently, we took fluorescence measurement read-outs (GFP, excitation 496 ± 9 nm, emission 516 ± 9 nm) roughly every 10 min in the plate reader. The results were plotted in the heatmap shown here. 
</P><CENTER><DIV class="figure" style="float:center; margin: 0px 10px 10px 40px; border:0px solid #aaa;width:500px;padding:10px 10px 0px 0px;"><TABLE><TBODY><TR><TD><B>K1319014 + K1319008 and K1319013 + K1319008 non-induced (top) and induced (bottom) in sensor cells </B>The induced double plasmid systems K1319013 + K1319008 and K1319014 + K1319008 exhibit a clear fluorescence response in our sensor cells which in response to induction with 2 µl IPTG.
</TD></TR></TBODY></TABLE></DIV></CENTER><P>The heatmap shows an increase of fluorescence from blue (no fluorescence) to red (high fluorescence). It is clearly visible that the induced chips are exhibiting a significantly higher fluorescence than the non-induced chips. This shows that the constructs also work  as intended in the sensor chips: The TEV protease cuts the linker so that the fusion protein is separated into GFP and a dark quencher, disabling the quenching. GFP has a clear fluorescence emission after the fusion protein has been successfully cut into two pieces by the TEV protease.
</P><H3><SPAN class="mw-headline" id="Comparing_the_kinetic_of_the_double_plasmid_systems_K1319013_.2B_K1319008_and_K1319014_.2B_K1319008_with_standard_GFP_expression">Comparing the kinetic of the double plasmid systems K1319013 + K1319008 and K1319014 + K1319008 with standard GFP expression</SPAN></H3><CENTER><DIV class="figure" style="float:center; margin: 0px 10px 10px 40px; border:0px solid #aaa;width:480px;padding:10px 10px 0px 0px;"><TABLE><TBODY><TR><TD><B>K1319013 + K1319008, K1319014 + K1319008 and K731520 in an non-induced (top) and induced (bottom) chip</B>Comparing the factor of fluorescence increase of induced (bottom) and not induced (top) sensor chips to the first frame of the measurement. The tested constructs are K1319013 + K1319008, K1319014 + K1319008 and K731520. After five hours, the fluorescence in our dual-plasmid REACh construct contaning K1319014 had increased by a factor of about 8, while K731520 fluorescence increased by only a factor of 2-3.
</TD></TR></TBODY></TABLE></DIV></CENTER><P>The analysis of the different <A href="https://2014.igem.org/Team:Aachen/Project/2D_Biosensor" class="external text" rel="nofollow">sensor chips</A> with the three different construct K1319013 + K1319008, K1319014 + K1319008 and K731520 demonstrates the same fluorescence response kinetic as in the shake flask experiments. The double plasmid systems exhibits a faster and stronger fluorescence response compared to a standard GFP expression in K731520. The build up pool of fusion proteins allows for a faster, stronger fluorescence response when the induced TEV protease cleaves the fusion proteins and releases GFP from its dark quencher. 
</P><H2><SPAN class="mw-headline" id="Outlook"> Outlook </SPAN></H2><P>The system of the GFP-REACh fusion proteins with an inducible TEV protease has been established and shows the desired results of being faster than standard expression. The next step will be to devise an expression of the TEV protease inducible by HSL instead of IPTG and then to incorporate both the HSL inducible TEV protease and the fusion protein into one plasmid backbone. This would also allow us to choose a high copy plasmid for both inserts, instead of a high copy plasmid for the TEV protease and a low to mid copy plasmid for the fusion protein which should yield an overall higher fluorescence readout.
</P><P>Subsequently the combined construct will be characterizes the same way as the double plasmid system and other options regarding different fluorescence proteins with different quenchers will be considered to be able to have multiple fluorescence responses readable at the same time while still being faster than normal expression. 
</P><P>Also finding and testing different promoters which can be used to express the TEV protease is planned to be able to detect not only <I>Pseudomonas aeruginosa</I> but also other pathogens. Also the technology can be expanded to other molecules enabling other research areas to profit from the faster fluorescence response.
</P><H1><SPAN class="mw-headline" id="References"> References </SPAN></H1><UL><LI> Sundar Ganesan, Simon M. Ameer-Beg, Tony T. C. Ng, Borivoj Vojnovic and Fred S. Wouters. &quot;A dark yellow fluorescent protein (YFP)-based Resonance Energy-Accepting Chromoprotein (REACh) for Förster resonance energy transfer with GFP&quot; Proceedings of the National Academy of Science of the United States of America | March 14,2006 | vol. 103 | no. 11 | 4089 - 4094
</LI></UL><UL><LI> Broussard, Joshua A, Benjamin Rappaz, Donna J Webb, and Claire M Brown. &quot;Fluorescence resonance energy transfer microscopy as demonstrated by measuring the activation of the serine/threonine kinase Akt.&quot; Nature protocols 8.2 (2013): 265-281. doi:10.1038/nprot.2012.147. 
</LI></UL><UL><LI> Broussard, J. A., Rappaz, B., Webb, D. J., &amp; Brown, C. M. (2013). Fluorescence resonance energy transfer microscopy as demonstrated by measuring the activation of the serine/threonine kinase Akt. Nature protocols, 8(2), 265-281. doi: 10.1073/pnas.0509922103 
</LI></UL><P><B>SWISS-MODEL</B></P><UL><LI> Arnold, K., Bordoli, L., Kopp, J., &amp; Schwede, T. (2005). The SWISS-MODEL Workspace: A Web-based Environment For Protein Structure Homology Modelling. Bioinformatics, 22(2), 195-201.
</LI></UL><UL><LI> Biasini, M., Bienert, S., Schwede, T., Waterhouse, A., Arnold, K., Studer, G., et al. (2014). Nucleic Acids Research. SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information. doi: 10.1093/nar/gku340.
</LI></UL><UL><LI> Guex, N., Peitsch, M. C., &amp; Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective. Electrophoresis, 30(S1), S162-S173.
</LI></UL><UL><LI> Kiefer, F., Arnold, K., Kunzli, M., Bordoli, L., &amp; Schwede, T. (2009). The SWISS-MODEL Repository and associated resources. Nucleic Acids Research, 37(Database), D387-D392.
</LI></UL><P><B>UCSF Chimera</B></P><UL><LI> Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng, E. C., et al. (2004). UCSF Chimera?A visualization system for exploratory research and analysis. Journal of Computational Chemistry, 25(13), 1605-1612. PubMed PMID: 15264254.
</LI></UL><P><B>POV-Ray</B></P><UL><LI> Persistence of Vision Pty. Ltd. (2004) Persistence of Vision Raytracer (Version 3.7) [Computer software]. Retrieved from <A href="http://www.povray.org/download/" class="external free" rel="nofollow">http://www.povray.org/download/</A></LI></UL></DIV><DIV class="footerDiv" align="center" style="margin-bottom:-10px; background:#0069b3;"><CENTER><A class="footerCenter" href="https://2014.igem.org/Team:Aachen/Contact">Contact</A><A class="footerCenter" href="https://2014.igem.org/Team:Aachen/Disclaimer">Disclaimer</A></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Aachen/Project/FRET_Reporter">http://2014.igem.org/Team:Aachen/Project/FRET_Reporter</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Aachen/Project/FRET_Reporter" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Aachen/Project/FRET_Reporter" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Aachen/Project/FRET_Reporter&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Aachen/Project/FRET_Reporter&amp;oldid=400067" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>