<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Ecuador_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Ecuador/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="ecuador_menu"><A href="https://2018.igem.org/Team:Ecuador"><DIV class="ecuador_home">
			iGEM ECUADOR
		  </DIV></A><DIV class="menu_options_container"><DIV id="project" class="menu_option"><A class="ecuador_href_text" href="https://2018.igem.org/Team:Ecuador/Description">
			PROJECT
			  </A><DIV class="menu_sub_options_container"><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Description"><DIV class="sub_menu_option">
				  DESCRIPTION
				</DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Background"><DIV class="sub_menu_option">
				  BACKGROUND
				</DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Model"><DIV class="sub_menu_option">
				MODELING
			  </DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Design"><DIV class="sub_menu_option">
				  DESIGN
				</DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Experiments"><DIV class="sub_menu_option">
				  EXPERIMENTS
				</DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Notebook"><DIV class="sub_menu_option">
				  NOTEBOOK
				</DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Results"><DIV class="sub_menu_option">
				  RESULTS
				</DIV></A></DIV></DIV><DIV id="parts" class="menu_option"><A class="ecuador_href_text" href="https://2018.igem.org/Team:Ecuador/Parts">
				PARTS
				</A></DIV><DIV id="human_practice" class="menu_option"><A class="ecuador_href_text" href="https://2018.igem.org/Team:Ecuador/Human_Practices">
				HUMAN PRACTICES
			  </A><DIV class="menu_sub_options_container"><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Human_Practices"><DIV class="sub_menu_option">
						HUMAN PRACTICES
				  </DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Public_Engagement"><DIV class="sub_menu_option">
						PUBLIC ENGAGEMENT
				  </DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Best-In-Human-Pract"><DIV class="sub_menu_option" style="font-size: 10px">
						BEST INTEGRADED HUMAN PRACTICES
				  </DIV></A></DIV></DIV><DIV id="safety" class="menu_option"><A class="ecuador_href_text" href="https://2018.igem.org/Team:Ecuador/Safety">
				SAFETY
				</A></DIV><DIV id="interlab" class="menu_option"><A class="ecuador_href_text" href="http://2018.iGEM.org/Team:Ecuador/Interlab">
			  INTERLAB
			</A></DIV><DIV id="judging_forms" class="menu_option"><A class="ecuador_href_text" href="http://2018.iGEM.org/Team:Ecuador/Judging">
			  JUDGING FORMS
			</A></DIV><DIV id="team" class="menu_option"><A class="ecuador_href_text" href="https://2018.igem.org/Team:Ecuador/Team">
			TEAM
			  </A><DIV class="menu_sub_options_container"><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Team"><DIV class="sub_menu_option">
						TEAM MEMBERS
					</DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Collaborations"><DIV class="sub_menu_option" style="font-size: 13.5px">
					COLLABORATIONS
				  </DIV></A></DIV></DIV><DIV id="atributions" class="menu_option"><A class="ecuador_href_text" href="https://2018.igem.org/Team:Ecuador/Attributions">
				  ATTRIBUTIONS
			  </A><DIV class="menu_sub_options_container"><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Attributions"><DIV class="sub_menu_option">
					ATRIBUTTIONS
				  </DIV></A><A class="ecuador_href_sub_text" href="https://2018.igem.org/Team:Ecuador/Sponsoring"><DIV class="sub_menu_option">
					SPONSORING
				  </DIV></A></DIV></DIV></DIV></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><TITLE>C-lastin, Interlab</TITLE><DIV class="ec--modeling--container"><DIV class="ec--first--section"><DIV class="ec--main--title"><H1>DESIGN</H1></DIV><DIV class="ec--modeling--items"><H3><A href="#INT" style="color:black; text-decoration: none">Index</A></H3><UL class="circle"><LI><A href="#l1" style="text-decoration: none;color: #000000">Introduction</A></LI><LI><A href="#l2" style="text-decoration: none;color: #000000">Resorbable Bacterial cellulose</A></LI><LI><A href="#l3" style="text-decoration: none;color: #000000">Fusion proteins</A></LI><UL><LI><A href="#l5" style="color:black; text-decoration: none">CBD-BMP2</A></LI><LI><A href="#l8" style="color:black; text-decoration: none">CBD cipA-sfGFP</A></LI><LI><A href="#l6" style="color:black; text-decoration: none">CBD-ELP_C5</A></LI></UL><LI><A href="#l7" style="text-decoration: none;color: #000000">Characteristics of C-lastin</A></LI></UL></DIV><DIV class="ec--nau--content"><DIV class="ec--h2">INTRODUCTION</DIV><DIV class="ec--p">
				 The main goal of our work is to develop a functionalized cellulose-based biomaterial for bone and cartilage regeneration in order to allow a faster injuries recovery time and overcome remaining challenges in repairing these connective tissues. In biomedicine, emergent strategies have focused the use biomaterials loaded with drugs.<SUP>1-3</SUP> We aimed to use bacterial cellulose, a biocompatible biomaterial, as scaffold for the delivery of the human bone morphogenetic protein 2 (BMP2). It is worth to note that the diffusion of a drug far of the action site can become ectopic tissue formation, representing a serious problem. Therefore, we focused to attach the BMP2 by the use of a cellulose binding domain (CBD). We choose the CBD of the scaffolding protein of C. thermocellum (CBD cipA) because the higher affinity compared to other CBDs, reported by the Team Imperial_2014. Moreover, in an effort for a longer broad application of a functionalized bacterial cellulose, such as wound healing, we also fused the CBD cipA with an elastin like polypeptide (ELP_C5) derived from the ELP- [V-150].</DIV><DIV class="ec--h2">RESORBABLE BACTERIAL CELLULOSE</DIV><DIV class="ec--BMP--container"><DIV class="ec--BMP--img" style="float: left; display: block; text-indent: inherit; width: 288px; text-align: justify; margin-right: 20px;margin-bottom: 20px"><CENTER><DIV class="ec--p"><STRONG>Figure 1.</STRONG> Surface topography imaged by AFM corresponding to a sample of dried bacterial cellulose. </DIV></CENTER></DIV><DIV class="ec--p">
                                High levels of cellulose is the first module of our project. Thus, we have focused to cotransform two plasmids in E. coli BL21 (DE3) because it is resistant to toxic proteins. The firs plasmid is  the operon bscABCD for cellulose biosynthesis while the second one is conformed by the genes Cmcax and CcpAx which are members of the cellulose synthase complex and allow an enhanced cellulose production rates.<SUP>4,5</SUP> Because the difficulties of chemical synthesis in four of the six genes, it was necessary to add aptamers at 5’ ands 3’ and the Gibson assembly, using psb1c3 as the host vector was delayed.  To date, we have successfully assembled the plasmid containing the genes Cmcax and CcpAx.</DIV><DIV class="ec--p">
                                Furthermore,  we started the atomic force microscopy (AFM) analysis of the bacterial  cellulose, analyzing its topography as the first step to approximate the total  cellulose binding sites where the fusion protein will bind. This analysis will  allow us to know the drug concentration in the cellulose.<SUP>6</SUP></DIV><DIV class="ec--p">
                                The way we are obtaining resorbable bacterial cellulose membranes is using electron beam irradiation.<SUP>7</SUP> This work is being a collaboration with hospitals of our country where there are electron beam accelerators which are used for cancer treatment.
                                
				  </DIV><DIV class="ec--h2">FUSION PROTEINS</DIV><DIV class="ec--BMP--container"><DIV class="ec--h3">CBD cipA-BMP2</DIV><DIV class="ec--BMP--img" style="float: left; display: block; text-indent: inherit; width: 288px; text-align: justify; margin-right: 20px;margin-bottom: 20px"><CENTER><DIV class="ec--p"><STRONG>Figure 2. </STRONG> Tridimensional structure of CBD cipA-BMP2 obtained from the server I-Tasser. </DIV></CENTER></DIV><DIV class="ec--p">
                                Two  gBlocks were synthetized for in-frame protein assembly. The first gBlock  consisting of lacI promoter, RBS and CDS of CBD cipA while the second gBlock  containing the CDS of BMP2. Both of them were codon optimized for <EM>E</EM>. <EM>coli</EM>.</DIV><DIV class="ec--p">
                                Because it has been demonstrated that the  C-terminal domain of BMP2 provides osteogenic activity, <SUP>8</SUP> we intended  to leave it free. Thus, it was necessary to fused it C-terminally to an  N-terminal CBD.<SUP>10</SUP> We decided to choose the CBD cipA because the  higher affinity to bacterial cellulose compared to other CBDs that was reported  by the team Imperial 2014.<SUP>10</SUP></DIV><DIV class="ec--p">
                                The CBD cipA has originally two linkers. The N-terminal linker is a signal sequence typical of the bacterial extracellular proteins while the C-terminal linker separating the CBD cipA from Coh3 module. </DIV><DIV class="ec--p">
                                Then, in contrast to the team Imperial 2014, we decided to use only the C-terminal linker hypothesizing that this can provide a better binding strength than the use of two linkers.</DIV><DIV class="ec--p"><STRONG>Figure 3. </STRONG>Schematic  overview of the CBD cipA-BMP2 construct cloned in psb1c3<STRONG>.</STRONG></DIV><DIV class="ec--p"> </DIV><DIV class="ec--p"> </DIV></DIV><DIV class="ec--BMP--container"><DIV class="ec--h3">CBD cipA-sfGFP</DIV><DIV class="ec--BMP--img" style="float: left; display: block; text-indent: inherit; width: 288px; text-align: justify; margin-right: 20px;margin-bottom: 20px"><CENTER><DIV class="ec--p"><STRONG>Figure 2. </STRONG>Tridimensional structure of CBD cipA-sfGFP  obtained from the server I-Tasser.</DIV></CENTER></DIV><DIV class="ec--p"><DIV class="ec--p">
                                The gBlocks synthetized for in-frame protein assembly corresponded to the first gBlock consisting of lacI promoter, RBS and CDS of CBD cipA; the second gBlock containing the CDS of sfGFP with a 6X His tag. Both of them were codon optimized for <EM>E</EM>. <EM>coli</EM>.</DIV><DIV class="ec--p">
                                We aimed to add a 6X His tag at C-terminus in order to visualize and measure at single-molecule level the interaction between the CBD cipA and bacterial cellulose.<SUP>6</SUP></DIV><DIV class="ec--p"><CENTER><DIV class="ec--p"><STRONG>Figure 5. </STRONG>Schematic overview of the CBD cipA-BMP2 construct cloned in psb1c3. 
					 		</DIV><DIV class="ec--p"> </DIV><DIV class="ec--p"> </DIV></CENTER></DIV></DIV></DIV><DIV class="ec--BMP--container"><DIV class="ecc--h3" style="font-size: 25px"><STRONG>CBD-ELP_C5</STRONG></DIV><DIV class="ec--BMP--img" style="float: left; display: block; text-indent: inherit; width: 288px; text-align: justify; margin-right: 20px;margin-bottom: 20px"><CENTER><DIV class="ec--p"><STRONG>Figure 6</STRONG>. Tridimensional structure of CBD cipA-ELP_C5 obtained from the server  I-Tasser.</DIV></CENTER></DIV><DIV class="ec--p">
                                ELP_C5 was obtained via overhang PCR using primers containing prefix and suffix in RFC25, and the plasmid pET-24a-ELP[V-150] as template.</DIV><DIV class="ec--p">
                                Each ELP undergoes a Tt that leads to its aggregation or solubilization. The Tt of an ELP fusion protein is different to that of the free ELP depending of the surface accessible surface. Because we aimed a future clinical application of ELP fusion proteins in aggregated state, we decided to use ELP[V-150] with a Tt=28.2℃,<SUP>11</SUP> lower than the corporal temperature. However we have cloning a 450 bp fragment of ELP[V-150] and we expect a similar Tt.</DIV><DIV class="ec--p"><CENTER><STRONG> Figure 7. </STRONG>Schematic  overview of the CBD cipA-ELP_C5 construct cloned in psb1c3<STRONG>. </STRONG></CENTER></DIV><DIV class="ec--p"> </DIV><DIV class="ec--p"> </DIV></DIV></DIV><DIV class="ec--h2">CHARACTERISTICS OF C-lastin</DIV><DIV class="ec--p"><UL><LI>In an effort to overcome the  unsolved challenges in bone healing that are mainly derived from pathological  and age related events, C-lastin takes advantage of the biocompatibility of  bacterial cellulose to develop a functionalized biomaterial for bone and cartilage  regeneration. </LI><LI>The principle of C-lastin is a  post-hoc functionalization of a chimeric protein formed by a CBD and the human  BMP2 with bacterial cellulose functionating as scaffold.</LI><LI>The fusion protein containing the  ELP_C5 attached to bacterial cellulose via CBD cipA can confer elastic  properties to the biomaterial tapping it for biomedical applications such as  wound healing.</LI></UL></DIV><DIV class="ec--h2">REFERENCES</DIV><DIV class="ec--p">
                                1. Mohtaram, N.K., Montgomery, A., Willerth, S.M.  (2013).Biomaterial-based drug delivery systems for the controlled release of  neurotrophic factors. <EM>Biomedical  biomaterials</EM>, 8(2):022001
				    2.  Shi, Q., Li, Y., Sun, J.,  Zhang, H., Chen, L., Chen, B., Yang, H., Wang, Z. (2012).The osteogenesis of  bacterial cellulose scaffold loaded with bone morphogenetic protein-2. <EM>Biomaterials</EM>, 33(28):6644-9
				    3. Winkler, T., Sass, F. A.,  Duda, G. N., &amp; Schmidt-Bleek, K. (2018). A review of biomaterials in bone  defect healing, remaining shortcomings and future opportunities for bone tissue  engineering: The unsolved challenge. <EM>Bone  &amp; Joint Research</EM>, 7(3), 232–243.  http://doi.org/10.1302/2046-3758.73.BJR-2017-0270.R1
				    4. Römling, U.,  &amp; Galperin, M. Y. (2015). Bacterial cellulose biosynthesis: diversity of  operons, subunits, products and functions. <EM>Trends  in Microbiology</EM>, 23(9), 545–557. http://doi.org/10.1016/j.tim.2015.05.005
				    5. Buldum, G.,  Bismarck, A., &amp; Mantalaris, A. (2018). Recombinant biosynthesis of  bacterial cellulose in genetically modified Escherichia coli. <EM>Bioprocess and Biosystems Engineering</EM>, 41(2),  265–279. 
				    6. Zhang, M.,  Sheng-Cheng, W., Zhou, W. &amp; Xu, B. (2012).Imaging and Measuring  Single-Molecule Interaction between a Carbohydrate-Binding Module and Natural  Plant Cell Wall Cellulose. <EM>J. Phys. Chem</EM>.  B, 2012, 116 (33), pp 9949–9956
                                    7. An, S. J., Lee, S. H., Huh, J. B., Jeong, S. I., Park, J. S., Gwon, H. J., Kang, E. S., Jeong, C. M., … Lim, Y. M. (2017). Preparation and Characterization of Resorbable Bacterial Cellulose Membranes Treated by Electron Beam Irradiation for Guided Bone Regeneration. International journal of molecular sciences, 18(11), 2236. doi:10.3390/ijms18112236

				    8. Schmoekel, H.  G., Weber, F. E., Schense, J. C., Grätz, K. W., Schawalder , P., &amp; Hubbell  , J. A. (2005). Bone repair with a form of BMP-2 engineered for incorporation  into fibrin cell ingrowth matrices. <EM>Biotechnology  Bioengineering</EM>, 89(3):253-62.
				    9. Yaniv, O., Morag,  E., Borovok, I., Bayer, E. A., Lamed, R., Frolow, F., &amp; Shimon, L. J. W.  (2013). Structure of a family 3a carbohydrate-binding module from the  cellulosomal scaffoldin CipA of Clostridium thermocellum with flanking linkers:  implications for cellulosome structure. <EM>Acta  Crystallographica Section F</EM>: <EM>Structural  Biology and Crystallization Communications</EM>, 69(Pt 7), 733–737. 
				    10. Team Imperial  College London. (2014). Aqualose. Recovery from https://2014.igem.org/Team:Imperial
			            11. Tang, N. &amp; Chilkoti,  A. (2016).Combinatorial codon scrambling enables scalable gene synthesis and  amplification of repetitive proteins. <EM>Nature  Materials</EM>, 15, pages 419–424 </DIV></DIV></DIV></DIV></DIV><DIV class="ecuador_footer"><DIV class="ecuador_footer_information_layout"><DIV class="ecuador_footer_information_container"><DIV class="ecuador_footer_information_title_container">
            SPONSORS
		</DIV></DIV><DIV class="ecuador_footer_information_container"><DIV class="ecuador_footer_information_title_container">
            FOLLOW US
            </DIV></DIV><DIV class="ecuador_footer_information_container"><DIV class="ecuador_footer_information_title_container">
            CONTACT US
			<DIV class="ecuador_information_data_container">
				biosin.espe@outlook.es
			UNIVERSIDAD DE LAS FUERZAS ARMADAS ESPE.
			Ecuador,Sangolquí, Av. General Rumiñahui s/n y Ambato
			</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>