<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Delft_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Delft/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-top"><DIV class="col-sm-6 col-xs-12"><UL class="list-unstyled list-inline header-contact"><LI><A href="mailto:tudelft.igem@gmail.com">tudelft.igem@gmail.com</A></LI></UL></DIV></DIV><DIV class="navbar-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse pull-right"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:TU_Delft/Achievements">ACHIEVEMENTS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Achievements#requirements">Medal Requirements</A></LI><LI class="submenu-item"><A href="">Special Prizes</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Design">Best Applied Design</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Hardware">Best Hardware</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Software">Best Software Tool</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Entrepreneurship">Best Entrepreneurship</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model">Best Model</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Composite_Part">Best Composite Part</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Integrated_Practices">Best Integrated Human Practices</A></LI></UL><LI><A href="https://2016.igem.org/Team:TU_Delft/Team">TEAM</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#members">Team Members</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#advisors">Advisors</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#fun">Fun Activities</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Project">PROJECT</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#description">Description</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#results">Experiments</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#fluorophores">Fluorophore expression</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#silicatein">Silicatein expression</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#Biolaser">Biolaser</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#Biolenses">Biolenses</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#Conclusions">Conclusions</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#conclusions">Conclusions and Recommendations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Safety">SAFETY</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Safety#workspace">Workspaces</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Safety#safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Notebook">NOTEBOOK</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Notebook#notes">Day Notes</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Notebook#protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Practices">PRACTICES</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#overview">Overview</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#product">Product Analysis</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#safety">Risk Assessment Tool</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#riskas">Risk Assessment Discussion</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#experts">Experts' Opinions</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#entrepre">Business Plan</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#igemana">iGEM Analysis</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#toolbox">iGEM Toolbox</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Model">MODELING</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model#lasers">Biolasers</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q1">Q1: Cavity size</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q2">Q2: Fluorophore concentration</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q3">Q3: Laser threshold</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q4">Q4: Cavity quality factor</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model#lenses">Biolenses</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q5">Q5: Focusing of light</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q6">Q6: Scattering of light</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q7">Q7: Influence of polysilicate layer thickness</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory">Theory</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory#light">What is light?</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory#lasers">What is a laser?</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory#mie">Mie Theory</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Hardware">HARDWARE</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Hardware#overview">Overview</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Hardware#building">Building the microscope</A></LI><LI><A href="/Team:TU_Delft/Collaborations">COLLABORATIONS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Collaborations#collab">Collaborations</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Collaborations#meetups">Meetups</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Parts">PARTS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#parts">All Parts</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#composite">Composite Parts</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#basic">Basic Parts</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#collection">Parts Collection</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Attributions">ATTRIBUTIONS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Attributions#attributions">Team Attributions</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Attributions#sponsors">Sponsors</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Attributions#acknowledgements">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Interlab">INTERLAB</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Interlab#results">Results</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Interlab#discussion">Discussion</A></LI></DIV></DIV><P><TITLE>iGEM TU Delft</TITLE></P><DIV class="page-heading text-center"><DIV class="container"><H1 class="page-header">Opticoli</H1><H3> The new age of optics: Producing biological lenses and lasers to improve microscopy </H3></DIV></DIV><DIV class="main-container"><DIV class="container project"><DIV class="our-project"><H2 class="title-style-1">Project Description</H2></DIV><DIV class="row"><H2 class="title-style-2 col-md-offset-1">Limitations of microscopy</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P>Microscopes have been around for hundreds of years and the technology behind these devices has been quickly developing over the past centuries. Especially fluorescence microscopy was an essential discovery for us biologists, since we are especially interested in what processes occur inside the cell. A popular approach to image intra- and extracellular processes is to use fluorescent tags to track a molecule- or gene of interest in the cell. These fluorescent tags can be imaged under a fluorescence microscope, allowing us to trace molecules and gene expression in a cell.</P><P>Fluorescence microscopy is popular technique that has been essential for cellular research over the past decade, and has helped us to find out several important basics of life. Even though most techniques are already very far developed, <STRONG>it is essential that we keep developing microscopy techniques even further</STRONG>. When we can image every process going on in the cell, we are able to use this for our own good. We can, for example, get a better understanding of a disease, which is essential in finding a cure. An example of a disease that has been studied for years but still not fully understood, is Alzheimer’s disease <A href="#references">(Hardy &amp; Selkoe, 2002)</A>. In an ideal future, if we could tag all molecules in a brain cell and image them, we might find what exactly causes the disease and hopefully develop a treatment. Also synthetic biology in general benefits from a good understanding of the cell. When we can trace all enzymes involved in a certain process, for example the alcoholic fermentation in yeast, we can more easily modify genes of interest and optimize this pathway for an improved production of biofuels.</P><DIV class="row"><DIV class="col-md-6"><P>Unfortunately, microscopy hasn’t yet advanced so far that we can track all processes in the cell. Though super-resolution microscopy is quickly developing, there are still several limitations that hinder a full visualization of the cell. At this point, the technology and knowledge of microscopy is not the biggest limit for making detailed images of the cell; <STRONG>it’s the cells itself</STRONG>. When using fluorescence microscopy, the limit of the resolution of the microscopy is the amount of photons that your sample emits. However, not all photons are observed by the detector of the microscope, simply because not all photons reach this it and get lost in its noise <A href="#references">(Heintzmann &amp; Ficz, 2006)</A>. Especially in tracing low intracellular concentrations or high-speed cellular processes, the amount of photons emitted is low <A href="#references">(Lakowicz, 2013)</A>. <STRONG>We aim to improve this limit of microscopy using synthetic biology.</STRONG></P></DIV><DIV class="col-md-6"><FIGURE><FIGCAPTION><B>Figure 1:</B> When a fluorescent cell is imaged, not all photons will reach the detector, so not all photons will be detected.
                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><DIV class="row"><H2 class="title-style-2 col-md-offset-1">Improving microscopy using biology</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P>The chance of a photon being observed is defined by both the chance of the photon being emitted and the photon being detected <A href="#references">(Heintzmann &amp; Ficz, 2006)</A>. We have thought of two possible solutions to improve the observation of a photon. The first one is increasing the chance of a photon being emitted by the sample, which in this case, are our cells. We will do this by modifying the cells in such a way they are <STRONG>emitting more photons</STRONG> for the same amount of fluorophores. This technique is based on the working principle of a laser. Therefore, we will call these cells <STRONG><A href="#Biolaser2"> the Biolaser.</A></STRONG></P><P>A second approach to increase the chance of a photon hitting the detector is by <STRONG>directing the photons towards it</STRONG>. An easy way to do this is to apply a layer of lenses over the detector to focus the light onto it. A layer of tiny lenses is also called a microlens array. However, these microlens arrays are hard to fabricate and their synthesis is harmful for the environment. Therefore, we will modify bacteria in such a way they will become lenses: <STRONG><A href="#Biolens2"> the Biolenses.</A></STRONG></P><DIV class="col-md-12"><FIGURE><FIGCAPTION><B>Figure 2:</B> two ways to improve the amount of photons detected in fluorescence microscopy.  First, by modifying a cell so it emits more photons at once using the same amount of fluorophores, we increase the chance of detecting a photon. This method is based on the working principle of a laser and is therefore called a Biolaser. Secondly, using an array of lenses we can focus light onto the detector, increasing the amount of photons detected. These lenses are synthesized using microorganisms, hence they are called Biolenses. Click on the images to find out more.
                                </FIGCAPTION></FIGURE></DIV></DIV></DIV><DIV class="row"><H2 class="title-style-2 col-md-offset-1"> Synthetic biology to produce biological lasers </H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P>In order to modify <I>Escherichia coli</I> in such a way that it is able to emit laser-like light, meaning it emits more photons, we first need to understand how a laser works. A laser has three essential components: a gain medium, an excitation source and a reflective agent. The gain medium is a medium with the ability to fluoresce. This gain medium is surrounded by a reflective agent, such as a mirror. This gain medium gets excited by the excitation source, which could be either an electric pulse or an external light source. Once this gain medium gets excited, it will emit photons. Because the gain medium is surrounded by a mirror, the photons cannot escape the gain medium but will ‘bounce’ back. When one of these photons hits an excited fluorescent molecule, something remarkable happens. This molecule will release an exact copy of the incident photon. This process is called stimulated emission. The result is that the light gets amplified each time it passes through the gain medium. Therefore, with only a limited amount of fluorescent molecules we can emit a lot more light compared to ‘conventional’ fluorescence. </P><DIV class="row"><DIV class="col-md-12"><FIGURE><FIGCAPTION><B>Figure 3:</B> a schematic representation of the process of lasing. First, the fluorescent gain medium is excited by an external excitation source. The excited molecules emit photons, which bounce back on the mirror surrounding the gain medium. When one of these photons hits another excited molecule, this molecule releases an exact copy of the incident photon, therefore ‘amplifying’ the light.
                                    </FIGCAPTION></FIGURE></DIV></DIV><P>In this project, we will use synthetic biology to modify <I>E. coli</I> in such a way that the bacterium will be able to emit laser-like light. In order to do this, we have to translate two of the main components of a laser, the gain medium and the mirrors, to biological alternatives that <I>E. coli</I> is able to produce. For the gain medium this is easy. We can transform <I>E. coli</I> with <STRONG>fluorescent proteins</STRONG> that will form the fluorescent gain medium. The biological alternative for mirrors is slightly less obvious. However, we believe to have found the solution in the enzyme <STRONG>silicatein</STRONG>. This enzyme is able to synthesize polysilicate, a biological glass <A href="#references">(Müller et al., 2008; Müller et al. 2003)</A>. By transforming <I>E. coli</I> with the gene for this enzyme, we can let the cells coat itself in a layer of glass that will reflect the photons emitted by the fluorescent proteins. As an excitation source we can simply use a laser to excite the proteins. A figure of our synthetic biology approach to create a biological laser can be seen in figure 4. This biolaser will be able to emit a higher number of photons compared to a cell that is merely fluorescent. Therefore, this cell will be useful in fluorescence microscopy, since its intracellular processes will be more easily detected. Therefore, this cell could be clearly imaged. Our parts, experiments and results can be found in the <A href="#results"><B>Experiments section</B></A></P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 4:</B> designing a laser using synthetic biology. The lasers gain
                                        medium can be substituted by fluorescent proteins, a layer of biologically synthesized
                                        polysilicate acts as the mirror around the gain medium. The fluorophores are excited by a laser. 
                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><DIV class="row"><H2 class="title-style-2 col-md-offset-1">Synthetic biology to produce biological lenses</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P>The second approach to capture more light in fluorescence microscopy is by focusing the photons onto the detector of the microscope. We will do this with lenses, since they have the ability to focus light onto an object, such as the detector of a microscope, which is schematically shown in figure 5. </P><FIGURE><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGCAPTION><B>Figure 5:</B> When shining a beam of light onto a detector, not all light might hit. Placing a lens in front of the detector the beam of light is focused towards it so all the photons are measured. </FIGCAPTION></DIV></DIV></FIGURE><P>In a microscope, applying this technique would mean that e.g. each photovoltaic cell of the detector gets a lens placed on it that will direct all light into each cell of the detector. This means we need a matrix, or “array”, of lenses that are only a few micrometers small.  These ‘microlens arrays’ already exist, and have been shown to be a good technique to focus more light onto photovoltaic cells, including the detector of a microscope <A href="#references"> (Jutteau, Paire, Proise, Lombez, &amp; Guillemoles, 2015)</A>. We will also use this technique. However, we will not use the conventional, chemically produced microlenses, since they are very costly and their production is difficult and bad for the environment <A href="#references"> (Nam et al., 2013)</A>. Therefore, we aim to produce our own, <STRONG>biological microlenses</STRONG> that will be much greener and environmentally friendly compared to the conventional microlenses. Furthermore, we will also apply our biological microlenses in applications, other than microscopy, where collecting light is important. One of these applications in which we will apply our biological lenses are <STRONG>solar cells</STRONG>, more information can be found on the  <A href="https://2016.igem.org/Team:TU_Delft/Practices" target="_blank"><B>practices page</B></A>.</P><P>So how do we aim to produce these biological lenses using synthetic biology? The approach is the same as for the biological lasers: we transform <I>E. coli</I> with the silicatein gene. This gene allows the cell to cover itself in glass, resulting in a glass sphere of only 1-2 µm small. We will test the optical properties of this glass sphere as well as its functionality as a microlens. Since the shape of lenses is of high importance, we will also research ways to manipulate the size and shape of the cells, enabling us to produce different shapes of microlenses. Also, since our biological microlens contains a core of live bacteria, we will look into ways to sterilize the lens. This way, our product will <STRONG>not harm the environment</STRONG> in any way. Our parts, experiments and results can be found in the <A href="#silicatein"><B>Experiments section</B></A></P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 6:</B> By transforming <I>E. coli</I> with a gene for the enzyme silicatein, the bacterium is able to coat itself with polysilicate, a kind of biological glass. This turns the bacterium in a glass sphere of 1-2 µm that can function as a microlens.
                                    </FIGCAPTION></FIGURE></DIV></DIV></DIV></DIV><DIV class="row"><H2 class="title-style-1">Experiments and results</H2></DIV><DIV class="row streepje "><H2 class="title-style-2 col-md-offset-1">Expression of different fluorophores</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><H3>Introduction</H3><P>One of the essential components of a laser is a fluorescent agent. Since our aim is to produce a fully biological laser, fluorescent proteins are favourable. To this end, we initially selected four fluorophores with different emission wavelengths: GFP, mVenus, mKate and mCerulean. These fluorophores were reported to have an increased fluorescent intensity compared to their wildtype <A href="#references">(Cormack <I>et al.</I>, 1996; Nagai <I>et al.</I>, 2002; Shcherboo <I>et al.</I>, 2007; Rizzo <I>et al.</I>, 2004)</A>.</P><P>Since mVenus, mKate and mCerulean did not exist in the iGEM registry yet, we constructed a brand new part for each of these fluorophores including strong constitutive promoter and RBS (parts <B><A href="http://parts.igem.org/Part:BBa_K1890010" target="_blank">K1890010</A></B> and <B><A href="http://parts.igem.org/Part:BBa_K1890011" target="_blank">K1890011</A></B>). GFP, on the other hand, was present in a whole range of biobricks. However, to our knowledge, there was no single biobrick available containing promoter, RBS and terminators. Hence, we constructed a new biobrick containing all of the above using the existing part <B><A href="http://parts.igem.org/Part:BBa_E0840" target="_blank">E0840</A></B>, consisting of RBS, coding sequence and terminators. By means of PCR we amplified this biobrick with primers designed to add a promoter while mainaining the biobrick prefix and suffix. Not only did we express GFP under the strong promoter <B><A href="http://parts.igem.org/Part:BBa_J23100" target="_blank">J23100</A></B>, but also under less strong promoters <B><A href="http://parts.igem.org/Part:BBa_J23113" target="_blank">J23113</A></B>, <B><A href="http://parts.igem.org/Part:BBa_J23117" target="_blank">J23117</A></B>, <B><A href="http://parts.igem.org/Part:BBa_J23105" target="_blank">J23105</A></B>, and <B><A href="http://parts.igem.org/Part:BBa_J23108" target="_blank">J23108</A></B>. This way we were able to see the influence of promoter strength on fluorescent output (parts <B><A href="http://parts.igem.org/Part:BBa_K1890020" target="_blank">K1890020</A></B> to <B><A href="http://parts.igem.org/Part:BBa_K1890024" target="_blank">K1890024</A></B>.</P><H3>Experiments and Results</H3><P>In order to characterize these biobricks before further use, the emission spectra of the fluorophores were measured. Additionally, their effect on cell growth was investigated.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseOne1" aria-expanded="false" aria-controls="collapseOne1"><DIV class="panel-heading" role="tab" id="headingOne1"><H4 class="panel-title">
                                            Emission spectra of different fluorophores
                                        </H4></DIV></A><DIV id="collapseOne1" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne1"><DIV class="panel-body"><H3>Introduction</H3><P>To assure the fluorophores were functional, the emission spectra were recorded at the given excitation wavelength.</P><H3>Methods</H3><P>Parts encoding the four different fluorophores, including promoter, RBS and terminators, were expressed in <I>E. coli</I> BL21 strain. GFP, mVenus, mKate and mCerulean were all expressed under the same strong constitutive promoter, J23100. Additionally, parts were constructed with GFP under control of promoters with different strengths, in order to investigate the influence of different fluorophore concentrations. </P><P>All of the mentioned biobricks were expressed in <I>E. coli</I> BL21. After growing in LB medium to an OD600 of ~1, the cells were washed and resuspended in the same volume of PBS. Aliquots of 100 µL were put in a 96 well plate.  </P><P>The emission spectrum of each fluorophore was determined by exciting at a given wavelength and measuring the output intensity at a range of wavelengths. Due to thelimit bandwidth of the spectrometer, Because of this, the emission at a wavelength too close to the excitation could not be measured without measuring the excitation source directly. This can be seen in the figures, where the left half of the emission peaks could not be measured. Especially for mVenus, where the excitation and emission wavelengths are very close together.</P><H3>Results and Discussion</H3><H4>Fluorophores expressed under strong constitutive promoter</H4><P>As all fluorophores were expressed under the strong constitutive promoter J23100, they were expected to show a strong fluorescence without the need of induction. Figure 1 shows  that this was the case for GFP, mVenus and mCerulean. mKate, however, did not show any fluorescent activity and was therefore not used in the subsequent steps of the project.</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Figure 1:</B> Emission spectra of the fluorophores GFP, mVenus, mCerulean, and mKate expressed under strong promoter J23100. Excitation wavelength was 488 nm, 510 nm, 433 nm or 558 nm, respectively.</FIGCAPTION></DIV></DIV></FIGURE><H4>GFP expressed under constitutive promoters of different strengths.</H4><P>Not only did we express GFP under the strong promoter J23100, but also under less strong promoters from weak to stronger J23113, J23117, J23105, and J23108 (increasing in strength).</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B> Emission spectra of GFP expressed under control of promoters with different strengths. Excitation wavelength was 488 nm.
                                                    </FIGCAPTION></FIGURE></DIV></DIV><P>For comparison, the emission was normalized by dividing by OD600. All strains were measured in the same dilution, in order to make the results reproducible. The emission intensity is as expected, corresponding to the strength of the promoters. All fluorophore spectra were also recorded in a dilution more suited for their emission intensity and normalized to their maximal emission to the maximum intensity of each construct (Figure 3). From this figure we can conclude that all GFP biobricks function properly.</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 3:</B> Emission spectra of GFP expressed under control of promoters with different strengths, normalized to their maximal intensity. Excitation wavelength was 488 nm.
                                                    </FIGCAPTION></FIGURE></DIV></DIV><A href="#fluorophores" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#fluorophores').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo2" aria-expanded="false" aria-controls="collapseTwo2"><DIV class="panel-heading" role="tab" id="headingTwo2"><H4 class="panel-title">                            The effect of expression of fluorophores on growth
                                        </H4></DIV></A><DIV id="collapseTwo2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo2"><DIV class="panel-body"><H3>Introduction</H3><P>As consitutive expression can sometimes be hard on the cells, we investigated the effect of the different consitutive promoters on cell growth. In a 24 hour kinetic cycle alterating shaking at 37°C with fluorescence and optical density measurements, we investigated whether this was the case.</P><H3>Methods</H3><P>An overnight culture in eM9 medium was inoculated in fresh eM9 to an OD600 of 0.1 in a 96 well plate. The emission at 522 nm was measured every 15 minutes. Measurements were done in quadruplicate with pure eM9 as a blank.</P><H3>Results and Discussion</H3><P>Figure 1 shows the 24 hour measurement of optical density and fluorescence intensity. The final OD600 is approximately equal for all different strains, suggesting that the level of constitutive expression was not influencing the growth. Furthermore, fluorescence intensity drops after the exponential growth phase, suggesting that GFP is being broken down by proteases as a response to nutrient limitation. After this event, growth continues at a slower pace, while GFP activity keeps decreasing. All in all, constitutive expression of GFP does not seem to have a detrimental effect on cell growth during exponential phase.</P><DIV class="row"><DIV class="col-md-12 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> GFP. Kinetic measurement of fluorescence intensity at 522 nm and optical density at 600 nm, while shaking at 37°C. Above 6·10<SUP>4</SUP> the intensity was too high to be measured.</FIGCAPTION></FIGURE></DIV></DIV><P>Cells expressing mCerulean or mVenus, however, seem to be having a longer lag phase before exponential growth starts (Figure 2). Also, they reach a lower final OD600 than the ones expressing GFP. These fluorophores might be slightly harmfull for cell growth. Nonetheless, they do grow and exhibit fluorescence, so they can be used in further experiments.</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B> Kinetic measurement of fluorescence intensity of mCerulean (left) and mVenus (right) at 544 nm and 475 nm, respecitively, and optical density at 600 nm, while shaking at 37°C.
                                                    </FIGCAPTION></FIGURE></DIV></DIV><A href="#fluorophores" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#fluorophores').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV></DIV><H3>Discussion and Conclusions</H3><P>We were able to succesfully construct and characterize two biobricks with brand new fluorophores for the iGEM registry: <B><A href="http://parts.igem.org/Part:BBa_K1890011" target="_blank">mVenus</A></B> and <B><A href="http://parts.igem.org/Part:BBa_K1890010" target="_blank">mCerulean</A></B>. Both of them exhibit a bright fluorescence, which is useful in our futher experiments. To our knowledge there was no part available in the registry consisting of promoter, RBS, GFP coding sequence and terminator Therefore, we constructed <B><A href="http://parts.igem.org/Part:BBa_K1890020" target="_blank">five new composite parts</A></B> to provide single complete biobricks for GFP expression. These new parts were characterized and show a bright fluorescence, according to their promoter strength. All in all, our new parts provide a ready-to-go expression device for green, yellow, or cyan fluorescence, which can be very usefull for future iGEM teams.</P></DIV></DIV><DIV class="row streepje "><H2 class="title-style-2 col-md-offset-1">Coating the cell in polysilicate using silicatein</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><H3>Introduction</H3><P>For both the biolaser and the biolenses we need a coating of polysilicate, biological glass, around the cell. For the biolaser this glass will form the cavity that will enable the cells to emit laser-like light. For the biolens, the glass will give optical properties for the cell. <I>E. coli</I> is intrinsically not able to coat itself in polysilicate. However, upon transformation of the silicatein-α gene, originating from sponges, it is possible to coat the bacterium in a layer of polysilicate <A href="#references">(Müller et al., 2008; Müller et al. 2003)</A>. Therefore, we are transforming <I>E. coli</I> with silicatein-α. We test the use of two different silicateins, one originating from the marine sponge <I>Suberites domuncula</I><A href="#references">(Müller, 2011)</A> and one originating from the marine sponge <I>Tethya aurantia</I><A href="#references">(Cha et al., 1999)</A>. We express the enzyme in three different ways. First of all, we expressed the gene from <I>S. domuncula </I> (Part <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1890000" target="_blank">K1890000</A></B>) and see if the enzyme is transported outside the cell as described by <A href="#references">Müller et al, 2008</A>. Furthermore, we express a fusion of silicatein from <I>T. aurantia</I> to the trans-membrane protein OmpA (outer membrane protein A) from <I>E. coli</I> to anchor the silicatein to the membrane (Part <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1890002" target="_blank">K1890002</A></B>) <A href="#&quot;references&quot;">(Curnow, Kisailus, &amp; Morse, 2006; Francisco et al. 1992)</A>, which might make coating the cell specifically in polysilicate more efficient. We also express a fusion of silicatein from <I>S. domuncula</I> to the transmembrane Ice Nucleation Protein (INP) from  <I>Pseudomonas syringae</I> (Part <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1890001" target="_blank">K1890001</A></B>), a popular protein for membrane fusions <A href="#references">(Kim &amp; Yoo, 1998)</A>. Using these different approaches we expect to coat the cell in polysilicate, an overview is shown in figure 1. This glass coating around the cell will be the basis of both our biolens and –laser. </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> (A) Silicatein is able to convert monosilicate to polysilicate, allowing the cell to cover itself in it. (B) We express silicatein in three ways: solely expressing silicatein and fusing it to the membrane proteins OmpA or INP. 
                                    </FIGCAPTION></FIGURE></DIV></DIV><H3>Experiments and Results</H3><P>To determine if we have successfully covered <I>E. coli</I> in polysilicate, and to characterize the properties of the polysilicate-coated cells, we have performed a series of tests. First of all, we have stained the cells with Rhodamine 123, a fluorescent stain that is able to bind to polysilicate. These stained cells were observed under a fluorescence microscope to determine whether the polysilicate shell was present. Furthermore, the polysilicate-synthesizing cells were observed using both Scanning Electron Microscopy (SEM) and Transmission Electron Microscopy (TEM). We have also determined the physical properties using Atomic Force Microscopy (AFM) to see whether the polysilicate layer changes the stiffness of the cells. Lastly, we have also performed a growth study of the polysilicate-coated cells to determine whether the polysilicate layer affects growth of the organisms.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseUno" aria-expanded="false" aria-controls="collapseUno"><DIV class="panel-heading" role="tab" id="headingUno"><H4 class="panel-title">
                                            Rhodamine 123 staining of polysilicate covered cells
                                        </H4></DIV></A><DIV id="collapseUno" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingUno"><DIV class="panel-body"><H3>Introduction</H3><P>After transforming <I>E. coli</I> with the different silicatein BioBricks, we wanted to confirm and image whether the cell had indeed synthesized a layer of polysilicate around itself. In a very intuitive experiment we can stain polysilicate with the fluorescent dye Rhodamine 123, which has shown to bind specifically to polysilicate  <A href="#references">(Li, Chu, &amp; Lee, 1989)</A>. We can image the stained polysilicate with a simple fluorescence microscope. </P><H3>Methods</H3><P> The experiment was performed using <I>E. coli</I> BL21 with the plasmids and conditions listed in table 1. All genes are expressed under the inducible Lac-promoter, which was present in the plasmid backbone we used, together with the LacI gene. This backbone was obtained from pBbS5a-RFP, a gift from Jay Keasling (Addgene plasmid # 35283) (<A href="#references">Lee <I>et al.</I>, 2011</A>).</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Table 1:</B> Plasmids and conditions used for the Rhodamine 123 staining experiment.
                                                    </FIGCAPTION></DIV></DIV><CENTER><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Plasmid(s)</TH><TH>IPTG</TH><TH>Silicic acid</TH><TH>Rhodamine 123</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn"> OmpA-Silicatein </TD><TD>+</TD><TD>+</TD><TD>+</TD></TR><TR><TD class="firstcolumn"> Silicatein  </TD><TD>+</TD><TD>+</TD><TD>+</TD></TR><TR><TD class="firstcolumn">INP-Silicatein </TD><TD>+</TD><TD>+</TD><TD>+</TD></TR><TR><TD class="firstcolumn">OmpA-Silicatein (negative control) </TD><TD>+</TD><TD>-</TD><TD>+</TD></TR><TR><TD class="firstcolumn">OmpA-Silicatein </TD><TD>+</TD><TD>+</TD><TD>-</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>The cells were stained with 0.1 vol% Rhodamine 123. And washed 5 times with PBS, prior to imaging in a fluorescence microscope at 488nm <A href="#references">(Li et al., 1989; Müller et al., 2005)</A>. Both widefield- (light microscopy) and fluorescence microscopy were used to image the cells.</P><H3>Results and discussion</H3><P>The stained cells were first imaged at maximum excitation intensity. At this excitation energy, only the OmpA-silicatein expressing cells showed fluorescence colocalized with the cells and not in the medium. Silicatein, INP-silicatein and the negative control (OmpA-silicatein without silicic acid) all caused overexposure of the camera. In figure 1 the imaging results of OmpA-silicatein and the negative control (OmpA-silicatein without silicic acid) are shown. The cells that were not stained with Rhodamine 123 showed no measurable fluorescence. (data not shown), indicating there is no autofluorescence at this wavelength.</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> Widefield and fluorescence images of OmpA-silicatein with silicic acid and OmpA-silicatein without silicic acid (negative control) at maximum excitation energy. Of the widefield and fluorescence images an overlay was made to show the colocalization of fluorescence with the cells. The negative control caused overexposure of the camera, attributed to the Rhodamine in the medium, therefore the fluorescent image only gives one uniform signal. 
                                                    </FIGCAPTION></FIGURE></DIV></DIV><P>Since Silicatein,  INP-silicatein and the negative control all caused overexposure of the camera, they all had the same output. We can thus not draw any conclusions for these strains. Therefore, samples where overexposure was observed were imaged again at only 1/3 of the excitation energy. The imaging results are displayed in figure 2. </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B> Widefield and fluorescence images of silicatein with silicic acid, INP-silicatein with silicic acid and OmpA-silicatein without silicic acid (negative control) at one-third of the maximum excitation energy. Of the widefield and fluorescence images an overlay was made to show the colocalization of fluorescence with the cells. 
                                                    </FIGCAPTION></FIGURE></DIV></DIV><P>From figures 1 and 2 we can see that the strain transformed with OmpA-silicatein clearly has a different output from the negative control (figure 1). The fluorescence of this sample is only localized at the cells. This might mean that the Rhodamine 123 has stained these cells and therefore the OmpA-silicatein cells could have the polysilicate layer around their membranes. We cannot distinguish a clear difference between silicatein, INP-silicatein and the negative control (figure 2). The entire medium is fluorescent, which causes overexposure of the camera at high excitation intensity. This might mean that the Rhodamine 123 is not specifically located at the cell walls, but still dissolved in the medium. </P><P>Because the cells were analyzed at different excitation energies, so different camera settings, it is impossible to draw a conclusion from solely the images. Therefore, we analyzed the pictures using ImageJ. We normalized the fluorescence intensity of the cells by the fluorescence intensity of the background. With this method, we are able to compare the fluorescence intensities of the cells, despite the different camera settings. The results of this calculation are shown in figure 3.</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 3:</B> The fluorescence intensity of our silicatein-strains, relative to the fluorescence of the medium. Fluorescence intensity was measured with ImageJ and normalized over the intensity of the medium (background intensity)
                                                    </FIGCAPTION></FIGURE></DIV></DIV><P>In figure 3 we see that the fluorescence intensity of the OmpA-silicatein strain is significantly higher than for the the silicatein and INP-silicatein strains. The silicatein and INP-silicatein strains have a fluorescence intensity comparable to the intensity of the medium (the background intensity). The negative controls (no silicic acid added, measured at two intensities) have the same output as silicatein and INP-silicatein. Therefore, we can conclude that these strains are not able to synthesize a polysilicate layer. However, we can conclude that the <I>E. coli</I> strain transformed with the OmpA-silicatein plasmid is successfully able to synthesize a layer of polysilicate around its membrane. </P><A href="#silicatein" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#silicatein').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTres" aria-expanded="false" aria-controls="collapseTres"><DIV class="panel-heading" role="tab" id="headingTres"><H4 class="panel-title">
                                            Imaging of silicatein-expressing cells and their elemental composition using TEM
                                        </H4></DIV></A><DIV id="collapseTres" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTres"><DIV class="panel-body"><H3>Introduction</H3><P>Electron microscopy is an imaging technique that can acquire images with resolutions far above the optical limit due to the small wavelength of electrons <A href="#references"> (Nellist, 2011)</A>. An electron gun forms an electron beam which is focused and accelerated by magnetic lenses. In order to make sure the electrons are not defracteddiffracted before reaching the sample, the beam and the sample are in high vacuum. When the electrons reach the sample, there are several interactions possible with the sample (figure 1). These various interactions can be used for different techniques. We have used high-angle annular dark-filed transmission electron microscopy (HAAFD-TEM) to make an image of the sample, energy dispersive x-ray spectroscopy to analyze the element composition of the sample and scanning electron microscopy (SEM) to analyze the phenotype of the samples.</P><FIGURE><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGCAPTION><B>Figure 1:</B> Interactions between the electron beam and sample in electron microscopy.
                                                    </FIGCAPTION></DIV></DIV></FIGURE><P>In HAAFD-TEM electrons pass through the sample and are inelastically scattered at a high angle and acquired by a detector. The amount of electrons scattered at a high angle depends on the thickness and the material of the sample. When the thickness is high, the sample scatters a lot it will appear bright in the image.</P><P>Energy dispersive x-ray spectroscopy (EDX) is an analytical technique to determine the elements present in the sample <A href="#references">(Friel <I>et al.</I>, 2006)</A>. When the electron beam interacts with the sample x-rays can emerge from the sample. These electrons can be detected and are characteristic for the element which emitted the x-ray. In that way we can specifically determine which elements are present in our sample. </P><H3> Methods</H3><P>The experiment were performed using <I><I>E. coli</I></I> BL21 cells with the plasmid containing OmpA-Silicatein. Two samples were made where the first sample was induced with IPTG but no silicic acid was added and a second sample which was both induced with IPTG and silicic acid was added, therefore this sample will have a polysilicate layer. The samples were prepared by fixation using 1% polylysine on the surface of a quantifoil carbon grid. Samples were imaged using a Titan transmission electron microscope from FEI company.</P><H3> Results and discussion</H3><P> Both samples with and without silicic acid added were imaged using HAAFD-TEM and energy dispersive x-ray spectroscopy (figure 2). The cells are fixed at a Quantifoil carbon grid, consisting of carbon film perforated with holes and mounted on a carbon grid. In figure 2A and 2C the white structure is a cell laying on a hole in the grid. In each sample, we measured elemental composition of our sample including the silicon content (figure 2 B,D) the blue spots in these images indicate where silicon is detected. The grid itself already contains silicon but in the holes of the grid no silicon is present (figure 2 B,D). Therefore we only measured the presence of silicon in bacteria laying on a hole on in the grid to make sure we do not have background silicon signal from the grid, which cannot be distinguished from the actual signal.</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Figure 2:</B> (A,C) HAAFD image  and (B,D) EDX spectroscopy of silicon. (A,B). Image of the same cell containing OmpA-silicatein without silicic added to the sample (negative control). (C,D) Image of the same cell containing OmpA-Silicatein with silicic acid added to the sample.
                                                    </FIGCAPTION></DIV></DIV></FIGURE><P>For the sample where no silicic acid is added (figure 2 A,B), we can see some silicon present at the position of the cell. However, there is a significant increase in silicon detected for the sample where silicic acid was added to the sample (figure 2D). This shows that silicon co-localizes with the cell which means there is indeed a polysilicate layer formed by the bacteria. </P><A href="#silicatein" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#silicatein').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseDos" aria-expanded="false" aria-controls="collapseDos"><DIV class="panel-heading" role="tab" id="headingDos"><H4 class="panel-title">
                                            Imaging of silicatein-expressing cells using SEM
                                        </H4></DIV></A><DIV id="collapseDos" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingDos"><DIV class="panel-body"><H3>Introduction</H3><P>The Scanning Electron Microscope, or SEM as it is called, is a widely used imaging technique in science. In SEM, a beam of electrons is accelerated by high voltage and it is directed towards the sample. When the beam hits the sample, x-rays, backscattered electrons and secondary electrons are generated (Figure 1 in the TEM section). For imaging, SEM uses the secondary electrons. Those secondary electrons are then collected and their kinetic energy is transformed to an image. <A href="#references">(Kwakernaak &amp; Sloof, 2005)</A> SEM is widely used due to its very high spatial resolution and Depth of Focus (DoF). In order to use SEM, samples have to be fixed and dried, because the measurements take place under vacuum. </P><P> With SEM, we hope to see the impact of the polysilicate layer on the phenotype of the cell. If we want to make microlenses, the surface of the cell should be smooth, despite the polysilicate layer. Any deformations in the layer can cause optical aberrations. Since SEM has a very high spatial resolution, we can use this technique to image what our polysilicate-covered cells look like and to see what effect the polysilicate layer has on the cells.</P><H3>Methods</H3><P>The polysilicate layer around the cells was prepared according to the polysilicate layer <A href="https://2016.igem.org/Team:TU_Delft/Notebook#protocols" target="_blank">protocol</A>. As a negative control, cultures without substrate (no sodium silicate) were used. The samples were fixed with glutaraldehyde and imaged with SEM. We used an FEI Niva Nano 450 SEM, under high vacuum. </P><DIV class="row"><DIV class="col-md-6"><FIGCAPTION>SEM experiments in the cleanroom!</FIGCAPTION></DIV></DIV><H3>Results and Discussion</H3><P>Figure 1 shows SEM images taken of samples in the presence (A, B) or absence (C, D) of silic acid. </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> SEM images of <I>E. coli</I> expressing OmpA-silicatein in the presence (A, B) or absence (C, D) of sodium silicate.</FIGCAPTION></FIGURE></DIV></DIV><P>We know from a previous experiment, the rhodamine 123 staining, that cells with OmpA-silicatein have a polysilicate layer. Figure 1 suggests that this layer does not seem to influence the cell shape as the cells have their normal size and shape. However, the cells that are expected to have a polysilicate layer appear to be somewhat fused together. According to <A href="#references">Müller et al (2008)</A>, cells possessing a polysilicate layer appear to be fused by a viscous cover. This might, however, also be the result of limited imaging resolution or an artifact in sample preparation. When using titanium oxide as a substrate, <A href="#references">Curnow et al (2005)</A> reported large aggregates visible by SEM. This was not observed in the current experiment, where we use silicic acid as a substrate. This comparison suggests that the polysilicate layer does not form aggregated or influence the shape of the cell, but form a homogeneous layer around the cell.</P><A href="#silicatein" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#silicatein').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseQuatro" aria-expanded="false" aria-controls="collapseQuatro"><DIV class="panel-heading" role="tab" id="headingQuatro"><H4 class="panel-title">
                                            Analysis of physical properties of polysilicate covered cells using AFM
                                        </H4></DIV></A><DIV id="collapseQuatro" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingQuatro"><DIV class="panel-body"><H3>Introduction</H3><P> The physical properties of polysilicate are different than those of the cell. We wanted to determine how the physical properties of the cell changes when encapsulated by a layer of polysilicate. Therefore we determine the stiffness of cells with and without a layer of polysilicate using atomic force microscopy (AFM).  The stiffness is the rigidity of the a material, describing how easy the cell is deformed. </P><P> AFM is a technique used to determine surface characteristics of a sample. In AFM a very sharp tip scans over the surface of the sample. While scanning over the surface every height change is detected by a laser which is reflected by the cantilever on a position-sensitive photodetector (Figure 1). We used the PeakForce QNM mode of AFM, in which also surface characteristics like the stiffness and deformation can be determined. The stiffness is computed from the relation between the force at which the cantilever pushes on the sample, the adhesion force and the deformation of the material.</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Figure 1:</B> AFM Setup. A laser is reflected by a cantilever and detected by a position sensitive photodetector. Every change in height of the sample results in a change of height of the cantilever and is measured. </FIGCAPTION></DIV></DIV></FIGURE><H3> Methods</H3><P> Experiments were performed using <I><I>E. coli</I></I> BL21 cells transformed with the plasmid containing the OmpA-Silicatein gene and induced with IPTG. Silicic acid was added to one sample to form the polysilicate layer around the cell. Another sample with no silicic acid added was used as a control. The cells weres spun down and resuspended in MilliQ and fixated on a glass slide using 1% ABTES. The samples were imaged using the Bruker FastScan-Bio.</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGCAPTION><B>Figure 2:</B> AFM experiments in the cleanroom. 
                                                    </FIGCAPTION></DIV></DIV><H3> Results and discussion</H3><P>We have imaged two samples with AFM. The first sample is OmpA-silicatein with silic acid added so that the cell can encapsulate itself with polysilicate (figure 3A, B). The second sample, that was used as a control, did not contain silicic acid (figure 3C-D).  From both samples a height map (figure 3A, C) and the stiffness (figure 3B, D) was determined. Both samples were fixed on a glass slide in the same way. Due to a tip change is there a factor 10 difference between the measured stiffness of the glass slide of both samples. </P><DIV class="row"><DIV class="col-md-12 col-sm-12"><FIGCAPTION><B>Figure 3:</B> Pictures taken with AFM of (A-B) <I>E. coli</I> transformed with OmpA-silicatein with silicic acid added, (C-D) <I>E. coli</I> transformed with OmpA-silicatein without silic acid added. (A,C) are height maps of the cell, (B,D) are stiffness maps. (E) Relative stiffness of <I>E. coli</I> cells covered with and without polysilicate layer, compared to the stiffness of a glass slide measured with Peakforce QNM AFM.
                                                    </FIGCAPTION></DIV></DIV>Multiple cells (n=3) were imaged and the relative stiffness of the cell compared to the stiffness of the glass slide was determined (figure 3E). We found that cells covered with a layer of polysilicate have a stiffness of 0.43 compared to the glass slide and the cells without a layer of polysilicate have a stiffness of 0.16 compared to the glass slide. We can see that due to the encapsulation of the cells in polysilicate the stiffness of the cells increases significantly. <A href="#silicatein" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#silicatein').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseCinco" aria-expanded="false" aria-controls="collapseCinco"><DIV class="panel-heading" role="tab" id="headingCinco"><H4 class="panel-title">
                                            Viability experiments of polysilicate covered cells
                                        </H4></DIV></A><DIV id="collapseCinco" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingCinco"><DIV class="panel-body"><H3>Introduction</H3><P>Since the silicatein expressing cells are to cover themselves in polysilicate, their nutrient supply might be limited by diffusion, which can eventually result in cell death. To investigate whether this is indeed the case,a growth study was performed.</P><H3>Methods</H3><P>Cells containing the three different silicatein biobricks were grown overnight in selective LB. They were transfered to fresh medium and grown until in exponential phase. Then IPTG was added to induce expression. After a subsequent incubation of three hours, the medium was supplemented with silicic acid as substrate for silicatein. During the following five hours samples were taken, of which a 10<SUP>-6</SUP> dilution was plated on selective LB plates. Colony forming units (cfu) were counted the day after.</P><H3>Results and Discussion</H3><P>Cells expressing either silicatein (Sil), silicatein fused to INP (INP-Silicatein) or silicatein fused to OmpA (OmpA-Silicatein) were tested. As a negative control, OmpA-silicatein expressing cells without silicic acid were used. After one hour no colonies were observed on the plates on which the cultures with silicic acid were plated (Figure 1). The cultures without silicic acid continued to grow until after five hours.</P><FIGURE><DIV class="row"><DIV class="col-md8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Figure 1:</B> Number of colony forming units (cfu) during incubation with silicic acid.</FIGCAPTION></DIV></DIV></FIGURE><P>This figure suggests that either the polysilicate layer inhibits nutrient diffusion into the cell, unlike reported by Müller et al. (2008), or the sodium silicate has a detrimental effect on growth.</P><A href="#silicatein" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#silicatein').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV></DIV><H3> Discussion and Conclusions </H3><P>In this part of our project we have transformed <I>E. coli</I> with a gene encoding the enzyme silicatein in three different ways. We solely expressed the silicatein protein, but have also fused the protein to two different membrane proteins, INP and OmpA. The goal was to let the cells produce a polysilicate layer around their membrane, so the cell was completely covered in this biological glass. We have characterized the polysilicate-covered cells by electron microscopy, AFM, spectroscopy, fluorescent staining and with a growth study.</P><P>After staining the silicatein-expressing cells with  the fluorescent stain Rhodamine 123, which binds specifically to the polysilicate layer, we found that the cells transformed with the OmpA-silicatein fusion <STRONG>successfully synthesized a polysilicate layer around the cell</STRONG>, therefore we continued all experiments with this strain. This result was confirmed with TEM and EDX spectroscopy, where <STRONG>a higher concentration of silicon was found</STRONG> on the polysilicate-synthesizing cells. SEM showed that this layer is a <STRONG>neat, homogeneous layer </STRONG>and does not form any aggregates of polysilicate on the cell. Aggregates could seriously disturb the optical properties of our cell, so it’s a good results that our cells are covered homogeneously. AFM showed that the cells covered in polysilicate have a <STRONG>higher stiffness </STRONG>compared to cells without the polysilicate layer. This proves the presence of this layer. The finding that the cells are more rigid is also useful for our idea to develop biolenses, since these should not easily deform. Lastly, our growth study shows that the cells covered in polysilicate live shorter that cells without this layer. This means that when we want to produce microlenses at a high scale, we should first grow the cells prior to incubating them with silicic acid. The fact that the cells die when they are encapsulated could also make it easier to use our microlenses in user-products. Bringing GMO’s out of the lab is often not permitted, so if we can show that our biolenses are inviable, this would not be a problem for our product. So, from these expreiments we can conclude that <STRONG>we have successfully genetically modified <I>E. coli</I> in such a way that it covers itself in a homogeneous layer of polysilicate.</STRONG> We can use these cells for further experiments on their ability to function as a laser or lens.</P></DIV></DIV><DIV class="row streepje"><H2 class="title-style-2 col-md-offset-1">Engineering a biological laser</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><H3>Introduction</H3><P>Our cellular laser consists out of two features: fluorophores will be the light of the laser and a polysilicate layer synthesized by silicatein will be the ‘mirrors’ that reflect a part of the photons emitted by these fluorophores. The fluorophores first need to be excited by an external light source. This could be either an LED source or a conventional solid laser. For fluorescence an LED excitation source suffices. However, we want lasing to happen in our cells. For this to happen, we need ‘population inversion’, which means the majority of the fluorophores is in an excited state <A href="#references">(Gather &amp; Yun, 2011; Svelto &amp; Hanna, 1976)</A>. More information on this can be found in the <B><A href="https://2016.igem.org/Team:TU_Delft/Hardware" target="_blank">hardware page</A></B>. In order to excite the majority of the fluorescent proteins at the same time, we need a strong excitation source. Therefore we need to use a laser to excite the fluorphores in our biolaser. However, a major downside to using lasers for fluorophore excitation is the occurrence of photobleaching <A href="#references">(Eggeling, Widengren, Rigler, &amp; Seidel, 1998)</A>. The laser power required for the excitation of fluorophores to induce population inversion in the cell is so high it will photobleach the fluorophores within microseconds <A href="#references">(Jonáš et al., 2014)</A>. Therefore, we need a custom laser setup to prevent photobleaching but establish the population inversion required for our biolaser. This laser setup should contain a pulsing laser which pulses at a frequency that will maintain the excited state but does not photobleach the proteins. </P><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> Our custom-built hardware setup to image our Biolaser cells.</FIGCAPTION></FIGURE></DIV></DIV><P>The appropriate set-up was not available, so we decided to build our own microscope out of separate optical parts. By discussing our problem with optics- and photonics companies and showing our motivation to solve this challenge, we were able to get all our required components sponsored or borrowed, making the entire set-up nearly cost-free. With our minimal set of available tools, we calculated and designed the optics in such a way that we could image fluorescent cells, while photobleaching was minimized. After days of laser aligning, we managed to do so. More information on the design of this setup can be found on the <B><A href="https://2016.igem.org/Team:TU_Delft/Hardware" target="_blank">hardware page</A></B>.</P><P>Using our custom-built setup, we analysed our ‘Biolaser’-cells, to see if they were able to produce a laser-like emission of light. </P><H3>Experiments and Results </H3><P>The first and foremost experiment to be done with the setup was to image the cells in order to see if the setup works properly and to see whether we can image cells with it. Once we have confirmed that the setup works properly, we can measure the output intensity of the cells to see whether the cells are able to emit laser-like light.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseEin" aria-expanded="false" aria-controls="collapseEin"><DIV class="panel-heading" role="tab" id="headingEin"><H4 class="panel-title">
                                            Imaging our biolaser using our home-built setup
                                        </H4></DIV></A><DIV id="collapseEin" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEin"><DIV class="panel-body"><H3>Introduction</H3><P>Building an optical setup is a very precise work. First, it is important to calculate the positions of all components in such a way that the laser beam will reach your sample, and the light emitted by the sample consequently reaches the detectors of the camera and spectrometer. Once this is done, all optical components are positioned on an optical table. This special table is made to prevent vibrations in you system and has mounting holes so all optical components can be screwed into place. Once the components are positioned on the table, the alignment begins. In this step, the beam coming from the laser is guided throughout the system. By slightly adjusting and repositioning all optical components the light is guided through the components until it reaches the detector of the camera. It is essential that the components are aligned correctly and are free of vibrations, because this could change the path of the light.</P><DIV class="row"><DIV class="col-md-12 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> the design of our custom self-built setup.</FIGCAPTION></FIGURE></DIV></DIV><P>After tens of hours of carefully placing components and aligning the light through the setup, we managed to direct the light from the laser, through all components onto the detector of the camera. However, this does not necessarily mean that when we add fluorescent cells to the setup it we are able to image the cells with the setup. Any error in the setup could cause it not to work. Therefore, we first had to confirm whether our setup was working.</P><H3>Methods</H3><P>In order to confirm whether the setup was working, we used <I><I>E. coli</I></I> BL21 cells that were transformed with our constitutive mCerulean BioBrick. In <A href="#fluorophores" target="_blank"><B>previous fluorescent plate reader experiments</B></A>, we have confirmed that these cells are able to fluoresce and that they can be excited at 405nm, the wavelength of our laser. To make sure the only output we were measuring was fluorescence, and not any ‘leakage’ of light from our laser beam, we also tested cells that were not transformed with the mCerulean BioBrick. These cells are not able to fluoresce after excitation at 405nm, so if the setup is working properly we should not get a signal from these cells.</P><P>The cells were adhered to the microscope slides using 3% agarose pads and imaged at an excitation intensity of 0.5 mW. This energy is low enough to not instantly photobleach the proteins, but observe fluorescence clearly.  Focusing on the cells was done manually with a 50x oil-immersed objective. The cells were excited with a Coherent OBIS LX 405nm laser and images were taken using a DeltaPix Invenio III CCD camera.</P><H3>Results and discussion</H3><P>Imaging the cells with the setup yielded the following results: </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B><I>E. coli</I> cells transformed with (A)OmpA-silicatein fusion plasmid and (B) mCerulean, imaged in our custom self-built optical setup. The cells were excited with a laser at a wavelength of 405nm and an intensity of 0.5 mW. </FIGCAPTION></FIGURE></DIV></DIV><P>As we can see in figure 2B, the cells transformed with a gene for the fluorescent protein mCerulean are clearly visible. When using a strain transformed with a plasmid that is not known to cause the cells to fluoresce, in this case OmpA-silicatein, no light was observed, as shown in figure 2A. To confirm that this sample did contain cells, we also inspected the sample with light microscopy, as shown in figure 3. </P><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 3:</B><I>E. coli</I> cells transformed with the OmpA-silicatein fusion plasmid imaged in a widefield microscope to confirm the presence of cells. </FIGCAPTION></FIGURE></DIV></DIV><P>In figure 3, we see that the negative control did contain cells, but we could not observe a fluorescent signal. From this we can conclude that we successfully built a setup that is able to observe and measure fluorescence in a cell. There is no leakage of light of our excitation laser in the camera, since we do not observe anything when we use non-fluorescent cells. Also at a higher excitation energy (50 mW) we did not observe anything on the camera. Therefore we can conclude that our setup works as expected as it is indeed able to measure fluorescence without measuring other light sources. </P><P>For this experiment we used <I>E. coli</I> transformed with OmpA-silicatein that was induced and incubated in silicic acid, so it would contain the polysilicate layer. We deliberately used this strain to test whether the polysilicate or the cells had any autofluorescence that could interfere with our laser experiments. We did not observe any fluorescent signal for these cells, so we can conclude that the polysilicate layer does not have any autofluorescence at 405 nm. Therefore, these cells are suitable for the laser experiments.</P><A href="#Biolaser" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#Biolaser').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseZwei" aria-expanded="false" aria-controls="collapseZwei"><DIV class="panel-heading" role="tab" id="headingZwei"><H4 class="panel-title">
                                            Intensity measurements of laser cells
                                        </H4></DIV></A><DIV id="collapseZwei" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingZwei"><DIV class="panel-body"><H3>Introduction</H3><P>One of the biggest differences between a laser and fluorescence is the amount of emitted photons. We can measure the amount of emitted photons by measuring the intensity of emitted light. The intensity of fluorescence increases linearly with the excitation energy. However, at a certain excitation energy it will reach the so-called ‘laser threshold’ and stimulated emission, and thus lasing will occur. From this point on, the output intensity of the fluorophores will still increase linearly, but with a much steeper slope, as shown in figure 1.</P><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> the relation of input energy vs. output intensity for fluorescence and lasing <A href="#references">(Fan &amp; Yun, 2014)</A></FIGCAPTION></FIGURE></DIV></DIV><P>We investigated whether we could find the same relation between input and output intensity for our Biolaser cells, using our self-built setup, to investigate whether our cells were able to emit a kind of laser light.</P><H3>Methods</H3><P>The plasmids and conditions used for this experiment are summarized in table 1.</P><FIGURE><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGCAPTION><B>Table 1:</B> Plasmids transformed into <I>E. coli</I> strain BL21 with conditions and functions used for the intensity measurements in laser cells.</FIGCAPTION></DIV></DIV><CENTER><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Plasmid(s) and conditions</TH><TH>Function</TH></TR></THEAD><TBODY><TR><TD> mCerulean (constitutively expressed) </TD><TD>Fluorescence (negative control)</TD></TR><TR><TD> mCerulean (constitutive) + OmpA-silicatein (induced, incubated in silicic acid)</TD><TD>Biolaser</TD></TR><TR><TD> OmpA-silicatein (induced, incubated in silicic acid)</TD><TD>Negative control (no fluorescence)</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>The cells were adhered on a microscope slide using 3% agarose pads. The cells were excited at a wavelength of 405 nm. The cells were imaged at excitation energies of 0.1 mW, 0.5 mW, 0.7 mW, 1 mW, 2 mW, 5 mW and 10 mW. These images were analysed using ImageJ to determine the output intensity and corrected for background noise <A href="#references">(McCloy et al., 2014)</A>. </P><H3>Results and discussion</H3><P>The output intensities of our cells were plotted against the excitation power to determine whether our cells emitted laser-like light. The results are shown in figure 2. </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B> Intensity measurements of cells transformed with mCerulean or mCerulean and OmpA-silicatein, we show two duplicate experiments per strain. The measured emission intensity was plotted against the excitation power (black dots) along a prediction of the increase of intensity for fluorescence (blue line) and lasing (red line). Left of the intensity graphs, a picture of the imaged spot is shown.</FIGCAPTION></FIGURE></DIV></DIV><P>In figure 2, the measured intensity is plotted against the excitation power (black dots). Since we expect a linear relation, the expected increase of intensity of fluorescence (blue line) and the expected increase of intensity for lasing (red line) was also plotted. The expected fluorescence is fitted to the fluorescence of the mCerulean, the expected lasing pattern is estimated from this. We show two duplicate experiments per strain. Comparing the measurement data to the expected relations, we cannot see a convincing result that our cells are lasing. Above an excitation power of 2 mW we see that in all cases the fluorescence stays at the same level. This means that the intensity is not increasing anymore and we believe that we observe photobleaching. So, both the fluorescent cells as well as the laser cells do not emit any laser-like light. For the fluorescent cells that were transformed with solely mCerulean, this result is as expected, these cells do not have any extra modifications that should cause them to emit laser light. The strain that was transformed with both mCerulean and OmpA-silicatein had a glass shell around the cell that could cause the cells to emit laser light. This effect was not observed. The strain transformed with solely OmpA-silicatein did not emit any light, as expected. </P><P>From this experiment we cannot see any clear proof of lasing in our cells. This could be both due to our setup or due to the constructed cells. Previous studies show that lasing usually occurs at an excitation power under 1mW <A href="#references">(Gather &amp; Yun, 2011; Fan &amp; Yun, 2014)</A>. However, up to this excitation power the Biolaser cells follow the same slope as the fluorescent cells, which means that no lasing occurred. From 2 mW and higher the cells most likely bleach. Since we expected the cells to lase at an excitation energy under 1 mW, we did not take such high energies into account while designing our setup. Probably we exposed the cells to the excitation laser for a too long time, or the power was probably too high that the cells eventually photobleached anyway. It is also possible the maximum fluorescence intensity of the cells was reached, and therefore the intensity doesn't increase. </P><P>Even though the excitation laser eventually most likely bleached or saturated the fluorophores, this was probably not the reason why lasing didn’t work. Lasing should have occurred at an excitation energy between 0 and 1 mW, and at these energies the fluorophores did not bleach. It is possible that our cells were lasing at the low energy prior to bleaching or saturation, but the system was not sensitive enough to measure this. However, we did find a more probable reason why the cells did not lase. <B><A href="https://2016.igem.org/Team:TU_Delft/Model">Modeling</A></B> showed, that in a cavity as big as <I>E. coli</I> (around 1 µm) we need an intracellular fluorophore concentration of 0.1 M to get lasing. In our cells the maximum concentration we can achieve is in the nM to µM range and therefore lasing physically not possible in our cells. To get lasing, we would either need much larger cells or a much higher concentration of fluorophores. Therefore, it was most likely not due to the self-built setup nor our synthetic biology design of a biolaser that we did not observe lasing.</P><A href="#Biolaser" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#Biolaser').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV></DIV><H3> Discussion and Conclusions </H3><P> We have successfully calculated, designed and built a custom optical setup that could allow us to measure lasing in cells. Using this self-built setup, we were able to image fluorescent cells and measure the intensity of the fluorescence. We have <STRONG>confirmed that the setup worked</STRONG>. We also measured whether lasing would occur in our fluorescent mCerulean polysilicate-covered cells. Despite the effort, we did not observe any lasing in cells. <STRONG><A href="https://2016.igem.org/Team:TU_Delft/Model">Our</A> models have shown that this is due to the size of our cells</STRONG>, if we would use a bigger organism, e.g. a mammalian cell line, we could be able to measure lasing. Unfortunately, we were not able to measure this in our setup, since we did not have the safety permit to use mammalian cells. However, this would be a very interesting future study.</P></DIV></DIV><DIV class="row streepje "><H2 class="title-style-2 col-md-offset-1">Engineering biological lenses</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><H3>Introduction</H3><P>Previously we have shown that <A href="#silicatein" target="_blank"><B>we were successfully able to express the enzyme silicatein</B></A> fused to the membrane protein OmpA in <I>E. coli</I>. These cells were able to synthesize a layer of polysilicate around the membrane of the cells. These polisilicate-covered cells could improve the capturing of light in microscopy in two ways: they could turn cells into lasers or they can make the cells act as microlenses. The lasing capacity of the cells were researched in our custom-built setup. The ability of our cells to act as a lens was also researched. We have already measured the stiffness of our lenses and imaged the shape, and it turned out that the polysilicate-covered cells were more stiff without disturbing the shape of the cell. However, before we can use the polysilicate-covered cells as a lens, we have to determine a few properties first. <A href="https://2016.igem.org/Team:TU_Delft/Practices#experts" target="_blank"><B>From our conversation with Nikon</B></A> we learned that the polysilicate cells should be as homogeneous as possible. Different cells should have as little variation as possible. Therefore, we researched if we are able to control the cell shape of our Biolenses using the BolA gene. Furthermore, the <A href="https://2016.igem.org/Team:TU_Delft/Practices#riskas"><B>risk assessment of our human practices part</B></A> showed that we cannot bring our cells outside the lab, since they have an antibiotic resistance marker. Therefore, we needed to develop a way to sterilize the Biolenses without disturbing their optical properties. </P><P>Even then, our microlenses are not ready to apply onto solar cells yet, because among others we are not yet able to make a good array out of the biolenses. However, we could already test the effect of our cells on the absorbance of a real solar cell. We should make shure that our Biolens cells do not absorb any light, because this would affect a solar cell in a negative way. Therefore, to make sure that we could use our cells on solar cells in the future, we tested the absorbance of our Biolenses in a real-world setup, by coating solar cells with Biolenses and testing them in a solar cell simulator.</P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> By coating cells in polysilicate using the enzyme silicatein, the bacteria might function as a lens.
                                    </FIGCAPTION></FIGURE></DIV></DIV><H3>Experiments and Results</H3><P>In order to confirm whether we can influence cell shape, we overexpressed the gene BolA and investigated the change of phenotype under a widefield microscope and SEM. Also, we investigated whether the cell shape changed and whether we were able to produce homogeneous cells. Then, before we could use our cells in a real-world setup, we had to find a way to sterilize the biolenses, since we could not take the cells out of the lab. Important here is that the technique was non-invasive, since the optical properties of the cells should not be disturbed. We experimented with autoclaving and UV sterilization. After we had found a way to sterilize the lenses, we tested them under real-world conditions. We applied the cells on solar cells to see whether this had any impact on the efficiency of the solar cells. We did this in a solar simulator to simulate real-world conditions as good as possible.</P><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseEen" aria-expanded="false" aria-controls="collapseEen"><DIV class="panel-heading" role="tab" id="headingEen"><H4 class="panel-title">
                                            Influencing cell shape for round lenses
                                        </H4></DIV></A><DIV id="collapseEen" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingEen"><DIV class="panel-body"><H3>Introduction</H3><P>When making biological lenses, the shape of the lens is of crucial importance. <I>E. coli</I> is a rod-shaped organism, so it’s not symmetrical along all axes. Shining light on the round parts of <I>E. coli</I> has a different effect on the focusing of light than shining light on the long sides, see figure 1. More information on this can be found on the <A href="https://2016.igem.org/Team:TU_Delft/Model#lenses" target="_blank"><B>modeling page</B></A>.</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Figure 1:</B> Different ways of focusing light by rod-shaped lenses and spherical lenses. The rod-shaped lens has various directions in which it can break the light, therefore the orientation of this lens is extremely important. The spherical cell only has one orientation, so the orientation of the cell does not matter. We do, however, observe some spherical aberations in our model for spherical lenses.</FIGCAPTION></DIV></DIV></FIGURE><P>When we want to use our microlenses in more advanced optical systems, such as microscopes or cameras, we need to make sure that the variation between the different lenses is minimized. Manufacturers of optical systems do not accept a high aberration between different lenses, so it’s crucial for us to be able to control the shape of our lenses. We have decided to engineer <I>E. coli</I> in such a way that it becomes spherical. This way we are able to create spherical lenses. Apart from the fact that it is crucial to be able to control cell shape, round cells offer the advantage of being symmetrical along all axes, so the orientation of your lens does not matter for the optical properties.</P><P>In order to create spherical <I>E. coli</I>, we overexpress the <I>BolA</I> gene. <I>BolA</I> is a gene that controls the morphology of <I>E. coli</I> in the stress response <A href="#references">(Santos, Freire, Vicente, &amp; Arraiano, 1999)</A>. By overexpressing this gene at 37°C, the rod-shaped <I>E. coli</I> cells, will become round <A href="#references">(Aldea, Hernandez-Chico, De La Campa, Kushner, &amp; Vicente, 1988)</A>. We will express this gene both under a constitutive promoter (Part <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1890031">K1890031</A></B>), as well as an inducible promoter (Part <B><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1890030">K1890030</A></B>). When we express both the <I>BolA</I> gene as well as silicatein, we are able to construct round cells, coated in glass.</P><H3>Methods</H3><P>The expected phenotype of the cells expressing BolA is  very different from the phenotype of wildtype <I>E. coli</I>. If the gene is successfully overexpressed, the cells become round, which we can easily observe under a widefield microscope. In widefield microscopy, the whole sample is simultaneously illuminated using a white light source so the phenotype of the sample can be inspected. This is comparable to normal light microscopy. </P><P>In order to obtain round cells we tested transforming <I>E. coli</I> BL21 with <I>BolA</I> under both a constitutive promoter (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_J23100">J23100</A>) and an inducible promoter (Lac-promoter), to do so it was cloned in a backbone containing the promoter and all machinery necessary for it to work. This backbone was obtained from pBbA5c-RFP, a gift from Jay Keasling (Addgene plasmid # 35281) (<A href="#references">Lee <I>et al.</I>, 2011</A>). Furthermore, we tried if transforming a strain with both <I>BolA</I> and the OmpA-silicatein fusion plasmid yielded round, glass covered cells. The strains and conditions tested are summarized in table 1.</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Table 1:</B> Strains and conditions tested to see whether we produced round-shaped cells.</FIGCAPTION></DIV></DIV><CENTER><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Plasmid(s)</TH><TH>IPTG</TH><TH>Silicic acid</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn"> OmpA-silicatein (inducible)  </TD><TD>+</TD><TD>+</TD></TR><TR><TD class="firstcolumn"> Lac-<I>BolA</I> (inducible)  </TD><TD>+</TD><TD>-</TD></TR><TR><TD class="firstcolumn">J23100-<I>BolA</I> (constitutive)  </TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">OmpA-Silicatein (<I>T. aurantia</I>) + Lac-<I>BolA</I> (inducible)  </TD><TD>+</TD><TD>+</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>The cells were heat-fixed on a slide and observed under the widefield microscope at 37°C. Furthermore, imaged <I>E. coli</I> with and without the BolA-gene (induced) under SEM. The samples were fixed with gluteraldehyde and imaged with SEM. We used an FEI Niva Nano 450 SEM, under high vacuum. </P><H3>Results and discussion</H3><P>The four different strains were imaged under the widefield microscope, the taken images are shown in figure 2.</P><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B> Widefield images of <I>E. coli</I> BL21 transformed with <B>(A)</B> the OmpA-silicatein gene, <B>(B)</B><I>BolA</I> under the inducible Lac promoter and induced with 1 mM IPTG, <B>(C)</B><I>BolA</I> under the constitutive promoter J23100, <B>(D)</B><I>BolA</I> and OmpA-silicatein fusion under the inducible Lac promoter and induced with 1 mM IPTG.  </FIGCAPTION></FIGURE></DIV></DIV><P>In figure 2, the widefield images of the four tested strains are shown. In figure 2A we can see cells with the characteristic wildtype <I>E. coli</I> phenotype; the cells are rod-shaped. Figure 2B shows that induction of the cells transformed wil <I>BolA</I> under the inducible Lac-promoter indeed has changed the phenotype of the cell. The cells have clearly become spherical. Constitutive expression of the <I>BolA</I> gene, as shown in 1C has the opposite result: the cells are rod-shaped and elongated. This is probably because of the stress the plasmid puts on the cells. As mentioned before,  <I>BolA</I> is a gene involved in the stress response of <I>E. coli</I> that changes the morphology of the cells. A too high expression of the gene could therefore change the morphology in an unexpected way, such as elongation, a phenomena that is often observed in <I>E. coli</I><A href="#references"> (Höltje, 1998)</A>. Since we require sphere-shaped cells, constitutive expression of the <I>BolA</I>-gene is not desired. Co-expression of the OmpA-silicatein fusion plasmid and the inducible <I>BolA</I> plasmid also yielded round cells, as seen in figure 2D.</P><P>To get a more detailed view of the shape of the cells transformed with <I>BolA</I>, we also imaged the cells using SEM, the results are shown in figure 3.
                                        </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 3:</B> SEM images of <B>(A)</B><I>E. coli</I> BL21 without the <I>BolA</I> gene, <B>(B)</B><I>E. coli</I> transformed with the <I>BolA</I> gene. </FIGCAPTION></FIGURE></DIV></DIV><P>In figure 3, we see that the cell shape of the cells transformed with <I>BolA</I> is indeed more spherical than the wildtype phenotype of <I>E. coli</I>. Also, we see that the size of the cells increased. Using ImageJ we analyzed the average size of 10 cells (table 1).</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Table 2:</B> Average sizes of a population of untransformed <I>E.coli</I> and <I>E. coli</I> transformed with BolA. We measured 10 cells for each case, the size was determined using ImageJ. </FIGCAPTION></DIV></DIV><CENTER><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Phenotype</TH><TH>Average length (µm)</TH><TH>Average diameter (µm)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn"> Wildtype  </TD><TD>1.2 +/- 0.12</TD><TD>0.5 +/-0.04</TD></TR><TR><TD class="firstcolumn"> BolA  </TD><TD>1.0 +/- 0.15</TD><TD>0.87 +/- 0.13</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>In the table we see the average diameter of the cells increased. The average size of microlenses is in the order of tens or hundreds of micrometers, because with current techniques it's difficult and costly to produce smaller microlenses <A href="#references">(Krupenkin, Yang, &amp; Mach, 2003)</A>. The average sizes of photodetectors are in the nano- to micrometer range <A href="#references">(Yang, Shtein, &amp; Forrest, 2005)</A>. Our spherical microlenses approach this size, and are therefore useful since you could place one microlens over one photovoltaic cell, increasing the efficiency of the photovoltaic cell.</P><P>So, by inducing the expression of <I>BolA</I>, we are indeed able to control the shape of <I>E. coli</I> and turn the cell into a sphere. Also, transforming a cell with both OmpA-Silicatein and <I>BolA</I> yields round cells, so the formation of the glass layer does not distort the cell shape. We have also shown that even though cell size increases upon expression of <I>BolA</I>, the size of our biolenses is smaller than the size of conventional microlenses. People have not been able to produce such small lenses in a cost-effective way, so our microlenses could be a possible solution to this. </P><P>Being able to control cell shape is of major importance if we want to create biological lenses, since the lenses are desired in various sizes and shapes. Especially spherical lenses are useful since they are symmetrical and therefore do not require a specific orientation; they will focus the light in the same way whatever their orientation is. From figure 2 we can see that the cells are not perfectly homogeneously shaped, there is some variation between the shape of the different cells. This variation is even clearer for the cells that contain the polysilicate layer. This is possibly because two plasmids with a lac promoter put stress on the cells, resulting in a greater variation. For precision optics, it is extremely important that there is little to none variation between the lenses. Therefore, it’s recommended to do more research in controlling cell shape. However, since there is always a variation in gene expression between cells it will be wiser to conduct research into selecting or sorting the cells on their size or shape. A possible solution could be sorting the cells using FACS. As a future experiment, we could stain the polysilicate-coated cells with Rhodamine 123 and let a FACS-machine sort the cells on their phenotype. </P><A href="#Biolenses" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#Biolenses').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwee" aria-expanded="false" aria-controls="collapseTwee"><DIV class="panel-heading" role="tab" id="headingTwee"><H4 class="panel-title">
                                            Sterilization of the biolenses
                                        </H4></DIV></A><DIV id="collapseTwee" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwee"><DIV class="panel-body"><H3>Introduction</H3><P>We wanted to test our biolenses in a real-world application: solar cells. This was chosen, since it was shown to make solar cells more efficient while using microlenses <A href="#references">(Jutteau <I>et al.</I>, 2015)</A>. Solar panels are currently unable to use all available solar energy, since parts of the light is lost due to non-focused beams. To decrease the losses, light needs to be focused, which can be done by lenses. Jutteau had shown already that solar panels are increased in efficiency while using microlenses. However, these microlenses are chemically produced, meaning under harsh environmental conditions. A high temperature combined with low pressures and harmful chemicals to produce these microlenses do not contribute positively in the total image of “environmentally friendly”. </P><P>In our opinion, the eventual product – microlenses – was promising. However, the production method did not add up with the purpose of increasing the potential of solar energy and thereby making general energy use more environmentally friendly. Our biologically produced <I>E. coli</I> cells covered in polysilicate were produced under physiological mild conditions (37 ºC, 1 atm and no harsh chemicals) and would fit much better in the image of environmental friendliness. </P><P>To put our cells to the test and see if these are just as promising in increasing the efficiency of solar cells as chemically produced microlenses, we have called for help in the department Photovoltaic Materials and Devices at the TU Delft. Here, prof. Dr. Arno Smets, had introduced us to Stefaan Heirman, a technician at the same research group. Stefaan helped us in our solar cell experiments. </P><P>Since the to be used solar simulator and all other necessary equipment were at another faculty of the TU Delft, our cells had to leave the ML-1 lab. Here, safety regulations from the executed practices were incorporated in the labwork, since the biolenses should be outside the lab – into the environment as well as soon as they could be used on solar panels. Even though <A href="#headingCinco"><B>the growth study of our in polysilicate covered <I>E. coli</I> cells</B></A> did not show any growth anymore after 1 hour, due to safety regulations we have decided to used an additional method to sterilize our biolenses. </P><P>To sterilize, different options are available. For example alcohols, chlorine and chlorine compounds are used within healthcare and autoclaving is a much-used method in laboratory settings. Besides that, UV sterilization is a much-used method in water treatment <A href="#references">(Chang <I>et al.</I>, 1985; Hijnen <I>et al.</I>, 2006; O’Flaherty <I>et al.</I>, 2016)</A>. This is used to remove among others <I>E. coli</I>. Since we do not want any chemicals to potentially interfere with our biolenses, we have chosen not to use any of these chemicals and to test whether autoclaving or UV sterilization would be a good option.</P><H3>Methods</H3><H4>Autoclaving biolenses</H4><P>Aliquots of <I>E. coli</I>cells transformed with OmpA-silicatein were induced and complemented with silicic acid and subsequently autoclaved for 20 minutes at 121 ºC before imaged with scanning electron microscopy (SEM). </P><H4>UV sterilization</H4><P>For this experiment, <I>E. coli</I> cells transformed with BolA pBbA5c were used, as the results of the growth experiments just showed that cells covered in polysilicate were not viable and we want a positive (absolutely living) control.</P><P>A volume of 30 µL <I>E. coli</I> BL21 cells that were transformed with BolA in pBbA5c was plated on LB plates complemented with chloramphenicol. As controls for sterilization aliquots of 500 µL of the same cells were either heated at 95 °C for 10 minutes or washed with 500 µL 70% ethanol for three times. For UV sterilization, light bulbs emitting 254 nm were used and samples were exposed to it for 3 minutes. After sterilization, also 30 µL of these cells were plated on LB+Cm plates. Besides that, 30 µL of non-sterilized cells were plated as well. Of the last mentioned cells, it was expected for them to be alive and grow after overnight incubation. </P><P>Two of the plates containing BolA transformed cells were UV sterilized in a Stratagene UV Stratalinker 1800. The manufacturer recommended power settings of 120000 µJ. In our experiment we have used a power of 120000 µJ and 240000µJ to sterilize our cells. </P><P>All plates were incubated overnight at 37 °C.</P><H3>Results</H3><H4>Autoclaving</H4><P>Cells containing the OmpA-silicatein gene were autoclaved and subsequently imaged with SEM. </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGCAPTION><B>Figure 1:</B><I>E. coli</I> BL21 cells transformed with OmpA-silicatein, provided with silicic acid and were autoclaved and subsequently imaged by with SEM.
                                                    </FIGCAPTION></DIV></DIV><P>As can be seen in figure 1, the cells were collapsed and lost their shape. Therefore they cannot function as biolenses, another sterilization method was used. </P><H4>UV sterilization</H4><P>To test the effectivity of UV sterilization on our <I>E. coli</I> cells, a growth study after UV sterilization was performed. For this 30 µL BL21 cells transformed with BolA in pBbA5c was plated on LB plates complemented with chloramphenicol. These plates were UV sterilized in a Strategene UV Stratalinker 1800. As positive controls the same cells were heated for 10 minutes at 95 ºC or washed three times with 500 µL ethanol. As negative control the same cells were plated on a selective plate. </P><FIGURE><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGCAPTION><B>Table 1:</B> Results of the growth study after UV sterilization. </FIGCAPTION></DIV></DIV><CENTER><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Cell strain</TH><TH>Treatment</TH><TH>Number of colonies formed</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="5">BolA (pBbA5c)</TD><TD>UV sterilization: 120000 µJ</TD><TD>none</TD></TR><TR><TD>UV sterilization: 240000 µJ</TD><TD>none</TD></TR><TR><TD>Heated 10 min at 95 ºC</TD><TD>none</TD></TR><TR><TD>Washed in ethanol</TD><TD>none</TD></TR><TR><TD>none</TD><TD>inf</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>None of the sterilized plates showed any growth, while the non sterilized plate did yield colonies.</P><P>Now it was shown that the <I>E. coli</I> cells are not viable after UV sterilization, we also wanted to see the influence of UV sterilization on the physical properties of the polysilicate layer. Since the polysilicate layer already ensured unviability of the cells, UV sterilization could be seen as an extra safety precaution to compline to current synthetic biology safety regulations. For this, we have UV sterilized <I>E coli</I> cells transformed with OmpA-silicatein, using 120000 µJ for 3 minutes and these were subsequently imaged with SEM (figure 2).
                                        </P><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 2:</B><I>E. coli</I> BL21 cells transformed with OmpA-silicatein, provided with silicic acid and were UV sterilized and subsequently imaged by with SEM. </FIGCAPTION></FIGURE></DIV></DIV><P>As shown in figure 2, the with silicate capturing of the cells remain undamaged after UV sterilization and could be used as biolenses now. </P><H3>Discussion</H3><P>In this experiment, different methods to sterilize our biolenses were tested, as we wanted to be able to put our biological microlenses in real-world conditions. For this, we have both autoclaved and UV sterilized our cells. From the growth study, it can be concluded that the biolenses were successfully sterilized by UV sterilization. Besides that, the shape of the biolenses was not affected by UV sterilization as opposed to autoclaving. </P><A href="#Biolenses" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#Biolenses').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseVier" aria-expanded="false" aria-controls="collapseVier"><DIV class="panel-heading" role="tab" id="headingVier"><H4 class="panel-title">
                                            Absorbance spectroscopy of our biolenses
                                        </H4></DIV></A><DIV id="collapseVier" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingVier"><DIV class="panel-body"><H3>Introduction</H3><P>One of the most common and useful methods of characterization of optical materials is done by spectroscopy. Using spectroscopy we can measure how much light can be transmitted through materials. The device used for those measurements is called spectrometer because it scans the electromagnetic spectrum and detects how much of the initial radiation passes through the materials, which are in our case our samples. For those measurements samples are loaded on glass plates and dried overnight, forming a solid material, then a beam of light is shined through it. 
                                            The machine measures which percentage of the light is absorbed by the sample. If we want to use our polysilicate-covered cells in an optical device or on a solar cell it is important to know whether the bacteria absorb light. If they do, they will significantly decrease the efficiency of the solar cells or optical device. Since our goal is to capture more light, we need to make sure that the cells do not block any light.</P><FIGURE><DIV class="row"><DIV class="col-md-6 col-md-offset-3 col-sm-12"><FIGCAPTION><B>Table 1:</B> Cells tested in the ‘real-world’ solar setup.</FIGCAPTION></DIV></DIV><CENTER><TABLE class="protocols table table-style-1"><THEAD><TR><TH>Plasmid(s)</TH><TH>IPTG</TH><TH>Silicic acid</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn"> OmpA-Silicatein </TD><TD>+</TD><TD>-</TD></TR><TR><TD class="firstcolumn"> OmpA-Silicatein  </TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">BolA + OmpA-Silicatein </TD><TD>+</TD><TD>-</TD></TR><TR><TD class="firstcolumn">BolA + OmpA-Silicatein </TD><TD>+</TD><TD>+</TD></TR><TR><TD class="firstcolumn">OmpA-Silicatein </TD><TD>+</TD><TD>+</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>The bacteria tested (table 1) were killed using UV sterilization and placed on glass pieces to dry overnight. Since our biolenses were placed on a glass plate instead of directly applied to the solar cell, another parameter was added: the glass plate could potentially also influence the spectroscopy measurements. Taking this into account, a control measurement of a clean glass plate had to be performed and then subtracted from the measurements of our biological samples. This way we could measure the effect of our lenses on the light  detected by the solar cells, which we call the ‘spectral response’. The spectral response was scanned from 300 nm to 1200 nm, since this is the range of wavelengths measured in the solar cells. Because we are not so interested about the exact spectral response in each wavelength we averaged the transmission data. </P><H3>Results &amp; discussion</H3><P>The amount of light that was absorbed by our cells was measured, as well as the absorbance of the substrate - a glass slide - 
                                            on which our cells were placed and dried. The absorbance of the substrate was subtracted from the measurements done with our cells 
                                            to get the absorbance of solely our biolenses. The amount of light that is transmitted by our cells is presented in figure 1 below.</P><DIV class="row"><DIV class="col-md-12 col-sm-12"><FIGURE><FIGCAPTION><B>Figure 1:</B> Percentage of light that is transmitted by our cells. The  transmittance was measured using spectroscopy, at a wavelength varying from 300 nm to 1200 nm.</FIGCAPTION></FIGURE></DIV></DIV><P>Figure 1 demonstrates how much light is transmitted by our cells. The light that is not transmitted is either absorbed or reflected. 
                                            Nevertheless, from figure 1 we can see that the light that was transmitted through our cells is almost 100%. </P><P>We do not see a significant difference between polysilicate-coated cells and uncoated cells. The intention of the polysilicate layer was to 
                                            focus the light, not to increase transmittance through the cells. However, due to the extra layer it was hypothesized that our biolenses 
                                            would absorb more light compared to uncoated cells. The fact that we see no significant difference means that the polysilicate layer does 
                                            not have a detrimental effect on the optical properties of the cells.</P><P>Concluding, the spectroscopy experiments showed that our biolenses almost have no influence on the amount of light that was passed through. More than 99% of the initial light is transmitted through our microlenses, making them a good option for any kind of optical application. Moreover, the results obtained from this experiments verify the assumption made in the <A href="https://2016.igem.org/Team:TU_Delft/Model#lenses" target="_blank"><B>modeling of microlenses</B></A> that we have non-absorbing materials, so the imaginary part of the refractive index was set to zero. Furthermore, it verifies the <A href="https://2016.igem.org/Team:TU_Delft/Model#lenses" target="_blank"><B>scattering model</B></A> where we concluded that we have hardly any backscattering from our devices. </P><A href="#Biolenses" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#Biolenses').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseDrie" aria-expanded="false" aria-controls="collapseDrie"><DIV class="panel-heading" role="tab" id="headingDrie"><H4 class="panel-title">
                                            Characterizing Biolenses on solar cells
                                        </H4></DIV></A><DIV id="collapseDrie" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingDrie"><DIV class="panel-body"><H3>Introduction</H3><P>Solar panels are used to convert the potential of solar energy into electrical energy. A lot of innovation had already happened in this field, however the efficiencies can still use quite some improvement. As it was stated by <A href="#references">Jutteau <I>et al.</I> (2015)</A>, the efficiency of solar cells could be increased significantly by adding a layer of microlenses: as parts of the available light get lost due to the fact that solar light is parallel and non-focused, the amount of electrical energy that can be retrieved from the sun could be higher. This limitation can be solved by applying a layer of microlenses onto the solar cell that focus the light in the solar panel, as shown by <A href="#references">Jutteau <I>et al.</I> (2015)</A>.</P><P>The microlenses used in the research of <A href="#references">Jutteau <I>et al.</I> (2015)</A> are chemically produced under harsh conditions, such as extremely high temperatures, low pressure and caustic chemicals. This does, in our opinion, not complement the aim of being more environmentally friendly. </P><P>Our <I>E. coli</I> cells covered in polysilicate however, could suppose a biological alternative for these chemically produced microlenses. The biological microlenses, were produced under physiological mild conditions (37 ºC, 1 atm and no harsh chemicals) and would fit much better in the image of environmental friendliness.</P><P>To put our cells to test and see if these are just as promising in increasing the efficiency of solar cells as chemically produced microlenses, we have called for help in the department Photovoltaic Materials and Devices at the TU Delft. Here, prof. Dr. Arno Smets, introduced us to Stefaan Heirman (see <A href="https://2016.igem.org/Team:TU_Delft/Attributions#acknowledgements"><B>acknowledgements</B></A>), a technician at the same research group who helped us in our solar cell experiments. After <A href="#Biolenses"><B>sterilization</B></A>, we compared efficiencies of solar cells using a solar simulator with and without our biologically produced microlenses. </P><H3>Methods</H3><P>There are different ways to measure the output of solar cells with accuracy and reproducibility. The most used characteristic of solar cells is their efficiency (Power Conversion Efficiency or PCE). It is defined as the energy produced by the solar cells with a set amount of light from an external light source. There is a standardized protocol for this type of measurement, enabling researchers to simulate ‘real-world’ conditions and compare different solar cells in different setups.</P><P>In order to achieve these reproducible and accurate ‘real world’ measurements of efficiency, the previously mentioned protocols need to be used, as well as a certain type of equipment. The equipment used to simulate the aforementioned conditions is called a <STRONG>solar simulator</STRONG>, which is basically a big Xenon lamp with well-defined light power output. In order to calibrate the simulator, a ‘standard’ solar cell is used with well-defined properties to measure the exact output of the simulator in each measurement. At last, the tested solar cells must be placed perpendicular to the light with nothing restricting the light path and kept still during the whole testing. Using this set of equipment, we are sure that we conduct all the experiments under the same scientifically accepted conditions so we remove the uncertainty factor from our experimental results.</P><H3>Results and discussion</H3><P>The samples prepared for the spectroscopy measurements of our cells are summarized in table 1.</P><FIGURE><DIV class="row"><DIV class="col-md-8 col-md-offset-2 col-sm-12"><FIGCAPTION><B>Table 1:</B> Samples prepared for the spectroscopy and solar cell measurements.</FIGCAPTION></DIV></DIV><CENTER><TABLE><THEAD><TR><TH>Plasmid(s)</TH><TH>IPTG</TH><TH>Silicic acid</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn"> OmpA-Silicatein </TD><TD>+</TD><TD>-</TD></TR><TR><TD class="firstcolumn"> OmpA-Silicatein  </TD><TD>-</TD><TD>-</TD></TR><TR><TD class="firstcolumn">BolA + OmpA-Silicatein </TD><TD>+</TD><TD>-</TD></TR><TR><TD class="firstcolumn">BolA + OmpA-Silicatein </TD><TD>+</TD><TD>+</TD></TR><TR><TD class="firstcolumn">OmpA-Silicatein </TD><TD>+</TD><TD>+</TD></TR></TBODY></TABLE></CENTER></FIGURE><P>The bacteria were killed using UV sterilization and placed on glass slides to dry overnight. The power conversion efficiency of the uncoated solar cell was measured, as well as the efficiency of the solar cell with an empty glass slide (negative control) and the efficiency of the solar cell with a glass slide coated with bacteria. </P><FIGURE><FIGCAPTION><B>Figure 1:</B> Testing the cells under real world conditions in the solar simulator </FIGCAPTION></FIGURE><H3>Results and discussion</H3><P>The measured power conversion efficiencies of the solar cell and the solar cell with various coatings is shown in figure 2. </P><FIGURE><FIGCAPTION><B>Figure 2:</B> Power conversion efficiencies of the solar cell, the solar cell covered with a glass slide and the solar cell covered with various biolens coatings and controls measured in real-world conditions under a solar simulator. </FIGCAPTION></FIGURE><P>
                                            In figure 2, we see that the uncoated solar cell has a power conversion efficiency of around 15%. Since our cells were affixed on a glass slide, we first had to measure the impact of placing a glass slide on the solar cell, so we can distinguish the effect that our biolenses have on the efficiency from the effect that the glass slide might have. We see that placing a glass slide, the substrate, on the solar cell reduced the efficiency of the solar cell by around 1%. Once we measure the power conversion efficiency with our cells on a glass substrate, we see that the efficiency is not different from the efficiency of the solar cell covered with solely the glass substrate. The efficiency has dropped about 1% compared to the uncovered solar cell, but this is most likely due to the fact that we placed the cells on a glass substrate. 
                                            From this we cannot conclude that adding our cells onto the solar cells has a measurable effect on the efficiency of the solar cells.
                                        </P><P>So why do the microlenses described in literature cause a 20% increase in the efficiency and our cells do not? We think this is most likely due to our experimental setup. When conventional microlenses are placed onto a solar cell, the cells are not placed directly onto the solar cell, but there is a short distance (an ‘optical spacer’) between the microlens and the solar cell <A href="#references">(Dal Zilio <I>et al.</I>, 2008)</A>. This optical spacer is at the distance of the focal point of the microlens, allowing the microlens to focus the light on the solar cell. We however, have directly placed our microlenses onto a glass slide, which was immediately placed on the solar cell. <A href="https://2016.igem.org/Team:TU_Delft/Model/Q5"><B>Modeling of our biolenses</B></A> has shown that our biolenses have a focal point at around 1 µm behind the cell. This means that if we are able to implement an optical spacer of 1 µm, we could focus the light on the surface of the solar cells. We now put the microlenses directly on the surface of the glass slide, which has its own optical properties, therefore we did not focus the light on the solar cell, as shown in figure 3. </P><FIGURE><FIGCAPTION><B>Figure 3:</B> The effect of using an optical spacer - an empty space between the lenses and the solar cell - and using a layer of glass, as we did. The layer of glass disturbs the focal point of the lenses, therefore we are not able to improve the solar cells.  </FIGCAPTION></FIGURE><P>Unfortunately we are not able yet to place our lenses in a structured array that is rigid enough to place it onto a solar cell with an optical spacer in between. This would be an essential future study if we want to further develop a technique where we want to improve solar cells. This is also applicable to our microlenses in optics and microscopy. For now, we have shown that our microlenses do not have any detrimental effects on the efficiency of solar cells. Most likely, this will also be the case for applications of our microlenses in optics. However, when we want to research improving solar cells or microscopy using our biolenses, we should first invest time into developing a technique to make a structured microlens array that is able to be placed onto a solar cell with an optical spacer in between.</P><A href="#Biolenses" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#Biolenses').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV></DIV><H3> Discussion and Conclusions </H3><P>From the experiments where we transformed <I>E. coli</I> with BolA, we can conclude that <STRONG>we were successfully able to control cell shape and produce spherical biological lenses</STRONG>. Biolenses with a spherical phenotype have an advantage over Biolenses with the rod-shaped <I>E. coli</I> phenotype, as for the round lenses, the orientation of the lens does not matter. The spherical cells we produced had an increased diameter compared to wildtype <I>E. coli</I>. The diameter of 1 µm that we observed matches the size of a photovoltaic cell, the smallest unit of a solar panel. Conventional microlenses are usually bigger. Therefore, <STRONG>our method of producing microlenses has an advantage over the conventional production</STRONG>, since we are able to produce far smaller lenses. Smaller lenses also means we can put more lenses on a surface, which increases the focusing effect.</P><P> In high-precision optics, it is important that all microlenses have the same shape and size. Although the size of our lenses was relatively homogeneous, there was still a significant error of 1% between the sizes of the lenses. We could solve this by sorting and selecting cells of various sizes using FACS. This would be a good future study to conduct.</P><P>Before we can use our Biolenses in real-world conditions, we need to sterilize them. The lenses contain a core of live bacteria that are carrying antibiotic resistance. This could cause a serious health risk when released outside the lab. Therefore we have used autoclaving and UV-sterilization to see if we could sterilize the biolenses. We found that <STRONG>we can sterilize the biolenses without disturbing the shape using UV-sterilization.</STRONG></P><P>With our sterilized cells, we could test the cells in real-world conditions. We tested whether our cells absorbed light using spectroscopy. If our cells would absorb light, they would have a negative impact on the solar cells or optics we plan to use them on. In these experiments we have found that the <STRONG>Biolenses do not absorb any light, so they are suitable for usein optics or solar cells</STRONG>. We have also tried whether our microlenses were able to improve solar cells under <STRONG>real-world conditions</STRONG>, by putting them on a solar cell in a solar simulator setup. Unfortunately, we did not see an increase, but we believe this is due to our experimental setup. We were not able to make a microlens array yet, therefore we did not see an increase in efficiency. The next experiment should be designing and testing a way to produce these arrays and place them on a solar cell. However, we did see that the biolenses do not impact the solar cells in a negative way. Therefore, we are confident we are able to increase the efficiency of solar panels once we have produced our microlens arrays. </P></DIV></DIV><DIV class="row"><H2 class="title-style-1">Conclusions and Recommendations</H2></DIV><DIV class="row"><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P>The ultimate goal of our project was to improve imaging techniques by capturing more light. To this end, we came up with two different approaches. The first one was to let the 
                            sample, in this case the bacterium <I>Escherichia coli</I>, emit more light without expressing more fluorophores: the Biolaser. Furthermore, we thought of a way to capture 
                            more light using biological microlenses, also made by <I>E. coli</I>: Biolenses</P></DIV></DIV><DIV class="row"><H2 class="title-style-2 col-md-offset-1">The Biolaser</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P> The laser consisted of two main synthetic biology components: expressing fluorophores and coating the cell in polysilicate with the enzyme silicatein. 
                            We have <A href="#fluorophores" target="_blank"><STRONG>successfully constructed two brand new BioBricks for fluorophore expression</STRONG></A>: mVenus and mCerulean. 
                            Additionally we improved the existing GFP BioBrick by adding a constitutive promoter. We also succeeded in expressing silicatein and locating it on the cell
                            membrane of <I>E. coli</I>. <A href="#silicatein" target="_blank"><STRONG>This enzyme enabled the cell to coat itself in polysilicate</STRONG></A>. We confirmed the presence of the polysilicate 
                            layer with imaging techniques, including AFM and TEM. </P><P>We tested our synthetic biology design in an <A href="https://2016.igem.org/Team:TU_Delft/Hardware" target="_blank"><STRONG>optical setup</STRONG></A> that we specially designed and built to enable lasing in bacteria. Although we successfully 
                            implemented all our synthetic biology components, this did not result in a lasing bacterium. To find out why, we adapted a conventional <A href="https://2016.igem.org/Team:TU_Delft/Model#lasers" target="_blank"><STRONG>laser model to 
                                    fit the characteristics of our biological laser</STRONG></A>. This led us to the conclusion that our design requires either a larger cell volume, or a higher fluorophore concentration. Therefore we recommend repeating this experiment in cells that are typically larger, such as mammalian cells. A previous study showed the potential of mammalian cells as a laser cavity, albeit with artificial fluorescent agents and mirrors<A href="#references">(Humar &amp; Yun, 2015)</A>. The goal of our project was to use a fully synthetic biology approach, therefore it would be interesting to apply this approach on mammalian cells.</P></DIV></DIV><DIV class="row"><H2 class="title-style-2 col-md-offset-1">Biolenses</H2><DIV class="col-md-10 col-md-offset-1 col-sm-12"><P>As stated before, we have successfully covered cells in polysilicate. We additionally engineered this strain by <A href="#Biolenses" target="_blank"><STRONG>changing its morphology </STRONG></A>from rod-shaped 
                            to spherical. By simulating the optical properties of these cells, we investigated whether they could act as a biological microlens.  Our models pointed 
                            out that <A href="https://2016.igem.org/Team:TU_Delft/Model#lenses" target="_blank"><STRONG>our cells are able to focus light</STRONG></A> and therefore act as a lens. To validate that our microlenses do not absorb any light but transmit it, we used 
                            spectroscopy. This test showed our microlenses indeed transmit the vast majority of the light. </P><P>We have discussed the potential of our microlenses with <A href="https://2016.igem.org/Team:TU_Delft/Practices#experts" target="_blank"><STRONG>experts</STRONG></A> in the field of imaging, who saw great potential of the microlenses in a number of applications. 
                            Apart from our original idea of applying the microlenses in optical systems, we also learned the microlenses could <STRONG>improve the efficiency of solar cells</STRONG>. For 
                            this application we have written a <A href="https://2016.igem.org/Team:TU_Delft/Practices#entrepre" target="_blank"><STRONG>business plan</STRONG></A>. One of the possible challenges we encountered implementing our product into a business, was bringing our GMOs 
                            in the outside world. To this end, we have investigated the most feasible way of sterilizing our microlenses so they can be applied on solar cells. </P><P>In our project, we have developed a<STRONG>new application for synthetic biology in the field of optics</STRONG>. Microlenses are a promising new field in optics, but due to
                            their environmental footprint and high costs, they are not widely used. The microlenses we developed could be a promising substitution for the conventional 
                            microlenses. Therefore, more research is necessary to implement our biological microlenses in daily life.</P></DIV></DIV></DIV></DIV><DIV class="references container"><H4 class="footer-title">References</H4><OL><LI>Aldea, M., Hernandez-Chico, C., De La Campa, A., Kushner, S., &amp; Vicente, M. (1988). Identification, cloning, and expression of bolA, an ftsZ-dependent morphogene of Escherichia coli. Journal of bacteriology, 170(11), 5169-5176. </LI><LI>Cha, J. N., Shimizu, K., Zhou, Y., Christiansen, S. C., Chmelka, B. F., Stucky, G. D., &amp; Morse, D. E. (1999). Silicatein filaments and subunits from a marine sponge direct the polymerization of silica and silicones in vitro. Proceedings of the National Academy of Sciences, 96(2), 361-365. </LI><LI>Chang, J. C., Ossoff, S. F., Lobe, D. C., Dorfman, M. H., Dumais, C. M., Qualls, R. G., &amp; Johnson, J. D. (1985). UV inactivation of pathogenic and indicator microorganisms. <I>Applied and environmental microbiology, 49</I>(6), 1361-1365.</LI><LI>Cormack, B. P., Valdivia, R. H., &amp; Falkow, S. (1996). FACS-optimized mutants of the green fluorescent protein (GFP). Gene, 173(1), 33-38.</LI><LI>Curnow, P., Bessette, P. H., Kisailus, D., Murr, M. M., Daugherty, P. S., &amp; Morse, D. E. (2005). Enzymatic synthesis of layered titanium phosphates at low temperature and neutral pH by cell-surface display of silicatein-a Journal of the American Chemical Society, 127(45), 15749–15755.</LI><LI>Curnow, P., Kisailus, D., &amp; Morse, D. E. (2006). Biocatalytic Synthesis of Poly (L‐Lactide) by Native and Recombinant Forms of the Silicatein Enzymes. Angewandte Chemie International Edition, 45(4), 613-616. </LI><LI>Dal Zilio, S., Tvingstedt, K., Inganäs, O., &amp; Tormen, M. (2009). Fabrication of a light trapping system for organic solar cells. Microelectronic Engineering, 86(4), 1150-1154. </LI><LI>Eggeling, C., Widengren, J., Rigler, R., &amp; Seidel, C. (1998). Photobleaching of fluorescent dyes under conditions used for single-molecule detection: Evidence of two-step photolysis. Analytical Chemistry, 70(13), 2651-2659. </LI><LI>Electron microscopy. (2013). Slideshare.net. Retrieved 15 October 2016, from http://www.slideshare.net/suniu/electron-microscopy</LI><LI>Fan, X., &amp; Yun, S.-H. (2014). The potential of optofluidic biolasers. Nature methods, 11(2), 141-147. </LI><LI>Francisco, J. a, Earhart, C. F., &amp; Georgiou, G. (1992). Transport and anchoring of beta-lactamase to the external surface of Escherichia coli. Proceedings of the National Academy of Sciences of the United States of America, 89(April), 2713–2717.</LI><LI>Friel, J. J., &amp; Lyman, C. E. (2006). Tutorial review: X-ray mapping in electron-beam instruments. Microscopy and Microanalysis, 12(01), 2-25.</LI><LI>Gather, M. C., &amp; Yun, S. H. (2011). Lasing from Escherichia coli bacteria genetically programmed to express green fluorescent protein. Optics letters, 36(16), 3299-3301. </LI><LI>Gather, M. C., &amp; Yun, S. H. (2011). Single-cell biological lasers. Nature Photonics, 5(7), 406-410. </LI><LI>Grossiord, N., Kroon, J. M., Andriessen, R., &amp; Blom, P. W. M. (2012). Degradation mechanisms in organic photovoltaic devices. <I>Organic Electronics, 13</I>(3), 432–456. http://doi.org/10.1016/j.orgel.2011.11.027</LI><LI>Hardy, J., &amp; Selkoe, D. J. (2002). The amyloid hypothesis of Alzheimer's disease: progress and problems on the road to therapeutics. Science, 297(5580), 353-356. </LI><LI>Heintzmann, R., &amp; Ficz, G. (2006). Breaking the resolution limit in light microscopy. Briefings in functional genomics &amp; proteomics, 5(4), 289-301. </LI><LI>Hijnen, W. A. M., Beerendonk, E. F., &amp; Medema, G. J. (2006). Inactivation credit of UV radiation for viruses, bacteria and protozoan (oo) cysts in water: a review. <I>Water research</I>, 40(1), 3-22.</LI><LI>Höltje, J.-V. (1998). Growth of the stress-bearing and shape-maintaining murein sacculus of Escherichia coli. Microbiology and Molecular Biology Reviews, 62(1), 181-203. </LI><LI>Humar, M., &amp; Yun, S. H. (2015). Intracellular microlasers. Nature Photonics. </LI><LI>Jonáš, A., Aas, M., Karadag, Y., Manioğlu, S., Anand, S., McGloin, D., Kiraz, A. (2014). In vitro and in vivo biolasing of fluorescent proteins suspended in liquid microdroplet cavities. Lab on a Chip, 14(16), 3093-3100. </LI><LI>Jutteau, S., Paire, M., Proise, F., Lombez, L., &amp; Guillemoles, J. F. (2015, 14-19 June 2015). Micro solar concentrators: Design and fabrication for microcells arrays. Paper presented at the Photovoltaic Specialist Conference (PVSC), 2015 IEEE 42nd</LI><LI>Kim, E.-J., &amp; Yoo, S.-K. (1998). Cell surface display of CD8 ecto domain on Escherichia coli using ice nucleation protein. Biotechnology techniques, 12(3), 197-201.</LI><LI>Krupenkin, T., Yang, S., &amp; Mach, P. (2003). Tunable liquid microlens. Applied Physics Letters, 82(3), 316-318. </LI><LI>Kwakernaak, I. C., &amp; Sloof, I. W. G. (2005). Introduction in the use and application of Scanning Electron Microscopy and Energy Dispersive X-ray Spectroscopy in materials., (November), 1–14.</LI><LI>Lakowicz, J. R. (2013). Principles of fluorescence spectroscopy: Springer Science &amp; Business Media.</LI><LI>Lee, T. S., Krupa, R. A., Zhang, F., Hajimorad, M., Holtz, W. J., Prasad, N., … Keasling, J. D. (2011). BglBrick vectors and datasheets: A synthetic biology platform for gene expression. Journal of Biological Engineering, 5, 12. http://doi.org/10.1186/1754-1611-5-12</LI><LI>Li, C.-W., Chu, S., &amp; Lee, M. (1989). Characterizing the silica deposition vesicle of diatoms. Protoplasma, 151(2-3), 158-163. </LI><LI>McCloy, R. A., Rogers, S., Caldon, C. E., Lorca, T., Castro, A., &amp; Burgess, A. (2014). Partial inhibition of Cdk1 in G2 phase overrides the SAC and decouples mitotic events. Cell Cycle, 13(9), 1400-1412. </LI><LI>Müller, W. E. G. (2003). Silicon biomineralization.</LI><LI>Müller, W. E. G., Rothenberger, M., Boreiko, A., Tremel, W., Reiber, A., &amp; Schröder, H. C. (2005). Formation of siliceous spicules in the marine demosponge Suberites domuncula. Cell and Tissue Research, 321(2), 285–297.</LI><LI>Müller, W. E., Engel, S., Wang, X., Wolf, S. E., Tremel, W., Thakur, N. L., Schröder, H. C. (2008). Bioencapsulation of living bacteria (Escherichia coli) with poly (silicate) after transformation with silicatein-α gene. Biomaterials, 29(7), 771-779. </LI><LI>Müller, W. E. (2011). Molecular biomineralization: aquatic organisms forming extraordinary materials: Springer Science &amp; Business Media.</LI><LI>Nagai, T. et al. A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications. Nat. Biotechnol. 20, 87–90 (2002).</LI><LI>Nam, M., Kim, K., Lee, J., Yang, S. S., &amp; Lee, K.-K. (2013). Solar cell packaged by a microlens array and its long-term optical efficiency enhancement. Paper presented at the Proc. of SPIE Vol.</LI><LI>Nellist, P. D. (2011). The principles of STEM imaging. In Scanning Transmission Electron Microscopy (pp. 91-115). Springer New York.</LI><LI>O’Flaherty, E., Membré, J. M., &amp; Cummins, E. (2016). META-ANALYSIS OF THE EFFECT OF UV WATER TREATMENT ON ESCHERICHIA COLI. <I>BIOSYSTEMS AND FOOD ENGINEERING RESEARCH REVIEW 21</I>, 147.</LI><LI>Rizzo, M. A., Springer, G. H., Granada, B. &amp; Piston, D. W. An improved cyan fluorescent protein variant useful for FRET. Nat. Biotechnol. 22, 445–449 (2004).</LI><LI>Santos, J. M., Freire, P., Vicente, M., &amp; Arraiano, C. M. (1999). The stationary‐phase morphogene bolA from Escherichia coli is induced by stress during early stages of growth. Molecular microbiology, 32(4), 789-798. </LI><LI>Shcherbo, D., Merzlyak, E. M., Chepurnykh, T. V, Fradkov, A. F., Ermakova, G. V, Solovieva, E. a, … Chudakov, D. M. (2007). Bright far-red fluorescent protein for whole-body imaging. Nature Methods, 4(9), 741–746. http://doi.org/10.1038/nmeth1083</LI><LI>Snaith, H. J. (2012). The perils of solar cell efficiency measurements, <I>6</I>(June), 337–340.</LI><LI>Svelto, O., &amp; Hanna, D. C. (1976). Principles of lasers: Springer.</LI><LI>Tvingstedt, K., Dal Zilio, S., Inganäs, O., &amp; Tormen, M. (2008). Trapping light with micro lenses in thin film organic photovoltaic cells. Optics Express, 16(26), 21608-21615. </LI><LI>Yang, F., Shtein, M., &amp; Forrest, S. R. (2005). Controlled growth of a molecular bulk heterojunction photovoltaic cell. Nature materials, 4(1), 37-41. </LI></OL></DIV><FOOTER class="main-footer"><DIV id="contact" class="footer-main"><DIV class="row"><DIV class="col-md-offset-1 col-md-3"><DIV class="footer-col"><H3 class="footer-title">Contact</H3><DIV class="footer-content"><STRONG> Address:</STRONG><UL><LI class="list-unstyled">Faculty of Applied Sciences</LI><LI class="list-unstyled">Room E0.100</LI><LI class="list-unstyled">Van der Maasweg 9</LI><LI class="list-unstyled"> 2629 HZ Delft</LI><LI class="list-unstyled">The Netherlands</LI></UL><STRONG>Email:</STRONG><UL><LI class="list-unstyled">tudelft.igem@gmail.com</LI></UL><STRONG>Social media:</STRONG><UL><LI class="list-unstyled"><A href="https://www.facebook.com/TUDelft.iGEM2016/" target="_blank"> Facebook</A></LI><LI class="list-unstyled"><A href="https://twitter.com/TUDelft_iGEM" target="_blank">Twitter</A></LI><LI class="list-unstyled"><A href="https://www.instagram.com/igemtudelft/" target="_blank">Instagram</A></LI><LI class="list-unstyled"><A href="https://www.youtube.com/channel/UCG_QXJIdl79P_zaHgzO6ppA" target="_blank">Youtube</A></LI></UL></DIV></DIV></DIV><DIV class="col-md-8"><DIV class="footer-col"><H3 class="footer-title">Sponsors</H3></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>