Team:TU Delft/Notebook
tudelft.igem@gmail.com
Toggle navigation
ACHIEVEMENTS
Medal Requirements
Special Prizes
Best Applied Design
Best Hardware
Best Software Tool
Best Entrepreneurship
Best Model
Best Composite Part
Best Integrated Human Practices
TEAM
Team Members
Advisors
Fun Activities
Media
PROJECT
Description
Experiments
Fluorophore expression
Silicatein expression
Biolaser
Biolenses
Conclusions
Conclusions and Recommendations
SAFETY
Workspaces
Safety
NOTEBOOK
Day Notes
Protocols
PRACTICES
Overview
Product Analysis
Risk Assessment Tool
Risk Assessment Discussion
Experts' Opinions
Business Plan
iGEM Analysis
iGEM Toolbox
Outreach
MODELING
Biolasers
Q1: Cavity size
Q2: Fluorophore concentration
Q3: Laser threshold
Q4: Cavity quality factor
Biolenses
Q5: Focusing of light
Q6: Scattering of light
Q7: Influence of polysilicate layer thickness
Theory
What is light?
What is a laser?
Mie Theory
HARDWARE
Overview
Building the microscope
COLLABORATIONS
Collaborations
Meetups
PARTS
All Parts
Composite Parts
Basic Parts
Parts Collection
ATTRIBUTIONS
Team Attributions
Sponsors
Acknowledgements
INTERLAB
Results
Discussion
iGEM TU Delft
Notebook
Day Notes
June
29th June 2016
María
Prepare 10mL of antibiotic stocks of both antibiotics that are going to be used during the project: Chloramphenicol (Cm) and Ampicillin (Amp). Make 500µL and 1mL aliquots.
Prepare 1L of LB medium and 3x400mL bottles of LB Agar. Send to autoclave.
30th June 2016
María
Add antibiotics to the LB Agar bottles prepared the previous day and pour plates to have them ready for when we start transforming.
Back to Top
July
4th July 2016
Lycka
Digestion gBlocks mVenus and mKate with EcoRI and PstI.
Made liquid culture of strain containing pSB4A5 bakcbone with RFP.
María
Digestion of pSB1C3 linearized backbone from the distribution kit with EcoRI and PstI.
Charlotte
Made two liquid overnight cultures of E. coli K12 harboring the standard backbone pSB4A5 containing RFP.
5th July 2016
María
Dissolution of necessary parts from distribution kit: GFP BioBrick BBa_E0840 and promoters BBa_J23100, BBa_J23105, BBa_J23108, BBa_J23113 and BBa_J23117.
Dissolution of K1149051 (a kind gift of Imperial College), which was sent dry on filter paper.
Lycka
Ligation of mKate into backbone pSB1C3.
Transformation of ligation product mKate and the BioBricks dissolved by María into Escherichia coli Top10 cells. These were plated on LB medium supplemented with Cm.
Charlotte
Did 10 minipreps on the overnight culture with RFP pSB4A5, according to protocol.
6th July 2016
Lycka
Stocked primers VF2 and VR: storage stock (100µM) and working stock (10µM).
Colony PCR of transformants from yesterday with primers VF2 and VR followed by gel electrophoresis. Gel picture yielded no bands. This might be attributed to a fault in the transformation.
7th July 2016
María
Transformation of same BioBricks as on 5th July using a different transformation protocol since we believed that the negative result was due to a mistake during transformation.
8th July 2016
María
Finally, the plasmids containing K1149051 and BBa_E0840 yielded a few colonies. However, the ones containing the promoters did not. Thus, we believe that the distribution kit plate containing the promoters is defective and the mKate ligation also featured some problems.
11th July 2016
Lycka
To test whether the distribution kit might be the problem, transformation of BBa_J23113 in OneShot® TOP10 chemically competent cells (Invitrogen) to make sure the transformation process is not the cause of the negative results. Also, a plasmid containing a construct from last year's team (CsgA) was used as a positive control.
For the mKate construct we will obtain the backbone from another construct instead of the linearized backbones from the iGem distribution kit.
12th July 2016
Lycka
Measure DNA concentration of biobricks from registry by nanodrop.
Nanodrop
Product
Concentration (ng/µl)
J23113
89.3
J23117
91.7
J23105
89.3
J23108
92.7
J23100
93.4
E0840
81.9
As can be seen from the table, there is DNA presence in the samples. Therefore, something else should be causing the transformations not to work. Since last year's distribution kit plates are still available in the lab and those plasmids were not taken we will try to get these parts from 2015 distribution kit.
María
Transfer colonies for K1149051 and BBa_E0840 to overnight culture on LB selective medium. A colony containing CsgA was transfered as well to use that plasmid to obtain the standard backbone for all synthesized parts.
13th July 2016
María
Plasmid isolation of overnight cultures prepared yesterday and preparation of K1149051 and BBa_E0840 samples for sequencing.
Lycka
Cryostocks of K1149051 and BBa_E0840 overnight culures. Stored at -80ºC.
Digestion of backbones pSB1C3, pSB4A5 and all gBlocks with EcoRI and PstI.
Lara
Preparation of 2 bottles of LB agar.
Liza
14th July 2016
Lycka
Gel purification of digested backbones. Dissolved in nuclease free water, stored at -20ºC.
Ligation of synthesized fragments into backbones.
María
After all the problems transforming in the past weeks, we tested if we could use our homemade chemically competent cells for future transformations by transforming a couple of positive and negative controls, which were taken from last year's plasmids. The transformations worked so these cells can still be used.
15th July 2016
María
Transformation of Lycka's ligation products from yesterday and the BioBricks from the distribution kit were given one last chance. Transformed cells were plated after doubling their concentration.
Lycka
Dissolution of promoter BioBricks from 2015 iGEM distribution kit.
17th July 2016
Lara
Poured plates using 2 bottles of LB Agar, one for chloramphenicol plates and one for ampicilin.
Iris
Purification IDTs IDT 1-12, 50 µL Mem 20 µL Nuclease free water
18th July 2016
Lycka
Restriction of backbones pSB1C3 and pSB4A5 with EcoRI and PstI. Gel electrophoresis of digested fragments. Gel picture yielded no bands.
Célina
Transformation of plasmids from Interlab Study into BL21.
Plated after doubling the concentration of cells by spinning down and removing half of the supernatant.
19th July 2016
Lycka
Transformation of BioBricks from the distribution kit of 2015, since the ones from 2016 yielded no colonies. Positive control: csgA. Negative control: water. Biobricks: J23100, J23108, J23105, J23117, J23113. Overnight cultivation yielded no result, we decide to drop this and add the promoters to the parts that need it by PCR.
María
Restriction of pSB1C3 and pSB4A5 backbones with EcoRI and PstI
Ligation of synthesized parts that could not be ligated on 14th of July: INP_Sil_Sdom in pSB1C3 and BolA_ind and BolA_con in both pSB1C3 and pSB4A5.
20th July 2016
Lycka and Tessa
Transformation of plasmids ligated by Maria (19th of July) together with the ones ligated on 14th of July. After overnight culture the following plates contained colonies. pSB1C3: INP_Sil_Sdom, OmpA_Sil_Taur, Sil_Sdom, SulA, BolA_ind, BolA_con, phaP. pSB4A5: BolA_con. The ones with a negative result were ligated again the next day.
21st July 2016
María
Colony PCR of yesterday's transformation results from Lycka and Tessa. However, not all transformations yielded colonies.
Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in selective LB overnight.
Lycka
Ligation of mKate, mVenus, mCerulean, OmpA_Sil_Sdom and LacI into pSB1C3. Ligation of OmpA_Sil_Taur, OmpA_Sil_Sdom, Sil_Sdom, SulA and BolA_ind into pSB4A5. Left at room temperature overnight.
22nd July 2016
María
Continue with colony PCR of 20th of July transformation results from Lycka and Tessa.
Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in selective LB overnight.
Lycka
Transformation of ligation products from yesterday. Cells containing backbones pSB1C3 or pSB4A5 were plated on plates supplemented with chloramphenicol or ampicilin, respectively. After overnight cultivation the following plates contained colonies: mKate, mVenus, mCerulean, OmpA_Sil_Sdom, Sil_Sdom, SulA, BolA_ind.
23rd July 2016
María
Continue with colony PCR of yesterday's transformation results from Lycka.
Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in selective LB overnight.
Pick all colonies selected by colony PCR on 21st, 22nd and 23rd of July and transfer to selective LB according to the resistance expressed by each plasmid.
24th July 2016
Lycka
Cryostock and plasmid isolation of overnight cultures prepared yesterday by Maria. Cryostocks stored at -80ºC, isolated plasmids stored at -20ºC.
25th July 2016
Lycka
Nanodrop and prepare for sequencing plasmids isolated yesterday.
Sequencing resulted in the following. Correct sequence: Sil_Sdom (pSB1C3), OmpA_Sil_Taur (pSB1C3), Sil_Sdom (pSB4A5), SulA (pSB4A5). Sequence with mutations: SulA (pSB1C3), BolA_ind (pSB1C3), J23108 (pSB1C3), BolA_con (pSB4A5), BolA_ind (pSB4A5). Different sequence: BolA_con (pSB1C3), INP_sil_Sdom (pSB1C3), OmpA_Sil_Sdom (pSB4A5). Mutated sequences can be repaired by PCR and blunt end ligation.
María
Prepare new digested pSB1C3 and pSB4A5 backbones by restricting with EcoRI and PstI and gel isolating the restricted product.
New ligation of those synthesized fragments that have not yielded any positive results (transformation or sequence confirmation) in the past week in their respective backbones.
26th July 2016
Célina
Transformation of mKate, mVenus, LacI and phaP in pSB1C3 and INP_Sil_Sdom in pSB4A5 into Top10 chemically competent cells. Also transformed one positive control per backbone and negative controls for both antibiotic resistances.                                                 Transformations were plated after increasing the concentration of cells by removing 200µL of medium.
27th July 2016
Lycka
Colony PCR of plates transformed yesterday.
Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated the same day in selective LB overnight.
María and Tessa
Make new chemically competent cells. They were tested for transformation capacity with a positive control plasmid (RFP in pSB4A5). Also, untransformed cells were plated in plates supplemented with Cm or Amp to test whether they had developed a resistance during the process.
28th July 2016
María
Cryostock, plasmid isolation and preparation of samples for sequencing of the overnight cultures prepared yesterday by Lycka. Only INP_Sil_Sdom in pSB4A5 was sequenced confirmed with a mutation to be fixed by PCR. The other sequences aligned to a fragment of GFP unknown to us, we assumed it either came from a contamination of the old restriction enzymes we were using or from the pSB1C3 backbone that was taken from last year's team plasmid stock and probably was poorly labeled. Thus, we decided to use another backbone and the new enzymes sponsored to all iGEM teams by New England Biolabs.
29th July 2016
Lycka and Célina
Repeat of the colony PCR of plates from the 26th of July that yielded a negative result.
Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in LB and the applicable antibiotic overnight.
Transformation of OmpA_Sil_Sdom in both different backbones.
Iris
Isolated plasmid with pSB1C3 backbone containing RFP to be used as positive control in transformations and to obtain the standard backbone by restriction.                                                 Obtained 3 tubes of isolated plasmid from 3mL of culture each.
30th July 2016
Lycka
Colony PCR of OmpA_Sil_Sdom in both backbones from the 29th of July. The PCR machine was filled with colonies from older plates which had not yielded any good colonies yet: mKate (26th of July), mKate (mKate 30th of June), INP_Sil_Sdom (20th of July).
Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in LB and the applicable antibiotic overnight.
Cryostocking and plasmid isolation of colonies picked by on 29th of July. Cryostocks stored at -80ºC, isolated plasmid stored at -20ºC.
31st July 2016
María
Cryostock and plasmid isolation of the overnight cultures prepared yesterday by Lycka and preparation of samples for sequencing of plasmids isolated yesterday and today. Sequence of mCerulean was confirmed (although it had a silent mutation). OmpA_Sil_Sdom in pSB4A5 and INP_Sil_Sdom in pSB1C3 had a deletion to be fixed by PCR. The rest of sequences either contained the random GFP sequenced mentioned before (since they are old ligation products) or multiple mutations that could not be fixed by PCR.
Back to Top
August
2nd August 2016
Tessa
Amplified E0840 (GFP) out of a pSB1C3-GFP plasmid using Phusion PCR. Four reactions of 50µl.
Nanodropped PCR products.
Nanodrop
Product
Concentration (ng/µl)
E0840
339.6
E0840
554.1
E0840
596.9
E0840
567.6
Ran a 1% agarose gel of the PCR product to verify product size.
Stored product 2 and 4 in the fridge for later use.
Lara
Digested the synthesized parts LacI, phaP, mKate, the backbones pSB1C3 and pSB4A5 and the Biobrick K1149051 with EcoRI and PstI. The restriction products were run over a 1% agarose gel and subsequently gel-purified.
Concentrations of purified DNA products.
Nanodrop
DNA construct
Concentration (ng/µl)
LacI
15.5
phaP
8.2
mKate
8.7
K1149051
13.8
pSB1C3
25.5
pSB1C3
13.1
pSB1C3
9.3
pSB4A5
10.4
pSB4A5
7.3
pSB4A5
6.4
3rd August 2016
Lara
Ligated the synthesized parts that were restricted and gel-purified yesterday and the backbones. The following ligation products were formed: LacI-pSB1C3, phaP-pSB1C3, mKate-pSB1C3, LacI-pSB4A5 and K1149051-pSB4A5. The ligation protocol was used and the below stated lengths for the vector and the inserts were used.
Lengths of inserts and backbones.
DNA construct
DNA length (bp)
LacI
1335
phaP
648
mKate
790
K1149051
4271
pSB1C3
2070
pSB4A5
3395
Transformations were performed with the just obtained ligation products and additionally the previously obtained mVenus-pSB1C3 was also transformed into TOP10 cells. A DNA amount of 2 µL was used; as positive control for the ampicilin plates the plasmid RFP-pSB4A5 was used during transformation and for the chloramphenicol plates the plasmid RFP-pSB1C3 was used. Transformed cells were plated on selective LB-agar plates and incubated overnight at 37ºC.
Charlotte
Restriction and ligation of mVenus into the standard backbone
Tessa
Purification of PCR products of 2nd August. Accidentally added wash solution in first step, added more bind solution to compensate and it yielded good concentrations (86-115ng/µL).
PCR to add different promoters to GFP (E08040) BioBrick from the distribution kit. Forward primers were designed to add promoters upstream of the GFP sequence, sequencing reverse primer (RV) was used as reverse primer in this PCR. Primers added were: J23100, J23105, J23108, J23113, J23117.
4th August 2016
Lara
Performed a DreamTaq colony PCR for yesterday's transformations; eight colonies per plate were picked and analyzed.
Gel electrophoresis of colony PCR of K1149051 pSB4A5, LacI pSB4A5, phaP pSB1C3, mKate pSB1C3, mVenus pSB1C3 and LacI pSB1C3.
Colonies were transferred into selective liquid LB medium.
DNA construct transformation
Selective medium
Colonies picked
K1149051-pSB4A5
LB-Amp
2
phaP-pSB1C3
LB-Cm
19, 20 and 21
mKate-pSB1C3
LB-Cm
31
mVenus-pSB1C3
LB-Cm
36, 40 and 41
LacI-pSB1C3
LB-Cm
46, 48 and 49
Tessa
Nanodrop of PCR products of GFP including promoters (sumarised in the following table) and subsequent restriction with EcoRI and PstI.
DNA construct
Concentration (ng/µL)
J23100 E0840
69.4
J23105 E0840
95.9
J23108 E0840
88.3
J23113 E0840
97.7
J23117 E0840
90.3
Concentration of DNA was measured after purification of the restriction products (see table).
DNA construct
Concentration (ng/µL)
J23100 E0840
3.3
J23105 E0840
3.5
J23108 E0840
3.5
J23113 E0840
2.8
J23117 E0840
3.4
These products were ligated with pSB1C3 and transformed into Top10 competent cells and incubated overnight.
5th August 2016
Lara
Some of the overnight cultures were opened inside the shaker, so these were not sterile anymore. Colonies 21 (phaP-pSB1C3) and 47 (LacI-pSB1C3) had to be picked and transferred into liquid LB-medium again.
From the other overnight cultures, plasmids were isolated using Miniprep.
Concentrations of plasmid isolations.
DNA construct
Colony number
Concentration (ng/L)
K1149051-pSB4A5
2
271.7
phaP-pSB1C3
19
398.0
phaP-pSB1C3
20
207.1
mKate-pSB1C3
31
83.3
mVenus-pSB1C3
36
142.5
mVenus-pSB1C3
40
306.5
mVenus-pSB1C3
41
65.3
These DNA samples were sent to sequencing.
6th August 2016
Tessa
Colony PCR of transformations from yesterday
Lanes 1, 2, 4, 5, 8, 9, 12, 15, 16, 17 and 19 yielded positive results and they were picked for overnight culture in selective LB medium.
7th August 2016
Tessa
Plasmid isolation of overnight cultures from yesterday using 3mL of culture. All products yielded good concentrations of DNA (90-200ng/µL). All of them were sent for sequencing.
8th August 2016
Tessa
Cryostock of overnight cultures from the other day.
Liza
Transformation of mCerulean into BL21.
9th August 2016
Charlotte
Started correcting point mutations using PCR in BioBricks that had a non-silent mutation in a coding area. The plasmids being corrected are SulA (pSB1C3), BolA_ind (pSB1C3), BolA_con (pSB4A5) and INP_Sil_Sdom (pSB4A5). Primers were ordered by María. Primers were first diluted for storage (100 µM) and working stock (10 µM). The concentrations of the plasmids with point mutations were determined using a Nanodrop.
Nanodrop
BioBrick
Concentration (ng/µl)
SulA (pSB1C3)
167.7
BolA_ind (pSB1C3)
109.1
BolA_con (pSB4A5)
130.4
INP_Sil_Sdom (pSB4A5)
166.7
The PCR to correct the mutations was done according to the Phusion HF protocol. The elongation time was set at 75 seconds, and the annealing time was set at 57ºC. The PCR product was loaded on a 1% TAE agarose gel for electrophoresis, in order to determine if the PCR worked.
Left is a simulated image by SnapGene, right the actual gel picture. The lane marked with L is the Promega 1kb ladder, lane 1 contains SulA (pSB1C3), lane 2 contains BolA_ind (pSB1C3), lane 3 contains BolA_con (pSB4A5) and lane 4 contains INP_Sil_Sdom (pSB4A5)
The bands for BolA_ind, BolA_con and INP_Sil_Sdom are correct (lanes 2-4). The band for SulA (lane 1) is not as expected. This sample is checked on gel again.
Left is a simulated image by SnapGene, right the actual gel picture. The lane marked with L is the Promega 1kb ladder, lane 1 contains SulA (pSB1C3). The other bands contain result from another experiment, irrelevant for this experiment.
The result from the second electrophoresis of SulA is still not as expected, we will start over with the correction of this BioBrick later. For now, we continue with BolA_ind (pSB1C3), BolA_con (pSB4A5) and INP_Sil_Sdom (pSB4A5).
BolA_ind (pSB1C3), BolA_con (pSB4A5) and INP_Sil_Sdom (pSB4A5) were purified according to protocol. The obtained concentrations were measured using Nanodrop.
Nanodrop
BioBrick
Concentration (ng/µl)
BolA_ind (pSB1C3)
99.9
BolA_con (pSB4A5)
98.5
INP_Sil_Sdom (pSB4A5)
61.5
The purified PCR product was restricted with DpnI, to restrict all methylated DNA containing the point mutation, leaving us with the corrected DNA. Restriction was done according to protocol. After 1 hour of incubation at 37ºC the restriction products were purified according to protocol. The prified restriction products were stored at -20ºC.
Liza
Colony PCR of transformations from yesterday
Lara
Colony PCR of LacI was performed again.
Colonies indicated with red arrows were transferred into liquid selective medium and grown overnight.
10th August 2016
Tessa
Restricted and purified PCR product of 3rd August, J23100 E0840, J23113 E0840 and J23117 E0840, with EcoRI-HF and PstI. Obtaining the following concentrations.
Nanodrop
Product
Concentration (ng/µl)
J23100 E0840
14.7
J23113 E0840
21.3
J23117 E0840
23.3
Ligated purified product into pSB1C3.
Transformed ligation product into TOP10 strain. Using RFP as positive control and sterile MilliQ as negative control. Plated on selective plates.
Lara
Of the previously sent DNA samples to sequencing, mVenus-pSB1C3, mKate-pSB1C3 and phaP-pSB1C3 were sequence confirmed. LacI-pSB1C3 and K1149051 were analysed again. Since the previous selected colonies for LacI-pSB1C3 turned out to contain the construct of the positive control RFP, a new colony-PCR was performed. Besides that, colony 47 of August 4th was also transferred into liquid medium.
Via Phusion-PCR, the promoters J23100, J23105, J23108, J23113 and J23117 were added to the yesterday sequence confirmed phaP-pSB1C3. The melting temperature of the designed primers were for all 58ºC; the template concentration was 102.0 ng/µl and the final construct should obtain 835 bp. The Phusion-PCR ran overnight.
Charlotte
The concentrations of the previously purified restriction products were measured using a Nanodrop:
Nanodrop
Product
Concentration (ng/µl)
BolA_ind (pSB1C3)
16.5
BolA_con (pSB4A5)
14.4
INP_Sil_Sdom (pSB4A5)
9.5
The restriction products were blunt-end ligated in order to circularize the plasmids. Since our ligation tool does not calculate the concentrations needed for blunt-end ligation, the following protocol was used:
Blunt-end ligation protocol
Component
Amount
T4 ligation buffer (10X)
2 µL
DNA
~80 ng
T4 Ligase
1 µL
Nuclease-free water
up to 20 µL
The T4 ligase was added last, the ligation mixture was left to incubate for 4 hours. After incubation, the ligation product was transformed into homemade chemically competent TOP10 cells, according to protocol. The tranformations were set to incubate overnight at 37ºC
Iris
PCR of phaP to fix mutations.
Célina and Liza
Plasmid isolation and cryostock of mCerulean overnight cultures obtaining the following concentrations.
11th August 2016
Tessa
Colony PCR of colonies from yesterdays transformation using Q5® mastermix.
Ran a 1% agarose gel of the PCR product.
L is the Promega 1kb ladder, 1-6 is J23100 E0840, 7-14 is J23113 E0840 and 15-22 is J23117 E0840.
Transferred colony 14 (J23113 E0840 pSB1C3) and 21 (J23117 E0840 pSB1C3) into liquid LB.
Lara
Plasmids were isolated from the colonies that were grown overnight.
Nanodrop
DNA construct
Colony number
Concentration (ng/µl)
LacI-pSB1C3
47
56.5
LacI-pSB1C3
51
86.4
J23117 E0840
54
141.4
LacI-pSB1C3 was sent to sequencing.
Charlotte
Picked 8 colonies for the plates containing cells tranformed with BolA_ind (psB1C3) and 8 for BolA_con (pSB4A5). The colonies on the plate containing cells transformed with INP_Sil_Sdom were still too small to pick so they were left to incubate at 37ºC a little longer before picking 8 colonies for colony PCR.
The colony PCR was done according to protocol, using DreamTaq polymerase. The PCR products were loaded on a 1% agarose gel for electrophoresis. Colony 16 didn't fit in the PCR machine anymore, so this colony is also not loaded on the gel.
Left is a simulated image by SnapGene, right the actual gel picture. For BolA_ind, colonies 2,3 and 4 had a band at the correct height. For BolA_con, colonies 11, 12 and 13 had a band at the correct height. For INP_Sil_Sdom, colonies 17, 19 and 20 were correct. These colonies were transferred to selective liquid LB and grown overnight in a shaker at 37ºC and 250 rpm.
Charlotte & Liza
Tested 15 µm fluorescent polystyrene beads in the MicroTime 200 confocal microscope to see if the microscope is able to establish lasing in a bead. We used a 1 mW laser with an excitation wavelength of 532 nm. The bead was imaged at different intensities to see if we were able to see a shift to lasing at higher excitation energies.
Liza
Colony PCR of tranformations of mVenus and mKate in the standard backbone.
12th August 2016
Tessa
Miniprepped colony 14 and 21 of yesterdays overnight culture.
Nanodropped miniprep product.
Nanodrop
Product
Concentration (ng/µl)
J23113 E0840 in pSB1C3
140.6
J23117 E0840 in pSB1C3
325.6
Cryostocked colony 14 and 21.
Send miniprep product for sequencing. [Sequence Confirmed]
Lara
The results for the sequencing of LacI-pSB1C3 were checked: For both colonies 47 and 51 there was a 124 bp long exact same insert in the middle of the LacI-gene. In colony 54, there were two point mutations. The first one at location 2215, changing an Arg into Leu; the second mutation was on location 2625 changing a Glu into a stopcodon.
These two point mutations need to be fixed. Since normal mutation fixing by PCR only handles one mutation at a time, a new strategy was thought of: Two different linear fragments should be made, both fixing one mutation and able to join together with the other part via Gibson Assembly. The primers were designed and ordered.
The Phusion-PCR products of the different promoters that were obtained were transformed into TOP10 cells. For this, 2µl DNA was used per transformation, RFP-pSB1C3 as a positive control and MilliQ as a negative control. All transformations were plated onto LB-Cm plates and stored at 37ºC.
Charlotte
The DNA of the colonies that were grown overnight was isolated using miniprep according to protocol. The isolated DNA was stored at -20ºC. The remaining culture was cryostocked according to protocol and stored at -80ºC.
Charlotte & Liza
Did a second attempt to get lasing in a bead with a confocal microscope.
Liza
Transformation mVenus and mKate into BL21, plated in selective plates and incubated at 37ºC overnight.
13th August 2016
Lara
A colony PCR was performed for the transformations of the different promoters attached to phaP-pSB1C3. The expected amplification length is approximately 1kb. Four colonies per plate were analyzed in the colony PCR.
This gel showed that only colony 35 (lane 15) did not contain the right PCR-product. The other colonies were transferred to liquid LB-Cm and cultivated overnight at 37ºC.
14th August 2016
Lara
The overnight grown cultures of the different promoters attached to phaP-pSB1C3 were plasmid isolated.
Nanodrop
DNA construct
Colony number
Concentration (ng/µl)
J23100 phaP pSB1C3
33
83.1
J23100 phaP pSB1C3
34
80.4
J23105 phaP pSB1C3
25
101.6
J23105 phaP pSB1C3
26
113.0
J23105 phaP pSB1C3
27
97.4
J23108 phaP pSB1C3
18
93.4
J23108 phaP pSB1C3
19
120.0
J23108 phaP pSB1C3
20
100.0
J23113 phaP pSB1C3
9
94.3
J23113 phaP pSB1C3
10
85.0
J23113 phaP pSB1C3
11
110.0
J23117 phaP pSB1C3
1
101.2
J23117 phaP pSB1C3
2
100.5
J23117 phaP pSB1C3
3
110.0
These plasmids were sent to sequencing.
15th August 2016
Charlotte
The concentrations of the previously isolated DNA was checked on Nanodrop. The following concentrations were measured:
Nanodrop
BioBrick
Colony number
Concentration (ng/µl)
BolA_ind (pSB1C3)
2
63.0
3
61.8
4
68.3
BolA_con (pSB4A5)
11
101.9
12
71.7
14
90.9
INP_Sil_Sdom (pSB4A5)
17
88.0
19
74.4
20
43.2
The isolated DNA was prepared and sent for sequencing, according to protocol.
Liza
Colony PCR of mVenus and mKate transformations.
16th August 2016
Charlotte
Sequencing had the following results:
BolA_ind (pSB1C3): All colonies had large inserts or deletions, so they couldn't be used for further experiments. We will try to repair the mutations again later.
BolA_con (pSB4A5): Colony 14 has an insert and colony 11 and 12 have a small gap. For colony 11 this is probably a readout error, when you look closer to the chromatogram. We can use colony 11 for further experiments.
INP_Sil_Sdom (pSB4A5): Colony 17 has a nonsilent mutation in the promoter and colony 20 has a gap in the promoter. Colony 19 has one silent mutation and one nonsilent mutation, but in the linker region. We can use colony 19 for further experiments.
Liza
Electrophoresis of colony PCR from 15th of August. Colonies 2, 3 and 8 from mVenus and 1, 3 and 4 from mKate were picked for overnight cultures.
17th August 2016
Charlotte & Liza
Did a third attempt to get lasing in a bead with a confocal microscope.
Tessa
Plasmid isolation from yesterday's overnight cultures from Liza. Concentrations obtained were between 120 and 150ng/µL.
18th August 2016
Tessa
Transformation of the InterLab Study plasmids into BL21.
Liza
Receivend purified GFP and made 5 aliquots of 15µL which were stored at -80ºC.
19th August 2016
Tessa
Only Test Device 2 and 3 and positive control have yielded (very little) colonies. Decided to try and transform them into Top10.
Lara
A colony PCR was performed for the transformation of LacI-pSB1C3 on IPTG plates.
Since none of the colonies yielded a positive result in the colony-PCR, the colony-PCR was performed again. This time with a new protocol, first boiling the colonies and using GoTaq buffer. A total of 5 µL DNA mixture was used.
The colony indicated with a red arrow was transferred into selective liquid LB.
22nd August 2016
Tessa
Transformation of the InterLab Study plasmids into Top10.
Colony PCR of J23100 E0840 pSB1C3 using GoTaq®. Plate from 11th August.
Ran a 1% agarose gel to verify part size
L is the Promega 1kb ladder, 1-9 are J23100 E0840.
Colony 1,2 and 3 were put in selective LB medium and incubated overnight.
Lara
A miniprep was performed to isolate plasmids from the LacI transformation, induced with IPTG. Concentrations can be found in the table below.
Nanodrop
DNA construct
Colony number
Concentration (ng/µl)
LacI pSB1C3
11
95.5
K1149051 pSB4A5
2
120.3
The LacI pSB1C3 plasmid was sent to sequencing, the K1149051 pSB4A5 plasmid yielded a too low concentration during plasmid isolation, since the sequencing company requested higher concentrations for larger plasmids. The BioBrick K1149051 pSB4A5 was cultivated from a cryostock overnight and for the LacI pSB1C3 BioBrick precautional cryostocks were made.
23rd August 2016
Tessa
Colony PCR of the InterLab Study plates using DreamTaq®. Picked eight colonies of each plate.
Cryostocked J23100 E0840 pSB1C3 in Top10.
Ran a 1% agarose gel of the InterLab Study colony PCR products.
L is the Promega 1kb ladder, 1-8 is Test Device 1, 9-16 is Test Device 2, 17-24 is Test Device 3, 25-32 is Negative Control and 33-40 is Positive control.
Colony 1, 2, 4, 9-11, 17-19, 26, 27, 29 and 33-36 were picked for overnight culture in liquid LB+CM.
Lara
Plasmids were isolated for the yesterday overnight cultured K1149051 pSB4A5 and for different colonies of J23100 E0480 pSB1C3.
Nanodrop
BioBrick
Colony number
Concentration (ng/µl)
K1149051 pSB4A5
2
328.7
J23100 E0480 pSB1C3
1
83.8
2
89.5
3
85.4
These plasmids were sent to sequencing.
Primers that were needed for Gibson Assembly of the fluorophores and PHB constructs were diluted.
24th August 2016
Tessa
Put InterLab Study cultures in fridge for later use.
Charlotte
Started a second round of correcting mutations in our designed BioBricks using PCR. The BioBricks that will be corrected are SulA (pSB1C3), BolA_ind (pSB1C3), INP_Sil_Sdom (pSB1C3) and OmpA_Sil_Sdom (pSB4A5). Prior to starting the correction of the mutations using PCR, the concentrations of these BIoBricks were measured on the Nanodrop:
Nanodrop
BioBrick
Concentration (ng/µl)
SulA (pSB1C3)
167.6
BolA_ind (pSB1C3)
109.1
INP_Sil_Sdom (pSB1C3)
139.2
OmpA_Sil_Sdom
170.8
The primers were diluted according to the manufacturers' instructions to a stock of 100µM. These stocks were consequently 10 times diluted to a working stock of 10µM.
Lara
Our advisor Esengül provided us with a protocol for the Gibson Assembly (see protocols).
The first step of the Gibson Assembly was to generate linear DNA segments by Phusion PCR. This was performed and to check the products, 5µL reaction product was loaded onto a 1% agarose gel.
None of the PCR products yielded the correct length. Especially K1149051 was not nearly the length expected. Sequences arrived, and it appeared that the were not correct. We figured that the backbone could have been pasted to each other twice, forming a plasmid like that with antibiotic resistance. For this, another round of colony PCR for K1149051 pSB4A5. Another colony PCR should be performed.
25th August 2016
Tessa
Resuspended FITC according to InterLab protocol.
Charlotte
Ran the PCR to fix the mutations in the BioBricks according to the Phusion HF PCR protocol, with an annealing time of 57ºC and an elongation time of 1.5 minutes. The results of the PCR were confirmed on gel, according to the electrophoresis protocol.
Left is a simulated image by SnapGene, right the actual gel picture. The lane marked with L is the Promega 1kb ladder, lane 1 contains SulA (pSB1C3), lane 2 contains BolA_ind (pSB1C3), lane 3 contains INP_Sil_Sdom (pSB1C3) and lane 4 OmpA_Sil_Sdom (pSB4A5)
The bands from BolA_ind (pSB1C3), INP_Sil_Sdom (pSB1C3) and OmpA_Sil_Sdom (pSB4A5) are at the correct height compared to the simulation. The lane containing SulA shows the same result as the previous attempt to correct this BoBrick. Probably something went wrong in the PCR, it's likely that the primers anneal at the wrong position. We will not continue this BioBrick and use SulA in another backbone. The other PCR products were purified according to protocol and the concentrations were measured using Nanodrop.
Nanodrop
BioBrick
Concentration (ng/µl)
BolA_ind (pSB1C3)
107.7
INP_Sil_Sdom (pSB1C3)
88.7
OmpA_Sil_Sdom
100.3
The purified PCR products were restricted with DpnI to remove any methylated and thus mutated DNA, according to the restriction protocol. Since our heat block was broken the samples were incubated in the stove at 37ºC. The restriction products were purified according to protocol and stored at -20ºC.
Lara
Another colony PCR was performed for K1149051 pSB4A5 to make sure we did not select a plasmid with a double backbone.
26th August 2016
Lara
The colony PCR of yesterday was gel analysed. The products in lanes 2, 3, 4, 9 and 10 were assumed to be correct. The corresponding colonies were transferred to LB+Amp and incubated overnight at 37ºC.
28th August 2016
Lara
Plasmid isolation was performed for the K1149051 pSB4A5 BioBrick.
Nanodrop
BioBrick
Colony number
Concentration (ng/µl)
K1149051 pSB4A5
2
340.3
3
278.6
4
320.9
9
290.1
10
315.6
These plasmids were sent to sequencing and were sequence confirmed.
29th August 2016
Tessa
Transformed J23100-, J23105-, J23108-, J23113- and J23117 E0840 pSB1C3, and BolA_con pSB4A5 into BL21.
Charlotte
Checked the concentrations of the purified restriction products on Nanodrop:
Nanodrop
BioBrick
Concentration (ng/µl)
BolA_ind (pSB1C3)
14.7
INP_Sil_Sdom (pSB1C3)
11.9
OmpA_Sil_Sdom
12.6
The restriction products were blunt-end ligated according to the ligation protocol. The ligated BioBricks were transformed into homemade TOP10 cells, according to protocol. For the controls only 1/3 of the volume was used due to high transformation efficiency observed before. The transformants were plated on selective plates and left to incubate overnight at 37ºC.
Lara
Prepared LB Amp+Cm plates.
30th August 2016
Lycka
PCR lineralization of vectors and inserts for gibson assembly phaCAB operon (BBa_K1149051) to fluorophores (GFP, mVenus, mKate, mCerulean). Phusion polymerase. Annealing temperature 60ºC, elongation time 2 minutes.
Tessa
Did the InterLab Study plate reader measurements according to the InterLab plate reader protocol.
Streaked J23105- and J23117 E0840 pSB1C3 out on new plates, because they had too many colonies.
Colony PCR of six colonies of BolA_con, J23100 E0840, J323108 E0840 and J23113 E0840 using GoTaq®.
Ran a 1% agarose gel of PCR products to verify part size.
L is the Promega 1kb ladder, 1-6 is BolA_con, 7-12 is J23100 E0840, 12-18 is J23108 E0840 and 19-24 is J23113 E0840.
Picked colony 1, 6, 11, 13, 18, 20 and 21 for overnight culture.
Charlotte
Of each transformation plate, 8 colonies were picked: Colonies 1-8 were of BolA_ind, colonies 9-16 of OmpA_Sil_Sdom and colonies 17-24 of INP_Sil_Sdom. A colony PCR was performed with the picked colonies, according to the GoTaq Colony PCR protocol. The annealing time was 1.5 minutes.
The PCR products were checked on gel according to the Electrophoresis protocol, which yielded the following gel:
L is the Promega 1kb ladder, 1-8 is BolA_ind, 9-16 is OmpA_Sil_Sdom, 17-24 is INP_Sil_Sdom
Colonies 1, 3, 4, 14, 15, 16, 20, 21 and 22 were picked and transferred to selective medium for an overnight culture, according to protocol.
Lara
Transformations into BL21 were performed for K1149051 pSB4A5, J23100 phaP pSB1C3 + K1149051 pSB4A5, J23105 phaP pSB1C3 + K1149051 pSB4A5, J23108 phaP pSB1C3 + K1149051 pSB4A5, J23113 phaP pSB1C3 + K1149051 pSB4A5 and J23117 phaP pSB1C3 + K1149051 pSB4A5. RFP pSB4A5 and RFP pSB4A5 + RFP pSB1C3 were used as positive controls and MilliQ as a negative control. In total 70 µL was plated on selective LB plates and they were incubated at 37ºC overnight.
Cryostocks were made for K1149051 pSB4A5 in Top10 cells.
31st August 2016
Lycka
Electrophoresis of PCR fragments yesterday.
Picture on the left shows simulated agarose gel. Picture on the right shows the actual agarose gel. 1 - 4: vectors for mCerulean, mKate, mVenus and GFP, respectively. 5 - 6 inserts for mVenus and GFP. Gel did not yield bands at the right height. PCR is repeated at an annealing temperature of 57ºC.
Second gel also did not yield the right bands. PCR repeated overnights by María at Tm 64ºC and elongation time 3.5 minutes in the presence of DMSO.
Transfer from cryostock to plate BL21 strains with fluorophores expressed. On LB agar supplemented with chloramphenicol.
Tessa
Cryostocked J23100-, J23108- and J23117 E0840 pSB1C3 in BL21. BolA_con didn't grow.
Cryostocked K1149051 pSB4A5 in BL21.
Charlotte
The overnight cultures of BolA_ind (pSB1C3), OmpA_Sil_Sdom (pSB4A5) and INP_Sil_Sdom (pSB1C3) were cryostocked according to protocol, 3 tubes per culture. The remaining culture was used for DNA isolation by Miniprep, according to protocol. Two tubes per colony were prepared. The concentration of the isolated DNA was measured using Nanodrop, the following concentrations were measured:
Nanodrop
BioBrick
Colony number
Tube 1 concentration (ng/µl)
Tube 2 concentration (ng/µl)
BolA_ind (pSB1C3)
1
91.2
113.5
3
87.9
79.6
4
104.5
87.7
OmpA_Sil_Sdom (pSB4A5)
14
133.5
143.7
15
152.5
141.4
16
171.1
159.4
INP_Sil_Sdom (pSB4A5)
20
109.6
90.6
21
112.2
101.5
22
112.0
111.9
The isolated DNA was prepared and sent for sequencing, according to protocol.
Lara
Yesterday's transformations in BL21 did not yield colonies for K1149051 pSB4A5 and for J23100 phaP pSB1C3 + K1149051 pSB4A5. These were transformed again, together with BolA_con pSB4A5, K1149051 pSB4A5 and RFP in pSB4A5, pSB4A5 and pSB1C3 as positive control and MilliQ as negative control. These transformations were plated on selective LB plates and stored overnight at 37ºC.
For the Gibson Assembly, new plasmid needed to be isolated.
Nanodrop
BioBrick
Concentration (ng/µl)
K1149051 pSB4A5
171.6
189.6
182.8
141.2
157.1
162.7
A colony PCR with GoTaq buffer was performed for the different promoters of phaP pSB1C3 + K1149051 pSB4A5.
Colonies 2, 3, 4, 8, 9, 10, 13, 14, 15, 18, 19 and 20 were transferred into liquid LB+Amp+Cm and stored overnight at 37ºC.
Back to Top
September
1st September 2016
María
PCR with new primers of those parts that need to be expressed under the lac promoter. These primers will allow cloning on Addgene plasmids from the pBb series. The plasmids chosen (pBbS5a and pBbA5c) express lacI and the cloning site is placed downstream of a lac promoter. The parts to be amplified are:
1. BolA
2. INP_Sil_Sdom
3. OmpA_Sil_Sdom
4. OmpA_Sill_Taur
5. Sil_Sdom
6. SulA
Lycka
Electrophoresis of overnight PCR for lineralization.
Picture on the left shows simulated agarose gel. Picture on the right shows the actual agarose gel. 1 - 4: vectors for mCerulean, mKate, mVenus and GFP, respectively. 5 - 6 inserts for mVenus and GFP. For all lanes except lane 5, the right bands are present. Also many other bands were present so the product was gel purified.
Transfered one colony of BL21 on each plate from yesterday in a 50mL falcon tube with 10mL filter sterilized M9 medium with glucose supplemented with chloramphenicol.
Tessa
Put three colonies of J23100-, J23105- and J23117 E0840 pSB1C3 in BL21 into selective LB medium.
Charlotte
Cryostocks: Were made for colonies 4, 10, 13 and 18 for the different promoters of phaP pSB1C3 + K1149051 pSB4A5. They were stored at -80ºC.
Liquid cultures: Each colony of BL21 transformation should be good. Liquid cultures were made of BolA_con pSB4A5, K1149051 pSB4A5 and J23100 phaP pSB1C3 + K1149051 pSB4A5 and stored at 37ºC to be used tomorrow to make cryostocks.
2nd September 2016
Tessa
Cryostocked J23100-, J23105- and J23117 E0840 pSB1C3 in BL21.
Lycka
Nanodrop gel purified PCR fragments from yesterday. Concentrations are too low to work with. Repeat PCR under the same conditions, but with double volume (100µL). Done by María.
Measure OD600 of cultivated M9 tubes.
OD600 measurements
Genotype
OD600
GFP J23108
1.24
mVenus
0.04
mKate
0.20
mCerulean
0.03
The strains containing GFP and mKate grew really well, the ones containing mVenus and mCerulean did not. All of the tubes were put back in the incubator. The one with GFP was diluted 10 times in M9 medium, the one with mKate was diluted 2 times. Aliquots of 150µL were loeaded in an 96 well plate and measured in a plate reader at 37ºC for 7 hours.
María
PCR of same samples as Lycka in a reaction volume of 100µL. After gel isolation and purification only for mCerulean and mKate good concentrations were reached (160 and 260 ng/µL, respectively), for the rest of samples either no bands were seen in the agarose gel or the concentration was too low again.
Charlotte
Analyzed sequencing results. Both the colonies of BolA_ind and OmpA_Sil_Sdom had an insert around the mutation spot, so sequences were not correct. Also all colonies of INP_Sil_Sdom had non-silent mutations.
3rd September 2016
Lycka
The cultures in M9 grew well in the incubator until an OD of approximately 1. Samples were diluted 10 times in M9 supplemented with chloramphenicol and measured in the plate reader.
4th September 2016
Lara
Doing things for Lycka and María: Plate was removed from plate reader and file was saved. Plate reader plate was stored at 4ºC.
mVenus pSB1C3 was miniprepped and stored at -20ºC.
Nanodrop
BioBrick
Tube number
Concentration (ng/µl)
mVenus pSB1C3
1
47.3
2
70.7
3
53.6
4
44.5
5
64.4
6
54.7
A Phusion PCR was performed again for Gibson Assembly. The vector for mKate and mVenus and the insert for GFP yielded a correct PCR product and were gel isolated.
Nanodrop
Linearized part
Concentration (ng/µl)
mKate vector
71.0
mVenus vector
69.2
GFP insert
31.1
Since there are an insert and a vector present that are compatible for mKate and K1149051 pSB4A5, these were Gibson Assembled according to the protocol and transformed into Top10 cells. As a negative control for the Gibson Assembly, nuclease-free water was used instead of the vector. Transformations were plated on selective LB plates by Lycka and stored overnight at 37ºC.
6th September 2016
Lycka & Lara
Colony PCR for the transformants of the Gibson Assembly.
Most colonies appear to be false positives, since the bands correspond to the length of the vector only. Colonies 43 and 44 might contain the desired plasmid (with the PHB operon fused to mKate), so these are transfered to LB medium.
8th September 2016
Lycka
Isolate plasmid, cryostock and prepare for sequencing colonies 43 and 44. The concentrations were 252.6 and 242.6 ng/µl, respectively.
Lara
For the Gibson Assemblies, we still needed the mCerulean vector, the GFP vector and the mVenus insert. For this, another phusion PCR was performed and it ran overnight.
9th September 2016
Lycka
In order for the cells to grow better, we prepared enriched M9 medium (eM9), instead of regular M9.                                                 The medium was supplemented with chloramphenicol and devided into aliquots of 10 mL. From all plates with fluorophore expressing cells,                                                 one colony was picked and grown in eM9 overnight.
Lara
The phusion PCRs of yesterday were gel-isolated. Since mVenus once again did not yield a proper product, we decided to discard this part.
Nanodrop
Linearized part
Concentration (ng/µl)
GFP vector
23.3
mCeruleanvector
12.3
A Gibson Assembly reaction was performed according to the protocol for the GFP + K1149051 assembled plasmid and the mCerulean + K1149051 assembled plasmid. After 60 minutes of incubation at 37ºC, they were transformed into Top10 cells. For this, RFP pSB4A5 was used as a positive control and vector without inserts were used as negative controls. Transformation products were plated on selective LB plates and stored overnight at 37ºC.
10th September 2016
Lycka
Transfer to LB cells with plasmids bought from Addgene: pBbS5a and pBbA5c
Measure OD of overnight cultures in eM9. Dilute them until OD of 0.1 in eM9 supplemented with chloramphenicol.                                                 Measure kinetic cycle of 7 hours in plate reader.
11th September 2016
Charlotte & Lara
Miniprepped plasmids bought from Addgene pBbS5a and pBbA5c from overnight LB cultures; 5 tubes each. The concentrations were confirmed with nanodrop and were between 30 and 60 ng/µL.
The transformations from the Gibson Assembly still did not yield any colonies.
María
Restriction of plasmids bought from Addgene pBbS5a and pBbA5c, and PCR products from September 1st with EcoRI and XhoI. The biggest fragment obtained with each plasmid is the backbone, which was gel isolated. Then, backbones and fragments were ligated and transformed into Top10 chemically competent cells.
Tessa
Transformation of the following plasmids in Top10 chemically competent cells and plated on selective agar plates.
BolA_con pSB1C3
INP_Sil_Sdom pSB4A5
OmpA_Sil_Taur pSB4A5
Sil_Sdom pSB4A5
Sul_A pSB1C3
OmpA_Sil_Sdom pSB4A5
RFP pSB1C3 (positive control)
RFP pSB4A5 (positive control)
Sterile MiliQ (negative control)
12th September 2016
María
Colony PCR of transformations from yesterday 16 colonies of each transformation were picked, but only analized 15 of each.
Red arrows point at colonies picked for overnight cultures in selective medium.
Lycka
Measure spectrum of mVenus and mCerulean cells in plate reader.
13th September 2016
María
Plasmid isolation, preparation of samples for sequencing and cryostock of overnight cultures of colonies picked yesterday.
Sequences of all colonies picked for OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom and SulA were confirmed to be good. In the case of BolA there's a deletion of a single bp, to be fixed by PCR. All sequences for OmpA_Sil_Sdom contained a strange long insert, this silicatein gene version was dropped for further experiments.
Tessa
Put 100µl of TOP10 competent cells into 10ml LB in a falcon tube and incubated overnight at 37ºC at 250rpm.
Made LB and LB agar.
Lycka
Measure spectrum of eM9 cultures in plate reader.
Sequences of mKate gibson assembled to phaCAB arrived and didn't yield the good results.
14th September 2016
Tessa
Made ~100 aliquots TOP10 competent cells with culture from yesterday according to protocol. OD600 before putting on ice was 0.34.
15th September 2016
María
Transformation of plasmids with confirmed sequences from 13th of September into BL21. Also, combined with GFP under promoter J23100 in pSB1C3 backbone. All 3 silicatein versions, SulA and the combination of Sil_Sdom and GFP yielded colonies. Other 2 silicateins together with GFP did not yield any positive result.
Tessa
Did positive and negative controls of competent cells made yesterday. Used RFP pSB1C3 for positive control and MiliQ for negative control.
Lycka
Wash 3x in PBS cultures grown in LB expressing fluorophores. Dissolve in PBS and measure spectrum in plate reader.
16th September 2016
Lycka
Repeat kinetic cycle experiment in plate reader with different settings, because the previous results didn't show good results.
18th September 2016
Lycka
Transfer from plate to liquid LB colonies with BL21 cells expressing OmpA_Sil_Taur, Sil_Sdom and INP_Sil_Sdom for widefield microscope experiment.
19th September 2016
Lycka & María
Transfer LB cultures from yesterday to fresh medium and grow until OD reached 0.65. Then added 1mM sterile IPTG to induce.                                                 Grown in the incubator for another 5 hours. Subsequently added 60µM Sodium Silicate and incubated for 3 hours before putting them in the fridge.
Diluted and filter sterilized Sodium Silicate in miliQ water up to a concentration of 1M.
Diluted and filter sterilized Rhodamine 123 in ethanol up to a concentration of 20 mg/mL and added 15µL of this to the indicated cultures.
20th September 2016
María
Transfer of colonies from 15th of September transformations into overnight culture for cryostocking.
Lycka & Charlotte
Add 7µM rhodamine to the cultures from yesterday, incubated 10 min. Centrifuged 1.5 mL of culture at 4000 rpm 5 min                                                 Washed 3x and then resuspended in 400µL PBS. Imaged in widefield microscope with excitation wavelength of 488 nm.
21st September 2016
María
Cryostock of BL21 strains containing sequence confirmed plasmids from yesterday's overnight cultures.
Transformation into BL21 of combinations of OmpA_Sil_Taur with all fluorophores (separately) and INP_Sil_Sdom with GFP under promoter J23100.
None of them yielded any colonies. We are going to try reducing the antibiotics concentration in the plates to half because probably the cells are not able to handle the 2 plasmids and such a high concentration of antibiotics.
Lycka
Measure OD600 of overnight cultures María prepared. They were around 1.46, 1.35, 1.49 for Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom respectively.                                                 1mL of culture was transfered to 10 mL fresh LB.
Dissolved and filter sterilized IPTG. 1.2 g in 5 mL.
Saved cultures in the fridge to continue tomorrow.
22th September 2016
Lycka
Inoculate 0.25 mL of yesterday's culture in 10 mL fresh LB. Cells did not grow after half a day. Inoculate                                                 new culture from plate, grown overnight.
23th September 2016
Lycka
Test silicic acid test, with 1mM aliquot of silic acid.
100 µL 1 mM silicate
4900 µL miliQ
3 drops reagent 1
3 drops reagent 2
500 µL reagent 3
24th September 2016
Lycka
Measure OD600 of overnight eM9 cultures, diluted 4 times before measuring.
OD600 measurements
Genotype
OD600
GFP J23100
0.90
GFP J23105
0.51
GFP J23108
0.88
GFP J23113
didnt grow
GFP J23117
didnt grow
mVenus
0.74
mCerulean
0.77
Dilute until OD 0.1 and divide into aliquots of 100µL in a 96 well plate. Performed kinetic cycle in plate reader.
26th September 2016
María
Repeat of transformations of 21st of September using plates with half the concentration of antibiotics. All of them yielded colonies.
27th September 2016
María
Transfer of colonies from yesterday's transformations to overnight culture.
Lycka
Prepare eM9 an divide into aliquots of 10 mL, inoculate with:
OmpA Sil Taur
INP Sil Sdom
Sil Sdom
BolA
OmpA Sil Taur
OmpA Sil Taur + mCerulean
mCerulean
Restreak plates with BL21 OmpA Sil Taur and BolA on fresh selective agar plates.
Cryostock OmpA + mCerulean, + mVenus and + all GFPs.
28th September 2016
María
Cryostock of BL21 cells from overnight culture from yesterday.
Fixing of BolA mutation by PCR.
Picture on the left shows simulated agarose gel. Picture on the right shows the actual agarose gel. PCR didn't work, we are going to try again with a gradient of annealing temperatures to see if this is the cause of the problem.
Lycka
Measure OD600 of overnight eM9 cultures, diluted 4 times before measuring.
OD600 measurements
Genotype
OD600
INP Sil Sdom
1.15
BolA
1.18
OmpA Sil Taur
1.26
Sil Sdom
1.00
Dilute until OD 0.05 in 200 mL eM9 and grown in shaker at 37ºC. Cultures didn't grow after half a day.
Prepared calibration curve for silicate test. Dilutions made with 1 mM silic acid stock.
Calibration curve
concentration (µM)
silica stock (µL)
miliQ (µL)
100
500
4500
80
400
4600
60
300
4700
40
200
4800
20
100
4900
10
50
4950
5
25
4975
0
0
5000
Prepare overnight eM9 cultures for:
Silic acid test: OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom, BolA
Laser setup: mCerulean, OmpA_Sil_Taur, mCerulean + OmpA_Sil_Taur
TEM: OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom
29th September 2016
Tessa
Made ~50 aliquots BL21 competent cells according to protocol. OD600 before putting on ice was 0.30.
María
PCR to fix BolA mutation.
All annealing temperatures resulted in a product of the desired length.
1. 66ºC; 2. 65ºC; 3. 63.4ºC; 4. 60ºC
Lycka & María
Measure OD600 of overnight eM9 cultures, diluted 4 times before measuring.
OD600 measurements
Genotype
OD600
INP Sil Sdom
0.77
BolA
0.74
OmpA Sil Taur
0.75
Sil Sdom
0.68
Inoculate 4 µL in 150 µL fresh eM9. After induction Erlenmeyer fell over and content was not usefull anymore.
Measure OD600 of overnight LB cultures, diluted 4 times before measuring.
OD600 measurements
Genotype
OD600
INP Sil Sdom
0.73
OmpA Sil Taur
0.76
Sil Sdom
0.91
Inoculate 100 µL in 10 mL fresh medium. Grow until OD 0.55 and add 10 µL 1 M IPTG.
30th September 2016
María
Silicic acid test: since cells grew too slow in eM9 medium we checked if LB absorbance spectrum interfered with the absorbance of the final product of the test (~815nm). It didn't so we decided to grow the cells in LB medium.
Measured OD600 of overnight cultures with a 1/4 dilution:
Genotype
OD600
OmpA Sil Taur
0.77
INP Sil Sdom
0.74
Sil Sdom
0.75
BolA
0.68
Carmen & Charlotte
Tested whether adding silicatein cells to a solar panel had any effect. We tested the strain expressing OmpA-fused silicatein which was incubated with silica, so it had a silicate coating.  The cells were washed 3x in PBS. We connected a multimeter that was measuring voltage to a solar cell with a limited area exposed to light. We measured the output power of the solar cell at the same time as the light intensity of the lamp. We repeated this experiment with the same area of the solar cell covered in PBS and covered in PBS + silicatein-expressing cells.
Charlotte & Liza
Tested the mCerulean-expressing cells in our self-built hardware setup. The cells were grown in LB and washed 3x with PBS. The cells were fixed on a slide by pipetting them under 3% agarose-TAE pads.
Lara
The PCR product of BolA of María was gel purified and blunt end ligated. Afterwards a transformation was performed into Top10 cells with RFP pSB1C3 as positive control and MilliQ as negative control. Transformations were plated on selective plates and stored overnight at 37ºC.
Tessa
Transformation of controls into new made BL21 competent cells.
RFP in psB4A5 plated on AMP plate
MQ plated on AMP plate and CM plate
31th September 2016
Lara
Plates of yesterday's transformations were transferred from the 37ºC incubator into the fridge.
Back to Top
October
2nd October 2016
Tessa
There is some AMP resistance in the new BL21 cells. This means colony PCR will be necessary when cloning AMP resistant plasmids into BL21.
3rd October 2016
María
Due to the problems with LB medium we considered the possibility of measuring the amount of silicic acid in polysilicate by hydrolysing it with NaOH ( Müller et al., 2003 ) ( Cha et al., 1999 ).
To do so we made a calibration curve of silicic acid diluted in NaOH and another diluted in equal concentrations of NaOH and HCl in to neutralize the pH in case it might suppose a problem. Both of them yielded successful results.
Since the silicic acid test worked in both calibration curves we decided that we would be using NaOH to hydrolyse the polysilicate layer in silicatein expressing cells to test silicatein kinetics. Thus, we prepared overnight cultures to do the test the following day.
Tessa
Poured plates: AMP, CM and AMP+CM
Célina
Colony PCR of BolA under an inducible promoter in the Addgene backbone (pBbA5c). It yielded some positive cololonies which were picked for overnight culture.
Liza
Preparation of TEM samples vesicles for Wageningen as protocol .
4th October 2016
María & Lycka
Silicic acid test and preparation of samples for the laser set up and for TEM following the same steps as on 30th September. OD600 of overnight cultures were (4 times diluted):
Genotype
OD600
OmpA Sil Taur
0.80
INP Sil Sdom
0.70
Sil Sdom
0.83
BolA
0.76
mCerulean
0.74
mCerulean + OmpA Sil Taur
0.88
Silicic acid was added in the form of sodium silicate to a final concentration of 60µM 3h after induction with IPTG. 5mL samples were taken every 20min and centrifuged to be kept as a pellet in the fridge for the day after. Also every hour an extra 1mL sample was taken for a growth study. For every sample OD600 was measured.
Charlotte & Liza
Tested the mCerulean-expressing cells, the OmpA-silicatein expressing cells and OmpA-silicatein + mCerulean expressing cells in our self-built hardware setup. The cells were grown in LB, spun down and resuspended in PBS. The cells were fixed on a slide by pipetting them under 3% agarose-TAE pads. The cells were excited at 405 nm and imaged at different excitation energies.
Célina
Cryostock of colonies picked yesterday.
5th October 2016
María & Lycka
Silicic acid test: pellets were washed 3 times with PBS (which was tested not to interfere with the test) to remove all remaining LB medium. Then cells were resuspended with a 10mM NaOH solution and all cells removed by centrifugation to perform the test on the supernatant, that now should contain hydrolysed silicic acid. However, we believe that the cells were partly lysed with the NaOH solution since when trying to perform the test aggregation occurred again. Thus, we had to drop the test and decided not to characterize silicatein kinetics.
Growth study: plated cells from samples taken yesterday at timepoints 0, 1h, 2h, 3h, 4h and 5h at different dilutions (no dilution, 1/10, 1/100, 1/100). All plates contained to many colonies to be counted so the growth study will be performed again at higher dilutions.
Célina
Plasmid isolation from overnight cultures from the other day.
6th October 2016
Lycka
Transformation to BL21 of BolA ind (corrected sequence, fixed by PCR). Tube 3 from minipreps Célina (yesterday)                                                 was used to transform.
Bola ind (CM)
BolA ind + OmpA Sil Taur (CM + AMP)
MiliQ - control
RFP pSB1C3 + control
Liza
Preparation of TEM samples OmpA-Silicatein + silicic acid and without silic acid as protocol on a Quantifoil carbon grid. TEM samples of 3rd october and today are imaged using TEM under supervision of Yoones Kabiri.
Preparation of OmpA-silicatein + silicic acid and without silicic acid for AFM measurements as AFM cell fixation protocol .
7th October 2016
Liza
AFM measurements of AFM samples of 7th October under supervision of Allard Katan.
10th October 2016
Lara
Restriction of plasmids to put the inserts into the standard backbone from the Addgene backbones.
Nanodrop
BioBrick
Concentration (ng/µL)
BolA_con
97.6
BolA
62.5
Sil_Sdom
35.5
OmpA_Sil_Taur
48.3
INP_Sil_Sdom
58.5
pSB1C3
150.4
Since the amount left of BolA and RFP pSB1C3 was too little, these needed to grow from cryostocks again to be miniprepped.
Restrictions were performed and the backbone was gel-isolated; the rest did not yield restriction products visible on the electrophoresis gel.
Lycka
Inoculate cultures in fresh LB for growth study and confocal. ODs measured at 600nm in spectrophotometer.
OD600
BioBrick
OD measured (4x diluted)
OD inucolated
OmpA Sil Taur
0.90
0.07
INP Sil Sdom
0.89
0.07
Sil Sdom
0.87
0.07
BolA ind
0.82
0.065
OmpA Sil Taur (no silicate)
0.90
0.07
BolA ind
0.82
0.065
BolA ind + OmpA Sil Taur
0.82
0.065
Cryostocked OmpA Sil Taur + BolA ind and BolA ind in BL21.
Tessa
Measured OD600 of [2 colonies of OmpA_Sil_Taur (OmpA)], INP_Sil_Sdom (INP), Sil_Sdom (Sil), OmpA+BolA_ind, and BolA_ind cells for growth study, confocal and SEM, and incubated them until the OD600 was around 0.7.
Induced cultures [2 colonies of OmpA], INP, Sil and OmpA+BolA_ind
After 3hr incubation silicate was added to 1 colony of OmpA, INP, Sil and OmpA+BolA_ind
BolA_ind and OmpA+BolA_ind was incubated for 4 hours and then studied with a confocal microscope.
After growth study all samples were autoclaved for SEM
Tessa and Lara
A growth study was performed for OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom and BolA_ind.
Growth study (number of colonies)
Dilution
OmpA_Sil_Taur
INP_Sil_Sdom
Sil_Sdom
BolA_ind
Negative control (OmpA without silicate)
t=0
10-2
inf
inf
inf
inf
inf
10-3
1300
inf
inf
inf
inf
10-4
337
inf
inf
1003
1150
10-5
190
2400
inf
447
610
10-6
128
1300
2400
814
700
t=1
10-2
5
no data
102
inf
inf
10-3
1
34
15
3080
inf
10-4
0
0
2
718
1000
10-5
0
0
2
872
860
10-6
0
1
0
494
900
t=2
10-2
0
51
2
inf
inf
10-3
0
6
1
inf
inf
10-4
0
0
0
inf
2000
10-5
0
2
1
inf
1300
10-6
0
0
0
1396
970
t=3
10-2
0
1400
2
inf
inf
10-3
0
317
1
inf
860
10-4
0
84
1
inf
410
10-5
0
29
0
678
538
10-6
0
22
0
664
560
t=4
10-2
0
0
1
inf
inf
10-3
0
0
0
inf
540
10-4
0
0
1
inf
323
10-5
0
0
0
inf
303
10-6
0
0
0
300
193
t=5
10-2
0
0
0
inf
inf
10-3
0
0
1
inf
409
10-4
0
0
0
inf
190
10-5
0
0
0
800
72
10-6
0
0
0
1400
890
Liza
Calibration curve EGFP. Dilutions of 0,  1/100, 1/200, 1/400, 1/800 and 1/1600 are made of the purified EGFP sample with a concentration of 98µM.                                                  Dilutions are made in PBS.
11th October 2016
Lara & Lycka
Tried miniprepping straight from the cryostocks to yield more isolated plasmids.
Nanodrop was performed by Lycka.
Nanodrop
BioBrick
Concentration (ng/µL)
Sil_Sdom
86.9
OmpA_Sil_Taur
55.2
INP_Sil_Sdom
55.7
BolA_con
196.7
Lycka
Pour LB agar plates with chloramphenicol and ampicilin.
Inoculate for SEM, TEM, confocal and solar cell experiments.
OmpA Sil Taur
INP Sil Sdom
Sil Sdom
BolA ind
BolA ind + OmpA Sil Taur
12th October 2016
María
Preparation of samples for imaging experiments: strains expressing OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom, BolA and OmpA_Sil_Taur together with BolA were grown following the Polysilicate layer in silicatein expressing cells protocol and divided for NileBlueA staining, Rhodamine123 staining, TEM, SEM and solar cell experiments. Stained samples were prepared the same day, the rest were kept as a pellet in the fridge for the following day.
Lycka
Restriction of Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom, BolA_con, BolA_ind with EcoRI and PstI to put in standard backbone. Load restriction product on gel. Run at 100 mA for 45 min. No bands visible.
Try to PCR with standard primers and digest the PCR product. Phusion Polymerase used. Annealing temp 56ºC; extention time 90 sec. 5µL PCR product was put on gel.
Purification and nanodrop done by Charlotte. Respective concentrations:
124.3 ng/µl
89.2 ng/µl
96.4 ng/µl
104.6 ng/µl
135.8 ng/µl
Lara
To test which settings for UV sterilization are effective, different powersettings within the Stratagene UV Stratalinker 1800 were used. 700 µL of cells transformed with BolA_con in pSB4A5 were loaded in an eppendorf tube and were separately treated with the lid open, so the UV light could be transmitted. The used powersettings were 100000µJ, 120000µJ, 140000µJ, 160000µJ, 180000 µJ, 200000 µJ, 220000µJ and 240000 µJ. Afterwards 50 µL was plated on LB+Amp plates.
13th October 2016
María
Preparation of samples for SEM, trying both freeze drying and glutaraldehyde fixation.
Lycka
Restriction of PCR products of Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom, BolA_con, BolA_ind with EcoRI and PstI. Gel purify BolA ind, because its current backbone has                                                 the same resistance as the standard backbone (chloramphenicol). Normally purify others (they come from an amp resistant backbone).                                                 Concentrations after purification:
40.8 ng/µl
22.6 ng/µl
34.8 ng/µl
36.0 ng/µl
5.0 ng/µl
Ligation of 100 ng vector with respective inserts. 1:3 molar ratio used. Incubated for 3 hours.
Transformation to TOP10. Plated after 1.5 hour incubation in nonselective LB. Plated on LB CM plates.
Lara
Checked if there were colonies after UV sterilization. Colonies did appear. We figured that the UV light was not transmitted through the entire layer of 700 µL of cells, so I actually plated non sterilized cells. For that, a new method was used.
New method UV sterilization: Plating 30 µL BolA pBbA5C transformed cells directly on selective LB plates and sterilizing these with the recommended powersetting of 120000µJ and 240000µJ. As positive controls 500 µL cells were washed 3 times with ethanol or heated at 95 ºC for 10 minutes; as negative control living cells were plated. All were incubated at 37 ºC overnight.
Solar cell experiment preparation: First, all cell pellets were diluted as such that same cell concentrations were yielded. Afterwards, these cells were UV sterilized 3 times with a power of 240000 µJ.
Liza
Preparation of TEM samples OmpA-Silicatein + silicic acid and without silic acid as protocol on a Quantifoil carbon grid and CUUB continuous grid. Sample without silicic acid added was imaged by TEM by Yoones Kabiri.
14th October 2016
María
Colony PCR of fragments cloned into the standard backbone to be submitted to iGEM.
There were positive results for all constructs, 3 colonies of each were picked to isolate plasmids and cryostock.
Lara
Checked if there were colonies after UV sterilization. No colonies appeared in both powersettings.
Lara & Giannis
Solar cell experiment: Visited Stefaan Heirmans at the TU Delft to properly test the change in efficiency in solar cells, using a solar simulator. A total of 3 µL was loaded onto the solar cell and the efficiency was measured.
Solar cell efficiency
Type of cells
Solar cell efficiency %
Empty solar cell
9.75
MilliQ
9.92
OmpA_Sil_Taur not induced and no silicate
9.64
OmpA_Sil_Taur induced and no silicate
9.75
OmpA_Sil_Taur induced and silicate
9.61
BolA + OmpA_Sil_Taur induced and no silicate
9.65
OmpA_Sil_Taur induced and silicate
9.71
Since the bubble of liquid also worked as a lens, we figured these were not the most conclusive results. For that, we have decided to redo these experiments, with cells dried onto glass plates.
15th October 2016
Lara
Plasmid isolation of Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom, BolA_con, BolA_ind in the standard backbone.
Nanodrop
BioBrick
Colony number
Concentration (ng/µL)
OmpA_Sil_Taur (pSB1C3)
1
104.9
2
103.5
3
97.4
INP_Sil_Sdom (pSB1C3)
20
141.2
21
100.7
22
107.5
Sil_Sdom (pSB1C3)
39
111.1
40
80.3
41
95.1
BolA_con (pSB1C3)
63
63.2
65
75.2
66
95.9
BolA_ind (pSB1C3)
83
120.4
95
75.3
These plasmids were sent to sequencing and were sequence confirmed.
16th October 2016
Lara
Made dilutions of the samples that are going to be tested in the solar cell simulations. All samples: OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom, BolA_ind and BolA_ind+OmpA_Sil_Taur were diluted 10 times, 100 times and 1000 times. Subsequently, 1.5 mL was pipetted on glass plates that were provided by Stefaan and previously devided in four separate compartments. Samples were dried overnight at 37 ºC.
Back to Top
Protocols
During our lab work we used the following protocols.
Chemically competent Cells
PCR (Phusion)
Gel Isolation (From Promega Wizard™ Kit)
Restriction (NEB enzymes)
Ligation
Transformation (TOP10)
Transformation (BL21)
Transformation (OneShot® TOP10)
Colony PCR (Q5® High-Fidelity 2X Master Mix)
Colony PCR (GoTaq)
Colony PCR (DreamTaq)
DNA Electrophoresis
Overnight Cell Cultures
Cell Frozen Stock
Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)
Restriction Product Purification (from Promega Wizard™ Kit)
Polysilicate layer in silicatein expressing cells
Growth study
Fluorophores kinetic cycle
Silicic acid test
Nile Blue A staining
Rhodamine123 staining
UV sterilization
Gibson Assembly
eM9 medium recipe
TEM sample preparation (samples from Wageningen UR)
TEM sample preparation (samples from TU Delft)
SEM sample preparation
Cell fixation for AFM
Chemically competent cells
Grow overnight culture from single colony or competent cells aliquot.
Inoculate 1:100 in the desired volume of LB.
Grow to OD 600 ~0.35 (important that it is close to 0.35, 0.3-0.4 is good).
Chill on ice for 20 min (it can chill for a 1-2 hours).
Centrifuge at 8000g for 8 minutes in Falcon tubes, decant supernatant.
Resuspend by pipetting gently in 1/10 (of the volume of LB from step 2) of ice-cold TSS.
NOTE: It can take a while to break up the pellet (you can use a 25 mL pipette and scrape the pellet off the side of the tube before pipetting).
Recipe for TSS (100 mL):
10 g PEG 3350
5 mL DMSO
2 mL 1 M MgCl2
LB to 100 mL
Filter sterilize and chill to 4ºC
Freeze 100 µL aliquots in liquid nitrogen & store at -80ºC in 1,5mL Eppendorf tubes.
ALWAYS do a transformation positive control using a plasmid that you know is efficiently taken up by competent cells to confirm competency.
Just as important: ALWAYS do a negative control transformation (add untransformed cells to a plate with antibiotics) to confirm that you don't have an antibiotic-resistant contaminant strain in your competent cell stock. This happens from time to time and can cause lots of agony!
Back to Protocols
PCR (Phusion polymerase)
Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
Component
Volume (µL) for 20µL reaction
Volume (µL) for 50µL reaction
5X Phusion HF Buffer
4
10
10mM dNTPs
0.4
1
FW Primer
1
2.5
RV Primer
1
2.5
Phusion polymerase
0.2
0.5
Template DNA (<250ng)
Variable
Variable
Nuclease-free H2O
Up to 20µL
Up to 50µL
NOTE: The 250ng of template DNA are approximate, choose a volume that works fine for all your samples and fill it in the excel sheet so you are able to prepare a mix for all samples at the same time.
Add the total volume per sample (without DNA) to each PCR tube.
Add the template DNA to each tube.
Put the tubes in the PCR machine and apply the following program (it needs to be adjusted for primers annealing temperature and extension time):
Step
Temperature (ºC)
Time
Initial denaturation
98
30sec
Denaturation
98
10sec
x30 cycles
Annealing
Tm-3
20sec
Extension
72
15-30sec/1kb
Final extension
72
5min
4
Hold
Run a DNA electrophoresis following the DNA electrophoresis protocol.
Back to Protocols
Gel isolation
Run a DNA electrophoresis (following the DNA electrophoresis protocol), loading the full volume of PCR or restriction product into the agarose gel. If necessary, use multiple wells.
Cut the band (or bands) corresponding to the fragment of interest with a razor blade and put it in a pre-weighted 1.5mL tube, then calculate the weight of the gel fragment.
Add 10µL of Membrane Binding Solution per 10mg of gel slice. Vortex and incubate at 50-65ºC until gel slice is completely dissolved.
Pre-warm an aliquot of Nuclease-free water at 50-65ºC.
Insert SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.
Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate at room temperature for 1 minute.
NOTE: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Add 700µL of Membrane Wash Solution (if it is the first use, dilute it with 95% ethanol following the bottle’s instructions).
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Repeat steps 9-11 with 500µL of Membrane Wash Solution and centrifuging for 5 minutes.
Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.
Transfer the SV Minicolumn to an empty 1.5mL tube carefully labelled.
Add 500µL of the pre-warmed water (300µL for higher concentrations or when small amounts of DNA are suspected) directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.
Incubate at room temperature for 5 minutes.
Centrifuge at maximum speed for 1 minute.
Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep at 4ºC (for immediate use) or -20ºC (for storage).
Back to Protocols
Restriction
Get the detailed protocol for restriction from the following website , according to the enzymes used. When high restriction product concentrations are needed (generally for backbones) the amount of DNA can be increased to 3-5g. The volume of DNA necessary can be obtained from the concentration measured at NANODROP.
Mix all the components stated by this restriction tool in a 1.5mL Eppendorf tube and incubate at 37ºC for 1-2h.
Back to Protocols
Ligation
Make your own ligation protocol.
Ligation Protocol
Ligation Protocol
Vector length (bp) :
Vector mass (ng)   :
Vector concentration (ng/µL) :
Insert length (bp)  :
Insert concentration (ng/µL) :
Ratio Insert/Vector  :
Calculate
Mix all the components in a tube (always add the T4 ligase at the end).
Incubate for at least 3 hours at 16ºC.
Back to Protocols
Transformation (Top10 chemically competent cells)
Get as many aliquots of competent cells (100L) from the -80ºC freezer as transformations to be done and put them on ice for 10-15min.
NOTE: Don’t forget positive (RFP in the backbone you are using, check plasmids box in the -20ºC freezer) and negative controls (no DNA).
Add to the 100µL of competent cells:
1µL of plasmid or 2µL of ligation product.
20µL 5X KCM buffer.
Sterile water up to 200µL.
Mix by flicking the tube.
Incubate on ice for 20-30 minutes.
Heat shock at 42ºC for exactly 90 seconds.
Return the cells to ice for 1-2 minutes.
Add 400µL of fresh LB.
Incubate at 37ºC with shaking for 1 hour.
NOTE: Not at the thermoblock, use a shaking incubator.
To increase the concentration of cells before plating, centrifuge the tubes at 4000rpm for 3 minutes, remove 500L of medium, resuspend the pellet in the remaining volume.
Plate the rest of the volume of the transformation on selective LB plates and incubate at 37ºC overnight.
Back to Protocols
Transformation (BL21 competent cells)
Put a tube of BL21 Competent E. coli cells on ice for 10 minutes.
Add 2µL of plasmid DNA to the cell mixture.
Mix by flicking the tube.
Place the tube on ice for 30 minutes.
Heat shock at 42ºC for exactly 10 seconds.
Place on ice for 5 minutes.
Add 950µL of SOC medium at room temperature.
Incubate at 37ºC with shaking for 1 hour.
NOTE: Not at the thermoblock, use a shacking incubator.
To increase the concentration of cells before plating, centrifuge the tubes at 4000rpm for 3 minutes, remove 700µL of medium, resuspend the pellet in the remaining volume.
Plate 70µL of the transformation on selective LB plates and incubate at 37ºC overnight.
Back to Protocols
Transformation (OneShot® Top10 Chemically competent cells)
Get half the amount of tubes of OneShot® TOP10 chemically competent cells (tubes with a purple lid) than the transformations to be done from the -80ºC freezer and put them on ice for 10-15min.
Divide the volume of competent cells of each tube in 2. Thus, transfer 25L to a new sterile tube.
Add 1µL of plasmid or 2µL of ligation product into each tube and mix by flicking.
Incubate on ice for 30 min.
Heat shock the cells for 30 seconds at 42ºC.
Put back on ice for 2 minutes.
Add 250µL of LB medium.
Incubate at 37ºC with shaking for 1 hour.
NOTE: Not at the thermoblock, use a shacking incubator.
To increase the concentration of cells before plating, centrifuge the tubes at 4000rpm for 3 minutes, remove 200µL of medium, resuspend the pellet in the remaining volume.
Plate the rest of the volume of the transformation on selective LB plates and incubate at 37ºC overnight.
Back to Protocols
Colony PCR (Q5® High-Fidelity 2X Master Mix)
Choose the colonies to be picked for testing and number them. When available pick at least 8 colonies per plate, or more if there are slots left in the PCR machine (it fits 25 samples).
NOTE: It is more convenient if the numbering is continued between plates of the same batch instead of starting at 1 in each plate. This way there is only one set of numbers per date.
NOTE: it is possible that not all samples fit in one PCR and then more than one would need to be performed, so keep that in mind for the timing of experiments. But don’t be sad, that would mean that the transformation worked so well that you have a lot of colonies to choose from!! ☺
Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
Component
Volume (µL)
Q5® HF 2X Master Mix
12.5
Primer VF2
1.25
Primer VR
1.25
Nuclease-free H2O
10
Total
25
Pipette 25µL of mix into each PCR tube.
Under sterile conditions, pick a colony making sure that part of it stays on the plate for later use and soak it in the content of the PCR tube. When finished, put them back in the incubator so they grow for a bit longer while the PCR runs.
Put the tubes in the PCR machine and apply the following program (applicable for the primers mentioned above, for other sets of primers the annealing temperature needs to be adjusted):
Step
Temperature (ºC)
Time
Cell lysis and initial denaturation
95
15min
Denaturation
95
30sec
x30 cycles
Annealing
53
30sec
Extension
72
30sec/1kb
Final extension
72
5min
4
Hold
While the PCR is running, prepare a 0.5mL tube with 2µL loading buffer for each PCR sample and one tube with 4µL of 1Kb DNA ladder (or 8µL if 2 gels are needed). This way you can take these tubes into the EtBr area and don’t worry about how to handle the PCR tubes.
Cast an agarose gel following the DNA electrophoresis protocol.
Prepare an agarose gel with SNAPGENE for comparison with the actual results.
Once the PCR program is finished, transfer 8L of PCR product into the tubes containing the loading buffer and follow the DNA electrophoresis protocol.
Back to Protocols
Colony PCR (GoTaq)
Under sterile conditions, pick a colony and dilute it in 25µL of miliQ water.
Incubate at 95ºC for 10 min.
Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
Component
Volume (µL)
GoTaq Mastermix 2X
25
Primer VF2 (10µM)
1
Primer VR (10µM)
1
Nuclease-free H2O
18
Total
50
Pipette 45µL of mix into each PCR tube (one per colony).
Centrifuge the colony mixture for 30 seconds at 10000rpm.
Add 5µL of supernatant of colony mixture to each PCR tube.
Put the tubes in the PCR machine and apply the following program (applicable for the primers mentioned above, for other sets of primers the annealing temperature needs to be adjusted):
Step
Temperature (ºC)
Time
Initial denaturation
95
2min
Denaturation
95
30sec
x30 cycles
Annealing
53
30sec
Extension
72
1min/1kb
Final extension
72
5min
4
Hold
While the PCR is running, prepare a 0.5mL tube with 2µL loading buffer for each PCR sample and one tube with 4µL of 1Kb DNA ladder (or 8µL if 2 gels are needed). This way you can take these tubes into the EtBr area and don’t worry about how to handle the PCR tubes.
Cast an agarose gel following the DNA electrophoresis protocol.
Prepare an agarose gel simulation with SNAPGENE for comparison with the actual results.
Once the PCR program is finished, transfer 8µL of PCR product into the tubes containing the loading buffer and follow the DNA electrophoresis protocol.
Back to Protocols
Colony PCR (DreamTaq)
Under sterile conditions, pick a colony and dilute it in 25µL of miliQ water.
Incubate at 95ºC for 10 min.
Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
Component
Volume (µL)
DreamTaq Hot Start PCR Master Mix (2X)
25
Primer VF2 (10µM)
1
Primer VR (10µM)
1
Nuclease-free H2O
18
Total
50
Pipette 45µL of mix into each PCR tube (one per colony).
Centrifuge the colony mixture for 30 seconds at 10000rpm.
Add 5µL of supernatant of colony mixture to each PCR tube.
Put the tubes in the PCR machine and apply the following program (applicable for the primers mentioned above, for other sets of primers the annealing temperature needs to be adjusted):
Step
Temperature (ºC)
Time
Initial denaturation
95
3min
Denaturation
95
30sec
x30 cycles
Annealing
53
30sec
Extension
72
1min for first 2kb + 1min/1kb
Final extension
72
5min
4
Hold
While the PCR is running, prepare a 0.5mL tube with 2µL loading buffer for each PCR sample and one tube with 4µL of 1Kb DNA ladder (or 8µL if 2 gels are needed). This way you can take these tubes into the EtBr area and don’t worry about how to handle the PCR tubes.
Cast an agarose gel following the DNA electrophoresis protocol.
Prepare an agarose gel simulation with SNAPGENE for comparison with the actual results.
Once the PCR program is finished, transfer 8µL of PCR product into the tubes containing the loading buffer and follow the DNA electrophoresis protocol.
Back to Protocols
DNA electrophoresis
Work clean! Handle with gloves all material labelled as EtBr contaminated, don’t take it outside of the EtBr area and don’t touch with gloves anything that is not labelled as EtBr contaminated.
Prepare TAE buffer: take the 10X concentrated TAE from the chemicals cabinet and dilute it 10 times with demi water. For 500mL, add 50mL to 450mL of dH2O.
Weigh agarose for a 1% gel. For 200mL, 2g of agarose is necessary.
Mix the TAE solution with the agarose and heat the solution until it is completely dissolved.
Add Ethidium bromide to the gel mould. 2.5L for the small gel and 5L for the big one.
Pour the solution into the mould making sure there are no bubbles and that the EtBr is completely mixed. Let it solidify (approx. 20 minutes).
Transfer the gel to the electrophoresis cell minding the arrow that indicates the direction of DNA migration, remove the combs and cover it with TAE.
Prepare tubes with 5/6 of PCR product and 1/6 of loading buffer.
Load the molecular weight marker (ladder) in the first well (check the appropriate volume for each marker, generally 5L works fine) and load the samples in the other wells, according to the order in your lab journal.
NOTE: Do not contaminate the loading buffer and ladder with EtBr! Do not touch it while wearing a glove.
Connect the cables following the colour code and run at 100-130V for 30-60 min.
Back to Protocols
Restriction product purification
For restriction products containing more than one large fragment follow the first 3 steps of the Gel isolation protocol.
When restriction yields only one large fragment, add an equal volume of Membrane Binding Solution to the volume of restriction product.
Pre-warm an aliquot of Nuclease-free water at 50-65ºC.
Insert SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.
Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate at room temperature for 1 minute.
NOTE: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Add 700µL of Membrane Wash Solution (if it is the first use, dilute it with 95% ethanol following the bottle’s instructions).
Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.
Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.
Repeat steps 9-11 with 500µL of Membrane Wash Solution and centrifuging for 5 minutes.
Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.
Transfer the SV Minicolumn to an empty 1.5mL tube carefully labelled.
Add 500µL of the pre-warmed water (300µL for higher concentrations or when small amounts of DNA are suspected) directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.
Incubate at room temperature for 5 minutes.
Centrifuge at maximum speed for 1 minute.
Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep at 4ºC (for immediate use) or -20ºC (for storage).
Back to Protocols
Cell frozen stock
Work sterile! If plasmid DNA needs to be purified from the overnight culture, remove the necessary volume from the Falcon tube before the cryostock procedure.
Centrifuge the 50mL tubes at 2000rpm for 10 min.
Decant the supernatant without disturbing the pellet.
Add fresh LB medium, around 1/3 of the starting volume of the culture.
Mix by vortexing until the pellet is completely resuspended.
Add sterile 80% glycerol solution, the same volume as fresh LB in step 4.
Mix by vortexing.
Make 1.5mL aliquots in cryotubes and label them with the cell type, the BioBrick and backbone they contain (if applicable) and the date.
Store them at -80ºC and update the inventory in the drive folder.
Back to Protocols
Plasmid isolation
Pre-warm an aliquot of Nuclease-free water at 50-60ºC.
Add 1.5 mL of bacterial culture in LB medium to a 1.5 mL micro-centrifuge tube. Centrifuge that tube at max speed for 3 min.
NOTE: For higher concentrations repeat step 1 in the same tube.
NOTE: If cryostocks need to be prepared from the same overnight culture tube, remove the necessary volume for plasmid isolation under sterile conditions!
Remove the supernatant.
Resuspend the pellet in 600 µL of MiliQ water.
Add 100 µL of Cell Lysis Buffer, and mix by inverting 6 times. The color change to blue indicates complete lysis.
Add 350 µL of cold (4-8ºC) Neutralization Buffer, and mix by inverting the tube. The color change to yellow indicates total neutralization.
Introduce a PureYield Minicolumn into a PureYield Collection tube.
Centrifuge at maximum speed for 3 minutes, and transfer the supernatant to the Minicolumn assembly.
NOTE: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.
Centrifuge at maximum speed for 15 seconds and discard the flowthrough.
Add 200 µL of Endotoxin Removal Wash to the minicolumn. Centrifuge at maximum speed for 15 seconds. It is not necessary to empty the Collection Tube now.
Add 400 µL of Column Wash Solution to the minicolumn, and centrifuge at maximum speed for 30 seconds.
Transfer the PureYield Minicolumn to a clean 1.5 mL tube, add 30 µL (or 50L if you expect high concentrations) of pre-warmed MiliQ water directly to the minicolumn matrix. Let stand for 5 minutes at room temperature.
Centrifuge at maximum speed in a microcentrifuge for 15 seconds to elute plasmidic DNA. Cap the tube, and store the DNA solution at -20 ºC (or keep it at 4ºC if it is to be used immediately).
NOTE: If possible, put the lids in line with the circle of the centrifuge (so not pointing out or inwards) so that the caps don’t break while centrifuging.
Back to Protocols
Overnight cell cultures
Prepare 20mL of selective LB (with the antibiotic your cells are resistant to) per colony to be picked.
NOTE: You can use a sterile bottle to prepare the total volume.
Transfer 10mL of selective LB to a 50mL Falcon tube. Prepare 2 tubes per colony: 1 for miniprep and 1 for cryostocks.
Pick 1 colony using an inoculation loop and soak it in the LB. Make sure you have cells left for the second tube.
Take the tubes to room F1.690 and incubate overnight at 37ºC in the shaking incubator.
Back to Protocols
Polysilicate layer in silicatein expressing cells
Inoculate a fresh culture of cells from an overnight culture. Make a dilution of around 1/100 of the overnight culture into fresh selective LB.
Incubate in the shaking incubator at 37ºC and 250 rpm until it reaches exponential phase (just measure OD600 from time to time until duplication time is close to 20 min).
Induce the culture with 1/1000 volume of IPTG 1M.
Incubate in the shaking incubator at 37ºC and 250 rpm for 3h.
Add sodium silicate to a final concentration of 60µM from the 1mM stock (maybe new stock needs to be made from the 1M stock, keep sterile!!).
Incubate at 37ºC and 250 rpm for 3-4h, after this the cells are ready for further experiments.
Back to Protocols
Growth study
Follow the Polysilicate layer protocol until step 5.
Take a 100µL sample directly after adding the sodium silicate, this is t=0.
Incubate at 37ºC and 250 rpm and take a 100µL sample every hour.
For each sample:
Make a series of dilutions in 1/10 steps until reaching a 1/106. It can be done in a 96 well plate by adding 10µL of the previous dilution in 90µL of LB medium.
Plate in agar plates 20µL of dilutions 1/100, 1/103, 1/104, 1/105, 1/106.
Incubate at 37ºC overnight.
Hopefully, in some of the dilutions the colonies will be countable. Count colonies in plates for at least one dilution of each strain and time point to make a growth/viability curve.
Back to Protocols
Fluorophores kintetic cycle
Grow cells expressing the fluorophores of interest in eM9 medium since LB has its own fluorescence.
Measure OD600 overnight cultures.
Dilute in eM9 medium an aliquot of each culture to an OD600 of 0.1.
Add 100µL of diluted culture to 4 wells per culture (that way we obtain 4 replicas of the experiment), also make blanks with sterile eM9.
Program the plate reading machine so it incubates the plate at 37ºC with shaking and makes an OD600 and fluorescence measurement every 15min for 24h.
Excitation and emission wavelengths to be used for each fluorophore are:
Excitation (nm)
Emission (nm)
mVenus
510
544
mCerulean
433
475
GFP
488
522
NOTE: Some of these wavelengths are adjusted to be at least 34nm apart due to machine limitations.
Back to Protocols
Silicic acid test
For a 1 mL sample:
Add 20µL of reagent Si-1 and mix.
Wait for 3 min.
Add 20µL of reagent Si-2 and 100µL of Si-3 and mix.
Wait for 10 min.
Measure absorbance at 824nm.
Back to Protocols
Nile Blue A staining
Heat fix cells in a glass slide.
Stain the glass slide with 1% Nile Blue A solution at 55ºC for 10 min.
Wash with tap water to remove excess stain.
Wash in 8% aqueous acetic acid for 1 min.
Wash and dry.
Remoist with tap water.
Cover with a cover slip and seal it with nail polish.
Nile Blue A is used to visualize PHB inside the cells. It is fluorescent when in contact with PHB and excited at around 460nm and fluoresces as bright orange ( Ostle & Holt, 1982 ).
Back to Protocols
Rhodamine123 staining
Add 1/1000 volume of Rhodamine123 to the culture to be stained.
Wait for 10 min.
Centrifuge for 1 min at maximum speed.
Wash the cells 5 times with PBS.
Prepare samples for microscopy in a glass slide.
Rhodamine123 is a fluorescent dye which is excited at 546nm and emits green fluorescence ( Li, Chu & Lee, 1989 )( Johnson, Walsh & Chen, 1980 )
Back to Protocols
UV sterilization
Plate 30µL of cells on selective LB plates.
Positive control: dead culture (for example, autoclaved).
Negative control: culture of living cells.
Test: culture of living cells.
Expose the test plates to UV light using the Stratagene UV Stratalinker 1800 at power 120000µJ and 240000µJ, respectively.
Incubate all plates at 37ºC overnight.
Back to Protocols
eM9 medium recipe
10X M9 stock solution
75 g/L Na2HPO4·2H2O
30 g/L KH2PO4
5 g/L NaCl
10 g/L NH4Cl
eM9 medium for protein overexpression
1X M9 stock solution
10 mL/L CaCl22H2O, 10mM
1 mL/L MgSO47H2O, 1M
5 mL/L L-Tryptophan, 1%
100mM Glucose
Back to Protocols
Gibson assembly
Put in a PCR tube:
100ng of vector
100ng of insert
2µL of buffer
Up to 10µL of Nuclease-free water
Incubate at 37ºC for 1h in a thermocycler.
Back to Protocols
SEM sample preparation
Centrifuge 1mL of culture for 1min at maximum speed.
Discard the supernatant
Resuspend the pellet with 3% glutaraldehyde solution.
NOTE: Glutaraldehyde is a dangerous chemical, wear gloves and safety goggles!
Let it stand for 10min.
Centrifuge for 1 min at maximum speed.
Discard the supernatant.
Resuspend the pellet in a 30% ethanol solution.
Centrifuge for 1 min at maximum speed.
Discard the supernatant.
Repeat steps 7-9 with ethanol solutions of 50%, 70%, 80%, 90%, 96% and 100%.
Resuspend the supernatant one last time in 100% ethanol.
The samples are ready to be put in a glass slide.
Preparation of slides and SEM images were done by Dominik Schmieden (see advisors page)
Back to Protocols
TEM sample preparation (Wageningen samples)
Take plasma cleaned 400 mesh carbon grid.
Put 3 µl of sample (lipids) on the carbon grid and incubate 1 min.
Absorb sample with a filter.
Wash the sample 3x with MiliQ water absorbing it every time with a filter.
Add 3µl 3% of uranyl acetate and incubate for 30 seconds.
Remove stain with filter.
Image with high angle annular dark field transmission electron microscopy with the help of Yoones Kabiri (see acknowledgements page).
Back to Protocols
TEM sample preparation (TU Delft samples)
Grow cells in LB centrifuge and remove supernatant.
Dilute pellet in MiliQ water.
Take a TEM grid and put 3µl polylysine (1%) and incubate 1 min.
Wash the grids with MiliQ water.
Drop cast 3µl of sample and incubate 1 min.
Wash with MiliQ water 3 times.
Image with high angle annular dark field transmission electron microscopy  with the help of Yoones Kabiri (see acknowledgements page).
Back to Protocols
Cell fixation for AFM
Dilute a cell pellet in MQ.
Take coverslips and plasma clean (oxygen) for 20 seconds.
Make 1% APTES in MQ.
Pipet a few drops of the APTES solution onto the coverslip and incubate for 2 minutes.
Dip the coverslip in MQ a few times to wash.
Pipet a few drops (~100µl) of bacterial culture onto the coverslip and incubate for 2 minutes.
Dip the coverslip in MQ a few times to wash.
Dry the coverslip with a nitrogen gun.
Optional: check density of fixed cells with normal microscope.
Cells are imaged and measured with the help of Allard Katan (see acknowledgements page) in an Atomic force microscope.
Back to Protocols
References
Cha, J. N., Shimizu, K., Zhou, Y., Christiansen, S. C., Chmelka, B. F., Stucky, G. D., & Morse, D. E. (1999). Silicatein filaments and subunits from a marine sponge direct the polymerization of silica and silicones in vitro. Proceedings of the National Academy of Sciences of the United States of America, 96(2), 361–365.
Johnson, L. V., Walsh, M. L., & Chen, L. B. (1980). Localization of mitochondria in living cells with rhodamine 123. Proceedings of the National Academy of Sciences of the United States of America, 77(2), 990–994.
Li, C. W., Chu, S., & Lee, M. (1989). Characterizing the silica deposition vesicle of diatoms. Protoplasma, 151(2-3), 158–163. http://doi.org/10.1007/BF01403453
Müller, W. E. G., Krasko, A., Pennec, G. Le, Steffen, R., Wiens, M., Ammar, M. S. A., Schröder, H. C. (2003). Molecular Mechanism of Spicule Formation in the Demosponge Suberites domuncula: Silicatein-Collagen-Myotrophin. In W. E. G. Müller (Ed.), Silicon Biomineralization: Biology --- Biochemistry --- Molecular Biology --- Biotechnology (pp. 195–221). Berlin, Heidelberg: Springer Berlin Heidelberg. http://doi.org/10.1007/978-3-642-55486-5_8
Ostle, A. G., & Holt, J. G. (1982). Nile blue A as a fluorescent stain for poly-beta-hydroxybutyrate. Applied and Environmental Microbiology, 44(1), 238–241.
Contact
Address:
Faculty of Applied Sciences
Room E0.100
Van der Maasweg 9
2629 HZ Delft
The Netherlands
Email:
tudelft.igem@gmail.com
Social media:
Facebook
Twitter
Instagram
Youtube
Sponsors
