<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Delft_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Delft/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-top"><DIV class="col-sm-6 col-xs-12"><UL class="list-unstyled list-inline header-contact"><LI><A href="mailto:tudelft.igem@gmail.com">tudelft.igem@gmail.com</A></LI></UL></DIV></DIV><DIV class="navbar-main"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV id="navbar" class="navbar-collapse collapse pull-right"><UL class="nav navbar-nav"><LI><A href="https://2016.igem.org/Team:TU_Delft/Achievements">ACHIEVEMENTS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Achievements#requirements">Medal Requirements</A></LI><LI class="submenu-item"><A href="">Special Prizes</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Design">Best Applied Design</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Hardware">Best Hardware</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Software">Best Software Tool</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Entrepreneurship">Best Entrepreneurship</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model">Best Model</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Composite_Part">Best Composite Part</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Integrated_Practices">Best Integrated Human Practices</A></LI></UL><LI><A href="https://2016.igem.org/Team:TU_Delft/Team">TEAM</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#members">Team Members</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#advisors">Advisors</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#fun">Fun Activities</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Team#media">Media</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Project">PROJECT</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#description">Description</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#results">Experiments</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#fluorophores">Fluorophore expression</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#silicatein">Silicatein expression</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#Biolaser">Biolaser</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#Biolenses">Biolenses</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#Conclusions">Conclusions</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Project#conclusions">Conclusions and Recommendations</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Safety">SAFETY</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Safety#workspace">Workspaces</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Safety#safety">Safety</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Notebook">NOTEBOOK</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Notebook#notes">Day Notes</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Notebook#protocols">Protocols</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Practices">PRACTICES</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#overview">Overview</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#product">Product Analysis</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#safety">Risk Assessment Tool</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#riskas">Risk Assessment Discussion</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#experts">Experts' Opinions</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#entrepre">Business Plan</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#igemana">iGEM Analysis</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#toolbox">iGEM Toolbox</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Practices#outreach">Outreach</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Model">MODELING</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model#lasers">Biolasers</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q1">Q1: Cavity size</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q2">Q2: Fluorophore concentration</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q3">Q3: Laser threshold</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q4">Q4: Cavity quality factor</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model#lenses">Biolenses</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q5">Q5: Focusing of light</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q6">Q6: Scattering of light</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Q7">Q7: Influence of polysilicate layer thickness</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory">Theory</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory#light">What is light?</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory#lasers">What is a laser?</A></LI><LI class="subsubmenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Model/Theory#mie">Mie Theory</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Hardware">HARDWARE</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Hardware#overview">Overview</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Hardware#building">Building the microscope</A></LI><LI><A href="/Team:TU_Delft/Collaborations">COLLABORATIONS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Collaborations#collab">Collaborations</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Collaborations#meetups">Meetups</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Parts">PARTS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#parts">All Parts</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#composite">Composite Parts</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#basic">Basic Parts</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Parts#collection">Parts Collection</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Attributions">ATTRIBUTIONS</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Attributions#attributions">Team Attributions</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Attributions#sponsors">Sponsors</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Attributions#acknowledgements">Acknowledgements</A></LI><LI><A href="https://2016.igem.org/Team:TU_Delft/Interlab">INTERLAB</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Interlab#results">Results</A></LI><LI class="submenu-item"><A href="https://2016.igem.org/Team:TU_Delft/Interlab#discussion">Discussion</A></LI></DIV></DIV><TITLE>iGEM TU Delft</TITLE><DIV class="page-heading text-center"><DIV class="container"><H1 id="description" class="page-header">Notebook</H1></DIV></DIV><DIV class="main-container"><DIV class="container"><DIV class="notebook"><H2 class="title-style-1">Day Notes</H2><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="false" aria-controls="collapseOne"><DIV class="panel-heading" role="tab" id="headingOne"><H4 class="panel-title">




                                        June




                                    </H4></DIV></A><DIV id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne"><DIV class="panel-body"><DIV class="row"><H2 class="title-style-3">29<SUP>th</SUP> June 2016 </H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Prepare 10mL of antibiotic stocks of both antibiotics that are going to be used during the project: Chloramphenicol (Cm) and Ampicillin (Amp). Make 500µL and 1mL aliquots.</P><P>Prepare 1L of LB medium and 3x400mL bottles of LB Agar. Send to autoclave.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">30<SUP>th</SUP> June 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Add antibiotics to the LB Agar bottles prepared the previous day and pour plates to have them ready for when we start transforming.</P></DIV></DIV><A href="#notes" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#notes').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo"><DIV class="panel-heading" role="tab" id="headingTwo"><H4 class="panel-title">




                                        July




                                    </H4></DIV></A><DIV id="collapseTwo" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo"><DIV class="panel-body"><DIV class="row"><H2 class="title-style-3">4<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Digestion gBlocks mVenus and mKate with EcoRI and PstI.</P><P>Made liquid culture of strain containing pSB4A5 bakcbone with RFP.</P><P><STRONG>María</STRONG></P><P>Digestion of pSB1C3 linearized backbone from the distribution kit with EcoRI and PstI.</P><P><STRONG>Charlotte</STRONG></P><P>Made two liquid overnight cultures of <I>E. coli</I> K12 harboring the standard backbone pSB4A5 containing RFP.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">5<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Dissolution of necessary parts from distribution kit: GFP BioBrick BBa_E0840 and promoters BBa_J23100, BBa_J23105, BBa_J23108, BBa_J23113 and BBa_J23117.</P><P>Dissolution of K1149051 (a kind gift of Imperial College), which was sent dry on filter paper.</P><P><STRONG>Lycka</STRONG></P><P>Ligation of mKate into backbone pSB1C3.</P><P>Transformation of ligation product mKate and the BioBricks dissolved by María into <I>Escherichia coli</I> Top10 cells. These were plated on LB medium supplemented with Cm.</P><P><STRONG>Charlotte</STRONG></P><P>Did 10 minipreps on the overnight culture with RFP pSB4A5, according to protocol.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">6<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Stocked primers VF2 and VR: storage stock (100µM) and working stock (10µM).</P><P>Colony PCR of transformants from yesterday with primers VF2 and VR followed by gel electrophoresis. Gel picture yielded no bands. This might be attributed to a fault in the transformation.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">7<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Transformation of same BioBricks as on 5<SUP>th</SUP> July using a different transformation protocol since we believed that the negative result was due to a mistake during transformation.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">8<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Finally, the plasmids containing K1149051 and BBa_E0840 yielded a few colonies. However, the ones containing the promoters did not. Thus, we believe that the distribution kit plate containing the promoters is defective and the mKate ligation also featured some problems.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">11<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>To test whether the distribution kit might be the problem, transformation of BBa_J23113 in OneShot® TOP10 chemically competent cells (Invitrogen) to make sure the transformation process is not the cause of the negative results. Also, a plasmid containing a construct from last year's team (CsgA) was used as a positive control.</P><P>For the mKate construct we will obtain the backbone from another construct instead of the linearized backbones from the iGem distribution kit.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">12<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Measure DNA concentration of biobricks from registry by nanodrop.</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Product</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>J23113</TD><TD>89.3</TD></TR><TR><TD>J23117</TD><TD>91.7</TD></TR><TR><TD>J23105</TD><TD>89.3</TD></TR><TR><TD>J23108</TD><TD>92.7</TD></TR><TR><TD>J23100</TD><TD>93.4</TD></TR><TR><TD>E0840</TD><TD>81.9</TD></TR></TBODY></TABLE><P>As can be seen from the table, there is DNA presence in the samples. Therefore, something else should be causing the transformations not to work. Since last year's distribution kit plates are still available in the lab and those plasmids were not taken we will try to get these parts from 2015 distribution kit.</P><P><STRONG>María</STRONG></P><P>Transfer colonies for K1149051 and BBa_E0840 to overnight culture on LB selective medium. A colony containing CsgA was transfered as well to use that plasmid to obtain the standard backbone for all synthesized parts.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">13<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Plasmid isolation of overnight cultures prepared yesterday and preparation of K1149051 and BBa_E0840 samples for sequencing.</P><P><STRONG>Lycka</STRONG></P><P>Cryostocks of K1149051 and BBa_E0840 overnight culures. Stored at -80ºC.</P><P>Digestion of backbones pSB1C3, pSB4A5 and all gBlocks with EcoRI and PstI.</P><P><STRONG>Lara</STRONG></P><P>Preparation of 2 bottles of LB agar.</P><P><STRONG>Liza </STRONG></P></DIV></DIV><DIV class="row"><H2 class="title-style-3">14<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Gel purification of digested backbones. Dissolved in nuclease free water, stored at -20ºC.</P><P>Ligation of synthesized fragments into backbones.</P><P><STRONG>María</STRONG></P><P>After all the problems transforming in the past weeks, we tested if we could use our homemade chemically competent cells for future transformations by transforming a couple of positive and negative controls, which were taken from last year's plasmids. The transformations worked so these cells can still be used.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">15<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Transformation of Lycka's ligation products from yesterday and the BioBricks from the distribution kit were given one last chance. Transformed cells were plated after doubling their concentration.</P><P><STRONG>Lycka</STRONG></P><P>Dissolution of promoter BioBricks from 2015 iGEM distribution kit.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">17<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Poured plates using 2 bottles of LB Agar, one for chloramphenicol plates and one for ampicilin.
                                            </P><P><STRONG>Iris</STRONG></P><P><B>Purification IDTs 
                                                    IDT 1-12, 50 µL Mem 
                                                    20 µL Nuclease free water</B></P></DIV></DIV><DIV class="row"><H2 class="title-style-3">18<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Restriction of backbones pSB1C3 and pSB4A5 with EcoRI and PstI. Gel electrophoresis of digested fragments. Gel picture yielded no bands.</P><P><STRONG>Célina</STRONG></P><P>Transformation of plasmids from Interlab Study into BL21.</P><P>Plated after doubling the concentration of cells by spinning down and removing half of the supernatant.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">19<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Transformation of BioBricks from the distribution kit of 2015, since the ones from 2016 yielded no colonies. Positive control: csgA. Negative control: water. Biobricks: J23100, J23108, J23105, J23117, J23113. Overnight cultivation yielded no result, we decide to drop this and add the promoters to the parts that need it by PCR.</P><P><STRONG>María</STRONG></P><P>Restriction of pSB1C3 and pSB4A5 backbones with EcoRI and PstI</P><P>Ligation of synthesized parts that could not be ligated on 14<SUP>th</SUP> of July: INP_Sil_Sdom in pSB1C3 and BolA_ind and BolA_con in both pSB1C3 and pSB4A5.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">20<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka and Tessa</STRONG></P><P>Transformation of plasmids ligated by Maria (19<SUP>th</SUP> of July) together with the ones ligated on 14<SUP>th</SUP> of July. After overnight culture the following plates contained colonies. pSB1C3: INP_Sil_Sdom, OmpA_Sil_Taur, Sil_Sdom, SulA, BolA_ind, BolA_con, phaP. pSB4A5: BolA_con. The ones with a negative result were ligated again the next day.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">21<SUP>st</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Colony PCR of yesterday's transformation results from Lycka and Tessa. However, not all transformations yielded colonies.</P><P>Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in selective LB overnight.</P><P><STRONG>Lycka</STRONG></P><P>Ligation of mKate, mVenus, mCerulean, OmpA_Sil_Sdom and LacI into pSB1C3. Ligation of OmpA_Sil_Taur, OmpA_Sil_Sdom, Sil_Sdom, SulA and BolA_ind into pSB4A5. Left at room temperature overnight.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">22<SUP>nd</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Continue with colony PCR of 20<SUP>th</SUP> of July transformation results from Lycka and Tessa.</P><P>Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in selective LB overnight.</P><P><STRONG>Lycka</STRONG></P><P>Transformation of ligation products from yesterday. Cells containing backbones pSB1C3 or pSB4A5 were plated on plates supplemented with chloramphenicol or ampicilin, respectively. After overnight cultivation the following plates contained colonies: mKate, mVenus, mCerulean, OmpA_Sil_Sdom, Sil_Sdom, SulA, BolA_ind.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">23<SUP>rd</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Continue with colony PCR of yesterday's transformation results from Lycka.</P><P>Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in selective LB overnight.</P><P>Pick all colonies selected by colony PCR on 21<SUP>st</SUP>, 22<SUP>nd</SUP> and 23<SUP>rd</SUP> of July and transfer to selective LB according to the resistance expressed by each plasmid.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">24<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Cryostock and plasmid isolation of overnight cultures prepared yesterday by Maria. Cryostocks stored at -80ºC, isolated plasmids stored at -20ºC.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">25<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Nanodrop and prepare for sequencing plasmids isolated yesterday.</P><P>Sequencing resulted in the following. Correct sequence: Sil_Sdom (pSB1C3), OmpA_Sil_Taur (pSB1C3), Sil_Sdom (pSB4A5), SulA (pSB4A5). Sequence with mutations: SulA (pSB1C3), BolA_ind (pSB1C3), J23108 (pSB1C3), BolA_con (pSB4A5), BolA_ind (pSB4A5). Different sequence: BolA_con (pSB1C3), INP_sil_Sdom (pSB1C3), OmpA_Sil_Sdom (pSB4A5). Mutated sequences can be repaired by PCR and blunt end ligation.</P><P><STRONG>María</STRONG></P><P>Prepare new digested pSB1C3 and pSB4A5 backbones by restricting with EcoRI and PstI and gel isolating the restricted product.</P><P>New ligation of those synthesized fragments that have not yielded any positive results (transformation or sequence confirmation) in the past week in their respective backbones.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">26<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Célina</STRONG></P><P>Transformation of mKate, mVenus, LacI and phaP in pSB1C3 and INP_Sil_Sdom in pSB4A5 into Top10 chemically competent cells. Also transformed one positive control per backbone and negative controls for both antibiotic resistances.
                                                Transformations were plated after increasing the concentration of cells by removing 200µL of medium.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">27<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Colony PCR of plates transformed yesterday.</P><P>Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated the same day in selective LB overnight.</P><P><STRONG>María and Tessa</STRONG></P><P>Make new chemically competent cells. They were tested for transformation capacity with a positive control plasmid (RFP in pSB4A5). Also, untransformed cells were plated in plates supplemented with Cm or Amp to test whether they had developed a resistance during the process.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">28<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Cryostock, plasmid isolation and preparation of samples for sequencing of the overnight cultures prepared yesterday by Lycka. Only INP_Sil_Sdom in pSB4A5 was sequenced confirmed with a mutation to be fixed by PCR. The other sequences aligned to a fragment of GFP unknown to us, we assumed it either came from a contamination of the old restriction enzymes we were using or from the pSB1C3 backbone that was taken from last year's team plasmid stock and probably was poorly labeled. Thus, we decided to use another backbone and the new enzymes sponsored to all iGEM teams by New England Biolabs.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">29<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka and Célina</STRONG></P><P>Repeat of the colony PCR of plates from the 26<SUP>th</SUP> of July that yielded a negative result.</P><P>Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in LB and the applicable antibiotic overnight.</P><P>Transformation of OmpA_Sil_Sdom in both different backbones.</P><P><STRONG>Iris</STRONG></P><P>Isolated plasmid with pSB1C3 backbone containing RFP to be used as positive control in transformations and to obtain the standard backbone by restriction.
                                                Obtained 3 tubes of isolated plasmid from 3mL of culture each.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">30<SUP>th</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Colony PCR of OmpA_Sil_Sdom in both backbones from the 29<SUP>th</SUP> of July. The PCR machine was filled with colonies from older plates which had not yielded any good colonies yet: mKate (26<SUP>th</SUP> of July), mKate (mKate 30<SUP>th</SUP> of June), INP_Sil_Sdom (20<SUP>th</SUP> of July).</P><P>Picture on the left shows simulated agarose gel. Pictures in the middle and on the right show the actual agarose gel. Picked colonies marked with an arrow were cultivated in LB and the applicable antibiotic overnight.</P><P>Cryostocking and plasmid isolation of colonies picked by on 29<SUP>th</SUP> of July. Cryostocks stored at -80ºC, isolated plasmid stored at -20ºC.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">31<SUP>st</SUP> July 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Cryostock and plasmid isolation of the overnight cultures prepared yesterday by Lycka and preparation of samples for sequencing of plasmids isolated yesterday and today. Sequence of mCerulean was confirmed (although it had a silent mutation). OmpA_Sil_Sdom in pSB4A5 and INP_Sil_Sdom in pSB1C3 had a deletion to be fixed by PCR. The rest of sequences either contained the random GFP sequenced mentioned before (since they are old ligation products) or multiple mutations that could not be fixed by PCR.</P></DIV></DIV><A href="#notes" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#notes').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseThree" aria-expanded="false" aria-controls="collapseThree"><DIV class="panel-heading" role="tab" id="headingThree"><H4 class="panel-title">




                                        August




                                    </H4></DIV></A><DIV id="collapseThree" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingThree"><DIV class="panel-body"><DIV class="row"><H2 class="title-style-3">2<SUP>nd</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Amplified E0840 (GFP) out of a pSB1C3-GFP plasmid using Phusion PCR. Four reactions of 50µl.</P><P>Nanodropped PCR products.</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Product</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>E0840</TD><TD>339.6</TD></TR><TR><TD>E0840</TD><TD>554.1</TD></TR><TR><TD>E0840</TD><TD>596.9</TD></TR><TR><TD>E0840</TD><TD>567.6</TD></TR></TBODY></TABLE><P>Ran a 1% agarose gel of the PCR product to verify product size.</P><P>Stored product 2 and 4 in the fridge for later use.</P><P><STRONG>Lara</STRONG></P><P>Digested the synthesized parts LacI, phaP, mKate, the backbones pSB1C3 and pSB4A5 and the Biobrick K1149051 with EcoRI and PstI. The restriction products were run over a 1% agarose gel and subsequently gel-purified. </P><P>Concentrations of purified DNA products.</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>LacI</TD><TD>15.5</TD></TR><TR><TD>phaP</TD><TD>8.2</TD></TR><TR><TD>mKate</TD><TD>8.7</TD></TR><TR><TD>K1149051</TD><TD>13.8</TD></TR><TR><TD>pSB1C3</TD><TD>25.5</TD></TR><TR><TD>pSB1C3</TD><TD>13.1</TD></TR><TR><TD>pSB1C3</TD><TD>9.3</TD></TR><TR><TD>pSB4A5</TD><TD>10.4</TD></TR><TR><TD>pSB4A5</TD><TD>7.3</TD></TR><TR><TD>pSB4A5</TD><TD>6.4</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row"><H2 class="title-style-3">3<SUP>rd</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Ligated the synthesized parts that were restricted and gel-purified yesterday and the backbones. The following ligation products were formed: LacI-pSB1C3, phaP-pSB1C3, mKate-pSB1C3, LacI-pSB4A5 and K1149051-pSB4A5. The ligation protocol was used and the below stated lengths for the vector and the inserts were used.</P><H4>Lengths of inserts and backbones.</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>DNA length (bp)</TH></TR></THEAD><TBODY><TR><TD>LacI</TD><TD>1335</TD></TR><TR><TD>phaP</TD><TD>648</TD></TR><TR><TD>mKate</TD><TD>790</TD></TR><TR><TD>K1149051</TD><TD>4271</TD></TR><TR><TD>pSB1C3</TD><TD>2070</TD></TR><TR><TD>pSB4A5</TD><TD>3395</TD></TR></TBODY></TABLE><P>Transformations were performed with the just obtained ligation products and additionally the previously obtained mVenus-pSB1C3 was also transformed into TOP10 cells. A DNA amount of 2 µL was used; as positive control for the ampicilin plates the plasmid RFP-pSB4A5 was used during transformation and for the chloramphenicol plates the plasmid RFP-pSB1C3 was used. Transformed cells were plated on selective LB-agar plates and incubated overnight at 37ºC. </P><P><STRONG>Charlotte</STRONG></P><P>Restriction and ligation of mVenus into the standard backbone</P><P><STRONG>Tessa</STRONG></P><P>Purification of PCR products of 2<SUP>nd</SUP> August. Accidentally added wash solution in first step, added more bind solution to compensate and it yielded good concentrations (86-115ng/µL).</P><P>PCR to add different promoters to GFP (E08040) BioBrick from the distribution kit. Forward primers were designed to add promoters upstream of the GFP sequence, sequencing reverse primer (RV) was used as reverse primer in this PCR. Primers added were: J23100, J23105, J23108, J23113, J23117.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">4<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Performed a DreamTaq colony PCR for yesterday's transformations; eight colonies per plate were picked and analyzed. </P><P>Gel electrophoresis of colony PCR of K1149051 pSB4A5, LacI pSB4A5, phaP pSB1C3, mKate pSB1C3, mVenus pSB1C3 and LacI pSB1C3.</P><P>Colonies were transferred into selective liquid LB medium.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct transformation</TH><TH>Selective medium</TH><TH>Colonies picked</TH></TR></THEAD><TBODY><TR><TD>K1149051-pSB4A5</TD><TD>LB-Amp</TD><TD>2</TD></TR><TR><TD>phaP-pSB1C3</TD><TD>LB-Cm</TD><TD>19, 20 and 21</TD></TR><TR><TD>mKate-pSB1C3</TD><TD>LB-Cm</TD><TD>31</TD></TR><TR><TD>mVenus-pSB1C3</TD><TD>LB-Cm</TD><TD>36, 40 and 41</TD></TR><TR><TD>LacI-pSB1C3</TD><TD>LB-Cm</TD><TD>46, 48 and 49</TD></TR></TBODY></TABLE><P><STRONG>Tessa</STRONG></P><P>Nanodrop of PCR products of GFP including promoters (sumarised in the following table) and subsequent restriction with EcoRI and PstI.</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>J23100 E0840</TD><TD>69.4</TD></TR><TR><TD>J23105 E0840</TD><TD>95.9</TD></TR><TR><TD>J23108 E0840</TD><TD>88.3</TD></TR><TR><TD>J23113 E0840</TD><TD>97.7</TD></TR><TR><TD>J23117 E0840</TD><TD>90.3</TD></TR></TBODY></TABLE><P>Concentration of DNA was measured after purification of the restriction products (see table).</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>J23100 E0840</TD><TD>3.3</TD></TR><TR><TD>J23105 E0840</TD><TD>3.5</TD></TR><TR><TD>J23108 E0840</TD><TD>3.5</TD></TR><TR><TD>J23113 E0840</TD><TD>2.8</TD></TR><TR><TD>J23117 E0840</TD><TD>3.4</TD></TR></TBODY></TABLE><P>These products were ligated with pSB1C3 and transformed into Top10 competent cells and incubated overnight.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">5<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Some of the overnight cultures were opened inside the shaker, so these were not sterile anymore. Colonies 21 (phaP-pSB1C3) and 47 (LacI-pSB1C3) had to be picked and transferred into liquid LB-medium again. </P><P>From the other overnight cultures, plasmids were isolated using Miniprep. </P><H4>Concentrations of plasmid isolations. </H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Colony number </TH><TH>Concentration (ng/L)</TH></TR></THEAD><TBODY><TR><TD>K1149051-pSB4A5</TD><TD>2</TD><TD>271.7</TD></TR><TR><TD>phaP-pSB1C3</TD><TD>19</TD><TD>398.0</TD></TR><TR><TD>phaP-pSB1C3</TD><TD>20</TD><TD>207.1</TD></TR><TR><TD>mKate-pSB1C3</TD><TD>31</TD><TD>83.3</TD></TR><TR><TD>mVenus-pSB1C3</TD><TD>36</TD><TD>142.5</TD></TR><TR><TD>mVenus-pSB1C3</TD><TD>40</TD><TD>306.5</TD></TR><TR><TD>mVenus-pSB1C3</TD><TD>41</TD><TD>65.3</TD></TR></TBODY></TABLE><P>These DNA samples were sent to sequencing.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">6<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Colony PCR of transformations from yesterday</P><P>Lanes 1, 2, 4, 5, 8, 9, 12, 15, 16, 17 and 19 yielded positive results and they were picked for overnight culture in selective LB medium.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">7<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Plasmid isolation of overnight cultures from yesterday using 3mL of culture. All products yielded good concentrations of DNA (90-200ng/µL). All of them were sent for sequencing.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">8<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Cryostock of overnight cultures from the other day.</P><P><STRONG>Liza</STRONG></P><P>Transformation of mCerulean into BL21.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">9<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Charlotte</STRONG></P><P>Started correcting point mutations using PCR in BioBricks that had a non-silent mutation in a coding area. The plasmids being corrected are SulA (pSB1C3), BolA_ind (pSB1C3), BolA_con (pSB4A5) and INP_Sil_Sdom (pSB4A5). Primers were ordered by María. Primers were first diluted for storage (100 µM) and working stock (10 µM). The concentrations of the plasmids with point mutations were determined using a Nanodrop.  </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>SulA (pSB1C3)</TD><TD>167.7</TD></TR><TR><TD>BolA_ind (pSB1C3)</TD><TD>109.1</TD></TR><TR><TD>BolA_con (pSB4A5)</TD><TD>130.4</TD></TR><TR><TD>INP_Sil_Sdom (pSB4A5)</TD><TD>166.7</TD></TR></TBODY></TABLE><P>The PCR to correct the mutations was done according to the Phusion HF protocol. The elongation time was set at 75 seconds, and the annealing time was set at 57ºC. The PCR product was loaded on a 1% TAE agarose gel for electrophoresis, in order to determine if the PCR worked.  </P><P>Left is a simulated image by SnapGene, right the actual gel picture. The lane marked with L is the Promega 1kb ladder, lane 1 contains SulA (pSB1C3), lane 2 contains BolA_ind (pSB1C3), lane 3 contains BolA_con (pSB4A5) and lane 4 contains INP_Sil_Sdom (pSB4A5) </P><P>The bands for BolA_ind, BolA_con and INP_Sil_Sdom are correct (lanes 2-4). The band for SulA (lane 1) is not as expected. This sample is checked on gel again. </P><P> Left is a simulated image by SnapGene, right the actual gel picture. The lane marked with L is the Promega 1kb ladder, lane 1 contains SulA (pSB1C3). The other bands contain result from another experiment, irrelevant for this experiment.</P><P>The result from the second electrophoresis of SulA is still not as expected, we will start over with the correction of this BioBrick later. For now, we continue with BolA_ind (pSB1C3), BolA_con (pSB4A5) and INP_Sil_Sdom (pSB4A5).</P><P> BolA_ind (pSB1C3), BolA_con (pSB4A5) and INP_Sil_Sdom (pSB4A5) were purified according to protocol. The obtained concentrations were measured using Nanodrop.

                                            </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>BolA_ind (pSB1C3)</TD><TD>99.9</TD></TR><TR><TD>BolA_con (pSB4A5)</TD><TD>98.5</TD></TR><TR><TD>INP_Sil_Sdom (pSB4A5)</TD><TD>61.5</TD></TR></TBODY></TABLE><P> The purified PCR product was restricted with DpnI, to restrict all methylated DNA containing the point mutation, leaving us with the corrected DNA. Restriction was done according to protocol. After 1 hour of incubation at 37ºC the restriction products were purified according to protocol. The prified restriction products were stored at -20ºC.</P><P><STRONG>Liza</STRONG></P><P>Colony PCR of transformations from yesterday</P><P><STRONG>Lara</STRONG></P><P>Colony PCR of LacI was performed again. </P><P>Colonies indicated with red arrows were transferred into liquid selective medium and grown overnight. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">10<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Restricted and purified PCR product of 3rd August, J23100 E0840, J23113 E0840 and J23117 E0840, with EcoRI-HF and PstI. Obtaining the following concentrations.</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Product</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>J23100 E0840</TD><TD>14.7</TD></TR><TR><TD>J23113 E0840</TD><TD>21.3</TD></TR><TR><TD>J23117 E0840</TD><TD>23.3</TD></TR></TBODY></TABLE><P>Ligated purified product into pSB1C3.</P><P>Transformed ligation product into TOP10 strain. Using RFP as positive control and sterile MilliQ as negative control. Plated on selective plates.</P><P><STRONG>Lara</STRONG></P><P>Of the previously sent DNA samples to sequencing, mVenus-pSB1C3, mKate-pSB1C3 and phaP-pSB1C3 were sequence confirmed. LacI-pSB1C3 and K1149051 were analysed again. Since the previous selected colonies for LacI-pSB1C3 turned out to contain the construct of the positive control RFP, a new colony-PCR was performed. Besides that, colony 47 of August 4<SUP>th</SUP> was also transferred into liquid medium. </P><P>Via Phusion-PCR, the promoters J23100, J23105, J23108, J23113 and J23117 were added to the yesterday sequence confirmed phaP-pSB1C3. The melting temperature of the designed primers were for all 58ºC; the template concentration was 102.0 ng/µl and the final construct should obtain 835 bp. The Phusion-PCR ran overnight.</P><P><STRONG>Charlotte</STRONG></P>
                                            The concentrations of the previously purified restriction products were measured using a Nanodrop:<H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Product</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>BolA_ind (pSB1C3)</TD><TD>16.5</TD></TR><TR><TD>BolA_con (pSB4A5)</TD><TD>14.4</TD></TR><TR><TD>INP_Sil_Sdom (pSB4A5)</TD><TD>9.5</TD></TR></TBODY></TABLE><P> The restriction products were blunt-end ligated in order to circularize the plasmids. Since our ligation tool does not calculate the concentrations needed for blunt-end ligation, the following protocol was used:</P><H4>Blunt-end ligation protocol</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Component</TH><TH>Amount</TH></TR></THEAD><TBODY><TR><TD>T4 ligation buffer (10X)</TD><TD>2 µL</TD></TR><TR><TD>DNA</TD><TD>~80 ng</TD></TR><TR><TD>T4 Ligase</TD><TD>1 µL</TD></TR><TR><TD>Nuclease-free water</TD><TD>up to 20 µL</TD></TR></TBODY></TABLE><P> The T4 ligase was added last, the ligation mixture was left to incubate for 4 hours. After incubation, the ligation product was transformed into homemade chemically competent TOP10 cells, according to protocol. The tranformations were set to incubate overnight at 37ºC</P><P><STRONG>Iris</STRONG></P><P>PCR of phaP to fix mutations.</P><P><STRONG>Célina and Liza</STRONG></P><P>Plasmid isolation and cryostock of mCerulean overnight cultures obtaining the following concentrations.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">11<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Colony PCR of colonies from yesterdays transformation using Q5<SUP>®</SUP> mastermix.</P><P>Ran a 1% agarose gel of the PCR product.</P><P> L is the Promega 1kb ladder, 1-6 is J23100 E0840, 7-14 is J23113 E0840 and 15-22 is J23117 E0840.
                                            </P><P>Transferred colony 14 (J23113 E0840 pSB1C3) and 21 (J23117 E0840 pSB1C3) into liquid LB.</P><P><STRONG>Lara</STRONG></P><P>Plasmids were isolated from the colonies that were grown overnight.</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Colony number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>LacI-pSB1C3</TD><TD>47</TD><TD>56.5</TD></TR><TR><TD>LacI-pSB1C3</TD><TD>51</TD><TD>86.4</TD></TR><TR><TD>J23117 E0840</TD><TD>54</TD><TD>141.4</TD></TR></TBODY></TABLE><P>LacI-pSB1C3 was sent to sequencing.</P><P><STRONG>Charlotte</STRONG></P><P>Picked 8 colonies for the plates containing cells tranformed with BolA_ind (psB1C3) and 8 for BolA_con (pSB4A5). The colonies on the plate containing cells transformed with INP_Sil_Sdom were still too small to pick so they were left to incubate at 37ºC a little longer before picking 8 colonies for colony PCR. </P><P> The colony PCR was done according to protocol, using DreamTaq polymerase. The PCR products were loaded on a 1% agarose gel for electrophoresis. Colony 16 didn't fit in the PCR machine anymore, so this colony is also not loaded on the gel.</P><P>Left is a simulated image by SnapGene, right the actual gel picture. For BolA_ind, colonies 2,3 and 4 had a band at the correct height. For BolA_con, colonies 11, 12 and 13 had a band at the correct height. For INP_Sil_Sdom, colonies 17, 19 and 20 were correct. These colonies were transferred to selective liquid LB and grown overnight in a shaker at 37ºC and 250 rpm. </P><P><STRONG>Charlotte &amp; Liza</STRONG></P><P> Tested 15 µm fluorescent polystyrene beads in the MicroTime 200 confocal microscope to see if the microscope is able to establish lasing in a bead. We used a 1 mW laser with an excitation wavelength of 532 nm. The bead was imaged at different intensities to see if we were able to see a shift to lasing at higher excitation energies.</P><P><STRONG>Liza</STRONG></P><P> Colony PCR of tranformations of mVenus and mKate in the standard backbone.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">12<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Miniprepped colony 14 and 21 of yesterdays overnight culture.</P><P>Nanodropped miniprep product.</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Product</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>J23113 E0840 in pSB1C3</TD><TD>140.6</TD></TR><TR><TD>J23117 E0840 in pSB1C3</TD><TD>325.6</TD></TR></TBODY></TABLE><P>Cryostocked colony 14 and 21.</P><P>Send miniprep product for sequencing. [Sequence Confirmed]</P><P><STRONG>Lara</STRONG></P><P>The results for the sequencing of LacI-pSB1C3 were checked: For both colonies 47 and 51 there was a 124 bp long exact same insert in the middle of the LacI-gene. In colony 54, there were two point mutations. The first one at location 2215, changing an Arg into Leu; the second mutation was on location 2625 changing a Glu into a stopcodon. </P><P>These two point mutations need to be fixed. Since normal mutation fixing by PCR only handles one mutation at a time, a new strategy was thought of: Two different linear fragments should be made, both fixing one mutation and able to join together with the other part via Gibson Assembly. The primers were designed and ordered.</P><P>The Phusion-PCR products of the different promoters that were obtained were transformed into TOP10 cells. For this, 2µl DNA was used per transformation, RFP-pSB1C3 as a positive control and MilliQ as a negative control. All transformations were plated onto LB-Cm plates and stored at 37ºC. </P><P><STRONG>Charlotte</STRONG></P><P> The DNA of the colonies that were grown overnight was isolated using miniprep according to protocol. The isolated DNA was stored at -20ºC. The remaining culture was cryostocked according to protocol and stored at -80ºC.</P><P><STRONG>Charlotte &amp; Liza</STRONG></P><P> Did a second attempt to get lasing in a bead with a confocal microscope.</P><P><STRONG>Liza</STRONG></P><P>Transformation mVenus and mKate into BL21, plated in selective plates and incubated at 37ºC overnight.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">13<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>A colony PCR was performed for the transformations of the different promoters attached to phaP-pSB1C3. The expected amplification length is approximately 1kb. Four colonies per plate were analyzed in the colony PCR. </P><P>This gel showed that only colony 35 (lane 15) did not contain the right PCR-product. The other colonies were transferred to liquid LB-Cm and cultivated overnight at 37ºC.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">14<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>The overnight grown cultures of the different promoters attached to phaP-pSB1C3 were plasmid isolated. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Colony number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>J23100 phaP pSB1C3</TD><TD>33</TD><TD>83.1</TD></TR><TR><TD>J23100 phaP pSB1C3</TD><TD>34</TD><TD>80.4</TD></TR><TR><TD>J23105 phaP pSB1C3</TD><TD>25</TD><TD>101.6</TD></TR><TR><TD>J23105 phaP pSB1C3</TD><TD>26</TD><TD>113.0</TD></TR><TR><TD>J23105 phaP pSB1C3</TD><TD>27</TD><TD>97.4</TD></TR><TR><TD>J23108 phaP pSB1C3</TD><TD>18</TD><TD>93.4</TD></TR><TR><TD>J23108 phaP pSB1C3</TD><TD>19</TD><TD>120.0</TD></TR><TR><TD>J23108 phaP pSB1C3</TD><TD>20</TD><TD>100.0</TD></TR><TR><TD>J23113 phaP pSB1C3</TD><TD>9</TD><TD>94.3</TD></TR><TR><TD>J23113 phaP pSB1C3</TD><TD>10</TD><TD>85.0</TD></TR><TR><TD>J23113 phaP pSB1C3</TD><TD>11</TD><TD>110.0</TD></TR><TR><TD>J23117 phaP pSB1C3</TD><TD>1</TD><TD>101.2</TD></TR><TR><TD>J23117 phaP pSB1C3</TD><TD>2</TD><TD>100.5</TD></TR><TR><TD>J23117 phaP pSB1C3</TD><TD>3</TD><TD>110.0</TD></TR></TBODY></TABLE><P>These plasmids were sent to sequencing.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">15<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Charlotte</STRONG></P><P>The concentrations of the previously isolated DNA was checked on Nanodrop. The following concentrations were measured:</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Colony number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="3" valign="middle">BolA_ind (pSB1C3)</TD><TD>2</TD><TD>63.0</TD></TR><TR><TD>3</TD><TD>61.8</TD></TR><TR><TD>4</TD><TD>68.3</TD></TR><TR><TD scope="row" rowspan="3" valign="middle">BolA_con (pSB4A5)</TD><TD>11</TD><TD>101.9</TD></TR><TR><TD>12</TD><TD>71.7</TD></TR><TR><TD>14</TD><TD>90.9</TD></TR><TR><TD scope="row" rowspan="3" valign="middle">INP_Sil_Sdom (pSB4A5)</TD><TD>17</TD><TD>88.0</TD></TR><TR><TD>19</TD><TD>74.4</TD></TR><TR><TD>20</TD><TD>43.2</TD></TR></TBODY></TABLE><P> The isolated DNA was prepared and sent for sequencing, according to protocol. </P><P><STRONG>Liza</STRONG></P><P>Colony PCR of mVenus and mKate transformations.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">16<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Charlotte</STRONG></P><P>Sequencing had the following results:</P><LI><B>BolA_ind (pSB1C3):</B> All colonies had large inserts or deletions, so they couldn't be used for further experiments. We will try to repair the mutations again later. </LI><LI><B>BolA_con (pSB4A5):</B> Colony 14 has an insert and colony 11 and 12 have a small gap. For colony 11 this is probably a readout error, when you look closer to the chromatogram. We can use colony 11 for further experiments.</LI><LI><B>INP_Sil_Sdom (pSB4A5):</B> Colony 17 has a nonsilent mutation in the promoter and colony 20 has a gap in the promoter. Colony 19 has one silent mutation and one nonsilent mutation, but in the linker region. We can use colony 19 for further experiments.</LI><P><STRONG>Liza</STRONG></P><P>Electrophoresis of colony PCR from 15<SUP>th</SUP> of August. Colonies 2, 3 and 8 from mVenus and 1, 3 and 4 from mKate were picked for overnight cultures.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">17<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Charlotte &amp; Liza</STRONG></P><P> Did a third attempt to get lasing in a bead with a confocal microscope.</P><P><STRONG>Tessa</STRONG></P><P>Plasmid isolation from yesterday's overnight cultures from Liza. Concentrations obtained were between 120 and 150ng/µL.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">18<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Transformation of the InterLab Study plasmids into BL21.</P><P><STRONG>Liza</STRONG></P><P>Receivend purified GFP and made 5 aliquots of 15µL which were stored at -80ºC.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">19<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Only Test Device 2 and 3 and positive control have yielded (very little) colonies. Decided to try and transform them into Top10.</P><P><STRONG>Lara</STRONG></P><P>A colony PCR was performed for the transformation of LacI-pSB1C3 on IPTG plates.</P><P>Since none of the colonies yielded a positive result in the colony-PCR, the colony-PCR was performed again. This time with a new protocol, first boiling the colonies and using GoTaq buffer. A total of 5 µL DNA mixture was used. </P><P>The colony indicated with a red arrow was transferred into selective liquid LB. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">22<SUP>nd</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Transformation of the InterLab Study plasmids into Top10.</P><P>Colony PCR of J23100 E0840 pSB1C3 using GoTaq<SUP>®</SUP>. Plate from 11th August.</P><P>Ran a 1% agarose gel to verify part size</P><P>L is the Promega 1kb ladder, 1-9 are J23100 E0840.</P><P>Colony 1,2 and 3 were put in selective LB medium and incubated overnight.</P><P><STRONG>Lara</STRONG></P><P>A miniprep was performed to isolate plasmids from the LacI transformation, induced with IPTG. Concentrations can be found in the table below. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>DNA construct</TH><TH>Colony number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>LacI pSB1C3</TD><TD>11</TD><TD>95.5</TD></TR><TR><TD>K1149051 pSB4A5</TD><TD>2</TD><TD>120.3</TD></TR></TBODY></TABLE><P>The LacI pSB1C3 plasmid was sent to sequencing, the K1149051 pSB4A5 plasmid yielded a too low concentration during plasmid isolation, since the sequencing company requested higher concentrations for larger plasmids. The BioBrick K1149051 pSB4A5 was cultivated from a cryostock overnight and for the LacI pSB1C3 BioBrick precautional cryostocks were made. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">23<SUP>rd</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Colony PCR of the InterLab Study plates using DreamTaq<SUP>®</SUP>. Picked eight colonies of each plate.</P><P>Cryostocked J23100 E0840 pSB1C3 in Top10.</P><P>Ran a 1% agarose gel of the InterLab Study colony PCR products.</P><P>L is the Promega 1kb ladder, 1-8 is Test Device 1, 9-16 is Test Device 2, 17-24 is Test Device 3, 25-32 is Negative Control and 33-40 is Positive control.</P><P>Colony 1, 2, 4, 9-11, 17-19, 26, 27, 29 and 33-36 were picked for overnight culture in liquid LB+CM.</P><P><STRONG>Lara</STRONG></P><P>Plasmids were isolated for the yesterday overnight cultured K1149051 pSB4A5 and for different colonies of J23100 E0480 pSB1C3. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Colony number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>K1149051 pSB4A5</TD><TD>2</TD><TD>328.7</TD></TR><TR><TD scope="row" rowspan="3">J23100 E0480 pSB1C3</TD><TD>1</TD><TD>83.8</TD></TR><TR><TD>2</TD><TD>89.5</TD></TR><TR><TD>3</TD><TD>85.4</TD></TR></TBODY></TABLE><P> These plasmids were sent to sequencing.</P><P>Primers that were needed for Gibson Assembly of the fluorophores and PHB constructs were diluted. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">24<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Put InterLab Study cultures in fridge for later use.</P><P><STRONG>Charlotte</STRONG></P><P>Started a second round of correcting mutations in our designed BioBricks using PCR. The BioBricks that will be corrected are SulA (pSB1C3), BolA_ind (pSB1C3), INP_Sil_Sdom (pSB1C3) and OmpA_Sil_Sdom (pSB4A5). Prior to starting the correction of the mutations using PCR, the concentrations of these BIoBricks were measured on the Nanodrop:</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>SulA (pSB1C3)</TD><TD>167.6</TD></TR><TR><TD>BolA_ind (pSB1C3)</TD><TD>109.1</TD></TR><TR><TD>INP_Sil_Sdom (pSB1C3)</TD><TD>139.2</TD></TR><TR><TD>OmpA_Sil_Sdom</TD><TD>170.8</TD></TR></TBODY></TABLE><P> The primers were diluted according to the manufacturers' instructions to a stock of 100µM. These stocks were consequently 10 times diluted to a working stock of 10µM. </P><P><STRONG>Lara</STRONG></P><P>Our advisor Esengül provided us with a protocol for the Gibson Assembly (see protocols).</P><P>The first step of the Gibson Assembly was to generate linear DNA segments by Phusion PCR. This was performed and to check the products, 5µL reaction product was loaded onto a 1% agarose gel.</P><P>None of the PCR products yielded the correct length. Especially K1149051 was not nearly the length expected. Sequences arrived, and it appeared that the were not correct. We figured that the backbone could have been pasted to each other twice, forming a plasmid like that with antibiotic resistance. For this, another round of colony PCR for K1149051 pSB4A5. Another colony PCR should be performed. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">25<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Resuspended FITC according to <A href="https://static.igem.org/mediawiki/2016/c/c5/InterLab_iGEM2016_Plate_Reader_Protocol_Updated_July.pdf" target="_blank">InterLab protocol.</A></P><P><STRONG>Charlotte</STRONG></P><P>Ran the PCR to fix the mutations in the BioBricks according to the Phusion HF PCR protocol, with an annealing time of 57ºC and an elongation time of 1.5 minutes. The results of the PCR were confirmed on gel, according to the electrophoresis protocol. </P><P>Left is a simulated image by SnapGene, right the actual gel picture. The lane marked with L is the Promega 1kb ladder, lane 1 contains SulA (pSB1C3), lane 2 contains BolA_ind (pSB1C3), lane 3 contains INP_Sil_Sdom (pSB1C3) and lane 4 OmpA_Sil_Sdom (pSB4A5) </P><P>The bands from BolA_ind (pSB1C3), INP_Sil_Sdom (pSB1C3) and OmpA_Sil_Sdom (pSB4A5) are at the correct height compared to the simulation. The lane containing SulA shows the same result as the previous attempt to correct this BoBrick. Probably something went wrong in the PCR, it's likely that the primers anneal at the wrong position. We will not continue this BioBrick and use SulA in another backbone. The other PCR products were purified according to protocol and the concentrations were measured using Nanodrop. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>BolA_ind (pSB1C3)</TD><TD>107.7</TD></TR><TR><TD>INP_Sil_Sdom (pSB1C3)</TD><TD>88.7</TD></TR><TR><TD>OmpA_Sil_Sdom</TD><TD>100.3</TD></TR></TBODY></TABLE><P> The purified PCR products were restricted with DpnI to remove any methylated and thus mutated DNA, according to the restriction protocol. Since our heat block was broken the samples were incubated in the stove at 37ºC. The restriction products were purified according to protocol and stored at -20ºC.</P><P><STRONG>Lara</STRONG></P><P>Another colony PCR was performed for K1149051 pSB4A5 to make sure we did not select a plasmid with a double backbone. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">26<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>The colony PCR of yesterday was gel analysed. The products in lanes 2, 3, 4, 9 and 10 were assumed to be correct. The corresponding colonies were transferred to LB+Amp and incubated overnight at 37ºC. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">28<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Plasmid isolation was performed for the K1149051 pSB4A5 BioBrick. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Colony number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="5">K1149051 pSB4A5</TD><TD>2</TD><TD>340.3</TD></TR><TR><TD>3</TD><TD>278.6</TD></TR><TR><TD>4</TD><TD>320.9</TD></TR><TR><TD>9</TD><TD>290.1</TD></TR><TR><TD>10</TD><TD>315.6</TD></TR></TBODY></TABLE><P>These plasmids were sent to sequencing and were sequence confirmed. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">29<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Transformed J23100-, J23105-, J23108-, J23113- and J23117 E0840 pSB1C3, and BolA_con pSB4A5 into BL21.</P><P><STRONG>Charlotte</STRONG></P><P>Checked the concentrations of the purified restriction products on Nanodrop:</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD>BolA_ind (pSB1C3)</TD><TD>14.7</TD></TR><TR><TD>INP_Sil_Sdom (pSB1C3)</TD><TD>11.9</TD></TR><TR><TD>OmpA_Sil_Sdom</TD><TD>12.6</TD></TR></TBODY></TABLE><P> The restriction products were blunt-end ligated according to the ligation protocol. The ligated BioBricks were transformed into homemade TOP10 cells, according to protocol. For the controls only 1/3 of the volume was used due to high transformation efficiency observed before. The transformants were plated on selective plates and left to incubate overnight at 37ºC.</P><P><STRONG>Lara</STRONG></P><P>Prepared LB Amp+Cm plates. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">30<SUP>th</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>PCR lineralization of vectors and inserts for gibson assembly phaCAB operon (BBa_K1149051) to fluorophores (GFP, mVenus, mKate, mCerulean). Phusion polymerase. Annealing temperature 60ºC, elongation time 2 minutes.</P></DIV><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Did the InterLab Study plate reader measurements according to the InterLab plate reader protocol.</P><P>Streaked J23105- and J23117 E0840 pSB1C3 out on new plates, because they had too many colonies.</P><P>Colony PCR of six colonies of BolA_con, J23100 E0840, J323108 E0840 and J23113 E0840 using GoTaq<SUP>®</SUP>.</P><P>Ran a 1% agarose gel of PCR products to verify part size.</P><P>L is the Promega 1kb ladder, 1-6 is BolA_con, 7-12 is J23100 E0840, 12-18 is J23108 E0840 and 19-24 is J23113 E0840.</P><P>Picked colony 1, 6, 11, 13, 18, 20 and 21 for overnight culture.</P><P><STRONG>Charlotte</STRONG></P><P> Of each transformation plate, 8 colonies were picked: Colonies 1-8 were of BolA_ind, colonies 9-16 of OmpA_Sil_Sdom and colonies 17-24 of INP_Sil_Sdom. A colony PCR was performed with the picked colonies, according to the GoTaq Colony PCR protocol. The annealing time was 1.5 minutes.</P><P> The PCR products were checked on gel according to the Electrophoresis protocol, which yielded the following gel: </P><P>L is the Promega 1kb ladder, 1-8 is BolA_ind, 9-16 is OmpA_Sil_Sdom, 17-24 is INP_Sil_Sdom </P><P> Colonies 1, 3, 4, 14, 15, 16, 20, 21 and 22 were picked and transferred to selective medium for an overnight culture, according to protocol. </P><P><STRONG>Lara</STRONG></P><P>Transformations into BL21 were performed for K1149051 pSB4A5, J23100 phaP pSB1C3 + K1149051 pSB4A5, J23105 phaP pSB1C3 + K1149051 pSB4A5, J23108 phaP pSB1C3 + K1149051 pSB4A5, J23113 phaP pSB1C3 + K1149051 pSB4A5 and J23117 phaP pSB1C3 + K1149051 pSB4A5. RFP pSB4A5 and RFP pSB4A5 + RFP pSB1C3 were used as positive controls and MilliQ as a negative control. In total 70 µL was plated on selective LB plates and they were incubated at 37ºC overnight.  </P><P>Cryostocks were made for K1149051 pSB4A5 in Top10 cells. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">31<SUP>st</SUP> August 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Electrophoresis of PCR fragments yesterday.</P><P>Picture on the left shows simulated agarose gel. Picture on the right shows the actual agarose gel. 1 - 4: vectors for mCerulean, mKate, mVenus and GFP, respectively. 5 - 6 inserts for mVenus and GFP. Gel did not yield bands at the right height. PCR is repeated at an annealing temperature of 57ºC.</P><P>Second gel also did not yield the right bands. PCR repeated overnights by María at Tm 64ºC and elongation time 3.5 minutes in the presence of DMSO.</P><P>Transfer from cryostock to plate BL21 strains with fluorophores expressed. On LB agar supplemented with chloramphenicol.</P></DIV><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Cryostocked J23100-, J23108- and J23117 E0840 pSB1C3 in BL21. BolA_con didn't grow.</P><P>Cryostocked K1149051 pSB4A5 in BL21.</P></DIV><DIV class="col-md-12"><P><STRONG>Charlotte</STRONG></P><P>The overnight cultures of BolA_ind (pSB1C3), OmpA_Sil_Sdom (pSB4A5) and INP_Sil_Sdom (pSB1C3) were cryostocked according to protocol, 3 tubes per culture. The remaining culture was used for DNA isolation by Miniprep, according to protocol. Two tubes per colony were prepared. The concentration of the isolated DNA was measured using Nanodrop, the following concentrations were measured:</P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Colony number</TH><TH>Tube 1 concentration (ng/µl)</TH><TH>Tube 2 concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="3">BolA_ind (pSB1C3)</TD><TD>1</TD><TD>91.2</TD><TD>113.5</TD></TR><TR><TD>3</TD><TD>87.9</TD><TD>79.6</TD></TR><TR><TD>4</TD><TD>104.5</TD><TD>87.7</TD></TR><TR><TD scope="row" rowspan="3">OmpA_Sil_Sdom (pSB4A5)</TD><TD>14</TD><TD>133.5</TD><TD>143.7</TD></TR><TR><TD>15</TD><TD>152.5</TD><TD>141.4</TD></TR><TR><TD>16</TD><TD>171.1</TD><TD>159.4</TD></TR><TR><TD scope="row" rowspan="3">INP_Sil_Sdom (pSB4A5)</TD><TD>20</TD><TD>109.6</TD><TD>90.6</TD></TR><TR><TD>21</TD><TD>112.2</TD><TD>101.5</TD></TR><TR><TD>22</TD><TD>112.0</TD><TD>111.9</TD></TR></TBODY></TABLE><P> The isolated DNA was prepared and sent for sequencing, according to protocol. </P><P><STRONG>Lara</STRONG></P><P>Yesterday's transformations in BL21 did not yield colonies for K1149051 pSB4A5 and for J23100 phaP pSB1C3 + K1149051 pSB4A5. These were transformed again, together with BolA_con pSB4A5, K1149051 pSB4A5 and RFP in pSB4A5, pSB4A5 and pSB1C3 as positive control and MilliQ as negative control. These transformations were plated on selective LB plates and stored overnight at 37ºC. </P><P>For the Gibson Assembly, new plasmid needed to be isolated. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="6">K1149051 pSB4A5</TD><TD>171.6</TD></TR><TR><TD>189.6</TD></TR><TR><TD>182.8</TD></TR><TR><TD>141.2</TD></TR><TR><TD>157.1</TD></TR><TR><TD>162.7</TD></TR></TBODY></TABLE><P>A colony PCR with GoTaq buffer was performed for the different promoters of phaP pSB1C3 + K1149051 pSB4A5. </P><P>Colonies 2, 3, 4, 8, 9, 10, 13, 14, 15, 18, 19 and 20 were transferred into liquid LB+Amp+Cm and stored overnight at 37ºC.</P></DIV></DIV><A href="#notes" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#notes').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseFour" aria-expanded="false" aria-controls="collapseFour"><DIV class="panel-heading" role="tab" id="headingFour"><H4 class="panel-title">




                                        September




                                    </H4></DIV></A><DIV id="collapseFour" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFour"><DIV class="panel-body"><DIV class="row"><H2 class="title-style-3">1<SUP>st</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>PCR with new primers of those parts that need to be expressed under the lac promoter. These primers will allow cloning on Addgene plasmids from the pBb series. The plasmids chosen (pBbS5a and pBbA5c) express lacI and the cloning site is placed downstream of a lac promoter. The parts to be amplified are:1. BolA2. INP_Sil_Sdom3. OmpA_Sil_Sdom4. OmpA_Sill_Taur5. Sil_Sdom6. SulA</P><P><STRONG>Lycka</STRONG></P><P>Electrophoresis of overnight PCR for lineralization.</P><P>Picture on the left shows simulated agarose gel. Picture on the right shows the actual agarose gel. 1 - 4: vectors for mCerulean, mKate, mVenus and GFP, respectively. 5 - 6 inserts for mVenus and GFP. For all lanes except lane 5, the right bands are present. Also many other bands were present so the product was gel purified.</P><P>Transfered one colony of BL21 on each plate from yesterday in a 50mL falcon tube with 10mL filter sterilized M9 medium with glucose supplemented with chloramphenicol.</P></DIV><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Put three colonies of J23100-, J23105- and J23117 E0840 pSB1C3 in BL21 into selective LB medium.</P></DIV><DIV class="col-md-12"><P><STRONG>Charlotte</STRONG></P><P><B>Cryostocks:</B> Were made for colonies 4, 10, 13 and 18 for the different promoters of phaP pSB1C3 + K1149051 pSB4A5. They were stored at -80ºC. </P><P><B>Liquid cultures:</B> Each colony of BL21 transformation should be good. Liquid cultures were made of BolA_con pSB4A5, K1149051 pSB4A5 and J23100 phaP pSB1C3 + K1149051 pSB4A5 and stored at 37ºC to be used tomorrow to make cryostocks. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">2<SUP>nd</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Cryostocked J23100-, J23105- and J23117 E0840 pSB1C3 in BL21.</P><P><STRONG>Lycka</STRONG></P><P>Nanodrop gel purified PCR fragments from yesterday. Concentrations are too low to work with. Repeat PCR under the same conditions, but with double volume (100µL). Done by María.</P><P>Measure OD600 of cultivated M9 tubes. </P><H4>OD600 measurements</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>GFP J23108</TD><TD>1.24</TD></TR><TR><TD>mVenus</TD><TD>0.04</TD></TR><TR><TD>mKate</TD><TD>0.20</TD></TR><TR><TD>mCerulean</TD><TD>0.03</TD></TR></TBODY></TABLE><P>The strains containing GFP and mKate grew really well, the ones containing mVenus and mCerulean did not. All of the tubes were put back in the incubator. The one with GFP was diluted 10 times in M9 medium, the one with mKate was diluted 2 times. Aliquots of 150µL were loeaded in an 96 well plate and measured in a plate reader at 37ºC for 7 hours.</P><P><STRONG>María</STRONG></P><P>PCR of same samples as Lycka in a reaction volume of 100µL. After gel isolation and purification only for mCerulean and mKate good concentrations were reached (160 and 260 ng/µL, respectively), for the rest of samples either no bands were seen in the agarose gel or the concentration was too low again.</P><P><STRONG>Charlotte</STRONG></P><P> Analyzed sequencing results. Both the colonies of BolA_ind and OmpA_Sil_Sdom had an insert around the mutation spot, so sequences were not correct. Also all colonies of INP_Sil_Sdom had non-silent mutations. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">3<SUP>rd</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>The cultures in M9 grew well in the incubator until an OD of approximately 1. Samples were diluted 10 times in M9 supplemented with chloramphenicol and measured in the plate reader.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">4<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P><B>Doing things for Lycka and María: </B>Plate was removed from plate reader and file was saved. Plate reader plate was stored at 4ºC. </P><P>mVenus pSB1C3 was miniprepped and stored at -20ºC. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Tube number</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="6">mVenus pSB1C3</TD><TD>1</TD><TD>47.3</TD></TR><TR><TD>2</TD><TD>70.7</TD></TR><TR><TD>3</TD><TD>53.6</TD></TR><TR><TD>4</TD><TD>44.5</TD></TR><TR><TD>5</TD><TD>64.4</TD></TR><TR><TD>6</TD><TD>54.7</TD></TR></TBODY></TABLE><P>A Phusion PCR was performed again for Gibson Assembly. The vector for mKate and mVenus and the insert for GFP yielded a correct PCR product and were gel isolated. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Linearized part</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="1">mKate vector</TD><TD>71.0</TD></TR><TR><TD scope="row" rowspan="1">mVenus vector</TD><TD>69.2</TD></TR><TR><TD scope="row" rowspan="1">GFP insert</TD><TD>31.1</TD></TR></TBODY></TABLE><P>Since there are an insert and a vector present that are compatible for mKate and K1149051 pSB4A5, these were Gibson Assembled according to the protocol and transformed into Top10 cells. As a negative control for the Gibson Assembly, nuclease-free water was used instead of the vector. Transformations were plated on selective LB plates by Lycka and stored overnight at 37ºC. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">6<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka &amp; Lara</STRONG></P><P>Colony PCR for the transformants of the Gibson Assembly. </P><P>Most colonies appear to be false positives, since the bands correspond to the length of the vector only. Colonies 43 and 44 might contain the desired plasmid (with the PHB operon fused to mKate), so these are transfered to LB medium.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">8<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Isolate plasmid, cryostock and prepare for sequencing colonies 43 and 44. The concentrations were 252.6 and 242.6 ng/µl, respectively.</P></DIV><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>For the Gibson Assemblies, we still needed the mCerulean vector, the GFP vector and the mVenus insert. For this, another phusion PCR was performed and it ran overnight. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">9<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>In order for the cells to grow better, we prepared enriched M9 medium (eM9), instead of regular M9.
                                                The medium was supplemented with chloramphenicol and devided into aliquots of 10 mL. From all plates with fluorophore expressing cells,
                                                one colony was picked and grown in eM9 overnight.</P><P><STRONG>Lara</STRONG></P><P>The phusion PCRs of yesterday were gel-isolated. Since mVenus once again did not yield a proper product, we decided to discard this part. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Linearized part</TH><TH>Concentration (ng/µl)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="1">GFP vector</TD><TD>23.3</TD></TR><TR><TD scope="row" rowspan="1">mCeruleanvector</TD><TD>12.3</TD></TR></TBODY></TABLE><P>A Gibson Assembly reaction was performed according to the protocol for the GFP + K1149051 assembled plasmid and the mCerulean + K1149051 assembled plasmid. After 60 minutes of incubation at 37ºC, they were transformed into Top10 cells. For this, RFP pSB4A5 was used as a positive control and vector without inserts were used as negative controls. Transformation products were plated on selective LB plates and stored overnight at 37ºC. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">10<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Transfer to LB cells with plasmids bought from Addgene: pBbS5a and pBbA5c</P><P>Measure OD of overnight cultures in eM9. Dilute them until OD of 0.1 in eM9 supplemented with chloramphenicol.
                                                Measure kinetic cycle of 7 hours in plate reader.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">11<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Charlotte &amp; Lara</STRONG></P><P>Miniprepped plasmids bought from Addgene pBbS5a and pBbA5c from overnight LB cultures; 5 tubes each. The concentrations were confirmed with nanodrop and were between 30 and 60 ng/µL.</P><P>The transformations from the Gibson Assembly still did not yield any colonies. </P></DIV><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Restriction of plasmids bought from Addgene pBbS5a and pBbA5c, and PCR products from September 1st with EcoRI and XhoI. The biggest fragment obtained with each plasmid is the backbone, which was gel isolated. Then, backbones and fragments were ligated and transformed into Top10 chemically competent cells.</P></DIV><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Transformation of the following plasmids in Top10 chemically competent cells and plated on selective agar plates.</P><UL class="list-unstyled"><LI>BolA_con pSB1C3</LI><LI>INP_Sil_Sdom pSB4A5</LI><LI>OmpA_Sil_Taur pSB4A5</LI><LI>Sil_Sdom pSB4A5</LI><LI>Sul_A pSB1C3</LI><LI>OmpA_Sil_Sdom pSB4A5</LI><LI>RFP pSB1C3 (positive control)</LI><LI>RFP pSB4A5 (positive control)</LI><LI>Sterile MiliQ (negative control)</LI></UL></DIV></DIV><DIV class="row"><H2 class="title-style-3">12<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Colony PCR of transformations from yesterday 16 colonies of each transformation were picked, but only analized 15 of each.</P><P>Red arrows point at colonies picked for overnight cultures in selective medium.</P></DIV><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Measure spectrum of mVenus and mCerulean cells in plate reader. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">13<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Plasmid isolation, preparation of samples for sequencing and cryostock of overnight cultures of colonies picked yesterday.</P><P>Sequences of all colonies picked for OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom and SulA were confirmed to be good. In the case of BolA there's a deletion of a single bp, to be fixed by PCR. All sequences for OmpA_Sil_Sdom contained a strange long insert, this silicatein gene version was dropped for further experiments.</P></DIV><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Put 100µl of TOP10 competent cells into 10ml LB in a falcon tube and incubated overnight at 37ºC at 250rpm.</P><P>Made LB and LB agar.</P></DIV><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Measure spectrum of eM9 cultures in plate reader. </P><P>Sequences of mKate gibson assembled to phaCAB arrived and didn't yield the good results.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">14<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Made ~100 aliquots TOP10 competent cells with culture from yesterday according to protocol. OD600 before putting on ice was 0.34.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">15<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Transformation of plasmids with confirmed sequences from 13<SUP>th</SUP> of September into BL21. Also, combined with GFP under promoter J23100 in pSB1C3 backbone. All 3 silicatein versions, SulA and the combination of Sil_Sdom and GFP yielded colonies. Other 2 silicateins together with GFP did not yield any positive result.</P></DIV><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Did positive and negative controls of competent cells made yesterday. Used RFP pSB1C3 for positive control and MiliQ for negative control.</P></DIV><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Wash 3x in PBS cultures grown in LB expressing fluorophores. Dissolve in PBS and measure spectrum in plate reader.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">16<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Repeat kinetic cycle experiment in plate reader with different settings, because the previous results didn't show good results.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">18<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Transfer from plate to liquid LB colonies with BL21 cells expressing OmpA_Sil_Taur, Sil_Sdom and INP_Sil_Sdom for widefield microscope experiment.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">19<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka &amp; María</STRONG></P><P>Transfer LB cultures from yesterday to fresh medium and grow until OD reached 0.65. Then added 1mM sterile IPTG to induce.
                                                Grown in the incubator for another 5 hours. Subsequently added 60µM Sodium Silicate and incubated for 3 hours before putting them in the fridge.</P><P>Diluted and filter sterilized Sodium Silicate in miliQ water up to a concentration of 1M.</P><P>Diluted and filter sterilized Rhodamine 123 in ethanol up to a concentration of 20 mg/mL and added 15µL of this to the indicated cultures.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">20<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Transfer of colonies from 15<SUP>th</SUP> of September transformations into overnight culture for cryostocking.</P><P><STRONG>Lycka &amp; Charlotte</STRONG></P><P>Add 7µM rhodamine to the cultures from yesterday, incubated 10 min. Centrifuged 1.5 mL of culture at 4000 rpm 5 min
                                                Washed 3x and then resuspended in 400µL PBS. Imaged in widefield microscope with excitation wavelength of 488 nm.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">21<SUP>st</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Cryostock of BL21 strains containing sequence confirmed plasmids from yesterday's overnight cultures.</P><P>Transformation into BL21 of combinations of OmpA_Sil_Taur with all fluorophores (separately) and INP_Sil_Sdom with GFP under promoter J23100. 
                                                None of them yielded any colonies. We are going to try reducing the antibiotics concentration in the plates to half because probably the cells are not able to handle the 2 plasmids and such a high concentration of antibiotics.</P><P><STRONG>Lycka</STRONG></P><P>Measure OD600 of overnight cultures María prepared. They were around 1.46, 1.35, 1.49 for Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom respectively.
                                                1mL of culture was transfered to 10 mL fresh LB.</P><P>Dissolved and filter sterilized IPTG. 1.2 g in 5 mL.</P><P>Saved cultures in the fridge to continue tomorrow.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">22<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Inoculate 0.25 mL of yesterday's culture in 10 mL fresh LB. Cells did not grow after half a day. Inoculate
                                                new culture from plate, grown overnight.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">23<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Test silicic acid test, with 1mM aliquot of silic acid.</P><UL class="list-unstyled"><LI>100 µL 1 mM silicate</LI><LI>4900 µL miliQ</LI><LI>3 drops reagent 1</LI><LI>3 drops reagent 2</LI><LI>500 µL reagent 3</LI></UL></DIV></DIV><DIV class="row"><H2 class="title-style-3">24<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Measure OD600 of overnight eM9 cultures, diluted 4 times before measuring.</P><H4>OD600 measurements</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>GFP J23100</TD><TD>0.90</TD></TR><TR><TD>GFP J23105</TD><TD>0.51</TD></TR><TR><TD>GFP J23108</TD><TD>0.88</TD></TR><TR><TD>GFP J23113</TD><TD>didnt grow</TD></TR><TR><TD>GFP J23117</TD><TD>didnt grow</TD></TR><TR><TD>mVenus</TD><TD>0.74</TD></TR><TR><TD>mCerulean</TD><TD>0.77</TD></TR></TBODY></TABLE><P>Dilute until OD 0.1 and divide into aliquots of 100µL in a 96 well plate. Performed kinetic cycle in plate reader.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">26<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Repeat of transformations of 21<SUP>st</SUP> of September using plates with half the concentration of antibiotics. All of them yielded colonies.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">27<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Transfer of colonies from yesterday's transformations to overnight culture.</P><P><STRONG>Lycka</STRONG></P><P>Prepare eM9 an divide into aliquots of 10 mL, inoculate with:</P><UL class="list-unstyled"><LI>OmpA Sil Taur</LI><LI>INP Sil Sdom</LI><LI>Sil Sdom</LI><LI>BolA</LI><LI>OmpA Sil Taur</LI><LI>OmpA Sil Taur + mCerulean</LI><LI>mCerulean</LI></UL><P>Restreak plates with BL21 OmpA Sil Taur and BolA on fresh selective agar plates.</P><P>Cryostock OmpA + mCerulean, + mVenus and + all GFPs.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">28<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Cryostock of BL21 cells from overnight culture from yesterday.</P><P>Fixing of BolA mutation by PCR.</P><P>Picture on the left shows simulated agarose gel. Picture on the right shows the actual agarose gel. PCR didn't work, we are going to try again with a gradient of annealing temperatures to see if this is the cause of the problem.</P><P><STRONG>Lycka</STRONG></P><P>Measure OD600 of overnight eM9 cultures, diluted 4 times before measuring.</P><H4>OD600 measurements</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>INP Sil Sdom</TD><TD>1.15</TD></TR><TR><TD>BolA</TD><TD>1.18</TD></TR><TR><TD>OmpA Sil Taur</TD><TD>1.26</TD></TR><TR><TD>Sil Sdom</TD><TD>1.00</TD></TR></TBODY></TABLE><P>Dilute until OD 0.05 in 200 mL eM9 and grown in shaker at 37ºC. Cultures didn't grow after half a day.</P><P>Prepared calibration curve for silicate test. Dilutions made with 1 mM silic acid stock.</P><H4>Calibration curve</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>concentration (µM)</TH><TH>silica stock (µL)</TH><TH>miliQ (µL)</TH></TR></THEAD><TBODY><TR><TD>100</TD><TD>500</TD><TD>4500</TD></TR><TR><TD>80</TD><TD>400</TD><TD>4600</TD></TR><TR><TD>60</TD><TD>300</TD><TD>4700</TD></TR><TR><TD>40</TD><TD>200</TD><TD>4800</TD></TR><TR><TD>20</TD><TD>100</TD><TD>4900</TD></TR><TR><TD>10</TD><TD>50</TD><TD>4950</TD></TR><TR><TD>5</TD><TD>25</TD><TD>4975</TD></TR><TR><TD>0</TD><TD>0</TD><TD>5000</TD></TR></TBODY></TABLE><P>Prepare overnight eM9 cultures for:</P><UL class="list-unstyled"><LI>Silic acid test: OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom, BolA</LI><LI>Laser setup: mCerulean, OmpA_Sil_Taur, mCerulean + OmpA_Sil_Taur</LI><LI>TEM: OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom</LI></UL></DIV></DIV><DIV class="row"><H2 class="title-style-3">29<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Tessa</STRONG></P><P>Made ~50 aliquots BL21 competent cells according to protocol. OD600 before putting on ice was 0.30.</P><P><STRONG>María</STRONG></P><P>PCR to fix BolA mutation.</P><P>All annealing temperatures resulted in a product of the desired length.
                                                1. 66ºC; 2. 65ºC; 3. 63.4ºC; 4. 60ºC</P><P><STRONG>Lycka &amp; María</STRONG></P><P>Measure OD600 of overnight eM9 cultures, diluted 4 times before measuring.</P><H4>OD600 measurements</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>INP Sil Sdom</TD><TD>0.77</TD></TR><TR><TD>BolA</TD><TD>0.74</TD></TR><TR><TD>OmpA Sil Taur</TD><TD>0.75</TD></TR><TR><TD>Sil Sdom</TD><TD>0.68</TD></TR></TBODY></TABLE><P>Inoculate 4 µL in 150 µL fresh eM9. After induction Erlenmeyer fell over and content was not usefull anymore.</P><P>Measure OD600 of overnight LB cultures, diluted 4 times before measuring.</P><H4>OD600 measurements</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>INP Sil Sdom</TD><TD>0.73</TD></TR><TR><TD>OmpA Sil Taur</TD><TD>0.76</TD></TR><TR><TD>Sil Sdom</TD><TD>0.91</TD></TR></TBODY></TABLE><P>Inoculate 100 µL in 10 mL fresh medium. Grow until OD 0.55 and add 10 µL 1 M IPTG.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">30<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>María</STRONG></P><P>Silicic acid test: since cells grew too slow in eM9 medium we checked if LB absorbance spectrum interfered with the absorbance of the final product of the test (~815nm). It didn't so we decided to grow the cells in LB medium.</P><P>Measured OD600 of overnight cultures with a 1/4 dilution:</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>OmpA Sil Taur</TD><TD>0.77</TD></TR><TR><TD>INP Sil Sdom</TD><TD>0.74</TD></TR><TR><TD>Sil Sdom</TD><TD>0.75</TD></TR><TR><TD>BolA</TD><TD>0.68</TD></TR></TBODY></TABLE><P><STRONG> Carmen &amp; Charlotte </STRONG></P><P>Tested whether adding silicatein cells to a solar panel had any effect. We tested the strain expressing OmpA-fused silicatein which was incubated with silica, so it had a silicate coating.  The cells were washed 3x in PBS. We connected a multimeter that was measuring voltage to a solar cell with a limited area exposed to light. We measured the output power of the solar cell at the same time as the light intensity of the lamp. We repeated this experiment with the same area of the solar cell covered in PBS and covered in PBS + silicatein-expressing cells.  </P><P><STRONG> Charlotte &amp; Liza </STRONG></P><P>Tested the mCerulean-expressing cells in our self-built hardware setup. The cells were grown in LB and washed 3x with PBS. The cells were fixed on a slide by pipetting them under 3% agarose-TAE pads.</P><P><STRONG>Lara</STRONG></P><P>The PCR product of BolA of María was gel purified and blunt end ligated. Afterwards a transformation was performed into Top10 cells with RFP pSB1C3 as positive control and MilliQ as negative control. Transformations were plated on selective plates and stored overnight at 37ºC. </P><P><STRONG>Tessa</STRONG></P><P>Transformation of controls into new made BL21 competent cells.</P><UL class="list-unstyled"><LI>RFP in psB4A5 plated on AMP plate</LI><LI>MQ plated on AMP plate and CM plate</LI></UL></DIV></DIV><DIV class="row"><H2 class="title-style-3">31<SUP>th</SUP> September 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Plates of yesterday's transformations were transferred from the 37ºC incubator into the fridge. </P></DIV></DIV><A href="#notes" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#notes').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseFive" aria-expanded="false" aria-controls="collapseFive"><DIV class="panel-heading" role="tab" id="headingFive"><H4 class="panel-title">




                                        October




                                    </H4></DIV></A><DIV id="collapseFive" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingFive"><DIV class="panel-body"><DIV class="row"><H2 class="title-style-3">2<SUP>nd</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> Tessa </STRONG></P><P>There is some AMP resistance in the new BL21 cells. This means colony PCR will be necessary when cloning AMP resistant plasmids into BL21.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">3<SUP>rd</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> María </STRONG></P><P>Due to the problems with LB medium we considered the possibility of measuring the amount of silicic acid in polysilicate by hydrolysing it with NaOH (<A href="#references">Müller <I>et al.</I>, 2003</A>) (<A href="#references">Cha <I>et al.</I>, 1999</A>).</P><P>To do so we made a calibration curve of silicic acid diluted in NaOH and another diluted in equal concentrations of NaOH and HCl in to neutralize the pH in case it might suppose a problem. Both of them yielded successful results.</P><P>Since the silicic acid test worked in both calibration curves we decided that we would be using NaOH to hydrolyse the polysilicate layer in silicatein expressing cells to test silicatein kinetics. Thus, we prepared overnight cultures to do the test the following day.</P><P><STRONG>Tessa</STRONG></P><P>Poured plates: AMP, CM and AMP+CM</P><P><STRONG>Célina</STRONG></P><P>Colony PCR of BolA under an inducible promoter in the Addgene backbone (pBbA5c). It yielded some positive cololonies which were picked for overnight culture.</P><P><STRONG> Liza</STRONG></P><P> Preparation of TEM samples vesicles for Wageningen as <A href="https://2016.igem.org/Team:TU_Delft/Notebook#TEMWag" target="_blank"> protocol</A>.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">4<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> María &amp; Lycka</STRONG></P><P>Silicic acid test and preparation of samples for the laser set up and for TEM following the same steps as on 30<SUP>th</SUP> September. OD600 of overnight cultures were (4 times diluted):</P><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Genotype</TH><TH>OD600</TH></TR></THEAD><TBODY><TR><TD>OmpA Sil Taur</TD><TD>0.80</TD></TR><TR><TD>INP Sil Sdom</TD><TD>0.70</TD></TR><TR><TD>Sil Sdom</TD><TD>0.83</TD></TR><TR><TD>BolA</TD><TD>0.76</TD></TR><TR><TD>mCerulean</TD><TD>0.74</TD></TR><TR><TD>mCerulean + OmpA Sil Taur</TD><TD>0.88</TD></TR></TBODY></TABLE><P>Silicic acid was added in the form of sodium silicate to a final concentration of 60µM 3h after induction with IPTG. 5mL samples were taken every 20min and centrifuged to be kept as a pellet in the fridge for the day after. Also every hour an extra 1mL sample was taken for a growth study. For every sample OD600 was measured.</P><P><STRONG> Charlotte &amp; Liza </STRONG></P><P>Tested the mCerulean-expressing cells, the OmpA-silicatein expressing cells and OmpA-silicatein + mCerulean expressing cells in our self-built hardware setup. The cells were grown in LB, spun down and resuspended in PBS. The cells were fixed on a slide by pipetting them under 3% agarose-TAE pads. The cells were excited at 405 nm and imaged at different excitation energies.</P><P><STRONG>Célina</STRONG></P><P>Cryostock of colonies picked yesterday.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">5<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> María &amp; Lycka </STRONG></P><P>Silicic acid test: pellets were washed 3 times with PBS (which was tested not to interfere with the test) to remove all remaining LB medium. Then cells were resuspended with a 10mM NaOH solution and all cells removed by centrifugation to perform the test on the supernatant, that now should contain hydrolysed silicic acid. However, we believe that the cells were partly lysed with the NaOH solution since when trying to perform the test aggregation occurred again. Thus, we had to drop the test and decided not to characterize silicatein kinetics.</P><P>Growth study: plated cells from samples taken yesterday at timepoints 0, 1h, 2h, 3h, 4h and 5h at different dilutions (no dilution, 1/10, 1/100, 1/100). All plates contained to many colonies to be counted so the growth study will be performed again at higher dilutions.</P><P><STRONG>Célina</STRONG></P><P>Plasmid isolation from overnight cultures from the other day.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">6<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG>Lycka</STRONG></P><P>Transformation to BL21 of BolA ind (corrected sequence, fixed by PCR). Tube 3 from minipreps Célina (yesterday)
                                                was used to transform. 
                                            <UL><LI>Bola ind (CM)</LI><LI>BolA ind + OmpA Sil Taur (CM + AMP)</LI><LI>MiliQ - control</LI><LI>RFP pSB1C3 + control</LI></UL></P><P><STRONG>Liza </STRONG></P><P>Preparation of TEM samples OmpA-Silicatein + silicic acid and without silic acid as  <A href=" https://2016.igem.org/Team:TU_Delft/Notebook#TEMTU" target="" _blank=""> protocol </A> on a Quantifoil carbon grid. TEM samples of 3<SUP>rd</SUP> october and today are imaged using TEM under supervision of Yoones Kabiri.</P><P> Preparation of OmpA-silicatein + silicic acid and without silicic acid for AFM measurements as <A href="" target="https://2016.igem.org/Team:TU_Delft/Notebook#AFM" _blank=""> AFM cell fixation protocol</A>.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">7<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG>Liza </STRONG></P><P> AFM measurements of AFM samples of 7<SUP>th</SUP> October under supervision of Allard Katan.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">10<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> Lara </STRONG></P><P>Restriction of plasmids to put the inserts into the standard backbone from the Addgene backbones. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>BolA_con</TD><TD>97.6</TD></TR><TR><TD>BolA</TD><TD>62.5</TD></TR><TR><TD>Sil_Sdom</TD><TD>35.5</TD></TR><TR><TD>OmpA_Sil_Taur</TD><TD>48.3</TD></TR><TR><TD>INP_Sil_Sdom</TD><TD>58.5</TD></TR><TR><TD>pSB1C3</TD><TD>150.4</TD></TR></TBODY></TABLE><P> Since the amount left of BolA and RFP pSB1C3 was too little, these needed to grow from cryostocks again to be miniprepped. </P><P>Restrictions were performed and the backbone was gel-isolated; the rest did not yield restriction products visible on the electrophoresis gel. </P><P><STRONG>Lycka</STRONG></P><P><B>Inoculate</B> cultures in fresh LB for growth study and confocal. ODs measured at 600nm in spectrophotometer.</P><H4>OD600</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>OD measured (4x diluted)</TH><TH>OD inucolated</TH></TR></THEAD><TBODY><TR><TD>OmpA Sil Taur</TD><TD>0.90</TD><TD>0.07</TD></TR><TR><TD>INP Sil Sdom</TD><TD>0.89</TD><TD>0.07</TD></TR><TR><TD>Sil Sdom</TD><TD>0.87</TD><TD>0.07</TD></TR><TR><TD>BolA ind</TD><TD>0.82</TD><TD>0.065</TD></TR><TR><TD>OmpA Sil Taur (no silicate)</TD><TD>0.90</TD><TD>0.07</TD></TR><TR><TD>BolA ind</TD><TD>0.82</TD><TD>0.065</TD></TR><TR><TD>BolA ind + OmpA Sil Taur</TD><TD>0.82</TD><TD>0.065</TD></TR></TBODY></TABLE><P><B>Cryostocked </B>OmpA Sil Taur + BolA ind and BolA ind in BL21.</P><P><STRONG>Tessa</STRONG></P><P>Measured OD600 of [2 colonies of OmpA_Sil_Taur (OmpA)], INP_Sil_Sdom (INP), Sil_Sdom (Sil), OmpA+BolA_ind, and BolA_ind cells for growth study, confocal and SEM, and incubated them until the OD600 was around 0.7.</P><P>Induced cultures [2 colonies of OmpA], INP, Sil and OmpA+BolA_ind</P><P>After 3hr incubation silicate was added to 1 colony of OmpA, INP, Sil and OmpA+BolA_ind</P><P>BolA_ind and OmpA+BolA_ind was incubated for 4 hours and then studied with a confocal microscope.</P><P>After growth study all samples were autoclaved for SEM</P><P><STRONG>Tessa and Lara</STRONG></P><P>A growth study was performed for OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom and BolA_ind. </P><H4>Growth study (number of colonies)</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Dilution </TH><TH>OmpA_Sil_Taur</TH><TH>INP_Sil_Sdom</TH><TH>Sil_Sdom</TH><TH>BolA_ind</TH><TH>Negative control (OmpA without silicate)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="5">t=0</TD><TD>10<SUP>-2</SUP></TD><TD>inf</TD><TD>inf</TD><TD>inf</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-3</SUP></TD><TD>1300</TD><TD>inf</TD><TD>inf</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-4</SUP></TD><TD>337</TD><TD>inf</TD><TD>inf</TD><TD>1003</TD><TD>1150</TD></TR><TR><TD>10<SUP>-5</SUP></TD><TD>190</TD><TD>2400</TD><TD>inf</TD><TD>447</TD><TD>610</TD></TR><TR><TD>10<SUP>-6</SUP></TD><TD>128</TD><TD>1300</TD><TD>2400</TD><TD>814</TD><TD>700</TD></TR><TR><TD scope="row" rowspan="5">t=1</TD><TD>10<SUP>-2</SUP></TD><TD>5</TD><TD>no data</TD><TD>102</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-3</SUP></TD><TD>1</TD><TD>34</TD><TD>15</TD><TD>3080</TD><TD>inf</TD></TR><TR><TD>10<SUP>-4</SUP></TD><TD>0</TD><TD>0</TD><TD>2</TD><TD>718</TD><TD>1000</TD></TR><TR><TD>10<SUP>-5</SUP></TD><TD>0</TD><TD>0</TD><TD>2</TD><TD>872</TD><TD>860</TD></TR><TR><TD>10<SUP>-6</SUP></TD><TD>0</TD><TD>1</TD><TD>0</TD><TD>494</TD><TD>900</TD></TR><TR><TD scope="row" rowspan="5">t=2</TD><TD>10<SUP>-2</SUP></TD><TD>0</TD><TD>51</TD><TD>2</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-3</SUP></TD><TD>0</TD><TD>6</TD><TD>1</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-4</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>inf</TD><TD>2000</TD></TR><TR><TD>10<SUP>-5</SUP></TD><TD>0</TD><TD>2</TD><TD>1</TD><TD>inf</TD><TD>1300</TD></TR><TR><TD>10<SUP>-6</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>1396</TD><TD>970</TD></TR><TR><TD scope="row" rowspan="5">t=3</TD><TD>10<SUP>-2</SUP></TD><TD>0</TD><TD>1400</TD><TD>2</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-3</SUP></TD><TD>0</TD><TD>317</TD><TD>1</TD><TD>inf</TD><TD>860</TD></TR><TR><TD>10<SUP>-4</SUP></TD><TD>0</TD><TD>84</TD><TD>1</TD><TD>inf</TD><TD>410</TD></TR><TR><TD>10<SUP>-5</SUP></TD><TD>0</TD><TD>29</TD><TD>0</TD><TD>678</TD><TD>538</TD></TR><TR><TD>10<SUP>-6</SUP></TD><TD>0</TD><TD>22</TD><TD>0</TD><TD>664</TD><TD>560</TD></TR><TR><TD scope="row" rowspan="5">t=4</TD><TD>10<SUP>-2</SUP></TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-3</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>inf</TD><TD>540</TD></TR><TR><TD>10<SUP>-4</SUP></TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>inf</TD><TD>323</TD></TR><TR><TD>10<SUP>-5</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>inf</TD><TD>303</TD></TR><TR><TD>10<SUP>-6</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>300</TD><TD>193</TD></TR><TR><TD scope="row" rowspan="5">t=5</TD><TD>10<SUP>-2</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>inf</TD><TD>inf</TD></TR><TR><TD>10<SUP>-3</SUP></TD><TD>0</TD><TD>0</TD><TD>1</TD><TD>inf</TD><TD>409</TD></TR><TR><TD>10<SUP>-4</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>inf</TD><TD>190</TD></TR><TR><TD>10<SUP>-5</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>800</TD><TD>72</TD></TR><TR><TD>10<SUP>-6</SUP></TD><TD>0</TD><TD>0</TD><TD>0</TD><TD>1400</TD><TD>890</TD></TR></TBODY></TABLE><P><STRONG> Liza </STRONG></P><P> Calibration curve EGFP. Dilutions of 0,  1/100, 1/200, 1/400, 1/800 and 1/1600 are made of the purified EGFP sample with a concentration of 98µM. 
                                                Dilutions are made in PBS. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">11<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> Lara &amp; Lycka </STRONG></P><P>Tried miniprepping straight from the cryostocks to yield more isolated plasmids.</P><P>Nanodrop was performed by Lycka. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD>Sil_Sdom</TD><TD>86.9</TD></TR><TR><TD>OmpA_Sil_Taur</TD><TD>55.2</TD></TR><TR><TD>INP_Sil_Sdom</TD><TD>55.7</TD></TR><TR><TD>BolA_con</TD><TD>196.7</TD></TR></TBODY></TABLE><P><STRONG>Lycka</STRONG></P><P>Pour LB agar plates with chloramphenicol and ampicilin.</P><P>Inoculate for SEM, TEM, confocal and solar cell experiments.</P><UL><LI>OmpA Sil Taur</LI><LI>INP Sil Sdom</LI><LI>Sil Sdom</LI><LI>BolA ind</LI><LI>BolA ind + OmpA Sil Taur</LI></UL></DIV></DIV><DIV class="row"><H2 class="title-style-3">12<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> María </STRONG></P><P>Preparation of samples for imaging experiments: strains expressing OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom, BolA and OmpA_Sil_Taur together with BolA were grown following the <I>Polysilicate layer in silicatein expressing cells</I> protocol and divided for NileBlueA staining, Rhodamine123 staining, TEM, SEM and solar cell experiments. Stained samples were prepared the same day, the rest were kept as a pellet in the fridge for the following day.</P><P><STRONG>Lycka </STRONG></P><P>Restriction of Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom, BolA_con, BolA_ind with EcoRI and PstI to put in standard backbone. Load restriction product on gel. Run at 100 mA for 45 min. No bands visible. 
                                            </P><P>Try to PCR with standard primers and digest the PCR product. Phusion Polymerase used. Annealing temp 56ºC; extention time 90 sec. 5µL PCR product was put on gel.</P><P>Purification and nanodrop done by Charlotte. Respective concentrations:</P><UL><LI>124.3 ng/µl</LI><LI>89.2 ng/µl</LI><LI>96.4 ng/µl</LI><LI>104.6 ng/µl</LI><LI>135.8 ng/µl</LI></UL><P><STRONG>Lara</STRONG></P><P>To test which settings for UV sterilization are effective, different powersettings within the Stratagene UV Stratalinker 1800 were used. 700 µL of cells transformed with BolA_con in pSB4A5 were loaded in an eppendorf tube and were separately treated with the lid open, so the UV light could be transmitted. The used powersettings were 100000µJ, 120000µJ, 140000µJ, 160000µJ, 180000 µJ, 200000 µJ, 220000µJ and 240000 µJ. Afterwards 50 µL was plated on LB+Amp plates.  </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">13<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> María </STRONG></P><P>Preparation of samples for SEM, trying both freeze drying and glutaraldehyde fixation.</P><P><STRONG>Lycka </STRONG></P><P><B>Restriction</B> of PCR products of Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom, BolA_con, BolA_ind with EcoRI and PstI. Gel purify BolA ind, because its current backbone has
                                                the same resistance as the standard backbone (chloramphenicol). Normally purify others (they come from an amp resistant backbone).
                                                Concentrations after purification:</P><UL><LI>40.8 ng/µl</LI><LI>22.6 ng/µl</LI><LI>34.8 ng/µl</LI><LI>36.0 ng/µl</LI><LI>5.0 ng/µl</LI></UL><P><B>Ligation</B> of 100 ng vector with respective inserts. 1:3 molar ratio used. Incubated for 3 hours.</P><P><B>Transformation</B> to TOP10. Plated after 1.5 hour incubation in nonselective LB. Plated on LB CM plates.</P><P><STRONG>Lara</STRONG></P><P>Checked if there were colonies after UV sterilization. Colonies did appear. We figured that the UV light was not transmitted through the entire layer of 700 µL of cells, so I actually plated non sterilized cells. For that, a new method was used. </P><P><B>New method UV sterilization:</B> Plating 30 µL BolA pBbA5C transformed cells directly on selective LB plates and sterilizing these with the recommended powersetting of 120000µJ and 240000µJ. As positive controls 500 µL cells were washed 3 times with ethanol or heated at 95 ºC for 10 minutes; as negative control living cells were plated. All were incubated at 37 ºC overnight. </P><P><B>Solar cell experiment preparation:</B> First, all cell pellets were diluted as such that same cell concentrations were yielded. Afterwards, these cells were UV sterilized 3 times with a power of 240000 µJ. </P><P><STRONG>Liza</STRONG></P><P>Preparation of TEM samples OmpA-Silicatein + silicic acid and without silic acid as  <A href=" https://2016.igem.org/Team:TU_Delft/Notebook#TEMTU" target="" _blank=""> protocol </A> on a Quantifoil carbon grid and CUUB continuous grid. Sample without silicic acid added was imaged by TEM by Yoones Kabiri.</P></DIV></DIV><DIV class="row"><H2 class="title-style-3">14<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG> María </STRONG></P><P>Colony PCR of fragments cloned into the standard backbone to be submitted to iGEM.</P><P>There were positive results for all constructs, 3 colonies of each were picked to isolate plasmids and cryostock.</P><P><STRONG>Lara</STRONG></P><P>Checked if there were colonies after UV sterilization. No colonies appeared in both powersettings. </P><P><STRONG>Lara &amp; Giannis</STRONG></P><P><B>Solar cell experiment:</B> Visited Stefaan Heirmans at the TU Delft to properly test the change in efficiency in solar cells, using a solar simulator. A total of 3 µL was loaded onto the solar cell and the efficiency was measured.  </P><H4>Solar cell efficiency</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>Type of cells</TH><TH>Solar cell efficiency %</TH></TR></THEAD><TBODY><TR><TD>Empty solar cell</TD><TD>9.75</TD></TR><TR><TD>MilliQ</TD><TD>9.92</TD></TR><TR><TD>OmpA_Sil_Taur not induced and no silicate</TD><TD>9.64</TD></TR><TR><TD>OmpA_Sil_Taur induced and no silicate</TD><TD>9.75</TD></TR><TR><TD>OmpA_Sil_Taur induced and silicate</TD><TD>9.61</TD></TR><TR><TD>BolA + OmpA_Sil_Taur induced and no silicate</TD><TD>9.65</TD></TR><TR><TD>OmpA_Sil_Taur induced and silicate</TD><TD>9.71</TD></TR></TBODY></TABLE><P>Since the bubble of liquid also worked as a lens, we figured these were not the most conclusive results. For that, we have decided to redo these experiments, with cells dried onto glass plates. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">15<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Plasmid isolation of Sil_Sdom, OmpA_Sil_Taur, INP_Sil_Sdom, BolA_con, BolA_ind in the standard backbone. </P><H4>Nanodrop</H4><TABLE class="notebook table table-style-1"><THEAD><TR><TH>BioBrick</TH><TH>Colony number</TH><TH>Concentration (ng/µL)</TH></TR></THEAD><TBODY><TR><TD scope="row" rowspan="3">OmpA_Sil_Taur (pSB1C3)</TD><TD>1</TD><TD>104.9</TD></TR><TR><TD>2</TD><TD>103.5</TD></TR><TR><TD>3</TD><TD>97.4</TD></TR><TR><TD scope="row" rowspan="3">INP_Sil_Sdom (pSB1C3)</TD><TD>20</TD><TD>141.2</TD></TR><TR><TD>21</TD><TD>100.7</TD></TR><TR><TD>22</TD><TD>107.5</TD></TR><TR><TD scope="row" rowspan="3">Sil_Sdom (pSB1C3)</TD><TD>39</TD><TD>111.1</TD></TR><TR><TD>40</TD><TD>80.3</TD></TR><TR><TD>41</TD><TD>95.1</TD></TR><TR><TD scope="row" rowspan="3">BolA_con (pSB1C3)</TD><TD>63</TD><TD>63.2</TD></TR><TR><TD>65</TD><TD>75.2</TD></TR><TR><TD>66</TD><TD>95.9</TD></TR><TR><TD scope="row" rowspan="2">BolA_ind (pSB1C3)</TD><TD>83</TD><TD>120.4</TD></TR><TR><TD>95</TD><TD>75.3</TD></TR></TBODY></TABLE><P>These plasmids were sent to sequencing and were sequence confirmed. </P></DIV></DIV><DIV class="row"><H2 class="title-style-3">16<SUP>th</SUP> October 2016</H2><DIV class="col-md-12"><P><STRONG>Lara</STRONG></P><P>Made dilutions of the samples that are going to be tested in the solar cell simulations. All samples: OmpA_Sil_Taur, INP_Sil_Sdom, Sil_Sdom, BolA_ind and BolA_ind+OmpA_Sil_Taur were diluted 10 times, 100 times and 1000 times. Subsequently, 1.5 mL was pipetted on glass plates that were provided by Stefaan and previously devided in four separate compartments. Samples were dried overnight at 37 ºC. </P></DIV></DIV><A href="#notes" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#notes').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV></DIV><DIV class="protocols"><H2 class="title-style-1">Protocols</H2><H3>During our lab work we used the following protocols.</H3><DIV class="container protocols"><DIV role="tabpanel"><UL class="notebook nav nav-tabs" role="tablist"><LI role="presentation"><A href="#competentcells" onclick="location.href=('#tryscroll');" aria-controls="home" role="tab" data-toggle="tab">Chemically competent Cells</A></LI><LI role="presentation"><A href="#pcr" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">PCR (Phusion)</A></LI><LI role="presentation"><A href="#gelisolation" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Gel Isolation (From Promega Wizard™ Kit)</A></LI><LI role="presentation"><A href="#restriction" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Restriction (NEB enzymes)</A></LI><LI role="presentation"><A href="#ligation" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Ligation</A></LI><LI role="presentation"><A href="#transformation" onclick="location.href=('#tryscroll');" aria-controls="profile" role="tab" data-toggle="tab">Transformation (TOP10)</A></LI><LI role="presentation"><A href="#bl21" onclick="location.href=('#tryscroll');" aria-controls="messages" role="tab" data-toggle="tab">Transformation (BL21)</A></LI><LI role="presentation"><A href="#oneshot" onclick="location.href=('#tryscroll');" aria-controls="messages" role="tab" data-toggle="tab">Transformation (OneShot<SUP>®</SUP> TOP10)</A></LI><LI role="presentation"><A href="#NEB" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Colony PCR (Q5<SUP>®</SUP> High-Fidelity 2X Master Mix)</A></LI><LI role="presentation"><A href="#gotaq" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Colony PCR (GoTaq)</A></LI><LI role="presentation"><A href="#dreamtaq" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Colony PCR (DreamTaq)</A></LI><LI role="presentation"><A href="#electrophoresis" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">DNA Electrophoresis</A></LI><LI role="presentation"><A href="#cultures" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Overnight Cell Cultures</A></LI><LI role="presentation"><A href="#cryostock" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Cell Frozen Stock</A></LI><LI role="presentation"><A href="#miniprep" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Plasmid Isolation (Promega PureYield™ Plasmid Miniprep Kit)</A></LI><LI role="presentation"><A href="#purification" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Restriction Product Purification (from Promega Wizard™ Kit)</A></LI><LI role="presentation"><A href="#polysilicate" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Polysilicate layer in silicatein expressing cells</A></LI><LI role="presentation"><A href="#growth" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Growth study</A></LI><LI role="presentation"><A href="#plate" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Fluorophores kinetic cycle</A></LI><LI role="presentation"><A href="#Test" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Silicic acid test</A></LI><LI role="presentation"><A href="#nileblue" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Nile Blue A staining</A></LI><LI role="presentation"><A href="#rhodamine" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Rhodamine123 staining</A></LI><LI role="presentation"><A href="#UV" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">UV sterilization</A></LI><LI role="presentation"><A href="#gibson" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Gibson Assembly</A></LI><LI role="presentation"><A href="#eM9" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">eM9 medium recipe</A></LI><LI role="presentation"><A href="#TEMWag" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">TEM sample preparation (samples from Wageningen UR)</A></LI><LI role="presentation"><A href="#TEMTU" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">TEM sample preparation (samples from TU Delft)</A></LI><LI role="presentation"><A href="#SEM" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">SEM sample preparation</A></LI><LI role="presentation"><A href="#AFM" onclick="location.href=('#tryscroll');" aria-controls="settings" role="tab" data-toggle="tab">Cell fixation for AFM</A></LI></UL><DIV class="tab-content" id="tryscroll"><DIV role="tabpanel" class="tab-pane" id="competentcells"><H2>Chemically competent cells</H2><OL><LI>Grow overnight culture from single colony or competent cells aliquot.</LI><LI>Inoculate 1:100 in the desired volume of LB.</LI><LI>Grow to OD 600 ~0.35 (important that it is close to 0.35, 0.3-0.4 is good).</LI><LI>Chill on ice for 20 min (it can chill for a 1-2 hours).</LI><LI>Centrifuge at 8000g for 8 minutes in Falcon tubes, decant supernatant.</LI><LI>Resuspend by pipetting gently in 1/10 (of the volume of LB from step 2) of ice-cold TSS.<U>NOTE</U>: It can take a while to break up the pellet (you can use a 25 mL pipette and scrape the pellet off the side of the tube before pipetting).</LI>
                                            Recipe for TSS (100 mL):
                                            <UL><LI>10 g PEG 3350</LI><LI>5 mL DMSO</LI><LI>2 mL 1 M MgCl<SUB>2</SUB></LI><LI>LB to 100 mL</LI></UL>
                                            Filter sterilize and chill to 4ºC
                                            <LI>Freeze 100 µL aliquots in liquid nitrogen &amp; store at -80ºC in 1,5mL Eppendorf tubes.</LI><LI>ALWAYS do a transformation positive control using a plasmid that you know is efficiently taken up by competent cells to confirm competency.</LI><LI>Just as important: ALWAYS do a negative control transformation (add untransformed cells to a plate with antibiotics) to confirm that you don't have an antibiotic-resistant contaminant strain in your competent cell stock. This happens from time to time and can cause lots of agony!</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="pcr"><H2>PCR (Phusion polymerase)</H2><OL><LI>Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Component</TH><TH>Volume (µL) for 20µL reaction</TH><TH>Volume (µL) for 50µL reaction</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">5X Phusion HF Buffer</TD><TD>4</TD><TD>10</TD></TR><TR><TD class="firstcolumn">10mM dNTPs</TD><TD>0.4</TD><TD>1</TD></TR><TR><TD class="firstcolumn">FW Primer</TD><TD>1</TD><TD>2.5</TD></TR><TR><TD class="firstcolumn">RV Primer</TD><TD>1</TD><TD>2.5</TD></TR><TR><TD class="firstcolumn">Phusion polymerase</TD><TD>0.2</TD><TD>0.5</TD></TR><TR><TD class="firstcolumn">Template DNA (&lt;250ng)</TD><TD>Variable</TD><TD>Variable</TD></TR><TR><TD class="firstcolumn">Nuclease-free H<SUB>2</SUB>O</TD><TD>Up to 20µL</TD><TD>Up to 50µL</TD></TR></TBODY></TABLE><U>NOTE</U>: The 250ng of template DNA are approximate, choose a volume that works fine for all your samples and fill it in the excel sheet so you are able to prepare a mix for all samples at the same time.</LI><LI>Add the total volume per sample (without DNA) to each PCR tube.</LI><LI>Add the template DNA to each tube.</LI><LI>Put the tubes in the PCR machine and apply the following program (it needs to be adjusted for primers annealing temperature and extension time):
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Initial denaturation</TD><TD>98</TD><TD>30sec</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>98</TD><TD>10sec</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>Tm-3</TD><TD>20sec</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>15-30sec/1kb</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>5min</TD></TR><TR><TD>4</TD><TD>Hold</TD></TR></TBODY></TABLE></LI><LI>Run a DNA electrophoresis following the <I>DNA electrophoresis</I> protocol.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="gelisolation"><H2>Gel isolation</H2><OL><LI>Run a DNA electrophoresis (following the <I>DNA electrophoresis</I> protocol), loading the full volume of PCR or restriction product into the agarose gel. If necessary, use multiple wells.</LI><LI>Cut the band (or bands) corresponding to the fragment of interest with a razor blade and put it in a pre-weighted 1.5mL tube, then calculate the weight of the gel fragment.</LI><LI>Add 10µL of Membrane Binding Solution per 10mg of gel slice. Vortex and incubate at 50-65ºC until gel slice is completely dissolved.</LI><LI>Pre-warm an aliquot of Nuclease-free water at 50-65ºC.</LI><LI>Insert SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.</LI><LI>Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate at room temperature for 1 minute. <U>NOTE</U>: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.</LI><LI>Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.</LI><LI>Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.</LI><LI>Add 700µL of Membrane Wash Solution (if it is the first use, dilute it with 95% ethanol following the bottle’s instructions).</LI><LI>Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.</LI><LI>Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.</LI><LI>Repeat steps 9-11 with 500µL of Membrane Wash Solution and centrifuging for 5 minutes.</LI><LI>Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.</LI><LI>Transfer the SV Minicolumn to an empty 1.5mL tube carefully labelled.</LI><LI>Add 500µL of the pre-warmed water (300µL for higher concentrations or when small amounts of DNA are suspected) directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.</LI><LI>Incubate at room temperature for 5 minutes.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI>Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep at 4ºC (for immediate use) or -20ºC (for storage).</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="restriction"><H2>Restriction</H2><OL><LI>Get the detailed protocol for restriction from the following <A href="http://nebcloner.neb.com/#!/redigest">website</A>, according to the enzymes used. When high restriction product concentrations are needed (generally for backbones) the amount of DNA can be increased to 3-5g. The volume of DNA necessary can be obtained from the concentration measured at NANODROP.</LI><LI>Mix all the components stated by this restriction tool in a 1.5mL Eppendorf tube and incubate at 37ºC for 1-2h.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="ligation"><H2>Ligation</H2><OL><LI>Make your own ligation protocol.</LI><FORM id="form1"><TITLE>Ligation Protocol</TITLE><H2>Ligation Protocol</H2><FIELDSET><LABEL>Vector length (bp) :</LABEL><LABEL>Vector mass (ng)   :</LABEL><LABEL>Vector concentration (ng/µL) :</LABEL><LABEL>Insert length (bp)  :</LABEL><LABEL>Insert concentration (ng/µL) :</LABEL><LABEL>Ratio Insert/Vector  :</LABEL></FIELDSET></FORM><BUTTON onclick="myFunction()">Calculate</BUTTON><LI>Mix all the components in a tube (always add the T4 ligase at the end).</LI><LI>Incubate for at least 3 hours at 16ºC.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="transformation"><H2>Transformation (Top10 chemically competent cells)</H2><OL><LI>Get as many aliquots of competent cells (100L) from the -80ºC freezer as transformations to be done and put them on ice for 10-15min.<U>NOTE</U>: Don’t forget positive (RFP in the backbone you are using, check plasmids box in the -20ºC freezer) and negative controls (no DNA).</LI><LI>Add to the 100µL of competent cells:
                                                </LI><LI>1µL of plasmid or 2µL of ligation product.</LI><LI>20µL 5X KCM buffer.</LI><LI>Sterile water up to 200µL.</LI></OL><LI>Mix by flicking the tube.</LI><LI>Incubate on ice for 20-30 minutes.</LI><LI>Heat shock at 42ºC for exactly 90 seconds.</LI><LI>Return the cells to ice for 1-2 minutes.</LI><LI>Add 400µL of fresh LB.</LI><LI>Incubate at 37ºC with shaking for 1 hour.<U>NOTE</U>: Not at the thermoblock, use a shaking incubator.</LI><LI>To increase the concentration of cells before plating, centrifuge the tubes at 4000rpm for 3 minutes, remove 500L of medium, resuspend the pellet in the remaining volume.</LI><LI>Plate the rest of the volume of the transformation on selective LB plates and incubate at 37ºC overnight.</LI><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="bl21"><H2>Transformation (BL21 competent cells)</H2><OL><LI>Put a tube of BL21 Competent E. coli cells on ice for 10 minutes.</LI><LI>Add 2µL of plasmid DNA to the cell mixture.</LI><LI>Mix by flicking the tube.</LI><LI>Place the tube on ice for 30 minutes.</LI><LI>Heat shock at 42ºC for exactly 10 seconds.</LI><LI>Place on ice for 5 minutes.</LI><LI>Add 950µL of SOC medium at room temperature.</LI><LI>Incubate at 37ºC with shaking for 1 hour.<U>NOTE</U>: Not at the thermoblock, use a shacking incubator.</LI><LI>To increase the concentration of cells before plating, centrifuge the tubes at 4000rpm for 3 minutes, remove 700µL of medium, resuspend the pellet in the remaining volume.</LI><LI>Plate 70µL of the transformation on selective LB plates and incubate at 37ºC overnight.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="oneshot"><H2>Transformation (OneShot<SUP>®</SUP> Top10 Chemically competent cells)</H2><OL><LI>Get half the amount of tubes of OneShot<SUP>®</SUP> TOP10 chemically competent cells (tubes with a purple lid) than the transformations to be done from the -80ºC freezer and put them on ice for 10-15min.</LI><LI>Divide the volume of competent cells of each tube in 2. Thus, transfer 25L to a new sterile tube.</LI><LI>Add 1µL of plasmid or 2µL of ligation product into each tube and mix by flicking.</LI><LI>Incubate on ice for 30 min.</LI><LI>Heat shock the cells for 30 seconds at 42ºC.</LI><LI>Put back on ice for 2 minutes.</LI><LI>Add 250µL of LB medium.</LI><LI>Incubate at 37ºC with shaking for 1 hour.<U>NOTE</U>: Not at the thermoblock, use a shacking incubator.</LI><LI>To increase the concentration of cells before plating, centrifuge the tubes at 4000rpm for 3 minutes, remove 200µL of medium, resuspend the pellet in the remaining volume.</LI><LI>Plate the rest of the volume of the transformation on selective LB plates and incubate at 37ºC overnight.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="NEB"><H2>Colony PCR (Q5<SUP>®</SUP> High-Fidelity 2X Master Mix)</H2><OL><LI>Choose the colonies to be picked for testing and number them. When available pick at least 8 colonies per plate, or more if there are slots left in the PCR machine (it fits 25 samples).<U>NOTE</U>: It is more convenient if the numbering is continued between plates of the same batch instead of starting at 1 in each plate. This way there is only one set of numbers per date.<U>NOTE</U>: it is possible that not all samples fit in one PCR and then more than one would need to be performed, so keep that in mind for the timing of experiments. But don’t be sad, that would mean that the transformation worked so well that you have a lot of colonies to choose from!! ☺</LI><LI>Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Component</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Q5<SUP>®</SUP> HF 2X Master Mix</TD><TD>12.5</TD></TR><TR><TD class="firstcolumn">Primer VF2</TD><TD>1.25</TD></TR><TR><TD class="firstcolumn">Primer VR</TD><TD>1.25</TD></TR><TR><TD class="firstcolumn">Nuclease-free H<SUB>2</SUB>O</TD><TD>10</TD></TR><TR><TD class="firstcolumn">Total</TD><TD>25</TD></TR></TBODY></TABLE></LI><LI>Pipette 25µL of mix into each PCR tube.</LI><LI>Under sterile conditions, pick a colony making sure that part of it stays on the plate for later use and soak it in the content of the PCR tube. When finished, put them back in the incubator so they grow for a bit longer while the PCR runs.</LI><LI>Put the tubes in the PCR machine and apply the following program (applicable for the primers mentioned above, for other sets of primers the annealing temperature needs to be adjusted):
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Cell lysis and initial denaturation</TD><TD>95</TD><TD>15min</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>95</TD><TD>30sec</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>53</TD><TD>30sec</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>30sec/1kb</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>5min</TD></TR><TR><TD>4</TD><TD>Hold</TD></TR></TBODY></TABLE></LI><LI>While the PCR is running, prepare a 0.5mL tube with 2µL loading buffer for each PCR sample and one tube with 4µL of 1Kb DNA ladder (or 8µL if 2 gels are needed). This way you can take these tubes into the EtBr area and don’t worry about how to handle the PCR tubes.</LI><LI>Cast an agarose gel following the <I>DNA electrophoresis</I> protocol.</LI><LI>Prepare an agarose gel with SNAPGENE for comparison with the actual results.</LI><LI>Once the PCR program is finished, transfer 8L of PCR product into the tubes containing the loading buffer and follow the <I>DNA electrophoresis</I> protocol.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="gotaq"><H2>Colony PCR (GoTaq)</H2><OL><LI>Under sterile conditions, pick a colony and dilute it in 25µL of miliQ water.</LI><LI>Incubate at 95ºC for 10 min.</LI><LI>Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Component</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">GoTaq Mastermix 2X</TD><TD>25</TD></TR><TR><TD class="firstcolumn">Primer VF2 (10µM)</TD><TD>1</TD></TR><TR><TD class="firstcolumn">Primer VR (10µM)</TD><TD>1</TD></TR><TR><TD class="firstcolumn">Nuclease-free H<SUB>2</SUB>O</TD><TD>18</TD></TR><TR><TD class="firstcolumn">Total</TD><TD>50</TD></TR></TBODY></TABLE></LI><LI>Pipette 45µL of mix into each PCR tube (one per colony).</LI><LI>Centrifuge the colony mixture for 30 seconds at 10000rpm.</LI><LI>Add 5µL of supernatant of colony mixture to each PCR tube.</LI><LI>Put the tubes in the PCR machine and apply the following program (applicable for the primers mentioned above, for other sets of primers the annealing temperature needs to be adjusted):
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Initial denaturation</TD><TD>95</TD><TD>2min</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>95</TD><TD>30sec</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>53</TD><TD>30sec</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>1min/1kb</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>5min</TD></TR><TR><TD>4</TD><TD>Hold</TD></TR></TBODY></TABLE></LI><LI>While the PCR is running, prepare a 0.5mL tube with 2µL loading buffer for each PCR sample and one tube with 4µL of 1Kb DNA ladder (or 8µL if 2 gels are needed). This way you can take these tubes into the EtBr area and don’t worry about how to handle the PCR tubes.</LI><LI>Cast an agarose gel following the <I>DNA electrophoresis</I> protocol.</LI><LI>Prepare an agarose gel simulation with SNAPGENE for comparison with the actual results.</LI><LI>Once the PCR program is finished, transfer 8µL of PCR product into the tubes containing the loading buffer and follow the <I>DNA electrophoresis</I> protocol.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="dreamtaq"><H2>Colony PCR (DreamTaq)</H2><OL><LI>Under sterile conditions, pick a colony and dilute it in 25µL of miliQ water.</LI><LI>Incubate at 95ºC for 10 min.</LI><LI>Prepare the mix for all the samples in a single 1.5mL tube (mind pipetting error!). For one sample:
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Component</TH><TH>Volume (µL)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">DreamTaq Hot Start PCR Master Mix (2X)</TD><TD>25</TD></TR><TR><TD class="firstcolumn">Primer VF2 (10µM)</TD><TD>1</TD></TR><TR><TD class="firstcolumn">Primer VR (10µM)</TD><TD>1</TD></TR><TR><TD class="firstcolumn">Nuclease-free H<SUB>2</SUB>O</TD><TD>18</TD></TR><TR><TD class="firstcolumn">Total</TD><TD>50</TD></TR></TBODY></TABLE></LI><LI>Pipette 45µL of mix into each PCR tube (one per colony).</LI><LI>Centrifuge the colony mixture for 30 seconds at 10000rpm.</LI><LI>Add 5µL of supernatant of colony mixture to each PCR tube.</LI><LI>Put the tubes in the PCR machine and apply the following program (applicable for the primers mentioned above, for other sets of primers the annealing temperature needs to be adjusted):
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Step</TH><TH>Temperature (ºC)</TH><TH>Time</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">Initial denaturation</TD><TD>95</TD><TD>3min</TD></TR><TR><TD class="firstcolumn">Denaturation</TD><TD>95</TD><TD>30sec</TD><TD rowspan="3">x30 cycles</TD></TR><TR><TD class="firstcolumn">Annealing</TD><TD>53</TD><TD>30sec</TD></TR><TR><TD class="firstcolumn">Extension</TD><TD>72</TD><TD>1min for first 2kb + 1min/1kb</TD></TR><TR><TD class="firstcolumn">Final extension</TD><TD>72</TD><TD>5min</TD></TR><TR><TD>4</TD><TD>Hold</TD></TR></TBODY></TABLE></LI><LI>While the PCR is running, prepare a 0.5mL tube with 2µL loading buffer for each PCR sample and one tube with 4µL of 1Kb DNA ladder (or 8µL if 2 gels are needed). This way you can take these tubes into the EtBr area and don’t worry about how to handle the PCR tubes.</LI><LI>Cast an agarose gel following the <I>DNA electrophoresis</I> protocol.</LI><LI>Prepare an agarose gel simulation with SNAPGENE for comparison with the actual results.</LI><LI>Once the PCR program is finished, transfer 8µL of PCR product into the tubes containing the loading buffer and follow the <I>DNA electrophoresis</I> protocol.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="electrophoresis"><H2>DNA electrophoresis</H2><P><B>Work clean!</B> Handle with gloves all material labelled as EtBr contaminated, don’t take it outside of the EtBr area and don’t touch with gloves anything that is not labelled as EtBr contaminated.</P><OL><LI>Prepare TAE buffer: take the 10X concentrated TAE from the chemicals cabinet and dilute it 10 times with demi water. For 500mL, add 50mL to 450mL of dH2O.</LI><LI>Weigh agarose for a 1% gel. For 200mL, 2g of agarose is necessary.</LI><LI>Mix the TAE solution with the agarose and heat the solution until it is completely dissolved.</LI><LI>Add Ethidium bromide to the gel mould. 2.5L for the small gel and 5L for the big one.</LI><LI>Pour the solution into the mould making sure there are no bubbles and that the EtBr is completely mixed. Let it solidify (approx. 20 minutes).</LI><LI>Transfer the gel to the electrophoresis cell minding the arrow that indicates the direction of DNA migration, remove the combs and cover it with TAE.</LI><LI>Prepare tubes with 5/6 of PCR product and 1/6 of loading buffer.</LI><LI>Load the molecular weight marker (ladder) in the first well (check the appropriate volume for each marker, generally 5L works fine) and load the samples in the other wells, according to the order in your lab journal.<U>NOTE</U>: Do not contaminate the loading buffer and ladder with EtBr! Do not touch it while wearing a glove.</LI><LI>Connect the cables following the colour code and run at 100-130V for 30-60 min.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="purification"><H2>Restriction product purification</H2><OL><LI>For restriction products containing more than one large fragment follow the first 3 steps of the <I>Gel isolation</I> protocol.
                                                When restriction yields only one large fragment, add an equal volume of Membrane Binding Solution to the volume of restriction product.</LI><LI>Pre-warm an aliquot of Nuclease-free water at 50-65ºC.</LI><LI>Insert SV Minicolumn into Collection Tube and label both of them according to the labelling of your samples.</LI><LI>Transfer the dissolved gel mixture to the Minicolumn assembly. Incubate at room temperature for 1 minute. <U>NOTE</U>: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.</LI><LI>Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.</LI><LI>Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.</LI><LI>Add 700µL of Membrane Wash Solution (if it is the first use, dilute it with 95% ethanol following the bottle’s instructions).</LI><LI>Centrifuge the SV Minicolumn assembly at maximum speed for 1 minute.</LI><LI>Discard the flowthrough and reinsert the SV Minicolumn into the Collection Tube.</LI><LI>Repeat steps 9-11 with 500µL of Membrane Wash Solution and centrifuging for 5 minutes.</LI><LI>Once the Collection Tube is empty, centrifuge the Minicolumn assembly at maximum speed for 1 minute with the microcentrifuge lid open to allow ethanol full evaporation.</LI><LI>Transfer the SV Minicolumn to an empty 1.5mL tube carefully labelled.</LI><LI>Add 500µL of the pre-warmed water (300µL for higher concentrations or when small amounts of DNA are suspected) directly to the centre of the SV Minicolumn, without touching the membrane with the pipette tip.</LI><LI>Incubate at room temperature for 5 minutes.</LI><LI>Centrifuge at maximum speed for 1 minute.</LI><LI>Discard the SV Minicolumn, cap the tube containing the eluted DNA and keep at 4ºC (for immediate use) or -20ºC (for storage).</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="cryostock"><H2>Cell frozen stock</H2><B>Work sterile!</B> If plasmid DNA needs to be purified from the overnight culture, remove the necessary volume from the Falcon tube before the cryostock procedure.
                                        <OL><LI>Centrifuge the 50mL tubes at 2000rpm for 10 min.</LI><LI>Decant the supernatant without disturbing the pellet.</LI><LI>Add fresh LB medium, around 1/3 of the starting volume of the culture.</LI><LI>Mix by vortexing until the pellet is completely resuspended.</LI><LI>Add sterile 80% glycerol solution, the same volume as fresh LB in step 4.</LI><LI>Mix by vortexing.</LI><LI>Make 1.5mL aliquots in cryotubes and label them with the cell type, the BioBrick and backbone they contain (if applicable) and the date.</LI><LI>Store them at -80ºC and update the inventory in the drive folder.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="miniprep"><H2>Plasmid isolation</H2><OL><LI>Pre-warm an aliquot of Nuclease-free water at 50-60ºC.</LI><LI>Add 1.5 mL of bacterial culture in LB medium to a 1.5 mL micro-centrifuge tube. Centrifuge that tube at max speed for 3 min.<U>NOTE</U>: For higher concentrations repeat step 1 in the same tube.<U>NOTE</U>: If cryostocks need to be prepared from the same overnight culture tube, remove the necessary volume for plasmid isolation under sterile conditions!</LI><LI>Remove the supernatant.</LI><LI>Resuspend the pellet in 600 µL of MiliQ water.</LI><LI>Add 100 µL of Cell Lysis Buffer, and mix by inverting 6 times. The color change to blue indicates complete lysis.</LI><LI>Add 350 µL of cold (4-8ºC) Neutralization Buffer, and mix by inverting the tube. The color change to yellow indicates total neutralization.</LI><LI>Introduce a PureYield Minicolumn into a PureYield Collection tube.</LI><LI>Centrifuge at maximum speed for 3 minutes, and transfer the supernatant to the Minicolumn assembly.<U>NOTE</U>: When pipetting into the column, aim the pipette to the wall not the membrane to avoid damaging it.</LI><LI>Centrifuge at maximum speed for 15 seconds and discard the flowthrough.</LI><LI>Add 200 µL of Endotoxin Removal Wash to the minicolumn. Centrifuge at maximum speed for 15 seconds. It is not necessary to empty the Collection Tube now.</LI><LI>Add 400 µL of Column Wash Solution to the minicolumn, and centrifuge at maximum speed for 30 seconds.</LI><LI>Transfer the PureYield Minicolumn to a clean 1.5 mL tube, add 30 µL (or 50L if you expect high concentrations) of pre-warmed MiliQ water directly to the minicolumn matrix. Let stand for 5 minutes at room temperature.</LI><LI>Centrifuge at maximum speed in a microcentrifuge for 15 seconds to elute plasmidic DNA. Cap the tube, and store the DNA solution at -20 ºC (or keep it at 4ºC if it is to be used immediately).<U>NOTE</U>: If possible, put the lids in line with the circle of the centrifuge (so not pointing out or inwards) so that the caps don’t break while centrifuging.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="cultures"><H2>Overnight cell cultures</H2><OL><LI>Prepare 20mL of selective LB (with the antibiotic your cells are resistant to) per colony to be picked.<U>NOTE</U>: You can use a sterile bottle to prepare the total volume.</LI><LI>Transfer 10mL of selective LB to a 50mL Falcon tube. Prepare 2 tubes per colony: 1 for miniprep and 1 for cryostocks.</LI><LI>Pick 1 colony using an inoculation loop and soak it in the LB. Make sure you have cells left for the second tube.</LI><LI>Take the tubes to room F1.690 and incubate overnight at 37ºC in the shaking incubator.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="polysilicate"><H2>Polysilicate layer in silicatein expressing cells</H2><OL><LI>Inoculate a fresh culture of cells from an overnight culture. Make a dilution of around 1/100 of the overnight culture into fresh selective LB.</LI><LI>Incubate in the shaking incubator at 37ºC and 250 rpm until it reaches exponential phase (just measure OD600 from time to time until duplication time is close to 20 min).</LI><LI>Induce the culture with 1/1000 volume of IPTG 1M.</LI><LI>Incubate in the shaking incubator at 37ºC and 250 rpm for 3h.</LI><LI>Add sodium silicate to a final concentration of 60µM from the 1mM stock (maybe new stock needs to be made from the 1M stock, keep sterile!!).</LI><LI>Incubate at 37ºC and 250 rpm for 3-4h, after this the cells are ready for further experiments.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="growth"><H2>Growth study</H2><OL><LI>Follow the <I>Polysilicate layer</I> protocol until step 5.</LI><LI>Take a 100µL sample directly after adding the sodium silicate, this is t=0.</LI><LI>Incubate at 37ºC and 250 rpm and take a 100µL sample every hour.
                                                For each sample:
                                                </LI><LI>Make a series of dilutions in 1/10 steps until reaching a 1/10<SUP>6</SUP>. It can be done in a 96 well plate by adding 10µL of the previous dilution in 90µL of LB medium.</LI><LI>Plate in agar plates 20µL of dilutions 1/100, 1/10<SUP>3</SUP>, 1/10<SUP>4</SUP>, 1/10<SUP>5</SUP>, 1/10<SUP>6</SUP>.</LI></OL><LI>Incubate at 37ºC overnight.</LI><LI>Hopefully, in some of the dilutions the colonies will be countable. Count colonies in plates for at least one dilution of each strain and time point to make a growth/viability curve.</LI><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="plate"><H2>Fluorophores kintetic cycle</H2><OL><LI>Grow cells expressing the fluorophores of interest in eM9 medium since LB has its own fluorescence.</LI><LI>Measure OD600 overnight cultures.</LI><LI>Dilute in eM9 medium an aliquot of each culture to an OD600 of 0.1.</LI><LI>Add 100µL of diluted culture to 4 wells per culture (that way we obtain 4 replicas of the experiment), also make blanks with sterile eM9.</LI><LI>Program the plate reading machine so it incubates the plate at 37ºC with shaking and makes an OD600 and fluorescence measurement every 15min for 24h.
                                                Excitation and emission wavelengths to be used for each fluorophore are:
                                                <TABLE class="protocols table table-style-1"><THEAD><TR><TH>Excitation (nm)</TH><TH>Emission (nm)</TH></TR></THEAD><TBODY><TR><TD class="firstcolumn">mVenus</TD><TD>510</TD><TD>544</TD></TR><TR><TD class="firstcolumn">mCerulean</TD><TD>433</TD><TD>475</TD></TR><TR><TD class="firstcolumn">GFP</TD><TD>488</TD><TD>522</TD></TR></TBODY></TABLE><U>NOTE</U>: Some of these wavelengths are adjusted to be at least 34nm apart due to machine limitations.
                                            </LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="Test"><H2>Silicic acid test</H2><P>For a 1 mL sample:</P><OL><LI>Add 20µL of reagent Si-1 and mix.</LI><LI>Wait for 3 min.</LI><LI>Add 20µL of reagent Si-2 and 100µL of Si-3 and mix.</LI><LI>Wait for 10 min.</LI><LI>Measure absorbance at 824nm.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="nileblue"><H2>Nile Blue A staining</H2><OL><LI>Heat fix cells in a glass slide.</LI><LI>Stain the glass slide with 1% Nile Blue A solution at 55ºC for 10 min.</LI><LI>Wash with tap water to remove excess stain.</LI><LI>Wash in 8% aqueous acetic acid for 1 min.</LI><LI>Wash and dry.</LI><LI>Remoist with tap water.</LI><LI>Cover with a cover slip and seal it with nail polish.</LI><LI>Nile Blue A is used to visualize PHB inside the cells. It is fluorescent when in contact with PHB and excited at around 460nm and fluoresces as bright orange (<A href="#references">Ostle &amp; Holt, 1982</A>).</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="rhodamine"><H2>Rhodamine123 staining</H2><OL><LI>Add 1/1000 volume of Rhodamine123 to the culture to be stained.</LI><LI>Wait for 10 min.</LI><LI>Centrifuge for 1 min at maximum speed.</LI><LI>Wash the cells 5 times with PBS.</LI><LI>Prepare samples for microscopy in a glass slide.</LI><LI>Rhodamine123 is a fluorescent dye which is excited at 546nm and emits green fluorescence (<A href="#references">Li, Chu &amp; Lee, 1989</A>)(<A href="#references">Johnson, Walsh &amp; Chen, 1980</A>)</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="UV"><H2>UV sterilization</H2><OL><LI>Plate 30µL of cells on selective LB plates.
                                                </LI><LI>Positive control: dead culture (for example, autoclaved).</LI><LI>Negative control: culture of living cells.</LI><LI>Test: culture of living cells.
                                                </LI></OL><LI>Expose the test plates to UV light using the Stratagene UV Stratalinker 1800 at power 120000µJ and 240000µJ, respectively.</LI><LI>Incubate all plates at 37ºC overnight.</LI><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="eM9"><H2>eM9 medium recipe</H2><OL><LI>
                                                10X M9 stock solution
                                                </LI><LI>75 g/L Na<SUB>2</SUB>HPO<SUB>4</SUB>·2H<SUB>2</SUB>O</LI><LI>30 g/L KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>5 g/L NaCl</LI><LI>10 g/L NH<SUB>4</SUB>Cl</LI></OL><LI>
                                                eM9 medium for protein overexpression
                                                </LI><LI>1X M9 stock solution</LI><LI>10 mL/L CaCl<SUB>2</SUB>2H<SUB>2</SUB>O, 10mM</LI><LI>1 mL/L MgSO<SUB>4</SUB>7H<SUB>2</SUB>O, 1M</LI><LI>5 mL/L L-Tryptophan, 1%</LI><LI>100mM Glucose</LI><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="gibson"><H2>Gibson assembly</H2><OL><LI>Put in a PCR tube:
                                                </LI><LI>100ng of vector</LI><LI>100ng of insert</LI><LI>2µL of buffer</LI><LI>Up to 10µL of Nuclease-free water</LI></OL><LI>Incubate at 37ºC for 1h in a thermocycler.</LI><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="SEM"><H2>SEM sample preparation</H2><OL><LI>Centrifuge 1mL of culture for 1min at maximum speed.</LI><LI>Discard the supernatant</LI><LI>Resuspend the pellet with 3% glutaraldehyde solution.<U>NOTE</U>: Glutaraldehyde is a dangerous chemical, wear gloves and safety goggles!</LI><LI>Let it stand for 10min.</LI><LI>Centrifuge for 1 min at maximum speed.</LI><LI>Discard the supernatant.</LI><LI>Resuspend the pellet in a 30% ethanol solution.</LI><LI>Centrifuge for 1 min at maximum speed.</LI><LI>Discard the supernatant.</LI><LI>Repeat steps 7-9 with ethanol solutions of 50%, 70%, 80%, 90%, 96% and 100%.</LI><LI>Resuspend the supernatant one last time in 100% ethanol.</LI><LI>The samples are ready to be put in a glass slide.</LI><LI>Preparation of slides and SEM images were done by Dominik Schmieden (see <A href="https://2016.igem.org/Team:TU_Delft/Team#advisors"><B>advisors</B></A> page)</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="TEMWag"><H2>TEM sample preparation (Wageningen samples)</H2><OL><LI>Take plasma cleaned 400 mesh carbon grid.</LI><LI>Put 3 µl of sample (lipids) on the carbon grid and incubate 1 min.</LI><LI>Absorb sample with a filter.</LI><LI>Wash the sample 3x with MiliQ water absorbing it every time with a filter.</LI><LI>Add 3µl 3% of uranyl acetate and incubate for 30 seconds.</LI><LI>Remove stain with filter.</LI><LI>Image with high angle annular dark field transmission electron microscopy with the help of Yoones Kabiri (see <A href="https://2016.igem.org/Team:TU_Delft/Attributions#acknowledgements"><B>acknowledgements</B></A> page).</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="TEMTU"><H2>TEM sample preparation (TU Delft samples)</H2><OL><LI>Grow cells in LB centrifuge and remove supernatant.</LI><LI>Dilute pellet in MiliQ water.</LI><LI>Take a TEM grid and put 3µl polylysine (1%) and incubate 1 min.</LI><LI>Wash the grids with MiliQ water.</LI><LI>Drop cast 3µl of sample and incubate 1 min.</LI><LI>Wash with MiliQ water 3 times.</LI><LI>Image with high angle annular dark field transmission electron microscopy  with the help of Yoones Kabiri (see <A href="https://2016.igem.org/Team:TU_Delft/Attributions#acknowledgements"><B>acknowledgements</B></A> page).</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV><DIV role="tabpanel" class="tab-pane" id="AFM"><H2>Cell fixation for AFM</H2><OL><LI>Dilute a cell pellet in MQ.</LI><LI>Take coverslips and plasma clean (oxygen) for 20 seconds.</LI><LI>Make 1% APTES in MQ.</LI><LI>Pipet a few drops of the APTES solution onto the coverslip and incubate for 2 minutes.</LI><LI>Dip the coverslip in MQ a few times to wash.</LI><LI>Pipet a few drops (~100µl) of bacterial culture onto the coverslip and incubate for 2 minutes.</LI><LI>Dip the coverslip in MQ a few times to wash.</LI><LI>Dry the coverslip with a nitrogen gun.</LI><LI>Optional: check density of fixed cells with normal microscope.</LI><LI>Cells are imaged and measured with the help of Allard Katan (see <A href="https://2016.igem.org/Team:TU_Delft/Attributions#acknowledgements"><B>acknowledgements</B></A> page) in an Atomic force microscope.</LI></OL><A href="#protocols" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#protocols').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Protocols</A></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="references container"><H4 class="footer-title">References</H4><OL><LI>Cha, J. N., Shimizu, K., Zhou, Y., Christiansen, S. C., Chmelka, B. F., Stucky, G. D., &amp; Morse, D. E. (1999). Silicatein filaments and subunits from a marine sponge direct the polymerization of silica and silicones in vitro. Proceedings of the National Academy of Sciences of the United States of America, 96(2), 361–365.</LI><LI>Johnson, L. V., Walsh, M. L., &amp; Chen, L. B. (1980). Localization of mitochondria in living cells with rhodamine 123. Proceedings of the National Academy of Sciences of the United States of America, 77(2), 990–994.</LI><LI>Li, C. W., Chu, S., &amp; Lee, M. (1989). Characterizing the silica deposition vesicle of diatoms. Protoplasma, 151(2-3), 158–163. http://doi.org/10.1007/BF01403453</LI><LI>Müller, W. E. G., Krasko, A., Pennec, G. Le, Steffen, R., Wiens, M., Ammar, M. S. A., Schröder, H. C. (2003). Molecular Mechanism of Spicule Formation in the Demosponge Suberites domuncula: Silicatein-Collagen-Myotrophin. In W. E. G. Müller (Ed.), Silicon Biomineralization: Biology --- Biochemistry --- Molecular Biology --- Biotechnology (pp. 195–221). Berlin, Heidelberg: Springer Berlin Heidelberg. http://doi.org/10.1007/978-3-642-55486-5_8</LI><LI>Ostle, A. G., &amp; Holt, J. G. (1982). Nile blue A as a fluorescent stain for poly-beta-hydroxybutyrate. Applied and Environmental Microbiology, 44(1), 238–241.</LI></OL></DIV><FOOTER class="main-footer"><DIV id="contact" class="footer-main"><DIV class="row"><DIV class="col-md-offset-1 col-md-3"><DIV class="footer-col"><H3 class="footer-title">Contact</H3><DIV class="footer-content"><STRONG> Address:</STRONG><UL><LI class="list-unstyled">Faculty of Applied Sciences</LI><LI class="list-unstyled">Room E0.100</LI><LI class="list-unstyled">Van der Maasweg 9</LI><LI class="list-unstyled"> 2629 HZ Delft</LI><LI class="list-unstyled">The Netherlands</LI></UL><STRONG>Email:</STRONG><UL><LI class="list-unstyled">tudelft.igem@gmail.com</LI></UL><STRONG>Social media:</STRONG><UL><LI class="list-unstyled"><A href="https://www.facebook.com/TUDelft.iGEM2016/" target="_blank"> Facebook</A></LI><LI class="list-unstyled"><A href="https://twitter.com/TUDelft_iGEM" target="_blank">Twitter</A></LI><LI class="list-unstyled"><A href="https://www.instagram.com/igemtudelft/" target="_blank">Instagram</A></LI><LI class="list-unstyled"><A href="https://www.youtube.com/channel/UCG_QXJIdl79P_zaHgzO6ppA" target="_blank">Youtube</A></LI></UL></DIV></DIV></DIV><DIV class="col-md-8"><DIV class="footer-col"><H3 class="footer-title">Sponsors</H3></DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>