<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:ZJU-China"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:ZJU-China/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:ZJU-China/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:ZJU-China/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Part_Collection"><DIV class="submenu_item">
					Part Collection
				</DIV></A></DIV><A href="https://2018.igem.org/Team:ZJU-China/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:ZJU-China/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:ZJU-China/Applied_Design"><DIV class="submenu_item">
					Applied Design
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Entrepreneurship"><DIV class="submenu_item">
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Hardware"><DIV class="submenu_item">
					Hardware
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Measurement"><DIV class="submenu_item">
					Measurement
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Plant"><DIV class="submenu_item">
					Plant
				</DIV></A><A href="https://2018.igem.org/Team:ZJU-China/Software"><DIV class="submenu_item">
					Software
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=ZJU-China"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><TITLE>Model</TITLE><DIV class="demo" style="width: 100%; padding: 0em 0; position: fixed; margin: 0; left:0; top: 0; z-index: 999; font-size: 1em !important"><DIV class="container"><DIV class="row"><DIV class="col-md-12"><NAV class="navbar navbar-default navbar-mobile bootsnav"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-menu">menu
							</BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-menu"><UL class="nav navbar-nav" data-in="fadeInDown" data-out="fadeOutUp"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/EnzymeScaffold">Enzyme Scaffold</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/LogicGate">Logic Gate</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Matrix">Matrix</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Electrodes">Electrodes</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Model">Model</A></LI><LI class="dropdown" style="min-width: 250px;"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">Silver&amp;Gold</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Public_Engagement" class="dropdown-toggle" data-toggle="dropdown">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/AllParts">All parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/BasicPart">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/CompositePart
">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Collection">Collection</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Improve">Improved Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Notebook">Lab book</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Safety" class="dropdown-toggle" data-toggle="dropdown">Safety</A></LI></DIV></NAV></DIV></DIV></DIV></DIV><DIV class="title"><DIV class="tl">MODEL  <H5>ABSTRACT  </H5></DIV></DIV><DIV class="intro"><P>After constructing hardware and biochip, we get the currents produced by the electrode. According to that we can help to judge the state of the illness. In our first model, we want to use machine learning to analyze the data. The k-nearest neighbors’ algorithm (k-NN) is a non-parametric method used for classification and regression. Our goal is to apply the sample data which contains three feature vectors and one decision vector on the sample data with unknown decision value to determine if the sample has brain damage.</P><P> In the whole system. one essential part of our wetlab experiments is to construct a multi enzyme system. So we need to ensure that this system can work normally during the whole testing procedure. That is what the second model does. we have built a mathematical model to carry out theoretical stimulation for the input and output of the enzyme system. we regard the enzyme system as logic gates and construct a kinetic model. The outputs of this theoretical model were compared with what we got from wetlab experiments to further verify out results and conclusions. </P><P> In the enzyme system, in order to find the most efficient way to construct multi-enzyme complex before the experiment, we adopted the protein de novo modeling. The modeling method used is the Robetta server. Computing method is distributed computing. After obtaining the protein spatial conformation, Z-DOCK was used for protein docking, which provides us a way to observe Tag-Catcher system' s effect on the spatial conformation of the enzyme complex. </P></DIV><DIV class="cnt_psg"><DIV class="psg"><SPAN class="psg_ttl">Cluster Analysis</SPAN><P>The <SPAN style="font-weight: bold;">k-nearest neighbors algorithm (k-NN)</SPAN> is a non-parametric method used for classification and regression.<SUP style="font-size: .5em !important;">[1]</SUP>It uses a vector space model to classify cases with the same category in which their similarity is high, so that the possible classification of unknown case cases can be evaluated by calculating the similarity with the known category cases.</P><P>k-NN is a type of instance-based learning, or lazy learning, where the function is only approximated locally and all computation is deferred until classification.</P><P>The training examples are vectors in a multidimensional feature space, each with a class label. The training phase of the algorithm consists only of storing the feature vectors and class labels of the training samples.</P><P>In the classification phase, k is a user-defined constant, and an unlabeled vector (a query or test point) is classified by assigning the label which is most frequent among the k training samples nearest to that query point.<SUP style="font-size: .5em !important;">[2]</SUP></P><SPAN class="psg_ttl psg_subtitle">Overview</SPAN><P>First, we used the 3-σ principle to detect anomalous data and make corrections. Then performed a box plot test and found that most of the data was within acceptable limits. </P><H5 style="color: black !important;">Fig. 1   Raw data.</H5><SPAN class="psg_ttl psg_subtitle">Data preprocessing</SPAN><H5 style="color: black !important;">Fig. 2   Box plot.</H5><P>The most important step after the k-NN algorithm is to standardize the data.</P><P>In our model, the min-max data standardization method is used.</P><P>The input sequence x<SUB style="font-size: .5em !important;">1</SUB>, x<SUB style="font-size: .5em !important;">2</SUB>……x<SUB style="font-size: .5em !important;">n</SUB> is transformed as follows:</P><P>The resulting new sequence y<SUB style="font-size: .5em !important;">1</SUB> y<SUB style="font-size: .5em !important;">2</SUB>……y<SUB style="font-size: .5em !important;">n</SUB> is dimensionless.</P><SPAN class="psg_ttl psg_subtitle">Data analysis</SPAN><P>Our goal is to apply the sample data which contains three feature vectors and one decision vector on the sample data with unknown decision value to determine if the sample has brain damage.</P><P>The followings are the classification effect:</P><P>From the above three figures, we can see that the indicators of non-brain damage are mostly concentrated in the lower left of the picture, while the cases of brain damage are scattered on the right side of the whole picture, although in our example, the visualization effect is not as obvious as the standard database in python, but our final accuracy has reached 70%~80%. So we have the reason to make a preliminary diagnosis based on the three indicators of lactic acid, glucose and lactate dehydrogenase.</P><SPAN class="psg_ttl psg_subtitle">Summary</SPAN><P>In the k-NN algorithm, the &quot;majority vote&quot; classification will have a defect when the category distribution is skewed. In other words, samples with more frequent frequencies will dominate the prediction of the test points because they are more likely to appear in the neighborhood of the test points and the properties of the test points are calculated from the samples in the neighborhood.</P><P>In order to overcome this defect, we first ensure that the two features of the sample are relatively uniform. Finally, the ratio of brain damage and non-brain damage in our sample is controlled at 3:5, and the number of both samples is much higher than that of k value.</P><SPAN class="psg_ttl psg_subtitle">References</SPAN><P style="vertical-align: top;display: inline-block; width: 3%;">[1]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ;">Altman, N. S. (1992). &quot;An introduction to kernel and nearest-neighbor nonparametric regression&quot;. The American Statistician. 46 (3): 175-185. doi:10.1080/00031305.1992.10475879.</P><P style="vertical-align: top; display: inline-block; width: 3%;">[2]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ; color: steelblue !important;"><A href="https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm" style:="">https://en.wikipedia.org/wiki/K-nearest_neighbors_algorithm</A></P><SPAN class="psg_ttl">Enzyme System - Logic-gate Model</SPAN><SPAN class="psg_ttl psg_subtitle">Overview</SPAN><P>One essential part of our wet lab experiments is to construct a multi-enzyme system. After applying the enzyme system to the electrodes, in order to make our data measured by the electrode accurate and believable, we need to ensure that this system can work normally during the whole testing procedure.  </P><P>So we have built a mathematical model to carry out theoretical stimulation for the input and output of the enzyme system. In the first place, under our certain circumstances, we regard the enzyme system as logic gates. Then we construct a kinetic model, during which we transformed original parameters into values between 0 and 1 to reduce the number of parameters. Finally, the outputs of this theoretical model were compared with what we got from wet lab experiments to further verify out results and conclusions.  </P><SPAN class="psg_ttl psg_subtitle">Logic-gate model</SPAN><P>According to research of Reiss, M.; Vladimir Privman et al<SPAN style="font-size: .5em !important;">[1]</SPAN>, for an enzyme system with multiple inputs, equivalence could be drawn between every enzymatic reaction and logic gates. So, to reduce the complexity of computation, an orderly combination of logic gates was used to imitate the enzyme system. The enzyme system we constructed and the its corresponding logic-gate model are shown in Fig.1 and Fig.2 respectively. </P><H5>Fig. 1   The Enzyme System.</H5><H5>Fig. 2   Logic-gate.</H5><SPAN class="psg_ttl psg_subtitle">Equation deductions</SPAN><P>After accomplishing the equivalent model, we first considered reactions without the interference brought by LDH (Fig.3).  </P><H5>Fig. 3   </H5><P>Based on the Michaelis-Menten equation, our enzymatic reactions can be described using the following equation, in which E denotes the enzyme, U and S represent substrates, C is the intermediate complex, P is the product and Ks, Ku are Michaelis constants of the enzymes. </P><P>During the reaction, in order to reduce the parameters, Ks and Ku can be omitted because lots of substrates exist and drive the reactions at the beginning.  </P><P>The proportion of enzyme the forms complex C is a constant, so we have:</P><P>Solving the differential equation and we get the following equation:</P><P>Meanwhile, according to the properties of enzymatic reactions, the beginning of the reaction is driven by lots of substrates. So, we supposed that U and S will have original value of U0 and S0 when t==0. We can then get the following equation of P when t=tq using the same methods. </P><P>Considering that our input and output should follow the principles of logic gates, we transform the input values into values between 0 and 1.</P><P>Where S0_max is the maximum value of the concentration of the input when the its logical value is 1. The U0_max and the P(tg)_max is the same.</P><P>So the above equation can be simplified into the following form:</P><P>a can be expressed by </P><P>And for y2, y3 and y4, we have:</P><P>Then after the influence of LDH began to appear. In our example, Fig.1, the third such process is biocatalyzed by enzyme HRP, which competes for the input (NADH) of the enzymes LDH. Assuming that this process competes for a fraction, F0, of the input NADH. According to the article<SUP style="font-size: .5em !important;">[2]</SUP>, we can get the complete equation for the output.</P><P>Also f and b can be denoted by:</P><P>Then we got the final relations between input and output in our logic-gate model.</P><SPAN class="psg_ttl psg_subtitle">Conclusions</SPAN><P>After establishing our theoretical model, we programed in Matlab to do the calculation. The inputs of the model are the two respective concentration values. of 0-0 and 0-1 logic input. Running the program, we got the results that are shown in Fig.4 and Fig.5. </P><P>According to the two figures, we can get when the logic value of concentration of the two inputs are 0 and 1, or 1 and 0, the value of the output varies from 0.95 to 1. when the logic value of the two inputs are 0 and 0, the value of output varies from 0.9 to 0.95. So, in the two different situations, the logic value of the output scatters in the two different ranges. Comparing the actual output values of our samples (Fig. 6) and the results of the logic-gate model, we can find in the same situation (0,1 and 0,0) the value of the output fluctuate in the range of (-1.05～-1.00) and (-1.17～-1.05). The two methods show the similar difference in the two situations. we can show that the corresponding experiments of 0-0 and 0-1 had an output value within error limit. Therefore, we can further conclude that the enzyme system worked normally on our electrodes.  </P><H5>Fig. 4 </H5><H5>Fig. 5</H5><H5>Fig. 6</H5><SPAN class="psg_ttl">References</SPAN><P style="vertical-align: top;display: inline-block; width: 3%;">[1]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ;">Reiss, M.; Heibges, A.; Metzger, J.; Hartmeier, W. Determination of BOD-Values of Starch-Containing Waste Water by a BOD-Biosensor, Biosens. Bioelectron. 1998, 13, 1083-1090.</P><P style="vertical-align: top;display: inline-block; width: 3%;">[2]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ;">Vladimir Privman. Theoretical Modeling Expressions for Networked Enzymatic Signal Processing Steps as Logic Gates Optimized by Filtering. Department of Physics, Clarkson University, Potsdam, NY 13699,2016,11.</P><SPAN class="psg_ttl">De Novo Modeling</SPAN><P>Most protein structure prediction algorithms are based on homology modeling and sequence alignment (such as Swiss-Model), which is not suitable for this experiment.<SUP style="font-size: .5em !important;">[1]</SUP>Taking the Horseradish Peroxidase (HRP) with SnoopTag (N-terminus) and SdyCatcher (C-terminus) as an example, the Swiss-Model predicted only the intermediate HRP fraction (due to the homology between this part and the HRP sequence in the library is extremely high), so the full-length structural prediction by Swiss-Model were not available. Therefore, this experiment used the full-length de novo prediction tool Robetta. </P><P>Since the amount of calculation conducted by de novo modeling was extremely large, the data was transmitted to the cloud server for distributed computing. [2] First, Ginzu analysis was performed, thus dividing the protein into several hypothetical domains. Then the algorithm models each section separately, and finally combines them together to obtain full length.</P><P>Come back to the Horseradish Peroxidase with SnoopTag and SdyCatcher. Robetta separated the enzyme into two domains (because the full length was too long and it was almost impossible to predict it directly), and the two structural units were predicted, with Confidence of 0.718500 and 0.728900 respectively. The Conf was calculated based on PFAM, MSA, and cutpref. The results are shown in Figure 1B. It can be seen that the prediction basically conformed the assumption and successfully obtained the spatial structure model. The remaining two enzymes were modeled based on similar principles.</P><H5 style="vertical-align: top;display: inline-block; width: 3%;">Fig. 1</H5><H5 style="text-align: left; width: 95% !important; display: inline-block; padding-left:2% ;">Protein modeling constructed by de novo modeling Robetta.  <I>(A)</I> Glucose Oxidase (GOx) modified by the Tag-Catcher system. The SnoopCatcher is shown as the blue part, which allows the GOx to be firmly linked to the protein with SnoopTag. <I>(B)</I> Horseradish Peroxidase (HRP) modified by the Tag-Catcher system. The SnoopTag is shown as the blue part, which can form band with SnoopCatcher to combine HRP with GOx. HRP also has a SdyCatcher (red-labeled in Fig.B) at the other end of the molecule that binds to proteins with SdyTag.  <I>(C)</I> Lactate dehydrogenase (LDH) modified by the Tag-Catcher system. The protein has a SdyTag (blue part) that binds to HRP. At the same time, it also has a red-labeled SpyCacher that allows the multi-enzyme complex to have an external interface, which could connect outside molecules through SpyTag-SpyCatcher interaction. </H5><P>After the de novo modeling of the proteins, we obtained the pdb file of the three enzymes modified by the Tag/Catcher system. Based on this, we conducted docking simulation. Protein docking is the simulation of the multi-enzyme complex’s structure in real situations. By docking the selected areas that must be combined and areas that must not be combined, the docking time can be reduced significantly. The method chosen for docking simulation is the Z-DOCK tool which performs calculations in cloud server and sends the results back to the local. <SUP style="font-size: .5em !important;">[3]</SUP> Figure 2 below is the results of the docking simulation.</P><H5 style="vertical-align: top;display: inline-block; width: 3%;">Fig. 2</H5><H5 style=";text-align: left; width: 95% !important; display: inline-block; padding-left:2% ;">Docking results of the multi-enzyme complex by Tag/Catcher ligation. From left to right are LDH, HRP and GOx. The leftmost stretched domain is SpyCatcher, which is the external interface of the multi-enzyme complex. </H5><SPAN class="psg_ttl psg_subtitle">References</SPAN><P style="vertical-align: top;display: inline-block; width: 3%;">[1]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ;">Baker, D. and A. Sali, Protein Structure Prediction and Structural Genomics. Science, 2001. 294(5540): p. 93-96.</P><P style="vertical-align: top;display: inline-block; width: 3%;">[2]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ;">David E. Kim, Dylan Chivian, David Baker; Protein structure prediction and analysis using the Robetta server, Nucleic Acids Research, Volume 32, Issue suppl_2, 1 July 2004, Pages W526-W531.</P><P style="vertical-align: top;display: inline-block; width: 3%;">[3]</P><P style="width: 95% !important; display: inline-block; padding-left:2% ;">Chen, R. , Li, L. and Weng, Z. (2003), ZDOCK: An initial-stage protein-docking algorithm. Proteins, 52: 80-87. doi:10.1002/prot.10389</P></DIV><DIV class="sponsors"><P class="sponsor_ttl">Our Sponsors</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>