<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ZJU-China_EnzymeScaffold skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ZJU-China/EnzymeScaffold</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>&lt;!DOCTYPE html&gt;<TITLE>Electrode</TITLE></P><DIV class="demo" style="width: 100%; padding: 0em 0; position: fixed; margin: 0; left:0; top: 0; z-index: 999; font-size: 1em !important"><DIV class="container"><DIV class="row"><DIV class="col-md-12"><NAV class="navbar navbar-default navbar-mobile bootsnav"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar-menu">menu
							</BUTTON></DIV><DIV class="collapse navbar-collapse" id="navbar-menu"><UL class="nav navbar-nav" data-in="fadeInDown" data-out="fadeOutUp"><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Matrix">Matrix</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/EnzymeScaffold">Enzyme Scaffold</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/LogicGate">Logic Gate</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Electrodes">Electrodes</A></LI></UL><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Applied Design</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Applied_Design">Product Design</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Model">Model</A></LI><LI class="dropdown" style="min-width: 250px;"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Human Practice</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Human_Practices" class="dropdown-toggle" data-toggle="dropdown">Silver&amp;Gold</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Public_Engagement" class="dropdown-toggle" data-toggle="dropdown">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/AllParts">All parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/BasicPart">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/CompositePart
">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Collection">Collection</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Improve">Improved Parts</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#" class="dropdown-toggle" data-toggle="dropdown">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Notebook">Lab book</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/InterLab">InterLab</A></LI><LI><A href="https://2018.igem.org/Team:ZJU-China/Safety" class="dropdown-toggle" data-toggle="dropdown">Safety</A></LI></DIV></NAV></DIV></DIV></DIV></DIV><DIV class="title"><DIV class="tl" style="font-size: 4.5em !important;">ENZYME SCAFFOLD  <H5>OVERVIEW  </H5></DIV></DIV><DIV class="intro"><P>In many real application scenarios, functional proteins sometimes provide the optimal solutions to industrial problems. While on the other hand, it is obvious to find use cases of functional proteins to be rare due to the limit control of biological systems. In our project, the well proved logic gate response to traumatic shock related injuries can only achieved with three or more metabolic interacting enzymes. Even in our HP visit to the bioinstrumentation industry, their R&amp;D department is also interested in a stable linked protein complex with a tough difficulty on a POCT(point of care test) creatinine-detecting device.</P><H5 style="text-align: center !important;">Fig. 1   Schematic representation of an industrial case and our project design.</H5><P>No matter in which case or model showed in Figure 1, there is always a problem on controlling the chaotic chemical process. </P><P style="margin-top: .3em !important;"><SPAN style="width: 5%; vertical-align: top;">•</SPAN><SPAN style="width: 92% !important; padding-left: 3%;">For this R&amp;D department, their problem lies on the quantification, since proportion of three enzymes is too dynamic and locally unstable in one solution. </SPAN></P><P style="margin-top: .3em !important;"><SPAN style="width: 7%; vertical-align: top;">•</SPAN><SPAN style="width: 90% !important; padding-left: 3%; display: inline-block;">For our design, the problem is how to keep the three enzymatic reactions in an interacting manner, which is reported to be related with average distances which means closer for a better performance.</SPAN></P><P>To mimic the well-defined <I>in vivo</I> protein machinery at a minimum, we have been seeking for a way to construct a stable, functional and designable multi-protein complex. So, a reported linking system named SpyTag-SpyCatcher system might provide us with a solution<SUP>[1]</SUP>. It has some basic properties that we desire.</P><P style="margin-top: .3em !important;"><SPAN style="width: 7%; vertical-align: top;">•</SPAN><SPAN style="width: 90% !important; padding-left: 3%;"><I>post-translational:</I>link process happens after translation and folding</SPAN></P><P style="margin-top: .3em !important;"><SPAN style="width: 7%; vertical-align: top;">•</SPAN><SPAN style="width: 90% !important; padding-left: 3%;"><I>covalent: </I>forming an isopeptide makes the construct very stable</SPAN></P><P>Obviously, a single tag-catcher system is insufficient to construct a complex of three or more proteins (with a certain order). And the use of a single system might cause an unexpected and uncontrollable polymerization. Therefore, to construct a desired multi-enzyme complex, several well-proved tag-catcher systems are needed (as reported SpyTag-SpyCatcher, SnoopTag-SnoopCatcher and SdyTag-SdyCatcher)<SUP>[2][3]</SUP>. Meanwhile, to make the whole reaction under control, these tag-catcher systems must have some robust properties. Except for the stability of the covalently linked construct, the orthogonality of different tag-catcher systems should be considered to offer additional possibilities to design a protein machinery for more complicated tasks. </P></DIV><DIV class="cnt_psg"><DIV class="psg"><SPAN class="psg_ttl">DESIGN OF THE TAG-XFP-CATCHER CONSTRUCTS</SPAN><P>The universal design of our devices is to construct on pET-26b(+). The expression plasmid includes a T7lac promotor and a unique pelB leader sequence which can direct the protein to the bacterial periplasm<SUP>[4]</SUP>. Double 6×His tag in both N- and C-terminal are used for subsequent purification. For convenience, our team has introduced GoldenGate assembly method to our lab and optimized a well-performing protocol <A href="https://2018.igem.org/Team:ZJU-China/Protocols">(details in PROTOCOLS section)</A>. One tag is designed to fuse to the N-terminal of target protein while corresponding catcher is at C-terminal. Since catchers are relatively big components in molecular weight, we divided the whole construct to two parts, tag-proteins and catchers. With GoldenGate assembly, not only can we construct 2 parts into the vector spontaneously, and also, we can easily replace one catcher or functional protein to another. To perform the GoldenGate assambly, we added 2 Bsa1 restriction site to the vector and meanwhile, mutated a needless Bsa1 site in gfp genes.</P><P>At the same time, the sequences of SpyTag-SpyCatcher, SnoopTag-SnoopCatcher and SdyTag-SdyCatcher systems are obtained by multi-sequence alignments. We used clustalW to find the most used tag-catcher amino acid sequences and applied codon optimizations for <I>E. coli</I> strains we use. ZJU-China has also contacted another team using SpyTag-SpyCatcher system, UNSW iGEM, to optimize linkers and the sequences of SpyTag-SpyCatcher system. All tag-catcher sequences can be accessed in the link here.</P><H5 style="vertical-align: top; width: 5% !important; display: inline-block;">Fig. 2</H5><H5 style="font-weight: 300 !important; text-align: left; padding-left: 2% !important; width: 93% !important; display: inline-block;"><SPAN style="font-weight: bolder !important;"> Sequence design of tag-catcher systems.</SPAN> (A) SnoopTag sequence alignment result. All raw sequence data was obtained from NCBI GenBank database. Accession number of each sequence is listed in the right. <SPAN style="font-weight: bolder;">KU608331.1</SPAN> Synthetic construct SnoopTag-beta-lactamase-SnoopCatcher gene, complete cds; <SPAN style="font-weight: bolder;">KY978851.1</SPAN> Synthetic construct Pfs28-SnoopTag gene, complete cds; <SPAN style="font-weight: bolder;">KU296975.1</SPAN> Synthetic construct SnoopTag-AffiHER2-SpyTag gene, complete cds; <SPAN style="font-weight: bolder;">KU356870.1</SPAN> Synthetic construct SnoopTag-MBP gene, complete cds. (B) SpyTag sequence alignment result. <SPAN style="font-weight: bolder;">LT986708.1</SPAN> synthetic construct glutathione s-transferase spytag fusion gene for GST-SpyTag; <SPAN style="font-weight: bolder;">KU302811.1</SPAN> Synthetic construct pENTR4-Pfs25-SpyTag sequence; <SPAN style="font-weight: bolder;">KJ906520.1</SPAN> Synthetic construct dead streptavidin SpyTag gene, complete cds.</H5><SPAN class="psg_ttl" style="font-size: 1em !important; margin-top: .4em !important;">OPTIMIZING THE EXPRESSION AND PURIFICATION PROCEDURES</SPAN><P>Since we had to express our enzymes in a large scale for printing use, the team has constructed several tag-fluorescent_protein-catcher complex to make a pilot expression, induction and purification assay. As plan, 4 kinds of complexes should be purified by Ni-NTA-Sefinose column pull down assays after an optimized 25℃ overnight IPTG induction. In all combinations, GFP-TC, RFP-TC and YFP-C are designed to imitate the enzymes in the complex. GFP-T is used for simulate the behavior of CsgA <I>in vitro</I><SUP>[5][6]</SUP>. </P><P>As originally conceived, fluorescent proteins are supposed to give fluorescent signals in native protein electrophoresis. But during the purification, unfortunately, we found a distinguishing difference in green fluorescence between GFP-T and GFP-TC. Since tags are only ~10 amino acids long and the only difference lies in catchers at C-terminal, we deduced that our linker is too short to mention the correct folding of XFPs. According to the GSG linkers UNSW iGEM team used, we lengthened our linkers from 2×GS-linkers to 6×GSG-linkers to retain the catalytic activities as possible<SUP>[7]</SUP>. In the following experiments, the relative enzyme activities were not largely influenced by the fusion tags and catchers (<A href="https://2018.igem.org/Team:ZJU-China/LogicGate">see more in LOGIC GATE</A>).</P><H5 style="vertical-align: top; width: 5% !important; display: inline-block;">Fig. 3</H5><H5 style="font-weight: 300 !important; text-align: left; padding-left: 2% !important; width: 93% !important; display: inline-block;"><SPAN style="font-weight: bolder !important;">  Expression and purification assays of 4 tag-XFP-cacther proteins.</SPAN>(A) The diagram shows the combination functional proteins and fused tags or catchers (the pairing of tags and catchers is exactly the same with final enzyme complex) with molecular weight in the right. (B) SDS-PAGE and Coomassie staining of 4 purification assays. All 4 proteins got acceptably purified as showed in 3 gel figures. (C) Comparison of GFP-T and GFP-TC in fluorescence intensity. Target protein with His tag was attached to Ni-NTA-Sefinose column (originally blue) at the bottom of EP tubes. GFP-T was green in room illumination while GFP-TC had barely no visible fluorescence.</H5><SPAN class="psg_ttl" style="font-size: 1em !important; margin-top: .4em !important;">PROOF OF THE ORTHOGONALITY OF THREE TAG-CATCHER SYSTEMS</SPAN><P>To prove the systems are capable to form a complex with a certain order, the orthogonality has to be tested. In order to make sure certain tags can and can only form covalent isopeptides with corresponding catchers, as the schematic in Figure 4A shows, all together 6 ligation reaction should be performed (the orthogonality of SnoopTag-SdyCatcher and SdyTag-SpyCatcher can be proved if the protein cannot polymerize to form polymers which is showed in Figure 3B). With molecular weight data showed in Figure 3A, an expected ligation product should be ~70KDa for GFP-T linked with another XFP or &gt;70KDa (~85KDa) for other combinations.</P><H5 style="vertical-align: top; width: 5% !important; display: inline-block;">Fig. 4</H5><H5 style="font-weight: 300 !important; text-align: left; padding-left: 2% !important; width: 93% !important; display: inline-block;"><SPAN style="font-weight: bolder !important;">Proof of the orthogonality of the three tag-catcher systems. </SPAN>(A) The schematic figure shows the experiment design. For each tag, reactions with all 3 catchers should be performed. The pairs showed in red indicates orthogonality of this kind of combinations can be proved if the purified protein cannot polymerize to form some heavier complexes in Figure 3B. (B) SDS-PAGE and Coomassie staining of ligation products. All three systems are proved orthoginal according to the figures.</H5><SPAN class="psg_ttl">References</SPAN><P style="padding-top:.4em;"><SPAN style="vertical-align:top; width: 3%; display:inline-block;">[1]</SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;"> Zakeri, Bijan, et al. &quot;Peptide tag forming a rapid covalent bond to a protein, through engineering a bacterial adhesin.&quot; Proceedings of the National Academy of Sciences 109.12 (2012): E690-E697.</SPAN></P><P style="padding-top:.4em;"><SPAN style="vertical-align:top;width: 3%; display:inline-block;">[2] </SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;"> Veggiani, Gianluca, et al. &quot;Programmable polyproteams built using twin peptide superglues.&quot; Proceedings of the National Academy of Sciences 113.5 (2016): 1202-1207.</SPAN></P><P style="padding-top:.4em;"><SPAN style="vertical-align:top; width: 3%; display:inline-block;">[3] </SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;">Tan, Lee Ling, Shawn S. Hoon, and Fong T. Wong. &quot;Kinetic controlled tag-catcher interactions for directed covalent protein assembly.&quot; PloS one 11.10 (2016): e0165074.</SPAN></P><P style="padding-top:.4em;"><SPAN style="vertical-align:top; width: 3%; display:inline-block;">[4] </SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;">Yoon, Sung H., Seong Keun Kim, and Jihyun F. Kim. &quot;Secretory production of recombinant proteins in Escherichia coli.&quot; Recent patents on biotechnology 4.1 (2010): 23-29.</SPAN></P><P style="padding-top:.4em;"><SPAN style="vertical-align:top; width: 3%; display:inline-block;">[5] </SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;">Botyanszki, Zsofia, et al. &quot;Engineered catalytic biofilms: Site‐specific enzyme immobilization onto E. coli curli nanofibers.&quot; Biotechnology and bioengineering 112.10 (2015): 2016-2024.</SPAN></P><P style="padding-top:.4em;"><SPAN style="vertical-align:top; width: 3%; display:inline-block;">[6]</SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;"> Barnhart, Michelle M., and Matthew R. Chapman. &quot;Curli biogenesis and function.&quot; Annu. Rev. Microbiol. 60 (2006): 131-147.</SPAN></P><P style="padding-top:.4em;"><SPAN style="vertical-align:top; width: 3%; display:inline-block;">[7]</SPAN><SPAN style="padding-left: 1em; width: 90%; display:inline-block;"> Reddington, Samuel C., and Mark Howarth. &quot;Secrets of a covalent interaction for biomaterials and biotechnology: SpyTag and SpyCatcher.&quot; Current opinion in chemical biology29 (2015): 94-99.</SPAN></P></DIV><DIV class="sponsors"><P class="sponsor_ttl">Our Sponsors</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>