<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ATOMS-Turkiye_Project_Ulcer2 skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ATOMS-Turkiye/Project/Ulcer2</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Tour" class="tour nav-item" id="tour">Tour</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project" class="project nav-item active" id="project">Project</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Results" class="results nav-item" id="results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices" class="pp nav-item" id="pp">P&amp;P</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/ATOMs" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras" class="extras nav-item" id="extras">Extras</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="project-sub" style="padding-right:350px"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Problems" class="sub-nav-item">Problems</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Ulcer" class="sub-nav-item">Ulcer</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Cancer_Switch" class="sub-nav-item">Cancer Switch</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Modeling" class="sub-nav-item">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Future_Plan" class="sub-nav-item">Future Plan</A></LI></UL><UL id="team-sub"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/ATOMs" class="sub-nav-item">ATOMs</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Attributions" class="sub-nav-item">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/Collaboration" class="sub-nav-item">Collaboration</A></LI></UL><UL id="extras-sub"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras/Protocols" class="sub-nav-item">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras/Achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2015&amp;group=ATOMS-Turkiye" class="sub-nav-item">Official Part Page</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/Interlab_Study" class="sub-nav-item">Interlab Study</A></LI></UL><UL id="pp-sub" style="padding-right:60px"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Virtual_Hospital" class="sub-nav-item">Virtual Hospital</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Video" class="sub-nav-item">Video</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/SynBio_Day" class="sub-nav-item">SynBio Day</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Outreach" class="sub-nav-item">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Street_Interview" class="sub-nav-item">Street interview</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Ethics" class="sub-nav-item">Ethics</A></LI></UL></DIV></DIV></DIV><DIV id="footer"><DIV id="footer-content"><H2>Team ATOMS-Turkiye</H2><H3>atoms2015@googlegroups.com</H3></DIV></DIV><DIV id="page-wrapper"><DIV class="main"><ARTICLE class="post__article"><DIV id="content-div" style="height:200px; background-image:url(https://static.igem.org/mediawiki/2015/a/ab/ATOMS-Turkiye_picture_1.jpg); padding-bottom:0px; background-size:cover"><H1 style="left:0px; right: 230px;">ULCER</H1></DIV><H2 class="scroll">Acid Resistance</H2><DIV id="content-div"><P>The main goal of this part of our project is to enable our E. Coli we used to eradicate H. Pylori to become acid resistant in order to live in gastric juice's low pH (pH:2) microenvironment. Wild-type E. coli already have couple systems to show acid resistance up to a certain pH level (pH: 5). Among those systems, the most stable-working and efficient one is known as Glutamate Dependent Acid Resistance(GDAR) system. The most important protein of Gad system is GadE, which controls the synthesis of all other proteins. By overexpressing this protein, we aim to have our bacteria resistant enough to live in gastric juice for a while.</P><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('1');">Background</A><DIV class="box"><H4>Escherchia coli Natural Acid Resistance System </H4><P>E. coli possesses four phenotypically distinct systems of acid resistance. These systems:<B>1.</B> Glutamate dependent acid resistance system (GDAR)<B>2.</B> Arginine dependent acid resistance system (ADAR)<B>3.</B> Lysine dependent acid resistance system (LDAR)<B>4.</B> Ornithine dependent acid resistance system (ODAR)</P><P>The most effective one of these systems is the glutamate dependent system so we decided to focus on glutamate dependent system (GDAR).</P><P>The GDAR system has in the acid resistance complex pathways. We decided to do the certain pathways that how to do acid resistance with our results of research.</P><P>EvgA is the most effective positive regulator of GDAR. (Efficiency: evgA&gt;ydeO&gt;gadE) But evgA takes part in countless number different genes regulation and cellular processes (2,6,7) and also most of this cellular processes are unclear. We cannot predict the results of the overexpression of evgA so we chose gadE and using by overexpression of gadE we can induce glutamate depended acid resistance system.</P><P>Briefly talk about the impact of the GadE on the mechanism.</P><P>The central activator is the LuxR-family member GadE (formerly known as YhiE). GadE binds to a 20-bp sequence called the gad box, which is located 63-bp upstream of the transcriptional start sites of gadA and gadBC. (4)GadE and GadBox are an important point for GadA and GadBC. This means that GadE and GadBox are an important for acid resistance. The basis section of acid resistance is GadE’s regulation. At least 10 different regulatory take part in this regulation.</P><P>GadE have three main activation mechanisms. The first of these is performed by evgA and ydeO. About GadE activation phase:<B>1.</B>	EvgS (sensor kinase) activate EvgA (response regulator)<B>2.</B>	YdeO and evgA are independently from each other and GadE takes an active role in transcription activation.</P><P>
The second GadE activation circuit includes CRP,RpoS and two AraC-like regulators, GadXand GadW. 
These steps are as follows:<B>1.</B>	GadX and GadW, are located downstream of gadA also GadX and GadW directly activate transcription of gadE<B>2.</B>	GadX and GadW also bind to the gadA and gadBC gad box sequences and seem to repress the gadA and gadBC promoters. (4)
GadW inhibits this GadX’s repression 
Also GadX and GadW regulate indirectly GadA/BC transcriptional functions.<B>3.</B>	The balance of power in this circuit is influenced by cAMP and CRP,which together inhibit the synthesis of RpoS.
Growth under acidic conditions reduces the concentration of cAMP in the cell.
	RpoS increases GadX’s transcriptional function
	The increase in GadX then stimulates transcription of GadE and also down regulates GadW.</P><P>The third activation of GadE contains TrmE and glucose. These steps are as follows:<B>1.</B>	The function of TrmE in the cell is not fully defined but it does have a clear effect on tRNA modification<B>2.</B>	TrmE and glucose increase independently GadE’s transcriptional function</P><P>We discourse GadE’s regulation and then now we will discourse GadE’s effect mechanisms.
GadA/BC’s effect mechanisms to acid resistance are as follows:<B>1.</B>	The external pH is normally neutral but if external pH turns to acidic pH, internal pH begins to become acidic pH with HCl diffusion.<B>2.</B>	If the external pH=2,5 and internal pH begins pH=4.2 ± 0.1. GDAR system will activate for acid resistance.<B>3.</B>	GadC is a transmembrane protein. The external pH begins to change from neutral pH to acidic pH, C-plug (is GadC’s subunit) will open. Then glutamate will take inside.</P><P><B>4.</B>	Glutamate in cells convert to GABA by GadB/C that’s glutamate decarboxylase isozymes
These steps are as follows:
*This step contains pyridoxal phosphate-containing enzymes that replace the α-carboxyl groups of their amino acid substrates with a proton that is recruited from the cytoplasm.
*HCl that diffusion from outside dissociates H+ and Cl-. This H+ is used and CO² is released by the agency of GadA/B isozymes than this is the last stage of glutamate changes to GABA. Besides this is the most important and the last step for acid resistance</P><P><B>5.</B>	If H+ leaves HCl it is staying back Cl and it will be export from the transport of chlorine channel.<B>6.</B>	The emitted CO² is taken out by diffusion.<B>7.</B>	Finally, the uncovered GABA is thrown out from the transmembrane protein GadC.</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('2');">Design</A><DIV class="box"><P>In this section, our aim is to synthesize the TlpB protein in our E.coli so that It can penetrate the stomach mucus layer, where our pathogenic bacteria, H.pylori, resides.</P><P>Tobenhance the production of our protein TIpB, we aim to assemble our TIpB gene with T7 promoter which is a fast and strong promoter whilst designing our parts.</P><P>To control the system, we decided to add a Lac Operator, where LacI protein can bind, between TIpB gene sequence and promoter. We searched for an expression vector which includes Lac Operon and found that peT45-b has the properties.</P><P>When ordering our genes we added RFC10 prefix site to 3’ end and RFC10 suffix site to the 5’ end of TlpB gene and also added BamHI and XhoI restriction site sequence on the Pet45-b for cloning.</P><P>We planned to clone TlpB gene we ordered as described above to the Pet45-b vector with using BamHI and XhoI enzymes.</P><P>When this cloning is achieved succesfully, there will be a construct which has the order T7 Promoter-Lac Operon (LacO)- HisTag-TlpB and also there are LacI constructive promoter and LacI protein sequence ,in front of promoter, on another area of Pet45-b vector . When these informations are considered, it is clear that production of TIpB protein is IPTG dependent and Western-Blot can be performed because of its His-Tag site.</P><P>As shown above,we planned to clone the ordered GadE gene into peT-45 vector by using BamHI and XhoI enzymes. Final construct after this cloning is, respectively T7 promoter-Lac operator(LacO)-HisTag-GadE. Besides, a constitutive promoter called LacI promoter and LacI protein sequence in front of that, found on peT45-b vector’s another part. It is obvious that this part is IPTG-dependent, so Western Blot can be performed easily with the help of His Tag.</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('3');">Results</A><DIV class="box"><H3>GadE-PSB1C3 CLONNING</H3><P>We cloned IDT G-Blocks GadE gene into PSB1C3 vector in order to make it ready to be submitted and have many copies of it. For this purpose we digested PSB1C3 vector and GadE G-Blocks with ECoRI and PstI restriction enzymes. Then we ligated these cut genes into the plasmid by using T4 DNA Ligase . Ocurring products were transformed into BL21 competent cell strain.</P><P>
To check if the cloning is correct, a colony PCR was perfomed with Verify Forward and Verify Reverse primers. If the cloning isn’t made properly t the band should be 314 bp long, but if colony PCR worked, then bands should be 858 bp long.</P><P>As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check for a second cloning control. We used EcorI and PstI restriction enzymes for cut-check.</P><H3>GadE-pET45-b CLONNING</H3><P>We cloned GadE into PSB1C3 vector successfully, then moved on cloning it into the expression vector pET45-b. For cloning into this plasmid, we removed our gene from PSB1C3-GadE plasmid with BamHI and XhoI enzymes. Then we ligated the cut gene with pET45-b which was cut with the same enzymes. We transformed ligated products into BL21 bacteria strain that we know it has T7 RNA polymerase.</P><P>To check if the cloning is correct, a colony PCR was performed with T7 Promoter Forward and T7 Terminator Reverse primers. If the cloning isn’t made properly the band should be 360 bp long, but if colony PCR worked, then bands should be 837 bp long.
</P><P>
As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check for a second cloning control. We used BamHI and XhoI restriction enzymes for cut-check.</P><H3>WESTERN BLOTTING</H3><P>After cloning GadE into pET45-b successfully, we did Western Blot experiment through  N-terminal located His Tag in proteins, so we managed to show the production of required proteins.</P><H3>FUNCTIONAL ASSAY</H3><P>To understand if the proteins we produced are functional, we designed and performed  a functional assay. We incubated these two types of bacteria in liquid culture for 13 hours at 37 C; pET45-GadE plasmid  containing BL21 bacteria and another BL21 bacteria which contains only pET45-b plasmid for negative control. At 13. our, we added 100 mM IPTG into these liquid cultures. By doing this, we removed the suppression on GadE protein expression. After adding IPTG, we incubated 3 hours more at 37 C.</P><P>
We prepared LB mediums with different pH values to show produced GadE proteins’ functionality. These LB mediums’ pH values are respectively 7, 5, 3,5; 2,5 and 2. We added thebacteria whichis incubated for 16 hours into LB mediums at the rate of 1:9. This means, for each pH value we added 0.5 ml liquid culture into 4.5 ml LB medium. We also added 1.5 mM Glutamat in each mix and incubated the final mix at 37 C. We made spectrophotometric measurement in 600 nm periodically for the samples that we incubated. Thus we observed how long the bacteria survives in different pH values.</P><P>
The measurement results are shown below.</P><DIV class="img-text"><P>1.5 mMGlutamate / 600 nm OD / 3h 100 mM IPTG</P></DIV><P>When the given data checked, it is obvious that the produced GadE is functional and makes E. coli survive in the acidic microenvironment in comparison to negative control results. Based on the data given, the E. coli with overexpressed GadE can survive in pH 2 gastric juice for 5-9 hours.</P></DIV></DIV><H2 class="scroll">Acid Repellency</H2><DIV id="content-div"><P>The main objective of this part of our project is to direct resistant E. Coli into the internal mucus layer where Helicobacter Pylori are abundant for a length of time. We will direct our E. Coli using the hydrogen ion difference between the stomach fluid and mucus layer. The PH of gastric juice is 2.0 whereas the pH of gastric mucus layer is 7.0. The chemoreceptor tlpb naturally found in the inner membrane of helicobacter pylori enable these bacteria to move away from acidic condition to a more neutral and basic environment. Therefore, we decided to use the TIpB protein and designed a biobrick for our E. Coli in order to ensure that our Bacteria reach the region where the helicobacter pylori are. </P><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('4');">Background</A><DIV class="box"><P>Chemotaxis, movement toward or away from chemicals, is a universal attribute of motile cells and organisms.  E. coli cells swim toward amino acids (serine and aspartic acid), sugars (maltose, ribose, galactose, glucose), dipeptides, pyrimidines and electron acceptors (oxygen, nitrate, fumarate). </P><B><P>E. coli's optimal foraging strategy</P></B><P>In isotropic chemical environments, E. coli swims in a random walk pattern produced by alternating episodes of counter-clockwise (CCW) and clockwise (CW) flagellar rotation (Fig. 3, left panel).  In an attractant or repellent gradient, the cells monitor chemoeffector concentration changes as they move about and use that information to modulate the probability of the next tumbling event (Fig. 3, right panel.  These locomotor responses extend runs that take the cells in favorable directions (toward attractants and away from repellents), resulting in net movement toward preferred environments.  Brownian motion and spontaneous tumbling episodes frequently knock the cells off course, so they must constantly assess their direction of travel with respect to the chemical gradient.</P><DIV class="img-text"><P><B>Figure 1: </B>Random and biased walks.  Left:  A random walk in isotropic environments.  When the cell's motors rotate CCW, the flagellar filaments form a trailing bundle that pushes the cell forward.  When one or more of the flagellar motors reverses to CW rotation, that filament undergoes a shape change (owing to the torque reversal) that disrupts the bundle.  Until all motors once again turn in the CCW direction, the filaments act independently to push and pull the cell in a chaotic tumbling motion.  Tumbling episodes enable the cell to try new, randomly-determined swimming directions.  Right  A biased walk In a chemoeffector gradient.  Sensory information suppresses tumbling whenever the cell happens to head in a favorable direction.  The cells cannot head directly up-gradient because they are frequently knocked off course by Brownian motion..</P></DIV><B><P>The chemotaxis signaling pathway of E.coli</P></B><P>E. coli senses chemoeffector gradients in temporal fashion by comparing current concentrations to those encountered over the past few seconds of travel.  E. coli has four transmembrane chemoreceptors, known as methyl-accepting chemotaxis proteins (MCPs), that have periplasmic ligand binding sites and conserved cytoplasmic signaling domains (Fig. 4).  MCPs record the cell's recent chemical past in the form of reversible methylation of specific glutamic acid residues in the cytoplasmic signaling domain (open and filled circles in Fig. 4).  Whenever current ligand occupancy state fails to coincide with the methylation record, the MCP initiates a motor control response and a feedback circuit that updates the methylation record to achieve sensory adaptation and cessation of the motor response. A fifth MCP-like protein, Aer, mediates aerotactic responses by monitoring redox changes in the electron transport chain.  Aer undergoes sensory adaptation through a poorly-understood, methylation-independent mechanism.
The five MCP-family receptors in E. coli utilize a common set of cytoplasmic signaling proteins to control flagellar rotation and sensory adaptation (Fig. 4).  CheW and CheA generate receptor signals; CheY and CheZ control motor responses; CheR and CheB regulate MCP methylation state.
</P><DIV class="img-text"><P><B>Figure 2: </B>Signaling components and circuit logic.  E. coli receptors employ a common set of cytoplasmic signaling proteins: CheW and CheA interact with receptor molecules to form stable ternary complexes that generate stimulus signals; CheY transmits those signals to the flagellar motors, CheZ controls their lifetime; CheR (methyltransferase) and CheB (methylesterase) regulate MCP methylation state.  Abbreviations: OM (outer membrane); PG (peptidoglycan layer of the cell wall); CM (cytoplasmic membrane)
</P></DIV><P>As in many biological signaling systems, the signaling currency in the E. coli chemotaxis pathway is reversible protein phosphorylation (Fig. 5).  However, the principal signaling chemistry is a bit different in prokaryotes and eukaryotes.  CheA is a kinase that uses ATP to autophosphorylate at a specific histidine residue.  Phospho-CheA molecules then serve as donors for autokinase reactions that transfer phosphoryl groups to specific aspartate residues in CheY and CheB.  Phospho-CheY enhances CW rotation of the flagellar motors; phospho-CheB has high MCP methylesterase activity.  The active forms of these response regulators are short-lived because they quickly lose their phosphoryl group through spontaneous self-hydrolysis. CheZ further enhances the dephosphorylation rate of phospho-CheY to ensure rapid locomotor responses to changes in the supply of signaling phosphoryl groups to CheY.</P><P>CheW couples the autophosphorylation activity of CheA molecules to chemoreceptor control.  Receptors, CheW, and CheA form stable ternary signaling complexes that modulate the influx of phosphoryl groups to the CheY and CheB proteins in response to chemoeffector stimuli.

</P><DIV class="img-text"><P><B>Figure 3: </B>Phosphorelay signaling.  The flagellar motors of E. coli spin CCW by default; the signaling pathway modulates the level of phospho-CheY, the signal for CW rotation.  Reactions and components that augment CW rotation are depicted in green; those that augment CCW rotation are depicted in red.

</P></DIV><P><B>Chemoreceptor signaling states in E.coli</B></P><P>
The signaling activities of chemoreceptors are described by a two-state model (Fig. 6).  Receptor complexes in the CW signaling state activate CheA, producing high levels of phospho-CheY.  Receptors in the CCW signaling state deactivate CheA, resulting in low levels of phospho-CheY.  Thus, the behavior of the flagellar motors reflects the relative proportion of receptor signaling complexes in the kinase-on and kinase-off conformations.  Both chemoeffector binding or release and methylation or demethylation can shift receptor signaling complexes from one state to the other.  For example, attractant ligands drive receptors toward the kinase-off state; subsequent addition of methyl groups shifts receptors toward the kinase-on state, reestablishing the steady-state (adapted) balance between the two states and restoring random walk movements.
</P><P><B>TlpB in Helicobacter Pylori: </B></P><P>H. pylori is a Gram-negative bacterium that resides in thestomachs of over half the world’s population. Its gastric habitatcontains a marked pH gradient from the highly acidic lumen,which can reach pH 2, to the more neutral environment adjacentto the epithelial lining, which is typically pH 7. 
Based on genome sequence analysis, the H. pylori chemotaxismachinery resembles that of the well-studied model, E. coli,with a few notable variations including the absence of themethylation enzymes involved in receptor adaptation (Sweeneyand Guillemin, 2011). </P><DIV class="img-text"><P><B>Figure 5: </B>Chematic representation of chemotaxis signaling in E. coli (A) and H. pylori (B). Chemoreceptors are shown in purple and flagellar motors in greenspanning the cell membrane. Cytoplasmic chemotaxis proteins CheA, CheW, CheY, CheZ, CheR, CheB, CheV, and ChePep are labeled. Protein modificationsare shown as pink circles for phosphorylation and purple hexagons for methylation. Activating interactions between signaling pathway components are indicatedby arrows, and speculative interactions are indicated by dotted lines.</P></DIV><P>When Helicobacter pylori and Escherichia coli bacteria which belong to two separate systems are examined, we recognize the vast similarity between the two. Both systems have chemoreceptors which belong to the MCP family enable the detection of chemical substances which do chemotaxis. Also, both bacteria also share in common the proteins required to send signals to necessary regions.</P><P>H. pylori has four chemoreceptors, TlpA,TlpB, TlpC, and TlpD, and several polar flagella that dictatesmooth swimming behavior in the presence of an attractantand increased stopping behavior in the presence of a repellent(Lowenthal et al., 2009; Rader et al., 2011). Only a small numberof chemotactic signals have been identified for H. pylori. Thebest-haracterized chemoreceptor is TlpB, which is requiredfor chemorepulsive responses to acid, as well as the quorumsensingmolecule autoinducer-2 (AI-2) and H+. 
TlpB is the first bacterial chemoreceptor of known function shown by crystallography to contain an extracellular PAS domain. Strikingly, a molecule of urea is found within the canonical ligand-binding site of the PAS domain, bound in a manner predicted to be sensitive to changes in pH.</P><DIV class="img-text"><P><B>Figure 6: </B>TlpB Forms a Dimer that ContainsUrea-Binding PAS Domains(A) Schematic of estimated TlpB domains is illustrated.Transmembrane region (TM, orange);PAS domain (light gray); HAMP domain (green);chemoreceptor trimer of dimers contact region(blue). The periplasmic domain is from aminoacids 32–209.(B) Ribbon diagram of TlpBpp homodimer andgray/blue/red/white urea molecules, with chaincolor gradation ranging from N terminus (blue) to Cterminus (green) for the monomer on the right andN terminus (light green) to C terminus (red) for themonomer on the left is presented. For orientationthe bacterial inner membrane would be below thelower part of the protein model diagram.(C) Diagram of the TlpBpp urea-binding siteincluding hydrogen bonds (dashed lines) betweenurea (gray, white, blue, and red) and the surroundingresidues and water molecule is demonstrated.Oxygen atoms are shown as red spheres,nitrogen as blue, and hydrogen as white..</P></DIV><P>Emily Goers Sweeney et.al. report that urea is bound with extremely high affinity andspecificity and is essential for the thermodynamic stability ofthe TlpB molecule. The urea-binding site includes an aspartategroup (Asp114), which we propose to be the key titratableresidue responsible for pH sensing. Mutational and biophysicalanalyses of the urea-binding site support a mechanism in whichthe urea cofactor, by binding in a pH-sensitive fashion, stabilizesthe secondary structure of TlpB and signals a pH response.
</P><DIV class="img-text"><P><B>Figure 7: Model for how TlpB Senses Acid:  </B>In low pH conditions (shown on the left), TlpB’s periplasmic domain is ina ‘‘relaxed’’ or expanded state due to decreased hydrogen bonding to ureaand consequent lowered urea-binding affinity. However, in high or moreneutral pH conditions, TlpB’s periplasmic domain is in a ‘‘tense’’ or condensedstate with increased urea-binding affinity. The state of the periplasmic domainis relayed through the transmembrane region, which affects CheA’s phosphorylationstate, ultimately affecting the flagellar motor and dictating stoppingbehavior. TlpB dimer is shown in orange, CheA in red, CheW in green, urea inpurple, protons in blue, and phosphate in black..</P></DIV><P>By sequence analysis, H. pylori TlpB is organized like a typical member of the MCP superfamily of transmembrane receptors with two transmembrane helices (tm1, tm2) per subunit bracketing an extracellularsensing domain[2,7]. The extracellular-sensing domain is responsible for detecting ligands directly or indirectly via interactions with periplasmic-binding proteins. Continuing from tm2, the C-terminal portion of the MCP is cytoplasmic and mostly helical. It contains a histidine kinase, adenylyl cyclase, methyl-binding protein, phosphatase (HAMP) domain, followed by a helical domain and a segment that binds to the CheA/CheW histidine autoki-nase complex[2,8]. Phosphorylation of CheA in response to an extracellular signal in turn controls downstream compo-nents that modulate activity of the flagellar motor. MCPs dimerize at the membrane, forming a four helix bundle with ligand-binding domains in the periplasm, whereas trimers of dimers assemble with CheA and CheW to form a high-perfor-mance signaling array located in the cytoplasm[2,9]. 
</P><P>In low pH conditions , TlpB’s periplasmic domain is in a ‘‘relaxed’’ or expanded state due to decreased hydrogen bonding to urea and consequent lowered urea-binding affinity. However, in high or more neutral pH conditions, TlpB’s periplasmic domain is in a ‘‘tense’’ or condensed state with increased urea-binding affinity. The state of the periplasmic domain is relayed through the transmembrane region, which affects CheA’s phosphorylation state, ultimately affecting the flagellar motor anddictating stopping behavior
</P><P>References</P><P><B>[1]</B>Sachs, G., Weeks, D.L., Melchers, K., and Scott, D.R. (2003). The gastric biology of Helicobacter pylori. Annu. Rev. Physiol. 65, 349–369.<B>[2]</B>Structure and Proposed Mechanism for the pH-Sensing Helicobacter pylori Chemoreceptor TlpB Emily Goers Sweeney,J. Nathan Henderson, John Goers, Christopher Wreden, Kevin G. Hicks, Jeneva K. Foster,Raghuveer Parthasarathy, S. James Remington,and Karen Guillemin <B>[3]</B>Sweeney, E.G., and Guillemin, K. (2011). A gastric pathogen moves chemo-taxis in a new direction. MBio 2, e00201-11.<B>[4]</B>Lowenthal, A.C., Simon, C., Fair, A.S., Mehmood, K., Terry, K., Anastasia, S., and Ottemann, K.M. (2009). A fixed-time diffusion analysis method determines that the three cheV genes of Helicobacter pylori differentially affect motility.Microbiology 155, 1181–1191. <B>[5]</B>Croxen, M.A., Sisson, G., Melano, R., and Hoffman, P.S. (2006). The Helicobacter pylori chemotaxis receptor TlpB (HP0103) is required for pH taxis and for colonization of the gastric mucosa. J. Bacteriol. 188, 2656–2665. <B>[6]</B>Hazelbauer, G.L., Falke, J.J., and Parkinson, J.S. (2008). Bacterial chemore-ceptors: high-performance signaling in networked arrays. Trends Biochem. Sci. 33, 9–19. <B>[7]</B>Wuichet, K., Alexander, R.P., and Zhulin, I.B. (2007). Comparative genomic and protein sequence analyses of a complex system controlling bacterial chemotaxis. Methods Enzymol. 422, 1–31. <B>[8]</B>Marchler-Bauer, A., Lu, S., Anderson, J.B., Chitsaz, F., Derbyshire, M.K., DeWeese-Scott, C., Fong, J.H., Geer, L.Y., Geer, R.C., Gonzales, N.R., et al. (2011). CDD: a Conserved Domain Database for the functional annotation of proteins. Nucleic Acids Res. 39 (Database issue), D225–D229. <B>[9]</B>Briegel, A., Ortega, D.R., Tocheva, E.I., Wuichet, K., Li, Z., Chen, S., Mu¨ller, A., Iancu, C.V., Murphy, G.E., Dobro, M.J., et al. (2009). Universal architecture of bacterial chemoreceptor arrays. Proc. Natl. Acad. Sci. USA 106, 17181– 17186. <B>[10]</B>http://chemotaxis.biology.utah.edu/Parkinson_Lab/projects/ecolichemotaxis/ecolichemotaxis.html</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('5');">Design</A><DIV class="box"><P>In this section, our aim is synthesize the TlpB protein in our E.coli for to E.coli can penetrate to the stomach mucus layer, which is our pathogenic bacteria, H.pylori, resides.</P><P>For enhance the production of our protein TIpB, we aim to assemble our TIpB gene with T7 promoter which is a fast and strong promoter during we design our parts.</P><P>For controlling the system, we decided to add a Lac Operator, which is LacI protein can bind, between TIpB gene sequence and promoter. We searched for a expression vector that include Lac Operon and we found that peT45-b has the properties we want and we decided to use that vector.</P><P>When we order our genes we added RFC10 prefix site to 3’ end and RFC10 suffix site to the 5’ end of TlpB gene and also we added BamHI and XhoI restriction site sequence on the Pet45-b for cloning.</P><P>We planed to cloning TlpB gene that we ordered as described above to the Pet45-b vector with using BamHI and XhoI enzymes.</P><P>When this cloning achieved succesfully, there will be a construct that with order T7 Promoter-Lac Operon (LacO)- HisTag-TlpB and also there are LacI constructive promoter and LacI protein sequence ,in front of promoter, on the another area of Pet45-b vector . When these informations considered, it is clear that production of TIpB protein is IPTG depented and Western-Blot can be performed because of His-Tag site.</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('6');">Results</A><DIV class="box"><P><B>PSB1C3-TlpB CLONING</B></P><P>First of all we planned to cloning our gene that we ordered from IDT as a G-blocks to the PSB1C3 plasmid for multiply our genes and also for submitting to the iGEM as a biobrick in PSB1C3, for this purpose we cut both our PSB1C3 vector and G-Block of TIpB with EcoRI and PstI restriction enzymes. Then we inserted these G-Blocks to the plasmid with used T4 DNA ligase enzyme and we transformed our plasmid to the BL21 competent bacteria.</P><P>For verifying our cloning process we did colony PCR with Verify Forward and Verify Reverse primers. If we got the cloning results wrongly we should see 314 bp band but if we achieved the cloning we should see  band.</P><P>After Colony PCR, for the clonings that we thought positive we put our bacteria to the 16 hours liquid culture and we isolated plasmid DNA with miniprep isolation method. We cutchecked our plasmid DNAs that we got from our isolation for controlling of cloning second time with using EcoRI restriction enzyme.</P><P><B>pET45-TlpB CLONING</B></P><P>After we cloned our genes to PSB1C3 plasmid successfully then we planned to cloning our genes to an expression vector pET45-b for high amount of protein synthesizing. For cloning our gene to this vector, we extracted our genes from PSB1C3-TIpB plasmid with using BamHI and XhoI enzymes. We performed ligation with pET45-b vector which is cutted with the same enzymes, then we transformed our plasmid to the BL21 competent bacteria which has a T7 RNA polymerase.</P><P>For verifying our cloning process we did colony PCR with T7 Promoter Forward and T7 Terminator Reverse primers. If we got the cloning results wrongly we should see 360 bp band but if we achieved the cloning we should see 2007 band.</P><P>After Colony PCR, for the clonings that we thought positive we put our bacteria to the 16 hours liquid culture and we isolated plasmid DNA with miniprep isolation method. We cutchecked our plasmid DNAs that we got from our isolation for controlling of cloning second time with using BamHI and XhoI restriction enzymes.</P><P><B>WESTERN BLOTTING</B></P><P>After we cloned our genes to pET-45b vector successfully then we did Western-Bloting for showing our proteins are synthesized with using the His-tag on the N-terminal of our proteins</P><P><B>SOFT AGAR PLUG ASSAY</B></P><P>For understanding that our proteins that produced are functional or not, we performed a chemotaxis motility assay which is called Agar Plug Assay. 
</P><P>We chose 2 different points on the soft agars that have 2 cm distance between them.We put the filter paper that is immersed in specially prepared liquid culture on the first point and then we put another filter paper that immersed in 0.2 mM HCL solution on the second point. Then we incubated our bacteria on 30C and examined their growing process on plates
</P><P>The results are below</P></DIV></DIV><H2 class="scroll">Increased Motility</H2><DIV id="content-div"><P>In this section, the basic goal of the parts that we are using is to ensure that E. Coli can move from the acidic pH of the stomach fluid into the basic pH of the thick mucus layer and be able to move around within this area with ease. In this manner, the bacteria will have reached the region of the stomach where Helicobacter pylori is found. . Unlike normal fluids, the mucus layer is very thick and the flagella of natural E. Coli does not have sufficient torque or speed to allow it to enter.. In this project, we aim to give E. Coli the ability to enter the stomach’s mucus layer by increasing both of these characteristics. In order to achieve this goal, we plan to use 2 different strategies. . First, we will use a knock-out mutant form (HNS T108I) of the NHS transcription factor found in E. Coli.  This mutant form has been shown to have a positive effect on the speed and torque of the flagella. The difference between the normal NHS and mutant NHS is a single amino acid change from threonine to isoleucine at residue 108 of the protein.</P><DIV class="img-text"><P><B>Figure 1: </B>Overall look of flagellar proteins n H. Pylori</P></DIV><P>As for the second strategy to reach this goal, we aim to increase the torque power and flagellar speed by replacing the MotA and MotB proteins which make up E. Coli’s rotor (they form a H+ channel) with proteins PotB59 and PomA from V. Alginolyticus(they form a Na+ channel).</P><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('7');">Background</A><DIV class="box"><P><B>HNS/HNST108I</B></P><P>Many species of bacteria have flagellar motility that is achieved by rotating surface-exposed organelles The rotation of the flagella is controlled by a motor complex embedded in the inner membrane (reviewed in Refs. 2 and 3 The motor region of E. Coli’s flagella is composed of several parts. The stator complex, made up of the proteins MotA-MotB, has a transmembrane characteristic and ensures the generation of a proton channel (4-6).  The rotor complex involves the interaction of three other proteins, FliG, FliM, and FliN(7,8). All three rotor proteins are involved in the processes of flagellar assembly, switching, and rotation (9,10). However, FliG is predominately involved in torque generation (8, 10, 11), whereas FliM mainly functions in switching rotor direction (12). The precise role of FliN is the least well defined, but it may participate in flagella protein-specific export and assembly (13,14)</P><P>HNS is a transcription factor that plays a role in the regulation of gene expression in E. Coli. It plays an activator role at the level of protein synthesis for many genes involved in bacterial motility.  . . In many studies it was shown that HNS is essential for the synthesis of proteins that make up the rotor and stator. The flhCD operon is a positive regulator of HNS which is responsible for the synthesis of many proteins that composes the structure of the bacterial flagella. Studies show that knock-out of the gene that codes the HNS protein results in non-motile E. Coli.</P><P>Above, we mentioned three proteins that make up the stator portion of the bacterial flagella.. These proteins are FliG, FliM and FliN and these three proteins have different activities. FliM is a regulator protein that is responsible for determining in which direction the bacterial flagella will be oriented. Although the role of FliN is not precisely known, it is thought that it acts as a guide for the assembly of the flagella’s membrane/for attachment of flagella to the membrane. As for FliG, this protein is responsible for the rotation speed and torque power of the bacterial flagella. For our aims in this part of the project, which are to increase the speed and power of the bacterial flagella, we decided to focus on FliG.
</P><P>In a study by Gina M. Donato and Thomas H. Kawula, it was shown that HNS is not only a transcription factor but also that it can change some protein’s activities through direct interactions with them. . In studies including fluorescence anistropy and chemical cross-linking, it was shown that there is a physical interaction between HNS and FliG proteins.</P><DIV class="img-text"><P><B>Figure 2: Fluorescence anisotropy of H-NS-FliG interactions. </B>Increasing amounts of FliG were added to fluorescein-labeled H-NS. Ten anisotropy values were measured at each FliG concentration and averaged. Graph is representative of two separate experiments.
.</P></DIV><DIV class="img-text"><P><B>Figure 3: H-NS-FliG cross-linked complexes.  </B>Reactions were incubated at room temperature and equal amounts were electrophoresed on denaturing poly acrylamide gels. A, Coomassie-stained 4–20%SDSpolyacrylamidegradient gel; B, 12% SDS-polyacrylamide gel transferred to nitrocellulose, and probed with H-NS antiserum; C, second half of gel in B probed with FliG antiserum. Protein. Standard sizes are indicated by lines; lmw, low molecular weight markers; hmw, high molecular weight markers; protein monomers, dimers, and H-NS-FliG complexes are indicated by arrows. Reactions for all panels: 1, wild-type H-NS only; 2, H-NST108I only; 3, FliG only; 4, wild-type H-NS with cross-linker; 5, H-NST108I with cross-linker; 6, FliG with cross-linker;7, 1:1 molar ratio of wild-type H-NS to FliG with cross-linker; 8, 1:1molar ratio of H-NST108I to FliG with cross-linker.
</P></DIV><P>Donato and Kawulainvestigated the effect on bacterial motility of random point mutations in the HNS protein. . As a result of their Swarm plate assay and experiments for the determination of flagellar rotational speed, it was found that some random mutations (HSN-T108I and HNS-A18E) resulted in bacteria with a stronger and faster flagella as compared with wild type bacteria. This single amino acid change caused a 50% increase in the binding of HNS to FliG.. It was also shown that this single amino acid change in HNS resulted in an approximately 50% increase in flagellar rotation speed and about a 2-fold increase in the bacteria’s swarm size. </P><DIV class="img-text"><P><B>Figure 4: Swarm plate assay.  </B>Fresh colonies from strains carrying the indicated alleles were inoculated onto semi-solid agar plates and grown at 30 °C. Growth was measured as the diameter of the bacterial swarm over several time points. Bacteria with swarm diameters under 10 mm at the end of 17 h incubation were considered non-motile. Data representative of three individual experiments. *, vector; f, hns2-tetR; ,,wild-type HNS; l, hnsT108I; l, hnsA18E..</P></DIV><DIV class="img-text"><P><B>Figure 5: </B> Flagella propel bacteria by rotating motor-driven helical filaments (35, 36) whereby swimming speed is directly related to flagellar rotational speed (39). Gina M. Donato and Thomas H. Kawula1 , in another experiment which they tried to compare rotational speeds of mutant and wild-type bacteria, they concluded that hnsA18E and hnsT108I accelerated flagellar speeds 44–62% over wild-type levels.</P></DIV><P>In the results from fluorescence anisotropy and chemical cross-linking, Donato and Kawula showed that there is an interaction between HNS and FliG, the protein responsible for flagellar torque; and they show that the mutant form of HNS binds FliG 50% more often than does wild-type HNS: They explain the increase in the bacteria’s swarm rate and swim speed caused by mutant HNS thusly “We position H-NS at the interface between the rotor and stator, directly linked to the C terminus of FliG(Fig. 5 A). Tighter binding of mutant H-NST108I toFliG (Fig.5 B) may cause increases in flagellar speed by altering the conformation of FliG relative to the other rotor proteins and/or the MotA·B complex, thus, compacting the motor complex and allowing fast errotation by creating less friction within the surrounding stationary MotA·B ring complex (56).</P><DIV class="img-text"><P>This study having captured our attention, we decided to use mutant HNS to increase flagellar speed and torque power. </P></DIV><P><B>PotB59/PomA</B></P><P>The flagella of bacteria is made up of 30 different proteins and at least three different parts. These parts are the basal body, hook and filament. . The basal body is made up of the rotor and stator and is the region responsible for the rotation of the flagella. The stator is the part made up of channel proteins that generate the necessary proton motive force. The rotor is a stable region of the basal body which controls things such as the speed and direction of the flagella and is found in the inner region of the stator. The interaction between the rotor and stator, found in the basal body, result in the movement of the flagella. The filament is the long string-like structure that reaches outwards. . The role of the hook is to connect the filament and the basal body together.</P><DIV class="img-text"><P><B>Figure 1. </B>Schematic diagram of the bacterial flagellum. The flagellum is a locomotive organelle for bacterial propulsion. The flagellum consists of the basal body, which acts as a reversible rotary motor, the hook, which functions as a universal joint and the filament, which works as a helical screw. OM, outer membrane; PG, peptidoglycan layer; CM, cytoplasmic membrane.
.</P></DIV><P>The driving force of the bacterial flagellar rotation is an ion concentration gradient between the two sides of the membrane. The energy needed to rotate the flagella is generated by the majority of bacteria as a H or Na ion concentration gradient. In proton(H)-driven motors, the rotation of the flagella is carried out by the MotA and MotB membrane proteins. These proteins are actually H+ channels and when H+ moves through these channels depending on the concentration gradient, the necessary energy for flagellal rotation is created. This system that uses H+ is used by E. Coli and many other bacterial strains to generate energy for flagellar rotation. Other bacterial strains rely on Na+ to power their flagellar motor.  These include Vibrio spp. (V. alginolyticus, V. parahaemolyticus ve V. cholerae) and alkaphilic Bacillus species.  These bacteria, in place of MotA and MotB proteins, use homologous membrane proteins PomA and PomB. These proteins function as Na+ channels and ensure that the necessary energy for flagellar rotation is generated as Na+ passes along the channel. Furthermore, in bacteria that use Na+ channels to power their flagella, there are MotX and MotY proteins which are localized in the outer membrane These two proteins stabilize the binding of PomA/B proteins to the motor Flagella that work by Na+ have greater speed and rotational power that those that depend on H+.</P><DIV class="img-text"><P><B>Figure 2. </B>Schematic diagram of the proton-driven bacterial flagellar motor. The flagellar motor consists of a reversible rotor made of FliF, FliG, FliM and FliN and a dozen stators, each of which consists of MotA and MotB. FliF forms the MS ring within the cytoplasmic membrane. FliG, FliM and FliN form the C ring on the cytoplasmic face of the MS ring. OM, outer membrane; PG, peptidoglycan layer; CM, cytoplasmic membrane.
</P></DIV><P>Both MotA and PomA proteins have 4 membrane-spanning region, a huge cytoplasmic fold and a C-terminal end. Yet MotB and PomB has only one membrane-spanning region.</P><DIV class="img-text"><P><B>Figure 3. </B>Arrangement of transmembrane segments of the MotA/B complex, which consists of four copies of MotA and two copies of MotB. The view is from the periplasmic side of the membrane. The complex has two proton conducting pathway shown by orange ellipsoids. </P></DIV><P>In work by M. Homma et al, a host of chimeric proteins were generated by combining different parts of the stator proteins (MotB and PomB) from the two systems mentioned above. .  Many experiments were done to show the activity of the chimeric stator proteins and to identify the most effective of these.  According to their data, the most effective protein was (PotB59)(PomB7E) which was generated by combining the N-terminal tail of PomB with the periplasmic C-terminal tail of MotB. . In order to understand how effective this protein worked, many experiments were conducted with this chimeric protein in combination with MotA and PomA.  According to results from these experiments, the most effective rotor system was PomA/PotB59. This stator system Works with Na When this two protein system was established in E. Coli, it showed increased flagellar rotation speed and strength as compared to the original MotA/MotB stator system.</P><DIV class="img-text"><P><B>Figure 4. </B>Amino acid alignments of V. alginolyticusPomB (Va), R. sphaeroides MotB (Rs), and E. coli MotB (Ec), and location of hypermotile mutations and junction sites of chimeric proteins. White letters in black boxes and arrows show identical residues and junction sites of chimeric proteins, respectively. Gray and hatched bars indicate putative transmembrane (TM) regions and peptidoglycan binding motif (PGB), respectively. Hypermotile mutations and strain numbers are indicated under the alignment.
</P></DIV><DIV class="img-text"><P><B>Figure 5. </B>Diagram of the properties of each chimeric B subunit. White, gray and hatched bars show the contributions of PomB or MotB from V. alginolyticus, R. sphaeroides and E. coli, respectively. TM, transmembrane region; PGB, peptidoglycan-binding motif. The stator proteins were expressed in V. alginolyticus or E. coli. R. sphaeroides MotA/MotB do not function in E. coli or V. alginolyticus, and PomA/PomB function only in V. alginolyticus. MotBE and PotB7E function in E. coli and V. alginolyticus. The properties of motors containing either protein are essentially the same in both species. Stators containing MomB7E or MomB7R function only in V. alginolyticus.</P></DIV><P>It is possible that the PomA/PotB59 system could be effective without the PotX and PotY proteins.</P><P>In light of the above information, we decided to use the PomA/PotB59 system in addition to the mutant HNS protein to increase the bacteria’s flagellar speed and torque strength.</P><P>References</P><P><B>[1]</B>Gina M. Donatoand Thomas H. Kawula(1998)TheJournalBiologicalChemistryVol. 273, No. 37, 24030-24036<B>[2]</B>Macnab, R. M. (1992) Annu. Rev. Genet. 26, 131–158
<B>[3]</B>Macnab, R. M. (1996) EscherichiacoliandSalmonellatyphimurium Cellular
andMolecularBiology (Neidhardt, F. C.,Curtiss, R., III, Ingraham, J. L.,
Lin, E. C. C.,Low, K. B., Magasanik, B., Reznikoff, W. S., Riley, M.,
Schaechter, M.,andUmbarger, H. E., eds) 2nd Ed., pp. 123–145, American
SocietyforMicrobiology, Washington, D. C.
<B>[4]</B>Blair, D. F.,andBerg, H. C. (1990) Cell 60, 439–449
 <B>[5]</B>Blair, D. F.,andBerg, H. C. (1991) J. Mol. Biol. 221, 1433–1442<B>[6]</B>Garza, A. G., Harris-Haller, L. W., Stoebner, R. A., andManson, M. D. (1995)
Proc. Natl. Acad. Sci. U. S. A. 92, 1970–1974
<B>[7]</B>Marykwas, D. L.,Schmidt, S. A., andBerg, H. C. (1996) J. Mol. Biol. 256,564–576<B>[8]</B>Tang, H.,Braun, T. F., andBlair, D. F. (1996) J. Mol. Biol. 261, 209–221 <B>[9]</B>Yamaguchi, S.,Fujita, H., Ishihara, A., Aizawa, S.-I., andMacnab, R. M.
(1986) J. Bacteriol. 166, 187–193
 <B>[10]</B>Irikura, V. M.,Kihara, M., Yamaguchi, S., Sockett, H., andMacnab, R. M.
(1993) J. Bacteriol. 175, 802–810
<B>[11]</B>Lloyd, S. A.,Tang, H., Wang, X., Billings, S., andBlair, D. F. (1996)
J. Bacteriol. 178, 223–231
<B>[12]</B>Welch, M.,Oosawa, K., Aizawa, S.-I., andEisenbach, M. (1993) Proc. Natl.
Acad. Sci. U. S. A. 90, 8787–8791

<B>[13]</B>Vogler, A. P.,Homma, M., Irikura, V. M., andMacnab, R. M. (1991)
J. Bacteriol. 173, 3564–3572

<B>[14]</B>Tang, H.,Billings, S., Wang, X., Sharp, L., andBlair, D. F. (1995) J. Bacteriol.
177, 3496–3503
<B>[15]</B>Asai, Y., T. Yakushi, I. Kawagishi, and M. Homma. 2003. Ion-couplingdeterminants of Na + -drivenand H + -drivenflagellarmotors. J. Mol. Biol. 327: 453-

</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('8');">Design</A><DIV class="box"><P><B>HNS/HNST108I</B></P><P>We aimed to compare bacteria motility between two kinds of HNS knock-out E. Coli; first one expresses HNS protein and the other one expresses HNST108I mutant gene. In order to produce these proteins in high amounts, we chose to combine GadE sequence with T7 promoter, a promoter which has a high rate of transcription. In addition to this, to have a controllable system, LacI protein attachable Lac operator was added between these two gene’s sequences and T7 promoter sequence. We searched to find a vector which can lay down all those conditions and found out that peT-45 expression vector is useful. When we ordered our genes, we put RFC10 prefix site on 3’ end of HNS and HNST108I and also BamHI restriction enzyme recognition site for cloning them into peT-45. We added RFC10 suffix site on 5’ end of the same gene sequences and also XhoI restriction enzyme recognition site for cloning to peT-45 vector, again.</P><P>As shown above,we planned to clone the ordered HNS and HNST108I genes into peT-45 vector by using BamHI and XhoI enzymes. Final constructs after this cloning are; respectively T7 promoter-Lac operator(LacO)-HisTag-HNS  and T7 promoter-Lac operator(LacO)-HisTag-HNST108I. Besides, a constitutive promoter called LacI promoter and LacI protein sequence in front of that, found on peT45-b vector’s another part. It is obvious that this part is IPTG-dependent, so Western Blot can be performed easily with the help of His Tag.
</P><P>
As the mutant version of HNS shows us that change of even a single amino acid can be effecting protein expression, we considered His-tag(6 amino acid long) could effect their expression too. So we planned to clone them into PSB1C3-T7 plasmid, which we designed. By doing this, we will have the proteins synthesized in high amounts with the help of T7 but there won’t be any additional amino acids. We expect to have better results from our experiments. 
</P><P><B>PotB59/PomA</B></P><P>
We aimed to increase motility and torque force of E. Coli flagella when we knock-out its natural rotator system proteins MotA and MotB and replace them with PotB59/PomA systems. As we needed these two proteins together, we combined them on the same biobrick when we ordered the gene sequences. In gene designing, we considered producing onlyone RNA for these two genes but having two different proteins when this RNA translated. In addition to this, to have a controllable system, LacI protein attachable Lac operator was added between this sequence and T7promoter sequence. We searched to find a vector which can lay down all those conditions and found out that peT-45 expression vector is useful. When we ordered our genes, we put RFC10 prefix site on 3’ end of PotB59/PomAgene sequence and also BamHI restriction enzyme recognition site for cloning them into peT-45. We added RFC10 suffix site on 5’ end of the same gene sequences and also XhoI restriction enzyme recognition site for cloning to peT-45 vector, again.
</P><P>
As shown above,we planned to clone the ordered PotB59/PomA gene sequence into peT-45 vector by using BamHI and XhoI enzymes. Final construct after this cloning is; respectively T7 promoter-Lac operator(LacO)-HisTag-PotB59-RBS-PomA-S-tag. Besides, a constitutive promoter called LacI promoter and LacI protein sequence in front of that, found on peT45-b vector’s another part. It is obvious that this part is IPTG-dependent, so Western Blot can be performed easily with the help of His Tag to observe PotB59 expression and S-tag to observe PomA expression.
</P><P>
In this part’s design we confronted with the same problem as we did in Hns part design . So we planned to clone them into PSB1C3-T7 plasmid, which we designed. By doing this, we will have the proteins synthesized in high amounts with the help of T7 but there won’t be any additional amino acids.
</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('9');">Results</A><DIV class="box"><P><B>PSB1C3-HNS/HNS-T108I/PotB59-PomA CLONING</B></P><P>We cloned IDT G-Block genes into PSB1C3 vector in order to make it ready to be submitted and have many copies of it. For this purpose we digested PSB1C3 vector and HNS,HNS_T108I, PotB59/PomA G-Blocks with ECoRI and PstI restriction enzymes. Then we ligated these cut genes into the plasmid by using T4 DNA Ligase . Ocurring products were transformed into BL21 competent cell strain.
To check if the cloning is correct, a colony PCR was perfomed with Verify Forward and Verify Reverse primers. If the cloning isn’t made properly t the band should be 314 bp long, but if colony PCR worked, then bands should be at 744 bp long line for HNS and HNST108I genes, and 2045 bp long line for PotB59/PomA gene.
As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check for a second cloning control. We used EcorI and PstI restriction enzymes for cut-check.
</P><P><B>RESULT FOR HNS</B></P><P><B> RESULT FOR HNS-T108I</B></P><P><B>RESULT FOR PotB59/PomA</B></P><P><B>pET45-HNS/HNS-T108I/PotB59-PomA CLONING</B></P><P>We cloned our genes into PSB1C3 vector successfully, then moved on cloning it into the expression vector pET45-b. For cloning into this plasmid, we removed our genes from PSB1C3-HNS, PSB1C3-HNS-T108I, PSB1C3-PotB59/PomA plasmids with BamHI and XhoI enzymes. Then we ligated the cut gene with pET45-b which was cut with the same enzymes. We transformed ligated products into BL21 bacteria strain that we know it has T7 RNA polymerase.</P><P>To check if the cloning is correct, a colony PCR was performed with T7 Promoter Forward and T7 Terminator Reverse primers. If the cloning isn’t made properly the band should be 360 bp long, but if colony PCR worked, then bands are expected to be at 744 bp line for HNS and HNS-T108I, 2045 bp line for PotB59/PomA.
</P><P>As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check for a second cloning control. We used BamHI and XhoI restriction enzymes for cut-check.</P><P><B>RESULTS FOR HNS</B></P><P><B>RESULT FOR HNS-T108I</B></P><P><B>RESULTS FOR PotB59/PomA</B></P><P><B>WESTERN BLOTTING</B></P><P>After cloning the genes into pET45-b successfully, we did Western Blot experiment through N-terminal located His Tag in proteins, so we managed to show the production of required proteins.(As we didn’t have any S-tag antibodies for Western blotting, the bands shown for PotB59/PomA indicate only the presence of PotB59 protein.)
</P><P><B>RESULT FOR HNS/HNS-T108I</B></P><P><B>RESULT FOR PotB59/PomA</B></P><P><B>SOFT AGAR PLUG ASSAY</B></P><P>After showing that we produced required proteins successfully, in order to learn if they are functional, we did a motility assay called ‘plug assay’. We inoculated bacteria in different concentrations into the soft agars and incubated them at 37°C.  After this, we measured their colony diameters at 24. and 48. hours. The results are shown below.
</P><B>(-) kontrol:</B>pET45-b plasmid transformed bacteria
<B>HNS:</B>pET45-b-HNS plasmid transformed bacteria
<B>HNS-T108I</B>pET45-b-HNS-T108I plasmid transformed bacteria
<B>PotB59/PomA :</B>pET45-b-PotB59/PomA plasmid transformed bacteria

<P>
The measurements of 0.25 and 0.4 concentrations in first 24. hour period, gives us significant positive results . If we compare the results,colony diameters are ordered like this; 
<B>  (-) kontrol &lt; Wild type HNS &lt; PotB59/PomA &lt; HNS-T108I --&gt; </B> . 
The results of 0.30 and 0.35 agar concentrations doesn’t give significant information. Besides, in 48. hour of incubation the diameter of colonies reached to plate’s diameter, so measurement results didn’t show any difference than 24. hour’s.
</P><P><B>PSB1C3T7-HNS/HNS-T108I / PotB59/PomA CLONING</B></P><P>As we didn’t very satisfying results from our functional assays, we checked everything about our experiments. Then we noticed that even one amino acid change in a protein can cause a difference in protein activity. We noticed that His-tag was added into the proteins when we cloned the genes into pET45-b. Hence we cloned these genes into our own design PSB1C3-T7 plasmid, in order to produce required proteins without any addition. For this cloning, we removed the gene sequences from PSB1C3-HNS, PSB1C3-HNS-T108I, PSB1C3-PotB59/PomA plasmids with BamHI and XhoI enzymes. Then we ligated the cut gene with pET45-b which was cut with BamHI and SalI enzymes.(XhoI and SalI enzymes occurs a scar when their cuts ligate)We transformed ligated products into BL21 bacteria strain.
</P><P>To check if the cloning is correct, a colony PCR was performed with Verify Forward and verify Reverse primers. If the cloning isn’t made properly the band should be at 503 bp line, but if colony PCR worked, then bands are expected to be at 906 bp line for HNS and HNS-T108I, 2204 bp line for PotB59/PomA.
</P><P>As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check in order to learn if the cloning was correct. For cut-check, we used EcoRI and PstI restriction enzymes for HNS and HNS-T108I, only EcoRI for PotB59/PomA.
</P><P><B>RESULT FOR HNS</B></P><P><B>RESULT FOR HNS-T108I</B></P><P><B>FUNCTIONAL ASSAY</B></P><P>We repeated functional assays after we cloned the genes into PSB1C3-T7. The results are shown below.</P><P>The measurements which made at  first 24. hour period, gives us significant positive results . If we compare the results,colony diameters are ordered like this;<B> (-) control &lt; Wild type HNS &lt; HNS-T108I&lt; PotB59/PomA </B></P>.













</DIV></DIV><H2 class="scroll">Sensing h.p.</H2><DIV id="content-div"><P>In this section, we aim to use our gene parts for making E. Coli, sensing H. Pylori’s existence and activating a pathogen-killer system, after it gets into the mucus layer. Targeting this pathogen will be provided by sensing Auto-inducer 2 (AI-2) and ammonia (NH3), both secreted from H. Pylori. AI-2 will be sensed by Lsr operon system and NH3 will be sensed by TnrA-pAlsT regulation system. Getting only one result of these two systems when they are sensed seperatley, is provided by Toehold-Triger RNA system. The output of this system, which works in the existence of these two molecules, is an endonuclease called TEV protease. Produced TEV protease will activate the antimicrobial peptide Pexiganan after this molecule synthesized in an inactive form and accumulated in cell. Activated Pexiganan molecules are expected to leave the cell, become free and kill H. Pylori.</P><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('10');">Background</A><DIV class="box"><H3>Helicobacter Pylori Sensing:</H3><P>Helicobacter pylori lives in stomach, by connecting to epithelium cells under the mucus layer.This bacteria releases some molecules around which comes from its metabolic waste. Two of these molecules are Auto-inducer 2 and ammonia.
</P><H4>Ammonia (NH3)</H4><P>H. pylori  synthesizes urease to buffer the pH of its microenvironment within the stomach. H. pylori dedicates several genes to the biosynthesis of its cytosolic urease, a Ni2+-containing enzyme which hydrolyses urea into NH3 and CO2. Bacteria has urea channel which is regulated positively by protons, opening at acidic pH values to allow more urea in to buffer cytosolic and surface pH, and closing at neutral pH to avoid over-alkalinization [1].
</P><P>Ammonia (NH3) buffers the cytosol and periplasm, and creates a neutral layer around the bacterial surface[1].With this surface created by bacteria, protects itself against low acid coditions of the stomach.
</P><DIV class="img-text"><P><B>Figure 1: </B>Helicobacter Pylori Environment.</P></DIV><H4>Autoinducer 2 (AI-2)</H4><P>Such as many gram-negative and gram positive bacteria has the quorum sensing (QS) molecule, H. Pylori has this molecule too. QS molecules are specific, low-molecular-weight signal molecules which used by bacteria to regulate expression of genes in response to changes in population density.[2]</P><P>H. pylori has Quorum Sensing molecule “Autoinducer-2 (AI-2)” which is produced depended on the activity of the LuxS enzyme [3].</P><P>Based on the information given above, we expect to find ammonia and Auto-inducer 2 molecules in the existence of H. pylori. Thus we aim to sense both of these molecules to find out if H. Pylori’s presence.</P><DIV class="img-text"><P><B>Figure 2:Our “AND GATE” system diagram:</B> According to our system, TnrA transcription factor represses TnrA promoter which produce Toehold. In the presence of NH3 this press will be eliminated and Toehold will be produced. Lsr transcription factor represses pLsr promoter which produce Trigger mRNA. In the presence of AI-2 (Autoinducer-2) this press will be eliminated and Trigger mRNA will be produced. In conclusion Trigger mRNA opens the toehold system and TEV-Protease will be formed.</P></DIV><P>As a result, our “SENSING” system composed of three parties: <B>1.</B>NH3 sensitive TnrA promoter</P><P><B>2. </B>AI-2 sensitive Lsr promoter</P><P><B>3. </B>Toehold system</P><H4>1. NH3 Sensitive TnrA Promoter</H4><P>Bacteria use nitrogen which is present in nearly all macromolecules such as proteins, carbonhydrates and peptidoglycan. Prokaryotes have developed transport and assimilation systems for a variety of nitrogen sources for living under optimal conditions and regulate their own systems. This regulatory network allows an adequate response to situations of nitrogen limitation.
</P><P>In the Bacillus subtilis, ammonium assimilation occurs via the glutamine synthetase - glutamate synthase pathway. Bacillus subtilis faces nitrogen- limiting conditions when it consumes glutamate as a prior nitrogen source,   while glutamine is the secondly preferred nitrogen source [3]. 
</P><P>Two transcription factors, TnrA and GlnR, and one enzyme, the Glutamine Synthase, are the major players in the B. Subtilis nitrogen regulatory network [3]. We use TnrA transcription factor in our system. 
</P><P>Under nitrogen-limited conditions, TnrA works as an activator and a repressor both. TnrA represses expression of glnRA (Glutamine Synthase) [4], gltAB (Glutamate Synthase) [5] and other genes. Also the form of Glutamine Synthase which is feedback inhibited by excess glutamine, directly interacts with and unbinds from TnrA, thus blocks its DNA-binding activity [6]. Based on all this information, if the amount of NH3 is not sufficient, glutamine synthase will not work properly, glutamine will be produced in a low amount, TnrA will bind to promoter and Toehold production will be repressed. But if there is a sufficient amount of NH3, glutamine will be produced in a high level, TnrA will not repress the promoter as previous and Toehold – Tev Protease will be produced in a high amount.
</P><DIV class="img-text"><P><B>Figure 3: </B> If there is a sufficient amount of NH3, Toehold and TEV protease will be produced. If there isn’t a sufficient amount of NH3, Toehold and TEV Protease will not be produced.</P></DIV><P>Seventeen TnrA targets were detected by a combination of DNA microarray hybridization, a genome-wide search for TnrA boxes, and gel retardation assays [7]. The TnrA box consensus delimited in this study to a 17- bp interrupted, inverted repeat sequence, <B>TGTNANAWWWTNTNACA.</B></P><H4>2. AI-2 Sensitive Lsr Promoter</H4><DIV class="img-text"><P><B>Figure 4:</B>LsrR-binding site recognition and regulatory characteristics in Escherichia Coli AI-2 quorum sensing (Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou and Baolin Sun).</P></DIV><P>In quorum sensing (QS) process, bacteria regulate gene expression by utilizing small signaling molecules called autoinducers in response to a variety of environmental cues. QS molecules secreted by bacteria are small, diffusible signaling molecules called autoinducers that accumulate in the external environment. When the concentration of the autoinducers reaches a threshold, an alteration of gene expression is induced, allowing the bacteria to adopt behaviors that are only productive when the bacteria are working together as a group [8].
</P><P>Many quorum sensing molecules have been identified until now. In contrast to other autoinducers that are specific for a narrow range of organisms, the widely conserved AI-2 has been hypothesized to be a universal language for interspecies communication [9]. In every case, AI-2 is synthesized by LuxS, which functions in the pathway for metabolism of S-adenosylmethionine (SAM), a major cellular methyl donor. In a metabolic pathway known as the activated methyl cycle, SAM is metabolized to S-adenosylhomocysteine, which is subsequently converted to adenine, homocysteine, and 4,5-dihydroxy-2,3-pentanedione (DPD, the precursor of AI-2) by the sequential action of the enzymes Pfs and LuxS [10]. 
</P><P>The regulatory network for AI-2 uptake is comprised of two other important components, lsrR and lsrK, which are located adjacent, but divergently transcribed from the lsr operon (Figure 1). LsrR is the repressor of the lsr operon and itself. LsrK is a kinase responsible for converting AI-2 to phospho-AI-2, which is required for relieving LsrR repression. It has also been postulated that phospho- AI-2 binds to LsrR and inactivates it to derepress the transcription of lsr [11].Since LsrR contains a helix-turn-helix (HTH) DNA-binding domain, it was hypothesized that LsrR represses the expression of lsr operon and itself by binding to their promoter regions [12]. 
</P><P>Two independent groups demonstrated that H. pylori secretes AI-2 into its extracellular environment by a luxS-dependent mechanism.[13,14] Therefore we planned to use LsrR system to sense AI-2 molecules synthesized by H. Pylori.
</P><P>In the absence of Autoinducer-2, LsrR repress the LsrR promoter to bind LsrR-Binding Site. In the presence of AI-2 extracellular AI-2 is imported into the cell (cytoplasmic AI-2) via LsrACDB transporter, where it is phosphorylated by LsrK. LsrK is a kinase responsible for converting AI-2 to phospho-AI-2, which is required for relieving LsrR repression. Phospho-AI-2 has been reported to bind to LsrR and relieve its repression effect on the lsrR promoter.
</P><H4>3. Toehold System</H4><P>Toehold switches provide a high level of orthogonality and can be forward engineered to provide average dynamic range above 400. Toehold switches, with their wide dynamic range, orthogonality, and programmability, represent a versatile and powerful platform for regulation of translation, offering diverse applications in molecular biology, synthetic biology, and biotechnology.New classes of regulatory components that offer wide dynamic range, low system crosstalk, and design flexibility represent a much-needed, enabling step toward fully realizing the potential of synthetic biology in areas such as biotechnology and medicine. (Khalil and Collins, 2010).
</P><P>Engineered riboregulators consist of cognate pairs of RNAs: a transducer strand that regulates translation or transcription and a trans-acting RNA that binds to the transducer to modulate its biological activity. Riboregulator designs can be classified according to the initial RNA-RNA interaction that drives hybridization between the transducer and trans-acting RNAs. Reactions initiated between loop sequences in both RNAs are termed loop-loop interactions, whereas those that occur between a loop sequence and an unstructured RNA are termed loop-linear(Takahashi and Lucks, 2013).
</P><P>A common limitation for riboregulators has been their dynamic range (Liu et al., 2012). Previous prokaryotic translational riboregulators have typically modulated biological signals by up to a maximum of 55-fold for activators (Callura et al., 2012) and up to 10-fold for repressors (Mutalik et al., 2012). In contrast, protein-based transcriptional regulators have demonstrated dynamic ranges over an order of magnitude higher, with widely-used inducible promoters regulating protein expression over 350-fold (Lutz and Bujard, 1997) and sigma factor-promoter pairs providing up to 480-fold modulation (Rhodius et al., 2013).Despite the inherent programmability of RNA-based systems, efforts at constructing large sets of orthogonal riboregulators have been limited to libraries of at most seven parts with crosstalk levels of 20% (Takahashi and Lucks, 2013). Typical RNA-based regulators employ interaction domains consisting of30 nts, which corresponds to a sequence space of over 1018 potential regulatory elements. Thus, the sheer diversity of possible RNA-based parts suggests that previous devices have not come close to realizing the potential of highly orthogonal regulation.
</P><DIV class="img-text"><P><B>Figure 5: </B>(A and B) Design schematics of conventional riboregulators (A) and toehold switches (B). Variable sequences are shown in gray, whereas conserved or constrained sequences are represented by different colors.</P></DIV><P>Much of this discrepancy arises from the significant sequence constraints imposed on riboregulators engineered thus far (Figure1A). Like natural riboregulators, engineered riboregulators of translation have invariably used base pairing to the ribosome binding site (RBS) to prevent ribosome binding, thereby preventing translation (Callura et al., 2012; Isaacs et al., 2004; Mutaliket al., 2012; Rodrigo et al., 2012). Because repression is caused by RBS binding, trigger RNAs that activate translation are engineered to contain an RBS sequence to displace the repressing sequence, which in turn reduces the potential sequence space for the riboregulator.
</P><P>Previous riboregulators have also relied on U-turn loop structures to drive loop-loop and loop-linear interactions between RNAs (Figure 1A) (Callura et al., 2012; Isaacs et al., 2004; Luckset al., 2011; Takahashi and Lucks, 2013). U-turn loops are common RNA structural motifs formed by tertiary interactions that have been identified in ribozymes, ribosomal RNAs, and transfer RNA anticodon loops (Gutell et al., 2000). Although recent work has begun to show that loops with canonical U-turn sequences are not essential for riboregulators (Mutalik et al., 2012; Rodrigoet al., 2012), the engineered systems reported to date have continued their reliance on the loop-mediated RNA interactions from natural systems. Although these loop interactions have been selected by evolution in nature, alternative approaches employing linear-linear RNA interactions are amenable to rational engineering and exhibit more favorable reaction kinetics and thermodynamics, factors that could be exploited to increase riboregulator dynamic range.
</P><P>A riboregulator that activates gene expression must switch from a secondary structure that prevents translation to a configuration that promotes translation upon binding of a cognate trans-acting RNA. Although the Shine-Dalgarno sequence is an important factor in determining the efficiency of translation from a given mRNA, studies have found that secondary structure in regions near by the start codon also plays a critical role (Kudla et al., 2009).Furthermore, genome-wide analyses have revealed strong biases toward low secondary structures around the start codon of mRNAs from a panel of hundreds of bacterial genomes.
</P><P>Toehold switch systems are composed of two RNA strands referred to as the switch and trigger (Figure 1B). The switch RNA contains the coding sequence of the gene being regulated. Upstream of this coding sequence is a hairpin-based processing module containing both a strong RBS and a start codon that is followed by a common 21 nt linker sequence coding for low-molecular-weight amino acids added to the N terminus of the gene of interest. A single-stranded toehold sequence at the 50 end of the hairpin module provides the initial binding site for the trigger RNA strand. This trigger molecule contains an extended single stranded region that completes a branch migration process with the hairpin to expose the RBS and start codon, thereby initiating translation of the gene of interest. The hairpin processing unit functions as a repressor of translation in the absence of the trigger strand. Unlike previous riboregulators, the RBS sequence is left completely unpaired within the 11 nt loop of the hairpin. Instead, the bases immediately before and after the start codon are sequestered within RNA duplexes that are 6 bp and 9 bp long, respectively. The start codon itself is left unpaired in the switches we tested, leaving a 3 nt bulge near the midpoint of the 18 nt hairpin stem. Because the repressing domain b (Figure 1B) does not possess complementary bases to the start codon, the cognate trigger strand in turn does not need to contain corresponding start codon bases, thereby increasing the number of potential trigger sequences. The sequence in the hairpin added after the start codon was also screened for the presence of stop codons, as they would prematurely terminate translation of the gene of interest when the riboregulator was activated. We employed a 12 nt Toehold domain at the 50 end of the hairpin to initiate its interaction with the cognate trigger strand. The trigger RNA contains a 30 nt single-stranded RNA sequence that is complementary to the toehold and stem of the switch RNA.
</P><DIV class="img-text"><P><B>Figure 6: </B>(C) Flow cytometry GFP fluorescence histograms for toehold switch number 2 compared to E. coli autofluorescence and a positive control. Autofluorescence level measured from induced cells not bearing a GFP-expressing plasmid.(D) GFP mode fluorescence levels measured for switches in their ON and OFF states in comparison to positive control constructs and autofluorescence. Error bars are the SD from at least three biological replicates.
Based on all information given above, we decided to use Toehold-Trigger RNA system fort he AND Gate part of our project.</P></DIV><P>At last, here is the place of toehold-trigger RNA system in our project as a diagram:</P><H4>Sources: </H4><P>[1]: Montecucco C, Rappuoli R. Living dangerously: how Helicobacter pylori survives in the human stomach. Nat Rev Mol Cell Biol. 2001;2(6):457-66.
[2]: Vendeville A, Winzer K, Heurlier K, Tang CM, Hardie KR. Making 'sense' of metabolism: autoinducer-2, LuxS and pathogenic bacteria. Nat Rev Microbiol. 2005;3(5):383-96.
[3]: Loh JT, Forsyth MH, Cover TL. Growth phase regulation of flaA expression in Helicobacter pylori is luxS dependent. Infect Immun. 2004;72(9):5506-10.
[4]: Brown SW, Sonenshein AL. Autogenous regulation of the Bacillus subtilis glnRA operon. J Bacteriol. 1996;178(8):2450-4.
[5]: Commichau FM, Wacker I, Schleider J, et al. Characterization of Bacillus subtilis mutants with carbon source-independent glutamate biosynthesis. J Mol Microbiol Biotechnol. 2007;12(1-2):106-13.
[6]: Wray LV, Zalieckas JM, Fisher SH. Bacillus subtilis glutamine synthetase controls gene expression through a protein-protein interaction with transcription factor TnrA. Cell. 2001;107(4):427-35.
[7]: Yoshida K, Yamaguchi H, Kinehara M, Ohki YH, Nakaura Y, Fujita Y. Identification of additional TnrA-regulated genes of Bacillus subtilis associated with a TnrA box. Mol Microbiol. 2003;49(1):157-65.
[8]: Ahmer BM. Cell-to-cell signalling in Escherichia coli and Salmonella enterica. Mol Microbiol 2004; 52:933-945
[9]: Federle MJ, Bassler BL. Interspecies communication in bacteria. J Clin Invest 2003; 112:1291-1299.

[10]: Vendeville A, Winzer K, Heurlier K, Tang CM, Hardie KR. Making ‘sense’ of metabolism: autoinducer-2, LuxS and pathogenic bacteria. Nat Rev Microbiol 2005; 3:383-396.

[11]: Taga ME, Miller ST, Bassler BL. Lsr-mediated transport and processing of AI-2 in Salmonella typhimurium. Mol Microbiol 2003; 50:1411-1427.

[12]: Xavier KB, Miller ST, Lu W, et al. Phosphorylation and processing of the quorum-sensing molecule autoinducer-2 in enteric bacteria. ACS Chem Biol 2007; 2:128-136.

[13]: Forsyth, M. H., and T. L. Cover. 2000. Intercellular communication in Helicobacter pylori: luxS is essential for the production of an extracellular signaling molecule. Infect. Immun. 68:3193–3199.
[14]: Joyce, E. A., B. L. Bassler, and A. Wright. 2000. Evidence for a signaling system in Helicobacter pylori: detection of a luxS-encoded autoinducer. J. Bacteriol. 182:3638–3643
[15]:Green, A., Silver, P., Collins, J., &amp; Yin, P. (n.d.). Toehold Switches: De-Novo-Designed Regulators of Gene Expression. Cell, 925-939.</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('11');">Design</A><DIV class="box"><H4>TnrA-pAlsT -RFP </H4><P>Main goal of this part is to show TnrA protein’s repressor activity on pAlst promoter and derepression of this promoter in the presence of NH3. Thereby we combined TnrA protein’s and pAlst’s gene sequences on the same biobrick. As a reporter, we put RFP gene sequence in front of the pAlst sequence. To make sure the repressive protein TnrA is produced in a high amount, we combined TnrA-pAlst-RFP gene sequence with T7 promoter which has a high transcription rate. Also, we put a Lac operator which LacI protein can bind, between this sequence and T7 promoter, in order to create a controllable system. We searched to find an expression vector working in harmony with this system, peT45-b expression vector was serving this purpose. . When we ordered our genes, we put RFC10 prefix site on 3’ end of TnrA-pAlst-RFP gene sequence and also BamHI restriction enzyme recognition site for cloning to peT45-b. We added RFC10 suffix site on 5’ end of the same gene sequence and also XhoI restriction enzyme recognition site for cloning to peT-45 vector, again.</P><P>As shown above,we planned to clone the ordered TnrA-pAlst-RFP gene into peT45-b vector by using BamHI and XhoI enzymes. Final construct after this cloning is, T7 promoter-Lac operator(LacO)-HisTag-TnrA-pAlst-RFP, respectively. Besides, a constitutive promoter called LacI promoter and LacI protein sequence in front of that, found on peT45-b vector’s another part. It is obvious that this part is IPTG-dependent, so Western Blot can be performed easily with the help of His Tag.
</P><P>According to this system, under the absence of IPTG; TnrA production  will be repressed, pAlsT promoter will be derepressed and RFP won’t be produced. If IPTG is present,  TnrA will be produced , pAlsT promoter becomes derepressed and red colored bacteria will be observed with production of RFP.
After showing stability of TnrA-pAlst sytem by using the RFP fluorescence protein, Toehold-TEV protease will be replaced with RFP for binding to AND gate system.
</P><H4>QS dependent promoter pLsr</H4><P>There are two promoters on LsrR’s operon Lsr . these promoters are called LsrA and LsrR, and they are repressed in the absence of AI-2. The location LsrR binds on these promoters(p-lsr-Box)  are shown below. Dark ones indicate the common sequences in both promoters. We preferred to use lsR promoter as it is one of ATOMS Team’s parts in 2013.
</P><DIV class="img-text"><P><B>FIGURE 5:</B>LsrR-binding site recognition and regulatory characteristics in Escherichia Coli AI-2 quorum sensing (Ting Xue, Liping Zhao, Haipeng Sun, Xianxuan Zhou and Baolin Sun).</P></DIV><P>We aimed to show repression of LsrR protein on pLsr promoter and dereppession of it in the presence of AI-2. For this purpose, we gathered LsrR protein gene sequence and pLsr sequence on the same biobrick. We put RFP as a reporter in front of pLsr promoter. To produce th repressor protein Lsr in a high amount, we put LsrR-pLsr-RFP gene sequence in front of J23100 constutive promoter .  We added RFC prefix on 3’end , RFC suffix on 5’ end of  LsrR-pLsr-RFP gene sequence.</P><P>LsrABCD operon, which makes AI-2 getting into the cell, already present in E.coli; so it is expected to be enough when an AI-2 sensing promoter system placed into E. coli. Thereby under the absence of AI-2, Lsr promoter will be heavily repressed by LsrR protein. If Quorum Sensing molecule sensed successfully AI-2 will be phosphorylated and LsrR’s suppression on pLsr will disappear. Thus RFP expression will occur.
</P><P>We chose to use the part J23100-LsrR-pLsR-RFP(BBa_K1202108) as 2013 ATOMS IGEM TEAM designed. After showing LsrR-pLsrR system’s stability by using RFP, Trigger RNA will be replaced with RFP to connect the inputs to the And gate system.
</P><H4>Trigger RNA-Toehold-GFP System</H4><P>We aimed to show Toehold switch system’s incapability of protein expression when it is alone, which means the Trigger RNA and Toehold Switch must coexist  for protein expression. To observe  Toehold swtich’s and and Toehold Swtich-Trigger RNA’s coexisting results both, we designed Toehold switch and trigger RNA in different biobricks. Because of origin incompatibility process, we had to clone these two different gene sequences into different plasmids. Thus we cloned Toehold-GFP into ColA originated PColA plasmid, and Trigger RNA into ColE originated PSB1C3 plasmid.
</P><P>The ordered form of Toehold G-blocks is shown above. We preferred to use T7 promoter, which is a very strong promoter in this gene’s design. Thus there won’t be any difficulty in Toehold-GFP RNA’s production. The RBS of GFP presented in Toehold structure. To clone Toehold-GFP sequence into PSB2C3 and pColA plasmids both, we added RFC 10 suffix region on 3’ end and RFC 10 prefix region  on 5’ end of Toehold-GFP. We also put BamHI restriction sitebetween Toehold and GFP. After GFP expression, which shows that Toehold-trigger system works, any protein could be replaced with GFP.  In our project, GFP will be replaced with TEV protease.
</P><P>Trigger RNA, which turns on Toehold switch system, is shown above in its ordered form as G-Blocks. In the design of this biobrick, we used T7 promoter, which has a strong mRNA production rate. By doing these, an enough amount of Trigger RNA will be produced to open Toeholds. There is no RBS in this product, because Trigger RNA doesn’t codes a protein. Its only task is opening Toehold switch in order to produce protein. To clone Trigger RNA sequence into PSB1C3 plasmid, we added RFC10 prefix on 3’ end and RFC10 suffix on 5’ end.
</P><DIV class="img-text"><P><B>Figure:</B> Toehold switch and Trigger RNA deatiled design</P></DIV></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('12');">Results</A><DIV class="box"><H3>NH3 Sensitive Promoter and TnrA</H3><H4>PSB1C3-TnrA-pAlst-RFP CLONING</H4><P>We cloned IDT G-Blocks GadE gene into PSB1C3 vector in order to make it ready to be submitted and have many copies of it. For this purpose we digested PSB1C3 vector and TnrA-pAlst-RFP G-Blocks with ECoRI and PstI restriction enzymes. Then we ligated these cut genes into the plasmid by using T4 DNA Ligase . Ocurring products were transformed into BL21 competent cell strain.</P><P>To check if the cloning is correct, a colony PCR was perfomed with Verify Forward and Verify Reverse primers. If the cloning isn’t made properly t the band should be 314 bp long, but if colony PCR worked, then bands should be 1800 bp long.
</P><P>As the result of Colony PCR experiments, we observed bands are either negative or less molecule-weighted than we expected. We made liquid cultures of the not negative colonies, and isolated DNA from these colonies after incubating for 16 hours. we did cut-check with EcoRI and PstI enzymes in order to control occurring DNA’s.
</P><P>After  cut-check, we observed less molecule-weighted bands again. Therefore we decided to control our ordered G-Blocks and did a PCR experiment with CMV forward and SV40 reverse primers. The gel  image of this PCR is shown below.
</P><H4>TnrA-pTnrA-RFP PCR</H4><P>After that PCR, we observed less molecule-weighted bands than normal.</P><H4>pET45 -TnrA-pTnrA-RFP CLONNING</H4><P>We questioned if the plasmid is incorrect because the results of PSB1C3 cloning weren’t matching with our expected results. We cut the G-Block with BamHI and XhoI and ligated it with pET45-b which was cut with same enzymes. We transformed this ligated product into BL21 bacteria strain, which has T7 RNA polymerase.
</P><P>In order to control if the cloning is made correctly, we did a colony PCR with T7 Promoter reverse and T7 promoter forward. If cloning isn’t made properly, the band should be at 360 bp line and if is made properly the band should be at 1779 bp line.</P><P>In colony PCR results, we observed lower molecule-weighted bands than expected, again. In this case we realized that this gene sequence was wrong.
</P><P>We didn’t have enough time to order and clone this gene so we didn’t continue to do this part’s experiments.
</P><H3>AI-2Sensitive Promoter and LsrR</H3><P>We got the results of Lsr promoter part from 2013 ATOMS iGEM team. Their results are shown below.</P><H4>Inducible promoter experiment</H4><P>To observe inducible promoters response, we co incubated two different bacteria. The first bacteria culture is expressing enzyme system and the second culture is including inducible promoter. At the experiment, we incubated them seperately for 16 hours and mixed them into one flask and 4 hour later, we add 0,1 mM SAH (with 10x PBS containing %1 BSA). Incubate with SAH for one day and, took 2 ml of liquid culture to santrifuge tube.Centrifuged it. We saw the red color at the pellet as you see below.</P><P>So, it means the enzyme system is working and inducible promoter is succesfully induced from AI-2.</P><H3>Toehold-Trigger RNA-GFP</H3><H4>PSB1C3-Toehold-GFP/Trigger RNA CLONING</H4><P>We cloned these two IDT G-Block genes into PSB1C3 vector in order to make it ready to be submitted and have many copies of it. For this purpose we digested PSB1C3 vector and TnrA-pAlst-RFP G-Blocks with ECoRI and PstI restriction enzymes. Then we ligated these cut genes into the plasmid by using T4 DNA Ligase . Ocurring products were transformed into BL21 competent cell strain.
</P><P>To check if the cloning is correct, a colony PCR was perfomed with Verify Forward and Verify Reverse primers. If the cloning isn’t made properly t the band should be 314 bp long, but if colony PCR worked, Toehold-GFP’s band should be at 1294 bp line and Trigger RNA’s should be at 443.
</P><P>As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check as a second control of cloning. We used EcorI and PstI restriction enzymes for cut-check.
</P><H4>RESULT FOR TOEHOLD-GFP</H4><H4>RESULT FOR TRIGGER RNA</H4><P>In order to make And gate work, Toehold-GFP and Trigger RNA must be transformed into the same bacteria. Origin incompatibility prevents us to transform two different originated plasmids in the same bacteria. Therefore we decided to clone Toehold-GFP into a ColA originated plasmid and clone Trigger RNA into a ColE originated plasmid. PSB1C3 plasmid has a ColE origin so we cloned Trigger RNA into it. We designed this ColA originated plasmid for cloning Toehold-GFP sequence and named it ‘pColA’.  
</P><H4>pColA-Toehold GFP CLONING</H4><P>After cloning Toehold-GFP into PSB1C3 vector successfully, as we are planning to transform this gene into the same bacteria with Trigger RNA, we cloned it into pColA plasmid.  In order to do cloning into this vector, we removed the gene from PSB1C3-Toehold GFP plasmid by cutting it with NotI enzyme. Then we ligated these cut genes into the plasmid by using T4 DNA Ligase . Ocurring products were transformed into BL21 competent cell strain.
</P><P>To check if the cloning is correct, a colony PCR was perfomed with ColA Forward and ColA Reverse primers. If the cloning isn’t made properly t the band should be 192 bp long, but if colony PCR worked, Toehold-GFP’s band should be at 1196 bp line.</P><P>As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check as a second control of cloning. We used NotI restriction enzyme for cut-check.</P><H4>PSB1C3-Trigger RNA/pColA-Toehold-GFP Cotransformation:</H4><P>To show And Gate system works properly, these two plasmids with different origins should be transformed into the same bacteria. T7 is the promoter which produces Trigger RNA and Toehold-GFP mRNA’s , so we assured that they were cotransformed into a bacteria strain including T7 RNA polymerase. BL21 served this purpose, and after cotransformation, we observed grown colonies.
</P><H4>Functional Assay:</H4><P>We designed a functional assay setup in order to figure out if Toehold-Trigger RNA system is functional. The main goal of this setup is to show that; Toehold-GFP doesn’t give fluorecence alone but if it comes together with Trigger RNA, GFP fluorescences. GFP production is also IPTG-dependent.
</P><P>  For the purposes given above, we made liquid culture of two different bacteria together; bacteria including only pColA-Toehold-GFP plasmid and bacteria including pCola-Toehold-GFP and PSB1C3-Trigger RNA. We incubated these bacteria in liquid culture for 13 hours at 37C and added IPTG into their mediums at 13. hour. After adding IPTG, we incubated them for more 3 hours at 37C. At the end of three hours, we firstly measured GFP fluorescence of the grown cultures by using VarioScan device.  The results of measurements are given above. They were made twice.
</P><P>When the results are analyzed, it is obvious that Toehold-Trigger RNA parts gave very high amounts of flourescence together while Toehold-GFP part gave flourescence in a very low amount. There is almost 15 times difference between these two fluoroscence amounts.  This proves that Toehold-Trigger system works successfully. Also it is shown that systems work IPTG-dependently.
</P><P>The graphs of measurement results are given below. The difference between two systems can be observed clearly.
</P><DIV class="img-text"><P><B>Figure : </B>GFP flourescent measurrement in LB mediums.</P></DIV><P>We also observed the liquid cultures under the fluorescence microscope.</P><DIV class="img-text"><P><B>Figure 2: </B>Flourescence microscope image of GFP producing E. coli.</P></DIV><P>For better results, we isolated protein and we fluorimetrically measured them. Firstly we centrifuged the liquid cultures which were incubated for 16 hours. We took pictures of tubes after centrifugation, they are shown belown.
</P><DIV class="img-text"><P><B>Figure :</B> Pellets can be seen after 16 hours of liquid culter precipitated.</P></DIV><P> We applied Standard protein isolation protocol after this centrifuge and managed to isolate protein from the occured pellets. We made GFP fluorescence measurement with VarioScan from those isolated proteins. The results of measurements are shown below.
</P><P>The measurement results of isolated proteins indicate that Toehold-Trigger system works very efficiently. GFP fluorescence amount of Toehold-trigger RNA including bacteria is about 175 times more than only Toehold including bacteria’s. also the leak of this system is almost negligible. In the absence of Trigger RNA, Toehold’s GFP fluorescence is almost zero. This indicates that our system works very efficient and in a very specific way.
</P><DIV class="img-text"><P><B>Figure :</B> Protein extract GFP florasans measurement</P></DIV></DIV></DIV><H2 class="scroll">Killing h.p.</H2><DIV id="content-div"><P>Our main goal is in this, eradicating from the body Helicobacter pylori that lives as the pathogen with pexiganan, which a powerful new antimicrobial peptide. In our project we use pexiganan to eradicate Helicobacter pylori because the pexiganan molecule has a powerful effect on the pathogen. Furthermore, it doesn't have a toxic effect  on our natural body cells. Pexiganan molecules’ dose for destroying erythrocytes is 250 ng/ml, the other dose for  destroying H.pylori is 16 ng/ml.
</P><P>In gastric fluid acidic pH conditions, the peptide degradation speed is very high; Pepsine, the stomach’s main digestion enzyme, crushes all of the peptide structure molecules; the drug taken for gastric fluid has a very short lifetime. These are all reasons that make it difficult to set an effective dose.
</P><P>Because of this reason, in our project we aim to start pexiganan synthesis only with the maximum closest reach.  
</P><P>We also aim to produce enough pexiganan molecules to eradicate all of the h. pylori population in the stomach.
</P><P>In line with our aims;<B>1. </B>With the parts described previously, our synthetic bacteria will enter into the mucosa layer where the h.pylori is in stomach.
<B>2. </B>We use E.coli in our treatment to continuously product DAMP-Pexiganan molecules. These continuously produce and accumulate proteins that don't have any lethal effects to the cells. Hence, DAMP-Pexiganan molecule’s active site is shut down by DAMP.( TEV protease recognition site located between the DAMP and Pexiganan molecules.) Thus, it goes to the area where Helicobacter pylori is in the stomach until the pathogen detects pexigan molecules  created by E.coli and that allow E.coli's ability to remain alive until producing enough pexiganan.</P><P><B>3. </B>Due to the presence of H. pylori in the environment, extremely high concentrations of NH3 and Al2 molecules induce AND GATE the system. As a result of this activation, TEV protease molecules will be produced. The produced TEV Protease break the large amount of connections accumulated between DAMP and Pexiganan in the cells and releases pexiganan molecules. These pexiganan molecules are active lysis molecules that cause cells to splinter. For this reason, pexiganan molecules lysing E.colis which contain pexiganan and get it out of the cells. The pexiganan which set the release of the cell will allow the elimination of Helicobacter pylori.</P><P>In this case, the protein that binds to DAMP and the tip imbedded in PEX. will be produced in the E. coli without rupturing the cell membrane. TEV Protease that produces in the presence of AND GATE and the active pexiganan will leave from DAMP,  and keep releasing the effective dose within and around the h.pylori. At the same time, we will provide eradication E.coli which is used as a producer and a carrier so that there won't be anymore possibility of colonization mucosa to have to be removed.
</P><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('13');">Background</A><DIV class="box"><H4>Pexiganan</H4><P>According to our researches, a very efficient protein called pexiganan, a molecule showing a broad spectrum of potent antimicrobial activities against both gram negative and positive bacteria, seemed beneficial for killing H. Pylori.(Figure 1) Antimicrobial peptides (AMP) are very popular because of their bactericidal activity and they seem potential alternatives to current antibiotics. Also, antimicrobial peptides are not affected by classical antibiotic mechanisms of antibiotic resistance, so this class of peptide have advantages to eradicate H.pylori instead of using current antibiotics. The other advantage of AMPs is that there aren’t any resistance to them-yet.  This was shown in a study too. (Figure 2)
</P><P>In a paper, pexiganan’s killer activity is proved under both in vitro and in vivo conditions. (Xiao-Lin Zhang et al. 2015) It has exhibited a broad-spectrum antibacterial activity in vitro, and it has been tested against 3109 clinical isolate strains of Gram-negative and Gram-positive, aerobic and anaerobic bacteria. (Ge, Y; MacDonald, D,L et al. 1999). By using normal H.Pylori treatment way, complete eradication is not possible because of the low antibiotic concentration in gastric mucosa. In addition, in the low pH of the gastric fluid, antibiotics are unstable. As a result of gastric emptying, the residence time is short for antibiotics in the stomach. Therefore, our strategy depends on providing production of pexiganan enter the mucosa according to presence of H.pylori or not. This system is based on the penetration of our engineered E.coli under the mucosa, and production of pexiganan under the control of the AND GATE system. After the production of pexiganan under the mucose, if this peptide is toxic for mammalian cells, this cannot be used for H.pylori eradication in the gastric mucosa. Therefore, toxicity level of pexiganan for mammalian cells must be higher than the requirement of H.pylori eradication. The potential toxicity of pexiganan has been investigated systematically by measuring the peptide’s hemolytic activity in human red blood cells in other studies. The reports suggested that at least 250 μg/mL is necessary to induce 100% hemolysis. (Navon, V; Feder, R et al. 2002 and Eren,T; Som, A et al. 2008) Two phase III clinical trials have reported that no adverse side effects have been found for pexiganan.    
(The Synthetic Antimicrobial Peptide Pexiganan and ItsNanoparticles (PNPs) Exhibit the Anti-Helicobacter pylori Activity in Vitro and in Vivo)</P><H4>DAMP</H4><P style="font-size:16px">  (A Simple and Low-Cost Platform Technology forProducing Pexiganan Antimicrobial Peptide in E. Coli,Chun-Xia Zhao, Mirjana Dimitrijev Dwyer, Alice Lei Yu, Yang Wu, Sheng Fang, Anton P.J. Middelberg)</P><P>DAMP,  a four-helix bundle protein connected to Pexiganan, used to ensure pexiganan’s intracellular antimicrobial activity was prevented successfully. A design, as showed above, was used to attach DAMP to Pexiganan with a cleavage site. That cleavage site makes it possible to release Pexiganan in an exact concentration. In addition, DAMP provides high levels of solubles in recombinant bacteria. It also forms a four helix bundle structure that  has thermo stability and remains soluble at high salt levels. Finally, pexiganan embeds into DAMP and this provides limitted electrostatic of pex. It prevents damage to the cell membrane in the E. coli.</P><H4>TEV Protease</H4><P>As a protease needed to cut the cleavage site between DAMP and Pexiganan, TEV protease is chosen to serve this purpose. TEV protease is the common name for the 27 kDa catalytic domain of the Nuclear Inclusion a (NIa) protein encoded by the tobacco etch virus (TEV). Because its sequence specificity, TEV protease is a very useful reagent for cleaving fusion proteins. It is also relatively easy to overproduce and purify large quantities of the enzyme. (Macromollecular Crystallography Laboratory, Protein Engineering Section, David Waugh)
</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('14');">Design</A><DIV class="box"><P>The project's main goal on this part is producing the inactive form of the pexiganan which have antimicrobial effects; DAMP-Pexiganan and producing TEV Protease which will activate DAMP-Pexiganan. For producing this proteins abundantly we use T7 promoter ,which has extremely strong transcription rate, while we designing our parts. We also decided to have Lac Operator site (that LacI protein can bind) between our gene sequences and T7 promoters to make protein expressions controllable. We researched for the expression vector that satisfy these requirements, and we found we could use the pet45-b expression vector provides the conditions we want. While we genes to order we added RFC10 prefix domain to the 3 'end of DAMP-Pexiganan and TEV Protease and we also added BamHI restriction enzyme recognition sequence for clonning inside to Pet45-b vector. We added RFC10 Suffix domain to the 5’ end of the same gene sequences, same way we added XhoI restriction enzyme recognition sequence for cloning inside to Pet45-b plasmid. In line with our ultimate goal we aim producing of  these two proteins in the same bacteria. To make this happen we need to ligate these two gene sequences with two plasmids these have different types of origin(Origin incompatibility).  Therefore, we have designed Damp PEX gene sequence in this manner so that at the same time we can ligate it with pColA vector. pColA plasmit do not include promoter in itself beceuse of that we added T7 promoter to 5' end of DAMP-Pexiganan gene sequence.
</P><P>As stated above, we planned clonning  ordered DAMP-Pexiganan and TEV Protease genes in Pet45-b vector with BamHI and XhoI restriction enzymes. When this clonning performed; T7 promoter-Lac operator (LacO)- HisTag- DAMP-pexiganan/TEV protease construction will ocur in order. In addition LacI promoter ,which is a constutive promoter in an other region, and also LacI protein sequence in front of this. In view of the above information DAMP-pexigan and TEV protease protein's production dependent IPTG and with help of His-tag it easily executed via Western blotting.</P><P>In addition to this if DAMP-Pexiganan gene sequence insert into pColA vector with NotI enzyme succesfully below stated construction will occur.</P></DIV><A href="#" class="clickme" id="toggle" onclick="toggle_visibility('15');">Results</A><DIV class="box"><H4>PSB1C3-DAMP-PEXIGANAN/TEV PROTEASE CLONNING</H4><P>We thought to ,ligate our gene sequence whichs ordered from IDT with the g-BLOCKS form, PSB1C3 firstly to clonning and submittions.  Therefore we digested both PSB1C3 vector and Damp-Pexigan and TEV protease-G-blocks with EcoRI and PstI restriction enzymes. Then we aimed to insert these cuted g-BLOCKS into plasmid with using T4 DNA Ligase. Following ligation, the resulting plasmids were transformed into BL21 competent bacteria.
</P><P>We did colony PCR in order to check the accuracy of cloning with Veritification Forward and Veritification Reverse Primers. If the clonning resulted negative expecting band would be 314 bp. On the other hand if we succesfully clonned these parts, for DAMP-Pex. Expected band would be 1006 bp, for TEV Protease it would be 1062 bp.
</P><P>After colony PCR consequences we have put the positive resulted clonnings 16 hours liquid culture  after that we isolated plasmid DNA with miniprep plasmid izolation method. We make cutcheck to our attained plasmid DNAs for put a second control step. Our restriction enzymes those we used are EcoRI and PstI.
</P><H4>pET45- DAMP-PEXIGANAN/TEV PROTEASE CLONNING</H4><P>We planned; clonning our genes ,whichs clonned succesfully to PSB1C3 vector, into pET45-b expression vector. For producing abundantly protein.  To allow for cloning this vector we withdraw our genes from PSB1C3- DAMP-Pexigan and PSB1C3-TEV protease gene plasmids with using BamHI and XhoI digestion enzymes. We did ligation within using pET45-b vector that digested with the same enzymes. Following ligation, the resulting plasmids were transformed into BL21 competent bacteria which have T7 RNA Polymerase that we know. 
</P><P>We did colony PCR in order to check the accuracy of cloning with using T7 promoter forward and T7 terminator reverse primers. If the clonning resulted negative expecting band would be 360 bp. On the other hand if we succesfully clonned these parts, for DAMP-Pex. Expected band would be 699 bp, for TEV Protease it would be 1041 bp.
</P><P>After colony PCR consequences we have put the positive resulted clonnings 16 hours liquid culture  after that we isolated plasmid DNA with miniprep plasmid izolation method. We make cutcheck to our attained plasmid DNAs for put a second control step. Our restriction enzymes those we used are BamHI and XhoI.
</P><H3>WESTERN BLOTTING</H3><P>After we clonned to pET45-b succesfully we showed our aim proteins production via western blotting with using His-tag which lacate in N-terminal of these proteins.</P><H4>RESULT FOR DAMP-Pexiganan</H4><H4>RESULT FOR TEV Protease</H4><H4>PColA- DAMP-PEXIGANAN CLONNIG:</H4><P>We demonstrated that DAMP-Pexiganan and TEV Protease proteins synthesing succesfully within clonning to pET45-b plasmid. We thought that we should insert both two genes(DAMP-Pexiganan and TEV Protease) at the same time to same bacteria to show TEV Protease breaking linker chain (to releasing active free pexiganan) that is between DAMP and Pexiganan. We have seen with the same origin in two plasmids can not be found in a bacterium simultaneously due to origin incompatibility process. For this reason, we decided to clonning one of these two gene sequences into pet45-b plasmid which have ColE origin and one of the gene sequence clonning into pColA plasmid which have ColA origin which also we designed it.
</P><P>Before that we achieved to clonning both of two gene sequences into pET45-b vector. Therefore, clonning only one of these genes would be enough for us. We decided to locate TEV Protease gene sequence into pET45-b because our essential want to control protein is TEV Protease. And we decided clonning DAMP-Pexiganan gene sequence into pColA vector.  For clonning DAMP-Pexiganan protein sequence into pColA vector we toke out our gene with NotI enzyme from PSB1C3-DAMP-Pexiganan plasmid. We did ligation with pET45-b vector which digested the same enzyme. Following ligation, the resulting plasmids were transformed into BL21 competent bacteria.</P><P>We did colony PCR in order to check the accuracy of cloning with using pColA forward and pColA reverse primers. If the clonning resulted negative expecting band would be 192 bp. On the other hand if we succesfully clonned these part, expected band would be 908 bp.</P><P>After colony PCR consequences we have put the positive resulted clonnings 16 hours liquid culture  after that we isolated plasmid DNA with miniprep plasmid izolation method. We make cutcheck to our attained plasmid DNAs for put a second control step. Our restriction enzymes those we used are XbaI and SpeI.
</P><H4>pColA- DAMP-PEXIGANAN/pET45-TEV PROTEASE COTRANSFORMATION</H4><P>After we clonning DAMP-Pexiganan gene sequence into pColA plasmid and TEV Protease gene sequence into pET45-b for transformate these both two plasmids into same BL21 bacteria we did cotransformation. After cotransformation process we put bacterias at 37C for 16 hours incubation. End of 16 hours as a result we did not see any bacteria colony over agar plate.
</P><P>We retried several times this cotransformation process but with these tryings we havent seen any bacteria colony on our agar plates.
</P><H4>TEV protease Activity Control:</H4><P>After the negative results obtained from cotransformation experiements we designed a new experimental apparatus to demonstrate TEV Protease's breaking or separator effect to DAMP-Pexiganan molecules linker site. We  firstly make broth culture secondly put incubation at 37C for 13 hours bacterias which contain pET45-DAMP-Pexiganan and pET45-TEV Protease plasmids. For inducing protein production we added 1mM IPTG into medium at 13rd hour. Next we left these incubation for 3 hours at 37C. We isolated protein from totally 16 hours incubated broth culture. We prepared a suitable buffer to create the conditions necessary to show the TEV Protease’s enzymatic activity.(Conditions are given below). Then we prepare our DAMP-Pex, and TEV protease isolated proteins by mixing in the buffer, we put the resulting mixture overnight incubation at 20° C. After incubation, to observe the fate of DAMP-pexigan proteins we did Western Blot. But in western blotting, we could not get any significant rational result.
</P></DIV></DIV><H2 class="scroll" style="display:none">...</H2></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>