Team:ATOMS-Turkiye/Project/Cancer Switch
Tour
Project
Results
P&P
Team
Extras
Problems
Ulcer
Cancer Switch
Modeling
Safety
Future Plan
ATOMs
Attributions
Collaboration
Notebook
Protocols
Achievements
Official Part Page
Interlab Study
Virtual Hospital
Video
SynBio Day
Outreach
Street interview
Ethics
Team ATOMS-Turkiye
atoms2015@googlegroups.com
CANCER
In this section of our project we tried to present and elucidate a new diagnostic and treatment method for the gastric cancer caused by Helicobacter pylori infection. The biggest problem of the existing treatment is that it cannot provide sufficient specificity to cancer cells, targeting healthy cells as well and killing them in the process. Exterminating solely the cancer cells without also damaging healthy cells in the body requires a sensitive detection system. As a result of our investigation, we have seen that the level of some microRNAs in gastric cancer cells arise, but have also recognized that there is a decrease in the level of some other miRNA molecules in cancer cells in respect to normal cells. Thus, we decided to use miRNA molecules to separate normal cells from cancer cells and designed a genetic circuit functioning in accordance to miRNA levels. This genetic circuit can detect the shifts in miRNA levels in both the cancerous and healthy cells, also reporting whether it is higher or lower from the average level. Understanding whether there is an increase or decrease of the miRNA levels also makes it easier to adopt treatment.
Figure 1: Cancer switch. working principles
The main function of this system is apoptosis, through the apoptotic protein Bax. This protein is a last chain and is affected by the shifts in miRNA levels. Decreasing level of miRNA in cancer cells directly starts the breakdown the mRNA responsible for the transcription of the Bax protein, causing a decrease in the expression of Bax. Thus, Bax transcribing mRNA is degraded in very low quantities in cancer cells due to the low amount miRNA present and high rates of Bax translation will occur. Nevertheless, in normal cells miRNA levels are higher than cancer cells so the majority of Bax’s mRNAs will be degraded and Bax expression will be very low in healthy cells. Increasing the miRNA levels in cancer cells higher than the average levels in healthy cells will supply degradation factors for the repressor in the transcription stage, removing the repressive factors in the synthesis of  the Bax protein; a high quantity of miRNAs in cancer cells will repress the repressors, synthesizing higher amounts of the Bax protein. However, in normal cells, low levels of  miRNA will not be sufficient to lift the pressure on Bax mRNAs’ transcription. Thus, healthy cells will produce small amounts of Bax protein. As a result, the elevated or relatively reduced miRNA levels in cancer cells will be the only target of the system, preventing the production of apoptotic proteins in normal cells.
Figure 2: The designed system operates according to a specific miRNA profile. The miRNA level for some was low, but the level of others were high in this profile, with the high miRNA profile belonging to those of gastric cancer cells. If the cell is a cancer cell, the kill switch system will enter the cell having the appropriate miRNA profile and the system will open, causing the cell to go into apoptosis. If the cell is a healthy body cell, the system will not operate and there will be no effect on the cell.
Background
According to the investigation of Lauren Peirce Carcas (2014), gastric cancer is defined as a lethal disease caused by different factors converting healthy mucosal cells to cancer cells. Although cancer tissue initially is only present in the mucous layer, the tumor mass spreads from the stomach to other organs from the digestive tract. Initially, pathologic findings of the diseases of the digestive system are only observed, but significant changes are formed in the later stages of the disease affecting the entire body. The spreading tumor mass uses lymph and blood vessels, invading many tissues and organs to form new tumor mass. With each tumor’s formation, the lethality and severity of the disease increases.
According to epidemiological studies of Lauren Peirce Carcas (2014), gastric cancer was the second cause of death for all cancer related deaths across the world. It was demonstrated by a complementary study of Parisa Karimi et. Al. (2014) that the gastric cancer is the fourth type of cancer among the most common cancer types globally. In addition, it was shown by Parisa Karimi in her article published in 2014 that every year 990,000 people are found to have gastric cancer diagnostics and each year 738 lives are lost to this phenomena. Every year in the US, 17.000 people are diagnosed to have cancer again and 24.000 people die as a result of this disease. The work done is more than enough to present the significance of gastric cancer as a major health problem.
Bryan J. Dicken has shown in 2004 that early diagnosis of gastric cancer is not possible in the initial stages of its development. The diagnosis of the latter stages (when tumor formation occurs), however, is done through endoscopy. Although there is no specific and effective way to treat gastric cancer, radiotherapy and chemotherapy are used in the right conditions. In patients that chemotherapy and radiotherapy do not yield results, surgical operation is conducted. As it can be seen, there are no satisfactory methods for the diagnosis and treatment of this deadly and widespread disease. The main issue that is faced in both stages is the inability to distinguish cancer cells from healthy cells. Many studies are conducted on this matter, but the desired achievements remain to be achieved. It is with our project that we aim to effectively detect cancerous cells and exterminate them to tackle this major health issue.Were many attempts at directly targeting cancerogenous cells. The most preferred of all methods has been the utilization of the proteins on the surface of the cell membranes of cancer cells that are present in higher density in contrast to healthy cells and show antigenic properties. Fortunato Ciardiello and Giampaolo Tortora(2001), Shantaram Kamath and John K. Buolamwini(2006), and the works of many more scientists have tested out this method. However, protein to protein interaction also has certain disadvantages. If we would like to target the cancer cell with a protein, the targeted protein should be on the cell surface. Among the proteins increased in cancer cells, only a small number is located on the membrane, most of them remains dissolved in the cytoplasm.  This situation seriously limits the number of the proteins can be used for cancer cell targeting. Besides, when this technique is used the healthy cells are damaged due to the existence of the same proteins on the healthy cells, even though this existence at lesser amounts. Additionally, sensing several features will provide much more specificity compared to sensing only one feature. Sensing two or more surface proteins together is a rather difficult process.
Although, some studies have been aiming sensing two antigenic proteins increased at cancer cells at once, most of the studies have been aiming sensing only one antigenic protein. Because of this, all the methods based on protein-protein interaction have been failing to provide enough specificity for the cancer targeting. These disadvantages have been directed scientists to search for new molecules for cancer targeting. Specific gene sequences, mRNA sequences of the certain proteins, altered molecules in the microenvironment of the cancer cell and the specific micro RNA sequences are some of the molecules can be used for this purpose. In our project, we decided to use micro RNA (miRNA) molecules to eliminate cancer cells without harming the normal cells.
Micro RNAs are 20-24 nucleotides long single stranded short RNA molecules and do not code proteins. Studies have been shown that they play important roles in many events in cells of mammals and other multicellular organisms (Lin He & Gregory J. Hannon (2004). These roles include regulating cellular events such as cell cycle, metabolism, cell differentiation and apoptosis. Some of these roles are regulating cellular events (Natascha Bushati and Stephen M. Cohen(2007)). miRNA molecules become active after several processing and inhibit protein expression by binding specifically to its mRNA. miRNA binds to its specific mRNA and either prevent binding of mRNA to ribosome or cause mRNA disruption. In both cases, it prevents the expression of a particular protein.
Martin D. Jansson and Anders H. Lund showed the level of miRNA molecules is altered in cancer cells compared to normal cells just like the protein levels. The level of certain miRNA molecules are increased while the level of certain miRNA molecules are decreased in cancer cells compared to the normal body cells. A study conducted by Wilson Roa et.al (2010) showed each cancer type has a specific miRNA profile. As a result of these studies miRNA profiles of different cancer types such as gastric cancer, lung cancer and the breast cancer have been revealed. Designing a system targets only a specific miRNA cancer profile will make possible to eliminate the cancer cells without damaging the healthy cells.
Utilizing miRNA profiles specific to each cancer types instead of cell surface antigens for cancer cell recognition has some certain advantages. Micro RNA recognition will be achieved by the genetic circuit placed in the cell. DNA and RNA loci can be easily inserted inside of the cell with the help of developed molecular biology techniques (Claudia Chen and Hiroto Okayama (1987)) so miRNAs don’t have to be on the surface of the cells to be recognized by the system designed. By this, all the intended miRNA molecules can be targeted for the cancer cell recognition. More than one miRNA can be easily recognized by the designed genetic system. The system we designed for the current project works depending on the different levels of five different miRNAs.
We are aiming to detect the level of five different miRNA levels, 3 increased and 2 decreased, in gastric cancer cells when compared to normal cells by the system set. A study performed by Xiaohua Li and friends showed miR323 level increased up to 30 fold in gastric cancer cells compared to healthy cells. Zhiyu Zhang and friends showed miR21 level increased 14.8 fold in gastric cancer cells compared to the normal cells. In another study, miR373 level was increased 22 fold in gastric cancer cells (Xiaoting Zhang and friends. In the other studies, miR375 level decreased 100 fold (Ling Ding et. Al. ,2010) and miR26a level decreased 16 fold in cancer cells compared to the healthy cells (Min Deng et. Al. 2013).
In order to achieve the gastric cancer treatment we aim, the system should recognize the miRNA profile above and respond accordingly. For this purpose, we decided to use a cancer switch system, targeting cervical cancer cells HeLa, has been used and succeeded by Zhen and friends.
The system mentioned above was used by Zhen and friends for targeting cervical cancer cells HeLa  (REFERANS). Positive results were obtained from the experiments. Intended protein was highly expressed in the HeLa cells whereas the expression was very low in the health cell line HEK293T. The expression level difference between these two cells was statistically significant. Additionally, Keller Rinaudo and friends also achieved positive results by using a similar system .
Cancer switch system consists of three steps. First two steps sense the increased miRNA levels, the last  step senses the decreased miRNA levels.  The following is the detailed diagram of the system.
Figure 3: A detailed diagram of miRNA recognition system
When thoroughly examined, it will be seen that the system consists of three separate steps.
1- Production of tTA which is an activator  2- Production of mLacI, a repressor, that is a result of activity of a promoter works in the presence of   tTA  3- Production of apoptotic protein that can be repressed at transcriptional level by the mLacI
Tetracyclin transactivator (tTA) is an activator protein activates pTRE promoter (Manfred Gossen and  Hermann Bujard(1992)). LacI is a repressor binds to LacO region. When it binds, it abolishes the promoter   activity (Jonathan R. Beckwith and David Zıpser (1970)). At the first step, in a system without the miRNAs,   tTA protein is produced continuously with a constantly active CMV promoter. tTA is required to activate   the pTRE promoter at the second step. As a result of pTRE activity, mLacI is synthesized. At the third step   produced mLacI inhibits the CMV IE promoter consists a Lac operator at the 3’ end. When promoter is   inhibited, the apoptotic protein hBax cannot be produced. Thus, the system will remain off until it reaches   the aimed cells so the genetic circuit created will not harm the tissues before entering into a cell.
If this system will enter into a cancer cell, with the effect of increased and decreased miRNAs,  the ample   amount hBax protein will be synthesized and the cell will enter to the apoptosis. If the system will enter   into a healthy cell, since the proper miRNA profile is not exist, there will be no hBax production and the   cell will continue to survive. As it described in the figures above, thanks to system constructed, the cancer   cells can be driven into the apoptosis without harming the healthy cells by using miRNAs. This method will   be both specific and effective in the cancer treatment.
As it is mentioned previously, our aim in this project is to create an effective treatment option by   integrating the existing system with the newly established system. Therefore delivery of the newly   established system to the cell will be executed by the antibodies specific to cell surface antigens increased   in the cancer cells.
EpCAM cell surface proteins are overexpressed in gastric cancer cells compared to healthy gastric   epithelieal cells (Du Wenqi et al.(2009)). Additionally, anti-EpCAM antibodies strongly attach to the   EpCAM proteins (Karolina Sterzynska et al. (2012)). Therefore, we decided to use Ep-CAM proteins to   target gastric cancer cells. We are planning to deliver the system to the cells by exosomes. Lamp2b is an   exosome membrane protein used as a vehicle to introduce any protein to the exosome surface. The   desired protein can be easily introduced to the exosome surface by fusing it with Lamp2b protein   (Matthew Wood, Samira Lakhal-Littleton(2014)). For binding exosomes strongly to EpCAM molecules   increased in the gastric cancer cells, we are planning to introduce anti-EpCAM antibodies to the exosome   surface. Thus, vast majority of the exosomes, enclosing the system works based on the miRNA levels, will   bind to the gastric cancer cell surface and empty their contents to these cells. A small number of   exosomes will bind to the healthy cell surface and empty their contents to these cells. However, since the   system only works in the cancer cells, the healthy cells will not be damaged.
Figure 4: Delivery of the miRNA recognition system to the gastric cancer cells
The miRNA system entered to the cancer cells by antigen-antibody recognition will result in the production   of the hBax protein based on the miRNA levels. Even though at the much lesser amounts, some of the   systems will enter into the EpCAM cell surface protein carrier healthy gastric mucosal cells. However, for   the system activation and the hBax protein synthesis, all the miRNAs should be at the intended levels. If   any of the miRNA level is different from the intended, the system will not work. Hence, the specifity that   cannot be obtained through antigen-antibody relationship will be provided by the miRNA system and the   cancer cells will directed to destruction without harming the healthy cells.
Figure 5: miRNA recognition system in the cancer cells
If the cell that our system enters is a gastric cancer cell, the system will work and at the end the hBax   protein will be synthesized. The hBax is one of the fundamental proteins of the apoptotic pathway of the   mammal’s cell. The homodimers created by the hBax proteins will cause the cell enter to the apoptosis.   hBax proteins in the homodimer form will cause the cytochrome c to pass to the cytosol from the   mitochondria, a key organelle of the apoptotic mechanism. The cytochrome c in the cytosol will gather the   Apaf-1 and the Caspase-9 proteins and will form the Apoptosome complex. The Apoptosome complex will   initiate the other processes required for cells to enter to the apoptosis (Susan Elmore(2007)).
Figure 6: Action mechanism of hBax protein. .
Design
We decided to use two plasmids to constitute the three step system described above. These plasmids are  pTET off (first step) and pTRE (second and third steps). The designs of the unmodified initial states of   these plasmids are given below.
pTET off:
pTRE:
pTET off plasmid normally includes a TTA coding gene sequence. For the first step, we will insert the  binding site (four repeats) of the miRNA that are expected to be elevated in the cancer cell to the 3’ end of   the TTA gene sequence. Since the miRNA binding sites are short sequences, visualization after the   cloning can be difficult. To come over this difficulty, instead of inserting only the binding sites, we decided   to insert the G-block including the last part of the tTA, miRNA binding sites, and the SV40 PolyA   signal. The G-blocks can be cloned into the pTET off plasmid by using SalI and HindIII enzymes. When   the G-block was designed, restriction sites for SpeI and PstI enzymes are included but restriction sites for   EcoRI and XbaI enzymes are not included, since the pTET off plasmids already have these restriction   sites. If this G-block is successfully cloned into the pTET off plasmid, the desired restriction sites will be   formed. In addition, MluI enzyme restriction sites exist before and after the miRNA binding sites. All these   restriction sites make possible to use any miRNA in any desired system. G-blocks designed to be cloned   into the Ptet off plasmid in the first step are shown below.
The pTRE plasmid, planned to be used in the second step, includes a TRE region that tTA, produced in   the first step, can bind. However, mLacI and the miRNA binding sites for increased miRNA should also be added since both required for the next step.  When the genes were designed, we decided to put mLacI   and the miRNA binding sites on the same biobrick. We designed G-blocks in a way so they can be cloned   to the pTRE plamids with the EcoRI-BamHI enzyme restriction sites. Additionally, to be able to clone these   G-blocks into the PSB1C3, we added the RFC10 prefix region to the 5’ end and the RFC10 suffix region to   the 3’ end. The following is the design of the G-blocks to be cloned into the pTRE plasmids for the second   step.
The pTRE plasmid that is to be present in the 3rd step contains the CMV minimal promoter. This  promoter and the TRE region will be cut with the Xho1 and EcoR1 restriction enzymes and the   CMV IE promoter will be placed in its stead. We decided to order BioBricks for LacO, which will   synthesized in step 2 that mLacI will be able to bond;  DsRed, which will function as the reporter;   and binding sites for the miRNA that are expected to have decrease in concentration levels.   Under this line, the design of the final state of the modified pTRE that will be used in step 3 and   the ordered G-blocks were given.
Results
PSB1C3- LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS/ mLacI - 4 X miR373 BS/ mLacI - 4 X miR223 - 4 X miR21 BS CLONNING
In order to both produce our genes and have them in a format for part submission, we decided to first clone  the G-blocks that we ordered from IDT into the PSB1C3 plasmid.  . In the cancer switch part of our project   there is a total of 5 G-blocks. For the first step, before cloning two of the biobricks that we designed into   PSB1C3, we first had to clone them into the pTET off plasmid.  . For the G-blocks used in the second and   third step, this was not necessary and we could directly clone the genes into PSB1C3.  Therefore, when   cloning the genes in the second and third step, we cut both the G-blocks for this gene and the PSB1C3   plasmid with EcoRI and PstI restriction enzymes.  . We then planned to add the cut G-blocks into the   plasmid using T4 DNA ligase.. After ligation, the plasmids were then transformed into BL21 competent   bacteria.
In order to verify our cloning, we performed colony PCR using Verify Forward and Verify Reverse primers.  .  If our cloning was unsuccessful, we expected a PCR product size of 314 bp.  On the other hand, if our   cloning was successful, we expected a band of 1495 bp for mLacI - 4 X miR373 BS, 1503 bp for 4 X   miR223 - 4 X miR21 BS and 1178 bp for LacO2-DsRed- 4 X miR375 BS .
Based on colony PCR results, we chose clones that appeared to be positive and grew them in liquid culture  for 16 hours, followed by isolation of plasmid DNA by MiniPrep plasmid isolation.  This isolated plasmid   DNA was checked once again by enzyme cut-check.  The enzymes used for the cut-check were EcoRI and   PstI.
. RESULT FOR mLacI - 4 X miR373 BS .
.RESULT FOR mLacI - 4 X miR223 - 4 X miR21 BS.
.RESULT FOR LacO2-DsRed- 4 X miR375 BS.
pTRE- LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS/ mLacI - 4 X miR373 BS/ mLacI - 4 X miR223 - 4 X miR21 BS CLONNING
We planned to subclone the genes that are part of the cancer switch system (which we had successfully  cloned into PSB1C3) into the TRE vector.. In order to clone PSB1C3- LacO2-DsRed- 4 X miR375 BS - 4 X   miR26a BS/ mLacI - 4 X miR373 BS/ mLacI - 4 X miR223 - 4 X miR21 BS into this vector, we cut the genes   from the plasmid using EcoRI and BamHI enzymes.. We then performed ligation with pTRE vector that had   been cut with these same enzymes.  . Following ligation, we transformed these plasmids into BL21   bacteria.
To confirm our clones, we performed colony PCR using CMV forward and SV40 reverse primers. If the   cloning was not successful, we expected a PCR product size of 224 bp.  On the other hand, if our cloning   was successful we expected a band size of 1397 for mLacI - 4 X miR373 BS/ mLacI, 1405 bp for mLacI -   4 X miR223 - 4 X miR21 BS and 1079 bp for LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS.
Clones that appears to be positive by colony PCR were grown in liquid culture for 16 hours and  subsequently plasmid DNA was isolated using MinpPrep plasmid isolation.  We performed a cut-check with   the plasmid DNA as a second control.  The restriction enzymes we used in the cut-check were EcoRI and   BamHI
pCMV- LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS CLONNING:
Once we successfully cloned pTRE- LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS, we needed to   change the minimal CMV promoter found in pTRE with the CMV IE promoter.  . To achieve this, we cut   the CMV IE promoter from the pTET off plasmid using XhoI and EcoRI enzymes.  We also cut pTRE-   LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS with the same enzymes so that the piece cut from pTET   off could be ligated into pTRE- LacO2-DsRed- 4 X miR375 BS - 4 X miR26a BS.  After cutting, we   performed gel extraction to isolate the portions we needed and then performed ligation.. After ligation, we   transformed the samples into BL21 bacteria.
To confirm positive colonies, we performed both colony PCR and cut-check.  The results from colony PCR   and cut-check can be found below.  Upon inspection of the results, we see that we acheived our aim.
pTET off- tTA - 4 X miR223 - 4 X miR21 BS/ tTA - 4 X miR373 BS CLONNING
In the first step, we had mentioned that we could not directly clone the 2 G-blocks into PSB1C3.  . We   planned to first clone these genes into the pTET off vector. To acheive this, we cut both the G-blocks and   pTET off vector with SalI and HindIII enzymes. We performed ligation and then transformed the   plasmids into BL21 bacteria. After 16 hours incubation at 370C, we performed colony PCR and cut-check   to check for positive clones. Despite trying this strategy 7 or 8 times, we were unable to successfully   clone these G-blocks into pTET off.
Transfection Experiment:
In order to show that the parts of the system work, after cloning we transfected the plasmids that can be   clonned into eukaryotic cells.  .In order to show that the system worked, we had planned to use the gastric   cancer cell line, AGS.  We obtained the cell line from another lab, but we had problems culturing them.  .   Therefore, for the time being, we performed our transfection experiments with HEK293-T cells.   Below are the transfection conditions and microscope images from the transfection experiments we   performed
In the images, DsRed protein production is seen.  However, since the cell line we used is not a cancer cell  line, the amount of protein produced is small.
Protein was isolated from the cells seen in the microscope images.  Following isolation, we measured the  DsRed florescence levels of the samples... Results are listed below
