<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ATOMS-Turkiye_Project_Safety skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ATOMS-Turkiye/Project/Safety</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="headerWrapper" class="hidden-xs hidden-sm"><DIV id="header"><DIV id="navigation"><UL><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Tour" class="tour nav-item" id="tour">Tour</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project" class="project nav-item active" id="project">Project</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Results" class="results nav-item" id="results">Results</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices" class="pp nav-item" id="pp">P&amp;P</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/ATOMs" class="team nav-item" id="team">Team</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras" class="extras nav-item" id="extras">Extras</A></LI></UL></DIV></DIV><DIV id="sub-navigation-wrapper"><DIV id="sub-navigation"><UL id="project-sub" style="padding-right:350px"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Problems" class="sub-nav-item">Problems</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Ulcer" class="sub-nav-item">Ulcer</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Cancer_Switch" class="sub-nav-item">Cancer Switch</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Modeling" class="sub-nav-item">Modeling</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Safety" class="sub-nav-item">Safety</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Project/Future_Plan" class="sub-nav-item">Future Plan</A></LI></UL><UL id="team-sub"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/ATOMs" class="sub-nav-item">ATOMs</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Attributions" class="sub-nav-item">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/Collaboration" class="sub-nav-item">Collaboration</A></LI></UL><UL id="extras-sub"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras/Notebook" class="sub-nav-item">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras/Protocols" class="sub-nav-item">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Extras/Achievements" class="sub-nav-item">Achievements</A></LI><LI><A href="http://parts.igem.org/cgi/partsdb/pgroup.cgi?pgroup=iGEM2015&amp;group=ATOMS-Turkiye" class="sub-nav-item">Official Part Page</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Team/Interlab_Study" class="sub-nav-item">Interlab Study</A></LI></UL><UL id="pp-sub" style="padding-right:60px"><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Virtual_Hospital" class="sub-nav-item">Virtual Hospital</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Video" class="sub-nav-item">Video</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/SynBio_Day" class="sub-nav-item">SynBio Day</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Outreach" class="sub-nav-item">Outreach</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Street_Interview" class="sub-nav-item">Street interview</A></LI><LI><A href="https://2015.igem.org/Team:ATOMS-Turkiye/Practices/Ethics" class="sub-nav-item">Ethics</A></LI></UL></DIV></DIV></DIV><DIV id="footer"><DIV id="footer-content"><H2>Team ATOMS-Turkiye</H2><H3>atoms2015@googlegroups.com</H3></DIV></DIV><DIV id="page-wrapper"><DIV class="main"><ARTICLE class="post__article"><DIV id="content-div" style="height:200px; background-image:url(https://static.igem.org/mediawiki/2015/4/49/ATOMS-Turkiye_back5.png); padding-bottom:0px; background-size:cover"><H1 style="left:0px; right: 230px;">Safety</H1></DIV><H2 class="scroll">Suicide Switch</H2><DIV id="content-div"><P>The kill switch designed to kill the E. Coli stil remaining after the treatment is as the following:
The TetR proteins, translated from the TetR sequence following the Constitutive Promoter, will repress Ptet that allows the synthesis of TEV protease.</P><P>By attaching to the TetR protein, the externally added ATC(anhydrotetracycline) causes conformational changes. The conformationally changed TetR protein cannot bind to Ptet and the repression on Ptet and the synthesis of TEV protease is initiated. The DAMP-PEX complex, which was previously synthesized by our E. coli, is inhibited by TEV protease. Pexiganan that is seperated from DAMP digests the cell membrane and the living E. Coli will be exterminated.</P></DIV><H2 class="scroll">Questions and Solutitions</H2><DIV id="content-div"><H3>ABOUT ULCER TREATMENT</H3><H4>1-Is there a risk of E. Coli to survive under the stomach mucus layer as H. Pylori does?</H4><P>A glycoprotein layer called mucin in stomach, prevents microorganims to penetrate and survive to grow colonies under that layer. This feature protects stomach epithelium cells.</P><P>According to our researches, H. Pylori has optimal ability of attaching to mucin layer. This attachment plays a very important role in H. Pylori infection. H. Pylori has specific proteins which helps it to attach gastric epithelium and colonize at there. Whereas E. Coli doesn’t have any structures for attaching to gastric epithelium. It expected to penetrate into mucus layer but stomach mucos layer renews itself continually, so bacteria can’t hold on to this layer. In this case, E. Coli won’t be able to grow a colony in stomach. </P><H4>2- Can E.coli approach sepsis within  mix with blood in the situation that stomach has ulcer perforation?</H4><P>H. pylori is placed in the mucosal layer of the stomach and produce NH3 with the urease enzyme that it has. H. pylori produces NH3 that neutralize around acide environment that causes to dissappearance of mucosa layer of stomach.Hydrocloric acid and gastric enzymes directly contact with gastric ephitalian cells and damage to tissue in that area which mucosa layer has dissappeared.H.pylori escapes from ulcer area because in that region acidic concentration is high.The E.coli that we will send would not go to the ulcer region because there were H.pylori and most likely it would enter to mucosa layer.If the patient loses a large portion of the stomach mucosa because of ulceration already there would be sepsis and we expect a great deal increased bacteria levels in the blood.But we will apply to patients who are not in an advenced stage of disease.</P><DIV class="img-text"><P><B>FIGURE1: </B>The NH3 that was produced by helicobacter pylori,reduces acid concentration of mucosa layer and it causes escaping of Helicobacter pylori from there. The E. Coli which will taken orally, will go to place where h. pylori was.Therefore the possibility of interference from injured place to blood will be reduced by this way.(Source: Y_tambe, dual-licensewith GFDL and CC-bySA,)</P></DIV><H4>3- Does E. Coli which we send to stomach cause a food poisoning-like effect?</H4><P>We thought our E.Coli will be taken orally as probiotics.The Probiotic bacterias are microorganisms that give to body alive and prevent the target disease.The side effects of these microorganisms can cause complications in immune deficient organisms such as deterioration of metabolic activity,repression of immune system.
We can reduce the probability of occurence without  any toxic effects in stomach by determining of the dose and time intervals of E.Coli without medication to the patient.
</P><H4>4- Can the Pexiganan molecule which will be appear in stomach, can cause a problem by going intestine?</H4><P>Even though pexiganan molecule  could be dientagle in intestine, it would be inactive cause of restriction from variable sites by the Trypsin enzyme in intestine and pepsin in stomach  as below. Hence, it wouldn’t be cause a problem in intestine.</P><DIV class="img-text"><P><B>FIGURE 2: </B>http://web.expasy.org (Sequence of Pexiganan: GIGKFLKKAKKFGKAFVKIL KK)</P></DIV><H4>SOURCES:</H4><P>[1]: Helicobacterpylori: Physiologyand Genetics. Chapter 34: AdherenceandColonization
[2]: Hirmo S.,Kelm S., Iwersen M., Hotta K., Goso Y., Ishihara K., Suguri T., Morita M., Wadström T. InhibitionofHelicobacterpylori sialicacid-specifichaemagglutinationbyhumangastrointestinalmucinsandmilkglycoproteins. FEMS Immunol. Med. Microbiol. 1998;20:275–281.
[3]: Veerman E. C., Bank C. M., Namavar F., Appelmelk B. J., Bolscher J. G., NieuwAmerongen A. V. Sulfatedglycans on oral mucin as receptorsfor Helicobacterpylori. Glycobiology. 1997;7:737–743.
[4]: Hirmo S.,Artursson E., Puu G., Wadström T., Nilsson B. Helicobacterpylori interactionswithhumangastricmucinstudiedwith a resonantmirrorbiosensor. J. Microbiol. Methods. 1999;37:177–182.
[5]: Tomb JF, White O, Kerlavage AR, Clayton RA, Sutton GG, Fleischmann RD, et al. Thecompletegenomesequence of thegastricpathogen Helicobacterpylori. Nature 1997; 388: 539-547. </P><H3>ABOUT CANCER TREATMENT</H3><H4>1-What is the risk of exosomes to enter into the healthy cells rather than cancer cells?</H4><P>EpCam antigenes are over-expressed in gastric cells so our antibody covered exosomes are expected to show more affinity to cancer cells rather than normal cells. Hence EpCam is expressed in normal cells even at low amounts and this means our switch system will enter to these cells too. As our designed miRNA switch senses 5 different miRNA amounts, it can only work at 3/32 situations. If exosomes get into the normal cells, because their miRNA levels doesn’t match with cancer cells. Thus we minimized the risk of damaging normal cells and kill cancer cells effectively. </P><H4>2-This system is working in the presence of 3 plasmids in the same exosome. Nevertheless some exosomes could occur without all three plasmids. Can this cause system to work incorrectly and damage healthy cells too?</H4><P>A possibility like this could occur if 5 different plasmids are used. Yet in the application, combinating these three plasmids into one plasmid could solve this problem. As our systems are presented in only one plasmid, there won’t be any cells which got the exosomes with missing plasmids, there will be only two kinds of cells; first one which got  the system and the second one which hasn’t. Thus we could prevent it from working incorrectly.</P></DIV><H2 class="scroll">Safety Guards</H2><DIV id="content-div"><H3>3. GENETIC SAFEGUARDS: BUILDING CONTAINMENT MECHANISMS INTO SYNTHETIC LIFE </H3><P>Decades of work in closed settings, such as research labs, might suggest that engineered organisms pose little threat. So far, no bio-hazardous incidents have been traced back to engineered organisms (Schmidt and de Lorenzo, 2012). Nonetheless, if speculations correctly predict the future use of synthetic biology, the technology will scale to large industrial volumes, introduce large numbers of synthetic organisms into the environment for bioremediation, and be used in private spaces where dispersal and disposal are difficult to monitor. Innovative containment mechanisms will improve safety in open synthetic systems. Genetic safeguards operate within the synthetic organisms themselves to prevent escaped microbes from proliferating unchecked and to prevent the spread of engineered genetic material into unintended host cells. 
</P><H4>1. Containment through engineered auxotrophy</H4><P>One method for biocontainment is to engineer auxotrophic organisms that are unable to synthesize an essential compound required for their survival. Once auxotrophic microbes escape the controlled environment where the compound is supplied, they rapidly die (Figures 1A,B).</P><P>Molin and colleagues designed a DNA cassette that could function as a conditional suicide system in any healthy bacterial strain (Molin et al., 1987). In the absence of an artificially supplied growth supplement, the cassette produced Hok, a toxic protein that damages bacterial cell membranes (Gerdes et al., 1986) and kills the cells.</P><P>A pioneering containment system for bioremediation applications was published in 1991 by Contreras et al. (1991). They designed a genetic switch to kill microbes once a mission was completed (e.g., after degrading an environmental pollutant). Cells engineered to destroy the pollutant compound benzoate remained alive in the presence of that compound. Benzoate depletion activated an artificial xylS gene switch, which produced Gef, a toxic protein that functions in a similar manner as Hok (Poulsen et al., 1989). </P><P>Recently, interleukin-10 secreting auxotrophic Lactococcuslactis (Steidleretal.,2003) has been used to treat Crohn’s Disease (Braatetal.,2006). In order to prevent uncontrolled proliferation, auxotrophy was created by eliminating thymidylate synthase (thyA) (Steidleretal.,2003). The population of engineered bacteria fell below detection limits in the absence of thymidine and did not acquire functional thymidylate synthase from other bacteria in controlled experiments in pigs.</P><H4>2. Active containment through induced lethality</H4><P>Induced lethality (Figure 1C), or “kill switch” mechanisms have been engineered as genetic safeguards. The engineered organisms survive normally until an inducer signal (e.g., IPTG) is added. Induced lethality could be used clean up synthetic microbe spills without harming other cells in the environment. An early proof of concept switch was created by placing the toxic hok gene under the control of the strong and inducible lac promoter (Bej et al., 1988). Later, other toxic proteins that are homologous to Hok (Poulsen et al., 1989), such as RelF (Knudsen and Karlström, 1991) and Gef (Bej et al., 1992), were tested in lac-controlled kill switches. In microcosm studies, Knudsen and colleagues demonstrated effective IPTG-induced kill switch activation of engineered microbes in soil, seawater, and an animal model (rat intestine) (Knudsen et al.,1995). Other inducers such as heat (Ahrenholtz et al., 1994), sucrose (Recorbet et al., 1993), and arabinose (Li and Wu, 2009) have been used to activate death in engineered cells.</P><P>Recent developments in artificial cell division counters have brought us closer to timed, automatic death of synthetic cells. A set of synthetic genetic components that includes a riboregulated transcriptional cascade and a recombinase-based cascade of memory units can count up to three events (Friedland et al., 2009). These counting circuits could be designed to limit the life span of synthetic cells by linking the circuit to intracellular cell cycle-cues. Genes such as hok, relF, or gef could be added so that a toxic protein is produced after a certain number of cell cycles (Lu et al., 2009).</P><H4>3. Gene-flow barriers</H4><P>In the absence of prohibitive mechanisms, plasmids are frequently transferred between microbes through conjugation (Heuer and Smalla, 2007). Furthermore, the death of an engineered organism is not necessarily accompanied by the disappearance of its rDNA. Cell-free DNA can remain functional and transferable even after exposure to harsh conditions (Lyon et al., 2010). Thus, scientists have developed systems to prevent the uptake and inheritance of engineered genetic material.</P><P>Gene-flow barriers are created by including a killer gene in the rDNA and placing the rDNA into an immune host. Immunity from the killer gene is provided by a repressor protein that blocks killer gene expression. If unintended hosts take up the engineered DNA, the lethal gene is decoupled from immunity and the new host cell dies (Figure 1D).</P><H4>Observed failures of engineered safeguards</H4><P>Unfortunately, not all genetic safeguards are completely fail-proof. Occasionally, an engineered microbe's DNA may undergo a spontaneous mutation that destroys the genetic switch (Knudsen and Karlström, 1991) or bestows immunity against the lethal gene (Bej et al., 1988), enabling the engineered cells to propagate outside of their contained environment.</P><P>The recommended limit of engineered microbe survival or engineered DNA transmission is less than 1 cell per 108 cells (Wilson, 1993), or less than 1000 cells per 2 liters, according to the National Institutes of Health. So far, only a few of the genetic safeguards meet this limit. Synthetic biologists should consider the difficulty in meeting this standard when designing genetically-contained synthetic organisms.</P><H4>Solutions for kill switch failure</H4><P>Toxic gene cassettes are attractive because they enable scientists to potentially add a biocontainment mechanism to any synthetic organism. Thus, lethal genes are the most widely used feature of genetic safeguards. Unfortunately, the lethal gene is a central cause of safeguard failure. Under certain conditions, both deactivation and activation of lethal gene expression may exacerbate the failure of biocontainment. As engineered cells are passaged in the laboratory, or as they propagate in large bioreactors, broken genetic safeguards can gradually accumulate in the population. If the utility of the biocontainment mechanism is lost, then the synthetic organisms might survive in the environment after disposal or accidental release.</P><P>Lethal gene expression can be deactivated by spontaneous genetic mutations that arise from DNA replication error (i.e., when newly replicated DNA is not identical to its template) and DNA rearrangements (i.e., transposon mobilizations or chromosome breakage and repair) as a population of synthetic cells increases through many rounds of cell division. As a result, the population becomes non-responsive to the genetic safeguard. Knudsen and Karlström applied a classic Nobel Prize-winning approach (Luria and Delbrück,1943) to measure the rate of spontaneous mutation of a relF kill switch (Knudsen and Karlström, 1991). In several trials, cells were grown for roughly 14 divisions and treated with IPTG to activate the toxic relFgene. Up to 49 cells survived in each experiment. Poisson distribution of survival showed that spontaneous mutations deactivated relF at various time points during population growth. In a population of synthetic organisms, cells carrying a mutated kill switch might gain a growth advantage and overwhelm the population (Figure 2). Experiments have demonstrated that slowing down growth by maintaining cells in a suboptimal medium and at a lowered incubation temperature prevented the accumulation of mutations that damage the lethal gene (Knudsen and Karlström, 1991). Presumably, these measures reduce the number of mutations by preventing rapid cell division.</P><DIV class="img-text"><P><B>FIGURE 4:</B> An illustration of the accumulation of damaged genetic safeguards in a population of synthetic organisms.</P></DIV><P><B>References:</B>
Moe-Behrens, G., Davis, R., &amp; Haynes, K. (n.d.). Preparing synthetic biology for the world. Front. Microbio. Frontiers in Microbiology. 

</P></DIV><H2 class="scroll">Safety Form</H2><DIV id="content-div"><A href="https://2015.igem.org/Safety/Final_Safety_Form?team_id=1639">You can reach our Safety Form from here..</A></DIV></ARTICLE></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>