Team:ATOMS-Turkiye/Project/Ulcer new1
Tour
Project
Results
P&P
Team
Extras
Problems
Ulcer
Cancer Switch
Modeling
Safety
Future Plan
ATOMs
Attributions
Collaboration
Notebook
Protocols
Achievements
Official Part Page
Interlab Study
Virtual Hospital
Video
SynBio Day
Outreach
Street interview
Ethics
Team ATOMS-Turkiye
atoms2015@googlegroups.com
ULCER
Acid Resistance
The main goal of this part is; making the E. coli, which we used for eradicating H. Pylori, acid resistant  for living in gastric juice’s low pH (pH: 2) microenvironment.  Wild-type E. coli already has couple systems to show acid resistance until a certain pH level (pH: 5). Among those systems, the most stable-working and efficient one is known as Glutamate Dependent Acid Resistance(GDAR) system. The most important protein of Gad system is GadE, which controls the synthesis of all other proteins. By overexpressing this protein, we aim to have our bacteria resistant enough to live in gastric juice for a while.
resim
Background
Escherchia coli Natural Acid Resistance System  E. coli possesses four phenotypically distinct systems of acid resistance. These systems: 1. Glutamate dependent acid resistance system (GDAR) 2. Arginine dependent acid resistance system (ADAR) 3. Lysine dependent acid resistance system (LDAR) 4. Ornithine dependent acid resistance system (ODAR)
The most effective one of these systems is the glutamate dependent system so we decided to focus on glutamate dependent system (GDAR). The GDAR system has in the acid resistance complex pathways. We decided to do the certain pathways that how to do acid resistance with our results of research.
resim
EvgA is the most effective positive regulator of GDAR. (Efficiency: evgA>ydeO>gadE) But evgA takes part in countless number different genes regulation and cellular processes (2,6,7) and also most of this cellular processes are unclear. We cannot predict the results of the overexpression of evgA so we chose gadE and using by overexpression of gadE we can induce glutamate depended acid resistance system. Briefly talk about the impact of the GadE on the mechanism.
resim
The central activator is the LuxR-family member GadE (formerly known as YhiE). GadE binds to a 20-bp sequence called the gad box, which is located 63-bp upstream of the transcriptional start sites of gadA and gadBC. (4)GadE and GadBox are an important point for GadA and GadBC. This means that GadE and GadBox are an important for acid resistance. The basis section of acid resistance is GadE’s regulation. At least 10 different regulatory take part in this regulation.
GadE have three main activation mechanisms. The first of these is performed by evgA and ydeO. About GadE activation phase:  1. EvgS (sensor kinase) activate EvgA (response regulator) 2. YdeO and evgA are independently from each other and GadE takes an active role in transcription activation. The second GadE activation circuit includes CRP,RpoS and two AraC-like regulators, GadXand GadW.  These steps are as follows: 1. GadX and GadW, are located downstream of gadA also GadX and GadW directly activate transcription of gadE 2. GadX and GadW also bind to the gadA and gadBC gad box sequences and seem to repress the gadA and gadBC promoters. (4) GadW inhibits this GadX’s repression  Also GadX and GadW regulate indirectly GadA/BC transcriptional functions. 3. The balance of power in this circuit is influenced by cAMP and CRP,which together inhibit the synthesis of RpoS. Growth under acidic conditions reduces the concentration of cAMP in the cell.  RpoS increases GadX’s transcriptional function  The increase in GadX then stimulates transcription of GadE and also down regulates GadW.  The third activation of GadE contains TrmE and glucose. These steps are as follows: 1. The function of TrmE in the cell is not fully defined but it does have a clear effect on tRNA modification 2. TrmE and glucose increase independently GadE’s transcriptional function  We discourse GadE’s regulation and then now we will discourse GadE’s effect mechanisms. GadA/BC’s effect mechanisms to acid resistance are as follows: 1. The external pH is normally neutral but if external pH turns to acidic pH, internal pH begins to become acidic pH with HCl diffusion. 2. If the external pH=2,5 and internal pH begins pH=4.2 ± 0.1. GDAR system will activate for acid resistance. 3. GadC is a transmembrane protein. The external pH begins to change from neutral pH to acidic pH, C-plug (is GadC’s subunit) will open. Then glutamate will take inside.
resim
4. Glutamate in cells convert to GABA by GadB/C that’s glutamate decarboxylase isozymes These steps are as follows: *This step contains pyridoxal phosphate-containing enzymes that replace the α-carboxyl groups of their amino acid substrates with a proton that is recruited from the cytoplasm. *HCl that diffusion from outside dissociates H+ and Cl-. This H+ is used and CO² is released by the agency of GadA/B isozymes than this is the last stage of glutamate changes to GABA. Besides this is the most important and the last step for acid resistance 5. If H+ leaves HCl it is staying back Cl and it will be export from the transport of chlorine channel. 6. The emitted CO² is taken out by diffusion. 7. Finally, the uncovered GABA is thrown out from the transmembrane protein GadC.
Design
In this section, we targeted to overexpress GadE protein. For this purpose, while we’re desinging the gene part, we chose to combine GadE sequence with T7 promoter, a promoter which has a high rate of transcription. In addition to this, to have a controllable system, LacI protein attachable Lac operator was added between T7 and GadE gene sequence. We searched to find a vector which can lay down all those conditions and found out that peT-45 expression vector is useful. When we ordered our genes, we put RFC10 prefix site on 3’ end of GadE and also BamHI restriction enzyme recognition site for cloning to peT-45. We added RFC10 suffix site on 5’ end of the same gene sequence and also XhoI restriction enzyme recognition site for cloning to peT-45 vector, again.
resim
As shown above,we planned to clone the ordered GadE gene into peT-45 vector by using BamHI and XhoI enzymes. Final construct after this cloning is, respectively T7 promoter-Lac operator(LacO)-HisTag-GadE. Besides, a constitutive promoter called LacI promoter and LacI protein sequence in front of that, found on peT45-b vector’s another part. It is obvious that this part is IPTG-dependent, so Western Blot can be performed easily with the help of His Tag.
resim
Results
GadE-PSB1C3 CLONNING
We cloned IDT G-Blocks GadE gene into PSB1C3 vector in order to make it ready to be submitted and have many copies of it. For this purpose we digested PSB1C3 vector and GadE G-Blocks with ECoRI and PstI restriction enzymes. Then we ligated these cut genes into the plasmid by using T4 DNA Ligase . Ocurring products were transformed into BL21 competent cell strain. To check if the cloning is correct, a colony PCR was perfomed with Verify Forward and Verify Reverse primers. If the cloning isn’t made properly t the band should be 314 bp long, but if colony PCR worked, then bands should be 858 bp long. As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check for a second cloning control. We used EcorI and PstI restriction enzymes for cut-check.
resimler
GadE-pET45-b CLONNING
We cloned GadE into PSB1C3 vector successfully, then moved on cloning it into the expression vector pET45-b. For cloning into this plasmid, we removed our gene from PSB1C3-GadE plasmid with BamHI and XhoI enzymes. Then we ligated the cut gene with pET45-b which was cut with the same enzymes. We transformed ligated products into BL21 bacteria strain that we know it has T7 RNA polymerase.
To check if the cloning is correct, a colony PCR was performed with T7 Promoter Forward and T7 Terminator Reverse primers. If the cloning isn’t made properly the band should be 360 bp long, but if colony PCR worked, then bands should be 837 bp long. As the result of colony PCR, the possible right cloned colonies were incubated in liquid culture for 16 hours. After this incubation, we isolated plasmid DNA from this bacteria culture by miniprep plasmid isolation method. We controlled obtained colonies with cut-check for a second cloning control. We used BamHI and XhoI restriction enzymes for cut-check.
resimler
WESTERN BLOTTING
After cloning GadE into pET45-b successfully, we did Western Blot experiment through  N-terminal located His Tag in proteins, so we managed to show the production of required proteins.
resim
FUNCTIONAL ASSAY
To understand if the proteins we produced are functional, we designed and performed  a functional assay. We incubated these two types of bacteria in liquid culture for 13 hours at 37 C; pET45-GadE plasmid  containing BL21 bacteria and another BL21 bacteria which contains only pET45-b plasmid for negative control. At 13. our, we added 100 mM IPTG into these liquid cultures. By doing this, we removed the suppression on GadE protein expression. After adding IPTG, we incubated 3 hours more at 37 C. We prepared LB mediums with different pH values to show produced GadE proteins’ functionality. These LB mediums’ pH values are respectively 7, 5, 3,5; 2,5 and 2. We added thebacteria whichis incubated for 16 hours into LB mediums at the rate of 1:9. This means, for each pH value we added 0.5 ml liquid culture into 4.5 ml LB medium. We also added 1.5 mM Glutamat in each mix and incubated the final mix at 37 C. We made spectrophotometric measurement in 600 nm periodically for the samples that we incubated. Thus we observed how long the bacteria survives in different pH values. The measurement results are shown below.
resim
1.5 mMGlutamate / 600 nm OD / 3h 100 mM IPTG
grafik
When the given data checked, it is obvious that the produced GadE is functional and makes E. coli survive in the acidic microenvironment in comparison to negative control results. Based on the data given, the E. coli with overexpressed GadE can survive in pH 2 gastric juice for 5-9 hours.
Acid Repellency
Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Background
Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance..
Design
Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance..
Results
Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance..
Increased Motility
Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Background
Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance..
Design
Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance..
Results
Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance..
Sensing h.p.
Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Background
Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance..
Design
Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance..
Results
Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance..
Killing h.p.
Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Some description.. Background
Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance.. Background of acid resistance..
Design
Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance.. Design of acid resistance..
Results
Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance.. Results of acid resistance..
