"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Edinburgh/Phage Display Old Idea
From 2011.igem.org
outline
in silico
in the lab
in society
postscript
abstract
safety
cell surface display
phage display
achievements
modelling
cellulases (C)
cellulases (Kappa)
cellulases (MATLAB)
model comparison
genetic instability
efficiency
artificial selection
phage replication
data overview
experiments
parts
BioSandwich
lab notebook
conclusions
the team
attributions
our sponsors
human practices
biorefinery design
biorefinery appendices
life cycle analysis
interviews: overview
interviews
interview analysis
collaboration
wiki watch
Hover over words in bold italics to see explanation.
This was an old idea considered in week 1. It was dropped in favour of a different idea using phage, here .
Contents
1 About phage display
2 Project idea: a vector for phage display
3 Biopanning
4 Proofs of concept
5 Impressing the iGEM judges?
About phage display
Phage are viruses of bacteria. They consist of DNA (or RNA) wrapped in a protein coat (or "capsid"). They inject their DNA into bacteria, then build new protein coats, into which they stuff copies of their DNA. They then get these capsids ejected from the cell. This is their reproductive cycle.
An interesting and potentially useful fact is that the coding sequence for the protein is (in natural phage, anyway) physically linked to it. Thus, if the protein has an interesting property, the coding DNA can be found and analysed.
It is also possible to get bacteria in which only some of the phage genes are incorporated into the genome; this means they can produce phage capsids which are not actually infectious to other bacteria.
Project idea: a vector for phage display
BioBricks come as a sequence within a plasmid. A plasmid is a circular piece of DNA; they are useful becuase they can be easily inserted into bacteria.
My idea is to create a new plasmid in which the BioBrick is automatically fused to a phage capsid protein. These will be expelled from the cell.
Ideally, the BioBrick DNA should be packaged into the phage as well.
The system has at least 2 purposes:
Once working, it would serve as an easy way to get a protein out of a cell. It can thus be useful for many projects that require this.
If the DNA is incorporated into the capsid, a large library of DNA sequences can be screened to find a sequence that actually produces something useful (see Biopanning, below).
Biopanning
Wikipedia has an article on this, here .
It should be possible to set the system up so that the DNA packed into the capsids contains the BioBrick. It could thus be reinserted into E. coli in repeated rounds:
E. coli makes phage with BioBrick in it...
BioBricks are extracted from phage and reinserted into new plasmids.
Plasmids are inserted into new E. coli.
This repeated cycle is "biopanning".
Edit: It's not as simple as this. It turns out the normal phage that people use for this encodes its genome as single stranded DNA. So you can't just use it directly; there would need to be some sort of step where we use a PCR machine to make the other strand?
It might be possible to use T7 or somesuch (which uses double stranded DNA), though this phage is lytic (meaning the creation of phage ultimately kills the bacteria). But that might not matter if production of phage can be made inducible, so it only happens when we want it.
This is useful for directed evolution. Suppose we have a protein sequence that weakly binds some target molecule (or "ligand"). We can mutate the BioBrick and insert it into lots of E. coli, then produce lots of phage, and dump it into a vessel that has the ligand bound round its sides. After washing the vessel of excess liquid, the phage that best binds the ligand will be present in the highest amount. If we extract DNA and insert it into new E. coli, these E. coli should (on average) produce phage better at binding the ligand. This can be repeated over multiple generations.
A different approach is to just generate a library of mutated genes right at the start and use biopanning to find which was the best (with no mutagenesis between rounds). This still takes several generations, but is perhaps easier. I think something like this is used to search for useful antibodies.
Proofs of concept
If we don't have time to get actual biopanning running, we could prove that the vector works by having 2 different colonies of E. coli each producing a different protein. On their own, nothing happens, but if the two proteins meet, some visible result occurs. Thus we could show the proteins were actually getting out of the cell.
Or even just getting one protein out might be good enough.
Impressing the iGEM judges?
This project ought to be quite interesting to the judges. Making a new vector is interesting, especially if it can be made compatible with existing BioBricks. The biopanning technology has been proven in the world of research but as far as I know nobody has taken steps to enable easy biopanning with iGEM BioBricks. Thus our project could help future teams.
Retrieved from " http://2011.igem.org/Team:Edinburgh/Phage_Display_Old_Idea "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
