<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Ideas"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Ideas">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Ideas&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Ideas&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Ideas&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Ideas" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Ideas</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Ideas_whose_shambling_corpses_lurched_on_into_week_3"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Ideas whose shambling corpses lurched on into week 3</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Ideas_that_died_late"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Ideas that died late</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Ideas_that_died_early"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Ideas that died early</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Ideas_whose_shambling_corpses_lurched_on_into_week_3">Ideas whose shambling corpses lurched on into week 3</SPAN></H2><UL><LI><A href="/Team:Edinburgh/Carbon_Capture_Bacterial_Storage" title="Team:Edinburgh/Carbon Capture Bacterial Storage">Carbon Capture Bacterial Storage</A>.
</LI></UL><UL><LI><A href="/Team:Edinburgh/Phage_Reactors" title="Team:Edinburgh/Phage Reactors" class="mw-redirect">Phage as scaffolds for reactions</A> taking place outside the cell.
</LI></UL><H2><SPAN class="mw-headline" id="Ideas_that_died_late">Ideas that died late</SPAN></H2><UL><LI><I>E. coli</I> lacks a Type II secretion system. (This is one of the ways bacteria can export a protein.) Add one! Test by export of a protein with signal added.
</LI><LI><B>Comment from CF</B>: actually, <I>E. coli</I> does possess a type II secretion system but it is normally inactive. When activated, it leads to secretion of a chitinase. You could use our BRIDGE system to make the necessary genetic modifications to have this pathway switched on in normal growth.
</LI></UL><UL><LI> Solvent tolerance
</LI></UL><UL><LI> Universal Biosensor (we discussed this yesterday but wasn't on the list): Can we abstract out the common parts of all biosensors and engineer one that might be more or less universal in nature.
</LI><LI> BioBricks needed: For testing, genes to make one antibody to a known ligand, as well as an output (signal transducing) domain, a promoter that it induces, and a response gene.
</LI></UL><UL><LI> Apply the principles of synthetic biology to 'innate immune response'- Binding of bacterial components to a family of Toll-like receptors (TLRs) activates the cells of the immune system but an exaggerated response may lead to systemic inflammation and sepsis has been looked at previously. Done by designing a feedback loop...
</LI></UL><UL><LI>Digital processing using DNA.&quot;Synthetic polymer systems could allow information densities several million times higher than current systems.&quot; This <A href="http://www.sciencedaily.com/releases/2010/06/100629081750.html" class="external text" rel="nofollow">link</A> is an article on the science daily website. Really good potential for this idea.
</LI></UL><UL><LI> Something with phage-assisted continuous evolution (PACE, see <A href="http://www.nature.com/nature/journal/v472/n7344/full/nature09929.html" class="external text" rel="nofollow">here</A>). This isn't related to phage display; instead it's a really interesting way of speeding up evolution drastically. It relies on having a certain gene (pIII) in the bacteria; this gene already exists in the Registry (<A href="http://partsregistry.org/wiki/index.php/Part:BBa_K415108" class="external text" rel="nofollow">BBa_K415108</A>) which is good. We then need to devise a way to link the gene we want to evolve to the rate of production of pIII.
</LI></UL><UL><LI> Combine phage display with BioBrick vectors. Create a new vector that attaches the product of a BioBrick to a phage capsule protein. As proof of concept, combine this with random mutagenesis of some protein that attaches to some ligand, and screen for phage that attach the ligand. These phage will differ in their DNA sequences, but multiple rounds of reinsertion (a.k.a. biopanning) into the vector will find a winner. This is a form of directed evolution. Or as simpler proof of concept, just get phage display working with GFP or somesuch. This project is quite ambitious yet has achievable subgoals.
</LI><LI> I wrote up a longer version of this proposal, <A href="/Team:Edinburgh/Phage_Display" title="Team:Edinburgh/Phage Display">here</A>.
</LI><LI><B>Comment from AC:</B> I don't really like this idea any more; it seems too complicated. See my other idea for a simpler phage project.
</LI></UL><UL><LI> Phage- based TB test (or to test other diseases)- MClee
</LI></UL><UL><LI> Extension to the phage display idea, we could turn it into synthetic antibodies library to screen for useful antibodies/ ligand selection process for protein purification/ ELISA / Phage based detection and measurement of small molecules- probably useful for food analysis, like detecting water soluble vitamins or drug residue or chemical contaminants. MClee
</LI></UL><UL><LI> Phage-based water purification device/Phage-based detection kit for V. cholerae/ Phage based typing scheme for any other bacteria (eg, Salmonella) Sylvia &amp; Mclee
</LI></UL><UL><LI> Microbial cell circuit. Sylvia
</LI><LI> I expanded this idea and wrote it up <A href="/Team:Edinburgh/Networking" title="Team:Edinburgh/Networking">here</A> - Lukasz
</LI><LI><B>Comment from Tosif:</B><A href="http://www.nature.com/nature/journal/v469/n7329/full/nature09565.html" class="external text" rel="nofollow">This</A> is very similar, these guys made NOR gate and then made an XOR by connecting the NORs. I am looking into the possibility of doing something similar for a very basic neural netowork.
</LI></UL><UL><LI> This is an interesting paper. It's not necessarily related to a specific topic but to the advancement of biology in general and the implications of that. <A href="http://www.design.philips.com/philips/shared/assets/design_assets/pdf/nvbd/march2011/BiologicalAge.pdf" class="external free" rel="nofollow">http://www.design.philips.com/philips/shared/assets/design_assets/pdf/nvbd/march2011/BiologicalAge.pdf</A> could be interesting for ideas around human aspects.
</LI></UL><UL><LI> (From Tuesday's Brainstorming Roundtable) Bioremediation; using the bacterial cell envelope to suck up toxic metals, or using internal enzymes to convert toxic metals into less toxic forms.
</LI><LI> Disadvantage: it's already been done, e.g. see <A href="http://dx.doi.org/10.1021/es0000652" class="external free" rel="nofollow">http://dx.doi.org/10.1021/es0000652</A></LI><LI> BioBricks needed: Genes coding for a more dense, negatively charged lipopolysaccharide? Reductases for internal detoxification?
</LI><LI> Peking seem to have done something like this <A href="https://2010.igem.org/Team:Peking/Project/Bioabsorbent" class="external text" rel="nofollow">last year</A>.
</LI></UL><H2><SPAN class="mw-headline" id="Ideas_that_died_early">Ideas that died early</SPAN></H2><UL><LI> Last year's winner used DNA as a scaffold to latch proteins onto, via DNA binding domains. Could two such domains be used to link two replicons together, for purposes of increasing recombination frequency?
</LI></UL><UL><LI>  Create a juxtacrine signalling pathway for <I>E. coli</I>. Probably impossible due to lipopolysaccharide. (Juxtacrine signalling is signalling by physical contact.)
</LI><LI><B>Comment from CF</B>: there have been some bizarre recent papers about bacteria being connected together by 'nanotubes', as well as a growing body of literature about electrically conductive pili (nanowires), but I'm not sure either of these systems is well enough characterized yet to form the basis of a project.
</LI><LI><B>Comment from LK</B>: I think this idea is similar to neural networks - <A href="http://en.wikipedia.org/wiki/Neural_network" class="external free" rel="nofollow">http://en.wikipedia.org/wiki/Neural_network</A> - if that was possible to model using <I>E. coli</I>, it would be awesome.
</LI></UL><UL><LI> (Chris French's idea) A sensor based on fusion of antibody domain to a signal transducing domain.
</LI></UL><UL><LI> Use the BRIDGE protocol to edit the chromosome of <I>E. coli</I> to report when a plasmid has been successfully introduced, e.g. by DNA binding proteins that would recognise the plasmid (either known sequences or common plasmid features like ORI) and cause some sort of effect.
</LI></UL><UL><LI> Global Transcription Machinery Engineering (see e.g. <A href="http://www.sciencedirect.com/science/article/pii/S1096717606001248" class="external text" rel="nofollow">Alper <I>et al</I>, 2007</A>) - use error-prone PCR to generate mutated copies of the <I>rpoD</I> gene (which codes for sigma-70, the key initiator of bacterial transcription). Transform these into cells. Because these mutant genes will affect transcription rates of many genes, some interesting phenotypes may be seen. Screen for e.g. solvent tolerance, heat and cold tolerance, etc. Interesting ones can be sequenced and biobricked. This project has natural quantitative aspects (how well can cells with various rpoD genes cope in various conditions) and may potentially generate multiple biobricks of use to others in the future.
</LI><LI><B>Comment from AC</B>: Alas, this may be impossible due to a <A href="http://www.wipo.int/patentscope/search/en/WO2009061429" class="external text" rel="nofollow">patent</A> on the technique.
</LI><LI> But Chris says it's probably OK anyway.
</LI></UL><UL><LI> Instead of the above, clone sigma factors from various species into <I>E. coli</I> and screen for useful phenotypes.
</LI></UL><UL><LI> From our first brainstorm:
</LI><LI> Bio-batteries
</LI><LI> Bio-etching - i.e. making the bacteria eat through silicone plates
</LI><LI> Bio-tweeting
</LI><LI> Bio-sensors for soil acidity
</LI><LI> Volcanic ash detection
</LI></UL><UL><LI> Our mouths are a host to a number of bacteria (especially in the morning), these bacteria cause lots of dental problems like caries, bad breath etc. Can we have genetically modified bacteria to help prevent dental caries or/and bad breath? I know there are parts available for fruity smells, not sure how one would do the former, also containment might be a problem (containment seems to be a big problem in general, would a biological self-containment mechanism be possible? ).
</LI></UL><UL><LI> Vaccines- Utilize design principles of synthetic biology, find a new approach. Possibly too complicated for a three month project, but worth a look. <A href="http://centerforvaccineethicsandpolicy.wordpress.com/2011/06/05/synthetic-biology-innovative-vaccines/" class="external text" rel="nofollow">Article link</A></LI><LI> Perhaps we could use synthetic biology to speed up the time it takes to make a vaccine. Previous research was presented at past synthetic biology conferences, the same one Alistair is at. Scientists have previously created a 'weakened versions of the polio virus by giving them a coat protein that contains the same sequence of amino acids as the natural virus, but using a gene with &quot;unfamiliar&quot; variants of the genetic code to make it.'They changed 700 codons, however they overdid it. Perhaps there is capacity to look into this further, explore different diseases...
</LI></UL><UL><LI> Various regulatory network motifs based on genetically engineered stem cells. Previously artificial tissue homeostasis system where genetically engineered stem cells maintain indefinitely a desired level of pancreatic beta cells despite attacks by the autoimmune response were created
</LI></UL><UL><LI> Not really an idea but this talk might lead to some <A href="http://www.ted.com/talks/angela_belcher_using_nature_to_grow_batteries.html" class="external free" rel="nofollow">http://www.ted.com/talks/angela_belcher_using_nature_to_grow_batteries.html</A></LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Ideas">http://2011.igem.org/Team:Edinburgh/Ideas</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Ideas" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Ideas" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Ideas&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Ideas&amp;oldid=57384" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>