<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_tech_Hints"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/tech/Hints">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/tech/Hints&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/tech/Hints&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/tech/Hints&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/tech/Hints" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/tech/Hints</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV><DIV class="hint" id="igem_hint"><I>iGEM</I> is the International Genetically Engineered Machine competition. Each year during (Northern hemispheric) summer, undergraduate-led teams from all over the world compete to produce biological devices with standard, reusable parts.</DIV><DIV class="hint" id="phage_hint"><I>Bacteriophage</I> are viruses that infect bacteria. They are the world's most numerous biological replicators, believed to outnumber bacteria 10 to 1.</DIV><DIV class="hint" id="bioreactor_hint">A <I>bioreactor</I> (as defined by Edinburgh) is a microscopic system containing multiple enzymes that work together in synergy to catalyse some reaction.<SMALL>Note though that others use the term to mean a large container in which biological processes occur.</SMALL></DIV><DIV class="hint" id="synergy_hint">Two or more enzymes act in <I>synergy</I> if their efficiency is increased in the presence of each other.</DIV><DIV class="hint" id="flagellum_hint">A <I>flagellum</I> is a long appendage that allows bacteria to swim. Not every species has them, and some have more than one per cell.</DIV><DIV class="hint" id="m13_hint"><I>M13</I> is a filamentous (long tubular) virus that infects <I>E. coli</I> bacteria but does not kill them.</DIV><DIV class="hint" id="cellulase_hint"><I>Cellulases</I> are a group of enzymes that break down the common plant material cellulose, ultimately producing glucose sugar.</DIV><DIV class="hint" id="biorefinery_hint">A <I>biorefinery</I> is any system that converts biomass into useful products.</DIV><DIV class="hint" id="promoter_hint">A <I>promoter</I> is a DNA sequence causing the following sequence to be transcribed into RNA.</DIV><DIV class="hint" id="rbs_hint">A <I>ribosome binding site</I> is a region of RNA where the ribosome binds to start translation into protein.</DIV><DIV class="hint" id="p8_hint">The <I>pVIII</I> protein is the major coat protein of M13, present in about 2,700 copies per phage.</DIV><DIV class="hint" id="leader_hint">A <I>leader</I> or <I>signal</I> peptide is a sequence of amino acids present at the start of a protein which instructs the cell to send it to some specific location.</DIV><DIV class="hint" id="linker_hint">A <I>linker</I> is a chain of amino acids used to connect two protein domains while giving them space to fold.</DIV><DIV class="hint" id="fusion_hint">A <I>fusion protein</I> contains the amino acid sequences of two or more proteins, effectively joining them together.</DIV><DIV class="hint" id="inp_hint"><I>Ice Nucleation Protein</I> is a protein isolated from the plant pathogen <I>Pseudomonas syringae</I>. It is present on the outer membrane of the bacteria and causes the formation of ice crystals, which harms the plant, helping the bacteria attack it. However, we will use it as a carrier to get other proteins displayed on the cell surface.</DIV><DIV class="hint" id="mals_hint"><I>malS</I> is a gene coding for amylase, which breaks down starch into its component glucose sugars.</DIV><DIV class="hint" id="p3_hint">The <I>pIII</I> protein is a minor coat protein of M13, present in 5 copies at the end of the phage.</DIV><DIV class="hint" id="oligo_hint">An <I>oligonucleotide</I> is a short piece of DNA or RNA. DNA oligonucleotides are often used as primers for PCR.</DIV><DIV class="hint" id="pcr_hint">The <I>Polymerase Chain Reaction</I> is a method for amplifying a section of DNA, producing many copies with the same sequence.</DIV><DIV class="hint" id="lic_hint"><I>Ligation Independent Cloning</I> is a method of assembling DNA fragments into a plasmid, which does not involve a ligase enzyme. Instead, long overhangs of homologous DNA are used to join the fragments, and the transformed bacterium repairs the phosphate backbone.</DIV><DIV class="hint" id="f_hint">The <I>F plasmid</I> is present in some bacteria, and contains genes coding for a sex pilus, which the plasmid uses to spread itself to other bacteria. The M13 phage requires the pilus to be present in order to infect <I>E. coli</I>.</DIV><DIV class="hint" id="codon_hint">A <I>codon</I> is a sequence of three DNA bases, coding for one amino acid in a protein. Since there are four types of base, there are 64 possible codons. Three of these are <I>stop codons</I> which terminate translation (protein production).</DIV><DIV class="hint" id="amylase_hint">The <I>amylase</I> family of enzymes break down starch into its component glucose sugars.</DIV><DIV class="hint" id="biobrick_hint">A <I>BioBrick</I> is a reusable biological part, containing a DNA sequence. BioBricks are designed to be joined together to produce new (composite) BioBricks.</DIV><DIV class="hint" id="ps_hint"><I>Pseudomonas syringae</I> is a species of Gram-negative bacteria and a pathogen of several plant species.</DIV><DIV class="hint" id="ec_hint"><I>Escherichia coli</I> is a species of Gram-negative bacteria commonly found in the mammalian gut. Most strains are harmless or even beneficial (though this is not a good reason to consume them). They have been well-studied and are often used in genetic engineering.</DIV><DIV class="hint" id="help_hint">Help! Help! We're being held prisoners in an iGEM laboratory! They make us do minipreps and agarose gels all day long!</DIV><DIV class="hint" id="c_hint">The <I>C terminal</I> of a protein is one end of the protein chain. That part of the protein is coded for by the <B>end</B> of the relevant DNA sequence.</DIV><DIV class="hint" id="n_hint">The <I>N terminal</I> of a protein is one end of the protein chain. That part of the protein is coded for by the <B>start</B> of the relevant DNA sequence.</DIV><DIV class="hint" id="sorbitol_hint"><I>Sorbitol</I> is a naturally occuring derivative of glucose. It can be used as a sweetener.</DIV><DIV class="hint" id="hfcs_hint"><I>High Fructose Corn Syrup</I> is a syrup containing a high level of fructose, making it very sweet.</DIV><DIV class="hint" id="metagenome_hint">The <I>metagenome</I> is all of the genetic information contained in the environment.</DIV><DIV class="hint" id="gfp_hint"><I>Green Fluorescent Protein</I> is a protein which emits green light if illuminated with a blue light. It is often used in genetic engineering as a reporter; i.e. to prove that a system is working.</DIV><DIV class="hint" id="fitness_hint"><I>Fitness</I> is the ability of an organism to compete with other organisms and successfully reproduce. If an organism outcompetes other organisms it is said to have greater fitness. Such an organism will spread.</DIV><DIV class="hint" id="algorithm_hint">An <I>algorithm</I> is a method of computing some function. It consists of a finite number of steps which you need to follow, to get the desired result.<SMALL>Examples: The annoying program inside a PCR machine, a flight checklist, or a pasta recipe.</SMALL></DIV><DIV class="hint" id="recombination_hint"><I>Recombination</I> is a natural process in which different pieces of DNA are joined together. Complex organisms use this process to combine the DNA of two parents during sex, while simpler organisms use it to repair damaged DNA by replacing it with an undamaged copy.</DIV><DIV class="hint" id="cellulose_hint"><I>Cellulose</I> is a tough plant material composed of chains of glucose sugar. Unlike starch (which is also a glucose chain) it is difficult to digest or degrade, so making usable sugar from it is hard.</DIV><DIV class="hint" id="starch_hint"><I>Starch</I> is an easily degradable chain of glucose sugars used by plants as a food store.</DIV><DIV class="hint" id="selectionpressure_hint">In evolution, a <I>selection pressure</I> exists when some external factor causes some members of a species to outcompete others, due to their having a genotype that is more suitable for the environment.</DIV><DIV class="hint" id="synonymouscodon_hint">In genetics, <I>synonymous codons</I> are different 3-base sequences that code for the same amino acid. For example, CAA and CAG both code for glutamine. By using synonymous codons, different DNA sequences can code for the same protein.</DIV><DIV class="hint" id="plasmid_hint"><I>Plasmids</I> are small circular pieces of DNA which replicate independently of the main bacterial chromosome. They are relatively easy to manipulate and so are used in genetic engineering.</DIV><DIV class="hint" id="artificialselection_hint"><I>Artificial selection</I> is the process where humans choose or influence which members of a species reproduce the most, thus altering the genetic makeup of the population.</DIV><DIV class="hint" id="rfc10_hint"><I>RFC10</I> is the original BioBrick standard. It specifies how restriction sites are to be placed in flanking positions around the BioBrick. Most iGEM-related parts are made in this format.</DIV><DIV class="hint" id="homology_hint">Two sequences of DNA have <I>homology</I> if the same sequence is present in both of them. Homology-based assembly relies on using base-pairing to join two pieces of DNA which have homologous ends, the start of one piece matching the end of the other.</DIV><DIV class="hint" id="frameshift_hint">A <I>frameshift</I> mutation in a gene involves the addition or deletion of a non-multiple of three bases. Because triplets of bases (codons) code for amino acids, subsequent codons will be &quot;out of frame&quot; and will code for the wrong amino acids.</DIV><DIV class="hint" id="synbio_hint"><I>Synthetic biology</I> is a broad range of disciplines, unified by the desire to take an engineering approach to biology. It is like genetic engineering, only more so.</DIV><DIV class="hint" id="presbyterian_hint">A <I>presbyterian</I> church is one governed by elected elders. At the highest level, policy is decided by a general assembly of these elders.</DIV><DIV class="hint" id="switchgrass_hint"><I>Switchgrass</I> (binomial name: <I>Panicum virgatum</I>) is a grass that can be grown on land that is unsuitable for normal crops, to produce high yields of cellulose.</DIV><DIV class="hint" id="bsr_hint"><I>Blue skies research</I> is research with no immediate practical use foreseen, though such uses often turn up later.</DIV><DIV class="hint" id="cytoplasm_hint">The <I>cytoplasm</I> is the internal volume of a cell; essentially everything that is contained within the cell's membrane(s).</DIV><DIV class="hint" id="stochastic_hint">A <I>stochastic</I> system is non-deterministic, meaning that the future state of the system cannot be predicted from the previous state, i.e. there is some random element involved.</DIV><DIV class="hint" id="agentbased_hint">An <I>agent-based</I> system is composed of many agents. Agents can perform actions of a varying level of complexity. Often, representing a system as many agents lets the programmer model complex behaviour using simple rules.</DIV><DIV class="hint" id="syntax_hint">The <I>syntax</I> of a (programming) language is the set of rules that result in working programs.</DIV><DIV class="hint" id="rk_hint">The <I>Runge-Kutta</I> methods are ways of generating an approximately correct solution to an ordinary differential equation.</DIV><DIV class="hint" id="deterministic_hint">A system is <I>deterministic</I> if future states of the system can be perfectly predicted from past states. Such a system does not incorporate any genuine randomness.</DIV><DIV class="hint" id="intractable_hint">A program is <I>computationally intractable</I> if it is <I>possible</I> to compute its output, but it takes an awful lot of time - e.g. exponential on the length of input.</DIV><DIV class="hint" id="adsorption_hint"><I>Adsorption</I> is the process in which molecules bind to a surface.</DIV><DIV class="hint" id="hydrolysis_hint"><I>Hydrolysis</I> is a type of chemical reaction in which water molecules are split up into a proton (H<SUP>+</SUP>) and a hydroxide (OH<SUP>−</SUP>) during the process of breaking down some other substance.</DIV><DIV class="hint" id="empirical_hint">To call some piece of knowledge <I>empirical</I> is to say that it is derived from data and evidence, rather than purely theoretical considerations.</DIV><DIV class="hint" id="cellobiose_hint"><I>Cellobiose</I> is a type of sugar containing two molecules of glucose bonded together in a certain way. There are other such sugars, such as maltose.</DIV><DIV class="hint" id="mm_hint">The <I>Michaelis-Menten</I> equation is a relatively simple formula predicting the rate of an enzymatic reaction from the concentration of the substance upon which the enzyme acts.</DIV><DIV class="hint" id="periplasm_hint">In bacteria that have two membranes, the <I>periplasm</I> is the space between them.</DIV><DIV class="hint" id="cryptic_hint">Genes are said to be <I>cryptic</I> if they are present in the genome but cannot normally be activated.</DIV><DIV class="hint" id="brownian_hint"><I>Brownian motion</I> is a type of motion where, at any time, a particle can move in any direction. It is also called a &quot;random walk&quot;.</DIV><DIV class="hint" id="assay_hint">An <I>assay</I> is a test of some biological agent's presence or activity.</DIV><DIV class="hint" id="pfd_hint">A <I>process flow diagram</I> is an engineering design tool used to show plant layout and connections between components.</DIV><DIV class="hint" id="insilico_hint">The phrase <I>in silico</I> was coined to refer to the study of biological processes in computer models, as opposed to <I>in vivo</I> (in life) or <I>in vitro</I> (in a test tube).</DIV><DIV class="hint" id="hemicellulose_hint"><I>Hemicellulose</I> is a weak sugar-based polymer found in plants. There are many possible forms.</DIV><DIV class="hint" id="lignin_hint"><I>Lignin</I> is a tough plant material quite resistant to degradation by enzymes, though some species of fungi and bacteria can slowly degrade it.</DIV><DIV class="hint" id="fractionation_hint"><I>Fractionation</I> is a separation technique whereby mixtures are divided into a number of smaller components.</DIV><DIV class="hint" id="depolymerisation_hint"><I>Depolymerisation</I> is the chemical reaction which results in a polymer chain being broken up into monomer units.</DIV><DIV class="hint" id="crystallisation_hint"><I>Crystallisation</I> is a chemical technique that separates solids and liquids using mass transfer.</DIV><DIV class="hint" id="aqueous_hint"><I>'Aqueous'</I> means 'in a solution of water'.</DIV><DIV class="hint" id="distillation_hint"><I>Distillation</I> is a technique for separating chemicals based on differences in boiling points.</DIV><DIV class="hint" id="hydrouspyrolysis_hint"><I>Hydrous pyrolysis</I> is a chemical engineering technique whereby organic compounds are heated to high temperature with water.</DIV><DIV class="hint" id="ode_hint"><I>Ordinary differential equations</I> (ODEs) are differential equations in which the dependent variable is a function of a single independent variable.</DIV><DIV class="hint" id="compcomplexity_hint">The <I>computational complexity</I> of an algorithm is the type of relationship between the length of input to the program and the number of operations it needs to perform. This is usually proportional to the program's runtime. For example, <I>logarithmic</I> computational complexity is good, whereas <I>exponential</I> is very, very bad.</DIV><DIV class="hint" id="seed_hint">The <I>random seed</I> is a number used to initialise a <I>pseudorandom number generator</I>. Initialising the same generator with a defined seed means that the generated numbers will be consistent between runs of a program.</DIV><DIV class="hint" id="polycistronic_hint">An RNA sequence is <I>polycistronic</I> if it codes for more than one protein.</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/tech/Hints">http://2011.igem.org/Team:Edinburgh/tech/Hints</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/tech/Hints" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/tech/Hints" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/tech/Hints&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/tech/Hints&amp;oldid=133565" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>