<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Phage_Reactors_1_0"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Phage_Reactors_1.0">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Phage_Reactors_1.0&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Reactors_1.0&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Reactors_1.0&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Phage_Reactors_1.0" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Phage Reactors 1.0</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P><B>Update: <A href="/Team:Edinburgh/Phage_Reactors_2.0" title="Team:Edinburgh/Phage Reactors 2.0" class="mw-redirect">Version 2.0</A> of this page now exists.</B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Technical_notes"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Technical notes</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Problems"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Problems</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#An_example_complete_system"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">An example complete system</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Alternatives"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Alternatives</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Two_options.2C_originality.2C_and_MIT_2010"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Two options, originality, and MIT 2010</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#A_modelling_component"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">A modelling component</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Testing"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Testing</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Other_useful_links"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Other useful links</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P>There exist situations where:
</P><UL><LI> several enzymes are needed to produce the desired product
</LI><LI> these enzymes work synergistically
</LI><LI> these enzymes must work outside the cell
</LI></UL><DIV class="thumb tright"><DIV class="thumbinner" style="width:305px;"><DIV class="thumbcaption">This is what we want to build: phage with enzymes (coloured circles) fused to the protein coat.</DIV></DIV></DIV><P>This proposal allows for the construction of scaffolds containing several enzymes, as well as ensuring that this complex is exported from the cell. This is accomplished by using phage as the scaffold, and attaching enzymes to it by protein fusion.
</P><P>There are probably many uses of an external reaction scaffold, so <B>this technique is fairly general</B> and could hopefully be used for many purposes, just by swapping in the correct BioBricks. One example is a <A href="http://en.wikipedia.org/wiki/Cellulosome" class="external text" rel="nofollow">cellulosome</A>, which is a complex of cellulolytic enzymes that can be used to turn cellulose (tough plant material) into sugar, which can then be fermented into fuel. This is something Chris is quite expert on, so could be a good fit for the lab.
</P><H2><SPAN class="mw-headline" id="Technical_notes">Technical notes</SPAN></H2><P>The project would probably use phage M13, which is non-lytic (does not kill the bacteria). The major coat protein (p8 aka pVIII) already exists in the Registry as &lt;partinfo&gt;BBa_M13008&lt;/partinfo&gt;, although DNA is not available. The entire genome is known (e.g. <A href="http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/sequences/m13ko7.txt" class="external text" rel="nofollow">here</A> is the entire M13KO7 genome) so primers can be designed. New England Bioworks <A href="http://www.neb.com/nebecomm/tech_reference/protein_tools/phdFaq.asp#1.6" class="external text" rel="nofollow">claims</A>:
</P><DL><DD><I>&quot;The major coat protein pVIII is present at ~2700 copies per virion, of which ~10% can be reliably fused to peptides or proteins.&quot;</I></DD></DL><P>Ideally, we wouldn't have to work with infectious phage at all. Apparently, if M13 lacks a viable pIII protein it is non-infectious (so claim <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1221525/pdf/11104694.pdf" class="external text" rel="nofollow">Cebe and Geiser, 2000</A>).
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:279px;"><DIV class="thumbcaption"><I>E. coli</I> with a pIII-deletion phage, forming what are essentially artificial pili.</DIV></DIV></DIV><P>Interestingly, MIT 2010 <A href="https://2010.igem.org/Team:MIT_phage_background" class="external text" rel="nofollow">made use of the fact</A> (from <A href="http://www.sciencedirect.com/science/article/pii/S002228369892006X" class="external text" rel="nofollow">Rakonjaca and Model, 1998</A>) that if pIII is deleted, very long phage particles will be formed and tethered to the membrane; this may be acceptable or even positive, if it anchors the bacteria to the substrate it's meant to use.
</P><P>Phage itself can be ordered if needed, e.g. <A href="http://www.neb.com/nebecomm/products/productN0315.asp" class="external text" rel="nofollow">here</A>. MIT 2010 used a phagemid with a pIII deletion, available from <A href="http://www.progen.de/en/hyperphage-m13-k07-piii-1x2ml.html" class="external text" rel="nofollow">Progen Biotechnik</A> (called hyperphage).
</P><P>We should be able to manufacture our own phagemids using &lt;partinfo&gt;BBa_K314110&lt;/partinfo&gt;, which is known to work! (A phagemid is a plasmid that contains both a plasmid-type origin of replication, but also a phage-type rolling circle origin so it can form single stranded DNA which is packaged into the phage capsid. The &lt;partinfo&gt;BBa_K314110&lt;/partinfo&gt; part is exactly that: a phage-type rolling circle origin.)
</P><P>Proteins can be fused to pVIII at its amino terminal (i.e. 5' end in the DNA), according to <A href="http://www.utoronto.ca/sidhulab/pdf/08.pdf" class="external text" rel="nofollow">Weiss and Sidhu, 2000</A>.
</P><P>To attach several different proteins to the resulting phage, different fusions can be created and all of them expressed on a plasmid. It may be simplest if this is a different plasmid from the phagemid, though plasmid compatibility will have to be kept in mind.
</P><P>Chris notes that the presence of repeated sequences on a plasmid can lead to genetic instability, but this will not be a problem in lab strains, which lack an important recombination enzyme. As for the use of this technology in industry, it will be possible to overcome this problem simply by synthesising coding sequences with as many altered (but synonymous) codons as possible. The (non-iGEM?) group <A href="http://partsregistry.org/cgi/partsdb/pgroup.cgi?pgroup=course&amp;group=MIT%2020.109%20Spring07" class="external text" rel="nofollow">MIT 20.109 Spring07</A> seem to have thought along these lines, e.g. compare &lt;partinfo&gt;BBa_M13008&lt;/partinfo&gt; with &lt;partinfo&gt;BBa_M31281&lt;/partinfo&gt;.
</P><P>(An alternative is just to attach different enzymes to different coat proteins; pIII is known to work; maybe one of the other coat proteins as well...)
</P><P>One thing to bear in mind is that the expression levels will need tuning; we need to make enough fusion proteins that some of them get onto the phage coat, but not so many as to gum up the works entirely.
</P><H2><SPAN class="mw-headline" id="Problems">Problems</SPAN></H2><UL><LI> Because M13 is not lytic, the pVIII protein includes a Sec leader sequence which it uses to get to the bacterial membrane. Our fusion would need to get to the membrane somehow as well.
</LI></UL><UL><LI> The pVIII mature protein is fairly small (50 residues) so whatever we fuse to it will dwarf it. It may not be able to get through the membrane channel that M13 creates. <B>This may be a fatal problem;</B> the literature seems to take a dim view of attaching large proteins to pVIII, e.g. <A href="http://www.sciencedirect.com/science/article/pii/S0022283699934654" class="external text" rel="nofollow">Sidhu <I>et al</I> (2000)</A> say &quot;In our experience, most large proteins display well below one copy per phage particle.&quot;
</LI></UL><UL><LI> It seems fusion to pIII works better, but there are only 5 copies of that on the phage, which sort of defeats the whole purpose...
</LI></UL><UL><LI><B>Can these problems be solved by other techniques or with other phage?</B></LI><LI> T7 is lytic so does not need Sec, but it is <A href="http://www.emdchemicals.com/showBrochure/200912.294.ProNet.pdf" class="external text" rel="nofollow">claimed</A> to be able to carry only 0.1 to 1 large protein per phage...
</LI><LI> It may be easier to fuse stuff to pIII but there are only five copies per phage.
</LI><LI><A href="http://www.springerlink.com/content/y206np5022357m37/" class="external text" rel="nofollow">Thammawong <I>et al</I> (2006)</A> report using the Tat pathway to get a (<A href="http://www.uniprot.org/uniprot/P35613" class="external text" rel="nofollow">385 aa?</A>) protein attached to M13 via pVIII.
</LI><LI> It may be possible to make pVIII with a small &quot;zipper&quot; protein attached to it; we can then attach another zipper to the protein of interest (i.e. the cellulase) and export it to the periplasm by whatever means necessary. The two will join together in the periplasm, and so will not need to pass through the membrane together. MIT 2010 <A href="https://2010.igem.org/Team:MIT_phage_results" class="external text" rel="nofollow">seem to have accomplished the first step</A>. The basic idea is similar to that of <A href="http://www.sciencedirect.com/science/article/pii/S0378111905000764" class="external text" rel="nofollow">Paschke and Höhne (2005)</A> (see Figure 1) and <A href="http://www.sciencedirect.com/science/article/pii/S0022283609014624" class="external text" rel="nofollow">Wang <I>et al</I> (2010)</A>.
</LI><LI> Actually, this might be impossible if the phage exit channel blocks the protein of interest from interacting with pVIII. Does it?
</LI></UL><UL><LI><B>Currently the Tat solution or the &quot;zipper&quot; solution look the most promising but there is no doubt that complete failure is a possibility.</B></LI></UL><H2><SPAN class="mw-headline" id="An_example_complete_system">An example complete system</SPAN></H2><P>A complete system could contain:
</P><UL><LI> A plasmid, chloramphenicol resistant, probably pSB1C3, with:
</LI><LI> An f1 origin of replication, namely &lt;partinfo&gt;BBa_K314110&lt;/partinfo&gt; (this is probably optional, but see below for rationale)
</LI><LI> Fusions of relevant proteins with Tat signals to mature pVIII, e.g.:
</LI><LI> Promoter-RBS-Tat-Endoglucanase-pVIII (based on &lt;partinfo&gt;BBa_K392006&lt;/partinfo&gt;)
</LI><LI> Promoter-RBS-Tat-Exoglucanase-pVIII (based on &lt;partinfo&gt;BBa_K392007&lt;/partinfo&gt;)
</LI><LI> Promoter-RBS-Tat-β-glucosidase-pVIII (based on &lt;partinfo&gt;BBa_K392008&lt;/partinfo&gt;)
</LI></UL><LI> A compatible plasmid, kanamycin resistant, with:
</LI><LI> The entire working M13KO7 genome, but perhaps with pIII deleted, maybe also the phage origin deleted?
</LI><P>The 2nd plasmid produces a normal phage capsid, while the 1st plasmid produces pVIII-fusions which will incorporate into it. The DNA incorporated into the capsid will mostly be from the 1st (safe, non-infectious) plasmid, because M13KO7 preferentially places DNA with an f1 origin into the capsid (according to <A href="http://www.neb.com/nebecomm/products/productN0315.asp" class="external text" rel="nofollow">New England Biolabs</A>). This won't matter if we delete pIII, since in that case there is no possibility of producing infectious phage.
</P><H3><SPAN class="mw-headline" id="Alternatives">Alternatives</SPAN></H3><P>An alternative would be to have everything on one plasmid.
</P><P>Another alternative would be to use <B>BRIDGE</B> to get the M13KO7 genome (again, perhaps minus pIII) into the <I>E. coli</I> chromosome, so we only have to worry about the plasmid that codes for our fusion proteins.
</P><H2><SPAN class="mw-headline" id="Two_options.2C_originality.2C_and_MIT_2010">Two options, originality, and MIT 2010</SPAN></H2><P>As this idea was originally conceived, it would use free-floating phage particles. Only after seeing the <A href="https://2010.igem.org/Team:MIT_phage" class="external text" rel="nofollow">phage component</A> of MIT's 2010 project did the pIII-deletion become a part of the idea.
</P><P>The original idea (free phage) looks nothing like MIT's project, whereas the later version (phage tendrils) looks somewhat similar to it. However, although the techniques used are the same, the goal is very different; MIT worked to cross-link phage filaments to create a novel biomaterial, whereas we will demonstrate this technique's value for catalytic uses.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:531px;"><DIV class="thumbcaption"><B>We have two options:</B> in the first, we use free-floating phage and are very original; in the second, we use phage attached to <I>E. coli</I> as &quot;tendrils&quot;, which are inherently cool but will remind people of MIT 2010.</DIV></DIV></DIV></DIV><P>Building upon other people's work is very much in keeping with the &quot;open-source biology&quot; ethos of iGEM. For example, <A href="https://2010.igem.org/Team:BCCS-Bristol" class="external text" rel="nofollow">Bristol 2010</A> showed that it's possible to do well with an adaptation of prior work. It would be sad if every team felt they had to start from scratch. Still, if we wish to appear more original, we could go with the free phage idea.
</P><P><B>One possibility is to do both</B>; it should be possible to convert a tendril phenotype into a free-phage phenotype by supplying the pIII gene (i.e. &lt;partinfo&gt;BBa_K415138&lt;/partinfo&gt;). This would enable comparisons so we find out which is best.
</P><H2><SPAN class="mw-headline" id="A_modelling_component">A modelling component</SPAN></H2><P>This project has a natural modelling component. Cellulosomes contain several different enzymes that assist in the conversion of cellolose into sugar. It would be good to optimise what the ratio of these various enzymes should be. Modelling will be needed for investigation of other possible phage reactors as well.
</P><P>Also, since both wildtype and modified pVIII proteins are being produced, we will want to choose what that ratio is as well.
</P><P>Vincent suggests that there are population dynamics to be modelled for the alternative system where the phage are actually infectious.
</P><H2><SPAN class="mw-headline" id="Testing">Testing</SPAN></H2><P>As proof of concept (i.e. something we can accomplish in a short time) perhaps it would be sufficient to get just one fusion protein working. We need to prove that the enzyme part is actually getting out of the cell, so we must demonstrate that some substance which cannot enter the cell is nevertheless being degraded.
</P><P>A fairly easy test would be to use a fusion of <A href="http://en.wikipedia.org/wiki/amylase" class="external text" rel="nofollow">amylase</A> to pVIII, and assay for starch degradation, which is very easy. There is no amylase BioBrick (with DNA available) in the Registry, so we'd have to make it.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Cebe R, Geiser M (2000) <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1221525/pdf/11104694.pdf" class="external text" rel="nofollow">Size of the ligand complex between the N-terminal domain of the gene III coat protein and the non-infectious phage strongly influences the usefulness of in vitro selective infective phage technology</A>. <I>Biochemical Journal</I><B>352</B>: 841-849.
</LI></UL><UL><LI> Paschke M, Höhne W (2005) <A href="http://www.sciencedirect.com/science/article/pii/S0378111905000764" class="external text" rel="nofollow">A twin-arginine translocation (Tat)-mediated phage display system</A>. <I>Gene</I><B>350</B>(1): 79-88 (doi: 10.1016/j.gene.2005.02.005).
</LI></UL><UL><LI> Rakonjaca J, Model P (1998) <A href="http://www.sciencedirect.com/science/article/pii/S002228369892006X" class="external text" rel="nofollow">Roles of pIII in filamentous phage assembly</A>. <I>Journal of Molecular Biology</I><B>282</B>(1): 25-41 (doi: 10.1006/jmbi.1998.2006).
</LI></UL><UL><LI> Sidhu SS, Weiss GA, Wells JA (2000) <A href="http://www.sciencedirect.com/science/article/pii/S0022283699934654" class="external text" rel="nofollow">High copy display of large proteins on phage for functional selections</A>. <I>Journal of Molecular Biology</I><B>296</B>(2): 487-495 (doi: 10.1006/jmbi.1999.3465).
</LI></UL><UL><LI> Thammawong P, Kasinrerk W, Turner RJ, Tayapiwatana C (2006) <A href="http://www.springerlink.com/content/y206np5022357m37/" class="external text" rel="nofollow">Twin-arginine signal peptide attributes effective display of CD147 to filamentous phage</A>. Applied Microbiology and Biotechnology <B>69</B>: 697-703 (doi: 10.1007/s00253-005-0242-0).
</LI></UL><UL><LI> Wang KC, Wang X, Zhong P, Luo PP (2010) <A href="http://www.sciencedirect.com.ezproxy.webfeat.lib.ed.ac.uk/science/article/pii/S0022283609014624" class="external text" rel="nofollow">Adapter-directed display: a modular design for shuttling display on phage surfaces</A>. <I>Journal of Molecular Biology</I><B>395</B>(5): 1088-1101 (doi: 10.1016/j.jmb.2009.11.068).
</LI></UL><UL><LI> Weiss GA, Sidhu SS (2000) <A href="http://www.utoronto.ca/sidhulab/pdf/08.pdf" class="external text" rel="nofollow">Design and evolution of artificial M13 coat proteins</A>. <I>Journal of Molecular Biology</I><B>300</B>: 213-219 (doi: 10.1006/jmbi.2000.3845).
</LI></UL><H2><SPAN class="mw-headline" id="Other_useful_links">Other useful links</SPAN></H2><UL><LI><A href="http://www.wwnorton.com/college/biology/microbiology2/ch/11/etopics.aspx" class="external text" rel="nofollow">Molecular overview of M13</A>.
</LI></UL><UL><LI> Karlsson F (2004) <A href="http://www.immun.lth.se/fileadmin/immun/Avhandlingar/Fredrik_Karlsson.pdf" class="external text" rel="nofollow">The biology of filamentous phage infection: implications for display technology</A>.
</LI></UL><UL><LI> Kehoe JW, Kay BK (2005) <A href="http://pubs.acs.org/doi/full/10.1021/cr000261r" class="external text" rel="nofollow">Filamentous Phage Display in the New Millennium</A>. <I>Chemical Reviews</I><B>105</B>(11): 4056-4072 (doi: 10.1021/cr000261r).
</LI></UL><UL><LI> Sidhu SS (2001) <A href="http://www.utoronto.ca/sidhulab/pdf/15.pdf" class="external text" rel="nofollow">Engineering M13 for phage display</A>. <I>Biomolecular Engineering</I><B>18</B>: 57-63.
</LI></UL><UL><LI> Willats WGT (2002) <A href="http://www.springerlink.com/content/u7v1763305k4mu30/" class="external text" rel="nofollow">Phage display: practicalities and prospects</A>. <I>Plant Molecular Biology</I><B>50</B>: 837-854 (doi: 10.1023/A:1021215516430).
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Phage_Reactors_1.0">http://2011.igem.org/Team:Edinburgh/Phage_Reactors_1.0</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Phage_Reactors_1.0" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Phage_Reactors_1.0" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Reactors_1.0&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Reactors_1.0&amp;oldid=57377" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>