"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Data
From 2011.igem.org
outline
in silico
in the lab
in society
postscript
abstract
safety
cell surface display
phage display
achievements
modelling
cellulases (C)
cellulases (Kappa)
cellulases (MATLAB)
model comparison
genetic instability
efficiency
artificial selection
phage replication
data overview
experiments
parts
BioSandwich
lab notebook
conclusions
the team
attributions
our sponsors
human practices
biorefinery design
biorefinery appendices
life cycle analysis
interviews: overview
interviews
interview analysis
collaboration
wiki watch
Hover over words in bold italics to see explanation.
Data Overview
This page provides an overview of the purely biological aspects of our feasibility study, and links to the relevant Parts Registry pages.
Contents
1 Cell Surface Display System
1.1 Cell Surface Display Animation
1.2 Progress
2 Phage Display System
2.1 Phage Display Animation
2.2 Progress
3 BioSandwich
4 Favourite BioBricks
5 Other BioBricks analysed
Cell Surface Display System
(Further details are at the dedicated Cell Surface Display page.)
This system aims at achieving synergy between the enzymes by displaying them at high copy number on the cell's outer membrane. Ice Nucleation Protein is used as a carrier for display of the enzymes; it carries them to the outer membrane.
Cell Surface Display Animation
The completed system should contain:
A promoter ( BBa_K523000 ) controlling:
an INP—Endoglucanase fusion ( BBa_K523008 + BBa_K523011 )
an INP—β-glucosidase fusion ( BBa_K523008 + BBa_K523004 )
an INP—Exoglucanase fusion ( BBa_K523008 + BBa_K523009 )
Ribosome Binding Sites are indicated as green ovals. Cellulose degradation is shown at the bottom. In reality, tens of thousands of enzymes will cover the outer membrane in random places.
A test system to prove that BBa_K523008 can be used to carry proteins to the outer membrane uses a fusion of INP to Yellow Fluorescent Protein (YFP) instead.
Progress
We have:
Shown that INP ( BBa_K523008 , based on BBa_K265008 ) can be used to carry proteins to the cell membrane, by constructing BBa_K523013 .
Made and tested a new β-glucosidase (bglX) BioBrick, BBa_K523002 .
Made bglX into a format compatible with our BioSandwich assembly protocol ( BBa_K523004 ).
Made new versions of other cellulases, in a format compatible with the BioSandwich protocol.
Tested an old exoglucanase gene, BBa_K118022 .
Made fusions of INP to bglX (β-glucosidase) and cex (exoglucanase).
Made a fusion of cex to cenA ( BBa_K523025 ) and shown that it works.
Phage Display System
(Further details are at the dedicated Phage Display page.)
This system aims at achieving synergy between the enzymes by displaying them on an M13phage. The major coat protein pVIII is used as a carrier for display of the enzymes; it incorporates them into the phage.
Phage Display Animation
The completed system should contain:
A promoter ( BBa_K523000 ) controlling:
an Endoglucanase—pVIII fusion
a β-glucosidase—pVIII fusion
an Exoglucanase—pVIII fusion
Ribosome Binding Sites are indicated as green ovals. "Signal" means a periplasmic signal sequence, directing the protein to the periplasm to be assembled into the phage.
A test system uses a fusion of pVIII to E. coli amylase MalS instead.
Progress
We have:
Made and tested a new amylase (malS) BioBrick, BBa_K523001 .
Attempted to fuse it to the major coat protein pVIII.
BioSandwich
BioSandwich ( RFC 81 ) is a new assembly protocol that we used.
Parts made using BioSandwich that we believe are correct (subject to final verification sequencing) include:
BBa_K523013 : PlacLacZ-INP-YFP
BBa_K523019 : PlacLacZ-INP-cex [update: sequencing shows this part has errors]
BBa_K523020 : PlacLacZ-INP-bglX [update: sequencing shows this part has errors]
BBa_K523022 : PlacLacZ-crtEIB
Favourite BioBricks
(A complete list of parts made during the project is found at the dedicated Parts page.)
BBa_K523000 : Plac + lacZ; with BglII site
Intended as a cloning vector (when present in pSB1C3), this part allows Blue/White selection of newly PCR'ed BioBricks. The PCR primers required are shorter than would be required using the standard method. It encodes LacZα; new BioBricks that have successfully replaced the part will be white on plates containing IPTG and Xgal. Parts created in this way will also be compatible with the BioSandwich assembly protocol. We proved this part worked by using it to create several other BioBricks.
BBa_K523006 : Plac + malS
malS is a periplasmicE. coliamylase gene. It ought to be usable for a fusion to INP or pVIII. But to test its normal activity, we placed it under the control of the lac promoter in a high copy number plasmid. We found evidence of starch degrading activity.
BBa_K523013 : Plac + INP-EYFP
A fusion of Ice Nucleation Protein and Enhanced Yellow Fluorescent Protein under the control of the Lac promoter. Fluoresces yellow under blue light. May be localised to the outer membrane.
BBa_K523014 : Plac + bglX
bglX is a crypticβ-glucosidase gene from E. coli. It ought to be usable for a fusion to INP or pVIII. But to test its normal activity, we placed it under the control of the lac promoter in a high copy number plasmid. We found it to be capable of degrading the cellobiose analog, MUG.
BBa_K523025 : Plac + cxnA
cxnA is a fusion we constructed (in line with our beads on a string proposal), containing an exoglucanase and an endoglucanse; it was found to have the functions of both enzymes.
Other BioBricks analysed
(More details of this are found at the dedicated Collaboration page. The information has also been added to the parts' "Experience" sections.)
BBa_K118022 : C. fimi exoglucanase
At the same time as we tested bglX for activity, we also tested this old part's ability to degrade the cellobiose analog MUG (weak ability) and the cellulose analog MUC (strong ability).
BBa_K415151 : p8-GR1
Supposed to encode a fusion of the pVIII protein to a GR1 zipper. It actually encodes part of the E. coli aminopeptidase N gene.
BBa_K392008 : C. fimiβ-glucosidase
We discovered that this part almost certainly starts its coding sequence at the second ATG present, not the first. This is highly relevant information for anyone trying to tune its expression via use of custom Ribosome Binding Sites.
BBa_K265008 : Ice Nucleation Protein
We successfully used this part, fusing it to Enhanced Yellow Fluorescent Protein, which was apparently transported to the outer membrane as a result.
Retrieved from " http://2011.igem.org/Team:Edinburgh/Data "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
