<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Phage_Display"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Phage_Display">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Phage_Display&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Display&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Display&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Phage_Display" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Phage Display</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Phage Display: Proposals</P><P>The second system our feasibility study will examine, while searching for a way to keep extracellular enzymes close together, is based on displaying proteins on a <SPAN class="hardword" id="phage">bacteriophage</SPAN>. This type of display is called &quot;phage display&quot;.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Outline"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Outline</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#The_F_plasmid"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The F plasmid</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Genetic_instability"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Genetic instability</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Bead_reactors"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Bead reactors</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Problems"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Problems</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#A_possible_solution:_zipper_adaptors"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">A possible solution: zipper adaptors</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Testing"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Testing</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Example_systems"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Example systems</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Results"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#References"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Outline">Outline</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">This is what we want to build: phage with enzymes fused to the protein coat.</DIV></DIV></DIV></DIV><P>The project will use phage <SPAN class="hardword" id="m13">M13</SPAN>, which is non-lytic (does not kill the bacteria). The entire genome is known (e.g. <A href="http://genome-www.stanford.edu/vectordb/vector_descrip/COMPLETE/M13MP18.SEQ.html" class="external text" rel="nofollow">here</A> is the entire M13mp18 genome) so primers can be designed. New England Bioworks <A href="http://www.neb.com/nebecomm/tech_reference/protein_tools/phdFaq.asp#1.6" class="external text" rel="nofollow">claims</A>:
</P><DL><DD><I>&quot;The major coat protein <SPAN class="hardword" id="p8">pVIII</SPAN> is present at ~2700 copies per virion, of which ~10% can be reliably fused to peptides or proteins.&quot;</I></DD></DL><P>We would infect our <SPAN class="hardword" id="ec">E. coli</SPAN> with wildtype phage. Our <I>E. coli</I> would have a plasmid coding for a fusion between an enzyme and pVIII. Proteins can be fused to pVIII at its amino terminal (i.e. 5' end in the DNA), according to <A href="http://www.utoronto.ca/sidhulab/pdf/08.pdf" class="external text" rel="nofollow">Weiss and Sidhu, 2000</A>. pVIII has a leader peptide (<A href="http://www.uniprot.org/uniprot/P69541" class="external text" rel="nofollow">residues 1-23</A>) that is cleaved out, slightly complicating fusion design.
</P><P>To attach several different proteins to the phage, different fusions can be created and all of them expressed on the plasmid.
</P><P>We would need to tune expression levels of the fusions versus the wildtype protein.
</P><H3><SPAN class="mw-headline" id="The_F_plasmid">The F plasmid</SPAN></H3><P>M13 can only infect <I>E. coli</I> if it has a sex pilus. This is coded for by the <SPAN class="hardword" id="f">F plasmid</SPAN>. However, such plasmids are quickly lost from cultures in a laboratory setting, as they confer a fitness cost. The JM109 strain has proline synthesis genes (proAB) absent from its main chromosome, but present on the F plasmid, and so when it is grown on minimal media, the plasmid will not be lost, as it is needed to keep the bacteria alive.
</P><H3><SPAN class="mw-headline" id="Genetic_instability">Genetic instability</SPAN></H3><P>In order to display several different proteins on one phage, it will be necessary to have several copies of the pVIII gene fused to different enzymes. The presence of repeated sequences on a plasmid can lead to genetic instability.
</P><P>This will not be a problem in the JM109 lab strain, which lacks an important <SPAN class="hardword" id="recombination">recombination</SPAN> enzyme. As for the use of this technology in industry, it will be possible to overcome this problem simply by synthesising coding sequences with as many altered (but <SPAN class="hardword" id="synonymouscodon">synonymous</SPAN>) codons as possible. We have written a software tool for designing such sequences... see the <A href="/Team:Edinburgh/Genetic_Instability" title="Team:Edinburgh/Genetic Instability">genetic instability</A> page.
</P><H2><SPAN class="mw-headline" id="Bead_reactors">Bead reactors</SPAN></H2><P>Maurice Gallagher, one of our university's phage experts, suggests a different approach: make several strains of bacteria each producing phage with just one type of pVIII-fusion. But also make each phage have a <SPAN class="hardword" id="p3">pIII</SPAN>-fusion with a protein which could bind some sort of bead. The bead now becomes the complete &quot;reactor&quot;. Since each bacteria codes for only one pVIII fusion, there is no repeated sequence problem.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:464px;"><DIV class="thumbcaption">Detail of M13 attached to a bead. We would link multiple phages to the bead; each phage would carry one enzyme type only.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Problems">Problems</SPAN></H2><P>The question is how efficiently fusions to pVIII can get onto the phage. There are some dire warnings in the literature:
</P><UL><LI><I>In our experience, most large proteins display well below one copy per phage particle.</I>â<A href="http://www.sciencedirect.com/science/article/pii/S0022283699934654" class="external text" rel="nofollow">Sidhu et al (2000)</A></LI><LI><I>A large 20 kDa protein (human growth hormone, hGH) is not displayed at detectable levels.</I>â<A href="http://www.utoronto.ca/sidhulab/pdf/08.pdf" class="external text" rel="nofollow">Weiss and Sidhu (2000)</A></LI><LI><I>The properties of the pIV channel may be one of the factors that limit the size of polypeptides that can be displayed on pVIII.</I>â<A href="http://www.immun.lth.se/fileadmin/immun/Avhandlingar/Fredrik_Karlsson.pdf" class="external text" rel="nofollow">Karlsson (2004)</A></LI><LI><I>As a general rule, the minor coat proteins will display larger proteins more effectively than pVIII.</I>â<A href="http://pubs.acs.org/doi/full/10.1021/cr000261r" class="external text" rel="nofollow">Kehoe and Kay (2005)</A></LI><LI><I>[It is plausible] that a phage containing pVIII with a large peptide may be too large in diameter to pass through the 7-nm pIV exit pore in the outer membrane.</I>â Barbas <I>et al</I> (2001)
</LI><LI><I>The pVIII site, although very popular for peptide phage display, is not suitable for the efficient display of large polypeptides such as antibodies.</I>â<A href="http://www.sciencedirect.com/science/article/pii/S0734975000000549" class="external text" rel="nofollow">Benhar (2001)</A></LI></UL><P>However, Maurice seemed quite upbeat about the prospects when we met him. It's definitely the case that large stuff has been displayed, but the question is whether this is some sort of fluke that only occurs (say) once per phage (i.e. see the Sidhu quote above).
</P><H3><SPAN class="mw-headline" id="A_possible_solution:_zipper_adaptors">A possible solution: zipper adaptors</SPAN></H3><P>If things don't work well, a possible solution (or at least something to attempt) exists...
</P><P>MIT 2010 attached small &quot;zipper&quot; peptides to pVIII. Their intention was to join phage together. We could use this system to attach our enzymes of interest (i.e. our cellulases) to the phage indirectly, by fusing the enzyme to a zipper and pVIII to the corresponding zipper. The two zippers will attach if we can get them to meet physically.
</P><P>The basic idea is similar to that of <A href="http://www.sciencedirect.com/science/article/pii/S0378111905000764" class="external text" rel="nofollow">Paschke and HÃ¶hne (2005)</A> (see Figure 1) and especially <A href="http://www.sciencedirect.com/science/article/pii/S0022283609014624" class="external text" rel="nofollow">Wang <I>et al</I> (2010)</A>.
</P><UL><LI> Sequence for zipper GR1: EEKSRLLEKENRELEKIIAEKEERVSELRHQLQSVGGC (38)
</LI><LI> Sequence for zipper GR2: TSRLEGLQSENHRLRMKITELDKDLEEVTMQLQDVGGC (38)
</LI></UL><P>The construct would then contain two coding sequences:
</P><UL><LI> Promoter--RBS--Leader--GR2--pVIII
</LI><LI> Promoter--RBS--Periplasm Signal--Enzyme--GR1
</LI></UL><H2><SPAN class="mw-headline" id="Testing">Testing</SPAN></H2><P>As proof of concept (i.e. something we can accomplish in a short time) it would be sufficient to get just one fusion protein working. We need to prove that the enzyme part is actually getting out of the cell, so we must demonstrate that some substance which cannot enter the cell is nevertheless being degraded.
</P><P>A fairly easy test would be to use a fusion of <SPAN class="hardword" id="amylase">amylase</SPAN> to pVIII, and assay for <SPAN class="hardword" id="starch">starch</SPAN> degradation. There is no amylase BioBrick (with DNA available) in the Registry, so we'd have to make it.
</P><H2><SPAN class="mw-headline" id="Example_systems">Example systems</SPAN></H2><P>A simple version of the system would work as follows:
</P><UL><LI><I>E. coli</I> are grown up containing a plasmid coding for a pVIII fusion gene, i.e:
</LI><LI> Promoter--RBS--Leader--Amylase--(Linker?)--pVIII.
</LI></UL><LI> These <I>E. coli</I> are infected with M13.
</LI><LI> They create new phage; some of the modified pVIII proteins incorporate into the capsid.
</LI><P>More complex versions would either incorporate more pVIII fusions, or multiple strains all of which have a pIII-fusion to attach to beads.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Please see the team's <A href="/Team:Edinburgh/Data" title="Team:Edinburgh/Data"> Data Page</A> for information about how far we got with this project.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Barbas CF, <I>et al</I> (2001) <I>Phage display: a laboratory manual.</I> Cold Spring Harbor Laboratory Press.
</LI></UL><UL><LI> Benhar I (2001) <A href="http://www.sciencedirect.com/science/article/pii/S0734975000000549" class="external text" rel="nofollow">Biotechnological applications of phage and cell display</A>. <I>Biotechnology Advances</I><B>19</B>(1): 1-33 (doi: 10.1016/S0734-9750(00)00054-9).
</LI></UL><UL><LI> Paschke M, HÃ¶hne W (2005) <A href="http://www.sciencedirect.com/science/article/pii/S0378111905000764" class="external text" rel="nofollow">A twin-arginine translocation (Tat)-mediated phage display system</A>. <I>Gene</I><B>350</B>(1): 79-88 (doi: 10.1016/j.gene.2005.02.005).
</LI></UL><UL><LI> Sidhu SS, Weiss GA, Wells JA (2000) <A href="http://www.sciencedirect.com/science/article/pii/S0022283699934654" class="external text" rel="nofollow">High copy display of large proteins on phage for functional selections</A>. <I>Journal of Molecular Biology</I><B>296</B>(2): 487-495 (doi: 10.1006/jmbi.1999.3465).
</LI></UL><UL><LI> Wang KC, Wang X, Zhong P, Luo PP (2010) <A href="http://www.sciencedirect.com/science/article/pii/S0022283609014624" class="external text" rel="nofollow">Adapter-directed display: a modular design for shuttling display on phage surfaces</A>. <I>Journal of Molecular Biology</I><B>395</B>(5): 1088-1101 (doi: 10.1016/j.jmb.2009.11.068).
</LI></UL><UL><LI> Weiss GA, Sidhu SS (2000) <A href="http://www.utoronto.ca/sidhulab/pdf/08.pdf" class="external text" rel="nofollow">Design and evolution of artificial M13 coat proteins</A>. <I>Journal of Molecular Biology</I><B>300</B>: 213-219 (doi: 10.1006/jmbi.2000.3845).
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Phage_Display">http://2011.igem.org/Team:Edinburgh/Phage_Display</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Phage_Display" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Phage_Display" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Display&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Phage_Display&amp;oldid=253952" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>