<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Genetic_Instability"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Genetic_Instability">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Genetic_Instability&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Genetic_Instability&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Genetic_Instability&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Genetic_Instability" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Genetic Instability</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Edinburgh/Genetic_instability&amp;redirect=no" title="Team:Edinburgh/Genetic instability">Team:Edinburgh/Genetic instability</A>)</DIV><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Genetic Instability Tool</P><P>Our project involves certain sequences of DNA being present in several copies on a <SPAN class="hardword" id="plasmid">plasmid</SPAN>. For example, to display three different enzymes on a cell via <SPAN class="hardword" id="inp">Ice Nucleation Protein</SPAN> (INP) requires three copies of the INP gene in the plasmid.
</P><P>But there exists this thing called &quot;genetic instability&quot;, and we wanted to know whether this made the whole project infeasible. (As it turns out, it doesn't.)
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#What_is_genetic_instability.3F"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">What is genetic instability?</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Possible_modes"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Possible modes</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Random_generation"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Random generation</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Best_codon"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Best codon</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Genetic_algorithm"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Genetic algorithm</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Conclusion"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Download"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Download</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="What_is_genetic_instability.3F">What is genetic instability?</SPAN></H2><P>When a lot of similar DNA sequences are introduced into a cell, this can potentially lead to those pieces of DNA undergoing <SPAN class="hardword" id="recombination">recombination</SPAN> and thus rearranging the DNA. This will not be a problem in our lab strain of <I>E. coli</I>, JM109, as it lacks the RecA recombinase enzyme which causes this to happen. However, industry requires hardy strains of <I>E. coli</I>, which must possess recombinases to deal with their high-stress working environment.
</P><P>So it initially seems as if the systems we are investigating could never be used in industry. <B>But in fact there is a solution!</B></P><P>If only we had several different (as far apart as possible) DNA sequences coding for the same protein, the project would be saved! According to <A href="http://www.pnas.org/content/82/14/4768.full.pdf" class="external text" rel="nofollow">Watt <I>et al</I> (1985)</A> even a small number of mismatches drastically reduce recombination frequency.
</P><P>Enter Edinburgh's <B>Genetic Stability Tool!</B> Using DNA's natural property of redundancy, it will find a number of different DNA sequences for you! And they will code for the same protein! Superb!
</P><P>So, how does it work?
</P><H2><SPAN class="mw-headline" id="Possible_modes">Possible modes</SPAN></H2><H3><SPAN class="mw-headline" id="Random_generation">Random generation</SPAN></H3><P>This is a very crude mode, in which a random sequence of codons coding for analogous amino acids is generated, e.g. if you had:
</P><P><CODE>
atgaaaaagtctttagtcctcaaagcctctgtagccgttgctaccctcgttccgatgctgtctttcgct...
</CODE>(the leader sequence for pVIII),
</P><P>the program would translate it to amino acids sequence, i.e.:
</P><P><CODE>
MKKSLVLKASVAVATLVPMLSFA...
</CODE></P><P>and then find a random codon for each amino acid, put them together, and spit out the result.
</P><P>If this sounds too simple to produce really good results... you're right!
</P><H3><SPAN class="mw-headline" id="Best_codon">Best codon</SPAN></H3><P>In this mode, the tool would choose the best codon per every amino acid. What is the 'best codon'? It's the codon which has the most base pairs different from the original one.
</P><P>For example, Leucine can be coded by TTA, TTG, CTT, CTC, CTA, or CTG. If the original sequence uses TTA, then the best codon method would choose CTT, CTC or CTG. There often are plenty of 'best' codons, so this method would be improved if it looked at the wider context...
</P><H3><SPAN class="mw-headline" id="Genetic_algorithm">Genetic algorithm</SPAN></H3><P>This is a very powerful <SPAN class="hardword" id="algorithm">algorithm</SPAN>, inspired by Computer Scientists observing Biology. It represents a problem using a string of characters - i.e. the 'DNA', then solutions are 'individuals', there are many of them in a 'generation', and they 'cross-over' and 'mutate' their 'genes' at some randomised rate. The most 'fit' individuals get into the next generation.
</P><P>A genetic algorithm is evolution <I>in silico!</I> It's perfect for use in Biology!
</P><P>We can use this algorithm to generate DNA sequences which are as far from the original as possible. <B>In this mode, the tool also avoids heavy use of rare codons</B>. And, it can return more than one sequence of DNA, which are also far from each other.
</P><P>What follows are 200 bases of two synthetic versions of <A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A> which are both dissimilar to it, as well as being dissimilar to each other. They also avoid using rare codons very often. These sequences were produced by the genetic algorithm.
</P><PRE style="background-color: rgba(255, 255, 255, 1.0);">
Alpha    1  atgaccctggacaaagcgctggtgctgcgtacctgcgccaacaacatggc   50
            |||||||||||.|||||.|||||.||||||||.|||||.|||||||||||
Beta     1  atgaccctggataaagctctggtcctgcgtacgtgcgcgaacaacatggc   50

Alpha   51  agatcactgcggtctgatttggccggcctctggcaccgtagagtcccgtt  100
            .||.||||||||.|||||.||||||||....|||||.||.||.||.||||
Beta    51  tgaccactgcggcctgatctggccggcaagcggcactgttgaatctcgtt  100

Alpha  101  actggcagagcactcgtcgtcatgaaaacggtctggttggcctgctgtgg  150
            |||||||....||.||||||||.|||||||||||.||.||.|||||.|||
Beta   101  actggcaatcgacccgtcgtcacgaaaacggtcttgtgggtctgctctgg  150

Alpha  151  ggcgcgggtacttcggccttcttatctgtccatgctgacgcgcgttggat  200
            ||.||.||.||....||.|||.|.||.||.|||||.||.|||||.|||||
Beta   151  ggtgcaggcaccagcgcattcctgtcggttcatgcggatgcgcgctggat  200
</PRE><H2><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H2><P>Using this tool we have shown that systems requiring multiple copies of a coding sequence can still be suitable for industrial use; recombination can be avoided by synthesising multiple different DNA sequences that code for the same protein.
</P><H2><SPAN class="mw-headline" id="Download">Download</SPAN></H2><P>You can download the all-new all-shiny tool here:
<CENTER><A class="nounderline" href="https://2011.igem.org/File:Edinburgh_gen_stab_0.1_Alpha_Americano.zip">
Team Edinburgh Genetic Stability Tool, v.0.1 Alpha Americano</A></CENTER></P><P>
To use, simply unpack, and run the gen_stab.py in python. Any questions? 
</P><UL><LI> Drop us a line on <A href="http://www.twitter.com/iGEMEdinburgh" class="external text" rel="nofollow">Twitter</A>, or
</LI><LI> E-mail: L (dot) Kopec (at) sms (dot) ed (dot) ac (dot) uk
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Watt VM, Ingles CJ, Urdea MS, Rutter WJ (1985) <A href="http://www.pnas.org/content/82/14/4768.full.pdf" class="external text" rel="nofollow">Homology requirements for recombination in <I>Escherichia coli</I></A>. <I>Proc. Natl. Acad. Sci. USA</I><B>82</B>: 4768-4772.
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Genetic_Instability">http://2011.igem.org/Team:Edinburgh/Genetic_Instability</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Genetic_Instability" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Genetic_Instability" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Genetic_Instability&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Genetic_Instability&amp;oldid=156907" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>