<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Data"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Data">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Data&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Data&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Data&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Data" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Data</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Edinburgh/Data_page&amp;redirect=no" title="Team:Edinburgh/Data page">Team:Edinburgh/Data page</A>)</DIV><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Data Overview</P><P>This page provides an overview of the purely biological aspects of our feasibility study, and links to the relevant Parts Registry pages.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cell_Surface_Display_System"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cell Surface Display System</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Cell_Surface_Display_Animation"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Cell Surface Display Animation</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Progress"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Progress</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Phage_Display_System"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Phage Display System</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Phage_Display_Animation"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Phage Display Animation</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Progress_2"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Progress</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#BioSandwich"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">BioSandwich</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Favourite_BioBricks"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Favourite BioBricks</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Other_BioBricks_analysed"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Other BioBricks analysed</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Cell_Surface_Display_System">Cell Surface Display System</SPAN></H2><P>(Further details are at the dedicated <A href="/Team:Edinburgh/Cell_Display" title="Team:Edinburgh/Cell Display"> Cell Surface Display</A> page.)
</P><P>This system aims at achieving synergy between the enzymes by displaying them at high copy number on the cell's outer membrane. <SPAN class="hardword" id="inp">Ice Nucleation Protein</SPAN> is used as a carrier for display of the enzymes; it carries them to the outer membrane.
</P><H3><SPAN class="mw-headline" id="Cell_Surface_Display_Animation">Cell Surface Display Animation</SPAN></H3><P>
The completed system should contain:<UL><LI>A <STRONG>promoter</STRONG> (<A href="http://partsregistry.org/Part:BBa_K523000">BBa_K523000</A>) controlling:
</LI><LI>an <STRONG>INP—Endoglucanase </STRONG>fusion (<A href="http://partsregistry.org/Part:BBa_K523008">BBa_K523008</A> + <A href="http://partsregistry.org/Part:BBa_K523011">BBa_K523011</A>)</LI><LI>an <STRONG>INP—β-glucosidase</STRONG> fusion (<A href="http://partsregistry.org/Part:BBa_K523008">BBa_K523008</A> + <A href="http://partsregistry.org/Part:BBa_K523004">BBa_K523004</A>)</LI><LI>an <STRONG>INP—Exoglucanase</STRONG> fusion (<A href="http://partsregistry.org/Part:BBa_K523008">BBa_K523008</A> + <A href="http://partsregistry.org/Part:BBa_K523009">BBa_K523009</A>)</LI></UL></P><P><STRONG>Ribosome Binding Sites</STRONG> are indicated as green ovals.
Cellulose degradation is shown at the bottom. In reality, tens of thousands of enzymes will cover the outer membrane in random places.
A test system to prove that <A href="http://partsregistry.org/Part:BBa_K523008">BBa_K523008</A> can be used to carry proteins to the outer membrane uses a fusion of INP to <STRONG>Yellow Fluorescent Protein (YFP)</STRONG> instead.</P><H3><SPAN class="mw-headline" id="Progress">Progress</SPAN></H3><P>We have:
</P><UL><LI> Shown that INP (<A href="http://partsregistry.org/Part:BBa_K523008" class="external text" rel="nofollow">BBa_K523008</A>, based on <A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A>) can be used to carry proteins to the cell membrane, by constructing <A href="http://partsregistry.org/Part:BBa_K523013" class="external text" rel="nofollow">BBa_K523013</A>.
</LI><LI> Made and tested a new β-glucosidase <I>(bglX)</I> BioBrick, <A href="http://partsregistry.org/Part:BBa_K523002" class="external text" rel="nofollow">BBa_K523002</A>.
</LI><LI> Made <I>bglX</I> into a format compatible with our BioSandwich assembly protocol (<A href="http://partsregistry.org/Part:BBa_K523004" class="external text" rel="nofollow">BBa_K523004</A>).
</LI><LI> Made new versions of other cellulases, in a format compatible with the BioSandwich protocol.
</LI><LI> Tested an old exoglucanase gene, <A href="http://partsregistry.org/Part:BBa_K118022" class="external text" rel="nofollow">BBa_K118022</A>.
</LI><LI> Made fusions of INP to <I>bglX</I> (β-glucosidase) and <I>cex</I> (exoglucanase).
</LI><LI> Made a fusion of <I>cex</I> to <I>cenA</I> (<A href="http://partsregistry.org/Part:BBa_K523025" class="external text" rel="nofollow">BBa_K523025</A>) and shown that it works.
</LI></UL><H2><SPAN class="mw-headline" id="Phage_Display_System">Phage Display System</SPAN></H2><P>(Further details are at the dedicated <A href="/Team:Edinburgh/Phage_Display" title="Team:Edinburgh/Phage Display"> Phage Display</A> page.)
</P><P>This system aims at achieving synergy between the enzymes by displaying them on an <SPAN class="hardword" id="m13">M13</SPAN><SPAN class="hardword" id="phage">phage</SPAN>. The major coat protein <SPAN class="hardword" id="p8">pVIII</SPAN> is used as a carrier for display of the enzymes; it incorporates them into the phage.
</P><H3><SPAN class="mw-headline" id="Phage_Display_Animation">Phage Display Animation</SPAN></H3><P>
The completed system should contain:<UL><LI>A <STRONG>promoter</STRONG> (<A href="http://partsregistry.org/Part:BBa_K523000">BBa_K523000</A>) controlling:
</LI><LI>an <STRONG>Endoglucanase—pVIII</STRONG> fusion</LI><LI>a <STRONG>β-glucosidase—pVIII</STRONG> fusion</LI><LI>an <STRONG>Exoglucanase—pVIII</STRONG> fusion</LI></UL></P><P><STRONG>Ribosome Binding Sites</STRONG> are indicated as green ovals. <STRONG>&quot;Signal&quot;</STRONG> means a periplasmic signal sequence, directing the protein to the periplasm to be assembled into the phage.A test system uses a fusion of pVIII to <EM>E. coli</EM> amylase <STRONG>MalS</STRONG> instead.</P><H3><SPAN class="mw-headline" id="Progress_2">Progress</SPAN></H3><P>We have:
</P><UL><LI> Made and tested a new amylase <I>(malS)</I> BioBrick, <A href="http://partsregistry.org/Part:BBa_K523001" class="external text" rel="nofollow">BBa_K523001</A>.
</LI><LI> Attempted to fuse it to the major coat protein pVIII.
</LI></UL><H2><SPAN class="mw-headline" id="BioSandwich">BioSandwich</SPAN></H2><P><A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich"> BioSandwich</A> (<A href="/File:RFC81.pdf" title="File:RFC81.pdf">RFC 81</A>) is a new assembly protocol that we used.
</P><P>Parts made using BioSandwich that we believe are correct (subject to final verification sequencing) include:
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_K523013" class="external text" rel="nofollow">BBa_K523013</A>: PlacLacZ-INP-YFP
</LI><LI><A href="http://partsregistry.org/Part:BBa_K523019" class="external text" rel="nofollow">BBa_K523019</A>: PlacLacZ-INP-cex [<B>update:</B> sequencing shows this part has errors]
</LI><LI><A href="http://partsregistry.org/Part:BBa_K523020" class="external text" rel="nofollow">BBa_K523020</A>: PlacLacZ-INP-bglX [<B>update:</B> sequencing shows this part has errors]
</LI><LI><A href="http://partsregistry.org/Part:BBa_K523022" class="external text" rel="nofollow">BBa_K523022</A>: PlacLacZ-crtEIB
</LI></UL><H2><SPAN class="mw-headline" id="Favourite_BioBricks">Favourite BioBricks</SPAN></H2><P>(A complete list of parts made during the project is found at the dedicated <A href="/Team:Edinburgh/Parts" title="Team:Edinburgh/Parts"> Parts</A> page.)
</P><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K523000" class="external text" rel="nofollow">BBa_K523000</A>: Plac + lacZ; with BglII site</B></DT><DD> Intended as a cloning vector (when present in pSB1C3), this part allows Blue/White selection of newly PCR'ed BioBricks. The PCR primers required are shorter than would be required using the standard method. It encodes LacZα; new BioBricks that have successfully replaced the part will be white on plates containing IPTG and Xgal. Parts created in this way will also be compatible with the BioSandwich assembly protocol. We proved this part worked by using it to create several other BioBricks.
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K523006" class="external text" rel="nofollow">BBa_K523006</A>: Plac + malS</B></DT><DD><I>malS</I> is a <SPAN class="hardword" id="periplasm">periplasmic</SPAN><I>E. coli</I><SPAN class="hardword" id="amylase">amylase</SPAN> gene. It ought to be usable for a fusion to INP or pVIII. But to test its normal activity, we placed it under the control of the lac promoter in a high copy number plasmid. We found evidence of <SPAN class="hardword" id="starch">starch</SPAN> degrading activity.
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K523013" class="external text" rel="nofollow">BBa_K523013</A>: Plac + INP-EYFP</B></DT><DD> A fusion of Ice Nucleation Protein and Enhanced Yellow Fluorescent Protein under the control of the Lac promoter. Fluoresces yellow under blue light. May be localised to the outer membrane.
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K523014" class="external text" rel="nofollow">BBa_K523014</A>: Plac + bglX</B></DT><DD><I>bglX</I> is a <SPAN class="hardword" id="cryptic">cryptic</SPAN>β-glucosidase gene from <I>E. coli</I>. It ought to be usable for a fusion to INP or pVIII. But to test its normal activity, we placed it under the control of the lac promoter in a high copy number plasmid. We found it to be capable of degrading the <SPAN class="hardword" id="cellobiose">cellobiose</SPAN> analog, MUG.
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K523025" class="external text" rel="nofollow">BBa_K523025</A>: Plac + cxnA</B></DT><DD><I>cxnA</I> is a fusion we constructed (in line with our <A href="/Team:Edinburgh/Cell_Display#An_alternative:_protein_chains" title="Team:Edinburgh/Cell Display">beads on a string</A> proposal), containing an exoglucanase and an endoglucanse; it was found to have the functions of both enzymes.
</DD></DL><H2><SPAN class="mw-headline" id="Other_BioBricks_analysed">Other BioBricks analysed</SPAN></H2><P>(More details of this are found at the dedicated <A href="/Team:Edinburgh/Collaboration" title="Team:Edinburgh/Collaboration"> Collaboration</A> page. The information has also been added to the parts' &quot;Experience&quot; sections.)
</P><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K118022:Experience" class="external text" rel="nofollow">BBa_K118022</A>: <I>C. fimi</I> exoglucanase</B></DT><DD> At the same time as we tested <I>bglX</I> for activity, we also tested this old part's ability to degrade the cellobiose analog MUG (weak ability) and the cellulose analog MUC (strong ability).
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K415151:Experience" class="external text" rel="nofollow">BBa_K415151</A>: p8-GR1</B></DT><DD> Supposed to encode a fusion of the pVIII protein to a GR1 zipper. It actually encodes part of the <I>E. coli</I> aminopeptidase N gene.
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K392008:Experience" class="external text" rel="nofollow">BBa_K392008</A>: <I>C. fimi</I>β-glucosidase</B></DT><DD> We discovered that this part almost certainly starts its coding sequence at the second ATG present, not the first. This is highly relevant information for anyone trying to tune its expression via use of custom Ribosome Binding Sites.
</DD></DL><DL><DT><B><A href="http://partsregistry.org/Part:BBa_K265008:Experience" class="external text" rel="nofollow">BBa_K265008</A>: Ice Nucleation Protein</B></DT><DD> We successfully used this part, fusing it to Enhanced Yellow Fluorescent Protein, which was apparently transported to the outer membrane as a result.
</DD></DL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Data">http://2011.igem.org/Team:Edinburgh/Data</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Data" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Data" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Data&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Data&amp;oldid=256707" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>