"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Cell Display
From 2011.igem.org
outline
in silico
in the lab
in society
postscript
abstract
safety
cell surface display
phage display
achievements
modelling
cellulases (C)
cellulases (Kappa)
cellulases (MATLAB)
model comparison
genetic instability
efficiency
artificial selection
phage replication
data overview
experiments
parts
BioSandwich
lab notebook
conclusions
the team
attributions
our sponsors
human practices
biorefinery design
biorefinery appendices
life cycle analysis
interviews: overview
interviews
interview analysis
collaboration
wiki watch
Hover over words in bold italics to see explanation.
Cell Surface Display: Proposals
The first proposed system our feasibility study will examine, while searching for a way to keep extracellular enzymes close together, is based on displaying proteins at high density on an E. coli outer membrane. This type of display is called "cell surface display".
We will attempt to design such a system for cellulases, and see if we can get it to work.
Contents
1 Outline
1.1 Linkers
1.2 Complete system
1.3 An alternative: protein chains
2 Genetic instability
3 Proof of concept: YFP
4 Results
5 References
Outline
In order to get a normal enzyme displayed on the E. coli outer membrane, the enzyme must be fused to a carrier protein; that is, one which is naturally transported to the outer membrane.
Berkeley 2009 tried several different carrier proteins with several different passenger enzymes, and had success in many areas. However, when they tried attaching cellulases, they weren't so successful - of the two quantified cellulases, one worked just as well without the carrier (Cel5b) and the other didn't work (Cel9a, as compared to negative control).
We will try a different carrier. The BioBrick BBa_K265008 made by UC Davis 2009 is a synthetic, codon-optimised sequence, based on GenBank AF013159 and coding for the first 211 and last 97 amino acids of Ice Nucleation Protein (INP, normally coded by the inaK gene) from the bacterium Pseudomonas syringae. It seems promising as a carrier of enzymes. Fusions are carried out at the INP C terminal.
Three strategies for INP-based cell display. After Van Bloois et al (2011)
Van Bloois et al (2011) speak highly of INP, and claim that it can be displayed at a copy number of around 100,000 copies per cell without affecting viability.
INP has major domains at its N and C terminals, as well as a number of internal repeating domains. There seem to be three strategies for using INP (see figure):
Use the entire INP protein; fuse at its C terminal
Delete the INP internal domains; fuse at its C terminal
Delete all of INP except the N domain; fuse at the new C terminal
BBa_K265008 should be suitable for the 2nd strategy.
Linkers
It is probably desirable to create linkers between the carrier and the protein of interest, to give the proteins space to fold. The new assembly protocol that we are investigating — BioSandwich — should be ideal for this.
Complete system
The complete 3 cellulase system could contain a promoter, driving expression of three coding fusions:
INP -- (Linker) -- endoglucanase (e.g. BBa_K392006 )
INP -- (Linker) -- exoglucanase (e.g. BBa_K392007 )
INP -- (Linker) -- β-glucosidase (e.g. BBa_K392008 )
An alternative: protein chains
The protein chain idea: a long fusion protein is created with INP fused to (say) 3 enzymes in a row...
Instead of making three different fusions, it might be possible to make one fusion that had all three cellulase enzymes linked together; we call this "beads on a string". As it happens, the exoglucanase (Cex) and the endoglucanase (CenA) both have a cellulose-binding module (CBM), but they are at different ends of the sequence. So here's the plan:
Create a fusion of:
Exoglucanase (catalytic domain) -- CBM -- Endoglucanase (catalytic domain)
This can be done by homology or by introducing an NcoI site into both Exo- and Endoglucanase at the appropriate locations, then ligating and doing fusion PCR.
We can then use KpnI in a similar way to attach a β-glucosidase at either end.
Then attach INP.
Genetic instability
In order to display several different proteins on one bacterium using the first strategy, it will be necessary to have several copies of the INP gene fused to different enzymes. The presence of repeated sequences on a plasmid can lead to genetic instability.
This will not be a problem in the JM109 lab strain, which lacks an important recombination enzyme. As for the use of this technology in industry, it will be possible to overcome this problem simply by synthesising coding sequences with as many altered (but synonymous) codons as possible. We have written a software tool for designing such sequences... see the genetic instability page.
Proof of concept: YFP
As far as we know, nobody has used BBa_K265008 for cell display. We could prove that it works by simply displaying the Yellow Fluorescent Protein on INP. Indeed, something similar was achieved by Li et al (2004) and Li et al (2009) for a different version of the gene.
Results
Please see the team's Data Page for information about how far we got with this project.
References
Li L, Kang DG, Cha HJ (2004) Functional display of foreign protein on surface of Escherichia coli using N-terminal domain of Ice Nucleation Protein . Biotechnology and Bioengineering 85(2): 214-221 (doi: 10.1002/bit.10892).
Li Q, Yu Z, Shao X, He J, Li L (2009) Improved phosphate biosorption by bacterial surface display of phosphate-binding protein utilizing ice nucleation protein . FEMS Microbiology Letters 299(1): 44-52 (doi: 10.1111/j.1574-6968.2009.01724.x).
Van Bloois E, Winter RT, Kolmar H, Fraaije MW (2011) Decorating microbes: surface display of proteins on Escherichia coli . Trends in Biotechnology 29(2): 79-86 (doi: 10.1016/j.tibtech.2010.11.003).
Retrieved from " http://2011.igem.org/Team:Edinburgh/Cell_Display "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
