<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Lab_Notebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Lab_Notebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Lab_Notebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Lab_Notebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Lab_Notebook&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Lab_Notebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Lab Notebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Lab Notebook</P><P><B>Note:</B> Protocols were generally similar to those found on Open Wetware's <A href="http://www.openwetware.org/wiki/French_Lab" class="external text" rel="nofollow">pages</A> dedicated to Chris French's Lab.
</P><P><B>June 1:</B>
Began work to add the Plac-lacZα sequence to the pSB1C3 vector. PCR was used to introduce the Bgl2 restriction site (so the final thing becomes E-X-B-Plac-lacZα-S-P). This gives us a vector that we like. For the principles behind the use of lacZα see <A href="http://utminers.utep.edu/rwebb/html/alpha_complementation.html" class="external text" rel="nofollow">here</A>. It will enable us to determine whether inserts are present using blue/white screening. In the future, vectors in which the desired part successfully replaces lacZα will be <B>white</B>.
</P><P>Having a Bgl2 site in the vector at the end of the BioBrick prefix is useful as it means any DNA sequence can easily have the sites needed for incorporation into the vector added (by PCR) at both ends, whereas in the standard RFC10 system the two most obvious sites (XbaI and SpeI, which come immediately before and after the BioBrick itself) cannot actually be used this way, because they generate compatible sticky ends, meaning the product would circularise.
</P><P><B>June 2:</B>
PCR seemed successful (though 2 bands were seen on the gel, 1 was the right size). We already have pSB1C3-RFP (Red Fluorescent Protein) so we cut out the RFP (using restriction enzymes XbaI and PstI) and inserted the new sequence.
</P><P><B>June 3:</B>
After transformation of competent cells we plated onto an <A href="http://en.wikipedia.org/wiki/X-gal" class="external text" rel="nofollow">Xgal</A>- and <A href="http://en.wikipedia.org/wiki/IPTG" class="external text" rel="nofollow">IPTG</A>- containing media.
</P><P>At this point we also prepared about 15 plates of agar, with chloramphenicol (which the pSB1C3 vector provides resistance to) as well as Xgal and IPTG for later blue/white screening.
</P><P><B>June 6:</B>
Colonies that have successfully lost RFP and gained lacZα should be blue (on this media). We found several such colonies (though many others were red and some were white). Six blue colonies were selected, replated and grown for 6 hours on the media mentioned above. Some cells were then transferred to bottles of media for further growth.
</P><P><B>June 7:</B>
Plasmid DNA from the bottled cultures was purified (by the &quot;miniprep&quot; protocol). We digested it with EcoRI to linearise it, and then ran it on a gel. All six cultures showed a band of the correct size (about 2600 b.p.) but we will send some DNA for sequencing.
</P><P><B>June 8:</B>
Clones 1 and 2 of pSB1C3-lacZα were selected for sequencing. Four tubes were sent to the university's sequencing service:
</P><OL><LI> Clone 1, Forward
</LI><LI> Clone 1, Reverse
</LI><LI> Clone 2, Forward
</LI><LI> Clone 2, Reverse
</LI></OL><P>We expect results on the 10th (Friday).
</P><P>We also began PCR to add the Bgl2 restriction site to some RFP and YFP (Yellow Fluorescent Protein) coding sequences (with no promoters). Bands on a gel showed that PCR apparently worked for the YFP but not the RFP. Nevertheless, we purified both and placed them in the pink box in the fridge. For the record, the YFP source is <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_E0030" class="external text" rel="nofollow">BBa_E0030</A>.
</P><P><B>June 9:</B>
We started to insert YFP into our new vector. We digested both pSB1C3-lacZα and our YFP product with Bgl2 and SpeI. We then started a ligation reaction and left it overnight. There was some concern about contamination as the reaction mixture picked up a yellowish colour, possibly from dirty (unused but old) pipette tips.
</P><P>If successful, some plasmids should become: E-X-B-YFP-S-P
</P><P>Since there's no promoter and no lacZ gene, these should give white colonies on Xgal/IPTG media. We expect to do the transformations and plating tomorrow.
</P><P><B>June 10:</B>
We transformed and plated our pSB1C3 + YFP (+ PlaclacZα, which we are trying to excise) ligation onto Xgal/IPTG plates. There are 4 plates; we decided to investigate whether the heat shock part of the transformation protocol has any effect, thus 2 were not heat shocked.
</P><P>We also got back sequences of our vector (see June 8). Eugene says the sequence from clone 2 appears better than that of clone 1. <A href="http://www.geospiza.com/Products/finchtv.shtml" class="external text" rel="nofollow">FinchTV</A> is freely available software we can use. Allan has added his best guess for the exact sequence of clone 2 to the <A href="/Team:Edinburgh/Sequences" title="Team:Edinburgh/Sequences">sequences</A> page (the chromatogram is a mess in a few places, but those bits can be filled in from the expected sequence).
</P><P><B>We should probably use our pSB1C3-Plac-lacZα clone 2 as our vector of choice in the future.</B> (16 June: the others were discarded)
</P><P><B>June 13:</B>
Our transformed plates showed a few colonies which were white, and yellow under the blue light. Some others were white and not yellow under the light, and most were blue. We are working on the assumption that the yellow ones are having YFP transcribed and translated even without a promoter or obvious RBS in the biobrick. Four colonies have been streaked onto a new Cml/IPTG/Xgal plate.
</P><P>However, Chris mentioned the possibility that the original PCR template plasmid could have made it all the way through the process and be causing this. <B>But:</B> did the original plasmid confer Cml resistance? If not, this isn't something to worry about. The current iGEM distribution has BBa_E0030 on pSB1AK3. Was this the source?? <B>Need to check this.</B> [Update: no, the source was also on pSB1C3. But because it was quite large, a gel of a miniprep will show us if we have the correct thing.]
</P><P><B>June 14:</B>
We started bottled cultures of our pSB1C3-YFP for minipreps tomorrow.
</P><P>We also did a PCR of a whole pSB1C3 backbone. This is mostly for Chris's BioSandwich idea. It looked OK on a gel.
</P><P>[Update: the resulting sequence is shown <A href="/Team:Edinburgh/Sequences#pSB1C3_for_BioSandwich" title="Team:Edinburgh/Sequences">here</A>. By design, there is a bunch of stuff between the BglII site and the BioBrick suffix.]
</P><P><B>June 15:</B>
Lee and Sylvia made 4 minipreps of (what we hope is) our pSB1C3-YFP plasmid.
</P><P>They also started on Chris's <A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich">BioSandwich</A> test, doing a digest with:
</P><UL><LI> lacZα - digest with Bgl2/SpeI
</LI><LI> YFP - digest with Bgl2/SpeI
</LI><LI> Vector - digest with Bgl2/XbaI
</LI></UL><P>Each was purified on 5 uL glass beads and eluted to 10 uL elution buffer.
</P><P>Afterwards, these three fragments were ligated to their respective spacers, and left overnight at 16 C.
</P><P>Meanwhile, Allan tested the size of the (putative) pSB1C3-YFP by cutting it with EcoRI and running it on a gel. The size should be about 2700? It seemed about right or maybe slightly larger. Clone 1 was barely visible, but clones 2, 3, or 4 seem OK.
</P><P><B>June 16:</B>
Allan sent clones 2 and 3 of the putative pSB1C3-YFP off to be sequenced. Results can be expected on Monday 20th. Note that the labels given to the sequencing people do not match our own internal names. Specifically:
</P><UL><LI> &quot;1F&quot; and &quot;1R&quot; are our clone 2
</LI><LI> &quot;2F&quot; and &quot;2R&quot; are our clone 3
</LI></UL><P>Lee and Sylvia purified the ligations from yesterday.
</P><P>We then tried two ways of joining everything up (see <A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich">BioSandwich page</A> for more).
</P><OL><LI> Ligation independent cloning
</LI><LI> Overlap Extension PCR
</LI></OL><P>The PCR failed; Chris later discovered we used the wrong primers. The LIC transformants were plated and left incubating overnight.
</P><P><B>June 17:</B>
A few colonies were seen on the LIC plates, but all were white [Update: or not, see June 20].
</P><P>Sylvia and Lee started on a fresh attempt of the OEPCR [Update: it didn't work].
</P><P><B>June 20:</B>
Sequencing results came back from our putative pSB1C3-YFP. The sequences show that the fluorescent colonies we made in fact have part <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216011" class="external text" rel="nofollow">K216011</A>, which includes a promoter. It is thus not what we were trying to amplify. Rather, it's the original PCR template that made it all the way through the process, from PCR to transformations.
</P><P>It is likely that colonies on the plate that are white but <B>don't fluoresce</B> are what we actually wanted.
</P><P>Over the weekend, Chris discovered that there was in fact a colony from the Ligation Independent Cloning attempt that was blue and fluoresced yellow. This is being miniprepped.
</P><P>Some of the LacZα cloning vector (clone 2) is being grown up in LB + cml in the shaking incubator (the rightmost one). This will be maxiprepped tomorrow.
</P><P><B>June 21:</B>
Allan did the maxiprep of pSB1C3-LacZα. This seemed to go well and the product has been placed in the box. It has not yet been tested on a gel however [Update: See June 27].
</P><P>Lee and Sylvia did a miniprep of one blue, three colourless, and two yellow colonies from the BioSandwich Ligation Independent Cloning. After digesting to linearise, a band for the blue colony on a gel was very faint, almost invisible.
</P><P>The whole LIC transformation process was repeated today.
</P><P><B>June 22:</B>
Lee and Sylvia sent some minipreps from the LIC attempt to be sequenced.
</P><P><B>June 23:</B>
Sylvia and Lee made some competent <I>E. coli</I> cells since we were running low.
</P><P><B>June 24:</B>
The sequencing results came back for the Ligation Independent Cloning attempt of the <A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich">BioSandwich</A> protocol. One colony has successfully incorporated LacZ and YFP; also spacers are seen; all indications are it worked perfectly.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:182px;"><DIV class="thumbcaption">June 27: Worst gel ever. This has been &quot;enchanced&quot; in GIMP.</DIV></DIV></DIV><P><B>June 27:</B>
Allan did a digest (SpeI) on the LacZα maxiprep and ran it on a gel. A band of the correct size was sort of visible if you used your imagination, but it was very faint. The maxiprep DNA may be quite dilute. [Update: Chris later said he successfully used this as a vector.]
</P><P><B>June 29:</B>
At Chris's suggestion, Allan designed primers for <I>E. coli</I> malS and bglX genes.
</P><P><B>July 3:</B>
Chris ordered primers for the <I>malS</I> and <I>bglX</I> genes. They should arrive on Wednesday (6th). Phage <A href="http://www.neb.com/nebecomm/products/productn4018.asp" class="external text" rel="nofollow">M13mp18</A> has also been ordered.
</P><P><B>July 4:</B>
We could do with a source for Ice Nucleation Protein. Gary Loake in Plant Science informs us they have <I>Pseudomonas syringae</I> pv. tomato DC3000 and <I>Pseudomonas syringae</I> pv. phaseolicola. Both these strains have problems. DC3000 seems to just lack the ice nucleation protein gene. Phaseolicola has a slightly different gene, <I>inaZ</I> (<A href="http://www.genome.jp/dbget-bin/www_bget?refseq:NC_005773" class="external text" rel="nofollow">see here</A>) and the published sequence has a transposable element in it.
</P><P>The gene that seems to be most commonly used (<I>inaK</I>) was first gotten from the Korean Collection for Type Cultures strain 1832.
</P><P><B>July 6:</B>
PCR primers for <I>malS</I> and <I>bglX</I> arrived. Eugene showed us how to make stock solutions of PCR primers, and then how to make working solutions.
</P><P><B>July 7:</B>
Allan has attempted to design BioSandwich-compatible primers for many of the genes involved in the project. See <A href="/Team:Edinburgh/Assembly" title="Team:Edinburgh/Assembly">the Assembly page</A>.
</P><P>Lee and Sylvia performed PCR to try and isolate <I>malS</I> and <I>bglX</I> genes from <I>E. coli</I>. The desired products are:
</P><OL><LI><I>malS</I> with RBS, start and stop codons; normal RFC10 part.
</LI><LI><I>malS</I> from start codon, no stop codon, intended for fusions.
</LI></OL><OL><LI><I>bglX</I> with RBS, start and stop codons; normal RFC10 part.
</LI><LI><I>bglX</I> from start codon, no stop codon, intended for fusions.
</LI></OL><P>According to the gel, the first three of these worked successfully (though <I>bglX</I> #1 is faint). The fourth will be repeated later, maybe the third also.
</P><P><B>July 8:</B>
Allan and Yassen continued searching for genes and designing primers.
</P><P>Sylvia and Lee redid PCR on the <I>bglX</I> gene (which seems to have worked this time, due to a lower annealing temperature of 55 C), as well as starting the process of inserting <I>malS</I> into our vector, starting with a purification of the <I>malS</I> PCR product. 
</P><P><B>July 11:</B>
Lee and Sylvia started the procedure for digesting and ligating <I>malS</I> and <I>bglX</I> into pSB1C3.
</P><P><B>July 12:</B>
Lee transformed <I>malS</I> and <I>bglX</I> into cells on chloramphenicol.
</P><P>We have also acquired the pG8SAET vector for display of proteins on pVIII; this might be useful as a fallback system. Lee has transformed this into cells on ampicillin.
</P><P><B>July 13:</B>
The pG8SAET transformation has worked and the plate has been placed in the cold room. Any colony on the plate should have the plasmid.
</P><P>New cml/Xgal/IPTG plates have been made and put in the cold room. We also seem to have acquired a bunch of Ampicillin plates.
</P><P>The two <I>malS</I> and the two <I>bglX</I> transformations seem to have worked (all gave a mix of white and blue colonies). Four white colonies from each have been streaked out onto plates and placed in the 37 C incubator.
</P><P>Overnight cultures of 2 colonies from each transformation have been started (8 colonies total). The source plates are still in the incubator and must be removed tomorrow...
</P><P>FedEx sent Allan a tracking thingy for BBa_K265008 - it seems to be getting treated as high priority and is supposed to arrive by July 15, 2011, 6:00 PM. It should be coming on plasmid pSB1AK3, so when it arrives it needs to be plated onto ampicillin.
</P><P><B>July 14:</B>
Lee did minipreps of the eight colonies. Afterwards, samples were cut with EcoRI and run on a gel:
</P><OL><LI> Ladder
</LI><LI><I>malS</I> f1/r1 colony 1
</LI><LI><I>malS</I> f2/r2 colony 1
</LI><LI><I>bglX</I> f1/r1 colony 1
</LI><LI><I>bglX</I> f2/r2 colony 1
</LI><LI><I>malS</I> f1/r1 colony 2
</LI><LI><I>malS</I> f2/r2 colony 2
</LI><LI> Ladder
</LI></OL><P>Allan prepared sequencing reactions (minus primers) for all the above &quot;colony 1&quot; minipreps. Depending on Lee's gel results stuff will be sequenced...
</P><P><B>July 15:</B>
Eugene has seen the gel and says it looks OK, so Allan sent forward and reverse sequencing reactions off for all the &quot;colony 1&quot; samples.
</P><P>In other news, the INP part <A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A> arrived from Boston; some bacteria were streaked from the agar onto an Amp50/Kan10 plate that Eugene happened to have; it was then placed in the incubator.
</P><P><B>July 18:</B>
Over the weekend Eugene has grown up some <A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A> in liquid culture and replated onto Ampicillin. Chris wants to miniprep this and get it sequenced. Eugene started overnight cultures.
</P><P>Eugene informs us there's a failure in one of the University's 3730 sequencers, which will likely delay sequencing of the samples sent on the 15th.
</P><P><B>July 19:</B>
Sequencing results came in on time. All four sequences look good. So all our &quot;colony number 1&quot; minipreps for <I>bglX</I> and <I>malS</I> are OK.
</P><P>Eugene has miniprepped the INP (<A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A>) part, still in vector pSB1AK3. He digested two different minipreps with EcoRI and ran both on a gel. They gave identical results, though the bands look too high up. We expect about 4.1 kb (part + plasmid) yet the bands suggest over 5 kb. We'll sequence it anyway.
</P><P><B>July 20:</B>
Allan sent INP miniprep 2 (in pSB1AK3) off to be sequenced. The samples are labelled simply F and R.
</P><P><B>July 22:</B>
The INP sequence is a spot on match to what's expected. The part came in <A href="http://dspace.mit.edu/bitstream/handle/1721.1/45139/BBFRFC12.txt?sequence=1" class="external text" rel="nofollow">RFC12</A> format and so has a non-standard prefix and suffix. The actual suffix contains an extra base that it shouldn't. This information was entered into the relevant <A href="http://partsregistry.org/Part:BBa_K265008:Experience" class="external text" rel="nofollow">experience page</A> on Partsregistry.org.
</P><P><B>July 25:</B>
Attending the UK meetup in Norwich. About this time, a whole bunch of primers were ordered, for pVIII, cellulases, and xylose isomerase.
</P><P><B>July 27:</B>
Lee and Sylvia did PCR for:
</P><UL><LI> The pVIII leader sequence
</LI><LI> The pVIII main sequence
</LI><LI> The <I>xylA</I> (xylose isomerase) gene
</LI></UL><P><B>July 28:</B>
The gel from yesterday showed bands of about the correct sizes for the two pVIII sequences. The xylose isomerase however is absent and is being redone at a lower annealing temperature of 50 C.
</P><P>This worked.
</P><P>Sylvia and Lee began work to add a Plac-RBS-LacZ construct (<S><A href="http://partsregistry.org/Part:BBa_J33207" class="external text" rel="nofollow">BBa_J33207</A></S> [Update 9 Aug: it seems <A href="http://partsregistry.org/Part:BBa_K523000" class="external text" rel="nofollow">BBa_K523000</A> was used]) upstream of the RBS-<I>malS</I> BioBrick, and a Plac-RBS-LacZ-RBS construct (<A href="http://partsregistry.org/Part:BBa_J33207" class="external text" rel="nofollow">BBa_J33207</A>+<A href="http://partsregistry.org/Part:BBa_J15001" class="external text" rel="nofollow">BBa_J15001</A>) upstream of the (no RBS) <I>malS</I> BioBrick, in both cases using <A href="http://partsregistry.org/Assembly:Standard_assembly" class="external text" rel="nofollow">standard assembly</A>.
</P><P>Successful clones, having gained the promoter (and RBS in case 2) should thus be blue.
</P><P>In other news, Chris informs us that the lac promoter is already extant in the lab as a BioSandwich part, which will save us the trouble of creating it.
</P><P><B>July 29:</B>
We had some starch agar helpfully made by the 6th floor lab tech people. There's now a bunch of plates of the stuff (with cml + IPTG) in the cold room. The recipe calls for:
</P><UL><LI> 1% starch
</LI><LI> Half-strength nutrient agar
</LI></UL><P>Also, the following PCR reactions were done:
</P><UL><LI> INP to add BglII and SpeI sites
</LI><LI> Exoglucanase to add BglII and SpeI sites
</LI><LI>β-glucosidase to add BglII and SpeI sites
</LI></UL><P>And transformations were done from the previous day's assembly.
</P><P><B>August 1:</B>
The gels for the cellulases were a mess (perhaps because of annealing temperatures) so PCR is being redone. The PCR reaction for the INP was done seperately and the gel looks good.
</P><P>MABEL is being done on the Endoglucanase plasmid to remove the BglII site.
</P><P>Four colonies of the Plac-RBS-LacZ-RBS-MalS[f1r1] successful transformations are being cultured overnight for minipreps. So are three colonies (numbers 2-4) of Plac-RBS-LacZ-RBS-MalS[f2r2], though this has no stop codon and so seems of dubious value.
</P><P>We decided to streak the four [f1r1] colonies onto starch also, so we could assay for starch degradation tomorrow; our first real experiment...
</P><P><B>August 2:</B>
Minipreps are being made of the Plac-malS cultures. The starch plate has growth on it but could probably do with a bit more time before the iodine-starch assay is carried out.
</P><P>The MABEL'd endoglucanase plasmid has been transformed into cells.
</P><P>Preparation is in progress for a real BioSandwich attempt to construct an INP-malS fusion.
</P><P>First the parts will have to be digested with the appropriate enzymes, as described on the <A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich">BioSandwich page</A>. Then spacers will have to be annealed as follows:
</P><UL><LI><B>Vector</B></LI><LI> Upstream spoligos: 
</LI><LI> RBS-2 FS
</LI><LI> RBS-2 RL
</LI></UL><LI> Downstream spoligos:
</LI><LI> Start FS
</LI><LI> Start RT
</LI><LI><B>Plac-LacZ</B></LI><LI> Upstream spoligos: 
</LI><LI> Start FS
</LI><LI> Start RL
</LI><LI> Downstream spoligos:
</LI><LI> RBS-1 FS
</LI><LI> RBS-1 RT
</LI><LI><B>INP</B></LI><LI> Upstream spoligos:
</LI><LI> RBS-1 FS
</LI><LI> RBS-1 RL
</LI><LI> Downstream spoligos:
</LI><LI> Linker FS
</LI><LI> Linker RT
</LI><LI><B>malS [f2/r2]</B></LI><LI> Upstream spoligos:
</LI><LI> Linker FS
</LI><LI> Linker RL
</LI><LI> Downstream spoligos:
</LI><LI> RBS-2 FS
</LI><LI> RBS-2 RT
</LI><P>Note that the RBS-2 spacer is not actually being used for its RBS, but just because its the only spacer we have left.
</P><P>The final construct would be:
</P><UL><LI> Vector - (spacer) - PlacLacZ - (RBS) - INP - (Linker) - malS - (spacer)
</LI></UL><P><B>August 3:</B>
Some Plac-LacZ-malS minipreps were sent for sequencing. [Update: these were Plac-LacZ-RBS-malS(f2r2) colonies 2 and 3; in neither case had standard assembly worked, see August 5]
</P><P>The BioSandwich annealments outlined above were set up. Ligation proceeded overnight.
</P><P><B>August 4:</B>
The 4 colonies of (putative, awaiting sequencing) <A href="http://partsregistry.org/Part:BBa_K523006" class="external text" rel="nofollow">BBa_K523006</A> were tested for starch degradation by flooding their starch agar plate with iodine. None showed a zone of clearing.
</P><P>The BioSandwich part/spoligo fragments have been purified. A gel was run to check for removal of excess oligos, but in fact everything had disappeared. [Update: in fact the quantity of DNA present is expected to be too low to see, so this gel is meaningless. In all probability the ligation and purification worked.]
</P><P>Sylvia and Lee made attempts to clone exoglucanase, C. fimi beta glucosidase and xylose isomerase into pSB1C3 vector. Ligation proceeded overnight.
</P><P><B>August 5:</B>
The sequencing results for clones 2 and 3 of our attempt to make <A href="http://partsregistry.org/Part:BBa_K523007" class="external text" rel="nofollow">BBa_K523007</A> show that assembly failed for those clones; the plasmids only contain Plac-RBS-LacZ-RBS (i.e. <A href="http://partsregistry.org/Part:BBa_K523005" class="external text" rel="nofollow">BBa_K523005</A>).
</P><P>We accidentally never got our attempts at Plac-RBS-LacZ-RBS-malS[f1r1] (i.e. <A href="http://partsregistry.org/Part:BBa_K523006" class="external text" rel="nofollow">BBa_K523006</A>) sequenced. This is the one that actually matters for control purposes, since it contains the <I>malS</I> gene with a proper stop codon. All four minipreps of this have now been sent off for sequencing.
</P><P>Lee and Sylvia continued to transform exoglucanase, C. fimi beta glucosidase and xylose isomerase into cells. Cloning of INP into pSB1C3 vector commenced. 
</P><P>Besides, work on BioSandwich continued. PCR was done for spacer1-INP-linker1 and linker1-malS-spacer2.
</P><P>
Chris informs us that the Ligation Independent Cloning version of BioSandwich assembly has not been working well; in 4 attempts there has been one successful colony. However, the OEPCR version of the protocol seems to work with some reasonable success rate (&quot;thousands of colonies&quot;) though a significant number of these are not the desired construct.
</P><P><B>August 6:</B>
Sylvia and Lee transformed INP into cells. Colonies from exoglucanase, C. fimi beta glucosidase and xylose isomerase were replated into chloramphenicol plates. 
</P><P>Gel electrophoresis was performed to confirm the PCR product of spacer1-INP-linker1 and linker1-malS-spacer2.
However, the expected bands were not seen for both samples. 
</P><P><B>August 7:</B>
Lee and Sylvia replated colonies of INP onto fresh chloramphenicol plates. We also made overnight culture for exoglucanase, C. fimi beta glucosidase and xylose isomerase as preparation for miniprep. 
</P><P><B>August 8:</B>
Sylvia and Lee continue with miniprep of exoglucanase, C. fimi beta glucosidase and xylose isomerase and sent them for sequencing.  
</P><P>The MABEL site-specific mutagenesis is causing problems; no colonies were seen after transformation. We realised this was because the plasmid encodes kanamycin and not chloramphenicol resistance.
</P><P>The modified INP (with BglII site and terminal &quot;gg&quot; bases, <A href="http://partsregistry.org/Part:BBa_K523008" class="external text" rel="nofollow">BBa_K523008</A>) has been successfully cloned into pSB1C3, as far as we know, and is being miniprepped.
</P><P>In case BioSandwich doesn't work well enough, primers for a hypothetical straight-up Gibson assembly of the various INP constructs <A href="/User:Allancrossman/Gibson" title="User:Allancrossman/Gibson">have been designed</A>. INP and malS have been ligated to oligos and the both ligations are used as templates for PCR and the PCR products were purified. The purified PCR product will be checked using gel electrophoresis the following day. 
</P><P>We used cen A mut miniprep as template for PCR, ran a gel to confirm that the PCR product was of expected size and purified the PCR product. 
</P><P><B>August 9:</B>
All of the putative Plac-LacZ-RBS-malS[f1/r1] sequences appear to be simply <A href="http://partsregistry.org/Part:BBa_K523000" class="external text" rel="nofollow">BBa_K523000</A>, our BglII-containing cloning vector. How is this possible? Presumably that was used as the Plac-LacZ source for the assembly. Anyway, it's failed.
</P><P>Lee and Sylvia miniprepped INP+pSB1C3 and digested it with EcoRI so that we can check it on the gel to confirm the size. We also performed linear digestion (EcoRI) for malS f1/r1, malS f2/r2, cex (exoglucanase), cf bglu (beta-glucosidase) and xyl A (xylose isomerase) in order to run them on the gel to check whether their sizes were correct as we forgot to run them on the gel before sending them for sequencing. 
</P><P>Sylvia ran a gel to confirm the purified PCR product of INP+oligos and malS+oligos. It appeared that for INP+oligos, expected bands were seen but not for malS+oligos. 
</P><P><B>August 10:</B>
Sequencing results came back for:
</P><UL><LI> our modified (BioSandwich ready) <I>C. fimi</I>β-glucosidase
</LI><LI> our modified (BioSandwich ready) <I>C. fimi</I> exoglucanase
</LI><LI> our modified (BioSandwich ready) xylose isomerase xylA
</LI></UL><P>For the β-glucosidase, there seems to be a frameshift in both clones that were sequenced (at the same place in both). [Update: after some discussion we believe the 2nd ATG in the gene is the true start codon. But this sequence will still not be usable for fusion proteins at its N terminal.]
</P><P>The sequences of clone 1 of the exoglucanase are bad. But clone 2 looks generally good but there are a couple of base changes, one of which is not synonymous (aspartic acid to glycine).
</P><P>Both sequences of xylA are bad.
</P><P>INP+pSB1C3 miniprep was sent for sequencing. PCR product of INP+oligos and malS+oligos were purified. The purified PCR product were then ran on a gel to confirm their sizes. Besides, primers for new linker (linker 2) arrived, thus ligation of malS and INP to new linker were done. 
</P><P>Lee and Sylvia prepared overnight culture of cex (exoglucanase) colony 3 and colony 4.
</P><P><B>August 11</B> 
PCR was done to amplify the amount of spacer 1-INP-linker 2 and linker 2-malS-spacer 2. The PCR products were checked on a gel to confirm that they are of correct sizes. A band of 1.0kb was seen for spacer 1-INP-linker 2 but no band for linker 2-malS-spacer 2. 
</P><P>Sylvia and Lee continued to miniprep cex (exoglucanase) colony 3 and colony 4, then performed a linear digestion with EcoRI and ran the minipreps on the gel. Only cex4 showed a strong band, at around 3.5 kb. So, cex4 was chosen to be sent for sequencing. 
</P><P>On the BioSandwich front, Chris tried two methods to assemble the BioSandwich parts (PlacLacZ-YFP); the Gibson method and LIC using blunt end PCR products. He gave Sylvia and Lee the plates in which he transformed the cells. From the Gibson method plate, 3 phenotypes were found; Blue and fluorescent (BY), Blue but not fluorescent (B) and White (W). Two colonies from each phenotype were inoculated into LB broth for overnight culture. From the LIC plate, there were 2 phenotypes; Blue (but not fluorescent) and white. One blue colony and 2 white colonies were grown overnight as preparation for miniprep.
</P><P><B>August 12:</B>
Sequencing came back for what was supposed to be a colony of our modified INP, but it was not present in the plasmid. Lee and Sylvia now plan to make minipreps of more white colonies from that transformation. INP colonies were grown overnight in culture to prepare for miniprep tomorrow. Miniprep was also continued for the BioSandwich colonies (9 colonies altogether). Linear digestion done for the miniprep and ran on gel. 
</P><P>Gel 1: Gibson method
Lane 1: 1kb DNA ladder
Lane 2: Blue 1
Lane 3: Blue 2
Lane 4: Blue and fluorescent 3
Lane 5: Blue and fluorescent 4
Lane 6: White 5
Lane 7: White 6 
Lane 8: 1kb DNA ladder 
</P><P>Gel 2: LIC 
Lane 1: 1kb DNA ladder
Lane 2: Blue 1
Lane 3: White 2
Lane 4: White 3
Lane 5: 1kb DNA ladder 
</P><P>
In case the worst comes to the worst and we need to start from scratch with INP, Allan recultured some colonies from the original pSB1AK3-K265008 plate and placed them in the incubator.
</P><P>Chris reports that the Gibson method of assembling BioSandwich parts is working about as well as OEPCR - that is, many correct colonies but also many incorrect ones.
</P><P>Exoglucanase colony 4 was sent for sequencing. New primers arrived for C. fimi beta glucosidase and BglX. 
</P><P>Lee and Sylvia attempted to make the INP-YFP construct. We tried ligating linker 1 to the first set and linker 2 to the second set. The final product will look like this:
</P><P>Spacer 1- INP – Linker 1 which will anneal to Linker 1- YFP – Spacer 2
Spacer 1- INP – Linker 2 which will anneal to Linker 2- YFP – Spacer 2
</P><P>Meanwhile,  malS and INP were also ligated to oligos.
</P><P><B>August 13:</B>
Sylvia and Lee miniprepped INP colony 3 and colony 4, performed linear digestion and ran the samples on a gel. Unfortunately, the gel does not look good. 
</P><P>Mabel was done for BglX to remove Pst I site. PCR reaction carried out for BglX and C.fimi beta glucosidase using new primers. 
</P><P>malS and INP that were ligated with oligos were purified; Purification removes excess oligos. The purified products will be used as templates for PCR and the PCR products will then be run on a gel to confirm ligation of parts to appropriate oligos. 
</P><P><B>August 14:</B>
We repeated cloning of INP colony 3 and colony 4 into pSB1C3 vector, ligation proceeded overnight. The PCR products of BglX and C.fimi beta glucosidase were purified and ran on a gel. 
</P><P>We also ran a PCR for malS+oligos and INP+oligos and used gel electrophoresis to confirm ligation. 
</P><P>Acquired Circular Polymerase Extension Cloning (CPEC) plates (PlaclacZ- YFP) from Chris- inoculated colonies into LB broth for miniprep tomorrow. 
</P><P><B>August 15:</B>
Allan's reculture of pSB1AK3 (from the cold room plate) produced only a single colony; hopefully it is the right thing. It was restreaked onto a new plate and placed in the incubator.
</P><P>Lee and Sylvia continue the main lab work such as transforming BglX f1/r1, BglX f2/r2 and INP into cells. C. fimi beta glucosidase was cloned into vector. 
</P><P>What we have done with the colonies from CPEC plates- we continued miniprep, digest with EcoRI and ran the samples on a gel to find which colony may have the correct PlaclacZ- YFP construct. The bands looked bigger than they were supposed to be, but later Chris informed us that he has added an extra 200 bases to the vector, pSB1C3. Besides, we have also made attempts on PCR-ing the YFP which has been ligated to either linker 1 or linker 2. 
</P><P>Linker 1- YFP- Spacer 2
Linker 2- YFP- Spacer 2
</P><P>We tried doing PCR reactions using Herculase and at two different temperatures, 65C and 58C. We will purify the PCR products tomorrow and ran them on a gel in hope of getting the right bands. 
</P><P><B>August 16:</B>
We have sent the colonies from CPEC plates for sequencing. We have sent one blue and yellow fluorescent colony, a blue colony but not fluorescent and a white colony. 
</P><P>The C. fimi beta glucosidase colony that was cloned into the vector has been transformed into cells. The colonies found on both MABEL’d BglX plates and the INP plate were replated.
</P><P>Purification of PCR products of YFP and oligos carried and ran on gel. We were relieved when we saw the correct bands for PCR products linker 2- YFP- spacer 2, for both PCR reaction carried out at 58C and 65C. 
</P><P>Chris taught Sylvia and Lee how to assemble BioSandwich parts using CPEC. It is basically doing a PCR reaction without using primers. The 4 parts+oligos we were trying to ligate together were as follows:
</P><P>Set 1
Spacer 2- vector- Start 
Start – PlaclacZ- Spacer 1
Spacer 1- INP – Linker 2
Linker 2- YFP – Spacer 2
</P><P>Set 2
Spacer 2- vector- Start 
Start – PlaclacZ- Spacer 1
Spacer 1- INP – Linker 1
Linker 1- malS – Spacer 2
</P><P>The parts that are ligated to oligos act as both templates and primers for the PCR, thus more of the parts must be added into the PCR tube, 2 microlitres recommended. As for malS+oligos, it showed a faint band on the gel, thus we have been advised to use a higher amount in the PCR reaction (3 microlitres). 
</P><P>After 3 cycles of PCR reaction, 5 microlitres of the PCR products are used to transform into cells. 
</P><P><B>August 17:</B>
Surprisingly, the transformation for CPEC and C.fimi beta glucosidase did not work. So, Lee and Sylvia attempted to re-clone C.fimi beta glucosidase into the vector again. We have also purified the PCR product of spacer1- INP – linker 2 and ran it on a gel so that we can use it for Gibson method of assembling BioSandwich parts tomorrow.
</P><P>Colonies from MABEL’d BglX f1/r1 and BglX 2/r2, and INP were inoculated into culture and grown overnight for miniprep tomorrow. 
</P><P>The MABEL on the endoglucanase has somehow given us a plasmid that contains only <A href="http://partsregistry.org/Part:BBa_J33207" class="external text" rel="nofollow">BBa_J33207</A>.
</P><P>We also received many sequencing results from BioSandwich tests... these colonies are supposed to code for LacZ and EYFP and so be blue but also fluoresce yellow.
</P><P>Note that after spacer2 we expect to see some extra bases before we reach the RFC10 suffix. I shall call these extra bases the &quot;stuffing&quot;. So, the expected sequence is:
</P><DL><DD> start1 -- PlacLacZ -- spacer1 -- EYFP -- spacer2 -- stuffing
</DD></DL><P><B>BioSandwich with Gibson:</B></P><UL><LI><B>gby [blue, yellow]</B> - expected scar is missing at approx base 680 of the forward chromatogram. Also, the spacer ends with ATG and the part starts with ATG, but only one ATG is seen. In total 9 bases are missing. There is homology of &quot;tatgg&quot; at both ends of the deleted fragment. There is a point mutation at approx base 500 of the forward chromatogram.
</LI><LI><B>gb [blue]</B> - there is a deletion of about 80 bases at approx base 700 of the chromatogram. This has been caused by homology of a &quot;gggcgagg&quot; found at both ends of the deleted sequence. The 9 bases mentioned for <B>gby</B> are also missing.
</LI><LI><B>gwy [yellow]</B> - seem correct except for a point mutation in the reverse chromatogram. The mutation is synonymous (ctg -&gt; ttg).
</LI></UL><P><B>BioSandwich with OEPCR:</B></P><UL><LI><B>bby [blue, yellow]</B> - mostly correct but the EYFP is followed not by spacer2 but by an RFC54 suffix, after which come a few mystery bases and the stuffing. [See update, below]
</LI><LI><B>bb [blue]</B> - has a massive deletion (several hundred bases) in the EYFP gene after about 60 bases. There is partial homology between the start and end of the deletion: &quot;tggtcgagctggac&quot; vs &quot;tggacgagctgtac&quot;. In addition to this, EYFP is not followed by spacer2 but instead by an RFC54 suffix (i.e. minus NotI). After this comes a few mystery bases, then the stuffing.
</LI><LI><B>bw [plain]</B> - has start1 joined to spacer1, then ~60 bases of the start of EYFP, the same massive deletion as <B>bb</B>, ~20 bases of the end of EYFP, an RFC54 suffix (i.e. minus NotI), then the stuffing.
</LI></UL><P><B>BioSandwich with Blunt-End LIC (expected to fail):</B></P><UL><LI><B>bb1 [blue]</B> - seems to be BBa_J33207 followed by part of <I>E. coli</I> ferrous iron transport gene A.
</LI><LI><B>bw2 [plain]</B> - has start1 followed by the stuffing.
</LI><LI><B>bw3 [plain]</B> - has RFC10 prefix followed by RFC10 suffix, as if XbaI and SpeI sites have joined in a normal vector.
</LI></UL><P>[Update 18 August] Chris informs us that the primers used for the OEPCR were not the normal ones, and in fact the expected product lacks spacer2 and has an RFC54 suffix.
</P><P><B>August 18:</B>
Transformed C. fimi beta glucosidase into cells. Sylvia and Lee also miniprepped MABEL’d BglX f1/r1 and BglX f2/r2 and INP.  Linear digestion of the two BglX with Pst I was carried out to investigate whether the Pst I site has been successfully removed. When ran on a gel, a single band appeared instead of two bands, thus we were confident of the results and will send the samples for sequencing tomorrow. 
We received results from some BioSandwich with CPEC tests...
</P><UL><LI><B>cby [blue, yellow]</B> - there is a single-base deletion near the start of the PlacLacZ part, with no obvious cause. It is too early to affect expression. The last few bases of spacer1 are missing, as well as the first few bases of the EYFP part (9 bases total). This is explained by homology of &quot;tatgg&quot;. Otherwise seems correct.
</LI><LI><B>cb [blue]</B> - Normal until the end of spacer1, which is followed immediately by spacer2 and the stuffing.
</LI><LI><B>cw [plain]</B> - start1 is followed immediately by a second copy of start1, after which comes spacer2 and a second copy of spacer2. Then the stuffing.
</LI></UL><P>Besides, Lee and Sylvia learned how to use Gibson method to assemble the BioSandwich parts together. We were required to put the parts ligated to oligos and the appropriate reagents into a tube and incubate in a 50C waterbath for an hour. Use 5 microlitres of the mixture for transformation and the remainder can be stored in the freezer. 
</P><P><B>August 19:</B>
Sylvia and Lee sent the minipreps for MABEL’d BglX f1/r1 and BglX f2/r2 and INP for sequencing. There were growth on the C.fimi beta glucosidase plate and so the colonies were replated into fresh plates. 
</P><P>For attempts on Gibson method, results were as follows:
</P><P>pSB1C3- PlaclacZ- INP- malS (100)
16 blue colonies and 74 white colonies were found, so we have replated some of them onto new plates. 
</P><P>pSB1C3- PlaclacZ- INP- YFP 
We have replated colonies from both 100 and 900 plates into fresh plates. There were supposingly 4 different phenotypes found on the plates, such as:
</P><P>Blue and Fluorescent
Blue but not fluorescent
White and fluorescent
White but not fluorescent
</P><P>However, the blue colour of the colonies may sometimes mask the fluorescence of the colonies and we might be able to distinguish Blue and Fluorescent from
Blue but not fluorescent. Thus we have replated blue colonies at random and will check how many of the blue colonies were actually fluorescent tomorrow. 
</P><P>We also attempted to make Vector-Placlacz-INP-malS and Vector-PlaclacZ-INP-YFP constructs using OEPCR. 
</P><P><B>August 20:</B>
Continuation of OEPCR- Lee and Sylvia ran a gel to confirm the sizes of OEPCR products. Unfortunately, the result on the gel was not good. 
</P><P>We have also set up overnight culture for C. fimi beta glucosidase and colonies from Gibson assembly method to make the following construct: pSB1C3- PlaclacZ – INP – YFP and pSB1C3- PlaclacZ – INP – malS. We were also supposed to replate the colonies from the CPEC plates but there was little growth observed on the plates. 
</P><P><B>August 21:</B>
Lee and Sylvia miniprepped C. fimi beta glucosidase and colonies from Gibson assembly plates. We have also replated colonies from CPEC plates. 
</P><P>Results for transformation of CPEC:
pSB1C3- PlaclacZ – INP – YFP
(100): 15 white colonies, no blue colonies found
(900): 1 Blue and Fluorescent colony, 24 white colonies
</P><P>
pSB1C3- PlaclacZ – INP – malS
(100): 4 blue colonies and 6 white colonies
(900): 32 blue colonies and 127 white colonies
</P><P><B>August 22:</B>
Sylvia and Lee performed linear digestion of the minipreps of C.fimi beta glucosidase and colonies from Gibson assembly plates with EcoRI and ran a gel to check their sizes. 
</P><P>We have also inoculated some colonies from CPEC plates into LB broth and grown them overnight for miniprep tomorrow. From plate for pSB1C3- PlaclacZ – INP – YFP, we took 1 blue and fluorescent colony and 3 white colonies. Meanwhile for pSB1C3- PlaclacZ – INP – malS, we took 2 blue and 2 white colonies. 
</P><P>We have also done PCR of Edinbrick (placlacZ) as we were running low. However, our PCR did not work at annealing temperature of 55C. We will have to re-do PCR tomorrow. 
</P><P>Primers for carotenoids (crt) arrived. Lee and Sylvia prepared working solution for the primers. 
</P><P><B>August 23:</B>
First thing in the morning, Lee and Sylvia sent the following for sequencing:
C.fimi beta glucosidase miniprep
pSB1C3- PlaclacZ – INP – malS (Colony Blue 1)
pSB1C3- PlaclacZ – INP – malS (Colony White 4)
pSB1C3- PlaclacZ – INP – malS (Colony White 5)
pSB1C3- PlaclacZ – INP – YFP (Colony White 3)
pSB1C3- PlaclacZ – INP – YFP (Colony Blue and Fluorescent 5)
pSB1C3- PlaclacZ – INP – malS (Colony Blue 1)
pSB1C3- PlaclacZ – INP – malS (Colony Blue 8)
</P><P>Transformation done for the blunt end ligation OEPCR products. Sylvia and Lee also miniprepped colonies from CPEC plates. We digested the minipreps with EcoRI and ran the samples on the gel, in search of the one with correct size and will send the sample for sequencing tomorrow morning. 
</P><P>As the PCR of Edinbrick (PlaclacZ) failed yesterday, we are repeating it today. We are also repeating PCR for endoglucanase. As we are short of time for today, we will run a gel tomorrow to check whether PCR has worked.
</P><P>We have also ligated the parts pVIII leader sequence, YFP and pVIII mature protein to corresponding oligos:
</P><P>Spacer 1- pVIII leader-linker 1 
Linker 1- YFP-linker 2
linker 2-pVIII- spacer 2
</P><P>Sequences came back for some stuff:
</P><UL><LI> We now have a perfect miniprep of INP for BioSandwich (i.e. with added BglII and GG), <A href="http://partsregistry.org/Part:BBa_K523008" class="external text" rel="nofollow">BBa_K523008</A></LI><LI> MABEL was successful in removing PstI from bglX (f1/r1, the non-BioSandwich version with RBS), <A href="http://partsregistry.org/Part:BBa_K523002" class="external text" rel="nofollow">BBa_K523002</A></LI><LI> MABEL was successful in removing PstI from bglX (f2/r2, the BioSandwich version), <A href="http://partsregistry.org/Part:BBa_K523004" class="external text" rel="nofollow">BBa_K523004</A></LI></UL><P><B>August 24:</B>
Again, the morning usually starts with sending samples for sequencing if there are any. We sent CPEC samples for sequencing: 
</P><P>For pSB1C3- PlaclacZ – INP – YFP, we sent samples from 1 blue and fluorescent colony and 1 white colony. Meanwhile for pSB1C3- PlaclacZ – INP – malS, we sent 1 blue and 2 white colonies.
</P><P>pVIII leader, YFP and pVIII that were ligated to oligos were purified and a PCR will be done and the PCR products will be ran on a gel to check whether we got the right parts+oligos. Unfortunately, we have forgotten that we should have used 100bp ladder as pVIII leader is around 90 bp and pVIII is around 175 bp.  For YFP+oligos, there was no band, so we have to repeat PCR tomorrow. 
</P><P>As for colonies from OEPCR plate, we have replated them onto fresh plates. 
Number and phenotypes of colonies from plates:
</P><P>pSB1C3- PlaclacZ – INP – YFP
(100): 1 white and fluorescent colony, no other phenotype found
(900): 1 blue, 7 white and fluorescent and 1 white colonies.
</P><P>pSB1C3- PlaclacZ – INP – malS
(100): 7 blue and 188 white colonies
(900): 165 blue colonies
</P><P>We then started PCR for the carotenoids; crt E, crt I and crt B. Gel electrophoresis used to check the sizes of the PCR products and the gel looks good. 
</P><P>
And last but not least, we ran a gel to check whether the PCR for Edinbrick and cenA mut has worked. Gel confirms that the PCR product of Edinbrick is correct. 
We then started to work on inserting PlaclacZ upstream of malS and BglX. 
</P><P><B>August 25:</B>
Lee and Sylvia repeated PCR for linker 1- YFP – linker 2 and cen A mut (endoglucanse with Bgl II site removed). Ran a gel to check the PCR products and also occupied a lane with 100bp ladder to check the pVIII leader+oligos and pVIII+oligos. The PCR products of the carotenoids looked good on a gel yesterday. So we continued to purify the PCR products and double-checked on a gel again, and the gel looks good. 
</P><P>PlaclacZ-malS and PlaclacZ-BglX were transformed into cells. 
</P><P>We have also set up overnight culture for OEPCR pSB1C3- PlaclacZ – INP – YFP
and pSB1C3- PlaclacZ – INP – malS. 
</P><P>Sequences came back for a lot of things:
</P><UL><LI><B>1/2: <I>C. fimi</I>β-glucosidase for BioSandwich, starting at the 2nd ATG of the template:</B> This is all correct except for a point mutation which turns an arginine into a cysteine.
</LI><LI><B>3/4: PlacLacZ-INP-malS (B1):</B> spacerT7, PlacLacZ, spacer1, INP, linker1, some mystery bases, and the stuffing.
</LI><LI><B>5/6: PlacLacZ-INP-malS (W4):</B> Only reverse sequence readable. INP present, malS not present.
</LI><LI><B>7/8: PlacLacZ-INP-malS (W5):</B> Has EcoRI site, NotI site, and PstI site.
</LI><LI><B>9/10: PlacLacZ-INP-YFP (BY):</B> Has the Start spacer (spacerT7) followed by Spacer2. Nothing else seen (how can this be blue and yellow?)
</LI><LI><B>11/12: PlacLacZ-INP-YFP (W3):</B> Has spacerT7, PlacLacZ, spacer1, and YFP
</LI><LI><B>13/14: PlacLacZ-INP-YFP (B8):</B> Has 2 copies of spacerT7, PlacLacZ, spacer1
</LI></UL><P>
The Plac-LacZ-INP construct has been saved with a label saying &quot;perfect sequence&quot; and today's date.
</P><P><B>August 26:</B>
We intended to replate PlaclacZ-malS and PlaclacZ-BglX but there were no colonies found. 
</P><P>We have transformed pG8SAET into JM109 cells. 
</P><P>Besides, we have also miniprepped colonies from OEPCR plates, Gibson assembly plates and CPEC plates. After that, we digested the minipreps with EcoRI and check them on a gel (except for CPEC minipreps).
</P><P>Next, we ran a PCR of PlaclacZ.INP. 
</P><P>New project for BioSandwich- assembling the carotenoids together with PlaclacZ. 
</P><P>Sequences came back for some attempts with CPEC:
</P><UL><LI><B>1/2: PlacLacZ-INP-YFP (BY1):</B> Unclear but not correct.
</LI><LI><B>3/4: PlacLacZ-INP-YFP (W4):</B> 2 copies of start spacer, 2nd of which is mutated and incomplete. Not sure afterwards.
</LI><LI><B>5/6: PlacLacZ-INP-malS (B2):</B> Chromotogram is missing the start area for some reason. Anyway, this contains start spacer, PlacLacZ, spacer1, INP, unsure afterwards; reverse sequence not readable.
</LI><LI><B>7/8: PlacLacZ-INP-malS (W3):</B> Unreadable.
</LI><LI><B>9/10: PlacLacZ-INP-malS (W4):</B> Chromotogram is missing the start area for some reason. Unclear.
</LI></UL><P><B>August 27:</B>
We have transformed PlaclacZ-malS and PlaclacZ-BglX into cells first thing in the morning. We ran a gel for PCR product of PlaclacZ. INP and CPEC digested minipreps that were left out yesterday. 
</P><P>Since the PCR for PlaclacZ.INP was successful, we intended to ligate this part upstream of exoglucanase and malS.  
</P><P>Besides, we tried cloning xylose isomerase into pSB1C3 again. 
</P><P><B>August 28:</B>
After allowing ligation of PlaclacZ.INP and exoglucanase with oligos to proceed overnight, we continued to purify the ligation to remove excess oligos. Then, we ran a PCR to confirm whether the parts have ligated to oligos. 
</P><P>We transformed xylose isomerase into cells. Since there was no growth on PlaclacZ-BglX plates, we have decided to re-transformed the construct into cells. For PlaclacZ-malS, we found some colonies and re-plated them into fresh plates. 
</P><P><B>August 29:</B>
We started the morning labwork by sending samples from OEPCR, CPEC, Gibson (PlaclacZ-INP-malS and PlaclacZ-INP-YFP) minipreps for sequencing. 
</P><P>We ran a gel to confirm the PCR products of PlaclacZ.INP+oligos and exoglucanase+oligos. There was a band for PlaclacZ.INP+oligos but not exoglucanase+oligos. 
</P><P>For xylose isomerase which we have transformed yesterday, we replated the colonies found on the plates today. For PlaclacZ-BglX, it was rather disappointing as we did not find any blue colonies. Transformation has to be repeated again. For PlaclacZ-malS, we are in the stage of growing the colonies overnight in culture for miniprep tomorrow. 
However, we have also attempted to add PlaclacZ upstream of MABEL’d BglX. 
</P><P>We intended to create the following BioSandwich constructs:
</P><P>PlaclacZ-INP-malS
PlaclacZ-INP-Cfbglu (C.fimi beta glucosidase)
PlaclacZ-INP-BglX 
</P><P>Thus, we have to digest our malS, C.fimi beta glucosidase and BglX with Bgl 2 and Spe 1 to create sticky ends, in order to ligate with oligos. Ligation of each parts with oligos linker 2 and spacer 2 proceeded overnight. 
</P><P><B>August 30:</B>
We started with miniprep of PlaclacZ-mals in the morning. Next, we digested the minipreps with EcoRI and performed a gel to confirm the sizes of digests. Besides, we have inoculated some colonies from xylose isomerase and set up overnight culture for miniprep tomorrow. 
</P><P>Ligation of the parts malS, Cfbglu (C.fimi beta glucosidase)
and BglX with oligos are being purified to remove excess oligos and a PCR will be run and then gel electrophoresis to check the sizes of parts and oligos. We are doing PCR at 58C and 62C to see which temperature will work best. 
</P><P>We have also transformed PlaclacZ-MABEL’d BglX into cells. For PlaclacZ-exoglcanase, we tried cloning the construct into pSB1C3. Besides, we have also tried cloning p8 leader and p8 mature protein into pSB1C3 vector. We allowed ligation to proceed overnight. 
</P><P>Once again, we were running low on the source of PlaclacZ, thus we have decided to do PCR of EdinBrick (PlaclacZ).  
</P><P><B>August 31:</B>
Again, our morning started with sending some samples for sequencing. This time it is PlaclacZ-malS. A busy morning will never be complete without miniprep. We miniprepped xylose isomerase and digested the minipreps with EcoRI and ran the samples on a gel. Those that looked like they have the right bands will be sent for sequencing tomorrow. 
</P><P>We wanted to replated PlaclacZ-MBEL’d BglX but was rather daunting that there was no blue colonies found. 
</P><P>BioSandwich news of the day- We checked the PCR products of the ligation of exoglucanase, C. fimi beta glucosidase, malS and BglX with oligos on a gel. But it was again disappointing that the gel did not look so good. However, that did not dampen our spirit and so we repeated PCR at 50C and 56C and check the PCR products on a gel. But still, the gel did not look so good. We ran the PCR product of PlaclacZ from yesterday on a gel.
</P><P>New primers arrived for spacer 4 and Gibson primers for malS and p8. We made preliminary attempts to use Gibson assembly to fuse malS and p8 together. Thus we used the Gibson primers to PCR p8 and malS. 
</P><P><B>September 1:</B>
We sent xylose isomerase minipreps for sequencing. We also transformed p8 leader and p8 into cells. 
</P><P>Once again, time for some update on BioSandwich- we ligated the parts exoglucanase, C. fimi beta glucosidase, BioSandwich version of malS and BglX  to linker 2 and spacer 2. We will use these parts+oligos to make construct PlaclacZ-INP-parts. In addition, we have ligated crt E, crt I and crt B to oligos. 
</P><P>Today we tried inserting PlaclacZ +RBS upstream of exoglucanase and C.fimi beta glucosidase. Working in parallel, Eugene attempted on putting PlaclacZ upstream of MABEL’d BglX. 
</P><P>We ran a gel to confirm the PCR products of p8 and malS (the ones produced using Gibson primers in PCR).
</P><P>Sequences came back for various PlacLacZ-INP-malS and PlacLacZ-INP-YFP attempts.
</P><P>In general the sequences are a bit weak, i.e. the strength falls off after a few hundred bases.
</P><UL><LI><B>1/2: malS by OEPCR (B2):</B> startT7, PlacLacZ, spacer1, INP, linker1, spacer2, stuffing (malS not present)
</LI><LI><B>3/4: malS by CPEC (B2):</B> startT7, PlacLacZ, spacer1, INP, linker1, spacer2, stuffing (malS not present)
</LI><LI><B>5/6: YFP by Gibson (BY2):</B> startT7, PlacLacZ [......] YFP, spacer2, stuffing (sequence too weak to check for INP)
</LI><LI><B>7/8: YFP by Gibson (BY3):</B> startT7, PlacLacZ, spacer1, INP [...] YFP, spacer2, stuffing (sequence too weak to check for linker1)
</LI><LI><B>9/10: YFP by Gibson (BY4):</B> startT7, PlacLacZ [......] YFP, spacer2, stuffing (sequence too weak to check for INP)
</LI><LI><B>11/12: YFP by Gibson (BY6):</B> startT7, PlacLacZ [......] YFP, spacer2, spacer2 repeated, stuffing (sequence too weak to check for INP)
</LI></UL><P>It seems that Gibson colony BY3 is the one we can be most confident about.
</P><P><B>September 2:</B>
Since the PCR product for p8 and malS looked good on the gel, we went on to purify them and ran them on a gel again. Unfortunately, Lee lost the PCR products during purification and PCR is being repeated for both p8 and malS. We ran a gel to confirm the PCR products and they looked good on the gel.
</P><P>On the BioSandwich front, ligation of crt+oligos and the cellulases (exoglucanase, C. fimi beta glucosidase, BioSandwich version of malS and BglX) that were ligated to linker 2 and spacer 2 are being purified. A PCR will be done and we will run a gel to check the bands. 
</P><P>We observed that there was no growth on the p8 leader and p8 plates. Thus we have attempted to clone p8 leader and p8 mature protein into pSB1C3 again. 
</P><P>We set up overnight culture of pSB1C3-PlaclacZ.INP-YFP. 
</P><P>Chao Kuo gave us the PlaclacZ-RBS construct which was in pSB1A2. We tried cloning this into pSB1C3. 
</P><P>We have also transformed PlaclacZ-RBS-exoglucanase and PlaclacZ-RBS-C. fimi beta glucosidase into cells. Since we now have the PlaclacZ-RBS construct, we tried inserting it upstream of malS and BglX. 
</P><P>Sequences came back for some attempts at PlacLacZ-malS (<A href="http://partsregistry.org/Part:BBa_K523006" class="external text" rel="nofollow">BBa_K523006</A>)...
</P><UL><LI><B>1/2:</B> Looks good.
</LI><LI><B>3/4:</B> Looks good.
</LI><LI><B>5/6:</B> Looks good.
</LI><LI><B>7/8:</B> Looks good.
</LI></UL><P>The colonies were streaked onto a starch plate (and also a normal plate by accident) and put in the incubator.
</P><P>Allan discovered that some of the starch plates in the cold room are contaminated with some sort of red bacteria. He therefore threw a bunch of plates out (seven remain).
</P><P><B>September 3:</B>
We ran a gel to confirm the PCR products of p8 and malS (using Gibson primers) and they looked good on the gel. Thus we proceeded with purification of the PCR products. Next, we used Gibson assembly to assemble the two parts together and transformed them into cells. We plated these cells on chloramphenicol plates. (Note: This is a mistake as pG8SAET vector we used has ampicillin resistance instead of chloramphenicol. See update later.)
</P><P>We also run a gel for PCR products of crt EIB +oligos. They looked very good on gel, having correct and strong bands. We also went on to purify the PCR products and ran them on a gel again. They still look good on a gel after purification. 
</P><P>On the flip side, there were some daunting news on the plates of PlaclacZ-RBS-exoglucanase and PlaclacZ-RBS-C. fimi beta glucosidase. We did not find any blue colonies. 
</P><P>We continued transformation of PlaclacZ-RBS, p8 leader and p8 into cells. 
</P><P><B>September 4:</B>
Lee and Sylvia replated PlaclacZ-RBS, p8 leader and p8 onto fresh plates. 
</P><P>Mission for today includes using Gibson method to assemble BioSandwich parts together. We will assemble the following:
</P><P>P8-malS
PlaclacZ-INP-BglX
PlaclacZ.INP-BglX
Set 1 of carotenoids: spacer 4-pSB1C3-start-PlaclacZ-spacer 1-crtE-spacer 2-crtI-spacer 3-crtB-spacer 4
</P><P>(Side note: We intended to construct 3 different sets for carotenoids, each set having carotenoids at different order. Refer to the following:
</P><P>Set 2: spacer 4-pSB1C3-start-PlaclacZ-spacer 2-crtI-spacer 3-crtB-spacer 1-crtE-spacer 4
</P><P>Set 3: spacer 4-pSB1C3-start-PlaclacZ-spacer 1-crtB-spacer 2-crtE-spacer 3-crtI-spacer 4)
</P><P>Since we only have the BioSandwich parts of Set 1 ready, we proceeded with that set first. We have also attempted to create BioSandwich part: start-PlaclacZ-spacer 2 for assembly of Set 2 crt.
</P><P><B>September 5:</B><B>September 5:</B>
Chris attempted to MABEL the endoglucanase, cen A. He gave us the cen A mut sample and asked us to send it for sequencing using pSB1A2 as the endoglucanase is still in this vector instead of pSB1C3. 
</P><P>We check the PCR products of exoglucanase, C. fimi beta glucosidase and malS which have been ligated to linker 2 and spacer 2 on the gel. Unfortunately, our template proved to be difficult and thus the result on the gel did not looked good. 
</P><P>We have also repeated transformation of p8-MalS and this time plating the cells onto ampicillin plate. 
</P><P>BioSandwich mission of the day- To use CPEC to assemble BioSandwich parts of the following constructs:
</P><P>Set 1 crt: spacer 4-pSB1C3-start-PlaclacZ-spacer 1-crtE-spacer 2-crtI-spacer 3-crtB-spacer 4
PlaclacZ-INP-BglX
</P><P>(Side note: We made a mistake of not checking the number of cycles on the PCR machine. It was re-set to 30 cycles when usually the programme for CPEC is only supposed to last for 3 cycles. When Sylvia and Lee realized that the PCR was running too long (was only supposed to take half an hour, the PCR was already in its 7th cycle. Thus, we stopped the PCR reaction but chose to transformed the PCR products anyway, out of curiousity. We then repeated the PCR according to the real procedure, which is running PCR for 3 cycles.)
</P><P>For the BioSandwich part start-PlaclacZ-spacer 2, we ran a PCR and then gel electrophoresis. Unfortunately, the gel result was not good and hence we did not managed to do Gibson assembly for set 2 crt. However, this did not hamper our intention to repeat the PCR of the BioSandwich part. In addition, we also do PCR for linker 2-exoglucanase-spacer 2, linker 2-C. fimi beta glucosidase-spacer 2 and linker 2-malS-spacer 2, this time using two protocols; Kod Hot Start Extreme and Herculase and at 56C and 65C. 
</P><P>On the other hand, Sylvia counted the colonies on the Gibson plates:
</P><P>PlaclacZ.INP-MABEL’d BglX
(100): 57 white colonies, no blue colonies
(900): 1 blue colony and 575 white colonies
</P><P>PlaclacZ-INP-MABEL’d BglX
(100): 54 white and 16 blue colonies
(900): 156 blue and 423 colonies
</P><P>PlaclacZ-crtE-crtI-crtB (set 1)
(100): 2 white colonies
(900): 1 blue and 80 white colonies
</P><P>We repeated the cloning of xylose isomerase into pSB1C3. 
</P><P>We grow a few colonies from p8 leader and p8 overnight in culture for miniprep tomorrow. PlaclacZ-RBS failed to give blue colonies. We informed Chao Kuo about this and he advised us to do a PCR using a template given by him to get the construct again. 
</P><P>We tried the iodine assay on the Plac-malS colonies. There was no zone of clearing, though the area directly under the cells did not go dark so much. We really should do a proper experiment with a negative control, so Allan set that up.
</P><P>Sequences came back for some stuff (allegedly xylA):
</P><UL><LI><B>1/2:</B> Full prefix not present. Seems to incorporate Hemolysin genes (from the main genome?). Reverse sequence unreadable; perhaps suffix not present.
</LI><LI><B>3/4:</B> Unreadable.
</LI></UL><P><B>September 6:</B>
We replated our p8-malS onto a fresh ampicillin plate. We have also transformed xylose isomerase into cells. 
</P><P>We also miniprep p8 leader and p8, digest the minipreps with EcoRI and ran them on a gel. 
</P><P>For BioSandwich, we set up overnight culture for Gibson set 1 crt, PlaclacZ-INP-BglX and PlaclacZ.INP-BglX. We have also replated colonies for CPEC assembly for set 1 crt and PlaclacZ-INP-BglX.
</P><P>Results for the CPEC plates were as follow:
</P><P>PlaclacZ-crtE-crtI-crtB  (7 cycles)
(100): 31 white and 19 blue colonies
(900): 257 white and 149 blue colonies
</P><P>PlaclacZ-crtE-crtI-crtB (3 cycles)
(100): 1 white colony
(900): 15 white and 4 blue colonies
</P><P>PlaclacZ-INP-BglX (7 cycles)
(100): 21 white colonies
(900): 363 white colony, 1 blue colony and surprisingly 1 purple colony
</P><P>
PlaclacZ-INP-BglX (3 cycles)
(100): 75 white and 4 blue colonies
(900): 494 white, 36 blue and 2 purple colonies
</P><P>More interesting BioSandwich news, we are running OEPCR for set 1 crt and PlaclacZ-INP-BglX. 
</P><P>We attempted PCR for start-PlaclacZ-spacer 2 for set 2 crt. Then we used gel electrophoresis to check the size but no band was observed. 
</P><P>Exciting news on the PCR using Herculase and Kod Hot Start Extreme for exoglucanase, C.fimi beta glucosidase and malS which have been ligated to oligos- we ran a gel to confirm sizes of PCR products and found out that Herculase worked for both exoglucanase and C.fimi beta glucosidase but Kod Hot Start also worked for C.fimi beta glucosidase. We then purified these PCR products.
</P><P>Chao Kuo gave us the template M6LZR-3 for PCR of PlaclacZ-RBS. We did a PCR using this and ran a gel to confirm PCR product. A band appeared at 650bp which confirms that we managed to get hold of PlaclacZ-RBS.
</P><P>Allan's controlled malS experiment failed to use a chloramphenicol resistant <I>E. coli</I> as the control strain. He has corrected this and a new plate is incubating in the incubator.
</P><P><B>September 7:</B>
We ran a gel to confirm the OEPCR products, but rather daunting as the expected bands were not observed. 
</P><P>For PlaclacZ-RBS, we purified the PCR product and ran it on the gel again. The gel result looked good and thus we did not hesitate to clone this upstream of exoglucanase and C. fimi beta glucosidase. 
</P><P>Morning was also spent doing minipreps for Gibson set1 crt and PlaclacZ-INP-BglX. We then digested the minipreps with EcoRI and ran them on a gel. We have inoculated colonies from CPEC plates into LB broth for miniprep tomorrow. 
</P><P>More BioSandwich news- we have attempted to assemble the parts for the following constructs using Gibson assembly:
PlaclacZ-INP-exoglucanase 
PlaclacZ-INP-C. fimi beta glucosidase
P8-malS
</P><P>We have also attempted OEPCR for PlaclacZ-INP-exoglucanase and PlaclacZ-INP-C. fimi beta glucosidase. We realized that we were low on the BioSandwich part spacer 2-pSB1C3-start and start-PlaclacZ-spacer 1, hence we ran a PCR to amplify the amount of these. 
</P><P>We found some colonies on the xylose isomerase plates, hence replated them into fresh plates. 
</P><P>Even more exciting was that we were finally commenced work on assaying our exoglucanase and BglX. We have assayed their MUC and MUG activities. 
</P><P>Allan notes that the negative control has a somewhat different morphology from the malS streaks. The negative control is a failed (white) attempt at PlacLacZ-INP-bglX. Maybe it's just growing slower because of increased plasmid size. The plate has been left in the incubator for another day.
</P><P>Sequence results came back for an attempt to remove the BglII site from a plasmid that evidently contained:
</P><DL><DD> J33207 -- RBS -- CenA.
</DD></DL><P>The reverse read shows the BglII site has been eliminated perfectly. However there is an inversion about 100 bases further upstream, with &quot;gc&quot; becoming &quot;cg&quot;. The effect is to convert a Proline to an Alanine.
</P><P><B>September 8:</B>
Some sequencing for Eugene came back. He has successfully made bglX under the control of the lac promoter, yielding new part <A href="http://partsregistry.org/Part:BBa_K523014" class="external text" rel="nofollow">BBa_K523014</A>. 
Sylvia and Lee attempted on using CPEC and OEPCR for assembling the BioSandwich ready cfbglu and cex together with INP, PlacLacZ into pSB1C3.  They also repeated OEPCR for assembling the following constructs: PlacLacZ-INP-BglX and PlacLacZ-INP-crtE-crtI-crtB. The same constructs made using CPEC was miniprepped and linearized with EcoR1. The analytical digest was run on an agarose gel and only 3 of the minipresps (PlacLacZ-INP-crtE-crtI-crtB) were confirmed to be of the right size and potentially the right construct. PlacLacZ+RBS-cex and PlacLacZ+RBS-cfbglu were transformed into competent JM109 cells. PlacLacZ-INP-BglX made using Gibson assembly was sent for sequence. 
</P><P><B>September 9:</B>
The controlled experiment on malS looked promising, with the negative control going dark when iodine was poured on it while the Plac-malS colonies stayed clear...
The 3 potentially correct sequences for PlacLacZ-INP-crtE-crtI-crtB obtained using CPEC was sent off for sequencing by Sylvia and Lee. CPEC for Plac-LacZ-INP-cex and Plac-LacZ-INP-cfbglu assembly was repeated. Miniprep of XylA in pSB1C3 was preformed and linearized with EcoR1. This was consequently run on an agarose gel to confirm any correct bands, potentially having correct construct. Though, the result only showed one having the right size. Sonication and fractionation of our cells containing the construct PlacLacZ-INP-YFP and control: PlacLacZ-YFP was performed. 
</P><P><B>September 10:</B>
Sylvia and Lee miniprepped PlacLacZ+RBS-cex and PlacLacZ+RBS-cfbglu, Plac-LacZ-INP-cex and Plac-LacZ-INP-cfbglu that was assembled using Gibson assembly, PlacLacZ-INP-crtE-crtI-crtB and PlacLacZ-INP-BglX that was assembled using CPEC. 
</P><P><B>September 11:</B>
Miniprep of Plac-LacZ-INP-cfbglu and Plac-LacZ-INP-cex assembled using CPEC was performed. Linearized digestion with EcoR1 of these and the miniprep from the previous day was performed and run on an agarose gel. The results showed that only one of the PlacLacZ+RBS-cex construct, eight of PlacLacZ+RBS-cfbglu, two of Plac-LacZ-INP-cfbglu (Gibson), and four of Plac-LacZ-INP-cex (Gibson) seemed to be of the correct size. Further, one of the PlacLacZ-INP-BglX (CPEC) and one of Plac-LacZ-INP-cex (CPEC) also seemed of the correct size. 
</P><P><B>September 12:</B>
A number of sequences came back for PlacLacZ-INP-bglX. We expect:
</P><DL><DD> Vector -- spacerT7 -- PlacLacZ -- spacer1 -- INP -- linker2 -- bglX -- spacer2 -- stuffing
</DD></DL><P>We have:
</P><UL><LI><B>1/2 (colony 3):</B> spacerT7, PlacLacZ, spacer1, INP [...too deep...] bglX -- spacer2 -- stuffing; <B>seems OK</B></LI><LI><B>3/4 (colony 9):</B> INP is missing, two copies of spacer2; otherwise as expected.
</LI><LI><B>5/6 (colony 10):</B> Unreadable.
</LI><LI><B>7/8 (colony 11):</B> Noisy low-quality sequence.
</LI><LI><B>9/10 (colony 15):</B> Unreadable.
</LI><LI><B>11/12 (colony 16):</B> Forward reaction had not enough red dye, but sequence might be OK. Hard to tell.
</LI><LI><B>13/14 (as above but using a dedicated PlacLacZ-INP part):</B> bglX is missing.
</LI></UL><P>We also got back a sequence for crtEIB:
</P><UL><LI><B>15/16:</B> Unreadable.
</LI></UL><P>Sylvia and Lee sent all the potentially right constructs from the previous day for sequencing. PCR of cenA using herculase protocol was done. Sylvia also performed DNS assay of PlacLacZ-MalS. 
</P><P><B>September 13:</B>
Allan started the final, definitive test of Plac-malS, by making two starch plates with:
</P><UL><LI> PlacLacZ only (old cells, might be dead)
</LI><LI> PlacLacZ-bglX (another periplasmic protein as a control)
</LI><LI> PlacLacZ-malS (colony 2)
</LI><LI> PlacLacZ-malS (colony 4)
</LI></UL><P>We can expose one plate to iodine and leave the other alone, and see what happens.
</P><P>Some sequences came back... all are supposed to be crt assemblies
</P><UL><LI><B>1/2:</B> spacerT7, PlacLacZ, crtB, spacer4, stuffing
</LI><LI><B>3/4:</B> spacerT7, PlacLacZ [...] crtB, spacer4, stuffing
</LI><LI><B>5/6:</B> Unreadable
</LI></UL><P>Miniprep of pVIII cfus and pVIII nfus inserted into pSB1C3 was performed by Sylvia and Lee. Re-PCR of cenA using the herculase protocol. MUC and MUG plates were inoculated with PlacLacZ-BglX and PlacLacZ-INP-BglX. 
</P><P><B>September 14:</B>
A small number of sequences came back...
</P><P><B>In a suspicious number of the Gibson sequences, the sequence strength drops off all of a sudden. This can't be coincidence.</B></P><DL><DT><B>1/2</B> PlacLacZ + RBS + cex (5)
</DT><DD> Forward read unreadable. Reverse read shows (BioSandwich format) cex <A href="http://partsregistry.org/Part:BBa_K523009" class="external text" rel="nofollow">BBa_K523009</A>. This has no stop codon.
</DD><DT><B>3/4</B> PlacLacZ + RBS + Cf bglu (1)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>5/6</B> PlacLacZ + RBS + Cf bglu (2)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>7/8</B> PlacLacZ + RBS + Cf bglu (3)
</DT><DD> Forward read unreadable. Reverse read shows (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. No stop.
</DD><DT><B>9/10</B> PlacLacZ + RBS + Cf bglu (4)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>11/12</B> PlacLacZ + RBS + Cf bglu (5)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>13/14</B> PlacLacZ + RBS + Cf bglu (8)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>15/16</B> PlacLacZ + RBS + Cf bglu (9)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>17/18</B> PlacLacZ + RBS + Cf bglu (10)
</DT><DD> J33207 + J15001 + (BioSandwich format) Cf bglu <A href="http://partsregistry.org/Part:BBa_K523010" class="external text" rel="nofollow">BBa_K523010</A>. This has no stop codon.
</DD><DT><B>19/20</B> Gibson Cf bglu (B1)
</DT><DD> spacerT7 + PlacLacZ + [...]. Reverse read unreadable.
</DD><DT><B>21/22</B> Gibson Cf bglu (B3)
</DT><DD> spacerT7 + PlacLacZ + [...] + PlacLacZ <B>in reverse orientation</B> + mystery bases + spacer2 + stuffing
</DD><DT><B>23/24</B> Gibson cex (2)
</DT><DD> spacerT7 + PlacLacZ + [...] + Ice Nucleation Protein <B>in reverse orientation</B> + spacer1 <B>(reversed)</B> + spacer2 + stuffing (PstI site has mutated to ctgcgg).
</DD><DT><B>25/26</B> Gibson cex (5)
</DT><DD> spacerT7 + PlacLacZ + [...] + Ice Nucleation Protein <B>in reverse orientation</B> + spacer1 <B>(reversed)</B> + spacer2 + stuffing
</DD><DT><B>27/28</B> Gibson cex (8)
</DT><DD> Forward read starts in the middle of spacerT7. Then PlacLacZ + [...]. Poor forward sequence quality. Reverse sequence completely falls off after a couple of hundred bases.
</DD><DT><B>29/30</B> Gibson cex (10)
</DT><DD> spacerT7 + PlacLacZ + spacer1 + INP + [...] + PlacLacZ <B>in reverse orientation</B> + spacer2 + stuffing
</DD><DT><B>31/32</B> CPEC crt(7) (B5)
</DT><DD> spacerT7 + PlacLacZ + spacer1 + crtI + [...] + crtB + spacer4 + stuffing
</DD><DT><B>33/34</B> CPEC crt(7) (B8)
</DT><DD> spacerT7 + PlacLacZ + spacer1 + crtE + [...] + crtB + spacer4 + stuffing
</DD><DT><B>35/36</B> CPEC bglX (5)
</DT><DD> spacerT7 + PlacLacZ + spacer1 + spacer2 + stuffing
</DD><DT><B>37/38</B> CPEC cex (1)
</DT><DD> spacerT7 + PlacLacZ + [...] + cex + spacer2 + stuffing
</DD><DT><B>39/40</B> xylA (5)
</DT><DD> Part of main genome has inserted via EcoRI and PstI sites. Not xylA.
</DD><DT><B>41/42</B> xylA (10)
</DT><DD> PlacLacZ only.
</DD></DL><P>The digested miniprep of pVIII cfus and pVIII nfus was run on an agarose gel, of which only one of pVIII cfus seemed to be correct in terms of the size of the band. Maxiprep of all of our constructs was set up! DNS assay for our PlacLacZ-MalS construct was repeated. 
</P><P><B>September 15:</B>
Sylvia and Lee continued with maxiprep! The potentially correct pVIII cfus was sent for sequencing. 
</P><P><B>September 19:</B> 
Sequencing results came back for something. But whatever it was has been replaced by a mysterious sequence, inserting via EcoRI and SpeI. BLAST suggests it's some sort of insertion sequence or something.
</P><PRE style="margin-left: 1em; margin-right: 1em;">
GAATTCCACT TGCCTTTAAC AACTGTATTT CAGACGttAT TTtAGGATCT CCATTTTCTT
CGATTAATTC GAAAGATGCT TCTATTTTTT TTAAGCACGT ATAAACTGTT AATTCAGGTT
CAATGCTACG AAATGCACTA GT
</PRE><P><B>September 29:</B>
Sequencing results came back for INP-EYFP, INP-cex, and INP-bglX using a forward primer internal to INP, so that the join to the linker and the enzyme can be seen:
</P><UL><LI><B>INP-YFP</B>: seems correct though a different linker was used than the one which was initially given on the Registry. The Registry sequence has been corrected.
</LI><LI><B>INP-bglX</B>: the sequence is poor quality but it seems like INP connects to the linker OK; but the linker does not seem to connect properly to bglX, for unknown reasons.
</LI><LI><B>INP-cex</B>: it appears that a massive chunk of the 5' end of cex is just missing from the construct. This is probably because of (very close) homology between linker1 and part of cex. While linker1 was derived from the CenA sequence, it also has homology to cex:
</LI></UL><PRE style="margin-left: 1em; margin-right: 1em;">
linker1         1 --------ccgacgaccagccccacgccgaccccgacg------------ 30
                          ||||||.|||   |||||||||||||||||            
BBa_K523009   951 gcccgacgccgacgccca---ccacgccgaccccgacgcccacgacgccg 997
</PRE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Lab_Notebook">http://2011.igem.org/Team:Edinburgh/Lab_Notebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Lab_Notebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Lab_Notebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Lab_Notebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Lab_Notebook&amp;oldid=212785" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>