<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Modelling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Modelling</P><P>One way of assessing the feasibility of the <SPAN class="hardword" id="synergy">synergistic</SPAN> approach to biorefineries is to use <SPAN class="hardword" id="insilico">in silico</SPAN> modelling. In particular, we would like to confirm that synergistic use of enzymes can make the process of <SPAN class="hardword" id="cellulose">cellulose</SPAN> degradation more efficient.
</P><P>Some other models, calculations, and tools were also developed by the team.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Cellulase_models"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Cellulase models</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Approaches"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Approaches</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Results"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Comparison_of_different_modelling_tools"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Comparison of different modelling tools</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Other_models_and_calculations"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Other models and calculations</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Energy_efficiency"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Energy efficiency</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Evolutionary_analysis_of_cell-display_vs._secretion"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Evolutionary analysis of cell-display vs. secretion</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Phage_replication"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Phage replication</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#Genetic_Stability_Tool"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Genetic Stability Tool</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Cellulase_models">Cellulase models</SPAN></H2><H3><SPAN class="mw-headline" id="Approaches">Approaches</SPAN></H3><P>As it happens, our <A href="/Team:Edinburgh/Team" title="Team:Edinburgh/Team"> team</A> includes:
</P><OL><LI> Two engineers experienced in using MATLAB.
</LI><LI> An informatician who quickly learned the Kappa modelling language.
</LI><LI> A biologist who, for no good reason, knows the C programming language.
</LI></OL><P>This led to three different attempts to model <SPAN class="hardword" id="cellulase">cellulase</SPAN> action.
</P><H3><SPAN class="mw-headline" id="Results">Results</SPAN></H3><UL><LI><A href="/Team:Edinburgh/Synergy_(C_model)" title="Team:Edinburgh/Synergy (C model)" class="mw-redirect">C model</A>— a simple model that showed a difference between synergistic and non-synergistic systems
</LI><LI><A href="/Team:Edinburgh/Synergy_(Kappa_model)" title="Team:Edinburgh/Synergy (Kappa model)" class="mw-redirect">Kappa model</A>— a more complex model that also showed a difference
</LI><LI><A href="/Team:Edinburgh/Synergy_(MATLAB_model)" title="Team:Edinburgh/Synergy (MATLAB model)" class="mw-redirect">MATLAB model</A>— the most complex model, only worked well for the non-synergistic system
</LI></UL><H3><SPAN class="mw-headline" id="Comparison_of_different_modelling_tools">Comparison of different modelling tools</SPAN></H3><P>Our analysis of the advantages and disadvantages of the various modelling approaches, together with conclusions, are on our <A href="/Team:Edinburgh/Model_Comparison" title="Team:Edinburgh/Model Comparison"> model comparison</A> page.
</P><H2><SPAN class="mw-headline" id="Other_models_and_calculations">Other models and calculations</SPAN></H2><H3><SPAN class="mw-headline" id="Energy_efficiency">Energy efficiency</SPAN></H3><P>Consider this: for bacteria to produce <SPAN class="hardword" id="phage">phage</SPAN> or <SPAN class="hardword" id="inp">INP</SPAN> requires energy. This energy could have been spent producing extra copies of the cellulases. In order for the phage and cell display projects to make sense, the benefits of synergy must outweigh the cost of producing all these extra proteins.
</P><P>This question can probably be investigated using <A href="/Team:Edinburgh/Efficiency" title="Team:Edinburgh/Efficiency">simple maths and back-of-envelope calculations</A>...
</P><H3><SPAN class="mw-headline" id="Evolutionary_analysis_of_cell-display_vs._secretion">Evolutionary analysis of cell-display vs. secretion</SPAN></H3><P>One potential benefit of attaching enzymes to the cell surface rather than secreting them into the media is that any mutations that increase enzyme efficiency will specifically benefit the cell with the mutation, as the increased sugar yield will be physically present at the cell. The mutation will thus confer a fitness advantage, potentially allowing it to take over the culture.
</P><P>By contrast, if a cell produces a <I>secreted</I> protein that is of higher efficiency, it will disperse and benefit random cells in the culture. To investigate this difference, we created our <A href="/Team:Edinburgh/Artificial_Selection" title="Team:Edinburgh/Artificial Selection">artificial selection</A> model.
</P><H3><SPAN class="mw-headline" id="Phage_replication">Phage replication</SPAN></H3><P>The phage display system requires infected <I>E. coli</I> to dominate the system and not be outcompeted by uninfected <I>E. coli</I>. Our <A href="/Team:Edinburgh/Phage_Replication" title="Team:Edinburgh/Phage Replication">phage replication model</A> verifies that this is indeed the expected outcome.
</P><H2><SPAN class="mw-headline" id="Genetic_Stability_Tool">Genetic Stability Tool</SPAN></H2><P>Our projects involve having multiple fusion proteins expressed, each of which uses a genetically identical carrier protein (e.g. ice-nucleation protein or the M13 pVIII gene). The presence of repeated sequences in DNA (i.e. the same sequence in multiple locations) can lead to genetic instability.
</P><P>However, it ought to be possible to design and synthesise different versions of the genes that code for the same amino acids but use different codons and so are as distinct as possible.
</P><P>To prove this, we made our <A href="/Team:Edinburgh/Genetic_instability" title="Team:Edinburgh/Genetic instability" class="mw-redirect"> Genetic Stability Tool</A>.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Van Zyl WH, Lynd LR, Den Haan R, McBride EJ (2007) <A href="http://www.springerlink.com/content/4l4m28lp06120253/" class="external text" rel="nofollow">Consolidated bioprocessing for bioethanol production using <I>Saccharomyces cerevisiae</I></A>. <I>Advances in Biochemical Engineering/Biotechnology</I><B>108</B>: 205-235 (doi: 10.1007/10_2007_061).
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Modelling">http://2011.igem.org/Team:Edinburgh/Modelling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Modelling&amp;oldid=236815" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>