<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_BioSandwich"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/BioSandwich">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/BioSandwich&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/BioSandwich&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/BioSandwich&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/BioSandwich" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/BioSandwich</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Edinburgh/RFC&amp;redirect=no" title="Team:Edinburgh/RFC">Team:Edinburgh/RFC</A>)</DIV><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">BioSandwich: a new assembly protocol</P><P><B>BioSandwich</B> is a new method for assembling multiple <SPAN class="hardword" id="biobrick">BioBricks</SPAN> (which can be <SPAN class="hardword" id="rfc10">RFC10</SPAN>-compliant) in a single reaction. BioSandwich was invented by Dr. Chris French of Edinburgh University, and the Edinburgh iGEM team carried out the first attempts at making it work.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Theory"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Theory</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Standards"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Standards</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Normal_parts"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Normal parts</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Normal_spacers"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Normal spacers</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Vector"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Vector</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Start_spacer"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Start spacer</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Protocols"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Protocols</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Digestion_of_parts_and_vector"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Digestion of parts and vector</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Ligation_of_parts_and_vector_to_spacer_oligos"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Ligation of parts and vector to spacer oligos</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Tests_of_BioSandwich"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Tests of BioSandwich</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#RFC_81"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">RFC 81</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Theory">Theory</SPAN></H2><P>The point of BioBricks is that any combination of parts can be assembled together in any order (as long as the parts are in the same format, e.g. RFC10). The main disadvantage is speed; combining two BioBricks into one can take days, so making a construct that incorporates a significant number of BioBricks can take weeks.
</P><P>Across the world, most synthetic biology labs are now using <SPAN class="hardword" id="homology">homology-based</SPAN> methods such as Gibson assembly. These methods have the advantage of speed: multiple parts can be assembled at once in a single reaction. However, the assembly process relies on the 3' end of every part having homology to the 5' end of another part. These regions of homology determine the order in which the parts assemble. The parts are therefore not reusable as they stand.
</P><P>BioSandwich is a hybrid that combines many of the benefits of both systems. Parts are reusable because they come in a standard format (much like ordinary BioBricks) with restriction sites flanking the part. The restriction sites are BglII (<B>agatct</B>) in the prefix, and SpeI (<B>actagt</B>) in the suffix. However, these restriction sites are not used directly for assembly; instead they are used to attach short (~35 bp) oligonucleotides (called &quot;spacers&quot;). These spacers serve two purposes:
</P><UL><LI> They create homology between the end of one part and the start of another; this allows homology-based assembly.
</LI><LI> They can incorporate short meaningful sequences such as ribosome binding sites, linkers for fusion proteins, etc.
</LI></UL><P>Any lab using BioSandwich will want to keep a small library of different spacers for different purposes. Once (carefully chosen) spacers have been attached to each part, homology based assembly can be carried out in a single reaction. Precisely which spacers have been attached to which parts will determine the order of the parts in the final assembly.
</P><P>The overall process works roughly like this:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:720px;"><DIV class="thumbcaption">1. The parts are made (each flanked by a BglII and a SpeI site). 2. They are cut with restriction enzymes. 3. The spacers are annealed and ligated. 4. Overhangs are created. 5. The parts are placed together, and homology-based assembly is carried out.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Standards">Standards</SPAN></H2><H3><SPAN class="mw-headline" id="Normal_parts">Normal parts</SPAN></H3><P>Parts must be free of internal BglII restriction sites (<B>agatct</B>) and SpeI sites (<B>actagt</B>). Each part must be made with a BglII site at the start, and a SpeI site at the end. If compliance with RFC10 is desired, the full format becomes:
</P><PRE>gaattcgcggccgcttctag<FONT color="red"><B>agatct</B></FONT><FONT color="#999999">NNN NNN NNN NN</FONT>t <FONT color="red"><B>actagt</B></FONT>agcggccgctgcag
</PRE><P>After cutting with BglII and SpeI, we have the following (shown in frame, which is relevant for fusion proteins):
</P><PRE>5'    ga tct <FONT color="#999999">NNN [...] NN</FONT>t a         3'
3'         a <FONT color="#999999">nnn [...] nn</FONT>a tga tc    5'
</PRE><P>When a part is used for fusion proteins, the &quot;t&quot; base at the start of the RFC10 suffix becomes the final base of the final codon in the part. Design of the part should take this into account. This is often done by making this final codon GGT, coding for an innocuous glycine residue. (If a fusion protein is not being made, no such consideration is needed. If RFC10-compliance is not required, this &quot;t&quot; base can be replaced with anything.)
</P><H3><SPAN class="mw-headline" id="Normal_spacers">Normal spacers</SPAN></H3><P>Spacers are designed as a set of oligonucleotides that are attached to the parts. Each type of spacer will be attached upstream of one part and downstream of another. Different forms (&quot;spoligos&quot;) of each spacer must be synthesised for each attachment. Thus one might expect that there would be four versions of each spacer; however it is possible to cheat and use three. They should be designed so that the spacer's non-ligating ends are blunt.
</P><P>The format for the <B>upstream spacers</B> is as follows (both strands shown):
</P><PRE>5'    ct ag<FONT color="blue"><B>c</B></FONT><FONT color="#999999">NNN [...] NNN</FONT><FONT color="blue"><B>g</B></FONT>         3'               (forward spoligo)
3'    ga tc<FONT color="blue"><B>g</B></FONT><FONT color="#999999">nnn [...] nnn</FONT><FONT color="blue"><B>c</B></FONT>ct ag    5'               (reverse spoligo ONE)
</PRE><P>The format for the <B>downstream spacers</B> is:
</P><PRE>5'    ct ag<FONT color="blue"><B>c</B></FONT><FONT color="#999999">NNN [...] NNN</FONT><FONT color="blue"><B>g</B></FONT>         3'               (forward spoligo)
3'         <FONT color="blue"><B>g</B></FONT><FONT color="#999999">nnn [...] nnn</FONT><FONT color="blue"><B>c</B></FONT>         5'               (reverse spoligo TWO)
</PRE><P>The bases indicated in blue should be used for fusion proteins as they result in small hydrophilic amino acids being coded for; however these bases are otherwise optional (though if other bases are chosen, care must be taken not to regenerate the BglII and SpeI sites).
</P><P>The format of different spacers should be varied to avoid homology. It is recommended that non-coding spacers have an in-frame stop codon, in case they are to be used after a coding part which lacks one.
</P><H3><SPAN class="mw-headline" id="Vector">Vector</SPAN></H3><P>A vector is made as a PCR product with a BglII site (<B>agatct</B>) at the 5' end and an XbaI site (<B>tctaga</B>) at the 3' end. Since XbaI produces sticky-ends compatible with SpeI, the vector is compatible with standard spacers.
</P><P>If one wishes the final products (after insertion into the vector) to be RFC10-compliant, the format for a vector is as follows (BglII and XbaI in bold):
</P><PRE><B>agatct</B> [...] tactagtagcggccgctgcag [<B>ori</B>, <B>cmlR</B>, etc] gaattcgcggccgct<B>tctaga</B></PRE><P>(The actual vector used by Edinburgh 2011 is shown <A href="/Team:Edinburgh/Sequences#pSB1C3_for_BioSandwich" title="Team:Edinburgh/Sequences">here</A>.)
</P><P>Note that a few additional bases must be present at the 5' and 3' ends to allow the restriction enzymes space to work. When spacers are to be annealed, the vector is cut with BglII and XbaI (<B>not SpeI</B>).
</P><H3><SPAN class="mw-headline" id="Start_spacer">Start spacer</SPAN></H3><P>For compliance with RFC10, the spacer that connects the vector to the first part must be in the following format:
</P><P>Spoligos attaching <B>upstream</B> of first part:
</P><PRE>5'    ctag<FONT color="red"><B>ag</B></FONT><FONT color="#999999"> NNN [...] NNN</FONT><FONT color="blue"><B>g</B></FONT>         3'               (forward spoligo)
3'    gatc<FONT color="red"><B>tc</B></FONT><FONT color="#999999"> nnn [...] nnn</FONT><FONT color="blue"><B>c</B></FONT>ctag     5'               (reverse spoligo ONE)
</PRE><P>Spoligos attaching <B>downstream</B> of vector:
</P><PRE>5'    ctag<FONT color="red"><B>ag</B></FONT><FONT color="#999999"> NNN [...] NNN</FONT><FONT color="blue"><B>g</B></FONT>         3'               (forward spoligo)
3'        <FONT color="red"><B>tc</B></FONT><FONT color="#999999"> nnn [...] nnn</FONT><FONT color="blue"><B>c</B></FONT>         5'               (reverse spoligo TWO)
</PRE><P>The &quot;a&quot; indicated in red is needed to regenerate the XbaI site, and the &quot;g&quot; indicated in red is needed to complete the RFC10 prefix. The blue bases are still optional, as they were in the normal spacer format.
</P><H2><SPAN class="mw-headline" id="Protocols">Protocols</SPAN></H2><P>The actual protocols that follow are merely suggestions. Each lab will want to follow its own favourite methods.
</P><H3><SPAN class="mw-headline" id="Digestion_of_parts_and_vector">Digestion of parts and vector</SPAN></H3><P>Each part must be digested separately.
</P><UL><LI><B>Parts:</B> digest with BglII and SpeI in NEB Buffer 2
</LI><LI><B>Vector:</B> digest with BglII and XbaI in NEB Buffer 2 or 3
</LI></UL><P>Each tube gets:
</P><PRE>36 uL     Water
 5 uL     DNA
 5 uL     Buffer
 2 uL     Enzyme 1
 2 uL     Enzyme 2
</PRE><P>The digestions are then left for 2 hours at 37 C.
</P><P>Afterwards they are purified with 5 uL glass beads, and eluted to 10 uL EB (<A href="http://www.openwetware.org/wiki/Cfrench:DNAextraction1" class="external text" rel="nofollow">protocol</A>).
</P><H3><SPAN class="mw-headline" id="Ligation_of_parts_and_vector_to_spacer_oligos">Ligation of parts and vector to spacer oligos</SPAN></H3><P>The parts and vector must now be ligated to the correct spacers...
</P><UL><LI> Each tube gets:
</LI></UL><PRE>10 uL     Water
 5 uL     DNA
 1 uL     Spacer pair #1
 1 uL     Spacer pair #2
 2 uL     Ligase buffer
 1 uL     T4 DNA ligase
</PRE><UL><LI> Tubes should then be left for 9 hours at 16 C.
</LI><LI> They must then be purified (<A href="http://www.openwetware.org/wiki/Cfrench:DNAextraction1" class="external text" rel="nofollow">protocol</A>).
</LI></UL><H2><SPAN class="mw-headline" id="Tests_of_BioSandwich">Tests of BioSandwich</SPAN></H2><P>Once you have your parts ligated to spacers, there are multiple ways in which they can be assembled...
</P><P>We tested several methods. Summaries are found on our <A href="/Team:Edinburgh/Experiments" title="Team:Edinburgh/Experiments">Experiments</A> page.
</P><H2><SPAN class="mw-headline" id="RFC_81">RFC 81</SPAN></H2><P>BioSandwich is <A href="/File:RFC81.pdf" title="File:RFC81.pdf">BBF RFC 81</A>.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Gibson DG, Young L, Chuang RY, Venter JC, Hutchison CA, Smith HO (2009) <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Enzymatic assembly of DNA molecules up to several hundred kilobases</A>. <I>Nature Methods</I><B>6</B>: 343-345 (doi: 10.1038/nmeth.1318).
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/BioSandwich">http://2011.igem.org/Team:Edinburgh/BioSandwich</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/BioSandwich" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/BioSandwich" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/BioSandwich&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/BioSandwich&amp;oldid=160597" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>