<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Collaboration"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Collaboration">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Collaboration&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Collaboration&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Collaboration&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Collaboration" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Collaboration</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Collaboration with the Synbio Community</P><P><SPAN class="hardword" id="igem">iGEM</SPAN> teams have always been a highly visible part of the synthetic biology community, and an important part of iGEM is contributing to that community. Thus, the relationship between our project and the projects of other teams, past and present, is of the highest importance.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Thanks_to_UC_Davis_and_KU_Leuven"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Thanks to UC Davis and KU Leuven</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Thanks_to_ETH_Zurich"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Thanks to ETH Zurich</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Cooperation_with_Trieste"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Cooperation with Trieste</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Wiki_Watch"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Wiki Watch</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Updating_the_Registry"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Updating the Registry</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#BBa_K118022"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">BBa_K118022</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#BBa_K415151"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">BBa_K415151</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#BBa_K392008"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">BBa_K392008</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#BBa_K265008"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">BBa_K265008</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#BBa_J33207"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">BBa_J33207</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#BBa_K118023"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">BBa_K118023</SPAN></A></LI></UL></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Thanks_to_UC_Davis_and_KU_Leuven">Thanks to UC Davis and KU Leuven</SPAN></H2><P>We gratefully acknowledge UC Davis 2009, who synthesised an <I>E. coli</I> optimised version of <SPAN class="hardword" id="inp">Ice Nucleation Protein</SPAN>, originally from the organism <SPAN class="hardword" id="ps">Pseudomonas syringae</SPAN>. We were delighted to discover that the part we received from the Registry had the correct sequence.
</P><P>We also acknowledge the help of this year's KU Leuven team, whose project involved Ice Nucleation Protein, and who sent us plasmid pUC1813ICE containing the <I>inaZ</I> version of the gene.
</P><H2><SPAN class="mw-headline" id="Thanks_to_ETH_Zurich">Thanks to ETH Zurich</SPAN></H2><P>After we presented <I>(inter alia)</I> our modelling at the European Jamboree, the team from ETH Zurich contacted us with some helpful bugfixes in one of our models — the <A href="/Team:Edinburgh/Cellulases_(MATLAB_model)" title="Team:Edinburgh/Cellulases (MATLAB model)">MATLAB cellulase model</A>— which we had some trouble with. These have been fixed, thank you Zurich!
</P><H2><SPAN class="mw-headline" id="Cooperation_with_Trieste">Cooperation with Trieste</SPAN></H2><P>Like us, Trieste's team this year are using the <SPAN class="hardword" id="biobrick">BioBrick</SPAN><A href="http://partsregistry.org/Part:BBa_K392008" class="external text" rel="nofollow">BBa_K392008</A>. This part by Osaka 2010 encodes a <SPAN class="hardword" id="cellulase">cellulase</SPAN>, β-glucosidase, from the bacterium <I>Cellulomonas fimi</I>. It is known to work in the lab of Chris French (Edinburgh's supervisor).
</P><P>Chris French sent Trieste a copy of the plasmid, however they reported some sequence errors and asked us for comments. We sequenced the gene independently and discovered that an apparent <SPAN class="hardword" id="frameshift">frameshift</SPAN> is present near the start of the BioBrick. This very same &quot;frameshift&quot; was seen in Trieste's sequencing results.
</P><P>Since a part with an early frameshift cannot possibly work, but the part does work, we looked for an explanation. Some 220 bases into the part, a second ATG codon is found. This codon is in-frame and there is a plausible <SPAN class="hardword" id="rbs">ribosome binding site</SPAN> (containing &quot;gaagga&quot;) just upstream of it. We therefore believe that this second ATG is the true start codon. The RBS would explain why the part can be expressed and work.
</P><P>Here is the start of the sequence, with these features highlighted:
</P><DIV style="margin-left: 4em;"><P><CODE><B>&gt; BBa_K392008 Part-only sequence (1671 bp)</B>
atgggcgaccggttccagcaggccggtcgcccacgccgccgcggcccggcgagggccgtt
aaccgtaccggtcaagaagacgcgtcgacggggtcgagggagcggtcccacgcgtgtatc
gtatcgtttcgacaccgccacccggccaccgggcacgcaccggggacgcagcagtccccg
ccccggccaccccctgtcaccgaaaccc<FONT color="blue"><B>gaagga</B></FONT>ccctc<FONT color="red"><B>atg</B></FONT>accaccacgcgcccctcg
[rest omitted]
</CODE></P></DIV><P>In agreement with this hypothesis, a <A href="http://www.ncbi.nlm.nih.gov/nuccore/332337569?from=3105074&amp;to=3106528" class="external text" rel="nofollow">recently published sequence</A> of <I>C. fimi</I>β-glucosidase (labelled as such by <A href="http://enzyme.expasy.org/EC/3.2.1.21" class="external text" rel="nofollow">Expasy</A> though NCBI calls it a β-galactosidase) starts at the 2nd ATG codon of K392008. This is in contrast to an <A href="http://www.ncbi.nlm.nih.gov/nuccore/304358?from=18&amp;to=1688" class="external text" rel="nofollow">older published sequence</A>.
</P><P>We passed this information on to the Trieste team, who agreed that it is a likely explanation.
</P><P>In addition to this, at the request of the Trieste team we sent details of an assay that can be used to test for activity of this β-glucosidase. This assay involves 4-methylumbelliferyl-β-D-glucuronide, which is cleaved by β-glucosidase to yield a fluorescent product visible under long-wave blue light. We used this assay ourselves for <A href="http://partsregistry.org/Part:BBa_K523014" class="external text" rel="nofollow">BBa_K523014</A>.
</P><H2><SPAN class="mw-headline" id="Wiki_Watch">Wiki Watch</SPAN></H2><P>We created a <A href="/Team:Edinburgh/Wiki_Watch" title="Team:Edinburgh/Wiki Watch"> Wiki Watch</A> page that attempted to convey the very basics of what every other team in the competition was doing. Early in the summer, this page was linked from the official iGEM <A href="/Community" title="Community">Community</A> page, and we hope it has helped other teams find partners for cooperation.
</P><H2><SPAN class="mw-headline" id="Updating_the_Registry">Updating the Registry</SPAN></H2><P>The hub of our community's collective experience is the Parts Registry. During our work, we have discovered some useful information about several parts in the Registry. We have entered this information in the &quot;experience&quot; section of the relevant pages.
</P><H3><SPAN class="mw-headline" id="BBa_K118022"><A href="http://partsregistry.org/Part:BBa_K118022" class="external text" rel="nofollow">BBa_K118022</A></SPAN></H3><P>This part by Edinburgh 2008 encodes an exoglucanase, capable of degrading <SPAN class="hardword" id="cellulose">cellulose</SPAN>. We tested its ability to degrade the cellulose analog MUC, and found that it could.
</P><P>This information has been added to the Registry on the relevant <A href="http://partsregistry.org/Part:BBa_K118022:Experience" class="external text" rel="nofollow">experience page</A>.
</P><H3><SPAN class="mw-headline" id="BBa_K415151"><A href="http://partsregistry.org/Part:BBa_K415151" class="external text" rel="nofollow">BBa_K415151</A></SPAN></H3><P>This part by MIT 2010 is supposed to encode a <SPAN class="hardword" id="fusion">fusion</SPAN> of the <SPAN class="hardword" id="p8">pVIII</SPAN> protein to a GR1 zipper. However, we noticed that the official verification sequencing carried out by the Registry did not match the expected sequence and did not contain a prefix or suffix.
</P><P>We tried to determine what the sequence actually is, and discovered that it is <A href="http://www.ncbi.nlm.nih.gov/nuccore/48994873?from=989379&amp;to=991367&amp;report=gbwithparts" class="external text" rel="nofollow">a fragment</A> of the <I>E. coli</I> main genome. This fragment naturally has EcoRI and PstI sites at its ends. It has evidently been cut with those enzymes and has somehow become inserted into pSB1C3 via those sites. This naturally wipes out the XbaI and SpeI sites, explaining their absence.
</P><P>This information has been added to the Registry on the relevant <A href="http://partsregistry.org/Part:BBa_K415151:Experience" class="external text" rel="nofollow">experience page</A>.
</P><H3><SPAN class="mw-headline" id="BBa_K392008"><A href="http://partsregistry.org/Part:BBa_K392008" class="external text" rel="nofollow">BBa_K392008</A></SPAN></H3><P>As mentioned above, Trieste and Edinburgh cooperated around the use of this part from Osaka 2010, and came to suspect that the second ATG present in the sequence is the real start codon.
</P><P>In addition, there are some other discrepancies between the published sequence and the physical K392008 as well.
</P><P>This information has been added to the Registry on the relevant <A href="http://partsregistry.org/Part:BBa_K392008:Experience" class="external text" rel="nofollow">experience page</A>.
</P><H3><SPAN class="mw-headline" id="BBa_K265008"><A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A></SPAN></H3><P>This is the synthetic Ice Nucleation Protein from UC Davis 2009. We updated the relevant <A href="http://partsregistry.org/Part:BBa_K265008:Experience" class="external text" rel="nofollow">experience page</A> to mention our successful use of it.
</P><H3><SPAN class="mw-headline" id="BBa_J33207"><A href="http://partsregistry.org/Part:BBa_J33207" class="external text" rel="nofollow">BBa_J33207</A></SPAN></H3><P>This part encodes LacZα, and provides a cunning way to make new BioBricks using shorter PCR primers than normal, by incorporating a SacI site. We made a new version deleting about 50 unnecessary bases and based on a different restriction site.
</P><P>This information has been added to the Registry on the relevant <A href="http://partsregistry.org/Part:BBa_J33207:Experience" class="external text" rel="nofollow">experience page</A>.
</P><H3><SPAN class="mw-headline" id="BBa_K118023"><A href="http://partsregistry.org/Part:BBa_K118023" class="external text" rel="nofollow">BBa_K118023</A></SPAN></H3><P>This part was made by Edinburgh 2008. When we sequenced it, we found a discrepancy between the physical DNA and the Registry sequence. We checked against a published genome sequence and found that the Registry sequence is incorrect.
</P><P>This information has been added to the Registry on the relevant <A href="http://partsregistry.org/Part:BBa_K118023:Experience" class="external text" rel="nofollow">experience page</A>.
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Collaboration">http://2011.igem.org/Team:Edinburgh/Collaboration</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Collaboration" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Collaboration" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Collaboration&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Collaboration&amp;oldid=253290" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>