<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Presentation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Presentation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Presentation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Presentation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Presentation&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Presentation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Presentation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Presentation</P><P>Here we prepare the talk we must give to the iGEM Jamboree.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Modelling"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Modelling</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#C"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">C</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Kappa"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Kappa</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#MATLAB"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">MATLAB</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Other_things"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Other things</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Biology"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Biology</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#Society"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Society</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><DL><DD><I>[Slide: Title]</I></DD></DL><P>Hello, we're the Edinburgh iGEM team...
</P><DL><DD><I>[Slide: What it's all about]</I></DD></DL><P>...and this is our project.
</P><P>Edinburgh's 2011 iGEM project is all about synergy - the ability some enzymes have to work better when close together. In 2010, the Slovenian team found a way to arrange enzymes in the cytoplasm so as to achieve synergy. We wondered whether it would be possible to get extracellular enzymes close together.
</P><P>Edinburgh's project is also about biorefineries, which are special types of refineries in which biomass, such as cellulose, is converted into useful products. Our project is a feasibility study into the creation of a biorefinery using cellulases arranged in a synergistic manner.
</P><DL><DD><I>[Slide: Cellulose]</I></DD></DL><P>Cellulose is an abundant material, produced by plants, but very hard to break down into useful products.
</P><H2><SPAN class="mw-headline" id="Modelling">Modelling</SPAN></H2><P>The first question we asked is: what is this synergy phenomenon and how does it work?
</P><DL><DD><I>[Slide: 3 enzyme actions]</I></DD></DL><P>Cellulose is a long chain of glucose sugars. It can be broken down by enzymes called &quot;cellulases&quot;. Consider the following statements about cellulose degradation:
</P><UL><LI> One type of cellulase, exoglucanase, attacks the end of a chain, producing disaccharides called cellobiose.
</LI><LI> Another type, endoglucanase, cuts the chain in the middle.
</LI></UL><P>We can immediately see that these two facts mean the enzymes work best together: the enzyme that cuts chains in the middle also produces new chain ends for the other enzyme to attack.
</P><P>Now consider the following:
</P><UL><LI> Exoglucanase is inhibited by the presence of cellobiose.
</LI><LI> A third enzyme, β-glucosidase, cuts cellobiose into glucose.
</LI></UL><P>Here then we have another case of enzymes working best together.
</P><P>We constructed computer models to investigate the above phenomena.
</P><H3><SPAN class="mw-headline" id="C">C</SPAN></H3><DL><DD><I>[Slide: C animation]</I></DD></DL><P>Our first model is an <I>ad hoc</I> model written in C. Here cellulose is placed on a two dimensional grid, and cellulases move over the grid. There are two simulations: in one, the enzymes move about freely. In the other, enzymes travel in triplets, each triplet containing one of each enzyme.
</P><P>The dark regions show areas where cellulose has been broken down to free glucose. As can be seen, the synergistic model breaks down cellulose faster...
</P><DL><DD><I>[Slide: C graph]</I></DD></DL><P>... and here's a graph of the results. What we're most interested in are the blue lines, which represent the amount of glucose produced in the synergistic and non-synergistic systems. 
</P><P>This model is very simplistic but does show that the synergy effect makes sense.
</P><H3><SPAN class="mw-headline" id="Kappa">Kappa</SPAN></H3><P>A more sophisticated model uses the Kappa modelling language. This includes two models - one preliminary model in Kappa, and another model in Spatial Kappa. Kappa is a language for running stochastic simulations of biochemical systems. It uses probabilistic kinetic rates to advance simulation. The preliminary model was developed based on Kadam <I>et al</I> (2004) and generates similar looking graphs for both synergistic and non-synergistic systems:
</P><DL><DD><I>[Slide: Normal Kappa graph]</I></DD></DL><P>The problem is that Kappa cannot see the advantages of synergy since it has no concept of space. Therefore we used Spatial Kappa by Team Edinburgh 2010 (Donal Stewart). The added functionality means we could develop a model where the enzymes were kept close together, and compare it to one where they were not:
</P><DL><DD><I>[Slide: Pacman animation]</I></DD></DL><P>This model proved quite scalable - we can manipulate initial conditions, add more cellulose - emulating a biorefinery.
</P><P>Most importantly - it shows synergy (dependent mostly on diffusion rate).
</P><DL><DD><I>[Slide: Spatial Kappa graph]</I></DD></DL><H3><SPAN class="mw-headline" id="MATLAB">MATLAB</SPAN></H3><DL><DD><I>[Slide: MATLAB graph]</I></DD></DL><P>We also tried MATLAB modelling, but found we couldn't model synergy without knowing the equations to use (which we didn't). Our MATLAB model is of the non-synergistic system only, and uses ordinary differential equations to model the progress of the reaction over time. We found that it works best within certain limits; if allowed to run indefinitely, the results violate the laws of thermodynamics; for example the amount of cellobiose going negative.
</P><H3><SPAN class="mw-headline" id="Other_things">Other things</SPAN></H3><DL><DD><I>[Slide: Modelling wrap-up]</I></DD></DL><P>We also modelled:
</P><UL><LI> Phage replication. We developed a model of phage replication in MATLAB using ODEs, but our project never got to the stage of using phages.
</LI><LI> Artificial selection: we showed that it is easier to select for enhanced cellulose degradation if the cellulases remain attached to the cell.
</LI></UL><H2><SPAN class="mw-headline" id="Biology">Biology</SPAN></H2><DL><DD><I>[Slide: Biology - the question]</I></DD></DL><P>The fundamental biology problem we had to solve was how to actually get different types of cellulases close together.
</P><DL><DD><I>[Slide: Cellulosome]</I></DD></DL><P>Nature has evolved a method called a cellulosome, in which various proteins connect to form a scaffold on which the cellulases sit. But such a structure won't work with <I>E. coli</I> due to its Gram-negative double membrane. <I>E. coli</I> has long been the workhorse of synthetic biology, so we wanted a solution compatible with it.
</P><DL><DD><I>[Slide: Phage display schematic]</I></DD></DL><P>Our first thought was to use phage display technology to place several enzymes on an M13 phage. Such a phage would be constructed as follows: fusion proteins would be created containing both a cellulase and a phage protein, and this fusion would integrate into the phage. The phage would then be excreted from the cell.
</P><DL><DD><I>[Slide: Phage display]</I></DD></DL><P>This approach has a number of difficulties, and has not advanced much beyond the theoretical, though we have made preliminary attempts to fuse the phage major coat protein to an amylase gene, using a phagemid vector. If we have more time we will assay this construct for activity and swap it into vector pSB1C3.
</P><P><B>[This might change by the time of the final presentation.]</B></P><DL><DD><I>[Slide: Cell display]</I></DD></DL><P>A different approach is to place the various enzymes on the cell surface. If they can be placed there at a high enough density, the synergistic effect should appear. Enzymes can be placed on the cell surface using Ice Nucleation Protein (or INP), which seems to automagically transport itself to the outer membrane. We used a synthetic INP gene created by the 2009 UC Davis team.
</P><DL><DD><I>[Slide: Cell display schematic]</I></DD></DL><P>By creating a fusion of a cellulase to Ice Nucleation Protein, it should be possible to have that cellulase carried to the outer membrane. A fully working system would incorporate multiple cellulases, displaying at a number of up to 100,000 in total.
</P><DL><DD><I>[Slide: BioSandwich]</I></DD></DL><P>The creation of fusion proteins poses interesting DNA assembly challenges, and is usually impossible using standard assembly of RFC10 parts. We pioneered the use of a new protocol, BioSandwich, invented by our supervisor Chris French. This assembly method is based on RFC10-compliant parts, but assembles them in a homology-based way. The homology between parts is created by attaching oligonucleotides to the end of each part.
</P><DL><DD><I>[Slide: INP-YFP]</I></DD></DL><P>As a proof both that BioSandwich works, and that INP works, we successfully attached INP to Yellow Fluorescent Protein. When we centrifuged the cells, we found that the fluorescence seemed to be localised to the membrane fraction, seen at the bottom of the tube. Ordinary YFP would be present to a much greater extent in the cytoplasm, as shown on the left.
</P><DL><DD><I>[Slide: bglX]</I></DD></DL><P>Before we attach enzymes to INP, we wanted to BioBrick them and test them. We successfully created a new β-glucosidase based on a cryptic <I>E. coli</I> gene, bglX. This slide shows that it is capable of degrading the cellobiose analog MUG. We also characterised the ability of an exoglucanase, <I>cex</I>, to degrade MUG and MUC, which is a larger cellulose-like molecule.
</P><P>We have fused this enzyme, and also <I>cex</I>, to INP. At this time, further experiments are required to test for activity.
</P><P><B>[This might improve by the Jamboree.]</B></P><DL><DD><I>[Slide: malS]</I></DD></DL><P>Another BioBrick we created was the amylase <I>malS</I>, the first amylase in the Registry. Amylases degrade starch, not cellulose, but a fusion of INP to <I>malS</I> could be useful as a test system, since degradation of starch requires just this one enzyme. We proved that this BioBrick has some starch-degrading activity with an iodine assay: iodine turns starch black, so a halo around the bacteria means starch has been degraded.
</P><P><B>[Discuss any advances in attaching cellulases to INP since this was written.]</B></P><DL><DD><I>[Slide: genetic instability]</I></DD></DL><P>Finally, we dealt with a serious objection raised by several people. The fully working system will require several copies of the INP gene in the construct, to fuse to the different cellulases. One worry is therefore the potential for genetic instability and recombination. However, we believe that in industry it will be possible to synthesise different versions of INP, which use synonymous codons to code for the same protein, while having different DNA sequences.
</P><P>We wrote a software tool to assist in this process.
</P><H2><SPAN class="mw-headline" id="Society">Society</SPAN></H2><P>We set out to answer two different questions:
</P><UL><LI> Could a biorefinery plant using this technology be built and be economically viable?
</LI><LI> Should we as a society proceed down this road?
</LI></UL><DL><DD><I>[Slide: Process flow diagram]</I></DD></DL><P>To answer the first question, we constructed a process flow diagram, which is an engineering design tool illustrating how a biorefinery would be arranged in the real world. The PFD is not a definitive answer to what a biorefinery should look like but an articulation of how it might look. It shows biomass raw materials subjected to pre-treatment via fractionation and depolymerisation of lignocellulose and hemicellulose. After lignin is seperated it undergoes hydrous pyrolysis to convert it into phenols. Cellulose is separated and undergoes conversion into glucose. Finally glucose is converted here to sorbitol, but there is potential for conversion to other food and chemical products.
</P><DL><DD><I>[Slide: Economics of Biorefineries]</I></DD></DL><P>Then we conducted an economic analysis of the biorefinery. While this analysis makes many assumptions, it shows that it is at least plausible that such a biorefinery could be economically viable. We estimated $2.5 million dollars of investment is required plus $4.5 million in production costs.
</P><DL><DD><I>[Slide: Industry, Church, Academia, Politics]</I></DD></DL><P>To answer the question of whether we should build a biorefinery is even harder. But we conducted interviews with a number of figures involved in the debate around synthetic biology: people from business, academia, philosophy, environmental groups, politics, and the Church.
</P><DL><DD><I>[Slide: Interviewees - names/pictures]</I></DD></DL><P>A number of key themes emerged.
</P><UL><LI> Concerns over the technology - will it work and is it safe?
</LI><LI> Concerns over who benefits, and whether anyone is exploited.
</LI><LI> Concerns over sustainable sourcing of raw materials for the biorefinies.
</LI></UL><P>One particular issue several people raised was a fear technology such as this would become a tool that rich nations use to clobber the world's poor; for example with crops grown in the Third World only to be shipped off and turned into ethanol or high fructose corn syrup.
</P><P>However, there was a general acceptance in our interviewees that synthetic biology has a major role to play in humanity's future. But several people warned us against the view that technological problems alone could solve the world's problems.
</P><DL><DD><I>[Slide: Sponsors]</I></DD></DL><P>Thanks for listening and thanks to all our sponsors!
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Presentation">http://2011.igem.org/Team:Edinburgh/Presentation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Presentation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Presentation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Presentation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Presentation&amp;oldid=138420" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>