<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Assembly"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Assembly">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Assembly&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Assembly&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Assembly&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Assembly" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Assembly</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Primers and Oligos</P><P>For the sake of posterity, a list of primers we made, or thought about, is kept here. This might conceivably be of use to teams in the future (though the odds are not high...)
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Notes"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Notes</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Primers_actually_made"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Primers actually made</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#malS_.28amylase.29"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">malS (amylase)</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#bglX_.28B-glucosidase.29"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">bglX (B-glucosidase)</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#pVIII_leader_sequence"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">pVIII leader sequence</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#pVIII_mature_protein"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">pVIII mature protein</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#BBa_K118023_.2F_BBa_K392006_.28Endoglucanase.29"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">BBa_K118023 / BBa_K392006 (Endoglucanase)</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#BBa_K118022_.2F_BBa_K392007_.28Exoglucanase.29"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">BBa_K118022 / BBa_K392007 (Exoglucanase)</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#BBa_K392008_.28B-glucosidase.29"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">BBa_K392008 (B-glucosidase)</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Xylose_isomerase"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Xylose isomerase</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#BBa_K265008_.28Ice_Nucleation_Protein.29"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">BBa_K265008 (Ice Nucleation Protein)</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#crtE"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">crtE</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#crtB"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">crtB</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#crtI"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">crtI</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-15"><A href="#Gibson_assembly_of_pG8SAET_with_malS"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Gibson assembly of pG8SAET with malS</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Primers_in_planning_.28inaZ.2C_3603_b.p..29"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Primers in planning (inaZ, 3603 b.p.)</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Primers_in_planning_.28amylase.2C_mature_peptide_only.29"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Primers in planning (amylase, mature peptide only)</SPAN></A></LI><LI class="toclevel-1 tocsection-18"><A href="#Primers_in_planning_.28bglX.29"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Primers in planning (bglX)</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#Mature_peptide_only"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Mature peptide only</SPAN></A></LI><LI class="toclevel-1 tocsection-20"><A href="#Primers_in_planning_.28biorefinery.29"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Primers in planning (biorefinery)</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Aldose_reductase"><SPAN class="tocnumber">7.1</SPAN><SPAN class="toctext">Aldose reductase</SPAN></A></LI><LI class="toclevel-1 tocsection-22"><A href="#Primers_in_planning_.28GFP:_green_fluorescent_protein.29"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">Primers in planning (GFP: green fluorescent protein)</SPAN></A></LI><LI class="toclevel-1 tocsection-23"><A href="#Primers_in_planning_.28zippers.29"><SPAN class="tocnumber">9</SPAN><SPAN class="toctext">Primers in planning (zippers)</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#BBa_K415124_.28GR1.29"><SPAN class="tocnumber">9.1</SPAN><SPAN class="toctext">BBa_K415124 (GR1)</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#BBa_K415125_.28GR2.29"><SPAN class="tocnumber">9.2</SPAN><SPAN class="toctext">BBa_K415125 (GR2)</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#BioSandwich_spoligos_actually_made"><SPAN class="tocnumber">10</SPAN><SPAN class="toctext">BioSandwich spoligos actually made</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#.22Start_Spacer.22_.28also_has_T7_promoter.29"><SPAN class="tocnumber">10.1</SPAN><SPAN class="toctext">&quot;Start Spacer&quot; (also has T7 promoter)</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Stop_-_RBS_-_Start_.281.29"><SPAN class="tocnumber">10.2</SPAN><SPAN class="toctext">Stop - RBS - Start (1)</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Stop_-_RBS_-_Start_.282.29"><SPAN class="tocnumber">10.3</SPAN><SPAN class="toctext">Stop - RBS - Start (2)</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Stop_-_Strongest_RBS"><SPAN class="tocnumber">10.4</SPAN><SPAN class="toctext">Stop - Strongest RBS</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#Stop_-_Weak_RBS"><SPAN class="tocnumber">10.5</SPAN><SPAN class="toctext">Stop - Weak RBS</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#PT_linker_.2810_aa.29"><SPAN class="tocnumber">10.6</SPAN><SPAN class="toctext">PT linker (10 aa)</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Serine-rich_linker_.2812_aa.29"><SPAN class="tocnumber">10.7</SPAN><SPAN class="toctext">Serine-rich linker (12 aa)</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Another_PT_linker_.2811_aa.29"><SPAN class="tocnumber">10.8</SPAN><SPAN class="toctext">Another PT linker (11 aa)</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#BioSandwich_spoligos_in_planning_.28ribosome_binding_sites.29"><SPAN class="tocnumber">11</SPAN><SPAN class="toctext">BioSandwich spoligos in planning (ribosome binding sites)</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Stop_-_Weakest_RBS"><SPAN class="tocnumber">11.1</SPAN><SPAN class="toctext">Stop - Weakest RBS</SPAN></A></LI><LI class="toclevel-1 tocsection-37"><A href="#BioSandwich_spoligos_in_planning_.28linkers.29"><SPAN class="tocnumber">12</SPAN><SPAN class="toctext">BioSandwich spoligos in planning (linkers)</SPAN></A></LI><LI class="toclevel-2 tocsection-38"><A href="#BBa_K105012_linker_.2810_aa.29"><SPAN class="tocnumber">12.1</SPAN><SPAN class="toctext">BBa_K105012 linker (10 aa)</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#BBa_K133132_linker_with_added_glycines_.2810_aa.29"><SPAN class="tocnumber">12.2</SPAN><SPAN class="toctext">BBa_K133132 linker with added glycines (10 aa)</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Glycine-rich_linker_.2812_aa.29"><SPAN class="tocnumber">12.3</SPAN><SPAN class="toctext">Glycine-rich linker (12 aa)</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#E-tag_.2813_aa.29"><SPAN class="tocnumber">12.4</SPAN><SPAN class="toctext">E-tag (13 aa)</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Notes">Notes</SPAN></H2><P>The setup for our two systems are as follows:
</P><UL><LI><A href="/Team:Edinburgh/Phage_Display" title="Team:Edinburgh/Phage Display"> Phage Display</A>: Promoter-RBS-Leader-Enzyme-pVIII
</LI><LI><A href="/Team:Edinburgh/Cell_Display" title="Team:Edinburgh/Cell Display"> Cell Surface Display</A>: Promoter-RBS-INP-Enzyme
</LI></UL><P>Phage/cell display passenger enzymes will need start codons so they can be used as controls for assays, where we ask: do they work without the carrier?
</P><P>When part of a display system, the different products will <B>either be fused only at their 5' end (cell display), or at both ends (phage display)</B>.
</P><P>Some advice that Chris, our supervisor, gave us on primer design:
</P><OL><LI> Check the annealing temperature of the portion without the non-complementary tail (as this is the only part that will anneal during the first round).
</LI><LI> Check preexisting BioBricks for BglII sites.
</LI><LI> We will have RFC10 reverse primers for the cellulases.
</LI><LI> It may not be necessary to have two different forward primers with and without start codon, since the start codon will just be interpreted as a methionine.
</LI><LI> It may not be necessary to have reverse primers with and without stop codons, as the stop codon can be supplied by the spacer.
</LI></OL><H2><SPAN class="mw-headline" id="Primers_actually_made">Primers actually made</SPAN></H2><H3><SPAN class="mw-headline" id="malS_.28amylase.29">malS (amylase)</SPAN></H3><P><I>malS</I> (<A href="http://www.ncbi.nlm.nih.gov/nuccore/48994873?from=3735520&amp;to=3737550&amp;report=gbwithparts" class="external text" rel="nofollow">sequence</A>) is an amylase gene from <I>E. coli</I>. It seems to be periplasmic, and for whatever reason does not cause noticable starch in wildtype <I>E. coli</I> degradation, whether because of its location or its expression levels.
</P><P>Using this as a test system would involve comparing the starch-degrading activity of the gene by itself, as compared to the relevant fusions.
</P><P>The actual primers ordered are as follows:
</P><UL><LI><B>EcMalSf1</B>: ACT AGATCT gaa atc gca gca ata agg act c 
</LI><LI> Forward; adds BglII site; starts upstream of gene to catch RBS (<A href="http://heptamer.tamu.edu/cgi-bin/gb2/gbrowse/MG1655/?plugin=FastaDumper;q=NC_000913%3A3735491..3737550;plugin_action=Go" class="external text" rel="nofollow">see here</A>).
</LI><LI> Complementary Tm = 63.7 C.
</LI></UL><LI><B>EcMalSf2</B>: ACT AGATCT atg aaa ctc gcc gcc tgt ttt c
</LI><LI> Forward; adds BglII site; starts at start codon.
</LI><LI> Complementary Tm = 70.1 C.
</LI><LI><B>EcMalSr1</B>: ACT ACTAGT A TTA tta ctg ttg ccc tgc cca g
</LI><LI> Reverse; adds SpeI site; RFC10 compliant; has double stop codon.
</LI><LI> Complementary Tm = 66.2 C.
</LI><LI><B>EcMalSr2</B>: ACT ACTAGT ACC ctg ttg ccc tgc cca gac g
</LI><LI> Reverse; adds SpeI site; RFC23 and BioSandwich compliant; no stop codons, adds a glycine.
</LI><LI> Complementary Tm = 71.6 C.
</LI><H3><SPAN class="mw-headline" id="bglX_.28B-glucosidase.29">bglX (B-glucosidase)</SPAN></H3><P><I>bglX</I> (<A href="http://www.ncbi.nlm.nih.gov/nuccore/48994873?from=2217714&amp;to=2220011&amp;report=gbwithparts" class="external text" rel="nofollow">sequence, but gene is on reverse strand</A>) is a periplasmic Î-glucosidase from <I>E. coli</I>. Again, it seems to not be expressed very strongly.
</P><P>Note presence of PstI site in sequence; we inserted this into vector &lt;partinfo&gt;BBa_K523000&lt;/partinfo&gt; via BglII and SpeI. It was then be necessary to use <A href="http://www.openwetware.org/wiki/Cfrench:MABEL" class="external text" rel="nofollow">MABEL</A> to remove the PstI site.
</P><P>The actual primers ordered are as follows:
</P><UL><LI><B>EcBglXf1</B>: ACT AGATCT gcc acg tcg ggc aac
</LI><LI> Forward; adds BglII site; starts upstream of gene to catch RBS.
</LI><LI> Complementary Tm = 65.6 C.
</LI></UL><LI><B>EcBglXf2</B>: ACT AGATCT atg aaa tgg cta tgt tca gta gg
</LI><LI> Forward; adds BglII site; starts at start codon.
</LI><LI> Complementary Tm = 60.8 C.
</LI><LI><B>EcBglXr1</B>: ACT ACTAGT A TTA tta cag caa ctc aaa ctc gcc
</LI><LI> Reverse; RFC10 compliant; has double stop codon.
</LI><LI> Complementary Tm = 64.2 C.
</LI><LI><B>EcBglXr2</B>: ACT ACTAGT ACC cag caa ctc aaa ctc gcc ttt c
</LI><LI> Reverse; adds SpeI site; RFC23 and BioSandwich compliant; no stop codons, adds a glycine.
</LI><LI> Complementary Tm = 67.0 C.
</LI><P>MABEL primers:
</P><UL><LI><B>EcbglXmutf</B>: a cca gca ctg gcg gat g
</LI><LI> Forward; first base is the change.
</LI><LI> Tm = 66.4 C.
</LI></UL><UL><LI><B>EcbglXmutr</B>: tg cag ggc cag act cac
</LI><LI> Reverse; perfect match to sequence.
</LI><LI> Tm = 64.1 C.
</LI></UL><H3><SPAN class="mw-headline" id="pVIII_leader_sequence">pVIII leader sequence</SPAN></H3><P>The pVIII sequence is <A href="/Team:Edinburgh/Sequences#pVIII" title="Team:Edinburgh/Sequences">here</A>.
</P><P>This part is intended to have <B>stuff fused at its 3' (C-terminal) end</B>.
</P><UL><LI><B>p8f_cfus</B>: ACT AGATCT atg aaa aag tct tta gtc c
</LI><LI> Forward, adds BglII site, start codon present.
</LI><LI> Complementary Tm = 49.7 C.
</LI></UL><UL><LI><B>p8r_cfus</B>: AAA ACTAGT ACC agc gaa aga cag cat cgg
</LI><LI> Reverse, adds a glycine, adds SpeI.
</LI><LI> Complementary Tm = 64.2 C.
</LI></UL><H3><SPAN class="mw-headline" id="pVIII_mature_protein">pVIII mature protein</SPAN></H3><P>The pVIII sequence is <A href="/Team:Edinburgh/Sequences#pVIII" title="Team:Edinburgh/Sequences">here</A>.
</P><P>This part is intended to have <B>stuff fused at its 5' (N-terminal) end</B>.
</P><UL><LI><B>p8f_nfus</B>: ACT AGATCT gct gag ggt gac gat ccc gc
</LI><LI> Forward, adds BglII site.
</LI><LI> Complementary Tm = 73.6 C.
</LI></UL><UL><LI><B>p8r_nfus</B>: AAA ACTAGT A tca gct tgc ttt cga ggt g
</LI><LI> Reverse, has stop codon, adds standard RFC10 suffix.
</LI><LI> Complementary Tm = 64.4 C.
</LI></UL><H3><SPAN class="mw-headline" id="BBa_K118023_.2F_BBa_K392006_.28Endoglucanase.29">BBa_K118023 / BBa_K392006 (Endoglucanase)</SPAN></H3><P><B>This part contains a BglII site, so that will first have to be dealt with, using MABEL:</B></P><UL><LI><B>cenAmutf1</B>: atc tcg cag cgg ctg gg
</LI><LI> Forward, perfect correspondance to sequence
</LI><LI> Tm = 70.3 C.
</LI></UL><UL><LI><B>cenAmutr1</B>: ttg ctg gcc gta cgc ctt g
</LI><LI> Reverse, replaces CAG (glutamine) with CAA (glutamine).
</LI><LI> Tm = 71.5 C.
</LI></UL><P>We can then PCR the part itself. SignalP predicts a cleavage site between amino acids 31 and 32. The following primers fix that problem:
</P><UL><LI><B>cenAf_nosig</B>: ACT AGATCT ATG GCA CCA GGA tgc cgc gtc gac tac gcc gtc <B>[Warning: strong secondary structure]</B></LI><LI> Forward, adds BglII site, adds start codon, starts at a.a. 32. but changes first three codons to avoid <I>very strong</I> primer secondary structure.
</LI><LI> Complementary Tm = 78.4 C.
</LI></UL><UL><LI><B>cenAr_nostop</B>: AAA ACTAGT ACC cca cct ggc gtt gcg cg
</LI><LI> Reverse, no stop codon; adds a glycine, adds SpeI.
</LI><LI> Complementary Tm = 75.1 C.
</LI></UL><H3><SPAN class="mw-headline" id="BBa_K118022_.2F_BBa_K392007_.28Exoglucanase.29">BBa_K118022 / BBa_K392007 (Exoglucanase)</SPAN></H3><P>The forward primer deletes the signal peptide.
</P><UL><LI><B>cexf_nosig</B>: ACT AGATCT ATG gcg acc acg ctc aag gag gc
</LI><LI> Forward, adds BglII site, adds start codon, starts at a.a. 43.
</LI></UL><UL><LI><B>cexr_nostop</B>: AAA ACTAGT AGA gcc gac cgt gca ggg cgt gc
</LI><LI> Reverse, no stop codon; adds a serine, adds SpeI.
</LI><LI> (I was unable to add the usual glycine due to very strong secondary structure arising.)
</LI></UL><H3><SPAN class="mw-headline" id="BBa_K392008_.28B-glucosidase.29">BBa_K392008 (B-glucosidase)</SPAN></H3><P>The forward primer is slightly misnamed as there is no signal peptide predicted for this; the primer just starts at the CDS start.
</P><UL><LI><B>Cfbgluf_nosig</B>: ACT AGATCT ATG ggc gac cgg ttc cag cag gc
</LI><LI> Forward, adds BglII site. Start codon.
</LI><LI> Complementary Tm = 77.2 C.
</LI></UL><UL><LI><B>cfbglur_nostop</B>: AAA ACTAGT ACC ggg ctg gta ggt cgc ggc g
</LI><LI> Reverse, no stop codon; adds a glycine, adds SpeI.
</LI><LI> Complementary Tm = 76.2 C.
</LI></UL><P><B>We discovered that the 2nd ATG in the K392008 sequence is probably the true start codon. The following is an alternative forward primer that starts there:</B></P><UL><LI><B>Cfbgluf2</B>: ACT AGATCT atg acc acc acg cgc ccc tc
</LI><LI> Forward, adds BglII site. Start codon.
</LI><LI> Complementary Tm = 75.8 C.
</LI></UL><H3><SPAN class="mw-headline" id="Xylose_isomerase">Xylose isomerase</SPAN></H3><P>From the <I><A href="http://www.ncbi.nlm.nih.gov/nuccore/148278?from=213&amp;to=1535&amp;report=gbwithparts" class="external text" rel="nofollow">xylA</A></I> gene of <I>E. coli</I>. Designed for fusion to INP or expression on its own. We probably won't try for phage.
</P><UL><LI><B>EcxylAf1</B>: ACT AGATCT atg caa gcc tat ttt gac c
</LI><LI> Forward, adds BglII site, has start codon.
</LI><LI> Complementary Tm = 59.2 C.
</LI></UL><UL><LI><B>EcxylAr1</B>: AAA ACTAGT A TTA tta ttt gtc gaa cag ata atg g
</LI><LI> Reverse, adds 2nd stop codon, adds standard suffix.
</LI><LI> Complementary Tm = 58.1 C.
</LI></UL><H3><SPAN class="mw-headline" id="BBa_K265008_.28Ice_Nucleation_Protein.29">BBa_K265008 (Ice Nucleation Protein)</SPAN></H3><P>Continuing on the assumption of <B>BioSandwich</B> assembly. Ice Nucleation Protein will be present at the start of the coding fusion. A promoter and RBS will have to be added upstream.
</P><P>This part will have <B>stuff fused at its 3' (C-terminal) end</B>.
</P><UL><LI> ACT AGATCT atg acc ctg gat aaa gcg c
</LI><LI> Forward, adds BglII site. Start codon is present.
</LI><LI> Complementary Tm = 63.9 C.
</LI></UL><UL><LI> AAA ACTAGT ACC ttt cac ttc gat cca atc
</LI><LI> Reverse, no stop codon; adds a glycine, adds SpeI.
</LI><LI> Complementary Tm = 55.9 C.
</LI></UL><H3><SPAN class="mw-headline" id="crtE">crtE</SPAN></H3><P>See <A href="http://www.ncbi.nlm.nih.gov/nuccore/22474502?from=225&amp;to=1133" class="external text" rel="nofollow">here</A>.
</P><UL><LI> ACT AGATCT atg acg gtc tgc gca aaa aaa c
</LI><LI> Forward, adds BglII site, start codon present.
</LI><LI> Complementary Tm = 68.9 C.
</LI></UL><UL><LI> AAA ACTAGT ACC act gac ggc agc gag ttt ttt g
</LI><LI> Reverse, adds SpeI site, adds a glycine, no stop codon.
</LI><LI> Complementary Tm = 69.3 C.
</LI></UL><H3><SPAN class="mw-headline" id="crtB">crtB</SPAN></H3><P>See <A href="http://www.ncbi.nlm.nih.gov/nuccore/22474502?from=5057&amp;to=5986" class="external text" rel="nofollow">here</A>.
</P><UL><LI> ACT AGATCT atg aat aat ccg tcg tta ctc aat c
</LI><LI> Forward, adds BglII site, start codon present.
</LI><LI> Long primer because Tm was low.
</LI><LI> Complementary Tm = 62.9 C.
</LI></UL><UL><LI> AAA ACTAGT ACC gag cgg gcg ctg cca gag
</LI><LI> Reverse, adds SpeI site, adds a glycine, no stop codon.
</LI><LI> Complementary Tm = 74.3 C.
</LI></UL><H3><SPAN class="mw-headline" id="crtI">crtI</SPAN></H3><P>See <A href="http://www.ncbi.nlm.nih.gov/nuccore/22474502?from=3582&amp;to=5060" class="external text" rel="nofollow">here</A>.
</P><UL><LI> ACT AGATCT atg aaa cca act acg gta att g
</LI><LI> Forward, adds BglII site, start codon present.
</LI><LI> Complementary Tm = 59.3 C.
</LI></UL><UL><LI> AAA ACTAGT ACC tat cag atc ctc cag cat caa ac
</LI><LI> Reverse, adds SpeI site, adds a glycine, no stop codon.
</LI><LI> Complementary Tm = 64.1 C.
</LI></UL><H2><SPAN class="mw-headline" id="Gibson_assembly_of_pG8SAET_with_malS">Gibson assembly of pG8SAET with malS</SPAN></H2><P>Make the 2 PCR products, assemble in Gibson way, and hope for the best...
</P><UL><LI><B>malSF4p8</B>: CA AAT GCT GCG CAA CAC GAT gcc agc tgg act tct ccg gg
</LI><LI> malS forward primer
</LI></UL><UL><LI><B>malSR4p8</B>: C TTT TGC GGG ATC GTC ACC CTC ctg ttg ccc tgc cca gac gac
</LI><LI> malS reverse primer
</LI></UL><UL><LI><B>vectF4p8</B>: GTC GTC TGG GCA GGG CAA CAG gag ggt gac gat ccc gca aaa g
</LI><LI> vector forward primer
</LI></UL><UL><LI><B>vectR4p8</B>: CC CGG AGA AGT CCA GCT GGC atc gtg ttg cgc agc att tg
</LI><LI> vector reverse primer
</LI></UL><H2><SPAN class="mw-headline" id="Primers_in_planning_.28inaZ.2C_3603_b.p..29">Primers in planning (inaZ, 3603 b.p.)</SPAN></H2><P>See <A href="http://www.ncbi.nlm.nih.gov/nuccore/X03035" class="external text" rel="nofollow">here</A>.
</P><UL><LI> ACT AGATCT atg aat ctc gac aag gcg ttg
</LI><LI> Forward, adds BglII site, start codon present.
</LI><LI> Complementary Tm = 66.2 C.
</LI></UL><UL><LI> AAA ACTAGT ACC ctt tac ctc tat cca gtc atc gtc
</LI><LI> Reverse, adds SpeI site, adds a glycine, no stop codon.
</LI><LI> Complementary Tm = 61.8 C.
</LI></UL><H2><SPAN class="mw-headline" id="Primers_in_planning_.28amylase.2C_mature_peptide_only.29">Primers in planning (amylase, mature peptide only)</SPAN></H2><P><I>malS</I> is a periplasmic gene which has a signal peptide. In order to fuse it to INP, it may be necessary to isolate the mature protein sequence, which is <A href="http://www.ncbi.nlm.nih.gov/protein/CAA41740.1" class="external text" rel="nofollow">claimed</A> to be amino acids 18-676; i.e. we would need to delete the first 51 bases in the DNA.
</P><UL><LI> ACT AGATCT gcc agc tgg act tct ccg gg
</LI><LI> Forward, adds SglII site.
</LI><LI> Complementary Tm = 72.5 C.
</LI></UL><H2><SPAN class="mw-headline" id="Primers_in_planning_.28bglX.29">Primers in planning (bglX)</SPAN></H2><H3><SPAN class="mw-headline" id="Mature_peptide_only">Mature peptide only</SPAN></H3><P><I>bglX</I> is a periplasmic gene which has a signal peptide. In order to fuse it to INP, it may be necessary to isolate the mature protein sequence, which SignalP predicts to be amino acids 21 onwards; i.e. we would need to delete the first 60 coding bases in the DNA.
</P><UL><LI> ACT AGATCT gat gat tta ttc ggc aac c
</LI><LI> Forward, adds BglII site.
</LI><LI> Complementary Tm = 59.4 C.
</LI></UL><H2><SPAN class="mw-headline" id="Primers_in_planning_.28biorefinery.29">Primers in planning (biorefinery)</SPAN></H2><P>These things may be produced on their own (as a control) or fused to INP. We probably won't bother with fusion to phage? In any case, the unfused versions used as controls will need start codons.
</P><H3><SPAN class="mw-headline" id="Aldose_reductase">Aldose reductase</SPAN></H3><P>From the <A href="/Team:Edinburgh/Biorefineries#Sorbitol" title="Team:Edinburgh/Biorefineries" class="mw-redirect">reverse complement</A> of <A href="http://www.ncbi.nlm.nih.gov/nuccore/15594346?from=538328&amp;to=539275&amp;report=gbwithparts" class="external text" rel="nofollow">this sequence</A> which we can get from <A href="http://www.straininfo.net/strains/149324/browser" class="external text" rel="nofollow">DSM 4680</A>. Might be problems with low GC content of gene. Some alternative strains that might be used instead:
</P><UL><LI><A href="http://www.ncbi.nlm.nih.gov/nuccore/159162017?from=1026949&amp;to=1027815&amp;report=gbwithparts" class="external text" rel="nofollow"><I>Lactobacillus acidophilus</I> NCFM</A></LI><LI><A href="http://www.ncbi.nlm.nih.gov/nuccore/209953764?from=197281&amp;to=198111&amp;report=gbwithparts" class="external text" rel="nofollow"><I>Providencia alcalifaciens</I> DSM 30120</A> (but BLAST says this is probably a 2,5-diketo-D-gluconate reductase)
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/nuccore/224798838?from=118139&amp;to=119005&amp;report=gbwithparts" class="external text" rel="nofollow"><I>Lactobacillus acidophilus</I> ATCC 4796 == LMG 11470</A></LI></UL><P>Primers for DSM 4680:
</P><UL><LI> ACT AGATCT gtg aat aat ctt aaa gac
</LI><LI> Forward, adds BglII site, has start codon (gtg).
</LI><LI> Complementary Tm = 42.3 C.
</LI></UL><UL><LI> AAA ACTAGT A TTA tta aat ttt ttt gtt aaa ta
</LI><LI> Reverse, adds 2nd stop codon, adds standard suffix.
</LI><LI> Complementary Tm = 46.6 C.
</LI></UL><H2><SPAN class="mw-headline" id="Primers_in_planning_.28GFP:_green_fluorescent_protein.29">Primers in planning (GFP: green fluorescent protein)</SPAN></H2><P>This can be PCR'd from &lt;partinfo&gt;BBa_E0040&lt;/partinfo&gt; which is has been in the distribution forever. I assume we'll have some floating around the lab somewhere?
</P><UL><LI> ACT AGATCT atg cgt aaa gga gaa gaa ctt ttc
</LI><LI> Forward, adds BglII site, has start codon.
</LI><LI> Complementary region Tm = 62.6 C.
</LI></UL><UL><LI> AAA ACTAGT ACC ttt gta tag ttc atc cat gcc atg
</LI><LI> Reverse, no stop codon; adds a glycine, adds SpeI.
</LI><LI> Complementary region Tm = 64.4 C.
</LI></UL><H2><SPAN class="mw-headline" id="Primers_in_planning_.28zippers.29">Primers in planning (zippers)</SPAN></H2><P>The GR1 and GR2 zippers may have applications in phage display. In both cases, stuff will be fused to both N and C terminals (see <A href="/Team:Edinburgh/Phage_Reactors" title="Team:Edinburgh/Phage Reactors" class="mw-redirect">Team:Edinburgh/Phage_Reactors</A>) so neither start nor stop codons are required.
</P><H3><SPAN class="mw-headline" id="BBa_K415124_.28GR1.29">BBa_K415124 (GR1)</SPAN></H3><P>It ought to be possible to PCR this out of &lt;partinfo&gt;BBa_K415151&lt;/partinfo&gt;, which is in the latest DNA distribution. <B>Note though that sequencing is claimed to be inconsistent.</B> The official sequence seems to show that the construct has somehow been replaced with the aminopeptidase PepN gene.
</P><UL><LI> ACT AGATCT gag gag aag tcc cgg ctg
</LI><LI> Forward, adds BglII site.
</LI><LI> Complementary Tm = 65.1 C.
</LI></UL><UL><LI> AAA ACTAGT ACC aca acc tcc tac aga ctg g
</LI><LI> Reverse, adds a glycine, adds SpeI.
</LI><LI> Complementary Tm = 56.6 C.
</LI></UL><H3><SPAN class="mw-headline" id="BBa_K415125_.28GR2.29">BBa_K415125 (GR2)</SPAN></H3><P>We can PCR this out of &lt;partinfo&gt;BBa_K415152&lt;/partinfo&gt;.
</P><UL><LI> ACT AGATCT aca tcc cgc ctg gag ggc c
</LI><LI> Forward, adds BglII site
</LI><LI> Complementary Tm = 74.9 C.
</LI></UL><UL><LI> AAA ACTAGT ACC gca acc tcc gac gtc ttg c
</LI><LI> Reverse, adds a glycine, adds SpeI.
</LI><LI> Complementary Tm = 68.3 C.
</LI></UL><H2><SPAN class="mw-headline" id="BioSandwich_spoligos_actually_made">BioSandwich spoligos actually made</SPAN></H2><H3><SPAN class="mw-headline" id=".22Start_Spacer.22_.28also_has_T7_promoter.29">&quot;Start Spacer&quot; (also has T7 promoter)</SPAN></H3><UL><LI><B>spacerT7fs:</B> CTAGAG ttaatacgactcactatagggagaggaggtac
</LI><LI><B>spacerT7rt:</B> gtacctcctctccctatagtgagtcgtattaa CT 
</LI><LI><B>spacerT7rl:</B> GATC gtacctcctctccctatagtgagtcgtattaa CTCTAG 
</LI></UL><H3><SPAN class="mw-headline" id="Stop_-_RBS_-_Start_.281.29">Stop - RBS - Start (1)</SPAN></H3><UL><LI><B>spacer1fs:</B> CTAGC tga tatttcgcgtaaggaaatccatt atg G
</LI><LI><B>spacer1rt:</B> C cat aatggatttccttacgcgaaata tca G
</LI><LI><B>spacer1rl:</B> GATCC cat aatggatttccttacgcgaaatatca GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="Stop_-_RBS_-_Start_.282.29">Stop - RBS - Start (2)</SPAN></H3><UL><LI><B>spacer2fs:</B> CTAGC aacatatca taa cggagtgatcgca atg G
</LI><LI><B>spacer2rt:</B> C cat tgcgatcactccg tta tgatatgtt G
</LI><LI><B>spacer2rl:</B> GATCC cat tgcgatcactccg tta tgatatgtt GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="Stop_-_Strongest_RBS">Stop - Strongest RBS</SPAN></H3><P>Explanation: stop + random junk + strongest RBS (&lt;partinfo&gt;BBa_B0034&lt;/partinfo&gt;). RBS strength is 100%
</P><UL><LI><B>spacer3fs</B>: CTAGC tga acgtaaatactcagt aaagaggagaaa G
</LI><LI><B>spacer3rt</B>: C tttctcctcttt actgagtatttacgt tca G
</LI><LI><B>spacer3rl</B>: GATCC tttctcctcttt actgagtatttacgt tca GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="Stop_-_Weak_RBS">Stop - Weak RBS</SPAN></H3><P>Explanation: stop + random junk + weak RBS (&lt;partinfo&gt;BBa_J61100&lt;/partinfo&gt;). RBS strength is 2.0% - 4.29%
</P><UL><LI><B>spacer4fs</B>: CTAGC tag gtcttagtcttcgtc aaagaggggaca G
</LI><LI><B>spacer4rt</B>: C tgtcccctcttt gacgaagactaagac cta G
</LI><LI><B>spacer4rl</B>: GATCC tgtcccctcttt gacgaagactaagac cta GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="PT_linker_.2810_aa.29">PT linker (10 aa)</SPAN></H3><P>Sequence of lowercase region below is PTTSPTPTPT.
</P><UL><LI><B>linker1fs</B>: CTAGC ccg acg acc agc ccc acg ccg acc ccg acg G
</LI><LI><B>linker1rt</B>: C cgt cgg ggt cgg cgt ggg gct ggt cgt cgg G
</LI><LI><B>linker1rl</B>: GATCC cgt cgg ggt cgg cgt ggg gct ggt cgt cgg GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="Serine-rich_linker_.2812_aa.29">Serine-rich linker (12 aa)</SPAN></H3><P>Back-translated from a.a. 141-152 of <I>Cellvibrio japonicus</I><A href="http://www.uniprot.org/blast/?about=P14768" class="external text" rel="nofollow">xylanase A</A>. Sequence of lowercase region below is SSIASSSPSSVA.
</P><UL><LI><B>linker2fs</B>: CTAGC agc agc att gcg agc agc agc ccg agc agc gtg gcg G
</LI><LI><B>linker2rt</B>: C cgc cac gct gct cgg gct gct gct cgc aat gct gct G
</LI><LI><B>linker2rl</B>: GATCC cgc cac gct gct cgg gct gct gct cgc aat gct gct GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="Another_PT_linker_.2811_aa.29">Another PT linker (11 aa)</SPAN></H3><P>SPTPTTPTPTS.
</P><UL><LI><B>linker3fs</B>: CTAGC tct ccg acc cct acc acg cca act ccg acc agc G
</LI><LI><B>linker3rt</B>: C gct ggt cgg agt tgg cgt ggt agg ggt cgg aga G
</LI><LI><B>linker3rl</B>: GATCC gct ggt cgg agt tgg cgt ggt agg ggt cgg aga GCTAG
</LI></UL><H2><SPAN class="mw-headline" id="BioSandwich_spoligos_in_planning_.28ribosome_binding_sites.29">BioSandwich spoligos in planning (ribosome binding sites)</SPAN></H2><H3><SPAN class="mw-headline" id="Stop_-_Weakest_RBS">Stop - Weakest RBS</SPAN></H3><P>Explanation: stop + random junk + weakest RBS (&lt;partinfo&gt;BBa_B0033&lt;/partinfo&gt;). RBS strength is 0.35%
</P><UL><LI> FS: CTAGC taa ttctgacctacttaa tcacacaggac G
</LI><LI> RT: C gtcctgtgtga ttaagtaggtcagaa tta G
</LI><LI> RL: GATCC gtcctgtgtga ttaagtaggtcagaa tta GCTAG
</LI></UL><H2><SPAN class="mw-headline" id="BioSandwich_spoligos_in_planning_.28linkers.29">BioSandwich spoligos in planning (linkers)</SPAN></H2><H3><SPAN class="mw-headline" id="BBa_K105012_linker_.2810_aa.29">BBa_K105012 linker (10 aa)</SPAN></H3><P>From &lt;partinfo&gt;BBa_K105012&lt;/partinfo&gt;. Sequence of lowercase region below is GENLYFQSGG.
</P><UL><LI> FS: CTAGC ggt gaa aat ttg tat ttt caa tct ggt ggt G
</LI><LI> RT: C acc acc aga ttg aaa ata caa att ttc acc G
</LI><LI> RL: GATCC acc acc aga ttg aaa ata caa att ttc acc GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="BBa_K133132_linker_with_added_glycines_.2810_aa.29">BBa_K133132 linker with added glycines (10 aa)</SPAN></H3><P>From &lt;partinfo&gt;BBa_K133132&lt;/partinfo&gt; with one extra glycine at each end. Sequence of lowercase region below is GSACYCELSG.
</P><UL><LI> FS: CTAGC gga tcc gct tgt tac tgt gag ctt tcc gga G
</LI><LI> RT: C tcc gga aag ctc aca gta aca agc gga tcc G
</LI><LI> RL: GATCC tcc gga aag ctc aca gta aca agc gga tcc GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="Glycine-rich_linker_.2812_aa.29">Glycine-rich linker (12 aa)</SPAN></H3><P>Truncated version of &lt;partinfo&gt;BBa_K416001&lt;/partinfo&gt;. Sequence of lowercase region below is GGGSGGGGSGGG.
</P><UL><LI> FS: CTAGC gga gga ggc tct ggt gga ggc ggt agc gga ggc gga G
</LI><LI> RT: C tcc gcc tcc gct acc gcc tcc acc aga gcc tcc tcc G
</LI><LI> RL: GATCC tcc gcc tcc gct acc gcc tcc acc aga gcc tcc tcc GCTAG
</LI></UL><H3><SPAN class="mw-headline" id="E-tag_.2813_aa.29">E-tag (13 aa)</SPAN></H3><P>Codes for GAPVPYPDPLEPR which can be detected by anti-E-tag antibodies.
</P><UL><LI> FS: CTAGC ggt gcg ccg gtg ccg tat ccg gac cca ctg gaa ccg cgt G
</LI><LI> RT: C acg cgg ttc cag tgg gtc cgg ata cgg cac cgg cgc acc G
</LI><LI> RL: GATCC acg cgg ttc cag tgg gtc cgg ata cgg cac cgg cgc acc GCTAG
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Assembly">http://2011.igem.org/Team:Edinburgh/Assembly</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Assembly" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Assembly" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Assembly&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Assembly&amp;oldid=110991" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>