"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Collaboration
From 2011.igem.org
outline
in silico
in the lab
in society
postscript
abstract
safety
cell surface display
phage display
achievements
modelling
cellulases (C)
cellulases (Kappa)
cellulases (MATLAB)
model comparison
genetic instability
efficiency
artificial selection
phage replication
data overview
experiments
parts
BioSandwich
lab notebook
conclusions
the team
attributions
our sponsors
human practices
biorefinery design
biorefinery appendices
life cycle analysis
interviews: overview
interviews
interview analysis
collaboration
wiki watch
Hover over words in bold italics to see explanation.
Collaboration with the Synbio Community
iGEM teams have always been a highly visible part of the synthetic biology community, and an important part of iGEM is contributing to that community. Thus, the relationship between our project and the projects of other teams, past and present, is of the highest importance.
Contents
1 Thanks to UC Davis and KU Leuven
2 Thanks to ETH Zurich
3 Cooperation with Trieste
4 Wiki Watch
5 Updating the Registry
5.1 BBa_K118022
5.2 BBa_K415151
5.3 BBa_K392008
5.4 BBa_K265008
5.5 BBa_J33207
5.6 BBa_K118023
Thanks to UC Davis and KU Leuven
We gratefully acknowledge UC Davis 2009, who synthesised an E. coli optimised version of Ice Nucleation Protein, originally from the organism Pseudomonas syringae. We were delighted to discover that the part we received from the Registry had the correct sequence.
We also acknowledge the help of this year's KU Leuven team, whose project involved Ice Nucleation Protein, and who sent us plasmid pUC1813ICE containing the inaZ version of the gene.
Thanks to ETH Zurich
After we presented (inter alia) our modelling at the European Jamboree, the team from ETH Zurich contacted us with some helpful bugfixes in one of our models — the MATLAB cellulase model — which we had some trouble with. These have been fixed, thank you Zurich!
Cooperation with Trieste
Like us, Trieste's team this year are using the BioBrick BBa_K392008 . This part by Osaka 2010 encodes a cellulase, β-glucosidase, from the bacterium Cellulomonas fimi. It is known to work in the lab of Chris French (Edinburgh's supervisor).
Chris French sent Trieste a copy of the plasmid, however they reported some sequence errors and asked us for comments. We sequenced the gene independently and discovered that an apparent frameshift is present near the start of the BioBrick. This very same "frameshift" was seen in Trieste's sequencing results.
Since a part with an early frameshift cannot possibly work, but the part does work, we looked for an explanation. Some 220 bases into the part, a second ATG codon is found. This codon is in-frame and there is a plausible ribosome binding site (containing "gaagga") just upstream of it. We therefore believe that this second ATG is the true start codon. The RBS would explain why the part can be expressed and work.
Here is the start of the sequence, with these features highlighted:
> BBa_K392008 Part-only sequence (1671 bp)
atgggcgaccggttccagcaggccggtcgcccacgccgccgcggcccggcgagggccgtt
aaccgtaccggtcaagaagacgcgtcgacggggtcgagggagcggtcccacgcgtgtatc
gtatcgtttcgacaccgccacccggccaccgggcacgcaccggggacgcagcagtccccg
ccccggccaccccctgtcaccgaaacccgaaggaccctcatgaccaccacgcgcccctcg
[rest omitted]
In agreement with this hypothesis, a recently published sequence of C. fimi β-glucosidase (labelled as such by Expasy though NCBI calls it a β-galactosidase) starts at the 2nd ATG codon of K392008. This is in contrast to an older published sequence .
We passed this information on to the Trieste team, who agreed that it is a likely explanation.
In addition to this, at the request of the Trieste team we sent details of an assay that can be used to test for activity of this β-glucosidase. This assay involves 4-methylumbelliferyl-β-D-glucuronide, which is cleaved by β-glucosidase to yield a fluorescent product visible under long-wave blue light. We used this assay ourselves for BBa_K523014 .
Wiki Watch
We created a Wiki Watch page that attempted to convey the very basics of what every other team in the competition was doing. Early in the summer, this page was linked from the official iGEM Community page, and we hope it has helped other teams find partners for cooperation.
Updating the Registry
The hub of our community's collective experience is the Parts Registry. During our work, we have discovered some useful information about several parts in the Registry. We have entered this information in the "experience" section of the relevant pages.
BBa_K118022
This part by Edinburgh 2008 encodes an exoglucanase, capable of degrading cellulose. We tested its ability to degrade the cellulose analog MUC, and found that it could.
This information has been added to the Registry on the relevant experience page .
BBa_K415151
This part by MIT 2010 is supposed to encode a fusion of the pVIII protein to a GR1 zipper. However, we noticed that the official verification sequencing carried out by the Registry did not match the expected sequence and did not contain a prefix or suffix.
We tried to determine what the sequence actually is, and discovered that it is a fragment of the E. coli main genome. This fragment naturally has EcoRI and PstI sites at its ends. It has evidently been cut with those enzymes and has somehow become inserted into pSB1C3 via those sites. This naturally wipes out the XbaI and SpeI sites, explaining their absence.
This information has been added to the Registry on the relevant experience page .
BBa_K392008
As mentioned above, Trieste and Edinburgh cooperated around the use of this part from Osaka 2010, and came to suspect that the second ATG present in the sequence is the real start codon.
In addition, there are some other discrepancies between the published sequence and the physical K392008 as well.
This information has been added to the Registry on the relevant experience page .
BBa_K265008
This is the synthetic Ice Nucleation Protein from UC Davis 2009. We updated the relevant experience page to mention our successful use of it.
BBa_J33207
This part encodes LacZα, and provides a cunning way to make new BioBricks using shorter PCR primers than normal, by incorporating a SacI site. We made a new version deleting about 50 unnecessary bases and based on a different restriction site.
This information has been added to the Registry on the relevant experience page .
BBa_K118023
This part was made by Edinburgh 2008. When we sequenced it, we found a discrepancy between the physical DNA and the Registry sequence. We checked against a published genome sequence and found that the Registry sequence is incorrect.
This information has been added to the Registry on the relevant experience page .
Retrieved from " http://2011.igem.org/Team:Edinburgh/Collaboration "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
