<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Experiments"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Experiments">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Experiments&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Experiments&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Experiments&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Experiments" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Experiments</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Experiments</P><P>Actual results of experiments are placed here, but <B>more detailed discussions are also found on the relevant Registry pages.</B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#malS"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">malS</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#bglX_and_cex"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">bglX and cex</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#INP-EYFP"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">INP-EYFP</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#cxnA"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">cxnA</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#BioSandwich"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">BioSandwich</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Parts_made_with_BioSandwich"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Parts made with BioSandwich</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Extensive_tests_-_what_can_go_wrong.3F"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">Extensive tests - what can go wrong?</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#BioSandwich_with_Gibson_Assembly"><SPAN class="tocnumber">5.2.1</SPAN><SPAN class="toctext">BioSandwich with Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#BioSandwich_with_Overlap_Extension_PCR"><SPAN class="tocnumber">5.2.2</SPAN><SPAN class="toctext">BioSandwich with Overlap Extension PCR</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#BioSandwich_with_Ligation_Independent_Cloning"><SPAN class="tocnumber">5.2.3</SPAN><SPAN class="toctext">BioSandwich with Ligation Independent Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#BioSandwich_with_Blunt-End_Ligation_Independent_Cloning"><SPAN class="tocnumber">5.2.4</SPAN><SPAN class="toctext">BioSandwich with Blunt-End Ligation Independent Cloning</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#BioSandwich_with_CPEC:_Circular_Polymerase_Extension_Cloning"><SPAN class="tocnumber">5.2.5</SPAN><SPAN class="toctext">BioSandwich with CPEC: Circular Polymerase Extension Cloning</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-13"><A href="#Semi-quantitative_results"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Semi-quantitative results</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Various_assemblies"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Various assemblies</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="malS"><I>malS</I></SPAN></H2><P>We placed <I>malS</I> (an <I>E. coli</I><SPAN class="hardword" id="amylase">amylase</SPAN> gene) under the control of the lac <SPAN class="hardword" id="promoter">promoter</SPAN> and grew the cells on <SPAN class="hardword" id="starch">starch</SPAN> agar. An iodine-based <SPAN class="hardword" id="assay">assay</SPAN> was carried out testing for starch degradation.
</P><CENTER><TABLE style="margin-top: 1em; margin-bottom: 1em;"><TBODY><TR><TD>  </TD><TD>  </TD></TR><TR><TD width="200px" valign="top"> Before the assay. Colony 1 failed to grow. <B>Control at top.</B></TD><TD>  </TD><TD width="200px" valign="top"> Immediately after iodine flooding.
</TD><TD>  </TD><TD width="200px" valign="top"> 40 minutes after iodine flooding.
</TD></TR></TBODY></TABLE></CENTER><P>We also conducted an assay based on cell extract's ability to degrade starch, producing glucose that would react with 3,5-dinitrosalicylic acid.
</P><P>
Finally, we tested the ability of bacteria with overexpression of <I>malS</I> to grow on a starch agar with no other carbon source. Indeed it could, while normal <I>E. coli</I> cannot:
</P><P>
For more information, see the Registry page on <A href="http://partsregistry.org/Part:BBa_K523006" class="external text" rel="nofollow">BBa_K523006</A>.
</P><H2><SPAN class="mw-headline" id="bglX_and_cex"><I>bglX</I> and <I>cex</I></SPAN></H2><P>We placed <I>bglX</I> (a <SPAN class="hardword" id="cryptic">cryptic</SPAN><I>E. coli</I>β-glucosidase gene) under the control of the lac promoter and compared its activity to the exoglucanase <I>cex</I>.
</P><CENTER><TABLE style="margin-top: 1em; margin-bottom: 1em;"><TBODY><TR><TD>  </TD></TR><TR><TD width="300px" valign="top"><B>MUG assay.</B><I>bglX</I> on left, <I>cex</I> on right.
</TD><TD>  </TD><TD width="300px" valign="top"><B>MUC assay.</B><I>bglX</I> on left, <I>cex</I> on right.
</TD></TR></TBODY></TABLE></CENTER><P><I>bglX</I> was capable of degrading the substrate MUG, which has a β(1→4) bond, similar to that of <SPAN class="hardoword" id="cellobiose">cellobiose</SPAN>.
</P><P>We tested the ability of <I>E. coli</I> with this <I>bglX</I> part to grow on minimal media with cellobiose as the only carbon source. It could not. By contrast, it did grow if glucose was the carbon source, showing that it is fundamentally capable of growing on minimal media:
</P><CENTER><TABLE style="margin-top: 1em; margin-bottom: 1em;"><TBODY><TR><TD>  </TD></TR><TR><TD width="300px" valign="top">Cellobiose as the only carbon source. K523014 (bottom) fails to grow.
</TD><TD>  </TD><TD width="300px" valign="top">Glucose as the only carbon source. K523014 can grow.
</TD></TR></TBODY></TABLE></CENTER><P>For more information, see the Registry page on <A href="http://partsregistry.org/Part:BBa_K523014" class="external text" rel="nofollow">BBa_K523014</A>.
</P><H2><SPAN class="mw-headline" id="INP-EYFP">INP-EYFP</SPAN></H2><P>We made a fusion (<A href="http://partsregistry.org/Part:BBa_K523013" class="external text" rel="nofollow">BBa_K523013</A>) of <SPAN class="hardword" id="inp">Ice Nucleation Protein</SPAN> and Enhanced Yellow Fluorescent Protein, and placed it under the control of the Lac promoter.
</P><P>Under blue light, the cells fluoresced yellow, as shown below (left image). Unfortunately, our imaging technology was not capable of showing whether the fluorescence was localised to the outer membrane, so we centrifuged the cells so that the membrane fraction was localised to the bottom of the tube and compared the result to a colony expressing EYFP without INP. This showed that the INP-EYFP was indeed localised to the cell membrane.
</P><CENTER><TABLE style="margin-top: 1em; margin-bottom: 1em;"><TBODY><TR><TD>  </TD><TD>  </TD></TR><TR><TD width="147px" valign="top"> Cells fluorescing yellow.
</TD><TD>  </TD><TD width="200px" valign="top"> Centrifuged cells. Control on left. <B>INP-EYFP on right.</B></TD><TD>  </TD><TD width="200px" valign="top"> The same image passed through GIMP's auto white balance filter.
</TD></TR></TBODY></TABLE></CENTER><P>For more information, see the Registry page on <A href="http://partsregistry.org/Part:BBa_K523013" class="external text" rel="nofollow">BBa_K523013</A>.
</P><H2><SPAN class="mw-headline" id="cxnA"><I>cxnA</I></SPAN></H2><P>In line with our <A href="/Team:Edinburgh/Cell_Display#An_alternative:_protein_chains" title="Team:Edinburgh/Cell Display">beads on a string</A> proposal, We created a fusion of <I>cex</I> and <I>cenA</I>. This showed the activity of both enzymes:
</P><UL><LI> Cex activity was tested on MUC plates.
</LI><LI> CenA activity was tested on carboxymethylcellulose (CMC) plates.
</LI></UL><P>The latter test produces a zone of clearing if CMC is degraded by the endoglucanase domain. We had three successful colonies:
</P><P>
For more information, see the Registry page on <A href="http://partsregistry.org/Part:BBa_K523025" class="external text" rel="nofollow">BBa_K523025</A>.
</P><H2><SPAN class="mw-headline" id="BioSandwich">BioSandwich</SPAN></H2><P>The BioSandwich system is outlined on <A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich">its own page</A>. BioSandwich creates parts with <SPAN class="hardword" id="homology">homologous</SPAN> ends, but there are a number of ways in which the final assembly could be carried out. Throughout the project we mostly used Gibson Assembly.
</P><H3><SPAN class="mw-headline" id="Parts_made_with_BioSandwich">Parts made with BioSandwich</SPAN></H3><P>Parts that we believe are correct (subject to final verification sequencing) include:
</P><UL><LI><A href="http://partsregistry.org/Part:BBa_K523013" class="external text" rel="nofollow">BBa_K523013</A>: PlacLacZ-INP-YFP
</LI><LI><A href="http://partsregistry.org/Part:BBa_K523019" class="external text" rel="nofollow">BBa_K523019</A>: PlacLacZ-INP-cex [<B>update:</B> sequencing shows this part has errors]
</LI><LI><A href="http://partsregistry.org/Part:BBa_K523020" class="external text" rel="nofollow">BBa_K523020</A>: PlacLacZ-INP-bglX [<B>update:</B> sequencing shows this part has errors]
</LI><LI><A href="http://partsregistry.org/Part:BBa_K523022" class="external text" rel="nofollow">BBa_K523022</A>: PlacLacZ-crtEIB
</LI></UL><H3><SPAN class="mw-headline" id="Extensive_tests_-_what_can_go_wrong.3F">Extensive tests - what can go wrong?</SPAN></H3><P>We also tested a number of final assembly methods, by attempting to join a <SPAN class="hardword" id="promoter">promoter</SPAN> and LacZ construct to a Yellow Fluorescent coding sequence, making <A href="http://partsregistry.org/Part:BBa_K523023" class="external text" rel="nofollow">BBa_K523023</A>. This part will produce a <SPAN class="hardword" id="polycistronic">polycistronic</SPAN> RNA transcript, coding for proteins that turn the bacteria blue (on Xgal) but also make it fluoresce yellow.
</P><P>BioSandwich inserts short spacers between each part to be assembled. Note that after spacer2 we expect to see some extra bases before we reach the RFC10 suffix, because of the choice of vector. We can call these extra bases the &quot;stuffing&quot;. So, the expected sequence is:
</P><DL><DD><B>Vector -- spacerT7 -- Plac,LacZ -- spacer1 -- EYFP -- spacer2 -- stuffing</B></DD></DL><P>Successful assemblies are expected to generate blue colonies that fluoresce yellow under a blue light. We tried several assembly methods, all of which generated many colonies with the correct phenotype, but also many colonies with an incorrect phenotype...
</P><P>From each plate we sequenced a colony with the correct phenotype (if there were any) and some colonies with incorrect phenotypes, in an attempt to understand what the failure modes were.
</P><H4><SPAN class="mw-headline" id="BioSandwich_with_Gibson_Assembly">BioSandwich with Gibson Assembly</SPAN></H4><P>Like, OEPCR, this method produces many correct colonies and many incorrect colonies.
</P><P>The Gibson method was described by <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Gibson <I>et al</I> (2009)</A>. A gentle introduction was written by the 2010 Cambridge iGEM team, and is found <A href="https://2010.igem.org/Team:Cambridge/Gibson/Introduction" class="external text" rel="nofollow">here</A>.
</P><P>We sequenced a colony with the correct phenotype, as well as others with incorrect phenotypes:
</P><UL><LI><B>Colony &quot;gby&quot; [blue, yellow]</B> - expected scar is missing at approx base 680 of the forward chromatogram. Also, the spacer ends with ATG and the part starts with ATG, but only one ATG is seen. In total 9 bases are missing. There is homology of &quot;tatgg&quot; at both ends of the deleted fragment. There is a point mutation at approx base 500 of the forward chromatogram.
</LI><LI><B>Colony &quot;gb&quot; [blue]</B> - there is a deletion of about 80 bases at approx base 700 of the chromatogram. This has been caused by homology of a &quot;gggcgagg&quot; found at both ends of the deleted sequence. The 9 bases mentioned for <B>gby</B> are also missing.
</LI><LI><B>Colony &quot;gwy&quot; [yellow]</B> - seems correct except for a point mutation in the reverse chromatogram. The mutation is synonymous (ctg -&gt; ttg). There's no clear reason why the colony wasn't blue.
</LI></UL><P>Lessons learned: bits of homology, even quite small, can cause problems with this method.
</P><H4><SPAN class="mw-headline" id="BioSandwich_with_Overlap_Extension_PCR">BioSandwich with Overlap Extension PCR</SPAN></H4><P>Overlap Extension PCR can be performed using spoligos as primers. The extension time should be calculated from the entire product size. The final product will not be circular, so can be run on a gel to test its length. Later, it will need to have its ends ligated.
</P><P>This method produces many correct colonies, although it also produces many incorrect colonies.
</P><P>[For our tests, different primers were used and spacer2 was not needed.]
</P><UL><LI><B>Colony &quot;bby&quot; [blue, yellow]</B> - correct sequence.
</LI><LI><B>Colony &quot;bb&quot; [blue]</B> - has a massive deletion (several hundred bases) in the EYFP gene after about 60 bases. There is partial homology between the start and end of the deletion: &quot;tggtcgagctggac&quot; vs &quot;tggacgagctgtac&quot;.
</LI><LI><B>Colony &quot;bw&quot; [plain]</B> - has spacerT7 joined to spacer1, and the same massive deletion as <B>bb</B>.
</LI></UL><P>Again homology seems to be an issue. Parts joining randomly (in bw) is worrying.
</P><H4><SPAN class="mw-headline" id="BioSandwich_with_Ligation_Independent_Cloning">BioSandwich with Ligation Independent Cloning</SPAN></H4><P>In our experience, this method has been only weakly successful. We have seen one correct colony in several attempts.
</P><P>To attempt the LIC method, simply mix the parts and vector (4 uL of each). Heat a beaker of water to 95 C, and float the reaction tube in it, and allow everything to cool. Afterwards, it may now be necessary to centrifuge the tube briefly to move the liquid back to the bottom.
</P><UL><LI><B>Colony &quot;b1&quot; [blue, yellow]</B> - sequence read is of low quality but things seem correct.
</LI><LI><B>Colony &quot;y2&quot; [yellow]</B> - appears to simply be <A href="http://partsregistry.org/Part:BBa_K216011" class="external text" rel="nofollow">BBa_K216011</A>, the template for the initial PCR
</LI><LI><B>Colony &quot;n4&quot; [plain]</B> - sequence is unreadable.
</LI></UL><P>The y2 colony is probably expressing a template for PCR which made it all the way through the various steps that came afterwards.
</P><H4><SPAN class="mw-headline" id="BioSandwich_with_Blunt-End_Ligation_Independent_Cloning">BioSandwich with Blunt-End Ligation Independent Cloning</SPAN></H4><P>This is expected to fail as Ligation Independent Cloning requires long overhangs at the end of each part, not blunt ends. No correct colonies were seen.
</P><UL><LI><B>Colony &quot;bb1&quot; [blue]</B> - seems to be <A href="http://partsregistry.org/Part:BBa_J33207" class="external text" rel="nofollow">BBa_J33207</A> followed by part of <I>E. coli</I> ferrous iron transport gene A.
</LI><LI><B>Colony &quot;bw2&quot; [plain]</B> - has spacerT7 followed by the stuffing.
</LI><LI><B>Colony &quot;bw3&quot; [plain]</B> - has RFC10 prefix followed by RFC10 suffix, as if XbaI and SpeI sites have joined in a normal vector.
</LI></UL><H4><SPAN class="mw-headline" id="BioSandwich_with_CPEC:_Circular_Polymerase_Extension_Cloning">BioSandwich with CPEC: Circular Polymerase Extension Cloning</SPAN></H4><UL><LI><B>Colony &quot;cby&quot; [blue, yellow]</B> - there is a single-base deletion near the start of the PlacLacZ part, with no obvious cause. It is too early to affect expression. The last few bases of spacer1 are missing, as well as the first few bases of the EYFP part (9 bases total). This is explained by homology of &quot;tatgg&quot;. Otherwise seems correct.
</LI><LI><B>Colony &quot;cb&quot; [blue]</B> - Normal until the end of spacer1, which is followed immediately by spacer2 and the stuffing.
</LI><LI><B>Colony &quot;cw&quot; [plain]</B> - spacerT7 is followed immediately by a second copy of spacerT7, after which comes spacer2 and a second copy of spacer2. Then the stuffing.
</LI></UL><P>This gave some of the weirder results.
</P><H3><SPAN class="mw-headline" id="Semi-quantitative_results">Semi-quantitative results</SPAN></H3><P>Our lab supervisor Chris French recorded the results of some early BioSandwich assembly attempts. Since correct assemblies of the construct described above (PlacLacZ-EYFP) should produce blue colonies that fluoresce yellow under blue light, we can get a rough idea of how many colonies are correct by counting phenotypes.
</P><DL><DT> BioSandwich with OEPCR
</DT><DD> Assembly produced around 100 colonies on a 100 microlitre plate, about half of these blue, and most of the blue also yellow.
</DD><DT> BioSandwich with Gibson
</DT><DD> Assembly produced about 100-200 colonies on a 100 microlitre plate, about half blue and half white, and the blue ones about 75% yellow.
</DD><DT> BioSandwich with CPEC
</DT><DD> Assembly produced 60 colonies, of which 26 were blue, of which half were also yellow.
</DD></DL><H3><SPAN class="mw-headline" id="Various_assemblies">Various assemblies</SPAN></H3><P>The core team made a number of constructs with PlacLacZ plus other genes. While these other genes usually did not give an easily spotted phenotype, we can at least see how many of them were blue on Xgal.
</P><P>As an explanation, &quot;100 plate&quot; and &quot;900 plate&quot; refer to the concentration of transformed cells plated.
</P><DL><DT><B>BioSandwich with OEPCR</B></DT><DD> pSB1C3-PlacLacZ-INP-YFP <B>(should be blue and yellow)</B></DD><DD> 100 plate&gt; 1 white+yellow
</DD><DD> 900 plate&gt; 1 blue, 7 white+yellow, 1 white
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-MalS <B>(should be blue)</B></DD><DD> 100 plate&gt; 188 white, 7 blue
</DD><DD> 900 plate&gt; 165 blue, 376 white
</DD></DL><DL><DT><B>BioSandwich with Gibson</B></DT><DD> pSB1C3-PlacLacZ-INP-cex <B>(should be blue)</B></DD><DD> 100 plate&gt; 46 white, 3 blue
</DD><DD> 900 plate&gt; 929 white, 48 blue
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-cex <B>(should be blue)</B></DD><DD> 100 plate&gt; 38 white, 2 blue, 1 red
</DD><DD> 900 plate&gt; 633 white, 61 blue, 2 red
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-bglX <B>(should be blue)</B></DD><DD> 100 plate&gt; 57 white
</DD><DD> 900 plate&gt; 575 white, 1 blue
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-bglX <B>(should be blue)</B></DD><DD> 100 plate&gt; 54 white, 16 blue
</DD><DD> 900 plate&gt; 423 white, 156 blue
</DD></DL><DL><DD> pSB1C3-PlacLacZ-crtEIB <B>(should be blue, or maybe red)</B></DD><DD> 100 plate&gt; 2 white
</DD><DD> 900 plate&gt; 1 blue, 80 white
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-malS <B>(should be blue)</B></DD><DD> 100 plate&gt; 16 blue, 74 white
</DD></DL><DL><DT><B>BioSandwich with CPEC</B></DT><DD> pSB1C3-PlacLacZ-crtEIB --- 7 cycles <B>(should be blue, or maybe red)</B></DD><DD> 100 plate&gt; 31 white, 19 blue
</DD><DD> 900 plate&gt; 257 white, 149 blue
</DD></DL><DL><DD> pSB1C3-PlacLacZ-crtEIB --- 3 cycles <B>(should be blue, or maybe red)</B></DD><DD> 100 plate&gt; 1 white
</DD><DD> 900 plate&gt; 15 white, 4 blue
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-bglX ---7 cycles <B>(should be blue)</B></DD><DD> 100 plate&gt; 21 white
</DD><DD> 900 plate&gt; 363 white, 1 red, 1 blue
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-bglX ---3 cycles <B>(should be blue)</B></DD><DD> 100 plate&gt; 75 white, 4 blue
</DD><DD> 900 plate&gt; 494 white, 36 blue, 2 red.
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-YFP <B>(should be blue and yellow)</B></DD><DD> 100 plate&gt; 15 white
</DD><DD> 900 plate&gt; 74 white, 1 blue+yellow
</DD></DL><DL><DD> pSB1C3-PlacLacZ-INP-MalS <B>(should be blue)</B></DD><DD> 100 plate&gt; 6 white, 4 blue
</DD><DD> 900 plate&gt; 32 blue, 127 white
</DD></DL><P>It seems our supervisor with his 15 years postdoctoral experience had slightly better results than we did. Hmm...
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Experiments">http://2011.igem.org/Team:Edinburgh/Experiments</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Experiments" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Experiments" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Experiments&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Experiments&amp;oldid=255404" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>