<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Edinburgh_Cell_Display"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Edinburgh/Cell_Display">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Edinburgh/Cell_Display&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Cell_Display&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Edinburgh/Cell_Display&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Edinburgh/Cell_Display" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Edinburgh/Cell Display</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV class="tops"><DIV class="mainmenu"><UL><LI><A href="/Team:Edinburgh" class="home">outline</A></LI><LI><A href="/Team:Edinburgh/Modelling" class="model">in silico</A></LI><LI><A href="/Team:Edinburgh/Data" class="lab">in the lab</A></LI><LI><A href="/Team:Edinburgh/Practices" class="practices">in society</A></LI><LI><A href="/Team:Edinburgh/Conclusions" class="team">postscript</A></LI></UL></DIV></DIV><DIV class="mids"><DIV id="sidebar"><DIV class="leftmenu" id="homemenu"><UL><LI><A href="/Team:Edinburgh" class="home_abstract">abstract</A></LI><LI><A href="/Team:Edinburgh/Safety" class="home_safety">safety</A></LI><LI><A href="/Team:Edinburgh/Cell_Display" class="home_cell">cell surface display</A></LI><LI><A href="/Team:Edinburgh/Phage_Display" class="home_phage">phage display</A></LI><LI><A href="/Team:Edinburgh/Achievements" class="home_achievements">achievements</A></LI></UL></DIV><DIV class="leftmenu" id="modelmenu"><UL><LI><A href="/Team:Edinburgh/Modelling" class="model_modelling">modelling</A></LI><LI><A href="/Team:Edinburgh/Cellulases (C model)" class="model_synergyc">cellulases (C)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (Kappa model)" class="model_synergykappa">cellulases (Kappa)</A></LI><LI><A href="/Team:Edinburgh/Cellulases (MATLAB model)" class="model_synergymatlab">cellulases (MATLAB)</A></LI><LI><A href="/Team:Edinburgh/Model Comparison" class="model_comparison">model comparison</A></LI><LI><A href="/Team:Edinburgh/Genetic_Instability" class="model_genetic_instability">genetic instability</A></LI><LI><A href="/Team:Edinburgh/Efficiency" class="model_efficiency">efficiency</A></LI><LI><A href="/Team:Edinburgh/Artificial_Selection" class="model_artificial_selection">artificial selection</A></LI><LI><A href="/Team:Edinburgh/Phage Replication" class="model_phagerep">phage replication</A></LI></UL></DIV><DIV class="leftmenu" id="labmenu"><UL><LI><A href="/Team:Edinburgh/Data" class="lab_data">data overview</A></LI><LI><A href="/Team:Edinburgh/Experiments" class="lab_results">experiments</A></LI><LI><A href="/Team:Edinburgh/Parts" class="lab_parts">parts</A></LI><LI><A href="/Team:Edinburgh/BioSandwich" class="lab_biosandwich">BioSandwich</A></LI><LI><A href="/Team:Edinburgh/Lab_Notebook" class="lab_notebook">lab notebook</A></LI></UL></DIV><DIV class="leftmenu" id="teammenu"><UL><LI><A href="/Team:Edinburgh/Conclusions" class="team_conclusions">conclusions</A></LI><LI><A href="/Team:Edinburgh/Team" class="team_people">the team</A></LI><LI><A href="/Team:Edinburgh/Attributions" class="team_attributions">attributions</A></LI><LI><A href="/Team:Edinburgh/Sponsors" class="team_our_sponsors">our sponsors</A></LI></UL></DIV><DIV class="leftmenu" id="practicesmenu"><UL><LI><A href="/Team:Edinburgh/Practices" class="practices_humanpractices">human practices</A></LI><LI><A href="/Team:Edinburgh/Biorefinery" class="practices_biorefinery">biorefinery design</A></LI><LI><A href="/Team:Edinburgh/Biorefinery_Appendices" class="practices_appendices">biorefinery appendices</A></LI><LI><A href="/Team:Edinburgh/Life_Cycle_Analysis" class="practices_lifecycle">life cycle analysis</A></LI><LI><A href="/Team:Edinburgh/Interviews (Overview)" class="practices_overview">interviews: overview</A></LI><LI><A href="/Team:Edinburgh/Interviews" class="practices_interviews">interviews</A></LI><LI><A href="/Team:Edinburgh/Interview_Analysis" class="practices_analysis">interview analysis</A></LI><LI><A href="/Team:Edinburgh/Collaboration" class="practices_synbio">collaboration</A></LI><LI><A href="/Team:Edinburgh/Wiki_Watch" class="practices_wiki_watch">wiki watch</A></LI></UL></DIV><DIV id="hintcontainer"><DIV class="hint" id="def_hint">Hover over words in <I><B>bold italics</B></I> to see explanation.</DIV></DIV></DIV><DIV class="main_body"><P class="h1">Cell Surface Display: Proposals</P><P>The first proposed system our feasibility study will examine, while searching for a way to keep extracellular enzymes close together, is based on displaying proteins at high density on an <SPAN class="hardword" id="ec"><I>E. coli</I></SPAN> outer membrane. This type of display is called &quot;cell surface display&quot;.
</P><P>We will attempt to design such a system for <SPAN class="hardword" id="cellulase">cellulases</SPAN>, and see if we can get it to work.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Outline"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Outline</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Linkers"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Linkers</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Complete_system"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Complete system</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#An_alternative:_protein_chains"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">An alternative: protein chains</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Genetic_instability"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Genetic instability</SPAN></A></LI><LI class="toclevel-1 tocsection-6"><A href="#Proof_of_concept:_YFP"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Proof of concept: YFP</SPAN></A></LI><LI class="toclevel-1 tocsection-7"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-1 tocsection-8"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Outline">Outline</SPAN></H2><P>In order to get a normal enzyme displayed on the <I>E. coli</I> outer membrane, the enzyme must be fused to a carrier protein; that is, one which is naturally transported to the outer membrane.
</P><P><A href="https://2009.igem.org/Team:Berkeley_Wetlab/Assay_Protocols" class="external text" rel="nofollow">Berkeley 2009</A> tried several different carrier proteins with several different passenger enzymes, and had success in many areas. However, when they tried attaching cellulases, they <A href="https://2009.igem.org/Team:Berkeley_Wetlab/Passenger:_Cellulases" class="external text" rel="nofollow">weren't so successful</A> - of the two quantified cellulases, one worked just as well without the carrier (Cel5b) and the other didn't work (Cel9a, as compared to negative control).
</P><P>We will try a different carrier. The <SPAN class="hardword" id="biobrick">BioBrick</SPAN><A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A> made by <A href="https://2009.igem.org/Team:UC_Davis" class="external text" rel="nofollow">UC Davis 2009</A> is a synthetic, codon-optimised sequence, based on <A href="http://www.ncbi.nlm.nih.gov/nuccore/AF013159" class="external text" rel="nofollow">GenBank AF013159</A> and coding for the first 211 and last 97 amino acids of <SPAN class="hardword" id="inp">Ice Nucleation Protein</SPAN> (INP, normally coded by the <I>inaK</I> gene) from the bacterium <SPAN class="hardword" id="ps">Pseudomonas syringae</SPAN>. It seems promising as a carrier of enzymes. Fusions are carried out at the INP C terminal.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Three strategies for INP-based cell display. After <A href="http://www.sciencedirect.com/science/article/pii/S016777991000199X" class="external text" rel="nofollow">Van Bloois <I>et al</I> (2011)</A></DIV></DIV></DIV></DIV><P><A href="http://www.sciencedirect.com/science/article/pii/S016777991000199X" class="external text" rel="nofollow">Van Bloois <I>et al</I> (2011)</A> speak highly of INP, and claim that it can be displayed at a copy number of around 100,000 copies per cell without affecting viability.
</P><P>INP has major domains at its N and C terminals, as well as a number of internal repeating domains. There seem to be three strategies for using INP (see figure):
</P><UL><LI> Use the entire INP protein; fuse at its C terminal
</LI><LI> Delete the INP internal domains; fuse at its C terminal
</LI><LI> Delete all of INP except the N domain; fuse at the new C terminal
</LI></UL><P><A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A> should be suitable for the 2nd strategy.
</P><H3><SPAN class="mw-headline" id="Linkers">Linkers</SPAN></H3><P>It is probably desirable to create <SPAN class="hardword" id="linker">linkers</SPAN> between the carrier and the protein of interest, to give the proteins space to fold. The new assembly protocol that we are investigating â<A href="/Team:Edinburgh/BioSandwich" title="Team:Edinburgh/BioSandwich">BioSandwich</A>â should be ideal for this.
</P><H3><SPAN class="mw-headline" id="Complete_system">Complete system</SPAN></H3><P>The complete 3 cellulase system could contain a promoter, driving expression of three coding fusions:
</P><UL><LI> INP -- (Linker) -- endoglucanase (e.g. <A href="http://partsregistry.org/Part:BBa_K392006" class="external text" rel="nofollow">BBa_K392006</A>)
</LI><LI> INP -- (Linker) -- exoglucanase (e.g. <A href="http://partsregistry.org/Part:BBa_K392007" class="external text" rel="nofollow">BBa_K392007</A>)
</LI><LI> INP -- (Linker) -- Î²-glucosidase (e.g. <A href="http://partsregistry.org/Part:BBa_K392008" class="external text" rel="nofollow">BBa_K392008</A>)
</LI></UL><H3><SPAN class="mw-headline" id="An_alternative:_protein_chains">An alternative: protein chains</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">The protein chain idea: a long fusion protein is created with INP fused to (say) 3 enzymes in a row...</DIV></DIV></DIV></DIV><P>Instead of making three different fusions, it might be possible to make one fusion that had all three cellulase enzymes linked together; we call this &quot;beads on a string&quot;. As it happens, the exoglucanase (Cex) and the endoglucanase (CenA) both have a cellulose-binding module (CBM), but they are at different ends of the sequence. So here's the plan:
</P><UL><LI> Create a fusion of:
</LI><LI><B>Exoglucanase</B> (catalytic domain) -- <B>CBM</B> -- <B>Endoglucanase</B> (catalytic domain)
</LI><LI> This can be done by homology or by introducing an NcoI site into both Exo- and Endoglucanase at the appropriate locations, then ligating and doing fusion PCR.
</LI></UL><LI> We can then use KpnI in a similar way to attach a <B>Î²-glucosidase</B> at either end.
</LI><LI> Then attach INP.
</LI><H2><SPAN class="mw-headline" id="Genetic_instability">Genetic instability</SPAN></H2><P>In order to display several different proteins on one bacterium using the first strategy, it will be necessary to have several copies of the INP gene fused to different enzymes. The presence of repeated sequences on a plasmid can lead to genetic instability.
</P><P>This will not be a problem in the JM109 lab strain, which lacks an important <SPAN class="hardword" id="recombination">recombination</SPAN> enzyme. As for the use of this technology in industry, it will be possible to overcome this problem simply by synthesising coding sequences with as many altered (but <SPAN class="hardword" id="synonymouscodon">synonymous</SPAN>) codons as possible. We have written a software tool for designing such sequences... see the <A href="/Team:Edinburgh/Genetic_instability" title="Team:Edinburgh/Genetic instability" class="mw-redirect">genetic instability</A> page.
</P><H2><SPAN class="mw-headline" id="Proof_of_concept:_YFP">Proof of concept: YFP</SPAN></H2><P>As far as we know, nobody has used <A href="http://partsregistry.org/Part:BBa_K265008" class="external text" rel="nofollow">BBa_K265008</A> for cell display. We could prove that it works by simply displaying the Yellow Fluorescent Protein on INP. Indeed, something similar was achieved by <A href="http://www.postech.edu/~hjcha/INP-N-GFP-OPH.pdf" class="external text" rel="nofollow">Li <I>et al</I> (2004)</A> and <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2009.01724.x/abstract" class="external text" rel="nofollow">Li <I>et al</I> (2009)</A> for a different version of the gene.
</P><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><P>Please see the team's <A href="/Team:Edinburgh/Data" title="Team:Edinburgh/Data"> Data Page</A> for information about how far we got with this project.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI> Li L, Kang DG, Cha HJ (2004) <A href="http://www.postech.edu/~hjcha/INP-N-GFP-OPH.pdf" class="external text" rel="nofollow">Functional display of foreign protein on surface of <I>Escherichia coli</I> using N-terminal domain of Ice Nucleation Protein</A>. <I>Biotechnology and Bioengineering</I><B>85</B>(2): 214-221 (doi: 10.1002/bit.10892).
</LI></UL><UL><LI> Li Q, Yu Z, Shao X, He J, Li L (2009) <A href="http://onlinelibrary.wiley.com/doi/10.1111/j.1574-6968.2009.01724.x/abstract" class="external text" rel="nofollow">Improved phosphate biosorption by bacterial surface display of phosphate-binding protein utilizing ice nucleation protein</A>. <I>FEMS Microbiology Letters</I><B>299</B>(1): 44-52 (doi: 10.1111/j.1574-6968.2009.01724.x).
</LI></UL><UL><LI> Van Bloois E, Winter RT, Kolmar H, Fraaije MW (2011) <A href="http://www.sciencedirect.com/science/article/pii/S016777991000199X" class="external text" rel="nofollow">Decorating microbes: surface display of proteins on <I>Escherichia coli</I></A>. <I>Trends in Biotechnology</I><B>29</B>(2): 79-86 (doi: 10.1016/j.tibtech.2010.11.003).
</LI></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Edinburgh/Cell_Display">http://2011.igem.org/Team:Edinburgh/Cell_Display</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Edinburgh/Cell_Display" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Edinburgh/Cell_Display" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Edinburgh/Cell_Display&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Edinburgh/Cell_Display&amp;oldid=255427" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>