"
Page
Discussion
View source
History
teams
Log in
 
Team:Edinburgh/Assembly
From 2011.igem.org
(Redirected from Team:Edinburgh/AssemblyContent )
outline
in silico
in the lab
in society
postscript
abstract
safety
cell surface display
phage display
achievements
modelling
cellulases (C)
cellulases (Kappa)
cellulases (MATLAB)
model comparison
genetic instability
efficiency
artificial selection
phage replication
data overview
experiments
parts
BioSandwich
lab notebook
conclusions
the team
attributions
our sponsors
human practices
biorefinery design
biorefinery appendices
life cycle analysis
interviews: overview
interviews
interview analysis
collaboration
wiki watch
Hover over words in bold italics to see explanation.
Primers and Oligos
For the sake of posterity, a list of primers we made, or thought about, is kept here. This might conceivably be of use to teams in the future (though the odds are not high...)
Contents
1 Notes
2 Primers actually made
2.1 malS (amylase)
2.2 bglX (B-glucosidase)
2.3 pVIII leader sequence
2.4 pVIII mature protein
2.5 BBa_K118023 / BBa_K392006 (Endoglucanase)
2.6 BBa_K118022 / BBa_K392007 (Exoglucanase)
2.7 BBa_K392008 (B-glucosidase)
2.8 Xylose isomerase
2.9 BBa_K265008 (Ice Nucleation Protein)
2.10 crtE
2.11 crtB
2.12 crtI
3 Gibson assembly of pG8SAET with malS
4 Primers in planning (inaZ, 3603 b.p.)
5 Primers in planning (amylase, mature peptide only)
6 Primers in planning (bglX)
6.1 Mature peptide only
7 Primers in planning (biorefinery)
7.1 Aldose reductase
8 Primers in planning (GFP: green fluorescent protein)
9 Primers in planning (zippers)
9.1 BBa_K415124 (GR1)
9.2 BBa_K415125 (GR2)
10 BioSandwich spoligos actually made
10.1 "Start Spacer" (also has T7 promoter)
10.2 Stop - RBS - Start (1)
10.3 Stop - RBS - Start (2)
10.4 Stop - Strongest RBS
10.5 Stop - Weak RBS
10.6 PT linker (10 aa)
10.7 Serine-rich linker (12 aa)
10.8 Another PT linker (11 aa)
11 BioSandwich spoligos in planning (ribosome binding sites)
11.1 Stop - Weakest RBS
12 BioSandwich spoligos in planning (linkers)
12.1 BBa_K105012 linker (10 aa)
12.2 BBa_K133132 linker with added glycines (10 aa)
12.3 Glycine-rich linker (12 aa)
12.4 E-tag (13 aa)
Notes
The setup for our two systems are as follows:
Phage Display : Promoter-RBS-Leader-Enzyme-pVIII
Cell Surface Display : Promoter-RBS-INP-Enzyme
Phage/cell display passenger enzymes will need start codons so they can be used as controls for assays, where we ask: do they work without the carrier?
When part of a display system, the different products will either be fused only at their 5' end (cell display), or at both ends (phage display).
Some advice that Chris, our supervisor, gave us on primer design:
Check the annealing temperature of the portion without the non-complementary tail (as this is the only part that will anneal during the first round).
Check preexisting BioBricks for BglII sites.
We will have RFC10 reverse primers for the cellulases.
It may not be necessary to have two different forward primers with and without start codon, since the start codon will just be interpreted as a methionine.
It may not be necessary to have reverse primers with and without stop codons, as the stop codon can be supplied by the spacer.
Primers actually made
malS (amylase)
malS ( sequence ) is an amylase gene from E. coli. It seems to be periplasmic, and for whatever reason does not cause noticable starch in wildtype E. coli degradation, whether because of its location or its expression levels.
Using this as a test system would involve comparing the starch-degrading activity of the gene by itself, as compared to the relevant fusions.
The actual primers ordered are as follows:
EcMalSf1: ACT AGATCT gaa atc gca gca ata agg act c
Forward; adds BglII site; starts upstream of gene to catch RBS ( see here ).
Complementary Tm = 63.7 C.
EcMalSf2: ACT AGATCT atg aaa ctc gcc gcc tgt ttt c
Forward; adds BglII site; starts at start codon.
Complementary Tm = 70.1 C.
EcMalSr1: ACT ACTAGT A TTA tta ctg ttg ccc tgc cca g
Reverse; adds SpeI site; RFC10 compliant; has double stop codon.
Complementary Tm = 66.2 C.
EcMalSr2: ACT ACTAGT ACC ctg ttg ccc tgc cca gac g
Reverse; adds SpeI site; RFC23 and BioSandwich compliant; no stop codons, adds a glycine.
Complementary Tm = 71.6 C.
bglX (B-glucosidase)
bglX ( sequence, but gene is on reverse strand ) is a periplasmic Β-glucosidase from E. coli. Again, it seems to not be expressed very strongly.
Note presence of PstI site in sequence; we inserted this into vector <partinfo>BBa_K523000</partinfo> via BglII and SpeI. It was then be necessary to use MABEL to remove the PstI site.
The actual primers ordered are as follows:
EcBglXf1: ACT AGATCT gcc acg tcg ggc aac
Forward; adds BglII site; starts upstream of gene to catch RBS.
Complementary Tm = 65.6 C.
EcBglXf2: ACT AGATCT atg aaa tgg cta tgt tca gta gg
Forward; adds BglII site; starts at start codon.
Complementary Tm = 60.8 C.
EcBglXr1: ACT ACTAGT A TTA tta cag caa ctc aaa ctc gcc
Reverse; RFC10 compliant; has double stop codon.
Complementary Tm = 64.2 C.
EcBglXr2: ACT ACTAGT ACC cag caa ctc aaa ctc gcc ttt c
Reverse; adds SpeI site; RFC23 and BioSandwich compliant; no stop codons, adds a glycine.
Complementary Tm = 67.0 C.
MABEL primers:
EcbglXmutf: a cca gca ctg gcg gat g
Forward; first base is the change.
Tm = 66.4 C.
EcbglXmutr: tg cag ggc cag act cac
Reverse; perfect match to sequence.
Tm = 64.1 C.
pVIII leader sequence
The pVIII sequence is here .
This part is intended to have stuff fused at its 3' (C-terminal) end.
p8f_cfus: ACT AGATCT atg aaa aag tct tta gtc c
Forward, adds BglII site, start codon present.
Complementary Tm = 49.7 C.
p8r_cfus: AAA ACTAGT ACC agc gaa aga cag cat cgg
Reverse, adds a glycine, adds SpeI.
Complementary Tm = 64.2 C.
pVIII mature protein
The pVIII sequence is here .
This part is intended to have stuff fused at its 5' (N-terminal) end.
p8f_nfus: ACT AGATCT gct gag ggt gac gat ccc gc
Forward, adds BglII site.
Complementary Tm = 73.6 C.
p8r_nfus: AAA ACTAGT A tca gct tgc ttt cga ggt g
Reverse, has stop codon, adds standard RFC10 suffix.
Complementary Tm = 64.4 C.
BBa_K118023 / BBa_K392006 (Endoglucanase)
This part contains a BglII site, so that will first have to be dealt with, using MABEL:
cenAmutf1: atc tcg cag cgg ctg gg
Forward, perfect correspondance to sequence
Tm = 70.3 C.
cenAmutr1: ttg ctg gcc gta cgc ctt g
Reverse, replaces CAG (glutamine) with CAA (glutamine).
Tm = 71.5 C.
We can then PCR the part itself. SignalP predicts a cleavage site between amino acids 31 and 32. The following primers fix that problem:
cenAf_nosig: ACT AGATCT ATG GCA CCA GGA tgc cgc gtc gac tac gcc gtc [Warning: strong secondary structure]
Forward, adds BglII site, adds start codon, starts at a.a. 32. but changes first three codons to avoid very strong primer secondary structure.
Complementary Tm = 78.4 C.
cenAr_nostop: AAA ACTAGT ACC cca cct ggc gtt gcg cg
Reverse, no stop codon; adds a glycine, adds SpeI.
Complementary Tm = 75.1 C.
BBa_K118022 / BBa_K392007 (Exoglucanase)
The forward primer deletes the signal peptide.
cexf_nosig: ACT AGATCT ATG gcg acc acg ctc aag gag gc
Forward, adds BglII site, adds start codon, starts at a.a. 43.
cexr_nostop: AAA ACTAGT AGA gcc gac cgt gca ggg cgt gc
Reverse, no stop codon; adds a serine, adds SpeI.
(I was unable to add the usual glycine due to very strong secondary structure arising.)
BBa_K392008 (B-glucosidase)
The forward primer is slightly misnamed as there is no signal peptide predicted for this; the primer just starts at the CDS start.
Cfbgluf_nosig: ACT AGATCT ATG ggc gac cgg ttc cag cag gc
Forward, adds BglII site. Start codon.
Complementary Tm = 77.2 C.
cfbglur_nostop: AAA ACTAGT ACC ggg ctg gta ggt cgc ggc g
Reverse, no stop codon; adds a glycine, adds SpeI.
Complementary Tm = 76.2 C.
We discovered that the 2nd ATG in the K392008 sequence is probably the true start codon. The following is an alternative forward primer that starts there:
Cfbgluf2: ACT AGATCT atg acc acc acg cgc ccc tc
Forward, adds BglII site. Start codon.
Complementary Tm = 75.8 C.
Xylose isomerase
From the xylA gene of E. coli. Designed for fusion to INP or expression on its own. We probably won't try for phage.
EcxylAf1: ACT AGATCT atg caa gcc tat ttt gac c
Forward, adds BglII site, has start codon.
Complementary Tm = 59.2 C.
EcxylAr1: AAA ACTAGT A TTA tta ttt gtc gaa cag ata atg g
Reverse, adds 2nd stop codon, adds standard suffix.
Complementary Tm = 58.1 C.
BBa_K265008 (Ice Nucleation Protein)
Continuing on the assumption of BioSandwich assembly. Ice Nucleation Protein will be present at the start of the coding fusion. A promoter and RBS will have to be added upstream.
This part will have stuff fused at its 3' (C-terminal) end.
ACT AGATCT atg acc ctg gat aaa gcg c
Forward, adds BglII site. Start codon is present.
Complementary Tm = 63.9 C.
AAA ACTAGT ACC ttt cac ttc gat cca atc
Reverse, no stop codon; adds a glycine, adds SpeI.
Complementary Tm = 55.9 C.
crtE
See here .
ACT AGATCT atg acg gtc tgc gca aaa aaa c
Forward, adds BglII site, start codon present.
Complementary Tm = 68.9 C.
AAA ACTAGT ACC act gac ggc agc gag ttt ttt g
Reverse, adds SpeI site, adds a glycine, no stop codon.
Complementary Tm = 69.3 C.
crtB
See here .
ACT AGATCT atg aat aat ccg tcg tta ctc aat c
Forward, adds BglII site, start codon present.
Long primer because Tm was low.
Complementary Tm = 62.9 C.
AAA ACTAGT ACC gag cgg gcg ctg cca gag
Reverse, adds SpeI site, adds a glycine, no stop codon.
Complementary Tm = 74.3 C.
crtI
See here .
ACT AGATCT atg aaa cca act acg gta att g
Forward, adds BglII site, start codon present.
Complementary Tm = 59.3 C.
AAA ACTAGT ACC tat cag atc ctc cag cat caa ac
Reverse, adds SpeI site, adds a glycine, no stop codon.
Complementary Tm = 64.1 C.
Gibson assembly of pG8SAET with malS
Make the 2 PCR products, assemble in Gibson way, and hope for the best...
malSF4p8: CA AAT GCT GCG CAA CAC GAT gcc agc tgg act tct ccg gg
malS forward primer
malSR4p8: C TTT TGC GGG ATC GTC ACC CTC ctg ttg ccc tgc cca gac gac
malS reverse primer
vectF4p8: GTC GTC TGG GCA GGG CAA CAG gag ggt gac gat ccc gca aaa g
vector forward primer
vectR4p8: CC CGG AGA AGT CCA GCT GGC atc gtg ttg cgc agc att tg
vector reverse primer
Primers in planning (inaZ, 3603 b.p.)
See here .
ACT AGATCT atg aat ctc gac aag gcg ttg
Forward, adds BglII site, start codon present.
Complementary Tm = 66.2 C.
AAA ACTAGT ACC ctt tac ctc tat cca gtc atc gtc
Reverse, adds SpeI site, adds a glycine, no stop codon.
Complementary Tm = 61.8 C.
Primers in planning (amylase, mature peptide only)
malS is a periplasmic gene which has a signal peptide. In order to fuse it to INP, it may be necessary to isolate the mature protein sequence, which is claimed to be amino acids 18-676; i.e. we would need to delete the first 51 bases in the DNA.
ACT AGATCT gcc agc tgg act tct ccg gg
Forward, adds SglII site.
Complementary Tm = 72.5 C.
Primers in planning (bglX)
Mature peptide only
bglX is a periplasmic gene which has a signal peptide. In order to fuse it to INP, it may be necessary to isolate the mature protein sequence, which SignalP predicts to be amino acids 21 onwards; i.e. we would need to delete the first 60 coding bases in the DNA.
ACT AGATCT gat gat tta ttc ggc aac c
Forward, adds BglII site.
Complementary Tm = 59.4 C.
Primers in planning (biorefinery)
These things may be produced on their own (as a control) or fused to INP. We probably won't bother with fusion to phage? In any case, the unfused versions used as controls will need start codons.
Aldose reductase
From the reverse complement of this sequence which we can get from DSM 4680 . Might be problems with low GC content of gene. Some alternative strains that might be used instead:
Lactobacillus acidophilus NCFM
Providencia alcalifaciens DSM 30120 (but BLAST says this is probably a 2,5-diketo-D-gluconate reductase)
Lactobacillus acidophilus ATCC 4796 == LMG 11470
Primers for DSM 4680:
ACT AGATCT gtg aat aat ctt aaa gac
Forward, adds BglII site, has start codon (gtg).
Complementary Tm = 42.3 C.
AAA ACTAGT A TTA tta aat ttt ttt gtt aaa ta
Reverse, adds 2nd stop codon, adds standard suffix.
Complementary Tm = 46.6 C.
Primers in planning (GFP: green fluorescent protein)
This can be PCR'd from <partinfo>BBa_E0040</partinfo> which is has been in the distribution forever. I assume we'll have some floating around the lab somewhere?
ACT AGATCT atg cgt aaa gga gaa gaa ctt ttc
Forward, adds BglII site, has start codon.
Complementary region Tm = 62.6 C.
AAA ACTAGT ACC ttt gta tag ttc atc cat gcc atg
Reverse, no stop codon; adds a glycine, adds SpeI.
Complementary region Tm = 64.4 C.
Primers in planning (zippers)
The GR1 and GR2 zippers may have applications in phage display. In both cases, stuff will be fused to both N and C terminals (see Team:Edinburgh/Phage_Reactors ) so neither start nor stop codons are required.
BBa_K415124 (GR1)
It ought to be possible to PCR this out of <partinfo>BBa_K415151</partinfo>, which is in the latest DNA distribution. Note though that sequencing is claimed to be inconsistent. The official sequence seems to show that the construct has somehow been replaced with the aminopeptidase PepN gene.
ACT AGATCT gag gag aag tcc cgg ctg
Forward, adds BglII site.
Complementary Tm = 65.1 C.
AAA ACTAGT ACC aca acc tcc tac aga ctg g
Reverse, adds a glycine, adds SpeI.
Complementary Tm = 56.6 C.
BBa_K415125 (GR2)
We can PCR this out of <partinfo>BBa_K415152</partinfo>.
ACT AGATCT aca tcc cgc ctg gag ggc c
Forward, adds BglII site
Complementary Tm = 74.9 C.
AAA ACTAGT ACC gca acc tcc gac gtc ttg c
Reverse, adds a glycine, adds SpeI.
Complementary Tm = 68.3 C.
BioSandwich spoligos actually made
"Start Spacer" (also has T7 promoter)
spacerT7fs: CTAGAG ttaatacgactcactatagggagaggaggtac
spacerT7rt: gtacctcctctccctatagtgagtcgtattaa CT
spacerT7rl: GATC gtacctcctctccctatagtgagtcgtattaa CTCTAG
Stop - RBS - Start (1)
spacer1fs: CTAGC tga tatttcgcgtaaggaaatccatt atg G
spacer1rt: C cat aatggatttccttacgcgaaata tca G
spacer1rl: GATCC cat aatggatttccttacgcgaaatatca GCTAG
Stop - RBS - Start (2)
spacer2fs: CTAGC aacatatca taa cggagtgatcgca atg G
spacer2rt: C cat tgcgatcactccg tta tgatatgtt G
spacer2rl: GATCC cat tgcgatcactccg tta tgatatgtt GCTAG
Stop - Strongest RBS
Explanation: stop + random junk + strongest RBS (<partinfo>BBa_B0034</partinfo>). RBS strength is 100%
spacer3fs: CTAGC tga acgtaaatactcagt aaagaggagaaa G
spacer3rt: C tttctcctcttt actgagtatttacgt tca G
spacer3rl: GATCC tttctcctcttt actgagtatttacgt tca GCTAG
Stop - Weak RBS
Explanation: stop + random junk + weak RBS (<partinfo>BBa_J61100</partinfo>). RBS strength is 2.0% - 4.29%
spacer4fs: CTAGC tag gtcttagtcttcgtc aaagaggggaca G
spacer4rt: C tgtcccctcttt gacgaagactaagac cta G
spacer4rl: GATCC tgtcccctcttt gacgaagactaagac cta GCTAG
PT linker (10 aa)
Sequence of lowercase region below is PTTSPTPTPT.
linker1fs: CTAGC ccg acg acc agc ccc acg ccg acc ccg acg G
linker1rt: C cgt cgg ggt cgg cgt ggg gct ggt cgt cgg G
linker1rl: GATCC cgt cgg ggt cgg cgt ggg gct ggt cgt cgg GCTAG
Serine-rich linker (12 aa)
Back-translated from a.a. 141-152 of Cellvibrio japonicus xylanase A . Sequence of lowercase region below is SSIASSSPSSVA.
linker2fs: CTAGC agc agc att gcg agc agc agc ccg agc agc gtg gcg G
linker2rt: C cgc cac gct gct cgg gct gct gct cgc aat gct gct G
linker2rl: GATCC cgc cac gct gct cgg gct gct gct cgc aat gct gct GCTAG
Another PT linker (11 aa)
SPTPTTPTPTS.
linker3fs: CTAGC tct ccg acc cct acc acg cca act ccg acc agc G
linker3rt: C gct ggt cgg agt tgg cgt ggt agg ggt cgg aga G
linker3rl: GATCC gct ggt cgg agt tgg cgt ggt agg ggt cgg aga GCTAG
BioSandwich spoligos in planning (ribosome binding sites)
Stop - Weakest RBS
Explanation: stop + random junk + weakest RBS (<partinfo>BBa_B0033</partinfo>). RBS strength is 0.35%
FS: CTAGC taa ttctgacctacttaa tcacacaggac G
RT: C gtcctgtgtga ttaagtaggtcagaa tta G
RL: GATCC gtcctgtgtga ttaagtaggtcagaa tta GCTAG
BioSandwich spoligos in planning (linkers)
BBa_K105012 linker (10 aa)
From <partinfo>BBa_K105012</partinfo>. Sequence of lowercase region below is GENLYFQSGG.
FS: CTAGC ggt gaa aat ttg tat ttt caa tct ggt ggt G
RT: C acc acc aga ttg aaa ata caa att ttc acc G
RL: GATCC acc acc aga ttg aaa ata caa att ttc acc GCTAG
BBa_K133132 linker with added glycines (10 aa)
From <partinfo>BBa_K133132</partinfo> with one extra glycine at each end. Sequence of lowercase region below is GSACYCELSG.
FS: CTAGC gga tcc gct tgt tac tgt gag ctt tcc gga G
RT: C tcc gga aag ctc aca gta aca agc gga tcc G
RL: GATCC tcc gga aag ctc aca gta aca agc gga tcc GCTAG
Glycine-rich linker (12 aa)
Truncated version of <partinfo>BBa_K416001</partinfo>. Sequence of lowercase region below is GGGSGGGGSGGG.
FS: CTAGC gga gga ggc tct ggt gga ggc ggt agc gga ggc gga G
RT: C tcc gcc tcc gct acc gcc tcc acc aga gcc tcc tcc G
RL: GATCC tcc gcc tcc gct acc gcc tcc acc aga gcc tcc tcc GCTAG
E-tag (13 aa)
Codes for GAPVPYPDPLEPR which can be detected by anti-E-tag antibodies.
FS: CTAGC ggt gcg ccg gtg ccg tat ccg gac cca ctg gaa ccg cgt G
RT: C acg cgg ttc cag tgg gtc cgg ata cgg cac cgg cgc acc G
RL: GATCC acg cgg ttc cag tgg gtc cgg ata cgg cac cgg cgc acc GCTAG
Retrieved from " http://2011.igem.org/Team:Edinburgh/Assembly "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
