<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Korea_U_Seoul_Project_Biological_Background_content skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Korea U Seoul/Project/Biological Background/content</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="content_wrapper" style=""><DIV class="title"><DIV class="title_">
                 1) KEGG database
                </DIV></DIV><DIV class="content_"><P class="ti2">
			    The program ‘Gil’ is a project based on KEGG (Kyoto Encyclopedia of Genes and Genomes) database (KanehisaM., et al. 2014; and KanehisaM., GotoS., 2000). There are numerous advantages of KEGG (Adriaens M. E., et al. n.d.). Above all, KEGG has a nice interface. In an overview graphic, we can take all reactions of a pathway, and see the enzymes that are involved in the reaction by clicking individual reactions. In addition, there are most information about the gene linked with this enzyme. So we organized and parsed the databases that provide information of compounds, reactions, genes and enzymes. 
            </P><P class="ti2">However, KEGG database is definitely not perfect. First, it doesn't provide thermodynamic data. Additionally, KEGG database shows reactions in a bidirectional way, which lacks the information of thermodynamical feasibility. Also, KEGG is a pure gene database. While each enzyme has a large amount of gene information attached, it lacks a bit in the other departments. Therefore, users easily encounter empty pages that do not have any information. Lastly, every glycan in KEGG database is not connected with its corresponding compound. When glycan IDs are not connected to compound IDs, it is not possible to find a reaction with a glycan.</P></DIV><DIV class="title"><DIV class="title_">
                 2) Thermodynamic information
                </DIV></DIV><DIV class="content_"><P class="ti2">
			   Thermodynamic constraint is one of the most important factors of biological experiments. Therefore, the program ‘Gil’ provides Gibbs energy changes of each metabolic reaction. 
</P><P class="ti2">The program ‘Gil’ obtained thermodynamic data from two separated databases even though they were obtained from the same program, eQuilibrator (Flamholz A., et al. 2012). The first database is composed of Gibbs free energy from standard condition—pH 7.0 and 0.1 M ionic strength. These data are correct, but the size of the database is smaller than the other one due to the lack of wet lab experiments. On the other hand, the second version is made up of values from a calculation. This database has much more values than the first one however, it is hard for researchers to apply this values directly to their experiments since they are not experimentally proved. Thus, additional attention is needed to utilize the second version of data. </P></DIV><DIV class="title"><DIV class="title_">
                 3) Four path-scoring criteria
                </DIV></DIV><DIV class="content_"><H2>① NADH and NADPH</H2><P class="ti2">Like an ATP (adenosine triphosphate) which serves as a convenient and versatile storage in cells, NADH (nicotinamide adenine dinucleotide) and NADPH (nicotinamide adenine dinucleotide phosphate) serve as activated carriers specialized in carrying high-energy electrons and hydrogen atoms. 

            </P><P class="ti2"> The NAD + and NADH molecules are indistinguishable in structure to NADP + and NADPH, respectively, except the fact that the phosphate group is absent from the former. However, this small difference is crucial for their distinctive roles. Although the extra phosphate group on NADPH is located far away from the region involved in electron transfer (see Figure 1-b), it gives the molecule a slight difference in its shape from that of NADH, so NADPH and NADH bind to different sets of enzymes as substrates. Thus, the two types of carriers are used to transfer electrons (or hydride ions) between different sets of molecules. This role separation is crucial for organisms to regulate two sets of electron-transfer reactions independently. An NADPH operates mainly with enzymes that catalyze anabolic reactions, providing the high-energy electrons needed to synthesize energy-rich biological molecules. In contrast, an NADH has a special role as an intermediate in the catabolic system of reactions that generate ATP through the oxidation of compounds (Alberts B., et al. 2002).
</P><P class="ti2">Then, why should we focus on the larger amount of NADH and NADPH as products? </P><P class="ti2">First, in the case of NADPH, it is the key for determining the efficiency of certain biosynthetic pathways. The oxidation during a pentose phosphorylation produces most NADPH which is needed in the biosynthesis of certain metabolites. Therefore, the carbon flow of the pentose phosphorylation path is considered to be the targets of metabolic engineering. For example, microorganisms found in the Corynebacterium genus, having enhanced L-ornithine production, not only activate pentose phosphate pathway but also increase the specific activities of G6PDH and 6PGD, which are enzymes related to the pentose phosphate pathway, by reducing or inactivating intrinsic GDH (glucose dehydrogenase). Consequently, the improved NADPH production makes L-ornithine yield to increase, which may have the possibility of being widely utilized for efficient and economical applications (Cho J., Ko Y. 2012). To summarize, if we find a path with improved NADPH yield, we could get more products in other relevant paths.
</P><P class="ti2">In the same context, if the NADH yield is increased, the efficiency of the path using the NADH will also be enhanced. In addition, cells grown aerobically may possibly use the extra NADH for ATP generation through the electron transport system. According to the research on the availability of intracellular NADH (Berríos-Rivera SJ., Bennett GN, San KY 2002), the higher NADH availability dramatically increased the ethanol to acetate (Et/Ac) ratio by 27-fold. The cause of this amazing increase is proven by an experiment comparing BS1 (pSBF2), which is inactivated by the native FDH (formate dehydrogenase) that regenerates NADH, with GJT001 (pDHK29), the control, under anaerobic conditions.
</P><P class="ti2">For these reasons, our program suggests the paths in the order of the amount of NADPH and NADH production. Users will be able to get efficient and productive paths with the information provided by our program, with less effort.
</P><H2>② ATP</H2><P class="ti2">Another scoring factor which the program ‘Gil’ provides is the ATP quantity. This function is expected to be useful for synthetic biologists to design reaction pathway necessary for their experiments which are designed to have higher efficiency when there is increased ATP level. For instance, according to the research that used Saccharomyces cerevisiae to generate ATP from adenosine in the process of glucose breakdown into ethanol and carbon dioxide (Murata K, et al.. 1981), the ATP-generating reaction enhances the biosynthesis of Escherichia coli which synthesizes glutathione and Brevibacterium ammoniagenes which produces NADP. Therefore, substituting the biological reaction with the ones that have higher ATP yield, designed by the program ‘Gil’, will possibly increase the productivity of both organisms. The following KEGG compound number order is the examples of the reaction pathway: C00031 (glucose), C00394, C00035, C00144, C00044, C04895, C00084, C00469 (ethanol).
</P><H2>③ CO2</H2><P class="ti2">Most chemical compounds with carbon produce a product through a synthetic-dissociation pathway. The decarboxylation, the dissociation of carbon, will lose carbon equivalent and limit the product’s theoretical carbon yield. The Formula 1 show you how to calculate the efficiency of a pathway.</P><P class="ti2">We assume that non-oxidative glycolysis (NOG) as a model mechanism to develop our software program. NOG is one of the complete carbon yield pathways that starts with sugar to acetyl-CoA. It needs conjunction with carbon dioxide or assimilation with other one-carbon (C1) source to produce desirable fuels and chemicals (Bogorad I. W., Lin T. S., Liao J. C. 2013).  </P><P class="ti2">While simulating the NOG pathway, we achieved complete carbon yield. So without losing any of carbons, we are able to obtain desirable results. During the reaction, there are lots of chemicals that result in carbon loss. The program ‘Gil’ calculates the CO2 reduction to calculate the carbon loss. This is due to the fact that CO2 is less likely to be utilized unless it is fixed compared to other chemicals.
</P><DIV class="title"><DIV class="title_">
               References
                </DIV></DIV><DIV class="content_"><P>Alberts, B., A. Johnson, J. Lewis, and et al. Molecular Biology of the Cell. 4th edition. New York: Garland Science, 2002.
</P><P>Berríos-RiveraSJ., BennettGN, SanKY. “Metabolic Engineering of Escherichia coli : Increase of NADH Availability by Overexpressing an NAD+-Dependent Formate Dehydrogenase.” “Metabolic Engineering”, 2002: 217-229.
</P><P>BogoradI. W., LinT. S., LiaoJ. C. “Synthetic non-oxidative glycolysis enables complete carbon conservation.” “Nature”, 2013: 693-697.
</P><P>Cho, J., and Y. Ko. Microorganism found in the corynebacterium genus having enhanced l-ornithine production and method for preparing l-ornithine using same. South Korea Patent WO 2012161522 A2. 5 23, 2012.
</P><P>FlamholzA., NoorE., Bar-EvenA., MiloR. “eQuilibrator—the biochemical thermodynamics calculator.” “Nucleic Acids Res.”, 2012: D770-D775.
</P><P>KanehisaM., GotoS. “KEGG: Kyoto Encyclopedia of Genes and Genomes.” “Nucleic Acids Res”, 2000: 27-30.
</P><P>KanehisaM., GotoS., SatoY., KawashimaM., FurumichiM., TanabeM. “Data, information, knowledge and principle: back to metabolism in KEGG.” “Nucleic Acids Res”, 2014: D199-D205.
</P><P>MurataK, TaniK., KatoJ., ChibataI. “Glycolytic pathway as an ATP generation system and its application to the production of glutathione and NADP.” “Enzyme and Microbial Technology”, 1981: 233-242.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>