<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RHIT_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RHIT/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="menu_wrapper"><DIV class="collapsable_menu_control"> MENU ▤ </DIV><UL id="accordion" class="accordion"><LI class="menu_item"><A href="https://2016.igem.org/Team:RHIT">HOME </A></LI><LI class="menu_item"> TEAM
			</LI><LI><A href=" https://2016.igem.org/Team:RHIT/Team"> Team   </A></LI><LI><A href="https://2016.igem.org/Team:RHIT/Collaborations">★  Collaborations </A></LI></UL><LI class="menu_item"> PROJECT  
			</LI><LI><A href="https://2016.igem.org/Team:RHIT/Description"> ★  Description </A></LI><LI><A href="https://2016.igem.org/Team:RHIT/Experiments"> Experiments </A></LI><LI><A href="https://2016.igem.org/Team:RHIT/Demonstrate"> ★ Demonstrate </A></LI><LI><A href="https://2016.igem.org/Team:RHIT/Results"> Results </A></LI><LI><A href="https://2016.igem.org/Team:RHIT/Notebook"> Notebook </A></LI><LI class="menu_item"> PARTS  
			</LI><LI><A href="https://2016.igem.org/Team:RHIT/Basic_Part"> ★ Basic Parts </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:RHIT/Safety"> SAFETY </A></LI><LI class="menu_item"><A href="https://2016.igem.org/Team:RHIT/Attributions">★  ATTRIBUTIONS </A></LI><LI class="menu_item"> HUMAN PRACTICES 
			</LI><LI><A href="https://2016.igem.org/Team:RHIT/HP/Silver">★ Silver </A></LI></DIV><DIV class="content_wrapper"><P> In this project, we successfully produced two yeast expression vectors with BioBrick prefix and suffix. Using the vectors we produced, we managed to verify function of mitochondrial localization signal (mls) by testing the hybrid construct mls-yeGFP under a fluorescence microscope. Expression of mls-yeGFP causes mitochondria targeted fluorescence regardless of whether the strain has functional mitochondrial ribosomes and a functional electron transport chain.  Unfortunately, experimental results indicated that expressing the mitoribosomal protein mRPS12 in a haploid mRPS12 knockout strain did not restore aerobic respiration and their ability to grow on non-fermentable carbon sources. We replicated this experiment several times while the same results were obtained. Expressing mRPS12 in wild type S. <I>cerevisiae</I> did not result any adverse effect, therefore we can conclude that its failure to restore mitochondria function in the knock out strain did not result from overexpression of the gene behind the GPD promoter.  Therefore we hypothesized that the knock out strain we used has lost its mitochondrial genome which explains why restoring mitoribosomal function does not restore the strains ability to process glycerol. We continued our investigation using mitochondria targeting red fluorescent stain, Mito ID Red.  Microscopy illustrated that the mRPS12 TU knock out strain still had mitochondria, and that the preprotein mitochondria targeting peptide from mls-yeGFP still function in that strain.  We purpose that the failure of mRPS12 TU complementation results from the loss of background sequence before and after the TU that could impact properties of the mRPS12 protein such as its half life.  To further investigate this we will clone mRPS12 TU and its surrounding genomic DNA into a vector and try to use it to complement the mRPS12 TU knock out phenotype.</P><P>Once the function of mRPS12 is verified, we would like to connect a light-switchable system [1] to the mRPS12 gene in order to achieve remote control. Essentially the system utilizes protein-protein interaction between phytochrome B (PhyB) and phytochrome interacting factor 3 (PIF3) to control gene expression. By attaching a chromophore phycocyanobilin (PCB) extracted from cyanobacterium to PhyB, exposure to red light would change conformation of PhyB from ground state Pr to active state Pfr. Such conformation change would alter interaction between PhyB and PIF3, resulting in turning the following gene on or off. This system is originally proposed in 2002  and it was based on Gal4 promoter [1], while an iGEM team from Munich modified it by replacing Gal4 promoter with LexA [2]. The advantage of using LexA is that, since LexA comes from prokaryotes, it minimizes the interference between endogenous yeast metabolism [2]. 
</P><P>References:</P><P>[1] S. Shimizu-Sato, E. Huq, J. Tepperman and P. Quail, &quot;A light-switchable gene promoter system&quot;, Nature Biotechnology, vol. 20, no. 10, pp. 1041-1044, 2002.</P><P>[2] <A rel="nofollow" class="external free" href="http://parts.igem.org/Part:BBa_K801043">http://parts.igem.org/Part:BBa_K801043</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>