<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NTHU_Taiwan_Collaborations skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NTHU Taiwan/Collaborations</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU  
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:NTHU_Taiwan"><DIV class="menu_button menu_right_margin">
				HOME
			</DIV></A><DIV class="submenu_wrapper"><BUTTON class="menu_button">TEAM</BUTTON><DIV class="submenu_button"><A href="https://2017.igem.org/Team:NTHU_Taiwan/Team"><DIV id="Team_page">
						Team 
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Attributions"><DIV id="Attributions_page">
						Attributions
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Collaborations"><DIV id="Collaborations_page">
						Collaborations
					</DIV></A></DIV></DIV><DIV class="submenu_wrapper"><BUTTON class="menu_button">PROJECT</BUTTON><DIV class="submenu_button"><A href="https://2017.igem.org/Team:NTHU_Taiwan/Description"><DIV id="Description_page">
						Description 
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Design"><DIV id="Design_page">
						Design
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Contribution"><DIV id="Contribution_page">
						Contribution
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Model"><DIV id="Model_page">
						Model
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Results"><DIV id="Results_page">
						Results
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Demonstrate"><DIV id="Demonstrate_page">
						Demonstrate
					</DIV></A></DIV></DIV><DIV class="submenu_wrapper"><BUTTON class="menu_button">WET LAB</BUTTON><DIV class="submenu_button"><A href="https://2017.igem.org/Team:NTHU_Taiwan/Experiments"><DIV id="Experiments_page">
						Experiments
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Protocol"><DIV id="Protocol_page">
						Protocol
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Notebook"><DIV id="Notebook_page">
						Notebook
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/InterLab"><DIV id="InterLab_page">
						InterLab
					</DIV></A></DIV></DIV><DIV class="submenu_wrapper"><BUTTON class="menu_button">PARTS</BUTTON><DIV class="submenu_button"><A href="https://2017.igem.org/Team:NTHU_Taiwan/Basic_Part"><DIV id="Basic_Part_page">
						Basic Parts
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Composite_Part"><DIV id="Composite_Part_page">
						Composite Parts
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Part_Collection"><DIV id="Part_Collection_page">
						Part Collection
					</DIV></A></DIV></DIV><A href="https://2017.igem.org/Team:NTHU_Taiwan/Safety"><DIV class="menu_button">
				SAFETY
			</DIV></A><DIV class="submenu_wrapper"><BUTTON class="menu_button">HUMAN PRACTICES</BUTTON><DIV class="submenu_button"><A href="https://2017.igem.org/Team:NTHU_Taiwan/HP/Silver"><DIV id="Silver_page">
						Silver HP
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/HP/Gold_Integrated"><DIV id="Gold_Integrated_page">
						Integrated and Gold
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Engagement"><DIV id="Engagement_page">
						Public Engagement 
					</DIV></A></DIV></DIV><DIV class="submenu_wrapper"><BUTTON class="menu_button">AWARDS</BUTTON><DIV class="submenu_button"><A href="https://2017.igem.org/Team:NTHU_Taiwan/Applied_Design"><DIV id="Applied_Design_page">
						Applied Design
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/HP/Gold_Integrated"><DIV id="Entrepreneurship_page">
						Integrated Human Practices
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Hardware"><DIV id="Hardware_page">
						Hardware
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Model"><DIV id="Model_page">
						Model
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Software"><DIV id="Software_page">
						Software
					</DIV></A><A href="https://2017.igem.org/Team:NTHU_Taiwan/Criteria"><DIV id="Criteria_page">
						Criteria
					</DIV></A></DIV></DIV><A href="https://igem.org/2017_Judging_Form?team=NTHU_Taiwan"><DIV class="menu_button">
				JUDGING FORM
			</DIV></A></DIV></DIV><DIV class="igem_2017_content_wrapper"><DIV style="text-align: center"><H1 style="color:#DF6A6A">Team Collaborations
		
		</H1></DIV><DIV class="content"><P><FONT><B>1. iGEM WLC: Wisconsin Lutheran College ( USA )</B></FONT></P><P><FONT>
-independent social investigation

-detector integration: E.coli detecting
</FONT></P><P><FONT><B>2. iGEM SECA NZ: The University of Auckland ( New Zealand )</B></FONT></P><P><FONT>
-independent social investigation

-device blueprint offer: temperature sensing system

-photobioreactor offer 
</FONT></P><P><FONT><B>3. iGEM NAU: Nanjing Agricultural University ( China )</B></FONT></P><P><FONT>
-troubleshooting: protocols

-technique exchange: data collection, model designing, and hardware modeling

-Data exchange: Aquaculture questionnaire results

-detector integration: nitrate detector integration
</FONT></P><P><FONT><B>4. iGEM NCKU: National Cheng Kung University ( Taiwan )</B></FONT></P><P><FONT>
-data exchange: Aquaculture questionnaire results

-detector integration: nitrate detector intergration
</FONT></P><P><FONT><B>5. iGEM TU Delft: Technische Universiteit Delft ( Netherlands )</B></FONT></P><P><FONT>
-shield protein offer: protecting protein polypeptide loop
</FONT></P><H1 style="color:#6c5070"><CENTER>iGEM NTHU x iGEM WLC</CENTER></H1><H2>．Collaborate college :</H2><P>Wisconsin Lutheran College ( USA )</P><H2>．Collaborate Division : </H2><P>1.Human Practice</P><P>2.Dry Lab</P><H2>．Collaborate item : </H2><P>1.Independent social investigation</P><P>2.Detector integration plan </P><H2>．Collaborate content :  </H2><P><A href="https://2017.igem.org/Team:WLC-Milwaukee">iGEM WLC</A> aims to detect <I><I>E. coli</I></I> in water while our team’s goal is to detect the EDCs in water. Because we both focus on water quality, we’ve planned to achieve multiple detection integrations that have the ability to detect various pollution, like a Swiss Knife with many functions! However, it would be very struggling for them to build up a detecting equipment and a 3D model because of a lack of an engineering expert in their team. Fortunately, our team has suggested some software resources that may be useful for them, which have given them an orientation to demonstrate their brilliant work. For the 3D modelling, we had gone through the software composition and made our own model. Based on experience, we could also associate them to learn the original model construction method.</P><BLOCKQUOTE><BLOCKQUOTE><BLOCKQUOTE>[ Figure. 1.1-1 ] The blueprint of our IoT system. This branch chart includes every important element that we integrate with. The left ions are the sensor input ends where is the very place to integrate the <I>E. coli</I> sensor.</BLOCKQUOTE></BLOCKQUOTE></BLOCKQUOTE><P>NTHU iGEM team’s detection method for EDCs in water is a different aspect of addressing water contamination. In the collaboration between the Taiwan and American team, we were able to help each other survey American’s opinion and knowledge on EDCs as well as Taiwanese knowledge on bacteriophages and their opinion in regard to <I>E. coli</I> contamination. With their help on designing our 3D model system, we were able to use one of the software programs that they suggested and designed a 3D model of our water test kit. We improved each other’s project based on these collaboration experiences, and it was a pleasure to work with them! </P><P><CENTER><FONT size="2">[ Figure. 1.1-2 ] 3D model designed by iGEM WLC with help and suggestion of iGEM NTHU.</FONT></CENTER></P><P><CENTER><FONT size="2">[ Figure. 1.1-3 ] Skype meeting with WLC, we have discussed how to build up a 3D model for their project.</FONT></CENTER></P><P><CENTER><FONT size="2">[ Figure. 1.1-4 ] Second-time meetup with iGEM WLC when we talk about further integration plan.</FONT></CENTER></P><H1 style="color:#6c5070"><CENTER>iGEM NTHU x iGEM SECA NZ</CENTER></H1><H2>．Collaborate college :</H2><P>The University of Auckland ( New Zealand )</P><H2>．Collaborate Division : </H2><P>1.Human Practice </P><P>2.Dry Lab</P><H2>．NTHU Collaborate Memo :</H2><P>In the project of <A href="https://2017.igem.org/Team:SECA_NZ">iGEM UoA</A>, they transferred antifreeze protein gene to crops and combined the ability to regulate gene expression. They are aiming for the gene to only be expressed at high levels when the plant is at a cold enough temperature. We thought that our temperature sensing system would be ideal for them to set up the initial temperature database. Because the data would be collected immediately and accurately, they could derive continuous data from different areas and decide whether it is necessary to plant antifreeze crops in every farmland or valley. So far we have provided our design detail including hardware and software to iGEM UoA. We hope it would show a better value for both of us.</P><H2>． SECA NZ Collaborate Memo :</H2><P>This year we, SECA NZ, collaborated with NTHU iGEM in the exchange of international viewpoints and software. NTHU is developing an ingenious system to monitor and adjust the level of contaminants in irrigation drains using copper filters and a Watergates. The filter cleans the water, and a complex set of sensors monitors the output. If the level of contaminants exceeds the capacity of the filter, the water gate will reduce the flow to a manageable level. They are also investigating the water quality in aquaculture. New Zealand’s economy is largely based on agriculture and fishery industries, and so as a team, we participated in a questionnaire to assess our opinions on aquaculture and water quality in the agriculture industry. This provided NTHU with both an international and consumer perspective on the issue.</P><P>Our own project focuses on the development of crop plants capable of surviving snap frosts. NTHU contributed to our project by graciously providing software and specifications on how to build a temperature sensor which would allow us to monitor the temperatures of cropping land.</P><P><CENTER><FONT size="2">[ Figure. 1.3-1 ] Meetup with iGEM UoA. The best part of this meetup is to exchange protocol and troubleshooting. </FONT></CENTER></P><H1 style="color:#6c5070"><CENTER>iGEM NTHU x iGEM NAU</CENTER></H1><H2>．Collaborate college :</H2><P>Nanjing Agricultural University ( China )</P><H2>．Collaborate Division :</H2><P>1.Wet Lab</P><H2>．Collaborate item :</H2><P>1.Data check</P><P>2.Technique exchange </P><P>3.Design discussion</P><H2>．Collaborate content : </H2><P>We had been collaborating with <A href="https://2017.igem.org/Team:NAU-CHINA">iGEM NAU</A> several times. Each time, we focused on sharing experiment schedules and techniques to solve problems in Wet Lab and Dry Lab. On account of their suggestion, we changed our protocols which made a great breakthrough on the performance of enzymes and solved the problem we made in gene transfer. In Dry Lab, we offered our experimental experience in hardware design. We not only gave them a deep look into measurement analysis and practical application, but also gave our suggestion on data collection method, model designing, and hardware modeling. On the other hand, we both exchange our plasmid for further check and study. Thanks to the profession in molecular biology from iGEM NAU and our outstanding talent in the device, we integrated both our advantages and made a significant improvement in both our project. </P><P><CENTER><FONT size="2">[ Figure. 1.3-1 ] Meetup with iGEM NAU. The best part of this meetup is to exchange protocol and troubleshooting.  </FONT></CENTER></P><H1 style="color:#6c5070"><CENTER>iGEM NTHU x iGEM NCKU</CENTER></H1><H2>．Collaborate college :</H2><P>National Cheng Kung University ( Taiwan )</P><H2>．Collaborate Division :</H2><P>1.Wet Lab</P><P>2.Dry lab</P><H2>．Collaborate item : </H2><P>1.Data check</P><P>2.Troubleshooting</P><P>3.Technique exchange</P><P>4.Detector integration plan</P><H2>．NTHU Collaborate Memo : </H2><P>In the beginning, we were interested in the <A href="https://2016.igem.org/Team:NCKU_Tainan">2016 iGEM NCKU project</A> which is about the signal detection on urine samples by single frequency laser reflection. After discussing the whole progress and detail, it inspires us to engage in the UV excitation fluorescence to detect our sample. In addition to mutual inspection and troubleshooting with both projects, we also presented our detector design to iGEM NCKU and shared different experimental strategies. During the last meeting, we learned that we had an identical commercial value in aquaculture. As a result, we offered our investigation results of Aquaculture questionnaire to them to strengthen their data integrity by the information of market demand and consumer’s ideas. Furthermore, we looked for other teams who were also participating in pollution detection and trying to combine all the detectors in our demo-board. Nitrate detector from iGEM NCKU offers us a great opportunity to strengthen this idea. On the other hand, they also developed a special boat combining several extra functions other than nitrate sensor, like GPS, radio, motor etc., which is also an independent and integral equipment ready for adding other sensors like our IoT system. Thus, we are also happy to offer them our blueprint of EDCs sensor which might also offer them a chance to broaden their sensing target! </P><P><CENTER><FONT size="2">[ Figure. 1.4-1 ] Cross section of EDCs sensor based on fluorescence. </FONT></CENTER></P><H2>． NCKU Collaborate Memo : </H2><P>In this September, we received the invitation to have a collaboration with NTHU. NTHU detected environmentally hazardous substances while our project is focusing on the sensing of nitrate concentration. They try to give a broader aspect. However, we try to give a more detail sensing system. Thus, we had a profound discussion of both teams’ projects. Moreover, we also shared our fluorescence idea of 2016 and 2017 with them.</P><P>After mutual inspection and troubleshooting with both projects, we were enlightened to expand our sensing device to detect more substances like environmental hormone even heavy metal in the future, which would make our project more practical. Also, NTHU offered their investigation results of aquaculture questionnaire to us. These questionnaires strengthened our data integrity and made our project more fit to the market demand.</P><P>This collaboration combined both teams’ detecting projects. We exchanged our hardware design, principle, commercial value, strategies so can make both projects more compatible. In the following paragraph, we will briefly show the function and mechanism of our nitrate sensor.</P><P><CENTER><FONT size="2">[ Figure. 1.4-2 ] Structure of nitrate sensor based on fluorescence. </FONT></CENTER></P><H2>． Sensor design</H2><P>We use a 450nm laser to excite our <I>E. coli.</I> Then, <I>E. coli.</I> will emit 510 nm green light and the green light will be received by the light sensor. By analyzing different fluorescence intensity. We will get concentration value of nitrate in the water..</P><H2>． Mechanism</H2><P>1. The water will enter the boat and flow through two solenoid valves.
			
			2. With the specific controlling, we can gather quantitative water for nitrate sensing.
			
			3. The sample of water will react with our <I>E. coli</I> in a tube fixed in a light absorber.  
			 	
			4. The laser will excite the <I>E. coli</I>, and the <I>E. coli</I> will emit the fluorescence.
			
			5. The fluorescence will be received by a light sensor.
			
			6. The Arduino will process the fluorescence we collected and send the data to server. 
			
			7. Our server will calculate fluorescence into the concentration of nitrate in the water.
		</P><P><CENTER><FONT size="2">[ Figure. 1.4-3 ] 3D model of nitrate sensor based on fluorescence. </FONT></CENTER></P><H1 style="color:#6c5070"><CENTER>GEM NTHU x iGEM TU Delft</CENTER></H1><H2>．Collaborate college :</H2><P>Technische Universiteit Delft ( Netherlands )</P><H2>．Collaborate Division :</H2><P>1.Wet Lab</P><H2>．Collaborate item : </H2><P>1.Data check</P><P>2.Troubleshooting</P><P>3.Enzyme improvement plan</P><H2>．NTHU Collaborate Memo :   </H2><P><A href="https://2017.igem.org/Team:TUDelft">iGEM TU Delft</A> is the first team we had Skype meeting with. They provided a method they had designed in which enzymes would be produced, packaged and released in vesicles in combination with “Tardigrade intrinsically disordered proteins” (TDPs)  stabilize enzymes just with a simple drying step rendering a functional enzyme upon subsequent rehydration. However, after a closer look to the integration of TU Delft’s approach, we reported some of our concerns about it: vesicles, could affect our  degradation ability, as  the volume might not be great enough as to carry enough ER-alpha or GFP and the variability in a number of enzymes packaged in the vesicles could compromise the reliability of the system. Nonetheless,  we believe the method should still be tested in the future.As time went by, we met some problems in the cloning and they helped us to inspect parts in detail and try to solve the problems. As a result, we changed the way of collaboration. Based on their advice, we transformed the plasmids of ER-INP and monobody into <I><I>E. coli</I></I> successfully(Figure.1.5-1) and we very much appreciated their help.</P><BLOCKQUOTE><BLOCKQUOTE><BLOCKQUOTE><CENTER><FONT size="2">[ Figure. 1.5-2 ]  1 : RFP ( mRFP F-primer and mRFP R-primer ); 2 : IPTG (promoter) + INP + RFP2 (ER F-primer and ER R-primer ); 3 : IPTG (promoter) + INP + RFP 1 (ER F-primer and ER R-primer ) ; 4 : IPTG (promoter) + ER with red spot  (ER F-primer and ER R-primer ); 5 : IPTG (promoter) + ER2  (ER F-primer and ER R-primer ); 6 : IPTG (promoter) + ER1  (ER F-primer and ER R-primer ); 7 : IPTG (promoter) + Monobody with red spot (Monobody F-primer and Monobody R-primer ); 8 : IPTG (promoter) + Monobody 2 (Monobody F-primer and Monobody R-primer ) ; 9 : IPTG (promoter) + Monobody 1 (Monobody F-primer and Monobody R-primer ); 10 : RFP ( VR primer and VF2 primer ); 11 : IPTG (promoter) + INP + RFP 2 ( VR primer and VF2 primer ); 12 : IPTG (promoter) + INP + RFP 1 ( VR primer and VF2 primer ); 13 : IPTG (promoter) + ER with red spot  ( VR primer and VF2 primer ); 14 : IPTG (promoter) + ER2  ( VR primer and VF2 primer ); 15 : IPTG (promoter) + ER1  ( VR primer and VF2 primer ); 16 : IPTG (promoter) + Monobody with red spot ( VR primer and VF2 primer ); 17 : IPTG (promoter) + Monobody 2 ( VR primer and VF2 primer ); 18 : IPTG (promoter) + Monobody 1 ( VR primer and VF2 primer )</FONT></CENTER></BLOCKQUOTE></BLOCKQUOTE></BLOCKQUOTE><P>The collaboration with team TU Delft is a brilliant experiment, the Skype meeting and part correction not only helps our team’s lab work but also build a nice friendship between two teams. We appreciate the works and really look forward to meeting TU Delft in Boston!</P><H2>．TU Delft Collaborate Memo :</H2><P>After stumbling upon the description of our project, NTHU Taiwan initiated conversations with us via Facebook.
Their project also consisted of a biosensor device, on this case featuring a series of overexpressed receptors and enzymes capable of both sensing and degrading EDC’s (Endocrine Disruptor Chemicals) that could be used on-site with sewage waters. A series of Skype meetings followed, in which the usefulness of the inclusion of any of our accomplished BioBricks into their project was discussed. Initially, vesicles and TDPs were deemed potentially interesting for NTHU Taiwan.However, after a closer study of our vesicles BioBrick by Taiwan, they came back to us with a series of points such as the variability in the content and size of vesicles, that made us aware of the weaknesses of our approach for packaging Cas13a and TDPs in our vesicles. Such a feedback resulted important for TU Delft iGEM 2017, as it made us aware of more aspects that should be addressed and optimised in our vesicles prior to its integration in our project.However, the collaboration took a different direction when Taiwan reported difficulties in the transformation involving their plasmids.</P><P><CENTER><FONT size="2">[ Figure. 1.5-3 ] Snapgene overview of NTHU Taiwan’s construct.</FONT></CENTER></P><P>TU Delft offered to revise their SnapGene files featuring their construct to discard a faulty design as the cause and shared a series of recommendations on how to make their design clearer in Snapgene and how to achieve transformations with higher yields.Nonetheless, a series of recommendations were still made by TU Delft as to eliminate redundancies and make their design in SnapGene clearer and more organized.Finally, NTHU communicated the accomplishment of their transformation after reordering their plasmids.We wish all the best to our fellow iGEMers of NTHU Taiwan and look forward to meeting them at the Giant Jamboree!</P><P><CENTER><FONT size="2">[ Figure. 2.5-5 ] iGEM TU Delft offers many suggestions for improving our project including to provide us one kind of protection protein to encapsulate our enzyme.</FONT></CENTER></P><P><CENTER>The hardware blueprint we offer to the collaborating teams.</CENTER></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>