<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCopenhagen_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCopenhagen/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container topnav"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/Project">Project</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/Project">Project description</A><A href="https://2017.igem.org/Team:UCopenhagen/Interdependency">Interdependency</A><A href="https://2017.igem.org/Team:UCopenhagen/Number-Control">Number control</A><A href="https://2017.igem.org/Team:UCopenhagen/Protein-Import">Protein import</A><A href="https://2017.igem.org/Team:UCopenhagen/Results">Results</A><A href="https://2017.igem.org/Team:UCopenhagen/Parts">Parts</A><A href="https://2017.igem.org/Team:UCopenhagen/Model">Modelling</A><A href="https://2017.igem.org/Team:UCopenhagen/InterLab">Interlab</A></DIV></DIV></LI><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/Safety">Safety &amp; Labnotes</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/Safety">Safety</A><A href="https://2017.igem.org/Team:UCopenhagen/Notebook">Notebook</A><A href="https://2017.igem.org/Team:UCopenhagen/Protocols">Protocols</A><A href="https://2017.igem.org/Team:UCopenhagen/Improve">Improve</A></DIV></DIV></LI><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/Team">People</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/Team">Team</A><A href="https://2017.igem.org/Team:UCopenhagen/Attributions">Attributions</A><A href="https://2017.igem.org/Team:UCopenhagen/Collaborations">Collaborations</A></DIV></DIV></LI><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/HP">Human Practices</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/HP/Silver">Overview</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Ethics">Ethics</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Gold_Integrated">Integrated</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Events">Events</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Media">Media</A></DIV></DIV></LI></UL></DIV></DIV><DIV class="intro-header4"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><DIV class="intro-message2"><H1><A name="Top">R E S U L T S</A></H1></DIV></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV><H2 class="section-heading">Introduction</H2><P class="lead">
On this page, we will present results from the experiments we have done within our three subprojects. 

The results we are the most proud of is the successful import of fluorescent proteins labeled with CPP-tag. 
On the other hand, our number control experiment didn’t show much - instead check out our modelling where we modelled how it could have worked. 
</P></DIV></DIV></DIV><DIV class="content-section-b"><DIV class="container"><DIV><H2 class="section-heading">Results from interdependency</H2><P class="lead"><STRONG>Gene expression and tryptophan production</STRONG>
In interdependency, we aimed to increase tryptophan biosynthesis in <I> E.coli</I> by overexpressing three proteins: the aromatic amino acid transporter YddG, the first protein in the shikimate pathway DAHP synthases (AroG), and the Anthranilate synthase component 1 (TrpE), responsible for anthranilate biosynthesis , precursor of L-Tryptophan (Trp). AroG and TrpE are naturally feedback inhibited, hence we modified them to be feedback resistant. This should allow higher rates of biosynthesis. (Gu et al., 2012)  
<I>aroG</I> and <I>trpE</I> were successfully mutated with the point mutation and verified by sequencing the genes inserted in our expression vector (Fig. 1). 

Expression of AroG and TrpE at 30 degrees in <I>E. coli</I> BL21 was verified via histidine column purification and western blot (Fig 2). 

Growth curves made of the transformed cells with empty vector, single YddG, AroG, TrpE, and combination YddG-AroG and YddG-TrpE, showed that all transformants grew well in YNB media without tryptophan (Fig 3). The lowest growth rate was observed in the transformant with the an empty vector (Control in Fig 3)
<FIGCAPTION><B>Figure 1 </B> In this figure the comparison between the natural Anthranilate synthase component 1 (TrpE) (a) and Phospho-2-dehydro-3-deoxyheptonate aldolase, Phe-sensitive (AroG) (b) and their respective negative feedback resistant (frb) version we obtained is shown. Above the DNA sequence, and the respective amino acid sequence, of the natural version is shown. Below the data obtained via the sequencing (Macrogen commercial service) of our modified genes. Is possible to see that TrpE becomes feedback resistant when 878T is substituted with cytosine, resulting in 293Met-&gt;Thr. Similarly, for AroG, the amino acid substitution 150Pro-&gt;Leu results in feedback resistance. </FIGCAPTION></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 2 </B>Western blot showing the expression levels of his-tagged AroG and TrpE expressed at 30℃. Three cultures of bacteria with empty expression vector, TrpE, or AroG were run. Size of AroG is 40 kDa, TrpE is 59 kDa.  </FIGCAPTION></FIGURE></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 3</B>Growth curves of <I>E.coli</I> in YNB pH 7.2 without tryptophan grown at 37 °C. <I>E. coli</I> transformed with vectors containing the different genes. Induction with 1 mM IPTG was performed 3 hours after inoculation. CK is empty vector. All measurement points are averages of two cultures.
 </FIGCAPTION></FIGURE></P><P class="lead"><STRONG>Tryptophan production</STRONG>
The expression of the proteins and growth of the cells, however, did not result in an increased production of tryptophan. 

Tryptophan concentration both in the media and from the cell lysate were measured with HPLC-MS. All concentrations measured in the media, however, were below the limit of quantification (LOQ), most being around 200 µg/L. In the cell lysate, concentrations above LOQ was only measured in two strains, and only after 58 hours growth: AroG (532.8 µg/L) and YddG-TrpE (404.2 µg/L). 

A higher concentration in cell lysate indicates that tryptophan synthesis is increased, but the export is not keeping up. However, even the 0.5 mg/L detected from the cell lysate is far below the 76 mg/L tryptophan in synthetic yeast media we aimed for. 


The tryptophan concentration in the media cleared from cells would be available for yeast growth in our serial growth experiments, or for the host cell in a real endosymbiotic system. 

Serial growth experiments performed in YNB media with tryptophan shows unhindered yeast growth, indicating that <I>E. coli</I> and yeast should be able to co-live (Fig 4). 

On the other hand, yeast in YNB without tryptophan that has to depend on tryptophan produced by our transformed <I>E. coli</I>, showed no growth, irrespective of the time <I>E. coli</I> had grown in the media (Fig 5). This finding fits with the tryptophan concentrations measured via HPLC-MS. 
</P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 4</B>Growth curves of Yeast grown in media with previous E.coli growth without added tryptophan
 </FIGCAPTION></FIGURE></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 5</B>Growth curve of yeast grown in YNB- media after media was cleared of our different <I>E. coli</I> strains. E.coli had grown in the media for 10.5, 31 or 58 hours before being removed by spinning and filtration to remove. 
 </FIGCAPTION></FIGURE></P></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV><H2 class="section-heading">Results from number control</H2><P class="lead"><STRONG>Sequencing og sgRNA</STRONG></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 6</B>Sequencing data of the plasmids pgRNA1, pgRNA2, pgRNA3 indicating that the proper seed sequence (highlighted in yellow) and sgRNA scaffold (following) has been inserted in all three plasmids. 
 </FIGCAPTION></FIGURE></P><P class="lead"><STRONG>Growth curve of induced/uninduced</STRONG>
The results obtained in number control sub-project were non-significant, therefore we decide to expand the modelling aspect of cell replication control. However, the results obtained in the wet-lab are described in the following paragraphs.

The data of the growth curve obtained via OD600 measurements without serial dilutions suggests the presence of detrimental effect on growth rate by all three construct pdCas9 – pgRNA1, pdCas9 – pgRNA2, pdCas9 – pgRNA3 when dCas9 expression is induced via tetracycline (200 ng/mL). However, further repetition of the experiment, with tuning of the protocol, did not significantly confirmed this trend.</P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 7</B>The figure shows the effect of dCas9-sgRNA complex targeting the OriC on the bacterial growth rate. Growth was measured via OD600 and the difference in density between inoculation time and 240 min incubation at 37 °C, 220 rpm was calculated. This suggests a detrimental effect of the system of growth when dCas9 is expressed (induction using tetracycline).
 </FIGCAPTION></FIGURE></P></DIV></DIV></DIV><DIV class="content-section-b"><DIV class="container"><DIV><H2 class="section-heading">Results from protein Import</H2><P class="lead">
In protein import, our aim is to express and import a fluorescent proteins linked to a 9 Arginine cell penetrating peptide (CPP). 
<STRONG>Expression of CPP and fluorescent proteins</STRONG>
Cells transformed with empty vector, CPP, YFP or BFP alone, or CPP-tagged fluorescent proteins (CPP-YFP, CPP-BFP) grew linearly both when induced and uninduced (Fig 8, 9 and 10). The emission of YFP and BFP were unaffected by CPP linkage. (Fig 11)
</P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 8</B>Growth curve of <I>E. coli</I> transformed with CPP-YFP, YFP or empty vector (CK). Dashed lines: uninduced, solid lines induced with 1 mM IPTG.  
 </FIGCAPTION></FIGURE></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 9</B>Growth curve of <I>E. coli</I> transformed with CPP-BFP, BFP or empty vector (CK). Dashed lines: uninduced, solid lines induced with 1 mM IPTG. 
 </FIGCAPTION></FIGURE></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 10</B>Growth curve of <I>E. coli</I> transformed with CPP or empty vector (CK). Dashed lines: uninduced, solid lines induced with 1 mM IPTG. 
 </FIGCAPTION></FIGURE></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 11</B>Normal and fluorescence microscopy of <I>E. coli</I> cells expressing (A-C): Empty vector, (D-F) CPP, (G) BFP, (H) YFP, (I) CPP-BFP, (J) CPP-YFP. Visualized with (A+D) Normal settings, (B, E, G and I) BFP filter settings, and (C,H and I) YFP filter settings. YFP filter settings: excitation 510-555 nm and emission 510-555 nm. BFP filter settings: excitation 330-385 and emission &gt;420 nm.  
 </FIGCAPTION></FIGURE></P><P class="lead"><STRONG>Purification</STRONG>
The fluorescent proteins with and without CPP tags were purified before testing the uptake into different <I>E. coli</I> cells via SDS-page and immunoblotting (His-tag). (Fig 12 and 13)</P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 12</B>SDS page of proteins purified from E coli cells with empty vector (CK), CPP alone and CPP-BFP. Numbers above the lanes is the elution number. Elution number 1 (combining the samples loaded on the two lanes within the red boxes) was used to test import.
 </FIGCAPTION></FIGURE></P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 13</B> SDS page of proteins purified from E coli cells with BFP, YFP and CPP-YFP. Numbers above the lanes is the elution number. Elution number 1 (combining the samples loaded on the two lanes within the red boxes) was used to test import.
 </FIGCAPTION></FIGURE></P><P class="lead"><STRONG>Import</STRONG>
Functionality of CPP in facilitating the fluorescent proteins cellular internalization was tested using fluorescent microscopy. Purified proteins CPP, YFP, BFP, CPP-YFP, or CPP-BFP were added to <I>E. coli</I> cultures. These were incubated for 10 minutes, washed to remove extracellular (or non membrane-bounded) proteins. Finally visualisation via fluorescence microscopy (Olymus BX60) was carried out. Fig 14 shows that treated cells emit fluoresc after treatment with CPP tagged YFP/BFP, but not when treated with untagged fluorescent proteins. This is a strong indicator that the CPP tag facilitates import of proteins in <I>E. coli</I>. </P><P class="lead"><FIGURE><FIGCAPTION><B>Figure 14</B>Fluorescence microscopy images of import experiment in <I>E.coli</I>. (A+B) BFP, (C+D)CPP-BFP, (E+F)YFP, (G+H)CPP-YFP. Top row is normal settings, while (B+D) is BFP filter settings and (F+H) is YFP filter settings. YFP filter settings: excitation 510-555 nm and emission 510-555 nm. BFP filter settings: excitation 330-385 and emission &gt;420 nm.  Images are cropped to focus on cells expressing fluorescence when any are observed. </FIGCAPTION></FIGURE></P></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV><H2 class="section-heading">Conclusions</H2><P class="lead"><STRONG>In Interdependency,</STRONG> we had successfully transformed and expressed three genes involved in tryptophan production and export. Namely, AroG, TrpE, and YddG. However, the level of tryptophan produced by our <I>E. coli</I> strains is not sufficient to sustain the growth of tryptophan auxotroph yeast. The level of intracellular Trp in <I>E. coli</I> showed an increase in the strains AroG and YddG-TrpE. Moreover, our experiment show that <I>E. coli</I> is able to grow onto yeast YNB media. This gave us a reason to say that a relationship between the two cells is not impossible.
<STRONG>In Number Control,</STRONG> we successfully created three sgRNA and separately transformed cells with these and a dCas9 expression plasmid. However, induction of dCas9 did not lead to a significant inhibition of growth as expected. Therefore, we continue our investigation of the number control mechanism in our modelling.<STRONG>In Protein impor,</STRONG>  we expressed fluorescent proteins with and without a cell penetrating peptide (CPP). This showed that the fluorescent proteins emission is not affected by the 9 arginine tag. And that CPP greatly enhanced the colocalization of cell and fluorescent CPP tagged proteins. Preliminary results (shown on the registry page of our cell penetrating USER casette biobrick) suggests that this mechanism is present also in other bacterial taxa (<I>Pseudomona aeruginosa</I>).</P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-5"><H2>Find Incell here:</H2></DIV><DIV class="col-lg-7"><UL class="list-inline banner-social-buttons"><LI><A href="https://twitter.com/iGEM_Copenhagen" target="_blank" class="btn btn-default btn-lg"><SPAN class="network-name">Twitter</SPAN></A></LI><LI><A href="https://www.facebook.com/iGEMCopenhagen" target="_blank" class="btn btn-default btn-lg"><SPAN class="network-name">Facebook</SPAN></A></LI><LI><A href="https://www.instagram.com/igemku2017" target="_blank" class="btn btn-default btn-lg"><SPAN class="network-name">Instagram</SPAN></A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV id="page-top" data-spy="scroll" data-target=".navbar-fixed-top"><NAV class="navbar scrollnav-custom navbar-fixed-bottom" role="navigation" id="scrollnav"><DIV class="container"><DIV class="navbar-header"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#Top">Introduction</A></LI><LI><A class="page-scroll" href="#Design">Final design</A></LI><LI><A class="page-scroll" href="#experiment">Experiments</A></LI><LI><A class="page-scroll" href="#process">Design Process</A></LI></UL></DIV><DIV class="collapse navbar-collapse navbar-right navbar-main-collapse"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="https://2017.igem.org/Team:UCopenhagen/Project">Previous</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:UCopenhagen/Number-Control">Next</A></LI></UL></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>