Team:UCopenhagen/Number-Control
Toggle navigation
Project
Project description Interdependency Number control Protein import Results Parts Modelling Interlab
Safety & Labnotes
Safety Notebook Protocols Improve
People
Team Attributions Collaborations
Human Practices
Overview Ethics Integrated Events Media
N U M B E R C O N T R O L
Introduction
The second mechanism we considered is number control. This is the intertwinement of the symbiont and host replication cycles. We believe this to be an imperative function of a healthy relationship between host and symbiont. In fact, the host cell is limited in size and it will be able to contain only a defined number of cells. Moreover, the exchange of nutrients also require a tightly regulated balance. For example, the mechanism of interdependency is based upon the providing of enough tryptophan to the host cell. This would not only depend on the production of amino acid per cell but also on the number of symbionts. Similarly, the symbionts won’t be able to directly access any resource not provided by the host cell. Hence, as in the financial market, a correct balance of consumer and producer must be kept to avoid our symbiotic relationship turning into a parasitic relationship. Number control can be seen as the New Deal of endosymbiosis.
Our vision is to use a modular system based on Clustered Regularly Inverted Short Palindromic Repeats interference (CRISPRi) technology to efficiently connect the symbiont cell replication cycle to the status of the endosymbiosis relationship. As a first step towards an automated number control system, we will investigate the control of cell replication through DNA replication inhibition and model the incorporation of a cell density based system - that is, a quorum sensing circuit - regulating the replication control mechanism.
Final Design
Background: Prior to cell replication a bacteria need to double all cell components, including the genomic DNA. The DNA replication begins with the opening of the double strand DNA (dsDNA) carried out by the protein helicase. Preventing the attachment of helicase to the dsDNA will prevent double strand separation, genome replication, and ultimately, cell division.
To disrupt the helicase-DNA interaction, we designed a genetic regulation tool based on CRISPR interference (CRISPRi). This technology was first developed by Qi et al (2013) and is constituted by two basic blocks, i.e. a guide RNA (sgRNA) complementary to a target sequence and a catalytically dead Cas9 endonuclease (dCas9). dCas9 differentiate from Cas9 by its lack of nuclease activity. Hence, dCas9 will bind to the sgRNA and sit on the target DNA sequence, without altering permanently its structure. The presence of dCas9 on the target site sterically blocks the binding of other proteins, such as helicase.
Our goal is to inhibit attachment of helicase to the origin of replication (ORI), using CRISPRi system targeting the ORI, thus disrupting the cell replication of E.coli.
Circuits and biobricks: Our CRISPRi system is composed by a catalytically dead Cas9 (dCas9) and three guide RNAs (sgRNAs) complementary to three distinct sites of E.coli origin of replication (OriC), i.e. on the DnaA helicase binding box (sgRNA1), before the DnaA binding box (sgRNA2) and after it (sgRNA3)(Wiktor et al., 2016). dCas9 is expressed from an anhydrotetracycline (aTc) inducible promoter and the sgRNAs are expressed separately from a minimal constitutive promoter (J23119). The choice of expressing the sgRNA constitutively arise from the higher efficiency of dCas9 to assemble with the sgRNA if the latter is already present in the cytosol (ref). The assembled protein-sgRNA will then bind to the complementary region on the bacterial chromosome and occupy that area. Detachment of dCas9-sgRNA complex from the target sequence can be achieved by incubating the cells at 42 °C (Wiktor et al., 2016)
The quorum sensing (QS) model is based on the Rhl genetic circuit found in Pseudomona aeruginosa.
Experiments
Overview
General procedure
Creation of working E. coli DH5-α strains:
OD600 growth curve
General procedure
To obtain large quantities of our plasmids 3xFLAG-pdCas9 and pgRNA-bacteria we transformed them into E. coli mach1 using Mix&Go (Zymo Research). To verify each transformation we purified the plasmid with column purification, linearized with proper restriction enzyme (RE) digestion to avoid supercoiling, and ran the linearized product in 1% agarose gel electrophoresis.
Moreover, all the purified plasmids were sent for sequencing (Macrogen EZ-seq commercial service).
Creation of working E. coli DH5-α strains
The following commercially available plasmids from AddGene were selected for our experiment:
3xFLAG-dCas9/p-bacteria (Addgene #64325)
pgRNA-bacteria (Addgene #44251)
Figure 1 Plasmid: 3xFLAG-dCas9/p-bacteria (Addgene #64325)
Figure 2 Plasmid: pgRNA-bacteria (Addgene #44251) which was used for insertion of seed sequence
The three seed sequences designed were inserted using PCR. Thus creating the new plasmids pgRNA1, pgRNA2, and pgRNA3.
We used the following primers for insertion of the seed sequences, where the underlined sequence is SpeI restriction site and the bold text is the seed sequence complementary to the E. coli DH5-α OriC.
fw.sgRNA1 CCACTAGTGCACTGCCCTGTGGATAACAGTTTTAGAGCTAGAAATAGCAAG
fw.sgRNA2 CCACTAGTTTGAGAAAGACCTGGGATCCGTTTTAGAGCTAGAAATAGCAAG
fw.sgRNA3 CCACTAGTGATCATTAACTGTGAATGATGTTTTAGAGCTAGAAATAGCAAG
rv.sgRNA GGACTAGTATTATACCTAGGACTGAG
After purification of the modified pgRNA plasmids, transformation of E. coli DH5-α was carried out using heat-shock protocol. And the following strains were obtained:
E. coli DH5-α pdCas9;
E. coli DH5-α pgRNA1;
E. coli DH5-α pgRNA2;
E. coli DH5-α pgRNA3;
E. coli DH5-α pdCas9 pgRNA1;
E. coli DH5-α pdCas9 pgRNA2;
E. coli DH5-α pdCas9 pgRNA3.
Growth curve (OD600)
The growth rate of the obtained strains was investigated in absence and presence of the dCas9 inductor tetracycline (tet) using absorbance measurements (protocol).
The strains E. coli DH5-α pdCas9, E. coli DH5-α pgRNA1, E. coli DH5-α pgRNA2, and E. coli DH5-α pgRNA3 cannot be considered proper negative control since they are cultivated with different antibiotics. Hence, three new negative control strain were designed and created:
E. coli DH5-α empty pdCas9 (EpdCas9). That is, the pdCas9 plasmid with truncated dCas9 CDS using EcoRI restriction);
E. coli DH5-α pdCas9 + pgRNA-bacteria (that is, the sgRNA scaffold without seed sequence);
E. coli DH5-α EpdCas9 pgRNA-bacteria (we did manage to obtain this strain).
Moreover, considering that the stop of DNA replication is expected to cause cell enlargement (Wiktor et al., 2016) the OD600­ measurement results might altered by the cell size. To overcome this and other issues a new protocol was design.
However, the difference in growth observed was never significant. Hence, we decided to focus on the numerical model describing the Number Control project, where we also explored the integration of a quorum sensing circuit.
Design process/future
After the design of the growth rate experiment, we started exploring other possible layers of complexity. Namely, Wiktor et al. (2016) has shown that upon successful genome replication arrest, the cell stays metabolically active, accumulating biomass. For this reason, we suggest the inhibition of lipid biosynthesis to limit cell membrane growth, hence cell enlargement. This could be obtained with a similar sgRNA-dCas9 circuit targeting the gene fabI, CDS for the native enoyl-acyl carrier protein (enoyl-ACP) reductase (ENR). This protein is in fact essential for the type II fatty acid biosynthesis and is commonly target of bacteriostatic drugs (Escaich, ‎2011).
Besides the autoinhibitory control of cell cycle, we are deeply interested in connecting the number control circuit to the host life cycle to further stabilize the relationship between host and symbiont. To do so, we propose to place the pgRNA-dCas9 system under control of three host-symbiont signals, i.e. symbiont abundance (that is, a quorum sensing sub system), host cell starvation, and host cell replication. Namely, a high symbiont abundance and/or a starvation status of the host cell would activate the dCas9 expression lowering or stopping symbiont replication. While the host cell initiation of replication would inactivate the dCas9 expression, so to allow an easy propagation of the symbiont.
References
Find Incell here:
Twitter
Facebook
Instagram
Introduction
Final design
Experiments
Design Process
Previous
Next
