Team:UCopenhagen/Interdependency
Toggle navigation
Project
Project description Interdependency Number control Protein import Results Parts Modelling Interlab
Safety & Labnotes
Safety Notebook Protocols Improve
People
Team Attributions Collaborations
Human Practices
Overview Ethics Integrated Events Media
I N T E R D E P E N D E N C Y
Introduction
The first mechanism that we have decided to investigate is the interdependency between two cell types. In order to have a stable relationship in which the host-endosymbiont relationship is maintained through generations, the host and endosymbionts must depend on each other for their continued survival.
Within the endosymbiotic relationship, we envision a resource based cross-network between host and symbiont. The exchange of vital metabolites, necessary for proliferation, would ensure the coexistence of the symbiotic pair, whilst also ensuring mutual demise in case of the host’s or the symbiont’s perishment. Our goal would be that the pair would survive only if the condition of endosymbiosis is met.
This is important, as our project is about laying the foundation for stable modular endosymbiosis. We are working to establish a dependency between free living cells, as a groundwork for a later endosymbiotic relationship. Natural endosymbiotic relationships have been found between fungi and bacteria in mycorrhiza strains ( Bianciotti, 2000 ), and we will be working with two cells from these two phylogenetic kingdoms in our project; a tryptophan auxotrophic yeast AM94 and E. coli.
Final Design
Background: Synthetic yeast media, suitable for growth of tryptophan auxotroph yeast, contains 76mg tryptophan pr. liter (Sigma-Aldrich, 2017). An accumulation of 1.7g tryptophan pr. liter was achieved by Gu et al. (2012) with a modified E.coli strain, which is sufficient for yeast growth. The production of tryptophan is regulated by negative feedback mechanisms that inhibits tryptophan synthesis in the presence of tryptophan or related amino acids.
Our goal is to produce and export enough tryptophan from E.coli to complement growth of an auxotroph yeast strain, grown in media without tryptophan. Additionally, we are interested in the number of endosymbionts necessary per host. This is included in our modelling .
Circuits and biobricks: In our circuit, we are overexpressing the three genes AroG, TrpE and YddG, which will also become our biobricks. We have made our selection based on the papers by Gu et al. (2012) and Wang et al. (2013) .
TrpE belongs to the tryptophan operon and has been overexpressed frequently in L-tryptophan producing E. coli strains.
AroG is the first enzyme of the shikimate pathway, thereby determining the carbon flow towards tryptophan synthesis.
Both AroG and TrpE (Figure 1) are regulated by the concentration of the amino acids produced. This feedback signalling reduces the tryptophan concentration that we can achieve with WT E.coli aroG and trpE genes, as simply overexpressing them would lead to inhibition due to increased production.
We have made use of known mutant feedback resistant alleles for trpE and aroG to overcome this regulation ( Gu et al. (2012) ). In the proteins TrpE, a mutation in a methionine to threonine at position 293 is required, and for AroG, the proline at position 150 is changed to leucine.
By making these point mutations, the feedback inhibition should be released, and overexpression would lead to elevated tryptophan production. We submitted the feedback resistant alleles as biobricks:
trpE: BBa_K2455001
aroG: BBa_K2455000
The last gene, yddG is an aromatic amino acid exporter and is responsible for the secretion of both tryptophan, tyrosine and phenylalanine.
Gu et al. (2012) have shown that the over-expression of YddG in E.coli increases the accumulation of tryptophan in the growth medium. This is likely due to the decrease in intracellular concentration, thus bypassing the feedback sensitive regulatory steps in tryptophan biosynthesis. We have designed a codon-optimised version of yddG that we have submittd as Biobrick BBa_K2455004 .
Constructs with each gene, or two of the genes together, will be evaluated in order to examine both the combined and isolated effects of the genes.
Figure 1 AroG and TrpE location in aromatic amino acid biosynthesis: AroG initiates the shikimate pathway, and is inhibited by Phenylalanine.  TrpE is the first protein in the trp operon and inhibited by tryptophan. The red crosses indicate feedback inhibition removed by the point mutations.
Experiments
Overview
Amplification of genes:
Synthetically produce a codon-optimised version of yddG
Amplify trpE and aroG with point mutations
Multi-gene constructs inserted in vector
Serial growth of E. coli and yeast
Identify a minimal media supporting both E. coli and yeast growth
Analyse auxotrophic yeast growth in the minimal media both with and without tryptophan added
Grow auxotrophic yeast in media only containing tryptophan produced by E.coli
Gene expression and tryptophan production
Western blotting to check expression of his-tagged proteins
HPLC-MS measurements of tryptophan production
Verifications
We have used gel electrophoresis in the majority of our experiments and sub-projects to continually monitor the progress.
All transformations in interdependency and protein import were done with USER cloning and the cells were grown ON on LB agar with the appropriate selection (Ampicillin or Chloramphenicol depending on the resistance marker). Colony PCR s were performed to pick colonies with the correct inserts for inoculation in liquid culture.
Subsequently, plasmids were cured from the cultures and PCR amplification and/or restriction digestion were used to check for correct insertion. Cells with appropriately sized inserts were sent to sequencing for sequence validation.
Biobrick creation
We amplified our biobricks with primers carrying the biobrick prefixes and suffixes as overhangs, and purified them with gel extraction. Subsequently, we used the standard iGEM protocol for its insertion into the pSB1C3 submission vector.
All biobricks were sequenced in the expression vector, and aroG, trpE, CPP and CPP-YFP were also sequenced following their insertion into the submission vector.
Vector design
A modified version of the pET102 vector was used for expression of all genes in this sub-project. See protein import for details on design and creation of the vector.
Codon optimization and synthetic yddG
Following problems with amplifying yddG from the genomic DNA, we decided to order a synthetically made version, and codon optimized it in the process.
Point mutations for feedback resistance in trpE and aroG
We amplified the aroG and trpE genes using genomic DNA from E.coli strain MG1655. The amplification was done using two sets of primers per gene, with overhangs allowing for insertion of a point mutation in each gene and for cloning with USER assembly.
The mutations were designed in accordance with previous work by Gu et al. (2012) . In TrpE, position 293 is changed from methionine to threonine, and for AroG position 150 is changed from proline to leucine.
Figure 2 Point mutation in trpE, and insertion in vector with USER assembly. To perform the point mutation, we designed two sets of primers for each gene. The outer primers are the same as used for amplification from the genomic DNA. The central primers have overhangs containing the point mutations. This way, the genes are split in two, and the two parts are combined when inserting in the USER cassette in the expression vector - and now containing a point mutation in the middle.
Multi-gene constructs
To evaluate the effect of the proteins, we made multi-gene constructs.  The constructs were made by designing primers with overhangs encoding Ribosome Binding Sites (RBS). RBS makes sure all genes will be transcribed, and the overhangs enable combination of the genes into multi-gene constructs (fig 3) and USER assembly.
We created biobricks of one or two genes in combination. We attempted to make all the following constructs into biobricks, but did not succeed with the 3 gene construct.
YddG-his
TrpE-his
AroG-his
YddG-TrpE-his: BBa_K2455007
YddG-AroG-his: BBa_K2455008
YddG-TrpE-AroG-his
Figure 3 Multi-gene constructs were created with a series of primers with overhangs coding for Ribosome Binding Sites (RBS) between the genes. The outer primers coded for USER overhangs for USER ligation.
Serial growth experiment of E.coli and yeast
E. coli strains MG1655 and BL21 were grown in several media, until we settled on the minimal yeast media YNB with the pH adjusted to 7 ( van Summeren-Wesenhagen and Marienhagen, 2014 ).
To ensure that E.coli do not produce substances hindering yeast growth, we performed an initial serial growth experiment, shown in figure 4. The serial growth experiment was performed with ON growth of E.coli in YNB pH 7, clearing the media of E.coli by spinning and filtration through 0.2 µm filters, and inoculating with yeast (AM94).
Figure 4 Serial growth experiment. 1) E.coli from liquid culture in LB media is inoculated in YNB media. 2) After sufficient growth of E.coli in YNB, the media is cleared of E.coli by spinning and filtration. 3) E.coli free media is then inoculated with tryptophan auxotroph yeast.
Serial growth experiment with transformed E.coli
To evaluate, if the complementation of the yeast amino acid auxotrophy by E.coli tryptophan production is successful, we made serial growth experiments with transformed E. coli.
We first growed the E.coli, transformed with the single, double and triple construct vectors described above in YNB media without a tryptophane source. Subsequently, we cleared the media and inoculated it with auxotroph yeast, then evaluated the growth using OD600 measurements.
We also checked the growth of auxotrophic yeast in the media with and without added tryptophan, to ensure that any growth after E.coli growth was due to tryptophan being produced by E.coli and not due to the yeast overcoming the auxotrophy.  All serial growth experiments used variations of the same protocol .
HPLC measurements of tryptophan production
In addition to the qualitative assessment of tryptophan production by yeast growth, we made a quantitative evaluation of tryptophan production by our transformed E.coli.
We used HPLC-MS with a program for amino acid detection to measure the concentration of tryptophan in the media after 10-40 hours growth. Internal and external standards were used. Simultaneously, we measured concentrations of tyrosine and phenylalanine, which should be increased when AroG is expressed.
We measure production from the following transformations of E. coli.
empty vector
YddG-his
TrpE-his
AroG-his
YddG-TrpE-his
YddG-AroG-his
YddG-TrpE-AroG-his
YddG-TrpE-AroG
Growth of these transformants are simultaneously monitored with OD600 measurements.
Figure 5 Photo of the HPLC-MS set-up.
Design process/future
Summary: We cloned E.coli strains to express proteins in the tryptophan synthesis pathway, and have measured the production of these cells. We have ensured that auxotroph yeast can grow after E.coli in the same media, and have attempted to grow it on the tryptophan produced by E. coli as the single tryptophan source.
In our design process, we have considered a wide range of possible gene combinations and interdependency mechanisms. We settled on a simple amino acid interdependency which is easy to test with an amino acid auxotroph yeast.
In a final version of an endosymbiotic relationship, amino acid dependency might not be the optimal dependency as it would not allow the system to be grown on a media with all amino acids that might be required for co-cultures for added modularity. Instead of tryptophan, a different, but vital metabolite could be supplied by the symbiont.
To perform our tryptophan overproduction, we chose genes that, when over-expressed would have the greatest impact, based on the article by Gu et al. (2012) . Initially, we considered simply overexpressing the tryptophan operon, but quickly realised this would be highly downregulated due to negative feedback regulation.
To release the feedback regulation, we decided on an amino acid translocator to reduce the intracellular tryptophan concentration. Several deletions of e.g. endogenous trpR has also been considered, but this would make our project overly complicated due to the difficulty of making such deletions in E.coli.
References
Summeren-Wesenhagen, P. V., & Marienhagen, J. (2014). Metabolic Engineering of Escherichia coli for the Synthesis of the Plant Polyphenol Pinosylvin. Applied and Environmental Microbiology, 81(3), 840-849. doi:10.1128/aem.02966-14
Gu, P., Yang, F., Kang, J., Wang, Q., & Qi, Q. (2012). One-step of tryptophan attenuator inactivation and promoter swapping to improve the production of L-tryptophan in Escherichia coli. Microbial Cell Factories, 11(1), 30. doi:10.1186/1475-2859-11-30
Wang, J., Cheng, L., Wang, J., Liu, Q., Shen, T., & Chen, N. (2013). Genetic engineering of Escherichia coli to enhance production of l-tryptophan. Applied Microbiology and Biotechnology, 97(17), 7587-7596. doi:10.1007/s00253-013-5026-3
Sigma-Aldrich. (2017). Yeast Synthetic Drop-out Medium Supplements Y1501. (n.d.). Retrieved October 31, 2017, from http://www.sigmaaldrich.com/catalog/product/sigma/y1501?lang=en&region=DK
Bianciotto, V., Lumini, E., Lanfranco, L., Minerdi, D., Bonfante, P., & Perotto, S. (2000). Detection and Identification of Bacterial Endosymbionts in Arbuscular Mycorrhizal Fungi Belonging to the Family Gigasporaceae. Applied and Environmental Microbiology, 66(10), 4503-4509. doi:10.1128/aem.66.10.4503-4509.2000
Reference for USER cloning:
Nour-Eldin, H. H., Hansen, B. G., Nørholm, M. H., Jensen, J. K., & Halkier, B. A. (2006). Advancing uracil-excision based cloning towards an ideal technique for cloning PCR fragments. Nucleic Acids Research, 34(18). doi:10.1093/nar/gkl635
Reference for USER fusion:
Geu-Flores, F., Nour-Eldin, H. H., Nielsen, M. T., & Halkier, B. A. (2007). USER fusion: a rapid and efficient method for simultaneous fusion and cloning of multiple PCR products. Nucleic Acids Research, 35(7). doi:10.1093/nar/gkm106
Find Incell here:
Twitter
Facebook
Instagram
Introduction
Final design
Experiments
Design Process
Previous
Next
