<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCopenhagen_Protein-Import skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCopenhagen/Protein-Import</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container topnav"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav navbar-right"><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/Project">Project</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/Project">Project description</A><A href="https://2017.igem.org/Team:UCopenhagen/Interdependency">Interdependency</A><A href="https://2017.igem.org/Team:UCopenhagen/Number-Control">Number control</A><A href="https://2017.igem.org/Team:UCopenhagen/Protein-Import">Protein import</A><A href="https://2017.igem.org/Team:UCopenhagen/Results">Results</A><A href="https://2017.igem.org/Team:UCopenhagen/Parts">Parts</A><A href="https://2017.igem.org/Team:UCopenhagen/Model">Modelling</A><A href="https://2017.igem.org/Team:UCopenhagen/InterLab">Interlab</A></DIV></DIV></LI><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/Safety">Safety &amp; Labnotes</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/Safety">Safety</A><A href="https://2017.igem.org/Team:UCopenhagen/Notebook">Notebook</A><A href="https://2017.igem.org/Team:UCopenhagen/Protocols">Protocols</A><A href="https://2017.igem.org/Team:UCopenhagen/Improve">Improve</A></DIV></DIV></LI><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/Team">People</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/Team">Team</A><A href="https://2017.igem.org/Team:UCopenhagen/Attributions">Attributions</A><A href="https://2017.igem.org/Team:UCopenhagen/Collaborations">Collaborations</A></DIV></DIV></LI><LI><DIV class="dropdown"><BUTTON class="dropbtn"><A href="https://2017.igem.org/Team:UCopenhagen/HP">Human Practices</A></BUTTON><DIV class="dropdown-content"><A href="https://2017.igem.org/Team:UCopenhagen/HP/Silver">Overview</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Ethics">Ethics</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Gold_Integrated">Integrated</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Events">Events</A><A href="https://2017.igem.org/Team:UCopenhagen/HP/Media">Media</A></DIV></DIV></LI></UL></DIV></DIV><DIV class="intro-header4"><DIV class="container"><DIV class="row"><DIV class="col-lg-12"><DIV class="intro-message2"><H1>P R O T E I N   I M P O R T</H1></DIV></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV><H2 class="section-heading">Introduction </H2><P class="lead">
The third mechanism that we have investigated in our project is protein import into bacterial cells, as a stand in for a symbiont. In the two best known endosymbiotic organelles, mitochondria and chloroplasts, a majority of gene expression has moved from the symbiont to the host. Because this relationship seems to be an evolutionary foundation of the known endosymbiotic relationships, we will attempt to imitate this concept.

Besides our interest in the evolutionary aspect of protein import, we believe it is valuable when approaching modular endosymbiosis. The principle of protein import would enable additional modularity in an endosymbiotic system, as one host can be manipulated to produce proteins to be utilized by a range of symbionts. 

The majority of the proteins, destined for mitochondria, are expressed in the cytosol and subsequently imported across the membranes via transport complexes taking up unfolded peptides with an N-terminal signal sequence targeting them to the mitochondria (<U><A href="#P5">Schmidt <I>et al</I> 2010</A></U>). Similar transport complexes are found in chloroplast. We want a similar targeted uptake, but in a simpler system. Thus, we utilise a small peptide shown to penetrate many types of cells; a Cell Penetrating Peptide (CPP) (<U><A href="#P1">Chang <I>et al</I> 2005</A></U>, and <U><A href="#P2">Chang<I>et al</I>, 2014</A></U>). 
</P></DIV></DIV></DIV><DIV class="content-section-b"><DIV class="container"><DIV><H2 class="section-heading">Final Design </H2><P class="lead"><STRONG>Background</STRONG>:  CPPs are small peptides, typically rich in arginines, which are able to facilitate transport of a wide variety of cargos across plasma membranes. Their origin in nature comes from viral domains such as the viral HIV tat domain (<U><A href="#P4">Eudes and Chugh, 2008</A></U>). In recent years, research on creating synthetics CPPs has been conducted and especially peptides constructed solely from arginine residues have been of interest. The arginine rich sequence has been shown to trigger endocytosis in a wide range of cell types, including onion and potato cells. These experiments have shown that GFP connected to a CPP has entered the cells contained in vesicles (<U><A href="#P1">Chang <I>et al</I> 2005</A></U>). 

If CPP can be used as a protein tag for import into an endosymbiotic symbiont, the host proteins targeted to the symbiont would simply need the CPP added. 
<STRONG>Goal</STRONG>: Evaluate the efficiency of protein uptake by our <I>Escherichia coli</I> chassis in presence and absence of the cell penetrating peptide (CPP). 
<STRONG>Circuits and Biobricks</STRONG>: The parts in our circuit are fluorescent proteins and CPP. 
We have chosen the yellow and blue fluorescent proteins (<A href="http://parts.igem.org/Part:BBa_K864100">YFP</A> and <A href="http://parts.igem.org/Part:BBa_K592100">BFP</A>) from the Biobrick repository, and improved them by adding the CPP sequence to the C-terminal end. 
YFP: <A href="http://parts.igem.org/Part:BBa_K2455002">BBa_K2455002 </A>
BFP: <A href="http://parts.igem.org/Part:BBa_K2455005">BBa_K2455005  </A>

Additionally, we created a biobrick of CPP with a USER casette, ready for insertion of any protein to be imported: <A href="http://parts.igem.org/Part:BBa_K2455003">BBa_K2455003</A>. 

YFP and BFP were chosen to avoid overlapping colours with FM4-64, a red staining used for membranes, in case we wanted to look into the localization and potential vesicle breaking using confocal microscopy. As it turned out, we did not get far enough in the lab to do this.
</P></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV><H2 class="section-heading">Experiments</H2><H4>Overview</H4><P class="lead">
General verification</P><P class="lead">
Vector creation
<UL style="text-align:left; color:white;"><LI>Biobrick compatible (failed) </LI><LI>Vector design </LI><LI>CPP tag insertion </LI></UL></P><P class="lead">
Evaluating protein import:
<UL style="text-align:left; color:white;"><LI>Expression of fluorescent proteins</LI><LI>Purification and import of the expressed proteins</LI></UL></P><H4>Verifications and biobrick creation</H4><P class="lead">

In all three sub projects, we have used gel electrophoresis and sequencing to verify our stepwise experiments. Read more about our general verifications and biobrick creation under <A href="https://2017.igem.org/Team:UCopenhagen/Interdependency">interdependency</A>.
</P><H4>Vector creation</H4><P class="lead">

Our vector is a modified version of the <A href="https://www.addgene.org/vector-database/2623/">pET102 vector</A>, which contains a USER cassette, and a his tag after the USER cassette. The USER cassette is used for cloning genes into. We build the vector design on prSET102; an existing vector in our lab, containing the USER cassette and some restriction sites.  
We have made two versions of the vector: one for expression of untagged YFP and BFP, that was also used in the interdependency subproject. The other has a CPP tag before the USER cassette, which will add CPP to the proteins. 

</P><FIGURE><FIGCAPTION><B>Figure 1 </B>Overview of final vectors. First level: The two vectors, unlinearized. Second level: linearized by opening in USER casette. Third level: Insertion of YFP. 4: Produced YFP protein with and without CPP tag. YFP used as example; the same method is used for expression of BFP with and without CPP tag. 
</FIGCAPTION></FIGURE><FIGURE><FIGCAPTION><B>Figure 2 </B>Our initial vector design was built on pRSET, and would have had possibility for 3A assembly and immediate biobrick formation. We worked a lot on this initial vector, but did not succeed in creating it.
</FIGCAPTION></FIGURE><H4>Expression of fluorescent proteins</H4><P class="lead">We inserted BFP and YFP with USER cloning, and transformed them into our expression strain BL21 using Mix&amp;Go. BFP and YFP was expressed with the CPP tag from the CPP vector, and without from the vector without CPP. Together with CPP alone, this gave us a total of 5 protein constructs expressed. <UL style="text-align:left; color:white;"><LI>CPP-USER-his</LI><LI>CPP-YFP-his</LI><LI>CPP-BFP-his</LI><LI>YFP-his</LI><LI>BFP-his
</LI></UL></P><P class="lead">The expression was verified using western blotting, and fluorescence of YFP and BFP both with and without CPP was verified with fluorescence microscopy.</P><H4>Expression of fluorescent proteins</H4><P class="lead">
In order to evaluate the import of the CPP tagged and non-tagged proteins, we first <A href="https://static.igem.org/mediawiki/2017/8/88/Protein_extractions_and_purification.pdf">purified</A> the fluorescent proteins, then incubated <I>E.coli</I> cells with the purified proteins. Afterwards we removed the non-endocytosed fluorescent proteins by washing with water. (Fig 2) 
Fluorescence microscopy was used to see if the fluorescent proteins had been imported into the cells. If the import system works with CPP tags, the fluorescent proteins should now be present in the cells treated with CPP-tagged fluorescent proteins, but not in the non-CPP tagged. <FIGURE><FIGCAPTION><B>Figure 3 </B>Protein import with YFP A: with and B: without CPP tag - asme is done with BFP. 1) Proteins are expressed in <I>E.coli</I>, then 2) purified, before 3) import in other cells is investigated; both in <I>E.coli</I> and in eukaryotic cells. Non-endocytosed YFP/BFP is removed before looking at the fluorescence of the treated cells.
</FIGCAPTION></FIGURE></P></DIV></DIV></DIV><DIV class="content-section-b"><DIV class="container"><DIV><H2 class="section-heading">Design process/future</H2><P class="lead"><STRONG>Summary</STRONG>: We have made vectors to express YFP and BFP, and checked that they are expressed and fluorescing both with and without the CPP tag. We investigated protein import into cells with fluorescence microscopy. <STRONG>Vector Design</STRONG>: Our initial vector design was Biobrick compatible, with XXX and XXX as prefixes and suffixes. We designed a long primer to span entire sections and generate the whole vector, but this never succeeded. Instead, we decided to scale down on the design and use a simpler vector based on pET102, where we needed to make fewer modifications. <STRONG>Import system</STRONG>: Initially, we looked into utilizing existing protein import systems found in endosymbionts: import systems from mitochondria, chloroplasts or even peroxisomes. Quickly, though, we realised that these systems are too complex to be achieved within an iGEM project. The mitochondrial translocation system contains at least 5 proteins in a complex just in the outer membrane (<U><A href="#P7">Truscott <I>et al</I> 2003</A></U>), which is already very different from bacterial membranes. The same problem arises with regards to the chloroplast translocation system (<U><A href="#P6">Shi and Theg 2013</A></U>), which also requires a large number of proteins to function. 

After this initial set-back in the design proccess, we found an article on CPPâ€™s: Cell Penetrating Peptides (Chang <I>et al</I> 2014), and decided to attempt to utilize these.<STRONG>Exit from vesicles</STRONG>: If the protein is endocytosed, but not released from the vesicle it can not be utilized in the cell. If we were able to confirm uptake in vesicles, the next step would be to break the vesicles and release the proteins. For this reason, we looked into a way of breaking the vesicles by adding a small peptide from the influenza hemagglutinin HA2, consisting of the 23 N-terminal amino acids (<U><A href="#P3">Erazo-Oliveras, 2012</A></U>). To attempt to break the vesicles, we would attach HA2 to the N-terminal end of the CPP tagged protein. 

                  </P></DIV></DIV></DIV><DIV class="content-section-a"><DIV class="container"><DIV><H2 class="section-heading">References</H2><P class="lead"><A name="P1">Chang, M., Chou, J., &amp; Lee, H. (2005). Cellular Internalization of Fluorescent Proteins via Arginine-rich Intracellular Delivery Peptide in Plant Cells. <I>Plant And Cell Physiology, 46</I>(3), 482-488. http://dx.doi.org/10.1093/pcp/pci046</A><A name="P2">Chang, M., Huang, Y., Aronstam, R., &amp; Lee, H. (2014). Cellular Delivery of Noncovalently-Associated Macromolecules by Cell- Penetrating Peptides. Current Pharmaceutical Biotechnology, 15(3), 267-275. http://dx.doi.org/10.2174/1389201015666140617095415</A><A name="P3">Erazo-Oliveras, A., Muthukrishnan, N., Baker, R., Wang, T., &amp; Pellois, J. (2012). Improving the Endosomal Escape of Cell-Penetrating Peptides and Their Cargos: Strategies and Challenges. Pharmaceuticals, 5(12), 1177-1209. http://dx.doi.org/10.3390/ph5111177</A><A name="P4">Eudes, F., &amp; Chugh, A. (2008). Cell-penetrating peptides. Plant Signaling &amp; Behavior, 3(8), 549-550. http://dx.doi.org/10.4161/psb.3.8.5696
Schmidt, O., Pfanner, N., &amp; Meisinger, C. (2010). Mitochondrial protein import: from proteomics to functional mechanisms. Nature Reviews Molecular Cell Biology, 11(9), 655-667. http://dx.doi.org/10.1038/nrm2959</A><A name="P5">Shi, L., &amp; Theg, S. (2013). The chloroplast protein import system: From algae to trees. Biochimica Et Biophysica Acta (BBA) - Molecular Cell Research, 1833(2), 314-331. http://dx.doi.org/10.1016/j.bbamcr.2012.10.002</A><A name="P6">Truscott, K., Brandner, K., &amp; Pfanner, N. (2003). Mechanisms of Protein Import into Mitochondria. Current Biology, 13(8), R326-R337. http://dx.doi.org/10.1016/s0960-9822(03)00239-2</A></P></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-lg-5"><H2>Find Incell here:</H2></DIV><DIV class="col-lg-7"><UL class="list-inline banner-social-buttons"><LI><A href="https://twitter.com/iGEM_Copenhagen" target="_blank" class="btn btn-default btn-lg"><SPAN class="network-name">Twitter</SPAN></A></LI><LI><A href="https://www.facebook.com/iGEMCopenhagen" target="_blank" class="btn btn-default btn-lg"><SPAN class="network-name">Facebook</SPAN></A></LI><LI><A href="https://www.instagram.com/igemku2017" target="_blank" class="btn btn-default btn-lg"><SPAN class="network-name">Instagram</SPAN></A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV id="page-top" data-spy="scroll" data-target=".navbar-fixed-top"><NAV class="navbar scrollnav-custom navbar-fixed-bottom" role="navigation" id="scrollnav"><DIV class="container"><DIV class="navbar-header"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="#Top">Introduction</A></LI><LI><A class="page-scroll" href="#Design">Final design</A></LI><LI><A class="page-scroll" href="#experiment">Experiments</A></LI><LI><A class="page-scroll" href="#process">Design Process</A></LI></UL></DIV><DIV class="collapse navbar-collapse navbar-right navbar-main-collapse"><UL class="nav navbar-nav"><LI><A class="page-scroll" href="https://2017.igem.org/Team:UCopenhagen/Project">Previous</A></LI><LI><A class="page-scroll" href="https://2017.igem.org/Team:UCopenhagen/Number-Control">Next</A></LI></UL></DIV></DIV></NAV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>