<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TU_Darmstadt_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TU Darmstadt/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="titleBar"><SPAN class="title"><A href="https://2017.igem.org/Team:TU_Darmstadt">ChiTUcare</A></SPAN></DIV><P><TITLE>MainPage</TITLE></P><H1 id="logo"><A href="https://2017.igem.org/Team:TU_Darmstadt">ChiTUcare</A></H1><NAV id="nav"><UL><DIV class="mainmenu"><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project" class="active">Project</A></LI></DIV><DIV class="nav-proj-drop"><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project/chitin_synthase">Chitin Synthase</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project/chitinase">Chitinase</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project/chitin_deacetylase" class="active">Chitin Deacetylase</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project/hydrogel">Hydrogels</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project/chemistry">Chemistry</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project
/parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/project
/notebook">Notebook</A></LI></DIV><DIV class="mainmenu"><DIV class="nav-follow"><DIV class="nav-drop"><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/human_practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/tech">Tech</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/team">Team</A></LI><LI><A href="https://2017.igem.org/Team:TU_Darmstadt/judging">Judging</A></LI></DIV></DIV></DIV></UL></NAV><DIV id="wrapper"><DIV id="main"><SECTION id="one"><DIV class="container"><HEADER class="major"><H2>Chitin Deacetylases NodB and COD</H2></HEADER><DIV class="post-it"><P>Chitosan is a polymeric product of deacetylated chitin, which exists in a wide variety of patterns differing in their degree of deacetylation. Our goal is to design chitosan oligomers with a specific pattern of deacetylation. It can then be used for the production of our hydrogels.We implemented chitin deacetylases originating from the organisms <I>Sinorhizobium meliloti</I> (NodB) and <I>Vibrio cholerae</I> (COD) into our <I>E. coli</I> cells. These enzymes deacetylate chitin individually. NodB targets the first position of the non-reducing end, while COD works similarly on the second unit. By implementing an orthogonal expression system to regulate the patterns, designer chitosan could be adjusted to the respective task. This would allow the expression of each enzyme separately, creating a defined deacetylation pattern.</P></DIV></DIV></SECTION><SECTION id="two"><DIV class="container"><H3>What are Chitin Deacetylases?</H3><P>Chitin deacetylases (CDA) mostly occur in marine bacteria, few in insects, and several in fungi <A href="#[1]">[1]</A>.</P><P>In fungi, for example, CDAs are involved in cell wall formation, sporulation, and catabolism of chitin oligosaccharides. Many plant fungal pathogens secrete CDAs during plant infection. Plants only detect fungal infections by registering chitin. Fungi “turn invisible” by deacetylating chitin into chitosan and thus, outwit the plant defence system <A href="#[2]">[2]</A>.</P><P>The CDAs generate chitosan oligomers from chitin by deacetylating the <I>N</I>-acetylglucosamine units of the substrate <A href="#[3]">[3]</A>. During deacetylation, acetic acid is cleaved off from a glucosamine unit. Some CDAs may even deacetylate chitosan, creating a double deacetylated oligomer <A href="#[2]">[2]</A>.</P><P>Chitin deacetylases belong to the carbohydrate esterase family 4. All family members, including NodB protein and chitin deacetylases, share the same primary structure called “NodB homology domain” or “polysaccharide deacetylase domain” <A href="#[4]">[4]</A>.</P><P>In medical applications and plant protection, CDAs are used for designing antifungal and antibacterial biofilms <A href="#[2]">[2]</A>.</P></DIV></SECTION><SECTION id="three"><DIV class="container"><H3>NodB - <I>Sinorhizobium meliloti</I></H3><H4>Introduction</H4><P>The CDA NodB is isolated from the organism <I>Sinorhizobium meliloti</I> (<I>strain 1021</I>) <A href="#[6]">[6]</A>, which belongs to the family of gram-negative proteobacteria <A href="#[5]">[5]</A>. <I>Rhizobium</I> sp. often form a root endosymbiosis with legumes in nature. Through this, nitrogen assimilation in legumes is provided. For cell signalling the microbial partners and plants exchange diffusible molecules, the so-called nodulation factors (Nod factors) <A href="#[7]">[7]</A>. Belonging to these Nod factors are NodC, NodB and NodA.For further information to NodC visit the following link to the <A href="https://2017.igem.org/Team:TU_Darmstadt/project/chitin_synthase">Chitin Synthase</A>.
The <I>nodB</I> gene is 653 base pairs long and translates into a hydrolase with a molecular weight of approximately 24.4 kDa <A href="#[2]">[2]</A>.
The enzyme works optimally in surroundings with a pH of 9 and temperatures reaching 37 °C <A href="#[2]">[2]</A>. It solely deacetylates the first position of the non-reducing end in a chitin oligomer <A href="#[2]">[2]</A>. Due to regioselectivity <A href="#[7]">[7]</A>, monomers are not deacetylated by NodB, therefore chains of dimers up to hexamers are converted to mono-deacetylated chitosan oligomers <A href="#[2]&lt;/a&gt;.&lt;br&gt;
After expression of &lt;i&gt;nodB&lt;/i&gt;, the enzyme accumulates within the cell in the form of inclusion bodies, which are normally insoluble and non-functional &lt;a href=">[7]</A>.
If NodB is incubated for a long time with the substrate and high enzyme concentrations, the possibility of double-deacetylated oligomers arises. However, the emerging amount is insignificant <A href="#[2]">[2]</A>.
<FIGURE><CENTER><FIGCAPTION><B>Figure 1. Structure of NodB.</B> Modeled from SwissModel. A 3D simulation showcasing the enzyme on molecular level. <A href="#[8]">[8]</A></FIGCAPTION></CENTER></FIGURE></P><H4>Mechanism</H4><P>As explained before, CDAs occur in many different organisms and produce chitosan out of chitin to outwit plant defense systems.
NodB deacetylates the first N-acetyl-D-glucosamine unit (GlcNAc) of the non-reducing end <A href="#[2]">[2]</A>. Deacetylation describes hydrolysis of the acetamido group of the GlcNAc units, thus generating acetic acid und D-glucosamine (GlcN) <A href="#[1]">[1]</A>.

<FIGURE><CENTER><FIGCAPTION><B>Figure 2. Mechanism of NodB.</B> The enzymatic hydrolization occurs at the first position of the non-reducing end. </FIGCAPTION></CENTER></FIGURE></P><H4>Methods</H4><H5>Cloning and Expression</H5><P>We ordered the <I>nodB</I> gene via IDT sequencing and inserted this gene into the pSB1C3 vector through a  BioBrick system and verified this via sequencing. The <I>nodB</I> was fused to an Anderson-promoter with defined cleavage sites <A href="http://parts.igem.org/Part:BBa_K2380025">(BBa_K2380025)</A>. The vector includes the RBS <A href="http://parts.igem.org/Part:BBa_K2380024">BBa_K2380024</A>.</P><FIGURE><CENTER><FIGCAPTION><B>Figure 3. Plasmid card of NodB without a promoter.</B> The <I>nodB</I> gene (in green) is located on the pSB1C3 vector without a promoter system. It is a basic part and can be used in combination with other parts. Biobrick (BB) prefix and suffix are shown in light blue. <A href="http://parts.igem.org/Part:BBa_K2380041">[BBa_K2380041]</A></FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION><B>Figure 4. Plasmid card of NodB with Anderson promoter.</B>The <I>nodB</I> gene (in green) is located on the pSB1C3 vector with an Anderson promoter (red) and a RBS (grey). This part is used for expression studies without the necessity of an induction. Biobrick (BB) prefix and suffix are shown in light blue. <A href="http://parts.igem.org/Part:BBa_K2380042">[BBa_K2380042]</A></FIGCAPTION></CENTER></FIGURE><P>Before cloning the <I>nodB</I> gene on pSB1C3, the gene was cloned in a pUPD-vector containing a T7-promoter. <I>nodB</I> gene was successfully expressed in <I>E. coli</I> BL21 after induction with 1 mM isopropyl β-D-1-thiogalactopyranoside (IPTG) and analysed via SDS-Page. NodB is known to aggregate inclusion bodies within <I>E. coli</I> cells <A href="#[7]">[7]</A>. Inclusion bodies are non-functional, insoluble aggregates that occur during overexpression <A href="#[7]">[7]</A>.  

There are many advantages with the occurrence of inclusion bodies. On one hand they yield a very high protein expression level, on the other, they are easy to isolation from cells because of their difference in size and density in comparison to cellular organelles. Inclusion body proteins also degrade slower and resist proteolytic attacks by proteases. Lastly, proteins expressed in inclusion bodies tend to be less contaminated with cellular substances and can be purified with less steps <A href="#[12]">[12]</A>.</P><H5>Purification and Refolding</H5><P>To purify the enzyme, a C-terminal-directed mutagenesis was performed to add a His-taq. The <I>nodB</I> gene was again expressed in <I>E. coli</I> BL21. The cells were disrupted via sonication and in a second sonication step the inclusion bodies were solubilized in a highly concentrated guanidine hydrochloride buffer containing β-mercaptoethanol. High concentrations of chaotropic denaturants such as guanidine hydrochloride and reducing agents (β-mercaptoethanol) provide the solubilization of inclusion bodies <A href="#[12]">[12]</A>. To prevent non-native disulfide bond formations β-mercaptoethanol was used. To refold the bound protein, the 6 M guanidine-hydrochloride buffer was exchanged to a 6 M urea buffer. During slow removal of the denaturant urea, the protein was refolded into the native state <A href="#[12]">[12]</A>.
Affinity purification of His-tagged recombinant proteins and subsequent on-column refolding, gives us the feasibility to purify and refold NodB in a single chromatographic step <A href="#[13]">[13]</A>. 
The protein was purified via an ÄKTA in combination with a 1 mL HisTrap column (GE Healthcare Bio-Sciences AB, Uppsala, Sweden).</P><H5>Enzyme Reaction and Assay</H5><H6>Reaction</H6><P>A reaction volume was set to 1 ml with 1 mM chitin pentamers (AG Prof. Dr. Bruno Moerschbacher, WWU Münster, Institut für Biologie und Biotechnologie der Pflanzen) and 2,5 µM purified NodB eluted in NH<SUB>4</SUB>HCO<SUB>3</SUB> adjusted to pH 9. Four reactions were initiated and incubated overnight at 37 °C.
After approximately 20 hours the reactions were heat-inactivated at 80 °C for 20 minutes to denature remaining enzymes.</P><H6>Assay</H6><P>
NodB is a carbohydrate esterase which deacetylates the first N-acetyl-D-glucosamine unit (GlcNAc) of the non-reducing substrate end <A href="#[4]">[4]</A>. Deacetylation describes hydrolysis of the acetamido group in the GlcNAc-units, thus generating acetic acid und D-glucosamine (GlcN) <A href="#[1]">[1]</A>.
Acetic acid was indirectly detected via an acetic acid assay kit (Acetate Kinase Manual Format, Megazyme, Bray, Ireland). NADH consumption is measured at 340 nm wavelength. Thus, the amount of acetic acid is stoichiometric with the amount of NAD<SUP style="font-size: 0.9em;">+</SUP> of the last reaction step.
<FIGURE><CENTER><FIGCAPTION><B> Figure 5.</B> Single reaction steps of the acetic acid assay kit (Acetate Kinase Manual Format, Megazyme, Bray, Ireland) </FIGCAPTION></CENTER></FIGURE></P><H4>Results and Discussion</H4><H5>Expression and Purification</H5><P>The <I>nodB</I> gene was successfully expressed in <I>E. coli</I> BL21 after induction with 1 mM isopropyl β-D-1-thiogalactopyranoside (IPTG). The expression temperature was reduced to 30 °C. 
Analysis utilizing sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) revealed single bands for the NodB enzyme (approximately 24,4 kDa). Results are shown in figure 6.
After tagging the protein with an His-tag, we performed purification and refolding through an ÄKTA pure system and verified the success through another SDS-Page. </P><FIGURE style="width: 100%"><FIGCAPTION><B>Figure 6. SDS Pages of NodB. </B> On the left: SDS-PAGE of chitin deacetylase <I>nodB</I> protein. The arrow marks the region of expression. From left to right: <I>E. coli</I> BL21 transformed with pSB1C3-nodB, <I>E. coli</I> BL21 transformed with pUPD-nodB after being induced with IPTG for 3h, 6h and 24h, non transformed BL21 parallel to induced cultures and 24h after, <I>E. coli</I> BL21 transformed with pSB1C3-nodB with and without (#) regulatory elements parallel to induced cultures. Usage of PageRuler Prestained Protein Ladder 10 to 180 kDa from ThermoFischer Scientific. 
On the right: SDS-PAGE of purified chitin deacetylase <I>nodB</I>. The arrow marks the region of expression. Fractions 1-3 show the first flow-through, while Fractions 4-8 are from Elution. Usage of PageRuler Prestained Protein Ladder 10 to 180 kDa from ThermoFischer Scientific. 
</FIGCAPTION></FIGURE><H5>Enzyme Reaction and Assay</H5><P>The following figures show data in which the background of chitin is already detracted from the measured results. Since the blank showed auto-hydrolysis of ATP, resulting in ADP, they were subtracted as well.
The measurement of four replicate enzyme reactions shows an indirect detection of acetic acid via NAD<SUP>+</SUP> absorbance. This proves that NodB has generated acetic acid during deacetylation of chitin into chitosan.</P><FIGURE><CENTER>4HCO<SUB>3</SUB> adjusted to pH 9 at 37 °C. Afterwards these were heat-inactivated for 20 minutes at 80 °C. The meausrement with Tecan200 infinte Pro plate reader ranged over 76 minutes.&quot;, width=70%,&gt;
<FIGCAPTION><B>Figure 7.</B> Detected acetic acid after 76 minutes in 0.2 x, 0.15 x and 0.1 x sample volume generated by NodB and in 60% standard solution. Four samples with 1 mM chitin pentamers and 2,5 µM purified NodB were previously incubated overnight in NH<SUB>4</SUB>HCO<SUB>3</SUB> adjusted to pH 9 at 37 °C. Afterwards these were heat-inactivated for 20 minutes at 80 °C. The measurement with <I>Tecan200 infinte Pro</I> plate reader ranged over 76 minutes.</FIGCAPTION></CENTER></FIGURE><P>Approximately 10 µg acetic acid in 0.2 x sample volume, 6 µg in 0.15 x sample volume and 4 µg in 0.1 sample volume were detected (figure 7). The 60 % standard solution resulted in 26 µg acetic acid, about 6.5 times more acetic acid as in our samples.</P><FIGURE><CENTER><FIGCAPTION><B>Figure 8.</B> Detailed view on the sample data of figure 7. Detected acetic acid after 76 minutes in 20%, 15% and 10% sample volume generated by NodB. Four samples with 1 mM chitin pentamers and 2,5 µM purified NodB were previously incubated overnight in NH<SUB>4</SUB>HCO<SUB>3</SUB> adjusted to pH 9 at 37 °C. Afterwards these were heat-inactivated for 20 minutes at 80 °C. The measurement with <I>Tecan200 infinte Pro</I> plate reader ranged over 76 minutes.</FIGCAPTION></CENTER></FIGURE><P>4 µg acetic acid were detected in the 0.1 x sample volume. In the 0.15 x sample volume the amount of acetic acid was 1.5 times higher and in the 0.2 x sample volume it was 2.5 times higher (figure 8).</P><FIGURE><CENTER><FIGCAPTION><B>Figure 9.</B> Final extrapolated amount of detected acetic acid in 1 ml sample after 76 minutes. The extrapolation mean of the measurements with <I>Tecan infinite200 Pro</I> plate reader is also shown.</FIGCAPTION></CENTER></FIGURE><P>About 1 mg of chitin was used as substrate. If chitin is completely deacetylated by NodB into chitosan about 60 µg acetic acid are expected to be produced. In our results about 50 µg acetic acid were produced.  Consequently, approximately 83 % chitin has been deacetylated. This extrapolated result is shown in figure 9.</P><H5>Conclusion</H5><P>We verified the expression of NodB in <I>E. coli</I> BL21 with a SDS-Page. After tagging the protein with a His-tag, we performed purification and refolding through an ÄKTA pure system, and verified the success through another SDS-Page.
To test whether NodB works properly, we used the acetic acid assay kit (Acetate Kinase Manual Format, Megazyme, Bray, Ireland). NodB deacetylates chitin to create chitosan. This chemical step releases acetic acid. As already explained in the methods, the amount of acetic acid is indirectly measured via amount of NAD<SUP>+</SUP>. Thus, the amount of acetic acid is stoichiometric with the amount of NAD<SUP>+</SUP> in the last reaction step. NADH consumption is measured at 340 nm.

As the graphs show, this verification was successful and indicates that NodB was refolded properly and is present in its active form. 
We are thankful for the advise Prof. Dr. Bruno Moerschbacher (also see <A href="https://2017.igem.org/Team:TU_Darmstadt/human_practices/integrated">integrated Human Practices</A>) gave us concerning NodB purification and the recommendation to use the acetic acid assay kit from Megazyme. After successfully purifying NodB, we would appreciate having the opportunity to share our results with his research group and eventually optimize our purification process. The many reaction steps of the kit result in somewhat inaccurate data. Additionally, we would need to try different acetic acid assay kits to evaluate the best one for our purpose. The measurement of background from NodB interaction with the assay reactions is another control, which is needed to be measured to complete our calculation.
At some point in the future, mass spectrometry is another point that has to be tackled as well.
Retrospectively, hoping to achieve even better purification results, we would like to try to express <I>nodB</I> in colder conditions for about 40 hours <A href="#[3]">[3]</A>. The expression at lower temperatures enables us to express <I>nodB</I> without the need of refolding from inclusion bodies.</P></DIV></SECTION><SECTION id="four"><DIV class="container"><H3>COD -<I>Vibrio Cholerae</I></H3><H4>Introduction</H4><P>In order to bring more variability in our produced chitosans, we decided to implement a second chitin deacetylase. Because of its bacterial origin, we picked COD isolated from the gram-negative organism <I>Vibrio cholerae</I> O1 biovar El Tor str. N16961 <A href="#[10]">[10]</A>. It is not part of its pathogenic activity. During our conversation with Professor Moerschbacher, he informed us that the expression of COD in <I>E. coli</I> has been successfully performed by himself and his group. In addition, it was already proven that both NodB and COD could be expressed in the same organism <A href="#[2]">[2]</A>.
The <I>cod</I> gene is 1296 base pairs long and translates into a hydrolase with a molecular weight of approximately 45.5 kDa <A href="#[2]">[2]</A>.
The enzyme works optimally in surroundings with a pH of 8 and temperatures reaching 45 °C <A href="#[2]">[2]</A> <A href="#[4]">[4]</A>.
As mentioned before, deacetylases target different units in a chitin molecule. Which unit is deacetylated depends on the chosen enzyme. In the case of COD, the second position from the non-reducing end is deacetylated <A href="#[2]">[2]</A> <A href="#[4]">[4]</A> <A href="#[9]">[9]</A>. If both enzymes – COD and NodB – are active at once we are able to create a deacetylation pattern involving the first two units.
In contrast to NodB, COD does not deacetylate chitosan twice, after long incubation periods <A href="#[2]">[2]</A>.
CODs catalytic part is its N-terminal domain, while the other two domains make up carbohydrate-binding molecules. The catalytic domain correlates to a carbohydrate esterase domain (CDA) <A href="#[4]">[4]</A>.
<FIGURE><CENTER><FIGCAPTION><B>Figure 10. Structure of COD.</B> Structural data from RCSB PDB. A 3D simulation of the enzymes molecular structure. <A href="#[11]">[11]</A></FIGCAPTION></CENTER></FIGURE></P><H4>Mechanism</H4><P>COD deacetylates the second N-acetyl-D-glucosamine unit (GlcNAc) of the non-reducing end <A href="#[2]">[2]</A>, thus generating acetic acid and D-glucosamine (GlcN). 
<FIGURE><CENTER><FIGCAPTION><B>Figure 11. Mechanism of COD.</B> The enzyme deacetylates at the second position of the non-reducing end. </FIGCAPTION></CENTER></FIGURE></P><H4>Methods</H4><P>We ordered the <I>cod</I> gene via IDT sequencing and inserted this gene into the pSB1C3 vector through a BioBrick system and verified this via sequencing. 
The <I>cod</I> was fused to a T7-promoter <A href="http://parts.igem.org/Part:BBa_I719005">(BBa_I719005)</A>. The vector includes the RBS <A href="http://parts.igem.org/Part:BBa_K2380024">BBa_K2380024</A>.
</P><FIGURE><CENTER><FIGCAPTION><B>Figure 12. Plasmid card of COD without a promoter.</B> The <I>cod</I> gene (in yellow) is located on the pSB1C3 vector without a promoter system. It is a basic part and can be used in combination with other parts. Biobrick (BB) prefix and suffix are shown in light blue. <A href="http://parts.igem.org/Part:BBa_K2380044">[BBa_K2380044]</A></FIGCAPTION></CENTER></FIGURE><FIGURE><CENTER><FIGCAPTION><B>Figure 13. Plasmid card of COD with a T7-promoter.</B> The <I>cod</I> gene (in yellow) is located on the pSB1C3 vector with a T7 promoter (marine blue) and a RBS (grey). Biobrick (BB) prefix and suffix are shown in light blue. <A href="http://parts.igem.org/Part:BBa_K2380043">[BBa_K2380043]</A></FIGCAPTION></CENTER></FIGURE><P>We induced the BL21 cells, containing the pSB1C3 vector with the T7 promoter system, with IPTG to start expression. In order to validate the successful expression, we performed a SDS-Page.</P><H4>Results and Discussion</H4><P>The <I>cod</I> gene was successfully expressed in <I>E. coli</I> BL21 after induction with 1 mM isopropyl β-D-1-thiogalactopyranoside (IPTG). The expression temperature was reduced to 30 °C. 
Analysis utilizing sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE), shown in figure 14, revealed single bands for the COD enzyme (approximately 45.5 kDa). 
Further approach would include adding His-tags and purification via affinity chromatography. To test the enzyme activity the amount of released acetic acid during enzymatic reaction can be measured, using an acetic acid assay kit, or via thin-layer chromatography. 
Considering the importance of using two CDAs, an orthogonal expression system could be implemented to create multiple patterns of deacetylation.
<FIGURE><CENTER><FIGCAPTION><B>Figure 14. SDS-Page of COD.</B> Protein analysis of the enzyme COD via SDS-Page. The arrow marks the region of expression. 
From left to right: Non transformed <I>E. coli</I> BL21 24 h after and simultaneous (0 h) to induced cultures, <I>E. coli</I> BL21 transformed with pUPD-nodB after being induced with IPTG for 24 h, 6 h, 3 h and before induction (0 h). Usage of PageRuler Prestained Protein Ladder 10 to 180 kDa from ThermoFischer Scientific.</FIGCAPTION></CENTER></FIGURE></P></DIV></SECTION><SECTION id="five"><DIV class="container"><H3>Designer Future</H3><P>Since chitin oligomers are deacetylated at specific positions by different deacetylases, the usage of different enzymes makes it possible to create chitosans with defined patterns and special properties. While our project only involves two deacetylases at this time, others could be introduced as well, giving us the opportunity to create varying patterns. To accomplish this future goal, we would require an orthogonal expression system. Unregulated parallel usage of several CDAs only results in chitosan oligomers that are deacetylated by all enzymes. If they could be individually induced, each combination of CDAs may be expressed and different chitosan oligomers could be constructed for various applications. To achieve an effective regulation, we have been searching for a suitable system. During this, we have stumbled upon a solution in a modern research issue – the T7-split-RNA-Polymerases.

The general concept is to split T7-RNAPs into a C-terminal and N-terminal domain. The biological activity of the split T7-RNAPs is restored when both fragments are in direct proximity to each other. The assembly of the split fragments can be made dependent on the dimerization of fusion proteins. Only when both domains are expressed the proteins can dimerize, reassembling the RNAP. If inducible dimerization systems are chosen as fusion proteins the dimerization must additionally be induced, for instance by small molecules or light.</P><FIGURE><CENTER><FIGCAPTION><B>Figure 15. </B>  The orthogonal expression of the two CDAs would result in the production of three different chitosans.</FIGCAPTION></CENTER></FIGURE><P>
Mutants of the T7-RNAP carrying amino acid substitutions in the region that mainly determine the polymerase’s promoter specificity address altered T7 promoter (P<SUB>T7</SUB>) sequences. This would allow the expression of the genes of interest orthogonally, when placing several orthogonal variants of T7 promoter in front of the gene. These promoters can be placed upstream of the genes of the different CDAs. Via chemical or light induction the dimerizations of fused dimer systems mediate the reassembly of various split T7-RNAPs, that are orthogonally addressing variants of P<SUB>T7</SUB>. Thus, a orthogonal regulatory system is established.

We have not been able to introduce this system into our project and the iGEM Registry of Standard Biological Parts yet, but we are planning to facilitate such a system in our next years project. </P></DIV></SECTION><SECTION id="six"><DIV class="container"><H3>Group Picture</H3><P><FIGURE><CENTER><FIGCAPTION><B>Group Picture of the Chitin Deacetylase team. </B>Back row, from left to right: Cristina Kurzknabe, Jennifer Zimmermann, Sophia Hein, Elena Nickels, Bea Spiekermann  Front row, from left to right: Feodor Belov, Claudia Kreher, Daniel Kelvin, Lara Steinel, Tim Maier </FIGCAPTION></CENTER></FIGURE></P></DIV></SECTION><SECTION id="seven"><DIV class="container"><H3>References</H3><P><TABLE class="ref"><TBODY><TR><TD id="[1]">[1]</TD><TD>Zhao, Y. <I>et.al.</I> , Park, R.-D., Muzzarelli, A.A. (2010) Chitin Deacetylases: Properties and Applications; <I>Marine Drugs</I>, 8(1), 24-46;
DOI: 10.3390/md8010024
</TD></TR><TR><TD id="[2]">[2]</TD><TD>Hamer, S.N. <I>et.al.</I> Enzymatic production of defined chitosan oligomers with a specific pattern of acetylation using a combination of chitin oligosaccharide deacetylases(2015); <I>Sci. Rep. 5</I>, 8716;DOI:10.1038/srep08716 </TD></TR><TR><TD id="[3]">[3]</TD><TD>Hamer, S.N. <I>et.al.</I>, Moerschbacher, B. M., Kolkenbrock, S. (2014) Enzymatic sequencing of partially acetylated chitosan oligomers; <I>Carbohydrate Research</I>, 392, 16–20;
DOI: 10.1016/j.carres.2014.04.006
</TD></TR><TR><TD id="[4]">[4]</TD><TD>Andrés, E. <I>et.al.</I>, Albesa-Jové, D., Biarnés, X., Moerschbacher, B.M., Guerin, M., Planas, A. (2014) Structural Basis of Chitin Oligosaccharide Deacetylation; <I>Angewandte Chemie International Edition</I>, 53, 6882-6887;DOI: 10.1002/anie.201400220</TD></TR><TR><TD id="[5]">[5]</TD><TD>Gargaud M. <I>et.al.</I>, Amils, R., Cernicharo Quintanilla, J. ,  Cleaves II, H.J., Irvine, W.M., Pinti, D., Viso, M.  (Eds.) (2011) Encyclopedia of Astrobiology, Springer-Verlag Berlin Heidelberg;DOI: 10.1007/978-3-642-11274-4</TD></TR><TR><TD id="[6]">[6]</TD><TD>Bateman, A., Wu, C., Xenarios, I.; UniProtKB - P02963 (NODB_RHIME); <A href="http://www.uniprot.org/uniprot/P02963">http://www.uniprot.org/uniprot/P02963</A>; last visited: 10/19/2017</TD></TR><TR><TD id="[7]">[7]</TD><TD>Chambon, R., Pradeau, S., Fort, S., Cottaz, S., Armand, S. (2011) High yield production of Rhizobium NodB chitin deacetylase and its use for in vitro synthesis of lipo-chitinoligosaccharide precursors; <I>Carbohydrate Research</I> 442, 25-30;DOI: 10.1016/j.carres.2017.02.007</TD></TR><TR><TD id="[8]">[8]</TD><TD>Biasini M, Bienert S, Waterhouse A, Arnold K, Studer G, Schmidt T, Kiefer F, Cassarino TG, Bertoni M, Bordoli L, Schwede T (2014). SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information Nucleic Acids Research 2014 (1 July 2014) 42 (W1): W252-W258

Bordoli, L., Kiefer, F., Arnold, K., Benkert, P., Battey, J. and Schwede, T. (2009) Protein structure homology modelling using SWISS-MODEL Workspace. Nature Protocols, 4,1.

Arnold K, Bordoli L, Kopp J, and Schwede T (2006) The SWISS-MODEL Workspace: A web-based environment for protein structure homology modelling. Bioinformatics.,22,195-201.

Bienert S, Waterhouse A, de Beer TA, Tauriello G, Studer G, Bordoli L, Schwede T (2017) The SWISS-MODEL Repository - new features and functionality Nucleic Acids Res. 45(D1):D313-D319.

Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009) The SWISS-MODEL Repository and associated resources. Nucleic Acids Res. 37, D387-D392.

Kopp J, and Schwede T (2006). The SWISS-MODEL Repository: new features and functionalities. Nucleic Acids Res.,34, D315-D318.

Guex, N., Peitsch, M.C. Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective. Electrophoresis, 30(S1), S162-S173. </TD></TR><TR><TD id="[9]">[9]</TD><TD>Li, X., Wang, L., Wang, X., Roseman, S. (2007) The Chitin Catabolic Cascade in the Marine Bacterium Vibrio Cholerae: Characterization of a Unique Chitin Oligosaccharide Deacetylase, <I>Glycobiology</I>, vol. 17, Issue 12, 1377–1387; DOI: 10.1093/glycob/cwm096</TD></TR><TR><TD id="[10]">[10]</TD><TD>National Center for Biotechnology Information, U.S. National Library of Medicine (NCBI); Vibrio cholerae O1 biovar eltor str. N16961 chromosome I, complete sequence, GenBank: AE003852.1; <A href="https://www.ncbi.nlm.nih.gov/nuccore/AE003852.1?from=1355388&amp;to=1356683&amp;sat=4&amp;sat_key=105780702">https://www.ncbi.nlm.nih.gov/nuccore/AE003852.1?from=1355388&amp;to=1356683&amp;sat=4&amp;sat_key=105780702</A>; last visited: 09/01/2017</TD></TR><TR><TD id="[11]">[11]</TD><TD>Image of 4NZ1 (Andres, E., Albesa-Jove, D., Biarnes, X., Moerschbacher, B.M., Guerin, M.E., Planas, A(2014) sturcture of Vibrio cholerae chitin de-N-acetylase at 2.05 A resolution.) created with The PyMOL Molecular Graphics System, Version 2.0 Schrödinger, LLC.</TD></TR><TR><TD id="[12]">[12]</TD><TD>Mohan Singh, S., Kumar Panda, A. (2015) Review: Solubilization and Refolding of Bacterial Inclusion Body Proteins; <I>Journal of bioscience and bioengineering</I> Vol. 99, No. 4, 303–310.
DOI: 10.1263/jbb.99.303</TD></TR><TR><TD id="[13]">[13]</TD><TD>GE Healthcare Bio-Sciences AB, Uppsala, Sweden (2007), Rapid and efficient purification and refolding of a (histidine) 6-tagged recombinant protein produced in E. coli as inclusion bodies; <I>Application note</I> 18-1134-37 AC;
<A href="https://www.gelifesciences.co.jp/catalog/pdf/18113437ac.pdf">https://www.gelifesciences.co.jp/catalog/pdf/18113437ac.pdf</A>; last visited: 10/09/2017 </TD></TR></TBODY></TABLE></P><DIV class="container"><P style="text-align: center;"><A href="http://www.igem.tu-darmstadt.de/igem_2/projekt_2/index.de.jsp">iGEM Technische Universität Darmstadt</A></P><P style="text-align: center;">We thank our sponsors, instructors and the iGEM organization!</P><P style="text-align: center;">Design: <A href="http://html5up.net">HTML5 UP</A></P></DIV></DIV></SECTION></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>