Team:TU Darmstadt/project/regulation system
ChiTUcare
MainPage
ChiTUcare
Project
Chitin Synthase
Chitinase
Chitin Deacetylase
Regulatory System
Hydrogels
Chemistry
Parts
Notebook
Human Practices
Tech
Team
Judging
Orthogonal Split T7‑RNA‑Polymerase Regulatory System
As applications of synthetic biology are growing in complexity, the design of orthogonal regulatory systems is becoming increasingly important. To realise the idea of producing designer chitosan with defined deacetylation patterns for different applications, it is necessary to implement a regulatory system for orthogonal expression of the chitin deacetylases. For this, we use a system based on Split-T7-RNA-Polymerases.These polymerases are split into an N- and C-terminal domain. They rearrange after being induced to with a specific signal, which is blue light for one, and rapamycin for the other polymerase. Both are only functional after assembly of the two domains.The two polymerases are each specific for a different promoter, which can be placed before a gene of interest. Because of this, each polymerase will only lead to the expression of one gene. Consequently, it is possible to express different genes orthogonally. It is also possible to implement more genes and more different split-polymerases.
Introduction
As applications of synthetic biology become wider and more complicated, the size of artificial genetic circuits increases continuously. One significant limitation for engineering of complex genetic circuits is the number of orthogonal regulatory systems that can be easily applied on the elements of the genetic circuit. In terms of our project chiTUcare, for example, the in vivo production of chitosan oligomers (deacetylated chitin oligomers) with defined deacetylation patterns requires an orthogonal regulatory system. Here, orthogonal expression of the chitin deacetylases (CDAs) [] is necessary, since different CDAs mediate different deacetylation patterns. Unregulated parallel usage of several CDAs only results in chitosan oligomers that were deacetylated by all CDAs. If all CDAs can be individually induced, each combination of CDAs may be expressed and various chitosan oligomers can be constructed. With the intention to expand the library of regulatory parts as well as to establish a regulatory system for our CDAs we here introduce a new gene regulatory system into the iGEM Registry of Standard Biological Parts based on the work of Pu et al. (2017) that focused on split T7‑RNA‑Polymerases [2017 Paper].
T7‑RNA‑Polymerase (T7‑RNAP) can be split into two fragments between residue 179 and 180 under conservation of its biological activity when both fragments are in direct proximity [2013]. Beyond that, it was shown that T7‑RNAP features a specificity loop between residues 742 and 773 that mainly determines the polymerase’s promotor specificity [1998]. Mutants carrying amino acid substitutions in this region show specificity to altered T7‑Promotor (PT7) sequences [2014]. Thus, a library of split T7‑RNAP that orthogonally address variants of PT7 can be set up for gene regulation purposes, especially for engineering synthetic transcriptional AND‑Gates [2013]. By protein engineering of the N‑terminal T7‑RNAP fragment via directed evolution Pu et al. (2017) made the reassembly of the split fragments strongly dependent on the dimerization of fusion proteins that were fused to each of the fragments. Only upon dimerization of the fused domains the T7‑Polymerases will reassemble, address its associated promotors and start transcribing downstream genetic elements.
Figure. 1: Structure of rapamycin induced Split-T7-RNA-Polymerase with FKBP/FRB-system, red: T7-N-Term, green: T7-C-Term, cyan: FRB, purple: FKBP
Since auto‑assembly of the split fragments is thus less probable and there are various proteins with inducible dimerization reactions, the engineered split T7‑RNAPs constitute new artificial and inducible transcriptional activators [2017]. Possible fused domains here are the chemically induced dimerization (CID) system of FK506 binding protein (FKBP) and FKBP‑Rapamycin binding domain (FRB) as well as the improved light‑induced dimer (iLID‑nano) system. The Dimerization of the FKBP/FRB system is chemically induced by the small molecule rapamycin whereas dimerization of the iLiD‑nano system is induced by blue light (488 nm) [FKBP Paper, nano iLID Paper]. Upon induction by blue light or rapamycin the split T7‑RNAP reassemble and their transcriptional activity is restored [2017].
In the framework of the chiTUcare‑project we aim to establish a split T7‑RNAP regulation system for two different CDAs. By efficient orthogonal regulation of the CDAs the biosynthesis of Chitosan oligomers with three different deacetylation patterns is possible (figure x). Since two enzymes shall be orthogonally regulated, two different split T7‑RNAP with unequal fused dimerization systems are necessary. As reassembly‑mediating fusion proteins the iLID‑nano system and the FKBP/FRB system were chosen.  The N‑terminal domain of T7‑RNAP (1‑179) engineered by Pu et al. (2017) was fused to FRB (N(T7)‑FRB) and to nano‑iLID (N(T7)‑iLID) [2017]. Since the C‑terminal domain of the split T7‑RNAP (180+) mediates the promotor specificity, two different C‑terminal domains were used. The native C‑terminus of T7‑RNAP was fused behind sspB (sspB‑C(T7)) while the C‑terminus of the T7‑RNAP variant N4 [N4 Paper] was fused to FKBP (FKBP‑C(N4)). As promotors for the CDAs the native PT7 and PN4 were used correspondingly (Figure 2).
Figure. 2: The orthogonal regulation of the chitin deacetylases enables the production of designer chitosan. red: T7-N-Term, green: T7-C-Term (N4-mutant), yellow: wt-T7-C-Term
Mechanism
The two polymerases we are using for the regulatory system are not functional while split. First they have to dimerize, which is mediated by the specific fusion domains. They join together after being triggered with a special inducing signal.
For the FKBP/FRB system the inducer is rapamycin, while blue light induces the nano-iLID system. The fusion of these parts results in a reassembly of the T7-Polymerases they are linked with, creating a functional protein. [PAPER: Pu et al.]
Caused by the specificity of the polymerases for different promoter sequences, it is possible to orthogonally express different genes of interest. The PT7 and the PN4 can be placed upstream of different genes, that should be expressed ortogonally. The Split-Polymerases themselves will be controlled with an Anderson-promoter. This causes them to be expressed continuously, but only start working after induction. The specific induction of one polymerase results in the production of RNA sequences, coding for the gene of interest, and consequently in the expression of the corresponding enzyme.
Figure. 3: mechanism of orthogonal regulation
In our case, the genes to be regulated are our two chitin deacetylases. The PT7 will control the COD, while the PN4 will be used for the CDA nodB.
In conclusion, it is possible to decide which chitin deacetylase is expressed simply by adding rapamycin or by using blue light. It is even possible to induce both at the same time. With this system, we are able to produce chitosan with three different deacetylation patterns.
This regulatory system can be used for any genetic circuits that require orthogonal expression.
For further information about how the two chitin deacetylases work, visit the following link:
Chitin Deacetylase
Design: HTML5 UP
