Team:TU Darmstadt/project/chitinase
ChiTUcare
MainPage
ChiTUcare
Project
Chitin Synthase
Chitinase
Chitin Deacetylase
Hydrogels
Chemistry
Parts
Notebook
Human Practices
Tech
Team
Judging
Chitinase A1
The chitinase is an enzyme whose ability to break down glycosidic bonds in chitin brings more variability into the molecules. Since not just the grade and pattern of deacetylation, but also the amount of connected chitin monomers influences the entire molecule´s behavior [1] , there is a great limitation of the properties and the bioactivity of the products. Its possible implementation in the project shows the future prospects of how chitins and chitosans with all kind of properties can be produced in E. coli.
Introduction
Chitin is a molecule found in fungal cell walls [1] . Many plants possess enzymes, so-called chitinases, which are able to break down chitin and thus help along with its digestion. These enzymes play a role in defense mechanisms of plants in case of fungal infections [2] . Even in human tissues chitinases appear, where they defend us against parasites [3] . Chitinases break down the glycosidic bonds between chitin monomer units and are classified as hydrolases.
In our project, we decided to focus mainly on chitosan pentamers, synthesized and deacetylated by NodC and NodB respectively. However, chitosan's bioactivity and properties are also defined by its polymer length. For this reason, we want to give a first look at how our project could be extended.
The enzyme we use is the ChiA1 from Bacillus circulans [4] . In the strain WL-12, the gene chiA encodes, together with chiB, chiC and chiD, an entire chitinase-system which primarily degrades chitin. This system is made up of at least six different chitinases [5] , in which ChiA1 is believed to be the key enzyme [6] .
The chiA gene is 1983 base pairs long and translates into an enzyme with a molecular weight of approximately 69.5 kDa [7] .
Since ChiA1 has been successfully expressed together with our chitin synthase NodC [8] , we originally decided on this specific enzyme. Its enzymatic activity has also previously been tested on chitin pentamers. ChiA1 breaks down these chitin pentamers in two dimers and one N-acetylglucosamine (GlcNAc) unit [8] . Its implementation would then give us a bigger variety of chitosan molecules.
The N-terminal domain in the ChiA1 is responsible for its catalytic activity. The C-terminal domain plays an important role in the hydrolysis of chitin, and is the reason ChiA1 has such a high affinity to the substrate [6] [8] [9] .
Figure 1. Structure of ChiA1. The Model shows the 3D structure of ChiA1. Structural Data from RCSB PDB. [10] .
Mechanism
A Chitinase's function is to break down chitin-oligomers into smaller molecules. ChiA1 turns chitinpentaose into two molecules of chitinbiose and one molecule of GlcNAc, by attacking the glycosidic bonds between the monomers [8] .
Figure 2. Mechanism of ChiA1. ChiA1 breaks down the glycosidic bonds between the GlcNAc units of a chitin pentamer, thus creating chitin dimers.
Methods
We ordered the chiA gene via IDT sequencing. First, we inserted this gene into the pUPD vector using a GoldenBraid assembly as this is a simple and fast cloning method [11] . For cloning the chiA gene into the pSB1C3 vector, we used the BioBrick system [12] .
After cutting the pUPD vector, which contains the chiA gene, and the pSB1C3 vector, with the restriction enzymes XbaI and PstI, dephosphorylating the backbone and subsequent ligation, we inserted the chiA gene succesfully into the pSB1C3 vector with an Anderson promoter with defined cleavage sites (BBa_K2380025) .
We used the same protocols inserting the chiA gene successfully into the pSB1C3 vector with an inducible promoter system. We altered the restriction enzymes to NheI and PstI for the pUPD vector with the chiA gene in order to exchange the Anderson promoter which is located on the pUPD vector. We have cut the pSB1C3 vector containing an AraC promoter system (BBa_K808000) with the restriction enzymes SpeI and PstI, dephosphorylated it and succesfully inserted the chiA gene via subsequent ligation. As NheI and SpeI are complementary the finalized construct does contain the BioBrick retriction sites.
Figure 3. Plasmidcard of ChiA with Anderson promoter. The chiA gene (in orange) is located on the pSB1C3 vector with an Anderson promoter (red) and a RBS (grey). This part is used for expression studies without the necessity of an induction. BioBrick (BB) suffix and prefix are shown in light blue. ( BBa_K2380006 ).
Figure 4. Plasmidcard of ChiA with AraC promoter. The chiA gene (in orange) is located on the pSB1C3 vector with the AraC promoter system (dark blue and blue) and a RBS (grey). BioBrick (BB) suffix and prefix are shown in light blue. ( BBa_K2380007 ).
Both the pSB1C3 vectors, containing the chiA gene, have the RBS BBa_K2380024 . After that, we transformed both plasmids into E. coli Top10 cells and BL21 cells. We verified the validity via eurofins tube sequencing, using a Mini-Prep-Kit first for DNA preparation.
We induced the E. coli BL21 cells, containing the pSB1C3 vector with the AraC promoter system, with arabinose to start expression. In order to validate the successful expression, we performed a SDS-Page.
Results and Discussion
Figure 5. SDS-Page. The page shows a successful expression of ChiA1 in E. coli BL21 cells. The lane marked with M) contains the protein marker, in 1) are cells 24 hours after the induction with arabinose, 2) has cells three hours after induction, 3) before induction and 4) contains empty BL21 cells. The arrow points to the expressed ChiA1. The amount of the protein at this position is highest in 1), while it is not existent in 4).
To verify if ChiA1 is produced in BL21 cells containing the pSB1C3-AraC-chiA, a SDS-Page was performed.  As we have described before, the page shows the expected results and proves that ChiA1 was successfully produced in E. coli BL21 cells.
Outlook
Chitosan is a derivative produced from chitin. As previously mentioned, chitosan's properties and bioactivity are heavily dependent on three factors: degree and pattern of deacetylation as well as its degree of polymerisation. The first two characteristics can be influenced and controlled via chitin deacetylases. Bringing variability into the length, however, is easiest done by implementing chitinases, whose ability to break down chitin would mean shorter chitosan molecules in turn.
Since chitosan is such a versatile molecule, the ability to select the influencing factors beforehand, means a great advantage over normally used methods.
Group Picture
Group Picture of the Chitinase team.
From left to right: Tim Maier, Bea Spiekermann, Feodor Belov, Sophia Hein, Lara Steinel, Isabelle Feinauer
References
[1]
Stefanie Nicole Hamer, Stefan Cord-Landwehr, Xevi Biarnés, Antoni Planas, Hendrik Waegeman, Bruno Maria Moerschbacher, and Staphan Kolkenbrock (2015) Enzymatic production of defined chitosan oligomers with a specific pattern of acetylation using a combination of chitin oligosaccharide deacetylases
DOI: 10.1038/srep08716
[2]
John G. Verburg and Q. Khai Huynh (1990) Purification and Characterization of an Antifungal Chitinase from Arabidopsis thaliana; Plant Physiol. 95, 450-455
DOI: https://doi.org/10.1104/pp.95.2.450
[3]
Paoletti MG, Norberto L., Damini R., and Musumeci S. (2007) Human gastric juice contains chitinase that can degrade chitin; Ann Nutr Metab 2007;51:244–251
DOI: 10.1159/000104144
[4]
National Center for Biotechnology Information, U.S. National Library of Medicine (NCBI); B.circulans chitinase A1 (chiA) gene, complete cds, GenBank: M57601.1; https://www.ncbi.nlm.nih.gov/nucleotide/1066341?report=genbank&log$=nuclalign&blast_rank=1&RID=Y46X5S0V014; last visited 10/28/2017
[5]
MD. Mustafa Alam, Takaaki Mizutani, Makoto Isono, Naoki Nikaidou, Takeshi Watanabe (1996) Three chitinase genes (chiA, chiC and chiD) comprise the chitinase system of Bacillus circulans WL-12; Journal of Fermantation and Bioengineering Vol.82, No. 1, 28-36
[6]
T Watanabe, W Oyanagi, K Suzuki, and H Tanaka (1990) Chitinase system of Bacillus circulans WL-12 and importance of chitinase A1 in chitin degradation; J Bacteriol. 1990 Jul; 172(7): 4017–4022
[7]
Watanabe et al., Gene Cloning of Chitinase A1 from Bacillus circulans Wl-12 Revealed Its Evolutionary Relationship to Serratie Chitinase and to the Type III Homology Units of Fibronectin (1990)
[8]
Sylvain Cottaz, Eric Samain (2005) Genetic engineering of Escherichia coli for the production of NI,NII-diacetylchitobiose (chitinbiose) and its utilization as a primer for the synthesis of complex carbohydrates; Metabolic Engineering 7, 311–317
[9]
Iwahori, F., Matsumoto, T., Watanabe, T., Nonaka, T. (2002) Catalytic Domain of Chitinase A1 from Bacillus circulans WL-12; https://www.rcsb.org/pdb/explore/explore.do?structureId=1ITX; last visited 10/28/2017
DOI: 10.2210/pdb1itx/pdb
[10]
Image of 1ITX ( Three-dimensional structure of the catalytic domain of chitinase A1 from Bacillus circulans WL-12 at a very high resolution Matsumoto, T., Nonaka, T., Hashimoto, M., Watanabe, T., Mitsui, Y. CRDT - 2002/02/13 12:00 AID - 10.2210/pdb1itx/pdb [doi] SO  - http://www.rcsb.org/pdb/explore/explore.do?structureId=1ITX)created with The PyMOL Molecular Graphics System, Version 2.0 Schrödinger, LLC.
[11]
GoldenBraid 2.0: A Comprehensive DNA Assembly Framework for Plant Synthetic Biology          Alejandro Sarrion-Perdigones, Marta Vazquez-Vilar, Jorge Palací, Bas Castelijns, Javier Forment, Peio Ziarsolo, José           Blanca, Antonio Granell, Diego Orzaez Plant Physiology Jul 2013, 162 (3) 1618-1631
DOI:10.1104/pp.113.217661
[12]
Knight, T. (2003) Idempotent Vector Design for Standard Assembly of Biobricks. MIT Artificial Intelligence         Laboratory
iGEM Technische Universität Darmstadt
We thank our sponsors, instructors and the iGEM organization!
Design: HTML5 UP
