Team:TU Darmstadt/Improve
ChiTUcare
MainPage
ChiTUcare
Project
Chitin Synthase
Chitinase
Chitin Deacetylase
Hydrogels
Chemistry
Parts
Notebook
Human Practices
Tech
Team
Judging
Chitin Synthase NodC
We set out to produce chitosan hydrogels that use chitin as a source material. Originally, this N-acetylglucosamine oligosaccharide (chitin) is extracted chemically from crustacean shells under usage of a lot of chemicals, producing chitin oligosaccharides of unspecified length [1] . To reduce the usage of chemicals, one aim of this project was to produce chitin in E. coli by insertion of a chitin synthase (CHS) into this organism, using the BioBrick system [2] . The CHS can produce chitin in E. coli in an enviromentally friendly way. The enzyme that was employed in this project is the CHS NodC from the bacteria Rhizobium  leguminosarum. NodC is an N-acetylglucosaminyl transferase which catalyzes the formation of chitin tetramers and pentamers using activated N-acetylglucosamine monomers. In addition, NodC reliably produces short oligosaccharides of certain lengths which can further be processed in vitro [3] .
Introduction
Besides cellulose, chitin is the most common polysaccharide in nature. Chitin is composed of β(1,4) linked 2-acetamido-2-deoxy-β-D-glucose (N-acetylglucosamine, Figure 1). The polymer is a white, hard nitrogenous polysaccharide and is a component of fungi cell walls and of the exoskeletons of insects and crustaceans, like crabs or shrimps [4] [5] .
Figure 1. Structure of Chitin. Chitin is one of the most common polymers in nature. It is composed of β(1,4) linked 2-acetamido-2-deoxy-β-D-glucose, which are acetylglucosamin monomers. The NAc stands for the acetamide group.
The extraction of chitin from crustaceans produces a lot of waste and uses a lot of chemicals. The waste of the seafood-processing industry, mostly the shells of crustaceans, contains 14 – 40 % chitin. This waste is treated with alternate acid and alkali to remove other components from the shells of the crustacean and to extract the chitin [1] . One approach to produce the polymer in an environmentally friendly way, are bacteria like E. coli which can produce chitin enzymatically via a CHS.
The production of chitin appears to be important as it is a useful substance which finds applications in medicinal, industrial and biotechnological research. Chitin, and its derivate chitosan, are non-toxic, biocompatible and biodegradable. Their bioactivities are for example the promotion of wound healing or hemostatic activity, immune enhancement, eliciting biological responses, and antimicrobial activity [1] .
Chitin oligomers are also of great biological interest as they lead to biological responses in plants and form the backbone of substituted lipochitooligosaccharides which induce the nodulation in leguminous plants [3] .
There are various kinds of CHS from several organisms. The  enzyme of our interest is the NodC which originates from the gram-negative bacterium Rhizobium leguminosarum. It is a homologue to the chitin synthase from yeast (Structure see Figure 2) [6] .
Figure 2. Structure of NodC. The 3D structure was modeled with SwissModel [7]
Rhizobium leguminosarum bv viciae is found to live in symbiosis with plants of the genera Pisum and Vicia of the family Fabaceae [9] . Rhizobium species live in symbiosis with legumes, where the bacteria form nitrogen-fixing nodules in the legume roots. The symbiotic interaction leads to an activation of the bacterial nodulation (nod) genes and the secretion of Nod factors. These nod genes create and modify the Nod factors. The Nod factors have a backbone consisting of β-1,4-N-acetylglucosamine oligosaccharides, most often tetra – or pentasaccharides with an acyl chain at C2 of the non-reducing end instead of an acetyl group [6] [10] .
Figure 3. Transmembrane domains of NodC. The NodC protein has strongly hydrophobic domains which indicate that it is an integral or transmembrane protein. The plot shows three transmembrane sequences. The plot was made with the TMHMM website. [8]
The NodC protein has strongly hydrophobic domains which indicate that it is an integral or transmembrane protein (Figure 3). Interestingly it is only found in the inner but not outer membrane of Rhizobium leguminosarum [10] . In contrast to most eukaryotic chitin synthases, NodC does not have a translocation domain for chitin. Therefore, the chitin synthesis is accomplished in the cytoplasm. NodC belongs to the class of glycosyltransferases which catalyze the transfer of sugar components from an activated donor molecule to a specific acceptor molecule [11] .
Mechanism
NodC is involved in the synthesis of chitin oligosaccharides, but only with a polymerization degree up to five. In earlier studies it was shown that investigating this CHS in E. coli is possible and leads to good results [12] .
NodC uses UDP-N-acetylglucosamine (UDP-GlcNAc) as a sugar donor, which is a precursor for the biosynthesis of peptidoglycan and therefore present in growing bacterial cells.
Another advantage is the unique property of the NodC which allows it to produce chitin pentamers in living E. coli without exogenous acceptors [3] . If an acceptor molecule and the substrate are added to the purified enzyme, the reaction can also be done in vitro.
The mechanism of elongation proceeds by a successive nucleophilic substitution reaction at C1 of the UDP-GlcNAc – molecule (Figure 4). UDP departs when the O4 atom of the growing sugar chain attacks as a nucleophile [11] .
With a low concentration of UDP-GlcNAc NodC produces a mixture of trimers, tetramers and pentamers and with high concentrations of UDP-GlcNAc it produces pentamers solely. It almost exclusively directs the formation of pentasaccharides [3] .
Figure 4. Mechanism of NodC. The elongation proceeds by a nucleophilic substitution reaction at C1 of the UDP-GlcNAc – molecule. The UDP departs when the O4 atom of the growing sugar chain attacks as a nucleophile. The UDP is set free and the chain is elongated. The colours indicate the various chitin monomers.
Methods
We ordered the nodC gene via IDT sequencing and inserted this gene into the pSB1C3 vector via the BioBrick system and verified this via sequencing.
One pSB1C3 vector has an AraC promoter system (BBa_K808000) and the other an Anderson promoter with defined cleavage sites (BBa_K2380025) . Both vectors have the RBS BBa_K2380024 .
Figure 5. Plasmidcard of NodC. The nodC gene (in pink) is located on the pSB1C3 vector without a promoter system. It is a basic part and can be used in combination with other parts. BioBrick (BB) suffix and prefix are shown in light blue. (BBa_K2380000) .
Figure 6. Plasmidcard of NodC with Anderson promoter. The nodC gene (in pink) is located on the pSB1C3 vector with an Anderson promoter (red) and a RBS (grey). This part is used for expression studies without the necessity of an induction. BioBrick (BB) suffix and prefix are shown in light blue. (BBa_K2380001) .
Figure 7. Plasmidcard of NodC with AraC promoter. The nodC gene (in pink) is located on the pSB1C3 vector with the AraC promoter system (dark blue and blue) and a RBS (grey). BioBrick (BB) suffix and prefix are shown in light blue. (BBa_K2380002) .
Afterwards we transformed the vector in E. coli BL21 for expression studies and started the expression, once by induction with 100 µL arabinose for the AraC promoter system and the other without induction by the constitutive Anderson promoter. To examine the successful expression, an SDS-PAGE was done.
The next step was the purification of the protein and the verification of the enzyme function. To purify the enzyme, a site-directed mutagenesis was done at the C-Terminus to add a His-tag and the protein was purified via an ÄKTA in combination with a 1 mL HisTrap column by GE Healthcare.  The activity of the NodC enzyme was evaluated via two different assays.
UDP-Glo™ Glycosyltransferase Assay
The other assay was the UDP-Glo™ Glycosyltransferase Assay from Promega. The Assay was used in order to test the functionality of the NodC enzyme. NodC is an N-acetylglucosamine transferase that uses UDP-GlcNAc as donor molecule. The NodC transfers N-acetylglucosamine from the UDP-GlcNAc to single N-acetylglucosamine bricks and UDP is set free. The free UDP is converted to ATP via a UDP Detection Reagent. This ATP generates light in a luciferase reaction which can be measured using a luminometer [13] .
TM Glycosyltransferase Assay", width=75%,>
Figure 8. Princip of the UDP-Glo™ Glycosyltransferase Assay. The NodC uses UDP-GlcNAc as donor molecule and transfers N-acetylglucosamine from the UDP-GlcNAc to single N-acetylglucosamine bricks and UDP. The free UDP is converted to ATP and the ATP generates light in a luciferase reaction. [13] .
Results
Expression and Purification
The nodC gene was expressed under the control of constitutive Anderson promoter (BBa_K2380025) and by arabinose-inducible AraC promoter (BBa_K808000) in E. coli Top10. Due to the E. coli cell's own proteins it was not possible to verify the NodC protein clearly. During the overexpression of the desired protein the cell's own proteins have also more time to express. To verify the expression of the NodC enzyme a SDS-PAGE was done after the purification instead (Figure 9).
We tagged the enzyme with a His-tag and purified the NodC enzyme via an ÄKTA system . The fractions were collected and the purity was examined via a SDS-PAGE. The SDS-PAGE shows a band at 21 kDA (Figure 9, red arrow) which fits to our protein size. The verification was done by using these fractions for a functional assay for the enzyme. The assay shows enzyme activity and therefore the successful expression.
Figure 9. SDS-PAGE after ÄKTA purification. The SDS-PAGE shows fraction 17 and 18 from the ÄKTA purification. A band at the size of 21 kDa can be seen (red arrow). This band was verfied to be our enzyme by the following activity assay. Marker is the Page Ruler Prestained Ladder from Thermo Fischer.
UDP-Glo™ Glycosyltransferase Assay
To verify the functionality of the NodC enzyme, the UDP-Glo™ Glycosyltransferase Assay was performed. This assay shows a luminescence, which indicates the conversion of free UDP to ATP. The UDP is set free by the reaction of the NodC enzyme.
The UDP standard curve shows increasing luminescence with increasing UDP concentration. With this curve the conversion to free UDP can be calculated.
The evaluation of the assay with sample 17 (see Figure 9) and 18 (data not shown) shows that the NodC enzyme converts the UDP-GlcNAc to free UPD and a growing oligo-GlcNAc-chain. So the assay shows that the NodC enzyme can create chitin oligomers.
Figure 10.UDP standard curve. The standard curve shows the linearity and sensitivity of the UDP-Glo™ Lycosyltransferase Assay. The curve was prepared over the indicated range of UDP concentration in 25 µl buffer. The luminescence was measured after 1 hour of incubation with a Tecan200 Infinite Pro plate reader. Values represent the mean of four replicates (N =4) and the standard deviation is shown. For better visibility the values for the small concentrations are shown in the additional graph. The following assays (Figure 11 and 12) are based on this standard curve. RLU = relative light units.
Figure 11. Activity assay of NodC. The NodC (40 ng) was titrated in 1X glycosyltransferase reaction buffer the presence of 100 μM of UDP-N-acetylglcosamine and 10 mM N-acetylglucosamine (GlcNAc) as an acceptor substrate. The reaction was performed as described before and the luminescence was measured after 1 hour of incubation with a Tecan200 Infinite Pro plate reader. Each point is an average of two experiments (N = 2), and the error bars represent the standard deviations. RLU = relative light units.
The higher the concentration of the NodC enzyme is, the more UDP is set free. This could be verified by employing a higher concentration of NodC in the assay (see Figure 12).
Figure 12.  Activity assay of NodC with higher NodC concentrations. The concentration of NodC starts with 1600 ng and was then titrated in 1X glycosyltransferase reaction buffer the presence of 100 μM of UDP-N-acetylglcosamine and 10 mM N-acetylglucosamine (GlcNAc) as an acceptor substrate. The higher concentration was used to show a higher turnover of the substrate. The reaction was performed as described before and the luminescence was measured after 1 hour of incubation with a Tecan200 Infinite Pro plate reader. Each point is an average of two experiments (N = 2), and the error bars represent the standard deviations. RLU = relative light units.
Discussion and Outlook
In our research, we show that the nodC gene can successfully be expressed in E. coli and the functional enzyme can produce chitin oligomers. This is a great chance to synthesize chitin oligomers in an environment-friendly way. Biosynthetical production of chitin oligomers forms the basis for the specific synthesis of chitosan (deacetylated chitin) and therefore is also highly relevant to all biotechnological and medical applications of chitosan.  The specific qualities of chitin or chitosan, the non-toxicity, biocompatibility and biodegradability, are supportive for application in medicine. In addition the bioactivities, like the promotion of wound healing, hemostatic activity or antimicrobial activity indicate an implementation in the field of wound management [1] . These beneficial properties improve by decreasing the length of chitosan. Hence, the oligomeric chitin produced by NodC is a great precursor for medicinal chitosan production. For our project we will use chitosan to manufacture a hydrogel for treating burn wounds.
The production of chitin is the first step in generating a chitosan hydrogel band-aid ecologically. Further modifications of the chitin are the deacetylation to chitosan ( see Deacetylase ) and linking with a fluorophore ( see Chemistry ).  The production of chitin in E. coli is not only beneficial for our project but for a variety of applications, like a fungus control agent in agricultural applications. Some of the applications of chitin or chitosan are quite promising, for instance in the fields of medicine, pharmacy, agriculture, and food processing [1] .
Group Picture
Group Picture of the Chitin Synthase team.
On the tree, from left to right: Isabelle Feinauer, Cristina Kurzknabe.
In front of the tree, from left to right: Isabelle Marquardt, Werner Kleindienst, Patrick Müller, Sven Storch and Lisa-Marie Brenner
References
[1]
Kurita, K. (2006) Chitin and Chitosan: Functional Biopolymers from Marine Crustaceans. Marine Biotechnology, 8, 203 – 226
DOI: 10.1007/s10126-005-0097-5
[2]
Knight, T. (2003) Idempotent Vector Design for Standard Assembly of Biobricks. MIT Artificial Intellignece Laboratory
[3]
Samain, E., Drouillard, S., Heyraud, A., Driguez, H., and Geremia, R. A. (1997) Gram-scale synthesis of recombinant chitooligosaccharides in Escherichia coli. Carbohydrate Research, 302, 35 – 42
DOI: 10.1016/S0008-6215(97)00107-9
[4]
Dutta, P. K., Dutta, J., and Tripathi, V. S. (2004) Chitin and Chitosan: Chemistry, properties and applications. Journal of Scientific & Industrial Research, 63, 20 – 31
[5]
Kumar, M. N. V. R. (2000) A review of chitin and chitosan applications. Reactive & Functional Polymers, 46, 1 – 27
DOI: 10.1016/S1381-5148(00)00038-9
[6]
Debellé, F., Rosenberg, C., and Dénarié, J. (1992) The Rhizobium, Bradyrhizobium, and Azorhizobium NodC proteins are homologous to yeast chitin synthases. Molecular Plant-Microbe Interactions, 5, 443 – 446
PMID: 1472721
[7]
Marco Biasini, Stefan Bienert, Andrew Waterhouse, Konstantin Arnold, Gabriel Studer, Tobias Schmidt, Florian Kiefer,     Tiziano Gallo Cassarino, Martino Bertoni, Lorenza Bordoli, Torsten Schwede; SWISS-MODEL: modelling protein tertiary and     quaternary structure using evolutionary information, Nucleic Acids Research, Volume 42, Issue W1, 1 July 2014, Pages     W252–W258
DOI:10.1093/nar/gku340
Bordoli, L., Kiefer, F., Arnold, K., Benkert, P., Battey, J. and Schwede, T. (2009). Protein structure homology     modelling using SWISS-MODEL Workspace. Nature Protocols, 4,1
DOI:10.1038/nprot.2008.197
Arnold K., Bordoli L., Kopp J., and Schwede T. (2006). The SWISS-MODEL Workspace: A web-based environment for protein     structure homology modelling.Bioinformatics, 22,195-201
DOI: 10.1093/bioinformatics/bti770
Stefan Bienert, Andrew Waterhouse, Tjaart A. P. de Beer, Gerardo Tauriello, Gabriel Studer, Lorenza Bordoli, Torsten     Schwede; The SWISS-MODEL Repository—new features and functionality,Nucleic Acids Research, Volume 45, Issue D1, 4     January 2017, Pages D313–D319
DOI:10.1093/nar/gkw1132
Kiefer F, Arnold K, Künzli M, Bordoli L, Schwede T (2009). The SWISS-MODEL Repository and associated resources. Nucleic Acids Research 37, D387-D392
DOI: 10.1093/nar/gkn750
Florian Kiefer, Konstantin Arnold, Michael Künzli, Lorenza Bordoli, Torsten Schwede; The SWISS-MODEL Repository and     associated resources, Nucleic Acids Research, Volume 37, Issue suppl_1, 1 January 2009, Pages D387–D392
DOI:10.1093/nar/gkn750
Kopp J, and Schwede T (2006). The SWISS-MODEL Repository: new features and functionalities. Nucleic Acids Res.,34, D315-    D318
DOI:10.1093/nar/gkj056
Guex, N., Peitsch, M.C. Schwede, T. (2009). Automated comparative protein structure modeling with SWISS-MODEL and Swiss-    PdbViewer: A historical perspective. Electrophoresis, 30(S1), S162-S173
DOI:10.1002/elps.200900140
[8]
A. Krogh, B. Larsson, G. von Heijne, and E. L. L. Sonnhammer.       Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes. Journal of Molecular Biology, 305(3):567-580, January 2001
DOI:10.1006/jmbi.2000.4315
E. L.L. Sonnhammer, G. von Heijne, and A. Krogh.       A hidden Markov model for predicting transmembrane helices in protein sequences.       In J. Glasgow, T. Littlejohn, F. Major, R. Lathrop, D. Sankoff, and C. Sensen, editors, Proceedings of the Sixth        International Conference on Intelligent Systems for Molecular Biology, pages 175-182, Menlo Park, CA, 1998. AAAI        Press
[9]
Long, S. R. (1996) Rhizobium Symbiosis: Nod Factors in Perspective. The Plant Cell, 8, 1885 – 1898
DOI: 10.1105/tpc.8.10.1885
[10]
Barny, M. A., and Downie, J. A. (1993) Identification of the NodC Protein in the Inner but Not the Outer Membrane of Rhizobium leguminosarum. Molecular Plant-Microbe Interactions, 6, 669 – 672
[11]
Dorfmueller, H.C., Ferenbach, A. T., Borodkin, V. S., and van Aalten, D. M. F. (2014) A Structural and Biochemical Model of Processive Chitin Synthesis. The Journal of Biological Chemistry, 289, 23020 – 23028
DOI: 10.1074/jbc.M114.563353
[12]
Kamst, E., van der Drift, K. M. G. M., Thomas-Oates, J. E., Lugtenberg, B. J. J., and Spaink, H. P. (1995) Mass Spectrometric Analysis of Chitin Oligosaccharides Produced by Rhizobium NodC Protein in Escherichia coli. Journal of Bacteriology, 177, 6282 - 6285
DOI:  10.1128/jb.177.21.6282-6285.199
[13]
Promega (2015) UDP-GloTM Glycosyltransferase Assay, Technical Manual
iGEM Technische Universität Darmstadt
We thank our sponsors, instructors and the iGEM organization!
Design: HTML5 UP
