<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UT-Tokyo_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UT-Tokyo/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UT-Tokyo/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UT-Tokyo/Project&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UT-Tokyo/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UT-Tokyo/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="global_wrapper"><DIV id="header"><DIV class="wrapper"><DIV class="box">
            The contents are...
                <OL class="menu"><LI><SPAN class="mhead">Multicellular Analog Clock</SPAN></LI><LI><A href="#mSummary">Summary</A></LI><LI><A href="#mBackground">Background</A></LI><LI><A href="#mSystems">Systems</A></LI><LI><A href="#mResults_and_Discussion">Results and Discussion</A></LI><LI><A href="#mReferences">References</A></LI></OL><LI><SPAN class="mhead">RNA Silencing</SPAN></LI><LI><A href="#rBackground">Background</A></LI><LI><A href="#rSystems">Systems</A></LI><LI><A href="#rResults_and_Discussion">Results and Discussion</A></LI><LI><A href="#rReferences">References</A></LI></DIV></DIV><UL id="navig"><LI class="home"><DIV class="naka">Back to HOME</DIV><DIV class="migi"><A href="https://2013.igem.org/Team:UT-Tokyo">Home</A></DIV></LI><LI class="project"><DIV class="naka"><A href="https://2013.igem.org/Team:UT-Tokyo/Project#Multicellular_Analog_Clock">Multicellular Analog Clock</A><A href="https://2013.igem.org/Team:UT-Tokyo/Project#RNA_Silencing">RNA Silencing</A></DIV><DIV class="migi"><A href="https://2013.igem.org/Team:UT-Tokyo/Project">Project</A></DIV></LI><LI class="modeling"><DIV class="migi"><A href="https://2013.igem.org/Team:UT-Tokyo/Modeling">Modeling</A></DIV></LI><LI class="humanpractice"><DIV class="migi"><A href="https://2013.igem.org/Team:UT-Tokyo/HumanPractice">Human Practice</A></DIV></LI><LI class="team"><DIV class="naka"><A href="https://2013.igem.org/Team:UT-Tokyo/members.html">Members</A><A href="https://2013.igem.org/Team:UT-Tokyo/labdata">Lab Data</A><A href="https://2013.igem.org/Team:UT-Tokyo/Protocol">Protocol</A></DIV><DIV class="migi"><A href="https://2013.igem.org/Team:UT-Tokyo/Team">Team</A></DIV></LI></UL></DIV><DIV id="main"><PRE>           PROJECT
       
</PRE><DIV id="contents"><H1 id="Multicellular_Analog_Clock">Multicellular Analog Clock</H1><H2 id="mSystems">Systems</H2><H3>What is the Multicellular Analog Clock?</H3><P class="ini">The goal of the project is to make the Multicellular Analog Clock which has following features;</P><P>1. One directional movement of position of gene expression on a multicellular scale (like the movement of clock hand).</P><P>2. Easy observation; visible under natural light without any special equipment.</P><P>3. Easy opreration; a familiar agar plate with <I>E. coli</I> population on it.</P><P>We designed “gear” and “clock hand”. The genetic “gear” of the clock, which generates gene expression in response to a stimulus and relays this stimulus to surrounding cells, consists of a positive feedback loop and a Multi-negative feedback system. The “clock hand” to visualize this gene expression is mCherry, a protein visibly red under natural light (Table 1). <I>E. coli</I> possessing this genetic behavior is placed on an agar plate so as to create a circle. Agar plates are relatively inexpensive and easier to use than microfluidic device, which is used to observe the spatiotemporal behavior of gene expression in previous research (Tal Danino et al., 2010 [1]). Agar plate is small and portable while microfluidic device has machines and tubes connected to it. These advantages of agar plates may contribute to the intuitive understanding of clock hand movement, and lead to  future applications.</P><P><B>Table 1</B></P><P>The clock device also contains a “control” reporter (amilGFP) for dual-reporter assay and a UV reset device (lambda cI repressor), which are explained later.</P><P><B>Figure 1</B> Multicellular Analog Clock</P><P><B>Figure 2</B> The clock device. Pink box genes (luxI and luxR) form the positive feedback loop of AHL production. Blue box genes (tetR, aiiA, and sRNAs) are the suppressors used for negative feedback loops. mCherry is a red reporter protein for visualization of the excitation wave (“clock hand”). amilGFP is the yellow reporter protein expressed constitutively as an internal standard.</P><H3>The neuron analogy</H3><P class="ini">Our genetic “clock hand” , is similar in many ways to the action potential propagating along nerve cell axons (Fig. 3, Fig.4). Both can be classified as excitation waves which are driven to propagate in one direction by positive and negative feedback loops. Imagine a circle of axon. The propagation of action potential is just like a clock hand movement (Fig. 5). We explain the mechanism of our clock “gear” by analogy with action potential of the nerve cell.</P><P><B>Figure 3</B> The clock hand moves in one direction; from the past to the future.</P><P><B>Figure 4</B> One directional propagation of action potential</P><P>When stimulated, a pulse of positive membrane potential (action potential) is generated. (Fig. 5). Positive membrane potential opens voltage-gated Na+ channel. Influx of Na+ increases positive membrane potential. This is positive feedback to amplify positive membrane potential. After a few milliseconds, positive membrane potential opens voltage-gated K+ channel. Increased K+ outflux decreases membrane potential. This is negative feedback to decrease positive membrane potential (Fig. 6).</P><P>Figure 5 Action potential of nerve cell</P><P><B>Figure 6</B> Positive and negative feedback loops regulate positive membrane potential to drive the action potential along nerve cell axon.</P><P>The positive membrane potential spreads along axon, coupling membrane potential of neighboring region, and causes another rising phase and falling phase. This is observed as action potential propagating along a nerve cell axon. </P><P>The feedback loops and coupling creates one-directional movement in nerve cell. Nerve cell and our Clock have the positive feedback followed by negative feedback and coupling in common (Table 2). Thus, as in a nerve cell, the one directional excitation wave may be generated on our Clock (Fig. 7).</P><P><B>Table 2</B> Action potential and clock hand</P><P><B>Figure. 7</B></P><H3>Positive feedback loop</H3><P class="ini">The positive feedback loop of the clock “gear” is composed of luxI (3OC6HSL(AHL) synthetase), the pLux/Tet hybrid promoter, and the luxR activator(Fig. 8). </P><P><B>Fig. 8</B> Positive feedback loop</P><P>This positive feedback loop makes rising phase of gene expression. It works as follows (Fig. 9);</P><P>0, LuxR is constitutively expressed in cells.</P><P>1, AHL flows in from out of the cell and binds to LuxR.</P><P>2, the AHL-LuxR complex binds to the pLux/Tet hybrid promoter and increase the transcription rate of genes including luxI.</P><P>3, LuxI synthesizes AHL.</P><P><B>Figure. 9</B> Positive feedback loop.</P><P>This process occurs not only in a single cell. Intercellular positive feedback loops are formed and function as intercellular coupling like a spreading positive membrane potential of a nerve cell axon (Figure 10); AHL is synthesized in a cell and diffuses across cell membrane, and activates transcription in other cells. This cell-cell communication enables intercellular synchronization of gene expression, which drives the one directional movement of the “clock hand”.</P><P><B>Figure 10</B> Intercellular coupling</P><H3>Multi-Negative feedback system</H3><P>Our clock “gear” has 4 negative feedback loops of different mechanisms (Table3), which are formed by tetR, aiiA, anti-mCherry sRNA, and anti-luxI sRNA (Fig. 11). All suppressors are regulated under the pLux/Tet promoter and expressed when stimulated by AHL. This negative feedback loops makes falling phase of gene expression.</P><P><B>Figure 11 </B>Multi-negative feedback system</P><P><B>Table 3</B> Multi-negative feedback system </P><P>TetR binds to pLux/tet and represses transcription of mCherry, luxI, and 4 suppressors including tetR. </P><P>AiiA degrades AHL in order to repress transcription indirectly.</P><P>The sRNAs might work as a rapid suppressor for two reasons; 1. they repress the translation, which is the final step of prokaryotic gene expression, 2. they are quickly produced since they do not require time for translation of themselves. Both of sRNAs facilitate degradation of mRNA that contains mCherry, luxI, tetR, and aiiA. Anti-mCherry sRNA represses translation of mCherry gene, and anti-luxI does that of luxI gene. Repression of luxI causes decreased production of AHL, while repression of mCherry doesn’t affect AHL.</P><P>The differences of mechanisms may lead to various patterns of suppression of various strength and time (Fig. 12). This multi-negative feedback system is a novel one which is not reported in previous papers or past iGEM projects. We designed 20 combinations of these 4 negative feedback loops and sought for the best combination for the Multicellular Analog Clock.</P><P>Hasty group reported[1] the system with only one negative feedback loop of AiiA. Their system generated synchronized oscillation. This system shows bursts without any external stimuli. The burst is not suitable for the Analog Clock, because it means inappropriate addition of clock hands. TetR and 2 sRNAs were added in our clock device. These additional suppressors may lead to stronger repression and prevents burst of other clock hands.</P><P><B>Figure 12</B> Multi-negative feedback system.</P><H3>The excitation wave</H3><P>Here, the propagation of excitation wave is explained concluding the discussion above .The positive feedback loop and Multi-negative feedback system drive the “clock hand”, the excitation wave of gene expression. The propagation of excitation wave is generated as follows (Fig. 13);</P><P>0. Cells are in the resting state. They are ready to respond to the stimulus by AHL.</P><P>1. Cells are activated by AHL synthesized by neighbor cells and express mCherry. These cells are the “clock hand”.</P><P>2. The LuxI is also expressed at the same time and synthesize AHL to activate neighbor cells.</P><P>3. The supressors are also expressed to make unresponsive state, which prevents “clock hand” go backwards.</P><P><B>Figure 13</B></P><H3>Clock hand (visualization for naked eyes)</H3><P>The clock device has 2 reporter genes for dual-reporter assay, which allows accurate measurement of gene expression intensities.</P><P>mCherry is a red reporter protein for the “clock hand”. mCherry expression intensity changes dynamically, being increased by AHL stimulus and decreased by 4 suppressors.</P><P>amilGFP is yellow “control” reporter protein for ratiometric measurement with mCherry. amilGFP is constitutively expressed independently from clock “gear”. </P><P>Without “control” reporter, background noises (changing agar plate environment surrounding cells, amount of resources for gene expression, cell population density, degradation tag activity, etc.) might obscure the dynamic gene expression of interest (=“clock hand”). Comparing the mCherry expression with amilGFP, you can clearly track the “clock hand” for a long time and will notice the unimportant noises disturbing the clock.</P><P>mCherry and amilGFP have strong color, which is visible to naked eyes under natural room light (Fig. 14). You can easily see the gene expression without special equipment.</P><P>Also, both of them are fluorescent protein and have different excitation and emission spectra. Measuring fluorescence (e.g. fluorescent microscopy, flow cytometry, …) will be a good way to measure the gene expression accurately. You can measure both fluorescence and color under natural light with one BioBrick device.</P><P>amilGFP with LVA degradation tag is a new BioBrick part of our design. This BioBrick part will be used as a good reporter for its strong yellow color, fluorescence, and short lifetime, which enables intuitive understanding and real-time measurement of gene expression at the same time.</P><H4>LVA degradation tag</H4><P>C-termini of almost all the proteins in the clock device (luxI, tetR, aiiA, mCherry, and amilGFP) have LVA degradation tag for rapid degradation [2]. Rapid degradation of components of feedback loops may cause rapid change of gene expression, which 
keeps “clock hand” small. Reporter genes should have short half-lives to visualize real-time changes of gene expression. LuxR has no degradation tag, since clock device needs constant expression of LuxR.</P><H3>UV reset device</H3><P>Feedback loops of genetic circuit are not sufficient to generate one directional movement of mCherry “clock hand” along an <I>E. coli</I> circle. If you had only feedback loops, after adding AHL at one point of the circle to start the Clock, you will get two-way movement, which will soon disappear by collision of two “clock hands”. </P><P>To turn this two-way movement to one-way, we need to make some part of a circle unresponsive before starting the clock. UV reset device was included in the clock device for this purpose. UV reset device contains UV sensor (UV-susceptible cI repressor and the phage lambda PR promoter and suppressors (Fig. 15). )</P><P><B>Figure 15</B></P><P>UV reset device makes the “unresponsive period” in response to exposure to UV. The detailed mechanism is as follows (Fig. 16)</P><P>0, cI repressors are constitutively expressed.</P><P>1, UV induces cleavage (inactivation) of cI repressor.</P><P>2, phage lambda PR promoter is derepressed.</P><P>3, suppressors are expressed, which makes unresponsive period.</P><P><B>Figure 16</B> UV reset device. UV induces suppressors to make unresponsive state.</P><P>UV treatment induces unresponsive period and AHL generate clock hand. One-way movement is generated when UV treatment was next to the point of AHL induction when you start the clock (Fig. 17).</P><P><B>Figure 17</B> Start of the Multicellular Analog Clock. UV induction is just next to AHL.</P><P>UV-susceptible cI repressors (E233K) were used in toggle switch of iGEM MIT 2010. However, MIT 2010[3] made only the composite toggle parts. Thus, we made a new BioBrick part of single UV-susceptible cI repressor (E233K) through site-directed mutagenesis. This standard basic part will be useful for future registry members.</P><H3>The full system</H3><P>“Gear”, “Clock hand”, and UV reset device make the full genetic circuit (<A href="http://parts.igem.org/Part:BBa_K1124100">BBa_K1124100</A>) (Fig. 18).The circle of <I>E. coli</I> population on agar plate with BBa_K1124100 works as the Multicellular Analog Clock.</P><P><B>Figure 18</B></P><H2 id="mApplicaton">Applicaton</H2><P>In this project, we constructed an analog clock of <I>E.coli</I> for which quorum sensing was utilized. The system is simple because it has only one hand. So other functions can be easily adopted. For example, if particular smell or light is given out every regular time, it becomes an alarm clock. Some past iGEM projects such as MIT2006 and NCBS2009 referred to how to produce scent. In addition, the fine adjustment device could be constructed which is based on the velocity of the hand of it changing by external input. For instance, it is possible to precisely control the clock device by employing promoterAHL which promotes transcription according to the density of AHL and constructing the sequences, such as promoterAHL-luxI and promoterAHL-(tetR, aiiA, sRNAs). While UV reset device resets the clock by decomposition of c1 repressor (E233K), it resets the clock by increase or decrease in AHL.                       </P><P>As another use of clock device, there is a quantity measuring device. The outline is as follows. First, we move the hand by means of the clock device, and then stop the movement of it after a fixed period of time by such as irradiation of UV. As a result, quantity can be visibly measured. For example, we could measure the density of a material by using this device.  The material to be measured defines “S”, we make a construction, promoterS-RBS-luxI-d.term-(clock-device)-(UV reset device). PromoterS promotes transcription in response to the existence of S. Then, because the velocity of the hand changes according to the density of S, we could conclude that if it swung longer, the density is higher and that if it swung shorter, the density is lower.(Fig.1)   </P><P>Fig1. The density of a material measuring device                      </P><P>Not only whole clock device, but also the parts we made for this project constitutive of it, such as amilGFP and UV sensor, could be applied to different purposes. amilGFP with LVA degradation tag gives out strong yellow light under natural light and degrades rapidly. Using these features enables us to measure visually, depending on the flow of time. Also, UV sensor can express any gene in any time by only irradiating UV.     </P><P>As stated above, the clock device and its parts can be used for various purposes and these applications would also give synthesis biology great progress.              </P><H1 id="RNA_Silencing">RNA Silencing</H1><H2 id="rBackground">Background</H2><P class="ini"><B>Introduction</B></P><P class="ini">It is needless to say that gene knockout is important in the study of biology. Genomic knockout by making the target gene mutate requires much labor and time. In eucaryotes, RNAi has contributed to the functional analysis of genes. <I>E. coli</I>, which belongs to procaryotes, does not have the mechanism needed for RNAi and this method is not usually used for <I>E. coli</I>.</P><P>But in recent years it has been becoming clear that <I>E. coli</I> has the mechanism of gene silencing by small regulatory RNAs (sRNA) which have complementary sequences with their target mRNAs [1].We used the mechanism for gene knockout by designing synthetic sRNAs and inducing plasmids coding them into bacterial cells. This method can be done by gene introduction and enables us to knock down genes by combining our new BioBrick parts &quot;micC scaffold&quot; and complementary sequences with the target gene [2].</P><P>We tested using our BioBrick parts by applying them for the following objectives.</P><P><UL><LI>Acceleration of Biosynthesis of L-DOPA, a substance utilized for treatment of Parkinson’s disease.</LI><LI>Improvement of H2 Productive Capacity of <I>E. coli</I></LI><LI>Suppressor of Multi-Negative Feedback System of Clock Device</LI></UL></P><H2 id="rSystem">System</H2><P class="ini"><B>Mechanism of RNA silencing in Escherichia coli</B></P><P class="ini">
Escherichia coli have several trans-acting sRNA molecules repressing the expressions of their target genes, such as micC and sgrS and so on. Their mechanism has been becoming clear in recent years. (Fig.1) These sRNA molecules consist of complementary sequences to their target mRNAs and a scaffold sequence that recruites the Hfq protein. Hfq protein, one of the RNA chaperones, facilitates the hybridization of sRNAs to mRNA by their complementary sequences and with RNase E. RNase E causes the degradation of the sRNAs by the target mRNAs. So the mechanism of gene repression by Hfq-dependent sRNAs is composed of the translational repression by their complementary sequences and Hfq proteins and the degradation of the target mRNAs by RNase E [3].</P><P>We tried to apply the Hfq protein-dependent RNA silencing for several objectives by establishing the design principle of sRNAs.</P><P><B>Fig.1</B> Mechanism of expression inhibition by Hfq protein-dependent sRNAs Design and use of synthetic regulatory small RNAs</P><P>In order to design Hfq protein-dependent sRNAs simply, we used the following design principle. Our sRNAs are composed of two regions: a target-binding sequence and a scaffold sequence. We made the sRNA generator by combining these with a promoter and a terminator. (Fig.2)</P><P>We chose a region of the target sequence that spans from the AUG to nucleotide +21 of the mRNA as the binding site. This was because a previous study suggested that it is effective especially to select the region overlapping the TIR (Translation Initiation Region) of the mRNA [2]. The scaffold sequence recruits the Hfq protein, which facilitates the hybridization of sRNA and target mRNA as well as translational inhibition. Among the scaffold sequences of <I>E. coli</I>, we selected MicC because of the superior repression capability and made the scaffold BioBrick part (BBa_K1124005). In this design, there is an unnecessary sequence, which is caused by using as BioBrick parts, between the promoter and the target-binding sequence.</P><P>We proved that the MicC scaffold and our sRNA design with pConst-sRNA(anti-tyrR) functions by qRT-PCR.</P><P><B>Fig.2</B> Design principle of our synthetic sRNAs</P><P class="ini"><B>Automatic RNA silencing primer designer</B></P><P class="ini">Here we present Automatic RNA silencing primer designer. You can get a pair of primer to generate sRNA sufficient to knockdown the gene of your choice.</P><H3><I>Program 1</I></H3><P>When you perform PCR with primers designed by Program 1, you will get the PCR product like the diagram below.</P><P>You can get promoter-free parts (target binding seq. - micC scaffold - d.term), so you can select any promoters in BioBrick registry. </P><DIV><B>CDS</B><B>Forward Primer for BioBrick</B></DIV><P><I>Procedures</I></P><P>1.	Input the first 24nt of the CDS of the gene you would like to knockdown.</P><P>2.	The sequence of forward primer is shown.</P><P>3.	The sequence of reverse primer is 5-tcaagaactctgtagcaccgcc-3.</P><P>4.	Perform PCR using primers given by the program. The template must be <A href="http://parts.igem.org/Part:BBa_K1124112">BBa_K1124112</A> on the plasmid backbone pSB1C3, pSB1A3, or pSB1AK3.</P><P>5.	Cut the PCR product by Xba1 and Pst1, and ligate it with backbone.</P><P>6.	(Alternative to Procedure 5) Cut the PCR product by Xba1 and Pst1, and the promoter part on a plasmid backbone by Spe1 and Pst1. Ligate these digests and you will get the full constructed part that expresses sRNA sufficient to knockdown the gene of your choice.</P><H3><I>Program 2</I></H3><P>
When you perform PCR with primers designed by Program 2 using BBa_K1124123.as the template, you will get the PCR product like the diagram below.</P><P>The PCR product is the full constructed part that expresses sRNA sufficient to knockdown the gene of your choice. You never have to digest and ligate the plasmid before <I><I>E. coli</I></I> transformation.
</P><DIV><B>CDS</B><B>Forward Primer for Inverse PCR</B><B>Reverse Primer for Inverse PCR</B></DIV><P><I>Procedures</I></P><P>1.	Input the first 24nt of the CDS of the gene you would like to knockdown.</P><P>2.	The sequences of forward primer and the reverse primer are shown.</P><P>3.	Perform inverse PCR with using primers given by the program. The template is <A href="http://parts.igem.org/Part:BBa_K1124123">BBa_K1124123</A>. There is no restriction on the plasmid backbone. You must use DNA polymerase with no displacement activity nor 5f¨3fexonuclease activity.</P><P>4.	The PCR product is a plasmid with nicks. Transform <I>E. coli</I> with the PCR product and the plasmid will expresses sRNA sufficient to knockdown the gene of your choice.</P><H2 id="rResults_and_Discussion">Application</H2><P class="ini"><B>Acceleration of Biosynthesis of L-DOPA</B></P><P class="ini">In order to make <I>E. coli</I> produce L-DOPA, we made hpaBC (encoding the enzyme 4-hydroxyphenylacetate 3-hydroxylase) BioBrick parts by cloning it from <I>E. coli</I> BL21 strain. HpaBC catalyzes the hydroxylation of tyrosine and produces L-DOPA (Fig.3a). But the production rate of L-DOPA was low. Then we designed sRNAs knocking down several genes involved in the pathway from glucose to L-Tyr in order to increase the carbon flow to tyrosine synthetic pathway, because the amount is thought to be dependent on that of L-Tyr, the precursor of L-DOPA [4].</P><P>We selected tyrR(encoding tyrosine repressor), csrA(encoding carbon-storage regulator, which regulates the expression of enzyme genes involved in glycolysis) and tyrC (phosphoribosylanthranilate isomerase) as the targets [2]. tyrR and csrA have been shown to promote L-Tyr biosynthesis in the previous study.(Fig.3b)</P><P>Because these sRNAs are thought to damage the growth of <I>E. coli</I>, we tried several promoters having different strength. </P><P><B>Fig.3a</B> Hydroxilation of tyrosine to L-DOPA by HpaBC</P><P><B>Fig.3b</B> The tyrosine biosynthetic pathway in <I>E. coli</I> and our metabolic engineering strategy</P><P class="ini"><B>Improvement of H2 Productive Capacity of E. coli</B></P><P class="ini">In 2012, we UT-Tokyo made &quot;H2 <I>E. coli</I>&quot;, which produces more H2 than usual <I>E. coli</I>. This year we tried to improve its H2 productive capacity further by designing anti-hycA. </P><P><I>E. coli</I> naturally produces hydrogen through a process called mixed acid fermentation (MAF pathway) (Fig.4a). HycA represses the transcription of the proteins of FHL complex, which produces H2 from formate. </P><P>Last year we tried to derepress it by introducing plasmids including many decoy binding sites in our project &quot;Inhibition without knockout&quot;. This method has several methods. It allows to knockdown by introducing BioBrick parts and does not make scars on the candidate's genome. But it can be used only for DNA binding proteins and the strength cannot be regulated without changing the plasmid.</P><P>Though the merit of &quot;Inhibition without knockout&quot; is retained in knockdown with Hfq protein-dependent sRNAs, our new method does not have these demerits.</P><P><B>Fig.4a</B> The function of FHL complex and HycA</P><P><B>Fig.4b</B> The system of &quot;Inhibition without knockout&quot;</P><P>By introducing plasmids with many decoy binding sites of the target proteins, we tried to prevent repressors from acting on genomic sites.</P><P class="ini"><B>Suppressor of Multi-Negative Feedback System of Clock Devise</B></P><P class="ini">In our project, Multicellular Analog Clock, we needed a suppressor to be used for the Multi-Negative Feedback System and UV reset devise. Our clock devise uses novel systems which are not seen in previous papers or past iGEM project and it is difficult to say what kind of suppressors is best unless we actually try them. So we designed two kinds of sRNAs that bind to the TIR of mCherry and luxI as the suppressors (anti-mCherry and anti-luxI).</P><P>sRNA suppressors will behave differently from AiiA and TetR. sRNAs express and decompose more rapidly than suppresors based on these proteins and act on different targets.</P><P>The two sRNAs are act on different molecules within the clock pathway and therefore expected to influence clock behavior differently. Translational inhibition is caused only by sRNAs binding to the TIR of the target gene while degradation by RNase E is caused by any sRNAs binding to the mRNA.</P><P>The differences of mechanisms may lead to various patterns of suppression of various strength and time.</P><H2 id="rReferences">References</H2><P class="ini">[1]Aiba, H. (2007). Mechanism of RNA silencing by Hfq-binding small RNAs. Current opinion in microbiology, 10(2), 134-139.</P><P class="ini">[2]Na, D., Yoo, S. M., Chung, H., Park, H., Park, J. H., &amp; Lee, S. Y. (2013). Metabolic engineering of Escherichia coli using synthetic small regulatory RNAs. Nature biotechnology.</P><P class="ini">[3]Yoo, S. M., Na, D., &amp; Lee, S. Y. (2013). Design and use of synthetic regulatory small RNAs to control gene expression in Escherichia coli. Nature protocols, 8(9), 1694-1707.</P><P class="ini">[4]Munoz, A. J., Hernandez-Chavez, G., de Anda, R., Martinez, A., Bolivar, F., &amp; Gosset, G. (2011). Metabolic engineering of Escherichia coli for improving L-3, 4-dihydroxyphenylalanine (L-DOPA) synthesis from glucose. Journal of industrial microbiology &amp; biotechnology, 38(11), 1845-1852.</P><P class="ini">[5]Maeda, T., Sanchez‐Torres, V., &amp; Wood, T. K. (2008). Metabolic engineering to enhance bacterial hydrogen production. Microbial Biotechnology, 1(1), 30-39.</P></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:UT-Tokyo/Project">http://2013.igem.org/Team:UT-Tokyo/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UT-Tokyo/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UT-Tokyo/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UT-Tokyo/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UT-Tokyo/Project&amp;oldid=364402" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>