<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UNC-Chapel_Hill_Background skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UNC-Chapel Hill/Background</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><CENTER><UL class="nav"><LI class="dropdown leftbookend"><A href="/Team:UNC-Chapel_Hill" style="font-size:13px;">HOME</A></LI><LI class="dropdown leftbookend"><A href="/Team:UNC-Chapel_Hill/Background" style="font-size:13px;">BACKGROUND</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UNC-Chapel_Hill/Project" style="font-size:13px;">PROJECT</A></LI><LI><A href="/Team:UNC-Chapel_Hill/Parts">PARTS</A></LI></UL><LI class="dropdown leftbookend rightbookend"><A href="/Team:UNC-Chapel_Hill/Notebook" style="font-size:13px;">NOTEBOOK</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UNC-Chapel_Hill/Protocols" style="font-size:13px;">PROTOCOLS</A></LI><LI class="dropdown leftbookend rightbookend"><A href="/Team:UNC-Chapel_Hill/Team" style="font-size:13px;">OUR TEAM</A></LI><LI class="dropdown rightbookend"><A href="/Team:UNC-Chapel_Hill/Practices" style="font-size:13px;">HUMAN PRACTICES</A></LI><LI><A href="/Team:UNC-Chapel_Hill/Attributions" style="font-size:13px;">ATTRIBUTIONS</A></LI></CENTER></P><DIV class="spacer"><CENTER><TABLE width="975" cellspacing="0" height="10px"><TBODY><TR><TD colspan="3" width="975px" bgcolor="#56A0D3" align="center"><P style="color:white;font-size:20px">BACKGROUND </P></TD></TR></TBODY></TABLE></CENTER><CENTER><TABLE width="975" cellspacing="4" height="115px"><TBODY><TR><TD width="100%" height="4px" bgcolor="#FFFFFF"><H2 style="color:#56A0D3"> Project Overview</H2><P>Our team aims to create a novel sensing device that enables one to quantitatively characterize the concentration, or presence of certain compounds. The theoretical advantages to our design include; a wider range of sensitivity, better accuracy, customizability, and ability to detect more than one input at any given time.</P><P>Our design combines three promoters upstream of three different chromoproteins onto one plasmid. This allows for the advantages listed above (see Figure 1-label theoretical comparison- comparison between tricolor and single color). To provide an example of our general design, we decided to create a tri-color system for glucose. The first two promoters in our system are already apart of the iGEM registry.</P><P>The first promoter is part BBa_K118011, submitted to iGEM in 2008 by University of Edinburgh in Scotland. This promoter is similar to the familiar promoter for the lactose operon in E. coli cells (the same one as described above). However, BBa_K118011 only has a lone cAMP receptor protein (CRP) binding site, as opposed to having a CRP binding site and an operator site that binds to the lactose repressor protein. At extremely low glucose concentrations cAMP levels in the cell are high. In these conditions, cAMP associates with CRP forming a complex that binds to the binding site and activates high levels of transcription. In our construct, downstream of this glucose repressible promoter is efor Red (BBa_K1073023), which contains its own ribosomal binding site (RBS). The next promoter was also submitted previously by another team. In 2012, Wuhan University in China submitted part BBa_K861171, which is a glucose inducible promoter. The mechanism for this promoter can be thought of as the opposite of the first promoter. Instead of having a CRP binding site upstream of the promoter, the CRP binding site is located within the binding site for DNA polymerase. Thus, has the opposite effect as the first; when glucose is low, cAMP levels are high. Thus, cAMP associates with CRP and blocks transcription. As glucose concentrations rise, transcription is induced. In our construct, the chromoprotein associated with the second promoter is aeBlue (BBa_K1073021).</P><P>There was not another glucose inducible promoter in the iGEM regresistry, so our team created a novel promoter. This third promoter takes advantage of a repressor protein known as MLC, which is discussed below. The third promoter in our design is upstream of a yellow chromoprotein.</P><P>At extremely low concentrations of glucose, the action of the most sensitive promoter dominates, which is our device is always a repressible promoter upstream of a red chromoprotein. Thus, at low concentrations (of the input) the solution appears red. However, when concentrations increase, the red color diminishes and the second promoter starts becoming activated. This produces a blue color as the result of transcription of a blue chromoprotein. Lastly, the third promoter will activate at even higher concentrations resulting in the transcription of a yellow chromoprotein. At these high concentrations the blue and yellow colors will mix and appear green. This unique tri-color system allows for the quantitative measurement of concentrations because they are associated with visible colors.</P><H2 style="color:#56A0D3"> General Background on MLC (Makes Large Colonies) </H2><P>The third promoter created and was novel promoter to the iGEM parts registry, consisting of MLC binding sites. MLC is a regulatory protein that stands for “Making Large Colonies”, and is encoded by the gene dgsA. Specifically, MLC is a repressor regulator of many phosphoenolpyruvate-dependent carbohydrate phosphotransferase systems (PTSs). These are pathways for carbohydrate uptake (including glucose).</P><P>MLC binds directly to palindromic DNA sequences and blocks RNA polymerase from proceeding with transcription.1,2 In E. coli, MLC is involved in the regulation of many genes involving PTS, including the gene pstG, which encodes for the transmembrane glucose permease also called enzyme IICBGlu (regulation of pstG also involves a CRP, which activates transcription in the presence of glucose: it is thought that the two counter balance one another to provide the correct levels of transcription specifically for enzyme IICBGlu). The protein MLC has high intracellular concentration when glucose concentration is low and becomes sequestered to the cell membrane when glucose concentrations rise. As a result, a promoter with MLC binding sites should function as a glucose inducible promoter (repressing transcription at low glucose concentrations).</P><P>For our project, we synthesized four variants of the MLC promoter. The promoters vary in two characteristics: the strength of the promoter associated with the MLC region and the placement of the MLC region. Using the iGEM parts registry, the two promoters selected were BBa_J23112 (weak) and BBa_J23100 (strong) and one MLC binding region was placed either prior or after (post) the promoter (strong and weak refer to the relative frequency to which the promoters actually transcription, with strong being more frequent than weak). The appropriate BioBrick restriction sites will also be included (prefixes and suffixes so that 3A assembly protocol can be used for the MLC promoters). Also primer sequences were synthesized around the promoters so that the promoter could undergo PCR. The sequences for the MLC promoters and the primer are shown below (Figure 2).</P><H2 style="color:#56A0D3"> MLC Mechanism Expanded </H2><P> The MLC protein was first discovered by overproducing it in E. coli cells. This caused a reduction in the uptake of glucose, resulting in a decrease in colony size.<SUP>3</SUP> As stated previously, MLC is a repressor involved in the transcription regulation of many proteins involved in PTS, this is why MLC is known as a global repressor.<SUP>4</SUP></P><P>As stated before, MLC binds to palindromic DNA sequences and blocks transcription, but what stops the action of MLC? Why it is associated with an increase in glucose concentration? Sung‐Jae Lee et al, discovered that the active transport of glucose across the cell membrane (into the cell) results in a phosphorylation cascade that causes certain domains of enzyme IICB to bind to the MLC protein and sequester it to the cell membrane. Recall that enzyme IICB is a transmembrane glucose permease (it is interesting to note that in the presence of glucose IICB sequesters MLC to the plasma membrane, boosting its own transcription).<SUP>4</SUP></P><P>Sung‐Jae Lee et al discovered that when dephosphorylated, the IIB domain of enzyme IICB sequestered MLC to the membrane and that the IIC domain, which is the membrane spanning portion of the protein, is also necessary for the sequestration to occur (see Figure 3). This causes more enzyme IICB to be expressed (gene ptsG), which will cause the sequestration of even more MLC. This positive feedback loop results in the vamping up of enzyme IICB on the cell membrane and maximizes the amount of glucose transported into the cell. There have also been additional research that has identified another protein-protein interaction that MLC participates in. It has been shown that MLC binds directly to mtfA (originally known as yeeI).<SUP>5,6</SUP> Mutants of mtfA cause a decrease in the expression of ptsG, whereas overexpression causes constitutive expression. It has been proposed that mtfA is a peptidase specifically for MLC and helps to increase the transcription of ptsG and other PTS genes.<SUP>5,6</SUP></P><H2 style="color:#56A0D3"> References </H2><P> 1) Notley-McRobb L, Death A, Ferenci T. The relationship between external glucose concentration and cAMP levels inside Escherichia coli: implications for models of phosphotransferase-mediated regulation of adenylate cyclase. Microbiology. 1997;143:1909–1918. [PubMed]</P><P> 2) Plumbridge, Jacqueline. “DNA Binding Sites for the MLC and NagC Proteins: Regulation of nagE, Encoding the N-Acetylglucosamine-Specific Transporter inEscherichia Coli.” Nucleic Acids Research 29.2 (2001): 506–514. Print.</P><P> 3) Hosono K, Kakuda H and Ichihara S (1995) Decreasing accumulation of acetate in rich medium by Escherichia coli on introduction of genes on a multicopy plasmid. Biosci Biotechnol Biochem, 59, 256–261.</P><P> 4) Sung‐Jae Lee, Winfried Boos, Jean‐Pierre Bouché, Jacqueline Plumbridge (2000) Signal transduction between a membrane‐bound transporter, PtsG, and a soluble transcription factor, MLC, of Escherichia coli. The EMBO Journal 19, 5353-5361: 16.10.2000. Online.</P><P> 5) Elisabeth Gabora, 1, Anna-Katharina Göhlera, 1, Anne Kosfelda, 1, Ariane Staaba, Andreas Kremlingb, Knut Jahreisa. The phosphoenolpyruvate-dependent glucose–phosphotransferase system from Escherichia coli K-12 as the center of a network regulating carbohydrate flux in the cell. European Journal of Cell Biology Volume 90, Issue 9, September 2011, Pages 711–720: 28 May 2011. Online</P><P> 6) Anna-Katharina Göhler, Ariane Staab, Elisabeth Gabor, Karina Homann, Elisabeth Klang, Anne Kosfeld, Janna-Eleni Muus, Jana Selina Wulftange and Knut Jahreis. Characterization of MtfA, a Novel Regulatory Output Signal Protein of the Glucose-Phosphotransferase System in Escherichia coli K-12. Journal of Bacteriology. March 2012 vol. 194 no. 5 1024-1035. Online.</P></TD></TR></TBODY></TABLE></CENTER><FOOTER><A href="#" class="go-top">Top</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>