Team:ShanghaiTech/Protocol
ShanghaiTech iGEM
PROJECT
Overview Negetive Feedback Loop Orthogonal Ribosome Combination InterLab Measurement
PARTS
Basic Parts Composite Parts Part Collection Improved Part
MODELING
Overview Negative Feedback Loop Orthogonal Ribosome Computer Experiments Interaction between Data and Model Software
HUMAN PRACTICE
Overview Education & Public Engagement Biosafety Integrated Human Practice Collaborations
NOTEBOOK
Protocol Calendar
TEAM
Meet Our Team Attribution
I. PCR amplification
A. Common PCR amplification:
1. Use 2×PCR Master Mix product (version number: KT121221) produced by TIANGEN, and use the genome or plasmid DNA as the template. The reaction system is 25μl as follows:
Component
Volume per 25μl
Template
<1μg
Forward Primer
1μl
Reverse Primer
1μl
2×Taq PCR Master Mix
12.5μl
ddH2O
Add to total volume 25μl
2. Setting of PCR program:
Step
Temperature
Time
Cycles
Initial Denaturation
94°C
3 minutes
1
Denaturation
94°C
30 seconds
25-35
Annealing
Variable
30 seconds
 
Extension
72°C
1 minute
 
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
Note: The annealing temperature should be calulated according to the base number of (A+T) and (C+G).
3. Prepare 1% agarose gel in advance. If only product verification is performed, load 5μl of the PCR product on the agarose gel for electrophoresis; if the products obtained by PCR amplification are all used for gel extraction, 25μl (or larger volume) PCR products are loaded. The DNA bands are detected by ultraviolet photography observation.
Note: The final product usually has base A bulged at the 3' end.
B. High fidelity PCR amplification:
1. Use NEBNext® High-Fidelity 2X PCR Master Mix (M0541) produced by NEB, and use genomic or plasmid DNA as template The reaction system is 50μl as follows:
Component
Volume per 50 ul RXN
Final Concentration
NEBNext High Fidelity 2x PCR Master Mix
25 μl
1x
10 μM Forward Primer
2.5 μl
0.5 μM
10 μM Reverse Primer
2.5 μl
0.5 μM
Template DNA*
Variable
Variable
Nuclease-free water
To 50 μl
 
*1 ng - 1 μg genomic DNA or 1 pg - 1 ng for plasmid or viral DNA
Gently mix the reaction and briefly centrifuge, and then begin thermocycling.
2. Setting of PCR programs:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
5-10 seconds
25-35
Annealing
50-72°C
10-30 seconds
 
Extension
72°C
20-30 seconds/kb
 
Final Extension
72°C
2 minutes
1
Hold
4-10°C
∞
1
*Annealing temperature needs to be calculated according to the base sequence of primers. (Use NEB Tm Calculator .) Further optimization may be required.
3. Prepare 1% agarose gel in advance. If only product verification is performed, load 5μl of the PCR product on the agarose gel for electrophoresis; if the products obtained by PCR amplification are all used for gel extraction, 25μl (or larger volume) PCR products are loaded. The DNA bands are detected by ultraviolet photography observation.
Note: The final product obtained is the blunt end product (no 3’ bulged base ).
 
II. Agarose gel electrophoresis
1. Prepare 1L 1×TAE buffer in advance.
2. Prepare agarose gel (usually 0.5%~2% agarose concentration): Weigh a certain amount of agarose and measure a certain volume of 1×TAE buffer. After mixing the agarose with 1×TAE buffer in an Erlenmeyer flask, cover the Erlenmeyer flask with a plastic wrap (prevent the 1×TAE buffer from volatilizing) and heat it in a microwave oven for 1-2 minutes. If the agarose is not completely dissolved, continue heating until the agarose is completely dissolved.
3. Take out the Erlenmeyer flask, cool the liquid to about 55°C, then add nucleic acid dye to the gel and mix well immediately, and pour the liquid gel (not solidified gel) into the gel mold.
4. Add loading buffer to the DNA sample at an appropriate proportion and mix them well, then load samples into the gel. Load markers into the gel as needed.
5. Perform electrophoresis at the voltage of 100~120V for 30~50 min.
6. When the blue band of the loading buffer reaches the 2/3 length of the lane, stop the electrophoresis. Perform photograph observation under ultraviolate light.
 
III. The purification of DNA products
A. The purification of common DNA products
Use TIANquick Midi Purification Kit produced by TIANGEN Company to remove proteins, other organic compounds, inorganic salt ions, and oligonucleotide primers to purify 100bp~10kb DNA fragments.
Note: Add anhydrous ethanol to the rinse buffer PW before use.
1. Column equilibration: Add 500μl of equilibration solution BL to the adsorption column CB2 (adsorption column into the collection tube), centrifuge at 12,000 rpm (~13,400×g) for 1 minutes, and pour the waste liquid from the collection tube. Place the adsorption column CB2 in the collection tube.
2. Estimate the volume of the PCR product solution or the digested product solution (except for the volume of paraffin oil or mineral oil), add 5 times the volume of the binding solution PB, and mix well (without removing paraffin oil or mineral oil).
3. Add the solution from the previous step to a column CB2 (the column is placed in the collection tube), place it at room temperature for 2 minutes, centrifuge at 12,000 rpm (~13,400×g) for 30-60 seconds, and drain the waste from the collection tube. Place the adsorption column CB2 in the collection tube.
4. Add 600 μl of rinse buffer PW to the adsorption column CB2 (check whether anhydrous ethanol has been added before use), centrifuge at 12,000 rpm (~13,400×g) for 30-60 seconds, and drain the waste liquid from the collection tube. The adsorption column CB2 is placed in a collection tube.
5. Repeat step 4.
6. Place the adsorption column CB2 back into the collection tube and centrifuge at 12,000 rpm (~13,400×g) for 2 minutes to remove the rinse solution as much as possible. The column CB2 is allowed to stand at room temperature for a few minutes and thoroughly dried to prevent residual rinse from affecting the subsequent experiments.
7. Place the adsorption column CB2 in a clean centrifuge tube, and add 30-50 μl ddH2O to the middle of the adsorption membrane and leave it at room temperature for 2 minutes. Centrifuge the DNA solution at 12,000 rpm (~13,400 x g) for 2 minutes and collect DNA solution. In order to increase the amount of extracted DNA, the centrifuged solution can be re-added to the centrifugal adsorption column, placed at room temperature for 2 minutes, centrifuged at 12,000 rpm (~13,400×g) for 2 minutes, and collect DNA solution into a centrifuge tube. Store the DNA product at -20°C.
8. DNA concentration and purity detection:
The extracted DNA fragment can be detected for concentration and purity by agarose gel electrophoresis and an ultraviolet spectrophotometer. DNA should have a significant absorption peak at A260 (with an A260 value of 1 corresponding to approximately 50 μg/ml double-stranded DNA and 40 μg/ml single-stranded DNA). The A260/A280 ratio should be 1.7-1.9.
B. The purification of plasmid DNA from PCR products
Purification of (low copy) plasmid DNA is performed using the NucleoSpin® Gel and PCR Clean-up kit produced by NucleoSpin.
Before starting the preparation: Check if Wash Buffer NT3 has added the indicated volume of ethanol.
1. Adjust DNA binding condition:
Mix 1 volume of sample with 2 volumes of Buffer NTI.
Note: For small sample volumes ＜30 μl, adjust the volume of the reaction mixture to 50~100 μl with water. It is not necessary to remove mineral oil.
2. Bind DNA:
a. Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.
b. Centrifuge for 30 seconds at 11,000×g. Discard flow-through and place the column back into the collection tube. (Load remaining sample if necessary and repeat the centrifugation step.)
3. Wash silica membrane:
Add 700 μl Buffer NT3 to the NucleoSpin® Gel and PCR Clean-up Column. Centrifuge for 30 s at 11,000×g. Discard flow-through and place the column back into the collection tube. Repeat previous washing step to minimize chaotropic salt carry-over and improve A260/A280 values.
4. Dry silica membrane:
a. Centrifuge for 1 min at 11,000×g to remove Buffer NT3 completely. Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube.
b. Incubate the columns for 2~5 minutes at 70°C to totally remove ethanol.
5. Elute DNA:
Place the NuleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube. Heat ddH2O to 70°C in advance. Add 20~50 μl heated ddH2O and incubate at 70°C for 5 min. Centrifuge for 1 min at 11,000×g. In order to increase the amount of extracted DNA, the centrifuged solution can be re-added to the centrifugal adsorption column, centrifuged at 11,000×g for 2 min, and collect DNA solution into a centrifuge tube. Store the DNA product at -20°C.
6. DNA concentration and purity detection:
The extracted DNA fragment can be detected for concentration and purity by agarose gel electrophoresis and an ultraviolet spectrophotometer. DNA should have a significant absorption peak at A260 (with an A260 value of 1 corresponding to approximately 50 μg/ml double-stranded DNA and 40 μg/ml single-stranded DNA). The A260/A280 ratio should be 1.7~1.9.
 
IV. DNA extraction from agarose gels
Use the NucleoSpin® Gel and PCR Clean-up kit produced by NucleoSpin to extract the target DNA fragments in the agarose gel.
Before starting the preparation: Check if Wash Buffer NT3 has added the indicated volume of ethanol.
1. Excise DNA fragment/solubilize gel slice:
Note: Minimize UV exposure time to avoid damaging the DNA before DNA extraction.
a. Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose. Determine the weight of the gel slice and transfer it to a clean tube.
b. For each 100 mg of agarose gel ＜2%, add 200 μl Buffer NTI. For gels containing ＞2% agarose, double the volume of Buffer NTI.
c. Incubate sample for 5~10 min at 50 °C. Vortexing the sample briefly every 2~3 min until the gel slice is completely dissolved.
2. Bind DNA:
a. Place a NucleoSpin® Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 μl sample.
b. Centrifuge for 30 s at 11,000×g. Discard flow-through and place the column back into the collection tube. Load remaining sample if necessary and repeat the centrifugation step.
3. Wash silica membrane:
Add 700 μl Buffer NT3 to the NucleoSpin® Gel and PCR Clean-up Column. Centrifuge for 30 s at 11,000×g. Discard flow-through and place the column back into the collection tube. Repeat previous washing step to minimize chaotropic salt carry-over and low A260/A280.
4. Dry silica membrane:
a. Centrifuge for 1 min at 11,000×g to remove Buffer NT3 completely. (Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube.)
b. Incubate the columns for 2~5 min at 70 °C to totally remove ethanol.
5. Elute DNA:
Place the NuleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube. Heat ddH2O to 70°C in advance. Add 20~50 μl heated ddH2O and incubate at 70°C for 5 min. Centrifuge for 1 min at 11,000×g. In order to increase the amount of extracted DNA, the centrifuged solution can be re-added to the centrifugal adsorption column, centrifuged at 11,000×g for 2 min, and collect DNA solution into a centrifuge tube. Store the DNA product at -20°C.
6. DNA concentration and purity detection:
The extracted DNA fragment can be detected for purity by an ultraviolet spectrophotometer. DNA should have a significant absorption peak at A260 (with an A260 value of 1 corresponding to approximately 50 μg/ml double-stranded DNA and 40 μg/ml single-stranded DNA). The A260/A280 ratio should be 1.7~1.9.
 
V. Plasmid DNA extraction
Plasmid DNA is extracted (slow copy) from saturated E.coli LB culture using the NucleoSpin Plasmid/Plasmid (Nolid) kit produced by NucleoSpin.
Before starting the preparation: Check if Wash Buffer A4 has added the indicated volume of ethanol.
1. Cultivate and harvest bacterial cells:
Use 5~10 ml of a saturated E.coli LB culture, pellet cells in a standard benchtop microcentrifuge for 30 seconds at 11,000×g. Discard the supernatant and remove as much of the liquid as possible.
2. Cell lysis:
a. Add 500 μl Buffer A1. Resuspend the cell pellet completely by vortexing or pipetting up and down. (Make sure no cell clumps remain before addition of Buffer A2.)
b. Add 500 μl Buffer A2. Mix gently by inverting the tube 6~8 times. Do not vortex to avoid shearing genomic DNA. Incubate at room temperature for up to 5 min or until lysate appears clear.
c. Add 600 μl Buffer A3. Mix thorougnly by inverting the tube 6~8 times until blue samples turn colorless completely.
3. Clarification of lysate:
Centrifuge for 10 min at 11,000×g at room temperature.
4. Bind DNA:
a. Place a NucleoSpin® Plasmid/Plasmid Column in a Collection Tube (2 ml) and decant the supernatant form step 3 or pipette a maximum of 750 μl of the supernatant onto the column. Centrifuge for 1 min at 11,000×g. Discard flow-through and place the NucleoSpin® Plasmid Column back into the collection tube.
b. Repeat this step to load the remaining lysate.
5. Wash silica membrane:
a. Add 500 μl Buffer AW, optionally preheated to 50 °C, and centrifuge for 1 min at 11,000×g. Discard flow-through and place the NucleoSpin® Plasmid/Plasmid (Nolid) Column back into the collection tube.
b. Add 600 μl Buffer A4 (supplemented with ethanol). Centrifuge for 1 min at 11,000×g. Discard flow-through and place the NucleoSpin® Plasmid/Plasmid (Nolid) Column back into the empty collection tube.
6. Dry silica membrane:
a. Centrifuge for 2 min at 11,000×g and discard the collection tube.
b. Incubate the columns for 2~5 min at 70°C to totally remove ethanol.
7. Elute DNA:
Place the NuleoSpin® Gel and PCR Clean-up Column into a new 1.5 ml microcentrifuge tube. Heat ddH2O to 70°C in advance. Add 20~50 μl heated ddH2O and incubate at 70°C for 5 min. Centrifuge for 1 min at 11,000×g. In order to increase the amount of extracted DNA, the centrifuged solution can be re-added to the centrifugal adsorption column, centrifuged at 11,000×g for 2 min, and collect DNA solution into a centrifuge tube. Store the DNA product at -20°C.
8. DNA concentration and purity detection:
The extracted DNA fragment can be detected for concentration and purity by agarose gel electrophoresis and an ultraviolet spectrophotometer. DNA should have a significant absorption peak at A260 (with an A260 value of 1 corresponding to approximately 50 μg/ml double-stranded DNA and 40 μg/ml single-stranded DNA). The A260/A280 ratio should be 1.7~1.9.
 
VI. Bacterial genomic DNA extraction
Bacterial genomic DNA is extracted using the TIANamp Bacteria DNA Kit produced by TIANGEN.
Note: Add absolute ethanol to the buffer GD and the rinse buffer PW before use.
1. Centrifuge 1-5 ml of the bacterial culture solution at 10,000 rpm (~11,500×g) for 1 min. Discard the supernatant as much as possible.
2. Add 200 μl of buffer GA to the cell pellet and shake until the cells are completely suspended.
Note:[1] For Gram-positive bacteria, skip the second step and add lysozyme for wall-breaking treatment by adding 180 μl buffer (20 mM Tris, pH 8.0; 2 mM Na2-EDTA; 1.2% Triton; lysozyme at a final concentration of 20 mg/ml (lysozyme must be prepared by dissolving lysozyme dry powder in buffer, otherwise it will cause lysozyme to be inactive), and treated at 37°C for more than 30min.
[2]If RNA needs to be removed, add 4 μl of RNase A (100 mg/ml) solution, shake the mixture for 15 seconds, and leave the mixture at room temperature for 5 min.
3. Add 20 μl Proteinase K solution to the tube and mix.
4. Add 220 μl buffer GB, shake for 15 seconds, and leave at 70°C for 10 min. The mixture should turn into a clear solution. Centrifuge briefly to remove the water droplets on the inner wall of the cap.
5. Add 220 μl of absolute ethanol and mix well for 15 seconds. At the moment, flocculent precipitation may occur. Briefly centrifuge to remove the water droplets on the inner wall of the cap.
6. Add the solution and the flocculent precipitate obtained in the previous step to an adsorption column CB3 (the adsorption column is placed in the collection tube), centrifuge at 12,000 rpm (~13,400×g) for 30 seconds, discard the flow-through, and place the adsorption column CB3 into the collection tube.
7. Add 500 μl of buffer GD to the adsorption column CB3 (check if anhydrous ethanol has been added before use), centrifuge at 12,000 rpm (~13,400×g) for 30 seconds, discard flow-through, and place the adsorption column CB3 back into the collection tube.
8. Add 600 μl of rinse buffer PW to the adsorption column CB3 (check if anhydrous ethanol has been added before use), centrifuge at 12,000 rpm (~13,400×g) for 30 seconds, discard flow-through, and place the adsorption column CB3 into the collection tube.
9. Repeat step 8.
10. Place the adsorption column CB3 back into the collection tube, centrifuge at 12,000 rpm (~13,400×g) for 2 min, and discard the flow-through. Then leave the adsorption column CB3 at room temperature for a few minutes to thoroughly dry the residual rinse buffer in the adsorption membrane.
11. Place the adsorption column CB3 in a clean centrifuge tube, and add 30-50 μl ddH2O to the middle of the adsorption membrane and leave it at room temperature for 2 min. Centrifuge at 12,000 rpm (~13,400 x g) for 2 min to collect DNA solution. For maximum yield, it is recommended to reapply the eluate from the first elution on the adsorption column, place at room temperature for 2 min, centrifuge at 12,000 rpm (~13,400×g) for 2 min, and collect DNA solution into a centrifuge tube. Store the collected DNA product at -20°C.
12. DNA concentration and purity detection:
The extracted DNA fragment can be detected for concentration and purity by agarose gel electrophoresis and an ultraviolet spectrophotometer. DNA should have a significant absorption peak at A260 (with an A260 value of 1 corresponding to approximately 50 μg/ml double-stranded DNA and 40 μg/ml single-stranded DNA). The A260/A280 ratio should be 1.7-1.9.
 
VII. Making E.coli DH5α competent cells with calcium chloride
1. Streak bacterial cells such as E.coli DH5α on a LB plate. Incubate the plate at 37°C overnight (15-20 hours).
2. Inoculate a monoclonal colonies into 2~5 ml of liquid LB medium with a sterile tip or a sterile inoculator. Grow the culture in a shaker at 200-250 rpm at 37 °C overnight (12-15 hours).
3. Pre-warm the liquid LB medium in a flask containing 100-250 ml of LB or SOB medium at 37 °C for 30 min to one hour. Transfer 1 ml of overnight bacterial culture to a flask containing 100-250 ml of LB or SOB medium. Place the culture in a shaker at 200~250 rpm at 37°C,
4. Harvest the bacterial once the OD600 of the culture reaches 0.25-0.3. (This is very critical! If the bacterial culture cell density is too high, then we need to discard it and restart the new culture. Otherwise, the efficiency will be low.)
5. Add all bacterial culture in centrifuge tubes, and centrifuge at 3000 rpm for 15 min. Discard the supernatant, and remove most of the residual LB medium by using a sterile pipet in the hood.
6. Add 25 ml of 0.1 M MgCl2 solution to the bacterial pellet at 4 °C in the 50 ml centrifuge tube, and centrifuge at 4°C, 2000 rpm for 15 min. Discard the supernatant.
7. Add 25 ml of 0.1 M CaCl2 solution (add 5 ml first to resuspend the cells, then add another 20 ml) at 4 °C in the 50 ml centrifuge tube, and centrifuge at 4°C, 2000 rpm for 15 min. Discard the supernatant.
8. Mix 0.1M CaCl2 solution and 30% glycerol in a ratio of 1:1, and put the mixture on ice.
9. Add 2 ml of 1:1 CaCl2-glycerol mixture to the 50 ml centrifuge tube and mix gently by pipetting.
10. Prepare a few 1.5 ml EP tubes, add 200 μL of the mixture to each 1.5 ml EP tube, and store the competent cells at -80 °C.
 
VIII. Competent cell transformation
A. Chemical transformation of DH5α/BL21 competent cells
1. Place DH5α competent cells or BL21 competent cells in an ice bath (dispense 25 μl of competent cells into each 1.5 ml EP tube).
2. After thawing the competent cells, add 1~10 ng of plasmid DNA to the competent cell suspension (add appropriate amount of DNA according to the experimental requirements), gently tap the tube to mix (do not use pipette!), ice bath for 30 minutes.
3. Heat shock at 42°C for 60 seconds, quickly transfer the centrifuge tube to the ice bath, and leave it on ice for 5 minutes.
4. Add 200~500μl sterile SOC or LB medium (without antibiotics) to each centrifuge tube, mix and place in a shaker at 37°C, shake culture at 210~250 rpm for 30~45 minutes.
5. According to the experimental requirements, take 50~200μl of transformed competent cells, add to the SOC or LB solid medium with antibiotics, and spread the cells. Until the liquid is absorbed, invert the culture and incubate at 37°C for 15 to 20 hours.
B. Electrotransformation (electroporation) of BL21 competent cells
1. Thaw BL21 competent cells on ice.
2. Transfer 50 μl of BL21 competent cells to a pre-chilled electroporation cuvette with 1 mM gap.
3. Add 1 μl plasmid solution(or 3 times diluted assembly plasmid product) to electrocompetent cells.
4. Mix gently by pipetting up.
5. Once DNA is added to the cells, electroporation can be carried out immediately. It is not necessary to incubate DNA with cells.
6. Add 950 μl of room temperature SOC media to the cuvette immediately after electroporation.
7. Place the tube at 37°C for 60 minutes. Shake vigorously (250 rpm) or rotate.
8. Warm selection plates (SOC solid medium with specific antibiotics) to 37°C.
9. Spread 100 μl of the cells onto the plates.
10. Incubate overnight at 37°C (15~20 hours).
 
IX. Restriction endonuclease digestion
A. Pst I digestion
1. Using NEB Pst I restriction endonuclease (catalog number: R0140), prepare the following digestion reaction system:
Reagents
Volume/μl
Restriction endonuclease
1
DNA
1
10×NEBuffer
5
ddH2O
43
Total volume
50
2. Vortex the mixture briefly or mix it with a pipette. Centrifuge the mixture briefly.
3. Digest the mixture at 37°C for 15 minutes.
4. Inactivate at 80°C for 20 minutes.
5. Add 5μl 10×loading buffer, use pipette and mix it, and load 5~10 μl for agarose gel electrophoresis detection. Store the remaining product at -20°C.
B. Xba I digestion
1. Using NEB Xba I Restriction endonuclease (catalog number: R0145), prepare the following digestion reaction system:
Reagents
Volume/μl
Restriction endonuclease
1
DNA
1
10×NEBuffer
5
ddH2O
43
Total volume
50
2. Vortex the mixture briefly or mix it with a pipette. Centrifuge the mixture briefly.
3. Digest the mixture at 37°C for 15 minutes.
4. Inactivate at 65°C for 20 minutes.
5. Add 5μl 10×loading buffer, use pipette and mix it, and load 5~10 μl for agarose gel electrophoresis detection. Store the remaining product at -20°C.
C. EcoR I digestion
1. Using NEB EcoR I Restriction endonuclease (catalog number: R0101), prepare the following digestion reaction system:
Reagents
Volume/μl
Restriction endonuclease
1
DNA
1
10×NEBuffer
5
ddH2O
43
Total volume
50
2. Vortex the mixture briefly or mix it with a pipette. Centrifuge the mixture briefly.
3. Digest the mixture at 37°C for 15 minutes.
4. Inactivate at 65 °C for 20 minutes.
5. Add 5μl 10×loading buffer, use pipette and mix it, and load 5~10 μl for agarose gel electrophoresis detection. Store the remaining product at -20°C.
D. Hind III digestion
1. Using NEB Hind III Restriction endonuclease (catalog number: R0104), prepare the following digestion reaction system:
Reagents
Volume/μl
Restriction endonuclease
10 U (The amount of Restriction Endonuclease is suitable for all types of DNA.)
DNA
1
10×NEBuffer
5
ddH2O
43
Total volume
50
2. Vortex the mixture briefly or mix it with a pipette. Centrifuge the mixture briefly.
3. Digest the mixture at 37°C for 1 hour.
4. Inactivate at 80°C for 20 minutes.
5. Add 5μl 10×loading buffer, use pipette and mix it, and load 5~10 μl for agarose gel electrophoresis detection. Store the remaining product at -20°C.
 
X. DNA ligation
A. DNA ligation by T4 DNA ligase
Use T4 DNA ligase (catalog number: M0202) produced by NEB to catalyze the ligation of blunt ends or sticky ends of double-strand DNA, as well as repairing single-strand nicking of double-stranded DNA, RNA or DNA/RNA hybrid duplexes.
1. Set up the following reaction system in a microcentrifuge tube on ice.     (T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.) Use NEBioCalculator ( http://nebiocalculator.neb.com/#!/ligation ) to calculate molar ratios.
Components
20 μl Reaction
T4 DNA Ligase Buffer (10X)*
2 μl
Vector DNA (4 kb)
50 ng (0.020 pmol)
Insert DNA (1 kb)
37.5 ng (0.060 pmol)
Nuclease-free water
to 20 μl
T4 DNA Ligase
1 μl
* The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
2. Gently mix the reaction by pipetting up and down and microfuge briefly.
3. For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
4. For blunt ends or single base bulged, incubate at 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10-minute ligation).
5. Heat inactivate at 65°C for 10 minutes.
6. Chill on ice and transform 1-2 μl of the reaction into 25 μl competent cells. Store the remaining products at -20 °C.
B. DNA ligation by E.coli DNA ligase
Use E.coli DNA ligase (catalog number: M0205) produced by NEB to catalyze the ligation of blunt ends of double-strand DNA.
1. Set up the reaction system as follows:
DNA
up to 5 µg
E. coli DNA Ligase Reaction Buffer (10X)
2 µl
E. coli DNA Ligase
1 µl (10 units)
H2O
up to 20 µl
2. Incubate at 16°C for 30 minutes.     3. Heat inactivate by incubating at 65°C for 20 minutes.
4. Chill on ice and transform 1-2 μl of the reaction into 25 μl competent cells. Store the remaining products at -20 °C.
 
XI. Using NEB Gibson Assembly Master Mix (or NEBuilder HiFi DNA Assembly Master Mix) to recombine DNA fragments
1. Set up the following reaction on ice:
 
Recommended Amount of Fragments Used for Assembly
 
 
 
2-3 Fragment Assembly
4-6 Fragment Assembly
Positive Control
Total Amounts of Fragments
0.02-0.5 pmols[1]X μl
0.2-1 pmolsX μl
10 μl
Gibson Assembly Master Mix (2×)(Or: NEBuilder HiFi DNA Assembly Master Mix)
10 μl
10 μl
10 μl
Deionized H2O
(10-X) μl
(10-X) μl
0
Total Volume
20 μl[2]
20 μl[2]
20 μl
Note: [1] Optimized cloning efficiency is 50–100 ng of vectors with 2–3 fold of excess inserts. Use 5 times more of inserts if size is less than 200 bps. Total volume of unpurified PCR fragments in Gibson Assembly reaction should not exceed 20%.
[2] If greater numbers of fragments are assembled, additional Gibson Assembly Master Mix may be required.
2. Incubate samples in a thermocycler at 50°C for 15 minutes when 2 or 3 fragments are being assembled (or 60 minutes when 4-6 fragments are being assembled). Following incubation, store samples on ice or at –20°C for subsequent transformation.
Note: Extended incubation up to 60 minutes may help to improve assembly efficiency in some cases.
 
XII. Dpn I digestion (Removal of templates in PCR products or homologous recombination ligation products)
Note: When higher amounts of plasmid template must be used in the PCR reaction, it is recommended to digest the PCR product with Dpn I (NEB #R0176) restriction endonuclease in order to destroy plasmid template before setting up the Gibson Assembly reaction. Dpn I cleaves only E. coli Dam methylase-methylated plasmid DNA, but does not cleave the PCR product since it is not methylated.
Dpn I Digestion Protocol:
1. In a total 10 µl reaction, mix 5–8 μl of PCR products with 1 μl of 10×Cutsmart and 1 μl (20 units) of Dpn I.
2. Incubate at 37°C for 30 minutes.
3. Heat-inactivate Dpn I by incubating at 80°C for 20 minutes.
4. Place the Dpn I digested product on ice, and the digested product can then be directly transformed. If transformation can not be performed immediately, it needs to be stored at -20 °C.
 
XIII. Site-directed mutation
1. Prepare the control and sample reaction(s) as indicated below:
Note: Set up a series of sample reactions using various concentrations ranging from 5 to 50 ng of dsDNA template (e.g., 5, 10, 20, and 50 ng of dsDNA template).
Control Reaction
Sample Reaction
5 μl of 10× reaction buffer
5 μl of 10× reaction buffer
2 μl (10 ng) of pWhitescript 4.5-kb control template (5 ng/μl)
X μl (5–50 ng) of dsDNA template
1.25 μl (125 ng) of oligonucleotide control primer #1 [34-mer (100 ng/μl)]
X μl (125 ng) of oligonucleotide primer #1
1.25 μl (125 ng) of oligonucleotide control primer #2 [34-mer (100 ng/μl)]
X μl (125 ng) of oligonucleotide primer #2
1 μl of dNTP mix
1 μl of dNTP mix
ddH2O to a final volume of 50 μl
ddH2O to a final volume of 50 μl
2. Then add 1 μl of PfuTurbo DNA polymerase (2.5 U/μl) to each control and sample reaction.
3. Overlay each reaction with 30 μl of mineral oil.
4. Cycle each reaction using the cycling parameters outlined in the following table:
Segment
Cycles
Temperature
Time
1
1
95°C
30 seconds
2
12-18
95°C
30 seconds
55°C
1 minute
 
 
68°C
2 minutes/kb of plasmid length
 
 
5. Adjust segment 2 of the cycling parameters in accordance with the type of mutation desired:
Type of mutation desired
Number of cycles
Point mutations
12
Single amino acid changes
16
Multiple amino acid deletions and insertions
18
6. Add 1 μl of the Dpn I Restriction endonuclease (10 U/μl) below the mineral oil overlay.
7. Gently and thoroughly mix each reaction, spin down in a microcentrifuge for 1 minute, and immediately incubate at 37°C for 1 hour to digest the parental supercoiled dsDNA.
8. Transform 1 μl of the DpnI-treated DNA from each control and sample reaction into separate 50-μl aliquots of XL1-Blue supercompetent cells. Store the remaining product at -20 °C.
 
XIV. Preparation of T-vector ligation fragments and T-vector ligation
A. Preparation of T-vector ligation fragments
For PCR amplification of the DNA fragment of interest, if normal Taq DNA polymerase is used, the T-vector ligation can be directly performed (see protocol: III); if high-fidelity DNA polymerase is used, the PCR product need to be further processed after purification of the PCR product, and then perform the T-vector ligation. The processing method is as follows:
1. Prepare the reaction system:
1-7ul of purified PCR fragment produced by high fidelity DNA polymerase (eg: Pfu)
Add 1 ul of Taq DNA polymerase 10× reaction buffer (containing MgCl2);
Add dATP to a final concentration of 0.2 mM;
Add 5 U of Taq DNA polymerase;
Add ultrapure water to a final reaction volume of 10ul.
2. Mix well and centrifuge briefly, then incubate at 72°C for 10-20 minutes.
3. Purify the PCR product followed by T-vector ligation.
B. T-vector ligation
Use the Easy Cloning T-vector (pEC-T) kit (catalog number: T003) produced by Novoprotein to perfrom T-vector ligation.
1. Choose the appropriate connection system:
(1) Common ligation reaction system:
Components
Volume
10×Ligation Buffer
2 μl
pEC-T vector（25 ng/μl）
2 μl
Target PCR fragment / Control Insert DNA
X μl/1 μl
T4 DNA Ligase
1 μl
ddH2O
Add to 20 μl
Reaction condition: Incubate at 16 °C for more than 3 hours or overnight.
(2) Rapid connection reaction system:
Components
Volume
2×Quick Ligation Buffer
2 μl
pEC-T vector（25 ng/μl）
2 μl
Target PCR fragment / Control Insert DNA
X μl/1 μl
T4 DNA Ligase
1 μl
ddH2O
Add to 20 μl
Reaction condition: Incubate at 16 °C or 25 °C for 15-30 minutes.
2. Add 10 μl ligation product to 100 μl competent cells, gently tap the EP tube 4-5 times, and place on ice for 30 minutes.
3. Place the tube in a 42°C water bath, heat shock for 60-90 seconds, and quickly place the tube on ice for 2-3 minutes.
4. Add 500 μl sterile antibiotic-free LB or SOC medium to each centrifuge tube, shake the centrifuge tube containing bacterial fluid at 37°C, 150 rpm for 45 minutes.
5. Pipet 200 μl of transformed competent cells, add to LB or SOC solid plate containing ampicillin, spread the cells, and place the plate upside down in a 37°C incubator until the liquid is completely absorbed. Incubate the plates for 12-16 hours.
6. Perform PCR amplification to detect the product (using the high-speed PCR amplification reagent 2×FastTaq Mast Mix provided by the kit to directly performe colony PCR identification).
 
XV. Preparation of solution and buffer
A. 1×TAE buffer
If there is 50×TAE buffer, directly dilute to 1L 1×TAE buffer by adding 980 ml ddH2O per 20 ml. If there is no 50×TAE buffer, first prepare 50×TAE buffer (take 1L 50×TAE buffer as an example), and then dilute it to 1×TAE buffer, the steps are as follows:
1. Dissolve 242 g tris in 500 ml H2O.
2. Add 100 ml of 0.5 M Na2-EDTA (pH 8.0) and 57.1 ml glacial acetic acid.
3. Adjust volume to 1 liter with H2O. Store in the glass bottle with grinding mouth lid.
4. Do not autoclave. No need to sterilize. Store at room temperature. Dilute it to 1X TAE solution for making or running agarose gels.
B. Liquid LB medium
1. Mix 10 g Bacto-tryptone, 5 g Bacto-yeast extract and 10 g NaCl in 900 ml ddH2O.
2. Adjust the pH to 7.0 with 2M NaOH (This step can generally be omitted).
3. Adjust volume to 1 liter with ddH2O.
4. Sterilize by autoclaving (121°C, 20 min) and store at 4°C.
C. LB with agar for LB plates
1. Mix 10 g Bacto-tryptone, 5 g Bacto-yeast extract and 10 g NaCl in 900 ml ddH2O.
2. Adjust the pH to 7.0 with 2M NaOH (This step can generally be omitted).
3. Adjust volume to 1 liter with ddH2O. Add 15 g of agar. Mix by swiveling the bottle a few times before autoclaving.
4. Sterilize by autoclaving (121°C, 20 min).
5. After autoclaving: When naturally cooled to 50~60°C, add the corresponding antibiotic stock solution (usually added in a ratio of 1:1000), mix them and pour into a blank Petri dish.
D. Antibiotic stock solution
1. Prepare antibiotic solution according to the following table:
Antibiotics
1000×Stock Concentration（mg/ml）
Mass of antibiotics solids added per 10 ml solution（g/10ml）
Solvent
Ampicillin
100
1.0
ddH2O
Chloramphenicol
30
0.3
EtOH
Gentamycin
10
0.1
ddH2O
Kanamycin
50
0.5
ddH2O
Streptomycin
30
0.3
ddH2O
2. After the preparation of solution is completed, filter the solution through a 0.22 μM filter membrane in the ultra-clean workbench (Do not autoclave!), then dispense the solution into 1.5 ml EP tubes for 1 ml per tube, and store in -20°C refrigerator.
Note: The stock solution is usually diluted to the working solution in a ratio of 1:1000.
E. 1×PBS solution
Final concentration of 1×PBS buffer (pH 7.2 to 7.4): NaCl 137 mmol/L, KCl 2.7 mmol/L, Na2HPO4 10 mmol/L, KH2PO4 2 mmol/L. The preparation steps are as follows:
1. Add 8 g NaCl, 0.2 g KCl, 1.42 g Na2HPO4 and 0.27 g of KH2PO4 to 800 ml ddH2O, and stir to dissolve thoroughly.
2. Add concentrated hydrochloric acid to adjust the pH to 7.2~7.4.
3. Adjust volume to 1 liter with ddH2O.
4. Autoclave the sollution (121°C, 20 min), and then store it at 4°C (or stored at room temperature).
F. 50% (w/v) glycerol
Note: 50% (w/v) is the mass percent concentration.Preparing 50% (w/v) glycerol is to prepare 50 g/L glycerol.
1. Weigh 20 g of glycerol, and then add ddH2O to 400 ml volume.
2. Stir the solution until fully mixed.
3. Autoclave the solution (121°C, 20 min), and then store it at 4 °C.
 
ShanghaiTech iGEM @ 2018
