<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ShanghaiTech_Project_Oribosome skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ShanghaiTech/Project Oribosome</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>ShanghaiTech iGEM</TITLE></P><DIV class="collapse navbar-collapse justify-content-center" id="navbarSupportedContent"><UL class="navbar-nav"><LI class="nav-item dropdown" id="navbar-project"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Project_Overview">
        PROJECT
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Project_Overview">Overview</A><A class="dropdown-item" href="/Team:ShanghaiTech/Project_NFBL">Negetive Feedback Loop</A><A class="dropdown-item" href="/Team:ShanghaiTech/Project_Oribosome">Orthogonal Ribosome</A><A class="dropdown-item" href="/Team:ShanghaiTech/Project_Combination">Combination</A><A class="dropdown-item" href="/Team:ShanghaiTech/InterLab">InterLab</A><A class="dropdown-item" href="/Team:ShanghaiTech/Measurement">Measurement</A></DIV></LI><LI class="nav-item dropdown" id="navbar-parts"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="#">
        PARTS
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Basic_Part">Basic Parts</A><A class="dropdown-item" href="/Team:ShanghaiTech/Composite_Part">Composite Parts</A><A class="dropdown-item" href="/Team:ShanghaiTech/Part_Collection">Part Collection</A><A class="dropdown-item" href="/Team:ShanghaiTech/Improve">Improved Part</A></DIV></LI><LI class="nav-item dropdown" id="navbar-model"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Model">
        MODELING
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Model">Overview</A><A class="dropdown-item" href="/Team:ShanghaiTech/Model_NFBL">Negative Feedback Loop</A><A class="dropdown-item" href="/Team:ShanghaiTech/Model_Oribosome">Orthogonal Ribosome</A><A class="dropdown-item" href="/Team:ShanghaiTech/Computer_Experiments">Computer Experiments</A><A class="dropdown-item" href="/Team:ShanghaiTech/Model_Interaction">Interaction between Data and Model</A><A class="dropdown-item" href="/Team:ShanghaiTech/Software">Software</A></DIV></LI><LI class="nav-item dropdown" id="navbar-hp"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Human_Practices">
        HUMAN PRACTICE
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Human_Practices">Overview</A><A class="dropdown-item" href="/Team:ShanghaiTech/Public_Engagement">Education &amp; Public Engagement</A><A class="dropdown-item" href="/Team:ShanghaiTech/Biosafety">Biosafety</A><A class="dropdown-item" href="/Team:ShanghaiTech/Integrated_HP">Integrated Human Practice</A><A class="dropdown-item" href="/Team:ShanghaiTech/Collaborations">Collaborations</A></DIV></LI><LI class="nav-item dropdown" id="navbar-notebook"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="#">
        NOTEBOOK
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Protocol">Protocol</A><A class="dropdown-item" href="/Team:ShanghaiTech/Notebook">Calendar</A></DIV></LI><LI class="nav-item dropdown" id="navbar-team"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Team">
        TEAM
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Team">Meet Our Team</A><A class="dropdown-item" href="/Team:ShanghaiTech/Attributions">Attribution</A></DIV></LI></UL></DIV><DIV class="row"><DIV class="col-sm-9" style="padding-top: -30px; margin-top: -10px"><P>Orthogonal ribosomes are a kind of ribosomes that have been engineered and reprogrammed. In our project, we mainly focus on the binding interaction between mRNA and the 16S ribosomal RNA(rRNA), hoping to create an isolate translation system. In the phase of translation, ribosome small subunit find the RBS(ribosome binding site) on the mRNA to start to translate. While the altered 16S rRNA can only recognize and bind to an altered Shine-Dalgarno sequence, which then initiates the translation of the downstream gene. </P><P>Since the 16S rRNA and the SD sequence is mutated and different from the endogenous system, the orthogonal ribosome’s control of translation is separated from the host’s.</P><H3><STRONG>Key achievement</STRONG></H3><UL><LI>Examination of the characteristic of the orthogonal ribosome, including feasibility, orthogonality and compatibility with the ribosome of the host and the other orthogonal ribosome.</LI><LI>Prepration for the combination of our negative feedback loop and orthogonal ribosome.</LI><LI>Introduction of 2 new orthogonal ribosomes and 3 new orthogonal RBS to the iGEM community.</LI><LI>Update of the BioBrick Registry library by designing a tool kit plasmid containing orthogonal RBS that the target gene can easily cloned into. </LI></UL><H3>Overview</H3><P>Our engineered cells need a <STRONG>three-node Negative Feedback Loop</STRONG> to construct a more sensitive and high-fidelity control system.  </P><P>Nevertheless, since there is limited resource in the engineered cells, there has to be a resource competition between the host genome and exogenous target gene. This kind of competition will finally induce unexpected interaction which will result in negative correlation between the input signal and output signal. </P><P>This means, if we put the output gene downstream the part B, the transcription of the target gene will be well regulated, and the amount of mRNA will follow the input signal well. However, because of the resource competition mentioned before, the translational level regulation will be disrupted, in this case, the expression of the target gene is not as expected.</P><P class="text-center"><SMALL>Fig.1 Competition of ribosomes with host genome. </SMALL></P><H3>Our approach </H3><P>To solve this problem, we introduce orthogonal ribosome into our system. Since its great orthogonality to the host genome, we believe that it can reduce the unexpected interaction of genome and our target gene. </P><P class="text-center" style="margin-left: 100px; margin-right: 100px"><SMALL>Fig.2 Expected effects orthogonal ribosomes may achieve.</SMALL></P><P>The bacteria ribosome consists of rRNA and protein responsible for translation of mRNA to protein. The Shine-Dalgarno(SD) sequence of the mRNA is a major determinant of translation efficiency as the interaction between the SD sequence and the anti-Shine-Dalgarno(ASD) sequence through RNA-RNA base pairing. Therefore, to change the sequence of the SD sequence and the RBS sequence could isolate the gene downstream the mutant RBS from the host ones as the mutant ribosome could only translate the target genes theoretically. In this way, the resource competition could be solved, which means our output will follow the input accurately.</P><P class="text-center" style="margin-left: 100px; margin-right: 100px"><SMALL>Fig.3 Left: Structure of ribosome, pink stands for the Shine-Dalgarno sequence of its 16S rRNA. Protein Data Bank accession number 4V4P ; video created with Chimera. Right: Example mutation of SD and RBS sequence that give rise to orthogonal ribosome.</SMALL></P><P>Through desktop research, we found several different editions of orthogonal ribosomes. After consideration, we chose four pairs of orthogonal ribosomes and orthogonal RBS(ribosome binding site) to test and try to find the best to use in our system. They are labeled as <STRONG>A2, B8, C9</STRONG> from Jason Chin's lab and <STRONG>TJU</STRONG> from TJU_2012 iGEM team. Although no constructed plasmid was available from Jason’s or the TJU team, the way they mutated the host ribosome 16s rRNA were provided in the paper they published. Therefore, we are luckily to have the chance to construct these orthogonal ribosomes by mutagenesis in our own lab.</P><H3>Experiment design</H3><H4>Construction</H4><P>As the ribosome binds to the mRNA only depending on the small subunit 16s rRNA, we only need to construct orthogonal 16S rRNA. To obtain the orthogonal 16s rRNA, we first cloned <EM>E.coli</EM> genomic DNA 16s rRNA rrnB to T-Vector. According to the sequence provided in the paper or website, we mutated the Shine-Dalgarno site for every version of orthogonal ribosome. And for A2, B8, C9 orthogonal 16s rRNA from Jason Chin's lab, an extra mutation was performed from 722 to 723 bp of rrnB. The extra mutation helps the ribosome to form a bulge proximal to the minor groove of the SD helix formed between the ribosome and mRNA, which increase the orthogonal ribosome’s orthogonality. </P><P class="text-center"><SMALL>Fig.4 A schematic representation of the orthogonal ribosome mutation site. </SMALL></P><P class="text-center"><SMALL>Fig.5 The detailed mutation sequence of the orthogonal ribosome and orthogonal RBS pairs.</SMALL></P><P> </P><H4>Characterization</H4><P>We develop two kinds of circuits to characterize the orthogonal ribosome.Specifically, the host's ribosome or orthogonal ribosome pool utilised for translation is controlled by the selection of ribosome binding site (RBS). </P><P>For the first kind of circuits, there is only one reporter gene GFP on it.</P><P>Plasmid A</P><P>Plasmid B</P><P class="text-center"><SMALL>Fig.6  A schematic representation of the reporter plasmid that GFP is under orthogonal RBS within or without J23100- orthogonal 16S rRNA.</SMALL></P><P> </P><P><STRONG>A. </STRONG></P><P>We characterize orthogonal ribosome system in terms of GFP expression under the orthogonal RBS(ribosome binding site) in the absence or presence of corresponding orthogonal 16S rRNA to characterize its orthogonality to the host genome. </P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>orthogonal 16s rRNA</STRONG></TH><TH><STRONG>orthogonal RBS</STRONG></TH></TR></THEAD><TBODY><TR><TD>A2</TD><TD>A2</TD></TR><TR><TD>C9</TD><TD>C9</TD></TR><TR><TD>TJU</TD><TD>TJU</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>B.</STRONG></P><P>As we obtain so many different orthogonal ribosomes, it is natural for us to try to test the interaction between them.  With the increase of the complexity of the gene circuit, we may need two or more orthogonal ribosome to meet the need of diverge different parts into different pool to achieve the goal of the precise control  from input to output.</P><P>We characterize orthogonal ribosome system in terms of GFP expression under orthogonal RBS in the presence of different non-corresponding orthogonal 16S rRNA to characterize its orthogonality with each other.</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>orthogonal 16s rRNA</STRONG></TH><TH><STRONG>orthogonal RBS</STRONG></TH></TR></THEAD><TBODY><TR><TD>A2</TD><TD>B8</TD></TR><TR><TD>A2</TD><TD>C9</TD></TR><TR><TD>C9</TD><TD>A2</TD></TR><TR><TD>C9</TD><TD>B8</TD></TR><TR><TD>TJU</TD><TD>A2</TD></TR><TR><TD>TJU</TD><TD>B8</TD></TR><TR><TD>TJU</TD><TD>C9</TD></TR></TBODY></TABLE></FIGURE><P>For the second kind of circuits, there are two reporter genes on it. One is RFP with degradation LVA tag under host RBS, the other is GFP with degradation LVA tag under orthogonal RBS.</P><P>We developed two new circuits both consists of two parts: the original RFP pool and an orthogonal GFP pool. Genes encoding RFP is supposed to express as the host's 16s rRNA always exists, while the expression of GFP mainly depends on the presence of orthogonal 16s rRNA. </P><P>We charactize the difference between the expression of different reporter genes at the same time to get a more detailed view of how the orthogonal ribosome improve the precise control system.</P><P>Plasmid C</P><P>Plasmid D</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>orthogonal 16s rRNA</STRONG></TH><TH><STRONG>orthogonal RBS</STRONG></TH></TR></THEAD><TBODY><TR><TD>C9</TD><TD>C9</TD></TR><TR><TD>TJU</TD><TD>TJU</TD></TR></TBODY></TABLE></FIGURE><P class="text-center"><SMALL>Fig. 7  A schematic representation of the reporter plasmid that GFP is under orthogonal RBS while RFP is under host RBS.</SMALL></P><H3><STRONG>Results</STRONG></H3><H4>Construction</H4><P>The orthogonal ribosomes we successfully constructed by mutation are the A2, C9, TJU. They were all sequenced and cloned into pSB1C3 backbone either for further cloning or for submission. We met great difficulties on the mutation of orthogonal 16s rRNA B8 but we are stilling working on it.</P><P>The plasmids contains reporter gene GFP or RFP under the orthogonal ribosome(plasmid A and plasmid B) was directly synthesized by the third-party company. After getting the plasmid, we cloned the orthogonal 16S rRNA with a  constitutive promoter into it.</P><P> </P><H4>Characterization</H4><P><EM>For characterisation experiments, transformed cells were grown from stocks overnight, and subcultured for three to four hours, the cultures were diluted to 0.05 O.D. and aliquoted into a 96 wells plate or cell culture tubes. The fluorescence signal of GFP and RFP were monitered over time using a microplate reader. Fluorescence values were divided by absorbance values to give a series of normalised data. Both the original and normalised data were presented below.</EM></P><P><STRONG>Characterization experiment A</STRONG></P><P>The experiment aims to test the orthogonality of orthogonal ribosome and host ribosome.</P><P class="text-center"><SMALL>Fig.8 Characterisation of orthogonal ribosome A2,C9,TJU in DH5α <EM>E.coli</EM>. Flourescence is monitered over 6 hours by plate reader. </SMALL></P><P class="text-center" style="margin-left:100px; margin-right:100px"><SMALL>Fig.9 Characterisation of orthogonal RBS A2,C9,TJU in DH5α <EM>E.coli</EM> in absence of the corresponding orthogonal 16S rRNA. The C9 RBS shows the best orthogonality.</SMALL></P><P>The experiment results showed the great orthogonality of the orthogonal ribosome A2 and C9 we introduce and verified the function of the orthogonal ribosome introduced by Tianjin_2012 iGEM team. The fluorescence signal of GFP is much weaker without orthogonal 16S rRNA, but when the orthogonal 16S rRNA existing, the fluorescence signal increases over time.</P><P><STRONG>Characterization experiment B</STRONG></P><P class="text-center" style="margin-left:100px; margin-right: 100px "><SMALL>Fig.10 Characterisation of orthogonal ribosome system interaction in DH5α <EM>E.coli</EM>. Flourescence is monitered over 6 hours by plate reader when different orthogonal ribosomes are under different RBS.</SMALL></P><P>The experiment results did not fully meet the data shown on the previous paper.  While achieving  perfect proof of the orthogonality of the orthogonal ribosome to host ribosome, the crosstalk between each of the orthogonal ribosome was much more serious that expected. Especially for the TJU O-16S, it can nearly translate all the orthogonal RBS from Jason Chin's Lab. However, we are still luckily to find that the A2 orthogonal ribosome. It is relatively orthogonal to other orthogonal ribosome while have good affainty to its own RBS. Meanwhile, the A2 RBS is perfectly isolated from the host genome. </P><P>We infer that the test of the orthogonality in previous paper is based on whether the bacteria can survive when the resistant gene is under the orthogonal RBS. This creates a far more strict screening than our experiment since expressing GFP is not a great burden to the host bacteria.</P><P>For now, we just need one pair of orthogonal ribosome and RBS, so the interaction do not actually bother us.  But we will still other orthogonal ribosome for further construction.</P><P><STRONG>Characterization experiment C</STRONG></P><P>Experimental results showed that Jason's orthogonal pool sufficiently controlled the expression of GFP. The fluorescence signal of GFP was nearly close to zero without o-16S rRNA, but when the o-16S rRNA existed, the fluorescent protein showed the tendency to increase over time. Besides, the fluorescence signal of RFP increased with time as well, no matter whether the o-16S rRNA presented. Due to some unknown reasons, we received some very small, even negative values of red fluorescence signals in measurements at the very beginning of culturing cells. For better understanding, we moved upward our data to make the lowest value be zero, and the whole values of red fluorescence signals were elevated synchronously. Fluorescence values were divided by absorbance values to give a series of normalized data. Both the original and normalized data were presented below.</P><P class="text-center"><SMALL>Fig.11 A)   Cells that under the control of circuits described above, and transcript TJU's o-16S rRNA or not both grew well and similarly. B)  Cells that under the control of circuits described above, and transcript C9's orthogonal pool or not both grew well and similarly. Cells contain orthogonal pool seemed to reach the plateau a few hours later than cells without orthogonal pool.</SMALL></P><P class="text-center"><SMALL>Fig.12 C (1). C (2) Cells that under the control of circuits described above and do not transcript TJU's o-16S rRNA hardly expressed GFP without o-16S rRNA, while RFP was expressed well as the host's 16S rRNA existed. D (1). D (2) Cells that under the control of circuits described above and do not transcript C9's o-16S rRNA hardly expressed GFP without o-16S rRNA, while RFP was expressed well as the host's 16S rRNA existed. E (1). E (2) Cells that under the control of circuits described above and transcript C9's o-16S rRNA expressed GFP and RFP well, but the expression of GFP was much lower than that of RFP. E(3) Cells that under the control of circuits described above and transcript TJU's o-16S rRNA expressed GFP and RFP well, but the expression of RFP was much lower than that of GFP.</SMALL></P><P class="text-center"><SMALL>Fig.13 F (1). F (2) The expression level of GFP in two kinds of cells containing plasmids under the control of C9's orthogonal pool differed, depending on whether they had genes encoding o-16S rRNA. F(3) The expression level of GFP in two kinds of cells containing plasmids under the control of TJU's orthogonal pool differed, depending on whether they had genes encoding o-16S rRNA. G (1). G (2) The expression level of RFP in two kinds of cells containing plasmids under the control of C9's orthogonal pool stayed the similar expression level as time went by. G(3) The expression level of RFP in two kinds of cells containing plasmids under the control of TJU's orthogonal pool stayed the similar expression level as time went by.</SMALL></P><H3>Experience</H3><P>Though most of the open database will show the 16S RNA only contains 1542bp, it is still necessary to obtain about 300 bp more in the construction since it will help to form the 16S rRNA precursor, otherwise it may not function appropriately. </P><H3>References</H3><P>Alexander P. S. Darlington, Juhyun Kim, José I. Jiménez, Declan G. Bates. (2018) Dynamic allocation of orthogonal ribosomes facilitates uncoupling of co-expressed genes. Nature Communications 9:1. </P></DIV><DIV class="container"><SPAN class="text-muted">ShanghaiTech iGEM @ 2018</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>