Team:ShanghaiTech/Notebook
ShanghaiTech iGEM
PROJECT
Overview Negetive Feedback Loop Orthogonal Ribosome Combination InterLab Measurement
PARTS
Basic Parts Composite Parts Part Collection Improved Part
MODELING
Overview Negative Feedback Loop Orthogonal Ribosome Computer Experiments Interaction between Data and Model Software
HUMAN PRACTICE
Overview Education & Public Engagement Biosafety Integrated Human Practice Collaborations
NOTEBOOK
Protocol Calendar
TEAM
Meet Our Team Attribution
I: Verification of the function of LuxR element
In this part of our experiments, we verified the function of LuxR in our project. The gene circuit on the part (BBa_K2315024 in 2017 Part Library, from 2017 ShanghaiTech iGEM team) is pCon-LuxR-Ter-pLux-GFP-Ter in pSB1C3 backbone. In the gene curcuit, AHL (exogenously added inducer) and LuxR co-activates the promoter pLux and co-induce the expression of the GFP gene.
08/17/2018
PCR + gel extraction (Vector and Insert were separately extracted)
Double digestion
Reaction system:
Reagents
Volume/μl
Vector (at a very low concentration)
20
Insert (at a very low concentration)
20
Buffer(10x)
5
Endonuclease BamH I
1
Endonuclease Pst I
1
ddH2O
3
Total volume
50
The reaction system was incubated at 37°C for 2 hours.
Restriction Endonuclease
Reaction system:
Reagents
Volume/μl
Template mix (at a very low concentration)
40
T4 Buffer(10x)
5
T4 ligase
1
ddH2O
4
Total volume
50
The reaction system was incubated at 16°C overnight.
08/18/2018
Perform these steps:
Purification
Transformation (early morning)
Result: No colonies on the LB solid medium.
[Repeat The Following Steps]
PCR products purification (Vector + Insert)
Restriction endonuclease digestion
Reaction system:
Reagents
Volume/μl
Vector (at a very low concentration)
20
Insert (at a very low concentration)
20
Buffer(10x)
5
Endonuclease 1
1
Endonuclease 2
1
ddH2O
3
Total volume
50
The reaction system was incubated at 37°C for 2 hours.
Restriction Endonuclease
Reaction system:
Reagents
Volume/μl
Template mix (at a very low concentration)
40
T4 Buffer(10x)
5
T4 DNA ligase
1
ddH2O
4
Total volume
50
The reaction system was incubated at 16°C overnight.
08/19/2018
Perform these steps:
Purification
Transformation (products on Aug 18th)
Result: No colonies on the LB solid medium.
08/20/2018
Preparation of Lux-AHL Solution:
Preparation of stock solution: 25mg AHL + 1.17ml DMSO → Concentration of AHL is 100mM.
Preparation of test solution: about 3mg AHL + 200μl → Concentration of AHL is 70.3mM.
Dilution of test solution:
Dilute the test solution to the final concentration of 7×10-4 M, 7×10-6 M, 7×10-8 M, 7×10-10 M.
Inducer addition to the bacterial fluid of A-GFP and the bacterial fluid made by Wenhan Fu:
A-GFP①: Add 10μl 10-8M AHL to 1ml bacterial fluid (final concentration: 10-10M)
A-GFP① Blank: No AHL addition in the bacterial fluid.
A-GFP②: Add 10μl 10-6M AHL to 1ml bacterial fluid (final concentration: 10-8M)
A-GFP② Blank: No AHL addition in the bacterial fluid.
FWH: Add 10μl 10-8M AHL to 1ml bacterial fluid made by Wenhan Fu (final concentration: 10-10M).
FWH Blank: No AHL addition in the bacterial fluid.
Detection on an ELISA plate
08/21/2018
Digestion of two types of restriction endonucleases:
Reaction system:
Reagents
Volume/μl
Plasmid (Template)
5
Buffer(10x)
3
Endonuclease BamH I
1
Endonuclease Pst I
1
ddH2O
20
Total volume
30
The reaction system was incubated at 37°C for 2 hours.
Agarose gel electrophoresis (1% agarose gel, 120V, 45min) was performed to verify the DNA bands.
08/22/2018
The plasmid (used in the restriction endonuclease digestion) was sent for sequencing.
[Repeat the steps on Aug 17th]
PCR + gel extraction (Vector and Insert were separately extracted)
Double digestion
Reaction system:
Reagents
Volume/μl
Vector (at a very low concentration)
20
Insert (at a very low concentration)
20
Buffer(10x)
5
Endonuclease BamH I
1
Endonuclease Pst I
1
ddH2O
3
Total volume
50
The reaction system was incubated at 37°C for 2 hours.
Restriction Endonuclease Digestion
Reaction system:
Reagents
Volume/μl
Template mix (at a very low concentration)
40
T4 Buffer(10x)
5
T4 ligase
1
ddH2O
4
Total volume
50
The reaction system was incubated at 16°C overnight.
08/24/2018
Purification of PCR products on Aug 22nd and transformation of the purified product.
08/25/2018
Monoclonal colonies were picked and inoculated in liquid LB medium with chloramphenol at 37°C for 15 hours.
Small molecule experiment
Reaction system:
Reagents
Volume/μl
Bacterial liquid
5000
AHL (10ng/uL)
200
Reaction conditions: The reaction system mixture was shaken overnight at 37°C.
Result: No GFP fluorescence was observed.
08/27/2018
The plasmid (used in the restriction endonuclease digestion) was sent for sequencing.
08/29/2018
Purification and transformation of the plasmid constructed on Aug 22nd.
Small molecule experiment
Reaction system:
Reagents
Volume/μl
Bacterial liquid
5000
AHL (10ng/uL)
200
Reaction conditions: The reaction system mixture was shaken overnight at 37°C.
Result: No GFP fluorescence was observed.
II: Verification of the function of pT181 element
In this part of our experiments, we verified the function of pT181 element in our constructed plasmid (BBa_2787027 in 2018 Part Library). The gene circuit on the part is pCon-pT181 Antisense-Ter-pCon-pT181 Sense Target-GFP-Ter. The pT181 Antisense gene express pT181 Antisense, and pT181 Antisense binds the pT181 Sense Target, thus represses the expression of GFP gene.
07/02/2018
Transformation of synthesis pT181 plasmids (the vector is pUC57, and the plasmid was synthesized by GenScript, China) into E.coli MG1655 strains.
Result: No colonies on the LB solid medium.
07/03/2018
Transformation of synthesis pT181 plasmids (the vector is pUC57, the plasmid was synthesized by GenScript, China) into E.coli DH5α strains.
Result: There were a few colonies on all the four LB solid medium plates. Label the four plates as: Plasmid①, Plasmid②; TOP 10①, TOP 10②. The colonies were picked and inoculated into liquid LB mediums. Shake the bacterial fluids at 37°C overnight (12 hours).
(Note: “Plasmid” represents the control, and “TOP 10” represents the plasmid containing the target DNA sequence.)
07/05/2018
Plasmid DNA Extraction of Plasmid①, Plasmid②, TOP 10①, TOP 10②:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
Plasmid①
44.2
1.97
Plasmid②
38.3
1.92
TOP 10①
293.2
1.85
TOP 10②
158.4
1.77
07/07/2018
Plasmid DNA Extraction
 
Concentration/(ng/**μl)**
A260/A280
A260/A230
Plasmid①
20.7
1.89
1.92
Plasmid②
93.7
1.88
2.09
07/15/2018
Vector linearization by circular PCR amplification:
The templates were pUC57, and was labelled as pUC57①, pUC57②, pUC57③, pUC57④.
Reagent
Volume/μl
TIANGEN 2×PCR Master   Mix
25
Primer-F
2
Primer-R
2
Template
1
ddH2O
20
Total Volume
50
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
94°C
3 min
1
Denaturation
94°C
30 seconds
25-35
Annealing
55°C
30 seconds
Extension
72°C
1 min
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
07/16/2018
Gel extraction of linearized pUC57:
(Plasmid pUC57① and pUC57② were added into the same adsorption column to perform gel extraction, and the products of gel extraction was labelled as Blank①; plasmid pUC57① and pUC57② were added into the same adsorption column, and the products of gel extraction was labelled as Blank②.)
Name of Plasmids
Concentration/(ng/**μl)**
A260/A280
Blank①
67.7
1.88
Blank②
54.7
1.87
Transformation of synthesis pT181 plasmids (the vector is pUC57, and the plasmid was synthesized by GenScript, China) into E.coli DH5α strains.
07/17/2018
Genomic DNA Extranction of E.coli MG1655 strains: E.coli DH5α strains was inoculated into LB solid medium, and the bacterial fliud was shaken at 37°C overnight (12 hours). The bacterial fluids were used to perform genomic DNA extraction.
Using Gibson Assembly Master Mix to perform pT181 Assembly (products labelled as PA):
Reagent
Volume of PA/μl
Volume of Control/μl
Gibson Assembly Master Mix (2×)
10
10
Linearized Vector (Blank)
2.7
2.7
Insert (pT181)
0.5
0.5
ddH2O
6.8
6.8
Total Volume
20
20
Then transformation of PA and control was followed by, and inoculation of colonies and shaking bacterial fluids were performed after transformation.
07/18/2018
Plasmid DNA extraction of PA (pT181-Blank Assembly):
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
PA①
538.6
1.84
PA②
372.9
1.85
Then high fidility PCR amplification was performed to amplify the target DNA sequence. Then the extracted plasmids were sent for sequencing.
Result: No target DNA (inserted pT181) in the pUC57 vector.
07/20/2018
pT181-Blank Assembly (Repeated the experimen again)
Reagent
Volume of PA**①, PA②/μl**
Volume of Control/μl
Gibson Assembly Master Mix (2×)
10
10
Linearized Vector (Blank)
2.7
2.7
Insert (pT181)
0.9
0
ddH2O
7.4
8.3
Total Volume
21
21
07/22/2018
pT181-Blank-Assembly:
The DNA fragments pT181 and linearized vector (Blank) was assembled by Gibson Assembly Master Mix (2×), which is labelled PA. Two systems were made to perform the assembly: PA①, PA②.
Reagent
Volume of PA**①, PA②/μl**
Volume of Control/μl
Gibson Assembly Master Mix (2×)
10
0
Linearized Vector (Blank)
2.7
2.7
Insert (pT181)
0.5
0.5
ddH2O
6.8
6.8
Total Volume
20
10
PCR and Transformation:
25μl DH5α competent cells + 0.2/10/9.8μl pT181-Blank
Circular PCR amplification of PA → 500μl DH5α competent cells + 1μl PA → Pipette 50μl to spread the cells on a solid LB medium
07/23/2018
Bacterial fluid (Blank/PA): shake for 12 hours
07/25/2018
Bacterial fluid (Blank/PA): plate streak
Bacterial fluid (Blank/PA): shake for 12 hours
07/25/2018
PA/Blank: Agarose gel electrophoresis (120 V, 30 min) + Gel extraction
Name of DNA
Concentration/(ng/**μl)**
A260/A280
A260/A230
PA
36.3
1.82
0.89
Blank
30.0
1.83
0.16
08/06/2018
Blank/pT181-Assembly transformation into E.coli DH5α
08/08/2018
Plasmid DNA extranction of pT181-Assembly(PA)/Blank again:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
pT181 Assembly (PA)
505.9
1.89
2.36
pT181 Blank (Blank)
265.1
1.91
2.09
Performing 1% agarose gel electrophoresis for the two extracted plasmids above, and performing gel extraction. The results of gel extraction are as follows:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
PA
15.3
1.79
Blank
6.5
1.65
08/10/2018
Colony PCR and Gel Electrophoresis of PCR Products
Templates: Blank 1, Blank 2, Blank 3, PA 1
Reagent
Volume/μl
TIANGEN 2×PCR Master   Mix
10
Primer-F
0.8
Primer-R
0.8
Template
0.8
ddH2O
7.6
Total Volume
20
Results of electrophoresis: No expected electrophoresis band.
08/11/2018
Plasmid PCR verification and Gel Electrophoresis of PCR Products
Templates: Blank pT181 top②, pT181 Assembly②; pT181 Blank (in plasmid pUC57), pT181 Assembly (in plasmid pUC57)
Reagent
Volume/μl
TIANGEN 2×PCR Master Mix
10
Primer-F
0.8
Primer-R
0.8
Template
0.8
ddH2O
7.6
Total Volume
20
08/12/2018
Dilution, Spreading Plates, followed by Picking Monoclonal Colonies and Shaking bacterial fluids.
Plates of the following bacterial fluids were diluted and spread: Blank (7.23)①; Blank (7.23)③; Blank (7.26)③; PA (7.26)②; PA (7.26)③.
Monoclonal colonies were picked and bacterial fluids were shaken after 17 hours.
Picking a Monoclonal Colony and Shaking bacterial fluids
Pick a Monoclonal Colony and Shake bacteria of the following bacterial fluid: Blank/PA (7.22), Blank/PA (8.8)
08/17/2018
Plasmid DNA Extraction
Plasmid DNA extraction of the bacterial fluids on August 12th : PA (7.22)①, PA (8.8)②, Blank (7.22)②, Blank (8.8)②
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
PA (7.22)①
426.5
1.89
2.28
PA (8.8)②
370.7
1.88
2.28
Blank (7.22)②
586.5
1.88
2.36
Blank (8.8)②
497.5
1.89
2.34
Gibson Assembly
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Insert
1.8
Vector
0.6
ddH2O
7.6
Total Volume
20
08/18/2018
Plasmid PCR verification and Gel Electrophoresis of PCR Products
Templates: PA (7.22)①, PA (8.8)②, Blank (7.22)②, Blank (8.8)② (extracted in August 17th )
Reagent
Volume/μl
TIANGEN 2×PCR Master   Mix
10
Primer-F
0.8
Primer-R
0.8
Template
0.8
ddH2O
7.6
Total Volume
20
Agarose gel electrophoresis: 120 V, 30 min.
III: Verification of the function of STAR element
In this part of our experiments, we verified the function of STAR element in the plasmid courtsey of 2018 SJTU-BioX-Shanghai (The sequence of Antisense and STAR Target was from 2016 Imperial College London). The gene circuit on the part is pCons-STAR Antisense-t500 (between the two multi-cleavage sites on both sides of the gene circuit) and pCons-STAR Target-GFP or pCons-STAR Target-RFP. The STAR Antisense gene express STAR Antisense, and STAR Antisense binds the STAR target, thus activates the expression of GFP gene or RFP gene.
07/05/2018
Plate spread of three kinds of competent cells containing different plasmids courtesy of SJTU.
Result: No colony on each of the three plates after 15 hours.
07/06/2018
Transformation of the three plasmids courtesy of SJTU.
Result: A few colonies grew on the LB plates.
07/07/2018
Picking a Monoclonal Colony and Shaking bacterial fluids
Monoclonal Colonies were picked and inoculated liquid LB mediums with streptomycin antibiotics,  and bacterial fluids were shaken at 37°C for 12 hours (three tubes were shaken for each plasmid PCDF1, PCDF3, T1RA1): PCDF1-1, PCDF1-2, PCDF1-3; PCDF3-1, PCDF3-2, PCDF3-3; T1RA1-1, T1RA1-2, T1RA1-3.
Plasmid DNA Extraction
Plasmid DNA extraction of the bacterial fluids: PCDF1-1, PCDF1-2, PCDF1-3; PCDF3-1, PCDF3-2, PCDF3-3; T1RA1-1, T1RA1-2, T1RA1-3.
Name of Plasmid
Concentration/(ng/μl)
A260/A280
PCDF1-1
51.5
1.92
PCDF1-2
128.4
1.90
PCDF1-3
351.9
1.86
PCDF3-1
161.3
1.88
PCDF3-2
230.7
1.89
PCDF3-3
265.4
1.84
T1RA1-1
84.4
1.97
T1RA1-2
72.5
1.98
T1RA1-3
93.3
1.98
These plasmids were then stored at -20°C, and they were sequenced right by sequencing company.
07/09/2018
Fluorescence Intensity Detection By a Microplate Reader
The bacterial liquids were added to 1.5ml EP tubes, and were centrifuged at 12,000 rpm for 5 min; the supernatant was discarded, and 200μl 1×PBS solution was added to the bacterial precipitate; resuspend the precipitate and add the resuspension solution to the wells on a 96-well ELISA plate; fluorescence intensity was detected by a microplate reader.
Shaking bacterial fluids
The detected bacterial fluids above were inoculated in liquid LB medium with streptomycin.
07/15/2018
Strain Recovery and Fluorescence Intensity Detection By a Microplate Reader
The DH5α E.coli strains containg following plasmids were recovered at 37°C for 20 min: PCDF1-1, PCDF1-2, PCDF1-3; PCDF3-1, PCDF3-2, PCDF3-3. Then the DH5α E.coli strain solutions were centrifuged at 12,000 rpm for 2 min, discard the supernatant, and resuspend the precipitate by addition of 200μl 1×PBS solution. Finally 200μl resuspended solution were added in the wells of a 96-well ELISA plate to detect fluorescence intensity of each solution.
07/26/2018
Shake Bacterial Fluids To Amplify Plasmid DNA
The following bacterial fluids were shaked overnight (12 hours): sfPCDF1-1, sfPCDF1-2, sfPCDF1-3; sfPCDF3-1, sfPCDF3-2, sfPCDF3-3.
Mix the bacterial fluids evenly, pick a monoclonal colony and inoculate it into 5ml LB medium with streptomycin antibiotics. Shake the bacterial fluid at 37°C overnight (12 hours).
07/27/2018
Microplate reader detection of GFP fluorescent expression results
The following bacterial fluids were performed to extract DNA plasmids: sfPCDF1-1, sfPCDF1-2, sfPCDF1-3; sfPCDF3-1, sfPCDF3-2, sfPCDF3-3; Blank 2-1, Blank 2-2; PA 2-1; PA 2-2.
For the bacterial fliuds above, 1.4 ml of each were centrifuged at 5,000 rpm, followed by discard of supernatants. Then 1.4 ml 1×PBS solution were added to resuspend the bacteria precipitate, and the resuspended solutions were added to the wells on a 96-well ELISA plate to detect the GFP fluorescence intensity.
The resuspended solutions of the following bacterial fluids were also aliquoted 5 μl to make a microscope mounts for viewing GFP fluorescence image: sfPCDF1-2, sfPCDF3-1, Blank 1, PA 2.
IV: Assembly of module A, B, C and orthogonal ribosome 16S DNA
In this part of our experiments, we assembled three gene circuit modules (module A, module B, module C) and orthogonal ribosome 16S DNA. The following plasmids were synthesized by GenScript, China:
Plasmid A: Gene circuit system containing module A, module B, module C and homologous sequence of the start and end of GFP gene (in pUC57 backbone).
Plasmid B: Gene circuit system containing module A, module B, module C and homologous sequence of the start and end of orthogonal ribosome 16S gene (in pUC57 backbone).
We assembled GFP gene sequence on Plasmid A, and assembled orthogonal ribosome 16S gene sequence on Plasmid B.
08/06/2018
PCR amplifications were performed to amplify four target fragments: h16S DNA sequence (labelled as “h16S”), GFP gene sequence (labelled as “GFP”),
Labels
Corresponding Amplified Fragments
Templates
h16S
h16S gene sequence
MG1655 genomic DNA
GFP
GFP gene sequence
Plasmid BBa_K2315022    (pCon-LuxR-pRpa-GFP)
R-G
Linearized vector of pUC57 containing GFP gene homologous sequence (start   and terminate position)
pUC57 containing GFP gene homologous sequence (start and terminate   position) (This vector is synthesized by GenScript, China)
O-ribo
Linearized vector of pUC57 containing h16S gene homologous sequence   (start and terminate position)
pUC57 containing h16S gene homologous sequence (start and terminate   position) (This vector is synthesized by GenScript, China)
PCR amplification of four fragments above was performed. The reaction systems were as follows:
Reagent
Volume/μl
NEB HiFi 2×PCR Master   Mix
25
Primer-F
2.5
Primer-R
2.5
Template
1
ddH2O
19
Total Volume
50
Programs of PCR amplifier:
(1) h16S:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
35
Annealing
67°C
30 seconds
Extension
72°C
90 seconds
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
(2) GFP:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
35
Annealing
55°C
30 seconds
Extension
72°C
30 seconds
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
(3) R-G:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
35
Annealing
65°C
30 seconds
Extension
72°C
2 minutes
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
(4) O-ribo:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
35
Annealing
65°C
30 seconds
Extension
72°C
2 minutes
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
08/07/2018
1% Agarose Gel Electrophoresis of PCR products (h16S, GFP, R-G, O-ribo) was performed at 120V for 45 min, then performed gel extraction. As there was no target band on the agarose gel, the results of gel extraction of the other tthree fragments was:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
h16S
11.5
2.79
0.01
GFP
8.0
2.08
1.81
O-ribo
4.1
2.11
0.03
Gibson Assembly of two fragments h16S and O-ribo (labelled as “h16S-O-ribo”)
Reagent
Volume/μl
Gibson Assembly   Master Mix (2×)
10
O-ribo
8.5
h16S
2.5
Total Volume
21
08/08/2018
Transformation of assembled products h16S-O-ribo into E.coli DH5α competent cells on the LB solid medium (Three with ampicillin, the other three with no antibiotics)
Results: High density colonies on the three LB solid medium plates with no antibiotics, but no colony on the three LB solid medium plates.
PCR amplification of pUC57 containing GFP gene homologous sequence
This time the forward primer (primer-F) and the reverse primer (primer-R) were changed to repeat PCR amplification.
Gel electrophoresis and gel extraction
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
R-G①
16.2
1.76
0.14
R-N-G①
12.3
1.82
0.21
08/09/2018
PCR products purification
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
h16S
220.0
1.84
1.67
GFP
132.6
1.68
0.88
R-G②
32.8
1.74
0.81
R-N-G②
36.6
1.67
1.00
O-ribo
25.3
1.46
1.02
Gibson Assembly of each products
Two gel extraction products were Gibson Assembled as the following reaction system:
Reagent
Volume/μl
Gibson Mix Courtesy of Haopeng Wang’s Lab
15
Vector (R-G① or R-N-G①)
4
Insert (GFP)
1
Total Volume
20
Five purified PCR products were Gibson Assembled as the following reaction system:
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Vector (R-G② or R-N-G② or O-ribo)
5
Insert (GFP)
5
Total Volume
20
Reagent
Volume/μl
Gibson Mix Courtesy of Haopeng Wang’s Lab
15
Vector (R-G② or R-N-G② or O-ribo)
4
Insert (GFP)
1
Total Volume
20
Then all reaction systems were incubated at 50°C for 4 hours.
08/10/2018
Transformation of the Gibson products of Aug 9th into E.coli DH5α Competent cells.
Result: No colonies on each LB solid medium plate.
08/11/2018~08/15/2018
Repeat PCR amplification, 1% agarose gel electrophoresis, gel extraction and Gibson assembly protocols from Aug 6th to Aug 9th .
The gel extraction results:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
h16S
75.5
1.79
1.23
GFP
36.7
1.79
1.92
R-G
10.5
1.71
0.05
O-ribo
15.5
1.73
0.48
The Gibson Assembly reaction systems:
(Reaction systems: G1: GFP + R-G; G2: GFP + R-G; G3: h16S + O-ribo; G4: h16S + O-ribo)
Reagent
Volume of G1/μl
Volume of G2/μl
Volume of G3/μl
Volume of G4/μl
Gibson Assembly Master Mix (2×)
10
10
10
10
Vector (R-G)
6
8
6
8
Insert (GFP)
4
2
4
2
Total Volume
20
20
20
20
Then performed transformation of the Gibson products into E.coli DH5α Competent cells.
Result: No colonies on each LB solid medium plate.
08/16/2018~08/22/2018
Repeat PCR amplification and 1% agarose gel electrophoresis of R-G and O-ribo, and gel extraction of the R-G. and Gibson assembly protocols from Aug 6th to Aug 9th .
The reaction systems of PCR amplification:
Reagent
Volume/μl
NEB HiFi 2×PCR Master   Mix
25
Primer-F
2.5
Primer-R
2.5
Template
1
ddH2O
19
Total Volume
50
The programs of PCR amplifier:
(1) R-G:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
30
Annealing
65°C
30 seconds
Extension
72°C
2 minutes
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
(2) O-ribo:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
30
Annealing
65°C
30 seconds
Extension
72°C
2 minutes
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
Then the 1% agarose gel electrophoresis was followed by.
Results: the expected R-G bands were observed on the agarose gel, but there were no expected O-ribo bands.
The gel extraction results:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
R-G
18.8
1.82
0.04
The Gibson Assembly reaction systems:
(Reaction systems: G1: GFP + R-G; G2: h16S + O-ribo)
Reagent
Volume of G1/μl
Volume of G2/μl
Gibson Assembly Master Mix (2×)
10
10
Vector
4.19
4.2
Insert
0.98
1
ddH2O
4.83
4.8
Total Volume
20
20
Then gradient transformation was transformed of the Gibson products (GFP + R-G, labelled as “GFP”) into E.coli DH5α Competent cells.
Transformation in gradient:
Group 1: 0.2μl plasmid + 25μl E.coli DH5α Competent cells (labelled as “0.2GFP-①”“0.2GFP-②”“0.2GFP-③”)
Group 2: 0.5μl plasmid + 25μl E.coli DH5α Competent cells (labelled as “0.5GFP-①”“0.5GFP-②”“0.5GFP-③”)
Group 3: 1μl plasmid + 25μl E.coli DH5α Competent cells (labelled as “1GFP-①”“1GFP-②”“1GFP-③”)
Result: High-density colonies were performed on each group of LB solid medium plates.
The colonies were picked and inoculated to liquid LB medium with chloramphenicol antibiotics, followed by plasmid DNA extraction.
The results of plasmid DNA extraction:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
0.2GFP-①
524.5
1.89
2.38
0.2GFP-②
600.0
1.89
2.38
0.5GFP-①
589.8
1.87
2.36
0.5GFP-②
475.0
1.91
2.35
0.5GFP-③
500.7
1.91
2.32
1GFP-①
388.0
1.86
2.35
1GFP-②
437.5
1.88
2.34
1GFP-③
491.5
1.87
2.36
Note: The bacterial fluid of 0.2GFP-③ wasn’t observed turbidity.
08/21/2018~08/29/2018
Repeat the following protocols (for many times) from Aug 21st to Aug 29th : PCR amplification and 1% agarose gel electrophoresis of R-G, and gel extraction of the R-G. and Gibson assembly protocols from Aug 6th to Aug 9th .
The labels of each assembly products were as follows:
Label
Protocol of Fragments Assembly
GHD
R-G + GFP: HiFi Assembly → DpnI Digestion
GGD
R-G + GFP: Gibson Assembly → DpnI Digestion
GHN
R-G + GFP: HiFi Assembly → No Digestion
PCR amplification of R-G:
The reaction systems of PCR amplification:
Reagent
Volume/μl
NEB HiFi 2×PCR Master   Mix
25
Primer-F
2.5
Primer-R
2.5
Template
1
ddH2O
19
Total Volume
50
The programs of PCR amplification were the same as that in Aug 6th:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
35
Annealing
55°C
30 seconds
Extension
72°C
30 seconds
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
Then the 1% agarose gel electrophoresis was followed by.
Results: The expected R-G bands were observed on the agarose gel.
Gel extraction was performed after 1% agarose gel electrophoresis. The gel extraction results:
Name of Plasmid
Concentration/(ng/μl)
A260/A280
A260/A230
R-G
48.0
1.84
0.38
The HiFi/Gibson Assembly reaction systems:
(Reaction systems: G1: GFP + R-G; G2: h16S + O-ribo)
Reagent
Volume of GHN/μl
Volume of GGN/μl
Gibson Assembly Master Mix (2×)
10
10
Vector
3.2
3.2
Insert
1.2
1.2
ddH2O
5.6
5.2
Total Volume
20
20
Then DpnI digestion was performed as the following reaction systems:
Reagent
Volume of GHD/μl
Volume of GGD/μl
Hifi/Gibson Assembly Products
8
8
10×Cutsmart
1
1
DpnI
1
1
Total Volume
10
10
Then gradient transformation was transformed of the DpnI digestion products into E.coli DH5α Competent cells.
Transformation in gradient (or in different groups):
Group 1: GHD① 0.5μl + 25μl E.coli DH5α Competent cells
Group 2: GGD① 1.5μl + 25μl E.coli DH5α Competent cells
Group 3: GHD② 0.2μl/0.5μl/1μl + 25μl E.coli DH5α Competent cells
Group 4: GGD② 0.7μl/1.5μl/2.5μl + 25μl E.coli DH5α Competent cells
Result: High-density colonies were performed on each group of LB solid medium plates.
The colonies were picked and inoculated to liquid LB medium with ampicillin antibiotics, followed by plasmid DNA extraction.
The results of plasmid DNA extraction:
Name of Plasmid
Concentration/(ng/μl)
A260/A280
A260/A230
0.2GFP-①
524.5
1.89
2.38
0.2GFP-②
600.0
1.89
2.38
0.5GFP-①
589.8
1.87
2.36
0.5GFP-②
475.0
1.91
2.35
0.5GFP-③
500.7
1.91
2.32
1GFP-①
388.0
1.86
2.35
1GFP-②
437.5
1.88
2.34
1GFP-③
491.5
1.87
2.36
Note: The bacterial fluid of 0.2GFP-③ wasn’t observed turbidity.
08/21/2018~08/29/2018
Repeat the following protocols (for many times) from Aug 21st to Aug 29th : PCR amplification and 1% agarose gel electrophoresis of O-ribo and h16S, and gel extraction of the R-G. and Gibson assembly protocols from Aug 6th to Aug 9th .
The labels of each assembly products were as follows:
Label
Protocol of Fragments Assembly
OHD
O-ribo + h16S: HiFi Assembly → DpnI Digestion
OGD
O-ribo + h16S: Gibson Assembly → DpnI Digestion
OHN
O-ribo + h16S: HiFi Assembly → No Digestion
o16S-Lin
Linearized vector of pUC57 containing h16S gene homologous sequence   (start and terminate position)
PCR amplification of R-G:
The reaction systems of PCR amplification:
Reagent
Volume/μl
NEB HiFi 2×PCR Master   Mix
25
Primer-F
2.5
Primer-R
2.5
Template
1
ddH2O
19
Total Volume
50
The programs of PCR amplification were the same as that in Aug 6th:
Step
Temperature
Time
Cycles
Initial Denaturation
98°C
30 seconds
1
Denaturation
98°C
10 seconds
40
Annealing
65°C
30 seconds
Extension
72°C
2 minutes
Final Extension
72°C
2 minutes
1
Hold
4°C
∞
1
Then the 1% agarose gel electrophoresis was followed by.
Results: The expected R-G bands were observed on the agarose gel.
Gel extraction was performed after 1% agarose gel electrophoresis. The gel extraction results:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
o16S-Lin
35.2
1.65
0.19
Overlap PCR was performed with h16S and o16S-Lin (The product was labelled as “o-ribo①”“o-ribo②”):
Reagants
Volume in o-ribo①/μl
Volume in o-ribo②/μl
NEB HiFi 2×PCR Master Mix
10
10
Insert
5
5
Vector
1
5
ddH2O
4
0
Total Volume
20
20
Then DpnI digestion was performed as the following reaction systems (The product was labelled as “o-ribo-I”“o-ribo-II”):
Reagent
Volume of o-ribo-I/μl
Volume of o-ribo-II/μl
Hifi/Gibson Assembly Products   (o-ribo①/o-ribo②)
8
8
10×Cutsmart
1
1
DpnI
1
1
Total Volume
10
10
Then transformation was transformed of the DpnI digestion products into E.coli DH5α Competent cells:
Group 1: 1.5μl o-ribo-I + 25μl E.coli DH5α Competent cells;
Group 2: 1.5μl o-ribo-II + 25μl E.coli DH5α Competent cells.
Result: A few colonies were performed on each group of LB solid medium plates.
The colonies were picked and inoculated to liquid LB medium with ampicillin antibiotics, followed by plasmid DNA extraction and sending extracted DNA for sequencing (many times).
V: Construction of a plasmid to validate the function of orthogonal ribosome
 
In this part of our experiments, we constructed the plasmid (BBa_K2787021 in 2018 Part Library) to validate the function of orthogonal ribosome. The part we constructed is pCon-hRBS-RFP-Ter-pCon-oRBS-GFP-Ter in pSB1C3 backbone. The orthogonality was verified by GFP fluorescence intensity and RFP fluorescence intensity.
07/11/2018
E.coli strain recovery by activation culture
Three LB solid plates were spread on by the bacterial fluids containing the following plasmids separately: PRSF①, PRSF②, PRSF③. Then the three plates were incubated at 37°C for 16 hours.
Results: The colonies on three LB solid plates were all extremely dense and small.
07/12/2018
The bacterial strains PRSF②, PRSF③ were reserved, while the PRSF① was discarded. The PRSF②, PRSF③ was relabelled as PRSF-1. PRSF-2.
The monoclonal colonies were picked on the LB solid plate on Jul 11th, and was separately inoculated in the liquid LB medium with antibiotics. Then these obtained bacterial liquid in different tubes was shaken for 13 hours, and were spread 20μl on the LB solid plate with antibiotics.
Results: The colonies on three LB solid plates were all extremely dense and small.
07/16/2018
For E.coli strain PRSF:
Monoclonal colonies were picked and innoculated to liquid LB medium with antibiotics, and these obtained bacterial fluids were shaken for 12 hours. Then plasmid DNA extraction was performed on each tube of bacterial fluids. The results of plasmid DNA extraction are as follows:
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
PRSF-1
176.1
1.96
2.11
PRSF-2
225.8
1.95
2.23
The plasmid (used in the restriction enzyme digestion) was sent for sequencing.
07/17/2018
PCR amplification was performed by each extracted plasmids PRSF-1
Note: the annealing temperature was set at 67°C, and the extending time was set at 105 seconds.
The 1% agarose gel electrophoresis was performed at 120V for 32 min.
Result: There are two clear and bright bands at the position of 2,000 bp and 3,500 bp.
07/18/2018
PCR amplification of vector and RFP:
Number
1
2
3
Template
Vector
RFP
RFP
Enzyme Manufacturer
NEB
NEB
TIANGEN Pfw
Annealing   Temperature
67
71
58
Extension time
1 min 45 seconds
30 seconds
1 min 30 senconds
1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.Then the No.3 DNA on the gel was discarded.
Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA.
Name of Plasmid
Concentration/(ng/**μl)**
A260/A280
A260/A230
pSEVA GFP (vector)
17.5
1.82
0.99
pSEVA RFP (insert)
161.1
1.83
2.08
Gibson Assembly of vector and insert
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Vector
5
Insert
1.5
ddH2O
3.5
Total Volume
20
Transformation of Gibson assembly products (Using E.coli DH5α strain)
Result: No colonies on the solid LB medium (while the positive control performed well).
07/19/2018
The plate was spreaded using the Gibson assembly product again.
Result: No colonies on the solid LB medium (while the positive control performed well).
07/20/2018
NEB HiFi assembly of vector and insert:
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Vector
5
Insert
1.5
ddH2O
3.5
Total Volume
20
Transformation of Gibson assembly products
Result: No colonies on the solid LB medium (while the positive control performed well).
07/21/2018~07/23/2018
The experiment of Gibson Assembly on Jul 18th  and Jul 21st was performed again on this day. Then the transformation of Gibson assembly products to DH5α was performed. After viewing monoclonal colonies 15 hours later, three colonies were picked to innoculate in the LB liquid with antibiotics, and the bacterial fluids were shaked in the 37°C shaker overnight. Then the plasmid DNA extraction was performed, and 20μl of the extracted plasmid were sent for sequencing.
08/06/2018~08/08/2018
PCR amplification of vector and RFP:
Number
1
2
Template
Vector
RFP
Enzyme Manufacturer
NEB
Pfu
Annealing   Temperature
67
58
Extension time
1 min 45 seconds
1 min 30 seconds
1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.
Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA.
Name of Plasmid
Concentration/(ng/μl)
A260/A280
A260/A230
pSEVA GFP (vector)
14.3
1.89
1.58
pSEVA RFP (insert)
23.0
1.97
1.67
Gibson Assembly of vector and insert
Reagent
Volume/μl
Gibson Assembly Master Mix
15
Vector
3.8
Insert
1.2
Total Volume
20
Transformation of Gibson assembly products (Using E.coli DH5α strain) were performed, and then   two of the colonies were picked to the liquid LB medium.
Result: No turbidity in the bacterial liquid.
08/12/2018~08/13/2018
PCR amplification of vector and RFP:
Number
1
2
Template
Vector
RFP
Enzyme Manufacturer
NEB
Pfu
Annealing   Temperature
67
58
Extension time
1 min 45 seconds
1 min 30 seconds
1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.
Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA.
Name of Plasmid
Concentration/(ng/μl)
A260/A280
A260/A230
pSEVA GFP (vector)
14.3
1.89
1.58
pSEVA RFP (insert)
23.0
1.97
1.67
Gibson Assembly of vector and insert
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Vector
7.5
Insert
2.5
Total Volume
20
Two parallel Gibson Assembly in the above form were performed.
Transformation of Gibson assembly products (Using E.coli DH5α strain) were performed, and then   two of the colonies were picked to the liquid LB medium.
Result: No turbidity in the bacterial liquid.
08/15/2018~08/17/2018
PCR amplification of vector and RFP:
Number
1
2
Template
Vector
RFP
Enzyme Manufacturer
NEB
NEB
Annealing   Temperature
66
71
Extension time
1 min 45 seconds
30 seconds
1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.
Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA and Gibson Assembly of vector and insert
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Vector
3.5
Insert
1.5
ddH2O
5
Total Volume
20
Two parallel Gibson Assembly in the above form were performed.
Transformation of Gibson assembly products (Using E.coli DH5α strain) were performed, and then two of the colonies were picked to the liquid LB medium.
Result: No turbidity in the bacterial liquid.
08/17/2018~08/20/2018
PCR amplification of vector and RFP:
Number
1
2
Template
Vector
RFP
Enzyme Manufacturer
NEB
NEB
Annealing   Temperature
66
71
Extension time
1 min 45 seconds
30 seconds
Then purification of PCR products was directly performed (No gel electrophoresis).
Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA, following by DpnI digestion was performed to eliminate the template DNA:
Reagent
Volume/μl
PCR Products of Vector (or Insert)
8
10×CutSmart
1
DpnI
1
Gibson Assembly of vector and insert
Reagent
Volume/μl
Gibson Assembly Master Mix (2×)
10
Vector
0.6
Insert
1.8
ddH2O
7.6
Total Volume
20
Two parallel Gibson Assembly in the above form were performed.
Transformation of Gibson assembly products (DpnI digested) were performed:
Group 1: 1μl Gibson products were added to 25μl DH5α competent cells.
Group 2: The following volume of Gibson poducts (DpnI digestion first, then Gibson assembly) were separately added to 25μl DH5α competent cells: 0.5μl, 1μl, 2.5μl, 5μl.
Group 3: The following volume of Gibson poducts (Gibson assembly first, then DpnI digestion) were separately added to 25μl DH5α competent cells: 0.5μl, 1μl, 2.5μl, 5μl.
After transformation, two of the colonies on each of the nine LB medium were picked to the liquid LB medium. Then the plasmid DNA extraction was performed, and 20μl of the extracted plasmid were sent for sequencing.
Result: The target fragment was successfully inserted to the vector.
VI: Experiment of site-directed mutation of hRBS to oRBS
 
In this part of our experiments, we performed site-directed mutation of host ribosome binding site (hRBS) to orthogonal ribosome binding site (oRBS).
 
09/03/2018
Experimental goal: T4 vector ligation with PCR products
Experimental steps:
Adding A tail to PCR products:
Reaction system:
Reagents
Volume/μl
Template (100ng/μl)
20
Datp (10ng/μl)
1
Taq
1
Buffer
3
ddH2O
5
Total volume
30
The reaction system was incubated at 72°C for 1 hour.
T4 vector ligation:
One fast reaction and one slow reaction.
Transformation, blue and white spot experiment
Reaction system:
Reagents
Volume/μl
IPTG
40
Xgal
13
09/04/2018
Experimental goal: h-ribo SD sequence mutation [mutA2 & mutB8]
Experimental steps:
PCR mutagenesis:
Reaction system:
Reagents
Volume/μl
Template (100ng/μl)
1
Primer F (10ng/μl)
1
Primer R (10ng/μl)
1
Q5_mix(2x)
20
ddH2O
17
Total volume
40
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial   Denaturation
95°C
5   minutes
1
Denaturation
95°C
30   seconds
30
Annealing
56°C
30   seconds
Extension
72°C
3   minutes
Final   Extension
72°C
5   minutes
1
Hold
4°C
∞
1
Dpn I digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours.
PCR product purification
Transformation
09/05/2018
Experimental goal: plasmid DNA amplification
Experimental steps:
a. Pick a single colony [over-density]
b. LB amplification (Amp+)
Experimental goal: h-ribo SD sequence mutation [9-4 repeat]
Experimental steps:
a. PCR mutagenesis
Reaction system:
Reagents
Volume/μl
Template (100ng/μl)
1
Primer F (10ng/μl)
1
Primer R (10ng/μl)
1
Q5_mix(2x)
20
ddH2O
17
Total volume
40
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
95°C
5 minutes
1
Denaturation
95°C
30 seconds
30
Annealing
56°C
30 seconds
Extension
72°C
3 minutes
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
b.DpnI digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours.
c. PCR product purification
d.Tranformation
09/06/2018
Experimental goal: plasmid amplification
Experimental steps:
a. Pick [9-5 mutant board] single colonies [over-density]
b. Plasmid subtle [9-5] inoculated bacterial solution
Experimental goal: h-ribo mutation [a2 and b8]
Experimental steps:
a.  PCR mutagenesis
Reaction system:
Reagents
Volume/μl
Template (100ng/μl)
1
Primer F (10ng/μl)
1
Primer R (10ng/μl)
1
Q5_mix(2x)
20
ddH2O
17
Total volume
40
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
95°C
5 minutes
1
Denaturation
95°C
30 seconds
30
Annealing
56°C
30 seconds
Extension
72°C
3 minutes
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
b. DpnI digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours.
c. PCR product purification
d. Transformation
09/07/2018
Experimental goal: verification of plasmid
Experimental steps:
Pick [9-6] single colonies
Plasmid DNA extraction of inoculated bacterial solution [from 9 am-7 pm]
Send the extracted plasmids for sequencing
09/09/2018
Experimental goal: h-ribo SD sequence mutation [mutA2 & mutB8]
Experimental steps:
PCR mutagenesis
Reaction system:
Reagents
Volume/μl
Template (100ng/μl)
1
Primer F (10ng/μl)
1
Primer R (10ng/μl)
1
Q5_mix(2x)
20
ddH2O
17
Total volume
40
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
95°C
5 minutes
1
Denaturation
95°C
30 seconds
30
Annealing
56°C
30 seconds
Extension
72°C
3 minutes
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
DpnI digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours.
09/10/2018
Experimental goal: plasmid amplification
Experimental steps:
Pick single colonies from the LB medium (on Sep 9th) with mutant strain [No colonies observed]
Experimental goal: h-ribo SD sequence mutation [mutA2 & mutB8][9-9 repeat]
a. PCR mutagenesis (Protocol of Teacher Gaofeng Fan)
Reaction system:
Reagents
Volume/μl
Template (10ng/μl)
5
Primer F (10 ng/μl)
1.25
Primer R (10ng/μl)
1.25
Restriction Endonuclease
1
Buffer(10x)
5
ddH2O
36.5
Total volume
50
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
95°C
5 minutes
1
Denaturation
95°C
30 seconds
18
Annealing
60°C
30 seconds
Extension
72°C
5 minutes
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
b. Dpn I digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours.
c. PCR product purification
d. Transformation
09/11/2018
Experimental goal: plasmid amplification
Experimental steps:
a. Pick single colonies from the LB solid medium (on Sep 10th) with mutant strain
b. Plasmid DNA extraction
Experimental goal: h-ribo mutation [m2 & m8]
Experimental steps:
a. PCR mutagenesis (Protocol of Teacher Gaofeng Fan)
Reaction system:
Reagents
Volume/μl
Template (10ng/μl)
5
Primer F (10 ng/μl)
1.25
Primer R (10ng/μl)
1.25
Restriction   Endonuclease
1
Buffer(10x)
5
ddH2O
36.5
Total volume
50
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
95°C
5 minutes
1
Denaturation
95°C
30 seconds
18
Annealing
60°C
30 seconds
Extension
72°C
5 minutes
Final Extension
72°C
5 minutes
1
Hold
4°C
∞
1
b. Dpn I digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours.
c. PCR product purification
d.Transformation
09/12/2018
Experimental goal: verification of plasmid
Experimental steps:
Pick single colonies (on the LB solid medium on Sep 11th)
Plasmid DNA extraction (in the morning of Sep 12th) of inoculated bacterial fluid
09/15/2018
Experimental goal: h-ribo SD sequence mutation [mutA2 & mutB8]
Experimental steps:
PCR mutagenesis (PFU protocol)
Reaction system:
Reagents
Volume/μl
Template (10ng/μl)
4
Primer F (10ng/μl)
1
Primer R (10ng/μl)
1
Pfu_mix(2x)
20
ddH2O
14
Total volume
40
Program of PCR amplifier:
Step
Temperature
Time
Cycles
Initial Denaturation
95°C
5 minutes
1
Denaturation
95°C
30 seconds
30
Annealing
56°C
30 seconds
Extension
68°C
10 minutes
Final Extension
68°C
5 minutes
1
Hold
4°C
∞
1
Dpn I digestion
Reaction system:
Reagents
Volume/μl
PCR product
20
Buffer(10x)
2.5
Dpn I
0.5
ddH2O
2
Total volume
25
The reaction system was incubated at 37°C for 2 hours, then incubated at 4°C overnight.
PCR product purification
Transformation (Followed by picking single colonies and inoculate into LB medium with antibiotics, shaking bacteria fluids and plasmid DNA extraction, sending for sequencing for verification of plasmids over the next few days)
ShanghaiTech iGEM @ 2018
