<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ShanghaiTech_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ShanghaiTech/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>ShanghaiTech iGEM</TITLE></P><DIV class="collapse navbar-collapse justify-content-center" id="navbarSupportedContent"><UL class="navbar-nav"><LI class="nav-item dropdown" id="navbar-project"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Project_Overview">
        PROJECT
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Project_Overview">Overview</A><A class="dropdown-item" href="/Team:ShanghaiTech/Project_NFBL">Negetive Feedback Loop</A><A class="dropdown-item" href="/Team:ShanghaiTech/Project_Oribosome">Orthogonal Ribosome</A><A class="dropdown-item" href="/Team:ShanghaiTech/Project_Combination">Combination</A><A class="dropdown-item" href="/Team:ShanghaiTech/InterLab">InterLab</A><A class="dropdown-item" href="/Team:ShanghaiTech/Measurement">Measurement</A></DIV></LI><LI class="nav-item dropdown" id="navbar-parts"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="#">
        PARTS
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Basic_Part">Basic Parts</A><A class="dropdown-item" href="/Team:ShanghaiTech/Composite_Part">Composite Parts</A><A class="dropdown-item" href="/Team:ShanghaiTech/Part_Collection">Part Collection</A><A class="dropdown-item" href="/Team:ShanghaiTech/Improve">Improved Part</A></DIV></LI><LI class="nav-item dropdown" id="navbar-model"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Model">
        MODELING
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Model">Overview</A><A class="dropdown-item" href="/Team:ShanghaiTech/Model_NFBL">Negative Feedback Loop</A><A class="dropdown-item" href="/Team:ShanghaiTech/Model_Oribosome">Orthogonal Ribosome</A><A class="dropdown-item" href="/Team:ShanghaiTech/Computer_Experiments">Computer Experiments</A><A class="dropdown-item" href="/Team:ShanghaiTech/Model_Interaction">Interaction between Data and Model</A><A class="dropdown-item" href="/Team:ShanghaiTech/Software">Software</A></DIV></LI><LI class="nav-item dropdown" id="navbar-hp"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Human_Practices">
        HUMAN PRACTICE
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Human_Practices">Overview</A><A class="dropdown-item" href="/Team:ShanghaiTech/Public_Engagement">Education &amp; Public Engagement</A><A class="dropdown-item" href="/Team:ShanghaiTech/Biosafety">Biosafety</A><A class="dropdown-item" href="/Team:ShanghaiTech/Integrated_HP">Integrated Human Practice</A><A class="dropdown-item" href="/Team:ShanghaiTech/Collaborations">Collaborations</A></DIV></LI><LI class="nav-item dropdown" id="navbar-notebook"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="#">
        NOTEBOOK
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Protocol">Protocol</A><A class="dropdown-item" href="/Team:ShanghaiTech/Notebook">Calendar</A></DIV></LI><LI class="nav-item dropdown" id="navbar-team"><A class="nav-link dropdown-toggle" data-toggle="dropdown" href="/Team:ShanghaiTech/Team">
        TEAM
      </A><DIV class="dropdown-menu"><A class="dropdown-item" href="/Team:ShanghaiTech/Team">Meet Our Team</A><A class="dropdown-item" href="/Team:ShanghaiTech/Attributions">Attribution</A></DIV></LI></UL></DIV><DIV class="row"><DIV class="col-sm-9" style="padding-top: -30px"><DIV class="card"><DIV class="card-header"><A href="#construction1" data-toggle="collapse"><H3>I: Verification of the function of LuxR element</H3></A></DIV><DIV class="card-body collapse" id="construction1"><P>In this part of our experiments, we verified the function of LuxR in our project. The gene circuit on the part (BBa_K2315024 in 2017 Part Library, from 2017 ShanghaiTech iGEM team) is pCon-LuxR-Ter-pLux-GFP-Ter in pSB1C3 backbone. In the gene curcuit, AHL (exogenously added inducer) and LuxR co-activates the promoter pLux and co-induce the expression of the GFP gene. </P><P><STRONG>08/17/2018</STRONG></P><OL start=""><LI>PCR + gel extraction (Vector and Insert were separately extracted)</LI><LI>Double digestion</LI></OL><P>Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Vector (at a very low concentration)</TD><TD>20</TD></TR><TR><TD>Insert (at a very low concentration)</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>5</TD></TR><TR><TD>Endonuclease <EM>Bam</EM>H I</TD><TD>1</TD></TR><TR><TD>Endonuclease <EM>Pst</EM> I</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>3</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><OL start="3"><LI>Restriction Endonuclease</LI></OL><P>Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template mix (at a very low concentration)</TD><TD>40</TD></TR><TR><TD>T4 Buffer(10x)</TD><TD>5</TD></TR><TR><TD>T4 ligase</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 16°C overnight.</P><P><STRONG>08/18/2018</STRONG></P><P>Perform these steps:</P><OL start=""><LI>Purification</LI><LI>Transformation (early morning)</LI></OL><P>        Result: No colonies on the LB solid medium.</P><P><STRONG>[Repeat The Following Steps]</STRONG></P><OL start=""><LI>PCR products purification (Vector + Insert)</LI><LI>Restriction endonuclease digestion</LI></OL><P>        Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Vector (at a very low concentration)</TD><TD>20</TD></TR><TR><TD>Insert (at a very low concentration)</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>5</TD></TR><TR><TD>Endonuclease 1</TD><TD>1</TD></TR><TR><TD>Endonuclease 2</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>3</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><OL start="3"><LI>Restriction Endonuclease</LI></OL><P>        Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template mix (at a very low concentration)</TD><TD>40</TD></TR><TR><TD>T4 Buffer(10x)</TD><TD>5</TD></TR><TR><TD>T4 DNA ligase</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 16°C overnight.</P><P><STRONG>08/19/2018</STRONG></P><P>Perform these steps:</P><OL start=""><LI>Purification</LI><LI>Transformation (products on Aug 18th)</LI></OL><P>Result: No colonies on the LB solid medium.</P><P><STRONG>08/20/2018</STRONG></P><OL start=""><LI><P>Preparation of Lux-AHL Solution:</P><P>Preparation of stock solution: 25mg AHL + 1.17ml DMSO → Concentration of AHL is 100mM.</P><P>Preparation of test solution: about 3mg AHL + 200μl → Concentration of AHL is 70.3mM. </P></LI><LI><P>Dilution of test solution:</P><P>Dilute the test solution to the final concentration of 7×10-4 M, 7×10-6 M, 7×10-8 M, 7×10-10 M.</P></LI><LI><P>Inducer addition to the bacterial fluid of A-GFP and the bacterial fluid made by Wenhan Fu:</P><P>A-GFP①: Add 10μl 10-8M AHL to 1ml bacterial fluid (final concentration: 10-10M)</P><P>A-GFP① Blank: No AHL addition in the bacterial fluid.</P><P>A-GFP②: Add 10μl 10-6M AHL to 1ml bacterial fluid (final concentration: 10-8M)</P><P>A-GFP② Blank: No AHL addition in the bacterial fluid.</P><P>FWH: Add 10μl 10-8M AHL to 1ml bacterial fluid made by Wenhan Fu (final concentration: 10-10M).</P><P>FWH Blank: No AHL addition in the bacterial fluid.</P></LI><LI><P>Detection on an ELISA plate</P></LI></OL><P><STRONG>08/21/2018</STRONG></P><OL start=""><LI><P>Digestion of two types of restriction endonucleases:</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Plasmid (Template)</TD><TD>5</TD></TR><TR><TD>Buffer(10x)</TD><TD>3</TD></TR><TR><TD>Endonuclease <EM>Bam</EM>H I</TD><TD>1</TD></TR><TR><TD>Endonuclease <EM>Pst</EM> I</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>20</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>30</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><OL start="2"><LI>Agarose gel electrophoresis (1% agarose gel, 120V, 45min) was performed to verify the DNA bands.</LI></OL><P><STRONG>08/22/2018</STRONG></P><P>The plasmid (used in the restriction endonuclease digestion) was sent for sequencing.</P><P><STRONG>[Repeat the steps on Aug 17th]</STRONG></P><OL start=""><LI><P>PCR + gel extraction (Vector and Insert were separately extracted)</P></LI><LI><P>Double digestion</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Vector (at a very low concentration)</TD><TD>20</TD></TR><TR><TD>Insert (at a very low concentration)</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>5</TD></TR><TR><TD>Endonuclease <EM>Bam</EM>H I</TD><TD>1</TD></TR><TR><TD>Endonuclease <EM>Pst</EM> I</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>3</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><OL start="3"><LI><P>Restriction Endonuclease Digestion</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template mix (at a very low concentration)</TD><TD>40</TD></TR><TR><TD>T4 Buffer(10x)</TD><TD>5</TD></TR><TR><TD>T4 ligase</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 16°C overnight. </P><P><STRONG>08/24/2018</STRONG></P><P>Purification of PCR products on Aug 22nd and transformation of the purified product. </P><P><STRONG>08/25/2018</STRONG></P><OL start=""><LI><P>Monoclonal colonies were picked and inoculated in liquid LB medium with chloramphenol at 37°C for 15 hours.</P></LI><LI><P>Small molecule experiment</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Bacterial liquid</TD><TD>5000</TD></TR><TR><TD>AHL (10ng/uL)</TD><TD>200</TD></TR></TBODY></TABLE></FIGURE><P>Reaction conditions: The reaction system mixture was shaken overnight at 37°C.</P><P>Result: No GFP fluorescence was observed.</P><P><STRONG>08/27/2018</STRONG></P><P>The plasmid (used in the restriction endonuclease digestion) was sent for sequencing.</P><P><STRONG>08/29/2018</STRONG></P><OL start=""><LI><P>Purification and transformation of the plasmid constructed on Aug 22nd.</P></LI><LI><P>Small molecule experiment</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Bacterial liquid</TD><TD>5000</TD></TR><TR><TD>AHL (10ng/uL)</TD><TD>200</TD></TR></TBODY></TABLE></FIGURE><P>Reaction conditions: The reaction system mixture was shaken overnight at 37°C.</P><P>Result: No GFP fluorescence was observed.</P></DIV></DIV><DIV class="card"><DIV class="card-header"><A href="#construction2" data-toggle="collapse"><H3>II: Verification of the function of pT181 element</H3></A></DIV><DIV class="card-body collapse" id="construction2"><P>In this part of our experiments, we verified the function of pT181 element in our constructed plasmid (BBa_2787027 in 2018 Part Library). The gene circuit on the part is pCon-pT181 Antisense-Ter-pCon-pT181 Sense Target-GFP-Ter. The pT181 Antisense gene express pT181 Antisense, and pT181 Antisense binds the pT181 Sense Target, thus represses the expression of GFP gene.</P><P><STRONG>07/02/2018</STRONG></P><P>Transformation of synthesis pT181 plasmids (the vector is pUC57, and the plasmid was synthesized by GenScript, China) into <EM>E.coli</EM> MG1655 strains.</P><P>Result: No colonies on the LB solid medium.</P><P><STRONG>07/03/2018</STRONG></P><P>Transformation of synthesis pT181 plasmids (the vector is pUC57, the plasmid was synthesized by GenScript, China) into <EM>E.coli</EM> DH5α strains.</P><P>Result: There were a few colonies on all the four LB solid medium plates. Label the four plates as: Plasmid①, Plasmid②; TOP 10①, TOP 10②. The colonies were picked and inoculated into liquid LB mediums. Shake the bacterial fluids at 37°C overnight (12 hours).</P><P>(Note: “Plasmid” represents the control, and “TOP 10” represents the plasmid containing the target DNA sequence.)</P><P><STRONG>07/05/2018</STRONG></P><P>Plasmid DNA Extraction of Plasmid①, Plasmid②, TOP 10①, TOP 10②:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH></TR></THEAD><TBODY><TR><TD>Plasmid①</TD><TD>44.2</TD><TD>1.97</TD></TR><TR><TD>Plasmid②</TD><TD>38.3</TD><TD>1.92</TD></TR><TR><TD>TOP 10①</TD><TD>293.2</TD><TD>1.85</TD></TR><TR><TD>TOP 10②</TD><TD>158.4</TD><TD>1.77</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>07/07/2018</STRONG></P><P><STRONG>Plasmid DNA Extraction</STRONG></P><FIGURE><TABLE class="table"><THEAD><TR><TH> </TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>Plasmid①</TD><TD>20.7</TD><TD>1.89</TD><TD>1.92</TD></TR><TR><TD>Plasmid②</TD><TD>93.7</TD><TD>1.88</TD><TD>2.09</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>07/15/2018</STRONG></P><P>Vector linearization by circular PCR amplification:</P><P>The templates were pUC57, and was labelled as pUC57①, pUC57②, pUC57③, pUC57④.</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>TIANGEN 2×PCR Master   Mix</TD><TD>25</TD></TR><TR><TD>Primer-F</TD><TD>2</TD></TR><TR><TD>Primer-R</TD><TD>2</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>20</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>94°C</TD><TD>3 min</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>94°C</TD><TD>30 seconds</TD><TD rowspan="3">25-35</TD></TR><TR><TD>Annealing</TD><TD>55°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>1 min</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>07/16/2018</STRONG></P><OL start=""><LI><P>Gel extraction of linearized pUC57:</P><P>(Plasmid pUC57① and pUC57② were added into the same adsorption column to perform gel extraction, and the products of gel extraction was labelled as Blank①; plasmid pUC57① and pUC57② were added into the same adsorption column, and the products of gel extraction was labelled as Blank②.)</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmids</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH></TR></THEAD><TBODY><TR><TD>Blank①</TD><TD>67.7</TD><TD>1.88</TD></TR><TR><TD>Blank②</TD><TD>54.7</TD><TD>1.87</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI>Transformation of synthesis pT181 plasmids (the vector is pUC57, and the plasmid was synthesized by GenScript, China) into <EM>E.coli</EM> DH5α strains. </LI></OL><P><STRONG>07/17/2018</STRONG></P><OL start=""><LI>Genomic DNA Extranction of <EM>E.coli</EM> MG1655 strains: <EM>E.coli</EM> DH5α strains was inoculated into LB solid medium, and the bacterial fliud was shaken at 37°C overnight (12 hours). The bacterial fluids were used to perform genomic DNA extraction.</LI><LI>Using Gibson Assembly Master Mix to perform pT181 Assembly (products labelled as PA):</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of PA/μl</STRONG></TH><TH><STRONG>Volume of Control/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Linearized Vector (Blank)</TD><TD>2.7</TD><TD>2.7</TD></TR><TR><TD>Insert (pT181)</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>6.8</TD><TD>6.8</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Then transformation of PA and control was followed by, and inoculation of colonies and shaking bacterial fluids were performed after transformation. </P><P><STRONG>07/18/2018</STRONG></P><P>Plasmid DNA extraction of PA (pT181-Blank Assembly):</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH></TR></THEAD><TBODY><TR><TD>PA①</TD><TD>538.6</TD><TD>1.84</TD></TR><TR><TD>PA②</TD><TD>372.9</TD><TD>1.85</TD></TR></TBODY></TABLE></FIGURE><P>Then high fidility PCR amplification was performed to amplify the target DNA sequence. Then the extracted plasmids were sent for sequencing.</P><P>Result: No target DNA (inserted pT181) in the pUC57 vector.</P><P><STRONG>07/20/2018</STRONG></P><P>pT181-Blank Assembly (Repeated the experimen again)</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of PA*</STRONG>*①, PA②/μl**</TH><TH><STRONG>Volume of Control/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Linearized Vector (Blank)</TD><TD>2.7</TD><TD>2.7</TD></TR><TR><TD>Insert (pT181)</TD><TD>0.9</TD><TD>0</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7.4</TD><TD>8.3</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>21</TD><TD>21</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>07/22/2018</STRONG></P><OL start=""><LI>pT181-Blank-Assembly:</LI></OL><P>The DNA fragments pT181 and linearized vector (Blank) was assembled by Gibson Assembly Master Mix (2×), which is labelled PA. Two systems were made to perform the assembly: PA①, PA②.</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of PA*</STRONG>*①, PA②/μl**</TH><TH><STRONG>Volume of Control/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD><TD>0</TD></TR><TR><TD>Linearized Vector (Blank)</TD><TD>2.7</TD><TD>2.7</TD></TR><TR><TD>Insert (pT181)</TD><TD>0.5</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>6.8</TD><TD>6.8</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD><TD>10</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI><P>PCR and Transformation:</P><P>25μl DH5α competent cells + 0.2/10/9.8μl pT181-Blank</P><P>Circular PCR amplification of PA → 500μl DH5α competent cells + 1μl PA → Pipette 50μl to spread the cells on a solid LB medium</P></LI></OL><P><STRONG>07/23/2018</STRONG></P><P>Bacterial fluid (Blank/PA): shake for 12 hours</P><P><STRONG>07/25/2018</STRONG></P><P>Bacterial fluid (Blank/PA): plate streak</P><P>Bacterial fluid (Blank/PA): shake for 12 hours</P><P><STRONG>07/25/2018</STRONG></P><P>PA/Blank: Agarose gel electrophoresis (120 V, 30 min) + Gel extraction</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of DNA</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>PA</TD><TD>36.3</TD><TD>1.82</TD><TD>0.89</TD></TR><TR><TD>Blank</TD><TD>30.0</TD><TD>1.83</TD><TD>0.16</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/06/2018</STRONG></P><P>Blank/pT181-Assembly transformation into E.<EM>coli</EM> DH5α </P><P><STRONG>08/08/2018</STRONG></P><P>Plasmid DNA extranction of pT181-Assembly(PA)/Blank again:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>pT181 Assembly (PA)</TD><TD>505.9</TD><TD>1.89</TD><TD>2.36</TD></TR><TR><TD>pT181 Blank (Blank)</TD><TD>265.1</TD><TD>1.91</TD><TD>2.09</TD></TR></TBODY></TABLE></FIGURE><P>Performing 1% agarose gel electrophoresis for the two extracted plasmids above, and performing gel extraction. The results of gel extraction are as follows:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH></TR></THEAD><TBODY><TR><TD>PA</TD><TD>15.3</TD><TD>1.79</TD></TR><TR><TD>Blank</TD><TD>6.5</TD><TD>1.65</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/10/2018</STRONG></P><OL start=""><LI><P>Colony PCR and Gel Electrophoresis of PCR Products</P><P>Templates: Blank 1, Blank 2, Blank 3, PA 1</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>TIANGEN 2×PCR Master   Mix</TD><TD>10</TD></TR><TR><TD>Primer-F</TD><TD>0.8</TD></TR><TR><TD>Primer-R</TD><TD>0.8</TD></TR><TR><TD>Template</TD><TD>0.8</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7.6</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Results of electrophoresis: No expected electrophoresis band.</P><P><STRONG>08/11/2018</STRONG></P><P>Plasmid PCR verification and Gel Electrophoresis of PCR Products</P><P>Templates: Blank pT181 top②, pT181 Assembly②; pT181 Blank (in plasmid pUC57), pT181 Assembly (in plasmid pUC57)</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>TIANGEN 2×PCR Master Mix</TD><TD>10</TD></TR><TR><TD>Primer-F</TD><TD>0.8</TD></TR><TR><TD>Primer-R</TD><TD>0.8</TD></TR><TR><TD>Template</TD><TD>0.8</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7.6</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/12/2018</STRONG></P><OL start=""><LI><P>Dilution, Spreading Plates, followed by Picking Monoclonal Colonies and Shaking bacterial fluids.</P><P>Plates of the following bacterial fluids were diluted and spread: Blank (7.23)①; Blank (7.23)③; Blank (7.26)③; PA (7.26)②; PA (7.26)③. </P><P>Monoclonal colonies were picked and bacterial fluids were shaken after 17 hours.</P></LI><LI><P>Picking a Monoclonal Colony and Shaking bacterial fluids</P><P>Pick a Monoclonal Colony and Shake bacteria of the following bacterial fluid: Blank/PA (7.22), Blank/PA (8.8)</P></LI></OL><P><STRONG>08/17/2018</STRONG></P><OL start=""><LI><P>Plasmid DNA Extraction</P><P>Plasmid DNA extraction of the bacterial fluids on August 12th : PA (7.22)①, PA (8.8)②, Blank (7.22)②, Blank (8.8)②</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>PA (7.22)①</TD><TD>426.5</TD><TD>1.89</TD><TD>2.28</TD></TR><TR><TD>PA (8.8)②</TD><TD>370.7</TD><TD>1.88</TD><TD>2.28</TD></TR><TR><TD>Blank (7.22)②</TD><TD>586.5</TD><TD>1.88</TD><TD>2.36</TD></TR><TR><TD>Blank (8.8)②</TD><TD>497.5</TD><TD>1.89</TD><TD>2.34</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI>Gibson Assembly</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Insert</TD><TD>1.8</TD></TR><TR><TD>Vector</TD><TD>0.6</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7.6</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/18/2018</STRONG></P><P>Plasmid PCR verification and Gel Electrophoresis of PCR Products</P><P>Templates: PA (7.22)①, PA (8.8)②, Blank (7.22)②, Blank (8.8)② (extracted in August 17th )</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>TIANGEN 2×PCR Master   Mix</TD><TD>10</TD></TR><TR><TD>Primer-F</TD><TD>0.8</TD></TR><TR><TD>Primer-R</TD><TD>0.8</TD></TR><TR><TD>Template</TD><TD>0.8</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7.6</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Agarose gel electrophoresis: 120 V, 30 min.</P></DIV></DIV><DIV class="card"><DIV class="card-header"><A href="#construction3" data-toggle="collapse"><H3>III: Verification of the function of STAR element</H3></A></DIV><DIV class="card-body collapse" id="construction3"><P>In this part of our experiments, we verified the function of STAR element in the plasmid courtsey of 2018 SJTU-BioX-Shanghai (The sequence of Antisense and STAR Target was from 2016 Imperial College London). The gene circuit on the part is pCons-STAR Antisense-t500 (between the two multi-cleavage sites on both sides of the gene circuit) and pCons-STAR Target-GFP or pCons-STAR Target-RFP. The STAR Antisense gene express STAR Antisense, and STAR Antisense binds the STAR target, thus activates the expression of GFP gene or RFP gene.</P><P><STRONG>07/05/2018</STRONG></P><P>Plate spread of three kinds of competent cells containing different plasmids courtesy of SJTU.</P><P>Result: No colony on each of the three plates after 15 hours.</P><P><STRONG>07/06/2018</STRONG></P><P>Transformation of the three plasmids courtesy of SJTU.</P><P>Result: A few colonies grew on the LB plates.</P><P><STRONG>07/07/2018</STRONG></P><OL start=""><LI><P>Picking a Monoclonal Colony and Shaking bacterial fluids</P><P>Monoclonal Colonies were picked and inoculated liquid LB mediums with streptomycin antibiotics,  and bacterial fluids were shaken at 37°C for 12 hours (three tubes were shaken for each plasmid PCDF1, PCDF3, T1RA1): PCDF1-1, PCDF1-2, PCDF1-3; PCDF3-1, PCDF3-2, PCDF3-3; T1RA1-1, T1RA1-2, T1RA1-3. </P></LI><LI><P>Plasmid DNA Extraction</P><P>Plasmid DNA extraction of the bacterial fluids: PCDF1-1, PCDF1-2, PCDF1-3; PCDF3-1, PCDF3-2, PCDF3-3; T1RA1-1, T1RA1-2, T1RA1-3.</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/</STRONG>μl)</TH><TH><STRONG>A260/A280</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCDF1-1</TD><TD>51.5</TD><TD>1.92</TD></TR><TR><TD>PCDF1-2</TD><TD>128.4</TD><TD>1.90</TD></TR><TR><TD>PCDF1-3</TD><TD>351.9</TD><TD>1.86</TD></TR><TR><TD>PCDF3-1</TD><TD>161.3</TD><TD>1.88</TD></TR><TR><TD>PCDF3-2</TD><TD>230.7</TD><TD>1.89</TD></TR><TR><TD>PCDF3-3</TD><TD>265.4</TD><TD>1.84</TD></TR><TR><TD>T1RA1-1</TD><TD>84.4</TD><TD>1.97</TD></TR><TR><TD>T1RA1-2</TD><TD>72.5</TD><TD>1.98</TD></TR><TR><TD>T1RA1-3</TD><TD>93.3</TD><TD>1.98</TD></TR></TBODY></TABLE></FIGURE><P>These plasmids were then stored at -20°C, and they were sequenced right by sequencing company.</P><P><STRONG>07/09/2018</STRONG></P><OL start=""><LI><P>Fluorescence Intensity Detection By a Microplate Reader</P><P>The bacterial liquids were added to 1.5ml EP tubes, and were centrifuged at 12,000 rpm for 5 min; the supernatant was discarded, and 200μl 1×PBS solution was added to the bacterial precipitate; resuspend the precipitate and add the resuspension solution to the wells on a 96-well ELISA plate; fluorescence intensity was detected by a microplate reader.</P></LI><LI><P>Shaking bacterial fluids</P></LI><LI><P>The detected bacterial fluids above were inoculated in liquid LB medium with streptomycin.</P></LI></OL><P><STRONG>07/15/2018</STRONG></P><P>Strain Recovery and Fluorescence Intensity Detection By a Microplate Reader</P><P>The DH5α E.coli strains containg following plasmids were recovered at 37°C for 20 min: PCDF1-1, PCDF1-2, PCDF1-3; PCDF3-1, PCDF3-2, PCDF3-3. Then the DH5α E.coli strain solutions were centrifuged at 12,000 rpm for 2 min, discard the supernatant, and resuspend the precipitate by addition of 200μl 1×PBS solution. Finally 200μl resuspended solution were added in the wells of a 96-well ELISA plate to detect fluorescence intensity of each solution.</P><P><STRONG>07/26/2018</STRONG></P><P>Shake Bacterial Fluids To Amplify Plasmid DNA</P><P>The following bacterial fluids were shaked overnight (12 hours): sfPCDF1-1, sfPCDF1-2, sfPCDF1-3; sfPCDF3-1, sfPCDF3-2, sfPCDF3-3.</P><P>Mix the bacterial fluids evenly, pick a monoclonal colony and inoculate it into 5ml LB medium with streptomycin antibiotics. Shake the bacterial fluid at 37°C overnight (12 hours).</P><P><STRONG>07/27/2018</STRONG></P><P>Microplate reader detection of GFP fluorescent expression results</P><P>The following bacterial fluids were performed to extract DNA plasmids: sfPCDF1-1, sfPCDF1-2, sfPCDF1-3; sfPCDF3-1, sfPCDF3-2, sfPCDF3-3; Blank 2-1, Blank 2-2; PA 2-1; PA 2-2.</P><P>For the bacterial fliuds above, 1.4 ml of each were centrifuged at 5,000 rpm, followed by discard of supernatants. Then 1.4 ml 1×PBS solution were added to resuspend the bacteria precipitate, and the resuspended solutions were added to the wells on a 96-well ELISA plate to detect the GFP fluorescence intensity.</P><P>The resuspended solutions of the following bacterial fluids were also aliquoted 5 μl to make a microscope mounts for viewing GFP fluorescence image: sfPCDF1-2, sfPCDF3-1, Blank 1, PA 2.</P></DIV></DIV><DIV class="card"><DIV class="card-header"><A href="#assembly" data-toggle="collapse"><H3>IV: Assembly of module A, B, C and orthogonal ribosome 16S DNA</H3></A></DIV><DIV class="card-body collapse" id="assembly"><P>In this part of our experiments, we assembled three gene circuit modules (module A, module B, module C) and orthogonal ribosome 16S DNA. The following plasmids were synthesized by GenScript, China:</P><P>Plasmid A: Gene circuit system containing module A, module B, module C and homologous sequence of the start and end of GFP gene (in pUC57 backbone).</P><P>Plasmid B: Gene circuit system containing module A, module B, module C and homologous sequence of the start and end of orthogonal ribosome 16S gene (in pUC57 backbone).</P><P>We assembled GFP gene sequence on Plasmid A, and assembled orthogonal ribosome 16S gene sequence on Plasmid B. 
        </P><P><STRONG>08/06/2018</STRONG></P><P>PCR amplifications were performed to amplify four target fragments: h16S DNA sequence (labelled as “h16S”), GFP gene sequence (labelled as “GFP”), </P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Labels</STRONG></TH><TH><STRONG>Corresponding Amplified Fragments</STRONG></TH><TH><STRONG>Templates</STRONG></TH></TR></THEAD><TBODY><TR><TD>h16S</TD><TD>h16S gene sequence</TD><TD>MG1655 genomic DNA</TD></TR><TR><TD>GFP</TD><TD>GFP gene sequence</TD><TD>Plasmid BBa_K2315022    (pCon-LuxR-pRpa-GFP)</TD></TR><TR><TD>R-G</TD><TD>Linearized vector of pUC57 containing GFP gene homologous sequence (start   and terminate position)</TD><TD>pUC57 containing GFP gene homologous sequence (start and terminate   position) (This vector is synthesized by GenScript, China)</TD></TR><TR><TD>O-ribo</TD><TD>Linearized vector of pUC57 containing h16S gene homologous sequence   (start and terminate position)</TD><TD>pUC57 containing h16S gene homologous sequence (start and terminate   position) (This vector is synthesized by GenScript, China)</TD></TR></TBODY></TABLE></FIGURE><P>PCR amplification of four fragments above was performed. The reaction systems were as follows:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>NEB HiFi 2×PCR Master   Mix</TD><TD>25</TD></TR><TR><TD>Primer-F</TD><TD>2.5</TD></TR><TR><TD>Primer-R</TD><TD>2.5</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>19</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P> Programs of PCR amplifier:</P><P>(1) h16S:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">35</TD></TR><TR><TD>Annealing</TD><TD>67°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>90 seconds</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>(2) GFP:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">35</TD></TR><TR><TD>Annealing</TD><TD>55°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>30 seconds</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>(3) R-G:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">35</TD></TR><TR><TD>Annealing</TD><TD>65°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P> (4) O-ribo:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">35</TD></TR><TR><TD>Annealing</TD><TD>65°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/07/2018</STRONG></P><OL start=""><LI>1% Agarose Gel Electrophoresis of PCR products (h16S, GFP, R-G, O-ribo) was performed at 120V for 45 min, then performed gel extraction. As there was no target band on the agarose gel, the results of gel extraction of the other tthree fragments was:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>h16S</TD><TD>11.5</TD><TD>2.79</TD><TD>0.01</TD></TR><TR><TD>GFP</TD><TD>8.0</TD><TD>2.08</TD><TD>1.81</TD></TR><TR><TD>O-ribo</TD><TD>4.1</TD><TD>2.11</TD><TD>0.03</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI>Gibson Assembly of two fragments h16S and O-ribo (labelled as “h16S-O-ribo”)</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly   Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>O-ribo</TD><TD>8.5</TD></TR><TR><TD>h16S</TD><TD>2.5</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>21</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/08/2018</STRONG></P><OL start=""><LI><P>Transformation of assembled products h16S-O-ribo into <EM>E.coli</EM> DH5α competent cells on the LB solid medium (Three with ampicillin, the other three with no antibiotics)</P><P>Results: High density colonies on the three LB solid medium plates with no antibiotics, but no colony on the three LB solid medium plates.</P></LI><LI><P>PCR amplification of pUC57 containing GFP gene homologous sequence</P><P>This time the forward primer (primer-F) and the reverse primer (primer-R) were changed to repeat PCR amplification.</P></LI><LI><P>Gel electrophoresis and gel extraction</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>R-G①</TD><TD>16.2</TD><TD>1.76</TD><TD>0.14</TD></TR><TR><TD>R-N-G①</TD><TD>12.3</TD><TD>1.82</TD><TD>0.21</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>08/09/2018</STRONG></P><OL start=""><LI>PCR products purification</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>h16S</TD><TD>220.0</TD><TD>1.84</TD><TD>1.67</TD></TR><TR><TD>GFP</TD><TD>132.6</TD><TD>1.68</TD><TD>0.88</TD></TR><TR><TD>R-G②</TD><TD>32.8</TD><TD>1.74</TD><TD>0.81</TD></TR><TR><TD>R-N-G②</TD><TD>36.6</TD><TD>1.67</TD><TD>1.00</TD></TR><TR><TD>O-ribo</TD><TD>25.3</TD><TD>1.46</TD><TD>1.02</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI><P>Gibson Assembly of each products</P><P>Two gel extraction products were Gibson Assembled as the following reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Mix Courtesy of Haopeng Wang’s Lab</TD><TD>15</TD></TR><TR><TD>Vector (R-G① or R-N-G①)</TD><TD>4</TD></TR><TR><TD>Insert (GFP)</TD><TD>1</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Five purified PCR products were Gibson Assembled as the following reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Vector (R-G② or R-N-G② or O-ribo)</TD><TD>5</TD></TR><TR><TD>Insert (GFP)</TD><TD>5</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Mix Courtesy of Haopeng Wang’s Lab</TD><TD>15</TD></TR><TR><TD>Vector (R-G② or R-N-G② or O-ribo)</TD><TD>4</TD></TR><TR><TD>Insert (GFP)</TD><TD>1</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Then all reaction systems were incubated at 50°C for 4 hours.</P><P><STRONG>08/10/2018</STRONG></P><P>Transformation of the Gibson products of Aug 9th into <EM>E.coli</EM> DH5α Competent cells.</P><P>Result: No colonies on each LB solid medium plate. </P><P><STRONG>08/11/2018~08/15/2018</STRONG></P><P>Repeat PCR amplification, 1% agarose gel electrophoresis, gel extraction and Gibson assembly protocols from Aug 6th to Aug 9th .</P><P>The gel extraction results:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>h16S</TD><TD>75.5</TD><TD>1.79</TD><TD>1.23</TD></TR><TR><TD>GFP</TD><TD>36.7</TD><TD>1.79</TD><TD>1.92</TD></TR><TR><TD>R-G</TD><TD>10.5</TD><TD>1.71</TD><TD>0.05</TD></TR><TR><TD>O-ribo</TD><TD>15.5</TD><TD>1.73</TD><TD>0.48</TD></TR></TBODY></TABLE></FIGURE><P>The Gibson Assembly reaction systems:</P><P>(Reaction systems: G1: GFP + R-G; G2: GFP + R-G; G3: h16S + O-ribo; G4: h16S + O-ribo)</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of G1/</STRONG>μl</TH><TH><STRONG>Volume of G2/</STRONG>μl</TH><TH><STRONG>Volume of G3/μl</STRONG></TH><TH><STRONG>Volume of G4/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD><TD>10</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Vector (R-G)</TD><TD>6</TD><TD>8</TD><TD>6</TD><TD>8</TD></TR><TR><TD>Insert (GFP)</TD><TD>4</TD><TD>2</TD><TD>4</TD><TD>2</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Then performed transformation of the Gibson products into <EM>E.coli</EM> DH5α Competent cells.</P><P>Result: No colonies on each LB solid medium plate.</P><P><STRONG>08/16/2018~08/22/2018</STRONG></P><P>Repeat PCR amplification and 1% agarose gel electrophoresis of R-G and O-ribo, and gel extraction of the R-G. and Gibson assembly protocols from Aug 6th to Aug 9th .</P><OL start=""><LI>The reaction systems of PCR amplification:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>NEB HiFi 2×PCR Master   Mix</TD><TD>25</TD></TR><TR><TD>Primer-F</TD><TD>2.5</TD></TR><TR><TD>Primer-R</TD><TD>2.5</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>19</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI>The programs of PCR amplifier:</LI></OL><P>(1) R-G:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>65°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P> (2) O-ribo:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>65°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>Then the 1% agarose gel electrophoresis was followed by.</P><P>Results: the expected R-G bands were observed on the agarose gel, but there were no expected O-ribo bands.</P><OL start="3"><LI>The gel extraction results:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>R-G</TD><TD>18.8</TD><TD>1.82</TD><TD>0.04</TD></TR></TBODY></TABLE></FIGURE><OL start="4"><LI>The Gibson Assembly reaction systems:</LI></OL><P>(Reaction systems: G1: GFP + R-G; G2: h16S + O-ribo)</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of G1/μl</STRONG></TH><TH><STRONG>Volume of G2/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>4.19</TD><TD>4.2</TD></TR><TR><TD>Insert</TD><TD>0.98</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4.83</TD><TD>4.8</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Then gradient transformation was transformed of the Gibson products (GFP + R-G, labelled as “GFP”) into <EM>E.coli</EM> DH5α Competent cells. </P><P>Transformation in gradient:</P><P>Group 1: 0.2μl plasmid + 25μl E.coli DH5α Competent cells (labelled as “0.2GFP-①”“0.2GFP-②”“0.2GFP-③”)</P><P>Group 2: 0.5μl plasmid + 25μl E.coli DH5α Competent cells (labelled as “0.5GFP-①”“0.5GFP-②”“0.5GFP-③”)</P><P>Group 3: 1μl plasmid + 25μl E.coli DH5α Competent cells (labelled as “1GFP-①”“1GFP-②”“1GFP-③”)</P><P>Result: High-density colonies were performed on each group of LB solid medium plates.</P><OL start="5"><LI>The colonies were picked and inoculated to liquid LB medium with chloramphenicol antibiotics, followed by plasmid DNA extraction.</LI></OL><P>The results of plasmid DNA extraction:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>0.2GFP-①</TD><TD>524.5</TD><TD>1.89</TD><TD>2.38</TD></TR><TR><TD>0.2GFP-②</TD><TD>600.0</TD><TD>1.89</TD><TD>2.38</TD></TR><TR><TD>0.5GFP-①</TD><TD>589.8</TD><TD>1.87</TD><TD>2.36</TD></TR><TR><TD>0.5GFP-②</TD><TD>475.0</TD><TD>1.91</TD><TD>2.35</TD></TR><TR><TD>0.5GFP-③</TD><TD>500.7</TD><TD>1.91</TD><TD>2.32</TD></TR><TR><TD>1GFP-①</TD><TD>388.0</TD><TD>1.86</TD><TD>2.35</TD></TR><TR><TD>1GFP-②</TD><TD>437.5</TD><TD>1.88</TD><TD>2.34</TD></TR><TR><TD>1GFP-③</TD><TD>491.5</TD><TD>1.87</TD><TD>2.36</TD></TR></TBODY></TABLE></FIGURE><P>Note: The bacterial fluid of 0.2GFP-③ wasn’t observed turbidity.</P><P><STRONG>08/21/2018~08/29/2018</STRONG></P><P>Repeat the following protocols (for many times) from Aug 21st to Aug 29th : PCR amplification and 1% agarose gel electrophoresis of R-G, and gel extraction of the R-G. and Gibson assembly protocols from Aug 6th to Aug 9th .</P><P>The labels of each assembly products were as follows:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Label</STRONG></TH><TH><STRONG>Protocol of Fragments Assembly</STRONG></TH></TR></THEAD><TBODY><TR><TD>GHD</TD><TD>R-G + GFP: HiFi Assembly → DpnI Digestion</TD></TR><TR><TD>GGD</TD><TD>R-G + GFP: Gibson Assembly → DpnI Digestion</TD></TR><TR><TD>GHN</TD><TD>R-G + GFP: HiFi Assembly → No Digestion</TD></TR></TBODY></TABLE></FIGURE><OL start=""><LI>PCR amplification of R-G:</LI></OL><P>The reaction systems of PCR amplification:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>NEB HiFi 2×PCR Master   Mix</TD><TD>25</TD></TR><TR><TD>Primer-F</TD><TD>2.5</TD></TR><TR><TD>Primer-R</TD><TD>2.5</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>19</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P> The programs of PCR amplification were the same as that in Aug 6th:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">35</TD></TR><TR><TD>Annealing</TD><TD>55°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>30 seconds</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P> Then the 1% agarose gel electrophoresis was followed by.</P><P>Results: The expected R-G bands were observed on the agarose gel.</P><OL start="2"><LI>Gel extraction was performed after 1% agarose gel electrophoresis. The gel extraction results:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/</STRONG>μl)</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>R-G</TD><TD>48.0</TD><TD>1.84</TD><TD>0.38</TD></TR></TBODY></TABLE></FIGURE><OL start="3"><LI>The HiFi/Gibson Assembly reaction systems:</LI></OL><P>(Reaction systems: G1: GFP + R-G; G2: h16S + O-ribo)</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH>Volume of GHN/μl</TH><TH><STRONG>Volume of GGN/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>3.2</TD><TD>3.2</TD></TR><TR><TD>Insert</TD><TD>1.2</TD><TD>1.2</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>5.6</TD><TD>5.2</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Then DpnI digestion was performed as the following reaction systems:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of GHD/</STRONG>μl</TH><TH><STRONG>Volume of GGD/</STRONG>μl</TH></TR></THEAD><TBODY><TR><TD>Hifi/Gibson Assembly Products</TD><TD>8</TD><TD>8</TD></TR><TR><TD>10×Cutsmart</TD><TD>1</TD><TD>1</TD></TR><TR><TD>DpnI</TD><TD>1</TD><TD>1</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></FIGURE><P>Then gradient transformation was transformed of the DpnI digestion products into <EM>E.coli</EM> DH5α Competent cells. </P><P>Transformation in gradient (or in different groups):</P><P>Group 1: GHD① 0.5μl + 25μl <EM>E.coli</EM> DH5α Competent cells</P><P>Group 2: GGD① 1.5μl + 25μl <EM>E.coli</EM> DH5α Competent cells</P><P>Group 3: GHD② 0.2μl/0.5μl/1μl + 25μl <EM>E.coli</EM> DH5α Competent cells</P><P>Group 4: GGD② 0.7μl/1.5μl/2.5μl + 25μl <EM>E.coli</EM> DH5α Competent cells</P><P>Result: High-density colonies were performed on each group of LB solid medium plates. </P><OL start="4"><LI><P>The colonies were picked and inoculated to liquid LB medium with ampicillin antibiotics, followed by plasmid DNA extraction.</P><P>The results of plasmid DNA extraction:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/</STRONG>μl)</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>0.2GFP-①</TD><TD>524.5</TD><TD>1.89</TD><TD>2.38</TD></TR><TR><TD>0.2GFP-②</TD><TD>600.0</TD><TD>1.89</TD><TD>2.38</TD></TR><TR><TD>0.5GFP-①</TD><TD>589.8</TD><TD>1.87</TD><TD>2.36</TD></TR><TR><TD>0.5GFP-②</TD><TD>475.0</TD><TD>1.91</TD><TD>2.35</TD></TR><TR><TD>0.5GFP-③</TD><TD>500.7</TD><TD>1.91</TD><TD>2.32</TD></TR><TR><TD>1GFP-①</TD><TD>388.0</TD><TD>1.86</TD><TD>2.35</TD></TR><TR><TD>1GFP-②</TD><TD>437.5</TD><TD>1.88</TD><TD>2.34</TD></TR><TR><TD>1GFP-③</TD><TD>491.5</TD><TD>1.87</TD><TD>2.36</TD></TR></TBODY></TABLE></FIGURE><P>Note: The bacterial fluid of 0.2GFP-③ wasn’t observed turbidity.  </P><P><STRONG>08/21/2018~08/29/2018</STRONG></P><P>Repeat the following protocols (for many times) from Aug 21st to Aug 29th : PCR amplification and 1% agarose gel electrophoresis of O-ribo and h16S, and gel extraction of the R-G. and Gibson assembly protocols from Aug 6th to Aug 9th .</P><P>The labels of each assembly products were as follows:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Label</STRONG></TH><TH><STRONG>Protocol of Fragments Assembly</STRONG></TH></TR></THEAD><TBODY><TR><TD>OHD</TD><TD>O-ribo + h16S: HiFi Assembly → DpnI Digestion</TD></TR><TR><TD>OGD</TD><TD>O-ribo + h16S: Gibson Assembly → DpnI Digestion</TD></TR><TR><TD>OHN</TD><TD>O-ribo + h16S: HiFi Assembly → No Digestion</TD></TR><TR><TD>o16S-Lin</TD><TD>Linearized vector of pUC57 containing h16S gene homologous sequence   (start and terminate position)</TD></TR></TBODY></TABLE></FIGURE><OL start=""><LI><P>PCR amplification of R-G:</P><P>The reaction systems of PCR amplification:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>NEB HiFi 2×PCR Master   Mix</TD><TD>25</TD></TR><TR><TD>Primer-F</TD><TD>2.5</TD></TR><TR><TD>Primer-R</TD><TD>2.5</TD></TR><TR><TD>Template</TD><TD>1</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>19</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>       The programs of PCR amplification were the same as that in Aug 6th:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>98°C</TD><TD>30 seconds</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>98°C</TD><TD>10 seconds</TD><TD rowspan="3">40</TD></TR><TR><TD>Annealing</TD><TD>65°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>2 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>2 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>       Then the 1% agarose gel electrophoresis was followed by.</P><P>        Results: The expected R-G bands were observed on the agarose gel.</P><OL start="2"><LI>Gel extraction was performed after 1% agarose gel electrophoresis. The gel extraction results:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>o16S-Lin</TD><TD>35.2</TD><TD>1.65</TD><TD>0.19</TD></TR></TBODY></TABLE></FIGURE><OL start="3"><LI>Overlap PCR was performed with h16S and o16S-Lin (The product was labelled as “o-ribo①”“o-ribo②”):</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagants</STRONG></TH><TH><EM>Volume in o-ribo</EM>①/μl</TH><TH><STRONG>Volume in o-ribo</STRONG>②/μl</TH></TR></THEAD><TBODY><TR><TD>NEB HiFi 2×PCR Master Mix</TD><TD>10</TD><TD>10</TD></TR><TR><TD>Insert</TD><TD>5</TD><TD>5</TD></TR><TR><TD>Vector</TD><TD>1</TD><TD>5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>4</TD><TD>0</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><OL start="4"><LI>Then DpnI digestion was performed as the following reaction systems (The product was labelled as “o-ribo-I”“o-ribo-II”):</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume of o-ribo-I/</STRONG>μl</TH><TH><STRONG>Volume of o-ribo-II/</STRONG>μl</TH></TR></THEAD><TBODY><TR><TD>Hifi/Gibson Assembly Products   (o-ribo①/o-ribo②)</TD><TD>8</TD><TD>8</TD></TR><TR><TD>10×Cutsmart</TD><TD>1</TD><TD>1</TD></TR><TR><TD>DpnI</TD><TD>1</TD><TD>1</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>10</TD><TD>10</TD></TR></TBODY></TABLE></FIGURE><P>Then transformation was transformed of the DpnI digestion products into <EM>E.coli</EM> DH5α Competent cells: </P><P>Group 1: 1.5μl o-ribo-I + 25μl <EM>E.coli</EM> DH5α Competent cells;</P><P>Group 2: 1.5μl o-ribo-II + 25μl <EM>E.coli</EM> DH5α Competent cells.</P><P>Result: A few colonies were performed on each group of LB solid medium plates.</P><P>The colonies were picked and inoculated to liquid LB medium with ampicillin antibiotics, followed by plasmid DNA extraction and sending extracted DNA for sequencing (many times).</P></DIV></DIV><DIV class="card"><DIV class="card-header"><A href="#construction5" data-toggle="collapse"><H3>V: Construction of a plasmid to validate the function of orthogonal ribosome</H3></A></DIV><DIV class="card-body collapse" id="construction5"><P> </P><P>In this part of our experiments, we constructed the plasmid (BBa_K2787021 in 2018 Part Library) to validate the function of orthogonal ribosome. The part we constructed is pCon-hRBS-RFP-Ter-pCon-oRBS-GFP-Ter in pSB1C3 backbone. The orthogonality was verified by GFP fluorescence intensity and RFP fluorescence intensity.</P><P><STRONG>07/11/2018</STRONG></P><P><EM>E.coli</EM> strain recovery by activation culture</P><P>Three LB solid plates were spread on by the bacterial fluids containing the following plasmids separately: PRSF①, PRSF②, PRSF③. Then the three plates were incubated at 37°C for 16 hours.</P><P>Results: The colonies on three LB solid plates were all extremely dense and small.</P><P><STRONG>07/12/2018</STRONG></P><OL start=""><LI><P>The bacterial strains PRSF②, PRSF③ were reserved, while the PRSF① was discarded. The PRSF②, PRSF③ was relabelled as PRSF-1. PRSF-2.</P></LI><LI><P>The monoclonal colonies were picked on the LB solid plate on Jul 11th, and was separately inoculated in the liquid LB medium with antibiotics. Then these obtained bacterial liquid in different tubes was shaken for 13 hours, and were spread 20μl on the LB solid plate with antibiotics.</P><P>Results: The colonies on three LB solid plates were all extremely dense and small.</P></LI></OL><P><STRONG>07/16/2018</STRONG></P><P>For <EM>E.coli</EM> strain PRSF:</P><P>Monoclonal colonies were picked and innoculated to liquid LB medium with antibiotics, and these obtained bacterial fluids were shaken for 12 hours. Then plasmid DNA extraction was performed on each tube of bacterial fluids. The results of plasmid DNA extraction are as follows:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>PRSF-1</TD><TD>176.1</TD><TD>1.96</TD><TD>2.11</TD></TR><TR><TD>PRSF-2</TD><TD>225.8</TD><TD>1.95</TD><TD>2.23</TD></TR></TBODY></TABLE></FIGURE><P>The plasmid (used in the restriction enzyme digestion) was sent for sequencing.</P><P><STRONG>07/17/2018</STRONG></P><OL start=""><LI><P>PCR amplification was performed by each extracted plasmids PRSF-1</P><P>Note: the annealing temperature was set at 67°C, and the extending time was set at 105 seconds.</P></LI><LI><P>The 1% agarose gel electrophoresis was performed at 120V for 32 min.</P><P>Result: There are two clear and bright bands at the position of 2,000 bp and 3,500 bp.</P></LI></OL><P><STRONG>07/18/2018</STRONG></P><OL start=""><LI>PCR amplification of vector and RFP:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Number</STRONG></TH><TH><STRONG>1</STRONG></TH><TH><STRONG>2</STRONG></TH><TH><STRONG>3</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>Vector</TD><TD>RFP</TD><TD>RFP</TD></TR><TR><TD>Enzyme Manufacturer</TD><TD>NEB</TD><TD>NEB</TD><TD>TIANGEN Pfw</TD></TR><TR><TD>Annealing   Temperature</TD><TD>67</TD><TD>71</TD><TD>58</TD></TR><TR><TD>Extension time</TD><TD>1 min 45 seconds</TD><TD>30 seconds</TD><TD>1 min 30 senconds</TD></TR></TBODY></TABLE></FIGURE><P> 1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.Then the No.3 DNA on the gel was discarded.</P><OL start="2"><LI>Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA.</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/*</STRONG>*μl)**</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>pSEVA GFP (vector)</TD><TD>17.5</TD><TD>1.82</TD><TD>0.99</TD></TR><TR><TD>pSEVA RFP (insert)</TD><TD>161.1</TD><TD>1.83</TD><TD>2.08</TD></TR></TBODY></TABLE></FIGURE><OL start="3"><LI>Gibson Assembly of vector and insert</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>5</TD></TR><TR><TD>Insert</TD><TD>1.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>3.5</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><OL start="4"><LI><P>Transformation of Gibson assembly products (Using <EM>E.coli</EM> DH5α strain)</P><P>Result: No colonies on the solid LB medium (while the positive control performed well).</P></LI></OL><P><STRONG>07/19/2018</STRONG></P><P>The plate was spreaded using the Gibson assembly product again.</P><P>Result: No colonies on the solid LB medium (while the positive control performed well).</P><P><STRONG>07/20/2018</STRONG></P><OL start=""><LI>NEB HiFi assembly of vector and insert:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>5</TD></TR><TR><TD>Insert</TD><TD>1.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>3.5</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI>Transformation of Gibson assembly products</LI></OL><P>Result: No colonies on the solid LB medium (while the positive control performed well).</P><P><STRONG>07/21/2018~07/23/2018</STRONG></P><P>The experiment of Gibson Assembly on Jul 18th  and Jul 21st was performed again on this day. Then the transformation of Gibson assembly products to DH5α was performed. After viewing monoclonal colonies 15 hours later, three colonies were picked to innoculate in the LB liquid with antibiotics, and the bacterial fluids were shaked in the 37°C shaker overnight. Then the plasmid DNA extraction was performed, and 20μl of the extracted plasmid were sent for sequencing.</P><P><STRONG>08/06/2018~08/08/2018</STRONG></P><OL start=""><LI>PCR amplification of vector and RFP:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Number</STRONG></TH><TH><STRONG>1</STRONG></TH><TH><STRONG>2</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>Vector</TD><TD>RFP</TD></TR><TR><TD>Enzyme Manufacturer</TD><TD>NEB</TD><TD>Pfu</TD></TR><TR><TD>Annealing   Temperature</TD><TD>67</TD><TD>58</TD></TR><TR><TD>Extension time</TD><TD>1 min 45 seconds</TD><TD>1 min 30 seconds</TD></TR></TBODY></TABLE></FIGURE><P> 1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.</P><OL start="2"><LI>Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA.</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/</STRONG>μl)</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>pSEVA GFP (vector)</TD><TD>14.3</TD><TD>1.89</TD><TD>1.58</TD></TR><TR><TD>pSEVA RFP (insert)</TD><TD>23.0</TD><TD>1.97</TD><TD>1.67</TD></TR></TBODY></TABLE></FIGURE><OL start="3"><LI>Gibson Assembly of vector and insert</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix</TD><TD>15</TD></TR><TR><TD>Vector</TD><TD>3.8</TD></TR><TR><TD>Insert</TD><TD>1.2</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><OL start="4"><LI>Transformation of Gibson assembly products (Using <EM>E.coli</EM> DH5α strain) were performed, and then   two of the colonies were picked to the liquid LB medium.</LI></OL><P>Result: No turbidity in the bacterial liquid.</P><P><STRONG>08/12/2018~08/13/2018</STRONG></P><OL start=""><LI>PCR amplification of vector and RFP:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Number</STRONG></TH><TH><STRONG>1</STRONG></TH><TH><STRONG>2</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>Vector</TD><TD>RFP</TD></TR><TR><TD>Enzyme Manufacturer</TD><TD>NEB</TD><TD>Pfu</TD></TR><TR><TD>Annealing   Temperature</TD><TD>67</TD><TD>58</TD></TR><TR><TD>Extension time</TD><TD>1 min 45 seconds</TD><TD>1 min 30 seconds</TD></TR></TBODY></TABLE></FIGURE><P>      1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.</P><OL start="2"><LI>Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA.</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Name of Plasmid</STRONG></TH><TH><STRONG>Concentration/(ng/</STRONG>μl)</TH><TH><STRONG>A260/A280</STRONG></TH><TH><STRONG>A260/A230</STRONG></TH></TR></THEAD><TBODY><TR><TD>pSEVA GFP (vector)</TD><TD>14.3</TD><TD>1.89</TD><TD>1.58</TD></TR><TR><TD>pSEVA RFP (insert)</TD><TD>23.0</TD><TD>1.97</TD><TD>1.67</TD></TR></TBODY></TABLE></FIGURE><OL start="3"><LI>Gibson Assembly of vector and insert</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>7.5</TD></TR><TR><TD>Insert</TD><TD>2.5</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>      Two parallel Gibson Assembly in the above form were performed.</P><OL start="4"><LI><P>Transformation of Gibson assembly products (Using <EM>E.coli</EM> DH5α strain) were performed, and then   two of the colonies were picked to the liquid LB medium.</P><P>Result: No turbidity in the bacterial liquid.</P></LI></OL><P><STRONG>08/15/2018~08/17/2018</STRONG></P><OL start=""><LI>PCR amplification of vector and RFP:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Number</STRONG></TH><TH><STRONG>1</STRONG></TH><TH><STRONG>2</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>Vector</TD><TD>RFP</TD></TR><TR><TD>Enzyme Manufacturer</TD><TD>NEB</TD><TD>NEB</TD></TR><TR><TD>Annealing   Temperature</TD><TD>66</TD><TD>71</TD></TR><TR><TD>Extension time</TD><TD>1 min 45 seconds</TD><TD>30 seconds</TD></TR></TBODY></TABLE></FIGURE><P>1% agarose gel electrophoresis (120V, 45 min); DL 5000 marker was used.</P><OL start="2"><LI>Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA and Gibson Assembly of vector and insert</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>3.5</TD></TR><TR><TD>Insert</TD><TD>1.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>5</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Two parallel Gibson Assembly in the above form were performed.</P><OL start="3"><LI><P>Transformation of Gibson assembly products (Using <EM>E.coli</EM> DH5α strain) were performed, and then two of the colonies were picked to the liquid LB medium.</P><P>Result: No turbidity in the bacterial liquid.</P></LI></OL><P><STRONG>08/17/2018~08/20/2018</STRONG></P><OL start=""><LI>PCR amplification of vector and RFP:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Number</STRONG></TH><TH><STRONG>1</STRONG></TH><TH><STRONG>2</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template</TD><TD>Vector</TD><TD>RFP</TD></TR><TR><TD>Enzyme Manufacturer</TD><TD>NEB</TD><TD>NEB</TD></TR><TR><TD>Annealing   Temperature</TD><TD>66</TD><TD>71</TD></TR><TR><TD>Extension time</TD><TD>1 min 45 seconds</TD><TD>30 seconds</TD></TR></TBODY></TABLE></FIGURE><P>Then purification of PCR products was directly performed (No gel electrophoresis).</P><OL start="2"><LI>Gel extraction was performed to collect vector and insert (RFP gene sequence) DNA, following by DpnI digestion was performed to eliminate the template DNA:</LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR Products of Vector (or Insert)</TD><TD>8</TD></TR><TR><TD>10×CutSmart</TD><TD>1</TD></TR><TR><TD>DpnI</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>Gibson Assembly of vector and insert</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagent</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Gibson Assembly Master Mix (2×)</TD><TD>10</TD></TR><TR><TD>Vector</TD><TD>0.6</TD></TR><TR><TD>Insert</TD><TD>1.8</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>7.6</TD></TR><TR><TD><STRONG>Total Volume</STRONG></TD><TD>20</TD></TR></TBODY></TABLE></FIGURE><P>Two parallel Gibson Assembly in the above form were performed.</P><OL start="3"><LI><P>Transformation of Gibson assembly products (DpnI digested) were performed:</P><P>Group 1: 1μl Gibson products were added to 25μl DH5α competent cells.</P><P>Group 2: The following volume of Gibson poducts (DpnI digestion first, then Gibson assembly) were separately added to 25μl DH5α competent cells: 0.5μl, 1μl, 2.5μl, 5μl.</P><P>Group 3: The following volume of Gibson poducts (Gibson assembly first, then DpnI digestion) were separately added to 25μl DH5α competent cells: 0.5μl, 1μl, 2.5μl, 5μl.</P><P>After transformation, two of the colonies on each of the nine LB medium were picked to the liquid LB medium. Then the plasmid DNA extraction was performed, and 20μl of the extracted plasmid were sent for sequencing.</P><P>Result: The target fragment was successfully inserted to the vector.</P></LI></OL></DIV></DIV><DIV class="card"><DIV class="card-header"><A href="#experiment6" data-toggle="collapse"><H3>VI: Experiment of site-directed mutation of hRBS to oRBS</H3></A></DIV><DIV class="card-body collapse" id="experiment6"><P> </P><P>In this part of our experiments, we performed site-directed mutation of host ribosome binding site (hRBS) to orthogonal ribosome binding site (oRBS).</P><P> </P><P><STRONG>09/03/2018</STRONG></P><P>Experimental goal: T4 vector ligation with PCR products</P><P>Experimental steps:</P><OL start=""><LI><P>Adding A tail to PCR products:</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (100ng/μl)</TD><TD>20</TD></TR><TR><TD>Datp (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Taq</TD><TD>1</TD></TR><TR><TD>Buffer</TD><TD>3</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>5</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>30</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 72°C for 1 hour.</P><OL start="2"><LI><P>T4 vector ligation:</P><P>One fast reaction and one slow reaction.</P></LI><LI><P>Transformation, blue and white spot experiment</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>IPTG</TD><TD>40</TD></TR><TR><TD>Xgal</TD><TD>13</TD></TR></TBODY></TABLE></FIGURE><P><STRONG>09/04/2018</STRONG></P><P>Experimental goal: h-ribo SD sequence mutation [mutA2 &amp; mutB8]</P><P>Experimental steps:</P><OL start=""><LI>PCR mutagenesis:</LI></OL><P>Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (100ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer F (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Q5_mix(2x)</TD><TD>20</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>17</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>40</TD></TR></TBODY></TABLE></FIGURE><P>Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial   Denaturation</TD><TD>95°C</TD><TD>5   minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30   seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>56°C</TD><TD>30   seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>3   minutes</TD></TR><TR><TD>Final   Extension</TD><TD>72°C</TD><TD>5   minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI>Dpn I digestion</LI></OL><P>       Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><OL start="3"><LI>PCR product purification</LI><LI>Transformation</LI></OL><P><STRONG>09/05/2018</STRONG></P><OL start=""><LI><P>Experimental goal: plasmid DNA amplification</P><P>Experimental steps:</P><P>a. Pick a single colony [over-density]  </P><P>b. LB amplification (Amp+)</P></LI><LI><P>Experimental goal: h-ribo SD sequence mutation [9-4 repeat]</P><P>Experimental steps:</P><P>a. PCR mutagenesis</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (100ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer F (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Q5_mix(2x)</TD><TD>20</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>17</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>40</TD></TR></TBODY></TABLE></FIGURE><P>       Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>56°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>3 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>       b.DpnI digestion</P><P>       Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><P>    c. PCR product purification</P><P>    d.Tranformation</P><P><STRONG>09/06/2018</STRONG></P><OL start=""><LI><P>Experimental goal: plasmid amplification</P><P>Experimental steps:</P></LI></OL><P>       a. Pick [9-5 mutant board] single colonies [over-density]</P><P>       b. Plasmid subtle [9-5] inoculated bacterial solution</P><OL start="2"><LI><P>Experimental goal: h-ribo mutation [a2 and b8]</P><P>Experimental steps:</P></LI></OL><P>      a.  PCR mutagenesis</P><P>      Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (100ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer F (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Q5_mix(2x)</TD><TD>20</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>17</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>40</TD></TR></TBODY></TABLE></FIGURE><P>      Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>56°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>3 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>      b. DpnI digestion</P><P>      Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><P>     c. PCR product purification</P><P>     d. Transformation</P><P><STRONG>09/07/2018</STRONG></P><P>Experimental goal: verification of plasmid</P><P>Experimental steps:</P><OL start=""><LI>Pick [9-6] single colonies</LI><LI>Plasmid DNA extraction of inoculated bacterial solution [from 9 am-7 pm]</LI><LI>Send the extracted plasmids for sequencing</LI></OL><P><STRONG>09/09/2018</STRONG></P><P>Experimental goal: h-ribo SD sequence mutation [mutA2 &amp; mutB8]</P><P>Experimental steps:</P><OL start=""><LI><P>PCR mutagenesis</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (100ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer F (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Q5_mix(2x)</TD><TD>20</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>17</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>40</TD></TR></TBODY></TABLE></FIGURE><P>     Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>56°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>3 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI><P>DpnI digestion</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><P><STRONG>09/10/2018</STRONG></P><OL start=""><LI><P>Experimental goal: plasmid amplification</P><P>Experimental steps:</P><P>Pick single colonies from the LB medium (on Sep 9th) with mutant strain [No colonies observed]</P></LI><LI><P>Experimental goal: h-ribo SD sequence mutation [mutA2 &amp; mutB8][9-9 repeat]</P></LI></OL><P>       a. PCR mutagenesis (Protocol of Teacher Gaofeng Fan)</P><P>       Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (10ng/μl)</TD><TD>5</TD></TR><TR><TD>Primer F (10 ng/μl)</TD><TD>1.25</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1.25</TD></TR><TR><TD>Restriction Endonuclease</TD><TD>1</TD></TR><TR><TD>Buffer(10x)</TD><TD>5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>36.5</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>    Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD><TD rowspan="3">18</TD></TR><TR><TD>Annealing</TD><TD>60°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>5 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>     b. Dpn I digestion</P><P>     Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours.</P><P>      c. PCR product purification</P><P>      d. Transformation</P><P><STRONG>09/11/2018</STRONG></P><OL start=""><LI><P>Experimental goal: plasmid amplification</P><P>Experimental steps:</P><P>a. Pick single colonies from the LB solid medium (on Sep 10th) with mutant strain</P><P>b. Plasmid DNA extraction</P></LI><LI><P>Experimental goal: h-ribo mutation [m2 &amp; m8]</P><P>Experimental steps:</P><P>a. PCR mutagenesis (Protocol of Teacher Gaofeng Fan)</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (10ng/μl)</TD><TD>5</TD></TR><TR><TD>Primer F (10 ng/μl)</TD><TD>1.25</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1.25</TD></TR><TR><TD>Restriction   Endonuclease</TD><TD>1</TD></TR><TR><TD>Buffer(10x)</TD><TD>5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>36.5</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></FIGURE><P>     Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD><TD rowspan="3">18</TD></TR><TR><TD>Annealing</TD><TD>60°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>72°C</TD><TD>5 minutes</TD></TR><TR><TD>Final Extension</TD><TD>72°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><P>       b. Dpn I digestion</P><P>       Reaction system:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P> The reaction system was incubated at 37°C for 2 hours.</P><P>    c. PCR product purification</P><P>    d.Transformation</P><P><STRONG>09/12/2018</STRONG></P><P>Experimental goal: verification of plasmid</P><P>Experimental steps:</P><OL start=""><LI>Pick single colonies (on the LB solid medium on Sep 11th)</LI><LI>Plasmid DNA extraction (in the morning of Sep 12th) of inoculated bacterial fluid</LI></OL><P><STRONG>09/15/2018</STRONG></P><P>Experimental goal: h-ribo SD sequence mutation [mutA2 &amp; mutB8]</P><P>Experimental steps:</P><OL start=""><LI><P>PCR mutagenesis (PFU protocol)</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>Template (10ng/μl)</TD><TD>4</TD></TR><TR><TD>Primer F (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Primer R (10ng/μl)</TD><TD>1</TD></TR><TR><TD>Pfu_mix(2x)</TD><TD>20</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>14</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>40</TD></TR></TBODY></TABLE></FIGURE><P>     Program of PCR amplifier:</P><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Step</STRONG></TH><TH><STRONG>Temperature</STRONG></TH><TH><STRONG>Time</STRONG></TH><TH><STRONG>Cycles</STRONG></TH></TR></THEAD><TBODY><TR><TD>Initial Denaturation</TD><TD>95°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95°C</TD><TD>30 seconds</TD><TD rowspan="3">30</TD></TR><TR><TD>Annealing</TD><TD>56°C</TD><TD>30 seconds</TD></TR><TR><TD>Extension</TD><TD>68°C</TD><TD>10 minutes</TD></TR><TR><TD>Final Extension</TD><TD>68°C</TD><TD>5 minutes</TD><TD>1</TD></TR><TR><TD>Hold</TD><TD>4°C</TD><TD>∞</TD><TD>1</TD></TR></TBODY></TABLE></FIGURE><OL start="2"><LI><P>Dpn I digestion</P><P>Reaction system:</P></LI></OL><FIGURE><TABLE class="table"><THEAD><TR><TH><STRONG>Reagents</STRONG></TH><TH><STRONG>Volume/μl</STRONG></TH></TR></THEAD><TBODY><TR><TD>PCR product</TD><TD>20</TD></TR><TR><TD>Buffer(10x)</TD><TD>2.5</TD></TR><TR><TD>Dpn I</TD><TD>0.5</TD></TR><TR><TD>ddH<SUB>2</SUB>O</TD><TD>2</TD></TR><TR><TD><STRONG>Total volume</STRONG></TD><TD>25</TD></TR></TBODY></TABLE></FIGURE><P>The reaction system was incubated at 37°C for 2 hours, then incubated at 4°C overnight.</P><OL start="3"><LI>PCR product purification</LI><LI>Transformation (Followed by picking single colonies and inoculate into LB medium with antibiotics, shaking bacteria fluids and plasmid DNA extraction, sending for sequencing for verification of plasmids over the next few days)</LI></OL></DIV></DIV></DIV></DIV><DIV class="container"><SPAN class="text-muted">ShanghaiTech iGEM @ 2018</SPAN></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>