<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RHIT_DesignNotebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RHIT/DesignNotebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="globalWrapper"><DIV id="mainContainer"><DIV id="contentContainer"><H2 class="attn">Design Work Notebook</H2><P style="width:85%, text-align:center"> This notebook describes the design, research, and planning that guided our project. Click on the week's date to reveal individual days' work. Weekly summaries can be found below the dates.</P><P class="Week" id="Weeks" style="color:#DBA246" onmouseover="changecursor('Weeks')">Week 1</P><P>This was the first week of work. We decided on a project, set goals, and started designing parts.</P><P class="Days" id="Days" style="color:#9CFBE4"> 6/8/15 </P><P class="Journal" id="Journal">Our team spent time discussing project ideas, establishing the overall goal to create an on/off “switch” for the mitochondria in<SPAN class="italics"> S. cerevisiae</SPAN>. We also spent time clarifying each of the medal requirements, inquiring from iGEM headquarters on questions that arose. We began discussing possibilities for parts submissions, including the ORF for<SPAN class="italics"> mRPS12</SPAN>, a promoter, and a reporter. Since yeast are able to survive without mitochondria on fermentable media, we will test the efficacy of our construct using a growth test on fermentable and nonfermentable carbon sources. The strains with their mitochondria turned off are expected to only grow on fermentable media. We also discussed Kozak sequences and determined that the Kozak sequence listed on the registry on the yeast page is actually the mammalian sequence. Our team began researching the consensus Kozak sequence in yeast. </P><P class="Days" id="Days2" style="color:#9CFBE4"> 6/9/15 </P><P class="Journal" id="Journal2">Today the group discussed two constructs that will be developed for our project. The first will be composed of the wild type<SPAN class="italics"> mRPS12</SPAN> sequence, preceded by the native Kozak sequence. We discussed the use of an ADH1 promoter. Our team designed and ordered two parts from IDT. The first sequence (construct 1 optimized) was generated by optimizing the wild type sequence of <SPAN class="italics">mRPS12</SPAN> from <SPAN class="italics">S. cerevisae</SPAN> for expression in yeast. Following optimization, two serine codons were modified from TCT to TCA in order to eliminate prohibited Xba1 restrictions sites. The <SPAN class="italics">mRPS12</SPAN> ORF was then flanked by 30 nucleotide overhangs homologous to the prefix and suffix listed in the registry to allow for insertion into pSB1C3 via homologous recombination. This part will be used as our new BioBrick part for the second silver medal requirement. 

In addition to the<SPAN class="italics"> mRPS12</SPAN> ORF construct, our team also designed and ordered a transcriptional unit to submit to the registry as a new part (construct 2 optimized). This part consists of the optimized<SPAN class="italics"> mRPS12</SPAN> ORF following the native Kozak sequence. This composite part was also flanked by 30 nucleotide overhangs homologous to the registry’s prefix and suffix. 

Lastly, our team discussed possibilities for promoters that could be used for our translational unit and concluded that a tetracycline operator system (TetO) could be useful for regulation of our construct. With this system, we would also need to utilize a construct consisting of a constitutive promoter (perhaps ADH1) and the TetR-VP64A coding region in order to produce the regulatory element (TetR-VP64A) required for the operator. </P><P class="Days" id="Days3" style="color:#9CFBE4"> 6/10/15 </P><P class="Journal" id="Journal3">Today our team began discussing methods for visualizing yeast mitochondria as well as methods for comparing the mitochondria between the on and off states. Several methods were proposed including staining with dyes, tagging with fluorescence, and viewing under a light microscope. 
It was also realized that the<SPAN class="italics">MRPS12</SPAN> ORF part order previously (construct 1) was designed with the prefix for non-coding regions rather than the prefix for coding regions, and must be reordered with the correct prefix sequence. </P><P class="Days" id="Days4" style="color:#9CFBE4"> 6/11/15 </P><P class="Journal" id="Journal4">The team discussed the creation of a new part, a modified version of the p416 plasmid which would be biobrick compatible and would be used in our experiments with our created translational unit.  We are still unsure of what assembly method to use to create this modified plasmid.</P><P class="Days" id="Days5" style="color:#9CFBE4"> 6/12/15 </P><P class="Journal" id="Journal5">Our team started today by having a meeting to establish what needed to be finished for the week’s end, as well as goals for the following week. A checklist for achieving the Gold medal was also created. Enzymes and materials for Gibson Assembly were ordered, and began planning nucleotide sequences of parts. Our team also requested to time to work with middle school teachers in order to help them understand what they can do to teach kids about synthetic biology. Construct 1 arrived from IDT. </P><P class="Week" id="Weeks2" style="color:#DBA246" onmouseover="changecursor('Weeks2')">Week 2</P><P>This week was primarily focused on finding a system to act as a 'switch' to turn the gene on and off. A main topic of research was a Tet O system.</P><P class="Days" id="Days6" style="color:#9CFBE4"> 6/15/15 </P><P class="Journal" id="Journal6">Our team performed minipreps to recover p416-GPD. Lab safety protocols were covered to Rose-Hulman’s standards. We discussed an outline for our parts documentation and submission status to the registry. The “About Our Lab” and “About Our Project” questionnaires had their first drafts completed. The TET system was investigated by our team as a potential candidate for an on/off switch. Construct 2 arrived from IDT. </P><P class="Days" id="Days7" style="color:#9CFBE4"> 6/16/15 </P><P class="Journal" id="Journal7">Our team spent more time researching the tet-o system.  We discussed regulating the tet-o system using the sugars galactose and glucose.  This system would be regulated using varying sugars in the media.  
</P><P class="Days" id="Days8" style="color:#9CFBE4"> 6/17/15 </P><P class="Journal" id="Journal8">Upon further looking into the tet-o system we found that there may have been potential copyright issues with using a tet-o system for our project.  In order to avoid any potential conflicts, we started looking into new promoter systems that could potentially be used in our project.</P><P class="Days" id="Days9" style="color:#9CFBE4"> 6/18/15 </P><P class="Journal" id="Journal9">Our team found three new potential promoter systems that may be used in the project:  MET-25, FET-3, and FIG-1.  Each promoter system was then discussed and thrown out due to issues with potential ligands needed to activate the systems and lack of information available that would prevent implementation.</P><P class="Days" id="Days10" style="color:#9CFBE4"> 6/19/15 </P><P class="Journal" id="Journal10">Upon continuing review, the team found a new promising promoter system using copper.  There are three possible promoters to use the two repressible CTR1 and CTR3 promoters and the inducible CUP1 promoter.  Additionally, in the reviewed paper a copper chelator bathocuprione disulfonate (BCS) is added to remove copper which derepresses the system.  We decided to continue research into this system as it is the most promising we have found.</P><P class="Week" id="Weeks3" style="color:#DBA246" onmouseover="changecursor('Weeks3')">Week 3</P><P> This week's focus was the Human Practices portion of our project. We planned and held three seminars for middle school summer campers on campus.</P><P class="Days" id="Days11" style="color:#9CFBE4"> 6/22/15 </P><P class="Journal" id="Journal11">We decided on continuing  our examination of literature about the CTR1 and CTR3 promoter systems.  The CUP1 promoter was ruled out since it is an inducible promoter and we prefer to use a repressible promoter.  Upon further reading, it was found that many laboratory strains of yeast possess a transposon in their CTR3 promoters, preventing function.  Thus, we chose to proceed with the CTR1 promoter.  We read that these copper systems require Copper Recognizing Elements, CuREs, in their sequences in order to bind Mac1 protein.  The activation of the system involves a binding event between the CuREs, Mac1 protein, and copper.  Our sequence of CTR1 was then analyzed for these CuREs and three were found.  Lastly, we found that a base concentration of 16nM of copper is necessary for the function of the promoter and the repression occurs around a copper concentration of 1µM.  Additionally, we began consulting the literature to delineate the mechanism by which Mac1 protein interacts with the promoter.</P><P class="Days" id="Days13" style="color:#9CFBE4"> 6/24/15 </P><P class="Journal" id="Journal13">The team planned for our human practices project with the middle schoolers.  We plan on introducing them to synthetic biology and the idea of how genes are turned into proteins.  A powerpoint presentation was made to accomplish this task and additionally a game or challenge was created to aide in this discussion.  The challenge is for the middle schoolers to put together varying biological circuits in increasing order.  The children will have access to various, promoters, genes, and terminators in order to accomplish these tasks.  The students will begin with a generic promoter, gene, terminator system.  This system will be used to explain the basic components in a general sense to the group before having them delve into the more advanced challenges.
</P><P class="Days" id="Days14" style="color:#9CFBE4"> 6/25/15 </P><P class="Journal" id="Journal14">Today the team had its synthetic biology education seminar with the middle schoolers.  The students enjoyed the presentation and activity and were extremely interested in learning about synthetic biology and the field of biology in general.  Each group was able to complete the challenges in a reasonable amount of time and were able to explain the purpose of each part in the biological circuit that they had constructed.</P><P class="Week" id="Weeks4" style="color:#DBA246" onmouseover="changecursor('Weeks4')">Week 4</P><P>This week revolved around researching leader sequences and how regulating MRPS12 might affect growth and production of fermentation products. We also began planning out new parts</P><P class="Days" id="Days15" style="color:#9CFBE4"> 6/28/15 </P><P class="Journal" id="Journal15">The team began a literature review regarding leader sequences and their function for mrps12. It was found that there are four common signal motifs that are found in leader sequences.  The proteins involved in entry are mitochondrial peptidases for cleaving the signal peptidase after entry and various TIM and TOM proteins that mediate entry.</P><P class="Days" id="Days16" style="color:#9CFBE4"> 6/29/15 </P><P class="Journal" id="Journal16">We compared the four cleavage motifs for mitochondrial peptidases with our sequence and matched our leader sequence with the R-10 cleavage motif.  The R-10 motif suggests that localization of mrps12 would occur in either the inner mitochondrial matrix or the mitochondrial matrix.  Additionally, this motif suggests that it is cleaved off in a two step cleavage event by mitochondrial processing peptidase and mitochondrial intermediate peptidase.
The team conducted a literature review to understand the genes encoded by the mitochondrial DNA in order to assess which proteins would no longer be translated as a result of the inactivation of mrps12.</P><P class="Days" id="Days17" style="color:#9CFBE4"> 6/30/15 </P><P class="Journal" id="Journal17">From the literature review, we found that some components of the electron transport chain and proteins involved in the krebs cycle would no longer be translated due to this inactivation.  The team then switched gears and wrote up our summary of our human practices events from the previous week as well as wrote a preliminary draft of our project description.  Lastly, we began discussing our track selection and narrowed our choices down to manufacturing, new application, and medicine.  </P><P class="Days" id="Days18" style="color:#9CFBE4"> 7/1/15 </P><P class="Journal" id="Journal18">Started discussion on which track to insert project into, started working on more part documentation and worked on creating a plan for which parts are needed to complete the project that we have/do not have. started idea on a potential model, which would be related to the amount of copper needed to activate/deactivate gene.</P><P class="Days" id="Days19" style="color:#9CFBE4"> 7/2/15 </P><P class="Journal" id="Journal19"> Wrapped up loose ends before week off. Created mini-preps and ran gels for a number of colonies, worked on project description.</P><P class="Week" id="Weeks5" style="color:#DBA246" onmouseover="changecursor('Weeks5')">Break Week!</P><P class="Week" id="Weeks6" style="color:#DBA246" onmouseover="changecursor('Weeks6')">Week 5</P><P> This week focused on researching how our system could be used in industry and the details of the CTR-1 copper-repressible promoter system functions.</P><P class="Days" id="Days26" style="color:#9CFBE4"> 7/13/15 </P><P class="Journal" id="Journal26">Wrote the project description for the wiki and the iGem deadline, reviewed work left to do after week off.</P><P class="Days" id="Days27" style="color:#9CFBE4"> 7/14/15 </P><P class="Journal" id="Journal27">The team completed the project description and placed it on our wiki.  We discussed track selection further and began discussing, in depth, the roles our project may play in a manufacturing setting.  In order to assess the application of our product more effectively we decided to meet with other faculty at Rose-Hulman with experience in industry using yeast for industrial processes.
</P><P class="Days" id="Days28" style="color:#9CFBE4"> 7/15/15 </P><P class="Journal" id="Journal28">Team arranged meeting with Dr. Serbezov to understand the industrial roles of yeast more effectively.</P><P class="Days" id="Days29" style="color:#9CFBE4"> 7/16/15 </P><P class="Journal" id="Journal29">The team met with Dr. Serbezov and, an incoming professor, Dr. Reizman in order to discuss the manufacturing application of our system.  We learned of various products that the general idea of our system may be used, including, ethanol, vanillin, indigo, etc. In terms of how our system could affect the overall manufacturing process.  Overall, the meeting was incredibly helpful in seeing the larger process to which our system fits and how our system may affect various aspects of that process.
</P><P class="Days" id="Days30" style="color:#9CFBE4"> 7/17/15 </P><P class="Journal" id="Journal30">Begun work on building the copper repressor part by assembling in silico the sequence for CTR1p with +/- 1kb up or downstream of the ORF as found on SGD. The region described in Labbé et. al (1997) as the CTR1 promoter as well as the CuRE transcript elements within this region were also identified. </P><P class="Week" id="Weeks7" style="color:#DBA246" onmouseover="changecursor('Weeks7')">Week 6</P><P>This week focused on the documentation of the project (poster, presentation, and potential papers) and learning more about repressible promoter systems.</P><P class="Days" id="Days31" style="color:#9CFBE4"> 7/20/15 </P><P class="Journal" id="Journal31">Discussion on how to design the copper repressor began. CTR3 was reintroduced as a possibility after concern over a transposon in the wild type CTR3 promoter was dismissed. Primers for isolating the CTR3 promoter via PCR were identified. </P><P class="Days" id="Days32" style="color:#9CFBE4"> 7/21/15 </P><P class="Journal" id="Journal32">Further reviewed paper by  Labbé et. al (1997) for designing the copper promoter. A 64bp region in the promoter region of the CTR3 gene was identified as the minimally required loci for the promoter to respond to copper input and function correctly. We also better defined our understanding of Mac1p’s interactions with CTR3 / CTR1 as involving epigenetic changes via methylation of promoter regions. 
The team created a first draft of the written content for the introduction, application, and human resources portions of the poster.
</P><P class="Days" id="Days33" style="color:#9CFBE4"> 7/22/15 </P><P class="Journal" id="Journal33">Began work on the presentation at the Giant Jamboree, including an outline of content as well as rough drafts of diagrams. The team also started efforts to secure an absorbance or fluorescence plate reader to use in experiments. 
The team typed up preliminary drafts for the documentation of the parts which we will be submitting to the iGEM parts database.
</P><P class="Days" id="Days34" style="color:#9CFBE4"> 7/23/15 </P><P class="Journal" id="Journal34">The safety page for the wiki was written by the team.  We looked into the equipment that would be needed for our ethanol assays.  The team found fluorescence readers in another biology lab on campus started to learn how to use them.  Discussions were conducted regarding how the experiments could be carried out and possible modeling that could be done.</P><P class="Days" id="Days35" style="color:#9CFBE4"> 7/24/15 </P><P class="Journal" id="Journal35">The team discussed possible papers to review for modeling yeast ethanol production with Dr. Goulet and read the user manual for the fluorescence reader for the ethanol assays.</P><P class="Week" id="Weeks8" style="color:#DBA246" onmouseover="changecursor('Weeks8')">Week 7</P><P>This week focused on learning about ethanol assays and how to use the fluorescence plate reader.</P><P class="Days" id="Days36" style="color:#9CFBE4"> 7/27/15 </P><P class="Journal" id="Journal36">We began working with the fluorescence plate reader in the Myers lab on campus.  We were able to find the user manual for the machine and the software user manual allowing us to gain an understanding of how to use the machine for our experiments.  Additionally, the team began to plan out the experiments for our ethanol assays.  We started to review the content written for the poster and plan the graphics that will accompany/replace the text currently there.</P><P class="Days" id="Days37" style="color:#9CFBE4"> 7/28/15 </P><P class="Journal" id="Journal37">The team further discussed the protocol and necessary lab work for the ethanol assays which we will be conducting.  Protocols and flowcharts were made to precisely lay out what will be occurring.  We also began thinking about how we could analyze our data.
</P><P class="Week" id="Weeks9" style="color:#DBA246" onmouseover="changecursor('Weeks9')">Week 8</P><P>This week focused on site-directed mutagenesis, the poster, and presentation for the Giant Jamboree.</P><P class="Days" id="Days41"> 8/3/15 </P><P class="Journal" id="Journal41">Sent off a set of DNA for sequencing. Planned out experiment for cell number standard curves.</P><P class="Days" id="Days42"> 8/4/15 </P><P class="Journal" id="Journal42">The team discussed site directed mutigenesis for removing illgeal PstI site and removing a premature atg in our vector  at the cleavage of the GPD promoter and the translation unit.</P><P class="Days" id="Days43"> 8/5/15 </P><P class="Journal" id="Journal43">The team ordered primers for the mutagenesis discussed on 8/4/15, discussed how to submit parts, and which parts go where, and how to document them.</P><P class="Days" id="Days44"> 8/6/15 </P><P class="Journal" id="Journal44">We started designing poster for giant Jamboree. Primary discussion revolved around color scheme and graphics for the poster. We also selected information sections that would be put onto the poster.</P><P class="Days" id="Days45"> 8/7/15 </P><P class="Journal" id="Journal45">Work continued on the poster and graphics. We also began discussing presentation. The primary focus was to allow members to present information that they were comfortable with. We also started ideas for graphics that would be good to have on our presentation.</P><P class="Week" id="Weeks10" style="color:#DBA246" onmouseover="changecursor('Weeks10')">Week 9</P><P> This week's design work focused on the presentation and poster, as well as discussing documentation for part submisson.</P><P class="Days" id="Days46" style="color:#9CFBE4"> 8/10/15 </P><P class="Journal" id="Journal46">We continued work on poster and presentation. Today’s focus was on the background biology and considering how much of it was needed for the jamboree, considering the significant biological background that attendees will have.</P><P class="Days" id="Days47" style="color:#9CFBE4"> 8/11/15 </P><P class="Journal" id="Journal47">We continued work on poster and presentation. Today’s focus was on fermentation/aerobic respiration, specifically considering what information is common knowledge and what information will help to expand the audience's understanding.</P><P class="Days" id="Days48" style="color:#9CFBE4"> 8/12/15 </P><P class="Journal" id="Journal48">We continued work on poster and presentation. Today’s focus was on potential uses of our project in industry, which is hopefully our track. Secondary metabolites is a significant portion that we wanted to address, since production of the metabolites are the primary use of fermentation in industry. We brainstormed potential questions from judges about any information thus far selected.</P><P class="Days" id="Days49" style="color:#9CFBE4"> 8/13/15 </P><P class="Journal" id="Journal49">Prepared for the creation of the mitochondrial localization sequence for submission to the database.</P><P class="Days" id="Days50" style="color:#9CFBE4"> 8/14/15 </P><P class="Journal" id="Journal50">We wrapped up for the week and summarized the work we still needed to get done.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>