<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_RHIT_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:RHIT/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="globalWrapper"><DIV id="mainContainer"><DIV id="contentContainer"><H2 class="attn">Wet Lab Notebook</H2><P style="width:85%, text-align:center"> This notebook regards the lab work that was done in order to complete our project. The highlights of our wet-lab work are listed below the week they occured. Click on the week's date to reveal individual days' work.</P><P class="Weeks" id="Weeks" style="color:#DBA246" onmouseover="changecursor('Weeks')">Week 1</P><P>Highlight: Commence lift-off!</P><P class="Days" id="Days" style="color:#9CFBE4"> 6/11/15 </P><P class="Journal" id="Journal"> p416-GPD was streaked on LB+Amp plate</P><P class="Weeks" id="Weeks2" style="color:#DBA246" onmouseover="changecursor('Weeks2')">Week 2</P><P>Highlight: NEBuilder and ligation techniques were used to insert our translational unit into pSB1C3 and p416-GPD.</P><P class="Days" id="Days2" style="color:#9CFBE4"> 6/14/15 </P><P class="Journal" id="Journal2"> Four overnight cultures of p416-GPD colonies were started. </P><P class="Days" id="Days3" style="color:#9CFBE4"> 6/15/15 </P><P class="Journal" id="Journal3"> We performed minipreps on overnight cultures of p416 and recovered eight aliquots of 50 µL. Miniprep #s 1, 3, 5, and 7 were digested with EcoR1. When run on a .9% gel, confirmed that minipreps are concentrated enough to use. </P><P class="Days" id="Days4" style="color:#9CFBE4"> 6/16/15 </P><P class="Journal" id="Journal4">IDT constructs 1 and 2 were resuspended to 10 ng/µL.</P><P class="Days" id="Days5" style="color:#9CFBE4"> 6/17/15 </P><P class="Journal" id="Journal5">The iGEM shipping vector, pSB1C3, and the translational unit (T.U.) were digested with EcoRI and PstI and the resulting fragments were ligated. The NEBuilder procedure was also used to assemble ligated pSB1C3 and T.U. from IDT. Both NEBuilder assembled, and ligation assembled plasmids were transformed with <SPAN class="italic"> E. coli </SPAN>and 100 µL aliquots were plated on LB+chlor plates. </P><P class="Days" id="Days6" style="color:#9CFBE4"> 6/18/15 </P><P class="Journal" id="Journal6">T.U. and p416 (from miniprep #1) were digested with SpeI and XbaI to remove the multiple cloning site in p416 and to create compatible ends. Digests were purified using the QIAquick Gel Extraction Kit. </P><P class="Days" id="Days7" style="color:#9CFBE4"> 6/19/15 </P><P class="Journal" id="Journal7">The gel purified fragments from yesterday were concentrated by ethanol precipitation, succeeded by resuspension in TE. V was resuspended in 10 µL and I was resuspended in 40 µL. The entire 10 µL V was ligated with 7 µL I and transformed into<SPAN class="italic"> E. coli</SPAN>. This ligation should insert the translational unit into p416 where the multiple cloning site was originally.

More aliquots of the pSB1C3/T.U. ligation and NEBuilder Assembly were plated (2 plates with 300 µL each).
</P><P class="Weeks" id="Weeks3" style="color:#DBA246" onmouseover="changecursor('Weeks3')">Week 3</P><P>Highlight: pSB1C3/translational unit assembly yields seemingly correct results.</P><P class="Days" id="Days8" style="color:#9CFBE4"> 6/22/15 </P><P class="Journal" id="Journal8">No colonies were seen on the p416/T.U. ligation plates, but colonies were seen on pSB1C3/T.U. plates from Friday. Eight colonies were picked and grown up in 3 mL cultures overnight. 
</P><P class="Days" id="Days9" style="color:#9CFBE4"> 6/23/15 </P><P class="Journal" id="Journal9"> The rest of the p416/T.U. ligation was plated (350µL on each of 2 plates). Boiling preps were performed on the 3 mL cultures of pSB1C3/T.U. </P><P class="Days" id="Days10" style="color:#9CFBE4"> 6/24/15 </P><P class="Journal" id="Journal10">There are colonies on one of the p416/T.U. plates, so we started 3 mL cultures of 10 colonies (A1-9, B1) for boiling preps tomorrow. Also, boiling preps of pSB1C3/T.U. (from yesterday) were digested with PvuII and SpeI. Bands of 1703 and 847 were expected. After running on a gel, three correct samples were found: NB1 (NEBuilder Assembly, plate B, colony 1), LA3 (Ligation assembly, plate A, colony 3), and LB2 (Ligation assembly, plate B, colony 2).  </P><P class="Days" id="Days11" style="color:#9CFBE4"> 6/25/15 </P><P class="Journal" id="Journal11">Fresh 3 mL cultures of three correct samples from yesterday were started for minipreps tomorrow. Overnight cultures for A1-8 grew, so boiling preps were performed on these. The preps were then digested with XbaI and PvuII and run on a gel, expecting correct bands at 4450, 879, and 931. It is possible that the insert ligated in the opposite direction due to compatibility between XbaI and SpeI sites. If this happened, we would expect bands at 4450 and 1810 because by mismatching of cohesive ends, the XbaI site is eliminated. On the gel we saw bands at ~1300, ~3000, and ~4500. 
</P><P class="Days" id="Days12" style="color:#9CFBE4"> 6/26/15 </P><P class="Journal" id="Journal12">Minipreps were performed on NB1, LA3, and LB2 cultures. Minipreps were then digested with SpeI to be run on a gel. Also today, 6 more transformants from p416/T.U. plate B (from 6/23) were used to create a master plate in order to start new cultures.</P><P class="Weeks" id="Weeks4" style="color:#DBA246" onmouseover="changecursor('Weeks4')">Week 4</P><P>Highlight: Cloned first part for submission.</P><P class="Days" id="Days13" style="color:#9CFBE4"> 6/28/15 </P><P class="Journal" id="Journal13"> A gel was run on SpeI digested NB1, LA3, and LB2 and affirmed that we have the correct plasmid.</P><P class="Days" id="Days14" style="color:#9CFBE4"> 6/29/15 </P><P class="Journal" id="Journal14"> A dot analysis was performed using NB1, LA3, and LB2 to estimate concentrations for sequencing. The resulting estimations were: NB1 - 180 ng/µL; LA3 - 180 ng/µL; LB2 - 100 ng/µL.</P><P class="Days" id="Days15" style="color:#9CFBE4"> 6/30/15 </P><P class="Journal" id="Journal15"> Remaining p416/T.U. ligation mixture was used in 6 transformation reactions: three 1 µL reactions (1, 2, and 3), and three 2 µL reactions (4, 5, and 6). Transformations were plated onto twelve plates, two plates (A and B) for each of the six transformations with 200 µL on each plate. </P><P class="Days" id="Days16" style="color:#9CFBE4"> 7/1/15 </P><P class="Journal" id="Journal16"> Colonies were found on five of the twelve plates from yesterday, and colonies were picked to start 3 mL overnight cultures (8 from 4A, 2 from 5A, 5 from 5B, 7 from 6A, and 10 from 6B) for boiling preps. </P><P class="Days" id="Days17" style="color:#9CFBE4"> 7/2/15 </P><P class="Journal" id="Journal17"> Twenty-seven of the thirty-two cultures grew, and boiling preps were performed on these cultures. Then, boiling preps were digested with PvuII and SpeI and run on a gel, expecting bands at 4450, 931, and 879. All samples returned incorrect results, with bands at 4450 and ~1300.</P><P class="Weeks" id="Weeks5" style="color:#DBA246;text-align:center" onmouseover="changecursor('Weeks5')">Break Week!</P><P class="Weeks" id="Weeks6" style="color:#DBA246" onmouseover="changecursor('Weeks6')">Week 5</P><P>Highlight: pSB416-GPD was created - a Biobrick compatible p416-GPD vector!</P><P class="Days" id="Days23" style="color:#9CFBE4"> 7/13/15 </P><P class="Journal" id="Journal23">p416 miniprep #3 was digested with SpeI and XbaI.</P><P class="Days" id="Days24" style="color:#9CFBE4"> 7/14/15 </P><P class="Journal" id="Journal24"> NB1 miniprep was digested with XbaI and SpeI in order to recover the translational unit for new ligation. After gel extraction of ligated p416 (Lp416) and T.U. from NB1, only Lp416 was recovered. Lp416 band was excised, but was not purified yet.</P><P class="Days" id="Days25" style="color:#9CFBE4"> 7/15/15 </P><P class="Journal" id="Journal25"> LA3 was digested with XbaI and SpeI to recover T.U. Both Lp416 and T.U. were purified by QIAquick gel extraction kit.</P><P class="Days" id="Days26" style="color:#9CFBE4"> 7/16/15 </P><P class="Journal" id="Journal26">The gel purified I and V (T.U. and Lp416 respectively). Lp416 was not concentrated enough after gel purification, so ethanol precipitation was used to concentrate the DNA. The two fragments were then ligated and transformed into <SPAN class="italics"> E. coli </SPAN> competent cells.</P><P class="Days" id="Days27" style="color:#9CFBE4"> 7/17/15 </P><P class="Journal" id="Journal27"> Colonies did not grow on ligation plates.</P><P class="Weeks" id="Weeks7" style="color:#DBA246" onmouseover="changecursor('Weeks7')">Week 6</P><P>Highlight: NEBuilder was used to insert our translational unit into pSB416.</P><P class="Days" id="Days28" style="color:#9CFBE4"> 7/20/15 </P><P class="Journal" id="Journal28">pSB416-GPD insert fragment was received and resuspended in TE to 10 ng/ml. Vector p416 was linearized by digestion with XbaI and XhoI. In order to create the pSB416 vector, 32.5 ng of insert fragment and 75 ng of linearized p416 were assembled using NEBuilder HiFi DNA Assembly Cloning Kit. Assembled product was then transformed into NEB 5-alpha competent <SPAN class="italics"> E. coli </SPAN> cells and plated on five plates of 100 µl each. Also, pSB1C3/T.U. constructs NB1, LA3, and LB2 were transformed into <SPAN class="italics"> E. coli </SPAN> in order to grow up for stock cultures in glycerol.
</P><P class="Days" id="Days29" style="color:#9CFBE4"> 7/21/15 </P><P class="Journal" id="Journal29"> Colonies were found on all plates from yesterday so we grew up 3 ml cultures for boiling preps tomorrow. Also, ran a lamp glass gel on digested p416 from yesterday to make sure it was cut, which was confirmed.
</P><P class="Days" id="Days30" style="color:#9CFBE4"> 7/22/15 </P><P class="Journal" id="Journal30">Today, we performed boiling preps on NB1, LA3, LB2, and pSB416 cultures from overnight. The iGEM/T.U. preps were then cut with XbaI and SpeI, and the pSB416 preps were cut with PvuII and BamHI. The digests were then run on a gel. the iGEM/T.U. digests were expected to result in bands at 2062 and 488, and the pSB416 digests were expected to result in bands at 4450, and 1303. NB1 returned positive results, though LA3 and LB2 did not. The remaining liquid cultures of these three were then placed in glycerol media for storage. All 6 pSB416 digests resulted in bands of the correct sizes, and the remaining liquid cultures of these were used for a miniprep to be sent off for sequencing. 
</P><P class="Days" id="Days31" style="color:#9CFBE4"> 7/23/15 </P><P class="Journal" id="Journal31">Today, pSB416 minipreps were digested with BamHI and PvuII and run on a gel. The resulting gel was messy due to star activity of PvuII, so the minipreps were then digested with PstI to run on a gel tomorrow to estimate concentrations in order to send off samples for sequencing.
</P><P class="Days" id="Days32" style="color:#9CFBE4"> 7/24/15 </P><P class="Journal" id="Journal32">A gel on pSB416-GPD digests, resulted in correct bands at 1868 and 3885.
</P><P class="Weeks" id="Weeks8" style="color:#DBA246" onmouseover="changecursor('Weeks8')">Week 7</P><P>Highlight: pSB1C3/translational unit construct sequencing came back correct for one sample.</P><P class="Days" id="Days33" style="color:#9CFBE4"> 7/26/15 </P><P class="Journal" id="Journal33">Sample 1B of pSB416 was digested with SpeI, and used with T.U. Gblock in the NEBuilder protocol. E. coli was then transformed with the assembly product and plated on LB + Amp plates.
</P><P class="Days" id="Days34" style="color:#9CFBE4"> 7/27/15 </P><P class="Journal" id="Journal34"> Transformation plates of pSB416 and T.U. have colonies. Today, we started 3 mL overnight cultures for boiling preps tomorrow. Yeast strains were also streaked on YPD plates to use for transformation later this week. New liquid LB was made.</P><P class="Days" id="Days35" style="color:#9CFBE4"> 7/28/15 </P><P class="Journal" id="Journal35"> The overnight cultures of NEBuilder transformants were used for boiling preps, and then digested with PvuII and run on a gel next to pSB416-GPD sample 1B to compare. Results showed no differences between the transformants and the empty vector. NEBuilder protocol was redone using less vector DNA, and incubated at 50º for 45 minutes instead of 15. Meanwhile, 1 µl of the Gblock was run on a gel to ensure it is still useful, which was confirmed.</P><P class="Days" id="Days36" style="color:#9CFBE4"> 7/29/15 </P><P class="Journal" id="Journal36"> Boiling preps from yesterday were digested with SpeI to determine if anything unusual was happening at the SpeI site. A gel showed that cut and uncut samples looked the same, indicating something else was happening. New 3 ml cultures were started by picking 8 transformants from NEB plates from yesterday. Also, single colonies of BY4742 and α disrupted strain were picked and smeared on a fresh YPD plate to use to inoculate 5 ml cultures tomorrow that will be used for yeast transformations on Friday.</P><P class="Days" id="Days37" style="color:#9CFBE4"> 7/30/15 </P><P class="Journal" id="Journal37">Overnight cultures were used for boiling preps, then cut with PvuII and run on a gel. Four preps (1A, 2A, 2B, and 4A) returned positive results and the remaining liquid cultures of these four were used for minipreps. The four minipreps and a pSB416-GPD miniprep, were then cut with PvuII and run on a gel. The four returned positive results again and will be used tomorrow for a yeast transformation.</P><P class="Days" id="Days38" style="color:#9CFBE4"> 7/31/15 </P><P class="Journal" id="Journal38">Today, we transformed yeast with pSB416-GPD (2A) and pSB416-GPD-S12 (2A) following the protocol for yeast transformation from the Breeden lab. Four transformation reactions were then plated on four plates each: BY4742 + pSB416-GPD; BY4742 + pSB416-GPD-S12; Disrupted + pSB416-GPD; and Disrupted + pSB416-GPD-S12. All transformations were plated on -ura plates to select for the plasmid.</P><P class="Weeks" id="Weeks9" style="color:#DBA246" onmouseover="changecursor('Weeks9')">Week 8</P><P>Highlight: Site-directed mutagenesis was used to change a premature start codon in our construct.</P><P class="Days" id="Days38" style="color:#9CFBE4"> 8/3/15 </P><P class="Journal" id="Journal38">Sequencing came back for pSB1C3 clones. Sample NB1 came back showing a single deletion (G 2372), LB2 came back with a double point deletion (TG 2452-2453), and LA3 sequence did not cover the entire <I>MRPS12</I> sequence and will be re-sequenced.
Three of the four yeast transformation reactions seem to have worked. The reaction that did not produce colonies was the BY4742/pSB416-GPD reaction. The other three reactions produced colonies. Three colonies were picked from each of these three, resuspended in water, and spotted on both dextrose and glycerol plates. These should show growth by Thursday and will tell us whether or not our translational unit was able to restore function to the strains with their endogenous copy of <I>MRPS12</I> disrupted.</P><P class="Days" id="Days39" style="color:#9CFBE4"> 8/4/15 </P><P class="Journal" id="Journal39">Today, 5 mL cultures of YPD were inoculated with BY4742 for transformation tomorrow.</P><P class="Days" id="Days40" style="color:#9CFBE4"> 8/5/15 </P><P class="Journal" id="Journal40">BY4742 was transformed with pSB416-GPD (2B) and pSB416-S12 (2A). Also, eight more colonies of Disrupted + and - each were resuspended in 50 µl water and 25 µl was spotted on a dextrose plate and 25 µl was spotted on a glycerol plate. The disrupted strain was also used to inoculate a new 5 ml culture of YPD for a new transformation tomorrow.</P><P class="Days" id="Days41" style="color:#9CFBE4"> 8/6/15 </P><P class="Journal" id="Journal41">Disrupted strain was transformed with pSB416-GPD (2B) and pSB416-S12 (1A). We got sequencing back for the pSB416-S12 sequences and 2A is the only wrong one. This means that the previous transformations are useless because they contained this plasmid. The parent strain will have to be transformed again with a different sample. </P><P class="Weeks" id="Weeks10" style="color:#DBA246" onmouseover="changecursor('Weeks10')">Week 9</P><P>Highlight: Plasmid corrections were made.</P><P class="Days" id="Days43" style="color:#9CFBE4"> 8/10/15 </P><P class="Journal" id="Journal43">Today, we will use the Q5 site-directed mutagenesis kit from NEB in order to change a nucleotide in pSB416-GPD to remove an illegal PstI site so we can submit the vector as a part. We also transformed E. coli with pSB416-GPD/A (1C) and with pSB416-GPD/A+MRPS12tu (4A) in order to store the DNA in glycerol stock tomorrow.</P><P class="Days" id="Days44" style="color:#9CFBE4"> 8/11/15 </P><P class="Journal" id="Journal44">Transformation plates did not have colonies. We redid the Q5 protocol with the new primers that came in today that will change the premature ATG site. This should make our constructs functional. We also replated pSB416-GPD/A and pSB416-GPD/A+MRPS12</P><P class="Days" id="Days45" style="color:#9CFBE4"> 8/12/15 </P><P class="Journal" id="Journal45">Today, we picked 6 colonies from each of the Q5 transformation plates and started overnight cultures, also picked one colony each from pSB416-GPD/A and pSB416-GPD/A+MRPS12 to start overnight cultures.</P><P class="Days" id="Days46" style="color:#9CFBE4"> 8/13/15 </P><P class="Journal" id="Journal46">Only pSB416-GPD/A culture grew, so we did a boiling prep on that and stored some in glycerol stock. Tomorrow, we will run a digest on the boiling prep and run on a gel to confirm we have the correct plasmid in glycerol. We also plated more transformations because none of the other cultures grew. We plated the remaining transformation mixture from Tuesday (-TU-ATG A; +TU-ATG A), transformations with the remaining KLD mixture from Tuesday (-TU-ATG B; +TU-ATG B), and transformations done after new KLD reactions were done using PCR products from before (-TU-Pst B; -TU-ATG C; +TU-ATG C).</P><P class="Days" id="Days47" style="color:#9CFBE4"> 8/14/15 </P><P class="Journal" id="Journal47">Started overnight cultures using four transformants of +TU-ATG and -TU-ATG.</P><P class="Weeks" id="Weeks11" style="color:#DBA246" onmouseover="changecursor('Weeks11')">Week 10</P><P>Highlight: Cloned MLS for submission.</P><P class="Days" id="Days48" style="color:#9CFBE4"> 8/17/15 </P><P class="Journal" id="Journal48">Digest pSB416, pSB416 - ATG, +MRPS12, and +MRPS12 - ATG with BtsCI and run on separate gels to determine if ATG problem was corrected. Gels did not return very clear results.</P><P class="Days" id="Days49" style="color:#9CFBE4"> 8/18/15 </P><P class="Journal" id="Journal49">Re-run gels from yesterday using 1.2% gel and running at 70V. Neither gel returned positive results. </P><P class="Days" id="Days50" style="color:#9CFBE4"> 8/19/15 </P><P class="Journal" id="Journal50">Redo Q5 mutagenesis with diluted DNA and primers to correct premature ATG. Also, use NEBuilder to insert mitochondrial leader sequence Gblock into pSB1C3 for submission.</P><P class="Days" id="Days51" style="color:#9CFBE4"> 8/20/15 </P><P class="Journal" id="Journal51">Picked 4 colonies from each Q5 transformation and started 3 ml cultures.</P><P class="Days" id="Days52" style="color:#9CFBE4"> 8/21/15 </P><P class="Journal" id="Journal52">Miniprep DNA from overnight cultures (pSB416-GPD+MRPS12, pSB416-GPD-MRPS12, and pSB1C3+MLS). Run gel on minipreps. Gel returned unclear results, so we sent off DNA for sequencing.</P><P class="Weeks" id="Weeks12" style="color:#DBA246" onmouseover="changecursor('Weeks12')">Weeks Following</P><P>Highlight: Parts submission and lab clean-up.</P><P class="Days" id="Days53" style="color:#9CFBE4"> 9/3/15 </P><P class="Journal" id="Journal53">Start yeast precultures with the disrupted strain and the parent strain.</P><P class="Days" id="Days54" style="color:#9CFBE4"> 9/4/15 </P><P class="Journal" id="Journal54">Transform P and D with pSB416-GPD+MRPS12-ATG clone #2 (correct sequence) and pSB416-GPD-ATG (control).</P><P class="Days" id="Days55" style="color:#9CFBE4"> 9/7/15 </P><P class="Journal" id="Journal55">Streak out glycerol stocks of pSB416-GPD+MRPS12-ATG clones 1 and 2.</P><P class="Days" id="Days56" style="color:#9CFBE4"> 9/8/15 </P><P class="Journal" id="Journal56">Start precultures for yeast transformation. Start 3 ml cultures of pSB416-GPD+MRPS12 clones for miniprep tomorrow. Pick 8 of each transformants with -ATG that worked for controls.</P><P class="Days" id="Days57" style="color:#9CFBE4"> 9/9/15 </P><P class="Journal" id="Journal57">Re-transform yeast with pSB416-GPD+MRPS12 clones 1 and 2.</P><P class="Days" id="Days58" style="color:#9CFBE4"> 9/14/15 </P><P class="Journal" id="Journal58">Transformations did not work. We will not have time to get positive results before part submissions are due.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>