<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Yale_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Yale/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Yale/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Yale/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Yale/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><TABLE style="color:#1b2c8a;background-color:#0c6;" cellpadding="3" cellspacing="1" border="1" width="62%" align="center"><TBODY><TR><TH align="center"><A href="/Team:Yale" title="Team:Yale">Home</A></TH><TH align="center"><A href="/Team:Yale/Team" title="Team:Yale/Team">Team</A></TH><TH align="center"><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Yale" class="external text" rel="nofollow">Official Team Profile</A></TH><TH align="center"><STRONG class="selflink">Project</STRONG></TH><TH align="center"><A href="/Team:Yale/Parts" title="Team:Yale/Parts">Parts Submitted to the Registry</A></TH><TH align="center"><A href="/Team:Yale/Modeling" title="Team:Yale/Modeling">Modeling</A></TH><TH align="center"><A href="/Team:Yale/Notebook" title="Team:Yale/Notebook">Notebook</A></TH><TH align="center"><A href="/Team:Yale/Safety" title="Team:Yale/Safety">Safety</A></TH><TH align="center"><A href="/Team:Yale/Attributions" title="Team:Yale/Attributions">Attributions</A></TH></TR></TBODY></TABLE><P><B>Biological diversity is a very powerful tool, but we are limited by the number of model organisms available to us today. Wouldn't it be wonderful to harness the cryptic pathways of any organism? This year, Yale tried to make a key step in that direction.</B></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Background"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Background</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Recombineering"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Recombineering</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#MAGE"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">MAGE</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-4"><A href="#Project_Overview"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Project Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Upcoming_goals"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Upcoming goals</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Selection_of_an_industrially-relevant_strain_of_the_species_of_interest"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Selection of an industrially-relevant strain of the species of interest</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Bacillus_subtilis"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Bacillus subtilis</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Acinetobacter_baylyi"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Acinetobacter baylyi</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Optimization_of_synthetic_oligonucleotides_for_efficient_genome_modification"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Optimization of synthetic oligonucleotides for efficient genome modification</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#lacZ_reporter"><SPAN class="tocnumber">2.3.1</SPAN><SPAN class="toctext">lacZ reporter</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Defective_selectable_marker"><SPAN class="tocnumber">2.3.2</SPAN><SPAN class="toctext">Defective selectable marker</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Active_selectable_marker"><SPAN class="tocnumber">2.3.3</SPAN><SPAN class="toctext">Active selectable marker</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Miscellaneous_plasmid_pieces:_ori_elements.2C_terminators.2C_restriction_sites"><SPAN class="tocnumber">2.3.4</SPAN><SPAN class="toctext">Miscellaneous plasmid pieces: ori elements, terminators, restriction sites</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Construction_of_strains_with_deficiencies_in_their_mismatch_repair_.28MMR.29_pathways"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Construction of strains with deficiencies in their mismatch repair (MMR) pathways</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Maintenance_of_bacterial_samples"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Maintenance of bacterial samples</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Verification_of_the_universal_replication_of_pBAV1K-T5-gfp_in_Ac.2C_Bs.2C_and_Ec"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Verification of the universal replication of pBAV1K-T5-gfp in Ac, Bs, and Ec</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Transformation_of_B._subtilis_and_Acinetobacter_baylyi_with_pBAV1K-based_construct"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Transformation of B. subtilis and Acinetobacter baylyi with pBAV1K-based construct</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Primer_Design"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Primer Design</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#cT5_.28normal.29"><SPAN class="tocnumber">2.8.1</SPAN><SPAN class="toctext">cT5 (normal)</SPAN></A></LI><LI class="toclevel-4 tocsection-20"><A href="#Sequencing_Primers_.28ordered_24_Jul_2012.29"><SPAN class="tocnumber">2.8.1.1</SPAN><SPAN class="toctext">Sequencing Primers (ordered 24 Jul 2012)</SPAN></A></LI><LI class="toclevel-4 tocsection-21"><A href="#Sequencing_Primers_II_-_full_cassette_sequencing"><SPAN class="tocnumber">2.8.1.2</SPAN><SPAN class="toctext">Sequencing Primers II - full cassette sequencing</SPAN></A></LI><LI class="toclevel-4 tocsection-22"><A href="#ADP1_Integration_Primers"><SPAN class="tocnumber">2.8.1.3</SPAN><SPAN class="toctext">ADP1 Integration Primers</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Homolog_Library_Primers"><SPAN class="tocnumber">2.8.2</SPAN><SPAN class="toctext">Homolog Library Primers</SPAN></A></LI><LI class="toclevel-3 tocsection-24"><A href="#Other_Primers"><SPAN class="tocnumber">2.8.3</SPAN><SPAN class="toctext">Other Primers</SPAN></A></LI><LI class="toclevel-3 tocsection-25"><A href="#Growth_curves_for_Ac.2C_Bs.2C_and_Ec"><SPAN class="tocnumber">2.8.4</SPAN><SPAN class="toctext">Growth curves for Ac, Bs, and Ec</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Effect_of_electroporation_on_competency_in_Ab_and_Bs"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Effect of electroporation on competency in Ab and Bs</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Generating_a_beta_protein_library"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">Generating a beta protein library</SPAN></A></LI><LI class="toclevel-1 tocsection-28"><A href="#Helpful_Links"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Helpful Links</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#On_MAGE_and_its_Components"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">On MAGE and its Components</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#On_the_Strains_Chosen_for_Experimentation"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">On the Strains Chosen for Experimentation</SPAN></A></LI><LI class="toclevel-3 tocsection-31"><A href="#Acinetobacter_baylyi_ADP1_.28formerly_BD413_before_1995.29"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Acinetobacter baylyi ADP1 (formerly BD413 before 1995)</SPAN></A></LI><LI class="toclevel-3 tocsection-32"><A href="#Bacillus_subtilis_2"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Bacillus subtilis</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#On_Expression.2FIntegration_Vectors_and_their_Components"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">On Expression/Integration Vectors and their Components</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#On_Various_Aspects_of_our_Oligo-Optimization_construct"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">On Various Aspects of our Oligo-Optimization construct</SPAN></A></LI><LI class="toclevel-2 tocsection-35"><A href="#On_Methods_and_Technical_Aspects_of_our_Project"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">On Methods and Technical Aspects of our Project</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Confirmation_of_the_universality_of_the_pBAV1K-T5_vector"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Confirmation of the universality of the pBAV1K-T5 vector</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Background">Background</SPAN></H1><P>The project this year relies on two relatively new technologies, Recombineering and Multiplex Automated Genome Engineering (MAGE).
</P><H2><SPAN class="mw-headline" id="Recombineering">Recombineering</SPAN></H2><P>Recombineering is recombinase mediated genetic engineering. It harnesses the function of &quot;recombinases&quot; like the bacteriophage Lambda protein Beta that bind single stranded pieces of DNA and target them to the lagging strand of the DNA replication fork. Because the replication fork is conserved over all Kingdoms of life, this technique, with the right recombinase, could potentially work in any species.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:871px;"><DIV class="thumbcaption">Costantino &amp; Court. PNAS (2003)</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="MAGE">MAGE</SPAN></H2><P>&quot;MAGE&quot; is recombineering on steroids. Instead of introducing one type of synthetic DNA, many areas of the genome are targeted. By repeating the procedure over and over again in discrete cycles, an incredible amount of diversity can be created. This can be used in a wide range of applications, including pathway optimization and entire codon engineering. So far, MAGE has only been developed for <I>E. coli</I>.
</P><P>But MAGE does have some challenges that limit its efficiency, principally among them the costly procedure of electroporation. Although this is the most efficient way to get the mutagenic oligonucleotides into the <I>E. coli</I> cells, it kills 90% of the cells in solution and requires a recovery growth period. This dramatically lowers efficiency rates.
</P><P><B>We aim to increase both the diversity of MAGE-able organisms and the efficiency of the technique by introducing recombinase-mediated genetic engineering to naturally competent organisms.</B></P><H1><SPAN class="mw-headline" id="Project_Overview">Project Overview</SPAN></H1><P>At the most fundamental level, the successful development of an optimized and fully functional MAGE system in a novel organism requires addressing five points:
</P><OL><LI>Selection of an industrially-relevant strain of the species of interest
</LI><LI>Optimization of synthetic oligonucleotides for efficient genome modification
</LI><LI>Construction of strains with deficiencies in their mismatch repair (MMR) pathways
</LI><LI>Isolation of ssDNA-binding recombinase proteins that bind mutagenic oligos
</LI><LI>Implementation of the MAGE methodology through continuous cycles of recombination
</LI></OL><P>Keeping those five main criteria of MAGE in mind, we propose to introduce the process in the naturally competent species <I>Bacillus subtilis</I> (Bs) and <I>Acinetobacter baylyi</I> (Ac, or ADP1) in an effort to enhance the efficiency and diversity of this method. Our collective research on each point is presented here, along with some further background where necessary.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:902px;"><DIV class="thumbcaption">A flow chart outlining the major steps of incorporating MAGE into our MMR-deficient strains of <I>B. subtilis</I> and <I>Acinetobacter baylyi</I></DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Upcoming_goals">Upcoming goals</SPAN></H2><P>Our project is multifaceted, but we have already completed most of the component projects and are rapidly approaching a proof of concept, and as of October 3 expect within weeks to have integrated our test cassette onto the genome of a naturally competent cell, transfected it with the beta protein submitted to the Parts Registry, and observed its behavior after exposure to a mutagenic oligo pool. If successful, by November we will have demonstrated an easier and widely applicable method for rapid mutagenesis and directed evolution.
</P><H2><SPAN class="mw-headline" id="Selection_of_an_industrially-relevant_strain_of_the_species_of_interest">Selection of an industrially-relevant strain of the species of interest</SPAN></H2><H3><SPAN class="mw-headline" id="Bacillus_subtilis"><I>Bacillus subtilis</I></SPAN></H3><P>Our efforts in Bs will be focused on two different MMR-deficient strains obtained separately:
</P><UL><LI><B>1A833</B> - <I>from BGSC (obtained 6/20 via spore dots on filter disks)</I></LI><LI><I>Reference</I>: BTS13 in Smith <I>et al.</I> (2001)
</LI><LI><I>Gene descriptor</I>: <I>ÎMutSL::<SPAN style="color:#660066">Spc</SPAN></I></LI><LI><I>Growth conditions</I>: LB, TBAB, Nutrient Agar, or other rich medium containing <SPAN style="color:#660066">spectinomycin</SPAN> (100 ug/mL); 37<SUP>o</SUP> C
</LI><LI><I>Ancestry</I>: Strain PY79 (BGSC no. 1A747 - <I>expected arrival imminent</I>)
</LI><LI>Protrophic, shares common ancestor as 168 with 28.6kb W23 island surrounding <I>trpC</I> locus
</LI></UL><LI><B>PERM739</B> - <I>from Pedraza-Reyes (obtained 7/12 via agar plates of questionable sterility)</I></LI><LI><I>Reference</I>: Pedraza-Reyes and Yasbin (2004)
</LI><LI><I>Gene descriptor</I>: <I>ÎMutSL::<SPAN style="color:#FF3399">Neo</SPAN></I></LI><LI><I>Growth conditions</I>: TBAB, Penassay broth (PAB) for liquid cultures, both supplemented with <SPAN style="color:#FF3399">Neo</SPAN></LI><LI><I>NOTE: Waiting for author response re: use of LB instead of TBAB, and recommended concentration of <SPAN style="color:#FF3399">Neo</SPAN></I></LI><LI><I>Ancestry</I>: YB955 (<I>hisC952 metB5 leuC427 xin-1 SpÎ²<SUP>SENS</SUP></I>)
</LI><LI>Prophage-&quot;cured&quot; auxotrophic derivative of 168
</LI><P>As both strains are already deficient in mismatch repair, Item 3 (<I>Construction of strains with deficiencies in their mismatch repair pathways</I>) has also been addressed by our choice of strains. The knockout of <I>MutSL</I> has been confirmed in 1A833 by selection with <SPAN style="color:#660066">spectinomycin</SPAN> in both the plating of the rehydrated spore disks and the O/N culture for glycerol stocks.
</P><H3><SPAN class="mw-headline" id="Acinetobacter_baylyi"><I>Acinetobacter baylyi</I></SPAN></H3><P>Efforts in Acinetobacter will be focused on one strain.
</P><UL><LI><B>ADP1 <I>ÎmutS</I></B> - <I>from Genescope (obtained 6/21 in liquid culture from Prof. Isaacs)</I></LI></UL><DL><DD><UL><LI><I>Reference</I>: de Berardinis <I>et al.</I>, 2008, supplementary table 1 - <A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Msb200810-s2.xls" class="new" title="Msb200810-s2.xls">Media:Msb200810-s2.xls</A></LI><LI><I>Gene Decriptor</I>: <I>ÎmutS::<SPAN style="color:#00CC00">kan</SPAN><SUP>r</SUP></I></LI><LI><I>Growth Conditions</I>: Same as <I>E. coli</I>, LB media
</LI><LI><I>Ancestry</I>: <I>Acinetobacter baylyi</I> ADP1
</LI></UL></DD></DL><P>As this strain is already deficient in mismatch repair, Item 3 (Construction of strains with deficiencies in their mismatch repair pathways) has also been addressed by our choice of strains.
</P><H2><SPAN class="mw-headline" id="Optimization_of_synthetic_oligonucleotides_for_efficient_genome_modification">Optimization of synthetic oligonucleotides for efficient genome modification</SPAN></H2><P>In an effort to streamline the optimization experiments for Bs and Ac, we aim to build a &quot;universal&quot; construct (using <I>E. coli</I> as a cloning behicle) that will allow us to test the success and efficiency of targeted incorporation of mutagenic oligonucleotides at the lagging strand of the replication fork. 
</P><P>This construct will require the following parts to be usable in both Bs and Ac:
</P><UL><LI>An active <I>lacZ</I> gene, to be inactivated by a mutagenic oligo to yield white colonies on Xgal/IPTG plates
</LI><LI>A promoter for <I>lacZ</I> capable of high expression in both Gram-negative (Ac) and Gram-positive (Bs) bacteria
</LI><LI>A defective copy of a gene conferring antibiotic resistance (plus its native promoter), to be corrected by a mutagenic oligo to yield colonies resistant to the antibiotic of choice (Chloramphenicol)
</LI><LI>An active (unique) antibiotic resistance gene with its own promoter, which will be used to select colonies (for Bs, Ac, and Ec) that have successfully incorporated the construct (we chose Tetracycline)
</LI><LI>An origin site recognized by the replication machinery for all three species
</LI><LI>Transcription terminators flanking the origin of replication to prevent RNA read-through to the plasmid origin part
</LI><LI>Unique restriction sites at appropriate locations for mobility of the construct and some of its elements
</LI></UL><P>The broad-host range expression vector <B>pBAV1K-T5</B> (as described by Bryksin and Matsumura, 2010) has been selected as the backbone of our construct, as the authors were able to obtain high copy numbers and high levels of expression in all three of the species we will be working with. Furthermore, this plasmid contains an IPTG-inducible T5 promoter within a BioBrick multiple cloning site, an optimized ori site flanked by the T1 and t0 terminators, and unique restriction sites for the incorporation of selectable markers driven by their own promoters.
</P><P>Our first priority upon receiving pBAV1K-T5-<I>gfp</I> from Addgene will be to verify that the plasmid can be incorporated (and can replicate) in Bs, Ac, and Ec. 
</P><H3><SPAN class="mw-headline" id="lacZ_reporter"><I>lacZ</I> reporter</SPAN></H3><H3><SPAN class="mw-headline" id="Defective_selectable_marker">Defective selectable marker</SPAN></H3><H3><SPAN class="mw-headline" id="Active_selectable_marker">Active selectable marker</SPAN></H3><H3><SPAN class="mw-headline" id="Miscellaneous_plasmid_pieces:_ori_elements.2C_terminators.2C_restriction_sites">Miscellaneous plasmid pieces: <I>ori</I> elements, terminators, restriction sites</SPAN></H3><H2><SPAN class="mw-headline" id="Construction_of_strains_with_deficiencies_in_their_mismatch_repair_.28MMR.29_pathways">Construction of strains with deficiencies in their mismatch repair (MMR) pathways</SPAN></H2><P><I>See Item 1 (Choosing an industrially-relevant strain) regarding the MMR-deficient </I>ÎMutSL<I> strains chosen for our experimentation.</I></P><H2><SPAN class="mw-headline" id="Maintenance_of_bacterial_samples">Maintenance of bacterial samples</SPAN></H2><P>As with the samples we have already received, the strains arriving imminently (Bs: 168 and PY79 for controls, PERM739; Ec: DH5Î± containing pBAV1K-T5-<I>gfp</I>) will be inoculated in O/N cultures for glycerol stocks.
</P><H2><SPAN class="mw-headline" id="Verification_of_the_universal_replication_of_pBAV1K-T5-gfp_in_Ac.2C_Bs.2C_and_Ec">Verification of the universal replication of pBAV1K-T5-<I>gfp</I> in Ac, Bs, and Ec</SPAN></H2><P>Upon receiving pBAV1k-T5-<I>gfp</I> (Bryksin and Matsumura) from Addgene, we will test the unaltered expression vector for transformation in electrocompetent cells of the following strains:
</P><UL><LI>EcNR2 (<I>ÎmutS, Î» red<SUP>+</SUP></I>)
</LI><LI>Bs 1A833 (<I>mutSL::<SPAN style="color:#660066">Spc</SPAN><SUP>r</SUP></I>)
</LI><LI>Bs 1A40 (<I>lys-3 trpC2 metA10</I>)
</LI><LI>ADP1 WT
</LI></UL><H2><SPAN class="mw-headline" id="Transformation_of_B._subtilis_and_Acinetobacter_baylyi_with_pBAV1K-based_construct">Transformation of <I>B. subtilis</I> and <I>Acinetobacter baylyi</I> with pBAV1K-based construct</SPAN></H2><H2><SPAN class="mw-headline" id="Primer_Design">Primer Design</SPAN></H2><P>Using Geneious in conjunction with sequences for the various components (obtained from various sources), we designed primers
</P><H3><SPAN class="mw-headline" id="cT5_.28normal.29">cT5 (normal)</SPAN></H3><UL><LI>Primers have been designed according to overhang specifications from Prof. Isaacs
</LI></UL><DL><DD><UL><LI>Tm overhang: 60-65 <SUP>o</SUP>C
</LI><LI>Tm annealing: 55-60 <SUP>o</SUP>C
</LI><LI>Low GC count or AT percentage
</LI></UL></DD></DL><UL><LI>Primers have been checked for secondary structures
</LI><LI>Some primers worrisome forÂ %GC
</LI><LI>Primers arrived (07/06/12) from Keck
</LI></UL><P>(Redesigned, see below.)
</P><UL><LI>Primers for restriction enzyme ends:
</LI></UL><DL><DD><UL><LI>Primers have been checked for secondary structures
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm (Tm overhang/Tm anneal) </TH><TH> Length (bp) </TH><TH> Sequence (overCUTSITEanneal)
</TH></TR><TR><TD> pAf </TD><TD> 66.64 (25.23/58.79) </TD><TD> 30 (10/20) </TD><TD> 5'-tgacTCTAGAtttgctttgtgagcggataa-3' (XbaI)
</TD></TR><TR><TD> pAr </TD><TD> 69.08 (30.28/55.61) </TD><TD> 25 (10/15) </TD><TD> 5'-gtaaCTGCAGgagagcgttcaccga-3' (PstI)
</TD></TR></TBODY></TABLE><UL><LI><B>New Primer List (19Jul12):</B></LI></UL><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm (Tm overhang/Tm anneal) </TH><TH> Length (bp) </TH><TH> Sequence (overhangANNEAL)
</TH></TR><TR><TD> pBf </TD><TD> 74.16 (68.90/56.06) </TD><TD> 66 (46/20) </TD><TD> 5'-tgactctagatttgctttgtgagcggataacaattataatagattcAATTGTGAGCGGATAACAAT-3'
</TD></TR><TR><TD> pBr </TD><TD> 79.55 (75.05/56.42) </TD><TD> 60 (44/16) </TD><TD> 5'-gtaactgcaggagagcgttcaccgacaaacaacagataaaacgaAAGGCCCAGTCTTTCG-3'
</TD></TR><TR><TD> p2f_2 </TD><TD> 74.20 (69.30/55.65) </TD><TD> 66 (46/20) </TD><TD> 5'-aattgtgagcggataacaattactagagaaagaggagaaatactagATGACCATGATTACGGATTC-3'
</TD></TR><TR><TD> p2r_2 </TD><TD> 72.60 (63.89/55.41) </TD><TD> 50 (30/20) </TD><TD> 5'-tctccatacctcatcactcctcaatattacTTATTTTTGACACCAGACCA-3'
</TD></TR><TR><TD> p3f_2 </TD><TD> 70.57 (62.42/56.57) </TD><TD> 54 (32/22) </TD><TD> 5'-caaaaataagtaatattgaggagtgatgaggtATGGAGAAAAAAATCACTGGAT-3'
</TD></TR><TR><TD> p3r </TD><TD> 72.61 (62.18/58.07) </TD><TD> 50 (37/13) </TD><TD> 5'-ttgttagatttcatatattctttctcctctttcttaaTTACGCCCCGCCC-3'
</TD></TR><TR><TD> p4f </TD><TD> 71.04 (62.15/56.45) </TD><TD> 50 (30/20) </TD><TD> 5'-ggcgtaattaagaaagaggagaaagaatatATGAAATCTAACAATGCGCT-3'
</TD></TR><TR><TD> p4r_2 </TD><TD> 81.11 (76.12/57.88) </TD><TD> 59 (44/15) </TD><TD> 5'-aaggcccagtctttcgactgagcctttcgttttatttgatgcctTCAGGTCGAGGTGGC-3'
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Sequencing_Primers_.28ordered_24_Jul_2012.29">Sequencing Primers (ordered 24 Jul 2012)</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm </TH><TH> Length(bp) </TH><TH>Â %GC </TH><TH> Sequence
</TH></TR><TR><TD> p2f_seq </TD><TD> 61.86 </TD><TD> 27 </TD><TD> 40% </TD><TD> 5'-GCGGATAACAATTACTAGAGAAAGAGG-3'
</TD></TR><TR><TD> p2r_seq </TD><TD> 59.45 </TD><TD> 21 </TD><TD> 52% </TD><TD> 5'-CTCCATACCTCATCACTCCTC-3'
</TD></TR><TR><TD> p3f_seq </TD><TD> 56.14 </TD><TD> 22 </TD><TD> 41% </TD><TD> 5'-GTAATATTGAGGAGTGATGAGG-3'
</TD></TR><TR><TD> p3r_seq </TD><TD> 61.76 </TD><TD> 22 </TD><TD> 50% </TD><TD> 5'-CCTCTTTCTTAATTACGCCCCG-3'
</TD></TR><TR><TD> p4f_seq </TD><TD> 60.35 </TD><TD> 25 </TD><TD> 40% </TD><TD> 5'-GGCGTAATTAAGAAAGAGGAGAAAG-3'
</TD></TR><TR><TD> p4r_seq </TD><TD> 64.01 </TD><TD> 27 </TD><TD> 40% </TD><TD> 5'-GACTGAGCCTTTCGTTTTATTTGATGC-3'
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="Sequencing_Primers_II_-_full_cassette_sequencing">Sequencing Primers II - full cassette sequencing</SPAN></H4><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm </TH><TH> Length(bp) </TH><TH>Â %GC </TH><TH> Sequence
</TH></TR><TR><TD> pCas_1F </TD><TD> 58.97 </TD><TD> 22 </TD><TD> 45.5% </TD><TD> 5'-GGCGTATAACATAGTATCGACG-3'
</TD></TR><TR><TD> pCas_1R </TD><TD> 60.73 </TD><TD> 20 </TD><TD> 50% </TD><TD> 5'-GTTGCACCACAGATGAAACG-3'
</TD></TR><TR><TD> pCas_2F </TD><TD> 59.53 </TD><TD> 20 </TD><TD> 50% </TD><TD> 5'-GTTTGTTCCCACGGAGAATC-3'
</TD></TR><TR><TD> pCas_2R </TD><TD> 62.59 </TD><TD> 21 </TD><TD> 52.4% </TD><TD> 5'-CAGGCTTCTGCTTCAATCAGC-3'
</TD></TR><TR><TD> pCas_3F </TD><TD> 61.85 </TD><TD> 20 </TD><TD> 55% </TD><TD> 5'-CGATGAGCGTGGTGGTTATG-3'
</TD></TR><TR><TD> pCas_3R </TD><TD> 59.53 </TD><TD> 21 </TD><TD> 47.6% </TD><TD> 5'-GATCGACAGATTTGATCCAGC-3'
</TD></TR><TR><TD> pCas_4F </TD><TD> 61.22 </TD><TD> 21 </TD><TD> 52.4% </TD><TD> 5'-GATCGTAATCACCCGAGTGTG-3'
</TD></TR><TR><TD> pCas_4R </TD><TD> 64.40 </TD><TD> 20 </TD><TD> 60% </TD><TD> 5'-CAGAGGCACTTCACCGCTTG-3'
</TD></TR><TR><TD> pCas_5F </TD><TD> 63.59</TD><TD> 21 </TD><TD> 52.4% </TD><TD> 5'-GAAGCAAAACACCAGCAGCAG-3'
</TD></TR><TR><TD> pCas_5R </TD><TD> 64.88 </TD><TD> 21 </TD><TD> 57.1% </TD><TD> 5'-CAGCAAGTGTATCTGCCGTGC-3'
</TD></TR><TR><TD> pCas_6F </TD><TD> 62.87 </TD><TD> 21 </TD><TD> 52.4% </TD><TD> 5'-CATTGGCGTAAGTGAAGCGAC-3'
</TD></TR><TR><TD> pCas_6R </TD><TD> 61.44 </TD><TD> 21 </TD><TD> 52.4% </TD><TD> 5'-GACCAACTGGTAATGGTAGCG-3'
</TD></TR><TR><TD> pCas_7F </TD><TD> 62.49 </TD><TD> 24 </TD><TD> 45.8% </TD><TD> 5'-GAAGAAGGCACATGGCTGAATATC-3'
</TD></TR><TR><TD> pCas_7R </TD><TD> 60.31 </TD><TD> 22 </TD><TD> 45.5% </TD><TD> 5'-GCGAAGAAGTTGTCCATATTGG-3'
</TD></TR><TR><TD> pCas_8F </TD><TD> 62.94 </TD><TD> 20 </TD><TD> 55% </TD><TD> 5'-GCAAGATGTGGCGTGTTACG-3'
</TD></TR><TR><TD> pCas_8R </TD><TD> 60.52 </TD><TD> 20 </TD><TD> 55% </TD><TD> 5'-GGTGATGTCGGCGATATAGG-3'
</TD></TR><TR><TD> pCas_9F </TD><TD> 63.33 </TD><TD> 23 </TD><TD> 52.2% </TD><TD> 5'-CTTCGCTACTTGGAGCCACTATC-3'
</TD></TR><TR><TD> pCas_9R </TD><TD> 64.43 </TD><TD> 20 </TD><TD> 60% </TD><TD> 5'-CGAACGCCAGCAAGACGTAG-3'
</TD></TR><TR><TD> pCas_10F </TD><TD> 62.89 </TD><TD> 22 </TD><TD> 50% </TD><TD> 5'-GCTTGCGGTATTCGGAATCTTG-3'
</TD></TR><TR><TD> pCas_10R </TD><TD> 60.89 </TD><TD> 20 </TD><TD> 55% </TD><TD> 5'-GGCGGATTTGTCCTACTCAG-3'
</TD></TR></TBODY></TABLE><H4><SPAN class="mw-headline" id="ADP1_Integration_Primers">ADP1 Integration Primers</SPAN></H4><P>Figure from de Berardinis et al 2008 showing locations of primers. Kan cassette in this case will be replaced with either the cT5 cassette (cas) or Zeocin cassette (zeo).
</P><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm (Tm overhang/anneal) </TH><TH> Length (bp) </TH><TH> Sequence (overhangANNEAL)
</TH></TR><TR><TD> P3 </TD><TD> 57.51 </TD><TD> 22 </TD><TD> 5'-GTTCGTTATTTCCTCACTCATC-3'
</TD></TR><TR><TD> P4_cas </TD><TD> 75.08 (63.97/61.78) </TD><TD> 47 (25/22) </TD><TD> 5'-cgctcacaaagcaaatctagagtcaATGTTGGTGTTCCAGCTTTACC-3'
</TD></TR><TR><TD> P4_zeo </TD><TD> 75.10 (62.85/61.78) </TD><TD> 47 (25/22) </TD><TD> 5'-atgccgatgattaattgtcaacaccATGTTGGTGTTCCAGCTTTACC-3'
</TD></TR><TR><TD> P5_cas </TD><TD> 77.52 (69.08/58.19) </TD><TD> 44 (25/19) </TD><TD> 5'-tcggtgaacgctctcctgcagttacTTACGCTCAAACAGTCGAG-3'
</TD></TR><TR><TD> P5_zeo </TD><TD> 75.04 (65.49/58.19) </TD><TD> 44 (25/19) </TD><TD> 5'-gctcgaaggctttaatttgcaagctTTACGCTCAAACAGTCGAG-3'
</TD></TR><TR><TD> P6 </TD><TD> 61.50 </TD><TD> 22 </TD><TD> 5'-ATGATGTTGTTTGGCGGTTTTG-3'
</TD></TR><TR><TD> Z.ab-F </TD><TD> 61.15 </TD><TD> 23 </TD><TD> 5'-GGTGTTGACAATTAATCATCGGC-3'
</TD></TR><TR><TD> Z.ab-R</TD><TD> 61.04 </TD><TD> 22 </TD><TD> 5'-AGCTTGCAAATTAAAGCCTTCG-3'
</TD></TR></TBODY></TABLE><P>Z.ab F and R are to amplify the Zeocin cassette.
</P><P>Potential problems: According to the IDT oligo analyzer, there are some unfortunate secondary structures/self-dimerization potentials for the longer primers, but I am not sure how to avoid this.
</P><H3><SPAN class="mw-headline" id="Homolog_Library_Primers">Homolog Library Primers</SPAN></H3><P>27 total primers:
</P><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm (Tm overhang/Tm anneal) </TH><TH> Length (bp) </TH><TH> Sequence (overhangANNEAL)
</TH></TR><TR><TD> pT5_F </TD><TD> 76.37 </TD><TD> 92 </TD><TD> 5'-CTAGTCTAGATTTGCTTTGTGAGCGGATAACAATTATAATAGATTCAATTGTGAGCGGATAACAATTACTAGAGAAAGAGGAGAAAGGTACC-3'
</TD></TR><TR><TD> pT5_R </TD><TD> 76.37 </TD><TD> 92 </TD><TD> 5'-GGTACCTTTCTCCTCTTTCTCTAGTAATTGTTATCCGCTCACAATTGAATCTATTATAATTGTTATCCGCTCACAAAGCAAATCTAGACTAG-3'
</TD></TR><TR><TD> pUni_F </TD><TD> 66.89 </TD><TD> 32 </TD><TD> 5'-CTAGTCTAGATTTGCTTTGTGAGCGGATAACA-3'
</TD></TR><TR><TD> pUni_R </TD><TD> 64.52 </TD><TD> 32 </TD><TD> 5'-GGTACCTTTCTCCTCTTTCTCTAGTAATTGTT-3'
</TD></TR><TR><TD> p40_F </TD><TD> 72.13 (64.83/55.53) </TD><TD> 56 (35/21) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGGCAACTAAAAAACAAGAG-3'
</TD></TR><TR><TD> p40_R </TD><TD> 66.39 (25.28/58.42) </TD><TD> 32 (10/22) </TD><TD> 5'-gcttactagtCTATTCATTTGTTTCCCCTCCT-3'
</TD></TR><TR><TD> p41_F </TD><TD> 73.76 (64.83/58.35) </TD><TD> 54 (35/19) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGGCTGAAAATGCTGTCA-3'
</TD></TR><TR><TD> p41_R* </TD><TD> 68.49 (25.28/59.97) </TD><TD> 25 (10/15) </TD><TD> 5'-gcttactagtTCATGCGTTGGGCCC-3'
</TD></TR><TR><TD> p42_F </TD><TD> 72.44 (64.83/55.76) </TD><TD> 55 (35/20) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGTCAACTAACGACGAATT-3'
</TD></TR><TR><TD> p42_R </TD><TD> 66.56 (25.28/59.83) </TD><TD> 33 (10/23) </TD><TD> 5'-gcttactagtTTAGATCATTGACCCTTGAACCT-3'
</TD></TR><TR><TD> p43_F </TD><TD> 72.34 (64.83/56.43) </TD><TD> 57 (35/22) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGGCAAATGAATTAGGAATCT-3'
</TD></TR><TR><TD> p43_R </TD><TD> 66.53 (25.28/58.73) </TD><TD> 30 (10/20) </TD><TD> 5'-gcttactagtTTAGAATCCCTCCAAAGGCT-3'
</TD></TR><TR><TD> p44_F </TD><TD> 72.07 (64.83/57.21) </TD><TD> 59 (35/24) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGGGGAATGAATTAATAGTAAGC-3'
</TD></TR><TR><TD> p44_R </TD><TD> 67.14 (25.54/58.91) </TD><TD> 28 (10/18) </TD><TD> 5'-gcAtactagtTCAGAAAGGGTAAGGGCC-3'
</TD></TR><TR><TD> p45_F </TD><TD> 72.09 (64.83/55.29) </TD><TD> 56 (35/21) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGGAAAAACCAAAGCTAATC-3'
</TD></TR><TR><TD> p45_R </TD><TD> 66.46 (25.28/58.33) </TD><TD> 31 (10/21) </TD><TD> 5'-gcttactagtCTAAGAAGCTAAAGGCTGTGT-3'
</TD></TR><TR><TD> p46_F </TD><TD> 73.48 (64.83/57.67) </TD><TD> 54 (35/19) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGAGCACAGCAGTACAAA-3'
</TD></TR><TR><TD> p46_R* </TD><TD> 66.50 (25.28/61.33) </TD><TD> 40 (10/30) </TD><TD> 5'-gcttactagtTTATGATGCCTTTTTCCTTAAAAAATCAGA-3'
</TD></TR><TR><TD> p47_F </TD><TD> 72.72 (64.83/55.62) </TD><TD> 54 (35/19) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGGCAACACAAAAAGTTG-3'
</TD></TR><TR><TD> p47_R* </TD><TD> 66.21 (25.28/60.97) </TD><TD> 40 (10/30) </TD><TD> 5'-gcttactagtTTAAGCCTTATCCTGATTAGTTTCTTTTTC-3'
</TD></TR><TR><TD> p48_F </TD><TD> 70.19 (64.83/55.02) </TD><TD> 63 (35/28) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGACTGAAAATAATAAATTACAAACTA-3'
</TD></TR><TR><TD> p48_R </TD><TD> 66.05 (25.28/58.60) </TD><TD> 31 (10/21) </TD><TD> 5'-gcttactagtTTAAAATGGCTCTTCTTCGCT-3'
</TD></TR><TR><TD> p49_F </TD><TD> 73.42 (64.83/57.24) </TD><TD> 54 (35/19) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGACTAAGCAACCACCAA-3'
</TD></TR><TR><TD> p49_R* </TD><TD> 66.40 (25.28/61.07) </TD><TD> 39 (10/29) </TD><TD> 5'-gcttactagtTTATTCCTCTGAATTATCGATTACACTGT-3'
</TD></TR><TR><TD> p50_F </TD><TD> 73.68 (64.83/56.95) </TD><TD> 52 (35/17) </TD><TD> 5'-gataacaattactagagaaagaggagaaaggtaccATGAGTACTGCACTCGC-3'
</TD></TR><TR><TD> p50_R </TD><TD> 67.27 (25.28/58.80) </TD><TD> 27 (10/17) </TD><TD> 5'-gcttactagtTCATGCTGCCACCTTCT-3'
</TD></TR><TR><TD> pExo_R </TD><TD> 66.38 (25.28/57.10) </TD><TD> 39 (10/16) </TD><TD> 5'-gcAtactagtTCATCGCCATTGCTCC-3'
</TD></TR></TBODY></TABLE><DL><DD><UL><LI> *Possible temperature issue
</LI></UL></DD></DL><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm </TH><TH> Length (bp) </TH><TH> Sequence
</TH></TR><TR><TD> pTest_F </TD><TD> 63.88 </TD><TD> 35 </TD><TD> 5'-CGGTGATATTCTCATTTTAGCCATTTATTATTTCC-3'
</TD></TR><TR><TD> pTest_R </TD><TD> 65.48 </TD><TD> 28 </TD><TD> 5'-GAGCCTTTCGTTTTATTTGATGCCTCTG-3'
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Other_Primers">Other Primers</SPAN></H3><TABLE class="wikitable"><TBODY><TR><TH> Name </TH><TH> Tm (Tm anneal) </TH><TH> Length (bp) </TH><TH> Sequence (overhangANNEAL)
</TH></TR><TR><TD> Zeo A (F) </TD><TD> 69 (57.13) </TD><TD> 35 (15/20) </TD><TD> 5'-CTATCAACAGGAGTCAGCTTGCAAATTAAAGCCTT-3'
</TD></TR><TR><TD> Zeo B (R) </TD><TD> 66.26 (61.15) </TD><TD> 38 (15/23) </TD><TD> 5'-ATCCTATATTTAAAAGGTGTTGACAATTAATCATCGGC-3'
</TD></TR><TR><TD> Zeo A' (R) </TD><TD> 68.38 (56.42) </TD><TD> 35 (15/20) </TD><TD> 5'-TTTAATTTGCAAGCTGACTCCTGTTGATAGATCCA-3'
</TD></TR><TR><TD> Zeo B' (F) </TD><TD> 63.07 (50.16) </TD><TD> 39 (15/24) </TD><TD> 5'-TTAATTGTCAACACCTTTTAAATATAGGATTTCATTTTC-3'
</TD></TR><TR><TD> BH mod F </TD><TD> 67.11 </TD><TD> 27 </TD><TD> 5'-gacgtcTTTGCTTTGTGAGCGGATAAC-3' (AatII)
</TD></TR><TR><TD> BH mod R </TD><TD> 69.62 </TD><TD> 25 </TD><TD> 5'-gacgtcGAGAGCGTTCACCGACAAA-3' (AatII)
</TD></TR><TR><TD> LacZ KO </TD><TD> - </TD><TD> 90 </TD><TD> 5'-TCAGGATATGTGGCGGATGAGCGGCATTTTCCGTGACGTCTCGTAGCTGCATAAACCGACTACACAAATCAGCGATTTCCATGTTGCCAC-3'
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Growth_curves_for_Ac.2C_Bs.2C_and_Ec">Growth curves for Ac, Bs, and Ec</SPAN></H3><P>While troubleshooting the unsuccessful transformation of pBAV1K-T5 in B. subtilis, we thought that it would be beneficial to obtain growth curves for the strains that we were working with. Unlike WT A. baylyi, which is constitutively competent at the stationary phase (12-16 hrs), B. subtilis requires more specific conditions for natural competence, namely starvation and stationary phase growth. Since each of the protocols that we looked at called for slightly different OD600 values â each set of which were largely for the WT strain (168) â we thought that characterizing the growth of all the B. subtilis strains would provide more insight into the optimal conditions for transformation.
The growth curves here were generated by growing starter cultures (n=3) of each of the strains in a 24-well plate that was kept incubating in a plate reader at 34â°C with shaking for 14.5 hours. The OD600 values were recorded automatically every 5 minutes, but the plots show the points at half-hour increments for clarity. What we found was that each of the different strains of B. subtilis showed notable differences in the mid- to late-exponential phase of growth, a fact that can be used in further efforts to transform our cassette-containing vector into B. subtilis via natural competence.
</P><H2><SPAN class="mw-headline" id="Effect_of_electroporation_on_competency_in_Ab_and_Bs">Effect of electroporation on competency in <I>Ab</I> and <I>Bs</I></SPAN></H2><P>To further test competency, we utilized electroporation to make Acinetobacter baylyi and Bacillus subtilis electrocompetent. A.baylyi strain ADP1 (wild type), B. subtilis strains BS168, BS PERM739, BS PY79, BS 1A833, and E.coli strain EcNR2 (E.coli strains implemented as a control due to known success with electroporation). Cells were inoculated from frozen glycerol aliquots and grown O/N in 10mL of LB media until OD600 reading was between 0.4 and 0.8 (midlog phase). Cells were prepared for electroporation by washing cells two times in ice-cold Nuclease-free water and by suspending in Nuclease-free water as the electroporation buffer (exact protocols available {see end of document}). Kanomycin resistance was introduced into the buffer to test successful uptake of plasmids. Electroporation conditions using BioRad Gene Pulser were as follows: 1.8kV, 200Î©, 25uF. Results are as follows:
</P><H2><SPAN class="mw-headline" id="Generating_a_beta_protein_library">Generating a beta protein library</SPAN></H2><P>We have succeeded in isolating DNA sequences of 16 different lambda red system/beta homolog variants as well as integrating a strong T4 promoter  upstream of each. As of right now, at least 6 of them have been integrated into a plasmid compatible with ADP1 and B. subtilis, while identification and confirmation of others is pending.
</P><UL><LI>To each homolog isolated, we added the T5 promoter (pBAV1K) and XbaI and SpeI sites
</LI><LI>We cloned into pBAV1K  and sequenced:
</LI><LI>bet
</LI><LI>recT
</LI><LI>plu2935
</LI><LI>orfC
</LI><LI>s065-exo
</LI><LI>recT-exo
</LI><LI>Other homologs in progress
</LI></UL><UL><LI>Re-cloned into pSB1C3 (Biobrick)
</LI><LI>bet - BBa_K810000
</LI></UL><H1><SPAN class="mw-headline" id="Helpful_Links">Helpful Links</SPAN></H1><H2><SPAN class="mw-headline" id="On_MAGE_and_its_Components">On MAGE and its Components</SPAN></H2><UL><LI>The original MAGE <A href="http://www.nature.com/nature/journal/v460/n7257/pdf/nature08187.pdf" class="external text" rel="nofollow">paper</A> (Wang <I>et al.</I>, 2009) with <A href="http://www.nature.com/nature/journal/v460/n7257/extref/nature08187-s1.pdf" class="external text" rel="nofollow">supplementary information</A> outlining the optimization of oligos for MAGE in EcNR2 (<I>mutS<SUP>-</SUP>, Î»-red<SUP>+</SUP></I>).
</LI><LI>Harvard's 2011 iGEM team's simplified <A href="https://2011.igem.org/Team:Harvard/Technology/MAGE" class="external text" rel="nofollow">outline</A> on MAGE.
</LI><LI>A <A href="http://www.pnas.org/content/105/5/1626.full.pdf?with-ds=yes" class="external text" rel="nofollow">paper</A> (Datta <I>et al.</I>) analyzing Î²-homologs from various Gm<SUP>-</SUP> and Gm<SUP>+</SUP> bacteria and their phages for recombinase activity.
</LI></UL><H2><SPAN class="mw-headline" id="On_the_Strains_Chosen_for_Experimentation">On the Strains Chosen for Experimentation</SPAN></H2><H3><SPAN class="mw-headline" id="Acinetobacter_baylyi_ADP1_.28formerly_BD413_before_1995.29"><I>Acinetobacter baylyi</I> ADP1 (formerly BD413 before 1995)</SPAN></H3><UL><LI>A broad <A href="http://nar.oxfordjournals.org/content/32/19/5780.full.pdf" class="external text" rel="nofollow">overview</A> of ADP1 (Metzgar <I>et al.</I>, 2004) presenting it as an ideal model organism for genetic analysis and genome engineering.
</LI><LI>An <A href="http://www.ajol.info/index.php/ajb/article/view/14828" class="external text" rel="nofollow">overview</A> (El Haleem <I>et al.</I>, 2004) of the industrial/biotechnical applications of the <I>Acinetobacter</I> genus
</LI><LI>The <A href="http://www.ncbi.nlm.nih.gov/pubmed/15514110" class="external text" rel="nofollow">paper</A> concerning the ADP1 genome (Barbe <I>et al.</I>, 2004)
</LI></UL><DL><DD><UL><LI>The associated <A href="http://www.genoscope.cns.fr/cgi-bin/ggb/thesaurus/gbrowse/thesaurus/" class="external text" rel="nofollow">baylyi genome browser</A></LI></UL></DD></DL><UL><LI>The <A href="http://aem.asm.org/content/72/1/932" class="external text" rel="nofollow">classification</A> of ADP1 as a new species, <I>A. baylyi</I> (Vaneechoutte <I>et al.</I>, 2006)
</LI><LI>A <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2290942/" class="external text" rel="nofollow">paper</A> concerning the construction of a library of single gene mutants in <I>A. baylyi</I>, including <I>ÎmutS</I> (de Berardinis <I>et al.</I>, 2008)
</LI></UL><H3><SPAN class="mw-headline" id="Bacillus_subtilis_2"><I>Bacillus subtilis</I></SPAN></H3><UL><LI>A <A href="http://bacillus.genome.ad.jp/" class="external text" rel="nofollow">genome browser</A> for <I>B. subtilis</I></LI><LI>The <A href="http://www.bgsc.org" class="external text" rel="nofollow">website</A> for the Bacillus Genetic Stock Center (BGSC) in Ohio, where we obtained our MMR-deficient strains and their relevant ancestral strains (for controls). In addition to strains of Bs, the BGSC also has integration vectors and information on Bs alleles and their mutant phenotypes.
</LI><LI>A <A href="http://ac.els-cdn.com/S1097276501004026/1-s2.0-S1097276501004026-main.pdf?_tid=5fa060af740627d2245f82e23a94992d&amp;acdnat=1341514852_2c79bf86cc2b32cd987872207fe9e741" class="external text" rel="nofollow">paper</A> (Smith, Grossman, and Walker, 2001) providing background information on the construction of Bs strain 1A833 (<I>mutSL::<SPAN style="color:#660066">Spc</SPAN><SUP>r</SUP></I>).
</LI><LI>A <A href="http://jb.asm.org/content/186/19/6485.full.pdf" class="external text" rel="nofollow">paper</A> (Pedraza-Reyes and Yasbin, 2004) providing background information on the construction of Bs strain PERM739 (<I>ÎmutSL::<SPAN style="color:#FF3399">Neo</SPAN></I>).
</LI><LI>A historical <A href="http://jb.asm.org/content/190/21/6983.full.pdf" class="external text" rel="nofollow">overview</A> (Zeigler <I>et al.</I>, 2008) of the various laboratory strains of <I>B. subtilis</I> and their genetic and anecdotal backgrounds.
</LI><LI>A <A href="https://2008.igem.org/Team:Newcastle_University/Bacillus_subtilis" class="external text" rel="nofollow">protocol</A> from the 2008 Newcastle iGEM team on inducing competence in Bs via starvation media.
</LI><LI>A two-step protocol (<A href="/wiki/index.php?title=Special:Upload&amp;wpDestFile=Bacillus_subtilis_competent_cell_tf.pdf" class="new" title="Bacillus subtilis competent cell tf.pdf">Media:Bacillus_subtilis_competent_cell_tf.pdf</A>) based on Harwood <I>et al.'</I>s &quot;Molecular Biological Methods for Bacillus&quot; that allows prior preparation of competent cells stored as a glycerol stock
</LI></UL><H2><SPAN class="mw-headline" id="On_Expression.2FIntegration_Vectors_and_their_Components">On Expression/Integration Vectors and their Components</SPAN></H2><UL><LI>A <A href="http://www.plosone.org/article/info%3Adoi%2F10.1371%2Fjournal.pone.0013244" class="external text" rel="nofollow">paper</A> (Bryksin and Matsumura, 2010) describing the design of a &quot;universal&quot; plasmid vector capable of replicating efficiently in various Gm<SUP>-</SUP> and Gm<SUP>+</SUP> bacteria.
</LI><LI>A <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3255645/pdf/zam280.pdf" class="external text" rel="nofollow">paper</A> (Murin <I>et al.</I>, 2011) outlining the construction and utility of inducible expression vectors and integration vectors in ADP1
</LI><LI>A <A href="http://www.sciencedirect.com/science/article/pii/104659289290012L" class="external text" rel="nofollow">paper</A> (Trumble <I>et al.</I>, 1992) describing the design and efficiency of a Ec/Bs shuttle plasmid with a synthetic promoter optimized for both species.
</LI><LI>Addgene's <A href="http://www.addgene.org/26702/" class="external text" rel="nofollow">description</A> on pBAV1K-T5-<I>gfp</I> (universal expression vector)
</LI><LI>Addgene's <A href="http://www.addgene.org/30505/" class="external text" rel="nofollow">description</A> on pIM1463 (ADP1 integration vector)
</LI></UL><H2><SPAN class="mw-headline" id="On_Various_Aspects_of_our_Oligo-Optimization_construct">On Various Aspects of our Oligo-Optimization construct</SPAN></H2><UL><LI>A <A href="http://www.sciencedirect.com/science/article/pii/0378111995006524" class="external text" rel="nofollow">paper</A> (GuÃ©rout-Fleury <I>et al.</I>, 1995) demonstrating the viability of several different antibiotic resistance cassettes in Bs.
</LI></UL><H2><SPAN class="mw-headline" id="On_Methods_and_Technical_Aspects_of_our_Project">On Methods and Technical Aspects of our Project</SPAN></H2><UL><LI><A href="http://arep.med.harvard.edu/kzhang/cgi-bin/myOligoTm.cgi" class="external text" rel="nofollow">Kun's Oligonucleotide Tm Calculator</A></LI><LI><A href="http://www.idtdna.com/analyzer/applications/oligoanalyzer/" class="external text" rel="nofollow">IDT Oligo Analyzer</A></LI><LI>A <A href="http://www.synbio.org.uk/dna-assembly/guidetogibsonassembly.html" class="external text" rel="nofollow">website</A> with links to info on Gibson Assembly
</LI></UL><H2><SPAN class="mw-headline" id="Confirmation_of_the_universality_of_the_pBAV1K-T5_vector">Confirmation of the universality of the pBAV1K-T5 vector</SPAN></H2><P>In order to determine whether the pBAV1K-T5 vector (Bryksin and Matsumura, 2010) would serve as an appropriate universal backbone for our cassette, we did a cross-species transformation experiment with the unaltered pBAV1K-T5-gfp plasmid. For E. coli, we used XL1-Blue cells that were made chemically competent, and for the other two species we used naturally competent cells. The success of transformation was gauged by selection on LB-Kan plates, with a concentration of 20 Î¼g/mL for B. subtilis, and 50 Î¼g/mL for Acinetobacter baylyi and E. coli.
</P><P>What we found was that the XL1-Blue and ADP1 WT cells were consistently transformed (i.e., growth when expected, selective colonies), while the four B. subtilis strains (168, PY79, PERM739, and 1A833) showed no indications of successful transformation despite several different attempts. While it is possible that pBAV1K-T5 simply does not replicate in high enough copy numbers in B. subtilis, it seems more likely that the problem lie somewhere in the transformation protocols. We initially used a two-step transformation protocol from Harwoodâs Molecular Biological Methods for Bacillus, and then tried electroporation, then a one-step transformation method also from Harwoodâs protocols. The results were consistently poor, with very little reproducibility for any positive results obtained. Upon correspondence with the authors, we learned that they too had trouble yielding consistent results for B. subtilis.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Yale/Project">http://2012.igem.org/Team:Yale/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Yale/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Yale/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Yale/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Yale/Project&amp;oldid=286756" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>