Team:METU HS Ankara/Results
Home
Team
Team
Collaborations
Project
Description
Design
Experiments
Proof of Concept
Demonstrate
Results
Notebook
Parts
Basic Parts
Composite Parts
Part Collection
Model
Attributions
Human Practice
Safety
Achievements
Gallery
Results
Overview
We used Team Harvard BioDesign 2015’s method for E. coli to bind to colon cancer cells by inserting the binding peptide RPMrel into the FimH protein and took their project to the next level by forcing the E. coli to produce butyrate upon binding. For the production of butyrate we used the enzyme But-CoAT which allowed for the direct conversion of Acetyl CoA into butyrate.
Achievements
We have succesfully cloned our genes of interest into pSB1C3.
We have confirmed that clones were obtained properly through the PCR amplification.
We have succesfully characterize the fusion of FimH together with GFP and ButCoaT together with RFP  and their compatibility.
We have obtained the Protein gel of previous year CAD1 gene that submitted in 2015.
We have optimized the  Co-culture protocol for our binding and treatment purposes and we have found that the optimal Co-culture time is 6 hours.
Harvard Bio Design’s Results
Team Harvard BioDesign 2015 had used a dot blot protocol in order to prove the specified binding of their cells to colon cancer cells. Their protocol involved dotting Caco-2 colorectal cancer cells, introducing their cancer-binding E. coli then staining the blots with certain antibodies to show binding. Their results, much like ours, were positive and indicated binding of the E. coli to colon cancer cells.
Figure 1. Binding of E.coli to colon cancer cells
Gel Results of submitted parts K2052014 and K202015
We have obtained our submitted parts as shown with gel images below:
K2052014
Gel image1: FimH + double terminator + arabinose promoter
In the first 4 lanes the gene is shown approximately 2100 base pairs and it is uncut. In the first 4 lanes after the second ladder the sequence is shown approximately at 3300 base pairs and it is single cut with EcoRI. With the single cut it’s been shown that the insert  is successfully in our vector.
K2052015
Gel Image 2: ButCoat and RBS + ButCoat
We have loaded uncut version of K2052015 next to EcoRI single digested one. In the third lane RBS ligated ButCoat (K2052018) was loaded with its single digested version. Digested ones gave sharp lane at 3600 bp as we expected.
K2052016
Gel Image 3: FimH+Double Terminator+ Arabinose Promoter+ RBS+ ButCoAT
Here, we have uncut and cut version of our whole construct. Since we have out of enzymes, we have digested them with NcoI. After digestion, we were expecting a lane at 1200 bp. Because it has 3 cut sites in the construct, we have extra lanes.
PCR Confirmation
Gel Image 4: PCR confirmation of Ligated Parts
The primers that we have designed bind and multiply the site when the insert and vector are ligated properly, forward binds to a region in vector and reverse binds to a region in insert and give a product at 781 bp for 1,2,3,4 which is K2052014 and 843 bp for 5 K2052018 ,6 which is K2052015.  Primers:
forward  5’-CGAAAAGCCAAAACCTGG-3’-18 bp
reverse  5’-GAAGCCTGCATAACGCGG-3’ -18 bp for 1,2,3,4
forward 5’ –TTGGAACCTCTTACGTGCCC-3’-20bp
reverse 5’- ATCAGTTTCGCCACGGTCTT-3’-20bp for 5,6
Single digestion and  confirmational PCR has been made to confirm that the vector and the insert unites properly.  As it can be seen in the first four lanes, PCR gave a hit at 781 bp. It has been confirmed that they united properly.
CHARACTERIZATION
We tried our experiments for expression vector, pet28a, however after hundreds of trials, we failed to get the expected results. So we decided to change our method and used our parts with psB1c3 backbone for characterization. By ligating GFP to the upstream of FimH and by ligating RFP to the downstream of ButCoAT, we have designed new constructs that we used for both characterization and experimental validation.
Our construct which includes GFP goes like this;
Promoter  / RBS / GFP(K608008) +  FimH / Terminator (K2052014)
We have ligated our construct(K2052014) with K608008. After ligation, we have obtained many colonies that we have confirm their proper ligation through PCR. Here you can see our results;
Ligated Parts Transformation Results
Figure 1: After ligating two construct we have  obtain colonies 2:1 (insert:vector) ratio, and transform them into E.coli BL21.
Confirmational PCR Result
Figure 2: Constitutive promoter-RBS-GFP- Fimh-Double terminator cloned into pSB1C3 and after ligation we have transformed into E.coli BL21. Colony PCR results here you can see; one primer stick on insert and another stick on vector. The expected  lenght of product is around 700bp.
CoCulture and Fluoresence Microscopy
Figure 3: Here, upper left picture shows the cells only and next to it we have taken picture of CaCo2 cells cocultured with Gfp tagged FimH. Bottom pictures were taken under the Fluoresence Microscope but we havent seen any illumination that we have expected. We thought that the fusion of these proteins can change the folding of each protein and resulted in no accumulation of bacteria on cancer cells and any  presence of  GFP illumination.
After we havent seen any GFP activity we prepared a new experimental set up for Flowcytometry to be sure whether our control group(only GFP producing bacteria) is working or not and then we wanted to summarize our findings.
Flowcytometry Result
Figure 4: After measuring GFP signals we have obtained these graphs. Here upper triplet stand for Mock(reference point) analysis that is the bacterial culture transformed with only FimH. Middle triplet stand  for only GFP expressed bacterial culture analysis which gave a peak that could be used as a control group. We have obtained the peaks in the FL1 channel of Fluorescence Microscope. The bottom triplet was GFP tagged FimH expressed bacterial group and as we have shown there is any signal depend on GFP activity.
Possible Outcomes of All Our Experimental Findings
After we did co-culture with CaCo-2 cells, we didn’t observe any clumps around cancer cells, but we observed our freely located bacteria without GFP illumination . We concluded from the results that because there wasn’t any ribosome binding site between GFP and FimH proteins,  GFP tagged FimH is unfunctionally formed and the fusion affected their folding
After we did co-culture with CaCo-2 cells, we didn’t observe any clumps around cancer cells, they were freely located but we observed illumination of GFP. We concluded from the results that because there wasn’t any ribosome binding site between GFP and FimH proteins, they fused together which resulted only FimH protein to be affected from this fusion and become unfunctional.
After we did co-culture with CaCo-2 cells, we observed clump formation around  only cancer cells but we didn’t observed illumination of GFP. We concluded from the results that because there wasn’t any ribosome binding site between GFP and FimH proteins, they fused together which resulted only GFP protein to be affected from this fusion and become unfunctional.
Our construct which includes RFG goes like this;
Promoter (J23119) +  RBS / ButCoAT (K2052015) +  RFP / Terminator (J23100)
Ligated Parts Transformation Results
Figure 5: After ligating ButcoaT and RFP, we have  obtain colonies 2:1 (insert:vector) ratio, and transform them into E.coli BL21.
Confirmational PCR Result
Figure 6: Constitutive promoter-RBS-ButCoaT-GFP-Double terminator cloned into pSB1C3 and after ligation we have transformed into E.coli BL21. Colony PCR results here you can see; one primer stick on insert and another stick on vector. The expected  lenght of product is around 850 bp.
Flowcytometry Result
After checking with confirmational PCR we have tried to validate RFP fluoresence experimenttaly through Fluorescence Microscope, Here you can see the analysis.
Figure 7: After measuring RFP signals with channel FL3 we have obtained these graphs. Here again upper triplet stand for Mock(reference point) analysis that is the bacterial culture transformed with only ButCoaT. Middle triplet stand  for only RFP expressed bacterial culture analysis which gave a peak that could be used as a control group. The bottom triplet was RFP tagged ButCoaT expressed bacterial group and as we have shown there is any signal depend on RFP activity.
Since ButCoAT was tagged with RFP, to test whether it is functional or not, we have used fluoresence microscopy technique to show its expression is properly done or not.However, because ButCoAT doesn’t have a stop codon it fused  with RFP and they both become unfunctional from this fusion.Therefore,  we couldn’t see any peak.
Experimentally Validation of Previous Device
CAD1 (cinnamyl alcohol dehydrogenase) enzyme that we were interested previous year was characterized with Fehling colorimetric assay. This year we wanted to show construct work with pSB1C3 vector on protein gel. After growing and obtaining colonies in Overnight culture, we have got optical density at 1.0 and using several techniques we have extracted protein and loaded on a SDS gel. Without induction, constitutive promoter that we have ligated previous year worked perfectly and give us a lane at 38kDA. The control group was the e. coli dh5a untransformed colonies. Here you can see image;
First well is ladder.
3rd lane of ladder from bottom stands for 25.
4th lane from bottom stands for 55kDa.
First well is loaded with empty dh5a and its lysate is just right next to it
3rd well is our first colony that we have successfully completed ligated into psb1c3
Here you can see 4 lane in-between 25 and 55kDa
Extra lane is our CAD1 enzyme denatured protein
To conclude, as you can see, the enzyme was not released in environment and we proved it in wells that we have loaded lysates next to precipitated ones.
Future Plans
After Pet28a expression vector clonning fail, we have tried to show characterization on iGEM cloning backbone with the use of reporters(GFP and RFP) and we couldnot prevent the formation of fusion proteins. However, we succesfully characterized the compatibility of those protein of interests when they expressied together with RFP and GFP.
Those finding show us that we need to seperate proteins with the use of RBS, for instance, to expressed proteins with their reporter properly and get expected results. Our first aim seperate these proteins from each other by constructing our biobricks depend on what we have found.
In parallel to these, as expression vector, we want to clone our constructs into PBlueScript to show the activity of bacteria when they properly induced.
REFERENCES
1. Harvard BioDesign (2015). Retrieved from: https://2015.igem.org/Team:Harvard_BioDesign
