<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Labjournal_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Labjournal/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Labjournal/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/July&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Labjournal/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Labjournal/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="width:720px; padding-left:20px; padding-right:20px; margin-top:60px;"><DIV id="buttonnav" style="height:30px; padding:10px 30px; margin-bottom:0px; margin-top:20px;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Prologue">Prologue</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/May">May</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/June">June</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/July">July</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/August">August</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/September">September</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/October">October</A></DIV></DIV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>July</H1><H3><SPAN class="mw-headline" id="Milestones">Milestones</SPAN></H3><UL><LI>Poster presentation at the congress ‘<A href="http://www.biotechnologie2020plus.de" class="external text" rel="nofollow">Next generation of biotechnological processes 2020+</A>’ in Berlin.
</LI><LI><A href="/Team:Bielefeld-Germany/Partners" title="Team:Bielefeld-Germany/Partners">Sponsoring</A> acquisition is nearly finished.
</LI><LI>Problems with cloning delay the progress in the laboratory.
</LI><LI>We have published our <A href="http://ekvv.uni-bielefeld.de/blog/uniaktuell/entry/mit_bakterien_batterie_strom_erzeugen" class="external text" rel="nofollow">first press release</A> with a great response. The press release was picked up worldwide.
</LI></UL><H3><SPAN class="mw-headline" id="Week_9">Week 9</SPAN></H3><H4><SPAN class="mw-headline" id="Organization">Organization</SPAN></H4><UL><LI>We’ve presented us at the congress ‘<A href="http://www.biotechnologie2020plus.de" class="external text" rel="nofollow">Next generation of biotechnological processes 2020+</A>’ in Berlin at June 27th, 2013. We met the expert (Dr. Falk Harnisch) on the topic of MFC and decided to participate in the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Human_Practice/Day_of_Synthetic_Biology" class="external text" rel="nofollow">'Day of Synthetic Biology in Germany'</A>.
</LI></UL><H4><SPAN class="mw-headline" id="MFC">MFC</SPAN></H4><UL><LI>Started designing 3D models of MFCs which can be printed out by a 3D-printer
</LI><LI>Conducted experiments with <I>saccharomyces cerevisiae</I> and <I>e. coli</I> and recorded power curves for both
</LI><LI>Ordered plastic parts for the new fuel cell model
</LI></UL><H4><SPAN class="mw-headline" id="Cytochromes">Cytochromes</SPAN></H4><UL><LI>Design and order of new primers
</LI><LI>The <I>mtrCAB</I> fragment has two illegal <I>Pst</I>I restriction sites at 1793 bp and 2078 bp, so specific primers to remove them had to be designed. One base in each restriction site was replaced by respective primer overlaps without affecting the coding triplet. The three different fragments will be ligated back together and into the pSB1C3 shipping plasmid via Gibson Assembly.
</LI><LI><I>mtrCAB</I>_Frag1_rev (30 nt): GACCTGTTTTATCTGTTGCAGCGGCAGTAT
</LI><LI><I>mtrCAB</I>_Frag2_fwd (30 nt): ATACTGCCGCTGCAACAGATAAAACAGGTC    
</LI><LI><I>mtrCAB</I>_Frag2_rev (30 nt): GGTGTGACGACTAATGCCATAATTGCAGAC   
</LI><LI><I>mtrCAB</I>_Frag3_fwd (30 nt): GTCTGCAATTATGGCATTAGTCGTCACACC   
</LI></UL><H4><SPAN class="mw-headline" id="Nanowires">Nanowires</SPAN></H4><UL><LI>Primer design for isolation of gene-cluster GSU 1776-1784 from <I>Geobacter sulfurreducens</I>, containing overlaps for BioBrick Prefix and Suffix:
</LI><LI>Forward GSU 1776-1784 (41):GAATTCGCGGCCGCTTCTAGATGCTGAAACGTTTTCGTAAC
</LI><LI>Reverse GSU 1776-1784 (44): CTGCAGCGGCCGCTACTAGTATCATCCTACAGTGCCGGCAATTT
</LI></UL><UL><LI>Successful genome isolation from <I>Geobacter sulfurreducens</I> strain, using the GenEluteTM Bacterial Genomic DNA-Kit to prevent vigorous fragmentation of the DNA, which arises out of the standard genome isolation method with a ribolyse-step.
</LI><LI>Successful PCR with forward and reverse primer GSU Promoter 1496-1505 and GSU 1496-1505 on the appropriate gene-cluster of <I>Geobacter sulfurreducens</I>, using the PCR Master Mix, described in Table 3 (June) (nanowires), with adjusted template volume and the standard PCR program for amplification of <I>Geobacter sulfurreducens</I> gene cluster (Table 4, June).
</LI><LI>GSU Promoter 1496-1505 and GSU 1496-1505 PCR products were isolated and purified by gel extraction.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 1:</B> Agarose gel from PCR on the gene-clusters GSU Promoter 1496-1505 (bands 2-5) and GSU 1496-1505 (bands 7-10) of <I>Geobacter sulfurreducens</I> strain. Ladder: GeneRuler™ 1 kb DNA Ladder from Thermo Scientific.</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Week_10">Week 10</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_2">Organization</SPAN></H4><UL><LI>We can present new sponsors of iGEM-Team Bielefeld: Stockmeier, Baxter, Promega, BIO.NRW and AppliChem will support our team.
</LI></UL><H4><SPAN class="mw-headline" id="MFC_2">MFC</SPAN></H4><UL><LI>Received plastic parts and built the first of our second generation fuel cells
</LI></UL><H4><SPAN class="mw-headline" id="Mediators">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI>Further work on <I>gldA</I> BioBrick is needed: Plasmid <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Colony_PCR" title="Team:Bielefeld-Germany/Labjournal/Molecular">colony PCR</A> could not give sufficient results. 
</LI><LI>Successful PCR (standard Phusion PCR) on the &lt;bbpart&gt;BBa_J04450&lt;/bbpart&gt; BioBrick plasmid using forward primer pSB1C3 and reverse primer pSB1C3 for later <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Gibson Assembly</A>.
</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A> with <I>gldA</I> PCR product and pSB1C3 PCR product.
</LI><LI>Screening of colonies from <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Gibson Assembly</A> with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Colony_PCR" title="Team:Bielefeld-Germany/Labjournal/Molecular"> colony PCR</A> and plasmid <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular"> restriction analysis</A> shows re-ligated pSB1C3, as already seen while using <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#NEB_BioBrick_Assembly_Kit" title="Team:Bielefeld-Germany/Labjournal/Molecular"> NEB Assembly Kit</A>.
</LI></UL><H4><SPAN class="mw-headline" id="Porines">Porines</SPAN></H4><UL><LI>We will try to use <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Gibson Assembly</A> for cloning <I>oprF</I> into pSB1C3.
</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Gibson Assembly</A> with <I>oprF</I> PCR product and pSB1C3 PCR product. Screening of colonies from <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A> with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Colony_PCR" title="Team:Bielefeld-Germany/Labjournal/Molecular"> colony PCR</A> and plasmid <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular"> restriction analysis</A> shows re-ligated pSB1C3, as already seen while using <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#NEB_Biobrick_Assembly_Kit" title="Team:Bielefeld-Germany/Labjournal/Molecular"> NEB Assembly Kit</A>.
</LI></UL><H4><SPAN class="mw-headline" id="Nanowires_2">Nanowires</SPAN></H4><UL><LI>PCR with forward and reverse primer GSU 1776-1784 on the appropriate gene-cluster of <I>Geobacter sulfurreducens</I> strain, using the PCR Master Mix, described in Table 3 (June) with adjusted template volume and the standard PCR program for amplification of <I>Geobacter sulfurreducens</I> gene cluster (Table 4, June) did not work as expected.
</LI><LI>Bands of several sizes, but no band at the expected size of 9000 bp, the length of GSU 1776-1784. Because of the correct band for the control PCR on the gene cluster GSU Promoter 1496-1505 a problem with the PCR program can be eliminated.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 2:</B> Agarose gel from PCR on the gene-clusters GSU 1776-1784 (bands 2-5), the GSU Promoter 1496-1505 (band 7) and <I>Geobacter sulfurreducens</I> strain. Ladder: NEB 2-Log DNA Ladder (0.1-10.0 kb).</DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Week_11">Week 11</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_3">Organization</SPAN></H4><UL><LI>We’ve presented us at the congress '<A href="http://www.bio.nrw.de/studentconvention" class="external text" rel="nofollow">BioNRW pHD Student Convention</A>' in Düsseldorf at July 13th, 2013 with a short presentation about iGEM and our project.
</LI><LI>We proudly present our <A href="http://ekvv.uni-bielefeld.de/blog/uniaktuell/entry/mit_bakterien_batterie_strom_erzeugen" class="external text" rel="nofollow">first press release.</A> It was amazing to see how often our press release was picked up.
</LI></UL><H4><SPAN class="mw-headline" id="MFC_3">MFC</SPAN></H4><UL><LI>Recorded power output of <I>e. coli</I> in our second generation fuel cell
</LI><LI>Manufactered different electrodes out of carbon tissue and tested them
</LI></UL><H4><SPAN class="mw-headline" id="Mediators_2">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI>Repeating <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Gibson Assembly</A> shows every time a re-ligated pSB1C3, a Gibson overlap of Prefix and Suffix with 20 bp seems to be too short, furthermore Prefix and Suffix have too many homologous regions.
</LI></UL><H4><SPAN class="mw-headline" id="Porines_2">Porines</SPAN></H4><UL><LI>Repeating <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Gibson Assembly</A> shows every time a re-ligated pSB1C3, a Gibson overlap of Prefix and Suffix with 20 bp seems to be too short, furthermore Prefix and Suffix have too many homologous regions.
</LI></UL><H4><SPAN class="mw-headline" id="Nanowires_3">Nanowires</SPAN></H4><UL><LI>Gibson Assembly with GSU 1491-1495, GSU 1496-1505 and GSU Promoter 1496-1505 PCR product as one fragment and the pSB1C3 PCR product, using the primer with Prefix and Suffix specific overlaps, as the second DNA fragment.
</LI><LI>Screening of colonies from Gibson Assembly by plasmid restriction analysis shows fragments of 2000 bp, which corresponds to re-ligated pSB1C3.
</LI><LI>PCR with forward and reverse primer GSU 1776-1784 on the appropriate gene-cluster of <I>Geobacter sulfurreducens</I> strain, using the PCR Master Mix, described in Table 3 (June) with adjusted template volume was repeated with the standard PCR program for amplification of <I>Geobacter sulfurreducens</I> gene cluster (Table 4, June) modified by a gradient of different annealing temperatures in a range of 45°C to 65°C. 
</LI><LI>No bands at the expected size of 9000 bp.
</LI></UL><H3><SPAN class="mw-headline" id="Week_12">Week 12</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_4">Organization</SPAN></H4><UL><LI>Dr. Falk Harnisch will have a presentation at CeBiTec colloquium as an expert for our team.
</LI><LI>Preparing experiments for the ‘<A href="https://2013.igem.org/Team:Bielefeld-Germany/Human_Practice/Day_of_Synthetic_Biology" class="external text" rel="nofollow">Day of Synthetic Biology</A>’. Our ideas are experiments like DNA isolation from fruit and vegetables, pipetting of bright colors, chromatography with markers, a potato battery and microscopy.
</LI><LI>The TV station <A href="http://www1.wdr.de/mediathek/video/sendungen/lokalzeit/lokalzeit-owl" class="external text" rel="nofollow">WDR</A> would like to contribute with us on our topic.
</LI></UL><H4><SPAN class="mw-headline" id="MFC_4">MFC</SPAN></H4><UL><LI>Visited the local hackerspace where we learned the basics of using a 3D printer and spent the night trying to make it work with our computers.
</LI><LI>Continued constructing and testing different electrode designs.
</LI></UL><H4><SPAN class="mw-headline" id="Mediators_3">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI>New primer design for pSB1C3 with longer overlaps individually for each part in order to stop the problem of re-ligated pSB1C3:
</LI><LI>Forward primer pSB1C3 <I>gldA</I> (43 bp): TCCTGCAAGAGTGGGAATAATACTAGTAGCGGCCGCTGCAGTC
</LI><LI>Reverse primer pSB1C3 <I>gldA</I> (43 bp): GATTGAATAATGCGGTCCATCTAGAAGCGGCCGCGAATTCCAG
</LI></UL><H4><SPAN class="mw-headline" id="Cytochromes_2">Cytochromes</SPAN></H4><UL><LI>Amplification of Fragment1 with <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" class="external text" rel="nofollow">Phusion polymerase</A></LI><LI>Size: 1833 bp
</LI><LI>Gradient: 54°C - 71°C over 8 steps
</LI><LI>Primer: 'mtrC'_fwd &amp; mtr_Frag1_rev
</LI><LI>Template: <I>S. oneidensis</I> PCR Template from genomic DNA
</LI><LI>Notes: Annealing temperature has no significant impact on the PCR
</LI></UL><UL><LI>Amplification of Fragment2 with  <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" class="external text" rel="nofollow">Phusion polymerase</A></LI><LI>Size: 321 bp
</LI><LI>Primer: mtr_Frag2_fwd &amp; mtr_Frag2_rev
</LI><LI>Template: <I>S. oneidensis</I> PCR Template from genomic DNA
</LI><LI>Notes: Annealing temperature has no significant impact on the PCR
</LI></UL><UL><LI>Amplification of Fragment3 with <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" class="external text" rel="nofollow">Phusion polymerase</A></LI><LI>Size: 3057 bp
</LI><LI>Gradient: 54°C - 71°C over 8 steps
</LI><LI>Primer: mtr_Frag3_fwd &amp; mtrB_rev
</LI><LI>Template: <I>S. oneidensis</I> PCR Template from genomic DNA
</LI><LI>Notes: 
</LI></UL><UL><LI>Amplification of <I>ccmAH</I> cluster
</LI><LI>Size:6311 bp
</LI><LI>Program: <A href="/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#cyt_ccmAH" title="Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms" class="mw-redirect">PhusionPCR</A></LI><LI>Gradient: 51.5°C - 68°C over 8 steps
</LI><LI>Primer: ccmAH_fwd &amp; ccmAH_rev
</LI><LI>Template: <I>E. coli</I> PCR Template from genomic DNA
</LI><LI>Notes: A high annealing temperature of 68°C or more increases yield as well as reduces unspecific bands and is therefore recommended.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 3: Results of a PAGE of an amplification of <I>ccmAH</I>; Expected Size: 6.3kb; Gradient:  51.5-68°C; Ladder:Thermo Scientific GeneRuler™ 1 kb DNA Ladder.</B></P></DIV></DIV></DIV><UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Used_Kits" class="external text" rel="nofollow">Gel extraction and cleanup</A> of Fragment 1, 2, 3 and <I>ccmAH</I></LI><LI>Elution buffer was preheated to 50°C
</LI><LI>We eluted each column twice, with 20 ul each and combined it afterwards
</LI><LI>Measurement of nucleic acid concentration via NanoDrop</LI><LI>Fragment1: 4-2607-304: 10.5 ng/ul
</LI><LI>Fragment2: 4-2707-003: 44.6 ng/ul
</LI><LI>Fragment2: 4-2707-004: 13.5 ng/ul
</LI><LI>Fragment3: 4-2607-301: 11.1 ng/ul
</LI><LI>Fragment3: 4-2607-302: 8.9 ng/ul
</LI><LI>Fragment3: 4-2607-303: 9.2 ng/ul
</LI><LI><I>ccmAH</I>: 4-2807-302: 11.0 ng/ul
</LI><LI><I>ccmAH</I>: 4-2807-303: 11.5 ng/ul
</LI></UL><H4><SPAN class="mw-headline" id="Porines_3">Porines</SPAN></H4><UL><LI>New primer design for pSB1C3 with longer overlaps individually for OprF part in order to stop the problem of re-ligated pSB1C3:
</LI><LI>Forward primer pSB1C3 <I>oprF</I> (43 bp): TTGAAGCCCAAGCTAAGTAATACTAGTAGCGGCCGCTGCAGTC
</LI><LI>Reverse primer pSB1C3 <I>oprF</I> (43 bp): AAGGTGTTTTTCAGTTTCATCTAGAAGCGGCCGCGAATTCCAG
</LI></UL><H4><SPAN class="mw-headline" id="Nanowires_4">Nanowires</SPAN></H4><UL><LI>Multiple Gibson Assembly retries with different molar ratios of the two fragments results every time in re-ligated pSB1C3 plasmids, so that the used Gibson overlap of Prefix and Suffix is not useable.
</LI><LI>Design of new primers for pSB1C3 amplification with longer sequence-specific overlaps for every part:
</LI><LI>Forward pSB1C3 GSU 1491-1505 / GSU 1496-1505 / GSU Promoter 1496-1505 (43 bp): GGGACGAGGTTCGTTTATGATACTAGTAGCGGCCGCTGCAGTC 
</LI><LI>Reverse pSB1C3 GSU 1496-1505 (43 bp): GTATGGGGGTAATTGGCCAACTCTAGAAGCGGCCGCGAATTCC 
</LI><LI>Forward pSB1C3 GSU 1491-1495 (43 bp): TTGGCATGGATGAGGAGTGATACTAGTAGCGGCCGCTGCAGTC
</LI><LI>Reverse pSB1C3 GSU 1491-1495 / GSU 1491-1505 (43 bp): CCCAGTCTGCTAGCCTGCATCTAGAAGCGGCCGCGAATTCCAG
</LI><LI>Reverse pSB1C3 GSU Promoter 1496-1505 (43 bp): ACTCGGTGACGGATCCTATCCTCTAGAAGCGGCCGCGAATTCC
</LI></UL><H3><SPAN class="mw-headline" id="Week_13">Week 13</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_5">Organization</SPAN></H4><UL><LI>Spontaneous visit of <A href="http://www.radiobielefeld.de" class="external text" rel="nofollow">Radio Bielefeld</A> in our laboratory. In addition to a radio interview a <A href="http://www.radiobielefeld.de/programm/bei-uns-im-programm/studenten-entwickeln-biobatterie.html" class="external text" rel="nofollow">short video clip</A> was filmed.
</LI></UL><H4><SPAN class="mw-headline" id="Mediators_4">Mediators</SPAN></H4><UL><LI> Phenazine
</LI></UL><P align="justify">
Almost ten PCRs were run up to this time. Unfortunately, none of them was successful. We show only two gel pictures out of many, where two different gradients were taken.
We have tried to add betaine as a stabilizing agent, work with <A href="http://www.millipore.com/publications.nsf/a73664f9f981af8c852569b9005b4eee/31140b3dbc2b4146852579d8005e9c72/$FILE/PR3366EN00.pdf" class="external text" rel="nofollow">KOD Hot Start polymerase</A> according to the manual. We tried different extention times and annealing in different temperatures. 
 
The fragment <A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Phenazine" class="external text" rel="nofollow"><I>PhzABCDEFGH</I></A> contains 8 illegal restriction sites and is 8505 bp long. 
After trying all strategies, we decided to abandon this sub-project and start with another endogenous mediator, riboflavin (see <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/August" class="external text" rel="nofollow">Labjournal August</A>)</P><OL><LI>Recipe PCR
</LI><LI>2x KOD Xtreme™ PCR Buffer: 25 µL</LI><LI>dNTPs: 1.0 µL</LI><LI>Primer1: 1.0 µL</LI><LI>Primer2: 1.0 µL</LI><LI>Template DNA: 1.5 µL</LI><LI>DMSO: 1.5 µL</LI><LI>Betaine: 12.5 µL</LI><LI>KOD Hot Start Polymerase: 0.5 µL</LI><LI>Dest. H2O: 6 µL</LI><LI>Total volume: 50 µL</LI><LI>Protocol PCR Two-step Cycling</LI><UL><LI>Polymerase Activation 94°C for 2 min</LI><LI>Denaturation 98°C for 10 s</LI><LI>Annealing gradient 50.8°C-70.2°C</LI><LI>Extension 72°C for 270 s/320 s</LI></UL></OL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>PCR for <I>PhzABCDEFGH</I>-gene cluster</B>, gradient with KOD-Polymerase, 270 s extension</DIV></DIV></DIV><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>PCR for <I>PhzABCDEFGH</I>-gene cluster</B>, gradient with KOD-Polymerase, 320 s extension</DIV></DIV></DIV><P align="justify">
As is clear from the gels, we were not able to amplify the fragment and have closed this sub-project.</P><H4><SPAN class="mw-headline" id="Biosafety">Biosafety</SPAN></H4><UL><LI>Amplification of different fragments:
</LI></UL><UL><LI>Recipe:
</LI><LI>PhusionBuffer: 5.5 µL
</LI><LI>dNTPs: 1 µL
</LI><LI>Primer1: 1 µL
</LI><LI>Primer2: 1 µL
</LI><LI>Template: 1.5 µL
</LI><LI>DMSO: 0.5 µL
</LI><LI>Phusion: 0.5 µL
</LI><LI>Dest. H<SUB>2</SUB>O: 39 µL
</LI></UL><UL><LI>1.<I>araC</I>:
</LI><LI>Primer: araC_d1+ araC_d2
</LI><LI>Annealing: 62°C
</LI><LI>Extension: 1 min
</LI><LI>Template: <I>E.coli</I> Genome
</LI><LI>Size: 600 bp
</LI></UL><UL><LI>2.Deletion of <I>araC</I>:
</LI><LI>Primer: araC_d3+ araC_d4
</LI><LI>Annealing= 62°C
</LI><LI>Extension: 1 min
</LI><LI>Template: <I>E.coli</I> Genome
</LI><LI>Size: 600 bp
</LI></UL><UL><LI>3.Amplification of <I>araC</I> into our Biosafety system:
</LI><LI>Primer: araC_fwd+ araC rev
</LI><LI>Annealing= 62°C
</LI><LI>Extension: 1 min
</LI><LI>Template: BBa_I13458
</LI><LI>Size: 900 bp
</LI></UL><UL><LI>4.Preparation of pSB1C3:
</LI><LI>Primer: pRha_rev+ pSB1C3_rha_suf
</LI><LI>Annealing: 62°C
</LI><LI>Extension: 1.30 min
</LI><LI>Template: BBa_I13541
</LI><LI>Size: 3 kb
</LI></UL><UL><LI>5.Preparation of pSB1C3:
</LI><LI>Primer: pSB1C3_alr_fwd+ pSB1C3_alr_rev
</LI><LI>Annealing: 62°C
</LI><LI>Extension: 1 min
</LI><LI>Template: <I>E.coli</I> Genome
</LI><LI>Size: 1 kb
</LI></UL><UL><LI>6.Preparation of pSB1C3:
</LI><LI>Primer: pSB1C3_plac_alr+ pSB1C3_alr_rev
</LI><LI>Annealing: 62°C
</LI><LI>Extension: 1 min
</LI><LI>Template: <I>E.coli</I> Genome
</LI><LI>Size: 1 kb
</LI></UL><UL><LI>7.Preparation of pSB1C3:
</LI><LI>Primer: pSB1C3_alr_pre+ pSB1C3_alr_suf
</LI><LI>Annealing: 62°C
</LI><LI>Extension: 1.30 min
</LI><LI>Template: pSB1C3
</LI><LI>Size: 2 kb
</LI></UL><UL><LI>8.Preparation of pSB1C3: 
</LI><LI>Primer: pSB1C3_plac_pre+ pSB1C3_alr_suf
</LI><LI>Annealing: 62°C
</LI><LI>Extension: 1.30min
</LI><LI>Template: pSB1C3
</LI><LI>Size: 2 kb
</LI></UL><UL><LI>Analysis by agarose gel
</LI><LI>Result:failed
</LI></UL><UL><LI>New PCR:
</LI><LI>Program: 06 phy_cyt mit 62°C Annealing
</LI><LI>10 µL 5xHF-Buffer
</LI><LI>4 µL dNTPs
</LI><LI>0.5 µL Primer1
</LI><LI>0.5 µL Primer2
</LI><LI>1 µL Template
</LI><LI>1.5 µL DMSO
</LI><LI>1 µL Enzyme
</LI><LI>31.5 µL dest. H<SUB>2</SUB>O
</LI></UL><UL><LI>Analysis by agarose gel:
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Gelanalysis of PCR 1-4.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Gel analysis of PCR 5-8.</DIV></DIV></DIV></DIV><UL><LI>PCR-Purification:
</LI><LI>BS1a (9-38-451) 113.6 ng/µL</LI><LI>BS1b (9-38-452) 75.1 ng/µL</LI><LI>BS2a (9-38-453) 85.5 ng/µL</LI><LI>BS3a (9-38-454) 107.6 ng/µL</LI><LI>BS4a (9-38-455) 14.4 ng/µL</LI><LI>BS5a (9-38-456) 44.5 ng/µL</LI><LI>BS5b (9-38-457) 49.0 ng/µL</LI><LI>BS6a (9-38-458) 63.6 ng/µL</LI><LI>BS7a (9-38-459) 6.9 ng/µL</LI><LI>BS8a (9-38-460) 8.6 ng/µL</LI></UL><UL><LI>Successful PCR on the BBa_J04450 BioBrick plasmid using part specific forward and reverse primer (Week 12) for later Gibson Assembly. Isolation of the pSB1C3 PCR product by agarose gel electrophoresis and gel extraction.
</LI><LI>Gibson Assembly, using the different new pSB1C3 PCR products and the according gene-cluster PCR products.
</LI></UL></DIV><DIV id="asdf" class="withMenu"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Milestones"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Milestones</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_9"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 9</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Organization"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#MFC"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Cytochromes"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Cytochromes</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Nanowires"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Nanowires</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-7"><A href="#Week_10"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 10</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Organization_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#MFC_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Mediators"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Porines"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Porines</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Nanowires_2"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Nanowires</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Week_11"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 11</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Organization_3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#MFC_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Mediators_2"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Porines_2"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Porines</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Nanowires_3"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Nanowires</SPAN></A></LI><LI class="toclevel-1 tocsection-19"><A href="#Week_12"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 12</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Organization_4"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#MFC_4"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Mediators_3"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Cytochromes_2"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Cytochromes</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Porines_3"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Porines</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Nanowires_4"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Nanowires</SPAN></A></LI><LI class="toclevel-1 tocsection-26"><A href="#Week_13"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Week 13</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Organization_5"><SPAN class="tocnumber">6.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Mediators_4"><SPAN class="tocnumber">6.2</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Biosafety"><SPAN class="tocnumber">6.3</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/July">http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Labjournal/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Labjournal/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/July&amp;oldid=354824" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>