<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Biosafety_Biosafety_System_M"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Biosafety/Biosafety_System_M&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_M&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_M&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Biosafety/Biosafety_System_M" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Biosafety/Biosafety System M</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>Biosafety System TetOR alive</H1><DIV id="buttonrow" style="padding-top:30px; padding-bottom:70px; padding-left:45px; clear:both;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">System Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#Genetic_Approach">Genetic Approach</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#Results">Results</A></DIV></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Biosafety-System TetOR alive.</B></DIV></DIV></DIV><P align="justify">
The Biosafety-System TetOR alive &lt;bbpart&gt;BBa_K1172915&lt;/bbpart&gt; is an improvement of the BioBrick &lt;bbpart&gt;BBa_K914014&lt;/bbpart&gt; by replacing the first promoter into the rhamnose promoter P<SUB><I>Rha</I></SUB>, integration of the alanine racemase &lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt; and utilization of the repressor TetR to regulate the transcription of the Barnase behind the <I>tetO</I> promoter. Because this system is known for a tight repression and a fast activation, it is expected that bacteria containing this Biosafety-System are dead or alive...
</P><H2><SPAN class="mw-headline" id="Genetic_Approach">Genetic Approach</SPAN></H2><H3><SPAN class="mw-headline" id="Rhamnose_promoter_PRha"><B>Rhamnose promoter P<SUB>Rha</SUB></B></SPAN></H3><P align="justify">
The promoter P<SUB><I>Rha</I></SUB> (&lt;bbpart&gt;BBa_K914003&lt;/bbpart&gt;) naturally regulates  the catabolism of the hexose L-rhamnose. The advantage to use this promoter is its solely positive regulation. The regulon consists of the gene <I>rhaT</I> for the  L-rhamnose transporter and the two operons <I>rhaSR</I> and <I>rhaBAD</I>.
The operon <I>rhaSR</I> encodes the two transcriptional activators RhaS and RhaR, who are responsible for the positive activation of the L-rhamnose catabolism, while the operon <I>rhaBAD</I> encodes the genes for the direct catabolism of L-rhamnose.
When L-rhamnose is present, it acts as an inducer by binding to the regulatory protein RhaR. RhaR regulates its own expression and the expression of the regulatory gene <I>rhaS</I> by repressing or, in the presence of L-rhamnose, activating the operon <I>rhaSR</I>. Normally, the expression level is modest, but it can be enhanced by a higher level of intracellular cAMP, which increases in the absence of glucose. So, in the presence of L-rhamnose and a high concentration of intracellular cAMP, the activator protein RhaR is expressed on higher level, resulting in an activation of the promoter P<SUB><I>rhaT</I></SUB> for an efficient L-rhamnose uptake and an activation of the operon <I>rhaBAD</I>. The L-rhamnose is than broken down into dihydroxyacetone phosphate and lactate aldehyde by the enzymes encoded by <I>rhaBAD</I> (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Wickstrum <I>et al.</I>, 2005</A>).
A brief schematic summary of the regulation is shown in Figure 1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1:</B> The catabolism of L-rhamnose in <I>E. coli</I> is turned off in general but inducible by L-rhamnose. The induction activates the transcription of the genes <I>rhaS</I> and <I>rhaR</I>, which regulate the L-rhamnose catabolism by positive activation of the rhamnose uptake (<I>rhaT</I>) and its metabolization (<I>rhaBAD</I>).</DIV></DIV></DIV></DIV><P align="justify">
Dihydroxyacetone phosphate can be metabolized in the glycolysis pathway, while lactate aldehyde is oxidized to lactate under aerobic conditions and reduced to L-1,2,-propandiol under anaerobic conditions.
The degradation of L-rhamnose can be separated in three steps. In the first step the L-rhamnose is turned into L-rhamnulose by an isomerase (gene <I>rhaA</I>). L-rhamnulose is in turn phosphorylated to L-rhamnulose-1-phosphate by a kinase (gene <I>rhaB</I>) and the sugar phosphate is finally hydrolyzed by an aldolase (gene <I>rhaD</I>) to dihydroxyacetone phosphate and lactate aldehyde (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Baldoma <I>et al.</I>, 1988</A>).
For our Safety-System <I>TetOR alive</I>, the rhamnose promoter P<SUB>Rha</SUB> is used to control the expression of the repressor TetR and the essential alanine racemase, because this promoter has a very low basal transcription. This is needed to tightly repress the expression of the alanine racemase (<I>alr</I>) and thereby take advantage of the double-kill switch. The Tet-System is also characterized by a tight repression, but the induction have to be realized with an antibiotic or anhydrotetracycline(<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Kamionka <I>et. al</I>, 2004</A>). Although the rhamnose promoter P<SUB><I>Rha</I></SUB> is characterized by a very low basal transcription it can not be used for the control of the toxic RNase Ba (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#RNase_Ba_.28Barnase.29" class="external text" rel="nofollow">Barnase</A>), which ist needed for the alanine racemase (<I>alr</I>), the other part of the double-kill switch. In conclusion this promoter is the best choice for the first part of the Biosafety-System, as it is tightly repressed in the absence of L-rhamnose, but activated in its presence.</P><H3><SPAN class="mw-headline" id="Tetracyclin_repressor.2Foperator"><B>Tetracyclin repressor/operator</B></SPAN></H3><P align="justify">
The tetracycline repressor (TetR)/ operator (TetO) is naturally used by <I>E. coli</I> to activate the detoxification of tetracycline if it is encountered by the bacteria. In the absence of tetracycline (continous line) TetR is expressed and forms a homodimer and binds with high affinity to the two tetracycline operator sides <I>tetO<SUB>1</SUB></I> and <I>tetO<SUB>2</SUB></I>. This results in the repression of the tetracycline efflux transporter TetA. If tetracycline is present, it diffuses through the cell membrane and can block protein biosynthesis by reversibly inhibiting the binding of aminoacyl-tRNA to the mRNA-ribosome. When the Tet-System is present the tetracycline forms a complex with Mg<SUP>2+</SUP> (red triangle). This complex binds to TetR causing a conformation switch in the repressor (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">S Orth <I>et. al.</I>, 2000</A>). This causes the repressor TetR to dissociate from the <I>tetO</I> operator, resulting in the expression of TetA. TetA is integrated into the cytoplasmic membrane and works as an antiporter transporting the tetracycline complex outside, thus allowing protein biosynthesis to proceed.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2:</B> Principle of TetR and <I>tetO</I> system in tetracycline resistant bacteria.</DIV></DIV></DIV></DIV><P>
For our Biosafety-System <I>TetOR alive</I> we used this System for the regualtion of the toxic RNase Ba. We used the TetR repressor (&lt;bbpart&gt;BBa_C0040&lt;/bbpart&gt;) for the regulation of the <I>tetO</I> operator (&lt;bbpart&gt;BBa_R0040&lt;/bbpart&gt;) containing the toxic RNase Ba. As this System is known for a tight repression and a high activation, it should be optimal for a Biosafety-System.
</P><H3><SPAN class="mw-headline" id="Alanine_racemase_Alr"><B>Alanine racemase Alr</B></SPAN></H3><P align="justify">
The alanine racemase Alr (EC 5.1.1.1) from the Gram-negative enteric bacteria <I>Escherichia coli</I> is an isomerase, which catalyses the reversible conversion of L-alanine into the enantiomer D-alanine (see Figure 2). For this reaction, the cofactor pyridoxal-5'-phosphate (PLP) is necessary. The constitutively expressed alanine racemase (<I>alr</I>) is naturally responsible for the accumulation of D-alanine. This compound is an essential component of the bacterial cell wall, because it is used for the cross-linkage of peptidoglycan (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Walsh, 1989</A>).
The usage of D-alanine instead of a typically L-amino acid prevents cleavage by peptidases. However, a lack of D-alanine causes to a bacteriolytic characteristics. In the absence of Dâalanine dividing cells will lyse rapidly. This fact is used for our Biosafety-Strain, a D-alanine auxotrophic mutant (K-12 â<I>alr</I> â<I>dadX</I>). The Biosafety-Strain grows only with a plasmid containing the alanine racemase (&lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;) to complement the D-alanine auxotrophy. Consequently the alanine racemase is essential for bacterial cell division. This approach guarantees a high plasmid stability, which is extremely important when the plasmid contains a toxic gene like the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#RNase_Ba_.28Barnase.29" class="external text" rel="nofollow">Barnase</A>. In addition this construction provides the possibility for the implementation of a double kill-switch system. Because if the expression of the alanine racemase is repressed and there is no D-alanine supplementation in the medium, cells will not grow.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:355px;"><DIV class="thumbcaption"><B>Figure 3:</B> The alanine racemase (&lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;) from <I>E. coli</I> catalyses the reversible conversion from L-alanine to D-alanine. For this isomerization the cofactor pyridoxal-5'-phosphate is necessary.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Terminator"><B>Terminator</B></SPAN></H3><P align="justify">
Terminators are essential to terminate the transcription of an operon. In procaryotes two types of terminators exist. The rho-dependent and the rho-independent terminator. Rho-independent terminators are characterized by their stem-loop forming sequence. In general, the terminator-region can be divided into four regions. The first region is GC-rich and constitutes one half of the stem. This region is followed by the loop-region and another GC-rich region that makes up the opposite part of the stem. The terminator closes with a poly uracil region, which destabilizes the binding of the RNA-polymerase. The stem-loop of the terminator causes a distinction of the DNA and the translated RNA. Consequently the binding of the RNA-polymerase is cancelled and the transcription ends after the stem-loop (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Carafa <I>et al.</I>, 1990</A>).
For our Bioafety-System <I>TetOR alive</I> the terminator is necessary to avoid that the expression of the genes under control of the rhamnose promoter P<SUB><I>Rha</I></SUB>, like the Repressor TetR and the alanine racemase (<I>alr</I>) results in the transcription of the genes behind the tetracylcine operon <I>tetO</I> which contains the toxic Barnase &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; and would lead to cell death.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 4:</B> Stem-loop structure of the terminator &lt;bbpart&gt;BBa_B0015&lt;/bbpart&gt;, which is used for the Biosafety-System <I>TetOR alive</I>. The terminator is used to make sure that solely both the repressor TetR and the alanine racemase Alr are expressed but the transcription of the toxic RNase Ba (Barnase) is avoided.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="RNase_Ba_.28Barnase.29"><B>RNase Ba (Barnase)</B></SPAN></H3><P align="justify">
The Barnase (EC 3.1.27) is a 12 kDa extracellular microbial ribonuclease, which is naturally found in the Gram-positive soil bacteria <I>Bacillus amyloliquefaciens</I> and consists of a single chain of 110 amino acids. The Barnase (RNase Ba) catalyses the cleavage of single stranded RNA, preferentially behind Gs. In the first step of the RNA-degradation a cyclic intermediate is formed by transesterification and afterwards this intermediate is hydrolyzed yielding in a 3'-nucleotide (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Mossakowska <I>et al.</I>, 1989</A>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:542px;"><DIV class="thumbcaption"><B>Figure 5:</B> Enzymatic reaction of the RNA-cleavage by the RNase Ba. First the transesterification by the Glu-73 residue is performed and then this cyclic intermediate is hydrolyzed by the His-102 of the Barnase.</DIV></DIV></DIV></DIV><P align="justify">
In <I>Bacillus amyloliquefaciens</I>, the activity of the Barnase (RNase Ba) is inhibited intracellular by an inhibitor called barstar. Barstar consists of only 89 amino acids and binds with a high affinity to the toxic Barnase. This prevents the cleavage of the intracellular RNA in the host organism (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#References" class="external text" rel="nofollow">Paddon <I>et al.</I>, 1989</A>). Therefore the Barnase normally acts only outside the cell and is translocated under natural conditions. For the Biosafety-System <I>TetOR alive</I> we modified the enzyme by cloning only the sequence responsible for the cleavage of the RNA, leaving out the N-terminal signal peptide part.

As shown in Figure 6 below, the transcription of the DNA, which encodes the Barnase produces a 474 nt RNA. The translation of the RNA starts about 25 nucleotides downstream from the transcription start and can be divided into two parts. The first part (colored in orange) is translated into a signal peptide at the amino-terminus of the Barnase coding RNA. This part is responsible for the extracellular translocation of the RNase Ba, while the peptide sequence for the active Barnase starts 142 nucleotides downstream from the transcription start (colored in red).
For the Biosafety-System <I>TetOR alive</I>, we only used the part (&lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt;) of the Barnase encoding the catalytic domain without the extracellular translocation signal of the toxic gene product. Translation of the barnase gene leads to rapid cell death if the expression of the Barnase is not repressed by the repressor TetR of our Biosafety-System.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6:</B> Sequence of the signal peptide amino terminal of the RNase Ba (Barnase). The Biobrick &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; does not contain the signal sequence for the extracellular translocation, but only the coding sequence for the mature enzyme.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Biosafety_system_TetOR_alive"><B>Biosafety system TetOR alive</B></SPAN></H3><P align="justify">
Combining the genes described above with the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> results in a powerful device, allowing us to control the bacterial cell division. The control of the bacterial growth is possible either active or passive. Active by inducing the <I>tetO</I> operator with tetracycline and passive by the induction of L-rhamnose. The passive control makes it possible to control the bacterial cell division in a defined closed environment, like the MFC, by continuously adding L-rhamnose to the medium. As shown in Figure 7 below, this leads to an expression of the essential alanine racemase (<I>alr</I>) and the TetR repressor, so that the expression of the RNase Ba is repressed. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 7:</B>Biosafety-System <I>TetOR alive</I> in the presence of L-rhamnose. The essential alanine racemase (Alr) and the repressor TetR are expressed, resulting in a repression of the expression of the RNAse Ba. Consequently the bacteria show normal growth behavior.</DIV></DIV></DIV></DIV><P align="justify">
In the event that bacteria exit the defined environment of the MFC or L-rhamnose is not added to the medium any more, both the expression of the alanine racemase (Alr) and the TetR repressor decrease, so that the expression of the toxic RNase Ba (Barnase) begins. The cleavage of the intracellular RNA by the Barnase and the lack of synthesized D-alanine, caused by the repressed alanine racemase inhibit the cell division. Through this it can be secured that the bacteria can only grow in the defined area or the device of choice respectively. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 8:</B>Active Biosafety-System <I>TetOR alive</I> outside of a defined environment lacking L-rhamnose. Both the expression of the alanine racemase (Alr) and TetR repressor are reduced and ideally completely shut down. In contrast, the expression of the RNase Ba (Barnase) is turned on, leading to cell death by RNA cleavage.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Characterization_of_the_tetracycline_tetO_operator"><B>Characterization of the tetracycline <I>tetO</I> operator</B></SPAN></H3><P align="justify">
First, the <I>tetO</I> operator was characterized to get a first impression of its basal transcription rate. Therefore, the bacterial growth was investigated using the unrepressed <I>tetO</I> operator on different carbon source using M9 minimal medium with either glucose or glycerol. The transcription rate was identified by fluorescence measurement of GFP &lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt; behind the <I>tetO</I> operator &lt;bbpart&gt;BBa_R0040&lt;/bbpart&gt; using the BioBrick &lt;bbpart&gt;BBa_K1172914&lt;/bbpart&gt;.
As shown in Figure 9 below, the bacteria grew slightly better on glucose then on glycerol. This is due to glucose being the better energy source of these two, because glycerol enters glycolysis at a later step and therefore delivers less energy. Moreover an additional ATP consumption is needed to drive glycerol uptake. For the investigation of the basal transcription the fluorescence measurements, shown in Figure 10, is more interesting.

In contrast to the other promoters characterized, like <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Characterization_of_the_arabinose_promoter_PBAD" class="external text" rel="nofollow">P<SUB><I>BAD</I></SUB></A> or <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#Characterization_of_the_lactose_promoter_Plac" class="external text" rel="nofollow">P<SUB><I>lac</I></SUB></A>, the fluorescence does not differ between the carbon sources used. This was expected in this case, because this operator is not enhanced by intracellular cAMP like the arabinoe or lactose promoter.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 9:</B> Characterization of the bacterial growth of the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> containing the plasmid &lt;bbpart&gt;BBa_K1172914&lt;/bbpart&gt; with GFP (&lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt;) under the control of the <I>tetO</I> operator. The M9 medium was supplemented with 5 mM D-alanine. It could be demonstrated, that the bacteria grow faster on M9 minimal medium containing glucose than on M9 minimal medium with glycerol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 10:</B> Characterization of the fluorescence of the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> containing the plasmid &lt;bbpart&gt;BBa_K1172914&lt;/bbpart&gt; with GFP (&lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt;) under control of the <I>tetO</I> operator. The Biosafety-Strain was cultivated on M9 minimal medium supplemented with 5 mM D-alanine..</DIV></DIV></DIV></DIV><P align="justify">
As for the other characterizations, the specific production rate was calculated to demonstrate in this case, that the carbon source does not influence the basal transcription, as shown in Figure 11. The specific production rates were calculated via equation (1)Â :</P><P align="justify">
With the calculation of the specific production rate of GFP it can be demonstrated that the GFP synthesis rates does not differ between the cultivation on glucose and glycerol. The specific production rate show fluctuation on both cultivation, resulting in an up and down, so that there is can no obviously obvious be seen. This demonstrates that <I>tetO</I> operator is not enhanced by cAMP and confirms the results of the lactose and arabinose promoter, as they can be enhanced by cAMP and their basal transcription differs on glucose and glycerol.
Moreover the specific production rate was calculated between every single measurement point, so the curve in Figure 11 is not smoothed and  the fluctuations have to be ignored, as they do not stand for real fluctuations in the expression of GFP. They are caused by measurement variations in the growth curve and the fluorescence curve. And as neither of this curves are ideal, the fluctuations are the result. Nevertheless this graph shows the difference between the two carbon sources.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 11:</B> Specific production rate of GFP expressed via the <I>tetO</I> operator in dependence of different carbon sources.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_the_Biosafety-System_TetOR_alive"><B>Characterization of the Biosafety-System TetOR alive</B></SPAN></H3><P align="justify">
The Biosafety-System <I>TetOR alive</I> was characterized on M9 minimal medium using glycerol as carbon source. As for the characterization of the pure <I>tetO</I> operator above, the bacterial growth and the fluorescence of GFP &lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt; was measured. Therefore, the wild type and the Biosafety-Strain <I>E. coli</I> K-12 â<I>alr</I> â<I>dadX</I> both containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172915&lt;/bbpart&gt; were cultivated once with the induction of 1% L-rhamnose and once only on glycerol.

It becomes obvious (Figure 12) that the bacteria, induced with 1Â % L-rhamnose (red and black curve), grow obviously slower, than on pure glycerol (orange and blue curve). This might be attributed to the high metabolic burden encountered by the induced bacteria. The expression of the repressor TetR and the alanine racemase (Alr) simultaneously causes a high stress on the cells, so that they grow slower than the uninduced cells which express only GFP.

Comparing the bacterial growth with the fluorescence in Figure 15, it can be seen that the fluorescence of the Biosafety-Strain is much higher than the fluorescence of the wild type. This was also observed for the Biosafety-Strain <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#Characterization_of_the_Biosafety-System_Lac_of_Growth" class="external text" rel="nofollow"><I>Lac of Growth</I></A> and therefore the wild tpye is taken for further analysis of the Biosafety-Plasmid. While the Biosafety-Strain shows an decreasing fluorescence, the wild type shows an expected increase during cultivation. As observed for the other Biosafety-Systems, the fluorescence seems to follow the same trend than the bacterial growth. The uninduced cells show approximately an exponential rise of fluorescence, while in comparison the fluorescence of the induced bacteria increases only slowly.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 12:</B> Characterization of the bacterial growth of the Biosafety-System <I>TetOR alive</I> on M9 minimal medium with glycerol. The Figure compares the wild type K-12 and the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172915&lt;/bbpart&gt; and the induction by 1% L-rhamnose to pure glycerol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 13:</B> Characterization of the fluorescence of the Biosafety-System <I>TetOR alive</I>. The Figure compares the wild type K-12 and the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172915&lt;/bbpart&gt; and the induction by 1% L-rhamnose to pure glycerol.</DIV></DIV></DIV></DIV><P align="justify">
From the data presented above, it cannot be determined if the expression of the repressor TetR does affect the transcription of GFP or not. Considering the wild type containing the Biosafety-System, the slower growth is a first indication that the repressor TetR and the alanine racemase (Alr) are highly expressed, but the growth of the bacteria shows nearly the same kinetics as the fluorescence. So it could be possible that the repressor does not affect the expression level of GFP under the control of the <I>tetO</I> operator. This becomes more clear by the calculation of the specific production rate of GFP by equation (1) . As shown in Figure 14 below the specific production rate does not really differ between the uninduced Biosafety-System and the Biosafety-System induced by 1% L-rhamnose.
At the beginning the production of GFP in the presence of L-rhamnose (red curve) is lower than in its absence (orange curve), so that the expression of GFP seems to be repressed in the presence of L-rhamnose, but later one this changes and the specific production rate is constantly higher in the induced Biosafety-Strain. The fluctuation can be ignored, because the specific production rate of GFP was calculated between every single measurement point. The curve in Figure 14 is not smoothed , as they do not stand for real fluctuations in the expression of GFP. They are caused by measurement variations in the growth curve and the fluorescence curve. But this unexpected tendency that the production of GFP is lower when the bacteria are uninduced is obvious, so the Biosafety-System  <I>TetOR alive</I> unfortunatly seems not to work. As the Biosafety-Systems <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#Conclusions" class="external text" rel="nofollow"><I>Lac of Growth</I></A> and <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Conclusions" class="external text" rel="nofollow"><I>AraCtive</I></A> work, there must be a problem in the interaction of the TetR repressor with the <I>tetO</I> operator. This is discussed in the next section.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 14:</B> Specific production rate of GFP for the Biosafety-System <I>TetOR alive</I>, calculated via equation (1). The production rate of GFP of the uninduced bacteria fluctuates so that the Biosafety-System might not working as expected.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Conclusions"><B>Conclusions</B></SPAN></H3><P align="justify">
As mentioned above it seems that the Biosafety-System <I>TetOR alive</I> does not work as expected. This is also confirmed by Figure 15 showing the specific production rates of GFP after 7,5 hours of the induced Biosafety-System <I>TetOR alive</I> (red bar), the uninduced Biosafety-System (orange bar) and the pure <I>tetO</I> operator (&lt;bbpart&gt;BBa_K1172914&lt;/bbpart&gt;). While induction does indeed results in a repression, the much lower expression from the pure <I>tetO</I> promoter/operator  was not expected.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 15:</B> Comparison of the specific production rate of GFP. Shown are the induced (1% L-rhamnose) Biosafety-System <I>TetOR alive</I>, the uninduced Biosafety-System <I>TetOR alive</I> and the second part of the Biosafety-System (<I>tetO</I> - GFP only).</DIV></DIV></DIV></DIV><P align="justify">
As the Biosafety-Systems <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#Conclusions" class="external text" rel="nofollow"><I>Lac of Growth</I></A> and <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Conclusions" class="external text" rel="nofollow"><I>AraCtive</I></A>  show the same genetic construction, but differ only in the repressor and promoter, this cannot be an error of the construction. Also the <A href="http://parts.igem.org/sequencing/part_analysis.cgi?part=BBa_K1172915" class="external text" rel="nofollow">sequence analysis</A> show the expected sequence, so that this system should actually work. In the Partsregistry similar problems were reported,  so this is might be due to an error in either of the parts, the repressor TetR &lt;bbpart&gt;BBa_C0040&lt;/bbpart&gt; and/or the <I>tetO</I> operator &lt;bbpart&gt;BBa_R0040&lt;/bbpart&gt;. So the Biosafety-Systems <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L" class="external text" rel="nofollow"><I>Lac of Growth</I></A> and <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S" class="external text" rel="nofollow"><I>AraCtive</I></A> work, while the Biosafety-System <I>TetOR alive</I> needs vigorous testing to identify and remove its problems. Nevertheless it would be worth to improve this part due to the tight shutdown and high activation of the Tet system which would be optimal for a biosafety system.</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Baldoma L and Aguilar J (1988) Metabolism of L-Fucose and L-Rhamnose in Escherichia coli: Aerobic-Anaerobic Regulation of L-Lactaldehyde Dissimilation <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210658/pdf/jbacter00179-0434.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 170: 416 - 421.</I></A>.
</LI><LI>Carafa, Yves d'Aubenton Brody, Edward and Claude (1990) Thermest Prediction of Rho-independent Escherichia coli Transcription Terminators - A Statistical Analysis of their RNA Stem-Loop Structures <A href="http://ac.els-cdn.com/S0022283699800059/1-s2.0-S0022283699800059-main.pdf?_tid=ede07e2a-2a92-11e3-b889-00000aab0f6c&amp;acdnat=1380629809_2d1a59e395fc69c8608ab8b5aea842f7|" class="external text" rel="nofollow"><I>Journal of molecular biology 216: 835 - 858</I></A>.
</LI><LI>Casali, N., Preston A. (2003) E. coli Plasmid Vectors - Methods and Applications <A href="http://www.springerprotocols.com/BookToc/doi/10.1385/1592594093" class="external text" rel="nofollow"><I>Methods in Molecular Biology 235</I></A><I>.</I></LI><LI>Kamionka, A., Sehnal, M., Scholz, O., Hillen, W. (2004) Independent Regulation of Two Genes in Escherichia coli by Tetracyclines and Tet Repressor Variants <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC421600/pdf/0151-04.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 186(13): 4399 â 4401.</I></A></LI><LI>Levchenko I., Grant R., Wah D., Sauer R., Baker T. (2003) Structure of a delivery protein for an AAA+ protease in complex with a peptide degradation tag. <A href="http://ac.els-cdn.com/S109727650300323X/1-s2.0-S109727650300323X-main.pdf?_tid=4fe9541a-3fed-11e3-a15a-00000aab0f01&amp;acdnat=1382977603_a779ff84590dc557902de5503c7e433a|" class="external text" rel="nofollow"><I>Molecular cell 12:365 - 72.</I></A></LI><LI>Mossakowska, Danuta E. Nyberg, Kerstin and Fersht, Alan R. (1989) Kinetic Characterization of the Recombinant Ribonuclease from Bacillus amyloliquefaciens (Barnase) and Investigation of Key Residues in Catalysis by Site-Directed Mutagenesis <A href="http://pubs.acs.org/doi/pdf/10.1021/bi00435a033|" class="external text" rel="nofollow"><I>Biochemistry 28: 3843 - 3850.</I></A>.
</LI><LI>Orth P. et al. (2000): Structual basis of gene regulation by the tetracycline inducible Tet repressor-operator system. In: <A href="http://life.nthu.edu.tw/~b871641/tetrepressor.pdf" class="external text" rel="nofollow">nature structural biology, volume 7 number 3</A>.
</LI><LI>Paddon, C. J. Vasantha, N. and Hartley, R. W. (1989) Translation and Processing of Bacillus amyloliquefaciens Extracellular Rnase <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC209718/pdf/jbacter00168-0575.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 171: 1185 - 1187.</I></A>.
</LI><LI>Voss, Carsten Lindau, Dennis and Flaschel, Erwin (2006) Production of Recombinant RNase Ba and Its Application in Downstream Processing of Plasmid DNA for Pharmaceutical Use <A href="http://onlinelibrary.wiley.com/doi/10.1021/bp050417e/pdf|" class="external text" rel="nofollow"><I>Biotechnology Progress 22: 737 - 744.</I></A>.
</LI><LI>Walsh, Christopher (1989) Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. <A href="http://www.jbc.org/content/264/5/2393.long|" class="external text" rel="nofollow"><I>Journal of biological chemistry 264: 2393 - 2396.</I></A></LI><LI>Wickstrum, J.R., Santangelo, T.J., and Egan, S.M. (2005) Cyclic AMP receptor protein and RhaR synergistically activate transcription from the L-rhamnose-responsive rhaSR promoter in Escherichia coli. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251584/?report=reader|" class="external text" rel="nofollow"><I>Journal of Bacteriology 187: 6708 â 6719.</I></A>.
</LI></UL><DIV id="asdf"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Genetic_Approach"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Genetic Approach</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Rhamnose_promoter_PRha"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Rhamnose promoter P<SUB>Rha</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Tetracyclin_repressor.2Foperator"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Tetracyclin repressor/operator</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Alanine_racemase_Alr"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Alanine racemase Alr</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Terminator"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Terminator</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#RNase_Ba_.28Barnase.29"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">RNase Ba (Barnase)</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Biosafety_system_TetOR_alive"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Biosafety system TetOR alive</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-9"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Characterization_of_the_tetracycline_tetO_operator"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Characterization of the tetracycline tetO operator</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Characterization_of_the_Biosafety-System_TetOR_alive"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Characterization of the Biosafety-System TetOR alive</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Conclusions"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_M&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_M&amp;oldid=358932" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>