<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Project_Cytochromes"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Project/Cytochromes">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Project/Cytochromes&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Cytochromes&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Cytochromes&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Project/Cytochromes" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Project/Cytochromes</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>Cytochromes</H1><DIV id="buttonrow" style="padding-top:30px; padding-bottom:70px; padding-left:45px; clear:both;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Projects Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes#Theory">Theory</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes#Genetic_Approach">Genetic Approach</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes#Results">Results</A></DIV></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 1:</B> Extracellular electron transfer via cytochromes in <I>E. coli</I> with a minimal set of proteins from <I>Shewanella oneidensins</I> MR-1.</DIV></DIV></DIV><P align="justify">
To enable transfer of electrons from the general metabolism to the outside of the cell, the <I>mtrCAB</I> operon from <A href="http://www.ncbi.nlm.nih.gov/genome/1082?project_id=57949" class="external text" rel="nofollow"><I>Shewanella oneidensis</I> MR-1</A> was heterologously expressed in <I>E. coli</I>. This operon encodes for a minimal set of genes required to build an electron shuttle pathway via different c-type cytochromes. Electrons from the native <I>E. coli</I> protein <A href="http://www.ncbi.nlm.nih.gov/protein/EGT66377.1" class="external text" rel="nofollow">NapC</A> are passed to the periplasmic <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A>, which transports them to the outer membrane protein <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A>. Via the membrane-bound <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> the corresponding electrons can be transferred to extracellular electron acceptors like the anode of a microbial fuel cell. 

The <I>mtrCAB</I> cluster contains two illegal restriction sites, which were removed by generating a silent mutation via site-directed mutagenesis. The resulting three fragments were combined and ligated with <A href="http://parts.igem.org/Part:pSB1C3" class="external text" rel="nofollow">pSB1C3</A> by <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>. Subsequently, this gene cluster was combined with three different promoters and ribosome binding sites of varying strength. It has not yet been possible to experimentally verify the functional expression of the cytochromes. Furthermore, it was ineffectual attempted to clone the <I>ccmAH</I> cluster as well. The Ccm proteins are required for the maturation of the decaheme cytochromes <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> and are natively not expressed under aerobic conditions.  This is, however, a minor issue, since the Microbial Fuel Cell will work under anareobic conditions. 

For a correct heme insertion into the decaheme cytochromes <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A>, the cytochrome c maturation machinery is required. The corresponding genes are naturally expressed in <I>E. coli</I> under anaerobic conditions, for aerobic expression they have to be expressed via plasmid.
</P><H2><SPAN class="mw-headline" id="Theory">Theory</SPAN></H2><P align="justify">
Cell membranes work as a natural insulator and prevent the flow from electrons out of the cell. To enable transfer of electrons from the general metabolism to the outside of the cell we had to alter the membrane of our organism <I>E. coli</I> without affecting cell growth, membrane stability or cell metabolism. 

Some species from the genera <I>Shewanella</I> and <I>Geobacter</I> have developed different mechanisms to allow extracellular electron transfer. Members of the <I>Shewanella</I> species are the Gram-negative γ-proteobacteria which are known for their respiratory versatility. They are reported to be using over 20 terminal electron acceptors for respiration (Nealson et al. 2003).<I> Shewanella oneidensis</I> MR-1 expresses many c-type cytochromes, membrane-bound redox-active proteins or soluble periplasmatic proteins, which play an important in the electron transport in the bacterial respiration and photosynthesis.

There have been already numerous approaches, where the Mtr proteins were introduced into <I>E.coli</I> strains and could be successfully expressed under anaerobic conditions (Goldbeck et al. 2012)
</P><H2><SPAN class="mw-headline" id="Genetic_Approach">Genetic Approach</SPAN></H2><H3><SPAN class="mw-headline" id="Isolation_of_the_mtrCAB_gene_cluster">Isolation of the <I>mtrCAB</I> gene cluster</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure2:</B> The genomic region of the <I>mtrCAB</I> operon is shown.<A href="http://ncbi.nlm.nih.gov" class="external text" rel="nofollow">(NCBI)</A>.</DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:552px;"><DIV class="thumbcaption"><B>Figure3:</B> Graphic representation of the illegal restriction sites within the <I>mtrCAB</I> operon and the resulting fragments from the site-directed mutagenesis approach. The colored overlaps indicate homologous sequences,that facilitate the religation via <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>. The parts labeled 'pSB1C3 prefix' and 'pSB1C3 suffix' are overlaps homologous to the vectors prefix and suffix, respectivly.</DIV></DIV></DIV><P align="justify" style="clear:both;">
As shown in Figure2, the three <I>mtr</I> genes are organized in an operon in the donor organism <I>Shewanella oneidensis</I> MR-1 and could therefore amplified altogether. Since two illegal PstI restriction sites exist in this cluster it had to be subdivided into three fragments. The illegal restriction sites were subsequently deleted by incorporating a silent mutation via site-directed mutagenesis. Eventually the fragments should be joined back together and ligated with the shipping backbone &lt;bbpart&gt;pSB1C3&lt;/bbpart&gt; via <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>.

This approach required specifically designed primers that can bind the targeted genetic sequence and hold an overlap to facilitate ligation with the other fragments and the backbone. Hence, eight primers were designed, two for each fragment of the <I>mtrCAB</I> operon and two for the &lt;bbpart&gt;pSB1C3&lt;/bbpart&gt; vector to generate proper backbone-overlaps as well. The resulting fragments are depicted in Figure 3.  The colored overlaps indicate homologous sequences that facilitate the re-ligation via <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A>. The parts labeled 'pSB1C3 prefix' and 'pSB1C3 suffix' are overlaps homologous to the vectors prefix and suffix, respectively. The genes and resulting fragments are listed in the table below. The fragment sizes incorporate the aforementioned primer overlaps of 20 bp each.
</P><TABLE class="wikitable" style="margin-left:20px;"><TBODY><TR><TH>Device
</TH><TH>Regulatory Part
</TH><TH>Promoter
</TH><TH>RBS
</TH><TH>Activity
</TH></TR><TR><TD>&lt;bbpart&gt;BBa_K1172403&lt;/bbpart&gt;</TD><TD>&lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;</TD><TD>Anderson 0.33
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0032" class="external text" rel="nofollow">medium</A></TD><TD>medium
</TD></TR><TR><TD>&lt;bbpart&gt;BBa_K1172404&lt;/bbpart&gt;</TD><TD>&lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;</TD><TD>Anderson 0.77
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034" class="external text" rel="nofollow">strong</A></TD><TD>strong
</TD></TR><TR><TD>&lt;bbpart&gt;BBa_K1172405&lt;/bbpart&gt;</TD><TD>&lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;</TD><TD>T7 induced
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034" class="external text" rel="nofollow">strong</A></TD><TD>very strong
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Promoter_strength">Promoter strength</SPAN></H3><P align="justify">
Due to the complex interaction of the Mtr proteins with both, themselves and the Ccm system the transcriptional balance is crucial for the complex formation. Previous works proved that only a small range for optimal MtrCAB expression exists. Most important, the synthesis of the mature holocytochromes <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> actually decreases at a high corresponding promoter activity. A possible explanation for this is the low reaction speed of the Ccm machinery which cannot cope with a high level of apocytochromes. This results in an accumulation of inmature apocytochromes which are then degraded by periplasmatic proteases. Furthermore, a too low level of Ccm proteins inhibits the maturation of the apocytochromes, whereas expression above a certain treshold decreases the overall efficiency of the Ccm system.

Since the genes are heterologously expressed on a high copy plasmid, the promoter strength plays a key role in a controlled and therefore successfull expression. To cover a wide range of expression levels three different promoters, with varying strength and ribosome binding sites were combined with the genes. An overview is given in the table below.
</P><TABLE class="wikitable" style="margin-left:20px;"><TBODY><TR><TH>Part
</TH><TH>Promoter
</TH><TH>RBS
</TH><TH>Activity
</TH></TR><TR><TD>&lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;</TD><TD>T7 uninduced
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034" class="external text" rel="nofollow">strong</A></TD><TD>weak (basal)
</TD></TR><TR><TD>&lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;</TD><TD>Anderson 0.33
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0032" class="external text" rel="nofollow">medium</A></TD><TD>medium
</TD></TR><TR><TD>&lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;</TD><TD>Anderson 0.77
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034" class="external text" rel="nofollow">strong</A></TD><TD>strong
</TD></TR><TR><TD>&lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;</TD><TD>T7 induced
</TD><TD><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_B0034" class="external text" rel="nofollow">strong</A></TD><TD>very strong
</TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Protein_Overview">Protein Overview</SPAN></H3><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Figure 4:</B> Extracellular electron transfer via cytochromes in <I>E. coli</I> with a minimal set of genes from <I>Shewanella oneidensins</I> MR-1.</DIV></DIV></DIV><P align="justify"><I>Shewanella oneidensis</I> MR-1 expresses many c-type cytochromes, membrane-bound redox-active proteins or soluble periplasmatic proteins, which play an important in the electron transport in the bacterial respiration and photosynthesis. One of the electron-transfer models is based on c-type cytochromes encoded by the <I>mtrCAB</I> gene cluster (Myers and Myers, 1997). Although additional proteins like OmcA and NapC are involved in the electron transport chain, a minimal set of the three proteins <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A>, <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A> is necessary to heterologously express this pathway in <I>E. coli</I> (Jensen et al. 2010). 

The process of the electron transport is illustrated in Figure 4. NapC ist natively expressed in <I>E.coli</I> and interacts very well with the proteins from <I>S. oneidensis</I>. The inner membrane cytochrome c NapC accepts electrons generated by the reduction of menaquinone to menaquinol. The electrons are then passed from NapC to the periplasmic decaheme cytochrome c <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> in the periplasm.  MtrA transports them to <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A>, a 72-kD β-barrel outer membrane protein which is physically connected with <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A>, a 69-kDa membrane-bound decaheme cytochrome c. From <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> the corresponding electrons can be transferred to extracellular electron acceptors such as iron oxide or, as in our case, an anode in a Microbial Fuel Cell. (Meyers and Meyers 2004)

The CcmAH complex composed of the eight membrane proteins CcmABCDEFGH is essential for the biosynthesis of the decaheme cytochromes <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A>. The system first transports the heme into the periplasm and catalyzes the formation of thioester bonds that link the heme to two cysteine residues in the apocytochrome. Afterwards, the axial ligands are located towards the heme and the holocytochrome is folded (Sanders et al. 2010). Naturally the Ccm proteins are expressed by <I>E. coli</I> solely under anaerobic conditions, which corresponds to our anaerobically operating Microbial Fuel Cell.
</P><TABLE class="wikitable" style="margin-left:20px; width:600px;"><TBODY><TR><TH style="width:25%;"><A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A></TH><TH style="width:25%;"><A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A></TH><TH style="width:25%;"><A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A></TH><TH style="width:25%;">CcmAH
</TH></TR><TR><TD>outer membrane decaheme type c cytochrome
</TD><TD>periplamatic decaheme type c cytochrome
</TD><TD>28 strand β-barrel membrane protein
</TD><TD>cytochrome c maturation machinery
</TD></TR><TR><TD>671 aa
</TD><TD>333 aa
</TD><TD>697 aa
</TD><TD><I>various</I></TD></TR><TR><TD>69 kDa
</TD><TD>32 kDa
</TD><TD>72 kDa
</TD><TD><I>various</I></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Difficulties">Difficulties</SPAN></H3><UL><LI><A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> seems to be necessary for the stability of <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A> (Schicklberger et al. 2011)
</LI><LI><A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A> itself is required for the correct location and incorporation of <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> (Meyers and Meyers 2002) and is involved in the interaction between <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> (Beliaev AS et al. 1998). 
</LI><LI>Furthermore extensive postranslational processing is required for correct incorporation of all hemes, folding, and localisation of the cytochromes. Considering these difficulties the expression of even a single cytochrome is a major challenge (Jensen et al. 2010). 
</LI></UL><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Genetics">Genetics</SPAN></H3><UL><LI>The amplification of the <I>mtrCAB</I> cluster from the genome of <I>S. oneidensis</I> and the deletion of illegal restriction sites was accomplished.
</LI><LI>The gene cluster was succesfully ligated into the shipping vector &lt;bbpart&gt;pSB1C3&lt;/bbpart&gt; forming the BioBrick &lt;bbpart&gt;BBa_K1172401&lt;/bbpart&gt;</LI><LI>The gene cluster was ligated into three expression vectors with varying promoter strength forming the following devices
</LI><LI>&lt;bbpart&gt;BBa_K1172403&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172404&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172405&lt;/bbpart&gt;</LI></UL><H3><SPAN class="mw-headline" id="Characterisation">Characterisation</SPAN></H3><UL><LI>To prove the expression of the Mtr proteins, <I>E. coli</I> cultures were transformed transformed with &lt;bbpart&gt;BBa_K1172403&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172405&lt;/bbpart&gt; (uninduced) and &lt;bbpart&gt;BBa_K1172401&lt;/bbpart&gt; as a control and then cultivated anaerobically. Membrane and periplasmatic fractions of the cells were isolated by <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Cold_osmotic_shock" class="external text" rel="nofollow">Cold osmotic shock fractioning.</A> and analyzed by <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" class="external text" rel="nofollow">SDS-PAGE</A>; the results are shown in Figure 5. 
</LI><LI>The membrane fraction should contain <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717385.1" class="external text" rel="nofollow">MtrB</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> of the Mtr complex, with sizes of 72 kDa and 69 kDa, respectively.
</LI><LI>The periplasmatic fraction should only contain the <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> protein of the Mtr complex (32 kDa). 
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5:</B> Image of a SDS-PAGE with periplasmatic and membrane fractions of colonies transformed with &lt;bbpart&gt;K1172401&lt;/bbpart&gt; as a control, besides &lt;bbpart&gt;K1172403&lt;/bbpart&gt; and &lt;bbpart&gt;K1172405&lt;/bbpart&gt; (undinduced) for analysis. Ladder: PageRuler unstained Protein Ladder (Fermentas)</DIV></DIV></DIV></DIV><UL><LI>It was not possible to confirm the expression of the Mtr proteins by <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" class="external text" rel="nofollow">SDS-PAGE</A> analysis, as no significant differences between control and experimental condition were visible at the relevant heights of the SDS-PAGE.
</LI></UL><UL><LI>The redox activity of <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717386.1" class="external text" rel="nofollow">MtrA</A> and <A href="http://www.ncbi.nlm.nih.gov/protein/NP_717387.1" class="external text" rel="nofollow">MtrC</A> was probed via absorption spectroscopy and did not show significant indications for a positive result as the expected shift of the Soret peak from 410 nm to 420 nm could not be observed.
</LI></UL><H3><SPAN class="mw-headline" id="Conclusion">Conclusion</SPAN></H3><UL><LI>The electron transfer system from <I>Shewanella oneidensis</I> MR-1 seems to be suitable for the usage in our Microbial Fuel Cell in principle. However, expression regulation, heme-loading, and correct folding as well as localization of the cytochromes are very complex and therefore our team was not able to produce the functional system in the available time.
</LI></UL><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Beliaev AS, Saffarini DA (1998)Shewanella putrefaciens mtrB encodes an outer membrane protein required for Fe(III) and Mn(IV) reduction. <A href="http://jb.asm.org/content/180/23/6292.short" class="external text" rel="nofollow"><I>J Bacteriol 180:6292–6297</I></A></LI></UL><UL><LI>Beliaev, A. S., D. A. Saffarini, J. L. McLaughlin, and D. Hunnicutt. 2001. MtrC, an outer membrane decahaem c cytochrome required for metal reduction in Shewanella putrefaciens MR-1. <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.2001.02257.x/full" class="external text" rel="nofollow"><I>Mol. Microbiol. 39:722-730</I></A></LI></UL><UL><LI>Grove, J., Tanapongpipat, S., Thomas, G., Griffiths, L., Crooke, H., and Cole, J. (1996)Escherichia coliK-12 genes essential for the synthesis of c-type cytochromes and a third nitrate reductase located in the periplasm. <A href="http://onlinelibrary.wiley.com/doi/10.1046/j.1365-2958.1996.383914.x/abstract" class="external text" rel="nofollow"><I>Mol. Microbiol. 19, 467−481</I></A></LI></UL><UL><LI>Jensen HM, Albers AE, Malley KR, Londer YY, Cohen BE, et al. (2010) Engineering of a synthetic electron conduit in living cells. <A href="http://www.pnas.org/content/107/45/19213.short" class="external text" rel="nofollow">Proc Natl Acad Sci USA. 10.1073/pnas.1009645107 <I>Proc. Natl Acad. Sci. USA 107, 19213–19218 (2010)</I>.</A></LI></UL><UL><LI>Myers CR, Myers JM (2002) MtrB is required for proper incorporation of the cytochromes OmcA and OmcB into the outer membrane of Shewanella putrefaciens MR-1. <A href="http://aem.asm.org/content/68/11/5585.short" class="external text" rel="nofollow"><I>Appl Environ Microbiol68:5585–5594</I></A></LI></UL><UL><LI>Sanders, C., Turkarslan, S., Lee, D.-W., and Daldal, F. (2010) Cytochromecbiogenesis: the Ccm system. <A href="http://www.sciencedirect.com/science/article/pii/S0966842X10000442" class="external text" rel="nofollow"><I>Trends Microbiol. 18, 266−274</I></A></LI></UL><UL><LI>Schicklberger, M., Bucking, C., Schuetz, B., Heide, H., and Gescher, J. (2011) Involvement of the Shewanella oneidensis decaheme cytochrome MtrA in the periplasmic stability of the beta-barrel protein MtrB. <A href="http://aem.asm.org/content/77/4/1520.short" class="external text" rel="nofollow"><I>Appl. Environ. Microbiol. 77, 1520−1523</I></A></LI></UL><UL><LI>Thony-Meyer L, Fischer F, Kunzler P, Ritz D, Hennecke H (1995)Escherichia coligenes required for cytochrome c maturation. <A href="http://jb.asm.org/content/177/15/4321.short" class="external text" rel="nofollow"><I>J Bacteriol 177:4321–4326</I></A></LI></UL></DIV><DIV id="asdf"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Theory"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Theory</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Genetic_Approach"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Genetic Approach</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Isolation_of_the_mtrCAB_gene_cluster"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Isolation of the mtrCAB gene cluster</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Promoter_strength"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Promoter strength</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Protein_Overview"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Protein Overview</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Difficulties"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Difficulties</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-8"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Genetics"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Genetics</SPAN></A></LI><LI class="toclevel-2 tocsection-10"><A href="#Characterisation"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Characterisation</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Conclusion"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Conclusion</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">http://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Project/Cytochromes" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Project/Cytochromes" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Cytochromes&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Project/Cytochromes&amp;oldid=360062" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>