<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Biosafety_Biosafety_System_S"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">Page               </A></LI><LI><A href="/Talk:Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_S&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_S&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Biosafety/Biosafety_System_S" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Biosafety/Biosafety System S</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>Biosafety System AraCtive</H1><DIV id="buttonrow" style="padding-top:30px; padding-bottom:70px; padding-left:45px; clear:both;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">System Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Genetic_Approach">Genetic Approach</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Results">Results</A></DIV></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Biosafety-System AraCtive</B>.</DIV></DIV></DIV><P align="justify">
The Biosafety-System araCtive &lt;bbpart&gt;BBa_K1172909&lt;/bbpart&gt; is an improvement of the BioBrick &lt;bbpart&gt;BBa_K914014&lt;/bbpart&gt; by replacing the first promoter by the rhamnose promoter P<SUB><I>Rha</I></SUB>, integration of the alanine racemase &lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt; and utilization of the regulator AraC to control the transcription of the Barnase behind the P<SUB><I>BAD</I></SUB> promoter. Because of the tight repression of this promoter, this system has the lowest basal transcription of the Barnase and is therefore the most active and attractive one.
</P><H2><SPAN class="mw-headline" id="Genetic_Approach">Genetic Approach</SPAN></H2><H3><SPAN class="mw-headline" id="Rhamnose_promoter_PRha"><B>Rhamnose promoter P<SUB>Rha</SUB></B></SPAN></H3><P align="justify">
The promoter P<SUB><I>Rha</I></SUB> (&lt;bbpart&gt;BBa_K914003&lt;/bbpart&gt;) naturally regulates  the catabolism of the hexose L-rhamnose. The advantage to use this promoter is its solely positive regulation. The regulon consists of the gene <I>rhaT</I> for the  L-rhamnose transporter and the two operons <I>rhaSR</I> and <I>rhaBAD</I>.
The operon <I>rhaSR</I> encodes the two transcriptional activators RhaS and RhaR, who are responsible for the positive activation of the L-rhamnose catabolism, while the operon <I>rhaBAD</I> encodes the genes for the direct catabolism of L-rhamnose.
When L-rhamnose is present, it acts as an inducer by binding to the regulatory protein RhaR. RhaR regulates its own expression and the expression of the regulatory gene <I>rhaS</I> by repressing or, in the presence of L-rhamnose, activating the operon <I>rhaSR</I>. Normally, the expression level is modest, but it can be enhanced by a higher level of intracellular cAMP, which increases in the absence of glucose. So, in the presence of L-rhamnose and a high concentration of intracellular cAMP, the activator protein RhaR is expressed on higher level, resulting in an activation of the promoter P<SUB><I>rhaT</I></SUB> for an efficient L-rhamnose uptake and an activation of the operon <I>rhaBAD</I>. The L-rhamnose is than broken down into dihydroxyacetone phosphate and lactate aldehyde by the enzymes encoded by <I>rhaBAD</I> (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Wickstrum <I>et al.</I>, 2005</A>).
A brief schematic summary of the regulation is shown in Figure 1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1:</B> The catabolism of L-rhamnose in <I>E. coli</I> is turned off in general but inducible by L-rhamnose. The induction activates the transcription of the genes <I>rhaS</I> and <I>rhaR</I>, which regulate the L-rhamnose catabolism by positive activation of the rhamnose uptake (<I>rhaT</I>) and its metabolization (<I>rhaBAD</I>).</DIV></DIV></DIV></DIV><P align="justify">
Dihydroxyacetone phosphate can be metabolized in the glycolysis pathway, while lactate aldehyde is oxidized to lactate under aerobic conditions and reduced to L-1,2,-propandiol under anaerobic conditions.
The degradation of L-rhamnose can be separated in three steps. In the first step the L-rhamnose is turned into L-rhamnulose by an isomerase (gene <I>rhaA</I>). L-rhamnulose is in turn phosphorylated to L-rhamnulose-1-phosphate by a kinase (gene <I>rhaB</I>) and the sugar phosphate is finally hydrolyzed by an aldolase (gene <I>rhaD</I>) to dihydroxyacetone phosphate and lactate aldehyde (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Baldoma <I>et al.</I>, 1988</A>).
For our Safety-System araCtive, the rhamnose promoter P<SUB>Rha</SUB> is used to control the expression of the repressor AraC and the essential alanine racemase, because this promoter has an even lower basal transcription then the arabinose promoter P<SUB><I>BAD</I></SUB>. This is needed to tightly repress the expression of the alanine racemase (<I>alr</I>) and thereby take  advantage of the double-kill switch. Although the rhamnose promoter P<SUB><I>Rha</I></SUB> is characterized by a very low basal transcription it can not be used for the control of the toxic RNase Ba (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#RNase_Ba_.28Barnase.29" class="external text" rel="nofollow">Barnase</A>), which ist needed for the alanine racemase (<I>alr</I>), the other part of the double-kill switch. In conclusion this promoter is the best choice for the first part of the Biosafety-System, as it is tightly repressed in the absence of L-rhamnose, but activated in its presence.</P><H3><SPAN class="mw-headline" id="Regulator_AraC"><B>Regulator AraC</B></SPAN></H3><P align="justify">
The <I>araC</I> gene is naturally found in <I>E. coli</I> and is coding for the 32 kDa AraC protein, which regulates the expression of the genes required for the uptake and catabolism of the pentose L-arabinose. The genes for the catabolism of arabinose are under the control of the arabinose promoter P<SUB><I>BAD</I></SUB>, which is both positively and negatively regulated by AraC. Naturally in the presence of L-arabinose the expression of those genes is activated, while it is repressed in its absence. In addition the AraC protein regulates its own transcription under the control of the so called P<SUB>C</SUB> promoter. Compared to the P<SUB><I>BAD</I></SUB> promoter the P<SUB><I>C</I></SUB> promoter and the associated <I>araC</I> gene are therefore transcribed in opposite direction (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Schleif, 2010</A>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 2:</B> Positive regulation of the arabinose promoter P<SUB><I>BAD</I></SUB>. When L-arabinose is present the dimer-structure of AraC is relaxed and the AraC protein functions as activator. The activation can additionally be enhanced by the CRP-cAMP-complex.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 3:</B> Negative regulation of the arabinose promoter P<SUB><I>BAD</I></SUB>. In the absence of L-arabinose, AraC functions as repressor by forming a dimer, which inhibits the binding of the RNA-polymerase.</DIV></DIV></DIV></DIV><P align="justify">
Naturally in the absence of L-arabinose the AraC protein is found in its repressor conformation, binding to the O<SUB>2</SUB>-region and I<SUB>1</SUB>-region. Binding of two AraC proteins to these domains results in a  protein-protein-interaction of AraC forming a dimer. This AraC dimer is responsible for the tight repression of the P<SUB><I>C</I></SUB> and the P<SUB>BAD</SUB> promoter by forming a DNA-loop to inhibit the initiation of the RNA-polymerase (see Figure 3 above).
When L-arabinose is present in the media the operon changes to the active form. Following this conditions the AraC protein functions as activator by binding to the  I<SUB>1</SUB>- and I<SUB>2</SUB>-region of the operator region. This causes transcription of the genes behind the P<SUB><I>BAD</I></SUB> and the P<SUB><I>C</I></SUB> promoter. Depending on the intracellular level of cAMP the transcription of both the genes behind the promoters P<SUB><I>BAD</I></SUB> and P<SUB><I>C</I></SUB> can even be enhanced. In the absence of glucose the cell reaches a high level of cAMP as a signal molecule. cAMP binds as the typically CRP-cAMP complex binds in front of the AraC regulator and increases the binding affinity of the RNA-polymerase, so that the genes behind the P<SUB><I>BAD</I></SUB> promoter are expressed on higher levels. Besides the transcription affinity of the P<SUB><I>BAD</I></SUB> promoter, the AraC protein also regulates the genes <I>araE</I> and the operon consisting the genes <I>araFGH</I>, which are necessary for an efficient uptake of L-arabinose (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Cass, 1988</A>).
The auto-regulation of the <I>araC</I> gene itself works both in the presence and absence of L-arabinose. In presence of L-arabinose, the DNA-loop which prevents also the transcription of the P<SUB><I>BAD</I></SUB> promoter also inhibits the transcription of <I>araC</I>. When L-arabinose is present and the AraC concentration is high enough, it auto-regulates its own transcription by dimerization between the <I>araO</I><SUB>1</SUB>- and the <I>araO</I><SUB>2</SUB>-region. This leads to an other DNA-loop, inhibiting solely the transcription of the genes behind the P<SUB><I>C</I></SUB> promoter (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Hamilton, 1988</A>).
For our Biosafety-System we decided to use the arabinose P<SUB><I>BAD</I></SUB> promoter, because this promoter is tightly regulated by AraC and shows no basal transcription in the complete absence of AraC. Additionally this promoter is repressed by glucose and basal transcription is activated in the presence of cAMP.</P><H3><SPAN class="mw-headline" id="Alanine_racemase_Alr"><B>Alanine racemase Alr</B></SPAN></H3><P align="justify">
The alanine racemase Alr (EC 5.1.1.1) from the Gram-negative enteric bacteria <I>Escherichia coli</I> is a racemase, which catalyses the reversible conversion of L-alanine into the enantiomer D-alanine (see Figure 4). For this reaction, the cofactor pyridoxal-5'-phosphate (PLP) is necessary. The constitutively expressed alanine racemase (<I>alr</I>) is naturally responsible for the accumulation of D-alanine. This compound is an essential component of the bacterial cell wall, because it is used for the cross-linkage of peptidoglycan (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Walsh, 1989</A>).
The usage of D-alanine instead of a typically L-amino acid prevents cleavage by peptidases. However, a lack of D-alanine causes to a bacteriolytic characteristics. In the absence of Dâalanine dividing cells will lyse rapidly. This fact is used for our Biosafety-Strain, a D-alanine auxotrophic mutant (K-12 â<I>alr</I> â<I>dadX</I>). The Biosafety-Strain grows only with a plasmid containing the alanine racemase (&lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;) to complement the D-alanine auxotrophy. Consequently the alanine racemase is essential for bacterial cell division. This approach guarantees a high plasmid stability, which is extremely important when the plasmid contains a toxic gene like the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#RNase_Ba_.28Barnase.29" class="external text" rel="nofollow">Barnase</A>. In addition this construction provides the possibility for the implementation of a double kill-switch system. Because if the expression of the alanine racemase is repressed and there is no D-alanine supplementation in the medium, cells will not grow.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:355px;"><DIV class="thumbcaption"><B>Figure 4:</B> The alanine racemase (&lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;) from <I>E. coli</I> catalyses the reversible conversion from L-alanine to D-alanine. For this isomerization the cofactor pyridoxal-5'-phosphate is necessary.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Terminator"><B>Terminator</B></SPAN></H3><P align="justify">
Terminators are essential to terminate transcription of an operon. In procaryotes,  two types of terminators exist. The rho-dependent and the rho-independent terminator. Rho-independent terminators are characterized by their stem-loop forming sequence. In general, the terminator-region can be divided into four regions. The first region is GC-rich and constitutes one half of the stem. This region is followed by the loop-region and another GC-rich region that makes up the opposite part of the stem. The terminator closes with a poly uracil region, which destabilizes the binding of the RNA-polymerase. The stem-loop of the terminator causes a distinction of the DNA and the translated RNA. Consequently the binding of the RNA-polymerase is cancelled and the transcription ends after the stem-loop (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Carafa <I>et al.</I>, 1990</A>).
For our Bioafety-System araCtive the terminator is necessary to avoid that the expression of the genes under control of the rhamnose promoter P<SUB><I>Rha</I></SUB>, like the Repressor AraC and the alanine racemase (<I>alr</I>) results in the transcription of the genes behind the arabinose promoter P<SUB><I>BAD</I></SUB>, which contains the toxic Barnase &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; and would lead to cell death.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 5:</B> Stem-loop structure of the terminator &lt;bbpart&gt;BBa_B0015&lt;/bbpart&gt;, which is used for the Biosafety-System araCtive. The terminator is used to make sure that solely both the repressor AraC and the alanine racemase Alr are expressed but the transcription of the toxic RNase Ba (Barnase) is avoided.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Arabinose_promoter_PBAD"><B>Arabinose promoter P<SUB>BAD</SUB></B></SPAN></H3><P align="justify">
The arabinose promoter P<SUB><I>BAD</I></SUB> controls the expression of the genes, which are necessary for the catabolism of the pentose arabinose. The expression is regulated by the bifunctional regulator AraC. The genes of this operon convert the pentose to xylose, which can be converted into fructose-1,6-diphosphate and subsequently be fed into glycolysis. The conversion of L-arabinose is realized in three steps. First the isomerase (AraA) catalyzes the reaction from L-arabinose to the isomer L-ribulose. In the second step the ribulose kinase (AraB) phosphorylates the L-ribulose to L-ribulose-phosphate. For this reaction ATP is needed. Last, the L-ribulose-phosphate is converted to D-xylose-5-phosphat by L-ribulose-5-phosphate-epimerase (AraD, <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Schleif, 2010</A>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6:</B> Overview of the arabinose regulon from <I>E. coli</I>. The transcription of the genes <I>araBAD</I>, which are coding for the enzymes of the L-arabinose catabolism are positively induced and negatively repressed by the AraC protein (depending on the presence of arabinose),  which additionally autoregulates his own transcription.</DIV></DIV></DIV></DIV><P align="justify">
For our Safety-System we used the arabinose promoter P<SUB>BAD</SUB> for the regulation of the toxic gene product Barnase. As this promoter has very low basal transcription and as the expression of the genes behind this promoter is strict depending on the presence of AraC for activating the transcription, it is possible to use the arabinose promoter P<SUB>BAD</SUB> to control the expression of a toxic gene product without causing cell death.</P><H3><SPAN class="mw-headline" id="RNase_Ba_.28Barnase.29"><B>RNase Ba (Barnase)</B></SPAN></H3><P align="justify">
The Barnase (EC 3.1.27) is a 12 kDa extracellular microbial ribonuclease, which is naturally found in the Gram-positive soil bacteria <I>Bacillus amyloliquefaciens</I> and consists of a single chain of 110 amino acids. The Barnase (RNase Ba) catalyses the cleavage of single stranded RNA, preferentially behind Gs. In the first step of the RNA-degradation a cyclic intermediate is formed by transesterification and afterwards this intermediate is hydrolyzed yielding in a 3'-nucleotide (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Mossakowska <I>et al.</I>, 1989</A>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:542px;"><DIV class="thumbcaption"><B>Figure 7:</B> Enzymatic reaction of the RNA-cleavage by the RNase Ba. First the transesterification by the Glu-73 residue is performed and then this cyclic intermediate is hydrolyzed by the His-102 of the Barnase.</DIV></DIV></DIV></DIV><P align="justify">
In <I>Bacillus amyloliquefaciens</I>, the activity of the Barnase (RNase Ba) is inhibited intracellular by an inhibitor called barstar. Barstar consists of only 89 amino acids and binds with a high affinity to the toxic Barnase. This prevents the cleavage of the intracellular RNA in the host organism (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Paddon <I>et al.</I>, 1989</A>). Therefore the Barnase normally acts only outside the cell and is translocated under natural conditions. For the Biosafety-System araCtive we modified the enzyme by cloning only the sequence responsible for the cleavage of the RNA, leaving out the N-terminal signal peptide part.

As shown in Figure 8 below, the transcription of the DNA, which encodes the Barnase produces a 474 nt RNA. The translation of the RNA starts about 25 nucleotides downstream from the transcription start and can be divided into two parts. The first part (colored in orange) is translated into a signal peptide at the amino-terminus of the Barnase coding RNA. This part is responsible for the extracellular translocation of the RNase Ba, while the peptide sequence for the active Barnase starts 142 nucleotides downstream from the transcription start (colored in red).
For the Biosafety-System araCtive, we only used the part (&lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt;) of the Barnase encoding the catalytic domain without the extracellular translocation signal of the toxic gene product. Translation of the barnase gene leads to rapid cell death if the expression of the Barnase is not repressed by the repressor AraC of our Biosafety-System.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 8:</B> Sequence of the signal peptide amino terminal of the RNase Ba (Barnase). The Biobrick &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; does not contain the signal sequence for the extracellular translocation, but only the coding sequence for the mature enzyme.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Biosafety-System_araCtive"><B>Biosafety-System araCtive</B></SPAN></H3><P align="justify">
Combining the gene described above with the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> results in a powerful device, allowing us to control the bacterial cell division. The control of the bacterial growth is possible either active or passive. Active by inducing the P<SUB><I>BAD</I></SUB> promoter with L-arabinose and passive by the induction of L-rhamnose. The passive control makes it possible to control the bacterial cell division in a defined closed environment, like the MFC, by continuously adding L-rhamnose to the medium. As shown in the Figure 9 below, this leads to an expression of the essential alanine racemase (<I>alr</I>) and the AraC regulator, so that the expression of the RNase Ba is repressed. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 9:</B> Biosafety-System araCtive in the presence of L-rhamnose. The essential alanine racemase (Alr) and the repressor AraC are expressed, resulting in a repression of the expression of the RNAse Ba. Consequently the bacteria show normal growth behaviour.</DIV></DIV></DIV></DIV><P align="justify">
In the event that bacteria exit the defined environment of the MFC or L-rhamnose is not added to the medium any more, both the expression of the alanine racemase (Alr) and the AraC regulatordecrease, so that the expression of the toxic RNase Ba (Barnase) begins. The cleavage of the intracellular RNA by the Barnase and the lack of synthesized D-alanine, caused by the repressed alanine racemase inhibit the cell division and makes sure that the bacteria can only grow in the defined environment or the device of choice respectively. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 10:</B> Active Biosafety-System araCtive outside of a defined environment lacking L-rhamnose. Both the expression of the alanine racemase (Alr) and AraC repressor are reduced and ideally completely shutdown. In contrast, the expression of the RNase ba (Barnase) is turned on, leading to cell death by RNA cleavage.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Characterization_of_the_arabinose_promoter_PBAD"><B>Characterization of the arabinose promoter P<SUB><I>BAD</I></SUB></B></SPAN></H3><P align="justify">
First of all, the arabinose promoter P<SUB><I>BAD</I></SUB> was characterized to get a first impression of its basal transcription rate. Therefore the bacterial growth was investigated under the pressure of the unrepressed P<SUB><I>BAD</I></SUB> promoter on different carbon source using M9 minimal medium with either glucose or glycerol. The transcription rate was determined by fluorescence measurement of GFP &lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt; transcribed via the P<SUB><I>BAD</I></SUB> promoter using the BioBrick &lt;bbpart&gt;BBa_I13541&lt;/bbpart&gt;.

As shown in Figure 12 below, the bacteria grew better on glucose than on glycerol. This is caused by the fact that glucose is the better energy source of this two, because glycerol enters glycolysis at a later step and therefore delivers less energy (in addition to the additional ATP consumption to drive glycerol uptake). For the investigation of the basal transcription the fluorescence measurements, shown in Figure 13, is more interesting. 
Both the wild type <I>E. coli</I> K-12 and the strain <I>E. coli</I> K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I> show about the same fluorescence on glucose (red and orange curve) but differ on glycerol. This can be explained by the fact that growth on glucose does not cause an additional induction of the arabinose promoter P<SUB><I>BAD</I></SUB>, while glycerol does. In the presence of glucose, the intracellular concentration of cAMP is low. The absence of cAMP results in inactive CAP (carbon utilization activator protein) and thus no induction of inefficient catabolisms like the catabolism of arabinose. Therefore, glucose is catabolized by the bacteria first, resulting in an optimal growth. In the absence of glucose, the cAMP level increases which enhances via CAP-cAMP the transcription of most operons encoding enzymes for the catabolism of alternative carbon sources. Therefore, the expression of GFP under the control of the P<SUB>BAD</SUB> promoter increases on glycerol. Another difference can be seen between the wild type (black curve) and the strain K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I> (blue curve). 
The Biosafety-Strain shows lower expression then the wild type, but has about the same growth rate, according to Figure 11. The reason for this characteristic is caused by the deleted AraC protein in the Biosafety-Strain. Because the AraC protein functions not only as are repressor but also as an activator the transcription rate decreases in the Biosafety-Strain.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 11:</B> Characterization of the bacterial growth of the strain K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I> containing the plasmid &lt;bbpart&gt;BBa_I13541&lt;/bbpart&gt; with GFP (&lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt;) under the control of the P<SUB><I>BAD</I></SUB> promoter. The M9 medium was supplemented with 5 mM D-alanine. It could be demonstrated, that the bacteria grow faster on M9 minimal medium containing glucose than on M9 minimal medium with glycerol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 12:</B> Characterization of the fluorescence of the strain K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I> containing the plasmid &lt;bbpart&gt;BBa_I13541&lt;/bbpart&gt; with GFP (&lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt;) under control of the P<SUB><I>BAD</I></SUB> promoter. The Biosafety-Strain was cultivated on M9 minimal medium with 5 mM D-alanine supplemented.</DIV></DIV></DIV></DIV><P align="justify">
The effect that glucose represses the transcription of the P<SUB><I>BAD</I></SUB> promoter becomes more obvious by calculating of the specific production rates of GFP, as shown in Figure 13. The specific production rates were calculated via equation (1)Â :</P><P align="justify">
With the specific production rate of GFP, it can be demonstrated that GFP synthesis rates differ extremely between the cultivation on glucose and glycerol. The specific production rate is constantly very low when using glucose as carbon source, but shows a higher more unstable level in the cultivation with glycerol.
Because the specific production rate was calculated between every single measurement point, the curve in Figure 13 is not smoothed and so the fluctuations have to be ignored, as they do not stand for are real fluctuations in the expression of GFP. They are caused by measurement variations in the growth curve and the fluorescence curve. And as neither of this curves are ideal, the fluctuations are the result. Nevertheless this graph shows clearly the difference between the two carbon sources.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 13:</B> Specific production rate of GFP transcribed via the P<SUB>BAD</SUB> promoter in dependence of different carbon sources.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_the_Biosafety-System_araCtive"><B>Characterization of the Biosafety-System araCtive</B></SPAN></H3><P align="justify">
The Biosafety-System araCtive was characterized on M9 minimal medium using glycerol as carbon source. As for the characterization of the pure arabinose promoter P<SUB><I>BAD</I></SUB> above, the bacterial growth and the fluorescence of GFP &lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt; was measured. Therefore, the wild type and the Biosafety-Strain <I>E. coli</I> K-12 â<I>alr</I> â<I>dadX</I>, both containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172909&lt;/bbpart&gt;, were cultivated once with the induction of 1% L-rhamnose and once only on glycerol.
It becomes obvious (Figure 14) that the bacteria, induced with 1Â % L-rhamnose (red and black curve) grow significantly slower than on pure glycerol (orange and blue curve). This is attributed to the high metabolic burden encountered by the induced bacteria. The expression of the repressor AraC and the alanine racemase (Alr) simultaneously causes a high strain on the cells, so that they grow slower than the uninduced cells, which express only GFP.
Comparing the bacterial growth with the fluorescence in Figure 15, it can be seen that the fluorescence seems to follow the same trend than the bacterial growth. The uninduced cells show approximately an exponential rise of fluorescence, while in comparision the fluorescence of the induced bacteria increases only slowly.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 14:</B> Characterization of the bacterial growth of the Biosafety-System on M9 minimal medium with glycerol. The Figure compares the wild type K-12 and the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> containing the Biosaftey-Plasmid &lt;bbpart&gt;BBa_K1172909&lt;/bbpart&gt; and the induction by 1% L-rhamnose to pure glycerol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 15:</B> Characterization of the fluorescence of the Biosafety-System araCtive. The Figure compares the wild type K-12 and the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172909&lt;/bbpart&gt; and the induction by 1% L-rhamnose to pure glycerol.</DIV></DIV></DIV></DIV><P align="justify">
From the data presented above it can not be determined if the expression of the repressor AraC does affect the transcription of GFP or not. The slower growth of the bacteria is a first indication that the repressor AraC and the alanine racemase (Alr) are highly expressed, but the growth of the bacteria shows nearly the same kinetics as the fluorescence. So it could be possible that the repressor does not affect the expression level of GFP under the control of the arabinose promoter P<SUB><I>BAD</I></SUB>. This becomes more clear by the calculation of the specific production rate of GFP by equation (1) . As shown in Figure 17 below the specific production rate differs clearly between the uninduced Biosafety-System and the Biosafety-System induced by 1% L-rhamnose. The production of GFP in the presence of L-rhamnose (red curve) is always lower than in its absence (orange curve), so that the expression of GFP is repressed in the presence of L-rhamnose.

Because the specific production rate of GFP was calculated between every single measurement point, the curve in Figure 16 is not smoothed and so the fluctuations have to be ignored, as they do not stand for are real fluctuations in the expression of GFP. They are caused by measurement variations in the growth curve and the fluorescence curve. But there is a clear tendency that the production of GFP is significantly lower when the bacteria are induced with 1% L-rhamnose. So the Biosafety-System araCtive works.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 16:</B> Specific production rate of GFP for the Biosafety-System araCtive, calculated via equation (1). The production rate of GFP of the uninduced bacteria is significantly higher compared to the bacteria induced with 1% L-rhamnose. The Biosafety-System AraCtive works.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Conclusions"><B>Conclusions</B></SPAN></H3><P align="justify">
It could be demonstrated that the Biosafety-System arActive works in principle. For example, the expression level of GFP is decreased in the presence of L-rhamnose, as can be seen in Figure 17, depicting the specific production rates of GFP after 7,5 hours. The induction of the Biosafety-System with L-rhamnose leads to a tight repression of the expression of GFP (red bar) when compared to the uninduced state (orange bar). Therefore the second part of the Biosafety-System, which contains normally the toxic RNase Ba &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; can be used as a kill-switch regulated by the pentose L-rhamnose. In <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System#Results" class="external text" rel="nofollow">comparison</A> to the other Biosafety-Systems it becomes clear, that the Biosafety-System araCtive would be probably the best choice of our three Biosafety-Systems, because of its tight repression by the P<SUB><I>BAD</I></SUB> promoter of the second part.
But a number of problems remain to be fixed. First of all, the basal expression of AraC under uninduced conditions  is too high. This becomes evident when comparing GFP expression in the uninduced Biosafety-System (orange bar) to the uninduced second part of the Biosafety-System &lt;bbpart&gt;BBa_I13541&lt;/bbpart&gt; (purple bar). The strong reduction of GFP expression is likely caused by the basal transcription of the first part of the Biosafety-System containing the AraC regulator, which  causes a tighter repression of the P<SUB><I>BAD</I></SUB> promoter, while there is no sufficient amount of AraC in the strain carrying the  second part only.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 17:</B> Comparision of the specific production rate of GFP. Shown are the induced (1% L-rhamnose) Biosafety-System araCtive, the uninduced Biosafety-System araCtive and the second part of the Biosafety-System (P<SUB><I>BAD</I></SUB> - GFP only).</DIV></DIV></DIV></DIV><P align="justify">
In <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System#Results" class="external text" rel="nofollow">comparison</A> to the other Biosafety-Systems it becomes clear, that the Biosafety-System araCtive would be probably the best choice of our three Biosafety-Systems, because of its tight repression by the P<SUB><I>BAD</I></SUB> promoter of the second part. And although the Biosafety-System works there are still some optimization remaining. 
This high basal expression also causes problems for the double kill-switch. As can be seen in Figure 14, the Biosafety-Strain carrying the Biosafety-System araCtive can grow even without L-rhamnose induction. Thus, a tighter repression of the alanine racemase is needed. This was tried by cultivation in M9 minimal medium with glucose to obtain a tighter repression of the L-rhamnose promoter P<SUB><I>Rha</I></SUB>. In this cultivation a tighter repression of both parts of the Biosafety-System was observed, but did not significantly improve the double-kill switch mechanism. This might be due to supplementation of  the preculture L-rhamnose and D-alanine. Although the cells were washed several times before cultivation, it is possible that the bacteria store D-alanine and L-rhamnose for some time, as observed for D-alanine in the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#Characterization_of_the_D-alanine_auxotrophy" class="external text" rel="nofollow">D-alanine-auxotrophic strain</A>. Additional also the alanine racemase (Alr) could accumulate because of the induction. This could be reason for the survival of the Biosafety-Strain in the absence of L-rhamnose. A possible solution here might be the introduction of a SsrA <A href="http://parts.igem.org/Protein_domains/Degradation" class="external text" rel="nofollow">degradation tag</A> at the C-terminus of Alr (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#References" class="external text" rel="nofollow">Levchenko <I>et. al</I>, 2003</A>).
Furthermore, as glucose does not occur in this amounts in nature. Thus, a  sufficiently low transcription  of the rhamnose promoter P<SUB><I>Rha</I></SUB> should be independent of the carbon source and the corresponding intracellular cAMP concentration. So another improvement of the Biosafety-System could be to adjust the plasmid copy number. The plasmid pSB1C3 we used for the Biosafety-System araCtive &lt;bbpart&gt;BBa_K1172909&lt;/bbpart&gt; is a high copy plasmid, resulting in 100 â 300 copies of the plasmid in each cell. Therefore it could be worthwhile to clone the Biosafety-System into a  medium (15 â 20 copies) or even a low copy plasmid (2 â 12 copies) to take advantage of the double-kill switch on one hand, and to reduce the metabolic pressure triggered by the plasmid on the other hand. Moreover the lower plasmid copy number might simplify the integration of the Barnase instead of GFP. Nevertheless the Biosafety-System araCtive we constructed can already be used as a kill-switch Biosafety-System and is characterized by a higher plasmid stability compared to common Biosafety-Systems.</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Baldoma L and Aguilar J (1988) Metabolism of L-Fucose and L-Rhamnose in Escherichia coli: Aerobic-Anaerobic Regulation of L-Lactaldehyde Dissimilation <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210658/pdf/jbacter00179-0434.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 170: 416 - 421.</I></A>.
</LI><LI>Carafa, Yves d'Aubenton Brody, Edward and Claude (1990) Thermest Prediction of Rho-independent Escherichia coli Transcription Terminators - A Statistical Analysis of their RNA Stem-Loop Structures <A href="http://ac.els-cdn.com/S0022283699800059/1-s2.0-S0022283699800059-main.pdf?_tid=ede07e2a-2a92-11e3-b889-00000aab0f6c&amp;acdnat=1380629809_2d1a59e395fc69c8608ab8b5aea842f7|" class="external text" rel="nofollow"><I>Journal of molecular biology 216: 835 - 858</I></A>.
</LI><LI>Casali, N., Preston A. (2003) E. coli Plasmid Vectors - Methods and Applications <A href="http://www.springerprotocols.com/BookToc/doi/10.1385/1592594093" class="external text" rel="nofollow"><I>Methods in Molecular Biology 235</I></A><I>.</I></LI><LI>Cass, Laura G. and Wilcoy, Gary (1988) Novel Activation of araC Expression and a DNA Site Required for araC Autoregulation in Escherichia coli B/r <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC211425/pdf/jbacter00187-0394.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 170: 4174 - 4180</I></A>.
</LI><LI>Hamilton, Eileen P. and Lee, Nancy (1988) Three binding sites for AraC protein are required for autoregulation of araC in Escherichia coli <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC279856/pdf/pnas00258-0029.pdf|" class="external text" rel="nofollow"><I>Proc Natl Acad Sci U S A 85: 1749 - 53</I></A>.
</LI><LI>Levchenko I., Grant R., Wah D., Sauer R., Baker T. (2003) Structure of a delivery protein for an AAA+ protease in complex with a peptide degradation tag. <A href="http://ac.els-cdn.com/S109727650300323X/1-s2.0-S109727650300323X-main.pdf?_tid=4fe9541a-3fed-11e3-a15a-00000aab0f01&amp;acdnat=1382977603_a779ff84590dc557902de5503c7e433a|" class="external text" rel="nofollow"><I>Molecular cell 12:365 - 72.</I></A></LI><LI>Mossakowska, Danuta E. Nyberg, Kerstin and Fersht, Alan R. (1989) Kinetic Characterization of the Recombinant Ribonuclease from Bacillus amyloliquefaciens (Barnase) and Investigation of Key Residues in Catalysis by Site-Directed Mutagenesis <A href="http://pubs.acs.org/doi/pdf/10.1021/bi00435a033|" class="external text" rel="nofollow"><I>Biochemistry 28: 3843 - 3850.</I></A>.
</LI><LI>Paddon, C. J. Vasantha, N. and Hartley, R. W. (1989) Translation and Processing of Bacillus amyloliquefaciens Extracellular Rnase <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC209718/pdf/jbacter00168-0575.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 171: 1185 - 1187.</I></A>.
</LI><LI>Schleif, Robert (2010) AraC protein, regulation of the L-arabinose operon in Escherichia coli, and the light switch mechanism of AraC action <A href="http://gene.bio.jhu.edu/Ourspdf/127.pdf|" class="external text" rel="nofollow"><I>FEMS microbial reviews 34: 779 - 796.</I></A>.
</LI><LI>Voss, Carsten Lindau, Dennis and Flaschel, Erwin (2006) Production of Recombinant RNase Ba and Its Application in Downstream Processing of Plasmid DNA for Pharmaceutical Use <A href="http://onlinelibrary.wiley.com/doi/10.1021/bp050417e/pdf|" class="external text" rel="nofollow"><I>Biotechnology Progress 22: 737 - 744.</I></A>.
</LI><LI>Walsh, Christopher (1989) Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. <A href="http://www.jbc.org/content/264/5/2393.long|" class="external text" rel="nofollow"><I>Journal of biological chemistry 264: 2393 - 2396.</I></A></LI><LI>Wickstrum, J.R., Santangelo, T.J., and Egan, S.M. (2005) Cyclic AMP receptor protein and RhaR synergistically activate transcription from the L-rhamnose-responsive rhaSR promoter in Escherichia coli. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251584/?report=reader|" class="external text" rel="nofollow"><I>Journal of Bacteriology 187: 6708 â 6719.</I></A>.
</LI></UL></DIV><DIV id="asdf"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Genetic_Approach"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Genetic Approach</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Rhamnose_promoter_PRha"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Rhamnose promoter P<SUB>Rha</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Regulator_AraC"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Regulator AraC</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Alanine_racemase_Alr"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Alanine racemase Alr</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Terminator"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Terminator</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Arabinose_promoter_PBAD"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Arabinose promoter P<SUB>BAD</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#RNase_Ba_.28Barnase.29"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">RNase Ba (Barnase)</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Biosafety-System_araCtive"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Biosafety-System araCtive</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-10"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Characterization_of_the_arabinose_promoter_PBAD"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Characterization of the arabinose promoter P<SUB>BAD</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Characterization_of_the_Biosafety-System_araCtive"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Characterization of the Biosafety-System araCtive</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Conclusions"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_S&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_S&amp;oldid=359363" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>