<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Biosafety_Biosafety_System_L"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Biosafety/Biosafety_System_L&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_L&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_L&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Biosafety/Biosafety_System_L" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Biosafety/Biosafety System L</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>Biosafety System Lac of Growth</H1><DIV id="buttonrow" style="padding-top:30px; padding-bottom:70px; padding-left:45px; clear:both;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">System Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#Genetic_Approach">Genetic Approach</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M#Results">Results</A></DIV></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption"><B>Biosafety-System Lac of growth.</B></DIV></DIV></DIV><P align="justify">
The Biosafety-System <I>Lac of Growth</I>&lt;bbpart&gt;BBa_K1172911&lt;/bbpart&gt; is an improvement of the BioBrick &lt;bbpart&gt;BBa_K914014&lt;/bbpart&gt; by replacing the first promoter P<SUB><I>BAD</I></SUB> into the rhamnose promoter P<SUB><I>Rha</I></SUB> and the integration of the alanine racemase (<I>alr</I>) &lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;. The transcription of the Barnase is regulated by the <I>lac</I> promoter and therefore LacI is used as repressor. The P<SUB><I>lac</I></SUB> promoter is characterized by a high basal transcription so that there is a high lethality rate in this systems. But as there are different challenging opportunities to overcome this problem it is also an interesting Biosafety-System with a high potential use.
</P><H2><SPAN class="mw-headline" id="Genetic_Approach">Genetic Approach</SPAN></H2><H3><SPAN class="mw-headline" id="Rhamnose_promoter_PRha"><B>Rhamnose promoter P<SUB>Rha</SUB></B></SPAN></H3><P align="justify">
The promoter P<SUB><I>Rha</I></SUB> (&lt;bbpart&gt;BBa_K914003&lt;/bbpart&gt;) naturally regulates  the catabolism of the hexose L-rhamnose. The advantage to use this promoter is its solely positive regulation. The regulon consists of the gene <I>rhaT</I> for the  L-rhamnose transporter and the two operons <I>rhaSR</I> and <I>rhaBAD</I>.
The operon <I>rhaSR</I> encodes the two transcriptional activators RhaS and RhaR, who are responsible for the positive activation of the L-rhamnose catabolism, while the operon <I>rhaBAD</I> encodes the genes for the direct catabolism of L-rhamnose.
When L-rhamnose is present, it acts as an inducer by binding to the regulatory protein RhaR. RhaR regulates its own expression and the expression of the regulatory gene <I>rhaS</I> by repressing or, in the presence of L-rhamnose, activating the operon <I>rhaSR</I>. Normally, the expression level is modest, but it can be enhanced by a higher level of intracellular cAMP, which increases in the absence of glucose. So, in the presence of L-rhamnose and a high concentration of intracellular cAMP, the activator protein RhaR is expressed on higher level, resulting in an activation of the promoter P<SUB><I>rhaT</I></SUB> for an efficient L-rhamnose uptake and an activation of the operon <I>rhaBAD</I>. The L-rhamnose is than broken down into dihydroxyacetone phosphate and lactate aldehyde by the enzymes encoded by <I>rhaBAD</I> (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Wickstrum <I>et al.</I>, 2005</A>).
A brief schematic summary of the regulation is shown in Figure 1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 1:</B> The catabolism of L-rhamnose in <I>E. coli</I> is turned off in general but inducible by L-rhamnose. The induction activates the transcription of the genes <I>rhaS</I> and <I>rhaR</I>, which regulate the L-rhamnose catabolism by positive activation of the rhamnose uptake (<I>rhaT</I>) and its metabolization (<I>rhaBAD</I>).</DIV></DIV></DIV></DIV><P align="justify">
Dihydroxyacetone phosphate can be metabolized in the glycolysis pathway, while lactate aldehyde is oxidized to lactate under aerobic conditions and reduced to L-1,2,-propandiol under anaerobic conditions.
The degradation of L-rhamnose can be separated in three steps. In the first step the L-rhamnose is turned into L-rhamnulose by an isomerase (gene <I>rhaA</I>). L-rhamnulose is in turn phosphorylated to L-rhamnulose-1-phosphate by a kinase (gene <I>rhaB</I>) and the sugar phosphate is finally hydrolyzed by an aldolase (gene <I>rhaD</I>) to dihydroxyacetone phosphate and lactate aldehyde (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Baldoma <I>et al.</I>, 1988</A>).
For our Safety-System <I>Lac of Growth</I>, the rhamnose promoter P<SUB>Rha</SUB> is used to control the expression of the repressor LacI and the essential alanine racemase, because this promoter has an even lower basal transcription then the arabinose promoter P<SUB><I>BAD</I></SUB>. This is needed to tightly repress the expression of the alanine racemase (<I>alr</I>) and thereby take  advantage of the double-kill switch. Although the rhamnose promoter P<SUB><I>Rha</I></SUB> is characterized by a very low basal transcription it can not be used for the control of the toxic RNase Ba (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#RNase_Ba_.28Barnase.29" class="external text" rel="nofollow">Barnase</A>), which ist needed for the alanine racemase (<I>alr</I>), the other part of the double-kill switch. In conclusion this promoter is the best choice for the first part of the Biosafety-System, as it is tightly repressed in the absence of L-rhamnose, but activated in its presence.</P><H3><SPAN class="mw-headline" id="Repressor_LacI"><B>Repressor LacI</B></SPAN></H3><P align="justify">
Naturally the <I>lacZYA</I> operon encodes the genes for the direct catabolism of the disaccharide lactose in <I>E. coli</I>. The operon contains the lactose promoter P<SUB><I>lac</I></SUB> and the genes for the catabolism of lactose to glucose and galactose. Upstream of the <I>lac</I> operator, in opposite orientation, the coding sequence for the repressor LacI under the control of a weak promoter is located (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Jacob <I>et al.</I>, 1961</A>). 
Compared to the catabolism of the sugars L-rhamnose or L-arabinose, lactose, as a disaccharide has a higher energy content and is therefore preferably used. For this reason the basal transcription of this promoter is higher than that of P<SUB><I>Rha</I></SUB>. The leakiness of the <I>lac</I> promoter is caused by the fact that LacY needs to be expressed for an efficient lactose uptake, while the uptake in the arabinose system is regulated separately (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Görke <I>et al.</I>, 2008</A>). 
In our Safety-System the LacI (&lt;bbpart&gt;BBa_C0012&lt;/bbpart&gt;) is used for repression of the lactose promoter P<SUB><I>lac</I></SUB>.
</P><H3><SPAN class="mw-headline" id="Alanine_racemase_Alr"><B>Alanine racemase Alr</B></SPAN></H3><P align="justify">
The alanine racemase Alr (EC 5.1.1.1) from the Gram-negative enteric bacteria <I>Escherichia coli</I> is an isomerase, which catalyses the reversible conversion of L-alanine into the enantiomer D-alanine (see Figure 2). For this reaction, the cofactor pyridoxal-5'-phosphate (PLP) is necessary. The constitutively expressed alanine racemase (<I>alr</I>) is naturally responsible for the accumulation of D-alanine. This compound is an essential component of the bacterial cell wall, because it is used for the cross-linkage of peptidoglycan (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Walsh, 1989</A>).
The usage of D-alanine instead of a typically L-amino acid prevents cleavage by peptidases. However, a lack of D-alanine causes to a bacteriolytic characteristics. In the absence of D‑alanine dividing cells will lyse rapidly. This fact is used for our Biosafety-Strain, a D-alanine auxotrophic mutant (K-12 ∆<I>alr</I> ∆<I>dadX</I>). The Biosafety-Strain grows only with a plasmid containing the alanine racemase (&lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;) to complement the D-alanine auxotrophy. Consequently the alanine racemase is essential for bacterial cell division. This approach guarantees a high plasmid stability, which is extremely important when the plasmid contains a toxic gene like the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#RNase_Ba_.28Barnase.29" class="external text" rel="nofollow">Barnase</A>. In addition this construction provides the possibility for the implementation of a double kill-switch system. Because if the expression of the alanine racemase is repressed and there is no D-alanine supplementation in the medium, cells will not grow.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:355px;"><DIV class="thumbcaption"><B>Figure 2:</B> The alanine racemase (&lt;bbpart&gt;BBa_K1172901&lt;/bbpart&gt;) from <I>E. coli</I> catalyses the reversible conversion from L-alanine to D-alanine. For this isomerization the cofactor pyridoxal-5'-phosphate is necessary.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Terminator"><B>Terminator</B></SPAN></H3><P align="justify">
Terminators are essential to terminate the transcription of an operon. In procaryotes two types of terminators exist. The rho-dependent and the rho-independent terminator. Rho-independent terminators are characterized by their stem-loop forming sequence. In general, the terminator-region can be divided into four regions. The first region is GC-rich and constitutes one half of the stem. This region is followed by the loop-region and another GC-rich region that makes up the opposite part of the stem. The terminator closes with a poly uracil region, which destabilizes the binding of the RNA-polymerase. The stem-loop of the terminator causes a distinction of the DNA and the translated RNA. Consequently the binding of the RNA-polymerase is cancelled and the transcription ends after the stem-loop (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Carafa <I>et al.</I>, 1990</A>).
For our Bioafety-System <I>Lac of Growth</I> the terminator is necessary to avoid that the expression of the genes under control of the rhamnose promoter P<SUB><I>Rha</I></SUB>, like the Repressor LacI and the alanine racemase (<I>alr</I>) results in the transcription of the genes behind the lactose promoter P<SUB><I>Lac</I></SUB> which contains the toxic Barnase &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; and would lead to cell death.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 3:</B> Stem-loop structure of the terminator &lt;bbpart&gt;BBa_B0015&lt;/bbpart&gt;, which is used for the Biosafety-System <I>Lac of Growth</I>. The terminator is used to make sure that solely both the repressor LacI and the alanine racemase Alr are expressed but the transcription of the toxic RNase Ba (Barnase) is avoided.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Lactose_promoter_Plac"><B>Lactose promoter P<SUB><I>lac</I></SUB></B></SPAN></H3><P align="justify">
Naturally the <I>lacZYA</I> operon encodes the genes for the direct catabolism of the disaccharide lactose in <I>E. coli</I>. The operon consists of a CAP-binding site, the <I>lac</I> promoter, the <I>lac</I> operator and the genes <I>lacZ</I>, <I>lacY</I> and <I>lacA</I> downstream of the promoter. The transcription of the lactose promoter is regulated by the LacI repressor, whose coding sequence is found upstream of the <I>lacZYA</I> operon under control of a weak promoter (Figure 4).
In the absence of lactose the transcription of the genes behind the lactose promoter is blocked, caused by the binding of the LacI repressor. In the presence of lactose the repressor is released from the operator sequence and so the genes can be transcribed. Typically the transcription is enhanced by a high intracellular level of cAMP (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Busby, 1999</A>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 4:</B>Structure of the lactose operon and its regulatory units. In the absence of lactose, transcription of the genes behind the lactose promoter is blocked. In the presence of lactose a side reaction of the ß-galactosidase (LacZ) synthesizes allolactose, which causes a conformation change in the repressor LacI. The repressor LacI releases the operator sequence and the transcription of the genes behind the lactose operons starts.</DIV></DIV></DIV></DIV><P align="justify">
The lactose promoter thereby regulates the transcription of the genes <I>lacZ</I>, <I>lacY</I> and <I>lacA</I>. <I>lacZ</I> encodes ß-galactosidase, an enzyme, which breaks down lactose to glucose and galactose. Additionally ß-galactosidase catalyzes the degradation of lactose to allolactose. By binding to the LacI repressor, allolactose changes the conformation of LacI. Thereupon, the repressor is not able to bind to the operator sequence and so the transcription is  not blocked any more. As only one enzyme is necessary to gain a substrate for glycolysis it becomes clear, why the degradation of lactose is more preferable compared to L-arabinose or L-rhamnose. 
Lactose, as a disachharide has a higher energy content. Hence, the cell needs to ensure the proper catabolism of lactose, which leads to a less tight regulation of the <I>lac</I> operon. This is caused by the fact that the <I>lacY</I> gene, coding for the integral membrane protein lactose permease, is necessary for the lactose uptake and has to be transcribed on a low level.
The last gene of the <I>lacZYA</I> operon, <I>lacA</I>, encodes for a transacetylase, which acetylizes glycosides that cannot be metabolized. The acetylated glycosides are transported outside the cell to avoid their accumulation in the cell.
In our Biosafety-System the <I>lac</I> promoter is used to regulate the expression of the Barnase. Because the native <I>lac</I> promoter shows a high basal transcription in comparison to the other promoters used, it might be not ideal for the regulation of a toxic gene product. Nevertheless the Biosafety-System <I>Lac of Growth</I> is ideal for the comparison of the basal transcription with other Biosafety-Systems and the leakiness of the <I>lac</I> promoter might be reduced by adding a second LacI-binding site 12 nt downstream of the existing binding site. As this distance corresponds to about one helix turn of the double helix, this should allow an additional LacI repressor to bind on the other site of the DNA and tighten the repression of the lactose promoter. Furthermore, the efficiency of the toxic gene product might be lower than expected, so that a higher basal transcription and activation may be needed. It can be concluded that the Biosafety-System <I>Lac of Growth</I> is  a promising Biosafety-System (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Lewis, 2005</A>).</P><H3><SPAN class="mw-headline" id="RNase_Ba_.28Barnase.29"><B>RNase Ba (Barnase)</B></SPAN></H3><P align="justify">
The Barnase (EC 3.1.27) is a 12 kDa extracellular microbial ribonuclease, which is naturally found in the Gram-positive soil bacteria <I>Bacillus amyloliquefaciens</I> and consists of a single chain of 110 amino acids. The Barnase (RNase Ba) catalyses the cleavage of single stranded RNA, preferentially behind Gs. In the first step of the RNA-degradation a cyclic intermediate is formed by transesterification and afterwards this intermediate is hydrolyzed yielding in a 3'-nucleotide (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Mossakowska <I>et al.</I>, 1989</A>).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:542px;"><DIV class="thumbcaption"><B>Figure 5:</B> Enzymatic reaction of the RNA-cleavage by the RNase Ba. First the transesterification by the Glu-73 residue is performed and then this cyclic intermediate is hydrolyzed by the His-102 of the Barnase.</DIV></DIV></DIV></DIV><P align="justify">
In <I>Bacillus amyloliquefaciens</I>, the activity of the Barnase (RNase Ba) is inhibited intracellular by an inhibitor called barstar. Barstar consists of only 89 amino acids and binds with a high affinity to the toxic Barnase. This prevents the cleavage of the intracellular RNA in the host organism (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L#References" class="external text" rel="nofollow">Paddon <I>et al.</I>, 1989</A>). Therefore the Barnase normally acts only outside the cell and is translocated under natural conditions. For the Biosafety-System Lac of Growth we modified the enzyme by cloning only the sequence responsible for the cleavage of the RNA, leaving out the N-terminal signal peptide part.

As shown in Figure 6 below, the transcription of the DNA, which encodes the Barnase produces a 474 nt RNA. The translation of the RNA starts about 25 nucleotides downstream from the transcription start and can be divided into two parts. The first part (colored in orange) is translated into a signal peptide at the amino-terminus of the Barnase coding RNA. This part is responsible for the extracellular translocation of the RNase Ba, while the peptide sequence for the active Barnase starts 142 nucleotides downstream from the transcription start (colored in red).
For the Biosafety-System <I>Lac of Growth</I>, we only used the part (&lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt;) of the Barnase encoding the catalytic domain without the extracellular translocation signal of the toxic gene product. Translation of the barnase gene leads to rapid cell death if the expression of the Barnase is not repressed by the repressor LacI of our Biosafety-System.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6:</B> Sequence of the signal peptide amino terminal of the RNase Ba (Barnase). The Biobrick &lt;bbpart&gt;BBa_K1172904&lt;/bbpart&gt; does not contain the signal sequence for the extracellular translocation, but only the coding sequence for the mature enzyme.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Biosafety-System_Lac_of_Growth"><B>Biosafety-System Lac of Growth</B></SPAN></H3><P align="justify">
Combining the genes described above with the Biosafety-Strain K-12 ∆<I>alr</I> ∆<I>dadX</I> results in a powerful device, allowing us to control the bacterial cell division. The control of the bacterial growth is possible either active or passive. Active by inducing the P<SUB><I>Lac</I></SUB> promoter with IPTG or classically with lactose and passive by the induction of L-rhamnose. The passive control makes it possible to control the bacterial cell division in a defined closed environment, like the MFC, by continuously adding L-rhamnose to the medium. As shown in Figure 7 below, this leads to an expression of the essential alanine racemase (<I>alr</I>) and the LacI repressor, so that the expression of the RNase Ba is repressed. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 7:</B> Biosafety-System <I>Lac of Growth</I> in the presence of L-rhamnose. The essential alanine racemase (Alr) and the repressor LacI are expressed, resulting in a repression of the expression of the RNAse Ba. Consequently the bacteria show normal growth behavior.</DIV></DIV></DIV></DIV><P align="justify">
In the event that bacteria exit the defined environment of the MFC or L-rhamnose is not added to the medium any more, both the expression of the alanine Racemase (Alr) and the LacI repressor decrease, so that the expression of the toxic RNase Ba (Barnase) begins. The cleavage of the intracellular RNA by the Barnase and the lack of synthesized D-alanine, caused by the repressed alanine racemase inhibit the cell division. Through this it can be secured that the bacteria can only grow in the defined area or the device of choice respectively. </P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 8:</B> Active Biosafety-System <I>Lac of Growth</I> outside of a defined environment lacking L-rhamnose. Both the expression of the alanine racemase (Alr) and LacI repressor are reduced and ideally completely shut down. In contrast, the expression of the RNase Ba (Barnase) is turned on, leading to cell death by RNA cleavage.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Characterization_of_the_lactose_promoter_Plac"><B>Characterization of the lactose promoter P<SUB><I>lac</I></SUB></B></SPAN></H3><P align="justify">
First the lactose promoter P<SUB><I>Lac</I></SUB> was characterized to get a first impression of its basal transcription rate. Therefore the bacterial growth was investigated under the pressure of the unrepressed P<SUB><I>lac</I></SUB> promoter on different carbon source using M9 minimal medium with either glucose or glycerol. The transcription rate was identified by fluorescence measurement of GFP &lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt; behind the P<SUB><I>lac</I></SUB> promoter using the BioBrick &lt;bbpart&gt;BBa_K741002&lt;/bbpart&gt;.
As shown in Figure 9 below, the bacteria grew better on glucose then on glycerol. This is due to glucose being the better energy source of these two, because glycerol enters glycolysis at a later step and therefore delivers less energy. Moreover an additional ATP consumption is needed to drive glycerol uptake. For the investigation of the basal transcription the fluorescence measurements, shown in Figure 10, is more interesting.

It can be demonstrated that the fluorescence differs corresponding to the carbon source used. The difference it not that high as observed for the arabinose promoter <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Characterization_of_the_arabinose_promoter_PBAD" class="external text" rel="nofollow">P<SUB><I>BAD</I></SUB></A>. But in comparison to the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S#Characterization_of_the_arabinose_promoter_TetO" class="external text" rel="nofollow">tetO</A> operator the difference is still obvious. This shows that the transcription of the <I>lac</I> promoter depends on the intracellular cAMP level, but is not as effective as the arabinose promoter P<SUB><I>BAD</I></SUB>. So growth on glucose does not cause an additional induction of the lactose promoter P<SUB><I>lac</I></SUB>, while growth on glycerol does. In the presence of glucose, the intracellular concentration of cAMP is low. The absence of cAMP results in inactive CAP (carbon utilization activator protein) and thus no induction of more inefficient catabolic routes, like the catabolism of lactose. Therefore, glucose is catabolized first by the bacteria, resulting in an optimal growth. In the absence of glucose, the cAMP level increases which enhances via CAP-cAMP the transcription of most operons encoding enzymes for the catabolism of alternative carbon sources. Therefore, the expression of GFP under the control of the P<SUB><I>lac</I></SUB> promoter increases on glycerol.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 9:</B> Characterization of the bacterial growth of the Biosafety-Strain K-12 ∆<I>alr</I> ∆<I>dadX</I> containing the plasmid &lt;bbpart&gt;BBa_K741002&lt;/bbpart&gt; with GFP (&lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt;) under the control of the P<SUB><I>lac</I></SUB> promoter. The M9 medium was supplemented with 5 mM D-alanine. It could be demonstrated, that the bacteria grow faster on M9 minimal medium containing glucose than on M9 minimal medium with glycerol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 10:</B> Characterization of the fluorescence of the Biosafety-Strain K-12 ∆<I>alr</I> ∆<I>dadX</I> containing the plasmid &lt;bbpart&gt;BBa_K741002&lt;/bbpart&gt; with GFP (&lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt;) under control of the P<SUB><I>lac</I></SUB> promoter. The Biosafety-Strain was cultivated on M9 minimal medium supplemented with 5 mM D-alanine.</DIV></DIV></DIV></DIV><P align="justify">
The effect that glucose represses the transcription of the P<SUB><I>lac</I></SUB> promoter becomes more obvious by calculating the specific production rates of GFP, as shown in Figure 11. The specific production rates were calculated via equation (1) :</P><P align="justify">
With the calculation of the specific production rate of GFP it can be demonstrated that the GFP synthesis rates differ between the cultivation on glucose and glycerol. The specific production rate is low, when using glucose as carbon source, but shows a higher more unstable level in the cultivation with glycerol.
Because the specific production rate was calculated between every single measurement point, the curve in Figure 11 is not smoothed and so the fluctuations have to be ignored, as they do not stand for real fluctuations in the expression of GFP. They are caused by measurement variations in the growth curve and the fluorescence curve. And as neither of this curves are ideal, the fluctuations are the result. Nevertheless this graph shows the difference between the two carbon sources.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 11:</B> Specific production rate of GFP expressed via the P<SUB><I>Lac</I></SUB> promoter in dependence of different carbon sources.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_the_Biosafety-System_Lac_of_Growth"><B>Characterization of the Biosafety-System <I>Lac of Growth</I></B></SPAN></H3><P align="justify">
The Biosafety-System <I>Lac of Growth</I> was characterized on M9 minimal medium using glycerol as carbon source. As for the characterization of the pure lactose promoter P<SUB><I>lac</I></SUB> above, the bacterial growth and the fluorescence of GFP &lt;bbpart&gt;BBa_E0040&lt;/bbpart&gt; was measured. Therefore, the wild type and the Biosafety-Strain <I>E. coli</I> K-12 ∆<I>alr</I> ∆<I>dadX</I> both containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172911&lt;/bbpart&gt; were cultivated once with the induction of 1% L-rhamnose and once only on glycerol.

It becomes obvious (Figure 12) that the bacteria, induced with 1 % L-rhamnose (red and black curve) grow obviously slower, than on pure glycerol (orange and blue curve). This might be attributed to the high metabolic burden encountered by the induced bacteria. The expression of the repressor LacI and the alanine racemase (Alr) simultaneously causes a high stress on the cells, so that they grow slower than the uninduced cells which express only GFP. Additionally, it can be observed, that the Biosafety-Strain <I>E. coli</I> K-12 ∆<I>alr</I> ∆<I>dadX</I> shows a very long lag-phase. 
The long lag-phase and the high measured fluorescence can not be explained so far, as the wild type shows normal grow. So further cultivation of the Biosafety-Strain containing the Biosafety-System <I>Lac of Growth</I> are necessary. In contrast the fluorescence of the wild type containing the Biosafety-System Lac of Growth shows the expected characteristics. It can be observed that the fluorescence of GFP increases in the cultivation on pure glycerol, while it is reduced by the induction of 1% L-rhamnose.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 12:</B> Characterization of the bacterial growth of the Biosafety-System <I>Lac of Growth</I> on M9 minimal medium with glycerol. The Figure compares the wild type K-12 and the Biosafety-Strain K-12 ∆<I>alr</I> ∆<I>dadX</I> containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172911&lt;/bbpart&gt; and the induction by 1% L-rhamnose to pure glycerol.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 13:</B> Characterization of the fluorescence of the Biosafety-System <I>Lac of Growth</I>. The Figure compares the wild type K-12 and the Biosafety-Strain K-12 ∆<I>alr</I> ∆<I>dadX</I> containing the Biosafety-Plasmid &lt;bbpart&gt;BBa_K1172911&lt;/bbpart&gt; and the induction by 1% L-rhamnose to pure glycerol.</DIV></DIV></DIV></DIV><P align="justify">
From the data presented above it cannot be determined if the expression of the repressor LacI does affect the transcription of GFP or not. Considering the wild type containing the Biosafety-System, the slower growth is a first indication that the repressor lacI and the alanine racemase (Alr) are highly expressed, but the growth of the bacteria shows nearly the same kinetics as the fluorescence. So it could be possible that the repressor does not affect the expression level of GFP under the control of the lactose promoter P<SUB><I>lac</I></SUB>. This becomes more clear by the calculation of the specific production rate of GFP by equation (1) . As shown in Figure 14 below the specific production rate differs between the uninduced Biosafety-System and the Biosafety-System induced by 1% L-rhamnose.
The production of GFP in the presence of L-rhamnose (red curve) is always lower than in its absence (orange curve), so that the expression of GFP is repressed in the presence of L-rhamnose.
Only at the end, the GFP synthesis of the uninduced cultivations seems to be lower. This might be caused by the very fast cell division in the end of the exponential phase, so that cells grow much faster, than expressing GFP.
Because the specific production rate of GFP was calculated between every single measurement point, the curve in Figure 14 is not smoothed and so the fluctuations have to be ignored, as they do not stand for real fluctuations in the expression of GFP. They are caused by measurement variations in the growth curve and the fluorescence curve. But there is a tendency that the production of GFP is lower when the bacteria are induced with 1% L-rhamnose. So the Biosafety-System <I>Lac of Growth</I> seems to work.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 14:</B> Specific production rate of GFP for the Biosafety-System <I>Lac of Growth</I>, calculated via equation (1). The production rate of GFP of the uninduced bacteria is significantly higher compared to the bacteria induced with 1% L-rhamnose. The Biosafety-System <I>Lac of Growth</I> works.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Conclusions"><B>Conclusions</B></SPAN></H3><P align="justify">
It could be demonstrated that the Biosafety-System <I>Lac of Growth</I> works in principle. For example, the expression level of GFP decreased in the presence of L-rhamnose, as can be seen in Figure 15 the specific production rates of GFP after 7,5 hours are compared within the Biosafety-System <I>Lac of Growth</I>. The induction of the Biosafety-System with L-rhamnose leads to a tight repression of the expression of GFP (red bar) when compared to the uninduced state (orange bar). But the level of gene expression is still quite high.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 15:</B> Comparison of the specific production rate of GFP. Shown are the induced (1% L-rhamnose) Biosafety-System <I>Lac of Growth</I>, the uninduced Biosafety-System <I>Lac of Growth</I> and the second part of the Biosafety-System (P<SUB><I>Lac</I></SUB> - GFP only).</DIV></DIV></DIV></DIV><P align="justify">
The Biosafety-System <I>Lac of Growth</I> works quite well, but in <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System#Results" class="external text" rel="nofollow">comparison</A> to the other Biosafety-Systems it shows a higher basal transcription in the uninduced cultivation. So this might result in a high lethality rate, when using the toxic Barnase &lt;bbpart&gt;BBa-K1179204&lt;/bbpart&gt; instead of GFP. Although this could also be useful for Biosafety-Systems with a lower toxicity than the RNase Ba, the basal transcription could also be reduced by a double <I>lac</I> promoter with two LacI binding sites.
But a number of problems remain to be fixed. For example, the basal expression of LacI under uninduced conditions is too high. This becomes evident when comparing GFP expression in the uninduced Biosafety-System (orange bar) to the uninduced second part of the Biosafety-System &lt;bbpart&gt;BBa_K741002&lt;/bbpart&gt; (purple bar). The strong reduction of GFP expression is likely caused by the basal transcription of the first part of the Biosafety-System containing the LacI repressor which causes a tighter repression of the P<SUB><I>lac</I></SUB> promoter, while there is no sufficient amount of LacI in the strain carrying the second part only.
Another improvement of the Biosafety-System could be to adjust the plasmid copy number. The plasmid pSB1C3 we used for the Biosafety-System <I>Lac of Growth</I>&lt;bbpart&gt;BBa_K1172911&lt;/bbpart&gt; is a high copy plasmid, resulting in 100 – 300 copies of the plasmid in each cell. Therefore it could be worthwhile to clone the Biosafety-System into a  medium (15 – 20 copies) or even a low copy plasmid (2 – 12 copies) to take advantage of the double-kill switch on one hand, and to reduce the metabolic pressure triggered by the plasmid on the other hand. Moreover the lower plasmid copy number might simplify the integration of the Barnase instead of GFP. Nevertheless the Biosafety-System <I>Lac of Growth</I> we constructed can already be used as a kill-switch Biosafety-System and is characterized by a higher plasmid stability compared to common Biosafety-Systems.</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><UL><LI>Baldoma L and Aguilar J (1988) Metabolism of L-Fucose and L-Rhamnose in Escherichia coli: Aerobic-Anaerobic Regulation of L-Lactaldehyde Dissimilation <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC210658/pdf/jbacter00179-0434.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 170: 416 - 421.</I></A>.
</LI></UL><UL><LI>Busby S, Ebright RH (1999) Transcription activation by catabolite activator protein (CAP).<A href="http://ac.els-cdn.com/S0022283699931613/1-s2.0-S0022283699931613-main.pdf?_tid=a8f22b74-2cf4-11e3-9457-00000aab0f6c&amp;acdnat=1380891687_f17fd24e5a3e15da96493226fdcaaa10|" class="external text" rel="nofollow"><I>Journal of molecular biology 239: 199 - 213.</I></A>.
</LI></UL><UL><LI>Carafa, Yves d'Aubenton Brody, Edward and Claude (1990) Thermest Prediction of Rho-independent Escherichia coli Transcription Terminators - A Statistical Analysis of their RNA Stem-Loop Structures <A href="http://ac.els-cdn.com/S0022283699800059/1-s2.0-S0022283699800059-main.pdf?_tid=ede07e2a-2a92-11e3-b889-00000aab0f6c&amp;acdnat=1380629809_2d1a59e395fc69c8608ab8b5aea842f7|" class="external text" rel="nofollow"><I>Journal of molecular biology 3: 835 - 858</I></A>.
</LI></UL><UL><LI>Casali, N., Preston A. (2003) E. coli Plasmid Vectors - Methods and Applications <A href="http://www.springerprotocols.com/BookToc/doi/10.1385/1592594093" class="external text" rel="nofollow"><I>Methods in Molecular Biology 235</I></A><I>.</I></LI></UL><UL><LI>Görke, Boris and Stülke, Jörg (2008) Carbon catabolite repression in bacteria: many ways to make the most out of nutrients <A href="http://www.nature.com/nrmicro/journal/v6/n8/full/nrmicro1932.html|" class="external text" rel="nofollow"><I>Nature Reviews Microbiology 6: 613 - 624</I></A>.
</LI></UL><UL><LI>Jacob, F Monod J (1961) Genetic regulatory mechanisms in the synthesis of proteins. <A href="http://www.ncbi.nlm.nih.gov/pubmed/13718526|" class="external text" rel="nofollow"><I>Journal of molecular biology 216: 318 - 356</I></A>.
</LI></UL><UL><LI>Lewis, Mitchell (2005) The lac repressor<A href="http://www.sciencedirect.com/science/article/pii/S1631069105000685|" class="external text" rel="nofollow"><I>Comptes rendues biologies 328: 521 – 548</I></A>.
</LI></UL><UL><LI>Mossakowska, Danuta E. Nyberg, Kerstin and Fersht, Alan R. (1989) Kinetic Characterization of the Recombinant Ribonuclease from Bacillus amyloliquefaciens (Barnase) and Investigation of Key Residues in Catalysis by Site-Directed Mutagenesis <A href="http://pubs.acs.org/doi/pdf/10.1021/bi00435a033|" class="external text" rel="nofollow"><I>Biochemistry 28: 3843 - 3850.</I></A>.
</LI></UL><UL><LI>Paddon, C. J. Vasantha, N. and Hartley, R. W. (1989) Translation and Processing of Bacillus amyloliquefaciens Extracellular Rnase <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC209718/pdf/jbacter00168-0575.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 171: 1185 - 1187.</I></A>.
</LI></UL><UL><LI>Voss, Carsten Lindau, Dennis and Flaschel, Erwin (2006) Production of Recombinant RNase Ba and Its Application in Downstream Processing of Plasmid DNA for Pharmaceutical Use <A href="http://onlinelibrary.wiley.com/doi/10.1021/bp050417e/pdf|" class="external text" rel="nofollow"><I>Biotechnology Progress 22: 737 - 744.</I></A>.
</LI></UL><UL><LI>Walsh, Christopher (1989) Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. <A href="http://www.jbc.org/content/264/5/2393.long|" class="external text" rel="nofollow"><I>Journal of biological chemistry 264: 2393 - 2396.</I></A></LI></UL><UL><LI>Wickstrum, J.R., Santangelo, T.J., and Egan, S.M. (2005) Cyclic AMP receptor protein and RhaR synergistically activate transcription from the L-rhamnose-responsive rhaSR promoter in Escherichia coli. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1251584/?report=reader|" class="external text" rel="nofollow"><I>Journal of Bacteriology 187: 6708 – 6719.</I></A>.
</LI></UL></DIV><DIV id="asdf"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Genetic_Approach"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Genetic Approach</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Rhamnose_promoter_PRha"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Rhamnose promoter P<SUB>Rha</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Repressor_LacI"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Repressor LacI</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Alanine_racemase_Alr"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Alanine racemase Alr</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Terminator"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Terminator</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Lactose_promoter_Plac"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Lactose promoter P<SUB>lac</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#RNase_Ba_.28Barnase.29"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">RNase Ba (Barnase)</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Biosafety-System_Lac_of_Growth"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Biosafety-System Lac of Growth</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-10"><A href="#Results"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Characterization_of_the_lactose_promoter_Plac"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Characterization of the lactose promoter P<SUB>lac</SUB></SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#Characterization_of_the_Biosafety-System_Lac_of_Growth"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Characterization of the Biosafety-System Lac of Growth</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Conclusions"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L">http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_L&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_System_L&amp;oldid=361399" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>