"
Page
Discussion
View source
History
teams
Log in
 
Team:Bielefeld-Germany/Labjournal/June
From 2013.igem.org
Home
Project
Overview
Microbial Fuel Cell
MFC Efficiency
Mediators
Porins
Cytochromes
Nanowires
Applications
Results
Overview
Since Regionals
Parts
Datapage
Achievements
Biosafety
Overview
Biosafety-Strain
Biosafety-Systems
System Lac of growth
System TetOR alive
System AraCtive
Biosafety-Form
Modeling
Overview
Mediator Reduction
Mediator Oxidation
Intermediates
Optimal Conditions
Team
Members
Pictures & Videos
Bielefeld University
Acknowledgements
Official Site
Contact us
Human Practice
Overview
Experts
Conventions
Bio Safety Motivation
Collaborations
Media
SynBio Day
Student Academy
Partners
Sponsors
Partners
Journal
Overview
Protocols
Protocols
Prologue
May
June
July
August
September
October
June
Milestones
Starting lab work on the sub-project Porins .
Successful PCR on oprF gene from Pseudomonas fluorescens. The oprF with pre- and suffix overlaps could be amplified from genome.
Planning of our Human Practice projects started and the first participations are fixed.
MFC: Designing of 3D models of MFC´s and visited a hacker space in order to print it in a 3D printer.
Week 5
Organization
iGEM-Team Bielefeld will support the ‘CeBiTec Student Academy’ from August 26th to 30th  with an own experiment.
MFC
Constructed a resistor box with different potentiometers and LEDs in order to be able to test our fuel cells
Porines
Primer design for isolation of oprF from Pseudomonas fluorescens strain, with overlaps for BioBrick Prefix and Suffix:
Forward primer oprF (49 bp): GAATTCGCGGCCGCTTCTAGATGAAACTGAAAAACACCTTGGGCTTTGC
Reverse primer oprF (51 bp): CTGCAGCGGCCGCTACTAGTATTACTTAGCTTGGGCTTCAACCTGCGCTTC
Nanowires
Primer design for isolation of gene-cluster GSU 1491-1495, GSU 1496-1505 and GSU Promoter-1496-1505 from Geobacter sulfurreducens, containing overlaps for BioBrick Prefix and Suffix:
Forward GSU 1491-1495 (45 bp):
GAATTCGCGGCCGCTTCTAGATGCAGGCTAGCAGACTGGGAGAAC
Reverse GSU 1491-1495 (38 bp):
CTGCAGCGGCCGCTACTAGTATCACTCCTCATCCATGC
Forward GSU 1496-1505 (42 bp):
GAATTCGCGGCCGCTTCTAGAGTTGGCCAATTACCCCCATAC
Reverse GSU 1496-1505 (51 bp):
CTGCAGCGGCCGCTACTAGTATCATAAACGAACCTCGTCCCAAGGCATCAG
Forward GSU Promoter-1496-1505 (52 bp):
GAATTCGCGGCCGCTTCTAGAGGATAGGATCCGTCACCGAGTGCGAACTGCC
Week 6
Organization
Thanks to NEB for the free iGEM kit with many useful laboratory things for all german iGEM teams.
We are working on our first press release.
Having a short radio contribution in the Bielefeld university campus radio ( radio 87.9 hertz ).
MFC
Connected film canister cells in series and parallel to increase voltage/current
Tested different redox mediators in our fuel cell
Mediators
Glycerol dehydrogenase
Isolation of shipping vector pSB1C3 out of 2013 Distribution Kit Plate 5 Well 3A with insert Part RFP (<bbpart>J04450</bbpart>) for better transformation characterization ( Distribution Kit BioBrick isolation ).
Transformation of <partinfo>BBa_J04450</partinfo> into Escherichia coli KRX strain.
Plasmid isolation of <partinfo>BBa_J04450</partinfo>.
Week 7
MFC
Constructed a stack of five film canister cells which was able to make a LED glow faintly
Cytochromes
Cultivation of Shewanella oneidensis MR-1 in liquid LB medium at 30 °C
Isolation of genomic DNA from S. oneidensis and dilution to the subsequently used PCR template:
4-2006-453: 5.5ng/µl
Amplification of the mtrCAB cluster with Phusion polymerase
Annealing: Gradient [55.8 - 56.7 - 57.8 - 59.1 - 60.4 - 61.7 - 62.9 - 63.9]
Elongation: 1:15 min
Notes: Clear Bands at the expected 5.2 kb, the annealing temperature seems not to have an effect.
PCR-CleanUp
Lane2: 4-2106-451: 7.4 ng/µl
Lane5: 4-2106-451: 8.5 ng/µl
Porines
Starting first cultivation of Pseudomonas fluorescens strain for complete genome isolation .
Successful genome isolation of Pseudomonas fluorescens.
Successful PCR with forward and reverse primer oprF on the oprF gene of Pseudomonas fluorescens strain.
Table 1: Standard Phusion PCR Master Mix.
Table 2: Two step standard Phusion PCR program for gldA amplification.
oprF PCR product was isolated by agarose gel electrophoresis and purified .
Bands are at expected size of 1300 bp.
Figure 1: Agarose gel from PCR on the oprF gene of Pseudomonas fluorescens strain with forward and reverse primer oprF. As a Ladder we used GeneRuler™ 1 kb DNA Ladder from Thermo Scientific .
Nanowires
Anaerobic cultivation of Geobacter sulfurreducens strain DSM-12127 in nitrogen-gassed Geobacter-medium , which was suggested by the strain-supplier: German Collection of Microorganisms and Cell Cultures DSMZ, using 30 mL cultivation-tubes and silicone stoppers with upending rim.
Week 8
Organization
Let’s go to Lyon, flights are booked for the European jamboree in Lyon from October 11 to 13, 2013.
We will participate at BioNRW pHD Student Convention in Düsseldorf at July 13th.
MFC
Started planning a new, air tight fue cell model with improved geometry.
Mediators
Glycerol dehydrogenase
Cloning of gldA into pSB1C3 shipping vector with NEB BioBrick assembly Kit did not work as expected.
Screening of colonies with colony PCR and plasmid restriction analysis shows re-ligated pSB1C3 shipping vector.
Bands are at size of 2000 bp, the length of linear pSB1C3.
Figure 2: Agarose gel with NEB 1 kb DNA Ladder as marker. Bands are showing restriction analysis from cloning of gldA into pSB1C3 shipping vector with NEB BioBrick assembly Kit . Assembly did not work, only one band at the size of 2000 bp showing re-ligated pSB1C3.
Primer design for pSB1C3 according an universal usable backbone for Gibson Assembly with Prefix and Suffix specific overlaps:
Forward primer pSB1C3 (23 bp): TACTAGTAGCGGCCGCTGCAGTC
Reverse primer pSB1C3 (23 bp): CTCTAGAAGCGGCCGCGAATTCC
Porines
Cloning of oprF into pSB1C3 shipping vector with NEB BioBrick assembly Kit did not work as expected.
Screening of colonies with colony PCR and plasmid restriction analysis shows re-ligated pSB1C3 shipping vector as described for gldA cloning.
Nanowires
Successful genomic DNA isolation of Geobacter sulfurreducens strain.
Successful PCR with forward and reverse primer GSU 1491-1495 and GSU 1496-1505 on the appropriate gene-cluster of Geobacter sulfurreducens.
Table3: Standard Phusion PCR Master Mix for amplification of Geobacter sulfurreducens gene cluster GSU 1491-1495 and GSU 1496-1505.
Component
Volume
H2O
to 50 µL
Template (100 ng)
20 µL
Betain 5 M
12.5 µL
5 x Phusion GC Buffer
10 µL
DMSO
2.5 µL
Primer-Mix 10 mM
2.5 µL
10mM dNTP´s
2 µL
Phusion DNA Polymerase
0.5 µL
Table 4: Standard two-step Phusion PCR program for amplification of Geobacter sulfurreducens gene cluster GSU 1491-1495 and GSU 1496-1505.
Step
Temperature
Time
Cycles
Denaturation
98 °C
30 sec
Denaturation
98 °C
30 sec
10
Annealing
55 °C
60 sec
10
Elongation
72°C
270 sec
10
Denaturation
98 °C
30 sec
25
Annealing
60 °C
60 sec
25
Elongation
72 °C
270 sec
25
Elongation
72 °C
300 sec
Clearly visible band at size of 7200 bp, the length of gene cluster GSU 1491-1495.
Clearly visible band at size of 9000 bp, the length of gene cluster GSU 1496-1505.
Several bands at smaller sizes suggest alternative primer binding sites in the genome of Geobacter sulfurreducens.
GSU 1491-1495 and GSU 1496-1505 PCR products were isolated and purified by gel extraction.
Figure 3: Agarose gel from PCR on the gene-clusters GSU 1491-1495 and GSU 1496-1505 of Geobacter sulfurreducens strain. Ladder: GeneRuler™ 1 kb DNA Ladder from Thermo Scientific.
Inoculation of new cultivation tubes, containing Geobacter-medium with Geobacter sulfurreducens culture for higher concentrated PCR-template production by whole genome isolation. Cultivation at 35°C on a rotary shaker with 110 rpm.
Contents
1 Milestones
2 Week 5
2.1 Organization
2.2 MFC
2.3 Porines
2.4 Nanowires
3 Week 6
3.1 Organization
3.2 MFC
3.3 Mediators
4 Week 7
4.1 MFC
4.2 Cytochromes
4.3 Porines
4.4 Nanowires
5 Week 8
5.1 Organization
5.2 MFC
5.3 Mediators
5.4 Porines
5.5 Nanowires
Retrieved from " http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/June "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
