<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Labjournal_August"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Labjournal/August">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Labjournal/August&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/August&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/August&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Labjournal/August" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Labjournal/August</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="width:720px; padding-left:20px; padding-right:20px; margin-top:60px;"><DIV id="buttonnav" style="height:30px; padding:10px 30px; margin-bottom:0px; margin-top:20px;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Prologue">Prologue</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/May">May</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/June">June</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/July">July</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/August">August</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/September">September</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/October">October</A></DIV></DIV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>August</H1><H3><SPAN class="mw-headline" id="Milestones">Milestones</SPAN></H3><UL><LI>Our great expert Dr. Falk Harnisch has <A href="/Team:Bielefeld-Germany/Human_Practice/Experts" title="Team:Bielefeld-Germany/Human Practice/Experts">answered numerous questions</A> about our project and helped us very well.
</LI><LI><I>gldA</I> BioBrick (&lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt;) was examined.
</LI><LI><I>gldA</I> Biobrick Devices with different promotors and RBS could be added: (&lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt;)
</LI></UL><LI><I>oprF</I> BioBrick &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; is available.
</LI><LI><I>oprF</I> Biobrick was functionalized with different promotors and RBS: (&lt;bbpart&gt;BBa_K1172502&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172503&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172504&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172505&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172507&lt;/bbpart&gt;)
</LI><LI>Successful protein expression and overproduction of glycerol dehydrogenase. SDS-PAGE shows <I>gldA</I> at expected size.
</LI><UL><LI> Start of the subproject Riboflavin
</LI></UL><UL><LI>MFC: Successfully printed out decent-looking model of a fuel cell out of ABS at hackerspace Bielefeld and smoothened its surface by vaporizing it over boiling acetone.
</LI></UL><UL><LI>We have produced a knockout. This AraC-Deletion in K12 Δalr Δdadx causes that this K12 E.coli can't produce AraC of itself and that it is dependent of the supplementation of D-alanine or the supplementation of a vector where araC is expressed.
</LI></UL><H3><SPAN class="mw-headline" id="Week_14">Week 14</SPAN></H3><H4><SPAN class="mw-headline" id="Organization">Organization</SPAN></H4><UL><LI>We had a second radio interview in the Bielefeld university campus radio (<A href="http://www.radiohertz.de" class="external text" rel="nofollow">radio 87.9 hertz</A>) to introduce our project and the iGEM competition.
</LI><LI>Having an expert interview with Dr. Falk Harnisch from the Helmholtz Institute in Leipzig, who has got a great knowledge in the field of MFC.
</LI></UL><H4><SPAN class="mw-headline" id="MFC">MFC</SPAN></H4><UL><LI>Spent more time trying to print something other than a deformed heap of plastic at hackerspace Bielefeld. Success limited.
</LI></UL><H4><SPAN class="mw-headline" id="Mediators">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI>Gradient PCR on the &lt;bbpart&gt;BBa_J04450&lt;/bbpart&gt; Biobrick with Forward and Reverse Primer pSB1C3 <I>gldA</I> to amplify pSB1C3 backbone with <I>gldA</I> specific overlaps for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson assembly</A>. The optimal Primer binding temperature was 60 °C.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Table 1: Optimal <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular">Phusion</A> PCR program for amplification of pSB1C3 with <I>gldA</I> and <I>oprF</I> specific overlaps for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson assembly</A>. </B></P></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 1: Agarosegel from gradient PCR on &lt;bbpart&gt;BBa_J04450&lt;/bbpart&gt; pSB1C3 plasmid with forward and reverse primer pSB1C3 <I>gldA</I> to amplify pSB1C3 backbone with <I>gldA</I> specific overlaps for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson assembly</A>. As marker we used <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder from Thermo Scientific</A>. </B></P></DIV></DIV></DIV></DIV><UL><LI>Optimization of PCR on the <I>gldA</I> gene of <I>Escherichia coli</I> with temperature gradient PCR using Forward and Reverse <I>gldA</I> Primer. The optimal Primer binding temperature was 60 °C.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Table 2: Optimal Phusion PCR program for amplification of <I>gldA</I> from <I>Escherichia coli</I>. </B></P></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><P align="justify"><B>Figure2: Agarosegel from gradient PCR on genome DNA of <I>Escherichia coli</I> KRX with forward and reverse primer <I>gldA</I> to amplify <I>gldA</I> for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A>. As marker we used <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder fromThermo Scientific</A>. </B></P></DIV></DIV></DIV></DIV><UL><LI><I>gldA</I> PCR product and the corresponding pSB1C3 Gibson backbone were isolated by Agarose gel electrophorese and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular">purificated</A>.
</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A> with optimized <I>gldA</I> PCR product and pSB1C3 PCR product with <I>gldA</I> specific overlaps using <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A> with 3:1 molar ratio of insert to vector with 100 ng insert.
</LI><LI><I>gldA</I> BioBrick was successfully cloned into pSB1C3 shipping vector.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 3: Agarosegel with <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder from Thermo Scientific</A> as marker. Bands are showing <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> from cloning of <I>gldA</I> into pSB1C3 shipping vector with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#NEB_Biobrick_Assembly_Kit" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A>. Bands are at the expected size of 1100 bp (GldA) and 2000 bp (pSB1C3). <I>gldA</I> BioBrick (&lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt;) was examined. </B></P></DIV></DIV></DIV><UL><LI><I>gldA</I> BioBrick plasmid (&lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt;) was examined by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> with restriction enzymes <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Eco</I>RI</A> und <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Xba</I>I</A>. Bands are at the expected size of 1100 bp (<I>gldA</I>) and 2000 bp (pSB1C3).
</LI></UL><H4><SPAN class="mw-headline" id="Cytochromes">Cytochromes</SPAN></H4><UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Restriction_analysis" class="external text" rel="nofollow">Restriction</A> of ccmAH and psB1C3 and Suffix-Insertion
</LI><LI>The backbone and shipping vector psB1C3 was cut with <I>Eco</I>RI, <I>Spe</I>I, the insert <I>ccmAH</I> with <I>Eco</I>RI and <I>Xba</I>I to perfom a <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Suffix_Insertion" class="external text" rel="nofollow">standard suffix insertion</A>.
</LI><LI>We used approx. 200 ng insert and 42 ng vector, which represents a 2-fold molar excess of insert.
</LI><LI>The Ligation was incubated for 15 min at 37°C, followed by a heat-inactivation at 80°C for 20 min.
</LI><LI>The subsequent PAGE did not yield any positiv results, probably due to the use of buffer and enzymes from two different manufactures.
</LI></UL><UL><LI>Validation of all intermediate products via PAGE
</LI><LI>A gel electrophoresis was performed to ensure that all former products have the correct size and this could be ruled out as an error source.
</LI><LI>It showed that all products were indeed correct.
</LI></UL><UL><LI>Amplification of psB1C3 with insert-specific primers
</LI><LI>The backbone and shipping plasmid psB1C3 was amplified with the new designed insert-specific primers for <I>mtrCAB</I>. These primers have a specific overlap complementary to 20nt at the beginning and end of the insert. This was necessary, because we had to deal with constant religation of the vector. We investigated this issue and found out, that prefix and suffix of the psB1C3 vector have a very similarity, which facilates religation. The correct band was <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Used_Kits" class="external text" rel="nofollow">extracted from the gel and cleaned up</A>.
</LI><LI>NanoDrop:
</LI><LI>4-0108-301: 1.1 ng/ul
</LI><LI>4-0108-302: 1.0 ng/ul
</LI></UL><UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson-Assembly</A> of psB1C3 and fragment 1,2 and 3 of the <I>mtrCAB</I> cluster
</LI><LI>After thawing on ice, a total amount of 5 ul DNA is added and the whole mixture is incubated at 50°C for approx. 60min. 
</LI></UL><DL><DD><UL><LI>psB1C3 (4-0108-301): 2.2 ng
</LI><LI>Fragment1 (4-2607-304): 13.1 ng 
</LI><LI>Fragment2 (4-2707-004): 16.9 ng 
</LI><LI>Fragment3 (4-2607-301): 14.4 ng
</LI></UL></DD></DL><DL><DD><UL><LI>Notes:The inserts were not equimolar and not added in a 3-fold excess to the vector
</LI><LI>Transformation to self-made electrocompetent <I>E.coli</I> cells. A total volume of 1ul could be used.
</LI></UL></DD></DL><DL><DD><UL><LI>The <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Transformation_via_electroporation" class="external text" rel="nofollow">Transformation</A> did not yield any positive colonies, which could be due to the wrong ratio of fragments and vector (see notes above).
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Biosafety">Biosafety</SPAN></H4><UL><LI>We did <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson-Assembly</A> according to the typical Gibson-Assembly protocol with the different purified plasmids. :
</LI><LI>Fragment: &quot;pSB1C3_plac_alr+ pSB1C3_alr_rev&quot;+&quot;pSB1C3_plac_pre+ pSB1C3_alr_suf&quot;,
</LI><LI>&quot;pSB1C3_alr_fwd+ pSB1C3_alr_rev&quot;+&quot;pSB1C3_alr_pre+ pSB1C3_alr_suf&quot;,
</LI><LI>&quot;araC_d1+ araC_d2&quot;+&quot;araC_d3+ araC_d4&quot;, with pKO4
</LI><LI>After the Gibson Assembly we <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega" class="external text" rel="nofollow">transformated</A> the Gibson fragments into self produced competent <I>E.coli</I> KRX cells and plated the cells on LB+CM20-plates
</LI></UL><H4><SPAN class="mw-headline" id="Porines">Porines</SPAN></H4><UL><LI>Gradient PCR on the &lt;bbpart&gt;BBa_J04450&lt;/bbpart&gt; Biobrick with Forward and Reverse Primer pSB1C3 <I>oprF</I> to amplify pSB1C3 backbone with <I>oprF</I> specific overlaps for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson assembly</A>. The optimal Primer Binding temperature was 60 °C. 
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Table 3: Optimal <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular">Phusion</A> PCR program for amplification of pSB1C3 with <I>gldA</I> and <I>oprF</I> specific overlaps for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson assembly</A>. </B></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 4: Agarosegel from gradient PCR on &lt;bbpart&gt;BBa_J04450&lt;/bbpart&gt; pSB1C3 plasmid with forward and reverse primer pSB1C3 <I>oprF</I> to amplify pSB1C3 backbone with <I>oprF</I> specific overlaps for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson assembly</A>. As marker we used <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder from Thermo Scientific</A>. </B></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Nanowires">Nanowires</SPAN></H4><UL><LI>Screening of colonies from Gibson Assembly by plasmid restriction analysis shows fragments of 2000 bp, which corresponds to religated pSB1C3.
</LI><LI>The negative result of several new Gibson Assembly attempts and subsequent analysis of the clones by plasmid restriction analysis illustrate the difficulty of using very big gene clusters in orders of 7000-9000 bp for Gibson Assembly.
</LI><LI>Based on this result and the not very promising prospects for a functional expression of nanowires in <I>Escherichia coli</I> the nanowire project was stopped until further notice to focus on other project aspects.
</LI></UL><H3><SPAN class="mw-headline" id="Week_15">Week 15</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_2">Organization</SPAN></H4><UL><LI>The <A href="http://www1.wdr.de/themen/index.html" class="external text" rel="nofollow">TV station WDR</A> made a video with us about our project. This can be seen in the OWL local time, a television program of the WDR.
</LI></UL><H4><SPAN class="mw-headline" id="MFC_2">MFC</SPAN></H4><UL><LI>Successfully printed out decent-looking model of a fuel cell out of ABS at hackerspace Bielefeld.. Took it back to the lab for testing.
</LI><LI>Tried to remove the oxygen from our cultivation media using nitrogen aeration with very little success.
</LI></UL><H4><SPAN class="mw-headline" id="Mediators_2">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI>The following parts were <A href="http://parts.igem.org/Help:2013_DNA_Distribution" class="external text" rel="nofollow">isolated from 2013 Distribution Kit</A> and transformed into <I>Escherichia coli</I> KRX strain:
</LI><LI>&lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt; (&lt;bbpart&gt;J23104&lt;/bbpart&gt; + &lt;bbpart&gt;B0034&lt;/bbpart&gt;)
</LI><LI>&lt;bbpart&gt;BBa_K608003&lt;/bbpart&gt; (&lt;bbpart&gt;J23104&lt;/bbpart&gt; + &lt;bbpart&gt;B0032&lt;/bbpart&gt;)
</LI><LI>&lt;bbpart&gt;BBa_K608004&lt;/bbpart&gt; (&lt;bbpart&gt;J23104&lt;/bbpart&gt; + &lt;bbpart&gt;B0031&lt;/bbpart&gt;)
</LI><LI>&lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt; (&lt;bbpart&gt;J23110&lt;/bbpart&gt; + &lt;bbpart&gt;B0032&lt;/bbpart&gt;)
</LI><LI>&lt;bbpart&gt;BBa_K608007&lt;/bbpart&gt; (&lt;bbpart&gt;J23110&lt;/bbpart&gt; + &lt;bbpart&gt;B0031&lt;/bbpart&gt;)
</LI><LI>&lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt; (T7 Promotor + &lt;bbpart&gt;B0034&lt;/bbpart&gt;)
</LI><LI>&lt;bbpart&gt;BBa_K081005&lt;/bbpart&gt; (&lt;bbpart&gt;J23100&lt;/bbpart&gt; + &lt;bbpart&gt;B0030&lt;/bbpart&gt;)
</LI></UL><LI>Plasmids were <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular">isolated</A> for later <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Suffix Insertion</A>.
</LI><LI><I>gldA</I> BioBrick plasmid (&lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt;) was examined by <A href="https://scf.cebitec.uni-bielefeld.de" class="external text" rel="nofollow">sequencing</A>. 
</LI><H4><SPAN class="mw-headline" id="Cytochromes_2">Cytochromes</SPAN></H4><UL><LI>The vector &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt; which consists of an IPTG inducible lac promoter and a ribosome binding site was isolated from the 2013 Plate 3
</LI><LI>J04500......4-1508-011......89.9ng/ul	
</LI></UL><UL><LI>The vector was subsequently digested with the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Used_Kits" class="external text" rel="nofollow">enzymes <I>Eco</I>RI and <I>Spe</I>I</A> and dephosphorylated afterwards to enable suffix insertion.
</LI></UL><UL><LI>Another attempt on a <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A> of the <I>mtrCAB</I> cluster was made.
</LI><LI>pSB1C3...4-1508-301......18.0 ng/ul......1.0 ul......18ng
</LI><LI>Frag1.......4-1408-301......95.0 ng/ul......1.0 ul......95ng
</LI><LI>Frag2.......4-1408-302......20.8 ng/ul......2.0 ul......40ng
</LI><LI>Frag3.......4-1408-303......45.4 ng/ul......1.0 ul......45ng
</LI><LI>Notes: molar excess not considered.
</LI></UL><UL><LI>The assembly seemed succesfull at first sight. Besides some red colonies due to not entirely clean vector template, the isolated white colonies had a plasmid high concentration. 
</LI><LI>A <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Restriction_analysis" class="external text" rel="nofollow">restriction analysis</A> with the enzymes <I>Xba</I>I and <I>Pst</I>I was analysed via PAGE and showed allegedly correct bands, which admittedly seemed a little to low. 
</LI></UL><UL><LI>It later became clear that the assembly itself worked, the problem was a wrong fragment3, which was amplified and then isolated by mistake and used in the following.
</LI></UL><H4><SPAN class="mw-headline" id="Biosafety_2">Biosafety</SPAN></H4><UL><LI>We did plasmid isolation with different samples. The first six samples used from the distribution plates and <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega" class="external text" rel="nofollow">transformed</A> into KRX electro competent cells to concentrate the DNA. The other samples were taken after the transformation. The reason for the red and white colony samples is that we wanted to use the Biobrick of Bettencourt 2012 but they were not physically in the parts registry. We asked Bettencourt to send us the Biobricks. After a week we got the Biobrick but after plating the sample there were red and white colonies. We decided to check both of them.:
</LI><LI>BBa_I13541 A: 95,1 ng/µL (9-88-451)
</LI><LI>BBa_I13541 B: 165,1 ng/µL (9-88-452)
</LI><LI>BBa_K914004 A white: 198,8 ng/µL (9-88-453)
</LI><LI>BBa_K914004 B white: 204,9 ng/µL (9-88-454)
</LI><LI>BBa_K914004 A red: 476,5 ng/µL (9-88-455)
</LI><LI>BBa_K914004 B red: 400,9 ng/µL (9-88-456)
</LI><LI>AraC_del A: 82,4 ng/µL (9-88-457)
</LI><LI>AraC_del B: 64,5 ng/µL (9-88-458)
</LI><LI>pSB1C3_alr A: 120,4 ng/µL (9-88-459)
</LI><LI>pSB1C3_alr B: 168,4 ng/µL (9-88-460)
</LI><LI>pSB1C3_alr C: 137,2 ng/µL (9-88-461)
</LI><LI>ptac_alr A: 24,7ng/µL (9-108-451)
</LI><LI>ptac_alr B: 39,6 ng/µL (9-108-452)
</LI></UL><H4><SPAN class="mw-headline" id="Porines_2">Porines</SPAN></H4><UL><LI>Optimization of PCR on the <I>oprF</I> gene of <I>Pseudomonas fluorescens</I> with temperature gradient PCR using Forward and Reverse <I>oprF</I> Primer. The optimal Primer binding temperature was 51 °C.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Table 4: Optimal <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular">Phusion</A> PCR program for amplification of <I>oprF</I> from <I>Pseudomonas fluorescens</I>. </B></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><B>Figure 5: Agarose gel from gradient PCR on genome DNA of <I>Pseudomonas fluorescens</I> with forward and reverse primer <I>oprF</I> to amplify <I>oprF</I> for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A>. As marker we used <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder fromThermo Scientific</A>. </B></DIV></DIV></DIV></DIV><UL><LI><I>oprF</I> PCR product and the corresponding pSB1C3 Gibson backbone were isolated by Agarose gel electrophorese and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular">purificated</A>.
</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A> with optimized <I>oprF</I> PCR product and pSB1C3 PCR product with <I>oprF</I> specific overlaps using <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A> with 3:1 molar ratio of insert to vector with 100 ng insert.
</LI><LI><I>oprF</I> BioBrick was successfully cloned into pSB1C3 shipping vector.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 6: Agarose gel with <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder from Thermo Scientific</A> as marker. Bands are showing <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> from cloning of <I>oprF</I> into pSB1C3 shipping vector with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Gibson Assembly</A>. Bands are at the expected size of 1300 bp (<I>oprF</I>) and 2000 bp (pSB1C3). <I>oprF</I> BioBrick &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; was examined. </B></DIV></DIV></DIV><UL><LI><I>oprF</I> BioBrick plasmid &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; was examined by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> with restriction enzymes <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Eco</I>RI</A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Xba</I>I</A>. Bands are at the expected size of 1300 bp (OprF) and 2000 bp (pSB1C3).
</LI><LI><I>oprF</I> BioBrick plasmid &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; was examined by <A href="https://scf.cebitec.uni-bielefeld.de" class="external text" rel="nofollow">sequencing</A>. 
</LI></UL><H3><SPAN class="mw-headline" id="Week_16">Week 16</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_3">Organization</SPAN></H4><UL><LI>Final preparations for <A href="/Team:Bielefeld-Germany/Human_Practice/Student_Academy" title="Team:Bielefeld-Germany/Human Practice/Student Academy">CeBiTec pupil academy</A> in the next week.
</LI></UL><H4><SPAN class="mw-headline" id="MFC_3">MFC</SPAN></H4><UL><LI>Conducted 24-hour experiments measuring the voltage generated by <I>e. coli</I> in our fuel cell
</LI><LI>Started designing a new model for 3D-printing, because the previous design had some major flaws
</LI></UL><H4><SPAN class="mw-headline" id="Mediators_3">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI>Adding the following parts with promoter and RBS to the <I>gldA</I> Biobrick &lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt; by Standard Biobrick assembly <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Suffix Insertion</A> modified from Silver lab: &lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Promotor insertion</A> was examined by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> and the following <I>gldA</I> Biobrick Devices with promotor and RBS could be added:
</LI><LI>&lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 7: Agarose gel with <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder from Thermo Scientific</A> as marker. Bands are showing <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> from <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">suffix insertion</A> of GldA (&lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt;) into &lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt; with Standard BioBrick assembly <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Suffix Insertion</A> modified from Silver lab. Bands are at the expected size. <I>gldA</I> BioBrick devices (&lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt;) were examined. </B></P></DIV></DIV></DIV><UL><LI><I>gldA</I> BioBrick plasmid (&lt;bbpart&gt;BBa_K1172201&lt;/bbpart&gt;) with different promotors and RBS were examined by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> with restriction enzymes <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Eco</I>RI</A> und <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Xba</I>I</A> for part &lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt; and with restriction enzyme <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Eco</I>RI</A> for parts &lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt;. All bands are at the expected size.
</LI></UL><H4><SPAN class="mw-headline" id="Cytochromes_3">Cytochromes</SPAN></H4><UL><LI>The <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson assembly</A> with the wrong insert (see week 15) was further analysed and the mistake was uncovered.
</LI><LI>A <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Restriction_analysis" class="external text" rel="nofollow">restriction analysis</A> with the enzymes <I>Hinc</I>II and <I>Nae</I>I showed an unexpected pattern of bands in the PAGE, which raised the suspicion of an incorrect fragment. A further investigation of the labjournal revealed the mistake. A sequencing confirmed it. 
</LI></UL><H4><SPAN class="mw-headline" id="Biosafety_3">Biosafety</SPAN></H4><UL><LI>We wanted to use Barnase (RNase Ba) from <I>Bacillus amyloliquefaciens</I> as a degradation part of our safety system. Because of this we first had to isolate the whole genome of it and get the <I>barnase</I> gene out with specially designed primer. We used different species because we didn't exactly know which species expresses the barnase. We isolated the genomes and measured the DNA concentration of <I>Bacillus amyloliquefaciens</I></LI><LI>DMS-1061: 20ng/µL (9-138-451)
</LI><LI>Wild type H 10A1: 766,6 ng/µL (9-138-452)
</LI><LI>Wild type R+M+ 10A3: 825,2 ng/µL (9-138-453)
</LI><LI>Wild type DSMZ7: 25,8 ng/µL (9-138-454)
</LI></UL><UL><LI>We did restriction analysis of the plasmids from 8.8.2013 to check if there is the correct insert in the vectors. We used the restriction enzymes <I>Eco</I>R1 and <I>Pst</I>1 to cut the vector.  The analysis did not show the right bands on the right higth apart from BBa_I13541.
</LI><LI>BBa_I13541, BBa_K914004 white colony, BBa_K914004 red colony, <I>araC</I>, alr
</LI><LI>Result: failed
</LI></UL><UL><LI>Further we did colony-PCR to check the insert. The gel also didn't show the right bands apart from BBa_I13541. We used 1 µL of a in 50µl resuspended colony suspension and the normal PCR protocol.:
</LI><LI>BBa_I15341, BBa_K194004 white, BBa_K194004 red, pSB1C3_alr, ptac_alr+insertprimern</LI><LI>Result: failed
</LI></UL><UL><LI>We also did a restriction analysis of our samples because we guessed that there was the alanine racemase in it. We cut the vector also with <I>Eco</I>R1 and <I>Pst</I>1 but the gel didn't show the right bands by the right hight:
</LI><LI>Alr 1,2,3,4
</LI><LI>Result: no bands as expected
</LI></UL><H4><SPAN class="mw-headline" id="Porines_3">Porines</SPAN></H4><UL><LI>Adding the following parts with promoter and RBS to the <I>oprF</I> BioBrick &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; by Standard BioBrick assembly <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Suffix Insertion</A> modified from Silver lab: &lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K608007&lt;/bbpart&gt;.
</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Promotor insertion</A> was examined by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> and the following <I>oprF</I> Biobrick Devices with promotor and RBS could be added:
</LI><LI>&lt;bbpart&gt;BBa_K1172502&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172503&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172504&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608007&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172505&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172507&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption"><B>Figure 8: Agarose gel with <A href="http://www.thermoscientificbio.com/nucleic-acid-electrophoresis/generuler-1-kb-dna-ladder-ready-to-use-250-to-10000-bp" class="external text" rel="nofollow">GeneRuler™ 1 kb DNA Ladder from Thermo Scientific</A> as marker. Bands are showing <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> from <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">suffix insertion</A> of <I>oprF</I> (&lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt;) into &lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_J04500&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K608007&lt;/bbpart&gt; with Standard Biobrick assembly <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">Suffix Insertion</A> modified from Silver lab. Bands are at the expected size. <I>oprF</I> BioBrick devices (&lt;bbpart&gt;BBa_K1172502&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172503&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172504&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172505&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K1172507&lt;/bbpart&gt;) were examined. </B></DIV></DIV></DIV><UL><LI><I>oprF</I> BioBrick plasmid (&lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt;) with different promotors and RBS was examined by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Restriction_analysis" title="Team:Bielefeld-Germany/Labjournal/Molecular">restriction analysis</A> with restriction enzymes <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>Eco</I>RI</A>. All bands are at the expected size and show a successful <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Standard_BioBrick_Assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular">suffix insertion</A>.
</LI></UL><H3><SPAN class="mw-headline" id="Week_17">Week 17</SPAN></H3><H4><SPAN class="mw-headline" id="Organization_4">Organization</SPAN></H4><UL><LI>Our Team will participate at the conference ‘<A href="http://www.naturkundemuseum-berlin.de/besucherinfos/archiv-kongress-60-jahre-dna" class="external text" rel="nofollow">60 Years of DNA</A>’ in Berlin on 14. September.
</LI></UL><H4><SPAN class="mw-headline" id="MFC_4">MFC</SPAN></H4><UL><LI>Printed out the new fuel cell model at hackerspace Bielefeld and smoothened its surface by treating it vaporizing it over boiling acetone.
</LI><LI>Asked Bielefeld universitys faculty of physics if we could use their 3D printer
</LI></UL><H4><SPAN class="mw-headline" id="Mediators_4">Mediators</SPAN></H4><UL><LI>Glycerol dehydrogenase
</LI><LI><I>Escherichia coli</I> KRX strain and modified <I>E. coli</I> KRX with pSB1C3 and &lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt; was cultivated over night, with induction of gene expression for &lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt; by 0,5 % L-rhamnose and for &lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt; by 0,7 mM IPTG.  Ribolysation of over night culture for total cell disruption. After centrifugation 15 min at maximal speed supernatant was prepared for <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" title="Team:Bielefeld-Germany/Labjournal/Molecular">SDS-PAGE</A>.
</LI><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" title="Team:Bielefeld-Germany/Labjournal/Molecular">SDS-PAGE</A> shows protein Glycerol dehydrogenase at expected size of 40 kDa. In contrast to <I>Escherichia coli</I> KRX wild type, weak Anderson promoter (&lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt;) shows only a slightly stronger band, whereas T7 (&lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt;) and Lac (&lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt;) promotor show a strong band, which is equated with a strong expression and overproduction of GldA.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 9: <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" title="Team:Bielefeld-Germany/Labjournal/Molecular">SDS-PAGE</A> with <A href="http://www.thermoscientific.com/ecomm/servlet/productsdetail_11152___13576050_-1" class="external text" rel="nofollow">Prestained Protein Ladder from Thermo Scientific</A> as marker.  Comparison of protein expression between <I>Escherichia coli</I> KRX wild type and <I>Escherichia coli</I> KRX with &lt;bbpart&gt;BBa_K1172205&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172204&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K1172203&lt;/bbpart&gt; after total cell disruption. The right-hand gel was loaded with a higher protein concentration. <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" title="Team:Bielefeld-Germany/Labjournal/Molecular">SDS-PAGE</A> shows protein Glycerol dehydrogenase at expected size of 40 kDa. Enhanced overproduction with increasing promotor strength. </B></P></DIV></DIV></DIV><UL><LI>For showing an endogenous mediator production, the plan is to establish a fast <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#NADH-Assay" title="Team:Bielefeld-Germany/Labjournal/Molecular">NADH assay</A> by direct using supernatant after cell disruption and testing specific NADH fluorescens with excitation at 340 nm and emission at 460 nm.
</LI><LI>First <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#NADH-Assay" title="Team:Bielefeld-Germany/Labjournal/Molecular">NADH assay</A> test shows the need of a cell washing step, because of high background signals with LB-medium.
</LI></UL><UL><LI>Riboflavin
</LI><LI><B>This is our first entry regarding the subproject riboflavin. Please make sure to read the chapter <A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Riboflavine#Genetic_Approach" class="external text" rel="nofollow">Genetic Approach</A> in the overview section aforegoing. It supplies you with all the information necessary for a good understanding of the following labjournal entries on riboflavin.</B></LI><LI>We were able to profit from the experiences we had already gained while working on other cloning subprojects. So we started right away with designing specific pSB1C3-gibson-primers. This way, we supplied every one of our four desired parts with an unique pSB1C3 with overlaps that were homologues to the ends of the insert. Thus, we were able to shut down the religation problem (see Failbook).
</LI><LI>pSB1C3_fwd_norM (43 bp) GCGACAGCGTGCACCATTAATACTAGTAGCGGCCGCTGCAGTC
</LI><LI>pSB1C3_rev_norM (43 bp) TGAAAGCCTGAATGTTTCATCTCTAGAAGCGGCCGCGAATTCC
</LI><LI>pSB1C3_fwd_RibC (44 bp) GTTTTAACGGACTATCTTAATACTAGTAGCGGCCGCTGCAGTCC
</LI><LI>pSB1C3_rev_RibC (43 bp) TGAACTATACCAGTAAACATCTCTAGAAGCGGCCGCGAATTCC
</LI><LI>pSB1C3_fwd_Rib (43 bp) AGCTTGAACAACAGTTGTAATACTAGTAGCGGCCGCTGCAGTC
</LI><LI>pSB1C3_rev_Rib (43 bp) TCGAGTACTGACCAATTCATCTCTAGAAGCGGCCGCGAATTCC
</LI><LI>pSB1C3_fwd_Rib_nus (43 bp) AACTGGTTGCACGTAAGTAATACTAGTAGCGGCCGCTGCAGTC
</LI><LI>Rib_rev_Gibson (50 bp) TGGCTTGAACTATACCAGTAAACATTTACAACTGTTGTTCAAGCTGTTGC
</LI><LI>RibC_fwd_Gibson (54 bp) GCAACAGCTTGAACAACAGTTGTAAATGTTTACTGGTATAGTTCAAGCCACCTG </LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Table 5: Our phusion-gradient-PCR protocol. For every following PCR regarding riboflavin we just adjusted the annealing temperatures and elongation times. </B></P></DIV></DIV></DIV></DIV><UL><LI>Gradient PCR´s with the above &quot;pSB1C3...&quot; Primers and pSB1C3+RFP as template gave good results. Actually, they were that good, that we just cleaned up and pooled all nanotubes for further experiments.
</LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 10: All phusion-gradient-PCR´s on pSB1C3+RFP with appropiate primers to establish homologues sequences for Gibson Assembly. </B></P></DIV></DIV></DIV></DIV><UL><LI>Meanwhile we started to isolate the genes responsible for riboflavin expression from the genome of <I>shewanella oneidensis</I> (link to genome isolation). 
</LI><LI>The following primers were used for isolation and amplification of  the four targeted sequences <I>rib</I>, <I>rib-nusB</I>, <I>ribC</I> and <I>norM</I>:
</LI><LI>Rib_fwd (44 bp) GAATTCGCGGCCGCTTCTAGATGAATTGGTCAGTACTCGATAAC
</LI><LI>Rib_rev (42 bp) CTGCAGCGGCCGCTACTAGTATTACAACTGTTGTTCAAGCTG
</LI><LI>Rib_Nus_rev (43 bp) CTGCAGCGGCCGCTACTAGTATTACTTACGTGCAACCAGTTTG
</LI><LI>RibC_fwd (43 bp) GAATTCGCGGCCGCTTCTAGATGTTTACTGGTATAGTTCAAGC
</LI><LI>RibC_rev (42 bp) CTGCAGCGGCCGCTACTAGTATTAAGATAGTCCGTTAAAACG
</LI><LI>norM_fwd (48 bp) GAATTCGCGGCCGCTTCTAGATGAAACATTCAGGCTTTCAAGCCAAAC
</LI><LI>norM_rev (45 bp) CTGCAGCGGCCGCTACTAGTATTAATGGTGCACGCTGTCGCCTTC 
</LI></UL><UL><LI><B>Through week 17 we were able to gain all desired DNA-sequences by running <A href="https://2013.igem.org/File:IGEM_Bielefeld_2013_phugrad50-70prot_23.08.13_Week17.jpg" class="external text" rel="nofollow">gradient PCR´s</A> followed by “production-PCR´s” at appropriate annealing temperatures.</B></LI></UL><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 11: Amplification of <I>rib</I> and <I>norM</I>. Annealing Temperatur: 60 C° and 70 C°; Elongation Time: 54 sec and 21 sec. </B></P></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 12: Amplification of <I>ribC</I>. Annealing Temperatur: 56 C°; Elongation Time: 10 sec. </B></P></DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 13: Amplification of <I>rib</I>-<I>nusB</I>. Annealing Temperatur: 67 C°; Elongation Time: 60 sec. </B></P></DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Cytochromes_4">Cytochromes</SPAN></H4><UL><LI>A selection of expression vectors of different strength, that were isolated from the iGEM-plates previously, were digested with the <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Used_enzymes" class="external text" rel="nofollow">enzymes <I>Spe</I>I and <I>Pst</I>I</A> to enable <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Suffix_Insertion" class="external text" rel="nofollow">suffix insertion</A>.
</LI></UL><UL><LI>&lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;:  Anderson 0.33 + weak RBS
</LI><LI>&lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;:  Anderson 0.77 + strong RBS
</LI><LI>&lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;:  T7 + strong RBS
</LI></UL><UL><LI>Subsequently the vectors were dephosphorylated and a PCR cleanup was performed.
</LI><LI>&lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;......4-3108-305......16.9 ng/ul
</LI><LI>&lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;......4-3108-305......14.9 ng/ul
</LI><LI>&lt;bbpart&gt;BBa_K525998&lt;/bbpart&gt;......4-3108-305......18.3 ng/ul
</LI></UL><H4><SPAN class="mw-headline" id="Biosafety_4">Biosafety</SPAN></H4><UL><LI>After we isolated the genome of <I>Bacillus amyloliquefaciens</I> we had to get the barnase. We picked the genome of DSMZ7. We amplified DNase Ba via PCR and designed primer. We had success as the gel shows. As expected the band lies on the higth of 350 bp.:
</LI><LI>Size: 350bp
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Gel analysis of RNase Ba PCR.</DIV></DIV></DIV><UL><LI>After we successfully got the Barnase we had to do gel purification of the fragments. We did this as described in the protocols. After purification we measured the concentration of DNA via nanodrop.:
</LI><LI>RNase Ba fwdrevinsert 27,5ng/µL (9-238-451)
</LI><LI>RNase Ba presufVektor 67,3ng/µL (9-238-452)
</LI></UL><UL><LI>We have produced a knockout. This <I>AraC</I>-Deletion in K12 Δalr Δdadx causes that this K12 <I>E.coli</I> can't produce AraC of itself and that it is dependent of the supplementation of D-alanine or the supplementation of a vector where <I>araC</I> is expressed. The reason for this knockout is that we want to raise plasmid stability of the strain. When the strain isn't able to produce AraC it dies.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">In this picture there is the marker (250pb Roth) on the left side and the following eight pockets are filled with the colony samples. 1,5,7,8 show the right band at the right hight. They will be picked for further experiments.</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">The second picture shows the 10-22 other colonies which were picked. Also the colonies in pocket 14 and 15 were chosen to for next experiments to analyse them.</DIV></DIV></DIV></DIV><UL><LI>After checking if the <I>araC</I> deletion was successful we plated the colonies 1,5,7,8,14,15.
</LI></UL><UL><LI>We cultivated K12-WT and K12 Δalr Δdadx ΔaraC and produced electro competent cells as it is written in the standard protocol.
</LI></UL><UL><LI>We did <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Gibson_assembly" class="external text" rel="nofollow">Gibson-Assembly</A> with alr, ptac und RNase Ba to clone it into the pSB1C3 shuttle vector.
</LI></UL><UL><LI>After the Assembly we transformated the vectors in <I>E.coli</I> KRX electro competent cells via electroporation for a plasmid isolation.
</LI></UL><UL><LI>We isolateed the plasmids of DNase Ba in pSB1C3. We did restriction analysis with <I>Eco</I>R1 and <I>Pst</I>1 to check if the correct insert is in the pSB1C3. But there were no expected bands.
</LI></UL><UL><LI>Restriction analysis:
</LI><LI>Result: no expected bands!
</LI></UL><H4><SPAN class="mw-headline" id="Porines_4">Porines</SPAN></H4><UL><LI>Starting with characterization of <I>Escherichia coli</I> KRX with porin <I>oprF</I> (&lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; up to &lt;bbpart&gt;BBa_K1172507&lt;/bbpart&gt;). 
</LI><LI>Testing <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Hexadecan_Assay" title="Team:Bielefeld-Germany/Labjournal/Molecular">Hexadecan-Assay</A> for characterization of hydrophobicity of the outer membrane. Using <I>Escherichia coli</I> KRX wild type and comparing it with:
</LI><LI>&lt;bbpart&gt;BBa_K1172504&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608007&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172505&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608006&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;BBa_K1172507&lt;/bbpart&gt;: &lt;bbpart&gt;BBa_K1172501&lt;/bbpart&gt; + &lt;bbpart&gt;BBa_K608002&lt;/bbpart&gt;</LI><LI><I>E. coli</I> KRX with <I>oprF</I> plasmids and Anderson promoters show higher membrane hydrophobicity than wild type. Hydrophobicity is about 37 up to 77 % higher in comparison to the KRX wild type. Assay must be improved because of too high standard deviation.
</LI></UL><UL><LI>Membrane permeability was further tested with two uptake assays. <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#ONPG_membrane_permeability" title="Team:Bielefeld-Germany/Labjournal/Molecular">ONPG</A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#NPN_membrane_permeability_assay" title="Team:Bielefeld-Germany/Labjournal/Molecular">NPN</A> uptake assay were performed. 
</LI><LI>Performing pre-experiments with different permeabilising chemicals: Isopropanol in water, EDTA in Tris-HCl and β-mercaptoethanol, each with 70% (v/v) concentration. β-mercaptoethanol is most suitable to permeabilize <I> Escherichia coli </I> KRX. β-mercaptoethanol shows 168% relative increase of permeability, EDTA 93% and isopropanol 46%. These data provide a benchmark to assess the <I>oprF</I> expression and membrane permeabilization.
</LI><LI>The values of pre-experiments of <I> Escherichia coli </I> with <I>oprF</I> indicate a successful expression of <I>oprF</I> porin in the outer membrane. The permeability increases by introducing the plasmids with <I>oprF</I>. However, the variation between samples is still too high and needs further optimization.
</LI></UL><UL><LI>Protein isolation of outer membrane porin oprF by Release of periplasmic protein fraction from <I>E. coli</I> by <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Cold_osmotic_shock" title="Team:Bielefeld-Germany/Labjournal/Molecular">cold osmotic shock</A> for later <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" title="Team:Bielefeld-Germany/Labjournal/Molecular">SDS-PAGE</A>. OprF could not been detected on the PAGE. It seems to be that only periplasmic proteins were isolated whereas outer membrane proteins werere not soluble in the supernatant.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><P align="justify"><B>Figure 14: <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29" title="Team:Bielefeld-Germany/Labjournal/Molecular">SDS-PAGE</A> with <A href="http://www.thermoscientificbio.com/protein-electrophoresis/pageruler-unstained-protein-ladder" class="external text" rel="nofollow">Unstained</A> (Left) and <A href="http://www.thermoscientific.com/ecomm/servlet/productsdetail_11152___13576050_-1" class="external text" rel="nofollow">Prestained</A> (Right) Protein Ladder from Thermo Scientific as marker.  Comparison of protein expression between <I>Escherichia coli</I> KRX wild type and <I>Escherichia coli</I> KRX with &lt;bbpart&gt;BBa_K1172502&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172503&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172504&lt;/bbpart&gt;, &lt;bbpart&gt;BBa_K1172505&lt;/bbpart&gt; and &lt;bbpart&gt;BBa_K1172507&lt;/bbpart&gt; after periplasmic protein fractioning.  </B></P></DIV></DIV></DIV></DIV><DIV id="asdf" class="withMenu"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Milestones"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Milestones</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Week_14"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Week 14</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Organization"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#MFC"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Mediators"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Cytochromes"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Cytochromes</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Biosafety"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Porines"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Porines</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Nanowires"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">Nanowires</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-10"><A href="#Week_15"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Week 15</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Organization_2"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-12"><A href="#MFC_2"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-13"><A href="#Mediators_2"><SPAN class="tocnumber">3.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Cytochromes_2"><SPAN class="tocnumber">3.4</SPAN><SPAN class="toctext">Cytochromes</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Biosafety_2"><SPAN class="tocnumber">3.5</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI><LI class="toclevel-2 tocsection-16"><A href="#Porines_2"><SPAN class="tocnumber">3.6</SPAN><SPAN class="toctext">Porines</SPAN></A></LI><LI class="toclevel-1 tocsection-17"><A href="#Week_16"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Week 16</SPAN></A></LI><LI class="toclevel-2 tocsection-18"><A href="#Organization_3"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-19"><A href="#MFC_3"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-20"><A href="#Mediators_3"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Cytochromes_3"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Cytochromes</SPAN></A></LI><LI class="toclevel-2 tocsection-22"><A href="#Biosafety_3"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#Porines_3"><SPAN class="tocnumber">4.6</SPAN><SPAN class="toctext">Porines</SPAN></A></LI><LI class="toclevel-1 tocsection-24"><A href="#Week_17"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Week 17</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Organization_4"><SPAN class="tocnumber">5.1</SPAN><SPAN class="toctext">Organization</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#MFC_4"><SPAN class="tocnumber">5.2</SPAN><SPAN class="toctext">MFC</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#Mediators_4"><SPAN class="tocnumber">5.3</SPAN><SPAN class="toctext">Mediators</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Cytochromes_4"><SPAN class="tocnumber">5.4</SPAN><SPAN class="toctext">Cytochromes</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Biosafety_4"><SPAN class="tocnumber">5.5</SPAN><SPAN class="toctext">Biosafety</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#Porines_4"><SPAN class="tocnumber">5.6</SPAN><SPAN class="toctext">Porines</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/August">http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/August</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Labjournal/August" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Labjournal/August" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/August&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/August&amp;oldid=354855" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>