<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Biosafety_Biosafety_Strain"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Biosafety/Biosafety_Strain&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_Strain&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_Strain&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Biosafety/Biosafety_Strain" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Biosafety/Biosafety Strain</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>Safety Strain</H1><DIV id="buttonrow" style="padding-top:30px; padding-bottom:70px; padding-left:45px; clear:both;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety Overview</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#Theory">Theory</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#Genetic_Approach">Genetic Approach</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#Results">Results</A></DIV></DIV><H2><SPAN class="mw-headline" id="Overview">Overview</SPAN></H2><P align="justify">
The basis for our Safety-System is an auxotrophic safety strain combined with a RNase from <I>Bacillus amyloliquefaciens</I> as a degradation part. In this K12 derived Safety-Strain the constitutive alanine racemase (<I>alr</I>) and the catabolic alanine racemase (<I>dadX</I>) are deleted. As the alanine racemase catalyzes the reversible isomerization from L-alanine into the enantiomer D-alanine our safety strain is no longer able to synthesis the essential amino acid D-alanine. D-alanine is an essential molecule for the Gram-negative bacterium <I>E. coli</I>, because it is responsible for the cross-linkage of the peptidoglycan layer. Therefore a lack of D-alanine inhibits cell division and leads to cell lysis in growing bacteria. This effect is comparable to bacteriolytic effect known from beta-lactam-antbiotics.
An important parameter, when expressing toxic gene products, like Rnase Ba (Barnase), is the maintenance of plasmid stability. The strict dependence on D-alanine ensures plasmid stability, when a plasmid containing the coding sequence for the alanine racemase (<I>alr</I>) is brought into the Safety-Strain and no D-alanine is supplemented. Therefore only plasmid carrying bacteria are able to grow. This constitutes a double kill-switch system, because not only the Barnase would ensure cell death, but also loss of the alanine racemase (<I>alr</I>) encoded on the plasmid.</P><H2><SPAN class="mw-headline" id="Theory">Theory</SPAN></H2><P align="justify">
The cell wall is essential for every living bacteria as it confers stability and structure, protects against osmotic pressure and regulates the transport of molecules.
The composition of the cell wall differs between bacteria, a feature commonly used in taxonomy. The most common division is based on the Gram-staining into Gram-negative, for example <I>Escherichia coli</I> and Gram-positive Bacteria for example <I>Bacillus subtillis</I>. Gram-negative bacteria are characterized by an inner plasma membrane, a thin peptidoglycan layer, periplasmatic spaces and the outer membrane. In contrast Gram-positive bacteria generally lack the outer membrane but have a thicker peptidoglycan layer (see Figure 2 and 3).</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption"><B>Figure 2:</B> The cell wall of a Gram-negative bacteria is characterized by a thin peptidoglycan-layer and a second membran.</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:399px;"><DIV class="thumbcaption"><B>Figure 3:'</B> The cell wall of a Gram-positive bacteria instead is characterized by a thick peptidoglycan layer and only one membrane</DIV></DIV></DIV></DIV><P align="justify">
Hence, the peptidoglycan layer is an interesting approach to control bacterial growth. Peptidoglycan itself is a polymer consisting of a linear chain of polysaccharides and short peptides. The polysaccharides component are of alternating residues of beta-(1,4) linked <I>N</I>-acetylglucosamine and <I>N</I>-acetylmuramic acid and they are cross-linked in <I>E. coli</I> by a tetra-peptide of L-alanine, D-glutamic acid, <I>meso</I>-diaminopimelic acid and finally D-alanine. The cross-linkage is thereby realized by a transpeptide-linkage of <I>meso</I>-diaminopimelic acid and D-alanine.(<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">Cava <I>et al.</I>, 2011</A>)
Therefore D-alanine is an essential component of the bacterial peptidoglycan layer. In conclusion a lack of D-alanine would have a bacteriolytic effect, comparable to beta-lactam-antibiotics, because it inhibits the building of the peptidoglycan-linkage. Hence, the bacteria without the ability to obtain D-alanine will lyse during cell division.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:355px;"><DIV class="thumbcaption"><B>Figure 4:</B> The alanine racemase from <I>E. coli</I> catalyses the reversible reaction from L-alanine to D-alanine. Pyridoxal-5'-phosphate is an essential cofactor for this reaction.</DIV></DIV></DIV></DIV><P align="justify">
The accumulation of D-alanine in <I>E. coli</I> can be catalyzed by an alanine racemase (EC 5.1.1.1). This enzyme enables the reversible reaction from L-alanine into the enantiomer D-alanine. For this reaction the cofactor pyridoxal-5'-phosphate (PLP) is also needed. <I>E. coli</I> posses two alanine racemases. One, encoded by <I>alr</I> is constitutively expressed and therefore normally responsible for the accumulation of D-alanine, while the other one encoded by <I>dadX</I> is under control of the <I>dad</I>-operon and usually used in catabolism (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">Walsh <I>et al.</I>, 1989</A>).
The deletion of the constitutive alanine racemase (<I>alr</I>) and the catabolic alanine racemase (<I>dadX</I>) will lead to a strict dependence of D-alanine, so that the bacteria with these mutations are only able to grow on media with D-alanine supplementation or by complementation of the alanine racemase on a separate plasmid. As shown in <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#Results" class="external text" rel="nofollow">Figure 5</A> below the D-alanine-auxotrophic mutant shows a strict dependence on D-alanine.</P><H2><SPAN class="mw-headline" id="Genetic_Approach">Genetic Approach</SPAN></H2><H3><SPAN class="mw-headline" id="Deletion_of_the_alanine_racemases"><B>Deletion of the alanine racemases</B></SPAN></H3><P align="justify">
For the construction of our Biosafety-Chassis we deleted both alanine racemases via homologous recombination with the plasmid pKO4 (a derivative of pKO3 containing the coding sequence of the <I>lacZ-alpha</I> fragment and a kanamycin resistance marker) to establish a D-alanine-auxotrophic mutant. For the deletion of the alanine racemase we used the following primers.
The primers alr_Ec_d1 and alr_Ec_d2 will amplify an about 600 bp long homologous fragment  upstream of the coding sequence of the alanine racemase, while the primers alr_Ec_d3 and alr_Ec_d4 will yield an about 600 bp long homologous fragment downstream of the corresponding alanine racemase. This two fragments can be ligated by Gibson-Assembly or alternative classically by gene SOEing (Gene splicing by overlap extension, <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">Horton <I>et al.</I>, 1990</A>). Afterwards the fuesd fragments flanking the alanine racemase were cloned into the vector pKO4 using again primer-extension (see box below).
By following the protocol for the double-cross over using the heat-sensitive RepA protein and the <I>sacB</I> gene from the vector <A href="http://arep.med.harvard.edu/labgc/pko3.html" class="external text" rel="nofollow">pKO3</A>, the two alanine racemases could be successfully deleted (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">Link <I>et al.</I>, 1990</A>).</P><P><B>Primer for the complete deletion of the constitutive alanine racemase (<I>alr</I>) and the catabolic alanine racemase (<I>dadX</I>) in <I>E. coli</I></B></P><PRE><B>Primer: alr_Ec_d1 (39 bp)</B>
5'-<B>TAGCTCACTCATTAGGCACCC</B>AGCTCGATGACGAAGACT-3'<B>Primer: alr_Ec_d2 (20 bp)</B>
5'-GCCGCTTGCATTTGTGTTCC-3'<B>Primer: alr_Ec_d3 (40 bp)</B>
5'-GGAACACAAATGCAAGCGGCTTGATTGTCTGTGCCGGATG-3'<B>Primer: alr_Ec_d4 (40 bp)</B>
5'-<B>GCTTTCTACGTGTTCCGCTT</B>CCGGGAAGTAGCGTTTCAGG-3'<B>Primer: dadX_Ec_d1 (39 bp)</B>
5'-<B>TAGCTCACTCATTAGGCACC</B>TGAAGTGTGGCGATGAAGT-3'<B>Primer:dadX_Ec_d2 (20 bp)</B>
5'-GGGTCATCTCGTTTCCTTAG-3'<B>Primer: dadX_Ec_d3 (40 bp)</B>
5'-CTAAGGAAACGAGATGACCCACTTGTTGTAAGCCGGATCG-3'<B>Primer: dadX_Ec_d4 (40 bp)</B>
5'-<B>GCTTTCTACGTGTTCCGCTT</B>CGAAGCCAGCGCCAAATATG-3'
</PRE><P align="justify">
To verify the deletion of the alanine racemases in <I>E. coli</I>, we designed some additional primers, which anneal outside the deletion region so that a successful deletion can be seen by a 1370 bp (<I>alr</I>) or a 1312 bp (<I>dadX</I>) amplified DNA fragment, while the wild type shows typically a DNA fragment with a length of 2439 bp (<I>alr</I>) or 2375 bp (<I>dadX</I>). The primers for the control are listed in the box below.</P><P><B>Primer for the control of the successful deletion of the constitutive alanine racemase (<I>alr</I>) and the catabolic alanine racemase (<I>dadX</I>) in <I>E. coli</I></B></P><PRE><B>Primer: alr_Ec_control1 (20 bp)</B>
5'-GCTGGAGATGCCATCAGAAC-3'<B>Primer: alr_Ec_control2 (20 bp)</B>
5'-CCGGCGAATATTGCTACGTG-3'<B>Primer: dadX_Ec_control1 (20 bp)</B>
5'-GCTTTAATACGCCCGTTGAC-3'<B>Primer: dadX_Ec_control2 (20 bp)</B>
5'-CTGGATCAACGCTTCTTTGG-3'</PRE><H3><SPAN class="mw-headline" id="Deletion_of_araC"><B>Deletion of araC</B></SPAN></H3><P align="justify">
In addition to the deletion of the alanine racemases we deleted the coding sequence for <I>araC</I> in the Safety-Strain. As the AraC protein is the repressor of the arabinose promoter P<SUB>BAD</SUB>, this mutation effects only the Biosafety-System araCactive, but is very useful because of two reasons: On the one hand, the repressor AraC works also as an activator, so that there is no basal transcription behind the P<SUB>BAD</SUB> promoter in the strain <I>E. coli</I> K-12 Î<I>alr</I> Î<I>dadX</I> Î<I>araC</I> as <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#Results" class="external text" rel="nofollow">verified</A>. This allows to transform the Biosafety-System araCative with the barnase as the toxic gene product in the bacteria without triggering cell death. On the other hand the P<SUB>BAD</SUB> promoter is possible too tightly regulated by the repressor AraC, so that a genomic deletion is necessary for the expression of the toxic gene barnase, when there is still a basal transcription of the <I>araC</I> gene from the plasmid.</P><P align="justify">
For the deletion of the repressor AraC we therefore used the same methods as for the deletion of the alanine racemases in <I>E. coli</I>. The primers araC_d1 and araC_d2 will amplify a 600 bp long homologous fragment upstream of the coding sequence for <I>araC</I>, while the primers araC_d3 and araC_d4 will amplify an about 600 bp long fragment downstream of  <I>araC</I>. This two fragments can be ligated by Gibson-Assembly or gene SOEing (Gene splicing overlap Extension, <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">Horton <I>et al.</I>, 1990</A>) and then cloned into to gene-targeting vector <A href="http://arep.med.harvard.edu/labgc/pko3.html" class="external text" rel="nofollow">pKO3</A>.
By following the protocol for the double-cross over by using the heat-sensitive RepA protein and the <I>sacB</I> from the vector pKO3, the repressor AraC could be successfully deleted (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">Link <I>et al.</I>, 1990</A>).</P><P><B>Primer for the complete deletion of the repressor coding gene <I>araC</I> in <I>E. coli</I></B></P><PRE><B>Primer: araC_d1 (40 bp)</B>
5'-<B>TAGCTCACTCATTAGGCACCC</B>CGGCAGGAATATCGATCGC-3'<B>Primer: araC_d2 (40 bp)</B>
5'-CTTCTCTGAATGGTGGGAGTGTCATAATTGGTAACGAATC-3'<B>Primer: araC_d3 (40 bp)</B>
5'-GATTCGTTACCAATTATGACACTCCCACCATTCAGAGAAG-3'<B>Primer: araC_d4 (40 bp)</B>
5'-<B>GCTTTCTACGTGTTCCGCTT</B>AACGCCAATCCCGACCACAG-3'</PRE><P align="justify">
To verify the deletion of the repressor AraC in <I>E. coli</I>, we designed primers, which anneal outside the deletion region so that a successful deletion can be seen by a 1404 bp amplified DNA fragment, while the wild type shows typically a DNA fragment with a length of 2268 bp. The primers for the control are listed in the box below.</P><P><B>Primer for the control of the successful Deletion of repressor araC in <I>E. coli</I></B></P><PRE><B>Primer: araC_control1 (20 bp)</B>
5'-CTTTACCGCTGCGCCATAAC-3'<B>Primer: araC_control2 (20 bp)</B>
5'-AACCGCAGTGTGGTCTTTCC-3'</PRE><H2><SPAN class="mw-headline" id="Results">Results</SPAN></H2><H3><SPAN class="mw-headline" id="Characterization_of_the_D-alanine_auxotrophy"><B>Characterization of the D-alanine auxotrophy</B></SPAN></H3><P align="justify">
The deletion of the alanine racemases and <I>araC</I> in <I>E. coli</I> was not possible in the commonly used strains like JM109, Top10 or KRX, but in the wild type strain K-12. This might be due to the <I>recA</I>1 mutations in these strains, which guarantees a better plasmid maintenance because of a defect in the RecA recombinase. 
To avoid a second recombination of the alanine racemase (<I>alr</I>) from the plasmid with the genome, the whole coding sequence was deleted in the genome and the characterization of the alanine racemase was performed with the antibiotic chloramphenicol. For the complementation the alanine racemase (<I>alr</I>) was brought under the control of the P<SUB><I>tac</I></SUB> promoter. The P<SUB><I>tac</I></SUB> promoter is a fusion promoter of the -35 region of the <I>trp</I> promoter and the -10 region the <I>lac</I> promoter, so that there only slight repression and the expression of the alanine racemase is highly activated (<A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain#References" class="external text" rel="nofollow">De Boer <I>et al.</I>, 1983</A>). Therefore an induction with IPTG was not necessary on M9 minimal medium, but surprisingly it was essential on LB-agar.
The deletion of the constitutive alanine racemase (<I>alr</I>) and the catabolic alanine racemase (<I>dadX</I>) in <I>E. coli</I> leads to a strict dependence on the amino acid D-alanine, as expected. As shown in Figure 5 below the bacteria with this deletions are not any more able to grow on normal M9 medium without D-alanine supplementation (purple curve), whereas the wild type does (red curve). The auxotrophic Safety-Strain grows only on media with D-alanine (5 mM) supplemented (blue curve) or by a complementation of the alanine racemase via plasmid. Furthermore, it could be demonstrated, that the auxotrophic mutant K-12 â<I>alr</I> â<I>dadX</I> grows slightly slower, than the wild type K-12. In contrast the bacteria containing the alanine racemase (<I>alr</I>) on the plasmid &lt;bbpart&gt;BBa_K1172902&lt;/bbpart&gt; does hardly show a disadvantage in the cell division compared to the wild type.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 5:</B> Characterization of the D-alanine auxotrophic Biosafety-Strain. The Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> depends strict on the presence of D-alanine in the media or a complementation via plasmid containing an intact alanine racemase.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Characterization_of_the_araC-deletion"><B>Characterization of the araC-deletion</B></SPAN></H3><P align="justify">
The deletion of the <I>araC</I> gene was added to the D-alanine auxotrophic Biosafety-Strain resulting in the <I>E. coli</I> strain K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I>. This strain is, in addition to the D-alanine auxotrophy, not able any more to use the pentose L-arabinose as carbon source. This opens the possibility to maintain a plasmid with a toxic gene behind the P<SUB>BAD</SUB> promoter without causing cell death. This huge advantage results from the AraC protein functioning as repressor and activator of the arabinose operon <I>araBAD</I> simultaneously. Lacking this activator, the transcription of the genes behind this promoter is shutdown completely.
To verify if the P<SUB>BAD</SUB> promoter is really shutdown, the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I> was cultivated on M9-minimal medium containing only L-arabinose as carbon source. As shown in Figure 6 below without the <I>araC</I> gene there is not even a slight basal transcription (blue curve). The Biosafety-Strain can not grow on L-arabinose, where as the positive control (red curve) does. In addition, the growth of the wild-type on L-arabinose was compared to D-glucose (black curve). The faster growth of <I>E. coli</I> on D-glucose in contrast to L-arabinose explains why the <I>araBAD</I> operon is naturally shut down. Because D-glucose is more efficient to metabolize, a tight regulation of the arabinose catabolism is needed to save the cell resources while the better substrate is present. But in synthetic biology this regulon can be used for different approaches. The Biosafety-System <A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S" class="external text" rel="nofollow">araCtive</A> is only one interesting application.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><B>Figure 6:</B> Characterization of the <I>araC</I> deletion. The AraC protein is essential for the transcription of the P<SUB>BAD</SUB> promoter, so that the Biosafety-Strain K-12 â<I>alr</I> â<I>dadX</I> â<I>araC</I> can not use L-arabinose as carbon source, while the wildtype K-12 does. To compare the efficiency of L-arabinose metabolism with D-glucose this cultivation is also shown.</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P align="justify"><UL><LI>Cava F, Lam H, de Pedro MA, Waldor MK (2011) Emerging knowledge of regulatory roles of d-amino acids in bacteria <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3037491/pdf/18_2010_Article_571.pdf|" class="external text" rel="nofollow"><I>Cell and Molecular Life Sciences 68: 817 - 831.</I></A></LI><LI>De Boer Hermann, Comstock J. Lisa and Vasser Mark (1983) The tac promoter: a functional hybrid derived from the trp and lac promoters. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC393301/pdf/pnas00627-0036.pdf|" class="external text" rel="nofollow"><I>Proceedings of the National Academy of Science of the United States of America 80: 21 - 25.</I></A></LI><LI>Horton R., Cai Z., Ho S. and Pease L. (1990) Gene splicing by overlap extension: tailor-made genes using the polymerase chain reaction <A href="http://www.ncbi.nlm.nih.gov/pubmed/2357375|" class="external text" rel="nofollow"><I>BioTechniques 8: 528 - 535.</I></A></LI><LI>Link, A.J., Phillips, D. and Church, G.M. (1997) Methods for generating precise deletions and insertions in the genome of wild-type Escherichia coli: Application to open reading frame characterization. <A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC179534/pdf/1796228.pdf|" class="external text" rel="nofollow"><I>Journal of Bacteriology 179: 6228-6237.</I></A></LI><LI>Walsh, Christopher (1989) Enzymes in the D-alanine branch of bacterial cell wall peptidoglycan assembly. <A href="http://www.jbc.org/content/264/5/2393.long|" class="external text" rel="nofollow"><I>Journal of biological chemistry 264: 2393 - 2396.</I></A></LI></UL></P></DIV><DIV id="asdf"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Overview"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Theory"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Theory</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Genetic_Approach"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Genetic Approach</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Deletion_of_the_alanine_racemases"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Deletion of the alanine racemases</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Deletion_of_araC"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Deletion of araC</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-6"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Characterization_of_the_D-alanine_auxotrophy"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Characterization of the D-alanine auxotrophy</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Characterization_of_the_araC-deletion"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Characterization of the araC-deletion</SPAN></A></LI><LI class="toclevel-1 tocsection-9"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">http://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_Strain&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Biosafety/Biosafety_Strain&amp;oldid=330915" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>