<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Bielefeld-Germany_Labjournal_Molecular"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Bielefeld-Germany/Labjournal/Molecular">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Bielefeld-Germany/Labjournal/Molecular&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/Molecular&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/Molecular&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Bielefeld-Germany/Labjournal/Molecular" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Bielefeld-Germany/Labjournal/Molecular</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="contentSub">(Redirected from <A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms&amp;redirect=no" title="Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms</A>)</DIV><DIV class="ribbon"><NAV id="nav"><UL id="navigation"><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany">Home</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Project</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Abstract">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC">Microbial Fuel Cell</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/MFC_Efficiency">MFC Efficiency</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Mediators">Mediators</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Porins">Porins</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Cytochromes">Cytochromes</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Nanowires">Nanowires</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Applications">Applications</A></LI></UL><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Results</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Results">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/SinceRegionals">Since Regionals</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Parts">Parts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Datapage">Datapage</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Project/Achievements">Achievements</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Biosafety</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_Strain">Biosafety-Strain</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System">Biosafety-Systems</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_L"> System Lac of growth</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_M">System TetOR alive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Biosafety_System_S">System AraCtive</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Biosafety/Safety-Form">Biosafety-Form</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Modeling</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Reduction">Mediator Reduction</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Mediator_Oxidation">Mediator Oxidation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Intermediates">Intermediates</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Modeling#Optimal_Conditions">Optimal Conditions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Team</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Team">Members</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Media">Pictures &amp; Videos</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Bielefeld">Bielefeld University</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Acknowledgements">Acknowledgements</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Bielefeld-Germany">Official Site</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Contact">Contact us</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Human Practice</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Experts">Experts</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Conventions">Conventions</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/HumanPractice/Biosafety Motivation">Bio Safety Motivation</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Collaborations">Collaborations</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Media">Media</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Day of Synthetic Biology">SynBio Day</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Human Practice/Student_Academy">Student Academy</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners">Sponsors</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Partners#Partners">Partners</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Journal</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal">Overview</A></LI><LI><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></LI></NAV></DIV><DIV id="globalwrapper" style="width:720px; padding-left:20px; padding-right:20px; margin-top:60px;"><DIV id="buttonnav" style="height:30px; padding:10px 30px; margin-bottom:0px; margin-top:20px;"><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms">Protocols</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Prologue">Prologue</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/May">May</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/June">June</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/July">July</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/August">August</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/September">September</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/October">October</A></DIV></DIV></DIV><DIV id="globalwrapper" style="padding-left:20px; padding-right:20px"><DIV id="leftcol" style="width:750px; float:left; overflow:auto;"><H1>Protocols</H1><DIV id="buttonrow" style="padding-top:30px; padding-bottom:70px; padding-left:150px; clear:both;"><DIV class="bigbutton"><P><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Genetic Engineering">Genetic Engineering</A></P></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Analytics">Analytics</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Cultivation">Cultivations</A></DIV><DIV class="bigbutton"><A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#Materials">Materials</A></DIV></DIV><H2><SPAN class="mw-headline" id="Genetic_Engineering">Genetic Engineering</SPAN></H2><UL><LI>In this section of our protocol pages you can read more about our methods for cloning and BioBrick assembly. 
</LI><LI>Genetic engineering is a basic tool of synthetic biology. With the help of standardized DNA building blocks (BioBricks) it is fairly easy to create new and modify existing natural systems. The methods we have used in our project to create BioBricks and to modify, mutate, transform and analyse DNA are presented in this section. 
</LI></UL><H3><SPAN class="mw-headline" id="Whole_Genome_Isolation">Whole Genome Isolation</SPAN></H3><UL><LI>For the isolation we used a <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Plasmid Miniprep Kit</A>.
</LI><LI>Standard Protocol:
</LI><LI>Centrifuge 10 mL of over-night liquid culture
</LI><LI>Resuspend pellet in 800 µL of resuspension solution
</LI><LI>Ribolyse three times for 60 s with 6500 rpm (30 s break between every run, 400 mg beads)
</LI><LI>Centrifuge 3 min at 10,000 rpm
</LI><LI>Transfer 500 µL of supernatant into new 2 mL tube
</LI><LI>Add 500 µL of lysis buffer, invert 4 - 6 times
</LI><LI>Add 700 µL of neutralisation buffer, invert 4 - 6 times
</LI><LI>Centrifuge 10 min at 12,000 rpm
</LI><LI>Transfer 2 x 600 µL of supernatant into spin column (centrifuge down each time for 30 s at 12,000 rpm)
</LI><LI>Two wash steps with 750 µL of washing buffer
</LI><LI>Dry column
</LI><LI>Elute in 75 µL of elution buffer
</LI></UL><H3><SPAN class="mw-headline" id="Generating_electrocompetent_cells">Generating electrocompetent cells</SPAN></H3><UL><LI>Material:
</LI><LI>550 mL LB-Medium
</LI><LI>1 L cooled bidest. H<SUB>2</SUB>O
</LI><LI>150 mL cooled 10 % glycerine
</LI><LI>10 pre-cooled 50 mL Falcons
</LI></UL><UL><LI>Protocol:
</LI><LI>Inoculate 2x3 mL LB with bacterial stock; incubate over night at 37 °C and 200 rpm
</LI><LI>Inoculate 2x250 mL LB with the over night cultures in 1-litre-flask at 37 °C and 140 rpm 
</LI><LI>Incubate until OD600 0,4-0,6
</LI><LI>Cool the culture 15-30 minutes on ice
</LI><LI>Onwards all steps at 4°C
</LI><LI>Divide the cultures into cooled 50 mL Falcons and centrifugate at 4000 rpm, 4 °C for 15 minutes, make sure to slowly accelerate and deccelerate
</LI><LI>Discard supernatant
</LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O (and don't get frustrated while doing it, keep shaking gently)
</LI><LI>Pool two suspensions each, add bidest H<SUB>2</SUB>O up to 50 mL and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Resuspend pellet in 5 mL cooled bidest H<SUB>2</SUB>O
</LI><LI>Add bidest H<SUB>2</SUB>O up to 50 mL and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Resuspend pellet in 5 mL cooled 10 % glycerine
</LI><LI>Transfer suspensions in two 50 mL Falcons and centrifugate again (see centrifugation above)
</LI><LI>Discard supernatant
</LI><LI>Add volume of 10 % glycerine that is approximately equal to the volume of the pellet and resuspend
</LI><LI>Divide cells in 50 μL aliquots and freeze in liquid N<SUB>2</SUB> immediately
</LI><LI>Store at -80 °C
</LI></UL><H3><SPAN class="mw-headline" id="Transformation_via_electroporation">Transformation via electroporation</SPAN></H3><UL><LI>Thaw 50 µL competent <I>E.coli</I> cells on ice, dilute with icecold 50 µL glycerine (10 %) if necessary
</LI><LI>Add 0.5-5 µL plasmid to 50 µl electrocompetent cells
</LI><LI>Store cells on ice for 1 minute
</LI><LI>Electroporate at U = 2.5 kV, C = 25 µF, R = 400 Ώ
</LI><LI>Transfer transformation reaction to 450 µL SOC-Medium and shake 1 h at 37 °C
</LI><LI>Centrifuge 2 min at 2000 rpm and plate on selective LB-Medium
</LI><LI>Incubate over night at 37 °C
</LI></UL><H3><SPAN class="mw-headline" id="Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></H3><UL><LI>Using <A href="http://www.promega.com/~/media/Files/Resources/Protocols/Technical%20Bulletins/101/Single%20Step%20Competent%20Cells%20Protocol.ashx" class="external text" rel="nofollow">protocol <I>E. coli</I> KRX single step competent cells by Promega</A></LI></UL><UL><LI>Remove Single Step (KRX) Competent Cells from –70 °C, and place on ice for 5 minutes or until just thawed
</LI><LI>Add 1–50 ng of DNA (in a volume not greater than 5 μL) to the Single Step (KRX) Competent Cells. Move the pipette tip through the cells while dispensing. Quickly flick the tube several times. Do not vortex!
</LI><LI>Immediately return the tubes to ice for 5–30 minutes
</LI><LI>Heat-shock cells for 15–20 seconds in a water bath at exactly 42 °C. Do not shake.
</LI><LI>Immediately place the tubes on ice for 2 minutes.
</LI><LI>Add 450 μL of room-temperature SOC-Medium to each transformation reaction, and incubate for 60 minutes at 37 °C with shaking (approximately 225 rpm). For best transformation efficiency, lay the tubes on their sides and tape them to the platform.
</LI><LI>For each transformation reaction, we recommend plating 100 μL of undiluted cells and 1:10 and 1:100 cell dilutions on antibiotic plates. Incubate the plates at 37 °C over night.
</LI></UL><H3><SPAN class="mw-headline" id="Standard_BioBrick_Assembly"> Standard BioBrick Assembly </SPAN></H3><UL><LI>modified from <A href="http://openwetware.org/wiki/Silver:_BB_Strategy" class="external text" rel="nofollow">Silver lab</A>:
</LI></UL><UL><LI>This assembly method can be used for BioBricks which are bigger than 150 bp. The BioBrick should be at least 500 bp bigger or smaller than the backbone. The BioBrick, which complies with these conditions, is used as the insert and is assembled into the prefix or suffix of the other used BioBrick, called vector. So you have to differentiate between a prefix and a suffix insertion.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Suffix Insertion</DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Silver Prefix Insertion</DIV></DIV></DIV></DIV><H4><SPAN class="mw-headline" id="Suffix_Insertion"> Suffix Insertion </SPAN></H4><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x NEBuffer 2.1, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>XbaI</I></A>, 1 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>PstI</I></A>. Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10x NEBuffer 2.1, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>SpeI</I></A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>PstI</I></A>. Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> AP (Antarctic phosphatase)</A> and 1.2 µL 10 x AP reaction buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H4><SPAN class="mw-headline" id="Prefix_Insertion"> Prefix Insertion </SPAN></H4><UL><LI> Digestion of insert: at least 700 ng DNA / 10 µL volume, 1 µL 10x NEBuffer 2.1, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>EcoRI</I></A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>SpeI</I></A>. Digest for 1 h at 37 °C, afterwards inactivation for 20 min at 80 °C. Clean up the insert via gel electrophoresis. When cutting the insert out of the gel try to avoid staining or exposure to ultraviolet light of the insert. 
</LI><LI> Digestion of vector about 700 ng DNA / 10 µL volume, 1 µL 10 x NEBuffer 2.1, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>EcoRI</I></A>, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>XbaI</I></A>. Digest for 1h at 37 °C, afterwards inactivation for 20 min at 80 °C. Add 1 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> AP (Antarctic phosphatase)</A> and 1.2 µL 10 x AP reaction buffer, incubate for 1 h at 37 °C. Clean up the vector with a <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular"> PCR clean-up kit</A>. 
</LI><LI> Ligation: after digestion and clean-up: 50 - 200 ng of vector, 3 - 10 fold molar access of insert, 20 µL ligation volume, 2 µL T4-Ligase-Buffer, 1 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> T4-Ligase</A>. Incubate for 20 - 30 min at room temperature, afterwards inactivation for 5 min at 70 °C. Then: store at -20 °C or transform.
</LI></UL><H4><SPAN class="mw-headline" id="Variations"> Variations </SPAN></H4><UL><LI> A digestion over night is possible. If you digest over night use only 0.1 µL restriction enzyme. 
</LI><LI> It is also possible to use PCR product as insert. Digest after PCR with corresponding restriction enzymes and clean up with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular"> PCR clean-up kit</A>. This could lead to higher yields of insert DNA because a lot of DNA gets lost during the gel electrophoresis clean up. 
</LI><LI> Sometimes some BioBricks are hard to assemble. Then you have to clean up the vector by gel electrophoresis as well.
</LI></UL><H3><SPAN class="mw-headline" id="NEB_Biobrick_Assembly_Kit">NEB Biobrick Assembly Kit</SPAN></H3><UL><LI>We used the BioBrick Assembly Kit to assemble an upstream part with a downstream part into destination plasmid.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">NEB BioBrick Assembly Kit.</DIV></DIV></DIV><UL><LI>Digestion Protocoll
</LI><LI>Digest upstream part with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>EcoRI-HF</I></A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>SpeI</I></A></LI><LI>Digest downstream part with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>XbaI</I></A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>PstI</I></A></LI><LI>Digest destiantion destination plasmid with<A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>EcoRI-HF</I></A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>PstI</I></A></LI><LI>500 ng part DNA
</LI><LI>1 μL of each enzyme
</LI><LI>5 μL 10x NEBuffer 2.1
</LI><LI>to 50 μL H<SUB>2</SUB>0
</LI></UL><LI>Incubate all three restriction digest reactions at 37°C for 10 minutes and then heat inactivate at 80°C for 20 minutes.
</LI><LI>Dephosphorylation Protocoll
</LI><LI> Add 1 μL of <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> AP (Antarctic phosphatase)</A> and 5 µL of 10x AP reaction buffer to digested destination plasmid, incubate for 1 h at 37 °C.
</LI><LI>Ligation Protocoll
</LI><LI>Ligate the upstream and downstream parts into the digested destination plasmid.
</LI><LI>2 μL of each part
</LI><LI>2 μL 10x T4 DNA Ligase Buffer
</LI><LI>1 μL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular">T4 DNA Ligase</A></LI><LI>11 μL H20
</LI><LI>Incubate at room temperature for 10 minutes and then heat inactivate at 80°C for 20 minutes.
</LI><UL><LI><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Transformation_via_electroporation" title="Team:Bielefeld-Germany/Labjournal/Molecular">Transform</A> 1-2 μL of the ligation product into 50 μL of <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Generating_electrocompetent_cells" title="Team:Bielefeld-Germany/Labjournal/Molecular">competent <I>E. coli</I> cells</A>.
</LI></UL><H3><SPAN class="mw-headline" id="Gibson_Assembly"> Gibson Assembly </SPAN></H3><UL><LI>Modified from <A href="http://www.nature.com/nmeth/journal/v6/n5/full/nmeth.1318.html" class="external text" rel="nofollow">Gibson <I>et al.</I> (2009)</A></LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Gibson Assembly</DIV></DIV></DIV><UL><LI>This assembly method is an isothermal, single-reaction method for assembling multiple overlapping DNA molecules. By coordinating the activity of a 5‘ exonuclease, a DNA polymerase and a DNA ligase two adjacent DNA fragments with complementary terminal sequence overlaps can be joined into a covalently sealed molecule, without the use of any restriction endonuclease.
</LI></UL><H4><SPAN class="mw-headline" id="Preparation_of_DNA_molecules_for_in_vitro_recombination">Preparation of DNA molecules for <I>in vitro</I> recombination</SPAN></H4><UL><LI> Generate the complementary sequence overlaps by PCR using the <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Phusion DNA-polymerase</A>. If necessary add 5 M Betain in the reaction mix by reducing the amount of H<SUB>2</SUB>O to decrease the number of false PCR products.
</LI><LI> Identify the PCR products of interest by gel electrophoresis with known DNA standards.
</LI><LI> Extract the PCR products from the gel by cutting out the DNA fragments and clean them up by using a commercial <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular"> PCR clean-up kit</A>.
</LI></UL><H4><SPAN class="mw-headline" id="In_vitro_recombination"><I>In vitro</I> recombination</SPAN></H4><UL><LI> Assembly mixture 
</LI><LI> 320 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#5x_isothermal_reaction_buffer_for_Gibson_assembly" title="Team:Bielefeld-Germany/Labjournal/Molecular"> 5x isothermal reaction buffer</A></LI><LI> 0.64 µL of 10 U mL<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> T5 exonuclease</A> (for DNA molecules overlapping by greater than 150 bp add 3.2 µL of 10 U ml<SUP>–1</SUP><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> T5 exonuclease</A>)
</LI><LI> 20 µL of 2 U mL<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Phusion DNA polymerase</A></LI><LI> 160 µL of 40 U mL<SUP>-1</SUP><A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>taq</I> DNA ligase</A></LI><LI> add ddH<SUB>2</SUB>O water up to a final volume of 1.2 mL
</LI><LI> aliquote 15 µL of the reagent-enzyme mix and store it at –20 ˚C
</LI></UL><LI> Thaw 15 µL assembly mixture aliquot and keep it on ice until use.
</LI><LI> Add 5 µL of the purified DNA molecules in equimolar amounts (between 10 and 100 ng of each DNA fragment).
</LI><LI> Incubate the resulting mixture at 50 ˚C for 15 to 60 min, with 60 min being optimal.
</LI><LI> Transformation (heatshock or <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Transformation_via_electroporation" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Transformation via electroporation</A>) without cleaning up the assembly product.
</LI><H3><SPAN class="mw-headline" id="Restriction_analysis"> Restriction analysis </SPAN></H3><UL><LI> Digest BioBrick of interest: about 400 ng DNA / 10 µL volume, 1 µL 10x NEBuffer 2.1, 0.5 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_enzymes" title="Team:Bielefeld-Germany/Labjournal/Molecular"><I>NotI</I> or <I>PstI</I></A>. Digest for 1 h at 37 °C. <I>NotI</I> is used to determine the length of the BioBrick and the plasmid backbone, <I>PstI</I> is used to determine the length of the BioBrick in the plasmid backbone. 
</LI><LI> Gel electrophoresis: add 2 µL loading buffer to every digestion mix, apply about 100 - 200 ng DNA / pocket in gel. Don't forget to apply the uncut BioBrick as well. A good agarose concentration for BioBricks between 0.2 and 3 kb is 1.5 %. The smaller your BioBrick of interest is the higher the agarose concentration should be and vice versa. The gel electrophoresis is made with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#TAE-Buffer" title="Team:Bielefeld-Germany/Labjournal/Molecular"> TAE-buffer</A>. Be sure that you melt your agarose gel in the same buffer you use for the electrophoresis later.
</LI></UL><H3><SPAN class="mw-headline" id="Colony_PCR"> Colony PCR </SPAN></H3><UL><LI> Pick one colony with a sterile tip and elute it in 100 µL ddH<SUB>2</SUB>0 or medium
</LI><LI> Store the colony in 4 °C while colony PCR is running
</LI><LI> One reaction mix contains: 
</LI><LI> 10 µL 5x buffer
</LI><LI> 2 µL MgCl<SUB>2</SUB> (25 mM stock)
</LI><LI> 1 µL dNTPs
</LI><LI> 0.5 µL primer mix (prefix/suffix primers or sequencing primers)
</LI><LI> 35.25 µL ddH<SUB>2</SUB>O
</LI><LI> 0.25 µL <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Used_Kits" title="Team:Bielefeld-Germany/Labjournal/Molecular"> GoTaq polymerase (Promega)</A></LI><LI> 1 µL template
</LI></UL><LI> PCR program: 
</LI><LI> Start: 3 min, 98 °C
</LI><LI> 30 cycles of: 
</LI><LI> 30 s, 98 °C
</LI><LI> 30 s, 55 °C
</LI><LI> 30 s / 1 kb template, 72 °C
</LI><LI> Finish: 5 min, 72 °C
</LI><UL><LI> Gel electrophoresis: check the fragment size
</LI><LI> Plate the correct colony
</LI></UL><H2><SPAN class="mw-headline" id="Analytics">Analytics</SPAN></H2><UL><LI>Protocols for the analytical methods we used are listed in the next chapter.
</LI><LI>DNA and proteins are very small and cannot be seen by the naked eye. To control the success and the results of the upstream and downstream processes, analytical methods are necessary that give reliable results to make DNA or proteins in any way visible for you. The analytical methods we used in our project can be found in this section.
</LI></UL><H3><SPAN class="mw-headline" id="Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29">Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)</SPAN></H3><UL><LI>This analytical method can be used for separation and identification of proteins according to their electrophoretic mobility. The mobility is a function of length of the molecular weight. Proteins that have identical charge per unit mass due to binding of SDS results in an equal electrophoretic mobility.
</LI></UL><H4><SPAN class="mw-headline" id="Pouring_the_polyacrylamide_gel">Pouring the polyacrylamide gel</SPAN></H4><UL><LI>Make a master mix for the <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#SDS-PAGE_gel" title="Team:Bielefeld-Germany/Labjournal/Molecular">stacking</A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#SDS-PAGE_gel" title="Team:Bielefeld-Germany/Labjournal/Molecular">separating gel</A> without adding  ammonium persulfate and TEMED.
</LI><LI>Aliquote 6,5 mL for each separating and 2,5 mL for each stacking gel.
</LI><LI>Add ammonium persulfate and  TEMED to each separating gel aliquote and pour the solution quickly into the gel casting form. Leave about 2 centimeters below the bottom of the comb for the stacking gel.
</LI><LI>Layer isopropanol on top of the gel.
</LI><LI>Leave the separating gel at room temperature for &gt;60 minutes to polymerize.
</LI><LI>Remove isopropanol and wait until the surface is dry.
</LI><LI>Add ammonium persulfate and TEMED to each stacking gel aliquote and pour the solution quickly into your gel casting form.
</LI><LI>Insert comb without getting bubbles stuck underneath
</LI><LI>Leave the gel at room temperature for &gt;60 minutes to polymerize.
</LI></UL><UL><LI>For storage
</LI><LI>Remove sealing and store the gel wrapped in moistened paper towel at 4°C.
</LI></UL><H4><SPAN class="mw-headline" id="Preparing_the_sample">Preparing the sample</SPAN></H4><UL><LI>Mix your protein mixture 4:1 with <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#4x_Laemmli-buffer" title="Team:Bielefeld-Germany/Labjournal/Molecular"> Laemmli-buffer</A> (30 mL protein solution + 10 mL Laemmli-buffer)
</LI><LI>Heat for 5 minutes at 95 °C.
</LI></UL><H4><SPAN class="mw-headline" id="Running_the_gel">Running the gel</SPAN></H4><UL><LI>Remove sealing, put the polymerized gel into gel box and pour <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#SDS_running_buffer" title="Team:Bielefeld-Germany/Labjournal/Molecular"> SDS running buffer</A> into the negative and positive electrode chamber.
</LI><LI>Remove comp without destroying the gel pocket.
</LI><LI>Pipet the sample into the gel pockets, adjusting the volume according to the amount of protein in your sample. Make sure to include a lane with molecular weight standards (PageRuler Prestained Protein Ladder™ (Fa. Fermentas)) to determinate the molecular weight of the sample. 
</LI><LI>Connect the power lead and run the stacking gel with 10 mA until the blue dye front enters the separating gel.
</LI><LI>Raise amperage up to 20 mA for running the separating gel.
</LI><LI>When the distance of the lowest molecular weight standard lane to the gel end is down to 0.5 cm stop the electrophoresis by turning off the power supply.
</LI></UL><H3><SPAN class="mw-headline" id="Polyacrylamide_gel_staining">Polyacrylamide gel staining</SPAN></H3><H4><SPAN class="mw-headline" id="Colloidal_Coomassie_Blue_staining">Colloidal Coomassie Blue staining</SPAN></H4><P>Modified staining protocol from <A href="http://http://newjournal.kcsnet.or.kr/main/j_search/j_abstract_view.htm?code=B021105&amp;qpage=j_search&amp;spage=b_bkcs&amp;dpage=ar" class="external text" rel="nofollow">Kang <I>et al</I>., 2002</A>.
</P><UL><LI>agitate the <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Colloidal_Coomassie_Brilliant_Blue_G-250_staining_solution" title="Team:Bielefeld-Germany/Labjournal/Molecular">  staining solution</A> at 37 °C over night to form the colloids
</LI><LI>After finishing the SDS-PAGE remove gel from gel casting form and transfer it in to a box.
</LI><LI>Add 100 mL of the staining solution to your polyacrylamid gel.
</LI><LI>Incubate the gel in the solution at room temperature until the protein bands got an intensive blue color. Shake the gel continuously during incubation.
</LI><LI>Remove the staining solution
</LI><LI>Wash the gel with dH<SUB>2</SUB>O
</LI><LI>Incubate the gel in ddH<SUB>2</SUB>O (2-6 h) for bleaching the background. Shake the gel continuously during incubation.  If necessary replace the colored water with new one.
</LI></UL><H3><SPAN class="mw-headline" id="MALDI-TOF">MALDI-TOF</SPAN></H3><UL><LI>Matrix-assisted Laser Desorption/Ionization – Time of flight (MALDI-TOF) is a procedure to analyze large biomolecules by their mass.
</LI></UL><DIV class="thumb tleft"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Our MALDI-TOF device</DIV></DIV></DIV><UL><LI>Before measurement, the analyte is co-crystallized on a metal plate within a solid matrix. 3,5-dimethoxy-4-hydroxycinnamic acid (sinapinic acid) and 2,5-dihydroxybenzoic acid (DHB) are most commonly used.
</LI><LI>Desorption of the analyte is taken by a laser beam, commonly with a wavelength of 337 nm. The matrix absorbs the laser light and the upper layer explosively vaporizes, ejecting both, matrix and analyte molecules. In addition to this vaporization, the laser beam also leads to an ionization of the analytes. The mass analysis is performed by the TOF (time of flight)-analysator. The evaporated ions are accelerated in an electric field. Typical acceleration voltages for this are 10-30 kV. The velocity of an ion depends on its mass and the charge. An ion detector converts the incoming ions into an electrical signal. The advantage of this method is the rapid analysis of a relatively large mass range.
</LI></UL><UL><LI>The MALDI-TOF analysis was performed with the <A href="http://www.bdal.com/products/maldi/ultraflextreme-series/overview.html" class="external text" rel="nofollow">ultrafleXtreme™</A> by Bruker Daltonics.
</LI></UL><UL><LI>We used the MALDI-TOF Analysis to characterize the following BioBricks: 
</LI><LI>&lt;bbpart&gt;K1172201&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;K1172501&lt;/bbpart&gt;</LI><LI>&lt;bbpart&gt;K1172303&lt;/bbpart&gt;</LI></UL><H3><SPAN class="mw-headline" id="Tryptic_digest_of_gel_lanes_for_analysis_with_MALDI-TOF">Tryptic digest of gel lanes for analysis with MALDI-TOF</SPAN></H3><UL><LI>Note:
</LI><LI>Make sure to work under a fume hood.
</LI><LI>Do not work with protective gloves to prevent contamination of your sample with plasticizers.
</LI></UL><UL><LI>Reaction tubes have to be cleaned with 60 % (v/v) CH<SUB>3</SUB>CN and 0.1 % (v/v) TFA. Afterwards the solution has to be removed completely followed by evaporation of the tubes under a fume hood. Alternatively microtiter plates from Greiner® (REF 650161) can be used without washing.
</LI><LI>Cut out the protein lanes of a Coomassie-stained SDS-PAGE using a clean scalpel. Gel parts are transferred to the washed reaction tubes/microtiter plate. If necessary cut the parts to smaller slices.
</LI><LI>Gel slices should be washed two times. Therefore add 200 µL 30 % (v/v) acetonitrile in 0,1 M ammonium hydrogen carbonate each time and shake lightly for 10 minutes. Remove supernatant and discard to special waste.
</LI><LI>Dry gel slices at least 30 minutes in a Speedvac.
</LI><LI>Rehydrate gel slices in 15 µL Trypsin-solution followed by short centrifugation.
</LI><LI>Gel slices have to be incubated 30 minutes at room temperature, followed by incubation at 37 °C over night.
</LI><LI>Dry gel slices at least 30 minutes in a Speedvac.
</LI><LI>According to the size of the gel slice, add 5 - 20 µL 50 % (v/v) ACN / 0,1 % (v/v) TFA.
</LI><LI>Samples can be used for MALDI measurement or stored at -20 °C.
</LI></UL><UL><LI>Trypsin-solution: 1 µL Trypsin + 14 µL 10 mM NH<SUB>4</SUB>HCO<SUB>3</SUB></LI><LI>Therefore solubilize lyophilized Trypsin in 200 µL of provided buffer and incubate for 15 minutes at 30 °C for activation. For further use it can be stored at -20 °C.
</LI></UL><H3><SPAN class="mw-headline" id="Preparation_and_Spotting_for_analysis_of_peptides_on_Bruker_AnchorChips">Preparation and Spotting for analysis of peptides on Bruker AnchorChips</SPAN></H3><UL><LI>Spot 0,5 - 1 µL of sample aliquot
</LI><LI>Add 1 µL HCCA matrix solution to the spotted sample aliquots. Pipet up and down approximately five times to obtain a sufficient mixing. Be careful not to contact the AnchorChip.
</LI><LI>Note: Most of the sample solvent needs to be gone in order to achieve a sufficiently low water content. When the matrix solution is added to the previously spotted sample aliquot at a too high water content in the mixture, it will result in undesired crystallization of the matrix outside the anchor spot area.
</LI></UL><LI>Dry the prepared spots at room temperature
</LI><LI>Spot external calibrants on the adjacent calibrant spot positions. Use the calibrant stock solution (Bruker’s “Peptide Calibration Standard II”, Part number #222570), add 125 µL of 0.1% TFA (v/v) in 30% ACN to the vial. Vortex and sonicate the vial.
</LI><LI>Mix the calibrant stock solution in a 1:200 ratio with HCCA matrix and deposit 1 µL of the mixture onto the calibrant spots.
</LI><H3><SPAN class="mw-headline" id="Substrate_HPLC">Substrate HPLC</SPAN></H3><UL><LI>This method has been used for measurement of the substrate concentration 
</LI><LI>Protocol: 
</LI><LI>Centrifugation (10 min at 14000 g) of the cell culture  
</LI><LI>Mix 900 μL of the supernatant with 2 μL 100 g/L sodium azide solution
</LI><LI>Store the sample at - 20 °C or direct measurement with HPLC </LI></UL><LI>HPLC parameters: 
</LI><LI>Stationary phase: 300 mm x 7.8 mm <I>NUCLEOGEL SUGAR 810H</I> column (<A href="http://www.mn-net.com/HPLCStart/SpecialHPLCphases/Sugars/NUCLEOGELSUGAR810/tabid/10328/language/en-US/Default.aspx" class="external text" rel="nofollow"><I>Macherey Nagel</I></A>) 
</LI><LI>Column temperature: 60 °C.
</LI><LI>Eluent: 2.5 mM H<SUB>2</SUB>SO<SUB>4</SUB>-solution 
</LI><LI>Flow: 0.8 mL min<SUP>-1</SUP></LI><LI>Sample volume: 20 μL with a retention time of 25 min
</LI><LI>Detection: RI detector
</LI><LI>Analysis: Software <I>Chromstar 32</I> version 6.3 (<A href="http://chromstaranalytical.com/index.htm" class="external text" rel="nofollow"><I>Chromstar Analytical</I></A>)
</LI><H3><SPAN class="mw-headline" id="NADH-Assay">NADH-Assay</SPAN></H3><UL><LI>This method has been used for measurement of intracellular NADH concentration.
</LI></UL><UL><LI>Protocol:
</LI><LI>Inoculate an overnight culture (30 mL with 1 mL of pre-culture) 
</LI><LI>Centrifugate (5 min at 5000 g) of 6-10 mL overnight culture (OD 4-6), adjust volume between different samples for the approximation of the number of cells 
</LI><LI>Discard supernatant and wash pellet 3 times with 1 mL of PBS buffer
</LI><LI>Resuspend pellet in 1 mL PBS buffer
</LI><LI>Cell disruption by ribolysation (3 x 30 sec at 6500 rpm) 
</LI><LI>Centrifugation for 10 min at maximum speed
</LI><LI>Store supernatant at - 20 ° C or direct measurement with <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A></LI></UL><UL><LI><A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A> parameters:
</LI><LI>Sample volume = 100 μL clear supernatant
</LI><LI>Excitation = 340 nm
</LI><LI>Emission = 460 nm
</LI><LI>Concentration calculation by NADH calibration curve
</LI></UL><H3><SPAN class="mw-headline" id="Hexadecan_Assay">Hexadecan Assay</SPAN></H3><UL><LI>This assay has been used to measure cell membrane hydrophobicity.
</LI></UL><UL><LI>Protocol
</LI><LI>Inoculate an overnight culture (30 mL with 1 mL of pre-culture)
</LI><LI>Centrifugate (5 min at 4000 g) of 2 mL overnight culture (OD 4-6)
</LI><LI>Discard supernatant and wash pellet 3 times with 1 mL of PBS buffer
</LI><LI>Resuspend pellet in 1 mL 0,9% NaCl and measurement of OD600
</LI><LI>Add x μL of washed cells to 0,9% NaCl for a final volume of 3 mL . Final OD600 should be approximately 0,3 (denoted as A0, calculate exact value)
</LI><LI>Add 3 mL Hexadecan and vortex for 60 sec
</LI><LI>Incubation for 15 min
</LI><LI>Discard the upper organic phase and measure OD600 of the aqueous phase (denoted as A)
</LI><LI>Hydrophobicity can be calculated using the equation: affinity [%] = 100 x [1 – (A/A0)]
</LI></UL><H3><SPAN class="mw-headline" id="Preparing_samples_for_Atomic_Force_Microscopy_.28AFM.29_analysis_of_the_outer_membrane_of_cells">Preparing samples for Atomic Force Microscopy (AFM) analysis of the outer membrane of cells</SPAN></H3><UL><LI>AFM: During the last years, atomic force microscopy has been used increasingly to investigate microbial surfaces at high resolution. The technique provides three-dimensional images of the surface ultrastructure with molecular resolution, in real time, under physiological conditions, and with minimal sample preparation. 
</LI><LI><P align="justify">AFM was carried out using the <A href="http://www.bruker.com/de/products/surface-analysis/atomic-force-microscopy/multimode-8/overview.html" class="external text" rel="nofollow">MultiMode® 8 AFM from Bruker</A>. The measurements were performed on air with ‘Tapping Mode’ and in water with ‘Peak Force Mode’.</P></LI></UL><UL><LI>Cell preparation:
</LI></UL><UL><LI>Inoculating an overnight culture (30 mL with 1 mL of pre-culture) and induction of gene expression if necessary at OD<SUB> 600</SUB> = 1.0.
</LI><LI>Centrifugation (5 min at 5000 g) of 2 mL overnight culture (OD<SUB> 600</SUB>  5 ~ 1 ∙ 10<SUP>9</SUP>  cells/mL). 
</LI><LI>Discard supernatant and wash pellet 3 times with 1 mL of ultra pure water.
</LI><LI>Resuspend pellet in 2 mL ultra pure water (2 ∙ 10<SUP>9</SUP>  cells).
</LI></UL><UL><LI>Preparing of AFM glass slides (0.5 cm<SUP>2</SUP>):
</LI></UL><UL><LI>10 minutes sonication of glass slides in acetone.
</LI><LI>10 minutes sonication of glass slides in ethanol.
</LI><LI>10 minutes sonication of glass slides in ultra pure water.
</LI><LI>Dry glass slides under gentle N<SUB>2</SUB> flow.
</LI></UL><UL><LI>Coating of AFM glass slides with prepared cells:
</LI></UL><UL><LI>Drop 5 μL of a 1:4 dilution of prepared cells (1,25 ∙ 10<SUP>6</SUP> cells) to the prepared glass slides for a submonolayer coverage.
</LI><LI>Dry glass surface with cells under gentle N<SUB>2</SUB> flow.
</LI><LI>Wash glass surface three times with ultra pure water and dry between under gentle N<SUB>2</SUB> flow.
</LI><LI>The coated glass slides were kept cool until AFM measurement.
</LI></UL><H3><SPAN class="mw-headline" id="NPN_membrane_permeability_assay">NPN membrane permeability assay</SPAN></H3><UL><LI>Cell preparation:
</LI></UL><UL><LI>Inoculate an overnight culture (30 mL with 1 mL of pre-culture).
</LI><LI>Centrifugation (8 min at 3000 g) of the whole overnight culture and discard supernatant.
</LI><LI>Resuspend pellet in equal volume of 10 mM PBS buffer.
</LI><LI>Repeat steps 2) and 3) twice.
</LI><LI>Mix 1 mL of washed cells with a 10 mM stock solution of NPN (1 N phenylnaphthylamine) to a final concentration of 10 µM.
</LI><LI>Transfer 200 µL of samples to a flat bottomed black 96 well plate. 
</LI><LI>Direct fluorescence measurement with <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A>.
</LI></UL><UL><LI><A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A> parameters:
</LI></UL><UL><LI>Sample volume = 200 μL 
</LI><LI>Excitation = 355 nm
</LI><LI>Emission Bandwidth = 280 – 550 nm
</LI></UL><H3><SPAN class="mw-headline" id="ONPG_membrane_permeability_assay">ONPG membrane permeability assay</SPAN></H3><UL><LI>Cell preparation:
</LI></UL><UL><LI>Inoculate an overnight culture (30 mL with 1 mL of pre-culture).
</LI><LI>Centrifugation (8 min at 3000 g) of the whole overnight culture and discard supernatant.
</LI><LI>Resuspend pellet in equal volume of 10 mM PBS buffer.
</LI><LI>Repeat steps 2) and 3) twice.
</LI><LI>Inside a clear, flat bottomed 96 well plate, mix 180 µL ONPG (o-nitrophenyl-b-galactopyranoside) stock solution (3 mM in 10 mM PBS buffer) with 20 µL of washed cells.
</LI><LI>Direct absorbance measurement with <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A>.
</LI></UL><UL><LI><A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A> parameters:
</LI></UL><UL><LI>Sample volume = 200 μL 
</LI><LI>Absorbance = 405 nm
</LI></UL><H3><SPAN class="mw-headline" id="Cold_osmotic_shock">Cold osmotic shock</SPAN></H3><UL><LI>Release of periplasmic protein fraction from <I>E. coli</I> by cold osmotic shock
</LI></UL><UL><LI>Modified protocol from <A href="http://www.jbc.org/content/240/9/3685.full.pdf+html?sid=4a90c176-0ec3-489f-8c82-4734274cebf5" class="external text" rel="nofollow">Neu &amp; Heppel, 1965</A>.
</LI><LI>Centrifuge <I>E. coli</I> cell suspension for 5 min at 14,000 g (4 °C) to collect the cells
</LI><LI>Discard the entire supernatant
</LI><LI>Resuspend the cells in ice-cold <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Cell_Fractioning_Buffers" title="Team:Bielefeld-Germany/Labjournal/Molecular">cell fractionating buffer 1</A>. The resulting volume should be 1/4 of the former suspension volume
</LI><LI>Incubate for 20 min on ice. Invert the suspension at regular intervals to counteract sedimentation
</LI><LI>Centrifuge the cell suspension for 15 min at 14,000 g (4 °C)
</LI><LI>Discard the entire supernatant
</LI><LI>Resuspend the cells in ice-cold <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Cell_Fractioning_Buffers" title="Team:Bielefeld-Germany/Labjournal/Molecular">cell fractionating buffer 2</A>. The resulting volume should be 1/4 of the former suspension volume
</LI><LI>Incubate for 10 up to 20 min on ice under regular invertion
</LI><LI>Centrifuge the cell suspension for 15 min at 14,000 g (4 °C)
</LI><LI>Save the supernatant, which contains the periplasmatic proteins and membrane proteins when using <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Cell_Fractioning_Buffers" title="Team:Bielefeld-Germany/Labjournal/Molecular">Cell fractionating buffer 2.2</A> and <A href="/Team:Bielefeld-Germany/Labjournal/Molecular#Cell_Fractioning_Buffers" title="Team:Bielefeld-Germany/Labjournal/Molecular">2.3</A></LI><LI>If the periplasmatic protein fraction is turbid, re-centrifuge and filter it through a 0.2 µm filter
</LI></UL><H3><SPAN class="mw-headline" id="Riboflavin_analysis">Riboflavin analysis</SPAN></H3><P><B>HPLC-Method</B></P><UL><LI> Procedure:
</LI><LI> Riboflavin was separated from disturbing substances on a C18 column and detected with a fluorescence detector.
</LI></UL><LI> Elution: 
</LI><LI> The sample was injected onto a C18-reversed phase column and eluted with a buffer consisting of 99 % 0.01 M sodium acetate pH 7.4 and 1 % tetrahydrofurane. Total flow is set to 1 ml/min
</LI><LI> Detection: 
</LI><LI> Excitation wavelength of 436 nm and emission wavelength of 535 nm
</LI><UL><LI> C18 reverse phase column
</LI><LI> Isocratic method: 99 % 0.01 M sodium acetate pH 7.4 + 1 % tetrahydrofurane
</LI><LI> Flow = 1 mL min-1
</LI><LI> UV-detection at 535 nm
</LI><LI> Internal standard: 5.31 x 10^-5 mg L<SUP>-1</SUP> Riboflavin
</LI><LI> Column:
</LI><LI> Eurospher II 100-5 C18p by Knauer
</LI><LI> Dimensions: 150 x 4.6 mm with precolumn
</LI><LI> Particle size: 5 µm
</LI><LI> Pore size: 100 Å
</LI><LI> Material: silica gel 
</LI></UL><LI> Software:
</LI><LI> Clarity (Version 3.0.5.505) by Data Apex 
</LI><LI> Autosampler:
</LI><LI> Midas by Spark Holland
</LI><LI> Tray cooling: 10 °C 
</LI><LI> Pump:
</LI><LI> L-6200A Intelligent Pump by Hitachi 
</LI><LI> UV-Detector:
</LI><LI> Series 1050 by Hewlett Packard 
</LI><P><B>LC-ESI-qTOF-MS(-MS)</B><B>HPLC method</B></P><UL><LI> Column: C18 reversed phase (Cogent <A href="http://www.mtc-usa.com/diamond_hydride_specs.asp" class="external text" rel="nofollow">Diamond Hydride</A>)
</LI><LI> dimension: 150 x 2,1 mm
</LI><LI> Pore size: 100 Å
</LI><LI> Particle size: 4 µm
</LI></UL><LI> Flow: 0.4 mL min<SUP>-1</SUP></LI><LI> Column temperature: 40 °C
</LI><LI> Injection: 5 µl
</LI><LI> Prerun: 5 min
</LI><LI> Eluent:
</LI><LI> A = 50 % acetonitrile + 50 % H<SUB>2</SUB>O + 0,1 % formic acid
</LI><LI> B = 90 % acetonitrile + 10 % H<SUB>2</SUB>O + 0,1 % formic acid
</LI><LI> Step gradient: 
</LI><LI> 0:00 min: 0 % eluent A
</LI><LI> 8:00 min: 100 % eluent A
</LI><LI> 13:00 min: 100 % eluent A
</LI><LI> 15:30 min: 0 % eluent A
</LI><LI> 18:00 min: 0 % eluent A
</LI><LI> VWR Hitachi LaChrom ULTRA HPLC equipment
</LI><LI> Software: HyStar 3.2, mircrOTOF Control, DataAnalysis 4.0
</LI><P><B>Ionization method</B></P><UL><LI> Using Bruker Daltonics ESI-QTOF micrOTOF<SUB>Q</SUB></LI><LI> ESI-QTOF-MS in positive mode
</LI><LI> Mass range: 50 - 1500 m/z
</LI><LI>Source:
</LI><LI> End plate offset: - 500 V, 107 nA
</LI><LI> Capillary: - 3500 V, 4 nA
</LI><LI> Nebulizer: 2 bar
</LI><LI> Dry gas: 8 L min<SUP>-1</SUP></LI><LI> Dry Temp.: 180 °C
</LI></UL><LI> Transfer:
</LI><LI> Funnel 1 RF: 200 Vpp
</LI><LI> Funnel 2 RF: 300 Vpp
</LI><LI> ISCID Energy: 0 eV
</LI><LI> Heyapole RF: 80 Vpp
</LI><LI> Quadrupole
</LI><LI> Ion energy: 4 eV
</LI><LI> Low mass: 50 m/z
</LI><LI> Collision Cell:
</LI><LI> Collision energy: 8 eV
</LI><LI> Collision RF: 130 Vpp
</LI><LI> Transfer time: 75 µs
</LI><LI> Pre puls storage: 4 µs
</LI><P><B>MS-MS</B></P><UL><LI> Isolated mass: 243.1 +/- 0.1
</LI><LI> Collision energy: 30 eV
</LI></UL><H3><SPAN class="mw-headline" id="Redox_activity_of_cytochromes_via_absorption_spectroscopy">Redox activity of cytochromes via absorption spectroscopy</SPAN></H3><UL><LI>Fractioning by <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Molecular#Cold_osmotic_shock" class="external text" rel="nofollow">cold osmotic shock</A></LI><LI>Dilution until measured spectrum is within the visible range
</LI><LI>Exposure to air to facilitate oxidation of the cytochromes
</LI><LI>Measurement of oxidized samples
</LI><LI>Addition of a few grains of sodium dithionite 
</LI><LI>Wait for approx. 10min
</LI><LI>Measurement of reduced samples
</LI><LI>Comparison of characteristic peaks
</LI><LI>Peak shifts oxidized to reduced
</LI><LI>Soret band: 410nm → 420nm
</LI><LI>β- and α-bands :broad, undefined peak → 525nm and 552nm
</LI></UL><LI>Measurements were performed with the <A href="http://www.tecan.com/platform/apps/product/index.asp?MenuID=1812&amp;ID=1916&amp;Menu=1&amp;Item=21.2.10.1" class="external text" rel="nofollow">Tecan Infinite® M200 platereader</A></LI><H2><SPAN class="mw-headline" id="Cultivation">Cultivation</SPAN></H2><UL><LI>These are the protocols for the cultivations and the downstream processing.
</LI></UL><H3><SPAN class="mw-headline" id="Cultivation_Method_for_UC_Davis_Collaboration">Cultivation Method for UC Davis Collaboration</SPAN></H3><UL><LI>Plasmid-DNA of the Anderson promoters J23101, J23108, J23109 and J23111-J23119 was rehydrated in 10 µL ddH<SUB>2</SUB>O
</LI><LI>Plasmid-DNA was transformed into competent cells of <I>E.coli</I>-strain DH5α using transformation protocol
</LI><LI>Each part was cultivated in LB-medium containing 20 mg/L chloramphenicol to an OD<SUB>700</SUB>  of 0.5 at 37°C with 120 rpm orbital shaking
</LI><LI>Strain control was cultivated under the same condutions without chloramphenicol
</LI><LI>Parts were diluted to a starting OD<SUB>700</SUB> of 0.01 in 200 µL LB and transferred into a clear, flat-bottomed 96-well plate
</LI><LI>Wells were covered by a gas-permeable foil
</LI><LI>The microplate reader(Tecan infinite 200) was ran with the following settings
</LI><LI>	Read duration: 18 hours
</LI><LI>	Read frequency: Once every ten minutes
</LI><LI>	Orbital shaking, amplitude 2.5 mm, frequency 244.5 rpm
</LI><LI>	Absorbance measurement at 700 nm
</LI><LI>	Fluorescence measurement at 588 nm excitation wavelength and 615 nm emission wavelength
</LI></UL><H3><SPAN class="mw-headline" id="Cultivation_Method_for_Riboflavin_Measurements_in_supernatans">Cultivation Method for Riboflavin Measurements in supernatans</SPAN></H3><UL><LI> Most importantly, all cultivations regarding riboflavin production where carried out in <A href="https://2013.igem.org/Team:Bielefeld-Germany/Labjournal/ProtocolsPrograms#M9_Medium" class="external text" rel="nofollow">M9 medium</A>. 
</LI></UL><DL><DD><UL><LI>One ml of a glycerol culture was poured in 30 ml of M9 medium. Cells were grown overnight.
</LI><LI>100 ml of M9 medium were inoculated with an adjusted amount of overnight culture, so that the starting cell density was identical in different shaking flasks.
</LI><LI>Two samples a one ml were transferred to 1,5 ml microcentrifuge tubes.
</LI><LI>The samples were spun down for 5 min at maximum speed. Afterwards the supernatant was transferred to fresh microcentrifuge tubes.
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Materials">Materials</SPAN></H2><UL><LI>Chemicals, enzymes and kits we used in our lab work.
</LI><LI>Chemical and biological reactions need defined conditions to work as expected. The chemicals, enzymes, kits, buffers and media we used in our project are listed in this section.
</LI></UL><H3><SPAN class="mw-headline" id="Used_enzymes">Used enzymes</SPAN></H3><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Enzyme </TH><TH style="padding-left:5px;">Producer
</TH></TR><TR><TD><I>DpnI</I></TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p296" class="external text" rel="nofollow">Fermentas</A></TD></TR><TR><TD><I>EcoRI</I></TD><TD style="padding-left:5px;"><A href="https://www.neb.com/products/r0101-ecori" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD> GoTaq DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/pcr/routine-pcr/gotaq-pcr-core-systems/" class="external text" rel="nofollow">Promega</A></TD></TR><TR><TD> KOD Hotstart DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.merck-chemicals.com/germany/life-science-research/kod-hot-start-dna-polymerase/EMD_BIO-71086/p_iFCb.s1O874AAAEj2Bl9.zLX" class="external text" rel="nofollow">Novagen</A></TD></TR><TR><TD> OneTaq DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.neb.com/nebecomm/products/productM0480.asp" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD><I>Pfu</I> DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/pcr/routine-pcr/pfu-dna-polymerase/" class="external text" rel="nofollow">Promega</A></TD></TR><TR><TD><I>PstI</I></TD><TD style="padding-left:5px;"><A href="https://www.neb.com/products/r0140-psti" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD> Phusion HF DNA-polymerase </TD><TD style="padding-left:5px;"><A href="http://www.finnzymes.com/pcr/phusion_high_fidelity_dna_polymerase.html" class="external text" rel="nofollow">Finnzymes</A></TD></TR><TR><TD> Antarctic phosphatase </TD><TD style="padding-left:5px;"><A href="https://www.neb.com/products/m0289-antarctic-phosphatase" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD><I>SpeI</I></TD><TD style="padding-left:5px;"><A href="https://www.neb.com/products/r0133-spei" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD> T4-DNA-Ligase </TD><TD style="padding-left:5px;"><A href="https://www.neb.com/products/m0202-t4-dna-ligase" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD> T5 exonuclease </TD><TD style="padding-left:5px;"><A href="http://www.neb.com/nebecomm/products/productM0363.asp" class="external text" rel="nofollow">NEB</A></TD></TR><TR><TD><I>XbaI</I></TD><TD style="padding-left:5px;"><A href="https://www.neb.com/products/r0145-xbai" class="external text" rel="nofollow">NEB</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Used_Kits">Used Kits</SPAN></H3><TABLE class="wikitable" style="text-align:left"><TBODY><TR><TH> Function </TH><TH style="padding-left:5px;"> Name
</TH></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"><A href="http://www.fermentas.de/product_info.php?info=p874" class="external text" rel="nofollow">Thermo Scientific GeneJET™ Plasmid Miniprep Kit</A></TD></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"><A href="http://www.promega.de/resources/protocols/technical-bulletins/0/wizard-plus-sv-minipreps-dna-purification-system-protocol" class="external text" rel="nofollow">Promega Wizard® Plus SV Minipreps DNA Purification System</A></TD></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"><A href="http://www.analytik-jena.de/de/life-science/produkte/prod/cat/isolierung-von-plasmid-dna/prod/innuprep-plasmid-mini-kit.html" class="external text" rel="nofollow">Analytik Jena Biometra innuPREP Plasmid Mini Kit</A></TD></TR><TR><TD> Plasmid purification </TD><TD style="padding-left:5px;"><A href="http://www.qiagen.com/products/catalog/sample-technologies/dna-sample-technologies/plasmid-dna/qiaprep-spin-miniprep-kit#orderinginformation" class="external text" rel="nofollow">Qiagen QIAprep Spin Miniprep Kit</A></TD></TR><TR><TD> PCR Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.mn-net.com/tabid/10745/default.aspx" class="external text" rel="nofollow">Macherey Nagel NucleoSpin® Extract II</A></TD></TR><TR><TD> PCR Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/dna-and-rna-purification/dna-fragment-purification/wizard-sv-gel-and-pcr-clean_up-system/" class="external text" rel="nofollow">Promega Wizard® SV Gel and PCR Clean-Up</A></TD></TR><TR><TD> PCR and Gel Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.mn-net.com/tabid/1452/default.aspx" class="external text" rel="nofollow">Macherey Nagel NucleoSpin® Gel and PCR Clean-up</A></TD></TR><TR><TD> PCR and Gel Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.promega.de/products/dna-and-rna-purification/dna-fragment-purification/wizard-sv-gel-and-pcr-clean_up-system/?origUrl=http%3a%2f%2fwww.promega.com%2fproducts%2fdna-and-rna-purification%2fdna-fragment-purification%2fwizard-sv-gel-and-pcr-clean_up-system%2f" class="external text" rel="nofollow">Promega Wizard® SV Gel and PCR Clean-up System</A></TD></TR><TR><TD> PCR and Gel Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.analytik-jena.de/de/life-science/produkte/prod/cat/cleanup-produkte/prod/innuprep-gel-extraction-kit.html" class="external text" rel="nofollow">Analytik Jena Biometra innuPREP Gel Extraction Kit</A></TD></TR><TR><TD> PCR and Gel Cleanup </TD><TD style="padding-left:5px;"><A href="http://www.thermoscientificbio.com/nucleic-acid-purification/genejet-gel-extraction-kit" class="external text" rel="nofollow">Thermo Scientific GeneJET Gel Extraction Kit</A></TD></TR><TR><TD> PCR core system </TD><TD style="padding-left:5px;"><A href="http://www.promega.com/products/pcr/routine-pcr/gotaq-pcr-core-systems/" class="external text" rel="nofollow">Promega GoTaq® PCR Core System I</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Media.2C_buffer.2C_solutions">Media, buffer, solutions</SPAN></H3><H4><SPAN class="mw-headline" id="Chloramphenicol_stock_solution">Chloramphenicol stock solution</SPAN></H4><UL><LI>Solubilize 20 mg mL<SUP>-1</SUP> Chloramphenicol in 100 % Ethanol
</LI><LI>Store at -20 °C
</LI></UL><H4><SPAN class="mw-headline" id="Ampicillin_stock_solution">Ampicillin stock solution</SPAN></H4><UL><LI>Solubilize 100 mg mL<SUP>-1</SUP> Ampicillin
</LI><LI>Store at -20 °C
</LI></UL><H4><SPAN class="mw-headline" id="DNA_loading_buffer">DNA loading buffer</SPAN></H4><UL><LI>50 % (v/v) glycerol
</LI><LI>1 mM EDTA 
</LI><LI>0.1 % (w/v) bromphenol blue 
</LI><LI>Solve in ddH<SUB>2</SUB>O
</LI></UL><H4><SPAN class="mw-headline" id="LB_medium">LB medium</SPAN></H4><UL><LI>10 g Trypton
</LI><LI>5 g yeast extract
</LI><LI>10 g NaCl 
</LI><LI>12 g Agar-Agar (for plates)
</LI><LI>Adjust pH to 7.4
</LI></UL><H4><SPAN class="mw-headline" id="SOC_medium">SOC medium</SPAN></H4><UL><LI>Add the following components for 900 ml of distilled H<SUB>2</SUB>O:
</LI><LI>20 g Trypton 
</LI><LI>5 g Bacto Yeast Extract
</LI><LI>2 mL of 5 M NaCl
</LI><LI>2.5 ml of 1 M KCl
</LI><LI>10 ml of 1 M MgCl<SUB>2</SUB></LI><LI>10 ml of 1 M MgSO4
</LI><LI>20 ml of 1 M glucose
</LI></UL><DL><DD>Adjust to 1L with distilled H<SUB>2</SUB>O. Sterilize by autoclaving.
</DD></DL><H4><SPAN class="mw-headline" id="Biotin_500x_Stock">Biotin 500x Stock</SPAN></H4><UL><LI>Dissolve 20 mg biotin in 100 mL of 0.05 M NaOH solution and filter sterilize
</LI><LI>Store at 4 °C
</LI><LI>Durable for one year
</LI></UL><H4><SPAN class="mw-headline" id="TAE-Buffer">TAE-Buffer</SPAN></H4><UL><LI>1 L of 50x TAE buffer
</LI><LI>242.48 g Tris
</LI><LI>41.02 g sodium acetate
</LI><LI>18.612 g EDTA
</LI><LI>Adjust pH to 7.8 with acetic acid
</LI><LI>Solve in dH<SUB>2</SUB>O
</LI></UL><UL><LI>Dilute 20 mL 50x stock in 1L dH<SUB>2</SUB>O for 1x Buffer for PAGE  
</LI></UL><H4><SPAN class="mw-headline" id="Cell_Fractioning_Buffers">Cell Fractioning Buffers</SPAN></H4><H5><SPAN class="mw-headline" id="CFB_1_.28pH_8.29">CFB 1 (pH 8)</SPAN></H5><UL><LI>0.2 M Tris
</LI><LI>200 g L<SUP>-1</SUP>  sucrose
</LI><LI>0.1 M EDTA
</LI></UL><H5><SPAN class="mw-headline" id="CFB_2.1_.28pH_8.29">CFB 2.1 (pH 8)</SPAN></H5><UL><LI>0.01 M Tris
</LI><LI>0.005 M MgSO<SUB>4</SUB></LI></UL><H5><SPAN class="mw-headline" id="CFB_2.2_.28pH_8.29">CFB 2.2 (pH 8)</SPAN></H5><UL><LI>0.01 M Tris
</LI><LI>0.005 M MgSO<SUB>4</SUB></LI><LI>1% Trition
</LI><LI>2% SDS
</LI></UL><H5><SPAN class="mw-headline" id="CFB_2.3_.28pH_8.29">CFB 2.3 (pH 8)</SPAN></H5><UL><LI>0.01 M Tris
</LI><LI>0.005 M MgSO<SUB>4</SUB></LI><LI>1% Trition
</LI><LI>0.2% SDS
</LI></UL><H5><SPAN class="mw-headline" id="CFB_2.4_.28pH_8.29">CFB 2.4 (pH 8)</SPAN></H5><UL><LI>0.01 M Tris
</LI><LI>0.005 M MgSO<SUB>4</SUB></LI><LI>1% Trition
</LI><LI>1% SDS
</LI></UL><H4><SPAN class="mw-headline" id="SDS-PAGE_gel">SDS-PAGE gel</SPAN></H4><P>The following amounts are for one gel.
Stacking gel 5 %:
</P><UL><LI>775 μL H<SUB>2</SUB>O
</LI><LI>1.25 mL 0,25 M Tris (pH 6,8)
</LI><LI>425 μL Bis/Acrylamide (0,8 %, 30 %)
</LI><LI>50 μL 5 % SDS
</LI><LI>25 μL 10 % Ammonium persulfate
</LI><LI>3 μL TEMED
</LI></UL><P>
Separating gel 12 %:
</P><UL><LI>1.5 mL H<SUB>2</SUB>O
</LI><LI>2.8 mL 1 M Tris (pH 8,8)
</LI><LI>3.0 mL Bis/ Acrylamide (0,8%, 30%)
</LI><LI>150 μL 5% SDS
</LI><LI>37.5 μL 10% Ammonium persulfate
</LI><LI>5 μL TEMED
</LI></UL><H4><SPAN class="mw-headline" id="SDS_running_buffer">SDS running buffer</SPAN></H4><UL><LI>25 mM Tris [pH 8,3]
</LI><LI>192 mM Glycerol
</LI><LI>0.1 % SDS
</LI></UL><H4><SPAN class="mw-headline" id="4x_Laemmli-buffer">4x Laemmli-buffer</SPAN></H4><UL><LI>250 mM Tris-HCl
</LI><LI>40 % [v/v] Glycerol
</LI><LI>20 % [v/v] 2-Mercapthoethanol
</LI><LI>80 g L<SUP>-1</SUP> SDS
</LI><LI>0.04 g L<SUP>-1</SUP> BPB
</LI></UL><H4><SPAN class="mw-headline" id="Colloidal_Coomassie_Brilliant_Blue_G-250_staining_solution">Colloidal Coomassie Brilliant Blue G-250 staining solution</SPAN></H4><UL><LI>for 1 L staining solution
</LI></UL><UL><LI>dissolve 50 g L<SUP>-1</SUP> (NH4)<SUB>2</SUB>SO<SUB>4</SUB> in ddH<SUB>2</SUB>O
</LI><LI>add 10 % (v/v) ethanol
</LI><LI>dissolve 0.2 g L<SUP>-1</SUP> Coomassie Brilliant Blue G-250
</LI><LI>add 2 % (v/v) phosphoric acid
</LI><LI>fill up to 1 L with ddH<SUB>2</SUB>O
</LI></UL><H4><SPAN class="mw-headline" id="5x_isothermal_reaction_buffer_for_Gibson_assembly">5x isothermal reaction buffer for Gibson assembly</SPAN></H4><UL><LI>storage -20˚C
</LI></UL><UL><LI>3 mL of 1 M Tris-HCl (pH 7.5)
</LI><LI>150 µL of 2 M MgCl<SUB>2</SUB>,
</LI><LI>60 µL of 100 mM dGTP
</LI><LI>60 µL of 100 mM dATP
</LI><LI>60 µL of 100 mM dTTP,
</LI><LI>60 µL of 100 mM dCTP
</LI><LI>300 µL of 1 M DTT
</LI><LI>1.5 g PEG-8000 and
</LI><LI>300 µL of 100 mM NAD
</LI></UL><H4><SPAN class="mw-headline" id="Geobacter_Medium"><I>Geobacter</I> Medium</SPAN></H4><UL><LI>1.5 g	NH<SUB>4</SUB>Cl<SUB>2</SUB></LI><LI>0.6 g	Na<SUB>2</SUB>HPO4
</LI><LI>0.1 g	KCl
</LI><LI>0.82 g	Na-acetate
</LI><LI>10 mL	100 X Trace element solution
</LI><LI>10 mL	100 X Vitamin solution
</LI><LI>1 mL	1000 X Selenite-tungstate solution
</LI><LI>8 g	Na-fumarate
</LI><LI>2.5 g 	NaHCO<SUB>3</SUB></LI><LI>980 mL	Distilled water
</LI></UL><UL><LI><B>100 X Trace element solution</B></LI><LI>1.5 g		Nitrilotriacetic acid
</LI><LI>3 g		MgSO<SUB>4</SUB> x 7 H<SUB>2</SUB>O
</LI><LI>0.5 g		MnSO<SUB>4</SUB> x H<SUB>2</SUB>O
</LI><LI>1 g 		NaCl
</LI><LI>0.1 g		FeSO<SUB>4</SUB> x 7 H<SUB>2</SUB>O
</LI><LI>0.18 g		CoSO<SUB>4</SUB> x 7 H<SUB>2</SUB>O
</LI><LI>0.1 g 		CaCl<SUB>2</SUB> x 2 H<SUB>2</SUB>O
</LI><LI>0.18 g		ZnSO<SUB>4</SUB> x 7 H<SUB>2</SUB>O
</LI><LI>0.01 g 		CuSO<SUB>4</SUB> x 5 H<SUB>2</SUB>O
</LI><LI>0.02 g		KAl(SO<SUB>4</SUB>)<SUB>2</SUB> x 12 H<SUB>2</SUB>O
</LI><LI>0.01 g 		H<SUB>3</SUB>BO<SUB>3</SUB></LI><LI>0.01 g		Na<SUB>2</SUB>MoO4 x 2 H<SUB>2</SUB>O
</LI><LI>0.03 g		NiCl<SUB>2</SUB> x 2 H<SUB>2</SUB>O
</LI><LI>0.3 mg		Na<SUB>2</SUB>SeO<SUB>3</SUB></LI><LI>1000 mL	Distilled water
</LI></UL><DL><DD><UL><LI>First dissolve nitrilotriacetic acid and adjust pH to 6.5 with KOH, then add minerals and adjust final pH to 7.0 with KOH.
</LI></UL></DD></DL><UL><LI><B>100 X Vitamin Solution</B></LI><LI>2 mg 		Biotin
</LI><LI>2 mg		Folic acid
</LI><LI>10 mg		Pyridoxine-HCl
</LI><LI>5 mg		Thiamine-HCl x 2 H<SUB>2</SUB>O
</LI><LI>5 mg 		Riboflavin
</LI><LI>5 mg		Nicotinic acid
</LI><LI>5 mg		D-Ca-pantothenate
</LI><LI>0,1 mg		Vitamin B<SUB>12</SUB></LI><LI>5 mg		p-Aminobenzoic acid
</LI><LI>5 mg		Lipoic acid<SUB>12</SUB></LI><LI>1000 mL	Distilled water
</LI></UL><UL><LI><B>1000 X Selenite-tungstate solution</B></LI><LI>0,5 g 		NaOH
</LI><LI>3 mg		Na<SUB>2</SUB>SeO3 x 5 H<SUB>2</SUB>O
</LI><LI>4 mg		Na<SUB>2</SUB>WO4 x H<SUB>2</SUB>O
</LI><LI>1000 mL	Distilled water
</LI></UL><DL><DD><UL><LI>Dissolve all ingredients, except fumarate and NaHCO<SUB>3</SUB>, bring to a boil, cool down to room temperature while gassing the medium with N<SUB>2</SUB> and add the NaHCO<SUB>3</SUB>. Add filter sterilized medium and 0.5 mL/10mL medium from an anoxic filter sterilized 16 % w/v Na-fumarate stock solution to autoclaved cultivation-tubes by injecting it through the silicone stoppers, using a second cannula to prevent overpressure.
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="M9_Medium">M9 Medium</SPAN></H4><UL><LI>To prepare 1 liter of M9 minimal medium add the following components to 836.7 mL sterile distilled water. To avoid precipitation, start with CaCl<SUB>2</SUB> and mix the solution thoroughly after addition of a component.
</LI></UL><UL><LI>100 mL		10 X M9 salt solution
</LI><LI>50 mL		20 X carbon source
</LI><LI>10 mL 		100 X trace element solution
</LI><LI>1 mL		autoclaved 1 M MgSO<SUB>4</SUB></LI><LI>0.3 mL		autoclaved 1 M CaCl<SUB>2</SUB></LI><LI>1 mL		filter sterilized 1 g/L biotin
</LI><LI>1 mL		filter sterilized 1 g/L thiamin
</LI></UL><UL><LI><B>10X M9 salt stock solution</B></LI><LI>75.2 g		Na<SUB>2</SUB>HPO<SUB>4</SUB> x 2H<SUB>2</SUB>O
</LI><LI>30 g		KH<SUB>2</SUB>PO<SUB>4</SUB></LI><LI>5 g		NaCl
</LI><LI>5 g		NH<SUB>4</SUB>Cl
</LI><LI>Dissolve the salts in 800 mL water and adjust the pH to 7.2 with NaOH. Add water to a final volume of 1 L and autoclave for 15 minutes at 121 °C.
</LI></UL><UL><LI><B>20X Carbon source stock solution</B></LI></UL><DIV align="center"><TABLE class="wikitable" style="text-align:center"><TBODY><TR><TH>carbon source -	</TH><TH> Concentration g/L 	</TH><TH> Concentration mol/L	</TH><TH> Concentration mol<SUB>C</SUB>/L	</TH><TH> Final concentration mmol/L
</TH></TR><TR><TD>glucose </TD><TD>	200 </TD><TD>	1,11</TD><TD rowspan="3">	6,66</TD><TD>	55,55
</TD></TR><TR><TD>glycerol	</TD><TD>204,46</TD><TD>	2,22	</TD><TD>	111
</TD></TR><TR><TD>NA-acetate	</TD><TD>273,15	</TD><TD>3,33	</TD><TD>	166,5
</TD></TR></TBODY></TABLE></DIV><DL><DD><UL><LI>To produce a 20X stock solution for a single carbon source dissolve the corresponding amount of the carbon source in distilled water and autoclave for 15 minutes at 121 °C.
</LI></UL></DD></DL><UL><LI><B>100X trace element solution</B></LI><LI>5 g		EDTA
</LI><LI>0,83 g		FeCl<SUB>3</SUB> x 6 H<SUB>2</SUB>O
</LI><LI>84 mg		ZnCl<SUB>2</SUB></LI><LI>13 mg		CuCl<SUB>2</SUB> x 2 H<SUB>2</SUB>O
</LI><LI>10 mg		CoCl<SUB>2</SUB> x 6 H<SUB>2</SUB>O
</LI><LI>10 mg		H<SUB>3</SUB>BO<SUB>3</SUB></LI><LI>1,6 mg		MnCl<SUB>2</SUB> x 4 H<SUB>2</SUB>O
</LI><LI>Dissolve 5g EDTA in 800 ml water and adjust the pH to 7.5 with NaOH. Then add the other components and add distilled water to a final volume of 1 L.
</LI><LI>Sterilize the solution over a 0.22-μm filter
</LI></UL></DIV><DIV class="withMenu" id="asdf"><DIV id="rightcol" style="width:210px; height:100%; overflow-y:auto; box-shadow:0px 0px 2px 0px grey;"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Genetic_Engineering"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Genetic Engineering</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Whole_Genome_Isolation"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Whole Genome Isolation</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Generating_electrocompetent_cells"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Generating electrocompetent cells</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Transformation_via_electroporation"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Transformation via electroporation</SPAN></A></LI><LI class="toclevel-2 tocsection-5"><A href="#Transformation_of_Single_Step_.28KRX.29_Competent_Cells_by_Promega"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">Transformation of Single Step (KRX) Competent Cells by Promega</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Standard_BioBrick_Assembly"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Standard BioBrick Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Suffix_Insertion"><SPAN class="tocnumber">1.5.1</SPAN><SPAN class="toctext">Suffix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Prefix_Insertion"><SPAN class="tocnumber">1.5.2</SPAN><SPAN class="toctext">Prefix Insertion</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#Variations"><SPAN class="tocnumber">1.5.3</SPAN><SPAN class="toctext">Variations</SPAN></A></LI></UL><LI class="toclevel-2 tocsection-10"><A href="#NEB_Biobrick_Assembly_Kit"><SPAN class="tocnumber">1.6</SPAN><SPAN class="toctext">NEB Biobrick Assembly Kit</SPAN></A></LI><LI class="toclevel-2 tocsection-11"><A href="#Gibson_Assembly"><SPAN class="tocnumber">1.7</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Preparation_of_DNA_molecules_for_in_vitro_recombination"><SPAN class="tocnumber">1.7.1</SPAN><SPAN class="toctext">Preparation of DNA molecules for in vitro recombination</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#In_vitro_recombination"><SPAN class="tocnumber">1.7.2</SPAN><SPAN class="toctext">In vitro recombination</SPAN></A></LI><LI class="toclevel-2 tocsection-14"><A href="#Restriction_analysis"><SPAN class="tocnumber">1.8</SPAN><SPAN class="toctext">Restriction analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-15"><A href="#Colony_PCR"><SPAN class="tocnumber">1.9</SPAN><SPAN class="toctext">Colony PCR</SPAN></A></LI><LI class="toclevel-1 tocsection-16"><A href="#Analytics"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Analytics</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#Sodium_dodecyl_sulfate_polyacrylamide_gel_electrophoresis_.28SDS-PAGE.29"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE)</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#Pouring_the_polyacrylamide_gel"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Pouring the polyacrylamide gel</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Preparing_the_sample"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">Preparing the sample</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Running_the_gel"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Running the gel</SPAN></A></LI><LI class="toclevel-2 tocsection-21"><A href="#Polyacrylamide_gel_staining"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Polyacrylamide gel staining</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Colloidal_Coomassie_Blue_staining"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Colloidal Coomassie Blue staining</SPAN></A></LI><LI class="toclevel-2 tocsection-23"><A href="#MALDI-TOF"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">MALDI-TOF</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#Tryptic_digest_of_gel_lanes_for_analysis_with_MALDI-TOF"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Tryptic digest of gel lanes for analysis with MALDI-TOF</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Preparation_and_Spotting_for_analysis_of_peptides_on_Bruker_AnchorChips"><SPAN class="tocnumber">2.5</SPAN><SPAN class="toctext">Preparation and Spotting for analysis of peptides on Bruker AnchorChips</SPAN></A></LI><LI class="toclevel-2 tocsection-26"><A href="#Substrate_HPLC"><SPAN class="tocnumber">2.6</SPAN><SPAN class="toctext">Substrate HPLC</SPAN></A></LI><LI class="toclevel-2 tocsection-27"><A href="#NADH-Assay"><SPAN class="tocnumber">2.7</SPAN><SPAN class="toctext">NADH-Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-28"><A href="#Hexadecan_Assay"><SPAN class="tocnumber">2.8</SPAN><SPAN class="toctext">Hexadecan Assay</SPAN></A></LI><LI class="toclevel-2 tocsection-29"><A href="#Preparing_samples_for_Atomic_Force_Microscopy_.28AFM.29_analysis_of_the_outer_membrane_of_cells"><SPAN class="tocnumber">2.9</SPAN><SPAN class="toctext">Preparing samples for Atomic Force Microscopy (AFM) analysis of the outer membrane of cells</SPAN></A></LI><LI class="toclevel-2 tocsection-30"><A href="#NPN_membrane_permeability_assay"><SPAN class="tocnumber">2.10</SPAN><SPAN class="toctext">NPN membrane permeability assay</SPAN></A></LI><LI class="toclevel-2 tocsection-31"><A href="#ONPG_membrane_permeability_assay"><SPAN class="tocnumber">2.11</SPAN><SPAN class="toctext">ONPG membrane permeability assay</SPAN></A></LI><LI class="toclevel-2 tocsection-32"><A href="#Cold_osmotic_shock"><SPAN class="tocnumber">2.12</SPAN><SPAN class="toctext">Cold osmotic shock</SPAN></A></LI><LI class="toclevel-2 tocsection-33"><A href="#Riboflavin_analysis"><SPAN class="tocnumber">2.13</SPAN><SPAN class="toctext">Riboflavin analysis</SPAN></A></LI><LI class="toclevel-2 tocsection-34"><A href="#Redox_activity_of_cytochromes_via_absorption_spectroscopy"><SPAN class="tocnumber">2.14</SPAN><SPAN class="toctext">Redox activity of cytochromes via absorption spectroscopy</SPAN></A></LI><LI class="toclevel-1 tocsection-35"><A href="#Cultivation"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Cultivation</SPAN></A></LI><LI class="toclevel-2 tocsection-36"><A href="#Cultivation_Method_for_UC_Davis_Collaboration"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Cultivation Method for UC Davis Collaboration</SPAN></A></LI><LI class="toclevel-2 tocsection-37"><A href="#Cultivation_Method_for_Riboflavin_Measurements_in_supernatans"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Cultivation Method for Riboflavin Measurements in supernatans</SPAN></A></LI><LI class="toclevel-1 tocsection-38"><A href="#Materials"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2 tocsection-39"><A href="#Used_enzymes"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Used enzymes</SPAN></A></LI><LI class="toclevel-2 tocsection-40"><A href="#Used_Kits"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Used Kits</SPAN></A></LI><LI class="toclevel-2 tocsection-41"><A href="#Media.2C_buffer.2C_solutions"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Media, buffer, solutions</SPAN></A></LI><LI class="toclevel-3 tocsection-42"><A href="#Chloramphenicol_stock_solution"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">Chloramphenicol stock solution</SPAN></A></LI><LI class="toclevel-3 tocsection-43"><A href="#Ampicillin_stock_solution"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">Ampicillin stock solution</SPAN></A></LI><LI class="toclevel-3 tocsection-44"><A href="#DNA_loading_buffer"><SPAN class="tocnumber">4.3.3</SPAN><SPAN class="toctext">DNA loading buffer</SPAN></A></LI><LI class="toclevel-3 tocsection-45"><A href="#LB_medium"><SPAN class="tocnumber">4.3.4</SPAN><SPAN class="toctext">LB medium</SPAN></A></LI><LI class="toclevel-3 tocsection-46"><A href="#SOC_medium"><SPAN class="tocnumber">4.3.5</SPAN><SPAN class="toctext">SOC medium</SPAN></A></LI><LI class="toclevel-3 tocsection-47"><A href="#Biotin_500x_Stock"><SPAN class="tocnumber">4.3.6</SPAN><SPAN class="toctext">Biotin 500x Stock</SPAN></A></LI><LI class="toclevel-3 tocsection-48"><A href="#TAE-Buffer"><SPAN class="tocnumber">4.3.7</SPAN><SPAN class="toctext">TAE-Buffer</SPAN></A></LI><LI class="toclevel-3 tocsection-49"><A href="#Cell_Fractioning_Buffers"><SPAN class="tocnumber">4.3.8</SPAN><SPAN class="toctext">Cell Fractioning Buffers</SPAN></A></LI><LI class="toclevel-4 tocsection-50"><A href="#CFB_1_.28pH_8.29"><SPAN class="tocnumber">4.3.8.1</SPAN><SPAN class="toctext">CFB 1 (pH 8)</SPAN></A></LI><LI class="toclevel-4 tocsection-51"><A href="#CFB_2.1_.28pH_8.29"><SPAN class="tocnumber">4.3.8.2</SPAN><SPAN class="toctext">CFB 2.1 (pH 8)</SPAN></A></LI><LI class="toclevel-4 tocsection-52"><A href="#CFB_2.2_.28pH_8.29"><SPAN class="tocnumber">4.3.8.3</SPAN><SPAN class="toctext">CFB 2.2 (pH 8)</SPAN></A></LI><LI class="toclevel-4 tocsection-53"><A href="#CFB_2.3_.28pH_8.29"><SPAN class="tocnumber">4.3.8.4</SPAN><SPAN class="toctext">CFB 2.3 (pH 8)</SPAN></A></LI><LI class="toclevel-4 tocsection-54"><A href="#CFB_2.4_.28pH_8.29"><SPAN class="tocnumber">4.3.8.5</SPAN><SPAN class="toctext">CFB 2.4 (pH 8)</SPAN></A></LI><LI class="toclevel-3 tocsection-55"><A href="#SDS-PAGE_gel"><SPAN class="tocnumber">4.3.9</SPAN><SPAN class="toctext">SDS-PAGE gel</SPAN></A></LI><LI class="toclevel-3 tocsection-56"><A href="#SDS_running_buffer"><SPAN class="tocnumber">4.3.10</SPAN><SPAN class="toctext">SDS running buffer</SPAN></A></LI><LI class="toclevel-3 tocsection-57"><A href="#4x_Laemmli-buffer"><SPAN class="tocnumber">4.3.11</SPAN><SPAN class="toctext">4x Laemmli-buffer</SPAN></A></LI><LI class="toclevel-3 tocsection-58"><A href="#Colloidal_Coomassie_Brilliant_Blue_G-250_staining_solution"><SPAN class="tocnumber">4.3.12</SPAN><SPAN class="toctext">Colloidal Coomassie Brilliant Blue G-250 staining solution</SPAN></A></LI><LI class="toclevel-3 tocsection-59"><A href="#5x_isothermal_reaction_buffer_for_Gibson_assembly"><SPAN class="tocnumber">4.3.13</SPAN><SPAN class="toctext">5x isothermal reaction buffer for Gibson assembly</SPAN></A></LI><LI class="toclevel-3 tocsection-60"><A href="#Geobacter_Medium"><SPAN class="tocnumber">4.3.14</SPAN><SPAN class="toctext">Geobacter Medium</SPAN></A></LI><LI class="toclevel-3 tocsection-61"><A href="#M9_Medium"><SPAN class="tocnumber">4.3.15</SPAN><SPAN class="toctext">M9 Medium</SPAN></A></LI></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Molecular">http://2013.igem.org/Team:Bielefeld-Germany/Labjournal/Molecular</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Bielefeld-Germany/Labjournal/Molecular" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Bielefeld-Germany/Labjournal/Molecular" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/Molecular&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Bielefeld-Germany/Labjournal/Molecular&amp;oldid=360543" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>