Team:IISc-Bangalore/Experiments
Home
Project
Overview PACMAN PAIR APES Design Experiments Parts Model
Lab
Notebook Protocols Assembly Results InterLab Improve
Software
PhageModifier
Community
Collaborations Human Practices Integrated HP Public Engagement Safety Survey
Team
Members Philosophy Attributions
Experiments
PACMAN
Binding Affinity Assay
PAIR
Secretion Assay
Chemotaxis Assay
Lambda Red recombination
APES
Phage MIC Calculation
UV Mutagenesis
EMS Mutagenesis
mCherry Improvement
Truncation assay
The following is a list of experiments we did to demonstrate the working of one or more parts of our project. Results for the same can be found on the Results page.
PACMAN
Binding Affinity Assay
A direct experimental assay for binding of gp37 to pEtN (phosphoethanolamine) would require a technique like ITC which asks for extremely large concentrations of protein for every run. As such, we decided to take a more structural approach to the assay by using Circular Dichroism measurements as our peek into what is going on at a molecular level. The binding of the ligand will change the structure of the protein giving a minor deviation from the normal spectrum. In the presence of a strongly binding ligand, the spectrum would be shifted further. The difference of the CD spectrum with and without the ligand can thus allow us to qualitatively check its binding affinity for the protein.
PAIR
Secretion Assay
A simple secretion assay for MCP-1 attached to the signal peptides was done to check the secretion efficiencies of both HlyA and PelB. The supernatant obtained after centrifugation of overexpressed flask culture was run on an SDS-PAGE gel (along with the cell pellet) to see if the protein was secreted out successfully.
The band size obtained in the supernatant can be used to determine whether the protein was cleaved after secretion or not.
Chemotaxis Assay
To check whether our recombinant Mcp-1 with and without the signal peptides is active, we used a chemotaxis experiment with the protein purified from E. coli. To get rid of LPS contamination the protein was washed multiple times when bound to the column and was dialysed against a sterile buffer.
For Mcp-1 specifically, a series of wash buffers containing decreasing concentrations of urea were used to dialyse the protein sample and refold it from an initial urea concentration of 8M. For Mcp-1, 8M urea solubilization needed to be done because of the tendency of Mcp-1 to form inclusion bodies when expressed in E. coli.
For the actual assay, we used Ibidi's Âµ-Slide designed for chemotaxis experiments. The slides were filled with the blank buffer on one side and buffer containing 75ng/ml of the protein on the other (final concentration). Following this, weakly adherent THP-1 cells were loaded in the middle of the slide and their movement was tracked over a span of 12 hrs.
Lambda Red recombination
The Lambda red recombination system is a combination of three enzymes (recombinases) isolated from the Lambda phage that together increase the basal DNA recombination efficiency in the cell by around four orders of magnitude. These recombinases - called Beta, Exo and Gam - can be expressed from a plasmid and ensure efficient recombination of a dsDNA carrier inside the cell to its homologous regions in either the bacterial genome or another target strand inside the cell. We added a T7 promoter to ZJU-China's 2014 Part BBa_K1433005 to convert it into a Lambda red generator ( BBa_K2609026 ).
For PAIR, the Mcp-1 CDS fused to a signal peptide was to be recombined into the T4 genome in place of the native endolysin gene with the use of 50bp homology on both sides of the construct and in the presence of the lambda red proteins. We were unable to do so because of our inability to electroporate the ~169 kbp T4 genome in multiple trials.
APES
Phage MIC Calculation
One of the few overlooked problems during the administration of phages for therapeutics is the decision of the dosage that a person will require to treat the infection successfully. We felt that a simple correspondence between antibiotics and phages would prove to be extremely useful in this case.
Using multiple dilutions of both phages and antibiotics, we quantified the percentage survival of bacteria over a wide range of concentrations and created curves that could effectively be used to choose the amount of phage that should be administered to create the same effect as a corresponding amount of antibiotic. Different initial concentrations of cells were incubated with varying concentrations of the agent and the percentage survival after 2 hrs was quantified using a resazurin-based viability dye.
UV Mutagenesis
Phage mutagenesis was induced by exposing phages in a suspension to UV (298nm) for varying amounts of time before propagating them in a susceptible host. The particles obtained after the first propagation were purified from the whole cell lysate and a part of this was used for further mutagenesis. The other part was utilized for a plaque assay to check the viability of phages obtained at each step of mutagenesis and to check their efficacy on resistant bacteria.
EMS Mutagenesis
For chemical mutagenesis, phage particles were incubated with susceptible cells (OD600 0.6, 104 pfu/mL) for 20 mins followed by addition of Ethyl methanesulfonate (EMS). After propogation for 2 hrs, the phages were purified and propogated in the susceptible strain (to ensure amplification of particles that can infect the susceptible strain). They were then extracted from the lysate and were either used for further mutagenesis or to check the susceptibiity of T4 resistant bacteria.
mCherry Improvement
Truncation assay
To quantify the truncation from the internal RBS in imCherry (and mCherry), we used a combination of two strategies. Firstly, a rough estimate was obtained by comparing the band intensities directly from the gel electrophoresis of the total cell lysate after expressing the protein.
To get a higher order estimate on the truncation, we purified the protein using Ni-NTA and measured the fluorescence obtained in all (supernatant, wash, and elution) fractions. In an ideal case, the ratio of fluorescence obtained in the supernatant and elution fractions would give the fraction of protein that was truncated (assuming truncated and non-truncated i/mcherry is equally fluorescent). However, the process of binding of 6xHis to Ni-NTA is not perfect and proteins tend to run off into the supernatant and the washes. To get rid of this experimental error, we performed PAGE with each fraction and divided the total fluorescence of the fraction into two parts - truncated/non-truncated - using the band intensities. The fluorescence for truncated protein from each fraction was then summed up and was divided by the fluorescence of the non-truncated part to get the relative fractions of the two varieties.
The results for the improvement can be found on the Improve page.
Links
IISc Bangalore
IISc UG Program
Complete list of references
Address
Undergraduate department,
Indian Institute of Science,
Bangalore - 560012
Contact
Bhaskar Kumawat
Team Lead
kbhaskar@iisc.ac.in
This website is best viewed on a 16:9 aspect ratio screen on the latest version of Google Chrome
