<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISc-Bangalore_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISc-Bangalore/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P>iGEMwiki toolssearchteamsdoucheno1
</P><P>Editing Team:IISc-Bangalore/Model
</P><P><NAV class="navbar navbar-expand-lg navbar-light fixed-top" style="background-color: #ffffff; top:16px; font-family: bodyfont;"><UL class="navbar-nav collapse navbar-collapse justify-content-end" id="navbarSupportedContent" style="padding-right: 40px;"><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:IISc-Bangalore">Home</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    Project
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Description">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/PACMAN">PACMAN</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/PAIR">PAIR</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/APES">APES</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Parts">Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Model">Model</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        Lab
                    </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Assembly">Assembly</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Software</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Software">PhageModifier</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Community
                        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Integrated_HP">Integrated HP</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Public_Engagement">Public Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Safety">Safety</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Survey">Survey</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    Team
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Team">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Philosophy">Philosophy</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Attributions">Attributions</A></DIV></LI></UL></NAV></P><DIV class="jumbotron" style="background-color: transparent; background-image: url(https://static.igem.org/mediawiki/2018/9/9a/T--IISc-Bangalore--page_modelmama.jpg); height: 300px; background-size: 100%"><H1 style="text-align: center; top: 60px; position: relative; color:white; font-family: headfont;"><B>Modelling</B></H1></DIV><DIV class="container"><H2><B>Overview</B></H2><P style="font-family: bodyfont; font-size: 16px;">Bacteria–phage coevolution, the reciprocal evolution between bacterial hosts and the phages that infect them, is an important part of this year's project. There is growing evidence from both laboratory and natural populations that coevolution can maintain phenotypic and genetic diversity, increase the rate of bacterial and phage evolution and divergence, affect community structure, and shape the evolution of ecologically relevant bacterial traits.<SUP>[6]</SUP> Also, phage dosage has been an elusive and non-trivial concept, proving to be a considerable challenge in the field of therapeutics. Hence we decided to have a go at it by employing delayed ODE-based methods.
Additionally, using a stochastic approach, we modeled induced mutations in bacteriophage T4 to determine feasibility of obtaining mutants capable of binding to resistant bacteria. 

</P><P style="font-family: bodyfont; font-size: 16px;">Two models were devised in the course of the project.</P><OL style="font-size: 16px;"><LI>Numerically analysing the phage-bacteria interaction using modified predator-prey equations.</LI><LI>Calculating the expected number of a particular genome sequence arising from the T4 phage mutations.</LI></OL><H2><B>Coevolution</B></H2><P>Due to the self-replicating nature of phages, some peculiarities arise in the use of phages as an antimicrobial drug. Two modes of therapy are possible with phages, active and passive. Active therapy takes place when the virus population increases along with the bacterial population, eventually overwhelming the bacterial population and drastically reducing bacterial concentration. Passive therapy is when the initial viral population is sufficiently large that the bacterial population is wiped out by primary infection alone, and displays kinetics somewhat similar to antibiotics.<SUP>[1]</SUP></P><P>Some necessary terminology:</P><UL style="font-size: 16px;"><LI>The threshold concentration of bacteria that must be present in order for the virus numbers to increase is known as the <I>proliferation threshold X<SUB>p</SUB></I>, which is required for active therapy.</LI><LI>The <I>viral inundation threshold V<SUB>I</SUB></I> is the minimum concentration of the phage required to cause a decrease in the total concentration of viable bacteria. </LI><LI>The bacteria are completely cleared if the concentration of phage is greater than a certain <I>clearance threshold V<SUB>C</SUB></I>. </LI></UL><P style="font-family: bodyfont; font-size: 16px;">To model coevolution, some assumptions have been taken into account.</P><P style="font-family: bodyfont; font-size: 16px;"><UL style="font-size: 16px;"><LI>Bacterial growth follows the logistic model<SUP>[1]</SUP> in the absence of phages due to the medium being a finite nutrient source.
</LI><LI>The phage-bacteria interaction is first order with respect to both the phage and the bacteria concentrations.<SUP>[2]</SUP></LI><LI>There is a time delay between a bacterial cell getting infected and its subsequent lysis, called the latent period.<SUP>[3]</SUP></LI><LI>Bacteria can undergo mutations and become resistant to the phages. Similarly, the phages can mutate to affect the resistant bacteria. This mutation rate is proportional to the number of bacteria, as each bacteria is equally likely to mutate. The phage mutation is taken into account by assuming that the resistant bacteria mutate back to being susceptible.</LI></UL></P><P style="font-family: bodyfont; font-size: 16px;"> The equations are as follows:
        </P><CENTER><BUTTON data-toggle="collapse" data-target="#demo" class="btn btn-default" style="margin-top: 30px;">Variables and Parameters</BUTTON><DIV id="demo" class="collapse" style="margin: auto;"><H5><B>Variables</B></H5><TABLE class="table table-dark" style="font-family: bodyfont; font-size: 16px;"><THEAD><TR><TH scope="col">Symbol</TH><TH scope="col">Details</TH></TR></THEAD><TBODY><TR><TH scope="row">Susceptible Bacteria</TH><TD>SB</TD><TD>Concentration of bacteria susceptible to infection by phages.</TD></TR><TR><TH scope="row">Resistant Bacteria</TH><TD>RB</TD><TD>Concentration of bacteria resistant to infection by phages.</TD></TR><TR><TH scope="row">Infected Bacteria</TH><TD>IB</TD><TD>Concentration of bacteria infected by phages.</TD></TR><TR><TH scope="row">Free Virions</TH><TD>V</TD><TD>Phage concentration in the medium.</TD></TR><TR><TH scope="row">Lysed Bacteria</TH><TD>LB</TD><TD>Concentration of dead bacteria after lysis.</TD></TR><TR><TH scope="row">Time</TH><TD>t</TD><TD>¯\_(ツ)_/¯</TD></TR></TBODY></TABLE><H5><B>Constants</B></H5><TABLE class="table table-dark" style="font-family: bodyfont; font-size: 16px;"><THEAD><TR><TH scope="col">Symbol</TH><TH scope="col">Details</TH></TR></THEAD><TBODY><TR><TH scope="row">Growth Rate Constant</TH><TD>a</TD><TD>Rate constant for bacterial growth.</TD></TR><TR><TH scope="row">Carrying Capacity</TH><TD>c</TD><TD>Maximum concentration of bacteria supported by the medium.</TD></TR><TR><TH scope="row">Forward Mutation Rate Constant</TH><TD>fr</TD><TD>Rate constant for mutations leading to resistance.</TD></TR><TR><TH scope="row">Backward Mutation Rate Constant</TH><TD>br</TD><TD>Rate constant for mutations leading to loss of resistance.</TD></TR><TR><TH scope="row">Interaction Constant</TH><TD>ir</TD><TD>Rate constant for phage-bacteria interaction.</TD></TR><TR><TH scope="row">Burst Size</TH><TD>bs</TD><TD>Average number of phages released per infected bacterial cell on lysis.</TD></TR><TR><TH scope="row">Decay Rate Constant</TH><TD>dr</TD><TD>Rate constant for phage decay.</TD></TR><TR><TH scope="row">Latent Period</TH><TD>t<SUB>L</SUB></TD><TD>The time delay between infection and lysis.</TD></TR></TBODY></TABLE><H4><B>Dosage Determination</B></H4><P style="font-family: bodyfont; font-size: 16px;">
		To determine the <I>proliferation threshold</I>: Consider a single infected bacterium, which upon lysis, releases <I>bs</I> virions. These virions either degrade, or infect more bacteria. The virions are distributed among the terms 
		</P><DIV class="alignCenter">  (for viral decay)
		</DIV><DIV class="alignCenter">  (for virus-bacteria interaction)
		</DIV>
		The fraction of infected bacteria must hence be
		giving the number of secondary infections per primary cell, <I>R<SUB>0</SUB></I>.
		For the net infected cell count to increase, we must have <I>R<SUB>0</SUB> &gt; 1</I>. Hence, this gives the proliferation threshold as<P style="font-family: bodyfont; font-size: 16px;">
		To determine the <I>inundation threshold</I>: For the viable bacterial concentration to decrease, 
		</P>
		This gives 
                
                The maximum value of this is the inundation threshold, giving 
		<P style="font-family: bodyfont; font-size: 16px;">
		The <I>clearance threshold</I> cannot be determined analytically from these equations. However, for any given value of bacterial concentration, the value can be determined numerically using the following equations, and taking the minimum value of the solutions: 
		</P><H4><B>Numerical Solutions</B></H4><P style="font-family: bodyfont; font-size: 16px;">The problem described above is a system of five differential equations, of which two are so called delayed differential equations. They contain a term that needs to be evaluated at a time-point in the past. A python script was used to simulate the growth curves numerically, using the method of finite differences. Each of the model parameters were optimised using the Levenberg-Marquadt algorithm<SUP>[4][5]</SUP> using the scipy.optimize python library. Initial values for the parameters were taken by using the data found <A href="http://bionumbers.hms.harvard.edu/files/The%20phages%20of%20the%20T%20system.pdf">here</A>, and a few were chosen 	by hand.</P><P style="font-family: bodyfont; font-size: 16px;">The code can be found <A href="https://github.com/AnishB1699/coevolution_model">here.</A></P><H4><B>Experimental Data</B></H4><P style="font-family: bodyfont; font-size: 16px;">The protocol we executed for obtaining the experimental curves is as follows:</P><OL style="font-size: 16px;"><LI>Put 190 μl of Lysogenic Broth (LB) and 10 μl of secondary inoculum at 0.6 OD in each well of a 96 well plate.</LI><LI>To each of the wells, add 2 μl of varying dilutions of phages.</LI><LI>Put the plate into a plate reader and take OD readings for required period of time.</LI></OL><P style="font-family: bodyfont; font-size: 16px;">The experimental data can be found <A href="https://github.com/AnishB1699/coevolution_model">here.</A></P><H4><B>Getting the fit</B></H4><P style="font-family: bodyfont; font-size: 16px;">Combinations of parameters that result in providing a fit closest to the experimental curve can most easily be found by analysing for different sets of parameters systematically.</P><OL style="font-size: 16px;"><LI>The theoretical curve was plotted using the numerical solutions.</LI><LI>Then a cost function was made between the theoretical and actual curve for reasonable values of parameters.</LI><LI>Then scipy.optimize.least_squares was used to minimise this cost and get required parameter values.</LI></OL><P style="font-family: bodyfont; font-size: 16px;">The code for fitting can be found <A href="https://github.com/AnishB1699/coevolution_model">here.</A></P><DIV id="container" style="width:100%"><H4><B>Conclusions</B></H4><P style="font-family: bodyfont; font-size: 16px;">Here are some of the graphs obtained after fitting our model with the experimental curves. For the bacterial concentration graphs, the experimental curve is coloured blue and the theoretical fit, orange. The viral concentrations are in the order of 4 x 10<SUP>4</SUP> pfu.</P><DIV class="row"><DIV class="col-sm-6"><CENTER><SPAN style="font-family: bodyfont; font-size: 12px;">Figure 1: SB(0) = 0.1422, V(0)=0.001</SPAN></CENTER><CENTER><SPAN style="font-family: bodyfont; font-size: 12px;">Figure 3: SB(0) = 0.1422, V(0)=0.001</SPAN></CENTER></DIV><DIV class="col-sm-6"><CENTER><SPAN style="font-family: bodyfont; font-size: 12px;">Figure 2: SB(0) = 0.1408, V(0) = 0.001</SPAN></CENTER><CENTER><SPAN style="font-family: bodyfont; font-size: 12px;">Figure 4: SB(0) = 0.1386, V(0) = 0.01</SPAN></CENTER></DIV></DIV><CENTER><BUTTON data-toggle="collapse" data-target="#demo" class="btn btn-default" style="margin-top: 30px;">Parameters</BUTTON><DIV id="demo" class="collapse" style="margin: auto;"><H5><B>Variables</B></H5><TABLE class="table table-dark" style="font-family: bodyfont; font-size: 16px;"><THEAD><TR><TH scope="col">Symbol</TH><TH scope="col">Average Value</TH></TR></THEAD><TBODY><TR><TH scope="row">Growth rate Constant</TH><TD>a</TD><TD>3.76 × 10<SUP>-3</SUP></TD></TR><TR><TH scope="row">Interaction Constant</TH><TD>ir</TD><TD>6.3 × 10<SUP>-2</SUP></TD></TR><TR><TH scope="row">Burst Size</TH><TD>bs</TD><TD>1.57 × 10<SUP>2</SUP></TD></TR><TR><TH scope="row">Forward Mutation Rate Constant</TH><TD>fr</TD><TD>8.1 × 10<SUP>-1</SUP></TD></TR><TR><TH scope="row">Backward Mutation Rate Constant</TH><TD>br</TD><TD>2.65 × 10<SUP>-2</SUP></TD></TR><TR><TH scope="row">Decay Rate Constant</TH><TD>dr</TD><TD>4.7 × 10<SUP>-3</SUP></TD></TR><TR><TH scope="row">Latent Period</TH><TD>t0</TD><TD>6.3 × 10<SUP>2</SUP></TD></TR><TR><TH scope="row">Carrying Capacity</TH><TD>c</TD><TD>5.6 × 10<SUP>-1</SUP></TD></TR></TBODY></TABLE><CENTER><P>Calculated values of the important dosage parameters: Proliferation threshold <I>X<SUB>p</SUB></I> = 11.1Inundation threshold <I>V<SUB>I</SUB></I> = 6 × 10<SUP>-2</SUP></P></CENTER><P>The conditions required for active therapy could not be attained in this system, as the carrying capacity of the system was lesser than the proliferation threshold.</P><H4><B>Limitations</B></H4><P style="font-family: bodyfont; font-size: 16px;">The curves predicted by the model agreed with the experimental curves only upto a limit, after which there was a deviation from the ideal model. This is because of a few factors:
</P><OL style="font-size: 16px;"><LI>The model did not account for the OD of lysed bacteria, which significantly increases with time.
</LI><LI>In the model, the backward rate of mutation is kept constant, effectively making the model similar to predator-prey dynamics. In reality, the backward rate of mutation is non zero in the beginning, but tends to 0 after some time. This accounts for the experimental curves showing no bacteria-phage interactions over large periods of time. 
</LI></OL><P style="font-family: bodyfont; font-size: 16px;">However, within reasonable time frames, the model predicts the phage-bacteria dynamics quite effectively, making it a useful tool to draw therapeutic correspondence between antibiotics and phage therapy.</P><H2><B>Mutagenesis</B></H2><P>To verify the feasibility of APES, a model was developed in parallel with the design for the hardware. The model was intended to predict the number of mutant phages produced that could infect and lyse the resistant bacteria. </P><P>To model the mutagenesis, the following assumptions were taken into account:
	<UL style="font-size: 16px;"><LI>Each nucleotide in the genome mutates independently, with the probability of mutation following an exponential distribution. </LI><LI>The process is assumed to happen discretely, with mutation from one nucleotide to any nucleotide being equally likely.</LI></UL></P><H4><B>The Transition Matrix</B></H4>
	Let the rate constants for each transition i-&gt;j be given by <I>q<SUB>ij</SUB></I> where both i and j belong to the set {0, 1, 2, 3}. (The state 0 represents the nucleotide with base A, 1 represents base T, 2 represents base G, and 3 represents base C. The base being in the state i implies that the base is currently the base which is repesented by i) We have the constraints
	
	Let  <I>p<SUB>ij</SUB></I> be defined as 
	
	These definitions allow a transition matrix to be defined for the mutation, and to treat it as a discrete Markov process, with a step size given as Δt, a sufficiently small time unit. The transition matrix for the mutations, T, is now given by
	
	where <I>m<SUB>ij</SUB></I> represents the element in the i<SUP>th</SUP> row and j<SUP>th</SUP> column of the matrix. 
	After n steps, the probability of transition from a state i to a state j is now given by the the element in the i<SUP>th</SUP> row and j<SUP>th</SUP> column of T<SUP>n</SUP>. 
<H4><B>The Calculations</B></H4><P>
		The length of the bacteriophage T4 genome is by 168903 base pairs, out of which about 65% is assumed to be essential for the viability of the phage.<SUP>[7]</SUP> Any change in any amino acids in the viable sites is assumed to result in loss in viability. The nucleotides in the genome encoding for the region of the tail fibres responsible for binding to a receptor on the bacterial cell surface are found to be in the range 156542-159622 in the T4 genome.<SUP>[8]</SUP></P><P>
		Given the time of mutation t, the number of steps is given by t/Δt, and hence, the transition matrix is given by T<SUP>n</SUP>. This is numerically computed using python and the numpy library. Using the diagonal elements of the transition matrix, which give the probability of no change in a nucleotides. These probabilities raised to the power of the number of nucleotides gives the probability that a cell remains viable. Next, the probability of transition of the nucleotide sequence between 156542-159622 to the random amino acid sequence chosen is also calculated, using data derived from an amino acid codon chart and the transition matrix. The events that there is no loss in viability and that a mutant develops which is capable of infecting the resistant bacteria are assumed to be independent. Hence, the probability of obtaining a required mutant phage is given by the product of these two probabilities. The expected value of the number of desired phages is found by multiplying the total number of phages subjected to mutagenesis times the probability calculated above.
	</P><P>
		The code used to perform the numerical analysis can be found <A href="https://github.com/AnishB1699/APES_">here.</A></P><H4><B>Conclusions</B></H4><P>
		For a time t=500 s, and <I>q<SUB>ij</SUB></I>=10<SUP>-7</SUP>, the probability of the virus remaining viable was found to be 10<SUP>-49</SUP>. The largest number of phages used for the experiments were of the order of 10<SUP>10</SUP> pfu, which gives an expected value of 10<SUP>-39</SUP> desired phages. This is an astronomically small number, clearly showing it is unreasonable to expect any kind of result from the APES experiments. 
		Increasing the values of <I>q<SUB>ij</SUB></I>=10<SUP>-7</SUP>, that is, using a more effective mutagen, fails to help as well, since it only results in lower amounts of viability. Decreasing the values of <I>q<SUB>ij</SUB></I>=10<SUP>-7</SUP> results in effectively no mutations, once again failing to produce results. </P><P>Thus, our modelling showed that no reasonable value of the mutation rates could produce desirable results. This was a major motivation for the team to decide to halt the APES experiments.
	</P><H5><B>References</B></H5><P id="references">
1]Payne, Robert JH, and Vincent AA Jansen. &quot;Understanding bacteriophage therapy as a density-dependent kinetic process.&quot; Journal of Theoretical Biology 208.1 (2001): 37-48.
2]Jason, A. C. &quot;A deterministic model for monophasic growth of batch cultures of bacteria.&quot; Antonie van Leeuwenhoek 49.6 (1983): 513-536.
3]Levin, Bruce R., Frank M. Stewart, and Lin Chao. &quot;Resource-limited growth, competition, and predation: a model and experimental studies with bacteria and bacteriophage.&quot; The American Naturalist 111.977 (1977): 3-24.
4]Levenberg, Kenneth. &quot;A method for the solution of certain non-linear problems in least squares.&quot; Quarterly of applied mathematics 2.2 (1944): 164-168.
5]Marquardt, Donald W. &quot;An algorithm for least-squares estimation of nonlinear parameters.&quot; Journal of the society for Industrial and Applied Mathematics 11.2 (1963): 431-441.
6]Koskella, Brockhurst. &quot;Bacteria–phage coevolution as a driver of ecological and evolutionary processes in microbial communities.&quot; US National Insitute of Medicine (2014).
7]Dedrick, Rebekah M., et al. &quot;Functional requirements for bacteriophage growth: gene essentiality and expression in mycobacteriophage G iles.&quot; Molecular microbiology 88.3 (2013): 577-589.
8]Bartual, Sergio G., et al. &quot;Structure of the bacteriophage T4 long tail fiber receptor-binding tip.&quot; Proceedings of the National Academy of Sciences 107.47 (2010): 20287-20292.
</P></DIV></CENTER></DIV><FOOTER id="footer"><DIV class="footer-text-boxes"><DIV class="footer-text-boxes-section"><DIV class="footer-text-boxes-section-heading">Links</DIV><UL><LI><A href="http://www.iisc.ac.in" target="_blank">IISc Bangalore</A></LI><LI><A href="https://iisc.ac.in/ug/" target="_blank">IISc UG Program </A></LI><LI><A href="https://2018.igem.org/Team:IISc-Bangalore/References">Complete list of references</A></LI></UL></DIV><A href="https://goo.gl/maps/BPKKwKU3hd32" target="_blank"><DIV class="footer-text-boxes-section"><DIV class="footer-text-boxes-section-heading">Address</DIV><UL><LI>Undergraduate department,</LI><LI>Indian Institute of Science,</LI><LI>Bangalore - 560012</LI></UL></DIV></A><DIV class="footer-text-boxes-section"><DIV><DIV class="footer-text-boxes-section-heading">Contact</DIV><UL><LI>Bhaskar Kumawat</LI><LI>Team Lead</LI><LI><A href="mailto:kbhaskar@iisc.ac.in" target="_blank">kbhaskar@iisc.ac.in</A></LI></UL></DIV></DIV></DIV><DIV class="footer-text-small" style="max-width:700px;">This website is best viewed on a 16:9 aspect ratio screen on the latest version of Google Chrome</DIV></FOOTER></DIV></CENTER></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>