<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_IISc-Bangalore_References skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:IISc-Bangalore/References</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="navbar navbar-expand-lg navbar-light fixed-top" style="background-color: #ffffff; top:16px; font-family: bodyfont;"><UL class="navbar-nav collapse navbar-collapse justify-content-end" id="navbarSupportedContent" style="padding-right: 40px;"><LI class="nav-item"><A class="nav-link" href="https://2018.igem.org/Team:IISc-Bangalore">Home</A></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    Project
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Description">Overview</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/PACMAN">PACMAN</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/PAIR">PAIR</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/APES">APES</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Design">Design</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Experiments">Experiments</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Parts">Parts</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Model">Model</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                        Lab
                    </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Notebook">Notebook</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Protocols">Protocols</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Assembly">Assembly</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Results">Results</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/InterLab">InterLab</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Improve">Improve</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Software</A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Software">PhageModifier</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                            Community
                        </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Collaborations">Collaborations</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Human_Practices">Human Practices</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Integrated_HP">Integrated HP</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Public_Engagement">Public Engagement</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Safety">Safety</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Survey">Survey</A></DIV></LI><LI class="nav-item dropdown"><A class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
                    Team
                </A><DIV class="dropdown-menu" aria-labelledby="navbarDropdown"><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Team">Members</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Philosophy">Philosophy</A><A class="dropdown-item" href="https://2018.igem.org/Team:IISc-Bangalore/Attributions">Attributions</A></DIV></LI></UL></NAV></P><DIV class="jumbotron" style="background-color: white; background-image: url(https://static.igem.org/mediawiki/2018/3/3b/T--IISc-Bangalore--pageReferences.png); height: 300px;"><H1 style="text-align: center; top: 40px; position: relative; color:white; font-family: headfont;">References</H1></DIV><DIV class="container"><P style="font-family: bodyfont; font-size: 16px;">
1. Young R. &quot;Bacteriophage lysis: mechanism and regulation&quot;. Microbiological Reviews. 1992;56(3):430-481<I>One among the most important source materials that helped us in formulating our project idea.</I>2. Eric S. Miller, Elizabeth Kutter, et al,&quot;Bacteriophage T4 Genome&quot;Microbiol. Mol. Biol. Rev. Mar 2003, 67 (1) 86-156; DOI: 10.1128/MMBR.67.1.86-156.2003<I>A complete guide about T4 Bacteriophages. Yet another most important referance for our work.</I>3. Abedon ST, García P, et al, &quot;Editorial: Phage Therapy: Past, Present and Future&quot;. Frontiers in Microbiology. 2017;8:981. doi:10.3389/fmicb.2017.00981.4. Cohen, Fredric S. “How Viruses Invade Cells.” Biophysical Journal 110.5 (2016): 1028–1032. PMC. Web. 12 Oct. 2018.<I>Gave us a detailed idea of functioning of viruses.</I>5. Inamdar, Mandar M., et al, “Dynamics of DNA Ejection from Bacteriophage.” Biophysical Journal 91.2 (2006): 411–420. PMC. Web. 12 Oct. 2018.6. Sagona, Antonia P., Grigonyte, Aurelija M., MacDonald, Paul and Jaramillo, Alfonso. (2016)
&quot;Genetically modified bacteriophages&quot;. Integrative Biology, 8 . pp. 465-474. <I>Gave us information about to a great extent about genetic modification of bacteriophages.</I>7. Pires, Diana P. et al. “Genetically Engineered Phages: A Review of Advances over the Last Decade”. Microbiology and Molecular Biology Reviews : MMBR 80.3 (2016): 523–543. PMC. Web. 12 Oct. 2018.<I>Recent advancements in the genetic modification of bacteriophages.</I>8. Navarro, Ferran, and Maite Muniesa. “Phages in the Human Body.” Frontiers in Microbiology 8 (2017): 566. PMC. Web. 13 Oct. 2018.9. Hadas H, Einav M, et al. &quot;Bacteriophage T4 Development Depends on the Physiology of its Host Escherichia Coli&quot;, Microbiology 143(1):179-185 doi:10.1099/00221287-143-1-179<I>Detailed description of development of T4 as a function of cell size, age, shape, rates of metabolismand chromosome replication, and time of lysis.</I>10. , (2016), Silk route to the acceptance and re‐implementation of bacteriophage therapy. Biotechnology Journal, 11: 595-600. doi:10.1002/biot.201600023<I>This gave us description of various steps to be undertaken for Phage Therapy to go big.</I>11. Maria-Neto S, de Almeida KC, et. al &quot;Understanding bacterial resistance to antimicrobial peptides: From the surface to deep inside&quot;. Biochim Biophys Acta. 2015 Nov;1848(11 Pt B):3078-88. doi: 10.1016/j.bbamem.2015.02.017.<I>Details in the paper was helpful for PACMAN.</I>12. Matsuda, Takeaki et al.&quot;Lysis-deficient bacteriophage therapy decreases endotoxin and inflammatory mediator release and improves survival in a murine peritonitis model&quot;. Surgery , Volume 137 , Issue 6 , 639 - 646<I>Gave us details about the effects of endotoxin release, a concern that arises due to lytic activity of phages.</I>13. Moussa, Samir H et al. “Protein Determinants of Phage T4 Lysis Inhibition.” Protein Science : A Publication of the Protein Society 21.4 (2012): 571–582. PMC. Web. 13 Oct. 2018.<I>Details regarding the pathway for lysis deficiency in T4 bcteriophages.</I>14. Dressman, Holly Kloos, and John W. Drake. “Lysis and Lysis Inhibition in Bacteriophage T4: rV Mutations Reside in the Holin t Gene.” Journal of Bacteriology 181.14 (1999): 4391–4396. Print.<I>Detailed study of lysis deficiency in T4 bacteriophages by modifications to the t-gene.</I>15. Deshmane, Satish L. et al. “Monocyte Chemoattractant Protein-1 (MCP-1): An Overview.” Journal of Interferon &amp; Cytokine Research 29.6 (2009): 313–326. PMC. Web. 13 Oct. 2018.<I>Gave us complete details about MCP-1, important for PAIR.</I>16. “RANTES and Macrophage Inflammatory Protein 1 Alpha Induce the Migration and Activation of Normal Human Eosinophil Granulocytes.” The Journal of Experimental Medicine 176.6 (1992): 1489–1495. Print.<I>Details about the specificity of MCP-1.</I>17. Gu, L. , Rutledge, B. , Fiorillo, J. , Ernst, C. , Grewal, I. , Flavell, R. , Gladue, R. and Rollins, B. (1997), &quot;In vivo properties of monocyte chemoattractant protein‐1&quot;. J Leukoc Biol, 62: 577-580. doi:10.1002/jlb.62.5.57718. <A href="https://en.wikipedia.org/wiki/CCL2">https://en.wikipedia.org/wiki/CCL2</A><I>Simple reference on MCP-1.</I>19. Christian K.W. Schwarz, et. al &quot;Using an E. coli Type 1 secretion system to secrete the mammalian, intracellular protein IFABP in its active form&quot;. J Biotechnol. 2012 Jun 15;159(3):155-61. doi: 10.1016/j.jbiotec.2012.02.005<I>Helped us formulate our idea of using MCP-1 for PAIR.</I>20. Petersen TN, Brunak S, et. al &quot;SignalP 4.0: discriminating signal peptides from transmembrane regions&quot;. Nat Methods. 2011 Sep 29;8(10):785-6. doi: 10.1038/nmeth.1701.21. Molino JVD, de Carvalho JCM, Mayfield SP.&quot;Comparison of secretory signal peptides for heterologous protein expression in microalgae: Expanding the secretion portfolio for Chlamydomonas reinhardtii&quot;.PLoS One. 2018 Feb 6;13(2):e0192433. doi: 10.1371/journal.pone.019243322. Han, SooJin et al. “Novel Signal Peptides Improve the Secretion of Recombinant Staphylococcus Aureus Alpha toxinH35L in Escherichia Coli.” AMB Express 7 (2017): 93. PMC. Web. 14 Oct. 2018.23. Schierle, Clark F. et al. “The DsbA Signal Sequence Directs Efficient, Cotranslational Export of Passenger Proteins to the Escherichia Coli Periplasm via the Signal Recognition Particle Pathway”. Journal of Bacteriology 185.19 (2003): 5706–5713. PMC. Web. 14 Oct. 2018.24. Valent QA, Scotti PA, High S, et al. &quot;The Escherichia coli SRP and SecB targeting pathways converge at the translocon&quot;. The EMBO Journal. 1998;17(9):2504-2512. doi:10.1093/emboj/17.9.2504.25. Beckwith J. &quot;The Sec-dependent pathway&quot;. Research in microbiology. 2013;164(6):497-504. doi:10.1016/j.resmic.2013.03.007.26. Lindsay Gray, Karen Baker, et. al &quot;A novel C-terminal signal sequence targets Escherichia coli haemolysin directly to the medium&quot;J Cell Sci 1989 1989: 45-57; doi: 10.1242/jcs.1989.27. Hanke, C., Hess, J.,&quot;Processing by OmpT of fusion proteins carrying the HlyA transport signal during secretion by theEscherichia coli hemolysin transport system&quot; et al. Molec. Gen. Genet. (1992) 233: 42. https://doi.org/10.1007/BF0058755930. Rosano GL, Ceccarelli EA. &quot;Recombinant protein expression in Escherichia coli: advances and challenges&quot;. Frontiers in Microbiology. 2014;5:172. doi:10.3389/fmicb.2014.00172.31. Sarthy PV, Meselson M. &quot;Single burst study of rec- and red-mediated recombination in bacteriophage lambda&quot;. Proceedings of the National Academy of Sciences of the United States of America. 1976;73(12):4613-4617.32. David C. Dale, Laurence Boxer, W. Conrad Liles &quot;The phagocytes: neutrophils and monocytes&quot;. Blood. 2008 112:935-945; doi: https://doi.org/10.1182/blood-2007-12-07791733. G Van Riper, S Siciliano,et. al &quot;Characterization and species distribution of high affinity GTP-coupled receptors for human rantes and monocyte chemoattractant protein 1&quot;.Journal of Experimental Medicine Mar 1993, 177 (3) 851-856; DOI: 10.1084/jem.177.3.851<I>Helped us with the details about different chemo-attractants and their effect on each other across various species.</I>34. Christoph A. Reichel, Daniel Puhr-Westerheide, et. al &quot;C-C motif chemokine CCL3 and canonical neutrophil attractants promote neutrophil extravasation through common and distinct mechanisms&quot;. Blood Jul 2012, 120 (4) 880-890; DOI: 10.1182/blood-2012-01-402164.<I>Gave us details about neutrophil attractants and was useful for PAIR.</I>35. <A href="https://www.ncbi.nlm.nih.gov/gene/2919#gene-expression">https://www.ncbi.nlm.nih.gov/gene/2919#gene-expression</A> Neutrophils.36. Christine Rouault, Vanessa Pellegrinelli, et. al &quot;Roles of Chemokine Ligand-2 (CXCL2) and Neutrophils in Influencing Endothelial Cell Function and Inflammation of Human Adipose Tissue&quot;, Endocrinology, Volume 154, Issue 3, 1 March 2013, Pages 1069–1079, https://doi.org/10.1210/en.2012-1415 <I>Details on effects of neutrophil attractants.</I>37. Silva MT. &quot;Macrophage phagocytosis of neutrophils at inflammatory/infectious foci: a cooperative mechanism in the control of infection and infectious inflammation&quot;.J Leukoc Biol. 2011 May;89(5):675-83. doi: 10.1189/jlb.0910536<I>Details about cooperation between macrophages and neutrophils.</I>38. Silva MT, Correia-Neves M. &quot;Neutrophils and Macrophages: the Main Partners of Phagocyte Cell Systems. Frontiers in Immunology&quot;. 2012;3:174. doi:10.3389/fimmu.2012.00174.39. Zhang X, Goncalves R, Mosser DM. The Isolation and Characterization of Murine Macrophages. Current protocols in immunology / edited by John E Coligan . [et al]. 2008;CHAPTER:Unit-14.1. doi:10.1002/0471142735.im1401s83.40. Lenski, R. E. (1988), &quot;Experimental studies of Pleiotropy and Epistasis in Escherichia coli. II. Compensation for maladaptive effects associated with resistance to virus T$&quot;. Evolution, 42: 433-440. doi:10.1111/j.1558-5646.1988.tb04150.x
    Exploring T4 resistance in <I>E. coli</I>, helping understand the dynamics behind our idea for APES.41. William T. Speck, Herbert S. Rosenkranz,&quot;Proflavin: an unusual mutagen&quot;. Mutation Research/Genetic Toxicology, Volume 77, Issue 1,1980,Pages 37-43,ISSN 0165-1218,https://doi.org/10.1016/0165-1218(80)90118-4.42. Konrad, M. (1960), &quot;The mutagenic effect of D2O on bacteriophage T4&quot;. Annals of the New York Academy of Sciences, 84: 678-684. doi:10.1111/j.1749-6632.1960.tb39100.x43. Abedon ST. &quot;Phage therapy dosing: The problem(s) with multiplicity of infection (MOI)&quot;. Bacteriophage. 2016;6(3):e1220348. doi:10.1080/21597081.2016.1220348.44. Anderson CW, Eigner J. &quot;Breakdown and Exclusion of Superinfecting T-Even Bacteriophage in Escherichia coli&quot;. Journal of Virology. 1971;8(6):869-886.<I>Superinfection issues in T4 bacteiophage.</I>45. Bode W. &quot;Lysis Inhibition in Escherichia coli Infected with Bacteriophage T4&quot;. Journal of Virology. 1967;1(5):948-955.46. <A href="https://www.ncbi.nlm.nih.gov/nuccore/AF158101">https://www.ncbi.nlm.nih.gov/nuccore/AF158101</A><I>This gave us the complete genome of bacteriophage T4. Helped us get the genes, make primers and order the DNA required for our work.</I>47. <A href="http://vlab.amrita.edu/?sub=3&amp;brch=76&amp;sim=719&amp;cnt=1">http://vlab.amrita.edu/?sub=3&amp;brch=76&amp;sim=719&amp;cnt=1</A><I>This was referred for understanding the mechanism of plaque assay.</I>48. Yao, Yao, and Stella E. Tsirka. &quot;Mouse monocyte chemoattractant protein 1 (MCP1) functions as a monomer.&quot; The international journal of biochemistry &amp; cell biology 55 (2014): 51-59.49. Leiman, Petr G., et al. &quot;Three-dimensional rearrangement of proteins in the tail of bacteriophage T4 on infection of its host.&quot; Cell 118.4 (2004): 419-429.50. Yoichi, Masatoshi, et al. &quot;Alteration of tail fiber protein gp38 enables T2 phage to infect Escherichia coli O157: H7.&quot; Journal of biotechnology 115.1 (2005): 101-107.51. Rakhuba, D. V., et al. &quot;Bacteriophage receptors, mechanisms of phage adsorption and penetration into host cell.&quot; Pol. J. Microbiol 59.3 (2010): 145-15552. Bartual, Sergio G., et al. &quot;Structure of the bacteriophage T4 long tail fiber receptor-binding tip.&quot; Proceedings of the National Academy of Sciences 107.47 (2010): 20287-20292.53. Wang, I N et al. “Holins: the protein clocks of bacteriophage infections.” Annual review of microbiology 54 (2000): 799-825.54. Lim, Jeong-A YangᆞSoon-Ryun et al. “Exogenous lytic activity of SPN9CC endolysin against gram-negative bacteria.” Journal of microbiology and biotechnology 24 6 (2014): 803-11.55. Jaeeun Park, Jiae Yun, et al. &quot;Characterization of an endolysin, LysBPS13, from a Bacillus cereus bacteriophage&quot;, FEMS Microbiology Letters, Volume 332, Issue 1, 1 July 2012, Pages 76–83, https://doi.org/10.1111/j.1574-6968.2012.02578.x56. Lim, Jeong-A, et al. &quot;Characterization of Endolysin From a Salmonella Typhimurium-infecting Bacteriophage Spn1s.&quot; Research in microbiology 163.3 (2012): 233-241. doi: 10.1016/j.resmic.2012.01.00257.  Zhang, Fang, et al. &quot;Secretion and circular dichroism analysis of the C-terminal signal peptides of HlyA and LktA.&quot; Biochemistry 34.13 (1995): 4193-4201.58.  Singh, Pranveer, et al. &quot;Effect of signal peptide on stability and folding of Escherichia coli thioredoxin.&quot; PloS one 8.5 (2013): e63442.59. Sletta, H., et al. &quot;The presence of N-terminal secretion signal sequences leads to strong stimulation of the total expression levels of three tested medically important proteins during high-cell-density cultivations of Escherichia coli.&quot; Applied and environmental microbiology 73.3 (2007): 906-912.
60. Mosberg, J. A., Lajoie, M. J., &amp; Church, G. M. (2010). Lambda Red Recombineering in Escherichia coli Occurs Through a Fully Single-Stranded Intermediate. Genetics, 186(3), 791–799. http://doi.org/10.1534/genetics.110.12078261. Sharan S K, Thomason L C, Kuznetsov S G, et al. Recombineering: a homologous recombination-based method of genetic engineering[J]. Nature protocols, 2009, 4(2): 206-223.62. Datsenko K A, Wanner B L. One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products[J]. Proceedings of the National Academy of Sciences, 2000, 97(12): 6640-6645.63.Cairns, Benjamin J., et al. &quot;Quantitative models of in vitro bacteriophage–host dynamics and their application to phage therapy.&quot; PLoS Pathogens 5.1 (2009): e1000253.64. Labrie, Simon J., Julie E. Samson, and Sylvain Moineau. &quot;Bacteriophage resistance mechanisms.&quot; Nature Reviews Microbiology 8.5 (2010): 317.65. <A href="https://www.uniprot.org/uniprot/P03697">https://www.uniprot.org/uniprot/P03697</A>66. <A href="https://www.uniprot.org/uniprot/P03698">https://www.uniprot.org/uniprot/P03698</A>67. <A href="https://www.uniprot.org/uniprot/P03702">https://www.uniprot.org/uniprot/P03702</A>66. <A href="https://www.thebureauinvestigates.com/stories/2018-01-30/a-game-of-chicken-how-indian-poultry-farming-is-creating-global-superbugs">https://www.thebureauinvestigates.com/stories/2018-01-30/a-game-of-chicken-how-indian-poultry-farming-is-creating-global-superbugs</A>67. World Health Organization. &quot;Critically important antimicrobials for human medicine: ranking of antimicrobial agents for risk management of antimicrobial resistance due to non-human use.&quot; (2017)68. A Ventola CL. The Antibiotic Resistance Crisis: Part 1: Causes and Threats. Pharmacy and Therapeutics. 2015;40(4):277-28369. Gould IM, Bal AM. New antibiotic agents in the pipeline and how they can overcome microbial resistance. Virulence. 2013;4(2):185–191. 70. Bartlett JG, Gilbert DN, Spellberg B. Seven ways to preserve the miracle of antibiotics. Clin Infect Dis. 2013;56(10):1445–1450 71. The antibiotic alarm. Nature. 2013;495(7440):14172. Bacteriophage resistance mechanisms. Nature Reviews Microbiology volume 8, pages 317–327 (2010)73. Grayson ML, Heymann D, Pittet D. The evolving threat of antimicrobial resistance. In: Martinez L, editor. The evolving threat of antimicrobial resistance: options for action. World Health Organization; Geneva: 2012. p. 1-10 74. Barlow M. What antimicrobial resistance has taught us about horizontal gene transfer. Methods Mol Biol 2009;532:397-41175. Non-genetic mechanisms communicating antibiotic resistance: rethinking strategies for antimicrobial drug design. Expert Opin. Drug Discov. (2012) 7(10):923-93376. Rossmann, Michael G., et al. &quot;The bacteriophage T4 DNA injection machine.&quot; Current opinion in structural biology 14.2 (2004): 171-180. 77. Leiman, Petr G., et al. &quot;Three-dimensional rearrangement of proteins in the tail of bacteriophage T4 on infection of its host.&quot; Cell 118.4 (2004): 419-429. 78. Moran G. Goren, Ido Yosef, Udi Qimron, Programming Bacteriophages by Swapping Their Specificity Determinants, Trends in Microbiology, Volume 23, Issue 12, 2015, Pages 744-746.79. Bartual, Sergio G., et al. &quot;Structure of the bacteriophage T4 long tail fiber receptor-binding tip.&quot; Proceedings of the National Academy of Sciences 107.47 (2010): 20287-2029280. Hinchliffe, Philip; Yang, Qiu E.; Portal, Edward; Young, Tom; Li, Hui; Tooke, Catherine L.; et al. (2017). &quot;Insights into the Mechanistic Basis of Plasmid-Mediated Colistin Resistance from Crystal Structures of the Catalytic Domain of MCR-1&quot;. Scientific Reports. 7: 39392.81. Yi-Yun Liu, Yang Wang, Timothy R Walsh et al. Emergence of plasmid-mediated colistin resistance mechanism MCR-1 in animals and human beings in China: a microbiological and molecular biological study, The Lancet Infectious Diseases, Volume 16, Issue 2, 2016, Pages 161-16882. Opal, S. M. (2010). Endotoxins and Other Sepsis Triggers. Endotoxemia and Endotoxin Shock, 14–24. doi:10.1159/000315915 83. Antibiotic-induced endotoxin release and clinical sepsis: a review 84. Dwayne R. Roach, Laurent Debarbieux. Phage therapy: awakening a sleeping giant. doi: 10.1042/ETLS20170002 85. Lysis-deficient bacteriophage therapy decreases endotoxin and inflammatory mediator release and improves survival in a murine peritonitis model; Matsuda, Takeaki et al.; Surgery , Volume 137 , Issue 6 , 639 - 646 86. Whiteside, T. L. (n.d.). Immune Cells in the Tumor Microenvironment. Cancer Immune Therapy, 95–118. doi:10.1002/3527600795.ch6 87. Mergulhão, F. J. M., Summers, D. K., &amp; Monteiro, G. A. (2005). Recombinant protein secretion in Escherichia coli. Biotechnology Advances, 23(3), 177–202. doi:10.1016/j.biotechadv.2004.11.003 88. Choi, J. H., &amp; Lee, S. Y. (2004). Secretory and extracellular production of recombinant proteins using Escherichia coli. Applied Microbiology and Biotechnology, 64(5), 625–635. doi:10.1007/s00253-004-1559-9 89. Labrie, Simon J., Julie E. Samson, and Sylvain Moineau. &quot;Bacteriophage resistance mechanisms.&quot; Nature Reviews Microbiology 8.5 (2010): 317.&quot;90. Yarosh, Daniel Bruce. &quot;UV-induced mutation in bacteriophage T4.&quot; Journal of virology 26.2 (1978): 265-27191. Beceiro, Alejandro, et al. &quot;Phosphoethanolamine modification of lipid A in colistin-resistant variants of Acinetobacter baumannii mediated by the pmrAB two-component regulatory system.&quot; Antimicrobial agents and chemotherapy (2011): AAC-00079.92. Bartual, Sergio G., et al. &quot;Structure of the bacteriophage T4 long tail fiber receptor-binding tip.&quot; Proceedings of the National Academy of Sciences 107.47 (2010): 20287-20292.93. Bartual, Sergio Galan, et al. &quot;Two-chaperone assisted soluble expression and purification of the bacteriophage T4 long tail fibre protein gp37.&quot; Protein expression and purification 70.1 (2010): 116-12194. Brzozowska, Ewa, et al. &quot;Recognition of bacterial lipopolysaccharide using bacteriophage-adhesin-coated long-period gratings.&quot; Biosensors and Bioelectronics 67 (2015): 93-9995. Paul, Vivek Daniel, et al. &quot;Lysis-deficient phages as novel therapeutic agents for controlling bacterial infection.&quot; BMC microbiology 11.1 (2011): 19596. Yu, Daiguan, et al. &quot;An efficient recombination system for chromosome engineering in Escherichia coli.&quot; Proceedings of the National Academy of Sciences 97.11 (2000): 5978-5983.97. Payne, Robert JH, and Vincent AA Jansen. &quot;Understanding bacteriophage therapy as a density-dependent kinetic process.&quot; Journal of Theoretical Biology 208.1 (2001): 37-4898. Jason, A. C. &quot;A deterministic model for monophasic growth of batch cultures of bacteria.&quot; Antonie van Leeuwenhoek 49.6 (1983): 513-536.99. Levin, Bruce R., Frank M. Stewart, and Lin Chao. &quot;Resource-limited growth, competition, and predation: a model and experimental studies with bacteria and bacteriophage.&quot; The American Naturalist 111.977 (1977): 3-24.100. Levenberg, Kenneth. &quot;A method for the solution of certain non-linear problems in least squares.&quot; Quarterly of applied mathematics 2.2 (1944): 164-168.101. Marquardt, Donald W. &quot;An algorithm for least-squares estimation of nonlinear parameters.&quot; Journal of the society for Industrial and Applied Mathematics 11.2 (1963): 431-441.102. <A href="https://salislab.net/software/">https://salislab.net/software/:Salis Lab - The Ribosome Binding Site Calculator</A>103. Le Guilloux V, Schmidtke P and Tuffery P, Fpocket: An open source platform for ligand pocket detection , BMC Bioinformatics, 2009, 10:168 104. Morris, Garrett M., et al. &quot;AutoDock4 and AutoDockTools4: Automated docking with selective receptor flexibility.&quot; Journal of computational chemistry 30.16 (2009): 2785-2791105. Nagarajan, Deepesh, et al. &quot;Design of a heme-binding peptide motif adopting a β-hairpin conformation.&quot; Journal of Biological Chemistry (2018): jbc-RA118106. Bhattacharya, Debswapna, and Jianlin Cheng. &quot;i3Drefine software for protein 3D structure refinement and its assessment in CASP10.&quot; PloS one 8.7 (2013): e69648107. Joosten, Robbie P., et al. &quot;A series of PDB related databases for everyday needs.&quot; Nucleic acids research 39.suppl_1 (2010): D411-D419. 108. Kabsch, Wolfgang, and Christian Sander. &quot;Dictionary of protein secondary structure: pattern recognition of hydrogen‐bonded and geometrical features.&quot; Biopolymers: Original Research on Biomolecules 22.12 (1983): 2577-2637.109. Park, Moon-Won, and Yeong-Dae Kim. &quot;A systematic procedure for setting parameters in simulated annealing algorithms.&quot; Computers &amp; Operations Research 25.3 (1998): 207-217.110. Davies, M. &quot;A game of chicken: how Indian poultry farming is creating global superbugs&quot;, The Bureau of Investigative Journalism, 30 January 2018. 111. Patel, Isha, et al. &quot;Antimicrobial resistance in India.&quot; Journal of pharmaceutical policy and practice 10.1 (2017): 27. 112. Bhutia, L. &quot;How India Became The Antibiotics Capital Of The World And Wasted The Wonder Cure&quot;, Huffington Post, 31 October 2015113. Holzheimer, R. G. &quot;Antibiotic induced endotoxin release and clinical sepsis: a review.&quot; Journal of Chemotherapy 13.sup4 (2001): 159-172114. <A href="http://www.readabilityformulas.com/free-dale-chall-test.php ">http://www.readabilityformulas.com/free-dale-chall-test.php </A>115. <A href="http://www.readabilityformulas.com/free-readability-formula-tests.php">http://www.readabilityformulas.com/free-readability-formula-tests.php</A>116. <A href="https://www.independent.co.uk/news/science/antibiotic-resistance-superbugs-united-nations-penicillin-doses-limit-a7197841.html">https://www.independent.co.uk/news/science/antibiotic-resistance-superbugs-united-nations-penicillin-doses-limit-a7197841.html</A>117. Koskella, Brockhurst. &quot;Bacteria–phage coevolution as a driver of ecological and evolutionary processes in microbial communities.&quot; US National Insitute of Medicine (2014).118. Dedrick, Rebekah M., et al. &quot;Functional requirements for bacteriophage growth: gene essentiality and expression in mycobacteriophage G iles.&quot; Molecular microbiology 88.3 (2013): 577-589.119. Bartual, Sergio G., et al. &quot;Structure of the bacteriophage T4 long tail fiber receptor-binding tip.&quot; Proceedings of the National Academy of Sciences 107.47 (2010): 20287-20292.
</P></DIV><FOOTER id="footer"><DIV class="footer-text-boxes"><DIV class="footer-text-boxes-section"><DIV class="footer-text-boxes-section-heading">Links</DIV><UL><LI><A href="http://www.iisc.ac.in" target="_blank">IISc Bangalore</A></LI><LI><A href="https://iisc.ac.in/ug/" target="_blank">IISc UG Program </A></LI><LI><A href="https://2018.igem.org/Team:IISc-Bangalore/References">Complete list of references</A></LI></UL></DIV><A href="https://goo.gl/maps/BPKKwKU3hd32" target="_blank"><DIV class="footer-text-boxes-section"><DIV class="footer-text-boxes-section-heading">Address</DIV><UL><LI>Undergraduate department,</LI><LI>Indian Institute of Science,</LI><LI>Bangalore - 560012</LI></UL></DIV></A><DIV class="footer-text-boxes-section"><DIV><DIV class="footer-text-boxes-section-heading">Contact</DIV><UL><LI>Bhaskar Kumawat</LI><LI>Team Lead</LI><LI><A href="mailto:kbhaskar@iisc.ac.in" target="_blank">kbhaskar@iisc.ac.in</A></LI></UL></DIV></DIV></DIV><DIV class="footer-text-small" style="max-width:700px;">This website is best viewed on a 16:9 aspect ratio screen on the latest version of Google Chrome</DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>