Team:IISc-Bangalore/Results
Home
Project
Overview PACMAN PAIR APES Design Experiments Parts Model
Lab
Notebook Protocols Assembly Results InterLab Improve
Software
PhageModifier
Community
Collaborations Human Practices Integrated HP Public Engagement Safety Survey
Team
Members Philosophy Attributions
Results
Improvement of mCherry BioBrick
Assembly of BioBrick Parts
PAIR
What Worked
What Didn't Work
PACMAN
What Worked
What Didn't Work
APES
The following are the results of all the experiments we performed over the  course of the summer.
Improvement of mCherry BioBrick
Aiming to eliminate the shortcomings of the existing BBa_J18932 (mCherry RFP), we replaced the internal RBS like sequence, with a generated weaker sequence (with conserved amino acid sequence). As a result, we obtained a BioBrick BBa_K2609006 that generates mCherry with reduced truncation. The magnitude of truncation is given below:
mCherry BBa_J18932: 38.8% ± 2.5% Truncation
imCherry BBa_K2609006: 20.4% ± 2.5% Truncation
Thus, we have produced a BioBrick which has 47.4% decrease in truncation from the original.
Complete details can be found on the Improve page.
Assembly of BioBrick Parts
The following parts were assembled, verified by sequencing, and characterised by expression and visualisation on SDS PAGE. These were also submitted to the iGEM Registry to aid teams and labs in their endeavors.
BBa_K2609007 (mcp-1 under T7 expression system)
Generates the Mcp-1 chemokine when transformed into a T7 expression strain like E coli. BL21(DE3).
BBa_K2609006 (improved mCherry construct)
Generator for improved version of mCherry RFP construct, modified to prevent truncation of protein. Produced mCherry when expressed in a T7 expression strain like E coli. BL21(DE3).
BBa_K2609017 (T4 Endolysin under constitutive promoter)
Generates T4 Endolysin (a lysozyme) constitutively.
BBa_K2609026 (Lambda RED recombinases under T7 expression system)
Generates the Lambda RED recombination proteins - exo, gam and beta - when transformed into a T7 expression strain like E coli. BL21(DE3).
BBa_K2609013 (gp37(modification 2) under T7 expression system)
Generates the tip of the T4 long tail fiber protein when transformed into a T7 expression strain like E coli. BL21(DE3).
The following parts were assembled, verified by sequencing and submitted to the iGEM Registry for the use of other teams and labs.
BBa_K2609008 (mcp-1 + HlyA in T4 recombination cassette) and
BBa_K2609009 (mcp-1 + PelB in T4 recombination cassette)
Mcp-1 attached to a C terminus HlyA signal peptide with a OmpT linker and N terminus PelB signal peptide respectively, inside a T4 Recombination Cassette. Intended for recombination and replacement of endolysin.
BBa_K2609010 (mcp-1 + HlyA under T7 expression system) and
BBa_K2609011 (mcp-1 + PelB under T7 expression system)
Generates mcp-1 fused with HlyA and PelB respectively, when transformed into a T7 expression strain like E coli. BL21(DE3). Intended for testing the functioning of the signal peptide HlyA.
The method used to assemble and their verification with all gel images can be found on the Assembly page.
PAIR
What Worked
Assembly of DNA Parts
All the BioBrick parts required for PAIR were assembled and verified to be correct by sequencing. The BioBricks that were assembled are: mcp-1 under T7 expression system (BBa_K2609007), Lambda RED recombinases under T7 expression system (BBa_K2609026), and mcp-1 with signal peptides HlyA/PelB under T7 expression system (BBa_K2609010 & BBa_K2609011).
Expression and purification of mcp-1
Mcp-1 was produced in E coli. BL21(DE3) by using the part mcp-1 under T7. Protein was produced by induction at OD600=0.6 with 250μM IPTG and grown for 16 hours at 25°C. The protein was forming inclusion bodies and hence was solubilized in 8M urea. This was directly applied to an Ni-NTA column and purified. The 6xHis tag purified protein was then dialized to get pure mcp-1 in PBS. Performing a Bradford Assay on the dialized protein gave a concentration of 49.8 μg/mL.
Characterisation of Lambda RED BioBrick
We characterised the Lambda RED BioBrick BBa_K1433005 using the generator BBa_K2609026 by expression and visualising on SDS PAGE. This part is under T7 expression system and hence was transformed into E coli BL21(DE3) and induced with 500 μM IPTG at OD600 = 0.6 and grown for 3 hours at 37°C. Cell pellet was boiled with SDS sample buffer and loaded ran on SDS PAGE.
Bands were observed slightly below 32kDa ladder band and at the 25kDa ladder band. These two correspond to the bet protein (29.7 kDa) and the exo protein (25.9 kDa). The other protein - gam (16.3 kDa) - could not be seen on the gel. This could be because the part has only one RBS sequence for all three proteins. It could be either that the third protein is being produced in very small amounts or not at all.
Characterisation of T4 Endolysin BioBrick
We characterised the BioBrick BBa_K2609001 using the generator BBa_K2609017 by the size of the protein produced and by its function. Since, this part is under a constitutive promoter, it cannot be induced. Hence, bands of E coli BL21(DE3) containing this part were compared with the bands of E coli BL21(DE3). The SDS PAGE is shown below.
We also characterised the lystic activity of recombinantly produced T4 Endolysin. For this, we extracted protein from E coli BL21(DE3) containing the endolsyin plasmid along with wild type. B Cereus, a gram positive bacteria, was grown to mid log phase and then suspended in 0.2mM Tris-HCl (pH 8). 0.1 volumes of the following were added:
Protein extract from E coli BL21(DE3) containing T4 Endolysin
Protein extract from wild type E coli BL21(DE3)
Protein lysis buffer
Control blank buffer
OD600 was measured using a plate reader continuously over a span of two hours. We obtained the graph shown above after taking average from multiple replicates. From this, we see that endolysin reduces B Cereus cell density faster than the other solutions. This is confirmation that recombinantly produced T4 Endolysin is functional and acts as a lysozyme.
Extraction of Phage DNA
Using an enzymatic protocol with Proteinase K and DNAse, we obtained T4 bacteriophage genomic DNA. The DNA could be seen precipitating after on addition of isopropanol. Following spectrophotometric analysis of the DNA, it was found to be very high concentration (~2 μg/μL) and high purity (A260/A280 = 1.8).
What Didn't Work
Electroporation of T4 phage DNA
We were unable to electroporate the extremely long ~169kbp genome of T4 bacteriophage. We tried electroporation of the extracted DNA into different strains of E coli in multiple trials with no success. We also consulted the highly experienced in bacteriophage research, Prof. Ryland Young and Prof. Graham Hatfull, for their inputs. Even so, we were not able to troubleshoot the problem.
Recombination with the Lambda RED System
We were unable to recombine with using the Lambda RED BioBrick BBa_K2609026 made from BBa_K1433005. We tried a test recombination of plasmids in multiple trials but were unable to generate any recombinants.
Generation of T4 phage recombinants
The success of this part depended on the success of the previous two parts. Since, neither of them gave any positive result, we were unable to proceed to this step.
PACMAN
What Worked
Expression and Purification of Wild Type gp37
In order to isolate gp37 for further studies, it was essential that it was properly folded. This protein needs two chaperones for correct folding i.e. gp38 and gp57.
Thus, we did a triple transformation of all the three gene-containing plasmids, expressed by induction with 500μM IPTG at OD600=0.6 and grown at 25°C for 10 hours.
Folding and Ligand Interaction of Wild Type gp37
A CD (Circular Dichroism) measurement was performed, and this CD spectrum showed a dip at the wavelength corresponding to exclusively the β pleated structure, hence giving us substantial evidence of its correct folding.
We measured CD Spectra of WT gp37 in the presence of different concentrations of the ligand PEtN. We plotted the minimum angle against ligand concentration and obtained the below graph. From the we can conclude that there is no binding between WT gp37 and PEtN as there is no observable trend. This can be explained by the fact that the thermal energy of the protein and the ligand are greater than the interaction energy between them.
Assembly and expression of gp37 M2 under T7 expression system
The BioBrick part BBa_K2609013 was assembled, and verified to be correct through DNA sequencing. It was then transformed into E coli BL21(DE3) and induced with 500μM IPTG at OD600 = 0.6 and grown for 3 hours at 37°C. The cell pellet was then loaded on SDS PAGE.
What Didn't Work
Assembly of gp37 M1,M3 and M4
The other three modifications of gp37 could not be assembled even after multiple trials.
Purification of Correctly Folded gp37 M2
Since gp37 needs its chaperones gp38 & gp57 to fold correctly, the three proteins had to be triple transformed and then expressed. Following multiple trials of triple transformation, we did not obtain a transformant that expressed all three proteins. Thus, we could not obtain correctly folded gp37 M2 to proceed further.
APES
The APES experiments failed to produce reproducibly, across trials, a sufficient number of mutant phages capable of infecting and lysing T4-resistant E. coli.
Plate showing sample 2 of resistant bacteria plated with T4 after irradiation with UV for 5 minutes. No plaques were observed.
Plate showing wild-type strain MG1655 plated with T4 after irradiation with UV for 5 minutes. Tiny plaques can be observed.
Links
IISc Bangalore
IISc UG Program
Complete list of references
Address
Undergraduate department,
Indian Institute of Science,
Bangalore - 560012
Contact
Bhaskar Kumawat
Team Lead
kbhaskar@iisc.ac.in
This website is best viewed on a 16:9 aspect ratio screen on the latest version of Google Chrome
