<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NCTU_Formosa_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NCTU Formosa/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>navigation</TITLE></P><DIV id="Navigation"><DIV id="nav"><UL><LI class="active has-sub"><A href="#"><SPAN>Notebook</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Protocol"><SPAN>Protocol</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Notebook"><SPAN>Lab Notes</SPAN></A></LI></UL><LI class="active has-sub"><A href="#"><SPAN>Team</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Team"><SPAN>Team Introduction</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Attributions"><SPAN>Attribution</SPAN></A></LI><LI class="active has-sub"><A href="#"><SPAN>Achievement</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Medal_Criteria"><SPAN>Medal Criteria</SPAN></A></LI><LI class="active has-sub"><A href="#"><SPAN>Human Practice</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Collaborations"><SPAN>Collaboration</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Silver"><SPAN>HP Silver</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Gold_Integrated"><SPAN>HP Gold Integrated</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Engagement"><SPAN>Education and Public Engagement</SPAN></A></LI><LI class="active has-sub"><A href="#"><SPAN>Parts</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Parts"><SPAN>Parts</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Basic_Part"><SPAN>Basic Parts</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Composite_Part"><SPAN>Composite Parts</SPAN></A></LI><LI class="active has-sub"><A href="#"><SPAN>Wet Lab</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Experiments"><SPAN>Experiment Design</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Fungal_Experiment"><SPAN>Fungal Experiment</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Fungal_Result"><SPAN>Fungal Result</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Protein_Expression"><SPAN>Protein Expression</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Safety"><SPAN>Safety</SPAN></A></LI><LI class="active has-sub"><A href="#"><SPAN>Modeling</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><SPAN>Peptide Prediction Model</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Disease_Occurrence_Model"><SPAN>Disease Occurrence Model</SPAN></A></LI><LI class="active has-sub"><A href="#"><SPAN>Project</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Description"><SPAN>Description</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Applied_Design"><SPAN>Design</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Peptide_Prediction"><SPAN>Peptide Prediction</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Disease_Occurrence_Prediction"><SPAN>Disease Occurrence Prediction</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Demonstrate"><SPAN>Demonstration</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Contribution"><SPAN>Contribution</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Improve"><SPAN>Improvement</SPAN></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa" id="Home"><SPAN>Home</SPAN></A></LI></DIV></DIV><DIV id="Navigation2"><DIV id="three_bar"><P>â˜°</P></DIV><DIV class="igem_2017_menu2_wrapper"><DIV class="menu2_button" id="display_menu2_control">
                MENU
            </DIV><DIV id="menu2_content"><A href="https://2017.igem.org/Team:NCTU_Formosa"><DIV class="menu2_button direct_to_page">
                        HOME
                    </DIV></A><DIV class="menu2_button"> Project
                </DIV><DIV class="submenu2_wrapper" id="Project_submenu"><A href="https://2017.igem.org/Team:NCTU_Formosa/Description"><DIV class="submenu2_button" id="Description_page">
                            Description
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Applied_Design"><DIV class="submenu2_button" id="Design_page">
                            Design
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Peptide_Prediction"><DIV class="submenu2_button" id="Peptide_Prediction_page">
                            Peptide Prediction
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Disease_Occurrence_Prediction"><DIV class="submenu2_button" id="Spore_Germinaition_Prediction_page">
                            Disease Occurrence Prediction
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Demonstrate"><DIV class="submenu2_button" id="Collaborations_page">
                            Demonstration
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Contribution"><DIV class="submenu2_button" id="Collaborations_page">
                            Contribution
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Improve"><DIV class="submenu2_button" id="Improvement_page">
                            Improvement
                        </DIV></A></DIV><DIV class="menu2_button"> Modeling
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu2_button" id="SCM_Model_page">
                            Peptide Prediction Model
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Disease_Occurrence_Model"><DIV class="submenu2_button" id="Spore_Germination_Model_page">
                            Disease Occurrence Model
                        </DIV></A></DIV><DIV class="menu2_button"> Wet Lab
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Experiments"><DIV class="submenu2_button" id="Fungal_Experiment_page">
                            Experiment Design
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Fungal_Experiment"><DIV class="submenu2_button" id="Fungal_Experiment_page">
                            Fungal Experiment
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Fungal_Result"><DIV class="submenu2_button" id="Plant_Experiment_page">
                            Fungal Result
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Protein_Expression"><DIV class="submenu2_button" id="Protein_Expression_page">
                            Protein Expression
                  </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Safety"><DIV class="submenu2_button" id="Safety_page">
                            Safety
                        </DIV></A></DIV><DIV class="menu2_button"> Parts
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Parts"><DIV class="submenu2_button" id="Basic_Part_page">
                            Parts
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Basic_Part"><DIV class="submenu2_button" id="Basic_Part_page">
                            Basic Parts
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Composite_Part"><DIV class="submenu2_button" id="Composite_Part_page">
                            Composite Parts
                        </DIV></A></DIV><DIV class="menu2_button"> Human Practice
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Collaborations"><DIV class="submenu2_button" id="Collaboration_page">
                            Collaboration
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Silver"><DIV class="submenu2_button" id="Silver_page">
                            HP Silver
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Gold_Integrated"><DIV class="submenu2_button" id="Gold_Integrated_page">
                            HP Gold Integrated
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Engagement"><DIV class="submenu2_button" id="Gold_Integrated_page">
                            Education and Public Engagement
                        </DIV></A></DIV><DIV class="menu2_button"> Achievement
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Medal_Criteria"><DIV class="submenu2_button" id="Medal_Criteria_page">
                            Medal Criteria
                        </DIV></A></DIV><DIV class="menu2_button"> Team
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Team"><DIV class="submenu2_button" id="Team_page">
                            Team Introduction
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Attributions"><DIV class="submenu2_button" id="Appreciation_page">
                            Attribution
                        </DIV></A></DIV><DIV class="menu2_button"> Notebook
                </DIV><DIV class="submenu2_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Protocol"><DIV class="submenu2_button" id="Protocol_page">
                            Protocol
                        </DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Notebook"><DIV class="submenu2_button" id="Lab_Notes_page">
                            Lab Notes
                        </DIV></A></DIV></DIV></DIV></DIV><DIV class="igem_2017_menu_wrapper"><DIV class="menu_button" id="display_menu_control">
		MENU
	</DIV><DIV id="menu_content"><A href="https://2017.igem.org/Team:NCTU_Formosa"><DIV class="menu_button direct_to_page">
				HOME
			</DIV></A><DIV class="menu_button"> TEAM
		</DIV><DIV class="submenu_wrapper" id="team_submenu"><A href="https://2017.igem.org/Team:NCTU_Formosa/Team"><DIV class="submenu_button" id="Team_page">
					Team
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Collaborations"><DIV class="submenu_button" id="Collaborations_page">
					Collaborations
				</DIV></A></DIV><DIV class="menu_button"> PROJECT
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Description"><DIV class="submenu_button" id="Description_page">
					Description
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Design"><DIV class="submenu_button" id="Design_page">
					Design
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Experiments"><DIV class="submenu_button" id="Experiments_page">
					Experiments
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Notebook"><DIV class="submenu_button" id="Notebook_page">
					Notebook
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/InterLab"><DIV class="submenu_button" id="InterLab_page">
					InterLab
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Contribution"><DIV class="submenu_button" id="Contribution_page">
					Contribution
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Results"><DIV class="submenu_button" id="Results_page">
					Results
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Demonstrate"><DIV class="submenu_button" id="Demonstrate_page">
					Demonstrate
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Improve"><DIV class="submenu_button" id="Improve_page">
					Improve
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Attributions"><DIV class="submenu_button" id="Attributions_page">
					Attributions
				</DIV></A></DIV><DIV class="menu_button"> PARTS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Parts"><DIV class="submenu_button" id="Parts_page">
					Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Basic_Part"><DIV class="submenu_button" id="Basic_Part_page">
					Basic Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Composite_Part"><DIV class="submenu_button" id="Composite_Part_page">
					Composite Parts
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Part_Collection"><DIV class="submenu_button" id="Part_Collection_page">
					Part Collection
				</DIV></A></DIV><A href="https://2017.igem.org/Team:NCTU_Formosa/Safety"><DIV class="menu_button direct_to_page">
				SAFETY
			</DIV></A><DIV class="menu_button"> HUMAN PRACTICES
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Silver"><DIV class="submenu_button" id="Silver_page">
					Silver HP
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Gold_Integrated"><DIV class="submenu_button" id="Gold_Integrated_page">
					Integrated and Gold
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Engagement"><DIV class="submenu_button" id="Engagement_page">
					Public Engagement
				</DIV></A></DIV><DIV class="menu_button"> AWARDS
		</DIV><DIV class="submenu_wrapper"><A href="https://2017.igem.org/Team:NCTU_Formosa/Applied_Design"><DIV class="submenu_button" id="Applied_Design_page">
					Applied Design
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Entrepreneurship"><DIV class="submenu_button" id="Entrepreneurship_page">
					Entrepreneurship
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Hardware"><DIV class="submenu_button" id="Hardware_page">
					Hardware
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Measurement"><DIV class="submenu_button" id="Measurement_page">
					Measurement
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Model"><DIV class="submenu_button" id="Model_page">
					Model
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Plant"><DIV class="submenu_button" id="Plant_page">
					Plant
				</DIV></A><A href="https://2017.igem.org/Team:NCTU_Formosa/Software"><DIV class="submenu_button" id="Software_page">
					Software
				</DIV></A></DIV><A href="https://igem.org/2017_Judging_Form?team=NCTU_Formosa"><DIV class="menu_button direct_to_page">
				JUDGING FORM
			</DIV></A></DIV></DIV><TITLE>NCTU_Formosa: Project Improvement</TITLE><DIV class="improve_content"><DIV id="NCTU2016"><DIV class="subtitle"><H6>Improvement - Finding more pest-resistant candidates for former NCTU_Formosa </H6><H5>- Using the same method SCM to build pest-resistant peptide prediction system</H5></DIV><DIV class="NCTU2016"><DIV class="show"><P>Â Â Â Â Â To improve the project of NCTU_Formosa 2016, we applied SCM to make an insecticidal peptide prediction system, using a quicker way to search for their target peptides and leaving them a group of potential target peptides.
                    </P><H2 style="position: relative; top: 10px;">Content:</H2><OL style="font-size: 1.3em"><LI>Datasets</LI><LI>Results and the candidates we suggested</LI></OL></DIV><DIV class="hide"><P>Â Â Â Â Â The way we use SCM to cure fungal diseases is just a part of its ability. In fact, the peptide prediction system based on the SCM can be specialized in different cases of evaluating sequences.
                    </P><P>Â Â Â Â Â We decided to apply the method to NCTU_Formosa 2016, which utilized spider toxin to kill the pests. We introduced the scoring card to the insecticidal protein to see whether we could also predict invertebrate proteins from ion channel impairing toxins, improving their searching tool while finding more candidates for the project last year.
                    </P><P>Â Â Â Â Â First, we collected the insecticidal and ion channel impairing toxins by <SPAN><A href="https://2016.igem.org/Team:NCTU_Formosa/Model#title1" target="_blank">2016 selection database</A></SPAN>. After
                        deleting peptides which contained non-standard amino acids, we randomly chose positive and negative data to our datasets and divided them into two datasets, training datasets and testing datasets.
                    </P><H3>Table 1: The dataset of the insecticidal protein </H3><P>Â Â Â Â Â For training parts, after initializing the first scorecard, we used IGA to optimize the scorecard for ten generations.
                    </P><H2>Results</H2><P>Â Â Â Â Â FullTrain_acc=91.70454568181819</P><P>Â Â Â Â Â CV acc(train)=93.8636343698348 </P><P>Â Â Â Â Â CV auc(train)=95.44599143143164 </P><P>Â Â Â Â Â Best theshold=498.75 </P><P>Â Â Â Â Â Best_acc(test)=88.86363681818182</P><P>Â Â Â Â Â Sensitivity(test)=0.7031249936523439 </P><P>Â Â Â Â Â Specitivity(test)=0.9202127637222726</P><H4>Figure 1: The score distribution of positive and negative(insecticidal and non-insecticidal) dataset</H4><H4>Figure 2: The ROC curve of the model</H4><H2>Discussion</H2><P>Â Â Â Â Â To improve the project of NCTU_Formosa 2016, we introduced the scoring card method to the insecticidal proteins. By using the method, we can predict more new insecticidal proteins.
                    </P><P>Â Â Â Â Â We collected about three thousands of ion channel impairing toxins.
                    </P><P>Â Â Â Â Â Below is the excerpt of the peptide list.

                    </P><H3>Table 2: Proteins that predicted insecticidal</H3></DIV></DIV></DIV><DIV id="ntnu2016"><DIV class="subtitle"><H6>NEW PART:</H6><H5>- fMt with a constitutive promoter</H5></DIV><DIV class="ntnu2016"><DIV class="show2"><H1>Introduction</H1><P>Â Â Â Â Â  fMt is a metallothionein that has high affinity with As(III). 2009 Groningen iGEM used it to purify the polluted water. However, 2009 Groningen iGEM ligased it with a low constitutive promotor, BBa_J23109. We failed serval times when and tried to ligased these two parts. After sequencing its plasmid, we found out there was two SpeI site in the plasmid. So we decided to replaced J23109 with J23119 and conducted a series of experiments to ensure the function.
                    </P></DIV><DIV class="hide2"><H1>Modifying and Improving the Existing Biobrick</H1><H2>
                        1. Previous biobrick: <SPAN><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K190031">BBa_K190031</A></SPAN> of 2009 iGEM Groningen
Â Â Â Â The new part: <SPAN><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2262015">BBa_K2262015</A></SPAN></H2><P style="margin-bottom: 50px;">Â Â Â Â Â The metallothionein (BBa_K190031) is a fMt(BBa_K190019) under control of a low constitutive promotor (BBa_J23109). We failed several times in replicating the ligation of these two parts. After sequencing these three plasmids,
                        we found BBa_J23109 has two Spel restriction sites in the prefix.(Figure 3.) Figure 4 shows the electrophoresis of BBa_K190019 when its plasmid was cut by XbaI and PstI and Figure 5 shows the electrophoresis of BBa_J23119
                        and BBa_J23109 when their plasmids were cut by SpeI and PstI. Thus, we decided to modify the biobrick by ligating fMt(BBa_K190019) with another constitutive promoter, BBa_J23119.(Figure 6.)
                    </P><H4>Figure 3: The sequence of J23109 Plasmid.</H4><H4>
                        Figure 4: The electrophoresis of BBa_K190019.
                    </H4><H4>
                        Figure 5: The electrophoresis of BBa_J23119 and BBa_J23109.
                    </H4><H4>
                        Figure 6: The new biobrick design.
                    </H4><H1>Results</H1><P>Â Â Â Â Â  We first examined the growth curve of <I>E. coli</I> DH5Î± in arsenic solution and compared the growth curve of <I>E. coli</I> DH5Î± in arsenic solution with that curve in solution without arsenic ions.
                        Table 3 shows the experimental design for the growth curve of <I>E. coli</I> DH5Î± and the result shows in Figure 7.
                    </P><H3>
                        Table 3: The experiment design for the growth curve of <I>E. coli</I> DH5Î±.
                    </H3><H4>Figure 7: The growth of <I>E.coli</I> DH5Î± in different conditions.</H4><P>Â Â Â Â Â  We find that <I>E. coli</I> DH5Î± wonâ€™t be affected by arsenic concentration below 100ppm.
                    </P><P>Â Â Â Â Â  Then we conducted the next experiment. We examined the growth curve of <I>E. coli</I> DH5Î± in arsenic solution and compared the growth curve of <I>E. coli</I> DH5Î± in arsenic solution with that curve
                        in solution with no arsenic ions. Table 4 shows the experimental design for the growth curve of <I>E. coli</I> DH5Î± and the result shows in Figure 8 .
                    </P><H3>
                        Table 4: The experiment design for the growth curve of <I>E. coli</I> DH5Î± with fMt plasmid.
                    </H3><H4>Figure 8: The growth of <I>E. coli</I> DH5Î± in different conditions.</H4><P>Â Â Â Â Â  The results of this experiment indicate that <I>E. coli</I> DH5Î± containing the transformed plasmid can survive in arsenic concentrations from 1 ppm to 100 ppm.
                    </P><P>Â Â Â Â Â  In conclusion, we modified the part of BBa_K190031 by replacing the promoter BBa_J23109 by BBa_J23119. The growth of <I>E. coli</I> with this new plasmid us not affected the arsenic concentration.
                    </P></DIV></DIV></DIV><DIV id="ntnu2016-2"><DIV class="subtitle"><H6>Increase Function:</H6><H5>- Application and Specificity</H5></DIV><DIV class="ntnu2016-2"><DIV class="show3"><H1>Introduction</H1><P>Â Â Â Â Â BBa_J33201 is a promoter containing the <I>E. coli</I> JM109 chromosomal arsenic detoxification operon (ars operon). Both 2013 Buenos Aires iGEM team and 2006 Edinburgh iGEM team used it to detect arsenic ions in the water. Buenos Aires team ligased it with GFP and Edinburgh team ligased it with LacZ Î±. The two teams also submitted biobricks, BBa_K1106004 and BBa_J33203. We wanted to know whether these two biobricks have more application, such as detecting the arsenic pollution in Chinese medicine. We also wanted to know whether they can detect other metal ions. Several experiments were done to evaluate the function of this promoter.
                    </P></DIV><DIV class="hide3"><H1>The Expression of GFP <SPAN><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K1106004">(BBa_K1106004)</A></SPAN></H1><H2>
                        1. Experimental Design: Growth curve of GFP biosensor in arsenic solutions of different concentration

                    </H2><P>Â Â Â Â Â  This experiment was to test the growth of <I>E. coli</I> DH5Î± with the arsenic solution in comparison to the solution without arsenic ions. Table 5 shows the experimental design for the growth curve
                        of <I>E. coli</I> DH5Î± and the result shows in Figure 9.
                        </P><H3>
                            Table 5: The experimental design for the growth curve of <I>E. coli</I> DH5Î±.
                        </H3><H4>
                            Figure 9: The growth of <I>E. coli</I> DH5Î± in different conditions.
                        </H4><P>Â Â Â Â Â The result shows that the growth of <I>E. coli</I> GFP biosensor in solutions was not affected by the arsenic ion, so GFP biosensor can be used to test arsenic ions.
                        </P><H2>
                            2. Experimental Design: Function test on detecting arsenic ion in Chinese medicine

                        </H2><P>Â Â Â Â Â  This experiment was to quantify the arsenic concentration in three kinds of Chinese medicine: Scutellaria baicale, Angelica, and Yanjing, based on the expression of GFP in arsenic solution. We compared the expression of
                            GFP in <I>E. coli</I> DH5Î± with plasmid BBa_K1106004 in three kinds of Chinese medicine solutions with the expression in the arsenic solution of 1 and 10 ppm. Table 8 shows the experimental design for the expression of GFP and the result shown in Figure 10.
                        </P><H3>
                            Table 6: The experimental design the expression of GFP. The excitation peak is 485nm, the emission peak is 538nm, and the auto cutoff is 515nm.
                        </H3><H4>
                            Figure 10: The expression of GFP.
                        </H4><P>Â Â Â Â Â  The result shows that the growth of <I>E. coli</I> GFP biosensor in solutions was not affected by the arsenic ion, so GFP biosensor can be used to test arsenic ions in Chinese medicine.
                        </P><H2>
                            3. Experimental Design: Specificity

                        </H2><P>Â Â Â Â Â  This experiment was to test the growth of <I>E. coli</I> DH5Î± with the arsenic solution in comparison to solution without arsenic ions.
                        </P><P>Â Â Â Â Â  This experiment was to test whether the GFP biosensor was responsive to other ions. We compared the expression of GFP in solutions of Copper ions and Lead ion. Table 7 shows the experimental design for the expression of
                            GFP and the result shown in Figure 11.
                        </P><H3>
                            Table 7: The experimental design the expression of GFP. The excitation peak is 485nm, the emission peak is 538nm, and the auto cutoff is 515nm.
                        </H3><H4>
                            Figure 11: The expression of GFP.
                        </H4><H1 style="margin-top:100px;">The Expression of LacZ Î± <SPAN><A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_J33203">(BBa_J33203)</A></SPAN></H1><H2>
                            1. Experimental Design: Growth curve of GFP biosensor in arsenic solutions of different concentration

                        </H2><P>Â Â Â Â Â  This experiment was to test the growth of <I>E. coli</I> DH5Î± with the arsenic solution in comparison to the solution without arsenic ions. Table 7 shows the experimental design for the growth
                            curve of <I>E. coli</I> DH5Î± and the result shows in Figure 12.
                        </P><H3>
                            Table 7: The experimental design the the growth curve of <I>E. coli</I> DH5Î±.
                        </H3><H4>
                            Figure 12: The experimental design the the growth curve of <I>E. coli</I> DH5Î±.
                        </H4><P>Â Â Â Â Â  The growth of <I>E. coli</I> DH5Î± with BBa_J33203 plasmid was not affected by the arsenic solution of different concentration.
                        </P><H2>
                            2. Experimental Design: detecting arsenic ion in arsenic solution

                        </H2><P>Â Â Â Â Â This experiment was to test the function of Lacz biosensor. We compared the blue color in the arsenic solution of different concentration. Table 8 shows the experimental design for detecting arsenic ion and the result shows
                            in Figure 13.
                        </P><H3>
                        Table 8: The experimental design for detecting arsenic ion.
                    </H3><H4>
                        Figure 13: The result of detecting arsenic ion.
                    </H4><P>Â Â Â Â Â  The results show that the density of blue color is associated with the concentration of the arsenic solution.
                    </P><H2>
                        3. Experimental Design: Specificity of LacZ Î± biosensor

                    </H2><P>Â Â Â Â Â This experiment was to test whether LacZ Î± biosensor was sensitive to copper and lead ions. We compare the blue color in three types of 10 ppm solution. Table 9 shows the experimental design for detecting arsenic ion and
                        the result shows in Figure 14.
                    </P><H3>
                        Table 9: The experimental design for detecting arsenic ion.
                    </H3><H4>
                        Figure 14: The pictures show LacZ Î± biosensor is not responsive to copper ions and lead ions.
                    </H4><P>Â Â Â Â Â  The results of our experiment show that the LacZ Î± biosensor can grow normally in the arsenic solution of different concentration. This LacZ Î± biosensor detects arsenic ion in arsenic solution and with good quality of specificity.
                        In a word, this LacZ Î± biosensor can be an effective fast screening instrument for detecting the arsenic pollution.
                    </P></DIV></DIV></DIV><P><TITLE>Untitled Document</TITLE></P><DIV class="footer"><DIV id="container"><DIV id="cont"><DIV><UL id="index_foot_list" style="list-style-image: none;list-style-type:none;"><LI><A href="https://2017.igem.org/Team:NCTU_Formosa" style="font-size:16px;color:#FFF;"><B>Home</B></A></LI></UL><LI><A href="#" style="font-size:16px;color:#FFF;"><B>Project</B></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Applied_Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Peptide_Prediction">Peptide Prediction</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Disease_Occurrence_Prediction">Disease Occurrence Prediction</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Demonstrate">Demonstration</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Improve">Improvement</A></LI><LI><A href="#" style="font-size:16px;color:#FFF;"><B>Modeling</B></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Model">Peptide Prediction Model</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Disease_Occurrence_Model">Disease Occurrence Model</A></LI><LI><A href="#" style="font-size:16px;color:#FFF;"><B>Wet Lab</B></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Experiments">Experiment Design</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Fungal_Experiment">Fungal Experiment</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Fungal_Result">Fungal Result</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Protein_Expression">Protein Expression</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Safety">Safety</A></LI><LI><A href="#" style="font-size:16px;color:#FFF;"><B>Parts</B></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Basic_Part">Basic Parts</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Composite_Part">Composite Parts</A></LI><LI><A href="#" style="font-size:16px;color:#FFF;"><B>Human Practice</B></A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Collaborations">Collaboration</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Silver">HP Silver</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/HP/Gold_Integrated">HP Gold Integrated</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Engagement">Education and Public Engagement</A></LI><LI><B>Achievement</B></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Medal_Criteria">Medal Criteria</A></LI><LI><B>Team</B></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Team">Team Introduction</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Attributions">Attribution</A></LI><LI><B>Notebook</B></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Protocol">Protocol</A></LI><LI><A href="https://2017.igem.org/Team:NCTU_Formosa/Notebook">Lab Notes</A></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>