"
Page
Discussion
View source
History
teams
Log in
 
Team:Baltimore US/JuneGroupNotebook
From 2010.igem.org
Home
Team
Official Team Profile
Project
Submitted Parts
Modeling
Notebook
Meeting/Lab Times
Safety
Group Notebook for June 2010
Contents
1 Group Notebook for June 2010
1.1 June 29th, 2010
1.2 June 28th, 2010
1.3 June 24th, 2010
1.4 June 16th, 2010
1.5 June 15th, 2010
1.6 June 14, 2010
1.7 June 10, 2010
1.8 June 9, 2010
1.9 June 8, 2010
1.10 June 7, 2010
June 29th, 2010
In attendance: Day - Duke, Tom, Patrick, Ryan, Robert
Evening: Gary, Liz, Tom, Patrick, Duke, Ryan
During the day-shift Robert continued to work on the PCR optimization, with the second set of Gels showing a continued contamination of one of the main components, pointing possibly to the template mixing into the dNTP's. Patrick helped Tom with some Thermophilic cultures, and Ryan completed the Plasmid Transformations of T3, K3, & C3 along with the Batch prep of the CAT8 plasmids.
We caught tonight's attendees up to speed as to where the group stands on it's individual projects. Then together we discussed in detail how to break up the foundational work-flow and what steps need to be accomplished to make them happen. Patrick detailed some of the foundational work previously performed by the earlier iGEM teams on Artificial Neural Networks.
DIY-Gem notes: Creation of Pol-1 Taq Polmerase in Bb format J04637.1
1. Amplify Pol-1 gene from T. Aquaticus
Convert to Bb format - Add Bb Prefix and Suffix
Change internal Pst-1 site - Single BP Mutation
2. Characterize Part
Error Rate?
Activity
Etc... (Versus Lab Ordered Purity)
3. Educational Supplementation
Techniques - How to do a single point mutation
Tools Discussion / Software & Hardware
DNA Computing Notes: Turing Automatons w. Back Propagation
1. What Species is it?
Identifying parts available.
Identify parts to be made.
Assemble parts.
Build Assemblies.
Test & Characterize System.
What is a Neural Network? The XOR switching system.
- Addressable Conjugation.
- RNA Hairpins.
- HSL Senders / Receivers.
- Others.
- Attaching varied weights among various pathways.
- Summing.
- Thresholdiing.
- Back Propagation Control through Lysing (Cellular Suicide model).
June 28th, 2010
In attendance: Day - Duke, Robert, Patrick, Robert, Ryan, Tom
Evening: Miles, Robert, Patrick, Tom, Ryan, Burnadette
We welcomed a new comer Bernadette Gallagher. And spoke more in depth about project design and workflow. Tom helped get our new team mate oriented, while Miles, Patrick and Robert worked on flushing out conceptual foundations for their projects of interest.
Ryan - Performed Restrictions (e/p)and Ligations on Pb1t3,c3,k3 while assisting the batch prep upon the CAT8 plasmids, performed first step lysis to instruction 14 putting DNA in alcohol.
June 24th, 2010
In Attendance: Patrick, Robert, Tom, Duke, Ryan
After a roughly a week's break from official activities the team met to discuss strategies and tactics.
A few hands have been taking up the lab tech training from Tom and Duke throughout the lazy summer days, while doing so they revisited the basic procedural steps of ligation, gel-phoresis, transformation, as well as the tasks of mixing and pouring media, basic chemical preparation, as well as safety and hygeine in the lab. Tom has also been introducing us to the concepts behind workflow and how to divide and stage the various processes towards a greater project concept.
Miles suggested that he will be working towards several ideas in hardware project construction from a PCR unit to a DNA synthesizer. We discussed in depth Patrick's idea for the creation of a cellular automaton. He broke down the previous tactics in DNA computing where everything is synthesized then re-sequenced for the answer, then posited the creation of an Artificial Neural Network type approach in utilizing various signaling proteins to reach consensus and establishing some way to create back propagation of error.
We've decided to restrict lab nights to Monday and Tuesday from 7-10pm, with a floater strategy meeting on Thursday nights so we can stimulate thought with a change of environment. With that in mind we are back tonight and look forward to seeing those of you who can make it in to keep cranking on our technique.
June 16th, 2010
In attendance: Lisa, Ryan, Tom, Colin (for a short time), Duke (for a moment)
Missing: Steve
Earlier in the day, Robert and Ryan performed a mini-prep of the pcr'd plasmid backbones 3A1(2) and 5A1(2). Robert also began working on finding optimized conditions for pcr.
With the small turnout, Ryan began the digestion restriction with Colin's assistance. We combined the 2 strains of 3A1 and 5A1 together, then put together 3 - .8 eppendorf's with 50 ul of rxn ingrediates.
(2 tubes of positive control of both along with 1 negative control that contained no enzymes and 40 ul of H2O)
5 ul of DNA
1 ul of Pst
1 ul of EcoR1
5 ul of Buffer
38 ul of H20
50 ul - Negative Controls Labelled with minus sign.
After preparation, the cells were put into the pcr block for a 30 minute heat cycle.
Tom will be heading to Arizona, tomorrow, so we called it a night so he could pack and go.
It has been suggested that we run a "Low-Heat Agar Gel" so we can find and physically splice out the appropriate sized plasmid structures.
June 15th, 2010
In attendance: Patrick, Ryan, Duke, Tom, Steve
Missing: Colin, Liz, Scott, Andy
Tom says results seemed to have worked for all of the transformations this time. Some better then others.
Steve asks the question.. "Why did transformation work better this time?"
Tom discusses differences in attempts.
Plasmid contains part a, another contains part b, another part c is simply a linear piece of DNA representing a Tet resistant backbone with the EcoRi and PstI restriction sites.
We cut Part A with EcoR1 and Xba1.
We cut Part B with Spe1 and Pst1.
We cut the Linear Backbone Part C wit EcoR1 and Pst1.
Then we combined everything into one tube using 2 ul of each part into 1 tube, and added t4 DNA ligase, hoping to get a Part A joined to Part B, because the Spe1 combines with the Xba1. And the Part A would join Part C with the EcoR1 and the Part B with Part C at the Pst1 site.
(See photo of board - )
Last week, we got nothing, so why?
Where could things go wrong?
They could go wrong at the restriction digestion reaction (less likely as process is fairly robust, and had been tested the day previously.)
Ligation reaction may not have worked.. (again less likely, process is fairly robust). If the buffer isn't mixed well it may separate.
How do we test? we could perform pcr to see if the ligation took place, using primers to amplify the ligated segment checking with gel electrophoreis to identify.
Transformation reaction - (Except we ran positive controls successfully)
Still possible to have mixed up the media, since positive used Amp instead..
Tom checked the media and found that wasn't the case.
Amount of Plasmid may not have been accurate. (The only thing we didn't physically check ahead of time.)
Pure operator/Pipetting action - such as the 30 second heating followed by kill cycle.
Detection limits for Ethidium Bromide 10-15 ngs of a product.
(3600 bp ) (650 gm/b.p molar weight)
2,340,000 Grams per mole of 3600 bp molecule.
50 nanograms
(50 x 10-9) / 2,340,000 = 2.1 x 10-14 moles
Transformation rxn is 1 x 10 (-8/-9), still fairly efficient.
We did 2 things last night, we did the transformations into the stocks..
By using assembly strategies that don't use the same resistance as the parts, we can more efficiently screen out non-transformed pieces.
Previously in Genetic Engineering all parts and pieces had variorus resistances where as bb tries to create a standard.
Cells can be lost in replication due to metabolic costs, and new generations lacking the Costly programming tend to outgrow the special cells.
When designing the assembly strategy try to use a plasmid backbone with a different resistance to allow the filtration/screening of parts that didn't transform correctly.
Gel-Electrophoresis Prep Review for tonights run.
Agarose 2 % for smaller parts will create a more viscous environment.
0.8% for the larger parts will allow longer molecules to move better.
50ml final volume
Weigh out the agarose (.4 gms for the 0.8%) (1 gm for the 2%)
Emphasize again the importance of NCBI databases... http://www.ncbi.nlm.nih.gov/
With links to BLAST, PubMed, and other official literature. Program called entree which links all the information together.
When reviewing the tubes from last evening left in PCR there was difficulty in identifying the tubes that were done by each individual. We spent a little time using the process of elimination. Knowing Patrick, Steve, Miles, Ryans and Gary's without issue, we had to guesstimate which was Roberts, and which was David's. The reason we needed differentiation was to create a loading template, in one of the two different agarose preparations.
June 14, 2010
In Attendance: Tom, Duke, Patrick, Robert, Steve, Gary, David, Ryan, Miles
3 Separate transformations attempted: <partinfo>pSB1C3</partinfo> [1-3A] (ChlorAmphenacol resistant), <partinfo>pSB1K3</partinfo> [1-5A](Kanamicin resistant), <partinfo>pSB1T3</partinfo> [1-7A] (Tetracycline resistant) Control Group: PET-17B  (still had separate resistance for Ampicillan) Tom will be culturing some adjacently resistant control groups for our next trial, thereby eliminating the extra variable in our testing.
Procedural experience:
Transformations using 1-3A 50uL EXPERIMENTAL
200uL EXPERIMENTAL
250uL NEGATIVE CONTROL
Cells were plated and put in the 37C incubator.  Plates labeled in RED.
9 Parts for PCR - 7 Distributed amongst us. Various participants were asked to look up their function and various bp lengths.
Patrick - <partinfo>Bba_R0010</partinfo> - 200 bp
Gary - <partinfo>Bba_R0063</partinfo> - 151 bp
Robert - <partinfo>Bba_J04450</partinfo> - 1,069 bp
Dave - <partinfo>Bba_R0062</partinfo> - 55 bp
Steve - <partinfo>Bba_J23009</partinfo> - 97 bp
Miles - <partinfo>Bba_I731014</partinfo> 1,938 bp
Ryan - <partinfo>Bba_I13507</partinfo> - Intermediate Screening Plasmid - 861 bp
PCR for part R0063 :
Vial labeled with RED X is the EXPERIMENTAL REACTION
Vial labeled with RED Y is the TEMPLATE CONTROL
Vial labeled with RED Z is the PRIMER CONTROL
PCR Polymerase Chain Reactions
Requirements
1. Template
2. 2 Primers (limiting)
3. dNTP's (limiting)
4. Polymerase (limiting by heatcycle) Using Taq which tends to transcribe more errors.
5. Buffer (mg++) (limiting)
Primers are 18-30 bp long. Have to be at least 15 bp long. Primers bind to template DNA, one primer for each (5'/3' strands). Review: What is a base..? Sugar with A, C, T or G is a nucleotide. BP are the matching set of 5' and 3' hydrogen bonded nucleotides.
Polymerase links free dNTP's to the opened strand edges of basepairs.
Denaturation 95 - 98 degrees c... DNA Melts
Anealing 40 - 65 degrees c... Primer Binds
Extension 72 degrees c... Strands Extend
30 second cycles...
As the DNA heats it denatures then as it cools the primers anneal and polymerization takes place creating twice as many strands. The strands are held together only by hydrogen bonds, making it very easy to melt and reform. The polymerase acts like a little machine stitching the free dNTP's into the template.
BioBrick plasmids have bb prefix and suffix. Primer binding sites vf2, vr. Theoretically any area can become a primer site.
With PCR we can identify and amplify particular strands/compositions. You have to know how much of the requirements to use. Gary asked if Tom could show us where to find the optimized volume information at, and he agreed to pull out the research to show how to optimize.
With PCR you can also design primers and introduce mutations. If primers are too short they can bind to the wrong spot. Actual reactions may vary based on bp length and combination. Usually they are tweaked for the optimal over a series of trials.
PCR Reactions (Specific to this evening's trial)
Experimental Reaction
Neg. Template Control
Neg. Primer Control>
Template
2 uL
0
2 uL
Forward Primer [VF2 310 uM]
16uL**
16uL**
0
Reverse Primer [VR 345 uM]
14uL**
14uL**
0
dNTPs
1uL
1uL
1uL
Rxn Buffer [5X-k]
10uL
10uL
10uL
Enzyme
0.5uL
0.5uL
0.5uL
Water
6.5uL
8.5uL
36.5uL
Total Volume
50uL
50uL
50uL
** indicates 1:100 dilution
34mM of dried DNA in primers ($12.50)
Several companies provide these primers: invitrogen is an example.
34 x 10-9 moles / 100 x 10-6 liters = 3.4 x 10 -7 moles/liter
Tomorrow we will look at the parts we pcr'd and then see how the bp length matches. If nothing happens its probably a pipetting error. If too many bands it may be an annealing issue.
We will also look at the cells we plated of the various transformed parts, 3A, 5A, 7A, and see again if any of the colonies grew.
June 10, 2010
In attendance: Duke, Gary, Robert, Patrick, Ryan, Miles, Steven, Liz, and Colin.
Tonight we tested the hypothesis that the plasmid did not code for the antibiotic resistance/that we did not use enough DNA.  A Restriction Digest reaction was set up on the ligation reactions completed on June 7.  The reactions were cut with EcoR1 and Pst1.  The reactions were allowed to incubate at 37 degrees C for one and a half hours.  The digests were then ran on a electrophoresis gel.  No DNA was present as there was not enough DNA, thus proving Duke's hypothesis.
June 9, 2010
In attendance: Duke, Tom, Ryan, Patrick, Lisa, Steven. Missing: Kyle and Friend(Forgot his name)
1) Registration for iGEM site & Team affiliation, required to update team wiki.
2) Access and update wiki..  Consider stylization, as well as information content. Who is our audience?  Team info - blurb and caption of team members. Couple of sentences about you and your interests.  Lab Notebook? - Openwetware connection.
3) Research biobrick parts by accessing in Registry of Standard Parts.  XF to see if new combination are already on file?  If not begin documentation of the 3 new assemblies, with image using standard BioBrick icons.  Document bp length of new parts to compare with Gel-Electrophoresis.  How do we test efficacy? (Part for PPM still in brick.)
Is the media okay, is the antibiotic right. Positive tells you whether the cells themselves were capable of taking up our DNA. (Ideally, they should use the same antibiotic resistance, as our parts.
Negative groups tells us whether the antibiotic and media was effective and that the cells were not resistant to the media.
We know that cells were competent they were able to be transformed, and the antibiotic was effective. So the remaining question is whether they were able to be transformed.
The ligation's are in question. Generally the restrictions and ligation's go smoothly.
Tom's Hypothesis is questioning whether the media plates may have been mis-labeled with the wrong antibiotic resistance. Since we used a different resistance for controls, there was no telling.
Possible denaturing of enzymatic proteins from the heat shock on the initial restriction? 80c kill cycle. Vector ? - Linear plasmids - was the amount to small...? Parts - size already was confirmed, according to Tom
No Transformations.... so now what? Primers still on the way. Don't need to go to scratch.
The Core enzymes needed to perform the biobricks constructions... Spe1 pst1 ecor1 xba1 t4dna ligase polymerase
Project idea:
Polymerase in e.coli,  tag, perhaps other enzymes could be self-manufactured/purified.
Basic tools/measurements
Micropipetters [Measuring volume]
Measuring Mass
DV/HD camera - you tube videos and components.
Mini-preps. Growing the cells and isolating DNA. Cell competency Preparations.
Lac polymerase - DIY-GEM
How do you purify?
What do you do with it... what is the process to create a project?
Hardware only takes you so far, these 5 basic tools. Restrictions Ligations Transformations/Plating PCR Gel Electrophoresis
The core will be good technique with these processes and then the ability to understand the existing database navigation with the proper questions to yield an experiment of interest.
Research educational tools from the MIT educator on IGEM site. Should we have an e-mail/Comments section added to wiki - openwetware, so observers can ask us questions?
June 8, 2010
In attendence: Colin, Patrick, Liz, Ryan, Duke and Tom. Missing: Andy, Scott.
So tonight we came in and took the 4 ligation's/new part combinations from yesterday and transformed them into the competent cells, using a heat-shock transformation, while also preparing the Control groups (one with nothing/one with the ampicillin resistance). 3 of the parts were Tetracycline resistant, and 1 was Chlorephenecol. After the 90 minute transformation cycle we plated 6 versions of each of the 3 parts along with 1 plate of control and one negative group.
The 6 plates were done in a 0, -1, -2 dilutions in 2 concentrations one of 50 ul, and one of 200 ul.
They will be left for tomorrows team to run gel electrophoresis to determine whether they have the appropriate combination links.
June 7, 2010
In attendance: Colin, Patrick, Robert, Ryan, David, Gary, Miles, Duke and Tom. Missing: Melissa.
Tom greeted us with 4 separate sheets that contained 3 reactions each for us to begin restrictions and began heating to let the enzymes cut. A lesson in what not to do, was offered as we began the heating cycle of restriction in the PCR blocks and he timing had been set to 35 seconds instead of 35 minutes, after which it heated to a kill cycle of 80 degrees and we had to reapply the enzymes, in case the enzymes had been denatured.
We had 6 individuals building the 4 sheets, 2 in redundancy.
The legend for the various parts is as follows...
1PO
<bbpart>BBa_R0063</bbpart>
2PO
<bbpart>BBa_P0412</bbpart>
3PO
<bbpart>pSBIT3</bbpart>
R (black marker)
<bbpart>BBa_R0062</bbpart>
I (black marker)
<bbpart>BBa_I13507</bbpart>
S (black marker)
<bbpart>pSBIT3</bbpart>
F (green marker)
<bbpart>BBa_F2620</bbpart>
I (green marker)
<bbpart><BBa_I13507</bbpart>
S (green marker)
<bbpart>SBIC3</bbpart>
R10 (green marker)
<bbpart>BBa_R0010</bbpart>
462 (green marker)
<bbpart>BBa_I0462</bbpart>
IT3 (green marker)
<bbpart>pSBIT3</bbpart>
A/R10 (green marker)
<bbpart>BBa_R0010</bbpart>
B/462 (green marker)
<bbpart>BBa_I0462</bbpart>
C/IT3 (green marker)
<bbpart>pSBIT3</bbpart>
Patrick cut parts <bbpart>BBa_r0063</bbpart> and <bbpart>BBa_p0412</bbpart>, and the plasmid backbone <bbpart>pSBIT3</bbpart>.
Robert cut parts <bbpart>BBa_R0062</bbpart> and <bbpart>BBa_I13507</bbpart>, and the plasmid backbone <bbpart>pSBIT3</bbpart>.
Collin/Miles cut parts <bbpart>BBa_F2620</bbpart>, <bbpart>BBa_I13507</bbpart> and the plasmid backbone <bbpart>pSBIC3</bbpart> (labeled w/green marker).
The next step is to ligate them.
After which we had a round table discussion about what kind of projects we may follow up with and the process of using the NCBI databases to discover pre-existing sequence information related to our various ideas. One idea we have discussed was the option of creating a smoother introductory curve for fellow DIY-Bio commmunity members and the creation of home-brewed enzymes that might be to pricey for the amateur scientists. We ended the night with the beginning of the various ligation reactions, as seen above.
Ligation Reaction for R10+462+IT3 in 'LIGATE 05X
Ligation Reaction for A/R10+B/462+C/IT3 in 'L/LIG
Ligation Reaction for F2620+I13507+IC3 with "Green Asetrisk"
Ligation Reaction for R0062+I13507 in "Squiggly Sigil"
Retrieved from " http://2010.igem.org/Team:Baltimore_US/JuneGroupNotebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
