<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USC_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:USC/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USC/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USC/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USC/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USC/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USC/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><TABLE><TBODY><TR><TD colspan="3"><DIV style="background: rgb(153,0,0);width: 954px;height: 36px;margin: 0 auto;padding: 0;"><UL><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC" title="Team:USC">Home</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC/Team" title="Team:USC/Team">Team</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><STRONG class="selflink">Project</STRONG></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC/Notebook" title="Team:USC/Notebook">Notebook</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC/Parts" title="Team:USC/Parts">Parts</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC/Safety" title="Team:USC/Safety">Safety</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC/Human_Outreach" title="Team:USC/Human Outreach">Human Outreach</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="/Team:USC/Attributions" title="Team:USC/Attributions">Attributions</A></SPAN></SPAN><SPAN style="float: left; padding: 15px;"><SPAN style="font-family: Arial, Helvetica, sans-serif;font-size: 15px;font-weight: bold;color: #FFFFFF;border: none;"><A href="https://igem.org/Team.cgi?year=2011&amp;team_name=USC" class="external text" rel="nofollow">Official Team Profile</A></SPAN></SPAN></UL></DIV></TD></TR><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Project"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Project</SPAN></A></LI><LI class="toclevel-1"><A href="#Overview"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Overview</SPAN></A></LI><LI class="toclevel-1"><A href="#Details"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Details</SPAN></A></LI><LI class="toclevel-2 tocsection-1"><A href="#Genetic_Arrangement_of_CRISPR.2FCas_System"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Genetic Arrangement of CRISPR/Cas System</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Synthetic_Version_of_CRISPR"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Synthetic Version of CRISPR</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Results"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Results</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#CRISPR.2FCas_System_in_K12_E._coli"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">CRISPR/Cas System in K12 E. coli</SPAN></A></LI><LI class="toclevel-3 tocsection-4"><A href="#HT115_w.2F_tetO::GFP"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">HT115 w/ tetO::GFP</SPAN></A></LI><LI class="toclevel-3 tocsection-5"><A href="#HT115_w.2F_tetO::GFP_.26_CRISPR-GFP"><SPAN class="tocnumber">4.1.2</SPAN><SPAN class="toctext">HT115 w/ tetO::GFP &amp; CRISPR-GFP</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#HT115_w.2F_CRISPR-GFP"><SPAN class="tocnumber">4.1.3</SPAN><SPAN class="toctext">HT115 w/ CRISPR-GFP</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#CRISPR.2Fcas_System_in_B_E._coli"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">CRISPR/cas System in B E. coli</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#BL21_.28DE3.29_strain_w.2F_tetO::GFP"><SPAN class="tocnumber">4.2.1</SPAN><SPAN class="toctext">BL21 (DE3) strain w/ tetO::GFP</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#BL21_.28DE3.29_strain_w.2F_tetO::GFP_.26_CRISPR-GFP"><SPAN class="tocnumber">4.2.2</SPAN><SPAN class="toctext">BL21 (DE3) strain w/ tetO::GFP &amp; CRISPR-GFP</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#BL21_.28DE3.29_strain_w.2F_tetO::GFP.2C_CRISPR-GFP.2C_cas3.2C_casABCDE"><SPAN class="tocnumber">4.2.3</SPAN><SPAN class="toctext">BL21 (DE3) strain w/ tetO::GFP, CRISPR-GFP, cas3, casABCDE</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#BL21_.28DE3.29_strain_w.2F_tetO::GFP.2C_CRISPR-GFP.2C_cas3.2C_casABCDE12"><SPAN class="tocnumber">4.2.4</SPAN><SPAN class="toctext">BL21 (DE3) strain w/ tetO::GFP, CRISPR-GFP, cas3, casABCDE12</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Future_Directions"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Future Directions</SPAN></A></LI></TD></TR></TBODY></TABLE><H1 style="font-family:Verdana;font-weight:700;"><SPAN class="mw-headline" id="Project">Project</SPAN></H1><H1 style="font-family:Verdana;font-weight:500;"><SPAN class="mw-headline" id="Overview">Overview</SPAN></H1><P>Bacteria protect their genome and remove foreign DNA elements through the use of the clustered, regularly interspaced short palindromic repeat (CRIPSR) system. Recently discovered, this defense mechanism can protect microorganisms against bacteriaphages.  The CRISPR system comprises a transcribed locus of repeats and spacers that is activated and made functional via a family of CAS proteins. The CRISPR repeats are short DNA sequences that are almost identical in size and sequence. Between a pair of repeats is a spacer, which if complementary to a DNA element of the invading bacteriaphage, facilitates the destruction of that sequence from the bacterial cell.  As such, the CRISPR/CAS system is a primitive biological defense mechanism, similar in many ways to the process of adaptive immunity in eukaryotic immune systems.  
Plasmids, which are commonly used in molecular biology are also the source of horizontal gene transfer in the wild and facilitate the spread of antibiotic resistant genes among microorganisms.  These plasmids can contain unique DNA sequences that could be targeted by the CRISPR/CAS system.  We exploit the CRISPR/CAS system to engineer a mechanism of plasmid curing and deactivation of antibiotic resistance genes in E. coli.
</P><P>We synthesized a version of CRISPR encoding a spacer that matches the GFP DNA sequence. We tested the synthetic CRISPR array against E.coli harboring a tetO::GFP plasmid that confers ampicillin resistance. Activation of CRISPER-GFP destroys the GFP-containing plasmid restoring the bacterial host's sensitivity to ampicillin. We will use the synthetic CRISPR system as a biological tool, combining it with other BioBricks for use in applications that will impact health and medicine, biotechnology, molecular biology, and genetics.
</P><H1 style="font-family:Verdana;font-weight:500;"><SPAN class="mw-headline" id="Details">Details</SPAN></H1><P>Studies have shown that <I>Escherichia coli</I> bacteria have a system that provides immunity against viral infections and plasmid conjugation in the natural world. This recently discovered system, also known as the CRISPR/Cas system, is made of eight cas (CRISPR-associated) genes and an array of repeats and spacers known as CRISPR (Clustered Regularly Interspaced Short Palindromic Repeats). The repeats are repeating DNA sequences of approximately thirty nucleotides that are separated by unique genetic elements, spacers. The sequences of these spacers are generally derived from foreign virus DNA or plasmids. The CRISPR array expands when the bacteria collects a new spacer in a process known as CRISPR adaptation. The bacteria only confers resistance to the virus or plasmid when its CRISPR repertoire contains a spacer that matches the DNA sequence of the invading DNA. Although parts of the pathway for CRISPR adaptation and CRISPR interference are being studied, many mechanisms such as spacer integration into the CRISPR array as well as the degradation signaling pathway for CRISPR-targeted DNA remain to be discovered. Below is a schematic on a possible pathway the CRISPR/Cas system in <I>E. coli</I> utilize for CRISPR interference:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Possible mechanism of CRISPR/cas system against viral DNA</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Genetic_Arrangement_of_CRISPR.2FCas_System">Genetic Arrangement of CRISPR/Cas System</SPAN></H3>
To the right is a schematic diagram of how the CRISPR/Cas system is arranged genetically.<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption">Diagram of CRISPR/Cas system. The repeats are displayed as rectangles. Spacers are colored.</DIV></DIV></DIV></DIV> Upstream of the CRISPR locus are a series of cas genes arranged sequentially in the following order:
<UL><LI>cas3
</LI><LI>casABCDE12 (short for casA, casB, casC...)
</LI></UL><P>What follows is the promoter of the CRIPSR locus, which has been sequenced to be A-T nucleotide rich. The CRISPR array of repeats and spacers completes the system.
</P><H3><SPAN class="mw-headline" id="Synthetic_Version_of_CRISPR">Synthetic Version of CRISPR</SPAN></H3><P>The idea of our project comes from utilizing the <I>E. coli</I> bacteria's natural way of conferring resistance to foreign DNA and plasmid and see if it can be engineered such that the CRISPR/Cas system can target specifically any piece of DNA by means of programming the system with a spacer of any DNA sequence. Such a system has been engineered by the Laboratory of Microbiology in the University of Wageningen and is displayed below:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">K12 Inspired CRISPR cassette with alternating K12-targeting spacers and restriction sites</DIV></DIV></DIV></DIV><P>This part, BBa_K512002, is an IPTG-inducible CRISPR array genetically engineered with alternating spacers that target the K12 genome and contain restriction sites. The restriction enzyme sites are designed so that a spacer of any DNA sequence can be inserted into this CRISPR system.
</P><P>In our particular project, we designed our synthetic CRISPR to target a DNA sequence coding for GFP. To do so, we took the synthetic CRISPR and GFP spacer and digested with two compatible restriction enzyme so that the spacer can be properly ligated into our synthetic CRISPR. The protocol is displayed below: 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Protocol for constructing CRISPR-GFP: yellow=restriction enzyme site, blue=repeat, red=K12 targeting spacer, green=GFP-targeting spacer</DIV></DIV></DIV></DIV><H1 style="font-family:Verdana;font-weight:500;"><SPAN class="mw-headline" id="Results">Results</SPAN></H1><P>Once the CRISPR-GFP plasmid has been constructed, we decided to test the system in K12 and B <I>E. coli</I>:
</P><H3><SPAN class="mw-headline" id="CRISPR.2FCas_System_in_K12_E._coli">CRISPR/Cas System in K12 E. coli</SPAN></H3><P>To test the CRISPR/Cas system in K12, we designed the following strains:
</P><UL><LI>HT115 (K12) strain w/ tetO::GFP (AmpR)
</LI><LI>HT115 (K12) strain w/ tetO::GFP &amp; CRISPR-GFP (AmpR &amp; CmR)
</LI><LI>HT115 (K12) strain w/ CRISPR-GFP (CmR)
</LI></UL><P>We tested each system by creating three batches for each strain so we can monitor the growth of the bacteria over time:
</P><UL><LI>One strain + antibiotics - IPTG
</LI><LI>Same strain + antibiotics + IPTG
</LI><LI>Same strain + antibiotics + 1 hr. before inducing system with IPTG
</LI></UL><H4><SPAN class="mw-headline" id="HT115_w.2F_tetO::GFP">HT115 w/ tetO::GFP</SPAN></H4><P>Regarding the first strain (HT115 w/ tetO::GFP), we used this test as our control.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">HT-A1=(-IPTG), HT-A2=(+IPTG), HT-A3=(1hr.+IPTG)</DIV></DIV></DIV></DIV><P>The three batches of the strain grew at approximately the same rates.
</P><H4><SPAN class="mw-headline" id="HT115_w.2F_tetO::GFP_.26_CRISPR-GFP">HT115 w/ tetO::GFP &amp; CRISPR-GFP</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Left: HT-B1 overnight; Right: HT-B2 overnight</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Upper &amp; lower left: HT-B1; Upper &amp; lower right: HT-B2</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">HT-B1=(-IPTG), HT-B2=(+IPTG), HT-B3=(1hr.+IPTG)</DIV></DIV></DIV></DIV><P>With the addition of CRISPR-GFP, there is a severe drop off in growth rate for the batches induced with IPTG. This indicates that CRISPR-GFP system is destroying the tetO::GFP plasmid that also contains the antibiotic resistance gene for ampicillin. Since the bacteria are growing in cultures containing ampicillin, the activated CRISPR renders the bacteria incapable of surviving in the presence of ampicllin. As a result, they are no longer able to survive.
</P><P>Above, the picture diagrams show our test for tetO::GFP. After the growth test, we clearly observed that the batch HT-B1 was more dense than that of HT-B2. More importantly, under a fluorescence scope, we also observed significant GFP expression from HT-B1. However, no GFP expression could be observed from HT-B2. This indicates that HT-B1 still harbors the tetO::GFP plasmid, whereas HT-B2 does not.
</P><H4><SPAN class="mw-headline" id="HT115_w.2F_CRISPR-GFP">HT115 w/ CRISPR-GFP</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">HT-D1=(-IPTG), HT-D2=(+IPTG), HT-D3=(1hr.+IPTG)</DIV></DIV></DIV></DIV><P>This test is used to confirm that CRISPR can also target and destroy genomic DNA from the K12 family. Note that this CRISPR system also contains spacers that match the genomic DNA of K12 bacteria. Although this data introduces a bias to the growth test of strain HT115 with tetO::GFP &amp; CRISPR-GFP, we are encouraged by the observation that the batches of bacterial strain HT115 with CRISPR-GFP induced by IPTG grew slightly more before dying from the presence of ampicillin.
</P><H3><SPAN class="mw-headline" id="CRISPR.2Fcas_System_in_B_E._coli">CRISPR/cas System in B E. coli</SPAN></H3><P>To test the CRISPR/Cas system in B, we designed the following strains:
</P><UL><LI>BL21 (DE3) strain w/ tetO::GFP (AmpR)
</LI><LI>BL21 (DE3) strain w/ tetO::GFP &amp; CRISPR-GFP (AmpR &amp; CmR)
</LI><LI>BL21 (DE3) strain w/ tetO::GFP, CRISPR-GFP, cas3, casABCDE (AmpR, CmR, KanR, StrepR)
</LI><LI>BL21 (DE3) strain w/ tetO::GFP, CRISPR-GFP, cas3, casABCDE12 (AmpR, CmR, KanR, StrepR)
</LI></UL><P>We tested each system by creating three batches for each strain so we can monitor the growth of the bacteria over time:
</P><UL><LI>One strain + antibiotics - IPTG
</LI><LI>Same strain + antibiotics + IPTG
</LI><LI>Same strain + antibiotics + 2 hr. before inducing system with IPTG
</LI></UL><H4><SPAN class="mw-headline" id="BL21_.28DE3.29_strain_w.2F_tetO::GFP">BL21 (DE3) strain w/ tetO::GFP</SPAN></H4><P>Regarding the first strain (BL21 w/ tetO::GFP), we used this test as our control. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">BL-A1=(-IPTG), BL-A2=(+IPTG), BL-A3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><P>The three batches of the strain grew at approximately the same rates.
</P><H4><SPAN class="mw-headline" id="BL21_.28DE3.29_strain_w.2F_tetO::GFP_.26_CRISPR-GFP">BL21 (DE3) strain w/ tetO::GFP &amp; CRISPR-GFP</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">BL-B1=(-IPTG), BL-B2=(+IPTG), BL-B3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Left:BL-B1=(-IPTG), Middle:BL-B2=(+IPTG), Right:BL-B3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">BL-B1=(-IPTG), BL-B2=(+IPTG), BL-B3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><P>The growth test for BL21 (DE3) strain with tetO::GFP &amp; CRISPR-GFP shows interesting results. Although we expected growth rates to be similar to our control strain, we instead observed lowered growth rates in batches induced with IPTG. Although studies have shown that CRIPSR interference against viruses in <I>E. coli</I> is governed by cas3, casABCDE, and CRISPR, this may indicate that CRISPR interference against plasmids in vivo may be governed by a slightly different cocktail of genes. This is a test we look forward to repeating. 
</P><P>Above, the picture diagrams display our test for GFP expression. After the growth test, we observed that BL-B1 was the most dense of the batches. In addition, it displayed the brightest GFP expression. These observations substantiate the possibility the CRISPR-GFP works under a mechanism that may require different genes or no cas genes.
</P><H4><SPAN class="mw-headline" id="BL21_.28DE3.29_strain_w.2F_tetO::GFP.2C_CRISPR-GFP.2C_cas3.2C_casABCDE">BL21 (DE3) strain w/ tetO::GFP, CRISPR-GFP, cas3, casABCDE</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">BL-D1=(-IPTG), BL-D2=(+IPTG), BL-D3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Left:BL-D1=(-IPTG), Middle:BL-D2=(+IPTG), Right:BL-D3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">BL-B1=(-IPTG), BL-B2=(+IPTG), BL-B3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><P>Compared to the growth test of strain BL21 with tetO::GFP &amp; CRISPR-GFP, we have a stronger CRISPR interference response as shown in the growth test curve.
</P><H4><SPAN class="mw-headline" id="BL21_.28DE3.29_strain_w.2F_tetO::GFP.2C_CRISPR-GFP.2C_cas3.2C_casABCDE12">BL21 (DE3) strain w/ tetO::GFP, CRISPR-GFP, cas3, casABCDE12</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">BL-E1=(-IPTG), BL-E2=(+IPTG), BL-E3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">Left:BL-E1=(-IPTG), Middle:BL-E2=(+IPTG), Right:BL-E3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">BL-E1=(-IPTG), BL-E2=(+IPTG), BL-E3=(2hr.+IPTG)</DIV></DIV></DIV></DIV><P>Compared to the growth test of strain BL21 with tetO::GFP &amp; CRISPR-GFP, we have a stronger CRISPR interference response as shown in the growth test curve.
</P><H1><SPAN class="mw-headline" id="Future_Directions">Future Directions</SPAN></H1><P>CRISPR is an effective method for plasmid curing, as seen by our own results. Although further research much be conducted to validate its ability to destroy all types of plasmids in bacterial cells, the future of CRISPR is positive. CRISPR, for example, has the potential to combat plasmids that carry antibiotic-resistant genes to other bacteria, hence reversing the evolution of antibiotic-resistance in bacteria. Moreover, we can prevent harmful phages from destroying certain bacteria by programming CRISPR to target foreign DNA. 
</P>
There are numerous benefits of CRISPR, from medical to environmental. However, CRISPRâs potential will not be fully realized until further research is accomplished. We must still figure out the effects combinations of spacers and CRISPR cassettes have on plasmids curing. We must also discover ways to practically implement these systems. Although there is much left to learn, CRISPR is definitely a promising area of synthetic biology.<DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:USC/Project">http://2011.igem.org/Team:USC/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USC/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USC/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USC/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USC/Project&amp;oldid=203200" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>