<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NYMU-Taipei_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NYMU-Taipei/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="panelsmartphone1"><DIV id="centered-list1"><UL id="panel-ul"><LI class="Home"><A href="https://2016.igem.org/Team:NYMU-Taipei">Home</A></LI></UL><UL id="panel-ul111"><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Achievement">Achievement</A></LI></UL><UL id="panel-ul222"><LI><A href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Project-Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Project-Experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Project-Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Description">Characterization</A></LI></UL><UL id="panel-ul111"><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Demonstrate">Prototype</A></LI></UL></DIV></DIV><DIV id="panelsmartphone2" style="top:79px;"><DIV id="centered-list2"><UL id="panel-ul333"><LI><A href="#">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/HP/Silver">Silver HP</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/HP/Gold">Gold HP</A></LI></UL><UL id="panel-ul444"><LI><A href="#">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Notebook-Lab_Book">Lab Note</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Notebook-Protocol">Protocol</A></LI></UL><UL id="panel-ul111"><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Notebook-Biosafety">Biosafety</A></LI></UL><UL id="panel-ul555"><LI><A href="#">Team</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Team-Member">Team</A></LI><LI><A href="https://igem.org/Team.cgi?team_id=2040">Official Profile</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Collaborations">Collaboration</A></LI></UL></DIV></DIV><DIV id="panelcomputer"><DIV id="centered-list"><UL id="panel-ul"><LI class="Home"><A href="https://2016.igem.org/Team:NYMU-Taipei">Home</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Achievement">Achievement</A></LI><LI><A href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Project-Background">Background</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Design">Design</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Project-Experiment">Experiment</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Project-Model">Model</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Parts">Parts</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Description">Characterization</A></LI></UL><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Demonstrate">Prototype</A></LI><LI><A href="#">Human Practice</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/HP/Silver">Silver HP</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/HP/Gold">Gold HP</A></LI><LI><A href="#">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Notebook-Lab_Book">Lab Note</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Notebook-Protocol">Protocol</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Notebook-Biosafety">Biosafety</A></LI><LI><A href="#">Team</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Team-Member">Team</A></LI><LI><A href="https://igem.org/Team.cgi?team_id=2040">Official Profile</A></LI><LI><A href="https://2016.igem.org/Team:NYMU-Taipei/Collaborations">Collaboration</A></LI></DIV></DIV><DIV id="wrap"><DIV class="fund"><H2 style="margin-top:20px; margin-bottom:10px;">Overview</H2><P style="font-size:16px;">Through the utilization of different gene expression and location of <I>M. anisopliae</I> cells during different stages of its infection cycle alongside with light-induced phototoxic fluorescent proteins, we aim to design entomopathogenic-fungi-specific killswitch that can greatly mitigate the safety concerns of the genetic modified fungal insecticides in hope that it will increase the feasibility of wide spread use of these enhanced bioinsecticides.</P></DIV><DIV class="fund"><H2 style="margin-top:20px; margin-bottom:10px;">Design</H2><B style="font-size:16px;">What is the infection cycle of <I>M anisopliae</I> ?</B><P style="font-size:16px;">Here we show a short overview: The infection process of <I>M. anisopliae</I> is similar to other entomopathogenic fungi, i.e. the infection pathway consists on the following steps: (1) attachment of the spore to the cuticle, (2) germination and formation of appressoria, (3) penetration through the cuticle, (4) overcoming of the host response and immune defence reactions of the host, (5) spreading within the host by formation of hyphal bodies or blastospores, i.e. yeast like cells, and (6) outgrowing the dead host and production of new conidia.<SUP>[1]</SUP></P><B style="font-size:16px;">What is <I>PMcl1</I> ?</B><P style="font-size:16px;"><I>Metarhizium</I>-collogen-like promoter, or <I>PMcl1</I>, is a strong insect-hemolymph inducible promoter. This promoter controls the production of <I>Metarhizium</I>-collagen-like-proteins in wildtype <I>Metarhizium anisopliae</I>, the transcripts of the <I>Mcl1</I> gene could be detected within 20 minutes of the fungus contacting the hemolymph, but could not be detected when the fungus is cultured in any other medium. Utilizing the promoter of <I>PMcl1</I> in genetically engineering <I>Metarhizium</I> allows the expression of target genes to be limited to the hemocoel of the fungus' host insects, ensuring the specificity of gene expression<SUP>[2]</SUP>.</P><B style="font-size:16px;">What is KillerRed ?</B><P style="font-size:16px;">The KillerRed protein is a red fluorescent protein with excitation and emission maxima at 585 and 610 nm respectively. It is engineered from anm2CP to be phototoxic.<SUP>[3]</SUP> When the protein comes in contact with light of wavelengths of 520-590 nm, it produces reactive oxygen species (ROS) along with intense photobleaching. The phototoxicity induced by KillerRed generated ROS is 1,000times greater than those produced by common fluorescent proteins<SUP>[4]</SUP>.  Expression of KillerRed and irradiation with light may act a kill-switch for biosafety applications<SUP>[3]</SUP>. The increase concentrations of ROS in the cytoplasm disrupt normal cellular functions and impede fungal growth and, in the best case scenario, induce programed necrosis<SUP>[5]</SUP>.

KillerRed effectively killed bacterial cells when exposed to white light for several minutes. However, in eukaryotic cells, irradiation of KillerRed localized in cell cytosol has a weaker effect on cell survival<SUP>[6]</SUP>. The following two ways have been found to be effective for killing the eukaryotic cells using KillerRed: (1) via an apoptotic pathway using KillerRed to target mitochondria, and (2) via membrane lipid oxidation using membrane-localized KillerRed. <SUP>[6]</SUP>. However, chromatin is also a ROS-sensitive intracellular localization.So we designed to fuse a SV40 nuclear localization signal to KillerRed protein in order to increase efficiency of KillerRed-mediated oxidative stress.
</P></DIV><DIV class="fund"><H2 style="margin-top:20px; margin-bottom:10px;">In-Out-Suicide</H2><P style="font-size:16px;">
We designed our circuit to utilize the gene expression change of <I>M. anisopliae</I> to activate the production phase of our killswitch. Utilizing <I>PMcl1</I> from <I>M. anisopliae</I>, a promoter with fast activation and high production rate, high amounts of KillerRed protein with SV40 NLS will be produced and transported to the nucleus. During <I>M. anisopliae</I>’s stay within the hemocoel and other interior organs of the insect, KillerRed molecules will remain inert due to the lack of yellow light. When <I>M. anisopliae</I> depletes the nutrients in host's body, the fungus will emerge from the carcass of its host for conidiation. This put fungal cells in direct contact with sunlight, allowing KillerRed to create reactive oxygen species like O2․-, which will disrupt the metabolic functions of the cells and eventually killing the fungi. In conclusion, our In-Out-Suicide system allows for more lethal genetically modified fungal pesticides to be developed because it provides them with a ability to clean up after itself, lowering their threat to the surrounding environment. 
</P></DIV><DIV class="fund"><B style="font-size:16px;">Our Killswitch Circuit:</B><P style="font-size:16px;">The killswitch circuit contains the hemolymph-induced promoter, <I>PMcl</I>, followed by the KillerRed gene and a fungal terminator, TtrpC. This sets the mass production of KillerRed proteins to the hemolymph infection phase of <I>M. anisopliae</I>’s infection cycle.</P></DIV><DIV class="fund"><H2 style="margin-top:20px; margin-bottom:10px;">Reference</H2><LI style="list-style-type:decimal;"><P style="font-size:16px;">Gisbert Zimmermann (2007): Review on safety of the entomopathogenic fungus Metarhizium anisopliae , Biocontrol Science and Technology, 17:9, 879-920. </P></LI><LI style="list-style-type:decimal;"><P style="font-size:16px;">Wang, C., &amp; St. Leger, R. J. (2006). A collagenous protective coat enables Metarhizium anisopliae to evade insect immune responses. Proceedings of the National Academy of Sciences of the United States of America, 103, 6647-6652.</P></LI><LI style="list-style-type:decimal;"><P style="font-size:16px;">2013 Carnegie_Mellon ;https://2013.igem.org/Team:Carnegie_Mellon/KillerRed</P></LI><LI style="list-style-type:decimal;"><P style="font-size:16px;">Pletnev, S., Gurskaya, N. G., Pletneva, N. V., Lukyanov, K. A., Chudakov, D. M., Martynov, V. I., … Pletnev, V. (2009). Structural Basis for Phototoxicity of the Genetically Encoded Photosensitizer KillerRed. The Journal of Biological Chemistry, 284(46), 32028–32039.  </P></LI><LI style="list-style-type:decimal;"><P style="font-size:16px;">Breitenbach, M., Weber, M., Rinnerthaler, M., Karl, T., &amp; Breitenbach-Koller, L. (2015). Oxidative Stress in Fungi: Its Function in Signal Transduction, Interaction with Plant Hosts, and Lignocellulose Degradation. Biomolecules,5(2), 318–342.   </P></LI><LI style="list-style-type:decimal;"><P style="font-size:16px;">Genetically-encoded photosensitizer KillerRed; http://evrogen.com/products/KillerRed/KillerRed_Detailed_description.shtml</P></LI></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>