<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aalto-Helsinki_Improve skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aalto-Helsinki/Improve</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Aalto-Helsinki</TITLE></P><DIV id="dropdown"><UL><LI class="active has-sub"><A class="lab" href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Laboratory</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Theory">Theoretical Background</A></LI><LI class="has-sub"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Experiments">Materials and Methods</A></LI><LI class="last"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Protocols">Protocols</A></LI></UL><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Results">Results and Discussion</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Future">Future Perspectives</A></LI><LI class="active has-sub"><A class="mod" href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Theory">Theoretical Background</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Setup">Simulation Setup</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Results">Results and Discussion</A></LI><LI class="active has-sub"><A class="hp" href="https://2017.igem.org/Team:Aalto-Helsinki/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Story">Project</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Engagement">Public Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/HP/Silver">Judging</A></LI><LI class="active has-sub"><A class="com" href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Community</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Composite_Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Improve">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Collaborations">Collaborations</A></LI><LI class="active has-sub"><A class="peo" href="https://2017.igem.org/Team:Aalto-Helsinki/Team">People</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Sponsors">Sponsors</A></LI></DIV><DIV id="color-block-com"><SPAN><DIV class="text1">COMMUNITY</DIV><DIV class="text2"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Overview</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Parts">Parts</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Composite_Part">Composite Part</A><A style="text-decoration: underline" href="https://2017.igem.org/Team:Aalto-Helsinki/Improve">Improved Part</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/InterLab">InterLab</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Collaborations">Collaborations</A></DIV></SPAN></DIV><DIV class="container"><DIV class="introtext"><H3 style="color: #006699">Improved Part</H3></DIV></DIV><DIV class="container"><DIV class="basictext"><H4><A href="http://parts.igem.org/Part:BBa_K2342009">BBa_K2342009</A> (His6x-Smt3-LL-37)</H4><P id="paragraph">
            Previously, team iGEM12_Trieste have used the part (BBa_K875009) to express LL-37 in <I>E. coli</I> as a part of sensing mechanism
            such that bacteria would produce LL-37 to terminate itself when sufficient conditions are met. However, in this
            form it wasn’t possible to use this part that encodes for antimicrobial peptide against other pathogens unless
            LL-37 is cloned to pathogen itself. This limits the possible applications of LL-37 as an antimicrobial agent. Therefore,
            we established a recombinant expression system for production of LL-37 antimicrobial peptide. We have improved
            the part by adding SUMO fusion tag, Smt3, such that it is first produced in inactive form and hence does not
            cause toxicity in host during expression. In order to ease the purification following recombinant expression,
            we further introduced a Histidine tag, His6x tag. In our project we succesfull expressed LL-37 in <I>E. coli</I> without
            killing it confirming that Smt3 tag fulfills its role as designed. We further proved that His6x tag facilitates
            purification of improved part with histidine tag affinity chromatography method. Therefore, we managed to obtain
            30 mg of pure LL-37 from 500 ml <I>E. coli</I> culture. Finally, we prove that it is possible to obtain LL-37 peptide alone
            by cleaving off His6x-Smt3 tag with commercially available and robust enzyme, Ulp1. In our project, LL-37 is used
            as a positive control for comparison due to its documented antimicrobial activity.
        </P><P id="paragraph">
            With our contribution to registry, other teams can benefit from our improved part (BBa_K2342009) and produce LL-37 in <I>E. coli</I>
            without killing the host, purify the peptide and use it for other applications.
        
         Here, we demonstrate in detail how we have improved and successfully used the part in our project.
        </P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Biology and system</H4><P id="paragraph">
                This part encodes for human antimicrobial peptide cathelicidin, LL-37 <A href="#refl1" name="ref1">[1]</A>. This part allows expression of LL-37 antimicrobial peptide in form of a fusion protein accompanied with His6x-Smt3 tag. The construct can be purified using with commercially available and widely used affinity chromatography columns designed for 6xHis tag. Smt3 is tag is used to keep antimicrobial peptide, LL-37 in inactive form by blocking its adhesion to phospholipid bilayer of the production host due to relatively large size of the tag. One major advantage of the fusion system used is that it facilitated easier detection of the peptide with a conventional method, SDS-PAGE. Also, SUMO tag is beneficial due to its effect on solubility of fusion peptide significantly easing purification step. After purification, to cleave off His6x-Smt3 tag, Ulp1 enzyme that is known for its robust and specific proteolytic activity against SUMO fusion proteins, is used to obtain free DCD-1L antimicrobial peptide. [<A name="ref2" href="#refl2">2</A>, <A name="ref3" href="#refl3">3</A>]
        </P><P id="paragraph"><I>Figure 1. Plasmid map of our composite part (Geneious version 10.1.3</I> [<A name="ref4" href="#refl4">4</A>, <A name="ref5" href="#refl5">5</A>]<I>).</I></P><P id="paragraph" style="word-wrap:break-word"><B>Promoter information</B>
            The pET28a(+) vector contains a T7<I>lac</I> promoter (TAATACGACTCACTATAGGGGAATTGTGAGCGGATAACAATTC) which consists of the T7 promoter
            and downstream of that there is the <I>lac</I> operator sequence. In addition, the vector contains the gene <I>lacI</I>, which encodes
            for the lac repressor (LacI) that binds to the <I>lac</I> operator. This promoter can be induced by isopropyl-β-D-thiogalactopyranoside
            (IPTG) [<A name="ref6" href="#refl6">6</A>].
        </P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Small scale production</H4><H5>Cultivations and Induction of protein expression</H5><P id="paragraph">
            For small scale production, 3-4 colonies from the transformation plates were inoculated (<A href="">the expression strain cells</A>) transformed with plasmids carrying
            the gene of interest) in 7 ml LB-kanamycin (50 μg/ml working concentration) and grew the cells at +37 °C until
            it reached the OD<SUB>600</SUB> value ~0,51. When finished growing the cells, the expression of the gene of interest was
            induced by adding a final concentration of 0,5 mM IPTG in the cultures and continued to grow at +37 °C shaking.
            Small scale purification was carried out using the Qiagen Ni-NTA spin columns.
        </P><H5>SDS-Page of protein purification</H5><P id="paragraph">
            Following small scale protein expression Qiagen Ni-NTA spin columns are used for purification. From different steps of purification,
            such as washing and elution, samples are loaded on SDS-PAGE along with samples collected from the small scale
            expression culture.
        </P><P id="paragraph">          SDS-PAGE image (Figure 2) shows that after induction of expression with IPTG, the band gets corresponding to gene of interest
            gets thicker gradually, from initial (well 1) to 4 hours (well 3). It proves that the T7 promoter system works
            as intended, without promoter leakage. Although, there is a strong band in both lysate (well 4) and pellet (well
            5) suggesting that protein of interest was partially precipitated. Soluble portion of the protein in lysate
            is successfully purified and can be seen in well 7 and 8.
        </P><P id="paragraph"><I>Figure 2. SDS-Page image from small scale expression of His6x-Smt3-LL37 (17.813 kDa): M. Marker (PageRuler<SUP>TM</SUP>
                Prestained Protein Ladder, Thermo Fisher), 1. Non-induced, 2. 2 hours of induction, 3. 4 hours of induction,
                4. Lysate, 5. Pellet, 6. Flow through, 7. Eluate 1, 8. Eluate 2.</I></P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Large Scale Production (Half liter batch)</H4><H5>Cultivations and Induction</H5><P id="paragraph">
                Large scale protein expression was started by inoculating 3-10 single colonies (of the expression strain cells transformed with plasmids carrying the gene of interest) in 25mL of LB medium with the 50ug/ml Kanamycin and Incubated at +30°C with shaking overnight. The next day we prewarmed 500 mL of LB medium to +37°C in a 2L Erlenmeyer flask and Added 50 ug/ml Kanamycin. Then inoculated 3-5 mL of the overnight grown preculture in 500 mL prewarmed LB. Flask was incubated at +37°C with shaking until OD<SUB>600</SUB> value reached 0.6.
        </P><P id="paragraph">
                Protein expression was then induced with a final concentration of 0.5mM IPTG and incubated the culture at +37°C for 4 hours. 
        </P><H5>Cell lysis and purification</H5><P id="paragraph">
                The 35 ml sample with harvested cells was then lysed using Emulsiflex machine and was injected into the ÄKTA machine for protein purification using His tag affinity method was done the Fractions were collected (Figure 3). We Selected elution fractions with the desired protein from the graph (Figure 4), and run flow-through and elution fractions on SDS-PAGE. Fractions that contain the desired protein were pooled, frozen in liquid nitrogen and stored at -20°C.
        </P><P id="paragraph"><I>Figure 3. Sample Image of the plate with all the eluates from the ÄKTA machine after purification.</I></P><P id="paragraph"><I>Figure 4. Curve obtained for 6xHis-Smt3-LL-37 after purification using the ÄKTA machine. The blue peak represents the eluted proteins.</I></P><P id="paragraph">
                The wells (D1, A2, B2…….. A5) were selected from the eluate plate based on  the peaks from the graph similar to one shown in (Figure 3) and analysed by running SDS PAGE for desired protein. 
        </P><H5>SDS-PAGE of protein purification</H5><P id="paragraph">
                The molecular weight of 6xHis-Smt3-LL-37 is 18.2kDa. After running the selected fractions on the gel, it could be seen that eluates D2, A3, B3, C3 had our desired protein. Hence, these were collected for further protein concentration and buffer exchange step and rest eluates were discarded.
        </P><P id="paragraph"><I>Figure 5. SDS-PAGE Gel for His6x-Smt3-LL-37: 1. Eluate D1, 2. Eluate A2, 3. Eluate B2, 4. Eluate C2, 5. Eluate
                D2, 6. Eluate A3, 7. Eluate B3, 8. Eluate C3, 9. Eluate D3, 10. Eluate A4, 11. Eluate B4, 12. Eluate C4,
                13. Eluate D4, 14. Eluate A5.</I></P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Concentration process and buffer exchange</H4><P id="paragraph">
                After pooling of the eluates it is important to concentrate the peptide samples. The protein is currently diluted in 20 ml of buffer. So the samples were concentrated using Sartorious VIVA SPIN 20 ultrafilter (Membrane: 5000 MWCO PES) and exchanged the buffer with 10mM Napi.
        </P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Ulp1 enzyme digestion and SDS PAGE</H4><P id="paragraph">
                For digesting H6-Smt3-containing DCD1l with Ulp1 to obtain free DCD1L 0.5μL of Ulp1 protease was added to 50μL of the purified protein (concentrated in e.g. NaPi buffer. The suitable digestion time was determined by incubating the protein with Ulp1 for different time points and running them on an SDS-PAGE. We can see that 10 mins of Incubation at RT is enough.
        </P><P id="paragraph"><I>Figure 6. Digestion of 6xHis-Smt3-LL37 with Ulp1. Samples on the gel are: 1. Ulp1 (control), 2. 6xHis-Smt3-DCD-1L (undigested), 3. 6xHis-Smt3-DCD-1L + Ulp1 (digested for 0 minutes), 4. 6xHis-Smt3-DCD-1L + Ulp1 (digested for 5 minutes), 5. 6xHis-Smt3-DCD-1L + Ulp1 (digested for 30 minutes), 6. 6xHis-Smt3-LL37 + Ulp1 (digested for 0 minutes), 7. 6xHis-Smt3-LL37 + Ulp1 (digested for 5 minutes), 8. 6xHis-Smt3-LL37 + Ulp1 (digested for 30 minutes).</I></P><P id="paragraph">
                Undigested samples are highlighted with a white box: 6xHis-Smt3-LL-37 18.2kDa. The LL-37 peptide (4.71kDa after digestion) are very low, it is challenging to observe the corresponding bands on the SDS-PAGE gel. Instead, the band corresponding to the 6xHis-Smt3 part can be clearly observed from the gel (Red Box). The image further illustrates that the activity of Ulp1 is very high, because directly after mixing the protein sample with Ulp1 (0 minutes digested), the digested 6xHis-Smt3 tag can be clearly distinguished from the undigested protein.
        </P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Antimicrobial activity</H4><P id="paragraph">
                Bacterial cultures were grown until the OD reached 0.05 with corresponding 1.4*10^8 CFU/ml. The samples were incubated at 37°C, shaking for 40 mins with the respective antimicrobial peptides Nisin, DCD1L and LL37 (concentration used: 100 ug/ml). After incubation, the OD of the cells were measured. It was observed that the OD values in all the three samples with LL37, Nisin and produced DCD1L dropped after 40 minutes indicating antimicrobial property of the peptides. 
        </P><P id="paragraph"><I>Figure 7. % of cells killed at t = 40 min, when incubated with DCD-1L, LL-37, nisin and chloramphenicol.</I></P><P id="paragraph">
                Undigested samples are highlighted with a white box: 6xHis-Smt3-LL-37 18.2kDa. The LL-37 peptide (4.71kDa after digestion) are very low, it is challenging to observe the corresponding bands on the SDS-PAGE gel. Instead, the band corresponding to the 6xHis-Smt3 part can be clearly observed from the gel (Red Box). The image further illustrates that the activity of Ulp1 is very high, because directly after mixing the protein sample with Ulp1 (0 minutes digested), the digested 6xHis-Smt3 tag can be clearly distinguished from the undigested protein.
        </P></DIV></DIV><DIV class="container"><DIV class="basictext"><H3>References</H3><P id="paragraph"><A name="refl1" href="#ref1">[1]</A> Turner, J., Cho, Y., Dinh, N. N., Waring, A. J., &amp; Lehrer, R. I. (1998). Activities of LL-37, a cathelin-associated antimicrobial peptide of human neutrophils. <I>Antimicrobial agents and chemotherapy, 42</I>(9), 2206-2214.
                <A name="refl2" href="#ref2">[2]</A> Malakhov, M. P., Mattern, M. R., Malakhova, O. A., Drinker, M., Weeks, S. D., &amp; Butt, T. R. (2004). SUMO fusions and SUMO-specific protease for efficient expression and purification of proteins. <I>Journal of structural and functional genomics, 5</I>(1), 75-86.
                <A name="refl3" href="#ref3">[3]</A> Marblestone, J. G., Edavettal, S. C., Lim, Y., Lim, P., Zuo, X., &amp; Butt, T. R. (2006). Comparison of SUMO fusion technology with traditional gene fusion systems: Enhanced expression and solubility with SUMO. <I>Protein Science : A Publication of the Protein Society, 15</I>(1), 182–189.
                <A name="refl4" href="#ref4">[4]</A><I> Geneious molecular biology and NGS analysis tools.</I> Accessible at: <A href="https://www.geneious.com/">[here]</A>.
                <A name="refl5" href="#ref5">[5]</A> Kearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., Buxton, S., Cooper, A., Markowitz, S., Duran, C., Thierer, T., Ashton, B., Mentjies, P., &amp; Drummond, A. (2012). Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data. 
                <I>Bioinformatics</I>, 28(12), 1647-1649.
                <A name="refl6" href="#ref6">[6]</A><I> Novagen pET System Manual.</I> Accessible at: <A href="https://research.fhcrc.org/content/dam/stripe/hahn/methods/biochem/pet.pdf">[here]</A>.
            </P></DIV></DIV><FOOTER><DIV class="footer-social"><DIV class="texts"><UL class="pe-social"><A href="https://2017.igem.org/Team:Aalto-Helsinki">HOME</A><A href="http://www.aalto.fi/en/">AALTO UNIVERSITY</A><A href="https://www.helsinki.fi/en">UNIVERSITY OF HELSINKI</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Map">SITE MAP</A></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>