Team:Aalto-Helsinki/Results
Aalto-Helsinki
Laboratory
Overview
Theoretical Background
Materials and Methods
Protocols
Results and Discussion
Future Perspectives
Modeling
Overview
Theoretical Background
Simulation Setup
Results and Discussion
Human Practices
Overview
Project
Public Outreach
Judging
Community
Overview
Parts
Composite Part
Improved Part
InterLab
Collaborations
People
Team Members
Advisors
Attributions
Sponsors
LABORATORY
Overview
Theoretical Background
Materials and Methods
Protocols
Results and Discussion
Future Perspectives
Results and Discussion
PCR check-up (for Smt3)
We first performed PCR amplification of the Smt3 gene fragment that was generously provided by Sesilija Aranko             (from Aalto University) for use in our project.
Figure 1. Smt3 PCR product. The PCR amplified Smt3 fragment is seen in the image as red. The size of the fragment                 is 324 bp.
We chose the plasmid pET28a(+) as our vector since it offers the advantage of having a His6x tag in the plasmid backbone, thus             enabling affinity purification of the produced proteins.
Figure 2. pET28a(+) vector. The vector carries an N-terminal and C-terminal His6x tag in the plasmid backbone                 (Geneious version 10.1.3 [ 1 , 2 ]). We introduced a stop codon in our constructs prior to the C-terminal His-tag, and thus used the N-terminal                 His-tag for purification.
Promoter information
The pET28a(+) vector contains a T7lac promoter (TAATACGACTCACTATAGGGGAATTGTGAGCGGATAACAATTC) which consists of the T7 promoter                 and downstream of that the lac operator sequence. In addition, the vector contains the gene lacI, which encodes                 for the lac repressor (LacI) that binds to the lac operator. This promoter can be induced by isopropyl-β-D-thiogalactopyranoside                 (IPTG) [ 3 ].
Cloning
We first cloned the Smt3 fragment into the pET28a(+) plasmid. After cloning and plasmid isolation, we performed a check-up digestion             with the same restriction enzymes that were used in cloning (NdeI, XhoI) to confirm that the plasmids carry the             right insert.
Figure 3. Check-up digestion of the pET28a(+)-Smt3 plasmid. The Smt3 insert (310 bp) is outlined with a red circle                 in the image. The pET28a(+) vector restricted by NdeI and XhoI enzymes is 5,290 bp.
Once we had the plasmid with Smt3, our next step was to insert the DCD-1L fragment into the plasmid. After cloning and plasmid             isolation, we performed a check-up digestion to confirm that the desired insert had been ligated in the vector.
Figure 4. Plasmid map of Smt3-DCD-1L (Geneious version 10.1.3 [ 1 , 2 ]).
DCD-1L being a short gene (150 bp), we decided to use enzymes NdeI and XhoI in the check-up digestion step. Thus, we screened             for a plasmid containing Smt3-DCD-1L insert (446 bp).
Figure 5. Check-up digestion of plasmids pET28a(+)-Smt3-DCD-1L (samples 1-4), alongside with pET28a(+)-Smt3 (sample                 5) which served as a negative control.
The gel image (Figure 5) shows that samples number 1, 2 and 4 contained the desired Smt3-DCD-1L insert (446 bp) circled with             red. The Smt3 insert (310 bp) in the negative control is circled with black, thus the insert in sample no. 3 seems             to correspond to Smt3.
Next, we cloned our other fragments into the pET28a(+)-Smt3 plasmid.
Figure 6. Plasmid map of Smt3-DCD-1L-22 aa linker-CBM (Geneious version 10.1.3 [ 1 , 2 ]).
Figure 7. Plasmid map of Smt3-CBM-22 aa linker-DCD-1L (Geneious version 10.1.3 [ 1 , 2 ]).
After cloning and plasmid isolation, we performed check-up digestion to confirm that the desired insert had been ligated             into the vector.
Figure 8. Check-up digestion of the following plasmids:
samples 1-4: pET28a(+)-Smt3-DCD-1L-10 aa linker-CBM (a-d),
samples 5-8: pET28a(+)-Smt3-DCD-1L-22 aa linker-CBM (a-d),
samples 9-12: pET28a(+)-Smt3-CBM-10 aa linker-DCD-1L (a-d),
samples 13-16: pET28a(+)-Smt3-CBM-22 aa linker-DCD-1L (a-d),
samples 17-18: pET28a(+)-Smt3-SSL-23 (a-b),
sample 19: pET28a(+)-Smt3 (negative control).
The red circles indicate inserts that were of the correct size: Smt3-DCD-1L-22 aa linker-CBM 983 bp (samples 5-7), Smt3-CBM-10             aa linker-DCD-1L 947 bp (samples 10, 12) and Smt3-CBM-22 aa linker-DCD-1L 983 bp (sample 13). None of the pET28a(+)-Smt3-DCD-1L-10             aa linker-CBM plasmids carried the correct insert (947 bp), nor did the pET28a(+)-Smt3-SSL-23 plasmids (371 bp).             Due to the long run time of the electrophoresis, the 310 bp Smt3 insert of the negative control is not clearly             visible on the gel.
Figure 9. Plasmid map of our composite part His6x-Smt3-LL-37 (Geneious version 10.1.3 [ 1 , 2 ]).
Figure 10. Check-up digestion of the following plasmids:
samples 1-2: pET28a(+)-Smt3-DCD-1L-10 aa linker-CBM (a-b),
samples 3-4: pET28a(+)-Smt3-LL-37 (a-b), samples 5-6: pET28a(+)-Smt3-SSL-23 (a-b),
sample 7: pET28a(+)-Smt3 (negative control).
The red circles indicate inserts that were of the correct size: Smt3-LL-37 419 bp (samples 3-4). The Smt3 insert (310 bp) in             the negative control is circled with white, thus the other plasmids seem to correspond to the negative control.             None of the pET28a(+)-Smt3-DCD-1L-10 aa linker-CBM plasmids carried the correct insert (947 bp), nor did the pET28a(+)-Smt3-SSL-23             plasmids (371 bp). As such, we did not continue to further steps with these two constructs and excluded them from our experiments.
Sequencing results as aligned graphs
To further confirm that the sequences of the cloned fragments were correct, we sent them for sequencing. Since our university             did not have a sequencing facility, we sent them to Eurofins Genomics in Germany. We used the primer T7 term             (CTA GTT ATT GCT CAG CGG T) from the available sequencing primers at Eurofins Genomics.
As observed from the sequence alignments, our part sequences were correct.
1. pET28a(+)-Smt3
Figure 11. Full alignment of the sequencing result with the cloned part in the biobrick backbone pET28a(+)-His6x-Smt3                 (Geneious version 10.1 [ 1 ]).
Figure 12. Zoomed up view of full alignment of the sequencing result with the cloned part the in biobrick backbone                 pET28a(+)-His6x-Smt3 (Geneious version 10.1 [ 1 ]).
2. pET28a(+)-Smt3-DCD-1L
We took sample no. 1 from the check-up digestion (Figure 5) for sequencing.
Figure 13. Full alignment of the sequencing result with the cloned part in the biobrick backbone pET28a(+)-His6x-Smt3-DCD-1L                 (Geneious version 10.1 [ 1 ]).
Figure 14. Full alignment of the sequencing result with the cloned part in biobrick the backbone pET28a(+)-His6x-Smt3-DCD-1L                 (Geneious version 10.1 [ 1 ]).
3. pET28a(+)-Smt3-DCD-1L-22-CBM
We took sample no. 7 from the check-up digestion (Figure 8) for sequencing.
The construct is long (1,029 bp), so we got only part of it sequenced due to using only one sequencing primer.
Figure 15. Partial alignment of the sequencing result with the cloned part in the biobrick backbone pET28a(+)-His6x-Smt3-DCD-1L-22                 aa linker-CBM (Geneious version 10.1 [ 1 ]).
NOTE: There were some unknown bases (N) in the end of the sequencing result (5’-end) because the quality             of sequencing decreases towards the end of the run. Hence, we looked into the chromatogram in order to verify             the results and some on the unknown N’s had peaks corresponding to the correct bases.
4. pET28a(+)-Smt3-CBM-22-DCD-1L
We took sample no. 13 from the check-up digestion (Figure 8) for sequencing. The construct is long (1029 bp), so we got only             part of it sequenced due to using only one sequencing primer.
Figure 16. Full alignment of the sequencing result with the cloned part in the biobrick backbone pET28a(+)-His6x-Smt3-CBM-22                 aa linker-DCD-1L (Geneious version 10.1 [ 1 ]).
As with the pET28a(+)-Smt3-DCD-1L-22 aa linker-CBM plasmid, there are some unknown bases (N) in the end of the sequencing             result (5’-end) because the quality of sequencing decreases towards the end of the run. Hence we looked into             the chromatogram in order to verify the results and some on the unknown N’s had peaks corresponding to the correct bases.
5. pET28a(+)-Smt3-LL-37
We took sample no. 3 from the check up digestion (Figure 10) for sequencing.
Figure 17. Full alignment of the sequencing result with the cloned part in the biobrick backbone pET28a(+)-His6x-Smt3-LL-37                 (Geneious version 10.1 [ 1 ]).
Figure 18. Zoomed in view of full alignment of the sequencing result with the cloned part in the biobrick backbone                 pET28a(+)-His6x-Smt3-LL-37 (Geneious version 10.1 [ 1 ]).
As observed from the sequence alignments, our part sequences were correct. Once the cloning of our desired parts was complete             and verified by sequencing, we went for small scale protein expression and purification.
Small scale protein expression and purification
Following small scale protein expression, Qiagen Ni-NTA spin columns are used for purification. From different steps of purification,             such as washing and elution, samples were loaded into SDS-PAGE along with samples collected from the small scale             expression culture.
The production of our antimicrobial peptide had not interfered with the bacterial growth.
His6x-Smt3-DCD-1L
For His6x-Smt3-DCD-1L, after induction of protein expression, the OD600 value of the culture was measured to observe the effect             of the protein of interest on cell growth due to its antimicrobial activity and therefore possible toxicity to expression             host. However, the cells were observed to be growing normally. The production of our antimicrobial peptide had not             interfered with the bacterial growth (Figure 19). Hence our implementation of the Smt3 fusion protein system for producing the antimicrobial             peptides in E. coli was successful.
Figure 19. OD values of the bacterial cells before and after protein expression His6x-Smt3-DCD-1L with respect                 to time.
The expression of desired protein was observed through SDS-PAGE analysis.
Figure 20. SDS-PAGE Gel for His6x-Smt3 and Ulp1. 1: Ladder (PageRulerTM Prestained Protein Ladder, Thermo Fisher), 2: Smt3 non-induced, 3: Smt3 2h after induction, 4: Smt3                 4h after induction, 5: Smt3 lysate, 6: Smt3 pellet, 7: Smt3 flow-through 1, 8: Smt3 flow-through 2.1 9: Smt3                 eluate 1, 10: Smt3 eluate 2, 11: Ulp1 non-induced, 12: Ulp1 2h after induction, 13: Ulp1 4h after induction,                 14: Ulp1 lysate, 15: Ulp1 pellet.
Figure 21. SDS-PAGE Gel for His6x-Smt3-DCD-1L and Ulp1: 1. Ladder (PageRulerTM Prestained Protein Ladder, Thermo Fisher), 2. Ulp1 flow-through 1, 3. Ulp1 flow-through 2.1, 4.                 Ulp1 eluate 1, 5. Ulp1 eluate 2, 6. Smt3-DCD-1L non-induced, 7. Smt3-DCD-1L 2h after induction, 8. Smt3-DCD-1L                 4h after induction, 9. Smt3-DCD-1L lysate, 10. Smt3-DCD-1L pellet, 11. Smt3-DCD-1L flow-through 1, 12. Smt3-DCD-1L                 flow-through 2.1, 13. Smt3-DCD-1L eluate 1, 14. Smt3-DCD-1L eluate 2.
His6x-Smt3-DCD-22 aa linker-CBM
SDS-PAGE image (Figure 22) shows that after induction of protein expression with IPTG, the band corresponding to the gene of interest             becomes gradually more prominent and thicker, from the initial induction (well 1) to 4 hours post-induction (well 3). This proves that the T7 promoter system works             as intended, without promoter leakage. However, there is a strong band in both the lysate (well 4) and the pellet (well             5) suggesting that protein of interest was partially precipitated. The soluble portion of the protein seen in the lysate             was successfully purified and can be seen in wells 7 and 8.
Figure 22. SDS-Page image from small scale expression of His6x-Smt3-DCD1L-22 Linker-CBM construct (36.910 kDa):                 M. Marker (PageRulerTM Prestained Protein Ladder, Thermo Fisher), 1. Non-induced, 2. 2 hours post-induction, 3. 4 hours post-induction,      4. Lysate, 5. Pellet, 6. Flow through, 7. Eluate 1, 8. Eluate 2.
His6x-Smt3-CBM-22 aa linker-DCD-1L
SDS-PAGE image (Figure 23) shows that after induction of protein expression with IPTG, the band gets corresponding to gene of interest             gets thicker and more prominent, from initial induction (well 1) to 4 hours post-induction (well 3). This proves that the T7 promoter system works as intended,             without promoter leakage. There are bands in both the pellet (well 3) and the lysate (well 4) suggesting that protein             of interest was partially precipitated. The soluble portion of the protein in the lysate is not visible in the elution samples             in wells 6 and 7, most likely due to its low concentration. See large scale production bellow.
Figure 23. SDS-Page image from small scale expression of His6x-Smt3-CBM-22 Linker-DCD1L construct (36.866 kDa):                 M. Marker (PageRulerTM Prestained Protein Ladder, Thermo Fisher), 1. Non-induced, 2. 4 hours post-induction, 3. Pellet, 4.                 Lysate 5. Flow through, 6. Eluate 1, 7. Eluate 2.
His6x-Smt3-LL-37
Following small scale protein expression Qiagen Ni-NTA spin columns were used for purification. From different steps of purification,             such as washing and elution, samples were loaded into SDS-PAGE along with samples collected from the small scale             expression culture. SDS-PAGE image (Figure 24) shows that after induction of protein expression with IPTG, the band             corresponding to the gene of interest gets gradually thicker, from the initial induction (well 1) to 4 hours post-induction (well 3). This proves             that the T7 promoter system works as intended, without promoter leakage. However, there is a strong band in             both the lysate (well 4) and the pellet (well 5) suggesting that the protein of interest was partially precipitated. Soluble             portion of the protein seen in the lysate was successfully purified and can be seen in wells 7 and 8.
Figure 24. SDS-Page image from small scale expression of His6x-Smt3-LL37 (17.813 kDa): M. Marker (PageRulerTM Prestained Protein Ladder, Thermo Fisher), 1. Non-induced, 2. 2 hours post-induction, 3. 4 hours post-induction,                 4. Lysate, 5. Pellet, 6. Flow through, 7. Eluate 1, 8. Eluate 2.
Large scale protein expression and purification
Approximately 35 ml sample is collected from Emulsiflex machine after lysis and was injected to the ÄKTA Machine for protein purification using the His-tag affinity method. During purification, elution fractions were collected on a well plate             (Figure 25). Elution fractions containing proteins were selected based on the graph and further analysed using SDS             PAGE (Figure 26).
His6x-Smt3-DCD-1L
Figure 25. Image of the plate with all the eluates from the ÄKTA machine after purification of His6x-Smt3-DCD-1L.                 The highlighted wells (D1, A2, B2…….. D4) were selected based on the peaks from the graph and analysed by                 running SDS PAGE for the desired protein.
Figure 26. Curve obtained for His6x-Smt3-DCD-1L after purification using the ÄKTA protein purification machine.                 The blue peak represents the eluted proteins.
Wells collected (D1 to D4) were selected based on the peaks from the graph and analysed by running SDS-PAGE for desired protein.
Figure 27. SDS-PAGE Gel for His6x-Smt3-DCD-1L: 1. Eluate D1, 2. Eluate A2, 3. Eluate B2, 4. Eluate C2, 5. Eluate                 D2, 6. Eluate A3, 7. Eluate B3, 8. Eluate C3, 9. Eluate D3.
The molecular weight of His6x-Smt3-DCD-1L-22 aa linker-CBM is 18.3 kDa. After running the selected fractions in the gel, it             could be seen that eluates D1, D1, A2, B2, C2, D2, A3, B3, C3 and D3 contained our desired protein. Hence these were             collected for further protein concentration and buffer exchange step, while the rest of the eluates were discarded.
His6x-Smt3-LL-37
Figure 28. Curve obtained for DCD-22 aa linker-CBM after purification using the ÄKTA machine. The blue peak represents                 the eluted proteins.
Wells (A2 to B5) were selected based on the peaks from the graph and analysed by running SDS-PAGE for the desired protein.
Figure 29. SDS-PAGE Gel for His6x-Smt3-DCD-22 aa linker-CBM: M. PageRulerTM Prestained Protein Ladder, Thermo                 Fisher; 1. Eluate A2, 2. Eluate B2, 3. Eluate C2, 4. Eluate D2, 5. Eluate A3, 6. Eluate B3, 7. Eluate C3,                 8. Eluate D3, 9. Eluate A4, 10. Eluate B4, 11. Eluate C4, 12. Eluate D4, 13. Eluate A5, 14. Eluate B5.
The molecular weight of His6x-DCD-1L-22 aa linker-CBM is 37.2 kDa. After running the selected fractions in the gel, it could             be seen that eluates A3, B3, C3, D3, A4, B4 and C4 contained our desired protein. Hence these were collected for further             protein concentration and buffer exchange step, while the rest of the eluates were discarded.
His6x-Smt3-CBM-22 aa linker-DCD-1L
Figure 31. SDS-PAGE Gel for His6x-Smt3-CBM-22 aa linker-DCD-1L: 1. Eluate A2, 2. Eluate B2, 3. Eluate C2, 4.                 Eluate D2, 5. Eluate A3, 6. Eluate B3, 7. Eluate C3, 8. Eluate D3, 9. Eluate A4, 10. Eluate B4, 11. Eluate                 C4.
The molecular weight of His6x-Smt3-CBM-22 aa linker-DCD-1 is 37.2 kDa. After running the selected fractions in the gel, it             can be seen that eluates in wells 2, 3, 9 and 10 contained our desired protein. Hence these were collected for further             protein concentration and buffer exchange step, while the rest of the eluates were discarded.
His6x-Smt3-LL-37
Figure 32. Curve obtained for His6x-Smt3-LL-37 after purification using the ÄKTA machine. The blue peak represents                 the eluted proteins.
The wells (D1 to A5) were selected from the eluate plate based on the peaks from the graph similar to the one shown in (Figure             25) and analysed by running SDS-PAGE for the desired protein.
Figure 33. SDS-PAGE Gel for His6x-Smt3-LL-37: 1. Eluate D1, 2. Eluate A2, 3. Eluate B2, 4. Eluate C2, 5. Eluate                 D2, 6. Eluate A3, 7. Eluate B3, 8. Eluate C3, 9. Eluate D3, 10. Eluate A4, 11. Eluate B4, 12. Eluate C4,                 13. Eluate D4, 14. Eluate A5.
The molecular weight of His6x-Smt3-LL-37 is 18.2 kDa. After running the selected fractions in the gel, it can be seen that             eluates D2, A3, B3, C3 contained the desired protein. Hence, these were collected for further protein concentration             and buffer exchange step, while the rest of the eluates were discarded.
For all the constructs, following protein purification, protein concentration columns were used both to exchange the buffer from             the elution buffer (buffer B) to a 10 mM Napi buffer and to increase the protein concentration. (Table 1)
Table 1. Table of final yields, after the concentration step.
Construct name
Concentration [mg/ml]
Volume [ml]
Yield [mg]
DCD-1L
2.83
9
25.47
D22C
2.33
10
23.30
C22D
3.37
11
37.07
LL-37
3.00
10
30.00
Ulp1 enzyme digestion and SDS PAGE
For digesting His6x-Smt3-DCD-1L with Ulp1 to obtain free DCD-1L, 0.5 μl of Ulp1 protease was added per 50 μl of the purified             protein (concentrated in e.g. NaPi buffer). The suitable digestion time was determined by incubating the protein             with Ulp1 for different time intervals and running them in an SDS-PAGE gel.
In our project, we were unfortunately unable to clone and produce Ulp1 after multiple attempts. Hence we ended up borrowing the enzyme from Sesilija             Aranko (Aalto University) who generously provided us with the Ulp1 protease. We incubated our produced proteins with Ulp1             protease for time intervals of 0 min, 5 min and 30 min. From the gel images, we observed that 5 min incubation already resulted in the digestion of peptides.
Figure 34. Digestion of His6x-Smt3-DCD-1L, His6x-Smt3-LL-37 and His6x-Smt3-DCD-1L-22 aa linker-CBM with Ulp1.                 Samples on the gel are:
1. Ulp1 (control),
2. His6x-Smt3-DCD-1L (undigested),
3. His6x-Smt3-DCD-1L + Ulp1 (digested for 0 minutes),
4. His6x-Smt3-DCD-1L + Ulp1 (digested for 5 minutes),
5. His6x-Smt3-DCD-1L + Ulp1 (digested for 30 minutes),
6. His6x-Smt3-LL-37 (undigested),
7. His6x-Smt3-LL-37 + Ulp1 (digested for 0 minutes),
8. His6x-Smt3-LL-37 + Ulp1 (digested for 5 minutes),
9. His6x-Smt3-LL-37 + Ulp1 (digested for 30 minutes),
10. His6x-Smt3-DCD-1L-22 aa linker-CBM (undigested),
11. His6x-Smt3-DCD-1L-22 aa linker-CBM + Ulp1 (digested for 0 minutes),
12. His6x-Smt3-DCD-1L-22 aa linker-CBM + Ulp1 (digested for 5 minutes),
13. His6x-Smt3-DCD-1L-22 aa linker-CBM + Ulp1 (digested for 30 minutes).
Undigested samples are highlighted with white boxes: His6x-Smt3-DCD-1L 18.3kDa, His6x-Smt3-LL-37 18.2kDa, His6x-Smt3-DCD1L-22             aa linker-CBM 37.3kDa. The desired peptides/proteins after digestion are highlighted with red boxes: DCD-1L 4.82 kDa,             DCD-1L-22 aa linker-CBM 23.8 kDa. The digested N-terminus containing the His6x-tag and the Smt3-tag (13.5 kDa)             is highlighted with black.
Since the mass of DCD-1L peptide and the LL-37 peptide (4.71 kDa after digestion) are very low, it is challenging to observe             the corresponding bands on the SDS-PAGE gel. Instead, the band corresponding to the His6x-Smt3 part can be clearly             observed from the gel. The image further illustrates that the activity of Ulp1 is very high, as directly             after mixing the protein sample with Ulp1 (0 minutes digested), the digested His6x-Smt3 tag can be clearly distinguished             from the undigested protein.
Figure 35. Digestion of His6x-Smt3-CBM-22 aa linker-DCD-1L with Ulp1. Samples on the gel are:
1. His6x-Smt3-CBM-22 aa linker-DCD-1L (undigested)
2. His6x-Smt3-CBM-22 aa linker-DCD-1L + Ulp1 (digested for 0 minutes)
3. His6x-Smt3-CBM-22 aa linker-DCD-1L + Ulp1 (digested for 5 minutes)
4. His6x-Smt3-CBM-22 aa linker-DCD-1L + Ulp1 (digested for 30 minutes).
Undigested samples are highlighted with white boxes: His6x-Smt3-CBM-22 aa linker-DCD-1L 37.2 kDa. The desired proteins after             digestion are highlighted with red boxes: CBM-22 aa linker-DCD-1L 23.8 kDa. The digested N-terminus containing             the His6x-tag and the Smt3-tag (13.5 kDa) is highlighted with black.
The image further illustrates that the activity of Ulp1 is very high, because directly after mixing the protein sample with             Ulp1 (0 minutes digested), the digested His6x-Smt3 tag can be clearly distinguished from the undigested protein.
MALDI-TOF-TOF Mass spectrometry
The cleaved peptides were further analysed with MALDI-TOF-TOF to verify that our final product was the right protein.
DCD-1L
The sample after Ulp1 digestion was analysed using mass spectrometry to identify DCD-1L. We used matrix-assisted laser desorption/ionization-time             of flight/time of flight (MALDI-TOF-TOF) mass spectrometer (UltrafleXtreamTM Bruker, Aalto department of biotechnology             and chemical technology facilities, Espoo, Finland) equipped with a 200-Hz smart-beam 1 laser (337 nm, 4 ns pulse)             to identify the masses of proteins/peptides. Since, the result is mass/charge ratio, when the charge is 2 we get             half the mass as the result, which is why we observe 2410.626 in addition to the correct mass peak in the mass spectrum for DCD-1L.
Figure 36. Image of Mass spectrometry spectrum for DCD-1L. Theoretical mass: 4,820 Da and result obtained: 4820.43                 Da.
DCD-22 aa linker-CBM
The sample after Ulp1 digestion was analysed using MALDI-TOF-TOF to identify DCD-1L-22 aa linker-CBM.
Figure 37. Image of Mass spectrometry spectrum for DCD-1L-22 aa linker-CBM. Theoretical mass: 23,800 Da and result                 obtained: 23,724 Da.
CBM-22 aa linker-DCD-1L
The sample after Ulp1 digestion was analysed using mass spec to identify CBM-22 aa linker-DCD-1L.
Figure 38. Image of Mass spectrometry spectrum for CBM-22 aa linker-DCD-1L Theoretical mass: 23800 Da and Result                 obtained: 23666 Da.
Hence, after production, purification and validation of our peptides, we went for antimicrobial activity testing and cellulose             nanofiber (CNF) binding assays to see if our peptides actually work.
Antimicrobial activity
Bacterial cultures (E. coli DH5 alpha) were grown until the OD reached 0.05 with corresponding 1.4*10^8 CFU/ml. The samples             were incubated at 37 °C with shaking for 40 min with the respective antimicrobial peptides Nisin, DCD-1L and LL-37             (concentration used: 100 μg/ml). After incubation, the OD of the cells were measured. It was observed that the             OD values in all the three samples with LL-37, Nisin and our produced DCD-1L dropped after 40 min indicating antimicrobial             property of the peptides.
Figure 39. OD values of the bacterial samples (E. coli DH5 alpha) incubated with DCD-1L, LL-37 and Nisin with                 respect to time.
The initial OD at 600 was observed to be 0.05 for the cells. The samples were incubated with different antimicrobial peptides             (conc. 100 μg/ml) for 40 mins. The reduction in the OD can be observed in Figure 39, with the OD value dropping             to 0.0147 in sample with DCD-1L and to 0.0202 in sample with LL-37. Nisin was used as a control peptide since it is             a well known and widely used antimicrobial peptide in the industry. From literature, the minimum inhibitory concentration             for DCD-1L was reported to be 10 μg/ml. However, in our experiments, we only observed antimicrobial activity with a high concentration             of 100 μg/ml this might be due to the salt concentration in the assay. We know from literature and our modelling             results that DCD-1L shows variance in activity at different salt concentrations.
Figure 40. Image representing data in terms of the percentage of cells (E. coli DH5 alpha) killed after 40 mins of incubation                 with Chloramphenicol, Nisin, DCD-1L and LL-37.
Form Figure 40, we can see that the antibiotic used was the most effective with 100 % of the cells killed after 40 min. Our produced DCD-1L             peptide killed 70 % of the cells, while LL-37 killed 57.58 % of cells. As such, the activity of LL-37 was lower than that of our DCD-1L peptide.             One possible reason could be that since we produced LL-37 as a recombinant protein with Smt3-tag and cleaved             the tag with Ulp1 protease (which cleaves between amino acids G and S), the N-terminus of our LL-37 is (theoretically)             SMLL. Thus, there are 2 additional amino acids in the N-terminus, which may affect the functional properties             of the antimicrobial peptide.
We carried out further antimicrobial activity tests with C256 E. coli cells and did not observe any antimicrobial activity             of DCD-1L, LL-37, CBM-22 aa linker-DCD-1L and DCD-1L-22 aa linker-CBM on these cells with 100 μg/ml effective concentration.             Also at 100 μg/ml concentration of CBM-22 aa linker-DCD-1L and DCD-1L-22 aa linker-CBM we did not observe any antimicrobial             activity. It may be that a change in buffer conditions or an increase in the peptide concentration is required.
Cellulose nanofibril binding
Efficacy of CBM binding of CBM-22 aa linker-DCD-1L
The efficacy of cellulose binding domain binding to a cellulose substrate was probed.       First, the purified protein construct was cleaved with Ulp1 to remove the His6x-Smt3 tag       coupled with subsequent heat inactivation of the enzyme. For the binding assay, concentrations of peptide varying from 0.1 μM to 50 μM       in low salt buffer were incubated with 150 μg of cellulose nanoFibrils (CNF).        Control samples of matching peptide concentrations were prepared without the addition of CNF.        The samples were mixed thoroughly and incubated at room temperature for 1 hour and then centrifuged for 10 min at 4000 g.        The supernatant from centrifugation was extracted for SDS-PAGE analysis.        Prepared SDS-PAGE samples were run in a 12 % acrylamide gel. The gels were fixed and stained with Coomassie blue.        For CBM-22 aa linker-DCD-1L a band at 23.8 kDa is expected.        The difference in the CBM-22 aa linker-DCD-1L bands between samples containing 150 μg CNF and those with no added CNF corresponds        to the amount of protein successfully bound to the cellulose substrate.        The SDS-PAGE gels for varying protein concentrations have been presented in Figure 41.        The gels containing samples with peptide concentrations below 4 μM have not been presented due to going below the detection limit of        Coomassie Blue staining. Based on the SDS-PAGE analysis,        we identify that up to a concentration of 10 μM close to 100 % of CBM-22 aa linker-DCD-1L appears to have successfully bound to the cellulose substrate.
Figure 41. SDS-PAGE gels of CBM binding assay of C22D. a.) Protein concentration of 4 μM to 15 μM b.) protein concentrations of 20 μM to 35 μM c.) protein concentrations of 40 μM to 50 μM.       In all gels samples without CNF and with 150 μg of CNF have been pipetted in adjacent wells.       The strong band at 23.8 kDa corresponds to CBM-22 aa linker-DCD-1L.
Efficacy of CBM binding of DCD-1L-22 aa linker-CBM
The efficacy of cellulose binding domain binding to a cellulose substrate was probed.         First, the purified protein construct was cleaved with Ulp1 to cleave off the His6x-Smt3 tag coupled with subsequent heat inactivation of the enzyme.         For the binding assay, concentrations of 5 μM to 50 μM of peptide in low salt buffer were incubated with 150 μg of cellulose nanoFibrils (CNF).
Control samples of matching peptide concentrations were prepared without the addition of CNF.         The total sample volume for both samples was 100 μl.         The samples were mixed thoroughly and incubated at room temperature for 1 hour, after which the samples were centrifuged for 10 min at 4000 g         and the supernatant was extracted for SDS-PAGE analysis. Prepared SDS-PAGE samples were run in a 12% acrylamide gel and the gels were fixed and stained with Coomassie blue.         For DCD-1L-22 aa linker-CBM we expect a band at 23.8 kDa. The difference in the DCD-1L-22 aa linker-CBM bands between samples containing 150 µg CNF and those with no added CNF should correspond to          the amount of protein successfully bound to the cellulose substrate.          The SDS-PAGE gels for varying protein concentrations have been presented in Figure 42.          Based on the SDS-PAGE analysis, we can determine that the CBM linked to our fusion protein remains active and successfully binds to CNF.
Figure 42. SDS-PAGE gels of CBM binding assay of DCD-1L-22 aa linker-CBM a.) Protein concentration of 5 μM to 10 μM b.) protein concentrations of 15 μM to 20 μM c.) protein concentrations of 30 μM to 40 μM. In all gels, two replicates of samples without CNF and with 150 µg of CNF have been pipetted in adjacent wells. The band at 23.8 kDa corresponds to DCD-1L-22 aa linker-CBM.
From the Figures 41(a,b) and 42(a,b) we can observe that our CBM binds to the CNF due to lack of or notably fainter bands in the lanes with CNF. As the concentration of peptides increases,         we also observe the protein bands in wells with CNF (Figure 41(c) and 42(c)). This is because the CNF is saturated with high concentration of protein and there are many unbound         proteins in the solution.
One thing to note here is the presence of  the strong band between 15 kDa and 20 kDa which is unidentified. Our initial suspicion regarding the unidentified band was that it corresponds to free CBM based on its molecular weight of 17.4 kD.         However, this band is also present in wells with CNF meaning that this band can not contain CBM given that CBM bands are not seen due to         CNF-CBM binding. Thus we conclude that it is most likely not the cleaved CBM. Another possibility is that Ulp1 enzyme cleaves the construct at a position between the 22 linker leaving His6x-Smt3-DCD-1L - with molecular weight 18.3 kDa- free from CBM. But we are unclear about it since we haven not observed this in any of our previous experiment         gels. For future perspectives, it would be interesting to probe the recombinant protein structure for homologous Ulp1 cleavega sites through modeling.
References
[1] Geneious molecular biology and NGS analysis tools. Accessible at: [here] .
[2] Kearse, M., Moir, R., Wilson, A., Stones-Havas, S., Cheung, M., Sturrock, S., Buxton, S., Cooper, A., Markowitz,             S., Duran, C., Thierer, T., Ashton, B., Mentjies, P., & Drummond, A. (2012). Geneious Basic: an integrated and extendable             desktop software platform for the organization and analysis of sequence data. Bioinformatics, 28(12), 1647-1649.
[3] Novagen pET System Manual. Accessible at: [here] .
HOME AALTO UNIVERSITY
UNIVERSITY OF HELSINKI SITE MAP
