<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aalto-Helsinki_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aalto-Helsinki/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Aalto-Helsinki</TITLE></P><DIV id="dropdown"><UL><LI class="active has-sub"><A class="lab" href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Laboratory</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Theory">Theoretical Background</A></LI><LI class="has-sub"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Experiments">Materials and Methods</A></LI><LI class="last"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Protocols">Protocols</A></LI></UL><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Results">Results and Discussion</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Future">Future Perspectives</A></LI><LI class="active has-sub"><A class="mod" href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Theory">Theoretical Background</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Setup">Simulation Setup</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Results">Results and Discussion</A></LI><LI class="active has-sub"><A class="hp" href="https://2017.igem.org/Team:Aalto-Helsinki/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Story">Project</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Engagement">Public Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/HP/Silver">Judging</A></LI><LI class="active has-sub"><A class="com" href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Community</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Composite_Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Improve">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Collaborations">Collaborations</A></LI><LI class="active has-sub"><A class="peo" href="https://2017.igem.org/Team:Aalto-Helsinki/Team">People</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Sponsors">Sponsors</A></LI></DIV><DIV id="color-block"><SPAN><DIV class="text1">LABORATORY</DIV><DIV class="text2"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Overview</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Theory">Theoretical Background</A><A style="text-decoration: underline" href="https://2017.igem.org/Team:Aalto-Helsinki/Experiments">Materials and Methods</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Protocols">Protocols</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Results">Results and Discussion</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Future">Future Perspectives</A></DIV></SPAN></DIV><DIV class="container"><DIV class="introtext"><H3 style="color: #339999">Materials and Methods</H3><P id="paragrapgh"><I>Note: For detailed steps of each method, please check the <A href="https://2017.igem.org/Team:Aalto-Helsinki/Protocols">Protocols</A> section.</I></P><H4>Cloning of DNA constructs in pET28a(+) expression vector</H4><P id="paragraph">
        Smt3 tag was amplified by PCR from a plasmid generously provided by Sesilja Aranko (from Aalto University) with forward primer TAT<U>CATATG</U>GGATCGGACTCAGAAGTC (NdeI restriction site underlined) and reverse primer TGAT<B>CTCGAG</B>TTA<U>GGATCC</U>ACCAATCTGTTC (XhoI restriction site in bold; BamHI restriction site underlined). The PCR product was ligated into the vector pET28a(+) (carrying an N-terminal His6x tag and the gene for kanamycin resistance) restricted with NdeI and XhoI restriction enzymes to obtain the plasmid pET28a(+)-Smt3. 
</P><P id="paragraph"><I>Figure 1. Illustration of cloning and transformation experiments including restriction, ligation and transformation.</I></P><P id="paragraph">
        The following DNA constructs were ordered from Integrated DNA Technologies (IDT) as gBlocks. Sequence of the CBM used is from family 3 cellulose binding domain from <I>Clostridium thermocellum</I>. 
</P><DIV style="word-wrap:break-word"><P id="paragraph"><B>1. DCD-1L</B>
        AAC<U>GGATCC</U>AGCCTGCTGGAAAAAGGCCTGGATGGCGCGAAAAAAGCGGTGGGCGGCCTGGGCAAACTGGGCAAAGATGCGGTGGAAGATCTGGAAAGCGTGGGCAAAGGCGCGGTGCATGATGTGAAAGATGTGCTGGATAGCGTGCTGTAA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold), 
</P></DIV><DIV style="word-wrap:break-word"><P id="paragraph"><B>2. DCD-1L-22 aa linker-CBM</B>
        AAC<U>GGATCC</U>TCCTTGCTTGAGAAAGGGCTGGACGGAGCAAAGAAAGCAGTGGGGGGCTTGGGCAAATTGGGCAAGGACGCGGTTGAGGATCTGGAGTCCGTAGGCAAGGGGGCCGTTCATGATGTAAAGGATGTACTTGATTCGGTTCTTTCAGCTTCTGCGAGTGCGAGCGCAGGGGCTAGTGCAGCCGCAAGCGCCGGAGCTGGTGCAGGGGCAATGGGAAACTTAAAGGTTGAATTTTACAATTCCAATCCATCGGACACAACCAATTCTATTAACCCACAATTTAAAGTAACGAATACCGGCTCTTCTGCGATCGATCTGTCTAAATTAACTTTACGTTATTACTACACAGTGGACGGACAAAAGGACCAAACATTTTGGTGTGACCACGCTGCGATCATTGGATCGAACGGGTCCTACAATGGCATCACTAGCAATGTCAAAGGCACTTTCGTAAAAATGTCGAGTAGCACAAACAATGCCGATACATATTTAGAGATTTCTTTCACGGGAGGCACCCTGGAGCCCGGAGCGCACGTACAGATTCAAGGCCGCTTCGCTAAGAATGACTGGTCGAATTATACGCAGTCGAACGATTATTCATTCAAGAGTGCGTCACAATTTGTGGAATGGGATCAGGTGACCGCTTATTTGAACGGAGTATTGGTATGGGGTAAAGAGCCATGA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold),
</P></DIV><DIV style="word-wrap:break-word"><P id="paragraph"><B>3. CBM-10 aa linker-DCD-1L</B>
        AAC<U>GGATCC</U>GGCAACCTGAAAGTGGAATTTTATAACAGCAACCCGAGCGATACCACCAACAGCATTAACCCGCAGTTTAAAGTGACCAACACCGGCAGCAGCGCGATTGATCTGAGCAAACTGACCCTGCGCTATTATTATACCGTGGATGGCCAGAAAGATCAGACCTTTTGGTGCGATCATGCGGCGATTATTGGCAGCAACGGCAGCTATAACGGCATTACCAGCAACGTGAAAGGCACCTTTGTGAAAATGAGCAGCAGCACCAACAACGCGGATACCTATCTGGAAATTAGCTTTACCGGCGGCACCCTGGAACCGGGCGCGCATGTGCAGATTCAGGGCCGCTTTGCGAAAAACGATTGGAGCAACTATACCCAGAGCAACGATTATAGCTTTAAAAGCGCGAGCCAGTTTGTGGAATGGGATCAGGTGACCGCGTATCTGAACGGCGTGCTGGTGTGGGGCAAAGAACCGGGCAGCGGCAGCGGCAGCGGCAGCGGCAGCAGCAGCCTGCTGGAAAAAGGCCTGGATGGCGCGAAAAAAGCGGTGGGCGGCCTGGGCAAACTGGGCAAAGATGCGGTGGAAGATCTGGAAAGCGTGGGCAAAGGCGCGGTGCATGATGTGAAAGATGTGCTGGATAGCGTGCTGTAA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold), 
</P></DIV><DIV style="word-wrap:break-word"><P id="paragraph"><B>4. CBM-22 aa linker-DCD-1L</B>
        AAC<U>GGATCC</U>GGAAACCTGAAAGTAGAATTTTACAATAGTAATCCCTCGGACACAACTAATTCTATTAACCCTCAGTTTAAGGTAACCAATACGGGGTCTAGTGCAATCGACCTTAGCAAACTTACTTTGCGCTACTATTATACAGTAGATGGGCAGAAGGACCAAACCTTCTGGTGTGATCATGCAGCAATTATTGGGTCTAATGGCTCGTATAATGGGATCACAAGCAATGTCAAAGGGACATTTGTAAAGATGAGCAGCAGCACCAATAACGCAGATACTTACTTAGAAATCTCATTTACTGGCGGGACATTGGAACCTGGCGCTCACGTCCAAATCCAAGGTCGCTTCGCAAAAAATGACTGGTCTAACTATACACAGAGCAACGACTATAGCTTTAAATCGGCATCGCAGTTTGTAGAGTGGGATCAGGTGACCGCTTACCTGAACGGAGTTCTGGTTTGGGGGAAAGAACCATCCGCATCAGCTAGCGCATCAGCAGGTGCCTCTGCGGCAGCCTCAGCCGGAGCAGGTGCCGGGGCTTCTTCCCTTCTGGAGAAGGGGCTGGATGGTGCTAAAAAAGCGGTGGGCGGCCTTGGAAAGTTGGGTAAAGATGCTGTAGAGGATCTTGAAAGCGTAGGAAAGGGGGCCGTGCATGACGTCAAGGATGTATTGGATTCAGTGCTGTAA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold),
</P></DIV><DIV style="word-wrap:break-word"><P id="paragraph"><B>5. LL-37</B>
        AAC<U>GGATCC</U>ATGCTGCTGGGCGATTTTTTTCGCAAAAGCAAAGAAAAAATTGGCAAAGAATTTAAACGCATTGTGCAGCGCATTAAAGATTTTCTGCGCAACCTGGTGCCGCGCACCGAAAGCTAA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold). 
</P></DIV><DIV style="text-align:left"><P id="paragraph">
        gBlocks were ligated into the pET28a(+)-Smt3 vector restricted with BamHI and XhoI restriction enzymes to obtain the plasmids: 
        </P><DIV style="font-size:initial"><OL type="1"><LI>pET28a(+)-Smt3-DCD-1L,</LI><LI>pET28a(+)-Smt3-DCD-1L-22 aa linker-CBM,</LI><LI>pET28a(+)-Smt3-CBM-10 aa linker-DCD-1L,</LI><LI>pET28a(+)-Smt3-CBM-22 aa linker-DCD-1L,</LI><LI>pET28a(+)-Smt3-LL-37.</LI></OL></DIV></DIV><P id="paragraph">
        Additionally, the following two gBlocks were obtained from IDT, but cloning in the pET28a(+)-Smt3 vector was unsuccessful:
</P><DIV style="word-wrap:break-word"><P id="paragraph"><B>DCD-1L-10 aa linker-CBM</B>
        AAC<U>GGATCC</U>AGCCTGCTGGAAAAAGGCCTGGATGGCGCGAAAAAAGCGGTGGGCGGCCTGGGCAAACTGGGCAAAGATGCGGTGGAAGATCTGGAAAGCGTGGGCAAAGGCGCGGTGCATGATGTGAAAGATGTGCTGGATAGCGTGCTGGGCAGCGGCAGCGGCAGCGGCAGCGGCAGCATGGGCAACCTGAAAGTGGAATTTTATAACAGCAACCCGAGCGATACCACCAACAGCATTAACCCGCAGTTTAAAGTGACCAACACCGGCAGCAGCGCGATTGATCTGAGCAAACTGACCCTGCGCTATTATTATACCGTGGATGGCCAGAAAGATCAGACCTTTTGGTGCGATCATGCGGCGATTATTGGCAGCAACGGCAGCTATAACGGCATTACCAGCAACGTGAAAGGCACCTTTGTGAAAATGAGCAGCAGCACCAACAACGCGGATACCTATCTGGAAATTAGCTTTACCGGCGGCACCCTGGAACCGGGCGCGCATGTGCAGATTCAGGGCCGCTTTGCGAAAAACGATTGGAGCAACTATACCCAGAGCAACGATTATAGCTTTAAAAGCGCGAGCCAGTTTGTGGAATGGGATCAGGTGACCGCGTATCTGAACGGCGTGCTGGTGTGGGGCAAAGAACCGTAA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold) and 
</P></DIV><P id="paragraph"><B>SSL-23</B> (the gBlock contains additionally the sequence of the CBM that was used, but restriction sites were introduced to digest only SSL-23) 
</P><DIV style="word-wrap:break-word"><P id="paragraph">
        GCTCCAGTTCATGGGCAACCTGAAAGTGGAATTTTATAACAGCAACCCGAGCGATACCACCAACAGCATTAACCCGCAGTTTAAAGTGACCAACACCGGCAGCAGCGCGATTGATCTGAGCAAACTGACCCTGCGCTATTATTATACCGTGGATGGCCAGAAAGATCAGACCTTTTGGTGCGATCATGCGGCGATTATTGGCAGCAACGGCAGCTATAACGGCATTACCAGCAACGTGAAAGGCACCTTTGTGAAAATGAGCAGCAGCACCAACAACGCGGATACCTATCTGGAAATTAGCTTTACCGGCGGCACCCTGGAACCGGGCGCGCATGTGCAGATTCAGGGCCGCTTTGCGAAAAACGATTGGAGCAACTATACCCAGAGCAACGATTATAGCTTTAAAAGCGCGAGCCAGTTTGTGGAATGGGATCAGGTGACCGCGTATCTGAACGGCGTGCTGGTGTGGGGCAAAGAACCGAAC<U>GGATCC</U>AGCCTGCTGGAAAAAGGCCTGGATGGCGCGAAAAAAGCGGTGGGCGGCCTGGGCAAACTGGGCAAATAA<B>CTCGAG</B>TGT (BamHI restriction site underlined; XhoI restriction site in bold).
</P></DIV><P id="paragraph">
        TOP10 Chemically competent <I>Escherichia coli</I> were transformed with ligated plasmids. Heat-shock was performed at +42 °C and cells were incubated at +37 °C with LB-medium prior to spread-plating on LB-agar plates containing 50 μg/ml kanamycin. Plates were incubated at +37 °C for over 12 hours before selecting kanamycin-resistant transformants for plasmid isolation, which was performed according to the manual of high copy number plasmid isolation in the Macherey-Nagel Nucleospin Plasmid kit. To verify that the isolated plasmids carried the desired insert cloned into the vector, check-up digestion of the plasmids was performed using NdeI and XhoI restriction enzymes. Plasmids that carried an insert of the correct size were sent for sequencing at Eurofins Genomics, using the available sequencing primer T7 term (CTA GTT ATT GCT CAG CGG T).
</P><H4>Protein expression and purification</H4><H5>Small Scale (5 ml culture)</H5><P id="paragraph">
        For small scale production, 3-4 colonies from the transformation plates were inoculated (<A href="http://www.neb.com/products/c2566-t7-express-competent-e-coli-high-efficiency">the expression strain cells</A> transformed with plasmids carrying the gene of interest) in 7 ml of LB-kanamycin (50 μg/ml working concentration), and the cells were grown at +37 °C until the culture reached the OD<SUB>600</SUB> value of ~0.51. When finished growing the cells, the expression of the gene of interest was induced by adding a final concentration of 0.5 mM IPTG in the cultures and continued to grow at +37 °C with shaking. 
</P><P id="paragraph">
        Then, 4 hours after induction, the cells were pelleted by centrifuging at 12,000 x g for 1 minute and the supernatant was discarded. The pellet was resuspended in 100 μl of <A href="http://www.thermofisher.com/order/catalog/product/78248">ThermoFisher Scientific B-PER Bacterial Protein Extraction Reagent</A>. After equilibrating the Qiagen Ni-NTA spin columns with 600 μl of NPI-10 buffer (50 nM NaPi, 300 mM NaCl, pH 8.0) the protein purification was started. Samples were loaded onto the spin columns and centrifuged at 1,600 rpm for 5 minutes. Followed by washing the columns with 600 μl of NPI-20 buffer (50 mM NaPi, 300 mM NaCl, 30 mM imidazole, pH 8.0). Proteins were eluted in 300 μl of NPI-500 buffer (50 mM NaPi, 300 mM NaCl, 250 mM imidazole, pH 8.0). Stored the used spin columns soaked in 0.1 M EDTA solution, at +4 °C. The samples from different flow throughs and elution were then analyzed using SDS-PAGE.
</P><P id="paragraph"><I>Figure 2. Illustration to represent our scale up operations in protein production from 5 ml culture to 500 ml culture.</I></P><H5>Large Scale (500 ml batch)</H5><P id="paragraph">
        T7 Express Competent <I>E. coli</I> were transformed with cloned plasmids by heat-shock transformation. Kanamycin-resistant transformants were pre-cultured in LB medium with 50 μg/mL kanamycin at +30 °C overnight. Next day the pre-culture was diluted 1:100 with fresh LB-kanamycin medium and grown at +37 °C until OD<SUB>600</SUB> value reached 0.6. Protein expression was induced with IPTG at a final concentration of 0.5 mM. Cells were harvested 4 hours after induction of protein expression by centrifuging in ThermoScientific Sorvall Lynx 4000 centrifuge at 5,000 x g for 10 minutes at +16 °C with the rotor F10-4x1000 LEX. Pellet was resuspended in Buffer A (50 mM NaPi, 300 mM NaCl, pH 8.0) and frozen in liquid nitrogen before storing at -20 °C.
</P><P id="paragraph">
        Frozen cells were thawed in hand-warm water bath before lysing them with EmulsiFlex-C3. Lysed cells were centrifuged in ThermoScientific Sorvall Lynx 4000 centrifuge at 18,000 x g for 30 minutes at +4 °C with the rotor F20-12x50 LEX. Supernatant was purified with GE Healthcare Life Sciences ÄKTA pure using the GE Healthcare Life Sciences HisTrap<SUP>TM</SUP> FF crude 5 ml column.
</P><P style="text-align: center" id="paragraph"><I>Figure 3. Image of AVESTIN EmulsiFlex-C3 used for cell lysis in our project.</I></P><P style="text-align: center" id="paragraph"><I>Figure 4. Image of GE Healthcare Life Sciences ÄKTA pure machine used for protein purification in our project.</I></P><P style="text-align: center" id="paragraph"><I>Figure 5. Image of GE Healthcare Life Sciences HisTrap<SUP>TM</SUP> FF crude 5ml column used in our project.</I></P><P id="paragraph">
        To remove imidazole from the buffer, buffer exchange and concentration was performed for purified proteins, which had been eluted with Buffer B (50 mM NaPi, 300 mM NaCl, 250 mM imidazole, pH 8.0). Protein samples were loaded onto Sartorius Vivaspin 20 ultrafiltration centrifugal tubes (5,000 MWCO). Samples were centrifuged at 3,000 x g for 45 minutes at +4 °C, repeated until sample volume was 5 ml. New buffer was added into the tubes three times, and tubes were centrifuged at 3,000 x g for 1 hour at +4 °C. Protein samples (in the new buffer) were frozen in liquid nitrogen before storing at -20 °C.
</P><P style="text-align: center" id="paragraph"><I>Figure 6. Image of Sartorius Vivaspin 20 ultrafiltration centrifugal tubes (5,000 MWCO) used for protein concentration and buffer exchange in our project. </I></P><P id="paragraph">
        Fractions from different steps of protein expression and purification (non-induced, 2 hours after induction, 4 hours after induction, lysate, pellet, flow-through fractions from purification, eluates) were run on an SDS-PAGE to determine which fractions contained the desired proteins. Samples were mixed with 2X SDS-PAGE Loading Dye, heated at +95 °C for 10 minutes, and run on 15% polyacrylamide gels. Gels were stained with Coomassie Blue and imaged with Bio-Rad Gel Doc<SUP>TM</SUP> XR+ imager and Image Lab software (version 5.1). 
</P><H4>Recombinant protein digestion</H4><P id="paragraph">
        Purified recombinant proteins containing His6x tag and Smt3 tag were cleaved with Ulp1 protease. 50 μL of recombinant protein mixture was incubated with 0.5 μL Ulp1 for 10-30 minutes at RT. Samples from cleaved and uncleaved proteins were run on 15% SDS-PAGE to verify the successful digestion.
</P><H4>MALDI-TOF-TOF (matrix-assisted laser desorption/ionization-time of flight/time of flight) Mass Spectrometry</H4><P id="paragraph">
        Masses of purified proteins and peptides were identified with MALDI-TOF mass spectrometry. Salt was removed from protein samples by dialysis with a dialysis tubing of 68 kDa MWCO overnight at +4 °C. Salt was removed from Ulp1-digested DCD-1L peptide with ZipTip. Protein/peptide samples and calibration standards (1 μL) were mixed with a final concentration of 10% acetic acid. Prepared Matrix A for proteins (&gt;10 kDa): dissolved a pinch of sinapinic acid in a 1:1-mixture of Milli-Q and acetonitrile (ACN), added trifluoroacetic acid (TFA) in a final concentration of ~0.1%. Prepared Matrix B for peptides (&lt;10 kDa): dissolved a pinch of α-cyano-4-hydroxycinnamic acid in a 7:3-mixture of MQ and ACN, added TFA in a final concentration of 0.05%. Mixed matrices thoroughly by vortexing for ~10 minutes. Mixed 1 μL of protein/peptide sample or standard with 1 μL of the appropriate matrix on MALDI target plates. Let the samples dry for 1 hour and conducted the measurement with UltrafleXtream<SUP>TM</SUP> Bruker MALDI-TOF-TOF mass spectrometer equipped with a 200-Hz smart-beam 1 lazer (337 nm, 4 ns pulse). Data collection was carried out by operating the instrument in positive ion mode controlled by the flex software packaged (FlexControl, FlexAnalysis). 5,000 laser shots were accumulated per each spectrum in MS modes. Protein Calibration Standard mixture I, II and peptide calibration standard II (Bruker Daltonics) were used to calibrated the MS spectra. 
</P><P id="paragraph"><I>Figure 7. Illustration representing salt removal from protein samples of bigger molecular weight by dialysis with a dialysis tubing  carried out for sample processing before MALDI-TOF-TOF.</I></P><P id="paragraph"><I>Figure 8. Illustration representing salt removal from Ulp1-digested DCD-1L peptide with ZipTip carried out for sample processing before MALDI-TOF-TOF.</I></P><H4>Antimicrobial assay</H4><P id="paragraph">
        Antimicrobial assays were performed to test the activity of DCD-1L and LL-37 against <I>E. coli</I>, using the expression strain (T7 Express competent <I>E. coli</I>). To obtain a standard curve for CFU/ml with respect to different OD<SUB>600</SUB> values, the cells were grown to the desired OD<SUB>600</SUB> values (0.1, 0.2, 0.3, 0.4, 0.5) and plated at 10<SUP>6</SUP> dilution. The plates were incubated at +37 °C for over 12 hours, and the CFUs corresponding to each OD<SUB>600</SUB> value were calculated from the plates. The experimental <I>E. coli</I> were grown at +37 °C until OD<SUB>600</SUB> reached 0.05 (corresponding to 1.4x10<SUP>8</SUP> CFU/ml). Samples of bacteria were incubated with different antimicrobial peptides (DCD-1L and LL-37 at a concentration of 100 μg/ml, Smt3 tag previously removed by cleaving with Ulp1) and controls (positive controls: chloramphenicol at a concentration of 35 μg/ml and the antimicrobial peptide nisin, generously provided by Prof. Per Saris from Faculty of Environmental and Agricultural sciences,  University of Helsinki, at a concentration of 100 μg/ml; negative control: without any peptide) at +37 °C with shaking for 45 minutes. After incubation, OD<SUB>600</SUB> values of the cells were measured and the bactericidal activity of the tested reagents was expressed as: [1 – (CFU after peptide incubation)/(CFU before peptide incubation)] × 100, which represents the percentage of cells that were killed.
</P><H4>Cellulose nanofiber binding</H4><P id="paragraph">
Binding of purified protein CBM-22 aa linker-DCD-1L containing a CBM was tested with cellulose nanofibrils (CNF).
</P><P id="paragraph">
First, cellulose nanofibrils (CNF) were prepared by fluidizing bleached birch pulp in deionized water six times (six passes) using a Voith LR40 homogenizer. A net specific energy of 300 kWh/t was used with specific edge load of 0.5 J/m and refiner speed of 150. The resulting samples were then used to make a 1 mg/ml solution in water with 0.02% NaN<SUP>3</SUP>. They were then were stored at 4 °C and taken out to room temperature 1 hour prior to use.
</P><P id="paragraph">
Binding of purified protein CBM-22 aa linker-DCD-1L containing a CBM was tested with cellulose nanofiber (CNF). His6x tag and Smt3 tag were removed from the protein by Ulp1 digestion. 150 μg of CNF was mixed with varying concentrations of protein (0.5 μM to 50 μM) and incubated for 1 hour at RT. Mixtures were centrifuged at 4,000 x g for 10 minutes. Samples from supernatants were run on 12% SDS-PAGE along with control samples that were not bound to CNF. Gel was fixed with acetic acid/methanol/water mixture, stained with Coomassie Blue and imaged with Bio-Rad Gel Doc<SUP>TM</SUP> XR+ imager and Image Lab software (version 5.1).
</P></DIV></DIV><FOOTER><DIV class="footer-social"><DIV class="texts"><UL class="pe-social"><A href="https://2017.igem.org/Team:Aalto-Helsinki">HOME</A><A href="http://www.aalto.fi/en/">AALTO UNIVERSITY</A><A href="https://www.helsinki.fi/en">UNIVERSITY OF HELSINKI</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Map">SITE MAP</A></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>