<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aalto-Helsinki_Model_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aalto-Helsinki/Model Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Aalto-Helsinki</TITLE></P><DIV id="dropdown"><UL><LI class="active has-sub"><A class="lab" href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Laboratory</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Description">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Theory">Theoretical Background</A></LI><LI class="has-sub"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Experiments">Materials and Methods</A></LI><LI class="last"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Protocols">Protocols</A></LI></UL><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Results">Results and Discussion</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Laboratory_Future">Future Perspectives</A></LI><LI class="active has-sub"><A class="mod" href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Modeling</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Theory">Theoretical Background</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Setup">Simulation Setup</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Results">Results and Discussion</A></LI><LI class="active has-sub"><A class="hp" href="https://2017.igem.org/Team:Aalto-Helsinki/Human_Practices">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Story">Project</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Engagement">Public Outreach</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/HP/Silver">Judging</A></LI><LI class="active has-sub"><A class="com" href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Community</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Community">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Parts">Parts</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Composite_Part">Composite Part</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Improve">Improved Part</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/InterLab">InterLab</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Collaborations">Collaborations</A></LI><LI class="active has-sub"><A class="peo" href="https://2017.igem.org/Team:Aalto-Helsinki/Team">People</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Team">Team Members</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Advisors">Advisors</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:Aalto-Helsinki/Sponsors">Sponsors</A></LI></DIV><DIV id="color-block-mod"><SPAN><DIV class="text1">MODELING</DIV><DIV class="text2"><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model">Modeling Overview</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Theory">Theoretical Background</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Setup">Simulation Setup</A><A style="text-decoration: underline" href="https://2017.igem.org/Team:Aalto-Helsinki/Model_Results">Results and Discussion</A></DIV></SPAN></DIV><DIV class="container"><DIV class="introtext"><H3 style="color: #CC9933">Results and Discussion</H3><P id="paragraph">
Based on molecular dynamics simulations of DCD-1L at different salt concentrations and temperatures, we evaluated occurring changes in protein secondary structure through secondary structure assignment using DSSP and observed Ramachandran plot representations. Additionally, we characterized the structure of the simulated peptide by scrutinizing the radius of gyration, solvent accessible surface area and hydrogen bonding.
</P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Characterization of the Amino Acid Sequence</H4><P id="paragraph">
Despite the great diversity of antimicrobial peptides, they exhibit many commonly shared characteristics. Such characteristics include the presence of multiple basic amino acids and amphipathic structures consisting of clusters of hydrophobic and hydrophilic amino acids. The length of the polypeptide chain of AMPs is quite short, generally ranging from 12 to 50 amino acids in length. Commonly, AMPs have a positive net charge, which has been shown to contribute to their initial binding to the often negatively charged bacterial cell membrane. The amphipathic structures found in AMPs have been found to enhance interactions with the phospholipids of the bacterial cell membrane, especially at the water-lipid interface.
</P><P id="paragraph">
DCD-1L consists of 48 amino residues and is processed proteolytically from a 110 amino acid precursor peptide. Contrary to many common AMPs, DCD-1L has a net negative charge at physiological pH. [<A name="ref10" href="#refl10">10</A>] The expected pI of DCD-1L is 5.07 and the expected molecular weight of the peptide is 4818.50 g/mol. Using the web-based tool Composition Profiler [<A name="ref16" href="#refl16">16</A>] we attempted to identify differences in amino acid residue expression in the sequence of DCD-1L when compared to the distribution of the 20 natural amino acids as commonly found in nature. A p-value of 0.05 was used as a threshold and Swissprot51 dataset [<A name="ref1" href="#refl1">1</A>], which closely mirrors the amino acid distribution found in nature, was used for background screening. Figure 1 shows the results of the amino acid expression screening with the amino acids being color coded according to how often they occur in α-helical structures. We find that especially the amino acids Aspartate, Glycine, Valine and Lysine are over-expressed, when compared to the frequency of these amino acids commonly found in nature. Of the aforementioned amino acids, Aspartate exhibits a negatively charged side chain, while Glycine and Valine are considered hydrophobic and Lysine a polar amino acid. According to figure 1, Lysine is an amino acid commonly found in α-helical structures, which are common for many AMPs [<A name="ref10" href="#refl10">10</A>, <A name="ref2" href="#refl2">2</A>]. The over-expression of negatively charged Aspartate residues is expected considering the overall negative net charge of DCD-1L. Additionally, in figure 1 we have included the amino acid sequence of DCD-1L in color, differentiating between polar and hydrophobic residues. Unsurprisingly, DCD-1L appears to have alternating clusters of polar and hydrophobic amino acids, with such amphipathic regions being common for AMPs as previously discussed.
</P><P style="text-align: left" id="paragraph"><I>Figure 1. Differences in expression of the 20 natural amino acids in DCD-1L sequence based on analysis by Composition profiler. The residues have been color coded according to the frequency at which they contribute to the α-helix secondary structure. Additionally, the amino acid sequence of DCD-1L has been provided and colored according to the presence of polar and hydrophobic amino acid residues.</I></P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Radius of Gyration and Solvent Accessible Surface Area</H4><P id="paragraph">
Protein interactions are often accompanied by significant changes in conformation. The radius of gyration describes the distribution of an object's mass around an axis. In polymer physics, the radius of gyration is one of the measures used to describe the dimensions of a polymer chain or, in this case, our polypeptide chain. The radius of gyration can be determined experimentally through static light scattering, small angle neutron- (SANS) and x-ray scattering (SAXS) experiments and thus allows a point of reference for checking a computational model’s accuracy. Based on our simulation trajectories, we calculated both the time evolution of the radius gyration over the course of our simulation and the average radius of gyration for the last 5 ns of simulation. The time evolution of the radius of gyration for DCD-1L simulated at different temperatures has been presented in figure 2. In all of our simulations we observe a clear decrease in the radius of gyration of our protein, which we link to the aggregation of our protein structure and the assumption of a random coil-like structure as the more constrained α-helical secondary structure is quickly decomposed due to protein-solvent interaction. The average radius of gyration as a function of both salt concentration and system temperature has been plotted in figure 3. We observe a rising trend in the radius of gyration as temperature is increased, based on which we can postulate that the protein assumes a looser random coil conformation as temperature is increased. No clear trends for the radius of gyration can be established as a function of NaCl concentration, as is evident from figure 3.    
</P><P style="text-align: left" id="paragraph"><I>Figure 2. Left: Radius of gyration of DCD-1L as a function of simulation time. Right: Solvent accessible surface area as a function of simulation time. Data for DCD-1L simulated at 290 K, 300 K, 310 K and 320 K has been presented.</I></P><P style="text-align: left" id="paragraph"><I>Figure 3. Radius of gyration averages over the last 5 ns of simulation time as a function of NaCl concentrations (left) and temperature (right). The error bars denote standard deviation.</I></P><P id="paragraph">
Additionally, we probed the changes in protein solvent accessible surface area [<A name="ref5" href="#refl5">5</A>] as a function of simulation time. The solvent accessible surface area was calculated using a probe radius of 0.14 nm which closely corresponds to that of a water molecule. Recently, it has been suggested that solvent accessible surface area can be used to help better predict assignment of protein secondary structure [<A name="ref9" href="#refl9">9</A>]. Generally, folding of a soluble protein decreases the area in contact with the solvent, which is reflected as a decrease in solvent accessible surface area. The decrease in solvent accessible surface area is generally more pronounced for hydrophobic residues. Lins et al. [<A name="ref9" href="#refl9">9</A>] analyzed the solvent accessible surface area of 587 proteins using NMR and X-ray diffraction and found that folding of the protein tends to equilibriate the hydrophobic and hydrophilic solvent-accessible areas, with the ratio of the two being close to 1. As such, the surface of a soluble protein is not only hydrophilic, as is often wrongly assumed. In our simulations, we observe a slight decrease in the solvent accessible area across all simulated systems. This hints that the resulting random coil structure is tight enough to restrict penetration of the structure by water molecules. The reduction of solvent accessible surface area can be observed in figure 2. We also calculated the average solvent accessible surface area over the last 5 ns of simulation time, which has been plotted in figure 4 as a function of both salt concentration and system temperature. While there appears no clear trend in the solvent accessible surface area as a function of salt concentration, an increasing trend is observed with respect to increased system temperature. This gives further evidence to the hypothesis that the protein assumes a looser conformation at higher temperatures, which is more permeable to water molecules.
</P><P style="text-align: left" id="paragraph"><I>Figure 4. Solvent accessible surface area averages over the last 5 ns of simulation time as a function of NaCl concentrations (left) and temperature (Right). The error bars denoting standard deviation.</I></P><P id="paragraph">
Lastly, we examined the number of hydrogen bonds present in the protein structure as a function of simulation time. The importance of hydrogen bonding is paramount in the formation of protein secondary structure and folding. The average number of hydrogen bonds present in the protein structure as a function of both concentrations of NaCl and system temperature have been presented in figure 5. The averages were calculated over the last 5 ns of simulation time. While the average number of hydrogen bonds within the structure appears to remain constant within the margin of error as salt concentration in the system is varied, there appears a decreasing trend in the number of hydrogen bonds found in the protein backbone as system temperature is increased. This corresponds with the assumption that the protein assumes a loosed random coil conformation at higher temperatures leading to increased distance between amino acid residues. Higher temperatures in general, also contribute to higher mobility of both the solvent water molecules and the protein backbone.
</P><P style="text-align: left" id="paragraph"><I>Figure 5. Averages of the number of hydrogen bonds in the protein structure over the last 5 ns of simulation time as a function of NaCl concentrations (left) and temperature (Right). The error bars denoting standard deviation.</I></P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Assignment of Protein Secondary Structure Elements</H4><H5>Secondary Structure Assignment Using DSSP</H5><P id="paragraph">
The DSSP algorithm calculates the most likely secondary structure assignment element-wise based on the protein three dimensional structure. Based on individual atom coordinates extracted from the simulation trajectory, the algorithm first calculates the H-bond energy between all atoms. The positions of hydrogen atoms are discarded from the original trajectory and instead replaced at 0.1 nm of the backbone N in the opposite direction from the backbone C=O bond. The DSSP algorithm uses the best two possible H-bonds to assign the most likely secondary structure element. The DSSP algorithm thus requires knowledge of the protein three dimensional structure for secondary structure assignment and cannot function with only the amino acid sequence. [<A name="ref15" href="#refl15">15</A>, <A name="ref7" href="#refl7">7</A>] However, this makes the algorithm easy to implement on a series of simulation trajectories as we have done here. Here, we used the Gromacs v.4.6.7 interface for DSSP to calculate secondary structure assignment for our simulated peptides as a function of simulation time as well as extract a distribution of the different secondary structure elements for the equilibrated protein structures. The secondary structure according to the DDSP algorithm as a function of simulation time has been presented in figure 6 for DCD-1L in varying concentrations of NaCl and in figure 8 for DCD-1L with counter ions at varying temperatures. Additionally, distributions of different secondary structure elements for DCD-1L at varying NaCl concentrations have been presented in figure 7. The retention of helical structure elements presented in figure 7 includes α-helical, 5-helical and 3-helical secondary structures. 
</P><P id="paragraph">
In the beginning of the simulation, the starting structure of the DCD-1L peptide is that of a standard right-handed α-helix. In a standard α-helix, each amino acid residue corresponds to a 100° turn in the helix and a translation of 0.15 nm along the helical axis, with the standard pitch per turn of the helix being 0.54 nm. The standard hydrogen bonding pattern of an α-helix is as follows: the N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid four residues earlier. [<A name="ref8" href="#refl8">8</A>]
</P><P id="paragraph">
Based on our simulations of DCD-1L in different NaCl concentrations, we see a rapid degradation of the α-helical structure of our peptide. However, in some cases of DCD-1L in different salt environments, some of the α-helical structure is maintained, though limited to only small, local regions of the peptide, mostly in the region 22th to 27th amino acid in the sequence.
</P><P style="text-align: left" id="paragraph"><I>Figure 6. Secondary structure assignment according the DSSP algorithm as a function of simulation time. Data for DCD-1L peptide simulated with NaCl concentrations of 20 mM, 50 mM, 70 mM, 100 mM, 120 mM, 150 mM, 200 mM, 300 mM and 500 mM is presented.</I></P><P style="text-align: left" id="paragraph"><I>Figure 7. a.) Presence of secondary structure elements in unfolded DCD-1L structures as assigned by the DSSP algorithm. b.) Presence of helical secondary structures in unfolded DCD-1L peptide as assigned by the DSSP algorithm. Data for DCD-1L peptide simulated with NaCl concentrations of 20 mM, 50 mM, 70 mM, 100 mM, 120 mM, 150 mM, 200 mM, 300 mM and 500 mM is presented</I></P><P id="paragraph">
However, more notable is the shift of the protein secondary structure from the standard α-helix to a 5-helix, also referred to as the pi helix. The 5-helix is characterized by its differing hydrogen bonding pattern from that of a typical α-helix, in which the N-H group of an amino acid forms a hydrogen bond with the C=O group of the amino acid five residues earlier. The 5-helix is thought to have originated from a destabilizing insertion of a single amino acid into an α-helix and is thus generally selected against in evolution unless it contributes to a functional advantage of the protein. [<A name="ref8" href="#refl8">8</A>] The fact that DCD-1L assumes the 5-helical structure in aqueous environments could in part explain its ability to remain antimicrobially active, as has been commonly reported [<A name="ref13" href="#refl13">13</A>, <A name="ref14" href="#refl14">14</A>]. Based on our simulations, we can additionally discern that the 5-helicity appears to be curiously confined to the first half of the peptide sequence. Interestingly, it has been previously hypothesized that this mainly cationic first half of the DCD-1L peptide is responsible for the initial interaction with the negatively charged bacterial cell membrane [<A name="ref4" href="#refl4">4</A>].
</P><P id="paragraph">
We observe somewhat similar results when we examine the effect of temperature on the DCD-1L secondary structure, with the amount of helicity that is retained varying wildly between different temperatures without any clear trend. However, it is notable that a significant amount of helicity is retained in simulations carried out at 310 K, a temperature that closely matches that of standard body temperature.
</P><P style="text-align: left" id="paragraph"><I>Figure 8. Secondary structure assignment according the DSSP algorithm as a function of simulation time. Data for DCD-1L peptide simulated with Na+ counterions at temperatures of 290 K, 300 K, 310 K and 320 K has been presented.</I></P><P id="paragraph">
Experimental examination of the secondary structure of DCD-1L in aqueous low salt environments, mainly using CD-spectroscopy, has reported similar degradation of the α-helical structure in favor of assuming an unfolded secondary structure consisting primarily of random coil, β-sheet and turn structures. However, experimental studies report no retention of the α-helicity in mostly aqueous environments. [<A name="ref12" href="#refl12">12</A>, <A name="ref6" href="#refl6">6</A>] This discrepancy can perhaps partly be explained by the rather short simulation time of 50 ns in our studies. However, 5-helices are often short in length and are commonly flanked by α-helical structures, which makes incorrect assignment of these structures as either α-helical or as turns [<A name="ref11" href="#refl11">11</A>]. It should be noted that this loss of helical structure is somewhat troubling, considering the linking of AMP helical structure to antimicrobial activity [<A name="ref2" href="#refl2">2</A>].
</P><H5>Ramachandran Plots</H5><P id="paragraph">
In a polypeptide chain, N-Cα and Cα-C bonds are relatively free to rotate. In a Ramachandran plot, the torsions of these angles are represented by the two dihedral angles Phi and Psi. The significance of examining these two dihedral angles lies in the fact that some combinations of phi and psi are forbidden due to steric hindrance, namely the collision of atoms, which allows ranking of Phi and Psi value combinations into favorable and unfavorable regions. The Ramachandran plot examines the distributions of different Phi and Psi angles in the protein structure and thus serves as an important tool in the assessment of the quality of protein three-dimensional structures, with torsion angles being the most important local structural parameters that control protein folding. The Ramachandran plot is effectively divided into four significant quadrants from right upper corner, moving counter clockwise. Quadrant I comprises of a region where some conformations are allowed and is where normally rare left-handed α-helices are located. Quadrant II is generally the most populated area of the graph and is the region that has the most favorable conformations of atoms. It includes the sterically allowed conformations for beta strands. Quadrant III contains the regions of allowed Phi and Psi combinations commonly found in right-handed helical structures and is normally the second most populated region of the plot. Quadrant IV is the least populated region of the Ramachandran plot and is generally not favored due to steric clash. Generally, clustering is found around the regions of the plot relating to α-helices and β-strands. An exception to this rule is glycine, which, due to lack of a side chain allowing high flexibility in the polypeptide chain, can assume a larger number of viable Phi-Psi combinations. This is one of the regions for the high conservation of glycine residues across protein families. [<A name="ref8" href="#refl8">8</A>] As we have mentioned earlier, Glycine is found over-expressed in the structure of DCD-1L.
</P><P id="paragraph">
Based on our simulation trajectories, we calculated the Ramachandran plots for DCD-1L over the firs 5 ns of the simulation for the starting configuration and over the last 10 ns of the simulation for the equilibrated, unfolded protein structure. The Ramachandran plots calculated over the first 5 ns of simulation exhibit strong clustering around the region commonly associated with right-handed α-helical structures, which is expected considering the starting configuration of our simulations. Based on the Ramachandran plots for DCD-1L simulated in the presence of salt in figure 9, we can observe that the α-helical starting structure of DCD-1L degrades during the 50 ns simulation and the unfolded protein structures over the last 10 ns of simulation exhibit clustering around the region commonly associated with β-strands. β-strand combinations of Phi and Psi appear most prominently in NaCl concentrations of 50 mM, 100 mM, 120 mM, and 300 mM. Additionally, some clusters occurring outside the aforementioned regions, with those in the more unfavorable regions being mostly contributed to the high Glycine content of DCD-1L. Most notably, however, we observe spreading of the clustering around α-helical phi and psi angles, with the phi angle range migrating towards more negative values, which may be indicative of the looser, 5-helical structure that we observed using DSSP secondary structure assignment.  
</P><P style="text-align: left" id="paragraph"><I>Figure 9. Ramachandran plots based on first 5 ns of simulation (left) and last 10 ns of simulation (right). Data for DCD-1L peptide simulated with NaCl concentrations of 20 mM, 50 mM, 70 mM, 100 mM, 120 mM, 150 mM, 200 mM, 300 mM and 500 mM has been presented.</I></P><P id="paragraph">
Ramachandran plots were also similarly calculated for DCD-1L simulated with Na+ counter ions while varying the system temperature. Plots for DCD-1L simulated at 290 K, 300 K, 310 K, and 320 K have been presented in figure 10. As we can see, the end structure appears to vary highly with regards to temperature, with degradation appearing the most throughout at 300 K, whereas the helical structure is mostly maintained at 310 K, which is most peculiar as this closely matches normal body temperature. The β- strand Phi-Psi combinations appear most dominantly at DCD-1L simulated at 320 K. Establishment of clear trends would require examination of a wider temperature range over a longer timescale.
</P><P style="text-align: left" id="paragraph"><I>Figure 10. Ramachandran plots based on first 5 ns of simulation (left) and last 10 ns of simulation (right). Data for DCD-1L peptide simulated with Na+ counter ions at temperatures of 290 K, 300 K, 310 K and 320 K has been presented.</I></P></DIV></DIV><DIV class="container"><DIV class="basictext"><H4>Conclusions</H4><P id="paragraph">
Through molecular dynamics simulations, using a united-atom force field, we observed the behavior of the anionic antimicrobial peptide DCD-1L in aqueous low salt environments. The study was initially motivated by the importance of understanding the behavior of DCD-1L in the additive containing water phase of our exfoliating hydrogel product. Initially, we set out to establish trends in the behavior of DCD-1L in aqueous environments when salt and temperature conditions are varied. This information could be used when planning the exact formula for our hydrogel water phase, as well as storage and transportation of our product, during which temperatures can vary greatly. Additionally, the data could prove useful when planning further antimicrobial testing, as well as production and purification protocols for our recombinant variant of the peptide.   
</P><P id="paragraph">
A number of simulations were carried out while varying the system salt concentration and temperature. The resulting trajectories were characterized thoroughly, with special emphasis placed on the evolution of the protein secondary structure. Based on our results, we find that DCD-1L assumes a mostly unfolded random coil conformation, specific make-up of which varies heavily across all examined salt and temperature conditions. Most curiously we find that in some cases, DCD-1L retains helicity in the form of a rare 5-helix, which is naturally present in just 15 % of proteins. This 5-helical structure is mostly concentrated in the first, mainly cationic half of the peptide. The cationic N-terminal half of DCD-1L has been previously postulated to be integral in the initial interaction between DCD-1L and the negatively charged bacterial cell membrane. To further investigate the importance of the helical structures we observed, studies spanning over longer time scales would be required. Additionally, it would be integral to study the interaction of the 5-helical structure with a phospholipid membrane to further unveil this new mechanism of action for DCD-1L.
</P></DIV></DIV><DIV class="container"><H3>References</H3><P id="paragraph"><A name="refl1" href="#ref1">[1]</A> Bairoch, A., Apweiler, R., Wu, C. H., Barker, W. C., Boeckmann, B., Ferro, S., . . . Yeh, L. S. (2005, Jan). The Universal Protein Resource (UniProt). <I>Nucleic Acids Res., 33</I>(Database issue), D154–159.<A name="refl2" href="#ref2">[2]</A> Brogden, K. A. (2005a, Mar). Antimicrobial peptides: pore formers or metabolic inhibitors in bacteria? <I>Nat. Rev. Microbiol., 3</I>(3), 238–250.<A name="refl3" href="#ref3">[3]</A> Brogden, K. A., De Lucca, A. J., Bland, J., &amp; Elliott, S. (1996, Jan). Isolation of an ovine pulmonary
        surfactant-associated anionic peptide bactericidal for Pasteurella haemolytica. <I>Proc. Natl. Acad.
        Sci. U.S.A., 93</I>(1), 412–416.
        <A name="refl4" href="#ref4">[4]</A> Burian, M., &amp; Schittek, B. (2015, Feb). The secrets of dermcidin action. <I>Int. J. Med. Microbiol., 305</I>(2), 283–286.<A name="refl5" href="#ref5">[5]</A> Eisenhaber, F., Lijnzaad, P., Argos, P., Sander, C., &amp; Scharf, M. (1995). The double cubic lattice method: Efficient approaches to numerical integration of surface area and volume and to dot surface contouring of molecular assemblies. <I>J. Comp. Chem., 16</I>(3), 273–284.<A name="refl6" href="#ref6">[6]</A> Jung, H. H., Yang, S. T., Sim, J. Y., Lee, S., Lee, J. Y., Kim, H. H., . . . Kim, J. I. (2010, May). Analysis of the solution structure of the human antibiotic peptide dermcidin and its interaction with phospholipid vesicles. <I>BMB Rep, 43</I>(5), 362–368.<A name="refl7" href="#ref7">[7]</A> Kabsch, W., &amp; Sander, C. (1983, Dec). Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. <I>Biopolymers, 22</I>(12), 2577–2637.<A name="refl8" href="#ref8">[8]</A> Kuriyan, J., Konforti, B., &amp; Wemmer, D. (2013). <I>The molecules of life: Physical and chemical principles.</I> Garland Science.<A name="refl9" href="#ref9">[9]</A> Lins, L., Thomas, A., &amp; Brasseur, R. (2003, Jul). Analysis of accessible surface of residues in proteins. <I>Protein Sci., 12</I>(7), 1406–1417.<A name="refl10" href="#ref10">[10]</A> Nguyen, L. T., Haney, E. F., &amp; Vogel, H. J. (2011, Sep). The expanding scope of antimicrobial peptide structures and their modes of action. <I>Trends Biotechnol., 29</I>(9), 464–472.<A name="refl11" href="#ref11">[11]</A> Pauling, L., Corey, R. B., &amp; Branson, H. R. (1951, Apr). The structure of proteins; two hydrogen-bonded helical configurations of the polypeptide chain. <I>Proc. Natl. Acad. Sci. U.S.A., 37</I>(4), 205–211.<A name="refl12" href="#ref12">[12]</A> Paulmann, M., Arnold, T., Linke, D., Ozdirekcan, S., Kopp, A., Gutsmann, T., . . . Schittek, B. (2012, Mar). Structure-activity analysis of the dermcidin-derived peptide DCD-1L, an anionic antimicrobial peptide present in human sweat. <I>J. Biol. Chem., 287</I>(11), 8434–8443.<A name="refl13" href="#ref13">[13]</A> Schittek, B., Hipfel, R., Sauer, B., Bauer, J., Kalbacher, H., Stevanovic, S., . . . Garbe, C. (2001, Dec). Dermcidin: a novel human antibiotic peptide secreted by sweat glands. <I>Nat. Immunol., 2</I>(12), 1133–1137.<A name="refl14" href="#ref14">[14]</A> Senyurek, I., Paulmann, M., Sinnberg, T., Kalbacher, H., Deeg, M., Gutsmann, T., . . . Schittek, B. (2009, Jun). Dermcidin-derived peptides show a different mode of action than the cathelicidin LL-37 against Staphylococcus aureus. <I>Antimicrob. Agents Chemother., 53</I>(6), 2499–2509.<A name="refl15" href="#ref15">[15]</A> Touw, W. G., Baakman, C., Black, J., te Beek, T. A., Krieger, E., Joosten, R. P., &amp; Vriend, G. (2015, Jan). A series of PDB-related databanks for everyday needs. <I>Nucleic Acids Res., 43</I>(Database issue), D364–368.<A name="refl16" href="#ref16">[16]</A> Vacic, V., Uversky, V. N., Dunker, A. K., &amp; Lonardi, S. (2007, Jun). Composition Profiler: a tool for discovery and visualization of amino acid composition differences. <I>BMC Bioinformatics, 8</I> , 211.</P></DIV><FOOTER><DIV class="footer-social"><DIV class="texts"><UL class="pe-social"><A href="https://2017.igem.org/Team:Aalto-Helsinki">HOME</A><A href="http://www.aalto.fi/en/">AALTO UNIVERSITY</A><A href="https://www.helsinki.fi/en">UNIVERSITY OF HELSINKI</A><A href="https://2017.igem.org/Team:Aalto-Helsinki/Map">SITE MAP</A></UL></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>