<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XMU-China_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XMU-China/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2017.igem.org/Team:XMU-China/Design</TITLE></P><DIV class="guide"><UL class="menu"><LI class="dropdown" id="Project"><A>PROJECT</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Parts">Parts</A></LI></UL><LI class="dropdown" id="Notebook"><A>NOTEBOOK</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Engineering">Engineering</A></LI><LI class="dropdown" id="Hardware"><A>HARDWARE</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Hardware">Overview</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Freeze-Dry">Freeze-Dry</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/East-Wind">East-Wind</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Accessories">Accessories</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/More_Chips">More Chips</A></LI><LI class="dropdown" id="Model"><A>MODEL</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Model">Overview</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Modeling">Modeling</A></LI><LI class="dropdown" id="HP"><A>HUMAN PRACTICES</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Human_Practices">Overview</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/HP/Silver">Silver</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/HP/Gold_Integrated">Gold</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Engagement">Engagement</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Entrepreneurship">Entrepreneurship</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Collaborations">Collaborations</A></LI><LI class="dropdown" id="other"><A>OTHER WORK</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/InterLab">Interlab</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Contribution">Contribution</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Improve">Improve</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Measurement">Measurement</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Safety">Safety</A></LI><LI class="dropdown" id="Team"><A>TEAM</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Attributions">Attributions</A></LI><LI><A href="https://2017.igem.org/Team:XMU-China/Judging">Judging</A></LI></DIV><DIV id="float"><DIV id="cebianlan"><DIV class="menu-list"><A href="#subtitle1">Ideas</A></DIV><DIV class="menu-list"><A href="#subtitle2">Sensing Device</A></DIV><DIV class="menu-list"><A href="#subtitle3">Amplifier</A></DIV><DIV class="menu-list"><A href="#subtitle4">Reporting device</A></DIV><DIV class="menu-list"><A href="#subtitle5">Future Plan</A></DIV><DIV class="menu-list"><A href="#subtitle6">References</A></DIV></DIV></DIV><DIV class="column" id="column"><DIV class="subtitle">--------* Ideas *--------</DIV><P>The pathway is as follows,
The construction of a programmable cell can be divided into sensor, amplifier, reporter three modules. The components of each module have different function.
Based on this idea, our team designed a gene pathway model. When putting the pathway into Escherichia coli, the bacterium detector of trace ions formed.</P><DIV class="subtitle">--------* Sensing Device *--------</DIV><P>The sensing device refers to the metal-sensitive promoters working on the strong positive correlation between the concentration of the specific metal and the activity intensity of the promoters. The major function of the sesing device is to detect metal ion and then input signal.
In the experiment, the metal sensitive promoters we used is ars promoter with arsR repressor gene that is sensitive to sodium arsenate or sodium arsenite. This promoters is described in detail as below,
This arsenite-sensitived sensor consists of the promoter of the E.coli JM109 chromosomal arsenic detoxification operon (ars operon), including the ArsR repressor binding site and the arsR gene encoding the arsR repressor protein, together with its ribosome binding site. Addition of any other genes to the 3' end of this part will result in their expression being dependent on the presence of sodium arsenate or sodium arsenite. Arsenite or arsenite anion binds to the repressor protein ArsR, resulting in inability to repress the promoter.
ArsR is a well-studied transcriptional repressor that regulates microbe-arsenic interactions. Most microorganisms have an arsR gene. The arsR gene is autoregulated by its product, ArsR, and is typically part of an operon that contains other ars genes involved in arsenic detoxification. The arsR repressor protein functions as a negative regulator, a membrane-associated protein need for extrusion of arsenite. It can be expressed by the arsR gene and then bind with ArsR repressor binding site in the absence of arsenate to stop the RNA polymerase from binding with the promoter. However, in the presense of arsenite, the arsR repressor protein can bind with the arsenite instead, thus the RNA polymerase can bind with the promoter to initiate transcription.
It can be seen that the commonality of the metal promoter as a sensor part of this metal ion semi-quantitative detector is that the metal ion concentration is positively correlated with the promoter activity.
But to be used in practice, the sensing device is not only limited to these three promoters. As soon as there is a positive correlation between the concentration and the activity intensity of the promoters, the metal sensitive promoters can be used as the sensing device in our semi-quantitative detection of metal ion concentration.</P><DIV class="subtitle">--------* Amplifier *--------</DIV><P>The detection limit we aim to achieve is quite low, so we designed a bio-amplifier to strengthen the signal provided by the sensing device.
T7 amplification systemIs the main feature of the amplifier system. The E.coli T7 system is regarded as the most widely used system for high-level gene expression. This system consists of a lambda DE3 lysogenic E.coli strain carrying a chromosomally integrated copy of the T7 RNA polymerase gene (gene 1) controlled by the lacUV5 promoter and a high-copy number vector allowing target gene expression from the T7 promoter. In contrast to other E.coli expression systems using host RNA polymerases for heterologous gene expression, an appropriate T7 system yields higher protein amounts since the bacteriophage RNA polymerase exhibits enhanced processivity.
T7 RNA polymerase is highly selective for its own promoters, which do not occur naturally in Escherichia coli. A relatively small amount of T7 RNA polymerase provided from a cloned copy of T7 gene 1 is sufficient to direct high-level transcription from a T7 promoter in a multicopy plasmid. Such transcription can proceed several times around the plasmid without terminating, and can be so active that transcription by E.coli RNA polymerase is greatly decreased.</P><DIV class="subtitle">--------* Reporting device *--------</DIV><P>Since the basic idea of the hardware is to measure the fluorescent signal, chemiluminescent signal or the electrochemical signal, we designed three different reporting systems including the green fluorescent protein, lacZ and luxAB. The following paragraphs will discuss and compare the three important reporting genes.
a) LacZ gene is widely used in gene expression regulation in genetic engineering. LacZ gene encoding β-galactosidase (referred to as β-gal) is composed of four subunits of tetramer, can catalyze the hydrolysis of lactose. Beta-gal is relatively stable, with X-Gal as the substrate colored blue, easy to detect and observe.
After LacZ is induced, the substrate PAPG is catalyzed by LacZ to form PAP. PAP is a kind of reducing substance which can be electrolyzed. It will oxidize and decompose under the action of a certain voltage. In theory, one molecule PAP decomposition will produce two electronic, and ultimately can produce a certain intensity of the current.
The following figure depicts the electrochemical method to detect the expression of β-galactosidase in the presence of toxicants. Initially, the E.coli cells transformed with the pUC-Ars plasmid are dielectrophoretically trapped in a small area above the embedded micro-electrodes. Due to the large molecular size ofβ-galactosidase, it will accumulate inside the concentrated cells. And then, PAPG can diffuse into the bacteria and catalyze by the β-galactosidase to generate galactopyranose and PAP. The PAP can diffuse out the bacteria and be detected via electrochemical method at 0.3V oxidation current.
b) The green fluorescent protein (GFP) is a protein composed of 238 amino acid residues (26.9 kDa) that exhibits bright green fluorescence when exposed to light in the blue to ultraviolet range. The GFP from A. victoria has a major excitation peak at a wavelength of 395 nm and a minor one at 475 nm. Its emission peak is at 509 nm, which is in the lower green portion of the visible spectrum. The fluorescence quantum yield (QY) of GFP is 0.79. GFP makes for an excellent tool in many forms of biology due to its ability to form internal chromophore without requiring any accessory cofactors, gene products, or enzymes / substrates other than molecular oxygen.In cell and molecular biology, the GFP gene is frequently used as a reporter of expression.
c) Luminous bacteria are the most abundant and widely distributed of the light-emitting organisms and are found in marine, freshwater, and terrestrial environments. What their most important feature is they can produce the luciferase called LUXAB, which can catalyzes the bioluminescence reactions. Almost all luminous bacteria have been classified into the three genera Vibrio, Photobacterium, and Xenorhabdus.In our progress, the luciferase what we use is from the Xenorhabdus luminescens.
LuxAB is a part of luxCDABEG which is the normal structure of the operon in most bioluminescent bacteria. The LuxCDE gene controls the synthesis/regenerate aldehyde and the FMNH<SPAN class="subscript">2</SPAN>, which is provided by an FMN reductase such as LuxG. The LuxAB luciferase is a heterodimeric enzyme of almost 80kDa composed of α and β-subunits whose molecular weight is 42kDa and 39kDa. For the two subunits, the α subunit plays a major role which is responsible for the light-emitting reaction and the β-subunit is important for stabling the protein, although there is about 40% identity in the amino acid sequence between the α and β subunits.
The light-emitting reaction which catalyzed by the LUXAB involves the oxidation of reduced riboflavin phosphate (FMNH<SPAN class="subscript">2</SPAN>) and a longchain fatty aldehyde with the emission of blue-green light(490nm). This reaction is as follows:
The reduced flavin, FMNH<SPAN class="subscript">2</SPAN>, bound to the enzyme, reacts with O<SPAN class="subscript">2</SPAN> to form a 4a-peroxyflavin. This complex interacts with aldehyde to form a highly stable intermediate, which decays slowly, resulting in the emission of light along with the oxidation of the substrates.
There are two ways to use the LUXAB as reporter system, in heterologous hosts such as Escherichia coli.Eithe luxAB alone may be used (in which case decanal must be provided as substrate), or luxCDABE can be used, in which case the organism can synthesise aldehyde itself.Because the E.coli is capable for reduceing the FMN to FMNH<SPAN class="subscript">2</SPAN>,so it is not necessary to add luxG as E.coli for the host.</P><DIV class="subtitle">--------* Future Plan *--------</DIV><P>Metallothioneins (MTs) is a novel kind of sulfur-based metal clusters. MTs contains some 200 amino acid residues, among them 20 Cys, and binding a total of 7 equiv of bivalent metal ions. Aromatic amino acid residues are absent. All Cys occur in the reduced form and are coordinated to the metal ions through mercaptide bonds. The abundance of Cys and their conspicuous arrangement in chelating Cys-Cys, Cys-X-Cys, and Cys-X-Y-Cys, where X and Y are residues other than Cys, predispose MT toward the binding of &quot;soft&quot; metal ions. Amino acid sequences are now known for over 36 class I MTs, for 4 class I1 MTs, and for 2 homologous sets of class 111MTs [cited in Kagi and Kojima (1987)l].
So, in order to minimize the toxic effects of metals on E.coli, metallothionein can be used as a part in the future work. One of the most common metallothionein in the prokaryotes has been submitted to the Registry: <A href="http://parts.igem.org/Part:BBa_K2310998">BBa_K2310998</A></P><DIV class="subtitle">--------* References *--------</DIV><P>
[1] Rosen BP, <EM>Families of arsenic transporters</EM>, Trends Microbiol, <STRONG>1999</STRONG>, 7, 207–212.
[2] Mukhopadhyay R, Rosen BP, Phung LT, Silver S, <EM>Microbial arsenic: from geocycles to genes and enzymes</EM>, FEMS Microbiol Rev, <STRONG>2002</STRONG>, 26, 311–325.
[3] Rosen BP, <EM>Biochemistry of arsenic detoxification</EM>, FEBS Lett, <STRONG>2002</STRONG>, 529, 86–92.
[4] Chen CM, Misra TK, Silver S, Rosen BP, <EM>Nucleotide sequence of the structural genes for an anion pump</EM>, J Biol Chem, <STRONG>1986</STRONG>, 261, 15030–15038.</P></DIV><DIV class="foot" id="foot"><P>Xiamen University, Fujian, China
No. 422, Siming South Road, Xiamen, Fujian, P. R. China 361005</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>