<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UPO-Sevilla_Project_Sensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UPO-Sevilla/Project/Sensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UPO-Sevilla/Project/Sensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Sensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Sensing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UPO-Sevilla/Project/Sensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UPO-Sevilla/Project/Sensing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="globalBC"><DIV class="header"><UL class="green"><LI><A id="home" href="https://2010.igem.org/Team:UPO-Sevilla" title="home"><SPAN>Home</SPAN></A></LI><LI onmouseover="showMenuTeam('teamList')" onmouseout="closeMenuTeam('teamList')"><A id="team" href="https://2010.igem.org/Team:UPO-Sevilla/Team" title="team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Team/Members" title="Members"><SPAN>Members &amp; Contact</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Team/Acknowledgments" title="Acknowledgments"><SPAN>Acknowledgments</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Team/Gallery" title="Gallery"><SPAN>Gallery</SPAN></A></LI></UL><LI onmouseover="showMenuProject('projectList')" onmouseout="closeMenuProject('projectList')"><A id="project" href="https://2010.igem.org/Team:UPO-Sevilla/Project" title="Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Abstract" title="Abstract"><SPAN>Abstract</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Sensing" title="Sensing"><SPAN>Sensing</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Chemotaxis" title="Chemotaxis"><SPAN>Chemotaxis</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Assays" title="Assays"><SPAN>Assays</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Results" title="Results"><SPAN>Results</SPAN></A></LI><LI onmouseover="showMenuBiobricks('biobricksList')" onmouseout="closeMenuBiobricks('biobricksList')"><A id="parts" href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks" title="Biobricks"><SPAN>Biobricks</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks/Circuits" title="Circuits"><SPAN>Circuits</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks/Devices" title="Devices"><SPAN>Devices</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks/Parts" title="Parts"><SPAN>Parts</SPAN></A></LI><LI onmouseover="showMenuModeling('modelingList')" onmouseout="closeMenuModeling('modelingList')"><A id="modeling" href="https://2010.igem.org/Team:UPO-Sevilla/Modeling" title="Modeling"><SPAN>Modeling</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Modeling/Chemotaxis" title="Chemotaxis"><SPAN>Chemotaxis</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Modeling/Signaling" title="Signaling"><SPAN>Signaling</SPAN></A></LI><LI><A id="notebook" href="https://2010.igem.org/Team:UPO-Sevilla/Notebook" title="Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A id="safety" href="https://2010.igem.org/Team:UPO-Sevilla/Safety" title="Safety"><SPAN>Safety</SPAN></A></LI></DIV><DIV class="contentBC"><H1>Introduction</H1><P>Bacteria can sense a lot of different stimuli. They have to detect changes in their environment and interpret them. Bacteria use different receptors to sense ionic state, chemicals, pH, nutrients, lights… and, when a stimuli is caused, they use an intracellular signaling pathway to produce an specific response. Usually, signals that bacteria can sense are diffusible-signals. However, we will use the first known system that senses a non-diffusible signal: the <STRONG>Prh system</STRONG> of <I>Ralstonia solanacearum</I>. This system detects an unknown polysaccharide ligand of plant cell walls and activates a signal transduction cascade that, in original system, causes virulence gene expression. </P><P>The aim of the <STRONG>sensing circuits</STRONG> is to ensure that our system is able to detect specifically the plant cells walls and causes the production of a chemoattractant through signal transduction pathways and its diffusion through cell membranes. In order to achieve effective bacterial accumulation around plant cells, we have designed four different circuits that attempt to prevent possible different problems. We have combined different regulated proteins from <I>Escherichia coli</I> and <I>Ralstonia solanacearum</I> with specific biobricks, and we have replaced virulence genes by chemoattranctant production genes. So, despite of <I>R. solanacearum</I> phito-patogenicity, the fact of working with biobricks avoids us to have to work with the security measures required by this spaecie. </P><P>A future improvement for our project would be to use an <STRONG>adhesin</STRONG> to keep the bacteria attached to the plant cell wall, which actually happend in <I>R.solanacearum</I>. We expect our system works without it.</P><H3>Brief description of <I>Ralstonia solanacearum</I>:</H3><P><I><STRONG>Ralstonia solanacearum</STRONG></I> belongs to the family of Gram-negative phyto-pathogens. This bacterium causes great losses in crops worldwide in tropical, subtropical and temperate environments. The genes involved in virulence are known as <I>hrp</I> (hypersensitive response and pathogenicity) and they are induced by contact with various plant species from three different families of dicotyledonous (Solanaceae, Cruciferae and legumes), including both host and non host species. These genes encode a type III secretion system (TTSS), required to develop the disease in the host or the hypersensitive response (HR) in resistant and non-host plants. The HR is a defense mechanism that certain plant species possess, in which cells infected by a pathogen are killed to prevent spread to healthy tissue. </P><P>The secretion system is expressed only when there is a <STRONG>physical interaction</STRONG> between <I>R. solanacearum</I> and the plant cell. PrhA (plant regulator of hrp genes), the protein that recognizes the plant cell ligand, is an outer membrane receptor that shows homology with some TonB-dependent siderophore receptors. However, PrhA is not involved in the bacteria-plant cell adhesion, but only in sensing. The interaction of bacteria with plant cells occurs in two steps: first, <I>R. solanacearum</I> binds to the cell surface. This union is independent of PrhA or any protein encoded by <I>hrp</I>. Once the union has been established, the PrhA receiver can detect an accessible ligand, which increases the transcription of the <I>hrp</I> regulatory gene. Induction of <I>hrp</I> expression is very fast, around 90 minutes, a period much shorter than generation time of <I>R. solanacearum</I> in optimal conditions. </P><P>PrhA fundamental feature is that it is the first known bacterial receptor that can detect a <STRONG>non-diffusible signal</STRONG> present in plant cell walls. The possibility of attaching bacteria to a specific tissue was what made us choose the Ralstonia system for our project. The induction of the expression of virulence hrp regulon integrates a complex signaling cascade that begins in the PrhA outer membrane protein. PrhA transduces the contact-dependent signal through a complex regulatory cascade composed of PrhR, PrhI, PrhJ, HrpG and HrpB. Finally, HrpB activates the expression of <I>hrp</I>, comprising the TTSS structural genes and genes that encode effector proteins that travel through the TTSS.</P><H1>Signal Transduction Circuits</H1><H2>Prh system of <I>Ralstonia solanacearum</I></H2><H3>Brief description of the original Prh system:</H3><P>The <STRONG>Prh system</STRONG> integrates the genes involved in the control of expression of <STRONG><I>hrp</I> virulence genes</STRONG> of <I>Ralstonia solanacearum</I>. <I>hrp</I> gene encodes a type III secretion system, necessary to develop disease in their hosts. The induction of these genes integrates a complex signaling network that begins when the bacteria and the plant cell contact. This signaling mechanism is composed of PrhA, PrhR and PrhI proteins and other regulators that, as a last resort, activate the expression of <I>hrp</I> or <I>hrc</I> (conserved hrp genes) genes.</P><P>The induction of virulence genes occurs when PrhA contacts with a plant cell. PrhA is an outer membrane protein that recognizes an unknown non-diffusible signal from the plant cell wall. PrhA-ligand binding causes that the periplasmic exposed N-terminal end of PrhA interacts with the carboxy terminal end of PrhR (an inner membrane protein) in the periplasm, transmitting the signal across the cytoplasmic membrane. In the cytoplasm inactive PrhI is actived by PrhR interaction by a still unkown mechanism.</P><P>The <I>prhIR</I> gene expression is induced in coculture with plant cells due to unknown environmental signal PrhA independent. PrhI is an ECF (extracytoplasmic function) sigma factor that, when it is activated, binds to RNA polymerase core enzyme and directs the polimerase to the promoter region of <I>PrhJ</I> gene to initiate transcription. In <I>R. solanacearum</I>, PrhJ protein induces <I>hrpG</I> transcription, which activates expression of <I>hrpB</I> gene and finally expresses <I>hrp</I> and <I>hrc</I> virulence genes.</P><P>The <STRONG>PrhA-PrhR-PrhI</STRONG> module of <I>Ralstonia</I> works similarly to FecA-FecR-FecI module of <I>E.coli</I>, with both similar sequences. PrhA shows homology with several members of the family of siderophore outer membrane receptors (as is the case of FecA). Two of the three boxes that this family of proteins presents (TonB-box, box II and boxIII) are well conserved and correctly located in PrhA. PrhR has a transmembrane domain (TM) in the same position as FecR and both proteins have a similar orientation. In addition, two of the three tryptophan residues of the N-terminal end of FecR required to activate FecI are present in PrhR. However, unlike most of the siderophores, both <I>prhIR</I> and <I>prhA</I> lack Fur-boxes which are necessary for the regulation in function of the internal iron status.</P><P>Another striking difference between PrhAIR and FecAIR is their gene organization: while there is a physical grouping between genes of FecAIR, in PrhAIR system, <I>prhA</I> constitutes a monocistronic operon at the left edge of <I>hrp</I> gene cluster and <I>prhIR</I> is on the right side of cluster, both <I>prhA</I> and <I>prhIR</I> separated by virulence genes. Moreover, in contrast to the Fec system where FecA is activated by FecI and repressed by Fur, PrhA is always expressed at very low level in the presence of the inducing signal and is PrhI independent.</P><H3>Circuit 1:</H3><P>In circuit 1 we wanted to use Prh system to transduce plant cell wall signals in chemoattractant synthesis. This first circuit integrates regulatory components PrhA-PrhI-PrhR and PprhJ of <I>R. solanacearum,</I> but they are transfered to <I>E. coli.</I> Genes required for synthesis and excretion of the chemoattractant are under PprhJ promoter control.</P><P>Because the Prh system is not fully characterized, unknown elements could be involved in, preventing the correct transmittion of signals to P<I>prhJ</I>. Also, it is possible that it could not perform its usual answers when expressed in <I>E.coli</I>. For example, it could have problems setting PrhA protein in the outer membrane. For those reasons, we have designed other circuits wich use <I>E. coli</I> proteins in the signal cascade.</P><H2>Fec system of <I>Escherichia coli</I></H2><H3>Brief description of the original Fec system:</H3><P><STRONG>Fec system</STRONG> includes genes involved in regulation and expression of <I>E. coli</I> iron transporters. <I>fecABCDE</I> genes express the <STRONG>ferric citrate transporter</STRONG> when bacteria iron status is low or deficient. The induction of genes <I>fecABCDE</I> integrates a signaling cascade that begins at the cell surface and is extended to the cytoplasm. To do this, three specific proteins are involved : FecA in outer membrane, FecR in cytoplasmic membrane and FecI in cytoplasm. This module <STRONG>FecA-FecR-FecI</STRONG> is known as a signal transduction system between three compartments (outside, periplasm and cytoplasm).</P><P>The signaling pathway begins when the outer membrane receptor FecA binds to its ligand, ferric dicitrate. This binding causes structural changes in FecA that allow the interaction of its amino terminal end to the carboxy terminal end of FecR in the periplasm. FecR, a transmembrane protein, transmits the signal to the cytoplasm, where it activates FecI. FecI is an extracitoplasmatic function (ECF) sigma factor that, when activated, binds to core RNA polymerase and directs the complex to the upstream promoter of <I>fecABCDE</I> transport genes to initiate transcription. </P><P>In addition, the transcription of regulatory genes <I>fecIR</I> is controlled by the internal iron status through the Fur repressor. When the Fur protein is loaded with Fe 2+, it represses <I>fecIR</I> transcription and prevents the <I>fec</I> gene expression. Therefore, the <I>fec</I> transport gene transcription is subjected to a double control: first, cells detect iron deficiency. Then, regulatory proteins FecI and FecR are synthesized, which, if ferric citrate is available, initiate the transcription of <I>fec</I> transport genes.</P><P>Dicitrate ferric transport through the outer membrane requires an energy transduction complex consisting of TonB, ExbB and Exb cytoplasmic membrane proteins.</P><H3>Circuit 2:</H3><P>In our second circuit, the iron transport genes (<I>fecABCDE</I>) are replaced by genes required for synthesis and excretion of chemoattractant. Those genes remain under control of the PfecABCDE promoter, being regulated by the FecA-FecI-FecR module, which depends on internal iron status and external ferric citrate concentration.</P><P>This circuit has the advantage that, besides being well characterized, is present in wild type <I>E. coli</I>. However, this system is not specifically directed to plant tissues and would be regulated by iron status of the bacteria and the environment. Nevertheless, the second signal transduction circuit could be use as a control of the chemoattractant synthesis. This way, if the plant cell wall signal is not properly transduced, we could induce the chemoattractant synthesis by changing medium conditions.</P><H2>(FecA/PrhA)-FecI-FecR hybrid protein system</H2><H3>Circuit 3:</H3><P>Our third circuit uses the FecA and PrhA <STRONG>sequence homology</STRONG>. We have designed an <STRONG>hybrid protein</STRONG> in order to detect the plant cell ligands and transmit the signal to FecR. The hybrid protein contains most of PrhA and the N-terminal end of FecA; binding both proteins by a shared sequence near the Ton-box. The signal would be transmited through the interaction between the periplasmic exposed N-terminal extension of FecA and the C-terminal part of FecR.</P><P>The third circuit would allow us to sense a non-diffusible signal and to transduce it using an <I>E. coli</I> system without problems of expression and function.</P><P>We have focused in this circuit. Below you can see a detailed description of hybrid protein structure.</P><H2>PrhA-fecI-FecR hybrid system</H2><H3>Circuit 4:</H3><P>Due to the proximity in the life tree, the similarities between N-terminal extension of PrhA and FecA is significant, in particular the amino acid sequence Gx10(L,A)L(D,Q,A)G(S,T)L is well conserved. Also PrhR shows sequence similarity with FecR (27% identity, 43% similarity). Cause this information we wanted to test if the interaction between these systems was possible without modification.</P><P>However, this construction is largely a test and we had not enough time to permorm it.</P><H1>Outer Membrane Protein Structures</H1><P>Now it is going to be shown the structure of the outer membrane proteins which starts the signal transduction in sensing systems described before. It is interesting to study the structure and domains of FecA and PrhA before seeing the hybrid protein, in which we have focused our project mainly.</P><H2>Fe(3+) dicitrate transport protein FecA</H2><P class="caption"><B><A src="http://www.pdb.org/pdb/explore/explore.do?pdbId=1KMO" target="_blank">Crystal structure of the Outer Membrane Transporter FecA.</A></B></P><P><A href="http://www.uniprot.org/uniprot/P13036" target="_blank">FecA</A> is the outer membrane receptor protein in the Fe(3+) dicitrate transport system of <I>Escherichia coli</I>. It binds and transports ferric citrate, and it is required to initiate transcription of the <I>fecABCDE</I> transport operon but not the regulatory fecIR genes. This is a well-known protein, compound of 773 amino acids, whose main domains are shown below:</P><P>The yellow left domain represents a <STRONG>signal peptide</STRONG> which takes from 1st to 33rd codon. The cleavage site of the signal peptidase has been found between residues 33 and 34<SUP><A href="#Reference_Sensing01">[1]</A></SUP>. Its function is to drive FecA protein to the outer membrane of <I>E. coli</I>, where the protein works.</P><P>The green illustration represents <STRONG>Secretin and TonB N-terminus Short Domain</STRONG> which takes from 57th to 107th codon. This domain is found at the N-terminus of the Secretins of the bacterial type II/III secretory system as well as the TonB-dependent receptor proteins. These proteins are involved in TonB-dependent active uptake of selective substrates. Thus, FecA interacts with TonB, which couples the electrochemical potential of the cytoplasmic membrane to active transport of ferric citrate across the outer membrane. The TonB box undergoes a substrate-induced disorder transition which produces an aqueous exposed, highly disordered protein fragment, which probably regulates transporter–TonB interactions<SUP><A href="#Reference_Sensing02">[2]</A></SUP>.</P><P>It is usual to find the TonB domain nearby signal and Plug domains. It is a common domain organization. At the left it is shown the structure of the periplasmic signaling domain of FecA by nuclear magnetic resonance. </P><P>Between both before domains it is a flexible 79-residue domain of FecA termed the <STRONG>NH2-terminal extension</STRONG>, which resides entirely within the periplasm. Its function is proposed to be to transmit the liganded status of the receptor to FecR<SUP><A href="#Reference_Sensing03">[3]</A></SUP>.</P><P>In red color in the schematic representation it is shown the <STRONG>TonB-dependent Receptor Plug Domain</STRONG> which takes from 129th to 244th codon. The Plug domain has been shown to be an independently folding subunit of the TonB-dependent receptors. It acts as the channel gate, blocking the pore until the channel is bound by ligand. At this point it undergoes conformational changes that open the channel. Also ligand induces allosteric transitions which are propagated through the outer membrane by the plug domain, signaling the occupancy of the receptor in the periplasm. The plug domain is located inside a barrel, comprising five helixes, two β strands, and a mixed four-stranded β sheet. Also three loops of the Plug domain extend above the plane of the upper leaflet of the outer membrane<SUP><A href="#Reference_Sensing03">[3]</A></SUP>.</P><P class="caption"><B>a.</B>Crystal structure of ferric citrate transporter FecA in the unliganded form <B>b.</B>Crystal structure of the outer membrane transporter FecA complexed with ferric citrate</P><P>Finally in the C-terminus there is a <STRONG>TonB Dependent Receptor Domain</STRONG> which takes from 525th to 773rd codon. The TonB dependent receptor domain is included in the 22-stranded β barrel that traverse de outer membrane. The barrel of a TonB dependent receptor is a dynamic entity that actively participates in the energy-dependent siderophore uptake. This barrel has elipsoidal shape as you can see in before representations of FecA. Below it is shown the C-terminal domain of FecA, from 525th codon to the end.</P><P class="caption">C-Terminal domain of FecA (representation made with RasWin program)</P><P>The common domain organization represents TonB dependent receptor domain at the same time as Plug domain because the interaction between the receptor (FecA) and the ligand (dinuclear ferric citrate molecule) is performed by Plug domain and the barrel. Formation of the liganded complex carries out changes on the conformation of the barrel and the Plug domain of FecA.</P><H2>Outer membrane receptor protein PrhA</H2><P>PrhA is the only known protein able to detect a non-diffusible signal and transduce this information into the cell. It is compound of 770 amino acids and it was found not too long ago. This is why there is not many information about it. Not being well-known is a point to use the hybrid protein FecA/PrhA instead of it. Anyway, their main domains are shown in <A href="http://pfam.sanger.ac.uk/protein?acc=B7ZJG7" target="_blank">Pfam website</A>, but it is not possible to see its structure because it has not been modeled yet.</P><P>Like in the case of FecA, PrhA has a putative <STRONG>signal peptide</STRONG> which takes from 1st to 35th codon. Its function would be direct PrhA to the outer membrane of <I>Rastonia solanacearum</I>. Despite its existence, you can not see it in the domain summary picture since it has not been well studied.</P><P>Next it is a not confirmed domain with unknown function which would take from the beginning of the protein to 130th amino acid. By now, it is called <STRONG>PfamB PB000342</STRONG> and its family was generated automatically from an alignment taken from Automatic Domain Decomposition Algorithm (<A href="http://ekhidna.biocenter.helsinki.fi/sqgraph/pairsdb/index_html" target="_blank">ADDA</A>). Since PrhA interacts with PrhR using its periplasmic domain, it is expected that this domain performs that function.</P><P>Then, PrhA presents the same domains that FecA: <STRONG>TonB-dependent Receptor Plug Domain</STRONG> (154 – 250 aa.) and <STRONG>TonB-dependent Receptor Domain</STRONG>(542 – 767 aa.), setting out the high similarities that exist between these two outer membrane proteins. Also their N-terminal extensions are quite similar as it was found by Marenda et al<SUP><A href="#Reference_Sensing04">[4]</A></SUP>.</P><P class="caption"><STRONG> Phylogenetic tree of TonB-dependent receptors </STRONG>. The tree was constructed as described by Rakin et al. (1994). Circled numbers indicate the number of times (from the whole 100) a particular node was supported by bootstrap analysis. The proteins used in this analysis are referenced in Rakin et al. (1994).<SUP><A href="#Reference_Sensing04">[4]</A></SUP></P><P>PrhA has high similarities with TonB-dependent receptors, which needs to interact with the TonB protein to perform their functions. It shares two of the three main domains those proteins have. Nevertheless, PrhA is lacking of the periplasmic Secretin and TonB N-terminus Short Domain, the necessary domain to interact with TonB. In its place there is an unknown domain still not well studied. It will be required to continue studying this protein to know if TonB is necessary in its function and understand the evolution that TonB interaction domain suffered.</P><H2>Hybrid protein FecA/PrhA</H2><P>Taking in advance that Prh system is not naturally expressed in <I>Escherichia coli</I> and that Prh system is not well-known, we decided to create a fusion protein. FecA/PrhA artificial coding sequence spans the first 92 codons of <I>fecA</I>, encoding the signal peptide, NH2 terminal extension and the proposed Ton-box, fused to the distal end of the <I>prhA</I> coding sequence at the conserved GSGL motif (aa. 89-92). We synthesized this biobrick using MrGene services, so we also optimized the sequence to be expressed in <I>Escherichia coli</I>.</P><P>Above is shown approximately the point where we fused FecA and PrhA proteins. To this way the hybrid protein include domains from both OM proteins:</P><UL><LI><STRONG>Signal peptide of FecA</STRONG> which will help to the accurate emplacement of the hybrid protein in the outer membrane of <I>E. coli</I>.</LI><LI><STRONG>NH2-terminal extension of FecA</STRONG>. Including this domain of FecA means including the periplasmic signaling domain of this protein. The signal transfer between the OM and the IM proteins is performed between the N-terminus of FecA and the C-terminus of FecR (both are shown in the periplasmic). The hybrid protein includes the N-terminus of FecA so our expectation is that FecA/PrhA protein was able to interact with FecR.</LI><LI>Most of the <STRONG>Secretin and TonB N-terminus Short Domain of FecA</STRONG>. This domain helps FecA to interact with TonB. If TonB interaction is required for the OM-IM signal transfer, our hybrid protein includes this domain. Also, doing this fusion the hybrid protein loses the unknown function domain set in the N-terminus of PrhA. <P class="caption"><STRONG>N-terminus</STRONG> (aa. 34-92) <STRONG>of FecA.</STRONG> Here is shown the FecA contribution to the hybrid protein. Our aim is that this structure was able to interact with FecR without the rest of FecA protein. Representation made with RasWin.</P></LI><LI><STRONG>TonB-dependent Receptor Plug Domain of PrhA</STRONG>. In 89<SUP>th</SUP> codon there is a conserved motif which was used to fuse FecA with PrhA. The function of the Plug domain is to propagate allosteric transitions through the outer membrane signaling the occupancy of the receptor.</LI><LI><STRONG>TonB Dependent Receptor Domain of PrhA</STRONG>. From the conserved motif GSGL to the end of the protein amino acids are the same that in PrhA protein. The hybrid protein includes most of the PrhA protein, from 92<SUP>sd</SUP> codon to the C-terminus. The aim of it is that FecA/PrhA was able to interact with the non-diffusible plant wall signal that PrhA detects. The mechanism of this interaction is unknown.</LI></UL><P>As you can see we work with a lot of uncertainty cause of the unknown mechanisms that manage the process we work with. Anyhow, we hope that this hybrid protein allows to sense non-difusible signals (with PrhA domains) and to transduce it by the Fec pathway (using the N-terminus of FecA). If this happened we would not have any problem with other Prh protein because the signal would continue by FecR and FecI in the Fec pathway of <I>E. coli<I>.</I></I></P><H1>Bibliography</H1><OL><LI id="Reference_Sensing01">Uwe Pressler, Horst Staudenmaier, Luitgard Zimmermann, And Volkmar Braun (1988), Genetics of the Iron Dicitrate Transport System of Escherichia coli. JOURNAL OF BACTERIOLOGY, June 1988, p. 2716-2724</LI><LI id="Reference_Sensing02">Miyeon Kim, Gail E. Fanucci, and David S. Cafiso (2007), Substrate-dependent transmembrane signaling in TonB-dependent transporters is not conserved. PNAS, July 17, 2007, vol. 104, no. 29, 11975–11980.</LI><LI id="Reference_Sensing03">Andrew D. Ferguson, et al (2002). Structural Basis of Gating by the Outer Membrane Transporter FecA. Sience 295, 1715.</LI><LI id="Reference_Sensing04">Marc Marenda, Belen Brito, Didier Callard, Stéphane Genin, Patrick Barberis, Christian Boucher and Matthieu Arlat (1998). PrhA controls a novel regulatory pathway required for the specific induction of Ralstonia solanacearum hrp genes in the presence of plant cells. Molecular Microbiology (1998) 27(2), 437–453.</LI></OL><UL><LI>Brito, B., Marenda, M., Barberis, P., Boucher, C., and Genin, S. 1999. <I>prhJ and hrpG: Two new components of the plant signal-dependent regulatory cascade controlled by PrhA in Ralstonia solanacearum</I>. Mol. Microbiol. 31:237-251.</LI><LI>Marenda, M., Brito, B., Callard, D., Genin, S., Barberis, P., Boucher, C. A., and Arlat, M. 1998. <I>PrhA controls a novel regulatory pathway required for the specific induction of Ralstonia solanacearum hrp genes in the presence of plant cells</I>. Mol. Microbiol. 27:437-453.</LI><LI>Aldon, D., Brito, B., Boucher, C., and Genin, S. 2000. <I>A bacterial sensor of plant cell contact controls the transcriptional induction of Ralstonia solanacearum pathogenicity genes</I>. EMBO (Eur. Mol. Biol. Organ.) J. 19:2304-2314.</LI><LI>Brito, B., Aldon, D., Barberis, P., Boucher, C., and Genin, S. 2002. <I>A Signal Transfer System Through Three Compartments Transduces the Plant Cell Contact-Dependent Signal Controlling Ralstonia solanacearum hrp Genes</I>. Molecular Plant-Microbe Interactions. Vol. 15, No. 2: 109/119</LI><LI>Braun V, Mahren S, Sauter A. <I>Gene regulation by transmembrane signaling</I>. 2006. Biometals. 19(2):103-13</LI><LI>Braun V, Mahren S, Ogierman M. 2003. <I>Regulation of the FecI-type ECF sigma factor by transmembrane signalling</I>. Curr Opin Microbiol. 6(2):173-80.</LI><LI>Enz, S., Brand, H., Orellana, C., Mahren, S., and Braun, V. 2003. <I>Sites of Interaction between the FecA and FecR Signal Transduction Proteins of Ferric Citrate Transport in Escherichia coli</I> K-12. J. Bacteriol. Vol. 185, No: 133745–3752 </LI><LI>Kim, M., Fanucci, G. E., and Cafiso, D. S. 2007. <I>Substrate-dependent transmembrane signaling in TonB-dependent transporters is not conserved</I>. PNAS. Vol. 104  N. 29: 11975/11980</LI><LI>http://www.mikrobio.uni-tuebingen.de/ag_braun/research_areas.html</LI><LI>www.uniprot.org </LI><LI>http://pfam.sanger.ac.uk/ </LI></UL><A class="return_button" href="/Team:UPO-Sevilla/Project" title="Project"><SPAN>Return to Project</SPAN></A></DIV><TITLE>Footer</TITLE><DIV class="footer"><H1>Sponsors</H1></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UPO-Sevilla/Project/Sensing">http://2010.igem.org/Team:UPO-Sevilla/Project/Sensing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UPO-Sevilla/Project/Sensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UPO-Sevilla/Project/Sensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Sensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Sensing&amp;oldid=166367" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>