<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UPO-Sevilla_Project_Chemotaxis"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UPO-Sevilla/Project/Chemotaxis">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UPO-Sevilla/Project/Chemotaxis&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Chemotaxis&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Chemotaxis&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UPO-Sevilla/Project/Chemotaxis" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UPO-Sevilla/Project/Chemotaxis</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV class="globalBC"><DIV class="header"><UL class="green"><LI><A id="home" href="https://2010.igem.org/Team:UPO-Sevilla" title="home"><SPAN>Home</SPAN></A></LI><LI onmouseover="showMenuTeam('teamList')" onmouseout="closeMenuTeam('teamList')"><A id="team" href="https://2010.igem.org/Team:UPO-Sevilla/Team" title="team"><SPAN>Team</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Team/Members" title="Members"><SPAN>Members &amp; Contact</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Team/Acknowledgments" title="Acknowledgments"><SPAN>Acknowledgments</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Team/Gallery" title="Gallery"><SPAN>Gallery</SPAN></A></LI></UL><LI onmouseover="showMenuProject('projectList')" onmouseout="closeMenuProject('projectList')"><A id="project" href="https://2010.igem.org/Team:UPO-Sevilla/Project" title="Project"><SPAN>Project</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Abstract" title="Abstract"><SPAN>Abstract</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Sensing" title="Sensing"><SPAN>Sensing</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Chemotaxis" title="Chemotaxis"><SPAN>Chemotaxis</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Assays" title="Assays"><SPAN>Assays</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Project/Results" title="Results"><SPAN>Results</SPAN></A></LI><LI onmouseover="showMenuBiobricks('biobricksList')" onmouseout="closeMenuBiobricks('biobricksList')"><A id="parts" href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks" title="Biobricks"><SPAN>Biobricks</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks/Circuits" title="Circuits"><SPAN>Circuits</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks/Devices" title="Devices"><SPAN>Devices</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Biobricks/Parts" title="Parts"><SPAN>Parts</SPAN></A></LI><LI onmouseover="showMenuModeling('modelingList')" onmouseout="closeMenuModeling('modelingList')"><A id="modeling" href="https://2010.igem.org/Team:UPO-Sevilla/Modeling" title="Modeling"><SPAN>Modeling</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Modeling/Chemotaxis" title="Chemotaxis"><SPAN>Chemotaxis</SPAN></A></LI><LI><A href="https://2010.igem.org/Team:UPO-Sevilla/Modeling/Signaling" title="Signaling"><SPAN>Signaling</SPAN></A></LI><LI><A id="notebook" href="https://2010.igem.org/Team:UPO-Sevilla/Notebook" title="Notebook"><SPAN>Notebook</SPAN></A></LI><LI><A id="safety" href="https://2010.igem.org/Team:UPO-Sevilla/Safety" title="Safety"><SPAN>Safety</SPAN></A></LI></DIV><DIV class="contentBC"><H1>Introduction</H1><P>The term <STRONG>chemotaxis</STRONG> is defined as the process in which organisms, both unicellular and multicellular, move toward or away from a chemical source that is present in the environment. Depending on their behavior to the chemical, chemotaxis can be positive, if cellular movements go toward higher concentration of the chemical, or negative, if they attempt to move away. In the first case, the chemical is called chemoattractant, in the second case, chemorepellent. Therefore, whereas the outcome of positive chemotaxis used to be the accumulation of organisms or cells in regions with higher concentration of chemoattractant, the outcome of negative chemotaxis would usually be separation and dispersal of them from the chemorepellent. Generally, this behavior is a consequence of using the chemoattractant as a nutrient, or due to the chemorepellent are excretion products or toxic. However, chemicals are not the only stimuli that cells and organisms can sense. They also are able to detect others stimuli like light, temperature, touch, electric fields, etc.</P><P>Bacteria have the capacity to sense a great diversity of <STRONG>chemical stimuli</STRONG>. Usually, there are families of substances that only act as chemoattractants and others that only do as chemorepelents, but this is not a primal rule. For example, <I>E. coli</I> has positive chemotaxis to sugars, oxygen, weak bases, dipeptids, whereas it present negative chemotaxis to alcohols, weak organic acids, inorganic ions and extreme extracellular pH values. As for amino acids, exist a division in their action and they exhibit a gradation. <I>E. coli</I> is strongly attracted to L-alanine, L-asparagine, L-aspatate, L-cystein, L-glutamate, glycine and L-serine, but is repelled from hydrophobic amino acids like L-isoleucine, L-leucine, L-phenylalanine, L-tryptophan and L-valine. Chemotactic response of <I>E. coli</I> to amino acids involves two MCPs (methyl-accepting chemotaxis proteins), which are the products of tar and tsr genes. In this case, <I>E. coli</I> and <I>Salmonella</I> are very similar, whereas <I>B. subtillis</I> and <I>Pseudomonas aeruginosa</I> are positive chemotatic to the common twenty L-amino acids. Sometimes taxis toward L-amino acids depends on the nitrogen availability.</P><P><STRONG>Chemotactic response</STRONG> is characterized by great sensitivity, huge dynamism and precise adaptation, that allow <I>E. coli</I> to amplify the sensed stimuli up to 50-fold. In bacteria as <I>E. coli</I> and <I>Salmonella</I>, some chemotactic stimuli bind directly to chemotaxis-specific receptors, whereas others bind first to a primary receptor, which then interact with the respective receptor. There are several genes involved in taxis: those that encodes chemotatic receptors for specific components (receptors, transducers and Che genes). <I>E. coli</I> tactic receptors are the product of five partially redundant genes: <I>tsr</I>, <I>tar</I>, <I>trg</I>, <I>tap</I> and <I>aer</I>., that sense a wide variety of chemicals. <I>Vibrio cholerae</I> has 43, <I>H. salinarum</I> has 3 subfamilies of MCPs, <I>M. xanthus</I> only one, and <I>B. subtilis</I> has ten.</P><P>The <STRONG>receptor complex</STRONG> is composed by MCP-CheW-CheA. Usually, <STRONG>MCPs</STRONG> are clustered at the bacterial poles (one or two). At least, MCPs have three main functions related to bacterial chemotaxis: they bind the ligands, they transduce the chemotaxis signal across the cytoplasmic membrane and they undergo methylation or demethylation (adaptation process). Received information is transduced by a group of proteins that are coordinated by phosphorilation-dephosphorilation reactions (CheW, CheA and CheY), and act on the flagellar motor changing its direction of rotation (clockwise or counterclockwise). The methylation-demethylation reactions (focusing in Tar MCP) take place in a carboxilic group of glutamate and glutamine residues as part of a chain reaction, which is carry out by the enzyme methyltransferase CheR, and the opposite reaction is carried out by the enzyme methylesterase CheB. As for chemorepellent, in most cases, the ligands bind to specific receptor, although sometimes this chemicals cause cellular perturbations that directly act on MCPs.</P><P class="caption">Diagram of the chemotactic pathway in E. coli. (Figure by MIT OCW. After figure 4 in Falke, J. J., R. B. Bass, S. L. Butler, S. A. Chervitz, and M. A. Danielson. &quot;The Two-component Signaling Pathway of Bacterial Chemotaxis: A Molecular View of Signal Transduction by Receptors, Kinases, and Adaptation Enzymes.&quot; In Annu Rev Cell Dev Biol. 13 (1997): 457-512.)</P><P>In bacteria, cell motility are a consequence of <STRONG>flagella rotation</STRONG> (in flagellated bacteria), which let them move in liquid mediums, and sometimes in solid mediums (agar plates). The name of these processes are <I>swimming</I> and <I>swarming</I>. However, there are others types of movements like: <I>gliding movements</I> (when cells have no cilium or flagella), <I>twitching movements</I> (intermittent, directionless and uncoordinated movement). <I>Pseudomonas</I> motility is due to the rotation of an only flagellum, <I>H. salinarum</I> motility is caused by a bunch of polarized flagella, but <I>Escherichia coli</I>, <I>Salmonella</I> y <I>Bacillus subtillis</I> have lots of flagella that are randomly spread out all over the membrane. Most of the time all flagella rotate in the same direction, provoking straight movements called <I>run</I>, but when the direction changes the bacteria begin to go round (<I>tumbles</I>).</P><P>Chemotaxis is a widespread phenomenon in organisms, because their natural tendency is approach to beneficial environments, according to their requirements, and avoid from damaging ones. Although it seems a very basic and easy process that only involves changes in speed and direction of rotation, actually it is a <STRONG>complex behavior</STRONG> that involves the continuous integration of extracellular stimuli and their intracellular coordination to respond fairly accurately.</P><P>Many bacteria use the chemotaxis to interact with both animal and plant hosts and chemotaxis plays an important role in the fitness and virulence of bacteria. Most plant-associated bacteria have swimming motility, which let them reaches plant tissues and causes invasion and colonization. Pathogenic bacteria are specifically attracted to diverse amino acids, sugars, aromatics, secondary metabolites and organic acids, natural component of plant exudates, that may act like chemoattractants. For example, sugars attract <I>Rizhobium leguminosarum</I>, <I>Azospirullum brasilense</I> and <I>Agrobacterium tumefaciens</I>, but not <I>Erwinia amylovora</I> or <I>Pseudomonas fluorescens</I>. Thus, chemotactic responses may be differentially selected traits that confer adaptation to various host and ecological conditions.</P><H1>Chemoattractants</H1><H2>Aspartate</H2><P>Also called aspartic acid, it is synthesize in a transamination reaction between oxalacetate and glutamate, thanks to the enzyme aspartate aminotransferase, in the Krebs cycle. Aspartate can also be produce from fumarate, using the enzyme aspartate ammonia lyase. In microorganisms, aspartate is the precursor to several amino acids, including methionine, threonine, lysine, isoleucine and asparagine. It is involve in others biological cycles ase in inosine synthesis (precursor of purine bases).</P><P>Triggering aspartate biosynthesis will be possible thanks to a new device, that will contain an specific promoter (PprhJ or PfecA) and an ORF which encodes for <STRONG>aspartate ammonia lyase</STRONG>. Over-expression of this enzyme will increase the amount of aspartate produced by <I><STRONG>E. coli</STRONG></I>. It is suppose that its concentration reach such levels that aspartate will spread through bacteria membranes. This construction would be made in a high copy plasmid.</P><H2>Glutamate</H2><P>It is a non-essential amino acid, that is denominated glutamic acid and play a key role in cellular metabolism, because its implication in some transamination reactions and its role as transporter of ammonia. Glutamate is produced in TCA, from 2-oxoglutarate, in the reaction catalyzed by glutamate deshidrogenase. However, this is not the only production route, because can also be synthesized from glutamine, in the reaction catalyzed by glutamate synthase, that generate two molecules of glutamate.</P><P>The procedure that will be employed to regulate and increase glutamate production will lie in overexpression of the two subunits that make up the enzyme <STRONG>glutamate synthase</STRONG>, called <STRONG>GltD</STRONG> and <STRONG>GltB</STRONG>, both under control of the promoter PprhJ/PfecA. This way, it is expected a higher flux in this route, that would produce an important amount of glutamate from glutamine, and that will be subordinated to activation by PrhI/PfecA. It is suppose that its concentration reach such levels that aspartate will spread through bacteria membranes, because there is no glutamate exporter known in <I><STRONG>E. coli</STRONG></I>. This construction would be made in a high copy plasmid.</P>
Glutamate causes lower chemotactic response than aspartate in Escherichia coli. That is why finally we focused on building aspartate and saliccylate devices and forgot glutamate as attractant to <I>E. coli.</I><H2>Salicylate</H2><P>Also known as salicylic acid, salicylate is a beta hydroxy acid that is used in organic synthesis and as a phytohormone. Salycilate is biosynthesized from the amino acid phenylalanine. In our proyect we will use another route, that involves two enzymes: <STRONG>isochorismate pyruvate lyase</STRONG> and <STRONG>isochorismate synthase</STRONG>. This chemical has a lot of medicinal uses. </P><P>In our project, there would be used a designed device, under control of PprhJ/PfecA promoter, that converts chorismate into salicylate, because of the production of <STRONG>PchB</STRONG> and <STRONG>PchA</STRONG> enzymes. This fact will only happened in response to bacteria interaction with plant cell walls, in the same way that the other chemoattractants. It is suppose that its concentration reach such levels that aspartate will spread through bacteria membranes. This construction would be made in a high copy plasmid.</P><P>Salicylate is a chemoattractant for many bacterias but not for <I>E. coli.</I> That is why using this chemoattractant in our circuit it is necessary the utilization of <I><STRONG>Pseudomonas putida G7</STRONG></I>, a strain that is naturally chemotactic to salicylate, as chemotactic population, while <I><STRONG>E. coli</STRONG></I> play the role of sensing population.</P><H1>References</H1><UL><LI>Eisenbach M, Lengeler J W, Varon M, Gutnick D, Meili R, Firtel R A, Segall J E, Omann G M, Tamada A, Murakami F. (2004) Chemotaxis. London: Imperial College Press. </LI><LI>Taylor B L, Zhulin I B, Johnson M S. (1999) <I>Aerotaxis and Other Energy-Sensing Behavior in Bacteria</I>. Annual Review of Microbiology. 53:103-28.</LI><LI>Mesibov R, Adler J. (1972) <I>Chemotaxis Toward Amino Acids in Escherichia coli</I>. Journal of Bacteriology. 112 (1):315-326.</LI><LI>Sourjik V, Berg H C. (2002) <I>Receptor sensitivity in bacterial chemotaxis</I>. PNAS. 99 (1): 123-127.</LI><LI>Taguchi K, Fukutomi H, Kuroda A, Kato J, Ohtake H. (1997) <I>Genetic identification of chemotactic transducers for amino acids in Pseudomonas aeruginosa</I>. Microbiology. 143: 3223-3229.</LI><LI>Hazelbauer G L, Falke J J, Parkinson J S. (2008) <I>Bacterial chemoreceptors: high-performance signaling in networked arrays</I>. Trends Biochem Sci. 33 (1): 9-19.</LI><LI>Yao J, Allen C. (2006) <I>Chemotaxis Is Required for Virulence and Competitive Fitness of the Bacterial Wilt Pathogen Ralstonia solanacearum</I>. Journal of Bacteriology. 188 (10): 3697-3708.</LI><LI>Baker M B, Wolanin P M, Stock J B. (2006) <I>System Biology of Bacterial Chemotaxis</I>. Current Opinion in Microbiology. 9: 187-192</LI><LI>Ikeda M. (2003) <I>Amino Acid Production Processes</I>. Advances in Biochemical Engineering/Biotechnology. 97.</LI><LI>Sano C. (2009) <I>History of glutamate production</I>. Am J Clin Nutr. 90(suppl):728S-32S.</LI><LI>Kimura E. (2003) <I>Metabolic Engineering of Glutamate Production</I>. Advances in Biochemical Engineering/Biotechnology. 97.</LI></UL><A class="return_button" href="/Team:UPO-Sevilla/Project" title="Project"><SPAN>Return to Project</SPAN></A></DIV><TITLE>Footer</TITLE><DIV class="footer"><H1>Sponsors</H1></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:UPO-Sevilla/Project/Chemotaxis">http://2010.igem.org/Team:UPO-Sevilla/Project/Chemotaxis</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UPO-Sevilla/Project/Chemotaxis" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UPO-Sevilla/Project/Chemotaxis" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Chemotaxis&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UPO-Sevilla/Project/Chemotaxis&amp;oldid=151489" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>