"
Page
Discussion
View source
History
teams
Log in
 
Team:UIUC-Illinois/Project
From 2011.igem.org
University of Illinois iGEM Team
Illinois iGEM
Home
Team
Project
Documentation
Human Practices
E. chiver
Project Navigation
Overall View
Lambda System
Lambda/P21 Concept
Protocol
Parts Submitted
Who We Are
Amanda Chang
"A watched gel never runs"
Track Selection: New Application
Abstract
Our project, E. chiver, drew inspiration from the commonly used CRIM system, a series of plasmids that allows the user to integrate constructs into lambdoid phage sites common to many bacterial chromosomes.  Our E. chiver system adds several elements yielding new applications.  Our team designed two E. chiver constructs utilizing Lambda and P21 machinery.  Each can in theory be used to shuttle a plasmid construct between two forms:  a single chromosomal insert and a high copy number plasmid.   In their current designs the systems must function separately, but possible routes have been identified by our team to make the co-functioning of these systems possible.  We can see elements of our project being used in drug delivery systems as a method to keep a gene of interest dormant unless in the correct condition/location, and with further exploration into the co-functioning routes it may be used to create a ‘bacterial filing cabinet’.
Project Goal:
To create a system that allows a construct to be expressed at a high copy number when induced, but then file back into the chromosome as a stable integrant for energetically favorable storage. The ultimate end goal is to create a bacterial filing system, E. chiver, which allows the coexistence of multiple constructs with their copy number and integration tied to a unique inducer.
Illustrative Overview of Goal:
Analogy:
A simple analogy for our E. chiver project is a filing cabinet.  The following are the basic components of the system and their workplace analogues.
Construct = Shuttle Plasmid = File
Gene of interest = File Contents
Chromosome = Filing Cabinet
Background/Tools:
Luckily for us, the basic machinery we need for our E. chiver design already exists.  The machinery is that of the Lambdiod phage family (i.e. λ, P21, P22, HK022, φ80), as well as the conditional R6K origin of replication and its trans-acting factor pir. The following are basic descriptions of how these parts function.
Lambdoid phage machinery:  Our main interest is the phage proteins excisionase (Xis) and integrase (Int), as well as the phage genetic element attP (phage attachment site) and the bacterial genetic element attB (bacterial attachment site).  Xis and Int are responsible for the site-specific recombination event between attP and attB.  The recombination event is reversible. The direction depends on the combination of proteins being expressed (see diagram below).
R6K origin and pir:  The pir gene encodes the trans-acting protein that allows replication of an R6K origin2.  When pir is turned on, a plasmid containing the R6K origin will be allowed to replicate, but when pir is not expressed the plasmid becomes a suicide vector (cannot replicate).  The CRIM system takes advantage of this by placing an attP site in the R6K vector.  When pir is off, the vector must insert into the chromosome or be lost.
CRIM (Conditional-Replication, Integration, Modular) plasmids utilize these factors and provide the current means of shuttling a construct into and out of a bacterial chromosome1.  However, the design of this system requires manual labor in the form of helper plasmid transformations and subsequent selection procedures each time we wish to excise or integrate our construct (see below figure).  In our design we rewire the CRIM machinery to place the integration and excision events under chemical inducers to eliminate this manual process.  To tie back into the filing cabinet analogy, we are reworking the system to make the process of removing and replacing a file easier, and taking the first step toward allowing this system to be utilized outside of the laboratory by placing the filing system under environmental signals rather than laboratory procedures.
Caption: The CRIM system uses phage machinery placed on helper plasmids to promote integration and excision events.  Preparation of competent cells, transformation of the appropriate helper plasmid, and a multi-step selection process must be performed for each integration and excision event.  The helper plasmids contain a temperature sensitive origin (oriR101) and may be cured via 37C incubation.
The Design:
The design of our E. chiver system seeks to obtain the following goals:  1) Link integration, excision, and replication of the construct (file) to the presence or absence of a chemical inducer.  2) Allow for modularity of the chemical inducer and gene of interest (file contents).  3) Ensure energetically favorable storage of construct by integration as a single copy (avoid multiple integration events).  4) Enable the system to report which machinery is being expressed.  5) Ultimately allow for the coexistence of multiple files to create a bacterial filing cabinet.
A single File contains two major components
Shuttle Construct (File)
This is the plasmid that shuttles in and out of the chromosome.  It contains the following elements:
R6K Origin, for control over replication (see pir component)
attP site, for integration into chromosome in absence of inducer
Modular Biobrick site, for choosing the controlling chemical inducer and for insertion of the gene of interest (File Contents)
pir gene (controlled by inducer), for replication of shuttle construct during presence of inducer
CI repressor (controlled by inducer), for repression of Integrase machinery (see helper construct) during presence of chemical inducer
Reversed Constitutive Promoter, for repression of Integrase machinery after initial integration event (absence of inducer).  This element is essential for addressing goal 3, avoiding multiple integration events.
Helper Construct (Filing Cabinet):
This is a modified chromosomal attB site. Our E. chiver strain must have genetically engineered attB sites in order to function.  The construct contains the following elements:
Xis/Int/RFP operon (under chemical inducer), for excision of an integrated construct during the presence of the inducer.  As soon as the construct is excised, however, this operon is switched off.
CI repressor, which is under the control of the shuttle construct’s Reversed Constitutive Promoter only when the shuttle construct is integrated.  The CI repressor, in this situation, acts to turn off the adjacent integrase machinery to prevent multiple integrations (see goal 3).
CI Promoter/Int/YFP, for expression of the integration machinery under the conditions that 1) no inducer is present, AND 2) no shuttle construct is integrated.
For our example of a single file design (addressing goals 1-4) we used lambda phage machinery.  However, any Lambdoid phage machinery can be by replaced into this system in order to change the site of chromosomal integration.  Follow the link below for a walk through of our single file design and an explanation of how it obtains goals 1-4.
Our design addressing goal 5, allowing for multiple files to coexist (a true filing cabinet), was explored by adding an analogous second filing system under the control of P21 phage machinery and the conditional replication of origin OriV to our original Lambda system.  The other difference in this P21 construct is the use of the CII repressor to repress the P21 integrase gene when appropriate.  In order to create a multi-file system each analogous construct (file) must have 1) a unique origin of replication, 2) a separate Lambdoid phage system whose activity does not overlap with other Lambdoid phage sites (i.e. the combination of HK022 and Lambda phage files is not recommended because HK022 recognizes Lambda attachment sites as well as it’s own), 3) no two files may have their helper construct integrase gene under the same repressor (i.e. lambda system utilizes cI while P21 system utilizes cII).  These three requirements are necessary to prevent crosstalk between two files.  Click on the link below to view a walk through of the multi-file, Lambda and P21, system.
References
1. Haldimann, A., Wanner, B. L. 2001. Conditional-Replication, Integration, Excision, and Retrieval Plasmid-Host Systems for Gene Structure-Function Studies of Bacteria. Journal of Bacteriology. 183:21 6384-6393. 2. Metcalf, W. W., Weihong, J., Wanner, B. L. 1994. Use of the rep technique for allele replacement to construct new Escherichia coli hosts for maintenance of R6Kϒ origin plasmids at different copy numbers. Gene. 138: 1-7.
2. Metcalf, W. W., Weihong, J., Wanner, B.  L.  1994.  Use of the rep technique for allele replacement to construct new Escherichia coli hosts for maintenance of R6Kϒ origin plasmids at different copy numbers.  Gene.  138:  1-7.
© Copyright 2011 Illinois iGEM
University of Illinois
Home
Team
Project
Parts Submitted
Tools
Documentation
Human Practices
Safety
Partners
iGEM
Retrieved from " http://2011.igem.org/Team:UIUC-Illinois/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
