<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Cholera_-_Detection_Winterexp_Period1_Dailylog"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Cholera - Detection/Winterexp/Period1/Dailylog</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL id="css3menu1" class="topmenu"><LI class="topfirst"><A href="https://2013.igem.org/Team:BYU_Provo" style="height:25px;line-height:25px;">
					Home
				</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Team" style="height:25px;line-height:25px;"><SPAN>
						Team
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Team">
							Overview
						</A></LI><LI><A href="https://igem.org/Team.cgi?id=1195">
							Official Team Profile
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Team_Video">
							Team Video
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Team_Members">
							Undergraduates
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Mentors">
							Mentors
						</A></LI></UL><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Project" style="height:25px;line-height:25px;"><SPAN>
						Project
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project/Background">
							Background
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project/ExperimentalDesign">
							Experimental Design
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Notebook" style="height:25px;line-height:25px;"><SPAN>
						Notebook
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook#Phage_Team">
							Phage Team
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook#Cholera_Team">
							Cholera Team
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Results" style="height:25px;line-height:25px;"><SPAN>
						Results
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Judge">
							Judging Criteria
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Experimental">
							Experimental
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Modeling">
							Modeling
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Parts">
							Parts Submitted to the Registry
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Toronto">
							A Taste of Toronto
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Outreach" style="height:25px;line-height:25px;">
					Human Practices
				</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach/OremLibrary">
							Orem Public Library Visit
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach/Baxtor">
							The Adventure of Baxter Bacteria
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Safety" style="height:25px;line-height:25px;"><SPAN>
						Safety
					</SPAN></A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Attribution" style="height:25px;line-height:25px;"><SPAN>
						Attributions &amp; Collaborations
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Attribution">
							Attributions &amp; Acknowledgements
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Collaboration">
							Collaborations
						</A></LI><LI class="topfirst"><A href="https://2013.igem.org/Main_Page" style="height:25px;line-height:25px;">
					iGEM 2013
				</A></LI></P><TABLE width="100%"><TBODY><TR><TD colspan="3"><FONT color="#333399" size="5" face="Calibri"><DL><DD><B>Cholera Detection March - April Notebook: March 15 - March 31 Daily Log</B></DD></DL></FONT></TD></TR><TR valign="top"><TD style="width: 22%; background-color: transparent;"><DL><DD><U><B>Cholera Detection</B></U></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Winterexp">March-April</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Springexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Springexp">May-June</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Summerexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Summerexp">July-August</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Fallexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Fallexp">September-October</A></DD></DL></TD><TD style="width: 82%; background-color: transparent;"><P><FONT size="4"><B>3/15/13</B></FONT></P><P>-KP Today as we compared our individual findings on methods to destroy the biofilm on cholera, we realized that there are many different approaches to take. We can try to stop biofilm production by disrupting the quorum sensing, destroy the internal mechanism that makes the biofilm, or we can attack the biofilm from the outside. We have decided to focus on that last approach. Our goal now, is to find an enzyme or chemical of some kind and then see if it will break down the biofilm of cholera. There are plenty of enzymes already discovered that break down biofilms on other bacteria, but we cannot find anything specific to cholera. We will probably have to experiment with different proven methods for other bacteria and see what effects it has on cholera. We can also possibly incorporate phage as a deliverer of a substance that breaks down cholera biofilm, but right now we are in search of that enzyme.
</P><P>-KK We reported on our investigation of various genes and enzymes that last year’s iGEM team had identified as candidates to disrupt the biofilm. Their approaches appear very disjointed: some proteins were meant to directly degrade the biofilm, some were meant to hydrolyze the QS autoinducer molecule, some (appear) to interfere with transcription and translation of HapR related genes. While the advantage to their multi-front approach is that perhaps one of their various technique could work, the disadvantage of being spread too thin I believes overrides the advantage. So, we’ll be homing in on only one approached: direct removal of the biofilm. Dr. Grose answered a question that had come up as I researched. I learned that two serogroups of Cholera are pathogenic to humans: O1 and O139. I wondered if it were important that the species of cholera that we were to work with be of those two serotypes. Dr. Grose explained that bacteria are grouped serotyped based upon how our immune system reacts to them. It isn’t important what serotype cholera is, because any type will make a biofilm. Also, if the biofilm is removed, any type can be destroyed by stomach acid. Questions to Research: What is the composition of the biofilm in Cholera? Exopolysaccharides (EPS) are a principal building block. EPS in most species has a negative charge and is (thus not surprisingly) chlorine resistant. What other components are there? Has anything successfully degraded its biofilm? We know of multiple papers that document biofilm reduction in seemingly every species BUT cholera, but we have failed to find any that is cholera-specific.
</P><P>- Whitney Hoopes -Compared research articles/data gathered for ideas on how to destroy the cholera biofilm -Quorum sensing is the regulation of gene expression in response to fluctuations in cell-population density. Quorum sensing bacteria produce and release chemical signal molecules called autoinducers that increase in concentration as a function of cell density. The detection of a minimal threshold stimulatory concentration of an autoinducer leads to an alteration in gene expression. Use quorum sensing to trigger transcription of genes that turn on and produce enzyme transcripts to degrade the biofilm? -Reviewed research papers and reviews to gather ideas and plan what genes to clone in -Discussed genes we researched: Dispersin (DspB) 106/107 Aiia 109/110---B. subtilis CytR 111/112 Deoxyribonuclease 113/114---B. subtilis Subtilisin sowirlane 115/116---B. subtilis Apple favonoid 117/118 Nuclease NuCB 119/120 Dnase 1 121/122 Amylase (AmyA) 123/124 Biofilm targets Holin endolysin 129/130 Anti-LPS 125/126 (bacteria target) Out of biofilm ChapK
</P><P><FONT size="4"><B>3/18/13</B></FONT></P><P>-KP Cholera Destroy Plan of attack- Get Vibrio Cholerae Grow Biofilm Find constructs from last year Test constructs against biofilm Construct our own biofilm destroyer to test out.
Cholera Detect There is already a lot done from last year, so we need to test the constructs from last year. The first step is sequencing and then getting it into E. Coli. Just like the destroy group, we first need to start growing some Cholera.
Big phage Need to do more research on what has already been done. One possible question is how to get a plasmid to form with medicine or something inside.
Small Phage They are trying to decide which phage to use. They are going to grow phage and see if any mutations produce a smaller phage. They need to come up with a screening method.
Here is what we need to do right now. We went to Dr. Robison’s lab and asked for a sample of cholera. We will return on Wednesday to pick it up. It is important that we can get the cholera to grow and produce biofilm. Next, we are looking through the work that the iGEM team did last year. We want to try some of their constructs on the biofilm to see if they actually work. We will also try some of our own ideas.
</P><P>-KK Our priority list right now is: 1) Get cholera growing a biofilm, and 2) Test existing constructs from last year’s iGEM.
We talked with Annette, Dr. Robison’s lab manager, and she promised us she would have a streak of cholera available for us by Wednesday. As soon as we get Cholera, we’re going to form a plan with Dr. Grose for how we can store it so as to not need to constantly ask Dr. Robison’s lab for cholera. On Wednesday we’ll begin to see if we can get cholera to grow biofilms.
Today I looked over existing plasmid constructs. Apparently iGEMers from last year did clone many genes into plasmids but the plasmids aren’t yet in the registry. Since the plasmids have not been sequenced, that will be another priority on Wednesday.
We also learned that Calgary in 2008 created “Champion,” a bacteria that sensed AHL and AI-2. They used colicin to succesfully eliminate “Bad Guy 1” (which I think is so tacky, and I struggled to determine what the heck “Bad Guy” is but I think it’s Vibrio Fischeri). Their quorum-sensing construct worked … they did NOT work with cholera, but in their presentation video they make reference to the potential usefulness of their system with cholera.
</P><P>- Whitney Hoopes Researched and read articles pertaining to biofilm degradation: (Also reviewed quorum sensing articles)
Alina Nakhamchik, C. W., and Dean A. Rowe-Magnus (2008). &quot;Cyclic-di-GMP regulates extracellular polysaccharide production, biofilm formation, and rugose colony development by Vibrio vulnificus.&quot; Applied and Environmental Microbiology 74(13): 4199-4209.
Anneleen Cornelissen, P.-J. C., Jeroen T'Syen, Helena Van Praet, Jean-Paul Noben, Olga V. Shaburova, Victor N. Krylov, Guido Volckaert, Rob Lavigne (2011). &quot;The T7-related Pseudomonas putida phage O15 displays virion-associated biofilm degradation properties.&quot; PLoS ONE 6(4): e18597.
Bassler, W.-L. N. a. B. L. (2009). &quot;Bacterial Quorum-Sensing Network Architectures.&quot; Annual Reviews 43: 197-222.
Christopher M. Waters, W. L., Joshua D. Rabinowitz and Bonnie L. Bassler (2008). &quot;Quorum sensing controls biofilm formation in Vibrio cholerae through modulation of cycli di-GMP levels and repression of vpsT.&quot; Journal of Bacteriology 190(7): 2527-2536.
Cynthia Wu, J. Y. L., Gerald G. Fuller, and Lynette Cegelski (2013). &quot;Disruption of Escherichia coli amyloid-integrated biofilm formation at the air-liquid interface by a polysorbate surfactant.&quot; Langmuir 29: 920-926.
D.H. Dusane, J. K. R., A.R. Kumar, Y.V. Nancharaiah, V.P. Venugopalan and S.S. Zinjarde (2008). &quot;Disruption of fungal and bacterial biofilms by lauroyl glucose.&quot; Letters in Applied Microbiology 47: 374-379.
Jun Zhu, M. B. M., Russell E. Vance, Michelle Dzlejman, Bonnie L. Bassler and John J. Mekalanos (2002). &quot;Quorum-sensing regulators control virulence gene expression in Vibrio cholerae.&quot; PNAS 99(5): 3129-3134.
Yildiz, J. C. N. F. a. F. H. (2007). &quot;The rbmBCDEF gene cluster modulates development of rugose colony morphology and biofilm formation in Vibrio cholerae.&quot; Journal of Bacteriology 189(6): 2319-2330.
Yildiz, J. C. N. F. a. F. H. (2008). &quot;Interplay between cyclic AMP-cyclic AMP receptor protein and cyclic di-GMP signaling in Vibrio cholerae biofilm formation.&quot; Journal of Bacteriology 190(20): 6646-6659.
-Begin growing cholera biofilms to test last years constructs -Sequence plasmid constructs on Wednesday (waiting until we have final decision on project)
</P><P><FONT size="4"><B>3/20/13</B></FONT></P><P>-KK We were thrilled to get out plate of cholera today! Then we happened upon about 5 iGEM projects from previous years that involve quorum sensing and/or cholera. We were on the verge of beginning our lab work but our paradigm was suddenly shifted a few steps backward: we needed (and still need) to decide if we should continue working on our cholera project. Calgary’s 2008 iGEM team successfully cloned the quorum-sensing circuit into E.Coli and used it to detect and destroy E.Coli that was expressing AHL - whose homologs appears to be autoinducers of V. Harveyi and V.Fischeri. St. Andrew’s iGEM team faced off against cholera specifically, attempted to render it avirulent through generating high levels of cholera’s autoinducer CAI-1, but they did not tame cholera. They only managed to get the CAI-1 autoinducer to work in E.Coli.
We got our cholera from Annette, Dr. Robison’s lab manager. She plated it on Columbia media, and it had been sitting at 37 degrees C for 24 hours when she gave it to us. We don’t know the exact assay for growing biofilm in cholera, but some protocols suggest LB broth. Our plan is to use LB broth and let cholera grow (NOT shaking it, so that biofilm may grow freely).
March 20th Overview of E. coli transformation protocol: Eppendorfs of DH5alpha cells---thaw on ice Mix 2-5ul of plasmid and keep on ice for 20-30minutes Heat shock for 1 min @ 42 Degrees Celsius Cool eppendorf on ice for 2-5 minutes Add 0.5 mL LB and incubate at 37 Degrees Celsius for 30 minutes (select for AMP) Plate on selective media (LB-AMP) Phage Titer Protocol
Plan: -Need to set up plasmids for sequencing -Grow a Cholera biofilm -Gram-stain of Cholera to check
</P><P>-KP Today we started off by picking up our cholera from Dr. Robison’s lab. Most of the day was spent researching previous iGEM teams that have already worked with cholera, quorum sensing, and biofilm degradation. Some of the previous teams projects are very similar to our own, so we are trying to decide whether to proceed with our current plans or to choose something else. Or, if we stick with our current plan, how can we incorporate phage or make our project unique?
Here are a few good papers on using phage to destroy biofilm.
<A href="https://www.novapublishers.com/catalog/product_info.php?products_id=21940" class="external free" rel="nofollow">https://www.novapublishers.com/catalog/product_info.php?products_id=21940</A><A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797112/" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797112/</A>
How Phages Kill Biofilm
<A href="http://www.biomedcentral.com/1472-6750/8/79" class="external free" rel="nofollow">http://www.biomedcentral.com/1472-6750/8/79</A>
Phage That Are Capable of Destroying Biofilms
Bacteriophage Phi S1 infection of Pseudomonas fluorescens planktonic cells versus biofilms. Pseudomonas fluorescens biofilms subjected to phage phiIBB-PF7A. Lytic Activity of Recombinant Bacteriophage 11 and 12 Endolysins on Whole Cells and Biofilms of Staphylococcus aureus.
</P><P><FONT size="4"><B>3/22/13</B></FONT></P><P>-KK March 22, 2013 (Friday) As a class we elected to continue forward with our cholera project, given that we emphasize elimination of the biofilm (which has NOT been accomplished by any other iGEM team) and using phage in our construct to do so. There are two approaches to using phage to eliminate the biofilm in Cholera. We may engineer a phage to do what we need, or we may go looking for a phage that has already evolved to do what we need.
Here are a few summaries of articles I read today:
The use of phages for the removal of infectious biofilms. Azeredo, J. and Sutherland I.W. (2008) Used two phages isolated from sewage; showed 2-4 log decreases in bacterial population both in vitro and in vivo (murine model). OF INTEREST: In vitro testing involved biofilm and took longer; in vivo testing involved only planktonic cells and was faster. OF INTEREST: Calcofluor biofilm staining. Efficacy of cocktail phage therapy in treating Vibrio cholerae infection in rabbit model Summary: Oral administration of a cocktail of 5 lytic cholera bacteriophages to rabbits yielded strong and statistically significant reductions of cholera. Researchers showed that administration of the cocktail at 6 hour and 12 hour time intervals after bacterial challenge reduced bacteria populations 100 fold and 10 fold, respectively. Administration of the cocktail before bacterial challenge apparently did not significantly decrease cholera OF INTEREST: The emergence of multiple drug resistant strains of V. cholerae is thus a cause for global concern [4,5] and is prompting the exploration for alternative ways of treatment OF INTEREST: To the best of our knowledge it is the first report of combatting V. cholerae infection in an animal model by phages adminis- tered through the oral route. Here are a few more articles that I've downloaded and I'm going to read:
Phage-bacterial interactions in the evolution of toxigenic Vibrio cholerae - this article exams the co-evolution of phage and cholera toxin. From what I read of the abstract, it is hypothesized that cholera's toxin was introduced by horizontal gene transfer from the CTXw phage. Written by a researcher from Harvard Medical School
O Antigen Is the Receptor of Vibrio cholerae Serogroup O1 El Tor Typing Phage VP4 - phage VP4 infects cholera, but certain mutations to the O-antigen gene provides resistance against VP4 (VP4 has anti-biofilm properties?)
Dispersing biofilms with engineered enzymatic bacteriophage - this is a synthetic biology approach to, like the name of the article implies, engineering phages to break down biofilms!! Written by a Harvard-MIT researcher.
</P><P>-KP Today we decided to stick with the cholera project. I am glad, because at first I was slightly disappointed when I realized that our project wasn’t as novel as I originally thought, but I am still very interested. Today we realized that the most important thing that we can do now is to find a way to disrupt the biofilm from the outside. I found a summary article on using phage against biofilms and each one of us chose a few article to read in order to understand the best ways to tackle the problem. I like the idea the paper that Kendall cited above that talks about bacteriophage that have been engineered to express enzymes that degrade the biofilm. I think that that is a good approach because we can combine our findings with the phage group and have a great presentation. Also, there is a lot of research that I have found from the papers that I have read suggesting that it is very possible to do what we will be trying to do.
</P><P><FONT size="4"><B>3/25/13</B></FONT></P><P>-Whitney Hoopes
Papers to read:
E. Suman, S.J. D’souza, P. Jacob, M.R. Sushruth, M.S. Kotian. Anti-biofilm and anti-adherence activity of Glm-U inhibitors.
Ilana Kolodkin-Gal, Shugeng Cao, Liraz Chai, Thomas Bottcher, Roberto Kolter, Jon Clardy, Richard Losick. A self-produced trigger for biofilm disassembly that targets exopolysaccharide.
Anneleen Cornelissen, Pieter-Jan Ceyssens, Victor N. Krylov, Jean-Paul Noben, Guido Volckaert, Rob Lavigne. Identification of EPS-degrading activity within the tail spikes of the novel Pseudomonas putida phage AF.
N. Ramasubbu, L.M. Thomas, C. Ragunath, J.B. Kaplan. Structural analysis of Dispersin B, a biofilm-releasing glycoside hydrolase from the periodontopathogen Actinobacillus actinomycetemocomitans.
Ranjit, Dev K.; Endres, Jennifer L.; Bayles, Kenneth W. Staphylococus aureus CidA and LrgA proteins exhibit holin-like properties.
CH 3/25/13 Cholera distribution in the US: Cholera can be found in brackish water or estuaries near the ocean (people who have eaten certain kinds of seafood have gotten cholera). One source I found suggested that the reported cases of cholera from Oct 2010-Feb2011 were almost all from people who had traveled out of the country. The only cases that had no travel time were found in north eastern US. <A href="http://wwwnc.cdc.gov/eid/article/17/11/11-0808-f1.htm" class="external free" rel="nofollow">http://wwwnc.cdc.gov/eid/article/17/11/11-0808-f1.htm</A> Generally in the world the most cholera cases are reported in Africa and South/Central America. The places where water purification is not so great. One problem with cholera is that countries don’t want to report that they have it because that scares away tourists.
Toxigenic Vibrio cholerae O1 in Water and Seafood, Haiti <A href="http://wwwnc.cdc.gov/eid/article/17/11/pdfs/11-0748.pdf" class="external free" rel="nofollow">http://wwwnc.cdc.gov/eid/article/17/11/pdfs/11-0748.pdf</A> This paper has an example of the difficultness involved in testing water for cholera contamination. They had to filter the water with a special filter then allow their sample to incubate for days and then plate test/PCR test.
I think we all need these shirts <A href="http://www.thebacteriophages.com/sales.htm" class="external free" rel="nofollow">http://www.thebacteriophages.com/sales.htm</A></P><P>-KK 25 In 2007 Timothy K. Lu and James J. Collins published a paper, “Dispersing biofilms with engineered enzymatic bacteriophage,” in which the showed that they had engineered T7 and T3 phage to infect their E. Coli hosts with the gene for Dispersin B, which was released along with the phage progeny. The researchers achieved 99.997% degradation of the biofilm. This is essentially what we are thinking to do: engineer a phage to infect a bacterial host with a gene that expresses a protein that will accelerate the degradation of cholera’s biofilm. These researchers show that it is possible for E.Coli. OUR PLAN: Our first step needs to be to get cholera-specific phages to work with. CTX theta phage is a very famous and well characterized (we found a paper elucidating it’s entire genome) cholera-specific bacteriophage. It is a temperate phage. Chinese researchers used VP4 phage to study serotype O1 cholera. In India, researchers successfully treated cholera-infected rabbits with a cocktail of 5 phages: ATCC 51352- B1, B2, B3, B4, and B5. Tomorrow I will write an email to send to these professors that Dr. Grose will then forward to them on our behalf. Another approach to obtaining cholera-specific bacteriophage is to search for them in samples of Cholera-sullied water. Clarice researched WHERE Cholera is in the United States, but unsurprisingly no one is keen to publish that their water is infected with cholera. She didn’t find anything helpfully conclusive. Once we have bacteriophages we will coordinate with Michael, Whitney, and Nathan to clone into its genome proteins that (hopefully) will degrade cholera’s biofilm. We are researching candidates proteins. We started growing cholera today and I’ve read several articles that mention assays for growing biofilms. Once we have our engineered phage we will begin testing.
</P><P>-CH 3/25/13 We again did some planning of what we need to do going forward. Plan of Attack: 1. Study phages 2. Obtain different cholera associated Phages 3. Engineer phage to penetrate biofilm then produce biofilm degrading enzymes upon infection of cholera 4. Testing I think we also need to do more research on V. cholerae in general. What makes up its biofilm etc. Today I set up a sequencing reaction with reverse primer BI227 (1ul undiluted) and pJG78 (2ul undiluted). Sequence obtained. Next time we need to set up a transformation with pJG78 because there is not much of it left.
</P><P><FONT size="4"><B>3/27/13</B></FONT></P><P>-CH Set up transformation. 1ul purified plasmid into 50ul E.coli. RESULT: contamination issues either from the LB we were using or the plates being old.
</P><P>-KK Dr. Grose has distributed the phage requests we sent her to send to professors in China, India, Switzerland, and Harvard. It would be wonderful to obtain CTXtheta phage, which is a cholera-specific phage whose genome has been well studied and inclusively its joint evolution with cholera has been studied. However, we when arrived at class and began discussing, Dr. Grose presented a better idea. We can use a lysogenic phage specific to E.Coli, rather than a cholera specific phage. We’ll engineer this phage with EPS depolymerases on its tail proteins, and then we will infect our E.Coli with the phage and grow E. Coli with copies of the virus prophage incorporated in its genome. If we can then engineer our bacteria so that the prophage DNA will be excised and expressed when E.Coli detects cholera, lysing the E.Coli and attacking the biofilm, then that will be an incredible iGEM project. Clarice, Kelton and I transformed plasmid IG78 into E.Coli and plated on about 10 plates of LB Amp. During the transformation of our first 50 microliters of E.Coli, we think the hot water bath failed to achieve 42 degrees Celsius, and it’s likely the transformation didn’t work. For that reason, we did the transformation again (using only 1 microliter of plasmid instead of 2 microliters because we’re very low). We shocked the DH5alpha with the hot water bath in Dr. Griffitts lab instead of the one in ours. We hope to harvest the plasmid on Friday.
</P><P><FONT size="4"><B>3/29/13</B></FONT></P><P>-CH More discussion of where our project is going. Redid the transformation with help from Jordan. RESULT: No contamination issues. The transformation rate was very high. Two colonies were streaked to singles and grown up in overnight cultures (LB Amp) in preparation for a mini prep.
</P><P>-KK Bad news: our transformations were infected by who knows what type of bacteria. It wasn’t the LB, because the solution was clear on wednesday and was clear today as well. The tips were autoclaved. We think the plates we used may not have been completely sterile. So today we did the transformation again with 1 microliter plasmid and 50 microliters of DH5alpha. It is about the last of the plasmid we have to use. Jordan supervised the transformation to ensure it went well. He made a few new plates of LB-AMP just today so he’s confident they are sterile. NOTE: during the transformaiton we again had issues with the temperature of the hot-shock plate. The reading was 42 degrees C but our thermometer showed that the true temperature was lower. We increased the temperature of the hot plate to 46 degrees and the thermometer read the correct temperature for the protocol. So, there was no error in our procedure today. Jordan plated the E.Coli on two plates, each plate with a different volume of E.Coli, because he didn’t know how dilute the plasmid was in solution.
On Monday we will present a 30 minute report on how our research is going. I’m going to research over the weekend the bacteriophage Lambda. We won’t be able to use T4 or T7 in our cholera activated E.Coli construct because neither is a lysogenic phage. Both are lytic :( that only means that we won’t be able to unite our project to the other group’s project so seamlessly.
GREAT NEWS: Dr. Reidl of the University of Graz returned our correspondence and is going to be extremely generous with us. He will send us his K139 cholera phage that we asked for AND biofilm-producing cholera AND more, including a o-antigen knockout strain of cholera that Reidl’s lab engineered. Exciting!
</P><P>-KP Today when we showed up at the lab, our beautifully plated e. coli infused with IG78 were not quite as beautiful as we had anticipated. Yes, something was growing, but unfortunately not what we had anticipated. So, with Jordan’s help, we re-did the transformation and hopefully we have what we need show up on Monday. I did some research and found that T1-T7 bacteriophage are lytic bacteriophage. We unfortunately need lysogenic bacteriophage to incorporate into our e. coli, because we want to to stay dormant in the host genome until cholera is sensed. The sensing of cholera will set off the chain of events leading to the expression and creation of the bacteriophage, so it has to be lysogenic bacteriophage that we use. I will do more research to see if there are any examples of one of the T# phages being used as lysogenic phages, because that will help us to connect with the phage group and contribute to an overall better presentation and iGEM project. We have our presentation on Monday and I will be giving a background of lytic and lysogenic phases of bacteriophage and why that is important to our project.
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog">http://2013.igem.org/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp/Period1/Dailylog&amp;oldid=150465" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>