<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_Cholera_-_Enzyme_March-April_Period1_Dailylog"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/Cholera - Enzyme/March-April/Period1/Dailylog</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL id="css3menu1" class="topmenu"><LI class="topfirst"><A href="https://2013.igem.org/Team:BYU_Provo" style="height:25px;line-height:25px;">
					Home
				</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Team" style="height:25px;line-height:25px;"><SPAN>
						Team
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Team">
							Overview
						</A></LI><LI><A href="https://igem.org/Team.cgi?id=1195">
							Official Team Profile
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Team_Video">
							Team Video
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Team_Members">
							Undergraduates
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Mentors">
							Mentors
						</A></LI></UL><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Project" style="height:25px;line-height:25px;"><SPAN>
						Project
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project/Background">
							Background
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project/ExperimentalDesign">
							Experimental Design
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Notebook" style="height:25px;line-height:25px;"><SPAN>
						Notebook
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook#Phage_Team">
							Phage Team
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook#Cholera_Team">
							Cholera Team
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Results" style="height:25px;line-height:25px;"><SPAN>
						Results
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Judge">
							Judging Criteria
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Experimental">
							Experimental
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Modeling">
							Modeling
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Parts">
							Parts Submitted to the Registry
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Toronto">
							A Taste of Toronto
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Outreach" style="height:25px;line-height:25px;">
					Human Practices
				</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach/OremLibrary">
							Orem Public Library Visit
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach/Baxtor">
							The Adventure of Baxter Bacteria
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Safety" style="height:25px;line-height:25px;"><SPAN>
						Safety
					</SPAN></A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Attribution" style="height:25px;line-height:25px;"><SPAN>
						Attributions &amp; Collaborations
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Attribution">
							Attributions &amp; Acknowledgements
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Collaboration">
							Collaborations
						</A></LI><LI class="topfirst"><A href="https://2013.igem.org/Main_Page" style="height:25px;line-height:25px;">
					iGEM 2013
				</A></LI></P><TABLE width="100%"><TBODY><TR><TD colspan="3"><FONT color="#333399" size="5" face="Calibri"><B>Cholera - Enzymes Notebook: March 15 - March 31 Daily Log</B></FONT></TD></TR><TR valign="top"><TD style="width: 18%; background-color: transparent;"><DL><DD><U><B>Cholera Enzyme</B></U></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Enzymes/March-April" title="Team:BYU Provo/Notebook/Cholera - Enzymes/March-April">March-April</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Enzymes/May-June" title="Team:BYU Provo/Notebook/Cholera - Enzymes/May-June">May-June</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Enzymes/July-August" title="Team:BYU Provo/Notebook/Cholera - Enzymes/July-August">July-August</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Enzymes/September-October" title="Team:BYU Provo/Notebook/Cholera - Enzymes/September-October">September-October</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Enzymes/Protocols" title="Team:BYU Provo/Notebook/Cholera - Enzymes/Protocols">Protocols</A></DD></DL></TD><TD style="width: 82%; background-color: transparent;"><P><FONT size="4"><B>3/15/13</B></FONT></P><P><B>Quorum sensing overview:</B> Quorum sensing is the regulation of gene expression in response to fluctuations in cell-population density. Quorum sensing bacteria produce and release chemical signal molecules called autoinducers that increase in concentration as a function of cell density. The detection of a minimal threshold stimulatory concentration of an autoinducer leads to an alteration in gene expression. 
</P><P><B>Our attack plan:</B> Use quorum sensing to trigger transcription of genes that produce enzymes capable of degrading <I>Vibrio cholerae</I> biofilms.
</P><P>
Last year’s team started on this project and was able to clone the necessary quorum sensing related genes from Vibrio cholerae into plasmids in preparation for insertion into E. coli. After reviewing their info and data, we need to sequence their plasmids to ensure that the plasmids have the correct genes in the correct sequence. Once we are able to confirm that everything got into the plasmid correctly we can move the plasmid into E. coli and make sure that it is working correctly. To help with this, we are creating a map of the plasmid to use as our reference for our sequencing. We have a generic map of the genes and are using them to compile a cohesive nucleotide sequence.
</P><P>We also compared research articles and data that we had gathered to discuss ideas on how to effectively destroy the V. cholerae biofilm. After our discussion we came up with a list of possible enzymes that we are going to research further before deciding on which enzymes we will focus our research on.
</P><P>
Genes/Enzymes we will further research:
</P><UL><LI>Dispersin (DspB) 106/107
</LI><LI>Aiia 109/110---B. subtilis
</LI><LI>CytR 111/112
</LI><LI>Deoxyribonuclease 113/114---B. subtilis
</LI><LI>Subtilisin sowirlane 115/116---B. subtilis
</LI><LI>Apple favonoid 117/118
</LI><LI>Nuclease NuCB 119/120
</LI><LI>Dnase 1 121/122
</LI><LI>Amylase (AmyA) 123/124
</LI><LI>Biofilm targets
</LI></UL><LI>Holin endolysin 129/130
</LI><LI>Anti-LPS 125/126 (bacteria target)
</LI><LI>Out of biofilm
</LI><LI>ChapK
</LI><P><FONT size="4"><B>3/18/13</B></FONT></P><P>We continued working on our plasmid map today. There are seven genes that we are cloning into E. coli to allow it to sense and respond to the V. cholerae quorum sensing system. Those genes are: 
</P><OL><LI>BamHI site: LuxPQ operon with med promoter
</LI><LI>BamHI/PstI site: HapR with own promoter
</LI><LI>EcoRI/PstI site: GFP with HapR promoter (forward direction)
</LI><LI>XhoI/EcoRI RFP with Qrr4 promoter (reverse direction)
</LI><LI>HIndIII/XhoI: Qrr4 with own promoter (reverse direction)
</LI><LI>HindIII/SacI: LuxU/O with med promoter (forward direction)
</LI><LI>SacI/XbaI: CqsS with med promoter (forward direction)
</LI></OL><P>
Because of interference with the restriction sites, the cloning sequence needs to be: 2,5,3,6,4,7,1
</P><P>The map of the final construct will be: (CP = constitutive promoter, hapRp = hapR responsive promoter, Qrr4Rp = Qrr4 responsive promoter)
CP-LUXPQ; hapRp-HapR; hapRp-GFP-term; Qrr4Rp-RFP-term; Qrr4Rp-Qrr4; CP-LuxO/U; CP-CqsS
</P><P>Associating GFP and RFP to the hapR and Qrr4 promoters respectively, will allow us to easily test whether our construct is working correctly. When placed in a high-cell density (HCD) solution of V. cholerae, the hapR linked GFP will become activated in our E. coli. Similarly, in low-cell density solutions of V. cholerae, the RFP should become activated. If our E.coli activates the same gene in both the HCD and LCD environments, then something in the construct is not working correctly. One of the problems that we might have is whether or not the sRNA genes will clone in successfully. We will be able to determine whether or not they do when we place the E coli in a site with HCD and LCD of cholera. If the rfp is activated in both then we will know that the sRNA’s weren’t transferred in succesfully and we will need to  figure out a different approach.
</P><P>In analyzing the plasmids from last year’s work, we found that last year they only cloned in genes 2-7. Gene 1 still needs to be cloned into our plasmids prior to our inserting the plasmid into our E. coli. We are also going to finish the plasmid sequence of the plasmids from last year so that we can sequence them to ensure that they were successful in cloning in the genes that they wanted. In all, we have a cloning site of around 1,000 bp that we are going to check. Clarise started working on the primer details to make sure we have enough primers to cover the entire gene during sequencing. So far it looks like the only gene that we need to insert another primer in is the first one, which is LuxPQ. We need to finish this by Wednesday so that we can submit the cloning site to the sequencing center by Thursday.
</P><P>We also requested a strain of V. cholerae from Dr. Robison’s lab. They will have it ready for us on Wednesday so that we can begin working on calibrating our growth conditions to get the best biofilm growth possible. Last year’s team had trouble getting good cholera biofilm growth, so we are going hit this issue immediately to solve the problem.
</P><P><B>Research</B>:
Whitney also researched the following articles pertaining to biofilm degradation in preparation for our attempts to degrade the cholerae biofilm using the quorum sensing system of cholera to trigger a response from our E. coli:
</P><DL><DD>Alina Nakhamchik, C. W., and Dean A. Rowe-Magnus (2008). &quot;Cyclic-di-GMP regulates extracellular polysaccharide production, biofilm formation, and rugose colony development by Vibrio vulnificus.&quot; Applied and Environmental Microbiology 74(13): 4199-4209.
</DD></DL><DL><DD>Anneleen Cornelissen, P.-J. C., Jeroen T'Syen, Helena Van Praet, Jean-Paul Noben, Olga V. Shaburova, Victor N. Krylov, Guido Volckaert, Rob Lavigne (2011). &quot;The T7-related Pseudomonas putida phage O15 displays virion-associated biofilm degradation properties.&quot; PLoS ONE 6(4): e18597.
</DD></DL><DL><DD>Bassler, W.-L. N. a. B. L. (2009). &quot;Bacterial Quorum-Sensing Network Architectures.&quot; Annual Reviews 43: 197-222.
</DD></DL><DL><DD>Christopher M. Waters, W. L., Joshua D. Rabinowitz and Bonnie L. Bassler (2008). &quot;Quorum sensing controls biofilm formation in Vibrio cholerae through modulation of cycli di-GMP levels and repression of vpsT.&quot; Journal of Bacteriology 190(7): 2527-2536.
</DD></DL><DL><DD>Cynthia Wu, J. Y. L., Gerald G. Fuller, and Lynette Cegelski (2013). &quot;Disruption of Escherichia coli amyloid-integrated biofilm formation at the air-liquid interface by a polysorbate surfactant.&quot; Langmuir 29: 920-926.
</DD></DL><DL><DD>D.H. Dusane, J. K. R., A.R. Kumar, Y.V. Nancharaiah, V.P. Venugopalan and S.S. Zinjarde (2008). &quot;Disruption of fungal and bacterial biofilms by lauroyl glucose.&quot; Letters in Applied Microbiology 47: 374-379.
</DD></DL><DL><DD>Jun Zhu, M. B. M., Russell E. Vance, Michelle Dzlejman, Bonnie L. Bassler and John J. Mekalanos (2002). &quot;Quorum-sensing regulators control virulence gene expression in Vibrio cholerae.&quot; PNAS 99(5): 3129-3134.
</DD></DL><DL><DD>Yildiz, J. C. N. F. a. F. H. (2007). &quot;The rbmBCDEF gene cluster modulates development of rugose colony morphology and biofilm formation in Vibrio cholerae.&quot; Journal of Bacteriology 189(6): 2319-2330.
</DD></DL><DL><DD>Yildiz, J. C. N. F. a. F. H. (2008). &quot;Interplay between cyclic AMP-cyclic AMP receptor protein and cyclic di-GMP signaling in Vibrio cholerae biofilm formation.&quot; Journal of Bacteriology 190(20): 6646-6659.
</DD></DL><P><FONT size="4"><B>3/20/13</B></FONT></P><P>Looking at previous projects by other iGem teams, we discovered that there was another iGEM project from the Calgary team that used Quorum sensing extensively for detection and destruction. We informed Dr. Grose and did some more research into possible projects that have already been done that are similar to ours. Since 2008 there has been at least one project every year that has used quorum sensing in some sense. Most used V. fischeri and only one used V. cholerae, but their project wasn’t to the extent that we are aiming for as they simply transferred the quorum sensing system, while we want to not only transfer this signal system, but use it to trigger a new reaction. We decided to look into the other projects more and  decide on Friday whether or not we will continue with our cholera research.
</P><P><B>Transformation Protocol:</B>
The following is an overview of the E. coli transformation protocol that we will be using:
</P><OL><LI>Eppendorfs of DH5alpha cells---thaw on ice
</LI><LI>Mix 2-5ul of plasmid and keep on ice for 20-30minutes
</LI><LI>Heat shock for 1 min @ 42 Degrees Celsius
</LI><LI>Cool eppendorf on ice for 2-5 minutes
</LI><LI>Add 0.5 mL LB and incubate at 37 Degrees Celsius for 30 minutes (select for AMP)
</LI><LI>Plate on selective media (LB-AMP)
</LI></OL><P><B>Research</B>: Michael also read the following research articles on the quorum sensing system in Vibrio cholerae to gain a better understanding of this system:
</P><DL><DD>Hoch, J. Two-Component and Phosphorelay Signal Transduction. Curr. Opin. Microbiol. 2000, 3, 165–170.
</DD></DL><DL><DD>Miller, M.; Skorupski, K.; Lenz, D.; Taylor, R.; Bassler, B. Parallel Quorum Sensing Systems Converge to Regulate Virulence in Vibrio cholerae. Cell 2002, 110, 303–314.
</DD></DL><DL><DD>Waters, C.; Bassler, B. Quorum Sensing: Cell-to-Cell Communication in Bacteria. Annu. Rev. Cell Dev. Biol. 2005, 21, 319–446.
</DD></DL><DL><DD>Zhu, J.; Miller, M.; Vance, R.; Dziejman, M.; Bassler, B.; Mekalanos, J. Quorum-Sensing Regulators Control Virulence Gene Expression in Vibrio cholerae. Proc. Natl. Acad. Sci. 2002, 99, 3129–3134.
</DD></DL><P><FONT size="4"><B>3/22/13</B></FONT></P><P>Today we met together as an entire team and discussed whether we would continue with the cholera project or not. After discussion, we decided that although work on this has already been done, the scope of what we are attempting with both detecting and destroying cholera biofilms is still unique, so we will continue with the cholera project. However, we will change the approach of our cholera project to incorporate phage into the destroy part of the project. This will make our project even more novel and will bring the two projects closer together so that our overall presentation will be more coherent.
</P><P>Due to the restructuring of our cholera approach we have decided to change the subgroups of our cholera team and our focuses. The two destroy teams will be the phage team and the enzyme team. Our cholera-enzyme team will focus on finding, characterizing, and transferring enzymes that will disrupt the biofilm into our E. coli system and phage. Our cholera-phage group will have a two pronged approach. One side will involved engineering a known cholera phage with modifications that will degrade biofilm. The other side is a search for cholera associated phage. They will work on designing, characterizing, and transferring our phage into the E. coli system and link it to the quorum sensing signal system.
</P><P><B>Research:</B>
Michael also read the following articles to research possible enzymes to focus our work on:
</P><DL><DD>Thallinger, B.; Prasetyo, E.; Nyanhongo, G.; Guebitz, G. Antimicrobial Enzymes: An Emerging Strategy to Fight Microbes and Microbial Biofilms. Biotechnol. J. 2013, 8, 97-109.
</DD></DL><DL><DD>Lamppa, J.; Griswold, K. Alginate Lyase Exhibits Catalysis-Independent Biofilm Dispersion and Antibiotic Synergy. Antimicrob. Agents Chemother. 2013, 57, 137-145.
</DD></DL><DL><DD>Molobela, I.; Cloete, T.; Beukes, M. Protease and Amylase Enzymes for Biofilm Removal and Degradation of Extracellular Polymeric Substances (EPS) Produced by Pseudomonas fluorescens Bacteria. Afr. J. Microbiol. Res. 2010, 4, 1515-1524.
</DD></DL><P>Whitney read the following articles:
</P><DL><DD>Ilana Kolodkin-Gal, S. C., Liraz Chai, Thomas Bottcher, Roberto Kolter, Jon Clardy, and Richard Losick (2012). &quot;A self-produced trigger for biofilm disassembly that targets exopolysaccharide.&quot; Cell 149: 684-692.
</DD></DL><DL><DD>N. Ramasubbu, L. M. T., C. Ragunath and J.B. Kaplan (2005). &quot;Structural analysis of Dispersin B, a biofilm-releasing glycoside hydrolase from the periodontopathogen Actinobacillus actinomycetemcomitans.&quot; Journal of Molecular Biology 349: 475-486.
</DD></DL><DL><DD>Anneleen Cornelissen, P.-J. C., Victor N. Krylov, Jean-Paul Noben, Guido Volckaert, Rob Lavigne (2012). &quot;Identification of EPS-degrading activity within the tail spikes of the novel Pseudomonas putida phage AF.&quot; Virology 434: 251-256.
</DD></DL><DL><DD>Dev K. Ranjit, J. L. E. a. K. W. B. (2011). &quot;Staphylococcus aureus CidA and LrgA proteins exhibit holin-like properties.&quot; Journal of Bacteriology 193(10): 2468-2476.
</DD></DL><P><FONT size="4"><B>3/23/13</B></FONT></P><P>The following are notes from the articles we researched about biofilm degradation using enzymes.

Lamppa, J.; Griswold, K. Alginate Lyase Exhibits Catalysis-Independent Biofilm Dispersion and Antibiotic Synergy. Antimicrob. Agents Chemother. 2013, 57, 137-145.
</P><UL><LI>P. aruginosa infects the airways of cystic fibrosis patients and forms a mucoid biofilm. The major component of the biofilm is alginate.
</LI><LI>Alginate is a copolymer of (1,4)-linked beta-D-mannuronic acid and alpha-L-guluronic acid.
</LI><LI>Two alginate lyases were studied: A1-III and AlgL.
</LI><LI>When tested individually, each showed slight biofilm degradation (25-30%) degradation. However, when either was used in conjunction with tobramycin there was 100% biofilm degradation.
</LI><LI>This was tested on forms of the bacteria that both do and don't create alginate, thus the biofilm degradation was not linked to the alginate lyase function specifically.
</LI><LI>The same results occurred whether or not they used the intact lyase protein, an inactive mutant, or the individual amino acids.
</LI><LI>They postulate that the lyase enzyme is used as a food source, the presence of which induces a natural partial breakdown of the biofilm as bacteria spread to grow. This spreading raises the efficiency of the tobramycin and allows for 100% biofilm degradation.
</LI><LI>I wonder if we could produce the environmental triggers that cause cholera biofilms to break apart in the intestines, we might be able to do something similar
</LI></UL><P>
Molobela, I.; Cloete, T.; Beukes, M. Protease and Amylase Enzymes for Biofilm Removal and Degradation of Extracellular Polymeric Substances (EPS) Produced by Pseudomonas fluorescens Bacteria. Afr. J. Microbiol. Res. 2010, 4, 1515-1524.
</P><UL><LI>One major problem of degrading biofilms is that the effectiveness of an enzyme to degrade a biofilm depends upon the specific composition of the biofilm and its extracellular polymeric substance.
</LI><LI>Synthetic polysaccharides have been used in the past to degrade the biofilms of several bacteria, but are not effective with all biofilms.
</LI><LI>They grew Pseudomonas fluorescens biofilms, then extracted the EPS to determine its carbohydrate and protein composition.
</LI><LI>A series of commercial enzymes were tested  with the proteases Savinase and Everlase the most effective in reducing protein concentration within the EPS.
</LI><LI>They postulate that proteases will be most effective against EPS with a higher protein concentration and amylase enzymes will be most effective against EPS with a higher carbohydrate concentration.
</LI><LI>However, they also believe that the relative concentrations of proteins and carbohydrates depends upon the growth environment of the biofilm and can change from location to location, even for the same species of bacteria.
</LI></UL><P>
Thallinger, B.; Prasetyo, E.; Nyanhongo, G.; Guebitz, G. Antimicrobial Enzymes: An Emerging Strategy to Fight Microbes and Microbial Biofilms. Biotechnol. J. 2013, 8, 97-109.
</P><UL><LI>This is a review of several enzymes shown to be effective in degrading biofilms and/or destroying bacteria
</LI><LI>There are two main types of enzymes used to degrade biofilms: proteases and amylases. This is because biofilms are primarily a mixture of proteins and polysaccharides.
</LI><LI>The list of effective enzymes in this review could help us supplement our list of potential enzymes to use
</LI></UL><P>
Nate read the following article:
<A href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3591221/" class="external free" rel="nofollow">http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3591221/</A>.
This review lists and talks about several categories of plant based products that degrade the biofilm
</P><P><FONT size="4"><B>3/25/13</B></FONT></P><UL><LI>Set up overnights (O/Ns) of cholera in 4 mL LB at room temperature, 30 C, and 37 C. All samples set up without shaking and with control blanks. We will see which environment provides the best biofilm growth.
</LI><LI>Last year's team only cloned in two enzymes to test for biofilm degradation, Cytr and AmyA. We took samples of the plasmids containing those  enzymes and prepared them for sequencing. Used pIB85 (forward) and  pIB86 (reverse ) primers. There are four samples to be sequenced. For each sample, 2 μL of plasmid and 1 μL of primer was used. The prepared samples were sent to be sequenced for us to determine if our plasmids are ready to go.
</LI><LI>Streaked cholera on LB plate in 30 C incubator.
</LI></UL><P>Possible additional enzymes to test:
</P><DL><DD><B>Subtilisins</B> - produced by Bacillus sp., are serine proteases that hydrolyze adhesins. Shown to remove biofilms of several bacteria, but not yet tested on cholera specifically.
</DD><DD><B>Dispersin B</B> - produced by Actinobacillus actinomycetemcomitans. Combining with proteases, DNAases, and glycolytic enzymes enhances biofilm removal.
</DD></DL><P><B>Research:</B></P><DL><DD>Sumantha, A., Larroche, C., Pandey, A., Microbiology and industrial biotechnology of food-grade proteases: a perspective. Food Technol. Biotechnol. 2006, 44, 211–220.
</DD></DL><DL><DD>Darouiche, R.O., Mansouri, M.D., Gawande, P.V., Madhyastha, S., Antimicrobial and  efficacy of triclosan and DispersinB¨ combination. J. Antimicrob. Chemother. 2009, 64, 88–93.
</DD></DL><DL><DD>Esperanza Torres, C., Lenon, G., Craperi, D., Wilting, R. et. al., Enzymatic treatment for preventing biofilm formation in the paper industry. Appl. Microbiol. Biotechnol. 2011, 92, 95–103.
</DD></DL><P><FONT size="4"><B>3/27/2013</B></FONT></P><P>Today we discussed in detail the plans for our upcoming experiments. We are currently growing V. cholerae cultures at 37 °C, 30 °C, and room temperature. We started growing our cultures on Monday (3/25) and have not observed any biofilm growth. Overnights of cholera were started again but left overnight in the shakers to better disperse the cholera growth. Once we observe biofilm growth we will focus on our biofilm degradation assays using the following three enzymes:
</P><UL><LI>Amylase (AmyA)
</LI><LI>Plasmid IG 86 were sequenced and verified as AmyA with appropriate forward and reverse primers. Subtilisin Savinase - from Bacillus lentus
</LI><LI>We have the primers from last year and need to get the genes so that we can start cloning them into E. coli
</LI></UL><LI>Dispersin B - from Actinobacillus pleuropneumoniae
</LI><LI>We have the primers from last year and need to get the genes so that we can start cloning them into E. coli.
</LI><P>
We will try to obtain Bacillus lentus from another professor in the department, Dr. Richard Robison for our DNA template. 

On Monday (4/1), we are going to give a presentation on the progress of our research and specify our experimental design. Each group member will research and present on a specific enzyme:
</P><DL><DD>Whitney: AmyA
</DD><DD>Nathan: Dispersin B
</DD><DD>Michael: Savinase 
</DD></DL><P>
The following research papers were studied and used to prepare for our presentation:
</P><DL><DD>Jason B. Harris, R. C. L., Firdausi Qadri, Edward T. Ryan, Stephen B. Calderwood (2012). &quot;Cholera.&quot; The Lancet 379(9835): 2466-2476.
</DD></DL><DL><DD>Luanne Hall-Stoodley, P. S. (2002). &quot;Developmental regulation of microbial films.&quot; Current Opinion in Biotechnology 13(3): 228-233.
</DD></DL><DL><DD>Shuyang Sun, S. K., Diane McDougald (2013). &quot;Relative Contributions of Vibrio Polysaccharide and Quorum Sensing to the Resistance of Vibrio cholerae to Predation by Heterotrophic Protists.&quot; PLoS ONE 8(2): e56338.
</DD></DL><DL><DD>Nicholas J. Shikuma, K. R. D., Jiunn N.C. Fong and Fitnat H. Yildiz (2012). &quot;The transcriptional regulator, CosR, controls compatible solute biosynthesis and transport, motility and biofilm formation in Vibrio cholerae.&quot; Environmental Microbiology.
</DD></DL><DL><DD>M. Kamruzzaman, S. M. N. U., D. Ewen Cameron, Stephen B. Calderwood, G. Balakrish Nair, John J. Mekalanos, and Shah M. Faruque (2010). &quot;Quorum-regulated biofilms enhance the development of conditionally viable, environmental Vibrio cholerae.&quot; PNAS 107(4): 1588-1593.
</DD></DL><DL><DD>Katrina L. Van Dellen, P. I. W. (2006). &quot;The Vibrio cholerae biofilm: A target for novel therapies to prevent and treat cholera.&quot; Drug Discovery Today: Disease Mechanisms 3(2).
</DD></DL><DL><DD>Mudrack, Benjamin Tamayo Rita (2012) “The Vibrio cholerae Pst2 Phosphate Transport System Is Upregulated in Biofilms and Contributes to Biofilm-Induced Hyperinfectivity”
<UL><LI>“Therefore, the physiological state of individual biofilm-associated cells is responsible for the observed hyperinfectivity.”
</LI><LI>Does this mean that even if we break down the bio-film that v. cholerae would still be infectious?
</LI></UL></DD></DL><DL><DD>Tapas Patra, Hemanta Koley, Thandavaryan Ramamurthy, Asoke C. Ghose, Ranjan K. Nandy. “The Entner-Doudoroff Pathway Is Obligatory for Gluconate Utilization and Contributes to the Pathogenicity of Vibrio cholerae”
<UL><LI>The Entner -Doudoroff pathway plays a role is the catabolism of sugars
</LI><LI>It can be turned on or off depending on which phase the V. cholerae is in.
</LI><LI>Deactivation of the ED pathway positively affects biofilm formation.
</LI></UL></DD></DL><DL><DD>Valeru SP, Wai SN, Saeed A, Sandström G, Abd H. “ToxR of Vibrio cholerae affects biofilm, rugosity and survival withAcanthamoeba castellanii”
</DD></DL><P><FONT size="4"><B>3/29/2013</B></FONT></P><P>Cholera overnights were checked and still no growth is visible. We received a response for Dr. Robison and he does not have the bacteria we are looking for. 
</P><P>The following papers were also read to prepare for our upcoming presentation:
</P><DL><DD>Sun, S.; Kjelleberg, S.; McDougald, D. Relative Contributions of Vibrio Polysaccharide and Quorum Sensing to the Resistance of Vibrio cholerae to Predation by Heterotrophic Protists. PLoS One 2013, 8, PMC3575383.
</DD></DL><DL><DD>Kalpana, B.; Aarthy, S.; Pandian, S. Antibiofilm Activity of alpha-Amylase from Bacillus subtilis S8-18 Against Biofilm Forming Human Bacterial Pathogens. Appl. Biochem. Biotechnol. 2012, 167, 1778-1794.
</DD></DL><DL><DD>Augustine, N.; Peter, W.; Kerkar, S.; Thomas, S. Arctic Actinomycetes as Potential Inhibitors of Vibrio cholerae Biofilm. Curr. Microbiol. 2012, 64, 338-342.
</DD></DL><DL><DD>He, H.; Cooper, J.; Mishra, A.; Raskin, D. Stringent Response Regulation of Biofilm Formation in Vibrio cholerae. J. Bacteriol. 2012, 194, 2962-2972.
</DD></DL><P>
The following are notes from the above papers:
</P><UL><LI>Biofilm formation regulated by a variety of signals, including salinity, bile, calcium, and phosphate.
</LI><LI>Genes for EPS formation are found in two vps (Vibrio exopolysaccharide) operons, one consisting of vpsU and vpaA to -K, and the other consisting of vpsL to -Q.
</LI><LI>There are two known transcriptional activators of the vps operons, VpsR and VpsT.
</LI><LI>vpsR mutants are completely defective in biofilm formation
</LI></UL><LI>Expression of vpsR and vpsT is regulated by the expression of hapR.
</LI><LI>At high cell density, hapR is expressed, and vpsR and vpsT are repressed.
</LI><LI>Stringent response induces biofilm formation, particularly when V. cholerae is forming biofilms in aquatic environments, where there may be low nutrient availability. Stringent response is induced when there is an increased concentration of the second messengers pppGpp (guanosine 3=-diphosphate 5=triphosphate) and ppGpp [guanosine 3=5=-bis(diphosphate)], together termed (p)ppGpp. An increased concentration of (p)ppGpp causes significant changes in gene expression that result in cessation of growth and induction of specific stress responses.
</LI><LI>Deletion of (p)ppGpp synthases led to loss of the ability to form biofilm and a decrease in expression of vpsR and vpsT.
</LI><LI>In stringent response-induced cultures, vpsR expression increased 2.5-fold and vpsT expression increased 4-fold.
</LI></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog">http://2013.igem.org/Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/Cholera_-_Enzyme/March-April/Period1/Dailylog&amp;oldid=221499" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>