"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Cholera - Detection
From 2013.igem.org
Home
Team
Overview
Official Team Profile
Team Video
Undergraduates
Mentors
Project
Overview
Background
Experimental Design
Notebook
Overview
Phage Team
Cholera Team
Results
Overview
Judging Criteria
Experimental
Modeling
Parts Submitted to the Registry
A Taste of Toronto
Human Practices
Overview
Orem Public Library Visit
The Adventure of Baxter Bacteria
Safety
Attributions & Collaborations
Attributions & Acknowledgements
Collaborations
iGEM 2013
Overview
March-April
May-June
July-August
September-October
Overview
ADD Cholera Detection project overview
The mission of Brigham Young University--founded, supported, and guided by The Church of Jesus Christ of Latter-day Saints--is to assist individuals in their quest for perfection and eternal life. That assistance should provide a period of intensive learning in a stimulating setting where a commitment to excellence is expected and the full realization of human potential is pursued.
All instruction, programs, and services at BYU, including a wide variety of extracurricular experiences, should make their own contribution toward the balanced development of the total person. Such a broadly prepared individual will not only be capable of meeting personal challenge and change but will also bring strength to others in the tasks of home and family life, social relationships, civic duty, and service to mankind.
.....
In meeting these objectives BYU's faculty, staff, students, and administrators should be anxious to make their service and scholarship available to The Church of Jesus Christ of Latter-day Saints in furthering its work worldwide. In an era of limited enrollments, BYU can continue to expand its influence both by encouraging programs that are central to the Church's purposes and by making its resources available to the Church when called upon to do so.
We believe the earnest pursuit of this institutional mission can have a strong effect on the course of higher education and will greatly enlarge Brigham Young University's influence in a world we wish to improve.
Contents
1March
1.13/15/13
1.23/18/13
1.33/20/13
1.43/22/13
1.53/25/13
1.63/27/13
1.73/29/13
2April
2.14/1/13
2.24/3/13
2.34/5/13
2.44/8/13
2.54/10/13
2.64/12/13
2.74/15/13
3May
3.15/1/13
3.25/3/13
3.35/6/13
3.45/8/13
3.55/10/13
3.65/13/13
3.75/14/13
3.85/15/13
3.95/17/13
3.105/20/13
3.115/22/13
3.125/24/13
3.135/27/13
3.145/29/13
3.155/31/13
4June
4.16/3/13
4.26/5/13
4.36/7/13
4.46/10/13
4.56/12/13
4.66/14/13
4.76/17/13
4.86/19/13
4.96/21/13
4.106/24/13
4.116/26/13
4.126/28/13
5July
5.17/1/13
5.27/3/13
5.37/5/13
5.47/8/13
5.57/10/13
5.67/12/13
5.77/15/13
5.87/17/13
5.97/19/13
5.107/22/13
5.117/24/13
5.127/26/13
5.137/29/13
5.147/31/13
6August
6.18/2/13
6.28/5/13
6.38/7/13
6.48/9/13
6.58/12/13
6.68/14/13
6.78/16/13
6.88/19/13
6.98/21/13
6.108/23/13
6.118/26/13
6.128/28/13
6.138/30/13
7September
7.19/2/13
7.29/4/13
7.39/6/13
7.49/9/13
7.59/11/13
7.69/13/13
7.79/16/13
7.89/18/13
7.99/20/13
7.109/23/13
7.119/25/13
7.129/27/13
7.139/30/13
March
3/15/13
KP 3/15/13 Today as we compared our individual findings on methods to destroy the biofilm on cholera, we realized that there are many different approaches to take. We can try to stop biofilm production by disrupting the quorum sensing, destroy the internal mechanism that makes the biofilm, or we can attack the biofilm from the outside. We have decided to focus on that last approach. Our goal now, is to find an enzyme or chemical of some kind and then see if it will break down the biofilm of cholera. There are plenty of enzymes already discovered that break down biofilms on other bacteria, but we cannot find anything specific to cholera. We will probably have to experiment with different proven methods for other bacteria and see what effects it has on cholera. We can also possibly incorporate phage as a deliverer of a substance that breaks down cholera biofilm, but right now we are in search of that enzyme.
KK 3/15/13 We reported on our investigation of various genes and enzymes that last year’s iGEM team had identified as candidates to disrupt the biofilm. Their approaches appear very disjointed: some proteins were meant to directly degrade the biofilm, some were meant to hydrolyze the QS autoinducer molecule, some (appear) to interfere with transcription and translation of HapR related genes. While the advantage to their multi-front approach is that perhaps one of their various technique could work, the disadvantage of being spread too thin I believes overrides the advantage. So, we’ll be homing in on only one approached: direct removal of the biofilm.  Dr. Grose answered a question that had come up as I researched. I learned that two serogroups of Cholera are pathogenic to humans: O1 and O139. I wondered if it were important that the species of cholera that we were to work with be of those two serotypes. Dr. Grose explained that bacteria are grouped serotyped based upon how our immune system reacts to them. It isn’t important what serotype cholera is, because any type will make a biofilm. Also, if the biofilm is removed, any type can be destroyed by stomach acid.  Questions to Research:  What is the composition of the biofilm in Cholera? Exopolysaccharides (EPS) are a principal building block. EPS in most species has a negative charge and is (thus not surprisingly) chlorine resistant. What other components are there? Has anything successfully degraded its biofilm? We know of multiple papers that document biofilm reduction in seemingly every species BUT cholera, but we have failed to find any that is cholera-specific.
March 15th- Whitney Hoopes -Compared research articles/data gathered for ideas on how to destroy the cholera biofilm -Quorum sensing is the regulation of gene expression in response to fluctuations in cell-population density. Quorum sensing bacteria produce and release chemical signal molecules called autoinducers that increase in concentration as a function of cell density. The detection of a minimal threshold stimulatory concentration of an autoinducer leads to an alteration in gene expression. Use quorum sensing to trigger transcription of genes that turn on and produce enzyme transcripts to degrade the biofilm? -Reviewed research papers and reviews to gather ideas and plan what genes to clone in -Discussed genes we researched: Dispersin (DspB) 106/107 Aiia 109/110---B. subtilis CytR 111/112 Deoxyribonuclease 113/114---B. subtilis Subtilisin sowirlane 115/116---B. subtilis Apple favonoid 117/118 Nuclease NuCB 119/120 Dnase 1 121/122 Amylase (AmyA) 123/124 Biofilm targets Holin endolysin 129/130 Anti-LPS 125/126 (bacteria target) Out of biofilm ChapK
3/18/13
KP Mon. March 18 Cholera Destroy Plan of attack-  Get Vibrio Cholerae Grow Biofilm Find constructs from last year Test constructs against biofilm Construct our own biofilm destroyer to test out.
Cholera Detect There is already a lot done from last year, so we need to test the constructs from last year. The first step is sequencing and then getting it into E. Coli. Just like the destroy group, we first need to start growing some Cholera.
Big phage Need to do more research on what has already been done. One possible question is how to get a plasmid to form with medicine or something inside.
Small Phage They are trying to decide which phage to use. They are going to grow phage and see if any mutations produce a smaller phage. They need to come up with a screening method.
Here is what we need to do right now. We went to Dr. Robison’s lab and asked for a sample of cholera. We will return on Wednesday to pick it up. It is important that we can get the cholera to grow and produce biofilm. Next, we are looking through the work that the iGEM team did last year. We want to try some of their constructs on the biofilm to see if they actually work. We will also try some of our own ideas.
KK March 18th,
Our priority list right now is: 1) Get cholera growing a biofilm, and 2) Test existing constructs from last year’s iGEM.
We talked with Annette, Dr. Robison’s lab manager, and she promised us she would have a streak of cholera available for us by Wednesday. As soon as we get Cholera, we’re going to form a plan with Dr. Grose for how we can store it so as to not need to constantly ask Dr. Robison’s lab for cholera. On Wednesday we’ll begin to see if we can get cholera to grow biofilms.
Today I looked over existing plasmid constructs. Apparently iGEMers from last year did clone many genes into plasmids but the plasmids aren’t yet in the registry. Since the plasmids have not been sequenced, that will be another priority on Wednesday.
We also learned that Calgary in 2008 created “Champion,” a bacteria that sensed AHL and AI-2. They used colicin to succesfully eliminate “Bad Guy 1” (which I think is so tacky, and I struggled to determine what the heck “Bad Guy” is but I think it’s Vibrio Fischeri). Their quorum-sensing construct worked … they did NOT work with cholera, but in their presentation video they make reference to the potential usefulness of their system with cholera.
March 18th, 2013- Whitney Hoopes
Researched and read articles pertaining to biofilm degradation: (Also reviewed quorum sensing articles)
Alina Nakhamchik, C. W., and Dean A. Rowe-Magnus (2008). "Cyclic-di-GMP regulates extracellular polysaccharide production, biofilm formation, and rugose colony development by Vibrio vulnificus." Applied and Environmental Microbiology 74(13): 4199-4209.
Anneleen Cornelissen, P.-J. C., Jeroen T'Syen, Helena Van Praet, Jean-Paul Noben, Olga V. Shaburova, Victor N. Krylov, Guido Volckaert, Rob Lavigne (2011). "The T7-related Pseudomonas putida phage O15 displays virion-associated biofilm degradation properties." PLoS ONE 6(4): e18597.
Bassler, W.-L. N. a. B. L. (2009). "Bacterial Quorum-Sensing Network Architectures." Annual Reviews 43: 197-222.
Christopher M. Waters, W. L., Joshua D. Rabinowitz and Bonnie L. Bassler (2008). "Quorum sensing controls biofilm formation in Vibrio cholerae through modulation of cycli di-GMP levels and repression of vpsT." Journal of Bacteriology 190(7): 2527-2536.
Cynthia Wu, J. Y. L., Gerald G. Fuller, and Lynette Cegelski (2013). "Disruption of Escherichia coli  amyloid-integrated biofilm formation at the air-liquid interface by a polysorbate surfactant." Langmuir 29: 920-926.
D.H. Dusane, J. K. R., A.R. Kumar, Y.V. Nancharaiah, V.P. Venugopalan and S.S. Zinjarde (2008). "Disruption of fungal and bacterial biofilms by lauroyl glucose." Letters in Applied Microbiology 47: 374-379.
Jun Zhu, M. B. M., Russell E. Vance, Michelle Dzlejman, Bonnie L. Bassler and John J. Mekalanos (2002). "Quorum-sensing regulators control virulence gene expression in Vibrio cholerae." PNAS 99(5): 3129-3134.
Yildiz, J. C. N. F. a. F. H. (2007). "The rbmBCDEF gene cluster modulates development of rugose colony morphology and biofilm formation in Vibrio cholerae." Journal of Bacteriology 189(6): 2319-2330.
Yildiz, J. C. N. F. a. F. H. (2008). "Interplay between cyclic AMP-cyclic AMP receptor protein and cyclic di-GMP signaling in Vibrio cholerae biofilm formation." Journal of Bacteriology 190(20): 6646-6659.
-Begin growing cholera biofilms to test last years constructs -Sequence plasmid constructs on Wednesday (waiting until we have final decision on project)
3/20/13
KK Today is March 20th, two thousand thirteen
We were thrilled to get out plate of cholera today! Then we happened upon about 5 iGEM projects from previous years that involve quorum sensing and/or cholera. We were on the verge of beginning our lab work but our paradigm was suddenly shifted a few steps backward: we needed (and still need) to decide if we should continue working on our cholera project. Calgary’s 2008 iGEM team successfully cloned the quorum-sensing circuit into E.Coli and used it to detect and destroy E.Coli that was expressing AHL - whose homologs appears to be autoinducers of V. Harveyi and V.Fischeri. St. Andrew’s iGEM team faced off against cholera specifically, attempted to render it avirulent through generating high levels of cholera’s autoinducer CAI-1, but they did not tame cholera. They only managed to get the CAI-1 autoinducer to work in E.Coli.
We got our cholera from Annette, Dr. Robison’s lab manager. She plated it on Columbia media, and it had been sitting at 37 degrees C for 24 hours when she gave it to us. We don’t know the exact assay for growing biofilm in cholera, but some protocols suggest LB broth. Our plan is to use LB broth and let cholera grow (NOT shaking it, so that biofilm may grow freely).
March 20th Overview of E. coli transformation protocol: Eppendorfs of DH5alpha cells---thaw on ice Mix 2-5ul of plasmid and keep on ice for 20-30minutes Heat shock for 1 min @ 42 Degrees Celsius Cool eppendorf on ice for 2-5 minutes Add 0.5 mL LB and incubate at 37 Degrees Celsius for 30 minutes (select for AMP) Plate on selective media (LB-AMP) Phage Titer Protocol
Plan: -Need to set up plasmids for sequencing -Grow a Cholera biofilm -Gram-stain of Cholera to check
KP March 20th
Today we started off by picking up our cholera from Dr. Robison’s lab. Most of the day was spent researching previous iGEM teams that have already worked with cholera, quorum sensing, and biofilm degradation. Some of the previous teams projects are very similar to our own, so we are trying to decide whether to proceed with our current plans or to choose something else. Or, if we stick with our current plan, how can we incorporate phage or make our project unique?
Here are a few good papers on using phage to destroy biofilm.
https://www.novapublishers.com/catalog/product_info.php?products_id=21940
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1797112/
How Phages Kill Biofilm
http://www.biomedcentral.com/1472-6750/8/79
Phage That Are Capable of Destroying Biofilms
Bacteriophage Phi S1 infection of Pseudomonas fluorescens planktonic cells versus biofilms. Pseudomonas fluorescens biofilms subjected to phage phiIBB-PF7A. Lytic Activity of Recombinant Bacteriophage   11 and  12 Endolysins on Whole Cells and Biofilms of Staphylococcus aureus.
3/22/13
KK  March 22, 2013 (Friday) As a class we elected to continue forward with our cholera project, given that we emphasize elimination of the biofilm (which has NOT been accomplished by any other iGEM team) and using phage in our construct to do so. There are two approaches to using phage to eliminate the biofilm in Cholera. We may engineer a phage to do what we need, or we may go looking for a phage that has already evolved to do what we need.
Here are a few summaries of articles I read today:
The use of phages for the removal of infectious biofilms. Azeredo, J. and Sutherland I.W. (2008)  Used two phages isolated from sewage; showed 2-4 log decreases in bacterial population both in vitro and in vivo (murine model). OF INTEREST: In vitro testing involved biofilm and took longer; in vivo testing involved only planktonic cells and was faster. OF INTEREST: Calcofluor biofilm staining. Efficacy of cocktail phage therapy in treating Vibrio cholerae infection in rabbit model Summary: Oral administration of a cocktail of 5 lytic cholera bacteriophages to rabbits yielded strong and statistically significant reductions of cholera. Researchers showed that administration of the cocktail at 6 hour and 12 hour time intervals after bacterial challenge reduced bacteria populations 100 fold and 10 fold, respectively. Administration of the cocktail before bacterial challenge apparently did not significantly decrease cholera OF INTEREST: The emergence of multiple drug resistant strains of V. cholerae is thus a cause for global concern [4,5] and is prompting the exploration for alternative ways of treatment  OF INTEREST: To the best of our knowledge it is the first report of combatting V. cholerae infection in an animal model by phages adminis- tered through the oral route. Here are a few more articles that I've downloaded and I'm going to read:
Phage-bacterial interactions in the evolution of toxigenic Vibrio cholerae - this article exams the co-evolution of phage and cholera toxin. From what I read of the abstract, it is hypothesized that cholera's toxin was introduced by horizontal gene transfer from the CTXw phage. Written by a researcher from Harvard Medical School
O Antigen Is the Receptor of Vibrio cholerae Serogroup O1 El Tor Typing Phage VP4 - phage VP4 infects cholera, but certain mutations to the O-antigen gene provides resistance against VP4 (VP4 has anti-biofilm properties?)
Dispersing biofilms with engineered enzymatic bacteriophage - this is a synthetic biology approach to, like the name of the article implies, engineering phages to break down biofilms!! Written by a Harvard-MIT researcher
KP 3/22/13 Today we decided to stick with the cholera project. I am glad, because at first I was slightly disappointed when I realized that our project wasn’t as novel as I originally thought, but I am still very interested. Today we realized that the most important thing that we can do now is to find a way to disrupt the biofilm from the outside. I found a summary article on using phage against biofilms and each one of us chose a few article to read in order to understand the best ways to tackle the problem. I like the idea the paper that Kendall cited above that talks about bacteriophage that have been engineered to express enzymes that degrade the biofilm. I think that that is a good approach because we can combine our findings with the phage group and have a great presentation.  Also, there is a lot of research that I have found from the papers that I have read suggesting that it is very possible to do what we will be trying to do.
NRS 3/22/13 After discussing it in class we decided to stick with the cholera project. We redivided the group and now I am in the biofilm destruction using enzymes group. We are going to try and read at least three articles by monday to help get a scope of the project and decide what we need to do with it.
3/25/13
Whitney Hoopes 3/25/13
Papers to read:
E. Suman, S.J. D’souza, P. Jacob, M.R. Sushruth, M.S. Kotian. Anti-biofilm and anti-adherence activity of Glm-U inhibitors.
Ilana Kolodkin-Gal, Shugeng Cao, Liraz Chai, Thomas Bottcher, Roberto Kolter, Jon Clardy, Richard Losick. A self-produced trigger for biofilm disassembly that targets exopolysaccharide.
Anneleen Cornelissen, Pieter-Jan Ceyssens, Victor N. Krylov, Jean-Paul Noben, Guido Volckaert, Rob Lavigne. Identification of EPS-degrading activity within the tail spikes of the novel Pseudomonas putida phage AF.
N. Ramasubbu, L.M. Thomas, C. Ragunath, J.B. Kaplan. Structural analysis of Dispersin B, a biofilm-releasing glycoside hydrolase from the periodontopathogen Actinobacillus actinomycetemocomitans.
Ranjit, Dev K.; Endres, Jennifer L.; Bayles, Kenneth W. Staphylococus aureus CidA and LrgA proteins exhibit holin-like properties.
CH 3/25/13 Cholera distribution in the US: Cholera can be found in brackish water or estuaries near the ocean (people who have eaten certain kinds of seafood have gotten cholera). One source I found suggested that the reported cases of cholera from Oct 2010-Feb2011 were almost all from people who had traveled out of the country. The only cases that had no travel time were found in north eastern US. http://wwwnc.cdc.gov/eid/article/17/11/11-0808-f1.htm Generally in the world the most cholera cases are reported in Africa and South/Central America. The places where water purification is not so great. One problem with cholera is that countries don’t want to report that they have it because that scares away tourists.
Toxigenic Vibrio cholerae O1 in Water and Seafood, Haiti http://wwwnc.cdc.gov/eid/article/17/11/pdfs/11-0748.pdf This paper has an example of the difficultness involved in testing water for cholera contamination. They had to filter the water with a special filter then allow their sample to incubate for days and then plate test/PCR test.
I think we all need these shirts http://www.thebacteriophages.com/sales.htm
KK 25 of March, 2013
In 2007 Timothy K. Lu and James J. Collins published a paper, “Dispersing biofilms with engineered enzymatic bacteriophage,” in which the showed that they had engineered T7 and T3 phage to infect their E. Coli hosts with the gene for Dispersin B, which was released along with the phage progeny. The researchers achieved 99.997% degradation of the biofilm.  This is essentially what we are thinking to do: engineer a phage to infect a bacterial host with a gene that expresses a protein that will accelerate the degradation of cholera’s biofilm. These researchers show that it is possible for E.Coli. OUR PLAN:  Our first step needs to be to get cholera-specific phages to work with. CTX theta phage is a very famous and well characterized (we found a paper elucidating it’s entire genome) cholera-specific bacteriophage. It is a temperate phage. Chinese researchers used VP4 phage to study serotype O1 cholera. In India, researchers successfully treated cholera-infected rabbits with a cocktail of 5 phages: ATCC 51352- B1, B2, B3, B4, and B5. Tomorrow I will write an email to send to these professors that Dr. Grose will then forward to them on our behalf. Another approach to obtaining cholera-specific bacteriophage is to search for them in samples of Cholera-sullied water. Clarice researched WHERE Cholera is in the United States, but unsurprisingly no one is keen to publish that their water is infected with cholera. She didn’t find anything helpfully conclusive.  Once we have bacteriophages we will coordinate with Michael, Whitney, and Nathan to clone into its genome proteins that (hopefully) will degrade cholera’s biofilm. We are researching candidates proteins. We started growing cholera today and I’ve read several articles that mention assays for growing biofilms. Once we have our engineered phage we will begin testing.
CH 3/25/13 We again did some planning of what we need to do going forward.  Plan of Attack: 1. Study phages 2. Obtain different cholera associated Phages 3. Engineer phage to penetrate biofilm then produce biofilm degrading enzymes upon infection of cholera 4. Testing  I think we also need to do more research on V. cholerae in general. What makes up its biofilm etc.  Today I set up a sequencing reaction with reverse primer BI227 (1ul undiluted) and pJG78 (2ul undiluted).  Sequence obtained: AGCAGCATGTTTTTCGTCTCANTGGCNTTTTTNGGCAACATGGTGTTCACTAACACGTCAGTTGGCTAGGTGACCCTCGGCTTANAAGGGTCAATCTAATAAATGCAGATTACGTGCCAACTTTTCATTAGCGACTTTTGCATGAAAATCGGGCAAAAAACGAACAGTTTCATTACAACCACACTCATAAACGAATGACGAAATGCGATTTCTATTGCAAATTGCAAAATCAAAATTTGCAAAATGCAAAGGCGAGCGAAGCTTTTTACGGCTAGCTCAGTCCTAGGTATAGTGCTAGCAGGAGGTATTATATATGGTAGAAGACACGGCGTCGGTGGCGGCGCTGTATCGTTCTTACCTCACACCGCTGGATATTGATATCAATATCGTGGGTACCGGACGCGATGCCATCGAGAGTATTGGCCGTCGCGAGCCTGACTTAATCCTGCTGGATTTACGTTTGCCAGATATGACGGGGATGGACGTACTCTATGCAGTGAAAGAGAAATCACCGGATGTGCCTATCGTGTTTATGACCGCTCATGGTTCGATTGATACGGCGGTTGAAGCCATGCGTCATGGTGCGCAAGACTTTTTGATCAAGCCGTGTGAAGCGGACCGACTGCGGGTCACAGTGAATAATGCGATTCGCAAAGCCTCGAAACTCAAAAACGATGTTGATAATAAAAATCAAAACTATCAAGGCTTTATTGGTAGCAGTCAAACCATGCAGGCGGTGTACCGCACCATTGACTCTGCGGCGAGCAGCAAAGCCAGTATTTTTATTACCGGCGAAAGTGGTACAGGTAAAGAAGTGTGCGCGGAAGCGATTCATGCCGCGAGCAAGCGTGGAGATAAGCCATTTATCGCCATCAACTGTGCGGCCATTCCGAAAGATCTGATTGAAAGTGAGTTGTTTGGTCACGTCAAAGGGGCTTTNCTGGGGCGGCGACTGAGCGTCAGGCGCGGCAGAAGCGGCTGANGGGGGACCCTNCTTTTGGATGAATTGTGCGAATGGATCTGGATCTGCAGACCAACTTCTGCGCTTATTCNGACGGGACATTCCAAAANNNNGNNTTTCCAAAATGAAANNNTGAATGTGCNTTTTGTGNGNCATCGAACCGCGATCCGTGAGGNANTGCANNAGGACGCTTCTGAAGACCTGACCTACCGNCTGTATGGATCCGNCTGACATTACNNCATTGCCCCGCCANGGTGATATGNTANTGTAATGCCTTATTCACTGCCTGGCTTATGTCCAGGACGANGANGTCNCAAGACTAGG Next time we need to set up a transformation with pJG78 because there is not much of it left.
3/27/13
CH 3/27/2013 Set up transformation. 1ul purified plasmid into 50ul E.coli. RESULT: contamination issues either from the LB we were using or the plates being old.
KK March 27th, 2013
Dr. Grose has distributed the phage requests we sent her to send to professors in China, India, Switzerland, and Harvard. It would be wonderful to obtain CTXtheta phage, which is a cholera-specific phage whose genome has been well studied and inclusively its joint evolution with cholera has been studied. However, we when arrived at class and began discussing, Dr. Grose presented a better idea. We can use a lysogenic phage specific to E.Coli, rather than a cholera specific phage. We’ll engineer this phage with EPS depolymerases on its tail proteins, and then we will infect our E.Coli with the phage and grow E. Coli with copies of the virus prophage incorporated in its genome. If we can then engineer our bacteria so that the prophage DNA will be excised and expressed when E.Coli detects cholera, lysing the E.Coli and attacking the biofilm, then that will be an incredible iGEM project.   Clarice, Kelton and I transformed plasmid IG78 into E.Coli and plated on about 10 plates of LB Amp. During the transformation of our first 50 microliters of E.Coli, we think the hot water bath failed to achieve 42 degrees Celsius, and it’s likely the transformation didn’t work. For that reason, we did the transformation again (using only 1 microliter of plasmid instead of 2 microliters because we’re very low). We shocked the DH5alpha with the hot water bath in Dr. Griffitts lab instead of the one in ours. We hope to harvest the plasmid on Friday.
3/29/13
CH 3/29/2013 More discussion of where our project is going. Redid the transformation with help from Jordan. RESULT: No contamination issues. The transformation rate was very high. Two colonies were streaked to singles and grown up in overnight cultures (LB Amp) in preparation for a mini prep.
KK March 29, 2013 Bad news: our transformations were infected by who knows what type of bacteria. It wasn’t the LB, because the solution was clear on wednesday and was clear today as well. The tips were autoclaved. We think the plates we used may not have been completely sterile. So today we did the transformation again with 1 microliter plasmid and 50 microliters of DH5alpha. It is about the last of the plasmid we have to use. Jordan supervised the transformation to ensure it went well. He made a few new plates of LB-AMP just today so he’s confident they are sterile. NOTE: during the transformaiton we again had issues with the temperature of the hot-shock plate. The reading was 42 degrees C but our thermometer showed that the true temperature was lower. We increased the temperature of the hot plate to 46 degrees and the thermometer read the correct temperature for the protocol. So, there was no error in our procedure today. Jordan plated the E.Coli on two plates, each plate with a different volume of E.Coli, because he didn’t know how dilute the plasmid was in solution.
On Monday we will present a 30 minute report on how our research is going. I’m going to research over the weekend the bacteriophage Lambda. We won’t be able to use T4 or T7 in our cholera activated E.Coli construct because neither is a lysogenic phage. Both are lytic :( that only means that we won’t be able to unite our project to the other group’s project so seamlessly.
GREAT NEWS: Dr. Reidl of the University of Graz returned our correspondence and is going to be extremely generous with us. He will send us his K139 cholera phage that we asked for AND biofilm-producing cholera AND more, including a o-antigen knockout strain of cholera that Reidl’s lab engineered. Exciting!
KP 3/29/13
Today when we showed up at the lab, our beautifully plated e. coli infused with IG78 were not quite as beautiful as we had anticipated. Yes, something was growing, but unfortunately not what we had anticipated. So, with Jordan’s help, we re-did the transformation and hopefully we have what we need show up on Monday.   I did some research and found that T1-T7 bacteriophage are lytic bacteriophage. We unfortunately need lysogenic bacteriophage to incorporate into our e. coli, because we want to to stay dormant in the host genome until cholera is sensed. The sensing of cholera will set off the chain of events leading to the expression and creation of the bacteriophage, so it has to be lysogenic bacteriophage that we use. I will do more research to see if there are any examples of one of the T# phages being used as lysogenic phages, because that will help us to connect with the phage group and contribute to an overall better presentation and iGEM project.  We have our presentation on Monday and I will be giving a background of lytic and lysogenic phases of bacteriophage and why that is important to our project.
April
4/1/13
KK April 1st 2013
Our transformations worked this time and we had several beautiful lawns of E.Coli. The E. Coli did NOT fluoresce either red or green to the naked eye. We haven’t yet looked at them under UV light. Clarisse streaked several singles, and also set up two overnight colonies. Kelton and I practiced streaking to singles for the first time, using old plates. We placed our E.Coli singles in the 37 degree room to grow, along with the overnights on the shaker.  Most of today we spent presenting our progress thus far. It appears that bacteriophage K139 will not be very useful to us, as we will need to use E.Coli bacteriophage lambda for our system. Dr. Grose has ordered Lambda and it is on it’s way. Our goal is to incorporate the Lambda genome as a lysogenic prophage into the genome of E.Coli, and have that prophage remain dormant until the presence of cholera’s autoreceptor triggers the phage to be excised from E.Coli’s genome and become lytic. The lysogeny/lysis balance is determined by the relative concentrations of two proteins, CRO and CI. The two proteins are mutually exclusive; if one is being expressed, then the other cannot be expressed, because they share a promoter region. Expression of CRO triggers the events that lead to replication and lysis. Expression of CI induces and maintains lysogeny. Our plan is clone the CI protein into E.Coli following the Qrr4 protein and the CRO protein following HapR.
Kp April 1st  We started today with each group presenting their research and game plan for the coming weeks/months. It seems like we all know, or at least have a direction, to our next plan of action. Our E. coli turned out well and we were able to single out a colony in order to prepare overnight. After further research, it is clear that we will not be able to use T phages in our E. coli as a means to destroy cholera biofilm. We have all done more research on lambda, and it appears to be our best lysogenic phage choice. It is well studied and the mechanism for turning on the lytic cycle is very well understood.
4/3/13
KK April 3rd 2013
Not this Friday, but next Friday, we will have our Final Presentations. They will be done by sub-group; my sub-group is the cholera-phage approach. As part of the project, I need to attend a science seminar. Tomorrow I’ll attend the seminar by John Roth on something to do with molecular biology, then write a one-paragraph summary of HOW the presentation was done, the showsmanship of the presentation. On the day of our presentation we need to explain the background of what we have done, then go into our methods and results, then give our conclusions.  Today we followed the mini-prep/Alkaline lysis procedure to purify plasmids from our overnight samples. We followed the step-by step protocol given in the kit we used. We chose to use 1 mL of our E.Coli overnight solution in our samples ... and at the end, our plasmid concentrations were low (24 ng/microL and 16ng/microL, respectively). We used the spectrophotometer in Dr. Grose’s lab to determine the concentrations of our plasmids. So, in the future, we ought to a) include Dr. Grose as we follow the protocol to ensure we are doing things correctly, and b) probably use more of the E.Coli sample to get more plasmids and hopefully a higher concentration.
KP April 3rd 2013  Today we purified the plasmids and got them ready to be sequenced. We used Dr. Grose’s spectrophotometer. We learned how to analyze the graph in order to know if we just have DNA or if there is something else in our mixture after the purification process. The bump of our graph was right over 260, so we know that we successfully purified our plasmids! The only object of concern, is the fact that we had a very low concentration of plasmids. The instructions in the kit advised 1-3 mL of E. coli potion for high copy DNA. We used 1 mL, so next time we will try 2-3 mL to see if we can get a higher concentration. We are unsure if we did something wrong, if there is something wrong with the kit, or if we just need to use more of the E. coli overnight potion. I need to become more familiar with the sequence of our plasmid, and we need to find out why our E.coli doesn’t glow. When I understand the plasmid sequence, hopefully we can know what is missing or defective in our E. coli that keeps it from glowing.
4/5/13
KP 4/5/13 Today we made a plan of exactly what we need to do in the next week and coming months. Our biggest priority at this time is sequencing the plasmid from last years iGEM team. I was reading in the iGEM registry, and I saw that there is a pIG78 A and B. We tried A, but we haven’t tried B. On Monday we will try to transfer B into E. Coli and see if B works better than A. Next week our goal is to completely sequence the plasmid to see if something is missing or incorrect. I also need to do more research to understand better what last years team did and also the specifics of how we are going to set up our phage and how we are going to induce the lytic cycle in the presence of cholera.
KK 4/5/2013 Today we set out some long-term priorities for what we should do from here on out. Our priorities: 1.    Determine is the QS circuit working? We aren’t getting RFP right now … what’s wrong? a. We need to determine what primers were used to amplify the HapR and Qrr4 out of cholera b. What is the difference between pIG78 A and pIG78 B? There are two plasmids that are the same? We hope to KNOW the answers to these questions by Thursday 2.     Clone CI behind Qrr4 promoter; determine where in plasmid and order primers order primers 3.     Clone CRO behind HapR promoter DONE by Mid-May 4.     Modify tails of phage (after C1 and CRO steps complete) DONE by summer 5.     If possible, use selectable marker on phage to purify many E. Coli that have prophage incorporated into the genome. So these are the steps we set out, we need to discuss with Dr. Grose where we should be the C1 and CRO genes, because the plasmid is already very full. Personally, I need to understand the plasmid we’re using, the Lambda repression circuit, and the Lambda tail proteins better.
4/8/13
KK 4/8/2013 The other cholera-subgroup is successfully growing biofilm. They have found that cholera biofilm grows best in a salty solution (their best imitation of seawater) at 37 C. Today we set up a quick experiment to see how our pIG78d+ E.Coli would respond to the presence of cholera. Using a cholera sample that was NOT growing much biofilm (the sample had been grown in normal LB at 30 C), we set up 5 test tubes. 1. 4 mL plain LB 2. 4 mL plain LB + 500 microL V.Cholerae solution 3. 4 mL plain LB + pIG78D DH5alpha E.Coli 4. 4 mL plain LB + 100 microL V. Cholerae solution + pIG78D DH5alpha E.Coli 5. 4 mL plain LB + 500 microL V. Cholerae solution + pIG78D DH5alpha E.Coli
We set the overnight tubes in 30 C room on the shaker. Also, we set up two E.Coli overnights for miniprep on Wednesday.
As part of our presentation on Friday we plan to present the results of whether our plasmid is functioning to detect cholera or not and why. We are still waiting for the sequencing results to come back to us. We also will present the research we have done on Lambda.
KP 4/8/13 I worked with kendall to set up an overnight so that we can do a mini-prep tomorrow to continue our sequencing of our plasmid. After we completed our overnight prep, we set up an experiment to test to ability of our plasmid to give off a color response to cholera. We prepared the 5 tubes that Kendall explained above, we then put them in the 30 C room overnight and we will check on them tomorrow.
4/10/13
KK 4/10/2013 Results from our overnight cholera preps: The first three tubes from the left are controls, one with pure LB, one with cholera + LB (which was grown at 30 degrees by the other subgroup), and one with E.Coli + LB, respectively. The fourth and fifth tubes have cholera, E.Coli, and LB. The fourth has a slightly greater number of cholera (500 microL of cholera-saturated broth) compared to the fifth (100 microL of cholera-saturated broth). These are the results after two days of shaker-incubation at 30 degrees C, when the cultures are viewed under UV light.
We went over the quorum-sensing circuit with Dr. Grose. The Cqss membrane receptor phosphorylates Lux U, which cascades phosphorylation to LuxO. LuxO, when phosphorylated, binds to the Qrr4 promoter, activating transcription of Qrr4 mRNA. Qrr4 mRNA interferes with HapR, so there is no protein HapR. HapR is a repressor whose binding site is at the promoter for GFP. When Qrr4 eliminates HapR, there is no repression of the expression of GFP. Thus, GFP should be ON when Cholera is NOT present. The reverse is also true. In the presence of Cholera autoinducer, CQSS acts as a phophatase on LuxU, which does the same to LuxO, which does NOT activate Qrr4 mRNA which does NOT degrade HapR which represses transcription of GFP. Thus GFP should NOT glow in the presence of cholera.
Our results in these test tubes aren’t exactly in agreement with what we want.
KP 4/10/2013 Today we worked on our presentation for Friday and we also went over the quorum sensing with Dr. Grose.
4/12/13
KP 4/12/13 We had our presentation today. It was good practice and I learned some things to improve on in the future. There was some confusion on the quorum sensing, because we thought that HapR was a repressor and that it shouldn’t glow green in the presence of cholera. It turn out that it is the other way around and that it should glow green in the presence of cholera and red in the absence of cholera. That still doesnt explain why our tests are glowing green in the absence of cholera. We will have to work backwards and find out exactly what it is that we have. I feel that we have a clearer view of what we need to do.  Also, I need to do more research other than wikipedia on lambda phage and its lytic and lysogenic cycles, because in our presentation, I said that that lambda enters the lytic cycle when it is in a cell that is healthy and lysogenic cycle when the cell is in trouble. That is what it says on wikipedia. Anyway, when I said that, I was told that I was wrong and that it is the other way around. I will look into that. We also hopefully got our quorum sensing system worked out now. All we need to do (once we figure out what we actually have in our plasmids) is add the cro transcription site after the GFP, and in the presence of cholera more cro will be produced and lambda will enter the lytic cycle!
KK 4/12/13 Today was our presentation in class. This is the link to our powerpoint presenation: https://docs.google.com/presentation/d/14mUxVrUdTvNXaPGifXZs_lpXeMrFd-4CxSZ9o1bPqYo/edit#slide=id.p Our presentation gave a background for our project - though we ought to have been clearer WHY we want to use phage - and also the work that we’ve accomplished thus far in our group. We went a long time over in our presentation; it took a long time to explain the quorum sensing circuit, Lambda, and other things. Dr Grose sent an email with several suggestions for presentations in the future. They include being more concise with our time management and better coordinating within our group so that we don’t end up repeating ourselves. In the future we will be expected to be more polished presenters.
4/15/13
KP 4/15/12 Today we started out by getting photographed. We also watched the phage group’s presentations. It seems as if they have a good start to their project. We didn’t have too much time to do our own experiments, but we did set up an overnight to do mini-prep tomorrow, and we streaked out our plates of lambda phage.
KK 4/15/13 The three phage groups presented today. I realized how very quickly each of our groups has specialized to the point that I have to concentrate deeply to understand what seems to be second nature to members of the phage group. That’s good! It means each of our groups is moving along nicely. The objective of the phage project is to make a library of sizes - the biggest and smallest phage capsids possible using phages that have already been well characterized and approved for medicinal purposes. One group is working on selecting for the largest phage possible, one for the smallest, and another group is developing a method to purify both phages as they are made. Afterwards, we laid out singles of our three E.Coli strains with Lambda prophage incorporated into its genome. Tomorrow Kelton and I are going to go in and check on them after our Intro to Medicine class.
May
5/1/13
KK 5/1/2013
Over the break we did little Lab work. Kelton was in Rexburg, and Clarice and I didn’t have the necessary primers to continue working. Today the primers came! We submitted primers for all the genes that have been cloned into pIG78 with pIG78 for sequencing. (I believe we included the primers for all the genes). We also have primers for working with CRO in the pBAD plasmid. Today our assignment was to make goals and set plans for what we hope to accomplish by the end of the term. Our plans are to be submitted by Friday. Our plans are spelled out in a table we’re printing, but they include understanding (via sequencing) what is ocurring in the plasmid by May 13th, correcting our system by May 29th, and demonstrating that we can induce Lambda into lysis through expression CRO. We will place CRO on the pBAD plasmid with the pBAD promoter (or on pLAT with a pBAD promoter), and the pBAD promoter is induced by arabinose.
KP 5/1/13 Today we were trained in Cholera handling safety and we made our plans and goals for a summer full of success.
5/3/13
KP 5/3/13 Today Kendall and I did PCR for the first time. Jordan and Clarice showed us how to do it. We did PCR on our E. Coli that has our lysogenic lambda encased within. We also froze down our lambda e. coli samples for future use.
KK 5/3/13 Kelton and I worked with Jordan today to PCR amplify the CRO gene from π9907-infected E.Coli. We boiled the E.Coli to use as template and then followed the PCR protocol as outlined. Jordan actually was performed most of the protocol so that we could learn. Our control was E.Coli that had not been infected with lambda.
5/6/13
KK 5/6/13 We ran a gel of our PCR product that we had created over the weekend. This was the first time I had ever done so, so it was fun and interesting! To make the gel, we mix 100 mL of TAE buffer with 1 gram of agar and heat in the microwave. The agar powder needs to completely dissolve. That mixture, with ethidium bromide, is added to the gel dock, and let set for about 20 minutes, or if you set the gel in the fridge it is a little less time. The CRO gene is about 300 base pairs. However, when we ran our PCR product against the ladder and against our negative control, our PCR product matched the control and did NOT match the length that indicates 300 base pairs. So, we know that our PCR reaction failed. It may be because we used a colony that did not include that lambda prophage. This is what our gel looks like: Today we did the PCR reaction to amplify CRO again, but this time we amplified CRO from colonies that had been infected with our three distinct strains of Lambda. We will check our product tomorrow.
KP 5/6/13 Today we ran a gel to check our PCR products. Unfortunately it ran farther than it should have and our control came out the same as our test, so we obviously didn’t get what we wanted. Today we started three new PCR’s. BI7701, BI7707, BI23----. We will check them on Wednesday and hopefully we get the results that we want.Our picture.
5/8/13
KP 5/8/13 We did a few different procedures today. We started out by doing a PCR purification for BI7701, BI7707, and BI23.... It was successful, but our concentration was very low (11.9). We also set up another PCR for BI7701, BI7707, and BI23... It will be done tomorrow. Today we also did a digest of our plasmid and our cro insert.
KK 5/8/13 Having confirmed that our CRO insert was indeed PCR amplified (see picture above; very faint lines about 500 base pairs; wells 1-3 represent three strains of lambda, while 4 is a negative control), we performed a PCR cleanup on ALL 3 of our samples, in an attempt to isolate concentration of CRO insert possible. On the spectrophotometer our A260 reading gave us a concentration of 11.9 ng/microL.   Following that we ran a digest of our CRO insert and pLAT plasmid with pBAD promoter using restriction enzymes PST1-HF and ECOR1-HF. Our pLAT sample was a mixture of pIG12 and pIG13, which according to the parts database are the same plasmid, taken from two different colonies.  Having set our vector and inserts to digest, we started a low-melt gel with Jordan’s help. Low melt gels follow a slightly different protocol than normal gels, and use a smaller dock. Our dock was broken so our gel didn’t set very well. Because we don’t have time today, tomorrow we’ll run our vector and insert on the low-melt gel to see what happens.
5/10/13
KK 5/10/13 We were able to cut out our vector from the low melt gel, but our insert was not visible. So, today, we made preparations to run our CRO insert on a low melt gel again. I made the low melt gel and set it to cool, and to remain in the fridge over the weekend. Kelton and Clarice performed the PCR cleanup of a second CRO insert PCR reaction that we ran. The PCR was more successful this time - the bands were much more clearly visible, in all three of our Lambda samples (see the photo that Kelton will upload).
KP 5/10/13 Today we did a PCR cleanup on our Cro PCR products. We also set up a slow melt gel so we can try again on Monday to get a cutout of the insert in order to combine our plasmid and insert. We already have the plasmid, we just have to wait on the insert because it wasn’t visible in our first slow melt gel. Here is a picture of our second PCR product run on a 1 kb ladder gel:
5/13/13
KK 5/13/13 Today we ran our digested CRO insert on the low melt gel, and under the UV light we saw the band we were looking for. Clarisse excised the band and we performed a ligation reaction according to the protocol and using the vector that we had already run on a low-melt gel last Thursday. After incubating for 30 minutes at room temperature, we performed a transformation into DH5alpha. The selectable marker of pLAT is ampicillin.
KP 5/13/13 We ran our low melt gel with our Cro insert. After 45 minutes, we put the gel under UV light and cut out the insert. We then performed a ligation of our vector and cro insert. After 30 min, we did a transformation and inserted our newly ligated plasmid into DH5alpha.
5/14/13
KK 5/14/13 (Thursday) When we dropped by today two of our plates (out of four) were contaminated. The two contaminated plates, we noticed, were the two plates that were old. The other two plates had a few colonies, but the was not a significantly greater number of colonies in our plasmid + insert + E.Coli plate than there were in our control plasmid + E.Coli plate. Kelton went ahead and streaked the few colonies we had to singles. We can PCR-verify if any of them have the plasmid. Meanwhile I am going to redo the transformation today. I will plate 4 plates: one 50 microL test, one 100 microL test, one 50 microL control, and one 100 microL control.
5/15/13
KK Yesterday's transformation of pIG13+CRO insert into E.Coli yielded several colonies on both the plate inoculated with 50 microL and the plate inoculated 100 microL. We plated 8 separate single colonies and will wait until Friday, when we will plan to PCR verify which of the colonies has our vector and insert. Afterward, we discussed what we will do on Friday. We'll spend the hour preparing samples for PCR and a few hours later Clarisse is going to drop by to run the samples on a gel.
Also, I streaked lines and waves of V.cholerae today on a plate and for curiosity streaked E.Coli with pIG78 next to it in lines, and in waves intersecting the wavy cholera lines. We want to see if there are varying degrees of fluorescence.
KP 5/15/13 Today I re-streaked our ligation products. I re-streaked the control (pLat vector only) and the test (pLat vector and cro insert.) The hope, is that we can get individual colonies from the plates, so that we can sequence our plasmid to see if the vector took up the insert and if we have what we think that we do. We we will need primers IG57 and IG58 for our sequencing of this plasmid on Friday.
5/17/13
KK All our singles grew up well, so today we set up 10 PCR reactions + our control to test which, if any, had our vector + CRO insert construct. We also checked the plate that we streaked with Cholera and E.Coli with pIG78 next to one another. We wanted to see if the E.Coli would fluoresce at all in the presence of cholera. Under a UV light we didn't observe any noticeable difference between the streaks of cholera and the streaks of E.Coli
5/20/13
KK Clarisse left for Russia on Sunday, so she'll be gone for perhaps a month working at her internship. At the beginning of class we talked about possible community projects. The first project we may want to do is publish a children's book. The other two projects that received votes were: creating a board game, and sponsoring a fun run to raise money for Haiti. I called my friend Redge Ballard, an animation major, to ask if he would be interested in designing the book.
We PCR verified our 10 colonies and only colony A had the CRO insert we were looking for. We set up overnights of colony A to do a plasmid prep and then sequence, and we also set up overnights of our three lambda strains to plate tomorrow as a lawn. Once we do that, we hope to be induce lambda to go lytic by plating our E.Coli with CRO.
5/22/13
KK, KP
Yesterday I came in and did a plasmid prep of colonies A and W, which looked promising for having CRO+pIG11 on our PCR readout. Today we confirmed on the spectrophotometer the concentrations of the plasmids. The plasmid from A was present at a concentration of 158 ng/microL, and that of W had a concentration of 62 ng/microL. Yesterday we also plated our three strains of lambda-infected E.Coli, TT9901, TT9907, and TT23281, so today we were ready to trasform the plasmid into these three plasmids using electrophoration. Dr. Grose showed us how to use the electrophorator. We shocked each strain with plasmid from colony A and then a control plasmid separately, for a total of 6 electrophorations. After letting the cells recover for 30 minutes at 37 degrees celsius, we plated about 1 mL of each on LB/Amp plates. Tomorrow, hopefully, we will see a few colonies indicating that they took up our plasmid! Also, we submitted plasmids from A and W for sequencing.
5/24/13
KK, KP Our electrophoration worked on 2 of the 3 strains - on TT9901, and TT9907, but not on TT23281. Yesterday we went ahead and set up overnights to plate the two successful transformed strains on Amp/Arabinose, but our Amp/Arabinose plates are no longer good, so we need to make more. We redid the electroporation today so that hopefully by next wednesday all three strains should be ready and we should have plates. We'll be testing to see if TT9901, TT9907, and TT23281 with Lambda's CRO gene cloned into them can induce the integrated prophage out of lysogeny and into the lytic cycle.
5/27/13
KK, KP
We saw the results of our second electroporation today. Again, strains TT9901 and TT9907 took up the plasmid by electroporation, but the TT25281 transformation was again unsuccessful. We don't know what's wrong. Last time around I failed to push the cuvette entirely into the probes of the electrophorator. This time however, we know we followed the protocol correctly ...
In reference to strains TT9901 and TT9907, which were successful transformations from the first time, yesterday we swabbed these strains and grew them in overnight test tubes at 37 degress. The medium we used was LB Amp/Arabinose. We were hopeful that the following day (that's today) we would not see any turbidity, because the arabinose would have induced the expression of CRO, inducing Lambda to its lytic cycle. However, today we found disappointing results: our overnights were saturated with bacterial growth. This assay isn't of course the best way to test whether our system is working, but as it was intended to at least give us a ballpark gauge as to the functionality of our clone, we are concerned that something is not right.
5/29/13
KP KK Today we plated TT9907 and TT9901. We plated them on LB arabinose/amp/x-gal plates. We plated different concentrations of TT9907 and TT9901 on each of the plates. We added 50, 100, and 300 microliters onto each plate. We will check the plates tomorrow hopefully to find plagues showing that arabinose triggered lysis of our lambda. We unfortunately didn't get any growth of our TT23821, so we streaked another plate from our original stab sample and we will try to re-electroporate TT23821 on Friday.
5/31/13
June
6/3/13
KK KP Our PCR product was consistent with every other attempt to amplify CRO: it failed. Consistency is only a virtue if you're not a screw-up :) Anyway, our gel showed a faint band at the 300 base pair level, but too faint to be considered successful. We considered a list of reasons why our PCR reaction is failing and discussed them with Dr. Grose:
We boiled our template DNA for 10 minutes, but Dr. Grose recommends 5 minutes
We added 1 microliter of template; Dr. Grose likes to add 2 microliters.
We did not use a master mix to combine our reagents with our template DNA. We added each reagent individually to each tube. Because we are pipetting such small volumes, there is a lot of room for error this way.
The Phusion program we used was set to an annealing temperature that was 1 degree too low for Phusion.
We did the PCR reaction one more time with Dr. Grose. Tomorrow we expect to see an enormous, fat, band!
6/5/13
6/7/13
6/10/13
6/12/13
6/14/13
6/17/13
6/19/13
6/21/13
6/24/13
6/26/13
6/28/13
July
7/1/13
7/3/13
7/5/13
7/8/13
7/10/13
7/12/13
7/15/13
7/17/13
7/19/13
7/22/13
7/24/13
7/26/13
7/29/13
7/31/13
August
8/2/13
8/5/13
8/7/13
8/9/13
8/12/13
8/14/13
8/16/13
8/19/13
8/21/13
8/23/13
8/26/13
8/28/13
8/30/13
This is  a test!!!
September
9/2/13
9/4/13
9/6/13
9/9/13
9/11/13
9/13/13
9/16/13
9/18/13
9/20/13
9/23/13
9/25/13
9/27/13
9/30/13
Retrieved from " http://2013.igem.org/Team:BYU_Provo/Cholera_-_Detection "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
