"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/SmallPhage/Cholera - Detection/Period4/Dailylog
From 2013.igem.org
Home
Team
Overview
Official Team Profile
Team Video
Undergraduates
Mentors
Project
Overview
Background
Experimental Design
Notebook
Overview
Phage Team
Cholera Team
Results
Overview
Judging Criteria
Experimental
Modeling
Parts Submitted to the Registry
A Taste of Toronto
Human Practices
Overview
Orem Public Library Visit
The Adventure of Baxter Bacteria
Safety
Attributions & Collaborations
Attributions & Acknowledgements
Collaborations
iGEM 2013
Cholera Detection March - April Notebook: April 15 - April 30 Daily Log
Overview
March-April
May-June
July-August
September-October
4/15/13
- KP Today we started out by getting photographed. We also watched the phage group’s presentations. It seems as if they have a good start to their project. We didn’t have too much time to do our own experiments, but we did set up an overnight to do mini-prep tomorrow, and we streaked out our plates of lambda phage. - KK The three phage groups presented today. I realized how very quickly each of our groups has specialized to the point that I have to concentrate deeply to understand what seems to be second nature to members of the phage group. That’s good! It means each of our groups is moving along nicely. The objective of the phage project is to make a library of sizes - the biggest and smallest phage capsids possible using phages that have already been well characterized and approved for medicinal purposes. One group is working on selecting for the largest phage possible, one for the smallest, and another group is developing a method to purify both phages as they are made. Afterwards, we laid out singles of our three E.Coli strains with Lambda prophage incorporated into its genome. Tomorrow Kelton and I are going to go in and check on them after our Intro to Medicine class.
4/3/13
- KK Not this Friday, but next Friday, we will have our Final Presentations. They will be done by sub-group; my sub-group is the cholera-phage approach. As part of the project, I need to attend a science seminar. Tomorrow I’ll attend the seminar by John Roth on something to do with molecular biology, then write a one-paragraph summary of HOW the presentation was done, the showsmanship of the presentation. On the day of our presentation we need to explain the background of what we have done, then go into our methods and results, then give our conclusions. Today we followed the mini-prep/Alkaline lysis procedure to purify plasmids from our overnight samples. We followed the step-by step protocol given in the kit we used. We chose to use 1 mL of our E.Coli overnight solution in our samples ... and at the end, our plasmid concentrations were low (24 ng/microL and 16ng/microL, respectively). We used the spectrophotometer in Dr. Grose’s lab to determine the concentrations of our plasmids. So, in the future, we ought to a) include Dr. Grose as we follow the protocol to ensure we are doing things correctly, and b) probably use more of the E.Coli sample to get more plasmids and hopefully a higher concentration.
- KP April Today we purified the plasmids and got them ready to be sequenced. We used Dr. Grose’s spectrophotometer. We learned how to analyze the graph in order to know if we just have DNA or if there is something else in our mixture after the purification process. The bump of our graph was right over 260, so we know that we successfully purified our plasmids! The only object of concern, is the fact that we had a very low concentration of plasmids. The instructions in the kit advised 1-3 mL of E. coli potion for high copy DNA. We used 1 mL, so next time we will try 2-3 mL to see if we can get a higher concentration. We are unsure if we did something wrong, if there is something wrong with the kit, or if we just need to use more of the E. coli overnight potion. I need to become more familiar with the sequence of our plasmid, and we need to find out why our E.coli doesn’t glow. When I understand the plasmid sequence, hopefully we can know what is missing or defective in our E. coli that keeps it from glowing.
4/5/13
- KP Today we made a plan of exactly what we need to do in the next week and coming months. Our biggest priority at this time is sequencing the plasmid from last years iGEM team. I was reading in the iGEM registry, and I saw that there is a pIG78 A and B. We tried A, but we haven’t tried B. On Monday we will try to transfer B into E. Coli and see if B works better than A. Next week our goal is to completely sequence the plasmid to see if something is missing or incorrect. I also need to do more research to understand better what last years team did and also the specifics of how we are going to set up our phage and how we are going to induce the lytic cycle in the presence of cholera.
- KK Today we set out some long-term priorities for what we should do from here on out. Our priorities: 1. Determine is the QS circuit working? We aren’t getting RFP right now … what’s wrong? a. We need to determine what primers were used to amplify the HapR and Qrr4 out of cholera b. What is the difference between pIG78 A and pIG78 B? There are two plasmids that are the same? We hope to KNOW the answers to these questions by Thursday 2. Clone CI behind Qrr4 promoter; determine where in plasmid and order primers order primers 3. Clone CRO behind HapR promoter DONE by Mid-May 4. Modify tails of phage (after C1 and CRO steps complete) DONE by summer 5. If possible, use selectable marker on phage to purify many E. Coli that have prophage incorporated into the genome. So these are the steps we set out, we need to discuss with Dr. Grose where we should be the C1 and CRO genes, because the plasmid is already very full. Personally, I need to understand the plasmid we’re using, the Lambda repression circuit, and the Lambda tail proteins better.
4/8/13
- KK The other cholera-subgroup is successfully growing biofilm. They have found that cholera biofilm grows best in a salty solution (their best imitation of seawater) at 37 C. Today we set up a quick experiment to see how our pIG78d+ E.Coli would respond to the presence of cholera. Using a cholera sample that was NOT growing much biofilm (the sample had been grown in normal LB at 30 C), we set up 5 test tubes. 1. 4 mL plain LB 2. 4 mL plain LB + 500 microL V.Cholerae solution 3. 4 mL plain LB + pIG78D DH5alpha E.Coli 4. 4 mL plain LB + 100 microL V. Cholerae solution + pIG78D DH5alpha E.Coli 5. 4 mL plain LB + 500 microL V. Cholerae solution + pIG78D DH5alpha E.Coli We set the overnight tubes in 30 C room on the shaker. Also, we set up two E.Coli overnights for miniprep on Wednesday. As part of our presentation on Friday we plan to present the results of whether our plasmid is functioning to detect cholera or not and why. We are still waiting for the sequencing results to come back to us. We also will present the research we have done on Lambda.
- KP I worked with kendall to set up an overnight so that we can do a mini-prep tomorrow to continue our sequencing of our plasmid. After we completed our overnight prep, we set up an experiment to test to ability of our plasmid to give off a color response to cholera. We prepared the 5 tubes that Kendall explained above, we then put them in the 30 C room overnight and we will check on them tomorrow.
4/10/13
- KK Results from our overnight cholera preps: The first three tubes from the left are controls, one with pure LB, one with cholera + LB (which was grown at 30 degrees by the other subgroup), and one with E.Coli + LB, respectively. The fourth and fifth tubes have cholera, E.Coli, and LB. The fourth has a slightly greater number of cholera (500 microL of cholera-saturated broth) compared to the fifth (100 microL of cholera-saturated broth). These are the results after two days of shaker-incubation at 30 degrees C, when the cultures are viewed under UV light. We went over the quorum-sensing circuit with Dr. Grose. The Cqss membrane receptor phosphorylates Lux U, which cascades phosphorylation to LuxO. LuxO, when phosphorylated, binds to the Qrr4 promoter, activating transcription of Qrr4 mRNA. Qrr4 mRNA interferes with HapR, so there is no protein HapR. HapR is a repressor whose binding site is at the promoter for GFP. When Qrr4 eliminates HapR, there is no repression of the expression of GFP. Thus, GFP should be ON when Cholera is NOT present. The reverse is also true. In the presence of Cholera autoinducer, CQSS acts as a phophatase on LuxU, which does the same to LuxO, which does NOT activate Qrr4 mRNA which does NOT degrade HapR which represses transcription of GFP. Thus GFP should NOT glow in the presence of cholera. Our results in these test tubes aren’t exactly in agreement with what we want.
- KP Today we worked on our presentation for Friday and we also went over the quorum sensing with Dr. Grose.
4/12/13
- KP We had our presentation today. It was good practice and I learned some things to improve on in the future. There was some confusion on the quorum sensing, because we thought that HapR was a repressor and that it shouldn’t glow green in the presence of cholera. It turn out that it is the other way around and that it should glow green in the presence of cholera and red in the absence of cholera. That still doesnt explain why our tests are glowing green in the absence of cholera. We will have to work backwards and find out exactly what it is that we have. I feel that we have a clearer view of what we need to do. Also, I need to do more research other than wikipedia on lambda phage and its lytic and lysogenic cycles, because in our presentation, I said that that lambda enters the lytic cycle when it is in a cell that is healthy and lysogenic cycle when the cell is in trouble. That is what it says on wikipedia. Anyway, when I said that, I was told that I was wrong and that it is the other way around. I will look into that. We also hopefully got our quorum sensing system worked out now. All we need to do (once we figure out what we actually have in our plasmids) is add the cro transcription site after the GFP, and in the presence of cholera more cro will be produced and lambda will enter the lytic cycle!
- KK Today was our presentation in class. This is the link to our powerpoint presenation: https://docs.google.com/presentation/d/14mUxVrUdTvNXaPGifXZs_lpXeMrFd-4CxSZ9o1bPqYo/edit#slide=id.p Our presentation gave a background for our project - though we ought to have been clearer WHY we want to use phage - and also the work that we’ve accomplished thus far in our group. We went a long time over in our presentation; it took a long time to explain the quorum sensing circuit, Lambda, and other things. Dr Grose sent an email with several suggestions for presentations in the future. They include being more concise with our time management and better coordinating within our group so that we don’t end up repeating ourselves. In the future we will be expected to be more polished presenters.
Retrieved from " http://2013.igem.org/Team:BYU_Provo/Notebook/SmallPhage/Cholera_-_Detection/Period4/Dailylog "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
