"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/CholeraDetection/Springexp/Period3/Dailylog
From 2013.igem.org
Home
Team
Overview
Official Team Profile
Team Video
Undergraduates
Mentors
Project
Overview
Background
Experimental Design
Notebook
Overview
Phage Team
Cholera Team
Results
Overview
Judging Criteria
Experimental
Modeling
Parts Submitted to the Registry
A Taste of Toronto
Human Practices
Overview
Orem Public Library Visit
The Adventure of Baxter Bacteria
Safety
Attributions & Collaborations
Attributions & Acknowledgements
Collaborations
iGEM 2013
Cholera Detection May - June Notebook: May 27 - June 9 Daily Log
Cholera Detection
March-April
May-June
July-August
September-October
5/27/13
KK, KP
We saw the results of our second electroporation today. Again, strains TT9901 and TT9907 took up the plasmid by electroporation, but the TT25281 transformation was again unsuccessful. We don't know what's wrong. Last time around I failed to push the cuvette entirely into the probes of the electrophorator. This time however, we know we followed the protocol correctly ...
In reference to strains TT9901 and TT9907, which were successful transformations from the first time, yesterday we swabbed these strains and grew them in overnight test tubes at 37 degress. The medium we used was LB Amp/Arabinose. We were hopeful that the following day (that's today) we would not see any turbidity, because the arabinose would have induced the expression of CRO, inducing Lambda to its lytic cycle. However, today we found disappointing results: our overnights were saturated with bacterial growth. This assay isn't of course the best way to test whether our system is working, but as it was intended to at least give us a ballpark gauge as to the functionality of our clone, we are concerned that something is not right.
5/29/13
-KP KK Today we plated TT9907 and TT9901. We plated them on LB arabinose/amp/x-gal plates. We plated different concentrations of TT9907 and TT9901 on each of the plates. We added 50, 100, and 300 microliters onto each plate. We will check the plates tomorrow hopefully to find plaques showing that arabinose triggered lysis of our lambda. We unfortunately didn't get any growth of our TT25821, so we streaked another plate from our original stab sample and we will try to re-electroporate TT23821 on Friday.
5/31/13
KK, KP NO! Our work over these past few weeks reached its climax today ... we checked our plates, but no dice. No plaques, that is. Lambda was not induced out of lysogeny. We are asking ourselves these questions: Was CRO not actually ligated into our vector? Are our plates not good (we used year old plates)? CRO, when overexpressed, downregulates itself; is there too high a concentration of CRO? Or is it possible that CRO alone isn't a sufficient factor to induce lysis. Dr. Grose talked with us, and told us she had our sequencing!! The results were depressing but revelatory: the sequence for CRO is not in our vector! So, on Monday we will begin the cloning process again. Today we set up a PCR reaction to amplify CRO from strains TT9901 and TT9907 template DNA.
6/3/13
-KK KP Our PCR product was consistent with every other attempt to amplify CRO: it failed. Consistency is only a virtue if you're not a screw-up :) Anyway, our gel showed a faint band at the 300 base pair level, but too faint to be considered successful. We considered a list of reasons why our PCR reaction is failing and discussed them with Dr. Grose: We boiled our template DNA for 10 minutes, but Dr. Grose recommends 5 minutes We added 1 microliter of template; Dr. Grose likes to add 2 microliters. We did not use a master mix to combine our reagents with our template DNA. We added each reagent individually to each tube. Because we are pipetting such small volumes, there is a lot of room for error this way. The Phusion program we used was set to an annealing temperature that was 1 degree too low for Phusion. We did the PCR reaction one more time with Dr. Grose. Tomorrow we expect to see an enormous, fat, band!
6/5/13
Regarding our PCR attempt yesterday, we did see a thick fat band ... at the bottom of the gel! Once again our PCR failed. This has led us to propose several reasons why we can't PCR amplify CRO: 1) the Lambda prophage isn't actually in TT 9901 or TT9907, or 2) our primers aren’t functioning to amplify CRO. We streaked a few plates of TT9901 and TT9907 to subject to UV light. If we see plaques, then we will know Lambda has been induced. Meanwhile Dr. Grose is going to check that our primers are the correct ones.
6/7/13
KK, KP After a little investigation, it turns out our primers were NOT for the CRO gene in Lambda, but for the CRO gene in another phage. Dr. Grose immediately ordered another set of primers, which arrived this morning! With the correct primers to amplify CRO, we performed the PCR protocol again today. Also, we set up an overnight of E.Coli with pIG12 to purify the plasmid tomorrow. We'll use pIG12 as our vector.
Our community project will be writing a children's book on synthetic biology. I wrote a story line which was distributed among all the team members; with their ideas I'm going to make changes and contact Redge Ballard, our illustrator. Since we hope to do our community project in one month, hopefully he can illustrate the story in such a short time!
6/9/13
CH Did a PCR Check for the Cro insert on 7 colonies of modified pIG12. The PCR was not promising so we decided to mini prep colonies labled 1 and 2 to sequence to check for the cro insert. The primer IG12 was used which is a reverse primer outside of the MCS. The sequencing results showed that Cro was cloned into pIG12 perfectly. Sequencing for pIG78 was also set up. Primers used: BI218 (HapR F), BI220 (GFP R), BI225 (RFP R), BI227 (Qrr4 R), B248 (LuxO mid), BI229 (LuxO R), BI230 (CqsS F), BI 247(CqsS mid), BI222 (GFP R).
Retrieved from " http://2013.igem.org/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period3/Dailylog "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
