"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/Cholera - Enzyme/September/Period1/Dailylog
From 2013.igem.org
Home
Team
Overview
Official Team Profile
Team Video
Undergraduates
Mentors
Project
Overview
Background
Experimental Design
Notebook
Overview
Phage Team
Cholera Team
Results
Overview
Judging Criteria
Experimental
Modeling
Parts Submitted to the Registry
A Taste of Toronto
Human Practices
Overview
Orem Public Library Visit
The Adventure of Baxter Bacteria
Safety
Attributions & Collaborations
Attributions & Acknowledgements
Collaborations
iGEM 2013
Cholera - Enzymes Notebook: September 1 - September 14 Daily Log
Cholera Enzyme
March-April
May-June
July-August
September-October
Protocols
9/4/13
Today we set up new overnights of cholera by adding 4 mL of SLB and 50 uL of our previous overnight from 8/26/13 into a test tubes and placing in the 30° incubator overnight. We prepared two overnight cultures and one control.
We also ran the GenElute Plasmid Miniprep Kit on the recombinant E. coli cells containing the piG91 plasmid using the protocol below to pull the iGem plasmid backbone out of the E. coli cells in preparation to clone our biobrick parts into the iGem plasmid backbone for submission to the iGem registry. We prepared two minipreps and the final product concentrations were: 28.1 ng/uL and 36.0 ng/uL.
Protocol
Pellet 1-5 mL of overnight culture of recombinant E. coli culture by centrifugation at ≥ 12,000 × g for one minute. Resuspend pellet in 200 uL of provided Resuspension Solution.
Lyse resuspended cells by adding 200 uL of Lysis solution. Immediately mix contents by gentle inversion 6-8 times, until the mixture becomes clear and viscous. Allow the Lysis reaction to proceed for approximately two minutes, but do not exceed five minutes.
Neutralize lysis and precipitate cell debris by adding 350 uL of Neutralization/Binding solution. Gently invert 4-6 times. Pellet cell debris by centrifuging at ≥ 12,000 × g for 10 minutes. Cell debris will precipitate out.
Insert a GenElute Miniprep Binding Column into a microcentrifuge tube. Add 500 uL of Column Preparation Solution and centrifuge at ≥ 12,000 × g for 30 seconds to 1 minute; discard flow through liquid.
Transfer clear lysate from step 3 to column from step 4 and centrifuge at ≥ 12,000 × g for 30 seconds to 1 minute; discard flow through liquid.
Add 750 uL of the diluted wash solution to the column. Centrifuge at ≥ 12,000 × g for 30 seconds to 1 minute. Discard flow through liquid and centrifuge again for 1 to 2 minutes, to remove excess ethanol.
Transfer column to a fresh collection tube. Add 50 uL of Elution Solution to the column and let stand for 5 minutes.
We then set up Restriction Digests for the Cro PCR, Qrr4/RFP PCR, and piG91 mini prep products using the setup shown below.
Restriction Digest setup
7.5 uL H2O    4.0 uL Buffer 4    4.0 uL 10x BSA    20 uL DNA template    1.5 uL Xba 1    1.5 uL Spe 1
Digests Labelled
Cro PCR
Qrr4/RFP PCR
piG91 miniprep
As well we set up phusion and Taq PCR for B. subtilis and phusion PCR for dspB
Phusion PCR setup
35 microliters ddH2O    10 microliters 5X phusion buffer    1.5 microliters 10mM dNTP's    1 microliter each primer    1 microliter diluted template DNA    .5 microliter phusion polymerase
Taq PCR setup
40 microliters ddH2O    5 microliters 10X TAQ buffer    1.5 microliters 10mM dNTP's    1 microliter each primer    1 microliter diluted template DNA    .5 microliter phusion polymerase
9/5/2013
I ran the PCR products from 9/4 on gel. There were no viable products.
9/6/2013
We set up PCRs with Desi with a variety of conditions to test all of the primers that we have been using and the new primers for the iGem plasmid backbone for both DspB and Savinase. This will allow us to see if the DNA templates that we are using are viable or not. If some of the PCRs work, but not all of them, then we know that the problem is with our primers, not the DNA templates we are using.
9/9/2013
We ran our PCR products from 9/6 on gel and got no results on any of them. This clearly shows that our DNA templates themselves are not viable. We will need to get new DNA templates to use for our DspB and Savinase. We ordered genomic DNA from Arabidopsis thaliana, which contains the DspB gene. We will also try to get purified Bacillus subtilis DNA from Dr. Robison's lab.
We set up new cholera overnights, seeding them with 50 ul of our previous overnights in 4 mL of SLB, and placed them in the 30°C incubator.
We set up sequencing on several of the Savinase and AmyA plasmids to check them:
1.  Savinase - Colony B with Forward primer     2.  Savinase - Colony B with Reverse primer     3.  Savinase - Colony D with Forward primer     4.  Savinase - Colony D with Reverse primer     5.  AmyA - Clone A with Forward primer     6.  AmyA - Clone A with Forward primer     7.  AmyA - Clone B1 with Forward primer     8.  AmyA - Clone B1 with Reverse primer     9.  AmyA - Clone B2 with Forward primer     10. AmyA - Clone B2 with Reverse primer     11. AmyA - Clone C with Forward primer     12. AmyA - Clone C with Reverse primer
9/11/2013
We got the sequencing results back on our Savinase and AmyA plasmids from 9/9. None of the Savinase plasmids sequenced with our primers, indicating that none of the plasmids took up our target gene. AmyA Clone A had the best sequencing results of the AmyA plasmids, we will use this clone for our future work in cloning AmyA into the iGem backbone plasmid.
We worked today on getting our AmyA and DspB genes cloned into the iGem backbone plasmid. We set up PCRs for AmyA and Savinase using the following:
PCR Protocol
70 uL ddH20     20 uL 5X Phusion Buffer     3 uL 10 mM dNTPs     2 uL Forward primer     2 uL Reverse primer     1 uL Phusion polymerase     2 uL template DNA
Primers
DspB Forward - BI258     DspB Reverse - BI259     AmyA Forward - BI260     AmyA Reverse - BI268
9/12/13
We ran our PCR products from yesterday out on gel, the AmyA didn't work at all. We will have to check those primers and rerun the PCR for that.
We ran plasmid preps on Savinase Colony B and Colony D to see if we can get any kind of product out of them. See protocol page for protocol followed. We then set up PCRs using the purified plasmid prep as our template DNA, but when we ran it out on gel there wasn't any product. Our plasmids containing Savinase are completely unusable.
9/13/13
We ran plasmid preps of the pIG91 and pJG648 plasmids. We then ran the digest for them and ran plate transformations in LB + CAM for AmyA/pIG91 Upper, AmyA/pIG91 lower, and DspB/pIG91. We then transformed DspB into pJG648, plated it to LB + AMP, and put it in the 37°C incubator. All work was done following the protocols listed on the protocols page.
We also purified our DspB and AmyA proteins today following the procedure listed on the protocols page. The purified proteins were placed in the -80°C freezer until ready for use.
Retrieved from " http://2013.igem.org/Team:BYU_Provo/Notebook/Cholera_-_Enzyme/September/Period1/Dailylog "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
