<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_BYU_Provo_Notebook_CholeraDetection_Springexp_Period2_Dailylog"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog&amp;action=history">History            </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:BYU Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL id="css3menu1" class="topmenu"><LI class="topfirst"><A href="https://2013.igem.org/Team:BYU_Provo" style="height:25px;line-height:25px;">
					Home
				</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Team" style="height:25px;line-height:25px;"><SPAN>
						Team
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Team">
							Overview
						</A></LI><LI><A href="https://igem.org/Team.cgi?id=1195">
							Official Team Profile
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Team_Video">
							Team Video
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Team_Members">
							Undergraduates
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/TeamMember#Mentors">
							Mentors
						</A></LI></UL><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Project" style="height:25px;line-height:25px;"><SPAN>
						Project
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project/Background">
							Background
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Project/ExperimentalDesign">
							Experimental Design
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Notebook" style="height:25px;line-height:25px;"><SPAN>
						Notebook
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook#Phage_Team">
							Phage Team
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Notebook#Cholera_Team">
							Cholera Team
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Results" style="height:25px;line-height:25px;"><SPAN>
						Results
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Judge">
							Judging Criteria
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Experimental">
							Experimental
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Results/Modeling">
							Modeling
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Parts">
							Parts Submitted to the Registry
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Toronto">
							A Taste of Toronto
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Outreach" style="height:25px;line-height:25px;">
					Human Practices
				</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach">
							Overview
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach/OremLibrary">
							Orem Public Library Visit
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Outreach/Baxtor">
							The Adventure of Baxter Bacteria
						</A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Safety" style="height:25px;line-height:25px;"><SPAN>
						Safety
					</SPAN></A></LI><LI class="topmenu"><A href="https://2013.igem.org/Team:BYU_Provo/Attribution" style="height:25px;line-height:25px;"><SPAN>
						Attributions &amp; Collaborations
					</SPAN></A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Attribution">
							Attributions &amp; Acknowledgements
						</A></LI><LI><A href="https://2013.igem.org/Team:BYU_Provo/Collaboration">
							Collaborations
						</A></LI><LI class="topfirst"><A href="https://2013.igem.org/Main_Page" style="height:25px;line-height:25px;">
					iGEM 2013
				</A></LI></P><TABLE width="100%"><TBODY><TR><TD colspan="3"><FONT color="#333399" size="5" face="Calibri"><DL><DD><B>Cholera Detection May - June Notebook: May 15 - May 26 Daily Log</B></DD></DL></FONT></TD></TR><TR valign="top"><TD style="width: 22%; background-color: transparent;"><DL><DD><U><B>Cholera Detection</B></U></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Winterexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Winterexp">March-April</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Springexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Springexp">May-June</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Summerexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Summerexp">July-August</A></DD></DL><DL><DD><A href="/Team:BYU_Provo/Notebook/Cholera_-_Detection/Fallexp" title="Team:BYU Provo/Notebook/Cholera - Detection/Fallexp">September-October</A></DD></DL></TD><TD style="width: 82%; background-color: transparent;"><P><FONT size="4"><B>5/13/13</B></FONT></P><P>Today we ran our digested CRO insert on the low melt gel, and under the UV light we saw the band we were looking for. Clarisse excised the band and we performed a ligation reaction according to the protocol and using the vector that we had already run on a low-melt gel last Thursday. After incubating for 30 minutes at room temperature, we performed a transformation into DH5alpha. The selectable marker of pLAT is ampicillin.
</P><P>We ran our low melt gel with our Cro insert. After 45 minutes, we put the gel under UV light and cut out the insert. We then performed a ligation of our vector and cro insert. After 30 min, we did a transformation and inserted our newly ligated plasmid into DH5alpha.
</P><P>KK, KP
</P><P><FONT size="4"><B>5/14/13</B></FONT></P><P>(Thursday) When we dropped by today two of our plates (out of four) were contaminated. The two contaminated plates, we noticed, were the two plates that were old. The other two plates had a few colonies, but the was not a significantly greater number of colonies in our plasmid + insert + E.Coli plate than there were in our control plasmid + E.Coli plate. Kelton went ahead and streaked the few colonies we had to singles. We can PCR-verify if any of them have the plasmid. Meanwhile I am going to redo the transformation today. I will plate 4 plates: one 50 microL test, one 100 microL test, one 50 microL control, and one 100 microL control.
</P><P>KK, KP
</P><P><FONT size="4"><B>5/15/13</B></FONT></P><P>KK Yesterday's transformation of pIG12+CRO insert into E.Coli yielded several colonies on both the plate inoculated with 50 microL and the plate inoculated 100 microL. We plated 8 separate single colonies and will wait until Friday, when we will plan to PCR verify which of the colonies has our vector and insert. Afterward, we discussed what we will do on Friday. We'll spend the hour preparing samples for PCR and a few hours later Clarisse is going to drop by to run the samples on a gel.
</P><P>Also, I streaked lines and waves of V.cholerae today on a plate and for curiosity streaked E.Coli with pIG78 next to it in lines, and in waves intersecting the wavy cholera lines. We want to see if there are varying degrees of fluorescence.
</P><P>KP 5/15/13 Today I re-streaked our ligation products. I re-streaked the control (pLat vector only) and the test (pLat vector and cro insert.) The hope, is that we can get individual colonies from the plates, so that we can sequence our plasmid to see if the vector took up the insert and if we have what we think that we do. We we will need primers IG57 and IG58 for our sequencing of this plasmid on Friday.
</P><P><FONT size="4"><B>5/17/13</B></FONT></P><P>KK All our singles grew up well, so today we set up 10 PCR reactions + our control to test which, if any, had our vector + CRO insert construct. We also checked the plate that we streaked with Cholera and E.Coli with pIG78 next to one another. We wanted to see if the E.Coli would fluoresce at all in the presence of cholera. Under a UV light we didn't observe any noticeable difference between the streaks of cholera and the streaks of E.Coli
</P><P>Ran PCR with 10 singles we got from our ligations with pIG12/13 + Cro insert. See gel below. The bottom band in the ladder in the first lane runs at 500bps. Lanes 2-11 are the 10 strains we tested and the last lane is our control. What we get from this gel is a little unsure. 
</P><P><FONT size="4"><B>5/20/13</B></FONT></P><P>KK Clarice left for Russia on Sunday, so she'll be gone for perhaps a month working at her internship. At the beginning of class we talked about possible community projects. The first project we may want to do is publish a children's book. The other two projects that received votes were: creating a board game, and sponsoring a fun run to raise money for Haiti. I called my friend Redge Ballard, an animation major, to ask if he would be interested in designing the book.
We PCR verified our 10 colonies and only colony A had the CRO insert we were looking for. We set up overnights of colony A to do a plasmid prep and then sequence, and we also set up overnights of our three lambda strains to plate tomorrow as a lawn. Once we do that, we hope to be induce lambda to go lytic by plating our E.Coli with CRO.
</P><P><FONT size="4"><B>5/22/13</B></FONT></P><P>Yesterday I came in and did a plasmid prep of colonies A and W, which looked promising for having CRO+pIG12 on our PCR readout. Today we confirmed on the spectrophotometer the concentrations of the plasmids. The plasmid from A was present at a concentration of 158 ng/microL, and that of W had a concentration of 62 ng/microL. Yesterday we also plated our three strains of lambda-infected E.Coli, TT9901, TT9907, and TT23281, so today we were ready to trasform the plasmid into these three plasmids using electrophoration. Dr. Grose showed us how to use the electrophorator. We shocked each strain with plasmid from colony A and then a control plasmid separately, for a total of 6 electrophorations. After letting the cells recover for 30 minutes at 37 degrees celsius, we plated about 1 mL of each on LB/Amp plates. Tomorrow, hopefully, we will see a few colonies indicating that they took up our plasmid! Also, we submitted plasmids from A and W for sequencing.
</P><P>KK
</P><P><FONT size="4"><B>5/24/13</B></FONT></P><P>KK, KP Our electrophoration worked on 2 of the 3 strains - on TT9901, and TT9907, but not on TT25281. Yesterday we went ahead and set up overnights to plate the two successful transformed strains on Amp/Arabinose, but our Amp/Arabinose plates are no longer good, so we need to make more. We redid the electroporation today so that hopefully by next wednesday all three strains should be ready and we should have plates. We'll be testing to see if TT9901, TT9907, and TT25281 with Lambda's CRO gene cloned into them can induce the integrated prophage out of lysogeny and into the lytic cycle.
</P></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog">http://2013.igem.org/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period2/Dailylog&amp;oldid=226394" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>