"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/CholeraDetection/Springexp/Period1/Dailylog
From 2013.igem.org
Home
Team
Overview
Official Team Profile
Team Video
Undergraduates
Mentors
Project
Overview
Background
Experimental Design
Notebook
Overview
Phage Team
Cholera Team
Results
Overview
Judging Criteria
Experimental
Modeling
Parts Submitted to the Registry
A Taste of Toronto
Human Practices
Overview
Orem Public Library Visit
The Adventure of Baxter Bacteria
Safety
Attributions & Collaborations
Attributions & Acknowledgements
Collaborations
iGEM 2013
Cholera Detection May-June Notebook: May 1 - May 14 Daily Log
Cholera Detection
March-April
May-June
July-August
September-October
5/1/2013
KK Over the break we did little Lab work. Kelton was in Rexburg, and Clarice and I didn’t have the necessary primers to continue working. Today the primers came! We submitted primers for all the genes that have been cloned into pIG78 with pIG78 for sequencing. (I believe we included the primers for all the genes). We also have primers for working with CRO in the pBAD plasmid. Today our assignment was to make goals and set plans for what we hope to accomplish by the end of the term. Our plans are to be submitted by Friday. Our plans are spelled out in a table we’re printing, but they include understanding (via sequencing) what is ocurring in the plasmid by May 13th, correcting our system by May 29th, and demonstrating that we can induce Lambda into lysis through expression CRO. We will place CRO on the pBAD plasmid with the pBAD promoter (or on pLAT with a pBAD promoter), and the pBAD promoter is induced by arabinose.
KP Today we were trained in Cholera handling safety and we made our plans and goals for a summer full of success.
CHFirst day of new semester. Our First assignment is due Friday which is a detailed plan for the semester. Need to think about the web page and register for the team. Explained the whole process of plasmid modification to Kendal and Kelton. Figured out plan to modify pLat (pIG12 or pIG13) so that cro will be inserted right after pBAD (using restriction sites PstI and ecoRI). List of primers we are using for sequencing: IG11- pLat MCS forward BI219- hap rev. BI220- GFP for. BI222-GFP rev. BI225- RFP for. BI227- Qrr4 rev. BI228- Lux O for. BI229- Lux U rev. BI230- CqsS for. BI247- CqsS rev. middle BI248- LuxO for. middle
5/3/13
KP Today Kendall and I did PCR for the first time. Jordan and Clarice showed us how to do it. We did PCR on our E. Coli that has our lysogenic lambda encased within. We also froze down our lambda e. coli samples for future use.
KK Kelton and I worked with Jordan today to PCR amplify the CRO gene from π9907-infected E.Coli. We boiled the E.Coli to use as template and then followed the PCR protocol as outlined. Jordan actually was performed most of the protocol so that we could learn. Our control was E.Coli that had not been infected with lambda.
5/6/13
We ran a gel of our PCR product that we had created over the weekend. This was the first time I had ever done so, so it was fun and interesting! To make the gel, we mix 100 mL of TAE buffer with 1 gram of agar and heat in the microwave. The agar powder needs to completely dissolve. That mixture, with ethidium bromide, is added to the gel dock, and let set for about 20 minutes, or if you set the gel in the fridge it is a little less time. The CRO gene is about 300 base pairs. However, when we ran our PCR product against the ladder and against our negative control, our PCR product matched the control and did NOT match the length that indicates 300 base pairs. So, we know that our PCR reaction failed. It may be because we used a colony that did not include that lambda prophage.
Today we did the PCR reaction to amplify CRO again, but this time we amplified CRO from colonies that had been infected with our three distinct strains of Lambda. We will check our product tomorrow.
KK, KP
PCR amplified Cro from one of our lamda phage ecoli. It didn’t give us a product so we need to redo it. Froze down lamda phage Ecoli strains in -80. (220ul DMSO added to 1.3ml saturated overnight culture) IGM46 = 23281 IGM47 =9901 IGM48 =9907 Figured out that the vector we need for our Cro insert is pIG12 or pIG13. Obtained this strain from the -80. Did PCR clean up on the PCRs we set up last time turns out no good. Redid the PCR this time having a reaction for each of the three strains (IGM46, IGM47, and IGM48) and a control.
CH
5/8/13
We did a few different procedures today. We started out by doing a PCR purification for BI7701, BI7707, and BI23.... It was successful, but our concentration was very low (11.9). We also set up another PCR for BI7701, BI7707, and BI23... It will be done tomorrow. Today we also did a digest of our plasmid and our cro insert.
Having confirmed that our CRO insert was indeed PCR amplified (see picture above; very faint lines about 500 base pairs; wells 1-3 represent three strains of lambda, while 4 is a negative control), we performed a PCR cleanup on ALL 3 of our samples, in an attempt to isolate concentration of CRO insert possible. On the spectrophotometer our A260 reading gave us a concentration of 11.9 ng/microL.   Following that we ran a digest of our CRO insert and pLAT plasmid with pBAD promoter using restriction enzymes PST1-HF and ECOR1-HF. Our pLAT sample was a mixture of pIG12 and pIG13, which according to the parts database are the same plasmid, taken from two different colonies.  Having set our vector and inserts to digest, we started a low-melt gel with Jordan’s help. Low melt gels follow a slightly different protocol than normal gels, and use a smaller dock. Our dock was broken so our gel didn’t set very well. Because we don’t have time today, tomorrow we’ll run our vector and insert on the low-melt gel to see what happens. KK, KP
Our PCR from last time gave us very dim bands as products so we did PCR clean up and added all three of the samples together into one column. We also set up restriction digests for the cleaned PCR insert of Cro and for the plasmid pIG13 with pstI and ecoRI. This should place the Cro insert right after pBAD in pIG13. I also worked a bunch on the QS construct updating the primers document from last year and working on a document that has the sequence for the entire construct as it should appear in pJG78. Need to redo sequencing for the primers BI222, BI227, and BI247. CH
5/9/13
Our low melt gel had holes in the bottom of the first few wells, so the already low-concentration plasmid was diluted between 4 wells. Our vector’s well was viable, and the vector stayed in the well. KK, KP
5/10/13
We were able to cut out our vector from the low melt gel, but our insert was not visible. So, today, we made preparations to run our CRO insert on a low melt gel again. I made the low melt gel and set it to cool, and to remain in the fridge over the weekend. Kelton and Clarice performed the PCR cleanup of a second CRO insert PCR reaction that we ran. The PCR was more successful this time - the bands were much more clearly visible, in all three of our Lambda samples (see the photo that Kelton will upload).
Today we did a PCR cleanup on our Cro PCR products. We also set up a slow melt gel so we can try again on Monday to get a cutout of the insert in order to combine our plasmid and insert. We already have the plasmid, we just have to wait on the insert because it wasn’t visible in our first slow melt gel. Here is a picture of our second PCR product run on a 1 kb ladder gel: KK, KP,CH
CH 5/13/13 Set up ligations for pIG12/13 + Cro both treated with pstI and ecoRI. Figured out the primers we need for sequencing the region where cro will be inserted (IG57 and IG58).
Retrieved from " http://2013.igem.org/Team:BYU_Provo/Notebook/CholeraDetection/Springexp/Period1/Dailylog "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
