"
Page
Discussion
View source
History
teams
Log in
 
Team:BYU Provo/Notebook/CholeraDetection/Fallexp/Period3/Dailylog
From 2013.igem.org
Home
Team
Overview
Official Team Profile
Team Video
Undergraduates
Mentors
Project
Overview
Background
Experimental Design
Notebook
Overview
Phage Team
Cholera Team
Results
Overview
Judging Criteria
Experimental
Modeling
Parts Submitted to the Registry
A Taste of Toronto
Human Practices
Overview
Orem Public Library Visit
The Adventure of Baxter Bacteria
Safety
Attributions & Collaborations
Attributions & Acknowledgements
Collaborations
iGEM 2013
Cholera Detection September-October Notebook: October 1 - October 28 Daily Log
Cholera Detection
March-April
May-June
July-August
September-October
10/9/2013
Set up overnights of SdiA+/- strains for later infection with bacteriophage Lambda. Lysogens will plated next to Cholera to prove that SdiA is necessary for lytic induction of bacteriophage Lambda.
Begin growing a stockpile of Cholera patches (x10)
Begin looking for 3D modeling software.
Set up overnights of CRO + DH5alpha for later infection with bacteriophage Lambda.
10/10 Plate top agar assays of SdiA+/-, infect with bacteriophage Lambda, grow at 37 C.
10/11/13
Streak singles and start overnights with LOTS of SdiA+/- lysogens, PLATE NEXT TO CHOLERA. PCR SdiA+ from boiled K12 template and from not-boiled K12.
10/12 Plate SdiA+/- Top Agar Assays with Cholera in the middle (6 SdiA+ lysogens and 6 SdiA- lysogens)
10/14/13
Failed (contaminated) top agar tests!! Remake overnight cultures, and plate more lysogens (Amber and Arick); PCR clean-up of SdiA+, digest with PST1 and Xba1
10/15 Top agar SdiA+/- assays, run SdiA on low-melt gel and ligate
10/16/13
Re-plate top agar assays for SdiA+ and SdiA- lysogens, with cholera or with H202 as a positive control.
10/18/13
This time, our top agar lawns were not contaminated, however, we did not see the results we expected!  Not one plate had any plaques, not our lawns of SdiA+ lysogens, nor SdiA- lysogens, not by H202 induction nor induction by v. cholerae's AHLs.  So, either our lysogens aren't extremely lysogenic, or the do not have prophage Lambda and are not lysogens at all.  There is evidence that they are indeed lysogens, because Wednesday I streaked lines of each lysogen across a place and attempted to infect with a microliter of bacteriophage Lambda.  None of the lysogens showed any plaques, meaning that none could be superinfected.
We started a PCR reaction for each plate with primers targeting the CRO gene in bacteriophage Lambda.  If we see bands on the gel for CRO, then we know that they are lysogens.
10/21/13
We visualized the PCR reactions from Friday on a gel.  The only plate that appears to indeed have bacteriophage Lambda is one SdiA- lysogenic lawn (see gel).  We began an overnight culture of this lysogen, and plated it on LB/CAM just to ensure that it isn't contaminated, because our SdiA- knockout strain has kanemycin resistance integrated into the genome.
10/23/13
Today we set up top agar tests to check our SdiA- knockout strain. We are hoping to see plaques tomorrow. We plated on four different plates. We plated on a kanemycin plate with cholera added, a kanemycin plate with just the cells, a LB plate with cholera, and a LB plate with H202.
10/25/13
None of our plates showed any plaques.  We either have some sort of super-prophage in a super-lysogen, OR it is possible that the SdiA protein is necessary for both H202-induced AND V.cholerae-induced lysis by Lambda.  The literature we had read (Nina Molin Høyland-Kroghsbo, Rasmus Baadsgaard Mærkedahl and Sine Lo Svenningsen. A Quorum-Sensing-Induced Bacteriophage Defense Mechanism. mBio 4(1). 2013) indicated that E.Coli's SOS pathway, which is triggered by H202 and which excites the Lambda prophage to its lytic cycle, is NOT the same pathway by which AHL quorum-sensing lytic induction operates.  We need further testing to characterize what is going on in our SdiA- lysogen.
We have cloned SdiA into the iGEM backbone, performed multiple site-directed mutagenesis, and purified plasmids with (we hope) mutant SdiA from 20 colonies.  We are in the process of testing these mutants.
10/28/13
After many attempts to clone in our parts (YdiV-GFP and FtsQ-GFP) in the the iGEM backbone using the EcoR1 and Pst1 sites and seeing no success we decided to use the Spe1 and Xba1 sites which worked for us before. This proved to be successful. We got only white colonies from our transformation where as before we would have more than a 20:1 ratio of red colonies (incorrect colonies because they still have mCherry in the insertion site) to white colonies. We suggest that the iGEM plasmid needs to go through testing to see what is wrong with the either the EcoR1 or Pst1 site. We are in the process of sequencing these new constructs so that we can have the correct parts for BBa_K1195007 and BBa_K1195008.
Retrieved from " http://2013.igem.org/Team:BYU_Provo/Notebook/CholeraDetection/Fallexp/Period3/Dailylog "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
