<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="Menu"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="project"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">Human Practices</A></LI><LI id="design"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Product Design</A></LI><LI id="parts"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">labwork</A></LI><LI id="achievements"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About Us</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Description">Overview</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Growing_Back_Nerves">Reconnect Nerves</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Fighting">Fight Infections</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Model">Model</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Membrane">Physical Interface</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Kill">Kill-Switch</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Entrepreneurship">Entrepreneurship</A></LI></UL><UL id="hp-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Human_Practices">Integrated HP</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Public_Engagement">Education &amp; Engagement</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Safety">Safety &amp; Ethics</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Law">Law</A></LI></UL><UL id="design-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Technicals">Methodology</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Scenario">Scenario</A></LI></UL><UL id="parts-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Basic_Part">Basic Parts</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Improve">Improve</A></LI></UL><UL id="labwork-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Notebook">Notebook</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Experiments">Protocols</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Results">Results</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/InterLab">InterLab</A></LI></UL><UL id="achievements-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Medals">Medals</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Awards">Awards</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Demonstrate">Demonstrate</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/GiantJamboree">Giant Jamboree</A></LI></UL><UL id="aboutus-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Team">Team Members</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Acknowledgements"> Acknowledgements</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Attributions">Attributions</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2018.igem.org/Team:Pasteur_Paris">Home</A></LI><LI id="project_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">HP</A></LI><LI id="design_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Design</A></LI><LI id="parts_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">Labwork</A></LI><LI id="achievements_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About us</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV id="GeneralContent"><DIV id="MainContent"><DIV id="timeline"><DIV class="flex-parent"><DIV class="input-flex-container"><DIV class="dot-info" data-description="Week_28"><SPAN class="year">Week 28</SPAN><SPAN class="label">9 - 15 July</SPAN></DIV><DIV class="dot-info" data-description="Week_29"><SPAN class="year">Week 29</SPAN><SPAN class="label">16 - 22 July</SPAN></DIV><DIV class="dot-info" data-description="Week_30"><SPAN class="year">Week 30</SPAN><SPAN class="label">23 - 29 July</SPAN></DIV><DIV class="dot-info" data-description="Week_31"><SPAN class="year">Week 31</SPAN><SPAN class="label">30 July - 5 Aug</SPAN></DIV><DIV class="dot-info" data-description="Week_32"><SPAN class="year">Week 32</SPAN><SPAN class="label">6 - 12 Aug</SPAN></DIV><DIV class="dot-info" data-description="Week_33"><SPAN class="year">Week 33</SPAN><SPAN class="label">13 - 19 Aug</SPAN></DIV><DIV class="dot-info" data-description="Week_34"><SPAN class="year">Week 34</SPAN><SPAN class="label">20 - 26 Aug</SPAN></DIV><DIV class="dot-info" data-description="Week_35"><SPAN class="year">Week 35</SPAN><SPAN class="label">27 Aug - 2 Sept</SPAN></DIV><DIV class="dot-info" data-description="Week_36"><SPAN class="year">Week 36</SPAN><SPAN class="label">3 - 9 Sept</SPAN></DIV><DIV class="dot-info" data-description="Week_37"><SPAN class="year">Week 37</SPAN><SPAN class="label"> 10 - 16 Sept</SPAN></DIV><DIV class="dot-info" data-description="Week_38"><SPAN class="year">Week 38</SPAN><SPAN class="label">17 - 23 Sept</SPAN></DIV><DIV class="dot-info" data-description="Week_39"><SPAN class="year">Week 39</SPAN><SPAN class="label">24 - 30 Sept</SPAN></DIV><DIV class="dot-info" data-description="Week_40"><SPAN class="year">Week 40</SPAN><SPAN class="label">1 - 7 Oct</SPAN></DIV><DIV class="dot-info" data-description="Week_41"><SPAN class="year">Week 41</SPAN><SPAN class="label">8 - 14 Oct</SPAN></DIV><DIV class="dot-info" data-description="Week_42"><SPAN class="year">Week 42</SPAN><SPAN class="label">14 - 17 Oct</SPAN></DIV><DIV id="timeline-descriptions-wrapper"><DIV data-description="Week_28"><DIV class="block full" id="Week_28" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_28" class="vignette" id="vign_2800"><DIV data-description="Week_28" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_28" class="panel" id="pan_2800" style="text-align:left;"><H2> 07.10.2018 </H2><P> Institut Curie’s lab MMBM (Macromolécules et Microsystèmes en Biologie et en Médecine) at IPGG (Institut Pierre Gilles de Gennes) handed us the molds for the microchannel chip and granted us access to their lab.
We have been taught the basic steps for the fabrication of the microfluidic microchannel chip: PDMS preparation, molding, demolding, punching, bonding. </P><P>Basic steps protocols <A href="https://static.igem.org/mediawiki/2018/3/3d/T--Pasteur_Paris--Microfluidics-general-protocols.pdf" target="_blank"> here </A>. </P><H2> 07.11.2018 </H2><P>4 microchannel chips have been made.
Molds for the PDMS well chip have been received.
Glass microscopic slides coated with 2 nm chromium and 50 nm gold have been received for use in PDMS well chip.
Glass slides have been cut into small strips with a diamond tip. Processing damaged the gold coating. </P><P> Microchannel chip fabrication protocols <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2> 07.12.2018 </H2><P> First pieces of the well chip have been successfully molded. 
8 microchannel chips have been made.</P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><H2> 07.13.2018 </H2><P> Tested the conductivity of gold coated strips of glass. Current doesn't seem to pass through the strips. It is supposed that the numerous scratches caused by the cutting process are the cause for the non-conductivity of the strips. It was realized after the cutting process, that the conductivity should also have been measured before cutting. </P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: Strips before cutting</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION>Figure 2: Cut strip with scratches</FIGCAPTION></FIGURE><P> Second successful set of PDMS pieces of the well chip molded.
4 microchannel chips have been made.
1 microchannel successfully bonded to an imaging dish, and filled with fluorescein. Observation under a microscope revealed that it is fully functional. </P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 3: Microchannel chip bonded to imaging dish</FIGCAPTION></FIGURE><P> Bonding protocol <A href="https://static.igem.org/mediawiki/2018/3/3d/T--Pasteur_Paris--Microfluidics-general-protocols.pdf" target="_blank"> here </A>. </P><P> Microchannel chip protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_29"><DIV class="block full" id="Week_29" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_29" class="vignette" id="vign_2900"><DIV data-description="Week_29" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_29" class="vignette" id="vign_2901"><DIV data-description="Week_29" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_29" class="panel" id="pan_2900" style="text-align:left;"><H2>07.16.2018 </H2><P>We transformed pET 43.1a and pSB1C3 in DH5-α competent cells, in order to constitute a stock of empty vectors for our manipulation.</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>C (ng/<FONT face="Raleway">μ</FONT>l)</B></TD><TD><B>Volume (<FONT face="Raleway">μ</FONT>l)</B></TD><TD><B>Competent cell</B></TD><TD><B>Medium</B></TD></TR><TR><TD>pET 43.1.a</TD><TD>4.95</TD><TD>1</TD><TD>DH5-α</TD><TD>LB/carbenicilline</TD></TR><TR><TD>pSB1C3</TD><TD>40</TD><TD>1</TD><TD>DH5-α</TD><TD>LB/chloramphenicol</TD></TR></TBODY></TABLE><P>See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight:bold;text-decoration:none;color:black;" target="_blank">here</A> the transformation of <I>E. coli </I> DH5-alpha protocol
                                 We let the transformed bacteria grow overnight (16 hours).</P><H2>07.17.2018</H2><P><B>Results: </B>We went to see our bacterial culture: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Bacteria transformed with pET 43.1 had not grown.</LI><LI>Bacteria transformed with pSB1C3 had formed colonies.</LI></UL></P><P><B>Interpretations: </B>
Bacteria transformed with pET 43.1 did not grow. We found a non-commercial tube of pET 43.1  in the freezer from last year team, and we decided to try to amplify it because we did not have any commercial tubes of pET 43.1. Transformation did not work as expected, probably because:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>There was no DNA left in the tube.</LI><LI>The concentration of DNA was too low .</LI></UL></P><P>We cultivated the transformed pSB1C3 bacteria in liquid medium 2 x 25ml + Chloramphenicol (25µg/ml) overnight at 37°C, 180 rpm.</P><P><B>Results: </B>
Bacteria successfully transformed with pSB1C3.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)</P><H2>07.18.2018</H2><P><B>Extraction</B>:
We extracted the pSB1C3 plasmid from the bacterial culture.
The protocol used was the Qiagen Plasmid Purification Kit (See Midiprep for plasmid extraction protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here </A>)
Measure of the DNA concentration in each tube thanks to the NanoDrop (Blank used : TE.1)
(See the NanoDrop protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A>)</P><P><B>Results</B>:
We used the Nanodrop to quantify the purified DNA.</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>1</B></TD><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD><TD><B>9</B></TD><TD><B>10</B></TD></TR><TR><TD>Volume (µl)</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD><TD>50</TD></TR><TR><TD>C (ng/µl)</TD><TD>-7.8</TD><TD>-8.5</TD><TD>54.8</TD><TD>5.6</TD><TD>237.4</TD><TD>-7.9</TD><TD>-7.2</TD><TD>-7.5</TD><TD>-6.9</TD><TD>-6.5</TD></TR><TR><TD>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></TD><TD>1.43</TD><TD>1.42</TD><TD>1.97</TD><TD>3.04</TD><TD>1.85</TD><TD>1.32</TD><TD>1.47</TD><TD>1.56</TD><TD>1.35</TD><TD>1.44</TD></TR></TBODY></TABLE><P><B>Remarks:</B>
 [DNA] &lt; 0 = no DNA
[DNA] &gt; 200 good
        Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB> [1;2]</P><P><B>Interpretations</B>:
<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Tube 1, 2, 6, 7, 8, 9 and 10 did not contain DNA at all. This problem can be due to the fact that we failed the precipitation with ethanol, meaning that when we removed all the supernatant, we also removed the DNA, or that we removed by mistake the DNA pellet when removing all the supernatant after precipitation (the pellet was so small that we couldn’t see it with the naked eye). </LI><LI>Tube 3, 4 and 5 had DNA inside. The amount of DNA obtained is enough for us. These tubes have been placed in the freezer for later use. </LI></UL></P><P><B>Transformation</B>:
We received from Eurofins our first sequences:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>3a_NGF construction Part1 (Seq1)</LI><LI>3a_NGF construction Part2 (Seq2)</LI><LI>T7 RIP construction (Seq8)</LI></UL></P><P>In order to constitute a stock of this commercial vectors, we transformed DH5-α competent cells with the plasmid we received, and then we cultivated them on agar plates covered with LB medium and carbenicillin.
As the bacteria transformed with pET 43.1a did not grow, we tried again to transform the bacteria with pET 43.1a in a higher quantity than previously.</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Plasmid</B></TD><TD><B>C (ng/<FONT face="Raleway">μ</FONT>l)</B></TD><TD><B>Volume (<FONT face="Raleway">μ</FONT>l)</B></TD><TD><B>Competent cell</B></TD><TD><B>Medium</B></TD></TR><TR><TD>pEX-A258 Seq1</TD><TD>10</TD><TD>1</TD><TD>DH5-α</TD><TD>SOC/carbenicilline</TD></TR><TR><TD>pEX-A258 Seq2</TD><TD>10</TD><TD>1</TD><TD>DH5-α</TD><TD>SOC/carbenicilline</TD></TR><TR><TD>pEX-A258 Seq8</TD><TD>10</TD><TD>1</TD><TD>DH5-α</TD><TD>SOC/carbenicilline</TD></TR><TR><TD>pET 43.1a</TD><TD>4.95</TD><TD>5</TD><TD>DH5-α</TD><TD>SOC/carbenicilline</TD></TR></TBODY></TABLE><P>See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight:bold;text-decoration:none;color:black;" target="_blank">here</A> the transformation of <I>E. coli </I> DH5-alpha protocol</P><H2>07.19.2018</H2><P><B>Results</B>:
We went to see our overnight bacterial culture plates:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Seq1 3a_NGF_Part1</B></TD><TD><B>Seq2 3a_NGF_Part2</B></TD><TD><B>Seq8 T7_RIP</B></TD><TD><B>Pet43.1a(+)</B></TD></TR><TR><TD> Selection </TD><TD>Amp</TD><TD>Amp</TD><TD>Amp</TD><TD>Amp</TD></TR><TR><TD> Growth </TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD>Liquid Culture</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR></TBODY></TABLE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Seq8 – T7_RIP</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Seq1 – 3a_NGF_Part1</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Seq2 – 3a_NGF_Part2</FIGCAPTION></FIGURE><P><B>Interpretations</B>: 
Transformation of Seq1 – 3a_NGF_Part1, Seq2 – 3a_NGF_Part2 and Seq8 – T7_RIP did work as expected. 
Transformation of Pet43.1a (+) did not work again. As mentioned before, the tube of pET43.1a (+) was a non-commercial tube that we found in the freezer from last year’s team, and we decided to try to amplify it because we did not have any commercial tubes of pET 43.1. We tried to transform our bacteria twice, with no results, so the most probable hypothesis is that the tube did not contain DNA anymore. After this experiment, we decided to buy a tube pET43.1a(+). 
We cultivated two different colonies from each plate (Seq1, Seq2, Seq8):<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>One in an Erlenmeyer in 25 ml LB medium + Carbenicillin (100µg/ml) overnight at 37°C, 180 rpm.</LI><LI>One in two Falcon tube in 15 ml LB medium + Carbenicillin (100µg/ml) overnight at 37°C, 180 rpm.</LI></UL></P><P>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)
</P><H2>07.20.2018</H2><P>Results of the liquid culture are good.</P><P><B>Extraction - MidiPrep</B>:
We extracted the 3 plasmids Seq1 – 3a_NGF_Part1, Seq2 – 3a_NGF_Part2 and Seq8 – T7_RIP from the bacterial cultures.
The protocol used was the Qiagen Plasmid Purification Kit (See Midiprep for plasmid extraction protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here </A>)
Measure of the DNA concentration in each tube thanks to the NanoDrop (Blank used : TE.1)
(See the NanoDrop protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A>)</P><P><B>Results</B>:
We used the Nanodrop to quantify the purified DNA.</P><P><B> For NGF part1, Colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>[DNA] (ng/µL) </B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 3569.2 </TD><TD>1.9</TD></TR><TR><TD> 1578.7 </TD><TD>1.89</TD></TR><TR><TD> 848.1 </TD><TD>1.90</TD></TR><TR><TD> 2322.1 </TD><TD>1.89</TD></TR></TBODY></TABLE><P><B> For NGF part1, Colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [DNA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 488.1</TD><TD>1.89</TD></TR><TR><TD> 493.7 </TD><TD>1.90</TD></TR><TR><TD> 445.1 </TD><TD>1.89</TD></TR><TR><TD> 644.6 </TD><TD>1.86</TD></TR></TBODY></TABLE><P><B> For NGF part2, Colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [DNA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1555.0</TD><TD>1.88</TD></TR><TR><TD> 493.7 </TD><TD>1.88</TD></TR><TR><TD> 666.2 </TD><TD>1.87</TD></TR><TR><TD> 367     </TD><TD>1.90</TD></TR></TBODY></TABLE><P><B> For NGF part2, Colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [DNA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 463.7</TD><TD>1.89</TD></TR><TR><TD> 420.8 </TD><TD>1.88</TD></TR><TR><TD> 461.4 </TD><TD>1.89</TD></TR><TR><TD> 420.6   </TD><TD>1.89</TD></TR></TBODY></TABLE><P><B> For RIP, Colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [DNA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 694.4</TD><TD>1.84</TD></TR><TR><TD> 1028.9 </TD><TD>1.87</TD></TR><TR><TD> 1408.7 </TD><TD>1.88</TD></TR><TR><TD> 466.8   </TD><TD>1.88</TD></TR></TBODY></TABLE><P><B> For RIP, Colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 228.4</TD><TD>1.89</TD></TR><TR><TD> 325.2 </TD><TD>1.89</TD></TR><TR><TD> 379.4 </TD><TD>1.89</TD></TR><TR><TD> 427.2   </TD><TD>1.89</TD></TR></TBODY></TABLE><P><B>Remarks</B>:
[NA] &lt; 0 = no DNA
[NA] &gt; 200 good
Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB> [1;2]</P><P><B>Interpretations</B>: 
We have DNA in our tubes. The DNA is too concentrated so we will have to dilute them before next use. We froze the tubes at -20°C until next use.</P></DIV><DIV data-description="Week_29" class="panel" id="pan_2901" style="text-align:left;"><H2> 18.07.2018</H2><P> 1 set of PDMS pieces for the well chip has been made. One of the pieces broke while demolding. It was discarded. 4 microchannel chips made. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2> 19.07.2018 </H2><P> 1 set of PDMS pieces for the well chip have been made. 4 microchannel chips made. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_30"><DIV class="block full" id="Week_30" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_30" class="vignette" id="vign_3000"><DIV data-description="Week_30" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_30" class="vignette" id="vign_3001" style="display:none"><DIV data-description="Week_30" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_30" class="vignette" id="vign_3002"><DIV data-description="Week_30" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_30" class="panel" id="pan_3000" style="text-align:left;"><H2>07.24.2018</H2><P><B>Culture of strain HB2151 transformed with pVDL 9.3 </B>(received on 23.07.2018 from Dr. Victor de Lorenzo)
Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>LB agar plate supplemented with chloramphenicol (25 mg/mL)</LI><LI>Bacteria HB 2151 transformed with pVDL9.3</LI></UL></P><P>The bacteria we received was in the form of a stock: LB + agar media containing the bacteria and the plasmid in an Eppendorf tube. Sample from the solid stock was streaked on LB agar plate complemented with chloramphenicol, and secondly grown in an Erlenmeyer with 25 mL LB medium and 25 <FONT face="Raleway">μ</FONT>L chloramphenicol at 25 mg/mL.</P><P><B>Biofilm culture on filter (0.2 <FONT face="Raleway">μ</FONT>m)</B>

Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Bacteria DH5α transformed with pSB1C3</LI><LI>Filters - 0.2 <FONT face="Raleway">μ</FONT>m Milipore (5 cm diameter) </LI><LI>Petri-dishes </LI><LI>Sterile Clamps</LI><LI>Sterile Rakes</LI><LI>Pipets</LI><LI>LB media </LI><LI>Falcon tubes </LI><LI>Chloramphenicol (25 mg/mL)</LI></UL><OL style="text-align: left;font-size: 16px;"><LI> Make a liquid culture of DH5α transformed with pSB1C3 from colonies we had on a petri dish made on the 07.16.2017 and stored at 4°C. Let grow over night at 37°C under agitation. 
Made the 07.23.2017 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)</LI><LI>Auto-cleavage of the 0.2 <FONT face="Raleway">μ</FONT>m filters</LI><LI>Deposit the filters on a petri dish with a clamp. Wait until the filter is totally “wet” with LB media. </LI><LI>Deposit the bacteria culture in the center of the filter with a micropipette and spread with a rake. We made 3 different petri dishes with 3 different volumes. </LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>DH5α-pSB1C3- biofilm culture – 1</B></TD><TD><B>DH5α-pSB1C3- biofilm culture – 2</B></TD><TD><B>DH5α-pSB1C3- biofilm culture – 3</B></TD></TR><TR><TD>Volume of culture deposit (<FONT face="Raleway">μ</FONT>L)</TD><TD>50</TD><TD>100</TD><TD>150</TD></TR></TBODY></TABLE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Picture of the 3 petri dishes at t = 0. </FIGCAPTION></FIGURE><P><OL start="5" style="text-align: left; font-size: 16px;"><LI>Incubate at 37°C overnight. </LI></OL></P><P><B>Transformation of Seq5, Seq7</B> from Eurofins Genomics, pET 43.1 and pBR322

Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>LB agar plate supplemented with carbenicillin</LI><LI>Sterile LB medium</LI><LI>pET 43.1a: 42.36 pg/<FONT face="Raleway">μ</FONT>L</LI><LI>pBR322: 65.00 ng/<FONT face="Raleway">μ</FONT>L</LI><LI>Seq5 &gt; AGR-RIP Part1: 1.1 <FONT face="Raleway">μ</FONT>g</LI><LI>Seq7 &gt; AGR-RIP Part3: 1.9 <FONT face="Raleway">μ</FONT>g</LI></UL></P><P>Sequences received on the 07.24.2018. 
Gene synthesis products are in the form of lyophilized powder, so we need to re-suspend them. 
Seq5 &gt; AGR-RIP Part1: 1.1 <FONT face="Raleway">μ</FONT>g
Seq7 &gt; AGR-RIP Part3: 1.9 <FONT face="Raleway">μ</FONT>g<OL style="text-align: left; font-size: 16px;"><LI>Centrifuge in order to make sure that all the plasmids are at the bottom of the tube and not in the lid. </LI><LI>Re-suspend the powder in TE 1X. 
Seq5: add 110 <FONT face="Raleway">μ</FONT>L to have a final concentration of 10 ng/<FONT face="Raleway">μ</FONT>L
Seq7: add 190 <FONT face="Raleway">μ</FONT>L to have a final concentration of 10 ng/<FONT face="Raleway">μ</FONT>L</LI><LI>Transformation of the 4 plasmids in DH5α competent cells:</LI><LI> Seq5, Seq7 and pBR322: 1 <FONT face="Raleway">μ</FONT>L</LI><LI>pET 43.1a: 10 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli</I> DH5-alpha protocol)<LI>Culture on petri dishes overnight at 37°C</LI></OL></P><H2>07.25.2018</H2><P><B>Transformed bacteria cultures</B>:
We went to see our overnight bacterial culture plates:
<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Seq5 RIP_Part1</B></TD><TD><B>Seq7 RIP_Part3</B></TD><TD><B>pBR322</B></TD><TD><B>Pet43.1a(+)</B></TD></TR><TR><TD> Selection </TD><TD>Amp</TD><TD>Amp</TD><TD>Amp</TD><TD>Amp</TD></TR><TR><TD> Growth </TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR><TR><TD> CFU </TD><TD>2 816</TD><TD>2 492</TD><TD>2 628</TD><TD>-</TD></TR><TR><TD>Liquid Culture</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD><TD>No</TD></TR></TBODY></TABLE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Seq8 – T7_RIP</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Seq7 – RIP_Part3</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 4: </B>pBR322</FIGCAPTION></FIGURE></P><P><B>Interpretations</B>: 
Transformation of Seq5 – RIP_Part1, Seq7 – RIP_Part3 and pBR322 did work as expected. 
Transformation of Pet43.1a (+) did not work again. The tube of pET43.1a (+) was another non-commercial tube that we found in the freezer from last year’s team, and we decided to try to amplify it because we did not have any commercial tubes of pET 43.1. We tried to transform our bacteria twice, with no results, so the most probable hypothesis is that the tube did not contain DNA anymore. 
We are still waiting for the commercial pET43.1a(+) (we ordered it last week).</P><P><B>Bacteria transformed pVDL 9.3</B>:
The culture on agar plate did work, but we were not able to isolate one colony.
We stopped the liquid culture and measured the OD600 of the culture medium.</P><P><B>Biofilm culture</B>:<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>pSB1C3-DH5α-biofilm culture – 1</B></TD><TD><B>pSB1C3-DH5α-biofilm culture – 2</B></TD><TD><B>pSB1C3-DH5α-biofilm culture – 3</B></TD></TR><TR><TD>Volume of culture deposit (<FONT face="Raleway">μ</FONT>L)</TD><TD>50</TD><TD>100</TD><TD>150</TD></TR><TR><TD>Biofilm formation</TD><TD>Yes</TD><TD>Yes</TD><TD>Yes</TD></TR></TBODY></TABLE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 5: </B>Picture of the 3 petri dishes at t = 24. </FIGCAPTION></FIGURE></P><P>We did a biofilm transfer. We took our filter with the biofilm and transferred it on another porous filter placed on a new petri dish LB + Cm. Then we removed the filter containing the biofilm and incubated the plate overnight at 37°C.</P><P><B>Liquid culture</B>:
We cultivated two different colonies from each plate (Seq5, Seq7 and pBR322):<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>One in an Erlenmeyer in 25 mL LB medium + Carbenicillin (100 µg/ml) overnight at 37°C, 180 rpm.</LI><LI>One in two Falcon tube in 15 mL LB medium + Carbenicillin (100 µg/ml) overnight at 37°C, 180 rpm.</LI></UL></P><P>Moreover, for each colony of each DNA part we did some streaks on a plate and incubated them at 37°C.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)</P><P><B>Midiprep pVDL 9.3</B>:
We extracted the pVDL 9.3 plasmid from the HB2151 strain.
The protocol use was the Qiagen Plasmid Purification Kit (See Midiprep for plasmid extraction protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here </A>)
We divided the eluted DNA into 8 tubes before centrifugation.

Measure of the DNA concentration in each tube by the NanoDrop. 
Blank used : TE.1 (See the NanoDrop protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A>)</P><P><B>Results</B>:
We used the Nanodrop to quantify the purified DNA for Seq: pVDL 9.3 1.</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>1</B></TD><TD><B>2</B></TD><TD><B>3</B></TD><TD><B>4</B></TD><TD><B>5</B></TD><TD><B>6</B></TD><TD><B>7</B></TD><TD><B>8</B></TD></TR><TR><TD>ng/µL [NA]</TD><TD>3.7</TD><TD>111.2</TD><TD>140</TD><TD>0</TD><TD>1.9</TD><TD>47</TD><TD>108</TD><TD>8.3</TD></TR><TR><TD>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></TD><TD>1.93</TD><TD>1.87</TD><TD>1.88</TD><TD>-</TD><TD>1.6</TD><TD>1.9</TD><TD>1.9</TD><TD>1.91</TD></TR></TBODY></TABLE><H2>07.26.2018</H2><P><B>Results</B>: Transfer of our biofilm growth on a filter (07.24.2018 loading of 50 µl of pSB1C3 liquid culture on a filter) on another filter on a plate and incubation overnight at 37°C:</P><FIGURE style="display:inline-block">μL of culture) 24 hours after transfer on another filter. &quot;&gt;<FIGCAPTION><B>Figure 6: </B>Picture of the petri dish (with 50 <FONT face="Raleway">μ</FONT>L of culture) 24 hours after transfer on another filter. . </FIGCAPTION></FIGURE><P><B>Observations</B>: 
The biofilm is able to grow again after a transfer from one filter to another as seen by the red color. </P><P><B>Midiprep Seq5, Seq7, pBR322:</B>
We extracted Seq5, Seq7 and pBR322 plasmid from transformed DH5α strain.
The protocol used was the Qiagen Plasmid Purification Kit (See Midiprep for plasmid extraction protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here </A>)
We divided the eluted DNA into 4 tubes before centrifugation.

Measure of the DNA concentration in each tube with NanoDrop. 
Blank used: TE.1 (See the NanoDrop protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A>)</P><P><B>Results</B>:
We used the Nanodrop to quantify the purified DNA.</P><P><B> For Seq5, Colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 245.7 </TD><TD>1.87</TD></TR><TR><TD> 349</TD><TD>1.87</TD></TR><TR><TD> 353.2 </TD><TD>1.87</TD></TR><TR><TD> 306.7 </TD><TD>1.87</TD></TR></TBODY></TABLE><P><B> For Seq5, Colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 258</TD><TD>1.88</TD></TR><TR><TD> 281 </TD><TD>1.86</TD></TR><TR><TD> 260 </TD><TD>1.86</TD></TR><TR><TD> 295 </TD><TD>1.86</TD></TR></TBODY></TABLE><P><B> For Seq7, Colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 293</TD><TD>1.88</TD></TR><TR><TD>299 </TD><TD>1.88</TD></TR><TR><TD> 305 </TD><TD>1.87</TD></TR><TR><TD>288  </TD><TD>1.87</TD></TR></TBODY></TABLE><P><B> For Seq7, Colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 217</TD><TD>1.87</TD></TR><TR><TD> 212 </TD><TD>1.87</TD></TR><TR><TD> 175</TD><TD>1.88</TD></TR><TR><TD> 244     </TD><TD>1.83</TD></TR></TBODY></TABLE><P><B> For pBR322, Colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 3.8</TD><TD>1.68</TD></TR><TR><TD> 55.8 </TD><TD>1.86</TD></TR><TR><TD> 54</TD><TD>1.9</TD></TR><TR><TD> 68  </TD><TD>1.84</TD></TR></TBODY></TABLE><P><B> For RIP, Colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 32</TD><TD>1.82</TD></TR><TR><TD> 28 </TD><TD>1.75</TD></TR><TR><TD> 41</TD><TD>1.88</TD></TR><TR><TD>30   </TD><TD>1.88</TD></TR></TBODY></TABLE><P><B>Interpretations</B>: 
There is DNA in all the tubes. For Seq5 and Seq7, we obtain high DNA concentrations, which is good. For pBR322, the DNA concentration is lower. This is probably due to the fact that pB322 is a low copy plasmid. However, the quantity of DNA is enough.</P><P><B>Preparing cultures for stocks</B>: 
We set some pre-cultures of Seq5, Seq7 and pBR322, in order to make some stocks to store at -80°C. 
In a 15 mL Falcon tube, we placed 5 mL of LB media, and 10 <FONT face="Raleway">μ</FONT>L of antibiotic (Carbenicillin). We added one of each colony inside each Falcon and placed the Falcon tubes inside the incubator (37°C – 180 rpm) and let grow overnight. </P></DIV><DIV data-description="Week_30" class="panel" id="pan_3002" style="text-align:left;"><H2> 07.24.2018</H2><P> Membrane filters from <A href="https://www.sterlitech.com/" target="_blank">Sterlitech</A>received. 
                                        8 microchannel chips and 1 set of PDMS pieces for the well chip have been made. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2> 07.25.2018 </H2><P> Platinum wires received and conductivity tested. Function generator was set on sine. The physical quantities measured here are Eg, the generator's tension amplitude and Ep, the voltage difference between the two extremities of a platinum wire. the quantity calculated here is 20*log(Ep/Eg) for different frequencies.</P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: Conductivity of a platinum wire for different frequencies</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Voltage difference calculated is extremely low, indicating a very good conductivity for the platinum wires, so its resistance (in low frequencies) can be neglected at first glance when it will be used in PDMS well chips. Resistance increases in higher frequencies, because of the skin-effect in metals: the strip transforms into an antenna. But as we are going to use only low frequencies, this doesn't affect us.</P><P> Conductivity measurement protocol (section 2, replace chip by platinum wire, no resistor) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><H2> 07.27.2018 </H2><P> 4 microchannel chips have been made. 1 microchannel chip bonded for imaging test purposes. </P><P> Microchannel chip fabrication and bonding protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_31"><DIV class="block full" id="Week_31" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_31" class="vignette" id="vign_3100"><DIV data-description="Week_31" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_31" class="vignette" id="vign_3101" style="display:none"><DIV data-description="Week_31" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_31" class="vignette" id="vign_3102"><DIV data-description="Week_31" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_31" class="panel" id="pan_3100" style="text-align:left;"><H2>07.30.2018</H2><P><B>Digestion of Seq8 and pBR322</B>:
We cut the commercial plasmid containing our Seq8 (T7-RIP) with restriction enzymes XbaI and SpeI, and pBR322 with AvaI and BsmI, in order then to insert our sequence into the plasmid pBR322. 
Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Water baths at 37°C and 80°C </LI><LI>Eppendorf tubes</LI><LI>Pipets and cones</LI></UL></P><P>
Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pBR322 [68 ng/<FONT face="Raleway">μ</FONT>L]</LI><LI>commercial plasmid containing Seq8 [466,8 ng/<FONT face="Raleway">μ</FONT>L]</LI><LI>Restriction enzymes: AvaI, BsmI, XbaI, SpeI</LI><LI>10X Cut Smart</LI><LI>Distilled water</LI></UL></P><P><OL style="text-align: left;font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 15 minutes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pBR322 </TD><TD>22</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>20</TD></TR><TR><TD>Bsml</TD><TD>1.5</TD></TR><TR><TD>Aval</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Eurofins plasmid with Seq8 </TD><TD>20</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>22</TD></TR><TR><TD>Xbal</TD><TD>1.5</TD></TR><TR><TD>Spel</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left;font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a heat block at 80 °C for 20 minutes.</LI></OL></P><P><B>Electrophoresis on agarose gel</B>
We prepared an Agarose gel. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>1 well with the digested pBR322 (n°1)</LI><LI>2 wells with the digested plasmids containing Seq8 (n°2 and 3)</LI><LI>1 well with a DNA ladder (n°4)</LI></UL></P><P>
We let the DNA migrate for 10 minutes at 85 V and then 40 minutes at 125 V.
We then revealed the gel under UV lamp.<B>Results</B>: </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B></FIGCAPTION></FIGURE><P><B>Interpretations</B>: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pBR322 (4361 bp) has been successfully linearized</LI><LI>pEXA258 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (459 bp) and the linearized plasmid (2909 bp)</LI></UL></P><P><B>DNA Gel extraction</B>:
Here is the same gel after DNA extraction of pBR322 and of Seq8.</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1-2:gel after DNA extraction of pBR322 and of Seq8 </B></FIGCAPTION></FIGURE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sequence name</B></TD><TD><B>Eppendorf weight (g)</B></TD><TD><B>Eppendorf + gel weights (g)</B></TD><TD><B>Gel weight (g)</B></TD></TR><TR><TD>Seq8 (1)</TD><TD> 1.08 </TD><TD> 1.29 </TD><TD>0.21</TD></TR><TR><TD>Seq8 (2)</TD><TD> 1.08 </TD><TD> 1.37 </TD><TD> 0.29 </TD></TR><TR><TD> pBR322 </TD><TD> 1.08 </TD><TD> 1.27 </TD><TD>0.19</TD></TR></TBODY></TABLE><P>It is useful to know how the weight of the gel to adapt de volumes of buffer to the amount of gel we have. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)<B>Results</B>: after extracting the DNA from the gel, we measured the DNA concentration using the Nanodrop.</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sequence name</B></TD><TD><B>DNA concentration (ng/<FONT face="Raleway">μ</FONT>L))</B></TD><TD><B>Abs<SUB>(260nm)</SUB>/Abs<SUB>(280nm)</SUB> (g)</B></TD><TD><B>Total extracted DNA (ng)</B></TD></TR><TR><TD>Seq8 (1)</TD><TD> 6.4 </TD><TD> 1.87 </TD><TD> 294.4 (620.8)</TD></TR><TR><TD>Seq8 (2)</TD><TD> 6.8 </TD><TD> 2.07 </TD><TD> 326.4 (620.8)</TD></TR><TR><TD> Linearized pBR322   </TD><TD> 9.9 </TD><TD> 1.97 </TD><TD> 475.2 </TD></TR></TBODY></TABLE><P><B>Interpretations</B>:
The initial amount of each DNA fragment before digestion was 1500 ng.
We can therefore evaluate the efficiency of our digestion and extraction:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>for Seq8 insert: r = 0.414 </LI><LI>for pBR322 plasmid: r = 0.317</LI></UL></P><P><B>Ligation of linearized pBR322 and our insert Seq8</B>
We used the In-Fusion cloning kit provided by Ozyme in order to clone our insert into the linearized pBR322.<OL style="text-align: left; font-size: 16px;"><LI>We mixed the following volumes into 1.5 mL tubes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Purified fragment</TD><TD>4</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>Linearized vector</TD><TD> 4 </TD><TD> 1 </TD><TD>1 (control)</TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 2 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 7</TD><TD> 5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 min in a water bath at 50°C</LI><LI>Ligation See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Ligation protocol)</LI><LI>Set in ice</LI><LI>Transformation of the 3 plasmids into Stellar competent cells (provided with the cloning kit)</LI><LI>Competent cell volume : 33 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli </I>DH5-alpha protocol)<LI>Culture on petri dishes LB agar gel supplemented with carbenicillin overnight at 37°C</LI></OL></P><P>Transformation of Seq6 from Eurofins Genomics
Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>LB agar plates supplemented with carbenicillin (25 <FONT face="Raleway">μ</FONT>g/mL)</LI><LI>LB medium</LI><LI>Seq6 &gt; AGR-RIP Part1: 4.6 <FONT face="Raleway">μ</FONT>g</LI></UL></P><P>Sequences are under the form of lyophilized powder, so we need to re-suspend it. </P><OL style="text-align: left; font-size: 16px;"><LI>Centrifuge in order to make sure that all the plasmids are at the bottom of the tube and not in the lid. </LI><LI>Re-suspend the powder in TE 1X. 
We resuspended it into 460 <FONT face="Raleway">μ</FONT>L TE, to obtain a concentration of 10 ng/<FONT face="Raleway">μ</FONT>L. </LI><LI>Transformation of the plasmid in DH5α competent cells
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli</I> DH5-alpha protocol)</LI><LI> Culture on petri dishes LB agar gel supplemented with carbenicillin overnight at 37°C</LI></OL><H2>07.31.2018</H2><P><B>Results</B>: ligation cultures<FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Negative Control</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Positive Control</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 4: </B>Ligation of pBR322 and Seq8 (T7-RIP)</FIGCAPTION></FIGURE>
As expected, we could observe a lot of colonies for the positive control. Some colonies had grown for the negative control (due to the linearized plasmid which circularized itself). For our ligated plasmid, we only observed a few colony (as much as in the negative control) so we didn’t know if the ligation succeeded. This is why we picked up 20 colonies and cultivated them overnight, hoping that one of these had the correct plasmid.

The plate containing DH5-α transformed with Seq6 also had a lot of colonies, as expected.  </P><P><B>Liquid culture of DH5-α with Seq6</B>
We cultivated 2 different colonies in 250 mL Erlenmeyer containing:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>25 mL LB medium</LI><LI>25 <FONT face="Raleway">μ</FONT>L carbenicillin (25 mg/mL)</LI></UL></P><P>
We cultivated it overnight in a shaking incubator at 37°C, 180 rpm.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)</P><P><B>Liquid culture of Stellar transformed with pBR322-Seq8</B>
We cultivated 20 different colonies in order to maximize our chance to get one successful ligated plasmid. Each colony was cultivated in a 15 mL Falcon tube containing:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>5 mL LB medium</LI><LI>5 <FONT face="Raleway">μ</FONT>L carbenicillin (25 mg/mL)</LI></UL></P><P>
We cultivated it overnight in a shaking incubator at 37°C, 180 rpm.
We also cultivated each colony on petri dishes LB agar plate supplemented with carbenicillin overnight at 37°C.</P><P><B>Transformation of Seq3</B> from Eurofins Genomics and commercial pET43.1a
Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>LB agar plate supplemented with carbenicillin </LI><LI>Seq3 : NGF-GFP Part1 : 4.0 <FONT face="Raleway">μ</FONT>g</LI><LI>pET 43.1a : 0.5 <FONT face="Raleway">μ</FONT>g/<FONT face="Raleway">μ</FONT>L</LI></UL></P><P>
Sequence from Eurofins Genomics under the form of lyophilized powder so we need to re-suspend it. <OL style="text-align: left; font-size: 16px;"><LI>Centrifuge in order to make sure that all the plasmids are at the bottom of the tube and not in the lid. 

</LI><LI>Re-suspend the powder in TE 1X. 
We resuspended it into 400 <FONT face="Raleway">μ</FONT>L TE, in order to obtain a concentration of 10 ng/<FONT face="Raleway">μ</FONT>L. 

</LI><LI>Transformation of the plasmids in DH5α competent cells:</LI><LI>volume of competent cells: 50 <FONT face="Raleway">μ</FONT>L</LI><LI>pET 43.1a: 50 pg (1 <FONT face="Raleway">μ</FONT>L of commercial plasmid diluted 1/10000)</LI><LI>Seq3 : 10 ng (1 <FONT face="Raleway">μ</FONT>L)</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli</I> DH5-alpha protocol)<LI>Culture on petri dishes LB agar plate supplemented with carbenicillin overnight at 37°C.</LI></OL></P><H2>08.01.2018</H2><P>We went to see our cultures:<UL style="text-align: left; list-style: disc;font-size: 16px;"><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 5: </B></FIGCAPTION></FIGURE><LI>All the colonies of our ligation (pBR322 + Seq8) grew except one.</LI><LI>DH5α transformed with Seq3 grew.</LI><LI>DH5α transformed with pET 43.1a did not grow. </LI></UL></P><P>Hypothesis: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>We might have diluted the plasmid or the bacteria too much.</LI><LI>We did not homogenize the solution enough during the dilution of the commercial solution.</LI><LI>We did not pipet correctly the 1 <FONT face="Raleway">μ</FONT>L of plasmid from the commercial tube. </LI></UL></P><P><B>Extraction of the ligated plasmid (pBR322 + Seq8)</B> from Stellar (miniprep)
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)<B>Results</B>: (tube 8 did not grow)
From NanoDrop<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA CONCENTRATION (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD>94.8</TD><TD>2.05</TD></TR><TR><TD> 2 </TD><TD>105.8</TD><TD>2.06</TD></TR><TR><TD> 3 </TD><TD>106.3</TD><TD>2.04</TD></TR><TR><TD> 4   </TD><TD>84.8</TD><TD>2.02</TD></TR><TR><TD> 5   </TD><TD>100.1</TD><TD>2.02</TD></TR><TR><TD> 6   </TD><TD>79.9</TD><TD>2.00</TD></TR><TR><TD> 7 </TD><TD>08.9</TD><TD>2.04</TD></TR><TR><TD> 8 </TD><TD>-</TD><TD>-</TD></TR><TR><TD> 9   </TD><TD>115.1</TD><TD>2.02</TD></TR><TR><TD> 10 </TD><TD>116.7</TD><TD>1.96</TD></TR><TR><TD> 11  </TD><TD>76.6</TD><TD>2.00</TD></TR><TR><TD> 12  </TD><TD>69.5</TD><TD>1.99</TD></TR><TR><TD> 13  </TD><TD>80.7</TD><TD>1.94</TD></TR><TR><TD> 14  </TD><TD>63.0</TD><TD>1.98</TD></TR><TR><TD> 15  </TD><TD>78.4</TD><TD>2.03</TD></TR><TR><TD> 16  </TD><TD>153.1</TD><TD>1.94</TD></TR><TR><TD> 17  </TD><TD>90.7</TD><TD>2.00</TD></TR><TR><TD> 18  </TD><TD>119.3</TD><TD>2.02</TD></TR><TR><TD> 19 </TD><TD>100.8</TD><TD>1.97</TD></TR><TR><TD> 20 </TD><TD>85.3</TD><TD>2.03</TD></TR></TBODY></TABLE></P><P>We chose to verify which colony had acquired the plasmid containing the Seq8 insert. To do so, we digested the plasmid with the restriction enzymes EcoRI and PstI and ran a 1% agarose gel. 
The size of the plasmid pBR322 is 4361 bp. Once we cut the sequence between the sites EcoRI (bp position 4359) and PstI (bp position 3611), this leaves us with an empty backbone of 3613 bp. The size of the insert Seq8 is 461 bp. 
Here we made a mistake, by confusing the right restriction enzymes to use. Indeed, we should have used XbaI and SpeI instead of EcoRI and PstI. Moreover, as these two restriction sites are present in the pBR322 backbone (748 bp away), this digestion should show two DNA fragments.</P><P><B>1% Agarose gel of the miniprep of the ligated plasmid (pBR322 + Seq8)</B> from Stellar
Only the first half of the samples (1 to 11) were deposed today. 
The preparation of the gel was done according to the Agarose Gel Preparation protocol.
The samples were made according to the following proportions:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI> 5 µL DNA </LI><LI>3 µL 10X Cutsmart Buffer </LI><LI> 1 µL EcoRI</LI><LI> 1 µL PstI</LI><LI> 20 µL H2O</LI></UL></P><P>
These samples were then incubated for 10 min at 37°C to allow digestion by the enzymes. 
The reaction was stopped by the addition of 6 µL of 6X Loading Dye. <B>Results</B>:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 6: </B></FIGCAPTION></FIGURE>/br&gt;
<P>This resulting gel shows the two DNA fragments we expected. Moreover, the column 3 shows a lower molecular weight (MW) than others for the plasmid part. As this part is supposed to contain our insert, it might suggest that colony 3 doesn’t possess the insert, but all others do. 
However, we will need to make the right experiment using restriction enzymes XbaI and SpeI (see date 08.02.2018) to verify that our insert was correctly put into the plasmid.</P><P><B>Extraction of Seq6 plasmid from DH5α (midiprep)</B>
We extracted the plasmids containing Seq6 from the DH5-α cells. 
(See Midiprep for plasmid extraction protocol <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here </A>)
Results from NanoDrop: (Blank: TE.1) </P><P><B> For the colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1 </TD><TD> 1572.9 </TD><TD>1.88</TD></TR><TR><TD> 2 </TD><TD> 2800.1 </TD><TD>1.88</TD></TR><TR><TD> 3 </TD><TD> 2598.7 </TD><TD>1.88</TD></TR><TR><TD> 4 </TD><TD> 2285.1 </TD><TD>1.89</TD></TR></TBODY></TABLE><P><B> For the colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1 </TD><TD> 4996.8 </TD><TD>1.87</TD></TR><TR><TD> 2 </TD><TD> 3263.2 </TD><TD>1.89</TD></TR><TR><TD> 3 </TD><TD> 3625.0 </TD><TD>1.90</TD></TR><TR><TD> 4 </TD><TD> 2339.7 </TD><TD>1.90</TD></TR></TBODY></TABLE><P>The DNA concentrations were too high and so did not answer to the Beer-Lambert’s law. We needed to dilute the samples before using the nanodrop again. We did 1/10 dilutions and obtained average concentrations of 34 ng/µL, with Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB> ratios of 1.92. </P><P><B>Digestion of pET 43.1a, NGF_Part1 (Seq1) and NGF_Part2 (Seq2)</B>

Digestion of the following plasmids:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET43.1a (500 ng/µl)</LI><LI>Commercial plasmid containing Seq1 (445.1 ng/<FONT face="Raleway">μ</FONT>L)</LI><LI>Commercial plasmid containing Seq2 (420.6 ng/<FONT face="Raleway">μ</FONT>L)</LI><LI>Restriction enzymes: NdeI, PacI, NheI, BamHI, MscI, HindIII</LI></UL></P><P><OL style="text-align: left;font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 15 min.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pET43.1a</TD><TD>1.6</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>40.4</TD></TR><TR><TD>Ndell</TD><TD>1.5</TD></TR><TR><TD>Pacl</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Eurofins plasmid with Seq1 </TD><TD>4.3</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>37.7</TD></TR><TR><TD>Nhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Eurofins plasmid with Seq2 </TD><TD>4.4</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>37.6</TD></TR><TR><TD>Mscl</TD><TD>1.5</TD></TR><TR><TD>MindIII</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left;font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a heat block at 80 °C for 20 min.</LI></OL></P><P><B>Electrophoresis on agarose gel</B>
We prepared an 0.8 % agarose gel to separate the different fragments of the cut plasmids.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 wells with digested pET43.1a (n°1 and n°2)</LI><LI>2 wells with insert Seq1 (n°3 and n°4)</LI><LI>2 wells with insert Seq2 (n°5 and n°6)</LI><LI>1 well with the DNA ladder</LI></UL></P><P>We let the DNA migrate for 10 minutes at 85 V on order to let the DNA penetrate the gel, and then 40 min at 125 V to efficiently separate the fragments.
We then revealed the gel under UV lamp.</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 7: </B>Electrophoresis gel 0.8% (1. 2. pET43.1a / 3. 4. Seq1 / 5. 6. Seq2) before DNA extraction</FIGCAPTION></FIGURE><P><B>Interpretations</B>: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET43.1a (7275 bp) has been successfully linearized into two fragments (5 kb and 2 kb)</LI><LI>pEXA258 – Seq1 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (1105 bp) and the linearized plasmid (2446 bp)</LI><LI>pEXA258 – Seq2 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (1159 bp) and the linearized plasmid (2446 bp)</LI></UL></P><P><B>DNA Gel extraction</B>:
We cut the gel to extract our inserts and linearized plasmid:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 8: </B>0.8% Agarose gel after excision of the DNA</FIGCAPTION></FIGURE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Eppendorf weight (g)</B></TD><TD><B>Eppendorf + gel weights (g)</B></TD><TD><B>Gel weight (g)</B></TD></TR><TR><TD>pET43.1a (1)</TD><TD>1.13</TD><TD>1.365</TD><TD>0.23</TD></TR><TR><TD>pET43.1a (2)</TD><TD> 1.08 </TD><TD> 1.286 </TD><TD> 0.21 </TD></TR><TR><TD>Seq1 (1)</TD><TD> 1.07 </TD><TD> 1.265 </TD><TD>0.20</TD></TR><TR><TD>Seq1 (2)</TD><TD> 1.08 </TD><TD> 1.306 </TD><TD> 0.23 </TD></TR><TR><TD>Seq2 (1)</TD><TD> 1.07 </TD><TD> 1.303 </TD><TD> 0.23 </TD></TR><TR><TD>Seq2 (2)</TD><TD> 1.07 </TD><TD> 1.284 </TD><TD> 0.21</TD></TR></TBODY></TABLE><P>It is useful to know how the weight of the gel to adapt de volumes of buffer to the amount of gel we have. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)<B>Results</B>: After extracting the DNA from the gel, we measured the DNA concentration using the Nanodrop.<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>C (ng/<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD>pET43.1a (+) (1)</TD><TD> 5.5 </TD><TD> 3.14 </TD></TR><TR><TD>pET43.1a (+) (2))</TD><TD> 7.1 </TD><TD> 2.50 </TD></TR><TR><TD>Seq1 (1)</TD><TD> 5.1 </TD><TD> 2.71 </TD></TR><TR><TD>Seq1 (2)</TD><TD> 7.8 </TD><TD> 1.90 </TD></TR><TR><TD>Seq2 (1)</TD><TD> 6.9 </TD><TD> 2.92 </TD></TR><TR><TD>Seq2 (2)</TD><TD> 6.0 </TD><TD> 3.11 </TD></TR></TBODY></TABLE></P><P><B>Ligation of linearized pET43.1a and our inserts Seq1,2 </B>/br&gt;
We used the In-Fusion cloning kit provided by Ozyme to clone our two inserts together in linearized pET43.1a.<OL style="text-align: left; font-size: 16px;"><LI>We mixed the following volumes into 1.5 mL tubes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Purified Seq1</TD><TD>6</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>Purified Seq2</TD><TD> 6 </TD><TD> - </TD><TD> 2 (control) </TD></TR><TR><TD>Linearized vector</TD><TD> 3 </TD><TD> 1 </TD><TD>1 (control)</TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> 1 </TD><TD> 7</TD><TD> 5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 min in a water bath at 50°C</LI><LI>Ligation (See Ligation protocol)</LI><LI>Set on ice</LI><LI>Transformation of the 3 plasmids into Stellar competent cells (provided with the cloning kit):</LI><LI>Competent cell volume: 50 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight:bold;text-decoration:none;color:black;" target="_blank">here</A> the transformation of <I>E. coli </I> DH5-alpha protocol)<LI>Culture on petri dishes LB agar plate supplemented with carbenicillin overnight at 37°C</LI></OL></P><H2>08.02.2018</H2><P><B>2nd Agarose gel of the miniprep of the ligated plasmid (pBR322 + Seq8)</B> from Stellar, using the correct enzymes
Yesterday we made a mistake, so today we carried out exactly the same protocol but with restriction enzymes SpeI and XbaI. As a negative control, we also deposited a sample we called 1x, without digesting it with restriction enzymes. </P><P><B>Liquid culture of pET43 1.a</B>
We cultivated 2 different colonies in 250 mL Erlenmeyer containing:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>25 mL LB medium</LI><LI>25 <FONT face="Raleway">μ</FONT>L carbenicillin (25 mg/mL)</LI></UL></P><P>We cultivated it overnight in a shaking incubator at 37°C, 180 rpm.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight:bold;text-decoration:none;color:black;" target="_blank">here</A> Liquid Culture protocol) </P><P><B>Liquid culture of Stellar transformed with pET43 1.a-Seq1-Seq2</B>
We cultivated 20 different colonies in order to maximize our chance to get one successful ligated plasmid. Each colony was cultivated in a 15 mL Falcon tube containing:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>5 mL LB medium</LI><LI>5 <FONT face="Raleway">μ</FONT>L carbenicillin (25 mg/mL)</LI></UL></P><P>
We cultivated it overnight in a shaking incubator at 37°C, 180 rpm.&lt;
We also cultivated each colony in an LB agar plate supplemented with carbenicillin.</P><P><B>Extraction of Seq3 plasmid </B>from Stellar (midiprep)
We extracted the plasmids containing Seq3 from the stellar cells
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight:bold;text-decoration:none;color:black;" target="_blank">here</A> the Midiprep for plasmid extraction protocol)
Results from NanoDrop (with Blank: TE.1)<B> For the colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 153.3 </TD><TD>1.90</TD></TR><TR><TD> 155.2 </TD><TD>1.90</TD></TR><TR><TD> 164.6 </TD><TD>1.89</TD></TR><TR><TD> 179.1 </TD><TD>1.91</TD></TR></TBODY></TABLE><P><B> For the colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 166.4 </TD><TD>1.91</TD></TR><TR><TD> 150.4 </TD><TD>1.90</TD></TR><TR><TD> 177.1 </TD><TD>1.91</TD></TR><TR><TD> 188.6 </TD><TD>1.89</TD></TR></TBODY></TABLE><P><B>Interpretations</B>: There is DNA in all the tubes</P><P><B>Digestion of pBR322, Agr-RIP-Part1 (Seq5), Agr-RIP-Part2 (Seq6) and Agr-RIP-Part3 (Seq7)</B>
Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Water baths at 37°C and heating block of 80°C </LI><LI>Eppendorf tubes</LI><LI>Pipets and cones</LI></UL></P><P>Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>PBR322 (conc. 68 ng/<FONT face="Raleway">μ</FONT>L)</LI><LI>Eurofins Genomics plasmid containing Seq5 (conc. 353 ng/ <FONT face="Raleway">μ</FONT>L)</LI><LI>Eurofins Genomics plasmid containing Seq6 (conc. 350 ng/ <FONT face="Raleway">μ</FONT>L)</LI><LI>Eurofins Genomics plasmid containing Seq7 (conc. 305 ng/ <FONT face="Raleway">μ</FONT>L)</LI><LI>Restriction enzymes: BsmI, AvaI, XbaI, BamHI, MscI, SphI, HindIII, SpeI</LI><LI>10X Cut Smart</LI><LI>Distilled water</LI></UL></P><P><OL style="text-align: left; font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 15 min</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pBR322</TD><TD>22</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>20</TD></TR><TR><TD>BsmI</TD><TD>1.5</TD></TR><TR><TD>Aval</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Plasmid containing Seq5 </TD><TD>12</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>30</TD></TR><TR><TD>Xbal</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Plasmid containing Seq7 </TD><TD>21</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>21</TD></TR><TR><TD>HindIIIl</TD><TD>1.5</TD></TR><TR><TD>Spel</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Plasmid </TD><TD>16</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>26</TD></TR><TR><TD>Mscl</TD><TD>1.5</TD></TR><TR><TD>Sphl</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left;font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a heat block at 80 °C for 20 min.</LI></OL></P><P><B>Electrophoresis on agarose gel</B>
We prepared an 0.8 % agarose gel to separate the different fragments of the cut plasmids.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 wells with digested pBR322 (n°1 and n°2)</LI><LI>2 wells with insert Seq5 (n°3 and n°4)</LI><LI>2 wells with insert Seq6 (n°5 and n°6)</LI><LI>2 wells with insert Seq7 (n°7 and n°8)</LI><LI>1 well with the DNA ladder</LI></UL></P><P>We let the DNA migrate for 10 minutes at 85 V on order to let the DNA penetrate the gel, and then 40 min at 125 V to efficiently separate the fragments.
We then revealed the gel under UV lamp.</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 9: </B>ElectroElectrophoresis gel 0.8 % before DNA extraction of pBR322, Seq5, Seq6, Seq7</FIGCAPTION></FIGURE><P style="display:inline-block"><UL style="text-align: left; list-style: none;font-size: 16px;"><LI>1. 2. : pBR322</LI><LI>3. 4. : Seq5 </LI><LI>5. 6. : Seq6 </LI><LI>7. 8. :  Seq7</LI></UL></P><P><B>Interpretations</B>: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET43.1a (7275 bp) has been successfully linearized into two fragments</LI><LI>pEXA258 – Seq5 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (1422 bp) and the linearized plasmid (2.5 kb)</LI><LI>pEXA258 – Seq6 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (960 bp) and the linearized plasmid (2.5 kb)</LI><LI>pEXA258 – Seq7 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (762 bp) and the linearized plasmid (2.5 kb)</LI></UL></P><P><B>DNA Gel extraction</B>
We cut the gel to extract our inserts and linearized plasmid:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 10: </B>Agarose gel electrophoresis after excision of the DNA</FIGCAPTION></FIGURE><P>We extract the DNA (our linearized vector and our plasmid) from the agarose gel.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)</P><P><B>Results</B>: After extracting the DNA from the gel, we measured the DNA concentration using the Nanodrop
<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> pBR322 </TD><TD> 10.3 </TD><TD> 2.07 </TD></TR><TR><TD> Seq5 </TD><TD> 8.3 </TD><TD> 3.07 </TD></TR><TR><TD> Seq6 </TD><TD> 14.0 </TD><TD> 2.06 </TD></TR><TR><TD> Seq7 </TD><TD> 7.6 </TD><TD> 1.97 </TD></TR></TBODY></TABLE></P><P><B>Ligation of linearized pBR322 and our insert Seq5,6,7</B>
We used the In-Fusion cloning kit provided by Ozyme (See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> theLigation protocol)  in order to clone our insert into the linearized pBR322.<OL style="text-align: left; font-size: 16px;"><LI>We mixed the following volumes into 1.5 mL tubes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Purified Seq 5</TD><TD>5</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>Purified Seq 6</TD><TD> 3</TD><TD> -</TD><TD>2 (control)</TD></TR><TR><TD>Purified Seq 7</TD><TD>5</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>Linearized vector</TD><TD> 3 </TD><TD> 1 </TD><TD> 1 (control) </TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 7</TD><TD> 5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 min in a water bath at 50°C</LI><LI>Ligation See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Ligation protocol)</LI><LI>Set in ice</LI><LI>Transformation of the 3 plasmids into Stellar competent cells (provided with the cloning kit)</LI><LI>Competent cell volume : 33 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli</I> DH5-alpha protocol)<LI>Culture on petri dishes LB agar gel supplemented with carbenicillin overnight at 37°C</LI></OL></P><H2>08.03.2018</H2><P><B>Midiprep pET43.1a:</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Midiprep for plasmid extraction protocol)
2 colonies from pET43.1a petri dish (08.01.2018 – pET43.1a) were placed in liquid culture the day before 

Results from NanoDrop (with Blank: TE.1)<B> For the colony 1</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 71.7 </TD><TD> 1.88 </TD></TR><TR><TD> 75.2 </TD><TD>1.88</TD></TR><TR><TD> 73.6 </TD><TD>1.84</TD></TR><TR><TD> 70.2 </TD><TD> 1.87 </TD></TR></TBODY></TABLE><P><B> For the colony 2</B>:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>ng/µL [NA]</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 65.0 </TD><TD>1.85</TD></TR><TR><TD> 34.7 </TD><TD>1.80</TD></TR><TR><TD> 64.9 </TD><TD>1.86</TD></TR><TR><TD> 55.5 </TD><TD>1.82</TD></TR></TBODY></TABLE><P><B>Observations</B>: Enough concentrations for further uses </P><P><B>Miniprep pET 43.1a + Seq1 + Seq2</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
20 colonies from pET43.1a + Seq1 + Seq2 from petri dish (08.01.2018) were placed in liquid culture the day before (5 mL of LB + 10 µL of Carbenicillin).
Results from NanoDrop (Blank used is Buffer EB):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 170.9 </TD><TD> 1.86 </TD></TR><TR><TD> 2 </TD><TD> 120.8 </TD><TD> 1.88 </TD></TR><TR><TD> 3 </TD><TD> 73.1 </TD><TD> 1.87 </TD></TR><TR><TD> 4   </TD><TD> 89.8 </TD><TD> 1.84 </TD></TR><TR><TD> 5   </TD><TD> 75.4 </TD><TD> 1.97 </TD></TR><TR><TD> 6   </TD><TD> 136.6 </TD><TD> 1.87 </TD></TR><TR><TD> 7 </TD><TD> 101.4 </TD><TD> 1.88 </TD></TR><TR><TD> 8 </TD><TD> 99.0 </TD><TD>1.91</TD></TR><TR><TD> 9   </TD><TD>72.7</TD><TD>1.88</TD></TR><TR><TD> 10 </TD><TD> 172.8 </TD><TD> 1.87 </TD></TR><TR><TD> 11  </TD><TD> 107.8 </TD><TD> 1.86 </TD></TR><TR><TD> 12  </TD><TD> 151.1 </TD><TD> 1.86 </TD></TR><TR><TD> 13  </TD><TD> 92.6 </TD><TD>1.92</TD></TR><TR><TD> 14  </TD><TD> 78.7 </TD><TD>1.93</TD></TR><TR><TD> 15  </TD><TD> 101.2 </TD><TD> 1.87 </TD></TR><TR><TD> 16  </TD><TD> 223.9 </TD><TD> 1.97 </TD></TR><TR><TD> 17  </TD><TD> 145.5 </TD><TD> 1.85 </TD></TR><TR><TD> 18  </TD><TD> 149.8 </TD><TD> 1.86 </TD></TR><TR><TD> 19 </TD><TD> 68.1 </TD><TD> 1.89 </TD></TR><TR><TD> 20 </TD><TD> 87.8 </TD><TD>1.93</TD></TR></TBODY></TABLE><P><B>Observations</B>: Samples were not transferred to the micro tubes at the right moment, so it is very probable that the samples are contaminated with genomic DNA.</P><H2>08.05.2018</H2><P><B>Liquid culture of pBR322 + Seq5 + Seq6 + Seq7</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)
6 colonies from the petri dish (pBR322 + Seq5,6,7 – 08.02.2018) were placed in a liquid culture containing: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>5 mL of LB </LI><LI>10 µL of Carbenicillin (25 mg/mL)</LI></UL></P><P>Cultures were incubated at 37°C, 180 rpm overnight. <B>Observations</B>: All the colonies grew. </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 11: </B>Petri dish with the 6 colonies that were placed in liquid culture</FIGCAPTION></FIGURE></DIV><DIV data-description="Week_31" class="panel" id="pan_3102" style="text-align:left;"><H2> 07.30.2018 </H2><P> First PDMS well chip entirely assembled and bonded, with platinum wire and gold membrane filter. Leaking problems required us to seal the leaks with additional PDMS. 4 microchannel chips and 1 set of PDMS pieces for the well chip have been made. </P><P> Well chip fabrication protocol (the chip assembled on that day is slightly different but doesn't work) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2> 07.31.2018 </H2><P> 2 additional PDMS wells assembled with gold-coated membranes. First PDMS vertical chip PDMS pieces prepared (cut and punched, silver paste applied and membrane filter glued). 4 microchannel chips and 1 set of PDMS pieces for the well chip have been made.  </P><P> Well chip fabrication protocol (the chip assembled on that day is slightly different but doesn't work) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Vertical chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/d/da/T--Pasteur_Paris--Vertical-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2> 08.01.2018 </H2><P> Conductivity tests on membranes assembled on chips. Well chips not functional: tension is applied on top of the membrane, but no current flow is detected. We conclude from that, that the current can't pass from one side of the gold-coated membrane to the other. It was decided for the well chips to stop using silver paste for connecting the platinum wire to the membrane and instead mount the wire on top of the membrane. </P><P> Bonding of the vertical chip PDMS pieces failed. </P><P> Well chip conductivity protocol (section 2) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Vertical chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/d/da/T--Pasteur_Paris--Vertical-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_32"><DIV class="block full" id="Week_32" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_32" class="vignette" id="vign_3200"><DIV data-description="Week_32" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_32" class="vignette" id="vign_3201" style="display:none"><DIV data-description="Week_32" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_32" class="vignette" id="vign_3202"><DIV data-description="Week_32" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_32" class="panel" id="pan_3200" style="text-align:left;"><H2>08.06.2018</H2><P><B>Miniprep pBR322 + Seq5 + Seq6 + Seq7</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)<B>Results</B> from NanoDrop (Blank used is buffer EB):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 61.2 </TD><TD> 2.00 </TD></TR><TR><TD> 2 </TD><TD> 86.3 </TD><TD>2.03</TD></TR><TR><TD> 3 </TD><TD> 58.8 </TD><TD>1.98</TD></TR><TR><TD> 4   </TD><TD> 65.9 </TD><TD>2.00</TD></TR><TR><TD> 5   </TD><TD> 74.5 </TD><TD>2.03</TD></TR><TR><TD> 6   </TD><TD> 84.4 </TD><TD>2.07</TD></TR></TBODY></TABLE><P><B>Observations</B>: Enough concentrations for further use. </P><P><B>Digestion of pET43.1a + Seq1 + Seq2 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: NdeI and XbaI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pET43.1a </B></TD><TD><B>linearized   </B></TD><TD><B>7.3 kB</B></TD></TR><TR><TD> pET43.1a + Seq1</TD><TD> 1 fragment</TD><TD>6.63 kB </TD></TR><TR><TD> pET43.1a + Seq2 </TD><TD> 2 fragments </TD><TD>6.63 kB + 0.1 kB</TD></TR><TR><TD> pET43.1a + Seq1+ Seq2   </TD><TD> 2 fragments </TD><TD>6.63 kB + 1 kB</TD></TR></TBODY></TABLE><P>
We digested 10 samples of our cloning products, and we also did some controls:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Colony 10 with no enzyme (well 11)</LI><LI>Plasmid pET43.1a + enzyme (well 12)</LI><LI>Plasmid pET43.1a + no enzyme (well 13)</LI></UL></P><P>
Well 1 to 10: samples from colony 1 to 10
Well 14: ladder<B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>5</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Ndel</TD><TD>1</TD></TR><TR><TD>Xbal</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>20</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 15 min</LI><LI>Addition of 6 µL of loading dye</LI></UL></P><P>We deposited all the samples in an agarose gel previously made. 
We let the gel run for 10 min at 85 V + 80 min at 125 V. <B>Results</B>:</P><P><B>Observations</B>: For samples 1 to 11, we can only see DNA streaks. In wells 12 and 13, we can clearly see the plasmid pET43.1a linearized (well 12) and circular (well 13). In the well 14 we can clearly see the ladder. <B>Interpretations</B>: The streaks observed on the gel are probably due to the genomic DNA contamination. As mentioned the 08.03.2018, there was an error in the Miniprep of pET43.1a + Seq1 + Seq2. </P><P><B>Digestion of pBR322+ Seq5 + Seq6 + Seq7 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzyme used</U>: EcoRI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pBR322 </B></TD><TD><B>linearized   </B></TD><TD><B>4 kB</B></TD></TR><TR><TD> pBR322 + Seq5</TD><TD> 1 fragment</TD><TD>5.4 kB </TD></TR><TR><TD> pBR322 + Seq5 + Seq6 </TD><TD> 2 fragments </TD><TD>6.3 kB </TD></TR><TR><TD>pBR322 + Seq5 + Seq6 + Seq7  </TD><TD> 2 fragments </TD><TD>7 kB</TD></TR></TBODY></TABLE><P><B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>5</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>20</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 15 min</LI><LI>Addition of 6 µL of loading dye</LI></UL></P><P>We digested 6 samples of our cloning products, and we also did some controls.  <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Colony 6 with no enzyme (well 7)</LI><LI>Plasmid pBR322 + enzyme (well 8)</LI><LI>Plasmid pBR322 + no enzyme (well 9)</LI></UL></P><P>Wells 1 to 6: samples from colony 1 to 6
Well 10: ladder

We deposited all the samples in an agarose gel previously made. 
We let the gel run for 10 min at 85 V + 90 min at 125 V. <B>Results</B>:</P><P><B>Observations</B>: The 6 samples are linearized and migrated exactly like the plasmid pBR322 + Enzyme. Moreover, sample 6 without enzyme migrated like pBR322 without enzyme. <B>Interpretations</B>: This leads to the conclusion that none of the colonies were transformed with the plasmid AND the insert. </P><P><B>Liquid culture of pET43.1a + Seq1 + Seq2</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)

8 colonies from the petri dish (pET43.1a + Seq1,2 from 08.01.2018) were placed in a liquid culture containing: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>5 mL of LB </LI><LI>5 µL of Carbenicillin (25 mg/mL)</LI></UL></P><P>Cultures were incubated at 37°C, 180 rpm overnight. </P><P><B>Cloning pBR322 + Seq8</B>
We used the In-Fusion cloning kit provided by Ozyme 
in order to clone our insert in the linearized pET 43.1a.<OL style="text-align: left;font-size: 16px;"><LI>We mixed the following volumes into 1.5mL tubes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Purified Seq8</TD><TD>4</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>Linearized vector</TD><TD>15 </TD><TD> 1 </TD><TD>1 (control)</TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 7</TD><TD> 5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 minutes in a water bath at 50°C</LI><LI>Ligation (See Ligation protocol)</LI><LI>Set in ice</LI><LI>Transformation of the 3 plasmids into Stellar competent cells (provided with the clonage kit):</LI><LI>Competent cell volume: 50 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI><LI>Culture on petri dishes LB agar plate supplemented with carbenicillin overnight at 37°C</LI></OL></P><H2>08.07.2018</H2>&lt;+br&gt;
<P><B>Miniprep pET43.1a + Seq 1 + Seq 2</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)<B>Results</B> from NanoDrop (Blank used is buffer EB):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 141.2 </TD><TD> 1.84 </TD></TR><TR><TD> 2 </TD><TD> 173.3 </TD><TD> 1.86</TD></TR><TR><TD> 3 </TD><TD> 155.2 </TD><TD>1.83</TD></TR><TR><TD> 4   </TD><TD> 151.9 </TD><TD>1.85</TD></TR><TR><TD> 5   </TD><TD> 106.9 </TD><TD> 1.85 </TD></TR><TR><TD> 6   </TD><TD> 143.0 </TD><TD> 1.88 </TD></TR><TR><TD> 7   </TD><TD> 151.1 </TD><TD> 1.89 </TD></TR></TBODY></TABLE><P><B>Observations</B>: Enough concentrations for further use. </P><P><B>Digestion of pET43.1a + Seq1 + Seq2 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: NdeI and XbaI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pET43.1a </B></TD><TD><B>linearized   </B></TD><TD><B>7.3 kB</B></TD></TR><TR><TD> pET43.1a + Seq1</TD><TD> 1 fragment</TD><TD>6.63 kB </TD></TR><TR><TD> pET43.1a + Seq2 </TD><TD> 2 fragments </TD><TD>6.63 kB + 0.1 kB</TD></TR><TR><TD> pET43.1a + Seq1+ Seq2   </TD><TD> 2 fragments </TD><TD>6.63 kB + 1 kB</TD></TR></TBODY></TABLE><P>
We digested 10 samples of our cloning products, and we also did some controls:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Colony 8 with no enzyme (well 8)</LI><LI>Plasmid pET43.1a + enzyme (well 9)</LI><LI>Plasmid pET43.1a + no enzyme (well 10)</LI></UL></P><P>
Well 1 to 7: samples from colony 1 to 8
Well 11: ladder<B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>5</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Ndel</TD><TD>1</TD></TR><TR><TD>Xbal</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>20</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 15 min</LI><LI>Addition of 6 µL of loading dye</LI></UL></P><P>We deposited all the samples in an agarose gel previously made. 
We let the gel run for 10 min at 85 V + 80 min at 125 V. </P><P><B>Liquid culture of pBR322 + Seq8</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the liquid culture protocol)

We cultivated:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>8 colonies of Stellar bacteria transformed with the result of our cloning </LI><LI>2 colonies from the negative control </LI></UL></P><P>Each colony is cultivated in a 15 mL Falcon tube with 5 mL LB medium and 5 <FONT face="Raleway">μ</FONT>L carbenicillin.
In parallel, we set each colony to grow on a LB agar plate supplemented with carbenicillin overnight at 37°C.</P><H2>08.08.2018</H2><P><B>Miniprep pBR322 + Seq8</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)<B>Results</B> from NanoDrop (Blank used is buffer EB):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 93.0 </TD><TD> 1.97 </TD></TR><TR><TD> 2 </TD><TD> 101.8 </TD><TD> 1.83 </TD></TR><TR><TD> 3 </TD><TD> 121.7 </TD><TD>1.98</TD></TR><TR><TD> 4   </TD><TD> 112.1 </TD><TD>2.00</TD></TR><TR><TD> 5   </TD><TD> 101.9 </TD><TD> 1.98 </TD></TR><TR><TD>6 </TD><TD> 97.5 </TD><TD> 1.98 </TD></TR><TR><TD> 7   </TD><TD> 113.4 </TD><TD> 1.98 </TD></TR><TR><TD> 8</TD><TD> 97.0 </TD><TD> 2.02 </TD></TR><TR><TD> Control (-) 1   </TD><TD> 91.9 </TD><TD>2.01 </TD></TR><TR><TD> Control (-) 2 </TD><TD> 73.5 </TD><TD> 1.90 </TD></TR></TBODY></TABLE><P><B>Observations</B>: Concentrations are good for further use. </P><P><B>Digestion of BR322 + Seq8</B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: NdeI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pET43.1a </B></TD><TD><B>linearized   </B></TD><TD><B>4 kB</B></TD></TR><TR><TD> pET43.1a + Seq8 </TD><TD> 2 fragments </TD><TD>4 kB + 0.9 kB</TD></TR></TBODY></TABLE><P>
We digested 8 samples of our cloning products, and we also did some controls:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Colony 8 with no enzyme (well 8)</LI><LI>Two colonies from the negative control of the cloning with enzyme</LI></UL></P><P>We filled the wells with:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Plasmid extracted from colonies 1 to 8 and digested with NdeI (n°1 to n°8)</LI><LI>Plasmid extracted from the negative control digested with NdeI (n°9 and n°10)</LI><LI>Plasmid extracted from colony 8 without enzyme (n°11)</LI></UL></P><P><B>Results</B>: <B>Interpretations</B>: We observed only one fragment for all the digested plasmids, and all the fragments have the same molecular weight as the negative control (vector without insert).
Cloning did not work again. 

We assume that the failure might be:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>A too low initial amount of linearized vector before cloning, resulting in all the Stellar bacteria being transformed with an empty and uncut vector.</LI><LI>Too long overlaps, or too GC rich overlaps.</LI></UL></P><P><B>Digestion of pBR322, PET 43.1, AND PLASMIDS CONTAINING Seq1, Seq2 and Seq8</B>
We plan to try cloning again with longer digestion and longer electrophoresis migration, in order to digest all the vector and to increase the efficiency of our separation.

Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pBR322 (conc. 54 ng/<FONT face="Raleway">μ</FONT>l)</LI><LI>pET 43.1a (conc. 65 ng/<FONT face="Raleway">μ</FONT>l)</LI><LI>Seq1 (conc. 445 ng/<FONT face="Raleway">μ</FONT>l)</LI><LI>Seq2 (conc. 460 ng/<FONT face="Raleway">μ</FONT>l)</LI><LI>Seq8 (conc. 379 ng/<FONT face="Raleway">μ</FONT>l)</LI></UL></P><P>We digest the 5 plasmids by mixing the following amounts:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA</TD><TD>22</TD></TR><TR><TD> Cut Smart </TD><TD>3.2</TD></TR><TR><TD>Enzyme 1</TD><TD>1</TD></TR><TR><TD>Enzyme 2</TD><TD>1</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>DNA</B></TD><TD><B>Enzyme 1</B></TD><TD><B>Enzyme 2</B></TD></TR><TR><TD> pET 43.1a</TD><TD> Ndel  </TD><TD> Pacl </TD></TR><TR><TD> pBR322 </TD><TD> Aval </TD><TD> Bsml </TD></TR><TR><TD> Seq1 </TD><TD> Nhal </TD><TD>BamHI</TD></TR><TR><TD> Seq2   </TD><TD> Mscl </TD><TD>HindIII</TD></TR><TR><TD>Seq8    </TD><TD> Xbal </TD><TD> Spel </TD></TR></TBODY></TABLE><P>Incubation time at 37°C : 1 hour
Inactivation time at 80°C : 20 min

After digestion, we added 8 <FONT face="Raleway">μ</FONT>L 6X Loading dye to each tube.</P><P><B>Electrophoresis on agarose gel</B>&lt;
We prepared a 0.8% agarose gel in order to separate the different fragments of the cut plasmids.

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 wells with digested pBR322 (n°1 and n°2)</LI><LI>2 wells with digested pET 43.1a (n°3 and n°4)</LI><LI>2 wells with insert Seq1 (n°5 and n°6)</LI><LI>2 wells with insert Seq2 (n°7 and n°8)</LI><LI>2 wells with insert Seq8 (n°9 and n°10)</LI></UL></P><P>We let the DNA migrate for 10 min at 85 V on order to let the DNA penetrate the gel, and then 80 min at 125 V to efficiently separate the fragments.
We then revealed the gel under UV lamp.<B>Results</B>: DNA Gel extraction
We cut the gel to extract our inserts and linearized plasmid:
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the  Gel Extraction protocol) <B>Results</B>: After extracting the DNA from the gel, we measured the DNA concentration using the Nanodrop.
<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> pBR322 </TD><TD> 7.0 </TD><TD> 3.85 </TD></TR><TR><TD> pET43.1a  </TD><TD> 5.1 </TD><TD> 3.40 </TD></TR><TR><TD> Seq1 </TD><TD> 5.7</TD><TD> 3.40 </TD></TR><TR><TD> Seq2 </TD><TD> 6.3 </TD><TD> 10.55 </TD></TR><TR><TD> Seq8 </TD><TD> 5.0</TD><TD> 4.48 </TD></TR></TBODY></TABLE></P><P><B>Interpretations</B>: Values of the ratio Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB> are very high, indicating that our DNA is highly contaminated. Moreover, the concentrations obtained are too low for further use for cloning.
We have several hypotheses to explain why our experiment did not work:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>The Gel extraction kit we used might be expired (particularly the final buffer with which we elute DNA might have turned unusable)</LI><LI>We need to check the wavelength of the nanodrop</LI></UL></P><P>We are going to try the digestion and extraction again tomorrow in order to try having better concentrations before proceeding to cloning.</P><H2>08.09.2018</H2>
The digestion and extraction of yesterday did not work well (the obtained concentrations of plasmid and insert were still too low to proceed to the cloning step with a suitable volume).
Today, we will proceed again with the digestion, extraction and then try the cloning again, using this time the right amount of insert and vector (as recommended by Clontech molar ratio tool) and following the pieces of advice given to us by Takara Clontech technician we called.<P><B>Digestion of pET 43.1a, pBR322, Seq1, Seq2 and Seq8</B>

Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Water baths at 37°C and heating block of 80°C </LI><LI>Eppendorf tubes</LI><LI>Pipets and cones</LI></UL></P><P>
Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET 43.1a (conc. 74.1 ng/<FONT face="Raleway">μ</FONT>L)</LI><LI>pBR322 (conc. 29.6 ng/<FONT face="Raleway">μ</FONT>L)</LI><LI>Eurofins Genomics plasmid containing Seq1 (conc. 1430.1 ng/ <FONT face="Raleway">μ</FONT>L)</LI><LI>Eurofins Genomics plasmid containing Seq2 (conc. 780.6 ng/ <FONT face="Raleway">μ</FONT>L)</LI><LI>Eurofins Genomics plasmid containing Seq8 (conc. 714.6 ng/ <FONT face="Raleway">μ</FONT>L)</LI><LI>Restriction enzymes : NdeI, PacI, NheI, BamHI, MscI, HindIII, BsmI, AvaI, SpeI, XbaI</LI><LI>10X Cut Smart</LI><LI>Distilled water</LI></UL></P><P><OL style="text-align: left;font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 20 min</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pET 43.1a </TD><TD>42</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>-</TD></TR><TR><TD>Nhel</TD><TD>1.5</TD></TR><TR><TD>Pacl</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pBR322 </TD><TD>42</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>-</TD></TR><TR><TD>Bsml</TD><TD>1.5</TD></TR><TR><TD>Aval</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq1 </TD><TD>4.5</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>31.5</TD></TR><TR><TD>Nhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq2 </TD><TD>8</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>32</TD></TR><TR><TD>Mscl</TD><TD>1.5</TD></TR><TR><TD>HindIII</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq8 </TD><TD>17.8</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>24.2</TD></TR><TR><TD>Xbal</TD><TD>1.5</TD></TR><TR><TD>Spel</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a water bath at 80 °C for 20 min</LI></OL></P><P><B>Electrophoresis on agarose gel</B>
We prepared a 0.8 % agar gel in order to separate the different fragments of the cut plasmids.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 wells with digested pBR322 (n°1 and n°2)</LI><LI>2 wells with digested pET 43.1a (n°3 and n°4)</LI><LI>2 wells with insert Seq1 (n°5 and n°6)</LI><LI>2 wells with insert Seq2 (n°7 and n°8)</LI><LI>2 wells with insert Seq8 (n°9 and n°10)</LI></UL></P><P>
We let the DNA migrate for 10 min at 85 V on order to let the DNA penetrate the gel, and then 80 min at 125 V to efficiently separate the fragments.
We then revealed the gel under UV lamp.
Results of the agarose gel after migration:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 4: </B> Agarose 1% gel after electrophoresis </FIGCAPTION></FIGURE><P><B>Interpretations:</B><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pBR322 has been linearized, and we observe one fragment corresponding to the linearized plasmid</LI><LI>pET 43.1a (7275 bp) has been successfully linearized into two fragments (5 kb and 2 kb)</LI><LI>pEX A258 – Seq1 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (1105 bp) and the linearized plasmid (2446 bp)</LI><LI>pEX A258 – Seq2 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (1159 bp) and the linearized plasmid (2446 bp)</LI><LI>pEX A258 – Seq8 from Eurofins Genomics has also been successfully digested. We obtained two fragments corresponding to our insert (459 bp) and the linearized plasmid (2446 bp)</LI></UL></P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 5: </B>Agarose gel 1% after electrophoresis of digested pBR322, pET43.1a, Seq1, Seq2 and Seq8 and extraction of the sequences of interest.</FIGCAPTION></FIGURE>
 (See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol) <B>Results</B>:
After extracting the DNA from the gel, we measured the DNA concentration using the Nanodrop<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> pBR322 </TD><TD> 11.1 </TD><TD> 2.19 </TD></TR><TR><TD> pET43.1a  </TD><TD> 17.0 </TD><TD> 2.00 </TD></TR><TR><TD> Seq1 </TD><TD> 16.9 </TD><TD> 2.19 </TD></TR><TR><TD> Seq2 </TD><TD> 14.8 </TD><TD> 2.61 </TD></TR><TR><TD> Seq8 </TD><TD> 15.7 </TD><TD> 2.17 </TD></TR></TBODY></TABLE><P><B>Cloning </B>:
We used the In-Fusion cloning kit provided by Ozyme in order to clone:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Our two inserts Seq1 and Seq2 together in linearized pET 43.1a.</LI><LI>Our insert Seq8 into linearized pBR322</LI></UL></P><P><OL style="text-align: left;font-size: 16px;"><LI>Following the advice of the Takara Clontech technician we contacted, we first incubated the linearized vectors and inserts at 50°C for 5 min</LI><LI>We then placed it on ice for 3 min</LI><LI>We mixed the following volumes into 1.5 mL tubes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Purified Seq1</TD><TD>5</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>Purified Seq2</TD><TD>6 </TD><TD> -</TD><TD>2 (control)</TD></TR><TR><TD>Linearized vector</TD><TD>9 </TD><TD> 9 </TD><TD>1 (control)</TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 9</TD><TD> 5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Purified Seq8</TD><TD>4.5</TD><TD>-</TD></TR><TR><TD>Linearized vector</TD><TD>13.5 </TD><TD> 13.5 </TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 4.5</TD></TR></TBODY></TABLE><P><OL start="4" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 minutes in a water bath at 50°C.</LI><LI>Set in ice.</LI><LI>Ligation (See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Ligation protocol)</LI><LI>Transformation of the 5 plasmids into Stellar competent cells (provided with the cloning kit):</LI><LI>Competent cell volume: 40 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of <I>E. coli</I> DH5-alpha protocol)<LI>Culture on LB agar plate supplemented with carbenicillin overnight at 37°C (with 1/5 dilution)</LI></OL></P><H2>08.10.2018</H2><P><B>Observation overnight plate culture of pBR322 + Seq8 and pET43.1a + Seq1 + Seq2 after InFusion</B></P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 6: </B>pBR322_ NegControl and pBR322 Seq8. </FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 7: </B>pET43.1 + Seq1+2 and pET43.1_ NegControl. </FIGCAPTION></FIGURE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>pBR322 Neg Control</B></TD><TD><B>pBR322 + Seq8</B></TD><TD><B>pET43.1 Neg Control</B></TD><TD><B>pET43.1 + Seq1 + Seq2</B></TD></TR><TR><TD>CFU</TD><TD>46</TD><TD>136</TD><TD>27</TD><TD>59</TD></TR></TBODY></TABLE><P><B>Liquid culture of pBR322 + Seq8 and pET43.1a + Seq1 + Seq2</B>
 We did 12 liquid cultures of 5 mL (6 of each).</P><H2>08.11.2018</H2><P><B>Miniprep pET43.1a + Seq1 + Seq2 and pBR322 + Seq8</B>
We extracted our plasmids from the colonies set in liquid culture yesterday. We had:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>5 colonies of pET43.1a + Seq1+ Seq2</LI><LI>6 colonies of pBR322 + Seq8</LI><LI>1 colony from negative control of pET43.1</LI><LI>1 colony from negative control of pBR322</LI></UL></P>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)

<B>Results</B> from NanoDrop (Blank used is buffer EB):<B>For pET 43.1a + Seq1 + Seq2 </B><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Colony</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 116.5 </TD><TD> 1.99 </TD></TR><TR><TD> 2 </TD><TD> 124.8 </TD><TD>1.95</TD></TR><TR><TD> 3 </TD><TD> 98.1 </TD><TD> 2.01 </TD></TR><TR><TD> 4   </TD><TD> 114.3 </TD><TD> 1.95 </TD></TR><TR><TD> 5   </TD><TD> 96.8 </TD><TD> 1.96 </TD></TR></TBODY></TABLE><P><B>For pBR322 + Seq8</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Colony</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 88.7 </TD><TD> 2.03 </TD></TR><TR><TD> 2 </TD><TD> 105.6 </TD><TD>1.95</TD></TR><TR><TD> 3 </TD><TD> 84.0 </TD><TD> 2.12 </TD></TR><TR><TD> 4   </TD><TD> 104.0 </TD><TD> 1.97 </TD></TR><TR><TD> 5   </TD><TD> 95.6 </TD><TD> 2.08 </TD></TR><TR><TD> 6   </TD><TD> 90.1 </TD><TD> 2.01 </TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Colony</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> pET 43.1 negative control</TD><TD> 110.6 </TD><TD> 1.98 </TD></TR><TR><TD> pBR322 negative control </TD><TD> 106.2 </TD><TD>2.02</TD></TR></TBODY></TABLE><P><B>Observations</B>: Concentrations are good enough for further use.  </P><P><B>Digestion of pET43.1a + Seq1 + Seq2 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: NdeI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pET43.1a </B></TD><TD><B>linearized   </B></TD><TD><B>7.3 kB</B></TD></TR><TR><TD> pET43.1a + Seq1</TD><TD> 1 fragment</TD><TD>6.63 kB </TD></TR><TR><TD> pET43.1a + Seq2 </TD><TD> 2 fragments </TD><TD>6.63 kB + 0.1 kB</TD></TR><TR><TD> pET43.1a + Seq1+ Seq2   </TD><TD> 2 fragments </TD><TD>6.63 kB + 1 kB</TD></TR></TBODY></TABLE><P>
We digested 5 samples of our cloning products, and we also did some controls.<B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>5</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Ndel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>21</TD></TR></TBODY></TABLE><P><B>Digestion of pBR322 + Seq8 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: NdeI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD> pBR322</TD><TD> linearized</TD><TD>4 kB </TD></TR><TR><TD> pBR322 + Seq8 </TD><TD> 2 fragments </TD><TD>4 kB + 0.9 kB</TD></TR></TBODY></TABLE><P>
We digested 6 samples of our cloning products, and 1 negative control.<B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>5</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Ndel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>21</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 15 min</LI><LI>Addition of 6 µL of 6X loading dye in each tube</LI></UL></P><P><B>Electrophoresis on agarose gel</B>
We prepared an Agarose gel. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Wells 1 to 6: pBR322 + Seq8 (colonies 1 to 6)</LI><LI>Well 7: pBR322 negative control</LI><LI>Wells 8 to 12: pET43.1a + Seq1 + Seq2 (colonies 1 to 5)</LI><LI>Well 13: pET 43.1 negative control</LI></UL></P><P>
We let the DNA migrate for 10 minutes at 85 V and then 40 minutes at 125 V.<B>Results</B>: </P><P><B>Remark</B>: Date shift -2 days on photograph<B>Interpretations</B>: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>For pBR322 + Seq8: On lanes corresponding to Col 1,3,4,5,6 two bands are clearly visible. One at 4 kb and another one at 1 kb, meaning that the plasmid digested by NdeI contained the insert Seq8. One band is visible on lane 2, Col 2 did not contain the insert. The control on lane 7 shows the plasmid digested without the insert at a higher molecular weight. The cloning worked for this construction, because bands observed between 700 bp and 1000 bp.</LI><LI>For pET43.1a + Seq1 + Seq2: From lanes 8 to 11, only one band is visible, and bands are visible at the same molecular weight as the control one (pET43.1a w/o insert and digested with NdeI). On lane 12, the band is lightly lower than the others but no other band corresponding to an insert is visible on lane 12. The cloning did not work for this construction.</LI></UL></P><P><B>Liquid culture pET43.1a + Seq1 + Seq2 and pBR322 + Seq8</B>
Colonies transformed with pET43.1a+Seq1+Seq2 (Colonies 7 to 25) were cultured in 5 mL LB media + 5 µL carbenicillin each (50 mg/mL)
Two culture of pBR322+ Seq8 (Col 3 and Col 4) were cultured in 25 mL LB media + 25 µl carbenicillin each (50 mg/mL)
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol) </P></DIV><DIV data-description="Week_32" class="panel" id="pan_3202" style="text-align:left;"><H2> 08.06.2018</H2><P> 20 microchannel chips bonded to imaging dishes. 8 microchannel chips and 1 set of PDMS pieces for the well chip have been made. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication and bonding protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2>08.07.2018 </H2><P> 20 bonded microchannel chips sterilized by UV exposure. 8 microchannel chips and 1 set of PDMS pieces for the well chip have been made. Preparation of vertical chip PDMS pieces. </P><P> Well chip fabrication protocol (the chip assembled on that day is slightly different but doesn't work) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Vertical chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/d/da/T--Pasteur_Paris--Vertical-chip.pdf" target="_blank"> here </A>. </P><P> Microchannel chip fabrication and sterilization protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><H2> 08.09.2018</H2><P> 10 membrane microchannel chips have been made. Leaks made this a difficult operation. 8 microchannel chips and 1 set of PDMS pieces for the well chip have been made. Preparation of vertical chip PDMS pieces. </P><P> Membrane microchannel chip and basic microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Vertical chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/d/da/T--Pasteur_Paris--Vertical-chip.pdf" target="_blank"> here </A>. </P><H2> 08.10.2018</H2><P> Membrane microchannel chips sterilized by UV exposure. Vertical chips bonded. 8 microchannel chips and 1 set of PDMS pieces for the well chip have been made. First successful vertical chip bonding, by modifying silver paste path. </P><P> Microchannel chip fabrication and sterilization protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_33"><DIV class="block full" id="Week_33" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_33" class="vignette" id="vign_3300"><DIV data-description="Week_33" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_33" class="vignette" id="vign_3301" style="display:none"><DIV data-description="Week_33" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_33" class="vignette" id="vign_3302"><DIV data-description="Week_33" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_33" class="panel" id="pan_3300" style="text-align:left;"><H2>08.13.2018</H2><P><B>Miniprep pET 43.1a + Seq1 + Seq2</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
20 colonies from pET43.1a + Seq1 + Seq2 from petri dish (08.01.2018) were placed in liquid culture the day before (5 mL of LB + 10 µL of Carbenicillin).
Results from NanoDrop (Blank used is Buffer EB):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 96.6 </TD><TD> 1.97 </TD></TR><TR><TD> 2 </TD><TD> 83.5 </TD><TD> 1.81 </TD></TR><TR><TD> 3 </TD><TD> 109.8 </TD><TD> 1.97 </TD></TR><TR><TD> 4   </TD><TD> 89.8 </TD><TD> 1.84 </TD></TR><TR><TD> 5   </TD><TD> 85.2 </TD><TD> 1.95 </TD></TR><TR><TD> 6   </TD><TD> 113.6 </TD><TD> 1.96 </TD></TR><TR><TD> 7 </TD><TD> 91.9 </TD><TD> 1.96 </TD></TR><TR><TD> 8 </TD><TD> 96.6 </TD><TD>1.97</TD></TR><TR><TD> 9   </TD><TD> 82.3 </TD><TD>1.96</TD></TR><TR><TD> 10 </TD><TD> 89.3 </TD><TD> 1.96</TD></TR><TR><TD> 11  </TD><TD> 133 </TD><TD> 1.98 </TD></TR><TR><TD> 12  </TD><TD> 67.7 </TD><TD> 1.90 </TD></TR><TR><TD> 13  </TD><TD> 94.6 </TD><TD>1.94</TD></TR><TR><TD> 14  </TD><TD> 87.3 </TD><TD>1.95</TD></TR><TR><TD> 15  </TD><TD> 80.5 </TD><TD> 1.94 </TD></TR><TR><TD> 16  </TD><TD> 94.2 </TD><TD> 1.98 </TD></TR><TR><TD> 17  </TD><TD> 94.2 </TD><TD> 1.93 </TD></TR><TR><TD> 18  </TD><TD> 103 </TD><TD> 1.95 </TD></TR><TR><TD> 19 </TD><TD> 90.4 </TD><TD> 1.95 </TD></TR><TR><TD> 20 </TD><TD> 131.5 </TD><TD>1.96</TD></TR></TBODY></TABLE><P><B>Digestion of pET 43.1a + Seq1 + Seq2</B>

Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET 43.1a + Seq1 + Seq2 </LI><LI>Restriction enzymes : NdeI, PacI, BsmI, AvaI, NheI, BamHI, MscI, HindIII, XbaI, SpeI</LI><LI>10X Cut Smart</LI><LI>Distilled water </LI></UL></P><P><OL style="text-align: left;font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 20 min</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pET 43.1a </TD><TD>42</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>-</TD></TR><TR><TD>Ndel</TD><TD>1.5</TD></TR><TR><TD>Pacl</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pBR322 </TD><TD>42</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>-</TD></TR><TR><TD>Bsml</TD><TD>1.5</TD></TR><TR><TD>Aval</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq1 </TD><TD>4.5</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>31.5</TD></TR><TR><TD>Nhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq2 </TD><TD>8</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>32</TD></TR><TR><TD>Mscl</TD><TD>1.5</TD></TR><TR><TD>HindIII</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq8 </TD><TD>17.8</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>24.2</TD></TR><TR><TD>Hhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a water bath at 80 °C for 20 min</LI></OL></P><P><B>Electrophoresis on agarose gel</B>
We prepared a 1 % agarose gel in order to separate the different fragments of the cut plasmids.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 wells with digested pET43.1a  (n°1 and n°2)</LI><LI>10 wells with digested pET43.1a with NdeI (n°4 until n°13)</LI></UL></P><P>
We let the DNA migrate for 10 min at 85 V on order to let the DNA penetrate the gel, and then 80 min at 125 V to efficiently separate the fragments.
We then revealed the gel under UV lamp.
Results of the agarose gel after migration:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 4: </B> Agarose gel 1% after electrophoresis of digested pBR322, pET43.1a, Seq1, Seq2 and Seq8. </FIGCAPTION></FIGURE><P><B>Interpretations:</B><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET43.1a has correctly been cut/linearized (lane 1 and 2). It was excised from the gel and extracted using Quiaquick Gel extraction kit.</LI><LI>pET43.1a+ Seq1+Seq2 cloning worked successfully on these colonies. We observed two bands as expected: one at ~ 6-7 kb and one at 1 kb. </LI></UL></P><P>We choose to do a Midiprep with sample 10 and 11 in order to have enough DNA for preparing samples for sequencing. We also transformed sample 11 in BL21(DE3) pLysS, our chassis.    

Results from Nanodrop for pET43.1a</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Concentration (ng/<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 11.3 </TD><TD>2.19</TD></TR><TR><TD> 10.8</TD><TD>2.22</TD></TR></TBODY></TABLE><P><B>Midiprep pBR322+Seq8:</B>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)

We extracted the plasmid from Col 3 and Col 4 from the cloning of pBR322+Seq8 (25 mL) from the 08.11.2018.
Results from Nanodrop:<B>For colony 3</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 41.0 </TD><TD> 1.86 </TD></TR><TR><TD> 2 </TD><TD> 2.6 </TD><TD> 1.90 </TD></TR><TR><TD> 3 </TD><TD> 38.3 </TD><TD> 1.91 </TD></TR><TR><TD> 4   </TD><TD> -0.4 </TD><TD> 0.41 </TD></TR><TR><TD> 5   </TD><TD> -1.1 </TD><TD> 0.67 </TD></TR><TR><TD> 6   </TD><TD> 1.0 </TD><TD> 1.44 </TD></TR><TR><TD> 7 </TD><TD> -0.6 </TD><TD> 1.19 </TD></TR><TR><TD> 8 </TD><TD> 27.2 </TD><TD>1.92</TD></TR></TBODY></TABLE><B>For colony 4</B><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 0.2 </TD><TD> 1.67 </TD></TR><TR><TD> 2 </TD><TD> -0.6 </TD><TD> 0.99 </TD></TR><TR><TD> 3 </TD><TD> 52.5 </TD><TD> 1.92 </TD></TR><TR><TD> 4   </TD><TD> -0.2 </TD><TD> 0.31 </TD></TR><TR><TD> 5   </TD><TD> 53.7 </TD><TD> 1.87 </TD></TR><TR><TD> 6   </TD><TD> 53.1 </TD><TD> 1.91 </TD></TR><TR><TD> 7 </TD><TD> -0.6 </TD><TD> 1.67 </TD></TR><TR><TD> 8 </TD><TD> -0.4 </TD><TD>0.52</TD></TR></TBODY></TABLE><P>Sample 1,3,8 from Col 3 and sample 3,5,6 from Col 4 are kept for future sequencing. The other samples are thrown away. However, the concentrations are too low. Samples are put in liquid culture again for new DNA extraction. 
The low concentrations can be due to a problem during the centrifugation steps, or to a too slow manipulation. </P><P><B>Bacterial transformation</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of<I> E. coli</I> BL21 (DE3) pLysS protocol)
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of <I>E. coli</I> DH5-alpha protocol)


3 transformations were processed: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>In BL21 (DE3)pLysS: pBR322 + Seq8 (Sample 11) and pET43.1a + Seq1 + Seq2 (Sample 3, Col4) </LI><LI>In DH5α: Seq4 </LI></UL></P><P><B>Bacterial stock preparation</B>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Bacterial Stock protocol)

Launch of pre-cultures of DH5α strains (5 mL LB + carbenicillin (50 <FONT face="Raleway">μ</FONT>g /mL) or 5 mL LB + chloramphenicol (25 <FONT face="Raleway">μ</FONT>g/mL)). 
Seq1, Seq2, Seq3, Seq5, Seq6, Seq7, Seq8, pET43, pBR322 and pVDL9.3</P><H2>08.14.2018</H2><P><B>Digestion of pBR322, Seq5, Seq6, Seq7</B>
Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pBR322, Seq5, Seq6, Seq7</LI><LI>Restriction enzymes : XbaI, BamHI, MscI, SphI, HindIII, SpeI, BsmI, AvaI. </LI><LI>10X Cut Smart</LI><LI>Distilled water</LI></UL></P><P><OL style="text-align: left;font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 20 min</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pBR322 (29.6 ng/<FONT face="Raleway">μ</FONT>L)</TD><TD>42</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>-</TD></TR><TR><TD>Bsml</TD><TD>1.5</TD></TR><TR><TD>Aval</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq5 (345 ng/<FONT face="Raleway">μ</FONT>L) </TD><TD>12</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>30</TD></TR><TR><TD>Nhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq6 (1572 ng/<FONT face="Raleway">μ</FONT>L) </TD><TD>6</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>36</TD></TR><TR><TD>Mscl</TD><TD>1.5</TD></TR><TR><TD>HindIII</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq7 (1572 ng/<FONT face="Raleway">μ</FONT>L) </TD><TD>36</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>6</TD></TR><TR><TD>Hhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a water bath at 80 °C for 20 min</LI></OL></P><LI>Incubation for 20 min at 37°C</LI><LI>Inactivation for 20 min at 80°C</LI><LI>Addition of 6X loading dye </LI><LI>See the deposition order on the result picture</LI><P>
We let the gel run for 10 min at 85 V and then for 80 min at 125 V.
We revealed the gel with a UV light. 
Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Agarose 0.8% gel after electrophoresis of digested pET43.1+Seq1+Seq2</FIGCAPTION></FIGURE><P><B>Observations</B>: Digestion went well for all samples. We excised the interesting bands (inserts or plasmids) form the gel and did a gel extraction. 

Results after excision:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Agarose 0.8% gel after electrophoresis of digested pET43.1+Seq1+Seq2 after excision</FIGCAPTION></FIGURE><P>Caption for fig 3:<UL style="text-align: left; list-style: none;font-size: 16px;"><LI>Wells 1-2: pBR322 cut</LI><LI>Wells 4-5: Seq5 cut</LI><LI>Wells 7-8: Seq6 cut</LI><LI>Wells 10-11: Seq7 cut</LI></UL></P><P>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)

Results from Nanodrop:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>[DNA] (ng/µL) </B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> pBR322 </TD><TD> 11.4 </TD><TD>2.49</TD></TR><TR><TD> Seq5 </TD><TD> 18.1 </TD><TD>2.05</TD></TR><TR><TD> Seq6 </TD><TD> 11.9 </TD><TD>3.10</TD></TR><TR><TD> Seq7 </TD><TD> 10.1 </TD><TD>2.14</TD></TR></TBODY></TABLE><P><B>Bacterial stock preparation</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Bacterial Stock protocol)

Culture of Seq1, Seq2, Seq3, Seq5, Seq6, Seq7, Seq8, pET43, pBR322 and pVDL9.3.
Measure the OD and when 0.6 </P><P><B>Midiprep pBR322 + Seq8 and pET43.1 + Seq1 + Seq2</B>

We extracted our plasmids from the colonies set in liquid culture yesterday. We had:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 colonies of pET43.1a + Seq1 + Seq2</LI><LI>2 colonies of pBR322 + Seq8</LI></UL></P><P>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Midiprep for plasmid extraction protocol)<B>Results</B> from NanoDrop (Blank used is buffer EB):<B>pET 43.1a + Seq1 + Seq2 Col 10</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 7.4 </TD><TD> 2.11 </TD></TR><TR><TD> 2 </TD><TD> 58.5 </TD><TD>1.91</TD></TR><TR><TD> 3 </TD><TD>0.8</TD><TD>3.57</TD></TR><TR><TD> 4   </TD><TD>3.2</TD><TD>1.26</TD></TR><TR><TD> 5   </TD><TD>64.5</TD><TD>1.90</TD></TR><TR><TD> 6   </TD><TD>2.8</TD><TD>1.72</TD></TR></TBODY></TABLE><P><B>pET 43.1a + Seq1 + Seq2 Col 11</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD>1.2</TD><TD>3.17</TD></TR><TR><TD> 2 </TD><TD>0.2</TD><TD>-0.33</TD></TR><TR><TD> 3 </TD><TD>0.1</TD><TD>-0.2</TD></TR><TR><TD> 4   </TD><TD>1.0</TD><TD>1.63</TD></TR><TR><TD> 5   </TD><TD>0.7</TD><TD>3.02</TD></TR><TR><TD> 6   </TD><TD>79.9</TD><TD>-0.38</TD></TR></TBODY></TABLE><P><B>pBR322 + Seq8 Col 3</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD>2.9</TD><TD>2.29</TD></TR><TR><TD> 2 </TD><TD>0</TD><TD>0.18</TD></TR><TR><TD> 3 </TD><TD>12.1</TD><TD>2.28</TD></TR><TR><TD> 4   </TD><TD>3.9</TD><TD>2.26</TD></TR><TR><TD> 5   </TD><TD>41.3</TD><TD>1.89</TD></TR><TR><TD> 6   </TD><TD>35.2</TD><TD>1.97</TD></TR></TBODY></TABLE><P><B>pBR322 + Seq8 Col 4</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD>0.3</TD><TD>-0.84</TD></TR><TR><TD> 2 </TD><TD>33.8</TD><TD>1.93</TD></TR><TR><TD> 3 </TD><TD>33.2</TD><TD>1.88</TD></TR><TR><TD> 4   </TD><TD>34.1</TD><TD>1.93</TD></TR><TR><TD> 5   </TD><TD>0.8</TD><TD>4.15</TD></TR><TR><TD> 6   </TD><TD>27.8</TD><TD>1.92</TD></TR></TBODY></TABLE><P><B>Observations</B>: Concentrations are not good. <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET43.1a + Seq1 + Seq2, Col 10: Samples 2 and 5 have enough concentration. </LI><LI>pBR322 + Seq8, Col 4: Samples 2,3,4,6 have enough concentration.</LI></UL></P><P><B>IN-FUSION ligation PBR322 + Seq5 + Seq6 + Seq7</B>
We used the In-Fusion cloning kit provided by Ozyme in order to clone our three inserts Seq5 and Seq6 and Seq7 together in linearized pBR322. (see above).<OL style="text-align: left;font-size: 16px;"><LI>Following the advice of the Takara Clontech technician we contacted, we first incubated the linearized vectors and inserts at 80°C for 5 min</LI><LI>We then placed it on ice for 3 minutes.</LI><LI>We mixed the following volumes into 1.5mL tubes.</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>90 ng Purified Seq5 (18.1ng/µl)</TD><TD>5</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>70 ng Purified Seq6 (14ng/µl)</TD><TD>5</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>70 ng Purified Seq7 (10.1ng/µl)</TD><TD>7</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Linearized vector</TD><TD> 10.5 </TD><TD> 1 </TD><TD>1 (control)</TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 7</TD><TD> 5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 min in a water bath at 50°C</LI><LI>Ligation See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Ligation protocol)</LI><LI>Set in ice</LI><LI>Transformation of the 3 plasmids into Stellar competent cells (provided with the cloning kit)</LI><LI>Competent cell volume : 33 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli </I>DH5-alpha protocol)<LI>Culture on petri dishes LB agar gel supplemented with carbenicillin overnight at 37°C</LI></OL></P><P><B>Liquid culture</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol) 
We cultivated 2 colonies from the transformation of DH5α with commercial plasmid containing Seq4.<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>One colony is cultivated in an Erlenmeyer in 25 mL LB medium + 25 <FONT face="Raleway">μ</FONT>L carbenicillin (50 mg/mL)</LI><LI>One colony is cultivated in two Falcon tubes, each containing 15 mL LB medium + 15 <FONT face="Raleway">μ</FONT>L carbenicillin (50 mg/mL)</LI></UL></P><P>
Since the results from this morning’s midiprep were not high enough, we cultivated again the results from cloning pBR322 + Seq8 and pET 43.1a + Seq1 + Seq2.
We cultivated:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Colony 3 of pBR322 + Seq8</LI><LI>Colony 4 of pBR322 + Seq8</LI><LI>Colony 11 of pET 43.1a + Seq1 + Seq2</LI><LI>Colony 12 of pET 43.1a + Seq1 + Seq2</LI></UL>/p&gt;
</P><P>
Each colony is cultivated in two Falcon tubes, each containing 15 mL LB medium + 15 <FONT face="Raleway">μ</FONT>L carbenicillin (50 mg/mL), in order to do a midiprep.

The bacteria are cultivated overnight in a rotative incubator at 37°C, 180 rpm.</P><H2>08.16.2018</H2><P><B>Miniprep pBR322 + Seq5,6,7</B>
We extracted our plasmids from the colonies set in liquid culture yesterday. We had 8 colonies of pBR322 + Seq5 + Seq6 + Seq7. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)

Results from NanoDrop (Blank used is buffer EB):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 117.6 </TD><TD> 2.00 </TD></TR><TR><TD> 2 </TD><TD> 122.0 </TD><TD> 2.00 </TD></TR><TR><TD> 3 </TD><TD> 152.4 </TD><TD> 1.97 </TD></TR><TR><TD> 4   </TD><TD> 148.0</TD><TD> 2.01 </TD></TR><TR><TD> 5   </TD><TD> 111.6 </TD><TD> 2.00 </TD></TR><TR><TD> 6   </TD><TD> 328.9 </TD><TD> 1.93 </TD></TR><TR><TD> 7 </TD><TD> 108 </TD><TD> 2.03 </TD></TR><TR><TD> 8 </TD><TD> 121.7 </TD><TD> 2.01 </TD></TR></TBODY></TABLE><P><B>Observations</B>: Concentrations are good, these samples can be saved for sequencing. (stored in the “SEQUENCING” box)</P><P><B>Digestion of pBR322 + Seq5,6,7 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: BamHI
Expected results: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pBR322 </B></TD><TD><B>linearized   </B></TD><TD><B>4 kB</B></TD></TR><TR><TD> pBR322 + Seq5 + Seq6 + 7 </TD><TD> 2 fragments </TD><TD>4.8 kB + 2.3 kB</TD></TR></TBODY></TABLE><P>We digested 8 samples of our clonage products, and we also did some controls: pBR322 with BamHI, pBR322 w/o BamHI, Col 8 w/o BamHI. </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>5</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>BamHI</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>20</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 15 min</LI><LI>Inactivation 80°C for 20 min.</LI><LI>Addition of 6 µL of loading dye</LI></UL></P><P><B>Electrophoresis on agarose gel PBR322 + Seq5 + Seq6 + Seq7</B>

We prepared a 1% agarose gel.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We deposited all the samples in the wells:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Wells 1 to 8: pBR322 + Seq5,6,7 (colonies 1 to 8)</LI><LI>Well 10: pBR322 + Seq5,6,7 Colony 8 w/o Enzyme</LI><LI>Well 11: pBR322</LI><LI>Well 12: pBR322 w/o enzyme</LI></UL></P><P>
We let the gel run for 10 min at 85 V and 80 min at 125 V. 
The gel was revealed with UV light.</P><P>
Digested DNA from colonies migrated to the same molecular weight as the control one. No expected fragments were visible representing a digestion of our construction with BamHI. None of the colonies contained the final construction. The cloning did not succeed.  
We performed another ligation. </P><P><B>IN-FUSION ligation PBR322 + Seq5 + Seq6 + Seq7</B>
We used the In-Fusion cloning kit provided by Ozyme in order to clone our three inserts Seq5 and Seq6 and Seq7 together in linearized pBR322. (see above).<OL style="text-align: left;font-size: 16px;"><LI>We mixed the vector pBR322 and Seq5 and Seq7 and incubate at 80°C for 5 min. Then we add Seq7 and incubate 5 min at 70°C</LI><LI>We then placed it on ice for 3 minutes</LI><LI>We add the Enzyme Mix</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Component </B></TD><TD><B>Cloning (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Negative control (<FONT face="Raleway">μ</FONT>L)</B></TD><TD><B>Positive control (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>90 ng Purified Seq5 (18.1ng/µl)</TD><TD>5</TD><TD>-</TD><TD>2 (control insert)</TD></TR><TR><TD>70 ng Purified Seq6 (14ng/µl)</TD><TD>5</TD><TD>-</TD><TD>2 (control)</TD></TR><TR><TD>70 ng Purified Seq7 (10.1ng/µl)</TD><TD>7</TD><TD>-</TD><TD>-</TD></TR><TR><TD>Linearized vector</TD><TD> 10.5 </TD><TD> 1 </TD><TD>1 (control vector)</TD></TR><TR><TD>5X In-Fusion enzyme premix</TD><TD> 4 </TD><TD> 2 </TD><TD> 2 </TD></TR><TR><TD>Deionized water</TD><TD> - </TD><TD> 7</TD><TD> 5</TD></TR></TBODY></TABLE><P><OL start="4" style="text-align: left; font-size: 16px;"><LI>Incubate for 15 min in a water bath at 50°C</LI><LI>Ligation See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Ligation protocol)</LI><LI>Set in ice</LI><LI>Transformation of the 3 plasmids into Stellar competent cells (provided with the cloning kit)</LI><LI>Competent cell volume : 33 <FONT face="Raleway">μ</FONT>L</LI><LI>DNA volume: 2 <FONT face="Raleway">μ</FONT>L</LI>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the transformation of <I>E. coli </I>DH5-alpha protocol)<LI>Culture on petri dishes LB agar gel supplemented with carbenicillin overnight at 37°C</LI></OL></P><P><B>Digestion of pET43 + Seq3 + Seq4</B>/br&gt;
<U>Aim</U>: Obtain higher concentration of DNA after the gel extraction in order to perform efficiently the ozyme IN-FUSION kit.<OL style="text-align: left; font-size: 16px;"><LI>Mix the following volumes of reagents and incubate at 37°C for 15 min</LI></OL></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pET43.1a</TD><TD>26</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>16</TD></TR><TR><TD>Ndel</TD><TD>1.5</TD></TR><TR><TD>Pacl</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>Seq3 </TD><TD>29</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>13</TD></TR><TR><TD>Nhel</TD><TD>1.5</TD></TR><TR><TD>BamHI</TD><TD>1.5</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Seq4 </TD><TD>18</TD></TR><TR><TD> 10X Cut Smart </TD><TD>5</TD></TR><TR><TD>Distilled water</TD><TD>24</TD></TR><TR><TD>Mscl</TD><TD>1.5</TD></TR><TR><TD>HindIII</TD><TD>1.5</TD></TR></TBODY></TABLE><P><OL start="2" style="text-align: left; font-size: 16px;"><LI>Inactivate the digestion by heating the mixture in a water bath at 80 °C for 20 min.</LI></OL></P><P><B>Electrophoresis on agarose gel</B>
We prepared a 0.8% agarose gel to separate the different fragments of the cut plasmids.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

We filled:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 wells with digested pET43.1a (n°3 and n°4)</LI><LI>2 wells with digested Seq3 (n°6 and n°7)</LI><LI>2 wells with digested Seq4 (n°9 and n°10)</LI></UL></P><P>
We let the DNA migrate for 8 min at 85 V on order to let the DNA penetrate the gel, and then 65 min at 125 V to efficiently separate the fragments.
We then revealed the gel under UV lamp.</P><P><B>Interpretations</B>: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>pET43.1a has not correctly been cut/linearized (lane 3 and 4). It was not excised from the gel and not extracted using Quiaquick Gel extraction kit.</LI><LI>Seq3 has correctly been cut/linearized (lane 3 and 4). It was excised from the gel and extracted using Quiaquick Gel extraction kit.</LI><LI>Seq4 has not correctly been cut/linearized (lane 3 and 4). It was not excised from the gel and not extracted using Quiaquick Gel extraction kit.</LI>/</UL></P><P>
We choose to repeat the experiment for pET43, Seq3 and Seq4 </P><H2>08.17.2018</H2><P><B>Miniprep pET 43.1a</B>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)

Results:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Colony</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 54.4 </TD><TD> 2.07 </TD></TR><TR><TD> 2 </TD><TD> 35.9 </TD><TD> 2.01 </TD></TR><TR><TD> 3 </TD><TD> 93.4 </TD><TD> 2.06 </TD></TR><TR><TD> 4   </TD><TD> 96.3</TD><TD> 2.05 </TD></TR><TR><TD> 5   </TD><TD> 92.8 </TD><TD> 2.03 </TD></TR><TR><TD> 6   </TD><TD> 80.9 </TD><TD> 1.99 </TD></TR><TR><TD> 7 </TD><TD> 59.0 </TD><TD> 2.02 </TD></TR><TR><TD> 8 </TD><TD> 53.6 </TD><TD> 2.03 </TD></TR><TR><TD> 9 </TD><TD> 52.4 </TD><TD> 1.99 </TD></TR></TBODY></TABLE><P><B>Transformation pBR322 + Seq5,6,7</B><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Colony</B></TD><TD><B>Control +</B></TD><TD><B>Control -</B></TD><TD><B>Cloning</B></TD></TR><TR><TD> CFU</TD><TD> 760 </TD><TD> 233 </TD><TD> 2 </TD></TR></TBODY></TABLE></P><P><B>Liquid cultures </B>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol) 
Culture of BL21(DE3)pLysS + pBR322+Seq8 (25 mL) + carbenicillin (50 <FONT face="Raleway">μ</FONT>g/mL)
Culture of DH5alpha + pSB1C3 (25 ml) + chloramphenicol  (25 <FONT face="Raleway">μ</FONT>g/mL)
We also did some cultures on plate. 
Culture of colonies 1 and 2 from ligation pBR322+Seq5,6,7 in 5 mL Falcon tubes</P><H2>08.18.2018</H2><P><B>Miniprep pBR322+Seq5,6,7</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
Results:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Colony</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 74 </TD><TD> 2.08 </TD></TR><TR><TD> 2 </TD><TD> 72.2 </TD><TD> 2.02 </TD></TR></TBODY></TABLE><P><B>Digestion of pBR322 + Seq5,6,7 </B><U>Aim</U>: Check that the ligation has successfully worked. <U>Restriction enzymes used</U>: BamHI, HindIII, SphI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pBR322</B></TD><TD><B>linearized   </B></TD><TD><B>4 kB</B></TD></TR><TR><TD> BamHI</TD><TD> pBR322 + Seq5 + Seq6 + Seq7</TD><TD> 2 fragments</TD><TD>4.8 kB + 2.3 kB </TD></TR><TR><TD> pET43.1a + Seq1</TD><TD> pBR322 + Seq5 + Seq6 + Seq7 </TD><TD> 2 fragments </TD><TD>3.2 kB + 4.2 kB</TD></TR><TR><TD> pET43.1a + Seq1</TD><TD> pBR322 + Seq5 + Seq6 + Seq7 </TD><TD> 2 fragments </TD><TD>3.7 kB + 3.6 kB</TD></TR></TBODY></TABLE><P>
We digested 1 samples of our cloning products, and we also did some controls pBR322 with BamHI.<B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Ndel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 20 min</LI><LI>Inactivation 80°C for 20 min</LI><LI>Addition of 6 µL of 6X loading dye in each tube</LI></UL></P><P><B>Electrophoresis on agarose gel PBR322 + Seq5 + Seq6 + Seq7</B>
We prepared a 1% agarose gel.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We let the gel run for 10 min at 85 V and 80 min at 125 V. </P><P>
Digested DNA from colonies migrated to the same molecular weight as the control one. No expected fragments are visible representing a digestion of our construction with BamHI. None of the colonies contain the final construction. The cloning did not succeed.  </P><P><B>Antibiotic test on liquid culture</B><U>Aim</U>: Evaluate the effect of an overnight culture media containing carbenicillin on a non-resistant strain. 
BL21- pBR322 + Seq8 were cultivated overnight on LB media containing carbenicillin. After centrifugation the supernatant was transferred into a new Erlenmeyer with 5 mL new LB media final volume 25 mL. 
250 µL of overnight culture of DH5alpha – pSB1C3 were added to each media a OD was monitored.<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> LB overnight culture</B></TD><TD><B>LB + CAM (chloramphenicol)   </B></TD><TD><B>LB + CARB (carbenicillin)</B></TD></TR><TR><TD> Add DH5alpha-pSB1C3 liquid culture</TD><TD> 250 µl</TD><TD> 250 µl</TD><TD>250 µl </TD></TR><TR><TD> Expected growth</TD><TD> -</TD><TD> Yes</TD><TD>No </TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> OD</B></TD><TD><B> LB overnight culture</B></TD><TD><B>LB + CAM </B></TD><TD><B>LB + CARB</B></TD></TR><TR><TD> t=0</TD><TD> 0.092</TD><TD>0.049</TD><TD>0.039</TD></TR><TR><TD> t = 30 min</TD><TD> 0.102</TD><TD> 0.079</TD><TD>0.052 </TD></TR><TR><TD> t = 1h</TD><TD> 0.108 </TD><TD> 0.128 </TD><TD> 0.098 </TD></TR><TR><TD> t = 1h30</TD><TD> 0.115 </TD><TD> 0.242 </TD><TD> 0.142 </TD></TR><TR><TD> t = 2h</TD><TD> 0.124</TD><TD> 0.390</TD><TD>0.127 </TD></TR><TR><TD> t = 2h30</TD><TD> 0.131</TD><TD>0.630</TD><TD>0.108</TD></TR><TR><TD> t = 3h</TD><TD> 0.147</TD><TD> 0.135</TD><TD>0.074 </TD></TR></TBODY></TABLE></P><P>
The antibiotic present in the overnight media did affect the growth of non-resistant bacteria.</P></DIV><DIV data-description="Week_33" class="panel" id="pan_3302" style="text-align:left;"><H2> 08.13.2018 </H2><P>  4 vertical chips bonded. Gold membrane PDMS well chips modification to enable current transmission.   </P><P> Well chip fabrication protocol (now the well chips correspond with the protocol) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Vertical chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/d/da/T--Pasteur_Paris--Vertical-chip.pdf" target="_blank"> here </A>. </P><H2> 08.16.2018 </H2><P> Conductivity measurements of  2 gold-membrane PDMS well chips. Function generator was set on sine. The physical quantities measured here are Eg, the generator's tension amplitude and Ep, the voltage difference between the extremity of the platinum wire outside the well chip and a point on the edge of the membrane of the chip. the quantity calculated here is 20*log(Ep/Eg) for different frequencies.  </P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: Gold-coated membrane conductivity for different frequencies</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Voltage difference calculated is very low, indicating a very good conductivity for the gold-coated membrane. Technically, we measured the conductivity of the system membrane+platinum wire, but we showed that the wire's conductivity could be neglected. Resistance increases in higher frequencies, because of the skin-effect in metals: it transforms into an antenna. But as we are going to use only low frequencies, this doesn't affect us.</P><P> Well chip conductivity measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_34"><DIV class="block full" id="Week_34" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_34" class="vignette" id="vign_3400"><DIV data-description="Week_34" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_34" class="vignette" id="vign_3401" style="display:none"><DIV data-description="Week_34" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_34" class="vignette" id="vign_3402"><DIV data-description="Week_34" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_34" class="panel" id="pan_3400" style="text-align:left;"><H2>08.20.2018</H2><P><B>Liquid culture BL21 – RIP AND IPTG INDUCTION</B>
Two colonies of transformed BL21 containing pBR322 and Seq8 (in order to produce RIP) are cultivated in 25 mL LB medium and 25 <FONT face="Raleway">μ</FONT>L carbenicillin.
OD is followed regularly
Results:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> OD</B></TD><TD><B> Col 3</B></TD><TD><B>Col 4</B></TD></TR><TR><TD> t=0</TD><TD> 0.004</TD><TD>0.011</TD></TR><TR><TD> t = 30 min</TD><TD> 0.010</TD><TD> 0.015</TD></TR><TR><TD> t = 60 min</TD><TD> 0.012 </TD><TD> 0.011 </TD></TR><TR><TD> t = 160 min</TD><TD> 0.019 </TD><TD> 0.029 </TD></TR><TR><TD> t = 255 min</TD><TD> 0.164</TD><TD> 0.242</TD></TR><TR><TD> t = 300 min</TD><TD> 0.426</TD><TD>0.562</TD></TR><TR><TD> t = 330 min</TD><TD> 0.809</TD><TD> 1.057</TD></TR><TR><TD> t = 355 min</TD><TD> 1.184</TD><TD> 1.369</TD></TR><TR><TD>Dilution 30 mL LB</TD><TD> Dilution 30 mL LB</TD></TR><TR><TD> t = 405 min</TD><TD> 1.285</TD><TD> 1.491</TD></TR><TR><TD> t = 420 min</TD><TD> 1.514</TD><TD> 1.732</TD></TR></TBODY></TABLE><P>
When the DO reaches 1.5 (and the bacteria are not in an exponential growth phase any more), we induced RIP production by adding IPTG.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the IPTG Induction of protein expression protocol)

Different IPTG concentrations are tested for induction:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>0.05 mM</LI><LI>0.1 mM</LI><LI>0.2 mM</LI><LI>0.5 mM</LI></UL></P><P>
DO600 is measured every 30 minutes (the sample is diluted 1/10 before each measure)
Results:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Time after induction</B></TD><TD><B> 0.05 mM</B></TD><TD><B>0.1 mM</B></TD><TD><B> 0.2 mM</B></TD><TD><B> 0.5 mM</B></TD></TR><TR><TD> t=0 min </TD><TD> 1.51</TD><TD>1.51</TD><TD>1.73</TD><TD>1.73</TD></TR><TR><TD> t = 30 min</TD><TD> 1.754</TD><TD> 1.746</TD><TD>1.874 </TD><TD>1.879 </TD></TR><TR><TD> t = 60 min</TD><TD> 2.118 </TD><TD> 2.219 </TD><TD> 2.346 </TD><TD> 2.265 </TD></TR><TR><TD> t = 90 min</TD><TD> 2.128 </TD><TD> 2.330 </TD><TD> 2.504 </TD><TD> 2.468 </TD></TR><TR><TD> t = 120 min </TD><TD> 2.552</TD><TD> 2.025</TD><TD>2.630 </TD><TD>2.493 </TD></TR><TR><TD>t = 180 min</TD><TD> 2.962</TD><TD>2.845</TD><TD>2.868</TD><TD>2.947</TD></TR></TBODY></TABLE><P>
At t = 60 min et t = 120 min, 1 mL culture is taken, centrifuged and pellet and supernatant are conserved separately at -20°C, for further use.</P><P>Liquid culture of <I>S. aureus</I>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)
Liquid culture of <I>S. aureus</I> in 35 mL LB medium and 35 <FONT face="Raleway">μ</FONT>L chloramphenicol.
We cultivated overnight in 37°C incubator.</P><P><B>Liquid culture</B> 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)
Liquid culture of DH5alpha- pSB1C3 and DH5alpha- pBR322 for plasmid stock.</P><H2>08.21.2018</H2><P><B>Digestion of pSB1C3 and sequences from EUROFINS GENOMICS (1 TO 8)</B>

Reagents:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Seq1 (conc. 493,7 ng/<FONT face="Raleway">μ</FONT>L) : EcoRI and BamHI</LI><LI>Seq2 (conc. 463,8 ng/<FONT face="Raleway">μ</FONT>L): MscI and PstI</LI><LI>Seq3 (conc. 179 ng/<FONT face="Raleway">μ</FONT>L): EcoRI and BamHI</LI><LI>Seq4 (conc. 354 ng/<FONT face="Raleway">μ</FONT>L): MscI and PstI</LI><LI>Seq5 (conc. 353 ng/<FONT face="Raleway">μ</FONT>L): EcoRI and BamHI</LI><LI>Seq6 (conc. 516 ng/<FONT face="Raleway">μ</FONT>L): MscI and SphI</LI><LI>Seq7 (conc. 305 ng/<FONT face="Raleway">μ</FONT>L): HindIII and PstI</LI><LI>Seq8 (conc. 325 ng/<FONT face="Raleway">μ</FONT>L): EcoRI and PstI</LI><LI>pSB1C3 from 2016 team (conc. 40 ng/<FONT face="Raleway">μ</FONT>L) : XbaI and SpeI</LI><LI>linearized iGEM plasmid pSB1C3 (conc. 25 ng/<FONT face="Raleway">μ</FONT>L): XbaI and SpeI</LI></UL></P><P>
Each digestion was performed in a total volume of 50 <FONT face="Raleway">μ</FONT>L, containing 1500ng insert, 1.5 <FONT face="Raleway">μ</FONT>L of each enzyme and 5 <FONT face="Raleway">μ</FONT>L of CutSmart buffer. 
Digestion was performed at 37°C for :<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>20 minutes for tubes without MscI</LI><LI>60 minutes for tubes containing MscI</LI></UL></P><P>
Digestion was then inactivated at 80°C for 20 minutes.</P><P><B>Electrophoresis on agarOSE gel</B>
A 0.8 % agar gel was prepared.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
10 <FONT face="Raleway">μ</FONT>L loading dye is added to each sample and each sample is put in two wells.
Results first gel:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Picture of the 3 petri dishes at t = 0. </FIGCAPTION></FIGURE><P>Results after excision: </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Picture of the 3 petri dishes at t = 0. </FIGCAPTION></FIGURE><P>Results second gel: a second gel was run with digested Seq3, Seq4, Seq5, Seq6 and Seq7.
Migration for 10 minutes at 85V and then 1 hour at 125V.</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Picture of the 3 petri dishes at t = 0. </FIGCAPTION></FIGURE><P><B>Interpretations</B>: all commercial plasmids have been successfully digested, except Seq4, for which the band corresponding to the insert we look for is unclear.</P><P><B>Gel extraction</B>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)
Results from Nanodrop:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> Seq1 (cut)</TD><TD> 31.6 </TD><TD> 2.00 </TD></TR><TR><TD> Seq2 (cut) </TD><TD> 23.9 </TD><TD> 1.92 </TD></TR><TR><TD> Seq3 (cut)</TD><TD> 16.8 </TD><TD> 2.00 </TD></TR><TR><TD> Seq4 (cut)  </TD><TD> 3.3</TD><TD> 3.11 </TD></TR><TR><TD> Seq5 (cut)  </TD><TD> 32.1 </TD><TD> 1.98 </TD></TR><TR><TD> Seq6 (cut)  </TD><TD> 25.8 </TD><TD> 1.91 </TD></TR><TR><TD> Seq7 (cut) </TD><TD> 24 </TD><TD> 1.97 </TD></TR><TR><TD> Seq8 (cut) </TD><TD> 10.8 </TD><TD> 2.25 </TD></TR><TR><TD> pSB1C3 2016</TD><TD> 7.6 </TD><TD> 2.7 </TD></TR><TR><TD> pSB1C3 iGEM</TD><TD> 9.0 </TD><TD> 2.07 </TD></TR></TBODY></TABLE><P><B>IN FUSION kit for pSB1C3 + Seq1 +Seq2 and pSB1C3 + Seq8</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Ligation protocol)
We used Ozyme cloning kit to insert Seq8 and Seq1 and 2 in iGEM plasmid.
Results: the volumes we mixed for the reaction<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> ng</B></TD><TD><B>ng/<FONT face="Raleway">μ</FONT>L</B></TD><TD><B><FONT face="Raleway">μ</FONT>L</B></TD></TR><TR><TD> pSB1C3 </TD><TD> 120</TD><TD>7.6</TD><TD>15</TD></TR><TR><TD>Seq1</TD><TD> 85</TD><TD> 31.6</TD><TD>3 </TD></TR><TR><TD>Seq2</TD><TD> 85 </TD><TD> 23.9 </TD><TD> 3.5 </TD></TR><TR><TD>Enzyme mix</TD><TD> - </TD><TD> - </TD><TD> 4 </TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> ng</B></TD><TD><B>ng/<FONT face="Raleway">μ</FONT>L</B></TD><TD><B><FONT face="Raleway">μ</FONT>L</B></TD></TR><TR><TD> pSB1C3 </TD><TD> 150</TD><TD>7.6</TD><TD>19</TD></TR><TR><TD>Seq8</TD><TD> 70</TD><TD> 15.7</TD><TD>5 </TD></TR><TR><TD>Enzyme mix</TD><TD> - </TD><TD> - </TD><TD> 4 </TD></TR></TBODY></TABLE></P><P> 
Plasmids and inserts were mixed together and heated at 80°C for 5 min, and then set on ice for 3 min. Then the enzyme mix was added. 
Samples were heated at 50°C for 15 min and then set on ice for 5 min. </P><P><B>Transformation in stellar competent cells</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of <I>E. coli</I> DH5-alpha protocol)</P><P><B>Miniprep pbr322 + pSB1C3</B>

Results from NanoDrop : Stock plasmid pSB1C3</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 161 </TD><TD> 1.95 </TD></TR><TR><TD> 2 </TD><TD> 143.9 </TD><TD> 1.95 </TD></TR><TR><TD> 3 </TD><TD> 129.1 </TD><TD> 1.95 </TD></TR><TR><TD> 4   </TD><TD> 265.3</TD><TD> 1.92 </TD></TR><TR><TD> 5   </TD><TD> 290.7 </TD><TD> 1.97 </TD></TR><TR><TD> 6   </TD><TD> 156.3 </TD><TD> 1.96 </TD></TR></TBODY></TABLE><P>
Results from NanoDrop : Stock plasmid pBR322</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 139.4 </TD><TD> 1.98 </TD></TR><TR><TD> 2 </TD><TD> 221.0 </TD><TD> 1.92 </TD></TR><TR><TD> 3 </TD><TD> 139.8 </TD><TD> 1.94 </TD></TR><TR><TD> 4   </TD><TD> 138.4</TD><TD> 1.99 </TD></TR><TR><TD> 5   </TD><TD> 129.1 </TD><TD> 1.95 </TD></TR><TR><TD> 6   </TD><TD> 120.1 </TD><TD> 1.95 </TD></TR></TBODY></TABLE><P><B>Miniprep Cloning pBR322 + Seq5,6,7</B>
Results from NanoDrop :</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 93.7 </TD><TD> 2.00 </TD></TR><TR><TD> 2 </TD><TD> 89.7 </TD><TD> 1.96 </TD></TR><TR><TD> 3 </TD><TD> 93.2 </TD><TD> 1.98 </TD></TR><TR><TD> 4   </TD><TD> 502.4</TD><TD> 1.89 </TD></TR><TR><TD> 5   </TD><TD> 98.9 </TD><TD> 2.00 </TD></TR><TR><TD> 6   </TD><TD> 105.5 </TD><TD> 1.96 </TD></TR><TR><TD> 7 </TD><TD> 90.5 </TD><TD> 1.98 </TD></TR><TR><TD> 8   </TD><TD> 111.5 </TD><TD> 1.96 </TD></TR><TR><TD> 9   </TD><TD> 238 </TD><TD> 1.93 </TD></TR><TR><TD> 10  </TD><TD> 100.2 </TD><TD> 2.00 </TD></TR></TBODY></TABLE><P><B>Digestion of pBR322 + Seq5,6,7 </B><U>Aim</U>: Check that the ligation has successfully worked on 10 other colonies. <U>Restriction enzymes used</U>: BamHI<B>Expected results</B>: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pBR322</B></TD><TD><B>linearized   </B></TD><TD><B>4 kB</B></TD></TR><TR><TD> BamHI</TD><TD> pBR322 + Seq5 + Seq6 + Seq7</TD><TD> 2 fragments</TD><TD>4.8 kB + 2.3 kB </TD></TR></TBODY></TABLE><P>
We digested 10 colonies of our clonage products, and we also did some controls: pBR322 with BamHI and without. <B>Procedure: </B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Ndel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Digestion at 37°C for 20 min</LI><LI>Addition of 6 µL of loading dye in each tube</LI></UL></P><P><B>Electrophoresis on agarose gel PBR322 + Seq5 + Seq6 + Seq7</B>
We prepared a 1% agar gel.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
We let the gel run for 10 min at 85 V and 80min at 125 V. <B>Observations</B>: Digested DNA from colonies migrated to the same molecular weight as the control one. No expecting fragments are visible representing a digestion of our construction with BamHI. None of the colonies contain the final construction. The cloning did not succeed.  </P><H2>08.22.2018</H2><P><B>Results from cloning of pSB1C3 + Seq1 + Seq2 and pSB1C3 + Seq8</B>
The two bacteria transformed with cloning results did grow on the agar plates. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)
10 colonies of each plate were placed in liquid culture and grew overnight.</P><P><B>Digestion pSB1C3 and pBR322</B></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pSB1C3 </TD><TD>10.3</TD></TR><TR><TD> Cut Smart </TD><TD>5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>28.7</TD></TR><TR><TD>Xbal</TD><TD>3</TD></TR><TR><TD>Spel</TD><TD>3</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> pBR322 </TD><TD>13.6</TD></TR><TR><TD> Cut Smart </TD><TD>5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>25.4</TD></TR><TR><TD>Xbal</TD><TD>3</TD></TR><TR><TD>Spel</TD><TD>3</TD></TR></TBODY></TABLE><P>We let the samples digest for 30 minutes. </P><P><B>BL21 – NGF culture and protein induction</B>
Yesterday’s preculture was centrifuged at 4500 x g for 15 minutes. Supernatant was discarded and pellet was washed with 5 mL fresh sterile LB medium. 
We centrifuged again for 10 minutes at 4500 x g and supernatant was discarded.
Pellet was resuspended in 10 mL LB medium + 10 <FONT face="Raleway">μ</FONT>L carbenicillin and 10 <FONT face="Raleway">μ</FONT>L chloramphenicol.
From this suspension, three liquid culture were launched in three 250 mL sterile Erlenmeyer.
Each Erlenmeyer contains:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>35 mL LB medium</LI><LI>35 <FONT face="Raleway">μ</FONT>L carbenicillin</LI><LI>35 <FONT face="Raleway">μ</FONT>L chloramphenicol</LI><LI>1 mL resuspended pellet</LI></UL></P><P>
The Erlenmeyers are incubated in a shaking incubator at 37 °C and DO is followed to induce protein expression when bacteria are in exponential growth phase.
At t = 80 min, when OD reaches 0.7, synthesis of the secretion system and NGF is induced by adding IPTG.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the IPTG Induction of protein expression protocol)

Different IPTG concentrations are tested:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Erlen 1 : 0.1 mM</LI><LI>Erlen 2 : 0.5 mM</LI><LI>Erlen 3 : 1 mM </LI></UL></P><P>
The Erlenmeyer are incubated again, DO is measured every hour for two hours.
Two hours after induction, cultures are stopped and 1 mL of each culture is pipetted and put in a 1.5 mL Eppendorf tube.
Tubes are centrifuged, and pellet and supernatant are conserved separately.</P><P><B>Analysis of protein expression</B>
We analyzed the protein present in the supernatant and in the cytoplasm of induced cells by SDS-Page gel.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the SDS-PAGE protocol)
The wells were filled as following:<OL style="text-align: left;font-size: 16px;"><LI> Culture medium before induction</LI><LI>Uninduced supernatant</LI><LI>Uninduced pellet</LI><LI>Culture medium after 0.1 mM induction</LI><LI>Supernatant after 0.1 mM induction</LI><LI> Pellet after 0.1 mM induction</LI><LI> Culture medium after 0.5 mM induction</LI><LI> Supernatant after 0.5 mM induction</LI><LI> Pellet after 0.5 mM induction</LI></OL></P><H2>08.23.2018</H2><P><B>Miniprep PSB1C3+ Seq8 and pSB1C3 + Seq1,2</B><U>Aim</U>: extract plasmid from the colonies put in liquid culture yesterday
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)<B>Results</B> from NanoDrop : plasmid pSB1C3 + Seq8</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 100.6 </TD><TD> 2.07 </TD></TR><TR><TD> 2 </TD><TD> 60 </TD><TD> 2.17 </TD></TR><TR><TD> 3 </TD><TD> 57 </TD><TD> 2.14 </TD></TR><TR><TD> 4   </TD><TD> 57.2</TD><TD> 2.10 </TD></TR><TR><TD> 5   </TD><TD> 55 </TD><TD> 2.16 </TD></TR><TR><TD> 6   </TD><TD> 52.3 </TD><TD> 2.19 </TD></TR><TR><TD> 7 </TD><TD> 45.4 </TD><TD> 2.16 </TD></TR><TR><TD> 8   </TD><TD> 38.8 </TD><TD> 2.25 </TD></TR><TR><TD> 9   </TD><TD> 56.5 </TD><TD> 2.11 </TD></TR><TR><TD> 10  </TD><TD> 61.5 </TD><TD> 2.09 </TD></TR></TBODY></TABLE><P><B>Results</B> from NanoDrop : plasmid pSB1C3 + Seq1+2</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 40.6 </TD><TD> 1.99 </TD></TR><TR><TD> 2 </TD><TD> 49.6 </TD><TD> 1.99 </TD></TR><TR><TD> 3 </TD><TD> 71.5 </TD><TD> 1.95 </TD></TR><TR><TD> 4   </TD><TD> 63.7</TD><TD> 1.99 </TD></TR><TR><TD> 5   </TD><TD> 55.6 </TD><TD> 2.03 </TD></TR><TR><TD> 6   </TD><TD> 46.8 </TD><TD> 1.98 </TD></TR><TR><TD> 7 </TD><TD> 86.6 </TD><TD> 1.96 </TD></TR><TR><TD> 8   </TD><TD> 74.9 </TD><TD> 1.93 </TD></TR><TR><TD> 9   </TD><TD> 57.1 </TD><TD> 1.97 </TD></TR></TBODY></TABLE><P><B>Enzymatic digestion of the extracted plasmids </B><U>Aim</U>: Verify that the cloning has correctly work for pSB1C3 + Seq8 and pSB1C3 + Seq1 + Seq2. 
The following volumes have been mixed:
For pSB1C3 + Seq8: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>18</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR></TBODY></TABLE><P>For pSB1C3 + Seq1 + Seq2: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>18</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR></TBODY></TABLE><P><B>Gel electrophoresis of psb1c3 + Seq8 </B><U>Digestion enzymes</U>: EcoRI/PstI
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

Expected results: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pSB1C3 </B></TD><TD><B>1 fragment</B></TD><TD><B>2070 bp</B></TD></TR><TR><TD> pSB1C3 + Seq8 </TD><TD> 2 fragments </TD><TD>2000 bp + 500 bp</TD></TR></TBODY></TABLE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 4: </B>Gel electrophoresis of psb1c3 + Seq8  </FIGCAPTION></FIGURE><P><B>Observations</B>: Colony 2, 4, 5, 6, 8, 9, 10 acquired the plasmid and the insert. 
BL21 cells were transformed with the plasmid form colony 2 and DH5α cells were transformed with plasmids from colony 2 and 4. 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of <I>E. coli</I> DH5-alpha protocol)</P><P><B>Gel electrophoresis of pSB1C3 + Seq1,2</B><U>Digestion enzymes</U>: EcoRI/PstI
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

Expected results: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pSB1C3 </B></TD><TD><B>1 fragment</B></TD><TD><B>2070 bp</B></TD></TR><TR><TD> pSB1C3 + Seq8 </TD><TD> 2 fragments </TD><TD>2000 bp + 2260 bp</TD></TR></TBODY></TABLE><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 5: </B>Agarose 1% gel after electrophoresis of digested psb1c3 + Seq1 + Seq2  </FIGCAPTION></FIGURE><P><B>Observations</B>: Colony 3, 7 and 8 did acquire the plasmid and the insert. </P><P><B>Antibiotic test on liquid culture</B><U>Aim</U>: Evaluate the effect of an overnight culture media containing AMP on a non-resistant strain.<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Culture of BL21 + pBR322 + Seq8 (25 ml) + ampicillin (10 µg/mL) </LI><LI>Culture of DH5alpha + pSB1C3 (5 ml) + chloramphenicol (50 µg/mL)</LI><LI>Culture of DH5alpha + pSB1C3 (20 ml) + ampicillin (10 µg/mL) </LI></UL></P><P>
Effect of AMP (10 µg/mL) on non-resistant cells: <UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Overnight BL21 pBR322 + Seq8 AMP (10 µg/ml): OD = 5.78</LI><LI>Overnight DH5a pSB1C3  AMP( 10 µg/ml): OD = 0.02</LI></UL></P><P>
At 10 µg/ml Amp do not allow the growth of non-resistant bacteria. 

After centrifugation the supernatant of Culture of BL21 + pBR322+Seq8 (25 ml) + AMP (50 µg/ml and 10 µg/ml) was transferred into a new erlenmeyer with 5ml new LB media final volume 25ml.`

1/10 of overnight culture of DH5alpha – pSB1C3 were added to each media a OD was monitored.  
Results:
<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> LB overnight culture AMP (50 µg/ml)</B></TD><TD><B> LB overnight culture AMP (10 µg/ml)</B></TD><TD><B>LB+ AMP (50 µg/ml) - 20 ml</B></TD><TD><B>LB+ AMP (50 µg/ml) - 10 ml</B></TD><TD><B>LB + CARB - 20 mL</B></TD></TR><TR><TD> Add DH5alpha-pSB1C3 liquid culture</TD><TD> 250 µl</TD><TD> 250 µl</TD><TD>200 µl </TD><TD>200 µl </TD><TD>200 µl </TD></TR><TR><TD> Expected growth</TD><TD> -</TD><TD> -</TD><TD> No</TD><TD>No </TD><TD>Yes </TD></TR></TBODY></TABLE></P><P>Results:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> OD</B></TD><TD><B> LB overnight culture AMP (50 µg/ml)</B></TD><TD><B> LB overnight culture AMP (10 µg/ml)</B></TD><TD><B>LB+ AMP (50 µg/ml) - 20 ml</B></TD><TD><B>LB+ AMP (50 µg/ml) - 10 ml</B></TD><TD><B>LB + CARB - 20 mL</B></TD></TR><TR><TD> t=1h</TD><TD> 0.116</TD><TD>0.094</TD><TD>0.082</TD><TD>0.091</TD><TD>0.087</TD></TR><TR><TD> t = 2h</TD><TD> 0.2</TD><TD> 0.172</TD><TD>0.043 </TD><TD> 0.231</TD><TD>0.258 </TD></TR><TR><TD> t = 3h</TD><TD> 0.272 </TD><TD> 0.231</TD><TD> 0.034 </TD><TD> 0.317 </TD><TD> 0.839 </TD></TR><TR><TD> t = 4h</TD><TD> 0.342 </TD><TD> 0.307 </TD><TD> 0.023 </TD><TD> 0.137 </TD><TD> 0.839 </TD></TR><TR><TD> t = 5h</TD><TD> 0.369</TD><TD> 0.349</TD><TD>0.021 </TD><TD> 0.092 </TD><TD> 1.33 </TD></TR><TR><TD> t = 6h</TD><TD> 0.426</TD><TD>0.394</TD><TD> 0.014 </TD><TD> 0.079 </TD><TD> 1.54 </TD></TR><TR><TD> t = 3h</TD><TD> 0.467</TD><TD> 0.477</TD><TD>0.017 </TD><TD> 0.063 </TD><TD> 1.71 </TD></TR></TBODY></TABLE><P>
The antibiotic present in the overnight media did affect the growth of non-resistant bacteria.</P><P><B>Liquid culture BL21 – NGF </B>
Two colonies of BL21 transformed with our plasmid producing NGF were cultivated in 5 mL LB medium + 5 <FONT face="Raleway">μ</FONT>L carbenicilline and 5 <FONT face="Raleway">μ</FONT>L chloramphenicol.
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)</P><H2>08.24.2018</H2><P><B>Miniprep PSB1C3+ Seq5,6,7 and Seq4</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)

Results from NanoDrop: pSB1C3 + Seq5,6,7</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 352.9 </TD><TD> 1.93 </TD></TR><TR><TD> 2 </TD><TD> 308.4 </TD><TD> 1.93 </TD></TR><TR><TD> 3 </TD><TD> 336 </TD><TD> 1.93 </TD></TR><TR><TD> 4   </TD><TD> 325.1 </TD><TD> 1.93 </TD></TR></TBODY></TABLE><P>
Results from NanoDrop: pSB1C3 + Seq4:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 95.8 </TD><TD> 2.05 </TD></TR><TR><TD> 2 </TD><TD> 104.8 </TD><TD> 1.99 </TD></TR><TR><TD> 3 </TD><TD> 87.9 </TD><TD> 2.01 </TD></TR><TR><TD> 4   </TD><TD> 101 </TD><TD> 1.95 </TD></TR></TBODY></TABLE><P><B>Enzymatic digestion of PSB1C3 + Seq5,6,7</B><U>Aim</U>: Verify that the IN-FUSION kit has correctly work for pSB1C3 + Seq5,6,7: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>18</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR></TBODY></TABLE><P>
To verify that Seq6 is in the ligation:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>18</TD></TR><TR><TD>Sphl</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR></TBODY></TABLE><P>
Digestion of Seq4: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA (325 ng/µl) </TD><TD>30</TD></TR><TR><TD> Cut Smart </TD><TD>5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>12</TD></TR><TR><TD>Mscl</TD><TD>1</TD></TR><TR><TD>HindIII</TD><TD>1</TD></TR></TBODY></TABLE><P><B>Gel electrophoresis of pSB1C3 + Seq5,6,7 and Seq4</B><U>Digestion enzymes</U>: EcoRI/PstI and SphI/PstI
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)

Expected results: </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> pSB1C3 </B></TD><TD><B>1 fragment</B></TD><TD><B>2070 bp</B></TD></TR><TR><TD> EcoRI/PstI: pSB1C3 + Seq5,6,7 </TD><TD> 2 fragments </TD><TD>2070 bp + 3144 bp</TD></TR><TR><TD><B> EcoRI/PstI: pSB1C3 + Seq5,6,7 </B></TD><TD><B>2 fragments</B></TD><TD><B>4500 bp + 762 bp</B></TD></TR><TR><TD> Seq4 </TD><TD> 2 fragments </TD><TD>2070 bp + 853 bp</TD></TR></TBODY></TABLE><P>
Results :
<FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 6: </B>Gel electrophoresis of pSB1C3 + Seq5,6,7 and Seq4 </FIGCAPTION></FIGURE><B>Gel electrophoresis of pSB1C3 + Seq5,6,7 and Seq4</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the IPTG Induction of protein expression protocol)
From yesterday’s pre-culture, 800 mL culture of BL21 producing NGF was induced at 0.1 mM for NGF analysis.</P><P><B><I>S. aureus</I> biofilm formation</B><I>S. aureus</I> alone was plated in a 96-well plate in different concentrations in order to test different methods to wash the wells.
Plate is incubated at room temperature over the week end.</P></DIV><DIV data-description="Week_34" class="panel" id="pan_3402" style="text-align:left;"><H2> 08.21.2018 </H2><P> Scanning electron microscopy (FEI Magellan 400 microscope) of a gold membrane. </P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: Gold-coated membrane</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Membrane imaging confirmed the presence of 0.4 micrometer pores on the surface, enough to confine bacteria like <I>E. coli</I>. </P><H2>  08.22.2018  </H2><P>   8 microchannel chips and 1 set of PDMS pieces for the well chip have been made.  </P><P> Microchannel chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><P> Well chip fabrication protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><H2>  08.23.2018  </H2><P>   PEDOT:PSS coated on alumina oxide membranes.  </P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 2: White alumina oxide membranes  before coating</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION>Figure 3: PEDOT:PSS coated  alumina oxide membranes</FIGCAPTION></FIGURE><P><B>Interpretation </B></P><P> The color of the alumina oxide membranes changed radically from light grey to black, suggesting the deposit of PEDOT:PSS on the membrane, as expected. Because of the lack of time and equipement, we won't try to verify that it is indeed PEDOT:PSS, however we are going to try to use electronic microscopy to visually evaluate the coating. </P><P> PEDOT:PSS coating protocol <A href="https://static.igem.org/mediawiki/2018/a/a9/T--Pasteur_Paris--Microfluidics-membranes.pdf" target="_blank"> here </A>. </P><H2>   08.24.2018 </H2><P>   PEDOT:Ts and PEDOT:Cl coated on alumina oxide membranes. </P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 4: PEDOT:Ts coated membranes</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION>Figure 5: PEDOT:Cl coated membranes</FIGCAPTION></FIGURE><P><B>Interpretation </B></P><P> The color of the alumina oxide membranes changed radically from light grey to black with green shades for PEDOT:Cl and blue shades for PEDOT:Ts, suggesting the deposit of the polymers on the membranes, as expected. Because of the lack of time and equipement, we won't try to verify that it is indeed PEDOT:Ts and PEDOT:Cl, however we are going to try to use electronic microscopy to visually evaluate the coating. </P><P> PEDOT:Cl and PEDOT:Ts coating protocol <A href="https://static.igem.org/mediawiki/2018/a/a9/T--Pasteur_Paris--Microfluidics-membranes.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_35"><DIV class="block full" id="Week_35" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_35" class="vignette" id="vign_3500"><DIV data-description="Week_35" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_35" class="vignette" id="vign_3501"><DIV data-description="Week_35" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_35" class="vignette" id="vign_3502"><DIV data-description="Week_35" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_35" class="panel" id="pan_3500" style="text-align:left;"><H2>08.27.2018</H2><P><B>Miniprep kill skitch</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
Results from NanoDrop: Kill Switch </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 661 </TD><TD> 1.86 </TD></TR><TR><TD> 2 </TD><TD> 889 </TD><TD> 1.87 </TD></TR><TR><TD> 3 </TD><TD> 665 </TD><TD> 1.87 </TD></TR><TR><TD> 4   </TD><TD> 1424 </TD><TD> 1.86 </TD></TR><TR><TD> 5   </TD><TD> 969 </TD><TD> 1.87 </TD></TR><TR><TD> 6   </TD><TD> 1093 </TD><TD> 1.87 </TD></TR></TBODY></TABLE><P><B>Digestion of pSB1C3 + Seq5,6,7</B><U>Aim</U>: Verify that the IN-FUSION kit has correctly work for pSB1C3 + Seq5,6,7:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Nhel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><B>Electrophoresis on agarose gel of pSB1C3 + Seq5,6,7</B><U>Digestion enzymes</U>: NheI 
(See<A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Agarose gel of  pSB1C3 + Seq5,6,7</FIGCAPTION></FIGURE><P><B>Interpretation</B>: wells 1, 2 and 4 have the expected fragments. The cloning did work for the three inserts. </P><P><B>Transformation</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of <I>E. coli</I> DH5-alpha protocol)
 
BL21(DE3) pLysS cells were transformed with the plasmid (pSB1C3 + Seq5,6,7) from colony 2 and DH5α cells were transformed with plasmids (pSB1C3 + Seq5,6,7 and pSB1C3 + Seq1,2) from colony 2 and 7. </P><P><B>Digestion Ks and pSB1C3</B><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> Ks (665ng/ <FONT face="Raleway">μ</FONT>L) </TD><TD>9.3</TD></TR><TR><TD> Cut Smart </TD><TD>5</TD></TR><TR><TD>EcoRI</TD><TD>1.5</TD></TR><TR><TD>PstI</TD><TD>1.5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>42</TD></TR></TBODY></TABLE><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD>pSB1C3 (156ng/ <FONT face="Raleway">μ</FONT>L)</TD><TD>16</TD></TR><TR><TD> Cut Smart </TD><TD>5</TD></TR><TR><TD>Xbal</TD><TD>1.5</TD></TR><TR><TD>Spel</TD><TD>1.5</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>26</TD></TR></TBODY></TABLE></P><P><B>Liquid culture</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)
DH5alpha + pSB1C3 + Seq8</P><P><B>Biofilm formation test</B>
Two colonies of BL21(DE3) pLysS cells that were transformed with pSB1C3+Seq8 were put in liquid culture in 30 mL LB medium with 30 <FONT face="Raleway">μ</FONT>L chloramphenicol (25 mg/mL). 
Two colonies of <I>S. aureus</I> were also put in liquid culture in 30 mL LB medium with 30 <FONT face="Raleway">μ</FONT>L chloramphenicol (25 mg/mL).
Results (DO is followed during the culture):</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> BL21 + pSB1C3 RIP</B></TD><TD><B><I>S. aureus</I></B></TD></TR><TR><TD> t=0</TD><TD> 0</TD><TD>0</TD></TR><TR><TD> t = 90 min</TD><TD> 0</TD><TD> 0</TD></TR><TR><TD> t = 180 min</TD><TD> 0.002 </TD><TD> 0.008 </TD></TR><TR><TD> t = 240 min</TD><TD> 0.002 </TD><TD> 0.023 </TD></TR><TR><TD> t = 300 min</TD><TD> 0.025</TD><TD> 0.076</TD></TR><TR><TD> t = 360 min</TD><TD> 0.120 </TD><TD> 0.281 </TD></TR><TR><TD> t = 430 min</TD><TD> 0.416 </TD><TD> 0.731 </TD></TR><TR><TD> t = 550 min</TD><TD> cf. below </TD><TD> 1.873 </TD></TR></TBODY></TABLE><P>When BL21(DE3) pLysS cells are in exponential growth phase (OD600 nm =0.416), the culture is separated into four 5 mL cultures in 50 mL Erlenmeyer. RIP production is induced with different concentrations of IPTG:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>One is not induced (negative control)</LI><LI>One is induced with 0.1 mM</LI><LI>One is induced with 0.5 mM</LI><LI>One is induced with 1 mM</LI></UL></P><P>
After two hours of induction (i.e. 550 minutes in the protocol above), OD600 nm is measured again:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> uninduced</B></TD><TD><B> IPTG 0.1 mM</B></TD><TD><B> IPTG 0.5 mM</B></TD><TD><B> IPTG 1 mM</B></TD></TR><TR><TD> OD<SUB>600 nm</SUB>/td&gt;
                                    </TD><TD> 2.495 </TD><TD> 2.464 </TD><TD>2.417 </TD><TD>2.337 </TD></TR></TBODY></TABLE><P>
1 mL of each culture is kept intact in a 1.5 mL Eppendorf tube and put at 4°C waiting for further treatment.
The 4 mL left are centrifuged, supernatant is collected.<I>S. aureus</I> is then grown overnight in a 96-well plate in different conditions in order to test the effect of both supernatant and our bacteria.
Each well contains 100 <FONT face="Raleway">μ</FONT>L of the following mix:<OL style="text-align: left;font-size: 16px;"><LI>A-B LB medium alone</LI></OL></P><OL style="text-align: left;font-size: 16px;"><P><LI>E-H 990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB un-induced supernatant + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB 0.1 mM-induced supernatant + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB 0.5 mM-induced supernatant + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB 1 mM-induced supernatant + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB + Cm + 100 <FONT face="Raleway">μ</FONT>L uninduced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB + Cm + 100 <FONT face="Raleway">μ</FONT>L 0.1 mM induced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB + Cm + 100 <FONT face="Raleway">μ</FONT>L 0.5 mM induced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB + Cm + 100 <FONT face="Raleway">μ</FONT>L 1 mM induced BL21(DE3)pLysS + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I>s culture</LI></P></OL><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B> Guild plate. </FIGCAPTION></FIGURE><P><B>800 mL culture NGF purification on a Nickel-NTA column</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Purification of his-tagged protein from culture medium protocol)
After purification, 10 fractions of 1 mL were collected from the column elution.</P><P><B>SDS-PAGE of the collected fractions &amp; Western blot for NGF production analysis</B>
This SDS-PAGE didn’t give satisfying results. There was a misunderstanding of the purification protocol as we had let the charging of the column go on over the weekend, which led to the emptying of the nickel from the column.</P><H2>08.28.2018</H2><P><B>Miniprep DH5alpha pSB1C3 Seq8 colony 2 and 4</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
Results from NanoDrop: DH5alpha pSB1C3 Seq8 colony 2 
<TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 210.7 </TD><TD> 1.97 </TD></TR><TR><TD> 2 </TD><TD> 234.9 </TD><TD> 1.94 </TD></TR><TR><TD> 3 </TD><TD> 201.2 </TD><TD> 1.94 </TD></TR><TR><TD> 4   </TD><TD> 210.7 </TD><TD> 1.95 </TD></TR><TR><TD> 5   </TD><TD> 217.1 </TD><TD> 1.93 </TD></TR></TBODY></TABLE></P><P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 222.7 </TD><TD> 1.95 </TD></TR><TR><TD> 2 </TD><TD> 264.4 </TD><TD> 1.95 </TD></TR><TR><TD> 3 </TD><TD> 224.2 </TD><TD> 1.95 </TD></TR><TR><TD> 4   </TD><TD> 212.4 </TD><TD> 1.96 </TD></TR><TR><TD> 5 </TD><TD> 225.3 </TD><TD> 1.94 </TD></TR></TBODY></TABLE></P><P>
Store at -20°C in the box named &quot;séquençage&quot;.</P><P><B>Liquid culture</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Liquid Culture protocol)<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>1 colony of DH5alpha + pSB1C3 + Seq1,2 (colony 7)</LI><LI>1 colony of DH5alpha + pSB1C3 + Seq5,6,7 (colony2)</LI><LI>2 colonies of BL21 + pSB1C3 + Seq5,6,7 (colony 2) </LI><LI>1 colonies<I> S. aureus </I></LI></UL></P><P><B>Electrophoresis on agarose gel </B><U>Digestion enzymes for pSB1C3</U>: XbaI + SpeI <U>Digestion enzymes for Ks</U>: EcoRI + PstI 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
Results:</P><P><B>Interpretation</B>: three bands are visible for pSB1C3, we will not excise this one. We excised the lower band for the kill switch </P><P><B>Gel extraction</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Extraction protocol)
Results from Nanodrop:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> Ks cut</TD><TD> 8.4 </TD><TD> 2.66 </TD></TR></TBODY></TABLE><P><B>IN FUSION kit for pSB1C3 + Ks</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the  Ligation protocol&gt;</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>mass (ng)</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> pSB1C3</TD><TD> 140 </TD><TD> 6.9 </TD><TD> 20 </TD></TR><TR><TD> Ks </TD><TD> 60 </TD><TD> 8.4 </TD><TD> 7 </TD></TR><TR><TD>Enzyme mix </TD><TD> - </TD><TD> - </TD><TD> 4 </TD></TR></TBODY></TABLE><P><B>Transformation in STELLAR competent cells</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Transformation of <I>E. coli</I> DH5-alpha protocol)</P><P><B>Midiprep <I>S. aureus</I></B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Midiprep for plasmid extraction protocol)

Results from NanoDrop: <I>S. aureus</I></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 222.7 </TD><TD> 1.95 </TD></TR><TR><TD> 2 </TD><TD> 264.4 </TD><TD> 1.95 </TD></TR><TR><TD> 3 </TD><TD> 224.2 </TD><TD> 1.95 </TD></TR><TR><TD> 4   </TD><TD> 212.4 </TD><TD> 1.96 </TD></TR></TBODY></TABLE><P><B>Purification of NGF from the bacterial pellet of an 800 mL culture using sonication and FPLC</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Fast Protein Liquid Chromatography (FPLC) protocol) </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 4: </B>FPLC proNGF purification with ÄKTA pure (General Electric) Ni-NTA column was equilibrated with buffer A (50 mM Tris, pH 7.4, 200 mM NaCl). Supernatant of lyzed bacteria was introduced through the column. Washing with 5% of buffer B. Elution by buffer B gradient (buffer A + imidazole 250 mM). UV absorbance at 280nm is shown in blue, conductivity in red, and concentration of buffer B in green. </FIGCAPTION></FIGURE><P>
Tomorrow will be deposited on an SDS-PAGE gel the fractions:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>A2 (flow through from the deposited proteins)</LI><LI>B3 (flow through from the elution at a 5% buffer B proportion)</LI><LI>A4, A5, A6, A7 (2 mL fractions around the A280 peak)</LI></UL></P><P><B>Kill switch test #1</B>
 
Cells pre-culture of DH5a + Seq9 + Carbenicillin (25 µg/mL) at different temperatures.

Three culture conditions were performed: 37°C, 15°C and 4°C without positive control. After 4 hours of incubation 5 ml of each samples were placed at 37°C, 15°C or 4°C and incubated overnight.</P><P>Bacterial cultures placed at 15°C did not grow up on the four-hour period compared to the one a 37°C. Once placed at 37°C all bacterial cultures grown up again even if it is only slightly for the ones placed at 15°C and 4°C. 
We supposed the Kill switch has a bacteriostatic effect. We will perform another test with a positive control and different temperatures. </P><H2>08.29.2018</H2><P><B>Miniprep DH5alpha pSB1C3 Seq1,2 colony 7 and DH5alpha pSB1C3 Seq5,6,7 colony 2</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
Results from NanoDrop: DH5alpha pSB1C3 Seq1,2 colony 7 </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 346.1 </TD><TD> 1.92 </TD></TR><TR><TD> 2 </TD><TD> 391.7 </TD><TD> 1.91 </TD></TR><TR><TD> 3 </TD><TD> 411.2 </TD><TD> 1.92 </TD></TR><TR><TD> 4   </TD><TD> 330.8 </TD><TD> 1.92 </TD></TR><TR><TD> 5   </TD><TD> 321.9 </TD><TD> 1.90 </TD></TR></TBODY></TABLE><P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 272 </TD><TD> 1.95 </TD></TR><TR><TD> 2 </TD><TD> 242.3 </TD><TD> 1.93 </TD></TR><TR><TD> 3 </TD><TD> 261.7 </TD><TD> 1.94 </TD></TR><TR><TD> 4   </TD><TD> 283 </TD><TD> 1.92 </TD></TR><TR><TD> 5   </TD><TD> 279.8 </TD><TD> 1.93 </TD></TR></TBODY></TABLE></P><P>
Store at -20°C in the box named &quot;séquençage&quot;.
Results from NanoDrop : Kill Switch</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 9.9 </TD><TD> 2.01 </TD></TR><TR><TD> 2 </TD><TD> 8.1 </TD><TD> 2.12 </TD></TR><TR><TD> 3 </TD><TD> 6.7 </TD><TD> 2.7 </TD></TR><TR><TD> 4   </TD><TD> 10.1 </TD><TD> 2.14 </TD></TR><TR><TD> 5   </TD><TD> 9.7 </TD><TD> 1.90 </TD></TR></TBODY></TABLE><P>
Store at -20°C in the box named <I>S. aureus</I></P><P><B>Digestion of DH5alpha pSB1C3 + Seq5,6,7</B><U>Aim</U>: Verify that the In-Fusion kit has correctly worked for DH5alpha pSB1C3 + Seq5,6,7  </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Nhel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><B>Digestion of DH5alpha pSB1C3 + Seq5,6,7</B><U>Aim</U>: Verify that the In-Fusion kit has correctly worked for DH5alpha pSB1C3 + Seq5,6,7  </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>18</TD></TR></TBODY></TABLE><P><B>Electrophoresis on agarose gel of DH5alpha pSB1C3 + Seq1,2 and DH5alpha pSB1C3 + Seq5,6,7</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol)
Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Electrophoresis on agarose gel of DH5alpha pSB1C3 + Seq1,2 and DH5alpha pSB1C3 + Seq5,6,7 </FIGCAPTION></FIGURE><P><B>BL21(DE3)pLysS liquid culture and RIP production induction</B>
Two colonies of BL21(DE3)PlysS cells transformed with pSB1C3+Seq8 were put in liquid culture in 25 mL LB medium with 25 <FONT face="Raleway">μ</FONT>L chloramphenicol (25 mg/mL). 
Results: OD is followed during the culture</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>BL21(DE3)pLysS + pSB1C3 RIP</B></TD></TR><TR><TD> t = 0 min </TD><TD>0.248</TD></TR><TR><TD>t = 90 min</TD><TD>1.550</TD></TR></TBODY></TABLE><P>
Culture is diluted 1/10, making the OD600nm 0.228.

When OD reaches 0.667,  BL21(DE3)pLysS culture is separated into four 5 mL cultures in 50 mL Erlenmeyer. RIP production is induced with different concentrations of IPTG:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>One is not induced (Negative control)</LI><LI>One is induced with 0.1 mM</LI><LI>One is induced with 0.5 mM</LI><LI>One is induced with 1 mM</LI></UL></P><P>
Results after two hours induction (i.e. 550 minutes in the protocol above), OD is measured again:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> uninduced</B></TD><TD><B> IPTG 0.1 mM</B></TD><TD><B> IPTG 0.5 mM</B></TD><TD><B> IPTG 1 mM</B></TD></TR><TR><TD> OD<SUB>600 nm</SUB>/td&gt;
                                    </TD><TD> 2.011 </TD><TD> 1.830 </TD><TD>1.741 </TD><TD>1.868 </TD></TR></TBODY></TABLE><P>
1 mL of each culture is kept intact in a 1.5 mL Eppendorf tube and put at 4°C waiting for further treatment.
The 4 mL left are centrifuged, supernatant is collected.</P><P><B>SDS-PAGE of the protein fractions from the FPLC purification to prepare the Mass Spectrometry samples</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the SDS-PAGE protocol)
Deposit plan and results:<FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 7: </B>SDS-PAGE of the protein fractions from the FPLC purification to prepare the Mass Spectrometry samples </FIGCAPTION></FIGURE></P><P>
To analyze this gel, we will perform a Western Blot tomorrow.</P><H2>08.30.2018</H2><P><B>Miniprep STELLAR pSB1C3 Seq5,6,7 colony 1,2,3,4 and STELLAR pSB1C3 Seq9 (Ks)</B>

(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Miniprep for plasmid extraction protocol)
Results from NanoDrop: STELLAR pSB1C3 Seq5,6,7 colony 1,2,3,4 </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 93.2 </TD><TD> 2.01 </TD></TR><TR><TD> 2 </TD><TD> 112 </TD><TD> 1.95 </TD></TR><TR><TD> 3 </TD><TD> 114 </TD><TD> 1.94 </TD></TR><TR><TD> 4   </TD><TD> 109 </TD><TD> 2.01 </TD></TR></TBODY></TABLE><P>
Results from NanoDrop: STELLAR pSB1C3 Seq9</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> 1</TD><TD> 83.4 </TD><TD> 2.01 </TD></TR><TR><TD> 2 </TD><TD> 156.5 </TD><TD> 1.88 </TD></TR><TR><TD> 3 </TD><TD> 69.2 </TD><TD> 1.98 </TD></TR><TR><TD> 4   </TD><TD> 103 </TD><TD> 1.98 </TD></TR><TR><TD> 5   </TD><TD> 98.3 </TD><TD> 2.00 </TD></TR><TR><TD> 4   </TD><TD> 123.7 </TD><TD> 1.97 </TD></TR></TBODY></TABLE><P><B>Digestion of STELLAR pSB1C3 + Seq5,6,7 </B><U>Aim</U>: Verify that the In fusion kit has correctly work for STELLAR pSB1C3 + Seq5,6,7   </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Nhel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><U>Aim</U>: Verify that the In fusion kit has correctly work for STELLAR pSB1C3 + Seq9</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><B>Electrophoresis on agarose gel of STELLAR pSB1C3 + Seq5,6,7 and STELLAR pSB1C3 + Seq9 </B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol) 
Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 8: </B>Electrophoresis on agarose gel of STELLAR pSB1C3 + Seq5,6,7 and STELLAR pSB1C3 + Seq9 </FIGCAPTION></FIGURE><P><B>Biofilm formation test</B><I>S. aureus </I>is now grown overnight in a 96-well plate in different conditions in order to test the effect of both supernatant and our bacteria.
Each well contains 100 <FONT face="Raleway">μ</FONT>L of the following mix (using the pellet suspension and supernatant from yesterday’s culture):<OL style="text-align: left;font-size: 16px;"><LI>A-B LB medium alone and  E-H    990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI> A-D    990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture + 0.2 <FONT face="Raleway">μ</FONT>L IPTG (0.5 M) and E-H   990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture + 1 <FONT face="Raleway">μ</FONT>L IPTG (0.5 M)</LI><LI>A-D 990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture + 2 <FONT face="Raleway">μ</FONT>L IPTG (0.5 M)</LI><LI>A-D     LB + BL21 without plasmid + <I>S. aureus</I></LI><LI>E-H     LB + BL21 without plasmid + <I>S. aureus</I> + IPTG (1 mM)</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB un-induced supernatant BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB 0.1 mM-induced supernatant BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB 0.5 mM-induced supernatant BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> us culture</LI><LI>990 <FONT face="Raleway">μ</FONT>L LB 1 mM-induced supernatant BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB + Cm + 100 <FONT face="Raleway">μ</FONT>L uninduced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>881 <FONT face="Raleway">μ</FONT>L LB + Cm + 109 <FONT face="Raleway">μ</FONT>L 0.1 mM induced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>875 <FONT face="Raleway">μ</FONT>L LB + Cm + 115 <FONT face="Raleway">μ</FONT>L 0.5 mM induced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>880 <FONT face="Raleway">μ</FONT>L LB + Cm + 110 <FONT face="Raleway">μ</FONT>L 1 mM induced BL21 + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI></OL></P><P>

To identical plates are incubated at 37°C overnight.</P><P><B>Transfer and immunodetection on membrane of the SDS-PAGE Gel</B> from the 08.29.18
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Gel Transfer for Western Blotting protocol)
Results:
<FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 8: </B>SDS-PAGE Gel </FIGCAPTION></FIGURE></P><P>
At around 33 kDa, one band more intense seems to be the NGF.
The result of this Western Blot is not very clear. Yet, there seems to be a band at 33 kDa in moleculrar weight (MW) (which is the size of un-cleaved proNGF with the signal sequence HLYA). Several other bands of upper and lower MW bands are also visible. To confirm that the expression of recombinant proNGF is effective, this band along with a set of lower MW ones will be studied by Mass Spectrometry as shown below. If confirmed, this will mean that our NGF is not retained on the Ni-NTA column for an unknown reason, and that our proNGF is not cleaved from the signal peptide by the TEV protease. Presence of proNGF in lower bands might mean that the signal is cleaved, or that the full length protein has suffered from protease attack.</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 9: </B>SDS-PAGE gel Bis-Tris 4-12% of bacterial lysate and proNGF purification fraction by SDS-PAGE.</FIGCAPTION></FIGURE><H2>08.31.2018</H2><P><B>Test Kill Switch #2</B>
From overnight pre-culture of Stellar pSB1C3+ KS and pSB1C3 backbone (positive control) in LB+ Chloramphenicol 25 <FONT face="Raleway">μ</FONT>g/mL (CAM) </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Stellar pSB1C3+KS</B></TD><TD><B>Stellar pSB1C3</B></TD></TR><TR><TD> OD<SUB>(600 nm)</SUB></TD><TD>1.240</TD><TD>1.660</TD></TR></TBODY></TABLE><P>
Launch of 8 bacterial cultures in Falcon 50 of 15 ml + CAM, with pre-cultures diluted 1/30.
Incubation all weekend at 180 rpm at different temperatures: 37°C, 25°C, 20°C and 18°C.</P><P><B>Test biofilm formation on various metals #1</B>
In order to test the adherence of our biofilm to metallic surfaces, two cylindrical pieces of metal, one made of titanium (TA6V4)  and the other of stainless steel (316L), approximately 5 cm in length and 1.0 cm in diameter each (Generous gift from I-CERAM, Limoges, France), were incubated in Falcon 15 + 7 ml of LB+ CAM.  50 µl of BL21(DE3)pLysSeq5,6,7 were inoculated. Incubation over the weekend at 37°C, 180 rpm. </P><P><B>Biofilm in 96-wells plate</B>
One of the two plates that was incubated yesterday was washed with the following procedure:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Supernatant in each well was discarded with a pipet</LI><LI>Each well was then washed two times by adding 100 <FONT face="Raleway">μ</FONT>L of distilled water </LI></UL></P><P>
Plates were read by measuring fluorescence at 510 nm before and after washing. We obtained the following results:</P><P>The washed plate was filled again with 100 <FONT face="Raleway">μ</FONT>L LB + CAM in each well, and left to incubate for the week-end at 37°C. The second plate was left for the week-end as well.</P>
The washed plate was filled again with 100 μL LB + CAM in each well, and left to incubate for the week-end at 37°C. The second plate was left for the week-end as well. </DIV><DIV data-description="Week_35" class="panel" id="pan_3501" style="text-align:left;"><H2> 08.30.2018</H2><P><B>Coating of microfluidic chips</B>
                                         Coating of 18 microfluidic chips with PLL, final concentration of 10 µg/mL
                                          Incubation at 37°C overnight</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the PDMS Chip Coating protocol.</P><H2> 08.31.2018</H2><P><B>Neuronal primary culture</B></P><P> See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the PDMS Chip Coating protocol.</P><P>PLL is drained from all microfluidic chips. Chips are rinsed one time with PBS 1X and Laminin is added at a final concentration of 1 mg/mL and incubated for 2 h at 37°C. 
Laminin is drained from the microfluidic chips. Cortex cells are dissociated following BrainBits.co.UK papain digestion protocol. </P><P> See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Cell Dissociation protocol.</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Cell Counting protocol.</P><P>Cellular concentration is counted using a Neubauer chamber and put at a final concentration of 40 000 cells / 1.7 <FONT face="Raleway">μ</FONT>L.</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Seeding Neurons into the Microfluidic Chip protocol.</P><P>Seeding of the cells is carried out in all microfluidic chips. Different media are then added depending on the experiment and put in culture for 6 days at 37°C, CO2 5%. </P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Final volume of media</B></TD><TD><B>Neurobasal</B></TD><TD><B>DMEM</B></TD><TD><B>SVF</B></TD><TD><B>B27</B></TD><TD><B>Glutamax</B></TD><TD><B>Commercial NGF (50ng/mL)</B></TD></TR><TR><TD><B>Experiment 1</B></TD><TD>1mL</TD><TD>/</TD><TD>920 <FONT face="Raleway">μ</FONT>L</TD><TD>50 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L </TD><TD>/ </TD></TR><TR><TD><B>Experiment 2</B></TD><TD>3 mL</TD><TD>/</TD><TD>2850 <FONT face="Raleway">μ</FONT>L</TD><TD>150 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>15 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 3</B></TD><TD>3 mL</TD><TD>/</TD><TD>2850 <FONT face="Raleway">μ</FONT>L</TD><TD>150 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>15 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B> Control +</B></TD><TD>1 mL</TD><TD>970 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD></TR><TR><TD><B>Control -</B></TD><TD>1 mL</TD><TD>/</TD><TD>950 <FONT face="Raleway">μ</FONT>L</TD><TD>50 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD> /</TD><TD>/</TD></TR><TR><TD><B>Experiment 7</B></TD><TD>1 mL</TD><TD>970 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD></TR></TBODY></TABLE></DIV><DIV data-description="Week_35" class="panel" id="pan_3502" style="text-align:left;"><H2> 08.27.2018</H2><P> 4 well chips have been made, each with a different type of membrane alumina oxide membrane filter (bare, PEDOT:PSS, PEDOT:Cl, PEDOT:Ts). 4 vertical chips bonded to tissue culture dishes. </P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: Bare alumina oxide, PEDOT:PSS and PEDOT:Ts membranes  on well chips</FIGCAPTION></FIGURE><P> Well chip fabrication protocol (the chip assembled on that day is slightly different but doesn't work) <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><H2> 08.28.2018</H2><P> 8 double membrane microchannel chips have been bonded to imaging dishes and sterilized by exposure to UV rays. Leaks made this a difficult and time-consuming operation. 8 microchannel chips have been made. </P><P> Scanning electron microscopy (FEI Magellan 400 microscope) of a bare alumina oxide membrane and a PEDOT:PSS-coated one. </P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 2: bare alumina oxide membrane</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION>Figure 3: PEDOT:PSS-coated alumina oxide membrane</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Bare alumina oxide membrane imaging confirmed the presence of 0.2 micrometer pores on the surface, enough to confine bacteria like <I>E. coli</I>. PEDOT:PSS-coated membrane revealed cluster-like formation of PEDOT:PSS deposits. It is thought that the lack of uniformity of the coating won't give the expected results in matters of biocompatibility and conductivity.</P><P> Basic and double membrane microchannel chip fabrication and sterilization protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P><P> Double membrane microchannel chip bonding and sterilization protocol, basic microchannel chip fabrication protocol  <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_36"><DIV class="block full" id="Week_36" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_36" class="vignette" id="vign_3600"><DIV data-description="Week_36" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_36" class="vignette" id="vign_3601"><DIV data-description="Week_36" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_36" class="panel" id="pan_3600" style="text-align:left;"><H2>09.03.2018</H2><P><B>Test Kill Switch after over weekend incubation</B>
Results after the week-end incubation at different temperatures: </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Table of results after the week-end incubation at different temperatures</FIGCAPTION></FIGURE><P>
RT: Room temperature
Interpretation: Bacterial cultures containing the Kill Switch (KS) below 20°C did not grow. At 25°C and over, cultures containing the KS can grow. Bacterial cultures incubated at 20°C and 37°C both KS and pSB1C3 were spread on plate and incubated at 37°C and RT.
All bacterial cultures were incubated at 37°C under shaking conditions. OD600nm for cultures previously incubated at 18°C and 20°C was monitored.  
Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Table of results</FIGCAPTION></FIGURE><P>
OD<SUB>600nm</SUB> was set to 0.01 and placed at 20°C. </P><H2>09.04.2018</H2><P><B>Miniprep pSB1C3+KS col2/6 and pSB1C3 + Seq5,6,7 col1/4</B>
Results from NanoDrop: pSB1C3+KS col 6 </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> Col6 1</TD><TD> 153.3 </TD><TD> 1.97 </TD></TR><TR><TD> Col6 2 </TD><TD> 117 </TD><TD> 1.81 </TD></TR><TR><TD> Col6 3 </TD><TD> 129.8 </TD><TD> 1.91 </TD></TR><TR><TD> Col6 4  </TD><TD> 130.6 </TD><TD> 1.94 </TD></TR></TBODY></TABLE><P>
Results from NanoDrop: pSB1C3 + Seq5,6,7 col1</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Sample</B></TD><TD><B>DNA Concentration (ng/µL)</B></TD><TD><B>Abs<SUB>(260 nm)</SUB>/Abs<SUB>(280 nm)</SUB></B></TD></TR><TR><TD> Col1 1</TD><TD> 183.4 </TD><TD> 1.95 </TD></TR><TR><TD> Col1 2 </TD><TD> 147.6 </TD><TD> 1.96 </TD></TR><TR><TD> Col1 3 </TD><TD> 149.8 </TD><TD> 1.91 </TD></TR><TR><TD> Col1 4  </TD><TD> 168.9 </TD><TD> 1.95 </TD></TR></TBODY></TABLE><P><B>Enzymatic digestion</B><U>Aim</U>: Verify that pSB1C3 + Seq5,6,7 col 1 contain KS insert for sequencing:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>Nhel</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>19</TD></TR></TBODY></TABLE><P><U>Aim</U>: Verify that pSB1C3 + KS col 6 contain KS insert for sequencing:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Volume (<FONT face="Raleway">μ</FONT>L)</B></TD></TR><TR><TD> DNA </TD><TD>7</TD></TR><TR><TD> Cut Smart </TD><TD>3</TD></TR><TR><TD>PstI</TD><TD>1</TD></TR><TR><TD>EcoRI</TD><TD>1</TD></TR><TR><TD>H<SUB>2</SUB>O</TD><TD>18</TD></TR></TBODY></TABLE><P><B>Electrophoresis on agarose gel of TELLAR pSB1C3 + Seq5,6,7 and STELLAR pSB1C3 + KS  </B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Agarose Gel Preparation protocol) 
Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 8: </B>Electrophoresis on agarose gel of STELLAR pSB1C3 + Seq5,6,7 and STELLAR pSB1C3 + KS </FIGCAPTION></FIGURE><P>Plasmids do contain inserts, and samples can be sent to sequencing. </P><P><B>Kill Switch test plates observations</B>
Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Results table </FIGCAPTION></FIGURE><P>
Bacterial culture placed at 20°C over-weekend and spread on plate placed at 37°C formed a mat, i.e. completely covered the plate, for either the Kill-Switch and controls. However, when they were placed at room temperature (RT), the positive control formed a mat whilst the one containing the Kill-Switch formed small colonies (58 CFU) at D+1.  
Cultures placed at 37°C over the week-end and incubated either at RT and 37°C grew as a mat. However, on the one containing the Kill-Switch less colonies grew. We counted 125 colonies at 1/50 dilution.</P></DIV><DIV data-description="Week_36" class="panel" id="pan_3601" style="text-align:left;"><H2> 09.03.2018 </H2><P><B> Media change for cell culture</B></P><P>Media is removed from microfluidic chip and fresh media is then added. Type of media used is dependant on the experiment. </P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Final volume of media</B></TD><TD><B>Neurobasal</B></TD><TD><B>DMEM</B></TD><TD><B>SVF</B></TD><TD><B>B27</B></TD><TD><B>Glutamax</B></TD><TD><B>Commercial NGF (50ng/mL)</B></TD></TR><TR><TD><B>Experiment 1</B></TD><TD>1mL</TD><TD>/</TD><TD>920 <FONT face="Raleway">μ</FONT>L</TD><TD>50 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L </TD><TD>/ </TD></TR><TR><TD><B>Experiment 2</B></TD><TD>3 mL</TD><TD> /</TD><TD>2850 <FONT face="Raleway">μ</FONT>L</TD><TD>150 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>15 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 3</B></TD><TD>3 mL</TD><TD>/</TD><TD>2850 <FONT face="Raleway">μ</FONT>L</TD><TD>150 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>15 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B> Control +</B></TD><TD>1 mL</TD><TD>970 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD></TR><TR><TD><B>Control -</B></TD><TD>1 mL</TD><TD>/</TD><TD>950 <FONT face="Raleway">μ</FONT>L</TD><TD>50 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD> /</TD><TD>/</TD></TR><TR><TD><B>Experiment 7</B></TD><TD>1 mL</TD><TD>970 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD><TD>/</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD></TR></TBODY></TABLE><P> Incubate at 37°C, 5% CO2</P><H2> 09.06.2018</H2><P><B>Fixation of cells remaining in the 9 microfluidic chips</B></P><P>Media is drained from the 9 remaining chips. Cells are fixed using paraformaldehyde (PFA) 4% for 30 min at room temperature. PBS 1X is added. Stored at 4°C overnight
</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Fixation and Staining Neurons protocol.</P><H2> 09.07.2018</H2><P><B>Staining of remaining 9 microfluidic chips</B></P><P>PBS 1X is drained from the 9 remaining chips. Cells are first permeabilized by adding Triton X-100 at 0.5 % for 10 min at room temperature and unspecific sites are saturated using bovine serum albumin (BSA) 1 % for 30 min at room temperature. Cells are stained for 1 h at room temperature using a mix of DAPI 1/20 000, anti MAP2 antibody coupled with Alexa Fluor 555 nm 1/500 and 1/500 Anti--tubulin antibody coupled with Alexa Fluor 448 nm. 
Antibody mix is drained, and cells are stored in PBS 1X in the dark at 4°C. </P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Fixation and Staining Neurons protocol.</P></DIV></DIV></DIV><DIV data-description="Week_37"><DIV class="block full" id="Week_37" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_37" class="vignette" id="vign_3700"><DIV data-description="Week_37" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_37" class="vignette" id="vign_3701"><DIV data-description="Week_37" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_37" class="vignette" id="vign_3702"><DIV data-description="Week_37" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_37" class="panel" id="pan_3701" style="text-align:left;"><H2> 09.12.2018</H2><P><B>Coating of microfluidic chips </B></P><P> Coating of different types of chips:  10 regular microfluidic chips, 4 microfluidic chips with a membrane and 4 microfluidic chips, used for electrical conductivity measurements with 50 µL per well of PLL (final concentration of 10 µg/mL).</P><P>Incubation at 37°C overnight.</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the PDMS Chip Coating protocol.</P><H2> 09.13.2018</H2><P><B> Neuronal primary culture</B></P><P>PLL is drained from all microfluidic chips. Chips are rinsed one time with PBS 1X and Laminin is added at a final concentration of 1 mg/ mL and incubated for 2h at 37°C. 
</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the PDMS Chip Coating protocol.</P><P>Laminin is drained from microfluidic chips. Cortex cells are dissociated using Trypsin 2.5% and DNAse 1000 U. See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Digestion protocol.</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Cell Dissociation protocol.</P><P>Cellular concentration is counted using a Neubauer chamber and put at a final concentration of 40 000 cells / 1.7 <FONT face="Raleway">μ</FONT>L</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Cell Counting  protocol.</P><P>Seeding of the cell is realized in all microfluidic chips. Different media are then added depending on the experiment and put in culture for 6 days at 37°C, 5% C02. </P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Types of chips</B></TD><TD><B>Final volume of media</B></TD><TD><B>Neurobasal</B></TD><TD><B>B27</B></TD><TD><B>Glutamax</B></TD><TD><B>Commercial NGF (Stock 10 000 ng/mL)</B></TD></TR><TR><TD><B>Experiment 1</B></TD><TD>1 regular chip</TD><TD>1 mL</TD><TD>965 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>50 ng/mL  5 <FONT face="Raleway">μ</FONT>L </TD></TR><TR><TD><B>Experiment 2</B></TD><TD>1 regular chip</TD><TD>3 mL</TD><TD>945 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>250 ng/mL  25 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 3</B></TD><TD>1 regular chip</TD><TD>3 mL </TD><TD>920 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>500 ng/mL50 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B> Experiment 4</B></TD><TD>1 regular chip</TD><TD>1 mL</TD><TD>890 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD> 750 ng/mL  75 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 5</B></TD><TD>1 regular chip </TD><TD>1 mL</TD><TD>895 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD> 900 ng/mL  90 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 6</B></TD><TD>5 regular chips + 2 chips with simple membrane and 2 chips with double membrane + 4 chips for conductivity measurements</TD><TD>1 mL</TD><TD>880 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD></TR></TBODY></TABLE><P> Incubate 37°C, 5% CO2</P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Seeding Neurons into the Microfluidic Chip protocol.</P></DIV><DIV data-description="Week_37" class="panel" id="pan_3702" style="text-align:left;"><H2> 09.10.2018 </H2><P> Scanning electron microscopy (FEI Magellan 400) of PEDOT:Cl-coated membranes. </P><P><B> Results </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: bare alumina oxide membrane</FIGCAPTION></FIGURE><FIGURE style="display:inline-block"><FIGCAPTION>Figure 2: PEDOT:Cl-coated membrane</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Microscopy reveals a uniform thickening of the membrane's surface, suggesting a uniform PEDOT:Cl coating of the membrane. We expect better results from the PEDOT:Cl-coated and PEDOT:Ts-coated membranes than PEDOT:PSS-coated ones </P><H2> 09.11.2018 </H2><P> Bonding to imaging dishes and UV sterilization of 20 microchannel chips. </P><P> Microchannel chip bonding and sterilization protocol <A href="https://static.igem.org/mediawiki/2018/4/46/T--Pasteur_Paris--Microfluidics-microchannel-chip.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_38"><DIV class="block full" id="Week_38" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_38" class="vignette" id="vign_3800"><DIV data-description="Week_38" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_38" class="vignette" id="vign_3801"><DIV data-description="Week_38" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_38" class="panel" id="pan_3800" style="text-align:left;"><H2>09.18.2018</H2><P><B>RIP expression and biofilm formation test</B>
Liquide culture of <I>S. aureus</I> and BL21-RIP. BL21-RIP. The OD<SUB>600nm</SUB> of BL21(DE3)pLysS where set at 0.1 and monitored for IPTG induction. 
Results:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B> Time after induction</B></TD><TD><B> BL21-RIP 0.1 mM</B></TD><TD><B>BL21-RIP 0.5 mM</B></TD><TD><B> BL21-RIP 1 mM</B></TD><TD><B> BL21-RIP 2 mM</B></TD><TD><B> BL21-RIP </B></TD><TD><B> BL21-RIP Mass 1 mM</B></TD></TR><TR><TD> t=1h </TD><TD> 0.148</TD><TD>0.159</TD><TD>0.163</TD><TD>0.141</TD><TD>0.169</TD><TD>0.074</TD></TR><TR><TD> t = 2h30</TD><TD> 0.523</TD><TD> 0.528</TD><TD>0.530 </TD><TD>0.849 </TD><TD>0.536 </TD><TD>0.545 </TD></TR></TBODY></TABLE><P>
IPTG induction and incubation for 2 hours
Samples were centrifugated. Pellets were resuspended in 1 mL of LB. The sample going for the mass spectrometry measurement was resuspended in Buffer (Tris 20 mM, pH 8, NaCl 200 mM) and thermolyzed (5-10 min in dry ice and then placed at 37°C for 3 min). The lysate was centrifugated at 16 000 rpm during 5 min and supernatant was kept. </P><P><B>Test in 96-well plates</B>
Each well contains 100 <FONT face="Raleway">μ</FONT>L of the following mix:<OL style="text-align: left;font-size: 16px;"><LI>A-D LB medium+ Cm </LI><LI>E-F 990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture + 0.2 <FONT face="Raleway">μ</FONT>L IPTG (0,5 M)</LI><LI>G-H 990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture + 1 <FONT face="Raleway">μ</FONT>L IPTG (0,5 M)</LI><LI>A-D 990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI> E-F    990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L<I> S. aureus</I> culture + 2 <FONT face="Raleway">μ</FONT>L IPTG (0,5 M)</LI><LI>G-H 990 <FONT face="Raleway">μ</FONT>L LB + Cm + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus </I>culture + 4 <FONT face="Raleway">μ</FONT>L IPTG (0,5 M)</LI><LI>750 <FONT face="Raleway">μ</FONT>L un-induced supernatant BL21-RIP+ 240 LB + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>750 <FONT face="Raleway">μ</FONT>L 0.1 mM supernatant BL21-RIP+ 240 LB + Cm+ 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>750 <FONT face="Raleway">μ</FONT>L LB 0.5 mM supernatant BL21-RIP+ 240 LB + Cm+ 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>750 <FONT face="Raleway">μ</FONT>L LB  1 mM supernatant BL21-RIP+ 240 LB + Cm+ 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>750 <FONT face="Raleway">μ</FONT>L LB 2 mM supernatant BL21-RIP + Cm+ 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB+ Cm+ 100µl pellet supernatant non-induced BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB+ Cm+ 100µl pellet supernatant 0.1 mM BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus </I>culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB+ Cm+ 100µl pellet supernatant 0.5 mM BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI><LI>890 <FONT face="Raleway">μ</FONT>L LB + CAM + 100 µl pellet supernatant 1 mM BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus </I>culture</LI><LI>880 <FONT face="Raleway">μ</FONT>L LB + CAM + 100 µl pellet supernatant 2 mM BL21-RIP + 10 <FONT face="Raleway">μ</FONT>L <I>S. aureus</I> culture</LI></OL></P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B> Guild plate. </FIGCAPTION></FIGURE><P>
96-well plate pattern: 4 identical plates are incubated at 37°C, 72h. 

Results:</P><H2>09.21.2018</H2><P><B>Biofilm formation test</B>
After 72h incubated at 37°C, non-adherent cells were removed by two ways:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>Using the micropipette</LI><LI>Remove by inversion</LI></UL></P><P>
For each way, 2 plates were done and one of the two was stained with crystal violet 0.1%. 
Plate1: Remove non-adherent cells with micropipette, water wash with micropipette. Read fluorescence <FONT face="Raleway">λ</FONT>EX485/<FONT face="Raleway">λ</FONT>EM510 nm
Plat2: Remove non-adherent cells by inversion, water wash with inversion. Read fluorescence <FONT face="Raleway">λ</FONT>EX485/<FONT face="Raleway">λ</FONT>EM510 nm
Plate 3: Remove non-adherent cells with micropipette, water wash with micropipette. Stain with crystal violet and wash with micropipette. Read absorbance at <FONT face="Raleway">λ</FONT>=590 nm
Plate 4: Remove non-adherent by inversion, water wash by inversion. Staining crystal violet and wash by inversion. <FONT face="Raleway">λ</FONT>=590 nm</P></DIV><DIV data-description="Week_38" class="panel" id="pan_3801" style="text-align:left;"><H2> 09.17.2018</H2><P><B> Media change for cell culture</B></P><P>Media is removed from microfluidic chips and fresh media is then supplied. The type of media used is dependent on the experiment. </P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Final volume of media</B></TD><TD><B>Neurobasal</B></TD><TD><B>B27</B></TD><TD><B>Glutamax</B></TD><TD><B>Commercial NGF (Stock 10 000 ng/mL)</B></TD></TR><TR><TD><B>Experiment 1</B></TD><TD>1 mL</TD><TD>965 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>50 ng/mL  5 <FONT face="Raleway">μ</FONT>L </TD></TR><TR><TD><B>Experiment 2</B></TD><TD>3 mL</TD><TD>945 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>250 ng/mL  25 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 3</B></TD><TD>3 mL </TD><TD>920 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>500 ng/mL50 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B> Experiment 4</B></TD><TD>1 mL</TD><TD>890 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD> 750 ng/mL  75 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 5</B></TD><TD>1 mL</TD><TD>895 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD> 900 ng/mL  90 <FONT face="Raleway">μ</FONT>L</TD></TR><TR><TD><B>Experiment 6</B></TD><TD>1 mL</TD><TD>880 <FONT face="Raleway">μ</FONT>L</TD><TD>20 <FONT face="Raleway">μ</FONT>L</TD><TD>10 <FONT face="Raleway">μ</FONT>L</TD><TD>/</TD></TR></TBODY></TABLE><P>Into 6 chips previously put in culture with Neuro / B27 / GlutaMAX (2 without membrane, 2 with simple membrane, 2 with double membrane), remove half the media and addition of the same volume of fresh media: Neurobasal / B27 / GlutaMAX/NGF 900 ng/mL for a final concentration of commercial NGF 450 ng/mL.</P><P> Incubate at 37°C, 5% CO2</P><H2> 09.19.2018 </H2><P><B> Fixation of remaining 13 microfluidic chips</B></P><P>Media is drained from the 13 remaining chips. Cells are fixed using paraformaldehyde (PFA) at 4% for 15 min at room temperature. PBS 1X is added. 
</P><P><B> Staining of remaining 13 microfluidic chips</B></P><P>PBS 1X is drained from the 13 remaining chips. Cells are first permeabilized by adding Triton X-100 at 0.5% for 10 min at room temperature and unspecific sites are saturated using Bovine serum albumin (BSA) at 1% for 30 min at room temperature. Cells are stained for 15 min at room temperature using a mix of DAPI 1/20 000, anti MAP2 antibody coupled with Alexa Fluor 555nm 1/500 and 1/500 Anti--tubulin antibody coupled with Alexa Fluor 448 nm. </P><P>Antibody mix is drained, and cells are stored in PBS 1X in the dark at 4°C. </P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Fixation and Staining Neurons protocol.</P><H2> 09.20.2018 </H2><P><B> Electric measurements on 4 remaining chips </B></P><P>Stimulation and measure of the electric signal at the interface of the neuronal system and the conductive membrane using an oscilloscope and a low frequency generator. 
</P><P><I><B>Materials:</B></I>
-Vertical chip with PEDOT:Ts membrane (see <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Membrane PEDOT:Ts and PEDOT:Cl coating protocol) seeded with neurons and filled with culture medium (see <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Seeding neurons into the microfluidic chip protocol). 

-Function generator (GW Intstek SFG-2010)

-Oscilloscope (Tektronix TBS 2000 Series)

-Phosphate buffered saline solution (PBS)
</P><P><I><B>Procedure:</B></I>
The goal is to measure the output of nerves in presence of a membrane by subjecting them to an electric signal. 
Function generator is set on sine mode, 240 mV at 50 Hz. The two wires are connected to the silver paste spots on the bottom of the imaging dish and between the two layers of PDMS composing the chip to allow the current to pass through the system. An oscilloscope is then connected to these spots as well. First measurements indicate that the input signal is not being transmitted at all (<I>Figure 1</I>) by the system neurons + membrane, suggesting something is not working well in the setup.
</P><FIGURE style="display:inline-block"><FIGCAPTION><I>Figure 1</I></FIGCAPTION></FIGURE><P>It is hypothesized, the ions in the culture medium might be perturbing the measure, so the decision is taken to remove the culture medium from the chip. The result is the same (<I>Figure 2</I>).</P><FIGURE style="display:inline-block"><FIGCAPTION><I>Figure 2</I></FIGCAPTION></FIGURE><P>The chip is then washed with PBS 1X and another measurement is made. Results remained the same (<I>Figure 3</I>).</P><FIGURE style="display:inline-block"><FIGCAPTION><I>Figure 3</I></FIGCAPTION></FIGURE><P><I><B>Results:</B></I>
Results: It can be concluded from this experiment, that testing the electric conductivity of the system neurons + membrane is harder than expected and might indicate the need for more precautions and more rigorous methods of measurement.</P><H2>09.21.2018</H2><P><B> Cell culture in 96-well plate </B></P><P>
In a 96-well plate
-Add 50 <FONT face="Raleway">μ</FONT>L Poly-L-ornithine 0.01% solution (Sigma Aldrich, ref : P4957) per well

-Incubate the plate 1h at 37°C

-Wash 3 times with 50 <FONT face="Raleway">μ</FONT>L PBS 1X per well

-Add 50 <FONT face="Raleway">μ</FONT>L laminin per well

-Incubate the plate 1h at 37°C

-Prepare 10 mL of culture medium at 37°C: 9.7mL neurobasal, 100 <FONT face="Raleway">μ</FONT>L Glutamax, 200 <FONT face="Raleway">μ</FONT>L B27, 100 <FONT face="Raleway">μ</FONT>L P/S

-Prepare dilutions
</P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Final [NGF] (ng/<FONT face="Raleway">μ</FONT>L)</B></TD><TD>0</TD><TD>50</TD><TD>250</TD><TD>500</TD><TD>750</TD><TD>900</TD></TR><TR><TD><B>Commercial NGF (<FONT face="Raleway">μ</FONT>L)</B></TD><TD>0</TD><TD>7.5</TD><TD>37.5</TD><TD>75</TD><TD>112.5</TD><TD>135</TD></TR><TR><TD><B>PSB 1X (<FONT face="Raleway">μ</FONT>L)</B></TD><TD>1500</TD><TD>1492.5</TD><TD>1462.5</TD><TD>1425</TD><TD>1387.5</TD><TD>1365</TD></TR></TBODY></TABLE><P>
-Defreeze cells at 37°C 

-Dillute them in 5 mL of neurobasal medium pre-warmed at 37°C

-Divide the 6 mL in 6 tubes (for the 6 experiments) 

-Centrifuge the tubes at 1200 rpm for 4 min 

-Remove the supernatant (leave about 100 <FONT face="Raleway">μ</FONT>L of medium) 

-Add 1.5 mL of appropriate medium 

-Remove the laminin

-Add 100 <FONT face="Raleway">μ</FONT>L of culture per well

-Incubate at 37°C, 5% CO2
</P><P><I>Guide plate</I></P><P><B> Imaging of microfluidic chips fixed on 09.06.2018 and 09.19.2018</B></P><P>The neurons were fixed after 7 days of culture. In green the axon and blue the nucleus.
The neurons grew, and we can observe the axon on the left chamber. The axons pass through the microchannels. </P></DIV></DIV></DIV><DIV data-description="Week_39"><DIV class="block full" id="Week_39" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_39" class="vignette" id="vign_3900"><DIV data-description="Week_39" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_39" class="vignette" id="vign_3901"><DIV data-description="Week_39" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_39" class="panel" id="pan_3900" style="text-align:left;"><H2>09.26.2018</H2><P><B>RIP expression and biofilm formation test</B>

Liquide culture of <I>S. aureus </I>and BL21(DE3)pLysS-RIP. BL21(DE3)pLysS -RIP. The OD600nm of BL21(DE3)pLysS where set at 0.1 and monitored for IPTG induction. 
IPTG induction at OD= 0.768 and incubation for 2 hours.
Sample was centrifugated. Pellet was resuspended in 1mL of LB and thermolyzed (5-10 min in dry ice and then placed at 37°C for 3 min) Lysate was centrifugated at 16 000 rpm during 5 min and the supernatant was saved. </P><P><B>Test in 96-well plates </B></P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>96-well plate pattern </FIGCAPTION></FIGURE><P><I>S. aureus </I>culture was diluted 1/100 in each well. 
SA+ PS BL21(DE3)pLysS -RIP 5: Pellet supernatant non-induced 5 µL/well
SA+ PS BL21(DE3)pLysS -RIP 1 mM 5: Pellet supernatant induced 1 mM IPTG 5 µL/well
SA+ PS BL21(DE3)pLysS -RIP 1 mM 10: Pellet supernatant induced 1 mM IPTG 10 µL/well
SA+ PS BL21(DE3)pLysS -RIP 1 mM 20: Pellet supernatant induced 1 mM IPTG 20 µL/well

5 identical plates are incubated at 37°C for 24h, 48h, 72h. 

Results: 
Fluorescence was read at 24h, 48h and 72h
Test of WPI Protocol at 48h </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: </B>Results</FIGCAPTION></FIGURE></DIV><DIV data-description="Week_39" class="panel" id="pan_3901" style="text-align:left;"><H2> 09.24.2018</H2><P><B> Media change for cell culture in 96-well plate </B></P><P>Media is removed from the plate and fresh media is then supplied. Type of media used is dependent on the experiment (different NGF concentrations). 
</P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Final [NGF] (ng/<FONT face="Raleway">μ</FONT>L)</B></TD><TD>0</TD><TD>50</TD><TD>250</TD><TD>500</TD><TD>750</TD><TD>900</TD></TR><TR><TD><B>Commercial NGF (<FONT face="Raleway">μ</FONT>L)</B></TD><TD>0</TD><TD>7.5</TD><TD>37.5</TD><TD>75</TD><TD>112.5</TD><TD>135</TD></TR><TR><TD><B>PSB 1X (<FONT face="Raleway">μ</FONT>L)</B></TD><TD>1500</TD><TD>1492.5</TD><TD>1462.5</TD><TD>1425</TD><TD>1387.5</TD><TD>1365</TD></TR></TBODY></TABLE><P>Add 100 <FONT face="Raleway">μ</FONT>L of culture per well and incubate at 37°C, 5% CO2.</P><H2> 09.27.2018 </H2><P><B> Media change for cell culture in 96-well plate </B></P><P>Media is removed from the plate and fresh media is then added. Type of media used is dependent on the experiment (different NGF concentrations). 
</P><TABLE class="tableData" style="margin:auto;"><TBODY><TR><TD><B>Final [NGF] (ng/<FONT face="Raleway">μ</FONT>L)</B></TD><TD>0</TD><TD>50</TD><TD>250</TD><TD>500</TD><TD>750</TD><TD>900</TD></TR><TR><TD><B>Commercial NGF (<FONT face="Raleway">μ</FONT>L)</B></TD><TD>0</TD><TD>7.5</TD><TD>37.5</TD><TD>75</TD><TD>112.5</TD><TD>135</TD></TR><TR><TD><B>PSB 1X (<FONT face="Raleway">μ</FONT>L)</B></TD><TD>1500</TD><TD>1492.5</TD><TD>1462.5</TD><TD>1425</TD><TD>1387.5</TD><TD>1365</TD></TR></TBODY></TABLE><P>Add 100 <FONT face="Raleway">μ</FONT>L of culture per well and incubate at 37°C, 5% CO2.</P></DIV></DIV></DIV><DIV data-description="Week_40"><DIV class="block full" id="Week_40" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_40" class="vignette" id="vign_4000"><DIV data-description="Week_40" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_40" class="vignette" id="vign_4001"><DIV data-description="Week_40" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_40" class="vignette" id="vign_4002"><DIV data-description="Week_40" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_40" class="panel" id="pan_4000" style="text-align:left;"><H2>10.02.2018</H2><P><B>Batch purification of 6xHIS-tagged proNGF (Seq1,2) with Ni-NTA beads under native and partial denatured conditions. </B><U>Aim</U>: Purify the proNGF under both native and partial denatured form to study whether the His-tag is accessible for Ni-NTA purification. We already tried on Ni-NTA column without any  retention of our His-tagged protein on the column. 
Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>2 x 200 µl Ni-NTA beads (100 µl beads)</LI><LI>Buffer A (Tris 20 mM, pH 8, NaCl 200 mM) and Buffer U (Tris 20 mM, pH 8, NaCl 200 mM, Urea 2M)</LI></UL></P><P>
Under native conditions, work at 4°C, under partial denatured work at room temperature (RT).
Ni-NTA beads were equilibrated with 4 volumes of beads (Buffer A or U), centrifuged for 5 min at 500 x g, two times. One volume of protein solution was added and allowed to bind for 1 hour at 4°C or RT. Beads were centrifuged for 5 min at 500 x g and an aliquot was taken. Beads were washed 4 times with 4 volumes of buffer containing 20 mM Imidazole. One volume of elution buffer (Imidazole 250 mM) was added and allowed to stand for 10 min. Centrifugation allow to get back the supernatant containing purified proteins.  

Addition of loading buffer and heat the sample at 95°C for 5 min. Sample were loaded on 2 Bis-tris 12% gel. 
One gel was stained with E/Z blue staining. The other gel was transferred on to a nitrocellulose membrane and stored in PBS at 4°C. </P><H2>10.03.2018</H2><P>Immunodetection of His-tagged proNGF </P>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Immunodetection on membrane protocol)<P>
We performed batch purification using NiNTA beads under native and partial denaturing conditions (Urea 2 M) followed by Western Blot analysis with immunodetection through Anti-His Antibodies Alexa Fluor 647. (Figure 10) Detection of His-tag in the pellet supernatant of induced BL21 with 1 mM IPTG and flow through when partially denatured.
His-tagged proNGF was not retained on NiNTA beads. N-terminal His tag may be hidden in the protein fold. Consequently, we did not manage to purify the proNGF.</P></DIV><DIV data-description="Week_40" class="panel" id="pan_4001" style="text-align:left;"><H2>10.02.2018</H2><P><B> Fixation of neuronal culture inside 96-well plate</B></P><P>Media is drained from the plate. Cells are fixed using 100 <FONT face="Raleway">μ</FONT>L paraformaldehyde (PFA) at 4% for 15 min at room temperature. PBS 1X is added, cells are conserved at 4°C for the night. </P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Fixation and Staining Neurons protocol.</P><H2>10.03.2018</H2><P><B> Staining of neuronal culture in 96-well plate</B></P><P>PBS 1X is drained from the plate. Cells are first permeabilized by adding 100 <FONT face="Raleway">μ</FONT>L Triton X-100 at 0.5% for 10 min at room temperature and unspecific sites are saturated using 100 <FONT face="Raleway">μ</FONT>L Bovine serum albumin (BSA) 1% for 30 min at room temperature. Cells are stained in 50 <FONT face="Raleway">μ</FONT>L of a mix of DAPI 1/20 000, anti MAP2 antibody coupled with Alexa Fluor 555nm 1/500 and Anti-<FONT face="Raleway">β</FONT>-tubulin antibody coupled with Alexa Fluor 448nm 1/500 for 15 min at room temperature. </P><P>Antibody mix is drained, and cells are rinsed one time in 100 <FONT face="Raleway">μ</FONT>L of PBS 1X and stored in 50 <FONT face="Raleway">μ</FONT>L PBS 1X in the dark at 4°C. </P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Fixation and Staining Neurons protocol.</P></DIV><DIV data-description="Week_40" class="panel" id="pan_4002" style="text-align:left;"><H2> 10.04.2018  </H2><P>  Liquid culture of BL21. Biofilm culture on 6 PDMS well chips (membranes:  2 gold, 1 alumina oxide, 1 PEDOT:PSS, 1 PEDOT:Cl and 1 PEDOT:Ts). </P><P> (Results on the 10.16.2018) </P><P> Biofilm culture protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P><H2> 10.05.2018  </H2><P>  Biofilm growth measurement via measurement of the optical density (see protocol below). Liquid culture of BL21 and biofilm culture on 6 PDMS well chips.   </P><P> (Results on the 10.16.2018) </P><P> Membrane microchannel chip bacteria confinement experiment. Optical density measure of a sample of liquid culture. Liquid culture was poured in a microchannel chip on one side, and optical density of the liquid that flowed to the other side of the microchannnels was measured. 
                                        </P><P><B> Results </B></P><P> Liquid culture OD: 0.44 </P><P> OD of liquid after flowing through the chip: 0.41. </P><P><B> Interpretation</B></P><P> We expected a much lower OD after liquid flow through the chip, so this suggests the presence of a leak in the chip, that allows the liquid culture to flow without retaining the bacteria. The decision was taken to investigate further and to observe the flow under a microscope. </P><P> Biofilm culture and growth measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_41"><DIV class="block full" id="Week_41" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_41" class="vignette" id="vign_4100"><DIV data-description="Week_41" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_41" class="vignette" id="vign_4101" style="display:none"><DIV data-description="Week_41" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_41" class="vignette" id="vign_4102"><DIV data-description="Week_41" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_41" class="panel" id="pan_4100" style="text-align:left;"><H2>10.08.2018</H2><P><B>Batch purification on Ni-NTA beads of His-tagged protein under denaturing conditions</B>
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the batch purification protocol)

Urea 8M 
Ni-NTA beads (200 µl)<U>Aim</U>: Purify the NGF under denaturing condition to know if the His tag is accessible for NiNTA purification. We already tried under native and partial denaturing condition but the protein was not retained on beads. 

Material:<UL style="text-align: left; list-style: disc;font-size: 16px;"><LI>200 µl Ni-NTA beads (100 µl beads)</LI><LI>Buffer Equilibration (Tris 20 mM, pH 8, NaCl 200 mM, Urea 8M)</LI><LI>Buffer Wash (Tris 20 mM, pH 8, NaCl 200 mM, Urea 8M, Imidazole 20 mM)</LI><LI>Buffer Elution (Tris 20 mM, pH 8, NaCl 200 mM, Urea 8M, Imidazole 250 mM)</LI></UL></P><P>
Under denaturing conditions work at room temperature (RT).

Ni-NTA beads were equilibrated with 1 ml of Equilibration buffer and centrifuged for 1 min at 3500 x g, two times. One volume of protein solution was added and allowed to bind for 1 hour at RT. Beads were centrifuged for 1 min, centrifuged at 3 500 x g and an aliquot was taken. Beads were washed 4 times with 4 volumes of buffer containing 20 mM Imidazole. Two volumes of elution buffer (Imidazole 250 mM) was added and allowed to stand for 10 min. Centrifugation allowed to get back the supernatant containing purified proteins. </P><H2>10.09.2018</H2><P><B>HIS-TAGGED NGF</B> 
(See <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:black; text-decoration:none;" target="_blank">here</A> the Immunodetection on membrane protocol)</P>
After protein transfer on nitrocellulose membrane on the 10.08.2018, protein identification was performed by immunodetection using His-tag antibodies.<FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 1: </B>Protein identification by immunodetection using His-tag antibodies. </FIGCAPTION></FIGURE><P>
Two bands (red arrows) seems to be the proNGF protein. The upper one corresponds to the non-cleaved form and the lower one either HlyA cleaved of protein degradation.</P><P><B><I>S.aureus</I> BIOFILM FORMATION </B>
3 biological replicates of <I>S. aureus</I>. </P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 2: Deposit plan of the 96-well plate</B></FIGCAPTION></FIGURE><P><UL style="text-align: left; list-style: disc;font-size: 16px;"><LI><I>S. aureus</I> cultures were diluted 1/100 in each well. </LI><LI>SA+ PS BL21-RIP 5: Pellet supernatant non-induced 5 µL/well</LI><LI>SA+ PS BL21-RIP 1 mM 5: Pellet supernatant induced 1 mM IPTG 5 µL/well</LI><LI>SA+ PS BL21-RIP 1 mM 10: Pellet supernatant induced 1 mM IPTG 10 µL/well</LI><LI>SA+ PS BL21-RIP 1 mM 20: Pellet supernatant induced 1 mM IPTG 20 µL/well</LI></UL></P><P>

5 identical plates are incubated at 37°C for 24h, 48h. </P><H2>10.10.2018</H2><P><B>S.aureus biofilm formation 96-wells plates</B>
After 24 hours of incubation, 4 plates were tested for Biofilm formation. Supernatant was removed, and biofilm were fixed with Bouin’s Solution during 20 min. Then plates were washed 3 times by flipping and stained with Crystal Violet 0.1% during 15 min. Excess of staining was removed with 4 wash by flipping. Biofilm were resuspended in 30% acetic acid solution under shaking conditions and absorbance was read at 590 nm. 

Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Results </FIGCAPTION></FIGURE><H2>10.11.2018</H2><P><B>S.aureus biofilm formation 96-wells plate</B>
After 48 hours of incubation, 4 plates were tested for Biofilm formation. Supernatant was removed, and biofilm were fixed with Bouin’s Solution during 20 min. Then plates were washed 3 times by flipping. Two plates were stained with Crystal Violet 0.1% during 15 min. Excess of staining was removed with 4 wash by flipping. Biofilms were resuspended in 30% acetic acid solution under shaking conditions and absorbance was read at 590 nm. For the two other plates, biofilms were resuspended in PBS 1X and absorbance was read at 600 nm and Fluorescence at 485/510. 

Results:</P><FIGURE style="display:inline-block"><FIGCAPTION><B>Figure 3: </B>Results </FIGCAPTION></FIGURE></DIV><DIV data-description="Week_41" class="panel" id="pan_4102" style="text-align:left;"><H2> 10.08.2018 </H2><P> Liquid culture of BL21. Biofilm culture on 6 PDMS well chips </P><P> Biofilm growth measurement. </P><P>(Results on the 10.16.2018) </P><P> Biofilm culture and growth measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P><H2> 10.11.2018 </H2><P> Liquid culture of BL21. Biofilm culture on 6 PDMS well chips </P><P> Biofilm growth measurement. </P><P> (Results on the 10.16.2018) </P><P> Biofilm culture and growth measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P><H2> 10.12.2018 </H2><P> Liquid culture of BL21. Biofilm culture on 6 PDMS well chips </P><P> Biofilm growth measurement. </P><P> Measurement of the electrical resistance of the membranes with the biofilm. </P><P> (Results on the 10.16.2018) </P><P> Biofilm culture and growth measurement protocol, conductivity measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P></DIV></DIV></DIV><DIV data-description="Week_42"><DIV class="block full" id="Week_42" style="display:flex;flex-flow: row wrap;justify-content:center;margin:auto;"><DIV data-description="Week_42" class="vignette" id="vign_4200"><DIV data-description="Week_42" class="vignette_text"><P>Bacteriology</P></DIV></DIV><DIV data-description="Week_42" class="vignette" id="vign_4201"><DIV data-description="Week_42" class="vignette_text"><P>Cell culture</P></DIV></DIV><DIV data-description="Week_42" class="vignette" id="vign_4202"><DIV data-description="Week_42" class="vignette_text"><P>Microfluidics/Membrane</P></DIV></DIV><DIV data-description="Week_42" class="panel" id="pan_4201" style="text-align:left;"><H2>10.15.2018</H2><P><B>Cell culture in 96-well plate</B></P><P>
In a 96-well plate
-Add 50 <FONT face="Raleway">μ</FONT>L Poly-L-ornithine 0.01% solution (Sigma Aldrich, ref: P4957) per well

-Incubate the plate 2h at 37°C

-Wash 3 times with 50 <FONT face="Raleway">μ</FONT>L distilled water per well

-Add 50 <FONT face="Raleway">μ</FONT>L laminin per well

-Incubate the plate 1h at 37°C

-Prepare 10mL of culture medium at 37°C: 9,7mL neurobasal, 100 <FONT face="Raleway">μ</FONT>L Glutamax, 200 <FONT face="Raleway">μ</FONT>L B27, 100 <FONT face="Raleway">μ</FONT>L P/S


-Dissociate cortex cells by adding 250 <FONT face="Raleway">μ</FONT>L of Trypsin 2.5% and DNAse 1000 U 

-Incubate 5 min at 37°C

-Rinse three times the cells using 5 mL warm CMF-HBSS

-Centrifugate at 1200 rpm for 4min
 
-Remove the supernatant (leave about 100 <FONT face="Raleway">μ</FONT>L of medium) 

-Resuspend the pellet by adding 1 mL of culture medium 

-  x 10 two-way trip with a sterile glass Pasteur pipette with large tip

-  x 10 two-way trip with a sterile glass Pasteur pipette with small tip

- Centrifuge at 200 rmp for 1min

-Keep the supernatant in a sterile Falcon 

-Count the cells using a Neubauer chamber (20 <FONT face="Raleway">μ</FONT>L cells and 80 <FONT face="Raleway">μ</FONT>L Trypan Blue) and dilute in order to obtain a final concentration of 30 000 cells/1.7 <FONT face="Raleway">μ</FONT>L


-Remove the laminin

-Add 100 <FONT face="Raleway">μ</FONT>L of culture per well

-Incubate at 37°C, 5% CO2
</P><H2>10.16.2018</H2><P><B> Observation of the seeding realized on 10.15.2018</B></P><FIGURE style="display:inline-block"><FIGCAPTION><I>Cortical cells 24 hours after seeding, 10X</I></FIGCAPTION></FIGURE><P><I><B>Interpretations: </B></I>the cells adhered and grown normally.</P><P><B> Addition of NGF (commercial and lysate) in the 96-well plate</B></P><P><I>Denaturation of lysate supernatant NGF</I>
1. Defroze the tubes in ice 
2. Incubate at 60°C for 5 min </P><P><I>Addition</I>
1. Add the appropriate volume of each condition in each well (see table below) 
2. Incubate at 37°C, 5% CO2 </P><P>Guide plate:</P><H2> 10.17.2018 </H2><P><B>Fixation of neuronal culture inside 96-well plate</B></P><P>Media is drained from the plate and cells are washed 3 times with PBS 1X. Cells are fixed using 100 <FONT face="Raleway">μ</FONT>L paraformaldehyde (PFA) at 4% for 15 min at room temperature. PBS 1X is added, cells are conserved at 4°C for the night. </P><P><B>Staining of neuronal culture in 96-well plate</B></P><P>PBS 1X is drained from the plate. Cells are first permeabilized by adding 100 <FONT face="Raleway">μ</FONT>L Triton X-100 at 0.5% for 10 min at room temperature and unspecific sites are saturated using 100 <FONT face="Raleway">μ</FONT>L Bovine serum albumin (BSA) 1% for 30 min at room temperature. Cells are stained in 50 <FONT face="Raleway">μ</FONT>L of a mix of DAPI 1/20 000, anti MAP2 antibody coupled with Alexa Fluor 555nm 1/500 and Anti-<FONT face="Raleway">β</FONT>-tubulin antibody coupled with Alexa Fluor 448nm 1/500 for 15 min at room temperature. </P><P>Antibody mix is drained, and cells are rinsed one time in 100 <FONT face="Raleway">μ</FONT>L of PBS 1X and stored in 50 <FONT face="Raleway">μ</FONT>L PBS 1X in the dark at 4°C. </P><P>See <A href="https://static.igem.org/mediawiki/2018/6/60/T--Pasteur_Paris--Cell-Culture.pdf" style="font-weight:bold; color:black; text-decoration:none; " target="_blank">here</A> the Fixation and Staining Neurons protocol.</P></DIV><DIV data-description="Week_42" class="panel" id="pan_4202" style="text-align:left;"><H2>  10.15.2018 </H2><P> Liquid culture of BL21. Biofilm culture on 6 PDMS well chips </P><P> Biofilm growth measurement. </P><P>  Measurement of the electrical resistance of the membranes with the biofilm. </P><P> (Results on the 10.16.2018) </P><P>  Measurement of the membrane's electrical resistance without biofilm. </P><P><B> Results: membrane conductivity </B></P><P> Function generator was set on square at 200 Hz.  The physical quantities measured are Eg, the generator tension amplitude, and Ep, the amplitude of the voltage difference between a point on the membrane inside the well and the extremity of the platinium strip outside the well. Tension amplitude of the resistor is given by Er = Eg - Ep. Current flowing through the electric circuit is calculated with I = Er/R. Conductivity of the membrane is given by I/Ep. Conductivity of each membrane was measured 3 times. Figure 1 shows the mean value for each membrane and the standard deviation. </P><P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 1: Membrane conductivity</FIGCAPTION></FIGURE></P><P><B> Interpretation </B></P><P> Bare alumina oxide and PEDOT:PSS-coated membranes show similar conductivies, indicating the incomplete coating of PEDOT:PSS on alumina oxide membranes. On the opposite, PEDOT:Cl and PEDOT:Ts seem to exhibit on average better conductivities, but in the same time, the coating of these membranes revealed by electron microscopy seemed to have covered the alumina oxide membranes in a more uniform way, assuring enhanced conductive capabilities. These results can be criticized because of the high deviation and because the membranes conductivity was measured after several biofilms were grown on them, which may have affected the measurements. </P><P> Biofilm culture and growth measurement protocol, conductivity measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P><H2>  10.16.2018 </H2><P> Measurement of the electrical resistance of the membranes with the biofilm.</P><P> Second try of the confinement experiment of bacteria with a membrane microchannel chip.  A few drops of RFP expressing DH5alpha <I> E. coli </I> liquid culture were poured in a membrane microchannel chip. Chip was then observed under a microscope. 
</P><P><B> Results: confinement experiment </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 2: Second confinement test</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Membrane is located on the right side, and liquid culture was poured on that side, before the membrane. Bacteria was still able to flow to the left side, but they were not following the microchannels, instead they were just flowing in a single direction, suggesting the membrane lifts the microfluidic chip from below and thus causes massive leakings in the microfluidic circuitry. We concluded that the microchannel chips we designed didn't really meet the requirements they were designed for. </P><P><B> Results: biofilm growth </B></P><P>  Biofilm growth was measured 4 times in total. For each series of measure, the measured optical densities were divided by the optical density of the base liquid culture, to normalize the measures. Figure 2 shows the mean value for each type of membrane and the standard deviation.  </P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 3: Biofilm growth</FIGCAPTION></FIGURE><P><B> Results: biofilm conductivity </B></P><P>For conductivity measurements, function generator was set on square at 200 Hz. The physical quantities measured are Eg, the generator tension amplitude, and Ep, the amplitude of the voltage difference between a point on the biofilm inside the well and the extremity of the platinium strip outside the well. Tension amplitude of the resistor is given by Er = Eg - Ep. Current flowing through the electric circuit is calculated with I = Er/R. Conductivity of the membrane is given by I/Ep. Conductivity of each membrane with a biofilm was repeated 3 times. Figure 4 shows the average conductivity of the membrane with the biofilm, and the standard deviation. </P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 4: Membrane conductivity with biofilm</FIGCAPTION></FIGURE><P>  To approximate very roughly the conductivity of the biofilm, the average conductivity values of the membranes with a biofilm were divided by the corresponding average biofilm growth values, and the conductivity of the membrane was then substracted. Figure 4 shows the estimated biofilm conductivity.  </P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 5: Biofilm conductivity</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> As told by the membrane manufacturer, biofilm formation on gold membranes seems indeed to be more difficult than on other membranes. However we expected PEDOT:PSS-coated membranes to stimulate more the growth of biofilm, but perhaps that may be just another indicator of the incomplete coating. Surprisingly, PEDOT:Cl tends to allow better formation of biofilms. We realized only after the experiments the need for a control biofilm culture without membrane. </P><P> Conductivity with a biofilm is better with gold membranes, although the conductivity of gold membranes themselves isn't the best. This may be explained by the fact, that because of the thinner biofilm formation on gold membranes, the electrical wires touched not only the biofilm, but also the membrane, bypassing the biofilm and leading to imprecise measurements.
                                        </P><P> Approximate biofilm conductivity is therefore probably wrong for the gold membrane. However, it is interesting to notice that the biofilm conductivity measured for the bare alumina oxyde, PEDOT:Ts-coated one and PEDOT:PSS-coated give more or less the same value, suggesting that with more measurements, adapted equipement and better methods it would be indeed possible to measure the biofilm's conductivity with our PDMS well chips. </P><P> Biofilm culture and growth measurement protocol, conductivity measurement protocol <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf" target="_blank"> here </A>. </P><P> Liquid culture protocol <A href="https://static.igem.org/mediawiki/2018/4/42/T--Pasteur_Paris--DNA_assembly.pdf" target="_blank"> here </A>. </P><H2> 10.17.2018 </H2><P> Third and last try of the confinement experiment of bacteria with a membrane microchannel chip.  A few drops of <I> E. coli </I> liquid culture were poured in a membrane microchannel chip. Chip was then observed under a microscope. 
</P><P><B> Results: confinement experiment </B></P><FIGURE style="display:inline-block"><FIGCAPTION>Figure 6: Third confinement test</FIGCAPTION></FIGURE><P><B> Interpretation </B></P><P> Membrane is located on the left side, and liquid culture was poured on that side, before the membrane. Bacteria this time wasn't able to flow to the right side, the membrane stopped their progression. It is clear on figure 6, that the left side is crowded with bacteria, and the right side is empty (apart from a few PDMS impurities). Final conclusion on the membrane microchannel chips is, that although the integration method of the membrane filter in the chip is complicated and a bit improvised, some chips apparently do fulfill their purpose, demonstrating this way the confinement of the bacteria with a membrane.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>