<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="Menu"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="project"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">Human Practices</A></LI><LI id="design"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Product Design</A></LI><LI id="parts"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">labwork</A></LI><LI id="achievements"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About Us</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Description">Overview</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Growing_Back_Nerves">Reconnect Nerves</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Fighting">Fight Infections</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Model">Model</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Membrane">Physical Interface</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Kill">Kill-Switch</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Entrepreneurship">Entrepreneurship</A></LI></UL><UL id="hp-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Human_Practices">Integrated HP</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Public_Engagement">Education &amp; Engagement</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Safety">Safety &amp; Ethics</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Law">Law</A></LI></UL><UL id="design-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Technicals">Methodology</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Scenario">Scenario</A></LI></UL><UL id="parts-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Basic_Part">Basic Parts</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Improve">Improve</A></LI></UL><UL id="labwork-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Notebook">Notebook</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Experiments">Protocols</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Results">Results</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/InterLab">InterLab</A></LI></UL><UL id="achievements-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Medals">Medals</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Awards">Awards</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Demonstrate">Demonstrate</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/GiantJamboree">Giant Jamboree</A></LI></UL><UL id="aboutus-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Team">Team Members</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Acknowledgements"> Acknowledgements</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Attributions">Attributions</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2018.igem.org/Team:Pasteur_Paris">Home</A></LI><LI id="project_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">HP</A></LI><LI id="design_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Design</A></LI><LI id="parts_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">Labwork</A></LI><LI id="achievements_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About us</A></LI></UL><A href="#" id="pull">Menu</A></DIV></DIV><DIV id="GeneralContent"><DIV id="index" class="block"><DIV id="indexContent"><P><A href="#Reconnect" class="link">Reconnect Nerves</A></P><P><A href="#Cell" class="link">Cell culture</A></P><P><A href="#Fight" class="link">Fight Infections</A></P><P><A href="#Kill" class="link">Kill Switch</A></P><P><A href="#Membrane" class="link">Membrane</A></P></DIV></DIV><DIV id="MainContent"><DIV class="block full bothContent"><DIV class="block dropDown" id="Reconnect"><H4>RECONNECT NERVES: DNA ASSEMBLY<I>Click to see more</I></H4></DIV><DIV class="block hiddenContent"><DIV class="block title"><H1>RECONNECT NERVES</H1></DIV><DIV class="block full"><H2>NGF Secretion <A href="http://parts.igem.org/Part:BBa_K2616000"> BBa_K2616000</A></H2><H4 style="text-align: left;">DNA assembly</H4><P>The <B>sequence</B> we designed codes for two different proteins: <B>proNGF (Nerve Growth Factor)</B> and <B>TEV protease</B> (from Tobacco Etch Virus). These two proteins are fused in C-terminal with a signal peptide for <I>Escherichia coli</I> Type I Secretion System which consists in the last 60 amino-acids of HaemolysinA (<B>HlyA</B>). Each coding sequence is separated from the signal peptide by the cleavage sequence for TEV, in order to get the protein without its signal peptide (Figure 1).</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 1: </B>proNGF and TEV production cassette </DIV></DIV><DIV class="block full"><P>This DNA construct was ordered in two parts, named Seq1 (1096 bp) and Seq2 (1153 bp) in commercial plasmids pEX-A258 from gene synthesis. Seq1 and Seq2 were amplified in competent <I>E. coli</I> DH5-<FONT face="Raleway">α</FONT>.   After bacteria culture and plasmid DNA extraction, we digested commercial vectors with restriction enzymes (<B>NheI</B> and <B>BamHI</B> for Seq1, <B>MscI</B> and <B>HindIII</B> for Seq2). We extracted the inserts from the gel and performed a ligation by using specific overlaps into <B>linearized pET43.1a</B> for proNGF expression and into <B>pSB1C3</B> for iGEM sample submission.We proved that our vector pet43.1a contained Seq1 and Seq2 (Figure 2) and that pSB1C3 contained Seq1 and Seq2 (Figure 3) after digestion and DNA electrophoresis. Plasmid DNA of pSB1C3 construction was purified and sent for sequencing (Figure 4).</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 2: </B>Agarose 1% gel after electrophoresis of digested pET43.1 containing Seq1 and Seq2 (Bba_K2616000) with NdeI. Colonies 6, 9, 10 ,11, 12 and 15 have the correct construction.</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 3: </B> Agarose 1% gel after electrophoresis of digested pSB1C3 containing Seq1 and Seq2 (Bba_K2616000) with EcoRI/PstI. Colonies 3, 7 and 8 have the correct construction.</DIV></DIV><DIV class="block full"><P><B>Sequencing</B> results, when aligned with our original construct using Geneious confirmed that pSB1C3 contained Seq1 and Seq2, <B><A href="">BBa_K2616000 </A></B>. </P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 4: </B> Alignment of sequencing results for BBa_K2616000. Sequencing perform in pSB1C3 and three primers were designed (FOR1, FOR2, FOR3) to cover the whole sequence. Image from Geneious. </DIV></DIV><DIV class="block full"><P>The construction was successfully assembled. In Figure 4, we show that we used three different primers, allowing us to cover the whole sequence without mistakes. As visible, the mismatches are only present at the extremities of each primer sequencing. The final basepair identity is 100%.</P></DIV><DIV class="block full"><H4 style="text-align: left;">proNGF characterization and purification</H4><P> Our chassis is <B><I>Escherichia coli</I> BL21(DE3) pLysS</B>, a specific strain dedicated to producing high amounts of desired proteins under a T7 promoter. Thus, we co-transformed our bacteria with <B><A href="">BBa_K2616000 </A></B> and pVDL 9.3, generously provided by Dr. Victor de Lorenzo, from Centro Nacional de Biotecnología of Madrid, bearing HlyB and HlyD (Type I secretion system) sequences, in order to get a chance to secrete proNGF out of the cell.
Bacteria were grown at a large scale (800 mL), and proNGF expression was induced with 0.1 mM IPTG for 2 hours at 37°C. 
We tried to achieve His-tagged proNGF purification using a single step <B>Ni-NTA affinity purification column</B>. We eluted our protein using a gradient of imidazole-containing buffer and one peak was detected (fraction A6).</P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 5: </B>FPLC proNGF purification with ÄKTA pure (General Electric) Ni-NTA column was equilibrated with buffer A (50 mM Tris, pH 7.4, 200 mM NaCl). Supernatant of lyzed bacteria was introduced through the column. Washing with 5% of buffer B. Elution by buffer B gradient (buffer A + imidazole 250 mM). UV absorbance at 280nm is shown in blue, conductivity in red, and concentration of buffer B in green. </DIV></DIV><DIV class="block full"><P>We analyzed bacterial lysate and purification fractions using SDS-PAGE electrophoresis and Mass spectrometry. </P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 6: </B>SDS-PAGE gel Bis-Tris 4-12% of bacterial lysate and proNGF purification fraction by SDS-PAGE. 
 </DIV></DIV><DIV class="block half"><P> The proNGF purification using a single step Ni-NTA column was not conclusive. Many proteins were found on elution fractions. His-tagged proNGF fused to HlyA export signal should be found at 33 kDa while the proNGF cleaved by TEV protease should be found at 27 kDa. We finally analyzed five gel slices around 20 to 35 kDa of the FPLC flow-through (lane 2, Figure 6)  by LC/MS/MS mass spectrometry, to verify the presence of proNGF.</P></DIV><DIV class="block half"><P>With the LC/MS/MS analysis, 14 coverage unique peptides corresponding to proNGF were found in all fractions. The sequence coverage represents 63%.<B> Results of mass spectrometry analysis demonstrate the expression of proNGF</B>. 

According to Figure 7, proNGF pattern are found on each fraction sent to mass spectrometry. The major amount is found on fraction 5, corresponding to 33 kDa. At this molecular weight, the proNGF is still fused to the signal peptide. However these results are also consistent with a mix of cleaved and uncleaved proNGF. The TEV protease, 34 kDa fused to export singal and 28 kDa cleaved from the export signal are found.  </P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 7: </B>Distribution of matching peptides of proNGF and TEV protease by gel fractions after mass spectrometry analysis. </DIV></DIV><DIV class="block full"><P>Analysis of Fraction 5 of the gel shows that our protein proNGF is present in a mix of cleaved and uncleaved polypeptide (Figure 8). Mass spectrometry spectrum of Peptide A, IDTACVCVLSR, from proNGF sequence and peptide B IISAAGSFDVKEER from fused HlyA export signal are shown in Figure 9. The presence of mass spectrometry identified peptides corresponding to the fusion of proNGF and HlyA indicate some proNGF uncleaved from the export signal.</P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 8: </B>Alignment sequences of proNGF fused to HlyA export signal and peptides identified by mass spectrometry. In light blue peptides that match proNGF amino acids sequence. In light yellow, peptides that match HlyA export signal. Sequence has been annotated to match corresponding protein amino acid sequences : In orange His tagged proNGF, in red TEV protease cleaving site, in pink HlyA export signal.</DIV></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 9: </B>Mass spectrometry spectrum. A) Peptide identified corresponding to proNGF. B) Peptide identified corresponding to the fusion of proNGF and HlyA export signal. </DIV></DIV><DIV class="block full"><P>The proNGF did not seem to be retained on the Ni-NTA affinity column, although in fraction A6 we also identified His-tag bound proNGF. To test if the His-tag is accessible for binding to Ni-NTA, we've performed a batch purification using Ni-NTA beads under native and partial denaturing conditions (Urea 2 M) followed by Western Blot analysis with immunodetection through Anti-His Antibodies Alexa Fluor 647 (Figure 10). Detection of His-tag in the pellet supernatant of induced BL21(DE3) pLysS with 1 mM IPTG and flow through when partially denatured.</P><P> Native His-tagged proNGF was not retained on Ni-NTA beads. We believe that the N-terminal His-tag may be hidden in the protein fold. Consequently, we denatured with 2M urea before purifying on the beads. As seen in lane 8 even 2M urea could not improve the binding. We also tried with an 8M urea concentration, without better results. 
</P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 10: </B>Western Blot analysis of batch purification of proNGF under native and partial denaturing conditions. </DIV></DIV></DIV><DIV class="block full" style="background-color: transparent;"><DIV class="block title"><H3>Summary</H3></DIV><P><I>Achievements: </I><UL style="text-align: left;list-style: disc;"><LI>Successfully cloned a biobrick coding for secretion of NGF in pET43.1a and iGEM plasmid backbone pSB1C3, creating a new part <A href="http://parts.igem.org/Part:BBa_K2616000" style="font-weight: bold ; color:#85196a;" target="__blank"> BBa_K2616000</A>. </LI><LI>Successfully sequenced <A href="http://parts.igem.org/Part:BBa_K2616000" style="font-weight: bold ; color:#85196a;" target="__blank"> BBa_K2616000</A> BBa_K2616000 in pSB1C3 and sent to iGEM registry. </LI><LI>Successfully co-transformed <I>E. coli</I> with plasmid secreting proNGF and plasmid expressing the secretion system, creating bacteria <B>capable of secreting NGF</B> in the medium.</LI><LI>Successfully characterized production of proNGF thanks to mass spectrometry and western blot.</LI></UL></P><P><I>Next steps:</I><UL style="text-align: left;list-style: disc;"><LI><B>Purify</B> secreted proNGF, and characterize its effects on neuron growth thanks to our microfluidic device. </LI></UL></P></DIV></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Cell"><H4>RECONNECT NERVES: CELL CULTURE<I>Click to see more</I></H4></DIV><DIV class="block hiddenContent"><DIV class="block title"><H1>CELL CULTURE</H1></DIV><DIV class="block full"><H3 style="text-align: left;">Neuron culture</H3><P><I>Imaging was performed in collaboration with the BioImagerie Photonique platform of the Institut Pasteur. Data are presented as MEAN ± SEM. Significance between 2 different groups was determined using an Ordinary one-way ANOVA test on the software Prism6 (GraphPad). (ns: non-significant, *: p&lt;0.05, **: p&lt;0.01, ***: p&lt;0.001, ****: p&lt;0.0001) </I></P><P>As an alternative to our recombinant proNGF for control experiments, we performed an <I>in vitro</I> neural primary culture with commercial NGF. For this, a pair of E18 Sprague Dawley cortexes were purchased from BrainBits.co.uk. We digested the tissue with manufacturer provided papain according to their protocol and seeded 40 000 dissociated neurons on our microfluidic chips with different conditions of culture for six days at 37°C, and 5% CO2.  </P><P>On our two-chamber microfluidic devices, we seeded neurons only on one side.  Fifteen chips were used in total. After six days, neurons are fixed with paraformaldehyde (PFA) 4% and stained with DAPI. For differentiated markers: MAP2 (coupled with Alexa Fluor 555), a cytoskeletal associated protein and Beta-III Tubulin (coupled with Alexa Fluor 488), one of the major components of microtubules and a neuron-specific marker were used.</P><P>We can see in Figure 11 that we had contaminations on many of our microfluidic chips because we could not use antibiotic selection otherwise our bacteria would have suffered from it, and that most of our experiments could not be analyzed. However, no contaminations were apparent for eight of them. Two of these successful ones are displayed in Figure 12. </P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 11: </B> In orange, are displayed bacteria found inside one of our microfluidic devices.</DIV></DIV><DIV class="block full"><DIV class="legend"><B>Figure 12: </B> Sprague Dawley E18 cortex neurons after six days of incubation at 37°C, and 5% CO2. Blue: DAPI stained nuclei, Green: Anti-Beta-III Tubulin coupled to Alexa Fluor 488, Yellow: Co-localization of anti-Beta-III Tubulin and MAP2.  <B>(A)</B> Neurons were put in culture in Neurobasal, B27, GlutaMAX medium. <B>(B)</B> Neurons were put in culture in DMEM FBS 10% medium. </DIV></DIV><DIV class="block full"><P>As we can see, we succeeded in growing the cells inside our device in the presence of Neurobasal, B27 and GlutaMAX medium. It is possible to see neurons passing through one chamber to the other in this experiment. Unfortunately, the PDMS of the microfluidic chips detached from the bottom of the glass culture dish, leading to the growth of cells not inside of the microchannels, but below them (Figure 13). </P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 13: </B> PDMS detachment from the glass bottom culture dish, we can observe the axonal development under the microchannels. Neurons were put in culture with Neurobasal, B27, GlutaMAX, and commercial NGF 50 ng/mL.</DIV></DIV><DIV class="block full"><P>We also tested the action of commercial NGF on our culture. Neurons were put in culture in the presence of commercial NGF at different concentrations: 50 ng/mL, 250 ng/mL, 500 ng/mL, 750 ng/mL and 900 ng/mL. The optimal concentration was determined by modeling of NGF diffusion inside the medium. It was possible to capture the cells passing through one of the chambers of the microfluidic chip to the other side during a time-lapsed using phase-contrast microscopy recorded for the first 48h of culture at the <I>BioImagerie Photonique </I> platform, proving that our device was working as expected (Video 1).</P></DIV><DIV class="block full"><VIDEO width="70%" height="auto" controls=""><SOURCE src="https://static.igem.org/mediawiki/2018/b/b9/T--Pasteur_Paris--vidéo.mp4" type="video/mp4">
Your browser does not support the video tag.
                    </SOURCE></VIDEO></DIV><DIV class="legend"><B>Video 1: </B> A video excerpt of a 48h time-lapsed in phase contrast. Neuron entering the microchannel are visible. Medium of culture: Neurobasal, B27, GlutaMAX and commercial NGF at a concentration of 50 ng/mL.   </DIV><DIV class="block full"><P>Because we were running out of fresh microfluidic chips, and since we already proved that our device was working as expected, we switched and put our next cultures in a 96-well plate for 10 days at 37°C, testing the influence of the different concentrations of NGF on the growth of the cells (Figure 14).  </P></DIV><DIV class="legend"><B>Figure 14: </B> Best representations of the imaging of the 96-well plate for each condition. A2: 0 ng/mL commercial NGF, B3: 50 ng/mL commercial NGF, C4: 250 ng/mL commercial NGF, D4: 500 ng/mL commercial NGF, E4: 750 ng/mL commercial NGF, F4: 900 ng/mL commercial NGF. </DIV><DIV class="block full"><P>Images were analyzed on ImageJ, and following results are shown in Figure 15. </P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 15: </B><B>(A)</B> Percentage area of <FONT face="raleway">β</FONT>-III Tubulin in each well and <B>(B)</B> number of stained nuclei in each well with no NGF, 50 ng/mL, 250 ng/mL, 500 ng/mL, 750 ng/mL and 900 ng/mL of commercial NGF added in our medium Neurobasal, B27, GlutaMAX. Each condition was compared to the control group without NGF. <I>(ns: non-significant, *: p&lt;0.05, **: p&lt;0.01, ***: p&lt;0.001, ****: p&lt;0.0001).</I></DIV></DIV><DIV class="block full"><P>As we can see in Figure 15 <B>(A)</B>, it was possible to observe a difference in the percentage of area taken by the <FONT face="raleway">β</FONT>-III Tubulin. Indeed, it is possible to observe a significant increase in this percentage when commercial NGF is put at a concentration of 250 ng/mL or higher. The concentration of NGF seems to influence the growth of axons. It was possible to observe the same significant increase of cell number at a concentration of 250 ng/mL or higher (Figure 15 <B>B</B>). We can also see that at a concentration of 900 ng/mL, it seems that both the percentage of area taken by the <FONT face="raleway">β</FONT>-III Tubulin and the number of cells decrease, even though it is not significant, we can hypothesize that at a high concentration of NGF, the receptor p75<SUP>NTR</SUP> is getting internalized, resulting in a decreasing number of available receptors. </P><P>It seems that commercial NGF has a dose-response effect on both the growth of neuronal axons and / or the survival of the cells. To determine in which category the NGF was affecting, we standardize the percentage area of <FONT face="raleway">β</FONT>-III Tubulin compared to the number of cells. </P></DIV><DIV class="legend"><B>Figure 16:</B> Ratio of the percentage area of <FONT face="raleway">β</FONT>-III Tubulin on the number of stained nucleus. <I>(ns: non-significant, * : p&lt;0.05, ** : p&lt;0.01, *** : p&lt;0.001, **** : p&lt;0.0001).</I></DIV><DIV class="block full"><P>As we can see in figure 16, we have a decreasing amount of <FONT face="raleway">β</FONT>-III Tubulin per nuclei each time the concentration of NGF gets higher. We can see a significant decrease of this ratio when the NGF is at 500 ng/mL and higher, which is not an expected result and an opposite result from the images that we occurred from the platform. </P><P>We assumed from the start that all of our cells put in culture were neuronal cells, which might not be the case. We know that the NGF has an effect of the survival of the cells <SUP>[1], [2]</SUP> (Figure 15 <B>B</B>). We did not have the suitable marker to differentiate the neuronal cells from the other types of cells, and should have stained the cells with NeuN, a neuronal nuclear antigen used as a biomarker for neurons. Therefore, the standardization we did with the number of cells is not an accurate one. We can still appreciate the qualitative results we had (Figure 14 and 15 <B>A</B>) and are positive on the effect NGF has on axon’s growth as well as cell survival.</P><P>After having collected the data on the effect of commercial NGF, we decided to put in culture our cells in the presence of our bacterial lysate to test the effect of our proNGF ( produced with <A href="http://parts.igem.org/Part:BBa_K2616000"> Bba_K2616000 </A> ). We put in culture for 2 days 30 000 cells with or without commercial NGF at 500 ng/mL and 900 ng/mL as well as our bacterial lysate in different dilutions. Since we wanted to inactivate as much bacterial proteins as possible (endotoxins), we checked the denaturation temperature for our proNGF, 70°C, and heat-inactivated the lysate at 60°C for 5 minutes before putting it in culture. Due to lack of time, only one well per condition was analyzed. </P></DIV><DIV class="legend"><B> Figure 17:</B><B>(A)</B> Percentage area of <FONT face="raleway">β</FONT>-III Tubulin in each well and <B>(B)</B> percentage area of nucleus in each well with no commercial NGF, 500 ng/mL or 900 ng/mL or bacterial lysate at 1/5, 1/10, 1/20 or 1/30 added in our medium Neurobasal, B27, GlutaMAX. </DIV><DIV class="block full"><P>We can see in Figure 17 that our lysate seems to increase the percentage area of the <FONT face="raleway">β</FONT>-III Tubulin compared to the control without NGF. Our results with the commercial NGF seem to be equivalent to the results we had from our first experiment (Figure 15), with a decrease of axons at a concentration of 900 ng/mL. We can hypothesize that the lysate does have an effect on axon’s growth from the increasing percentage area of <FONT face="raleway">β</FONT>-III Tubulin, increase similar to the one we observe in our first experiment (Figure 15) and that the activity of our proNGF could be equivalent to commercial NGF with a concentration between 500 ng/mL and 900 ng/mL.</P><P>We also could see an influence of the commercial NGF on the survival of the cells, similar to our first experiment (Figure 15). Our lysate, put at a concentration of 1/10 and higher, seems to have the same effect (Figure 18). </P></DIV><DIV class="legend"><B>Figure 18: </B> Image of the whole well of the 96-well plate. Neurons were put in culture in Neurobasal, B27, GlutaMAX, and our lysate at a concentration of 1/10 medium.</DIV><DIV class="block full"><P>Of course, those data require further statistical tests, since we only had time to analyze one well per condition, and for only 2 days of culture due to French customs administrative delays that came with the order of the E18 cortex pair from the USA.  Still, in those 2 days of culture, we have been able to observe a difference in both the percentage area of <FONT face="raleway">β</FONT>-III Tubulin and nuclei counts. </P></DIV><DIV class="block title"><H1 id="References">REFERENCES</H1></DIV><DIV class="block full"><UL style="text-align: left;list-style: disc;"><LI style="list-style-type: decimal;">Matsumoto,  T.,  Numakawa,  T.,  Yokomaku,  D.,  Adachi,  N.,  Yamagishi,  S.,
Numakawa, Y., Kunugi, H., and Taguchi, T. (2006). <I>Brain-derived neurotrophicfactor-induced potentiation of glutamate and GABA release: Different dependency on signaling pathways and neuronal activity.</I>Mol. Cell. Neurosci. 31, 70–84 </LI><LI style="list-style-type: decimal;">Price, R. D., Yamaji, T., and Matsuoka, N. (2003). <I>FK506 potentiates NGF-induced neurite outgrowth via the Ras/Raf/MAP kinase pathway.</I> Br. J. Pharmacol.140,825–829. </LI><DIV class="block full" style="background-color: transparent;"><DIV class="block title"><H3>Summary</H3></DIV><P><I>Achievements: </I><UL style="text-align: left;list-style: disc;"><LI>Successfully <B>observed axon growth</B> in microfluidic chip in presence of commercial NGF.</LI><LI>Successfully observed <B>activity of our proNGF</B> in invitro cellular culture compared to commercial NGF with a concentration between 500 ng/mL and 900 ng/mL.</LI></UL></P><P><I>Next steps:</I><UL style="text-align: left;list-style: disc;"><LI><B>Statistical analysis</B> of our <I>in vitro</I> culture in presence of bacterial lysate. </LI><LI><B>Global proof of concept</B> in a microfluidic device containing neurons in one of the chamber, and our engineered bacteria in the other.</LI></UL></P></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Fight"><H4>FIGHT INFECTIONS<I>Click to see more </I></H4></DIV><DIV class="block hiddenContent"><DIV class="block title" id="Fight"><H1>FIGHT INFECTIONS</H1></DIV><DIV class="block full"><H2>RIP Secretion <A href="http://parts.igem.org/Part:BBa_K2616001"> BBa_K2616001</A></H2><P>The <B>sequence</B> we designed contains two <B>RIP (RNAIII Inhibiting Peptide)</B> sequences fused to two different export signal peptides for <I>E. coli</I> Type II Secretion System: <B>DsbA</B>  and <B>MalE</B>, placed on their N-termini (Figure 11).</P><DIV class="block two-third center"><DIV class="legend"><B>Figure 11: </B> Schematic representation of the RIP production cassette.</DIV></DIV><P>We gene synthesized our DNA constructs commercially. Once we received the sequence encoding for this production cassette, named Seq8 (461 bp) in the commercial plasmid pEX-A258, we amplified it in competent <I>E. coli</I> DH5<FONT face="Raleway">α</FONT>. After bacterial culture and plasmid DNA extraction, we digested the commercial vector with <B>EcoRI</B> and <B>PstI</B> restriction enzymes. We extracted the inserts from the gel and performed a ligation by using specific overlaps into <B>linearized pBR322</B> for RIP expression and into <B>pSB1C3</B> for iGEM sample submission. We proved that our vectors contained the insert by electrophoresis (Figure 12, 13).</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 12: </B> Agarose 1% gel after electrophoresis of digested pSB1C3 containing Seq8 (Bba_K2616001) with PstI and EcoRI. All colonies except 1, 3 and 7 contained the insert. </DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 13: </B> Agarose 1% gel after electrophoresis of digested pBR322 containing Seq8 (Bba_K2616001) with NdeI (lane 1 to 7). All colonies except colonies 2 and 7 contained the insert.  </DIV></DIV><DIV class="block full"><P><B>Sequencing</B> results, when aligned with our original construct using Geneious, confirmed that pSB1C3 contained Seq8, <A href="http://parts.igem.org/Part:BBa_K2616001"> Bba_K2616001</A>. </P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 14: </B> Alignment of sequencing results for BBa_K2616001. Sequencing perform in pSB1C3 plasmid and one primer was designed (FOR1) to cover the whole sequence. Image from Geneious. Pairwise % Identity: 100%.  </DIV></DIV><DIV class="block full"><P>Once checked, we cloned our construct into the <I>Escherichia coli</I><B>BL21(DE3) pLysS</B> strain, a specific dedicated strain to produce high amounts of desired proteins under a T7 promoter. Bacteria were grown in 25 mL culture, and <B>protein expression</B> was induced with different IPTG concentrations during exponential phase at an OD<SUB>600nm</SUB> at 37°C. A 1 mL aliquot was centrifuged and the pellet stored at -20°C.
                After two hours of induction, we centrifuged and collected both supernatant and pellet separately.</P></DIV><DIV class="block title"><H3>Test of RIP effect on <I>S. aureus</I> biofilm formation</H3></DIV><DIV class="block full"><H4 style="text-align: left;">Fluorescence reading experiments</H4><P>Since RIP is only a seven-aminoacid peptide, we were not able to check its production by classic SDS-PAGE. Thus, we tried to check its expression by <B>observing its effect</B> on <I>Staphylococcus aureus</I> growth and adhesion. We grew a <I>S. aureus</I> strain expressing GFP (Green Fluorescent Protein), (kindly provided by Pr. Jean-Marc Ghigo) on 96-well microtiter plates with different fractions of supernatant or pellet of our BL21(DE3) pLysS bacterial cultures containing BBa_K26160001.</P><P>After 48h or more of incubation at 37°C, we washed the plates in order to discard planktonic bacteria, and read fluorescence (excitation at 485 nm and measuring emission at 510 nm).</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 15: </B>Measurement of the impact of RIP on biofilm formation of <I>S. aureus</I>. In yellow, <I>S. aureus</I> alone with different concentrations of IPTG. In blue, <I>S. aureus</I> in the presence of culture Medium from induced BL21(DE3) <I>E. coli</I> expressing RIP. In green, <I>S. aureus</I> in the presence of the cell lysate supernatant from induced BL21(DE3) <I>E. coli</I> expressing RIP. Every measurement was done eight times and the bars show the average fluorescence.</DIV></DIV><DIV class="block half"><P>Some of the results we got were extremely encouraging. For example, Figure 15 shows an average 3-fold reduction of fluorescence from <I>S. aureus</I> biofilms when they were cultivated in presence of the bacterial lysate of an induced culture of BL-21 <I>E. coli</I> transformed with  BBa_K2616001. </P><P>However, we performed those experiments several times, and the results were not always as concluding. This variability is very likely due to a bias linked to the different approaches used for supernatant removal and washes. When using the flicking approach, we damaged the biofilms. Therefore, we removed planktonic cells by micropipeting. This variability is often encountered when using this protocol, even in Pr. Jean-Marc Ghigo's laboratory.</P></DIV><DIV class="block full"><H4 style="text-align: left;">Crystal violet staining</H4><P>Since fluorescence measurements were not satisfying enough, we tried to improve our methods for quantifying biofilm formation. Thus, we began staining biofilms by Crystal violet 0.1% and measuring absorbance at 570 nm. Again, the results were very heterogeneous between our different experiments, and between the different protocols. 
We tried to compare our protocol within the Institut Pasteur, but also outside of it. This was the occasion to collaborate with another iGEM team, namely the team WPI Worcester, who was also working on biofilm disruption. We decided to exchange our protocols. The results of this comparative experiment are shown in Figure 16.</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 16: </B>Measurement of the absorbance at 570 nm of <I>S. aureus</I> biofilms after 0.1% crystal violet staining. We compared the washing protocols of our team (in red) with the one of WPI Worcester team (in blue). All biofilms were cultivated with varying concentrations of cell lysate supernatant from a BL21(DE3) <I>E. coli</I> culture induced with 0.1 mM IPTG for RIP peptide production. LS = Lysis Supernatant from the induced BL21(DE3)<I>E. coli</I> culture. NI=Non Induced. Every measurement was done eight times and the bars show the average measured absorbance.</DIV></DIV><DIV class="block full"><P>We show that our method gave lower biofilm retention than WPI Worcester's. However, even if we obtained higher retention values with theirs, we still met the same variability, as seen by the error bars. This may be related to the use of various solvents, namely ethanol and acetone in our method, and acetic acid in their case. Mechanically, we applied the same steps in our first approach. Since there was no improvement, we switched to pipetting and then finally back to full tray washing again. Both protocols can be found <A href="https://2018.igem.org/Team:Pasteur_Paris/Protocols/CellBio" style="font-weight: bold ; color:#85196a;" target="__blank">here</A>.</P></DIV><DIV class="block two-third"><H4 style="text-align: left;">Biofilm PFA fixation before staining</H4><P>We wanted to avoid biofilm damage or loss during these steps. In order to do that, we used Bouin solution to fix the formed biofilm after 24 and 48 hours of culture (Figure 17). Biofilms were then either stained with crystal violet 0.1% and resuspended in acetic acid 30% or directly resuspended in PBS 1X.  Surprisingly, with this method, the biofilm formation was higher when cultivated with cell extracts containing RIP. For now, we are not able to explain why.</P></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 17: </B>Biofilm culture fixed with Bouin's solution in 96-well microtiter plate</DIV></DIV><DIV class="block full"><P>With more time, we would certainly have been able to optimize our protocols to best fit with the strain we use, but for the time being, we are not able to give a final conclusion on whether or not our RIP peptide inhibits <I>S. aureus</I> biofilm formation.
Potential ideas for improvement would first be to better standardize starting amounts of biofilm cultures. Secondly, to find more gentle planktonic cells removal methods. Thirdly, better staining methods in order to get better absorbance readouts that can also take into account biofilm formation on the walls of the 96-wells plate and not only on its floor. Finally, the use of RIP peptides that have been processed through the export machinery and that would be cleaved from their export signal might have higher activities. 
</P></DIV><DIV class="block full"><H2><I>S. aureus</I> Detection and RIP secretion <A href="http://parts.igem.org/Part:BBa_K2616003"> BBa_K2616003</A></H2><P>The sequence we designed contains the <I> agr </I> detection system from <I>S. aureus</I> and secretion of  RIP (RNAIII Inhibiting Peptide) sequences fused to two different export signal peptides for <I>E. coli</I> Type II Secretion System: DsbA and MalE, placed in their N-termini (Figure 18).</P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 18: </B><I> S. aureus </I> sensor device and RIP production cassette</DIV></DIV><DIV class="block full"><P>We gene synthesized our DNA commercially by Eurofins-Genomics. We received this genetic construct in three parts that we called Seq5 (1422 bp), Seq6 (960 bp) and Seq7 (762 bp) in the commercial plasmid pEX-A258 which we amplified in competent <I>E. coli</I> DH5<FONT face="Raleway">α</FONT>. 
 After bacterial culture and plasmid DNA extraction, we digested the commercial vector with XbaI and BamHI for Seq5, MscI, and SphI for Seq6, and HindIII and SpeI for Seq7. We extracted the insert from the gel and ligated by specific overlaps into linearized pBR322 for expression and into pSB1C3 for iGEM sample submission.</P><P>We had trouble to proceed with the ligation of the three inserts to linearized pBR322 and pSB1C3. We discussed with Takara Bio about our ligation issues, the GC percentage on our overlaps was too high to allow for a good ligation.  Due to the lack of time, we were not able to redesign the overlaps for this construction.  </P></DIV></DIV><DIV class="block full" style="background-color: transparent;"><DIV class="block title"><H3>Summary</H3></DIV><I><P>Achievements:</P></I><UL style="text-align: left;list-style: disc;"><LI>Successfully cloned a biobrick coding for RIP secretion in pBR322 and in pSB1C3, creating a new part <A href="http://parts.igem.org/Part:BBa_K2616001"> Bba_K2616001 </A>.
                        </LI><LI>Successfully sequenced <A href="http://parts.igem.org/Part:BBa_K2616001"> Bba_K2616001 </A> in pSB1C3 and sent to iGEM registry.
                        </LI><LI>Successfully cultivated S. aureus biofilms in 96-well plates with different supernatants. Although there was a high variability in our results, and we used several protocols to overcome it, in one case, we were able to observe a reduction in biofilm formation in the presence of our RIP.</LI></UL><P><I>Next steps:</I><UL style="text-align: left;list-style: disc;"><LI>Clone the sensor device with inducible RIP production upon <I>S. aureus</I> detection.</LI><LI>Improve the characterization of RIP effect on biofilm formation with a more standardized assay.</LI></UL></P></DIV></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Kill"><H4>KILL SWITCH<I>Click to see more </I></H4></DIV><DIV class="block hiddenContent"><DIV class="block full"><DIV class="block title"><H1 style="padding-top: 50px;">KILL SWITCH</H1><H2><A href="http://parts.igem.org/Part:BBa_K2616002"> BBa_K2616002</A></H2></DIV><DIV class="block full"><P>The sequence designed codes for two different proteins: CcdB toxin and CcdA antitoxine. The antitoxin production is under an constitutive promoter (PLac) and the toxin production under a thermosensitive one (PcspA). </P></DIV><P>We gene synthesized the genetic construct of our kill-switch commercially. Once we received the sequence, called Seq9, in a commercial plasmid, we transformed competent bacteria <I>E. coli</I> DH5<FONT face="Raleway">α</FONT>. After bacterial culture and plasmid DNA extraction, we digested our DNA with restriction enzymes, extracted the inserts from the gel, and ligated it into linearized pSB1C3 for iGEM submission and expression in BL21(DE3).</P><P>We proved that our vector contained the insert by DNA electrophoresis (Figure 19).</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 19: </B> Agarose gel after electrophoresis of digested pSB1C3 containing Seq9 (Bba_K2616002) in columns 6 to 11. Colonies 2 and 6 have the correct plasmid. </DIV></DIV><DIV class="block full"><P><B>Sequencing</B> results, when aligned to our original construct using Geneious, confirmed that pSB1C3 contained Seq9. This sequence was sent to the registry as <A href="http://parts.igem.org/Part:BBa_K2616002" style="font-weight: bold ; color:#85196a;" target="_blank"> Bba_K2616002</A>. </P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 20: </B> Alignment of sequencing results for BBa_K2616002. Sequencing perform in pSB1C3 and two primers were designed (FOR1 and FOR2) to cover the whole sequence. Image from Geneious. Pairwise Identity: 100%.  </DIV></DIV><DIV class="block full"><P>The construction was successfully assembled. In Figure 20, we show that we used two different primers, allowing us to cover the whole sequence without mistakes. As visible, the mismatches are only present at the extremities of each primer sequencing. The final basepair identity is 100%.</P></DIV><DIV class="block title"><H3>Test of kill-switch efficiency</H3></DIV><DIV class="block full"><P>To test the efficiency of our kill-switch, we decided to cultivate transformed BL21(DE3) pLysS <I>E. coli</I> at several temperatures (15°C, 20°C, 25°C and 37°C). We used BL21(DE3) pLysS <I>E. coli</I> transformed with the empty pSB1C3 plasmid as the negative control. The bacteria growth was followed by measuring the optical density at 600 nm every 30 minutes for 6 hours, followed by two additional points at 18 hours and at 72 hours. Each experiment was done in triplicate and the standard deviation was calculated for every point. We showed that bacteria transformed with the kill-switch presented <B>no measurable growth</B> at 15°C and at 20°C during the 72 hours of the experiment, whereas the control population grew normally (Figure 21).</P><P>At 25°C, the kill-switch population grew more slowly than the control for the first 18 hours, but the growth eventually started to reach normal values at 72 hours. </P><P>Finally, at 37°C there was no difference in the growth of the kill-switch population compared to the control bacteria. </P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 21: </B>Effect of different temperatures on the growth of Cryodeath kill-switch transformed BL21(DE3) pLysS <I>E. coli</I></DIV></DIV><DIV class="block full"><P>Thus, we successfully guarantee that our engineered bacteria will not be able to grow if they happened to be released in the environment.</P></DIV></DIV><DIV class="block full" style="background-color: transparent;"><DIV class="block title"><H3>Summary</H3></DIV><P>Achievements:<UL style="text-align: left;list-style: disc;"><LI>Successfully cloned the biobrick <A href="http://parts.igem.org/Part:BBa_K2616002" style="font-weight: bold ; color:#85196a;" target="_blank"> Bba_K2616002</A> coding for toxin/antitoxin (CcdB/CcdA) system in pSB1C3, creating a <B>new part</B>.</LI><LI>Successfully sequenced  <A href="http://parts.igem.org/Part:BBa_K2616002" style="font-weight: bold ; color:#85196a;" target="_blank"> BBa_K2616002</A> in pSB1C3 and sent it to iGEM registry.</LI><LI>Successfully observed <B>normal growth</B> of our engineered bacteria at 25°C and 37°C and <B>absence of growth</B> at 18°C and 20°C, showing the <B>efficiency of the kill switch</B>.</LI></UL></P><P><I>Next steps:</I><UL style="text-align: left;list-style: disc;"><LI>Find a system that kills bacteria when released in the environment rather than just stopping their growth.</LI></UL></P></DIV></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Membrane"><H4>MEMBRANE<I>Click to see more </I></H4></DIV><DIV class="block hiddenContent"><DIV class="block title" id="Membrane"><H1 style="padding-top: 50px;">Membrane</H1></DIV><DIV class="block two-third"><P>The <B> membrane filter </B> is a key element of our prosthesis system, allowing the <B> confinement </B> of the genetically modified bacteria and the <B> conduction of neuron impulses </B>. We tested two types of membranes: Sterlitech Polycarbonate <B> Gold-Coated </B> Membrane Filters (pores diameter of 0.4 micrometers) and Sterlitech <B> Alumina Oxide </B> Membrane Filters (pores diameter of 0.2 micrometers).
                         Sterlitech Alumina Oxide Membrane Filters were <B> coated </B> with different types of <B> biocompatible conductive polymers: PEDOT:PSS </B> (poly(3,4-ethylenedioxythiophene) polystyrene sulfonate), <B> PEDOT:Cl </B>  and <B> PEDOT:Ts </B> .
                         To characterize the potential of the different types of membranes to be integrated into our prosthesis system, <B> we designed a PDMS (polydimethylsiloxane) well chip for that exact purpose </B>, a modified <B>culture well </B>. The bottom of the well is a membrane, and a platinum wire touching the membrane electrically connects the inside of the well with the exterior.</P></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 22: </B> PEDOT:PSS </DIV></DIV><DIV class="block full"><H2 style="text-align: left;">Confinement</H2><P> The first requirement for the membrane is that it needs to meet is the ability to <B> retain bacteria </B> of the size of <I>E. coli</I> (1-2 micrometers). Theoretically, confinement should be garanteed because the membrane's pore size is smaller than <I>E. coli</I>. Three experiments were conducted on membrane microchannel chips to <B> prove that the gold-coated membranes can retain bacteria </B>. </P><H3> First experiment </H3><P> The optical density of an <I>E. coli</I> liquid culture was measured. Liquid culture was poured in a microchannel chip on one side, and optical density of the liquid that flowed to the other side of the microchannnels was measured. </P><H4 style="text-align: left;"> Results </H4><P> OD (600 nm) of liquid culture: 0.44 </P><P> OD (600 nm) of liquid after flowing through the chip: 0.41. </P><H4 style="text-align: left;"> Interpretation </H4><P> We expected a much lower OD after liquid flow through the chip, so this suggests the presence of a leak in the chip, that allowed the liquid culture to flow without retaining the bacteria.</P></DIV><DIV class="block one-third"><H3> Second experiment </H3><P> A few drops of RFP expressing DH5alpha <I>E. coli</I> liquid culture were poured in a membrane microchannel chip. The chip was then observed under a microscope.  </P><H4 style="text-align: left;"> Interpretation </H4><P> Membrane is located on the right side, and liquid culture was poured on that side, before the membrane. Bacteria was still able to flow to the left side, but they were not following the microchannels, instead they were all just flowing in a single direction, suggesting the membrane lifted the microfluidic chip from below and thus caused massive leakings in the microfluidic circuitry. </P></DIV><DIV class="block two-third"><DIV class="legend"><B>Figure 23: </B> Membrane microchannel chip under microscope</DIV></DIV><DIV class="block full"><H3> Third experiment </H3><P> A few drops of E. coli liquid culture were poured in a membrane microchannel chip. The chip was then observed under a microscope. </P><H4 style="text-align: left;"> Interpretation </H4><P> The membrane is located on the left side, and liquid culture was poured on that side, before the membrane. Bacteria this time wasn't able to flow to the right side, <B> the membrane stopped their progression </B>. It is clear on figure 24, that the left side is crowded with bacteria, and the right side is empty (apart from a few PDMS impurities). Final conclusion on the membrane microchannel chips is, that although the integration method of the membrane filter in the chip is complicated and a bit improvised, <B> some chips apparently do fulfill their purpose </B>, demonstrating this way the confinement of the bacteria with a membrane. Leaks observed in previous experiments were also probably caused by membrane filters that were not correctly stretching across the whole chip.</P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 24: </B> Membrane microchannel chip under microscope with retaining membrane</DIV></DIV><DIV class="block full"><H2 style="text-align: left;"> Polymer coating </H2><P> Bare alumina oxide membranes were coated with different polymers to enhance their conductivity values and their biocompatibility.  </P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 25: </B> White alumina oxide membranes before coating</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 26: </B> Scanning electron microscopy  of bare alumina oxide membranes</DIV></DIV><DIV class="block full"><H3> PEDOT:PSS coating</H3><P> The color of the alumina oxide membranes changed radically from <B> light grey to black </B>, suggesting the deposit of PEDOT:PSS on the membrane, as expected. Scanning electron microscopy (courtesy of Bruno Bresson, Sciences et Ingénierie de la Matière Molle Physico-chimie des Polymères et Milieux Dispersés) of a PEDOT:PSS-coated membrane revealed <B> cluster-like formation of PEDOT:PSS deposits </B>. It is thought that the lack of uniformity of the coating won't give the expected results in matters of biocompatibility and conductivity. </P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 27: </B> PEDOT:PSS-coated membranes</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 28: </B> Scanning electron microscopy of PEDOT:PSS-coated membrane</DIV></DIV><DIV class="block full"><H3> PEDOT:Cl and PEDOT:Ts coating</H3><P> The color of the alumina oxide membranes changed radically from <B> light grey to black with green shades for PEDOT:Cl and blue shades for PEDOT:Ts </B>, suggesting the deposit of the polymers on the membranes, as expected. Scanning electron microscopy reveals a uniform thickening of the membrane's surface, suggesting a <B> uniform </B> PEDOT:Cl coating of the membrane. We expect better results from the PEDOT:Cl-coated and PEDOT:Ts-coated membranes than PEDOT:PSS-coated ones.  </P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 29: </B> PEDOT:Cl-coated membranes</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 30: </B> PEDOT:Ts-coated membranes</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 31: </B> Scanning electron microscopy of PEDOT:Cl-coated membrane</DIV></DIV><DIV class="block full"><H2 style="text-align: left;">Conductivity</H2><P> The membranes used in our system should possess good electric conductive capabilities for nerve influx conduction. The goal here is to <B> evaluate the conductivity of the membranes </B>.</P></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 32: </B> Hand-made PDMS well chip </DIV></DIV><DIV class="block two-third"><P>The conductivity of the membranes was measured on a <B> self-made device </B>. It consists of a culture well made of PDMS (polydimethylsiloxane), with a <B> membrane filter </B> at its bottom and a <B> platinum wire </B> linking the conductive membrane filter with the exterior.</P></DIV><DIV class="block full"><H3>Platinum wire</H3><P>As in the end we were going to measure the conductivity of the system biofilm+membrane+platinum wire, we wanted to simplify the measurements and <B> neglect the impact of the platinum wire </B>. Function generator was set on sine. The physical quantities measured here are Eg, the generator's tension amplitude and Ep, the voltage difference between the two extremities of a platinum wire. the quantity calculated here is 20*log(Ep/Eg) for different frequencies. </P><H4 style="text-align: left;"> Results </H4><DIV class="legend"><B>Figure 33: </B> Conductivity of a platinum wire for different frequencies </DIV><H4 style="text-align: left;"> Interpretation </H4><P> Voltage difference calculated is extremely low, indicating a very good conductivity for the platinum wires, so its resistance (in low frequencies) <B> could be neglected </B> at first glance when it would be used in PDMS well chips. Resistance increases in higher frequencies, because of the skin-effect in metals: the strip transforms into an antenna. But as we were going to use only low frequencies, this didn't affect us. </P></DIV><DIV class="block full"><H3>Frequency impact on membrane conductivity</H3><P> Before measuring the conductivity of multiple membranes, we needed to have an overview of the impact of the frequency on the conductivity of a membrane. We tested two gold-coated membranes.Function generator was set on sine. The physical quantities measured here are Eg, the generator's tension amplitude and Ep, the voltage difference between the extremity of the platinum wire outside the well chip and a point on the edge of the membrane of the chip. the quantity calculated here is 20*log(Ep/Eg) for different frequencies.  </P><H4 style="text-align: left;"> Results </H4><DIV class="legend"><B>Figure 34: </B> Conductivity of a platinum wire for different frequencies </DIV><H4 style="text-align: left;"> Interpretation </H4><P> Voltage difference calculated is very low, indicating a very good conductivity for the gold-coated membrane. Technically, we measured the conductivity of the system membrane+platinum wire, but we showed that the wire's conductivity could be neglected. Resistance increases in higher frequencies, again because of the skin-effect in metals. But as we were going to use only low frequencies, this doesn't affect us, and moreover,<B> the frequency response is flat for wide range of low frequencies </B>. </P></DIV><DIV class="block full"><H3>Membrane conductivity</H3><P> We measured the <B> conductivity of 6 membranes on PDMS well chips </B> (2 gold-coated, 1 bare alumina oxide, 1 PEDOT:PSS-coated, 1 PEDOT:Cl-coated, 1 PEDOT:Ts-coated). Here we show the electric circuit that we used for the following experiments.   </P><DIV class="legend"><B>Figure 35: </B> Electric circuit used for the different conductivity measurements </DIV><P> Function generator was set on square at 200 Hz. The physical quantities measured are Eg, the generator tension amplitude, and Ep, the amplitude of the voltage difference between a point on the membrane inside the well and the extremity of the platinium strip outside the well. Tension amplitude of the resistor is given by Er = Eg - Ep. Current flowing through the electric circuit is calculated with I = Er/R. Conductivity of the membrane is given by I/Ep. Conductivity of each membrane was measured 3 times. </P><H4 style="text-align: left;"> Results </H4><DIV class="legend"><B>Figure 36: </B> Conductivity of a platinum wire for different frequencies (mean value and standard deviation for each membrane) </DIV><H4 style="text-align: left;"> Interpretation </H4><P> Bare alumina oxide and PEDOT:PSS-coated membranes show similar conductivities, indicating the <B> incomplete coating of PEDOT:PSS </B> on alumina oxide membranes. On the opposite, PEDOT:Cl and PEDOT:Ts exhibit on average better conductivities, but in the same time, the coating of these membranes revealed by electron microscopy seemed to have covered the alumina oxide membranes in a more uniform way, <B> ensuring enhanced conductive capabilities </B>. These results can be criticized because of the high deviation and because the membranes conductivity was measured after several biofilms were grown on them, which may have affected the measurements. </P></DIV><DIV class="block full"><H2 style="text-align: left;">Biocompatibility and biofilm conductivity</H2><P> One last important property of the membranes to measure was the <B> capability of bacteria to form a biofilm </B> on them, as in our prosthesis system, the membrane is going to be directly in contact with the genetically modified biofilm, as well as the human body.  </P><P> We used the last section of the following <A href="https://static.igem.org/mediawiki/2018/b/b5/T--Pasteur_Paris--PDMS-well-chip.pdf">protocol</A> to form biofilms in our PDMS well chips and to measure the biofilm growth.</P><H4 style="text-align: left;"> Results: biofilm growth  </H4><P> Biofilm growth was measured 4 times in total. For each series of measurements, the measured optical densities were divided by the optical density of the base liquid culture, to normalize the measures.</P><DIV class="legend"><B>Figure 37: </B> Biofilm growth (mean value and standard deviation for each type of membrane)</DIV><H4 style="text-align: left;"> Results: biofilm conductivity  </H4><P> For conductivity measurements, we used the same electric circuit as in figure 35 . Function generator was set on square at 200 Hz. The physical quantities measured are Eg, the generator tension amplitude, and Ep, the amplitude of the voltage difference between a point on the biofilm inside the well and the extremity of the platinium strip outside the well. Tension amplitude of the resistor is given by Er = Eg - Ep. Current flowing through the electric circuit is calculated with I = Er/R. Conductivity of the membrane is given by I/Ep. Conductivity of each membrane with a biofilm was repeated 3 times. </P><DIV class="legend"><B>Figure 38: </B> Membrane conductivity with biofilm (mean value and standard deviation for each type of membrane)</DIV><P> To approximate very roughly the conductivity of the biofilm, the average conductivity values of the membranes with a biofilm were divided by the corresponding average biofilm growth values, and the conductivity of the membrane was then substracted.  </P><DIV class="legend"><B>Figure 39: </B>Estimated biofilm conductivity </DIV><H4 style="text-align: left;"> Interpretation  </H4><P> As told by the membrane manufacturer, biofilm formation on gold membranes seems indeed to be more difficult than on other membranes. However we expected PEDOT:PSS-coated membranes to stimulate more the growth of biofilm, but perhaps this may be just another indicator of the incomplete coating. Surprisingly, <B> PEDOT:Cl tends to allow better formation of biofilms</B>. We realized only after the experiments the need for a control biofilm culture without membrane. </P><P> Conductivity with a biofilm is better with gold membranes, although the conductivity of gold membranes themselves isn't the best. This may be explained by the fact, that because of the thinner biofilm formation on gold membranes, the electrical wires touched not only the biofilm, but also the membrane, bypassing the biofilm and leading to imprecise measurements. </P><P> Approximate biofilm conductivity is therefore probably wrong for the gold membrane. However, it is interesting to notice that the biofilm conductivity measured for the bare alumina oxyde, PEDOT:Ts-coated one and PEDOT:PSS-coated give more or less the same value, suggesting that with <B> more measurements, adapted equipement and better methods it would be indeed possible to measure the biofilm's conductivity with our PDMS well chips </B>. </P></DIV></DIV><DIV class="block full" style="background-color: transparent;"><DIV class="block title"><H3>Summary</H3></DIV><P><I>Achievements: </I><UL style="text-align: left;list-style: disc;"><LI> Successfully demonstrated the <B> confinement of bacteria </B> by a membrane filter. </LI><LI> Successfully <B> coated </B> alumina oxide membranes with PEDOT:Cl and PEDOT:Ts .</LI><LI> Partially <B> coated </B> alumina oxide membranes with PEDOT:PSS.</LI><LI> Successfully demonstrated the <B> enhanced conductivity </B> induced by the PEDOT:Cl and PEDOT:Ts coating. </LI><LI> Successfully <B> enhanced biocompatibilty </B> with PEDOT:Cl coating.   </LI></UL></P><P><I>Next steps:</I><UL style="text-align: left;list-style: disc;"><LI> Enhance <B> measurement precision </B> for membrane conductivity with and without biofilm.</LI><LI> Improve <B> PEDOT:PSS coating </B> to form a uniform layer.</LI></UL></P></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>