<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="Menu"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="project"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">Human Practices</A></LI><LI id="design"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Product Design</A></LI><LI id="parts"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">labwork</A></LI><LI id="achievements"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About Us</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Description">Overview</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Growing_Back_Nerves">Reconnect Nerves</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Fighting">Fight Infections</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Model">Model</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Membrane">Physical Interface</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Kill">Kill-Switch</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Entrepreneurship">Entrepreneurship</A></LI></UL><UL id="hp-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Human_Practices">Integrated HP</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Public_Engagement">Education &amp; Engagement</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Safety">Safety &amp; Ethics</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Law">Law</A></LI></UL><UL id="design-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Technicals">Methodology</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Scenario">Scenario</A></LI></UL><UL id="parts-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Basic_Part">Basic Parts</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Improve">Improve</A></LI></UL><UL id="labwork-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Notebook">Notebook</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Experiments">Protocols</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Results">Results</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/InterLab">InterLab</A></LI></UL><UL id="achievements-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Medals">Medals</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Awards">Awards</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Demonstrate">Demonstrate</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/GiantJamboree">Giant Jamboree</A></LI></UL><UL id="aboutus-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Team">Team Members</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Acknowledgements"> Acknowledgements</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Attributions">Attributions</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2018.igem.org/Team:Pasteur_Paris">Home</A></LI><LI id="project_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">HP</A></LI><LI id="design_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Design</A></LI><LI id="parts_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">Labwork</A></LI><LI id="achievements_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About us</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV id="GeneralContent"><DIV id="index" class="block"><DIV id="indexContent"><P><A href="#Introduction" class="link">Introduction</A></P><P><A href="#Production" class="link">NGF Production</A></P><P><A href="#Diffusion" class="link">NGF Diffusion</A></P><P><A href="#Growth" class="link">Neurons Growth</A></P><P><A href="#Mechanical" class="link">Mechanical Model</A></P><P><A href="#References" class="link">References</A></P></DIV></DIV><DIV id="MainContent"><DIV class="block title" id="Introduction"><H1> First aspect modeled : secretion, diffusion and influence of NGF </H1></DIV><DIV class="block two-third"><P>The aim of our mathematical model is to simulate the growth of neurons towards our biofilm in response to the presence of pro Nerve Growth Factor (NGF) (Figure 1). NGF is part of a family of proteins called neurotrophins. They are responsible for the development of new neurons, and for the growth and maintenance of mature ones. We created a deterministic model to help the wet lab establish the optimal concentration gradients of NGF needed for the regrowth of the nerves. NGF concentration and concentration gradient are key parameters affecting the growth rate and direction of neurites. Neurites growth has shown to be NGF dose-dependent: if NGF concentration is too low or too high, the growth rate is attenuated. In order to visualize the results of the model on a microchannel, we used MATLAB and Python. This is an important part of our project since it creates the link between the wet lab and dry lab. </P></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 1: </B>Drawing of the NGF (purple) required for the growth of the neurons</DIV></DIV><DIV class="block full"><P style="text-align: center;">We divided our model in three parts:
                    <OL style="text-align: left;"><LI>Production of NGF by the genetically modified <I>Escherichia coli</I></LI><LI>Simulation of the diffusion of NGF in a given environment</LI><LI>Neurons growth in the presence of NGF</LI></OL></P></DIV><DIV class="block title"><H1>Context of our model</H1></DIV><DIV class="block half"><P>Our project aims at creating a biofilm composed of genetically modified <I>E. coli</I> able to release a neurotrophic factor: NGF. It helps to accelerate the connection between the neurons and the implant of the prosthesis; hence aiming at connecting the prosthesis and the amputee's neurons directly. This will enable the patient to have a more instinctive control of his prosthetic device. The nerves will be guided towards a conductive membrane surrounding our genetically modified biofilm (Figure 2). This membrane will then pass the neural signal of the regenerated nerves towards the electronic chip of the implant through wires.  It will allow the patient to have a more instinctive and natural control than any other current prosthesis, and a reduced re-education time.</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 2: </B>Drawing of the <I>E. coli</I> confined in our membrane</DIV></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 3: </B>Drawing of our microfluidic chip</DIV></DIV><DIV class="block two-third"><P>The aim of the wet lab is to test the biofilm on a microfluidic chip as a proof of concept. The chip is composed of two compartments: one contains the genetically modified <I> E. coli </I> that produce NGF and the other one contains neurons (Figure 3). Microchannels link the two compartments in the middle of the chip, allowing the diffusion of NGF and the growth of the neurites.  Our model will hence be established on a microfluidic chip shape in order to share our results with the wet lab and indicate them the optimal concentration of NGF needed according to our model. All the codes we used in this part are available <A href="https://github.com/samueljaoui/iGEM-Pasteur-Paris-2018-codes-for-secretion-diffusion-and-influence-" style="font-weight: bold ; color:#85196a;" target="_blank">here.</A></P></DIV><DIV class="block two-third center"><P><I>We introduce different parameters in order to create our model :</I></P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD>g</TD><TD>Length of the neurite outgrowth</TD></TR><TR><TD><SPAN><DIV class="frac"><SPAN>dg</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></DIV></SPAN></TD><TD>Neurite outgrowth rate</TD></TR><TR><TD>u(x,t)</TD><TD>Concentration of NGF at the position x and time t</TD></TR><TR><TD><SPAN><DIV class="frac"><SPAN>du</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></DIV></SPAN></TD><TD>NGF concentration gradient at the position x and time t</TD></TR><TR><TD>C<SUB>diff</SUB></TD><TD>Diffusion coefficient of NGF</TD></TR><TR><TD>K</TD><TD>Gradient factor (growth rate of the neurite under the stimulation of the NGF concentration gradient)</TD></TR><TR><TD>G<SUB><FONT face="Raleway">θ</FONT></SUB></TD><TD>Baseline growth rate (neurite growth rate in absence of NGF concentration gradient)</TD></TR><TR><TD>L</TD><TD>Length of the conduit </TD></TR></TBODY></TABLE></DIV><DIV class="block half"><DIV class="legend"><B>Table 1: </B>Model variables</DIV></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Production"><H4>NGF production by genetically modified <I>E. coli</I></H4></DIV><DIV class="block hiddenContent"><DIV class="block title"><H1 style="padding-top: 50px;">NGF production by genetically modified <I>E. coli</I></H1><P><I>As we want to obtain the best fitted NGF concentration, we first simulate the production and secretion of our recombinant NGF by transformed <I> E. coli</I>, in order to help the wetlab to optimize the induction and obtain the desired concentration, and to check whether we can theoretically obtain the optimal concentration for neurite growth.</I></P></DIV><DIV class="block full"><H3>Model Description</H3><P>In this model, we include transcription, translation, translocation through <I> E. coli </I> membrane, protein folding and mRNA and protein degradation in cytoplasm and medium. NGF synthesis is placed under Plac promoter, so we also modeled the IPTG induction. Finally, NGF is secreted in the medium through Type I secretion system in which the export signal peptide is not cleaved during translocation. Our Biobrick is designed to synthetize and export TEV protease in order to cleave signal peptide and thus produce functional NGF.</P><P>The molecular mechanism included in our model appears schematically in Figure 4.</P></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 4: </B>Secretion mechanism of TEV and NGF by our engineered bacteria</DIV></DIV><DIV class="block full"><P style="">Our model includes the following variables:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>Name</B></TD><TD><B>Meaning</B></TD></TR><TR><TD><B>I<SUB>ex</SUB></B></TD><TD>IPTG outside the cell</TD></TR><TR><TD><B>I<SUB>in</SUB></B></TD><TD>IPTG in the cytoplasm</TD></TR><TR><TD><B>P<SUB>o</SUB></B></TD><TD><I>Plac</I> promoter occupied by repressor, prevent transcription</TD></TR><TR><TD><B>P<SUB>f</SUB></B></TD><TD><I>Plac</I> promoter with free <I>lacO</I> site</TD></TR><TR><TD><B>m</B></TD><TD>mRNA for TEV and NGF</TD></TR><TR><TD><B>m-r</B></TD><TD>Ribosome-bound mRNA </TD></TR><TR><TD><B>NGF<SUB>c</SUB></B></TD><TD>NGF in cytoplasm</TD></TR><TR><TD><B>TEV<SUB>c</SUB></B></TD><TD>TEV protease in cytoplasm</TD></TR><TR><TD><B>(N-T)<SUB>c</SUB></B></TD><TD>NGF-TEV complex in cytoplasm</TD></TR><TR><TD><B>NGF<SUB>cc</SUB></B></TD><TD>Cleaved NGF in cytoplasm, cannot be exported</TD></TR><TR><TD><B>NGF<SUB>t</SUB></B></TD><TD>NGF bound to transporter channel</TD></TR><TR><TD><B>TEV<SUB>t</SUB></B></TD><TD>TEV bound to transporter channel</TD></TR><TR><TD><B>t</B></TD><TD>Transmembrane transporter</TD></TR><TR><TD><B>NGF<SUB>um</SUB></B></TD><TD>Unfolded NGF in medium with export peptide</TD></TR><TR><TD><B>NGF<SUB>m</SUB></B></TD><TD>Folded NGF in medium with export peptide</TD></TR><TR><TD><B>N-T<SUB>m</SUB></B></TD><TD>Complex between NGF with export peptide and functional TEV</TD></TR><TR><TD><B>TEV<SUB>m</SUB></B></TD><TD>TEV in medium with export peptide</TD></TR><TR><TD><B>NGF<SUB>f</SUB></B></TD><TD>Functional NGF in the medium</TD></TR></TBODY></TABLE></DIV><DIV class="block half"><DIV class="legend"><B>Table 2: </B>Model parameters</DIV></DIV><DIV class="block title"><H4 style="text-align: left;">1. NGF and TEV synthesis in the cytoplasm</H4></DIV><DIV class="block full"><P>The synthesis of NGF and TEV is placed under the control of the Plac promoter. The promoter can be in two different states: occupied (Po) by the repressor lacI, preventing RNA polymerase from binding and thus preventing transcription, or free (Pf) thanks to IPTG binding to the repressor. We assume that one IPTG molecule binds with one repressor molecule, freeing the promoter and restoring RNA polymerase binding capacity. The real mechanism of promoter Plac is more complex, as described in [1], but this simplification is sufficient for our model.</P></DIV><DIV class="block full"><P>The transport of IPTG from outside the cell to cytoplasm is considered to be only due to free diffusion through the membrane by two first order reactions with the same kinetic constant.</P></DIV><DIV class="block full"><P>IPTG is not considered to be degraded neither in the cytoplasm nor in the medium.</P><P>For the TEV and NGF transcription, we use a first-order reaction where the rate of mRNA production (m) depends on the concentration of the free promoter (Pf).</P></DIV><DIV class="block full"><P>For the TEV and NGF translation, we first consider binding of ribosomes to ribosome binding site (the same association constant is used since the r.b.s. are the same), and then translation rate is proportional to the protein length. Since TEV and NGF have approximately the same length, we consider only one translation rate <FONT face="Raleway">β</FONT>.</P></DIV><DIV class="block full"><P>Even though it still has an export peptide, TEV is assumed to be functional in the cytoplasm (although less functional than if it had no export peptide). Since NGF has TEV cleaving site between the coding sequence and the export peptide, a fraction of NGF is cleaved inside the cytoplasm and thus cannot be secreted. We use a simple model to simulate TEV kinetics: TEV recognizes the signal sequence ENLYFQ, binds to its substrate and then cleaves the export peptide. This process can thus be modeled by the following equations:</P></DIV><DIV class="block full"><P>K1, k-1 and k2 are taken lower than constants found in literature, in order to model the fact that TEV still has its signal peptide and is consequently less functional than usually.</P></DIV><DIV class="block title"><H4 style="text-align: left;">2. NGF and TEV secretion to the medium</H4></DIV><DIV class="block full"><P>The transport of NGF and TEV with their export signal peptide from inside the cell to the medium is assumed to follow Michaelis-Menten enzymatic kinetics in which the transporter channel (composed of HlyB in the inner membrane, bounded to HlyD and recruiting TolC in the outer membrane) plays the role of the enzyme and intracellular protein the role of the substrate.</P></DIV><DIV class="block two-third"><P>Each protein (NGF and TEV) via its export signal peptide HlyA can bind to the HlyB-HlyD complex pore, forming a protein-transporter complex (NGFt or TEVt). Translocation corresponds to the dissociation of this complex, resulting in restoring a free transporter and secreting NGF or TEV in the medium (NGFum and TEVm), which are the products.</P></DIV><DIV class="block title"><H4 style="text-align: left;">3. Including growth rate</H4></DIV><DIV class="block full"><P>This model is valid for one bacterial cell, but for our model to fit with our proof of concept system, which is a microfluidic chip chamber containing 100 <FONT face="Raleway">μ</FONT>L of bacterial culture, we need to integrate the number of bacteria contained in the chamber. Therefore, our model helps to determine which is the most accurate bacteria amount we need to put in our chip to produce the appropriate NGF concentration.</P></DIV><DIV class="block title"><H4 style="text-align: left;">4. NGF folding and export peptide cleavage by TEV</H4></DIV><DIV class="block full"><P>Once in the medium, both NGF and TEV are still bounded to the export signal peptide HlyA. We assume there is a very small amount of functional TEV, that is sufficient to cleave TEV signal peptide, producing more functional TEV.</P><P>As for the transporter, we use a simple model in which TEV recognizes the signal sequence ENLYFQ, bind to its substrate (which can be either NGF with its export peptide or TEV with its export peptide) and then cleave the export peptide. This process can thus be modeled by the following equations:</P></DIV><DIV class="block title"><H4 style="text-align: left;">5. mRNA and protein degradation</H4></DIV><DIV class="block full"><P>Finally, in cytoplasm and in the medium, mRNA and protein are degraded and all degradations are assumed to follow first-order kinetic reactions.</P></DIV><DIV class="block full"><H3>MODEL PARAMETRISATION</H3><P>From these equations, we obtained a system of differential equations mostly based on mass action kinetics (get it <A href="https://static.igem.org/mediawiki/2018/d/d4/T--Pasteur_Paris--secretion_equations.pdf" style="font-weight: bold ; color:#85196a;" target="__blank">here</A>. We numerically solved the ordinary differential equations system using Euler method implemented in Python. The constants we used were mainly determined from literature and are given in the following table.</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD><B>NAME</B></TD><TD><B>DESCRIPTION</B></TD><TD><B>VALUE</B></TD><TD><B>UNIT</B></TD><TD><B>SOURCE</B></TD></TR><TR><TD>k<SUB>t</SUB></TD><TD>IPTG diffusion rate across the membrane</TD><TD>0.92</TD><TD>min<SUP>-1</SUP></TD><TD>[1]</TD></TR><TR><TD>k<SUB>i</SUB></TD><TD>Association rate for derepression mechanism by IPTG</TD><TD>3 x 10<SUP>-5</SUP></TD><TD>nM<SUP>-1</SUP>min<SUP>-1</SUP></TD><TD>[1]</TD></TR><TR><TD>k<SUB>-i</SUB></TD><TD>Dissociation rate for derepression mechanism</TD><TD>4.8 x 10<SUP>3</SUP></TD><TD>min<SUP>-1</SUP></TD><TD>[1]</TD></TR><TR><TD><FONT face="Raleway">α</FONT></TD><TD>Transcription rate</TD><TD>2</TD><TD>mRNA.min<SUP>-1</SUP>nM<SUP>-1</SUP></TD><TD>[3]</TD></TR><TR><TD>k<SUB>r</SUB></TD><TD>Association rate of ribosome with r.b.s</TD><TD>1</TD><TD>min<SUP>-1</SUP>mRNA<SUP>-1</SUP></TD><TD>[2]</TD></TR><TR><TD>k<SUB>-r</SUB></TD><TD>Dissociation rate of ribosome with r.b.s</TD><TD>1</TD><TD>min<SUP>-1</SUP></TD><TD>[2]</TD></TR><TR><TD><FONT face="Raleway">β</FONT></TD><TD>Translation rate</TD><TD>4</TD><TD>nM.min<SUP>-1</SUP>mRNA<SUP>-1</SUP></TD><TD>[3]</TD></TR><TR><TD>k<SUB>1</SUB></TD><TD>Association rate of TEV with its substrate in the cytoplasm</TD><TD>7.8 x 10<SUP>-7</SUP></TD><TD>min<SUP>-1</SUP>nM<SUP>-1</SUP></TD><TD>Estimated from [4]</TD></TR><TR><TD>k<SUB>-1</SUB></TD><TD>Dissociation rate of TEV with its substrate in the cytoplasm</TD><TD>6 x 10<SUP>-4</SUP></TD><TD>min<SUP>-1</SUP></TD><TD>Estimated from [4]</TD></TR><TR><TD>k<SUB>2</SUB></TD><TD>Cleaving rate by TEV in cytoplasm</TD><TD>1.38 x 10<SUP>-2</SUP></TD><TD>min<SUP>-1</SUP></TD><TD>Estimated from [4]</TD></TR><TR><TD>k<SUB>3</SUB></TD><TD>Association rate of NGF and TEV with transmembrane transporter</TD><TD>6 x 10<SUP>-4</SUP></TD><TD>min<SUP>-1</SUP>nM<SUP>-1</SUP></TD><TD>[5]</TD></TR><TR><TD>k<SUB>-3</SUB></TD><TD>Dissociation rate of NGF and TEV with transporter</TD><TD>2.34</TD><TD>min<SUP>-1</SUP></TD><TD>[5]</TD></TR><TR><TD>k<SUB>4</SUB></TD><TD>Translocation rate within the transporter</TD><TD>2.1</TD><TD>min<SUP>-1</SUP></TD><TD>[5]</TD></TR><TR><TD>k<SUB>f</SUB></TD><TD>NGF folding rate in the medium</TD><TD>0.28</TD><TD>min<SUP>-1</SUP></TD></TR><TR><TD>k<SUB>5</SUB></TD><TD>Association rate of TEV with its substrate in the medium</TD><TD>7.8 x 10<SUP>-5</SUP></TD><TD>min<SUP>-1</SUP>nM<SUP>-1</SUP></TD><TD>[4]</TD></TR><TR><TD>k<SUB>-5</SUB></TD><TD>Dissociation rate of TEV with its substrate in the medium</TD><TD>0.06</TD><TD>min<SUP>-1</SUP>nM</TD><TD>[4]</TD></TR><TR><TD>k<SUB>6</SUB></TD><TD>Cleaving rate by TEV in the medium</TD><TD>1.38</TD><TD>min<SUP>-1</SUP>nM<SUP>-1</SUP></TD><TD>[4]</TD></TR><TR><TD><FONT face="Raleway">δ</FONT><SUB>m</SUB></TD><TD>mRNA degradation rate</TD><TD>0.462</TD><TD>min<SUP>-1</SUP></TD><TD>[1]</TD></TR><TR><TD><FONT face="Raleway">δ</FONT><SUB>pc</SUB></TD><TD>Protein degradation rate in cytoplasm</TD><TD>0.2</TD><TD>min<SUP>-1</SUP></TD><TD>[1]</TD></TR><TR><TD><FONT face="Raleway">δ</FONT><SUB>pm</SUB></TD><TD>Protein degradation rate in extracelular medium</TD><TD>0.1</TD><TD>min<SUP>-1</SUP></TD><TD>[1]</TD></TR></TBODY></TABLE></DIV><DIV class="block half"><DIV class="legend"><B>Table 3: </B>Values of constants</DIV></DIV><DIV><H3>MODEL RESULTS</H3></DIV><DIV class="block full"><P>We determined the temporal evolution of secreted NGF concentration in the medium, in order to get the u(0,t) term used in our following diffusion model.</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 5: </B>Comparison of cytoplasmic and secreted NGF with a single-cell model (IPTG induction 1 mM)</DIV></DIV><DIV class="block half"><P> After the initial dynamics, concentration of secreted NGF quickly reaches a <B>steady state </B>, which is then only driven by the bacterial population dynamics. If we consider a bacterial culture in stationary phase, we can consequently consider that the initial NGF concentration is constant. Our model predicts that the majority of recombinant protein remains cytoplasmic or is secreted but not functional (we consider as &quot;non-functional NGF&quot; the recombinant proteins that are not folded or still have a C-terminal HlyA signal peptide), as it appears in Figure 4.</P></DIV><DIV class="block full"><P>The aim of this first model is to demonstrate that we can expect an appropriate secreted recombinant NGF concentration to observe neurite growth. However, we had to make several assumptions to parametrize the model. We scanned different parameter values for the values we assumed (such as the number of transporters or kinetic parameters for translocation) in order to check the range of NGF amount we can reasonably expect. We also studied the influence of IPTG induction and number of bacteria, since they are parameters our wet lab can control to best fit recombinant NGF secretion with what we need.</P></DIV><DIV class="block title"><H4 style="text-align: left;">Influence of number of transporters</H4></DIV><DIV class="block full"><P>We co-transformed our bacteria with a plasmid expressing HlyB and HlyD, two of the components of the secretion pore. However, we did not quantify the number of pores each cell contains, and we are only able to estimate it, based on assumptions made in [5]. Consequently, we scanned a range of different values for the number of transporters in order to see the range of NGF concentration we can expect.</P><P>The following graph shows the predicted NGF concentration in the microfluidic chip chamber for a number of pores varying: no pore (A.), 10 per cell (B.), 100 per cell (C.) and 500 per cell (D.):</P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 6: </B>Comparison of cytoplasmic and secreted NGF when the number of transporters varies</DIV></DIV><DIV class="block title"><H4 style="text-align: left;">Influence of translocation rate</H4></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 7: </B>Secreted NGF as a function of translocation rate</DIV></DIV><DIV class="block two-third"><P>As expected, the more transporters the cell has, the more recombinant NGF is secreted, but the amount of functional secreted NGF (in blue) remains limited due to TEV protease cleaving efficiency. </P><P>Taking in account the number of <I> E. coli </I> cells and the dilution factor between intracellular and extracellular space, we obtain for 500 transporters a concentration of functional NGF of 1 nM, which corresponds to 24 ng/mL. This is still 10 times lower than what we need to observe neurite growth.
Enhancing signal peptide cleavage by a more efficient enzyme should help solve the problem since we could expect 5 nM functional NGF if the totality of the secreted NGF were cleaved. 
</P></DIV><DIV class="block title"><H4 style="text-align: left;">IPTG induction level</H4></DIV><DIV class="block two-third"><P> One of the parameters our wet lab team is able to adjust is IPTG induction in the microchannel chip in order to optimize the obtained NGF concentration. Consequently, we studied the dependence of secreted NGF with IPTG initial concentration.</P><P> As expected the final NGF concentration (both in the cytoplasm and in extracellular medium) is an increasing function of IPTG induction. As our wet lab did not succeed in quantifying the secreted NGF, it is hard to figure out whether or not the desired concentration was obtained, but if our assumptions are valid, it could be reached with reasonable IPTG concentrations. Production of NGF with the tag has been detected by Mass spectrometry.</P></DIV><DIV class="block one-third"><DIV class="legend"><B>Figure 8: </B>Comparison of cytoplasmic and secreted NGF for different IPTG induction level</DIV></DIV><DIV><H3>PERSPECTIVES</H3></DIV><DIV class="block full"><P>Our model is based on assumptions but it shows that within <B>realistic parameters values</B>, we can reasonably expect to obtain the optimal NGF concentration needed for neurite growth in the microfluidic chamber and it consequently paves the way to a functional proof of concept. </P><I style="text-align: left;"><P>Next modeling steps:<UL style="list-style: disc;"><LI> It would be worth isolating and <B>quantifying secreted recombinant NGF</B> in order to confront model and experiments, and be able to determine some of the kinetics parameters values we used (such as translocation rate)</LI><LI> This program is designed to model the microchip proof-of-concept experiment but we will adapt it to our final <B>biofilm</B> device to predict its behavior</LI></UL></P></I></DIV></DIV></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Diffusion"><H4>NGF diffusion simulation in a given environment</H4></DIV><DIV class="block hiddenContent"><DIV class="block title"><H1 style="padding-top: 50px;">NGF diffusion simulation in a given environment</H1><P><I>We are trying to understand the way the NGF spreads inside the conduit once it is produced. This will help us determine the NGF concentration u(x,t) (ng.mL<SUP>-1</SUP>) as a function of the distance x (cm) from the production site of NGF.</I></P></DIV><DIV class="block full"><H3>Fick’s diffusion law </H3><P>To simulate NGF diffusion in the microfluidic chip we consider a unidimensional conduit of axe x (cm) and a constant concentration rate of NGF introduced at one end of the canals. In this part, diffusion is assumed to be the only mechanism producing the gradient decay in the micro canals. According to Fick's diffusion law :<SPAN style="position: relative; display: inline-block; width: 100%; text-align: center;"><SPAN class="frac"><SPAN>du</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></SPAN>
                                     = C<SUB>diff</SUB><SPAN class="frac"><SPAN>d<SUP>2</SUP>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dx<SUP>2</SUP></SPAN></SPAN>(1)
                                </SPAN></P><P>C<SUB>diff</SUB> is assumed to be constant inside the conduit and depends on the material used.</P><P>The equation (1) can be solved with Euler’s method and we find the NGF concentration gradient at the position x and time t. We displayed our results showing a decrease in the concentration of NGF (u(x,t)) depending on the distance of the conduit x.</P></DIV><DIV class="block half center"><P>We used the following parameters for the model: <SUP>[8]</SUP></P><TABLE class="tableData"><TBODY><TR><TD>Length of the conduit: L</TD><TD>0.1 cm </TD></TR><TR><TD>Diffusion coefficient of NGF : Cdiff</TD><TD>7,8*10<SUP>-7</SUP> cm<SUP>2</SUP>.s<SUP>-1</SUP></TD></TR><TR><TD>Time length of the experiment: t_final</TD><TD>3 600 s </TD></TR></TBODY></TABLE></DIV><DIV class="block full"><DIV class="legend"><B>Table 4: </B>Fick's diffusion law parameters</DIV></DIV><DIV class="block full"><P style="text-align: center;">We obtain the following graphs: </P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 9: </B>NGF gradient</DIV></DIV><DIV class="block full"><H3>Optimization of the NGF gradient</H3></DIV><DIV class="block two-third center"><P>To optimize the accuracy of the NGF gradient we interpolate the curve u(x)=f(x). Consequently, we obtain the f polynomial function easier to derive and a polynomial function of the gradient with a better accuracy than with the first method.</P><P>With the same parameters as with the previous model we obtain the following graphs: </P></DIV><DIV class="block full"><H3>Analysis of the model </H3><P><I>To validate the model, we vary two parameters (L and C<SUB>diff</SUB>) to verify if the program corresponds to a diffusion phenomenon described in Fick’s second law of diffusion. </I></P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 10: </B>Fick's second law model validation (a)</DIV></DIV><DIV class="block full"><P>Observations:<OL style="text-align: left; list-style-type: disc;"><LI>When the length of the conduit increases but the duration of the experiment is fixed the NGF doesn’t have the time to diffuse in the entire conduit.</LI><LI>For instance, with a t_final= 3 600s the NGF molecules can’t diffuse further than x=0.2cm.</LI></OL></P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 11: </B>Fick's second law model validation (b)</DIV></DIV><DIV class="block full"><P>The higher the diffusion coefficient, the faster the molecules will diffuse in the conduit. Indeed, we observe in the model that with a fixed t_final:<OL style="text-align: left; list-style-type: disc;"><LI>NGF concentration at x=0.1 cm is 675 ng.ml<SUP>-1</SUP> for a diffusion coefficient C<SUB>diff</SUB> = 15*10<SUP>-7</SUP> cm<SUP>2</SUP>.s<SUP>-1</SUP></LI><LI>For a diffusion coefficient two times lower, the NGF concentration is 380 ng.ml<SUP>1</SUP></LI></OL></P><P>The results confirm the prediction of the Fick’s law model. </P></DIV><DIV class="block full"><P>When the time length of the experiment lasts from 1 hour to 2 hours, the concentration of NGF is almost homogeneous in the entire conduit. At the end of the conduit, for x= 0.1 cm, the concentration of NGF equals to 910 ng.ml-1 when t_final= 7 200s whereas the concentration is 3 90 ng.ml<SUP>-1</SUP> when t_final=3 600s. </P><P>It is interesting to observe that when the duration of the experiment increases, the stationary regime is established: the NGF concentration in the conduit becomes independent of the position and time. Indeed, the concentation gradient of NGF in the conduit moves toward 0 for any position. </P></DIV></DIV></DIV><DIV class="block full bothContent"><DIV class="block dropDown" id="Growth"><H4>Neurons growth in the presence of NGF</H4></DIV><DIV class="block hiddenContent"><DIV class="block title"><H1>Neurons growth in the presence of NGF</H1><P><I>In this part our goal is to determine the length of the neurite outgrowth (g(t)) in response to the gradient concentration of NGF. This step is the last one in our neurotrophin modelization. It aims at building a persistent model which should  give two relevant pieces of information regarding the use of the interface NeuronArch :
                            -The model must be able to indicate an estimated value of the time needed for the nerves to grow of a certain distance
                            -The model must be of use to provide the optimized parameters to boost the nerves growth
                            </I></P></DIV><DIV class="block full"><H3>Explanation of the model</H3></DIV><DIV class="block full"><H4 style="text-align: left">Baseline growth rate: </H4><P>In the mathematical model studied <SUP>[8]</SUP>, neurites grow at a constant growth rate defined as the baseline growth rate G0  when the concentration is below the threshold (assumed to be 995 ng.mL<SUP>-1</SUP>). Neurites stop growing when the NGF concentration is higher than the threshold concentration. The value for the baseline growth rate G0 has been fixed at 20 <FONT face="Raleway">μ</FONT>m.h<SUP>-1</SUP> for this model. </P><H4 style="text-align: left">Concentration Gradient:</H4><P>The extent of directional guidance is gradient steepness-dependent provided that the concentration gradient reaches the threshold value. The gradient factor k is a gradient steepness-dependent positive effect on the neurite growth rate. </P><P>In this model we assume that the baseline growth rate and the growth rate in the presence of concentration gradient follow an additive rule. This can be explained by the fact that both the NGF concentration and its gradient can individually contribute to neurite extension. The equation governing neurite outgrowth thus becomes:<SPAN style="position: relative; display: inline-block; text-align: center; width: 100%"><SPAN class="frac"><SPAN>dg</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></SPAN>
                                     = G<SUB>0</SUB> + k
                                    <SPAN class="frac"><SPAN><FONT face="Raleway">∂</FONT>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom"><FONT face="Raleway">∂</FONT>x</SPAN></SPAN>
                                     |<SUB>(g(t),t)</SUB> (4)</SPAN></P></DIV><DIV class="block full"><P>We can introduce a time parameter Tlag because the time taken to transmit the NGF signal is finite. The experiments show that the time lag for the cells to respond to NGF is approximately 1 day. The experiments show:
                                if t <FONT face="Raleway">≤</FONT> T<SUB>lag</SUB> : <SPAN class="frac"><SPAN>dg</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></SPAN> = 0
                                else, if: t <FONT face="Raleway">≥</FONT> T<SUB>lag</SUB> : <SPAN class="frac"><SPAN>dg</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></SPAN> = G<SUB>0</SUB> + k
                                    <SPAN class="frac"><SPAN><FONT face="Raleway">∂</FONT>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom"><FONT face="Raleway">∂</FONT>x</SPAN></SPAN>
                                 |<SUB>(g(t),t)</SUB></P></DIV><DIV class="block full"><H3>Euler's Method</H3></DIV><DIV class="block full"><P>To solve the equation (4) we are using Euler’s method forward because the gradient concentration of NGF depends on the length of the neurite (since neurites consume NGF). 
                                The Equation (4): <SPAN style="position: relative; display: inline-block; width: 100%; text-align: center;"><SPAN class="frac"><SPAN>dg</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></SPAN> = G<SUB>0</SUB> + k
                                    <SPAN class="frac"><SPAN><FONT face="Raleway">∂</FONT>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom"><FONT face="Raleway">∂</FONT>x</SPAN></SPAN>
                                 |<SUB>(g(t),t)</SUB></SPAN>
                                Can be written as: <SPAN style="position: relative; display: inline-block; width: 100%; text-align: center;">  
                                    g' = G<SUB>0</SUB> + k*f(g,t)
                                </SPAN>
                                Which can be written as : <SPAN style="position: relative; display: inline-block; width: 100%; text-align: center;"><SPAN class="frac"><SPAN>g<SUB>n+1</SUB> - g<SUB>n</SUB></SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom">dt</SPAN></SPAN>
                                     |<SUB>(g(t),t)</SUB> = G<SUB><FONT face="Raleway">θ</FONT></SUB> + k
                                    <SPAN class="frac"><SPAN><FONT face="Raleway">∂</FONT>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom"><FONT face="Raleway">∂</FONT>x</SPAN></SPAN>
                                     |<SUB>(g(t),t)</SUB></SPAN>
                                We can therefore have an expression of g<SUB>n+1</SUB>: <SPAN style="position: relative; display: inline-block; width: 100%; text-align: center;">  
                                    g<SUB>n+1</SUB> = g<SUB>n</SUB> + dt*[G<SUB>0</SUB> + k
                                    <SPAN class="frac"><SPAN><FONT face="Raleway">∂</FONT>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom"><FONT face="Raleway">∂</FONT>x</SPAN></SPAN>
                                     |<SUB>(g(t),t)</SUB>]
                                </SPAN>
                                With initial values of g<SUB><FONT face="Raleway">θ</FONT></SUB>, t<SUB><FONT face="Raleway">θ</FONT></SUB> and 
                                    <SPAN class="frac"><SPAN><FONT face="Raleway">∂</FONT>u</SPAN><SPAN class="symbol">/</SPAN><SPAN class="bottom"><FONT face="Raleway">∂</FONT>x</SPAN></SPAN>
                                     |<SUB>(g(t),t)</SUB>
                                  we can find all the values of the g.</P></DIV><DIV class="block full"><DIV class="legend"><B>Figure 12: </B>Schematic representation of NGF diffusion </DIV></DIV><DIV class="block full"><H3>Solving the Model</H3></DIV><DIV class="block full"><P>We noticed that there was a mistake in the article regarding the gradient steepness-dependent factor k. Therefore, by examining the results of the article, we were able to find a new coherent value of k, different from what was originally written in the article. To see the details of the mistake and our strategy to find the right value, <A href="https://static.igem.org/mediawiki/2018/2/28/T--Pasteur_Paris--ModelisationMistakes.pdf" style="font-weight: bold ; color:#85196a;" target="__blank">click here.</A></P><P>In the following graphs, the red curve corresponds to a nerve growth inside a unidimensional canal without any NGF while the blue one corresponds to the situation where there is an NGF gradient inside the canal. </P><P>Our strategy to compute this phenomenon is the following. </P><P>For each different time, (spaced by the value dt) the position (in cm) of an axon is put inside two unidimensional matrices, g and g<SUB>control</SUB>. The matrix g holds the values of positions when there is a gradient of NGF, while there isn’t for g<SUB>control</SUB>. At each time and for the corresponding position, the script we used to calculate the gradient of NGF (in part 2 ) is ran with those new parameters. As we obtain the value of the gradient of NGF at this time and at a position g<SUB>n</SUB>, we can calculate the new position at the end of the axon g<SUB>n+1</SUB> by using the formula written above. Tlag is set as 10 000 s for the rest of the modeling. </P><P>The first set of parameters we use is the following : </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD>Length of the device L </TD><TD>0.4 cm </TD></TR><TR><TD>Time of the experiment t_final</TD><TD>100 000 s </TD></TR><TR><TD>Initial concentration of NGF U1 </TD><TD>995 ng.mL<SUP>-1</SUP></TD></TR></TBODY></TABLE><DIV class="block full"><DIV class="legend"><B>Table 5: </B>Parameters for testing Euler's method (a)</DIV></DIV></DIV><DIV class="block two-third center"><DIV class="legend"><B>Figure 13: </B>Evolution of the Nerve growth as a function of time</DIV></DIV><DIV class="block full"><P>Not only does the result clearly demonstrates the need of an NGF gradient in order to boost the nerve growth, it also indicates how fast will a neuron grow in a certain amount of time, knowing the NGF concentration at the beginning of the canal.</P><P>We then carried on the study regarding two different parameters.</P></DIV><DIV class="block title"><H4 style="text-align: left;">Dependence of the length of the canal </H4></DIV><DIV class="block full"><P>We studied the influence of the length of the canal on the nerve growth. We worked with 2 sets of parameters:</P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD>Set 1</TD><TD>Set 2</TD></TR><TR><TD>Length of the device L</TD><TD>0.4 cm</TD><TD>0.1 cm </TD></TR><TR><TD>Time of the experiment t_final</TD><TD>50 000 s </TD><TD>50 000 s </TD></TR><TR><TD>Initial concentration of NGF U1</TD><TD>995 ng.mL<SUP>-1</SUP></TD><TD>995 ng.mL<SUP>-1</SUP></TD></TR></TBODY></TABLE><DIV class="block full"><DIV class="legend"><B> Table 6: </B> Parameters for testing Euler's method (b)
                            </DIV><DIV class="block half"><DIV class="legend"><B>Figure 14: </B>SET 1: L=0.4 cm</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 15: </B>SET 2: L=0.1 cm</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 16: </B>SET 1: L=0.4 cm</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 17: </B>SET 2: L=0.1 cm</DIV></DIV><DIV class="block full"><P>The more the length of the device rises, the longer it will take for the NGF concentration to be homogenous inside the canal. Therefore, to boost the growth nerve by having a gradient concentration of NGF in the media, the length of the device must be taken into account. The difference is significant since, at t=50 000 s, the magnitude of the gradient concentration of NGF in the canal worth 10<SUP>3</SUP> when L = 0.4 cm while it worth 10<SUP>-5</SUP> when L= 0.1 cm.</P><P>If the length is too small, the gradient concentration won’t be high enough to boost the growth nerves significantly, as shown in the graph Evolution of the nerve growth above.</P></DIV><DIV class="block title"><H4 style="text-align: left;">Dependence of the initial concentration </H4></DIV><DIV class="block full"><P>We studied the influence of the length of the canal on the nerve growth. We worked with two sets of parameters </P><TABLE class="tableData" style="margin: auto;"><TBODY><TR><TD>Set 1</TD><TD>Set 2</TD></TR><TR><TD>Length of the device L</TD><TD>0.4 cm</TD><TD>0.1 cm </TD></TR><TR><TD>Time of the experiment t_final</TD><TD>50 000 s </TD><TD>50 000 s </TD></TR><TR><TD>Initial concentration of NGF U1</TD><TD>400 ng.mL<SUP>-1</SUP></TD><TD>995 ng.mL<SUP>-1</SUP></TD></TR></TBODY></TABLE><DIV class="block full"><DIV class="legend"><B>Table 7:</B>Parameters for testing Euler's method (c) </DIV></DIV><P>The results are shown below.</P></DIV><DIV class="block half"><DIV class="legend"><B>Figure 18: </B>SET 1</DIV></DIV><DIV class="block half"><DIV class="legend"><B>Figure 19: </B>SET 2</DIV></DIV><DIV class="block full"><P>The results show that the more the initial concentration of NGF increases, the more its gradient reaches higher values and therefore the faster the neurons grow. It would appear that increasing the initial concentration of NGF would help to boost the nerve growth. However, if the concentration of NGF is too high, it would cause the opposite effect as neurons would start to die. Finding the right compromise, depending on the length of our final device and the coefficient diffusion of NGF in the future media, will be of paramount importance. Finding this compromise will be possible thanks to our model. </P></DIV><DIV class="block title"><H3>COMPARISON WITH EXPERIMENTAL WORKS</H3></DIV><DIV class="block full"><P>Our modeling work has permitted us to study the secretion, diffusion, and influence of NGF on the growth of the neurons.  With our model, we are able to optimize two parameters: the length of the microchannels and the initial concentration. 
The wet lab took into consideration our results to do the experiments on the influence of the concentration of NGF on the growth of the axons. Indeed, they observed the growth of the axons of E18 cortex cells for different concentrations of NGF: 0, 50, 250, 500, 750 and 900 ng/mL.  The wet lab’s results were coherent with our model.
</P><P>The experiments show that until a certain concentration the growth of the neurons increases with the presence of NGF. For a concentration between 250 and 750 ng/mL, the presence of NGF increases significantly the growth of the axons. On the model (Figure 20 and 21), the higher the NGF concentration, the higher the gradient concentration of NGF so the faster the neurons will grow.</P><DIV class="block full"><DIV class="legend"><B>Figure 20: </B> Experimental results of the wet lab showing the importance of the growth of the axons for different concentrations of NGF. The <FONT face="Raleway">β</FONT>-III Tubulin is a bio marker of neuron cell differentiation that indicates the growth of axons (to know more about the experimental results of the cell culture, click <A href="https://2018.igem.org/Team:Pasteur_Paris/Results" style="font-weight: bold;" target="_blank">here</A>)</DIV></DIV><DIV class="block full"><DIV class="legend"><B>Figure 21:</B> The modeling results showing the nerves growth (cm) for different concentrations of NGF (purple: 0 ng/mL, blue: 250 ng/mL, red: 500 ng/mL, yellow: 750 ng/mL)</DIV></DIV><P>The model helped the wet lab establish the concentration limit of NGF above which the NGF doesn’t have any more influence on the growth of the neurons. The wet lab’s concentration limit is coherent with ours: their concentration limit is approximately 900 ng/mL whilst the model shows a concentration limit of 995 ng/mL<SUP>[8]</SUP>.</P><P>The wet lab has done the series of experiments on a 96 wells plate in order to optimize the number of samples. The next step for the wet lab is to experimentally verify the influence of the length of the microchannels in the microfluidic chip on the growth of the nerves. The model is able to provide information on the optimization of the length of the microchannels which could be of use for the wet lab. Another improvement would be to calculate the diffusion coefficient in the microfluidic chip media.  
</P></DIV><DIV class="block title"><H3>PERSPECTIVES </H3></DIV><DIV class="block full"><P>Our model will be used to prototype the final device to help and establish the NGF concentration needed to control nerves’ growth. The length of the nerves needed to reach the interface depends on the individual. As mentioned in the <A href="https://2018.igem.org/Team:Pasteur_Paris/Scenario" style="font-weight: bold ; color:#85196a;" target="_blank">design scenario</A>, chemical induction for bacteria regarding NGF production might be considered. Since the model manages to link induction to diffusion to nerves growth, it will enable to know how much NGF needs to be produced for each individual.</P><P>The next step consists in keep trying to get in touch with the authors of the article or contacting other experts to make our model completely fulfill its major role in NeuronArch.  
</P></DIV></DIV></DIV></DIV></DIV><DIV class="block title" id="Introduction"><H1 id="Mechanical"> Second aspect modeled : mechanical modeling </H1></DIV><DIV class="block two-full"><P>Neuronarch aims at making the prosthesis of the future and making it more comfortable and protective for the patient. For this sake and to facilitate surgical interventions we modeled the behavior of a bone under mechanical stress. We presented our tools and scripts to Dr. Laurent Sedel, an orthopedic surgeon at Hôpital Lariboisière and researcher at the Hôpital Ambroise Paré – Hôpitaux Universitaires Paris Ile-de-France Ouest, in the hopes of using our tools to improve the lifespan of prosthesis.</P></DIV><DIV class="block two-third center"><DIV class="legend">Representation of our calculated model of the deformation and the stress (von Mises) on a straight line inside a humerus</DIV></DIV><DIV class="block title"><H3><A href="https://static.igem.org/mediawiki/2018/2/2c/T--Pasteur_Paris--MechanicalModeling.pdf" style="font-weight: bold ; color:#85196a;" target="__blank">Download here the full PDF of the Mechanical Modeling</A></H3></DIV></DIV></DIV><DIV class="block title"><H1 id="References">REFERENCES</H1></DIV><DIV class="block full"><UL style="text-align: left;list-style: disc;"><LI style="list-style-type: decimal;">M. Stamatakis and N. V. Mantzaris, &quot;Comparison of deterministic and stochastic models of the lac operon genetic network,&quot; Biophys. J., vol. 96, no. 3, pp. 887-906, 2009.</LI><LI style="list-style-type: decimal;">A. Y. Weiße, D. A. Oyarzún, V. Danos, and P. S. Swain, &quot;Mechanistic links between cellular trade-offs, gene expression, and growth,&quot; Proc. Natl. Acad. Sci., vol. 112, no. 9, pp. E1038-E1047, 2015.</LI><LI style="list-style-type: decimal;">R. Milo, &quot;Useful fundamental BioNumbers handout.doc,&quot; pp. 1-2, 2008.</LI><LI style="list-style-type: decimal;">M. S. Packer, H. A. Rees, and D. R. Liu, &quot;Phage-assisted continuous evolution of proteases with altered substrate specificity,&quot; Nat. Commun., vol. 8, no. 1, 2017.</LI><LI style="list-style-type: decimal;">H. Benabdelhak et al., &quot;A specific interaction between the NBD of the ABC-transporter HlyB and a C-terminal fragment of its transport substrate haemolysin A,&quot; J. Mol. Biol., vol. 327, no. 5, pp. 1169-1179, 2003.</LI><LI style="list-style-type: decimal;">Defining the concentration gradient of nerve growth factor for guided neurite outgrowth, XCao M.SShoichet, March 2001</LI><LI style="list-style-type: decimal;">Immobilized Concentration Gradients of Neurotrophic Factors Guide Neurite Outgrowth of Primary Neurons in Macroporous Scaffolds, Moore K, MacSween M, Shoichet M, feb 2006</LI><LI style="list-style-type: decimal;">Mathematical Modeling of Guided Neurite Extension in an Engineered Conduit with Multiple Concentration Gradients of Nerve Growth Factor (NGF), Tse TH, Chan BP, Chan CM, Lam J, sep 2007</LI><LI style="list-style-type: decimal;">Mathematical modeling of multispecies biofilms for wastewater treatment, Maria Rosaria Mattei, november 2005</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>