<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Pasteur_Paris_Fighting skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Pasteur Paris/Fighting</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>&quot;&quot;</TITLE></P><DIV id="Menu"><DIV id="naviconwrapper"><DIV id="naviconbar"><DIV id="navicons"><UL><LI id="project"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">Human Practices</A></LI><LI id="design"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Product Design</A></LI><LI id="parts"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">labwork</A></LI><LI id="achievements"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About Us</A></LI></UL></DIV></DIV></DIV><DIV id="navtextwrapper"><DIV id="navtext"><UL id="project-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Description">Overview</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Growing_Back_Nerves">Reconnect Nerves</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Fighting">Fight Infections</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Model">Model</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Membrane">Physical Interface</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Kill">Kill-Switch</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Entrepreneurship">Entrepreneurship</A></LI></UL><UL id="hp-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Human_Practices">Integrated HP</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Public_Engagement">Education &amp; Engagement</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Safety">Safety &amp; Ethics</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Law">Law</A></LI></UL><UL id="design-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Technicals">Methodology</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Scenario">Scenario</A></LI></UL><UL id="parts-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Basic_Part">Basic Parts</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Improve">Improve</A></LI></UL><UL id="labwork-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Notebook">Notebook</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Experiments">Protocols</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Results">Results</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/InterLab">InterLab</A></LI></UL><UL id="achievements-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Medals">Medals</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Awards">Awards</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Demonstrate">Demonstrate</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Collaborations">Collaborations</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/GiantJamboree">Giant Jamboree</A></LI></UL><UL id="aboutus-sub"><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Team">Team Members</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Acknowledgements"> Acknowledgements</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Attributions">Attributions</A></LI><LI><A class="navsub1" href="https://2018.igem.org/Team:Pasteur_Paris/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV><DIV class="clearfix" id="smallnav"><UL class="clearfix"><LI id="home_small"><A href="https://2018.igem.org/Team:Pasteur_Paris">Home</A></LI><LI id="project_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Project">Project</A></LI><LI id="hp_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/HP">HP</A></LI><LI id="design_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Applied_Design">Design</A></LI><LI id="parts_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Parts">Parts</A></LI><LI id="labwork_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Labwork">Labwork</A></LI><LI id="achievements_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/Achievements">Achievements</A></LI><LI id="aboutus_small"><A href="https://2018.igem.org/Team:Pasteur_Paris/AboutUs">About us</A></LI></UL><A href="#" id="pull">Menu</A></DIV><DIV id="GeneralContent"><DIV id="index" class="block"><DIV id="indexContent"><P><A href="#Background" class="link">Background</A></P><P><A href="#Solution" class="link">Our Solution</A></P><P><A href="#Results" class="link">Results</A></P><P><A href="#References" class="link">References</A></P></DIV></DIV><DIV id="MainContent"><DIV class="block title"><H1 id="Background">BACKGROUND</H1></DIV><DIV class="block full"><P>Biomedical implants, such as prostheses, catheters and several other implanted devices, can improve the patient’s quality of life. Nevertheless, because they are artificial, handled from outside the body, and rarely have an antimicrobial property, they increase the risk of infection. In fact, device-associated infections account for 25.6% of all health-care-associated infections in the USA <SUP>[1]</SUP>.</P><P>Orthopedic implant infections are caused by the development of pathogenic microorganisms on an inert material, allowing bacterial adhesion and biofilm formation on the implant. Biofilm-related infections frequently lead to chronic infections and require implant replacement. They are therefore responsible for a significant medical (on both the patient’s and surgeon's point of view) and economic burden <SUP>[2] [3]</SUP>.</P></DIV><DIV class="block title"><H3 style="text-align: left;">What are biofilms? </H3></DIV><DIV class="block full"><P>A biofilm is a community of microorganisms surrounded by an extracellular matrix produced by the microbes themselves. They can attach to various surfaces whether living or inert. Biofilm formation is a process that begins with the attachment of the microorganism, in our case bacteria, to a surface on which they can grow under the control of specific biofilm genes expression. Bacteria living in a biofilm produce extracellular polymeric substances (EPS) composed mostly of water with polysaccharides, proteins, lipids and extracellular DNA <SUP>[4] [5]</SUP>.</P></DIV><DIV class="block title"><H3 style="text-align: left;">Stages of Biofilm formation</H3></DIV><DIV class="block half" style="margin-bottom: 0;"><P>The process of biofilm formation applies to both Gram-negative and Gram-positive bacteria. It involves different stages, from adhesion of planktonic cells, microcolony formation, EPS production and biofilm maturation to dispersal stage, as shown in Figure 1. </P></DIV><DIV class="block half" style="margin-top: 0;"><DIV class="legend"><B>Figure 1: </B>Biofilm formation process. Adapted from E. Maunders et al., 2017<SUP>[6]</SUP></DIV></DIV><DIV class="block title"><H3 style="text-align: left;">How can biofilms form on an implanted device and elude immunity and antibiotic treatments?</H3></DIV><DIV class="block full"><P>A major issue in the fight against bacteria is their organization as biofilms. These bacterial aggregates strongly adhere to the biomaterial surfaces. In this bio-mechanical configuration, the implant-infecting biofilms can elude innate and adaptive host defenses as well as antibiotic therapies <SUP>[7]</SUP>.</P><P>In fact, the biomaterial becomes encapsulated in layers of tissues which isolate the biofilm from the surrounding tissues and so shields it from the immune system. At the same time, the extracellular matrix also protects the microorganisms from immune cells <SUP>[8]</SUP>.</P><P>Biofilms, by their architecture, give the bacteria that compose them an inherent resistance to antimicrobial agents by making it difficult for them to reach the bacteria <SUP>[4]</SUP>. Antibiotic treatments may lead to the removal of most of the bacterial population, but some of the bacteria frequently survive and can reconstitute the biofilm later when the antibiotic therapy is stopped <SUP>[7]</SUP>. The evasion of host defense and tolerance to antibiotics leads to the establishment of chronic infections.</P></DIV><DIV class="block title"><H3 style="text-align: left;">Which microorganism is the most frequently responsible for prosthetic-joint infections?</H3></DIV><DIV class="block half"><P><I>Staphylococcus aureus</I>, a Gram-positive coccus, is the most commonly isolated microorganism from orthopedic-associated infections, accounting for more than 40% of hip and knee prosthetic-joint infections shown by Table 1 [9]. Other species such as <I>Staphylococcus epidermidis</I> and Gram-negative bacteria including <I>Pseudomonas aeruginosa</I> and <I>Escherichia coli</I> are less frequent implant-infecting bacteria. The situation is alarming as implant-infecting <I>S. aureus</I> strains have high rates of antibiotic resistance against commonly used antibiotics (<I>e.g.</I> methicillin, gentamycin) <SUP>[7]</SUP>. </P></DIV><DIV class="block half"><DIV class="legend"><B>Table 1: </B>Distribution of organisms (the top 10 ranking organisms are identified to genus/species level) causing surgical site infection in patients undergoing knee and hip arthroplasty. <SUP>[9]</SUP></DIV></DIV><DIV class="block title"><H3 style="text-align: left;">Which treatments are currently used?</H3></DIV><DIV class="block full"><P>In most cases, the treatment after a prosthetic-joint infection involves the removal of the implant followed by several weeks of antibiotic treatment. Apart from the fact that it presents a heavy burden for the patient, the extended hospitalization and potential re-operation represent a considerable excess of cost for the health industry. A study led in France <SUP>[9]</SUP> estimates that the revision of infected prostheses multiplies the cost of the total procedure by a 3.6 factor.  </P><P>Re-operation is almost always necessary because bacteria growing in biofilms are more resistant to antimicrobial therapies and dissolving biofilms using antibiotics is a complicated task. To fight this serious problem, new treatment strategies have already appeared, such as antimicrobial peptides (AMP), bacteriophages, matrix-degrading enzymes or quorum sensing inhibitors <SUP>[4]</SUP>. An alternative approach to fight against prosthetic infections is to directly prevent pathogenic bacteria from forming a biofilm, thus blocking the pathogen development of resistance to the immune system and antibiotics. </P></DIV><DIV class="block title"><H1 id="Solution">OUR SOLUTION</H1><I><P>Our strategy is to concentrate our efforts on fighting against <I>S. aureus</I>, since it is the leading cause of infections. It is known that over time, <I>S. aureus</I> strains can develop resistance to antimicrobial peptides or toxin/antitoxin systems due to the selective pressure <SUP>[10]</SUP>. We decided to focus on preventing the formation of a biofilm by S. aureus, because it is the state in which pathogens develop resistance to both the immune system and to antibiotics <SUP>[11] [12]</SUP>. To overcome the problem, we chose the disturbance of quorum sensing as an anti-biofilm strategy. Communication within a bacterial community relies on the production and detection of signal molecules, in a system called quorum sensing. For instance, pathogens use quorum sensing to regulate and coordinate the biofilm’s architecture and the production of toxins and virulence factors <SUP>[12]</SUP>. This is why we focused on quorum sensing inhibitors as a promising anti-biofilm strategy.</P></I></DIV><DIV class="block title"><H3 style="text-align: left;" id="Quorum">Quorum sensing in <I>S. aureus</I></H3></DIV><DIV class="block full"><P><B>Quorum sensing (QS)</B> is a mechanism of cell-cell communication. It works thanks to the regulation of gene expression that produces and releases chemical signal molecules called <B>auto-inducers</B> in response to fluctuations in cell-population density <SUP>[13]</SUP>. There are three major types of QS systems: the acylhomoserine lactone (AHL) QS system in Gram-negative bacteria, the <B>autoinducing peptide</B> (AIP) QS system in Gram-positive bacteria and the autoinducer-2 (AI-2) QS system in both Gram-negative and -positive bacteria <SUP>[14]</SUP>.</P><P><I>S. aureus</I> is a gram-positive coccus that uses AIP quorum sensing to regulate the colonization and the production of virulence factors <SUP>[13]</SUP>. This microorganism is able to go from a benign phase to invasion thanks to a switch in gene expression controlled by the <B>accessory gene regulator (<I>agr</I>)</B> QS system <SUP>[15]</SUP>. This system is a signalling circuit that produces and senses AIP. Its chromosomal locus encodes two transcripts: RNAII and RNAIII (Figure 2).
                </P><P>
                    The first system, <B>RNAII</B>, is a polycistronic transcript that is produced by the <B>promoter P2</B> and encodes for four proteins – <I>agr</I>A, <I>agr</I>B, <I>agr</I>C and <I>agr</I>D – that allow the sensing properties of <I>S. aureus</I><SUP> [13] [16]</SUP>. First, a small peptide is encoded by <I>agr</I>D, and is transformed into an <B>auto-induced peptide (AIP)</B> by AgrB, a transmembrane protein. AIPs from all the bacteria accumulate in the extracellular environment. Another transmembrane protein, AgrC, binds the extracellular AIPs and in turn, phosphorylates AgrA (the intracellular regulator). When phosphorylated, AgrA has an increased affinity for promoters P2 and P3<SUP> [4] [6]</SUP>. Thus, there is an upregulation of all the <I>agr</I> system. 
                </P><P>
                    The <B>promoter P3</B> allows the transcription of the <B>RNAIII transcript</B>, that is the regulatory molecule that upregulates the production of numerous secreted toxins <SUP>[16]</SUP>.
                    When colonies of <I>S. aureus</I> multiply and grow, they secrete a protein called <B>RAP</B>. RAP accumulates outside the cells and when it reaches a threshold, it induces a histidine phosphorylation of its target molecule, <B>TRAP</B>. When TRAP is phosphorylated, it activates the <I>agr</I> system and the production of toxins. 
                </P><DIV class="legend"><B>Figure 2: </B>The accessory gene regulator (<I>agr</I>) system in <I>S. aureus</I>. The P2 operon encodes the RNAII transcript, which upregulates the production of AIPs.</DIV></DIV><DIV class="block title"><H3 style="text-align: left;" id="Detect">Detection of <I>S. aureus</I> by our biofilm and inhibition of QS</H3></DIV><DIV class="block full"><P>
                    We wanted our interface to produce a protein that would inhibit the development of <I>S. aureus</I> in the environment of the implant, but we did not want this protein to be secreted continuously and to accumulate outside the biofilm. We wanted our biofilm to start producing growth inhibiting molecules only in the presence of a pathogen. We decided to use the Biobrick <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_I746100" style="font-weight: bold ; color:#85196a;" target="__blank">BBa_I746100 from iGEM Cambridge 2007 Team</A> and <B>improve</B> it by optimizing it for our chassis:  <I>E. coli</I> BL21 strain. </P><P>
                    We engineered the bacteria composing our biofilm by introducing the genes encoding for AgrC and AgrA proteins, the two proteins responsible for the detection of AIPs. We encoded them under the constitutive promoter BBa_J23107, from <A href="https://2006.igem.org/wiki/index.php/Berkeley" style="font-weight: bold ; color:#85196a;" target="__blank">iGEM Berkeley 2006 Team.</A> When AIPs are detected in the environment by the transmembrane protein AgrC, AgrA is phosphorylated and has an increased affinity for the promoter P2. </P><P>
                    In our engineered bacteria, P2 encodes for a protein called <B>RNAIII Inhibiting Peptide (RIP)</B>. RIP is a small peptide (seven amino-acids) that has been proven to inhibit biofilm formation of <I>S. aureus</I>. Indeed, <B>RIP competes with RAP</B> by phosphorylating TRAP, which leads to an inhibition of TRAP phosphorylation. This leads to the inhibition of RNAIII transcription and thus to the attenuation of virulence factor production, inhibition of cell-cell communication and decreased adhesion capacities (Figure 3) <SUP>[16] [17]</SUP>.
                </P><P>
                    Because we needed our peptide to be secreted by our biofilm, we targeted RIP to the periplasm of <I>E. coli</I> by using the Sec-dependent Type II secretion system. Our peptide is fused to an amino-terminal signal sequence, that is recognized by the chaperone SecB, then addressed to SecA and translocated across the inner membrane through the SecYEG complex. One advantage of this system is that the signal sequence gets cleaved during translocation through SecYEG. Several different signal sequences have been characterized (all formed of 18 to 30 amino acids) <SUP>[18]</SUP>. Following the advice of <B>Dr. Jean-Michel Betton</B>, Research Director in Structural Microbiology at the Institut Pasteur, we chose to try two different signal sequences to export our peptides to the periplasm, which he knew were efficient: <B>MalE</B> and <B>DsbA</B>. Once in the periplasm, due to the small size of RIP (7 amino acids), a leaky release through the outer membrane of the bacteria should allow us to obtain RIP in the medium.
                </P><P>
                    To sum up, in the presence of <I>S. aureus</I> in the environment of our biofilm, our engineered bacteria secrete a small peptide, RIP, that competes with RAP and inhibits the formation of a pathogenic biofilm. Unlike common antibiotics, RIP inhibits cell-to-cell communication rather than killing pathogenic bacteria. Thanks to such an approach, we avoid the development of antibiotic resistance by pathogens and we force pathogens to remain in their planktonic stage, in which they stay susceptible to the host immune system. 
                </P><DIV class="block legend"><B>Figure 3: </B>The detection proteins AgrC and AgrA are present in the engineered bacteria composing our biofilm, and thus, can detect the AIPs in the environment if S. aureus is present. AIPs will bind to AgrC, that will phosphorylate AgrA. This will induce the production and secretion of RIP, that inhibits S. aureus QS and thus, biofilm formation.</DIV></DIV><DIV class="block title"><H1 id="Results">RESULTS</H1><I><P>Achievements:</P></I><UL style="text-align: left;list-style: disc;"><LI>Successfully cloned a biobrick coding for RIP secretion in pBR322 and in pSB1C3, creating a new part <A href="http://parts.igem.org/Part:BBa_K2616001"> Bba_K2616001 </A>.
                        </LI><LI>Successfully sequenced <A href="http://parts.igem.org/Part:BBa_K2616001"> Bba_K2616001 </A> in pSB1C3 and sent to iGEM registry.
                        </LI><LI>Successfully cultivated <I>S. aureus</I> biofilms in 96-well plates with different supernatants. Although there was a high variability in our results, and we used several protocols to overcome it, in one case, we were able to observe a reduction in biofilm formation in the presence of our RIP.</LI></UL><P><I>Next steps:</I><UL style="text-align: left;list-style: disc;"><LI>Clone the sensor device with inducible RIP production upon <I>S. aureus</I> detection.</LI><LI>Improve the characterization of RIP effect on biofilm formation with a more standardized assay.</LI></UL></P></DIV><DIV><P style="text-align: center;"> Visit the <A href="https://2018.igem.org/Team:Pasteur_Paris/Results" style="font-weight: bold ; color:#85196a;" target="__blank">RESULTS page</A>to get more information</P></DIV><DIV class="block title"><H1 id="References">REFERENCES</H1></DIV><DIV class="block full"><UL style="text-align: left;list-style: disc;"><LI style="list-style-type: decimal;">S. S. Magill et al., “Multistate Point-Prevalence Survey of Health Care–Associated Infections,” N. Engl. J. Med., vol. 370, no. 13, pp. 1198–1208, 2014.</LI><LI style="list-style-type: decimal;">T. Lamagni, S. Elgohari, and P. Harrington, “Trends in surgical site infections following orthopaedic surgery,” Curr. Opin. Infect. Dis., vol. 28, no. 2, pp. 125–132, 2015.</LI><LI style="list-style-type: decimal;">M. Sciences, “Annals of Microbiology and Research,” no. May, 2018.</LI><LI style="list-style-type: decimal;">L. Drago and M. Toscano, Biofilm formation and the biological response. Elsevier Ltd., 2016.</LI><LI style="list-style-type: decimal;">C. N. Melton and G. G. Anderson, Biofilms and Disease: A Persistent Threat, 4th ed. Elsevier Inc., 2018.</LI><LI style="list-style-type: decimal;">E. Maunders and M. Welch, “Matrix exopolysaccharides; the sticky side of biofilm formation,” FEMS Microbiol. Lett., vol. 364, no. 13, pp. 1–10, 2017.</LI><LI style="list-style-type: decimal;">C. R. Arciola, D. Campoccia, and L. Montanaro, “Implant infections: Adhesion, biofilm formation and immune evasion,” Nat. Rev. Microbiol., vol. 16, no. 7, pp. 397–409, 2018.</LI><LI style="list-style-type: decimal;">M. Ribeiro, F. J. Monteiro, and M. P. Ferraz, “Infection of orthopedic implants with emphasis on bacterial adhesion process and techniques used in studying bacterial-material interactions,” Biomatter, vol. 2, no. 4, pp. 176–194, 2012.</LI><LI style="list-style-type: decimal;">T. Lamagni, “Epidemiology and burden of prosthetic joint infections,” J. Antimicrob. Chemother., vol. 69, no. SUPPL1, pp. 5–10, 2014.</LI><LI style="list-style-type: decimal;">D. Lebeaux and J. Ghigo, “associées aux biofilms Quelles perspectives thérapeutiques,” 2012.</LI><LI style="list-style-type: decimal;">M. R. Shakibaie, “Bacterial Biofilm and its Clinical Implications,” Ann. Microbiol. Res., vol. 2, no. 1, pp. 45–50, 2018.</LI><LI style="list-style-type: decimal;">K. I. Wolska, A. M. Grudniak, Z. Rudnicka, and K. Markowska, “Genetic control of bacterial biofilms,” J. Appl. Genet., vol. 57, no. 2, pp. 225–238, 2016.</LI><LI style="list-style-type: decimal;">J. M. Yarwood and P. M. Schlievert, “Quorum sensing in Staphylococcus infections,” J. Clin. Invest., vol. 112, no. 11, pp. 1620–1625, 2003.</LI><LI style="list-style-type: decimal;">G. Brackman and T. Coenye, “Quorum Sensing Inhibitors as Anti-Biofilm Agents,” Curr. Pharm. Des., vol. 21, no. 1, pp. 5–11, 2014.</LI><LI style="list-style-type: decimal;">C. M. Waters and B. L. Bassler, “QUORUM SENSING: Cell-to-Cell Communication in Bacteria,” Annu. Rev. Cell Dev. Biol., vol. 21, no. 1, pp. 319–346, 2005.</LI><LI style="list-style-type: decimal;">Y. Gov, A. Bitler, G. Dell’Acqua, J. V. Torres, and N. Balaban, “RNAIII inhibiting peptide (RIP), a global inhibitor of Staphylococcus aureus pathogenesis: Structure and function analysis,” Peptides, vol. 22, no. 10, pp. 1609–1620, 2001.</LI><LI style="list-style-type: decimal;">A. Giacometti et al., “RNA III Inhibiting peptide inhibits in vivo biofilm formation by drug-resistant Staphylococcus aureus,” Antimicrob. Agents Chemother., vol. 47, no. 6, pp. 1979–1983, 2003.</LI><LI style="list-style-type: decimal;">R. Freudl, “Signal peptides for recombinant protein secretion in bacterial expression systems,” Microb. Cell Fact., vol. 17, no. 1, pp. 1–10, 2018.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>