<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_Genetics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Alberta/Genetics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/Genetics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Genetics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Genetics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/Genetics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/Genetics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="main-content"><DIV id="page-title"><SPAN id="page-title-text">
            PROJECT
        </SPAN></DIV><DIV id="contents-box"><UL id="subpage-links"><LI><A class="" href="https://2011.igem.org/Team:Alberta/project">Project Overview</A></LI><LI><A class="" href="https://2011.igem.org/Team:Alberta/Growth">Growth</A></LI><LI><A class="selected" href="https://2011.igem.org/Team:Alberta/Genetics">Genetics</A></LI><LI><A class="" href="https://2011.igem.org/Team:Alberta/EsterificationExtraction">Esterification &amp; Extraction</A></LI></UL></DIV><DIV id="page-content"><H2>Genetics</H2><H3>Assembly of Genetic Constructs</H3><P>We believe <I>Neurospora crassa</I> is a very strong model organism for synthetic biology, and we want it to be as easy as possible to work with. Its ease of use is largely complemented by its full genetic sequence being freely available at the website of the Broad Institute.<SUP>1</SUP></P><P>This year, we wanted to make things easy for you. We created a <A href="https://static.igem.org/mediawiki/2011/a/ae/RFC-82.pdf">new RFC</A> for assembling parts to be transformed into <I>Neurospora crassa</I>.</P><P>This RFC is a modification of last years BioBytes 2.0 assembly method, adapted to fit into the schema of transformation in <I>Neurospora crassa</I>. It delineates a clear, concise, and characterized method of assembly which allows genetic constructs to be assembled faster and more efficiently than the current BioBrick method.</P><P>Any genetic construct to be inserted into <I>Neurospora crassa</I> requires the following parts, assembled in the following order.</P><OL><LI>A 5’ homologous recombination region</LI><LI>The gene of interest. Can be broken down into:
                    </LI><LI>Promoter</LI><LI>Open Reading Frame</LI><LI>Terminator</LI><LI>A reporter gene (selectable marker). Most commonly a fungicide or antibiotic resistance. </LI><LI>A 3’ homologous recombination region</LI></OL><P>Homologous recombination regions are defined as ~1000 bp sections of DNA which correspond to sections of DNA in the <I>N. crassa</I> genome flanking the point of insertion. This means that anything present between these homologous recombination regions will be REMOVED from the <I>Neurospora crassa</I> genome in the properly selected transformant. This is useful for knocking out genes, but must be taken into account either way.</P><P>The gene of interest is defined as whatever gene is to be inserted into <I>N. crassa</I>. A reporter gene is defined as any gene that allows you to select for transformants. The most commonly used reporter genes are fungicide or antibiotic resistances (ex. Hygromycin B).</P><P>Because our new RFC allows assembly of these parts in a modular format, genetic constructs to be inserted into <I>N. crassa</I> can be mixed and matched in whatever ways you see fit. Swapping out promoters or selectable markers is as easy as running another ligation with your new part of choice.</P><CENTER><P>A typical genetic construct, linked by 7 unique ends. UTR=Untranslated Regions, also known as homologous recombination regions. S.M.=Selectable Marker, i.e reporter gene</P></CENTER><P>We believe this is a powerful tool for working in <I>Neurospora crassa</I>, because it allows quick, simple, easy genetic manipulation</P><H3>Strains</H3><P>Because wildtype <I>Neurospora crassa</I> has both homologous recombination as well as non-homologous end-rejoining, it is not ideal for genetic manipulation. While homologous recombination is ideal for accurately inserting genetic constructs into the genome, non-homologous end-rejoining simply inserts genetic constructs into random parts of the genome. This is not very useful for synthetic biology.</P><P>However, there is a solution. Mus strains of <I>Neurospora crassa</I> have their non-homologous end-rejoining machinery knocked out, and so have much greater stability when it comes to genetic manipulation. In fact, accurate transformation efficiencies are near 100%<SUP>2</SUP> in some cases. Mus strains are widely available from the Fungal Genetic Stock Centre, and can be used for all necessary transformations.</P><H3>Metabolic Engineering</H3><P>But what is all this genetic manipulation without a purpose? Genetic manipulation is just a tool; it’s what you use it for that counts.</P><P>To increase fatty acid in <I>Neurospora crassa</I>, we figured that we could modify its metabolic pathways to shunt more of its energy into Fatty Acid Synthesis, away from degradation. This buildup of fatty acid would allow us to harvest <I>N. crassa</I> and esterify to Fatty Acid Methyl- or Ethyl- esters; biodiesel. BUt how to increase fatty acid levels?</P><P>By introducing a primarily C16 thioesterase from e. coli, codon-optimized for <I>N. crassa</I>, we believe we can cleave off fatty acids attached to the Fatty Acid Synthase complex and increase free fatty acid (FFA) levels. However, we also realize that this higher FFA level may just be degraded anyway for more energy. That’s why we decided to knock out a fatty-acyl:CoA synthetase as well.</P><CENTER><P>The Plan. By introducing novel thioesterase activity, and knocking out degradation of FFA, fatty acid levels build up in the cell. Theoretically.</P></CENTER><P>The way we can accomplish this is elegant. Because genetic constructs for <I>Neurospora</I> require that two homologous recombination regions flank the sequence on the genome that will be removed after transformation, we figured: why not kill two birs with one genetic stone?</P><P>We created a construct that would both knock out our fatty-acyl:CoA synthetase, FadD1, as well as insert our novel thioesterase. Because homologous recombinations end up knocking out a section of the genomic sequence, it was easy to do. We assembled the following construct, using ~1000 bp of UTR (untranslated regions) both upstream and downstream of the FadD1 gene, to ensure proper homologous recombination.</P><P>After transforming <I>Neurospora crassa</I> conidia with this construct, all that needs to be done is to plate on an L-sorbose agar plate inoculated with hygromycin, and pick colonies of transformants.</P><H3>Results</H3><P>Unfortunately, results for our proof-of-concept transformation are still ongoing. However, the strength of this genetic assembly method and the proper submission of an RFC should be taken as a strong indicator of the direction and viability of our research. More parts for genetic transformation in <I>N. crassa</I> will be added as soon as they are made, with more available in time for Boston.</P><H3>References</H3><OL><LI>Galagan, et al. The genome sequence of the filamentous fungus <I>Neurospora crassa</I>. Nature 422, 859-868, 2003.</LI><LI>Yuuko Ninomiya, Keiichiro Suzuki, Chizu Ishii, and Hirokazu Inoue. Highly efficient gene replacements in <I>Neurospora</I> strains deficient for nonhomologous end-joining. PNAS 2004 101 (33) 12248-12253</LI></OL></DIV></DIV><DIV id="content-footer-box"><DIV class="footer-left-half"><DIV class="footer-left-col"><P><B>About our Wiki</B></P><P>Seven students from three departments across the University of Alberta campus aspired to complete an ambitious project over the course of the summer. This wiki is all about our successes and our many trials and tribulations.</P></DIV><DIV class="footer-right-col"><P><B>Search</B></P><P><B>Contact Us</B></P><P>University of Alberta</P><P>4-055 Katz Group Research Centre</P><P>University of Alberta</P><P>Edmonton, Alberta T6G 2E1</P><P>teamalbertaigem@gmail.com</P></DIV></DIV><DIV class="footer-right-half"><DIV class="footer-right-left-col"><P><B>Navigate Our Wiki</B></P><UL><LI><A href="https://2011.igem.org/Team:Alberta">Home</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Neurospora">Neurospora crassa</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Achievements/Overview">Results &amp; Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Overview">Methodology</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/HumanPractices/Overview">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/team/Students">Team</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Press">Interactive</A></LI></UL></DIV><DIV class="footer-right-right-col"><A style="color:#808080;" href="http://twitter.com/#!/Team_Alberta" class="twitter-follow-button">Follow @Team_Alberta</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Alberta/Genetics">http://2011.igem.org/Team:Alberta/Genetics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/Genetics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/Genetics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/Genetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/Genetics&amp;oldid=200536" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>