<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Alberta_Methodology_Notebook_Genetics"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Alberta/Methodology/Notebook/Genetics">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Alberta/Methodology/Notebook/Genetics&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Methodology/Notebook/Genetics&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Alberta/Methodology/Notebook/Genetics&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Alberta/Methodology/Notebook/Genetics" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Alberta/Methodology/Notebook/Genetics</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><DIV id="main-content"><DIV id="page-title"><SPAN id="page-title-text">
            METHODOLOGY
        </SPAN></DIV><DIV id="contents-box"><UL id="subpage-links"><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Overview" class="">Overview</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Parts" class="">Parts</A></LI><LI><A class="" href="https://2011.igem.org/Team:Alberta/Methodology/Protocols">Protocols</A></LI><LI><A href="#growth">Growth</A></LI><LI><A href="#genetics">Genetics</A></LI><LI><A href="#extraction">Extraction and Esterification</A></LI></UL><LI><A class="selected" href="https://2011.igem.org/Team:Alberta/Methodology/Notebook">Notebook</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Notebook/Growth">Growth</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Notebook/Genetics">Genetics</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Notebook/Esterification">Esterification</A></LI><LI><A class="" href="https://2011.igem.org/Team:Alberta/Safety">Safety</A></LI><LI><A href="#Section_1">Section I.</A></LI><LI><A href="#Section_2">Section II.</A></LI><LI><A href="#Section_3">Section III.</A></LI><LI><A href="#Section_4">Section IV.</A></LI><LI><A href="#References">References</A></LI></DIV><DIV id="page-content"><H2>Notebook</H2><H3>Genetics</H3><H4>Pre-Week 1</H4><UL><LI>Many of the members of our iGEM team are enrolled in BIOCHEMISTRY 482, a class designed to introduce students to the methodology and some of the techniques in the iGEM competition.  Over the course of the school year we have been learning about common synthetic biology themes and techniques, and reading papers related to the field. Many of them are research into possible projects that may turn into this years iGEM Team Alberta focus. Many presentations are given over possible projects.</LI><LI>Group Meetings to discuss the focus of this year's iGEM team. Narrowed it down to Biofuels, but still unclear as to what biofuel and in what organism. Choices are:
        </LI><LI>Organism:
             </LI><LI>Algae</LI><LI>Yarrowia Lipolytica, a fatty yeast</LI><LI>Neurospora Crassa, a common cellulose-degrading fungus</LI></UL><LI> Biofuel:
                </LI><LI>Bioethanol</LI><LI>Biodiesel (Fatty-acid Methyl Esters, Fatty-acid Ethyl Esters)</LI><LI>Bioalkanes (butane, pentane, etc.)</LI><LI>Higher order Bioalcohols (Butanol, pentanol, etc.)</LI><LI>Decided on Neurospora Crassa as organism of choice. The organism's ability to degrade cellulose is its strength for this project, although we are currently unfamiliar with how to manipulate it. We figure that degrading cellulose is a better starting place for a project instead of Y. Lipolytica's ability to accumulate fat. Also, we know more about N. crassa at this time.</LI><LI>Biodiesel is the biofuel of choice. We figure that it is manageable to increase fatty acid content in an organism in the short time available, and then convert that fatty acid chemically to biodiesel. Accumulating fatty acid seems much easier than trying to make and secrete a higher-order alkane or alcohol. Biodiesel also has advantages over bioethanol in that it has a greater energy density. </LI><LI>Research into all the ways we might be able to increase fatty acid content in N. crassa. Many genes are discussed, from many different organisms. All 11 members of the class pick a relevant metabolic gene, codon-optimize it for N. crassa, and submit a referenced report detailing why they think the gene or pathway might work. We now have a small library of possible genetic modifications, optimized for Neurospora Crassa.</LI><LI>Research into possible promoters and terminators that have been used in N. crassa, and how we might set about getting a sample of N. crassa to work with.</LI><H4>Week 2  May 1-7</H4><UL><LI>Deliberations on how to achieve the project vision. In collaboration with Dr. Dave Stewart and Dr. Frank Nargang, we decide that it may be possible to increase fatty acid content by introducing a new thiosterase into the N. crassa genome, allowing more fatty acid to be cleaved off the Fatty Acid Synthase complex (FAS) and accumulate in membranes and incorporated into fat. Also, knocking out a Fatty-acyl:CoA Synthetase may help shunt fatty acids away from being turned into acetyl-CoA (energy) and into storage.</LI><LI>We have many possible choices for thioesterases and fatty-acyl:CoA synthetases. After conferring with Dr. Dave Stewart, and researching the Neurospora Crassa genome available at the Broad Institute website, we decide that FadD1 is the Fatty-acyl:CoA synthetase we will try to knock out.</LI><LI>Possible choices for thioesterases include one from E. coli, one from a plant species (Umbellularia Californica), and another one from N. crassa itself. Each are specific for different fatty-acid chain lengths. We decide that the thioesterase from E. coli, which cleaves primarily C16 products off of the Fatty-acid:FAS complex is the safest bet, because it has the least chance of being accidentally recombined into N. crassa's genome at an undesired locus.</LI><LI>Decide that knocking out FadD1 and inserting our thioesterase in its place is the easiest way to achieve both genetic modifications at once. Our construct will take the form of:</LI><UL><LI>5' FadD-Untranslated Region (5'-UTR), thioesterase, Hygromycin B resistance, and 3' FadD-Untranslated Region.</LI></UL><LI>Decide to use Team Alberta's previous year's project, GENOMIKON, to assemble our genetic constructs. For more information, see 2010.igem.org/Team:Alberta</LI></UL><H4>Week 3  May 8-14</H4><UL><LI>Decided to synthesize the N. crassa-optimized E. coli thioesterase gene using IDT's custom gene synthesis service. First, further codon optimization has to take place, as the %GC content of our codon-optimized thioesterase is too high. </LI><LI>Decided to use the Actin promoter, native to N. crassa's genome, as the promoter for our thioesterase gene. Because actin is such an essential protein in eukaryotic cells, we believe this will yield very high expression levels of our synthesized gene. </LI><LI>Decided to use the trpC terminator, from Aspergillus nidulans, as the terminator for our thioesterase gene. This is based on previous papers that have used the trpC terminator and obtained successful results.</LI><LI>We decide to synthesize the promoter, thioesterase gene, and terminator all at once in one construct. This allows us to maintain a high degree of control over the genetic sequence, without the need to sequence parts obtained from other labs to ensure their accuracy.</LI></UL>Week 4 May 15-21<UL><LI>Second version of our codon-optimized construct. Used DNA 2.0's 'Gene Designer 2.0' to optimize the secondary structure of the thioesterase mRNA, and it came up with the exact same sequence as our first purely codon-optimized form. %GC content is still too high, and large repeated sequences make it difficult and costly to synthesize</LI><LI>Third version of our codon-optimized construct. Went through the sequence for the thioesterase and changed the codons for Alanine, Proline, Arginine, and Glycine to their second-most used codon. This lowers the %GC content to ~59%, and gets rid of a large number of the repeats. However, there are still internal BsaI cut sites, which are incompatible with the Genomikon assembly method.</LI><LI>Fourth version of our codon-optimized construct. Modified the codons of 2 amino acids to get rid of BsaI recognition sequences. Changed Gly-76 and Gly-101 from GGT back to GGC. Upon consultation with IDT, we receive the go-ahead that this gene can be made. The current iteration will cost less, and be made faster.</LI></UL><H4>Week 5 May 22-28</H4><UL><LI>May 25: Finally ordered our promoter, thioesterase, and terminator synthetic gene construct. It will take approximately one month to make.</LI><LI>Ordered all the primers we'll need to assemble our genetic construct. Ordered pairs of primers for the 5' and 3'-UTRs, Hygromycin B, and for our synthesized gene from IDT.</LI><LI>Went to Dr. Frank Nargang's laboratory and received instruction from Jeremy Wideman on how to manipulate and transform N. crassa. </LI></UL><H4>Week 6 May 29-June 4</H4><UL><LI>Received cosmid pMOcosX G6 C9 from Jeremy Wideman of Dr. Frank Nargang's lab. This cosmid contains the genomic sequence for FadD1 in N. crassa. It was received plated on an amp plate.</LI><LI>Picked 4 colonies from the plate, grew overnight cultures in amp, and ran minipreps of each. The concentrations of each miniprep look good. </LI><LI>Received samples of plasmids pCSN44, which carries the Hygromycin resistance gene, pBARK, which carries BASTA resistance, and pBA, which carries Bleomycin resistance. All samples were graciously donated by Dr. Frank Nargang.</LI></UL><H4>Week 7 June 5-11</H4><UL><LI>All primers arrived on June 7, 2011. They were all diluted in sterile TE Buffer (10 mM EDTA, 1 mM Tris, pH 8.0) to a concentration of 100uM. This was further diluted 1:10 in sterile TE to 10uM. All primers used for PCR are at a stock concentration of 10uM.</LI><LI>Used cosmid miniprep #2 to attempt to PCR out both the 5' FadD-UTR and 3' FadD-UTR regions. Diluted 1:100 in TE to a final concentration of ~3 ng/uL. Set the PCR machine to run a suitable program for our fragments (both ~800 bp), and ran a typical PCR reaction. Ran the results on a 1% agarose gel and saw successful PCR products, free of contamination.</LI><LI>PCR-purified the PCR products according to the 'QIAQEN Bench Protocol: QIAquick PCR Purification' protocol using a vacuum manifold. The final concentrations of each PCR tube look good.</LI><LI>Research into other possible promoter candidates:
            </LI><LI>Bli-3/Bli-4/Bli-7: All are blue-light-sensitive promoters that respond greatly to exposure to blue light. Many of these promoters are involved in either conidiation or carotenoid production in N. crassa. Having these promoters would be useful because it allows us to control gene expression through external stimuli</LI><LI>cmt: Copper-inducible metallotionein gene promoter. Because this promoter is inducible by copper, it would allow us to regulate gene expression through copper as an external stimuli. Could be an effective way to ensure genetically modified N. crassa cannot survive in the wild.</LI></UL><LI>Research into other possible terminator candidates:
            </LI><LI>Am: Terminator for the amination deficient gene. Has been used before and has optimized vectors already.</LI><H4>Week 8 June 12-18</H4><UL><LI>Procured samples of solid and liquid pulp mill sludge from Milo Mihajlovich, of Incremental Forest Technologies, Ltd. Gave the samples to Growth team to run growth experiments on. Mr. Mihajlovich received the samples graciously from Peace River Pulp Division, run by DMI (Daishowa-Marubeni International Ltd.).</LI><LI>Transformed DH5-alpha E. coli with pCSN44, to make strains that would carry the hygromycin-resistance gene, HygBr. After plating on ampicillin-inoculated plates, no colonies grew overnight. Stored plate in fridge.</LI><LI>Transformed DH5-alpha E. coli with pBARK, to make strains that would carry the BASTA-resistance gene. After plating on ampicillin-inoculated plates, ~3000 colonies grew overnight. Stored plate in fridge.</LI></UL><H4>Week 9 June 19-25</H4><UL><LI>Transformed pCSN44 into E. coli, again. Performed exact same procedure as in Week 8. Only 3 colonies grew on overnight amp plates. The negative water control plate has one colony, indicating that maybe our plasmid isn't quite correct.</LI><LI>Grew three 5mL overnight tubes. Only one of the inoculated overnights grew after 16hr. Restreaked the overnight onto another amp-plate to ensure that the plasmid has actually been transformed, and inoculated 3 more overnight tubes.</LI><LI>Miniprepped the overnights, and got good concentrations. Tried PCRing out HygBr out of the resulting plasmid using a program fit for parts 2200 bp long. Results were negative; no PCR products appeared on the gel. In fact, no DNA appeared on the gel. Tried PCR purifying the results anyway, and came up with no DNA.</LI><LI>Digested the pCSN44 miniprep with BamHI, to check if pCSN44 is correct. No DNA showed up on the resulting gel, indicating that our pCSN44 miniprep is either incorrect or something in our procedure is very wrong.</LI></UL><H4>Week 10 June 26-July 2</H4><UL><LI>Tried a PCR of HygBr using direct pCSN44 (the pure plasmid Jeremy gave us, and not the miniprep) as the template. Double checked that our cells were competent, and double-checked our primers.</LI><LI>Ran a gel of the PCR products: no DNA. Something is wrong with our pCSN44.</LI><LI>Tried digesting the pure pCSN44 sample (not the miniprep) with BamHI. After running a gel of the pure pCSN44 along with a digested sample, we saw very faint bands at ~10000 bp in the non-digested sample, and at ~4500 bp in the digested sample. This does not correspond with the actual size of pCSN44 (5.3 kbp). We should have seen 3 fragments.</LI><LI>Tried digesting pure pCSN44 (not miniprep) again, and this time loaded 3x as much DNA as before onto the gel. We see clear bands that correspond with the previous gel; something is very wrong.</LI><LI>Went to Dr. Frank Nargang's lab, and went through a transformation under the supervision of Jeremy Wideman. Used a new sample of pCSN44 taken out of -20 degrees celsius storage. Plated 3 amp plates and left to grow overnight at 37 degrees celsius.</LI><LI>Too many colonies to count grew on all 3 plates. Restreaked two of the plates on to more amp plates to make sure we get a clean sample. Inoculated overnights, and miniprepped them the next day. The concentrations of each miniprep vary considerably from tube to tube.</LI></UL><H4>Week 10 July 3-July 9</H4><UL><LI>The synthetic gene we ordered from IDT has arrived. Resuspended the gene in 20 uL of sterile TE buffer, and stored in the freezer.</LI><LI>Ran a digestion of miniprep #3 from the most previous pCSN44 transformation attempt. Also ran a PCR of HygBr off of miniprep #3.</LI><LI>The gel shows the same as before. The undigested pCSN44 has a band at ~10000 bp, digested pCSN44 has a band at ~4500 bp, and the PCR products have no DNA in each lane.</LI><LI>Tried PCR purifying the PCR products anyway, and got reasonable concentrations. However, after running on a gel, we see no DNA again.</LI><LI>Transformed our new synthetic gene (thioesterase with promoter and terminator in pUCminusMCS) into E. coli, and plated onto amp plates. Colonies grew. Made overnights. Minipreps of the overnights all yielded good concentrations.</LI><LI>Tried a BamHI digestion of the engineered thioesterase, and ran the products on a 1% agarose gel per usual procedure. Gel shows that the undigested pUCminusMCS runs correctly, but that the digested plasmid runs at &gt;5000 bp. This is not correct. After double-checking the sequence file, we realized that at one of the BsaI recognition sites there is Dcm methylation, which is interfering with the restriction enzyme activity.</LI></UL><H4>Week 11 July 10-16</H4><UL><LI>Generously received 1 uL of ApaLI from Dr. Joel Weiner's laboratory. Will use ApaLI to check the accuracy of our synthesized thioesterase in pUCminusMCS, since ApaLI has no methylation sensitivity. After digesting miniprep #3, and running on a gel, we see that the gene is in fact correct. We get 3 bands, as we expected.</LI><LI>Because pCSN44 isn't working, we get a sample of pCSN43 from Dr. Frank Nargang. pCSN43 is identical to pCSN44, except the orientation of HygBr in relation to the plasmid backbone is reversed. </LI><LI>Tried a PCR of HygBr using direct pCSN43 (not miniprepped) as the template and the primers designed to work only with pCSN44. This PCR should not work, as the primers do not anneal correctly to this sequence. However, we are still going to try on the off chance it does. </LI><LI>After running a gel of the PCR products, we see a strong band at 2200 bp and a fainter band at 1000 bp. This is very surprising, as this PCR should in no way have worked. Yet there is correct product on the gel. Perhaps pCSN43 and pCSN44 were mislabelled?</LI><LI>Ran a PCR touchdown procedure, varying the concentrations of magnesium ion in each PCR tube. Set the PCR machine to anneal at slightly lower temperature each cycle, and varied the Mg++ concentration from 1-10 mM. Ran 4 tubes using pCSN44 as a template and 4 using pCSN43 as a template. The gel of the products showed very little conclusive results. It appears as if neither template produced the correct product; however, both have contamination at ~1000 bp. There are two strong possibilities as to why none of our PCRs have worked:
            </LI><LI>The primers are binding non-specifically to sequences in both the pCSN43 and pCSN44 plasmid backbones, or;</LI><LI>The plasmids themselves are incorrect</LI></UL><LI>Tried digesting the pCSN43 PCR product that produced the correct 2200 bp band with PstI. When run on a gel, we see the correct fragment sizes, indicating that it is indeed HygBr. However, we still see contamination at 1000 bp. </LI><H4>Week 12 July 17-23</H4><UL><LI>Tried another PCR of HygBr out of pCSN43, using the same primers and conditions as the last time when we got product at 2200 bp. After PCR purifying the products, we get reasonable concentrations of DNA (~40 ng/uL). </LI><LI>Ran one of the cleaned up PCR products on a gel for gel extraction. Loaded ~800 ng into one lane of the gel, and ran per usual. Used a transilluminator to visualize the 2200 bp piece of DNA, and cut it out of the gel. After performing a normal gel extraction procedure from QIAGEN, we measure the concentration to be 0 ng/uL.</LI><LI>Ran a PCR of HygBr using the gel extraction (0 ng/uL) as a template. Although there appears to be no DNA in the extraction sample, if there is any at all PCR may still be able to amplify it up. Ran the PCR products on a gel, and saw no DNA at all. </LI><LI>Tried another PCR of HygBR out of pure pCSN43, this time using a separate program with longer extension times and a lower annealing temperature than before. After running the PCR products on a gel, we see only contamination at 1000 bp and no band at 2200 bp. </LI><LI>Tried yet another PCR of HygBr out of pure pCSN43, this time using another program with less melting time per cycle than our other programs. Again, we see only contamination at 1000 bp on the gel.</LI></UL><H4>Week 13 July 24-30</H4><UL><LI>Decided to hold off on PCRing out HygBr for a little bit, to think of new solutions. We have ordered a new sample of pCSN44 from the Fungal Genetic Stock Centre (FGSC)</LI><LI>For mass amplification of parts, we have designated a PCR program 'BIG'. This program carries an extension time of 2 min and 30 sec, meaning it can amplify any of our parts. It also has a lower annealing temp than our other programs, to ensure all sets of primers will anneal correctly.</LI><LI>Tested PCR of our synthetic gene construct (thioesterase) using BIG. Used the normal Mg++ concentration of 2 mM. Ran the PCR products on a gel, and saw an inconclusive band at ~1700 bp. Not sure what it means.</LI><LI>Our pCSN44 has arrived from the FGSC. Diluted to appropriate concentration, and tried a normal PCR of HygBr using it as a template. The gel of the PCR products shows clean bands at 2200 bp. It is now clear that whatever pCSN44 templates we were using before were incorrect or faulty in some way, and that FGSC has provided us with the correct pCSN44.</LI><LI>Tried running a massive PCR to amplify up all of our parts at once. Used all primer+template combinations that have been working so far, and ran BIG. The gel of the PCR products shows that PCR of the 5' FadD-UTR, 3' FadD-UTR, and HygBr all worked very well. The PCR of the synthetic thioesterase, however, did not yield very much product. There is also contamination in the lower bp range for our thioesterase. Need to make the primers more specific.</LI><LI>Ran a gradient Mg++ concentration PCR touchdown on our synthetic thioesterase to see what Mg++ concentration yielded specific, correct PCR products. After running a gel, it appears as if 0.5mM Mg++ yields pretty clean products at the correct 1700 bp band.</LI></UL><H4>Week 14 July 31-August 6</H4><UL><LI>Research into other possible thioesterases, since ours is having trouble. Decide on several candidates. One candidate is a thioesterase from an elm tree, which preferentially cleaves C8/C10 fatty acids off of FAS. This is god because it allows the diesel to be more liquid at lower temperatures.</LI><LI>Most promising is a thioesterase from Umbellularia Californica, the Californian bay tree. Has been shown in studies to preferentially cleave C12 fatty acids off the FAS, to the point where C16 concentrations are significantly lower than normal. (Mol. Breeding, 12:71-81, 2003). This severely reduced C16 concentration would mean less inhibition of Fatty acid synthesis by the Fatty Acid:FAS complex, and allow for even greater FA yields. Also, C12 is more fluid at lower temperatures, and so would be more ideal as a diesel fuel.</LI><LI>Codon-optimized the U. californica thioesterase, made sure to remove all BsaI cut sites, and ordered from IDT in a pIDTsmart w/ amp plasmid. Synthesized the gene with both the signal and mature peptides, and designed primers such that we can fit the new thioesterase into existing infrastructure, if need be.</LI><LI>FIrst attempts at assembly of our genetic construct, as well as our control construct. Put all digested, cleaned up parts into a ligation mixture and left overnight. Ran the products of the ligation on a gel, and saw a large smear. No evidence that ligation occurred correctly.</LI><LI>Tried a PCR of the entire construct, using Taq. Gel had only smears, and no conclusive results. Ran a PCR touchdown using gradient Mg++ concentrations of the entire construct. No conclusive results.</LI><LI>Tried to ethanol precipitate the results of genetic assembly ligation. No DNA was present at the end of the procedure.</LI></UL><H4>Week 15 August 7-13</H4><UL><LI>Tried genetic construct assembly two more times. Gels of both attempts showed no product at 5.5 kbp, and only contamination at 2000 bp.</LI><LI>Designed 22 more primers from IDT. These primers will be used to extensively characterize our parts into parts plasmids. Also designed the necessary 5'-phosphorylated oligonucleotides to adapt our parts into GENOMIKON base plasmid v.2. Ordered the primers and oligos next week.</LI></UL><H4>Week 16 August 14-20</H4><UL><LI>Logistics of RFC, part submission, and work on WIki design.</LI><LI>Cataloguing of all parts in any freezer boxes, and development of parts assembly method. Research on BioBrick systems.</LI><LI>Helped out Growth and Esterification teams.</LI></UL><H4>Week 17 August 21-27</H4><UL><LI> All ordered primers and oligos from IDT arrive. All are diluted to appropriate concentrations, and stored in the freezer.</LI><LI>Make all oligo âadaptersâ for base plasmid v.2. by mixing 45 uL of each 100uM oligo, 10uL of 10X React2 Buffer (from Invitrogen), and annealing by heating all pairs up to 95 degrees celsius and slowly cooling (over 2.5 hrs) to room temperature.</LI><LI>Ran 2 massive PCRâs to check all primers. Run PCRs of 5â FadD-UTR, HygBr, synthetic thioesterase gene, 3â FadD-UTR, actin promoter, trpC terminator, thioesterase CDS, and the U. californica thioesterase; one with the signal + mature peptide, and one with just the mature peptide.</LI><LI>Checked all parts on a large 1% gel. All parts look like they are PCRing up very clean and correctly at the proper length EXCEPT
        </LI><LI>Synthetic thioesterase gene: the gel here is smeary, and we donât see clearly 	defined product.</LI><LI>Actin promoter: there is very little PCR product on the gel. There are some faint, smeary bands at extremely low bp range (~50 bp).</LI></UL><LI>PCRâed up all parts required for multiple assemblies. PCRed at least 9 100uL tubes of each part, and PCR purified them in groups of three (i.e. put 3 PCR tubes through the same column). Digested one tube of each part with BsaI-HF, and ran on a gel. The gel shows that all parts worked EXCEPT the synthetic thioesterase gene and the actin promoter. These lanes have no DNA at the correct bp range, and are smeary. This is unfortunate, because those are two pieces we NEED to complete this assembly.</LI><LI>Ran a PCR touchdown using gradient Mg++ concentrations on the Actin promoter. Gel shows that the Actin promoter PCRâed out correctly at both 0.5mM and 1mM Mg++. </LI><LI>Tested the digested thioeserase gene construct that gave smeary bands on the gel by trying a test ligation of the thioesterase with 5â FadD-UTR. The test of the ligation is negative, and we see no bands for the thioesterase or the ligated product.</LI><H4>Week 18 August 28-September 3</H4><UL><LI>Brainstormed strategies to try an get our genetic construct assembly to work. Decided we may not be able to approach it directly. </LI><LI>PCRâed and digested more synthetic thioesterase gene. Gel shows product at right length, but there is still extensive smearing in the gel lane. Not sure as to why.</LI><LI>Tried PCRing up the 5â and 3â FadD-UTR regions off of a sample of N. crassa genomic DNA, harvested via a yeast DNA extraction protocol. Previously, we have been using a cosmid for PCR purposes; however, we want to see if its possible to PCR off of genomic DNA. The results are overwhelmingly positive: we get clear, bright bands for every PCR we try.</LI><LI>Tried PCRing up 5â and 3â FadD-UTR regions off of dry conidia dusted into the 	PCR tube. One of the PCRs works perfectly, with no contamination. The other 2 attempts do not work. This is promising, although we do not have enough time to investigate this further.</LI><LI>Tried a variety of PCR programs on the Actin promoter. None work.</LI><LI>Received a generous donation of Phusion polymerase from NEB. Tried to PCR up the Actin promoter using several different conditions and Phusion polymerase. None of the PCRâs work. There is most likely something wrong with our primers or template.</LI><LI>Made chlor plates for amplification of our base plasmids for parts assembly. These plasmids were designed last year to fit within the existing BioBrick framework, but be useful for BioBytes 2.0 (and by extension this yearâs Biobytes 3.0) as well. Pick 4 red colonies, grow overnights for both the pAB and pBA plasmids, and miniprep the results. Concentrations look good.</LI><CENTER>
		The Genomikon base plasmid v.2. has the prefixes and suffixes of the BioBrick format, but is designed to work with BioBytes. An RFP bounded by BsaI-HF cut sites allows easy visual selection of proper E. coli transformants.
    </CENTER><LI>Ran multiple ligation tests involving some of our already-cut parts. All work except for the ligations involving 3â Fad-UTR. Re-PCR and digest more 3â FadD-UTR.</LI></UL><H4>Week 19 September 4-10</H4><UL><LI>Multiple PCRs, digestions, and test ligations of each of our parts required for the synthetic thioesterase gene construct. Checked each of them on gels to ensure they are correct. Most worked, but some still arenât. The synthetic thioesterase gene is causing problems.</LI><LI>Attempt PCR of all possible ways to PCR and link the Actin promoter, the thioesterase CDS, and the trpC terminator together. All possible primer combinations and Mg++ concentrations are considered. From this, we find several candidates. Troubleshooting PCR reactions, and further tests.</LI><LI>Digested all parts plasmid minipreps with BsaI-HF, and some with PstI or EcoRI as well. After cleaning up, none of the samples have any DNA left. At some point in the proceedings, all DNA was lost.</LI><LI>Grew more base-plasmid transformed E. coli, and miniprepped. Digested these with BsaI-HF, and digested some with EcoRI or PstI as well. Digested some with all 3 enzymes. Concentrations of these digested, purified plasmids look good, and the gel shows correct bands at ~2000 bp (plasmid backbone) and at ~1000 bp (RFP). The reason for the multiple digestions is that our oligo âadaptorsâ will connect at those sites, and allow any plasmid to be adapted to our partâs need. </LI></UL><H4>Week 20 September 11-17</H4><UL><LI>Further ligation tests and part characterization. Getting parts ready for assembly, and trying many different kinds of assembly (normal bead assembly, one-pot ligations, etc.). Trying many types of PCR (both Taq and Phusion) on assembly products.</LI></UL><H4>Week 21 September 18-24</H4><UL><LI>More ligation tests, and PCRs. Prep for wiki freeze. Parts plasmids to be submitted to the registry are made during this week, and are checked on a gel for correctness. Decide on parts to submit to the registry before Indianapolis. </LI><LI>Work on presentation for aGEM, and work on the Wiki freeze. Ongoing work from here on.</LI></UL></DIV></DIV><DIV id="content-footer-box"><DIV class="footer-left-half"><DIV class="footer-left-col"><P><B>About our Wiki</B></P><P>Seven students from three departments across the University of Alberta campus aspired to complete an ambitious project over the course of the summer. This wiki is all about our successes and our many trials and tribulations.</P></DIV><DIV class="footer-right-col"><P><B>Search</B></P><P><B>Contact Us</B></P><P>University of Alberta</P><P>4-055 Katz Group Research Centre</P><P>University of Alberta</P><P>Edmonton, Alberta T6G 2E1</P><P>teamalbertaigem@gmail.com</P></DIV></DIV><DIV class="footer-right-half"><DIV class="footer-right-left-col"><P><B>Navigate Our Wiki</B></P><UL><LI><A href="https://2011.igem.org/Team:Alberta">Home</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Neurospora">Neurospora crassa</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/project">Project</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Achievements/Overview">Results &amp; Achievements</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Methodology/Overview">Methodology</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/HumanPractices/Overview">Human Practices</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/team/Students">Team</A></LI><LI><A href="https://2011.igem.org/Team:Alberta/Press">Interactive</A></LI></UL></DIV><DIV class="footer-right-right-col"><A style="color:#808080;" href="http://twitter.com/#!/Team_Alberta" class="twitter-follow-button">Follow @Team_Alberta</A></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Alberta/Methodology/Notebook/Genetics">http://2011.igem.org/Team:Alberta/Methodology/Notebook/Genetics</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Alberta/Methodology/Notebook/Genetics" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Alberta/Methodology/Notebook/Genetics" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Alberta/Methodology/Notebook/Genetics&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Alberta/Methodology/Notebook/Genetics&amp;oldid=195982" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>