<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kyoto_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kyoto/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header"><HEADER><UL class="nv"><LI><A href="https://2018.igem.org/team:Kyoto/Project"><FONT face="Segoe UI">Project</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Project"><FONT face="Segoe UI" font="" size="2px"><B>Description</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Design"><FONT face="Segoe UI" font="" size="2px"><B>
Design</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Result"><FONT face="Segoe UI" font="" size="2px"><B>
Results</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Demonstrate"><FONT face="Segoe UI" font="" size="2px"><B>
Demonstration</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Discussion"><FONT face="Segoe UI" font="" size="2px"><B>
Discussion</B></FONT></A></LI></UL><LI><A href="https://2018.igem.org/Team:Kyoto/Model"><FONT face="Segoe UI">
Modeling</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Notebook"><FONT face="Segoe UI">
Lab Work</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Notebook"><FONT face="Segoe UI" font="" size="2px"><B>
Notebook</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/MaterialsMethods"><FONT face="Segoe UI" font="" size="2px"><B>
Materials&amp;Methods</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/SpecialMethods"><FONT face="Segoe UI" font="" size="2px"><B>
Special Methods</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Safety"><FONT face="Segoe UI" font="" size="2px"><B>
Safety</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Basic_Parts"><FONT face="Segoe UI">
Parts</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Basic_Parts"><FONT face="Segoe UI" font="" size="2px"><B>
Basic Parts</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Composite_Parts"><FONT face="Segoe UI" font="" size="2px"><B>
Composite Parts</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Improve"><FONT face="Segoe UI" font="" size="2px"><B>
Improving parts</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Parts_Collection"><FONT face="Segoe UI" font="" size="2px"><B>
Parts Collection</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Human_Practices"><FONT face="Segoe UI" font="" size="2%">
Human Practices</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Human_Practices"><FONT face="Segoe UI" font="" size="2px"><B>
                                                                        Human Practice/Silver</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Integrated_Human_Practices"><FONT face="Segoe UI" font="" size="2px"><B>
Integrated Human Practice/Gold</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Collaborations"><FONT face="Segoe UI" font="" size="2px"><B>
Collaborations</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Team"><FONT face="Segoe UI">
Team</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Team"><FONT face="Segoe UI" font="" size="2px"><B>
Members</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Attributions"><FONT face="Segoe UI" font="" size="2px"><B>
Attributions</B></FONT></A></LI></HEADER></DIV><TITLE>Team:Kyoto/Design - 2018.igem.org</TITLE><DIV id="BACKGROUND"><DIV style="padding-top: 100px;"><DIV class="box27"><SPAN class="box-title"><FONT face="Segoe UI">Table of contents</FONT></SPAN><UL class="index1"><LI><A href="#Our Design"> 1) Our Design</A></LI><LI><A href="#Preparation of salt resistance enhancing plasmid in budding yeast"> 2) Preparation of salt resistance enhancing plasmid in budding yeast</A></LI><LI><A href="#Preparation of yeast to incorporate Na+"> 3) Preparation of yeast to incorporate Na+</A></LI><LI><A href="#Reduce the concentration of NaCl in the medium"> 5) Reduce the concentration of NaCl in the medium</A></LI><LI><A href="#Development of aggregation system"> 5) Development of aggregation system</A></LI></UL></DIV><H5 id="Our Design">1) Our Design</H5><P> Our device, Swallowmyces cerevisiae, bravely dives into dangerous saltwater and swallow Na+, reducing salt concentration of the water. In order to achieve this, our device i.e. microorganisms should fulfill the four criteria shown below. 

1. Survive in high salt water. 
2. Uptake Na+ into their cytoplasm or vacuoles. 
3. Reduce salt concentration of the water by absorbing Na+.
4. Change to a form which is easy to be collected quickly and trash easily after use.
 We'll introduce our project design regarding these criteria.
</P><H5 id="Preparation of salt resistance enhancing plasmid in budding yeast
">2) Preparation of salt resistance enhancing plasmid in budding yeast
</H5><P> When Na+ ions are collected in S. cerevisiae cells, a high concentration of Na+ might damage the cells. We have to develop tools which protect S. cerevisiae cells from salt damage. We used two devices here, “compatible solute synthesis” and “chaperon-like protein”.

Did you know that a certain group of yeast is working hard in high salt condition, to provide a great contribution to tables in the world? They are, <I>Zygosaccharomyces rouxii</I>, soy sauce-brewing yeast. They produce glycerol as a compatible solute to counteract osmotic stress. We cloned ZrGPD1 (glycerol-3-phosphate dehydrogenase) and ZrFPS1 (glycerol transporter)from <I>Z. rouxii</I>, both of which are important for the glycerol metabolism in <I>Z. rouxii</I>. We tried to increase salt tolerance of our Swallowmyces cerevisiae by these two genes.　Another gene we focused on is mangrin, a small peptide which derived from mangrove. It is a chaperone-like protein and believed to repair salt damaged proteins. We decided to test this gene to increase yeast salt tolerance.

</P><P><CENTER><FONT face="Segoe UI" font="" size="2px" font="" color="#000000">Figure1. ZrGPD1/ZrFPS1/mangrin in Yeast</FONT></CENTER></P><H5 id="Preparation of yeast to incorporate Na+"> 3) Preparation of yeast to incorporate Na+
</H5><P><I>S. cerevisiae</I> has Na+ transporter to remove Na+ from their cytoplasm. The main transporters include NHA1, ENA1, ENA2, ENA4. When all of these genes were knocked out, the deletion strain shows high sensitivity against NaCl. To produce yeast strain that uptake even more Na+, we knocked out all of the above genes by using a homologous recombination system,　Furthermore, we found a protein called AtHKT1 which involves in an influx of Na+ in plants.  By overexpressing AtHKT1, we expected that more Na+ will be collected by the yeast. Another candidate gene we found is McHKT2. McHKT2 is a Na+ transporter of <I>Mesembryanthemum crystallinum</I>in other words, “ice-plant”, a salt tolerance plant. It is reported that McHKT2 is involved the salt compartmentalization in this plant.　High concentration Na+ in cytoplasm might damage the cells. To overcome this problem, we will use the salt plants’ salt tolerance system, where Na+ in the cytoplasm is sequestered into vacuole by Na+/H+ exchanger. These factors include Na+/H+ antiporter AtNHXS1 from A. thaliana, SseNHX1, a paralog of AtNHXS1 from the salt plant, and a vacuolar protein AVP1 which increase H+ concentration in a vacuole.　By enhancing Na+ influx and preventing Na+ efflux at the same time. Finally, by redirecting cytoplasmic Na+ into vacuoles, we aimed to create yeast strain which accumulates more Na+. 
</P><P><CENTER><FONT face="Segoe UI" font="" size="2px" font="" color="#000000">Figure2.Transporters related to Na+ in Yeast</FONT></CENTER></P><H5 id="Reduce the concentration of NaCl in the medium
">4) Reduce the concentration of NaCl in the medium
</H5><P> The goal of our Swallowmyces cerevisiae is not the uptake of Na+. We aim to reduce the salt concentration of the water by this device. By combining experimental data and mathematical modeling, we try to optimize our system, to achieve maximum desalination. What will happen when we put our best strain into high salt-containing media?  
</P><P><CENTER><FONT face="Segoe UI" font="" size="2px" font="" color="#000000">Figure3. transporters and peptide in this project</FONT></CENTER></P><H5 id="Development of aggregation system">4) Development of aggregation system
</H5><P> Even if our device efficiently reduce NaCl concentration in the media, we will never stop our research and development. We try to construct a robust safety system for the biocontainment of our genetically modified yeast cells. For this purpose, we selected two genes, SdrG and FgBeta.
</P><P> SdrG is a surface component of Staphylococcus epidermidis. It is known that SdrG tightly binds to a small N-terminus domain of human fibrinogen beta. Reportedly, the binding between SdrG and fibrinogen beta is as strong as a covalent bond. If we express these proteins on the surface of yeast separately, we might see strong coupling of two yeast cells mediated by this interaction. In such a case, as one cell will display multiple handles, we might see a big ball of conjugated yeast cells, when we mix the two “handle-displaying” strains.</P><P> SdrG is a surface component of Staphylococcus epidermidis. It is indicated that this protein binds extremely strongly to a short sequence of 25 amino acids derived from human fibrinogen beta and this is involved in the sticking of biofilm to the body. If these proteins are expressed by a surface display in separate yeasts, can we realize the strong binding of these cells by the strong binding force? If one of the yeast expresses a plurality of handle, it makes us realize large cell mass one across just by blending the two types of yeast……??

</P><DIV class="reference"><H6>Reference</H6><UL class=""><LI>[1] R. Haro, B. Garciadeblas, A. Rodriguez-Navarro (1991) A novel P-type ATPase from yeast involved in sodium transport, <I>FEBS Letters</I> Vol.291 Issue2 189-191</LI><LI>[2] Jos6 A. Miirquez, Ramdn Serrano (1996) Multiple transduction pathways regulate the sodium-extrusion gene PMR2/ENA1 during salt stress in yeast, <I>FEBS Letters</I> Vol.382 Issue1-2 89-92</LI><LI>[3] A. Yamada, T. Saitoh, T. Mimura et al. (2002) Expression of Mangrove Allene Oxide Cyclase Enhances Salt Tolerance in <I>Escherichia coli</I>, Yeast, and Tobacco Cells, <I>Plant and cell physiology</I> 903-910
</LI><LI>[4] Hou,Lihua Wang,Meng Wang,Cong Wang,Chunling Wang,Haiyong (2013) Analysis of salt-tolerance genes in zygosaccharomyces rouxii, <I>Applied Biochemistry and Biotechnoloogy</I> 1417-1425 </LI><LI>[5] L. Milles, K. Schulten, H. Gaub et al. (2018) Molecular mechanism of extreme mechanostability in a pathogen adhesin, <I>Science</I> Vol.359 Issue6383 1527-1533 </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="pop_why_box"><DIV class="pop_close">× </DIV><DIV class="pop_why_content"><H3> Loading ... </H3></DIV></DIV></DIV></BODY></HTML>