<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Kyoto_Result skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Kyoto/Result</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header"><HEADER><UL class="nv"><LI><A href="https://2018.igem.org/team:Kyoto/Project"><FONT face="Segoe UI">Project</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Project"><FONT face="Segoe UI" font="" size="2px"><B>Description</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Design"><FONT face="Segoe UI" font="" size="2px"><B>
Design</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Result"><FONT face="Segoe UI" font="" size="2px"><B>
Results</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Demonstrate"><FONT face="Segoe UI" font="" size="2px"><B>
Demonstration</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Discussion"><FONT face="Segoe UI" font="" size="2px"><B>
Discussion</B></FONT></A></LI></UL><LI><A href="https://2018.igem.org/Team:Kyoto/Model"><FONT face="Segoe UI">
Modeling</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Notebook"><FONT face="Segoe UI">
Lab Work</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Notebook"><FONT face="Segoe UI" font="" size="2px"><B>
Notebook</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/MaterialsMethods"><FONT face="Segoe UI" font="" size="2px"><B>
Materials&amp;Methods</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/SpecialMethods"><FONT face="Segoe UI" font="" size="2px"><B>
Special Methods</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Safety"><FONT face="Segoe UI" font="" size="2px"><B>
Safety</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Basic_Parts"><FONT face="Segoe UI">
Parts</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Basic_Parts"><FONT face="Segoe UI" font="" size="2px"><B>
Basic Parts</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Composite_Parts"><FONT face="Segoe UI" font="" size="2px"><B>
Composite Parts</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Improve"><FONT face="Segoe UI" font="" size="2px"><B>
Improving parts</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Parts_Collection"><FONT face="Segoe UI" font="" size="2px"><B>
Parts Collection</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Human_Practices"><FONT face="Segoe UI" font="" size="2%">
Human Practices</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Human_Practices"><FONT face="Segoe UI" font="" size="2px"><B>
                                                                        Human Practice/Silver</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Integrated_Human_Practices"><FONT face="Segoe UI" font="" size="2px"><B>
Integrated Human Practice/Gold</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Collaborations"><FONT face="Segoe UI" font="" size="2px"><B>
Collaborations</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Team"><FONT face="Segoe UI">
Team</FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Team"><FONT face="Segoe UI" font="" size="2px"><B>
Members</B></FONT></A></LI><LI><A href="https://2018.igem.org/Team:Kyoto/Attributions"><FONT face="Segoe UI" font="" size="2px"><B>
Attributions</B></FONT></A></LI></HEADER></DIV><P><TITLE>Team:Kyoto/Project - 2018.igem.org</TITLE></P><DIV id="BACKGROUND"><DIV style="padding-top: 100px;"><DIV class="accbox"><LABEL for="label1">1) Salt concentration has various effects on protein-protein interaction</LABEL><DIV class="accshow"><P> To start, we conducted an experiment demonstrating how the salt concentration of the solution actually affects protein-protein interaction. As the model protein, we chose GFP(<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0040</A>), which is commonly used in many experiments as a model protein. TDH3 promoter and CYC1 terminator were added to either end of ORF and cloned into pRS316, a shuttle vector from <I>E. coli</I> to <I>S. cerevisiae</I>. The resulting plasmid was transformed into wild-type yeast strain BY4741 to overexpress GFP. In a comparative experiment, yeast expressing RFP(<A href="http://parts.igem.org/Part:BBa_E0040">BBa_E0010</A>) with the same set of promoter and terminator was used.

In Figure 1 we show cell pellets recovered from the yeast culture used in this experiment.</P><P class="fig">Figure1. A picture of yeast expresses RFP or GFP</P><P> As can be easily seen, the yeast pellet overexpressing GFP was a pale yellow color while the yeast overexpressing RFP was a thin red color. From this, it was confirmed that both GFP of BBa_E0040 and RFP of BBa_E0010 can be highly expressed in yeast cells even without codon optimization, and that the expression level is high enough to be observed under visible light without lysing the yeast or protein purification.

 In order to investigate yeast proteins interacting with GFP, an experiment was conducted to purify expressed GFP protein from yeast lysate using immunoprecipitation. The yeast was placed in a mortar with liquid nitrogen and crushed, and a lysate was prepared with a buffer having a salt concentration of zero. The color from both fluorescent proteins was clearly transferred to the supernatant by this treatment. Anti-GFP nanobody (GST-GFPnb, GST fusion protein on Glutathione sepharose beads) was used for sedimentation, and the obtained precipitate was visualized by SDS-PAGE and subsequent Silver Stain. Figure 2 shows the results of electrophoresis of GFP pull down after adding Sepharose beads conjugated with anti-GFP nanobody to this lysate.

</P><CENTER><P class="fig">Figure 2. Immunoprecipitation of GFP from yeast lysate
</P></CENTER><P> As shown in Figure 2, the band labelled ① is GST-GFPnb, and the band labelled ② (5-8) is GFP. GFP is clearly immunoprecipitated by GST-GFPnb. Furthermore, since the band intensities of ① and ② are equal, we can expect that GFP and GFPnanobody interact at a 1:1 ratio. This indicates that the binding between GFP and GST-GFPnb is strong and GFP is in excess in the lysate.

For the band labelled ③, protein is seen at low salt concentration, yet becomes thinner as the salt concentration becomes higher. This is thought to be due to the nonspecific interaction under low salt concentration between GST-GFPnb and some protein present in the lysate. In addition, the band labelled ④ (lane 8) can only be detected at 1000 mM salt and in the presence of GFP (note that it is not detected in lane 4). This is thought to be due to the non-specific interaction under high salt concentration between GFP and some protein in the lysate. These data indicate that the salt concentration can influence three-dimensional structure of a protein, influencing protein-protein interactions.

 In the next section, we will show how we aimed to prepare yeast which adjusts salt concentration in solution.
</P></DIV><LABEL for="label2">2) Creation of salt-absorbing yeast</LABEL><DIV class="accshow"><P><B>  2-1) To inhibit Na+ efflux</B> In order to make yeast that effectively uptake salt from the environment, we aimed to reconstruct the Na+-isolation system of salt-resistant plants, which sequester Na+ into their vacuoles. We first tried to knockout NHA1 and ENA1, the critical components of Na+ efflux of budding yeast.[1][2]

Since there is no ENA1 knockout strain in the Yeast Knockout Collection, we amplified a hygromycin resistance cassette from pFA6a-hphMX6 and aimed to knockout the ENA1 gene using homologous recombination. We transformed PCR products into wild type yeast and obtained hygromycin-resistant colonies. Genomic colony PCR identified clones with the desired mutation. Using the same method, we succeeded to make a ΔENA1ΔNHA1 double-mutant strain from a ΔNHA1 strain.

Moreover, we obtained the G19 strain that lacks all of ENA1 family genes (W303 yeast have a cluster comprised of ENA1-ENA2-ENA3-ENA4). Although we obtained the G19 strain, we also introduced ENA1,2,5 knockouts (the equivalents of ENA1-4 in W303 yeast) to ΔNHA1 strain, due to the limited repertoire of selection markers that G19 strain has available.
 
 Finally, we prepared 4 strains: ΔNHA1 / ΔENA1ΔNHA1 / ΔENA1 / ΔENA1,2,5ΔNHA1. All genes we disrupted so far encode Na+ or K+ transporters that are responsible for Na+ export from the cytoplasm. Therefore, disruption of any of these genes should lead to decreased export efficiency of excess salt. Some researchers reported that yeast lacking these genes show an increased NaCl-sensitivity. To compare salt-resistance of knockout strains we prepared, we performed in vivo spotting assay.
</P><P class="fig">Figure 3. Spotting assay of KO yeast strains (cultured for 2 days on YPD plate)
</P><P>
Figure 3 shows representative results of spotting assay. With 200 mM NaCl in YPD, wild type strain formed large colonies after 2 days of incubation. In contrast, G19 strain and ΔENA1,2,5ΔNHA1 showed little or no growth, indicating a dramatic increase in salt-sensitivity for these strains. Under the same condition, <I>Zygosaccharomyces rouxii</I>, a yeast strain known for its high salt-resistance (as they have been used to make soy sauce in Japan), made large colonies. These results encouraged us to integrate genes of <I>Z. rouxii</I> to our device.

<B>2-2）To increase salt-resistance of yeast</B> The result we described above indicates that we successfully disrupted proper genes for our purpose, and thus mutants could not export Na+ from their cytoplasm, which is favorable to make salt-absorbing yeasts. However, from these results, we also noticed a serious problem.

As long as we use the mutants that absorb salt, we have to face the problem of salt-sensitivity. We hypothesized that if we could sequester Na+ in the yeast's vacuoles, salt-resistance might be recovered.
 However, even the yeast equipped with Na+ sequestration system, it would suffer from high salt condition to some extent. Hence, we concluded to overcome this problem, we would need to engineer salt-resistance into the yeast.
 In order to make salt-resistant yeast, we took three approaches: (1) to sequester Na+ into vacuoles by active transport, (2) to prepare compatible solutes to mitigate cell stress caused by osmotic pressure, and (3) to minimize the alteration of the higher order structures of proteins, which is caused by high Na+ concentration, by efficiently expressing chaperons. </P><P class="fig">Figure 4. Spotting assay of ΔENA1 expressing Mangrin (pRS316), AVP1 (YCplac111), AtNHXS1 (pRS313), AtHKT1 (pRS426), ZrGPD1 (pRS426), SseNHX1 (pRS426), and Control (empty pRS316). Yeast were cultured for 4 days on SD plates.
</P><P>
Figure 4 shows that all expressed genes, except for AtHKT1, successfully increased salt-resistance of the ΔENA1 strain. Among them, AVP1 and AtNHXS1 showed the greatest effect. Except for AtHKT1, all genes were already reported to confer NaCl-resistance to strains that lack the ENA1 gene family.[3][4][5][6][7] Also, there is a report that indicates AtHKT1 works on the plasma membrane to import increased amount of NaCl. The same study reported AtHKT1-expressing budding yeast showed high sensitivity to salt, consistent with our result.[8] However, in our experiment, we could not conclude AtHKT1 increased salt-sensitivity because AtHKT1-expressing yeast showed remarkably slow growth in the medium without salt.
 All  genes successfully increased salt-resistance of the ΔENA1 strain, making our device viable in high salt medium. Therefore, we are all set to develop a device that accumulates high concentrations of Na+ in their vacuoles. Importantly, these parts are not only relevant for our device development, but may also used for various other purposes. We expect that all of our submitted parts will be widely used as tools to increase the salt-resistance of budding yeast.
</P><P><B>2-3） How much Na+ can be absorbed by the device</B> With the assistance and advice of many experts, we successfully made the parts of interest. We then sub-cloned these genes into the plasmid that could be used in budding yeast and transformed them into yeast. We used a high copy number plasmid derived from 2-micron as an expression vector. We also used a low copy number plasmid derived from CEN plasmid because expression of some exogenous genes from high copy plasmids might lead to cytotoxicity (e.g. growth inhibition). As overexpression of exogenous membrane proteins may lead to stress response such as ERAD(endoplasmic reticulum-associated degradation), we have to be careful about expression of membrane proteins, which are the main components of our parts. Therefore, we have to carefully tune the expression level. Yeast harboring AVP1, AtNHXS1, SseNHX1, and mangrin in high copy plasmids showed significantly low growth rate. Strikingly, yeast harboring the mangrin gene in a high copy plasmid could not even form a colony.

We cultured various mutants we made in  media containing 400 mM NaCl and measured the amount of Na+ uptake by the flame photometry method during yeast growth (from OD=0.05 to OD=1.0). We show the average intracellular Na+ concentrations in yeast below.
</P><P class="fig">Figure 5. Intracellular sodium ion concentration (N=2)</P><P> First, we measured Na+ uptake by mutants that do not harbor any plasmids. As shown in Figure 5, even the wild type strain was capable of Na+ import to some extent. As we expected, the strains that lack the elements for Na+ export, such as NHA1 and ENA1, retained more Na+ than wild type in their cytoplasm. Among them, G19 strain, which lack all of the ENA1 family, and ΔENA1,2,5ΔNHA1 strain, showed especially high Na+ concentrations. These results support our ΔENA1,2,5ΔNHA1 hypothesis that these elements are main components of Na+ export and indicate that ΔENA1,2,5ΔNHA1 should be the chassis of Swallowmyces cerevisiae. 

</P><P class="fig">Figure 6. Intracellular potassium ion concentration (N=2)</P><P>
Interestingly, K+ concentrations of the same sample sets were greatly different from those of Na+. Again, ΔENA1,2,5ΔNHA1 that we made showed a remarkable feature. Although this strain took in the greatest amount of the Na+, it took up the smallest amount of K+ among 6 strains compared, showing that its selectivity of Na+/K+ uptake is the best among them.

Next we conducted a similar experiment using the ΔENA1 strain which expressed the parts we constructed and tested them for salt absorption.</P><P class="fig">Figure 7. Intracellular sodium ion concentration in ΔENA1 (N=2)</P><P> The result is shown in Figure 7. It turned out that control strain transformed with empty vectors takes up Na+ until the intracellular concentration reaches about 25mM. (NaCl concentration of culture medium is 400 mM). The most efficient intracellular retention of NaCl was observed in the strain expressing SseNHX1, and the strain in which AtNHXS1 was expressed absorbed NaCl at almost the same level, ranking second.


 These proteins are expressed in vacuoles and they transport H+ in vacuoles to the cytoplasm and transport cytoplasmic Na+ to vacuoles. In simplest model is like this; Saccharomyces cerevisiae that expressed these factors behaves just as like of salty tolerance plants that store Na+ taken from extracellular into vacuoles, so the amount of Na+ that can be stored by the whole cells remarkably increased. 
They showed more Na+ uptake when expressed from low copy plasmid, and when expressed by multicopy plasmid, the effect of both genes was greatly attenuated.
 The AVP1 gene is a Pyrophosphate-driven H+ pump localized on the vacuolar membrane and plays a major role in the enrichment of H+ in vacuoles. There is a report that by expressing this gene derived from <I>A. thaliana</I> will improve salt tolerance in budding yeast.[5]

This is interpreted as an increase in the H+ concentration in the vacuole, which promotes the relatively inactive endogenous NHX1 function of <I>Saccharomyces cerevisiae</I> and sequesters cytoplasmic Na+ into vacuoles. Increasing  intracellular Na+ concentrations in strains expressing AVP1 is considered to result from a similar mechanism.

 Interesting results were also obtained for AtHKT1. In contrast to the other factors, this factor has been reported to make yeast highly sensitive to salt as described before. However, in this NaCl absorption assay, it was found that it contributes to an increase in the average Na+ concentration in yeast as well as other factors. This suggests that AtHKT1 is involved in afflux of Na+ and anchoring to yeast by a different mechanism from other localized pumps expressed on vacuolar membrane. It has been reported that AtHKT1 is a co-transporter of Na+/ K+, it is localized on the cell membrane and seems to be directly involved in an influx of Na+ from external solution.[8]

 We saw interesting results from expression of the genes mangrin and ZrGPD1 as well. The former is a small peptide that works like a chaperone[3], and so we expected them to protect the host yeast from damage caused by salt. We have never thought that mangrin itself impacts NaCl absorption. However, when we did the experiment, it seemed that mangrin encourages the absorption of Na+ very strongly. Similarly, ZrGPD1, which is involved in the production of glycerol as a compatible solute, was also introduced to protect yeast from osmotic stress.[4] In this case also, although the mechanism is unknown, Na+ uptake of host yeast increased by an introduction of overexpression plasmid.

 We also tested their absorbance of Na+ using ΔENA1,2,5ΔNHA1. Strikingly, as shown in Figure 8 below, the strain expressing SseNHX1 gave the highest Na+ uptake value of all the engineered strains. We concluded that SseNHX1 is the best part in our hands to uptake Na+ from the media.
</P><P class="fig">Figure 8. Intracellular sodium ion concentration in ΔENA1,2,5ΔNHA1 (N=2)</P><P> Using our strains established in 2-2, we inquired how much Na+ can be removed from the media. For this purpose, we mixed 1 g yeast to 1 mL culture containing 100 mM NaCl. After incubation for 3.5h, aliquots were obtained and analyzed. As shown in the Figure 9, the strain expressing AVP1-SseNHX1 showed a rapid decrease in Na+ concentration in the media. These results indicate that we have successfully demonstrated our device, Swallowmyces cerevisiae, under realistic conditions.
</P><P class="fig">Figure 9. Sodium ion concentration of supernant of ΔENA1,2,5ΔNHA1 culture</P><P> As an aim to construct Swallowmyces cerevisiae, we have successfully made several devices that absorbs NaCl from a culture. Furthermore, we proved that each part actively transports Na+ from the medium into the cell.



</P></DIV><LABEL for="label3">3)Implement reliable biocontainment of genetically modified yeast</LABEL><DIV class="accshow"><P> Our primary goal is to construct a device which removes excessive Na+ from the environment. As indicated above, the construction of yeast which actually absorbs and reduces NaCl in the medium has been completed. However, during our Human Practice activities, the initial projects underwent major changes due to two points. The first change is the extension of future goal. We realized that this biological desalination system could be applied outside the laboratory as a measure against environmental salt damage, which is a big social and economic problem. The second problem is the problem of biocontainment that must be seriously considered when using genetically modified yeast outside the laboratory.


As shown in the Figure of Human Practice, a column type container for fixing and using yeast has already been used practically. Originally we planned to use this container as it was.

However, we noticed that this system is inefficient as it is. There is a problem that the column cannot be reused because it is difficult to replace the yeast grown inside. Yeast that has absorbed a certain amount of NaCl needs to be recovered or eliminated, such as removing them from the column or incinerating it, so that the incorporated salt does not return to the environment. Because the system utilizes live yeast, the user must pay close attention so that even a small number of engineered yeast do not leak into the environment.
In this situation, as a way to simplify the handling of yeast as much as possible, we worked on the creation of yeast artificial aggregation induction system. For this purpose, we focused on a very strong interaction between SdrG and FgBeta.[9] If yeast's surface display system can be used to present these two proteins on different yeast surfaces, yeasts will be tied together by strong interactions and form large cell clusters. Such a cell mass can be handled easier than handling a single small yeast cell. We considered this approach to be effective for preventing leakage of genetically modified yeast to the natural world.
 We fused the binding domain of SdrG/FgBeta to the following two parts; an artificial sequence which becomes the stalk region as a linker, and the anchoring domain of Sed1 for displaying SdrG/FgBeta on the yeast cell surface.[10] Plasmids expressing GFP and RFP (used for the immunoprecipitation described above) were simultaneously introduced into yeast expressing SdrG and FgBeta, respectively, so that each cell type could be distinguished.
 In order to see the interaction between SdrG and FgBeta created in vitro, these proteins were synthesized in a cell-free translation system and pulled down with a His tag fused to SdrG and a Flag tag fused to FgBeta. The results are shown in Figure 10 below. 

</P><P class="fig">Figure 10. TnT assay Gel</P><P>
As shown in Figure 10, SdrG was synthesized in a cell-free translation system and showed a clear band of the expected size. Pull-down by Ni-NTA is also shown, indicating that the target tag sequence is also working. However, it appeared that FgBeta was hardly expressed in the cell-free translation system. Since the monoclonal antibody used for immunoprecipitation was strongly detected at the time of Western blotting and became noise interfering with protein detection, it can not be concluded that translation products do not necessarily exist. However, both Rabbit Reticulocyte Lysate and Wheat Germ Extract showed the same tendency (for both SdrG can be observed but FgBeta cannot), leading us to conclude that FgBeta is unstable in lysate. Probably due to this reason, coprecipitation of FgBeta and SdrG could not be observed in either lysate.
 From another experiment we could obtain evidence that FgBeta is actually expressed in the cell and displayed on the surface of the yeast.


In the experiment, we pulled down yeast displaying FgBeta on the cell surface with magnetic beads coated with anti-Flag antibody, and after washing with PBS, the bound cells were observed with a fluorecent microscope. 

</P><P class="fig">Figure 11. Fluorecent picture after fishing with magnetic beads (upper left: FdBeta・anti-Flag antibody beads, upper right: FdBeta・SdrG・anti-Flag antibody beads, bottom: SdrG・anti-Flag antibody beads)</P><P>

The results are shown in the Figure 11. As expected, yeast containing FeBeta (Flag) bound very well to anti-Flag antibody beads. It was observed that yeast with many RFP signals were bound to the washed beads. In contrast, when yeast displaying SdrG (His 6) was extracted in the same manner, yeast with GFP signal were not recovered. This indicates that the anti-Flag antibody beads do not randomly precipitate yeast at all, but rather specifically recognizes the Flag sequence displayed on the yeast cell surface. From the above results, the following was clarified.

(1) FgBeta (Flag) was appropriately displayed on the surface of yeast cells by fusion with Sed1 anchoring domain.

(2) Only specific yeasts could be immobilized on magnetic beads using the antigen placed on the surface.

SdrG (His 6) was similarly displayed on the cell surface by Sed 1 anchoring domain, but SdrG (His 6) expressing cells could not be recovered with His-trap magnetic beads. This may be due to the presented His6 tag sequence being inaccessible to magnetic beads due to steric hindrance, or because SdrG (His 6) has a lower expression level per cell.
In addition, FgBeta(FLAG) expressing cells mixed with SdrG (His 6) expressing cells was captured with magnetic beads coated with anti-Flag antibody. The upper right figure shows significant decrease in the number of captured cells compared to the upper left image. From this data we cannot draw a clear conclusion, however we suspect it may be consequence of poor interaction between FgBeta and SdrG. </P></DIV></DIV><DIV class="reference"><H6>Reference</H6><UL class=""><LI>[1] R. Haro, B. Garciadeblas, A. Rodriguez-Navarro (1991) A novel P-type ATPase from yeast involved in sodium transport, <I>FEBS Letters</I> Vol.291 Issue2 189-191</LI><LI>[2] Jos6 A. Miirquez, Ramdn Serrano (1996) Multiple transduction pathways regulate the sodium-extrusion gene PMR2/ENA1 during salt stress in yeast, <I>FEBS Letters</I> Vol.382 Issue1-2 89-92</LI><LI>[3] A. Yamada, T. Saitoh, T. Mimura et al. (2002) Expression of Mangrove Allene Oxide Cyclase Enhances Salt Tolerance in <I>Escherichia coli</I>, Yeast, and Tobacco Cells, <I>Plant and cell physiology</I> 903-910
</LI><LI>[4] Hou,Lihua Wang,Meng Wang,Cong Wang,Chunling Wang,Haiyong (2013) Analysis of salt-tolerance genes in zygosaccharomyces rouxii, <I>Applied Biochemistry and Biotechnoloogy</I> 1417-1425 </LI><LI>[5] R. Gaxiola, R. Rao, A. Sherman et al. (1999) The Arabidopsis thaliana proton transporters, AtNhx1 and Avp1, can function in cation detoxification in yeast, <I>Proceedings of the National Academy of Sciences of the United States of America</I> Vol.96 Issue4 1480-5 </LI><LI>[6] K. Xu, H. Zheng, E. Blumwald et al. (2010) A novel plant vacuolar Na+/H+antiporter gene evolved by DNA shuffling confers improved salt tolerance in yeast, <I>Journal of Biological Chemistry</I> Vol.285 Issue30 22999-23006 </LI><LI>[7] G. Wu, G. Wang, J. Ji et al. (2015) A chimeric vacuolar Na+/H+antiporter gene evolved by DNA family shuffling confers increased salt tolerance in yeast, <I>Journal of Biotechnology</I> Vol.203 1-8 </LI><LI>[8] N. Uozumi, E. Kim, F. Rubio et al. (2000) The Arabidopsis HKT1 gene homolog mediates inward Na(+) currents in xenopus laevis oocytes and Na(+) uptake in Saccharomyces cerevisiae., <I>Plant physiology</I> Vol.122 Issue April 1249-1259 </LI><LI>[9] L. Milles, K. Schulten, H. Gaub et al. (2018) Molecular mechanism of extreme mechanostability in a pathogen adhesin, <I>Science</I> Vol.359 Issue6383 1527-1533 </LI><LI>[10] K. Inokuma, T. Bamba, J. Ishii et al. (2016) Enhanced cell-surface display and secretory production of cellulolytic enzymes with Saccharomyces cerevisiae Sed1 signal peptide, <I>Biotechnology and Bioengineering</I> Vol.113 Issue11 2358-2366 </LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>