<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCSD_Software_Teaml"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UCSD_Software/Teaml">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCSD_Software/Teaml&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSD_Software/Teaml&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCSD_Software/Teaml&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCSD_Software/Teaml" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCSD Software/Teaml</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><DIV class="btn-group  btn-group btn-group-justified"><DIV class="btn btn-lg " id="home">Home</DIV><DIV class="btn btn-lg " id="team">Team</DIV><DIV class="btn btn-lg " id="project">Project</DIV><DIV class="btn btn-lg " id="doc">Documentation</DIV><DIV class="btn btn-lg" id="notebook">Notebook</DIV><DIV class="btn btn-lg" id="safety">Safety</DIV><DIV class="btn btn-lg" id="attributions">Sponsors</DIV><DIV class="btn btn-lg" id="achieve">Achievements</DIV></DIV><DIV class="neighborhood-guides"><DIV class="container"><H1 class="row text-center">Meet the Team </H1><DIV id="undergraduates"><H2><B>Undergraduates</B></H2></DIV><H2><B>Advisors</B></H2><DIV class="row" id="p1"><DIV class="col-md-12"><P><FONT size="3px">
UCSD’s iGEM team will join over 200 teams from across the country for a chance to compete for the BioBrick trophy. 
UCSD’s computational iGEM team will focus on recruiting a team from a wide range of academic disciplines in order to expose students to interdisciplinary research. iGEM team members will have a chance to work closely with mentors during the summer, fostering a diverse skillset necessary for interdisciplinary fields such as Systems Biology. In addition to acquiring technical skills while working on the iGEM project, students in iGEM will also refine their presentation skills – typically iGEM students present their work in: an oral presentation at the iGEM World Jamboree, on a research poster, online wiki, and ocassionally, a submission to an academic journal. Participation in UCSD’s iGEM team will raise not only students’ confidence in their research abilities, but also their awareness of opportunities in emerging fields including Systems Biology.
             </FONT></P></DIV></DIV><DIV id="w1"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/in/hyeerna"><FONT color="2F4F4F">Huwate (Kwat) Yeerna (Ernar)</FONT></A></H2><P>Huwate (Kwat) is from Kazakhstan, and he studies Bioengineering: Bioinformatics and Mathematics at University of California, San Diego (UCSD). He is passionate about cancer genomics, and has been conducting bioinformatics research on cancer at the Trey Ideker laboratory at UCSD. He wants to become a pediatric oncologist/medical scientist to improve pediatric cancer treatments. His immediate goal is to obtain an M.D./Ph.D. degree by expanding his bioinformatics research on cancer heterogeneity. He is burning with a desire to improve pediatric cancer treatment and beyond.</P><UL><LI>Bioengineering: Bioinformatics, Mathematics2015</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                                Collaboration was the theme of this past spring, summer, and this fall.

We have created a computational synthetic biology tool, SBiDer, that fosters collaboration among the global synthetic biology community. SBiDer also has a potential to bridge between other fields of biology and the field of synthetic biology by enabling ontological interpretations of biochemical reactions as well as genetic interactions. Furthermore, SBiDer database can become the first universal database for the field of synthetic biology. Lastly, we have minimized the barriers for collaborative development of SBiDer by modularly engineering SBiDer on an open source platform. We encourage the global community to extend, develop, and optimize SBiDer - together.

Our team has developed SBiDer from the ground up, and my primary tasks have been the engineering of the database and the search algorithm. Development of the database and the search algorithm were daunting tasks that challenged me in many ways. Yet, I embraced the challenge. The more arduous the project became, the harder I worked. As a result, this assiduous process has provided me with a fantastic opportunity to improve my skills in computer programming, bioinformatics, and mathematical modeling. Now, I truly appreciate the value of the challenges that I faced and the priceless opportunity of this education.

Developing SBiDer codes was hard, and effectively leading a groups of people was even harder. Technical challenges not only made me a critical thinker, but also harnessed my skills as leader that ultimately made me a better scientist. In the process of interacting with my teammates, I felt a maturation of myself. I have learned to be more conscious about others’ struggles. I learned to sacrifice for others in developing the project, which is key in teamwork. I learned a lot about proper teamwork ethics. Looking back, I am surprised how much I have learned about communication, effective leadership, project development, and more. I believe these skills have made me a better scientist. Without working together, we would not have been able to develop SBiDer. SBiDer is a result of our communal effort.

I believe that science and collaboration are closely intertwined, for collaboration fundamentally advances science. I am grateful for the opportunity in developing SBiDer and working with my team: Joaquin, Jenhan, Justin, Ben, Ryan, Andreas, KP, Lauren, Valeriy, Gary, Fernando, Luyao, and Rohit. I learned tremendously from the collaboration that was coalesced with the science. And finally, we made SBiDer. The past seven months have been filled with learning opportunities, arising in unexpected manners that have led a an enjoyable and fruitful journey.

I hope the community enjoys SBiDer and can contribute to its further development. 

All in all, collaboration was not only the theme of this past spring, summer, and this fall, but it will be the theme of my life.



                                </P></DIV></DIV></DIV><DIV id="w2"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/profile/view?id=281609511&amp;trk=nav_responsive_tab_profile"><FONT color="2F4F4F">Joaquin Reyna</FONT></A></H2><P>4th year bioinformatics undergraduate at UCSD. He worked at the Salk Institute lab as a research assistant. His research interests include genetic diseases specifically muscular dystrophies and has strong interest using programming skills for the advancement of biological knowledge. After graduating he plans on continuing research and attending graduate school. </P><UL><LI>Bioengineering: Bioinformatics, 2015</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                         I was excited for the start of summer 2014 for the experiences and work that would be done by our UCSD iGEM team. The focus of my work was the database creation. The initial progress of the SBiDer database included lots of learning SQL and Python for the usage in the SBiDer database creation. Although I had some programming experience in Python which helped populate the database, the creation of the database itself was not an easy task. To create just any database is easy, but what I found to be really difficult was creating a database schema that planned ahead for the SBiDer network. I had great team members and together we managed to both create a reliable schema and populated a database that helped move our project forward.

I came into iGEM expecting to learn a lot of about programming, but iGEM was also a great experience to learn teamwork. Communicating seems like a given in a team project, but that doesn’t mean that ideas are put into clear words. I learned to talk clearly with my teammates to establish tasks we had to do and how we could work together. As the project was coming to a close we really depended on one another and I learned to sit down and work out problems with people.
For the future I expect UCSD iGEM teams to be able to build upon our successes and failures. Being part of iGEM this summer has encouraged me to begin a club on campus that focuses on synthetic biology, stressing programming and teamworking skills that would make future UCSD iGEM teams successful and competitive.      
                    </P></DIV></DIV></DIV><DIV id="w3"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/profile/view?id=306167421&amp;trk=nav_responsive_tab_profile"><FONT color="2F4F4F">Valeriy Sosnovskiy</FONT></A></H2><P>Valeriy is finishing up his bachelor in Biochemistry and Cell biology with a minor in computer science. This is Valeriy's first time participating in IGem. Valeriy has worked in two labs dealing with the mechanisms of apoptosis but his interest is to become a software developer. </P><UL><LI>Biochemistry, Computer Science, 2015</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             Having little experience with programming, the possibility of applying my knowledge in Biochemistry to help create an application was very appealing. The chance to improve in programming and possibly learn different coding languages definitely was the icing on the cake.
Biochemistry is something I have always been fond of, but having a chance to translate that knowledge into something that people could use and benefit from was extremely exciting. iGEM was a chance to make that happen.
 
       During my time as a member of the UCSD Software  iGEM team, I learned many different coding languages including  html, JavaScript, Python,  and Java.  Throughout the entire development process I got a chance to work with the front-end and the back-end parts of the application development, with my primary focus being web development, as well as the user interface portion of the application. This was an enriching experience as I learned many different techniques as time went on, and now feel more comfortable as ever with the ability to design web applications in the future.  The project taught me many skills and bonded me with all my teammates, and as a result we were able to develop an application that we believe could possibly change data mining. 
                    </P></DIV></DIV></DIV><DIV id="w4"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/profile/view?id=210400159&amp;trk=nav_responsive_tab_profile_pic"><FONT color="2F4F4F">Lauren Crudup</FONT></A></H2><P>Lauren Crudup is a bioengineering student at the University of California, San Diego. Lauren works in Dr. Coleman's Neural Interactions Laboratory where the purpose of her current research is to contribute to this expanding body of work related to neurological signal processing by developing a reliable testing mechanism for measuring the physiological and neurological changes associated with empathetic behavioral patterns in rats. She is also currently leading an engineering project team through Engineers Without Borders focused on determining an effective and efficient way to automate a large-scale algal media testing system. Lauren hopes to continue to develop the programming skills she has learned through the iGEM program and her other research experiences in her future career.</P><UL><LI>Bioengineering, 2015</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             When I applied to become part of the UCSD iGEM team, I understood that it would be a substantial challenge given my lack of knowledge about synthetic biology. As the Spring quarter progressed and we were introduced to a plethora of programming concepts, what was being asked of us as members of the iGEM team became clear. Instead of shirking at the responsibility that lay before us, everyone started the project with the enthusiasm about the task we had been given.

As we moved into summer, I had the opportunity to work on code for the search algorithm, JSON file creation, and generation of the Pigeon images as well as work on the complexity analysis. Each project has enabled me to build a strong background in programming and synthetic biology, and I hope to continue to pursue both in the future.

We have had a great opportunity to build a complex, novel application that has the ability to change the landscape of synthetic biology by allowing community collaboration. We have enabled the community to stand upon the shoulders of a SBiDer-sized giant.
                    </P></DIV></DIV></DIV><DIV id="w5"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/pub/gary-le/a3/720/639"><FONT color="2F4F4F">Gary Le</FONT></A></H2><P>Gary is a second year undergraduate at UC San Diego born in Milpitas, California with a strong interest in biological studies and medicine. Prior to involvement with the UCSD Software team, he had little exposure to web application development. Aiding the team in building the SBiDer web tool has given him a greater understanding of synthetic biology, and he is motivated to continue pursuing study in the new field.</P><UL><LI>Human Biology, Business, 2017</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                          I started aboard the UCSD Software iGEM team after hearing about a project for undergraduates centering around the relatively new field of synthetic biology. The subject was one I had no practical experience with but was one that I had been avidly following since I had first heard of the term &quot;synthetic&quot; being applied to a field of biology. With no knowledge of how to program in any language and with only an introductory knowledge into genetic circuitry, I decided to apply for a chance to help in the endeavor. At first, I was both intimidated by the scope of the project planned and the skills set of my future team members and mentors, but I grew more excited and impressed the more skill I had begun to learn and concepts I started to understand. My skill set expanded rapidly in the oncoming months to include ways of analyzing problems algorithmically and constructing methods to approach a solution above the various programming languages I learned.

These skills were implemented with equal rapidity as I worked to create a user interface for the manually-curated network of synthetic genetic devices. The weeks passed before I had realized, and I felt overwhelmed at times by the material I needed to not only learn but master in order to create a viable component to the SBiDer project. However, with guidance from the graduate students and practice, I was able to implement the front-end application that was capable of rendering a visual network of the database information that could dynamically be altered and manipulated to allow users to explore connections of the database through a graphic interface. I additionally spent time learning from other team members about the nature of the genetic circuits to improve my biological foundation as well as how to manipulate and represent data in order to begin constructing a way to aid in creating models of the circuit behavior.

Like all goals worth pursuing, working with the UCSD iGEM team required many hours of difficult work and challenges to be solved. The result , however, improved my own personal ability and produced a project with a growing potential to aid a large audience of people. After the conclusion of this year's jamboree, I hope to have the pleasure to continue working with my team to create a lasting impact upon our campus by educating other undergraduates and providing them the skills to tackle an opportunity like this both in an academic setting and in other aspects of their life. I am proud of my team and our work and grateful for the many outstanding people that allowed this event to unfold.   
                    </P></DIV></DIV></DIV><DIV id="w6"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/pub/kartikeya-puri/73/557/a09"><FONT color="2F4F4F">Kartikeya Puri</FONT></A></H2><P>Kartikeya Puri (KP) is a third-year Computer Science major at UCSD. Originally from Mumbai, India, he came to UCSD as a freshman to pursue a bachelors of science (B.S.) degree in Computer Science, a field that he's completely in love with. He’s passionate about web development, and also serves as the web master at the Undergraduate Bioinformatics Club (UBIC) at UCSD. Outside of school and work, he enjoys playing and watching soccer, &amp; religiously follows Manchester City Football Club. 
                        <UL><LI>Computer Science, 2016</LI></UL></P></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             This is my first year taking part in iGEM, and also my first research project as an undergraduate student. When I read about UCSD building a team to take part in iGEM, I knew wanted to be a part of the team but was apprehensive at first as I wasn’t sure if, as a Computer Science major, I’d fit into a team participating in a Synthetic ‘Biology’ competition. After meeting Jenhan for the first time, though, a lot of my fears had subsided. He eased me into the team and let me know that I didn’t require a lot of background or experience in biology as this project was meant to be a learning experience for everybody.

He couldn’t have been more right about that; we learned &amp; picked up a ton of skills this summer, ranging from reading scientific papers, mining data, building a database based on that data to figuring out how a real world software project works. We learned about little details such as setting realistic expectations and deadlines for tasks. I personally learned a lot about what entails the field of Synthetic Biology (pro advice: reading nature journals on the topic helps with that ;).  Having worked on the website and the application that we built, I also picked up several web development skills.  

Overall, taking part in iGEM 2014 and representing UCSD at an international level was a fun &amp; highly enriching learning experience.  Over the course of the summer, we put together an awesome app with the help of our advisors. I feel proud to have been part of the team that built one of the most breathtaking Syn Bio apps this year – SbiDer. Special shout-out to all our advisors –Jenhan, Justin, Ben &amp; Ryan for guiding us and inspiring us throughout summer. Stay  awesome, people. 


                    </P></DIV></DIV></DIV><DIV id="w7"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/profile/view?id=240077984&amp;authType=NAME_SEARCH&amp;authToken=k-dM&amp;locale=en_US&amp;trk=tyah&amp;trkInfo=tarId%3A1411758557752%2Ctas%3Aluyao%20zhou%2Cidx%3A1-4-4"><FONT color="2F4F4F">Luyao Zhou</FONT></A></H2><P>Luyao (Fred) is a forth year Electrical Engineering student from UCSD. He is an international student form China. He speaks Chinese, English and entry-level Spanish. He has pervious experience of programming in Java, C++, Python, and Assembly Language, and circuit design. He has perviously worked as Tutor for CSE21 in UCSD and Software Development intern in SINOPEC. He expects to be a Software Engineer in the future. </P><UL><LI>Electrical Engineering, 2015</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             I am really excited for being part of our IGEM team. When I first joined IGEM, I expected to use my programming knowledge to help the team build our project, but IGEM was a great experience to learn new things. Although I have never programmed in python, but IGEM helped us learn these necessary skills before project started. My work is to create a reliable database. Thought my work, I managed to search the materials online and learn by myself. The creation of the initial database didn’t take long, but then we realized that building a database with a robust schema is more important. Working together with my teammates, we finally find a way to populate the database. Teamwork is the next important thing I learned in IGEM. IGEM isn’t just a individual project. We can only put small pieces together with efficient communication. So I learned to working out problems with my team. Another thing I learned in IGEM is planning. We break the whole project into smaller parts and set up clear goals for each one. It helped move our project forward. For the future, I hope I can join more projects like IGEM so I can create meaningful products and also improve myself.
                    </P></DIV></DIV></DIV><DIV id="w8"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="https://www.linkedin.com/profile/view?id=372964518&amp;trk=nav_responsive_tab_profile"><FONT color="2F4F4F">Fernando Contreras</FONT></A></H2><P>Fernando is an undergraduate student in the Bioengineering program at UCSD with a

specialization in Biosystems. His research interests lie in biological signal transmission and 

network modeling. As a pastime, Fernando enjoys jamming with other UCSD musicians.</P><UL><LI>Biosystems, 2017</LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             An interest in artificial intelligence coupled with a philosophical inclination developed an 

affinity for the notion of consciousness, which, in turn, entertained the idea of &quot;biological

programming.&quot; With this newfound interest in research, I sought to find a lab position in 

biotechnology at UCSD; however, research in a DNA biotechnology lab did not suffice. Prior 

to iGEM, I never considered Synthetic Biology as a potential avenue in biotechnology research. 

However, after Jenhan, our team coordinator, provided me with a brief description of Synthetic 

Biology, I was undoubtedly convinced that I had found my niche in research. 

As our software project unfolded during the summer, I became responsible for the visualization 

and analysis of our complex genetic network. Following the integration of the different genetic 

circuits that we found in literature, I proceed to generate a graphical representation of our 

database that conveyed the relationship between the different genetic circuits. In addition, I 

performed a structural and connectivity analysis, to demonstrate the increase in potential genetic 

circuits that our network can provide.

This project introduced me to the potential of Synthetic Biology, and demonstrated the 

importance of engineering and computational principles in synthetic and systems biology.
                    </P></DIV></DIV></DIV><DIV id="w9"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2><A href="http://www.linkedin.com/pub/rohit-mande/7b/809/516"><FONT color="2F4F4F">Rohit Mande</FONT></A></H2><P>Rohit is a fourth-year Bioengineering: Bioinformatics major who is
interested in entering the bioinformatics or synthetic biology industry
after graduation.  He has an active interest in the design and analysis of
biological systems,, and hopes to continue to do DIYbio work outside of
college.    He is primarily looking forward to iGEM as  a way to
collaborate and work with  undergraduates who share the same interest. 
Outside of academics, Rohit is the Internal Officer for the Undergraduate
Bioinformatics Club (UBIC) and is an active member of the UCSD Quiz Bowl
club.</P><UL><LI>Bioengineering: Bionformatics, 2015 </LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             I initially heard about iGEM through an email from the Undergraduate Bioinformatics

Club (UBIC) at UCSD, of which I am currently serving as the Internal Officer. I was looking for a

research opportunity over the summer to further gain exposure to computational biology before my

final year of college. When I first met Jenhan, I was a little taken aback to discover that most of

the team had already been involved with the competition for a few weeks, but when I heard the

scope of the project, my interest immediately grew and I was excited to be a part of UCSD's first

ever iGEM team.

At first, familiarizing myself with iGEM proved to be a greater challenge than I had

expected. I had no previous exposure to the field of synthetic biology, and because I had joined at

a later date than most of the team, I started the summer already behind everyone else's knowledge

and frequently felt lost during meetings. In addition, difficulties in finding housing in San Diego

over the summer meant that I was making an hour­long commute to meetings twice a week and

working remotely for the rest. Despite this, iGEM has been a great experience. Through this

project, I learned the Python programming language, refined gaps in my programming knowledge

base, and learned a great deal about synthetic biology theory. In particular, I focused on network

analysis and network motifs. For my part of the project, I read through literature detailing various

network motif finding algorithms and implemented one of them on our network in order to conduct

analysis and demonstrate an increase in functional genetic circuits in our network.

I definitely feel that I chose the right way to spend my summer by joining the iGEM

team. Not only have I learned a great deal about synthetic biology, a field which I previously

knew nothing about, but I have learned how to work in a large group environment, how to

collaborate on a brand new project, and how to meet tight deadlines. IGEM has provided me with

a valuable skill set which I plan to use in a hopefully long and successful career in the

bioinformatics industry.
                    </P></DIV></DIV></DIV><DIV id="w10"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2>Jenhan Tao</H2><P>Jenhan is a PhD student in the Bioinformatics and Systems Biology program at UCSD. He's interested in utilizing computational technqiues to increase our capacity to understand complex biological problems such as cell signalling. This is Jenhan's second year working as an iGEM mentor. Previously Jenhan was involved in synthetic biology research as an iGEM team member at Boston University and as a research assistant at UC Berkeley, the BioFAB, and Boston University.</P><UL><LI>Bioinformatics PhD, 2018
                            Bioengineering, UC Berkeley, 2012
                            </LI><LI><A href="https://www.linkedin.com/pub/jenhan-tao/1a/763/83b">LinkedIn</A></LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                         This is my third year being involved with iGEM - previously I've been an undergraduate iGEM team member 2011 as well as a mentor in 2013 - and so I thought I was ready for anything and everything. However, iGEM at UC San Diego this year caught me by surprise. Before I get into the summer's surprises, I'd like to thank our sponsors. Without generous support from Synthetic Genomics and the San Diego Center for Systems Biology, none of this would have been possible - their support gave UCSD students a chance to experience a genuine research experience in an interdisciplinary and collaborative setting. I had an excellent summer designing the project with the undergraduates, teaching them, and working with them. I think UCSD's first iGEM team has managed to impart useful lessons to all those involved, and I'm highly hopeful a second team will follow next year.

Even as an iGEM veteran, the amount of work involved in coordinating this project and mentoring each of the students was quite unanticipated. Fortunately, I had quite a bit of help. I'd also like to thank my fellow mentors, Ben, Justin, Ryan, and Andreas, for helping out and stepping up as needed. Each one of them selflessly gave their time to give patient guidance for the undergraduates. Barry Demchak, Kei Ono, and the rest of the Cytoscape group gave us some great technical advice that really helped to make sure that the team took the most effective software development strategy. Other individuals that gave us some great advice include Jason Kreisberg and Prof. Todd Coleman. Risa Shibata and Anna Lu also deserve a lot of thanks for their invaluable help with logistics and making sure that we could fly to Boston and be at the Jamboree. So I'd guess the amount of work involved in iGEM was ultimately balanced by the amount of support we received. 

As the project is finally wrapping up, I'm amazed at how much the students were able to accomplish in the past few months. Many of our students had little exposure to coding, and I'm glad that iGEM at UCSD was able to help them grow into proficient and innovative programmers. On the other side of the coin, we had a few computationally oriented students who managed to leverage synthetic biology concepts to help build a project that everyone on the team can take pride in. I think the project was very ambitious in its design, and I was definitely surprised when all the pieces finally clicked together. I think this was due in no small part to the leaders that emerged from amongst the undergraduates. Without the motivated students involved in this project, we would not have even gotten halfway through the project. Each of the students who worked hard on this project should be proud of themselves. 

Overall, I'm very glad to have had an opportunity to be involved in UCSD's first ever iGEM team. iGEM is a great program, and my experience with iGEM in the past is one of the main reasons why I decided to pursue graduate studies. And I hope that I've done my small part to show students that research can be exciting, meaningful, and enjoyable.    
                    </P></DIV></DIV></DIV><DIV id="w11"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2>Justin Huang</H2><P>Justin Huang is a second year Bioinformatics PhD student at UC San Diego. Originally hailing from the great state of North Carolina, Justin completed his undergraduate studies with a BS in Biology and a BA in Mathematics from the University of North Carolina at Chapel Hill in 2013. Justin's research interests are in translational medicine, specifically in the role of translational medicine in cancer. Justin is also an avid sports fan, and an advocate for Asian American issues. You can follow him @huangger on Twitter.</P><UL><LI>Bioinformatics PhD, 2018
                            Biology,Mathematics University of North Carolina-Chapel Hill, 2013
                            </LI><LI><A href="www.linkedin.com/in/justinkhuang">LinkedIn</A></LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             When Jenhan first approached me about mentoring undergraduate students for a summer software project in synthetic biology, I was initially very excited at the opportunity, but at the same time I was nervous. Many of the students we recruited had very little experience programming, and our project was very ambitious. We spent the better part of the Spring quarter teaching skills to the undergraduates ranging from synthetic biology concepts, to algorithm development, to learning how to read scientific papers. I was pleased and impressed at how quickly the undergraduate students were able to pick up these new skills, and when summer arrived, we were ready to hit the ground running.

My undergraduate background was in biology and mathematics, but all of my research experience had been in bioinformatics up to this point. I had never worked on a synthetic biology project either, but after attending IWBDA 2014, I was able to learn more about the field, giving me the confidence to move forward in mentoring this group of undergraduates. Software development was also a new project style for me, so the process has also been a great learning experience for me as well. Much of my summer was spent helping the students develop and analyze algorithms to perform searches and queries across our manually-curated networks. I was also involved with the database schema development to better represent the network we were trying to present. As a member of the Trey Ideker lab at UCSD, I have spent the past year working closely with network biology and tools used in network biology, which has been important to the development of both the back-end database and the front-end application. 

Additionally, as a new graduate student, I felt it was an excellent opportunity to begin sharpening my skills as a teacher and mentor to students not only on their projects but also their academic careers. I have found that I myself have a great love for teaching and lecturing, and it has truly been rewarding to see how far many of these students have come this year. This year, we have also been part of UCSD's very first iGEM team ever, which in itself, comes with a unique set of challenges. Our team did all of its own fundraising to pay for the registration costs of iGEM as well as the travel costs to Boston. Along the way, we have also pitched our team in addition to the project itself to several labs at UCSD in hopes of creating a sustainable team for years to come. However, even with these limited resources and barriers to our project's development, particularly in terms of our wet lab validation for this project, I am extremely pleased with what we've been able to accomplish this summer and I am proud to have been a part of UCSD's first iGEM team.
                    </P></DIV></DIV></DIV><DIV id="w12"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2>Benjamin Kellman</H2><P>My name is Benjamin Kellman. I am a PhD student at the University of California at San Diego studying Bioinformatics and Systems Biology. My work uses existing formalisms in mathematics and computer science to consolidate the spurious information of experimental biology to a more codified rule set; many such reductions have already been made. I hope to apply fundamental biological axioms formalism to increase the power of my biological predictions.</P><UL><LI>Bioinformatics PhD, 2018
                            Biology, University of Rochester, 2012
                            </LI><LI><A href="https://www.linkedin.com/profile/view?id=331388590&amp;authType=NAME_SEARCH&amp;authToken=3p2B&amp;locale=en_US&amp;trk=tyah2&amp;trkInfo=tarId%3A1412133206770%2Ctas%3Aben%2Cidx%3A2-1-2">LinkedIn</A></LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             Advising for iGEM has been a fascinating engagement and a positive force towards my personal and professional development this summer.  I enjoyed discussing and formulating our project with the undergraduates. The novelty of their insight was exciting and fun. Their perspectives are untainted by the priors of the field and often stemmed new and exciting conversations.  Educating new students was also a great way to consolidate and confirm my understandings and preconceptions.  Revisiting details assumed in professional conversation is hard but ultimately beneficial.  I was delighted to discover several shortcomings in my current understanding of Synthetic Biology over the course of teaching this project.

I also had the opportunity to learn new skills such as database construction, normalization, maintenance and accession.  I had never managed a database before.  This posed a special challenge to advising its construction.  Learning enough to advise is certainly more difficult but ultimately very rewarding.  I had the immediate enjoyment of learning something new followed promptly by the joy of sharing my understanding with an eager audience.

It was also interesting to managing a team of this size for the first time.  There were some pitfalls and lost time in the beginning but eventually my co-advisors and I learned enough about distributed management and collaborative infrastructure--mostly google docs--to effectively manage a team of 10 undergraduates.

I realize that I saw many professional benefits in my ongoing experience with iGEM, but all professionalism aside, I really enjoyed my time teaching, learning and leading this iGEM project.
                    </P></DIV></DIV></DIV><DIV id="w13"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2>Andreas Dräger</H2><P>Andreas Dräger works as an independent research scholar in the Systems Biology Research Group at UCSD. He studied bioinformatics at the Martin Luther University in Halle and worked as an intern at the Max Planck Institute for Molecular Genetics, Berlin, and at the University of Illinois at Chicago. He did his PhD at the Center for Bioinformatics in Tübingen and worked as visiting research student at Keio University in Yokohama. The Faculty of Science (University of Tuebingen) awarded his PhD thesis as the most outstanding work of 2011 and granted him a funding for an own graduate student.</P><UL><LI>Systems Biology PhD, 2011
                            Master of Bioinformatics, 2006
                            </LI><LI><A href="https://www.linkedin.com/in/andreasdraege">LinkedIn</A></LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                             After my first discussion with the graduate students, who were just
about to launch an iGEM project at UCSD, my impression was that the aims
of this project were extremely ambitious. This made me very curious and
I was excited to be on board. It soon fleshed out what would be the
precise goal of the project and which tasks should be done. The great
organization and strict planing, the regular meetings and terrific
support of the undergraduate students by their mentors finally made this
project an ongoing success. 
During the project, everybody could learn and experience a lot of new
aspects of research in diverse fields, reaching from project planing,
organization, and conduction on the one side through scientific
disciplines bioinformatics, database management systems, software
design, interactive graph drawing, data exchange formats and
standardization, and synthetic biology to communication, writing, and
presentation skills on the other side. It can be assumed that this large
interdisciplinary and highly collaborative team work was the first
experience of this kind for the majority of students and therefore a
great experience for all participants. It is remarkable that the project
was always focused on the overall goal to obtain a useful product at the
end rather than endlessly discussing tiny details of each aspect.
As assumed at the very beginning, this was an enormously challenging
effort, but all participants made this project a success. To me, it is
still amazing how all this could be achieved in this short time and
while in parallel educating the undergraduate students in so many
disciplines.
                    </P></DIV></DIV></DIV><DIV id="w14"><DIV class="row" id="graduates"><DIV class="col-md-6"><DIV class="thumbnail text-center" id="gradPic"><H2>Youbin Mo (Ryan)</H2><P>Youbin Mo is one of the advisors of UCSD_Software iGEM team in 2014. As a PhD student in biophysics, he is an unquestionable master of biological model and computer programming on which he helped team members to develop a novel modeling approach that modeling bio-brick individually then constructed circuits as wet-lab experiment. In the meanwhile, website constructing is also Mo's technical ability which he acquired during the iGEM previous competition. Youbin gives play to his talent by present his modeling skills to new iGEMers as well as directing them to be self-reliant synthetic biologists.</P><UL><LI>Biophysics PhD, 2019
                             Bachelor's degree, Physics, 2013
                            </LI><LI><A href="http://la.linkedin.com/pub/youbin-mo/71/41/9b">LinkedIn</A></LI></UL></DIV></DIV><DIV class="col-md-6"><P class="pull-left">
                       In this summer, I donated my efforts on helping UCSD iGEM team to set up a new modeling format which modularized devices as individual function and standardized the function’s structure. Based on the concept of iGEM, we started from building up dozens of sub-functions that represents a set of simple devices which were the fundamental bio-brick in synthetic biology. Then we searched some literatures and picked up a system for testing our novel modeling method. Two arrays of ODEs were defined for representation of the circuits we chose. A python script was written to integrate these ODEs. 

As result, the plot from wet-lab experimental data was reproduced by our differential equations. The simulative output approximately matches the curve plotted in citation which  shows the idea that modeling single devices and linking them as connecting bio-brick works like we expected. In next step, we tried to plugin this function into our web app but we didn’t finish because time was over. It is hopefully to be done in future. 
</P></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:UCSD_Software/Teaml">http://2014.igem.org/Team:UCSD_Software/Teaml</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCSD_Software/Teaml" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCSD_Software/Teaml" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCSD_Software/Teaml&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCSD_Software/Teaml&amp;oldid=401225" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>