<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UCAS_Attributions skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UCAS/Attributions</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="catalog"><LABEL><FONT color="blue">ucas</FONT>lab</LABEL><UL><LI><A>Safety</A><P>safe project, lab and shipment</P></LI><LI><A>Methods</A><P>experiment protocol</P></LI><LI><A>Notebook</A><P>notes and progress</P></LI><LI><A>Interlab</A><P>our interlab work </P></LI><LI><A>Contributions</A><P>our contribution work</P></LI></UL><LABEL>Team</LABEL><UL><LI><A>Attributions</A><P>achievements and supports</P></LI><LI><A>Acknowledgement</A><P>the ones we sincerely appreciate</P></LI><LI><A>Collaborations</A><P>extended and intensive collaboration</P></LI></UL><LABEL>Parts</LABEL><UL><LI><A>Improve</A><P>parts that we improved</P></LI><LI><A>BasicParts</A><P>basic parts that we constructed</P></LI><LI><A>CompositeParts</A><P>composition parts that we constructed</P></LI><LI><A>PartCollection</A><P>collection of our parts</P></LI></UL><LABEL>HumanPractices</LABEL><UL><LI><A>Silver</A><P>HP for silver medal</P></LI><LI><A>Integrated&amp;Gold</A><P>HP related to our project</P></LI><LI><A>PublicEngagement</A><P>communication and popularization</P></LI><LI><A>Entrepreneurship</A><P>product and marketing</P></LI></UL><LABEL>Application</LABEL><UL><LI><A>AppliedDesign</A><P>effective convenient cheap</P></LI><LI><A>Hardware</A><P>design and measurement of our hardware</P></LI><LI><A>Demonstrate</A><P>proof of concept</P></LI></UL><LABEL>Project</LABEL><UL><LI><A>Description</A><P>brief introduction to we fit fish</P></LI><LI><A>Sensor</A><P>sensitive economic convenient</P></LI><LI><A>Degradation</A><P>efficient eco-friendly radical</P></LI><LI><A>Modelling</A><P>calculate analyze abstract optimize</P></LI></UL></DIV><DIV id="material"><DIV id="Project"><P>We sought to solve a major problem in aquaculture - high ammonia concentration in aquaculture water. High ammonia concentration is resulted from metabolite expelled from fish, which accumulated in pond water and cannot be all utilized by microorganisms in water. This may lead to widespread death of pond fish and cause damage to fish farmers. Therefore, UCAS iGEM developed an ammonia sensor to monitor water quality and a degradation machine to remove excessive ammonia. We hope to save lives of fish and provide people with more delicacies.  </P><P>We characterized three nitrogen-dependent promoters glnAp2, glnHp2 and astCp to report high nitrogen amount in aquaculture water. These three promoters are regulated by NtrC, which composes a two-component system with NtrB is activated under low-nitrogen stress. We tested the signal-noise ratio, response delay to ammonium-starvation and maximal strength of these promoters and selected the best one for our reporter RFP. By expressing RFP from such a nitrogen-regulated promoter, we constructed a nitrogen sensor particularly sensitive to ammonium concentration which emitts red fluorescence in low-ammonium environment, representing safe environment for fish.</P><P>We designed two pathways to tackle excessive ammonia in aquaculture water, one of which converts ammonium into dinitrogen gas through a NH3-NH2OH-N2H4-N2 process and the other converts ammonia into uric acid precipitation by introducing recombinant human xanthine oxidoreductase which oxidises hypoxanthine to xanthine and xanthine to uric acid into <I>E. coli</I>. Both pathways figure to remove excessive ammonia and facilitate fish to live healthily in aquaculture water.</P><P>We employed a two-step model to describe and analyze our sensor system. In the first model, the cell growth curve is fitted with Baranyi   Roberts model by applying Runge-Kutta method and PSO algorithm. This gives a rather effective description of cell growth despite of random noises. In the second model, we studied the fluorescence intensity generated by unit cells. We resolved the evolving law of our TCS system into a differential equation by Markov methods,the general behaviors of which was solved through dynamic system analysis. The total flourescence intensity is the product of the two variable that we studied, therefore we are able to validate the effectiveness of our methodologies in hardware design.</P></DIV><DIV id="Project.Description"><P>Fish provide us with abundant proteins, which is a main reason for its consumption. Yet we might not notice that in aquaculture water, ammonia concentration is always high due to fish excretion and decomposition of organics in sediment. In pond water, this is particularly deteriorated because of a lack of fresh water, which leads to continuous accumulation of excessive ammonia. High ammonia concentration harms aquaculture creatures in a variety of ways. With its high lipid solubility, it readily diffuses through cell membranes. Once it enters  the central nervous system and disrupts the homeostasis by disturbing the equilibrium between α-Ketoglutaric acid and glutamate, severe consequences can occur, including death. Accumulation of ammonia can cause lowered growth rate and reduced disease resistance as well as poor feed conversion. If exposed to continuous high-ammonia stress, fish will be more susceptible to bacterial infections, and eventually succumb to disease or starvation. Ammonia can also affect the reproduction system of aquatic organisms and ultimately affect the total output. Thus, a resolution to high ammonia threat is in urgent need.</P><P>This year, our project aims to resolve this problem by creating a sensitive ammonia sensor and an efficient ammonia utilizer to secure healthy living of fish.</P></DIV><DIV id="Project.Sensor"><H3>Design</H3><P>To facilitate reliable and convenient judgement of ammonia concentration in aquaculture water, we invented a bacterial ammonia sensor. Considering the equilibrium between ammonia and ammonium, ammonia concentration can be deduced from ammonium concentration under specific pH and temperature <A data-jumpto="projectsensor-references">[1]</A>. Therefore, we exploited endogenous nitrogen metabolism regulation system of <I>E. coli</I>.</P><P>In <I>E. coli</I>, nitrogen regulatory protein C (NtrC) and NtrB compose a two-component system where NtrC is activated by NtrB under low-nitrogen stress and activates downstream gene expression to survive such stress <A data-jumpto="projectsensor-references">[2]</A><A data-jumpto="projectsensor-references">[3]</A>. Theoretically, NtrC/B system responds to global nitrogen change in environment, nevertheless <I>E. coli</I> preferentially utilizes ammonium as its nitrogen source and is much more sensitive to ammonium concentration change compared to other nitrogen sources<A data-jumpto="projectsensor-references">[4]</A>. Thus, NtrC/B is selected as the key player in our bacterial sensor. By expressing reporter proteins from NtrC-regulated promoters, we can easily monitor ammonia concentration change in aquaculture water.</P><P>Among all NtrC-regulated promoters, three of them have been well-documented and functionally validated, including glnAp2<A data-jumpto="projectsensor-references">[5]</A>, glnHp2<A data-jumpto="projectsensor-references">[6]</A> and astCp<A data-jumpto="projectsensor-references">[7]</A> (for more information see <A data-jumpto="ucaslabnotebook">notebook</A>). We tested different properties of these promoters and decided that glnAp2 is our best choice because of its exceptional stability and high transcription level (see <A data-jumpto="projectsensor-results">results</A> for details).</P><P>We chose mRFP as our reporter since it produces high level of fluorescence and is visible under natural light. These traits are especially advantageous in hardware application. To facilitate real-time detection of aquaculture water ammonium concentration, we added a SsrA-tag at the C-terminus of mRFP to accelerate its degradation and decrease fluorescence accumulation in time and space <A data-jumpto="projectsensor-references">[8]</A>. The efficiency and ability to induce protein degradation of three SsrA-like tags (LAA, LVA and DAS) were compared (for more information see <A data-jumpto="ucaslabnotebook">notebook</A>) and DAS was selected in our system (see <A data-jumpto="projectsensor-results">results</A> for details). </P><P>To sum up, mRFP with DAS-tag is expressed from a NtrC-regulated promoter glnAp2, enabling <I>E. coli</I> to emit red fluorescence in low-nitrogen environment and quench this signal at a higher rate to facilitate real-time detection of water ammonium concentration.</P><H3>Results</H3><H4>Determining best culture conditions</H4><P>In order to select suitable promoters responding variously to different ammonium concentration, we used sfGFP as reporter to indicate transcription level, thus evaluating activities of different promoters (See <A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287001">BBa_K2287001</A>, <A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287002">BBa_K2287002,</A><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287003"> BBa_K2287003</A>). Since we were dealing with ammonium and nitrogen metabolism, traditional LB medium was much so complicated for analysis that we used basic M9 medium for bacteria culture and our first task was to determine best culture conditions. </P><P>At the beginning, we used basic M9 medium for bacteria culture (see <A data-jumpto="ucaslabnotebook">notebook</A> for composition) and applied different NH4Cl concentration to induce differentiated fluorescence intensity. We detected fluorescence intensity and OD600 using Cytation Multi-Mode reader of bacteria culture over a 10-hour range at 37℃. No fluorescence signal was detectable (the first bar in Figure 1). We hypothesized it was because basic M9 medium was short of nutrients for bacteria growth and protein expression.</P><P>To ameliorate growth conditions of these bacteria, we added 340mg vitamin B1 and 2g casamino acid per liter basic M9 medium and detected fluorescence intensity and OD600 of bacteria culture for 10 hours. This time, fluorescence signal was detected; nevertheless no apparent difference in fluorescence intensity was visible between bacteria cultured under different ammonium concentration (Figure 1). We speculated that high concentration of casamino acid, which also provides nitrogen for bacteria growth, masked the difference in ammonium concentration.</P><P>Therefore, we prepared M9 medium with same Vitamin B1 concentration but different casamino acid concentration ranging from 0.2‰ to 2‰ (see <A data-jumpto="ucaslabnotebook">notebook</A>) and tested fluorescence intensity and OD600 of bacteria culture for 10 hours. Bacteria growth curves and signal-noise ratio of fluorescence intensity under different casamino acid concentration were analyzed simultaneously. Results showed that the optimal relative casamino acid concentration is 0.3 where our bacteria not only grew properly but also exhibited apparently different responses to different ammonium concentrations in medium (Figure 1,2). Eventually, we fixed our medium composition for characterization of different NtrC-regulated promoters (see <A data-jumpto="ucaslabnotebook">notebook</A>). This modified M9 medium was named “working medium”</P><H4>Selection and characterization of promoters</H4><P>To evaluate signal-noise ratio, response delay and strength of the three promoters glnAp2, glnHp2 and astCp, we analyzed bacteria growth curves and dynamics of fluorescence intensity. Results showed that all three promoters responded to low-ammonium environment at similar rates while glnAp2 demonstrated a significantly higher signal-noise ratio and signal strength compared to the other two promoters (Figure 3,4,5). Thereby, we decided to use glnAp2 in our working system.</P><P>Next, we reduced ammonium addition in our working medium (ranging from 0 to 0.02-fold of standard M9 medium) to study the relationship between ammonium concentration and response time of glnAp2. Consistent with our expectation, the lower the ammonium concentration, the shorter the response time (Figure 6,7).</P><P>To test the interference of other nitrogenous substances in ammonium detection, we collected and analyzed water sample from local farming ponds. We found that NO3- is the main nitrogenous substance other than ammonium in pond water with a concentration of approximately 60uM. Therefore, we added potassium nitrate of different concentration into working medium with 0.02-fold of standard ammonium concentration and detected fluorescence intensity and OD600 of bacteria culture. Results showed that 100uM nitrate had slight influence on response of glnAp2 to ammonium concentration, suggesting that glnAp2 is suitable for ammonia detection in aquaculture water (Figure 8).</P><H4>Selecting SsrA-like tags</H4><P>Three SsrA-like tags (LAA, LVA and DAS) were added to the C-terminus of mRFP downstream the promoter glnAp2. By analyzing the dynamics of fluorescence intensity in working medium, we compared protein-degradation effects of these tags. We found that mRFP with LAA- and LVA-tag were unable to emit detectable fluorescence, indicating a high degradation rate, while mRFP with DAS-tag still produced red fluorescence, which in turn suggested the inefficiency of DAS-tag (Figure 9). In the meantime, the response time mRFP with DAS-tag reveals a remarkable difference, indicating a reliable resolution, while the LAA and LVA do not (Figure 10).</P><H4>Demonstrating function of our sensor system</H4><P>We constructed a sensor circuit where a reporter mRFP with a moderate-degradation DAS-tag was expressed from an NtrC-regulated promoter glnAp2, and tested its function in laboratory.</P><P>In laboratory, we used Cytation3 Multi-Mode reader to obtain the kinetic curves. Obviously, the degradation tag leads to a remarkable drop in the maximum RFU, which makes the fluorescence signal harder to be detected by normal photoresistors (fig.9). This problem can be solved by employed other photoelectric sensors with higher sensitivity, or we can give up the idea of involving degradation tag, as the circuits without this tag has displayed reliable performances. Moreover, after being added a degradation tag, the system becomes more unstable, which is revealed from the decreased signal-noise ratio(fig 11,12). With help from other iGEM teams in China who volunteered to validate our circuit in their own lab, we obtained a large amount of data for modelling and modifying our system (see collaboration).</P><H3>References</H3><P>[1]Anthonisen A, Loehr R, Prakasam T, et al. Inhibition of nitrification by ammonia and nitrous-Acid [J]. Journal Water Pollution Control Federation, 1976,48(5):835-852.</P><P>[2] Li, W., &amp; Lu, C. D. (2007). Regulation of carbon and nitrogen utilization by CbrAB and NtrBC two-component systems in Pseudomonas aeruginosa. Journal of bacteriology, 189(15), 5413-5420.</P><P>[3] Huo, Yi‐Xin, et al. &quot;Protein‐induced DNA bending clarifies the architectural organization of the σ54‐dependent glnAp2 promoter.&quot; Molecular microbiology 59.1 (2006): 168-180.</P><P>[4] Wang, Jilong, et al. &quot;Deciphering the principles of bacterial nitrogen dietary preferences: a strategy for nutrient containment.&quot; MBio 7.4 (2016): e00792-16.</P><P>[5] Reitzer, Lawrence J., and Boris Magasanik. &quot;Transcription of glnA in <I>E. coli</I> is stimulated by activator bound to sites far from the promoter.&quot; Cell 45.6 (1986): 785-792.</P><P>[6] Claverie-Martin, Felix, and Boris Magasanik. &quot;Role of integration host factor in the regulation of the glnHp2 promoter of Escherichia coli.&quot; Proceedings of the National Academy of Sciences 88.5 (1991): 1631-1635.</P><P>[7] Kiupakis, Alexandros K., and Larry Reitzer. &quot;ArgR-independent induction and ArgR-dependent superinduction of the astCADBE operon in Escherichia coli.&quot; Journal of bacteriology 184.11 (2002): 2940-2950.</P><P>[8] McGinness, K. E., Baker, T. A., &amp; Sauer, R. T. (2006). Engineering controllable protein degradation. Molecular cell, 22(5), 701-707.</P></DIV><DIV id="Project.Degradation"><H3>Gas Group-Design</H3><P>We sought to convert ammonia into dinitrogen gas which can be directly released into air. This is our “Gas Group”. Our inspiration came from ANaerobic AMMonium Oxidation bacteria (Anammox) which converts nitrite and ammonium (NH4+) into dinitrogen gas (N2) in nature. However, Anammox is sensitive to oxygen, which makes it impossible to apply it in aquaculture. To tackle this problem, we designed a novel aerobic pathway combining enzymes from ammonia oxidizing bacteria (AOB) and Anammox. In this novel pathway, the ammonia monooxygenase (AmoA) from AOB converts ammonium (NH4+) into hydroxylamine(NH2OH), then the alpha subunit of hydrazine synthase (HZSα) uses NH2OH and NH4+ as substrates to synthesize hydrazine (N2H4), and finally an oxidase oxidizes N2H4 into N2 (Figure D1). In this case, soluble nitrogen is turned into insoluble dinitrogen gas and fish no longer had to worry about ammonia threat. (For more details about anammox metabolism, see <A data-jumpto="ucaslabnotebook">Notebook</A>.)</P><P>To facilitate this NH4+ - NH2OH - N2H4 - N2 pathway, we have to express ammonia monooxygenase (AmoA) and the alpha subunit of hydrazine synthase (HZSα) from AOB in <I>E. coli</I>, and select an enzyme which is capable of catalyzing the oxidation of hydrazine into dinitrogen gas by oxygen with high efficiency and specificity.</P><H3>Gas Group-Results</H3><H4>From Hydrazine to dinitrogen gas</H4><P>We first tackled the final step of our pathway, since the substrate hydrazine is highly toxic to bacteria and the rate of this reaction directly determines the efficiency of our system. Though no successful expression of the original hydrazine dehydrase (HDH, for more information, see <A data-jumpto="ucaslabnotebook">Notebook</A>) from Anammox has been reported yet, that it is a multiheme protein is clear. Therefore, we decided to try heme-containing proteins and other metal-containing proteins available in our laboratory. We tested N2H4 oxidation activity of 6 metal-containing proteins, including Degrado, 3His, Pf ferritin, DPS, CueO, and Urease (for detailed protein information, see <A data-jumpto="ucaslabnotebook">Notebook</A>). Purified and assembled (for Pf ferritin and DPS, detailed assembly protocol can be found in <A data-jumpto="ucaslabmethods">Method</A>) proteins were incubated overnight with 100mM hydrazine, respectively, and hydrazine leftover was measured by adding 4-Dimethylaminobenzaldehyde and HCl to the system (for detailed protocol, see <A data-jumpto="ucaslabmethods">Method</A>). Sample N2H4·H2SO4 exhibited apparent color change and UV absorption at 454nm after such reaction (Figure 1). Significant decrease of hydrazine concentration was observed after overnight incubation with all 6 proteins. However, since CuCl2 was added in culture medium of CueO-expressing bacteria to help correct folding, and Cu2+ alone exhibited considerable catalytic activity, it was hard to measure the catalytic activity of CueO. As for urease, additional experiments showed that it was not as efficient as Pf ferritin and DPS (Figure 3). Therefore, we stopped with these two enzymes and focused on the other 4 proteins.</P>https://static.igem.org/mediawiki/2017/7/7d/T--UCAS--Degradation_Fig1.png
		<P>We tested the catalytic kinetics of 3His, Degrado, Pf ferritin and DPS and found that the most efficient one was Pf ferritin (Figure 4). Unfortunately, even though Pf ferritin was already the most effcicient candidate for Hydrazine Dehydrogenase (HDH), it took about an hour to reduce the concentration of hydrazine from 100uM to 50Um, which was far from efficient in general sense. In future, we will introduce point mutations into these proteins to improve their catalytic efficiency, find more multiheme proteins and test their N2H4-oxidation activity, and try to express codon-optimized HDH in <I>E. coli</I>.</P><H4>From ammonia(ammonium) to hydroxylamine</H4><P>We cast our sight on AmoA, a membrane protein in Anammox, which was anticipated to transform ammonia into hydroxylamine. <I>E. coli</I> bearing AmoA was incubated overnight with 1mM NH4Cl at 37°C in a shaker. Since AmoA is a copper-containing protein, 100uM CuCl2 was added to the medium. After centrifuging at 13,000 rpm for 10 minutes, the supernatant was mixed with 8-Hydroxyquinoline and K2CO3 for hydroxylamine detection and tested its UV absorbance at 700nm (see <A data-jumpto="ucaslabnotebook">notebook</A> for detailed protocol). Compared to sample NH2OH (Figure 5), no absorption was detected in culture supernatant (figure 6). Apart from the possibility that no NH2OH was produced, it was also possible that the produced NH2OH was exploited by <I>E. coli</I>. Since NH4Cl is natural nitrogen source for <I>E. coli</I>, decrease of NH4+ concentration does not necessarily indicate AmoA activity nor NH2OH production. Therefore, we decided to use a coupling assay to test the activity of AmoA which depends on the activity of the alpha subunit of HZS (HZSα).</P><H4> From hydroxylamine to hydrazine</H4><P>A senior in the laboratory helped us expressing the alpha subunit of HZS, but no expression was detected.</P><H3>Solid Group-Design</H3><P>We came up with a second solution of converting ammonia nitrogen into insoluble precipitation– uric acid, which ends up in sediment. This is our “solid group”. Uric acid is produced in human’s purine catabolism process with xanthine as its precursor. This reaction is catalyzed by Xanthine Oxidoreductase (XOR) in human and this enzyme also catalyzes the oxidation of hypoxanthine into xanthine. <I>E. coli</I>, however, lacks the ability to transfer purines into uric acid because of lack of this enzyme. Therefore, we hypothesized that introducing XOR into <I>E. coli</I> may enable it to produce uric acid (Figure D2). Furthermore, to elevate the efficiency at which purines are transformed into uric acid, we over expressed key enzymes in de novo purine biosynthesis pathway of <I>E. coli</I> to “pull” the reaction towards uric acid production. In this way, excessive nitrogen, especially ammonia nitrogen, is transferred into nitrogen in sediments and fish don’t have to bear the high concentration of ammonia.</P><H3>Solid Group-Results</H3><DIV data-special="namecard"><H4>Key achievements</H4><P>Successful expression of functional recombinant human xanthine oxidoreductase (rhXOR) in <I>E. coli</I> and optimization of expression conditions.</P><P>Successful production of uric acid in <I>E. coli</I> with extra xanthine supply.</P><P>Increase of uric acid production by over-expressing key enzymes in de novo purine biosynthesis pathway.</P></DIV><H4>Expressing XOR in <I>E. coli</I></H4><P>We constructed the plasmid bearing recombinant human xanthine oxidoreductase (rhXOR, shorted as XOR) on a pETDuet-1 vector and transformed it into <I>E. coli</I>, thus obtaining <I>E. coli</I><SUP>XOR</SUP>. We first cultured <I>E. coli</I><SUP>XOR</SUP> in regular LB medium at 37℃ at 200 rpm for 12-16 hours, then added IPTG to induce expression of XOR and cultured at 30℃ at 200 rpm overnight. Then XOR activity was tested by incubating bacteria with OD600 of 100 with xanthine for 6 hours at 1100 rpm and detecting uric acid with HPLC-MS (for detailed assay, see <A data-jumpto="ucaslabnotebook">NOTEBOOK</A>). In regular LB medium, no uric acid production was detected (Figure 7).</P><P>Since uric acid is reported to cause cell death, we speculated that expression of XOR in large quantity was baneful and thus expression was suppressed. Therefore, we removed IPTG and counted on leakage expression of XOR. XOR activity was tested following the same procedure, but still no uric acid production was detected (figure 8).</P><P>When we learnt that XOR contains molybdopterin cofactors as active catalytic site, we added 1mM sodium molybdate into regular LB medium and cultured <I>E. coli</I><SUP>XOR</SUP> under same conditions as before. IPTG was added in experiment group while removed in control group. We used HPLC-MS to detect uric acid and integrated area was calculated and compared to decide the relative amount of uric acid. Since peaks of xanthine and uric acid were sometimes overlapped in UV absorbance plot (Figure 9), we integrated the MS plot. Results showed that compared to bacteria not bearing XOR (Control in Figure 10), <I>E. coli</I><SUP>XOR</SUP> cultured with 1mM sodium had the exceptional ability to transfer external xanthine into uric acid. The function of IPTG differed regarding plasmid vectors. Control bacteria bore a pETDuet-1 plasmid same as pET-XOR, and gained resistance to ampicillin, while RSF-XOR bore a pRSFDuet-1 plasmid and was endowed a resistance to kanamycin. In pET-XOR bearing bacteria, addition of IPTG significantly inhibited its ability to oxidize xanthine, while in pRSF-XOR bearing bacteria, IPTG seemed to promote expression and activity of XOR. This phenomena still needs further research.</P><H4>Harnessing nitrogen to produce purines</H4><P>Though we detected uric acid production using <I>E. coli</I><SUP>XOR</SUP> cultured in LB with sodium molybdate (1mM), it was achieved by adding extra xanthine. In order to produce more uric acid without extra xanthine supply, we wanted our bacteria to harness more nitrogen and store it in purines, the precursors of uric acid. In de novo purine biosynthesis pathway of <I>E. coli</I>, AMP (Adenosine monophosphate) and GMP (Guanosine monophosphate) share a common precursor – IMP (inosine monophosphate), and there are 11 steps to produce IMP from R5P (Figure 11). We over-expressed the two rate-limiting enzymes phosphoribosylpyrophosphate synthase (encoded by prs gene, denoted as Prs in below) and glutamine phosphoribosylpyrophosphate amidotransferase (encoded by purF gene, denoted as PurF in below), along with other 4 enzymes involved in 4 ATP-consuming reactions (Figure 11). </P><P>Genes of all 6 enzymes but phosphoribosylformylglycinamidine synthase (encoded by purL) were successfully cloned and transformed into <I>E. coli</I>. These enzymes and selective combinations of them were co-transformed with XOR into <I>E. coli</I> and uric acid production were measured. Considering the feedback inhibition of phosphoribosylpyrophosphate amidotransferase (encoded by purF) imposed by AMP and GMP, we mutated the 326th amino acid from K to Q (designated as K326Q), to inhibit binding of AMP and GMP. PurFK326Q was also co-transformed with XOR to test uric acid production ability. </P><P>We first tested their uric acid production activity by providing extra xanthine, and found that there was a significant increase in uric acid compared to control <I>E. coli</I><SUP>XOR</SUP> except for co-expression of PurC and Prs(Figure 12). Particularly, co-expression of PurC and PurD and co-expression of purM and purD demonstrated largest increase among all over expressed enzymes. Then we provided no extra xanthine and measured uric acid production following a similar procedure. Results showed that over-expression of enzymes involved in purine biosynthesis did elevate uric acid production compared to <I>E. coli</I><SUP>XOR</SUP> (Figure 13). Interestingly, this time the combination of purM and purD exhibited lowest oxidation activity and we will design more experiments to explore the reason behind. Notably, comparing uric acid produced without extra xanthine by these bacteria with that of <I>E. coli</I><SUP>XOR</SUP> with extra xanthine, bacteria co-transformed with Prs, PurC and PurD, Prs and PurD, PurF and Prs still showed higher oxidation activity (Figure 14). These results confirmed our idea that by over-expressing related enzymes, more purines can be produced for uric acid synthesis. Our future work is to co-express other enzymes and calibrate the ratio of different enzymes to achieve maximum uric acid production.</P><H3>References</H3><H4>Gas Group:</H4><P>[1]Dietl, A., et al. (2015). &quot;The inner workings of the hydrazine synthase multiprotein complex.&quot; Nature 527(7578): 394-397.</P><P>[2]Jetten, M. S., et al. (2009). &quot;Biochemistry and molecular biology of anammox bacteria.&quot; Critical Reviews in Biochemistry and Molecular Biology 44(2-3): 65-84.</P><P>[3]Kartal, B. and J. T. Keltjens (2016). &quot;Anammox Biochemistry: a Tale of Heme c Proteins.&quot; Trends in Biochemical Sciences 41(12): 998-1011.</P><P>[4]Kartal, B., et al. (2011). &quot;Molecular mechanism of anaerobic ammonium oxidation.&quot; Nature 479(7371): 127-130.</P><P>[5]Strous, M., et al. (2006). &quot;Deciphering the evolution and metabolism of an anammox bacterium from a community genome.&quot; Nature 440(7085): 790-794.</P><H4>Solid Group:</H4><P>[6]Ferreira Antunes, M., et al. (2016). &quot;Human xanthine oxidase recombinant in <I>E. coli</I>: A whole cell catalyst for preparative drug metabolite synthesis.&quot; Journal of Biotechnology 235: 3-10.</P><P>[7]Shimaoka, M., et al. (2007). &quot;Effect of amplification of desensitized purF and prs on inosine accumulation in Escherichia coli.&quot; Journal of Bioscience and Bioengineering 103.</P><P>[8]Zhou, G., et al. (1993). &quot;Identification of sites for feedback regulation of glutamine 5-phosphoribosylpyrophosphate amidotransferase by nucleotides and relationship to residues important for catalysis.&quot; Journal of Biological Chemistry 268(14): 10471-10481.</P><P>[9]Zhou, G., et al. (1994). &quot;Binding of purine nucleotides to two regulatory sites results in synergistic feedback inhibition of glutamine 5-phosphoribosylpyrophosphate amidotransferase.&quot; Journal of Biological Chemistry 269.</P></DIV><DIV id="Project.Modelling"><H3>Overview</H3><P>Modeling is an art of description. Grounded on experimental data and theoretical principals, the modeling practice combines both of them and expectedly will produce feedbacks unnoticeable by direct data analysis (though almost every modeling starts from this process).</P><P>The goal of our project is to build engineering bacteria with the ability to detect ammonium in water. By data analysis, we select the delay time (detailed definition is explained in the RESULT PART) as the signal to report the concentration of ammonium. To quantify the effect,<STRONG>we build models of bacteria growth under different concentration of ammonium. Also, we apply analysis method in dynamic systems to mathematically explain the promoters’ function feature: sudden transition from close to open at the threshold ammonium concentration. </STRONG>This proves that choice for delay are not only practically practicable, as is supported by experiment data, but also theoretically reasonable.</P><P>We divide our model into 2 sections. In <STRONG>part 1</STRONG>, we employ numerical algorithm to describe bacteria growth with accuracy and perform a deep-going analysis, using statistical methods.<STRONG> Part 2</STRONG>, we abstract the mechanism of these promoter into a TCS (Two Components System), which helps to get the evolution equation of the system. With methods from dynamic systems researches, we successfully explain the mechanism of the promoters we applied and this might be universal for common TCSs.</P><P>Bacteria growth is a fundamental issue in synthetic biology and the TCS is commonly used in synthetic biology.<STRONG> Our model is conducive to the core of the project, finding a suitable signal, and  potentially useful for modeling of other projects.</STRONG></P><H3>Dynamics of Free Growth</H3><P>To describe growth of bacteria, many growth curves have been proposed. Under substrate-sufficient conditions, the Logistic curve, Gompertz equation, polynomial fitting, Stannard model, and their modifications[1] are widely applied. Meanwhile, Monod equation[2], Baranyi &amp; Roberts model and many computational models[3][4] are more suitable in substrate-limited conditions.</P><P>As the promoter used in our project only functions in conditions with low concentration of ammonium, we adopt Baranyi &amp; Roberts model which is suitable to explain substrate-limited condition. Its general form performs as below:</P><P>Parameter v(t) reflects the physiological state of the cells, which is determined by initial conditions and is independent of the history of bacteria and previous environments [6]. It can be assumed that the derivative of v is a constant under invariant conditions. Therefore v(t) can be solved: </P><P>Optical density(OD) we measure in experiments is proportional to the cell density. Then the growth rate equation can be expressed as followings.</P><P>The parameters of equation (3) cannot be fitted by simple regression, so we develop another approach. For any given parameters, we use Runge-Kutta method [7] to calculate how OD evolve with time after the initial value, and use Particle Swarm Optimization(PSO) algorithm [8] to find the suitable parameters that minimize the standard deviation between evolution and measurement. </P><P>	The fitting results are shown in Fig. 1. Dots are experimental values, and the curve is the fitted results. As is shown by the graph, this model performs well in predicting the bacteria growth in our program.</P><P>It is fair to consider the ammonium consuming velocity is roughly in proportion to the bacteria’s amount. Thus, the growth model could roughly predict the ammonium quantity change of the system.</P><H3>Mechanism of Promotors</H3><P>The promotors we used in our project, like glnAp2, glnHp and astC, have the same feature that it does not start the transcription process until the environmental ammonium concentration reaches a low threshold. Researchers have proposed the biological mechanism of these promotors [9]. The general process is shown in the graph below.</P><P>As is shown in the graph, the promoters are singularly activated by NtrC. Thus, it is fair to assume the promoter strength is proportion to the concentration of NtrC. Previous research indicates that factors from cellular amino acid metabolism like PII would bind to NtrB’s active site and prohibit the further reaction.[11] Researchers also found lower environmental ammonium concentration will result in an increase of NtrC-Pi. We simplify this process as the graph shows below.</P><P>As is shown in the graph, this system can be reasonably considered as a TCS. The amount of NtrC-Pi (Conc) and NtrB-Pi (Conb) are two key parameters of this system. Four factors influence Conc and Conb: NtrB helps NtrC’s phosphorylation, NtrC starts the production of NtrB, Conb reduction caused by PII and constant NtrC production. Also, it should not be neglected that the inherent degradation of NtrC and Ntrb is in proportion to their own concentration.</P><P>Thus, the evolution equation of this system can be written as:</P><P>Where y<SUB>0</SUB> can be expressed as:</P><P>The four parameters b, c, y0, x0 invited here respectively represent the velocity of the four processes mentioned before. To be emphasized,y<SUB>0</SUB> describing the change of Conb caused by PII. Parameters a, d is the coefficient of proportionality between Conc or Conb and its depredating velocity. This is a differential equation and the evolution process could be treated as a Markov Process*. </P><P>Here we apply methods from <STRONG>dynamic system</STRONG> to conduct further analysis: </P><P>First, Consider evolution matrix A, calculate the eigenvalue λ of A</P><P>Solve this equation, get</P><P>and the eigenvector is</P><P>Since parameters a, d are relatively small compared with c, b and they are both negative, it can be easily spotted thatλ<SUB>1</SUB>&gt;0,λ<SUB>2</SUB>&lt;0 and the ordinate of the vector I<SUB>1</SUB>&gt;0 while that of I<SUB>2</SUB>&lt;0. If we temporarily put aside the parameters x0, y0 and the definition domain of Conc and Conb, this system could be explained and predicted by the phase flow graph shows below.</P><P>Every curve in the graph is an orbit of the system with different initial Conc and Conb. This means that the system with certain initial condition evolves alongside the curve that passes through that point. Thus, with any certain set of parameters, we can determine how the system evolves according to the graph. </P><P>So how to draw this graph? How to calculate the function of the curves in the graph? If we consider a system with the evolving law written as </P><P>Rewrite r(t) using the eigenvector I1, I2, defined vector X=(Conc, Conb)</P><P>Take this back into the differential equation, after simplify, get</P><P>Solve this set of equation, where C1, C2 is determined by the initial condition of the system.</P><P>Therefore, any system with specific initial condition could be settled. If we clear t out of the set of equation, we get the functions of the curves in the graph above.</P><P>Now consider what have been put aside in previous discussion.</P><P>Conc and Conb have their definition domains, let them respectively be [0,Conc<SUB>Max</SUB>]and [0,Conb<SUB>Max</SUB>]. This gives a limit to the possible state the systems can evolve. In other word, it limits the system inside a square frame. </P><P>As is supported by the experimental data, at the beginning stage of experiment with high environmental ammonium concentration, the gene circuit remains closed. Thus, we can consider Conc is small at this stage. The initial point of the system falls above L1. </P><P>From the graph, we can easily find the system confines Conc at a low level at the beginning stage of experiment. This explained why there is no GFP produced at the beginning of experiment.</P><P>On the other side, parameters x0, y0 cause the whole system to perform a spatial translation. In any timepoint, x0, y0 give the velocity of the motion. According to y0’s definition, the translation in Conb direction at any time always equals y0 multiplies the change of PII by that time. y0 is negative and significantly bigger than x0.</P><P>As the experiment proceeds, the environmental ammonium concentration decreases and the concentration of PII decreases together. The phase flow graph of our system gradually rises up and conduct a constant small translation in Conb direction. There is a special time when the concentration of PII reaches a threshold that the point of our system falls below L1 and L2. Therefore, the system evolves to maximize the Conc. That is when the promoter begins to function and this is the delay time we used as signal in our project. </P><P>After that point, the graph keeps rising, the Conc consequently keeps increasing. Thus, the promoter strength keeps increases over time and the reaction goes on for certain time.</P><P>In a nutshell, we simplify the gene circuits as a TCS and furtherly construct a differential equation describing the evolving law of this system. To solve this equation, we apply methods from studies of dynamic system. From detailed analysis, we prove that the delay time is a principally reasonable characteristic signal for this reaction. With experimental data supported, the delay time could be used as a special mark for both the promoter and the environmental ammonium concentration.</P><H3>Conclusion</H3><P>To conclude, in part 1 we successfully model the bacteria growth in our project. We use Baranyi &amp; Roberts model and we apply Runge-Kutta method and PSO algorithm to get the parameters using the experimental data. This gives a rough prediction for the delay time signal.</P><P>In part 2, we simplify the gene circuits as a TCS and furtherly construct a differential equation describing the evolving law of this system. To solve this equation, we apply methods from studies of dynamic system. From detailed analysis, we prove that the delay time is a principally reasonable characteristic signal for this reaction. </P><P>With all these analysis and experimental data supported, the final conclusion of the modelling is that the delay time could be used as a special mark for both the promoter and the environmental ammonium concentration.</P><H3>References</H3><P>[1] Liu, Y. Overview of some theoretical approaches for derivation of the Monod equation, Applied Microbiology and Biotechnology. 2007;73(6):1241–1250. </P><P>[2] Bruce R. Levin, Frank M. Stewart &amp; Lin Chao, Resource-Limited Growth, Competition, And Predation: A Model And Experimental Studies With Bacteria And Bacteriophage, The American Naturalist. 1977; Vol 111,No.977. </P><P>[3] M. L. Shuler, S. Leung, and C. C. Dick, Mathematical Model For The Growth Of A Single Bacterial Cell, Annals of the New York Academy of Sciences, 1979, 326(326):35–52. </P><P>[4] M. H. Zwietering, I. Jongenburger, F. M. Rombouts, And K. Van 'T Riet. Modeling of the Bacterial Growth Curve. Applied and Environmental Microbiology. June 1990, p. 1875-1881, Vol. 56, No. 6. </P><P>[5] J. Baranyi, T. A. Roberts, A dynamic approach to predicting bacterial growth in food, International Journal of Food Microbiology, 23(1994) 227-294. </P><P>[6] Vadasz P, Vadasz A S. Biological implications from an autonomous version of Baranyi and Roberts growth model. International Journal of Food Microbiology, 2007, 114(3):357-65. </P><P>[7] Timothy Sauer. Numerical Analysis (Second Edition). New York: Pearson Education, 2012. Chapter 5. </P><P>[8] Kennedy J. Particle Swarm Optimization[M]. Springer US, 2011. </P><P>[9] Klein, Adam H., et al. &quot;The intracellular concentration of acetyl phosphate in Escherichia coli is sufficient for direct phosphorylation of two-component response regulators.&quot; Journal of bacteriology 189.15 (2007): 5574-5581.</P><P>[10] Huo, Yi‐Xin, et al. &quot;Protein‐induced DNA bending clarifies the architectural organization of the σ54‐dependent glnAp2 promoter.&quot; Molecular microbiology 59.1 (2006): 168-180.</P><P>[11] Jiang, Peng, et al. &quot;Functional dissection of the dimerization and enzymatic activities of Escherichia coli nitrogen regulator II and their regulation by the PII protein.&quot; Biochemistry 39.44 (2000): 13433-13449.</P><P>* Roughly speaking, a process satisfies the Markov property if one can make predictions for the future of the process based solely on its present state just as well as one could knowing the process's full history, hence independently from such history.</P></DIV><DIV id="Application"><P>Given the fact that the problem of excessive ammonia in the fish pond need be solved while the current detecting and degrading system is imperfect, we designed our project and hardware to solve the problem in a more efficient, convenient and economic way. We used engineered <I>E. coli</I> to detect and degrade ammonia, considerably reducing the cost. We also designed a hardware to convert optical signal transmit by bacteria into warning message for fish farmers to read.</P><P>We have implement our idea of detecting and degrading ammonia with our engineered <I>E. coli</I>. Data and graphs in this page demonstrate what we have achieved in our project. Three promotors we chose to detect ammonia were proved to work well and we successfully detected considerable amount of uric acid in the culture medium our bacteria bred in. we also improved our project by adding ssrA tag to the RFP our detecting bacteria express and transform enzymes into degrading bacteria that enhance the produce of uric acid.</P><P>For convenient detection of ammonia nitrogen in fishponds, a device was designed based upon our sensor bacteria, which express RFP in low ammonia concentration water. The device is able to provide relatively accurate information while the cost is lower than most ammonia detection equipment on the market. The user only needs to fix our equipment on a buoy or a platform, and to purify the water when receiving the text message that the device automatically sends to him/her.</P></DIV><DIV id="Application.AppliedDesign"><H3>Demand</H3><P>As we have demonstrated, excessive ammonia in the fish pond is one of the main problems that limited the development of fresh water aquaculture. Excessive ammonia will lead to the deterioration of water quality and effect the taste of the fish. It will also lead to the outbreak of fish diseases and cause fish death, heavily decreasing the yield of the fish pond. If we cannot solve the problem of excessive ammonia, it will be problematic to increase the breeding density and to improve the fish quality, thus effecting the income of fish farmers.</P><P>So essential as the problem of excessive ammonia is, there are certain solutions fish farmers will apply empirically. During the high-risk period, fish farmers will frequently visit their ponds and observe the water quality and fish conditions. Once there is any manifestation indicting excessive ammonia or fish diseases, they will take emergency measures such as disinfecting the pond and changing pond water.</P><P>Given the fact that fish farmers are judging by virtue of experience, there will always be deviation. When any manifestation occurs, excessive ammonia has actually erupted already. Although there are some products of ammonia detection on the market, they are either too expensive for fish farmers to afford or inconvenient to use (See more details on products comparison in <A data-jumpto="humanpracticesentrepreneurship">entrepreneurship</A>). In addition, they measures they usually take to deal with excessive ammonia are either too expensive or too troublesome. According to our research (See more details on Integrated HP), big aquaculture companies usually choose running water to breed fish, which, in places lack of surface water, will cost up to 200 thousand yuan per month and waste thousands tons of underground water. Self-employed fish pond tend to choose chemical agents to remove ammonia, but this method is not efficient enough and is not eco-friendly.</P><P>Therefore, an effective, convenient and cheap way to detect and degrade ammonia is in strong demand of fish farmers.</P><H3>Design</H3><P>Based on the principle of efficiency, convenience and economy, we designed a device to detect the ammonia concentration in the fish pond with the application of our engineered bacteria and introduced another strain of engineered <I>E. coli</I> to covert ammonia in the pond water to nitrogen or uric acid precipitation. </P><P>By introducing bacteria into the sensing system can reduce the cost of detection components. Our engineered bacteria will express RFP when the ammonia concentration is below the threshold. When the ammonia concentration exceeds the safe value, our bacteria will stop expressing RFP. With the help of our engineered bacteria, we can convert the concentration of ammonia nitrogen into a more readable optical signal.</P><P>After transmitting optical signal, our device should be able to convert optical signal into electrical signal, and then make the electrical signal readable to fish farmers. We use photoresistance to achieve the first step and code a program to accomplish the second step. When the concentration of ammonia is above the threshold, fish farmers will receive a message on their cellphone sent via Bluetooth to warn them (See more details on in <A data-jumpto="applicationshardware">hardware</A>).</P><P>As for ammonia degradation, we planned to express three different enzymes form different organisms to achieve the aim of convert ammonia into nitrogen gas. </P><H3>Adjustment</H3><P>Nothing can be accomplished at one stroke. After we completed the design of our device, we still need to customize it to the practical conditions.</P><P>The first thing we need to know is the composition of fish pond water. We visited some fish ponds nearby and collected water sample from the ponds. We then analyzed the composition of our water sample and adjust the composition of our culture medium to stimulate the pond water (See more details on water sampling in Integrated HP). We also asked other iGEM Teams to help us collected water samples in their surroundings (See more details in collaboration).</P><P>The second thing we need to do is to engineer our bacteria to make it express RFP under certain concentration of ammonia. To achieve this, we designed and tested three different promotors and chose the best one among them (See more details on promotors in sensor).</P><P>Last but not least, we did market research and contacted with our potential customers, fish farmers, to optimize our device according to the demand (See more details in <A data-jumpto="humanpracticesintergratedgold">integrated HP</A> and <A data-jumpto="humanpracticesentrepreneurship">entrepreneurship</A>).</P></DIV><DIV id="Application.Hardware"><P>For convenient detection of ammonia nitrogen in fishponds, a device was designed based upon our sensor bacteria, which express RFP in low ammonia concentration water. The device is able to provide relatively accurate information while the cost is lower than most ammonia detection equipment on the market. The user only needs to fix our equipment on a buoy or a platform, and to purify the water when receiving the text message that the device automatically sends to him/her.</P><H3>Design</H3><H4>Immobilization</H4><P>As a mature technology which has fully developed in the past 100 years, the immobilization of bacteria at present is widely applied in various fields ranging from pharmaceutical industry to sewerage system. Embedding method is the most common one to realize bacterial immobilization. Immobilizing living bacteria in carrier like sodium alginate, cellulose and agar, etc., increases the density and survival time of them, as well as reduces the flow of bacteria into natural environment. Our team produces immobilized <I>E. coli</I> by embedding the sensor bacteria in polyvinyl alcohol/sodium alginate (PVA/SA) beads.(UCAS-APPLICATION-DESIGN-1)</P><P>We planned to breed our immobilized bacteria in fish pond water, but as our analysis of fish pond water shows, in nutrition in fish pond water is limited and cannot meet the demand of our engineered bacteria. Therefore, we changed our plan and decided to breed bacteria in our nitrogen limited M9 culture medium, mixed with fish pond water. Nevertheless, our technique in immobilization might have further use since we are still seeking for a better solution.</P><H4>Pumping System</H4><P>The equipment contains a detection system, a pumping system, a control system and a battery. In order to properly inject water, a peristaltic pump is employed to inject 1.5mL pool water into the detecting cell, and another pump will pump it out once the detection process finishes. The second pump delivers the water to a cup and, considering the potential danger of escaping bacteria, a microporous filter at the top of the cup.</P><H4>Detection System</H4><P>Detection part mainly consists of a detecting cell, a LED and a photoresistor. In our first plan, we planned to applied bacterial immobilizing technique. Firstly the polyvinyl alcohol/sodium alginate (PVA/SA) beads with immobilized <I>E. coli</I> soak in the water in the detecting cell for approximately 5 hours. Thus the promoter begins to respond to the concentration of nitrogen, and then RFP gene is expressed if the concentration is below the threshold. A 585nm-595nm LED is applied to excite red fluorescence and a photoresistor are used to detect the fluorescence. The detecting cell is the only consumable item in the whole equipment. In our second plan, we will breed our bacteria in nitrogen limited M9 culture medium for approximately 5 hours, then inject the fish pond water to mix with the culture medium. It will take 2-3 hours before the GFP being expressed and detected.</P><H4>Control System</H4><P>To ensure that the hardware perform properly, a control system was designed. We used a small circuit board to control the relays and the Bluetooth module. The relays’ opening or closure makes different parts in the equipment start or stop working at proper time. And the Bluetooth module sends the information of ammonia concentration to the user’s mobile phone during the detection process.</P><H4>Safety</H4><P>In our first plan, we install a valve and, considering the potential danger of escaping bacteria, a microporous filter at the bottom of the detection cell. Once the detecting process finishes, the valve opens and the water sample flow into the pool through the microporous filter.</P><P>In our second plan, we make the breeding and detection cell separated from other parts of the device, and can be pulled out like a draw. We can then pour out the bacteria solution and autoclave it for sterilization. As for the detection cell and pump tube, the only component in pump that will contact with the bacteria we choose sodium hypochlorite (NaClO) to sterilize.</P><H4>Cost</H4><P>In our researches we find that most fishermen are aware of the serious consequences of excessive ammonia. Yet instead of monitoring the water precisely, they always decide whether to clean the pool merely by experience. The main reason, we believe, is that ammonia detection machines are rather expansive. Many of them are priced at over 10000 yuan each, and even the cheapest one could cost more than 2000. Our device, by contrast, is much more available, which costs approximately 300 yuan each.</P><H3>Hardware</H3><P>3D printing technique was applied in constructing the detection system. We employed a 3D printer, FlashPrint Creator Pro, and produced some components with ABS (acrylonitrile butadiene styrene) material.</P><H4>Pumping System</H4><P>For pumping water in and out the detecting cell, we bought two Kamoer peristaltic pumps NKP-DE-S04 (6V, 5W) which weigh 110g, with the silicone pump pipe (service life≥200h) with internal and external diameter of 1.0/3.3mm. The pump can transports more than 11mL water per minute. Each pump has a pipe inserted into the hole on the lid of detecting cell. One for pumping pool water into the cell, and another for pumping it from cell to the cup. </P><H4>Detection System</H4><P>We printed a frame of the detecting cell, and fixed two pieces of quartz glass on it, to imitate a cuvette. Quartz glass has relatively high light transmission, thus can be an alternative for cuvette, which is expensive. The columns at the corner are designed to help to fix the glass, while the outreaching bar enables users to replace the detecting cell more conveniently. We drilled two holes on the lid of the cell, to insert the pump pipe.</P><P>A 585nm-595nm 1W LED light bead, with the rated voltage around 2.0V, is applied in our hardware. Two 1kΩ resistors were put in series with the LED to reduce the voltage across it, as our circuit board can only provide 5V output.</P><P>To sense the fluorescence, we installed a LDR (light dependent resistor) sensor of LXD55 series, whose resistance ranges from 8 to 20 kΩ in brightness (10 Lux), and 1MΩ in darkness (0 Lux) with response time of around 20ms. A 10kΩ resistor is in series with the LDR sensor to divide the voltage, and then the resistance value of sensor is collected by the circuit board.</P><P>We printed some small components in this part to fix or support the items mentioned above, with the 3D printer.</P><H4>Control System</H4><P>An Arduino Nano V3.0 board is employed to control the whole equipment. It is powered by a battery and provides 5V output to different parts. Meanwhile it collects the signal from LDR sensor and transports it to the Bluetooth module, and later sends to user’s phone. </P><P>Five relays under the control of Arduino Nano separately regulate the power supplies of the pumps, the LED, the LDR sensor, and the Bluetooth module. We chose Risym 5V electromagnetic relay which could bear up to DC30V/10A. </P><P>For transmitting signal we applied HC-60 wireless Bluetooth module.</P><P>Firstly we plan to apply a much more complex design, which required us to 3D-print the total equipment. In fact it is difficult for our machine to print such a large box. Thus we simplified it and decided to only print some small components and use a relatively flat PC box to hold all the parts. The heaviest item, the battery, is put on the bottom and the whole detection system is placed at the corner, to stabilize the equipment.</P><H3>Measurement</H3><P>Different parameters of the hardware were acquired through a series of experiments. We attached importance to the performance of filter membrane, investigating the efficiency of filtration of bacteria and flow velocity. The working state of the pumps and the LED light bead were checked. And we conducted experiments to simulate the expected working condition of our hardware, to inquiry whether it functions well when floating in the ponds.</P><H4>Filter membrane</H4><P>To test the effect of the filter membrane, 2mL bacterial suspension flew pass the pump, and the absorbance at OD600 was tested by a spectrophotometer. LB medium was test as control group. And for each type of membrane we repeated for 3 times.(MEASURE-2/3)</P><TABLE><TBODY><TR><TD>Original OD600</TD><TD>1</TD><TD>2</TD><TD>3</TD></TR><TR><TH>0.45μm</TH><TD>1.505</TD><TD>0.001</TD><TD>0.005</TD><TD>0.002</TD></TR><TR><TH>0.22μm</TH><TD>1.505</TD><TD>0.000</TD><TD>0.019</TD><TD>0.554*</TD></TR></TBODY></TABLE><P>In the experiment we found that parafilm is not strong enough to seal the joint. In the third test for 0.22μm membrane, some liquid leak from the gap and flow into the collection tube, making the outcome rather strange.</P><P>Considering that we will applied immobilized bacteria for application, which means there will be quite rare bacteria in the water after detection process, we believe it is reasonable to applied a 0.45μm filter membrane once we solve the leakage problem.</P><H4>Pumps</H4><P>For pumping water into and out the detecting cell, we chose two Kamoer peristaltic pumps NKP-DE-S04. We measured that, when powered by Arduino Nano (5V output), the voltage across the pump is 5.40V, while the current intensity is 0.22A. (pic1.pump v/pic2.pump a)</P><P>Also we measured the pumping speed under the same condition. Pumping 50mL water cost 3’54’’ on average, and it means the pump should work approximately 7 seconds each time to pump 1.5-2mL water, filling the detecting cell. </P><P>Another flow velocity experiment was conducted using 0.45μm and 0.22μm filter membrane. Different membranes were installed at the end of pump pipe and the joint was sealed with parafilm. It took 3’41’’ for 50mL water to flow past the 0.45μm membrane and 3’35’’ for 0.22μm membrane. (MEASURE-1)</P><H4>LED</H4><P>A 585nm-595nm 1W LED light bead, with the rated voltage around 2.0v, is applied in our hardware. Two 1kΩ resistors were put in parallel with the LED to reduce the voltage across it, as Arduino Nano can only provide 5V output. Then we found the working voltage was 1.90V and the current is 1.75mA. (pic3.LED v/pic4.LED a)</P></DIV><DIV id="Application.Demonstrate"><P>To solve the problem of excessive ammonia in fishing ponds, we designed a biosensor and a degrading pathway.</P><H3>Sensor</H3><P>To efficiently detect the ammonia in fishing ponds, we employed the nitrogen-sensitive promoter glnAp2. As the results shown by <A data-jumoto="projectsensor-results">sensor-results</A>, the promoter is able to discriminate the difference of the ammonium concentration in an extremely low level (<STRONG>Fig 1</STRONG>), with a remarkable signal-noise ratio and fluorescence intensity (<STRONG>fig 2, 3</STRONG>). </P><P>In terms of our real application, we constructed a sensor circuit where a reporter mRFP with a moderate-degradation DAS-tag was expressed from an NtrC-regulated promoter glnAp2, and tested its function in laboratory. </P><P>In laboratory, we used Cytation3 Multi-Mode reader to obtain the kinetic curves. Although after being added a degradation tag, the system becomes more unstable, it still displays the robust ability to respond to different ammonium concentrations specifically (<STRONG>Fig 5</STRONG>).</P><P>Considering the response delay, we found that glnAp2 responded to ammonium-deficiency most rapidly (fig 6), taking less than 5 hours. Since our sensor will be used in aquaculture ponds, where 5 hours is a period short enough for farmers to take actions, we believe this promoter can be used in our device. Furthermore, with future optimization, response delay can be even shorter, enabling more efficient responses to elevate water quality and reduce harm caused to fish. </P><P>In reality, however, aquaculture water is a complex mixture of different compounds, including a lot of nitrogenous compounds. One of the main nitrogenous compounds in aquaculture water is nitrate, which reaches a concentration of 60uM. Therefore, to see whether real-world nitrate would interfere with ammonium-sensing ability of our selected promoter glnAp2, we we added potassium nitrate of different concentration into working medium with 0.02-fold of standard ammonium concentration and detected fluorescence intensity and OD600 of bacteria culture. Results showed that 100uM nitrate had slight influence on response of glnAp2 to ammonium concentration, suggesting that glnAp2 is suitable for ammonia detection in aquaculture water (<STRONG>Fig 4</STRONG>).</P><P>Undoubtedly that the working circuit still needs to be further modified, but right now we have demonstrate that it is able to sense the concentration of ammonium and give valid response.</P><H3>Degradation</H3><P>For gas part, we successfully found 6 metal-containing enzymes capable of oxidizing N<SUB>2</SUB>H<SUB>4</SUB> with O2 as oxidant, and this sets as a footstone for our aerobic ammonium oxidation pathway.</P><P>For solid part, we first demonstrate that our bacteria did produce uric acid, which is not an easy task. We cultured bacteria expressing xanthine oxidoreductase (XOR) and bacteria expressing sfGFP under same conditions, and both genes were cloned to the same vector. After 3 days we centrifuged the bacteria culture and resuspended the bacteria in 400mM Tris-HCl buffer (pH=8.5), and added 25uL of xanthine (21mM) as substrate. After shaking at 1100 rpm at 37℃ and centrifuging for 15min, we detected uric acid in the supernatant using HPLC-MS (for detailed protocol, see <A data-jumpto="ucaslabmethods">methods</A>). By co-analyzing UV absorbance plot at 275nm and MS plot of molecular weight 169, and comparing retention time with standard uric acid sample, we confirmed the presence of uric acid in the supernatant of XOR bearing bacteria but not sfGFP-bearing bacteria (Figure 5). Thus, we successfully fulfilled the production of uric acid.</P><P>Next, we demonstrate that by over-expressing key enzymes in de novo purine biosynthesis pathway, uric acid production was elevated in xanthine-provided reactions. Key enzymes were selected according to textbooks (PurF and Prs) and ATP-dependence of corresponding reaction (that is, ATP-dependent enzymes were selected) (Figure 7.). Following the same procedure, we measured the uric acid produced in supernatant after reactions. Results showed that this strategy worked quite well, since 8 groups out of 9 exhibited higher uric acid-producing activity than control group (that is, expressing XOR alone) (Figure 8).</P><P>Most importantly, we demonstrate that by over-expressing some of these enzymes, comparative amount of uric acid was produced even without extra xanthine supply (Figure 9). This means that normal nitrogenous substrates can be transferred into purines and be stored in bacteria for uric acid production. Thus, our idea of transforming aquatic ammonia into uric acid is highly feasible. Time permitted, we would test this hypothesis in field. </P></DIV><DIV id="HumanPractices"><DIV data-special="two_column"><P>Interviews with different professors and experts help us knew more about ammonia-nitrogen pollution, and gave us inspirations to improve our design; communications with fish farmers make it clear what they need, thus we decide to publicize our article and write a brochure to provide specific help; achievements on social media give us a chance to reach more people, and share our experience with the world!</P></DIV><DIV data-special="two_column"><P>Our human practice work is integrated to the project from the very beginning to the end. Communications with professors help us to be clear with background situation, talks with technicians provide information on designing improvements, interviews with government staff give advice on application issues, and visits to fish farmers bring about new ideas on hardware design! </P></DIV><DIV data-special="two_column"><P>In order to get more people know synthetic biology, we joined and organized activities such as video competition and public talks, introducing knowledge and sharing experiences. Also, we used our own WeChat official account to show descriptions about our project, knowledge about synthetic biology, and update our current activities. We really enjoyed sharing stories with the public!</P></DIV></DIV><DIV id="HumanPractices.Silver"><H4>Talks and interviews</H4><DIV data-special="two_column"><P>In the communication with professors and experts, we knew more about ammonia-nitrogen pollution in water, as well as the current methods to deal with it. How could we make a different on the problem? How could we improve our design to be more helpful? University professors and factory technicians gave us useful inspirations and suggestions.</P></DIV><H4>Outreach for farmers</H4><DIV data-special="two_column"><P>Our project focused on ammonia pollution, and we also wanted to provide farmers with more specific help. We contacted with the official account of Chinese aquaculture website, and publicized our article which gained more than 4000 clicks. Also, a 24-pages brochure was written and delivered to farmers over China, helping with problems in fish-farming and solutions to fish diseases. </P></DIV><H4>Social media</H4><DIV data-special="two_column"><P>Social media efficiently helped us to reach lots of people at the same time. This year, we used WeChat official account to show descriptions about our project, introduce knowledge about synthetic biology, and update our current activities. We really enjoyed sharing stories with public!</P></DIV></DIV><DIV id="HumanPractices.IntegratedGold"><H3>Interviews with experts in aquaculture</H3><P>To customize our project to the practical conditions, we interviewed some experts in related fields. We explained our aim purpose of promoting the development of fresh water aquaculture by dealing with excessive ammonia and presented the prototype of our project to them. Thanks to their suggestions, we attained a more comprehensive understanding of the practical conditions and improved our own project.</P><H4>Mingxian HUANG</H4><SMALL>Editor of the official account of China Aquaculture Website:</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>We interviewed Mingxiang Huang, the editor of the official account of China Aquaculture Website, the largest online aquatic products market and the communication platform of fish raisers.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>The water pollution is severe in China, and aquaculture waste water contributes a lot to the ammonia-nitrogen pollution.</LI><LI>The current methods of ammonia-nitrogen degradation are circulating stagnant water and using activated sludge.</LI><LI>Administration and proper guidance is needed for fish raisers in the rural area.</LI><LI>It is important to concern about bio-safety when applying engineered bacteria to ponds.</LI></UL></DIV></DIV><P>The information and suggestion we obtained from this interview inspired us to go to synthetic biology for help. We expect to use genetically engineered bacterium to solve this problem. Let’s find a solution to detect and degrade ammonia-nitrogen pollution in a cheap and easy way!</P><H4>Zhisheng YU</H4><SMALL>Professor of Energy and Environmental Microbiology in UCAS</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>In order to customize our project to practical conditions, we interviewed Zhisheng YU, professor of Energy and Environmental Microbiology in University of Chinese Academy of Sciences.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>The water pollution is severe in China, and aquaculture waste water contributes a lot to the ammonia-nitrogen pollution.</LI><LI>The current methods of ammonia-nitrogen degradation are circulating stagnant water and using activated sludge.</LI><LI>Administration and proper guidance is needed for fish raisers in the rural area.</LI><LI>It is important to concern about bio-safety when applying engineered bacteria to ponds.</LI></UL></DIV></DIV><P>Prof. YU raised up many issues that we had never concerned before and inspired us to think more comprehensively. Let’s divide the detection part and the degradation part, and do more things on bio-safety!</P><H4>Mr. XU</H4><SMALL>Technician of Yancheng Cheng Dong foul water purifying factory</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>Does the use of bacteria in water purification have any reference meaning to the degradation of ammonia in fish pond? We interviewed Mr. XU to find out the answer. </P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>Activated sludge which is cultivated by their workers has been used in the factory.</LI><LI>There is not a full-blown industry in China for cultivating bacteria strains that can purify the water</LI><LI>A low-cost device to detect and degrade nitrogen will be very useful.</LI><LI>They advised us to pay more attention on engineering and application.</LI></UL></DIV></DIV><P>Thanks to the advice Mr. XU gave us, we notice the connection between laboratory work and marketing. Let’s design a device for the application of our project!</P><H4>Mr. LI</H4><SMALL>Management Staff of the sturgeon breeding technology engineering center of the CAFS</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>How will our device and bacteria be applied in practical fish breeding? We interviewed Mr. LI, Management Staff of the sturgeon breeding technology engineering center of the Chinese Academy of Fishery Sciences to find out the answer.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>At present, full-flow aquaculture was an advanced aquatic cultivation model, which isn’t wildly adapted due to the high cost.</LI><LI>Chemicals or biological agents are rarely used in the full-flow aquaculture factory.</LI><LI>Ammonia nitrogen index isn’t contained in the on-line water monitor system. </LI><LI>It’s quite troublesome and dangerous to detect ammonia nitrogen in laboratory.</LI></UL></DIV></DIV><P>The staff of the sturgeon breeding center was grad to apply genetically engineered bacteria product as far as it is highly efficient. Let’s do more things on application!</P><H3>Interviews with government officials</H3><H4>Wenqi PENG</H4><SMALL>Deputy Director of China Institute of Water Resources and Hydropower Research</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>Is excessive ammonia genuinely fatal to aquatic creatures? We interviewed Wenqi PENG, Deputy Director of China Institute of Water Resources and Hydropower Research to find out the answer.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>Ammonia: the largest source of water pollution in China after 2010</LI><LI>Fish cannot live in water with the concentration of ammonia above 2mg/L or that of dissolved oxygen below 0.2mg/L</LI><LI>March, 2007, because of Tai Hu’s eutrophication, more than 3 million citizens of Wu Xi City were affected.</LI><LI>No approach was widely adopted to treat ammonia pollution that is both efficiently and economically in China.</LI></UL></DIV></DIV><P>We learned from this communication that excessive ammonia is a crucial problem all over the world and. Let’s try to use bacteria to treat excessive ammonia in the fish pond efficiently and economically!</P><H4>Mr. LI</H4><SMALL>Supervisor of Yancheng Ocean and Fisheries Bureau</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>As our device is designed for aquaculture industry, we should have knowledge of what kind of technologies are already used and will to be used in aquaculture. This is why we intervewed Mr. LI, supervisor of Yancheng Ocean and Fisheries Bureau.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>Aquaculture business in Yan Chen City is significantly prosperous.</LI><LI>Biological products is the major method taken to balance and control aqua environment.</LI><LI>IoT (internet of Things) technology currently shows multiple application in aquaculture management.</LI></UL></DIV></DIV><P>Li is majorly concerned about the biosafety of our design and we also think it is a key point on whether our product can be sold on the market or not. Let’s pay more attention on safety of our device!</P><H3>Contact with fish farmers</H3><P>To make our project and device have practical meaning, we contacted with different fish farmers and presented our project and device to them. Thanks to their feedback, we had better understanding about what fish farmers actually concerned about. In addition, we also found that most of the farmers are lack of basic scientific knowledge to deal with outbreak of fish diseased and ammonia nitrogen. This inspired us to put some effort on publicizing scientific knowledge among fish farmers.</P><H4>Crab Farmers</H4><SMALL>Yancheng, Jiangsu Provice, China</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>Crab is one of the fresh water creatures that are extremely sensitive to water quality and to be specific, ammonia concentration. Therefore, we interviewed some crabs farmers to see what they think about our project.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>Individual farmers lack of cultivation knowledge, so factory farming is not implemented on a large scale.</LI><LI>Experience still plays an important role in assessing pond water quality.</LI><LI>As for the pond water detection, online equipment is expensive while offline kits are complex and abstruse.</LI></UL></DIV></DIV><P>According to our communication with the crab farmers, we feet it important to design a product that farmers without much knowledge can use easily. Let’s make a convenient, cheap online device to help the farmers!</P><H4>Sturgeon Farmers</H4><SMALL>Beijing, China</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>Since it is important to and learn about their opinions about ammonia pollution in ponds, we contacted with several farmers in Beijing, paid visits to their ponds and had face-to-face talks with farmers to get more information.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>Fish farmers themselves do not care about biosafety, they only care about how customers think about their aquatic product.</LI><LI>Changing water is an efficient but expensive way to improve the water quality, only big companies can afford 200 thousand yuan per month on changing water.</LI><LI>Our project should aim at self-employed fish ponds instead of those big aquaculture companies.</LI><LI>Good water quality does make fish more tasty and fish can be sold for a higher price as long as the fish tastes good.</LI></UL></DIV></DIV><P>Good water quality does make fish more tasty and fish can be sold for a higher price as long as the fish tastes good.</P><H3>Analyze composition of fish pond water</H3><P>In order to help our engineered bacteria accustom to the practical water condition, we collected water samples from fish ponds and analyzed the composition of the water sample. We then cultured our bacteria in the culture medium that stimulating the practical fish pond water condition.</P><H4>Analyze composition of fish pond water</H4><SMALL>Beijing, China; Mumbai, India;Hangzhou, China</SMALL><DIV data-special="two_column"><DIV data-special="two_row"><P>We received help from ICT-Mumbai and ZJU-China. We analyzed the inorganic components and the amino acid components of the fish pond water sample to help us know better about the practical water condition.</P></DIV><DIV data-special="two_row"><STRONG>Key point</STRONG><UL><LI>The ammonia concentration in fish pond water is actually rather low, even though it can do harm to the fish</LI><LI>The sensitivity of our device should be high enough to detect the fluctuation of ammonia concentration</LI><LI>There is not enough nutrition in the fish pond water for our bacteria to gcolumn with.</LI><LI>We probably need to add some culture medium into our device to breed our bacteria</LI></UL></DIV></DIV><P>What we learned from the data is of crucial value for the design of our device and sensor. Let’s make our device more sensitive and add culture medium into the device for our bacteria!</P><H3>Beijing Tianjin meetup</H3><DIV data-special="two_column"><P>On Oct 7th and 8th, UCAS iGEM team invited other 4 undergraduate teams and a high school team to our school and held a meet-up, including Zhejiang University, Beijing Normal University, Beijing Institute of Technology, Tianjin University of Science   Technology and The High School Affiliated to Renmin University of China. During the two-day activity, six teams gave presentation of their project, shared ideas and opinions.</P></DIV></DIV><DIV id="HumanPractices.PublicEngagement"><H3>For Farmers</H3><H4>Publicizing knowledge of ammonia</H4><DIV data-special="two_column"><DIV data-special="two_row"><P>In order to help more farmers to know the harm of ammonia pollution in ponds, we contacted with the official account of Chinese aquaculture website, and publicized our article though it. The article gained more than 4000 clicks, and fish farmers from all over the china came to share with us their problems or solutions towards ammonia pollution.</P></DIV></DIV><H4>Contact with farmers and paying visits to their ponds</H4><DIV data-special="two_column"><P>We visited some fish ponds around Beijing to find out their problems in dealing with excessive ammonia and other questions they may be faced with when breeding fish. They provided us with useful information and we also solved their questions about science and technology. We helped them to have better understanding of biotechnology especially the most controversial transgene technology.</P></DIV><H4>Aquaculture tips brochure</H4><DIV data-special="two_column"><DIV data-special="two_row"><P>How can we provide more help to fish farmers facing water quality and fish diseases problems? Our team members searched information and wrote a brochure for them. The brochure contained typical problems and tips to improve the water quality in each season, as well as symptoms and solutions for some widespread fish disease in China.</P><P>After finishing the brochure, we delivered the brochures to farmers who contacted us online though China. In brief, nearly 400 brochures were sent to twelve cities and counties in China, including Suzhou, Yanan, Yancheng, Jining, Guanzhou and so on.</P></DIV></DIV><H3>Bio-safety</H3><H4>Questionnaire about laws on transgene and bio-safety</H4><DIV data-special="two_column"><P>In order to know more about the opinions on transgene laws from iGEMers and students majoring in synthetic biology, we designed questionnaires and finished the survey. 177 students finished our questionnaire, most of whom were iGEMers or undergraduates and graduates majoring in biology. We found out that laws on bio-safety problems still need to be advertised, since a number of students were not familiar with them and thought it would be helpful to know more. Also, we provided some of the details in the laws, and majority students replied that they were useful to their study and research work.</P></DIV><H4>Video about bio-safety in laborites</H4><DIV data-special="two_column"><P>This year, UCAS iGEM team collaborated with other nine Chinese iGEM team, making a series of videos which focused on bio-safety to help Chinese students learn more about bio-safety. The topic of our team’s video is personal protective equipment, mainly concerns about the proper use of gloves, lab coat etc in laborites. Here comes our video.</P></DIV><P><A data-jumpto="https://www.youtube.com/channel/UCZEYJNccLLPRro1SX1AfVAA">Watch our VIDEO!</A></P><H3>Communication activities</H3><H4>Introduction video to GFP protein and awards</H4><DIV data-special="two_column"><P>This August, Institute of Physics Chinese Academy of Science, Beijing Association for Science and Technology etc organization held a “3 minutes for science” activity, aiming to collect videos which introduce scientific phenomenon to the public. Our team members made a video to introduce GFP protein and relative knowledge, won the special award and had our video broadcast in the Scientific Carnival in Beijing Olympic Parks.
		     </P></DIV><H4>Our own We Chat official account</H4><DIV data-special="two_column"><P>Since WeChat has been widely used in China, we decided to create our own official account to get more people know us and iGEM competition. In the past two months, we have shared introduction of iGEM competition and some details of our team's project. To sum up, we received 7731 clicks and more than 3783 persons read our passages. </P></DIV><H4>Advertising synthetic biology on SELF Conference</H4><P>SELF Conference is a TED-like forum jointly set up by CAS Computer Network Information Center and CAS Science Communication Bureau, inviting elites to share new ideas and discoveries. Its name is the abbreviation respective of Science, Education, Life and Future.</P><P>This time, we invited Ph.D. Haoqian Zhang, who is once an iGEMer and now the chief technology officer of a syn-bio company Bluepha, to give a speech on SELF Conference. Aiming at promoting synthetic biology, the speech talked about its principle, significance and various applications. In this conference which more than two hundred people attended, our UCAS iGEM team members offered volunteer services and have a really good time!</P></DIV><DIV id="HumanPractices.Entrepreneurship"><H3>Overview</H3><P>Aiming at designing an effective, convenient and cheap device to detect ammonia for fish farmers, we believe that our device has the potential to be applied in the real ponds and reach commercial success. The following is a summary of our work on entrepreneurship, including the discussion on product application, market analysis and financial summary.</P><H3>Product Application</H3><P>Since a patent is needed if we want to apply our device on real ponds, we went to the department of transformation of scientific and technological achievements in Institute of Biophysics, and asked for advice. During the communication with the staff there, we learned about the process of applying a patent and noted significant issues. At present, we are planning to apply a patent for our designs after the project is finished.</P><H3>Market Analysis Summary</H3><DIV data-special="two_column"><P>In China, freshwater aquaculture is the most important production method in the fishery industries. Among all aquaculture modes, ponds account for over 75% of the product. Nevertheless, few farming ponds in China are equipped with ammonia monitoring and water purification systems, causing pollution and fish diseases and reduced aquaculture production.</P></DIV><P>We compared our designs with other methods on the market which are widely used currently, and find our designs having the obvious advantages such as cheaper, cleaner and more convenient.</P><TABLE><TBODY><TR><TH>Indexes/Products</TH><TD>Biochemistry test kit</TD><TD>Portable water quality monitor</TD><TD>Multi-parameter water quality monitor</TD><TD>Automatic water quality monitor</TD></TR><TR><TH>Method</TH><TD>Visual colorimetry</TD><TD>Photoelectric  colorimetry</TD><TD>Photoelectric  colorimetry</TD><TD>Automatic electrochemical detection</TD></TR><TR><TH>Accuracy</TH><TD>About 0.3mg/l</TD><TD>0.01 mg/l</TD><TD>0.001mg/l</TD><TD>Unsure</TD></TR><TR><TH>Range</TH><TD>0.1-1.8mg/l</TD><TD>0.05-12.0mg/l</TD><TD>0.05 -50.0mg/l</TD><TD>0.1-300mg/l</TD></TR><TR><TH>Prize time</TH><TD>≦15$</TD><TD>422$</TD><TD>1508$</TD><TD>4524$</TD></TR><TR><TH>Usage count</TH><TD>≦50 times</TD><TD>50-100 times</TD><TD>limitless</TD><TD>limitless</TD></TR><TR><TH>Advantages</TH><TD>Cheap and test quickly</TD><TD>Portable and quite accurate， can test several compounds together</TD><TD>Accurate and effective, can store the data</TD><TD>Most accurate, convenient to use and can store the data</TD></TR><TR><TH>Disadvantages</TH><TD>Less accurate and toxic</TD><TD>Expensive for ordinary fish farmers</TD><TD>Much expensive for farmers</TD><TD>Extremely expensive for farmers, too large and need regular maintenance</TD></TR><TR><TH>Specific examples</TH><TD>Beijing Sangpu Biochemistry Science Company, Ammonia Test Kit</TD><TD>Okedan Biological Technology Company, Portable Water Quality Monitor</TD><TD>Jiangsu Shengaohua Environmental Protection Technology Company, 6B-220N Ammonia Detection Monitor</TD><TD>Nanjing Baowei Instrument Company, On-line Ammonia Detection  Monitor</TD></TR></TBODY></TABLE><H3>Financial Summary</H3><H4>Cost for a single device:</H4><TABLE><TBODY><TR><TH>Items</TH><TH>Price/$</TH></TR><TR><TH>Peristaltic Pump*2</TH><TD>7.5</TD></TR><TR><TH>Relay</TH><TD>2.9</TD></TR><TR><TH>Arduino Nano</TH><TD>3.9</TD></TR><TR><TH>HC60 Bluetooth module</TH><TD>2.7</TD></TR><TR><TH>Mobile power supply</TH><TD>10</TD></TR><TR><TH>LED</TH><TD>0.3</TD></TR><TR><TH>Piezoid</TH><TD>1.4</TD></TR><TR><TH>PC Box*2</TH><TD>4.5</TD></TR><TR><TH>3D Print wire and resistance</TH><TD>3.0</TD></TR><TR><TH>Total</TH><TD>36.2</TD></TR></TBODY></TABLE><H4>Cost summary:</H4><TABLE><TBODY><TR><TH>Item</TH><TH>Cost/$</TH></TR><TR><TH>Apply a patent</TH><TD>755</TD></TR><TR><TH>Device*5000^[1]</TH><TD>150000</TD></TR><TR><TH>Advertisement</TH><TD>15000</TD></TR><TR><TH>Price*5000^[2]</TH><TD>375000</TD></TR></TBODY></TABLE><SMALL>[1] In the bulk-production, we estimate that the cost of a single device will decrease to 30$</SMALL><SMALL>[2] For sale, we suppose that 75$ for a single device is proper, since we can also provide maintainance for users</SMALL><H3>The Future Plan</H3><P>We strongly hope to popularize and transform our achievements in lab, and benefit more fish farmers in China and even the world. In the way of reaching commercial success, we are confident to deal with challenges and difficulties. Chances remain that we can patent our device, realize batch production and bring our device to market. We are looking forward to the future in which a large number of fish farmers can use our product to solve the ammonia pollution problems efficiently and conveniently.</P></DIV><DIV id="Parts.Improve"><P>Degradation tag is a useful part in synthetic biology. It can be attached to any protein and then help engineer controllable protein degradation. This year, to construct a sensor system which clears its previous signals fast and automatically, we employed mRFP1 with different ssrA degradation tags, Ba_K1339001, BBa_K1339002, BBa_K1339003.</P><P>In the meantime, we noticed that there exists a series of parts composed by mut3b GFP and ssrA tags, for example, BBa_K1339004. Considering that the highly efficient folding capability of superfolder GFP (sfGFP) should be more exploited, we replaced the mut3b GFP with a superfolder GFP, to see if there is any improvement for this part. This time, we took the part BBa_K1339004, which is a mut3b GFP with a LVA tag, as a try.</P><P>By comparing the kinetic curves of these two parts (see <A data-jumpto="ucaslabmethods">methods</A> for more information), we observed that there is no remarkable improvement in total relative fluorescence intensity at the steady state, and the degradation trends are highly similar as well. The minimum fluorescence intensity of sfGFP with the LVA tag is a little bit weaker than mut3b GFP, which may because the set wavelength of the exciting light suits the mut3b GFP best but not the sfGFP. In another word, this may not be seen as an improved feature.</P><P>However, when analyzing the gcolumnth curves, we found that the curve of mut3b GFP reveals stronger fluctuation than sfGFP. That is to say, the sfGFP with a LVA tag shows a more stable gcolumnth state (fig.1). This is a significant improvement, because for engineered bacteria expected to work properly in real world, they must have robust viability. A part leaves bacteria gcolumning unstably cannot be further programmed efficiently.</P></DIV><DIV id="Parts.BasicParts"><TABLE><TBODY><TR><TH>Part number</TH><TH>Name</TH><TH>Type</TH><TH>Description</TH><TH>Length/bp</TH><TH>Designer</TH></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287000">BBa_K2287000</A></TH><TD>sfGFP+LVA</TD><TD>Coding</TD><TD>sfGFP with a LVA degradation tag</TD><TD>753</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287001">BBa_K2287001</A></TH><TD>glnAp2</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter</TD><TD>299</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287002">BBa_K2287002</A></TH><TD>glnHp2</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter</TD><TD>287</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287003">BBa_K2287003</A></TH><TD>astCp</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter</TD><TD>427</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287021">BBa_K2287021</A></TH><TD>Prs</TD><TD>coding</TD><TD>phosphoribosylpyrophosphate synthase</TD><TD>948</TD><TD>Jianyi Huang</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287022">BBa_K2287022</A></TH><TD>rhXOR</TD><TD>coding</TD><TD>recombinant human xanthine oxidase</TD><TD>4002</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287023">BBa_K2287023</A></TH><TD>purF</TD><TD>coding</TD><TD>glutamine phosphoribosylpyrophosphate amidotransferase</TD><TD>1518</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287024">BBa_K2287024</A></TH><TD>purD</TD><TD>coding</TD><TD>glycinamide ribonucleotide synthetase</TD><TD>1280</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287025">BBa_K2287025</A></TH><TD>purM</TD><TD>coding</TD><TD>aminoimidazole ribonucleotide </TD><TD>1018</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287026">BBa_K2287026</A></TH><TD>purC</TD><TD>coding</TD><TD>succinylaminoimidazolecarboxamide ribonucleotide synthetase</TD><TD>714</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287027">BBa_K2287027</A></TH><TD>K326Q(mpurF)</TD><TD>coding</TD><TD>mutant of glutamine phosphoribosylpyrophosphate amidotransferase</TD><TD>1518</TD><TD>Jiang Wu</TD></TR></TBODY></TABLE></DIV><DIV id="Parts.CompositeParts"><TABLE><TBODY><TR><TH>Part number</TH><TH>Name</TH><TH>Type</TH><TH>Description</TH><TH>Length/bp</TH><TH>Designer</TH></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287010">BBa_K2287010</A></TH><TD>glnAp2+sfGFP</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter with a sfGFP reporter</TD><TD>1177</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287011">BBa_K2287011</A></TH><TD>glnHp2+sfGFP</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter with a sfGFP reporter</TD><TD>1165</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287012">BBa_K2287012</A></TH><TD>astCp+sfGFP</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter with a sfGFP reporter</TD><TD>1305</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287013">BBa_K2287013</A></TH><TD>glnAp2+mRFP1+LVA</TD><TD>reporter</TD><TD>Lack-nitrogen induced promoter with a mRFP1 reporter and a LVA degradation tag</TD><TD>1171</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287014">BBa_K2287014</A></TH><TD>glnAp2+mRFP1+DAS</TD><TD>Reporter</TD><TD>Lack-nitrogen induced promoter with a mRFP1 reporter and a DAS degradation tag</TD><TD>1017</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287015">BBa_K2287015</A></TH><TD>glnAp2+Mrfp1+LAA</TD><TD>Reporter</TD><TD>Lack-nitrogen induced promoter with a mRFP1 reporter and a LAA degradation tag</TD><TD>1171</TD><TD>Ruiling Cai</TD></TR></TBODY></TABLE></DIV><DIV id="Parts.PartCollection"><TABLE><TBODY><TR><TH>Part number</TH><TH>Name</TH><TH>Type</TH><TH>Description</TH><TH>Length/bp</TH><TH>Designer</TH></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287000">BBa_K2287000</A></TH><TD>sfGFP+LVA</TD><TD>Coding</TD><TD>sfGFP with a LVA degradation tag</TD><TD>753</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287001">BBa_K2287001</A></TH><TD>glnAp2</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter</TD><TD>299</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287002">BBa_K2287002</A></TH><TD>glnHp2</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter</TD><TD>287</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287003">BBa_K2287003</A></TH><TD>astCp</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter</TD><TD>427</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287010">BBa_K2287010</A></TH><TD>glnAp2+sfGFP</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter with a sfGFP reporter</TD><TD>1177</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287011">BBa_K2287011</A></TH><TD>glnHp2+sfGFP</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter with a sfGFP reporter</TD><TD>1165</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287012">BBa_K2287012</A></TH><TD>astCp+sfGFP</TD><TD>regulatory</TD><TD>Lack-nitrogen induced promoter with a sfGFP reporter</TD><TD>1305</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287013">BBa_K2287013</A></TH><TD>glnAp2+mRFP1+LVA</TD><TD>reporter</TD><TD>Lack-nitrogen induced promoter with a mRFP1 reporter and a LVA degradation tag</TD><TD>1171</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287014">BBa_K2287014</A></TH><TD>glnAp2+mRFP1+DAS</TD><TD>Reporter</TD><TD>Lack-nitrogen induced promoter with a mRFP1 reporter and a DAS degradation tag</TD><TD>1017</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287015">BBa_K2287015</A></TH><TD>glnAp2+Mrfp1+LAA</TD><TD>Reporter</TD><TD>Lack-nitrogen induced promoter with a mRFP1 reporter and a LAA degradation tag</TD><TD>1171</TD><TD>Ruiling Cai</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287021">BBa_K2287021</A></TH><TD>Prs</TD><TD>coding</TD><TD>phosphoribosylpyrophosphate synthase</TD><TD>948</TD><TD>Jianyi Huang</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287022">BBa_K2287022</A></TH><TD>rhXOR</TD><TD>coding</TD><TD>recombinant human xanthine oxidase</TD><TD>4002</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287023">BBa_K2287023</A></TH><TD>purF</TD><TD>coding</TD><TD>glutamine phosphoribosylpyrophosphate amidotransferase</TD><TD>1518</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287024">BBa_K2287024</A></TH><TD>purD</TD><TD>coding</TD><TD>glycinamide ribonucleotide synthetase</TD><TD>1280</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287025">BBa_K2287025</A></TH><TD>purM</TD><TD>coding</TD><TD>aminoimidazole ribonucleotide </TD><TD>1018</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287026">BBa_K2287026</A></TH><TD>purC</TD><TD>coding</TD><TD>succinylaminoimidazolecarboxamide ribonucleotide synthetase</TD><TD>714</TD><TD>Jiang Wu</TD></TR><TR><TH><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2287027">BBa_K2287027</A></TH><TD>K326Q(mpurF)</TD><TD>coding</TD><TD>mutant of glutamine phosphoribosylpyrophosphate amidotransferase</TD><TD>1518</TD><TD>Jiang Wu</TD></TR></TBODY></TABLE></DIV><DIV id="Team"><DIV data-special="two_column"><P>We are team UCAS, consisting of 13 UCAS (University of Chinese Academy of Sciences) students from different grades and different majors. We gather together for the same aim of innovating in synthetic biology and disperse apart to strive in our areas of expertise. We think, we talk, we practice, we progress. We wrangle, we reconcile, we laugh, we love. We are a team, we are a family, we are team UCAS.</P></DIV><H3>Instructors</H3><H3>Advisors</H3><H3>Team Members</H3><DIV data-special="album" data-arrangement="2 5 6"><DIV><H5>Ruiling CAI</H5><Q>I will definitely arrive at the lab before 9 tomorcolumn! (Never)</Q><P>Ruiling is a junior majoring in biology at University of Chinese Academy of Sciences.</P><P> Devoted to iGEM for two years, she has got lots of training from lab work to project design. Besides, she really treasures the precious and memorable experience of working with all her teammates and instructors. About them, the future is still very long. To be short, she is the leader of the team this year.</P><P> “Deadline gathers, and now my watch begins. It shall not end until my death. I shall have no breakfast, play no games, watch no films. I shall wear no ccolumnns and win no glory. I shall live and die at my post. I am the alarm of the clock. I am the watcher for the schedule. I am the fire that burns against the laziness, the light that brings the energy, the horn that wakes the sleepers, the shield that guards the team of UCAS. I pledge my life and honor to the Deadline’s Watch, for this ddl and all the ddls to come.”</P><P> Apart from iGEM she is also a singer and a geek design aficionado. To her, the beat and melody have the magic power to sweep away all the tiredness and stress, while the fascinating design works are the spring of imagination and creativity.</P></DIV><DIV><H5>Jianyi HUANG</H5><Q>If ONLY I’m a MATH MAJOR!!!</Q><P>Jianyi is a junior majoring in biology and minoring in mathematics.</P><P>This is the second year that she has been on UCAS iGEM team and also the second year that UCAS has joined iGEM community. She likes playing with DNA fragments and create something interesting out of basic elements.Unfortunately, she always regrets she hadn’t become a math major, especially when tons of data drove her mad. The good thing is that she would be happy with everything as long as she could stay in lab.</P><P> As a second-year iGEMer, she organized group meetings and trainings of new team members in the first stage. Responsible for our DEGRADATION MODULE, she devoted most of her time in lab. </P><P>She is also fluent in English and wrote the PROJECT part of our wiki. She plays the guitar and doesn’t talk much.</P></DIV><DIV><H5>Jingnan YE</H5><Q>Easily pissed off when just getting up</Q><P>Jingnan is a junior double majoring in biology and material sciences and engineering at UCAS.</P><P>She sincerely appreciates her excellent teammates and awesome instructors working and learning with her during this enjoyable year, from which she discovered more about what she hates and will never try in her future research. </P><P>She spent most of her time in the lab and on her way to the lab, while successfully spared some time to arrange a couple of HP activities and do some paperwork. </P><P>Outside the lab she is also a music lover, a pop dancer and an amateur bodybuilder. She believes exposing herself to music and art can help her keep good mood when the experiment fails again, and regular exercise can keep her working in the lab for more than 15 hours without feeling exhausted.</P></DIV><DIV><H5>Jiang WU</H5><Q>Crazy fan of Arya Stark</Q><P>Jiang who wants to study alchemy is a sophomore majoring in chemistry at UCAS.</P><P>Though sufferring the frustrating failures which shouldn’t have been experienced at his age, he learned a lot and knew better about synthetic biology. Through all this one-year journey, he confirmed his ideal again to attempt establishing the biological world based on axiom system.</P><P>Making a plan, doing experiments and fail and planning again. Following this circle and eventually gained payback. </P><P>He is a chemist with an artistic heart, deeply attracted by CS and mathematics and has a final goal to become a philosopher.</P></DIV><DIV><H5>Yi WU</H5><Q>Always satisfied to be a menial reseacher(Not exactly).</Q><P>Yi is a sophomore majoring in physics in UCAS.</P><P>During a couple of months spent on IGEM competition. She came across many outstanding teammates, from whom she harvested many new pieces of knowledge and experience. Though she may not continue the research work in the biological field, the precious experience thought she what &quot;scientific research&quot; really is.</P><P>Her summer vacation was spent in the laboratory where she served as an assistant. Out of lab she also did some paperwork such as translation and collecting some materials. She did some trivial works in HP and Hardware.</P><P>Although most of the time she acted as a &quot;manual worker&quot;, she is willing to contribute to the team.</P></DIV><DIV><H5>Xiaohan HU</H5><Q>More detail please!</Q><P>Xiaohan is a junior majoring in mathematics and applied mathematics at UCAS.</P><P>Taking apart in iGEM and working with so many outstanding teammates were some interesting attempts for her. At least she knows how to keep calm now.</P><P>She was responsible for the artistic design of the project. </P><P>She has a bunch of seemingly unrelated hobbies in her daily life, such as debating, painting and literature. What she wants to do further is to make all these hobbies be related.</P></DIV><DIV><H5>Yutao JIANG</H5><Q>Kinds of fun.</Q><P>Junior majors in physics, minors in biology at UCAS.</P><P>Happy to meet guys with quite deep knowledge about biological systems who changed my view towards biological research. </P><P>Awesome at regular performance in complex biological systems. Majorly deals with ocean of numbers this year. Concentrates on modally and numerical analysis, take a bit of contribution on the design of hardware.</P><P>Away from the computer screen, favorites are songs and joggings. Likes traveling and eating special local foods. Fan of sci-fic movies and tech. product. </P></DIV><DIV><H5>Yuqing LEI</H5><Q>Tomorcolumn is another day.</Q><P>Yuqing is a sophomore majoring in biology at UCAS.</P><P>The year just passed witnessed the vigorous gcolumnth of her, which was due to the learning and working experience with iGEM team members.</P><P>While she used to be quiet and nervous, experience in iGEM really changed her a lot. Now, she has been more outgoing and warm-hearted, and never afraid to face challenges and difficulties.</P><P>She still enjoys reading and traveling alone, but becomes more open to new friends and new ideas.</P></DIV><DIV><H5>Yuze WU</H5><Q>I will go to bed before 12 today!</Q><P>Yuze is majoring in material sciences and engineering at UCAS.</P><P>He learns a lot in the project and is always inspired by his teammates. “I did many things I’ve never tried before. And when my teammates are taking pains, I do feel encouraged and believe that I have to work harder”, he says.</P><P>Yuze first worked in the lab, while later he focused on designing and making the hardware. He also took part in some HP activities. Attracted by biology science as a participants in iGEM, Yuze plans to minor in biology. </P><P>Besides, he keeps doing sport and sometimes play music for relax. He is interested in numerous things and enjoys a fulfilling college life.</P></DIV><DIV><H5>Yunfan LI</H5><Q>Riding the subway with my MP3 is the most enjoyable thing I can imagine.</Q><P>Yunfan is a sophomore majoring in biology in UCAS.</P><P>He gets a pretty memorable experience with his fantastic teammates and sapiential tutors and learns a lot in the lab. That really promotes his gcolumnth in both knowledge level and spiritual level. He also recognizes what he is truly interested in and figures out his future researching plan in cell oxidation and reduction.</P><P>He spends most of his time on fighting with the experiment in lab and draws the vestigial energy out for the hardware designing and circuit welding. The interlab work is also finished by him.</P><P>Like most schoolboys in campus, he loves computer games and sports and can play badminton quite well. Taking the public transport for aimless roaming is also his favor. However, no one can strip the comic out of his life. Remember! No one!!!</P></DIV><DIV><H5>Shubo YUAN</H5><Q>If I rest, I rust.</Q><P>Shubo is a junior majoring in biological science at UCAS.</P><P>He learns some experimental operation of molecular biology, some methods to organize material and data, and many routes from the lab to the school. He wants to make biological experiments more automated to liberate biotech hands.</P><P>He undertook most of the experimental tasks in the group of ammonia degradation. He spent most of his time adding samples. He also did some work of human practice, but mostly ended in failure.</P><P>Outside the lab he has little hobby in addition to watching anime. Learning and experimenting make him happy.</P></DIV><DIV><H5>Fengzhi LI</H5><Q>The one who thinks differently.</Q><P>Fengzhi is a sophomore majoring in Computer Science and Technology in UCAS. </P><P>He participated in both China National Biology Olympiad and National Olympiad in Informatics in High school. With interests in these research directions, he is dedicated to Molecular biology and bioinformatics.</P><P>He is the art designer and the electronics designer of this team. He tries hard not to get mad because he must polish his artwork dozens of times, and fix the numerous bugs when designing the hardware.</P><P>To relieve psychological stress, he plays saxophone and some other music instruments. He enjoys more than 10k different music each year, which makes him an amateur Music connoisseur.</P></DIV><DIV><H5>Shiyang WANG</H5><Q>You can relax at any time, but not today.</Q><P>Shiyang is a senior majoring in both chemistry and computer sciences at UCAS.</P><P>He greatly appreciates all those understanding and encouragement from his wonderful teammates and instructors. He have made great progress in website techniques, experimental operations and communication skills in this year. All of these make his iGEM journey a precious and memorable experience in his life.</P><P>He participated in topic selection, devoted all his summer to wiki programs, and assisted other teammates in modeling and hardware design. He used to be a piano player, outdoors amateur and movie fan, but now he only enjoys sweet sleeping after overnight debugging.</P></DIV></DIV></DIV><DIV id="Team.Attributions"><H3>Team Accomplishments</H3><H4>Project:</H4><P>The prototype of our project was raised by Shiyang WANG in our brainstorm session in Feb. 2017. This idea was then modified and improved by Ruiling CAI and Jianyi HUANG. Our investigators Prof. Xien-en ZHANG, Prof. Chunbo LOU and Prof. Jiangyun WANG helped in our project design. Our advisors Fankang MENG, Xudong LUO and Sun ZHI also offered help in project design.</P><H4>Experiments:</H4><P>Our experiments started in July 2017. As our team's leaders, Ruiling CAI and Jianyi HUANG, separately responsible for the sensor part and degradation part, designed and monitored our experiments. They also did the primary work of data analysis. Our advisors Fankang MENG, Xudong LUO and Sun ZHI gave their suggestion on project design. Our experiments were accomplished with the joint effort of the following 9 teammates: Ruiling CAI, Jianyi HUANG, Shubo YUAN, Jiang WU, Jingnan YE, Yi WU, Yunfan LI, Yuze WU, Yuqing LEI.</P><H4>Hardware:</H4><P>Our hardware was designed by Ruiling CAI, Yunfan LI, Yuze WU, Shiyang WANG and Fengzhi LI. Yuze WU visualized our idea by drawing the blueprint. He also cooperated with Yunfan LI to build the prototype device with accessible materials in our lab. Fengzhi LI helped design and construct the electric circuits of our device. He and Yutao JIANG also contributed to the 3D printing of certain parts of our device.</P><H4>Modelling:</H4><P>Our modeling work was mainly accomplished by Yutao JIANG, with the help of Shiyang WANG.</P><H4>Human Practice:</H4><P>Yuqing LEI is the leader of our HP Squad. She accomplished most of the contact and interview work and record every HP activities. In addition, Jingnan YE helped arrange a couple of HP activities and Shubo YUAN helped a lot in contacting work. The whole team all contributed to the implementation of all the HP work.</P><H4>Wiki:</H4><P>Our wiki was designed by Shiyang WANG, Ruiling CAI, Fengzhi LI and Xiaohan HU. The coding work was finished by Shiyang WANG, assisted by Fengzhi LI. The paperwork was finished by the joint effort of Ruiling CAI, Jianyi HUANG, Jiang WU, Jingnan YE, Yi WU, Yunfan LI, Yuze WU and Yuqing LEI.</P><H4>Art work:</H4><P>Our art work was accomplished by our excellent art designer Fengzhi LI, Xiaohan HU and omnipotent team leader Ruiling CAI.</P><H3>Supports</H3><H4>General support</H4><P>University of Chinese Academy of Sciences</P><P>College of Life Sciences, University of Chinese Academy of Sciences</P><P>Institute of Biophysics, Chinese Academy of Sciences</P><P>Institute of Microorganism, Chinese Academy of Sciences</P><H4>Project support and advice</H4><P>Prof. Xiaohong LIU generously provided advice on project design and experiment design.</P><P>	Prof. Xiuzhu DONG attended our project defence and offered her advice on project design.</P><P>Prof. Chenli LIU attended our project defence and offered her advice on project design.</P><P>FAFU-CHINA, Shanghaitech, ZJU-China, SDU_CHINA, Worldshaper-XSHS to help us perform circuit-verifying experiments.</P><P>ZJU-China and ICT-Mumbai helped us take and analyze water samples from their local lakes.</P><H4>Fundraising help and advice:</H4><P>College of Life Sciences, University of Chinese Academy of Sciences</P><H4>Lab support:</H4><P>Prof. Jiangyun WANG, IBP, CAS provided his lab for our experiments.</P><P>Prof. Xien-en ZHANG, IBP, CAS provided his lab for our experiments.</P><P>Prof. Dianbing WANG helped in laboratory management.</P><P>College of Life Sciences, UCAS in Institute of Tibetan Plateau Research, CAS helped us analysis the inorganic components of the fish pond water sample.</P><P>Peking University Health Science Center helped us analysis the amino acid components of the fish pond water sample.</P><P>Beijing National Day School provided their lab for part of our device experiments.</P><H4>Difficult technique support</H4><P>Cheng HU provided important advice and instructions in laboratory.</P><P>Jian HUANG gave consistent help and instructions on HPLC-MS analysis.</P><H4>Hardware support:</H4><P>Prof. Pingyong XU provided generous help in our hardware design by lending us a light filter.</P><H4>Wiki support:</H4><P>Lewis Sandler proofread our paperwork for wiki.</P><H4>Presentation coaching:</H4><P>Prof. Chunbo LOU coached us in our presentation.</P><P>Lewis Sandler coached us in our presentation.</P><H4>Human Practice support:</H4><P>Wenqi PENG participated in our Human Practice Interview and provided important background information and helpful advice.</P><P>Prof. Zhisheng YU participated in our Human Practice Interview and provided important background information and helpful advice.</P><P>Bluepha Microbe Technology Co. gave juristic advice on application of GM products.</P><P>China Aquaculture Website published the article about our project and helped us contact with fish farmers over China.</P><P>Beijing Shuianyuge Food Company provided fieldwork opportunities and important water samples of fishing ponds.</P><P>Sturgeon Breeding Technology Engineering Center of the Chinese Academy of Fishery Sciences provided fieldwork opportunities and important information about application.</P><P>Yancheng Ocean and Fisheries Bureau provided detailed information on aquaculture and governmental perspectives for our project.</P><P>Yancheng Chengdong foul water purifying factory provided helpful advice on the engineering and application of our project.</P><P>Dr. Haoqian ZHANG accepted our invitation of giving a SELF talk on synthetic biology.</P><H4>Other support</H4><P>Leying CHEN generously provide us with sound recording outfit for one of our HP activities.</P><P>Xiaoding LI took the photos of our team members on the team page.</P></DIV><DIV id="Team.Acknowledgement"><DIV data-special="namecard"><H4>Lewis Sandler, BS, JD</H4><P>Foreign Expert English teacher</P><P>proofreaded for iGEM.</P></DIV><DIV data-special="namecard"><H4>Xiaohong LIU, Ph.D, Prof.</H4><P>Members of the Youth Innovation Promotion Association, CAS</P><P>Key Laboratory of RNA Biology, IBP, CAS</P><P>Provided generous advice on project design and experiment design</P></DIV><DIV data-special="namecard"><H4>Xiuzhu DONG, Ph.D, Prof.</H4><P>Anaerobe research group, IMB, CAS</P><P>Director of State Key Lab. Microbial Resources, IM, CAS</P><P>Provided generous advice on project design and experiment design.</P></DIV><DIV data-special="namecard"><H4>Chenli LIU, Ph.D, Prof.</H4><P>Executive Director of the Center for Synthetic Biology Engineering Research, SIAT, CAS</P><P>Provided generous advice on project design and experiment design</P></DIV><DIV data-special="namecard"><H4>Dianbing WANG, Ph.D, Prof.</H4><P>National Laboratory of Biomacromolecules, IBP, CAS</P><P>Provided help on laboratory management</P></DIV><DIV data-special="namecard"><H4>Pingyong XU, Ph.D, Prof.</H4><P>Key Laboratory of RNA Biology, IBP, CAS</P><P>Provided generous help in hardware design</P></DIV><DIV data-special="namecard"><H4>Cheng HU</H4><P>Key Laboratory of RNA Biology, IBP, CAS</P><P>Provided important advice and instructions in laboratory</P></DIV><DIV data-special="namecard"><H4>Jian HUANG</H4><P>Key Laboratory of RNA Biology, IBP, CAS</P><P>Gave consistent help and instructions on HPLC-MS analysis</P></DIV><DIV data-special="namecard"><H4>Wenqi PENG</H4><P>Deputy Director of China Institute of Water Resources and Hydropower Research</P><P>Provided important background information and helpful advice</P></DIV><DIV data-special="namecard"><H4>Zhisheng YU</H4><P>Professor of Energy and Environmental Microbiology in UCAS</P><P>Provided important background information and helpful advice</P></DIV><DIV data-special="namecard"><H4>Haoqian ZHANG, Ph.D</H4><P>Co-founder of BluePHA (Beijing) Co., Ltd</P><P>Accepted our invitation of giving a SELF talk on synthetic biology.</P></DIV><DIV data-special="namecard"><H4>Institute of Tibetan Plateau Research, Chinese Academy of Sciences</H4><P>Helped analysis the inorganic components of the fish pond water sample</P></DIV><DIV data-special="namecard"><H4>Peking University Health Science Center</H4><P>Helped analysis the amino acid components of the fish pond water sample</P></DIV><DIV data-special="namecard"><H4>Bluepha Microbe Technology Co.</H4><P>Gave juristic advice on application of GM products</P></DIV><DIV data-special="namecard"><H4>Beijing Shuianyuge Food Company</H4><P>Provided fieldwork opportunities and important water samples of fishing ponds</P></DIV><DIV data-special="namecard"><H4>China Aquaculture Website</H4><P>Published the article about our project and helped us contact with fish farmers over China</P></DIV><DIV data-special="namecard"><H4>Sturgeon Breeding Technology Engineering Center of the Chinese Academy of Fishery Sciences</H4><P>Provided fieldwork opportunities and important information about application</P></DIV><DIV data-special="namecard"><H4>Yancheng Ocean and Fisheries Bureau</H4><P>Provided detailed information on aquaculture and governmental perspectives for our project</P></DIV><DIV data-special="namecard"><H4>Yancheng Chengdong foul water purifying factory</H4><P>Provided helpful advice on the engineering and application of our project</P></DIV><DIV data-special="namecard"><H4>iGEM teams CIT-Mumbai and ZJU-China</H4><P>Sent analysis data of water samples from their local lakes</P></DIV><DIV data-special="namecard"><H4>iGEM team BNDS_China</H4><P>Offered lab to test hardware</P></DIV><DIV data-special="namecard"><H4>iGEM teams FAFU-CHINA, shanghaitech, ZJU-China, SDU_CHINA, Worldshaper-XSHS</H4><P>Helped verify circuits and sent critical data</P></DIV><H4 data-noline="true">College of Life Sciences, University of Chinese Academy of Sciences</H4><H4 data-noline="true">University of Chinese Academy of Sciences</H4><H4 data-noline="true">Institute of Biophysics, Chinese Academy of Sciences</H4><H4 data-noline="true">Institute of Microorganism, Chinese Academy of Sciences</H4><H4>Special thanks to:</H4><P>Zelun LI</P><P>Jie BIAN</P><P>Leying CHEN</P><P>Xiaoding SUN</P></DIV><DIV id="Team.Collaborations"><H4>Measuring data for BIT-China</H4><P>When iGEMers from BIT-China asked us whether we could allow them to use our Cytation Multi-Mode readers to monitor the gcolumnth state and fluorescence emissions of their engineered yeast, we agreed without hesitation. BIT-China wanted to detect both OD and fluorescence intensity simultaneously for up to 30 hours to evaluate the function of their yeast. With our help, they needn’t get up and take samples early in the morning but can obtain a much larger amount of data, saving them trouble and improving the reliability.</P><P><A data-jumpto="https://2017.igem.org/Team:BIT-China/Collaborations">Visit BIT-China wiki</A></P><H4>Air Samples Testing for ICT-Mumbai</H4><P>ICT-Mumbai is working on dealing with the problem of excessive ammonia in toilets and chemistry labs. They contacted us via email after reading our abstract and found out that we both deal with excessive ammonia. We collected air samples from the toilets and chemistry laboratories in Huairou Campus of UCAS, and assayed them following their standardized protocol. We are glad that this data is helpful for improving ICT-Mumbai’s project.</P><P><A data-jumpto="https://2017.igem.org/Team:ICT-Mumbai/Collaborations">Visit ICT-Mumbai wiki</A></P><H4>Mentoring a New Team HFLS_H2Z_Hangzhou</H4><DIV data-special="two_column"><P>Every year, there will be some newly established teams in iGEM. As beginners on this rough ride, they might encounter many difficulties and need of help. HFLS_H2Z_Hangzhou is one of those iGEM teams this year. We paved the way for this newly established team by mentoring them, offering some advice and guidance on their project design and troubleshooting their circuits. In addition, we taught them DNA cloning methods like Golden Gate Assembly and helped them construct biobricks. This year, HFLS_H2Z_Hangzhou is focusing on removing excessive nitrite in canned food. Wish them good results!</P></DIV><P><A data-jumpto="https://2017.igem.org/Team:HFLS_H2Z_Hangzhou/Collaborations">Visit HFLS_H2Z_Hangzhou wiki</A></P><H4>Helping ZJU-China construct their biobrick</H4><P>We helped ZJU-China construct a calcium responsive promotor which plays a significant role in their crops-protecting project. This promotor can sense the increase of calcium ion concentration cause by a medium wave signal and express downstream genes to achieve the goal of using wave signal to regulate chassis gene expression. We used Golden Gate Assembly to assemble seven different fragments of the promotor in <I>E. coli</I>, and used Gibson Assembly to shift the promotor onto shuttle vector for their further use. We cultivated sincere friendship with ZJU-China through our close collaboration, they also attended our Beijing-Tianjin iGEM Meetup in Beijing in spite of the 1500km distance between us.</P><P><A data-jumpto="https://2017.igem.org/Team:ZJU-China/Collaborations">Visit ZJU-China wiki</A></P><H4>Adding Biobricks to the Parts Library of Shanghaitech</H4><P>We provided our three promoter parts constructed this year for Shanghaitech, helping to build up their parts library of input and output biobricks. Shanghaitech is aiming at creating an intercellular connection between bacteria containing different input/output circuits. We are looking forward to more data showing the performance of our input elements working with their system.</P><P><A data-jumpto="https://2017.igem.org/Team:Shanghaitech/Collaborations">Visit Shanghaitech wiki</A></P><H4>Offering Biobricks to RDFZ-China</H4><P>Biobricks to an iGEM project is like bricks to a skyscraper, without which a project cannot even start. This is why when RDFZ-China asked us for some essential biobricks, we offered without hesitation. This year, RDFZ-China is paying attention to helping solve the petroleum leakage issue in the soil. Rather awesome!</P><P><A data-jumpto="https://2017.igem.org/Team:RDFZ-China/Collaborations">Visit RDFZ-China wiki</A></P><H4>World-wide data base of water pollution</H4><P>Invited by Tianjin, we collaborated with FAFU-CHINA, XMU-China, SCUT-China A and other 5 teams to construct a world-wide data base of water pollution, especially copper pollution and cadmium pollution. After collecting all the data we need, we drew a map to visualize the data we collected. By constructing this map, we can get better understanding of the overall condition of water pollution around the world. As ammonia pollution is one of the severe water pollutions, we believed by collaborating with these wonderful teams, we can have better understanding of our own project from a brand new point of view. </P><P><A data-jumpto="https://2017.igem.org/Team:Tianjin/Collaborations">Visit Tianjin wiki</A></P><H4>Newsletter</H4><P>Collaboration and communication are always of crucial value in iGEM and there are many ways for iGEMers to communicate efficiently. This year, we wrote a description of our project and human practices for the September newsletter hosted by XMU-China. The Newsletters are widely read across the world by many other iGEMers. It’s our pleasure to share and discuss our preliminary work with teams all over the world.</P><H4>Safety Videos</H4><P>We worked with other Chinese iGEM teams (listed below) to create a series of videos introducing the critical safety instructions in laboratory. The video we made is about the personal protective equipment in the lab, which is quite important for every scientific researcher. Up to now, these videos have been watched for XXX times in total. We hope this collaboration can truly raise the awareness of the lab workers about safety.</P><P>Participated teams: AHUT_China, BNU-China, UESTC-China, FAFU-CHINA, HZAU-China, NAU-CHINA, Shanghaitech, Tianjin, TUST_China, WHU-China, UCAS</P><P><A data-jumpto="https://www.youtube.com/channel/UCZEYJNccLLPRro1SX1AfVAA">Watch our VIDEO!</A></P><H4>Online Conferences</H4><P>We took part in two online conferences with dozens of wonderful Chinese teams. One was with teams interested in water pollution, and the other was with teams involved in hardware design. We shared our ideas and suggestions with each other. As we are working on water related issue and applied device, these conferences were really great help for our following work.</P><H4>Helping Us Verify Circuits</H4><P>To fully characterize a circuit, a great amount of data is required, as only one team cannot accomplish such heavy work. Fortunately, we have FAFU-CHINA, Shanghaitech, ZJU-China, SDU_CHINA, Worldshaper-XSHS to help us perform circuit-verifying experiments and send us considerable amount of essential data, which helps us characterize our circuits better and improve our project further. Thanks to all the teams mentioned above.</P><H4>Fishing Ponds Water Sampling and Analyzing</H4><P>As our project is focusing on promoting fresh water aquaculture by dealing with excessive ammonia in the fish pond, it is essential to learn what the real fish pond water is like. ZJU-China and ICT-Mumbai helped us take and analyze water samples from their local lakes. Thanks to their contribution, we were able to make the media protocol simulating the real fish ponds, which is important for our future application test.</P><H4>Offering equipment and reagent</H4><P>Although we have more than two well equipped laboratories in the Institute of Biophysics (IBP, CAS) where we can carry out our experiments, we have to spend 3 hours commuting between IBP and our campus. Thanks to BNDS_China, who generously provided us with some equipment we do not have on campus, we managed to finish some of our experiment without wasting time on the subway. We also want to thank BIT-China. They generously provided us with 4 liters of Methanol for HPLC-MS testing during National Day vacation when we were out of stock and all our suppliers were off work, which facilitated the progress of our experiments.</P></DIV><DIV id="UCASLab.Safety"><H3>Safe Project Design</H3><P>Aimed at dealing with excessive ammonia in the fish pond, we engineered our bacteria to detect and degrade ammonia. To ensure the safety of our project, we chose <I>E. coli</I> (TOP10 strain for detection, BL21 strain for degradation and DH5α for InterLab) as our project chassis, which is of group 1 risk. Although we introduce resistance genes into our circuits, they are the ones that commonly used in the lab and have been widely confirmed to be safe.</P><P>For ammonia detection, we also designed a device to apply our bacteria into practical use, of which safety is of crucial value. We used filter to isolate the component that will contact with bacteria from other components of the device and open environment. Afterwards, we have a complete decontamination protocol to deal with our engineered bacteria and the component of the device that contact the bacteria.</P><P>For ammonia degradation, we have successfully expressed all the enzymes we need in <I>E. coli</I>. However, considering the harm <I>E. coli</I> can do to aquatic creatures, our engineered bacteria will not be directly released into fish ponds. Instead, we planned to replace our chassis with other harmless bacteria such as B.subtilis now that our concept has been proved success in <I>E. coli</I>.</P><H3>Safe Laboratory Work</H3><STRONG>Lab Safety Rules</STRONG><P>We have already completed and submitted the safety form. We guarantee to follow the safety rules provided by iGEM in our lab work. Besides, we have safety rules in our lab as well. We separate the experimenting area and living area apart, no experimental material is allowed in living area and no living item is allowed in experimenting area. When we do experiments, we strictly follow the protocol and are monitored by the lab manager.</P><STRONG>Safe Parts</STRONG><P>In detection part, we transformed three different promotors and a RFP gene into <I>E. coli</I> and in degradation part, we transformed some enzymes in purine metabolic pathway into our engineered bacteria. Therefore, no toxic matters will be produce in our bacteria. Our engineered bacteria have either chloramphenicol resistance or ampicillin resistance, which are widely used in the lab and have been widely confirmed to be safe.</P><STRONG>Safe organism</STRONG><P>We used and only used <I>E. coli</I> in our experiments. The strain of <I>E. coli</I> we use are TOP10, BL21 and DH5α, which means that all the organism we used in our lab are of group 1 risk. In addition, we separated our organisms from other organisms in the lab that are used by others. There is no phage or virus in our lab and our lab is of group 2 risk.</P><H3>Safe Shipment</H3><P>We have already sent our parts to iGEM Headquarters. Our parts are not in the select agents and toxins list. They are safe and well packed. There’s no liquid, no organism, no toxin in the package, only plasmids that are allowed to ship. Our parts are on their way to Boston, bon voyage!</P></DIV><DIV id="UCASLab.Methods"><H3>Protocol</H3><DIV data-special="protocol" data-heading="Agarose Gel Electrophoresis of DNA"><OL><LI>Add 1g agarose into 100mL 1×TAE buffer. Boil in miccolumnave oven to dissolve the agarose. The agarose concentration depends on the DNA size. 1% is the most commonly used concentration. Add in 5µl view dye in 100mL solution when it is cooled down to 60℃.</LI><LI>Choose the suitable gel tank and insert the comb. The electrophoresis comb according to the purpose. Pour the fluid agarose gel into the tank. After a complete solidification soak the gel in 1×TAE buffer to make the DNA strips clearer.</LI><LI>Pour enough running buffer into the electrophoresis tank and put the agarose gel in the electrophoresis tank.</LI><LI>Mix the sample with loading buffer sufficiently and load them into the sample lane together with proper marker. The choice of the marker is dictated by the size of the target DNA. Some PCR system already add loading.</LI><LI>Run the electrophoresis under 120V.</LI><LI>fter 15-20 min, put the gel in an UV detector and record the picture.</LI></OL></DIV><DIV data-special="protocol" data-heading="Bacteria Preservation Protocol"><OL><LI>The 50% glycerin should be autoclaved before use.</LI><LI>Mix 600µl bacteria solution with 600µl 50% glycerin separately in two 1.5mL tubes.</LI><LI>Seal the tubes with parafilm.</LI><LI>Preserve one tube in -20℃ refrigerator and another one in -80℃ refrigerator.</LI><LI>Use the bacteria store in -20℃ first.</LI></OL></DIV><DIV data-special="protocol" data-heading="Cell Culture and Harvest for Adenine Detection"><OL><LI>Gcolumn overnight culture of cells in 5mL LB-medium containing appropriate antibiotic at 37℃ and 200 rpm.</LI><LI>The bacterium solutions of step1 were transferred into 15mL centrifuge tubes.</LI><LI>The tubes were sterilized by autoclaving at 121℃ and 1.2 bar, then centrifuge at 4000r and room temperature for 10min.</LI><LI>Discard the supernatant and resuspended the cells in 2mL SDS lysis buffer and 1mL SDS, then put the tubes into drying oven at 70℃ for 30min.</LI><LI>Add 2mL 30% HClO4 and mix thoroughly for 2min.</LI><LI>Add 5mL 200mM KH2PO4.</LI><LI>Transfer 1mL solution into 1.5mL centrifuge tubes.</LI><LI>Centrifuge for 1.5min with 15000r/min at room temperature.</LI><LI>The supernatants were analyzed by HPLC.</LI></OL></DIV><DIV data-special="protocol" data-heading="SDS lysis buffer"><TABLE><TBODY><TR><TH>500mM EDTA</TH><TD>50mL</TD></TR><TR><TH>5M NaCl</TH><TD>100mL</TD></TR><TR><TH>1M TRIS-HCl pH=8.0</TH><TD>100mL</TD></TR><TR><TH>ddH2O</TH><TD>To 1L</TD></TR></TBODY></TABLE></DIV><DIV data-special="protocol" data-heading="Cell Culture and Harvest for Uric Acid and Xanthine"><OL><LI>Gcolumn overnight culture of cells in 5mL LB-medium containing 1mM sodium molybdate and 5μL appropriate antibiotic at 37℃ and 200 rpm.</LI><LI>5μL appropriate antibiotic at 37℃ and 200 rpm.</LI><LI>The bacterium solutions of step1 were transferred into 200 mL flasks with 100 mL of LB medium supplemented with 1mM sodium molybdate and 100μL appropriate antibiotic. </LI><LI>The flasks were incubated at 37 ℃ and 200 rpm and then changed the temperature to 30℃ until an OD600 of 1.5 was reached. </LI><LI>The cultures were incubated for 72h.</LI><LI>The cells were harvested by centrifugation at 3200r and room temperature for 10min, and resuspended in TRIS-HCl buffer (400 mM, pH 8.5) to a final OD600 of 100.</LI></OL></DIV><DIV data-special="protocol" data-heading="Detection of Hydrazine"><OL><LI>Set-up the reaction in 1.5mL centrifuge tubes as follows: 
			    <TABLE><TBODY><TR><TH>test solution</TH><TD>100µL</TD></TR><TR><TH>pure hydrochloric acid</TH><TD>5µL</TD></TR><TR><TH>3,2-dimethyl-4-aminobiphenyl</TH><TD>10µL</TD></TR></TBODY></TABLE></LI><LI>Incubate the tubes at room temperature for 2 minutes </LI><LI>The absorption peak at 484nm of wavelength was measured by ultraviolet spectrophotometer.</LI></OL></DIV><DIV data-special="protocol" data-heading="Detection of Hydroxylamine"><OL><LI>Set-up the reaction in 1.5mL centrifuge tubes as follows: 
			   <TABLE><TBODY><TR><TH>test solution</TH><TD>100µL</TD></TR><TR><TH>1% of 8-hydroxyquinoline ethanol solution</TH><TD>10µL</TD></TR><TR><TH>1M potassium carbonate</TH><TD>10µL</TD></TR></TBODY></TABLE></LI><LI>Incubate the tubes at room temperature for 2 minutes</LI><LI>Boil the tubes in boiling water bath for 3 minutes</LI><LI> The absorption peak at 700nm of wavelength was measured by ultraviolet spectrophotometer.</LI></OL></DIV><DIV data-special="protocol" data-heading="Analytical procedure"><OL><LI>Set-up the reactions in duplicate as follows: 
			    <TABLE><TBODY><TR><TH>cell suspension with an OD600 of 100</TH><TD>26µL</TD></TR><TR><TH>TRIS-HCl 400mM,pH 8.5</TH><TD>474µL</TD></TR><TR><TH>substrate solution</TH><TD>25µL</TD></TR></TBODY></TABLE><P>Note:</P></LI><LI>Substrate solution is acetonitrile/water 1:1 mixed with 21 mM xanthine and addition of 1M NaOH until solubilization.</LI><LI>Control incubations were performed without substrate addition.</LI><LI>The analytical biotransformations were performed in 1.5ml centrifuge tube at 37°C and 1100 rpm for 6h using an Eppendorf Thermoshaker. </LI><LI>The reactions were stopped by the addition of 200μL acetonitrile/methanol 1:1(v:v)</LI><LI>After 5 min of mixing the vials were centrifuged for 10 min with 15000r/min at 4°C.</LI><LI>The supernatants were analysed by HPLC.</LI></OL></DIV><DIV data-special="protocol" data-heading="Digestion"><OL><LI>Set-up the reaction in 1.5mL centrifuge tubes as follows.
			    <TABLE><TBODY><TR><TH>Restriction enzyme I</TH><TD>0.5µL</TD></TR><TR><TH>Restriction enzyme II</TH><TD>0.5µL</TD></TR><TR><TH>Plasmid (PCR product)</TH><TD>2 mg (500 ng)</TD></TR><TR><TH>10x Cutsmart</TH><TD>15µL(5µL)</TD></TR><TR><TH>H2O</TH><TD>To 150µL(50µL)</TD></TR></TBODY></TABLE></LI><LI>The plasmid (PCR product) digestion reactions were incubated at 37°C for 1h(2-3h).</LI><LI>Heat inactivate by incubating at 65°C for 20 minutes.</LI></OL></DIV><DIV data-special="protocol" data-heading="HPLC method for adenine"><TABLE><TBODY><TR><TH>Flowing condition</TH><TD>H2O with 1% TFA as solvent A and methanol with 1% TFA as solvent B</TD></TR><TR><TH>Gradient</TH><TD><UL><LI>0-5min 10% solvent B 90% solvent A</LI><LI>5-8min 10% solvent A 90% solvent B</LI><LI>8-10min 10% solvent B 90% solvent A</LI></UL></TD></TR><TR><TH>Flow rate</TH><TD>1.2mL/min</TD></TR><TR><TH>Injection volume</TH><TD>20μL</TD></TR><TR><TH>Column temperature</TH><TD>not in control </TD></TR><TR><TH>Detection wavelength</TH><TD>254nm. </TD></TR></TBODY></TABLE><H4>HPLC method for xanthine and uric acid</H4><TABLE><TBODY><TR><TH>Flowing condition</TH><TD>H2O with 1% TFA as solvent A and methanol with 1% TFA as solvent B</TD></TR><TR><TH>Gradient</TH><TD><UL><LI>0-5min 10% solvent B 90% solvent A</LI><LI>5-8min 10% solvent A 90% solvent B</LI><LI>8-10min 10% solvent B 90% solvent A</LI></UL></TD></TR><TR><TH>Flow rate</TH><TD>1.2mL/min</TD></TR><TR><TH>Injection volume</TH><TD>20μL</TD></TR><TR><TH>Detection wavelength</TH><TD> 275nm&amp;254nm.</TD></TR></TBODY></TABLE></DIV><DIV data-special="protocol" data-heading="Ligation"><OL><LI>Set-up the reaction in microcentrifuge tubes as follows: 
			    <TABLE><TBODY><TR><TH>Vector DNA</TH><TD>0.02 pmol</TD></TR><TR><TH>Insert DNA</TH><TD>0.06 pmol</TD></TR><TR><TH>T4 Ligase Reaction Buffer (10X)</TH><TD>2 µl</TD></TR><TR><TH>T4 Ligase</TH><TD>1 µl</TD></TR><TR><TH>H2O</TH><TD>H2O	up to 20 µl</TD></TR></TBODY></TABLE></LI><LI>Incubate the tubes at room temperature for 1 hour or at 16°C overnight.</LI><LI>Heat inactivate by incubating at 65°C for 20 minutes.</LI></OL></DIV><DIV data-special="protocol" data-heading="Polymerase Chain Reaction Protocol"><OL><LI>Choosing Target Substrates and PCR Primers
			    <P>Possible target substrate:</P></LI><LI>Single Colonies</LI><LI>Bacteria Solution</LI><LI>Plasmid</LI><LI>DNA Fragments</LI><P>Design PCR Primers according to the purpose of the experiment</P><LI><P>The choice of the reaction system is dictated by the specific experiment</P></LI><LI>Taq System: to primarily validate the DNA sequence.</LI><LI>KOD System: to amplificate long DNA fragments in high fidelity.</LI><LI>Golden Mix System: for rapid amplification of DNA fragments in high fidelity.</LI><P>Set up the reaction system according to the protocol of the product you use</P><UL><LI>If the target substrate is single colony, use pipette to pick up a minute quantity of bacteria. Inoculate the bacteria onto a plate for further use. Then pipet up and down for several times in the solution.</LI><LI>If the target substrate is liquid, add proper quantity of substrate according to the protocol of specific reaction system.</LI></UL><LI>Choosing the Reaction Conditions
			    <P>The choice of reaction conditions mainly depends on the reaction system you choose.</P></LI><LI>Validating the Reaction
			    </LI><LI>Use agarose gel electrophoresis of DNA to validate the reaction.</LI><LI>If the result of agarose gel electrophoresis of DNA is primarily positive, send the sample for DNA sequence to confirm the result if necessary.</LI><LI>If the sequence is confirmed, reproduce and preserve the target bacteria strain for further use.</LI></OL></DIV><DIV data-special="protocol" data-heading="Purification of Ferritin"><OL><LI>Express and harvest cells. Resuspend cells in 10mM phosphate buffer (pH=7) in 15mL centrifuge tubes. </LI><LI>Freeze thawing the cells with liquid nitrogen.</LI><LI>Unfreeze the cells in 37°C water bath. </LI><LI>Use Ultrasonication Extraction to get cell lysis solution. Transfer cell lysis solution into 1.5mL centrifuge tubes.</LI><LI>Put the tubes into 60°C drying oven. </LI><LI>After 1 hour, centrifuge the tubes in 1.5mL centrifuge for 10min with 13000r/min at 4°C.</LI><LI>Collect the supernatant and transfer 700μL to each 1.5mL centrifuge tube.</LI><LI>Add 700μL saturated (NH4)2SO4 </LI><LI>After a few minutes, centrifuge for 5min with 10000r/min. </LI><LI>Collect the supernatant and add 10μL 1M phosphate buffer (pH=7).</LI></OL></DIV><DIV data-special="protocol" data-heading="Purification of Protein with His_tag"><OL><LI>Express and harvest cells. Resuspend cells in Lysis Buffer in 15mL centrifuge tubes.
			    <P>Lysis Buffer:</P><TABLE><TBODY><TR><TH>Stoke B (Na2HPO4 200mM + NaCl 5M)</TH><TD>60mL</TD></TR><TR><TH>Stoke A (NaH2PO4 200mM + NaCl 5M)</TH><TD>40mL</TD></TR><TR><TH>H2O</TH><TD>To 1L</TD></TR></TBODY></TABLE></LI><LI>Freeze thawing the cells with liquid nitrogen.</LI><LI>Unfreeze the cells in 37°C water bath.</LI><LI>Use Ultrasonication Extraction to get cell lysis solution.</LI><LI>Centrifuge the cell lysis solution in 1.5mL .centrifuge for 10min with 13000r/min at 4°C.</LI><LI>Collect the supernatant and transfer it to a injection syringe to bind with nickel column for 1 hour in 4°C.</LI><LI>Dilute the elusion buffer from 500mM imidazole to 50mM imidazole.
			    <P>Elution Buffer:</P><TABLE><TBODY><TR><TH>Stoke B (Na2HPO4 200mM + NaCl 5M)</TH><TD>60mL</TD></TR><TR><TH>Stoke A (NaH2PO4 200mM + NaCl 5M)</TH><TD>40mL</TD></TR><TR><TH>imidazole</TH><TD>0.5mol</TD></TR><TR><TH>H2O</TH><TD>To 1L</TD></TR></TBODY></TABLE></LI><LI>Flow down the liquid in the injection syringe</LI><LI>Add the diluted elusion buffer, then flow down the liquid as well.</LI><LI>Repeat step 9.</LI><LI>Add the elusion buffer, and then preserve the flow down liquid in 4°C.</LI></OL></DIV><DIV data-special="protocol" data-heading="SDS-PAGE Gel Electrophoresis"><OL><LI>Prepare the solution as follow to make 15% separation gel 
			    <TABLE><TBODY><TR><TH>ddH2O</TH><TD>6.8mL</TD></TR><TR><TH>30%Acr-Bis(29:1)</TH><TD>15mL</TD></TR><TR><TH>1.5M Tris, pH8.8</TH><TD>7.6mL</TD></TR><TR><TH>10%SDS</TH><TD>0.3mL</TD></TR><TR><TH>10%APS</TH><TD>0.3mL</TD></TR><TR><TH>TEMED</TH><TD>0.012mL</TD></TR></TBODY></TABLE></LI><LI>Pour the solution into a gel mould. And then, we take advantage of the density of H2O to press separation gel to be flat. </LI><LI>After about 30 minutes, pour out the upper H2O and prepare the solution as follow to make 5% spacer gel 
			     <TABLE><TBODY><TR><TH>ddH2O</TH><TD>6.8mL</TD></TR><TR><TH>30%Acr-Bis(29:1)</TH><TD>1.7mL</TD></TR><TR><TH>1.5M Tris, pH8.8</TH><TD>1.25mL</TD></TR><TR><TH>10%SDS</TH><TD>0.1mL</TD></TR><TR><TH>10%APS</TH><TD>0.1mL</TD></TR><TR><TH>TEMED</TH><TD>0.01mL</TD></TR></TBODY></TABLE></LI><LI>After 1 hour, transfer the SDS-PAGE gel to a tank, remove the comb and apply:  5μL marker ; 10μL protein samples with 5μL loading dye. 
			    <P>Electrophoresis buffers:</P><TABLE><TBODY><TR><TH>Tris</TH><TD>3g</TD></TR><TR><TH>Gly</TH><TD>19g</TD></TR><TR><TH>SDS</TH><TD>1g</TD></TR><TR><TH>H2O</TH><TD>To 1L</TD></TR></TBODY></TABLE></LI><LI>Run the gel for 45-60 minutes at 230V</LI></OL></DIV><DIV data-special="protocol" data-heading="Spread Plate Method of Isolation"><OL><LI>All the operations should be sterile and completed in the super clean bench.</LI><LI>Prepare some sterile Petri dishes with solid culture medium.</LI><LI>Soak the glass spreader in the Ethyl Alcohol.</LI><LI>Pass the spreader through the alcohol burner flame to ignite the alcohol. Wait for the alcohol to burn away. Put the glass spreader to cool it down.</LI><LI>Pipet 100μL bacteria solution into the plate. Use the glass spreader to spread the bacteria solution till the plate is dry.</LI><LI>Burn the spreader in the alcohol burner flame. Soak it back in the Ethyl Alcohol.</LI><LI>Incubate plate. If the bacteria grow into few colony, centrifuge the bacteria solution for 1 min at 10,000×g, discard part of the supernatant, vortex or pipet up and down to mix thoroughly, spread again. If the bacteria fail to isolate, dilute the bacteria solution with liquid culture medium, spread again, or apply streak plate method.</LI></OL></DIV><DIV data-special="protocol" data-heading="Streak Plate Method of Isolation"><OL><LI>All the operations should be sterile and completed in the super clean bench.</LI><LI>Prepare some sterile Petri dishes with solid culture medium.</LI><LI>Burn the inoculating loop in the alcohol burner flame till the loop glows. Put the glass spreader to cool it down.</LI><LI>Dip the loop in the bacteria solution. Streak on the plate for several time.</LI><LI>Repeat step 3 to sterilize the loop.</LI><LI>Streak on the plate for several times from the former lines to another direction.</LI><LI>Repeat step 5 and step 6 for 2-3 times.</LI><LI>Incubate plate.</LI><LI>If the bacteria fail to isolate, dilute the bacteria solution with liquid culture medium, streak again.</LI></OL></DIV><DIV data-special="protocol" data-heading="Transformation Protocol"><OL><LI>All the operations should be sterile and completed in the super clean bench.</LI><LI>Fill lab ice bucket with ice.Thaw competent cells on ice for 15-20 min for a 100µl stock. Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.</LI><LI>Pipette 50µl of competent cells into 1.5ml tube. Keep all tubes on ice.</LI><LI>Pipette 1-10µl of resuspended DNA into 1.5ml tube. Gently pipette up and down a few times. Keep all tubes on ice. The quantity of the resuspended DNA depends on the concentration. Remember the quantity of DNA should not be excess.</LI><LI>Close 1.5ml tubes, incubate on ice for 30min.</LI><LI>Heat shock tubes at 42°C for 1 min. Timing is critical.</LI><LI>Incubate on ice for 2min.</LI><LI>Pipette 950µl liquid culture medium without resistence to each transformation</LI><LI>Incubate at 37°C for 1 hours, shaking at 180rpm</LI><LI>Spread 150µl on a resistent plate for isolution. Incubate plate overnight.</LI><LI>Colony PCR.</LI><LI>Agarose gel electrophoresis of DNA.</LI><LI>Sequence the PCR product or the bactaria solution.</LI><LI>Preserve the target bacteria for further use.</LI></OL></DIV></DIV><DIV id="UCASLab.Notebook"><P>click <A data-jumpto="https://static.igem.org/mediawiki/2017/0/03/T--UCAS--NOTEBOOK.pdf">here</A> to read full pdf.</P><STRONG>INDEX:</STRONG><H4>1.NOTES FOR PROJECT</H4><UL><LI>Mechanism of NtrC/B system</LI><LI>Mechanism of SsrA tag</LI><LI>Details about Anammox and AOB</LI><LI>Protein information</LI></UL><H4>2.NOTEBOOKS OF EXPERIMENTS</H4><UL><LI>Sensor and Other Collaborations</LI><LI>Fusion Protein</LI><LI>DNA Scaffold</LI><LI>Collaboration with ZJU</LI></UL></DIV><DIV id="UCASLab.Interlab"><H3>Summary</H3><P>InterLab is a global lab cooperation which aims to gain an objective and universal accepted evaluation to some modeled parts for engineering. All labs participating in InterLab measure the OD and the fluorescence following the same protocol with the same type of instruments, resulting in a large reduction in error and an easily building global comparing system. Plus, the idea of the InterLab study also enables a common measurement platform in terms of samples and protocols so as to allow interconversion of data between different devices in the following methods.  </P><H3>Procedure</H3><P>OD: A standard colloidal solution, LUDOX S30, was provided in 2017 measurement kit.</P><P>Fluorescence: Dry Fluorescence Sodium salt (100µM by resuspending in 1ml 1 x PBS ). Fluorescence was measured under the standard condition(EX:483nm  Em: 507nm)</P><P>Use of standardized measurements:</P><P>	A gcolumnth curve for E coli DH5α transformed with the following plasmids was performed</P><UL><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_I20270">Positive Control (BBa_I20270): well 20B</A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_R0040">Negative Control (BBa_R0040): well 20D</A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_J364000">Test Device 1 (BBa_J364000): well 20F</A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_J364001">Test Device 2 (BBa_J364001): well 20H </A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_J364002"> Test Device 3 (BBa_J364002): well 20J</A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_J364003">Test Device 4 (BBa_J364003): well 20L</A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_J364004">Test Device 5 (BBa_J364004): well 20N </A></LI><LI><A data-jumpto="http://parts.igem.org/wiki/index.php?title=Part:BBa_J364005">Test Device 6 (BBa_J364005): well 20P</A></LI></UL><SMALL>All the gcolumnth curves were performed with initial uncorrected OD of 0.2</SMALL><P>Error: It is hard for us to perform the standard fluorescence measurement because the detection limit for our plate reader is up to 100000 which correspond to nearly 0.78µM fluorescence. 
		</P><H3>Results</H3><SMALL>Fig 1: A standard curve of fluorescein.</SMALL><SMALL>Fig 2: The absorbance of the <I>E. coli</I> with standard plasmids incubating in LB broth. The absorbance was measured every two hours in the cytation palte reader.</SMALL><SMALL>Fig 3: The fluorescence readings of the <I>E. coli</I> with standard plasmids incubating in LB broth. The absorbance was measured every two hours in the cytation plate reader. The Device 1 performs a too high fluorescent intensity that makes our plate reader overflow.</SMALL></DIV><DIV id="UCASLab.Contributions"><P>Among the requires for Bronze Medal, we chose to complete InterLab instead of Contribution, see more details in <A data-jumpto="ucaslabinterlab">InterLab</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>