"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/Our Project/Methods/
From 2010.igem.org
iGEM Yale
home
our project
our team
our project
introduction
methods
plasmid
H2S production
cu growth assay
cu localization
parts
lab notebook
safety
applications
experimental methods
The experimental methods used for generation of plasmid constructs containing IGEM parts registry promoter (insert name here) as well as the phsABC thiosulfate reductase gene and the terminator are described as follows. The vector containing phsABC was provided to us by the Keasling Lab. This vector underwent two successive single digestions in which each digestion was allowed to proceed overnight. This deviation from the standard double digestion was necessary to insure successful cutting of the phsABC insert out of the provided plasmid. Following these digestions, the DNA components in the digestion reaction solution were analyzed via gel electrophoresis and the dna band that corresponded to the length of the phsABC insert was excised from the gel and gel purified. Plasmid pSB74 which has been shown to posses the highest activity, was also digested and purified in a manner similar to what is described above for the insert. These purified dna segments were then combined in a ligation reaction that used T4 dna ligase to combine the sticky ends of the degenerate restriction sites, allowing for the formation of phsABC insert containing pSB74 plasmid. The resultant plasmid was sequenced and then transformed to increase its concentration. The transformed cells were then miniprepped using the Qiagen miniprep system.   (sequencing) (resistance) This plasmid then underwent an very similar series of steps to insert the promoter and regulatory biobricks. Plasmid Production for Thiosulfate reducase PHS gene: Important part of anaerobic bacterial respiration/metabolism • Studied since 1970's; several plasmids found (ie: phs pEB40, pSB103, pSB77, pSB107 etc) • Thiosulfate reductase is a transmembrane protein involved in the second step of the Sulfate-Reducing Bacteria pathway of thiosulfate reduction. Thiosulfate reductase catalyzes the dissimilatory reduction of inorganic thiosulfate to hydrogen sulfide and sulfite. • 1995, phs locus of Salmonella chromosome located, sequenced, and confered H2S production to E.Coli. – Sequence similar to reductases in other bacteria – Later confirmed to contain structural gene for thiosulfate reductase • Since before 1970 people have observed bacteria producing H2S. The earliest samples were taken from patients (Stoleru and Bouanehaud of Institut Pasteur, 1972 and 1975). Better yet, they showed that H2S production was mediated by a plasmid! A later study in 1978 by Jones et al of Texas Tech characterized another one of these plasmids, saw that it could be used in and transferred between E. Coli as well! • In 1987 an extensive review was written collecting lots of existing research about sulfate-reducing bacteria (SFB). In this broad review of SFB, it mentioned that sulfate reductases are membrane bound. Possibly useful for us, purified samples of thiosulfate reductase can also apparently be controlled, stopped in the presence of NADH , NADPH, and cysteine. Anyhow, the main development in this paper showed that thiosulfate-reduction ability is fairly common, usually used by anaerobic bacteria as an energy source (although some aerobic variants have been reported) and that one of the key players is thiosulfate reductase, which by this point had been isolated and had some characterization available. If required, there are lots of other sulfur-reducing pathways that are common as well, with a notable one being the reduction of tetrathionate. • Perhaps not surprising, the same authors of the review paper actually in the same year published a paper on the phs gene mediating hydrogen-sulfide production! Better yet, this gene is not coupled to methyl-viologen, which is common in anaerobic bacteria! score! Of note, the authors write that the phs gene is not actually the structural gene itself, but instead codes for a regulatory protein important for the reduction of thiosulfate to H2S. This paper also looked at different ways to optimize H2S production. • Something happened over a decade (particularly, see 1995 paper by Erika Barrett analyzing the phs sequence), because by 1999 a study from UC Berkeley, in Keasling lab, had the actual structural genes! In a couple of studies (1999-2000) they optimized the phs H2S production system to produce lots of H2S and remove heavy metals from solution! • Keasling lab, used phs gene sequences transformed into DH5α for bioremediation studies (2000). •  • Optimized reading frame and plasmid for H2S production • Optimized conditions for precipitation of metals •   (zinc, lead, and cadmium) Thus, our team created a phsABC vector in e.coli bacteria strands, and successfully optimized copper deposition using varying concentrations of revalant media.
Retrieved from " http://2010.igem.org/Team:Yale/Our_Project/Methods/ "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
