<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Yale_Our_Project_Notebook_Week_4"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Yale/Our_Project/Notebook/Week_4">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Yale/Our_Project/Notebook/Week_4&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_4&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_4&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Yale/Our_Project/Notebook/Week_4" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Yale/Our Project/Notebook/Week 4</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>iGEM Yale</TITLE></P><DIV id="wikiwrapper"><DIV id="wikiheader"><UL id="nav"><LI><A href="https://2010.igem.org/Team:Yale">home</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project">our project</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Team">our team</A></LI></UL></DIV></DIV><DIV id="main-content"><DIV id="left-col"><H5>
our project
</H5><P><UL id="proj-nav"><LI><A href="https://2010.igem.org/Team:Yale/Our Project">introduction</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Applications">applications</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Methods">methods</A></LI><LI><B><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook">lab notebook</A></B></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook">week 1</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 2">week 2</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 3">week 3</A></LI><LI id="nb"><B><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 4">week 4</A></B></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 5">week 5</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 6">week 6</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 7">week 7</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 8">week 8</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 9">week 9</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Protocols">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Safety">safety</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Future Work">future work</A></LI></UL></P></DIV><DIV id="right-col"><H5>
lab notebook: week 4 (6/28 -7/4)
</H5><P><UL><LI>

Monday 6/28--PCR amplification of thiosulfate reductase operon phsABC

</LI><A id="link" href="javascript:ReverseDisplay('monday')">See more/less</A><DIV style="display:none;" id="monday"><B> pSB74 transformants</B>
Extensive colony growth was seen on plated transformations of pSB74 into DH5alpha and LE392 from <A href="https://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_3"> 6/25 </A> but none was visible on BL21 plate.  Continued failure of BL21 transformation suggests that the homemade competent BL21 cells being used are somehow faulty.  </DIV></UL></P><H4> PCR amplification of phsABC </H4><UL><LI> Prior to PCR, took aliquots of all primer samples and diluted them first to 100 uM as detailed below by the chart and then again to 10 uM </LI><TABLE><TBODY><TR><TD>Primer</TD><TD> phsABC_F </TD><TD> phsABC_R </TD><TD> phsAB_R </TD><TD> phsC_F </TD></TR><TR><TD> Initial Amount </TD><TD>38.56 nm</TD><TD>46.58 nm</TD><TD>29.03 nm</TD><TD>21.94 nm</TD></TR><TR><TD> Volume of water </TD><TD>385.6 uL</TD><TD>465.8 uL</TD><TD>290.3 uL</TD><TD>219.4 uL</TD></TR></TBODY></TABLE><LI> Similarly, diluted the DNA sample to 10 ng DNA/uL.  Started with 0.5 uL of miniprepped pSB74 (sample a) at 67.3 ng/uL and added 2.865 uL of water to get the desired concentration.</LI><B> PCR reaction </B><LI> Added the following to each of three PCR tubes: 10 uL 5x Phusion HF buffer, 27.5 uL water, and 1 uL of dNTPs.  5 uL were added of both the forward and reverse primers (at 10 uM), as was 1 uL of the template DNA. The matching of tube numbers, template DNA, and primers was as follows:</LI><TABLE><TBODY><TR><TD> Tube #</TD><TD>2</TD><TD>3</TD><TD> 4</TD></TR><TR><TD> Template DNA</TD><TD>phsC</TD><TD>phsAB</TD><TD> phsABC</TD></TR><TR><TD>Primers</TD><TD>phsC_F &amp; phsC_R</TD><TD>phsABC_F &amp; phsAB_R</TD><TD> phsABC_F &amp; phsC_R</TD></TR></TBODY></TABLE><LI> Chilled tubes on ice, then added 0.5 uL of Phusion DNA polymerase and put in thermocycler. </LI><LI> Prior to designing thermocycler protocol, used oligocalc to get the following Tm values for the primers. </LI><TABLE><TBODY><TR><TD> Primer </TD><TD>phsABC_R </TD><TD>phsABC_F </TD><TD>phsAB_R </TD><TD>phsC_F </TD></TR><TR><TD> Tm </TD><TD>73˚C </TD><TD>66˚C</TD><TD>64˚C</TD><TD>72˚C </TD></TR></TBODY></TABLE>
However, these values seem excessively high, so will just start out with an annealing temperature of 55˚C.  Devised the following &quot;phs&quot; Thermocycler Protocol
1. Initial Denaturation-- 2 minutes at 98˚C
2. Denaturation--15 seconds at 98˚C
3. Annealing--15 seconds at 55˚C
4. Extension--3 minutes at 72˚C 
Steps 2-4 are repeated for 25 cycles
5. Final Extension--10 minutes at 72˚C 
6. Hold--indefinitely at 4˚C<B> Gel of PCR Products </B>
Ran PCR products on an 0.8% agarose gel with 10 uL of ethidium bromide.  Loaded 1 uL of each sample with 9 uL of water and 2 uL of loading buffer.  Ran gel until done at 60 V, but had difficulty imaging so left aside to deal visualize the next day, unaware that it would spread. </UL><I>This day's labwork is also recorded on pages 23-28 of the hard copy lab notebook </I></DIV><LI>

Tuesday 6/29--Gel extraction of PCR amplified phs, TSI agar slant assay for hydrogen sulfide production, &amp; pre-ligation double digestion
</LI><A id="link" href="javascript:ReverseDisplay('tuesday')">See more/less</A><DIV style="display:none;" id="tuesday"><H4>PCR amplification of phs </H4><LI> Because the gel run the previous day was no longer good, had to run a second 0.8% agarose gel of the PCR products to determine the success of the PCR reaction. Below is the image of the gel, with a 1 kb ladder in the leftmost lane, phsC in next lane, phsAB in lane 3, and phsABC in lane 4. </LI><DIV id="caption">
The phsC fragment seems to have been amplified with decent success, but phsABC and phAB  are only present in trace amounts, so the thermocycler protocol will have to be tweaked.  </DIV><LI> In the mean time, gel extract the phsC from lane 2 and the phsABC from lane 4 using the microcentrifuge variant of the  <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols/gel_extract">standard gel extraction protocol </A></LI><H4> TSI Agar Slant Assay </H4><LI> Want to confirm that IPTG will induce H<SUB>2</SUB>S production in pSB74 transformants, so inoculate 5 mL Amp/LB liquid cultures of DH5alpha and LE392 transformed with pSB74 and grow them up on shaker at 37˚C for use in TSI agar assay. </LI><LI> The slants don't contain IPTG, so must inject IPTG with the transformants into the slants. Took 200 uL of each cell solution and added to it 50 uL of 1 M IPTG, then injected the resulting mixture into a slant.  Left the slants to sit overnight in the fume hood </LI><H4> Double digestion in preparation for ligation </H4>
In order to create the desired Biobricks, must insert the phsABC operon into terminator B0015 and put the constitutive promoter J23114 into the generator P0312.  Following the Biobrick combination protocol, we then must digest phsABC and J23114 with EcoRI and SpeI and digest B0015 and P0312 with EcoRI and XbaI.  The former can be done as a double digest, but the latter must be done sequentially because the enzymes require different buffers.
Set up the following 50 uL digestions:<B>Digest of phsABC</B>--5 uL EcoRI buffer, 0.5 uL 100x BSA, 40 uL phsABC fragment (from gel extraction), 1.8 uL EcoRI, 1.8 uL SpeI, 0.9 uL water <B>Digest of J23114</B>--5 uL EcoRI buffer, 0.5 uL 100x BSA, 8 uL miniprepped J23114 (1 ug DNA), 1.8 uL EcoRI , 1.8 uL SpeI, 32.9 uL water <B>Digest of B0015</B>--5 uL EcoRI buffer,  12 uL miniprepped B0015 (1 ug DNA), 3.6 uL EcoRI , 29.4 uL water <B>Digest of P0312</B>--5 uL EcoRI buffer,  9 uL miniprepped P0312 (1 ug DNA), 3.6 uL EcoRI , 32.4 uL water 
Let each digestion run for two hours at 37˚C, then ran the result through the microcentrifuge version of the <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols/PCR_purify">PCR purification protocol.</A><I>This day's labwork is also recorded on pages 28-31 of the hard copy lab notebook </I></DIV><LI>

Wednesday 6/30--Troubleshooting PCR amplification of thiosulfate reductase gene &amp; continued work toward ligation

</LI><A id="link" href="javascript:ReverseDisplay('wednesday')">See more/less</A><DIV style="display:none;" id="wednesday"><B>TSI Agar slant </B><UL><LI>No visible color change, and decide that it is safe to accelerate growth by moving slants to the 37˚C incubator as research fails to turn up any safety concerns regarding the hydrogen sulfide emitted in this kind of assay. </LI></UL><B>PCR efforts continue </B><UL><LI>Redid 6/29 PCR of phsABC and phsAB, but this time ran two trials of each, with and without 3% DMSO, and changed the thermocycler protocol so that the annealing temperature was 50˚C rather than 55˚C, calling the new protocol &quot;phs50.&quot;</LI><LI> After running the protocol, found that PCR tubes had popped open and contents had evaporated, so set up PCR a second time to run overnight.</LI></UL><H4>Vector digestion &amp; 1st ligation attempt</H4><UL><LI> Ran the second portion of the B0015 and P0312 sequential digests, with reaction contents of each as follows: 5 uL NEB Buffer 4, 0.5 uL 100x BSA, 3.6 uL XbaI, 40 uL DNA from PCR purification protocol, and 0.9 uL water.  Let the reactions run for 2.5 hours at 37˚, but after 1.5 hours added to each reaction 1 uL of Calf Intestinal Phosphatase (CIP) to cleave terminal phosphates and discourage self-ligation in the next step. </LI><B>Ligation of phsABC and B0015</B>
By this point the DNA concentrations of the digestion reactions are dismally low (around 2 ng/uL for B0015 and phsABC), but will try a ligation anyways while at the same time preparing for a possible redo. Set up digestion reactions of two sizes, 10 uL and 20 uL, with a control reaction for each in which there is no insert.<B> Reaction 1--small control</B> 1 uL ligase buffer, 0.5 uL T4 ligase, 4.25 uL B0015, 4.25 uL water<B> Reaction 2--small 1:1 insert:vector</B>1 uL ligase buffer, 0.5 uL T4 ligase, 4.25 uL B0015, 4.25 uL phsABC<B> Reaction 3--small 2:1 insert:vector</B>1 uL ligase buffer, 0.5 uL T4 ligase, 2.5 uL B0015, 6 uL phsABC<B> Reaction 4--large control</B>2 uL ligase buffer, 1 uL T4 ligase, 8.5 uL B0015, 8.5 uL water<B> Reaction 5--large 1:1 insert:vector</B>2 uL ligase buffer, 1 uL T4 ligase, 8.5 uL B0015, 8.5 uL phsABC
Insert to vector ratios are rough and based on the fact that the two are at similar concentrations and are of similar sizes. 
Each reaction was let to run 20 minutes at room temperature before being transformed into Xl1-Blue cells according to the standard <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols/transformation"> transformation protocol </A> and then plated on ampicillin LB plates and let to grow overnight.</UL><B> Cultures for miniprep &amp; glycerol stock redo </B>
In preparation for potential ligation failure, once again inoculated liquid cultures of Amp LB  for miniprep with pSB74,P0312,R0011, B0015, and J23114 transformants, making two 5 mL cultures for each plasmid.  When OD reached 0.6, once again made glycerol stocks from 0.5 mL of each solution and 0.5 mL of filter-sterilized 50% glycerol.  This time flash-froze the stocks immediately, rather than waiting overnight as on <A href="https://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_3">6/23</A><I> This day's labwork is also recorded on pages 32-36 of the hard copy lab notebook. </I></DIV><LI>

Thursday 7/1--Smelly bacteria (yay!) &amp; ongoing ligation efforts

</LI><A id="link" href="javascript:ReverseDisplay('thursday')">See more/less</A><DIV style="display:none;" id="thursday"><H4> TSI agar assay </H4><UL><LI>LE392 and DH5alpha cultures in TSI agar showed some growth. No black color change was visible, but both slants had a rotten egg scent, so suspected generation of H<SUB>2</SUB>S at concentrations too low to visibly trigger color change yet.</LI></UL><H4>PCR work</H4><UL><LI>Ran 1.0% agarose gel of four 6/30 PCR products at 90 V.</LI><DIV id="caption">
Gel of phsABC and phsAB PCR products or a demonstration of the utility of DMSO Lane 1: 1 kb ladder and 3.6 kb spillover from Lane 2, Lane 2: 3.6 kb phsABC fragment from PCR protocol without DMSO, Lane 4: 2.9 kb phsAB fragment from PCR protocol without DMSO, Lane 6:3.6 kb phsABC fragment from PCR protocol with DMSO, Lane 8: 2.9 kb phsAB fragment from PCR protocol with DMSO
      Gel of phsABC and phsAB PCR products or a demonstration of the utility of DMSO Lane 1: 1 kb ladder and 3.6 kb spillover from Lane 2, Lane 2: 3.6 kb phsABC fragment from PCR protocol without DMSO, Lane 4: 2.9 kb phsAB fragment from PCR protocol without DMSO, Lane 6:3.6 kb phsABC fragment from PCR protocol with DMSO, Lane 8: 2.9 kb phsAB fragment from PCR protocol with DMSO
</DIV><LI>Based on gel, it appears that the phs50 thermocycler protocol is an improvement from phs protocol and that including DMSO also improves yield</LI><LI>Cut out largest band for each lane and ran through <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols/gel_extract">standard gel extraction protocol.</A></LI></UL><B>Stockpiling plasmids for later</B><UL><LI><A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols/miniprep">Miniprepped </A> LE392 cultures grown overnight,resulting in two Eppendorfs apiece of the following plasmids: B0015 (double terminator), J23114 (constitutive promoter), pSB74 (thiosulfate reductase), R0011 (IPTG-inducible promoter), and P0312 (lacI needed with R0011). Nanodropping gave the following concentrations:</LI><TABLE><TBODY><TR><TD> Sample </TD><TD> B0015-1</TD><TD> B0015-2 </TD><TD>P0312-1</TD><TD> P0312-1</TD><TD> J23114-1 </TD><TD> J23114-2</TD><TD> pSB74-1</TD><TD> pSB74-2 </TD><TD> R0011-1 </TD><TD> R0011-2 </TD></TR><TR><TD> Concentration (ng DNA/uL) </TD><TD> 262.3</TD><TD> 28569</TD><TD>129.1</TD><TD> 92.3</TD><TD> 189.0 </TD><TD> 202.6</TD><TD> 214.4</TD><TD> 284.4 </TD><TD> 102.6</TD><TD> 79.6 </TD></TR></TBODY></TABLE></UL><H4>Results of 6/30 Ligation (Ligation Attempt #1)</H4><UL><LI>Plated post-ligation transformants showed the following results: </LI>

Reaction #1 (small control)	0 colonies
Reaction #2 (1:1 insert:vector) 	2 colonies, numbered 1A and 2A
Reaction #3 (2:1 insert:vector) 	0 colonies
Reaction #4 (large control) 	2 colonies
Reaction #5 (large 1:1 insert vector) 	2 colonies, numbered 3A and 4A

Concerned about the small number of colonies and low ratio of number of colonies from actual ligation to number of colonies from control ligation.<LI>Started liquid cultures and index plates of four colonies from reactions 2 and 5.</LI><LI>Also set up colony PCR for all four colonies, spotting the template DNA into tubes with the following contents and running on thermocycler protocol phs50 </LI><TABLE><TBODY><TR><TD>Component</TD><TD>Volume</TD></TR><TR><TD>Distilled Water</TD><TD>27 μL</TD></TR><TR><TD>Phusion 5x Buffer</TD><TD>10 μL</TD></TR><TR><TD>DMSO</TD><TD>1.5 μL</TD></TR><TR><TD>10 mM dNTPs </TD><TD>1 μL</TD></TR><TR><TD>10 uM phsABC_F primer</TD><TD>5 μL</TD></TR><TR><TD>10 uM phsABC_R primer </TD><TD>5 μL</TD></TR><TR><TD>Phusion Polymerase</TD><TD>0.5 μL</TD></TR><TR><TD>Template DNA </TD><TD>spotting</TD></TR><TR><TD>Total</TD><TD>50 μL</TD></TR></TBODY></TABLE><LI>After PCR, ran 1.0% agarose gel of colony PCR reaction solutions at 90 V versus a 1 kb ladder. A preliminary visualization showed no visible non-primer DNA except for a band between 3 and 4 kb for the PCR product from colony 1. Based on a later visualization said band appeared to be closer to 3 kb, suggesting the 3.2 kb B0015 vector rather than the 3.6 kb insert. Unfortunately gel was dropped and damaged irreparably before picture could actually be taken. </LI></UL><I>Wetlab work for this day is also described on pages 37-41 of the hard copy lab notebook </I></DIV><LI>

Friday 7/2--Confirmation of hydrogen sulfide production plus more ligation work

</LI><A id="link" href="javascript:ReverseDisplay('friday')">See more/less</A><DIV style="display:none;" id="friday"><H4> TSI Agar slants </H4><UL><LI>Slant inoculated with the LE392 transformants containing pSB74 showed a mass of black precipitate near the surface, definitively confirming the production of hydrogen sulfide (!), the smell of which was by now very strong. The DH5alpha transformants did not exhibit the same H<SUB>2</SUB>S odor.</LI><DIV id="caption">blurry (apologies!) image of LE392 slant showing FeS precipitate(right) next to un-inoculated slant (left)</DIV></UL><H4>Ligation Attempt #2</H4><UL><LI>Still don't know whether ligation attempt #1 was successful, as colony PCR on 7/1 was not definitive. Will sequence the results for a certain answer, but given the small number of colonies think it best to redo ligation in the interim in case of failure.</LI><LI>To this effect redid double digestion of phsABC insert with EcoRI and SpeI and the serial digestion of B0015 with EcoRI and then XbaI. Simultaneously digested J23114 to be inserted into P0312, which lacks a promoter. Each digestion reaction had a total volume of 100 uL and included 2 ng of DNA, double the amount used previously, so that even after purification there would still be a sizable amount of DNA remaining for ligation. Otherwise digestion and purification was run the same as on 6/29 &amp; 6/30. </LI></UL><I>This day's activities are also recorded on pages 42-45 of the hard copy lab notebook </I></DIV><LI>

Saturday 7/3--More attempts to put a terminator on thiosulfate reductase and a promoter on lacI

</LI><A id="link" href="javascript:ReverseDisplay('saturday')">See more/less</A><DIV style="display:none;" id="saturday"><H4>Ligation Attempt #2 continued</H4><UL><LI>Following digestion and purification, took concentrations of digestion products, which were still miserably low (phsABC: 3.2 ng/uL, P0312: 16.88 ng/uL, J23114: 23.98 ng/uL, B0015: 13.24 ng/uL) and set-up ligation attempt #2. For each of the two ligations (phsABC into B0015 and J23114 into P0312) four different 20 uL reactions were run: a control ligation with vector, ligase and buffer but no insert, a reaction with a 2:1 stoichiometric ratio of insert to vector, a reaction with a 3:1 insert to vector ratio, and a reaction with a 5:1 insert to vector ratio. All reactions used 1 uL NEB T4 ligase and 2 uL of accompanying 10x buffer and were run for 15 minutes at room temperature. Following this the (theoretically) ligated plasmids were transformed into competent XL1-Blue cells according to the <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols"> standard transformation protocol </A> and plated onto Amp plates</LI><LI>Index plates from ligation attempt #1 on 7/1 belatedly showed growth. </LI></UL><I>This day's activities are also recorded on page 45 of the hard copy lab notebook </I></DIV><LI>

Sunday 7/4--Ligation attempt #2 transformants

</LI><A id="link" href="javascript:ReverseDisplay('sunday')">See more/less</A><DIV style="display:none;" id="sunday"><H4> Transformed Colonies from Ligation Attempt #2</H4><UL><LI>Checked on plated transformants and observed the following: </LI><B>phsABC/B0015 system</B>	
Reaction #1 (control)	2 colonies	
Reaction #2 (2:1 insert:vector) 	4 colonies, numbered 1B-4B
Reaction #3 (2:1 insert:vector) 	5 colonies, numbered 5B-9B
Reaction #4 (large control) 	0 colonies<B>J23114/P0312 system	</B>
Reaction #5 (control) 	numerous* colonies 
Reaction #6 (2:1 insert:vector) 	numerous colonies, somewhat more than #5
Reaction #7 (3:1 insert:vector) 	numerous colonies,about double #5
Reaction #8 (5:1 insert:vector) 	numerous colonies, about double #5

numerous=more than readily countable<LI>Inoculated 5 mL liquid Amp/LB cultures with colonies 1B-9B and let grow overnight at 37 on shaker </LI><LI> Set aside plates 6-8 in the fridge to work with later. </LI></UL><I>This day's activities are also recorded on page 46 in the hard copy of the lab notebook. </I></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_4">http://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_4</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Yale/Our_Project/Notebook/Week_4" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Yale/Our_Project/Notebook/Week_4" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_4&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_4&amp;oldid=206092" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>