"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/Our Project/Notebook/Week 5
From 2010.igem.org
iGEM Yale
home
our project
our team
our project
introduction
applications
methods
lab notebook
week 1
week 2
week 3
week 4
week 5
week 6
week 7
week 8
week 9
protocols
safety
future work
lab notebook: week 5 (7/5 -7/11)
Monday 7/5--Colony PCR of Ligation Attempt #2 Transformants
See more/less
Colony PCR
Ran colony PCR of colonies 1B-9B from ligation attempt #2 according to the same protocol as used on 7/1 .
Then ran a 1.0% agarose gel of the nine PCR reaction solutions against a 1 kb ladder, but the resulting gel failed to show any evidence of either insert or vector, so will have to rely on sequencing and digests of miniprepped plasmids.
Gel of colony PCR from ligation attempt #2 Lane 1: 1 kb ladder, Lanes 3-11:PCR products from colonies 1B-9B, in order
Miniprepped liquid cultures grown from colonies of first ligation attempt (colonies 1A-4A) and second ligation attempt (colonies 1B-9B) according to standard miniprep protocol
This day's activities are also recorded on page 47 and 48 of the hard copy lab notebook.
Tuesday 7/6--Analysis of Results of Ligation Attempts 1 & 2
See more/less
More Diagnostics of Ligation Results
In order to determine success of ligation attempts 1 and 2, digested the miniprepped plasmids 1A-4A and 1B-9B from 7/5 with XbaI and SpeI and then looked for evidence of two fragments, one at 3.2 kb and the other at 3.6 kb. Reactions were conducted for 2 hours at 37°C, with the following contents for each of the thirteen reactions
Component
Volume
Distilled Water
6.4 μL
10x NEB Buffer 4
2 μL
DNA solution
10 μL
NEB XbaI
0.8 μL
NEB SpeI
0.8 μL
Total
20 μL
The diagnostic 1.0% agarose gel was run at 90 V vs. a 1 kb ladder. Each of the thirteen samples gave one diffuse band around 3 kb and no evidence of the 3.6 kb phsABC fragment, so both ligation attempts #1 and #2 were unsuccessful.
Gel of XbaI and SpeI double digest of plasmids from ligation attempts #1 and #2 Lane 1: 1 kb ladder, Lanes 2-5:1A-4A, Lanes 6-14: 1B-9B
During digestion prepped plasmid samples for sequencing. Each sample contained 3 μL of plasmid soln (average concentration of 96 ng/μL), 2 μL of 4 mM VF2 (forward primer), and 13 μL of distilled water. Intended to follow up with reverse sequencing of more promising samples, but gel results made it a moot point.
Wetlab work for this day is also recorded on pages 47, 49, and 50 in the hard copy lab notebook.
Wednesday 7/7--Triple ligation attempt--promoter + thiosulfate reductase + terminator
See more/less
Ligation Strategy
Failure of previous two ligation efforts prompted rethinking of ligation strategy. Will attempt a one-step triple ligation that will link phsABC to B0015 and insert them both into a standard single-resistance iGEM background. phsABC will be digested with EcoRI and SpeI, B0015 with XbaI and PstI, and pSB1C3 with EcoRI and PstI, allowing for them all to be ligated together.
Prep work
Needed more phsABC, so ran five PCR reactions to amplify it from plasmid pSB74. Reaction mixture contents were the same as the DMSO variant used on 6/30 as was the thermocycler protocol (phs50). The results were then run on a gel
Gel of PCR amplified phsABC Extreme right lane:1 kb ladder, all other lanes PCR reaction mixtures and their spillover
Needed chloramphenicol plates for pSB1C3-based constructs, so mixed 350 μL of 50 μg/μL chloramphenicol with an autoclaved solution of 7.5 g agar and 12.5 g LB in 500 mL H2O and poured plates.
Digestions
Ran all three digestion reactions for 30 minutes at 37 °C before heat killing for 20 minutes at 80 °C. Digestion mixtures had the following components:
Digestion Reaction Components
phsABC Digestion
B0015 Digestion
pSB1C3 Digestionhttps://2010.igem.org/Main_Page
5 μL EcoRI Buffer 10x
5 μL NEB Buffer 3 10x
5 μL EcoRI Buffer 10x
0.5 μL BSA 100x
0.5 μL BSA 100x
0.5 μL BSA 100x
21.1 μL phsABC soln (1 μg DNA)
3.8 μL B0015 soln (1 μg DNA)
15 μL phsABC soln (375 ng DNA)
1.8 μL EcoRI & 1.8 μL SpeI
1.8 μL XbaI & 1.8 μL PstI
1.2 μL EcoRI, 1.2 μL PstI, & 1.2 μL DpnI
20.8 μL H2O
37.1 μL H2O
15.9 μL H2O
Wetlab work for this day is also recorded on page 51 of the hard copy lab notebook.
Thursday 7/8--Further ligation efforts
See more/less
Ligation Efforts Continue
In addition to trying the triple-ligation strategy started on 7/7 (ligation attemp #3) will try trouble-shooting the ligation strategy used in attempts #1 and #2. In particular, will modify digestion protocol and test EcoRI and XbaI (older enzymes) for activity.
Ligation Attempt #3 (Triple Ligation)
Ran triple ligation all day at room temperature and stored at 4°C. Ligation contents were as follow to have a 1:1:1 stoichiometric ratio of all three components.
Component
Volume
Distilled Water
6.4 μL
T4 Ligase buffer 10x
2 μL
NEB T4 Ligase
1 μL
pSB1C3 solution
2 μL
B0015 solution
2.4 μL
phsABC solution
6.2 μL
Total
20 μL
Ligation Attempt #4
Concerns about enzyme activity led to reversing the order of the serial digestion of B0015 background so that the first digestion is XbaI (low salt buffer) and the second is EcoRI (high salt buffer). Will also omit the pre-ligation PCR purification step.
Overnight digestion of ligation components--Set up the following digestions to run overnight at 37°C. The EcoRI was newly purchased as there was concern about the age of the previously used enzyme.
phsABC double digestion
NEB EcoRI Buffer
5 μL
BSA 100x
0.5 μL
phsABC solution
30.8 μL (1 μg DNA)
NEB EcoRI
1.8 μL
NEB SpeI
1.8 μL
Sterile H2O
10.1 μL
B0015 digestion
NEB Buffer 4
5 μL
BSA 100x
0.5 μL
NEB XbaI
3.6 μL
B0015 solution
3.8 μL ( 1 μg DNA)
Sterile H2O
37.1 μL
Wetlab work for this day is also recorded on pages 52 and 53 of the hard copy lab notebook.
Friday 7/9--Ligation attempt #4 to join phsABC and B0015 terminator.
See more/less
Ligation Attempt #4
Removed both overnight digestions, the XbaI digestion of B0015 and the EcoRI and SpeI digestion of phsABC, from 7/8 from the incubator.
Heatkilled enzymes of phsABC digest with 20 minutes at 80°C.
Ran PCR purification protocol on XbaI digestion using microcentrifuge. In order to give a concentrated sample, eluted in only 35 μL of EB Buffer that had been heated at 50°C. Resulting concentration could not be determined as there was residual ethanol contamination.
Ran EcoRI digestion of purified XbaI digest product at 37 °C, with components as follows. Digestion was run for an hour before addition of CIP, after which it was run for another hour.
B0015 digestion with EcoRI
NEB EcoRI buffer
5 μL
BSA 100x
0.5 μL
NEB EcoRI
3.6 μL
B0015 digested with XbaI
25 μL
NEB Calf Intestinal Phosphatase
1 μL
Sterile H2O
15.9 μL
Followed with 20 minute heatkill at 80°C.
In order to set up ligation, need concentration of B0015, but ethanol contaminant continues to preclude measurement. Will estimate that the purification between digestion steps resulted in a 50% loss of DNA, leaving behind a 10 ng/μL concentration. Will also, as usual, run different ligation trials with different ratios of insert to vector, which should help combat any issues if this estimate is inaccurate
Set up the following ligation trials, with stoichiometric ratios of insert to vector (I:V) based on B0015 concentration estimate. Ran the ligations for 20 minutes at room temperature.
Ligation Reaction Components
Control Ligation
1:1 I:V
2:1 I:V
3:1 I:V
4:1 I:V
2 μL NEB T4 ligase buffer for all reactions
1 μL NEB T4 ligase for all reactions
5 μL B0015 solution for all reactions
12 μL H2O
8.9 μL
5.8 μL H2O
2.7 μL H2O
--
--
3.1 μL phsABC
6.2μL phsABC
9.3 μL phsABC
12 μL phsABC
Following ligation, transformed 1 μL of each ligation reaction into commercial grade cells according to standard transformation protocol. Also transformed triple ligation reaction.
Enzyme Activity Test
To test enzyme activity, will do single digests of B0015 with enzymes in question and then run them on a gel to see if the plasmid linearized. XbaI and EcoRI are old, so will test them first. For XbaI activity test, simply set aside 10 μL of digestion solution from overnight digestion. Testing of EcoRI required additional digests. Digests using both old and new EcoRI samples were run for two hours at 37 °C with the following components:
EcoRI Activity
NEB EcoRI buffer
5 μL
BSA 100x
0.5 μL
NEB EcoRI
3.6 μL
undigested B0015
3.8 μL ( 1 μg DNA)
Sterile H2O
37.1 μL
Followed with 20 minute heatkill at 80°C.
Loaded all three enzyme activity tests on a 1.0% agarose gel and ran them at 90 V versus a 1 kb ladder and circular B0015 plasmid.
Gel of single digests of B0015 Left to Right lane 1:1 kb ladder, lane 2: uncut circular B0015, its dimer, and lane 3 spillover, lane 3: XbaI digest of B0015, lane 4: old EcoRI digest of B0015, lane 5: new EcoRI digest of B0015
Based on the results, all enzymes appeared to work, as they successfully created a linearized 3.2 kb fragment.
Wetlab work for this day is also recorded on pages 54-57 of the hard copy lab notebook.
Saturday 7/10--Transformants from ligation attempts
See more/less
Transformants from ligation attempts #3 an #4
The plate from ligation attempt #3 shoed in excess of 100 differnt colonies.
Plates from ligation attempt #4 had the following colony counts:
Control-- 5 colonies
1:1--28 colonies
2:1--38 colonies
3:1--18 colonies
4:1--27 colonies
Took colonies from various plates and used them to establish index plates on Amp LB, labeled as follow: eight colonies from triple ligation as #1-#8, four colonies from 1:1 as#9-#12, four colonies from 2:1 as #13-#16, four colonies from 3:1 as #17-#20, and four colonies from 4:1 as #21-#24.
Wetlab work for this day is also recorded on page 58 of the hard copy lab notebook.
Sunday 7/11--Prep for copper growth assays of transformants
See more/less
Ligation work
Realize that mistakenly plated triple ligation reaction on ampicillin plate rather than the required chloramphenicol, so none of the colonies present are of value.  Fortunately still have transformation solution, so replate it, this time on a chloramphenicol
Inoculated overnight liquid cultures of ligation attempt #4 transformants  (#9-#24 on index plates) for miniprep the following morning
Growth assay prep work
Want to see if pSB74 confers any copper resistance on E. coli, since it should help them precipitate nearby copper and thus reduce the copper concentration that they experience.
Inoculated liquid cultures of  LE392 with pSB74 in ampicillin LB and untransformed LE392 in plain LB and left to grow overnight on shaker at 37˚C for growth assays the following day
Prepared CuSO4 solutions in LB.  Each well in the 96-well plate will have 225 uL of copper solution and 25 uL of cell solution, so the copper solution must have a CuSO4 concentration that is 10/9 the desired final values. The copper solutions must also contain ampicillin and IPTG at the same levels as the cultures that will be introduced. The desired  CuSO4 concentrations are those used in the wide and narrow concentration range trials on 6/10 and 6/14 .
Wetlab work for this day is also recorded on pages 59-61 of the hard copy lab notebook.
Retrieved from " http://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_5 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
