"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/Our Project/Notebook/Week 6
From 2010.igem.org
iGEM Yale
home
our project
our team
our project
introduction
applications
methods
lab notebook
week 1
week 2
week 3
week 4
week 5
week 6
week 7
week 8
week 9
protocols
safety
future work
lab notebook: week 6 (7/12-7/18)
Monday 7/12--Copper growth assay of pSB74 transformants & continued ligation work
See more/less
Copper Growth Assay Comparing LE392 With And Without pSB74
Measured the OD of the overnight LE392 cultures and found that the LE392 containing pSB74 had an OD of 0.58 while the untransformed culture had an OD of 0.87.  Wanted to let the transformed culture grow a while with IPTG prior to beginning of assay (to allow time for protein expression), but didn't want either culture to overgrow in the mean time, so diluted each as follows:
To 1 mL of the  untransformed solution added 4 mL of plain LB, while to 3 mL of the transformed LE392 added 2 mL of LB with Amp and 10 uL of IPTG, bringing the IPTG concentration to the desired 2 mM.  Let diluted cultures grow another hour at 37˚C to reach mid-log phase.
Following the second growth period, measured the OD of the untransformed culture as 0.75 while the OD of the pSB74 transformants was 0.88, but it was diluted to 0.75.
Finally the solutions were loaded into the 96-well plate as follows, with each row having 12 wells and copper concentration decreasing left to right as before.
Row A: Wide copper concentration range blank series with Amp & IPTG
Row B: Narrow copper concentration range blank series with Amp & IPTG
Row C: Wide copper concentration range transformant series with Amp & IPTG
Row D: Narrow copper concentration range transformant series with Amp & IPTG
Row E: Wide copper concentration range blank series
Row F: Narrow copper concentration range blank series
Row G: Wide copper concentration range untransformed series
Row H: Narrow copper concentration range untransformed series
Continuing ligation work
Miniprep and sequencing of attempt #4 transformants
Using the vacuum manifold protocol miniprepped the cultures grown up from the sixteen colonies (#9-#24) that resulted from the 4th ligation attempt.
Nanodropped three to get a sense of concentration and found that the average was 65.3 ng/uL
Prepared all sixteen samples for sequencing, taking 4 uL of each miniprep, 2 uL of 4 uM VF2 primer, and 12 uL of water.
Digest of miniprepped attempt #4 products
Digested each of the ligation products with EcoRI to linearize them prior to running them on a gel and determining their size. Each digestion reaction was composed of 5 uL of EcoRI buffer, 0.5 uL 100x BSA, 15 uL of plasmid DNA, 3.6 uL EcoRI, and 25.9 uL of water and was run overnight in the 37˚C incubator.
Colony PCR of ligation attempt #4 transformants
Ran colony PCR for each of the sixteen transformants, relying on the index plates as a cell source.  Spotted cells in 20 uL of water to lyse them, then added 1 uL of the resulting solution to a PCR tube.  To each tube also added 2 uL of VF2 primer (10 mM), 2 uL of VR primer (10 mM), 10.8 uL water, 4 uL 5x Phusion Buffer, 0.6 uL of DMSO, 0.4 uL dNTPs, and 0.2 uL of Phusion polymerase. Ran the reactions on the "VR" thermocycler protocol
Wetlab work for this day is also recorded on pages 59 and 62-64 of the hard copy lab notebook.
Tuesday 7/13--Transformant Copper growth assay, ligation attempt #4 results, and copper removal assay prep
See more/less
pSB74 transformant copper growth assay
Looked at results of the previous day's assay and saw that the presence of pSB74 actually decreased the copper tolerance of the LE392.  Maybe the effort associated with protein production actually weakens the cell's ability to deal with the copper?
Spotted each culture from the 96-well plate onto an agar plate (LB and ampicillin in the case of transformants, plain LB in the case of the untransformed) and put in incubator to see if it grows.
Ligation attempt #4 results
Ran 1.0% agarose gels of both the colony PCR reactions and EcoRI digestion of minipreps.  Gels contained 10 uL of ethidium bromide and were run at 90 V with a 1 kb ladder, but the power source malfunctioned and turned off at some point, so the gel sat for an unknown amount of time. Restarted power source, let run, and then visualized the gels
Digestion of miniprep shows that all ligation efforts failed--had they succeeded there would have been fragments at 7.8 kb, but as the gel below shows, all the samples run at slightly over  3 kb (ladder rungs are 500 bp, 1 kb, 1.5 kb, 2 kb, 3 kb, 4 kb, 5 kb, 6 kb, 8 kb, & 10 kb).
Efforts to visualize the colony PCR gel failed entirely--maybe forgot to add ethidium bromide? But given above results, it's a moot point.
Copper Removal Assay Prep
Want to run an assay measuring whether pSB74 transformants remove copper(II) from their surroundings.  If had a large bacterial culture with an intermediate CuSO4 concentration (say 500 uM), could periodically remove small aliquots, centrifuge out the cells, and spectrophotometrically determine the copper content.
Need to determine what wavelength should be used to monitor copper concentration and create a calibration curve relating absorbance at that wavelength to copper concentration.
Start by making serial dilutions of CuSO4 in LB.  Made samples of  0.1 M, 10 mM, 1 mM, 100 uM, 10 uM, 1 uM and 0.1 uM concentrations.
While spectrum had no clear peak, the CuSO4 did absorb strongly at 700 nm compared to just LB, so chose that as wavelength to monitor.  Attempted to  establish curve, but got the odd result that the 10 uM absorbed orders of magnitude more than the 100 uM.  Will redo dilutions and try again another day.
Inoculated 5 mL liquid cultures of LE392, both with and without pSB74, adding 5 uL of 1000x ampicillin to the transformant culture. Let grow overnight on shaker at 37˚C.
Wetlab work for this day is also recorded on pages 65-67 of the hard copy lab notebook.
Wednesday 7/14--Copper Removal assay work and 5th attempt to ligate phsABC into terminator B0015
See more/less
Digest/Ligation Attempt #5
Concerned that ligation components may not be thoroughly digested, so ran an all day digestion of phsABC, which containted 5 uL EcoRI buffer, 0.5 uL 100x BSA, 36.1 uL phsABC (at 27.7 ng/uL for 1 ug total), 1.8 uL EcoRI, 1.8 uL SpeI, and 4.8 uL water.
Also concerned about SpeI activity, so ran the following diagnostic SpeI digestion of B0015 that will then be run on a gel versus the circularized plasmid: 5 uL NEB buffer 4, 0.5 uL 100x BSA, 4 uL B0015(1 ug DNA), 3.6 uL SpeI, and 36.9 uL water, let to run for 2 hours at 37˚C before heat killing at 80˚C for twenty minutes
Simultaneously digested more B0015 with XbaI according to the same protocol as used on 7/8 to ensure there will be enough vector.
Purified the XbaI-digested B0015 with a standard PCR purification protocol , eluting in 40 uL, and then digested all of the result with 3.6 uL EcoRI, 5 uL EcoRI buffer, 0.5 uL 100x BSA, and 0.9 uL of water, letting it run for two hours. An hour into the digestion, added 1 uL of CIP to the reaction, and after the digestion heat-killed the enzymes with 20 minutes at 80˚C.
Copper(II) Absorbance Calibration Curve Redo
Once again carefully make serial dilutions of copper sulfate in LB, creating a series of solutions running from 0.1 M to 1 uM and separated by a power of ten.  Absorbances prove to be highly nonlinear again, so research and find that copper(II) concentration cannot directly be measured directly spectrophotometrically.
Bacterial Survival in Copper Solution
Retrieve from incubator plates spotted with copper solution cultures from 6/13 copper growth assay.  Find that all cultures up to and including 4 mM copper levels survived, both in the transformed and untransformed LE392.  Also see a colony were the  pSB74 transformant in 50 mM copper was spotted, but as there is no growth at 10 mM, wonder if there was an accidental drip during spotting.
Wetlab work for this day is also recorded on pages 67-68 of the hard copy lab notebook.
Thursday 7/15--Confirmation of SpeI activity and EtOH precipitation of ligation components
See more/less
Analysis of SpeI activity
Ran the SpeI digest of B0015 on a gel versus the circular B0015 and a 1 kb ladder (leftmost lane).  The digested plasmid (rightmost lane) ran slower than the undigested one (middle lane), confirming that SpeI is active and successfully linearized the plasmid.
Ethanol precipitation of ligation components
Guessing that higher concentrations of insert and vector might improve ligation results, ethanol precipitate the digested samples from 7/14 according to the following protocol
In case need more material, started five overnight liquid cultures of  each pSB74 and B0015 in Amp LB for morning miniprep.
Wetlab work for this day is also recorded on page 69 of the hard copy lab notebook.
Friday 7/16--Experimentation with different treatments of vector & insert DNA prior to ligation
See more/less
Stockpiling starting materials
Miniprepped the overnight cultures of pSB74 and B0015 by vacuum manifold and found the resulting concentrations by nanodrop.  The five B0015 samples had concentrations of 83.7, 83.4, 111.7, 130.0, and 103.3 ng/uL while the pSB74 samples had concentration values of 118.7, 129.5, 118.2, 137.3, and 124.9 ng/uL.
Set up 8 PCR reactions with pSB74 samples using the "phs50" thermocycler protocol and the DMSO variant of PCR reaction contents from 6/30 .
Also ran four digestions of B0015 with XbaI, each of which included 5 uL NEB buffer 4, 0.5 uL 100x BSA, 3.6 uL XbaI, 7.7 uL B0015 (1 ug worth), and 33.2 uL of water.  Let these run for two hours at 37˚C
Meanwhile attempted to resuspend ethanol precipitated DNA, but nanodrop showed no DNA present--pellets must have fallen out of tubes when inverted to dry.
After XbaI digestion of B0015 skipped PCR purification step (concerned that it was leading to product loss)and simply added 0.5 uL of 5 M NaCl to each digestion so that the buffer solution would have the salt content required by EcoRI.  Then added 2 uL of EcoRI to each digestion and let incubate for 2 hours at 37˚C.  One hour into this digestion, added 1 uL of CIP to half (two) of the digestion reactions (want to see if it makes a difference).
Following the PCR amplification of phsABC, ran the vacuum manifold PCR cleanup protocol on four out of the eight reaction solutions and measured the DNA concentrations of the resulting solutions as 99.5, 131.2, 75.5, 36.2, and 26.2 ng/uL.
Concerned that phsABC may not be getting digested properly, especially given that its cut sites are so near its ends.  While running the required EcoRI/SpeI double digestion, will run in parallel single digestions of phsABC with each EcoRI and SpeI.  While the amount cut off makes these digestions impossible to detect directly, if the digestions occur properly the resulting fragments should be able to self-ligate, so it is possible to test for that.
The two double digestions had contents as follows: 5 uL EcoRI buffer, 0.5 uL 100x BSA, 10.5 uL phsABC (1 ug DNA), 1.8 uL EcoRI, 1.8 uL SpeI, and 30.4 uL water.
The diagnostic single digestions were run with 13.2 uL phsABC (at 75.5 ng/uL, 1 ug), 5 uL EcoRI buffer, 0.5 uL BSA, 27.7 uL water, and 3.6 uL of the relevant enzyme whether EcoRI or SpeI.
All of the above digestions were run for eight hours at 37˚C before being heat-killed with 20 minutes at 80˚C.
Wetlab work for this day is also recorded on pages 70 & 71 of the hard copy lab notebook.
Retrieved from " http://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_6 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
