<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Yale_Our_Project_Notebook_Week_5"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Yale/Our_Project/Notebook/Week_5">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Yale/Our_Project/Notebook/Week_5&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_5&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_5&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Yale/Our_Project/Notebook/Week_5" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Yale/Our Project/Notebook/Week 5</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><P><TITLE>iGEM Yale</TITLE></P><DIV id="wikiwrapper"><DIV id="wikiheader"><UL id="nav"><LI><A href="https://2010.igem.org/Team:Yale">home</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project">our project</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Team">our team</A></LI></UL></DIV></DIV><DIV id="main-content"><DIV id="left-col"><H5>
our project
</H5><P><UL id="proj-nav"><LI><A href="https://2010.igem.org/Team:Yale/Our Project">introduction</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Applications">applications</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Methods">methods</A></LI><LI><B><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook">lab notebook</A></B></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook">week 1</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 2">week 2</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 3">week 3</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 4">week 4</A></LI><LI id="nb"><B><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 5">week 5</A></B></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 6">week 6</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 7">week 7</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 8">week 8</A></LI><LI id="nb"><A href="https://2010.igem.org/Team:Yale/Our Project/Notebook/Week 9">week 9</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Protocols">protocols</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Safety">safety</A></LI><LI><A href="https://2010.igem.org/Team:Yale/Our Project/Future Work">future work</A></LI></UL></P></DIV><DIV id="right-col"><H5>
lab notebook: week 5 (7/5 -7/11)
</H5><P><UL><LI>

Monday 7/5--Colony PCR of Ligation Attempt #2 Transformants


</LI><A id="link" href="javascript:ReverseDisplay('monday')">See more/less</A></UL></P><H4> Colony PCR</H4><UL><LI>Ran colony PCR of colonies 1B-9B from ligation attempt #2 according to the same protocol as used on <A href="https://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_4"> 7/1</A>.</LI><LI>Then ran a 1.0% agarose gel of the nine PCR reaction solutions against a 1 kb ladder, but the resulting gel failed to show any evidence of either insert or vector, so will have to rely on sequencing and digests of miniprepped plasmids.</LI><DIV id="caption">Gel of colony PCR from ligation attempt #2 Lane 1: 1 kb ladder, Lanes 3-11:PCR products from colonies 1B-9B, in order</DIV><LI>Miniprepped liquid cultures grown from colonies of first ligation attempt (colonies 1A-4A) and second ligation attempt (colonies 1B-9B) according to <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols"> standard miniprep protocol</A></LI></UL><I>This day's activities are also recorded on page 47 and 48 of the hard copy lab notebook. </I></DIV><LI>

Tuesday 7/6--Analysis of Results of Ligation Attempts 1 &amp; 2

</LI><A id="link" href="javascript:ReverseDisplay('tuesday')">See more/less</A><DIV style="display:none;" id="tuesday"><H4> More Diagnostics of Ligation Results</H4><UL><LI>In order to determine success of ligation attempts 1 and 2, digested the miniprepped plasmids 1A-4A and 1B-9B from 7/5 with XbaI and SpeI and then looked for evidence of two fragments, one at 3.2 kb and the other at 3.6 kb. Reactions were conducted for 2 hours at 37°C, with the following contents for each of the thirteen reactions </LI><TABLE><TBODY><TR><TD>
Component </TD><TD> 	Volume
</TD></TR><TR><TD>
Distilled Water	</TD><TD>6.4 μL
</TD></TR><TR><TD>
10x NEB Buffer 4	</TD><TD>2 μL
</TD></TR><TR><TD>
DNA solution	</TD><TD>10 μL
</TD></TR><TR><TD>
NEB XbaI 	</TD><TD>0.8 μL
</TD></TR><TR><TD>
NEB SpeI 	</TD><TD>0.8 μL
</TD></TR><TR><TD>
Total 	</TD><TD>20 μL
</TD></TR></TBODY></TABLE><LI>The diagnostic 1.0% agarose gel was run at 90 V vs. a 1 kb ladder. Each of the thirteen samples gave one diffuse band around 3 kb and no evidence of the 3.6 kb phsABC fragment, so both ligation attempts #1 and #2 were unsuccessful.</LI><DIV id="caption">Gel of XbaI and SpeI double digest of plasmids from ligation attempts #1 and #2 Lane 1: 1 kb ladder, Lanes 2-5:1A-4A, Lanes 6-14: 1B-9B </DIV><LI>During digestion prepped plasmid samples for sequencing. Each sample contained 3 μL of plasmid soln (average concentration of 96 ng/μL), 2 μL of 4 mM VF2 (forward primer), and 13 μL of distilled water. Intended to follow up with reverse sequencing of more promising samples, but gel results made it a moot point. </LI></UL><I>Wetlab work for this day is also recorded on pages 47, 49, and 50 in the hard copy lab notebook. </I></DIV><LI>

Wednesday 7/7--Triple ligation attempt--promoter + thiosulfate reductase + terminator

</LI><A id="link" href="javascript:ReverseDisplay('wednesday')">See more/less</A><DIV style="display:none;" id="wednesday"><H4>Ligation Strategy </H4><UL><LI>Failure of previous two ligation efforts prompted rethinking of ligation strategy. Will attempt a one-step triple ligation that will link phsABC to B0015 and insert them both into a standard single-resistance iGEM background. phsABC will be digested with EcoRI and SpeI, B0015 with XbaI and PstI, and pSB1C3 with EcoRI and PstI, allowing for them all to be ligated together. </LI></UL><B>Prep work </B><UL><LI>Needed more phsABC, so ran five PCR reactions to amplify it from plasmid pSB74. Reaction mixture contents were the same as the DMSO variant used on <A href="https://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_4">6/30 </A>as was the thermocycler protocol (phs50). The results were then run on a gel</LI><DIV id="caption">Gel of PCR amplified phsABC Extreme right lane:1 kb ladder, all other lanes PCR reaction mixtures and their spillover</DIV><LI>Needed chloramphenicol plates for pSB1C3-based constructs, so mixed 350 μL of 50 μg/μL chloramphenicol with an autoclaved solution of 7.5 g agar and 12.5 g LB in 500 mL H2O and poured plates. </LI><B>Digestions</B><LI>Ran all three digestion reactions for 30 minutes at 37 °C before heat killing for 20 minutes at 80 °C. Digestion mixtures had the following components: </LI><TABLE><TBODY><TR><TD>
Digestion Reaction Components
</TD></TR><TR><TD>
phsABC Digestion </TD><TD>	B0015 Digestion	</TD><TD>pSB1C3 Digestionhttps://2010.igem.org/Main_Page
</TD></TR><TR><TD>
5 μL EcoRI Buffer 10x 	</TD><TD>5 μL NEB Buffer 3 10x </TD><TD>	5 μL EcoRI Buffer 10x
</TD></TR><TR><TD>
0.5 μL BSA 100x </TD><TD>0.5 μL BSA 100x </TD><TD>0.5 μL BSA 100x 
</TD></TR><TR><TD>
21.1 μL phsABC soln (1 μg DNA)</TD><TD> 	3.8 μL B0015 soln (1 μg DNA)	</TD><TD>15 μL phsABC soln (375 ng DNA)
</TD></TR><TR><TD>
1.8 μL EcoRI &amp; 1.8 μL SpeI 	</TD><TD>1.8 μL XbaI &amp; 1.8 μL PstI 	</TD><TD>1.2 μL EcoRI, 1.2 μL PstI, &amp; 1.2 μL DpnI
</TD></TR><TR><TD>
20.8 μL H2O 	</TD><TD>37.1 μL H2O </TD><TD>	15.9 μL H2O
</TD></TR></TBODY></TABLE></UL><I>Wetlab work for this day is also recorded on page 51 of the hard copy lab notebook. </I></DIV><LI>

Thursday 7/8--Further ligation efforts

</LI><A id="link" href="javascript:ReverseDisplay('thursday')">See more/less</A><DIV style="display:none;" id="thursday"><H4> Ligation Efforts Continue</H4><UL><LI>In addition to trying the triple-ligation strategy started on 7/7 (ligation attemp #3) will try trouble-shooting the ligation strategy used in attempts #1 and #2. In particular, will modify digestion protocol and test EcoRI and XbaI (older enzymes) for activity. </LI></UL><B>Ligation Attempt #3 (Triple Ligation)</B><UL><LI>Ran triple ligation all day at room temperature and stored at 4°C. Ligation contents were as follow to have a 1:1:1 stoichiometric ratio of all three components.</LI><TABLE><TBODY><TR><TD>
Component </TD><TD>	Volume
</TD></TR><TR><TD>
Distilled Water	</TD><TD>6.4 μL
</TD></TR><TR><TD>
T4 Ligase buffer 10x	</TD><TD>2 μL
</TD></TR><TR><TD>
NEB T4 Ligase</TD><TD>	1 μL
</TD></TR><TR><TD>
pSB1C3 solution </TD><TD>	2 μL
</TD></TR><TR><TD>
B0015 solution</TD><TD> 	2.4 μL
</TD></TR><TR><TD>
phsABC solution </TD><TD>	6.2 μL
</TD></TR><TR><TD>
Total </TD><TD>	20 μL
</TD></TR></TBODY></TABLE></UL><B>Ligation Attempt #4</B><UL><LI>Concerns about enzyme activity led to reversing the order of the serial digestion of B0015 background so that the first digestion is XbaI (low salt buffer) and the second is EcoRI (high salt buffer). Will also omit the pre-ligation PCR purification step.</LI><LI>Overnight digestion of ligation components--Set up the following digestions to run overnight at 37°C. The EcoRI was newly purchased as there was concern about the age of the previously used enzyme.</LI><TABLE><TBODY><TR><TD><B>phsABC double digestion</B></TD></TR><TR><TD>	
NEB EcoRI Buffer</TD><TD>	5 μL
</TD></TR><TR><TD>
BSA 100x 	</TD><TD>0.5 μL
</TD></TR><TR><TD>
phsABC solution </TD><TD>	30.8 μL (1 μg DNA)
</TD></TR><TR><TD>
NEB EcoRI 	</TD><TD>1.8 μL
</TD></TR><TR><TD>
NEB SpeI 	</TD><TD>1.8 μL
</TD></TR><TR><TD>
Sterile H2O </TD><TD>	10.1 μL
</TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD><B>B0015 digestion </B></TD></TR><TR><TD>	
NEB Buffer 4 </TD><TD>	5 μL
</TD></TR><TR><TD>
BSA 100x </TD><TD>	0.5 μL
</TD></TR><TR><TD>
NEB XbaI </TD><TD>	3.6 μL
</TD></TR><TR><TD>
B0015 solution </TD><TD>	3.8 μL ( 1 μg DNA)
</TD></TR><TR><TD>
Sterile H2O 	</TD><TD>37.1 μL
</TD></TR></TBODY></TABLE></UL><I>Wetlab work for this day is also recorded on pages 52 and 53 of the hard copy lab notebook.</I></DIV><LI>

Friday 7/9--Ligation attempt #4 to join phsABC and B0015 terminator.
</LI><A id="link" href="javascript:ReverseDisplay('friday')">See more/less</A><DIV style="display:none;" id="friday"><H4> Ligation Attempt #4</H4><UL><LI>Removed both overnight digestions, the XbaI digestion of B0015 and the EcoRI and SpeI digestion of phsABC, from 7/8 from the incubator.</LI><LI>Heatkilled enzymes of phsABC digest with 20 minutes at 80°C.</LI><LI>Ran <A href="https://2010.igem.org/Team:Yale/Our_Project/Protocols/PCR_purify">PCR purification protocol </A>on XbaI digestion using microcentrifuge. In order to give a concentrated sample, eluted in only 35 μL of EB Buffer that had been heated at 50°C. Resulting concentration could not be determined as there was residual ethanol contamination.</LI><LI>Ran EcoRI digestion of purified XbaI digest product at 37 °C, with components as follows. Digestion was run for an hour before addition of CIP, after which it was run for another hour. </LI><TABLE><TBODY><TR><TD><B>B0015 digestion with EcoRI</B></TD></TR><TR><TD>	
NEB EcoRI buffer </TD><TD>	5 μL
</TD></TR><TR><TD>
BSA 100x </TD><TD>	0.5 μL
</TD></TR><TR><TD>
NEB EcoRI 	</TD><TD>3.6 μL
</TD></TR><TR><TD>
B0015 digested with XbaI 	</TD><TD>25 μL
</TD></TR><TR><TD>
NEB Calf Intestinal Phosphatase 	</TD><TD>1 μL
</TD></TR><TR><TD>
Sterile H2O </TD><TD>	15.9 μL
</TD></TR></TBODY></TABLE>

Followed with 20 minute heatkill at 80°C.<LI>In order to set up ligation, need concentration of B0015, but ethanol contaminant continues to preclude measurement. Will estimate that the purification between digestion steps resulted in a 50% loss of DNA, leaving behind a 10 ng/μL concentration. Will also, as usual, run different ligation trials with different ratios of insert to vector, which should help combat any issues if this estimate is inaccurate </LI><LI>Set up the following ligation trials, with stoichiometric ratios of insert to vector (I:V) based on B0015 concentration estimate. Ran the ligations for 20 minutes at room temperature. </LI><TABLE><TBODY><TR><TD><B>Ligation Reaction Components</B></TD></TR><TR><TD>
Control Ligation</TD><TD>	1:1 I:V	</TD><TD>2:1 I:V </TD><TD>	3:1 I:V</TD><TD>	4:1 I:V	
</TD></TR><TR><TD>
2 μL NEB T4 ligase buffer for all reactions
</TD></TR><TR><TD>
1 μL NEB T4 ligase for all reactions
</TD></TR><TR><TD>
5 μL B0015 solution for all reactions
</TD></TR><TR><TD>
12 μL H<SUB>2</SUB>O </TD><TD>	8.9 μL </TD><TD>5.8 μL H<SUB>2</SUB>O 	</TD><TD>2.7 μL H<SUB>2</SUB>O </TD><TD>	--
</TD></TR><TR><TD>
-- 	</TD><TD>3.1 μL phsABC </TD><TD>	6.2μL phsABC </TD><TD>	9.3 μL phsABC</TD><TD> 	12 μL phsABC
</TD></TR></TBODY></TABLE><LI> Following ligation, transformed 1 μL of each ligation reaction into commercial grade cells according to standard transformation protocol. Also transformed triple ligation reaction. </LI></UL><H4>Enzyme Activity Test</H4>

To test enzyme activity, will do single digests of B0015 with enzymes in question and then run them on a gel to see if the plasmid linearized. XbaI and EcoRI are old, so will test them first. For XbaI activity test, simply set aside 10 μL of digestion solution from overnight digestion. Testing of EcoRI required additional digests. Digests using both old and new EcoRI samples were run for two hours at 37 °C with the following components:<TABLE><TBODY><TR><TD><B>EcoRI Activity </B></TD></TR><TR><TD>
NEB EcoRI buffer </TD><TD>	5 μL
</TD></TR><TR><TD>
BSA 100x 	</TD><TD>0.5 μL
</TD></TR><TR><TD>
NEB EcoRI 	</TD><TD>3.6 μL
</TD></TR><TR><TD>
undigested B0015 	</TD><TD>3.8 μL ( 1 μg DNA)
</TD></TR><TR><TD>
Sterile H<SUB>2</SUB>O 	</TD><TD>37.1 μL
</TD></TR></TBODY></TABLE>

Followed with 20 minute heatkill at 80°C.<UL><LI>Loaded all three enzyme activity tests on a 1.0% agarose gel and ran them at 90 V versus a 1 kb ladder and circular B0015 plasmid. </LI>

Gel of single digests of B0015 Left to Right lane 1:1 kb ladder, lane 2: uncut circular B0015, its dimer, and lane 3 spillover, lane 3: XbaI digest of B0015, lane 4: old EcoRI digest of B0015, lane 5: new EcoRI digest of B0015<LI>Based on the results, all enzymes appeared to work, as they successfully created a linearized 3.2 kb fragment.</LI><I>Wetlab work for this day is also recorded on pages 54-57 of the hard copy lab notebook. </I></UL></DIV><LI>

Saturday 7/10--Transformants from ligation attempts

</LI><A id="link" href="javascript:ReverseDisplay('saturday')">See more/less</A><DIV style="display:none;" id="saturday"><B> Transformants from ligation attempts #3 an #4</B><UL><LI> The plate from ligation attempt #3 shoed in excess of 100 differnt colonies. </LI><LI> Plates from ligation attempt #4 had the following colony counts:</LI>
Control-- 5 colonies 
1:1--28 colonies 
2:1--38 colonies 
3:1--18 colonies 
4:1--27 colonies <LI>Took colonies from various plates and used them to establish index plates on Amp LB, labeled as follow: eight colonies from triple ligation as #1-#8, four colonies from 1:1 as#9-#12, four colonies from 2:1 as #13-#16, four colonies from 3:1 as #17-#20, and four colonies from 4:1 as #21-#24.</LI><I>Wetlab work for this day is also recorded on page 58 of the hard copy lab notebook.</I><LI>

Sunday 7/11--Prep for copper growth assays of transformants
</LI><A id="link" href="javascript:ReverseDisplay('sunday')">See more/less</A><DIV style="display:none;" id="sunday"><B>Ligation work </B><UL><LI> Realize that mistakenly plated triple ligation reaction on ampicillin plate rather than the required chloramphenicol, so none of the colonies present are of value.  Fortunately still have transformation solution, so replate it, this time on a chloramphenicol </LI><LI> Inoculated overnight liquid cultures of ligation attempt #4 transformants  (#9-#24 on index plates) for miniprep the following morning </LI><B>Growth assay prep work </B><LI> Want to see if pSB74 confers any copper resistance on <I>E. coli</I>, since it should help them precipitate nearby copper and thus reduce the copper concentration that they experience. </LI><LI> Inoculated liquid cultures of  LE392 with pSB74 in ampicillin LB and untransformed LE392 in plain LB and left to grow overnight on shaker at 37˚C for growth assays the following day </LI><LI> Prepared CuSO<SUB>4</SUB> solutions in LB.  Each well in the 96-well plate will have 225 uL of copper solution and 25 uL of cell solution, so the copper solution must have a CuSO<SUB>4</SUB> concentration that is 10/9 the desired final values. The copper solutions must also contain ampicillin and IPTG at the same levels as the cultures that will be introduced. The desired  CuSO<SUB>4</SUB> concentrations are those used in the wide and narrow concentration range trials on <A href="https://2010.igem.org/Team:Yale/Our_Project/Notebook"> 6/10 </A> and <A href="https://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_2"> 6/14 </A>. </LI></UL><I>Wetlab work for this day is also recorded on pages 59-61 of the hard copy lab notebook.</I></DIV></UL></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_5">http://2010.igem.org/Team:Yale/Our_Project/Notebook/Week_5</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Yale/Our_Project/Notebook/Week_5" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Yale/Our_Project/Notebook/Week_5" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_5&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Yale/Our_Project/Notebook/Week_5&amp;oldid=206111" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>