"
Page
Discussion
View source
History
teams
Log in
 
Team:Yale/Our Project/Methods
From 2010.igem.org
iGEM Yale
home
our project
our team
our project
introduction
applications
methods
plasmid
H2S production
Cu growth assay
Cu localization
parts
lab notebook
protocols
safety
future work
experimental methods
Our plasmid is composed of three parts: a promoter and a terminator Biobrick as well as a novel addition to the biobrick library, the phsABC gene that is known to encode Thiosulfate Reductase.
(1) phsABC gene and vector
phsABC in pSB74
This central component encodes Thiosulfate Reductase. The gene phsABC was obtained through Addgene from Dr. Jay Keasling's laboratory at University of California, Berkeley. According to their results, thiosulfate reductase encoded in the plasmid pSB74 showed the highest activity catalytic activity, so we obtained phsABC from the plasmid pSB74.
Table from Keasling’s research: comparison of Thiosulfate reductase activity
Figure from Keasling’s research: Sulfide production by phsABC in various plasmids. pSB74 (orange) showed the highest reactivity.
(2) Biobrick Promoter
Promoter used was designed by Caitlin Conboy and was found within the parts registry. This promoter is a Quad Part Inverter: “that is, a PoPS-based inverter composed of four sub-parts: a ribosome binding site, a coding region for a repressor protein (e.g., lambda cI), a terminator, and the promoter (e.g., pLambda) regulated by the encoded repressor protein.” Research into promoter activity by previous groups has suggested that this promoter has a strong on state with a noticeable  background in the off state .
Promoter B0034
Biobrick Part:BBa_Q04121.
Length 1370 bp
IPTG-induced (regulatory)
(3) Biobrick Terminator The terminator used was the 129 bp BBa_B0015 designed by Reshma Shetty.  It is actually a double terminator composed of BBa_B0010 and BBa_B0012 and the BioBrick assembly scar & was chosen for its reliability and availability.
Restriction Enzyme Sites:
The sites are shown in New England BioLabs Inc.
Plasmid Construction
The digested sticky ends of the enzymes Xba I and Spe I are complimentary. Once two ends from different combine, neither Xba I nor Spe I can recognize its restriction site in the gene.
By using this ligation method, inserted phsABC into B0015:
Read more about the background of the phsABC gene
phsABC gene: Important part of anaerobic bacterial respiration/metabolism • Studied since 1970's; several plasmids found (ie: phs pEB40, pSB103, pSB77, pSB107 etc) • Thiosulfate reductase is a transmembrane protein involved in the second step of the Sulfate-Reducing Bacteria pathway of thiosulfate reduction. Thiosulfate reductase catalyzes the dissimilatory reduction of inorganic thiosulfate to hydrogen sulfide and sulfite. • 1995, phs locus of Salmonella chromosome located, sequenced, and confered H2S production to E.Coli. – Sequence similar to reductases in other bacteria – Later confirmed to contain structural gene for thiosulfate reductase • Since before 1970 people have observed bacteria producing H2S. The earliest samples were taken from patients (Stoleru and Bouanehaud of Institut Pasteur, 1972 and 1975). Better yet, they showed that H2S production was mediated by a plasmid! A later study in 1978 by Jones et al of Texas Tech characterized another one of these plasmids, saw that it could be used in and transferred between E. Coli as well! • In 1987 an extensive review was written collecting lots of existing research about sulfate-reducing bacteria (SRB). In this broad review of SFB, it mentioned that sulfate reductases are membrane bound. Possibly useful for us, purified samples of thiosulfate reductase can also apparently be controlled, stopped in the presence of NADH , NADPH, and cysteine. Anyhow, the main development in this paper showed that thiosulfate-reduction ability is fairly common, usually used by anaerobic bacteria as an energy source (although some aerobic variants have been reported) and that one of the key players is thiosulfate reductase, which by this point had been isolated and had some characterization available. If required, there are lots of other sulfur-reducing pathways that are common as well, with a notable one being the reduction of tetrathionate. • Perhaps not surprising, the same authors of the review paper actually in the same year published a paper on the phs gene mediating hydrogen-sulfide production! Better yet, this gene is not coupled to methyl-viologen, which is common in anaerobic bacteria! score! Of note, the authors write that the phs gene is not actually the structural gene itself, but instead codes for a regulatory protein important for the reduction of thiosulfate to H2S. This paper also looked at different ways to optimize H2S production. • Something happened over a decade (particularly, see 1995 paper by Erika Barrett analyzing the phs sequence), because by 1999 a study from UC Berkeley, in Keasling lab, had the actual structural genes! In a couple of studies (1999-2000) they optimized the phs H2S production system to produce lots of H2S and remove heavy metals from solution! Optimization of their system is given in the slide to the left • Keasling lab, used phs gene sequences transformed into DH5α for bioremediation studies (2000). •  • Optimized reading frame and plasmid for H2S production • Optimized conditions for precipitation of metals •   (zinc, lead, and cadmium)
Retrieved from " http://2010.igem.org/Team:Yale/Our_Project/Methods "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
