Team:IISER-Mohali-INDIA/Demonstrate
Description
Circuit
Genetic Design
Clonning Strategies
Co-culturing and Growth
Lab Journal
Results
Team
Attributions
Collaboration
Interlab
Parts Submission
Composite Part
Safety
Modelling
Hardware
Measurement
Demonstration
Software
Judging Form
Entrepreneurship
Applied Design
Plant Synthetic Biology
Improve
Contributions
Int. Human Practices
Education & Engagement
Public Awareness
Navigate
Home
Project
Description
Circuit
Genetic Design
Clonning Strategies
Co-culturing and Growth
Lab Journal
Results
Team
Attributions
Collaborations
Interlab
Parts Submission
Composite Parts
Safety
Competition
Modelling
Hardware
Measurement
Demonstration
Software
Application
Entrepreneurship
Applied Design
PLant Synthetic Biology
Practices
Int. Human Practices
Education & Engagement
Public Awareness
Demonstrate
1. Cloning of module 1:
Construct 1 --> Ptet - RBS1 - T7 RNA polymerase - terminator 1 - terminator 2. Two PCR reactions were performed. The first PCR reaction was performed with primer 1 and 2 using the template R0040 to amplify the Ptet. The size of PCR product is 77 bp. The second PCR reaction was performed with primer 3 and 4 using the template K1450004 to amplify the RBS1 - T7 RNA polymerase - terminator 1 - terminator 2. The size of this PCR product is 2338 bp. Thirdly, SOE PCR was performed using the primers 1 and 4 (for PCR conditions and reaction mixture, refer notebook).
The final PCR product was attempted to be cloned at AatII and ApaI sites of pZS21MCS. However, we did not get positive results for cloning, resulting in failure to clone the construct 1. Similarly, cloning of construct 2 (PT7 - RBS2 - Chromoprotein II - RBS3 - tetR - terminator3), three PCR reactions were performed.
The first PCR reaction was performed with primer 5 and 6 using the template K1343022 to amplify PT7-RBS2. The size of the PCR product is 132 bp. The second PCR reaction was performed with primer 7 and 8 using the template K1033910 to amplify chromoprotein II. The size of the PCR product is 759 bp. The third PCR was performed using the primers 9 and 10 (for PCR conditions and reaction mixture, refer notebook). Next, SOE1 PCR was performed using primers 5 and 8 to get an 891 bp amplified fragment.  Then, SOE2 PCR was performed using primers 5, primer 10 and the template SOE1 (891bp) fragment to amplify the 1783 bp fragment. The cloning of this SOE2 PCR fragment was attempted in the low copy number plasmid pZS21MCS, at SalI and BamHI sites, but due to problems with digestion of the vector, we were not able to clone in pZS21MCS plasmid.  Therefore, construct 2 was finally cloned at SalI and BamHI sites of pRC10 (modified ptrc99a) which is an intermediate copy number plasmid. The E. coli transformants were screened for positive clones of construct 2 by restriction digestion and a positive clone 9 was obtained as shown in the image below.
2. Cloning of module 2:
Construct 1 --> Pmar - RBS4 - ToxR - terminator 4.
The first PCR was performed with primers 11 and 12 using the template E. coli genome to amplify the pmar - RBS4. The size of the PCR product is 524 bp. The second PCR was performed with primers 13 and 14 using the template K641009 to amplify ToxR. The size of this PCR product is 1441 bp. The third PCR was performed using the primers 15 and 16 using the template pVenus to amplify terminator (271 bp). For PCR conditions and reaction mixture, refer notebook. Next, SOE1 PCR was performed using primers 11 and 14 to get a 1965 bp amplified fragment.  Then, SOE2 PCR was performed using primer 11, primer 16, template SOE1 (1965bp) and third PCR amplified fragment (271 bp) to amplify the 2236 bp fragment. This SOE2 PCR fragment was attempted to be cloned in low copy number plasmid pACYC177, at XhoI and XmaI sites (high copy number plasmid).
The E. coli transformants were screened for positive clones of construct 2 by restriction digestion and positive clones were obtained in lane 2,3,4,6,7,8,9,10 and 11 as shown in image below:
Construct 2 --> Pctx - RBS5 - chromoprotein I - RBS6 - TetR - terminator5.
Two PCR reactions were performed. The first PCR reaction was performed with primers 17 and 18 to amplify the Pctx. The size of PCR product is 180 bp. The second PCR reaction was performed with primers 19 and 20 using the template K1343022 to amplify the RBS5 - chromoprotein I - RBS6 - TetR - terminator5. The size of PCR product is 1645 bp. Thirdly, SOE PCR was performed using the primers 17 and 20 to amplify 1825 bp (for PCR conditions and reaction mixture, refer notebook). Finally, construct II of module II was cloned at BamHI and AatII site of pACYC177 plasmid. The E. coli transformants were screened for positive clones of construct 2 by PCR and obtained positive clone 4,6,7,8 and 10 shown in image below:
To check the sensitivity of the mar promoter, we used an E. coli strain (taken from C.R. Rao’s lab), which contains a mar cis-element cloned upstream of the Venus reporter gene (a fast folding variant of yellow fluorescent protein) integrated at the attλ site. We monitored the amount of fluorescence in the reporter strain with increasing amount of salicylate as shown in the figure below:
The reporter strain was grown to it's exponential phase (0.5 OD600) and aliquoted 200 µl per well into three wells of 96-well, clear bottom, black plate. A desired amount of salicylate was added from a stock into each well. LB media with and without salicylate were used as blanks. The plates were incubated at 37°C with shaking, and fluorescence was measured after 40 minutes of incubation at λexcitation = 515 nm and λemission= 547 nm. We observed that below 1.25 mM salicylate, very little fluorescence was obtained. Thus, the sensitivity of the promoter is limited to mM concentrations of salicylate. To increase the sensitivity of the reporter, a stronger promoter or mutations in the MarR binding site can be done.
We also investigated the time kinetics of the reporter, by measuring the fluorescence of the reporter strain induced with various concentrations of salicylate.  The 200 µl aliquots of the exponentially growing reporter strain cells were induced with desired concentrations of salicylate and their fluorescence was measured after every 20 minutes. As seen in the figure below, the mar promoter responds very quickly to the salicylate - eg in case of 0.625 mM salicylate concentration, the minimal fluorescent signal reached saturation in 150 minutes.
With higher concentrations of salicylate, the response was as quick as before, but with greater magnitude. The saturation of the fluorescence signal could not be studied with higher concentrations as the fluorescence intensity crossed the upper-limit of the instrument. Thus, we observe that the mar promoter responds within 20 minutes of adding salicylate.
Cloning of chromoprotein for selection and part submission for registry:
We selected five different chromoproteins to study their kinetics and to evaluate their utility in our circuit.
We have observed that out of these five different chromoproteins, chromoprotein sequence numbers 1-3 developed color relatively fast (24-30 hours) but chromoprotein sequence numbers 4-5 developed color relatively slow. Hence, 4 and 5 are not suitable for our circuit. Therefore, we selected chromoproteins: amilCP Blue chromoprotein and fwYellow chromoprotein for use in our circuit. Further, all these chromoproteins were cloned with RBS and LacI promoter by 3A assembly and submitted in the depository (accession number from BBa_K2320001- K2320005).
Software
We seek to automate the process of reading gel doc slides and give reliable band sizes as outputs in a comprehensible format with an easy-to-use software. A manual mode has also been developed for graphical analysis to ease the detection of fainter bands that are too close to each other to distinguish using one's eyes. The software requires minimal resources: a few Python modules, any gel doc machine and a variety of standard ladders. The main difficulty in differentiating very closely spaced bands (due to the inefficiency of human eyes), is done away with in this module. Furthermore, we are working on introducing machine learning in the software to give better accuracy and to detect extremely faint bands. The software will help students and researchers to automate the mundane and routine task of reading gel doc slides.
