<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Gifu_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Gifu/Project</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="header"><UL><LI id="menu002" onmouseover="document.getElementById('pro').style.visibility='visible'" onmouseout="document.getElementById('pro').style.visibility='hidden'"><FONT size="5">Project</FONT></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project">Abstract </A></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/Project#Back ground">Back ground </A></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/Project#How to make sake">How to make sake </A></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/Project#Cataboliterepression">Catabolite repression </A></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/Project#Gap-repair cloning for biobricks users">Gap-repair cloning </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Terminatome">Terminatome </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Experiments">Experiments </A></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/Project#Plasmid construction"> Plasmid construction </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Results">Results </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Conclusion">Conclusion </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Discussion">Discussion </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Future work">Future work </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Reference">Reference</A></LI></UL><LI id="menu009" onmouseover="document.getElementById('note').style.visibility='visible'" onmouseout="document.getElementById('note').style.visibility='hidden'"><FONT size="5">Lab Note</FONT></LI><LI><A href="https://2017.igem.org/Team:Gifu/Protocol">Protocol</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Calendar">Calendar</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/InterLab">InterLab</A></LI><LI id="menu004" onmouseover="document.getElementById('parts').style.visibility='visible'" onmouseout="document.getElementById('parts').style.visibility='hidden'"><FONT size="5">Parts</FONT></LI><LI><A href="https://2017.igem.org/Team:Gifu/Basic Part">Basic Part</A></LI><LI id="com"><A href="https://2017.igem.org/Team:Gifu/Composite Part">Composite Part</A></LI><LI id="coll"><A href="https://2017.igem.org/Team:Gifu/Part Collection">Part Collection</A></LI><LI id="menu007" onmouseover="document.getElementById('hupra').style.visibility='visible'" onmouseout="document.getElementById('hupra').style.visibility='hidden'"><FONT id="menu010" size="5">Human Practices</FONT></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/HP/Silver">Human Practices</A></LI><LI id="back"><A href="https://2017.igem.org/Team:Gifu/HP/Gold_Integrated">Integrated HP</A></LI><LI id="menu003" onmouseover="document.getElementById('us').style.visibility='visible'" onmouseout="document.getElementById('us').style.visibility='hidden'"><FONT size="5">About Us</FONT></LI><LI><A href="https://2017.igem.org/Team:Gifu/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Attributions">Attributions</A></LI><LI id="back"><A href="https://igem.org/2017_Judging_Form?id=2438">Judging Form</A></LI></DIV><DIV id="menu"><LI><A href="https://2017.igem.org/Team:Gifu/Project">Abstract </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Background">Background </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#How to make sake">How to make sake </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Cataboliterepression">Catabolite repression </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Gap-repair cloning for biobricks users">Gap-repair cloning</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Terminatome">Terminatome</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Plasmid construction"> Plasmid construction </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Results">Results </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Conclusion">Conclusion </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Discussion">Discussion </A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Future work">Future work</A></LI><LI><A href="https://2017.igem.org/Team:Gifu/Project#Reference">Reference</A></LI></DIV><DIV id="pagebody"><H2>PROJECT</H2><H3 id="Abstract">Abstract</H3><P>Our team will try to make a new technology called LAM (Lactic acid mediated) communication utilizing GAR<SUP>+</SUP> prion. GAR, or glucose associated repression controls the usage of glucose of yeast. In 2016, it was proved that this pathway is managed by the certain amount of lactic acid. Lactic acid produced by lactic acid bacteria, affects the prion formed gar- and induces the formation of GAR<SUP>+</SUP>. Considering this mechanism and terminators of fission yeast and regulating the amount of mRNA of GFP and antisense-GFP, we will enable yeast to communicate each other beyond the species of gram-negative bacteria and make a biosensor to sense hiochi. 
. </P><H3 id="Background">Background</H3><P>In Gifu prefecture, there are many rivers that are called “clear river”. The water coming from the streams is famous for its good quality and good deliciousness. Therefore, it is natural that the development of sake brewing. However, there is a trouble which is called “an act of God” in the process of sake brewing. This is the phenomenon of “Hiochi”. </P><P>Now the risk of Hiochi has generally decreased by the complete control, the measurement of Hiochi. However, in the current sake market, the popularity of Namazake is increasing. Namazake have not passed the step of Hiire, pastuerization. Hence, Namazake have not been sterilized by heating. The taste of Namazake is fresh and delicate and the smell is pretty good. Also, there remains a feeling of sparkling. However, simply because Namazake have not been done “Hiire”, there may occur “Hiochi” after being opened the bottle when people store at home. So, we plan to make a biosensor which detects the causative bacteria of Hiochi. </P><P>Sake is a kind of fermented liquor and produced by three kinds of bacterial co-cultivate. Most sake is made from rice, water and rice Koji. At first glance, you might think sake is so simple and modest. However, while we refer to them all as sake, there are large varieties of sake like wine and beer. Sake is classified by criteria which is defined by Japanese national tax agency. The sake is distinguished by the following four criteria, what is used for, how much is the ratio of rice polishing, how much ratio of the Koji rice is and what smell is it. According to Japanese national tax agency, rice polishing ratio refers the ratio of weight of white rice to brown rice. Koji rice refers to white rice used for manufacturing rice Koji. Also, rice Koji is bred the Koji to white rice and can glycosylate starch of sake rice. Now you know, sake is not simple and modest but luxury.</P><FONT size="5">   ・History of sake </FONT><P>According to “Harimanokuni Fudoki ”(records of the culture and geography of the Harima province) , there has been clearly described about the oldest sake which was made from rice(A.C.700). After reaching the culture of rice cultivation from Mainland China, people started to make sake mainly using rice. Sake brewing was started as a part of offerings that is dedicated to a god. As a part of offerings, the sake was made by the first rice of the year. At that time, people make sake by the way which is called “Kuchikami”( =mouth chewed). By chewing the heated crops, amylase contained in saliva converts starch into sugar. After that, wild yeast ferments sugar into alcohol. In Japan, people say “Kamosu”(=ferment at the process of brewing sake). It is said that the word of “Kamosu” came from “Kamu”(=chewing).</P><P>After this era, according to some historical literature, the culture of sake brewing spreaded to the whole country. However, in those days, sake was high viscosity because the proportion of water in sake is low. Therefore, people ate sake with chopsticks. </P><P>Early in the Nara period(710~794), brewers started to use the brewing method of using rice Koji. The way of sake brewing was the same as modern and there has been applied a lot of kinds of sake in Heian period. At that time, the sake was yet cloudy sake. From this period, “Sobosyu”(=sake produced by monks) started to develop till Japan government banned making sake by monks. The quality of “Sobosyu” was told to be high and their sake was so precious. </P><P>During middle ages, when commerce became more prosperous, the economical value of sake as a kind of goods is equal to rice. And in this era, there were three breakthrough renovation. First, it made a dramatically quality improvement to use white rice for both steamed rice and Koji. This is called “Moroshiro”. Using this way, brewers can make sake with high clarity. Second, brewers realized that making sake at winter, it took a little longer time, but they can produce pretty good quality of sake. Third, they made it possible to control the fermentation process by “Dankake”. “Dankake” is a kind of ingenuity. When they prepared, they added steamed rice and Koji rice and water in some steps. By doing this, they could increase the quantity of acid in stages. </P><P>Furthermore controlling their fermentation process they became to supply stable quality sake. In this way, mainstream of sake migrated from cloudy sake into clarified sake. In Edo period, some of innovative processing technology had developed. Here are some examples. To begin with, we will introduce the technology of “Hiire”(=pasteurization) which is most related to our team project. Pasteurization is a way to raise preserving property. </P><P>Another technology is the allegation medial of adding alcohol. These two technologies are means to avoid the risk of “Hiochi”. And from this period, sake became popular among common people because sake began to carry-out to whole Japan by ships. Though the innovative processing technology had developed, “Hiire” is not a complete measure to prevent “Hiochi”. So, there were many large scale of decays in Meiji period. </P><P>In 1904, Japan government established the National Research Institute of Brewing. NRIB was made in order to study yeast for seishu and improve brewing way. In NRIB, Kamajiro Eda who is a head of NRIB invented “Sokujyomoto”. “Sokujyomoto ” is sake mush which is used for fast brewing method by adding enzyme. Also he had improved the method of “Hiire”. Brewers do “Hiire” after they bottled their sake. 
</P><P>In the early of Showa period, the new yeast which can make good smell such as the sixth yeast was picked, isolated and pure-cultured. And by the time, most of the devices for measuring has appeared. During the W.W.Ⅱ, the production of sake declined because of rice shortage. After the war, reconstruction of sake brewers started every places.
</P><FONT size="5">   ・The phenomenon of “Hiochi” </FONT><P>Growing of Hiochi bacteria in sake and bacterial pollution of seishu is called Hiochi. After the phenomenon of “Hiochi” , seishu became cloudy and take on peculiar odor by producing acid inside of it. Hiochi bacteria is high resistant to alcohol so they can grow in seishu. Hiochi bacteria can be separated to hetero type and homo type by the style of fermentation. Furthermore, they can be classified by whether they need mevalonic acid when they grow. Hiochi bacteria are a kind of Lactic acid bacteria and cannot grow except for sake because  they need mevalonic acid produced by Koji for living.</P><H3 id="How to make sake">How to make sake </H3><P><FONT size="4"> ① Seimai, rice polishing  </FONT></P><P>Japanese Sake breweries mainly utilize sake-brewing rice which is only used for making sake. As a first step of brewing sake, brown rice of the sake rice is polished to exclude the germ and the surface fraction. They contain abundant proteins, lipids and inorganic substances. These ingredients are negative factors for sake because taste and the color can be rough. Also without highly polished rice, deterioration of sake is accelerated easily. Therefore, the center of rice grain called shinpaku is basically used in production of sake. Shinpaku is also called pearl of rice from this reason.</P><P><FONT size="4">② Seimai and Shinseki, washing and soaking the rice</FONT></P><P>After polishing the rice, Nuka, rice bran powder and foreign matters are washed away as these substances have a bad effect on the final quality of sake. Following this process, the rice is soaked carefully. Measuring the time at this stage is necessary to make the amount of absorbed water appropriate. Koji, culture of a special species of mold on rice can easily grow on the rice.</P><P><FONT size="4">③ Steaming and cooling the rice </FONT></P><P>The rice is steamed for approximately one hour. Steamed rice is cooled gradually and sufficiently.</P><P><FONT size="4"> ④ Seikiku, making koji</FONT></P><P>One of the most important processes is making koji, <I>Aspergillus oryzae</I> is mainly used for propagating koji mold. In koji-muro, koji-making room at 30 ℃, koji spores are inoculated on the rice. After inoculation, the rice is kneaded and mixed well to put koji spores evenly onto rice. Under the condition of koji-muro, koji rice is incubated for one day.</P><FONT size="4"> ⑤ Making shubo, starter culture of sake</FONT><P>Rice koji, a kind of mold grown on rice, sake yeast and steamed rice is added to shikomi-sui, water for sake making. With strict thermal management, mass propagation of sake yeast is accelerated. Toji, a chief of sake maker continues the pumping over the starter liquid. Taking out and pouring of starter culture liquid during early stage of the starter enhances the absorption of liquid by koji and steamed rice. The liquid of the fermentation starter is full of enzyme for saccharification. The chief verifies the quality of shubo overnight.</P><FONT size="4"> ⑥ Shikomi, preparation for fermentation mash</FONT><P>After being transferred to a larger tank, more koji, the rice and water are added. To prepare the fermentation mash of sake, steamed rice and koji are divided into three portions and added over four days. Three steps are called hatsu-zoe or first step, naka-zoe or second stage and tome-zoe or third stage respectively. This method is the basic and traditional method of sake brewery and astonishingly effective to prevent microbial contamination. After incubation for 15 to 20 days, moromi, fermentation mash, is completed.</P><FONT size="4">⑦ Pressing and filtration</FONT><P>Pressing is a process to separate sake kasu, sake cake, and sake. This is conducted by putting moromi in sakebukuro, cloth bags used at pressing stage. Filtration is also performed to let more solids such as rice settle out. Charcoal filtration is usually used to adjust flavor and color.</P><FONT size="4">⑧ Pasteurization</FONT><P>Following filtration, sake is heated to 60 ℃ to sterilize and deactivate the enzyme. This is called hiire and indispensable to prevent spoilage.</P><FONT size="4">⑨ Storing(Aging)</FONT><P>Sake is fundamentally left in the tank to age about a half of a year. The period depends on types and brands. </P><H3 id="Cataboliterepression">Catabolite repression and GAR prion</H3><P> For <I>S.cerevisiae</I>, glucose is the most principal source to maintain its life. Yeast is fundamentally able to utilize a variety of carbon source such as glycerol. However, under the condition of presence of glucose, yeast only uses glucose as its energy. This phenomenon is coordinated by several signaling and metabolic interactions that is controlled by glucose.</P><P>GAR stands for glucose associated repression. GAR prion regulates usage of glucose in yeast. Basically the conformation of the prion is [gar<SUP>-</SUP>]. Adding the certain amount of lactic acid to media of yeast, the conformation is changed into [GAR<SUP>+</SUP>]. GAR<SUP>+</SUP> prion effects on signal transmission in yeast and expression of enzymes or protein related to glucose utilization is repressed strongly. This mechanism was confirmed by the growth test of GLY+GluN media in <A href=" https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2758746/" target="_blank">this research</A>. As the cause of controlling catabolite repression is one kind of prions, this state is heritable to next generation. Technically, one of the common metabolite, lactic acid, plays a role of signal and may make it possible to conduct cross-kingdom communication.</P><H3 id="Gap-repair cloning for biobricks users">Gap-repair cloning for biobricks users</H3><FONT size="4"><STRONG>What is GRC?</STRONG></FONT><P>GRC or gap-repair cloning was invented by <A href="https://www.ncbi.nlm.nih.gov/pubmed/2828185" target="_blank">this research.</A></P><P>Utilizing homologous recombination, the process of transformation of yeast can be simplified and easy to conduct. The picture below shows a brief explanation of how to perform GRC. One of the reasons why not so many teams choose yeast in their projects is that it takes much longer time to conduct transformation of yeast. In this competition, all teams must construct plasmids which include pSB1C3 backbone. However since this backbone does not contain replication origin and selectable marker gene, users who would like to utilize yeast must make at least two plasmids, one of pSB1C3 and one of a shuttle vector. In addition, all teams use the same method of transformation. We thought prefix and suffix can be homologous  recombination regions. GRC may overcome this big problem and be a revolutionary solution for all iGEM teams. To develop this technology and adjust to biobricks, we decided to conduct the research of this technique. 
</P><H3 id="Terminatome">Terminatome</H3><P><STRONG>What is “Terminatome”?</STRONG></P><P>Terminatome is a new concept suggested by <A href=" https://www.ncbi.nlm.nih.gov/pubmed/23654277" target="_blank"> A Genome-Wide Activity Assessment of Terminator Regions in <I>Saccharomyces cerevisiae</I> Provides a “Terminatome” </A>
This breakthrough system will clearly develop synthetic biology. According to the article, terminators of yeast change the amount of expressed protein. It is mentioned that Terminator region activities relative to that of the PGK1 standard terminator ranged from
0.036 to 2.52, with a mean of 0.87. As the terminators effect on the amount of mRNA, selection of appropriate terminators is indispensable for all iGEM teams that would like to use yeast in their project. It is feasible to regulate the final amount of protein utilizing inducible or constitutive promoters. We chose RPL15A, PGK1 and Glc1 to assay this phenomenon on ADH1 promoter.</P><H3 id="Plasmid construction">Plasmid construction </H3><P> Here are ideal plasmids of this project and brief explanations below. As shuttle vector of the plasmid, we used pESC-LEU. For submission to HQ, we used pSB1C3 backbone.</P><P><STRONG>plasmid 1</STRONG></P><P>ADH1 promoter is one of the standard promoters for yeast and has function constitutively. HXT3 promoter is anticipated to be regulated by [GAR<SUP>+</SUP>] prion and repressed strongly according to  <A href="https://www.ncbi.nlm.nih.gov/pubmed/12861007
">this article</A>. When the concentration of lactic acid is relatively low, the prion makes its conformation [gar<SUP>-</SUP>]. Then naturally coding regions of SF-GFP and antisense-GFP are transcribed. As a result, mRNA of GFP is expected to be combined with mRNA of antisense-GFP. When the concentration of lactic acid is relatively high, the prion makes its conformation [GAR<SUP>+</SUP>]. Under this condition HXT3 promoter is inhibited and the final amount of mRNA of antisense-GFP is decreased. Thus, mRNA of GFP is not combined with mRNA of anti- GFP. Utilizing this system, lactic acid is apparently anticipated to control the expression of GFP and be a trigger of starting translation of GFP protein.</P><P><STRONG>plasmid 2</STRONG></P><P>This plasmid is for assaying the strength of HXT3 promoter. Our team planned to quantify mRNA of antisense-SFGFP by Real-time PCR. To normalize the data, we also decided to measure the amount of PDA1. In addition, to confirm the repression of HXT3 promoter by lactic acid, we decided to prepare suspension of yeast of YPD medium and suspension of YPD medium adding 0.625% lactic acid. Comparing with the control, the function of HXT3 promoter in yeast with lactic acid is expected to be decreased. As a terminator of this part, we used RPL15A terminator. RPL15A terminator is one of the strongest terminators.</P><P><STRONG>plasmid 3</STRONG></P><P>This plasmid is for confirming the new concept, terminatome, works or not if the promoter is changed to ADH1 promoter. We chose 3 terminators for this experiment, RPL15At, PGK1t and GIC1t. As GIC1t includes Spe1 restriction site, we did not submit this part. The alignment of RPL15At and GLC1t is <A href="https://2017.igem.org/Team:Gifu/Basic_Part" target="_blank">here</A>. By measuring FI of GFP in each transformant, the strength of terminators was anticipated to be decided easily like <A href="https://2017.igem.org/Team:Gifu/InterLab" target="_blank">interlab</A>. </P><H3 id="Results"> Results </H3><P><STRONG>Plasmid construction</STRONG></P><P>This year we submitted 3 basic parts, 2 terminators and 1 promoter, and 2 composite parts. Here, we show how we constructed these parts.</P><P>To construct plasmid 3, we amplified 3 terminators from <I>Saccharomyces cerevisiae</I> genome. We designed  primers below to perform PCR.</P><TABLE class="keln_table"><TBODY><TR><TD> RPL15At Fw</TD><TD>5'-TTCTAGAGTAAGCTGGTTGATGGAAAATATAA-3'</TD></TR><TR><TD>RPL15At Rv</TD><TD>5'-CTAGTAGTAGGAAAAACGGGAAGAAAAGG-3'</TD></TR><TR><TD>GIC1t Fw</TD><TD>5'-TTCTAGAGACTAGTTTTCTTCTTTCCTCCTCTT-3'</TD></TR><TR><TD><SUB>GIC1t Rv</SUB></TD><TD>5'-CTAGTAGTAGGTGGAGTACTGTCCGTTCC-3'</TD></TR><TR><TD><SUB>PGK1t Fw</SUB></TD><TD>5'-TTCTAGAGGAATTGAATTGAAATCGATAGAT-3'</TD></TR><TR><TD><SUB>PGK1t Rv</SUB></TD><TD>5'-CTACTAGTATTTTGTTGCAAGTGGGATGA-3'</TD></TR></TBODY></TABLE><P>We utilized Prime STAR max DNA polymerase(Takara bio,  Otsu, Japan) and amplified the region following the protocol of the polymerase. The picture below is the result of PCR. We confirmed all terminators were amplified as expected.</P><P>To clone HXT3 promoter and kozak sequence, we used IDT free synthesis service of gBlock. Here we cloned two kinds of artificial genes to construct plasmid 2. As usual part, this two sequences include prefix and suffix. A-ko stands for ADH1 promoter-kodak sequence-SF-GFP alignment. H-an stands for HXT3 promoter and antisense-GFP. Since yeast is eukaryotes, the amount of GC is really low. Thus we cloned terminators by PCR from genomic DNA.</P><P>Performing biobricks, we constructed two composite parts. First one is the part which consists of HXT3 promoter and antisense-GFP and RPL15At. Second one is the part which includes ADH1 promoter , kosak sequence, SF-GFP gene and RPL15At. We confirmed the success of transformation by colony PCR and sequencing.
</P><H3 id="Conclusion"> Conclusion </H3><P> This year our team completed the composite parts of ADH1p-kozak sequence-SF-GFP-RPL15At and HXT3p-antisense-SF-GFP-RPL15At.  </P><H3 id="Discussion"> Discussion </H3><P><STRONG>Gap-repair cloning</STRONG></P><P>22 bp(prefix and suffix) of homologous recombination region may not be enough to perform plasmid construction effectively. According to<A href="https://www.ncbi.nlm.nih.gov/pubmed/2828185" target="_blank"> this research</A>, Gap-repair cloning is basically stable and efficient. An alternative idea of the cloning method for iGEM should be developed soon. </P><P><STRONG>Strain for this experiment</STRONG></P><P>The reason why our team selected  <I>S. cerevisiae</I>W303 this year is that  <I>S. cerevisiae</I> S288C is not able to grow on Gly-GluN media and the assay of this strain would be difficult. We thought degradation of glycerol is one of the causes of this phenomenon. In this case, W303 is more appropriate compared with S288C.</P><P><STRONG>Terminators</STRONG></P><P>As mentioned above, terminators of yeast contain less than 50% of GC. The ratio of GC affects on stability of DNA alignment. Thus terminators must be cloned from genomic DNA of yeast, not depending on DNA synthesis service. This matter may make it complicated to conduct researches of yeast in this competition. Also, each terminator has its own function. In iGEM catalog, just a few terminators are registered currently. Analysis and submission of terminators of yeast should be conducted more next year.</P><H3 id="Future work"> Future work </H3><P> One of our purposes of the project is detection of the causative bacteria of Hiochi. To detect lactic acid, we can use HPLC which is a more precise way. But if we apply our parts or system into wild type and association selected yeast for making sake, we expect to get some features of sake making microorganisms such as interaction between yeast and lactobacillus in sake. This means we can confirm ancient Japanese sake breweries select microorganisms which has some special characteristics. Sadly, we could not assay our parts this year, but we would like to keep researching this project and show this system is workable in yeast. Finally we would like to emphasize the fact that this system may clarify new quorum sensing-like interactions of yeast.</P><H3 id="Reference"> Reference </H3><P><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5132342/">David M, David D, Jon C, Daniel J. 2016. A common bacterial metabolite elicits prion-based bypass of glucose repression. eLIFE <STRONG>5</STRONG>: e17978.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23654277">Mamoru Y, Yoichiro I, Reiko K, Chine I, Satoshi K, Akinori I, Hisao M. 2013. A Genome-Wide Activity Assessment of Terminator Regions in <I>Saccharomyces cerevisiae</I> Provides a ″Terminatome″ Toolbox. Synthetic Biology <STRONG>2(6)</STRONG>: 337−347.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=A+heritable+switch+in+carbon+source+utilization+driven+by+an+unusual+yeast+prion">Jessica B, Susan L. 2009. A heritable switch in carbon source utilization driven by an unusual yeast prion. GENES &amp; DEVELOPMENT <STRONG>23(19)</STRONG>: 2320–2332.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=An+acid+tale+of+prion+formation%3A+Some+bacteria+use+lactic+acid+to+communicate+with+yeast+cells">Mick T. 2016. An acid tale of prion formation: Some bacteria use lactic acid to communicate with yeast cells. eLIFE <STRONG>5</STRONG>: e22256.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Characterization+of+different+promoters+for+designing+a+new+expression+vector+in+Saccharomyces+cerevisiae
">Siavash P, Verena S, Sara B, Jens N, Jerome M. 2010. Characterization of different promoters for designing a new expression vector in <I>Saccharomyces cerevisiae</I>. Yeast <STRONG>27(11)</STRONG>: 955–964.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/18950675
">Jeong-Ho K. 2009. DNA-binding properties of the yeast Rgt1 repressor. Biochimie <STRONG>91(2)</STRONG>: 300–303.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Integration+of+Transcriptional+and+Posttranslational+Regulation+in+a+Glucose+Signal+Transduction+Pathway+in+Saccharomyces+cerevisiae
">Jeong-Ho K, Valerie B, Hisao M, Mark J. 2006. Integration of Transcriptional and Posttranslational Regulation in a Glucose Signal Transduction Pathway in <I>Saccharomyces cerevisiae</I>. EUKARYOTIC CELL <STRONG>5(1)</STRONG>: 167–173.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/2828185
">Hong M, Sam K, Peter S, David B. 1987. Plasmid construction by homologous recombination in yeast. Gene <STRONG>58(2-3)</STRONG>: 201-216.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26205245
">Omur K, Jens N. 2015. Glucose repression in Saccharomyces cerevisiae. FEMS Yeast Research <STRONG>15(6)</STRONG>: fov068.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Grr1-dependent+Inactivation+of+Mth1+Mediates+Glucose-induced+Dissociation+of+Rgt1+from+HXT+Gene+Promoters
">Karin F, Nathalie S, Tatyana K, Marisela G, Qianzheng Z, Hui-Chu C, Curt W. 2003. Grr1-dependent Inactivation of Mth1 Mediates Glucose-induced Dissociation of Rgt1 from HXT Gene Promoters. Molecular Biology of the Cell <STRONG>14(8)</STRONG>: 3230–3241.</A><A href="https://www.frontiersin.org/articles/10.3389/fevo.2016.00137/full
">Vidhya R, Gordon W, Qingwen F, Minami O, Yan L, Peter L, Lucy J, Linda B. 2016. Inter-Kingdom Modiﬁcation of Metabolic Behavior: [GAR<SUP>+</SUP>] Prion Induction in <I>Saccharomyces cerevisiae</I> Mediated by Wine Ecosystem Bacteria. Frontiers in Ecology and Evolution <STRONG>14</STRONG>:  Article 13.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Lactate+production+yield+from+engineered+yeasts+is+dependent+from+the+host+background%2C+the+lactate+dehydrogenase+source+and+the+lactate+export
">Paola B, Michael S, Luca G, Giuseppe Z, Minoska V, Diethard M, Danilo P. 2006. Lactate production yield from engineered yeasts is dependent from the host background, the lactate dehydrogenase source and the lactate export. Microbial Cell Factories <STRONG>5</STRONG>: 4.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Three+Different+Regulatory+Mechanisms+Enable+Yeast+Hexose+Transporter+(HXT)+Genes+To+Be+Induced+by+Different+Levels+of+Glucose
">Sabire O, Mark J. 1995. Three Different Regulatory Mechanisms Enable Yeast Hexose Transporter (HXT) Genes To Be Induced by Different Levels of Glucose. MOLECULAR AND CELLULAR BIOLOGY <STRONG>15(3)</STRONG>: 1564–1572.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/12861007
">Jeong-Ho K, Jeffrey P, Mark J. 2003. Speciﬁcity and Regulation of DNA Binding by the Yeast Glucose Transporter Gene Repressor Rgt1. MOLECULAR AND CELLULAR BIOLOGY <STRONG>23(15)</STRONG>: 5208–5216.</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Transformation+of+Intact+Yeast+Cells+Treated+with+Alkali+Cations">Hisao I, Yasuki F, Kousaku M, Akira K. 1983. Transformation of Intact Yeast Cells Treated with Alkali Cations. JOURNAL OF BACTERIOLOGY <STRONG>153(1)</STRONG>: 163-168.</A></P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>