<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_NRP-UEA-Norwich_NOSensing"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:NRP-UEA-Norwich/NOSensing">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:NRP-UEA-Norwich/NOSensing&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:NRP-UEA-Norwich/NOSensing&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:NRP-UEA-Norwich/NOSensing&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:NRP-UEA-Norwich/NOSensing" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:NRP-UEA-Norwich/NOSensing</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE>NRP UEA iGEM 2012</TITLE></P><DIV id="colortab" class="ddcolortabs"><UL><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich" title="Home"><SPAN>Home</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Team" rel="dropmenu1_a"><SPAN>Team</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Project" title="Home"><SPAN>Project</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Safety" rel="dropmenu1_a"><SPAN>Safety</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Notebook" rel="dropmenu1_a"><SPAN>Lab Book</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Parts" rel="dropmenu1_a"><SPAN>Parts</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Future" rel="dropmenu1_a"><SPAN>Future Applications</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Quanticare" rel="dropmenu1_a"><SPAN>Quanticare</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Human Outreach" rel="dropmenu1_a"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Attributions" rel="dropmenu1_a"><SPAN>Attributions</SPAN></A></LI></UL></DIV><DIV class="ddcolortabsline"> </DIV><DIV id="dropmenu1_a" class="dropmenudiv_a"><A href="https://2008.igem.org/Team:Edinburgh/Team">Overview</A><A href="https://2008.igem.org/Team:Edinburgh/Team">Step1</A><A href="https://2008.igem.org/Team:Edinburgh/Team">Step2</A></DIV><P><B>Welcome to the NRP UEA iGEM 2012 Wiki Projects Menu</B></P><P>Please choose the relevant link to view an overview of each project!
</P><P><STRONG class="selflink"> Nitric Oxide Sensing &amp; The Hybrid Promoters</STRONG>  |  <A href="/Team:NRP-UEA-Norwich/ComparatorCircuit" title="Team:NRP-UEA-Norwich/ComparatorCircuit"> The Comparator Circuit</A>  |  <A href="/Team:NRP-UEA-Norwich/TheoreticalProjects" title="Team:NRP-UEA-Norwich/TheoreticalProjects"> Theoretical Projects</A></P><P><CENTER><B><FONT size="4pt">Our hybrid promoter hopes to add to the systems already in the registry by creating a hybrid promoter that combines the bacterial promoter PyeaR and the mammalian CArG element , both of which respond to exogenous nitrogenous species. Combining the two would allow a more modular nitric oxide sensor that can be used in mammalian and bacterial cells interchangeably.</FONT><FONT size="2pt">Six new BioBricks produced and submitted to the registry with characterisation from fluorescence-based experiments.</FONT></B></CENTER></P><P>Parts produced from this project:
</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774000" class="external text" rel="nofollow">Bacterial-Mammalian/B-M (PyeaR-CArG) Hybrid Promoter</A> -- <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774001" class="external text" rel="nofollow">Mammalian-Bacterial/M-B (CArG-PyeaR) Hybrid Promoter</A> -- <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774004" class="external text" rel="nofollow">B-M + eCFP</A> -- <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774005" class="external text" rel="nofollow">B-M + RFP</A> -- <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774006" class="external text" rel="nofollow">M-B + eCFP</A> -- <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774004" class="external text" rel="nofollow">M-B + RFP</A></P><P>Parts characterised from this project:
</P><P><A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216005" class="external text" rel="nofollow">PyeaR Promoter</A> -- <A href="http://partsregistry.org/Part:BBa_K381001" class="external text" rel="nofollow">PyeaR Promoter + GFP</A> -- <A href="http://partsregistry.org/Part:BBa_E0420" class="external text" rel="nofollow">enhanced Cyan Fluorescent Protein + RBS + Terminators</A> -- <A href="http://partsregistry.org/Part:BBa_K081014" class="external text" rel="nofollow">Red Fluorescent Protein + RBS + Terminators</A></P><P>Our main project has resulted in the production of a hybrid bacterial and mammalian promoter optimised for induction by nitric oxide, nitrates and nitrites. We have ligated PyeaR, a known bacterial promoter and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216005" class="external text" rel="nofollow">Part BBa_K216005</A> (Cambridge 2009) in the parts registry, with its mammalian counterpart, CArG. The resulting hybrid promoter has been synthesised in two orientations; PyeaR (bacterial, B) upstream of CArG (mammalian, M), nicknamed (B-M); and CArG upstream of PyeaR (M-B). These orientations were submitted to the parts registry as our first two BioBricks.
</P><P>Each orientation of the promoter was ligated to enhanced cyan fluorescent protein (eCFP) and red fluorescent protein (RFP) to produce four new BioBricks which have been submitted to the parts registry. These promoter + fluorescent protein BioBricks have been characterised following transformation into <I>Escherichia coli</I> and induction by potassium nitrate using methods such as flow cytometry, fluorescence-activated cell sorting (FACS) and scanning with a fluorometer. The data from these experiments has proved that our promoter works as we expected it to. We have also transfected M-B + eCFP into a human breast cancer cell line, MCF7, and have proved the system is flexible and can be used in both eukaryotes and prokaryotes.
</P><P>We believe the promoters we have produced have relevant uses in cancer therapeutics, soil fertilisation and detection of emissions from industries such as construction.
</P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-1 tocsection-2"><A href="#Characterisation_of_Existing_BioBrick:_BBa_K381001_.28PyeaR_.2B_GFP_BioBrick.29"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Characterisation of Existing BioBrick: BBa_K381001 (PyeaR + GFP BioBrick)</SPAN></A></LI><LI class="toclevel-1 tocsection-3"><A href="#Creating_Novel_Hybrid_Promoters:_B-M_and_M-B_into_BioBricks"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Creating Novel Hybrid Promoters: B-M and M-B into BioBricks</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Studies_into_the_effect_of_the_hybrid_promoter_on_growth_of_E._coli_competent_cells"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Studies into the effect of the hybrid promoter on growth of E. coli competent cells</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Generation_of_B-M_and_M-B_with_eCFP_and_RFP"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Generation of B-M and M-B with eCFP and RFP</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Qualitative_Results"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Qualitative Results</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Fluorometer_Experiments"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Fluorometer Experiments</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Flow_Cytometry"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Flow Cytometry</SPAN></A></LI><LI class="toclevel-3 tocsection-9"><A href="#B-M_.2B_RFP_Flow_Cytometry_Data"><SPAN class="tocnumber">4.3.1</SPAN><SPAN class="toctext">B-M + RFP Flow Cytometry Data</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#M-B_.2B_eCFP_Flow_Cytometry_Data"><SPAN class="tocnumber">4.3.2</SPAN><SPAN class="toctext">M-B + eCFP Flow Cytometry Data</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#Transfection_of_M-B_.2B_eCFP_into_the_MCF7_human_breast_cancer_cell_line"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">Transfection of M-B + eCFP into the MCF7 human breast cancer cell line</SPAN></A></LI><LI class="toclevel-1 tocsection-12"><A href="#Discussion"><SPAN class="tocnumber">6</SPAN><SPAN class="toctext">Discussion</SPAN></A></LI><LI class="toclevel-1 tocsection-13"><A href="#Future_Experiments"><SPAN class="tocnumber">7</SPAN><SPAN class="toctext">Future Experiments</SPAN></A></LI><LI class="toclevel-1 tocsection-14"><A href="#References"><SPAN class="tocnumber">8</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 1.</B></I><I>A graphical representation of PyeaR. In the higher image PyeaR's activity is being repressed by both Nar and NsrR preventing transcription and the ultimate expression of a reporter. In the lower image nitrate/nitrite molecules have inhibited the activity of Nar, and nitric oxide has inhibited activity of NsrR, allowing for transcription to occur and subsequent expression of a reporter.</I></DIV></DIV></DIV><P>The University of East Anglia hosts many research teams whose work focuses on studying nitrogenous species and the way in which bacteria use and modify those species. One major problem that is faced by these teams involves working with nitric oxide (NO), as NO is highly reactive with a low half life, therefore making it difficult to trace and quantitatively measure accurately. Many of the methods currently used to measure NO levels are unable to distinguish between homogenous species, such as nitrates (NO3) and nitrites (NO2), therefore the figure given for NO levels is often inaccurate as other nitrogenous species are taken into account.
</P><P>
The ability to be able to accurately detect NO levels is one with a great deal of potential for the future. Nitric oxide has been noted as a possible cancer therapy due to its physiological use as an apoptosis inducer by macrophages, however NO is also known to be used by cancerous cells to establish a baseline and use it to induce apoptosis and promote proliferation of a tumour; being able to accurately sense nitric oxide and go on to act on that information could be very useful to prevent the NO baseline being established by cancerous cells, but to also use NO for its apoptosis-inducing abilities. There are also other potential applications in the construction business, in 2008 Modern Building Services published an article regarding legislation released by the Building Regulation Establishment Environmental Assessment Method (BREEAM) encouraging construction companies to monitor their NO output as emission of NO can contribute to smog and acid rain levels; the ability to accurately detail levels of NO being released in these circumstances would be highly useful.
</P><P>
Nitric oxide is an extremely physiologically relevant molecule found within both eukaryotes and prokaryotes, where it is utilised by different enzymes and systems for various roles. The aim of the experiment was to devise a hybrid promoter that could be applied to eukaryotes and prokaryotes in order to begin to more accurately sense NO and report on its specific levels.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 2.</B></I><I>A graphical representation of the hybrid promoter in its two orientations; the top image shows B-M with the bottom image showing M-B. Also included are the restriction sites and their location within the system, with the elements being read from left to right.</I></DIV></DIV></DIV><P>The hybrid promoter was designed to take on both bacterial and mammalian elements in order to be compatible with both bacterial and mammalian cells. After research around the subject and searching the parts registry a promoter known as PyeaR was decided upon as the bacterial element of the hybrid promoter; PyeaR is found in the <I>yeaR/yoaG</I> operon of <I>Escherichia coli</I> and is associated with induction by nitric oxide, nitrates and nitrites (Lin <I>et al.</I>, 2007). PyeaR is repressed by two main repressors; Nar, which is regulated by nitrates and nitrites; and NsrR, which is regulated by nitric oxide (Figure 1.). One of the key elements of PyeaR is that it is not repressed in aerobic conditions, allowing for easier carrying out of experiments. The PyeaR aspect of the hybrid promoter has been known throughout the project as the bacterial promoter, or simply B.
</P><P>
The mammalian element of the hybrid promoter was produced by nine CArG elements (repeated elements of CC(A/T)(6)GG), a promoter previously used synthetically for nitric oxide synthase as a cancer therapy (Worthington <I>et al.</I>, 2005) and developed from the EGR1 gene for early growth response protein 1 (Scott <I>et al.</I>, 2002). The CArG aspect of the hybrid promoter has been known throughout the project as the mammalian promoter, or simply M.
</P><P>
Following identification of the two elements of the hybrid promoters the B (PyeaR) and M (CArG) aspects were ligated to one another in two orientations; B upstream of M (B-M) and M upstream of M (M-B) (Figure 2.). The hybrid promoters were synthesised in a pUC57 backbone with the standard iGEM restriction sites of EcoR1/Xba1 upstream of the promoter, and Spe1/Pst1 downstream of the promoter. A BamH1 restriction site was included in between the B and M sequences in order to allow for the B and M elements to be separated, as well as for easy verification of the promoter having been ligated into the iGEM backbone in future experiments (as BamH1 does not already exist in the pSB1C3 backbone).
</P><H2><SPAN class="mw-headline" id="Characterisation_of_Existing_BioBrick:_BBa_K381001_.28PyeaR_.2B_GFP_BioBrick.29">Characterisation of Existing BioBrick: BBa_K381001 (PyeaR + GFP BioBrick)</SPAN></H2><P>In order to begin to develop experiments to characterise the hybrid promoters + fluorescent proteins experiments were also carried out on a BioBrick containing PyeaR + GFP (Part <A href="http://partsregistry.org/Part:BBa_K381001" class="external text" rel="nofollow">BBa_K381001</A>, Bristol 2010). In these experiments transformed <I>E. coli</I> was inoculated into liquid culture, which in turn had varying potassium nitrate concentrations added to it. They were then left to grow before being spun down and viewed under a UV box in order to observe. The different concentrations of potassium nitrate that the transformed <I>E. coli</I> was grown in were: 0 mM, 1 mM, 2 mM, 3 mM, 4 mM, 5 mM, 6 mM, 8 mM, 10 mM.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 3.</B></I><I>A photograph of spun-down media containing potassium nitrate (to induce the promoter) and E. coli transformed by PyeaR + GFP (art BBa_K381001). Each sample was grown with a different concentration of potassium nitrate, from left to right: 0 mM, 1 mM, 2 mM, 3 mM, 4 mM, 5 mM, 6 mM, 8 mM, 10 mM.</I></DIV></DIV></DIV></DIV><P>The figure suggests that fluorescent proteins have been expressed by the bacteria grown in media containing potassium nitrate due to the fluorescence shown under the UV box. It also suggests that different concentrations of potassium nitrate correlate with different intensities of expression due to the observable differences in fluorescence as the tubes are viewed from left to right, going up the gradient. The negative control of 0 mM potassium nitrate appears to show no fluorescence, suggesting it is indeed the potassium nitrate that is inducing the promoter.
</P><H2><SPAN class="mw-headline" id="Creating_Novel_Hybrid_Promoters:_B-M_and_M-B_into_BioBricks">Creating Novel Hybrid Promoters: B-M and M-B into BioBricks</SPAN></H2><P>The DNA for the synthesised genes of B-M and M-B had been supplied in the pUC57 backbone, therefore it was necessary for B-M and M-B to be digested from the pUC57 backbone and ligated into the pSB1C3 backbone. The synthesised gene was transformed into competent ’’E. coli’’ cells, which in turn were grown on agar plates containing 100 µg/ml ampicillin (due to pUC57 containing ampicillin resistance); colonies that had grown were then inoculated into liquid culture, and the liquid culture was subsequently mini-prepped using either the Bioline ISOLATE Plasmid DNA Mini Kit or the Promega Wizard® Plus SV Minipreps DNA Purification System. The DNA that had been extracted through mini-preps and the pSB1C3 backbone, as provided by the iGEM registry, were then digested using EcoR1 and Pst1 and a ligation was carried out using standard assembly protocol. The product of ligation was then transformed into competent <I>E. coli</I> cells, which were grown on agar plates containing 2.5 µg/ml chloramphenicol (due to pSB1C3 containing chloramphenicol resistance); this was done to eliminate any bacteria that had been transformed with undesirable ligation products. 
</P><P>The colonies that had grown were then grown in liquid culture and mini-prepped in order to extract the DNA; the extracted DNA was then sent for sequencing, and the returned sequenced matched the expected sequence. The DNA was then sent to the parts registry as the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774000" class="external text" rel="nofollow">Bacterial-Mammalian Hybrid Promoter</A> and the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774001" class="external text" rel="nofollow">Mammalian-Bacterial Hybrid Promoter</A>. The BioBricks for B-M and M-B were then used for further experiments, including ligation with a fluorescent protein reporter and growth studies.
</P><P>From weeks one through to five the team worked on producing the first BioBricks of the hybrid promoter. Despite this proving difficult due to various reasons such as low amounts of DNA being produced from early mini-preps and ligation strategies not working, <I>E. coli</I> transformed by the BioBrick DNA was successfully grown and proven to have the relevant antibiotic resistance by the beginning of <A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Week6" class="external text" rel="nofollow">week six</A>.
</P><H3><SPAN class="mw-headline" id="Studies_into_the_effect_of_the_hybrid_promoter_on_growth_of_E._coli_competent_cells">Studies into the effect of the hybrid promoter on growth of <I>E. coli</I> competent cells</SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 4.</B></I><I>A calibration curve produced using E. coli alpha cells grown over time. The inoculations were run through a spectrophotometer at OD 600 nm every hour and then a sample was plated in order to calculate CFU/ml over time.</I></DIV></DIV></DIV><P>As part of the characterisation of the hybrid promoters, as well as PyeaR alone (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K216005" class="external text" rel="nofollow">Part BBa_K216005</A>, studies were conducted to see if the growth of the <I>E. coli</I> was affected by transformation of the promoters. 
</P><P>
To being untransformed <I>E. coli</I> were inoculated into liquid culture and left to grow overnight at 37 ºC before being diluted down. The diluted liquid cultures were run through a spectrophotometer at OD 600 nm and their absorbance noted, before each culture was plated and once again left overnight at 37 ºC; the next morning the colonies on the plates were counted, allowing a calibration curve of absorbance at OD 600 nm against colony-forming units per ml (CFU/ml) to be set up (Figure 4.)
</P><P>
Following establishment of the calibration curve liquid cultures were made up of <I>E. coli</I> that was:
</P><P>. Untransformed (otherwise referred to as alpha cells)
</P><P>. Transformed by B-M
</P><P>. Transformed by M-B
</P><P>. Transformed by PyeaR
</P><P>
These cultures were then diluted down to the same starting level (an OD 600 nm absorbance of 0.2 +/- 0.1) and cuvettes filled with LB media were inoculated. The cuvettes were then placed in a spectrophotometer every hour and the absorbance at 600 nm was established; in between readings the cuvettes were placed into a 37 ºC incubator in order to encourage bacterial growth. This was repeated for 12 hours and the absorbance readings compared to the calibration curve in order to give data on the level of growth of <I>E. coli</I> transformed with the different promoters/untransformed over time.
</P><P>We found that there was a significant difference between Alpha cells and PyeaR cells. Initially, Alpha cells had a greater growth rate, but after the third hour into the study, the growth rate of PyeaR was faster than that of Alpha cells. The overall growth rate of PyeaR cells was significantly faster that Alpha cells (Levenes Test, F = 1.009  p = 0.372; T Test,  t = 4.196, df = 4, p = 0.014).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 5</B></I><I>Growth of PyeaR transformed E.coli cells relative to Alpha cell (untransformed cells. Error bars show the standard deviation between the three repeats.  For clarity reasons, lines of best fit are not shown.</I></DIV></DIV></DIV></DIV><P>The growth pattern and rate of E.coli cells with or without transformation with B-M and M-B show little difference. Any differences in growth rate were not significant. There was lots of overlap. As previously described, there was a significant difference between the growth rate of PyeaR and Alpha cells. There was also a significant difference between MB/BM and PyeaR cells. The statistical results can be seen in Table 1
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 6:</B></I><I>Growth over 12 hours of Alpha, M-B and B-M. Error bars and lines of best fit are not shown for clarity reasons.</I></DIV></DIV></DIV></DIV><P><I><B>Table 1:</B></I><I>ANOVA readings of statistical differences between Alpha (1) PyeaR (2), MB (3) and BM (4).</I></P><P>From all the above graphs, it can be seen that with the starting concentration of cells as high as they are, the cultures are in exponential stage and do not undergo lag phase. A further growth study will be carried out on purely the lag phase with lower starting concentrations. As the starting absorbances here are approximately 0.2 at a wavelength of 600nm, the lag phase study will involve starting absorbances of 0.04 and lower.
</P><P>
Following the above study, we found that a lag phase only study needed to be carried out to see if there was a significant difference in the lag phase.  Again the study protocol was the same except that the starting concentration absorbances at 600nm was lowered to &lt;0.04. It was extremely difficult to keep the absorbances ranges within 0.005 so the range is actually 0.3±0.1.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 7</B></I><I>A mean average of all the data; using the data from the calibration curve, the absorbances were converted to colony forming units per ml (CFU/ml). The trend lines of alpha cells, BM/MB and PyeaR transformed cells are shown within this order from highest to lowest trendlines. One single trendline was used to represent BM and MB because the actual trendlines were extremely similar.</I></DIV></DIV></DIV></DIV><P>Using the initial concentrations of 0.3±0.1 showed that there is little difference between the growth rates. Using statistical analysis, it was found that there was no significant difference between any of the transformed cells relative to Alpha cells or to each other (Anova, p &gt; 0.05). From this study we have found that changes in growth occur during exponential growth phase and not the lag growth phase.
</P><H2><SPAN class="mw-headline" id="Generation_of_B-M_and_M-B_with_eCFP_and_RFP">Generation of B-M and M-B with eCFP and RFP</SPAN></H2><P>In order to test the activity of the hybrid promoters a reporter needed to be ligated. As the hybrid promoter did not already contain a ribosome binding site (RBS) both the RBS and the reporter were needed to be ligated to the promoter; in order to help improve experimental efficiency the parts registry was searched for relevant reporters that also contained an RBS. In <A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Week3" class="external text" rel="nofollow">week three</A> two reporters were identified as <A href="http://partsregistry.org/Part:BBa_E0420" class="external text" rel="nofollow">BBa_E0420</A>, a BioBrick for enhanced CFP (eCFP) + RBS + terminators, and <A href="http://partsregistry.org/Part:BBa_K081014" class="external text" rel="nofollow">BBa_K081014</A>, a BioBrick for RFP + RBS + terminators.
</P><P>Once the B-M and M-B BioBricks had been created in week six work began in earnest on the fluorescent proteins and ligating the promoters to them in order to begin characterisation. Due to many set-backs with low levels of DNA and having to order more BioBricks from the registry, a successful ligation of the promoter to a fluorescent protein reporter was finally achieved in <A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Week10" class="external text" rel="nofollow">week ten</A>. In order to carry out the ligation the promoter was first digested using Spe1 and Pst1 in order to linearise the backbone downstream of the promoter; the fluorescent proteins were digested using Xba1 and Pst1 in order to remove the insert. A ligation was then carried out using standard assembly protocol and the ligation products were transformed into <I>E. coli</I> competent cells, which in turn were grown on agar plates 2.5 µg/ml chloramphenicol (due to pSB1C3 containing chloramphenicol resistance). 
</P><P>In order to quickly identify colonies of bacteria containing the promoter, RBS and reporter in a likely correct sequence a range of colonies were inoculated into media also containing potassium nitrate (KNO3) solution; this was done in order to inhibit the Nar repressor in PyeaR and result in activation of the promoter/induction of transcription/expression of the fluorescent protein reporter. Samples of the inoculated media containing KNO3 were then added to an eppendorf and spun down to form a pellet, which was viewed under a UV box and observed for fluorescence. After a week of various ligation and transformation experiments both promoters were successfully ligated to both fluorescent proteins and fluorescence of eCFP and RFP was observed under a UV box.
</P><P>From these experiments four more BioBricks were produced and submitted to the registry: <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774004" class="external text" rel="nofollow">B-M + eCFP</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774005" class="external text" rel="nofollow">B-M + RFP</A>, <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774006" class="external text" rel="nofollow">M-B + eCFP</A>, and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774004" class="external text" rel="nofollow">M-B + RFP</A>.
</P><P>In order to characterise the hybrid promoters ligated to fluorescent proteins a number of experiments were carried out to measure the level of fluorescent output at different concentrations of KNO3 (used to induce the promoter's activity). All of these experiments were carried out in <A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Week11" class="external text" rel="nofollow">week eleven</A></P><H3><SPAN class="mw-headline" id="Qualitative_Results">Qualitative Results</SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 8.</B></I><I>A photograph of spun-down media containing potassium nitrate (to induce the hybrid promoters) and E. coli transformed by M-B + eCFP. The concentrations of potassium nitrate added to the media were, from left to right: 100 mM, 50 mM, 10 mM, 0 mM.</I></DIV></DIV></DIV></DIV><P>This figure appears to show fluorescence from the spun-down pellets of each hybrid promoter + fluorescent protein that had been grown in media contain potassium nitrate; it can be inferred from this that the potassium nitrate has induced the hybrid promoter, resulting in expression of the fluorescent protein reporter. The negative control of 0 mM potassium nitrate appears to show no fluorescence, suggesting it is indeed the potassium nitrate that is inducing the promoter.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 9.</B></I><I>A photograph of spun-down media containing potassium nitrate (to induce the hybrid promoters) and E. coli transformed by the four BioBricks containg the promoters and fluorescent proteins; the photograph has been taken from a UV box. From left to right: B-M + eCFP, M-B + eCFP, B-M + RFP, M-B + RFP.</I></DIV></DIV></DIV></DIV><P>The figure appears to show fluorescence from the spun-down pellets of each hybrid promoter + fluorescent protein that had been grown in media contain potassium nitrate; it can be inferred from this that the potassium nitrate has induced the hybrid promoter, resulting in expression of the fluorescent protein reporter.
</P><H3><SPAN class="mw-headline" id="Fluorometer_Experiments">Fluorometer Experiments</SPAN></H3><P>The main characterisation of the BioBricks was carried out using a fluorometer. Five tubes of media containing 200 µL transformed bacteria and potassium nitrate were grown for each BioBrick in concentrations as follows:
</P><P>. B-M + RFP with 0 mM, 5 mM, 10 mM, 15 mM and 20 mM potassium nitrate
</P><P>. B-M + eCFP with 0 mM, 5 mM, 10 mM, 15 mM and 20 mM potassium nitrate
</P><P>. M-B + RFP with 0 mM, 5 mM, 10 mM, 15 mM and 20 mM potassium nitrate
</P><P>. M-B + eCFP with 0 mM, 5 mM, 10 mM, 15 mM and 20 mM potassium nitrate
</P><P>
The samples were then spun down in a centrifuge and resuspended in Tris buffer before being subjected to sonication in order to lyse the cells. The resulting solution was then spun down in a centrifuge again before the supernatant was extracted (in order to separate the proteins from the rest of the cell). The supernatant was then run through a fluorometer in order to measure the intensity of fluorescence as a method to gauge the activity of the hybrid promoters. RFP-containing samples were excited with a wavelength of 560 nm and the emission measured from 600 – 650 nm; eCFP-containing samples were excited with a wavelength of 410 nm and emission measured from 440-490 nm.
</P><P>
These characterisation experiments gave rise to the characterisation which is now available on the main page for all hybrid promoter + fluorescent protein parts on the registry, and the experience section of the hybrid promoters alone on the registry.
</P><P><B>NOTE: All data for the fluorometer has had the equivalent 0 mM reading subtracted from it in order to nulify the affects of light scattering due to cell debris</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 10.</B></I><I>A graph of the intensity of CFP fluorescence at different wavelengths ranging from 440 - 500 nm where the samples were excited at 410 nm. The samples of E. coli were transformed by the B-M + CFP BioBrick and grown overnight in different concentrations of potassium nitrate (0 mM, 5 mM, 10 mM, 15 mM and 20 mM) before being lysed in order to release proteins from the cells for fluorometer analysis.</I></DIV></DIV></DIV></DIV><P>The graph above shows the flourescence measured from the expression of eCFP due to the response of the bacterial-mammalian promoter to  different concentrations of potassium nitrate.  The wavelength reading which corresponds to eCFP is between 440-500nm.  The graph clearly demonstrates that between 0mN and 15mM there is a proportional relationship between fluorescence intensity and potassium nitrate concentration. There appears to be a sharp increase in fluorescence intensity between 5mM and 10mM, and the rate at which intensity increase gradually decreases so that there is only a small increase between 15mM and 20mM.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 11.</B></I><I>A graph of the intensity of CFP fluorescence at different wavelengths ranging from 440 - 500 nm where the samples were excited at 410 nm. The samples of E. coli were transformed by the M-B + CFP BioBrick and grown overnight in different concentrations of potassium nitrate (0 mM, 5 mM, 10 mM, 15 mM and 20 mM) before being lysed in order to release proteins from the cells for fluorometer analysis.</I></DIV></DIV></DIV></DIV><P>
The graph above shows the flourescence measured from the expression of eCFP due to the response of the mammalian-bacterial promoter to  different concentrations of potassium nitrate.  The wavelength reading which corresponds to eCFP is between 440-500nm.  The graph clearly demonstrates that between 0mN and 15mM there is a proportional relationship between fluorescence intensity and potassium nitrate concentration. It can be noted that at a 20mM concentration the intensity of fluorescence sharply decreases back down to the level of 5mM potassium nitate concentration. This may be due to the cell overexpressing eCFP up to the point at which the excess protein begins to form inclusion bodies which can no longer fluoresce; alternatively, this could be due the potassium nitrate concentration reaching the critical concentration at which it becomes toxic to the cell. This data differs to the readings taken from the bacterial-mammalian promoter ligated to eCFP, as well as the hybrid promoters to RFP, which may suggest there is a difference in the molecular mechanisms that these promoters function by; however at this point the change in intensity at 20mM is inconclusive and is an area which we would like to look into further. 
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 12.</B></I><I>A graph comparing the intensity of CFP fluorescence at 475 nm where the samples were excited at 410 nm. The samples of E. coli were transformed by the B-M + CFP BioBrick and grown overnight in different concentrations of potassium nitrate (0 mM, 5 mM, 10 mM, 15 mM and 20 mM) before being lysed in order to release proteins from the cells for fluorometer analysis.</I></DIV></DIV></DIV></DIV><P>
We were initially unsure of the effect that the orientation of the bacterial (pYEAR) and the mammalian (CaRG) genes would have in gene expression, therefore we synthesised two hybrid promoters in the orientation bacterial-mammalian and mammalian-bacterial. The graph above compares the intensity of fluorescence of the two hybrid promoters (BBa_K774004 and BBa_K774006)  ligated to eCFP. There is a distinct difference between the intensity of fluorescence produced by the bacterial-mammalian promoter and the mammalian-promoter which is something that we would like to look into further. It is particularly interesting that at an intensity of 109a.u. the mammalian-bacterial promoter returns to the same level of intensity as the apparent maxiumum of the bacterial-mammalian promoter at 40a.u.
</P><P>
We also ligated both of our hybrid promoters to Red Fluorescent Protein (RFP), and the results can be seen below.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 13.</B></I><I>A graph of the intensity of RFP fluorescence at different wavelengths ranging from 600 - 650 nm where the samples were excited at 560 nm. The samples of E. coli were transformed by the B-M + RFP BioBrick and grown overnight in different concentrations of potassium nitrate (0 mM, 5 mM, 10 mM, 15 mM and 20 mM) before being lysed in order to release proteins from the cells for fluorometer analysis.</I></DIV></DIV></DIV></DIV><P>
The graph above shows the flourescence measured from the expression of RFP due to the response of the bacterial-mammalian promoter to  different concentrations of potassium nitrate.  The wavelength reading which corresponds to RFP is between 600-650nm.  The graph clearly demonstrates that between 0mN and 15mM there is a proportional relationship between fluorescence intensity and potassium nitrate concentration. A similar pattern can be seen here as for  the mammalian- bacterial promoter with eCFP as at a 20mM concentration the intensity of fluorescence sharply decreases, however the intensity here decreases down to a level between 10mM and 15mM  potassium nitate concentration. There is also only a small difference between 5mM and 10mM potassium nitrate, which differs to the pattern seen with the bacterial-mammalian promoter ligated to eCFP.  As previously stated, this may be due to the cell overexpressing eCFP up to the point at which the excess protein begins to form inclusion bodies which can no longer fluoresce; alternatively, this could be due the potassium nitrate concentration reaching the critical concentration at which it becomes toxic to the cell. This data differs to the readings taken from the bacterial-mammalian ligated to eCFP, as well as the hybrid promoters to RFP, which may suggest there is a difference in the molecular mechanisms that these promoters function by; however at this point the change in intensity at 20mM is inconclusive and is an area which we would like to look into further.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 14.</B></I><I>A graph of the intensity of RFP fluorescence at different wavelengths ranging from 600 - 650 nm where the samples were excited at 560 nm. The samples of E. coli were transformed by the M-B + RFP BioBrick and grown overnight in different concentrations of potassium nitrate (0 mM, 5 mM, 10 mM, 15 mM and 20 mM) before being lysed in order to release proteins from the cells for fluorometer analysis.</I></DIV></DIV></DIV></DIV><P>
The graph above shows the flourescence measured from the expression of RFP due to the response of the mammalian-bacterial promoter to  different concentrations of potassium nitrate.  The wavelength reading which corresponds to RFP is between 600-650nm.  The graph clearly demonstrates that between 0mN and 15mM there is a proportional relationship between fluorescence intensity and potassium nitrate concentration. It has been found that for all BioBricks apart from the mammalian-bacterial promoter ligated to eCFP at a 20mM concentration the intensity of fluorescence sharply decreases.  
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption"><I><B>Figure 15.</B></I><I>A graph comparing the intensity of RFP fluorescence at different wavelengths ranging at 610 nm where the samples were excited at 560 nm. The samples of E. coli were transformed by the B-M + RFP BioBrick and grown overnight in different concentrations of potassium nitrate (0 mM, 5 mM, 10 mM, 15 mM and 20 mM) before being lysed in order to release proteins from the cells for fluorometer analysis.</I></DIV></DIV></DIV></DIV><P>
As previously stated, we were initially unsure of the effect that the orientation of the bacterial (pYEAR) and the mammalian (CaRG) genes would have in gene expression, therefore we synthesised two hybrid promoters in the orientation bacterial-mammalian and mammalian-bacterial. The graph above compares the intensity of fluorescence of the two hybrid promoters (BBa_K774007 and BBa_K774005)  ligated to RFP. There appears to be no pattern if the difference between the intensities of these two promoters; however both promoters do show a decrease in intensity at 20mM potassium nitrate and decrease from a maximum intensity of 82a.u. (bacterial-mammalian) and 66a.u. to approximately 36a.u.
</P><H3><SPAN class="mw-headline" id="Flow_Cytometry">Flow Cytometry</SPAN></H3><P>Three tubes of media were inoculated with E. coli transformed by the <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774005" class="external text" rel="nofollow">B-M + RFP</A>  or <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K774006" class="external text" rel="nofollow">M-B + eCFP</A> BioBrick. Each tube then had potassium nitrate added to it at different concentrations; 0 mM, 1 mM and 10 mM respectively. The E. coli were grown over night and then spun down, fixed in 4% PFA and re-suspened in 500ul PBS. The samples were then analysed in an Acuri C6 or BD Aria II flow cytometer.
</P><P><A href="https://2012.igem.org/Team:NRP-UEA-Norwich/Protocol" class="external text" rel="nofollow">Full Protocol</A></P><P><I>Note: This was the first time the flow cytometers at the University of East Anglia had been used with E. coli</I></P><H4><SPAN class="mw-headline" id="B-M_.2B_RFP_Flow_Cytometry_Data">B-M + RFP Flow Cytometry Data</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 16.</B></I><I>Flow cytometry data for B-M RFP transformed E. coli that were grown in either 0 mM, 1 mM or 10 mM potassium nitrate. Top row: Scatter plot of raw data and gating strategy utilised. Middle row: RFP Fluorescence profiles of samples. Lower left: Fluorescence profiles of the three samples overlay on the same plot.</I></DIV></DIV></DIV></DIV><P>
In this figure the image on the lower left of the fluorescence profiles overlayed on one another suggests that there is a slight difference in fluorescence intensity between the samples grown in media containing 0 mM potassium nitrate and 1 mM potassium nitrate, however there is a very noticeable difference in fluorescence between samples grown in 1 mM potassium nitrate and 10 mM potassium nitrate.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 17.</B></I><I>Flow cytometry fluorescence data: B-M RFP transformed E. coli that were grown in either 0 mM, 1 mM or 10 mM potassium nitrate.</I></DIV></DIV></DIV></DIV><P>The data in this figure appears to corroborate with the data in Figure 13, showing a small difference in fluorescence between 0 mM potassium nitrate and 1 mM potassium nitrate, and then a much larger difference in fluorescence intensity following induction by 10 mM potassium nitrate.
</P><H4><SPAN class="mw-headline" id="M-B_.2B_eCFP_Flow_Cytometry_Data">M-B + eCFP Flow Cytometry Data</SPAN></H4><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 18.</B></I><I>Flow cytometry data for M-B eCFP transformed E. coli that were grown in either 0 mM, 1 mM or 10 mM potassium nitrate. Top row: Scatter plot of raw data and gating strategy utilised. Middle row: RFP Fluorescence profiles of samples.</I></DIV></DIV></DIV></DIV><P>The data in this figure suggests a slight increase in fluorescence intensity from 0 mM to 1 mM, and then a higher increase in fluorescence intensity from 1 mM to 10 mM.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:302px;"><DIV class="thumbcaption"><I><B>Figure 19.</B></I><I>Flow cytometry fluorescence data: M-B eCFP transformed E. coli that were grown in either 0 mM, 1 mM or 10 mM potassium nitrate.</I></DIV></DIV></DIV></DIV><P>The data in this figure corroborates with the data in figure 15 in suggesting a slight increase in fluorescence intensity from 0 mM to 1 mM, and then a higher increase in fluorescence intensity from 1 mM to 10 mM.
</P><H2><SPAN class="mw-headline" id="Transfection_of_M-B_.2B_eCFP_into_the_MCF7_human_breast_cancer_cell_line">Transfection of M-B + eCFP into the MCF7 human breast cancer cell line</SPAN></H2><P>In order to ascertain the flexibility of the hybrid promoter and help prove our original aim of producing a promoter that could be used in both eukaryotes and prokaryotes correct, the DNA for M-B + eCFP was transfected into the MCF7 human breast cancer cell line. An experiment was set up using S-Nitroso-N-acetyl-DL-penicillamine (SNAP), a nitric oxide donor, in order to induce expression of the fluorescent protein. A six-well transfection slide was produced containing:
</P><P>The cells were then left for a day before they were imaged with a fluorescence microscope in order to observe expression of eCFP.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I><B>Figure 20.</B></I><I> Transfection of MCF7 cells with images taken via a Zeiss CCD2 inverted microscope to detect CFP expression. Images in the left two columns are controls and have not been transfected, images in the right two columns have been transfected with M-B + CFP DNA; SNAP is a nitric oxide donor, therefore addition of SNAP was used to try and induce promoter activity.</I></DIV></DIV></DIV></DIV><P>The figure appears to show fluorescence in the mammalian cells that had been transfected with M-B + CFP compared with the cells that had not been transfected. The figure also appears to show stronger fluorescence in the cells that had been transfected with M-B + CFP and had been grown with the nitric oxide donor SNAP compared to the cells that had been transfected with M-B + CFP and grown without SNAP.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I><B>Figure 21.</B></I><I> Transfection of MCF7 cells with images taken via a Zeiss CCD2 inverted microscope to detect CFP expression. Image shows mammalian cells transfected with M-B + CFP DNA however no SNAP (a nitric oxide donor) has been administered.</I></DIV></DIV></DIV></DIV><P>Close up image of what appears to be eCFP fluorescence from a transfected MCF7 cell.
</P><P>The figure appears to show low levels of fluorescence in the cells as there are small blue circles appearing in the cells indicative of CFP expression. MCF7 cells do endogenously express endothelial cell nitric-oxide synthase (ecNOS), which can produce low levels of NO. Perhaps it is endogenous NO production which is inducing the M-B promoter to express low levels of eCFP.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption"><I><B>Figure 22</B></I><I> Transfection of MCF7 cells with images taken via a Zeiss CCD2 inverted microscope to detect CFP expression. Image shows mammalian cells transfected with M-B + CFP DNA where SNAP (a nitric oxide donor) has been administered.</I></DIV></DIV></DIV></DIV><P>
The figure suggests that expression of CFP has occurred due to the large blue shape appearing in the centre of the image following the use of a filter to view fluorescent cyan more easily.
</P><P>Future experiments are needed to confirm these results and also to provide more information.
</P><H2><SPAN class="mw-headline" id="Discussion">Discussion</SPAN></H2><P>Through the experiments detailed above a lot of information has been collated regarding the hybrid promoters and their activity when induced with potassium nitrate. Through all figures from the fluorometer, flow cytometry and qualitative results showing fluorescence as a result of potassium nitrate induction we can conclude that the hybrid promoter in both orientations (B-M and M-B) does indeed react to nitrates and result in translation and expression of the downstream reporter. We can assume due to the transcriptional repressors associated with PyeaR (Figure 1.) that similar results would be given by nitrites, however future experimentation is recommended to test this theory and fully characterise this aspect of the promoter.
</P><P>
In Figures 20, 21 and 22 it is shown that human breast cancer cells (MCF7) can be transfected with the M-B orientation of the promoter and that fluorescence does not occur in untransfected cells, however does occur in the transfected cells; the figures also appears to show that fluorescence intensifies upon addition of a nitric oxide donor. This data would suggest that the hybrid promoter indeed responds to nitric oxide as well as nitrates, however future experiments are needed to prove this. This appears to show that the hybrid promoter is indeed flexible as we first intended it to be, and systems can be developed in bacteria and then applied to mammalian cells with similar affects.
</P><P>
In Figures 11, 13 and 14 the data shows the intensity of fluorescence increasing as the concentration of potassium nitrate added to the media the <I>E. coli</I> were grown in increases; this trend continues until the 15 mM sample (which appears to be the highest), before dramatically dropping in the case of the 20 mM potassium nitrate samples. One of the ideas surrounding this would be the toxicity levels of potassium nitrate, however in Figure 5 <I>E. coli</I> samples are shown to be fluorescing (and thus likely still be alive) after being subjected to 100 mM potassium nitrate. Another possibility may be the formation of inclusion bodies due to the vast quantities of fluorescent proteins being produced, leading to their aggregation and thus inability to fluoresce as expected. From the data given it can likely concluded that the maximum level of fluorescence can be attained from potassium nitrate levels between 15 mM and 20 mM, though future experimentation with narrower parameters in this area will be needed to conclude where the maximum tolerable level of potassium nitrate is found.
</P><P>
In Figure 12 there appears to be a stark difference in CFP fluorescence intensity when M-B and B-M are compared, with M-B giving much higher intensities than B-M on all occasions, however in Figure 15 where the same experiments have been carried out using RFP instead of CFP the differences are not conclusively favouring M-B or B-M as the more active promoter. It is possible, therefore, that in the CFP experiments that there was a difference in the amount of cultured media that was used to inoculate fresh tubes (with more <I>E. coli</I> transformed by M-B + CFP being added than those transformed with B-M + CFP, resulting in more growth and thus increased production of CFP).
</P><P>
In Figure 7 and through statistical analysis it is shown that there is no significant difference between the growth of cells transformed by M-B, B-M or PyeaR compared to untransformed cells; this shows that the hybrid promoter does not affect the growth of its chassis.
</P><P><B>Following the experiments over the 11-week project we have successfully produced two hybrid promoters including bacterial and mammalian elements in two orientations that sense nitrates and nitric oxide (and therefore likely nitrites). We have also successfully ligated the promoters to CFP and RFP as reporters, and have successfully transfected mammalian cells with M-B + CFP, proving the flexibility of the system.</B></P><H2><SPAN class="mw-headline" id="Future_Experiments">Future Experiments</SPAN></H2><P><B>Full quantitative analysis to see where the values lie</B></P><P>In order to gauge how the promoter responds to different levels of substrates experiments must be first designed to test the sensitivity to nitrates, nitrites and nitric oxide. The experiments detailed above have characterised a great deal of the nitrate sensitivity; moving on from this a salt such as sodium nitrite could be used as a donor for nitrites to assess how the promoter responds to nitrites as a substrate. A donor for nitric oxide must be found as well in order to ascertain more easily how the promoter responds to this compound (nitrite reductase has been looked at as a possibilty thus far). Finely-tuned experiments will be carried out to find the area at which fluorescence intensity moves from steadily increasing to dramatically dropping off (e.g. 15 mM - 20 mM for nitrates), and once a range has been found experiments would be carried out with small incriments of change of substrate concentration. Repeats will be made of all experiments in order to more accurately detail the change in intensity compared to substrate concentration. 
</P><P><B>Combining the system with different reporter/effector enzymes</B></P><P>Now we have completed experiments involving fluorescent protein reporters the next logical step for our future applications would be to attach different reporters or effector enzymes. We could attach an enzyme such as nitrite reductase in order to increase expression of nitric oxide as a next step experiment in the future application of cancer therapeutics; we could also investigate adding an enzyme to break down nitric oxide in order to take the next step in the future application of combating nitric oxide pollution. One idea would be to produce a new BioBrick of M-B + Nitric Oxide Synthase (or another nitric oxide donor) + eCFP to test the concept of generating a synthetic gene network which can act as a cancer theraeutic.
</P><P><B>Repeat M-B + eCFP transfection into MCF7 cells. Utilise more experimental controls (e.g. M-B only transfection) and construct an M-B + eCFP plasmid which is optimised for mammalian systems</B></P><P>We are very pleased with the results of our mammalian cell transfection, however it was just step one of what is likely very many more steps to optimise the experiment. In the future we would like to test transfection with M-B and B-M to see if both produce similar results in mammalian cells as well as using them as controls to promoters combined with reporters. We would also like to ligate the promoter into a plasmid which is more optimised for mammalian systems in order to test it fully within MCF7 cells.
</P><P><B>Optimise use of SNAP NO donor and use alternatives to enhance the induction of Nitric Oxide production</B></P><P>In order to fully test nitric oxide sensation with the promoter we will need to test different nitric oxide donors e.g. SNAP, nitrite reductase, nitric oxide synthase. This will help us fully characterise how the promoter reacts the nitric oxide levels as well as help us decide on the best nitric oxide donor moving forward into producing systems for the future applications that will require nitric oxide production (e.g. cancer therapeutics)
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><P>Lin H.Y., Bledsoe P.J., Stewart V., (2007), <I>Activation of yeaR-yoaG Operon Transcription by the Nitrate-Responsive Regulator NarL Is Independent of Oxygen- Responsive Regulator Fnr in Escherichia coli K-12▿</I>, Journal of Bacteriology, <B>189: 7539 - 7548</B></P><P>
Modern Building Services, (2008), <I>Understanding NOx emissions [online]</I>, Available at: <A href="http://www.modbs.co.uk/news/archivestory.php/aid/5380/Understanding_NOx_emissions.html" class="external free" rel="nofollow">http://www.modbs.co.uk/news/archivestory.php/aid/5380/Understanding_NOx_emissions.html</A>, accessed on 26/09/2012
</P><P>
Scott S.D., Joiner M.C., Marples B., (2002), <I>Optimizing radiation-responsive gene promoters for radiogenetic cancer therapy.</I>, Gene Therapy, <B>9: 1396-1402</B></P><P>
Worthington J., Robson T., Scott S., Hirst, D., (2005), <I>Evaluation of a synthetic CArG promoter for nitric oxide synthase gene therapy of cancer</I>, Gene Therapy, <B>12: 1417–1423</B></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:NRP-UEA-Norwich/NOSensing">http://2012.igem.org/Team:NRP-UEA-Norwich/NOSensing</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:NRP-UEA-Norwich/NOSensing" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:NRP-UEA-Norwich/NOSensing" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:NRP-UEA-Norwich/NOSensing&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:NRP-UEA-Norwich/NOSensing&amp;oldid=226942" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>