"
Page
Discussion
View source
History
teams
Log in
 
Team:NRP-UEA-Norwich/Week7
From 2012.igem.org
NRP UEA iGEM 2012
Home
Team
Project
Safety
Lab Book
Parts
Future Applications
Quanticare
Human Practices
Attributions
 
Overview Step1 Step2
Welcome to the NRP UEA iGEM 2012 Wiki Lab Book
Please choose the relevant link to access our diary of that week!
Week Zero | Week One | Week Two | Week Three | Week Four | Week Five | Week Six | Week Seven | Week Eight | Week Nine | Week Ten | Week Eleven | Lab Protocols | Experiments
Contents
1Week 7
1.1Day 1 (20/08/12)
1.1.1Research
1.1.2Labs
1.2Day 2 (21/08/12)
1.2.1Labs
1.3Day 3 (22/08/12)
1.3.1Labs
1.4Day 4 (23/08/12)
1.4.1Labs
1.5Day 5 (24/08/12)
1.5.1Labs
Week 7
With half the team away for the week, this was a though week in terms of lab work. The remaining team members concentrated mainly on the completion of the growth studies which produced good results that we are looking forward to analysing. However, there were also many highlights such as the confirmation of our arduously designed gene constructs being made and the arrival of primers allowing us to send our BioBricks to be sequenced.
Day 1 (20/08/12)
Research
Research into companies and their requirements for sequencing was carried out. This is in preparation for sequencing of our own BioBricks (BM and MB in pSB1C3). Sequencing is required to confirm, that our genes have the correct base sequence before we send them to iGEM. Lukas found that the requirements for Bio LifeSciences was 5-6µL of plasmid DNA at a concentration of 100ng/µL and the primers at a quantity of 10µL and concentration of 3.2pmol/µL. The primers were ordered and the plasmid samples aquilotted in preparation.
The genes of our comparator circuit DNA constructs were finalised within the requirements of GenScript. With three weeks left, we are hoping they will arrive at the end of week 9 or the start of week 10. This will hopefully give us enough time to get into the BioBrick format, send them off to iGEM and characterise and prove their function.
Labs
As the previous transformations of the BioBricks BBa_E0420 (CFP) and BBa_K081014 (RFP) were either unsuccessful or showed contamination so we decided to retry the transformation with a different team member. The procedure used was the one seen on the lab protocol page . In addition, the BBa_K206009 (GFP) BioBrick was transformed. Again, this will also be utilised in characterisation of our BioBricks as these will eventually be ligated to our promoter. The transformants were plated on ampicillin plates of a concentration of 100µg/ml. A negative control of E.coli cells was implemented as well.
Day 2 (21/08/12)
Labs
A message from GenScript confirming the synthesising of the ordered genes!
The transformation was unsuccessful as there was no sign of contamination and no growth showed on any of the plates. A transformation of CFP, RFP and GFP into alpha competent cells was repeated. Some changes were made: positive controls were made (plasmids containing MB within backbones that contain both ampicillin and chloramphenicol resistance). Through some slight confusion, GFP had been plated on ampicillin plates the previous day, as the BioBrick contains only chloramphenicol resistance, it was plated on chloramphenicol plates (25µg/ml) today. The rest of the protocol was unchanged from the previous day. Some of the transformation culture was retained and more LB medium was added and incubated over night.
A growth study involving the PyeaR-GFP (BBa_K381001) BioBrick and alpha cells was planned again but this time involving MB and BM BioBricks as well. We hope to obtain results that will allow us to compare the data sets. The plan is to grow these cells in LB medium without antibiotic resistance in sterile conditions overnight to increase the number of cells. The next day, these will be pelleted down to obtain a larger cell concentration and these samples will be diluted down with LB both till an absorbance of 0.2 is reached. From then on, measurements will be run for 12 hours ( Results ).
Day 3 (22/08/12)
Labs
Figure 1. B-M/M-B in pSB1C3 plasmid preps, double digested using PstI and SpeI to linearize for later ligation with fluorescent reporter
Following the protocol of the growth study set up the previous day, Khadija, Lukas and Rebecca arrived into the labs bright and early. The full experiment details can be found on the Experiments page .
The transformations of the GFP, RFP and CFP were successful. Growth was observed on all plates except the negative control plates. Some showed greater numbers of colonies than others. The positive control showed lots of viable cells. The GFP plates showed lots of growth, too. The RFP and CFP showed less growth. There were less than 10 colonies on these plates. Inoculations with viable cells from all samples were made into LB media and incubated overnight at 37°C.
A restriction digest was carried out by Khadija to both validate and linearise BM and MB within the iGEM pSB1C3 backbone. It was linearised in preparation of ligating a reporter to the insert (B-M and M-B hybrid promoter). The restriction digest was carried out using PstI and SpeI. The samples were left overnight to fully complete the digestion. From previous experiments which ligated MB and BM into pSB1C3, we used 4 BM's and 4 BMs samples labeled BM1-4 and M-B1-4 respectively. These were used in the digestion (Figure 1.).
The running inoculations of MB and BM BioBricks were miniprepped, following the protocols on the lab protocol page . This produced more plasmid DNA which was nano dropped.
Day 4 (23/08/12)
Labs
The overnight restriction digest was run on an agarose gel (1% w/v) for an hour and a half. The image showed that MB1, MB2, MB4 and BM2 were fully cut and presented with only one band. The smaller excised fragment, which would be only a few base pairs long, was not visible as expected. Some of the samples ran further than exspected and we hypothesised that these might be uncut plasmids (Figure 1.).
BioBrick (BBa_K206009 (GFP), BBa_E0420 (CFP) and BBa_K081014 (RFP) transformants were miniprepped by Lukas to attain pure plasmid DNA. These was stored in the freezer.
The growth study carried out the previous day did not include the lag phase as the starting concentrations of cells were too high. A lag phase only study was carried out. This involved starting measurements at a lower cell concentration (relative to absorbance). The absorbances starting point was chosen at around 0.04, however reaching these values precisely was very difficult due to the error margin of the spectrophotometer being great in this range. Therefore 10µl of cultures grown overnight were pipetted into 1ml of LB broth media in a cuvette. The study lasted for 5 hours with readings taken every hour. Results are shown on the experiments page .
The sequencing primers have arrived! iGEM sequencing primer (VF and VR) stock was made up to a concentration of 100µM by resuspension the stock solution in distilled water. These aquilots were diluted to match the concentration requirements of Bioscience at 3.2pml/µL. Following the arrival and preparation of primers, the plasmid DNA of our BioBricks to be sent for sequencing was prepared. The made DNA was packaged into separate eppendorfs to be sent off. Each eppendorf contained 6µL of the DNA (MB1-4 and BM(1-4) and was labelled accordingly. As there was a delay, these were unable to be sent off immediately and were frozen again until postage.
Day 5 (24/08/12)
Labs
Figure 2. Gel electrophoresis of Isolated plasmids of GFP, CFP, RFP BioBricks, single digested with PstI
Rachel carried out a gel purification (on both MB and BM fragments) using a Promega Wizard kit (Ref: A9281) as described in the lab protocol page . The purified samples were stored in the freezer.
Rebecca transformed BioBrick BBa_K561002 (PFDHF +RFP + TETR) into Bioline Alpha Select Gold Standard cells following the transformation protocol as seen on the lab protocol page . The negative control was simple, untransformed alpha cells. The samples were plated onto agar plates containing chloramphenicol at 25µg/ml.
Khadija nanodropped miniprepped plasmid BioBrick samples of BBa_K206009 (GFP), BBa_E0420 (CFP) and BBa_K081014 (RFP). As several morphologically different colonies had grown an several plates, the samples of these cells were stored in labeled epppendorfs. In total there were 4 different plasmid isolations of CFP, RFP and GFP. These were shown to have high concentrations of DNA ranging from 165ng/µL to 332 ng/µL. Following a nanodrop, two restriction digests were carried out to validate the DNA. One cut at the brick insert at the suffix/prefix, the other digested at a specific restriction site within the backbone. As BBa_K206009 is found in pSB1AK3, there is a HinIII site and  BBa_K081014 is in pSB1C3 and contains an EcoRV site. BBa_E0420 contains many restriction sites, but none that can be cut by enzymes that we had in stock. Therefore, a second validation restriction digest was not carried out on this BioBrick. The protocol involved 1µL DNA, 2µL buffer, 0.2µL BSA, 0.5µL of restriction enzyme and 16.3 µL of distilled water. Buffer H was used in the digests with EcoR1, Buffer B for HindIII and Buffer D for EcoRV. These completed digestions were stored in the freezer after 2 and a half hours of digestion ('Figure 2.)
Retrieved from " http://2012.igem.org/Team:NRP-UEA-Norwich/Week7 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
