"
Page
Discussion
View source
History
teams
Log in
 
Team:NRP-UEA-Norwich/Week3
From 2012.igem.org
NRP UEA iGEM 2012
Home
Team
Project
Safety
Lab Book
Parts
Future Applications
Quanticare
Human Practices
Attributions
 
Overview Step1 Step2
Welcome to the NRP UEA iGEM 2012 Wiki Lab Book
Please choose the relevant link to access our diary of that week!
Week Zero | Week One | Week Two | Week Three | Week Four | Week Five | Week Six | Week Seven | Week Eight | Week Nine | Week Ten | Week Eleven | Lab Protocols | Experiments
Contents
1Week 3
1.1Day 1 (23/07/12)
1.1.1Meetings
1.1.2Labs
1.2Day 2 (24/07/12)
1.2.1Labs
1.3Day 3 (25/07/12)
1.3.1Research
1.3.2Labs
1.4Day 4 (26/07/12)
1.4.1Research
1.4.2Labs
1.5Day 5 (27/07/12)
1.5.1Research
1.5.2Labs
Week 3
Over the weekend, the group met up to discuss the two potential projects, we could go forward with, using our synthesised genes and the BioBrick it will ultimately produce. We developed two main themes; one was a multi-sensor system sensing oxygen-containing physiologically-relevant molecules and reporting on the levels of each molecule; the other was a comparator circuit sensing levels of molecular species and using novel gene expression to express reporters for specific molecules within the environment to quantitatively measure them. We decided on the latter as we felt it would have the highest scientific impact, a diagram of the gene system for one of the sensors is given below.
The concept behind the project involves the mRNA control region of one sensor binding to the mRNA control region of a second sensor and thus for one sensor to 'switch off' the other to a certain degree. The overhang is used to measure levels of a substrate molecule. For example in our nitric oxide-sensing project a sensor for nitric oxide and nitrites (sensor 1) may be used in conjunction with a sensor for only nitrites (sensor 2); the control region of mRNA produced from sensor 2 would switch off sensor 1 per the levels of nitrite in the environment, and therefore all expression of sensor 1 would be directly related to nitric oxide levels in the atmosphere.
Day 1 (23/07/12)
Meetings
Amy (our team artist) came to visit us today in order to discuss our video and brainstorm ideas to best present the future applications of your project to the public. Her, Joy and Rachel began to develop the storyboard and work out how to incorporate the comparative circuit and its potential into the video. They came up with some fantastic ideas and the whole team sat down to discuss and review them together with Amy before she left again at the end of the day. Subsequently Joy contacted the UEA Drama School in order to book one of their studios so that we could have a professional working environment to film our video.
Rachel at one of our various planning and brainstorm sessions
Labs
From the cells transformed with the two synthetic genes BM and MB, samples were used to inoculate LB broth for plasmid DNA isolation and later transfer from their vector pUC57 to the standart iGEM Backbone pSB1C3 via restriction digest and ligation. We also hoped to use these cells to further characterise our gene promoters. When visually inspecting the agar plates on which the transformants of the two genes were growing, it was detectable that the colonies of cell transformed with M-B were larger then the cells transformed with BM after the same time frame of incubation. We formed that hypothesis that BM might negatively influence the cell growth of "E.coli" and we were planning to further investigate this after we have inserted the genes into the iGEM backbone and worked with them in the standard BioBrick format.
To characterise the synthetic promoters and other Bio-Bricks, that are involved with NO, an input of NO will be needed. Our solution to this was to use a nitrate salt like potassium nitrate in the growth media. A stock solution with 1M of KNO3 was made up today by Lukas. This was taken to be autoclaved. The protocol for this can be found here.
Day 2 (24/07/12)
Labs
The LB cultures inoculated with cells, containing the M-B plasmid, were mini prepped to isolate the synthetic gene in its backbone pUC57.  As in preparation for the ligation of the BM/MB BioBrick into the pSB1C3 backbone, the ladder was isolated in its plasmid form from cell cultures and restriction digested with EcoRI and PstI to form sticky ends at the end of the IGEM prefix and the beginning of the suffix. In this slot, we hoped to ligate our genes, that were being cut out of the transportation backbone with the analog enzymes to give them complementary sticky ends to pSB1C3, which will hopefully allow us to to create two new BioBricks. For the preparation of pSB1C3, plasmid isolation samples of the PyeaR BioBrick in the mentioned backbone were digested as described. Protocol of double digest
Day 3 (25/07/12)
Research
After thorough research and consulting with Dr. K. Yeoman the previous day, we came to the conclusion that, to further design the DNA sequence for the complementary control region of the comparative circuit and to choose reporter/enzymes, we needed to understand if and which fluorescent proteins we could use in our constructs. This was strongly influenced by our access to fluorometer and after a social media call-out to our faculty, we were pointed in the direction of Dr. Tom Clarke who very kindly offered us the use of his fluorometer, as well as training on it, and potentially even the use of a fluorescence microscope. In addition he gave advice on the project and suggested that pigments and the use of a spectrophotometer would perhaps give better quantitative results for characterisation and informed us that fluorescence may be particularly labour-intensive and time consuming. We concluded that more research on other ways of reporting substrate levels was necessary.
Various other pigments were investigated and projects such as E. Chromi were looked at to find BioBricks that would allows to use them in the comparative circuit. One problem we came across was the overlapping absorbance of various pigments, which would ultimately cause our results to be inaccurate if they were produced in an environment. We then decided to look at proteins with unique absorbencies to utilise as our reporters, as well as enzymes that would produce colourful reporter products. Our main problems involving proteins were that they might not already be incorporated into BioBricks and that they might interfere with the ribosome binging site.
Labs
To prepare the M-B promoter to be ligated into the pSB1C3 iGEM backbone, the plasmid containing the gene and pUC57 was double digested with EcoR1 and PstI to cut out the desired synthetic DNA sequence at the iGEM prefix and suffix (Figure 1.).
Figure 1. Restriction map of B-M in pUC57 double digested with EcoRI and PstI
To further characterise the PyeaR + GFP BioBrick and to give us important insight into measuring fluorescence, that we will utilise in our gene construct that measures No concentrations as well as in the comparative circuit, cell cultures were grown up in media containing different concentrations of potassium nitrate. After overnight incubation we hope to measure a correlation between the potassium nitrate concentration and the intensity of GFP expression  The isolated hybrid promoter MB was digested and a 1.4% agarose gel was prepared to separate the fragments the following morning.
Day 4 (26/07/12)
Research
Extensive research went into which reporter gene would be the best for our project including fluorescence, absorbing proteins, pigments and enzymes which produce coloured products. A decision was made to test fluorescent BioBricks as we had already determined how easy they would be to use. Using IDTs OligoAnalyser, we also began constructing the synthetic gene needed for our comparator circuit idea. It soon became clear that the previous designs for the gene would be too complicated for the timescale of the project, so we decided that our efforts should be focused on designing 'zips' that engulf the ribosome binding site of the reporter gene of interest. These 'zips' on the two constructs will complimentary bind together when both are present in the cell and form a duplex that will sequester the ribosome binding site and inhibit translation.
However, these 'zips' needed to consist of non-rare codons on both constructs and still display complimentary base pairing to one another. This is a challenge we hope to solve over the coming weeks and have the synthetic gene sent of in good time so that after it has been delivered we have enough time to bring it into the BioBrick form and fully characterise it to prove its function it will have in the comparative circuit.
Labs
The GFP Gradient
Using the cell cultures grown in potassium nitrate media, Russell spun down a proportion of the culture and viewed them under UV light. He found a graduated intensity of green fluorescence increasing with increasing potassium nitrate concentration. This proven correlation is important information for our comparative circuit as well as the NO sensing compound. Results can be viewed here.
To keep constant stock of media and other supplies, Rebecca prepared chloramphenicol plates at a concentration of 25µg/ml and made up a stock solution of ampicillin.
The restriction digest of the hybrid promoter MB in pUC57 with EcoRI and PstI from yesterday was run on a 1.4% agarose gel in order to separate the desired insert that is the promoter, from the larger fragment that is the backbone. The band that represents the MB DNA sequence was excised from the gel and the DNA contained in it was purified using a BioLine gel-purification kit. Unfortunately another gel electrophoresis of the obtained sample revealed that no DNA was present. We suspected that the failed extraction might have been caused by a faulty kit or misread instruction on our part. Another explanation might also have been the very low concentration of DNA present in the sample and the subsequent loss during the purification.  In the afternoon, Lukas purified the hybrid promoter BM from cell cultures via mini prep and the presence of DNA was proven via gel electrophoresis. The samples were double digested with EcoRI and PstI to excise the promoter from the pUC57 backbone at the iGEM suffix and suffix to insert it later in the pSB1C3 backbone.
Day 5 (27/07/12)
Research
Pascoe and Khadija continued their work on planning the comparator circuit construct. They had a limited number of codons to work with, as they needed to be both common codons in E. coli and have complimentary codons which are also common. In addition both codons needed to code for small, non polar amino acids that would not affect the reporter proteins tertiary structure enough to affect its reporter properties. The list of codons were collated into a list affectionately called 'The Good List' and from this Pascoe and Khadija began piecing them together. Utilising OligoAnalyser the folding of the constructs were visualised and evalulated. If the ribosome binding site or start codon were obstructed in the single construct strands, translation could be inhibit, which would hinder the comparator circuits functionality.
Labs
Lukas and Joy ran a gel electrophoresis of the double digest of the B-M hybrid promoter to separate the the pUC57 backbone from the inserted promoter. DNA was found and hence the 250bp fragment, representing the B-M sequence, was cut from the gel and purified. In preparation of the ligation of the BM fragment with the linearised pSB1C3 backbone, two methods were planned out. The first one involves the unpurified double digest sample to be mixed with the linearised pSB1C3 backbone. Three expected plasmid constructs might be observed and the desired one selected with staggered growth on two media with different antibiotics present. The other protocol that would in the end be used can be seen here.
Rebecca made up LB media in 5ml sample bottles and LB plates containing ampicillin at a concentration of 100 µg/ml. Lukas and Rebecca also isolated plasmids containing the  hybrid promoters in both orientations to ensure enough purified DNA would be available for subsequent experiments.
After the previous days' investigation into various reporter proteins that are suitable for our projects, we decided to look at fluorescent proteins with vastly differing peaks. Ultimately we decided to investigate Red Fluorescence Protein (RFP) and Blue Fluorescence Protein (BFP) as there was very little overlap in their peaks, and it was concluded that we should characterise some BioBricks that we already had access to, to assess the fluorescent protein levels and their usefulness in our project. After checking their availability in our kit plates we decided on using:
- BBa_R0080 - A promoter that should activate transcription in the presence of arabinose, in order to allow simple and easy activation of the fluorescence in transformed cells.
- BBa_E0420 - A reporter expressing enhanced Cyan Fluorescence Protein (eCFP), a fluorescent protein with a similar peak to BFP that shouldn't overlap with RFP too much. This BioBrick also contains the RBS and terminators meaning we can simply add the promoter of choice to complete the system.
- BBa_K081014 - A reporter expression RFP. This BioBrick also contains the RBS and terminators meaning we can simply add the promoter of choice to complete the system.
Russell and Rachel transformed E. coli with the three BioBricks of choice using the standard protocol described in previous experiments and plated onto agar plates containing 100ug/µl ampicillin due to the plasmids for each BioBrick including ampicillin resistance. The cultures were incubated overnight at 37 °C and stored in the fridge the next day over the weekend.
The transformation protocol can be found here
Retrieved from " http://2012.igem.org/Team:NRP-UEA-Norwich/Week3 "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
