<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Stuttgart"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Sponsoring"><DIV class="submenu_item">
							Sponsoring
						</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Description"><DIV class="submenu_item">
					Description
			</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Part_Collection"><DIV class="submenu_item">
					Part Collection
				</DIV></A></DIV><A href="https://2018.igem.org/Team:Stuttgart/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Applied_Design"><DIV class="submenu_item">
					Applied Design
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Entrepreneurship"><DIV class="submenu_item">
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Hardware"><DIV class="submenu_item">
					Hardware
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Measurement"><DIV class="submenu_item">
					Measurement
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Plant"><DIV class="submenu_item">
					Plant
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Software"><DIV class="submenu_item">
					Software
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Stuttgart"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1 align="center"> Notebook </H1><DIV class="text-light"><H2>Abbreviations</H2><P>Team Rhamnolipid: Rhl <B>-</B> Team Nisin: Nis <B>-</B> Team Chitosan: Chi <B>-</B> Team Linkage: Link <B>-</B> Team Modeling: Mod</P><H2>Timeline</H2></DIV><DIV class="container"><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>15.10. - 19.10.2018</H3><SPAN>Preparation for Giant Jamboree</SPAN><P><B>Nis:</B> Repeated SDS-PAGE of nisin and staining with silver nitrate, to obtain a more clearly visible result.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>08.10. - 12.10.2018</H3><SPAN>Last week in the lab / Wiki time</SPAN><P><B>Rhl:</B> Wiki was completed.</P><P><B>Nis:</B> Additionally, a SDS-PAGE of the purified nisin was performed and the gel was stained with silver nitrate.</P><P><B>Chi/Nis:</B> Disruption of cells from last weeks expression was performed by high pressure homogenization (EmulsiFlex C5). Protein purification was done by HPLC using a His-Trap column (GE-Healthcare). Proteins (COD, NotC, NisA) were analyzed by SDS-PAGE and Coomassie staining.</P><P><B>Mod:</B>The text for the wiki was written and last analyzes were performed.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>01.10. - 05.10.2018</H3><SPAN>Last few days of our project </SPAN><P><B>Rhl:</B> Temperature gradient PCR of <I>rhlC</I> with GC-Enhancer due to the fact that the gene contains high GC content. After this, the project was dropped because time was running out.</P><P><B>Nis: </B>Restriction digestion of the Nisin-Composite part with XbaI and BbsI. Subsequent gel-extraction, religation and transformation into DH5 alpha <I>E. coli</I>. was performed to obtain a proper Basic Biobrick. DNA samples were isolated via Mini-Prep-Kit and were sent to sequencing. </P><P><B>Chi: </B>Alternative strategy for construction of BBa_K2380002 was tested by digesting BBa_K28480001 with BbsI and XbaI, gelextraction of the 3.4 kb fragment and religation.</P><P>
Expression of BBa_K2380043 and BBa_K28480001 at 30 °C in 250 mL. Cells were harvested by centrifugation, resuspended in phosphate buffer and stored at -20 °C.</P><P><B>Link:</B> In this week we coated a door handle with our polymer and started writing the wiki.</P><P><B>Mod:</B>All MD simulation was started again under npt condition to achieve a simulation time of 200 ns.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>24.09. - 28.09.2018</H3><SPAN>Last Steps </SPAN><P><B>Rhl:</B> Transformation of <I>E. coli</I> DH5α with BB_B0034 (ribosome binding site), BBa_K1331006 (<I>rhlAB</I>) and a vector carrying a kanamycin resistance. Overnight cultivation thereof, Miniprep and digestion with EcoRI, SpeI or XbaI, respectively. This plan was dropped since the yield was not satisfying.</P><P><B>Nis: </B>Sequencing confirmed the Composite Biobrick for Nisin, containing our modified Nisin with different tags, a RBS and an arabinose promotor. In order to include the whole processing cluster of Nisin Gibson Assembly was performed with NisA_LY, NisBTCIP and pSB1C3.</P><P><B>Chi: </B>Construction of BBa_K28480001 (NodC+COD-HisTag) from BBa_K2380002 (NodC, SpeI) and the PCR product from BBa_K2380043 (XbaI/SpeI). Multiple colonies were picked and the orientation of the COD fragment in the construct was tested by colonyRCA and restriction digestion using MscI.</P><P>
Activity assay was performed for the chitin synthase NodC using the UDP Glo Kit (Promega) and UDP-GlcNAC as substrate. Activity was compared for NodC from BBa_K2380002 and BBa_K28480000.
</P><P><B>Link:</B>The company dispendix provided us their iDot to print bacteria suspension on agar plate. With this we conducted experiments for antimicrobial.</P><P><B>Mod:</B>The first equilibration run successfully but for the production run with npt condition the equilibration time was to short, so the step size for equilibration was extended activity.</P><P><B>German Red Cross: </B>Two team members interviewed Michael Teutsch about the requirements of an antimicrobial surface in practice.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>17.09. - 21.09.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Order of new optimized primers. Temperature gradient PCR with different template concentrations.</P><P><B>Nis:</B>The vector pSB1C3 with the araBAD promotor was amplified using new PCR components. Finally it worked. Gibson Assembly was performed for both our NisA inserts and the products were transformed into <I> E. coli </I>.</P><P><B>Chi:</B>Adding a His<SUB>6</SUB>-Tag to COD by PCR using BBa_K2380043, and -044 respectively, as templates. PCR was checked by agarose gel electrophoresis.</P><P>
Testing of a chitosan assay in the presence of proteins. Standard calibration curves for chitosan and BSA in the range of 20-100 µg mL<SUP>-1</SUP> were recorded.
</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>10.09. - 14.09.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Temperature gradient PCR of all templates with different concentrations.</P><P><B>Nis:</B> We decided for a new approach: cloning our NisA-inserts into pSB1C3 that already contains an arabinose promoter and a RBS (araBAD promotor). Design of new primers for our construct. PCRs were performed for our inserts and the vector + araBAD promotor and checked on an agarose gel. Again only the amplification of the NisA inserts worked.</P><P><B>Chi:</B>Expression of BBa_K2380002/-042 and the combined construct BBa_K2848000 was repeated using a higher culture volume (100 mL). Cells were disrupted in phosphate buffer by high pressure homogenization using an EmulsiFlex C5 (Avestin, ATA Scientific). Proteins were purified by their His-Tag using HPLC (HisTrap column, ÄKTA, GE Healthcare).</P><P><B>Link:</B>IR analysis of samples from chemical linkage.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>03.09. - 07.09.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Overnight cultivation of transformed <I>E. coli</I> DH5α and Miniprep for higher DNA concentrations. Control of templates by restriction digest of biobricks with EcoRI and SpeI. <I>RhlC</I> was loaded without digestion  since it does not have the same restriction sites. Sizes of fragments accorded with theoretical values leading to the conclusion of having the correct templates. To verify these results, all biobricks were sequenced with positive results. Sequencing of <I>rhlC</I>.</P><P><B>Nis:</B> We changed every parameter possible in our PCR to amplify pSB1C3 for Gibson cloning but nothing worked.</P><P><B>Chi:</B>Expression of BBa_K2380002/-042 and the combined construct BBa_K2848000. Wt DH5α expression was performed in parallel. Cells were disrupted by ultrasound and purified using a Ni-NTA agarose column. Purified fractions were analyzed by SDS-PAGE and Coomassie staining.</P><P>
All contructs from Darmstadt 2017 were tested by XbaI+PstI double digestion and agarose gel electrophoresis.
</P><P><B>Link:</B>Analyisis of enzymatic linkage and chemical approaches for coupling.</P><P><B>Mod:</B>A lot of problem solving was performed in the simulation team to get a running md simulation.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>27.08. - 31.08.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Temperature gradient PCR with <I>araC</I>, <I>rhlA</I>, <I>rhlB</I> and <I>rhlC</I>.</P><P><B>Nis:</B> Since assembly by restriction cloning didn’t work, we decided for a new cloning strategy and switched to GibsonAssembly. Design of new primers. PCRs were performed on our NisA inserts and pSB1C3 and checked on an agarose gel.</P><P><B>Chi:</B>Contruction of BBa_K2848000 from BBa_K2380002 (NodC, SpeI + PstI) and BBa_K2380042 (NodB, NheI + PstI). BBa_K2848000 contains NodC and NodB under the control of a pBAD arabinose promotor (BBa_K808000). Each ORF is flanked by its own RBS (BBa_K2380024) and terminator (BBa_B0015).
An alternative clone was also constructed using BBa_K2380041, resulting in a plasmid without a second RBS for the NodB ORF.</P><P>
Expression of BBa_K2380002/-042 and -043 was performed again and the proteins analyzed by SDS-PAGE and Coomassie staining.
</P><P><B>Link:</B>In this week we performed further different chemical and enzymatic approaches for coupling chitosan.</P><P><B>Mod:</B>A code was adapted from the group of Pleiss to perfume the equilibration and production simulation on multi cluster.   
</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>20.08. - 24.08.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> PCR with plasmids containing <I>araC</I>, <I>rhlA</I>, <I>rhlB</I> and <I>rhlC</I>.</P><P><B>Nis:</B> To obtain pSB1C3 for cloning we transformed two biobricks (<A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K398326">BBa_K398326</A> and <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K398331">BBa_K398331</A>) from the distribution Kit 2018 into <I>E. coli</I>. We isolated both plasmids and digested them with SpeI and XbaI before ligating them to both our NisA gBlocks. Screening was done with colony PCR but no positive clones were detected.</P><P><B>Chi:</B>Expression of NodC (BBa_K2380001) and COD (BBa_K2380043) at 30 °C and room temperature in triplicates each. Proteins were analyzed by SDS-PAGE and Coomassie staining.</P><P><B>Link:</B>Further animicrobial assays to check for the antimicrobial potential of TAGC. We also tested different ratios of reactants for chemical linkage of rhamnolipids with chitosan. Differnt methods of precipitation of the product were tested. The different batches of enzymatic coupling were analyzed.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>13.08. - 17.08.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Overnight cultivation of transformed <I>E. coli</I> DH5α and Miniprep to amplify the plasmids. Control of concentration with Nanodrop.</P><P><B>Nis:</B>We finally started working in the lab. To clone our ordered NisA (NisA_LY and NisA_YL) into the iGEM backbone vector pSB1C3, we performed several PCRs on the inserts and pSB1C3 and checked them on agarose gels. We managed to amplify both our inserts. However, though we changed various parameters of our PCR protocol like annealing temperature, number of cycles or elongation time we didn’t manage to amplify pSB1C3. Digest of both NisA inserts with SpeI and XbaI.</P><P><B>Link:</B>First antimicrobial assays to prove the efficacy of our chemical produced surface. We tried different approaches of enzymatic coupling of GFP with chitosan by tyrosinase. Those approaches were analysed by PAGE.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>06.08. - 10.08.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Transformation of <I>E. coli</I> DH5α with existing biobricks BBa_K1331001 (<I>rhlA)</I>, BBa_K1331002 (<I>rhlB</I>), BBa_K808000 (araC) and BBa_J04450 (mRFP) as well as pVLT33 containing <I>rhlC</I>.</P><P><B>Link:</B>Further tyrosinase based experiments for linking GFP and chitosan. We met with experts of the institute of plastics engineering.</P><P><B>Mod:</B>A few tutorials were read and the setup for the simulation of md model were performed .</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>30.07. - 03.08.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Transformation of <I>E. coli</I> DH5α with existing biobricks BBa_K1331001 (<I>rhlA</I>), BBa_K1331002 (<I>rhlB</I>) and BBa_K808000 (<I>araC</I>).</P><P><B>Link:</B>Chemical reactions to link rhamnolipids to chitosan. First enzymatic approach to couple GFP with chitosan, by tyrosinase reaction.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>23.07. - 27.07.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B>Overnight cultivation of transformed <I>E. coli</I> DH5α and Miniprep to amplify the plasmids. A stock in 30 % glycerol was generated, too.</P><P><B>Chi:</B>Expression of NodC (BBa_K2380001), NodB (BBa_K2380042) and Wild Type <I>E. coli</I> DH5α at 30 °C and room temperature in triplicates each. Proteins were analyzed by SDS-PAGE and Western Blot using an anti-His antibody coupled to horseradish peroxidase. Detection was performed with luminol.</P><P><B>Link:</B>Chemical reactions to couple rhamnolipids with chitosan.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>16.07 - 20.07.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Transformation of <I>E. coli</I> DH5α with pVLT33 containing <I>rhlC</I>.</P><P><B>Link:</B>Production of GFP: We purified GFP from a GFP expressing <EM>E. coli</EM>culture by high-pressure homogenisation and affinity chromatography. We planned to couple GFP to chitosan by enzymatic (tyrosinase) reaction. This sould work as a proof of concept for coupling nisin to chitosan. Improvement of the chemical reaction to link rhamnolipids to chitosan.</P><P><B>European iGEM Meetup</B> Four team members attended the European iGEM Meetup in Munich, Germany. There, we presented our poster and visited some interesting presentations and workshops.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>09.07. - 13.07.2018</H3><SPAN>Work in the lab</SPAN><P><B>Link:</B>Further experiments to couple rhamnolipids with chitosan.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>02.07. - 06.07.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Contact to Prof. Dr. Blank and Dr. Germer from RWTH Aachen for scientific support and request for a plasmid containing rhamnosyltransferase II.</P><P><B>Link:</B>First chemical reaction for linking rhamnolipid to chitosan was conducted.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>25.06. - 29.06.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Pre-culture of <I>Pseudomonas putida</I> KT2440 and storage in 50 % glycerol stocks.</P><P><B>Chi:</B>Expression of NodC (BBa_K2380001) and NodB (BBa_K2380042) at 30 °C and room temperature in triplicates each. Proteins were analyzed by SDS-PAGE and Coomassie staining.</P><P><B>InterLab:</B> We started the iGEM InterLab experiments. </P><P><B>Science day:</B></P><P> A few team members presented our project to the people.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>18.06. - 22.06.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Request of composite part BBa_K1331006 from the regsitry and we contacted Prof. Dr. Syldatk from KIT for scientific support.</P><P><B>Link:</B>The ordered tyrosinase was tested for activity by enzyme activity assay.</P><P><B>Mod:</B>Two members of the team attended a three week practical course provided by Prof. Dr. Pleiss about MD simulations. </P><P><B>German iGEM Meetup:</B> Our Team visited the German iGEM Meetup at Marburg, Germany. Here we presented our poster and got an impress of the work of the other German iGEM Teams.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>11.06 - 15.06.2018</H3><SPAN>Work in the lab</SPAN><P><B>Chi:</B>Transformation of Darmstadt BioBricks in <I>E. coli</I> DH5α. Test transformation of BBa_J04450 in pSB1C3 for the calculation of transformation efficiency. Transformation efficiency was calculated to about 10<SUP>7</SUP> cfu µg<SUP>-1</SUP>.</P><P><B>Mod:</B>Two members of the team attended a three week practical course provided by Prof. Dr. Pleiss about MD simulations.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>04.06. - 08.06.2018</H3><SPAN>Work in the lab</SPAN><P><B>Rhl:</B> Order of missing chemicals and TLC-layers.</P><P><B>Chi:</B>Preparation of chemical ultra-competent <I>E. coli</I> DH5α. Protocol is based on Inoue et al (1990), Gene, 96:23-28, with modifications (<A href="http://staceylab.missouri.edu/preparation-of-ultra-competent-e-coli-cells-for-transformation/" title="protocol ultra-competent &lt;i&gt;E. coli&lt;/i&gt;">check here</A>).</P><P><B>Link:</B>We ordered chemicals and talked with Dr. Claasen about IR analysis of our chemical reaction products. The amount of chemicals and substrate for the reactions was calculated.</P><P><B>Mod:</B>Two members of the team attended a three week practical course provided by Prof. Dr. Pleiss about MD simulations.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>28.05. - 01.06.2018</H3><SPAN>First steps in the lab</SPAN><P><B>Rhl:</B> Preparation of media and agar plates.</P></DIV></DIV><DIV class="timeline-block timeline-block-left"><DIV class="timeline-content"><H3>21.05. - 25.05.2018</H3><SPAN>Some lab experience</SPAN><P><B>Rhl:</B> Primer Design and order, planning of cloning and gene assembly.</P><P><B>Chi:</B>BioBricks from Darmstadt 2017 were requested: BBa_K238000/-01/-02/-41/-42/-43/-44.</P><P><B>Link:</B> Preparation of buffers, media and enzyme stock solutions.</P><P><B>Mod:</B>organising the task and to performe a literature search.</P></DIV></DIV><DIV class="timeline-block timeline-block-right"><DIV class="timeline-content"><H3>January - May 2018</H3><SPAN>Beginning of the iGEM Stuttgart 2018 team</SPAN><P>The old team of 2017 recruited students for the 2018 iGEM Stuttgart team. We started brain storming for new possible projects. Different subteams were created for the different task (e.g. Financing/Sponsoring, Lab-management, Homepage). In parallel we did research for our project (literature and experts), developed experiments and planned our lab work. We met weekly to report our progress and discuss with our Advisors and PIs.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>