<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Stuttgart"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Sponsoring"><DIV class="submenu_item">
					Sponsoring
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A></DIV><A href="https://2018.igem.org/Team:Stuttgart/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Entrepreneurship"><DIV class="submenu_item">
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Model"><DIV class="submenu_item">
					Model
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Stuttgart"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1 align="center">Results</H1><DETAILS><SUMMARY><H2 align="center" span="" title="click to enlarge">Chitosan</H2></SUMMARY><P>Aim of the chitosan subproject was the production of chitosan in an expression system. To achieve this, the chitin synthase NodC and one of the chitin deacetylases NodB or COD had to be encoded on a plasmid.</P><P>Already existing BioBricks from the iGEM Team Darmstadt 2017 (<A href="https://2017.igem.org/Team:TU_Darmstadt">ChiTUcare</A>) were used for the cloning of the enzymes in a plasmid. BioBrick BBa_K2380002 encoding NodC combined with an arabinose inducible promoter was used as vector.</P><P>After having recieved plasmid DNA from iGEM Team Darmstadt transformation was successfully conducted and plasmid DNA isolated using Monarch Plasmid Miniprep Kit (NEB). In parallel, 10 ng of pUC19 plasmid DNA were transformed to evaluate the transformation efficiency of the prepared competent <I>E. coli</I> DH5α cells. Transformation efficiency proofed to be about 10<SUP>7</SUP> cfu/µg.</P><P>Plasmids were checked by restriction digestion using XbaI and PstI to excise the inserts from the pSB1C3 backbone. The respective agarose gel is shown in figure C.1.</P><FIGCAPTION style="width: 60%" class="figure-caption"><I>Figure C.1: </I>1 % agarose gel of plasmid DNA after restriction digestion with XbaI and PstI.</FIGCAPTION><P>Except for BBa_K2380044 (COD basic part) all fragment sizes matched the expections.</P><P><U>Construction of combined BioBrick containing <I>nodC</I> and <I>nodB</I></U></P><P>BBa_K2380002 was opened at the BioBrick suffix using SpeI and PstI. The COD ORF including the preceeding RBS was excised from BBa_K2380042 by double digestion using NheI and PstI. After agarose gel electrophoresis of the COD sample the respective band at about 1400 bp was cut out and eluted in 10 µL ddH<SUB>2</SUB>O using Zymoclean<SUP>TM</SUP> Gel DNA Recovery Kit (Zymo Research). The NodC restriction reaction was purified using Monarch PCR &amp; DNA Cleanup Kit (NEB). After ligation and transformation in <I>E. coli</I> DH5α colonies were picked for overnight culture and plasmid DNA checked by double digestion with XbaI and PstI and agarose gel electrophoresis (Figure C.1, rightmost lane). DNA sequence and plasmid map can be found in the registry (<A href="http://parts.igem.org/Part:BBa_K2848000">BBa_K2848000</A>).</P><P>Expression of BBa_K2380002/-042 and our combined construct BBa_K2848000 was performed using a culture volume of 100 mL. Cells were disrupted in phosphate buffer by high pressure homogenization using an EmulsiFlex C5 (Avestin, ATA Scientific). Proteins were purified using immobilized metal affinity chromatography (HisTrap column, ÄKTA, GE Healthcare) and analyzed by SDS-PAGE and Coomassie staining (Figure C.2).</P><FIGCAPTION class="figure-caption"><I>Figure C.2: </I>SDS-PAGE of purified protein fractions after expression using BBa_K2380002 (NodC), BBa_K2380042 (NodB) and BBa_K2848000 (NodC+NodB). Red arrow: NodC.</FIGCAPTION><P>As successful expression of NodC was noticed, enzyme activity was tested by UDP Glo assay. Results of the corresponding luminescence measurements are displayed in figure C.3.</P><FIGCAPTION><I>Figure C.3: </I>Activity assay of purified NodC after expression using BBa_K2380002 (NodC) and BBa_K2848000 (NodC + NodB). Assay was performed using the UDP-Glo Kit (Promega) and UDP-GlcNAC as substrate. Activity was measured by detecting luminescence with a TECAN microplate reader, integration time = 1 s.</FIGCAPTION><P>In figure C.3 the activity of NodC expressed from our combined BioBrick has a much higher activity than NodC expressed from BBa_K2380002. As there was no change in the sequence of the NodC expression cassette the reason for this higher activity is not clear. Further experiments have to be conducted to validate the activity of NodC.</P><P>With respect to the complicated purification of NodB from inclusion bodies built during expression which team Darmstadt 2017 has already described. We exchanged the chitin deacetylase from NodB to COD. Prior to cloning by restriction and ligation, we added a His-tag to COD (<A href="http://parts.igem.org/Part:BBa_K2380043">BBa_K2380043</A>) by PCR. In addition to the C-terminal His-tag a BioBrick suffix compatible SpeI restriction site was added.</P><P>After successful PCR (Figure C.4), product was digested using XbaI and SpeI. BBa_K2380002 was again used as vector. After opening the vector by SpeI restriction digestion and dephosphorylation by using alkaline phosphatase, ligation of COD + His-tag into the vector was performed overnight. The COD fragment could be inserted in either orientation. Therefore, 16 colonies were picked and colonyRCA was performed for amplifying circular (plasmid) DNA. The correct orientation was approved by restriction digestion using MscI and agarose gel electrophoresis (Figure C.5)</P><FIGCAPTION style="width: 100%" class="figure-caption"><I>Figure C.4: </I>1% agarose gel of His-tag PCR using BBa_K2380043 as template. Expected fragment size is 1522 bp as indicated by red arrow.</FIGCAPTION><FIGCAPTION style="width: 100%" class="figure-caption"><I>Figure C.5: </I>1% agarose gel of RCA reaction after restriction digestion with MscI to check correct orientation of the COD ORF. Expected fragments are 2590 bp, 2355 bp and 318 bp.</FIGCAPTION><P>Expression of BBa_K2380043 and BBa_K2848001 was performed using a culture volume of 250 mL. Cells were disrupted in phosphate buffer by high pressure homogenization using an EmulsiFlex C5 (Avestin, ATA Scientific). Proteins were purified using immobilized metal affinity chromatography (HisTrap column, ÄKTA, GE Healthcare) and analyzed by SDS-PAGE and Coomassie staining (Figure C.6).
</P><FIGCAPTION style="width: 100%" class="figure-caption"><I>Figure C.6:</I> 12.5 % SDS-PAGE after His-tag purification of cell lysates and elutions from BBa_K2848001 and BBa_K2380043. For both constructs two approaches were analyzed.</FIGCAPTION><P>NodC (approximately 21 kDa) and COD (approximately 45.5 kDa) could not be detected by SDS-PAGE and Coomassie staining. Further experiments are necessary to evaluate the expression of NodC and COD in the plasmid BBa_K2848001. Culturing and purification of BBa_K2380043 acted as a negative control because the COD on this plasmid has no His-tag for purification.</P><H3>Outlook</H3><P>Further experiments for the subproject chitosan are the validation of the new generated BioBricks BBa_K2848000 and BBa_K2848001. By varying the amount of IPTG and arabinose induction of expression can be optimized. Purification of NodC and COD from cell culture with a His-Trap column should be analyzable by SDS-PAGE. Eventually the staining method has to be more sensitive (e.g. silver staining, western blot). After proof of function chitosan could be produced by <I>E. coli</I> in one culturing step by adding UDP-N-acetylglucosamine (UDP-GlcNAc) to the medium.</P><P>The final purification of chitosan from the cell culture has to be investigated. The fact that chitosan is possibly toxic to <I>E. coli</I> must be considered. Eventually, induction of chitin synthase and chitin deacetylase has to be separated to protect <I>E. coli</I> from early cell death <A href="https://2018.igem.org/Team:Stuttgart/Design">(see Design)</A>. Otherwise, intracellular chitosan has perhaps no toxic effect, because it interacts with peptidoglycan <A name="refC.1" href="#reflC.1">[C.1]</A>.</P></DETAILS><DETAILS><SUMMARY><H2 align="center" span="" title="click to enlarge">Nisin</H2></SUMMARY><P>Team Nisin generated BioBricks that enable the production of a modified nisin in <I>E. coli</I>. The template for the construct of nisin (containing two tags: one for purification: His<SUB>6</SUB>-tag, one for covalent linkage using tyrosinase: YAAY-tag) was synthesized by IDT. This template then was used for successful amplifications of the NisA inserts (Figure N.1).
</P><P><CENTER><B><I>Figure N.1</I>: Agarose gel of the PCR products of the araBAD_NisA  insert.</B> As marker we used the 1 kb Plus DNA ladder from NEB. 
YL: NisA with YAAY and His<SUB>6</SUB>-tag at the N-Terminus. 
LY: NisA with YAAY and His<SUB>6</SUB>-tag after the leader peptide.
OH: amplicons with overhangs for Gibson with vector pSB1C3 and the Genecluster NisBTCIP</CENTER></P><P>The inserts were cloned into the BioBrick <A href="http://parts.igem.org/Part:BBa_K2380002">BBa_K2380002</A> from Darmstadt using Gibson assembly to replace the chitin synthase <I>nodC</I> and preserve the araBAD promoter. This resulted in the composite BioBrick <A href="http://parts.igem.org/Part:BBa_K2848004">BBa_K2848004</A> encoding the modified nisin with the inducible araBAD promoter and a strong RBS confirmed by sanger sequencing (Figure N.2).</P><P><CENTER><B><I>Figure N.2</I>:</B> BioBrick <A href="http://parts.igem.org/Part:BBa_K2848004">BBa_K2848004</A> confirmed by sanger sequencing.</CENTER></P><P>To obtain the basic BioBrick encoding only <I>nisA</I>, <A href="http://parts.igem.org/Part:BBa_K2848004">BBa_K2848004</A> was digested with BbsI and XbaI and ligated after removal of the promoter and RBS fragment (Figure N.3). This resulted in the basic BioBrick <A href="http://parts.igem.org/Part:BBa_K2848003">BBa_K2848003</A> confirmed by sanger sequencing (Figure N.4).</P><P><CENTER><B><I>Figure N.3</I>: Agarose gel of the pSB1C3_araBAD_NisA_LY construct digested
with XbaI and BbsI.</B> As marker we used the 1 kb Plus DNA ladder from NEB.
The product containing the araBAD promoter is 1242 bp long. The product
holding pSB1C3 and the protein coding region for NisA is 2275 bp long.
</CENTER></P><P><CENTER><B><I>Figure N.4</I>:</B> BioBrick <A href="http://parts.igem.org/Part:BBa_K2848003">BBa_K2848003</A> confirmed by sanger sequencing</CENTER></P><P>To confirm the production of pre-nisin in <I>E. coli</I> we used western blot (Figure N.5) for visualization (pre-nisin having higher molecular weight then natural nisin).</P><P><CENTER><B><I>Figure N.5</I>: Western Blot of pre-nisin.</B> As marker we used Roti-Mark Standard Protein Marker. Pre-nisin has a molecular weight of 7.37 kDa and was detected using an alphaHis antibody. The red arrow indicates NisA.</CENTER></P><P>Since we aimed for a visually more pleasing confirmation, we used SDS-PAGE of pre-nisin with silver staining in figure N.6. </P><P><CENTER><B><I>Figure N.6</I>: SDS-PAGE of pre-nisin with silver staining.</B> As marker Roti-Mark Standard Protein Marker was used. The black arrow indicates NisA with a molecular weight of 7.37 kDa.</CENTER></P></DETAILS><H3>Outlook</H3><P>We are currently able to produce pre-nisin with the required tags for purification and linkage but for further application nisin has to be fully processed in order to enable its complete antimicrobial effects. Therefore, we cloned the processing enzyme cluster <I>nisB</I>, <I>nisT</I>, <I>nisC</I>, <I>nisI</I> and <I>nisP</I> (required for post-translational modification, resistance and export) in our BioBrick <A href="http://parts.igem.org/Part:BBa_K2848004">BBa_K2848004</A> downstream of <I>nisA</I>. This theoretically enables us to produce fully processed nisin containing the YAAY- and His<SUB>6</SUB>-tag C-terminal in a resistant <I>E. coli</I> which also exports nisin to the surrounding medium. Unfortunately, the construct still contains two forbidden SpeI restriction sites so it cannot be registered as a BioBrick. Those restriction sites will have to be removed using <I>site directed mutagenesis</I>.
Due to secretion of nisin and its inducible expression, production of large amounts of nisin is possible in the future using for example a bioreactor in a perfusion process.</P><DETAILS><SUMMARY><H2 align="center" span="" title="click to enlarge">Rhamnolipid</H2></SUMMARY><P>The transformation of <I>E. coli</I> DH5α with existing BioBricks BBa_K1331001 (<I>rhlA</I>), BBa_K1331002 (<I>rhlB</I>), BBa_K808000 (<I>araC</I>) and BBa_J04450 (mRFP) as well as pVLT33 containing <I>rhlC</I> was successfully carried out. Unfortunately not all required PCRs worked and it was not possible to amplify <I>rhlC</I>. However, the restriction digest provided the right sizes of fragments accorded with theoretical values leading to the conclusion of having the correct templates (Figure R.1). Since most of the PCRs were not successful (with temperature gradient, GC Enhancer and different concentrations) the project was dropped because time was running out. A possible reason could be the high complexity of the gene and especially the high GC-content. This could lead to an inhibition of the proofreading DNA polymerase <A name="refR.1" href="#reflR.1">[R.1]</A>. The content of <I>rhlC</I> is indicated by about 82 % in a window of 100 bases starting at base 843 (IDT).</P><FIGCAPTION class="figure-caption"><I>Figure R.1</I>: Results of agarose gel electrophoresis: AraC = PCR of araC at given temperature, <I>Rhl A</I> = PCR of <I>rhlA</I> at given temperature, <I>Rhl B</I> = PCR of <I>rhlB</I> at given temperature, <I>Rhl C</I> = PCR of <I>rhlC</I> at given temperature, <I>RhlC</I> BB = PCR of <I>rhlC</I> at given temperature with second primer pair for direct creation of a basic part BioBrick, <I>Rhl</I> A digest: digestion of pSB1C3 containing <I>rhlA</I> with EcoRI and SpeI, <I>Rhl</I> B digest: digestion of pSB1C3 containing <I>rhlB</I> with EcoRI and SpeI, AraC digest: digestion of pSB1C3 containing araC with EcoRI and SpeI, <I>Rhl C</I>: pVLT33 containing <I>rhlC</I> without digestion, mRFP digest: digestion of pSB1C3 containing mRFP with EcoRI and SpeI</FIGCAPTION><H3>Outlook</H3><P>Although production of rhamnolipids in <I>P. putida</I> could not be carried out successfully here, it has been demonstrated that establishing thereof is feasible. For example, the iGEM teams from Uppsala 2015 produced rhamnolipids in <I>E. coli</I> and University of Columbia 2016 in <I>P. putida</I>. Wittgens <I>et al.</I> also reported the construction of new production strains for rhamnolipid synthesis in <I>P. putida</I> KT2440 <A name="refR.2" href="#reflR.2">[R.2]</A>. 
</P><P>Heterologous production of di-rhamnolipids is feasible as previous mentioned examples show, since the molecular biological techniques are all standard methods nowadays. As <I>rhlC</I> could not be added to the registry as sought, this task is still open for following teams. 
</P></DETAILS><DETAILS><SUMMARY><H2 align="center" span="" title="click to enlarge">Linkage</H2></SUMMARY><P>The objective of our experiments was the chemical and enzymatic linkage of the produced compounds. Through linkage of the antimicrobial agents to the chitosan scaffold we generated our anti germ coating.</P><H3>Chemical linkage</H3><P>The aim of the experiment was the linking of the antimicrobial agent rhamnolipid to the chitosan scaffold by divinyladipate <A name="refL.1" href="#reflL.1">[L.1]</A>. Thereby, divinyladipate acts as a linker. By transesterfication an ester bond is formed between the components and they become connected.</P><P>First, we tried to verify the successful linkage by IR-spectroscopy. No specific peak was found to identify the formed ester bond. There was an ester peak in each spectrograph of the product. This peak likely originates from the ester bond in the rhamnolipid. Therefore, IR-spectroscopy was no appropriate method to verify the reaction and we tried an other approach to prove the success of the reaction.</P><P>We analyzed different mixtures of the educts as controls for the reaction (Figure L.1).</P><FIGCAPTION class="figure-caption"><I>Figure L.1:</I> Control reactions for chemical linkage. All reactions were executed according to the same protocol. Top left to right: chitosan-chitosan in ionic liquid, chitosan+rhamnolipid+divinyladipate-all educts were dissolved in ionic liquid, rhamnolipid-rhamnolipid in ionic liquid. Bottom left to right: chitosan+divinyladipate-chitosan and divinyladipate were dissolved, chitosan+rhamnolipid-chitosan and rhamnolipid in ionic liquid, rhamnolipid+divinyladipate-rhamnolipid and divinyladipate were dissolved in ionic liquid.</FIGCAPTION><P>Figure L.1 indicates that a solid compound only precipitated if all educts were added to the reaction. In the control approaches no solid precipitated. Following, the reaction was successful when all educts where mixed.</P><P>In order to find the ideal molecular ratios between the educts we tried different amounts of the substances. After diverse approaches we concluded that a ratio of one part AGU (anhydro glucose unit) chitosan per two molecules of rhamnolipid and divinyladipate (1:2:2) resulted in an optimal surface. This  ratio is equivalent to 1 % (w/v) chitosan and 0.93 mM of rhamnolipid and divinyladipate. The resulting surface showed the best properties regarding precipitation and antimicrobial effect. This surface was used for <A href="https://2018.igem.org/Team:Stuttgart/Demonstrate">antimicrobial tests and &quot;Demonstrate&quot;</A>.</P><H3>Enzymatic linkage</H3><P>For enzymatic linkage of nisin to chitosan we used a tyrosinase. This enzyme is described in diverse publications to link proteins to chitosan (<A name="refL.2" href="#reflL.2">[L.2]</A>, <A name="refL.3" href="#reflL.3">[L.3]</A>).</P><P>Prior to further experiments the activity of the tyrosinase was tested. Figure L.2 shows the activity of the substrate turnover of different tyrosine concentrations by tyrosinase.</P><FIGCAPTION class="figure-caption"><I>Figure L.2:</I> Enzyme activity of tyrosinase EC(1.14.18.1). The absorbance 280 nm was plotted over time in minutes. Different tyrosine concentrations were diluted in phosphate buffer. The reaction was started by adding 1 Unit tyrosinase.</FIGCAPTION><P>In Figure L.2 you can see the increase of absorbance at 280 nm in time indicating the oxidization of tyrosine. By oxidization of tyrosine the electron system of the aromatic ring is changed leading to a shift in the absorbance. The graph shows a sigmoidal curve which is typical for an enzyme reaction. The higher the tyrosine concentration the higher the final absorbance. Following the tyrosinase works as expected.</P><P><U>Proof of concept:</U></P><P>For linkage of nisin to chitosan a tyrosine residue has to be introduced first. While the Nisin-team was working on the modification of nisin we linked the tyrosine containing green fluourescence protein (GFP) to chitosan as a proof of concept. Therefore we tried two different <A href="https://2018.igem.org/Team:Stuttgart/Experiments">protocols</A> for the reaction and analyzed the reaction product by SDS-PAGE subsequently. Figure L.3 depicts the SDS-PAGE results for protocol 1 and 2. There were loaded samples from supernatants of different time points on the gel. For protocol 1 chitosan was mixed with GFP and tyrosinase and incubated for 3 days. In protocol 2 GFP and tyrosinase were incubated for 3 days before adding the chitosan solution.</P><FIGCAPTION class="figure-caption"><I>Figure L.3:</I> SDS-PAGE of tyrosinase based linkage of GFP to chitosan. Sample supernatants of different time points were loaded. A: Sample supernatants of time points (0, 24, 48, 72 h) of control and sample treated acording to protocol 1. B: Sample supernatants of time points (0, 72(1), 72(2), 144 h) of control and sample treated acording to protocol 2.</FIGCAPTION><P>The supernatant was loaded after short centrifugation of the samples. After centrifugation the chitosan was found in the pellet. Therefore bound GFP was also in the pellet. A decrease of protein in the supernatant indicates a linkage of GFP with chitosan. Figure L.3.A shows that the GFP band (~25 kDa) of samples of time points 24, 48 and 72 h  is smaller and less intense in contrast to the control. This indicates a decrease of GFP in the supernatant and therefore a successful linkage of GFP to chitosan. In addition you can see a new band (~63 kDa) for the samples at 24 and 48 h. This band likely originates from cross linked GFP. The oxydized tyrosine can form a covalent bond with the amine group of the chitosan but also with the amine group of the amino acids of the GFP. This side reaction was stronger for protocol 2 (Figure L.3.B). In this case the tyrosine of the GFP was first oxidized before adding the chitosan. Whilst oxidation the oxidized tyrosine residues reacted with amine groups of GFP because no chitosan was present. Nevertheless the connection of GFP to chitosan was successful, too. The GFP band (~25 kDa) of sample after 144 h was smaller and less intense in comparison to the control. However, the protocol 1 leaded to better results and less side products. Thus we suggest protocol 1 for linking proteins to chitosan.</P><H3>Outlook</H3><P>We showed successfully the production of a version of TAGC by chemical linkage. Tyrosine-containing proteins could be linked enzymatically to chitosan.</P><P>The procedure of chemical linkage can be improved by varying the ratios of the educts. The following  precipitation and wash steps can be optimized as well. To remove remaining solute (ionic liquid) and ease the processing of the polymer a liquid-liquid extraction might be helpful. This step enables also the recycling of the ionic liquid. The ionic liquid is quite expensive. Therefore you can save lot of money and resources by recycling of the reaction solute. Moreover the liquid-liquid extraction provides further possibilities to analyze the polymer (e.g. NMR-spectroscopy). To improve linkage and the features of TAGC other linkers than divinyladipate might be utilized in the reaction.</P><P>Basically linkage of proteins to chitosan is possible. The next objective is the linkage of nisin to chitosan. Hence varying ratios of nisin, chitosan and tyrosinase need to be examined for finding best degree of substitution (DS). Besides analyzing the reaction by PAGE, MALDI-TOF is a suitable high-end analyzing method. Precipitation of the generated polymer is an important step for processing. Further research and development are necessary to apply the product as coating.</P><P>The major goal for future development of TAGC is the final linkage of all components (rhamnolipid and nisin) to chitosan. Due to the synergistic effect of rhamnolipid and nisin we expect stronger antimicrobial effects.</P></DETAILS><H2 align="center">References</H2><P><A name="refl1" href="#ref1">[C.1]</A> Evaluation of the antimicrobial activity of chitosan and its quaternized derivative on <I>E. coli</I> and <I>S. aureus</I> growth. Rejane C. Goy, Sinara T.B. Morais Odilio B.G. Assis. <I>Brazilian Journal of Pharmacognosy</I>. 2016. Volume 26. Page 122-127.</P><P><A name="refR.1" href="#reflR.1">[R.1]</A> Guanine-rich sequences inhibit proofreading DNA polymerases, Zhu et al. 2016. <I>SCIENTIFIC REPORTS</I>. DOI: 10.1038/srep28769.</P><P><A name="refR.2" href="#reflR.2">[R.2]</A> Growth independent rhamnolipid production from glucose using the non-pathogenic <I>Pseudomonas putida</I> KT2440. A. Wittgens, T. Tiso, T.T. Arndt, P. Wenk, J. Hemmerich, C. Müller, R. Wichmann, B. Küpper, M. Zwick, S. Wilhelm, R. Hausmann, C. Syldatk, F. Rosenau, L.M. Blank. 2011. <I>Microb Cell Fact</I>, 10:80.</P><P><A name="refL.1" href="#reflL.1">[L.1]</A>  Homogeneous vinyl ester-based synthesis of different cellulose derivatives in 1-ethyl-3-methyl-imidazolium acetate, Hinner et al., 2016, Green Chemistry, DOI: 10.1039/C6GC02005D</P><P><A name="refL.2" href="#reflL.2">[L.2]</A> Tyrosinase-catalyzed modification of Bombyx mori silk fibroin: grafting of chitosan under heterogeneous reaction conditions, Freddi et al., 2006, Journal of Biotechnology, DOI: 10.1016/j.jbiotec.2006.03.003</P><P><A name="refL.3" href="#reflL.3">[L.3]</A> Enzyme-catalyzed gel formation of gelatin and chitosan: potential for in situ applications. Chen et al. 2003. <I>Biomaterials</I>. DOI: 10.1016/S0142-9612(03)00096-6.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>