<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Stuttgart_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Stuttgart/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="igem_2018_team_menu displaying_menu"><A href="https://2018.igem.org/Team:Stuttgart"><DIV class="menu_item direct_link">
				HOME
			</DIV></A><DIV class="menu_item">
			TEAM
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Team"><DIV class="submenu_item">
					Team Members
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Collaborations"><DIV class="submenu_item">
					Collaborations
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Sponsoring"><DIV class="submenu_item">
					Sponsoring
				</DIV></A></DIV><DIV class="menu_item">
			PROJECT
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Description"><DIV class="submenu_item">
					Description
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Design"><DIV class="submenu_item">
					Design
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Experiments"><DIV class="submenu_item">
					Experiments
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Notebook"><DIV class="submenu_item">
					Notebook
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/InterLab"><DIV class="submenu_item">
					InterLab
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Model"><DIV class="submenu_item">
					Model
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Results"><DIV class="submenu_item">
					Results
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Demonstrate"><DIV class="submenu_item">
					Demonstrate
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Improve"><DIV class="submenu_item">
					Improve
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Attributions"><DIV class="submenu_item">
					Attributions 
				</DIV></A></DIV><DIV class="menu_item">
			PARTS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Parts"><DIV class="submenu_item">
					Parts Overview
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Basic_Part"><DIV class="submenu_item">
					Basic Parts
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Composite_Part"><DIV class="submenu_item">
					Composite Parts
				</DIV></A></DIV><A href="https://2018.igem.org/Team:Stuttgart/Safety"><DIV class="menu_item direct_link">
				SAFETY
			</DIV></A><DIV class="menu_item"> 
			HUMAN PRACTICES 
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Human_Practices"><DIV class="submenu_item">
					Human Practices
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Public_Engagement"><DIV class="submenu_item">
					Education &amp; Engagement
				</DIV></A></DIV><DIV class="menu_item"> 
			AWARDS
		</DIV><DIV class="submenu"><A href="https://2018.igem.org/Team:Stuttgart/Entrepreneurship"><DIV class="submenu_item">
					Entrepreneurship
				</DIV></A><A href="https://2018.igem.org/Team:Stuttgart/Model"><DIV class="submenu_item">
					Model
				</DIV></A></DIV><A href="https://igem.org/2018_Judging_Form?team=Stuttgart"><DIV class="menu_item direct_link">
				JUDGING FORM ⇗ 
			</DIV></A></DIV><DIV class="igem_2018_team_content"><DIV class="igem_2018_team_column_wrapper"><DIV class="column full_size"><H1 align="center">Design</H1><DETAILS><SUMMARY><H3 align="center" span="" title="click to enlarge">Chitosan</H3></SUMMARY><P>Chitosan is a biopolymer of N-acetylglucosamine and glucosamine units in irregular patterns. It is built by chitin synthases and chitin deacetylases. Chitin synthases polymerize UDP-N-acetylglucosamine units to chitin. Chitin deacetylases attack at different positions and thereby generate chitosan of different length and degree of deacetylation. Naturally, chitosan occurs in cell walls of fungi. Up to now chitosan production is mostly based on the processing of waste parts from marine industries, e.g. from crustaceans.
</P><FIGCAPTION style="width: 100%" class="figure-caption"><I>Figure C.1: </I>Metabolic flow of chitosan synthesis. The chitin synthase NodC uses UDP-GlcNAc as sugar donor for synthesis of chitin oligomers. The chitin deacetylases NodB or COD hydrolysis the acetoamido group of the GlcNAc units. </FIGCAPTION><P>
Chemical and physical properties of chitosan are determined by degree and pattern of deacetylation as well as by the polymers length.
Short to midrange chitosan with low degrees of acetylation have strong antimicrobial properties <A name="refC.1" href="#reflC.1">[C.1]</A>.
Therefore, we concluded that chitosan could be an important part of <B>The Anti Germ Coating</B> as a base polymer. </P><P>We wanted to design a plasmid containing the chitin synthase NodC and chitin deacetylase NodB or rather COD based on the plasmids from last years iGEM Team Darmstadt (<A href="https://2017.igem.org/Team:TU_Darmstadt">ChiTUcare</A>).
</P><FIGCAPTION style="width: 100%" class="figure-caption"><I>Figure C.2: </I>Cloned construct of BBa_K2380002 and K2380043 (with His-tag). This BioBrick is registered as BBa_K2848001.</FIGCAPTION><P> Figure C.2 visualizes the planned plasmid of the chitosan subproject. Combining of chitin synthase and deacetylase in one cell should simplify the production of chitosan. 
Because of the challenges to extract NodB from the inclusion bodies, we focused on the production of COD as a chitin deacetylase.</P></DETAILS><DETAILS><SUMMARY><H3 align="center" span="" title="click to enlarge">Nisin</H3></SUMMARY><P>Antimicrobial peptides (AMPs) are a common defensive mechanism against pathogens and competitors used in mammalian complement system <A name="refN.1" href="#reflN.1">[N.1]</A> and as bacteriocines to inhibit the growth of similar or closely related bacterial strains <A name="refN.2" href="#reflN.2">[N.2]</A>. Nisin, a post translational modified lantibiotic (i.e., lanthionine containing antibiotic) is such a bacteriocine produced and secreted by <I>Lactococcus lactis</I>. We chose nisin as AMP since it has been used commercially for decades and is already approved as food additive, therefore minimizing the risk of negative health effects on humans.
For nisin production, <I>Lactococcus lactis</I> holds a nisin operon of around 15 kb comprising 9 individual genes of which <I>nisA</I> is coding for the nisin precursor shown in figure N.1.</P><P><CENTER><B><I>Figure N.1</I>:</B> Schematic representation of the organization of the nisin operon and location of the nisin promoters (modified after <A name="refN.3" href="#reflN.3">[N.3]</A>).</CENTER></P><P>Since we used an inducible arabinose promoter (araBAD) for overexpression, we excluded the natural regulating genes <I>nisR</I>, <I>nisK</I>, <I>nisF</I>, <I>nisE</I> and <I>nisG</I>. Whereas <I>nisB</I>, <I>nisT</I>, <I>nisC</I>, <I>nisI</I> and <I>nisP</I> are required for post-translational modification, resistance and export. The post-translational modifications of nisin are shown in figure N.2.</P><P><CENTER><B><I>Figure N.2</I>: Post-translational modifications in nisin.</B> Like all other lantibiotics, nisin is produced in an inactive precursor form containing a leader peptide that directs it to the modification and transport machinery. The modifications are performed in two steps. First, dehydration of the serines and threonines (green) by NisB requiring didehydroalanines (Dha) and didehydrobutyrines (Dhb), respectively (orange). Next, NisC catalyses the cyclization of the didehydro amino acids with cysteine residues that are located downstream (towards the carboxy terminus), which forms the lanthionine rings (Ala-S-Ala = lanthionine; Abu-S-Ala = methyl-lanthionine). During the dehydration and cyclization steps the stereochemistry of the serine (or threonine) residue is changed from the l- to the D-form. Only when the modifications are completed the peptide is exported (by NisT) from the cell and the precursor peptide is cleaved (by NisP), rendering nisin. The five lanthionine rings of nisin are labelled a–e from the N-terminus. <I>Abu</I>, aminobutyric acid <A name="refN.3" href="#reflN.3">[N.3]</A>.</CENTER></P><P>The modified nisin acts as bacteriocine and therefore as antibiotic against gram-positive bacteria via two mechanisms. First lipid II is bound by nisin and no longer available for incorporation into the growing peptidoglycan network. Following, the binding of lipid II a pore formation in the bacterial plasma membrane is observed (Figure N.3) leading to lysis of the bacterial cell.</P><P><CENTER><B><I>Figure N.3</I>: Dual mode of action of the lantibiotic nisin.</B> 1) Nisin forms a stoichiometric complex with lipid II, which triggers two fatal events. The cell wall precursor is abducted and no longer available for incorporation into the growing peptidoglycan network (left, modified after <A name="refN.2" href="#reflN.2">[N.2]</A>). 2) Target-directed pore-formation mechanism of nisin. First, nisin reaches the bacterial plasma membrane, where it binds to lipid II via two of its amino-terminal rings. This is then followed by pore formation, which involves a stable transmembrane orientation of nisin. During or after assembly of four 1:1 (nisin: lipid II) complexes, four additional nisin molecules are recruited to form the pore complex (right, modified after <A name="refN.3" href="#reflN.3">[N.3]</A>).</CENTER></P><P>Since we decided that we wanted to go for nisin as our AMP a few more requirements occurred. We wanted to produce nisin in <I>E. coli</I>. For that purpose, we introduced the araBAD promoter with strong RBS in front of the <I>nisA</I> gen replacing the natural promoter for inducible expression. For efficient purification of nisin we also introduced a His-tag. Since we created an antimicrobial surface, nisin had to be linked to the chitosan matrix. Therefore, we also introduced a YAAY-tag (tyr-ala-ala-tyr peptide). The tyrosine residues can be covalently linked to the amine group of chitosan by a tyrosinase. Those requirements lead to the design of several nisin constructs starting with pre-NisA production holding a YAAY and His-tag (Figure N.4) and resulting in a construct holding the complete post-translational modification, resistance and export machinery (Figure N5).</P><P><CENTER><B><I>Figure N.4</I>:</B> Map of our BioBrick <A href="http://parts.igem.org/wiki/index.php/Part:BBa_K2848004">BBa_K2848004</A></CENTER></P><P><CENTER><B><I>Figure N.5</I>:</B> Map of our theoretical BioBrick containing the araBAD promotor, and <I>NisA</I>, <I>NisB</I>, <I>NisT</I>, <I>NisC</I>, <I>NisI</I>, <I>NisP</I> of the Nisin operon.</CENTER></P></DETAILS></DIV><DETAILS><SUMMARY><H3 align="center" span="" title="click to enlarge">Rhamnolipid</H3></SUMMARY><P>As an essential part of <B>T</B>he <B>A</B>nti <B>G</B>erm <B>C</B>oating (<B>TAGC</B>) the production of rhamnolipids with their natural antimicrobial properties was established in <I>Pseudomonas putida</I> KT2440. According to Díaz De Rienzo and Martin, 2016 <A name="refR.1" href="#reflR.1">[R.1]</A>, a mixture of mono- and di-rhamnolipids enhances these properties. Therefore, the construction of a novel BioBrick containing the genes <I>rhlA</I>, <I>rhlB</I> and <I>rhlC</I> encoding rhamnosyltransferase II, an enzyme essential for the synthesis of di-rhamnolipids was planned.</P><DIV class="row"><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><I>Figure R.1:</I> Pathway: rhamnolipid biosynthesis in <I>Pseudomonas aeruginosa</I> PAO1 <A name="refR.2" href="#reflR.2">[R.2]</A></FIGCAPTION></FIGURE></DIV><P><I>P. putida</I> KT2440 is natively able to produce activated rhamnose on the one side as well as 2hydroxyacyl-ACP, two precursors of the rhamnolipid synthesis (figure R.1). The gene product of <I>rhlA</I>, an alkanoate synthase can link two fatty acid chains together. This gene as well as <I>rhlB</I> and <I>rhlC</I> are only present in <I>P. aeruginosa</I> but not in <I>P. putida</I>. In the next step, rhamnosyltransferase I (derived from <I>rhlB</I>) is capable of fusing the activated form of rhamnose (dTDP-rhamnose) with the hydroxyalkanoate precursor resulting in mono-rhamnolipids. In the last step, expression of rhamnosyltransferase II (<I>rhlC</I>) results in production of di-rhamnolipids by adding another activated dTDP-rhamnose to the mono-rhamnolipid.</P><P>The aim was to construct a plasmid containing the three genes <I>rhlA</I>, <I>rhlB</I> and <I>rhlC</I> for production of rhamnolipids under control of an <I>araC/P<SUB>BAD</SUB></I> (figure R.2), making the expression of all three genes inducible by presence of arabinose. On top of that, we aimed to insert additional ribosome binding sites before <I>rhlB</I> and <I>rhlC</I> to facilitate the expression. Positive transformants are selected by the chloramphenicol resistance. The correct composition of the plasmid is verified either by sequencing of the part or by restriction digest with EcoRI and SpeI followed by agarose gel electrophoresis. 
</P><DIV class="row"><DIV class="col-md-6"><FIGURE class="figure"><FIGCAPTION class="figure-caption"><I>Figure R.2:</I> Final gene construct containing the arabinose-inducible promoter <I>araC</I> as well as the genes necessary for rhamnolipid-production, <I>rhlA</I>, <I>rhlB</I> and <I>rhlC</I>. </FIGCAPTION></FIGURE></DIV><P>In the first step of cloning vector generation, the four necessary genes are amplified by polymerase chain reaction (PCR) while also adding ribosome binding sites before <I>rhlB</I> and <I>rhlC</I> as well as prefix to araC (already included since the existing BioBrick was used) and suffix to <I>rhlC</I>, respectively.</P><P>After successful cloning of all four fragments, two approaches of overlap extension PCR are performed to couple <I>araC</I> and <I>rhlA</I> as well as <I>rhlB</I> and <I>rhlC</I>, respectively.</P><P>In the last step, Gibson assembly is used to create one DNA fragment containing all four genes. Therefore, the primers rhlA_rev and rhlB_fwd were created with a long (40 bp) complementary overlap.</P><P>When the final DNA construct is achieved, it can be digested with EcoRI and SpeI and then inserted into the pSB1C3 backbone derived from BBa_J04450 by restriction digest and agarose gel cleanup. Insertion is mediated by T4-Ligase followed by transformation of <I>E. coli</I> DH5α for amplification of the new generated BioBrick. In the last step, the QuikChange site-directed mutagenesis kit (Agilent) is used to eliminating the illegal PstI restriction site inside the gene body of <I>rhlC</I>. Therefore, primers with a mismatch in position 327 in <I>rhlC</I> were created leading to a G → A mutation.</P><P>In parallel, <I>rhlC</I> is amplified by a second primer pair for production of the new basic part BioBrick of <I>rhlC</I>. In this case, a forward primer carrying the BioBrick prefix is used in combination with the same reverse primer used in the other setup. On top of that, the QuikChange site-directed mutagenesis kit (Agilent) is performed in the same way than before to eliminate the PstI restriction site. The exchange of base 327 is controlled by sequencing of this region.</P><P>For expression in <I>P. putida</I>, the whole construct is cloned into a vector carrying a compatible Ori, e. g. pBBR1MCS-2 which also contains a XbaI and SpeI restriction site for insertion of the constructed BioBrick.</P></DIV></DIV></DETAILS><DIV class="column full_size"><DETAILS><SUMMARY><H3 align="center" span="" title="click to enlarge">Linkage</H3></SUMMARY><P> After developing our idea and deciding for antimicrobial substances we needed to clarify how we manage to create TAGC. In order to receive a surface, all components need to be linked and stay together without losing their characteristic properties. Therefore, identification of functional moieties and possible attackable groups is essential. </P><P>TAGC consinsts of rhamnolipid and nisin connected with the polysaccharide chitosan which builds a stable base for the surface. Rhamnolipid and chitosan offer hydroxyl groups to make a linkage possible. Nisin is modified by introducing a tyrosine which can be oxidized by the enzyme tyrosinase and as a result a linking with amino groups of chitosan is possible.</P><P>Intensive researching and advisory support emerged a linkage achievable by chemical and enzymatical methods. To connect rhamnolipid and chitosan, transesterification can be conducted chemically or ester formation are catalyzed enzymatically. For transesterification, divinyladipate is used as a linker between the scaffold polymer chitosan and rhamnolipids <A name="refL.1" href="#reflL.1">[L.1]</A>. To connect nisin and chitosan in a enzymatical manner, the ability of tyrosinase to oxidate tyrosine is used. Resulting quinones attack nucleophilicly amino groups of the scaffold polymer chitosan <A name="refL.2" href="#reflL.2">[L.2]</A>. A tyrosines in nisin is introduced to enable desired linkage to chitosan.</P><P> First step to validate our designed plan was the actual production of a coating (see <A href="https://2018.igem.org/Team:Stuttgart/Results">Results</A>). Afterwards for examining antimicrobial effects we developed an assay and for testing as a coating we aplicated TAGC on a door handle (see <A href="https://2018.igem.org/Team:Stuttgart/Demonstrate">Demonstrate</A>).</P></DETAILS><H3 align="center">References</H3><P><A name="reflC.1" href="#refC.1">[C.1]</A> Insights into the Mode of Action of Chitosan as an Antibacterial Compound. D. Raafat, K. von Bargen, A. Haas, H-G. Sahl. <I>Applied and Environmental Microbiology</I>. 2008. Volume 74, Issue 12, pp 3764–3773. http://doi.org/10.1128/AEM.00453-08.</P><P><A name="reflN.1" href="#refN.1">[N.1]</A> Activation of the complement system generates antibacterial peptides. E.A. Nordahl <I>et al.</I> 2004. <I>Proc. Natl. Acad. Sci.</I>, Volume 101, pp 16879–16884.</P><P><A name="reflN.2" href="#refN.2">[N.2]</A> Multiple activities in natural antimicrobials. H.G. Sahl, G. Bierbaum. 2008.  <I>Microbe</I>, Volume 3, pp 467–473 .</P><P><A name="reflN.3" href="#refN.3">[N.3]</A> Lipid II as a target for antibiotics. E. Breukink and B. de Kruijff. 2006. <I>Nat. Rev. Drug Discov.</I> Volume 5, pp 321–323.</P><P><A name="reflR.1" href="#refR.1">[R.1]</A> Effect of Mono and Di-rhamnolipids on Biofilms Pre-formed by <I>Bacillus subtilis</I> BBK006. M.A. Díaz De Rienzo and P.J. Martin. <I>Curr Microbiol</I>. 2016. Volume 73, pp 183-189. DOI 10.1007/s00284-016-1046-4.</P><P><A name="reflR.2" href="#refR.2">[R.2]</A><A href="http://www.pseudomonas.com:1555/PSEUDO/NEW-IMAGE?object=PWY-6648"><I>Pseudomonas aeruginosa</I> PAO1 Pathway: rhamnolipid biosynthesis</A>.</P><P><A name="reflL.1" href="#refL.1">[L.1]</A> Homogeneous vinyl ester-base synthesis of different cellulose derivates in 1-ethyl-3-methyl-imidazolium acetate. Hinner et al. 2016. <I>Green Chemistry</I>, Volume 18, pp 6099-6107.</P><P><A name="reflL.2" href="#refL.2">[L.2]</A> Tyrosinase-catalyzed modification of Bombyx mori silk fibroin: Grafting of chitosan under heterogeneous reaction conditions. Freddi et al. 2006. <I>Journal of Biotechnology</I>, Volume 125, pp 281-294.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>