<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Greensboro-Austin_Smell_degradation"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Greensboro-Austin/Smell_degradation">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Greensboro-Austin/Smell_degradation&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Greensboro-Austin/Smell_degradation&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Greensboro-Austin/Smell_degradation&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Greensboro-Austin/Smell_degradation" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Greensboro-Austin/Smell degradation</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL class="cssmenu" style="float:left;"><LI class="home"><A href="/Team:Greensboro-Austin" title="home"><SPAN class="displace">Home</SPAN></A></LI><LI class="team"><A href="/Team:Greensboro-Austin/Team" title="team"><SPAN class="displace">Team</SPAN></A></LI><LI class="official_team_profile"><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Greensboro-Austin" title="official_team_profile"><SPAN class="displace">Official Team Profile</SPAN></A></LI><LI class="ncAAs"><A href="/Team:Greensboro-Austin/ncAAs" title="ncAAs"><SPAN class="displace">ncAAs</SPAN></A></LI><LI class="MAPs"><A href="/Team:Greensboro-Austin/MAPs" title="MAPs"><SPAN class="displace">MAPs</SPAN></A></LI><LI class="smell_degradation"><A href="/Team:Greensboro-Austin/Smell_degradation" class="selected" title="smell_degradation"><SPAN class="displace">Smell degradation</SPAN></A></LI><LI class="bacto_art"><A href="/Team:Greensboro-Austin/Bacto-Art" title="bacto_art"><SPAN class="displace">Bacto-Art</SPAN></A></LI><LI class="human_practices"><A href="/Team:Greensboro-Austin/Human_Practices" title="Human Practices"><SPAN class="displace">Human Practices</SPAN></A></LI><LI class="Standard_Proposal"><A href="/Team:Greensboro-Austin/Standard_Proposal" title="Standard_Proposal"><SPAN class="displace">Standard_Proposal</SPAN></A></LI><LI class="parts_submitted"><A href="/Team:Greensboro-Austin/Parts" title="parts_submitted"><SPAN class="displace">Parts Submitted</SPAN></A></LI><LI class="notebook"><A href="/Team:Greensboro-Austin/Notebook" title="notebook"><SPAN class="displace">Notebook</SPAN></A></LI><LI class="safety"><A href="/Team:Greensboro-Austin/Safety" title="safety"><SPAN class="displace">Safety</SPAN></A></LI><LI class="attributions"><A href="/Team:Greensboro-Austin/Team#Attributions" title="attributions"><SPAN class="displace">Attributions</SPAN></A></LI></UL></P><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Introduction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Introduction</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Pseudomonas_putida_NCIMB_9866"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Pseudomonas putida NCIMB 9866</SPAN></A></LI><LI class="toclevel-2 tocsection-3"><A href="#Bacillus_sp._JJ-1b"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Bacillus sp. JJ-1b</SPAN></A></LI><LI class="toclevel-2 tocsection-4"><A href="#Clostridium_difficile"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">Clostridium difficile</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-5"><A href="#Strategy"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Strategy</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#Plasmid_Design"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Plasmid Design</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#Plasmid_Assembly"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Plasmid Assembly</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#Assays"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Assays</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#Directed_Evolution"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">Directed Evolution</SPAN></A></LI><LI class="toclevel-1 tocsection-10"><A href="#Conclusions_and_Future_Directions"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Conclusions and Future Directions</SPAN></A></LI><LI class="toclevel-1 tocsection-11"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Introduction">Introduction</SPAN></H2><P><I>p</I>-Cresol (4-methylphenol) is a volatile organic compound present in most mammal fæces, correlated with having an odor characteristic to humans as fæcal and detectable at very dilute concentrations <A href="#References">[1]</A>.  The compound is present in its highest concentrations in pig manure and is formed in this context by the anaerobic degradation of the amino acid L-tyrosine, which occurs in the metabolism of the natural intestinal flora <A href="#References">[2,3]</A>.
</P><H3><SPAN class="mw-headline" id="Pseudomonas_putida_NCIMB_9866"><I>Pseudomonas putida</I> NCIMB 9866</SPAN></H3><P>The <I>pch</I> gene cluster follows the protocatechuate branch of the ortho pathway of <I>p</I>-cresol degradation, encoding for the enzymes <I>p</I>-cresol methylhydroxylase and  <I>p</I>-hydroxybenzaldehyde dehydrogenase, which degrade <I>p</I>-cresol to <I>p</I>-hydroxybenzoic acid <A href="#References">[4]</A>. This <I>pch</I> cluster has been studied in multiple strains of <I>Pseudomonas putida</I>and other microorganisms. One of these <I>P. putida</I> strains, NCIMB 9866, contains the <I>pchACXF</I> gene cluster, its <I>pch</I> genes have been sequenced and annotated, and it is readily available. If this gene cluster could be inserted into and expressed by a bacterial species that is as manipulable as <I>Escherichia coli</I>, it could become an effective tool for the bioremediation of <I>p</I>-cresol pollution.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:585px;"><DIV class="thumbcaption">Pathway for degrading p-cresol from <I>P</I>. putida NCIMB 9866 <A href="#References">[4]</A>.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Bacillus_sp._JJ-1b"><I>Bacillus</I> sp. JJ-1b</SPAN></H3><P>To create a self-sufficient bioremediation tool for <I>p</I>-cresol, the <I>pch</I> cluster will be accompanied by the <I>pra</I> cluster. This second gene cluster encodes for several enzymes that degrade the end product of the <I>pch</I> gene cluster, <I>p</I>-hydroxybenzoic acid, down a pathway that results in the formation of acetyl coenzyme A and pyruvate, which then feed into the citric acid cycle for energy production and cell growth <A href="#References">[5]</A>. The <I>praABEGFDCHI</I> gene cluster from the <I>Bacillus</I> sp. strain JJ-1b has been successfully expressed in <I>E. coli</I>, its <I>pra</I> genes sequenced and annotated, and is readily available <A href="#References">[6]</A>. A functional <I>E. coli</I> bacterium containing both the <I>pch</I> and <I>pra</I> gene clusters could be able to survive on <I>p</I>-cresol as its sole carbon source. It should be noted, however, that the <I>pra</I> plasmid is not necessary for deodorization of <I>p</I>-cresol. If successfully expressed, the <I>pch</I> plasmid alone would degrade <I>p</I>-cresol to the effectively odorless intermediate, <I>p</I>-hydroxybenzoic acid.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:602px;"><DIV class="thumbcaption">Pathway for degrading <I>p</I>-hydroxybenzoic acid from <I>Bacillus</I> sp. JJ-1b <A href="#References">[6]</A>.</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Clostridium_difficile"><I>Clostridium difficile</I></SPAN></H3><P><I>p</I>-Cresol has one final element of interest to humans: it is secreted by the dangerous microorganism <I>Clostridium difficile</I> when forming an infection in the intestine, which is common as a hospital acquired infection and can be fatal. The <I>p</I>-cresol secreted by <I>C. difficile</I> is toxic to almost all other microbes in the intestinal flora, severely disrupting gut function and allowing <I>C. difficile</I> to flourish in the absence of others <A href="#References">[7]</A>. An organism that could efficiently degrade the <I>p</I>-cresol being secreted might be able to restore gut function and inhibit the progression of the infection.
</P><H2><SPAN class="mw-headline" id="Strategy">Strategy</SPAN></H2><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:810px;"><DIV class="thumbcaption">Overall Design</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Plasmid_Design">Plasmid Design</SPAN></H3><P>The <I>pch</I> plasmid was designed to insert the 4.2kb <I>pchACXF</I> gene cluster into a backbone vector containing the pColA origin of replication, chloramphenicol resistance, and a double terminator from the Registry of Standard Biological Parts. In addition, the hybrid trp/lac promoter, or Ptac promoter system, was built in using ultramer oligonucleotides (Integrated DNA Technologies).
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:441px;"><DIV class="thumbcaption"><I>pch</I> Plasmid Design</DIV></DIV></DIV></DIV><P>The <I>pra</I> plasmid was designed to insert the 8.3kb <I>praABEGFDCHI</I> gene cluster into a backbone vector containing the p15A origin of replication, spectinomycin resistance, and a single terminator. Transcription of <I>pra</I> was controlled by the Ptac promoter system.
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:441px;"><DIV class="thumbcaption"><I>pra</I> Plasmid Design</DIV></DIV></DIV></DIV><P>The plasmid backbones were chosen due to their medium plasmid copy number (20-30), strong hybrid promoter with IPTG inducible expression, and a multiple antibiotic resistance selection scheme allowing for better cotransformant selection.
</P><H3><SPAN class="mw-headline" id="Plasmid_Assembly">Plasmid Assembly</SPAN></H3><P>In order to construct these plasmids, the organisms <I>P. putida</I> NCIMB 9866 and <I>Bacillus</I> sp. JJ-1b were ordered from the repositories NCIMB and ATCC, respectively. Both backbones were obtained from existing samples. After being revived, genomic DNA was extracted from both organisms and purified (Invitrogen). The <I>pchACXF</I> gene cluster was amplified as one fragment using a standard PCR reaction with a genomic DNA sample and Phusion polymerase (New England Biolabs). The <I>praABEGFDCHI</I>  gene cluster was split into three fragments and amplified separately as <I>praABE</I>, <I>praGFD</I>, and <I>praCHI</I> using a standard PCR reaction with a genomic DNA sample and Phusion polymerase. Amplifications were roughly checked for success by agarose gel electrophoresis, comparing the amplified DNA’s length against a ladder. Gels were recorded using a gel imager system, and then the amplified DNA was gel extracted (Invitrogen). After gel extraction, the purified DNA’s concentration was checked using a Nanodrop ultraviolet-visible spectrophotometer (Thermo Scientific).
</P><P>The plasmids containing the synthetic <I>pch</I> and <I>pra</I> operons were both constructed using Gibson isothermal assembly <A href="#References">[8]</A>. Primers were designed to create homology overlaps of approximately 35-45 base pairs in adjacent fragments and introduce synthetic sequences, such as the Ptac promoter system, where necessary. Purified fragments were placed in an aliquot of Gibson assembly mix and incubated at 50ºC for one hour. After plasmids were Gibson assembled, they were desalted on filters (Millipore), and then transformed by electroporation into the “odorless” and kanamycin resistant BW25113 ∆tnaA strain.
</P><P>After transformation, cells were recovered in Super Optimal Broth with 20mM glucose, and then spread onto LB agar plates with chloramphenicol antibiotic for the <I>pch</I> transformants, and LB agar plates with spectinomycin antibiotic for the <I>pra</I> transformants. From these plates, if there was any growth, individual colonies of varying characteristics were selected and a colony PCR performed using each pair of primers that were used to originally amplify the fragments of the plasmid.
</P><P>After each colony was picked, the pipette tip used to pick it was dropped in a test tube containing 4mL of LB and placed in a shaking incubator. The resulting amplified DNA from the colony PCR reaction was checked for length with agarose gel electrophoresis and recorded with the gel imager. Any colonies that passed the gel electrophoresis test for length were recovered from their test tubes, a sample was preserved in a frozen stock, and purified plasmid extracted for Sanger sequencing.
</P><H3><SPAN class="mw-headline" id="Assays">Assays</SPAN></H3><P>In order to assay if the synthetic operon is functioning correctly, the <I>pch</I> fragments were assembled and have been transformed into the <I>E. coli</I> strain BW25113 ∆<I>tnaA</I>, which is a single gene knockout mutant with the gene coding for the tryptophanase enzyme removed <A href="#References">[9]</A>. Tryptophanase breaks down the amino acid tryptophan into indole, pyruvate, and ammonium. With the tryptophanase gene intact, the production of indole is responsible for the characteristic fecal odor of <I>E. coli</I>. However, the BW25113 ∆<I>tnaA</I> strain does not produce indole, and therefore serves as an ideal chassis for a deodorizing construct because it is odorless. The <I>pch</I> and <I>pra</I> gene clusters will likely need to be refactored to function properly, replacing native <I>Pseudomonas</I> and <I>Bacillus</I> ribosome binding sites with those of <I>E. coli</I><A href="#References">[10]</A>.
</P><P>An ideal, albeit qualitative, experiment would be to cotransform both the <I>pch</I> and <I>pra</I> plasmids into a BW25113 ∆<I>tnaA</I> chassis and grow it in Davis and Mingioli (DM) minimal media supplemented only with <I>p</I>-cresol as a carbon source and no glucose. If the synthetic operons function as intended, the cells should be odorless before adding <I>p</I>-cresol, and when <I>p</I>-cresol is added, there should be a new, distinct malodor. Growth of the cells should result in the removal of some or all of the malodor.
</P><P>A more quantitative assay will involve using ultraviolet-visible absorbance spectroscopy on the media supernatant of centrifuged cells grown in minimal media at wavelengths shown to be absorbed by <I>p</I>-cresol and its degradation intermediates. Further confirmation could be achieved by high performance liquid chromatography.
</P><H3><SPAN class="mw-headline" id="Directed_Evolution">Directed Evolution</SPAN></H3><P>We are currently running a directed evolution experiment to evolve an <I>E. coli</I> chassis that can thrive at the highest concentrations of <I>p</I>-cresol possible. By implementing a strong selection pressure for survival at increasing <I>p</I>-cresol concentrations, mutations that improve fitness in a <I>p</I>-cresol saturated environment would be selected for.
</P><P>The experiment began with the “odorless” and kanamycin resistant BW25113 ∆<I>tnaA</I> strain, grown in LB overnight and streaked out on an LB agar plate with kanamycin to select individual colonies for the three cell lines.  Each selected colony was then inoculated in a flask with 10mL liquid DM media at a 500 mg/L glucose concentration. On the first day, each cell line was supplemented with <I>p</I>-cresol at concentrations of 2mM, 3mM and 4mM from a working stock of 0.5M <I>p</I>-cresol in ethanol solution along with a control flask with media and the lowest concentration of <I>p</I>-cresol used. After every day’s culturing, the lines were grown at 37º C in a shaking incubator at 200rpm for 24 hours, then transferred to a new flask by a 1:1000 dilution of 10µL of cells into 10mL of new minimal media.
</P><H2><SPAN class="mw-headline" id="Conclusions_and_Future_Directions">Conclusions and Future Directions</SPAN></H2><P>The pch and pra plasmids were assembled, and one clone transformed with the pch gene cluster was confirmed to be correctly assembled after sequencing. This clone was grown in liquid DM 500 media supplemented with <I>p</I>-cresol and grown overnight. The cells were induced with IPTG before assaying. The media containing the overnight growth was spun down in a centrifuge and the supernatant was analyzed with UV-visible spectroscopy. There was no perceptible difference in the supernatant after the cells were grown and then induced.
</P><P>One probable explanation is that the <I>E. coli</I> cells are not recognizing the <I>Pseudomonas</I> ribosome binding site sequences native to the genetic material that was transformed with the plasmid. Without a recognizable sequence for the ribosome to bind to, the genes would have no way of being expressed and degrading the <I>p</I>-cresol. Another possible explanation is that the UV-visible spectroscopy analysis is insufficient for evaluating this amount and type of degradation.
</P><P>The <I>pch</I> plasmid has since been redesigned to incorporate an <I>E. coli</I> ribosome binding site sequence, specificially the BBa_B0034 biobrick part, for each of the four genes in the cluster. Assembly is currently in progress and has been less successful than the previous construct without the <I>E. coli</I> sequences. In addition, future confirmed constructs will be assayed using high performance liquid chromatography with an apHera C18 polymer column (Sigma-Aldrich), which is a more accurate measure of success for this amount and type of degradation.
</P><H2><SPAN class="mw-headline" id="References">References</SPAN></H2><OL><LI>Zahn, J. A., Dispirito, A. A., Do, Y. S., Brooks, B. E., Cooper, E. E., &amp; Hatfield, J. L. (2001). Correlation of Human Olfactory Responses to Airborne Concentrations of Malodorous. J. Environ. Qual., 30(April), 624–634.
</LI><LI>Antoine, P., Taillieu, X., &amp; Thonart, P. (1997). The Degradation of L-Tyrosine to Phenol and Benzoate in Pig Manure. Biotechnology for Fuels and Chemicals (pp. 707–717).
</LI><LI>Mathus, T. L., &amp; Miller, W. (1995). Anaerobic biogenesis from L-tyrosine of phenol and <I>p</I>-cresol. Fuel, 74(10), 1505–1508.
</LI><LI>Jõesaar, M., Heinaru, E., Viggor, S., Vedler, E., &amp; Heinaru, A. (2010). Diversity of the transcriptional regulation of the <I>pch</I> gene cluster in two indigenous <I>p</I>-cresol-degradative strains of <I>Pseudomonas fluorescens</I>. FEMS Micriobiology Ecology, 72(3), 464-475.
</LI><LI>Crawford, R. L. (1975). Novel pathway for degradation of protocatechuic acid in <I>Bacillus</I> species. Journal Of Bacteriology, 121, 531–536.
</LI><LI>Kasai, D., Fujinami, T., Abe, T., Mase, K., Katayama, Y., Fukuda, M., &amp; Masai, E. (2009). Uncovering the protocatechuate 2,3-cleavage pathway genes. Journal of Bacteriology, 191(21), 6758–6768. 
</LI><LI>Dawson, L. F., Stabler, R. A., &amp; Wren, B. W. (2008). Assessing the role of <I>p</I>-cresol tolerance in <I>Clostridium difficile</I>. Journal of Medical Microbiology, 57(Pt 6), 745-749. 
</LI><LI>Gibson, D. G., Young, L., Chuang, R.-Y., Venter, J. C., Hutchison, C. A., &amp; Smith, H. O. (2009). Enzymatic assembly of DNA molecules up to several hundred kilobases. Nature Methods, 6(5), 343–345.
</LI><LI>Baba, T., Ara, T., Hasegawa, M., Takai, Y., Okumura, Y., Baba, M., Datsenko, K. A., et al. (2006). Construction of <I>Escherichia coli</I> K-12 in-frame, single-gene knockout mutants: the Keio collection. Molecular Systems Biology, 2, 2006.0008. 
</LI><LI>Quandt, E. M., Hammerling, M. J., Summers, R. M., Otoupal, P. B., Slater, B., Alnahhas, R. N., Dasgupta, A., et al. (2013). Decaffeination and Measurement of Caffeine Content by Addicted <I>Escherichia coli</I> with a Refactored <I>N</I>-Demethylation Operon from <I>Pseudomonas putida</I> CBB5. ACS synthetic biology, 2(6), 301–307.
</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Greensboro-Austin/Smell_degradation">http://2013.igem.org/Team:Greensboro-Austin/Smell_degradation</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Greensboro-Austin/Smell_degradation" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Greensboro-Austin/Smell_degradation" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Greensboro-Austin/Smell_degradation&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Greensboro-Austin/Smell_degradation&amp;oldid=329938" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>