<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Greensboro-Austin_MAPs"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Greensboro-Austin/MAPs">Page               </A></LI><LI><A href="/Talk:Team:Greensboro-Austin/MAPs">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Greensboro-Austin/MAPs&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Greensboro-Austin/MAPs&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Greensboro-Austin/MAPs" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Greensboro-Austin/MAPs</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL class="cssmenu" style="float:left;"><LI class="home"><A href="/Team:Greensboro-Austin" title="home"><SPAN class="displace">Home</SPAN></A></LI><LI class="team"><A href="/Team:Greensboro-Austin/Team" title="team"><SPAN class="displace">Team</SPAN></A></LI><LI class="official_team_profile"><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Greensboro-Austin" title="official_team_profile"><SPAN class="displace">Official Team Profile</SPAN></A></LI><LI class="ncAAs"><A href="/Team:Greensboro-Austin/ncAAs" title="ncAAs"><SPAN class="displace">ncAAs</SPAN></A></LI><LI class="MAPs"><A href="/Team:Greensboro-Austin/MAPs" class="selected" title="MAPs"><SPAN class="displace">MAPs</SPAN></A></LI><LI class="smell_degradation"><A href="/Team:Greensboro-Austin/Smell_degradation" title="smell_degradation"><SPAN class="displace">Smell degradation</SPAN></A></LI><LI class="bacto_art"><A href="/Team:Greensboro-Austin/Bacto-Art" title="bacto_art"><SPAN class="displace">Bacto-Art</SPAN></A></LI><LI class="human_practices"><A href="/Team:Greensboro-Austin/Human_Practices" title="Human Practices"><SPAN class="displace">Human Practices</SPAN></A></LI><LI class="Standard_Proposal"><A href="/Team:Greensboro-Austin/Standard_Proposal" title="Standard_Proposal"><SPAN class="displace">Standard_Proposal</SPAN></A></LI><LI class="parts_submitted"><A href="/Team:Greensboro-Austin/Parts" title="parts_submitted"><SPAN class="displace">Parts Submitted</SPAN></A></LI><LI class="notebook"><A href="/Team:Greensboro-Austin/Notebook" title="notebook"><SPAN class="displace">Notebook</SPAN></A></LI><LI class="safety"><A href="/Team:Greensboro-Austin/Safety" title="safety"><SPAN class="displace">Safety</SPAN></A></LI><LI class="attributions"><A href="/Team:Greensboro-Austin/Team#Attributions" title="attributions"><SPAN class="displace">Attributions</SPAN></A></LI></UL></P><H1><SPAN class="mw-headline" id="Mussel_adhesion_proteins_.28MAPs.29"> Mussel adhesion proteins (MAPs) </SPAN></H1><DIV class="thumb tright"><DIV class="thumbinner" style="width:202px;"><DIV class="thumbcaption">The &quot;foot&quot; of a mussel. Source: <A href="http://www.bigelow.org/" class="external text" rel="nofollow">bigelow.org</A></DIV></DIV></DIV><P>The <A href="http://en.wikipedia.org/wiki/Mussel" class="external text" rel="nofollow">mussel</A> is a marine mollusk that lives in the <A href="http://en.wikipedia.org/wiki/Intertidal_zone" class="external text" rel="nofollow">inter-tidal zone</A> on wave washed rocks. To withstand the tidal forces, they must bind themselves to these rocks with their endogenous glue. Mussels have an organ called the foot, which is used to both pull the mussel through terrain and to excrete a structure called the <A href="http://en.wikipedia.org/wiki/Byssus" class="external text" rel="nofollow">byssus</A>. These byssal threads are made up of collagen and other proteins and are what allow the mussel to attach to surfaces. 
</P><P>The properties of mussel glue make it an attractive target for synthetic improvement; mussel glue allows attachment to many different types of surfaces, both organic and inorganic. It is strong and durable as it is able to withstand the harsh ocean environment. It is made of protein, so it is biodegradable and is environmentally friendly. Most importantly, however, it is biocompatible (non-immunogenic) and—unlike most glues—it works in aqueous environments.
</P><P>There are several different proteins that make up the byssal threads, but the ones we’re most interested in are the ones that lie at the interface with the surface to which the mussel adheres. These proteins are responsible for the binding to the substrates and therefore have the most potential for use as a glue.
</P><P>All of the different mussel foot proteins have a percentage of their tyrosines post-translationally modified to a molecule called L-DOPA. It turns out that the proteins closer to the adhesion surface have a much higher L-DOPA content, and in fact the L-DOPA content is linearly correlated with adhesiveness.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:216px;"><DIV class="thumbcaption">Amino acid sequence of fp-151 with high tyrosine content</DIV></DIV></DIV><P>To the right is the sequence of a recombinant mussel adhesion protein, fp-151, that has good adhesive properties. Notably, it has high tyrosine content, denoted by the letter Y.
</P><P>It turns out that tyrosines are critical for adhesion, but first they must undergo tyrosinase-mediated hydroxylation to L-DOPA, which can be further oxidized to dopaquinone:
</P><P>These modified amino acids contribute to adhesion via two separate mechanisms: L-DOPA interacts <B>non-covalently</B> with <B>inorganic surfaces</B> by chelating metal ions, whereas dopaquinone interacts with <B>organic surfaces</B> through the formation of <B>covalent</B> bonds.
</P><H2><SPAN class="mw-headline" id="Previous_work"> Previous work </SPAN></H2><DIV class="thumb tleft"><DIV class="thumbinner" style="width:252px;"><DIV class="thumbcaption">Previous work with MAPs and L-DOPA generation</DIV></DIV></DIV><P>In 2007, a recombinant protein called fp-151 was first produced, which was notable for its lower cytotoxicity than previous recombinant foot proteins. After purifying the protein, tyrosine residues were hydroxylated <I>in vitro</I> with tyrosinase, so the protein was already folded before this modification could occur, possibly limiting access to the tyrosines <A href="#References">[1]</A>.
Choi, et al. extended this work with a coexpression system, such that the residues were modified <I>in vivo</I>, simultaneously along with translation. These foot proteins had better adhesive properties <A href="#References">[2]</A>.
Also in 2012, a system to generate L-DOPA from free tyrosine was engineered <A href="#References">[3]</A>.
</P><DIV class="thumb tright"><DIV class="thumbinner" style="width:352px;"><DIV class="thumbcaption">Limitations of previous L-DOPA work</DIV></DIV></DIV><P>However, these systems are not yet optimal for studying the effects of L-DOPA content on adhesive properties. Ideally, you need the ability to choose exactly which sites that L-DOPA should be incorporated. In current systems, you can only choose where there should be tyrosines. 
</P><P>Additionally, in current systems there is no guarantee that a tyrosine placed at your desired location will even end up as an L-DOPA, so we would also want the ability to know for certain that the residues of interest are in fact L-DOPA and not tyrosine.
</P><H2><SPAN class="mw-headline" id="Our_solution:_direct_non-canonical_amino_acid_incorporation"> Our solution: direct non-canonical amino acid incorporation </SPAN></H2><P>Incorporating the non-canonical amino acid L-DOPA <B>directly</B> would solve these problems of control and predictability. The <I>M. jannaschii</I> tRNA synthetase has been evolved to recognize the non-canonical amino acid L-DOPA by sequential rounds of positive and negative selection. The tRNA’s anticodon loop recognizes the UAG Amber stop codon and therefore this codon is now repurposed to code for a 21st amino acid. 
</P><P>In addition, release factor 1, which normally recognizes this Amber codon, can be knocked out, ensuring no competition between the stop codon’s old effect (termination) and its new effect (L-DOPA incorporation). The result of this system is an expansion of the cell’s genetic code to 21 amino acids, allowing precise control of targeted L-DOPA incorporation.
</P><H3><SPAN class="mw-headline" id="E._coli_as_a_self-sufficient_microbial_factory"><I>E. coli</I> as a self-sufficient microbial factory </SPAN></H3><P>Our vision is of <I>E. coli</I> as a biosynthetic &quot;factory&quot; able to make complicated products with simple inputs. By giving it the machinery to charge tRNA with L-DOPA, and expanding the genetic code to include L-DOPA, this non-canonical amino acid can be incorporated at rationally targeted locations, generating a library of foot protein variants. 
</P><P>Appropriate with this characterization of self-sufficiency and synthetic capability, these E. coli would not even need to be supplemented with exogenous L-DOPA, as they could continually generate it from intracellular free tyrosine.
</P><H3><SPAN class="mw-headline" id="Experimental_design"> Experimental design </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Plasmids containing fp-151 variant library and L-DOPA biosynthesis cassette (top) and synthetase/tRNA pair (bottom).</DIV></DIV></DIV><P>A two-plasmid system would be appropriate for our aims. One plasmid would code for a library of fp-151 variants, each containing Amber codons in unique locations and quantities, yielding a diverse set of foot proteins with characteristic locations and numbers of L-DOPA residues. The adhesive properties of these variants would be compared against both in vitro and in vivo tyrosinase-treated foot proteins, which would serve as positive controls. 
</P><P>This plasmid would also contain our L-DOPA biosynthesis cassette, ensuring a continual source of non-canonical amino acid without supplementation. 
</P><P>The second plasmid would code for the synthetase/tRNA pair that is responsible for adding L-DOPA to the genetic vocabulary. 
</P><H3><SPAN class="mw-headline" id="Diversity_of_Parts"> Diversity of Parts </SPAN></H3><P>The individual components of this system are derived from a variety of organisms. The foot proteins come from mussels, the genes in the L-DOPA biosynthesis cassette come from mice and humans, and the synthetase/tRNA pair is derived from archaea, and all these components are put in bacteria. 
</P><P>This covers 5 different organisms from all three domains of life, and as such would be a remarkable illustration of synthetic biology!
</P><H2><SPAN class="mw-headline" id="Results"> Results </SPAN></H2><P>We have built a small library of 17 different versions of fp-151, with various amounts and locations of Amber stop codons. 
</P><P>Variants with ambers only in fp5:
</P><TABLE cellpadding="5" border="1" cellspacing="0"><TBODY><TR><TH>Name
</TH><TH>5'fp1
</TH><TH>fp5
</TH><TH>3'fp1
</TH><TH># Ambers
</TH></TR><TR><TD>MAP01
</TD><TD>fp1_5’_NoAmb
</TD><TD>fp5_NoAmb
</TD><TD>fp1_3’_NoAmb
</TD><TD>0
</TD></TR><TR><TD>MAP02
</TD><TD>fp1_5’_NoAmb
</TD><TD>fp5_4Amb
</TD><TD>fp1_3’_NoAmb
</TD><TD>4
</TD></TR><TR><TD>MAP03
</TD><TD>fp1_5’_NoAmb
</TD><TD>fp5_7Amb
</TD><TD>fp1_3’_NoAmb
</TD><TD>7
</TD></TR><TR><TD>MAP04
</TD><TD>fp1_5’_NoAmb
</TD><TD>fp5_10Amb
</TD><TD>fp1_3’_NoAmb
</TD><TD>10
</TD></TR><TR><TD>MAP05
</TD><TD>fp1_5’_NoAmb
</TD><TD>fp5_20Amb
</TD><TD>fp1_3’_NoAmb
</TD><TD>20
</TD></TR></TBODY></TABLE><P>Variants with ambers only in fp1:
</P><TABLE cellpadding="5" border="1" cellspacing="0"><TBODY><TR><TH>Name
</TH><TH>5'fp1
</TH><TH>fp5
</TH><TH>3'fp1
</TH><TH># Ambers
</TH></TR><TR><TD>MAP06
</TD><TD>fp1_5'_2Amb
</TD><TD>fp5_NoAmb
</TD><TD>Fp1_3’_NoAmb
</TD><TD>2
</TD></TR><TR><TD>MAP08
</TD><TD>fp1_5'_2Amb
</TD><TD>fp5_NoAmb
</TD><TD>fp1_3'_5Amb
</TD><TD>7
</TD></TR><TR><TD>MAP09
</TD><TD>fp1_5'_5Amb
</TD><TD>fp5_NoAmb
</TD><TD>fp1_3'_5Amb
</TD><TD>10
</TD></TR><TR><TD>MAP10
</TD><TD>fp1_5'_5Amb
</TD><TD>fp5_NoAmb
</TD><TD>fp1_3'_9Amb
</TD><TD>14
</TD></TR><TR><TD>MAP11
</TD><TD>fp1_5'_10Amb
</TD><TD>fp5_NoAmb
</TD><TD>fp1_3'_9Amb
</TD><TD>19
</TD></TR></TBODY></TABLE><P>Mixed variants:
</P><TABLE cellpadding="5" border="1" cellspacing="0"><TBODY><TR><TH>Name
</TH><TH>5'fp1
</TH><TH>fp5
</TH><TH>3'fp1
</TH><TH># Ambers
</TH></TR><TR><TD>MAP02
</TD><TD>fp1_5’_NoAmb
</TD><TD>fp5_4Amb
</TD><TD>fp1_3’_NoAmb
</TD><TD>0
</TD></TR><TR><TD>MAP12
</TD><TD>fp1_5'_2Amb
</TD><TD>fp5_4Amb
</TD><TD>fp1_3'_2Amb
</TD><TD>4
</TD></TR><TR><TD>MAP13
</TD><TD>fp1_5'_2Amb
</TD><TD>fp5_7Amb
</TD><TD>fp1_3'_2Amb
</TD><TD>9
</TD></TR><TR><TD>MAP14
</TD><TD>fp1_5'_5Amb
</TD><TD>fp5_4Amb
</TD><TD>fp1_3'_5Amb
</TD><TD>14
</TD></TR><TR><TD>MAP15
</TD><TD>fp1_5'_5Amb
</TD><TD>fp5_7Amb
</TD><TD>fp1_3'_5Amb
</TD><TD>17
</TD></TR><TR><TD>MAP16
</TD><TD>fp1_5'_5Amb
</TD><TD>fp5_10Amb
</TD><TD>fp1_3'_5Amb
</TD><TD>20
</TD></TR><TR><TD>MAP17
</TD><TD>fp1_5'_5Amb
</TD><TD>fp5_20Amb
</TD><TD>fp1_3'_5Amb
</TD><TD>30
</TD></TR><TR><TD>MAP18
</TD><TD>fp1_5'_10Amb
</TD><TD>fp5_20Amb
</TD><TD>fp1_3'_9Amb
</TD><TD>39
</TD></TR></TBODY></TABLE><P>These variants are sequence-verified and testing is in progress.
</P><H2><SPAN class="mw-headline" id="References"> References </SPAN></H2><OL><LI> Hwang DS, Gim Y, Yoo HJ, Cha HJ. Practical recombinant hybrid mussel bioadhesive fp-151. Biomaterials. 2007 Aug;28(24):3560-8. Epub 2007 May 3. PubMed PMID: 17507090.
</LI><LI> Choi YS, Yang YJ, Yang B, Cha HJ. In vivo modification of tyrosine residues in recombinant mussel adhesive protein by tyrosinase co-expression in Escherichia coli. Microb Cell Fact. 2012 Oct 24;11:139. doi: 10.1186/1475-2859-11-139. PubMed PMID: 23095646; PubMed Central PMCID: PMC3533756.
</LI><LI> Satoh Y, Tajima K, Munekata M, Keasling JD, Lee TS. Engineering of L-tyrosine oxidation in Escherichia coli and microbial production of hydroxytyrosol. Metab Eng. 2012 Nov;14(6):603-10. doi: 10.1016/j.ymben.2012.08.002. Epub 2012 Aug 29. PubMed PMID: 22948011.
</LI></OL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Greensboro-Austin/MAPs">http://2013.igem.org/Team:Greensboro-Austin/MAPs</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Greensboro-Austin/MAPs" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Greensboro-Austin/MAPs" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Greensboro-Austin/MAPs&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Greensboro-Austin/MAPs&amp;oldid=241680" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>