<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Greensboro-Austin"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Greensboro-Austin">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Greensboro-Austin&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Greensboro-Austin&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Greensboro-Austin&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Greensboro-Austin" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Greensboro-Austin</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><P><UL class="cssmenu" style="float:left;"><LI class="home"><A href="/Team:Greensboro-Austin" class="selected" title="home"><SPAN class="displace">Home</SPAN></A></LI><LI class="team"><A href="/Team:Greensboro-Austin/Team" title="team"><SPAN class="displace">Team</SPAN></A></LI><LI class="official_team_profile"><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=Greensboro-Austin" title="official_team_profile"><SPAN class="displace">Official Team Profile</SPAN></A></LI><LI class="ncAAs"><A href="/Team:Greensboro-Austin/ncAAs" title="ncAAs"><SPAN class="displace">ncAAs</SPAN></A></LI><LI class="MAPs"><A href="/Team:Greensboro-Austin/MAPs" title="MAPs"><SPAN class="displace">MAPs</SPAN></A></LI><LI class="smell_degradation"><A href="/Team:Greensboro-Austin/Smell_degradation" title="smell_degradation"><SPAN class="displace">Smell degradation</SPAN></A></LI><LI class="bacto_art"><A href="/Team:Greensboro-Austin/Bacto-Art" title="bacto_art"><SPAN class="displace">Bacto-Art</SPAN></A></LI><LI class="human_practices"><A href="/Team:Greensboro-Austin/Human_Practices" title="Human Practices"><SPAN class="displace">Human Practices</SPAN></A></LI><LI class="Standard_Proposal"><A href="/Team:Greensboro-Austin/Standard_Proposal" title="Standard_Proposal"><SPAN class="displace">Standard_Proposal</SPAN></A></LI><LI class="parts_submitted"><A href="/Team:Greensboro-Austin/Parts" title="parts_submitted"><SPAN class="displace">Parts Submitted</SPAN></A></LI><LI class="notebook"><A href="/Team:Greensboro-Austin/Notebook" title="notebook"><SPAN class="displace">Notebook</SPAN></A></LI><LI class="safety"><A href="/Team:Greensboro-Austin/Safety" title="safety"><SPAN class="displace">Safety</SPAN></A></LI><LI class="attributions"><A href="/Team:Greensboro-Austin/Team#Attributions" title="attributions"><SPAN class="displace">Attributions</SPAN></A></LI></UL></P><H1 style="clear:left;">Bringing non-canonical amino acids (ncAAs) to iGEM</H1><P>The natural set of 20 amino acids permits a wide range of chemistry, but the ubiquity of post-translational modifications indicates that these 20 are often not enough for the diverse catalysis of life. The redundancy of the genetic code means that there is room for expansion—the &quot;Amber&quot; stop codon (UAG), the least-used stop codon in <I>E. coli</I>, has been successfully repurposed to code for a 21st amino acid (Wang 2001). However, this technology has not yet come to iGEM. We have BioBricked and tested an aminoacyl-tRNA synthetase / tRNA pair that &quot;suppresses&quot; the Amber stop codon, and can be easily mutagenized to incorporate many possible non-canonical amino acids. 
</P><H1 style="clear:left;">GluE.coli: Incorporating ncAAs to improve adhesive properties</H1><P>This project aims to use the technology of non-canonical amino acid incorporation to produce the mussel adhesion proteins (MAPs) that mussels employ to anchor themselves to the environment. Mussel adhesive proteins have amassed much attention as a potential adhesive for biomedical, underwater and other commercially relevant applications. Mussel adhesive proteins are also sought for their biodegradability, biocompatibility and ability to adhere to various substrates. However, production of mussel adhesive proteins proves to be an arduous task. Extraction-based production and in-vitro based production are expensive, inefficient, and unsustainable. Thus, our team focused on improving the efficiency of in-vivo production of MAPs. MAPs derive their adhesive properties from the hydroxylation of tyrosine to L-3,4-dihydroxyphenylalanine (L-DOPA) then to dopaquinone.  This project aims to reprogram the amber stop codon (TAG) to incorporate the non-canonical amino acid L-DOPA. In turn, this technique provides tighter control of L-DOPA incorporation whereas previous in-vivo projects depended on post-translational modification of tyrosine residues. Our team aims to develop the technology for large-scale MAP production in <I>E. coli</I> that will ultimately allow for rapid, cost-effective, and commercially viable production of an adhesive for biomedical applications.
</P><H1 style="clear:left;"><I>D. odori</I></H1><P>As long as animals have been domesticated, farmers have sought a solution to decreasing odor emissions with a close eye on cost-effectiveness. In swine manure, the worst culprit is the compound <I>p</I>-cresol.  To remedy this, UT iGEM is using engineered E. coli to degrade <I>p</I>-cresol down to Acetyl-CoA and pyruvate, two potential carbon sources for our bacteria to use for growth. Using directed evolution, strains with a high efficiency of degradation and the ability to use <I>p</I>-cresol as their sole carbon source could be isolated. Ultimately, this strain could be potentially used in probiotics for livestock and pets for odor reduction. 
</P><H1 style="clear:left;">Bacto-Art</H1><P>Using protein engineering techniques such as random mutagenesis and directed evolution, researchers have developed a toolkit of fluorescent protein (FP) color variants that spans the rainbow. This, in turn, has inspired a new brand of “living art” in which a palette of bacterial clones, each expressing a different FP color variant, are “painted” onto a petri dish to create colorful, glowing works of art (see, for example, <A href="http://tsienlab.ucsd.edu/Images.htm" class="external free" rel="nofollow">http://tsienlab.ucsd.edu/Images.htm</A>). Bacto-Art takes this concept one step further. With Bacto-Art, each bacterium contains a complete palette of FP genes encoded on a single plasmid. The specific color variant that is expressed is determined by the local environment of the bacterium. To accomplish this, we created a plasmid, termed pBactoArt, that contains a series of three inducible promotors (pLac, pHyc, and pXyl), each controlling the expression of a different FP color variant (ECFP, EYFP and EGFP, respectively). In the uninduced state, the expression of all FP variants is repressed leading to white colonies. Meanwhile, in the presence of a given inducer, the corresponding FP color variant is expressed, leading to colonies that are colored accordingly. By patterning the inducers on a petri dish—using either a paintbrush or an inkjet printer—new images emerge when pBactoArt-containing bacteria are plated on the dish. In the future, we plan to incorporate additional promotors and FPs, some with unique photophysical properties (e.g., the photo-switchable FP, Dronpa), to expand the types of art that can be done using BactoArt (e.g., simple animation). Aside from its applications in art (and possibly cryptology), we envision that Bacto-Art can be a valuable tool to introduce middle- and high school students to the concepts of inducible promotors and regulated gene expression. 
</P><DIV id="footer"><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:Greensboro-Austin">http://2013.igem.org/Team:Greensboro-Austin</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Greensboro-Austin" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Greensboro-Austin" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Greensboro-Austin&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Greensboro-Austin&amp;oldid=241537" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>