"
Page
Discussion
View source
History
teams
Log in
 
Team:OUC-China/Project/Supplementary
From 2012.igem.org
Home
Project
Modeling
Human Practice
Team
Parts
Safety
Note
Project Overview
HumanPractice Overview
JudgingForm
Contact Us
Project Overview
Sensor
Decision-making Device
Gas vesicle
ODEModel
Parameter Sensitivity Analysis
Parameter Sweep
Noise Analysis
HumanPractice Overview
Meeting and Academic Communication
Camps, Class and Lectures
Special HP
Team Members
Instructors
Acknowledgement and Cooperation
Lab
Parts
Safety
Labnote
Modeling Note
ProtocolS
meet
sensor
decision-making
gas vesicle
Supplementary materials for comparator and ratio sensor
——Please follow our chase for the most brilliant star of sRNA!
Brainstorming is going on and on throughout the whole summer vacation. We have considered quite a lot mechanisms involved in RNA-RNA interactions.
Firstly, we focused a lot on cis-acting sRNA because they have completely complementary pairing sites. We speculated that they could bind strongly together. Besides, completely complementary binding might facilitate our calculation for reaction rates(k), which is likely to serve as our key factor for predicting RNA interaction.
However, something in our minds changed when we coincided with Hfq which showed that this RNA chaperon can play an extremely important role in RNA interactions. And we began to target moreinformation about trans-encoded RNAs as our candidates for comparators and ratio sensor to neglect the impacts exerted by Hfq and the results suggest that our persistence is correct.
When referring to our ratio sensor design, many ideas died while seldom of them survived. RNase-coupled degradation was once the most brilliant one we regarded as. However, it seems that RNase tends to destroy the key pairing sites, rather than the whole structure. So we immediately gave up that idea and focus more on inert structure forming and orthogonal design. In fact, both of them concentrated our knowledge for sRNA-mRNA interactions. And fortunately we do finish our experimental design two weeks before.
Our modelers always give us interesting ideas for designs which enable us to adjust our direction. It is the mutual cooperation that yields today’s achievement!
Fig. Microarray Analysis of Genes Modulated at Least 2-Fold following Spot 42 Overexpression across Three Independent Experiments[1] The right end indicated the ratio of mRNA levels for pBRplac and pSpot42 samples(experimental control). Values reflect the average and standard deviation from three independent experiments. Positive and negative values reflect induction and repression, respectively. We have investigated all those genes in E.coli genome. According to the value of repression rate and corresponding error magnitude, we have chosen three of them, nanC, srlA, ytfJ to check our designs. SrlA is our first choice due to its highest repression rate. The second choice is ytfJ because its relatively higher repression rate and lower error magnitude. However, we could not find out the transcription start site(TSS) of ytfJ, the only one we didn’t operate PCR from its corresponding TSS. We chose nanC for its 47-fold repression when experimented in LacZ-fusion assays.
FigPredicted Base-Pairing Interactions between the Single-Stranded Regions of Spot 42 and Target mRNAs[1]
(A) Secondary structure of Spot 42, reported previously[2]. The three single-stranded regions(highlighted in gray). Three consecutive nucleotides (white) in each single-stranded region were mutated to disrupt predicted base-pairing interactions with target mRNAs (I–III).It speculates that spot42 may pair with different targets through three seed regions and their mechanisms may be different as well.
(B) Northern blot analysis of NM525 Dspf::kanR (GSO433) cells transformed with pBRplac, pSpot42, or pSpot42 variants I–III. Cells were grown in LB to an OD600 of 0.3, treated with 1mMIPTG, and incubated for 0 min or 30 min. 5S RNA served as a loading control.  It provided that spot42I-III, introduced with different bases mutation, won’t lead to altered degradation rate.
(C) Genes identified by microarray analysis and base-pairing interactions with Spot 42 predicted by the folding algorithm NUPACK. Mutations introduced into pSpot42 and the lacZ translational fusions are designated. The bar below each target gene designates the predicted location of base pairing with Spot 42. The number above each promoter, indicated as a dark arrow, specifies the number of nucleotides between a transcriptional start site and the start codon of the first gene in the operon.[1]  Figure C suggests that the locations of complementary pairing site in transcripts of many spot42 targets are quite different. Given that the complexity of sRNA-mRNA interactions, we tried our best to protect the original secondary structure from alteration or destroy. Moreover, both transcripts of nanC and srlA are picked out from their transcription start site(+1) by PCR to reduce the probability that RNA pairs are out of action after assembling.
Fig. Mutational Analysis of Base-Pairing Interactions between Spot 42 and Target mRNAs[1]
Results for b-galactosidase assays for lacZ translational fusions with compensatory mutations in the predicted location of base pairing with Spot 42. Derivatives of PM1205 Dspf::kanR transformed with the indicated plasmid were grown in LB to an OD600 of 0.1 and treated with 0.2% arabinose or 0.2% arabinose and 1 mM IPTG for 1 hr before being subjected to b-galactosidase assays. The mutations in pSpot42 variant I–III correspond to the indicated mutations in Figure A. The reported averages and standard deviations are from measurements of cultures from three separate colonies[1].
The assays showed that nanC::lacZ fusion can ben perfectly silenced by spt42 as high as 47-fold repression rate. And it also indicated that mutation in the first location (5’-3’) for predicted complementary pairing can strongly alter the interaction which tells us this site is crucial for interaction between spot42 and nanC. Other sites seem negligible to the interaction. SrlA, though no longer the highest repressing target, still showcased its trapping capacity for being inhibited by spot42.
Figure B showed that the complemetary mutation in nanC can offset the corresponding mutation in spot42, which displays great potential for our orthogonal ratio sensor design .
The location of complementary pairing sites for further understanding the sRNA-mRNA reaciton mechanisms
Visit Freiburg intaRNA
Visit our orthogonal ratio sensor design
Experimental flow chart
Referrence
1. Beisel, C.L. and G. Storz, The base-pairing RNA spot 42 participates in a multioutput feedforward loop to help enact catabolite repression in Escherichia coli. Mol Cell, 2011. 41(3): p. 286-97.
2. Moller, T., Spot 42 RNA mediates discoordinate expression of the E. coli galactose operon. Genes & Development, 2002. 16(13): p. 1696-1706.
 
Retrieved from " http://2012.igem.org/Team:OUC-China/Project/Supplementary "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
