<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_OUC-China_Project_GVP_DesignandResult"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:OUC-China/Project/GVP/DesignandResult">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:OUC-China/Project/GVP/DesignandResult&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:OUC-China/Project/GVP/DesignandResult&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:OUC-China/Project/GVP/DesignandResult&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:OUC-China/Project/GVP/DesignandResult" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:OUC-China/Project/GVP/DesignandResult</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV id="menu-wrapper"><UL class="menu"><LI><A href="https://2012.igem.org/Team:OUC-China">Index</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Project/Overview">Project</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Modeling/Overview">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/HumanPractice/Overview">Human Practice</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Team/members">Team</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Parts">Parts</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Safety">Safety</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/labnote">Note</A></LI></UL></DIV><DIV id="submenu-wrapper"><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China">
					Home
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China#hmo">
					HumanPractice index
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Judgingform">
					JudgingForm
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/contactus">
Contact Us
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/Project/Overview">
				Project Overview
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Project/Sensor/AbstractandBackground">
				Sensor
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Project/DesignMaking/Abstract">
					Decision-making Device
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Project/GVP/GasandBackground">
				Gas vesicle
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/Modeling/ODEModel">
				ODEModel
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Modeling/ParameterSensitivityAnalysis">
				Parameter Sensitivity Analysis
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Modeling/ParameterSweep">
				Parameter Sweep
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Modeling/NoiseAnalysis">
					Noise Analysis
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/HumanPractice/Overview">
				HumanPractice Overview
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/HumanPractice/Communication">
			Meeting and Academic Communication
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/HumanPractice/Camps">
					Camps, Class and Lectures
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/HumanPractice/SpecialHM">
				Special HP
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/Team/members">
					Team Members
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Team/instructor">
					Instructors
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Team/AcknowledgementandCooperation">
Acknowledgement&amp;Cooperation
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/Team/Lab">
				Lab
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/Parts">
				Parts
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/Safety">
				Safety
				</A></LI></UL><UL class="submenu"><LI><A href="https://2012.igem.org/Team:OUC-China/labnote">
				Labnote
				</A></LI><LI><A href="https://skydrive.live.com/redir?resid=6A13E80E2A176A23!249&amp;authkey=!AGsNOi9excznIt8">
			Modeling Note
				</A></LI><LI><A href="https://2012.igem.org/Team:OUC-China/protocol">
					Protocols
				</A></LI></UL></DIV><DIV id="page"><DIV id="slidenav"><DIV id="slidenav-bar"><UL id="social" class="group"><LI class="meet"><A href="https://2012.igem.org/Team:OUC-China"><SPAN>Back</SPAN></A></LI><LI class="sensor"><A href="https://2012.igem.org/Team:OUC-China/Project/GVP/GasandBackground">Background</A></LI><LI class="decision-making"><A href="https://2012.igem.org/Team:OUC-China/Project/GVP/DesignandResult">Design&amp;result</A></LI><LI class="gas-vesicle"><A href="https://2012.igem.org/Team:OUC-China/Project/GVP/DiscussionandFutureWork">Disscusion</A></LI></UL></DIV></DIV><DIV class="article-list"><DIV class="list-box"><DIV class="list-text"><H1 style="border-bottom:none;">Design and Result</H1><H2>Cloning strategy</H2><P><STRONG>1. Gene cluster</STRONG>For our <I>gvp</I> gene cluster we use the specificity primer form Walsby 2002 and add the necessary restriction sites that annealed these pieces of DNA have EcoRI and SpeI sticky ends. We use the vector psb4K5 because the uncoordinated expression of the entire <I>gvp</I>A/C cluster from its own constitutive promoter when carried on a high-copy number vector could result in the accumulation of lethal quantities of gas vesicle proteins in the host cells.Hence we use the low-copy number vector to ensure that the gene cluster can in the host to the normal expression.[3] so we use The vector pSB4K5 containing <I>gvp</I> gene cluster was cut with EcoRI and PstI and was ligated with B0015. ( Figure 4 5) After PCR inspection by using Specific primer we get the target fragment. ( Figure7)
On the other hand, In order to prove that the integrity of the <I>gvp</I> gene cluster we use a set of genetic circuit controlled by the methods OMPR promoter reverse connection in the back of the K73710 using another light controlled element in laser under excitation of start OMPR promoter that the expression of gene cluster is restrained to prove <I>gvp</I> gene cluster integrity. ( Figure 23)
</P><P style="text-align:center; font-size:90%;">Figure 4 This 1 kbp part contains 2 open reading frames coding for gas vesicle genes (<I>gvp</I>A and <I>gvp</I>-20) and the <I>Cyanobacteria</I> gene promoter and RBS (BBa_K737019 and BBa_K737020) from <I>Bacillus Planktothrix rubescens</I> strain BC-Pla 9402. Promotion of the sequence results in expression of gas vesicles, organelles made entirely out of protein. These organelles contain gas and therefore provide buoyancy to the cell. 
</P><P style="text-align:center; font-size:90%;">Figure 5 and 6The cloning strategy of buoyancy device and light control device. Because the uncoordinated expression of the entire <I>gvp</I>A/C cluster with its own constitutive promoter could result in the accumulation of lethal quantities of gas vesicle proteins when carried on a high-copy number vector in the host cells. 
</P><P style="margin-left:360px; margin-top:-280px;">Use the low-copy number vector pSB4A to ensure that the gene cluster can in the normal expression in the host
The light control gene circuit from our iGEM lab is based on UC-iGEM 2005.we use part BBa_ I15008 (ho1) and BBa_I15009 and Cph1 / EnvZ FP redesigned photosensitive gene line, fixed iGEM part library of the BBa_I150010 Cph1 / EnvZ FP mistakes, successful synthesis of 660 nm red light sensitive gene expression device, which in the red light input conditions closed gas vesicle expression. (Figure7)
</P><P style="font-size:90%; margin-top:50px;">(Figure7) the 660 nm laser induced device
It is made up of three 660 nm wavelength of
laser device and a cooling fan composition can
be fixed a root tube that nutrient solution to
get full sunshine has reached laser induced effect
                          ( OUC-China team)</P><P><STRONG>2.<I>gvp</I>A and <I>gvp</I>C genes</STRONG>It’s out second step to take out the <I>gvp</I>A and <I>gvp</I>C gene from <I>Planktothrix rubescens</I> 9402 strain (CCAP 1460/9). Since there is no sequence information in NCBI about Pla 9402, we decided to take advantages of its translocation mechanism. So we use the Pla 9401 (GenBank: AJ132354.1) and Pla 9402/b (GenBank: AJ494991.1) gene sequence as the template to design primers for <I>gvp</I>A, <I>gvp</I>C-20psi and <I>gvp</I>C-16omega. In consideration of Pla 9402’s repetitive sequences of gvp gene, we took two different approaches to take out the genes: one is directly doing PCR with the genome of Pla 9402 as template, while the other is PCRed on the BssSI-enzymed template of Pla 9402 genome. The gene <I>gvp</I>A and <I>gvp</I>C-20psi was successfully cloned through both these pathways, but the attempts to clone <I>gvp</I>C-16omega gene failed. 
</P><P><STRONG>3.Synthetic <I>gvpA/C</I> gene cluster</STRONG>In order to realize the gas vesicle as the decision making output actuator， we clone the gvp structure gene gvpA and gvpC from the natural gvp gene cluster and ligate the promoter（<A href="http://partsregistry.org/Part:BBa_J23106">BBa-J23106</A>）RBS（<A href="http://partsregistry.org/Part:BBa_B0034">BBa-B0034</A>）and terminator（<A href="http://partsregistry.org/Part:BBa_B0015">BBa-B0015</A>）of the E.coli expression system. Because we use the standardized promoter and RBS in this part, so this part can be used in more expression system and control its expression by ourselves as the decision making output actuator. It may extend the applicability of gvp floating device.
</P><P style="font-size:90%; text-align:center;">Fig 7: We want to recombine the gvpA/C cluster and control its expression by ourselves to realize the decision making output.</P><P style="font-size:90%; text-align:center;">Fig 8：the expression situation of th e Gas vesicle synthetic gene cluster in escherichia coli JM109,from the picture, we can see obviously gas vesicle due to the expression of bacteria produce clear separation phenomenon.</P><H2>Floating <I>E.Coli</I></H2><P>	We transformed 1Kbp <I>gvp</I> gene cluster into JM109 and stationary culture for 36 hours. You can find the E.Coli floating on the surface! (Figure 10 and 11)
The expression situation of the Gas Vesicle gene cluster in Escherichia coli JM109, from the picture, we can see obviously gas vesicle due to the expression of bacteria produce clear separation phenomenon. The picture on the right of the experimental group were liquid surface layer and intermediate layer of fluid sampling results, we can see that the fluid layer surface bacteria volume density is obviously higher than that of the middle layer of fluid.
</P><P style="text-align:center; font-size:90%;">Figure 10 and 11. The expression situation of the Gas vesicle gene cluster in <I>Escherichia coli</I> JM109, from the picture, we can see obviously gas vesicle due to the expression of bacteria produce clear separation phenomenon. The picture on the right of the experimental group were liquid surface layer and intermediate layer of fluid sampling results, we can see that the fluid layer surface bacteria volume density is obviously higher than that of the middle layer of fluid.
</P><P>	The buoyancy of Gvp was tested by using the buoyancy test protocol. The cells were grown in a test tube with LB medium for 36 hours (Figure 10) and were left for a while before detected in order to give the cells time to sink or float.
	Trace stratified take liquid apparatus. It is made up of a 10 ml volumetric syringe and five a 1 ml volumetric syringe composition. It can be used as a culture cell containers and can be performed every layer of the synchronous sampling the aim is to avoid the sampling process caused by vibration caused by fluid layer surface thallus under. The whole device can make quantitative analysis more accurate.
</P><P style="text-align:center; font-size:90%;">Figure 12. Trace stratified take liquid apparatus. It is made up of a 10 ml volumetric syringe and five a 1 ml volumetric syringe composition. It can be used as a culture cell containers and can be performed every layer of the synchronous sampling the aim is to avoid the sampling process caused by vibration caused by fluid layer surface thallus under. The whole device can make quantitative analysis more accurate. (OUC-China team)
</P><P>	Because when the bacteria die, it may float on the surface. There is a question, do the bacteria floating on the surface die or not  So we made a microscopic examination. We use a vital stain to examine the bacteria die or not. Dead bacteria can’t decompose the stain, so they appear blue, while the living cells turn to be transparent (Figure 11 and 12).
</P><P style="text-align:center; font-size:90%;">Figure 13 and 14.  The left picture shows that most of surface bacteria are living because there are few blue spot here. The right picture is dead bacteria as control. We can see that, almost all the bacteria are dead.
</P><P style="text-align:center; font-size:90%;">Figure 15 and 16.  Under the microscope, we use the blood count method to observe the different liquid layer cell capacity, the left is from the liquid layer on the surface of the samples taken from the experimental group, and the right is middle layer of fluid samples. Through the under the microscope observation can obviously observed as the cells expressed in the gas vesicle that cells in the culture solution of concentrated in the surface layer of fluid.
</P><P style="text-align:center; font-size:90%;">Figure 17 and 18. Under the microscope we use the blood count method to observe the different liquid layer cell capacity,  the cell from the bottom of the control samples taken from control group（left）the surface layer of fluid samples.(right) Because the control group cells without expression of Gas vesicle that the force of gravity has a large number of somatic settlement in the test tube through the bottom under the microscope observation we can clear observation the samples of bottom and the difference between the surface sample.</P><P>	According to the different layer accurately sampling, the samples blood count determine different height body effusion layer cell number so as to reflect the situation of gas vesicle expression. The results were best visible with and cell density of OD600=2.382 in Liquid surface. And we find that in the test tube, in addition to liquid layer surface and bottom cell is dense in the rest of the liquid layer densities and roughly the same and surface bacteria volume density difference of an order of magnitude. From the control group of counting the results to the distribution of cells with normal cells in the test tube, the distribution so can also explain the experimental results is due to gas vesicle expression in cells. We also do each liquid layer absorbance inspection results and cell number experimental results high similarity from the side proved the correctness of blood count experiment.
</P><P style="text-align:center; font-size:90%;">Figure 19. The picture shows the gas vesicle in the cells after expressed in the overall trend. We in the cultivation of the cells into logarithmic phase after transferring them to test tube, quiet place and culture 36 hours after using Trace stratified take liquid apparatus at the same time to each liquid layer for sampling. Then each layer of fluid samples of blood count the number of cells with statistics. Results show that gas vesicle due to the existence of the cell concentration in the test tube surface layer of fluid, and control group due to the action of gravity that most cells sedimentation in the test tube bottom.
</P><P style="text-align:center; font-size:90%;">Figure 20. Graph of OD measurements at 600nm of both experience group and control group. From the experimental results can be seen in the liquid surface can see cell number have significant abrupt transformation.
</P><P style="margin-left:400px; margin-top:-100px; font-size:90%">Figure21. Fitting function that function relation is y=25300x4-4726000x3+30515000x2-7.7444000x+70902000. In the picture X axis representative distance test tube bottom every centimeter liquid column and ordinate for cell number.</P><P>Through experiment data f and significant t test. We draw conclusion that in the experimental group and control group on the surface of the liquid layer cell count there exist significant differences this difference is due to the experimental group cells express gas vesicle.
</P><P style="margin-left:140px; margin-top:-310px; width:600px;">Besides detecting OD/600, we use blood counting method to get cell number in different water-courses. With the help of MATLAB analysis，we can get fitting function of cell number in different water-courses.
Fitting function that function relation is y=25300x4-4726000x3+30515000x2-7.7444000x+70902000. In the picture X axis representative distance test tube bottom every centimeter liquid column and ordinate for cell number.
After getting the plasmids of constitutive GvpA and GFP generator, constitutive GvpC and GFP generator, we transform the two plasmids into E.Coli to see if they can float. As expected, they float! It shows that the <I>gvp</I>A and <I>gvp</I>C genes work. In other word, only two genes (about 700bp) can make bacteria floating.
</P><P style="font-size:90%; margin-left:10px;"> Figure 22.Floating bacteria with double plasmids.</P><H2>Gas vesicle protein and Expressive inspection</H2><P>We cloned the <I>gvp</I> gene cluster and <I>gvp</I> coding gene form <I>Planktothrix rubescens</I>, which can make the protein GvpA and GvpC-20.The two types of gas vesicle structure protein can be assembled into an empty bubble structure of the Protein polymers which can be in the electron microscope observation under.
After successfully transform the BBa_K737010 into the Expression strains JM109, the experiment results show that gas vesicle gene can be expressed in <I>E.coli</I> when gas vesicle gene expression in cells it can make floating bacteria in nutrient solution layer surface. The results showed that some of our speculations include that the <I>gvp</I> genes in prokaryotes of generality.
</P><P style="font-size:90%; width:200px; margin-left:20px; margin-top:-10px;">Figure 23. Gas vesicles in a freeze-fractured cell of Prosthecomicrobium
Pneumaticum showing the straight –sided cone (Micrograph by
D. Branton and A.E. Walsby) Magnification 210000x)
</P><P style="font-size:90%; width:200px; margin-left:250px; margin-top:-160px;">Figure 24.Gas vesicles in E.coli protoplasts (BBa_K190033).
The cells were treated with Lysozyme and SDS to
create the protoplasts, uranyl acetate was used for
staining. Magnification: 11500x.(2009 Groningen iGEM )
</P><P style="font-size:90%; margin-top:-150px; width:250px; margin-left:470px;">Figure 25.The <I>gvp</I> gene cluster in 1% agarose gel form 1 to 4
1 and 2 representative gvp gene cluster + B0015 + R0082 + B0034
3 and 4 gvp representing gene cluster + B0015 and gvp gene cluster
</P><P>To check whether gas vesicles really were present in the cells we did some electron microscopy. In Figure 28 a picture of gas vesicles in a protoplast can be seen. This protoplast comes from an <I>E. coli</I> cell that contained a plasmid with the <I>gvp</I> gene cluster （<A href="http://partsregistry.org/Part:BBa_K737010">BBa_K737010</A>）. Figure 29 a picture of blank plasmid in the <I>E. coli</I> JM109 in electron microscope.
</P><P style="font-size:90%; text-align:center;">Figure 26. Gas vesicles in <I>E. coli</I> protoplasts (<A href="http://partsregistry.org/Part:BBa_K737010">BBa_K737010</A>). The cells were treated with Lysozyme and SDS to create the protoplasts, uranyl acetate was used for staining. Magnification: 60k x.</P><P style="font-size:90%; text-align:center;">Figure 27. A blank plasmid in the <I>E. coli</I> JM109 The cells were treated with Lysozyme and SDS to create the protoplasts, uranyl acetate was used for staining. Magnification: x 60k</P><H2>Gas vesicle protein quantitative analysis</H2><P>In order to quantitative analyze the expression of gvp gene inside the cell, we design a genetic circuit that could be realized by inserting the <I>gvp</I> gene alongside GFP as a polycistron[5]. Using this part, we can know the expression level of Gvp protein by the detecting the fluorescence of GFP.It will be transcribed into an mRNA and the two genes each have a same promoter (BBa_ J23106). So we can through the expression of GFP to detect gvp gene expression of roughly quantitative. GFP expression can through the determination expression of the specific excitation wavelength of fluorescence value to calculate. [5][6]
We made 2 polycistron circuits of gvpA/C and gfp genes. 
Our devices are:
Constitutive GvpA and GFP generator (BBa_K737006):
Promoter (BBa_J23106) + RBS (BBa_B0034) + gvpA + RBS + GFP + terminator (BBa_E0840)
Constitutive GvpC and GFP generator (BBa-_K737007):
Promoter (BBa_J23106) + RBS (BBa_B0034) + gvpC + RBS + GFP + terminator (BBa_E0840)
</P><P>In order to check out the expression of polycistronic genetic circuit .We use the SDS-PAGE（protocol）to get the intracellular total protein. Here is the result of SDS-polyacrylamide gel electrophoresis. (Figure22) We can see GFP and GvpC protein directly. Lane 4 is top10 control group, lane 3 is constitutive GvpA and GFP generator, lane 5 is constitutive GvpC and GFP generator. The upper blue and red boxes are in the position about 26KD are GFP protein (25.8KD). The lower red box in the position of about 20KD is GvpC-20 (20KD). Because the MW(molecular weight) of GvpA protein is too small, it’s very difficult to see it on the SDS-polyacrylamide gel. But the expression of GFP suggests it may be expressed.
Some papers say that GvpA is difficult to see is that GvpA is too hydrophobic to run out of sample hole. Because the gas vesicle is multimodal protein so it cannot isolate by using SDS-PAGE, too. (Figure28)
Finally, we consider that GVP as exogenous protein may be in escherichia coli form inclusion body so we through the Wilkinson–Harrison solubility model to predict GVP protein solubility and judgment GVP protein formation the possibility of inclusion body.
The revised Wilkinson–Harrison solubility model involves calculating[9]
a canonical variable (CV) or composite parameter for the protein for which
the solubility is being predicted. The canonical variable in the two parameter model is defined as:
</P><P>where: n 4 number of amino acids in protein
N,G,P,S 4 number of Asn, Gly, Pro, or Ser residues, respectively.
R,K,D,E 4 number of Arg, Lys, Asp, or Glu residues, respectively.
λ1 andλ2 coefficients (15.43 and 29.56, respectively)
The probability of the protein solubility is based on the parameter CV-CV8,
where CV8 is the discriminant equal to 1.71. If CV-CV8 is positive, the
protein is predicted to be insoluble, while if CV-CV8 is negative, the
protein is predicted to be soluble. The probability of solubility or insolubility can be predicted from the following equation:
Probability of solubility or insolubility =
</P><P>The Calculation results show that the gas vesicle protein is insoluble protein.it may be form the 
inclusion body in the escherichia coli expression system.( Table 2)
</P><P style="margin-left:20px; margin-top:20px; width:700px; font-size:90%;">Figure 28.the result of 10% SDS-polyacrylamide gel electrophoresis.
the expression of protein gas vesicle in E. coli JM109 analyzed by
SDS-PAGE and quantitative measurement. Equal amount of sample
from cell lysate was fractioned into the  pellet and subjected
to SDS-PAGE. 1 and 2 bands in pellet represent respectively the
JM109 blank control and import GVP gene cluster strains.
</P><P style="text-align:center; font-size:90%;">Figure 29.The result of SDS-polyacrylamide gel electrophoresis. Lane 4 is top10 control group, lane 3 is constitutive GvpA and GFP generator, lane 5 is constitutive GvpC and GFP generator. The upper blue and red boxes are in the position about 26kDa are GFP protein (25.8kDa). The lower red box in the position of about 20kDa is GvpC-20 (20kDa).
</P><P>Because the GvpA protein can’t be seen on the SDS-polyacrylamide gel electrophoresis, we wanted to use GFP to detect the expression of GvpA. We use a polycistron of gvpA (gvpC) and gfp to detect the RFU. We can see that, the RFU of Gvp and GFP Generator is about 2000, and the RFU of Gvp and GFP Generator is about 4000, while top 10 is about 0 as control. (Figure26)First, the RFU of these parts are rather high. Second, the E.Coli with these two plasmids can float. Third, we can see GvpC on the SDS-polyacrylamide gel. So, we can see the two parts can express Gvp Protein and work well.
</P><P style="margin-left:20px; margin-top:10px;margin-bottom:20px; font-size:90%;">Figure 30.We can see that, the RFU of Gvp and GFP Generator is about 2000, and the RFU of Gvp and GFP Generator is about 4000, while top 10 is about 0 as control. 
</P></DIV></DIV><DIV id="ouc-footer"><DIV id="ouc-tools"><UL><LI style="width:160px;"><FORM action="/Special:Search" id="searchform" style=" border-color:#000;"> </FORM></LI></UL></DIV></DIV></DIV></DIV><P id="back-top"><A href="#top">Back to Top</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:OUC-China/Project/GVP/DesignandResult">http://2012.igem.org/Team:OUC-China/Project/GVP/DesignandResult</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:OUC-China/Project/GVP/DesignandResult" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:OUC-China/Project/GVP/DesignandResult" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:OUC-China/Project/GVP/DesignandResult&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:OUC-China/Project/GVP/DesignandResult&amp;oldid=295017" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>