"
Page
Discussion
View source
History
teams
Log in
 
Team:OUC-China/Project/DesignMaking/Background
From 2012.igem.org
Index
Project
Modeling
Human Practice
Team
Parts
Safety
Note
Home
HumanPractice index
JudgingForm
Contact Us
Project Overview
Sensor
Decision-making Device
Gas vesicle
ODEModel
Parameter Sensitivity Analysis
Parameter Sweep
Noise Analysis
HumanPractice Overview
Meeting and Academic Communication
Camps, Class and Lectures
Special HP
Team Members
Instructors
Acknowledgement&Cooperation
Lab
Parts
Safety
Labnote
Modeling Note
Protocols
HOME
Abstract
Background
Goal&Design
Result
Discussion
Background
Precise control of gene expression is at the core of any genetic engineering-dependent discipline.
What is small RNA?
Definition & MechanismOBacteria employ a range of RNA regulators collectively termed small RNAs (sRNAs) to help respond to changes in the environment.They have important functions as genetic regulators in prokaryotes. sRNAs act post-transcriptionally through complementary pairing with target mRNAs to regulate protein expression.The subsequent effects of trans-ecoded  complementary pairing on 5’ UTR include repression(through blocking RBS like spot42-galk::mRNA) and activation(through unlocking inert secondary structures near RBS like dsrA-rpos::mRNA).
Fig.  Small RNA regulation mechanisms.
Modular achitecture Almost all the trans-encoded small RNAs consist of repression stem-loops,a RNA chaperone binding site and a endogenous terminator(unclear termination efficiency),even a few RNase recognition sites on vital functional domain.
The repression stem-loop initializes complementary pairing through several nucleotides motif called seed region,which has been proven crucial in small RNA regulation.
RNase recognition site is conincident with RNA chaperone binding site in some cases,thus the chaperone Hfq protects small RNA from degradation.
RNA chaperoneRNA chaperone,that is the hexameric Escherichia coli Hfq,is involved in riboregulation of target mRNAs by  many small trans-encoded RNAs.Hfq is a highly abundant protein 20000~40000 copies,always plays a significant role in sRNA regulation by binding to RNA.It binds to sRNA and mRNA respectively, then hfq-RNA complexes binds to each other,increasing the local concentration for pairing,accelerating strand-exchange,altering RNA structures for initiation,protect sRNA from degradation and exposure of seed region.Hfq molecular titration experiments have convinced us that the essential status of hfq in sRNA regulation
Fig. Variable requirement for Hfq in sRNA-mediated gene regulations in bacteria.
How is current study going on?
Unfortunately, analogous technology for use in bacteria that is as reliable and efficient as mammalian RNAi has remained elusive due to its complicated mechanism,but many more indepth researches have shed some light on rational designation of bacterial small RNA.
Approaches artificial sRNA in synbioHowever, active artificial sRNAs reperssion stem or orthogonal seed region can be screened from error-prone libraries,in which natural sRNA scaffolds are fused to a randomized antisense domain.
Fig. Heat map of percentage repression profile of 23 RNA-IN mutants in presence of 23 antisense RNA-OUT mutants
What’s in sRNA topology? On the other hand,ongoing characterization of base pairing sRNAs in bacteria has started for several years to reveal the modular architecture and how these sRNAs participate in global regulatory networks. These networks can be broken down into smaller regulatory circuits that have characteristic behaviors and functions.
Fig. Overview of regulatory circuits in which sRNAs are found in E. coli and Salmonella.
Quantitative simulation Now, the precise control of sRNA has been quantitatively understood preliminaryily.A  threshold-linear response with a tunable threshold, a robust noise resistance characteristic, and a built-in capability for hierarchical cross-talk, the outstanding nature of riboregulator makes it more popular in synthetic biology.
Fig. (A)The strength of sRNA repression decreases as the target transcription increases.
(B)Steady-state solution of a model for protein regulators, where the strength of repression is independent of target transcription rate.
(C)Temporal behavior in a single stochastic simulation of the expression of two model genes, regulated by sRNA(blue line) and protein regulators(red) respectively.Parameter set is given in reference
Why choose small RNA as information processing medium?
Advantage of Ribo/Protein regulators:
What is spot42?
spot42 is a multitarget small RNA that mediates the discoordinate expression of the E.coli galactose operon and other sugar metabolic pathway,such as galK,nanC,ytfJ,srlA,which are the 5’ leader sequence of the corresponding metabolic enzymes.
Take galK for example,spot42 causes translation repression by base pairing to RBS in the 5’ leader sequence,then block the recognition of the antiSD sequence on 30S subunit. It’s a very strong pairing(up to 20bases),but it only causes 2.6 fold repression according to Johannes H. Urban and Jo¨ rg Vogel,for it doesn’t result in the degradation of the RNA complex by RNaseE(ssRNA degradation) and RNaseIII(dsRNA degradation),both of them are major enzymes that causes RNA degradation in vivo.spot42-galK complex degrades in a slow and presently unclear way. The base pairing is initiated by the recognition of seed region,which plays a significant role in the repression efficiency
Fig. Basepairing between spot42 and galK
Hfq binds to the AU-rich domain of spot42,and footprinting experiments has shown that spot42.It is essential to most of small RNAs in E.coli,extremely enchances the repression efficiency.
Fig. Repression efficiency of many known sRNA in WT,RNase deficient strain,RNA chaperone deficient strain
spot42 have a weak Hfq binding site (whereas galK weaker),an endogenous terminator(unclear efficiency,maybe weak),and the multitarget repression stem-loop.
Fig. spot42 modular structure. 5’→3’:Repression stemloop,  Hfq binding stem,terminator stem.
How about dsrA?
dsrA is an 87-nt untranslated RNA that regulates both the global transcriptional silencer and nucleoid protein H-NS and the stationary phase and stress response sigma factor rpos (ss). Both dsrA and rpos have strong Hfq binding site.
Fig. Models for dsrA riboregulation.
At the top, dsrA forms RNA:RNA interactions with target transcripts.
On the left (-) is a model for translational repression of hns.
On the right (+) is a model for translational activation of rpos
 
Back to Top
Retrieved from " http://2012.igem.org/Team:OUC-China/Project/DesignMaking/Background "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
