Team:FAU Erlangen/Safety
iGEM Erlangen
Safety
"When there’s a huge solar energy spill, it’s just called a ‘nice day’."
votesolar.org
☰Menu
Project
Inspiration
Biofilm
Grätzel Cell
Parts
References
Results
Growing Biofilms
Binding of Heavy Metals
ZnO Mineralization
Solar cell results
References
Notebook
Biology
Chemistry/Physics
References
Human Practices
School laboratory
Science Day
Collaborations
iGEM Team Aachen
iGEM Team Munich
iGEM Team Marburg
Safety
Killswitch
Binding of Heavy Metals
References
Achievements
Team
Killswitch
Binding of Heavy Metals
References
The perfect killswitch
...does not exist. Let us elaborate why.
Any killswitch must contain a sequence that is capable of destroying the cell carrying it. This is, per definition, a selective disadvantage and therefore prone to either being lost entirely, or else being silenced by mutation. The question now is “how probable is that?”
Without going too much into detail, the average mutation rate in Escherichia coli (genome size around 4.5 Mbp) is   1 x 10-3 base pairs per genome replication and while some base substitutions are more likely to happen than others, this does not matter much in the large scale of things. One mutation every one thousand replicated genomes may not seem like much, but it does mean that, in theory, the probability of a given base mutating in at least one cell as calculated by 1-(1- Pmutation / Genome size)2n exceeds 0.95 after n=34 divisions – under ideal conditions this takes less than a day.
Figure 1: Left: total predicted number of mutations in a growing culture; right: probability of one specific mutation occuring in at least one cell in the culture.
So how probable is it that a single mutation will kill the killswitch?
Mutations come in different flavors: single-base substitutions that can result in silent mutations with no effect on the amino acid sequence, missense mutations that lead to a single substitution on amino acid level and nonsense mutations resulting in a stop codon, thus terminating translation and usually yielding no functional protein. Furthermore, there are single-base insertions or deletions, which result in a so-called frame-shift that leads to a completely different amino acid sequence – usually gibberish – being translated.
For the sake of simplicity, we will now focus on nonsense mutations. For each codon there are nine possible mutations, three for each base. With 61 amino acid encoding base triplets that means 549 possible mutations, 21 (or 3.8%) of which create a stop codon. Now, let us look at the first ten amino acids of the killswitch’s core protein. Out of 61 possible codons, 18 can be turned into a stop codon by a single-base substitution, which is about a one in three chance. With ten codons, we now have a chance of less than 1% for not including a one-hit kill spot in the sequence. This spot will likely be mutated by the end of the day, resulting in a useless peptide and a very happy cell (because it gets to live).
Table 1: Codons that can be converted to a stop codon with a single-base substitution.
Of course, these issues can be alleviated by e.g. codon optimization. There are methods of controlling and restraining cell growth and the fact that most of us are not dying of cancer right now is solid evidence that this is true. However, to quote Dr. Ian Malcolm from “Jurassic Park”:
If there is one thing the history of evolution has taught us it's that life will not be contained.
The fact that some people are dying of cancer right now is solid, if sad, evidence that this is also true. This leaves us with exactly two options when considering the applications of genetically engineered organisms in the environment:
Live with the fact that we cannot control the cells, especially simple, fast-dividing organisms such as bacteria, or
Make sure they never leave the lab alive.
We chose the second option.
Kill it with fire
Of course, killing the bacteria was a challenge since the cells are an integral part of the structural stability of our solar cell. Fortunately, our chemists had a neat little trick called “sintering”, i.e. heating to several hundred degrees Celsius for a few minutes. The chemists do this to partially melt metal compounds and improve the cohesion of nanoparticles. Since we were quite sure that doing this would not endanger the functionality of our solar cell, sintering quickly became our favorite technique to sterilize our functionalized biofilms. We also found that it had the pleasant side effect of improving the efficiency of our solar cells. To determine whether this killed all bacteria on the glass surface, we performed a quick experiment with two microscope slides that we streaked some of the E. coli suspension on. One was heated for two minutes with a Bunsen burner, the other was not. Both slides were touched down on an agar plate with the bacteria-coated surface pointing towards the agar. Unsurprisingly, no cells grew from the heated sample, while the control showed a lot of healthy bacteria (Figure 2).
Figure 2: Lethality test. Left, no heating; right, heated 2 Minutes with a Bunsen burner. "+" streaked from liquid culture as positive control. Cells plated were NEB-5-alpha E. coli containing pQE-9 on LB agar with ampicillin (100 µg/ml).
We have therefore implemented a sure-fire method of sterilizing the final product into the fabrication process of our solar cells.
Binding of Heavy Metals
Due to the metal binding domains on the curli fibers, the biofilms are able to bind not only zinc oxide, but also heavy metals. This might present a safety application since biofilms could scavenge toxic heavy metal ions out of wastewater.  Copper, nickel, gold and cadmium were tested. Two different methods have been applied. While the first two were detected by a decrease in absorption of UV and visible light, gold and cadmium nanoparticles were synthesized directly on the biofilm. A color change indicates a successful binding event. For further information, see results.
References
Lee, H., Popodi, E., Tang, H., & Foster, P. L. (2012). Rate and molecular spectrum of spontaneous mutations in the bacterium Escherichia coli as determined by whole-genome sequencing. Proceedings of the National Academy of Sciences, 109(41), E2774-E2783. doi: 10.1073/pnas.1210309109
< ▲
Contact: Department Biology Friedrich-Alexander-University Erlangen-Nuremberg Staudtstr. 5 91058 Erlangen igem2016fau@gmail.com
