Team:HebrewU/notebookdates.html
March 1st.
March 2nd.
March 3rd.
March 4th.
March 5th.
March 6th.
Group Meeting in Jerusalem (Ein Karem + Givat Ram)
Recruit students for our project from the faculties of computer enineering and medicine.
Lior B, Lea, Noa and Alon.
March 7th.
Meeting with Tomer Malchi (CultivAid, The Hebrewu University)
Tomer mentioned the  KOW value (hydro/lipo-philic qualities, polarity) which can be used to predict how well molecules will be able to diffuse through root membranes.
Molecules that are between 2-3 can penetrate easily through the membrane while higher or lower values may have trouble:
a. Molecules that are bigger than 500 D will also have trouble passing through the membrane.
b. Molecules that have net charge can't [easily/passively] pass passively through the membrane.
c.  Molecules PKA  effects the charge, and as such soil pH can play a role in determining uptake(causing the plant to affect pH may help us uptake molecules).
Specifically, TCDD has KOW of 4.5 meaning it won't enter a plant in a passively. When we checked the dechlorinated version of the molecule we saw that its KOW is 3 meaning the plant should be able to absorb it [passively].
Moreover, Tomer suggested to check if we can enhance oxygenation in soil and by doing so, we can boost the degradation of different compounds.
When Tomer was asked about different methods to measure how well a plant absorbed/ broke down a certain molecule he said that we the machinery to check how well a chemical is present in soil solution or plant solution he said that we need to check if someone made a protocol for it before. If we can find a protocol, we probably have the machinery needed in the faculty (HPLC chromatography)
Further on Tomer mentioned that he has a special pump that can measure the ratio between the concentration of chemical in plant phloem to the solution that the plant was based in (while using aquaponics).
Furthermore, we understood that in order to build a system to detoxify TCDD in Vietnamese soil we need to pay attention to the soil properties (clay % and organic matter concentration).
Tomer recommended to discuss our idea with Baruch Robin, professor from the agriculture faculty that work in the field of herbicides.
Also, Tomer recommended to discuss our idea with Itzhack Hadar, professor from the agriculture faculty that deals with white rot fungi that can break down different chemicals (Without the need to pass them through a biologic membrane).
When Tomer was asked about the possibility of cleaning the water from the toxin, he recommended to check "Constructed wetlands".
Alon, Amichai, Aliza, Omri, Shai, Eliya
March 8th.
Whole Team Meeting, Discussing the near Future with Assaf and Shai (Mentors).
Meeting summary:
- We couldn’t find the sequnce of TCDD degradation enzymes  that we found in the literture.
- Shai (Metnor) said that we should contact labs that already showed lever cells that degrade any kind of dioxin and see if they can give us the specific enzymes that were part of the degradation.
- Do the same thing as above with the recombinant E.coli (from one of lior`s articles) (when we ask this kind of things we should ask for everything, the Ecoli and all the plasmids with the enzymes and the DNA sequences and start checking how much time and money it would take to express them ourselves) we should ask if they tried TCDD as well.
- We should check references of the article with 3 enzymes in other articles and see what others had to say about it.
- We should summaries all the candidates for degradation, what kind of reactions we know they do and think how we can check the products of TCDD degradation by them. What the intake and out take of the prosses as well (not just the substrate), maybe we can use it to create a test for degradation.
- Maybe looking for agent orange, and check where in the degradation prosses it becomes detoxify and the enzymes relevant for that.
- Send a mail to the beijing team to understand their work.
- Mosquna and shai says that trying to understand the uptake of TCDD to the plant is a big task.
- If we start researching the uptake by plants we should add it to our project even if we weren`t able to combine the degradation and uptake in the same plant.
- Shai expresses his positivity in adding the arabidopsys to the project.
- Mosquna says that when they want to check something that it works the inject agrobacterium to the leaves of tobacco.
- We need to grow the arabidopsis in advance to shorten the time frame of every generation.
- Fungi has a much better uptake then arabidopsis so it might be better.
- We should send mails to all relevant labs.
- Maybe we can search for a place they already have full grown symbiotic mushrooms (talk with tomer again).
- When we read articles, we should care much about concentration.
- If we use the fungi, maybe we can find a relevant assay for the connection between the fungi and plant.
- Assaf (Mentor) said that we should concentrate on the next topics when we read articles:
○ Enzymes.
○ What kind of reactions.
○ What are the products.
○ How can we detect and follow the reaction prosses.
- We might be able to do a research with bacteria to create bacteria that can live on TCDD and mutagenesis. Mosquna sent us an article about the method of finding the bacteria that prosses the T4D. Mosquna says he has a friend that deals with this kind of things so he might ask him for bacteria populations.
- We should check if someone knows about a bacteria that knows to use dioxins as carbon sources.
Everyone.
March 9th.
March 10th.
March 11th.
March 12th.
March 13th.
Library Meeting with Assaf and Shai(Mentors)
- We reviewed all the important dates for lab work.
- We outlined the lab work.
- Mosquna said that we need to check what the enzymes we found need to work (if transferring them from bacteria to plants will work).
- All anaerobic enzymes may not work in aerobic organisms.
- Red team should be divided in the different enzymes. (we might ask the help of our associated students).
Red team should check:
- Do the enzymes work in the context of our organisms?
- Do they need any other enzymes to work?
- Do they work in plants and bacteria as well?
- We should look again for a way to break the products to CO2 and water.
- We need to reconstruct the 4 experimental pathways to start constructing our experiments.
- Regarding the assay, mosquna thinks we should check the formation of our product to test the enzymes.
- We should check how much the Aliza and Hagit proposes cost and where we can get it.
- We should look for NEB sponsorship.
- We should check more thoroughly the use of biobricks and DNA in the competition.
- We should decide what parts we want from igem before we receive the biobricks kit.
- Addgene can send us vectors needed.
- Igem parts might not be reliable enough.
- Mosquna wants to be part of the choosing of vectors.
- Computational biology - use them for a model of the plants work beforehand.
- How and where exactly the enzymes work (coenzymes, cofactors, where in the cell they work, what kind of environment they need).
- We should check what receptors we know for TCDD and its products to create dependency in plant and bacteria. (we can use it as a kill switch and an assay).
- We should write a vision of all the genes we would like to add to complete the TCDD cycle.
- We can't rely on the faculty time table, they will react too slow. Mosquna will give us a banch in his lab.
- We should check if AHR exist in E.coli and Arabidopsis.
- If AHR exist in yeast already, we should order it now.
• We should contact the guy from givat ram and get his E.coli.
- We should look again at all the old articles we have and look if anyone conducted more work later on.
- Mosquna will contact berta and talk about how to conduct orders.
- What regulation exist for working with TCDD.
- We will conduct another meeting in Sunday after the conference in Weizmann.
- We should search for an industrial multigene vector.
Everyone.
March 14th.
March 15th.
March 16th.
March 17th.
March 18th.
Synthetic Biology Conference in Weizmann
The conference included lectures, round table discussion and a showcase of former I-GEM project from Technion, Tel Hai and Beer Sheva teams.
Everyone.
Team Meeting - Funds and Research
We reviewed all the numbers that we got to see if we can guess our budget for research.
- Assaf (Mentor) think we will need about 50,000$ for research alone, including 15,000$ DNA synthesis.
- There is 20kb from IDT - https://2018.igem.org/Sponsors/Special_Offers/IDT.
- Twist might be able to aid us in funding with DNA synthesis or cash.
After reviewing our current information, we concluded that:
1. PR has to do research to understand where TCDD is located and considered a problem to guess which companies and organizations will be contacted for support.
2. Write a scheme of our proposed solution for marketing reasons (just the idea, not much details, focus on problem and end results). Magnify the problem (it`s around the world, show a lot of research that support that, how bad it is for human health and environment). Show a simple figure of how the solution works.
3. PR team should contact Berta and arrange a meeting to regulate the donation searching prosses.
4. We should find a reductase in yeast cell for the cyp metabolic pathway. Plus we should look at people who tried to express this kind of enzymes in general in yeasts and see what they did and what worked best.
5. We should also look how the fungi extract the enzymes.
Organizations that popped up during the meeting:
1. Veterans of Vietnam.
2. Kakal.
3. Bill gates.
4. Jews from the states.
Lab space:
- We have a small lab desk but we have no equipment, Assaf says we will work in his lab for now.
- Plus he says his doctors will be our mentors and help us with the lab work.
Everyone.
March 19th.
March 20th.
March 21st.
March 22nd.
Team Meeting about the Enzymatic Pathways.
We discussed the first enzymatic pathway that we found in the literture.
- For now we concluded that we need a reductase for NADPH to recycle the prosses.
- We can degrade the tetra to tri, there are a few enzymes that we can add to degrade the tri (including other mutations of cyp1a1.
- There is a site called "addgene" that cells plasmids and viruses so we might find what we need there.
- We might try to send the cyp crystallography to Weizmann to analyze it and improve it.
- We should find a candidate with the most structural information and send it to them.
- We should choose one of the cyps that we think that might work for us, find structural information and send it to Weizmann and try to improve its specificity to TCDD and maybe even degrade it further then triCDD.
Asaf presented his idea for experiments:
- Use a yeast vector with a promotor called "gal1-gal10" that can provide for two genes at the same time.
- When we introduce it to the yeast, we can merge different yeasts with different promotors (they merge their genetic information) and so we can get more enzymes in 1 yeast.
- If we could find an assay for the products that is quick and cheap, we might use a total random reactions and collect the best yeasts.
We presented another product of a reaction that is pyrocatechol which is toxic for human. We need to check if we can work with it, it has good assays.We concluded that every assay for potential products or byproducts of the system is worth doing.
We discussed the second enzymatic pathway:
- We concluded that we might get from TCDD to catechol which every plant can degrade.
- It seems that we can use TCDD as a carbon source, which is easy to check.
- Any way we need a reductase for NADH/NADPH (depends on the enzyme).
- Cyp1 might help this enzymatic pathway as a Cl ridder.
We discussed the forth enzymatic pathway:
- With a dioxin without the Cl`s atoms being degraded by two enzymes to catechol.
- There are more enzymes that might be easier to use then the second enzymatic pathway. (we should combine the pathways).
- We should send a mail to the Chinese research team, and maybe even try their sequence ourselves.
We discussed the third enzymatic pathway.
We concluded that all 4 enzymatic pathways are viable solution.
- We need to optimize the sequences we found for yeasts.
- Because we want to use plants in the future, we want to use codons that are good for plants and yeasts.
- if it takes too much time (more than an hour) we should go for yeasts optimization alone.
- We should be able to do that in sequencing company's sites.
- Enzyme names are not Absolut! We should use the exertion numbers of each gene.
- We should check if there are splices to each gene.
- When we have those sequences we should check with IDT and twist to get price tag and supply timeline.
- We need to contact twist and IDT and see if we can get the sequences.
- We have a notebook of the yeast vectors to read and understand how to work with.
We found out that there are 2 options to insert the genes to the yeasts:
1. Plasmids, which will give us more enzymes in the cells but the difference between cells is high (10,000 times more enzymes between cells).
2. We can insert it to the yeast`s genome which will be more stable between cells but we will receive less enzymes in general.
- We need to decide how we want to insert the genes to the plasmids, using restriction enzymes or gibson technic.
- Restriction enzymes might cut inside the gene so we need to check all the different sequences that we want to use and maybe even change them a bit so it wont happen. Mosquna thinks that gibson technic might be easier because of that.
- We should check if we can use gibson technic for biobrick creation.
- Safety: Asaf thinks that the personal equipment is very important, so only we use it and nobody else contaminate it.
- Asaf introduced a program called "genius" for planning gene sequences and experiments.
Everyone.
March 23rd.
March 24th.
March 25th.
Team Meeting - Assays and Sequences
We started discussing the sequences and reviewed if they might work in our organisms or not.
- Assaf spoke with Berta about green light for purchasing materials and she gave it.
- He says that now there is more pressure on Berta to receive more money.
- He says we should talk with yonathan about the degradation of the Cl atoms.
- Assaf wants us to be prepared to plan the cloning for after pesah.
- We might try to use the igem`s creation for plasmid quantification - https://2017.igem.org/Team:Vilnius-Lithuania.
- We should contact "genious" and ask if they support iGEM.
- We should check if anyone tried to syntisys the same enzymes we want to use with some sort of a FLAG.
We should talk to IDT`s representative in israel about the kbs.
How we do cloning:
- Download the annotated sequence.
- Look for relevant restriction enzymes in the pESC plasmid using the program and mosquna`s notebook
- We should use 2 resistances for the large enzymatic pathway and another two for all other enzymes we need (like the reductase).
After we find our FLAG, we should delete the stop codon in our enzyme sequence (if we want to insert it before the flag) or not if we add it after.
Everyone.
March 26th.
Team Meeting - Rebuilding our Enzyme`s Genes
Wev'e built gene combinations for our experiments.
For easier typing' the following groups should be named:
Alpha&beta+223+26=AB
44a+223+26=44A
Lip+mnp=Lipp
Suggested groups:
1. Only AB(4)
2. Only 44A(4)
3. deHalo+AB(5)
4. deHalo+44A(5)
5. CYP+AB(5)
6. CYP+44A(5)
7. Only lipp(2)
8. CYP+lipp(3)
9. Only CYP(1)
10. Only deHalo(1)
If aiming for redundency:
1. AB+44a(6)
2. deHalo+AB+44a(7)
3. CYP+AB+44a(7)
4. AB+44a+lipp(8)
5. CYP+AB+44a+lipp(9) -> (!notice: our maximum insert includes 8 genes!).
6. CYP+deHalo+AB(6)
7. CYP+deHalo+44A(6)
Everyone.
March 27th.
Marketing Group Meeting
Summary:
- We need to drip some social media sparks (The Facebook page still get weekly viewers without any movement for the past 3 months).
- Discuss needed presentation files for introducing to the university`s vice president.
- Discuss companies and organizations that we want to approach.
- Discuss content of presentations to aid us to present and for sending to others.
- Plan our next moves and presentation practices.
- We should Ask Issum in personal about free tickets for Agrivest.
- Discuss community project management (Each faculty team with different project with a line that combine all of them). A twist is needed for spreading the word in a viral way.
- Discuss initiation of video concepts.
- Video Concept - pluralism of Israel's population. Exposing people from the street to genetic engineering and documenting their reaction.
- We decided to check on other teams and their marketing plan and take good ideas from there.
- We need to come up with an idea for the script of the animation video.
- We thought about promoting Bilkis's idea for creating a Regulation system for TCDD manufacturing.
- We should try and open Instagram page and a Musical.ly page.
- On the website there should be a tab for stories of people who got injured from exposure to TCDD .
- And in the same breath - we need to build a website !!
- About Maintenance of the Facebook page - apparently a rotation between us will be the best.
- We discussed the fact that it is now the holidays so it`s not a good time to try and approach companies.
Eliya, Lior B, Alon and Sivan.
Lab Meeting with Assaf, discussing our gene combinatorics.
Amichai, Lior S, Aliza and Hagit.
March 28th.
March 29th.
March 30th.
March 31st.
April 1st.
April 2nd.
April 3rd.
April 4th.
April 5th.
April 6th.
April 7th.
April 8th.
April 9th.
April 10th.
April 11th.
Meeting with University vice presidents, Prof. Oron Shagrir and Mr. Yossi Gal.
Oron concluded that he will work with Karen from the international office and will try cover our Jamboree expanses.
Eliya, Badash and Alon.
Meeting with University vice presidents, Prof. Oron Shagrir and Mr. Yossi Gal.
Yosi concluded that because the president of the university will fly to the US next week, we should send him the PPT and one-pager that describe the project and he will try to recruit donations for the project.
Eliya, Badash and Alon.
April 12th.
April 13th.
April 14th.
April 15th.
First Lab meeting.
Everyone.
Plasmid Restriction
Restriction in preperation for Gibson Assembly
Everyone.
April 16th.
April 17th.
Meeting with Dr. Yonatan Friedman (Microbiology)
Alon.
April 18th.
Community project meeting. Deciding on team members that will lead community projects efforts.
Everyone, Elyia administrated the effors.
April 19th.
April 20th.
April 21th.
April 22nd.
Gel Analysis of Plasmid Restriction
As can be seen, we got 2 large amounts of fragments in uncut plasmids, and 2 strands in the cut fragments of U, T, H Vectors.
The L vector, which is outlying (with a single band) was the only vector not restricted with the Cla enzyme.
To be safe, we decided grow additional plasmids from Glycerol Stock, and preform a Mini Prep and restriction again.
Amichai.
April 23rd.
Overnight Cultures for miniprep on 24.4
Amichai.
April 24th.
First group meeting in Ein-Karem Campus: Discussing Gene therapy experiment, the option of making a research about TCDD breakdown in the  biosphere and recruitment of staff members in Ein Karem Faculty.
We decided to fuse Ein-Karem  students with Givat Ram cell, leaving the option of having a lab experiment in  Ein-Karem.
Alom, Lior and Amichai
Miniprep and Gel analysis
Bacteria grew well overnight, opaque media in the morning
Miniprep went according to protocol, and gel analysis confirms good results
Amichai.
April 25th.
Meeting  with Danny Porat, V. Dean in Givat Ram Faculty.
Lea.
April 26th.
April 27th.
April 28th.
April 29th.
April 30th.
May 1st.
Restriction PCR
In preparation for Gibson Assembly [GA]
Amichai
May 2nd.
PCR Purification and Gel Analysis
Bands are as expected, we can move forward with GA
Hagit
May 3rd.
1st wiki  design meeting.
a. We agreed that Wix is no a possible way to build our wiki (Because it  requires to store the information on their server). 
b. We concluded that each team member will bring a rough template of  how he sees the wiki, and we will choose the best one to go with next week.
c. When we will have the official logo, we will be able to start  adding protocols with the logo as a header.
d. We should find a mascot, Lior brought an idea to have a kind of "Odish" Pokémon traveling in our website.
e. Omri will organize the new section for wiki.        We made a rough schedule for our overall plan (Wiki should be  ready at 19.07.18).
Alon, Lior B, Aliza and Omri.
May 4th.
May 5th.
May 6th.
May 7th.
Amplification of Reductase Gene
The reductase is an enzyme which assists with activity of CYP450
We used Phusion enzyme for this amplification
As shown, there was both an expected band (~2000bp)
an unexpected band at ~500 bp.
To compensate for this, we will read more colonies after transformation
in an attempt to ensure we cloned a vector with the 2000bp insert and not the 500 bp insert
Amichai
5 GA reactions were run:
Leu Vector + Dehalo
Leu Negative CTRL (NC)
Ura + CYP450 + Reductase + Gal1/10
Ura Negative CTRL (NC)
Positive CTRL (PC)
Amichai
E. coli Transformation with plasmids from GA
Amichai
May 8th.
Agrivest Convention (Trask, Tel Aviv) : Israel's premier ag-tech investment event.
We concluded that the convention was  dedicated for investments opportunities with possible profit and that our project didn't fuse well into the general purpose that the business man came for. We Managed to get 25 businness cards and sent mails with information about our  research and community project plants, wishing for possible donations.
Lior B, Lior S, Hasan and Alon.
May 9th.
Community Project: Meeting With the designer from Betzalel art school (Eyal Shushan). We discussed cost of the project and rough timeline.
Lior S, Alon, Eliya, Lior B.
Colony screen for transformation from 7.5
Negative Control plates showed approximately the same amount
of colonies as the transformed bacteria, but now screen showed any bands
Amichai
May 10th.
May 11th.
May 12th.
May 13th.
After troubleshooting our failed transformation, it became clear
that we used un restricted vectors for the GA reaction, due to a
simple miscommunication. To confirm our suspicion, we ran an
additional GA reaction with the RESTICTED vectors (Leu, Leu NC)
Aliza, Hagit, Amichai
May 14th.
Preperation meeting for "Basakarabidopsis" event which took place in May 16th.
Elyia and Sivan.
May 15th.
MIXiii bioMED (David InterContinental Hotel, Tel Aviv): We participated in bioMED conference which was focused on healthcare. We got business cards and sent Emails with details describing our research and our community project.
Eliya, Lior B, Yaarit, Amichai, Aliza and Alon.
Created Glycerol Stocks of the strains grew from 14 5 (all four)
Additionally Mini Prep was preformed on Clone #1 and #2
and sent for sequencing.
Lior S.
May 16th.
"Baskarbidopsis" event in the  faculty:
We invited our entire university, including the staff, to join us in a fun ring toss competition on our campus, which we called “Baskarabidopsis" – Put the Plasmid in the Plant.
We constructed plasmid-shaped loops to throw and plants that were made from recycled plastic bottles.  While enjoying the game, we introduced ourselves and our project goals to our visitors, opening the minds of our student body to the fascinating world of synthetic biology. Even the administration of our university participated! Prizes were won and everybody had a great time!
Everyone.
May 17th.
4 GA reactions were run:
His Vector + 266 + 233TBD 1 +233TBD 2 + Gal 1/10
His Negative CTRL (NC)
Trp +  44a sS + 44a LS 1 + 44a LS 2 + Gal 1/10
Trp Negative CTRL (NC)
Amichai
May 18th.
May 19th.
May 20th.
May 21st.
E. coli transformation on the His and Trp vectors from 17.5
His Vector + 266 + 233TBD 1 +233TBD 2 + Gal 1/10
His Negative CTRL (NC)
Trp +  44a sS + 44a LS 1 + 44a LS 2 + Gal 1/10
Trp Negative CTRL (NC)
Amichai
May 22nd.
Colony screen for transformation from 21.5
Amichai
May 23rd.
Miniprep for Sequencing
Amichai
May 24th.
May 25th.
May 26th.
May 27th.
May 28th.
May 29th.
Re-Amplification of Reductase Gene
Due to the issue we had when amplifying earlier in the month, we decided
to amplify the gene from a plasmid instead of cDNA.
There is a minor secondary product, but it appears less than the previous amplification
As shown, there was both an expected band (~2000bp)
Amichai
PCR Purification
Preformed only on the 1:1000 plasmid dilution to avoid as much plasmid as possible
Eliya
2 Vectors sent for sequencing [His, Trp]
Aliza
May 30th.
E. coli transformation for Ura[+CYP450, Gal 1/10, Reductase] vector from 17.5
Ura +CYP450 + Gal 1/10 + Reductase vector
Ura Negative CTRL (NC)
Amichai
https://static.igem.org/mediawiki/2018/c/c9/T--Hebrewu--310518_fun_time.jpeg
May 31th.
Social evening - cooking dinner and playing card games.
Everyone.
Colony screen for transformation from 30.5
Gel 1: Cyp Amplification:
Gel 2: Reductase Amplification:
Due to the unclear results(no plasmid appears to have both inserts),
we grew colonies all colonies showing at least one insert for Mini Preps
Aliza
June 1st.
MiniPrep from starters from 31 5
Amichai
Glycerol stocks from same starters
Amichai
Miniprep Gel Analysis
Gel Analysis came back inconclusive.
Because the samples are plasmids (with tertiary structures)
while the ladder is linear, it is hard to tell the exact size of the plasmids,
PCR analysis was conducted later in the week
Amichai
June 2nd.
June 3rd.
1# Meeting with prof. Yaakov (Kobe) Nahmias (Givat Ram Campus).
a. We reviewed our team goals and possible directions concerning liver cells.
b. When we discussed about using plants and moss to remediate lands, Kobe pointed out that he thinks that a more practical way for remediation will be by using yeast (Spray them). Spraying can work in forests where we can't grow any plants (too dark and competitive). Moreover he mentioned that choosing the right yeast could be an interesting path (With the right kill switch).
c. When we discussed about experimenting in his lab, he said that all the options that we discussed concerning the liver cells is feasible, would take about 2 months and can be made by 1 dedicated person from our team with one of his mentor in his lab. Kobe offered to fund the experiment with his lab budget, except analysis part.
d. For experimenting with TCDD in his lab we need to have someone that will be dedicated also for the safety part - how to deal with that material in the lab.   e. Also, Kobe lab can easily detect LD and TC levels in human liver cells relatively easily (Toxicity).
After the conversation with Kobe, we continued to discuss our ideas with Merav Cohen (His lab director) and Avner Ehrlich, a PhD student.
We concluded that we need to understand exactly what we want to do in the experiment, understand which team members will prepare and take part in the wet lab. After that, we should aim to start with wet lab after the exam period.
Alon, Eliya, Lea and Dania.
2# Meeting with prof. Shimshon Belkin (Givat Ram Campus).
a. We reviewed our group goals and possible directions concerning our research.
b. Shimshon agreed to lead the group efforts in Givat Ram campus!
Within that Shimshon will help the team members with logistics (getting rooms for the meeting, budgeting and contacts in the campus).
c. Shimshon pointed out a cheap way for analyzing level of TCDD breakdown:      Analyze the level of free chloride in the solution that will indicate the level of breakdown.
d. For funding, Shimshon suggested to try and get funding from:
a. Teva.
b. "Ramat Hovav" industries ("Brohm", "Machteshim")
c. Gal Erlich patent office.
Alon, Eliya, Lea and Dania.
PCR to check Ura clones
Due to inconclusive results on 1.6, we will ran an additional PCR for gel analysis.
Due to continued inconclusive results, we will conduct PCR analysis on other colonies
Amichai
June 4th.
Meeting with Dr. Yair Mau, Dr. Nimrod Schwartz:  a.  We introduced the project, the tean abd and our calculator idea. They concluded that it will be feasible to make such calculator in our time frame.   b.  In order to make such calculator we should look for this variables:
1. uptake rate of TCDD in our plant.
2. Interaction between the soil and the TCDD.
3. Solubility of TCDD.
4. volatility of TCDD.
5. Level of contamination in specific areas (Alon).
c. Nimrod pointed out that the uptake might be relevant only for a very short distance from the root (The rhizosphere), and this property might be problematic while taking our solution to the environment.  Although this property is shared with Heavy metals which has similar practical implementation.
We concluded that we should aim for a tool that will be able to understand the spread TCDD in a given soil and it's uptake while looking at time and space variables. In order to make that tool we need to assume information about the root system and about its flow in the soil.
Hagit, Yaarit and Alon
Colony PCR for additional colonies from transformation on 31.5
Overnight culture started form positive colonies
Aliza
June 5th.
MiniPrep from starters from 31 5
Amichai
Glycerol stocks from same starters
Amichai
Miniprep Gel Analysis:
We decided to send clones 5 and 6 for sequencing.
Amichai
Sterilized and Planted Arabidopsis seeds (Strain- WT-Col) on MSA medium
Aliza
June 6th.
June 7th.
After reviewing Sequencing results:
Both the His and Trp Vectors look good, and we will proceed with cl. 2 for both vecotrs
Ura vector is incomplete. We will start the cloning process again next week from mini prep and restriction
Amichai
June 8th.
Meeting about the wiki
We discussed design ideas, phones compatability, wiki main menu and a version of KEGG map for introducting the work that hss been done by each team member.
Aliza, Hasan, Omri and Alon.
50 ml Yeast Starters (prep for Yeast Transformation):
yms2 W303 mat a Cl 1
yms2 W303 mat a Cl 1
Set for incubation- 28°C for 48 hours, 200 RPM
Aliza, Amichai
June 9th.
June 10th.
Yeast Transformation for Leu, Trp, and His Vectors
Lior S.
Set up Miniprep Starters for Ura and His Vectors
Lior Si
June 11th.
Meeting with Rosalin (Weizmann institute, Flieshman lab)
Meeting Summary:
a. Their newest tool of the lab named funclib and is used to make certain in the proteins more ligand specific (in up to 15 sites).
b. It's possible to run the program with a bound ligand so if we will have a crystallographic structure that contains TCDD ore some derivative of it, the process might perform better results.
c. Rosaline mentioned that We should use pross tool (Also from Sarel lab) just in case the expression level of the protein in the cell is very low, otherwise it's not needed for our purpose. Also the lab has a special pross for membrane proteins like our CYP seem to be.
Eliya and Alon
Meeting with Noam Kirshennbaum: Analyzing of our experimentt results (Rehovot Campus)
Analytical methods suggested by Noam:
a. HPLC  with UV detector
b. GC with FID/ECD detector.
Noam suggested we should check for the best method that works with our medium. If both doesn't show up well we might need another stage of purification.
Hagit mentioned that we might be able to dissolve TCDD in DMSO while measuring the breakdown in yeast.
Moreover, Hagit found an old and simpler method called TLC that we should check (if it fits our needs).
Furthermore, when TCDD will arrive we could try analyze it by GC-MS.
Tasks for next meeting:
a. We need to find libraries that contains properties about our reacting molecules.
b.  We might find information concerning our molecules in M/Z cloud library (High resolution database).
While looking at another analyzing methods we should aim to understand the concentration range and sensitivity of each one of them in order to find the best that can fit.
We must prepare an archive of all the data we are able to find regarding the materials (Tetrachlorobenzene, dibenzofuran and TCDD)- physical properties, chemical properties etc..
We concluded to have a tour at Noam lab later this week (13.06.18 at 1100).
Hagit, Alon, Omri and Shai.
Miniprep and Restriction for His and Ura vectors
Aliza
June 12th.
PCR Purification for Restricted Ura vector
Eliya
Gibson Assembly for Ura vector and E. coli Transformation
Amichai
June 13th.
Colony Screen:
Colonies from highlighted products were grown over night
Aliza
June 14th.
Glycerol stocks from starters from 13.6
Amichai
Miniprep from same starters
Amichai
PCR and Gel Analysis for new Ura Clones
Cl. 12 was sent for sequencing
Amichai
June 15th.
50 ml Yeast Starters (prep for Yeast Transformation):
yms2 W303 mat a Cl 1
yms2 W303 mat a Cl 1
Set for incubation- 28°C for 48 hours, 200 RPM
Amichai
June 16th.
June 17th.
Yeast Transformation for Ura Cl. 13,13,15 and Ctrl Vectors (w/o genes) for each strain
Amichai.
June 18th.
Sequencing sent for Ura cl.12
Amichai.
June 19th.
inoculate Starters with Yeast Colonies from 17 6 transformation  14 1ml starters (a & a):
Ura cl 12
Ura Cl 13
Ura Cl 15
Leu CTRL Vector
Ura CTRL Vector
Trp CTRL Vector
His CTRL Vector
Amichai.
June 20th.
Glycerol stocks from starters from 13.6
Lior S
June 21st.
Prepare dropout media for starters
Amichai
June 22nd.
June 23rd.
June 24th.
Yeast transformation (a & a) according to protocol for:
1. Dehalo (adding Cyp)
2. 44a (adding 226.24-223)
Grown on SD Agar for 48 hours at 30C on
1. Leu + Ura Dropout
2. His + Trp Dropout
Eliya, Amichai
June 25th.
June 26th.
Selection for transformed yeast from 24 6 was ineffective and there was a "blanket" on both plates
Everyone.
June 27th.
Poster design meeting.      a. Aliza Set important dates for the making of the poster.   b. We choose elements to have inside the poster : dark background, center piece, up to 3 colors, no need for explicit titles, minimal borders (maybe interesting upon further inspection) with minimally curved edges and no pictures, only illustrations.
Aliza, Hasan, Omri, Badash and Alon.
June 28th.
June 29th.
June 30th.
July 1st.
01.07.18 - Meeting with Teva Pharmaceutical Industries
Eliya describeed the competition and our project.   Alon Described the Community project and that we were aiming to build  our exhibit in the summer and operate it in September - January while traveling with it across the country (Looking to reach periphery schools with Education for excellence NPO).
July 2nd.
July 3rd.
July 4th.
July 5th.
July 6th.
July 7th.
Community project: Moving from The Big Cell to Caroucell (Skype)
a. Eliya gave a Budget overview .
b. we decided to Cut the main structure so the exhibition will be in other closed spaces (Classes / Museums).
c. We decided to start an headstart campain to recruit money for the project.
E. Shushan (designer), Y. Meroz (Designer), Eliya, Alon, Lior B, Lior S.
July 8th.
July 9th.
Yeast Growth Curve Experiment:
2 Yeast lines- Leu Ctrl, Ura Ctrl (3 repetitions of each line)
Grown at 28C, 200 RPM
Read times [h] t=0,8,10,12,14,16,18,20,24,34
.
Amichai
Sterilized and Planted Arabidopsis seeds (Strain- WT-Col) on MSA medium
Aliza
July 10th.
July 11th.
July 12th.
July 13th.
July 14th.
July 15th.
July 16th.
Move WT Arabidopsis from MSA plate to soil
Alon
Inoculate LB with e.coli pHeGHPB DHSalpha for  miniprep
Aliza
Dissolve 1,2,3,4-Tetrachlorobenzene in methanol- final concentration 1mM
Dr. Mosquna, Amichai
July 17th.
Miniprep: e.coli pHeGHPB DHSalpha
Aliza
Growth curve experiment for TCB with Dehalo and Cyp enzymes
Amichai
July 18th.
Meeting with TecCEM I-GEM team (Mexico).
We started by describing our project and hearing about their's. Afterwards we discussed collaboration possibilities in the inter-lab or modelling part of the project.
Lior B, Eliya, Amichai
Growth curve experiment for TCB with Dehalo and Cyp enzymes:
200ul samples were collected from each sample for chemical analysis.
Amichai
July 19th.
Samples from each treatment were put in DO media and set to incubate [for 48 hours at 30C, 200 RPM] for genotyping
Amichai
July 20th.
Community project meeting: The Caroucell
We decided that the exhibit will include 8 stations with different backgrounds (By order). Each station will show different animation that reacts to the background.
1. Human Body
2. Cell membrane and cytoplasm fluidity
3. Mitochondria as an energy factory
4. Nuclei
5. Ribosome (In the background of ER)
6. Protein (In the cytoplasm)
7. Cytoskeleton
8. Membrane (Exit)
The background is static, while the phones are moving between the different backgrounds. The spectators will walk with their phones through the different organelles.
The exhibit will have 2 routes (Polycistronic!)  with different type of information:
1 - The screen on the phone.
2 - the circular table under the phone.
The duration of a full walk between the organelles will take 1:30 min.    ** Proteins that we designed with the kids (on the card boards) will hang out from the top towards the spectators.
Alon, Eliya, Lior B, Lior S, E. Shushan (Designer), Y. Meroz (Designer),
July 21st.
July 22nd.
Zymolyase treatment for cell wall degradation before genotyping PCR
results were inconclusive
Amichai
July 23rd.
Arabidopsis Plasmid Cloning:
CYP and Dehalo gene Amplified with new primers
Continued with dehalo, Cyp will be amplified again
Aliza
July 24th.
Yeast genotyping:
Running a PCR to ensure that the samples (from 17.7 Ex ) were labeled correctly, ,br/> and strains were properly transformed using gene specific primers.   .
Amichai
July 25th.
July 26th.
Arabidobsis cloning: Cyp amplification with new primers:
Q5 55C chosen for PCR PURIFICATION.
Aliza
July 27th.
Wiki-Poster meeting: discussing team's proposal for poster structure.
Aliza, Lior B, Eliya, Amichai, Omri, Hasan.
Yeast genotyping- no  comprehensible results from 24.7 PCR
Amichai
July 28th.
July 29th.
Yeast genotyping- Rerun gel from 27.7, still no  comprehensible results from 24.7 PCR
Amichai.
July 30th.
Arabidopsis Cloning: Restriction for CYP, Deh and plasmid with EcoRI and BamHI.
Aliza.
Arabidopsis Cloning: Ligation reaction
Aliza.
July 31st.
Arabidopsis Cloning: E coli transformation with CYP.arab, Deh.arab and control (empty plasmid).
Aliza
Yeast experiment #2:
Repeat of the 17.7 growth curve experiment for TeCB with Dehalo and Cyp enzymes:
Amichai
August 1st.
Meeting with Assaf (Mentor)
Alon and Lior presented the current efforts regarding our community project and our livestream. Later we discussed Jamboree logistics.
Alon, Lior B
Yeast experiment #2 Results
Amichai
Colony PCR for31.7 s overnight e coli colonies with CYP.arab & Deh.arab plasmids
Aliza
August 2nd.
Yeast Genotyping PCR (repeat)
Amichai.
August 3rd.
August 4th.
August 5th.
Gel analysis for 2.8 (genotyping) PCR
Amichai
August 6th.
Sequencing for Cyp (plant vector) came back faulty
because of multiple issues with Cyp, work with this enzyme was
until a fully sequenced clone is found
Aliza.
August 7th.
Meeting With NCHU Taichung I-GEM team(Taiwan)
We started by explaining both projects respectively, and understanding the similarities that we found in the NCHU wiki.
- As for now, our research is focused on same problem (TCDD as a byproduct of Agent orange). Within that, NCHU team visited Vietnam and visited the "War Museum".
- It's the first time their university is participating in iGEM    - The team consists of 17 students.
- Although the problem is almost identical to ours, their approach is different. They are looking to engineer a plant symbiont (Bacteria), while expressing genes that will help the transfer and the degradation of the molecule inside the plant.
- NCHU focus on 3 enzymes and have obtained results showing transfer and dechlorination of TCDD. They work with TCDD in their lab and not similar molecules like us (In terms of safety, they use 2 disposal masks when performing the experiments with the material).
- NCHU use mainly HPLC for analyzing their experiments. Also, they use HTC for toxicity tests.
- NCHU team showed interest to know more about our livestream and take an active part.
Alon, Lior B, Amichai, Shai
PCR to ensure dehalo seq (primer) is gene specific and that Dehalo gene is in Leu vector as labeled
Amichai
August 8th.
Meeting with Tartu I-GEM team (Estonia)     We described both our projects and decided to combine forces in few subjects. It was decided that Huji Team will answer Tartu survey about sun screen and that Tartu will research about TCDD problem in Estonia. (See more details about the research that has been done by Tartu in our collaboration page). Beside that, Tartu showed interest to take an active part in our Live stream show.
Alon, Lior, Omri, Sivan, and Eliya
Deh.arab starter inoculated for overnight growth
Aliza
August 9th.
Miniprep from 8.8 starter
Aliza
deh.arap plasmid sent for sequencing
Aliza
"Dirty" DNA extraction from yeast
Still no clear results for yeast genotyping
Amichai
August 10th.
August 11th.
August 12th.
August 13th.
Prepared new SD Drop Out media
Hagit
Prepared starters for Yeast transformation
-Trp 44a (to add His 226,233)
-Trp C.V. (to add His CV)
-"WT" (to add Ura Cl. 15)
Amichai
August 14th.
AtDeh cl.3 seq results comeback positive.
Aliza
Yeast transformation using yeast starters from 13.8
Amichai
Rerunning Genotyping PCR from 9.8 (due to inconclusive results):
Amichai
Re-plate all yeast on selective agar plates
Amichai
August 15th.
Team meeting
We discussed current stae of our crowdfunding efforts to the community project, we summerized the latest work that been done in the lab and we shared tasks concerning our next Live Stream program.
Moreover, We discussed the level of dedication to the project and that we will have to switch gears for the months to come. We decided to try a new way to manage the team's tasks.
Everyone.
August 16th.
Zymolyase cell wall degradation protocol (for yeast genotyping)
Amichai
Miniprep for Zymolyase treated samples
Amichai
Ura Cl. 13 sent for sequencing
Amichai
Single colony starter from 14.8 transformation inoculated
Amichai
August 17th.
Genotyping PCR from 16.8 Miniprep
Amichai
Gel analysis
Results still inconclusive
Amichai
August 18th.
August 19th.
Interlab Study: Calibration Protocols and E coli Trnsformation
Eliya
Prepare SD DO media for transformations planned for 21.8
Aliza
Rerun PCR from Miniprep on 16.8 with new primers
Aliza
Gel Analysis for PCR
Due to continued negative results we will reperform transformation to yeast with sequenced vectors
Amichai
From seq results it would appear that Ura Cl 13 has two plasmids, with different seqs of the Cyp gene To separate the plasmids we ran a transformation from the cl 13 miniprep, and will continue with colony pcr and standard genotyping protocol.
Amichai
E coli transformation with Ura vector
Amichai
August 20th.
Interlab Study: Transformed Colony Inoculation
Eliya
5 single colony starters from 20.8 transformation inoculated
Aliza
August 21st.
Interlab Study: Abs​600​ and Fluorescence measurement
Eliya
Interlab Study: CFU plate innoculation
Eliya
Colony PCR for 19.8 transformation
Aliza.
Yeast transformation for all strains
Leu- DH
Leu- CV
Trp- 44a
Trp- CV
His- 22
His CV
Trp+ His 44a, 22
Trp+ His CVCV
Amichai
August 22nd.
Interlab Study: CFU Measurements and Form Submission
Eliya
August 23rd.
August 24th.
August 25th.
August 26th.
Single and Multiple Colony starters from 22.8 transformation
Amichai
Ura cl. 13A sent for sequencing
Amichai
August 27th.
Agrobacterium Transformation- Dehalo plasmid
Amichai
August 28th.
Group Meeting with Assaf
a. Captains presented red tasks for medals.
b. Each team member presented their latest work regarding the project and plan for the month to come.
Everyone
Cell wall Degradation and Miniprep for successful transformations from 21.8i
Amichai
Gel Analysis
Both transformation and genotyping were successful
Amichai
August 29th.
Yeast transformation: transformation with following plasmids:  His+Trp (22,44)
His (233,266)
Trp (44a)
NC (H20)
Seeded on selective SD plates
Amichai
August 30th.
Arabidopsis inoculated with transformed agrobacterium
Aliza, Lior S
August 30th.
Infected Arabidopsis set up right and left to flower
Aliza
September 1st.
September 2nd.
September 3rd.
Dehalo Growth Curve Experiment
Amichai
September 4th.
Biobrick Synthesis
Genes amplification for biobricks with prefix and suffix
Aliza.
September 5th.
Dehalo Growth Curve Experiment
Amichai.
September 6th.
Dehalo Growth Curve Experiment Results
Amichai and Aliza.
September 7th.
September 8th.
September 9th.
Yeast transformation preformed on Leu/DH  strain:
1. His+trp [22,44a]
2. His [22]
3. Trp [44a]
4. NC [DDW]
Amichai
September 10th.
September 11th.
September 12th.
His+trp [22,44a] transformation (on Leu Strain) appears successful 4 starters inoculated
Amichai
September 13th.
BioBrick e. coli transformation
3 transformations: Dehalo BB, Gal1/10 BB, Negative CTRL
Eliya
September 14th.
Yeast cell wall treatment and miniprep
Amichai
PCR and Gel analysis for Yeast miniprep
Amichai, Aliza.
Biobrick Colony Screen
Aliza.
September 15th.
September 16th.
Biobrick Miniprep
Aliza
Inoculated yeast starters :
1. His Trp CV (for Leu CV Transformation)
2. LHT-D24 (for Glycerol Stock) .
Amichai
Dibenzofuran Dissolved in Methanol solution [30mM]
Dr. Mosquna, Amichai
September 17th.
Yeast Transformation : Leu CTRL to HIS-TRP CVCV
Lior S
Gly stock for D24 Strain (entire pathway in one yeast strain)
Lior S
September 18th.
Yeast starters inoculated for qPCR experiment to test Gal1/10 Promoter
Amichai
September 19th.
Yeast from 18.9 starters moved to new medium:
1,2: D24 Galactose (induced)
3,4: D24 Glucose (induced)
5,6: Trp CV Galactose (uninduced)
7,8: Trp CV Glucose (uninduced)
3 Hours Growth in new media
Eliya
September 20th.
RNA extraction (for qPCR)
RNA Gel:
Two clear bands representing rRNA indicating a successful extraction
Eliya
September 21st.
cDNA synthesis (for qPCR)
Eliya
September 22nd.
September 23rd.
September 24th.
September 25th.
September 26th.
GibsonBrick- 3 BB to be stitched:
1. Promoter + RBS (BBa_K608002)
2. Blue Chromoprotein (BBa_K592009)
3. Terminator (BBa_B0015)
Step 1: PCR to Amplify Parts from BB plasmids
Eliya
Step 2: PCR Purification for all samples
Eliya
Step 3: Gibson Assembly
Eliya, Amichai
Step 4: Ecoli Transformation
Amichai
Dibenzofuran (DBF) Growth Curve Experiment
Amichai
September 27th.
Dibenzofuran (DBF) Growth Curve Experiment
Amichai
GibsonBrick Experiment: Colonies but no expression
Amichai
September 28th.
Dibenzofuran (DBF) Growth Curve Experiment
Amichai
September 29th.
none, 12:00.
Everyone.
September 30th.
GibsonBrick- 3 BB to be stitched:
[Vector (GAL1) =>]
ECFP, yeast-optimized (coding region)
Constitutive strong promoter with medium RBS and GFP
amilCP Reporter (RBS+Coding+Term)
=>Vector
Step 1: PCR to Amplify Parts from BB plasmids
Eliya
Step 2: PCR Purification for all samples
Eliya
Step 3: Gibson Assembly
Eliya, Amichai
Step 4: Ecoli Transformation
Amichai
October 1st.
DBF Growth Curve Experiment
Checking multiple concentrations of DBF
Amichai
October 2nd.
DBF Growth Curve Experiment (two parallel experiments)
Checking multiple concentrations of DBF
Amichai
October 3rd.
DBF Growth Curve Experiment
Checking multiple concentrations of DBF (two parallel experiments)
Amichai
Biobricks sent for sequencing
Aliza
Biobricks overnight started inoculated
Aliza
October 4th.
Biobrick plasmid Miniprep
Amichai
October 5th.
October 6th.
DBF Growth Curve Experiment
Amichai
October 7th.
DBF Growth Curve Experiment
Amichai
October 8th.
DBF Growth Curve Experiment
Amichai
Biobrick Genotyping
Due to problems with sequencing, we ran multiple PCRs to ensure our parts were in the vector
Aliza
October 9th.
GC-SM for both TCB and DBF samples
University Chemistry Labs
October 10th.
Chemical Analysis
Amichai
October 11th.
Chemical Analysis
Amichai
October 12th.
Chemical Analysis
Amichai
October 13th.
Chemical Analysis
Amichai
October 14th.
Chemical Analysis
Amichai
October 15th.
Chemical Analysis
Amichai
October 16th.
October 17th.
October 18th.
October 19th.
October 20th.
October 21st.
October 22nd.
October 23rd.
Huji Team flight to Boston
Everyone.
October 24th.
October 25th.
October 25th.
October 26th.
Giant Jamboree - Huji Presentation!/li>
Everyone.
October 27th.
October 28th.
October 29th.
October 30th.
October 30th.
