Team:HebrewU/Parts
HebrewU HujiGEM 2018
Hebrew
Project
Description
Model
Results
Parts
MOOLTi
Lab
Yeast Design
Plant Design
Notebook
Inter-lab
Safety
Protocols
Human Practices
Ethics
Global Aspects
Public Engagment
Open Source
Team
Members
Attributions
Collaborations
Photo Gallery
Judging
Project
Description
Model
Results
Parts
MOOLTi
Lab
Yeast Design
Plant Design
Notebook
InterLab
Safety
Protocols
Human Practice
Human Practice - Main Page
Ethics
Global Aspects
Public Engagment
Open Source
Team
Members
Attributions
Collaborations
Photo Gallery
Judging
Gal 1/10
   
Gibson brick
Gal 1/10
GAL 1- GAL 10 is a divergent promoter region of Saccharomyces cerevisiae, allowing it to regulate two genes simultaneously.                Each complimentary strand promotes translation in opposing directions.                Galactose induces transcription, while Glucose serves as a repressor.                The promoter is inducible by as low as 2% Galactose the in medium.
It is particularly useful for measuring protein-protein interactions, expressing proteins with two subunits,              or can simply be used to effectively cut the number of plasmids required  for cloning in half, as two genes can be cloned into a single plasmid.
To validate this part we performed Real-Time PCR on yeast with our Leu-44a vector,             containing two subunits of the same protein- 44a-Dioxygenase (44a).             This enzyme contains to subunits: 44a Large Subunit (LS) and 44a Small Subunit (SS).              We used Leu2 as a regulating gene, as it is found on the same plasmid,              but its transcription is not dependent on the Gal 1/10 Promoter.
We grew 2 strains of yeast in SD media (containing glucose) for 36 hours. The strains were as follows:
1. 44a- Containing the vector shown in the map above,                      with the LS transcription being promoted by Gal 10 and SS transcription being promoted by Gal1.
2. CTRL strain, containing a similar but "empty" vector.                      This vector also contains the Leu2 gene, and Gal 1/10 promoters, but does not have the 44a genes inserted.
After growth, be split each strain into 2 treatments:
1. Induced:      SG media, containing 2% galactose in the media.
2. Uninduced: SD media, containing 2% glucose in the media.
As the primers for the RT-PCR are gene specific to the subunits, we expected to see no product in the CTRL vectors, in both treatments.                  The results of this experiment are detailed in the graph below.              After normalizing the raw expression data of our target genes to the expression of the Leu2 gene, we get the figures presented.
In the induced treatment, we see expression rise more than 20 fold over the uninduced treatment.
Additionally, we see approximately equal expression between the uninduced treatment and the control group,              indicating the minor levels could be background noise caused by the non-specificity of primers, or primer-dimer amplification.              (We used SYBR, and not a probe, therefore the possibility of such background noise arises)
Gibson brick
An obstacle many team face is finding a way to assemble multiple genetic components into a single                plasmid effectively and quickly. Though the biobrick standard addresses this issue,                we found that using newer, more advanced methods of can help save time and resources.                We, as many iGEM teams before us, decided to use Gibson Assembly. Since no restriction                is required for this method, the multi-cloning sites in the biobrick prefix and suffix were not particularly helpful.       Gibson Assembly utilizes homologous overlaps on each DNA fragment allowed for their cloning into a single vector.       This set of 8 oligos, that help to convert any biobrick into a Gibson Assembly ready fragment.                Each of our oligos is approximately 40 Bp long, consisting of two components:
1. Biobrick prefix/suffix region- allowing them to be used as primers for any (RFC10) biobricks currently in the iGEM library.
2. 20 bp region with no secondary structures and about 50% GC content, allowing for extremely efficient Gibson Assembly.
This means that teams can, in one PCR, clone all their biobricks with these 20 bp overlaps, and then in one Gibson reaction                stitch them all together into a plasmid, ready for transformation.
Parts in Registry:
BioBrick
Name
Sequence
BBa_K2667002
GibsonBrick 1 Fw
GCACTGAAGGTCCTCAATCGCGAATTCGCGGCCGCTTCTAG
BBa_K2667003
GibsonBrick 1 Rv
TACTAGTAGCGGCCGCTGCAGGCACTGAAGGTCCTCAATCGC
BBa_K2667004
GibsonBrick 2 Fw
CTGACCTCCTGCCAGCAATAGGAATTCGCGGCCGCTTCTAG
BBa_K2667005
GibsonBrick 2 Rv
TACTAGTAGCGGCCGCTGCAGCTGACCTCCTGCCAGCAATAG
BBa_K2667006
GibsonBrick 3 Fw
CTATTGCTGGCAGGAGGTCAGGAATTCGCGGCCGCTTCTAG
BBa_K2667007
GibsonBrick 3 Rv
TACTAGTAGCGGCCGCTGCAGCTATTGCTGGCAGGAGGTCAG
BBa_K2667008
GibsonBrick 4 Fw
TCTTAGGTGGCAGCGAACGAGGAATTCGCGGCCGCTTCTAG
BBa_K2667009
GibsonBrick 4 Rv
TACTAGTAGCGGCCGCTGCAGTCTTAGGTGGCAGCGAACGAG
Fragments should be amplified in the following order:
Fragment 1: Custom Vector Primer 1 +  GibsonBrick 1 Rv.
Fragment 2: GibsonBrick 1 Fw + GibsonBrick 2 Rv.
Fragment 3: GibsonBrick 2 Fw + GibsonBrick 3 Rv.
Fragment 4: GibsonBrick 3 Fw + GibsonBrick 4 Rv.
Fragment 5: GibsonBrick 4 Fw + Custom Vector Primer 2.
The Custom vector primers should contain the Prefix and Suffix respectively, along with                 homologous regions that will allow for assembly with the Vector your team is using.
