<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HebrewU_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HebrewU/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HebrewU HujiGEM 2018</TITLE></P><DIV class="w3-blue-grey"><UL id="ddmenu_hebrewu"><LI style="float:left;margin-top:12px;padding-left:15px;">Hebrew</LI><LI><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Software">MOOLTi</A></LI></UL><LI><A>Lab</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Design">Yeast Design</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Plant">Plant Design</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/InterLab">Inter-lab</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Ethics">Ethics</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Global_aspects">Global Aspects</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Public_engagment">Public Engagment</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Open_Source">Open Source</A></LI><LI><A>Team</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Photo_Gallery">Photo Gallery</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Judging">Judging</A></LI><DIV class="huji_smallnav" id="huji_shownav"><BUTTON class="huji_small_navbar">Project</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Description">Description</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Model">Model</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Demonstrate">Results</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Parts">Parts</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Software">MOOLTi</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Lab</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Design">Yeast Design</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Plant">Plant Design</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Notebook">Notebook</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/InterLab">InterLab</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Safety">Safety</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Protocols">Protocols</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Human Practice</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Human_Practices">Human Practice - Main Page</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Ethics">Ethics</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Global_aspects">Global Aspects</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Public_Engagment">Public Engagment</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Open_Source">Open Source</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Team</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Members">Members</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Attributions">Attributions</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Collaborations">Collaborations</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Photo_Gallery">Photo Gallery</A></BUTTON></DIV><BUTTON class="huji_small_navbar"><A href="https://2018.igem.org/Team:HebrewU/Judging">Judging</A></BUTTON></DIV><DIV class="w3-blue-gray"><DIV class="w3-center w3-animate-left"><DIV class="content"><P style="padding-left:150px;padding-right:150px;text-align:justify;line-height:1.5;color:white;"> 
        Our team has created an exploratory model of a metabolic pathway for the enzymatic degradation of TCDD. This model served as the foundation of our project design and the backbone for our lab work. We completed in depth research on enzymes families relevant to TCDD degradation, narrowing down contenders to the enzymes we would eventually use in the lab. While building this model, we utilized a multitude of tools including, but not limited to biochemical analysis, phylogenetic comparison and pathway mapping. Through these tools, we compiled a comprehensive list of enzymes, is strongly rooted in prior academic studies. </P></DIV><DIV style="text-align:center"><DIV class="content"><P style="padding-left:150px;padding-right:150px;text-align:justify;line-height:1.5;color:white;"> 
       The final pathway we tested in the lab, consists of dehalogenase, dioxygenase and hydrolase enzymes. Additionally, Cyp 450 and Lignin Peroxidase enzymes were researched, enzymes have been previously found to breakdown TCDD. Despite that neither of these enzymes catalyze a specific step in the final engineered pathway, both are excellent candidates for TCDD Degradation. 
		</P></DIV></DIV><DIV id="huji_model_1" class="w3-modal"><DIV class="w3-modal-content w3-animate-zoom"><HEADER class="w3-container w3-blue"><SPAN onclick="document.getElementById('huji_model_1').style.display='none'" class="w3-button w3-display-topright">×</SPAN><H2 class="w3-center">Haloacid dehalogenase </H2></HEADER><DIV class="w3-container"><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">Dehalogenases currently reported are involved in the cleavage of P-X and C-X bonds in the Brenda database. More than 90% of dehalogenases cleave C-X bonds. Such C-X bonds mainly exist in the halocarbons, halohydrins, and haloacids and their derivatives.  2-Haloacid dehalogenases are phylogenetically classified into two groups, those acting on L-2 haloacids, and those acting on D-haloacting on D-haloacids.<SUP>1</SUP></P><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                        According to the Fig. mechanism, a carboxylate group of the enzyme acts as a nucleophile to attack the a-carbon of L-2-haloalkanoic acid, leading to the formation of an ester intermediate. This is hydrolyzed by an attack of the water molecule activated by a basic amino acid residue of the enzyme. Alternatively, water is activated by a catalytic base of the enzyme and directly attacks the a-carbon of L-3-haloalkan6ic acid to displace the halogen atom (Fig. 2B). <SUP>2</SUP></P><H2 class="w3" style="color:black;padding-left:90px;">Further Reading:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                        1.<A href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1751-7915.2009.00155.x">&quot;Sequence‐ and activity‐based screening of microbial genomes for novel dehalogenases&quot;; Wing Yiu Chan et. al, Microbial Biotechnology Volume 3, Issue 1
</A>

                        2.<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4899344/">&quot;l-2-Haloacid dehalogenase (DehL) from Rhizobium sp. RC1&quot;; Aliyu Adamu et. al, SpringerPlus. 2016;5(1):695</A></P><H2 class="w3" style="color:black;padding-left:90px;">References:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                        1. <A href="https://www.nature.com/articles/s41598-017-19050-x">&quot;Insights into the molecular mechanism of dehalogenation catalyzed by D-2-haloacid dehalogenase from crystal structures&quot;; Yayue Wang, Yinghui Liu &amp; Song Xue. Scientific Reportsvolume 8, Article number: 1454 (2018) 
                         </A>            2. <A href="https://cwru.pure.elsevier.com/en/publications/bacterial-2-haloacid-dehalogenases-structures-and-catalytic-prope-2">&quot;Bacterial 2-haloacid dehalogenases: Structures and catalytic properties&quot; by Kenji Soda and Nobuyoshi Esah. Pure and Applied Chemistry, Volume 68, Issue 11, Pages 2097–2103 (1996). 
                         </A></P></DIV></DIV></DIV><DIV id="huji_model_2" class="w3-modal"><DIV class="w3-modal-content w3-animate-zoom"><HEADER class="w3-container w3-purple"><SPAN onclick="document.getElementById('huji_model_2').style.display='none'" class="w3-button w3-display-topright">×</SPAN><H2 class="w3-center">Cytochrome P450</H2></HEADER><DIV class="w3-container"><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5"> Collectively, the involvement of P450s has been proven in the metabolism of aliphatic, alicyclic, and aromatic molecules in reactions resulting in hydroxylation, epoxidation, dealkylation, sulfoxydation, deamination, desulphuration, dehalogenation, and N-oxide reduction. The majority of P450s catalyze the reactions after interacting with one or more protein components which transfer electrons from NADH or NADPH to the P450s, while some of them do not require any additional protein components to achieve the reductive activation of oxygen. While bacterial CYPs are water-soluble, mammalian CYPs are bound to either ER or mitochondrial membranes. <SUP>1,2</SUP></P><H2 class="w3-center" style="color:black">Mammalian CYP450:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                       Though human CYP's showed the ability to breakdown PCD's, it appeared that they were unable to utilize 2,3,7,8 TCDD as substrate. By contrast, CYP450 deriving from canines have showed some ability to catalyze  hydroxylation of 2,3,7,8-TCDD. In 2003, an article published by Kyoto university, described a single amino acid mutation [F240A] which allowed a rat CYP enzyme to catalyze TCDD dechlorination in E. coli.<SUP>3</SUP></P><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                       In vivo studies suggest that the CYP-dependent metabolism includes multiple reactions such as hydroxylation at an unsubstituted position, hydroxylation with migration of a chloride substituent, hydroxylation with elimination of a chloride substituent, and opening of the dioxin ring. All of these reactions appear to be detoxication reactions of PCDDs. 
                       
                       Aromatic hydroxylation reactions of arenes, 7, are catalyzed by P450. The initial product is a reactive arene epoxide, 8, which then undergoes an interesting rearrangement (NIH shift) to yield a reactive keto 2,4-diene intermediate, 11. This reactive intermediate rapidly undergoes a tautomeric shift of a proton to restore aromaticity. 
                        </P><DIV align="center"><BUTTON type="button" class="w3-button w3-teal" data-toggle="collapse" data-target="#Results_3">Further Reading:</BUTTON></DIV><DIV id="Results_3" class="collapse"><H2 class="w3-center" style="color:black">Plant CYP450:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                       Cytochrome P450 monooxygenases (P450s) represent the largest enzyme family of the plant metabolism. Plants typically devote about 1% of the protein-coding genes for the P450s to execute primary metabolism and also to perform species-specific specialized functions including metabolism of the triterpenes. The members of the subfamilies CYP51G, CYP85A, CYP90B-D, CYP710A, CYP724B, and CYP734A are generally conserved across the plant kingdom to take part in plant primary metabolism related to the biosynthesis of essential sterols and steroid hormones. However, the members of the subfamilies CYP51H, CYP71A,D, CYP72A, CYP81Q, CYP87D, CYP88D,L, CYP93E, CYP705A, CYP708A, and CYP716A,C,E,S,U,Y are required for the metabolism and might perform species-specific functions including chemical defence toward specialized pathogens.<SUP>4</SUP></P><H2 class="w3-center" style="color:black">Fungal CYP450:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                      Cytochrome P450 monooxygenases of fungi are involved in many essential cellular processes and play diverse roles. The enzymes catalyze the conversion of hydrophobic intermediates of primary and secondary metabolic pathways, detoxify natural and environmental pollutants and allow fungi to grow under different conditions. Fungal genome sequencing projects have enabled the annotation of several thousand novel cytochromes P450, many of which constitute new families.  Previous reports demonstrated that the white-rot fungus P. chrysosporium has been shown to possess biodegradative capabilities 2,3,7,8-TCDD. The degradation of more highly chlorinated DDs by P. chrysosporium has also been demonstrated. Based on the fact that P. chrysosporium genome contains 148 CYP genes, it is possible to assume that some P. chrysosporium CYPs are involved in the metabolism of PCDDs. Unfotunately, The specific 2,3,7,8-TCDD-metabolizing enzyme has not yet discovered  in P. chrysosporium.<SUP>2,5</SUP></P><H2 class="w3-center" style="color:black">Prokaryotic CYP450:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                      Bacterial CYP102A1 is a water-soluble fusion protein between CYP and NADPH-P450 reductase containing FAD and FMN. Interestingly, CYP and the reductase domains of CYP102A1 are significantly homologous with animal CYP and its reductase, suggesting it maybe the eukaryotic origin of CYP102A1 (Omura 2010).  These cytochromes are involved in the synthesis of secondary metabolites, such as antibiotics, in the utilization of hydrophobic low molecular substrates, such as alkanes and aromatic carbohydrates. These enzymes also have monooxygenase activity towards different organic low molecular substrates. Cytochromes C450 play an important role in the degradation of environmental toxins and mutagens. The hydroxylation activities of these enzymes towards PCDDs are much lower than those of mammalian CYPs.<SUP>1</SUP></P><H2 class="w3" style="color:black;padding-left:90px;">References:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">   1. <A href="https://link.springer.com/article/10.1007/s00253-010-2765-2">&quot;Enzyme systems for biodegradation of polychlorinated dibenzo-p-dioxins&quot;; Toshiyuki Sakaki et. al, Applied Microbiology and Biotechnology September 2010, Volume 88, Issue 1, pp 23–30. 
                         </A>
                        2. <A href="https://www.ncbi.nlm.nih.gov/pubmed/20619366">&quot;Cytochrome P450 enzymes in the fungal kingdom&quot;; Crešnar B et. al, Biochim Biophys Acta. 2011 Jan;1814(1):29-35. 
                         </A>
                          3. <A href="https://link.springer.com/article/10.1007/s00253-005-0286-1">&quot;Biodegradation of dioxins by recombinant Escherichia coli expressing rat CYP1A1 or its mutant&quot;; Raku Shinkyo, Applied Microbiology and Biotechnology, September 2006, Volume 72, Issue 3, pp 584–590.
                         </A>
                          4. <A href="https://www.frontiersin.org/articles/10.3389/fpls.2017.01886/full">&quot;Triterpene Structural Diversification by Plant Cytochrome P450 Enzymes&quot;;Sumit Ghosh, Front. Plant Sci., 09 November 2017 
                         </A>
                          5. <A href="https://www.sciencedirect.com/science/article/pii/S1087184597910215">&quot;Cytochrome P450 Enzyme Systems in Fung&quot;; Hans J.M et. al, Fungal Genetics and Biology Volume 23, Issue 1, February 1998, Pages 1-17 
                         </A></P></DIV></DIV></DIV></DIV><DIV id="huji_model_3" class="w3-modal"><DIV class="w3-modal-content w3-animate-zoom"><HEADER class="w3-container w3-light-blue"><SPAN onclick="document.getElementById('huji_model_3').style.display='none'" class="w3-button w3-display-topright">×</SPAN><H2 class="w3-center">Dioxygenase </H2></HEADER><DIV class="w3-container"><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">Many of these reactions have little or no precedent in organic chemistry. Di-oxygenase and mono-oxygenase enzymes that are able to activate dioxygen are divided into oxidases, which use oxygen as an oxidant, and reduce dioxygen to hydrogen peroxide or water; and oxygenases, which incorporate oxygen atoms from dioxygen into the product(s).Dioxygenases incorporate both atoms of oxygen into the product(s).  </P><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                        Oxidization of Dibenzofuran, a dioxin like molecule we used in the lab1 <SUP>1</SUP>. 

Most dioxygenase enzymes require a metal cofactor, which is most often iron(II or III). Dioxygenases primarily oxidize aromatic compounds and, therefore, have applications in environmental remediation. Dioxygenases serve as part of nature’s strategy for degrading aromatic molecules in the Environment. They are often found in the soil bacteria as well as almost all higher plants. Dioxygenase enzymes are used for a multitude of separate steps in the pathway described.
						</P><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
Proposed initial dioxygenolytic attack on the tested CDFs and CDDs by DFDO and CARDO. Panel 1, 2-CDF, 5′-chloro-2,2′,3-THB (reaction A) and 5-chloro-2,2′,3-THB (reaction B); panel 2, 2,8-DCDF, 5,5′-dichloro-2,2′,3-THB (reaction A); panel 3, 2-CDD, 5′-chloro-2,2′,3-THDE (reaction A), 4′-chloro-2,2′,3-THDE (reaction A′), 5-chloro-2,2′,3-THDE (reaction B), and 4-chloro-2,2′,3-THDE (reaction B′); panel 4, 2,3-DCDD, 4′,5′-dichloro-2,2′,3-THDE (reaction A) and 4,5-dichloro-2,2′,3-THDE (reaction B); panel 5, 2,7-DCDD, 4,5′-dichloro-2,2′,3-THDE (reaction A) and 4′,5-dichloro-2,2′,3-THDE (reaction B); panel 6, 1,2,3-TCDD, 4′,5′,6′-trichloro-2,2′,3-THDE (reaction A), 3′,4′,5′-trichloro-2,2′,3-THDE (reaction A′), and 4,5,6-trichloro-2,2′,3-THDE (reaction B). Possible sites attacked by angular dioxygenases are indicated by A or A′ on the nonsubstituted rings and by B or B′ on the halogen-substituted rings, with the exception of those in panels 2 and 5. ND, not detected. <SUP>2</SUP> 
Numerous studies have been done on a number of Dioxygenase enzymes, and their ability to degrade dioxins:
</P><H2 class="w3" style="color:black;padding-left:90px;">Further Reading</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                     1.<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC206875/">&quot;Characterization of 2,2',3-Trihydroxybiphenyl Dioxygenase, an Extradiol Dioxygenase from the Dibenzofuran- and Dibenzo-pDioxin-Degrading Bacterium Sphingomonas sp. Strain RW1&quot;;B Happe et. al, J Bacteriol. 1993 Nov; 175(22): 7313–7320 </A>
                     2.<A href="https://pdfs.semanticscholar.org/b33d/229d6320620e843a0ab71562c66a38b71075.pdf"> &quot;Molecular Bases of Aerobic Bacterial Degradation of Dioxins: Involvement of Angular Dioxygenases&quot;; H. Nojiri et. al, Biosci. Biotchnol. Biochem.,66 (10) 2001-2016,2002 </A>
                      3. <A href="https://link.springer.com/article/10.1007/s00203-005-0045-9"> &quot;Isolation and characterization of a gene cluster for dibenzofuran degradation in a new dibenzofuran-utilizing bacterium, Paenibacillus sp. strain YK5&quot;; T. Iida, Archives of Microbiology January 2006, 184:305. </A>
                        4.<A href="http://www.craftychemist.net/inorganic/wp-content/uploads/2010/02/tetrahedron2003v59p7075.pdf"> &quot;Dioxygenase enzymes: catalytic mechanisms and chemical models&quot;; T.D.H. Bugg, Tetrahedron 59 (2003) 7075–7101. </A></P><H2 class="w3" style="color:black;padding-left:90px;">References</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
 					 1.<A href="https://www.semanticscholar.org/paper/Dibenzofuran-4,4a-dioxygenase-from-Sphingomonas-sp.-B%C3%BCnz-Cook/3d6600bf8164f0d6570ca9d7208f865607cdd631">&quot;Dibenzofuran 4,4a-dioxygenase from Sphingomonas sp. strain RW1: angular dioxygenation by a three-component enzyme system&quot;; Patricia V. Bünz et. al, Journal of bacteriology 175 20 (1993): 6467-75.7313–7320 </A>
                     2.<A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC93062/"> &quot;Degradation of Chlorinated Dibenzofurans and Dibenzo-p-Dioxins by Two Types of Bacteria Having Angular Dioxygenases with Different Features&quot;;Hiroshi Habec et. al, Appl Environ Microbiol. 2001 Aug; 67(8): 3610–3617. </A></P></DIV></DIV></DIV><DIV id="huji_model_4" class="w3-modal"><DIV class="w3-modal-content w3-animate-zoom"><HEADER class="w3-container w3-green"><SPAN onclick="document.getElementById('huji_model_4').style.display='none'" class="w3-button w3-display-topright">×</SPAN><H2 class="w3-center">Hydrolases </H2></HEADER><DIV class="w3-container"><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">Hydrolases are hydrolytic enzymes, biochemical catalysts that use water to cleave chemical bonds, usually dividing a large molecule into two smaller molecules. Examples of common hydrolases include esterases, proteases, glycosidases, nucleosidases, and lipases. Hydrolases carry out important degradative reactions in almost all organisms. These enzymes have a catalyze reactions in a multitude of biological processes including digestion and neurotransmitter degradation in animals, bacterial cell-wall growth, and extracellular degradation of cellulose, xylan, and chitin, in fungi.<SUP> 1,2 </SUP></P><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                       General scheme Hydrolase catalyzed reactions. <SUP>3</SUP></P><H2 class="w3" style="color:black;padding-left:90px;">Further Reading:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                     1. <A href="https://www.ncbi.nlm.nih.gov/pubmed/24118890">&quot;Shotgun proteomics suggests involvement of additional enzymes in dioxin degradation by Sphingomonas wittichii RW1&quot;; EM Hartmen, Environ Microbiol. 2014 Jan;16(1):162-76. </A>
                     2. <A href="https://www.nature.com/articles/s41598-017-00258-w">&quot;Distribution and diversity of enzymes for polysaccharide degradation in fungi&quot;; R. Berlemont, Scientific Reportsvolume 7, Article number: 222 (2017).
 </A></P><H2 class="w3" style="color:black;padding-left:90px;">References:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                     1. Devlin, T. M. (2002). Textbook of Biochemistry, 5th edition. New York: John Wiley.  
                     2. <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3986426/">&quot;The role of hydrolases in bacterial cell-wall growth&quot;; T.K. Lee et. al, Curr Opin Microbiol. 2013 Dec; 16(6): 760–766.</A>  
                     3. <A href="https://pubs.rsc.org/en/content/articlehtml/2003/ob/b302109b">&quot;Reagents for (ir)reversible enzymatic acylations&quot;; U Hanefeild, Organic &amp; Biomolecular Chemistry, Issue 14, 2003.</A></P></DIV></DIV></DIV><DIV id="huji_model_5" class="w3-modal"><DIV class="w3-modal-content w3-animate-zoom"><HEADER class="w3-container w3-purple"><SPAN onclick="document.getElementById('huji_model_5').style.display='none'" class="w3-button w3-display-topright">×</SPAN><H2 class="w3-center">Peroxidase </H2></HEADER><DIV class="w3-container"><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">White rot fungi, the fungi responsible for the biodegradation of lignin in wood, are remarkable in their ability to degrade a wide variety of environmental pollutants. Insoluble, generally very recalcitrant chemicals are mineralized by the fungi. The fungi also mineralized (oxidized to CO2) some chemicals that are already highly oxidized. In general it is thought that this biodegradative ability is related to the ability of these fungi to degrade lignin. This ability, which is unique to white rot fungi, is thought to be dependent on a family of peroxidases secreted by the fungi; however, a purely oxidative, peroxidase- based system cannot be used to completely degrade lignin. Lignin is highly oxidized so it is difficult to oxidize further. Lignin is a complex heteropolymer with no stereochemical regularity, due at least in part to the free radical mechanism of synthesis. Lignin biodegradation must therefore involve a nonspecific and non-stereoselective mechanism.  

  
                        
                       The lignin peroxidases are somewhat unique in that they have higher oxidation potentials (- -1.35 V) than do most peroxidases ( -0.8 V). In this way these enzymes have a somewhat greater range of chemicals that they can oxidize. Due to this none specific, and highly effective oxidation abilities, Lignin peroxidase is an excellent candidate for bioremediation- not only for dioxins, but a wide range of aromatic pollutants.<SUP>1</SUP></P><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                        The C-O--C bond cleavage occurs via the nucleophilic attack of water on a cation rather than via the scavenging of a radical by molecular 02. The first step in this proposed mechanism is the one-electron oxidation of I by the oxidized enzyme intermediate LiP compound I, resulting in the formation of the aryl cation radical A. The LiP oxidation of a variety of nonphenolic aromatics to their corresponding aryl cation radicals has been well established. Indeed, the LiP oxidation of the related nonchlorinated compound dibenzop-dioxin to a cation radical has been reported. However, DCDD is considerably less soluble than dibenzo-p-dioxin. Furthermore, chloride is a much better leaving group than hydrogen; therefore, the aryl cation radical A is probably short-lived. For these reasons, the cation radical A was not detected during the oxidation of DCDD by LiP. 
                        Attack of H20 at the cation would result in the loss of chloride and the formation of the carbon-centered radical intermediate B. One-electron oxidation of B by LiP or MnP would result in the formation of the cation intermediate C. Attack of H20 on intermediate C would lead to the first C--O-C bond cleavage and the formation of the quinone intermediate D. Subsequent oxidation of the phenolic function of D would generate the phenoxy radical E, which is in resonance with the carbon-centered radical E'. Oxidation of E' by either LiP or MnP would yield the cation F. Finally, attack of H20 on the cation F would result in the cleavage of the second C-O-C bond and generation of 4-chloro-1,2-benzoquinone (V) and 2-hydroxy-1,4-benzoquinone (VIII). In the metabolic pathway, both chlorine atoms are removed from the aromatic rings as chloride before ring cleavage takes place. This is advantageous because the possible formation of toxic chlorinated aliphatics is avoided. Indeed, peroxidasecatalyzed oxidative dechlorination followed by reduction of the quinone products results in the introduction of phenolic groups, which facilitate subsequent ring opening. Similar peroxidase-catalyzed oxidative dechlorinations of polychlorinated phenols have been reported.  We propose that a similar metabolic pathway is probably involved in the fungal degradation of TCDD and have initiated studies to examine its metabolism.<SUP>2</SUP></P><H2 class="w3" style="color:black;padding-left:90px;">Further Reading:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                     1. <A href="https://www.sciencedirect.com/science/article/pii/S014102290400153X">&quot;Bioremediation of 2,3,7,8-Tetrachlorodibenzo-p-Dixoin in Soil by Fungi Screened from Nature Potential of extra cellular enzymes in remediation of polluted soils: a review&quot;; L. Gianfreda, Enzyme and Microbial Technology Volume 35, Issue 4, 1 September 2004, Pages 339-354. </A>
                     2. <A href="https://ehp.niehs.nih.gov/doi/pdf/10.1289/ehp.95103s459">&quot;Mechanisms of Degradation by White Rot Fungi&quot;;S.D. Aust, Environmental Health Perspectives, Vol. 103, Supplement 5: Biodegradation (Jun., 1995), pp. 59-61.
 </A></P><H2 class="w3" style="color:black;padding-left:90px;">References:</H2><P style="padding-left:90px;padding-right:90px;text-align:justify;line-height:1.5">
                     1. <A href="http://science.sciencemag.org/content/228/4706/1434">&quot;Oxidation of persistent environmental pollutants by a white rot fungus&quot;; JA Bumpus et. al, Science  21 Jun 1985: Vol. 228, Issue 4706, pp. 1434-1436.</A>  
                     2. <A href="https://jb.asm.org/content/jb/174/7/2131.full.pdf">&quot;Degradation of 2,7-Dichlorodibenzo-p-Dioxin by the Lignin Degrading Basidiomycete Phanerochaete chrysosporium &quot;; K Valli et. al, Journal of Bacteriology, Apr. 1992, p. 2131-2137</A></P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>