<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HebrewU_Parts skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HebrewU/Parts</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HebrewU HujiGEM 2018</TITLE></P><DIV class="w3-blue-gray"><UL id="ddmenu_hebrewu"><LI style="float:left;margin-top:12px;padding-left:15px;">Hebrew</LI><LI><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Software">MOOLTi</A></LI></UL><LI><A>Lab</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Design">Yeast Design</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Plant">Plant Design</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/InterLab">Inter-lab</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Ethics">Ethics</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Global_aspects">Global Aspects</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Public_engagment">Public Engagment</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Open_Source">Open Source</A></LI><LI><A>Team</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Photo_Gallery">Photo Gallery</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Judging">Judging</A></LI><DIV class="huji_smallnav" id="huji_shownav"><BUTTON class="huji_small_navbar">Project</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Description">Description</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Model">Model</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Demonstrate">Results</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Parts">Parts</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Software">MOOLTi</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Lab</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Design">Yeast Design</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Plant">Plant Design</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Notebook">Notebook</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/InterLab">InterLab</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Safety">Safety</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Protocols">Protocols</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Human Practice</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Human_Practices">Human Practice - Main Page</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Ethics">Ethics</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Global_aspects">Global Aspects</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Public_Engagment">Public Engagment</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Open_Source">Open Source</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Team</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Members">Members</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Attributions">Attributions</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Collaborations">Collaborations</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Photo_Gallery">Photo Gallery</A></BUTTON></DIV><BUTTON class="huji_small_navbar"><A href="https://2018.igem.org/Team:HebrewU/Judging">Judging</A></BUTTON></DIV><DIV class="w3-center w3-animate-bottom"><BUTTON class="w3-button w3-padding-large w3-large w3-margin-top" style="background-color:#D1F2EB;border-radius: 12px;width:100px;"><A href="#Gal_main_toggle">Gal 1/10</A></BUTTON>  <BUTTON class="w3-button w3-padding-large w3-large w3-margin-top" style="background-color:#D1F2EB;border-radius: 12px;width:100px;"><A href="#Gibson_main_toggle">Gibson brick</A></BUTTON><DIV id="Gal_main_toggle" class="w3-container w3-content"><H1 class="w3-center">Gal 1/10</H1><P class="w3-justify" style="padding-right:30px;padding-left:30px;"> 
              GAL 1- GAL 10 is a divergent promoter region of Saccharomyces cerevisiae, allowing it to regulate two genes simultaneously. 
              Each complimentary strand promotes translation in opposing directions. 
              Galactose induces transcription, while Glucose serves as a repressor. 
              The promoter is inducible by as low as 2% Galactose the in medium. </P><P class="w3-justify" style="padding-right:30px;padding-left:30px;"> 

            It is particularly useful for measuring protein-protein interactions, expressing proteins with two subunits, 
            or can simply be used to effectively cut the number of plasmids required  for cloning in half, as two genes can be cloned into a single plasmid.
    
            To validate this part we performed Real-Time PCR on yeast with our Leu-44a vector,
            containing two subunits of the same protein- 44a-Dioxygenase (44a).
            This enzyme contains to subunits: 44a Large Subunit (LS) and 44a Small Subunit (SS). 
            We used Leu2 as a regulating gene, as it is found on the same plasmid, 
            but its transcription is not dependent on the Gal 1/10 Promoter. 
            
            We grew 2 strains of yeast in SD media (containing glucose) for 36 hours. The strains were as follows:
            
                1. 44a- Containing the vector shown in the map above, 
                    with the LS transcription being promoted by Gal 10 and SS transcription being promoted by Gal1.
                 2. CTRL strain, containing a similar but &quot;empty&quot; vector. 
                    This vector also contains the Leu2 gene, and Gal 1/10 promoters, but does not have the 44a genes inserted.</P><P class="w3-justify" style="padding-right:30px;padding-left:30px;"> 
            After growth, be split each strain into 2 treatments:
            
                1. Induced:      SG media, containing 2% galactose in the media.  
                2. Uninduced: SD media, containing 2% glucose in the media. 
           
    
            As the primers for the RT-PCR are gene specific to the subunits, we expected to see no product in the CTRL vectors, in both treatments.
    
            The results of this experiment are detailed in the graph below. 
            After normalizing the raw expression data of our target genes to the expression of the Leu2 gene, we get the figures presented.
            
            In the induced treatment, we see expression rise more than 20 fold over the uninduced treatment.   
            Additionally, we see approximately equal expression between the uninduced treatment and the control group, 
            indicating the minor levels could be background noise caused by the non-specificity of primers, or primer-dimer amplification. 
            (We used SYBR, and not a probe, therefore the possibility of such background noise arises) 
        </P></DIV><DIV id="Gibson_main_toggle" class="w3-row-padding w3-padding-64 w3-container w3-content"><H1 class="w3-center">Gibson brick</H1><P class="w3-justify" style="padding-right:30px;padding-left:30px;"> 
              An obstacle many team face is finding a way to assemble multiple genetic components into a single 
              plasmid effectively and quickly. Though the biobrick standard addresses this issue, 
              we found that using newer, more advanced methods of can help save time and resources. 
              We, as many iGEM teams before us, decided to use Gibson Assembly. Since no restriction 
              is required for this method, the multi-cloning sites in the biobrick prefix and suffix were not particularly helpful. 
			  Gibson Assembly utilizes homologous overlaps on each DNA fragment allowed for their cloning into a single vector. 
			  This set of 8 oligos, that help to convert any biobrick into a Gibson Assembly ready fragment. 
              Each of our oligos is approximately 40 Bp long, consisting of two components:
               
                    
                    1. Biobrick prefix/suffix region- allowing them to be used as primers for any (RFC10) biobricks currently in the iGEM library.
                    
                    
                    2. 20 bp region with no secondary structures and about 50% GC content, allowing for extremely efficient Gibson Assembly. 
                    
              
			  This means that teams can, in one PCR, clone all their biobricks with these 20 bp overlaps, and then in one Gibson reaction 
              stitch them all together into a plasmid, ready for transformation. </P><P class="w3-justify" style="padding-right:30px;padding-left:30px;"> 
            Parts in Registry: <TABLE class="w3-table w3-striped w3-white" style="color:black;"><TBODY><TR><TH><STRONG>BioBrick</STRONG></TH><TH><STRONG>Name</STRONG></TH><TH><STRONG>Sequence</STRONG></TH></TR><TR><TD>
                BBa_K2667002
                </TD><TD>
                GibsonBrick 1 Fw
                </TD><TD>
                GCACTGAAGGTCCTCAATCGCGAATTCGCGGCCGCTTCTAG
                </TD></TR><TR><TD>
                BBa_K2667003
                </TD><TD>
                GibsonBrick 1 Rv
                </TD><TD>
                TACTAGTAGCGGCCGCTGCAGGCACTGAAGGTCCTCAATCGC
                </TD></TR><TR><TD>
                BBa_K2667004
                </TD><TD>
                GibsonBrick 2 Fw
                </TD><TD>
                CTGACCTCCTGCCAGCAATAGGAATTCGCGGCCGCTTCTAG
                </TD></TR><TR><TD>
                BBa_K2667005
              </TD><TD>
                GibsonBrick 2 Rv
                </TD><TD>
                TACTAGTAGCGGCCGCTGCAGCTGACCTCCTGCCAGCAATAG
                </TD></TR><TR><TD>
                BBa_K2667006
                </TD><TD>
                GibsonBrick 3 Fw
                </TD><TD>
                CTATTGCTGGCAGGAGGTCAGGAATTCGCGGCCGCTTCTAG
                </TD></TR><TR><TD>
                BBa_K2667007
                </TD><TD>
                GibsonBrick 3 Rv
                </TD><TD>
                TACTAGTAGCGGCCGCTGCAGCTATTGCTGGCAGGAGGTCAG
                </TD></TR><TR><TD>
                BBa_K2667008
                </TD><TD>
                GibsonBrick 4 Fw
                </TD><TD>
                TCTTAGGTGGCAGCGAACGAGGAATTCGCGGCCGCTTCTAG
                </TD></TR><TR><TD>
                BBa_K2667009
                </TD><TD>
                GibsonBrick 4 Rv
                </TD><TD>
                TACTAGTAGCGGCCGCTGCAGTCTTAGGTGGCAGCGAACGAG
                </TD></TR></TBODY></TABLE></P><P class="w3-justify" style="padding-right:30px;padding-left:30px;"> 

            Fragments should be amplified in the following order: 
            	
                	Fragment 1: Custom Vector Primer 1 +  GibsonBrick 1 Rv.
                    Fragment 2: GibsonBrick 1 Fw + GibsonBrick 2 Rv. 
                    Fragment 3: GibsonBrick 2 Fw + GibsonBrick 3 Rv. 
                    Fragment 4: GibsonBrick 3 Fw + GibsonBrick 4 Rv. 
                    Fragment 5: GibsonBrick 4 Fw + Custom Vector Primer 2.
                
                
                The Custom vector primers should contain the Prefix and Suffix respectively, along with
                homologous regions that will allow for assembly with the Vector your team is using. </P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>