<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HebrewU_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HebrewU/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>HebrewU HujiGEM 2018</TITLE></P><DIV class="w3-blue-gray"><UL id="ddmenu_hebrewu"><LI style="float:left;margin-top:12px;padding-left:15px;">Hebrew</LI><LI><A>Project</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Model">Model</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Demonstrate">Results</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Software">MOOLTi</A></LI></UL><LI><A>Lab</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Design">Yeast Design</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Plant">Plant Design</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/InterLab">Inter-lab</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Safety">Safety</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Ethics">Ethics</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Global_aspects">Global Aspects</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Public_engagment">Public Engagment</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Open_Source">Open Source</A></LI><LI><A>Team</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Team">Members</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Attributions">Attributions</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Collaborations">Collaborations</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Photo_Gallery">Photo Gallery</A></LI><LI><A href="https://2018.igem.org/Team:HebrewU/Judging">Judging</A></LI><DIV class="huji_smallnav" id="huji_shownav"><BUTTON class="huji_small_navbar">Project</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Description">Description</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Model">Model</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Demonstrate">Results</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Parts">Parts</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Software">MOOLTi</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Lab</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Design">Yeast Design</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Plant">Plant Design</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Notebook">Notebook</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/InterLab">InterLab</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Safety">Safety</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Protocols">Protocols</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Human Practice</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Human_Practices">Human Practice - Main Page</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Ethics">Ethics</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Global_aspects">Global Aspects</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Public_Engagment">Public Engagment</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Open_Source">Open Source</A></BUTTON></DIV><BUTTON class="huji_small_navbar">Team</BUTTON><DIV class="huji_small_navbar_panel"><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Members">Members</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Attributions">Attributions</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Collaborations">Collaborations</A></BUTTON><BUTTON class="b_huji_small_subnav"><A href="https://2018.igem.org/Team:HebrewU/Photo_Gallery">Photo Gallery</A></BUTTON></DIV><BUTTON class="huji_small_navbar"><A href="https://2018.igem.org/Team:HebrewU/Judging">Judging</A></BUTTON></DIV><DIV class="blue-gray"><DIV class="w3-center w3-animate-left"><DIV class="content"><P style="padding-left:150px;padding-right:150px;text-align:justify;"> 
       As the enzymes in the pathway come from multiple organisms and biological settings, it would be naïve to expect high specificity to the substrates used in the experiments. Thus monitoring the change of the parent molecule might be challenging. On the other hand, following newly created degradation products could prove feasible. However, these processes occur in the presence of many other yeast enzymes, and as such tracking, their, and only their, activity is extremely difficult. We had to be creative with the experiments we ran to examine the degradation process. As the chemicals we are attempting to degrade are toxic, an option for biological assays arises. To complement the biological assay, we employed a wide-reaching whole metabolic comparison through high-resolution chemical analysis. The experiment involved growing transgenic yeast in medium with TCDD-like chemicals and then examining two parameters: </P><P style="padding-left:200px;padding-right:150px;text-align:justify;"> 
1.	Growth-rate alteration monitored using O.D. measurements. 
2.  Changes in the chemical composition of the yeast monitored by Mass Spectrometry.   </P></DIV></DIV><DIV align="left"><DIV style="padding-left:150px;padding-right:150px;padding-top:15px;padding-bottom:15px;"><BUTTON type="button" class="w3-button" style="color:white;border-color:#EBEEF2;border-style: solid;" data-toggle="collapse" data-target="#Results_1"> + Control Groups</BUTTON></DIV><DIV id="Results_1" class="collapse" style="padding-left:150px;padding-right:150px;text-align:justify"><P style="text-align:justify">As with all experiments, the significance of the results is contingent on control groups. Every experiment run had three control groups for each strain tested (Table 1).  </P><DIV align="center"><I><B>Table 1:</B> Explanation and title of control groups for each experiment </I></DIV><P style="text-align:justify">
Each experiment involved yeast expressing catabolic enzymes treated with a compatible toxic compound (Treatment 1). By growing the yeast with the toxic chemical, we could then analyze their survival and growth to understand if the enzymes are indeed breaking down the compounds. To fully understand the extent of growth inhibition, this treatment was compared to the growth rate of yeast not treated with said chemical (Treatment 3); as well as not expressing the tested enzymes (Treatments 2 and 4) with and without the chemical, respectively. Treatment 2 gave us a proper understanding of how ‘Wild-Type’(WT) yeast would react to being exposed to these compounds.
</P><P style="text-align:justify">
An expression vector with no catabolic enzymes, as opposed to actual WT yeast, were used, so they could be grown in the same medium. As discussed on the <A href="https://2018.igem.org/Team:HebrewU/Design">yeast design</A> page, we used amino acid auxotrophic mutants as selection markers. This means we grow the yeast in a dropout media lacking a specific amino acid. This on its own is enough to alter growth rates, and as such yeast was transformed with the same vector (allowing for growth in the absence of amino acids) but without the enzymes for breaking down the compounds. In addition, these treatments also allowed for the measurement of the “expression load” of the enzymes. Clearly, gene expression comes at a biological cost. By comparing differences in growth- with and without the enzymes- not in the presence of toxic compounds the effect of this cost can be understood. 
  

As both the chemicals were dissolved in methanol, all treatment contained identical concentrations of methanol. This is because similar to the amino acid dropout, methanol in the medium also affects growth rates and metabolism of the yeast.
 

The team considered leveraging the ability to induce the GAL1/10 promoter as a control group but concluded that creating different media – one with glucose and one with galactose – could create other undesired changes in the yeast’s metabolism. As such, this would not give us an accurate comparison and empty vectors were therefore used as the control group.



With all of these controls in hand, we could begin our experiments. Because using TCDD in the lab was not possible due to safety reasons, we used two separate compounds to simulate the two stages of the pathway. To test the first stage of the pathway - dechlorination - 1,2,4,5-Tetracholobenzen (TCB) was used. After dechlorination, the second stage is the breakdown of the carbon backbone, similar in all dioxins. To test this stage, Dibenzofuran (DBF) – a non-chlorinated dioxin-like compound – was utilized.</P></DIV><DIV style="padding-left:150px;padding-right:150px;padding-top:15px;padding-bottom:15px;"><BUTTON type="button" class="w3-button" style="color:white;border-color:#EBEEF2;border-style: solid;" data-toggle="collapse" data-target="#Results_2"> + Biological Analysis</BUTTON></DIV><DIV id="Results_2" class="collapse w3-blue-gray" style="padding-left:150px;padding-right:150px;"><P><U><B>Chemically Induced Growth Rate Alteration:</B></U>  Though the chemicals used in our experiments were less toxic than TCDD, they are still recognized as harmful molecules. As such, it was reasoned that the presence of these molecules in the medium would inhibit or completely halt the growth of yeast. As some of the breakdown products of these compounds are also toxic (i.e. pyrocatechol), it was not clear if the enzymatic activity would reduce the inhibition or further decrease the growth rate. 

To measure growth, measurements were taken with a spectrophotometer to track absorbance at 600nm. By reading the absorbance regularly (every 5-10 hours during lag phase and every 2-4 hours during log phase) we could accurately track the growth rates of each yeast line for comparison. Each experiment was conducted in triplicate and repeated twice. </P><P>
To compare growth rates and discern if there were significant changes in growth, the following data analyses were conducted:</P><P style="padding-left:30px">
1.	Isolation of exponential growth phase. 
2.	Rate of change (slope) calculated for each sample. 
3.	Comparison of these rates using Two-way T-test for Equal Means.</P></DIV><DIV class="w3-blue-gray" style="padding-left:150px;padding-right:150px;padding-top:15px;padding-bottom:15px;"><BUTTON type="button" class="w3-button" style="color:white;border-color:#EBEEF2;border-style: solid;" data-toggle="collapse" data-target="#Results_3"> + Chemical Analysis </BUTTON></DIV><DIV id="Results_3" class="collapse w3-blue-gray" style="padding-left:150px;padding-right:150px;"><P>
    After testing yeast growth in the presence of these toxic compounds, the samples were analyzed to monitor changes in the chemical composition of the different treatments. To this end, cells were lysed, and total metabolites were extracted in methanol. Chemical and statistical analyses were conducted to survey the metabolic differences. Using chemical analysis, two parameters were tracked that would allow for quantifying the activity of the researched enzymes:
    </P><P style="padding-left:30px">
1.	Concentration changes of the compound added to the medium. 
2.	Concentration changes of the expected metabolites in the sample. </P><P>
    At first, High-Pressure Liquid Chromatography (HP-LC) was employed, but the molecules in question were not easily tracked by this method. This resulted in a switch to High-Resolution Liquid chromatography-mass spectrometry (HR-LC-MS), specifically using atmospheric pressure chemical ionization (APCI). APCI is a method of ionization which ionizes the sample through the addition of atoms, or small molecules, creating a charge that is read by the machine. The results produced are chemical compounds found in the sample, represented by molecular mass. The masses produced are accurate to 5 digits after the decimal- which increases accuracy in identifying chemical equations from the mass values. 
    
The LC-MS analysis was untargeted. This means the equipment was not directed to look for a specific molecule or mass; rather it processed all the available data from each sample. As it was not certain that the starting compound could be easily tracked, this method allowed unpredicted products of the compounds’ breakdown to be assessed. 

To sift through the thousands of compound masses delivered by these assays, Sieve and Compound Discoverer software was used to align the sample peaks and to produce the primary statistical analysis of the differences between treatments. Even after this help from software, there were thousands of outputted compounds. To narrow the field of compounds, the following criteria were outlined:</P><P style="padding-left:30px">
1.       Compounds with at least a 60% increase in samples containing both the compound and the enzymes, compared to the control groups. This would suggest that the compound is the result of the presence of both the enzymes and substrate.
2.       Statistical significance was determined using an alpha value of ≥ 0.05 for T-test. 
3.       Compounds with a molecular weight in the range of expected metabolites. 
4.       Chemical equations corresponding to anticipated metabolites with minor variations as a result of the ionization process. </P></DIV><DIV class="w3-blue-gray" style="padding-left:150px;padding-right:150px;padding-top:15px;padding-bottom:15px;"><BUTTON type="button" class="w3-button" style="color:white;border-color:#EBEEF2;border-style: solid;" data-toggle="collapse" data-target="#Results_4"> + 1,2,4,5-Tetracholobenzen Results</BUTTON></DIV><DIV id="Results_4" class="collapse w3-blue-gray" style="padding-left:150px;padding-right:150px;"><P>To examine whether 1,2,4,5-Tetracholobenzen (TCB) affected the growth rate of the yeast, we treated six subculture replicates (OD<SUB>600</SUB>~0.025): three with 0.45uM of TCB and three methanol controls. Monitoring the growth from the lag- through the linear-phase, we could not detect any differences between the growth rate of chemical treatment and the control medium (Figure 1) regardless of the presence of the enzyme. Statistical analysis of the differences between the slopes is presented in Table 2. </P><DIV align="center"><I><B>Table 2:</B> Two way T-test analysis for equal means to measure change in growth rates</I></DIV><DIV align="center"><I><B>Figure 1:</B> Linear phase growth rates for both yeast strains in 0.45[mM] TCB and Control media</I></DIV><P>
As there was no observed growth-inhibition, it was concluded that the IC50 for TCB is higher than its solubility in water and that its metabolic products are not toxic; or that there was no substantial enzymatic activity. Since TCB and its potential degradation products could not be monitored by this bioassay, we tested the chemical composition of the cells. 

The expected chemical reactions were the removal of chlorine molecules. As the benzene ring is a very stable structure, and the Dehalogenase does not act to dismantle it, the expected metabolites would be a 6-carbon ring with hydrogen molecules or OH groups replacing the Cl molecules. The chemical analysis showed both the increase and decrease of numerous compounds in the media (Figure 2) Following the parameters outlined in the chemical analysis section, ion masses possibly representing metabolites of TCB were found (Table 1).
 </P><DIV align="center"><DIV align="justify" style="width:80%"><I><B> Figure 2:</B> Differential Analysis Plot TCB Treatment: Each point represents the identification of specific ion mass. The green square represents a region of a ≥ 20% increase in the concentration of ions in the ‘Degrading Enzymes’ strain compared to the ‘Empty Vector’ strain (P value ≤ 0.05).
The red square represents a region of a ≥ 20% decrease in the concentration of ions in the ‘Degrading Enzymes’ strain compared to the ‘Empty Vector’ strain (P value ≤ 0.05).
 </I></DIV></DIV><DIV style="text-align:center"><I><B>Table 3:</B> LCMS results for TCB samples in range of relevant molecular weight. </I></DIV><P>
 Not all of the masses with relevant chemical equations could be accounted for due to a large number of enzymatic processes in yeast. Based on the range in which they fell, however, it is likely they are products of the dehalogenation of TCB. 


Additionally, some of the masses represent anticipated molecules produced by the metabolic pathway. For example, a molecular weight of 165.0763 represents the molecular equation: C<SUB>6</SUB>H<SUB>12</SUB>O<SUB>5</SUB>. One of the spatial structures this equation can take is Cyclohexanepentol (Figure 3a). This molecule could likely represent our original 6-carbon ring, with all 4 of its chlorine molecules replaced by hydroxy groups, and an additional hydroxy group added during ionization or protonation of an oxygen molecule already present. 


Moreover, the molecular weight 153.04249 is of interest, representing the molecular equation C<SUB>7</SUB>H<SUB>7</SUB>O<SUB>3</SUB>N. This is possibly our original benzene ring with hydroxy groups. Though a specific reaction which would introduce an N molecule was not anticipated, nitrogen is abundant in yeast cells (Figure 3b). Both of these molecules represent promising results. They point to the complete dechlorination of the Tetrachlorobenzene and suggest the possibility that this enzyme could help transform 2,3,7,8 TCDD into the non-chlorinated Dibenzo-p-dioxin. 
 </P><DIV align="center"><DIV style="width:80%"><I><B> Figures 3a and 3b:</B> Predicted degradation process of TCB in yeast. Possible products in (A) Cyclohexanepentol (B) 4-amino-3-hydroxy benzoic acid  </I></DIV></DIV></DIV><DIV class="w3-blue-gray" style="padding-left:150px;padding-right:150px;padding-top:15px;padding-bottom:15px;"><BUTTON type="button" class="w3-button" style="color:white;border-color:#EBEEF2;border-style: solid;" data-toggle="collapse" data-target="#Results_5"> + Dibenzofuran Results</BUTTON></DIV><DIV id="Results_5" class="collapse w3-blue-gray" style="padding-left:150px;padding-right:150px;"><P>
To examine Dibenzofuran’s (DBF) effect on yeast growth rate, we treated 12 subculture replicates (OD<SUB>600</SUB>~0.025), at 4 varying concentrations: Methanol Control (mock), 0.5, 1, and 1.5 mM. As shown in Figure 4, the effect was significant and dose-dependent.
</P><DIV align="center"><P style="width:70%"><I><B> Figure 4:</B> Yeast growth curve (A) Growth inhibition caused by varying concentrations of DBF in growth medium (B): Quantitative growth inhibition as a function of DBF concentration.</I></P></DIV><P>
Using the Control group as the “maximum potential” for growth under the given conditions, we measured inhibition as affected by DBF (Figure 5B). This graph is based on measurements after 48 hours of growth. 


With the above data, it was decided to run experiments with the enzymes at a concentration of 0.5 mM. This appears to be enough to inhibit growth, but not high enough to inhibit yeast growth even with 50% of the compound broken-down; such as could be the case at a higher dosage i.e. 1.5 mM.

When tested the growth rate of a yeast strain with degrading enzymes (DE) compared with to empty vector (EV) stains. We expected to see a change in growth rate as both substrate and degradation products may be toxic. As shown in Figure 6, the DE’s growth was further inhibited by the enzymes in the presence of DBF, meaning that the toxicity of the product exceed that of the substrate. </P><DIV align="center"><I><B>Table 4:</B> Two-way T-test analysis for equal means to measure the change in growth rates.</I></DIV><DIV align="center"><I><B>Figure 5:</B> Linear phase growth rates for both yeast strains in 0.5 [mM] DBF and Control media.</I></DIV><P>
It was discernible from the bio-assay that there was metabolic activity based on changes in growth rate. We then progressed to chemical analysis to better understand these processes. As with the TCB, we saw a number of changes in the medium (Figure 6). Employing a process similar to that described in the TCB section, we identified the compounds enriched in the treated sample (Table 5).    

</P><DIV align="center"><DIV style="width:65%"><P><I><B>Figure 6:</B> Differential Analysis Plot DBF Treatment: Each point represents the identification of a specific ion mass. The green square represents a region of a ≥ 20% increase in the concentration of ions in the ‘Degrading Enzymes’ strain compared to the ‘Empty Vector’ strain (P-value ≤ 0.05). The red square represents a region of a ≥ 20% decrease in the concentration of ions in the ‘Degrading Enzymes’ strain compared to the ‘Empty Vector’ strain (P-value ≤ 0.05). </I></P></DIV></DIV><DIV align="center"><I><B>Table 5:</B> LCMS results for DBF samples in range of relevant molecular weight.</I></DIV><P>
As these molecules consisted of more carbon atoms, anticipating the given structures for each mass became substantially harder, and our predictions less accurate. However, the masses 206.16661 and 137.03550 correlate closely with molecules in the designed pathway. 2,2',3-trihydroxybiphenyl and salicylic acid (a breakdown product of Dibenzofuran but not TCDD) weigh 202.2 and 138.1 Da, respectively. The replacement of a carbon molecule with oxygen or changes in the hydrogen content during ionization could account for these changes in mass.</P></DIV><DIV class="w3-blue-gray" style="padding-left:150px;padding-right:150px;padding-top:15px;padding-bottom:15px;"><BUTTON type="button" class="w3-button" data-toggle="collapse" style="color:white;border-color:#EBEEF2;border-style: solid;" data-target="#Discussion"> + Discussion</BUTTON></DIV><DIV id="Discussion" class="collapse w3-blue-gray" style="padding-left:150px;padding-right:150px;"><P>
The results here are preliminary but substantial. To further understand the chemical makeup of the breakdown products, continued Mass-Spec analysis is required. To do this we can employ a number of methods:</P><P style="padding-left:30px">
1.       Obtain pure chemical standards corresponding to the molecules we suspect as breakdown products. We can run these standards against our sample and see if similar profiles appear. 

2.       LC-MS/MS: we can break down specific molecules we suspect as metabolites and run them in Mass-Spec. These broken-down products have a very specific time signature. If we put together the molecules from this small window of time and are able to rebuild the molecule in question, it would strongly imply that we have identified the correct molecule.

3.       Use heavier molecules such as radioactive isotopes or Hydrogen-deuterium exchange, to see if there are shifts in the masses of the observed compounds.</P><P>
These methods would further strengthen our ability to correctly identify the breakdown molecules we are working with. Additionally, growth-curve experiments with more toxic chemicals, even using 2,3,7,8-TCDD itself could potentially show clearer results in the biological assay. We have completed the first stages of research, and have already transformed plants with the Haloacid Dehalogenase. Further examination is required as we continue to develop a genetically engineered solution to TCDD, chlorinated pollutants, and dioxins.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>