Team:ASTWS-China/Design
ASTWS-China
Project▾
Applied Design
Attributions
Demonstrate
Description
Design
Improve
Modeling
Experiments▾
Protocols
InterLab
Results
Team▾
Team
Collaborations
Human Practices▾
Silver
Gold Integrated
Judging Form
Design
First thing first, we start to develop our topic inspired by the article from 2018, March, Science. It mainly talked about the gene expression of methanobactin which is originated from methanotrophy, a kind of anaerobic bacteria which consumes methane as their only energy source, which triggers us the idea of using this gene to eliminate the heavy metal (copper) pollution in water-bodies. The property of Methanobactin shows it's not the best suit for solving the real world problems because as a chalkophore, it not only shows high affinity to copper but also attracts a variety of metals when there is no copper ion presents in the environment. This property means we couldn't put the engineered bacteria into natural environments, or it may cause more damage than before, such as eroding soils. With some advice from experts with our experience from visiting a local sewage treatment plant, we learned that we can only achieve the project's greatest potential by dividing tasks. We put all of our resources into two major parts, and at last, they may combine to make a more sophisticated final project. The first part is responsible for the detection of copper ion concentration in polluted water. The second part could chelate copper ions to prevent them from entering our food chain which is the modern way of preventing heavy metal poisoning.
Copper Detect 1.0
Our first approach to detect the copper ion concentration is based on the NJAU 2014 iGEM team's pCOPA promoter. In which could trigger the translation process if there is a copper concentration higher than the pCOPA's lowest detection concentration, which we will test in our experiment section. pCOPA is followed by bioJ(2 REFERENCE(K1679038)), which is a stabilizer for protein production. At last, a reporter, eRFP(k1357010 which includes an RBS), following the bioJ to report the number of copper ions in the media accordingly. The reason why we picked the eRFP protein rather than GFP from NJAU's gene product is that eRFP not only behaves like a fluorescent protein but also has the property of a chromoprotein. This improvement helps us to achieve a cheaper, less expensive, relatively accurate way of detecting the copper ion concentration in the field.
Copper Detect 2.0
The second approach is added with a logic gate of EXOR(Which means it will function if there is the presence of either input, not both.) [With the production of tetR, ptet will not be triggered to produce chromoprotein 2. It leads to the increasing intensity of red color from RFP in media. On the other hand, if the copper ion concentration in the media is not sufficient enough to start the pCOP A promoter, tetR will not be produced. At last, this would lead to the increase of GFP content. This working system makes our logic gate based on the concentration of copper ion concentration.]  In short, if the copper concentration is not detectable by pCOPA, then the promoter ptet will be triggered to produce another chromoprotein(E0840) and alert the observer.
Copper Treatment 1.0
Absorb 1.0 is specifically designed to put MbnABC gene in use. Starting with a strong promoter(K608002) to initiate the reaction automatically. Then followed by our target gene MbnABC. These series of genes(MbnABC) are responsible for producing the Methanobactin protein to attach to the copper ions in the media. Finally, attached with a reporter gene eRFP, which we have mentioned above.
Copper Treatment 2.0
Absorb 2.0 is the ultimate product of our project, which is composited of the pCOPA, bioJ & RBS pair, RFP from detection part. Followed by our target gene MbnABC. We take away the logic gate we used in the previous parts because we don't need that function in this situation.
References
Choi, D. W., Do, Y. S., Zea, C. J., McEllistrem, M. T., Lee, S., Semrau, J. D., . . . DiSpirito, A. A. (2006). Spectral and thermodynamic properties of Ag(I), Au(III), Cd(II), Co(II), Fe(III), Hg(II), Mn(II), Ni(II), Pb(II), U(IV), and Zn(II) binding by methanobactin from Methylosinus trichosporium OB3b. Journal of Inorganic Biochemistry,2150-2161. doi:10.1016/j.jinorgbio.2006.08.017
Hao, Z., Luo, H., Zhu, R., Zhu, J., Zhang, D., Zhao, B. S., . . . Chen, P. R. (2014). The multiple antibiotic resistance regulator MarR is a copper sensor in Escherichia coli. Nature Chemical Biology,10, 21-28. doi:10.1038
Kenney, G. E., & Rosenzweig, A. C. (2011). Chemistry and Biology of the Copper Chelator Methanobactin. ACS Chemical Biology,7(2), 260-268. doi:10.1021/cb2003913
Kenney, G. E., Dassama, L. M., Kelleher, N. L., & Rosenzweig, A. C. (2018). The biosynthesis of methanobactin. BIochemistry,1411-1416. Retrieved March 23, 2018.
Keown, W., Gary, J. B., & P. Stack, T. D. (2016). High‐valent copper in biomimetic and biological oxidations. J Biol Inorg Chem,289-305. Retrieved September 28, 2016.
Kim, H. J., Galeva, N., Larive, C. K., Alterman, M., & Graham, D. W. (2005). Purification and Physical -- Chemical Properties of Methanobactin: A Chalkophore from Methylosinus trichosporium OB3b. Biochemistry,5140-5148.
Kulczycki, E., Fowle, D. A., Knapp, C., Graham, D. W., & Roberts, J. A. (2007). Methanobactin-promoted dissolution of Cu-substituted borosilicate glass. Geobiology,5(3), 251-263. doi:10.1111/j.1472-4669.2007.00102.x
Li, L., Luo, L., Mu, X., Sun, T., & Guo, L. (2010). A reagentless signal-on architecture for electronic, real-time copper sensors based on self-cleavage of DNAzymes. Analytical Methods,2(6), 627. doi:10.1039/c0ay00176g
McDonald, I. R., & Murrell, J. C. (1997). The methanol dehydrogenase structural gene mxaF and its use as a functional gene probe for methanotrophs and methylotrophs. Applied and Environmental Microbiology,63(8), 3218-3224. Retrieved August, 1997.
Outten, F. W., Outten, C. E., Hale, J., & Ohalloran, T. V. (2000). Transcriptional Activation of anEscherichia coliCopper Efflux Regulon by the Chromosomal MerR Homologue, CueR. Journal of Biological Chemistry,275(40), 31024-31029. doi:10.1074/jbc.m006508200
Rensing, C., Fan, B., Sharma, R., Mitra, B., & Rosen, B. P. (2000). CopA: An Escherichia coli Cu(I)-translocating P-type ATPase. Proceedings of the National Academy of Sciences,97(2), 652-656. doi:10.1073/pnas.97.2.652
Stein, L. Y., Yoon, S., Semrau, J. D., DisSpirito, A. A., Crombie, A., Murrell, J. C., . . . Zeytun, A. (2010). Genome Sequence of the Obligate Methanotroph Methylosinus trichosporium Strain OB3b. Journal of Bacteriology,192(24), 6497.
Vita, N., Platsaki, S., Basle, A., Allen, S. J., Paterson, N. G., Crombie, A. T., . . . Dennison, C. (2016). A four-helix bundle stores copper for methane oxidation. Nature. Retrieved March 03, 2013.
Xin, J., Dong, J., Yan, C., Qiao, J., Liang, H., & Xia, C. (2011). 甲烷氧化菌素的产生和铜捕获作用. China Biotechnology,40-46. doi:10.13523/j.cb.20110808
Xing, J., Yan, M., Zhou, Q., Song, H., & Xia, C. (n.d.). 甲烷 氧化细菌的铜捕获机理. Journal of Molecular Catalysis(China).
Yamamoto, K., & Ishihama, A. (2005). Transcriptional Response of Escherichia coli to External Zinc. Journal of Bacteriology,187(18), 6333-6340. doi:10.1128/jb.187.18.6333-6340.2005
Zhou, Q., Xin, J., Zhang, Y., Dong, J., Song, H., & Xia, C. (2009). 甲烷氧化菌素的生物活性研究进展. Letters In Biotechnology,20(5), 723-725. doi:10.3969/j.issn.1009-0002.2009.05.035
Copyright © 2018 Team:ASTWS-China All Rights Reserved.
