"
Page
Discussion
View source
History
teams
Log in
 
Team:Tokyo-NoKoGen/g3dh
From 2014.igem.org
Project
Team
Result
Parts
Protocols
Notebook
Safety
Human Practice
Achievement
About G3DH
The enzyme, glucoside 3-dehydrogenase (E.C.1.1.99.13) or glucose-3-dehydrogenase (G3DH) catalyzes the oxidation of the C-3 hydroxyl group of the glucosides and converting them to corresponding 3-ketoglucosides (1). Because G3DH has wide substrate specificity, it can convert not only monosaccharides but disaccharides, including trehalose.
G3DH is composed of three subunits: catalytic subunit, cytochrome c subunit, and small subunit. The catalytic subunit has a flavin adenine dinucleotide (FAD) as cofactor, and the cytochrome c subunit is bound to the cytoplasmic membrane in the periplasm.
In our project, we used G3DH derived from Rhizobium tumefaciens EHA101, formerly known as Agrobacterium tumefaciens. The enzymatic activity of G3DH in this microorganism was first reported in 1967 (2). The gene encoding this enzyme was found from a putative proteins, which is homologous to G3DH derived from Halomonas sp. α-15 (3), which was reported to convert trehalose to 3,3’dkT (see below). The functional expression of R.tumefaciens derived putative enzyme confirmed that the gene encodes the G3DH complex (unpublished data from Sode laboratory, Tokyo univ. of Agric. and Technol.).
About 3,3'-diketotrehalose (3,3'-dkT)
3,3’-dkT is a novel trehalose derivative in which the third hydroxyl group of both glucose moieties are oxidized. It was already reported that 3,3-dkT showes an inhibitory effect toward the trehalase from pig-kidney and Bombyx mori (silkworm)(3).
Construction of Biobrick
We amplified G3DH gene from Rhizobium tumefacience by PCR. PCR products were inserted in pSB1C3 using standard cloning techniques. Original G3DH gene has two EcoRI restriction sites, which is incompatible with BioBrick parts. These restriction sites was removed without altering the codon by overlap extension PCR using designed primers(Fig.1).
Fig.1. The schematic image to remove internal EcoRI sites in G3DH.
G3DH gene fragments were amplified and three PCR products were ligated. Then, G3DH (removed illegal restriction sites) were ligated with four different promoters and double terminator (BBa_B0015)(Fig.2).
Fig 2. BioBrick parts containing G3DH.
One promoter is arabinose inducible (PBAD), and the others are constitutive promoters (Table.1).
Table 1. Promoter connected with G3DH genes.
Evaluation
The G3DH requires the maturation of cytochrome c subunit for the catalytic activity. Therefore, we transformed E. coli TOP 10 with the constructed plasmids and pEC86, which expresses cytochrome c maturation (CCM) enzymes (3).
This is the method of culturing and extraction of production (Fig. 3).
We cultured TOP10 transformed with below plasmids in LB medium containing 20 mM Trehalose. Phigh, medium-RBS-G3DH-DT and PBAD-RBS-G3DH-DT were cultured at 80 rpm, 37 ℃, for 24 hours and at 150 rpm, 37 ℃ respectively.
When OD660 achieves 0.6, 0.2 % arabinose was added to the medium for induction (PEC86,PBAD-RBS-G3DH-DT). The transformants were harvested, suspended in 10 mM sodium phosphate buffer (SPB) and pellet was obtained by centrifugation.
First, we investigated expression of G3DH by SDS-PAGE analysis. Second, we measured the glucose dehydrogenase activity of G3DH. Finally we tried to detect 3,3’-dkT by thin-layer chromatography (TLC).
Fig. 3 Evaluation of G3DH
SDS-PAGE
By SDS-PAGE analysis, there was the band which showed about 68 kDa on G3DH under the constitutive promoter Pmedium and Phigh, PBAD. Therefore, we confirmed the expression of G3DH (Fig. 4).
Fig. 4 SDS-PAGE analysis of G3DH expression.
Measurement of G3DH productivity
We measured the oxidase activity of G3DH by PMS-DCIP assay (Fig. 5). On this assay, we measured the decrease in absorbance of DCIP at 600 nm.  Substrates were glucose and trehalose.  PMS stands for phenazine methosulfate, and DCIP stands for 2,6-dichlorophenol-indophenol.
Fig.5 The scheme of PMS-DCIP assay.
Fig.6 Measurement of the G3DH productivity
Productivity is the enzyme activity(U) from 1L culture. Phigh had the highest productivity. It is reason that  we confirmed expression of Phigh-G3DH by SDS-PAGE. Both of PBAD-G3DH(+) and (-) had comparatively high productivity. Increase of PBAD(-) productivity results that regulation by non-arabinose was not enough. PBAD (+) was also seemed to express at insoluble fraction (misfolded), therefore we could not confirm activity improvement.
Trehalase inhibition measurement
This is the result of trehalase inhibition activity assay(Fig. 7). Each value of activity was normalized at the value of activity of empty vector which were not induced by arabinose.  Samples of G3DH which induced by Phigh and Pmedium activity was lower than that of empty vector.
We concluded that G3DH which induced by Phigh and Pmedium expressed G3DH. And G3DH converted trehalose to 3,3’-dkT.
Fig. 7 Trehalase inhibition activity assay
Reference
(1) K Kojima et al., (2001) Cloning and Expression of Glucose 3-Dehydrogenase from Halomonas sp. α-15 in Escherichia coli. Biochem Biophys Res Commun, 282, 21-27
(2) K Hayano et al., (1967) Purification und properties of 3-ketosucrose-forming enzyme from the cells of Agrobacterium tumefaciens. J. Biol. Chem., 242, 3665-3672
(3) K Sode et al., (2001) Enzymatic synthesis of a novel trehalose derivative, 3,3’-diketotrehalose, and its potential application as the trehalase enzyme inhibitor. FEBS Letters, 489, 42-45
(4) E Arslan et al., (1998) Overproduction of the Bradyrhizobium japonicum c-Type Cytochrome Subunits of the cbb3 Oxidase in Escherichia coli. Biochem Biophys Res Commun, 251, 744-747
>> BRENDA
Future work
We confirmed the production of trehalose and 3,3’-dkT respectively.  To produce 3,3’-dkT from glucose directly in E. coli, we are going to cotransform E. coli with 3 different plasmids which harbor OtsA+OtsB, G3DH and cytochrome c maturation enzymes, respectively.
In this study, we confirmed low productivity of 3,3’-dkT in E. coli. It might be caused by low amount of precursor (trehalose).  In our project, we used OtsA and OtsB derived from E. coli to produce trehalose, but E. coli and other microorganisms have different kind of pathway to produce trehalose. To increase the production of trehalose, we are planning to use other kind of enzyme to produce trehalose.
Further improvement
To improve the specificity of Exterminator coli, we are going to introduce cockroach pheromone expression system to it. If we want to create an Exterminator coli for other insects, we can engineer it by introducing specific pheromone for each insect.
Retrieved from " http://2014.igem.org/Team:Tokyo-NoKoGen/g3dh "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
