<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SydneyUni_Australia_Modelling_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SydneyUni_Australia/Modelling_Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SydneyUni_Australia/Modelling_Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Results&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SydneyUni_Australia/Modelling_Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SydneyUni Australia/Modelling Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="nav"><UL><LI><A href="/Team:SydneyUni_Australia">Home</A></LI><LI>Team
</LI><LI><A href="/Team:SydneyUni_Australia/Team">Team Page</A></LI><LI><A href="/Team:SydneyUni_Australia/Publicity">Publicity</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=SydneyUni_Australia">Team Profile</A></LI><LI><A href="https://igem.org/2013_Judging_Form?id=1115">Judging Form</A></LI></UL><LI>Project
</LI><LI><A href="/Team:SydneyUni_Australia/Project/Background">Background</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Design">Design</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Protocols">Protocols</A></LI><LI><A href="/Team:SydneyUni_Australia/Notebook">Notebook</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Results">Results</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/SinceHK">Since Hong Kong</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Parts">Registry Parts</A></LI><LI>Modelling
</LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Intro">Introduction</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Model">Mathematical model</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Validation">Validation</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Output">Output</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Conclusion">Conclusion</A></LI><LI><A href="/Team:SydneyUni_Australia/Safety">Safety</A></LI><LI><DIV style="width:120px; margin-left:-15px; margin-top;-1px;"><A href="/Team:SydneyUni_Australia/Strange_Nature">Strange Nature</A></DIV></LI><LI><A href="/Team:SydneyUni_Australia/Attributions">Attributions</A></LI></DIV><H2><SPAN class="mw-headline" id="Running_the_Model"><B>Running the Model</B></SPAN></H2><P>The model was run using MATLAB through the ode solver ODE45.
</P><P>
The modelling for both the pathays had initial conditions of:
</P><CENTER><TABLE class="wikitable"><TBODY><TR><TH>Constant </TH><TH> Value </TH><TH> Comment
</TH></TR><TR><TD>K<SUB>M A</SUB></TD><TD> 0.530 mM </TD><TD> From literature [1]
</TD></TR><TR><TD>k<SUB>cat A</SUB></TD><TD> 3.3 s<SUP>-1</SUP></TD><TD> From literature [1]
</TD></TR><TR><TD>K<SUB>M B</SUB></TD><TD> 0.940 mM </TD><TD> From literature [2]
</TD></TR><TR><TD>k<SUB>cat B</SUB></TD><TD> 0.0871 s<SUP>-1</SUP></TD><TD> From literature [2]
</TD></TR><TR><TD>K<SUB>M C</SUB></TD><TD> 7.2 mM </TD><TD> From literature [3]
</TD></TR><TR><TD>k<SUB>cat c</SUB></TD><TD> 89.8 s<SUP>-1</SUP></TD><TD> From literature [3]
</TD></TR><TR><TD>K<SUB>M D</SUB></TD><TD> 0.160 mM </TD><TD> From literature [4]
</TD></TR><TR><TD>k<SUB>cat D</SUB></TD><TD> 0.600 s<SUP>-1</SUP></TD><TD> From literature [4]
</TD></TR><TR><TD>K<SUB>M E</SUB></TD><TD> 20 mM </TD><TD> From literature [5]
</TD></TR><TR><TD>k<SUB>cat E</SUB></TD><TD> 25.4 s<SUP>-1</SUP></TD><TD> From literature [5]
</TD></TR><TR><TD>β, γ, δ &amp; ε</TD><TD> 0 mM </TD><TD> Not naturally present in cells.
</TD></TR><TR><TD>α<SUB>out</SUB></TD><TD> 1 mM </TD><TD> Initial concentration of DCA in solution (arbitrary)
</TD></TR><TR><TD>α<SUB>in</SUB></TD><TD> 0.001 mM </TD><TD> Initial concentration of DCA in cell
</TD></TR><TR><TD>A, B, C, D, E </TD><TD> 25.55 mM </TD><TD> Estimation from literature [11], as described in &quot;principles&quot;.
</TD></TR><TR><TD> 2 x 10<SUP>8</SUP> cells / mL </TD><TD> Initial cell concentration which allows appropriate growth
</TD></TR></TBODY></TABLE></CENTER><P>
Physical conditions: the model assumes that the cells are present in minimal media prior to DCA exposure and that the DCA is instantaneously and evenly present at time = 0 min in a solution of homogeneously mixed cells
Many assumptions have been made in the construction of the model and are outlined in the 'principles' section.
</P><P>By using the constants summarised in the previous section the flux, J, took the value (alongside the bacterial surface area, S):
</P><H2><SPAN class="mw-headline" id="The_Non-monooxygenase_Pathway"><B>The Non-monooxygenase Pathway</B></SPAN></H2><P><B>ODE overview:</B></P><P>The summary of the ODE (explained and justified in the previous section).
</P><P><B>Raw MATLAB code:</B></P><PRE>

function dy = nop450(t,y)
 
dy=zeros(7,1);
 
dy(1)= -y(7)*(6*10^(-12))*0.0463067*(y(1)-y(2));
dy(2)= ((6*10^(-12))*0.0463067*(y(1)-y(2)))-3.3*25.55*(y(2)/(0.53+y(2)));
dy(3)= 3.3*25.55*(y(2)/(0.53+y(2)))-0.0871*25.55*(y(3)/(0.94+y(3)));
dy(4)= .0871*25.55*(y(3)/(0.94+y(3)))- 0.6*25.55*(y(4)/(0.16+y(4)));
dy(5)= 0.6*25.55*(y(4)/(0.16+y(4))) - 25.4*25.55*(y(5)/(20+y(5)));
 
   if y(6) &gt;  2*10^(-10)
      dy(6)= 25.4*25.55*(y(5)/(20+y(5))) -1.5789*10^(-10)
   else
       dy(6) = 25.4*25.55*(y(5)/(20+y(5)))
   end
   
   if y(6) &gt; 0.0005 
        if y(7) &gt; 1.6*10^11
            dy(7)=0
        else
            dy(7) = 5*10^6
        end
   else
        dy(7) = 0
   end
 
end



</PRE><H4><SPAN class="mw-headline" id="MATLAB_output:"><B>MATLAB output:</B></SPAN></H4><P><B>The rate at which DCA is removed in solution:</B></P><P>Graph 1: The blue line represents how the concentration of DCA in solution (extracellular) decreases due to the action of our DCA degraders. The red line represents the intracellular concentions of the metabolites (disregard in this graph). One can see that an initial concentration of 2E8 cells/mL completely removes the DCA with a concentration 1mM in (roughly) 150mins.
</P><P><B>The Rate at which the intracellular concentration of the metabolites change over time:</B></P><P>Graph 2: Each line represents the concentration of each of the metabolites . This graph is simply a rescaling of graph 1. Note: the glycolate won’t accumulate in the cell – it is metabolised – the model had glycolate as the final product. It is used to show that the presence of glycolate can attribute to cell growth.
</P><P><B>Rescaling the graph once again: the rate at which the metabolic intermediate change over time.</B></P><P>Graph 3: Each line represents the concentration of each of the metabolites . This graph is simply a rescaling of graph 1 and 2.
</P><P><B>The rate at which the cells grow over time:</B></P><P>Graph 4: the blue line represents the linear increase of cells due to the presence of intracellular glycolate. 
</P><H2><SPAN class="mw-headline" id="The_Non-monooxygenase_Pathway_2"><B>The Non-monooxygenase Pathway</B></SPAN></H2><P><B>ODE overview</B>:
</P><P><B>Raw MATLAB code:</B></P><PRE>
function dy = p450(t,y)
 
dy=zeros(6,1);
 
dy(1)= -y(6)*(6*10^(-12))*0.0463067*(y(1)-y(2));
dy(2)= ((6*10^(-12))*0.0463067*(y(1)-y(2))) - 0.0113*25.55*(y(2)/(.12+y(2)));
dy(3) = 0.0113*25.55*(y(2)/(.12+y(2))) - 0.6*25.55*(y(3)/(0.16+y(3)));
dy(4)= 0.6*25.55*(y(3)/(0.16+y(3))) - 25.4*25.55*(y(4)/(20+y(4)));
 
 
   if y(5) &gt;  2*10^(-10)
      dy(5)= 25.4*25.55*(y(4)/(20+y(4))) -1.5789*10^(-10)
   else
       dy(5) = 25.4*25.55*(y(4)/(20+y(4)))
   end
   
   if y(5) &gt; 0.0005 
        if y(6) &gt; 1.6*10^11
            dy(6)=0
        else
            dy(6) = 5*10^6
        end
   else
        dy(6) = 0
   end
 
end


</PRE><P><B>MATLAB output:</B></P><P><B>The rate at which DCA is removed in solution:</B></P><P>Graph 5: The blue line represents how the concentration of DCA in solution (extracellular) decreases due to the action of our DCA degraders. 
</P><P><B>The Rate at which the intracellular concentration of the metabolites change over time:</B></P><P>Graph 6: Each line represents the intracellular concentration of each of the metabolites over time within a single cell. The colour associated with each metabolite is depicted in the figure legend. This graph is simply a rescaling of graph 5.
</P><P><B>Rescaling the graph once again: the rate at which the metabolic intermediate change over time.</B></P><P>Graph 7: Again, each line represents the intracellular concentration of each of the metabolites over time within a single cell. This graph is simply a rescaling of graph 5 and 6.
</P><P><B>The rate at which the cells grow over time:</B></P><P>Graph 8: The blue line represents the linear increase of the total number of cells due to the presence of intracellular glycolate. 
</P><P><B>Rescaling Shows a lag effect on cell growth:</B></P><P>Graph 9: The blue line represents the increase of cells due to the presence of intracellular glycolate. This graph is a rescaling of graph 8 to show the inital lag in cellualr growth.
</P><H2><SPAN class="mw-headline" id="Conclusions:"><B>Conclusions:</B></SPAN></H2><P>From Graphs 1 and 5, one can see that 1mM of DCA is removed from solution within roughly 50 minutes when the DCA degrading cells are at a concentration of 2E8 cells/mL.
</P><P>From Graphs 4, 8 and 9, it is evident that bacterial growth occurs. This growth is due to the production of glycolate, and by comparing graphs 6 and 9, one can see that bacterial growth correlates with glycolate accumulation.
</P><P>The cytotoxic metabolic intermediate chloroactealdehyde doesn't accumulate to a significant concentration in any of the pathways and is consistently at a negligibly small concentration. From Graphs 3 and 7 one can see that chloroacetaldehyde reaches a maximum concentration of roughly 0.2 mM in both pathways. Chloroacetaldehyde is seen to be metabolised very quickly; this concentration maximum is very short lived where it peaks at roughly 0.03 seconds and returns back to 0 mM by 0.5 seconds. It is expected that chloroacetaldehyde toxicity will not be a problem in our engineered cells.
</P><P>It is also possible to conclude that the pathways remove DCA at the same rate (through comparing graphs 1 and 5). 
</P><H2><SPAN class="mw-headline" id="References:"><B>References:</B></SPAN></H2><P>[1] Krooshof, G. H., Ridder, I. S., Tepper, A. W., Vos, G. J., Rozeboom, H. J., Kalk, K. H., Dijkstra, B. W. &amp; Janssen, D. B. (1998). Kinetic Analysis and X-ray Structure of Haloalkane Dehalogenase with a Modified Halide-Binding Site. <I>Biochemistry</I>, <B>37</B>(43), 15013-15023.
</P><P>[2]  Janecki, D. J., Bemis, K. G., Tegeler, T. J., Sanghani, P. C., Zhai, L., Hurley, T. D., Bosron, W. F. &amp; Wang, M. (2007). A multiple reaction monitoring method for absolute quantification of the human liver alcohol dehydrogenase ADH1C1 isoenzyme. <I>Analytical Biochemistry</I>, <B>369</B>(1), 18-26.
</P><P>[3] Pandey, A. V. &amp; Flück, C. E. (2013). NADPH P450 oxidoreductase: Structure, function, and pathology of diseases. <I>Pharmacology &amp; Therapeutics</I>, <B>138</B>(2), 229-254.
</P><P>[4]  van der Ploeg, J., Shmidt, M. P., Landa, A. S., &amp; Janssen, D. B. (1994). Identification of Chloroacetaldehyde Dehydrogenase Involved in 1,2-Dichloroethane Degradation. <I>Applied Environmental Microbiology</I><B>60</B>(5), 1599-1605.
</P><P>[5]  van der Ploeg, J., van Hall, G. &amp; Janssen, D. B. (1991) Characterization of the haloacid dehalogenase from Xanthobacter autotrophicus GJ10 and sequencing of the dhlB gene. <I>Journal of Bacteriology</I>, <B>173</B>(24), 7925-33.
</P><P>[6] Sinensky, M. I. (1974). Homeoviscous Adaption – A Homeostatic Process that Regulates the Viscosity of Membrane Lipids in <I>Escherichia coli</I>. <I>Proceedings from the National Academy of Science of the United States of America</I>, <B>71</B>(2), 522-525.
</P><P>[7] CyberCell Database
</P><P>[8]
</P><P>[9]
</P><P>[10] <A href="http://www.dtsc.ca.gov/AssessingRisk/Upload/12dca.pdf" class="external free" rel="nofollow">http://www.dtsc.ca.gov/AssessingRisk/Upload/12dca.pdf</A></P><P>[11] Ishihama, Y., Schmidt, T., Rappsilber, J., Mann, M., Hartl F. U., Kerner, M. J. &amp; Frishman, D. (2008) Protein abundance profiling of the Escherichia coli cytosol. BMC Genomics, <B>9</B>:102
</P><P>[12] Lord, J. M. (1972) Glycolate oxidoreductase in Escherichia coli. Biochemica et Biophysica Acta <B>267</B>:2, 227-327.
</P><CENTER><DIV style="font-family: 'Rockwell',Georgia, serif;"><H3>With thanks to:</H3></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SydneyUni_Australia/Modelling_Results">http://2013.igem.org/Team:SydneyUni_Australia/Modelling_Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SydneyUni_Australia/Modelling_Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SydneyUni_Australia/Modelling_Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Results&amp;oldid=335241" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>