<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SydneyUni_Australia_Modelling_Conclusion"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SydneyUni_Australia/Modelling_Conclusion">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SydneyUni_Australia/Modelling_Conclusion&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Conclusion&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Conclusion&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SydneyUni_Australia/Modelling_Conclusion" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SydneyUni Australia/Modelling Conclusion</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="nav"><UL><LI><A href="/Team:SydneyUni_Australia">Home</A></LI><LI>Team
</LI><LI><A href="/Team:SydneyUni_Australia/Team">Team Page</A></LI><LI><A href="/Team:SydneyUni_Australia/Publicity">Publicity</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=SydneyUni_Australia">Team Profile</A></LI><LI><A href="https://igem.org/2013_Judging_Form?id=1115">Judging Form</A></LI></UL><LI>Project
</LI><LI><A href="/Team:SydneyUni_Australia/Project/Background">Background</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Design">Design</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Protocols">Protocols</A></LI><LI><A href="/Team:SydneyUni_Australia/Notebook">Notebook</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Results">Results</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/SinceHK">Since Hong Kong</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Parts">Registry Parts</A></LI><LI>Modelling
</LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Intro">Introduction</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Model">Mathematical model</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Validation">Validation</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Output">Output</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Conclusion">Conclusion</A></LI><LI><A href="/Team:SydneyUni_Australia/Safety">Safety</A></LI><LI><DIV style="width:120px; margin-left:-15px; margin-top;-1px;"><A href="/Team:SydneyUni_Australia/Strange_Nature">Strange Nature</A></DIV></LI><LI><A href="/Team:SydneyUni_Australia/Attributions">Attributions</A></LI></DIV><H2><SPAN class="mw-headline" id="Conclusions:"> Conclusions: </SPAN></H2><UL><LI>In our model we find that 1 mM of DCA is removed from solution within roughly 150 minutes when the DCA degrading cells are at a concentration of 2 x 10<SUP>8</SUP> cells/mL (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph1" class="external text" rel="nofollow">graph 1</A> and <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph5" class="external text" rel="nofollow">graph 5</A>).
</LI></UL><UL><LI>It is also evident that bacterial growth can occur (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph4" class="external text" rel="nofollow">graph 4</A>, <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph8" class="external text" rel="nofollow">graph 8</A> and <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph9" class="external text" rel="nofollow">graph 9</A>). This growth is due to the degradation of DCA to glycolate. We can also see that that bacterial growth correlates with glycolate accumulation (by comparing <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph6" class="external text" rel="nofollow">graph 6</A> and <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph9" class="external text" rel="nofollow">graph 9</A>).
</LI></UL><UL><LI>The cytotoxic metabolic intermediate chloroactealdehyde does not accumulate to a significant concentration in either of the pathways and is consistently at a negligibly low concentration. We can see that chloroacetaldehyde reaches a maximum concentration of roughly 0.2 mM in both pathways (<A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph3" class="external text" rel="nofollow">graph 3</A> and <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph7" class="external text" rel="nofollow">graph 7</A>). Chloroacetaldehyde is apparently metabolised very quickly; this concentration maximum is short lived, peaks at roughly 0.03 seconds and returns back to 0 mM by 0.5 seconds. It is expected that chloroacetaldehyde toxicity will not be a problem in our engineered cells.
</LI></UL><UL><LI> We also conclude that the two pathways remove DCA at the same rate (by comparing <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph1" class="external text" rel="nofollow">graph 1</A> and <A href="https://2013.igem.org/Team:SydneyUni_Australia/Modelling_Output#graph5" class="external text" rel="nofollow">graph 5</A>). 
</LI></UL><H2><SPAN class="mw-headline" id="References:"> References:</SPAN></H2><P>[1] Krooshof, G. H., Ridder, I. S., Tepper, A. W., Vos, G. J., Rozeboom, H. J., Kalk, K. H., Dijkstra, B. W. &amp; Janssen, D. B. (1998). Kinetic Analysis and X-ray Structure of Haloalkane Dehalogenase with a Modified Halide-Binding Site. <I>Biochemistry</I>, <B>37</B>(43), 15013-15023.
</P><P>[2]  Janecki, D. J., Bemis, K. G., Tegeler, T. J., Sanghani, P. C., Zhai, L., Hurley, T. D., Bosron, W. F. &amp; Wang, M. (2007). A multiple reaction monitoring method for absolute quantification of the human liver alcohol dehydrogenase ADH1C1 isoenzyme. <I>Analytical Biochemistry</I>, <B>369</B>(1), 18-26.
</P><P>[3] Pandey, A. V. &amp; Flück, C. E. (2013). NADPH P450 oxidoreductase: Structure, function, and pathology of diseases. <I>Pharmacology &amp; Therapeutics</I>, <B>138</B>(2), 229-254.
</P><P>[4]  van der Ploeg, J., Shmidt, M. P., Landa, A. S., &amp; Janssen, D. B. (1994). Identification of Chloroacetaldehyde Dehydrogenase Involved in 1,2-Dichloroethane Degradation. <I>Applied Environmental Microbiology</I><B>60</B>(5), 1599-1605.
</P><P>[5]  van der Ploeg, J., van Hall, G. &amp; Janssen, D. B. (1991) Characterization of the haloacid dehalogenase from Xanthobacter autotrophicus GJ10 and sequencing of the dhlB gene. <I>Journal of Bacteriology</I>, <B>173</B>(24), 7925-33.
</P><P>[6] Sinensky, M. I. (1974). Homeoviscous Adaption – A Homeostatic Process that Regulates the Viscosity of Membrane Lipids in <I>Escherichia coli</I>. <I>Proceedings from the National Academy of Science of the United States of America</I>, <B>71</B>(2), 522-525.
</P><P>[7] CyberCell Database. Retrieved from <A href="http://ccdb.wishartlab.com/CCDB/" class="external free" rel="nofollow">http://ccdb.wishartlab.com/CCDB/</A>.
</P><P>[8] Currie, R. C., Chiao, F. F., McKone, T. E. (1994) Intermedia Transfer Factors for Contaminants Found at Hazardous Waste Sites: 1,2 dichloroethane (DCA). Retrieved from <A href="http://www.dtsc.ca.gov/AssessingRisk/Upload/12dca.pdf" class="external free" rel="nofollow">http://www.dtsc.ca.gov/AssessingRisk/Upload/12dca.pdf</A></P><P>[9] Lord, J. M. (1972) Glycolate oxidoreductase in <I>Escherichia coli</I>. <I>Biochemica et Biophysica Acta</I><B>267</B>:2, 227-327.
</P><P>[10] Ishihama, Y., Schmidt, T., Rappsilber, J., Mann, M., Hartl, F. U., Kerner, M. J. &amp; Frishman, D. (2008) Protein abundance profiling of the <I>Escherichia coli</I> cytosol. <I>BMC Genomics</I>, <B>9</B>:102.
</P><CENTER><DIV style="font-family: 'Rockwell',Georgia, serif;"><H3>With thanks to:</H3></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SydneyUni_Australia/Modelling_Conclusion">http://2013.igem.org/Team:SydneyUni_Australia/Modelling_Conclusion</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SydneyUni_Australia/Modelling_Conclusion" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SydneyUni_Australia/Modelling_Conclusion" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Conclusion&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Conclusion&amp;oldid=355763" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>