<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_SydneyUni_Australia_Modelling_Principles"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:SydneyUni_Australia/Modelling_Principles">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:SydneyUni_Australia/Modelling_Principles&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Principles&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Principles&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:SydneyUni_Australia/Modelling_Principles" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:SydneyUni Australia/Modelling Principles</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2013.igem.org</H3><DIV id="nav"><UL><LI><A href="/Team:SydneyUni_Australia">Home</A></LI><LI>Team
</LI><LI><A href="/Team:SydneyUni_Australia/Team">Team Page</A></LI><LI><A href="/Team:SydneyUni_Australia/Publicity">Publicity</A></LI><LI><A href="https://igem.org/Team.cgi?year=2013&amp;team_name=SydneyUni_Australia">Team Profile</A></LI><LI><A href="https://igem.org/2013_Judging_Form?id=1115">Judging Form</A></LI></UL><LI>Project
</LI><LI><A href="/Team:SydneyUni_Australia/Project/Background">Background</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Design">Design</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Protocols">Protocols</A></LI><LI><A href="/Team:SydneyUni_Australia/Notebook">Notebook</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Results">Results</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/SinceHK">Since Hong Kong</A></LI><LI><A href="/Team:SydneyUni_Australia/Project/Parts">Registry Parts</A></LI><LI>Modelling
</LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Intro">Introduction</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Model">Mathematical model</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Validation">Validation</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Output">Output</A></LI><LI><A href="/Team:SydneyUni_Australia/Modelling_Conclusion">Conclusion</A></LI><LI><A href="/Team:SydneyUni_Australia/Safety">Safety</A></LI><LI><DIV style="width:120px; margin-left:-15px; margin-top;-1px;"><A href="/Team:SydneyUni_Australia/Strange_Nature">Strange Nature</A></DIV></LI><LI><A href="/Team:SydneyUni_Australia/Attributions">Attributions</A></LI></DIV><H2><SPAN class="mw-headline" id="A_Schematic_of_the_Engineered_Metabolic_Pathway:"><B>A Schematic of the Engineered Metabolic Pathway:</B></SPAN></H2><DIV align="justify">
The above figure is a simplified schematic of the two metabolic pathways (<B>i</B> and <B>ii</B>) we have modelled. The symbols in squares signify the intracellular concentrations of the associated metabolite (Greek letters) or enzyme (capital English letters).  These symbols are used throughout the analysis. </DIV><H2><SPAN class="mw-headline" id="Summary_of_the_enzymes_involved_in_the_Metabolic_Pathway"><B>Summary of the enzymes involved in the Metabolic Pathway </B></SPAN></H2><CENTER><TABLE class="wikitable"><TBODY><TR><TH>Enzyme </TH><TH> Gene </TH><TH> Symbol </TH><TH> Constants </TH><TH> Substrate </TH><TH> Product </TH><TH> Ref
</TH></TR><TR><TD>1,2-Dichloroethane Dechlorinase </TD><TD> dhlA </TD><TD> A </TD><TD> K<SUB>M A</SUB> = 0.53 mM, k<SUB>cat A</SUB> = 3.3 s<SUP>-1</SUP></TD><TD> DCA </TD><TD> 2-chloroethanol </TD><TD> [1]
</TD></TR><TR><TD>Alcohol Dehydrogenase </TD><TD> Adhlb 1/2* </TD><TD> B </TD><TD> K<SUB>M B</SUB> = 0.94 mM, k<SUB>cat B</SUB> = 0.0871 s<SUP>-1</SUP></TD><TD> 2-chloroethanol </TD><TD> chloroacetaldehyde </TD><TD> [2]
</TD></TR><TR><TD>p450 </TD><TD> p450 </TD><TD> C </TD><TD> K<SUB>M C</SUB> = 0.120 mM, k<SUB>cat C</SUB> = 0.0113 s<SUP>-1</SUP></TD><TD> DCA </TD><TD> chloroacetaldehyde </TD><TD> [3]
</TD></TR><TR><TD>Chloroacetaldehyde Dehydrogenase </TD><TD> aldA </TD><TD> D </TD><TD> K<SUB>M D</SUB> = 0.06mM, k<SUB>cat D</SUB> = 0.60 s<SUP>-1</SUP></TD><TD> chloroacetaldehyde </TD><TD> chloroacetate </TD><TD> [4]
</TD></TR><TR><TD>Haloacetate Dehydrogenase </TD><TD> dhlB </TD><TD> E </TD><TD> K<SUB>M E</SUB> = 20 mM, k<SUB>cat E</SUB> = 25.4 s<SUP>-1</SUP></TD><TD> chloroacetate </TD><TD> glycolate </TD><TD> [5]
</TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="ODE_Model"><B>ODE Model</B></SPAN></H2><P>These are the two systems of ODEs we used to model each pathway:
</P><P><B>i)</B> The non-monooxygenase pathway (with dhlA (A) and adh1b1*2  (B)).
</P><P><B>ii)</B>The monooxygenase pathway (with p450 (C)).
</P><P>Generally, each line represents how the metabolic intermediate changes over time; this is dictated by the relative rate at which the intermediate is formed vs. the rate at which it is used/removed. 
</P><P>The two above systems of ODEs were used to model two things:
</P><P>1. The rate at which extracellular DCA is removed from solution (given the number of cells in solution) . This is considering the rate at which DCA crosses the plasma membrane (from solution to cytoplasm) of all cells in solution. 
</P><P>2. The rate at which the intracellular concentrations of the metabolic intermediates change over time within a single cell. This considers the relative activities of the enzymes of the metabolic pathway. 
</P><P>
The symbols A, B, C, D &amp; E represent the intracellular enzyme concentrations of 1,2-Dichloroethane Dechlorinase, Alcohol Dehydrogenase, Cytochrome p450, Chloroacetaldehyde Dehydrogenase and Haloacetate Dehydrogenase respectively. They have units of mM.
</P><P>
The symbols α<SUB>in</SUB>, β, γ, δ and ε represent the intracellular concentrations of the metabolic intermediates DCA, 2-chloroethanol, chloroacetaldehyde, chloroacetate and glycolate respectively. They have units mM.
α<SUB>out</SUB> represents the extracellular concentration of DCA, i.e. the concentration of DCA in solution. It has units mM.
</P><P>
The symbol Ψ represents the concentration of cells in solution and has units cells/mL.
</P><P>
The function J(α<SUB>out</SUB>,α<SUB>in</SUB>) represents the rate which extracellular DCA moves across the plasma membrane and into the single cell (the flux rate). It is a function of the extra and intracellular concentrations of DCA and has a value of surface area per time m<SUP>2</SUP>s<SUP>-1</SUP>. The value S represents the average surface area of the cellular membrane of <I>E. coli</I>. By multiplying S by J one achieves the total amount of DCA flowing into a single cell per unit time. The total amount of DCA flowing from solution is achieved by multiplying Ψ by S and J.
</P><H2><SPAN class="mw-headline" id="Constructing_a_model_for_the_metabolic_pathway"><B>Constructing a model for the metabolic pathway</B></SPAN></H2><P>The rate of change for the intracellular concentration of metabolites (β, γ, δ and ε) is purely determined through the relative activity (described by MM equations) of the enzymes creating or removing the metabolite. 
</P><P>Luckily the enzymes of our metabolic pathway are accurately described by MM kinetics and all constants were available in the literature. The system is based on the forward and reverse kinetic rate (k<SUB>1</SUB> and k<SUB>-1</SUB> respectively) of substrate-enzyme binding followed by the irreversible catalytic step of product formation (k<SUB>2</SUB>) as depicted in the figure below:
</P><P>The rate of catalysis for an enzyme is a function of the enzyme and substrate concentration and incorporates the catalytic constant k<SUB>cat</SUB> and the Michaelis constant K<SUB>M</SUB>, which are given in most enzyme kinetics studies.
</P><P>The symbol R<SUB>E</SUB> denotes the reaction rate: i.e. the rate of product formation (which is equal to the rate of substrate removal).
</P><P>So, by observing:
</P><P>i) that the rate at which a substrate is removed is directly equal to the rate at which the associated product is created (e.g. the rate at which γ is removed is the rate at which δ is formed)
ii) and how each intermediate simultaneously acts as a substrate and product (e.g. how D creates δ, and E uses δ to create ε)
</P><P>it is easy to see how this process gives rise to a connected system of ODEs. 
</P><H2><SPAN class="mw-headline" id="Estimating_Protein_Concentration"><B>Estimating Protein Concentration</B></SPAN></H2><P>The accuracy of this model is strongly dependent on the intracellular enzyme concentrations; these cannot be determined theoretically as there is a weak correlation with the level of gene expression and protein abundance. 
</P><P>Ishihama <I>et al.</I> ([11]) profiled the protein concentration in <I>E. coli</I> as:
</P><P>The enzyme-encoding-genes are under the regulation of an artificial promoter termed Psyn. The promoter is designed to be constitutive and drive high expression of the enzymes. We estimated the protein concentration as 10,000 protein units per cell - the upper quartile limit in the ‘Highly abundant group’ box in the figure above.
</P><P>By taking the cell volume as 0.65µ<SUP>3</SUP>  = 0.63 x 10<SUP>-18</SUP> m<SUP>3</SUP> [7]
</P><P>So the intracellular concentration for all proteins is predicted to be 25.55 mM
</P><H2><SPAN class="mw-headline" id="DCA_Diffusion_Across_the_Plasma_Membrane"><B>DCA Diffusion Across the Plasma Membrane</B></SPAN></H2><P>We were presented with a scenario where it was necessary to model how DCA would move from the solution into the cell where it’s metabolised: i.e. linking α<SUB>out</SUB> and α<SUB>in</SUB>. 
</P><P>This was tackled by modelling the rate of DCA diffusion across a cell membrane through Fick’s first law of diffusion:
</P><P>Fick’s first law of diffusion is justified since 1,2-DCA is non-polar and the cellular membrane is thin. The law states that the flux, J, of DCA across the membrane is equal to the permeability coefficient, P, times the concentration difference of DCA across the cell membrane. The flux has units m<SUP>2</SUP>  s<SUP>-1</SUP>
The permeability coefficient of DCA is not reported in the literature, so it had to be estimated from the definition of the permeability constant.
</P><P>Where D is the diffusion constant for DCA diffusion across the plasma membrane and d is the length of the membrane. 
The partition coefficient for DCA across a plasma membrane is not documented, but can be estimated to be similar to the octanol-water partition coefficient (K<SUB>ow</SUB>). This constant determines the equilibrium ratio of DCA in octanol and water.
Like many properties of DCA, the diffusion constant, D, is not documented in the literature. It was determined through the famous Stokes-Einstein equation:
</P><P>Where k<SUB>B</SUB>, T, η  &amp; r represent the Boltzmann constant (1.3806488 × 10-23 m<SUP>2</SUP> kg s-2 K-1), temperature, viscosity of the membrane and radius of DCA respectively. Here we must assume DCA is spherical.
The ‘radius’ of DCA isn’t described in the literature. But the van der Waal constant, b, can be used to calculate the van der Waal volume, V<SUB>W</SUB> , and hence the van der Waal radius, r<SUB>W</SUB><SUP>3</SUP>. Note DCA is not spherical, and this method is used to calculate atoms.
</P><P>By combining all of the above, the flux can be described as:
</P><CENTER><B>Summary of Constants</B></CENTER><CENTER><TABLE class="wikitable"><TBODY><TR><TH>Symbol </TH><TH> Name </TH><TH> Value (units) </TH><TH> Ref
</TH></TR><TR><TD>η </TD><TD> Cellular Membrane Viscosity </TD><TD> 1.9 kg/m/s </TD><TD> [6]
</TD></TR><TR><TD>S </TD><TD> E. coli Membrane Surface Area </TD><TD> 6x10<SUP>-12</SUP> m<SUP>2</SUP></TD><TD> [7]
</TD></TR><TR><TD>r </TD><TD> DCA radius </TD><TD> 0.3498 nm </TD><TD> [8]
</TD></TR><TR><TD>K<SUB>ow</SUB></TD><TD> Octanol-water Partition Coefficient for DCA </TD><TD> 28.2 </TD><TD> [9]
</TD></TR><TR><TD>d </TD><TD> Length of Cellular Membrane </TD><TD> 2 nm </TD><TD> [10]
</TD></TR></TBODY></TABLE></CENTER><H2><SPAN class="mw-headline" id="Extending_the_System_to_Cell_Cultures"><B>Extending the System to Cell Cultures</B></SPAN></H2><P>The model describes the process of DCA diffusion and metabolism of a single cell. So given a concentration of our engineered cells in solution one can calculate the rate at which the DCA concentration in solution (αα<SUB>out</SUB>) decreases by simply multiply the flux rate of DCA across the membrane by the total concentration of cells in the solution. 
So, by letting Ψ represent the concentration of cells in solution. The overall rate of decrease of DCA in solution is:
</P><P>Since the latter portion of the system of ODEs describes the intracellular concentrations within a single cell, the rate at which the DCA comes into the cell and is metabolised (dα<SUB>in</SUB>/dt) is simply:
</P><P>
The motivation for doing this as it seems more natural to consider the intracellular metabolite concentrations of a single cell, rather than averaged across many. It also allows one to gauge whether chloroacetaldehyde reaches cytotoxic levels. The assumption made here is that all the cells in the solution are exactly the same.
</P><H2><SPAN class="mw-headline" id="Factoring_in_Cell_Growth"><B>Factoring in Cell Growth</B></SPAN></H2><P>The cells are expected to grow due to the production of glycolate (used as a carbon source for growth).  So one can model the rate at which the cell concentration, Ψ, increases. 
Ideally one would want to experimentally determine the cell growth as function of time, but unfortunately we did not have enough time to do this. 
We turned to other, more approximate measures:
The growth of <I>E. coli</I> over time due to the presence of glycolate was initially described by Lord [12]:
</P><P>Firstly it must be noted that this analysis is highly approximate, and is used only to obtain a very rough estimate of the growth rate as a function of glycolate produced.
By using the approximation that an OD550 of 0.1 = 1 x 10<SUP>8</SUP> cells/mL one can use the above graph to generate rates of E. coli growth due to glycolate production.
From graph 1, it can be seen that a 2.2 x 10<SUP>8</SUP>cells us 3.33 mM of glycolate in 95 seconds. Or in other words, 1 cell uses 1.5789 x 10<SUP>-10</SUP> mM of glycolate per second: 
</P><P>This assumes that only the original cells existing in solution from the graphs above (the 2.2E8 cells) use glycolate for growth.
From the graph b, the OD550 increases at a rate of 0.25 per 50 seconds (gradient of graph 2) which shows that the cells increase at a maximum rate of 5x10<SUP>6</SUP> cells / mL / s when growing in saturated glycolate.
From the graphs, it can be seen that the point where maximal growth no longer occurs (where the straight line becomes curved) is when glycolate is at a very low concentration. We will take it that the saturation point is so small that it is negligible – i.e. the growth rate is always maximal when in the presence of glycolate.
So the cellular growth rate can be described as: 
</P><P>Using this approach, the model for growth rate is most appropriate for cell concentrations in the range of 2 x 10<SUP>8</SUP> to 6 x 10<SUP>8</SUP> cells/mL.
</P><CENTER><DIV style="font-family: 'Rockwell',Georgia, serif;"><H3>With thanks to:</H3></DIV></CENTER></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2013.igem.org/Team:SydneyUni_Australia/Modelling_Principles">http://2013.igem.org/Team:SydneyUni_Australia/Modelling_Principles</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:SydneyUni_Australia/Modelling_Principles" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:SydneyUni_Australia/Modelling_Principles" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Principles&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:SydneyUni_Australia/Modelling_Principles&amp;oldid=343508" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2013.igem.org:Privacy_policy" title="2013.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2013.igem.org:General_disclaimer" title="2013.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>