<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUST-China_Description skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:HUST-China/Description</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Team:HUST-China:Description</TITLE></P><DIV id="mid"><DIV class="container header"><DIV class="navbar"><DIV class="navbar-inner"><DIV class="container"><UL class="nav"><LI class="first-menu"><A href="https://2015.igem.org/Team:HUST-China">HOME</A></LI><LI class="dropdown other-menu" id="accountmenu"><A href="https://2015.igem.org/Team:HUST-China/Project">PROJECT</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Background">Background</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Description">Description</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Design">Design</A></LI></UL><LI class="dropdown other-menu" id="accountmenu"><A href="https://2015.igem.org/Team:HUST-China/Wetlab">WETLAB</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Results">Results</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Experiments">Experiments&amp;protocol</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/InterLab Study">InterLab Study</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Notebook">Notebook</A></LI><LI class="dropdown other-menu" id="accountmenu"><A href="https://2015.igem.org/Team:HUST-China/Parts">PARTS</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Basic_part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Basic_part#2">Composite Parts</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Part_Collection">Part Collection</A></LI><LI class="dropdown other-menu" id="accountmenu"><A href="https://2015.igem.org/Team:HUST-China/Modeling">MODELING</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Modeling">Overiew</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Modeling on Cellular Level">Modeling on Cellular Level</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Modeling on Ecosystem Level">Modeling on Ecosystem Level</A></LI><LI class="dropdown other-menu" id="accountmenu"><A href="https://2015.igem.org/Team:HUST-China/Practices">HUMAN PRACTICES</A></LI><LI class="dropdown other-menu" id="accountmenu"><A href="https://2015.igem.org/Team:HUST-China/Safety">OTHERS</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Collaborations">Collaborations</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:HUST-China/Achievements">Achievements</A></LI></DIV></DIV></DIV></DIV><DIV id="pic"><DIV class="pic_a"><H3 align="center" style="color:white"><B>click it~</B></H3></DIV></DIV><DIV id="maodian"><UL class="ul"><LI class="li"><A href="#description" class="btn btn-default btn-lg">Description</A></LI><LI class="li"><A href="#darkness" class="btn btn-default btn-lg">Darkness induction system</A></LI><LI class="li"><A href="#supporting" class="btn btn-default btn-lg">Supporting system</A></LI><LI class="li"><A href="#flocculating" class="btn btn-default btn-lg">Flocculating system</A></LI><LI class="li"><A href="#host" class="btn btn-default btn-lg">Yarrowia lipolytica</A></LI><LI class="li"><A href="#improvement" class="btn btn-default btn-lg">Biobrick Improvement</A></LI><LI class="li"><A href="#reference" class="btn btn-default btn-lg">References</A></LI></UL></DIV><VIDEO width="800" height="400" controls="controls" style="margin-top:50px;margin-left:300px;z-index:-2"><SOURCE src="https://static.igem.org/mediawiki/2015/8/89/IGEMHUSTVIDEO.mp4" type="video/webm">
Your browser does not support the video tag.
</SOURCE><DIV class="description" style="margin-top:0px;margin-bottom:30px"><P>According to requirement of iGEM wiki construction, we hosted our cartoon instruction on iGEM server. For those can not play this video on your Browser such as IE11, please turn to view our wiki with google chrome or directly donwload this video <B><A href="https://static.igem.org/mediawiki/2015/8/89/IGEMHUSTVIDEO.mp4">HERE</A></B> from iGEM server.</P></DIV><DIV align="center" class="description"><H2 style="color:black" align="left"><B>1.Description</B></H2><P>In our project of Euk.cement, we employ Yarrowia lipolytica JMY1212 as our chassis and generate three circuits respectively. We use expression vector p414TEF to load the light control part of darkness induction system and JMP62 as backbone to carry the negative control part together with flocculating part and supporting part. The whole project is aiming to consolidate sands or soil particles more flexibly in proper way, time and region. Next, we will show all the details. 
     		</P><DIV><DIV class="pizhu">Figure 1：The complete circuit</DIV></DIV><P>As mentioned above, our design is briefly divided into three parts: Darkness induction system, flocculating system and supporting system. The latter two are both under control of the former one with the same upstream promoter Anb1. Fusion protein CRY2-BD and CIB1-AD are respectively expressed by constitutive promoter pTEF and pADH1. CRY2-BD has high affinity to bind CIB1-AD only when induced by blue light (even contained in nature sunlight) . This interaction can activate pGal1 owing to the combination between CIB1 and CRY2’s special structures thus induce the downstream promoter pGal1 and express ROX1 gene. Then expressed ROX1 protein can inhibit the promoter Anb1. So our engineered circuit is turned off when exposed to light. On the contrary, in darkness, the structural-reversed CRY2 will dissociate with CIB1. Without CRY2-BD’s target function, AD is hard to combine and active pGal1 in the complex environment of nucleus, so the downstream ROX1 is no longer transcribed. With the degradation of translated ROX1 protein, pAnb1 starts to express the downstream gene as the supporting matter and flocculating agent :LIP2 pro-Si-tag-YLcwp3 and LIP2-pro-Mcfp-3 fusion proteins(shown in Fig 1). </P></DIV><DIV class="description"><H2 style="color:black" align="left"><B>2.Darkness Induction System</B></H2><DIV><DIV class="pizhu">Figure 2：Darkness induction system</DIV></DIV><P>The first one is the expression control system. Each system should apply a kind of trigger of working system. While a number of inducible systems are available for controlling protein expression in yeast, almost none of them make sense for us, as our project’s practical applications are currently considered to exist mainly in marine environment, which makes the amount of chemical inducers to be used a huge burden.</P><P> So we give up several common promoters including those naturally responsive to molecules such as galactose, methionine, and copper, or engineered regulatory systems that respond to orthogonal molecules such as estrogen (Gal-ER-VP16)<SUP>[1]</SUP> or doxycycline (Tet-OFF)<SUP>[2]</SUP> .In fact, some of them are really robust and well-designed pathways. However, we find a better method instead, a darkness induction system based on light control yeast two-hybrid principles, because darkness is the most common environment factor that our Euk.cement cells will face in working.</P><P>Our original idea comes from dimers that allow inducible control of protein-protein interactions, which are powerful tools for manipulating biological processes. On this basis, we successfully utilized a split transcription factor that is reconstituted by light-dependent protein–protein interactions as in yeast two-hybrid systems. </P><P>	Compared to those chemical-genetic tools, the light control system not only meets our demand, but also possesses many other advantages. Despite the general success of chemical inducers, the potential problems, such as toxic, nonspecific, or pleiotropic effects of these treatments can impose limitations on their use in sensitive chassis or environment. Also, light control is far more fast, precise, low cost and easy. It even allows exquisite dose-dependent control over protein levels, as light can be delivered for defined amounts of time and be instantly removed.</P><H3>Principle of light control yeast two hybrid</H3><DIV><DIV class="pizhu">Figure 3: yeast two-hybrid principles</DIV></DIV><P>To regulate DNA transcription by light, the system is based on a two-hybrid interaction in which a light-mediated protein interaction brings together two halves (a binding domain and an activation domain) of a split transcription factor of pGal. One of the light-mediated protein is Arabidopsis CIB1, a basic helix-loop-helix (bHLH) protein, and another is cryptochrome 2 (CRY2), a blue light-absorbing photosensor that binds CIB1 in its photoexcited state <SUP>[3]</SUP>. When applying the two-hybrid system to light control switch, there generated a couple of fusion protein with each half containing a light-mediated part and transcription-factor-related part. If we remove the stimulation of blue light, dark reversion of CRY2 will dissociate its interaction with CIB1 and halt Gal4-dependent transcription. These modules require no exogenous chromophore, and are reversible within minutes thus triggering protein translocation at a sub-second time scale, and it even allow potential use in vivo in whole organisms (shown in Fig 3). </P><DIV><DIV class="pizhu">Figure 4: Light control part</DIV></DIV><P>In the circuit, the two halves, CIB1 and CRY2 are fused separately with activation domain (AD) of Gal4 and DNA binding domain (BD) of Gal4,the fusion of which are respectively expressed by constitutive promoters pADH1 and pTEF. Both of them are essential to yeast genetics as the trigger of transcription requires their attachment to the promoter sequence at the same time. Commonly, it is almost impossible because they are too far from each other. But when exposed to blue light, such interaction of CIB1 and CRY2 brings together the split Gal1(similar to Gal4) transcription factor to control DNA transcription. As BD combines to the upstream activation sequence (UAS) and AD is successfully targeted to activate the promoter, downstream gene are able to be transcribed and translated, following the principle of yeast two-hybrid(shown in Fig 4).</P><H3>Logic reverse control</H3><DIV><DIV class="pizhu">Figure 5：Logic reverse control</DIV></DIV><P>Then a couple of promoter Panb1 and its inhibitory protein ROX1 <SUP>[4]</SUP> are set afterwards (shown in Figure 5). Actually, a light-activated promoter as we described above is still one step away from our needs. Since the applications of this project are expected to be put into marine environment, it is highly possible that the marine yeasts are unable to get to sunlight in most of the blue water region. Based on the above, we generated a logic reverse control. After introducing Panb1 and ROX1, the system’s function is exquisitely switched: in the condition of light, ROX1 in the downstream of pGal1 are expressed to inhibit expressing our functional proteins. Only total darkness can remove the inhibition and allow the expression of target protein in abysmal sea of all time.</P></DIV><DIV class="description"><H2 style="color:black" align="left"><B>3.Supporting System</B></H2><P>Next, to preserve the kit’s function, we placed a silica binding protein Si-tag right after Panb1. It’s not for direct consolidation, but to target marine yeasts to the surface of sands. Considering waves may occur occasionally underwater, a guarantee for the yeasts not to flow away is really necessary, which equals to immobilize them on silicon. When we fused an anchor protein YLcwp3 at the C-terminal of Si-tag and a signal peptide LIP2 prepro at the N-terminal of Si-tag, we can display Si-tag onto the surface of cell wall and link the yeasts’ fate to sands thoroughly.</P><DIV><DIV class="pizhu">Figure 6: Supporting system：displaying Si-tag on the cell wall of Y.L</DIV></DIV><H3 style="color:black;margin-top:30px" align="left">Si-tag</H3><P style="width:500px;float:left">Si-tag is 50S ribosomal protein L2 in the genome of E.coli<SUP>[5,6]</SUP> . It has been found from several bacterial strains and proved to have a silica-binding property. It is demonstrated that an L2-protein fusion binds to silica surfaces 20- to 100-fold more strongly than poly-Arg-tagged proteins, showing its strong superiority in silica binding. The Si-tag consists of three domains. Based on previous study, it is found that domain 1 and 3 of recombinant Si-tag bind much stronger than domain 2. Following this result, we performed further study to see how the combinations of different domains work in our surface display system. Indeed, we find it meaningful as a kit that fits in several circumstances. This varying intensity of viscosity is just what we want as a kit to be to utilized following different requirement. In our study, the separate ones, domain 1, 2, 3, and combinations as fusion domains1+2, 1+3, 2+3, 1+2+3, are all tested. Hoping a linker could bring better flexibility between two domains, we also put a short GS linker between domain 1&amp;3, creating the 1+Linker+3 Si-tag.</P><DIV style="color:black;margin-top:30px" align="left"><B>Anchor Protein</B></DIV><P style="width:820px;float:left">Ylcwp3 is the covalently bound GPI-anchored cell wall protein of Yarrowia lipolytica. The protein allows the target to be covalently bound to β-1, 6-glucan of the cell wall. The nucleotide sequence encoding 110 amino acids of the Ylcwp3 C terminus has been widely utilized for the construction of the cell surface display vector in Y. Lipolytica. In previous study, the observed high activity of the immobilized lipase LIP2 on the surface clearly shows the efficiency of the surface display system<SUP>[7,9]</SUP>.Therefore we adopt YLcwp3 as an anchor to target the interest protein on the surface of the cell wall.</P><DIV style="color:black;margin-top:30px" align="left"><B>Signal Peptide</B></DIV><P style="width:500px;float:left">Yarrowia.lipolytica naturally secretes lipases LIP2. The signal peptide LIP2 prepro we employed is a modified signal peptide from Y.lipolytica LIP2 with the sequence consisting 13 aa pre, 4 XA or XP dipeptides, 10 aa pro and KR cleavage site. It can leads the flocculating protein Mcfp-3 to be secreted out of cell to perform its flocculating function, during the secretion progress, the signal peptide will be deleted in the Golgi apparatus to eliminate its interference to the secreting protein<SUP>[8,9]</SUP> . 
</P></DIV><DIV class="description"><H2 style="color:black" align="left"><B>4.Flocculating System</B></H2><DIV><DIV class="pizhu">Figure 7: Flocculating system：Secreting Mcfp-3 among sand</DIV></DIV><P>Following is flocculating agent secretion circuit which briefly consists of the promoter anb1, LIP2 prepro signal peptide, Mcfp-3 flocculating protein and its terminator yADH1. Mcfp-3 is the main flocculating material. It can perform self-assembly and thus get high affinity to silica and other solid surface. So the secreted Mcfp-3 protein can cross link sand particles together and complete flocculation.</P><H3>Flocculating agent</H3><P style="width:500px;float:left">Mcfp-3 is the filopodia protein secreted from Mytilus californianus<SUP>[10]</SUP> . The protein is important to the formation of filopodia to help mussels permanently or temporarily tether to the rough surface of reef or ship-body. This coherent substance shows an excellent adhesion performance with no substitute under water for sustaining in the repeating wash of waves and having no any known toxicity. The properties are owing to the amino acid content which are successively DOPA (28mol％), glycine, lysine and asparagine. The presence of DOPA in the foot protein suggests a role in protein-protein and protein-substrate(glass eg.) interactions related to the formation of adhesive plaques. The electron rich nature of DOPA and its susceptibility to oxidation can lead to the formation of o-quinone with subsequent tanning or sclerotization to form a cross-linked  stable products. Furthermore, it is noteworthy that the protein shows an intrinsic ability of self-assemble into stable, higher molecular weight forms that may further enhance our consolidation function<SUP>[11]</SUP>.</P></DIV><DIV class="description"><H2 style="color:black" align="left"><B>5. Yarrowia Lipolytca</B></H2><P style="width:500px;float:left">We adopt a marine yeast Yarrowia lipolytca JMY1212 as our chassis <SUP>[12]</SUP> for the specific application this year: building artificial reef. Our host is generally regarded as safe microorganism and has an abroad application prospect in industries of foods and medicine. Researches show that Y.lipolytica is widely found in marine environment and exists stably in the ocean and healthy marine fishes’ intestine. It is able to make use of Glucose, ethanol, acetate or other hydrophobic substrate (alkanes, fatty acid and lipid ) as carbon source with fast growing capacity and adaptability for high-density fermentation. Moreover in the expression system of Y.lipolytica, transformed plasmids can integrate into multiple site and they are all genetically stable in the genome. And the signal peptide XPR2 pre, LIP2 prepro and so on are all demonstrated to own high efficiency of secreting protein. So it is obvious Y.lipolytica is a powerful host to express and secrete complicated recombinant protein. Based on the above mentioned reasons, Yarrowia lipolytica are regarded as an ideal strain for our marine application of micro-consolidation.</P></DIV><DIV class="description"><H2 style="color:black" align="left"><B>6.Biobrick Improvement</B></H2><P style="width:500px;float:left">In the early period of our project design, we found two SpeI illegal sites in a Part BBa_K191007 when glancing over parts Registered in pools. Though we wouldn't use this part in our project, we still removed its illegal sites by site-directed mutagenesis, in order to make it meet the requirements of RFC10 and more convenient to be assembled. According to the reference <SUP>[12]</SUP>,we successfully found the appropriate sites for mutation without affecting the part’s function.
		    Finally, we created 3 different mutants of Part <A href="http://parts.igem.org/Part:BBa_K191007">BBa_K191007</A>. In order to test and verify our mutants’ function, we inserted RBS <A href="http://parts.igem.org/Part:BBa_B0032">(BBa_B0032)</A>  + GFP  <A href="http://parts.igem.org/Part:BBa_E0040">(BBa_E0040)</A> into the downstream of them. We have intended to measure their fluorescence value. However,unfortunately, we had some trouble in getting the tryptophan analog, 3-indoleacrylicacid, a competitive inhibitor of tryptophan in the formation of the specific aporepressor, so we couldn't complete the verification work. We hope that their verification can be finished in the future.
		     Moreover, we also improved the characterization of five existing BioBrick Parts. They are <A href="http://parts.igem.org/Part:BBa_K207001">BBa_K207001</A>
,
<A href="http://parts.igem.org/Part:BBa_K1144001">BBa_K1144001</A>
,
<A href="http://parts.igem.org/Part:BBa_K105007">BBa_K105007</A>
,
<A href="http://parts.igem.org/Part:BBa_K950002">BBa_K950002</A>
,
and 
<A href="http://parts.igem.org/Part:BBa_K950001">BBa_K950001</A>.</P></DIV><DIV class="description"><TABLE border="1" style="width:810px;height:auto"><TBODY><TR><TH>Type</TH><TH>Part Number</TH><TH>Description</TH><TH>Length(bp)</TH></TR><TR><TD>Regulatory</TD><TD><A href="http://parts.igem.org/Part:BBa_K1592020">BBa_K1592020</A></TD><TD>Part mutant1</TD><TD>85</TD></TR><TR><TD>Regulatory</TD><TD><A href="http://parts.igem.org/Part:BBa_K1592021">BBa_K1592021</A></TD><TD>Part mutant2</TD><TD>85</TD></TR><TR><TD>Regulatory</TD><TD><A href="http://parts.igem.org/Part:BBa_K1592022">BBa_K1592022</A></TD><TD>Part mutant3</TD><TD>85</TD></TR><TR><TD>Composite</TD><TD><A href="http://parts.igem.org/Part:BBa_K1592023">BBa_K1592023</A></TD><TD>Part mutant1+RBS+GFP</TD><TD>832</TD></TR><TR><TD>Composite</TD><TD><A href="http://parts.igem.org/Part:BBa_K1592024">BBa_K1592024</A></TD><TD>Part mutant2+RBS+GFP</TD><TD>832</TD></TR><TR><TD>Composite</TD><TD><A href="http://parts.igem.org/Part:BBa_K1592025">BBa_K1592025</A></TD><TD>Part mutant3+RBS+GFP</TD><TD>832</TD></TR></TBODY></TABLE></DIV><DIV class="description"><H2 style="color:black" align="left"><B>Reference</B></H2><P>
         	[1] Braselmann S, Graninger P, Busslinger M. A selective transcriptional induction system for mammalian cells based on Gal4-estrogen receptor fusion proteins[J]. Proceedings of the National Academy of Sciences, 1993, 90(5): 1657-1661.
[2]Brengues M, Teixeira D, Parker R. Movement of eukaryotic mRNAs between polysomes and cytoplasmic processing bodies[J]. Science, 2005, 310(5747): 486-489.
[3]Hughes R M, Bolger S, Tapadia H, et al. Light-mediated control of DNA transcription in yeast[J]. Methods, 2012, 58(4): 385-391.
[4] Balasubramanian B, Lowry C V, Zitomer R S. The Rox1 repressor of the Saccharomyces cerevisiae hypoxic genes is a specific DNA-binding protein with a high-mobility-group motif[J]. Molecular and cellular biology, 1993, 13(10): 6071-6078.
[5]Taniguchi K, Nomura K, Hata Y, et al. The Si‐tag for immobilizing proteins on a silica surface[J]. Biotechnology and bioengineering, 2007, 96(6): 1023-1029.
[6]Ikeda T, Hata Y, Ninomiya K, et al. Oriented immobilization of antibodies on a silicon wafer using Si-tagged protein A[J].Analytical biochemistry, 2009, 385(1): 132-137.
[7]Yuzbasheva E Y, Yuzbashev T V, Laptev I A, et al. Efficient cell surface display of Lip2 lipase using C-domains of glycosylphosphatidylinositol-anchored cell wall proteins of Yarrowia lipolytica[J]. Applied microbiology and biotechnology, 2011, 91(3): 645-654.
[8]Nicaud J M, Madzak C, Broek P, et al. Protein expression and secretion in the yeast Yarrowia lipolytica[J]. FEMS yeast research, 2002, 2(3): 371-379.
[9]Madzak C, Gaillardin C, Beckerich J M. Heterologous protein expression and secretion in the non-conventional yeast Yarrowia lipolytica: a review[J]. Journal of Biotechnology, 2004, 109(1): 63-81.
[10]Platko J D, Deeg M, Thompson V, et al. Heterologous expression of Mytilus californianus foot protein three (Mcfp-3) in Kluyveromyces lactis[J]. Protein expression and purification, 2008, 57(1): 57-62.
[11]Zhong C, Gurry T, Cheng A A, et al. Strong underwater adhesives made by self-assembling multi-protein nanofibres[J]. Nature nanotechnology, 2014.
[12]Madzak C, Gaillardin C, Beckerich J M. Heterologous protein expression and secretion in the non-conventional yeast Yarrowia lipolytica: a review[J]. Journal of Biotechnology, 2004, 109(1): 63-81.
[13] Yanofsky C, Platt T, Crawford I P, et al. The complete nucleotide sequence of the tryptophan operon of Escherichia coli [J]. Nucleic acids research, 1981, 9(24): 6647-6668.
     		</P></DIV></VIDEO></DIV></DIV></DIV></DIV></DIV></BODY></HTML>