Team:HUST-China/Experiments
Team:HUST-China:Modeling
HOME
PROJECT
Background
Description
Design
WETLAB
Results
Experiments&protocol
InterLab Study
Notebook
PARTS
Basic Parts
Composite Parts
Part Collection
MODELING
Overiew
Modeling on Cellular Level
Modeling on Ecosystem Level
HUMAN PRACTICES
OTHERS
Safety
Collaborations
Attributions
Team
Achievements
click it~
Molecular Experiments
The protocol of experiment
The transformation of lipolytica
Verification Experiments
Molecular Experiments
1.Molecular Experiments Part in E.coli
We constructed following vectors carrying our parts and transformed these vectors into corresponding host strains.
Project Related
Part
Plasmid Construction
Transformation to Host Strain
ST1
JMP62
Yarrowia lipolytica JMY1212
ST2
JMP62
Yarrowia lipolytica JMY1212
ST3
JMP62
Yarrowia lipolytica JMY1212
ST12
JMP62
Yarrowia lipolytica JMY1212
ST13
JMP62
Yarrowia lipolytica JMY1212
ST23
JMP62
Yarrowia lipolytica JMY1212
ST123
JMP62
Yarrowia lipolytica JMY1212
ST1L3
JMP62
Yarrowia lipolytica JMY1212
XPR2-Mcfp-3
JMP62
Yarrowia lipolytica JMY1212
Mcfp-3
JMP62
Yarrowia lipolytica JMY1212
His-ST1
JMP62
Yarrowia lipolytica JMY1212
His-ST2
JMP62
Yarrowia lipolytica JMY1212
His-ST3
JMP62
Yarrowia lipolytica JMY1212
His-ST12
JMP62
Yarrowia lipolytica JMY1212
His-ST13
JMP62
Yarrowia lipolytica JMY1212
His-ST23
JMP62
Yarrowia lipolytica JMY1212
His-ST123
JMP62
Yarrowia lipolytica JMY1212
His-ST1L3
JMP62
Yarrowia lipolytica JMY1212
His-Mcfp-3
JMP62
Yarrowia lipolytica JMY1212
TT180
PSB1C3
E.coli.DH5α
TT240
PSB1C3
E.coli.DH5α
ROX1
PSB1C3
E.coli.DH5α
Php4d
PSB1C3
E.coli.DH5α
pGal1
PSB1C3
E.coli.DH5α
Panb1
PSB1C3
E.coli.DH5α
XPR2-Mcfp-3
PSB1C3
E.coli.DH5α
Development
Ptrp1-GFP
PSB1C3
E.coli.DH5α
Ptrp2-GFP
PSB1C3
E.coli.DH5α
Ptrp3-GFP
PSB1C3
E.coli.DH5α
Cooperation
CI
PSB1C3
E.coli.DH5α
2. The Protocol of Molecular Experiment
(1) The PCR Reaction System
Components (50uL)
Volume(uL)
PrimerStar Buffer
10
dNTPs(2.5mM)
4
Primer-F(10uM)
1
Primer-R(10uM)
1
Template
1
PrimerStar
0.5
ddH2O
32.5
To find the optimal temperature for our parts composite amplification, we set a gradient of annealing temperature. The result shows that 58℃ is suitable to the PCR.(Specific temperature according to different primers of components)
(2) The double enzyme digestion system (Q.cut)
Components (50uL)
Volume(uL)
10 x H buffer
5
EcoRI
1.5
PstI
1.5
ddH2O
1
Conditions
37℃ 30~40min
(3) The gene conjunction system
Components (10uL)
Volume(uL)
Solution I
5
Plasmid Skeleton
0.5
Insert Gene
4.5
Conditions
16℃ 1h
(4) Transforming the Plasmids into E. coli DH5a Strain
• The conjunction product was transformed into E. coli DH5a Strain. The strains were grown in LB plate medium containing 25ug/mL chloramphenicol at 37℃.
(5) Reaction System of Colony PCR validation
Components (10uL)
Volume(uL)
Es Taq Mix(2×)
5
Primer-F(10uM)
0.3
Primer-R(10uM)
0.3
ddH2O
4.4
3. The Transformation of The Yarrowia Lipolytica
(1) Linearize the plasmid by digestion
Unit：uL
Plasmid
30(plasmid in higher concentration)
50( plasmid in lower concentration)
10 x H buffer
6
8
Triton
6
8
10 x BSA
6
8
Not I
2
2
ddH2O
10
4
Total
60
80
Digest at 37℃ for 3 to 4 h
(2) Separate the digested product by agarose gel electrophoresis. The larger size of fragment should be extracted.
（The digested product should be in high concentration because of the low transformation rate of the Yarrowia lipolytica）
(3) Transformation (one-step protocol)
a) Inoculate the Yarrowia lipolytica into YPD medium (5mL). Incubate for 16 to 20 hr at 28℃ with vigourous shaking.
b) Divide the yeast culture suspension in 1.5mL centrifuge tubes (each tube for 1.4mL liquid) ( All the operations should be in sterile environment.) Pellet the suspension by centrifugation at 12000rpm for 2min. Decant the medium and discard.
c) Preparation of one-step transformation mixture solution
PEG 40%
90uL
DTT
5uL
LiCl2
5uL
Total
100uL
(The solutions can be mixed in 1.5mL sterilized centrifuge tube. All the operations should be in sterile environment.)
(4) Add 90uL mixture and 10uL linearized plasmid in teach tube (from step 2) and mix.（All the operations should be in sterile environment.）
(5) Incubate for 1h at 39℃ in water bath.
(6) Coat the mixture on the YNBDcasein  plate，incubate for 1.5 to 2 days at 28℃.
(7) Pick the transformants.
(8) Extract yeast transformants’ genome
a) Inoculate the Yarrowia lipolytica into YPD medium (5mL). Incubate for 16 to 20 h at 28℃ with vigourous shaking.
b) Aspirate 2mL yeasty suspension in centrifuge tube. Centrifuge at 12000rpm for 2min. Decant the medium and discard.
c) Add 500 ~1000uL sterilized water. Mix and suspend the sediment. Centrifuge at 12000rpm for 1min. Decant the medium and discard.
d) Add 200uL Trition/SDS and suspend the mixture by vortex oscillator.
e) Add 0.2~0.3g acid-washed glass beads and mix them by vortex oscillator.
f) Add 200uL phenol / chloroform / ISO (use the substratum). Mix for 1~2min by vortex oscillator.
g) Add 200uL TE (pH 8.0). Shake fast and mix.
h) Centrifuge at 12000rpm for 10min.
i) Cut the cusp of the pipette tip. Use the cut tip to carefully aspirate the supernatant to a new 1.5mL centrifuge tube.
j) Add 1mL absolute ethyl alcohol in the supernatant and mix. Stand the mixture for 0.5 to 1h at-20℃ (white sediments will form ).
k) Centrifuge at 12000rpm for 10min and discard the supernatant.
l) Stand the tube at 37℃ to dry the ethyl alcohol until the white sediments turn transparent and the smell of alcohol die out.
m) Dissolve the sediments with 30 to 50uL sterilized water.
Verification Experiments
Part 1:Optical Control System
1. β-galactosidase activity assay to verify optical control system Material:
（1）Y187 containing with plasmid PRMH120 and wild-type Y187
（2）1M Na2CO3，ONPG，β-ME，YPDA liquid medium, SD/-Trp liquid medium
（3）Z-buffer：0.04M NaH2PO4, 0.06M Na2HPO4, 0.001M MgSO4, 0.01M KCl
（4）Z-buffer/β-ME: Per 10mL Z-buffer with 27uL β-ME
（5）4mg/mL ONPG: 0.008gONPG dissolved in 2mLZ-buffer/β-ME
Procedure:
(1) Pick up 3 colonies of Y187 containing with PRMH120 and separately inoculate them in 2mL SD/-Trp liquid medium. Fully mix the medium and the colony and split the mixture in 2 tubes (each contains 1mL mixture). One is cultivated in light environment; the other is cultivated in dark environment. Both are inoculated at 28℃ for 16-18 hr with shaking. Pick up 2 colonies of wild-type Y187, and separately inoculate them in 2mL YPDA medium. Fully mix the medium and the colony and split the mixture in 2 tubes (each contains 1mL mixture). One is cultivated in light environment; the other is cultivated in dark environment. Both are inoculated at 28℃ for 16-18 h with shaking.
(2) Add 250uL above-mentioned culture in 2mLYPDA liquid medium（in 20mL small glass bottle）. Each colony is split to 3 bottles. Inoculate them at 28℃ with shaking (200rpm) for 5h.
(3) Pellet 2mL culture by centrifugation at 12000rpm for 30s. Discard the supernatant.
(4) Add 2mL Z-buffer and resuspend. Centrifuge the mixture at 1200rpm for 30s and discard the supernatant.
(5) Add 300uL Z-buffer and resuspend. Add 0.2-0.3g pickling glass beads and vibrate them by vortexing for 2 min.
(6) Cool the mixture with liquid nitrogen for 30s and warm them with water bath at 37℃. Repeat this procedure 3 times. Vibrate them by vortexing for 2 min. (determine the OD at 600 nm)
(7) Aspirate 100uL mixture to a new centrifuge tube. Add 700uL Z-buffer/β-ME and 200Ul ONPG. Fully mix. Warm it at 30℃ in water bath and do the timekeeping until yellow substance appear. Add 400uL 1M Na2CO3 and fully mix to stop the reaction.
(8) Centrifuge the mixture at 1200rpm for 2min. Aspirate the supernatant and add them to 96-well plates (200uL for each well). Use the Multiskan Spectrum to determine the absorbance.
Enzyme-activity unit (U) = 1000*OD420/(OD600*V) *TIME *Dilution Ratio
100uL  min   14
1 U：The amount that one cell catalyzes the conversion of 1uL mol substrate (ONPG) per minute.
Unit：Miller
Part 2: Verication of Cell Surface Display
1.fluorescence immunoassay to verify the cell surface display
(1) The fermentation of recombinant yeast
(2) Aspirate 1mL culture to an Erlenmeyer flask with 50mL YPD. Inoculate the culture at 28℃ with shaking (rotate speed:200rpm) till the OD600 is between 0.4 and 0.6.
(3) Aspirate 1mL cell suspension to centrifuge tube. Pellet the suspension by centrifugation at 2000rpm, 4℃,1min. Discard the supernatant.
(4) Use 1mL PBS buffer (pH=7.4) to resuspend the cells. Pellet the suspension by centrifugation at 2000rpm, 4℃ for 1 min. Discard the supernatant. Repeat this step twice.
(5) Resuspend the cells with 50uL PBS buffer，add 1uL mouse anti-His-6 tag monoclonal antibody. Seal the centrifuge tube the parafilm.
(6) Blend the cell suspension. Put the tube in the ice for 2 h’s standing (blend the suspension with inverting at times during the 2 h) or overnight standing for 4℃.
(7) Pellet the suspension by centrifugation at 2000rpm, 4℃ for 1 min. Discard the supernatant.
(8) Use 1mL PBS buffer (pH=7.4) to resuspend the cells. Pellet the suspension by centrifugation at 2000rpm, 4℃ for 1 min. Discard the supernatant. Repeat this step twice.
(9) Resuspend the cells with 100μL PBS buffer，add 1μL FITC tagged goat anti-mouse IgG antiantibody.
(10)  Blend the suspension. Seal the centrifuge tube the parafilm and wrap it with tinfoil.   Put the tube in the ice and incubate the cells for 1~2h (blend the suspension with  inverting at times during the period).
(11)  Pellet the suspension by centrifugation at 2000rpm, 4℃ for 1 min. Discard the supernatant. Repeat this step twice.
(12)  Suspend the cells with 50μL PBS. Aspirate 8μL suspension to the slide. Cover it with coverslip. Use the fluorescence microscopy to observe under the excitation light of 492nm.
The preparation of PBS pH7.4:
NaCl 8.00g/L
KCl 8.00 g/L
Na2HPO4 3.58g/L
KH2PO4 0.244g/L
Use sterile filter membrane (pore size=0.22 um) to filtrate bacteria.
2.The study of yeast’s silica-binding protein surface display on the glass slide
(1) Preparations
a) Preparation of the buffer：About 300mL 25 mM Tris-HCl (pH= 7.5)
b) Preparation of the object slide ：We invent an observing method in which we use scotch tape to divide the slide into small chambers. It solved the problem of counting the amounts of cells in unit area. It can also use the tension to reduce droplet thickness so that it is more convenient to do microscopic examination. The experience show that after the mid-speed flow form tap washing and wiped by lens paper，the slide has no bacteria remains according to the microscopic examination. It has no influence on the results of experiments. The effectiveness is excellent. We think this method is outstanding and should be extended.
(2) Procedures
a) Inoculate 50 mL culture in 250 mL Erlenmeyer flask overnight. Add 1.5 mL culture to each EP column (2 columns for experimental group and control group). Adjust the bacteria density to approximately the same and mix the culture.
b) Aspirate the about 4.7 to 4.8 ul culture and drop it in the small chambers of the slide. Experimental group and control group both have three slides.
c) Do the microscopic examination immediately. If the densities of what are similar, stand the slides for 10 min.  The samples should stay moist during this 10 min. If they are drying, add 1ul buffer. Take pictures and count the amounts of bacteria before washing.
d) Put the slide into the culture dish with buffer（The component of the buffer: 25 mM Tris-HCl [pH 7.5]3.5 NaCl）. Shake genteelly. After 5 to 10 min shaking, take the slides out flatly. Oversee them under the microscope. Compare the experimental group with control group. Take pictures and count the accounts of bacteria after washing. Compare the accounts of bacteria before and after washing. Analyze the changes.
Part 3: Verification of Flocculating Protein Secretion
SDS-PAGE for flocculating protein
The SDS-PAGE protocol:
Stock Solutions:
1) 1.5M TrisHCl pH 8.3 + 0.4% SDS (adjust pH before you add the SDS). Keep RT.
2) 30% Acrylamide 0.8% Methylene bis Acrylamide or 40% Acrylamide / Methylene bis Acrylamide (ratio: 37.5:1). Keep 4°C
3) 0.5M TrisHCl pH 6.8+ 0.4% SDS (adjust pH before you add the SDS). Keep RT.
4) 10% Ammonium Persulfate (APS). Keep 4°C less than 1 month.
Running Buffer x5
Trizma
0.125M
15.1gr
Glycine
0.96M
72.0gr
SDS
0.5%
5.0gr
ddH2O up to
1L
Keep at RT
Dissolving (Sample) Buffer x5
Glycerol
5mL
SDS
1gr
ß Mercapto Ethanol (or 0.25M DTT)
2.56mL
0.5M TrisHCl pH 6.8+ 0.4% SDS
2.13mL
Bromo Phenol Blue
traces
Keep in aliquots of 1mL at -20C
Separating Gel:
% Acrylamide
10%
10%
12%
12%
15%
Number of Minigels
5
8
5
8
5
1.5M TrisHCl pH 8.3 + 0.4% SDS
7.0 mL
10.5 mL
7.0 mL
10.5 mL
7.0 mL
30% Acrylamide 0.8% Methylene bis Acrylamide
9.3 mL
13.9 mL
11.3 mL
16.9 mL
13.9 mL
H2O
12.3 mL
18.4 mL
9.3 mL
13.9 mL
6.3 mL
10% APS
100 uL
150 uL
100 uL
150 uL
100 uL
TEMED
23 uL
35 uL
23 uL
35 uL
23 uL
% Acrylamide
10%
10%
12%
12%
15%
Number of Minigels
5
8
5
8
5
1.5M TrisHCl pH 8.3 + 0.4% SDS
7.0 mL
10.5 mL
7.0 mL
10.5 mL
7.0 mL
40% Acrylamide / Methylene bis Acrylamide (ratio: 37.5:1)
7.2 mL
10.8 mL
8.6 mL
12.9 mL
10.8 mL
H2O
14.4 mL
21.5 mL
12 mL
17.9 mL
9.4 mL
10% APS
100 uL
150 uL
100 uL
150 uL
100 uL
TEMED
23 uL
35 uL
23 uL
35 uL
23 uL
Add TEMED and APS at the end. Gently swirl the flask to mix, being careful not to generate bubbles. Pipette the solution to a level of 4cm of the top. Add 0.3mL of n-buthanol. A very sharp liquid interface will be visible within 10-20min. Let polymerize the gel for another hour at least. Rinse the surface of the gel with watter before pouring the stacking gel.
Stacking Gel :
Number of Minigels
2
5
8
0.5M TrisHCl pH 6.8 + 0.4% SDS
2.5 mL
4.0 mL
5.2 mL
30% Acrylamide 0.8% Methylene bis Acrylamide
1.0 mL
1.5 mL
2.0 mL
H2O
6.4 mL
9.6 mL
12.8 mL
10% APS
100 uL
150 uL
200 uL
TEMED
10 uL
15 uL
20 uL
Number of Minigels
2
5
8
0.5M TrisHCl pH 6.8 + 0.4% SDS
2.5 mL
4.0 mL
5.2 mL
40% Acrylamide / Methylene bis Acrylamide (ratio: 37.5:1)
0.75 mL
1.1 mL
1.4 mL
H2O
6.6 mL
10 mL
13.4 mL
10% APS
100 uL
150 uL
200 uL
TEMED
10 uL
15 uL
20 uL
Fill each sandwich with stacking gel solution and insert a comb into each place taking care not to trap any bubbles bellow the teeth. The gel should fully polymerized after 1hour. Cover  gel with a nylon wrap.  Keep gels no more than 2 weeks at 4°C.
Sample Preparation
Prior to adding the sample buffer, keep samples at 0°C. Add the SDS sample buffer (RT) to the sample (still on ice), and boil at 100°C immediately 3 to 5 min. Do not leave the sample in SDS sample buffer without heating; endogenous proteases are very active in SDS sample buffer and can cause severe degradation. Once heated, sample could sit at RT for a short time until loading, or at -20°C for a long time.
For a gel thickness of 0.75mm and 15 wells applied 10-25ug protein of a complex mixture, when staining with Coomasie Blue and 0.5 to 5ug for samples for one or few proteins. If silver stain is used 10 to 100-fold less protein can be used.
Samples can be concentrated or interferences (salts, etc.) eliminated with TCA, acetone, TCA-DOC, ethanol, etc. (see attached Protocol). Potassium ions in particular must be removed since they precipitate the SDS.
Some proteins such as nuclear non-histone proteins and membrane proteins, require the presence of 8M urea in the SDS sample buffer to get complete solubilization.
Some membrane bound proteins undergo aggregation at temperatures above 40-50°C. In this case incubate 30min at 40°C with sample buffer.
A shift in the migration distances of proteins with internal disulfide bridges could be observed by incubating samples in SDS in the presence or absence of reducing agents (mercaptoethanol, DTT, DTE, etc)
Staining Solution :
Methanol CP
500mL
50%
Acetic Acid CP
100mL
10%
H2O
400mL
Coomasie Brilliant Blue R
2.5gr
0.25%
Keep flask on dark at RT
Destaining Solution:
Methanol CP
150mL
15%
Acetic Acid CP
100mL
10%
H2O
750mL
Keep flask on dark at RT
Stain overnight at RT or put gel with staining solution 8 sec in microwave on the high position, and then shake for another 15min at RT. Wash with water 2-3 times and distain by several changes of destaining solution in the presence of a sponge.
Part 4:The Test of Microbial Cement Sand Column
Hypotheses and theoretical basis
1. Hypotheses
Silica-binding protein has long self-decomposition time. It can bind with sand and form products visible for naked eyes.The strain will not decompose in large amount within a short time.
2. Theories Scientific literatures show that calcite produced by microorganism can hold loose particles together and form intensified units. This quality of calcite is wildly applied in many fields, such as improving mechanical property and permeability of porous materials, repairing cement-based materials, etc.
Si-tag1+2+3 protein binding with silicon will help attach our strain to sand and rocks. Flocculation protein Mcfp3 will co-function with Si-tag123 to further flocculate the substrates. Therefore, we used the yeasts transformed Si-tag123-PZS and Mcfp3-PZS plasmid to modify its function.
Our engineering strain expresses silicon-binding protein, flocculation protein, and generates CO2 to form CaCO3 with Ca2+, which is more advantageous in forming calcite  comparing with natural microorganism.
Procedure:
1. Build experimental device: constant flow pump, iron support, glass  syringe, quartz sand(4g 100 mesh particle and 36g 60 mesh particle ) gauze, latex tube.
2. Keep the experiment temperature at 28 Celsius degree.
3.Cushion the glass syringe with 0.5cm gauze. Add fine sand. Rinse with distilled water. Remove bubbles from the sand. Put 1cm gauze on the top.
4.Add 15mL yeast solution to the syringe at maximum speed. Seal process for 2 hours. Add 20mL 1:1 mixture of tris-hcl(PH7-7.5) and calcium chloride(1mol/L) to the syringe. Seal process for 3 hours.
5.Repeat procedure 4 for 20 times. Demold. Rinse with distilled water. Remove the top gauze. Dry out the glass syringe in drying oven at 120 Celsius degree.
6.Remove the syringe from drying oven after 24 hours. Compare with the control group.
7.Use wild yeasts as control group and follow the same procedure with the experimental group.
