<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="navbar-header"><DIV id="footer"><P>Footer</P></DIV><CENTER><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color:green;margin-top: 9px;font-weight:bold !important;">≡ Menu ≡</A></CENTER></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI class="dropdown"><A href="#">People </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#">Application </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A href="#">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A href="#">Lab Book </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2018.igem.org/Team:Cardiff_Wales"> Cardiff iGEM 2018</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px; text-align:center !important; color:green !important;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px; color:green !important"> Click to close sidebar <SPAN style="color:transparent">---</SPAN></A></DIV><UL class="nav navmenu-nav" id="sidenav" style="color:green !important;"><LI><A data-toggle="collapse" data-target="#Project_side">Project</A></LI><LI style="color:green !important;"><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI><A data-toggle="collapse" data-target="#Team_side">People</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI><A data-toggle="collapse" data-target="#Human_side">Human Practices </A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Application_side">Application</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI><A data-toggle="collapse" data-target="#Achievements_side">Achievements</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI><A data-toggle="collapse" data-target="#Book_side">Lab Book</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></NAV><DIV class="footer"><DIV id="container"><DIV class="footer-right"><DIV class="pageContentWrapper"><DIV class="footer-left"><H3> With thanks to our sponsors! </H3><H3> Visit our social media pages! </H3></DIV></DIV></DIV></DIV></DIV><DIV class="maincontent"><DIV class="column full_size"><H1>Results</H1><P> We managed to successfully create several new functional PhytoBricks, and added experimental data to already existing ones. Our new PhytoBricks, the GUS and mCherry reporter genes and the RTBV promoter, could prove to be valuable parts within the registry and we have extensive data for these parts.
Unfortunately (or fortunately if you're an aphid), we didn't manage to detect the presence of siRNAs from tobacco leaves that were transiently transformed with our RNAi constructs. This might be because our detection system was sub-optimal, rather than due to a lack of expression. We also did not manage to test the toxicity of these siRNAs on the aphids themselves, due to the final constructs only being produced in the last week of our lab time and as the aphid toxicity assays taking several weeks. Therefore future steps in our project would involve using a more specialised detection protocol for siRNAs, or perhaps a northern blot, and perform the toxicity assay on our aphids to provide a definite toxicity range of our RNAs.</P><H3 style="color:green !important">Characterisation of plants expressing RNAi constructs: Abundant Aphids and Sad Scientists</H3><P>We successfully generated a composite level 1 GoldenGate clone containing five individual level 0 parts (<A href="http://parts.igem.org/Part:BBa_P10101">35S</A>-<A href="http://parts.igem.org/Part:BBa_K2810005">BCR3</A>-<A href="http://parts.igem.org/Part:BBa_K2810000">WRKYintron</A>-<A href="http://parts.igem.org/Part:BBa_K2810006">BCR*</A>-<A href="http://parts.igem.org/Part:BBa_P10401">NosT</A>). This clone was expressed in tobacco leaves, and we attempted to detect expression of this construct using RT-PCR. Below we present some of the final gels that show these attempts to demonstrate transgene expression by RT-PCR. Our protocol relies on our RNAi constructs producing a double stranded intermediate that would be cleaved into siRNAs. Therefore our RT-PCR protocol aimed to detect the expression of the full length unprocessed RNAi construct. </P><H5><CENTER>Figure 1: BCR3 pre-siRNA detection results. Primers were designed to amplify regions of the unprocessed pre-siRNA, using primers 70-73 which bind in the BCR3, BCR3*, or loop of the RNAi constructs. Bands were present in many lanes of the gels, including the positive controls. However, bands were also present in some negative control lanes, possibly suggesting that there was unspecific binding of primers to RNAs (or contaminating DNA) within the plant transcriptome (or genome).
 </CENTER></H5><P>
The above figure shows the RT-PCR using primers that are designed to amplify a sequence that includes sequences from BCR3 and the WRKY intron. For these experiments we extracted RNA from leaf discs from plants infiltrated with <A href="http://parts.igem.org/Part:BBa_P10101">35S</A>-<A href="http://parts.igem.org/Part:BBa_K2810005">BCR3</A>-<A href="http://parts.igem.org/Part:BBa_K2810000">WRKYintron</A>-<A href="http://parts.igem.org/Part:BBa_K2810006">BCR*</A>-<A href="http://parts.igem.org/Part:BBa_P10401">NosT</A>. We generated cDNA using random primers and performed PCR reactions using the stated primers. Unfortunately these experiments were inconclusive. The positive control (denoted by a * in the figure) show that the PCR reactions are working correctly. However, expression of the construct in RT+ reactions lead us to conclude that there is DNA contamination in our RNA samples.
In hindsight this is not the optimal experimental strategy to detect siRNA expression even though this protocol had been successfully used in <A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640634/#!po=26.7857">published experiments.</A> Ideally we would have used a protocol specifically designed to detect the siRNAs generated from the cleaved RNAi construct, but protocols of this type were too expensive for our budget. However, given the success of our other attempts at transient gene expression in tobacco leaves (see below) we would predict that the RNAi construct was expressed but that unfortunately in this case we were unable to definitively prove this expression.
In addition we did not manage to perform a toxicity assay on the aphids themselves, but our laboratory <A href="https://2018.igem.org/Team:Cardiff_Wales/Model">model</A> shows some possible scenarios, assuming a specific toxicity value for our siRNA constructs. In  future, given more time for the project, we would perform this toxicity assay to obtain our own data in an experiment analogous to <A href="https://link.springer.com/protocol/10.1007%2F978-1-62703-986-4_11"> this one,</A> and quantify it by qPCR.</P></DIV><H3 style="color:green !important">A colourful display</H3><P>We had great success with the PhytoBrick-focused side of our project. We managed to successfully create a new PhytoBrick reporter gene, beta-glucuronidase (or GUS), which is widely used in plant biology, but was not within the registry. We made several composite parts containing the GUS reporter outlined in our <A href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">composite parts</A> page. We managed to get high quality experimental data for GUS itself, and for the 35S and RTBV promoters, and <A href="http://parts.igem.org/Part:BBa_P10401:Experience">Nos</A>, <A href="http://parts.igem.org/Part:BBa_P10402:Experience">G7</A>, and <A href="http://parts.igem.org/Part:BBa_P10400:Experience">35S</A> terminators, characterising these parts in the process, further strengthening the PhytoBrick registry. These results can be seen below, and are on each respective part's page. </P>
The left GUS assay shows experiments testing the 35S and RTBV promoters. Clearly expression is constitutive for both, though the RTBV promotor has been extensively reported in the literature to be vascular specific. The right two GUS assays show tests of the three terminators. We concluded from these that the G7 terminator produced strong expression consistently, and the nopaline synthase terminator (NOS), used as standard in all our constructs, has the most inconsistent expression. Ideally we would perform more assays on these terminators with each of GUS, mCherry and eGFP, but from these results we would suggest that future iGEM teams generating plant expression constructs should use the G7 terminator to obtain more consistent higher expression.<H5><CENTER>Figure 2: Results using the GUS reporter gene. Using the GUS reporter gene, we managed to characterise the 35S CaMV and RTBV promoters (left). This showed that both have constitutive expression, but that the 35S promoter produces stronger gene expression. We also characterised the influence of the nopaline synthase, G7, and 35S CaMV terminators on gene expression, all with the 35S CaMV promoter (centre and right). These results suggest that the G7 terminator contributes to the strongest and most consistent gene expression.
</CENTER></H5><P>
Initially, we were visualising eGFP in plants, but quickly found that expression levels were very close to background levels. We knew that GFP functions as a dimer, meaning it requires more protein copies to fluoresce effectively. For these reasons, we soon created codon-optimised <A href="http://parts.igem.org/Part:BBa_K2810009">mCherry,</A> which we had excellent results with. The only construct we managed to get even tentative eGFP results with was RTBV-OTMV-eGFP-NosT, which are shown below. The pattern of RTBV driving eGFP expression does not match the expression pattern when using GUS and mCherry, creating uncertainty in this result. In addition, the experiment performed with this construct was assessed prior to changing our protocol to include a heat-shock step before we assessed gene expression. Looking at the GFP experiments to identify differences between the terminators, there is no significant difference in fluorescence between the 3 samples (35S-OTMV enhancer-eGFP-NosT/G7/35ST) and the negative control. As there is no difference between any samples, only the RTBV assay's (bottom panels) quantification report is shown as a table below.</P><H5><CENTER>Figure 3: Results using the eGFP reporter gene. The top three panels show replicates of results to characterise the three terminators, nopaline synthase, G7, and 35S CaMV. There is no significant difference between the eGFP constructs and the negative control leaves. The lower two panels show results when testing the RTBV promoter, with eGFP and the nopaline synthase terminator. The left of these shows raw expression output, and the right quantified values of expression. There was still no significant difference between these constructs and the negative control leaves.
</CENTER></H5><H5><CENTER> Table 1. Raw quantification of eGFP expression. There is no significant difference between sample and control leaves, despite raw values being higher.</CENTER></H5><P>
Finally, we had excellent success with our improved <A href="http://parts.igem.org/Part:BBa_K2810009">mCherry,</A> which we made compatible with GoldenGate and codon-optimised it for <I>Nicotiana benthamiana</I>. It is important to note that chlorophyll autofluoresces in the red region of light (around 680nm), which could explain the very low levels of expression in negative control leaves. Using this reporter, we only had time to perform a single experiment with mCherry under control of either the 35S or RTBV promoter. In this experiment we heat shocked transgenic plants 1 DPI and tested them 3 DPI. This means that the protocol was shortened by one day which may effect the results. The <A href="http://parts.igem.org/Part:BBa_K133055">original mCherry sequence</A> that is present in the registry is not GoldenGate compatible so we couldn't create composite level 1 constructs in order to compare its expression with the GoldenGate compatible tobacco codon-optimised <A href="http://parts.igem.org/Part:BBa_K2810009">variant</A> that we generated.</P><P>In order to demonstrate that our part still functions in bacteria, we created the <A href="http://parts.igem.org/Part:BBa_K2810011">part</A> that contains a bacterial promoter, RBS, our mCherry, and a bacterial terminator. Both the plant and bacterial experiments had extremely promising results, as shown below. Below the figure is the table showing the quantification report of the lower two panels, showing our use of mCherry to quantify our 35S CaMV and RTBV promoters.
</P><H5><CENTER>Figure 4: Results using the mCherry reporter gene. The top left of the figure shows two plate images of our mCherry reporter gene working as expected in E. coli. The top right image shows the results of mCherry under control of the ribitiol control operon, and the nopaline synthase terminator, used for our collaboration with WashU. This has strong expression only at infiltration sites, presumably due to the Agrobacterium expressing it, rather than the plant, and the bacteria are most concentrated here. The lower two panels show our mCherry results when testing the 35S CaMV and RTBV promoters. The left of these shows raw output, and the right shows quantified expression. This suggests that both promoters have constitutive expression, but that the 35S CaMV promoter is stronger.
 </CENTER></H5><H5><CENTER> Table 2. Raw quantification of mCherry expression. The 35S CaMV promoter causes mCherry expression at around 1000 times background level, and the RTBV promoter at between 10-100x background level.</CENTER></H5><P>
Thus, we are happy with our contribution to the PhytoBrick registry, and believe that we have massively improved the opportunities for future plant teams who will use GoldenGate cloning as part of their experiments. We hope that our parts, especially GUS, mCherry, and the RTBV promoter, will be used commonly in the future and that they will be included in future collections. As discussed above because our reporter constructs worked with high efficiency, we think that the the expression of the RNAi constructs was also likely successful even though it could not be verified. Even though the RTBV promoter was meant to have vascular-specific expression, its weak constitutive expression in our tobacco plants suggests that it could still make a useful tool. However, to minimise the amount of siRNA eaten by non-target insects, we would need to find an alternative, truly vascular specific promoter.</P><H3 style="color:green !important">The dry stuff</H3><P>
It wasn't only in the lab that we made progress. We managed to create a piece of <A href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">hardware</A> that proved to be very useful for human practices, especially in helping the public understand plant genetic modification. This side of our project gave us new leads, and allowed us to chat to a wide variety of stakeholders, who voiced their opinions and concerns about GMOs, GM food, and our project. With the help of our supervisor, Dr. Daniel Pass, we managed to <A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">bioinformatically analyse</A> the potential ecological effects if our RNAi constructs were used in field trials, and allowed us to compare this to other strategies. In addition, we modelled several scenarios which could occur when aphids fed on the transgenic plants, in a laboratory, greenhouse, or field setting. We communicated with several teams for <A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">collaborations</A>, and ended up testing a regulatory sequence of DNA in our plants, and 3D printed a model detection chamber for <A href="https://2018.igem.org/Team:WashU_StLouis/Collaborations">WashU,</A> giving detailed feedback. Finally, we completed the InterLab study and provided iGEM with a highly valued +1 to their sample size.
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>