<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="footer"><DIV id="container"><DIV class="footer-right"><DIV class="pageContentWrapper"><DIV class="footer-left"><H3> With thanks to our sponsors! </H3><H3> Visit our social media pages! </H3></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="navbar-header"><DIV id="footer"><P>Footer</P></DIV><CENTER><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color:green;margin-top: 9px;font-weight:bold !important;">≡ Menu ≡</A></CENTER></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI class="dropdown"><A href="#">People </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#">Application </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A href="#">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A href="#">Lab Book </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2018.igem.org/Team:Cardiff_Wales"> Cardiff iGEM 2018</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px; text-align:center !important; color:green !important;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px; color:green !important"> Click to close sidebar <SPAN style="color:transparent">---</SPAN></A></DIV><UL class="nav navmenu-nav" id="sidenav" style="color:green !important;"><LI><A data-toggle="collapse" data-target="#Project_side">Project</A></LI><LI style="color:green !important;"><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI><A data-toggle="collapse" data-target="#Team_side">People</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI><A data-toggle="collapse" data-target="#Human_side">Human Practices </A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Application_side">Application</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI><A data-toggle="collapse" data-target="#Achievements_side">Achievements</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI><A data-toggle="collapse" data-target="#Book_side">Lab Book</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></NAV><DIV class="maincontent"><DIV class="column full_size"><H1>Design</H1><H2 style="color:green !important"><CENTER>Principles</CENTER></H2><P> Being one of the few iGEM teams that uses plants as part of our project, we needed to take into consideration the extra steps required to generate genetically modified plants. More specifically, the transformation procedure was both modular and sequential, requiring transformation of <I> E. coli</I>, then <I> Agrobacterium tumefaciens</I>, followed by infiltration into <I>Nicotiana benthamiana</I>. </P><H4 style="color:green !important"> GoldenGate cloning: </H4><P>As a team, we decided to use the efficient GoldenGate modular assembly system, creating a range of parts that are assembled in a specific order. This relies on using Type IIS restriction enzymes that cut downstream of a recognition site and create sticky ends of the designer's choice. This allows a range of custom sticky ends to be created to allow the effective, clean ligation of different DNA sequences. This system requires the synthesis of DNA that contain external <A href="https://international.neb.com/products/R0580-BsmBI">BsmBI</A>or <A href="https://international.neb.com/products/r0734-esp3i#Product%20Information">Esp3I</A>recognition sites 5'- CGTCTC(N)<SUB>1</SUB> | -3' and 3'- GCAGAG(N)<SUB>5</SUB> | -5', where &quot; | &quot; is the site of endonuclease activity. For our project, these sequences were ordered as gBlocks from IDT. These were then set up in a simultaneous digest/ligation reaction to insert these parts into the level 0 acceptor plasmid, and iGEM plasmid backbone, pSB1C3. These sticky ends are created so that when the level 0 parts ligate into the pSB1C3 plasmid, the cut sites for another Type IIS restriction endonuclease, <A href="https://international.neb.com/products/r0535-bsai#Product%20Information">BsaI,</A>is created. These are the sequences 5' - GGTCTC(N)<SUB>1</SUB> | -3' and 3'- CCAGAG(N)<SUB>5</SUB> | -5'. Following this, a combined reaction with several level 0 parts can be set up that, due to their custom sticky ends, can be simultaneously ligated into one specific order with very high efficiency. Following each digestion/ligation reaction, <I>E. coli</I> would be transformed and grown overnight on antibiotic containing plates.</P><CENTER><H5><CENTER>Figure 1: GoldenGate cloning allows modular construction of constructs. It uses type IIS restriction endonucleases that cut downstream of a recognition sites, creating custom sticky ends. These sticky ends can be designed so that several level 0 parts can be assembled into a level 1 construct with ease in a single reaction. Following cutting by Esp3I (red) and ligation into a level 0 acceptor plasmid (pSB1C3), the cut site of BsaI is created (purple).

</CENTER></H5></CENTER><H4 style="color:green !important"> Bacterial selection: </H4><P>To detect which bacterial colonies had the desired insert, we used both negative and positive selection. Bacteria that had not taken up the plasmid, be it pSB1C3 or pGBA2 for a level 0 and level 1 reaction, respectively, would be killed by the antibiotic in the plates (either chloramphenicol for level 0 plates, or kanamycin for level 1 plates). The resulting colonies on the plates would either be blue, containing the original LacZ gene in the acceptor plasmid which breaks down X-gal, or white, where LacZ had been lost, so X-gal is not cleaved, and the colony remains colourless. Consequently, white colonies would be screened by colony PCR to detect the presence of the insert. Colonies identified as promising were then picked, grown in antibiotic-containing broth, and had plasmid DNA extracted for DNA sequencing. Following this, <I>Agrobacterium tumefaciens</I> were transformed with sequence-confirmed DNA, and grown on plates containing kanamycin and rifamycin. </P><CENTER><H5><CENTER>Figure 2: Identification of colonies from plates requires positive and negative selection. Cells containing level 0 constructs are grown on X-gal, IPTG, and chloramphenicol-containing plates, and the white colonies are picked and tested (left). Cells containing level 1 constructs are grown on X-Gal, IPTG, and kanamycin-containing plates, and have the same blue/white selection procedure. <I>Agrobacterium</I> are grown on kanamycin and rifamycin containing plates, with no blue/white selection.
</CENTER></H5></CENTER><H4 style="color:green !important">Plant transformation: </H4><P><I>A. tumefaciens</I> colonies were subsequently picked, activated, and syringe-infiltrated by hand into around 50 day old <I>Nicotiana benthamiana</I> plants. We found the best transgene expression measuring reporter gene output 4 days post infiltration (DPI), following a heat shock at 37 degrees Celsius for 30 minutes at 2DPI. For more detailed, replicable protocols, please visit our <A href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">protocols</A> page. </P><H5><CENTER>Figure 3: Activated <I>Agrobacterium tumefaciens</I> are hand infiltrated into the underside of the leaf. The tDNA from the <I>Agrobacterium</I> transforms the local plant cells which express the transgene of interest. Plants were heat shocked at 37oC for 30minutes 2DPI, and analysed 4DPI.
</CENTER></H5><H2 style="color:green !important"><CENTER>Iterations</CENTER></H2><H4 style="color:green !important">The core stuff: </H4><P>Initially, we aimed to target the aphid bacterial symbiont, <I>Buchnera aphidicola</I> as an effective aphid insecticide. Very early discussions ruled out the possibility of using a protein based insecticide, due to potential restrictions implemented by plasmodesmata size. We then realised that a nucleic acid, namely an RNA, would not have this issue, so we turned to looking at these as insecticides. Seeing as we were targeting bacteria, we looked into producing a sgRNA to kill the bacteria using its own CRISPR defense system, but a quick <A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">bioinformatic</A> search ruled this out too, as the bacteria lack a CRISPR system. Thus, we decided to express an siRNA, targeting aphid-encoded genes that are essential for the survival of the symbiont, in the hope that these could be stacked with siRNA constructs that are directly essential for the aphids own survival. </P><H4 style="color:green !important">Sticky ends: </H4><P>As we were using GoldenGate cloning, we had to design parts with sticky ends that are complementary, and so would ligate into the correct order. Seeing as we used some parts that were already in the registry, we had to design sticky ends that had not been used before. Typically, teams such as the Cardiff team from 2017, would use only 3 parts in level 1 constructs. These would usually be a promoter, coding sequence, and terminator. However, due to synthesis size limits, we had to create constructs using up to 5 level 0 parts. Whilst GoldenGate cloning reports that around 11 level 0 parts can be combined at once, it means that for each order a different set of sticky ends must be created and used. For example, the most 5' sequence of our constructs was the promoter, with the 5' sticky end having GAGG to ligate to the plasmid backbone, and its 3' end having TACT to ligate to the region of DNA to be second in position. In our case, this was either the OTMV enhancer, or the 5' sequence of our siRNAs. Visit our individual <A href="https://2018.igem.org/Team:Cardiff_Wales/Parts">part</A> pages to find out all the sequences.
</P><H4 style="color:green !important">From BsmBI to Esp3I: </H4><P>Early on in the project, we found difficulty running the level 0 digestion and ligation reactions simultaneously due to the optimum temperature of the restriction endonuclease. Typically, GoldenGate reactions are with digestion and ligation occurring in the same PCR tube, running in a thermocycler. However, initially we were using the type IIS restriction enzyme, BsmBI, which has an optimum temperature of 55 degrees Celsius. Running a cycled reaction with T4 DNA ligase (with an optimum temperature of 16 degrees Celsius and a denaturation temperature of around 65 degrees Celsius) raised some red flags. We found our reaction efficiency to be very low, with some results running the two reactions separately, but much less efficiently. One team member discovered that BsmBI has an isoschizomer, a restriction endonuclease that cuts at the same region, called Esp3I that has an optimum temperature of 37 degrees Celsius, well within the temperature range that allows T4 ligase to maintain is form. Once we were using this enzyme for our reactions, nearly all of them worked first time, with nearly 100% of colonies appearing white after an overnight cycle.</P><H4 style="color:green !important">Changes to GUS reagents: </H4><P>Later on in the project, we found that our GUS staining solution assay was not giving the desired results. One of the researchers in the university, Dr. Scofield, allowed us to use some of his GUS staining solution, and gave us his modified protocol for GUS staining. We immediately found clearer and stronger GUS staining.</P><H2 style="color:green !important"><CENTER>Experimental design</CENTER></H2><P> 
Please see the <A href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A> and <A href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A> pages for thorough and replicable experimental design. This section only provides a very brief overview.</P><H4 style="color:green !important">Fluorescent proteins: </H4><P>To visualise fluorescent proteins, we examined infiltrated plants 4DPI as whole leaves, cut from the parent plant, under a the Photon Imager. Both eGFP and mCherry were visualised this way.</P><H4 style="color:green !important"> Beta-glucuronidase (GUS) expression: </H4><P>GUS expression levels were analysed 4DPI through a staining assay. This involved cutting leaf discs from plats, lysing the leaf membranes with 90% acetone, washing with NaPO<SUB>4</SUB> buffer, and staining overnight with staining solution. Following this, leaf discs were destained with ~100% ethanol.</P><H4 style="color:green !important"> siRNA expression: </H4><P>We aimed to produce RNAi constructs that would degrade mRNAs for the BCR3 and SP3 genes within the aphid pest <I>Myzus persicae</I>. These two genes are thought to be essential for the aphid's bacteriocytes, which are cells that harbour their essential bacterial symbiont, <I>Buchnera aphidicola</I>. Without these bacteria, the aphids cannot metabolise their food effective and die. We aimed to express these constructs in tobacco plants, where the aphids would feed from the vascular tissue, take up the processed siRNAs and be affected by them. 
So, we aimed to detect the presence of our siRNA constructs within the tobacco leaves which were ground up 4 days post infiltration. RNAi has been used as an insecticide against aphids previously, but never before to target this symbiotic relationship. 
Despite not being able to detect siRNAs, their presence in leaves was attempted to be detected using 'semi-quantitative' PCR methods, involving primer pairs that should anneal to the pre-siRNA, and amplify a region. This method has been used to detect siRNA presence before, but is not a dedicated method. The primer binding sites can be seen below for our BCR3 RNAi construct. The logic behind this is that a population of siRNAs produced in the plant would not be folded or processed, meaning the loop from the intron would still be present, and primers could still bind in the BCR3 or BCR3* sequence. Due to lack of time and resources, siRNA specific kits were not used, but could be in the future. On the diagram below, the numbers (70-73) denote different primer sequences, and their arrows their direction of amplification.
</P><H5><CENTER>Figure 4: The primers used to detect the presence of our RNAi constructs. Primers 70-73 were used to amplify our pre-siRNAs from plant tissue. Their directions are as indicated in the diagram, and we used a variety of different combinations.
</CENTER></H5><H4 style="color:green !important"> Toxicity assays: </H4><P>We aimed to test the toxicity by allowing aphids to feed on the transgenic plants and measuring a change in birth or death rate over several generations. However, this was not achievable in the time scale, but could be with a one or two months more work. This was going to involve putting adult aphids (parents) on agar-supplied leaf discs, and taking off any nymphs onto another leaf disc (F1), then their offspring to another (F2), and so on for as many generations as possible, with the hope that we would see a gradual decline in aphid populations when feeding on the transgenic plants relative to the controls.</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>