<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_Bioinformatics skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/Bioinformatics</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="footer"><DIV id="container"><DIV class="footer-right"><DIV class="pageContentWrapper"><DIV class="footer-left"><H3> With thanks to our sponsors! </H3><H3> Visit our social media pages! </H3></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="navbar-header"><DIV id="footer"><P>Footer</P></DIV><CENTER><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color:green;margin-top: 9px;font-weight:bold !important;">≡ Menu ≡</A></CENTER></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI class="dropdown"><A href="#">People </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#">Application </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A href="#">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A href="#">Lab Book </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2018.igem.org/Team:Cardiff_Wales"> Cardiff iGEM 2018</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px; text-align:center !important; color:green !important;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px; color:green !important"> Click to close sidebar <SPAN style="color:transparent">---</SPAN></A></DIV><UL class="nav navmenu-nav" id="sidenav" style="color:green !important;"><LI><A data-toggle="collapse" data-target="#Project_side">Project</A></LI><LI style="color:green !important;"><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI><A data-toggle="collapse" data-target="#Team_side">People</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI><A data-toggle="collapse" data-target="#Human_side">Human Practices </A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Application_side">Application</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI><A data-toggle="collapse" data-target="#Achievements_side">Achievements</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI><A data-toggle="collapse" data-target="#Book_side">Lab Book</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></NAV><DIV class="maincontent"><CENTER><H1> Bioinformatics</H1></CENTER><H2 style="color:green !important"><CENTER>Identifying a target</CENTER></H2><P>
When early discussions about the project took place, we considered targeting the aphid microbiome using single guide RNAs (sgRNAs), hoping that the bacteria they hold have a CRISPR system that we could essentially hijack. However, this idea was quickly put to rest as we ran a short bioinformatic test to identify whether the bacteria had a CRISPR system. These results can be seen below, and show that there is no identifiable CRISPR system naturally in <I>Buchnera aphidicola</I>.
</P><H5><CENTER>Figure 1. Bioinformatic analysis showing that <I>Buchnera aphidicola</I> lack a CRISPR system. </CENTER></H5><P>
The above screenshot shows &quot;3 Analysed Sequences&quot; as the bacterial genome FASTA sequence was assembled into 3 contigs. This genome can be found on NCBI, or <A href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000009605.1?expt_dict=b&amp;utm_expid=.AUTBVZdtRVGcQs7InIDH9A.2&amp;utm_referrer=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fsearch%2F%3Fexpt_dict%3Db%26term%3Dbuchnera%2Baphidicola%26dict%3Dgquery_dev">here.</A></P><H2 style="color:green !important"><CENTER>Integrating Human Practices - Protecting Local Fauna</CENTER></H2><P>For our human practices, we communicated with several stakeholders about our project, and their general knowledge and opinions of GM. One concern we found, especially when talking to retirees, was for the well-being of the local fauna. More specifically, we initially spoke to an amateur beekeeper, who expressed concerns about honeybee welfare by raising a previously unknown issue. That is, they suggested that wasps feed on aphid honeydew, and have been known to attack bees when they are low on food. Obviously any risk to bee populations would make the project completely unfavourable, so we took up the issue with the <A href="http://www.wbka.com/">Welsh Beekeepers’ Association (WBKA).</A> These professional beekeepers rapidly informed us that the honeybees themselves actually feed on aphid honeydew. The reduction of a food source for these bees is of minor concern; instead, the risk of the siRNAs that are secreted in the aphid honeydew raises more immediate and interesting concerns. 

To address these concerns and prove that our project is safe for the field, we analysed binding sites of every potential siRNA produced from our RNAi gene constructs, and ran a BLASTn of each of the produced siRNAs against any of the transcriptomes of any aphid predators, bees, and humans. Where the transcriptome sequences were not available, we used the genomic sequences. Here, it is important to note that as siRNAs only bind to RNA sequences, they can only affect transcribed sequences, so much of the matches to genomic sequences will not be in coding regions. This is why we used transcriptomic sequences where available. For our target organism, <I>Myzus persicae</I> we used both genomic and transcriptomic sequences to allow comparison between other samples of the same type.

 In addition, many genomes were not available for direct aphid predators, so instead of ruling these out, we took the genome sequence of the next closely available organism(s), moving up taxa, using the assumption that the genomic regions between these relatives and the actual predators must be at least somewhat conserved, and thus still serve as good indicators. Finally, we did not attempt to analyse the genomes of organisms that may eat the honeydew, but not the aphid directly, as this information is fairly sparse and indefinite. However, if interested, one can download the program from the link at the bottom of this page, and run the analysis themselves.
</P><H2 style="color:green !important"><CENTER>Modelling toxicity <I>in silico</I></CENTER></H2><P>To analyse the potential toxicity of each siRNA, we entered each of their full FASTA sequences (BCR3, SP3, and C002) into the program, which splits them into a full complement of 21 nucleotide siRNAs (for maximum siRNA toxicity, assuming the minimum length for an siRNA is 21 nucleotides) using a single base sliding window. Each of these output siRNAs was then run against each of the genomes or transcriptomes we were interested in with a BLASTn, appropriately paramitised for short sequence searches. Standard BLAST alignment scoring optimises for non-gapped and non-mismatched results, so to ensure accurate alignment we implemented a strict E-value cutoff of &lt;0.05. As a result, all matches are of 16-21 nucleotides in length with 100% alignment. We also specified that these must be unique matches, which removes multiple hits from the same siRNA of slightly different lengths (16-21 nucleotides) in the same sequence. Adding mismatches would rapidly increase the E-values to non-significant levels using the standard BLAST model. In the future, we would run a motif analysis that would allow for gaps and mismatches, and may increase the number of hits as a result. However, if one desires to run the script themselves, these values can be changed, showing a greater number of results as the minimum limit for the E-value increases. Below you can see the output windows with some annotation for each of the three siRNAs.
</P><H3 style="color:green !important">BCR3</H3><P>
The full unedited output for BCR3 can be downloaded as an xlsx file <A href="https://2018.igem.org/File:T--Cardiff_Wales--BCR3_complete_bioinformatics.xlsx">here.</A> This shows where each siRNA has matches in the host genome, allowing for more detailed analysis. In addition, should the above PDF not load, it can be downloaded <A href="https://static.igem.org/mediawiki/2018/d/d2/T--Cardiff_Wales--BCR3BioinformaticsShort.pdf">here.</A> Despite us running the analysis against both the genome and transcriptome of <I>Myzus persicae,</I> hits only showed up against the genome for the BCR3 RNAi construct, unlike the other two RNAi constructs which both had hits against the genome and transcriptome. A graphical output is seen in Figure 2 below.
</P><H5><CENTER>Figure 2. Graphical representation showing the number of hits from a BLASTn search of each possible siRNA produced from BCR3. Note that the Y axis is logarithmic.</CENTER></H5>
For example, the above PDF shows a single match against <I>Apis mellifera</I>, the Western honeybee. Of course, harming this species could have huge ecological consequences, so before our insecticides could be applied we would need to ensure that it could cause no harm to essential species. When we look at where the siRNA binds in the transcriptome of the bee, we find the following result:<H5><CENTER>Figure 3. Bioinformatic analysis showing transcriptomic hit location against <I>Apis mellifera</I>. The region has been annotated and has been decided it is non-functional, and the siRNA is thereby safe.</CENTER></H5><P>
The locus in the dataset where the siRNA matched to has been removed as it was not predicted in a newer annotation since running the bioinformatic analysis. Therefore, this siRNA is safe to the honeybee, according to its current sequencing data (01/10/2018). </P><H3 style="color:green !important">SP3</H3><P>
The full xlsx file showing where each siRNA hits within each genome can be downloaded <A href="https://2018.igem.org/File:T--Cardiff_Wales--SP3_unprocessed.xlsx">here.</A> If the above PDF doesn't load, this can also be downloaded <A href="https://static.igem.org/mediawiki/2018/b/b2/T--Cardiff_Wales--SP3_processed.pdf">here.</A> A graphical output is seen in Figure 4 below.
</P><H5><CENTER>Figure 4. Graphical representation showing the number of hits from a BLASTn search of each possible siRNA produced from SP3. Note that the Y axis is logarithmic.</CENTER></H5>
This siRNA has no hits against the honeybee genome. The outputs at the bottom of the PDF were once hits against transcripts, but NCBI has had annotation added to them, and considers them non-functional, and thereby safe.<H3 style="color:green !important">C002 (positive control)</H3><P>
The full xlsx file can be downloaded <A href="https://2018.igem.org/File:T--Cardiff_Wales--C002_unprocessed.xlsx">here</A>, and the above PDF, should it not load, can be downloaded <A href="https://static.igem.org/mediawiki/2018/1/12/T--Cardiff_Wales--C002_processed.pdf">here.</A> This is the positive control, an siRNA that has been used in research before and targets transcripts in the salivary glands of aphids. A graphical output is seen in Figure 5 below.
</P><H5><CENTER>Figure 5. Graphical representation showing the number of hits from a BLASTn search of each possible siRNA produced from the positive control C002.</CENTER></H5>
The C002 siRNA is a much shorter sequence than BCR3 and SP3, meaning fewer siRNAs are produced, reducing the potential off-target effects. In future, we would optimise our BCR3 and SP3 sequences using this bioinformatic analysis, to create a shorter pre-siRNA from these genes, thus reducing the chance of off-target effects, potentially to include sequences that only occur in the aphid genomes. Of course, as more complete sequencing data becomes available, these results have the potential to change. 
<H2 style="color:green !important"><CENTER>Conclusions</CENTER></H2><P>
As you can see, there are potential risks of the deployment of our siRNAs. However, for genomes with annotation, such as the <I>Apis mellifera</I> genome, when we analyse the locus of the siRNA match, we find no alignment to known genes (some previous gene models did identify matches, but are now defunct), thereby making it safe to bees. Initially, we wanted to analyse other loci where the siRNAs matched, but quickly found that these genomes are currently (01/10/2018) completely unannotated, or have such poor functional annotation that it is impossible to actually look at the function of the DNA at the matched loci. It was possible for <I>Apis mellifera</I> as this genome has been extensively analysed, and because we used the transcriptomic data due to it being available. Due to the other organisms sequences being genomic as opposed to transcriptomic, there is a high chance that the regions where our siRNAs matched are actually not transcribed, making the siRNA safe. However, because of the lack of annotation, we cannot completely rule this out. In the future, as more annotation becomes available, a clearer picture will be painted, and more accurate conclusions can be drawn.

 Of course there are likely many more organisms that may come in contact with these siRNAs due to massive food webs, but we decided to focus our attention on the most important species, or those that are most likely to come in contact with them. We also set the minimum e-value to be 0.05. As you can see, all of the hits against the host aphids genome (<I>Myzus persicae</I>) have the lowest e-values of 0.002, and therefore have the highest accuracy. When you increase the minimum e-value, to 0.5 for example, hits do come up against other genomes, including the human genome. This is because the e-value is calculated from the bit-score, a complex algorithm calculated based on the number of matches, mismatches, or gaps between sequences. These bit-scores can be seen on the original xlsx files that can be downloaded.

 Fortunately, for those who are interested or want to try their own analysis, a link to the bioinformatics script can be found on Dr. Daniel Pass's <A href="https://github.com/passdan/IGEMCardiff2018">GitHub.</A> This script could prove to be a very useful tool for anyone who wishes to analyse potential siRNA toxicity. The input sequence to be made into siRNA sequences can be changed, the siRNA length (the default is 21 because siRNAs are usually 21-25 nucleotides in length, and so 21 nucleotides provides the highest toxicity, as shorter sequences are likely to come up with more matches, and will include all matches for siRNAs of larger lengths). In addition, the genomic or transcriptomic sequences to have a BLASTn run against can be changed. Thus, this tool is very flexible, and potentially useful to other siRNA using projects, be them for iGEM or not. Finally, the script comes with annotated help for all the variables and how to change them in the command. To see this, in the command window, enter the script name with the &quot;-h&quot; or &quot;-help&quot; switch (note, you don't need the quotation marks!).
</P></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>