<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Cardiff_Wales_Plant skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Cardiff Wales/Plant</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="footer"><DIV id="container"><DIV class="footer-right"><DIV class="pageContentWrapper"><DIV class="footer-left"><H3> With thanks to our sponsors! </H3><H3> Visit our social media pages! </H3></DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="navbar-header"><DIV id="footer"><P>Footer</P></DIV><CENTER><A type="button" data-canvas="body" id="sidebarCollapse" class="navbar-toggle" data-toggle="offcanvas" data-target="#sidebar" style="color:green;margin-top: 9px;font-weight:bold !important;">≡ Menu ≡</A></CENTER></DIV><DIV class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav pull-right"><LI class="dropdown"><A href="#">Project </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI class="dropdown"><A href="#">People </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#">Human Practices </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="#">Application </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI class="dropdown"><A href="#">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI class="dropdown"><A href="#">Achievements </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A href="#">Lab Book </A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></DIV></DIV><NAV id="sidebar" style="z-index: 10001; background-color: white;border: none;box-shadow:1px 0px 0px #393939;" class="navmenu navmenu-default navmenu-fixed-left offcanvas" role="navigation"><A class="navmenu-brand" href="https://2018.igem.org/Team:Cardiff_Wales"> Cardiff iGEM 2018</A><DIV id="sidebarHide" data-toggle="offcanvas" data-target="#sidebar" style="padding-left: 20px; text-align:center !important; color:green !important;"><A style="font-family: 'Josefin Sans' !important; font-size: 16px; line-height: 18px; height: 18px; color:green !important"> Click to close sidebar <SPAN style="color:transparent">---</SPAN></A></DIV><UL class="nav navmenu-nav" id="sidenav" style="color:green !important;"><LI><A data-toggle="collapse" data-target="#Project_side">Project</A></LI><LI style="color:green !important;"><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Description">Description</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Design">Design</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Experiments">Experiments</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/InterLab">Interlab</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Plant">Plant</A></LI></UL><LI><A data-toggle="collapse" data-target="#Team_side">People</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Team">Team</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Attributions">Attributions</A></LI><LI><A data-toggle="collapse" data-target="#Human_side">Human Practices </A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Human_Practices">Human Practices</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Public_Engagement">Public Engagement</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">Bioinformatics</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Survey">Survey</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Collaborations">Collaborations</A></LI><LI><A data-toggle="collapse" data-target="#Application_side">Application</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Model">Modelling</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Hardware">Hardware</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Applied_Design">Applied Design</A></LI><LI><A data-toggle="collapse" data-target="#Parts_side">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Parts">Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Basic_Part">Basic Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Composite_Part">Composite Parts</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Part_Collection">Part Collection</A></LI><LI><A data-toggle="collapse" data-target="#Achievements_side">Achievements</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Results">Results</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Medal_Fulfillment">Medal Fulfilment</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Improve">Improve</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Demonstrate">Demonstrate</A></LI><LI><A data-toggle="collapse" data-target="#Book_side">Lab Book</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Notebook">Notebook</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/Protocols">Protocols</A></LI><LI><A style="color:green !important;" href="https://2018.igem.org/Team:Cardiff_Wales/gallery">Gallery</A></LI><LI><A href="https://2018.igem.org/Team:Cardiff_Wales/Safety">Safety</A></LI></NAV><DIV class="maincontent"><DIV class="column full_size"><H1> Plant Synthetic Biology</H1><P> As a team, we believe we are eligible for the 'Best Advancement in Plant Synthetic Biology' prize for several reasons. Firstly, we attempted to tackle a very serious and worldwide pest problem, using a fairly unexplored system in a unique way. Secondly, and perhaps most importantly, we added several parts to the iGEM phytobrick registry, all of which are GoldenGate compatible for teams who wish to use the system, but also can be used with the conventional iGEM cloning if desired. Two of these parts are the two potentially extremely useful reporter genes, GUS and mCherry, both codon optimised for <I>Nicotiana benthamiana</I>. Thirdly, we experimentally characterised several parts that are used in plant synthetic biology, through a series of experiments using different reporter genes. </P><H2 style="color:green !important"><CENTER> The adversity of aphids </CENTER></H2><P> Aphids are global crop pests, reportedly causing severe damage to crops, reducing yields by highly significant numbers. In Australia alone, for example, <I>Myzus persicae</I> caused Canola yields to decline by around 34%, and lupin yields a decline of 43%, costing the country 241 million and 482 million Australian dollars per year due to direct feeding and indirect virus transmission, respectively<SUP>(1)</SUP>. In Pakistan, damage from aphids directly, due to feeding, only represents an average <I>Triticum aestivum L.</I> (wheat) yield decline of between 35-40%, compared to between 20-80% indirectly due to viral or fungal disease transmission<SUP>(2)</SUP>. Consequently, attempts are made to control aphid populations. Traditionally, chemical insecticides are used for this purpose, but the Grains Research and Development Corporation (GRDC) has shown that aphids are developing resistance to these pesticides<SUP>(3)</SUP>. They found that <I>Myzus persicae</I> have developed widespread resistance to Pirimicarb, an aphid-specific insecticide that causes minimal harm to other invertebrates. They also found resistance to other carbamates, synthetic pyrethroids, and organophosphates. The authors also note that resistance to these widely used pesticides has been found outside of Australia, up to 10,000 times resistance of control populations. Of course, this means that researchers are attempting to find chemical alternatives, which is costly, time-consuming, and depending on the chemical product, potentially have off-target effects. 

 Genetic modification opens up new avenues to explore. It is possible that expressing toxic proteins, such as Bt Cry toxins or VIPs, in leaf tissues could target these insects. However, this increases the risk to other organisms that eat the leaf tissue, especially when constitutively expressed under control of promoters like the 35S cauliflower mosaic virus (35S CaMV) promoter. Due to insect anatomy often being very similar, it also still poses fairly high risk of off-target effects. This is demonstrated, for example, by the diversity of Bt Cry toxins and VIP proteins, all of which work by targeting and degrading insect midguts when ingested, with subtly different protein variants working better on different taxa<SUP>(4)</SUP>. However, resistance has developed to these proteins in many different insects. Furthermore, aphids feed from leaf vascular tissue, taking sap from the plant phloem. The 'hollow' regions of phloem contain the sap, and are living, but only due to support of essential organic compounds from supporting cells called companion cells. These companion cells supply these hollow sieve cells with small proteins, sugars and other molecules essential for the sieve cells, through pores linking the cells called plasmodesmata. However, these have size restrains, preventing larger sized molecules from freely diffusing between cells<SUP>(5)</SUP>. To get around this problem, a smaller toxic molecule can be used, such as an RNA.</P><CENTER><H5><CENTER>Figure 1: There is a size limit to what can pass through plasmodesmata. Aphids feed on sap contained within the sieve cells of phloem. These sieve cells are living but hollow tubes, with their essential molecules supplied by companion cells, and are joined by plasmodesmata. These plasmodesmata have size restrictions, preventing large proteins from passing though. Nucleic acids are small enough to pass into the sieve tubes.
</CENTER></H5></CENTER><P> RNAi has been shown to work as an effective pesticide against aphids, though hasn't been widely adopted yet. Researchers showed that producing RNAi constructs against aphid genes has successfully controlled aphid population dynamics, showing that the effects can be transgenerational too<SUP>(6)</SUP>. Expressing an RNA that interferes with the interaction between the aphid and the host plant, by targetting a gene, C002, expressed in aphid salivary glands<SUP>(7)</SUP>. However, salivary glands are not unique to aphids, and this could have off-target effects. We found that all aphids have specialised cells called bacteriocytes, which harbour an essential bacterial symbiont, <I>Buchnera aphidicola</I>. These are unique to aphids, and are essential for their survival. Thus, we decided to produce an RNAi construct that targets aphid-encoded genes that allow development of these bacteriocytes, and thus are essential for this symbiotic relationship. We performed <A href="https://2018.igem.org/Team:Cardiff_Wales/Bioinformatics">bioinformatic</A> analysis, influenced by our human practices, to assess any potential unforeseen effects.

Thus, our project has been thoroughly researched, and a potentially extremely viable alternative to traditional treatments found, and developed, for anyone to use with the GoldenGate system. The bioinformatic tool developed could also be invaluable outside of plant synthetic biology, for anyone who wants to assess any potentially toxic effects of any ingested siRNA to any organism, simply by changing the inputted siRNA FASTA sequences, and the genomic or transcriptomic sequences that these should have a BlastN run against.

</P><H2 style="color:green !important"><CENTER> Reporting on reporters </CENTER></H2><P>One of the key aims of Cardiff iGEM teams is to expand the highly limited Phytobrick registry, by introducing useful parts for other teams to use in plants. This year, we have added several useful parts, but the ones we are most happy with are the reporter genes. The Cardiff iGEM team of 2017 added GoldenGate compatible luciferase for expression in plants. For the 2018 team, we have used three. eGFP was the initial reporter gene we used, which was already in the registry before us. We found that eGFP functions as a dimer, and fluoresces at a similar wavelength of light to background fluorescence produced in low amounts by chlorophyll. This makes eGFP data in plants somewhat noisy, so we thought it would be better to use a fluorescent protein that fluoresces at a different wavelength to green light, naturally not absorbed by chlorophyll, and ideally one that functions as a monomer, so should have higher and more consistent expression. To do this, we took a sequence of mCherry from the iGEM registry, part <A href="http://parts.igem.org/Part:BBa_K133055">BBa_K133055,</A> codon optimised it for tobacco plants, and made it GoldenGate compatible, creating the new and improved part <A href="http://parts.igem.org/wiki/index.php?title=Part:BBa_K2810009">BBa_K2810009,</A> optimal for expression in plants. Chlorophyll does autofluoresce in the red region of light (around 680nm), potentially explaining the low level fluorescence in the controls and some of the RTBV leaves. In addition, we decided to add GoldenGate compatible beta-glucuronidase (GUS) to the registry. This is a widely used reporter gene in plants, and we were unable to find a level 0 part of exclusively the GUS sequence in the registry. We did manage to get some tentative results with eGFP, but as you can see below, these expression patterns were extremely close to the background level, and do not show the same localisation patters as GUS and mCherry. </P><H5><CENTER>Figure 2: Characterisation of promoters. We used 3 reporter genes to attempt to characterise the promoters. The top panel shows our results using the GUS reporter gene in leaf discs. It shows that both RTBV and 35S promoters are constitutive in tobacco, but 35S is stronger. The middle two panels show our results using mCherry, with the left of the two panels showing raw expression, and the right showing a heatmap with data values. This also shows that 35S has stronger expression. The lower two panels show eGFP used to test the RTBV promoter. Despite the left of these two panels having clear fluorescence, it is not significant relative to the controls, and the expression pattern does not match the other two reporter gene constructs.
</CENTER></H5><P>
As the images above show, we also added a new functional promoter sequence to the registry. We wanted to use a promoter that uniquely expressed in the vasculature of the plant, so as to reduce the amount of siRNA that any leaf-eating organisms ingested. Thus, we tried to characterise two promoters, RTBV<SUP>(8)</SUP> and Athspr<SUP>(9)</SUP>, that have both been shown to transcribe DNA exclusively in plant phloem and vascular tissue, respectively. We were unable to successfully clone the Athspr promoter into a level 0 acceptor plasmid within the 10 weeks we had in the lab, but we managed to successfully run the full course of reporter gene expression with the RTBV promoter. The images above indicate the leaves that had which reporter gene under control of which promoter. Clearly, the RTBV promoter did not have expression exclusively in the vascular tissue (except for eGFP results, bizarrely), but rather an expression pattern similar to a weaker version of the 35S CaMV promoter. Therefore, this promoter could also be a useful addition to the phytobrick registry. Perhaps a shorter version of this promoter may have exclusively vascular expression, as suggested by other researchers<SUP>(10)</SUP>. Thus, this promoter sequence could be developed further for truly vascular specific gene expression.
</P><H2 style="color:green !important"><CENTER> Defining DNA </CENTER></H2><P>Finally, we have characterised several DNA sequences that are already in the registry. This includes the 35S CaMV promoter, part <A href="http://parts.igem.org/Part:BBa_P10003">BBa_P10003</A>, and three terminator sequences, from 35S CaMV, Nopaline synthase gene sequence, and G7. These are parts <A href="http://parts.igem.org/Part:BBa_P10400">BBa_BBa_P10400,</A><A href="http://parts.igem.org/Part:BBa_P10401">BBa_P10401,</A> and <A href="http://parts.igem.org/Part:BBa_P10402">BBa_P10402,</A> respectively. While similar parts had experimental characterisation previously, these specific ones had not. Some of the data can be seen in the images below. All of these parts have the 35S CaMV promoter element, and terminator sequences are indicated.</P><H5><CENTER>Figure 3: Characterisation of terminators. We used 2 reporter genes to attempt to characterise the terminators. We made several attempts using eGFP to measure expression levels when different terminators were used. All promoters were using the 35S CaMV promoter. The top three panels all show replicates testing eGFP expression, but all show that there is no significant difference between each construct and the negative control, suggesting that eGFP is not expressed at all. The lower two panels are two replicates when using GUS as our reporter gene. These suggest that the nopaline synthase terminator is the least consistent and potentially weakest, whilst the G7 terminator is the most consistent and strongest.
</CENTER></H5><H2 style="color:green !important"><CENTER> References </CENTER></H2><P>
(1) - Valenzuela, I. and Hoffmann, A. (2014). Effects of aphid feeding and associated virus injury on grain crops in Australia. Austral Entomology, 54(3), pp.292-305.

(2) - Muhammad, A. (2012). Wheat Crop Yield Losses Caused by the Aphids Infestation. Journal of Biofertilizers &amp; Biopesticides, 03(04).

(3) - Clarry, S. (2013). Insecticide resistance increasing in aphids. [online] Grains Research and Development Corporation. Available at: https://grdc.com.au/resources-and-publications/groundcover/ground-cover-issue-106-sept-oct-2013/insecticide-resistance-increasing-in-aphids [Accessed 23 Sep. 2018].

(4) - Palma, L., Muñoz, D., Berry, C., Murillo, J. and Caballero, P. (2014). Bacillus thuringiensis Toxins: An Overview of Their Biocidal Activity. Toxins, 6(12), pp.3296-3325.

(5) - Ueki, S. and Citovsky, V. (2014). Plasmodesmata-associated proteins. Plant Signaling &amp; Behavior, 9(2), p.e27899.

(6) - Coleman, A., Wouters, R., Mugford, S. and Hogenhout, S. (2014). Persistence and transgenerational effect of plant-mediated RNAi in aphids. Journal of Experimental Botany, 66(2), pp.541-548.

(7) - Mutti, N., Louis, J., Pappan, L., Pappan, K., Begum, K., Chen, M., Park, Y., Dittmer, N., Marshall, J., Reese, J. and Reeck, G. (2008). A protein from the salivary glands of the pea aphid, Acyrthosiphon pisum, is essential in feeding on a host plant. Proceedings of the National Academy of Sciences, 105(29), pp.9965-9969.

(8) - Bhattacharyya-Pakrasi, M., Peng, J., Elmer, J., Laco, G., Shen, P., Kaniewska, M., Kononowicz, H., Wen, F., Hodges, T. and Beachy, R. (1993). Specificity of a promoter from the rice tungro bacilliform virus for expression in phloem tissues. The Plant Journal, 4(1), pp.71-79.

(9) - Zhang, L., Yang, T., Li, X., Hao, H., Xu, S., Cheng, W. and Sun, Y. et al. (2014). Cloning and characterization of a novel Athspr promoter specifically active in vascular tissue. Plant Physiology and Biochemistry 78:88-96.

(10) - Yin, Y., Chen, L. and Beachy, R. (1997). Promoter elements required for phloem-specific gene expression from the RTBV promoter in rice. The Plant Journal 12:1179-1188.
</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>