<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Valencia_Modeling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Valencia/Modeling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Valencia/Modeling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia/Modeling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Valencia/Modeling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Valencia/Modeling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Valencia/Modeling</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><DIV><UL class="menu"><LI class="top"><A href="https://2012.igem.org/Team:Valencia" target="_self" class="top_link"><SPAN>Home</SPAN></A></LI><LI class="top"><A href="https://2012.igem.org/Team:Valencia/Project" target="_self" class="top_link"><SPAN>Project</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Project" target="_self">Overview</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/future" target="_self">Our future vision</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/genetic_engineering" target="_self">Genetic engineering</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/cultures" target="_self">Cultures</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/other_microalgae" target="_self">Testing other microalgae</A></LI></UL><LI class="top"><A href="https://2012.igem.org/Team:Valencia/Drylab" target="_self" class="top_link"><SPAN>Dry Lab</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Drylab" target="_self">Introduction</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Modeling" target="_self">Modeling</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Engineering" target="_self">Engineering</A></LI><LI class="top"><A href="https://2012.igem.org/Team:Valencia/human_practices" target="_self" class="top_link"><SPAN>Human Practices</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Valencia/human_practices" target="_self">Activities</A></LI><LI><A href="http://igemvalencia2012.wordpress.com/" target="_blank">Syn(ethic) Biology -&gt; Blog</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Safety" target="_self">Safety</A></LI><LI class="top"><A href="https://2012.igem.org/Team:Valencia/Team" target="_self" class="top_link"><SPAN>Team</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Team" target="_self">Team description</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/profiles" target="_self">Members</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Acknowledgments" target="_self">Acknowledgments</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/gallery;" target="_self">Gallery</A></LI><LI class="top"><A href="https://2012.igem.org/Team:Valencia/labwork" target="_self" class="top_link"><SPAN>Lab Work</SPAN></A></LI><LI class="tap"><A href="https://2012.igem.org/Team:Valencia/labwork" target="_self">What we did</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/Parts" target="_self">Submitted parts</A></LI><LI class="tap"><A href="https://2012.igem.org/Team:Valencia/prototypes" target="_self">Designed Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/notebook" target="_self">Notebook</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/protocols" target="_self">Protocols</A></LI><LI class="top"><A href="https://2012.igem.org/Team:Valencia/achivements" target="_self" class="top_link"><SPAN>Achievements</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:Valencia/achivements" target="_self">Achievements</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/judging" target="_self">Judging criteria</A></LI><LI><A href="https://2012.igem.org/Team:Valencia/collaboration" target="_self">Collaborations</A></LI></DIV><DIV id="Titulos">Modeling</DIV><DIV id="HomeCenterCenter"><H2><B>Light, sucrose and AHL yield in <I>Synechococcus elongatus</I>:</B></H2>
In first place we adapted a metabolic model of <I>Synechococcus</I> with the functions we included with our constructs. The model is an algorithm applied to a <B>metabolic network</B> of reactions, originally designed for <I>Synechocystis sp.</I> by Montagud et al. 2010 and 2011. and recently adapted for <I>Synechococcus sp.</I>, where optimization of determined fluxes with constraints on certain reactions which make it behave like a living cell.
We transformed a wildtype <I>S. elongatus</I> with the luciferase genes (luxAB) regulated by the promoter psbA. The <B>reactions of bioluminescence</B> expressed by the LuxAB construct were introduced into the network as follows:
<UL style="list-style-type: square"><LI>LuxE8  : octanoic acid + ATP + NADPH -&gt; octanal + AMP + NADP+ + diphosphate</LI><LI>LuxAB8 : octanal + FMNH<SUB>2</SUB> + oxygen -&gt; FMN + H<SUB>2</SUB>O + octanoic acid + light</LI><LI>LuxE14  : tetradecanoic acid + ATP -&gt; tetradecanal + AMP + NADP+ + diphosphate</LI><LI>LuxAB14 : tetradecanal + FMNH<SUB>2</SUB> + oxygen -&gt; FMN + H<SUB>2</SUB>O + tetradecanoic acid + light</LI><LI>LuxE16  : hexadecanoic acid + ATP -&gt; hexadecanal + AMP + NADP+ + diphosphate</LI><LI>LuxAB16 : hexadecanal + FMNH2 + O<SUB>2</SUB> -&gt; FMN + H<SUB>2</SUB>O + hexadecanoic acid + luz</LI><LI>FMN_red : FMNH<SUB>2</SUB> + NADP+ &lt;-&gt; FMN + NADPH</LI></UL>
May be noted that the <B>luciferin</B> (FMNH<SUB>2</SUB>) is reduced (regenerated) by the reaction FMN_red, which <B>is not naturally present in <I>Synechococcus</I></B>, but which we added to our model to ensure light production in a persistent way. As a future aim, we plan to include luciferin regeneration gene cassettes to this wildtype. If we didn’t insert this adjustment, the model would calculate flow 0, as luminescence would extinguish in a few seconds with the scarce FMNH<SUB>2</SUB>, so the model has no solutions for infinite time flow. This helped us to notice (and then assured from bibliographic support) this handicap of cyanobacteria as light producers. This partly triggered the reorientation of our research towards the bispecific coculture idea. 
Moreover, we programmed bioluminescence functions with <B>3 types of fatty acid</B> (8, 14 and 16 carbon chain length) which are most abundant in the cyanobacterial cell so could be used as ‘fuel’ with greatest probability, and lied between the operational chain-lenght limits for the luciferase.

This yielded results as we optimized the model’s algorithm to maximize light production, by adjusting the flow rates of the other reaction of the metabolism under certain restrictions which make it a biologically reasonable maximum (principle of operation for every result we obtained from the algorithm)value of 22509mmol/g DCW/h, equivalent to 6.48x10<SUP>8</SUP>photons/cell/s. 

In second place, we adapted the wildtype model with the <B>sucrose export</B> induced by the expression of the <B>cscB transporter</B> gene included in the transformed strain from Harvard. Here we modeled the maximum values of sucrose export for 2 different growth restrictions, 0.09mmol Biomass/gDCW*/h (a value near to the maximum growth, similar to an averaged exponential phase culture) and 1x10<SUP>-6</SUP>mmol Biomass/gDCW/h (a negligible growth value which will keep all the vital reactions working but not divert a significant amount of fixed carbon to increasing biomass, as in an averaged dynamic equilibrium of a stationary phase culture).
<B>Maximum export values yielded:</B> With minimal growth constraint: 0.3075mmol/g DCW/h
 With maximal growth constraint: 0.027mmol/g DCW/h

As we planned to transform this cscB strain to express luxI, the protein that synthesizes <B>AHL (autoinducer molecule)</B>, we introduced the reaction in the model as follows:

AHL: Hexanoyl-(acyl carrier protein) + S-adenosyl-L-methionine -&gt; AHL + S-methyl-5'-thioadenosine + acyl-carrier protein 

Our biomachine has 2 synthetic functions inserted, for which we have an interest of flow maximization. Unfortunately, export of sucrose and export of AHL, both carbon based molecules, sets a <B>biochemical competition for carbon redirection</B>. Therefore we came out with 3 modeled outputs: Parabolic curve of maximization of AHL+Sucrose (fig a), and the linear functions of AHL vs. sucrose export in both growth scenarios (fig b, c). All units are expressed in mmol/g DCW/h.
<CENTER><B>Fig a.</B> Parabolic curve representing the product of AHL·Sucrose export flows versus net sucrose export constraint, at the maximum growth constrain. The maximum value, of near 0.016mmol/g DCW/h, corresponds to the 0.013mmol/g DCW/h sucrose export constraint.</CENTER><CENTER><B>Fig b.</B> Linear equation representing AHL export versus sucrose export constraint, at the minimum growth constraint.</CENTER><CENTER><B>Fig c.</B> Linear equation representing AHL export versus sucrose export constraint, at the maximum growth constraint. In this scenario, the value of AHL export for a sucrose export of 0.049 mmol/g DCW/h (units adapted from Ducat et al, 2012, at the experimental conditions of our setting) is 0.221mmol/g DCW/h, which is a very high value, near to the the maximum (0.2633mmol/g DCW/h).</CENTER>
We just require 1 or 2 molecules per cell of <I>Aliivibrio fischeri</I>  to induce biolumminescence (Kaplan &amp; Greenberg, 1985). In Fig c we can see the how small sucrose export values, such as our real fluxes affect very little the export of AHL. This was tested on a maximal growth restriction, as the forward idea is to keep the bioreactor in a continuous exponential phase, which corresponds to a flow of 0.00974 mmol/g DCW/h . This imposes a constant flux of sucrose and AHL, which is optimum for our system. The answer of the model to this constant export of sucrose is a flow of 0.015mmol AHL/g DCW/h, which is a pretty high value. As you will see later, this value of AHL export is fundamental for the development of our next model.

-*: DCW=Dry Cell Weight, where an average <I>Synechococcus</I> cell has a dry weight of 3.87ng (Rosales et al. 2005).
<H2><B>Synergic model:</B></H2>
We achieved our main goal of <B>integrating and connecting all the system into a single model</B> capable of predicting the luminescence of the consortium system in different bioreactor setting scenarios, assuming fast diffusion mechanisms of substances in the medium with a pump system and a pump-filter-return control of population density in both <I>S. elongatus</I> and <I>A. fischeri</I> modules.

The main part is based on the model of bioluminescence regulation developed by Belta et al. 2001 (source of the following figures), a hybrid model of 9 differential equations which predicts the behavior of the whole <I>quorum sensing</I> regulatory mechanism of luciferase expression (including cAMP, AHL, LuxR and LuxI expression) in <I>A. fischeri</I>.

We modified this model to restrict the growth of the <I>A. fischeri</I> population into a <B>constrained volume</B> (biolamp compartment) meanwhile we adopted a greater volume for the dilution of the autoinducer (AHL), as the population cannot cross the membrane to colonize the whole of the medium. Such 'dilution volume' represents the annexing of the photobioreactor of <I>S. elongatus</I> and the extra volume from tubing and pumping systems where the medium flows (50ml).

We gave the system a new input of AHL derived from the <I>S. elongatus</I> population, which has a Ks (constant rate of AHL production/cell resultant from our modified <I>Synechococcus</I> metabolic network model) of  <B>2.38x10<SUP>-7</SUP> nmol AHL /cell/s</B>, multiplying the number of cells (culture density x culture volume – of the photobioreactor compartment). The model still counts with the AHL produced by the <I>A. fischeri</I> population, and the degradation half-life in the cells. As we said above, the <B>diffusion speed of AHL is considered instantaneous</B> (as the rate’s scale is very high compared with other variables, such as cell growth or gene expression).

The application of this model is focused in the <B>optimization of the design of the future bioreactor</B> which may contain the system. We are looking for the values of compartment volumes, relative volume, population densities and total volume which can assure a <B>diel control of luminescence</B> (in 12h cycles, considering export rates and the AHL degradation half-life of 10h) by the light induced activity of our <I>S. elongatus</I> biomachine; and, within those values, the settings with <B>maximum luminescence values</B> (values higher than 1000nM luciferase in the <I>A. fischeri</I> module volume).

A second filter is be applied to the group of solutions obtained from these premises, with the result of a further model on <B>export and consumption rates of sucrose</B>. This filter discards the settings where insufficient sucrose is exported to the culture to sustain the growth of the <I>A. fischeri</I> population. We managed to adapt some experimental values of sucrose export and consumption from the literature:

The export of sucrose from our cyanobacteria <B>(Ke)</B> is: 2.07x10<SUP>-11</SUP> nmol/cell/s (units adapted from Ducat et al, 2012)

The consumption of sucrose by <I>A. fischeri</I><B>(Kc)</B> is: 0.00000150 nmol/cell/s (units adapted from oxygen consumption rates at maximum bioluminescent activity, at normal glycolytic route assumptions in aerobic conditions, from Makemson 1985).<B>dSucrose=Ke·(S. elongatus cell density).(photobioreactor module volume) - Kc·(<I>A. fischeri</I> cell density)·(biolammp module volume)</B> ; where only dSucrose (&gt; or =) 0nM/s are accepted.
This amendment model sets the total volume of dilution, the compartmental volumes of occupation (in a range between 0.001-10l) and the cell densities for S. elongatus cscB and for <I>A. fischeri</I> (in a range between 0 and 10<SUP>9</SUP> cell/ml) as controlled variables.
The output gives the multivariate scenarios where the rate of change of sucrose concentration in the common broth is 0 or positive, <B>so that the energy budget is not a shortfall</B>.
The results of our model were <B>non-unimodal</B>. This means that we cannot rationally present in short the outcome of multivariate solutions we obtained as min/max limits for each variable. Our high resolution discrete outcome is an enormous multidimensional permutation. If we lower the resolution for a presentable report, the interpolation error is too high. We look forward to rationalize this by the means of analytic resolution of the differential equations system into a simpler function applicable to an easy assessment of the design parameters for the bioreactor.
<B>References</B>
Belta, C., Schug, J., Dang, T., Kumar, V., Pappas, G.J., Rubin, H., Dunlap, P. 2001. Stability and reachability analysis of a hybrid model of luminescence in the marine bacterium Vibrio fischeri. Decis. In Cont., 1, 869-874.
Ducat, D. C., Avelar-Rivas, A. J., Way, J. C. &amp; Silver, P. A. (2012) Rerouting carbon flux to enhance photosynthetic productivity. Applied and Environmental Microbiology, 78(8):2660–2668. 
Kaplan H., Greenberg, E.B. 2004. Diffusion of autoinducer is involved in regulation of the Vibrio fischeri  luminescence system. Journ. Of Bacteriol. 1210-1214.
Makemson J., 1985. Luciferase-dependent oxygen consumption by bioluminescent Vibrios. Amer. Soc. for Microb.165(2): 465-466.
Montagud, A., Navarro E., Fernández de Córdoba, P., Urchueguía, J.F., Patil, K.R. (2010). Reconstruction and analysis of genome-scale metabolic model of a photosynthetic bacterium. BMC Syst. Biol. 4:156.
Montagud, A., Zelezniak, A., Navarro E., Fernández de Córdoba, P., Urchueguía, J.F., Patil, K.R. (2011). Flux coupling and transcriptional regulation within the metabolic network of the photosynthetic bacterium Synechocystis sp. PCC6803. Biotechnol. J. 6(3):330-42.
Rosales, N., Ortega, J., Mora, R., Morales, E. 2005. Influence of salinity on the growth and biochemical composition of the cyanobacterium Synechococcus sp. Ciencias Marinas, 31(2): 349–355.</DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Valencia/Modeling">http://2012.igem.org/Team:Valencia/Modeling</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Valencia/Modeling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Valencia/Modeling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Valencia/Modeling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Valencia/Modeling&amp;oldid=236602" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>