"
Page
Discussion
View source
History
teams
Log in
 
Team:Valencia Biocampus/Yeast
From 2012.igem.org
The Team Know the team
Student Members Advisors Instructors The Universities Acknowledgements Attributions Gallery Contact Us
The Project Where ideas get born
Overview Bacteria Yeast Modeling Talking Interfaces Notebook Protocols
Results Our dialogues
Talking to bacteria Talking to yeast Dissecting cheaters Submitted Biobricks
Modeling Inside knowlegde
Introduction Bacteria Yeast for Human Practices
Human Practices Talking life
Ethical issues Safety
Achievements What we did
Yeast Subteam
THE IDEA
Our aim in this part of the project is to detect when the yeast starts fermenting. At the end of the project we will be able to “ask”  the yeast if there is still any glucose in the medium or not through the addition of H2O2. Furthermore, we will be able to know for how long the media has been running out of  glucose. In conclusion, this project allows us to know how much time has elapsed since the fermentation began.
To do this, we are going to use two gene constructions:
• The ADH2 promoter is fused to the YAP1 protein coding sequence. The YAP1 protein is a yeast transcription factor regulator of H2O2 adaptive response. It is stored in the cytoplasm in normal conditions and, in the presence of H2O2, it is transported to the nucleus where it acts as a transcription factor. The ADH2 promoter is activated in the absence of glucose.
Thus, complete disappearance of glucose triggers the production of YAP1 in the cytoplasm, and YAP1 concentration increases if the lack of glucose continues. Notice that we are working with a delta-yap1 mutant strain as a recipient of our construct.
• The TRR promoter is fused to the GFP (Green Fluorescence Protein) coding sequence. The green fluorescent protein can be detected by fluorescent emission. The tiorredoxin reductase promoter is activated by two transcriptional factors, YAP1 and SKN7 in the oxidative form. Both of them can only bind to the promoter if H2O2 has been previously added to the culture medium.
MOLECULAR MECHANISMS
Click on each plasmid to learn how our constructions work!
Each construction was carried in different plasmids for their expression in yeast:
YEplac181 carried the Fermentative response construction. To see further information about the vector, click here .
YEp352 carried the Oxidative response construction. To see further information about the vector, click here .
INDUSTRIAL APPLICATIONS
The main industrial application of our talking yeast is to know the moment at which fermentation started. This is possible because we can calculate the time elapsed since the medium ran out of glucose according to the fluorescence intensity. Taking this into account, we could ask the yeast, through the addition of hydrogen peroxide: “How long have you been fermenting?” And the cells would answer telling us how long it has been since all the glucose was consumed. One possible answer could be “I have been fermenting for 1 hour”.
There is also a second possible application, but it needs a more detailed explanation:
There are several factors affecting yeast ethanol production during alcoholic fermentation. Saccharomyces cerevisiae seems to have adapted all along its evolution to optimize its growing rate in environments that are rich in easily assimilable nutrients, such as sugars and amino acids.
There are two important characteristics responsible of S. cerevisiae adaptation to this particular niche. One of them is its capability of metabolizing glucose and fructose through both the respiration and the fermentation pathways. The second one is its ability to grow in aerobic and anaerobic conditions. All of this makes this species exhibit some metabolic peculiarities, such as the Pasteur and the Crabtree effects. The Crabtree effect, described in S. cerevisiae and in a few other yeast species, must be taken into account when ethanol production is being studied or modified during fermentation. This effect causes that, even under high concentration of oxygen and with a relatively low amount of glucose, a considerable fraction of the consumed sugar is used to produce ethanol through the fermentative pathway. This is why in every industrial process which requires yeast growth (such as starter yeast production) it is necessary to provide the optimum amount of glucose that permits its direct consumption through the respiratory pathway (or the other way round if fermentation is desirable, such in bioethanol production).
We present here one way of monitoring the fate of glucose in the medium. By adding a small amount of hydrogen peroxide as a chemical input, we plan to ask our culture: “Is there any glucose left?”, and the culture would answer -according to the amount of glucose already present in the medium- what concentration would be necessary to optimize either yeast growth or fermentation through the Crabtree effect. In this way, the Crabtree effect could be monitored (perhaps even online) by measuring fluorescence and used to regulate the amount of glucose to be fed.
EXPERIMENTAL OUTLINE
- We ordered the DNA constructions: pADH2-YAP1 protein and pTRR-GFP protein that were cloned in plasmid pUC57 with a bacterial origin of replication.
- We obtain the Yeplac181 and Yep352 yeast vectors by our laboratory.
- We carried out four transformations of E. coli strain DH5, one for each plasmid DNA (the two constructions and the two vectors), in order to amplify them. See the Transformation Protocol Using Heat Shock .
- We obtained several E. coli transformants in four plates and took some colonies of each DNA construction (from both constructions and both vectors) and cultured them in liquid medium over night at 37ºC in shaking flasks.
- A day after, we extracted the plasmid DNA. See the Mini-preps protocol. See the Mini-prep Protocol .
- We obtained the purified constructions (both in pUC57 plasmid) and the vectors for yeast (YEplac181 and YEp352) also purified. See the Protocol for Gel Extraction
- We digested the four DNAs with restriction enzymes EcoRI and PstI in order to obtain compatible ends. See the Digestion Protocol .
- We ligated the pTRR-GFP construction with the Yep352 vector and the ADH2-YAP1 construction with the Yeplac181 vector. See the ligation Protocol .
- The day after that we transformed E. coli with the results of the ligation in order to amplify the final constructions (pTRR-GFP/Yep352 and pADH2-YAP1/YEplac181). We located the recombinant constructs using X-Gal and white/blue selection.
- We took some of these white colonies and cultured them in liquid medium overnight at 37ºC in shaking flasks.
- The day after we extracted the plasmid DNA. See the Mini-prep Protocol .
- After this, we checked the final purified recombinant constructs by electrophoresis, restriction digest and DNA capilar sequencing.
- We introduced the first of the DNA recombinant plasmids in the yeast. See the Yeast transformation protocol .
- We selected the transformants by growth in solid and liquid mineral medium attending to the auxotrophic markers and checked the presence of the construction by PCR. See the protocol here .
- We used the transformed yeast obtained at that moment and transformed it with the second recombinant construct. See the yeast transformation protocol. See the Yeast transformation protocol .
- Once again we identified the transformants by growth in selective medium and after that we used a PCR protocol to check the presence of both constructions.
- In order to detect fluorescence, we carried out a protocol to induce the expression of GFP. See the Yeast induction protocol .
Retrieved from " http://2012.igem.org/Team:Valencia_Biocampus/Yeast "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
