<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Notebook_Week5"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Notebook/Week5">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Notebook/Week5&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week5&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week5&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Notebook/Week5" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Notebook/Week5</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="left"><H4><SPAN class="mw-headline" id="Categories_:">Categories :</SPAN></H4><UL><A href="/Team:Cambridge" title="Team:Cambridge"> Home</A><A href="/Team:Cambridge/Team" title="Team:Cambridge/Team"> Team</A><A href="/Team:Cambridge/Sponsors" title="Team:Cambridge/Sponsors"> Sponsors</A><A href="/Team:Cambridge/Parts" title="Team:Cambridge/Parts"> Parts Submitted to Registry</A><A href="/Team:Cambridge/ImageGallery" title="Team:Cambridge/ImageGallery"> Image Gallery</A><A href="/Team:Cambridge/Contact" title="Team:Cambridge/Contact"> Leave a Message!</A></UL><H4><SPAN class="mw-headline" id="Project_:">Project :</SPAN></H4><UL><A href="/Team:Cambridge/Project" title="Team:Cambridge/Project"> Overview</A><A href="/Team:Cambridge/Project/Amplification" title="Team:Cambridge/Project/Amplification"> Sensitivity Tuner</A><A href="/Team:Cambridge/Project/Amplification/Characterisation" title="Team:Cambridge/Project/Amplification/Characterisation"> --- Characterisation</A><A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling"> --- Modelling</A><A href="/Team:Cambridge/Project/Pigments" title="Team:Cambridge/Project/Pigments"> Colour Generators</A><A href="/Team:Cambridge/Project/CA01" title="Team:Cambridge/Project/CA01"> --- Carotenoids (Orange/Red)</A><A href="/Team:Cambridge/Project/ME01" title="Team:Cambridge/Project/ME01"> --- Melanin (Brown)</A><A href="/Team:Cambridge/Project/VI01" title="Team:Cambridge/Project/VI01"> --- Violacein (Purple/Green)</A><A href="/Team:Cambridge/Future" title="Team:Cambridge/Future"> The Future</A><A href="/Team:Cambridge/Safety" title="Team:Cambridge/Safety"> Safety</A></UL><H4><SPAN class="mw-headline" id="Notebook_:">Notebook :</SPAN></H4><UL><A href="/Team:Cambridge/Notebook/Week1" title="Team:Cambridge/Notebook/Week1"> Week 1</A><A href="/Team:Cambridge/Notebook/Week2" title="Team:Cambridge/Notebook/Week2"> Week 2</A><A href="/Team:Cambridge/Notebook/Week3" title="Team:Cambridge/Notebook/Week3"> Week 3</A><A href="/Team:Cambridge/Notebook/Week4" title="Team:Cambridge/Notebook/Week4"> Week 4</A><STRONG class="selflink"> Week 5</STRONG><A href="/Team:Cambridge/Notebook/Week6" title="Team:Cambridge/Notebook/Week6"> Week 6</A><A href="/Team:Cambridge/Notebook/Week7" title="Team:Cambridge/Notebook/Week7"> Week 7</A><A href="/Team:Cambridge/Notebook/Week8" title="Team:Cambridge/Notebook/Week8"> Week 8</A><A href="/Team:Cambridge/Notebook/Week9" title="Team:Cambridge/Notebook/Week9"> Week 9</A><A href="/Team:Cambridge/Notebook/Week10" title="Team:Cambridge/Notebook/Week10"> Week 10</A></UL><H4><SPAN class="mw-headline" id="Team_Logistics_:">Team Logistics :</SPAN></H4><UL><A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"> Protocols</A><A href="/Team:Cambridge/Stock" title="Team:Cambridge/Stock"> Stock List</A><A href="/Team:Cambridge/Research" title="Team:Cambridge/Research"> Research</A><A href="/Team:Cambridge/Paste" title="Team:Cambridge/Paste"> Shared Links and Help</A></UL></DIV><DIV id="contentbox"><H1><SPAN class="mw-headline" id="Week_5"> Week 5 </SPAN></H1><DIV id="daytop"><UL><A href="#Monday"> Monday</A><A href="#Tuesday"> Tuesday</A><A href="#Wednesday"> Wednesday</A><A href="#Thursday"> Thursday</A><A href="#Friday"> Friday</A></UL></DIV><P><I>The plate-reader data for the promoter/amplifier constructs is analysed - DNA-purification debugged - modelling the amplifier/theshold system - melA primers recieved - vio gene planned using GeneDesigner</I></P><H2><SPAN class="mw-headline" id="Monday"> Monday </SPAN></H2><H3><SPAN class="mw-headline" id="Dry_Work">Dry Work</SPAN></H3><H4><SPAN class="mw-headline" id="Violacein_Synthesis">Violacein Synthesis</SPAN></H4><P>Used GeneDesigner from DNA2.0 to plan what we wanted synthesised, taking into account the different genes in the operon. This would be synthesised with codon optimisation for E. coli. All forbidden restriction sites have been removed (EcoRI, XbaI, SpeI, PstI, NotI)
</P><H4><SPAN class="mw-headline" id="Data_Analysis">Data Analysis</SPAN></H4><P>Produced a plot making program, producing subplots across cell type and repeats for all measurements.
</P><H3><SPAN class="mw-headline" id="Wet_Work">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Protocol_de-bugging">Protocol de-bugging</SPAN></H4><P>For the threshold device project, we need to amplify pSB3K3 as a low-copy number plasmid vector in order to transfer the activator constructs into low copy number plasmid.  We tried 4 different PCR reaction mixtures, and used a Zymogen PCR wash kit before running them on an 0.8% Agarose E-gel.  After looking at the gel, the ladder was quite smeared and no PCR product was visible.  It was found that the samples run on the gel had a very low DNA concentration.  However, some PCR solution was left before the wash kit was used, and using the Nano-Drop it was determined that it had a high concentration of DNA, so the PCR worked, we just need to refine the protocols after PCR.
</P><H4><SPAN class="mw-headline" id="Threshold_Device">Threshold Device</SPAN></H4><P>Mini-prepped I746374, I746375, I746380, and I746391; incubated overnight cultures of the rest of the 11 activator constructs in preparation for miniprep.
</P><H4><SPAN class="mw-headline" id="Pigments">Pigments</SPAN></H4><P><B>Melanin</B></P><P>Retransformed melanin plasmid into Top10, and transformed it into MG1655 to (hopefully) achieve better expression.  Plated on plates optimized for pigment production, and prepared for solid-phase pigment extraction
</P><P><B>Violacein</B></P><P>Retransformed violacein plasmid into Top10, and transformed it into MG1655 to (hopefully) achieve better expression.  
</P><P><B>Mixing pigments</B></P><P>Plated Top10 transformed with melanin plasmid and violacein plasmid and the Roche strain transformed with Duncan’s orange plasmid on the same plate, optimized for melanin production (as this is the only pigment that requires media supplementation) but without any antibiotic resistance.
</P><H4><SPAN class="mw-headline" id="Progress_on_Carotenoids_.28Monday.29"> Progress on Carotenoids (Monday) </SPAN></H4><P>Today we carried out restriction digest of K118006/CrtB (as recipient plasmid vector) using SpeI and PstI (two cuts both within suffix), and of K118005/CrtI (as gene insert) using XbaI and PstI (one cut within prefix, the other cut within suffix).
</P><P>Recall that the sizes are:
K118006/CrtB: insert: 949bp, backbone: 2079bp, total: 3028bp.
K118005/CrtI: insert: 1498bp, backbone: 2079bp, total: 3577bp.
</P><P>Hence we expect one band of about 3000bp for K118006/CrtB, and two bands of about 1500bp and 2100bp for K118005/CrtI, after restriction digest with their respective enzymes.
</P><P>The restriction digest products were run on gel, and the results were shown below:
</P><P><I>Lane 1: Log-2 DNA ladder (NEB).</I><I>Lanes 2 and 3: Restriction digest products of K118006/CrtB (less DNA loaded in lane 3).</I><I>Lane 4: Undigested K118006/CrtB (negative control).</I><I>Lane 5 empty.</I><I>Lane 6: Restriction digest products of K118005/CrtI.</I><I>Lane 7: Undigested K118005/CrtI (negative control).</I></P><P>This indicated that the restriction digest was successful.
</P><P>We then carried out gel extraction on Lane 2 and Lane 6, ligation with T4 ligase, and electroporation into E. coli Top10 cells. Results of transformation will be ready for inspection tomorrow.
</P><H2><SPAN class="mw-headline" id="Tuesday"> Tuesday </SPAN></H2><H3><SPAN class="mw-headline" id="Dry_Work_2">Dry Work</SPAN></H3><H4><SPAN class="mw-headline" id="Violacein_gene">Violacein gene</SPAN></H4><P>Confirmed that correct protein sequence was present for each gene, by using MUSCLE to compare the gene sequence given by GeneDesigner to the sequence in the NCBI database. All protein sequences were identical.
</P><P>Prepared the GeneDesigner file and the ApE file of the violacein operon to be sent for sequencing. Jim suggested that we refactor the DNA so that each individual ORF is preceeded by an RBS (the biobrick part B0034) and a constituitive promoter added. DNA2.0 kindly agreed to provide the promoter sequence for us, and add it in along with the operon. We also included the vio E gene after reading the Sanchez paper: <A href="http://www3.interscience.wiley.com/cgi-bin/fulltext/112732008/HTMLSTART?CRETRY=1&amp;SRETRY=0" class="external free" rel="nofollow">http://www3.interscience.wiley.com/cgi-bin/fulltext/112732008/HTMLSTART?CRETRY=1&amp;SRETRY=0</A></P><DL><DD><B>New Vio Plan</B></DD></DL><P>The protein structures given for each gene were also compared with vio proteins from other organisms, ( <I>Janthinobacterium lividum</I> and Duganella sp. B2) to confirm that the ORFs were coding for the correct protein.
</P><H4><SPAN class="mw-headline" id="Modelling">Modelling</SPAN></H4><P>Using our model for the rate of transcription from pBAD, dependent on arabinose concentration:
</P><P> , where </P><P>we discussed how this model would behave, and how to determine the parameters.
</P><P>The behaviour of this model to changing arabinose concentrations is shown in these graphs
</P><H3><SPAN class="mw-headline" id="Wet_Work_2">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Threshold_Device_2"> Threshold Device </SPAN></H4><P>Continued to refine protocols for successful PCR and purification of pSB3K3; tried 4 different PCR mixtures, all of which were successful.
</P><P>Miniprepped all remaining amplifier constructs.
</P><H4><SPAN class="mw-headline" id="Pigments_2">Pigments</SPAN></H4><P>Yesterdays transformations were successful, continued to incubate to trigger pigment production.
</P><H2><SPAN class="mw-headline" id="Wednesday"> Wednesday </SPAN></H2><H3><SPAN class="mw-headline" id="Dry_Work_3">Dry Work</SPAN></H3><H4><SPAN class="mw-headline" id="Pseudomonas_Green_and_Red">Pseudomonas Green and Red</SPAN></H4><P>Received confirmation that we will be receiving clones containing the appropriate plasmids (YAY) soon.
</P><H4><SPAN class="mw-headline" id="Violacein">Violacein</SPAN></H4><P>Removed restriction sites from the new vio plan (shown on yesterdays page) and send it to Jim, along with the ApE plasmid editor file of the operon. After getting the go-ahead from Jim, this plan will be send to DNA2.0 for synthesis.
Drafted the design criteria for Violacein synthesis:
</P><DL><DD><UL><LI>The gene consists of five open reading frames (vioA-E) each preceded by a ribosome binding site (part B0034). The whole sequence is held within the BioBrick registry prefix and suffix
</LI><LI>The DNA sequence of the prefix and suffix CAN NOT be altered in any way
</LI><LI>Ideally the operon would be under the control of the repressible promoter suggested by Jeremy. This promoter would be upstream of the operon and ribosome binding sites, but downstream of the prefix
</LI><LI>With the exception of the prefix and suffix (which cannot be altered) the following restriction sites are forbidden within the operon: EcoRI, XbaI, SpeI, PstI, NotI
</LI><LI>If possible, we would like the genes to be optimised for E. coli and preferably B. subtilis as well
</LI><LI>In order to easy gene separation, we would like restriction sites on either side of the vio C and the vio D genes.
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="Modelling_2">Modelling</SPAN></H4><P>We have now developed several versions of our model for the amplifier. These involve a simple model, positive feedback and degradation of the activator protein. We also included dependence on a parameter for input transcription, describing changes in both arabinose concentration and binding properties.
</P><P>We have some interesting results for the positive feedback latch. Our <I>continuous</I> model indicates that whatever the transcriptional input, the reporter is produced in the same way. This means that the latch would be switched at any input level. We plan to look at stochastic modelling to see if the same applies here.
</P><P>We produced several graphs to show what happens when different parameters are changed for the latch. The most important is the following, showing the very small effect of changing transcriptional input - encapsulating the effect of different arabinose concentrations.
</P><H4><SPAN class="mw-headline" id="Data_Analysis_2">Data Analysis</SPAN></H4><P>We now have fluorescence rate calculated for all runs, and have normalised this for od. The next step is to take the steady state fluorescence rate per cell, and plot this against arabinose concentration. This should allow us to show the effect of the amplifier construct.
</P><H3><SPAN class="mw-headline" id="Wet_Work_3">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Pigment_Stocks">Pigment Stocks</SPAN></H4><P>Retransformed melanin and violacein pigment into MG1655 (previous attempt was unsuccessful)
</P><H4><SPAN class="mw-headline" id="Melanin_Biobrick">Melanin Biobrick</SPAN></H4><P>Did a PCR to amplify the melanin fragment that will be made into a BioBrick. Purpose was to measure the length of the MelA fragment, check that the A and F primers were functional (as they are used in every PCR cycle in the plan), and produce samples for testing our different DNA clean-up protocols. We used the Finnzymes MasterMix with added DMSO and ran on an ethidium bromide gel.
</P><P>Gel was run at 80V for 45mins. The two samples were made of the same master mix (as in Finnzymes PCR protocol) with added 1.5ul of DMSO to each 50ul sample. The bands are approximately 2kb, which matches the 1.9kb size of the melA fragment. The smaller bands near the bottom of the gel are the primers.
</P><H4><SPAN class="mw-headline" id="Threshold_Device_3">Threshold Device</SPAN></H4><P>Made glycerol stocks for -20 freezer of all activator constructs.
</P><P>Continued to de-bug DNA clean-up after PCR, PCR amplified more pSB3K3.
</P><H2><SPAN class="mw-headline" id="Thursday">Thursday</SPAN></H2><H3><SPAN class="mw-headline" id="Dry_Work_4">Dry Work</SPAN></H3><H4><SPAN class="mw-headline" id="Data_Analysis_3">Data Analysis</SPAN></H4><UL><LI>For each data set, the maximum normalised fluorescence rate was taken. This is the measurement used to calculate Relative Promoter Units, and by extention, RNA Polymerase per second (PoPS). These data were plotted for each cell type against concentration of arabinose:
</LI></UL><P>The most important graphs above are those for GFP in cells 40 (pBAD) and 91 (amplifier). These are effectively the input and output for the amplifier device. One can see that the pBAD curve has no real increase until at least 10μM, whereas an equivalent large increase for the amplifier starts at about 1μM. Very low level transcription can be seen to be amplified. The amplified graph (91) starts to drop off at high concentrations: this may be due to toxicity.
</P><UL><LI>Next, in order to remove the effect of pBAD, the gain (ratio output/input) at each concentration was calculated:
</LI></UL><P>This graph shows that the amplifier has steady gain at low concentrations. The point at &quot;0.01&quot; is actually that for zero, showing the basal amplification. What is interesting in this graph is that the gain drops off, well before the reduction of expression for cells 91. This is because cells 40 start to rise here.
</P><UL><LI>Finally, in order to fully abstract the behaviour of the amplifier device from arabinose concentration, the output signal was plotted directly against the input signal:
</LI></UL><P>A linear device would have a straight line here. Instead we observe a rapid increase in output at a low range of input, followed by a plateau, then a decrease at higher input. There is scope for the device to be used in both discrete and continuous applications.
</P><H3><SPAN class="mw-headline" id="Minutes_from_Lab_Meeting"> Minutes from Lab Meeting </SPAN></H3><UL><LI>Melanin and Violacein
</LI></UL><DL><DD><UL><LI>Characterisation of the pigments - consider LCMS
</LI><LI>Plate all colours as streaks on the same plate, for presentation purposes
</LI><LI>Check the Melanin rbs is compatible with B. subtilis
</LI><LI>We could probably have a  think about why our pigments don't grow in liquid culture.
</LI></UL></DD></DL><UL><LI>Carotenoids
</LI></UL><DL><DD><UL><LI>Check the gene homology for the different genes
</LI><LI>Contact Edinburgh to ask about the promoter
</LI><LI>Double-check the orange colour is actually being formed by the genes we think they are
</LI><LI>The Roche strain can only be used for testing, not for the actual iGEM product
</LI></UL></DD></DL><UL><LI>DNA purification
</LI></UL><DL><DD><UL><LI>Overall, we are still not sure why the column washes are not working
</LI><LI>Suggested that we try purification with 0.5M Ammonium acetate
</LI></UL></DD></DL><UL><LI>Melanin retention
</LI></UL><DL><DD><UL><LI>More controls needed to check whether the melanin diffusion out of the cell is the cause of the staining.
</LI></UL></DD></DL><UL><LI>Modelling
</LI></UL><DL><DD><UL><LI>Presented the continuous modelled behaviour of the latch: to a large extent independent of input transcription. This means it is likely to turn on under basal input.
</LI><LI>Stochastic modelling may agree or disagree with this finding.
</LI></UL></DD></DL><UL><LI>Overall Plan
</LI></UL><DL><DD><UL><LI>We need a lab meeting to set out the following:
</LI></UL></DD><DD><UL><LI>What we want
</LI><LI>What is feasible
</LI><LI>What is justifiable within the overall project plan
</LI></UL></DD></DL><UL><LI>Work towards the end-result, and the talk/presentation at MIT
</LI></UL><DL><DD><UL><LI>Results with theoretical underpinning
</LI><LI>What we have managed and what our model days we could achieve
</LI><LI>Finished PRODUCT
</LI></UL></DD></DL><UL><LI>Ideas for finished product - dipstick, 'lollypop' etc.
</LI></UL><H4><SPAN class="mw-headline" id="Modelling_3">Modelling</SPAN></H4><UL><LI>Basic promoter-activator-phage promoter system modelled. A variety of parameters changed and the effects collated.
</LI><LI>Addition of positive feedback for first 'latch' design - Note that the same output is gained for any input, so it would destroy any natural switching level on-off property of the activators.
</LI><LI>Began to develop 'threshold' switching level system as detailed at bottom of modelling page. Considered different mechanisms by which it could work:
</LI></UL><DL><DD><UL><LI>activator and repressor bind to different sites on operator, both can be bound at once, but transcription only occurs when the activator alone is bound.
</LI></UL></DD></DL><DL><DD><UL><LI>activator and repressor compete for binding to DNA; they do not necessarily bind the same site, but if one is present the other cannot bind.
</LI></UL></DD></DL><DL><DD><UL><LI>the activator prevents production of the repressor by inhibiting transcription, but it is only capable of doing this once it has reached a particular concentration. This is effectively just the activator system with a second step, as the switching level depends on the properties of the promoter, not the quantity of repressor present. It would still be interesting to see what happens when this extra level is included and whether this helps remove basal transcription level problems.
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Wet_Work_4">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Amplification">Amplification</SPAN></H4><P>Overnight culture of a single colony of J69591, phenotypically confirmed, in preparation to make glycerol stocks, steak out on a plate, and miniprep.  This will be the parent colony for all of our characterizations.
</P><H4><SPAN class="mw-headline" id="Melanin_Retention">Melanin Retention</SPAN></H4><P>Streaks of the strains requested from Philip Oliver containing multi-drug transporter knockouts arrived - MC1060 containg a deletion of the acrAB operon, and MG1655 containing a deletion of the acrF gene (part of the acrEF operon). Competent cell stocks were prepared in readiness for the next day's transformations with the MelA-containing plasmid.
</P><H2><SPAN class="mw-headline" id="Friday">Friday</SPAN></H2><H3><SPAN class="mw-headline" id="Wet_Work_5">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Melanin_Pigment">Melanin Pigment</SPAN></H4><P>Made controls (as  suggested in the meeting yesterday) to confirm that the melanin pigment was being produced by the bacteria with the MelA gene, rather than by oxidation of plate substrates. Schematic is shown below - all plates contain tyrosine, copper sulfate, and IPTG. Brown colour represents melanin bacteria, while pale stain represents the TOP10 control species. All plates were stored over the weekend at 37 degrees.
</P><H4><SPAN class="mw-headline" id="Melanin_Retention_2">Melanin Retention</SPAN></H4><P>Electroporation was used to transform the MC1060 (containing an acrAB deletion) and MG1655 (containing an acrF deletion) strains with the MelA-containing plasmid before plating them out onto ampicillin plates. Hopefully these should show limited melanin diffusion into the agar medium and delineated black colonies after a spell in the incubator.
</P><H4><SPAN class="mw-headline" id="DNA_Wash">DNA Wash</SPAN></H4><P>As part of troubleshooting the column wash procedure, the QUIGEN kit was used to purify the DNA after a PCR. The gel below shoes, from left to right, 2-log ladder, unwashed PCR-product, 1/10 diluted PCR product, and washed DNA. The QUIGEN kit therefore produces a reasonable yield of pure DNA, and will be used in the melanin protocol which will be startedon tuesday.
</P><P>(GEL PICTURE)
</P><H3><SPAN class="mw-headline" id="Dry_Work_5">Dry Work</SPAN></H3><H4><SPAN class="mw-headline" id="Modelling_4">Modelling</SPAN></H4><P>The threshold switching system was modelled, giving model 6.1 (transcription only when activator bound) and model 6.2 (competition between activator and repressor).  A plot of output against repressor present is given for a set input which shows the expected switching off of output at a threshold (this is the reverse of an activator switch on, effectively).
</P><H4><SPAN class="mw-headline" id="Violacein_Synthesis_2">Violacein Synthesis</SPAN></H4><P>Sent the order to DNA2.0! The GeneDesigner file was send, along with the ApE files for the vio operon and the vector we hope to splice the operon into (pPB3K3). The final design is shown below:
</P><P>Design specifications were also finalised:
</P><DL><DD><UL><LI>The gene consists of five open reading frames (vioA-E) each preceded by a ribosome binding site. The whole sequence is held within the BioBrick registry prefix and suffix
</LI><LI>The DNA sequence of the prefix and suffix cannot be altered in any way
</LI><LI>As overexpression of the violacein operon can be toxic for the bacteria, we would like the operon under control of an inducible/repressible promoter, such as the T5 promoter suggested by Jeremy.
</LI><LI>With the exception of the prefix and suffix (which cannot be altered) the following restriction sites are forbidden within the operon: EcoRI, XbaI, SpeI, PstI, NotI
</LI><LI>If possible, we would like the genes to be optimised for E. coli and preferably B. subtilis as well
</LI><LI>In order to easy gene separation, we would like restriction sites on either side of the vio C and the vio D genes.  Ideally ones with compatible overhanging ends such as BamHI, BclI and BglII
</LI></UL></DD></DL><P>Vector: pJexpress 401-406 (T5) Cloning Cassette
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Cambridge/Notebook/Week5">http://2009.igem.org/Team:Cambridge/Notebook/Week5</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Notebook/Week5" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Notebook/Week5" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week5&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week5&amp;oldid=59337" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>