"
Page
Discussion
View source
History
teams
Log in
 
Team:Cambridge/Project/Carotenoids
From 2009.igem.org
Categories :
Home Team Sponsors Parts Submitted to Registry Image Gallery Leave a Message!
Project :
Overview Sensitivity Tuner --- Characterisation --- Modelling Colour Generators --- Carotenoids (Orange/Red) --- Melanin (Brown) --- Violacein (Purple/Green) The Future Safety
Notebook :
Week 1 Week 2 Week 3 Week 4 Week 5 Week 6 Week 7 Week 8 Week 9 Week 10
Team Logistics :
Protocols Stock List Research Shared Links and Help
Carotenoids
Introduction Previous Research Our Action Plan Result Analysis Reference
Introduction
Carotenoids are organic pigments naturally present in plants, algae and some bacteria. There are more than 600 carotenoids, which can be categorised into xanthophylls (hydrocarbons containing oxygen element) and carotenes (hydrocarbons containing no oxygen). Carotenoids perform a range of functions, including light energy absorption, protection against photo-damage, acting as antioxidants, and as precursor to other organic compounds. In human, for example, beta-carotene is the precursor to vitamin A.
The common starting point for carotenoid synthesis is farnesyl pyrophosphate (FPP), which derives from two precursors, isopentenyl pyrophosphate (IPP) and dimethylallyl pyrophosphate (DMAPP). In general, there are two pathways for synthesising IPP and DMAPP: the Mevalonate Pathway (starting with acetyl CoA) and the Non-mevalonate Pathway (starting with pyruvate and glyceradehyde-3-phosphate). While the Mevalonate Pathway is present in all higher eukaryotes, the Non-mevalonate Pathway is present in E. coli.
Part of the carotenoid biosynthesis pathway is shown below (figure adopted from iGEM 2007 Team Edinburgh):
Some of the intermediates in the above pathways are coloured, e.g. lycopene (red), beta-carotene (orange), and zeaxanthin (yellow). The enzymes involved in the biosynthetic pathway are CrtE, CrtB, CrtI, CrtY, and CrtZ. Since our team aims to use different colour pigments as signal output, the above pathway can be incorporated in our design of colour production.
Previous Research
A few iGEM teams of previous years (e.g. Edinburgh 07, Edinburgh 08, Guelph 08) conducted research in the carotenoid system. We are fortunate to have access to some of their Biobricks in the registry. Here is a summary of the Biobricks available:
Registry Code
Team
Sequence Description
Size
K118014
Edinburgh 08
rbs + CrtE
insert: 922bp, backbone: 2079bp, total: 3001bp.
K118006
Edinburgh 08
rbs + CrtB
insert: 949bp, backbone: 2079bp, total: 3028bp.
K118005
Edinburgh 08
rbs + CrtI
insert: 1498bp, backbone: 2079bp, total: 3577bp.
2 PstI sites removed.
K118013
Edinburgh 08
rbs + CrtY
insert: 1162bp, backbone: 2079bp, total: 3241bp.
K152005
Guelph 08
(rbs+CrtE) + (rbs+CrtB) + (rbs+CrtI)
+ (rbs+CrtY) + (rbs+GFP)
insert: 5412bp, backbone: 2079bp, total: 7491bp.
2 PstI sites in CrtI removed.
I742158
Edinburgh 07
rbs + CrtZ
insert: 547bp, backbone: 2079bp, total: 2626bp.
The above enzymes all come from Pantoea ananatis (GenBank: D90087.2).
However, we have encountered certain issues during our preliminary research.
In part K152005, the “CrtE” sequence (which originated from part K152001) contains 3 nucleotides different from the NCBI data base.
Originally, the registry annotation for K152005 indicated that there was no ribosome binding site before gene CrtI (i.e. “…rbs+CrtB+CrtI+rbs+CrtY…”). After clarification with Team Guelph 08, we were told that it was solely an error in annotation. The registry entry for K152005 has been updated to the sequence in the table above.
Teams of previous years constructed various composite parts containing two or more enzymes of the synthetic pathways (e.g. CrtEBI, CrtEBIY), but they were not available in the 2009 Distribution Plates.
Our Action Plan
In our project this year, we aim to incorporate the carotenoid biosynthesis pathway in E. coli and produce different coloured pigments. Potential candidates are lycopene (red), beta-carotene (orange), and zeaxanthin (yellow). Since these three compounds are sequential in the biosynthetic pathway, we hope to demonstrate colour changes from red to orange to yellow, by controlling expression of the enzymes involved in conversion.
Our action plan is as follows:
1. Test the activity of K152005 (CrtEBIY) .
2. Test pigment production in E. coli strain MG1655 and its derivative (up-regulation in precursors) .
3. Construct composite biobrick containing CrtE, CrtB and CrtI (i.e. CrtEBI) and test its activity .
4. Transform E. coli with CrtEBI and CrtY. Test if activation of CrtY can cause colour changes in vivo .
5. Characterise our composite biobricks and the colour output .
6. Test compatibility with biobricks from previous iGEM teams and see if we can convert their outputs into pigment production .
Our Action Plan (Updated regularly). Green: completed. Yellow: Work in progress. Blue: Partially completed. Red: Planning.
Test activity of K152005
Close-up of K152005 in TOP10
We wish to test the feasibility of incorporating the carotenoid biosynthetic pathway into E. coli, which does not produce carotenoids naturally. K152005 contains the enzymes for beta-carotene formation, and is a quick way to verify our concept.
We transformed E. coli (TOP10) using K152005 from the 2009 Distribution Plate ( Week 2 ). After incubation for about 3 days, some light yellowish pigmention was visible. The long incubation time may be due to the fact that K152005 does not contain promoter. Although only small amount of yellow pigment was observed*** (instead of the expected orange colour), this gave us confidence that the carotenoid biosynthetic pathway was working in E. coli. ***Later experiments showed that the yellow pigment was probably not beta-carotene. Please refer to " Characterisation of Biobricks and colour output " for more details.
Mapping of Biobrick K152005 (update):
We will then put K152005 under a strong promoter (R0011, constitutive) to see if stronger colour production can be achieved.
Test pigment production in E. coli MG1655 and its derivative
Pathway from pyruvate and G3P to FPP. Figure from Yuan et al. 2006.
The first compound in the carotenoid biosynthetic pathway, farnesyl pyrophosphate (FPP), derives ultimately from pyruvate and glyceraldehyde 3-phosphate (see figure on the right). The rate of carotenoid synthesis is limited by the rate of supply of FPP.
Certain strains of E. coli, e.g. MG1655, have been genetically engineered to up-regulate the enzymes involved in the pathway leading to FPP, thus increasing FPP supply and enhancing carotenoid synthesis. In Yuan et al., plasmid pPCB15 was used as reporter to assess the effect of such up-regulation. pPCB15 contains genes CrtEXYIB derived from Pantoea stewartii (homologous to the Biobricks available, which are derived from Pantoea ananatis) and produces orange beta-carotene as the main coloured product. The results showed that upregulation of enzymes upstream of FPP led more beta-carotene production.
Map of plasmid pPCB15.
In our project, we wish to compare carotenoid pigment production in E. coli strains TOP10, MG1655 and its derivative. Due to greater supply of FPP, we will expect higher level of carotenoid production in MG1655 and its derivative than in TOP10. Working on MG1655 and its derivative makes it easier to assess the effects of Biobricks on carotenoid synthesis, and allows us to develop and refine methods for pigment measurement and characterisation, which can then be applied on TOP10.
We have access to the original MG1655 ("Parent") and its derived strain, MG1655 PT5-dxs PT5-idi PT5-ispB PTe-ispDF ("Roche"). We first transformed these two strains with pPCB15 for orange beta-carotene production ( Week 3 ). pPCB15 was also transformed into TOP10 to see if it would work in a strain with normal FPP flux. pPCB15 has chloramphenicol-resistance selection marker so the transformed bacteria were grow on media with chloramphenicol.
On agar plates, pPCB15 produced yellow pigments in all three strains, with increasing intensity in the order: TOP10 < original MG1655 < MG1655 PT5-dxs PT5-idi PT5-ispB PTe-ispDF (photo below). The results indicated that it is feasible to introduce carotene synthesising pathway in E.coli, and that even in strains with normal FPP flux (i.e. TOP10), the yellow pigment production is still relatively fast (overnight incubation) and easily visualised. However, as mentioned above, since the effects in MG1655 and its derivative are much stronger and easier to assess, we used them as a tool to prove our concepts and test our Biobricks.
Top row from the left: pPCB15 in MG1655-PT5 ("Roche"), pPCB15 in TOP 10, pPCB15 in MG1655 ("Parent"). Bottom row from the left: K152005 in TOP10, R0011 in TOP10 (negative control).
Construct composite Biobrick CrtEBI
We wish to produce the first coloured compound in the pathway, lycopene (red), which can then be converted into other colours by subsequent enzymatic reaction.
In order to “pause” at lycopene, we need enzymes CrtE, CrtB and CrtI only. Currently in the registry there is no available Biobrick containing CrtEBI, so we decide to construct our own.
After transforming E. coli with K118014 (rbs+CrtE), K118006 (rbs+CrtB) and K118005 (rbs+CrtI), we construct composite biobrick by ligating the three genes together.
Plan mapping of CrtEBI:
Test colour changes by CrtY
Characterisation of Biobricks and colour output
Carotene: We wish to not only visually identify yellowish/orange pigment formation, but also quantify the amount of beta-carotene produced. To this end, we performed carotene extraction with acetone using protocol described in literature and took absorbance measurement using Omega Microplate Readers (BMG-Labtech).
As a pilot project to test out the carotene measurement protocol, we prepared overnight cultures for pPCB15 in original MG1655, pPCB15 in MG1655 PT5-dxs,idi,ispBDF, pPCB15 in TOP10, K152005 in TOP10 and R0011 in TOP10 (as negative control). Absorbance spectrum measurements were taken, first on intact cells, and then on carotene extract in acetone. The results indicated that absorbance measurements on intact cells were unable to demonstrate the characteristic absorbance peak of beta-carotene at 450 nm. In contrast, absorbance measurement on carotene extract in acetone showed clear peak at 450nm, which was a much better indicator of beta-carotene production. Furthermore, the visually identified "yellow pigment" in K152005 was probably not beta-carotene. This could not have been distinguished using visual inspection along. Given these results, we decided to use carotene extraction in acetone and absorbance at 450 nm for future quantification of amount of carotene.
The colours of the acetone extracts were shown below:
Results of the Microplate Readers (left: absorbance spectrum of intact cells; right: absorbance spectrum of acetone extracts).See Week 3 Notebook for detailed analysis of the results.
Test compatibility with other biobricks
Result Analysis
Reference
Nishizaki T, Tsuge K, Itaya M, Doi N, Yanagawa H. Metabolic engineering of carotenoid biosynthesis in Escherichia coli by ordered gene assembly in Bacillus subtilis. Appl Environ Microbiol. 2007 Feb;73(4):1355-61. PMCID: PMC1828653.
Luke Z. Yuan, Pierre E. Rouvière, Robert A. LaRossa and Wonchul Suh. Chromosomal promoter replacement of the isoprenoid pathway for enhancing carotenoid production in E. coli. Metabolic Engineering 8 (2006) 79-90.
Luan Tao, Raymond E. Jackson, Pierre E. Rouvière, Qiong  Cheng. Isolation of chromosomal mutations that affect carotenoid production in Escherichia coli: mutations alter copy number of ColE1-type plasmids. FEMS Microbiology Letters 243 (2005) 227-233.
von Lintig J, Vogt K. Filling the gap in vitamin A research. Molecular identification of an enzyme cleaving beta-carotene to retinal. J Biol Chem. 2000 Apr 21;275(16):11915-20. PMID: 10766819.
Retrieved from " http://2009.igem.org/Team:Cambridge/Project/Carotenoids "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
