<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Cambridge_Notebook_Week3"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:Cambridge/Notebook/Week3">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Cambridge/Notebook/Week3&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week3&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week3&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Cambridge/Notebook/Week3" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Cambridge/Notebook/Week3</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="left"><H4><SPAN class="mw-headline" id="Categories_:">Categories :</SPAN></H4><UL><A href="/Team:Cambridge" title="Team:Cambridge"> Home</A><A href="/Team:Cambridge/Team" title="Team:Cambridge/Team"> Team</A><A href="/Team:Cambridge/Sponsors" title="Team:Cambridge/Sponsors"> Sponsors</A><A href="/Team:Cambridge/Parts" title="Team:Cambridge/Parts"> Parts Submitted to Registry</A><A href="/Team:Cambridge/ImageGallery" title="Team:Cambridge/ImageGallery"> Image Gallery</A><A href="/Team:Cambridge/Contact" title="Team:Cambridge/Contact"> Leave a Message!</A></UL><H4><SPAN class="mw-headline" id="Project_:">Project :</SPAN></H4><UL><A href="/Team:Cambridge/Project" title="Team:Cambridge/Project"> Overview</A><A href="/Team:Cambridge/Project/Amplification" title="Team:Cambridge/Project/Amplification"> Sensitivity Tuner</A><A href="/Team:Cambridge/Project/Amplification/Characterisation" title="Team:Cambridge/Project/Amplification/Characterisation"> --- Characterisation</A><A href="/Team:Cambridge/Modelling" title="Team:Cambridge/Modelling"> --- Modelling</A><A href="/Team:Cambridge/Project/Pigments" title="Team:Cambridge/Project/Pigments"> Colour Generators</A><A href="/Team:Cambridge/Project/CA01" title="Team:Cambridge/Project/CA01"> --- Carotenoids (Orange/Red)</A><A href="/Team:Cambridge/Project/ME01" title="Team:Cambridge/Project/ME01"> --- Melanin (Brown)</A><A href="/Team:Cambridge/Project/VI01" title="Team:Cambridge/Project/VI01"> --- Violacein (Purple/Green)</A><A href="/Team:Cambridge/Future" title="Team:Cambridge/Future"> The Future</A><A href="/Team:Cambridge/Safety" title="Team:Cambridge/Safety"> Safety</A></UL><H4><SPAN class="mw-headline" id="Notebook_:">Notebook :</SPAN></H4><UL><A href="/Team:Cambridge/Notebook/Week1" title="Team:Cambridge/Notebook/Week1"> Week 1</A><A href="/Team:Cambridge/Notebook/Week2" title="Team:Cambridge/Notebook/Week2"> Week 2</A><STRONG class="selflink"> Week 3</STRONG><A href="/Team:Cambridge/Notebook/Week4" title="Team:Cambridge/Notebook/Week4"> Week 4</A><A href="/Team:Cambridge/Notebook/Week5" title="Team:Cambridge/Notebook/Week5"> Week 5</A><A href="/Team:Cambridge/Notebook/Week6" title="Team:Cambridge/Notebook/Week6"> Week 6</A><A href="/Team:Cambridge/Notebook/Week7" title="Team:Cambridge/Notebook/Week7"> Week 7</A><A href="/Team:Cambridge/Notebook/Week8" title="Team:Cambridge/Notebook/Week8"> Week 8</A><A href="/Team:Cambridge/Notebook/Week9" title="Team:Cambridge/Notebook/Week9"> Week 9</A><A href="/Team:Cambridge/Notebook/Week10" title="Team:Cambridge/Notebook/Week10"> Week 10</A></UL><H4><SPAN class="mw-headline" id="Team_Logistics_:">Team Logistics :</SPAN></H4><UL><A href="/Team:Cambridge/Protocols" title="Team:Cambridge/Protocols"> Protocols</A><A href="/Team:Cambridge/Stock" title="Team:Cambridge/Stock"> Stock List</A><A href="/Team:Cambridge/Research" title="Team:Cambridge/Research"> Research</A><A href="/Team:Cambridge/Paste" title="Team:Cambridge/Paste"> Shared Links and Help</A></UL></DIV><DIV id="contentbox"><H1><SPAN class="mw-headline" id="Week_3_-_Development"> Week 3 - Development </SPAN></H1><DIV id="daytop"><UL><A href="#Monday"> Monday</A><A href="#Tuesday"> Tuesday</A><A href="#Wednesday"> Wednesday</A><A href="#Thursday"> Thursday</A><A href="#Friday"> Friday</A></UL></DIV><P><I>Designed primers for making the melanin biobrick - produced brown, purple and orange pigment colours on solid media - characterised orange pigment - researched degredation tags (later turned out not to be needed) - researched the amplification/threshold processing system</I></P><H2><SPAN class="mw-headline" id="Monday">Monday</SPAN></H2><H3><SPAN class="mw-headline" id="Primer_Design">Primer Design</SPAN></H3><P>Primers were designed to convert MelA into a biobrick. Primers A and F are for either end (including prefix and suffix - highlighted in green). The other primers surround the unwanted restiction sites (highlighted in yellow) with the changed base in red.
</P><P>These correspond to the parts on the gene shown below. The stars correspond to the two unwanted restriction sites.
</P><H3><SPAN class="mw-headline" id="Orange_Pigment">Orange Pigment</SPAN></H3><P>Used plasmid editor to examine the genes used for the production of orange pigments. The (supposedly) orange-producing pigment from the biobricks:
</P><P>We also tried a preliminary design for a biobrick we could construct, in case the orange biobrick does not function. This uses the same genetic synthetic pathway for gene consruction:
</P><H3><SPAN class="mw-headline" id="Transformed_Pigments">Transformed Pigments</SPAN></H3><P>The MelA plate left on the bench overnight produced a brown-coloured pigment! Growth on media containing IPTG should produce this pigment a lot faster.
</P><H3><SPAN class="mw-headline" id="Amplification_system">Amplification system</SPAN></H3><UL><LI>Possible plan
</LI></UL><DL><DD><UL><LI>We would want to tag the activator proteins then quantify their degradation in some way
</LI></UL></DD><DD><UL><LI>GFP fusion: one idea was to create a functional activator-GFP fusion, to which we would then add a C-terminal degradation tag.  Thus we could express the activator-GFP-tag in E. coli, and could measure flourescence to quantify activator degradation directly.  However, this idea depends on us constructing a functional activator-GFP fusion.
</LI><LI>Western blotting: a second idea is to add an N-terminal flag tag to the activator proteins, and then add the C-terminal degradation tag.  We could measure the amount of activator using Western blotting and an anti-flag antibody by taking samples over time.  However, Western blotting isn't that easy. 
</LI><LI>Finally, what we're really doing is solving a problem found in the Cambridge 2007 team's system.  If we can show that overnight bacterial growth is less hindered with our tagged activators, then we are effectively showing that because of the degradation tags, the activators are indeed  being degraded. 
</LI></UL></DD></DL><UL><LI>We could also attempt to see what this effect has on amplification
</LI></UL><UL><LI>Modelling potential - activator half life, change in Pops out?
</LI><LI>First we need to dig up the 2007 work and recreate their data, this time with a plate reader that can take OD and fluorescence readings simultaneously.
</LI></UL><H4><SPAN class="mw-headline" id="Activators">Activators</SPAN></H4><UL><LI>Found parts submitted by Cambridge '07 team amplifier project. Three translational units (ribosome binding sites and protein coding sequence) for the three activators. Sequence below, 5' to 3':
</LI></UL><DL><DD><UL><LI><B>Ogr activator from P2 phage: Part I746350</B></LI></UL></DD><DD><A href="http://partsregistry.org/Part:BBa_I746350" class="external text" rel="nofollow">BBa_I746350 Part-only sequence (237 bp)</A></DD></DL><DL><DD><UL><LI><B>pag activator from PSP3 phage: Part I746351</B></LI></UL></DD><DD><A href="http://partsregistry.org/Part:BBa_I746351" class="external text" rel="nofollow">BBa_I746351 Part-only sequence (237 bp)</A></DD></DL><DL><DD><UL><LI><B>delta activator from phiR73 phage: Part I746352</B></LI></UL></DD><DD><A href="http://partsregistry.org/Part:BBa_I746352" class="external text" rel="nofollow">BBa_I746352 Part-only sequence (264 bp)</A></DD></DL><H4><SPAN class="mw-headline" id="Promoters">Promoters</SPAN></H4><P>Available as individual parts, with a GFP reporter or as part of a composite with the activator upstream. 
Five promoters as used by Cambridge '07:
</P><P>PF promoter, part: I746360
PO promoter, part: I746361
PP promoter, part: I746362
Psid promoter, part: I746364
PLL promoter, part: I746365
</P><H4><SPAN class="mw-headline" id="Degradation_Tags">Degradation Tags</SPAN></H4><UL><LI>Three C-terminal tags are well described in Andersen, J. B., Sternberg, C., Poulsen, L. K., Bjorn, S. P., Givskov, M. &amp; Molin, S. (1998). New unstable variants of green fluorescent protein for studies of transient gene expression in bacteria. <A href="http://aem.asm.org/cgi/reprint/64/6/2240" class="external text" rel="nofollow">Appl Environ Microbiol 64, 2240–2246.</A>, which detailed tagged-GFP degredation. 
</LI></UL><UL><LI>The tags as described in the paper that we could use are as follows (all DNA sequences are 5' to 3')
</LI></UL><DL><DD><UL><LI><B>LAA</B>: Amino acid sequence AANDENYALAA, DNA sequence GCAGCAAACGACGAAAACTACGCTTTAGCAGCT
</LI></UL></DD></DL><P>(primer – CGTCGTTTGCTGCTTTTGATGCGAAATCGTCGA) (currently the &quot;planning&quot; biobrick BBa_M0042)
</P><DL><DD><UL><LI><B>AAV</B>: Amino acid sequence AANDENYAAAV, DNA sequence GCAGCAAACGACGAAAACTACGCTGCAGCAGTT
</LI></UL></DD></DL><P>(primer – CGTCGTTTGCTGCTTTTGATGCGACGTCGTCAA, currently the &quot;planning&quot; biobrick BBa_M0044 with DNA sequence GCAGCAAACGACGAAAACTACGCTGCTGCTGTT)
</P><DL><DD><UL><LI>NOTE: We need to use the biobrick DNA sequence as the DNA sequence described in the paper has a forbidden PstI site
</LI></UL></DD></DL><UL><LI><B>ASV</B>: Amino acid sequence AANDENYAASV, DNA sequence GCAGCAAACGACGAAAACTACGCTGCATCAGTT
</LI></UL><P>(primer – CGTCGTTTGCTGCTTTTGATGCGACGTAGTCAA, currently the &quot;planning&quot; biobrick BBa_M0046 with DNA sequence GCTGCAAACGACGAAAACTACGCTGCATCAGTT)
</P><H3><SPAN class="mw-headline" id="Wet_Work">Wet Work</SPAN></H3><UL><LI>Made up 4 15ug/mL tetracycline agar plates - argh should have been trimethoprim!
</LI><LI>Electroporated the following plasmids into our TOP10 competent E. coli cells
</LI></UL><DL><DD><UL><LI>K118005
</LI><LI>K118006
</LI><LI>K118013
</LI><LI>K119014
</LI><LI>Orange plasmid from Dupont strain
</LI></UL></DD></DL><UL><LI>Plated each biobrick out onto ampicillin plates, plated the orange plasmid onto a tetracycline plate, left to incubate overnight
</LI><LI>Innoculated 6 single colonies from each of the following cultures, grown over the weekend on plates, in the appropriate antibiotic LB to shake over night in preparation for minipreps and colony PCRs tomorrow
</LI></UL><DL><DD><UL><LI>Violacein plasmid
</LI><LI>Melanin plasmid
</LI><LI>K152005
</LI><LI>R0011
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Amplifiers">Amplifiers</SPAN></H3><P>Basal amplification could potentially be reduced by using a suppressor.
The &quot;amplification factors&quot; from Cambridge 2007 appear to be long term, ie steady state. It would be interesting to look at the transient response, as the degradation rate will certainly have an impact on this.
</P><H2><SPAN class="mw-headline" id="Tuesday">Tuesday</SPAN></H2><H3><SPAN class="mw-headline" id="Long_Term_Aims">Long Term Aims</SPAN></H3><H4><SPAN class="mw-headline" id="Pigments">Pigments</SPAN></H4><UL><LI>Create biobricks
</LI></UL><H4><SPAN class="mw-headline" id="Amplification">Amplification</SPAN></H4><UL><LI>Find biobricks
</LI><LI>Organise system
</LI><LI>Transform
</LI><LI>Test
</LI><LI>Attach to pigments (if time)
</LI></UL><H4><SPAN class="mw-headline" id="Degredation_Tags">Degredation Tags</SPAN></H4><UL><LI>Synthesise
</LI><LI>Test
</LI><LI>Attach
</LI></UL><H4><SPAN class="mw-headline" id="Colour_Logic">Colour Logic</SPAN></H4><UL><LI>Get colour biobricks
</LI><LI>Attach inducible promoters
</LI><LI>Test
</LI></UL><H3><SPAN class="mw-headline" id="Results_of_Melanin_transformation">Results of Melanin transformation</SPAN></H3><P>Plate on the left is the melA plasmid plated on LB agar with 15ug/ml copper, 0.2ug/ul tyrosine and 100ug/ml Amp. The plate was incubated at 37 degrees for 24 hours and then left on the bench at room temperature over the weekend. Quicker induction of the melanin would be achieved by growing on plates containing IPTG. The relative concentrations of copper, tyrosine and IPTG could also be optimized.  Of particular note is the fact that the pigment appears to have diffused out of the colonies.
</P><P>The plate on the right is the  control plate, containing untransformed TOP10 E. coli. These were the cells used for the transformation.  We will also need to do a second control experiment to show that pigmentation is due to the bacteria and not due to auto-oxidation of any of the supplements we added to the media.
</P><H3><SPAN class="mw-headline" id="Wet_Work_2">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Colony_PCR">Colony PCR</SPAN></H4><UL><LI>K152005
</LI><LI>R0011
</LI></UL><P>PCR cycle = two minutes at 98 degrees followed by thirty cycles:
</P><UL><LI>Denaturation 98 degrees 30 seconds
</LI><LI>Annealing 65 degrees 30 seconds
</LI><LI>Elongation 22 degrees 3 minutes
</LI></UL><P>Followed by a final elongation at 72 degrees for 10 minutes.
</P><P>After the PCR run the products were run out on a gel (15ul of PCR product with 2ul of 1X loading buffer were added to a 0.8% agarose Invitrogen E-GEL and run for 30 mins).
</P><H4><SPAN class="mw-headline" id="Mini_Prep">Mini Prep</SPAN></H4><UL><LI>K152005
</LI><LI>R0011 - not a high enough concentration of plasmid DNA, will have to cut out of gel from colony PCR
</LI><LI>Violacein cultures didn't grow overnight, will try again tomorrow with new cultures
</LI><LI>Melanin 
</LI></UL><UL><LI>Used nanodrop spectrophotometer to measure DNA concentration.
</LI></UL><H4><SPAN class="mw-headline" id="Preparation_for_Colony_PCRs_and_Minipreps_for_Tomorrow">Preparation for Colony PCRs and Minipreps for Tomorrow</SPAN></H4><UL><LI>Overnight cultures of the cultures containing the following plasmids in LA with the appropriate antibiotic
</LI></UL><DL><DD><UL><LI>K118005 (amp)
</LI><LI>K118006 (amp)
</LI><LI>K118013 (amp)
</LI><LI>K118014 (amp)
</LI><LI>Violacein (trimethoprim)
</LI></UL></DD></DL><H4><SPAN class="mw-headline" id="New_E._Coli_Strains">New E. Coli Strains</SPAN></H4><UL><LI>Roche parent strain and FPP high flux strain, for orange plasmid
</LI><LI>Prepared to make competent cells tomorrow
</LI></UL><H4><SPAN class="mw-headline" id="Amplification_system_plan">Amplification system plan</SPAN></H4><UL><LI><B>Thursday</B></LI></UL><DL><DD><UL><LI>Prepare to make arabinose strain (BW27783) competent
</LI><LI>Incubate single colony overnight 
</LI><LI>Make more amp plates if necessary
</LI></UL></DD></DL><UL><LI><B>Friday</B></LI></UL><DL><DD><UL><LI>Do competent cell protocol
</LI><LI>Transform all 15 combination constructs.
</LI><LI>Plate  (30 amp plates)
</LI></UL></DD></DL><UL><LI><B>Saturday</B></LI></UL><DL><DD><UL><LI>Hope they grew
</LI></UL></DD></DL><UL><LI><B>Sunday</B></LI></UL><DL><DD><UL><LI>Incubate single colonies overnight in culture
</LI></UL></DD></DL><UL><LI><B>Monday</B></LI></UL><DL><DD><UL><LI>Colony PCR of each 
</LI><LI>PCR low copy plasmid
</LI></UL></DD></DL><UL><LI><B>Tuesday</B></LI></UL><DL><DD><UL><LI>Use plate reader to gather data for the 15 combination constructs in BW27783 cells on high copy number plasmid after induction with arabinose
</LI><LI>Assembly: restriction digest of combination constructs, ligation into low copy plasmid 
</LI><LI>Transform new constructs into Top10
</LI><LI>Plate (30 kan plates)
</LI></UL></DD></DL><UL><LI><B>Wednesday</B></LI></UL><DL><DD><UL><LI>Colony PCR of new constructs the sneaky way
</LI><LI>Incubate single colonies overnight in culture
</LI></UL></DD></DL><UL><LI><B>Thursday</B></LI></UL><DL><DD><UL><LI>Colony PCR of new constructs if yesterdays didn’t work
</LI><LI>Miniprep
</LI><LI>Transform into BW27783 strain
</LI><LI>Plate (30 kan plates)
</LI></UL></DD></DL><UL><LI><B>Friday</B></LI></UL><DL><DD><UL><LI>Do another plate reader analysis after induction with arabinose
</LI></UL></DD></DL><H2><SPAN class="mw-headline" id="Wednesday">Wednesday</SPAN></H2><H3><SPAN class="mw-headline" id="Dry_Work">Dry Work</SPAN></H3><P>Organising a road-map for the procedures, to be presented at the meeting tomorrow
</P><H3><SPAN class="mw-headline" id="Wet_Work_3">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Miniprep">Miniprep</SPAN></H4><P>Violacein - got a decent DNA concentration this time.
</P><H4><SPAN class="mw-headline" id="Making_more_competent_cells">Making more competent cells</SPAN></H4><P>We made the MG1655 Roche and Parent strains competent. The Roche strain has been modified from the parent strain in order to upregulate the pathways that produce the precursors for the orange pigment. We aim to transform both of them with the orange plasmid from Duncan and the orange pigmented biobrick in order to compare the difference in pigment intensity.
</P><P>We also started the procedure to make strain BW27783 competent.  This strain constitutively expresses arabinose transporters and is perhaps better suited to arabinose induction for the amplification project.
</P><H4><SPAN class="mw-headline" id="Testing_DNA_ladders">Testing DNA ladders</SPAN></H4><P>We ran a quick gel to test the ladders in the freezer from the iGEM competition last year (using the 0.8% agarose Bioline E-GELs). All hyperladders were confirmed to be working. It was decided to order more Hyperladder I as this gives the most useful spread of bands for the DNA we will be working with. This was also a test of the E-GEL system confirming the gels worked well.
</P><P>Picture above shows the gel test run for the ladders. From left: hyperladder I, four samples of hyperladder IV found in the freezer. There were no meaningful results from the PCR run, so we went back to check over the proceedure for errors.
</P><H4><SPAN class="mw-headline" id="Transformations">Transformations</SPAN></H4><P>We transformed Duncan's orange plasmid into Top10, Roche, and parent strain competent cells.  Plated onto chloramphenicol and tetracycline plates, as we're not 100% sure of the antibiotic resistance properties of the plasmid.
</P><H4><SPAN class="mw-headline" id="Preparing_culture">Preparing culture</SPAN></H4><P>Single cultures of the melA transformed cells and TOP10 cells (for a control) were incubated at 37 degrees in 20 mL of agar overnight. The aim is to try to characterise the melanin pigment using the OD and flourescence plate-reader.
</P><H2><SPAN class="mw-headline" id="Thursday">Thursday</SPAN></H2><H3><SPAN class="mw-headline" id="Dry_Work_2">Dry Work</SPAN></H3><H4><SPAN class="mw-headline" id="Primers">Primers</SPAN></H4><P>Redesigned Primer A for the melA plasmid to include the native ribosome binding site:
</P><P>Green is the suffix, purple is the ribosome binding site and the cyan is the start of the gene.
</P><P>Also characterised the vio operon in terms of the seperate genes:
</P><DL><DD>vio A = 1256bp (two unwanted restriction sites)
</DD><DD>vio B = 2996bp (six unwanted restriction sites)
</DD><DD>vio C = 1289bp (two unwanted restriction sites)
</DD><DD>vio D = 1121bp (one unwanted restriction sites)
</DD></DL><H4><SPAN class="mw-headline" id="Road_maps">Road maps</SPAN></H4><P>Made road maps for melanin, violacein, pseudomonas, detailing the constructions and biobricks we plan to make all the way to making logic circuits.
</P><H4><SPAN class="mw-headline" id="Notes_from_2:30_lab_meeting">Notes from 2:30 lab meeting</SPAN></H4><UL><LI>Need to further integrate amplification and pigments into a &quot;big idea&quot;
</LI></UL><DL><DD><UL><LI>what is the rate limiting step in pigment formation - number of enzymes or substrate?
</LI></UL></DD></DL><UL><LI>Avenues to consider
</LI></UL><DL><DD><UL><LI>Strains of bacteria that detect concentrations of something and respond by turning a different colour - we could base that on the band pass project and use a system that produces 2 pigments with one pathway, for example violacein or pseudomonas
</LI><LI>for example, as in band pass, at low concentrations of AHL make cyan, have high concentrations of AHL turn on VioC and product violet instead
</LI></UL></DD></DL><H3><SPAN class="mw-headline" id="Wet_Work_4">Wet Work</SPAN></H3><H4><SPAN class="mw-headline" id="Results_of_Previous_Colour_Transformations">Results of Previous Colour Transformations</SPAN></H4><P>We took the violacein pigment bacteria (right plate in photo) out of the fridge to find that the purple colour had started to develop. They were therefore left at room temperature overnight. The colour appears to be within the bacteria, with little or no bleeding into the media. The control plate (left plate) is the untransformed TOP10 E. coli.
</P><P>The Orange plasmid transformed into the Roche strain (with upregulated upstream pathways) had also produced colour. The two controls are above the trannsformed strain; the plate on the left shows TOP10 E. coli (i.e 'normal' E. coli colour) while the plate on the right shows the untransformed Roche strain.
</P><H4><SPAN class="mw-headline" id="Amplification_system_2">Amplification system</SPAN></H4><UL><LI>Made arabinose strain electrocompetent
</LI></UL><H4><SPAN class="mw-headline" id="Melanin">Melanin</SPAN></H4><UL><LI>Started using the plate reader to quantify melanin absorption.
</LI></UL><H4><SPAN class="mw-headline" id="Progress_on_carotenoids_.28Thursday.29">Progress on carotenoids (Thursday)</SPAN></H4><P>On Wednesday afternoon we transformed pPCB15 (CrtEXYIB, producing beta-carotene) into <B>MG1655</B> (&quot;Parent&quot;), <B>MG1655 PT5-dxs,idi,ispBDF</B> (&quot;Roche&quot;) and <B>TOP10</B>. Results of overnight culture are as follows:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:562px;"><DIV class="thumbcaption"><I>Top row from the left: pPCB15 in MG1655-PT5 (&quot;Roche&quot;), pPCB15 in TOP 10, pPCB15 in MG1655 (&quot;Parent&quot;). Bottom row from the left: K152005 in TOP10, R0011 in TOP10 (negative control).</I></DIV></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><I>Close-up of pPCB15 in MG1655-PT5 (&quot;Roche&quot;)</I></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:222px;"><DIV class="thumbcaption"><I>Close-up of K152005 in TOP10</I></DIV></DIV></DIV><DIV class="thumb tleft"><DIV class="thumbinner" style="width:212px;"><DIV class="thumbcaption"><I>Close-up of R0011 in TOP10 (negative control)</I></DIV></DIV></DIV><P>
All three strains show yellowish/orange pigmentation, indicating production of beta-carotene. We are confident that the Crt enzymes (E, B, I, Y) can function normally in E. coli, and the rate of pigment formation is fast (overnight). Previously, K152005 (CrtEBIY, without promoter) took about 3 days for yellowish/orange pigment to develop, which might be due to the absence of promoter. To confirm this, we will put K152005 under strong promoter (R0011, constitutive), transform the construct into E.coli TOP10 and the two MG1655 strains, and see if there is improvement in the rate of pigment formation.
</P><P><B>Preparation for tomorrow</B>:
</P><P>1. Overnight cultures (5ml each) for K118014 (rbs+CrtE), K118006 (rbs+CrtB), K118005 (rbs+CrtI), K118013 (rbs+CrtY), K152005 (CrtEBIY), R0011, and pPCB15, all in TOP10, with appropriate antibiotics (Amp for biobricks, Cam for pPCB15). For <B>mini-prep</B> tomorrow.
</P><P>2. Overnight cultures (10ml each) for pPCB15 in MG1655, pPCB15 in MG1655 PT5-dxs,idi,ispBDF, pPCB15 in TOP10, and R0011 in TOP10 (as control). For <B>carotene measurement</B>: try out both plate reader and acetone extraction* (*:need more reading on acetone extraction).
</P><H2><SPAN class="mw-headline" id="Friday">Friday</SPAN></H2><H3><SPAN class="mw-headline" id="Dry_work_3">Dry work</SPAN></H3><H4><SPAN class="mw-headline" id="The_Big_Idea">The Big Idea</SPAN></H4><P>Developed an outline for our system as a device made up of three parts - a sensor, an adaptor, and a pigment producer.  Our idea is to develop different bacterial strains that are sensitive to different concentrations of an inducer.  Certain concentrations of the inducer will cause violet pigment production, for example, while other concentrations will cause cyan pigment production.  This overall aim requires further attention as to how we will develop multiple sensitivities.
</P><H4><SPAN class="mw-headline" id="Synthesis_and_primers">Synthesis and primers</SPAN></H4><P>Finalised the sequence for our melanin biobrick on the ApE plasmid editor. Pictorial version shown below:
</P><P>Restriction sites are all within the registry prefix and suffix and will be used to join the gene to the plasmid backbone.
</P><UL><LI>Primers have been ordered from Invitrogen and should get here next week.
</LI></UL><P>Also examined the vio genes using the Gene-designer from DNA 2.0 with the aim of optimising them for synthesis. As the vio genes are origonally from the chromobacterium we looked at optimising codon usage for E. coli as well as adding a B0034 ribosome binding site to the beginning of each gene.
</P><H4><SPAN class="mw-headline" id="Modelling">Modelling</SPAN></H4><P>During the course of this week we have been reading about mathematical modelling of gene circuits. We now have some useful books and several papers. We are looking at using either Matlab, or software related to the Systems Biology Workbench. We may well use Cell Designer to work on our circuits graphically.
</P><P>We would like to model the amplifier such that we are able to predict transient and steady state responses to various inputs. The amplifier, at a simple level, takes a Polymerase per Second (PoPS) signal input, and produces a larger signal as an output, but this will have an inherent time lag. We are also expecting the amplifier's steady state behaviour not to be linear. This however, could be useful if it introduces semi-discrete behaviour.
</P><P>We would like to also model the pigment production response curves to a PoPS input at various levels. For this, we would need to look beyond molecular modelling, towards the optical properties of the different pigments at various concentrations.
</P><P>The nature of biological modelling means that we will introduce assumptions at certain stages in the models, but we should have an idea of what impact these assumptions have. Modelling our iGEM project has the potential to be very interesting!
</P><H3><SPAN class="mw-headline" id="Wet_work_5">Wet work</SPAN></H3><P><B>Amplifiers</B></P><P>Transformed the following biobricks from Cambridge 2007 into Top10 competent bacteria:
</P><P>Each of these biobricks is a composite construct in a high copy plasmid vector that follows the following format:
</P><P><A href="https://2007.igem.org/Cambridge/Amplifier_project" class="external text" rel="nofollow">(from the Cambridge 2007 iGEM team's wiki)</A></P><P>Plated neat and 1/10 dilutions onto amp plates.
</P><P><B>Melanin</B></P><P>Plated melanin bacteria on the following plates:
</P><DL><DD>100ug/ml Ampicillin, 7.5 ug/ml CuSO4, 0.075 mg/mL tyrosine (my bad, meant to be double that)
</DD><DD>100ug/ml Ampicillin, 15 ug/ml CuSO4, 0.3 mg/mL tyrosine
</DD><DD>100ug/ml Ampicillin, 15 ug/ml CuSO4, 0.6 mg/mL tyrosine
</DD><DD>100ug/ml Ampicillin, 7.5 ug/ml CuSO4, 0.075 mg/mL tyrosine, 0.5mM IPTG (my bad, meant to be double that)
</DD><DD>100ug/ml Ampicillin, 15 ug/ml CuSO4, 0.3 mg/mL tyrosine, 1mM IPTG
</DD><DD>100ug/ml Ampicillin, 15 ug/ml CuSO4, 0.6 mg/mL tyrosine, 1mM IPTG
</DD></DL><P>Also, left one 100ug/ml Ampicillin, 15 ug/ml CuSO4, 0.3 mg/mL tyrosine plate out, without any bacteria, just to ensure that the colouration is due to tyrosinase action and not natural oxidation of any of the agar supplements.
</P><P>Finally, incubated a melanin colony in 5 mL of 100ug/ml Ampicillin, 15 ug/ml CuSO4, 0.3 mg/mL tyrosine, 1mM IPTG over the weekend in preparation for pigment characterisation using the plate reader.
</P><H4><SPAN class="mw-headline" id="Progress_on_carotenoids_.28Friday.29">Progress on carotenoids (Friday)</SPAN></H4><P><B>Mini-prep</B> (using Zyppy Plasmid MINI PREP kit) were performed for K118014 (rbs+CrtE), K118006 (rbs+CrtB), K118005 (rbs+CrtI), K118013 (rbs+CrtY), K152005 (CrtEBIY), R0011, and pPCB15 (all in TOP10) using preparation from Thursday.
</P><P><B>Carotene extration</B> were performed on overnight cultures for pPCB15 in original MG1655 (&quot;MG1655&quot;), pPCB15 in MG1655 PT5-dxs,idi,ispBDF (&quot;MG1655 PT5&quot;), pPCB15 in TOP10, K152005 in TOP10 and R0011 in TOP10 (as control), using &quot;<A href="/Team:Cambridge/Protocols#Carotene_extraction_with_acetone" title="Team:Cambridge/Protocols">Carotene extraction with acetone</A>&quot; protocol.
</P><P>The photo below shows the results of extraction with acetone. Visual inspection put the order of intensity of yellow colour as: <B>pPCB15 in TOP10 &lt; pPCB15 in MG1655 &lt; pPCB15 in MG1655 PT5</B>. This is consistent with the fact that the FPP flux increases in same order in these strains.
</P><P><B>Measurement using microplate reader</B> was performed on both intact cells (from overnight culture) and carotene extract. A full spectrum scan was taken, covering 450 nm (absorbance by carotene) and 600 nm (cell density). Results were as follows:
</P><P>In the top graph showing absorbance spectrum of <B>intact cells</B>, there was no distinct peak at 450 nm (expected absorbance of beta-carotene). The OD 600 nm values, which indicate cell density, were also different for different cells. This indicated that the measurements on intact cells were unsuitable for detecting differences in carotene production.
</P><P>In the bottom graph showing absorbance spectrum of <B>carotene extract in acetone</B>, there were two distinct peaks, one at about 450 nm (beta-carotene), and the other at about 490 nm (unknown). The 450 nm peak was very obvious for pPCB15 in MG1655 and pPCB15 in MG1655 PT5, barely discernible for pPCB15 in TOP10, and absent in K152005 in TOP10 and R0011 in TOP10. After normalising for the differences in OD 600nm values (which were independent of carotene concentration), the OD 450 nm values increased in order: <B>pPCB15 in TOP10 &lt; pPCB15 in MG1655 &lt; pPCB15 in MG1655 PT5</B>, which was consistent with previous observations.
</P><P>The 450 nm peak in pPCB15 in TOP10 was very weak, demonstrating the importance of FPP flux in carotene synthesis. For K152005 in TOP10, the OD 450 nm was much lower. Although visibly yellow in acetone extraction, it did not show up in the absorbance spectrum, suggesting that the yellow colour might not be due to beta-carotene. An alternative hypothesis would be that insufficient time (20 hour) was given for beta-carotene production in TOP10. This could be resolved by trying out extraction with cultures kept at longer incubation time.
</P><H2><SPAN class="mw-headline" id="Weekend">Weekend</SPAN></H2><P>Checked to see how the amplification project transformations went: all but I1746370, I13540, and I13520 were growing.
</P><P>Additionally, the melanin plates were growing rather poorly, probably because they were inoculated with an overnight culture that was a few days old. 
</P><P>Primer order conformation recieved from Invitrogen
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:Cambridge/Notebook/Week3">http://2009.igem.org/Team:Cambridge/Notebook/Week3</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Cambridge/Notebook/Week3" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Cambridge/Notebook/Week3" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week3&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Cambridge/Notebook/Week3&amp;oldid=86665" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>