Team:XJTLU-CHINA/Design/qbeta
MarkdownPad Document
Qbeta replicase
Main components of Qβ replicase
During the infection of the bacteriophage Qβ into a host, an RNA-dependent RNA polymerase named Qβ replicase is formed. The polymerization activity is carried out by the core complex of this enzyme, comprising the virus-based catalytic βsubunit and the host-derived elongation factors EF-Ts and EF-Tu (Tomita, 2014).
Figure.1 replicase holoenzyme
Functions and patterns of replicase
One remarkable feature of the Qβ replicase is that during RNA replication process, it introduces high rate (~7 × 10-2) of base pair substitution with no significant preference towards transitions and transversions or skew biases (Kopsidas et al., 2007), which could be utilized to generate an RNA random mutation pool. Another notable trait of β subunit, as demonstrated by Vasiliev et al. (2010), is that without the co-expression of EF-Tu and EF-Ts, the improper folding pattern will lead to insolubility, indicating weak enzymatic activity. Therefore it is vital to insert both EF-Ts and EF-Tu into the engineered E.coli along with β subunit.
Figure.2 Mechanism of Qβ replicase
Important regions <1/p>
Based on previous study, Qβ replicase is only active on some specific genes including Qβ genome and MDV-1 which may contain particular secondary structures influencing the function of Qβ replicase. Therefore, the target gene scientists desire to study through mutagenesis should be embedded into Qβ genome to make sure that Qβ replicase will recognize and thus replicate this gene. According to literature (Schuppli, 1994) 3’ terminal region which appears to be engaged in long-range base-pairing, 5’ end containing a potential stem-loop and M site are important regions for recognition. Therefore we reserve these regions and posit our construct leaving these sites unaffected.
Figure.3 Secondary structure of Qβ genome
Genome modification
Figure.4 Important regions
Though directly inserting our construct into Qbeta genome will guarantee the effective recognition and thus replication of our template, adequate deletion of some region is required to inactivate phage infectivity while maintaining recognition capability. A closely related study conducted by Schuppli (1994), Barrera and Weber shows deleting the central region (approximately from 600 to 3342) will have little influence on replication activity. The study of Mills (1988) demonstrated the significance of replicase encoding region (from 2360 to 4120), which is necessary for both in vivo and in vitro replication. It also indicates that insertion or substitution in the region before 2360 is acceptable by Qβ replicase without abolishing its function.
Figure.5 Our design
Based on research, our project decided to delete the Qβ genome from 701 to 2340, where intron and target gene will then be inserted.  To continue explore our design principle, please click here – (intron description)
References:
Kopsidas, G., Carman, R.K., Stutt, E. L., Raicevic, A., Roberts, A. S., Siomos, M. V., Dobric, N., Pontes-Braz, L. and Coia, G., (2007) RNA mutagenesis yields highly diverse mRNA libraries for in vitro protein evolution. BMC Biotechnology, 7(1), pp.18.
Mills. D. R., Priano. C., Dimauro. P., Binderow. B. D. (1988). ‘Qp Replicase: Mapping the Functional Domain of an RNA-dependent RNA Polymerase’ Journal of Molecular Biology, 205(4) 751-764
Schuppli. D., Barrera. I., Weber. H. (1994). ‘Identification of Recognition Elements on Bacteriophage Qβ Minus Strand RNA that are Essential for Template Activity with Qβ Replicase’ Journal of Molecular Biology, 243(5): 811-815
Tomita, K., (2014) Structures and Functions of Qβ Replicase: Translation Factors beyond Protein Synthesis. International Journal of Molecular Sciences, 15(9), pp.15552-15570.
Vasiliev, N. N., Jenner, L., Yusupov, M. M., and Chetverin, A. B., (2010) Isolation and Crystallization of a Chimeric Qβ Replicase Containing Thermus thermophilus EF-Ts. Biochemistry (Moscow), 75(8), pp. 989-994.
