Team:XJTLU-CHINA/Design/intron
Home
Team
Members
Attributions
Collaborations
Project
Project Description
Experiments
Notebook
Mathematical Model
Parts
Parts
Registry Parts Improvement
Human practices
Overview
Silver
Gold
Public Engagement
Safety
Toggle navigation
XJTLU CHINA
Home
Team
Members
Attributions
Collaborations
Project
Project Description
Experiments
Proof of Concept
Demonstrate
Notebook
Mathematical Model
Parts
Basic Parts
Composite Parts
Registry Parts Improvement
Human Practices
Overview
Silver
Gold
Public Engagement
Safety
How group II intron works?
Now welcome Intron!
Now that mRNA has been replicated by Qβ replicase, the incoming step is to convert RNA into a stable form, DNA. Reverse transcription is required. Cousineau et al. (1998) reported a mobile group II intron Lactococcus lactis, Ll.LtrB. This intron has activity of retrohoming in which the intron integrates into recipient DNA by reverse splicing and reverse transcription. In other words, RNA after error-prone replication can be integrated into recipient plasmid and then be converted into DNA. Therefore, Ll.LtrB intron is the ideal tool to collect mutated DNA. The mechanism of Ll.LtrB intron is relatively simple. The whole process contains five steps: self-splicing, ribonucleoprotein formation, reverse-splicing, reverse transcription and DNA repair. Importantly, the first four steps are all associated with a vital intron encoded protein, LtrA. It is a 599 amino acids protein containing activities of maturase, reverse transcriptase and endonuclease. Specific activities will be elucidated in the following.
Mechanism
Figure 1. Flexible ORF expresses LtrA
Figure 2. Flow chart of retrohoming
Intron RNA precursor is transcribed, and LtrA is expressed.
The maturase activity of the LtrA assists with self-splicing the intron RNA from the precursor transcript. Firstly, LtrA helps RNA form a catalytic active structure. Secondly, intron binding sequence (IBS) interacts with exon binding sequence (EBS) to locate splicing site. Thirdly, two transesterification reactions cut off 5’ and 3’ exons and form an intron lariat.
Figure 3. Secondary structure of intron RNA. EBS-IBS interaction
Following splicing, the intron lariat binds to LtrA to form a RNA-protein complex called ribonucleoprotein (RNP).
Subsequently, the RNP scans the genome and plasmids enquiring each site by base pairing using the EBS of the intron RNA (EBS1–IBS1, EBS2–IBS2, δ–δ’). Apart from base pairing, the approximate 35bp region is regarded to influence target recognition efficiency. After target recognition, intron reverse splices into the position between 5’ and 3’ exon on the sense strand.
Figure 4. Intron base pairing with target DNA.
EBS1–IBS1, EBS2–IBS2, δ–δ’. An approximate 35bp region is displayed upon reverse splicing. LtrA approaches the antisense strand and cleaves +9 with endonuclease activity, creating a free 3’-OH.
Starting from the free 3’-OH, LtrA synthesizes cDNA taking reverse-spliced intron RNA as the template. Reverse transcription takes place in this process.Since cDNA has been synthesized, RNA at the sense strand is replaced by DNA. This mechanism has not been clear yet. Cousineau et al. (1998) indicates DNA repair events account for it.
Design
Because the open reading frame of Ll.LtrB intron is not conservative, it is possible to insert foreign gene into that region, and the gene can be carried with intron along retrohoming. According to Cousineau et al. (1998), a two plasmids system can convey kanamycin resistance gene (KanR) from the donor to the recipients. Ll.LtrB intron was engineered into the donor and target exons was engineered into the donor. Also, Yao and Lambowitz (2007) moved LtrA sequence to the downstream without influencing its function.
Figure 5. The two plasmids system designed by Cousineau et al. (1998)
Inspired by the previous design, we put forward an innovative two plasmids system. In this system, one plasmid is taken both as donor and recipient, and another plasmid undertakes the expression of enzymes. Ll.LtrB intron module and target site are designed on the same plasmid, pETDuet-1, but LtrA is placed in another plasmid, pACYCDuet-1. It is assumed the two plasmids would collaborate and function more stably by doing RNA transcription and protein expression separately.
Figure 6. pACYCDuet-1, LtrA expression plasmid
Figure 7. pETDuet-1 containing intron and a target site
To validate this two plasmid system, we introduce an inactivated KanR-RBS, which is placed in the reverse direction, into Ll.LtrB intron. Moreover, a pair of wild-type exons, which are also placed in the reverse direction, are taken as the target. Therefore, once Ll.LtrB intron is inserted into the target, the direction of KanR-RBS will be recovered, and its expression is activated.
Figure 8. Reversed KanR with a RBS in the intron sequence
Figure 9. A target site is place at the reverse direction
To prevent leaky expression of KanR prior to retrohoming, KanR with RBS is placed in the reverse direction so that kanamycin resistance can only be expressed when intron is inserted into target DNA.
Figure 10. KanR with a RBS
According to Mohr et al. (2000), EBS and IBS can be modified through PCR to retarget to a specific DNA sequence according to a certain principle. Therefore, we developed an algorithm to search for capable target sites and generate corresponding EBS and IBS (Kimmel and Axelrod, 2015) . Here, a LacZα gene is put adjacent to wildtype target as the subject of retargeting. Hence, the effectiveness of the algorithm can be validated.
Figure 11. lacZα and a reversed target site.
In order to switch on retrohoming process independently, the expression of the LtrA is designed to be controlled by a pBAD promoter, which is an arabinose inducible promoter.
Figure 12. LtrA expression is controlled by pBAD promoter.
Overall, the final construction is displayed in the following two figures.
Figure 13. Plasmid construction
Cousineau, B., Smith, D., Lawrence-Cavanagh, S., Mueller, J. E., Yang, J., Mills, D., Manias, D., Dunny, G., Lambowitz, A. M. and Belfort, M. (1998) 'Retrohoming of a bacterial group II intron: mobility via complete reverse splicing, independent of homologous DNA recombination', Cell, 94(4), pp. 451-462. Kimmel, M. and Axelrod, D. E. (2015) Branching Processes in Biology. Second edn. New York Heidelberg Dordrecht London: Springer. Mohr, G., Smith, D., Belfort, M. and Lambowitz, A. M. (2000) 'Rules for DNA target-site recognition by a lactococcal group II intron enable retargeting of the intron to specific DNA sequences', Genes & development, 14(5), pp. 559-573. Yao, J. and Lambowitz, A. M. (2007) 'Gene targeting in gram-negative bacteria by use of a mobile group II intron (“targetron”) expressed from a broad-host-range vector', Applied and environmental microbiology, 73(8), pp. 2735-2743.
Qbeta Intron Results
Location
Rm 363, Science Building
Xi'an Jiaotong-Liverpool University
111 Ren'ai Road, Suzhou,China
215123
Around the Web
Last Year Project
About XJTLU-CHINA
XJTLU-CHINA igem is sponsored by
Synbio, XJTLU, Snapgene, NEB
Learn More
XJTLU-CHINA igem 2016
