<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_XJTLU-CHINA_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:XJTLU-CHINA/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="fixed-nav-bar text-center"><UL class="breadcrumb"><LI style="color:#428bca;"><A href="https://2016.igem.org/Team:XJTLU-CHINA" style="color: #428bca">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style="color: #428bca">Team</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style="color: #428bca">Project</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Design">Project Description</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Model">Mathematical Model</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style="color: #428bca">Parts</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Basic_Part">Parts </A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Description">Registry Parts Improvement  </A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#" style="color: #428bca">Human practices</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/HP/overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Engagement">Public Engagement</A></LI><LI style="color:#002bb8;"><A href="https://2016.igem.org/Team:XJTLU-CHINA/Safety" style="color: #428bca">Safety</A></LI></DIV><NAV class="navbar navbar-default navbar-fixed-top" role="navigation"><DIV class="container"><DIV class="logo"><DIV class="navbar-header"><BUTTON type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"><SPAN class="sr-only">Toggle navigation</SPAN></BUTTON><A class="navbar-brand" href="Team:XJTLU-CHINA"><DIV style="margin-top: 5px;"><SPAN class="light">XJTLU</SPAN> CHINA </DIV></A></DIV><DIV class="collapse navbar-collapse navbar-right" id="bs-example-navbar-collapse-1"><UL class="nav navbar-nav "><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA">Home</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Team</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Team">Members</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Attributions">Attributions</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Collaborations">Collaborations</A></LI></UL><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Project</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Design">Project Description</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Experiments">Experiments</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Proof">Proof of Concept </A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Demonstrate">Demonstrate</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Notebook">Notebook</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Model">Mathematical Model</A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Parts</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Basic_Part">Basic Parts </A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Composite_Part">Composite Parts </A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Description">Registry Parts Improvement  </A></LI><LI class="dropdown"><A class="dropdown-toggle" data-toggle="dropdown" href="#">Human Practices</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/HP/overview">Overview</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/HP/Silver">Silver</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/HP/Gold">Gold</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Engagement">Public Engagement</A></LI><LI><A href="https://2016.igem.org/Team:XJTLU-CHINA/Safety">Safety</A></LI></DIV></DIV></DIV></NAV><DIV class="container"><DIV style="margin-top: 300px;"><DIV class="row"><DIV class="col-12"><DIV class="page-header"><H1 style="font-weight:400;">Experiments</H1></DIV></DIV></DIV><DIV class="panel-group" id="accordion" role="tablist" aria-multiselectable="true"><DIV class="panel panel-delft"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseOne1" aria-expanded="true" aria-controls="collapseOne1"><DIV class="panel-heading" role="tab" id="headingOne1"><H4 class="panel-title">

                                        Working with Qβ replicase
                                    </H4></DIV></A><DIV id="collapseOne1" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne1"><DIV class="panel-body"><H2>Introduction</H2><P>In the model that we proposed, random mutations were accomplished by the error-prone amplification of RNAs, which is different from the DNA-based way of building mutagenesis library.  As we have discussed in the design section, RNA have several superior characteristics over DNA in the case of introducing mutations, including evasion of efficient host-repair systems and abundancy in numbers. However, the main drive that determined RNA as the chassis of mutagenesis in our project was the finding of a RNA-dependent-RNA-polymerase (RdRp) from bacteria phage Qβ, named as Qβ replicase. This RdRp was previously reported of having specific activity in amplifying Qβ genomic RNA and a small RNA variant called MDV-1. Experiment results indicate that a single MDV-1 template can produce 10<SUP>12</SUP> replicates in only 10-15 mins at 37 ℃, a preferred characteristic since the errors that the RdRp generated can be accumulated.
The first priority of our wet lab is to express functional Qbeta replicase holoenzyme in bacterium Escherichia coli BL21 and perform function assays to confirm the robust RNA amplification activity of the enzyme. </P><DIV class="row"><DIV class="col-6"><H2>Method and Results</H2><H3>Primary attempt in expressing β subunit</H3><P>As a first step, β subunit of Qβ replicase was cloned into a commercial vector pETDuet-1 by Gibson assembly (Figure 1) and the resulting plasmid was named as pETDuet-Rep. The plasmid was transformed into <EM>Escherichia coli</EM> strain BL21. The correct colony was picked and cultured in LB culture that contains ampicillin at concentration of 50μg/mL and induced with 0.4mM IPTG for 8 hours once the OD600 value reaches 0.6. The cells was then harvested, sonicated and sampled for SDS-PAGE.</P><H3>Results</H3><P>This design failed because somehow the expression of β subunit cannot be detected in SDS-PAGE gels（see combined results in Figure 3).</P><H3>How we restore the expression of β subunit</H3></DIV><DIV class="col-6"><P>Figure 1. Plasmid map of PetDuet-1-Rep</P></DIV></DIV><DIV class="row"><DIV class="col-6"><P>Figure 2. Plasmid map of pACYC-TS-TU</P></DIV><DIV class="col-6"><H3>How we restore the expression of β subunit</H3><P>In order to aid the solubility and increase the expression level of β subunit, the other two domains of Qβ holoenzyme complex (learn more) were also cloned into another commercial vector pACYCDuet-1 (Fig.2) to get a construct named as pACYC-TS-TU. This cloning process generates a intermidiate plasmid-pACYC-TU, which contains the ORF of EF-TU and a upstream T7 promoter. pACYC-TU was transformed into BL21 and tested for the production of EF-TU and pACYC-TS-TU was co-transformed with pETDuet-Rep into <EM>Escherichia coli</EM> strain BL21.</P><P>Cells transformed with two plasmids were spread on LB agar plate that contains 50μg/mL ampicillin and 30μg/mL chloramphenicol while cells transformed with pETDuet-1-Rep were spread on the plates that only contains 50μg/mL ampicillin. The correct colonies was picked next morning and cultured in LB culture with respective antibiotics and induced with 0.4mM IPTG for 8 hours once the OD600 value reaches 0.6. The cells were then harvested, sonicated and sampled for SDS-PAGE.
</P></DIV></DIV><H3>Results</H3><P>Whole cell lysate, supernatant and precipitations of cells with different constructs are loaded into SDS-PAGE gels. Figure 3 shows the electrophoresis results of samples that transformed with three different constructs:
1. pETDuet-1-Rep (Expressing β subunit alone)		
2. pACYC-TS-TU (co-expressing EF-Ts and EF-Tu)
3. pACYC-TU (Expressing EF-Tu alone)
It can be clearly observed that, unpon inducement of IPTG, pETDuet-1-Rep doesn't lead to the abundant production of β subunit. None of the three lanes of cells that transformed with pETDuet-1-Rep display a detectable band at 65.6 kDa, the size of β subunit. The cells that transformed with pACYC-TS-TU and pACYC-TU, however, shows unambiguous production of EF-Ts (position indicated by red arrow) and EF-Tu (position indicated by orange arrow).  
</P><P>
Figure 4. SDS-PAGE results of samples transformed with pETDuet-1-Rep alone or co-transformation of pACYC-TS-TU and pETDuet-1-Rep. A wild-type control was included.</P><H2>Discussion and Conculsion</H2><P>Based on our results, it is clear that co-expression of EF-TS and EF-TU is essential for the production of β subunit as well as the generation of Qβ replicase holoenzyme. However, in our experiment conditions, inclusion bodies still take a considerable porpotion of the total production of β subunit under the help of EF-TS and EF-TU.</P><A href="#top" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#top').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV><DIV class="panel panel-default"><A class="collapsed" data-toggle="collapse" data-parent="#accordion" href="#collapseTwo2" aria-expanded="false" aria-controls="collapseTwo2"><DIV class="panel-heading" role="tab" id="headingTwo2"><H4 class="panel-title">

                                        Retro-homing of Group II intron

                                    </H4></DIV></A><DIV id="collapseTwo2" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingTwo2"><DIV class="panel-body"><H2>Introduction</H2><P>LtrB Group II introns, branded by Sigma-Aldrich as TargeTron® has been applied to the industry for many years. However, in this project, the disruption target was no longer a genomic gene but instead, a piece of plasmid DNA was preset as the target. Our lab decided to test this plasmid disruption design first before we combined it with RNA error-prone amplification system. </P><DIV class="row"><DIV class="col-6"><H2>Method and results</H2><H3>Primary attempt in customizing LtrB intron</H3><P>We adapted the TargeTron® design from Sigma-Aldrich to our two plasmid system. One of our plasmids pACYC-Maturase (Figure 5) encodes LtrA protein, which is respensible for the retro-homing of intron. The intron transcript was produced by a plasmid called pET-Duet-intron (Figure 6). These two plasmids were transformed into <EM>Escherichia coli</EM> strain BL21. Correct colonies were picked and cultured in LB culture that contains 50μg/mL ampicillin and 30μg/mL chloramphenicol and induced with 0.4mM IPTG and 4mM arabinose for 8 hours once the OD600 value reaches 0.2. The cells were then plated on three antibiotics plate (CmR, Amp, Kan) for the selection of colonies with insert.</P></DIV><DIV class="col-6"><P>Figure 5. Plasmid map of pACYC-Maturase. The expression of maturase is under the control of pBAD promoter (Col. 13 of registry pBAD SPL).</P></DIV></DIV><DIV class="row"><DIV class="col-6"><P>Figure 6. Design of pET-Duet-intron. </P></DIV><DIV class="col-6"><H3>Results</H3><P>After 12 hrs incubation, handful amount of conlonies grew on three antibiotics plate. However, cells transformed with pET-Duet-intron alone were found grow on kanamycin plates.</P></DIV></DIV><H3>Discussion</H3><P>It was unexpected that cells transformed with pET-Duet-intron can grow on kanamycin plates. By looking back at our design, it turns out that one of the terminator we used, known as T7Te terminator was a medium strength promoter in the oppsite direction. We suspected this bug might compromise the selection of kanamycin and accounts for the growth of pET-Duet-intron hosts.</P><H3>T7Te terminator Debugging</H3><P>In order to prove our hypothesis of the naughty function of T7Te terminator, primers were design to remove T7Te terminator and put a bi-directional terminator (BBa B1006) at the original T7Te position as a subsitute. This design was belived to have the power of eliminating noise from the promoter on the plasmid backbone and restore the selection of kanamycin ultimately. </P><H3>Results</H3><P>After terminator substitution, the amount of viable colonies on kanamycin plate is significantly reduced while the amount of colonies grew on amplicilin plate remains unaffected (Figure 7). This suggested that T7Te terminator might be the main cause of compromise kanamycin selection.

</P><DIV class="row"><DIV class="col-12">
Figure 7. BL21 cells transformed with debuged pET-Duet-intron spreaded on Kan and Amp plates.</DIV></DIV><A href="#top" class="btn btn-info" role="button" onclick="$('html,body').animate({scrollTop: $('#top').offset().top}, 'slow');" style="text-decoration:none; color:#f3f4f4; float:right;">Back to Top</A></DIV></DIV></DIV></DIV></DIV></DIV><DIV class="footer-above"><DIV class="container-fluid"><DIV class="row"><DIV class="footer-col col-4"><H3>Location</H3><P style="text-align: center;">Rm 363, Science Building
						Xi'an Jiaotong-Liverpool University 
	111 Ren'ai Road, Suzhou,China215123
</P></DIV><DIV class="footer-col col-4"><H3>Around the Web</H3><A href="https://2015.igem.org/Team:XJTLU-CHINA/Team" class="btn-square">Last Year Project</A></DIV><DIV class="footer-col col-4"><H3>About XJTLU-CHINA</H3><P style="text-align: center;">XJTLU-CHINA igem is sponsored by Synbio, XJTLU, Snapgene, NEB</P><A href="2016.igem.org/Team:XJTLU-CHINA/Sponsors" class="btn-square">Learn More</A></DIV></DIV></DIV></DIV><DIV class="footer-below"><DIV class="container"><DIV class="row"><DIV class="col-12">
                        XJTLU-CHINA igem 2016
                    </DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>