Team:XJTLU-CHINA/Description
Home
Team
Members
Attributions
Collaborations
Project
Project Description
Experiments
Notebook
Mathematical Model
Parts
Parts
Registry Parts Improvement
Human practices
Overview
Silver
Gold
Public Engagement
Safety
Toggle navigation
XJTLU CHINA
Home
Team
Members
Attributions
Collaborations
Project
Project Description
Experiments
Proof of Concept
Demonstrate
Notebook
Mathematical Model
Parts
Basic Parts
Composite Parts
Registry Parts Improvement
Human Practices
Overview
Silver
Gold
Public Engagement
Safety
MarkdownPad Document
Motivation
RNA thermometers (RNATs) are RNA sequences that can post-transcriptionally regulated gene expression in response to temperature shifts by a way of undergoing conformation changes in the secondary structure of RNA. below shows the secondary structure of natural formed RNA thermometer found in 5鈥?untranslated region of many eubacteria. Graph 1. Last year XJTLU_CHINA 2015, tested more than 10 nature existed and artificial RNATs using Egfp as reporter gene. Though all of these RNATs are reported perfect performance in original experiments, the expression of EGFP can only be controlled by three RNATs. < a href="http://parts.igem.org/Part:BBa_K1824556 " style="color: #428bca" title="qbeta">A1from BIT China showed highest protein expression and temperature induction (more than 10 times of induction). This year again, we want to further characterize the A1 RNAT using MRFP as reporter gene. However, when the green fluorescence gene was replaced by red florescence gene, surprisingly, no protein was expressed. (Figure X) Hence the aim of this year is to optimize the performance of RNAT A1 in the expression of Mrfp.
Figure. 1 The secondary structure of RNAT A1.
Principle of RNATs
The principle is shown in (Figure 2.), the hairpin structure harbors the Shine-Dalgarno sequence (SD sequence) and, in this way, make it inaccessible to the 30S unit of the bacterial ribosome, resulting in inactivation of translation. (SD sequence is polypurine sequence located in around 8 nt upstream from the start codon and responsible for anchoring ribosome in the RNA single strand). Once reaching a certain temperature, hairpin structure would vanish and as a result, exposing the SD sequence to trigger the translation process.
Figure 2. Responsiveness of mRNA structures to environmental cues (TUDelft, 2008)
Methods
At first, the RNAT A1 was constructed under the control of constitutive promoter J23119. The first 25 bp of RNAT A1 forming the secondary structure with the SD sequence (AAGAG) was deleted for positive control. In order to optimize the performance of RNAT A1, constructing a mutation library is the first choice. Yet, the sequence of RNAT A1 is only 42bp, it is difficult to mutate such a short sequence and construct it onto plasmid through error prone PCR. To combat this problem, instead, degenerative PCR, which using degenerative primers to amplify DNA, was employed for the plasmid construction. Degenerative primers are mixture of oligonucleotide sequences in which some positions contain any type of nucleotides (ATGC), giving a population of primers with similar sequences that cover all possible nucleotide combinations at certain sequence (Iserte 2013).
Table 1. The abbreviations for degenerate bases
Two strategies were recruited when design the degenerative primers. The first is to mutate the anti-SD sequence (CTCTT) to lower the energy of the hairpin structure, and the second is to change the poly A sequence of the RNAT, which may form complex structures to interfere with the SD, anti-SD structure in long distance. (Table 2.) The second strategy may also increase the robust of the RNAT, which may result in a RNAT structure that is suitable to a wide range of proteins.
To construct the mutant RNAT, the J23119A1MRFP in pEt28a plasmid was first built. Then, the whole J23119 RFP sequence was amplified using degenerative primers. The amplified sequence was recombined with the PCR linearized plasmid using Gibson assembly. The recombined plasmid was then transformed into Top 10 competent cell.
To find out the best performed colonies, 288 colonies were picked out from the library constructed by degenerative primer 1 and 2. The colonies were inoculated in 96 well plate and shake at 37 鈩?and 30 鈩?for 18 h. The fluorescence of MRFP (excitation: 580, emission: 607) were tested triplicate. However, the results of culture in 96 well plat were unstable due to the low culture system. Hence, the red colour of the 30 鈩?and 37 鈩?culture were compared and the best cultures were streaked on the LB agar plate, and incubated at 37 鈩?and 30 鈩?for 18 h separately. (Figure 3.) The degree of red was compared, and again the best performed samples were shake in 150 ml flasks at 20 ml system. The fluorescence of the shaking flasks were tested and compared.
Figure 3. Streaked plate of different RNATs at 30 鈩?and 37鈩? (F3: JB1F3, F4: JB1F4, G1: JB1G1, G2: JB1G2, G3: JB1G4, G3: JB1G4, H1: JB1_H1, J23119 mut, J23119 PC: positive control)
Results
The Result of the fluorescence of different RNAT at 26 鈩?and 37 鈩? (Figure 4.) As is shown on the graph, the natural RNAT A is too strong to even express the MRFP. Normally, the positive control (no anti-SD and poly A) showed no significant difference between 26 鈩?and 37 鈩? The mutation of RNAT truly improve the performance of RNAT up to 4.5 times difference and increase the expression level of Mrfp. These RNAT were sequenced, and the sequence are shown in Table 2.
Figure 4. RNAT performance test. The expression level of MRFP at 37 鈩?and 26 鈩?are tested.
Further plans
The test of RNAT library is still on going. The future plan is to find out the RNAT with lower leakage and higher expression. Also we want to further test the performance of the RNAT using different reporter genes such as GFP and luciferase to find out the most rebut RNAT.
References
Iserte, J.A., Stephan, B.I., Goni, S.E., Borio, C.S., Ghiringhelli, P.D., Lozano, M.E. (2013) Family-specific degenerate primer design: a tool to design consensus degenerated oligonucleotides. Biotechnol Res Int 2013:38364
Location
Rm 363, Science Building
Xi'an Jiaotong-Liverpool University
111 Ren'ai Road, Suzhou,China
215123
Around the Web
Last Year Project
About XJTLU-CHINA
XJTLU-CHINA igem is sponsored by
Synbio, XJTLU, Snapgene, NEB
Learn More
XJTLU-CHINA igem 2016
