<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU_Denmark_modelling"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:DTU_Denmark/modelling">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU_Denmark/modelling&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU_Denmark/modelling&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU_Denmark/modelling&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU_Denmark/modelling" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU Denmark/modelling</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Welcome to the DTU iGEM wiki!</TITLE></P><TABLE class="links"><TBODY><TR><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark">Home</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/team">The Team</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/project">The Project</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/parts">Parts submitted</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/modelling">Modelling</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/notebook" title="Day to day lab activity">Notebook</A></FONT></TD></TR></TBODY></TABLE><TABLE><TABLE cellpadding="10px"><TBODY><TR><TD width="556px" height="100%" valign="top"><FONT color="#990000" face="arial" size="5"><B>Modelling of The Redoxilator</B></FONT><FONT color="#333333" face="arial" size="2.5"><FONT size="3"><P><B>Introduction</B></P></FONT>

Here a model for the GFP expression under the control of “The Redoxilator” expression system is derived. The model is based on modular structured kinetics and links the level of the signal molecules NADH and NAD to the final expression of GFP through the events of 

  i) transcription factor activation,
  ii) promoter activation,  
  iii) synthesis and degradation of mRNA and finally, 
  iv) synthesis, dilution and degradation of GFP. 

This model formed the basis of the genetic design of the The redoxilator. Importantly it provided understanding of the influence of each of the parameters when designing the system, e.g. that in order to achieve fast response time a fast degradable GFP was needed as reporter. However, since many parameters specific for this system are not yet available, the model is not expected to make exact predictions but to ease the understanding of the design &quot;principles&quot;.

<DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The redox coupled system</DIV></DIV></DIV></DIV><P align="justify"><I><B>Gene design and redox regulation</B>
NADH and NAD competes about binding to REX, however only when NADH is bound, the trancription factor is inhibited from binding to DNA. When the transcription factor is active (NADH not bound), the fused activator domain summons the RNA polymerase and the reporter gene is transcribed.</I></P><FONT size="3"><P><B>NADH and NAD binding to the transcription factor</B></P></FONT>

Both NADH and NAD binds reversible to the same site of the transcription factor (TF): 

Dissociation of TF and NADH


at equilibrium: 


Dissociation of TF and NAD:


Dissociation constant of TF and NAD:  


Balance of NAD and NADH. Constant total level of NADH + NAD in the cell are assumed:


Balance of TF. TF is constitutively expressed leading to constant total value . 


NADH inhibits DNA binding activity of TF and thus transcription activity. Active TF is designated TF’.


Leading to the conserved moiety:


Using (3), the fraction of TF’ (<I>tf</I>) can be written as


With (1) and (2) in (4) the <I> tf </I> becomes


Leading to


Importantly figure 1 shows how the transcription factor is affected by the cofactor pool of NAD(H). The competing binding of NADH and NAD makes the system sense the redox poise rather than the concentration of each of the cofactors [2]. <I><B>Figure 1 - Transcription factor activity with increasing NAD/NADH ratio is hyperbolic.</B> At NAD/NADH = 0.1, <I>tf</I> = 0.5, i.e. half of the transcription factors are active. (For this plot the following parameters were used: assuming K_H = 0,02 μM and K_N = 2 μM since REX NADH affinity &gt; REX NAD affinity [2].  = 2mM, estimated TF concentration expressed from a constitutive TEF promoter.)</I><FONT size="3"><P><B>Promoter binding of active transcription factor</B></P></FONT>

When the active TF (TF') is bound to the promoter (P), P becomes active (P’) and transcription is initiated. The number of transcription factors needed for promoter activation is designated m:


Here it assumed that binding of TF and TFN is equal.  
The dissociation equilibrium for this reaction:



The number of promoters is constant, making the sum of fractions equal to



Where the fraction of active promoter <I>Pf</I> is

 

Introducing P’ from (6) gives 


Now introducing  we can obtain a function of the fraction of active promoter sites as a function of NAD(H) level:



For simplicity the expression of (5) has not been inserted. Figure 2 shows how the redox poise modulates transcription from the promoter P, which is controlled by The Redoxilator.<I><B>Figure 2 - Fraction of active promoters with increasing NAD/NADH ratio.</B> The curve is sigmoidal and has almost switch like behavior. From NAD/NADH = 0 to 0.01 the difference in active promoter is almost 60 %.(Assumed value of K_P = 1 mM, and m = 2). 
</I><FONT size="3"><P><B>mRNA production</B></P></FONT>

mRNA is produced from the transcription of the active promoter. Concentration of intracellular levels of mRNA () can be derived from the mRNA balance:



Here it is assumed that mRNA is degraded corresponding to a first order reaction and that mRNA is not exported from the cell. 

The term  represents the dilution due to growth. However mRNA is highly unstable and is thus degraded within tens of seconds to order few minutes, making <I> k_dm</I> very small compared to growth rate. This makes , simplifying the mRNA balance to:

                     (9)<B>Production rate of mRNA </B>
It is assumed that when RNA polymerase is efficiently bound to the promoter, the promoter is active and mRNA is transcribed at constant maximum rate . The binding of RNA polymerase is dependent on the presence of transcription factor and determined by signal strength (NAD/NADH) as well as Kp. In the overall production rate of mRNA, of course also the number of promoters (and genes) Np is influencing:



For a given organism  and Np are constant. Inserting (10) into (9) the mRNA balance becomes:

<B>Steady state concentration of mRNA</B>
At steady state (11) is equal to zero since no mRNA accumulates. Here the mRNA concentration is:

<I><B>Figure 3 - mRNA levels at steady state with increasing NAD/NADH ratios.</B>Dilution is neglected since degradation rate of mRNA is much larger. (Assumed parameters: kdm = 8.3 h-1 (half life = 5 min), Np = 60 (2 micron plasmid is used). and   = 180 U/h (assuming 75 bases per second, and 1500 bases in gene).)
</I><FONT size="3"><P><B>GFP production</B></P></FONT>
Finally, the mRNA is used as template for the process of translation to protein, here GFP. The GFP protein balance:



As with mRNA, it is assumed that GFP is degraded by first order rate. The dilution cannot be neglected since protein degradation is much slower than mRNA degradation. Furthermore it is assumed that GFP is not transported out of the cell.



The protein is synthesized by ribosomes. The number of ribosomes bound to one mRNA (Np) and the protein production rate of the ribosome (kp) should be taken into account. The number of ribosomes in the cell, ATP and available charged amino acids are assumed to be non-limiting. Taking this into account, the protein synthesis rate can be described as



Making the final GFP balance



This final relation relates the GFP balance directly to the levels of NADH and NAD through mRNA level, promoter activity and transcription factor activity. <B>Steady state concentration of GFP</B>
From (15) we find that the steady state concentration of GFP is



This relation opens certain possibilities: By knowing the NAD/NADH ratio of the cell, one can predict the GFP expression using this model. Or opposite and even more relevant, knowing (by measurement) the  GFP level of the cell, one can point out the redox state. In this way The redoxilator acts as an <I>in vivo</I> redox sensor. Figure 4 shows how the steady state concentration of GFP changes at different redox poises. 
<I><B>Figure 4 - Steady state GFP levels as a direct function of NAD/NADH signal levels.</B>  of a cell in steady state. (Assumed parameters Nrib = 1. kp = 288 U/h, assuming 40 amino acids per second. k_dGFP = 1.386 h-1 equal to a half time of 30 min [3]. Growth rate = 0.1 h-1).</I>

So far a model has been derived showing how steady state GFP levels changes with different values of NAD/NADH poise. This is applicable when the NAD/NADH ratios are constant. However, when NAD/NADH changes in time, GFP also changes in time <I>dynamically</I>. In order to sense changes in NAD/NADH ratios it is important that the reporter signal (GFP level) changes relatively fast. The following paragraph deals with the dynamic state.
<FONT size="3"><P><B>The dynamic state</B></P></FONT>

When the cells are exposed to a certain NAD/NADH level mRNA and in turn GFP levels will change in time according to their balances. In figure 5 the NAD+ level is changed from 0 M to 5mM equal to total amount of NAD(H).  

<I><B>Figure 5 - Dynamics of GFP expression when NAD+ level is changed</B><B>A:</B> At t = 1, the NAD+ level is suddenly shifted from 0 M to the total concentration of NAD(H). At t = 4, the NAD is shifted to 0 M. <B>B:</B> dynamic mRNA level. A fast response in mRNA levels is observed in both production and degradation. <B>C:</B> dynamic GFP levels. The response is much slower than mRNA. After app. 1/2 hour from signal burst, the GFP level has reached half of the maximum level. Degradation and dilution is relatively fast for a protein with a response time of approximately 40 min.
</I>

Figure 5 illustrates the importance of using a fast degradable GFP when using the redoxilator system as a sensor. If the cells that are subject for an <I>in vivo</I> analysis of redox poise using The redoxilator suddenly shift to a new state, the sensor will respond fast (in about 40 minutes with the assumed parameters). 



If the NAD/NADH level changes in time, all values depending on NAD(H) concentrations will also be time depending. This will result in periodic burst in the production of the protein under control of the redox system:

<I><B>Figure 6 - Dynamics of GFP expression when NAD+ level is changed</B><B>A:</B>Simulated change in NAD+/NADH level.<B>B:</B> Since NAD+/NADH ratio changes periodically, also transcription factor activity changes.<B>C:</B> The changes in NAD+ affects the dynamic GFP levels.
</I>
Figure 6 shows how the protein under control of the redoxilator is produced in bursts. Due to the fast degradation of GFP in this case, GFP levels will change in response to the redox level with a fast response time. 

<B>References</B>
[1] Heijnen J.J 2009. Modular structured kinetic models: protein production from genes.
[2] Brekasis, D., Paget, M.S., 2003. A novel sensor of NADH/NAD+ redox poise in Streptomyces coelicolor A3(2). EMBO J. 22, 4856–4865.
[3] Mateus and Avery, 2000. Destabilized green fluorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry. Yeast 2000; 16: 1313±1323. </FONT></TD><TD width="182px" height="100%" valign="top" class="greybox"><FONT face="arial" size="2"><B>Modelling facts</B><B>Abbreviations used in the model</B><I>TF</I> = the transcription factor based on the REX protein <I>TF'</I> = active transcription factor <I>N</I> = concentration of NAD <I>H</I> = concentration of NADH <I>TFH</I> = concentration of the transcription factor bound to NADH. <I>TFN</I> = concentration of the transcription factor bound to NAD. <I>P</I> = Promoter upstream of GFP <I>P'</I> = activated/transcribed promoter <I>tf</I> = fraction of active transcription factor <I>Pf</I> = fraction of active promoter</FONT></TD></TR></TBODY></TABLE><TABLE><TBODY><TR height="25px"><TD width="967px" valign="center" align="center"><FONT face="arial" size="1">
 Comments or questions to the team? Please <A href="mailto:igem@bio.dtu.dk" class="email">Email us</A> -- Comments of questions to webmaster? Please <A href="mailto:lronn@bio.dtu.dk" class="email">Email us</A></FONT></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:DTU_Denmark/modelling">http://2009.igem.org/Team:DTU_Denmark/modelling</A>&quot;</DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU_Denmark/modelling" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU_Denmark/modelling" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU_Denmark/modelling&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU_Denmark/modelling&amp;oldid=163045" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TABLE></DIV></DIV></DIV></BODY></HTML>