"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU Denmark/protocols
From 2009.igem.org
Welcome to the DTU iGEM wiki!
Home
The Team
The Project
Parts submitted
Modelling
Notebook
Activities relating to our two sub-projects:
- The Redoxilator
- The USER assembly standard
- Biobricks
- Protocols
Protocols
Gel electrophoreses. 1
PCR amplification. 2
DNA sequencing. 4
Digestion with restriction enzymes. 5
Ligation. 7
Miniprep for plasmid purification. 7
DNA purification from gel 8
DNA purification from enzymatic reaction. 9
Transformation of chemically competent E. coli 10
High Efficiency Yeast Transformation. 11
 
Gel electrophoreses
Materials and chemicals
Agarose
1X TAE buffer
MiliQ water
Ethidium Bromide 0.625 mg/ml (Amresco)
Mini Sub Cell GT (BioRad)
Hyperladder I (Bioline) (Appendix 6.1.1)
5x DNA Loading Buffer, Blue (Bioline)
Procedure
Preparation of 1% agarose gel For a final volume of 500 ml, 5 g of agarose was dissolved into 500 ml of TAE buffer and mixed with a magnetic stirrer. The agarose was melted in microwave for 5 min. The agarose is completely melted when the solution is totally clear (no veil).
Preparation of DNA Loading buffer was added to DNA. When mixing small amounts this can be done on a piece of parafilm.
Agarose gel electrophoresis Capacity of small wells app.: ~ 15 μl, big wells: ~ 50μl.. Standard 75 mV 400 mA 1 hour / 45 min. Standard amount of hyperladder loaded: 5 μl (4 μl hyperladder 1 μl loading buffer).
Imaging the gel Gel Doc 2000 (Bio Rad). Quantity One 4.0.2 (Bio Rad). The exposure varies.
HyperladderTM I   
HyperladderTM I was used throughout the study as ladder for all gels
PCR amplification
The following recipe was used for amplifying genes used for cloning with the Phusion-polymerase. If more than one PCR-reaction was made, a mastermix containing the polymerase, dNTP, PCR-buffer and water could be made giving more precise concentrations in the mix.
 
It is critical that the DNA Polymerase is the last component added to the PCR mixture, since the enzyme exhibits 3´→5´ exonuclease activity that can degrade primers in the absence of dNTPs.
When running PCR reactions with Taq the PCR buffer is 10x so only 5μl needs to be added
Phusion polymerase
Phusion Polymerase has the highest fidelity of any commercial thermo stable polymerase (50X greater than Taq). This is the reason that this enzyme was used for all cloning procedures described in this report. Taq is mostly used for colony PCR where the purpose is to detect the presence of a certain sequence and not to use the PCR product for further cloning.
T [ºC]
Time
 
 
98
30
sec
 
98
30
sec
Repeat 30 times
57*
30
sec
72
15-60**
sec
72
8
min
 
12
Hold
 
 
 
Table 2: Standard PCR program with Phusion polymerase.
* for primers > 20nt, anneal for 10 – 30 seconds at a Tm +3°C of the lower Tm primer. for primers ≤ 20nt, use an annealing temperature equal to the Tm of the lower Tm primer for calculating of melting temperature see Primer design. **15 sec per kb for low complexity DNA (e.g. plasmid, lambda or BAC DNA). 30 sec per kb for high complexity genomic DNA
Taq polymerase
T [ºC]
Time
 
 
94
2
min
 
94
30
sec
Repeat 30 times
56*
30
sec
72
4**
min
72
8
min
 
12
Hold
 
 
 
Table 3: Standard program with Taq Polymerase.
*Annealing temperatures should be chosen to match the Tm values of the primer pair.
**use 1 minute/kb
Colony PCR
For each colonyPCR to be performed a PCR tube is filled with 35 µl MiliQ Water. A sterile tooth pick which has touched a desired colony is stirred around in the water for the bacteria to stay in the water. Then primers, buffer, dNTP and DNA Polymerase (usually Taq) is added in the same amounts as stated earlier for PCR reactions
A similar program as the before mentioned standard is used, with the modification of starting with 3 minutes at 98º C to get the bacteria to lyse so the DNA is available for the primers to anneal.
T [ºC]
Time
 
 
94
3
min
 
94
30
sec
Repeat 35 times
52
30
sec
72
4
min
72
10
min
 
12
Hold
 
 
 
Table 4: Typical colony PCR program.
Primer design
The annealing temperature (Tm) of a primer is an important value to set in a PCR reaction is by definition the temperature at which one half of the DNA duplex will dissociate to become single stranded. At temperatures lower than the melting temperatures unspecific binding is possible so best PCR results is achieved at an annealing temperature close to the lowest melting temperature of the two primers. This means that primer pairs that have similar melting temperatures will yield a PCR reaction with reduced background noise by unspecific binding of the primers to the template.
Tm can be calculated using the “Oligo analyzer” www.genelink.com/tools/gl-downloads.asp
Or be approximated using: Tm(°C) ≈ 2(NA+NT) + 4(NG+NC)
The primers described in this report is designed using http://fokker.wi.mit.edu/primer3/input.htm , which is a handy tool to find primer pairs to amplify a specific sequence and discards potential primers  that have a tendency to e.g. self anneal. Usually a primer length of 22 was used based on experience. Primers was ordered at www.sigma.com .
DNA sequencing
Preparing and sending DNA
Description from http://www.starseq.com/nomenu.php?nomenu=1&ln=7n8
Mix Each PCR tube contains DNA, MQ H2O and a single sequence primer in a total volume of 6μl. Add 3-5 μl plasmid and 1μl primer – see Table 5.
Label legible, with indelible black pen (e. g. Staedtler permanent lumocolor; Art. Nr. 318-9; EAN 40 07817 304563 or similar). Note your abbreviation on the lids and number them continuously.
Send Protect the reaction containers with a box or something similar and send together with order form in a padded envelope to: StarSEQ, GENterprise GMBH, Johann-Joachim-Becher-Weg 30a, 55099 Mainz, Germany´
Table 5: Relevant sequencing informations
DNA
dissolved in water or TrisHCl (10mM), pH 7-8
Amounts
PCR product: 200 bp: 50 ng, 500 bp: 100 ng; 1 kb: 200 ng
Plasmid DNA
400 ng - 700 ng
Cosmid DNA, PACs, BACs
>1 µg
Primer
10 pmol, i.e. 1 µl of a 10 µMolar primer solution
Melting temperature
52 - 60°C
Optimal length
18 - 25mer
Tube
200 µl PCR tubes with flat lids (e. g. Starlab, Art. Nr. I 1402-8100 or similar); Close only, no Parafilm
Analyzing retrieved files
The files return as SEQ and AB1 files. AB1 files can be opened with the freeware program FinchTV.
Open all files. In each file the sequence that look reliable (normally from base 20 to ~700) is copied and pasted in a txt file.
Assemble the contigs using Vector NTI from Invitrogen
Use the “Assemble” function, and click“Open New Assembly Project”.
Add fragments (txt. files)
Select fragments and assemble.
Digestion with restriction enzymes
Materials and chemicals
MiliQ water
Restriction enzymes 10,000 U/ml
NEBuffer 10X (1,2,3,4)
Bovine serum albumin (BSA) 100X (10X)
DNA
All restriction enzymes, enzyme buffers and bovine serum albumin (BSA) used in this study were purchased through New England Biolabs. Information about specific enzyme conditions e.g. type of NEBuffer and whether BSA was needed or not was obtained through the NEB website: http://www.neb.com/nebecomm/products/category1.asp?#2 .
Initial concentrations of NEBuffers were provided as 10X and diluted to 1X in the final mix. BSA was provided as 10 mg/ml (100X) and should optimally have a final concentration of 100 μg/ml (1X). In this study the original BSA solution had been diluted to 10X which then was used as stock. Enzymes that do not require BSA should not be affected if BSA is present.
Procedure
Short protocol MiliQ H2O was added first, buffer next, BSA when required, then the DNA solution, and finally the enzyme. The reaction mixture was mixed by gently pipetting up and down or by flicking the tube.
Amount of enzyme Generally, 10 U (1 μl) enzyme is normally added to 1 μg of purified DNA in a final volume of 50 μl. Thus, the amount of enzyme depends on the DNA concentration which can be verified by gel electrophoresis. A little more enzyme was added when double digestion was performed with enzymes requiring different types of buffers for optimal activity. E.g. 30 U of AsiSI and 40 U of KpnI was used with NEBuffer 2, because KpnI has got only 75% activity in this buffer and AsiSI had 100%.
Incubation In the 1st hour of incubation most of the DNA will be digested according to NEB, but for most of our reactions a more complete digestion was obtained by incubating “over night”. When this was done new enzymes were added halfway. All reaction were incubated overnight at 37º C.
Enzyme
Recognition site
Used for
KpnI
Cloning
AsiSI
Cloning, Restriction Analysis
ApaLI
Restriction Analysis, linearized BGHA P8 before transformation
NotI
Restriction Analysis
Table 6: Restriction endonucelases used in this study.
More information
NEB restriction enzymes:
http://www.neb.com/nebecomm/products/faqCategory1.asp#661
 
NEBuffers:
http://www.neb.com/nebecomm/products/productB7000.asp
Ligation
 
1 µl ligase
 
4 µl 10x ligase buffer
 
X µl plasmid, cut
X = 50-100 ng plasmid
Y µl PCR product, cut
Y = 300 ng PCR product
Z µl MQ up to 40 µl total
 
 
Everything is mixed, and incubated – either at room temperature for 2 hours, or overnight using the idea of Lund et al (26) of incubating on PCR machine with the following PCR program
 
 
T [ºC]
Time
 
 
10
30
sec
 
30
15
sec
Repeat 99 times
30
15
sec
10
30
sec
10
30
sec
 
16
Hold
 
 
 
Table 7: Ligation program on PCR machine
After the incubation, the mix can be stored at -18˚C. Lund et al (26) showed that ligation with cohesive ends carried out on with the temperature shifts could give an increased ligation yield of 7.7 times. This is because ligase works best at 30º C but template annealing is most effective at 10º C.
Miniprep for plasmid purification
(Sigma GenEluteTM Plasmid Miniprep Kit, from manual)
1.       Harvest & lyse bacteria
a.        Pellet cells from 1 – 5 ml overnight culture 1 min (1 ml from TB or 2xYT; 1-5 ml from LB medium). Discard supernatant.
b.       Resuspend cells in 200 µl Resuspension Solution. Pipet up and down or vortex.
c.        Add 200 µl of Lysis Solution. Invert gently to mix. Do not vortwx. Allow to clear for ≤ 5 min.
2.       Prepare cleared lysate
a.        Add 350 µl of Neutralization Solution (S3). Invert 4- 6 times to mix.
b.       Pellet debris 10 min. at max speed.
3.       Prepare binding column
a.        Add 500 µl Column Preparation Solution to binding column in a collection tube.
b.       Spin at ≥ 12.000 x g, 1 min. Discard flow-through.
4.       Bind plasmid DNA to column
a.        Transfer cleared lysate into binding column.
b.       Spin 30” – 1 min. Discard flow-through.
5.       Wash to remove contaminants
a.        Optional (EndA+ strains only): Add 500 µl Optional Wash Solution to column. Spin 30” – 1 min. Discard flow-through.
b.       Spin 1 min to dry column
6.       Elute purified plasmid DNA
a.        Transfer column to new collection tube.
b.       Add 100 µl Elution Solution. Spin 1min.
DNA purification from gel
Using illustraTM GFX PCR DNA and Gel Band Purification Kit. Protocol from manual has been slightly altered by eluting using MiliQ Water and not elution buffer.
1. Sample Capture
a. Weigh a DNase-free 1.5 ml micro centrifuge tube and record the weight.
b. Using a clean scalpel, long wavelength (365 nm) ultraviolet light and minimal exposure
time, cut out as small an agarose band as possible containing the sample of interest.
Place agarose gel band into a DNase-free 1.5 ml microcentrifuge tube.
c. Weigh the microcentrifuge tube plus agarose band and calculate the weight of the agarose
slice.
d. Add 10 μl Capture buffer type 2 for each 10 mg of gel slice.
e. Mix by inversion and incubate at 60°C until the agarose is completely dissolved. Mix by
inversion every 3 minutes.
f. For each purification that is to be performed, place one GFX MicroSpin column into one Collection tube.
2. Sample Binding
a. Centrifuge Capture buffer type 2- sample mix briefly to collect the liquid at the bottom of the tube.
b. Transfer 600 μl Capture buffer type 2- sample mix onto the assembled GFX MicroSpin column
and Collection tube.
c. Incubate at room temperature for 1 minute.
d. Spin the assembled column and Collection tube at 16 000 × g for 30 seconds.
e. Discard the flow through by emptying the Collection tube. Place the GFX MicroSpin
column back inside the Collection tube.
f. Repeat Sample Binding steps b. to e. as necessary until all sample is loaded.
3. Wash & Dry
a. Add 500 μl Wash buffer type 1 to the GFX MicroSpin column.
b. Spin the assembled column and Collection tube at 16 000 × g for 30 seconds.
c. Discard the Collection tube and transfer the GFX MicroSpin column to a fresh DNase-free 1.5 ml
microcentrifuge tube (supplied by user).
4. Elution
a. Add 10–50 μl Mq water to the center of the membrane in the assembled GFX MicroSpin column and sample Collection tube.
b. Incubate the assembled GFX MicroSpin column and sample Collection tube at room temperature for 1 minute.
c. Spin the assembled column and sample Collection tube at 16 000 × g for 1 minute to recover the purified DNA.
d. Proceed to downstream application. Store the purified DNA at -20°C.
DNA purification from enzymatic reaction
Using illustraTM GFX PCR DNA and Gel Band Purification Kit. Protocol from manual has been slightly altered by eluting using MiliQ Water and not elution buffer.
1. Sample Capture
a. Add 500 μl Capture buffer type 2 to up to 100 μl sample.
b. Mix thoroughly.
c. For each purification that is to be performed, place one GFX MicroSpin column into one Collection tube.
2. Sample Binding
a. Centrifuge Capture buffer type 2-sample mix briefly to collect the liquid at the bottom of the tube.
b. Load the Capture buffer type 2-sample mix onto the assembled GFX MicroSpin column and Collection tube.
c. Spin the assembled column and Collection tube at 16 000 × g for 30 seconds.
d. Discard the flow through by emptying the Collection tube. Place the GFX MicroSpin column back inside the Collection tube.
3. Wash & Dry
a. Add 500 μl Wash buffer type 1 to the GFX MicroSpin column.
b. Spin the assembled column and Collection tube at 16 000 × g for 30 seconds.
c. Discard the Collection tube and transfer the GFX MicroSpin column to a fresh DNase-free 1.5 ml microcentrifuge tube (supplied by user).
4. Elution
a. Add 10–50 μl Mq-water to the center of the membrane in the assembled GFX MicroSpin column and sample Collection tube.
b. Incubate the assembled GFX MicroSpin column and sample Collection tube at room temperature for 1 minute.
c. Spin the assembled column and sample Collection tube at 16 000 × g for 1 minute to recover the purified DNA.
d. Proceed to downstream application. Store the purified DNA at -20°C.
Transformation of chemically competent E. coli
Defrost the frozen cells on ice. Use 50μl cells pr transformation for transformation of plasmids and 100 μl of cells for transformations of ligation mix.
Add an appropriate volume of plasmid [1] (often 1μl) to 50μl cell suspension. If transforming with a ligation mix add half of the ligation mix (5-10 μl ligation mix) to the 100μl cells.
Keep on ice for 30 minutes
Heat chock for 90 seconds at 42°C.
Transfer quickly to ice and keep them for 5 minutes.
Plate 100 μl cell suspension on selective medium (LB-amp) and incubate at 37°C O/N. Store the rest of the transformed cells in the fridge.
Between 5 and 6 the following can be done:
Add 1ml LB medium. Use 0,5 ml for ligation mixtures. Mix.
Incubate samples for 1 hour at 37°C.
High Efficiency Yeast Transformation
DAY 1
Inoculate the yeast strain in 2-5ml of liquid YPD and incubate O/N at 30°C on a rotary shaker. Leave a 50 mL shakeflask in the incubater with YPD, so it will be 30 C in the morning.
In the morning, inoculate a shake flask containing 50ml YPD with the 2-5ml cultures. Incubate 3-5 h at 30°C on a rotary shaker. (Option: take 1 mL of o/n culture and leave it for 5 hours)
NB: For a transformation with a basic plasmid, the O/N culture is plenty.
DAY 2
Harvest the cells by centrifugation at 3000g for 5 min (for the centrifuge in 220 this is equal to 4220 rpm). You need the rotor F-16 from the first floor that can take falcon tubes. The rotor code should be 30.
Wash the cells in 25ml sterile water. Centrifuge at 3000g for 5 min and resuspend cells in 1ml water.
Boil a 1.0 ml sample of carrier DNA for 5 min and chill in an ice/water bath while harvesting the cells. It is not necessary to boil the carrier DNA every time. Keep the aliquot in freezer and boil again after 3-4 freeze-thaws. (They can usually be found in lab 015)
Transfer the cell suspension to a 1.5ml micro centrifuge tube and centrifuge for 30s. Remove the supernatant with a micropipette.
Resuspend the cells to a final volume of 1ml with water and vortex vigorously to resuspend the cells. Put the cells on ice.
Pipette 100µl samples (108 cells) into 1.5ml micro centrifuge tubes, one for each transformation and one extra for each culture for SC plating. Centrifuge at top speed for 30s and remove the supernatant with a micropipette. (NB: Remember to set water baths at 42).
Make sufficient Transformation Mix corresponding to the number of transformations that need to be performed (count 1 extra). Keep the mix on ice/water.
 All volumes are mentioned in µl
(for a deletion you will need approx 300-500 ng of each fragment)
 
·         Place Xµl of plasmid on top of the cells. Add (360-X)µl of Mix to each transformation tube and resuspend the cells by vortex mixing vigorously.
 
·         Incubate the tubes at 42°C for 40 min
 
·         Micro centrifuge at top speed for 30s and remove the transformation mix with a pipette.
 
·         Redissolve in 200µl of sterile water into each tube. Stir the pellet by pipetting and vortex.
 
·         For transformation with an integrative plasmid, linear construct or nucleotide, plate 100µl onto 2 selective plates.
 
Incubate the plates at 30°C for 2-3 days.
 
 
[1] Approximately 50ng should be added if transforming with a plasmid.
Work process
Our team works parallel in smaller sub-teams. Some of us work hard in the lab, while others are in the process of developing software and the in silico model of the Redoxilator system. However, we constantly keep each other updated, and meet often to exchange ideas and take turns at the different tasks, thus exhausting all of our combined knowledge in every aspect of this project.
Comments or questions to the team? Please Email us -- Comments of questions to webmaster? Please Email us
Retrieved from " http://2009.igem.org/Team:DTU_Denmark/protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
