"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU Denmark/genetic design
From 2009.igem.org
Theoretical background
Home
The Team
The Project
Parts submitted
Modelling
Notebook
The redoxilator
- Genetic design
- Applications and perspectives
- Safety considerations
The USER assembly standard
- USER fusion of biobricks
USER fusion primer design software
- Abstract
- Instructions
- Output format
Experimental results
- Results and discussion
The project
Genetic design
We have designed and physically constructed a genetic system that couples the intracellular NAD+/NADH level to the gene expression of a reporter protein. The system has potentially many applications including in vivo online monitoring of the redox poise, production optimization and cancer research with yeast as a model organism (see Applications).
The NAD+/NADH ratio is sensed by a system originating in Streptomyces coellicolor. In S. coellicolor the protein REX is a repressor and controls the gene expression of multiple genes by recognizing and binding to a specific DNA-sequence termed ROP (Rex operator). NAD+ and NADH compete to associate with Rex, but only a REX:NAD+ association can bind the ROP DNA-sequence (Brekasis and Paget, 2003).
In S. coellicolor REX DNA binding represses expression of target genes, by physically hindering RNA-polymerases from binding the promoter. As the transcription machinery of eukaryotes is different and more complicated, there are no guarantee that repression will be effective in eukaryotes. REX has therefore been fused to an eukaryotic transcriptional activator, a widely used technique applied for the investigation of the GAL proteins and other systems (Sadowski et al. 1988). The REX-activator fusion-protein is able to bind the ROB sequence placed upstream of a minimal eukaryotic promoter that only supports transcription upon activation. A certain NAD+/NADH ratio will activate the Redoxilator to recognize the ROB promoter, resulting in transcription of the reporter gene.
The redox coupled system
The redox coupled system
Gene design and redox regulation
A: The Rex gene has been fused to an activator domain and is transcribed constitutively, leading to constant concentration of the Rex-activator protein in the cell. The ROB sequence and a minimal promoter is followed by a reporter gene, which is only transcribed when the REX-activator fusion protein is bound to the promoter. B: The REX-activator only binds the ROB DNA sequence under the condition of having NAD+ bound. Under these circumstances the fused activator domain summons the RNA polymerase and the reporter gene is transcribed.
Design specifications
The genetic system consists of two synthetic genes: one coding for the REX-activator fusion protein, and one coding for a yeast optimized GFP gene under control of ROB fused to a minimal promoter. With this system GFP is only be expressed when REX-Activator is bound to ROB, which occurs at high NAD+/NADH levels.
Simplified schematic representation of the synthetic genetic system on the DNA-level. The individual parts are described below.
The genetic elements and the requirements they need to fulfill are listed in the following table. The detailed description of the used genetic elements will not be made publicly available due to IP rights. The elements have been selected solely on their properties, and are from a variety of organisms - several of them are biobricks. All elements are codon optimized for S. cerevisiae
The genetic elements are from a variety of organisms.
Modelling
To guide the design process and to simulate how our genetic device would operate in vivo, we performed thorough mathematical modelling of the relation between the input and output of our system (the detailed model is available here ). The model predicts how the GFP-concentration in a transformed cell relates to the internal NAD+/NADH level (indicative of the redox state).
The model takes the following into account:
i) transcription factor activation,
ii) promoter activation,
iii) synthesis and degradation of mRNA and finally,
iv) synthesis, dilution and degradation of GFP.
The model provided understanding of the influence of the various parameters on the behavior of the system. The most important discovery, was the fact that due to the relatively slow degradation of GFP, the system would have a slow response time: a change in NAD+/NADH level would not be reflected fast enough for the system to be efficient.
The solution to our problem was to bring down the half-life of the GFP. This was done by the fusion of GFP and a PEST degradation signal from the yeast protein Cln2, which has been demonstrated to reduce the half-life from 7 hours to 30 minutes (Mateus and Avery, 2000). The fast degradable GFP-Cln2 has been created as a biobrick and submitted to the registry ( available here ).
Design and submission of a biobrick allowing fast degradation of any protein
The possibility of making a fast degradable version of specific proteins is essential when engineering biological systems by a synthetic biology approach. To bring this possibility to other registry users, we have designed a biobrick that allows fast degradation of any protein in S. cerevisiae, in a simple and easy way: the biobricks are assembled by any standard that allows protein-fusions.
Description and requirements of the genetic elements
constituting the Redoxilator device.
Genetic element
Required function
Constitutive promoter
Constitutive expression of   REX-Activator fusion protein
Kozak sequence
Ribosome start-codon   recognition and enhanced initiating of translation
REX
REX (redox regulator) that binds to   ROP at high NAD+/NADH ratio
Linker
To couple two protein   domains without disrupting their individual functions
Activator domain
Protein domain able to   activate transcription in eukaryotes in proximity of a minimal promoter.
Nuclear Localization   Sequence (NLS)
Translocation of the   REX-Activator protein to the nucleus
Terminator 1
Sequence that terminates   transcription
 
 
ROB
DNA sequence that REX binds   at high NAD+/NADH ratio
Minimal promoter
Promoter devoid of   regulatory motifs. Only expression if an activator is bound upstream.
GFP
The amount of green   fluorescent protein can be quantitatively measured.
Degradation signal
Ensures fast degradation of   GFP
Terminator 2
Terminates transcription.   Different from terminator 1 to avoid direct repeats, which can cause the DNA to loop out.
References
[Brekasis and Paget, 2003] Brekasis, D. and Paget, M. S. B. (2003). A novel sensor of NADH/NAD+ redox poise in streptomyces coelicolor a3(2). EMBO J, 22(18):4856–4865.
[Mateus and Avery, 2000] Mateus, C. and Avery, S. V. (2000). Destabilized green fluorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry. Yeast, 16(14):1313–1323.
[Sadowski et  al., 1988] Sadowski, I., Ma, J., Triezenberg, S., and Ptashne, M. (1988). Gal4-vp16 is an unusually potent transcriptional activator. Nature, 335(6190):563–564.
Why yeast?
The utilization of improved micro-organisms for industrial processes is a fact for centuries. From the early stages in the preparation of fermented food and beverages until nowadays. Recent advances in biochemistry, engineering and genetic manipulation techniques, led scientist and engineers to improve micro-organisms in order to enhance their metabolic capabilities for biotechnological applications. Along with these improvements, a far more rational and direct approach to strain improvement have been employed, of what we call Metabolic Engineering. What distinguishes Metabolic Engineering from the classical approaches is the application of advanced analytical tools for identification of suited targets for genetic modifications or even the use of mathematical models to perform in silico design of optimized micro-organisms. The consequences of the changes introduced in these engineered strains can then suggest further modifications to improve cellular performance and therefore Metabolic Engineering can be seen as a cyclic process made of continuous iterations between experimental and analytical work.
Among all possible micro-organisms, Saccharomyces cerevisiae is a very well-suited candidate since it is recognized as being GRAS (“generally regarded as safe”).
Due to its long history of application in the production of consumable products such as ethanol and baker’s yeast, Saccharomyces cerevisiae has a very well-established fermentation and process technology for large-scale production. The availability of its complete genome sequence of in 1996, and the numerous possibilities for genetic modifications by recombinant DNA technology that came with that, made of yeast a perfect model organism within the field of biotechnology.
Comments or questions to the team? Please Email us -- Comments of questions to webmaster? Please Email us
Retrieved from " http://2009.igem.org/Team:DTU_Denmark/genetic_design "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
