"
Page
Discussion
View source
History
teams
Log in
 
Team:DTU Denmark/results
From 2009.igem.org
The project
Home
The Team
The Project
Parts submitted
Modelling
Notebook
The redoxilator
- Genetic design
- Applications and perspectives
- Safety considerations
The USER assembly standard
- USER fusion of biobricks
USER fusion primer design software
- Abstract
- Instructions
- Output format
Experimental results
- Results and discussion
The project
Results
In this section the results of our experimental work is described. In short 4 biobricks are made. One of the biobricks are amplified by PCR and USER-fused to a TEF promoter to demonstrate that the biobrick works. The Redoxilator construct which we designed is transformed into yeast together with a copy to check if our minimal promoter is leaky. In total 5 strains are characterized in a 40 hour fermentation showing growth and GFP fluorescence in 4 different carbon sources.
Construction of biobrick BBa_K194000 , BBa_K194001 and BBa_K194002
To prepare our biobricks to be sent to the registry the plasmid pSB1A2 from the iGEM collection kit was digesting with XbaI and SpeI. This linearized vector was used as backbone for insertions of all of our 4 biobricks amplified by PCR and inserted. (see "parts submitted" and a more through description of the experimental procedure to obtain these at the notebook ). Following insertion the vectors were digested with XbaI and SpeI to make sure the inserts had integrated correctly, and subsequent PCR on the biobrick to make sure the right fragment had been inserted
Construction of biobrick BBa_K194003 and yeast plasmid containing biobrick BBa_K194003 for demonstration.
Two complentary oligoes were ordered so that they could anneal forming the biobrick BBa_K194003 holding a PacI restriction site flanked by two Nt.BbvCI-sites and with XbaI and SpeI sites in the end for upholding the biobrick standard.
The USER cassette consisting of two complementary synthetic oligoes.
The two primers were warmed up to 95 °C on PCR machine and slowly cooled down with 5°C every 5 minutes until it reached 25°C for the two primers to anneal correctly. Then the double stranded oligo was digested with XbaI and SpeI o/n (over night). In the morning the oligo was warmed up to 95 °C again and cooled down as former described. This method was chosen to denature the restriction enzyme before ligation as the oligoes would have been lost in a standard DNA purification.  The oligo was inserted into two different plasmids:
1. pRS416 : in-house yeast plasmid with URA marker that we wished to use to demonstrate that Biobrick BBa_K194002 worked in yeast.
2. pSB1A2 : a iGEM plasmid what would be used to send the biobrick to MIT as we cannot send pRS416 with the biobrick due to intellectual property issues.
The two plasmids were digested seperately using two enzymes, XbaI (20U) and SpeI (10U). The mixtures were incubated at 37 °C for eight hours after which more enzyme was added to ensure maximum digestion. After an additional two hours of incubation the digested plasmids were dephosphorilised to prevent reanealing during ligation. This was achieved by an Antartic Phosphatase treatment (25 U). The DNA was gel-purified and the resulting concentration was determined using fluorometrics ( The Qubit Quantitation PlatformTM from invitrogen). The ligation was performed using T4-DNA ligase using a 3:1 ratio of fragment to vector and 9 ng DNA in total at a volume of 20 µl according to the manufactors recommendations. The actual ligation was carried out on a PCR machine for 1½ hours shifting the temperature from 10 °C to 30 °C at 30 second intervals as desribed by [1]. The final product was finally chemically transformed into competent E coli cells.
Selection of transformants
Ten random colonies were picked for overnight growth in liquid media. The next day the plasmids were purified and digestion were performed with PacI and XbaI to verify the presence of the biobrick. This was done as the presence of the biobrick is introduced a PacI site in the vector. The XbaI digestion was done to ensure that the biobrick had been inserted with the right directionality.
Figure 1. Left: PacI restriction assay, BBa_K194003 insertion into pRS416. Restriction performed on ten randomly picked, o/n grown and plasmid purified samples. Samples are run on a 1 % agarose gel. Digestion should leave a single band at approxiamately 6 kb. Colonies 5 and 9 are believed to contain the correct plasmid. Right: The ladder used in the corresponding experiment, Hypper Ladder 1.
Demonstration of BBa_K194003
From the gel it was seen that the BBa_K194003 had been inserted correctly into pRS416. Now this was used to demonstrate that biobrick BBa_K194002 was functional. This was done by opening the USER cassette in BBa_K194003 with PacI and Nt.BbvCI, purifying the backbone from gel and adding two PCR fragments:
1. The constitutive TEF promoter (400bp) using the primers BGHA200/370
2. Biobrick BBa_K194002 (1500bp) using the primers BGHA371/372
Figure 2. The cloning strategy for demonstrating that biobricks BBa_K194002 works. Two PCR products were fused using USER fusion and inserted into the pRS416 backbone by USER cloning.
After treatment with USERTM mix, the mixture was transformed into E. coli.
Verification
Following selection of transformants and overnight liquid cultures the plasmids was purified . The plasmids were linearized using AsiSI, which is also included in the BBa_K194003 . Plasmids holding the fusion of TEF and BBa_K194002 will show to be 2kb longer than the ones without.  The presence of the gene were only verified by PCR of the genes inserted.
Extraction of p241i and cloning
500 µl Eppendorf tubes were pierced at the buttom and sterilized. The filter containing our construct (also referred to as p241i) was obtained from DNA2.0 and stuffed into the 500 µl Eppendorf tube that again was put into a 1.5 ml Eppendorf tube. 100 µl MilliQ H2O was added and the filter was soaked for 2 minutes. The Eppendorf tubes were centrifuged at 13.000 g for 2 minutes. The eluation was used to chemically transforme E. coli.
Ten random colonies were picked for overnight growth in liquid media. The next day the plasmids were purified and digestion was performed with SnaBI, BsaI & BamHI to make sure that this was the plasmid we had ordered. The gel showed that this was the case.
Creation of the Leaky construct
As desided by our genetic design the redoxilator can be removed from p241i by NotI digestion and subsequent gel-purification of the backbone. Following religation of the backbone the DNA was transformed into E. coli. This plasmid was purified and digested with XmaI and run on gel to check whether the expected 2500 bp & 4500 bp bands appeared. The gel gave ambiguous results so a test PCR was done on the purified plasmid to check whether the redoxilator had been removed. This test showed that this was the case.
Transformation into yeast
Yeast (CEN PK 113 5D (Ura-)) was transformed with the pRS416c-plasmid using the Ura marker present on pRS416 Yeast was also transformed with pSC011 kindly provided by Simon Carlsen, CMB containing the sequence from BBa_K194001 under the constitutive expression of the TEF promoter. pSC011 holds the same promoter and terminator as the pRS416 holding the BBa_K194003 and can therefore be used to analyze the difference of the addition of the Cln2 degradation signal.
Also the construct (p241i) holding the redoxilator and the leaky construct was transformed into CEN PK 113 5D
BioLector fermentation
To characterize the yeast strains that we have constructed in the project we used the BioLector machine kindly provided by the Center for Biological Sequence Analysis at DTU. The BioLector is able to keep the microtiterplate stirring and measures OD and GFP levels every 3rd minut making it ideal for our experimentt. Our main goal with this experiment was to determine if there was a difference in GFP signal for different carbon sources and this way prove if the GFP signal is anyway dependent on the NADH/NAD+ ratio in the cell or not
We characterized:
1. Wildtype as background: CEN PK 113 7D without any auxotrophic marker was chosen. If there is GFP fluorescence of the wildtype this needs to be accounted for.
2. Our Redoxilater construct : This construct holds the redoxilator sensor controlling the expression of BBa_K194002 . This plasmid was transformed into CEN PK 113 5D (Ura-).
3. Leaky construct: To check whether the redoxilator actually induces expression of BBa_K194002 or our minimal promoter is simply leaky. This plasmid was transformed into CEN PK 113 5D (Ura-).
4. pSC011: Demonstration of BBa_K194001 . This plasmid was transformed into CEN PK 113 5D (Ura-).
5. pRS416c: Demonstration of BBa_K194002 . This plasmid was transformed into CEN PK 113 5D (Ura-).
To minimize the background fluorescence from the media, synthetic complete without riboflavin and folic acid was used since they have been reported to be fluorescent [2].
The cells were grown in this media overnight to reach an OD high enough for inoculation of the microtiter plate for the BioLector.  The cells were inoculated in the mirotiterplate in 1 ml media with an OD of 0.15 with the following carbon sources
1.glucose (20 g/l)
2.ethanol (15,7 g/l  = same cmol as glucose)
3.galactose (20 g/l = same cmol as glucose)
4.glucose (10 g/l) + galactose (10 g/l) – so we should see an diauxic shift over the course of the experiment
Growth rates (h-1) obtained for the two strains on diferent carbon sources
Carbon Source
Leaky
Redoxilator
Glucose
0,114 ± 0,006
0,112 ± 0,003
Galactose
0,084 ± 0,010
0,076 ± 0,003
Ethanol
0,013 ± 0,005
0,007 ± 0,001
As expected, both strains showed different profiles of growth accordingly to the substrate. For Glucose and Galactose, the growth curve showed two exponential phases, the first one corresponding to the growth on Glucose and the second corresponding to the growth on Galactose.  Although the initial OD was similar in both cases, the growth on Galactose showed an even longer lag phase than the glucose. Finally, as expected, the substrate reporting less growth was ethanol. For the Redoxilator, the growth solely on Glucose showed a longer lag phase compared to the mixed substrate, probably due to a lower initial OD.
Growth profile (OD) of the Redoxilator for the different carbon sources
Growth profile (OD) of the Leaky for the different carbon sources
Regarding the GFP signal, all substrates showed different profiles. Nevertheless, the fermentative substrates, Glucose and Galactose, showed higher values of GFP than the respirative substrate, Ethanol.
GFP signal of the Redoxilator for the different carbon sources
GFP signal of the Leaky for the different carbon sources
Although these results are accordingly to what was expected, it is important to mention that these cultures didn’t have comparable growths, so, in order to subtract this variability of the growth, one should normalize the GFP expression with OD. Additionally, to the GFP values, the residual values for each case were subtracted.
GFP signal corrected from the residual signal, normalized with growth of the Redoxilator for the different carbon sources
GFP signal corrected from the residual signal, normalized with growth of the Leaky for the different carbon sources
One other aspect that is worth instigating is the variation of the GFP signal with growth. In other words, if growth is correlated with GFP signal.
Variation of the corrected GFP signal with growth of the Redoxilator for the different carbon sources
Variation of the corrected GFP signal with growth of the Leaky Redoxilator for the different carbon sources
Results of the fermentations and demonstration of biobrick BBa_K194001 and BBa_K194002
Growth rates (h-1) obtained for the two strains containing the two biobricks BBa_K194002 (GFP_Cln) and BBa_K194001 (GFP) with glucose as Carbon source
Strain
Growth Rate
GFP_Cln
0,080 ± 0,003
GFP
0,129 ± 0,001
The growth profile of these two strains in Glucose and in a mixed carbon source of Glucose and Galactose is presented below.
Growth profile of the yeast strain tranformed with the degradable GFP
Growth profile of the yeast strain tranformed with the non-degradable GFP
both strains show very similar behavior. With Glucose as the solely carbon source, the curve presents a distinct exponential phase corresponding to the consumption of this substrate. After Glucose is consumed, OD seems to drop. Although there is not a decrease in cells, this drop might be due to morphological changes in the cells typical of a shift in the metabolism. As for the curves representing the mixed carbon, there are two exponential phases, the first corresponding to the glucose consumption and the second to the Galactose.
The next two graphs represent the variation of the GFP signal corrected and normalized with OD to subtract any variability that could come from the differences of growth
GFP signal, corrected and normalised for the degradable GFP
GFP signal, corrected and normalised for the non-degradable GFP
As one can see, there is a significant difference between this two constructs in terms of GFP signal. As expected, the signal from the degradable GFP is much lower than the signal showed by the non-degradable GFP, as the GFP that is not being degraded is accumulating and giving a stronger signal.
This results go in line with the results published by Avery and Mateus [3] that state that the degradable GFP has a 14 times shorter half-life than the non-degradable.
Therefore, based on this results we demontrate that the biobrick BBa_K194002 submited, presents a good option as a fast degradable GFP for yeast. We show that both biobricks codes for Green Fluorescent Protein and that the addition of the sequence from Cln2 results in a strain with lower levels of GFP when the genes are expressed from the same promoter in plasmids with same copy number in the same strain
References
[1] Lund, A. H., M. Duch, and F. S. Pedersen. 1996. Increased cloning efficiency by temperature-cycle ligation. Nucleic Acids Res. 24:800-801. doi:l50400
[2] Mark A. Sheff and Kurt S. Thorn. 2004. Optimized cassettes for fluorescent protein tagging in Saccharomyces cerevisiae. Yeast 24:800-801. doi:10.1002/yea.1130
[3] Mateus, C, Avery, S V 2000. Destabilized green fuorescent protein for monitoring dynamic changes in yeast gene expression with flow cytometry . Yeast 16:1313-1323.
Achievements
Redox sensing device
Two novel genes have been designed and synthesized each comprised of 5 genetic elements. Together they function as a device termed the Redoxilator that can sense the internal redox state of a yeast cell, and output a reporter signal.
Biobricks
DNA of several new biobricks have been designed and submitted including a yeast optimized GFP reporter protein, a protein degradation sequence and a fast degradable yeast GFP. (Bronze medal)
We have demonstrated that our USER fusion biobrick works as expected and documented it (silver medal)
USER fusion Assembly standard
A new biobrick assembly standard that allows the rapid construction of multi-part devices have been developed and documented. The assembly standard offers many benefits: All restriction sites are allowed, multiple biobricks can be joined in one step, the result is scar-free making it ideal for protein fusions and more. (Gold medal)
USER-fusion primer design software
A novel and very useful software tool have been developed that can automatically design the optimal primers for USER fusion assembly of 2-9 biobricks, taking several parameters into account.
Comments or questions to the team? Please Email us -- Comments of questions to webmaster? Please Email us
Retrieved from " http://2009.igem.org/Team:DTU_Denmark/results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
