<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_DTU_Denmark_theory_private_securkey_hjg1mab2ak47"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:DTU Denmark/theory private securkey=hjg1mab2ak47</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><P><TITLE>Theoretical background</TITLE></P><TABLE class="links"><TBODY><TR><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark">Home</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/team">The Team</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/project">The Project</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/parts">Parts submitted</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/modelling">Modelling</A></FONT></TD><TD align="center"><FONT face="arial" size="3"><A class="mainLinks" href="https://2009.igem.org/Team:DTU_Denmark/notebook" title="Day to day lab activity">Notebook</A></FONT></TD></TR></TBODY></TABLE><TABLE><TABLE cellpadding="10px"><TBODY><TR><TD width="163px" height="100%" valign="top"><FONT color="#990000" face="arial" size="3"> 
   The redoxilator</FONT><FONT color="" face="arial" size="2"><A href="https://2009.igem.org/Team:DTU_Denmark/theory" class="leftbar">- Introduction</A><A href="https://2009.igem.org/Team:DTU_Denmark/yeast" class="leftbar">- Results</A><A href="https://2009.igem.org/Team:DTU_Denmark/practicalapproach" class="leftbar">- Applications and perspectives</A></FONT><FONT color="#990000" face="arial" size="3">The USER<SUP>TM</SUP> assembly standard</FONT><FONT face="arial" size="2"><A href="https://2009.igem.org/Team:DTU_Denmark/USERprinciple" class="leftbar">- Principle</A><A href="https://2009.igem.org/Team:DTU_Denmark/USERconcept" class="leftbar">- Proof of concept</A><A href="https://2009.igem.org/Team:DTU_Denmark/USERmanual" class="leftbar">- Manual</A></FONT><FONT color="#990000" face="arial" size="3">USER<SUP>TM</SUP> fusion primer design software</FONT><FONT face="arial" size="2"><A href="https://2009.igem.org/Team:DTU_Denmark/USERprogram" class="leftbar">- Abstract</A><A href="https://2009.igem.org/Team:DTU_Denmark/USERprograminstructions" class="leftbar">- Instructions</A><A href="https://2009.igem.org/Team:DTU_Denmark/USERprogramoutputformat" class="leftbar">- Output format</A></FONT></TD><TD width="556px" height="100%" valign="top"><FONT color="#990000" face="arial" size="5"><B>The project</B></FONT><FONT color="#333333" face="arial" size="2.5"><FONT size="4"><B>Theoretical background</B></FONT><P align="justify">The NAD<SUP>+</SUP>/NADH ratio sensor-protein Rex (<U>Re</U>do<U>x</U> regulator) has been discovered in the bacterium Streptomyces
coelicolor. In its host organism, the sensor works as a repressor and controls the gene expression of a large number of genes by recognizing and binding to a specific DNA-sequence termed ROP (<U>R</U>ex <U>OP</U>erator). NAD<SUP>+</SUP> and NADH compete for Rex binding, and the protein binds the ROP DNA-sequence only when NAD<SUP>+</SUP> is bound.</P><FONT size="3"><B>Our synthetic biology project: The Redoxilator</B></FONT><P align="justify">To achieve a system that senses changing levels in the NAD<SUP>+</SUP>/NADH ratio in the eukaryote <I>S. cerevisiae</I>, the gene encoding the Rex protein will be fused to a yeast activator domain, resulting in a new synthetic protein: the Redoxilator. The ROP sequence - the DNA binding site Rex can bind to - will be inserted into a yeast promoter, resulting in a promoter activated by the Redoxilator.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The redox coupled system</DIV></DIV></DIV></DIV><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The redox coupled system</DIV></DIV></DIV></DIV><P align="justify"><I><B>Figure 1 - Gene design and redox regulation</B><B>A:</B> The Rex gene will be fused to an activator domain forming the Rexivator and will be transcribed constitutively leading to constant concentration of the sensor in the cell. The ROB sequence will be inserted into a specific promoter followed by a reporter gene, which will only be transcribed if the Rexivator complex is bound to the promoter. <B>B:</B> The Rexivator only binds the ROB DNA sequence under the condition of having NAD+ bound. Under these circumstances the fused activator domain summons the RNA polymerase and the reporter gene will be transcribed</I></P><P align="justify">A certain NAD<SUP>+</SUP>/NADH ratio will activate the Redoxilator to recognize the ROB promoter resulting in transcription of a downstream gene. In this way the ROB promoter and the Redoxilator comprises the complete sensing system. The system can be coupled to the expression of virtually any gene of interest; making transcription solely dependent on the ratio of NAD<SUP>+</SUP>/NADH in the cell. In our iGEM project, the system will be used for two selected applications considered highly relevant: i) in vivo monitoring of NAD<SUP>+</SUP>/NADH in yeast, and ii) NAD<SUP>+</SUP>/NADH ratio regulated production of yeast products in chemostat processes.</P><B><I>i) Reporter gene expression regulated by the Rexivator – an in vivo redox sensor</I></B><P align="justify">The gene encoding green fluorescent protein (GFP) is widely used as a reporter gene in molecular biology. By placing the ROB promoter upstream of a GFP gene on a plasmid, and transforming the whole system into a yeast cell, GFP will be expressed at certain NAD<SUP>+</SUP>/NADH levels. When the Rexivator is bound to DNA, GFP expression will produce a visible and quantitatively measurable signal, which will be an indirect measure of the NAD<SUP>+</SUP>/NADH ratio.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The redox coupled system</DIV></DIV></DIV></DIV><P align="justify"><I><B>Figure 2 – Schematic overview of overall approach.</B> 
After the design, synthesis and transformation of the NAD<SUP>+</SUP>/NADH sensor, online measurement of reporter gene expression will be measured and oscillative behaviour of the productivity will be evaluated and used for further optimization.</I></P><P align="justify">The measuring of cellular NAD<SUP>+</SUP>/NADH levels is usually a difficult process, especially due to fast changes in NAD<SUP>+</SUP>/NADH ratio that can occur when a sample is taken and exposed to slightly new conditions. With this sensor system, the quantitative measurement of the reporter gene will provide a fast and reliable way to determine the NAD<SUP>+</SUP>/NADH ratio in vivo. The plasmid can be transformed into a yeast strain allowing the NAD<SUP>+</SUP>/NADH ratio to be continuously monitored. As an example this plasmid can be used to study whether an engineered yeast strain has an altered NAD<SUP>+</SUP>/NADH ratio. The plasmid can be transformed into different yeast strains (e.g. wild type versus an engineered strain or two production strains) and the NAD<SUP>+</SUP>/NADH ratio can be compared.</P><B><I>ii) Product formation regulated by the Rexivator – an attempt to improve and prolong chemostat processes</I></B><P align="justify">When S. cerevisiae are grown continuously in a chemostat, the productivity of e.g. antibiotic or protein product gradually decreases [Personal correspondence with Novo Nordisk A/S]. This occurs during maximum production and is believed to be the result of metabolic adaption8 with reduced product formation as a consequence. The metabolic adaption is believed to occur because the cells are stressed by the extensive production. The cells will adapt to the new metabolic situation, which will gradually lead to lower production rates. This is highly undesirable in the biotech industry, as the chemostats will have to be restarted on a regular basis, which is costly and time consuming.</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">The redox coupled system</DIV></DIV></DIV></DIV><P align="justify"><I><B>Figure 3 – Chemostat fermentation with Rexivator system</B>
Chemostat oscillations in the NAD+/NADH ratio as a result of the Yeast metabolic cycle (<B>A:</B> top graph) lead to change in the Rex DNA binding activity (<B>B:</B> middle graph) leading to controlled bursts in product of reporter gene expression (<B>C:</B> lower graph).</I></P><P align="justify">A strategy to lower the effect of the metabolic adaption could be to couple the productivity with the yeast metabolic cycle (see grey box to the right). The NAD<SUP>+</SUP>/NADH poise oscillates in parallel with the yeast metabolic cycle. The synthesis of a given product will be put under control of the Rexivator, which will lead to periodic burst of gene expression and thus enzyme production (See figure 2). Consequently the cells will have time to recover periodically from the metabolic stress that occurs during the production phase. We believe that this will lead to a prolonged effective chemostat operation, and ultimately higher accumulation of product before a costly restart of the chemostat is required.</P><P align="justify">By using computer modelling and experimental data from literature we will determine when in the metabolic cycle, it is favorable to produce, and when to let the cells rest. These predictions will be used to adjust and tune the sensing system, so that it initiates gene expression at appropriate times. The production over an extended period of time will then be monitored in a chemostat fermentation, to test if the yield has improved before restart is required.</P><P><FONT size="3"><B>Yeast as a model organism for humans: Rexivator in cancer research</B></FONT></P><P align="justify">Yeast is used as a model organism for humans in many research projects. As an example, studies of the yeast cell cycle have paved the way for most of the knowledge about the cell cycle in mammalian cells. As yeast has been widely used as a model organism for studying cancer an integrated tool for detecting a significant change in the redox potential would ease this research, especially because a rising NADH level is seen as a hallmark of carcinogenesis. The prospect for a cancer drug based on a DNA vaccine transcribed only at abnormally high NADH concentrations killing the cell could also be facilitated by research in this field.</P></FONT></TD><TD width="182px" height="100%" valign="top" bgcolor="DDDDDD" class="greybox"><FONT face="arial" size="2"><B>The yeast metabolic cycle</B><P>It has recently been shown by Tu <I>et al.</I> and Klevecz <I>et al.</I> that the expression of at least half of the genes monitored on a standard yeast gene chip will oscillate in a coordinated manner when grown under glucose limited conditions. The cells will shift between oxidative and reductive metabolism in a synchronized metabolic cycle with three phases: oxidative, reductive/building and reductive/
charging. As oxygen will only be consumed in the oxidative phase, the dissolved oxygen will oscillate. Many metabolites and cofactors including NADH and NAD+ will also oscillate during this cycle as NADH is converted to NAD+ when oxygen is consumed.</P></FONT></TD></TR></TBODY></TABLE><TABLE><TBODY><TR height="25px"><TD width="967px" valign="center" align="center"><FONT face="arial" size="1">
 Comments or questions to the team? Please <A href="mailto:igem@bio.dtu.dk" class="email">Email us</A> -- Comments of questions to webmaster? Please <A href="mailto:lronn@bio.dtu.dk" class="email">Email us</A></FONT></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47">http://2009.igem.org/Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47</A>&quot;</DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:DTU_Denmark/theory_private_securkey%3Dhjg1mab2ak47&amp;oldid=106624" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></TABLE></DIV></DIV></DIV></BODY></HTML>