View source for Team:Queens Canada/Description&action=edit
You do not have permission to edit this page, for the following reason:
Edit conflict: The Team Wikis are frozen. They will thaw after the Jamboree.
You can view and copy the source of this page:
<html lang="en"> <head> <meta charset="utf-8"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <meta name="viewport" content="width=device-width, initial-scale=1"> <meta name="description" content=""> <meta name="author" content=""> <title>Team:Queens Canada/Description</title> <!-- Bootstrap Core CSS --> <link rel="stylesheet" href="css/bootstrap.min.css" type="text/css"> <!-- Custom Fonts --> <link href='http://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800' rel='stylesheet' type='text/css'> <link href='http://fonts.googleapis.com/css?family=Merriweather:400,300,300italic,400italic,700,700italic,900,900italic' rel='stylesheet' type='text/css'> <link rel="stylesheet" href="font-awesome/css/font-awesome.min.css" type="text/css"> <!-- Plugin CSS --> <link rel="stylesheet" href="css/animate.min.css" type="text/css"> <!-- Custom CSS --> <link rel="stylesheet" href="css/creative.css" type="text/css"> <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries --> <!-- WARNING: Respond.js doesn't work if you view the page via file:// --> <!--[if lt IE 9]> <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script> <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script> <![endif]--> </head> <body id="page-top"> <nav id="mainNav" class="navbar navbar-default navbar-fixed-top"> <div class="container-fluid"> <!-- Brand and toggle get grouped for better mobile display --> <div class="navbar-header"> <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button> <ul class="nav navbar-nav navbar-left"> <li> <a class="navbar-brand page-scroll" href="https://2016.igem.org/Team:Queens_Canada"> <img class="img-responsive" src="img/logos/qgemlogoind.png"> </a> </li> </ul> </div> <!-- Collect the nav links, forms, and other content for toggling --> <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1"> <ul class="nav navbar-nav navbar-right"> <li class="nav-padtop text-center"> <a href="https://2016.igem.org/Team:Queens_Canada/Team">Team</a> </li> <li class="dropdown nav-padtop text-center"> <button id=dLabel href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Project <span class="caret"></span></button> <ul class="dropdown-menu" aria-labelledby="dLabel"> <li class="text-center"><a href="https://2016.igem.org/Team:Queens_Canada/Background">Background</a></li> <li role="separator" class="divider"></li> <li class="text-center"><a href="https://2016.igem.org/Team:Queens_Canada/Description">Description</a></li> <li role="separator" class="divider"></li> <li class="text-center"><a href="https://2016.igem.org/Team:Queens_Canada/Thermo_Ind">Thermo Ind</a></li> <li role="separator" class="divider"></li> <li class="text-center"><a href="https://2016.igem.org/Team:Queens_Canada/COM_Domains">COM Domains</a></li> <li role="separator" class="divider"></li> <li class="text-center"><a href="https://2016.igem.org/Team:Queens_Canada/Notebook">Notebook</a></li> </ul> </li> <li class="nav-padtop text-center"> <a href="https://2016.igem.org/Team:Queens_Canada/Parts">Parts</a> </li> <li class="nav-padtop text-center"> <a href="https://2016.igem.org/Team:Queens_Canada/Safety">Safety</a> </li> <li class="nav-padtop text-center"> <a href="https://2016.igem.org/Team:Queens_Canada/Attributions">Attributions</a> </li> <li class="nav-padtop text-center"> <a href="https://2016.igem.org/Team:Queens_Canada/HP/Silver">Human Practices</a> </li> <li class="nav-padtop text-center"> <a href="https://2016.igem.org/Team:Queens_Canada/Collaborations">Collaborations</a> </li> <li> <a href="https://2016.igem.org/Main_Page"> <img class="img-responsive" src="img/logos/igem-logo.png"> </a> </li> </ul> </div> <!-- /.navbar-collapse --> </div> <!-- /.container-fluid --> </nav> <header id="pageHeader"> <div class="header-content"> <div class="header-content-inner"> <br><br><br><br> <h1>Description</h1> <hr> <p></p> </div> </div> </header> <div class="container"> <div class="row"> <div class="col-sm-1"></div> <div class="middle-column col-sm-10"> <br><br><br><br> <h2 class="section-heading text-center">Investigating IndC Robustness</h2> <hr class="primary"> <br> <p class="text-muted paragraph">In 2013, Heidelberg iGEM introduced indigoidine tagging of nonribosomal peptides (NRPs) to the iGEM community. It was noted however that there was no paper published on the ability of Indigoidine to tag NRPs, nor were there any NMR or MS data present to demonstrate its ability to tag NRPS. <br><br>                 The Heidelberg 2013 team mentioned that they experienced trouble tagging bulky peptides such as tryptophan. Therefore, one possibility we wanted to explore was steric hindrance of the oxidation domain found in the IndC module incorporating indigoidine. <br><br>                 One other reason behind further investigating the oxidation domain in IndC was because the exact mechanism of indigoidine formation was unknown. It has been debated whether the oxidation domain acts on the cyclized final compound as seen in other FMN-dependent oxidation domains found within NRPS systems, or whether it would oxidize the glutamine attached to the PCP. </p> <br><br> <img class="bodyimageleft" align="left" src="img/content/results1.png"> <img class="bodyimageright" align="right" src="img/content/results2.png"> </div> <div class="col-sm-1"></div> </div> <div class="row"> <div class="col-sm-1"></div> <div class="middle-column col-sm-10"> <p class="text-muted paragraph">Our first hypothesis was the former mentioned above, that the oxidation domain acts on the cyclized product after it is released by the thioesterase (TE) domain. This hypothesis was derived from the fact that cyclized glutamine was observed as the final amino acid in the molecule Padanamide B. <br><br> <img class="bodyimageleft" align="left" src="img/content/results3.png"> <img class="bodyimageright" align="right" src="img/content/results4.png"> <br><br><br>                 This leads us to believe that the TE domain would cyclize glutamine, which would then be oxidized by the oxidation domain in order to spontaneously combine into indigoidine. <br><br>                 The precursor was purchased from Astatech, and it was dissolved in water. The pyroglutamine had a blue colour in solution with absorbance similar to indigoidine at 600nm. However, the pyroglutamine was soluble in water and insoluble in DMSO, opposite of what indigoidine was found to be soluble in in literature [2]. <br><br>                 NMR was carried out in order to determine if the compound we ordered was indeed pyroglutamine. The compound was dissolved in deuterated ethanol, and the NMR spectrum is pictured below.</p> <br><br> <img class="bodyimageleft" align="left" src="img/content/results5.png"> <img class="bodyimageright" align="right" src="img/content/results6.png"> <br><br> </div> <div class="col-sm-1"></div> </div> <div class="row"> <div class="col-sm-1"></div> <div class="middle-column col-sm-10"> <p class="text-muted paragraph"> Despite the solution containing a blue colour, no traces of indigoidine were observed. <br><br>                 The oxidation domain was ordered from IDT and cloned into a pET-16b vector containing a 10x C-terminal His tag. It was purified via Ni+ affinity chromatography. The final concentration was 12.2 mg/ml of protein. The purified oxidation domain contained a yellow colour demonstrating that it was co-purifying with bound FMN. <br><br>                 An enzymatic assay was carried out by adding pyroglutamine and our enzyme to HEPES buffer (pH 7.2) in a plastic cuvette and scanning for changes in Absorbance at 600nm every minute for ten minutes total. </p> <br><br> <img class="bodyimageleft" align="left" src="img/content/results8.png"> <img class="bodyimageright" align="right" src="img/content/results9.png"> <br><br> </div> <div class="col-sm-1"></div> </div> <div class="row"> <div class="col-sm-1"></div> <div class="middle-column col-sm-10"> <p class="text-muted paragraph">                 As seen from above there was no increase in absorbance as one would expect. There is a decrease in absorbance for the enzyme plus pyroglutamine, which one might expect if the indigoidine being formed was precipitating out of solution. However Takahashi et al. saw an increase in absorbance when characterizing BPSA, an IndC homologue in vitro. <br><br>                 From this hypothesis it is noted that the oxidation domain in IndC most likely works on the glutamine while it is tethered to the PCP. If that is the case it would most likely fix the glutamine in a cis configuration, bringing together the residue amine group to act as an intramolecular nucleophile in order to finalize the cyclization of pyroglutamine. To test this new hypothesis, we are currently synthesizing a S-N-acetylcystamine (SNAC), which mimics the phosphopantathiene arm found on the PCP. The use of peptide SNAC to probe NRPS domain substrate specificity has been previously demonstrated [3]. <br><br>                 We hope that by understanding the mechanism of glutamine cyclization and oxidation we can begin to create more robust IndC variants. </p> <br><br> <h2 class="text-left">References</h2> <br><br>                 1. Hahn, M., & Stachelhaus, T. (2004). Selective interaction between nonribosomal peptide synthetases is facilitated by short communication-mediating domains.Â <em>Proceedings of the National Academy of Sciences of the United States of America,Â 101</em>(44), 15585-15590. <br><br>                 2. Takahashi, H., Kumagai, T., Kitani, K., Mori, M., Matoba, Y., & Sugiyama, M. (2007). Cloning and characterization of a Streptomyces single module type non-ribosomal peptide synthetase catalyzing a blue pigment synthesis. <em>Journal of Biological Chemistry,Â 282</em>(12), 9073-9081. <br><br>                 3. Marahiel, M. A., & Essen, L. O. (2009). Nonribosomal peptide synthetases: mechanistic and structural aspects of essential domains. <em>Methods in enzymology,Â 458,</em> 337-351. <br><br><br><br> </div> <div class="col-sm-1"></div> </div> </div> <footer> <div class="container"> <div class="row"> <div class="col-md-4"> <br><br> <a href="https://www.facebook.com/iGEMQueens/?fref=ts"><img class="img-responsive img-footer" src="img/logos/facebook-logo.png"></a> <br> </div> <div class="col-md-4"> <br><br> <a href="http://queensigem.ca/"><img class="img-responsive img-footer img-center" src="img/logos/website-blu.png"></a> <br> </div> <div class="col-md-4"> <br><br> <a href="https://twitter.com/iGEMQueens"><img class="img-responsive img-footer" src="img/logos/twitter-logo.png"></a> <br> </div> </div> </div> </footer> <!-- jQuery --> <script src="js/jquery.js"></script> <!-- Bootstrap Core JavaScript --> <script src="js/bootstrap.min.js"></script> <!-- Plugin JavaScript --> <script src="js/jquery.easing.min.js"></script> <script src="js/jquery.fittext.js"></script> <script src="js/wow.min.js"></script> <!-- Custom Theme JavaScript --> <script src="js/creative.js"></script> </body> </html>
Return to Team:Queens Canada/Description&action=edit .
