<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_WHU-China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:WHU-China/Experiments</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="container"><DIV class="middle-left"><DIV class="secmenu" id="sec"><UL id="tit1"><LI><A href="https://2015.igem.org/Team:WHU-China/Team">Member</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Collaborations">Collaborations</A></LI></UL><UL id="tit2"><LI><A href="https://2015.igem.org/Team:WHU-China/Description?num=1">Overview</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Description?num=2">Criticality Detector</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Description?num=6">Binary System</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Description?num=8">Dosage Control System</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Results">Results</A></LI></UL><UL id="tit3"><LI><A href="https://2015.igem.org/Team:WHU-China/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Basic_Part">Basis Parts</A></LI></UL><UL id="tit4"><LI><A href="https://2015.igem.org/Team:WHU-China/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Experiments">Protocols</A></LI></UL><UL id="tit5"><LI><A href="https://2015.igem.org/Team:WHU-China/Practices?num=5">Meet ups</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Practices?num=1">Sketch in dishes</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Practices?num=8">Smile mates</A></LI></UL><UL id="tit6"><LI><A href="https://2015.igem.org/Team:WHU-China/Modeling">Modeling</A></LI></UL></DIV></DIV><DIV class="middle-left"><DIV class="secmenu" id="sec"><UL id="tit1"><LI><A href="https://2015.igem.org/Team:WHU-China/Team">Member</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Collaborations">Collaborations</A></LI></UL><UL id="tit2"><LI><A href="https://2015.igem.org/Team:WHU-China/Description">Overview</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Criticality">Criticality Detector</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/BinarySystem">Binary System</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/DentalCP">Dental Caries Prevention</A></LI><LI><A href="javascript:return false;">Results</A></LI></UL><UL id="tit3"><LI><A href="https://2015.igem.org/Team:WHU-China/Parts">Team Parts</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Basic_Parts">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Composite_Part">Composite Parts</A></LI><LI><A href="javascript:return false;">Favorite Parts</A></LI></UL><UL id="tit4"><LI><A href="https://2015.igem.org/Team:WHU-China/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Safety">Safety</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Experiments">Protocols</A></LI></UL><UL id="tit5"><LI><A href="https://2015.igem.org/Team:WHU-China/Practices?num=5">Meet ups</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Practices?num=1">Sketch in dishes</A></LI><LI><A href="https://2015.igem.org/Team:WHU-China/Practices?num=8">Smile mates</A></LI></UL><UL id="tit6"><LI><A href="https://2015.igem.org/Team:WHU-China/Modeling">Modeling</A></LI></UL></DIV></DIV><DIV class="text"><DIV class="textarea"><DIV class="editarea" id="member1"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 1 of 10</H2><DIV class="member"><DIV class="txt"><H2>Agarose Gel Electrophoresis</H2><UL><LI>1. Seal the edges of a clean, dry glass plate (or the open ends of the plastic tray supplied with the electrophoresis apparatus) with tape to form a mold. Set the mold on a horizontal section of the bench.</LI><LI>
2. Prepare sufficient electrophoresis buffer (usually 1x TAE or 0.5x TBE) to fill the electrophoresis tank and to cast the gel.
It is important to use the same batch of electrophoresis buffer in both the electrophoresis tank and the gel.</LI><LI>
3. Prepare a solution of agarose in electrophoresis buffer at a concentration appropriate for separating the particular size fragments expected in the DNA sample(s): Add the correct amount of powdered agarose (please see table below) to a measured quantity of electrophoresis buffer in an Erlenmeyer flask or a glass bottle.Range of Separation in Cells Containing Different Amounts of Standard Low-EEO Agarose.</LI></UL><CENTER><TABLE border="1"><TBODY><TR><TH>Agarose Concentration in Gel 
(% [w/v])</TH><TH>Range of Separation of Linear DNA Molecules 
(kb)</TH></TR><TR><TH>0.3</TH><TH>5-60</TH></TR><TR><TH>0.6</TH><TH>1-20</TH></TR><TR><TH>0.7</TH><TH>0.8-10</TH></TR><TR><TH>0.9</TH><TH>0.5-7</TH></TR><TR><TH>1.2</TH><TH>0.4-6</TH></TR><TR><TH>1.5</TH><TH>0.2-3</TH></TR><TR><TH>2.0</TH><TH>0.1-2</TH></TR></TBODY></TABLE></CENTER><SPAN style="font-size:16px;margin-left:5%">Agarose gels are cast by melting the agarose in the presence of the desired buffer until a clear, transparent solution is achieved. The melted solution is then poured into a mold and allowed to harden. Upon hardening, the agarose forms a matrix, the density of which is determined by the concentration of the agarose.
                </SPAN><UL><LI>4. Loosely plug the neck of the Erlenmeyer flask with Kimwipes. If using a glass bottle, make certain the cap is loose.Heat the slurry in a boiling-water bath or a microwave oven until the agarose dissolves.Heat the slurry for the minimum time required to allow all of the grains of agarose to dissolve.</LI><LI>
                5. Use insulated gloves or tongs to transfer the flask/bottle into a water bath at 55°C. When the molten gel has cooled,add ethidium bromide to a final concentration of 0.5 μg/ml. Mix the gel solution thoroughly by gentle swirling.IMPORTANT SYBR Gold should not be added to the molten gel solution.
                </LI><LI>6. While the agarose solution is cooling, choose an appropriate comb for forming the sample slots in the gel. Position the comb 0.5-1.0 mm above the plate so that a complete well is formed when the agarose is added to the mold.
                </LI><LI>7. Pour the warm agarose solution into the mold.The gel should be between 3 mm and 5 mm thick. Check that no air bubbles are under or between the teeth of the comb. Air bubbles present in the molten gel can be removed easily by poking them with the corner of a Kimwipe.
                </LI><LI>
                8. Allow the gel to set completely (30-45 minutes at room temperature), then pour a small amount of electrophoresis buffer on the top of the gel, and carefully remove the comb. Pour off the electrophoresis buffer and carefully remove the tape. Mount the gel in the electrophoresis tank.
                </LI><LI>
                9. Add just enough electrophoresis buffer to cover the gel to a depth of approx. 1 mm.
                </LI></UL></DIV></DIV></DIV><DIV class="editarea" id="member2"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 2 of 10</H2><DIV class="member"><DIV class="txt"><H2>Age</H2><UL><LI>
                        10. Mix the samples of DNA with 0.20 volume of the desired 6x gel-loading buffer.The maximum amount of DNA that can be applied to a slot depends on the number of fragments in the sample and their sizes. The minimum amount of DNA that can be detected by photography of ethidium-bromide-stained gels is approx. 2 ng in a 0.5-cm-wide band (the usual width of a slot). More sensitive dyes such as SYBR Gold can detect as little as 20 pg of DNA in a band.
                        </LI><LI>
                        11. Slowly load the sample mixture into the slots of the submerged gel using a disposable micropipette, an automatic micropipettor, or a drawn-out Pasteur pipette or glass capillary tube. Load size standards into slots on both the right and left sides of the gel.
                        </LI><LI>
                        12. Close the lid of the gel tank and attach the electrical leads so that the DNA will migrate toward the positive anode (red lead). Apply a voltage of 1-5 V/cm (measured as the distance between the positive and negative electrodes). If the leads have been attached correctly, bubbles should be generated at the anode and cathode (due to electrolysis), and within a few minutes, the bromophenol blue should migrate from the wells into the body of the gel. Run the gel until the
bromophenol blue and xylene cyanol FF have migrated an appropriate distance through the gel.
The presence of ethidium bromide allows the gel to be examined by UV illumination at any stage during electrophoresis. The gel tray may be removed and placed directly on a transilluminator. Alternatively, the gel may be examined using a hand-held source of UV light. In either case, turn off the power supply before examining the gel!
                        </LI><LI>
                        13. When the DNA samples or dyes have migrated a sufficient distance through the gel, turn off the electric current and remove the leads and lid from the gel tank. If ethidium bromide is present in the gel and electrophoresis buffer, examine the gel by UV light and photograph the gel as described in Chapter 5, Protocol 2 . Otherwise, stain the gel by immersing it in electrophoresis buffer or H2O containing ethidium bromide (0.5 μg/ml) for 30-45 minutes at room temperature or by soaking in a 1:10,000-fold dilution of SYBR Gold stock solution in electrophoresis buffer.
                        </LI></UL></DIV></DIV></DIV><DIV class="editarea" id="member3"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 3 of 10</H2><DIV class="member"><DIV class="txt"><H2>Double Digestion</H2><SPAN>
                    Pipette the following into a 1.5ml microfuge tube:
                    </SPAN><CENTER><TABLE border="1"><TBODY><TR><TH colspan="2">For Identification(10μl system)</TH></TR><TR><TH>Enzyme A</TH><TH>0.3μl</TH></TR><TR><TH>Enzyme B</TH><TH>0.3μl</TH></TR><TR><TH>10× buffer</TH><TH>1μl</TH></TR><TR><TH>DNA</TH><TH>1μl(according to specific concentration)</TH></TR><TR><TH>dd H2O</TH><TH>up to 10μl</TH></TR><TR><TH colspan="2">For Recovery(50μl system)</TH></TR><TR><TH>Enzyme A</TH><TH>0.5μl</TH></TR><TR><TH>Enzyme B</TH><TH>0.5μl</TH></TR><TR><TH>10× buffer</TH><TH>5μl</TH></TR><TR><TH>DNA</TH><TH>2μg</TH></TR><TR><TH>dd H2O</TH><TH>up to 50μl</TH></TR></TBODY></TABLE></CENTER><SPAN style="text-indent:2em">Incubate at recommended temperature (37℃) for at least 2 hour; </SPAN><SPAN style="text-indent:2em">
Purify the digestion product; </SPAN><P style="text-indent:2em">
Notes: The enzymes used here are TaKaRa enzymes (EcoRI/ XbaI/ SpeI/ PstI),
 and buffer M or H is suitable for most of double digestion.
                    </P></DIV></DIV></DIV><DIV class="editarea" id="member4"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 4 of 10</H2><DIV class="member"><DIV class="txt"><H2>Expression of Cloned Genes in <I>E</I>. coli</H2><UL><LI>
                        1. Modify by PCR, or isolate by restriction enzyme digestion, a fragment of DNA carrying 5'- and 3'-restriction enzyme sites compatible with sites in a bacteriophage T7 promoter expression plasmid (e.g., pET vectors).
                        </LI><LI>
                        2. Ligate the DNA fragment containing the cDNA/gene of interest into the expression vector.
                        </LI><LI>
                        3. Transform E. coli strain BL21(DE3) or HMS174(DE3) with aliquots of the ligation reaction. Select for ampicillin-resistant transformants by plating aliquots of the transformation reaction on NZCYM agar plates containing 50 μg/ml ampicillin.Incubate the plates overnight at 37°C.
                        </LI><LI>
                        4. Screen the transformants by colony hybridization and/or restriction enzyme analysis, oligonucleotide hybridization, or direct DNA sequence analysis of plasmid minipreparations.
                        </LI><LI>
                        5. Inoculate 1-ml cultures (NZCYM medium containing 50 μg/ml ampicillin) with a transformed colony containing positive control vectors, negative control vectors, and one containing the recombinant vector. Incubate the cultures overnight at 37°C to obtain a saturated culture.
                        </LI><LI>
                        6. Inoculate 5 ml of NZCYM medium containing 50 μg/ml ampicillin in a 50-ml flask with 50 μl of a saturated culture.Incubate the cultures for 2 hours at 37°C.
                        It is important to monitor the number of bacteria inoculated into the growth medium, the length of time cells are grown before induction, and the density to which cells are grown after induction.
                        </LI><LI>
                        7. Transfer 1 ml of each culture (zero-time aliquot) to a microfuge tube. Immediately process the zero-time aliquots as described in Steps 9 and 10.
                        </LI><LI>
                        8. Induce the remainder of each culture by adding IPTG to a final concentration of 1.0 mM and continue incubation at 20-37°C with aeration.
                        </LI><LI>
                        9. At 0.5, 1, 2, and 3 hours after induction, transfer 1 ml of each culture to a microfuge tube, measure the A550 in a spectrophotometer, and centrifuge the tubes at maximum speed for 1 minute at room temperature in a microfuge.Remove the supernatants by aspiration.
                        </LI><LI>
                        10. Resuspend each pellet in 100 μl of 1x SDS gel-loading buffer, and heat the samples to 100°C for 3 minutes. Centrifuge.the tubes at maximum speed for 1 minute at room temperature in a microfuge, and store them on ice until all of the samples are collected and ready to load on a gel.
                        </LI><LI>
                        11. Warm the samples to room temperature and load 0.15 OD550 units (of original culture) or 40 μg of each suspension on a 10% SDS-polyacrylamide gel.
                        </LI><LI>
                        12. Run the gel at 8-15 V/cm until the bromophenol blue reaches the bottom of the resolving gel.
                        </LI><LI>
                        13. Stain the gel with Coomassie Brilliant Blue or silver, or carry out an immunoblot to visualize the induced protein.
                        </LI><LI>
                        14. For large-scale expression and purification of the target protein, inoculate 50 ml of NZCYM containing 50 μg/ml ampicillin in a 250-ml flask with individual colonies of E. coli containing the recombinant and control plasmids. Incubate the cultures overnight at 37°C.
                        </LI><LI>
                        15. Inoculate 450-500 ml of NZCYM containing 50 μg/ml ampicillin in a 2-liter flask with 5-50 ml of overnight culture of E.coli containing the recombinant plasmid. Incubate the culture with shaking at 37°C until the culture has reached the midlog phase of growth (A550 = 0.5-1.0).
                        </LI><LI>
                        16. Induce expression of the target protein based on the optimal values of IPTG concentration, incubation time, and incubation temperature determined in the previous section.
                        </LI><LI>
                        17. After the induced cells have grown for the proper length of time, harvest the cells by centrifugation at 5000g (5500 rpm in a Sorvall GSA rotor) for 15 minutes at 4°C and proceed with a purification protocol
                        </LI></UL></DIV></DIV></DIV><DIV class="editarea" id="member5"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 5 of 10</H2><DIV class="member"><DIV class="txt"><H2>Ligation Protocol with T4 DNA Ligase</H2><UL><LI>
                        1.Set up the following reaction in a microcentrifuge tube on ice.
(T4 DNA Ligase should be added last. Note that the table shows a ligation using a molar ratio of 1:3 vector to insert for the indicated DNA sizes.).
						<CENTER><TABLE><TBODY><TR><TD><STRONG>COMPONENT</STRONG></TD><TD><STRONG>20 μl REACTION</STRONG></TD></TR><TR><TD>10X T4 DNA Ligase Buffer*</TD><TD>2 μl</TD></TR><TR><TD>Vector DNA (4 kb)</TD><TD>50 ng (0.020 pmol)</TD></TR><TR><TD>Insert DNA (1 kb)</TD><TD>37.5 ng (0.060 pmol)</TD></TR><TR><TD>Nuclease-free water</TD><TD>to 20 μl</TD></TR><TR><TD>T4 DNA Ligase</TD><TD>1 μl</TD></TR></TBODY></TABLE></CENTER></LI><LI>
                        2.* The T4 DNA Ligase Buffer should be thawed and resuspended at room temperature.
                        </LI><LI>
                        3.Gently mix the reaction by pipetting up and down and microfuge briefly.
                        </LI><LI>
                        4.For cohesive (sticky) ends, incubate at 16°C overnight or room temperature for 10 minutes.
                        </LI><LI>
                        5.For blunt ends or single base overhangs, incubate at 16°C overnight or room temperature for 2 hours (alternatively, high concentration T4 DNA Ligase can be used in a 10 minute ligation).
                        </LI><LI>
                        6.Heat inactivate at 65°C for 10 minutes.
                        </LI><LI>
                        7.Chill on ice and transform 1-5 μl of the reaction into 50 μl competent cells.

                        </LI></UL></DIV></DIV></DIV><DIV class="editarea" id="member6"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 6 of 10</H2><DIV class="member"><DIV class="txt"><H2>Operation of Microplate Reader</H2><STRONG>Plate format</STRONG><SPAN>
                    Select the type of microplate to be used. The default options are 6
wells, 12 wells, 24 wells, 48 wells, 96 wells, 192 wells or 384 wells.
The graphical representation of the microplate will change
according to the number of wells chosen.
                    </SPAN><STRONG>Auto-zero</STRONG><SPAN>Select this option to perform an auto-zero when the Zero button is selected from the Kinetics Application window. A zero reading will be taken at the selected well position using the current method setup. To select the well position to be zeroed, use the Select Well option. These readings will be offset to zero and subsequent readings will have this single value subtracted from each well in the sequence. The reading is taken only once, stored, and subtracted from the
ratioed intensity reading.</SPAN><STRONG>Select well</STRONG><SPAN>
                    Choose Select well to open the Go To well dialog where you can select the well position to be zeroed. This option is only available if you have selected Auto-zero. 
                    </SPAN><STRONG>Well</STRONG><SPAN>
                    Displays the well position of the well selected for auto-zeroing. This field is not editable.
                    </SPAN><STRONG>Display options</STRONG><SPAN>Select this option to open the Display Options dialog where you can set up the display parameters associated with the microplate reader.</SPAN><STRONG>Sample names</STRONG><SPAN>Select Sample Names to open the Microplate Sample List dialog where you can enter or modify the sample names. Highlight one or more wells on the microplate reader diagram to view and change their sample information.</SPAN><STRONG>Go to well</STRONG><SPAN>
                    Select this option to open the Go To Well dialog.
                    </SPAN><STRONG>Well position</STRONG><SPAN>Displays the well position that the mouse cursor is pointing to. This field is not editable.</SPAN><STRONG>Reset</STRONG><SPAN>Select Reset to move the microplate reader to the first well position defined by the read sequence.</SPAN><STRONG>Park</STRONG><SPAN>Select Park to move the microplate reader to the Park position. The microplate reader accessory is moved clear of the sample compartment to enable you to easily access all wells.</SPAN><STRONG>Microplate reader diagram</STRONG><SPAN>The microplate reader diagram represents the number of wells present for the chosen microplate configuration. The labels for the rows will be 1 .. 12 and for the columns A .. H (for
a 96 microplate). Similar labels will occur for different microplates according. The sequence for numbering the read order will be from A1 to A12, B12 - B1, C1 - C12…etc for a 96 microplate. A similar sequence will be for other plates. To select or clear a single well, simply click or right-click on it. To select or clear a block of wells, hold down the left mouse and
drag around the block required. All wells within the area will be selected or cleared. For additional information, refer to the Kinetics, Scan and Advanced Reads software help sections for specific operational details.</SPAN></DIV></DIV></DIV><DIV class="editarea" id="member7"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 7 of 10</H2><DIV class="member"><DIV class="txt"><H2>PCR</H2><SPAN>To prepare several parallel reactions and to minimize the possibility of pipetting errors, prepare a PCR master mix by mixing water, buffer, dNTPs, primers and template DNA. Pfu DNA Polymerase should be the last component added. Prepare sufficient master mix for the number of reactions plus one extra to allow for pipeting error. </SPAN><UL><LI>
                    	1. Gently vortex and briefly centrifuge all solutions after thawing. 
                    </LI><LI>
                    2. Place a thin-walled PCR tube on ice and add the following components for each 50 µL reaction: 
                    <CENTER><TABLE border="1"><TBODY><TR><TD>Water, nuclease-free (#R0581) 
                                </TD><TD>variable </TD></TR><TR><TD>10X Pfu Buffer with MgSO4* 
                                </TD><TD>5 µL</TD></TR><TR><TD>dNTP Mix, 2 mM each (#R0241)
                                </TD><TD>5 µL (0.2 mM of each)</TD></TR><TR><TD>Forward primer
                                </TD><TD>0.1-1.0 µM </TD></TR><TR><TD>Reverse primer
                                </TD><TD>0.1-1.0 µM</TD></TR><TR><TD>Template DNA
                                </TD><TD>50 pg - 1 µg</TD></TR><TR><TD>Pfu DNA Polymerase
                                </TD><TD>1.25-2.5 U</TD></TR><TR><TD>Total volume 
                                </TD><TD>50 µL</TD></TR></TBODY></TABLE></CENTER></LI><LI>
                    3. Gently vortex the samples and spin down.
                    </LI><LI>
                    4. If using a thermal cycler that does not use a heated lid, overlay the reaction mixture with 25 µL of mineral oil. 
                    </LI><LI>
                    5. Perform PCR using the following thermal cycling conditions: 
                    <CENTER><TABLE border="1"><TBODY><TR><TD>Step</TD><TD>Temperature, °C</TD><TD>Time</TD><TD>Number of cycles</TD></TR><TR><TD>Initial denaturation</TD><TD>95 </TD><TD>1-3 min </TD><TD>1</TD></TR><TR><TD>Denaturation</TD><TD>95</TD><TD>30 s </TD><TD>25-35</TD></TR><TR><TD>Annealing </TD><TD>tm-5</TD><TD>30 s</TD><TD>25-35</TD></TR><TR><TD>Extension</TD><TD>72</TD><TD>2 min/kb</TD><TD>25-35</TD></TR><TR><TD>Final extension  </TD><TD>72</TD><TD>5-15 min</TD><TD>1</TD></TR></TBODY></TABLE></CENTER></LI><LI>
                    *If using 10X Pfu Buffer without MgSO4, a 25 mM MgSO4 solution should be added to 50 µL of the master mix: 
                    <CENTER><TABLE border="1"><TBODY><TR><TD>Final concentration of MgSO4, mM</TD><TD>1</TD><TD>1.25</TD><TD>1.5</TD><TD>1.75</TD><TD>2</TD><TD>2.5</TD><TD>3</TD><TD>4</TD></TR><TR><TD>Volume of 25 mM MgSO4, µL</TD><TD>2</TD><TD>2.5</TD><TD>3</TD><TD>3.5</TD><TD>4</TD><TD>5</TD><TD>6</TD><TD>8</TD></TR></TBODY></TABLE></CENTER></LI></UL></DIV></DIV></DIV><DIV class="editarea" id="member8"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 8 of 10</H2><DIV class="member"><DIV class="txt"><H2>Plasmid DNA Mini Kit I</H2><UL><LI>1. Isolate a single colony from a freshly streaked selective plate, and inoculate a culture of 1- 5 mL LB medium containing the appropriate selective antibiotic. Incubate for ~12-16 hours at 37°C with vigorous shaking (~ 300 rpm). Use a 10-20 mL culture tube or a flask with a volume of at least 4 times the volume of the culture. It is strongly recommended that an endA negative strain of E. coli be used for routine plasmid isolation. 
                        </LI><LI>2. Centrifuge at 10,000 x g for 1 minute at room temperature.
                        </LI><LI>3. Decant or aspirate and discard the culture media.
                        </LI><LI>4. Add 250 μL Solution I/RNase A. Vortex or pipet up and down to mix thoroughly.Complete resuspension of cell pellet is vital for obtaining good yields.
Note: RNase A must be added to Solution I before use. 
                        </LI><LI>5. Transfer suspension into a new 1.5 mL microcentrifuge tube.
                        </LI><LI>6. Add 250 μL Solution II. Invert and gently rotate the tube several times to obtain a clear lysate. A 2-3 minute incubation may be necessary.

Note: Avoid vigorous mixing as this will shear chromosomal DNA and lower plasmid purity. Do not allow the lysis reaction to proceed more than 5 minutes. Store Solution II tightly capped when not in use to avoid acidification from CO2 in the air.
                        </LI><LI>7. Add 350 μL Solution III. Immediately invert several times until a flocculent white precipitate forms.

Note: It is vital that the solution is mixed thoroughly and immediately after the addition of Solution III to avoid localized precipitation.
                        </LI><LI>8. Centrifuge at maximum speed (≥13,000 x g) for 10 minutes. A compact white pellet will form. Promptly proceed to the next step.
                        </LI><LI>9. Insert a HiBind? DNA Mini Column into a 2 mL Collection Tube.Optional Protocol for Column Equilibration:</LI><LI>
1. Add 100 μL 3M NaOH to the HiBind? DNA Mini Column.</LI><LI>
2. Centrifuge at maximum speed for 30-60 seconds.</LI><LI>
3. Discard the filtrate and reuse the collection tube.</LI></UL><LI>10. Transfer the cleared supernatant from Step 8 by CAREFULLY aspirating it into the HiBind? DNA Mini Column. Be careful not to disturb the pellet and that no cellular debris is transferred to the HiBind? DNA Mini Column.
                        </LI><LI>11. Centrifuge at maximum speed for 1 minute.
                        </LI><LI>12. Discard the filtrate and reuse the collection tube.
                        </LI><LI>13. Add 500 μL HB Buffer.
                        </LI><LI>14. Centrifuge at maximum speed for 1 minute.E.Z.N.A.? Plasmid DNA Mini Kit I Spin Protocol
                        </LI><LI>15. Discard the filtrate and reuse collection tube.
                        </LI><LI>16. Add 700 μL DNA Wash Buffer.

Note: DNA Wash Buffer must be diluted with 100% ethanol prior to use. 
                        </LI><LI>17. Centrifuge at maximum speed for 1 minute.
                        </LI><LI>18. Discard the filtrate and reuse the collection tube.

Optional: Repeat Steps 16-18 for a second DNA Wash Buffer wash step.
                        </LI><LI>19. Centrifuge the empty HiBind? DNA Mini Column for 2 minutes at maximum speed to dry the column matrix.

Note: It is important to dry the HiBind? DNA Mini Column matrix before elution.Residual ethanol may interfere with downstream applications.
                        </LI></DIV></DIV></DIV><DIV class="editarea" id="member9"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 9 of 10</H2><DIV class="member"><DIV class="txt"><H2>Recovery of DNA</H2><UL><LI>1. Prepare a gel containing the appropriate concentration of low-melting-temperature agarose in 1x TAE buffer.</LI><LI>2. Cool the gel to room temperature, and then transfer it and its supporting glass plate to a horizontal surface in a gel box.</LI><LI>3. Mix the samples of DNA with gel-loading buffer, load them into the slots of the gel, and carry out electrophoresis at 3-6V/cm.</LI><LI>4. If needed, stain the agarose gel with ethidium bromide or with SYBR Gold, and locate the DNA band of interest using a hand-held, long-wavelength (302 nm) UV lamp.</LI><LI>5. Use a sharp scalpel or razor blade to cut out a slice of agarose containing the band of interest and transfer it to a clean,disposable plastic tube.</LI><LI>6. After cutting out the band, photograph the gel as described in Chapter 5, Protocol 2 to record which band of DNA was removed.</LI><LI>7. Add approx. 5 volumes of LMT elution buffer to the slice of agarose, close the top of the tube, and melt the gel by incubation for 5 minutes at 65°C.</LI><LI>8. Cool the solution to room temperature, and then add an equal volume of equilibrated phenol. Vortex the mixture for 20 seconds, and then recover the aqueous phase by centrifugation at 4000g (5800 rpm in a Sorvall SS-34 rotor) for 10 minutes at 20°C.

The white substance at the interface is agarose.</LI><LI>9. Extract the aqueous phase once with phenol:chloroform and once with chloroform.</LI><LI>10. Transfer the aqueous phase to a fresh centrifuge tube. Add 0.2 volume of 10 M ammonium acetate and 2 volumes of absolute ethanol at 4°C. Store the mixture for 10 minutes at room temperature, and then recover the DNA by centrifugation, for example, at 5000g (6500 rpm in a Sorvall SS-34 rotor) for 20 minutes at 4°C.</LI><LI>11. Wash the DNA pellet with 70% ethanol and dissolve in an appropriate volume of TE (pH 8.0).</LI></UL></DIV></DIV></DIV><DIV class="editarea" id="member10"><H2 style="display:inline; float:left">Protocols</H2><H2 style="display:inline; float:right">Page 10 of 10</H2><DIV class="member"><DIV class="txt"><H2>Transformation(Heat Shock)</H2><UL><LI>1. Thaw one vial (or more) of cells (50 ul volume) on ice. The cells are stored in the -80 fridge. Depending on other parameters (nature of ligation, selection criteria for clones, etc.), a smaller volume (down to 15 ul) may be used. You can aliquot volumes into regular 0.5 or 1.5 ml eppendorf tubes and use those tubes.</LI><LI>2. Add 1-5 ul of ligation reaction (up to 10-15% of volume of cells) or plasmid DNA (10-50 pg) and mix by using tip of the pipette tip.</LI><LI>3. Incubate on ice for 15-30 min.</LI><LI>4. Heat shock for exactly 30 secs at exactly 42 deg. You can use a thermocycler, a water bath or a small beaker filled with water at the right temperature.</LI><LI>5. Place on ice for 0.5-1 minute.</LI><LI>6. Add 250 ul (for 50 ul bacteria) of SOC medium (commercial source) that has been prewarmed to 37 deg (not necessary).</LI><LI>7. Shake at about 225 rpm on a horizontal shaker at 37 deg for 1 hr. Tape the tubes to the shaker in a horizontal position if required.</LI><LI>8. Spread a tenth to all (usually half to all) of the culture on a 10 cm LB agar plate (usually stored in the cold room; different types such as LB with ampicillin or LB with kanamycin may be available).</LI><LI>9. Incubate the plates, inverted (you may have to wait 1-2 hours before inverting if a high volume of culture was plated) at 37 deg overnight.</LI><LI>10. Bacterial colonies are visible after about 12 hours and they can be picked at 15-18 hours. Beyond 20-22 hours, satellite colonies appear.</LI><LI>11. The plates can be stored, after parafilm sealing, at 4 deg. The colonies remain viable for up to a week.</LI></UL></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>