<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Antigen_Receptors skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Antigen Receptors</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Antigen, receptors</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right orangeBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Antigen, Receptors</SPAN></P></DIV></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner orangeBg"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Antigen, Receptors</H1></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Antigen, Receptors</H1></DIV></DIV><DIV class="container"><MAIN style="margin: 0"><DIV id="summary" class="section container"><H2>Quick summary of our receptors</H2><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to demonstrate the evolution of synthetic Notch receptors and our ENABLE toolbox present in 2018 iGEM GJ.</P></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to introduce our assay for measuring the signal-to-noise ratio of SynNotch. Please note that all cells expressing the receptors having mCherry expressed at the same time/level via P2A.</P></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to summarize our picks of SynNotch receptors with different extracellular domains and intracellular domains.</P></DIV></DIV><DIV id="full" class="section container"><H2>Transmembrane signal transduction systems
                        </H2><P>In order to be able to implement a customable multiplexed transmembrane signal input and
                            output relationship, the first thing to do is to engineer modular receptors to enable it
                            to recognize extracellular signals and transduce them into customized intracellular signals.
                            To this end, a variety of techniques have been developed, such as Tango
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/18165312" target="_blank">(Barnea et al., 2008)</A>,
                            CAR (Porter et al., 2011), GEMS (Scheller et al., 2018), MESA
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/24611683" target="_blank">(Daringer et al., 2014;</A>
                            Hartfield et al., 2017; Schwarz et al., 2017), SynNotch <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830878" target="_blank">(Morsut et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26830879" target="_blank">Roybal et al., 2016a;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/27693353" target="_blank">Roybal et al., 2016b;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/29853554" target="_blank">Toda et al., 2018)</A>, etc.
                            High modularity of the extracellular and intracellular domains of synthetic Notch (SynNotch),
                            as well as its orthogonality and adaptation to contact-dependent signaling, fully meets our needs.
                            Thus, we ultimately use SynNotch technology as receiver for extracellular signals to build our
                            transmembrane logic gates.
                        </P><P>Notch receptors are single-pass transmembrane receptors that transduce extracellular surface
                            signals from sender cells to the inside of receiver cells. It features structural modularity and
                            orthogonality in intracellular signal transduction. The signal strength and intensity of wild-type
                            Notch receptors plays a crucial role in cell-fate decisions such as proliferation in development
                            as well as homeostasis. This is true for functional outcomes that mainly rely on the dosage of
                            the gene regulation and context of Notch signaling <A href="https://www.ncbi.nlm.nih.gov/pubmed/18039126" target="_blank">(Aster et al., 2008;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/27507209" target="_blank">Bray, 2016)</A> For SynNotch,
                            specialized factors will exclusively participate in regulation of genetic circuits that allow
                            novel user-defined transcriptional responses. In our project ENABLE, we have engineered the cells
                            to perform logical operations on multiple transmembrane signals to produce a variety of
                            customizable outputs.
                        </P><DIV class="expFigureHolder" style="width:40%;margin:0 auto;"><P><B>Figure 1. SynNotch activation.</B>
        SynNotch has customizable extracellular and intracellular domains, as well as a core regulatory region.
        After the extracellular domain of the single-chain antibody recognizes the corresponding surface antigen,
        the core regulatory region undergoes two cleavage (S2, S3 sites) to finally release the intracellular
        segment of the transcription factor into the nucleus to activate the downstream circuits.
    </P></DIV><H3>What is different in signal input?</H3><H4>1.	Ligands</H4><P>The largest family of ligands that can activate wild type Notch receptors features three
                            related structural motifs, which are DSL (Delta/Serrate/LAG-2) motif, EGF-like repeats,
                            and DOS (Delta and OSM-11-like proteins) domain that are tandem EGF repeats
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/19379690" target="_blank">(Kopan et al., 2009)</A>.
                            Wild type Notch receptors features cis-inhibition and trans-activation due to the antiparallel
                            orientation of extracellular domains.
                        </P><P>Similar to Notch receptors, the cis-inhibition mechanism was recapitulated in SynNotch receptors
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830878" target="_blank">(Morsut et al., 2016)</A>.
                            This indicates that the activating and inhibiting mechanism of wild type Notch and SynNotch
                            are probably alike in nature. Therefore, as long as the Notch core can be activated, any
                            ectodomain could work. In 2015, Gordon and her colleagues demonstrated that Notch with FBP
                            as ectodomain targeting FKBP in the presence of chemical rapamycin can activate Notch receptors
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/26051539" target="_blank">(Gordon et al., 2015)</A>.
                            Lim and his colleagues have showed that SynNotch could recognize antigens such as CD19 and EGFP
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830878" target="_blank">(Morsut et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26830879" target="_blank">Roybal et al., 2016a)</A>.
                        </P><H4>2.	The ectodomain of Notch receptors</H4><P>Previous work by Morsut et al. showed that the natural surface antigen CD19 and the non-natural
                            surface antigen EGFP have good orthogonality <A href="https://www.ncbi.nlm.nih.gov/pubmed/29853554" target="_blank">(Toda et al., 2018)</A>.
                            We applied this and designed two surface antigens, surCD19 and surEGFP. By immunostaining,
                            we were able to clearly detect surCD19 and surEGFP expressed on the surface of 293T cells
                            <FONT color="purple">(Figure 2a)</FONT>.
                            Thus, these two surface antigens can serve as ideal dual antigens for our transmembrane binary
                            Boolean logic inputs.
                        </P><P>We selected αCD19 (anti-CD19) <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830878" target="_blank">(Morsut et al., 2016)</A>,
                            LaG17 (anti-EGFP with low affinity), LaG16-2 (anti-EGFP with high affinity) (Fridy et al., 2014)
                            as the extracellular domain of SynNotch. Mouse Notch1 core (mN1c) regulatory region as the core
                            transmembrane region of SynNotch. Two transcription factors, tTAA (TetR-VP48) and GV2 (Gal4DBD-VP64),
                            which are orthogonal to each other, are used as the intracellular domain of SynNotch
                            <FONT color="purple">(Figure 2a)</FONT>.
                            We found that the combination of different extracellular domains and intracellular domains has
                            differentiated activation characteristics <FONT color="purple">(Figure 2c)</FONT>
                            in transient transfection experiments.
                            In all combinations, αCD19-mN1c-GV2 was not able to be activated efficiently
                            <FONT color="purple">(Figure 2c)</FONT>.
                            LaG16-2-mN1c-GV2 and αCD19-mN1c-tTAA have high activation and low background expression.
                            LaG16-2 was able to be efficiently activated only when activated with surEGFP but not surCD19
                            <FONT color="purple">(Figure 2d)</FONT>. The excellent orthogonality between LaG16-2 and
                            αCD19 ensures that we will
                            not undergo signal crosstalk when using dual surface antigens as signal inputs in the future.
                            In the following experiments, we used the LaG16-2-mN1c-GV2 and αCD19-mN1c-tTAA which has
                            good performance, as actual transmembrane signal transduction elements. We used mCherry-tagged
                            LaG16-2-mN1c-GV2 and EGFP-tagged αCD19-mN1c-tTAA to construct a stable cell line
                            co-expressing dual-SynNotch receptors for subsequent experiments.
                        </P><P>Meanwhile, as we mentioned above, some types of SynNotch have high background expression.
                            This phenomenon aroused our great interests. We designed a systematic experiment to study the
                            mechanism of SynNotch activation and hope to optimize it. We specifically covered the details
                            of this section in <A href="/Team:Fudan/Optimization">the Optimization page</A>.
                        </P><DIV class="expFigureHolder" style="width:80%;margin:0 auto;"><P><B>Figure 2. Engineering SynNotch can receive different extracellular signals and generate orthogonal
        intracellular signals.</B>
        a. Use surEGFP and surCD19 as the antigens. Upper left, surEGFP schematic. Green, surEGFP autofluorescence;
        red, surEGFP was stained with anti-GFP labeled with AF647. Red shows a clear contour line. Lower left, surCD19
        schematic. The extracellular region of surCD19 with HA tag was stained with anti-HA labeled with AF488.
        Scale bar, 5 μm.
        b. Schematic diagram of our SynNotch testing setting. Different extracellular scFvs and different intracellular
        transcription factors were used as extracellular or intracellular domains of SynNotch.
        c. SynNotch with different extracellular domain and intracellular domains has different activation strength.
        SynNotch expressing cells (receiver cells) were stimulated with excessive surface antigen (surAg) expressing sender cells or Mock cells
        carrying no surAg. The percentage of cells expressing d2EGFP (EGFP++%) was quantified by flow cytometry after
        24 hours of co-culture of sender and receiver cells.
        d. LaG16-2 and αCD19 were able to recognize surEGFP and surCD19 orthogonally. LaG16-2-mN1c-tTAA or
        αCD19-mN1c-tTAA was activated by surEGFP or surCD19, respectively, and TRE3GV-d2EGFP was used as the
        downstream reporter. The median fluorence intensity of receiver cells were color coded on the right.
        The error bar indicates SD of biological replicates, n=3.
    </P></DIV><P>The ectodomain of wild type Notch is composed of 29-36 modular epidermal growth factor (EGF)-like
                            repeats, each around 40 residues in length and can be categorized into calcium binders and
                            non-binders. Though former studies have assumed the shape of this ectodomain to be a long
                            rod-like structure, taking a linear topology
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/19379690" target="_blank">(Kopan and Ilagna,2009)</A>,
                            recent studies have elucidated that there are multiple rods joined by flexible joints
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/25503468" target="_blank">(Weisshuhn et al., 2015a,</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/25930016" target="_blank">Weisshuhn et al., 2015b)</A>.
                        </P><H3>Signal transduction of the Notch core</H3><P>The signal transduction mechanism of Notch signaling is very unique. While conserved signaling
                            pathways are activated at the efforts of receptors, Notch receptor signaling and proteolysis
                            is caused by the bing of corresponding ligands.</P><P>One interesting feature of Notch signaling is that its intracellular domains, released by
                            intramembrane proteolysis during activation, will translocate to the cell nucleus
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/15485896" target="_blank">(Schroeter EH et al., 1998)</A>
                            This renders almost no amplification of the signal, and signal orthogonality.
                        </P><P>Notch specific ligands are in constant endocytosis-surface presenting cycle in the sender
                            cell. When the N terminal recognition domain of Notch is bound to the ligand, this would
                            produce resistance that will stimulate the ubiquitylation of the ligand and also enhance
                            the recruitment of ubiquitin-binding Epsin endocytic adaptors
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/21763614" target="_blank">(Weinmaster G. 2011)</A>.
                            Complementary to this model, Ligand and cells defective in endocytosis cannot activate Notch
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/17296795" target="_blank">(Nichols JT et al., 2007)</A>.
                            This is like a cellular level <I>tug-of-war</I> between the two surface proteins, with the
                            Notch receptor eventually giving up. When Notch extracellular domain is not bound by ligands
                            with enough affinity or is under disturbances from small molecules, Notch 1 receptor will not
                            activate because its negative regulatory region (NRR) will refrain it from being cleaved.
    </P><DIV class="expFigureHolder" style="width:60%;margin: 0 auto;"><P><B>Figure 3. Structure of the negative regulatory region of the human Notch 1 receptor.</B>
    Adapted from <A href="https://www.ncbi.nlm.nih.gov/pubmed/19075186" target="_blank">Gordon et al., 2009a</A>
    and <A href="https://www.ncbi.nlm.nih.gov/pubmed/19701457" target="_blank">Gordon et al., 2009b</A>. Rendered using
    PDB structure <I>3eto</I>.
</P></DIV><P>Notch core consists of the negatively regulated region and transmembrane region of the mouse
                            Notch 1 receptor. It is the joint that empowers SynNotch the high modularity and proper
                            activation 需要参考文献. From the N terminal to C terminal, it is composed of three
                            lin12-repeats (LNR) domains LNR-A, LNR-B, LNR-C, the heterodimerization domain (HD),
                            and the transmembrane domain. The LNR-AB linker between LNR-A and B is like a plug that
                            occludes ADAM from cleaving wild type extracellular domain can be substituted by anti-GFP
                            single-chain antibodies like LaG16 and αCD19, while its intracellular domain can be
                            replaced by specialized transcription factors. Notch core is the key to the activation
                            of both wild type Notch receptors and SynNotch. Lin-12 repeats (LNR)
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/12795601" target="_blank">(Vardar et al. 2003)</A>.
                        </P><P>The majority of Notch receptors are cleaved during the secretory pathway by furin-like
                            convertase at cleavage site 1 (S1)
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/9653148" target="_blank">(Logeat et al., 1998)</A>,
                            which protrudes as a part of a linker of the two parts of the Notch negative regulatory
                            region's heterodimerization domain <A href="https://www.ncbi.nlm.nih.gov/pubmed/19075186" target="_blank">(Gordon et al., 2009a;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/17401372" target="_blank">Gordon et al., 2007)</A>.
                            This action separates Notch extracellular domain and the Notch transmembrane domain, but
                            noncovalent interactions between the heterodimerization domains hold the receptor together.
                            Despite disparate results on the role of S1 cleavage is a prerequisite for later activation,
                            it has been suggested that it is not required for contact-dependent activation
                            on the cell membrane <A href="https://www.ncbi.nlm.nih.gov/pubmed/19701457" target="_blank">(Gordon et al., 2009b)</A>.
                        </P><P>When ligand binds to the Notch extracellular domain (NECD) and pulls it with mechanical force
                            and thereby activating it, it initiates a process called regulated intramembrane proteolysis (RIP)
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/9245493" target="_blank">(Weinmaster et al., 1997</A>;
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/10882063" target="_blank">Brou et al. 2000;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26051539" target="_blank">Gordon et al., 2015)</A>
                            In this process, the Notch receptor is first cleaved at S2 site by two types of metalloproteases in the
                            ADAM family (a disintegrin and metalloprotease), ADAM10 or ADAM17, with ADAM10 the dominant
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/10882063" target="_blank">(Brou et al., 2000;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/11112321" target="_blank">Mumm et al., 2000;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/19704010" target="_blank">Bozkulak et al., 2009)</A>.
                            The remaining truncated transmembrane subunit is subsequently cleaved by γ-secretase at S3 site,
                            and results in the releasing the Notch intracellular domain (NICD). The wild type NICD will
                            then enter the nucleus and interact with CSL and MAML to activate transcription
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/9620803" target="_blank">(Schroteter et al., 1998;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/9604939" target="_blank">Struhl et al., 1998)</A>.
                        </P><H3>What is different in transcriptional signal output?
                        </H3><P>
                            The wild type NICD will enter the nucleus and interact with CSL and MAML
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/9620803" target="_blank">(Schroteter et al., 1998;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/9604939" target="_blank">Struhl et al., 1998)</A>
                            to regulate endogenous pathways that decide cell fate. The customizable output of SynNotch receptors
                            have been integrated with chimeric antigen receptors to engineer T cells with customizable
                            therapeutic response programs <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830879" target="_blank">(Roybal et al., 2016a;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/27693353" target="_blank">Roybal et al., 2016b)</A>, or to
                            program self-organizing multicellular structures
                            <A href="https://www.ncbi.nlm.nih.gov/pubmed/29853554" target="_blank">(Toda et al., 2018)</A>.
                            In our ENABLE project, we have used two types of SynNotchs in order to empower mammalian cells
                            the ability of binary computation. This promises more novel application potentials. Please
                            continue the reading of <A href="/Team:Fudan/Results">our Results</A> on intracellular binary logic gates.
                        </P></DIV><DIV class="section container"><H4 style="margin-left: 0;">References 下面完全是凌乱的</H4><OL id="ref" style="margin: 23px 0 0 0;"><LI>Aster J C, Pear W S, Blacklow S C. Notch signaling in leukemia[J]. Annu. Rev. pathmechdis. Mech. Dis., 2008, 3: 587-613.
                            </LI><LI>
                                Bozkulak, Esra Cagavi, and Gerry Weinmaster. &quot;Selective use of ADAM10 and ADAM17 in activation of Notch1 signaling.&quot; Molecular and cellular biology 29.21 (2009): 5679-5695.

                        </LI><LI>     Bray, Sarah J. &quot;Notch signalling in context.&quot; Nature reviews Molecular cell biology 17.11 (2016): 722.

                        </LI><LI>     Brou C, Logeat F, Gupta N, Bessia C, LeBail O, et al. (2000) A novel proteolytic cleavage involved in Notch signaling: the role of the disintegrin-metalloprotease TACE. Mol Cell 5: 207–216.

                        </LI><LI>    Chiba S. Concise review: Notch signaling in stem cell systems[J]. Stem cells, 2006, 24(11): 2437-2447.

                        </LI><LI>    Fehon, R.G. et al. Molecular interactions between the protein products of the neurogenic loci Notch and Delta, two EGF-homologous genes in Drosophila. Cell 61, 523–534 (1990).

                        </LI><LI>   Fridy, P. C. et al. A robust pipeline for rapid production of versatile nanobody repertoires. Nature methods 11, 1253--1260 (2014).

                        </LI><LI>    Gordon WR, Roy M, Vardar-Ulu D, Garfinkel M, Mansour MR, et al. (2009) Structure of the Notch1-negative regulatory region: implications for normal activation and pathogenic signaling in T-ALL. Blood 113: 4381–4390.

                        </LI><LI>   Gordon WR, Vardar-Ulu D, Histen G, Sanchez-Irizarry C, Aster JC, et al.(2007) Structural basis for autoinhibition of Notch. Nat Struct Mol Biol 14: 295–300.

                        </LI><LI>    A: Gordon, Wendy R., et al. &quot;Structure of the Notch1-negative regulatory region: implications for normal activation and pathogenic signaling in T-ALL.&quot; Blood 113.18 (2009): 4381-4390.
                        </LI><LI>     B: Gordon, Wendy R., et al. &quot;Effects of S1 cleavage on the structure, surface export, and signaling activity of human Notch1 and Notch2.&quot; PloS one 4.8 (2009): e6613.

                        </LI><LI>   Gordon, Wendy R., et al. &quot;Mechanical allostery: evidence for a force requirement in the proteolytic activation of Notch.&quot; Developmental cell 33.6 (2015): 729-736.

                        </LI><LI>   Greenwald, Iva, and Geraldine Seydoux. &quot;Analysis of gain-of-function mutations of the lin-12 gene of Caenorhabditis elegans.&quot; Nature 346.6280 (1990): 197.

                        </LI><LI>  Grupp, S. A. et al. Chimeric antigen receptor-modified T cells for acute lymphoid leukemia. New England Journal of Medicine 368, 1509--1518 (2013).

                        </LI><LI>   Kovall R A, Gebelein B, Sprinzak D, et al. The canonical Notch signaling pathway: structural and biochemical insights into shape, sugar, and force[J]. Developmental cell, 2017, 41(3): 228-241.

                        </LI><LI>     Kopan, R., and Ilagan, M.X. (2009). The canonical notch signaling pathway: unfolding the activation mechanism. Cell 137, 216–233.

                        </LI><LI>     Logeat, F. et al. The Notch1 receptor is cleaved constitutively by a furin-like convertase. Proceedings of the National Academy of Sciences 95, 8108-8112 (1998).

                        </LI><LI>  Mansour, M. R., et al. &quot;High incidence of Notch-1 mutations in adult patients with T-cell acute lymphoblastic leukemia.&quot; Leukemia 20.3 (2006): 537.

                        </LI><LI>   Mumm J S, Kopan R. Notch signaling: from the outside in[J]. Developmental biology, 2000, 228(2): 151-16

                        </LI><LI>   Mumm JS, Schroeter EH, Saxena MT, Griesemer A, Tian X, et al. (2000) A ligand-induced extracellular cleavage regulates gamma-secretase-like proteolytic activation of Notch1. Mol Cell 5: 197–206.

                        </LI><LI>    Nichols JT, Miyamoto A, Olsen SL, D’Souza B, Yao C, Weinmaster G. DSL ligand endocytosis physically dissociates Notch1 heterodimers before activating proteolysis can occur. J Cell Biol 2007;176:445–58.


                        </LI><LI>   Rand, Matthew D., et al. &quot;Calcium depletion dissociates and activates heterodimeric notch receptors.&quot; Molecular and cellular biology 20.5 (2000): 1825-1835.

                        </LI><LI>   Sanchez-Irizarry, Cheryll, et al. &quot;Notch subunit heterodimerization and prevention of ligand-independent proteolytic activation depend, respectively, on a novel domain and the LNR repeats.&quot; Molecular and cellular biology 24.21 (2004): 9265-9273.

                        </LI><LI>   Schroeter EH, Kisslinger JA, Kopan R. Notch-1 signalling requires ligand-induced proteolytic re- lease of intracellular domain. Nature. 1998;393: 382-386.

                        </LI><LI>   Struhl, G. &amp; Adachi, A. Nuclear access and action of notch in vivo. Cell 93, 649–660 (1998).

                        </LI><LI>   Vardar, Didem, et al. &quot;Nuclear magnetic resonance structure of a prototype Lin12-Notch repeat module from human Notch1.&quot; Biochemistry 42.23 (2003): 7061-7067.

                        </LI><LI>   Weinmaster, G. The ins and outs of notch signaling. Mol. Cell. Neuroscience. 9, 91–102 (1997).
                        </LI><LI>   Weinmaster G, Fischer JA. Notch ligand ubiquitylation: what is it good for. Dev Cell 2011;21:134–44.

                        </LI><LI>   Weisshuhn, P.C., Handford, P.A., and Redﬁeld, C. (2015a). (1)H, (13)C and (15)N assignments of EGF domains 4 to 7 of human Notch-1. Biomol. NMR Assign. 9, 275–279.

                        </LI><LI>    Weisshuhn, P.C., Handford, P.A., and Redﬁeld, C. (2015b). (1)H, (13)C and (15)N assignments of EGF domains 8-11 of human Notch-1. Biomol. NMR Assign. 9, 375–379.

                        </LI><LI>   Weisshuhn, P.C., Sheppard, D., Taylor, P., Whiteman, P., Lea, S.M., Handford, P.A., and Redﬁeld, C. (2016). Non-linear and ﬂexible regions of the human Notch1 extracellular domain revealed by high-resolution structural studies. Structure 24, 555–566.
                        </LI><LI>
                                Cho J H, Okuma A, Al-Rubaye D, et al. Engineering Axl specific CAR and SynNotch receptor for cancer therapy[J]. Scientific reports, 2018, 8(1): 3846.

                        </LI><LI>  Morsut L, Roybal K T, Xiong X, et al. Engineering customized cell sensing and response behaviors using synthetic notch receptors[J]. Cell, 2016, 164(4): 780-791.
                                 
                        </LI><LI>      Roybal K T, Williams J Z, Morsut L, et al. Engineering T cells with customized therapeutic response programs using synthetic notch receptors[J]. Cell, 2016, 167(2): 419-432. e16.
                                 
                        </LI><LI>     Roybal K T, Rupp L J, Morsut L, et al. Precision tumor recognition by T cells with combinatorial antigen-sensing circuits[J]. Cell, 2016, 164(4): 770-779.
                                 
                        </LI><LI>     Toda S, Blauch L R, Tang S K Y, et al. Programming self-organizing multicellular structures with synthetic cell-cell signaling[J]. Science, 2018: eaat0271.

                            </LI></OL></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;padding: 10px 0;">Abstract</H2><P style="margin: 0">Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4 active"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                           </DIV><DIV class="col s12 m6 l2"> +86-21-31246727
                           </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                           </DIV><DIV class="col s12 m6 l4"> Fudan_iGEM
                           </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>