<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Measurement skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Measurement</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Measurement</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right orangeBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Quantification</SPAN></P></DIV></LI><LI>Navigator on this page</LI><LI class="onThisPageNav"><A href="#section1">Introduction</A></LI><LI class="onThisPageNav"><A href="#section2">Method</A></LI><LI class="onThisPageNav"><A href="#section3">Results</A></LI><LI class="onThisPageNav"><A href="#section4">Discussion</A></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner orangeBg"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Quantification</H1></DIV><DIV class="col s12 hide-on-med-and-up"><SPAN>A dual-fluorescence dual-plasmid quantification system was used to rapidly and robustly measure the performance of synthetic transcription factor-promoter pairs.</SPAN></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Quantification</H1><P><SPAN>A dual-fluorescence dual-plasmid quantification system was used to rapidly and robustly measure the performance of synthetic transcription factor-promoter pairs.</SPAN></P></DIV></DIV><DIV class="container"><UL id="pageContentNav" class="hide-on-med-and-down z-depth-0"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI>Quantification</LI><LI class="onThisPageNav"><A href="#section1">Introduction</A></LI><LI class="onThisPageNav"><A href="#section2">Method</A></LI><LI class="onThisPageNav"><A href="#section3">Results</A></LI><LI class="onThisPageNav"><A href="#section4">Discussion</A></LI></UL><MAIN><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to summarize our measurement used for receptor optimization and logic gate characterization.</P></DIV><DIV id="section1" class="section container scrolSpy"><H2>Introduction</H2><P>
                                Transcription factors and promoters are the basis of the genetic circuit. Compared with prokaryotic promoters, the mechanism of action of eukaryotic promoters is complex <A href="https://www.ncbi.nlm.nih.gov/pubmed/10412974" target="_blank">(Struhl, 1999)</A>, which increases the difficulty of our new design and transformation. The limited number of eukaryotic transcription factors and promoters is currently limiting the design and implementation of complex gene circuits in mammalian cells from meeting a variety of clinical or industrial applications.
                            </P><P>
                                In order to obtain a sufficient number of orthogonal synthetic transcription factor-promoter pairs, it is often necessary to carry out a certain number of constructions and select candidates with an excellent performance from the complex construction. This requires us to develop a robust, fast and cost-controllable method suitable for large-scale screening. Therefore, testing using transient transfection methods is an effective method.
                            </P><DIV class="expFigureHolder width60 left-on-med-and-up"><P><B>Figure 1. Using a dual fluorescence system to effectively remove data-to-data differences due to differences in transient efficiency.
                                </B>
                                (a) Testing with a single fluorescence system is susceptible to transient effects resulting in measurement errors. U, untransfected cell; T, transfected cell. The two distinct peak shapes here are only for convenience of display. In actual tests, it is often difficult to distinguish a clear trough because of the copy number variability of the transiently transferred plasmid. This effect is more pronounced when a promoter has a higher background expression or an unstable expression.
                                (b) Testing with dual fluorescence system can effectively eliminate the difference in transfection efficiency. Orange indicates more cell aggregation in this area, and yellow indicates less cell aggregation in this area.
                            </P></DIV><P>Since transient transfection experiments will introduce a heterogeneous number of plasmid copies into cells <A href="https://www.ncbi.nlm.nih.gov/pubmed/19211029" target="_blank">(Cohen et al., 2009)</A>, the differences between experiments will be expanded compared to stable expression cell lines. We found that a reasonable way to distinguish successfully transfected cells is to maintain test robustness while quantifying the promoter. As an example taken from a group of experiments, when a transcription factor and its corresponding promoter which controls a downstream reporter gene are co-transfected, transient transfection will result in a high degree of heterogeneity in the fluorescent expression of the cells. If it is impossible to gate out cells that have been unsuccessfully transfected by any means, the transfection efficiency of each change in each experiment will make these untransfected cells have a catastrophic effect on the statistical results <FONT color="”purple”">(Figure 1a)</FONT>. However, by introducing an additional fluorescent signal, we can clearly distinguish cells that have been successfully transfected <FONT color="”purple”"> (Figure 1b) </FONT>. Thus, this method contains important implementation value.
  </P><P>In transient transfection, the introduction of additional plasmids may result in that not all cells are able to take up all of the components of the plasmid set. Thus, the use of such a method does not effectively distinguish whether those cells are successfully transferred to all of the plasmid groups based on the presence or absence of the internal reference fluorescence. However, in contrast, the success rate of co-transfection can be significantly improved by reducing the amount of plasmid used.
                            </P><P>Therefore, we developed a dual-fluorescence dual-plasmid (DFDP) quantification system (DFDP), in which we keep only two plasmids that allow us to confidently perform a co-transfection experiment. Simultaneously, the P2A element couples the expression of the internal reference fluorescence with transcription factor (TF) so that we can gate out the cells that have not been successfully transferred into the plasmid by distinguishing the presence or absence of one of the fluorescences. Thereby we can minimize the interference caused by utilizing the difference in transfection efficiency between the repeats. Thus, DFDP quantification system helps us measure transcription factor-promoter pair strength quickly and robustly.
                            </P><P>Here we summarize the comparison of using a 2A element to construct DFDP relative to co-transfecting an internal reference plasmid (Table 1).
                            </P><DIV class="tableHolder"><TABLE><TBODY><TR style="border-top: #AAAAAA solid 1px"><TH>Dual-fluorescence dual-plasmid (DFDP)</TH><TH>Co-transfecting an internal reference fluorescence</TH></TR><TR><TD rowspan="2">Pro</TD><TD>Using only two plasmids for co-transfection experiments, it is better to ensure that the fluorescence expressed in association with TF is able to indicate that the cell already contains two other plasmids.</TD><TD rowspan="2">Molecular cloning may be relatively simple due to the elimination of the need to construct a fusion protein.
                                    </TD></TR><TR><TD>Using 2A to link TF and fluorescent protein, you can roughly understand the expression level of transcription factor by mCherry intensity.</TD></TR><TR><TD>Con</TD><TD>
                                        Need to construct a fusion protein, so molecular cloning requires some experience (this can be improved by our careful plasmid design).
                                    </TD><TD>
                                        Due to the need to introduce an additional plasmid, co-transfection with 3 plasmids (even if 4 plasmids are required for co-transfection if the NIMPLY-form promoter is to be tested) actually reduces the efficiency of transfection. It is not guaranteed that cells with internal reference fluorescence do contain the two or three other plasmids.
                                    </TD></TR><CAPTION style="text-align:left">Table 1. Comparison of advantages and disadvantages between DFDP and the method of co-transfecting an internal reference fluorescence.


                                    </CAPTION></TBODY></TABLE></DIV></DIV><DIV id="section2" class="section container scrolSpy"><H2>Method</H2><H3>The design of DFDP
                            </H3><P>
                                Based on theoretical assumptions (see <A href="#section5">Supplementary Materials</A> for details), we designed a dual-fluorescence-double plasmid (DFDP) system containing two plasmids <FONT color="”purple”">(Figure 2)</FONT>. Spectral overlap problems in multicolor fluorescence signal measurements can interfere with the effectiveness of the measurement.
                                We used mCherry and d2EGFP here as two dual fluorescent signals because of the minimal overlap of their spectra.
                            </P><DIV class="expFigureHolder" style="width: 100%"><P><B>Figure 2. Dual-fluorescence dual-plasmid (DFDP) assay.</B>
                                    Structure of the plasmid with TF (P-TF): conPro-mCherry-P2A-SynTF. mCherry and TF are in one open reading frame linked by P2A and controlled by CMV promoter. P2A is a high efficiency self-cleaving 2A peptide
                                    <A href="https://www.ncbi.nlm.nih.gov/pubmed/21602908" target="_blank">(Kim et al., 2011)</A>. P2A are thought to function by making the ribosome skip the synthesis of a peptide bond at the C-terminus element, leading to “cleavage” between the end of the 2A sequence and and the next peptide downstream. The efficiency of cleavage can be improved by adding the Gly-Ser-Gly sequence at the N-terminus of the 2A peptide <A href="https://www.ncbi.nlm.nih.gov/pubmed/15064769" target="_blank">(Szymczak et al., 2004)</A>. Cleavage occurs between the Gly and Pro residues on the C-terminus causing the upstream cistron will have a few additional residues added to the C-terminus, while the downstream cistron will start with an additional Proline. Another advantage of using the 2A structure is that each molecule of mCherry expresses one molecule of TF at the same time, so that the relative expression level of TF can be indirectly known by detecting the fluorescence intensity of mCherry (see <A href="#section5">Supplementary Materials</A> for details).
                                    Structure of the plasmid with Pro (P-Pro): Pro-d2EGFP. Promoter drives the expression of d2EGFP, which allows us to indirectly reflect the expression intensity of Pro in different states (for example, conditions with the presence of TF and conditions without the presence of TF) by detecting the fluorescence intensity of d2EGFP (see <A href="#section5">Supplementary Materials</A> for details). At the same time, since our theoretical assumptions is derived under steady-state conditions, the key to the establishment of this test is that [d2EGFP] has reached steady state at the time of testing. Therefore, it is important to speed up the system to achieve fast response time. The response time can be accelerated by accelerating the rate of protein degradation
                                    <A href="https://www.ncbi.nlm.nih.gov/pubmed/12787666" target="_blank">(Rosenfeld and Alon, 2003)</A>. Therefore, compared to the traditional fluorescence reporting system, we used rapidly degradable EGFP (d2EGFP)
                                    <A href="https://www.ncbi.nlm.nih.gov/pubmed/9857028" target="_blank">(Li et al., 1998)</A> as a surrogate for EGFP. Comparing with EGFP, d2EGFP has a faster degradation rate, which will make promoter-driven fluorescence expression reach steady state faster, ensuring that we are as steady as possible in each test.
                                </P></DIV><H3>Optimize DHDP plasmid vector backbone for easier molecular cloning</H3><P>In our design, the P-TF plasmid requires the construction of a fusion protein, we specifically designed p2Y-C-mCherry-P2A as the basic skeleton for P-TF construction. We also designed multiple cloning sites (MCS) at the mCherry-P2A backend and we inserted a BamHI site after the last Pro on the C-terminus of P2A. Furthermore, TF can be inserted directly after digestion with BamHI and HindIII. In addition to the restriction enzyme digestion method, we inserted two BbsI sites after the 3' end of BamHI and 5' of HindIII to meet the potential requirements of the Golden Gate cloning method while the restriction sites were specially designed. Make the same as the cleavage site of BamHI and HindIII. A stop codon was also inserted between the two BbsI sites, such that the p2Y-C-mCherry-P2A backbone without the insertion of TF was normally terminated after transcription initiation. Therefore, the p2Y-C-mCherry-P2A plasmid can be used as a control plasmid (P-Ctrl). We also optimized the base sequence after the 5' end of BamHI and the 3' end of HindIII to reduce the hairpin structure which makes it very easy to use when using ClonExpress or Gibson assembly. Although insertion of the BamHI site will result in three amino acids at the N-terminal residual (Pro-Gly-Ser) of the TF; yet as far as we know, this does not affect the function of SynTF. While, you can also use the ClonExpress or NEBuilder cloning method to remove the BamHI site during assembly, leaving only one Pro.
                            </P><P>Molecular cloning of the plasmids based on this specially optimized backbone system enables DFDP, making much easier and more convenient like never before.
                            </P><P>We use the ClonExpress (Vazyme #C115) or the standard BioBrick assembly for all plasmid assembly.
                            </P><P>Download the file to view our MCS design.
<A href="https://static.igem.org/mediawiki/2018/b/ba/T--Fudan--Measurement-MCS.txt" target="_blank"><I class="fa fa-download">MCS</I></A></P><H3>Cell culture</H3><P>293T (ATCC #CRL-3216) was cultured in DMEM (Gibco #11965092) supplemented with 10% fetal bovine serum (Gibco # 10437036), 1x Penicillin-Streptomycin and GlutaMax.
                            </P><H3>Transient transfection</H3><P>Transfection was performed using the cost-effective cationic polymer polyethyleneimine Max (Polysciences #24765). All transient transfection experiments of cells were performed in 24-well plates where the cells were plated one day earlier and transfected the next day at 80% cell confluence. For a well, 1.5 µl 1 µg/µl were added into 23.5 µl Opti-MEM (Thermo Fisher Scientific #11058021). A total of 500 or 450 ng plasmid was added into 25 µl Opti-MEM for 5 min. PEI-Opti-MEM and DNA-Opti-MEM mixture were mixed and placed at room temperature for 10 min before being added into the well.
                            </P><P>The amount of plasmids used for activating-, silencing-, NIMPLY-form promoter tests is shown in <FONT color="”purple”">Table 2</FONT>.
                            </P><DIV class="tableHolder"><TABLE class=""><TBODY><TR><TH>-TF group</TH><TH>+aTF group</TH><TH>+sTF group</TH><TH>+aTF +sTF (both) group</TH></TR><TR><TD>aTF-aPro
                                        </TD><TD>P-Ctrl = 250 ngP-aPro = 250 ng</TD><TD>P-aTF = 250 ngP-aPro = 250 ng</TD></TR><TR><TD>sTF-sPro
                                        </TD><TD>P-Ctrl = 250 ngP-sPro = 250 ng</TD><TD>P-sTF = 250 ngP-sPro = 250 ng</TD></TR><TR><TD>aTF-aPros-sTF
                                        </TD><TD>P-Ctrl = 300 ngP-aPros = 150 ng</TD><TD>P-Ctrl = 150 ngP-aTF = 150 ngP-aPros = 150 ng</TD><TD>P-Ctrl = 150 ngP-sTF = 150 ngP-aPros = 150 ng</TD><TD>P-aTF = 150 ngP-sTF = 150 ngP-aPros = 150 ng</TD></TR><CAPTION style="text-align:left">Table 2. Plasmids for activating-, silencing-, NIMPLY-form promoter test.

                                    </CAPTION></TBODY></TABLE></DIV><H3>Flow cytometry</H3><P>All the flow cytometry data is acquired by FACSJazz (BD Biosciences). After 48 hours of transfection, the cells were fixed with 4% paraformaldehyde for 15 min, and washed twice with PBS before analysis.
                                Keep in dark as much as possible between operations. The d2EGFP is recorded under the excitation laser of 488 nm with a 530/40 emission filter while the mCherry is recorded under the excitation laser of 561 nm with a 610/20 emission filter.
                                The photomultiplier tube (PMT) amplification values for FSC, SSC, EGFP, mCherry channel are 28, 28, 28 and 44 separately.
                                Overall, the spectral overlap between EGFP and mCherry is very small; emissions of EGFP and mCherry activated by 2 lasers separated well so no compensation is needed.
                            </P><H3>Data analysis</H3><P>Three biological replicates were performed for each experiment.
                                Flow cytometry data are analyzed by FlowJo 10 (TreeStar). Three basic hierarchical populations are gated: FSC-SSC → FSC-Trigger pulse width → 488_530/40-561_610/20. Successfully transfected cells are gated by mCherry positive for the same group. MFI represents the median fluorescence intensity of EGFP in the top 50% of cells with the strongest expression of d2EGFP. Different biological replicates were normalized using the fluorescence intensity of mCherry. For the Fold value, it is calculated according to the formula in the theoretical hypothesis (see <A href="#section5">Supplementary Materials</A> for details). All statistical analysis was performed with Prism 7 (Graphpad).
                            </P></DIV><DIV id="section3" class="section container scrolSpy"><H2>Results</H2><H3>Testing activating-form promoters using DFDP</H3><P>We used the artificial zinc finger protein (Synthetic Zinc finger, SynZF)
                                <A href="https://www.ncbi.nlm.nih.gov/pubmed/22863014" target="_blank">(Khalil et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22323524" target="_blank">Lohmueller et al., 2012)</A>  to construct three mammalian-adapted DNA binding domains (DBD) of ZF21.16, ZF42.10, and ZF43.8, based on
                                <A href="https://2017.igem.org/Team:Fudan" target="_blank">iGEM 2017 Fudan</A> standardized synthetic transcription factors (SynTF) – synthetic promoter (SynPro) construction paradigm. The tests using DFDP showed that the ZF21.16-, ZF42.10-, ZF43.8-VP64 had good activation characteristics <FONT color="”purple”">(Figure 3)</FONT>, and the activation magnification ratio ranged from 51 to 454 fold. Among them, 8×ZF21.16-minCMV had the largest activation ratio (454 fold), while 8×ZF43.8-minCMV had the smallest activation ratio (51 fold). The difference between them is mainly due to the relatively high background expression of 8×ZF43.8-minCMV. Thus, we believe that 8×ZF21.16-minCMV is a
                                better promoter (for our opinion on excellent promoters, please see the discussion section).
                            </P><DIV class="expFigureHolder row width80" style="margin: 23px auto 0 auto;"><DIV class="col s12 m6"><P><B>Figure 3. IDENTITY-form promoter.</B>
                                The diagram located at the top is the IDENTITY-form promoter working mechanism diagram. The activating-form transcription factor (aTF) is formed by a DNA binding domain (DBD) and a transcriptional activation domain (AD) linked via a linker. The IDENTITY-form promoter is just an activating-form promoter (aPro) because it is expressed only in the presence of aTFs. The aPro is constructed by inserting a plurality of response elements corresponding to aTF (aTF REs) before the minimal promoter (minPro, eg minCMV). At the bottom, DFDP was used to test the aPros. It can be seen that the aPro is fully activated in the presence of its corresponding aTF (n = 3, error bar, SD).
                            </P></DIV></DIV><H3>
                                Testing the orthogonality between synthetic DBDs using cross-paired DFDP
                            </H3><DIV class="expFigureHolder width40 right-on-med-and-up"><P><B>
                                    Figure 4. testing of the Orthogonality of aTF-aPro pairs by using the cross-paired DFDP.</B>
                                The five aTF-aPro pairs have good orthogonality.
                            </P></DIV><P>
                                Cross-paired DFDP can be used to verify promoter orthogonality by using the co-transfection strategy of P-TF with both corresponding or uncorresponding P-Pro. Since there are 5 aSynTF-aSynPro pairs, 25 (5 x 5) sets of experiments are required. The intensity of d2EGFP under different conditions was examined, and it was observed that the group on the diagonal
                                (the correctly paired group) had the most significant expression <FONT color="”purple”">(Figure 4)</FONT>, indicating that the 5 pairs of aSynTF-sSynProdui pairs were orthogonal to each other. The advantage of using the activated promoter for DNA binding domain orthogonality testing over the use of sSynTF-sSynPro testing is that there is a difference in the basal expression of sSynPro so more normalization is required to rule out differences in basic expression. Abnormal results:
                            </P><H3>
                                Testing silencing-form promoters using DFDP
                            </H3><P>
                                Based on the SynTF of <A href="https://2017.igem.org/Team:Fudan" target="_blank">iGEM 2017 Fudan</A> (see the <FONT color="”purple”">improvement</FONT> page), we adjusted the position of the sSynTF corresponding response element (sTF REs) from the 3' end of the promoter to the 5' end, and replaced the original pSV40 with a stronger CMV promoter to serve as the constitutive expression promoter (conPro). Then we constructed a new generation of inhibitory transcription factors <FONT color="”purple”">(Figure 5)</FONT>. Such a construction helps to attenuate the interference of sTF REs on the under expression of conPro. Tests using DFDP show that ZF21.16-, ZF42.10-, ZF43.8-KRAB have good inhibition characteristics, with the magnification ranging from 9 to 13 times <FONT color="”purple”">(Figure 5)</FONT>. The reason why aSynTF-aSynPro differs by a hundred fold while only having a difference of about tenfold here does not mean that our sSynPro function is not good. Rather, it's because the basic expression of aSynPro is particularly weak, and it was close to the minimum of our flow cytometry detection at the time of testing.
                            </P><DIV class="expFigureHolder row width80" style="margin: 23px auto 0 auto;"><DIV class="col s12 m6"><P><B>Figure 5. NOT-form promoter.</B>
                                Above is the NOT-form promoter working mechanism diagram. Silencing-form transcription factors (sTFs) are formed by a (DBD) and a transcriptional silencing domain (SD, eg KRAB) linked via a linker. The NOT-form promoter is just a silencing-form promoter (sPro) because it is expressed only in the absence of sTFs. The sPro is structure by inserting a plurality of REs corresponding to sTFs (sTF REs) before a constitutive expression promoter (conPro, eg CMV). At the bottom, DFDP was used to test silencing-form promoters. It is observed that sPro are sufficiently inhibited in the presence of their corresponding inhibitory transcription factors. The dashed line indicates the intensity of expression of the CMV promoter under the same test conditions. Relative to the insertion of REs on the 3-terminus of conPro, the 5’-terminal design can reduce the interference to the basic expression of conPro (n = 3, error bar, SD).
                            </P></DIV></DIV><H3>
                                Testing NIMPLY-form promoters using DFDP
                            </H3><DIV class="expFigureHolder width45 right-on-med-and-up"><P><B>Figure 6. NIMPLY-form promoter. </B>
                                Top, the NIMPLY-form promoter working mechanism diagram. The NIMPLY-form promoter is highly expressed in the presence of only aTF and no sTF. When aTF and sTF coexist, sTF plays a major role. Bottom, using DFDP to Test NIMPLY-form promoters. 8×ZF21.16-minCMV-2×ZF43.8 is highly expressed only in the presence of ZF21.16-VP64 without ZF43.8-KRAB, while 8×ZF43.8-minCMV-2×ZF21.16 is only in High expression under conditions of ZF43.8-VP64 without ZF21.16-KRAB.
                            </P></DIV><P>
                                In addition to traditional activating- and silencing-from promoters, we have also designed a novel NIMPLY-form promoter where we added multiple REs corresponding to the activating- or silencing-form transcription factors at the 5' and 3' ends of minCMV, respectively. The dominant inhibition can be achieved by (1) KRAB recruitment of transcriptional inhibitors can inhibit promoter expression <A href="https://www.ncbi.nlm.nih.gov/pubmed/8183939" target="_blank">(Margolin et al., 1994)</A>, (2) inhibitory transcription factor binding to the REs. As sTF REs is located downstream of the promoter, when the inhibitory transcription factor binds to the REs, it can cause steric hindrance and enhance the ability of transcriptional inhibition by inhibiting the forward movement of RNA polymerase. This type of promoter exhibits the logical selectivity of NIMPLY under conditions of overexpression of aTFs and sTFs. However, it can only be expressed in the presence of aTFs and the absence of sTFs. 8xZF21.16-minCMV-2xZF43.8 can only be activated in the presence of ZF21.16-VP64 and in the absence of ZF43.8-KRAB. 8xZF43.8-minCMV-2xZF21.16 can only be activated in the presence of ZF43.8-VP64 and in the absence of ZF21.16-KRAB <FONT color="”purple”">(Figure 5)</FONT>. It can be observed that in the group transfected with both aTF and sTF, the expression of the NIMPLY-form promoter has increased a little because although sTF is dominant, it can slightly reverse sTF in the condition of an overexpressed aTF.
                            </P></DIV><DIV id="section4" class="section container scrolSpy"><H2>Discussion</H2><H3>What is a good promoter?</H3><P>The performance of a promoter is not always correlated to its expression. For some experiments that require a good control of protein expression,
                                scientists may not even insert a promoter on the viral vector and rely solely on the LTR element to maintain the protein at a weak expression
                                <A href="https://www.ncbi.nlm.nih.gov/pubmed/17350576" target="_blank">(Cai et al., 2007)</A>. It was also observed in our SynNotch experiment that comparing with using CMV to control SynNotch, by using a weak promoter of PGK can significantly reduce the basal expression (data not shown). For inducible promoters, we always expect that the activating-form promoter maintains very low background expression when not activated, and produces
                                increased expression when activated. Note that we do not believe that an activating-form promoter is activated as much as possible after being induced, indicating that it works well, and more importantly is whether the background expression of its background is low enough and whether it has sufficient tunability. Similarly, for silencing-form promoters, we always want to maintain stable expression under uninhibited conditions and can be shut down as much as possible after being inhibited. Similarly, we are more concerned about the stability of expression and the degree of inhibition.
                            </P><H3>Factors that interfere with DFDP</H3><P>The d2EGFP intensity was used as an indirect characterization of promoter strength in our test system, but it should be noted that d2EGFP stability may vary by cell type. Therefore, the comparison of d2EGFP intensity across cell types may not truly reflect differences in promoter strength in different cell types. Only the same cell type can be used as a common chassis for the promoter test. Under these conditions, the d2EGFP intensity driven by different promoters can be compared to reflect the difference between the strengths of the promoters. At the same time, for mammals, even the same constitutive promoter, its differential performance in different cell lines is still very significant.
                                For example, the CMV promoter is very active in some cell types (such as 293T cells). But in some other cell types (such as mesenchymal stem cells) it may be quite weak
                                <A href="https://www.ncbi.nlm.nih.gov/pubmed/20485554" target="_blank">(Qin et al., 2010)</A>. In our measurement protocol, we used 293T cells as chassis cells to screen for functional synthetic transcription factor-promoter pairs. If you need to migrate the potential synthetic transcription factor-promoter pair obtained from the initial screening to other cell lines, we strongly recommend you to use a commonly used transcription factor-promoter that works well in the
                                chosen cell line as a positive control. And then repeat DFDP test on your target cell line to qualitatively verify the synthetic transcription factor-promoter pair again.
                            </P><P>The success of multicolor flow cytometry depends on many hardware factors.
                                First, the shape and position of the excitation laser beam; second, the choice and quality of the optical filter; and third, the sensitivity and resolution of photoelectron detection
                                <A href="https://www.ncbi.nlm.nih.gov/pubmed/23138348" target="_blank">(Perfetto et al., 2012)</A>. The intensity and sensitivity of the excitation fluorescence of different flow cytometers have a certain degree of difference. To ensure that the data between each test is quantitatively comparable, the same flow cytometer should be used for the test. Quality control is required prior to each analysis even if using the same flow cytometer.
                                If multiple flow machines were used, automatic and manual calibrations between the flow are required.
                                Dual fluorescent signals should be received using the same optical channel (filter configuration) and should ensure that the sensor-photomultiplying tubes (PMTs) for each channel set the same gain for each test. An unfixed PMT setting will destroy your experiment.
                                If a flow cytometer with user adjustable PMT value is used, it is important to select and set an appropriate PMT value for the subsequent experimental needs during the initial experiment.
                                We recommend the following. When you first start your synthetic transcription factor-promoter pair measurement, you could set up a strong double positive cell to determin the proper PMT setting. For Example, we use CMV-mCherry and CMV-d2EGFP in our experiment and add just the two channels to ensure that the brightest group of cells are within the maximum detecting range of the flow cytometer.
                            </P><P>Transient transfection efficiency will interfere with comparability between different experiments. Different transfection reagents, reagent dosages, as well as cell density at the time of transfection, cell status, and the duration of transfection can significantly affect the efficiency of transfection. So you should control these variables as much as possible during the test. For example, try to use the same batch of transfection reagents; optimize the transfection efficiency and then use the same amount of transfection reagent and fix the ratio between the DNA and the transfection reagent; use the same transfection density; use the cells with low passage number; and fix the cell at the same time after transfection for future Flow analysis.
                            </P></DIV><DIV id="section5" class="section container"><H2>Supplementary Materials</H2><UL class="collapsible expandable"><LI style="list-style-type: none"><DIV class="collapsible-header">Click to expand Supplementary Materials</DIV></LI></UL></DIV><DIV class="section container"><H4 style="margin-left: 0;">References</H4><OL id="ref" style="margin: 23px 0 0 0;"><LI>Cai, L., Marshall, T.W., Uetrecht, A.C., Schafer, D.A., and Bear, J.E.J.C. (2007). Coronin 1B coordinates Arp2/3 complex and cofilin activities at the leading edge.  128, 915-929.</LI><LI>Cohen, R.N., van der Aa, M.A., Macaraeg, N., Lee, A.P., and Szoka Jr, F.C.J.J.o.C.R. (2009). Quantification of plasmid DNA copies in the nucleus after lipoplex and polyplex transfection.  135, 166-174.</LI><LI>Khalil, A.S., Lu, T.K., Bashor, C.J., Ramirez, C.L., Pyenson, N.C., Joung, J.K., and Collins, J.J. (2012). A synthetic biology framework for programming eukaryotic transcription functions. Cell 150, 647--658.</LI><LI>Kim, J.H., Lee, S.-R., Li, L.-H., Park, H.-J., Park, J.-H., Lee, K.Y., Kim, M.-K., Shin, B.A., and Choi, S.-Y. (2011). High cleavage efficiency of a 2A peptide derived from porcine teschovirus-1 in human cell lines, zebrafish and mice. PLoS ONE 6, e18556.</LI><LI>Li, X., Zhao, X., Fang, Y., Jiang, X., Duong, T., Fan, C., Huang, C.-C., and Kain, S.R. (1998). Generation of destabilized green fluorescent protein as a transcription reporter. Journal of Biological Chemistry 273, 34970-34975.</LI><LI>Lohmueller, J.J., Armel, T.Z., and Silver, P.A. (2012). A tunable zinc finger-based framework for Boolean logic computation in mammalian cells. Nucleic Acids Research 40, 5180--5187.</LI><LI>Margolin, J.F., Friedman, J.R., Meyer, W., Vissing, H., Thiesen, H.-J., and Rauscher, F. (1994). Krüppel-associated boxes are potent transcriptional repression domains. Proceedings of the National Academy of Sciences 91, 4509-4513.</LI><LI>Perfetto, S.P., Ambrozak, D., Nguyen, R., Chattopadhyay, P.K., and Roederer, M.J.n.p. (2012). Quality assurance for polychromatic flow cytometry using a suite of calibration beads.  7, 2067.</LI><LI>Qin, J.Y., Zhang, L., Clift, K.L., Hulur, I., Xiang, A.P., Ren, B.-Z., and Lahn, B.T.J.P.o. (2010). Systematic comparison of constitutive promoters and the doxycycline-inducible promoter.  5, e10611.</LI><LI>Rosenfeld, N., and Alon, U.J.J.o.m.b. (2003). Response delays and the structure of transcription networks.  329, 645-654.</LI><LI>Struhl, K.J.C. (1999). Fundamentally different logic of gene regulation in eukaryotes and prokaryotes.  98, 1-4.</LI><LI>Szymczak, A.L., Workman, C.J., Wang, Y., Vignali, K.M., Dilioglou, S., Vanin, E.F., and Vignali, D.A.J.N.b. (2004). Correction of multi-gene deficiency in vivo using a single'self-cleaving'2A peptide–based retroviral vector.  22, 589.</LI></OL></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;padding: 10px 0;">Abstract</H2><P style="margin: 0">Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4 active"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                           </DIV><DIV class="col s12 m6 l2"> +86-21-31246727
                           </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                           </DIV><DIV class="col s12 m6 l4"> Fudan_iGEM
                           </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>