<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Model</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right blueBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Model</SPAN></P></DIV></LI><LI>Navigator on this page</LI><LI class="onThisPageNav"><A href="#Transcriptional_Amplifer">Transcriptional amplifer</A></LI><LI class="onThisPageNav"><A href="#NotchLigandKinetics">Receptor-Ligand kinetics</A></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Model</H1></DIV><DIV class="col s12 hide-on-med-and-up"><SPAN>It is important to observe and anticipate how one's product will work in the real world in order to make it more applicable. </SPAN></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Model</H1><P><SPAN>It is important to observe and anticipate how one's product will work in the real world in order to make it more applicable. </SPAN></P></DIV></DIV><DIV class="container"><MAIN><UL id="pageContentNav" class="hide-on-med-and-down z-depth-0"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model">Model</A></LI><LI class="onThisPageNav"><A href="#Transcriptional_Amplifer">Transcriptional amplifer</A></LI><LI class="onThisPageNav"><A href="#NotchLigandKinetics">Receptor-Ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><DIV class="section container"><P>Our modeling focuses on two main aspects. The first is optimizing the transcriptional modules to increase signal-to-noise ratio, and the second is forecasting clinical outputs for multiple types of mixtures of immune cells and cancer cells. We experimentally showed that an amplification step is required. We also used a stochastic process to model receptor-ligand interaction kinetics, a possibility theory to model the transcriptional amplifier, and differential equations to model signal integration. ENABLE constructs with modeled parameters have increased signal-to-noise ratio and should have larger dynamic range.</P></DIV><DIV id="Transcriptional_Amplifer" class="section container scrolSpy"><H2>The essence of the Amplifier for transmembrane signaling</H2><P>By modeling, we demonstrated that our 3-layer design balances adjustability and stability for transmembrane logic processing.
</P><P>
                            Cellular logic gate lies at the foundation of our system, but the core and essential design of ENABLE is the detection of transmembrane signals. With our powerful SynNotch receptors transferring extracellular signals into intracellular transcription factors, we aim to detect real-world signals through ligand-receptor binding. In previous designs that detect intracellular signals with logic gates, researchers merely control the input to the system by not adding or adding superfluous small molecule drugs <A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">(Auslnder et al., 2012)</A>, or by not performing transfection or performing superfluous transfection, which is analogous to the 0 or 1 binary input. In such scenarios, just one single element is suffice to faithfully represent the on and off of the input. However, these previous designs are unable to transduce transmembrane signals in realistic circumstances. Our analysis through modelling leads to the 3-layer design principle of the ENABLE toolbox, which is to introduce an amplifier layer between the SynNotch (input, which we call the Receptor) system and the responsive element (output, which we call the Combiner).</P><H3>Previous designs fail to transduce transmembrane signals</H3><P>
                            Signal detection in engineered cells is composed of three parts: input, the representation of information through transcription factors (TFs); process, the processing of information through TF-Promoter interactions; and output, the response to the information through gene expressions. When the input is experimentally controllable or even artificially manipulated, researchers can explore the capacity of cellular information processing by designing specific circuits with only one element <FONT color="purple">(Figure 1)</FONT>. Using transduction or small molecule drugs for input manipulation, scientists in previous studies <A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">(Gaber et al., 2014)</A> have shown the logical processing ability of engineered cells and synthetic gene circuits.
                        </P><DIV class="expFigureHolder width40 left-on-med-and-up"><P><B>Figure 1. Previous designs usually incorporate one pair of transcription factor and promoter as the detection system.</B>The amount of transcription factor is manually controlled using small molecule drug, transduction, etc. As a result, the input of the system is nearly arbitrary, which can easily leads to the on/off output, represented by the activity of the promoter.
                            </P></DIV><P>Unlike previous designs, the ENABLE system aims to detect real-world transmembrane signals represented by SynNotch receptors. The imperfect and variable situations and scenarios in the real world are far from experimentally controllable; yet they are unavoidable and must be dealt with.</P><P>1) The number of Notch molecules on the cell membrane is relatively limited. Thus, the amount of released transcription factors is constrained (under detection level and have never been directly visualized) even under superfluous activation. This will prevent the input from being arbitrarily amplified.</P><P>2) Experimental observations show that SynNotch comes with a certain level of background activation without external stimuli, which may most likely be a result of thermodynamic randomness.
                        </P><P>These factors prevent the previous designs from faithfully detecting the on and off of external stimuli. In fact, any synthetic circuits with only one input-process-output element will be limited in its detection ability. Here, we conventionally used the Hill Equation to characterize such an element, in which the output (denoted by X) relates to the input (activator for example, denoted by A) through the equation:
                        </P><P>Both X and A are described by their concentrations. Kd denotes the dissociation constant between the activator A and its binding site on the gene circuit whereas the n denotes the Hill coefficient. Please refer to <A href="https://2017.igem.org/Team:Fudan/Model/HE" target="_blank">the model from our team in 2017 for details on the simulation</A>.
                        </P><P>
                            The two free parameters, the dissociation constant (Kd) and n (Hill Coefficient), can be controlled to adjust the element. As shown in the following interactive figure, this allows us to manipulate the element to achieve different input (A) - output (X) relationships.
                        </P><P class="hide-on-med-and-up">(Hidden content only visible on a desktop computer.)</P><DIV class="expFigureHolder width40 right-on-med-and-up"><P><B>Figure 2. Previous designs with single element are not able to handle transmembrane signal processing task.</B>The input-output relationship of a single element is characterized by Hill Equation, which comes with a 'detection range' defined by Kd and n. When the input range does not match the detection range, the system cannot faithfully represent the on and off of the input.
                            </P></DIV><P>
                            However, the free parameters are in reality not 'free' at all. While Kd is determined by the stability of the transcription factor-binding complex. The available transcription factor-promoter pairs, unfortunately, are limited, thus this constrains the dynamics that the one single input element can achieve to very few possibilities. More specifically, membrane receptors such as SynNotch that detect transmembrane signals and interact with ligands via non-covalent interactions require extremely low dissociation constants that may not even be realistic. On the other hand, the sensibility of the system needs to be high while the possible input range remains low and narrow. Furthermore, the Hill coefficient n representing the sensibility is almost uncontrollable and requires the specific and accurate designing of the promoter. Thus the previous designs that utilize a single element are not able to handle transmembrane signal processing tasks <FONT color="purple">(Figure 2)</FONT>.
                        </P><P>
                            One potential solution is to change the number of binding sites of the transcription factors. As illustrated in Figure 3, different binding situations can lead to different activation levels. While a single binding site can be characterized by the Hill Equation.
                        </P><DIV class="expFigureHolder width40 left-on-med-and-up"><P><B>Figure 3. Changing the number of binding sites of the transcription factors can potentially lead to more complex dynamics.</B>Different numbers of bound transcription factors have different effects on the activity of the promoter, denoted by α in the figure.
                            </P></DIV><P>We characterized the input-output relationship when there are multiple binding sites (of an activator A for example).</P><P>While n and Kd are the Hill coefficient and dissociation constants, N represents the total number of binding sites, α<SUB>i</SUB> denotes the activation level while i activators are bound to the binding sites (bigger i leads to bigger α). Unfortunately, the following interactive Figure demonstrates that such dynamics is still Hill-like, and still relies on different pairs of transcription factor-binding sites to manipulate the dynamical range (four binding sites are used as an example).</P><P>In conclusion, changing both the transcription factor - promoter pairs and the number of binding sites can be used to adjust how a synthetic gene circuit responds to the input. However, both are limited and cannot enable a single limited-number element to faithfully detect transmembrane signals. Nevertheless, they are still valuable in our ENABLE after the addition of an amplifier layer <FONT color="purple">(Figure 4)</FONT>.</P><DIV class="expFigureHolder width40 right-on-med-and-up"><P><B>Figure 4. In the design of our system, the input will first come from the SynNotch receptors.</B>Then it will be amplified by a tunable amplifier circuit, which will then feed input to the final responsive element, our Combiner.
                            </P></DIV><H3> An amplifier circuit</H3><P>The realistic constraints and our analysis through modeling prompted us to introduce the amplifier circuit. With the amplifier circuit present, the eventual input into the final responsive element now comes from the output of the amplifier circuit instead of the output of the SynNotch receptors. This enables our system to be much more adjustable and stable.
</P><P>
                            With the amplifier circuit, the input from SynNotch will no longer directly enter the responsive element but will first go through the Amplifier. As demonstrated before, the input signal from SynNotch is weak and has a narrow range, making it unsuitable to be directly processed by the responsive element. However, with the amplifier, input strength to the responsive element can be manipulated by the amplifier, providing a higher level of control.
                    </P><DIV class="expFigureHolder width40 right-on-med-and-up"><P><B>Figure 5. The amplifier circuit introduces nested dynamics into our system.</B>The output of the amplifier A and its input from SynNotch B are characterized by the Hill Equation. The A subsequently leads to the final output X, which again follows the Hill Equation in relation to A. While the input range of B is usually narrow and hard to detect, it can be amplified to the range of A through the amplifier circuit.
                            </P></DIV><P>
                            To remain consistent, we denoted the final output of the responsive element as X and the input as A. With amplifier circuit present, the A now means the output of the amplifier circuit while the B now means the direct input from SynNotch, which is again still the input to the amplifier circuit. As shown in Figure 5, the narrow range of the SynNotch signal does not support faithful detection. But with the amplifier circuit, the input range will be converted to an output range, which will then become the input range to the responsive element. </P><P>The output range of the amplifier is currently still undiscriminatable to the responsive element, but the amplification range can be manipulated. First, we can freely engineer the transcription factors to change the dissociation constant Kd. Since the input signal is now amplified, it will be easier to find an appropriate A to X relationship <FONT color="purple">(Figure 6)</FONT>.</P><P>Secondly, with a certain and fixed A to X relationship, we can change the binding sites of transcription factor B of the amplifier circuit which determines how the signal from SynNotch will be amplified. This allows us to very freely adjust the dynamical properties of the ENABLE toolbox. The previously unusable input range now gets amplified and enters the detection range of the responsive element; hence, allowing accurate detection <FONT color="purple">(Figure 7)</FONT>.</P><DIV class="row expFigureHolder"><DIV class="col s12 m6" style=""><P><B>Figure 6. Adjusting the transcription factor - its promoter pair between the amplifier circuit and the responsive element changes the relationship between X and A.</B>Comparing with Figure 5, the organe line moves, which suggests a response to lower concertation of input from the Amplifer output, allowing the final response to be a full range.
                                </P></DIV><DIV class="col s12 m6" style=""><P><B>Figure 7. Adjusting the binding sites of transcription factor B on the amplifier circuit leads to a dramatic change in the amplification magnitude.</B>The former narrow and undetectable range of B can now be easily manipulated in terms of A. This modeling suggests to increase the copy number of our Amplifer's DNA binding domain in our 3-layer design.
                                </P></DIV></DIV><UL class="collapsible expandable"><LI style="list-style-type: none"><DIV class="collapsible-header">Click to see more about the formula</DIV><DIV class="collapsible-body container"><P>Derivation of the formula we use to characterize a single transcription factor - promoter pair with multiple binding sites is straightforward under a few appropriate assumptions. We will keep using activator for example. Since we have N binding sites, the number of actual bound activators ranges from 0 to N. First, we assume that the activity of the promoter is solely controlled by the number of bound activators, but not their spatial arrangement (or spatial arrangement has little influence on the activation mechanism). Similar evidence has been reported before. Thus we could simply consider in total the N+1 states representing N+1 bound activators. We use α<SUB>i</SUB> to denote the activation level while i activators are bound to the binding sites (bigger i leads to bigger α). Second, zinc finger proteins are used as the building block of our transcription factor. Structure analysis shows no known interaction sites between the proteins. We thus assume different biding sites are independent to each other. This allows us to easily assign probabilities to the N+1 states. For each binding site, the probability of the activator being bound is again describe by 'Hill Equation' (refer to the model of our team in 2017 for the probabilistic explanation of Hill Equation and more <A href="https://2017.igem.org/Team:Fudan/Model/HE" target="_blank">Link here</A>).</P><P>To account for the four independent binding sites, elementary combination would show that the probability of i (ranging from 0 to 4) activator being bound is</P><P>Thus the final output can be characterized by the expectation of the promoter activity under the N+1 states, which is</P></DIV></LI></UL></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above in addition to the modeling just describe to add two points:
        (1) A summary of the Receptor-Ligand kinetics described below that with the same number of receptors and proteases, due to the increased complexity with more diversed ligand-receptor interactions, the release of Notch intracellular domains significantly decreased.
        (2) There was a previous study highlighting the requirement of amplification for signaling transduction across many cells.</P></DIV><DIV id="NotchLigandKinetics" class="section container scrolSpy"><H2>Receptor-Ligand kinetics</H2><P>
                            The Receptor-Ligand collects extracellular signals for further intercellular processing, which constitutes the most significant part for any transmembrane logic gate. To reduce the background activation of SynNotch, we put a huge <A href="/Team:Fudan/Optimization" target="_blank">experimental effect based on its protein structure</A>.
                        </P><P>
                            We focused on the signal-to-noise ratio (SNR) of the Notch/SynNotch receptor. In previous research, some quantitative descriptions of Notch-Ligand have been published for explanation and exploration of systematic design <A href="https://www.ncbi.nlm.nih.gov/pubmed/23839946" target="_blank">(Andrawes MB, et al., 2013)</A>. For example, cis-inhibition was modeled via chemical kinetics, which precisely predict the mechanism of Notch-induced pattern formation <A href="https://www.ncbi.nlm.nih.gov/pubmed/20418862" target="_blank">(Sprinzak D, et al., 2010)</A>. However, stochastic models for Notch-Ligand simulations have not been reported yet.
                    </P><P>
                        Here we present a mathematical model for Notch-Ligand kinetics using Stochastic <A href="https://en.wikipedia.org/wiki/Petri_net#Mathematical_properties_of_Petri_nets" target="_blank">Petri nets</A>, which takes random intercellular processes into consideration. We found that the SNR of our system is not only dependent on the affinity of Notch-Ligands, but also the Secretase Complex. We also expanded our model from targeting just molecular-level chemical reactions to cell colony-level chemical reactions, which offers clues for oriented optimization of Notch. Last but not least, our object-oriented programming (OOP) makes it easy to transplant into <A href="/Team:Fudan/Software">further application</A>.
                        </P><H3>Using ChemicalReactions toolkit</H3><P>
                            Not like chemical reactions happening in tubes, Notch-Ligand interaction occurs in a 2D manner. That is to say, chemical reactions between Notch and Ligand takes place on the membrane of two neighboring cells. Two cells may exchange their components on the membrane by touching to each other. Due to physical constraints, the chemical constitution of those cells remains relatively independent. However, when Ligand proteins binding to the extracellular domain of Notch receptor, proteolytic cleavage and release of the intracellular domain are induced.
                        </P><P>
                        Proteolytic cleavage of Notch involves a few steps, including S2-cleaveage by metalloprotease ADAM10,  S3-cleavage by γ-secretase complex or γ-secretase Tetering <A href="https://www.ncbi.nlm.nih.gov/pubmed/21506924" target="_blank">(van Tetering G, et al., 2011)</A>. Here we simplify the cleavage of Notch after ligand binding, and we suppose that the cleavage is a one-step reaction with the smallest rate constant of all cleavages mentioned above. This simplification is coming down to the rate-limiting step in physics chemistry. The simplified equations are as follows.
                                     N+L⇌NL    NL+S⇌NLS→icd  
                        Here N refers to Notch, L refers to Ligand, NL Notch-Ligand complex, S protease, NLS Notch-Ligand-protease complex, and icd means the intracellular domain of Notch.
                    </P><P>
                        Our mathematic tool is <A href="https://en.wikipedia.org/wiki/Petri_net#Mathematical_properties_of_Petri_nets" target="_blank">Petri net</A>. For the history and definition of Petri net,
                        please check reference <A href="https://www.crcpress.com/Stochastic-Modelling-for-Systems-Biology/Wilkinson/p/book/9781439837726" target="_blank">Wilkinson DJ, et al., 2006</A>. This method views each chemical or reaction intermediate as nodes in a network, element reactions as edges, and weight and direction of each edge for stoichiometric number and reaction direction. Especially, we use {P, T, Pre, Post, M} to describe a Petri net precisely: P={p1, …, pu} is the space of chemicals, T={t1, …, tv} is the spaces of all transitions (element reactions) , Pre is a v*u integer matrix containing the weights from chemicals to transitions， and the (i,j)th element of this matrix is the weight of the arc going from chemical j to transition i, and Post is a v*u integer matrix containing the weights from transitions to chemicals, and the (i,j)th element of this matrix is the weight of the arc going from transition i to chemical j. M is a u-dimensional integer vector that represents the current state of the system (i.e. the number of molecules).
                    </P><P>
                        For system with only one kind of Notch and one kind of Ligand, we have
                        
                        the subscripts refer to the index of a cell in a cell-pair, and the index is designated arbitrarily.
                        Also, we have
                               Here the lower capital g and d refer to the generation and degradation of following chemicals.
                        Similarly, we can write out the matrix Post and Pre.
                    </P><P>
                        We set W as a u-dimensional zero vector for initialization. Then we need to designate when and how this system chooses to finish a certain element reaction. For this purpose, we consider that the occurrence of an element reaction is a heterogeneous Poisson process, and certain reaction selection via sampling. That’s, the possibility that an element reaction R<SUB>i</SUB> happening in the time interval (t,δt] is given by h<SUB>i</SUB>(x,<SUB>i</SUB>) δt. With additivity assumption, we can get the possibility an arbitrary reaction happening in the time interval (t,δt] is
                        </P><P>
                        The form of h<SUB>i</SUB>(x,<SUB>i</SUB>) is given by mass-action stochastic kinetics. For a given system, h<SUB>i</SUB>(x,<SUB>i</SUB>)=c<SUB>i</SUB>*C(n<SUB>i1</SUB>,n<SUB>i2</SUB>, …, n<SUB>in</SUB>), where c<SUB>i</SUB> is the rate constant for reaction i, n<SUB>in</SUB> refers to the molecule number of reactant k for reaction i, and function C() means
                        combinatorial number of all n<SUB>in</SUB>. In practice, C() can be replaced by Π() with some modifications for c<SUB>i</SUB>.
                    </P><P>
                        Similarly, we can easily derive the expression for system with i kinds of Notch and j kind of Ligand.
                        </P><H3>Constitutive expression for stationary system</H3><P>For certain parameters, Notch signaling component is equivalent to constitutive expression for stationary system <FONT color="purple">(Figure 8A)</FONT>. It’s a good property for using Notch-Ligand in colony design, which uses the simplified version for higher-layer system simulation and shorten the simulation time without side effect of poor prediction.
                        </P><H3>The rate of Notch cleavage by proteases</H3><P>Cleavage rate affects system response in a linear way. Notch-Ligand specificity may not affect system response.</P><P>
                            For certain parameters, generation of Notch-Ligand-protease complex may be the rate-limiting step <FONT color="purple">(Figure 8A)</FONT>. For example, decreased rate constant of Notch-Ligand-protease complex generation reduce the icd generating rate in a linear manner (Figure 8B with rate constant of gN<SUB>A</SUB>LS decreased to one tenth of that of Figure 8A). Also, changing Notch-Ligand binding affinity may not significantly change icd generating rate, which strongly corroborated this view (Figure 8C with rate constant of gN<SUB>A</SUB>L<SUB>B</SUB> decreased to one tenth of that of Figure 8A; and in Figure 8D where heter/homo refers to the rate constant ratio of gN<SUB>A</SUB>L<SUB>B</SUB>/gN<SUB>A</SUB>L<SUB>A</SUB>.
                        </P><H3>The signal-to-noise ratio (SNR)</H3><P>
                            SNR can be tuned via Notch-Ligand binding affinity in a power-law manner.
                        </P><P>
                        For certain parameters, signal-to-noise ratio can be tuned via Notch-Ligand binding affinity in a power-law manner (<FONT color="purple">Figure 8E</FONT> with rate constant of gN<SUB>A</SUB>L<SUB>B</SUB>varied compared to Figure 8A). This offers clue for <A href="/Team:Fudan/Optimization">SynNotch optimization</A>.
                    </P><P>
                            Amplification is required for Notch-Ligand (see above).
                        For certain parameters, signal of a certain pair of Notch-Ligand coupling may be diluted by occurrence of other reactions (<FONT color="purple">Figure 8F</FONT> with types of overall Notch/Ligand varied compared to Figure 8A).
                        </P><DIV class="expFigureHolder" style="width:100%"><P><B>Figure 8. Simulation results of Notch-Ligand kinetics.</B>(A) Simulation of a 2 Notch-2 Ligand system. The graph's horizontal axis shows the time range, and #molecules shows the number of 2 kinds of intracellular domain (icdA and icdB) in 2 neighboring cells. Both Cell1 and Cell2 are armed with all four kinds of Notch/Ligand A/B.
(B) Weak binding affinity for cleavage leads to low response. The graph's horizontal axis shows the time range, and #molecules shows the number of 2 kinds of intracellular domain (icdA and icdB) in 2 neighboring cells. The affinity of protease to NotchB-LigandB complex is weakened, leading to poor NotchB response to Ligand signal.
(C) High Notch-Ligand specificity may not affect system response. The graph's horizontal axis shows the time range, and #molecules shows the number of 2 kinds of intracellular domain (icdA and icdB) in 2 neighboring cells. The specificity of Notch-Ligand binding is enhanced, making no significant difference of system response with the cleavage being the rate-limit reaction in this system.
(D) Variant Notch-Ligand specificity may not affect system response. The graph's horizontal axis shows the ratio of rate constant of gN<SUB>i</SUB>L<SUB>j</SUB> (i≠j) and that of gN<SUB>i</SUB>L<SUB>j</SUB> (i=j), and #molecules shows the number of 2 kinds of intracellular domain (icdA and icdB) in Cell2. The specificity of Notch-Ligand binding is tuned, making no significant difference of system response with the cleavage being the rate-limit reaction in this system. This strongly corresponds to the fact that the cleavage reaction is the rate-limit reaction in this system.
(E) SNR can be tuned by Notch-Ligand specificity. The graph's horizontal axis shows the ratio of rate constant of gN<SUB>i</SUB>L<SUB>j</SUB> (i≠j) and that of gN<SUB>i</SUB>L<SUB>j</SUB> (i=j), and the SNR are defined as the ratio of the number of 2 kinds of intracellular domain (icdA and icdB) in Cell2, with Cell1 only have LigandA. The specificity of Notch-Ligand binding is tuned, changing SNR in a power-law manner.
(F) Amplification is needed for multi-Notch/Signal system. The graph's horizontal axis shows the diversity of the Notch-Ligand system, and the diversity are defined as the species number of Notch/Ligand. With the diversity increasing, the number of a certain type of Notch intracellular domain molecules reduces.
                            </P></DIV><UL class="collapsible expandable"><LI style="list-style-type: none"><DIV class="collapsible-header">Click to see more about the modeling parameters</DIV></LI></UL><P>
                            The stochastic Notch-Ligand kinetics can be simplified as a single chemical constant for some certain conditions. Though this may greatly reduce the workload of transplant our model into a higher-scale application (e.g. to model a cell colony made up of cells armed with Notch and Ligand), this OOP modeling style makes it easy to transplant Notch-Ligand kinetics at the molecular level to macroscale level. Please continue to our <A href="/Team:Fudan/Software">software</A>, where we abstracted mammalian cells into blocks (with parameters modeled and simulated above), to predict cellular behaviors. We quantified the behavior of individual cells within a population. We found that besides ENABLE signaling, cell proliferation speed, cell life-span and cell adhesion greatly impact cancer elimination effectiveness. Our modeling and software gives ENABLE gates a population perspective, and test them in a clinical scenario.
                        </P><P>A document on <A href="https://static.igem.org/mediawiki/2018/c/c1/T--Fudan--model-cell-colony.pdf" target="_blank">model cell colony</A>, and source code is available on <A href="https://github.com/0vioiano/iGEM2018_Team_Fudan" target="_blank">GitHub</A>.</P></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;line-height: 0.8rem;">Abstract</H2><P>Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4 active"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                           </DIV><DIV class="col s12 m6 l2"> +86-21-31246727
                           </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                           </DIV><DIV class="col s12 m6 l4"> Fudan_iGEM
                           </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>