<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Software skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Software</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Software</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right orangeBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Software</SPAN></P></DIV></LI><LI>Navigator on this page</LI><LI class="onThisPageNav"><A href="#section1">Abstract</A></LI><LI class="onThisPageNav"><A href="#section2">Introduction</A></LI><LI class="onThisPageNav"><A href="#section3">Method</A></LI><LI class="onThisPageNav"><A href="#section4">Tutorials</A></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner orangeBg"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Software</H1></DIV><DIV class="col s12 hide-on-med-and-up"><SPAN>...</SPAN></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Software</H1><P><SPAN>...</SPAN></P></DIV></DIV><DIV class="container"><UL id="pageContentNav" class="hide-on-med-and-down z-depth-0"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI>Software</LI><LI class="onThisPageNav"><A href="#section1">Abstract</A></LI><LI class="onThisPageNav"><A href="#section2">Introduction</A></LI><LI class="onThisPageNav"><A href="#section3">Method</A></LI><LI class="onThisPageNav"><A href="#section4">Tutorials</A></LI></UL><MAIN><DIV id="section1" class="section container scrolSpy"><P>Our software tool was built based on (1) <A href="/Team:Fudan/Model">modeled parameters</A> characterizing our 3-layer design; (2) populational description of individual cellular behaviors. We found that several cell behaviors and ENABLE signaling have key impact on the evolution of a cell colony with mixed cell types. In an object-oriented-programming and user-friendly style, our software allows users to adjust those key factors and profile the fate of their own mixed cells. Our software bridges a nanoscopic transcriptional design of biological circuits, with microscopic cellular behaviors, up to a macroscopic population output, from which clinical outcome could be predicted, artificial tissue could be assembled, etc.
                        </P></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to demonstrate a possible clinical outcome with cells having 100-fold granzyme release.</P></DIV><DIV id="section2" class="section container scrolSpy"><H2>Introduction
                        </H2><P>
                            Modeling is necessary for quantifying biological processes and designing biological systems for customized function. With the increasingly rapid development of synthetic biology, many models are based on different branches of applied mathematics, as exampled by stochastic process, cellular automaton, dynamic systems, and Boltzmann kinetics for different biological processes. For example, automaton is often used to model birth and death processes, stochastic processes is often used to model transcription factor-DNA binding，dynamic systems is often used in population ecology to predict the evolution of colony, and Boltzmann kinetics is often used to quantitatively describe chemical reactions. Some toolkit based on this model, like <A href="http://www.compucell3d.org/" target="_blank">CompCell3D</A>, has been developed for computational nanotechnology and simulating tissue development.
                        </P><P>
                            However, there hasn’t a ready single model for our ENABLE project. It is because an overall model for our project should not only consist the 3-layer standard design (for more details please visit <A href="/Team:Fudan/Results">our Results page</A>) but also the biological mechanism underlying it (for more details please visit <A href="/Team:Fudan/Model">our Modeling page</A>).
                        </P><P>
                            Here we present a software (<A href="https://github.com/0vioiano/iGEM2018_Team_Fudan" target="_blank">github.com/0vioiano/iGEM2018_Team_Fudan</A>) using multiscale mathematical tools for different biological processes, which serves as a reusable tool for cell colony design. We use different modules packaged in different <I>Classes</I> to simulate different biological processes. Based on the object-oriented principal, it is easy for users to realize customized Classes and simulate cell colony using our software.
                        </P><P>
                            A colony is occupied by different populations, a population is unitized by abundant individuals, be it a cell or an ensemble of various cells, and a cell is a network of chemicals, for example, proteins, lipids, nucleotides, etc. To simulate cell colony, our software bridges a nanoscopic transcriptional design of biological circuits, with microscopic cellular behaviors, up to a macroscopic population output, from which clinical outcome could be predicted, artificial tissue could be assembled, etc.
                        </P><P>
                            A demonstration using our software to simulate the process of our engineered cells collaborating to wipe out cancer cells are offered as <A href="https://github.com/0vioiano/iGEM2018_Team_Fudan/tree/master/demo" target="_blank">a demo</A>.

                        </P></DIV><DIV id="section3" class="section container scrolSpy"><H2>Method
                        </H2><P>Our software is built based on OOP algorithm and MATLAB. The workflow of our software is shown below. For more details, please refer to <A href="https://github.com/0vioiano/iGEM2018_Team_Fudan" target="_blank">src and demo folders on GitHub</A>.
                        </P><DIV class="expFigureHolder" style="width:100%"><P>Figure.1 UML of our toolbox.
                            </P></DIV><DIV class="expFigureHolder" style="width:100%;margin-top: 23px"><P>Figure.2 Workflow of our software.
                            </P></DIV><P><B>Initialization:</B> First, initialize the system. Use the formative text as input, then designate the initial state of each cell (life span, type, vitality), properties of each cell type (type of Ligand and Notch, proliferation rate, mean vitality, and special chemical reactions in cells of this type), and relationship between cells (binding affinity between cells, which can be related to cell type and expression of membrane proteins, such as Notch and Ligand).

                        </P><DIV class="tableHolder"><TABLE><TBODY><TR><TH>Parameter</TH><TH>Meaning</TH></TR><TR><TD>T</TD><TD>Temperate, measuring the effect of random move.</TD></TR><TR><TD>E_neighbor</TD><TD>Affinity, measuring the effect of directional movement of cell based on cell-cell recognition.</TD></TR><TR><TD>Nm</TD><TD>Sampling rate in dt, measuring the rate of cell movement.</TD></TR><TR><TD>dt</TD><TD>The step length of Euler method in our simulation.</TD></TR><TR><TD>T</TD><TD>Simulation time, measuring how long we want to simulate with our software.
                                    </TD></TR></TBODY></TABLE></DIV><P><B>Iteration:</B> After initiation, an iteration is made. In a period of δt (very short time, and at the scale at 1-10 seconds), the cell may try to migrate, proliferate, or contact to neighboring cells to gather information for choice making. What is worth noticing is that if the cell moves/divides/dies, the Notch-Ligand kinetics may change abruptly for the changing cell-cell interaction, and a movement is relatively rapid compared with cell size. Therefore, it only takes little time for the switching of cell-cell network, but the process of finding proper movement takes some.

                        </P><P>
                            Within the interval of two “cellular movements (proliferation/migration/death included)”, “chemical movements” happens. This refers to the Notch-Ligand kinetics between cell membranes, the amplification of the signal of Notch ICD (intracellular domain) and the combination of augmented signals. Using <A href="https://en.wikipedia.org/wiki/Euler_method" target="_blank">Euler method</A> (for kinetics) and discrete <A href="https://en.wikipedia.org/wiki/Gillespie_algorithm" target="_blank">Gillespie algorithm</A> (for stochastic process), we have predicted how a single cell works in a period within the interval of <B>cellular movements</B> (proliferation/migration/death included) and <B>chemical movements</B>, which refers to the <A href="/Team:Fudan/Model#NotchLigandKinetics">Notch-Ligand kinetics</A> between cell membranes, the amplification of the signal of Notch intracellular domain and the combination of augmented signals.

                        </P><P>
                            After this prediction, we construct some functions to record the general state of each cell, including index, position, age, and then tendency to die or divide, for further data analysis. A snapshot of our cell colony is taken simultaneously for further simulation visualization. A judgement statement is executed to determine whether to terminate (when δt multiples iteration time is greater than T, the full length of the simulation) or continue to iterate (both cellular movements and chemical movements). Movements in a single iteration seems negligible, but a big number of iterations would show a difference.

                        </P><P>For more details of cellular movement, please refer to the <A href="#">supplementary material</A>; for more details of chemical movement, please refer to our modeling of <A href="/Team:Fudan/Model#NotchLigandKinetics">Notch-Ligand Kinetics</A>.
                        </P><P><B>Data analysis:</B> Upon simulation termination, data will be analyzed using prepared functions. We offer APIs for cell track, cell census and cell network analysis.

                        </P><P><B>Simulation visualization:</B> Using <I>clips</I> recorded in Iteration step, it’s easy to get our simulation visualized using built-in Matlab function <I>videowrite</I>. Cell colony composition can be checked by watching the output video.
                        </P></DIV><DIV id="section4" class="section container scrolSpy"><H2>Tutorials (single functions)
                        </H2><P>To make users familiar to our toolbox, a tutorial is as follows.</P><H4>Notch-Ligand kinetics (using <I>ChemicalReactions</I> toolkit)
                        </H4><P><I>ChemicalReactions</I> toolkit is a toolkit for chemical reaction modeling using <A href="https://en.wikipedia.org/wiki/Petri_net#Mathematical_properties_of_Petri_nets" target="_blank">Petri net</A> and Possion process. Here we demonstrate the usage of <I>ChemicalReactions</I> through an example of Notch-Ligand kinetics modeling.
                        </P><DIV class="expFigureHolder" style="width:100%;margin-top: 23px"><P>
                                Figure 3. Notch-Ligand kinetics analysis using ChemicalReactions toolkit. Code are offered on <A href="https://github.com/0vioiano/iGEM2018_Team_Fudan" target="_blank">Github</A>.
                            </P></DIV><H4>STEP 1</H4><P>Open MATLAB, and open the file ChemicalReactions.md in folder @ChemicalReactions.</P><H4>STEP 2</H4><P>prepare MATLAB variables.
                            &gt;&gt;[Pre,Post]=Pre_Notch_Ligand(2)
                                    &gt;&gt;obj= ChemicalReactions(‘’,’’,Post,Pre,@H_Notch_Ligand,zeros(1,12),2)
</P><P>
                                    Here Pre_Notch_Ligand() is a function to generate transition matrix for chemical reactions, @H_Notch_Ligand is a function handle for calculation of reaction possibilities, zeros(1,12) designates the initial condition of the system, and 2 refers to 2 kinds of Notch/Ligand exist.

                        </P><H4>STEP 3</H4><P> Enter in the command line window
                            &gt;&gt;XMat=obj.iteration(100,1)
                            &gt;&gt;for i =1:12
                            &gt;&gt;plot(XMat(:,i));
                            &gt;&gt;hold on
                            &gt;&gt;end
                        </P><P>
                            Here iteration() is a method of class <I>ChemicalReactions</I> for predicting the state of the system. (100, 1) is the setting step for data recording, and XMat is for data storage.
                        </P><P>
                            To repeat the results of Figure 3, please refer to <A href="https://github.com/0vioiano/iGEM2018_Team_Fudan" target="_blank">Github</A> for code.
                        </P><H3>War predictor (using <I>Cell2DProl</I> toolkit)
                        </H3><P><I>Cell2DProl</I> toolkit is a toolkit for Cell colony simulation. For more details, please refer to the <A href="#">supplementary material at the end of this page</A> or <A href="https://github.com/0vioiano/iGEM2018_Team_Fudan" target="_blank">Github</A>.
                            Here we demonstrate the usage of <I>Cell2DProl</I> through an example of therapeutic engineered cell design.
                        </P><DIV class="expFigureHolder" style="width:100%;margin-top: 23px"><P>
                                Figure 4. Demo - visualization function in the <I>Cell2DProl</I> toolkit
                            </P></DIV><DIV class="expFigureHolder" style="width:100%;margin-top: 23px"><P>
                                Figure 5. Demo - Data analysis functions in <I>Cell2DProl</I> toolkit
                            </P></DIV><H4>STEP 1</H4><P>Open MATLAB, and open the file Cell2DShadeProl.md in your folder.</P><H4>STEP 2</H4><P>Click the ‘run’ on the panel.</P><P>Here all the parameters are set proper previously. For more details on parameters, please refer to the supplementary PDF below. If your simulation results are visualized successfully, a video called 1_6_1_6 can be found in your current folder.
                        </P><H4>STEP 3</H4><P>The left Figure in Figure 5 will be plotted automatically. To get the right figure, Enter in the command line window
                            &gt;&gt; res1 = -Culture.*(Culture&lt;-1);
                            &gt;&gt; res2 = res1(:);
                            &gt;&gt; ecdf(res2);</P><H3>OOP-based AND gate design (using <I>Cell</I> toolkit)</H3><P><I>Cell</I> toolkit is a toolkit to design combinational circuits by linking user-defined cellular movement (element) together. Here we show you how to use our toolkit <I>Cell</I> by an example of AND gate circuit design.
                        </P><DIV class="expFigureHolder" style="width:100%;margin-top: 23px"><P>
                                Figure 6. Demo - Simulation of 3-layer ENABLE using <I>Cell</I> toolkit
                            </P></DIV><H4>STEP 1</H4><P>Open MATLAB, and open the folder with subfolder @Cell.
                        </P><H4>STEP 2</H4><P>Run kinetic_notch_test.m, kinetic_addgate_test_signal.m, kinetic_addgate_test.m
                        Here we use kinetic model for Notch activation and signal amplification, and 3-step kinetics for intein kinetics to validate the function of our toolkit.
                        </P><H4>STEP 3</H4><P>Figure 6 will be output automatically.
                        </P></DIV><DIV class="section container"><H4>Supplementary PDF with references
                        </H4><P><A href="https://static.igem.org/mediawiki/2018/c/c1/T--Fudan--model-cell-colony.pdf" target="_blank">Model_cell_colony</A></P></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;padding: 10px 0;">Abstract</H2><P style="margin: 0">Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4 active"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                           </DIV><DIV class="col s12 m6 l2"> +86-21-31246727
                           </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                           </DIV><DIV class="col s12 m6 l4"> Fudan_iGEM
                           </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>