<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Addon skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Addon</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Addon</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right blueBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Model</SPAN></P></DIV></LI><LI>Navigator on this page</LI><LI class="onThisPageNav"><A href="#ribo">ENABLE<SUP>ribo</SUP></A></LI><LI class="onThisPageNav"><A href="#TALE">ENABLE<SUP>TALE</SUP></A></LI><LI class="onThisPageNav"><A href="#T2">ENABLE<SUP>T2</SUP></A></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Addon: ENABLE</H1></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Addon: ENABLE</H1></DIV></DIV><DIV class="container"><MAIN><UL id="pageContentNav" class="hide-on-med-and-down z-depth-0"><LI>Addon: ENABLE</LI><LI class="onThisPageNav"><A href="#ribo">ENABLE<SUP>ribo</SUP></A></LI><LI class="onThisPageNav"><A href="#TALE">ENABLE<SUP>TALE</SUP></A></LI><LI class="onThisPageNav"><A href="#T2">ENABLE<SUP>T2</SUP></A></LI><LI><A href="/Team:Fudan/Model">Model</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><DIV id="ribo" class="section container scrolSpy"><H2>HOW TO IMPLANT ENABLE<SUP>ribocomputing</SUP></H2><DIV style="background:rgba(0,0,0,0.2)"><P style="line-height:25px;text-indent:0;"><B>Complex cellular logic computation using ribocomputing devices. Green AA, Kim J, Ma D, ..., Collins JJ, Yin P. </B>
                            Nature, 2017 Aug;548(7665):117-121 PMID: 28746304; <A href="https://dx.doi.org/10.1038/nature23271" target="_blank">DOI: 10.1038/nature23271</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28746304" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed/28746304/</A></P></DIV><H3>In summary</H3><UL><LI>The ribocomputing devices they have developed in bacteria use RNA molecules as input signals and use whether or not producing a designed protein as the output.</LI><LI>Their RNA-based biological circuits exploit the programmable base-pairing properties of RNA.
                            </LI><LI>
                                The gate RNA carries out the signal processing by combining sensing and output modules to enable complex intracellular computations in a single circuit layer.
                            </LI><LI>AND, OR and NOT logic were built from self-assembly of input RNAs and gate RNAs. Input RNAs can interact with one another cooperatively to activate a gate RNA for AND logic, or to inhibit another gate RNA for NOT logic. Inputs that could bind to separated sensor domains on a third gate RNA can trigger protein production and thus used for OR logic.
                            </LI></UL><H3>Side-by-side compare ENABLE and Ribocomputing</H3><DIV class="tableHolder"><TABLE class="striped"><TBODY><TR><TH>Ribocomputing circuits  </TH><TH>ENABLE </TH></TR><TR><TD>Orthogonality </TD><TD>Possible </TD><TD>Yes </TD></TR><TR><TD>Transmembrane </TD><TD>no</TD><TD>Yes </TD></TR><TR><TD>Amplification before inducing transcription</TD><TD>no </TD><TD>Yes </TD></TR><TR><TD>Standardized information processing</TD><TD>Yes </TD><TD>Yes </TD></TR><TR><TD>Binary logic output </TD><TD>Yes </TD><TD>Yes </TD></TR><TR><TD>Information mediator</TD><TD>Toehold Switches </TD><TD>Zinc-finger based activators and repressors </TD></TR><TR><TD>Test in eukaryotic cells</TD><TD>no</TD><TD>Yes</TD></TR></TBODY></TABLE></DIV><H3>Strengths of Ribocomputing circuits</H3><UL><LI>Easy to scale up.</LI><LI>Use synthetic RNAs and operate purely at the post-transcriptional level with no intermediate transcriptional or translational steps, which minimizes delays and improves the reliability of signal transduction.</LI><LI>
                                In principle, it could be applied in prokaryotic hosts beyond <I>E. coli</I>.
                            </LI></UL><H3>Integrate Ribocomputing circuits into ENABLE</H3><P>Using AND logic as an example
                        </P><P style="margin-top:0;text-indent:0;"><B>Figure legend in the original study: Two-input AND gate constructed from two input RNAs that bind to yield a complete trigger RNA.</B>Both input RNAs are required to activate the toehold switches which will activate translation of the output gene. (e) A two-input AND gate constructed from two input RNAs that bind to yield a complete trigger RNA. (f) Flow cytometry measurements of the two-input AND circuit under four combinations of input RNAs.  (g,h) The truth table for the AND computation on linear (g) and logarithmic (h) scales.
</P><H4 style="line-height:45px">ENABLE<SUP>ribocomputing</SUP></H4><P>
                            ENABLE<SUP>ribocomputing</SUP> will use our Receptors to pass the signal across the membrane and will use our transcriptional module to amplify the production of Ribocomputing input RNAs. Ribocomputing AND gate RNAs should be stably produced. The computation by input and gate RNAs will behave as our Combinator to execute binary logic functions.
                        </P></DIV><DIV id="TALE" class="section container scrolSpy"><H2>HOW TO IMPLANT ENABLE<SUP>TALE-NOR</SUP></H2><DIV style="background:rgba(0,0,0,0.2)"><P style="line-height:25px;text-indent:0;"><B>Designable DNA-binding domains enable construction of logic circuits in mammalian cells.</B>
                            Gaber R, Lebar T, Majerle A, ..., Benčina M, Jerala R. Nat Chem Biol, 2014 Mar;10(3):203-8 PMID: 24413461; <A href="https://dx.doi.org/10.1038/nchembio.1433" target="_blank">DOI: 10.1038/nchembio.1433</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed/24413461/</A></P></DIV><H3>In summary</H3><P>
                            This study has shown that TALE-NOR gates can be combined to form all 16 binary logic gates.
                        </P><P>
                            Before this study, knowing the specific binding nucleotide sequences of TALEs (transcription activator-like effectors) has enabled people to design and construct a large number of protein domains that can bind almost any nucleotide sequence in genomes <A href="https://www.ncbi.nlm.nih.gov/pubmed/19933106" target="_blank">(Moscou MJ, et al., 2009;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/19933106" target="_blank">Boch J, et al., 2009)</A>. Moreover, fusing DNA sequence-binding TALE domains with the effector domain has been used to construct either transcriptional activators or repressors. These fusions have been proven orthogonal <A href="https://www.ncbi.nlm.nih.gov/pubmed/22581776" target="_blank">(Garg A, et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22828628" target="_blank">Cong L, et al., 2012)</A>.
                        </P><P>In this study, the authors used designable TALE repressors to construct the orthogonal NOR gates in HEK293T cells, serving as complex information-processing devices (like our Collectors). They used TALE repressors to serve as both their input and output of their genetic logic gates. They have demonstrated 16 two-input logic functions in transiently transfected mammalian cells from the combinations of the same type of orthogonal TALE repressor-based NOR gates. Furthermore, they also designed a NOR logic function that accepts three inputs and emulates logic functions.
                        </P><P>
                            The authors suggested that their platform can enable the design of any advanced information processing devices in mammalian cells that can take any signal combination as input and convert into a transcriptional output.
                        </P><H3>Side-by-side compare ENABLE and TALE-NOR</H3><DIV class="tableHolder"><TABLE class="striped"><TBODY><TR><TH>TALE-NOR gates </TH><TH>ENABLE </TH></TR><TR><TD>Orthogonality </TD><TD>Yes </TD><TD>Yes </TD></TR><TR><TD>Transmembrane </TD><TD>no</TD><TD>Yes </TD></TR><TR><TD>Amplification before inducing transcription</TD><TD>no </TD><TD>Yes </TD></TR><TR><TD>Standardized information processing</TD><TD>Yes </TD><TD>Yes </TD></TR><TR><TD>Binary logic output </TD><TD>Yes </TD><TD>Yes </TD></TR><TR><TD>Information mediator </TD><TD>TALE repressors </TD><TD>Zinc-finger based activators and repressors</TD></TR><TR><TD>Generalized intracellular logic layers</TD><TD>no</TD><TD>Yes</TD></TR></TBODY></TABLE></DIV><H3>Strengths of TALE-NOR gates</H3><UL><LI>The NOR gates can be optimized by placing the operators upstream from the promoter as it produces minimal variability in reporter expression and retains efficient repression.
                            </LI><LI>The same type of NOR gates can be used for all their circuit functions and allows for the construction of complex circuits with precision and interoperability.
                            </LI><LI>The implementation of TALE-based cellular circuits could be easily extended to other cellular chassis, such as yeast or bacteria.
                            </LI><LI>TALEs can accommodate only one or two mismatches out of the 18-nucleotide-recognition site, supporting the potential for large orthogonality.
                            </LI></UL><H3>Integrate TALE-NOR based logic gates with ENABLE</H3><P>Using NAND gate as an example</P><P style="margin-top:0;text-indent:0;"><B>Figure legend in the original study: Implementation of the two-input Boolean logic NAND function constructed from combinations of designed TALE repressor-based NOR gates.</B>The logic connectivity of biological NOR gates and experimental results from the firefly luciferase and confocal microscopy are shown, with the truth table below the luciferase reporter results. Their NAND gate, constructed from two NOR gates, will produce an output if at least one of the two signals is off.  Only those TALE binding sites that were functionally relevant for the circuit are illustrated. The appropriate combination of processing logic and input plasmids were introduced into HEK293T cells for each combination of input values. Values are the mean of <I>n</I> = 4 ± s.d. For all functions, the statistical significance of the separation between the high and low states is significant at the level *** P &lt; 0.001. Microscopic images are representative of five separate observations. Scale bars, 125 μm. nRLU, normalized relative light units.
                        </P><H4 style="line-height:45px">ENABLE<SUP>TALE-NOR</SUP></H4><P>ENABLE<SUP>TALE-NOR</SUP> will use our Receptors to pass the signal across the membrane and will use our transcriptional modules to amplify the production of TALE-NOR transcription factors. Different combinations of TALE-NOR will behave as our Combinator to execute binary logic functions.</P></DIV><DIV id="T2" class="section container scrolSpy"><H2>HOW TO IMPLANT ENABLE<SUP>T2</SUP></H2><DIV style="background:rgba(0,0,0,0.2)"><P style="line-height:25px;text-indent:0;"><B>Programmable single-cell mammalian biocomputers.</B>
    Ausländer S, Ausländer D, Müller M, Wieland M, Fussenegger M. Nature, 2012 Jul;487(7405):123-7 PMID: 22722847; <A href="https://dx.doi.org/10.1038/nature11149" target="_blank">DOI: 10.1038/nature11149</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">https://www.ncbi.nlm.nih.gov/pubmed/22722847/</A></P></DIV><H3>In summary</H3><UL><LI>
                                The authors used RNA-binding proteins and their RNA target units to create a set of synthetic transcription-translation control devices that can be rewired in a plug-and-play manner.
                            </LI><LI>
                                The transcription factors are triggered in the sense that they are activated only when they interact with specific inputs.
                            </LI><LI>
                                They use RNA-binding proteins that inhibit the translation of transcripts containing specific RNA target units.
                            </LI><LI>
                                They used a half-subtractor and half-adder to perform fundamental arithmetic operations (addition and subtraction) of two bits. Their half-subtractor was made by a combination of the XOR gate and the NIMPLY gate. Their half-adder was made by a combination of the XOR gate and the AND gate. For more details, please refer to Figure 4.
                            </LI><LI>
                                They have achieved two-input-two-output integration.
                            </LI></UL><H3>Side-by-side comparison</H3><DIV class="tableHolder"><TABLE class="striped"><TBODY><TR><TH>Transcription-translation control devices  </TH><TH>ENABLE </TH></TR><TR><TD>Orthogonality </TD><TD>Yes</TD><TD>Yes </TD></TR><TR><TD>Transmembrane </TD><TD>no</TD><TD>Yes </TD></TR><TR><TD>Amplification before inducing transcription</TD><TD>no </TD><TD>Yes </TD></TR><TR><TD>Standardized information processing </TD><TD>Yes</TD><TD>Yes</TD></TR><TR><TD>Binary logic output </TD><TD>Yes </TD><TD>Yes </TD></TR><TR><TD>Information mediator</TD><TD>RNA-binding proteins and their RNA target units</TD><TD>Zinc-finger based activators and repressors </TD></TR><TR><TD>Test in eukaryotic cells</TD><TD>Yes</TD><TD>Yes</TD></TR></TBODY></TABLE></DIV><H3>Strengths of transcription-translation control devices
                        </H3><UL><LI>These transcription-translation control devices are rational and has a predictable plug-and-play characteristic. Individual components can be readily rewired to perform computing activities, such as NOT, NAND, and NIMPLY.</LI><LI>Their single-cell biocomputers are scalable to tissue structures and are straightforward to wire with host metabolism to have therapeutic impact.</LI></UL><H3>Integrate T2 into ENABLE</H3><P>Using NIMPLY gate as an example</P><P style="margin-top:0;text-indent:0;"><B>Figure legend in the original study: Design synthetic NIMPLY gates in human cells.</B>(a) A ANDNOT B logic gate. By combining the two input signals erythromycin and phloretin in accordance with the truth table, transfected HEK-293 cells are programmed to produce d2EYFP exclusively in the presence of erythromycin and not phloretin as shown by fluorescence microscopy and FACS analysis. b.t., below the threshold of 104 fluorescence units. Error bars represent s.d.; n = 3.
                        </P><H4 style="line-height:45px">ENABLE transcription-translation control devices</H4><P>ENABLE<SUP>transcription-translation</SUP> (ENABLE<SUP>T2</SUP>) will use our Receptors to pass the signal across the membrane and will use our transcriptional module to amplify the production of trigger-controlled transcription factors, including both RNA-binding proteins and their RNA target units. Different combinations of trigger-controlled transcription factors will behave as our combinator to execute binary logic functions.
                        </P></DIV><DIV class="section container"><H4 style="margin-left: 0;">References</H4><OL id="ref" style="margin: 23px 0 0 0;"><LI>Moscou, M.J. &amp; Bogdanove, A.J. A simple cipher governs DNA recognition by TAL effectors. Science 326, 1501 (2009).
                            </LI><LI>Boch, J. et al. Breaking the code of DNA binding specificity of TAL-type III effectors. Science 326, 1509–1512 (2009).
                            </LI><LI>Cong, L., Zhou, R., Kuo, Y.C., Cunniff, M. &amp; Zhang, F. Comprehensive interrogation of natural TALE DNA-binding modules and transcriptional repressor domains. Nat. Commun. 3, 968 (2012).
                            </LI><LI>Garg, A., Lohmueller, J.J., Silver, P.A. &amp; Armel, T.Z. Engineering synthetic TAL effectors with orthogonal target sites. Nucleic Acids Res. 40, 7584–7595 (2012).
                            </LI></OL></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;line-height: 0.8rem;">Abstract</H2><P>Contact-dependent signaling is critical for multicellular biological
                            events, yet customizing contact-dependent signal transduction between
                            cells remains challenging. Here we have developed the ENABLE toolbox, a
                            complete set of transmembrane binary logic gates. Each gate consists of
                            3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                            Notch receptors to enable cells to respond to different signals across the
                            membrane reliably. These signals, individually amplified intracellularly by
                            transcription, are further combined for computing. Our engineered zinc finger-based
                            transcription factors perform binary computation and output designed products.
                            In summary, we have combined spatially different signals in mammalian cells,
                            and revealed new potentials for biological oscillators, tissue engineering,
                            cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                            contact-dependent signaling networks in mammals. The 3-layer design principle
                            underlying ENABLE empowers any future development of transmembrane logic circuits,
                            thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4 active"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                            </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                            </DIV><DIV class="col s12 m6 l2"> Fudan_iGEM
      </DIV><DIV class="col s12 m6 l4"> +86-21-31246727
                            </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>