<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Results skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Results</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Results</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right blueBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Transmembrane logic</SPAN></P></DIV></LI><LI>Navigator on this page</LI><LI class="onThisPageNav"><A href="#section1">Introduction</A></LI><LI class="onThisPageNav"><A href="#section2">Result</A></LI><LI class="onThisPageNav"><A href="#section3">Discussion</A></LI><LI class="onThisPageNav"><A href="#section4">Future work</A></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Transmembrane logic</H1></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Transmembrane logic</H1></DIV></DIV><DIV class="container"><MAIN><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to introduce the significance of our work.</P></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to highlight our work.</P></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to summarize our unified binary computing gates.</P></DIV><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we presented evidences that we have demonstrated ENABLE with 3-layer design.</P></DIV><UL id="pageContentNav" class="hide-on-med-and-down z-depth-0"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI>Transmembrane logic</LI><LI class="onThisPageNav"><A href="#section1">Introduction</A></LI><LI class="onThisPageNav"><A href="#section2">Result</A></LI><LI class="onThisPageNav"><A href="#section3">Discussion</A></LI><LI class="onThisPageNav"><A href="#section4">Future work</A></LI></UL><DIV id="section1" class="section container scrolSpy"><H2>Introduction</H2><P>
                            Sensing and integrating various transmembrane signals is a key aspect of cellular decision making. For example, activation of CD8+ cells requires co-activation of TCR and CD28 molecules, meanwhile, this activation can be inhibited by the PD-1 pathway <A href="https://www.ncbi.nlm.nih.gov/pubmed/28280247" target="_blank">(Hui et al., 2017)</A>. By abstracting this biological process, we can get: the activation of CD8+ cell = activated TCR AND (activated CD28 NIMPLY activated PD-1) <FONT color="purple">Figure 1a</FONT>. Programming cells with predictable complex transmembrane signal inputs – customized intracellular signal outputs logic relationships are significant for expanding the widespread applications of mammalian cells, such as cellular immunotherapy <A href="https://www.ncbi.nlm.nih.gov/pubmed/24337479" target="_blank">(Fedorov et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23242161" target="_blank">Kloss et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26830879" target="_blank">Roybal et al., 2016a;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/27693353" target="_blank">Roybal et al., 2016b)</A>, tissue patterning <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830878" target="_blank">(Morsut et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/29853554" target="_blank">Toda et al., 2018)</A><FONT color="purple">(Figure 1b)</FONT>.
                        </P><DIV class="expFigureHolder" style="width: 100%"><P><B>Figure 1. The logical integration of complex transmembrane signals has crucial biological implications.</B>(a) Immune cells recognize target cells by integration of multiple transmembrane logic signals. Immune cells need to be co-stimulated by TCR and CD28 to be activated. Tumor cells can induce PD-1 activation of immune cells by expressing PD-L1 molecules. PD-1 can in turn interrupt the progression of the immune response by terminating the co-stimulatory signal of CD28. Thus, tumor cells cannot be recognized by immune cells and cause immune escape.
                                (b) Engineering the cells to perform logical operations on multiple transmembrane signals to produce a variety of customizable outputs could enable cells to gain novel application potential.
                            </P></DIV><P>
                            The transmembrane signal transduction is obstructed by the cell membrane, a unique feature that makes designing multiple transmembrane signal sensing and integrating systems a huge challenge. As we have seen, synthetic biologists have achieved great success in the integration of non-transmembrane signals. Although researchers already built partially <A href="https://www.ncbi.nlm.nih.gov/pubmed/17700541" target="_blank">(Anderson et al., 2007;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">Auslnder et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28746304" target="_blank">Green et al., 2017;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/25417166" target="_blank">Green et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22009040" target="_blank">Wang et al., 2011)</A> or completely <A href="https://www.ncbi.nlm.nih.gov/pubmed/23539178" target="_blank">(Bonnet et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">Gaber et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26829588" target="_blank">Macia et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150900" target="_blank">Regot et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23396014" target="_blank">Siuti et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150903" target="_blank">Tamsir et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28346402" target="_blank">Weinberg et al., 2017)</A> binary logic gates in both prokaryocyte <A href="https://www.ncbi.nlm.nih.gov/pubmed/17700541" target="_blank">(Anderson et al., 2007;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23539178" target="_blank">Bonnet et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28746304" target="_blank">Green et al., 2017;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/25417166" target="_blank">Green et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23396014" target="_blank">Siuti et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150903" target="_blank">Tamsir et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22009040" target="_blank">Wang et al., 2011)</A> and eukaryocyte <A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">(Auslnder et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">Gaber et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26829588" target="_blank">Macia et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150900" target="_blank">Regot et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28346402" target="_blank">Weinberg et al., 2017)</A>. All of these mentioned design limited by they can only integrate signals that are confined within the cell membrane <A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">(Gaber et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28746304" target="_blank">Green et al., 2017;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/25417166" target="_blank">Green et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28346402" target="_blank">Weinberg et al., 2017)</A>, or are the small molecules which could freely penetrate the cell membrane <A href="https://www.ncbi.nlm.nih.gov/pubmed/17700541" target="_blank">(Anderson et al., 2007;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">Auslnder et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23539178" target="_blank">Bonnet et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/26829588" target="_blank">Macia et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150900" target="_blank">Regot et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23396014" target="_blank">Siuti et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150903" target="_blank">Tamsir et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22009040" target="_blank">Wang et al., 2011)</A>. Therefore, developing a complete system that enables it to sense and integrate complex transmembrane signals is a challenge for synthetic biology.
                        </P><P>
                            We designed the ENABLE (Engineered, Across-membrane, Binary Logic in Eukaryotes) system and achieved the first complete transmembrane binary Boolean logic in mammals. The three-layer modular design
                            (<A href="/Team:Fudan/Antigen_Receptors">Receptor</A>, Amplifier, and Combiner) of the ENABLE system gives the system great expandability, which not only promising a host of application potentials, but also provides a design paradigm for the future transmembrane logic decision system. At the same time, the ENABLE system is not only capable of running single-cell-based centralized logic calculations, but also enables distributed logic calculations based on single-cell-single-cell contacts through spatial wiring by the surface antigens. These characteristics give the ENABLE system the possibility to perform sophisticated cellular computation.

                        </P></DIV><DIV id="section2" class="section container scrolSpy"><H2>Three-layer logic processing circuit design paradigm for dual transmembrane signals
                        </H2><P>
                            The expression of membrane proteins on the cell membrane is limited. For example, cells stably expressing chimeric antigen receptor (CAR) on the membrane are expected to have expression level in the order of 50,000 molecules/cell <A href="https://www.ncbi.nlm.nih.gov/pubmed/26705086" target="_blank">(Harris and Kranz, 2016)</A>. It can be compared that in a transient experiment using cationic liposomes, the number of plasmids entering each cell’s nucleus can be as high as 50,000 molecules <A href="https://www.ncbi.nlm.nih.gov/pubmed/19211029" target="_blank">(Cohen et al., 2009)</A>. Therefore, it is critical to design a circuit so that it can accommodate the limited transmembrane signal. <A href="/Team:Fudan/Model#Transcriptional_Amplifer" target="_blank">Our modeling</A> shows that the use of transcription cascade can effectively amplify the signal.
                        </P><DIV class="expFigureHolder width60 right-on-med-and-up"><P><B>Figure 2. The multiple transmembrane signal is processed using a three-layer logic processing circuit design paradigm. </B>
                                On the first layer, the &quot;Receptor&quot; layer, SynNotch receives extracellular signals and then transduces into intracellular signals. The second layer, the &quot;Amplifier&quot; layer, contains two channels, receives signals from the &quot;Receptor&quot; layer, amplifies them and converts them into a variety of signal types based on transcription factors. The third layer, the &quot;Combiner&quot; layer, receives the dual channel signal from the Amplifier layer and integrates to produce a logical output.
                            </P></DIV><P>
                            Based on this, we designed a three-layer transmembrane logic circuit paradigm and limited the  transcription units inside the cell to three. The first layer is the &quot;Receptor&quot; layer on the cell membrane, the function of which is to transduce the dual orthogonal antigen signal into the cell by SynNotch to the signal carried by two activating-form transcription factors. The second layer is the &quot;Amplifier&quot; layer within the cell membrane. It contains up to two transcription units. TRE3GV (7xTetO-minCMV) and URE2G (4xUAS-minCMV) are paired activating-form promoters of tTAA and GV2, acting as two channels, receiving signals from two SynNotchs of the &quot;Receptor&quot; layer, respectively. The &quot;Amplifier&quot; layer amplifies the signal from the cell membrane and converts it into a composite signal type carried by engineered zinc finger-based transcription factors. The last layer is the &quot;Combiner&quot; layer within the cell membrane. This layer contains only one transcription unit that integrates the composite signals from the two channels of the &quot;Amplifier&quot; layer and performs the final logical decision output <FONT color="purple">(Figure 2)</FONT>.
                        </P><P>
                            In order to be able to receive dual transmembrane signals and produce complete binary Boolean logic, we designed a set of interactive gramma for the interaction between the &quot;Amplifier&quot; layer and the &quot;Combiner&quot; layer within the cell membrane. This grammar consists mainly of the following elements, a transcription system based on the activating-form, silencing-from or NIMPLY-form promoters (the three are collectively referred to as a synthetic transcription factor-promoter pairs based transcription system); intein-based protein in vivo fusion systems, proteolytic enzyme-based protein in vivo destruction systems (the two are collectively referred to as a protein fusion/destruction-based transcription factor logic).
                        </P><DIV class="expFigureHolder width45 left-on-med-and-up"><P><B>Figure 4. IDENTITY-form promoter.</B>Top. the IDENTITY-form promoter working mechanism diagram. The activating-form transcription factor (aTF) is formed by a DNA binding domain (DBD) and a transcriptional activation domain (AD) linked via a linker. The IDENTITY-form promoter is just an activating-form promoter (aPro) because it is expressed only in the presence of aTFs. The aPro is structured by inserting a plurality of response elements corresponding to aTF (aTF REs) before the minimal promoter (minPro, eg minCMV). Middle. use dual-fluorescence dual-plasmid assay to test the aPros. aPro is fully activated in the presence of its corresponding aTF (n = 3, error bar, SD).
                            Bottom. Testing of the Orthogonality of aTF-aPro pairs by using the cross-paired dual-fluorescence dual-plasmid assay.
                            The five aTF-aPro pairs have good orthogonality.
                        </P></DIV><H3>Constructing a synthetic transcription factor-promoter pairs based logic
                        </H3><P>
                            When constructing an artificial gene circuit, it is necessary to consider the interference of the host's endogenous signal. By using a cross-species promoter or synthetic promoter, and ensuring the designed gene circuit to be orthogonal to the host endogenous gene is the key to maintaining system robustness <A href="https://www.ncbi.nlm.nih.gov/pubmed/16738572" target="_blank">(Andrianantoandro et al., 2006)</A>. We constructed three mammalian adapted DNA binding domains (DBD) by by using synthetic Zinc finger (SynZF) <A href="https://www.ncbi.nlm.nih.gov/pubmed/22863014" target="_blank">(Khalil et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/22323524" target="_blank">Lohmueller et al., 2012)</A> of ZF21.16, ZF42.10, and ZF43.8. We <A href="/Team:Fudan/Improve" target="_blank">improved</A> the design paradigm about synthetic transcription factors (SynTF) – synthetic promoter (SynPro) pairs proposed by <A href="https://2017.igem.org/Team:Fudan/Demonstrate" target="_blank">2017 iGEM Team:Fudan</A>, constructed three mammalian adapted activating-form transcription factor (aTF) – activating-form promoter (aPro) pairs and three silencing-form transcription factor (sTF) – silencing-form promoter (sPro) pairs to serve as candidate downstream elements of SynNotch. Using <A href="/Team:Fudan/Measurement">a dual-fluorescence dual-plasmid (DFDP) system</A>, we confirmed that ZF21.16-, ZF42.10-, ZF43.8-VP64 have good activation characteristics <FONT color="purple">(Figure 4)</FONT>, ZF21.16-, ZF42.10 -, ZF43.8-KRAB has good inhibition properties <FONT color="purple">(Figure 5)</FONT>. At the same time, ZF21.16, ZF42.10, ZF43.8 are orthogonal to each other and to both TetR and Gal4, which are the DBD of selected aTF for SynNotch <FONT color="purple">(Figure 4)</FONT>. By using an identical intermediate transcription factor as a common wiring molecule for the two upstream transcription factors, we can generate an implicit OR gate <FONT color="purple">(Figure 6)</FONT> and an implicit NOR gate <FONT color="purple">(Figure 7)</FONT>.
                        </P><DIV class="expFigureHolder row width80" style="margin: 60px auto 0 auto;"><DIV class="col s12 m7"><P><B>     Figure 5. NOT-form promoter.
                                </B>
                                Top, the NOT-form promoter working mechanism diagram. Silencing-form transcription factors (sTFs) are formed by a (DBD) and a transcriptional silencing domain (SD, eg KRAB) linked via a linker. The NOT-form promoter is just an silencing-form promoter (sPro) because it is expressed only in the absence of sTFs. The sPro is structure by inserting a a plurality of REs corresponding to sTFs (sTF REs) before a constitutive expression promoter (conPro, eg. CMV). Bottom, Using dual-fluorescence dual-plasmid assay to test silencing-form promoters. sPro are sufficiently inhibited in the presence of their corresponding inhibitory transcription factors. The dashed line indicates the intensity of expression of the CMV promoter under the same test conditions. Relative to the insertion of REs on the 3-terminus of conPro, the 5’-terminal design can reduce the interference to the basic expression of conPro (n = 3, error bar, SD).

                            </P></DIV></DIV><DIV class="expFigureHolder row width80" style="margin: 23px auto 0 auto;"><DIV class="col s12 m7"><P><B>Figure 6. Implicit OR gate.</B>Top. Schematic diagram of implicit OR. By adding an intermediate layer, signals from aTF1, aTF2 are respectively received using two orthogonally aPros (aPro1, aPro2), then generate the aTF3 as a common wiring molecule.
                                Bottom. Implicit tTAA OR GV2 gate. Using tTAA, GV2 as the input signals. The TRE3GV, URE2G promoters receive signals from tTAA and GV2, respectively. ZF21.16-VP64 was used as aZF3, and its downstream 8xZF21.16-minCMV promoter controls the expression of d2EGFP (n = 3, error bar, SD).

                            </P></DIV></DIV><DIV class="expFigureHolder row width80" style="margin: 23px auto 0 auto;"><DIV class="col s12 m7"><P><B>Figure 7. Implicit NOR gate.</B>Top. Schematic diagram of the implicit NOR gate. Unlike implicit OR gate, here we use a sTF as a common wiring molecule.
                                Bottom. Implicit tTAA NOR GV2 gate. sTF1 is ZF21.16-KRAB, and its downstream 8xZF21.16-CMV controls d2EGFP expression (n=3, error bar, SD).
                            </P></DIV></DIV><DIV class="expFigureHolder width45 right-on-med-and-up"><P><B>Figure 8: NIMPLY-form promoter.</B>Top. the NIMPLY-form promoter working mechanism diagram. The NIMPLY-form promoter is highly expressed in the presence of only aTF and no sTF. When aTF and sTF coexist, sTF plays a major role. Upper, using dual-fluorescence dual-plasmid assay to Test NIMPLY-form promoters. 8×ZF21.16-minCMV-2×ZF43.8 is highly expressed only in the presence of ZF21.16-VP64 without ZF43.8-KRAB, while 8×ZF43.8-minCMV-2×ZF21.16 is only in High expression under conditions of ZF43.8-VP64 without ZF21.16-KRAB. Lower. The NIMPLY-form promoter shows NIMPLY logic selectivity at different aTF, sTF levels.
                                Bottom. The maximum activation intensity of the NIMPLY-form can be tuned by changing the number of repeats of the aTF REs.

                            </P></DIV><P>
                            In addition to traditional activating- and silencing-form promoters, we have also designed a novel NIMPLY-type promoter by adding multiple response elements corresponding to the activating-form transcription factor (aTF REs) and response elements corresponding to the silencing-form transcription factors (sTF REs) at the 5' and 3' ends of minCMV, respectively <FONT color="purple">(Figure 8)</FONT>. Thus, the sTF can simultaneously apply (1) transcriptional inhibitors recruited by KRAB can inhibit promoter expression <A href="https://www.ncbi.nlm.nih.gov/pubmed/8183939" target="_blank">(Margolin et al., 1994)</A>; (2) due to RE Located downstream of the promoter, when a sTF binds to the sTF REs, it can cause steric hindrance and enhance the ability of transcriptional inhibition by inhibiting the forward movement of RNA polymerase. This type of promoter exhibits the logical selectivity of NIMPLY type for aTF and sTF. NIMPLY-form promoter can be expressed only in the presence of an aTF and the absence of a sTF <FONT color="purple">(Figure 8)</FONT>. 8xZF21.16-minCMV-2xZF43.8 can only be activated in the presence of ZF21.16-VP64 and in the absence of ZF43.8-KRAB, while 8xZF43.8-minCMV-2xZF21.16 can only be activated in the presence of ZF43.8-VP64 and in the absence of ZF21.16-KRAB <FONT color="purple">(Figure 8)</FONT>. Through gradient transfection experiments, we also confirmed that the NIMPLY-type promoter exhibits the logic of NIMPLY in the ratio of different aTFs and sTFs <FONT color="purple">(Figure 8)</FONT>. In addition, as previously reported cases of aPros <A href="https://www.ncbi.nlm.nih.gov/pubmed/22863014" target="_blank">(Khalil et al., 2012)</A> and sPros <A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">(Gaber et al., 2014)</A> with intensity tunability, NIMPLY-form promoter can also be tuned by using a different number of aRE sites corresponding to the. When the number of sTF REs is fixed and the number of aTF REs is increased, the maximum activation value increase <FONT color="purple">(Figure 8)</FONT>.
                        </P><H3>Constructing a protein fusion/destruction-based transcription factor logic
                            </H3><DIV class="expFigureHolder width45 left-on-med-and-up"><P><B>Figure 9: Split intein system.

                                </B>
                                Top left. Construct an AND gate by using split aTF and intein. aZF'N is a fusion of AD (VP64) and the N-terminus of DBD (DBD'N) with the N-terminus of Cfa (Cfa'N). While aZF’N meeting with aZF’C (consisted of DBD’C and Cfa’C), rapid and seamless assembly is performed, which in turn generates the active aTF to induce its corresponding aPro. Top right. Construct an AND gate with VP64-ZF21.16'N and ZF21.16'C. When VP64-ZF21.16'N and ZF21.16'C are present simultaneously, the expression of downstream 8xZF21.16-minCMV relatively rises. (n = 3 technical repeats, error bar, SD).
                                Bottom left. Constructing an NAND gate by using split sTF and intein. Use KRAB as the SD. When both sZF'N and sZF'C are present, active sTF is produced to inhibit its corresponding sPro. Bottom right. Constructing NAND gate with KRAB-ZF21.16'N and ZF21.16'C. The downstream sPro is 8xZF21.16-CMV (n = 3 technical repetitions, error bar, SD).


                            </P></DIV><P>
                                To build some of the complex logic, we used intein and proteolytic enzyme technologies. Cfa is a highly efficient fast split intein <A href="https://www.ncbi.nlm.nih.gov/pubmed/26854538" target="_blank">(Stevens et al., 2016)</A>. Cfa’s residue tolerance of the C-extein proximal sequence was improved by directed mutagenesis of the catalytically related key loop <A href="https://www.ncbi.nlm.nih.gov/pubmed/28739907" target="_blank">(Stevens et al., 2017)</A>. We split ZF21.16 in the first cysteine <A href="https://www.ncbi.nlm.nih.gov/pubmed/22323524" target="_blank">(Lohmueller et al., 2012)</A> of the outer region of the ZF structure, resulting in the N-terminal ZF21.16 (ZF21.16'N) and the C-terminal ZF21.16 (ZF21.16'C) . We co-transfected VP64-ZF21.16'N-Cfa'N and Cfa'C-ZF21.16'C into the cell. Comparing with the cell only transfected the VP64-ZF21.16'N-Cfa'N or Cfa’C-ZF21.16’C, the co-transfected cells showed a relatively higher activation level of 8xZF21.16-minCMV <FONT color="purple">(Figure 9)</FONT>. Similar experiments were performed using KRAB-ZF21.16'N-Cfa'N and Cfa'C-ZF21.16'C, and expression of the 8xZF21.16-CMV promoter in the co-transfected group was inhibited <FONT color="purple">(Figure 9)</FONT>. Combining use of the aZF’N and aZF’C can generate the AND gate <FONT color="purple">(Figure 9)</FONT>, while the co-effect of sZF’N and sZF’C results the NAND gate <FONT color="purple">(Figure 9)</FONT>.

                            </P><P>
                                We next tried to use proteolytic techniques to build more complex logic gates. Due to the stringent sequence specificity of Tobacco etch virus protease (TEVp), it is widely used in the field of synthetic biology <A href="https://www.ncbi.nlm.nih.gov/pubmed/18165312" target="_blank">(Barnea et al., 2008;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/24611683" target="_blank">Daringer et al., 2014)</A>. We replaced the amino acid sequence of the ZF21.16-KRAB linker region by using the high affinity TEV cleavage sequence (TCS), and added a certain of Gly or Ser at both ends of the TCS to provide flexibility and then constructed destroyable sTF (dsTF). In the experimental group whicn co-transfected with dsTF and TEVp, the expression of inhibited 8xZF21.16-CMV was partially restored <FONT color="purple">(Figure 10)</FONT>. Therefore, the IMPLY logic has been constructed <FONT color="purple">(Figure 10)</FONT>.
                            </P><DIV class="expFigureHolder row width80" style="margin: 23px auto 0 auto;"><DIV class="col s12 m6"><P><B>
                                    Figure 10: protease system.

                                </B>
                                Left. IMPLY gate can be achieved by using dsTF and TEVp. The linker of dsTF between SD and DBD contains a TEVp high affinity sequence. When TEVp is present, the linker of dsZF is destroyed, and KRAB and DBD are separated. At this time, even if the DBD is able to bind to the sTF REs on the sPro, it is unable to suppress the expression of sPro due to the lack of SD. Right. Constructing NAND gate with KRAB-ZF21.16'N and ZF21.16'C. When TEVp and dKRAB-ZF21.16 are simultaneously present, TEVp disrupts dKRAB-ZF21.16, resulting in a certain degree of recovery of downstream 8xZF21.16-CMV expression (n = 3 technical replicates, error bar, SD).


                            </P></DIV></DIV><P>
                                So far, we have tested all the components needed to build intracellular logic gates, and they all
                                worked. At the same time, we completed most of the construction intracellular logic gates by using the synthetic transcription factor-promoter pairs based logic and the protein fusion/destruction-based transcription factor logic (we have now built the OR, NOR, AND, NAND, IMPLY, NIMPLY logic gates.
                                We only have XOR and XNOR gates not characterized, so we have completed 6 of 8 complex binary gates, which is 14 of 16 complete binary logic. Our next goal is to integrate these intracellular elements with “Receptor” layer to construct transmembrane binary Boolean logic.

                            </P><H3>Transmembrane binary Boolean logic</H3><DIV class="expFigureHolder width30 right-on-med-and-up"><P><B>Figure 11: Using the three-layer paradigm to build a transmembrane OR gate.

                                </B>
                                This is a schematic diagram of the OR gate Receiver cell.
                            </P></DIV><P>
                                Any of the transmembrane binary Boolean logic has been constructed by integrating the &quot;Receptor&quot; layer in which SynNotch is located with the intracellular logic elements we constructed above (view our specific design on the <A href="/Team:Fudan/Parts_Collection" target="_blank">parts collection page</A>). Here, we use the OR gate as a proof of concept for a three-layer paradigm-based transmembrane binary Boolean logic. In our test, we used the previously constructed bistable cells as the cell chassis of the Recever cell, on which the elements of the &quot;Amplifier&quot; and &quot;Combiner&quot; layers corresponding to the OR gate were transiently transfected <FONT color="purple">(Figure 11)</FONT>. LifeAct-EGFP-labeled 293T-surEGFP and H2B-EGFP-labeled 293T-surCD19 were used as Sender Cell. In this way, we can clearly distinguish the type of antigen carried by the sender cells according to the position of EGFP under the microscope. Microscopy showed that both surEGFP <FONT color="purple">(Figure 12)</FONT> and surCD19 <FONT color="purple">(Figure 13)</FONT> were able to activate our Receiver cells, respectively. This shows that both the surEGFP and surCD19 signals can be received by the SynNotch paired with them and passed to the final &quot;Combiner&quot; layer via the &quot;Amplifier&quot; layer. Therefore, we have conceptually demonstrated the feasibility of the transmembrane binary Boolean logic based on the three-layer paradigm.

                            </P><DIV class="expFigureHolder" style="width: 100%"><P><B>Figure 12: surEGFP could induce the OR-Receiver.

                                </B>
                                OR-Receiver can be activated by surEGFP to generate a signal. Bottom right, pattern mode diagram. S, Sender cell with surEGFP. A, activated Receiver cells. I, inactevted Receiver cells. Receiver cells that are in direct contact with sender cells are activated, and Receiver cells that aren’t contacted by sender cells cannot be activated. U, Receiver cell that failed to transfect the OR gate intracellular element. We firstly transfected the dual stable Reicerver Cell with 293T-LaG16-2-mN1c-GV2 and αCD19-mN1c-tTAA on two glass petri dishes that have been Coat by fibronectin. at 0h. the required elements of OR gate’s &quot;Amplifier&quot; and &quot;Combiner&quot; layers were transiently transfected into two culture dishes in duplicate. After 8 hours of transfection, the two sender cells were plated onto Receiver cells seperately. For the experimental group using surEGFP Sender cell, we detected the SynNotch activation at 54 hours after transfection. For the experimental group activated with surCD19 Sender cell, continuous live motion was taken for 30 h at 24 h after transfection using a live cell workstation. The picture shown is a representative picture of all fields of view taken. We have removed the weak background green fluorescence of SynNotch stable cells and surEGFP cells in the picture.

    </P></DIV><DIV class="expFigureHolder" style="width: 100%"><P><B>Figure 13: surCD19 could induce the OR-Receiver (a different field as of Figure 12).</B>The Receiver cell equipped with an OR gate can be activated by the surCD19 to generate a signal.
                                The time projection was used to illustrate the dynamically trajectory of cell movement for 30 hours. Scale bar, 20 μm.
                            </P></DIV></DIV><DIV id="section3" class="section container scrolSpy"><H2>Discussion of our results </H2><H3>Single cell based logical operations</H3><P>
                            In the ENABLE system, it is theoretically possible to construct complete single-cell logic calculators with different dual signal processing logic. These cell-based logic calculators have customizable input and output units that inspire us to define cell functions at new level. We named this new concept &quot;CellBrick&quot;. CellBrick is similar to BioBrick, a well-known modular concept at gene level for synthetic biology, but is even more different in that it is actually a standardized biological device based on cell levels. With mutiple Cellbrick, each cell can be explicitly defined as a microprocessing unit with specific inputs and outputs. By using different combinations of input and output, Cellbrick is able to perform specific tasks in a coordinated manner in terms of processes or results <FONT color="purple">(Figure 14)</FONT>.
                        </P><P>
                            In particular, CellBricks can be “wired” by standardizing orthogonal surface antigens, allowing cells and cells to be connected in a gear-like or brick-like manner. Different Cellbricks form an interaction system with upper and lower levels. By optimizing the performance of each Cellbrick, the overall interaction system can perform a wider range of functions at a higher level.Previous logic gate systems often have mismatches in the input signal-output signal class (for example, we could design circuits that enable cells to logically process complex small molecule signals. However, because the output port product is often a protein rather than a homogeneous small molecule. That will result in an irreversible signal transmission chain <A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">(Auslnder et al., 2012)</A>) or the type of input signal cannot spontaneously cross the cell membrane into the cell in a natural way (for example, input the signal by transient transfection <A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">(Gaber et al., 2014)</A>). These logic gate designs can only choose to perform centralized computing <A href="https://www.ncbi.nlm.nih.gov/pubmed/22722847" target="_blank">(Auslnder et al., 2012;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23539178" target="_blank">Bonnet et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/24413461" target="_blank">Gaber et al., 2014;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/23396014" target="_blank">Siuti et al., 2013;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/28346402" target="_blank">Weinberg et al., 2017)</A> (in simple terms, the process of logical operations is limited to one cell) or distributed computing <A href="https://www.ncbi.nlm.nih.gov/pubmed/26829588" target="_blank">(Macia et al., 2016;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150900" target="_blank">Regot et al., 2011;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/21150903" target="_blank">Tamsir et al., 2011)</A> (in simple terms, that is, the logical operation can be performed by any of a plurality of cells or cell groups.). However, by using a library of Cellbricks which are spatially wired by antigen, we can allow cells to perform intercellular iterative distributed computing systems in a cell-cell contact-dependent manner. Moreover, this kind of distributed computing also has the characteristics of centralized computing. For example, by correlating Cellbricks with different logical operations (the internal operations of each cell actually run in a centralized computing way), or the Cellbricks equipped with the logic gate with logical complete such as NOR, we can construct a very complex logic processing system in theory <FONT color="purple">(Figure 14)</FONT>.
                        </P><H3>Building cell legion with CellBricks</H3><P>
                            In today's cellular immunotherapy, people tend to focus on the use of tools such as CAR to customize a straightforward immune response (recognition → killing). In contrast, the natural immune system achieves extraordinary regulation through a highly complex and networked interaction between multi-cells. Just by expanding the cell killing function of the TCR pathway, the CAR system shows such an exciting clinical application prospect <A href="https://www.ncbi.nlm.nih.gov/pubmed/23527958" target="_blank">(Grupp et al., 2013)</A>. If there is a method that allows people to fully invoke the specialized mechanisms in the natural immune system, would it allow cell immunotherapy to enter a new era? For example, in cellular immunity, the interaction of helper T cells and effector T cells plays a key role <A href="https://www.ncbi.nlm.nih.gov/pubmed/15286726" target="_blank">(Bevan, 2004)</A>. But as far as we know, there is still no system that attempts to simulate this natural mechanism and use multiple cells for interactive design. We imagine that through CellBrick, people can think and customize cell-cell interaction logic in a new dimension. In the future, people may be able to cope with the challenges of various diseases by using a more ingenious design and building a Cell Legion <FONT color="purple">(Figure 14)</FONT> with different functions based on a comprehensive consideration of the challenges faced by different diseases <A href="https://www.ncbi.nlm.nih.gov/pubmed/28187291" target="_blank">(Lim and June, 2017)</A>.
                        </P><DIV class="expFigureHolder" style="width: 100%"><P><B>Figure 14: Single cell based logical operation group could be achieved via ENABLE system.</B>(a) By using an antigen or a secretory molecule for wiring, it is possible that cells form a network with each other, thereby implementing CellBrick. Each cell is then similar to a microprocessor, interacting in a manner similar to building blocks.
                                (b) In the ENABLE system, each cell carries a logic component and the cells operate in a centralized computing manner. Cells that are equipped with different ENABLE elements can be linked tighter by using antigen wiring. They then interact in a mannar similar to distributed computing. In this way, it is expected to be able to implement a very complex logic processing system. (d). Using Cellbrick, you can construct a series of cell types with different functions to form a fully functional cell Legion. Cell Legion may be a new era for future cellular immunotherapy

                            </P></DIV></DIV><DIV id="section4" class="section container scrolSpy"><H2>Future work</H2><H3>Further exploration of SynNotch design principle
                        </H3><P>In order to achieve complete transmembrane binary Boolean logic, in addition to using our well-designed three-layer transmembrane logic paradigm. In our initial trials, we attempted to directly use different of synthetic activing-, silencing-form transcription factors to serve as the intracellular domains of SynNotch, attempting to construct logic gate system just by transcriptional network. But disappointingly, these novel SynNotchs equipped with synthetic zinc finger based transcription factor were not able to be efficiently activated (data not shown). We hypothesize that this may be due to two reasons:
                        </P><P>First, we use a three-finger synthetic zinc finger protein to construct a SynTF that recognizes a RE sequence specific to about 9 bp, which may limit its extent to bind the RE with high affinity. Therefore, it may be necessary to reach a certain working concentration to drive the function. Since the expression of SynNotch on the cell membrane is limited, when the SurAg activates SynNotch, only a limited intracellular domains can be cleaved to produce an effect. These may lead to the disability of SynZF-TF. Therefore, we design the excellent three-layer transmembrane logic paradigm to solve this problem. At the same time, we are also using the newly designed SynTFs based on six-finger zinc finger protein to construct a novel SynNotch. These novel synthetic six-finger zinc finger proteins recognize response elements up to about 18 bp and theoretically have superior affinities compared to the three-finger one.
                        </P><P>Second, as mentioned in the initial research by Lim, although the intracellular domain of SynNotch has excellent programmability, in practice, not all the combination of the extracelluar and intracellular domains can be grafted to SynNotch with function <A href="https://www.ncbi.nlm.nih.gov/pubmed/26830878" target="_blank">(Morsut et al., 2016)</A>. In our fundamental <A href="/Team:Fudan/Optimization" target="_blank">research on SynNotch</A>, we found that its S3 cleavage site has crucial biological functions. We believe that it is possible that the conformation of SynZF interferes with the proximity of γ-secretase and that SynNotch cannot be efficiently cleaved. In this regard, as we mentioned in the work of the SynNotch Optimization project, we are trying to design a more versatile linker for the joints at both ends of the SynNotch core region to extend the broad adaptability of the SynNotch design.
                        </P><H3>Optimization of the protein fusion/destruction-based transcription factor logic
                        </H3><P>In our experiments, it was found that transcription factor logics based on protein fusion/destruction are still insufficient in efficiency. Although Cfa is the fastest assembled and robust intein known to us, it only exhibits limited assembly efficiency in our application systems (such as AND and NAND gate). At present, we believe that the following two possibilities may limit its efficiency:
                        </P><P>First, we split the C2H2-type ZF21.16 at the first Cys site on the first β-fold of the second zinc finger motif and then construct the ZF21.16’N-Cfa’N and Cfa’C-ZF21.16’C. Although the Cys split site in theory will not cause scarring of ZF21.16 after assembly of the intein, it is still possible to interfere with the natural conformation of ZF21.16 to some extent after incorporation of the intein.
                        </P><P>Second, as previously reported, Cfa constructed based on the DnaE type intein groups still retains the preference for a bulky hydrophobic residue (e.g., Phe) at the +2 position of C-extein <A href="https://www.ncbi.nlm.nih.gov/pubmed/26854538" target="_blank">(Stevens et al., 2016)</A>. In subsequent work, we are ready to try other cleavage sites and follow the stricter Cfa C-extein proximal amino acid preference rules for design, or use a more appropriate linker to design the Cfa-splited SynTF.
                        </P><H3>Construction of the complete transmembrane logic gates
                        </H3><P>
                            Limited to the time factor, in this year's work, we only use the OR gate as a proof of concept for the transmembrane logic gate. But as we have shown, we have completed most of the intracellular logic gate construction and characterized their performances. We believe that the construction of complete transmembrane logic gates will be just around the corner in the near future.
                        </P></DIV><DIV class="section container"><H4 style="margin-left: 0;">References</H4><OL id="ref" style="margin: 23px 0 0 0;"><LI>Anderson, J.C., Voigt, C.A., and Arkin, A.P. (2007). Environmental signal integration by a modular AND gate. Molecular systems biology 3, 133.
                            </LI><LI>Andrianantoandro, E., Basu, S., Karig, D.K., and Weiss, R. (2006). Synthetic biology: new engineering rules for an emerging discipline. Molecular systems biology 2.
                        </LI><LI>  Auslnder, S., Auslnder, D., Mller, M., Wieland, M., and Fussenegger, M. (2012). Programmable single-cell mammalian biocomputers. Nature 487, 123--127.
                        </LI><LI>  Barnea, G., Strapps, W., Herrada, G., Berman, Y., Ong, J., Kloss, B., Axel, R., and Lee, K.J. (2008). The genetic design of signaling cascades to record receptor activation. Proceedings of the National Academy of Sciences 105, 64--69.
                        </LI><LI>   Bevan, M.J.J.N.R.I. (2004). Helping the CD8+ T-cell response.  4, 595.
                        </LI><LI>    Bonnet, J., Yin, P., Ortiz, M.E., Subsoontorn, P., and Endy, D.J.S. (2013). Amplifying genetic logic gates.  340, 599-603.
                        </LI><LI>      Cohen, R.N., van der Aa, M.A., Macaraeg, N., Lee, A.P., and Szoka Jr, F.C.J.J.o.C.R. (2009). Quantification of plasmid DNA copies in the nucleus after lipoplex and polyplex transfection.  135, 166-174.
                        </LI><LI>      Daringer, N.M., Dudek, R.M., Schwarz, K.A., and Leonard, J.N. (2014). Modular extracellular sensor architecture for engineering mammalian cell-based devices. ACS synthetic biology 3, 892-902.
                        </LI><LI> Fedorov, V.D., Themeli, M., and Sadelain, M. (2013). PD-1–and CTLA-4–based inhibitory chimeric antigen receptors (iCARs) divert off-target immunotherapy responses. Science translational medicine 5, 215ra172-215ra172.
                        </LI><LI>Gaber, R., Lebar, T., Majerle, A., ter, B., Dobnikar, A., Benina, M., and Jerala, R. (2014). Designable DNA-binding domains enable construction of logic circuits in mammalian cells. Nature Chemical Biology 10, 203--208.
                        </LI><LI>Green, A.A., Kim, J., Ma, D., Silver, P.A., Collins, J.J., and Yin, P. (2017). Complex cellular logic computation using ribocomputing devices. Nature 548, 117.
                        </LI><LI>  Green, A.A., Silver, P.A., Collins, J.J., and Yin, P. (2014). Toehold switches: de-novo-designed regulators of gene expression. Cell 159, 925-939.
                        </LI><LI>   Grupp, S.A., Kalos, M., Barrett, D., Aplenc, R., Porter, D.L., Rheingold, S.R., Teachey, D.T., Chew, A., Hauck, B., Wright, J.F., et al. (2013). Chimeric antigen receptor-modified T cells for acute lymphoid leukemia. New England Journal of Medicine 368, 1509--1518.
                        </LI><LI>  Harris, D.T., and Kranz, D.M. (2016). Adoptive T cell therapies: a comparison of T cell receptors and chimeric antigen receptors. Trends in pharmacological sciences 37, 220--230.
                        </LI><LI>   Hui, E., Cheung, J., Zhu, J., Su, X., Taylor, M.J., Wallweber, H.A., Sasmal, D.K., Huang, J., Kim, J.M., and Mellman, I. (2017). T cell costimulatory receptor CD28 is a primary target for PD-1–mediated inhibition. Science 355, 1428-1433.
                        </LI><LI>    Khalil, A.S., Lu, T.K., Bashor, C.J., Ramirez, C.L., Pyenson, N.C., Joung, J.K., and Collins, J.J. (2012). A synthetic biology framework for programming eukaryotic transcription functions. Cell 150, 647--658.
                        </LI><LI>    Kloss, C.C., Condomines, M., Cartellieri, M., Bachmann, M., and Sadelain, M. (2013). Combinatorial antigen recognition with balanced signaling promotes selective tumor eradication by engineered T cells. Nature biotechnology 31, 71.
                        </LI><LI>    Lim, W.A., and June, C.H. (2017). The Principles of Engineering Immune Cells to Treat Cancer. Cell 168, 724--740.
                        </LI><LI>    Lohmueller, J.J., Armel, T.Z., and Silver, P.A. (2012). A tunable zinc finger-based framework for Boolean logic computation in mammalian cells. Nucleic Acids Research 40, 5180--5187.
                        </LI><LI>    Macia, J., Manzoni, R., Conde, N., Urrios, A., de Nadal, E., and Sol (2016). Implementation of Complex Biological Logic Circuits Using Spatially Distributed Multicellular Consortia. PLOS Computational Biology 12, e1004685--1004624.
                        </LI><LI>    Margolin, J.F., Friedman, J.R., Meyer, W., Vissing, H., Thiesen, H.-J., and Rauscher, F. (1994). Krüppel-associated boxes are potent transcriptional repression domains. Proceedings of the National Academy of Sciences 91, 4509-4513.
                        </LI><LI>    Morsut, L., Roybal, K.T., Xiong, X., Gordley, R.M., Coyle, S.M., Thomson, M., and Lim, W.A. (2016). Engineering Customized Cell Sensing and Response Behaviors Using Synthetic Notch Receptors. Cell 164, 780--791.
                        </LI><LI>    Regot, S., Macia, J., Conde, N., Furukawa, K., Kjellén, J., Peeters, T., Hohmann, S., de Nadal, E., Posas, F., and Solé, R. (2011). Distributed biological computation with multicellular engineered networks. Nature 469, 207.
                        </LI><LI>   Roybal, K.T., Rupp, L.J., Morsut, L., Walker, W.J., McNally, K.A., Park, J.S., and Lim, W.A. (2016a). Precision Tumor Recognition by T Cells With Combinatorial Antigen-Sensing Circuits. Cell 164, 770--779.
                        </LI><LI>   Roybal, K.T., Williams, J.Z., Morsut, L., Rupp, L.J., Kolinko, I., Choe, J.H., Walker, W.J., McNally, K.A., and Lim, W.A. (2016b). Engineering T Cells with Customized Therapeutic Response Programs Using Synthetic Notch Receptors. Cell 167, 419--432.e416.
                        </LI><LI>   Siuti, P., Yazbek, J., and Lu, T.K. (2013). Synthetic circuits integrating logic and memory in living cells. Nature biotechnology 31, 448.
                        </LI><LI>    Stevens, A.J., Brown, Z.Z., Shah, N.H., Sekar, G., Cowburn, D., and Muir, T.W. (2016). Design of a split intein with exceptional protein splicing activity. Journal of the American Chemical Society 138, 2162-2165.
                        </LI><LI>    Stevens, A.J., Sekar, G., Shah, N.H., Mostafavi, A.Z., Cowburn, D., and Muir, T.W. (2017). A promiscuous split intein with expanded protein engineering applications. Proceedings of the National Academy of Sciences 114, 8538-8543.
                        </LI><LI>    Tamsir, A., Tabor, J.J., and Voigt, C.A. (2011). Robust multicellular computing using genetically encoded NOR gates and chemical ‘wires’. Nature 469, 212.
                        </LI><LI>   Toda, S., Blauch, L.R., Tang, S.K., Morsut, L., and Lim, W.A. (2018). Programming self-organizing multicellular structures with synthetic cell-cell signaling. Science, eaat0271.
                        </LI><LI>   Wang, B., Kitney, R.I., Joly, N., and Buck, M. (2011). Engineering modular and orthogonal genetic logic gates for robust digital-like synthetic biology. Nature Communications 2, 508.
                        </LI><LI>  Weinberg, B.H., Pham, N.H., Caraballo, L.D., Lozanoski, T., Engel, A., Bhatia, S., and Wong, W.W.J.N.b. (2017). Large-scale design of robust genetic circuits with multiple inputs and outputs for mammalian cells. Nat Tech 35, 453.
                            </LI></OL></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;line-height: 0.8rem;">Abstract</H2><P>Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4 active"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                           </DIV><DIV class="col s12 m6 l2"> +86-21-31246727
                           </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                           </DIV><DIV class="col s12 m6 l4"> Fudan_iGEM
                           </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>