<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan-CHINA_Results_PR skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan-CHINA/Results PR</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><UL id="navUl" style="z-index: 888;"><LI class="navLi navImg" style="float: left;position: relative;"><A class="navA activeA teamName" href="https://2018.igem.org/Team:Fudan-CHINA" style="color: white;">Fudan-CHINA</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Awards">Awards</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Team">Team</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Team">Team</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Attributions">Attributions</A></LI></UL><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Notebook">Notebook</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Human_Practices">Human Practices</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Human_Practices">Overview</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Human_Practices#mainTitle2">Silver HP</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Human_Practices#mainTitle3">Integrated HP</A></LI><LI class="li2"><A class="navA2 muchWords muchWords2" href="https://2018.igem.org/Team:Fudan-CHINA/Public_Engagement">Education &amp; Engagement</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Collaborations">Collaborations</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Safety">Safety</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Part_List">Parts</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Improve">Improved Part</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Basic_Part">Best New Basic Part</A></LI><LI class="li2"><A class="navA2 muchWords muchWords2" href="https://2018.igem.org/Team:Fudan-CHINA/Composite_Part">Best New Composite Part</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Part_Collection">Best Part Collection</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Part_List">Part List</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Model">Model</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Model">Model</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Software">Software</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Results">Results</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Results">Overview</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Results_STEP">STEP System Test</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Results_PR">Receptor Optimisation</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Demonstrate">Demonstrate</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/InterLab">Interlab</A></LI><LI class="navLi"><A class="navA" href="https://2018.igem.org/Team:Fudan-CHINA/Design">Project</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Background">Background</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Design">Design</A></LI><LI class="li2"><A class="navA2" href="https://2018.igem.org/Team:Fudan-CHINA/Applied_Design">Applied Design</A></LI><UL class="leftNav" style="margin:0;padding:0;"><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle1">Summary</A></LI><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle2">Methods</A></LI><LI class="leftNavLi2"><A class="leftNavA2" href="#mainTitle2_1">Computational macromolecule design</A></LI><LI class="leftNavLi2"><A class="leftNavA2" href="#mainTitle2_2">Ni affinity chromatography</A></LI></UL><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle3">VEGF-scFv</A></LI><LI class="leftNavLi2"><A class="leftNavA2" href="#mainTitle3_1">Redesigning in silicon</A></LI><LI class="leftNavLi2"><A class="leftNavA2" href="#mainTitle3_2">Testing in wet lab</A></LI><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle4">KDR Ig domain 3</A></LI><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle5">D-Dimer scFv</A></LI><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle6">tTA promoter</A></LI><LI class="leftNavLi"><A class="leftNavA" href="#mainTitle7">Reference</A></LI><DIV id="firstDiv"><DIV class="fiName">Receptor Optimisation</DIV><DIV class="pSentence">
		&quot;No road of flowers lead to glory.&quot;
	</DIV></DIV><DIV class="mainWords"><DIV class="mainTitle1">
		Summary
	</DIV><DIV class="mainContents">
    In our dry lab, to improve the binding affinity, we have redesigned 3 receptor binding domains: 
    1.VEGF-scFv
    2.KDR Ig domain 3
    3.D-Dimer scFv
    Also, we have redesigned the DNA sequence of the tTA promoter, to enhance downstream expression. <DIV class="imgState"><DIV><SPAN class="figureNum">Figure 0.</SPAN> Binding energy variation after redesigning 
			</DIV></DIV></DIV><DIV class="mainTitle1">
		Methods
	</DIV><DIV class="mainTitle2">
		Computational macromolecule design 
	</DIV><DIV class="mainContents"><DIV class="imgState"><DIV><SPAN class="figureNum">Figure 1.</SPAN> Computational macromolecule design</DIV></DIV>
   
    Aiming at enhancing the binding affinity of the signal molecule and the receptor by sequence redesigning, we should firstly make it clear: 
    WHAT DO WE HAVE? 
    To simulate the macromolecule interaction, the structure information is necessary. At best, the crystal structure of the signal-receptor complex is already tested by experiment. That means the interface of those two molecules is known, the interaction of the two molecules can be calculated, and further protein designing will be carried out. If there is only individual structures of the signal molecule and the receptor, we can use RosettaDock to simulate the binding process of the two molecules. If there are only amino acid sequences of those two molecules, we can still predict the structure by large-scale computations. If there is a homologous structure (for example, antibodies), this operation would be easier. Otherwise, modeling only with sequence is still a tough work. 
    Next step is to design the sequence with the complex structure. If the receptor is an antibody, or have Immunoglobulins as its binding site, we can use RosettaAntibodyDesign, an automatic application that designs antibodies with Monte Carlo method. Another thing we should remember is that before antibody design, the input structure must be renumbered, so that the program would know which polypeptide chain, or what secondary structure, is to be designed. 
    For more details, go to <A class="contentLink" href="https://2018.igem.org/Team:Fudan-CHINA/Design">principles</A>.
    </DIV><DIV class="mainTitle2">
		Ni affinity chromatography
	</DIV><DIV class="mainContents">
    1. Pick 3 colonies from plate and grow in 30mL LB + antibiotic (1‰ Kana or Amp according to vector) at 37℃ overnight. Transfer 10mL culture at 1:30 ratio into 300mL LB + antibiotic. After new culture could be grown at 37℃ into OD600 0.6~0.8 (which normally takes ~3hrs), add IPTG to induce protein expression at 16℃ for 18~20hrs.
    2. Collect culture in a 50mL tube, centrifugate consecutively at 4℃ 5,000rpm for 5 min and discard of LB supernate. Resuspend sediment in 10mL lysis buffer (20mM HEPES, 500nM KCl) and centrifugate again. Then discard supernate and resuspend sediment in 20mL lysis buffer.
    3. Add protease inhibitor at 1:1000 ratio and extract suspension through ultrasonication at 3sec/3sec for 10 min twice with 10 min interval. Centrifugate lysate at 4℃ 11,000rpm for 60 min. Take samples from supernate and sediment for future examination.
    4. Blend supernate with Ni resin in a 50mL tube and incubate horizontally on ice at 160rpm shake for 1.5hr, then erect the tube and place on ice for 0.5 hr to settle mixture. Take sample from solution which would be marked as flow-through.
    5. Segregate protein from affinity chromatography at 20nM-500nM imidazole gradient elution.
    6. Equilibrate with storage buffer (20mM HEPES, 150mM NaCl).
    7. Examine protein purification through SDS-PAGE.</DIV><DIV class="mainTitle1">
		VEGF-scFv
	</DIV><DIV class="mainTitle2">
		Redesigning in silicon
	</DIV><DIV class="mainContents">
	Sequence before and after mutation:
    Heavy chain WT:
    E V Q L V E S G G G L V Q P G G S L R L S C A A S G F T I S D Y W I H W V R Q A P G K G L E W V A G I T P A G G Y T Y Y A D S V K G R F T I S A D T S K N T A Y L Q M N S L R A E D T A V Y Y C A R F V F F L P Y A M D Y W G Q G T L V
    Heavy chain after mutation:
    E V Q L V E S G G G L V Q P G G S L R L S C A A S G F T I S D Y W I H W V R Q A P G K G L E W V A G I <SPAN style="background: lightskyblue;">V</SPAN> P A G G Y T <SPAN style="background: lightskyblue;">E</SPAN> Y A D S V K G R F T I S A D T S K N T A Y L Q M N S L R A E D T A V Y Y C A R F V F F <SPAN style="background: lightskyblue;">W</SPAN> P <SPAN style="background: lightskyblue;">F</SPAN> A M D Y W G Q G T L V
    This is the simplest job, since the VEGF &amp; VEGF antibody complex structure have been texted by X-ray diffraction. That means the interface of those two proteins has been determined. What we have to do is just remolding the amino acid sequence to have a stronger binding affinity. However, this is the best chance to test RosettaDock, since we have got the correct response. After 5000 times of global docking, we finally chose a best structure, in consideration of total energy, binding energy and interface (Antibodies have stationary interface, CDRs). Then, another 1000 times of local docking was operated. (see more from <A class="contentLink" href="https://2018.igem.org/Team:Fudan-CHINA/Design">principles</A>)
    		<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 2.</SPAN> Docked structure compared to crystal structure 
				(RMSD= 6.030 Å)
				(A)	Crystal structure
				(B)	Docked structure</DIV></DIV>
    Finally, we have got a not-so-bad docked structure. As Figure 2 shows, the docked structure has only minor difference compared to the crystal structure. 
    Next step is sequence design of the VEGF-scFv. With 200 rounds of sequence design of the CDR loop by RosettaAntibodyDesign, we finally mutated 4 residues on the heavy chain: (using AHo Scheme numbering) 
    Thr-59 → Val
    Tyr-69 → Glu
    Leu-113 → Trp
    Tyr-134 → Phe
    The Thr-59 and Tyr-69 belongs to CDR H2, while Leu-113 and Tyr-134 belongs to CDR H3. Let’s see what happens after antibody design. 
    1.Thr-59 → Val<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 3.</SPAN> Energy before and after mutating Thr-59 to Val
				 Other terms of energy have minor influence
				 (Terms of Rosetta Energy Function can be found in <A class="contentLink" href="https://2018.igem.org/Team:Fudan-CHINA/Design">principles</A>)</DIV></DIV>
	As we all know, Thr has polar side chain, while Val has hydrophobic side chain. After mutating Thr-59 to Val, the solvation energy (Figure 3 fa_sol) has dropped dramatically, while the Van der Waals interaction energy (Figure 3 fa_atr &amp; fa_rep, while calculating total energy, fa_rep should multiply by 0.55) has decreased a little. That means this mutation is in favor of the folding of protein, benefiting total energy. 	
		
	2.	Tyr-69 → Glu
	This mutation results in notable changes in charge distribution. With negative electrification, Glu has accumulated negative charge. (Figure 4) Since the area surrounding residue 69 has positive electrical charge, the electrostatic energy has reduced. Calculating by the Rosetta energy function, the electrostatic energy is decreasing by 1.25 kcal/mol after the mutation. <DIV class="imgState"><DIV><SPAN class="figureNum">Figure 4.</SPAN> Vacuum electrostatics plot around residue 69
				 (A)Before mutation
				 (B)After mutation</DIV></DIV>		
	3.	Leu-113 → Trp &amp; Tyr-134 → Phe
	These two residues are on CDR H3. In fact, they are close to each other, although they seem separate in AHo Scheme residue numbering. More important, they are directly on the interface of VEGF and VEGF-scFv. The pentapeptide on the interface, Phe Phe Leu Pro Tyr, has been in placed of by Phe Phe Trp Pro Phe. <DIV class="imgState"><DIV><SPAN class="figureNum">Figure 5.</SPAN> Interface of VEGF-scFv CDR H3 with VEGF
				 (A)Before mutation
				 (B)After mutation
				 Spheres shows VEGF molecule, while sticks shows the pentapeptide on the interface.
				 Polar contracts between VEGF and VEGF-scFv are marked with distance. (unit: Å)</DIV></DIV>		
	This area collected high ratio of aromatic amino acids. π–π stacking plays an important role in the stabilization of conformation. After mutation, the Leu-113 is replaced by Trp, which has an indolyl, an aromatic heterocyclic substituent. This causes another π–π stacking interaction (Figure 6), which increases the Van der Waals interactions, thus stabilizing the conformation. What’s more, Trp-113 generates another H-bond between VEGF and VEGF-scFv, which is contributes directly to the binding energy. <DIV class="imgState"><DIV><SPAN class="figureNum">Figure 6.</SPAN> π–π stacking between Trp-113 and Phe-112 after mutation</DIV></DIV>		
	The mutation of Tyr-134, however, has no significant influence on the Gibbs free energy. It may have minor effects on the conformation stability. Based on Monte Carlo method, RosettaAntibodyDesign does not always give the best answer. 
	
	After 4 point mutations, the binding affinity has increased a bit. The interface energy has decreased from -46.405 to -50.641 REU, calculated by RosettaDock. 
	
	Also, we have tested a previous part, BBa_K1694003. This is another VEGF-scFv, differing from the one we used. First, we used PIGSPro [1][2][3][4][5][6][7], a web-based tool to build the structure of immunoglobulins. The next step is finding the interface. After 5000 times of global docking and 1000 times of local docking, finally we find the interface of the previous parts, along with VEGF. After some structure refinements, we tested the binding energy of them. From the interface score calculated by RosettaDock, we can see a big difference. Therefore, our parts improvement work is successful.<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 7.</SPAN>Comparing binding energy between three types of VEGF-scFv
				The interface score is calculated by RosettaDock, which directly shows binding energy.</DIV></DIV><DIV class="imgState"><DIV><SPAN class="figureNum">Figure 8.</SPAN> Parts (BBa_K1694003) local docking RMSD
				The interface score is calculated by RosettaDock, which directly shows binding energy.</DIV></DIV>	
	RMSD is calculated compared to the final docking result, expressing the difference between local docking and the final result. There is clear interdependency between interface score and RMSD, which shows a potential well near the final result. Within the potential well, the structure will be stable. Therefore, our docking result is reliable. 	
	

	</DIV><DIV class="mainTitle2">
		Testing in wet lab
	</DIV><DIV class="mainContents">
   Above, VEGF-scFv was modified in silico to enhance its binding ability to VEGF ligand in our modelling. In protein expression and interaction section, we hope to mutate wildtype VEGF-scFv according to modification provided by dry lab and monitor its binding process with VEGF through a thermodynamic method known as isothermal titration calorimetry (ITC) developed by Adrian Velazquez-Campoy and Ernesto Freire in 2006 [8]. 
   1. Plasmid construction
   VEGF-scFv and VEGF-scFv’ were cloned into pGEX expression vector. VEGF sequence was synthesized by Sangon Biotech on a pET28a expression vector.
   
   2. Protein expression and purification<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 9.</SPAN> Gradient elution in affinity chromatography for VEGF-scFv’(above) and VEGF (below). 
				M: marker; L: lysate; SM: sediment; SN: supernate; FT: flow-through; numbers: eluting concentration of imidazole (in mM)</DIV></DIV><DIV class="imgState"><DIV><SPAN class="figureNum">Figure 10.</SPAN> protein purification products equilibrated with storage buffer (see protocols above for detail). 
				M: marker; L: lysate; SM: sediment; SN: supernate; FT: flow-through; numbers: eluting concentration of imidazole (in mM)</DIV></DIV>
   3. ITC
   The titration curve shown in Figure 11 is isothermal titration of VEGF and wild-type VEGF-scFv, of which concentrations are 90uM, 15uM respectively. However, there is no significant exotherm in the titration curve, since fluctuations may be related to the influx of the solution and the background noise of the instrument.<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 11.</SPAN> Isothermal titration of VEGF and VEGF-scFv</DIV></DIV><DIV class="imgState"><DIV><SPAN class="figureNum">Figure 12.</SPAN> Isothermal titration of VEGF and VEGF-scFv’</DIV></DIV>     
  The titration curve shown in Figure 12 is isothermal titration of VEGF and mutation VEGF-scFv (named VEGF-scFv’), of which concentrations are 90uM, 15uM respectively. It can be seen that there is a significant exothermic phenomenon when the first two drops of VEGF are instilled, indicating that VEGF is completely bound to VEGF-scFv', which is superior to wild type. However, subsequent data has no obvious meaning. 
  Because the ITC experiment requires a higher protein concentration and buffer composition. The preliminary analyses lead to unsatisfactory results are as follows. 1) The low concentration of ligand and receptor leads to less meaningful data. 2)VEGF is an insoluble protein, probably due to its poor solubility in buffer (20 mM HEPES + 150 mM NaCl) resulting to lower protein concentration in solution. 3) The activation of proteins in solution buffer were weaken. 
  We’ ll do further experiments to confirm our conclusion, by optimizing the method of protein expression and purification as well as the contains of solution buffer.</DIV><DIV class="mainTitle1">
		KDR Ig domain 3
	</DIV><DIV class="mainContents">
        
    Sequence before and after mutation:    
    WT:    
   
    S H G I E L S V G E K L V L N C <SPAN style="background: lightskyblue;">T A R T E L N V G I D F N</SPAN> W E Y P S S K H Q H K K L V N R D L K T Q S G S E M K K F L S T L T I D G V T R S D Q G L Y T C <SPAN style="background: lightskyblue;">A A S S G L M T K</SPAN> K N S T F V R V
    After mutation:
    S H G I E L S V G E K L V L N C <SPAN style="background: lightskyblue;">V A E T S L N V G I D M T</SPAN> W E Y P S S K H Q H K K L V N R D L K T Q S G S E M K K F L S T L T I D G V T R S D Q G L Y T C <SPAN style="background: lightskyblue;">T T M T L I V I Q T S E R Q Y T I W R M D V </SPAN>K N S T F V R V
    
  KDR is a receptor with immunoglobulins-like domain. Not a standard antibody, KDR still has some homology with antibodies. We got the crystal structure of KDR binding to VEGF from protein data bank (3v2a.pdb). However, some key residues are missing, probably because of the accuracy of solution structure. We used SWISS-MODEL [9][10][11][12][13] to rebuild the structure. Then, RosettaAntibodyDesign was operated. We tried GraftDesign, replacing whole CDR H3 with a CDR structure from antibody database. The interface energy has decreased from -41.850 to -43.639 REU. Thanks to the GraftDesign, the CDR H3 has been replaced by a longer one, and the contact area has increased a lot.       
        
     	<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 13.</SPAN> Interface of KDR and VEGF 
				Green=VEGF; Cyan=KDR
				(A)	Before mutation
				(B)	After mutation
				The interface is enlarged after mutation.</DIV></DIV></DIV><DIV class="mainTitle1">
		D-Dimer scFv
	</DIV><DIV class="mainContents">
        
Sequence before and after mutation:    
Heavy chain WT:
        Q V Q L K Q S G P G L V Q P S Q S L S I T C <SPAN style="background: lightskyblue;">T V S G F S L T T Y G V H</SPAN> W I R Q S P G K G L E W L G <SPAN style="background: lightskyblue;">V I W S G G S T D</SPAN> Y N A A F I S R L S I N K D N S K S Q V F F K M N S L Q A N D T A I Y Y C <SPAN style="background: lightskyblue;">A R N Y W G T S M D Y</SPAN> W G Q G T S V T V S S
Heavy chain after mutation:
        Q V Q L K Q S G P G L V Q P S Q S L S I T C <SPAN style="background: lightskyblue;">E A S E T D M K N A G F G</SPAN> W I R Q S P G K G L E W L G <SPAN style="background: lightskyblue;">G V D G G L G K E H G T T</SPAN> Y N A A F I S R L S I N K D N S K S Q V F F K M N S L Q A N D T A I Y Y C <SPAN style="background: lightskyblue;">T S A T G S A N</SPAN> W G Q G T S V T V S S
Light chain WT:
        D I K M T Q S P S S M Y A S L G E R V T V T C <SPAN style="background: lightskyblue;">K A S Q D I N S Y</SPAN> L S W I Q Q K P G K S P K T L I Y <SPAN style="background: lightskyblue;">R G N R L V A</SPAN> G V P S R F S G S G S G Q D Y S L T I S S L E Y E D V G V Y Y C <SPAN style="background: lightskyblue;">L R Y D E F P F</SPAN> T F G S G T K L E I K R A G Q G S S V
Light chain after mutation:
        D I K M T Q S P S S M Y A S L G E R V T V T C <SPAN style="background: lightskyblue;">Q A S Q D I G N D</SPAN> L S W I Q Q K P G K S P K T L I Y <SPAN style="background: lightskyblue;">K V S K R A S</SPAN> G V P S R F S G S G S G Q D Y S L T I S S L E Y E D V G V Y Y C <SPAN style="background: lightskyblue;">Q Q G Y S S P S</SPAN> T F G S G T K L E I K R A G Q G S S V  
 
  Another signal molecule we are fascinating with is D-Dimer. What we have found is the crystal structure of D-Dimer (PDB ID: 2Q9I), and the amino acid sequence of D-Dimer scFv. SWISS-MODEL helps us to model the structure of D-Dimer scFv. Next is docking, finding out the interface. <DIV class="imgState"><DIV><SPAN class="figureNum">Figure 14.</SPAN> D-Dimer scFv local docking RMSD</DIV></DIV>         
        
    After rounds of docking, finally we found the interface (Figure 14). As has mentioned before, this RMSD plot shows a reliable result of docking. The final step is RosettaAntibodyDesign. Here, we made a bold attempt, execute the command -random_start. This will start graft design with a new set of CDRs from the database, not biasing the run with native CDRs. The result is successful. The binding energy has been optimized from -31.561 to -54.875 (unit:REU). Since the CDR region is totally different after mutation, we can claim to create a new antibody. Merely, the position of interface is the same. </DIV><DIV class="mainTitle1">
		tTA promoter
	</DIV><DIV class="mainContents">
        
DNA Sequence before and after mutation:

WT:
T C T A T C A C T G A T A G G


After mutation:
T C T A T C A C T G <SPAN style="background: lightskyblue;">C</SPAN> T A G G

 We can also redesign DNA sequence. Here is an example, a promoter and transcription factor complex, tTA. In fact, the structure of tTA has not been tested through experiment. However, TetR, another transcription factor that has the same sequence on the protein-DNA interface, has known structure (PDB ID: 1QPI). So it is equivalent to redesign the interface of TetR. <DIV class="imgState"><DIV><SPAN class="figureNum">Figure 15.</SPAN> Vacuum electrostatics plot of interface near base pair 11
				 (A)Before mutation
				 (B)After mutation
				 The transcription factor is colored to show its electrostatics distribution. It should be mentioned that the area with less electric charge is always hydrophobic. </DIV></DIV>        
     
After mutation on base pair 11 (A to C), the interface energy has decreased from-37.467 to -38.411 REU, tested by RosettaDock. The most significant improve is lk_ball_wtd, or orientation-dependent solvation energy. Because of the hydrophobic area on the transcription factor’s interface, solvation energy has major effect. (Figure 16)<DIV class="imgState"><DIV><SPAN class="figureNum">Figure 16.</SPAN> Δ lk_ball_wtd on base pair 10-12
				 The energy term lk_ball_wtd means orientation-dependent solvation energy.
				 (Terms of Rosetta Energy Function can be found in <A class="contentLink" href="https://2018.igem.org/Team:Fudan-CHINA/Design">principles</A>)</DIV></DIV></DIV><DIV class="mainTitle1">
		References
	</DIV><DIV class="mainContents" style="margin: -40px 0 0 0;"><SPAN class="duanjian">
[1] Marcatili, P., Rosi, A. and Tramontano, A. (2008) PIGS: automatic prediction of antibody structures. Bioinformatics, 24, 1953-1954.
		</SPAN><SPAN class="duanjian">
[2] Chothia, C. and Lesk, A.M. (1987) Canonical structures for the hypervariable regions of immunoglobulins. J Mol Biol, 196, 901-917.
		</SPAN><SPAN class="duanjian">
[3] Morea, V., Tramontano, A., Rustici, M., Chothia, C. and Lesk, A.M. (1998) Conformations of the third hypervariable region in the VH domain of immunoglobulins. J Mol Biol, 275, 269-294.
		</SPAN><SPAN class="duanjian">
[4] Tramontano, A., Chothia, C. and Lesk, A.M. (1990) Framework residue 71 is a major determinant of the position and conformation of the second hypervariable region in the VH domains of immunoglobulins. J Mol Biol, 215, 175-182.
		</SPAN><SPAN class="duanjian">
[5] Chailyan, A., Marcatili, P., Cirillo, D. and Tramontano, A. (2011) Structural repertoire of immunoglobulin lambda light chains. Proteins, 79, 1513-1524.
		</SPAN><SPAN class="duanjian">
[6] Chailyan, A., Marcatili, P. and Tramontano, A. (2011) The association of heavy and light chain variable domains in antibodies: implications for antigen specificity. FEBS J, 278, 2858-2866.
		</SPAN><SPAN class="duanjian">
[7] Messih, M.A., Lepore, R., Marcatili, P. and Tramontano, A. (2014) Improving the accuracy of the structure prediction of the third hypervariable loop of the heavy chains of antibodies. Bioinformatics, 30, 2733-2740.
		</SPAN><SPAN class="duanjian">
[8] Adrian Velazquez-Campoy and Ernesto Freire. Isothermal titration calorimetry to determine association constants for high-affinity ligands. Nature Protocols, 2006, 1(1):186-191. doi:10.1038/nprot.2006.28
		</SPAN><SPAN class="duanjian">
[9] Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R., Heer, F.T., de Beer, T.A.P., Rempfer, C., Bordoli, L., Lepore, R., Schwede, T. SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. 46(W1), W296-W303 (2018). 
		</SPAN><SPAN class="duanjian">
[10] Bienert, S., Waterhouse, A., de Beer, T.A.P., Tauriello, G., Studer, G., Bordoli, L., Schwede, T. The SWISS-MODEL Repository - new features and functionality. Nucleic Acids Res. 45, D313-D319 (2017). 
		</SPAN><SPAN class="duanjian">
[11] Guex, N., Peitsch, M.C., Schwede, T. Automated comparative protein structure modeling with SWISS-MODEL and Swiss-PdbViewer: A historical perspective. Electrophoresis 30, 
		</SPAN><SPAN class="duanjian">
[12] Benkert, P., Biasini, M., Schwede, T. Toward the estimation of the absolute quality of individual protein structure models. Bioinformatics 27, 343-350 (2011). 
		</SPAN><SPAN class="duanjian">
[13] Bertoni, M., Kiefer, F., Biasini, M., Bordoli, L., Schwede, T. Modeling protein quaternary structure of homo- and hetero-oligomers beyond binary interactions by homology. Scientific Reports 7 (2017).
		</SPAN></DIV></DIV><DIV id="contactUs"><DIV class="cuCol" id="cuCol1" style="width: 17%;"><H1 class="cuH1">  Useful Links</H1><DIV class="cuContent"><A target="_blank" href="https://2018.igem.org/Main_Page"> iGEM Main Page</A><A target="_blank" href="https://2018.igem.org/Main_Page"> iGEM Special Pages</A></DIV></DIV><DIV class="cuCol" id="cuCol2" style="width: 17%;"><H1 class="cuH1">  Email</H1><DIV class="cuContent"><A href="mailto:igem@fudan.edu.cn"> igem@fudan.edu.cn</A><A href="mailto:kfjin16@fudan.edu.cn"> kfjin16@fudan.edu.cn</A></DIV></DIV><DIV class="cuCol" id="cuCol3" style="width: 30%;"><H1 class="cuH1">  Address</H1><DIV class="cuContent">
		G604, School of Life Sciences, Fudan University
		2005 Songhu Road, Yangpu, Shanghai, China
		</DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>