<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Fudan_Optimization skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Fudan/Optimization</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>2018 iGEM Team:Fudan - Optimization</TITLE></P><DIV id="FudanWrapper" class="white"><DIV id="FudanBody" class="white"><HEADER><NAV id="topNav" class="white z-depth-0_5"><DIV class="nav-wrapper"><UL id="nav-mobile" class="right orangeBorder"><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown1">Project</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown2">Dry lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown3">Wet lab</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger thisPageLink" data-target="dropdown4">Toolbox</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown5">Outreach</A></LI><LI class="hide-on-med-and-down"><A class="dropdown-trigger" data-target="dropdown6">Team</A></LI></UL></DIV></NAV><UL id="dropdown1" class="dropdown-content"><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL><UL id="dropdown2" class="dropdown-content"><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL><UL id="dropdown3" class="dropdown-content"><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL><UL id="dropdown4" class="dropdown-content"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><UL id="dropdown5" class="dropdown-content"><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL><UL id="dropdown6" class="dropdown-content"><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL><UL id="slide-out" class="sidenav"><LI style="padding: 0"><DIV class="sidenavBanner"><P style="width: 100%;text-align: center;font-size: 24px"><SPAN class="white-text">Optimization</SPAN></P></DIV></LI><LI>Navigator on this page</LI><LI class="onThisPageNav"><A href="#section1">Briefing</A></LI><LI class="onThisPageNav"><A href="#section2">Results</A></LI><LI class="onThisPageNav"><A href="#section3">Conclusion</A></LI><LI>Navigator on wiki</LI><LI><DIV class="collapsible-header">Project</DIV></LI><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI></UL></HEADER></DIV><LI><DIV class="collapsible-header">Dry Lab</DIV></LI><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></DIV><LI><DIV class="collapsible-header">Wet Lab</DIV></LI><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></DIV><LI><DIV class="collapsible-header">Toolbox</DIV></LI><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></DIV><LI><DIV class="collapsible-header">Outreach</DIV></LI><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></DIV><LI><DIV class="collapsible-header">Team</DIV></LI><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></DIV><DIV id="pageContent" style=""><DIV id="contentBanner" class="figureBanner orangeBg"><DIV class="row"><DIV class="col s12 hide-on-med-and-up"><H1>Optimization</H1></DIV></DIV><DIV id="figureBannerTitle" class="hide-on-small-only"><H1>Optimization</H1></DIV></DIV><DIV class="container"><UL id="pageContentNav" class="hide-on-med-and-down z-depth-0"><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI>Optimization</LI><LI class="onThisPageNav"><A href="#section1">Briefing</A></LI><LI class="onThisPageNav"><A href="#section2">Results</A></LI><LI class="onThisPageNav"><A href="#section3">Conclusion</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL><MAIN><DIV class="expFigureHolder" style="width:100%"><P>In our GJ presentation (10/25 Room 311 9:00-9:25), we used the image above to demonstrate the improvement we got for the Notch core domain.</P></DIV><DIV id="section1" class="section container scrolSpy"><H2>A quick briefing on Notch activation</H2><P>
                            Mature Notch receptors are cleaved at a S1 in the negative regulatory region (NRR) heterodimerization domain during its delivery to cell membrane. A currently widely accepted model of Notch activation begins as the receptor’s ectodomain binds to ligands on sender cells with high affinity. The Notch core is then &quot;opened up&quot; at its negative regulatory region (NRR) by the mechanical force exerted by Notch-bound ligand endocytosis process on signal-sender cell, exposing its cleavage sites to proteases ADAM10/17 and Gamma secretase, releasing the Notch intracellular domain (NICD). In the majority of cellular interactions, the free Notch intracellular domain then translocate into the cell nucleus via nuclear localization sequence to regulate downstream signaling. For wild type Notch, the Notch intracellular domain would interacted with its major downstream effector CBF-1/Suppressor of Hairless/Lag-1) (CSL) on their target DNA. Together they recruit co-factors to activate endogenous downstream transcription. For SynNotch, specialized factors will exclusively participate in regulation of genetic circuits that allow user-defined cellular responses.
                            n our project, from Dry lab results as well as Wet lab results, we realized that high functioning SynNotch receptors with high signal-to-noise ratio is its cornerstone. (See reference in <A href="Antigen_Receptors">Antigen, receptors page</A>.)</P><P>
                            So, in this section, we refer to the structural information from former very clarifying structural research on human Notch 1 and human Notch 2 receptors for their striking similarity to try to optimize the mouse Notch 1 receptor core for better performance of the SynNotch in our project. Through truncations and point mutations of the negative regulatory region (NRR), we have attained some results and characterization with better signal-to-noise ratio, though there is still a long way to go.
                        </P><DIV class="expFigureHolder row" style="width:100%"><P>
                                Figure 1. Published diagram and crystal structure of human Notch receptor. See <A href="/Team:Fudan/Antigen_Receptors">Antigen and Receptors</A> for more detailed information.
                            </P></DIV></DIV><DIV id="section2" class="section container scrolSpy"><H2>Notch core optimization</H2><H3>SynNotch and surface antigen (surAg)</H3><P>For our project’s Wet lab part, we first built a variety of SynNotch with the same synthetic intracellular domains (SynICDs), but has different synthetic extracellular domains (SynECDs). We chose three different Synthetic extracellular domains: LaG17 (low affinity single domain antibody against EGFP), LaG16-2 (high affinity scFv against EGFP) <A href="https://www.ncbi.nlm.nih.gov/pubmed/25362362">(Fridy PC, et al., 2014)</A>, αCD19 (scFv against CD19) <A href="https://www.ncbi.nlm.nih.gov/pubmed/23527958" target="_blank">(Grupp SA, et al., 2013)</A> with same SynICD: tTAA (tTA-Advanced. Its corresponding promoter is TRE3GV) which can theoretically respond to surface EGFP (surEGFP) and surface CD19 (surCD19), as summarized in Table 1.
                        </P><P>
                            We first propose a simple way of measuring the performance of SynNotch activation with:
                        </P><P><B>MFI<SUB>EGFP</SUB>:</B> Mean fluorescence intensity (MFI) of 293T cells activated and expressing EGFP as signal output of SynNotch activation in flow cytometry</P><P><B>Freq<SUB>EGFP++</SUB>:</B> Frequency of EGFP positive cells in parent cell population, in other words the percentage of cells that has SynNotch activated</P><P><B>MFI<SUB>mCherry</SUB>:</B> Mean fluorescence intensity of mCherry fluorescence internal control.</P><DIV class="tableHolder"><TABLE><TBODY><TR><TH>SynNotch</TH><TH>Class</TH><TH>ARF<SUB>aS</SUB><SUP>*)</SUP></TH><TH>Preferred</TH></TR><TR><TD>LaG17-mN1c-tTAA</TD><TD>II</TD><TD>1.67 ± 0.20</TD></TR><TR><TD>LaG16-2-mN1c-tTAA</TD><TD>I</TD><TD>1.89 ± 0.21</TD><TD>☆</TD></TR><TR><TD>αCD19-mN1c-tTAA</TD><TD>I</TD><TD>5.27 ± 0.29</TD><TD>☆</TD></TR><TR><TD>LaG17-hN1c-tTAA</TD><TD>II</TD><TD>1.46 ± 0.11</TD></TR></TBODY></TABLE><P>*) <B>ARF<SUB>aS</SUB>:</B> activation ratio fold of activating-form SynNotch</P></DIV><H3>SynNotch and surAg are delivered onto cell membrane</H3><P>The activation of SynNotch is dependent on cell-cell contact between ligand and receptor. In order to verify the function of SynNotch, the first thing we need to know is whether the protein and its corresponding surAg are expressed outside the cell membrane. We used immunofluorescence to verified the surface-EGFP (surEGFP), surface-CD19 (surEGFP) and LaG16-2-SynNotch are indeed expressed on the cell membrane. The sharp fluorescence outline confirms that these proteins are expressed outside of the membrane (Figure 1). Since LaG17 and LaG16-2 are similar in function, we did not test the membrane expression of LaG17. For αCD19-SynNotch, we tried another method by using flow cytometry staining. The αCD19-SynNotch cell shows a positive population comparing to the negative control. Overall, we confirmed that all of surEGFP, surCD19, anti-EGFP-, anti-CD19-SynNotch express on the membrane.
                        </P><DIV class="expFigureHolder" style="width:100%"><P>Figure 2. Confirmation of surface expression of surEGFP, surCD19, and the SynNotch with LaG16-2 NECD, by live-cell antibody staining.
                            </P></DIV><H3>Comparison of activation between different types of SynNotch with the same intracellular domain
                        </H3><P>Considering the inherent limited number of intracellular domains that will enter the cell nucleus and the range of output theoretically needed for our Amplifier layer, we chose the mouse Notch core as the foundation of our Receptor layer.
                        </P><DIV class="expFigureHolder" style="width:100%"><P><B>Figure 1. Comparison between different SynNotch receptors</B>
                                    a.	Mouse Notch core and human Notch core with the same extracellular domain and intracellular domain manifest different signal-to-noise ratio. While mouse SynNotch with LaG17 and tTAA as NECD and NICD exhibit significantly high activation to basal activation, the corresponding human SynNotch shows lower activation.
                                    b.	Performance of SynNotch receptors with different NECDs and the same NICD shows LaG17 SynNotch with the highest activation but also the highest leakage, while CD19 SynNotch shows relatively lower activation but the least leakage. In addition, SynNotch receptors with the same NECD, the one with NICD of tTA-Advanced shows better signal-to-noise ratio compared those with NICD of Gal4-VP64.
                            </P></DIV><P>Mouse Notch core is highly similar in amino acid sequence to human Notch core. Therefore, in this study, even though we do not have information on the exact structure of the mouse Notch NRR domain, we referred to human Notch 1 and 2 receptors’ NRR structures deciphered by Gordon and her colleagues in the past few years.
                        </P><P>On the basis of the mouse Notch core, we furthered compared the performance of SynNotch with different scFv (single chain variable fragment). The basal activation of anti-CD19 is the lowest compared to LaG16 -2 and LaG17, and it also has a moderate activation performance, making it the SynNotch with the highest signal-to-noise ratio. On the contrary, we found that although the activation rate of LaG17 SynNotch is the highest, so is its basal activation, making it the SynNotch with the lowest signal-to-noise ratio <FONT color="purple">(Figure 3b)</FONT>. SynNotch with the same NECD but different NICD comparison shows that SynNotch with tTAA is highly functionging, while SynNotch with Gal4-VP64 NICD shows poorer acitivity. Therefore, it indicates that choice of the extracellular domain as well as intracellular domain can influence SynNotch performance although the Notch core, which includes the protective negative regulatory region and transmembrane region, and the intracellular domain are the same.
                        </P><P>Hence, we chose LaG17 SynNotch with the mouse core and tTAA NICD that demonstrates high sensitivity but also high basal activation for optimization.
                        </P><H3>Exploring multiple truncation of the LNR-A domain in the Notch core</H3><P>Although Notch mutations has been associated with T-cell acute lymphocytic leukemia (T-ALL) and developmental disorders <A href="https://www.ncbi.nlm.nih.gov/pubmed/21157484" target="_blank">(Ferrando A., 2010;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/19075186" target="_blank">Gordon WR, et al., 2009;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/16424867" target="_blank">Mansour MR, et al., 2006)</A>, the optimization of SynNotch is conducted with the goal of adding to the characterization of the Notch core and SynNotch from a preliminary structural perspective in order to use it as a well-defined tool in synthetic biology in the future.
                        </P><P>Consistent with past studies that removed the lin12-repeats (LNR) <A href="https://www.ncbi.nlm.nih.gov/pubmed/15485896" target="_blank">(Sanchez-Irizarry C, et al., 2004;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/17401372" target="_blank">Gordon WR, et al., 2007;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/2164160 " target="_blank">Greenwald I, et al., 1990)</A>, deleting the LNR-A region in its entirety as well as deleting it with the LNR-AB linker that protects the S2 cleavage site both can lead to higher activation of SynNotch. This also seems to cause irregular activation activity. Of course, more repeats are needed for further verification. However, when the last helix of the LNR-A domain remains and the rest of it is deleted, there is a significant decrease in activation compared to the former two truncated versions, even slightly lower than intact Notch core. This is inconsistent with former studies that proposed LNR-A as a blockage against ADAM 10/17 metalloproteases to prevent premature activation via S2 site, smaller LNR-A domain might render easier activation of Notch, not harder. Further verification needs to be done.
                        </P><DIV class="expFigureHolder" style="width:100%"><P><B>Figure 2. Truncation of different parts of the LNR-A domain of LaG17 SynNotch</B>
                                a.	With LaG17 SynNotch as control, the more we removed from the LNR-A region, the higher activation. Though leaving helix3 in LNR-A makes the activation even lower than our control is a huge surprise, it is possible this is a deviation due to SynNotch expression.
                                b.	This is a graph that “translates” fluorescence data into a relative performance of truncated SynNotch to LaG17 SynNotch with normal Notch core. X axis indicates performance relative to wt Notch core without induction, while y axis is when surEGFP antigens are presented to Notch cells.
                                c-e. These three structures of the human Notch1 core from (Gordon et al., 2009) (PDB accession code 3eto) has been adapted to represent mouse Notch core. Grey domains indicate truncation.
                                ** Please note that amino acid numbers are consistent with human Notch1.
                            </P></DIV><H3>Addition of flexible glycine linker between the truncated NRR LNR-A domain and scFv can make up for the deletion to an extent</H3><P>From what we saw in Figure 1b where SynNotch with different ectodomains would exhibit differed activity, and from the truncation results, guided by implications from past research in NRR mechanism and ADAM10/17 <A href="https://www.ncbi.nlm.nih.gov/pubmed/29224781" target="_blank">(Seegar TCM, et al., 2017)</A>, it occurred to us that the steric hinderance of NECD would influence Notch cleavage, therefore adding residues back might lower the basal activation or lower induced activation. </P><P>From the results from Figure 4, it seems as if there is a threshold length or structural hinderance of LNR-A that is within the range of optimal enzymatic activity for ADAM10/17 cleavage. Of course, further experiments must be done to rule out possibilities of SynNotch malfunction.</P><DIV class="expFigureHolder" style="width:100%"><P><B>Figure 3. Adding glycine linker to truncated versions of LaG17 mouse SynNotch core</B>
                                    a,d. Adding glycine linker to the ΔLNR-A hlx3+ NRR significantly lowered activation. This needs further verification.
                                    b,e. Adding a six-glycine linker between the ΔLNR-A NRR and scFv lowered induced activation and little of noise. However, a nine- glycine linker raised both basal and induced activation.
                                    c,f. Adding a nine-glycine linker between the ΔLNR-ABlinker NRR could increase activation, but a bold addition of 39 glycines significantly lowered it.
                     </P></DIV><H3>Mutating the calcium binding site of LNR-A and LNR-AB linker for better SynNotch receptors, some of them showing improvement</H3><P>
                            Studies have shown that calcium depletion of the negative regulatory region <A href="https://www.ncbi.nlm.nih.gov/pubmed/19075186" target="_blank">(Gordon et al., 2009a</A>
                            and <A href="https://www.ncbi.nlm.nih.gov/pubmed/19701457" target="_blank">Gordon et al., 2009b;</A><A href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10669757" target="_blank"> Rand MD, et al., 2000)</A>
                            could activate Notch, implying the electrostatic interactions of calcium ions with LNR domains and heterodimerization domain are fundamental in holding the NRR together. It occurred to us that perhaps slightly maneuvering the interaction strength by changing neighboring side chains of amino acids could either lead to either high basal activation with higher sensitivity, or lower basal activation with same or lower activation. We first analyzed the how conserved these amino acids are and chose ones that are quite conserved. Mutating the amino acids into others with residues with diverged features, we found that changing the charge from negative to positive (Lysine, K) significantly sensitized SyNotch.
                        </P><DIV class="expFigureHolder" style="width:100%"><P><B>Figure 4. LNR-A calcium binding site and LNR-AB linker mutation shows improvement in LaG17 SynNotch performance</B>
                                    a,c,e.  Mutation of the D1433 to K and Q (for mouse, shown here with adapted NRR structure from Gordon’s work, it is D1458 for human) shows higher activation and lower basal activation than LaG17 SynNotch. Due to the variance of data, further repeats must be done to confirm. It can be seen in e. that there are five electrostatic interaction between the calcium ion at LNRA with D1433, N1436, V1438, D1451 and D1454 (for mouse).
                                    b,d,f. Mutation of the LNR-AB linker mostly shows functional SynNotch with heighten sensitivity. L 1457 (mouse) mutation to V and G (or A) showed LaG17 SynNotch with better signal-to-noise ratio. It can be seen in f., L1457 (for mouse) / L1482 (for human)‘s side chain is locked in a hydrophobic pocket.
                                    g. Evolutionary analysis on the NRR sequence with MEGA software. Though D1433 (for mouse) is not highly conserved, L1457 (for mouse) is.

                            </P></DIV><P>Similarly, we learnt from Gordon’s work in 2007 and 2009 that both natural Notch 1 and 2 heterodimerization domains contain a hydrophobic pocket that is plugged by a highly conserved Leucine, and the two amino acids in close proximity seems to help to hold the “plug” in place. Therefore, either changing the residue to more hydrophilic ones would lead to higher basal and induced activation, or changing it to hydrophobic ones might have the opposite result. In addition, the hinderance sterically exerted by amino acids with larger side chains might either make it “unpluggable”, or and amino acids with smaller side chains might make it “unplugged” with little mechanical force.
                        </P><P>Analyzing the results, it seems more likely that the size of side chains plays a significant role. Mutating Leucine to Valine and Glycine or Alanine is downsizing the side chains. So, the LNR-AB linker might be easier to pull out and give way to ADAM10/17 proteolysis.
                        </P><P>To find out if the same principle would apply to LaG16-2 SynNotch and anti-CD19 SynNotch, we did the leucine mutation on them. We didn’t have the time to finish and do more repeats to verify, but luckily mutated LaG16-2 SynNotch are still well functioning.
                        </P><DIV class="expFigureHolder" style="width:100%"><P><B>Figure 5. Further mutations on LaG16-2 SynNotch LNR-AB-linker shows functional results.</B></P></DIV></DIV><DIV id="section3" class="section container scrolSpy"><H2>Conclusion
                        </H2><P>Through carefully designed experiments, we have attained D1433K, D1433Q, L1457V, and L1457G (for mouse) mutations for LaG17-mNc-tTAA SynNotch optimization. Of course, we still need more experimental repeats to further verify their validity.
                        </P><DIV class="tableHolder"><TABLE><TBODY><TR><TH>SynNotch</TH><TH>Class</TH><TH>ARF<SUB>aS</SUB><SUP>#)</SUP></TH><TH>Preferred</TH></TR><TR><TD>LaG17-mN1c-tTAA</TD><TD>II</TD><TD>1.67 ± 0.20</TD></TR><TR><TD>LaG17-6G-LNRA[-]mN1c-tTAA</TD><TD>I</TD><TD>1.73 ± 0.29</TD><TD>☆</TD></TR><TR><TD>LaG17-LNRA cbs(D1433K)mN1c-tTAA</TD><TD>II</TD><TD>2.69 ± 0.11</TD><TD>☆</TD></TR><TR><TD>LaG17-LNRA cbs(D1433Q)mN1c-tTAA</TD><TD>II</TD><TD>Needs more repeat</TD></TR><TR><TD>LaG17-LNRAlnkr(L1457V)mN1c-tTAA</TD><TD>I</TD><TD>2.12 ± 0.39</TD><TD>☆</TD></TR><TR><TD>LaG17-LNRAlnkr(L1457G)mN1c-tTAA</TD><TD>II</TD><TD>2.55± 0.55</TD></TR></TBODY></TABLE><P>* <B>cbc:</B> calcium binding site; <B>lnkr:</B> linker; <B>mN1c:</B> mouse Notch 1 core.</P><P>#) <B>ARF<SUB>aS</SUB>:</B> activation ratio fold of activating-form SynNotch</P></DIV></DIV><DIV class="section container"><UL class="collapsible expandable"><LI style="list-style-type: none"><DIV class="collapsible-header">Click to see the Supplementary Data
                                </DIV><DIV class="collapsible-body"><H4>Supplementary Data
                                    </H4><P>The following at data graphs of the above results of means and SD error bars.
                                    </P></DIV></LI></UL><H4 style="margin-left: 0;">References</H4><OL id="ref" style="margin: 23px 0 0 0;"><LI>Ferrando, A. (2010). NOTCH mutations as prognostic markers in T-ALL (Nature Publishing Group).
                            </LI><LI>Fridy, P.C., Li, Y., Keegan, S., Thompson, M.K., Nudelman, I., Scheid, J.F., Oeffinger, M., Nussenzweig, M.C., and Feny (2014). A robust pipeline for rapid production of versatile nanobody repertoires. Nature methods 11, 1253--1260.
                        </LI><LI>Gordon, W.R., Roy, M., Vardar-Ulu, D., Garfinkel, M., Mansour, M.R., Aster, J.C., and Blacklow, S.C. (2009). Structure of the Notch1-negative regulatory region: implications for normal activation and pathogenic signaling in T-ALL. Blood 113, 4381-4390.
                        </LI><LI>Gordon, W.R., Vardar-Ulu, D., Histen, G., Sanchez-Irizarry, C., Aster, J.C., and Blacklow, S.C. (2007). Structural basis for autoinhibition of Notch. Nature structural and molecular biology 14, 295.
                        </LI><LI>Greenwald, I., and Seydoux, G.J.N. (1990). Analysis of gain-of-function mutations of the lin-12 gene of Caenorhabditis elegans.  346, 197.
                        </LI><LI>Grupp, S.A., Kalos, M., Barrett, D., Aplenc, R., Porter, D.L., Rheingold, S.R., Teachey, D.T., Chew, A., Hauck, B., Wright, J.F., et al. (2013). Chimeric antigen receptor-modified T cells for acute lymphoid leukemia. New England Journal of Medicine 368, 1509--1518.
                        </LI><LI>Mansour, M., Linch, D., Foroni, L., Goldstone, A., and Gale, R.J.L. (2006). High incidence of Notch-1 mutations in adult patients with T-cell acute lymphoblastic leukemia.  20, 537.
                        </LI><LI>Rand, M.D., Grimm, L.M., Artavanis-Tsakonas, S., Patriub, V., Blacklow, S.C., Sklar, J., Aster, J.C.J.M., and biology, c. (2000). Calcium depletion dissociates and activates heterodimeric notch receptors.  20, 1825-1835.
                        </LI><LI>Sanchez-Irizarry, C., Carpenter, A.C., Weng, A.P., Pear, W.S., Aster, J.C., Blacklow, S.C.J.M., and biology, c. (2004). Notch subunit heterodimerization and prevention of ligand-independent proteolytic activation depend, respectively, on a novel domain and the LNR repeats.  24, 9265-9273.
                        </LI><LI>Seegar, T.C., Killingsworth, L.B., Saha, N., Meyer, P.A., Patra, D., Zimmerman, B., Janes, P.W., Rubinstein, E., Nikolov, D.B., and Skiniotis, G. (2017). Structural Basis for Regulated Proteolysis by the α-Secretase ADAM10. Cell 171, 1638-1648. e1637.
                            </LI></OL></DIV></MAIN></DIV><DIV id="abstractContent" class="z-depth-2"><DIV class="container"><H2 style="margin: 0;padding: 10px 0;">Abstract</H2><P style="margin: 0">Contact-dependent signaling is critical for multicellular biological
                        events, yet customizing contact-dependent signal transduction between
                        cells remains challenging. Here we have developed the ENABLE toolbox, a
                        complete set of transmembrane binary logic gates. Each gate consists of
                        3 layers: Receptor, Amplifier, and Combiner. We first optimized synthetic
                        Notch receptors to enable cells to respond to different signals across the
                        membrane reliably. These signals, individually amplified intracellularly by
                        transcription, are further combined for computing. Our engineered zinc finger-based
                        transcription factors perform binary computation and output designed products.
                        In summary, we have combined spatially different signals in mammalian cells,
                        and revealed new potentials for biological oscillators, tissue engineering,
                        cancer treatments, bio-computing, etc. ENABLE is a toolbox for constructing
                        contact-dependent signaling networks in mammals. The 3-layer design principle
                        underlying ENABLE empowers any future development of transmembrane logic circuits,
                        thus contributes a foundational advance to Synthetic Biology.
                    </P></DIV></DIV><FOOTER id="FudanFooter" class="page-footer grey"><DIV class="container"><DIV class="row"><DIV id="sponsor" class="col m3 s12 row"><H3 class="col s12" style="text-align: left; color: rgba(255, 255, 255, 0.8); font-size: 18px">ENABLE: making cells even smarter</H3></DIV><DIV id="usefulLinks" class="col m9 s12 row"><DIV class="col s12 l6 row"><DIV class="col s12 m4"><SPAN>Project</SPAN><UL><LI><A href="/Team:Fudan/Demonstrate">Demonstration</A></LI><LI><A href="/Team:Fudan/Antigen_Receptors">Antigen, Receptors</A></LI><LI><A href="/Team:Fudan/Results">Transmembrane logic</A></LI><LI><A href="https://2017.igem.org/Team:Fudan">2017.iGEM</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Dry lab</SPAN><UL><LI><A href="/Team:Fudan/Addon#ribo">Addon: ribo</A></LI><LI><A href="/Team:Fudan/Addon#TALE">Addon: TALE</A></LI><LI><A href="/Team:Fudan/Addon#T2">Addon: T2</A></LI><LI><A href="/Team:Fudan/Model#Transcriptional_Amplifer">Model: transcriptional amplifer</A></LI><LI><A href="/Team:Fudan/Model#NotchLigandKinetics">Model: Notch-ligand kinetics</A></LI><LI><A href="/Team:Fudan/Software">Software</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Wet lab</SPAN><UL><LI><A href="/Team:Fudan/InterLab">iGEM interLab</A></LI><LI><A href="/Team:Fudan/Notebook">Our notebook</A></LI><LI><A href="/Team:Fudan/Primers">Primers used</A></LI><LI><A href="/Team:Fudan/Protocols">Protocols</A></LI><LI><A href="/Team:Fudan/Safety">Safety</A></LI></UL></DIV></DIV><DIV class="col s12 l6 row"><DIV class="col s12 m4 active"><SPAN>Toolbox</SPAN><UL><LI><A href="/Team:Fudan/Basic_Parts">Basic parts</A></LI><LI><A href="/Team:Fudan/Composite_Parts">Composite parts</A></LI><LI><A href="/Team:Fudan/Optimization">Optimization</A></LI><LI><A href="/Team:Fudan/Parts_Collection">Parts collection</A></LI><LI><A href="/Team:Fudan/Improve">Parts improvement</A></LI><LI><A href="/Team:Fudan/Measurement">Quantification</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Outreach</SPAN><UL><LI><A href="/Team:Fudan/Bio-Art">Bio-Art display</A></LI><LI><A href="/Team:Fudan/Collaborations">Collaborations</A></LI><LI><A href="/Team:Fudan/Design_Intention">Design intention</A></LI><LI><A href="/Team:Fudan/Human_Practices">Human practices</A></LI><LI><A href="/Team:Fudan/Public_Engagement">Public engagement</A></LI></UL></DIV><DIV class="col s12 m4"><SPAN>Team</SPAN><UL><LI><A href="/Team:Fudan/Acknowledgement">Acknowledgement</A></LI><LI><A href="/Team:Fudan/Attributions">Attributions</A></LI><LI><A href="/Team:Fudan/Heritage">Heritage</A></LI><LI><A href="/Team:Fudan/Team">Members</A></LI><LI><A href="/Team:Fudan/Sponsors">Sponsors</A></LI></UL></DIV></DIV></DIV></DIV></DIV><DIV class="footer-copyright"><DIV class="container"><DIV class="contactUS row"><DIV class="col s12 m6 l4"> Life Sci Bldg E301, 2005 Songhu Rd, Shanghai
                           </DIV><DIV class="col s12 m6 l2"> +86-21-31246727
                           </DIV><DIV class="col s12 m6 l2"> igem@fudan.edu.cn
                           </DIV><DIV class="col s12 m6 l4"> Fudan_iGEM
                           </DIV></DIV></DIV></DIV></FOOTER></DIV></DIV></BODY></HTML>