<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_NU_Kazakhstan_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:NU Kazakhstan/Notebook</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Nazarbayev University Team</TITLE></P><DIV class="header-bottom"><DIV class="container"><DIV class="top-nav cl-effect-5"><UL class="nav1"><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan"><SPAN data-hover="Home">Home</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/Team"><SPAN data-hover="Team">Team</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/Project"><SPAN data-hover="Project">Project</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/Parts"><SPAN data-hover="Parts">Parts</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/Notebook"><SPAN data-hover="Notebook">Notebook</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/Attributions"><SPAN data-hover="Attributions">Attributions</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/InterLab_Study"><SPAN data-hover="Interlab_Study">Interlab Study</SPAN></A></LI><LI><A href="https://2015.igem.org/Team:NU_Kazakhstan/Practices"><SPAN data-hover="Practices">Practices</SPAN></A></LI></UL></DIV></DIV></DIV><DIV class="container"><DIV class="news"><H2>Notebook</H2><H3>1.06.15</H3><P><OL><LI>Preparation of the LB agar
We used 37 g of nutrient agar for 400 mL of distilled water</LI><LI>Extraction of genome from S.mutans</LI><LI>First, culture S.mutans in 5 mL liquid  BHI + bacitracin</LI><LI>Centrifuge for 15 min at 4000 rpm</LI><LI>Then dissolve the pellet in 500 microliters of Lysis Buffer<B>How to prepare Lysis Buffer (1 mL):</B>Lysis Buffer contains EDTA,
         Tween 80%, tris HCl,125 microliters of 8 M EDTA,5 µl of Tween 80,
         Tris HCl 1M 50 µl, Proteinase K (200 µg/mL). 0.0002 grams of powder Proteinase K were put into 1 mL of Lysis Buffer.                                                      The balance could not read 0.0002 g of proteinase K, so 0.02 g of proteinase K were taken.</LI></OL><LI>Incubation for 2 hours at 55°C. Heat at 90°C for 5 minutes.</LI><LI>Then add equal volume of cold isopropyl alcohol.</LI><LI>Incubate in freezer for 20 minutes.</LI><LI>Centrifuge at the maximum speed for 30 min. Remove the supernatant.</LI><LI>Add enough amount of ethanol to the pellet in order to wash</LI><LI>Then add 50 µl of TE buffer</LI><LI>Add 0.5 µl of the RNAase</LI><LI>Incubate at 37°C for 1 hour</LI><LI>Inactivate at 60°C for 10 min</LI><LI>Run it in an agarose gel</LI></P><H3>2.06.15</H3><OL><LI>Construction of the light system
pFixK2(K592006) + rbs + tetR(C0040) + double terminator + Ptet(R0040) + rbs + RFP(J06505) + double terminator
 </LI><LI>[rbs] = 139.15 ng/µl</LI><LI>[Ptet  + GFP] = 199.2 ng/µl</LI><LI>[pFixK2] = 131.6 ng/µl</LI><LI>[double terminator] = 70.21 ng/µl</LI><LI>[tetR] = 78.76 ng/µl</LI><LI>Restriction digests: 
Protocol for Restriction digest:
Take following amounts of reagents:
-1000 ng of DNA
-0.5 µl of each Restriction Enzyme(we used NEB enzymes)
-5 µl of CutSmart Buffer
-dH2O to get final volume of 50 µl
Incubate this mixture at 37 C for 1-2 hours and heat inactivate for 20 min at the temperature needed for particular enzyme.<B>Example mixture:</B>
-pFixK2 (250 bp) was cut with EcoRI and SpeI.
</LI><LI>pFixK2 – 7.6 µl</LI><LI>EcoRI  = 0.5 µl</LI><LI>SpeI = 0.5 µl</LI><LI>dH2O = 36.4 µl</LI><LI>cut smart = 5 µl</LI><LI>Overall: 50 µl</LI>
-RBS (15 bp) was cut with .
-TetR (685 bp) was cut with EcoRI and SpeI.
-Double terminator (95 bp) was cut with XbaI.<LI>Gel extraction of the pFixK2, RBS, tetR and double terminator
</LI><LI>Invitrogen by Life Technologies PureLink Quick Gel Extraction Kit was used to purify DNA.</LI><LI>The small area of the gel containing the DNA fragment of interest was cut under UV.</LI><LI>Mass of FixK2 = 220 mg, RBS = 80 mg, tetR = 150 mg, dTer = 140 mg</LI><LI>The protocol of the PureLink Quick Gel Extraction was used to dissolve the gel and extract the DNA
</LI><LI>[FixK2] = 5.727 ng/µl</LI><LI>[Rbs] = 4.499 ng/µl</LI><LI>[tetR] = 5.216 ng/µl</LI><LI>[double terminator] = 2.694 ng/µl</LI><LI>Ligation of the Parts:
Protocol for Ligation:
-Take 50 ng of vector or just of bigger DNA part
-Take amount of smaller DNA part to have 1:3 molar ratio
-1 µl of T4 ligase enzyme
-2 µl of T4 buffer
-Add dH2O to have final volume of 20 µl
Incubate this mixture for 16 hours at 16 C and heat inactivate for 10 min at 65 C.<B> Example Ligation mixture:</B> 
-Ligation of pFixK2 + rbs
</LI><LI>pFixK2 = 8.5 µl</LI><LI>RBS = 8.5 µl</LI><LI>T4 ligase = 1 µl</LI><LI>T4 buffer = 2 µl</LI><LI>Overall: 20 µl</LI>
-Ligation of TetR + double terminator 
When ligated DNA was transformed, the plate with pFixK2 + RBS had colonies grown.
The mini-prep of pFixK2 + rbs was done<H3>3.06.15</H3><OL><LI>The concentrations of the transformed parts:
</LI><LI>FixK2+ rbs= 75.79 ng/µl</LI><LI>FixK2+ rbs= = 72.80 ng/µl</LI><LI>TetR + dter= 56. 93 ng/µl</LI><LI>TetR + dter= 95.86ng/µl</LI><LI>Pveg= 84.84 ng/µl</LI><LI>FixJ= 161.1 ng/µl</LI><LI>PCR (Thermo Scientific Phusion High Fidelity PCR Master-mix):<B>Protocol for PCR reaction:</B>
Take following reagents:
-10 ng of DNA 
-10 µl of PCR MasterMix
-2 µl of each 5 µM Primer
-Add dH2O to get final volume of 20 µl<B>Example PCR mixture:</B>
-PCR of FixK2
</LI><LI>DNA = 0.03 µl *10 reactions = 0.3 µl</LI><LI>Water = 5.97 µl= 59.7 µl</LI><LI>Master Mix = 10 µl= 100 µl</LI><LI>VF2 = 2µl = 20 µl</LI><LI>VR= 2 µl = 20 µl</LI><UL>
-PCR of RBS
-PCR of FixK2 + rbs
-PCR of tetR+dter
-PCR of tetR
-PCR of Double terminator
Result:
-The ligation of the FixK2+ rbs did not work
-The ligation of the tetR+ dter worked<LI>Restriction Digests of:
-tetR
-Double terminator</LI><H3>4.06.15</H3><OL><LI>PCR of the ligated parts after transformation and mini-prep:
-PCR of [FixK2+rbs+tetR] = 108.54 ng/µl
-PCR of [tetR+ double terminator] = 166 ng/µl</LI><LI>Running of PCR products on a gel in the following order:</LI><LI>Ladder</LI><LI>FixK2+ RBS</LI><LI>FixK2+ rbs+tetR</LI><LI>TetR</LI><LI>tetR+ dter</LI></OL><H3>5.06.15</H3><P>Protocol for making the competent dh5alpha</P><OL><LI>Inoculate a single colony into 5 mL LB in 50 mL falcon tube (taped on a loosed tap)</LI><LI>Grow on 37°C with shaking for 130 rpm overnight</LI><LI>Use 1 mL to inoculate 100 mL to inoculate 100 mL of LB in 250 mL bottle the next morning</LI><LI>Shake 37°C for 1.5-3 hours</LI><LI>When OD is between 0.3-0.4 put cell on ICE</LI><LI>Hold cells on ice for the 10 minutes</LI><LI>Collect cells by centrifugation for 3 min at maximum speed (4700 rpm)</LI><LI>Decant supernatant and gently resuspend on 10 mL ice-cold 0.1 M CaCl2  (prepared in ddH2O). Treat them gently.</LI><LI>Incubate on ice for 20 minutes.</LI><LI>Centrifuge again at maximum speed (4700 rpm)</LI><LI>Discard supernatant and gently resuspend in 5 mL cold 0.1 M CaCl2 (15% glycerol)</LI><LI>Dispence into chilled microtubes, put on the dry ice. Perform this procedure very quickly!</LI><LI>Freeze in -80°C</LI></OL><H3>6.06.15</H3><P><OL><LI>PCR clean-up of: 
-[FixK2+rbs] = 30.15 ng/µl
-[tetR] = 57.47 ng/</LI><LI>Gel extraction after digestion:
-[FixK2 + rbs] = 5.862 ng/µl
-[tetR] = 4.621 ng/µl</LI><LI>
Ligation with Fermentas enzymes:
-[tetR] = 20/4.621ng/µl = 4.3 µl + [FixK2+ rbs] = 40ng/5.862 ng/µl = 7 µl
-[tetR]=4 ng/µl + [double terminator]=2ng/µl
-TetR + double terminator
-Pveg + FixJ</LI><LI>Ligation with NEB enzymes:
-Pveg + FixJ
-FixK+ rbs+tetR
-[FixK2 + rbs]=10.66 ng/µl + [tetR]=23.08ng/µl
-TetR+double terminator</LI></OL>
Results: Ligation  of tetR + double terminator (NEB) 50 ng: 50 ng have worked.
</P><H3>8.06.15</H3><P>Transformation of Ligated products:
<UL><LI>Pveg + FixJ</LI><LI>tetR+ double terminator</LI><LI>FixK2+rbs+tetR</LI><LI>FixK2+rbs+tetR (PCR)</LI><LI>FixK2+rbs+tetR</LI><LI>FixK2+tetR+ double terminator</LI><LI>tetR+ double terminator</LI></UL></P><H3>9.06.15</H3><P>PCR after ligation:
-tetR+ double terminator
-FixK2+rbs+ tetR
-Pveg + FixJ
-FixK2+rbs+tetR             
-tetR+ double terminator
-tetR+ double terminator (fermentas)
Results: Only ligation of the tetR + double terminator with NEB enzyme have worked</P><H3>10.06.15</H3><P>Restriction Digest: 
-FixK2+ rbs
-TetR + double terminator</P><H3>11.06.15</H3><P>1.Gel extracts of:
[Pveg ] = 3.174 ng/µl
[FixJ] = 2.9045 ng/µl
2.Ligation of Pveg + FixJ</P><H3>12.06.15</H3><P>PCR of S.mutans 16s  rRNA with synthesized primers

Figure 1: There were two types of colonies growing on bacitracin selective plate. They are expected to be S. mutans and S.sobrinus. It was suggested that Sm479F/R primer pair is highly specific for identification of S.mutans(Chen et al.,2007). In the picture you see two sets of lanes corresponding to the two types of colonies from plate, and first set of lanes is of correct size.</P><H3>14.06.15</H3><P>1.Digest of the Pet 21(plasmid) with NotI
2.Digest of the GFP with NotI
3.Ligation of Pveg with FixJ
4.Digest of FixK2+rbs with SpeI  (NEB)
5.Digest of FixK2+rbs with Aah1  (Syberian Enzyme)
Results: The size of FixK2+rbs (265 bp) does not coincide with the gel</P><H3>15.06.15</H3><P>1.Transformation of parts from Kit 2014:<UL><LI>Promoter FixK2 (Plate 1, 19G) – 250 bp</LI><LI>RBS (Plate 4, 4G) – 15 bp</LI><LI>FixJ  (Plate 1, 10N) – 1796  bp</LI><LI>RFP mCherry with double terminator (Plate 1, 13K) = 861 bp</LI><LI>Promoter Veg  (plate1, 20G) = 237 bp</LI></UL>
2.Genome extraction from S.mutans with Instagene Matrix
3.PCR of 16S rRNA of the S.mutans</P><H3>16.06.2015</H3><P>1.Mini-prep: 
<OL><LI>pFixK2 = 78.03 ng/µl</LI><LI>Rbs = 171.8  ng/µl</LI><LI>FixJ = 124.5 ng/µl</LI><LI>Rfp + double terminator = 82.75 ng/µl</LI><LI>Promoter Veg = 42.20 ng/µl</LI></OL> 
2.Double Digest of FixK2 with SpeI and EcoRI
3.Single digest of FixK2 with SpeI
4.Double Digest of RBS with EcoRI and XbaI</P><H3>17.06.2015</H3><P>Gel extraction of FixK2 and rbs 
-FixK2 (double digest) = 4.128 ng/µl
-FixK2 (single digest) = 3.315 ng/µl
-Rbs = 3.051 ng/µl
2.Ligation of FixK2 (double digested) + rbs
3.Ligation of FixK2(single digested) + rbs in order to obtain linear plasmid 
4.Double digest of:
-Pveg (EcorI-SpeI) 
-FixJ (EcorI-XbaI)
-RFP (EcorI - XbaI)
5.Transformation of Fixk2 + rbs.</P><H3>18.06.2015</H3><P>1.Double check Transformation of [Fixk2 + rbs]
We did not simultaneously perform digest and ligation since we wanted to check the work of enzymes in double digest
2.Gel extraction of parts: Pveg,  FixJ, RFP
3.Ligation reaction:
-Pveg + FixJ
-Pveg +RFP
4.Chrm and Bacitracin plates were done (Bacitracin Mol. weight is 1422.71 g/mol  and the concentration in stock (ex: 500 ml LB agar) should be 0.2  µM, while the conc. of bacitracin is 50 mg/ml):
(50 mg/ml)/ 1422. 71 = 0.035 M in eppendorf tube
0.035  M  x Y= 0.2 µM x 500 ml
Y = 0.00286 ml = 2. 86 µl
Chrm conc: 500 µl for 500 ml of LB agar
5. Miniprep of Fixk2+rbs:
[FixK2+ rbs] = 217.7 ng/µl</P><H3>19.06.2015</H3><P><OL><LI>Transformations of parts:</LI><LI>dCas9 under xylose  (2015 Kit, plate=5, 8L) chloromphenicol resistant</LI><LI>Anderson promoter (plate 1, 20 K) chloromphenicol resistant</LI><LI>Anderson promoter (plate1, 22A)</LI><LI>Anderson promoter (plate 1, 22 K)</LI><LI>GFP (plate 4, 21J)</LI><LI>Pveg+ FixJ</LI><LI>Pveg+ RFP</LI><LI>Pveg+ FixJ  (20 µl)</LI><LI>Pveg + RFP (20 µl )</LI></OL><LI>PCR confirmation of the Fixk2+rbs ligation part obtained from miniprep
</LI><LI>ladder</LI><LI>FixK2</LI><LI>FixK2 + rbs</LI><LI>FixK2 + rbs</LI>
Results: Size of FixK2, FixK2 + rbs do not coincide with the right one.<LI>Genome extraction from S.mutans by Instagene Genome Extraction and PCR
RESULTS:
S.Mutans = No band amplification
FixK2 = 600 bp part was shown as amplified
</LI></P><H3>25.06.2015</H3><P><OL><LI>PCR of S.mutans. Plate #2 (grown from single colony). Genome isolated by Instagene Matrix</LI><LI>PCR of S.mutans (colony taken from plate #2)</LI><LI>Negative control with the Bacillus Subtilis genome. Genome of the Bacillus was extracted with Instagene matrix</LI></OL>
Results:  The amplicon of size 479 base pair is detected. First raw is the ladder, second is the amplicon which was PCR-ed with extension time 1 min, third is the amplicon with extension time of 14 seconds.</P><H3>30.06.2015</H3><OL><LI>Transformation of FixK2 ( Kit 2015, Plate 1, 19G)</LI><LI>Restriction digest of FixJ with EcoRI and XbaI</LI><LI>Restriction Digest of Pveg with EcoRI and SpeI </LI><LI>Gel extraction of the FixJ and Pveg 
FixJ = 2.217 ng/ul 
Pveg =2.892 ng/ul</LI><LI>Ligation was performed in two ways: 
-DNA was taken from gel extraction
-DNA was taken directly from digestion solution without clean-up </LI><H3>08.07.2015</H3><P>1. Sequential Digest of Pveg with SpeI (sib enzyme) and EcoRI (fermentas)
Result: Pveg of size 237 bp was seen on an agarose gel</P><H3>9.07.2015</H3><P>1. Transformations:
<OL><LI>FixK2+rbs</LI><LI>GFP</LI><LI>Anderson promoter 101</LI><LI>Anderson promoter 106</LI><LI>Anderson promoter 117</LI>
2. Digest of the RFP mcherry + double terminator with Invitrogen EcoRI and XbaI
3. Gel extraction of the Pveg that was sequentially digested with SpeI (syberian enzyme) and EcoRI (fermentas):
Pveg = 30.78 ng/ul
mcherryRFP + double terminator (digested with EX from Invitrogen) = 9.36 ng/ul 
4. Ligation of Pveg+ mcherryRFP with double terminator</OL></P><H3>10.07.2015</H3><P>1. Checking Ligation of Pveg+FixJ mini prep product by making sequential digest with SpeI (Syberian enzyme) and EcoRI (fermentas)
2. Single digest of Pveg+FixJ with the EcoRI (Neb enzyme) 
3. Transformation </P><H3>5.08.2015</H3><P>1.PCR of:
<UL><LI>GFP</LI><LI>Anderson promoter 101</LI><LI>Anderson promoter 106</LI><LI>Anderson promoter 117</LI><LI>plasmid backbone</LI><LI>sgRNA for VicK</LI></UL></P><H3>6.08.2015</H3><P>1. CPEC of sgRNA for VicK and plasmid backbone.
2. miniprep of pet21 plasmid from S.mutans
3. restriction digest and ligation of GFP with each Anderson promoter</P><H3>7.08.2015</H3><P>1. PCR of ligation products of GFP with Anderson promoters</P><H3>8.08.2015</H3><P>1. CPEC of GFP+Anderson promoter 101  with plasmid backbone</P><H3>9.08.2015</H3><P>1. minipreps of pet21+sgRNA(Vick) and sgRNA(VicK) in a plasmid backbone
2. PCR of pVeg and FixJ
3. Restriction digest of pVeg and FixJ
4. Ligation of pVeg and FixJ</P><H3>10.08.2015</H3><P>1. CPEC of GFP+Anderson promoters 106 and 117 with plasmid backbone
2. PCR of parts:
<UL><LI>TetR+dt+pTet</LI><LI>pVeg+FixJ</LI></UL></P><H3>12.08.2015</H3><P>1. Transformation of GFP+Anderson promoters in plasmid backbone</P><H3>17.08.2015</H3><P>
1.	Vector pMSP3535 for Nisin controlled expression for Gram-Positive bacteria kindly provided by Dr.Gary Dunny's Lab. 
2.	                    Restriction digest of sgVicK 
•	SgVicK = 6.3 ul 
•	3.1 NEB buffer = 5 ul 
•	PstI (Neb) = 1 ul 
•	XbaI = 1 ul 
•	S.H2O = 36.7 ul 
3.	                 Restriction digest of XrpA 
•	xrpA = 4 ul 
•	3.1 buffer = 5 ul 
•	PstI = 1 ul 
•	XbaI = 1 ul 
•	s.water = 39 ul 
4.	                Restriction digest of pMSP3535 
•	pMSP3535= 4 ul 
•	3.1 buffer = 5 ul 
•	PstI = 1 ul 
•	XbaI = 1 ul 
•	s.water = 39 ul 
5.	               Ligation of pMSP3535 + XrpA 
•	xrpA (after gel extract) = 1 ul 
•	vector = 5.2 ul 
•	T4 buffer = 1 ul 
•	T4 ligase = 0.5 ul NEB 
•	S.water = 2.3 ul 
6.	                      Ligation of pMSP3535 + sgVicK 
•	sgVicK = 1 ul 
•	vector = 1.5 ul 
•	T4 buffer = 1 ul 
•	T4 ligase = 0.5 ul 
•	s.water = 6 ul 
 Results: Ligation was done overnight. After that, transformation was done. Colonies were screened and successful incorporation can be seen from figure

Figure 1. Mini prep products cut with PstI and XbaI. First line is ladder, 2,3,4 lines are xrpA incorporated into pMSP3535, 5 line is sgVicK incorporated into pMSP3535. 
Size of pMSP3535 is 8354 base pairs, xrpA under pH sensitive promoter is 475 base pairs, and sgVicK is 275 base pairs</P><H3>sgRNA for VicK construction</H3><P>1. sgVicK gene was synthesized at TechnoPark of Nazarbayev University and size is 275 base pairs. sgVicK gene was received in pGEM-T Easy Vector with ampicillin resistance. Mini - prep was done and concentration was measured with Nanodrop. 
2. sgRNA for VicK gene was amplified with Prefix-F and Suffix-R primers. After that sgRNA for VicK gene was pcr purified using  QIAquick PCR Purification Kit
3. Circular polymerase extension cloning (CPEC) method was used for incorporation of sgVicK into chloramphenicol linearized plasmid backbone, pSB1C3</P><H4> Reaction set up for CPEC</H4>

1.Linearized plasmid backbone – 200ng 
2.sgVicK PCR purified product – equimolar to backbone 

3.High Fidelity Master Mix – 10 ul 

4.DMSO – 0.75 ul 

5.Sterile water to 25 ul  

      PCR program for CPEC: 

1. 98°C = 30sec

2. 98°C = 10 sec

3. 55°C = 30 sec 

4. 72°C = 70 sec

5. 15X 

6. 72°C = 10 min 

7. 4°C  

4. 5 ul of CPEC reaction was transformed into dh5alpha strain of E.coli.

Next day, one colony was put into mini- prep and concentration was 

measured with Nanodrop. Mini-prep product of pSB1C3 + sgVicK was 

cut with NotI restriction enzyme. Also, mini prep product was amplified 

with Prefix-F and Suffix-R primers. 

Results: Second line is pcr product of mini prep and there is a band on 

275 base pairs according to size of sgVicK. Third line is mini prep 

product cut with NotI and there is a band of 275 base pairs. These 

results indicate that sgVicK gene successfully incorporated into 

chloramphenicol resistant plasmid backbone

Figure 1. First line is ladder, second is pcr amplified product of sgVicK 

(275 bp), mini prep product digested with NotI
</OL></UL></OL></OL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>