<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Model_Environment_Sensing_parameter_space skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Model/Environment Sensing/parameter space</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="eth-main-contents" class="eth-main-contents"><HEADER id="header"><NAV><UL class="main-menu col-to-row enlarge-line-height custom_cursor"><LI onclick="return true"><SPAN class="menu-entry">Project Idea</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Description">Story of CATE</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Applied_Design">CATE in Action</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Circuit">Circuit</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Design">Design</A></LI></UL><LI onclick="return true"><SPAN class="menu-entry">Engineering</SPAN></LI><LI onclick="return true"><SPAN class="dry">Dry Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Environment_Sensing">Tumor Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Heat_Sensor">Heat Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Vivo">3D Model</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Silico_Final">Final Integration</A></LI><UL class="subsubmenu"><LI onclick="return true"><SPAN class="wet">Wet Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Notebook">Lab Journal</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Protocols">Protocols</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Safety">Safety</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sequences">Sequences</A></LI></UL></NAV></HEADER></DIV><LI onclick="return true"><SPAN class="menu-entry">Achievements</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Results">Results</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Parts">Parts</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Medal_Criteria">Medal Criteria</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/InterLab">InterLab</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI onclick="return true"><SPAN class="menu-entry">Human Practices</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Human_Practices">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Gold_Integrated">Integrated Practices</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Silver">Public Engagement</A></LI><LI onclick="return true"><SPAN class="menu-entry">Our Team</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Team">Members</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sponsors">Sponsors</A></LI><MAIN role="main"><H1 class="headline">Model-based design guidelines used to choose our parts rationally</H1><SECTION class="abstract"><H1 class="goal">Why do we need to search for functional parameters?</H1><P>A biological circuit often has different functioning regimes and can only achieve a particular and interesting behavior for a few given combinations of parameters (among which protein expression levels, or promoter sensitivity and leakiness for example). This is why we had to get some insights into the sets of parameters of our circuit and determine a target combination that would make our circuit work, to give out design guidelines for the choice of the parts we would use in the lab.
</P><P><STRONG>Please check the <A href="https://2017.igem.org/Team:ETH_Zurich/Model/Environment_Sensing/model">full detailed model</A> if you are interested in knowing how our whole model works.</STRONG></P></SECTION><SECTION id="parameter_search_sec"><H1>Parameter search</H1><P>Even before we got our first parts cloned and characterized, we attempted to predict the requirements that they should meet to achieve the criteria <A href="https://2017.igem.org/Team:ETH_Zurich/Model/Environment_Sensing/system_specifications">previously</A> established from literature data. For this, we extensively explored the parameter space controlling our model, and simulated the response of potential systems to find subsets of parameter combinations satisfying the performance specifications needed to get a sensitive as well as specific tumor sensing circuit.</P><FIGURE class="fig-nonfloat" style="max-width: 600px;"><FIGCAPTION>Figure 1. Goal of the parameter search: deduce genetic design guidelines</FIGCAPTION></FIGURE><P>The parameters satisfying the specifications are called functional space (green ellipse on Figure 1). We selected the biological parts that were the most likely to fall in the functional parameter space.</P><DIV class="multi-summary"><DETAILS><SUMMARY>Different categories of parameters</SUMMARY><P>Our model is relying on a dozen of parameters, some of which we can have a leverage on (typically maximal expression of the proteins, via RBS tuning), and others not (binding constants, promoter leakiness...). Some of these latter parameters have been precisely characterized and others are not very well known. This is why we have chosen to set some parameters to a certain value when we could find a reasonably reliable source in the literature, or when their influence would be redundant with other parameters (such as protein degradation rates and maximal expression rates that can alleviate each other's influence when co-varied), and leave other parameters free to vary to check their influence on our system.</P><H2>Fixed parameters, because well known</H2><TABLE><TBODY><TR><TH>Symbol</TH><TH>Description</TH><TH>Value</TH><TH>Reference</TH></TR><TR><TD>a<SUB>AHL</SUB></TD><TD>AHL synthesis rate by LuxI</TD><TD>0.01 min<SUP>-1</SUP></TD><TD><A href="#bib2" class="forward-ref">[2]</A></TD></TR><TR><TD>d<SUB>AHL</SUB></TD><TD>AHL degradation rate</TD><TD>5x10<SUP>-4</SUP> min<SUP>-1</SUP></TD><TD><A href="#bib4" class="forward-ref">[4]</A></TD></TR><TR><TD>D</TD><TD>AHL diffusion coefficient in water</TD><TD>3x10<SUP>-8</SUP> m<SUP>2</SUP>min<SUP>-1</SUP></TD><TD><A href="#bib4" class="forward-ref">[4]</A></TD></TR><TR><TD>K<SUB>LuxR-AHL</SUB></TD><TD>LuxR-AHL quadrimer binding constant</TD><TD>5x10<SUP>-10</SUP> nM<SUP>-3</SUP></TD><TD><A href="#bib5" class="forward-ref">[5]</A></TD></TR><TR><TD>w</TD><TD>Width of the colonized shell area</TD><TD>5x10<SUP>-10</SUP> nM<SUP>-3</SUP></TD><TD><A href="#bib5" class="forward-ref">[5]</A></TD></TR></TBODY></TABLE><H2>Fixed parameters, not very well known but redundant with other parameters</H2><TABLE><TBODY><TR><TH>Symbol</TH><TH>Description</TH><TH>Value</TH><TH>Reference</TH></TR><TR><TD>d<SUB>LuxI</SUB></TD><TD>LuxI degradation rate</TD><TD>0.017 min<SUP>-1</SUP></TD><TD><A href="#bib4" class="forward-ref">[4]</A></TD></TR><TR><TD>d<SUB>LuxR</SUB></TD><TD>LuxR degradation rate</TD><TD>0.023 min<SUP>-1</SUP></TD><TD><A href="#bib4" class="forward-ref">[4]</A></TD></TR><TR><TD>d<SUB>Azu</SUB></TD><TD>Azurin degradation rate</TD><TD>0.1 min<SUP>-1</SUP></TD><TD>estimated</TD></TR></TBODY></TABLE><H2>Parameters allowed to vary because not very well known and which may have a significant effect on our circuit</H2><TABLE><TBODY><TR><TH>Symbol</TH><TH>Description</TH><TH>Typical value (initial value in the parameter search)</TH><TH>Reference</TH><TH>Lower bound</TH><TH>Higher bound</TH></TR><TR><TD>a<SUB>LuxR</SUB></TD><TD>Maximum expression of luxR</TD><TD>5 nM min<SUP>-1</SUP></TD><TD><A href="https://2014.igem.org/Team:ETH_Zurich/modeling/parameters">iGEM ETH 2014</A></TD><TD>1x10<SUP>-2</SUP> nM min<SUP>-1</SUP></TD><TD>1x10<SUP>4</SUP> nM min<SUP>-1</SUP></TD></TR><TR><TD>a<SUB>LuxI</SUB></TD><TD>Maximum expression of luxI</TD><TD>1x10<SUP>3</SUP> nM min<SUP>-1</SUP></TD><TD><A href="#bib5" class="forward-ref">[5]</A></TD><TD>1x10<SUP>-2</SUP> nM min<SUP>-1</SUP></TD><TD>1x10<SUP>4</SUP> nM min<SUP>-1</SUP></TD></TR><TR><TD>K<SUB>Lac</SUB></TD><TD>Half-activation lactate concentration of the hybrid promoter</TD><TD>2x10<SUP>6</SUP> nM</TD><TD><A href="http://parts.igem.org/Part:BBa_K1847008">Characterized lactate sensing part on which our AND-gate is based</A></TD><TD>1x10<SUP>4</SUP> nM</TD><TD>1x10<SUP>8</SUP> nM</TD></TR><TR><TD>k<SUB>LuxI</SUB></TD><TD>Leakiness of the hybrid promoter</TD><TD>0.01</TD><TD><A href="http://parts.igem.org/Part:BBa_K1847008">Characterized lactate sensing part on which our AND-gate is based</A></TD><TD>0.0001</TD><TD>0.1</TD></TR><TR><TD>K<SUB>LuxR</SUB></TD><TD>Half-activation LuxR-AHL concentration of the hybrid promoter</TD><TD>5 nM</TD><TD><A href="https://2013.igem.org/Team:ETH_Zurich/Parameter">iGEM ETH 2013</A></TD><TD>1 nM</TD><TD>100 nM</TD></TR><TR><TD>n<SUB>LuxR</SUB></TD><TD>Hill coefficient of the hybrid promoter regarding LuxR-AHL concentration</TD><TD>1.7</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">iGEM ETH 2015</A></TD><TD>1.1</TD><TD>1.9</TD></TR><TR><TD>n<SUB>Lac</SUB></TD><TD>Hill coefficient of the hybrid promoter regarding lactate concentration</TD><TD>1.7</TD><TD><A href="https://2015.igem.org/Team:ETH_Zurich/Modeling/Parameters">iGEM ETH 2015</A></TD><TD>1.1</TD><TD>1.9</TD></TR><TR><TD>k<SUB>Azu-LuxI</SUB></TD><TD>Relative expression of azurin compared to LuxR</TD><TD>10 times the luxI expression</TD><TD>estimated</TD><TD>10<SUP>-5</SUP></TD><TD>10<SUP>5</SUP></TD></TR></TBODY></TABLE></DETAILS><DETAILS><SUMMARY>How do we quantitatively define the functional space?</SUMMARY><H2>Cost function</H2><P>To be able to distinguish systems satisfying the <A href="https://2017.igem.org/Team:ETH_Zurich/Model/Environment_Sensing/system_specifications">criteria</A> about specificity and azurin production and the ones that do not, we need to use a numerically evaluable condition quantifying how well the criteria are met. Based on this, the script will either accept or discard the parameter set. For this, we will use the following cost function, taking a parameter vector as argument:</P><P><SPAN class="math">\[\text{cost}(p) = \max \left( \frac{10\cdot {[Azu]_{\mathrm{(low\; lac,HIGH\; d_{cell})}}}}{[Azu]_{\mathrm{(HIGH\; lac,HIGH\; d_{cell})}}}, \frac{10\cdot {[Azu]_{\mathrm{(HIGH\; lac,low\; d_{cell})}}}}{[Azu]_{\mathrm{(HIGH\; lac,HIGH\; d_{cell})}}}, \frac{1\times10^{6}}{[Azu]_{\mathrm{(HIGH\; lac,HIGH\; d_{cell})}}}\right)\]</SPAN></P><P>Each argument of the max function represents in the same order the following criteria:</P><OL><LI>Specificity of the sensing for lactate</LI><LI>Specificity of the sensing for bacterial cell density</LI><LI>Achieving a large amount of produced azurin</LI><OL><P>Interpretation of the result of the cost function goes as follows: the smaller the value the better better the criterium is met. If the highest value (e.g. the value for the criterium is met the worst) is below 1, the paramter set is accepted. Over 1, a ratio is not good enough. This monotonicity enables us to rely on optimization algorithms to reach the best combination of parameters available. Also, we can say that every system that has a cost function value below 1 is good enough for us, while &quot;the smaller the better&quot; still applies.
    </P></OL></OL></DETAILS><DETAILS><SUMMARY>Intermediate modeling result: Analysis of the functional parameter space</SUMMARY><P>Using an optimization toolbox developed for biological systems, MEIGO <A href="#bib6" class="forward-ref">[6]</A>, followed by a package exploring parameter spaces, HYPERSPACE <A href="#bib7" class="forward-ref">[7]</A>, we could obtain the following graphs describing, in the high-dimension space of all possible circuits, a subset of systems satisfying our performance criteria:
    </P><P>On this figure are shown the systems suitable for our application. All the axis are logarithmic, except for <EM>n</EM><SUB>Lac</SUB> and <EM>n</EM><SUB>LuxR</SUB>. The yellow points are good systems, the blue ones are even better and surpass the specifications that we demand. From this figure, we can draw the following interpretations (see corresponding sub-graphs referred to on the figure).
    <OL><LI>Only some given combination of expression of LuxI and LuxR are suitable for our needs. This is expected as the tuning of the bacterial cell density at which the quorum sensing is triggered is mainly done with these two proteins</LI><LI>High amounts of azurin are more easily achieved when LuxI maximal expression is high: then the expression of azurin does not need to be that much more compared to luxI to reach the desired level.</LI><LI>The tipping point of the lactate sensing must be either around or above the lactate levels to be distinguished (1 mM in healthy tissues and 5mM in tumors). The first possibility makes sense as the promoter should ideally be unactivated at low lactate level and activated above. However, the combination of this lactate sensing and quorum sensing into the hybrid promoter seems to allow for a second possibility: that the full activation of the promoter happens at much higher concentrations. In both cases, the differential expression at 1 mM and 5 mM plays the role of &quot;increasing the leakiness&quot; of the promoter in regard to luxR so that the quorum sensing is more easily activated in presence of lactate.</LI><LI>The leakiness is a very important parameter to be able to achieve a good performance for our system. The smaller the leakiness, the more probable it is to find a good system.</LI><LI>The Hill coefficient of our hybrid promoter in regard to lactate will allow more or less possibilities of systems: when over 1.5, a population of systems is present (more on the yellow side) that allows for a larger set of <EM>a</EM><SUB>LuxR</SUB>/<EM>a</EM><SUB>LuxI</SUB>combinations (see also <EM>n</EM><SUB>Lac</SUB> vs <EM>a</EM><SUB>LuxR</SUB> and <EM>n</EM><SUB>Lac</SUB> vs <EM>a</EM><SUB>LuxI</SUB> graphs). As we won't be able to tune it, we should prepare for the worst and try to aim for the best systems (the blue ones) on graph 1 to keep a security margin</LI><LI><EM>K</EM><SUB>LuxR</SUB> and <EM>n</EM><SUB>LuxR</SUB> don't have a significant influence on our system, we can stop studying them</LI></OL></P></DETAILS></DIV><H1>Final modeling result: Experimental guidelines used for circuit design</H1><P> From these observations, we can deduce guidelines regarding the parameters on which we can exert an active control, that is to say the expression level of the genes LuxI and LuxR (<EM>a</EM><SUB>LuxR</SUB> and <EM>a</EM><SUB>LuxI</SUB> here) as well as a judicious choice of a previously characterized lactate sensor circuit (comprising lldR and lldP genes) among the <A href="https://2015.igem.org/Team:ETH_Zurich/Part_Collection">iGEM ETH 2015 part collection </A>.
    </P><DIV class="multi-summary" id="experiment_guidelines"><DETAILS><SUMMARY>Target parameters and restrictions</SUMMARY><P>To translate these insights into experimental results in the lab, we need to chose a target in the range of parameters that works for our application. With the help of the previously characterized initial values for <EM>a</EM><SUB>LuxR</SUB> (5 nMmin<SUP>-1</SUP>) and <EM>a</EM><SUB>LuxI</SUB> (1x10<SUP>3</SUP> nMmin<SUP>-1</SUP>), we can hope to tune our system and reach our target in the parameter space via simple RBS tuning which is supported by the <A href="https://salislab.net/software/forward">Salis Lab RBS Calculator</A><A href="#bib8" class="forward-ref">[7]</A>.
    </P><P>As it turned out, the regulatory sequence in front of the luxR gene on the part at our disposal induced already a relatively high expression level. It was hard to get more than 10 times more expression for this gene on the Salis calculator, this is why the range <EM>a</EM><SUB>LuxR</SUB> &gt; 1x10<SUP>2</SUP> nMmin<SUP>-1</SUP> is inaccessible to us (grey area), and that we have to choose LuxI in consequence. We also get to chose <EM>K</EM><SUB>Lac</SUB> among the ones available in the promoter collection of parts ranging from <A href="http://parts.igem.org/Part:BBa_K1847002">BBa_K1847002</A> to <A href="http://parts.igem.org/Part:BBa_K1847009">BBa_K1847009</A>: between 0.3 mM and 2.4mM.
    </P></DETAILS></DIV><P>Taking into account the experimental constraints (forbidden grey area), the targeted parameters (red squares) were chosen on the following plot, with an extensive compatibility for different potential leakiness of our hybrid promoter (red frame):
    </P><DIV><FIGURE><FIGCAPTION>Parameter space of vectors of parameters satisfying our criteria (cost below 1). Yellow points are good enough systems, blue ones are even better and surpass the specifications that we demand. Red features highlight our choice for the operating choice that we will implement in the lab through genetic design guidelines.</FIGCAPTION></FIGURE></DIV><SECTION class="emphasize" id="GuidelineToParts"><P>With <EM>a</EM><SUB>LuxR</SUB> = 1x10<SUP>2</SUP> nMmin<SUP>-1</SUP>, <EM>a</EM><SUB>LuxI</SUB> = 1x10<SUP>4</SUP> nMmin<SUP>-1</SUP> and <EM>K</EM><SUB>Lac</SUB> =  1x10<SUP>6</SUP> nM, we should be at a suitable operating point for our system and still have some security margin in case the genetic design does not yield the exact expression levels that we would expect from it. To achieve these parameters, we gave the following directions for the design of our parts:

<OL><LI>Use a 10 times stronger RBS than on the <A href="https://2014.igem.org/Team:ETH_Zurich/lab/sequences">piG0047 sequence of iGEM ETH 2014 team</A> for the expression of LuxR</LI><LI>Use a 10 times stronger RBS than on the <A href="https://2014.igem.org/Team:ETH_Zurich/lab/sequences">piG0050 sequence of iGEM ETH 2014 team</A> for the expression of LuxI</LI><LI>Use the <A href="http://parts.igem.org/Part:BBa_K1847008">BBa_K1847008 part</A> with J23118-B0034 regulatory sequences, giving <EM>K</EM><SUB>lac</SUB> = 1.8 mM</LI></OL></P><P>These value were the basis for the design of our parts and the <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Tumor_Sensor">subsequent experimentations</A>. 
</P></SECTION><DIV class="multi-summary"><DETAILS><SUMMARY>Sanity check: in silico behavior for the chosen target parameters</SUMMARY>

We can validate on our model that they would work well to distinguish the specific levels dictated by our application:
    <DIV><FIGURE><FIGCAPTION>Expected behavior of the circuit for the chosen parameter set target: intracellular azurin concentration (in nM) depending on both lactate level and bacterial cell density. The white lines correspond to low levels of lactate and bacterial cell density (in healthy tissues) and the black lines represent the high levels (in tumor tissues). The output level of azurin is only significant if both inputs are high, which is the behavior that suits our application.</FIGCAPTION></FIGURE></DIV><P>We can confirm that the obtained parameter set target would lead to a functioning circuit.</P></DETAILS></DIV><SECTION><H1>Going further: a 3D tumor model</H1><P>To assess the validity of this analytical model, and especially validate the assumptions regarding <A href="https://2017.igem.org/Team:ETH_Zurich/Model/Environment_Sensing/model#diff_model">the AHL diffusion model</A>, we developed a <A href="https://2017.igem.org/wiki/index.php?title=Team:ETH_Zurich/Model/In_Vivo">comprehensive 3D model</A> of a colonized tumor on the software COMSOL, on which we implemented the same equations governing our bacterial circuit.</P></SECTION><SECTION class="references"><H1>References</H1><OL><LI id="bib1"><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC219261/?page=4">Kaplan HB, Greenberg EP. Diffusion of autoinducer is involved in regulation of the Vibrio fischeri luminescence system. <SOURCE>Journal of Bacteriology.</SOURCE> 1985;163(3):1210-1214.</A></LI><LI id="bib2"><A href="http://www.pnas.org/content/101/30/10955.full">Jordi Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing <SOURCE>PNAS</SOURCE> 2004 101 (30) 10955-10960</A></LI><LI id="bib3"><A href="https://www.ncbi.nlm.nih.gov/pubmed/20100181">Fekete, A., Kuttler, C., Rothballer, M., Hense, B. A., Fischer, D., Buddrus-Schiemann, K., Lucio, M., Müller, J., Schmitt-Kopplin, P. and Hartmann, A. (2010), Dynamic regulation of N-acyl-homoserine lactone production and degradation in Pseudomonas putida IsoF. <SOURCE>FEMS Microbiology Ecology</SOURCE>, 72: 22–34. doi:10.1111/j.1574-6941.2009.00828.x</A></LI><LI id="bib4"><A href="http://www.sciencedirect.com/science/article/pii/S0303264705001267">A.B. Goryachev, D.J. Toh T.Lee, Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants <SOURCE>Biosystems</SOURCE>, Volume 83, Issues 2–3, February–March 2006, Pages 178-187</A></LI><LI id="bib5"><A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html">A synthetic multicellular system for programmed pattern formation Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss<SOURCE>Nature</SOURCE> 434, 1130-1134 (28 April 2005) | doi:10.1038/nature03461</A></LI><LI id="bib6"><A href="http://gingproc.iim.csic.es/meigo.html">Egea JA, Henriques D, Cokelaer T, Villaverde AF, MacNamara A, Danciu DP, Banga JR and Saez-Rodriguez J. (2014) MEIGO: an open-source software suite based on metaheuristics for global optimization in systems biology and bioinformatics. <SOURCE>BMC Bioinformatics</SOURCE> 15:136.</A></LI><LI id="bib7"><A href="https://gitlab.com/csb.ethz/HYPERSPACE">Zamora-Sillero E, Hafner M, Ibig A, Stelling J, Wagner A (2011) Efficient characterization of high-dimensional parameter spaces for systems biology. <SOURCE>BMC Syst Biol</SOURCE> 5: 142.http://doi.org/10.1186/1752-0509-5-142</A></LI><LI id="bib8"><A href="https://www.nature.com/nbt/journal/v27/n10/full/nbt.1568.html">Salis, Howard M., Ethan A. Mirsky, and Christopher A. Voigt. &quot;Automated design of synthetic ribosome binding sites to control protein expression.&quot;<SOURCE>Nature biotechnology</SOURCE> 27.10 (2009): 946-950. http://dx.doi.org/10.1038/nbt.1568</A></LI></OL></SECTION></SECTION><FOOTER id="footerN"><BUTTON onclick="goto_top()" id="goto-top" title="Go to top">Top</BUTTON><A class="sponsor-a" href="/Team:ETH_Zurich/Sponsors">Sponsors</A></FOOTER></MAIN></DIV></DIV></DIV></DIV></DIV></BODY></HTML>