<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments_Cell_Lysis skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Experiments/Cell Lysis</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="eth-main-contents" class="eth-main-contents"><HEADER id="header"><NAV><UL class="main-menu col-to-row enlarge-line-height custom_cursor"><LI onclick="return true"><SPAN class="menu-entry">Project Idea</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Description">Story of CATE</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Applied_Design">CATE in Action</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Circuit">Circuit</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Design">Design</A></LI></UL><LI onclick="return true"><SPAN class="menu-entry">Engineering</SPAN></LI><LI onclick="return true"><SPAN class="dry">Dry Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Environment_Sensing">Tumor Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Heat_Sensor">Heat Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Vivo">3D Model</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Silico_Final">Final Integration</A></LI><UL class="subsubmenu"><LI onclick="return true"><SPAN class="wet">Wet Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Notebook">Lab Journal</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Protocols">Protocols</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Safety">Safety</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sequences">Sequences</A></LI></UL></NAV></HEADER></DIV><LI onclick="return true"><SPAN class="menu-entry">Achievements</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Results">Results</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Parts">Parts</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Medal_Criteria">Medal Criteria</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/InterLab">InterLab</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI onclick="return true"><SPAN class="menu-entry">Human Practices</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Human_Practices">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Gold_Integrated">Integrated Practices</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Silver">Public Engagement</A></LI><LI onclick="return true"><SPAN class="menu-entry">Our Team</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Team">Members</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sponsors">Sponsors</A></LI><MAIN role="main"><H1 class="headline">Cell Lysis Experiments</H1><P><EM>This is a detailed experiment page dedicated to an individual function. To access other experiments, go to our <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments">Experiments page</A>. To get a quick glimpse at all of our achievements, check out <A href="https://2017.igem.org/Team:ETH_Zurich/Results">Results</A>.</EM></P><SECTION class="emphasize"><H1>Achievements</H1><UL><LI>We rationally designed an RBS library in order to <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Cell_Lysis#phaseII">tune protein E expression strength</A>. We managed to decrease expression levels at uninduced state far enough to successfully co-transform protein E with the thermosensing system.</LI><LI>We show that it is possible for our engineered bacteria to grow at 37 °C when transformed with the heat-inducible cell-lysis system. After inducation at 45 °C, we can show that the <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Cell_Lysis#phaseIII">cells lyse and release their protein-content into the environment</A>.</LI></UL></SECTION><SECTION><H1>Introduction</H1><P>CATE needs to release the previously accumulated <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Anti-Cancer_Toxin">Anti-Cancer Toxin</A> to the extracellular space. We implemented the cell lysis mechanism from bacteriophage Phi X174. It is initiated upon recognition of the heat signal by the <A href="https://2017.igem.org/Team:ETH_Zurich/Circuit/Fd_Heat_Sensor">Heat Sensor</A>. Development and test of the heat sensor is shown on the <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor">Heat Sensor Experiments</A> page. Here we show the induction of cell lysis with the heat sensor and subsequent GFP release to the supernatant.</P><FIGURE class="fig-nonfloat" style="width:700px;">Figure 1. The genetic circuit of our <A href="http://parts.igem.org/Part:BBa_K2500003">TlpA heat sensor</A>. TlpA represses the P<SUB>TlpA</SUB> Promoter. A temperature of 45 °C releases the repression leading to induction of protein E. Protein E molecules interfere with cell wall synthesis and lead to cell lysis. Previously accumulated toxins get released.</FIGURE></SECTION><P>For more details about the lysis mechanism, go to <A href="https://2017.igem.org/Team:ETH_Zurich/Circuit/Fe_Cell_Lysis">Cell Lysis</A>.</P></MAIN><SECTION id="phaseI"><H1>Phase I: Initial System Design</H1><P>The cell lysis mechanism of phage Phi X174 needs a single gene called protein E to get activated. In the initial design we placed protein E under an inducible promotor called P<SUB>Lux</SUB>. A ribosome binding site with large translation initiation rate was calculated with the <A href="https://salislab.net/software/">Salis Lab RBS calculator</A> and placed in front of the protein E coding sequence.
   </P><FIGURE class="fig-nonfloat" style="width:300px;"><FIGCAPTION>Figure 2. Cell Lysis test plamids. AHL inducible protein E is placed on a pSEVA291 vector. PF and SF are abbreviations for BioBrick Prefix and BioBrick Suffix restriction sites. RS1-RS4 are restriction sites that we introduced for subsequent cloning. 
        </FIGCAPTION></FIGURE><P>Initially, no transformants could be obtained. This was probably due to a high leakiness of the P<SUB>TlpA</SUB>, which lead to enough expression of protein E to lyse all successful transformants.</P><P>CONCLUSION: We knew now that the protein E must be regulated by a very tight promotor. We used this knowledge to engineer the Heat Sensor to a low base level expression of the regulated gene. Read <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor#phaseII">here</A> how we engineered leakiness of the P<SUB>TlpA</SUB>.</P></SECTION><SECTION id="phaseII"><H1>Phase II - Optimization of co-transformation efficiency</H1><P>Initially, it was not possible to transform a protein E, regulated by the <H href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor">Heat Sensor into <SPAN class="bacterium">E. coli</SPAN>. Therefore we reduced the translation initiation rate of the protein E RBS.
  </H></P><P><B>Protein E RBS library creation</B></P><P>
We constructed a ribosome binding site library to find variants expressing little enough protein E to avoid lysis from leaky expression, but still enough to lyse the cells when the expression of protein E is induced. The RedLibs algorithm <A href="#bib1" class="forward-ref">[1]</A> was used.
</P><DETAILS><P>We used the RedLibs algorithm to design a RBS library that would allow us vary protein expression and screen for improved variants. Therefore, we initially calculated the TIR values for a fully degenerate RBS library with 8 times N (fully degenerate base) at the positions -13 to -5 upstream of the ATG start codon using the Salis RBS calculator. <A href="#bib2" class="forward-ref">[2]</A><A href="#bib3" class="forward-ref">[3]</A> This library with 65’536 variants would be too large to efficiently screen and contain too many unfunctional RBS sequences. Therefore, we used the RedLibs algorithm to reduce the library to smaller size and distribute it’s values uniformly[Rationally reduced libraries for combinatorial pathway optimization minimizing experimental effort https://www.nature.com/articles/ncomms11163]. The algorithm then provided us with a partially degenerate sequence, that could be implemented by a single cloning step and codes for an as uniform as possible distribution of TIR values.</P><FIGURE class="fig-nonfloat" style="width:800px;"><FIGCAPTION>Distribution of TIR values calculated with RedLibs. Most sequences have a low TIR and should therefore enable transformants to grow despite some leakiness of the promotor.</FIGCAPTION></FIGURE></DETAILS><P>
The degenerate primers were ordered and the library was created with a PCR amplification and subsequent Gibson assembly and transformation. The plasmid was designed in a way that transformants with correct insert produce GFP constitutively and the protein E is controlled by the heat sensor.

</P><FIGURE class="fig-nonfloat" style="width:500px;"><FIGCAPTION>Figure 5. Heat inducible cell lysis test device. The plasmids contained RBS libraries of protein E (above) and TlpA (below).</FIGCAPTION></FIGURE><P>

The double transformation with the plasmids (Figure 5) grown at 37 °C, yielded green fluorescent colonies. This shows successfull inhibition of protein E at 37 °C (colonies are not lysed) and successful insertion of the protein E gene (+ const. promotor) between P<SUB>Lux</SUB> and gfp (constitutive green at 37 °C)



 <FIGURE class="fig-nonfloat" style="width:500px;"><FIGCAPTION>Figure 6. Double transformation of the heat inducible cell lysis test device (protein E RBS library and TlpA RBS library).</FIGCAPTION></FIGURE></P><P><B>Protein E RBS library variant selection</B></P><P>
All fluorescent colonies were picked and inoculated to a 96 well plate and grown overnight (16 h) to stationary phase at 37°C. Continuing with the 96 well format, the samples were inoculated into a fresh 96 well culture plate (OD 0.1) and grown to OD<SUB>600</SUB> 0.4. At this point the cultures were split to fresh plates (flat transparent bottom) and incubated at 37 °C and 45 °C for 3 h. The OD<SUB>600</SUB> was measured from the beginning of the OD<SUB>600</SUB> 0.1 culture to track the growth curve during induction. Four variants were selected for the subsequent experiment. They were restreaked to obtain multiple single clones for triplicate measurements.
  </P></SECTION><SECTION id="phaseIII"><H1>Phase III - Heat-Induction of GFP Release by TlpA-controlled Protein E Expression</H1><P>We tested the function of heat-induced cell lysis by inducing <SPAN class="bacterium">E. coli</SPAN> Top10 for 3 h at 45 °C in culture tubes in a shaking incubator. Because the fluorescence of each sample was different due to intrinsic noise, only the ratio of total fluorescence and fluorescence in the supernatant can give a cue about the lysis efficiency. Therefore we measured the total fluorescence and the supernatant fluorescence every hour for the induction period (Figure 7). 
</P><FIGURE class="fig-nonfloat" style="width:1000px;"><FIGCAPTION>Figure 7. Protein E RBS library variants fluorescence ratio (supernatant/total). Four library variants were selected and induced with a heat shock of 45 °C (lower row). The negative control consists of a constitutively expressed GFP without protein E.</FIGCAPTION></FIGURE><P>We performed the experiment according to the <A href="https://static.igem.org/mediawiki/2017/e/e9/T-ETH_Zurich--Fe-protocol.pdf" download="">protocol</A> with three protein E and TlpA RBS library variants and one protein E RBS library and improved TlpA RBS variant. (Read <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor#phaseII">here</A> how we produced it). The protein E RBS variants were sequenced and compared to the predicted translation initiation rates:

<FIGURE class="fig-nonfloat" style="width:600px;"><FIGCAPTION>Table 1. Protein E RBS library variants -13 to -5 upstream sequences with their corresponding calculated translation initiation rates.</FIGCAPTION></FIGURE></P><P>
We showed that the Heat Sensor effectively induces protein E expression with 3 h of induction at 45 °C. The variant C (protein E RBS: CGGGGGGG, Table 1) has a tight repression because it was cotransformed with the engineered TlpA RBS (Figure 7, D).

CONCLUSION: We showed that heat induced cell lysis happens and about 70% of the total GFP is found in the supernatant after 3 h induction at 45 °C. This value underestimates the effective amount of released protein, because cell lysis might continue after the measurement period of 3 h and release even more protein. We also showed effective inhibition of cell lysis at 37 °C if the protein E is regulated by a highly expressed TlpA such as our engineered TlpA RBS variant (Figure 7, D).
</P></SECTION><SECTION class="references"><H1>References</H1><OL><LI id="bib1">Jeschek, Markus, Daniel Gerngross, and Sven Panke. &quot;Rationally reduced libraries for combinatorial pathway optimization minimizing experimental effort.&quot; <CITE>Nature communications</CITE> 7 (2016). <A href="https://doi.org/10.1038/ncomms11163">doi: 10.1038/ncomms11163</A></LI><LI id="bib2">Espah Borujeni, Amin, Anirudh S. Channarasappa, and Howard M. Salis. &quot;Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites.&quot; <CITE>Nucleic acids research</CITE> 42.4 (2013): 2646-2659. <A href="https://doi.org/10.1093/nar/gkt1139">doi: 10.1093/nar/gkt1139</A></LI><LI id="bib3">Salis, Howard M., Ethan A. Mirsky, and Christopher A. Voigt. &quot;Automated design of synthetic ribosome binding sites to control protein expression.&quot; <CITE>Nature biotechnology</CITE> 27.10 (2009): 946-950. <A href="https://doi.org/10.1038/nbt.1568">doi: 10.1038/nbt.1568</A></LI></OL></SECTION><FOOTER id="footerN"><BUTTON onclick="goto_top()" id="goto-top" title="Go to top">Top</BUTTON><A class="sponsor-a" href="/Team:ETH_Zurich/Sponsors">Sponsors</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>