<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Experiments_Heat_Sensor skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Experiments/Heat Sensor</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="eth-main-contents" class="eth-main-contents"><HEADER id="header"><NAV><UL class="main-menu col-to-row enlarge-line-height custom_cursor"><LI onclick="return true"><SPAN class="menu-entry">Project Idea</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Description">Story of CATE</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Applied_Design">CATE in Action</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Circuit">Circuit</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Design">Design</A></LI></UL><LI onclick="return true"><SPAN class="menu-entry">Engineering</SPAN></LI><LI onclick="return true"><SPAN class="dry">Dry Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Environment_Sensing">Tumor Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Heat_Sensor">Heat Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Vivo">3D Model</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Silico_Final">Final Integration</A></LI><UL class="subsubmenu"><LI onclick="return true"><SPAN class="wet">Wet Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Notebook">Lab Journal</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Protocols">Protocols</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Safety">Safety</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sequences">Sequences</A></LI></UL></NAV></HEADER></DIV><LI onclick="return true"><SPAN class="menu-entry">Achievements</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Results">Results</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Parts">Parts</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Medal_Criteria">Medal Criteria</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/InterLab">InterLab</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI onclick="return true"><SPAN class="menu-entry">Human Practices</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Human_Practices">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Gold_Integrated">Integrated Practices</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Silver">Public Engagement</A></LI><LI onclick="return true"><SPAN class="menu-entry">Our Team</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Team">Members</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sponsors">Sponsors</A></LI><MAIN role="main"><H1 class="headline">Heat Sensor Experiments</H1><P><EM>This is a detailed experiment page dedicated to an individual function. To access other experiments, go to our <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments">Experiments page</A>. To get a quick glimpse at all of our achievements, check out <A href="https://2017.igem.org/Team:ETH_Zurich/Results">Results</A>.</EM></P><SECTION class="emphasize"><H1>Achievements</H1><UL><LI>We <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor#phaseI">characterized</A> our initial design and found that the highest fold-changes in activation can be induced at 45 °C.</LI><LI>We rationally designed an RBS library to tune expression levels of tlpA. By screening for the best variant we were able to dramatically <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor#phaseII">improve our initial design</A>.</LI><LI>Finally, we <A href="https://2017.igem.org/Team:ETH_Zurich/Experiments/Heat_Sensor#phaseII">showed</A> that our thermosensing system is tight enough to put the lysis-inducing protein E under it's control without harming the cells.</LI></UL></SECTION><SECTION><H1>Introduction</H1><P>We incorporated a module into our system that allows our engineered bacteria to sense the external toxin-release signal of the doctor. The signal is produced with focused ultrasound which increases the temperature in the tumor region to 45 °C. To detect this signal, we needed to fit a naturally occurring heat sensor from <SPAN class="bacterium">Salmonella enterica serovar Typhimurium</SPAN> into our novel genetic circuit. <A href="#bib1" class="forward-ref">[1]</A><A href="#bib2" class="forward-ref">[2]</A> Once the heat sensor detects the temperature increase, it activates the next step (<A href="2017.igem.org/Team:ETH_Zurich/Circuit/Fe_Cell_Lysis">Cell Lysis</A>), by promoting expression of protein E.</P><FIGURE class="fig-nonfloat" style="width:800px;"><FIGCAPTION>Figure 1. The genetic circuit of our <A href="http://parts.igem.org/Part:BBa_K2500003">TlpA heat sensor</A>. TlpA represses the P<SUB>TlpA</SUB> Promoter. A temperature of 45 °C releases the repression leading to induction of protein E.</FIGCAPTION></FIGURE><P>For more details about the mechanism, go to <A href="https://2017.igem.org/Team:ETH_Zurich/Circuit/Fd_Heat_Sensor">Heat Sensor</A>.</P></SECTION><SECTION id="phaseI"><H1>Phase I: Initial System Design</H1><P>The TlpA heat sensor consists of two parts: the TlpA regulator protein and the P<SUB>TlpA</SUB> promoter. We implemented the digital sequences from a plasmid used by Piraner et al. <A href="#bib2" class="forward-ref">[2]</A>and introduced 8 silent mutations to remove forbidden restriction sites inside the coding sequence. We designed test-plasmids to ensure proper function of the system. The initial design of a test system constists of two plasmids, one containing the repressor and the other the promoter and a reporter protein. The first plasmid was designed to have an <A href="http://parts.igem.org/Part:BBa_J23100">Anderson Promoter</A> with relative strength of 0.71 followed by a RBS (designed with Salis Lab RBS calculator <A href="#bib3" class="forward-ref">[3]</A>) with a calculated translation initiation rate of 5000 and the TlpA coding sequence. 

    <FIGURE class="fig-nonfloat" style="width:500px;"><FIGCAPTION>Figure 2. The heat sensor test plamids. The TlpA coding sequence is placed on <A href="https://2017.igem.org/Team:ETH_Zurich/Sequences#heatsensor">piG17-2-002</A> (pSEVA361) and the heat inducible gfp is placed on <A href="https://2017.igem.org/Team:ETH_Zurich/Sequences#heatsensor">piG17-1-005</A> (pSEVA291). PF and SF are abbreviations for BioBrick Prefix and BioBrick Suffix restriction sites. RS1-RS4 are restriction sites that we introduced for subsequent cloning steps. 
        </FIGCAPTION></FIGURE></P><P>The optimal amount of the TlpA repressor protein in the cytoplasm was not known to us from the beginning, that's why we chose a medium amount of TlpA expression. The physical DNA sequences were ordered as gBlocks from IDT and inserted to our test plasmids pSEVA361 and pSEVA291 via Ligase Cycling Reactions.</P><H2>Key questions to answer in first experiments</H2><UL><LI>Does the heat sensor work in our lab with our experimental setup?</LI><LI>Does the heat sensor work despite the changes we made to the coding sequence, the synthetic ribosome binding site and the BioBrick promotor?</LI><LI>Which temperature is needed to activate the heat sensor?</LI><LI>How long does the heat sensor have to be induced for decent expression of the regulated gene?</LI></UL><P>A sequence of experiments was performed to find optimal induction times and experimental setup. <SPAN class="bacterium">E. coli</SPAN> Top10 chemical competent cells or Nissle electrocompetent cells were used. Single colonies of double transformants were inoculated to 12 mL round bottom culture tubes in 5 mL LB and grown for 16 h at 37 °C shaking 230 rpm. After growth to stationary phase, they were diluted to OD 0.1 and grown to exponential phase (OD 0.4). At this point the induction procedure was initiated in different formats, for different times and temperatures.</P><P> Our findings were:</P><UL><LI>Induction times of 1 to 15 minutes don't induce the reporter gene measurably, even though temperatures above 42 °C lead to slightly higher fluorescence after 15 min induction</LI><LI>Strong induction takes place in a timescale of 1-5 hours. (more would not be feasible for <A href="https://2017.igem.org/Team:ETH_Zurich/Applied_Design">our application</A>)</LI><LI>A suitable experiment procedure was found</LI></UL><FIGURE class="fig-nonfloat" style="width:500px;"><FIGCAPTION>Figure 3. Fluorescence readouts of the TlpA Heat sensor after induction times from 0.5 to 5 h. Biological triplicates were measured and the standard deviation is depicted as error bars.</FIGCAPTION></FIGURE><P>Longer induction times lead to higher fluorescence values (Figure 7). Even though induction is significant, the fold change is very low. Induction was performed in 60 uL cell culture in 100 uL PCR tubes in thermocyclers for the time and temperature indicated and after induction stored at 30 °C until start of the readout (5 h after start of induction). The experimental setup was changed from induction in thermocycler to induction in shaking incubators. Shake flasks were used to test whether low oxygen availability could have been an issue for culture growth and induction potency.</P><FIGURE class="fig-nonfloat" style="width:800px;"><FIGCAPTION>Figure 4. Fluorescence of the TlpA Heat Sensor test device after 3 h induction at 37 °C and 45 °C in shake flasks in a shaking incubator. Biological triplicates were measured and the standard deviation is depicted as error bars. The negative control lacked the gene for gfp (GFP-, TlpA+) and the positive control lacked the TlpA regulator protein (GFP+, TlpA-).</FIGCAPTION></FIGURE><P>The fluorescence of the supernatant was also measured to find out if we can use supernatant fluorescence in future cell lysis measurements. These experiments (Figure 3 and 4) showed that induction is possible, but the fold change of ~6 is very low. This is either due to a low maximum induction of the promotor compared to a constitutive promotor of gfp, or to a high leaky expression when not induced. We hypothesized that the RBS of TlpA was not initiating translation enough, and low amounts of the TlpA reporessor protein don't inactivate base level expression enough. 
Since we intend to heat induce protein E mediated cell lysis, a promoter system with high basal expression is useless. Initial trials to clone protein E under P<SUB>TlpA</SUB> failed, probably due to insufficient inhibition of protein E expression.</P></SECTION><SECTION id="phaseII"><H1>Phase II: Optimization of the Heat Sensor</H1><P><B>Reducing the heat sensor's leakiness</B></P><P>
It is very important for proper function of CATE to have a very tightly controlled activation of cell lysis. Only if CATE releases the anti-cancer compound upon the external heat signal, an improvement of the control of toxin delivery is possible. Additionally, it is very difficult to introduce the heat inducible protein E expression construct under a leaky promoter, as the basal expression already induces cell death in the successful transformants.
  </P><P><B>TlpA RBS library creation</B></P><P>
A ribosome binding site library was then created to find variants expressing more TlpA. The RedLibs algorithm <A href="#bib4" class="forward-ref">[4]</A> was used and set to calculate degenerate sequences that produce 12 variants. The variants should all have a rather high expression rate to increase the cytoplasmic amount of TlpA dimers able to repress the promotor. Degenerate primers were used to create the library with a simple PCR and subsequent gel cleanup and transformation.
  </P><FIGURE class="fig-nonfloat" style="width:500px;"><FIGCAPTION>Figure 5. TlpA RBS Library. A RBS library of the TlpA RBS was calculated with the RedLibs algorithm. It contained 12 different RBS sequences that should exhibit uniformly distributed translation initiation rates. Green colonies have lost the repressor activity of TlpA probably due to very weak TlpA expression. Non-fluorescent colonies might have a strongly repressed gfp, because of a higher amount of TlpA.</FIGCAPTION></FIGURE><P><B>TlpA RBS library variant selection</B></P><P>
Single colonies were picked and inoculated to a 96 well plate and grown to a stationary phase. Continuing with the 96 well format, the samples were inoculated into a fresh 96 well culture plate and grown to OD<SUB>600</SUB> 0.4. At this point the cultures were split and induced at 37 °C and 45 °C for 3 h. Samples were diluted in PBS and the fluorescence measured in a plate reader. The eight variants with the highest fold-change were selected for further experiments.
  </P><P>
The best eight TlpA RBS variants were tested for fluorescence induction according to the protocol.
  </P><FIGURE class="fig-nonfloat" style="width:800px;"><FIGCAPTION>Figure 6. Distribution of GFP expression of the library variants (A) and their fold change (B).</FIGCAPTION></FIGURE><P>Our results (Figure 6) suggest that different variants of the original RBS were obtained in the library creation.  The fold changes vary from 20 to 1200. Most of the new variants yield higher fold changes than the parent variant. The high fold change of H1 is caused by a low leakiness, not by high expression.

<FIGURE class="fig-nonfloat" style="width:500px;"><FIGCAPTION>Figure 7. The 96 well plate with the four technical replicates of the induction of gfp with the thermosensitive TlpA repressor. On top: 4 wells per column induced for 3 h at 37 °C, bottom: 4 wells per column induced at 45 °C. The positive control did not grow.</FIGCAPTION></FIGURE></P><P>The best variants were sequenced and single colonies restreaked for subsequent triplicate measurements. The sequencing results were compared to the predicted translation initiation rates from the RedLibs algorithm:
   </P><P> The variants A9 and D9 share the same sequence and have the highest predicted translation initiation rate (T.I.R.). Interestingly the T.I.R. of H1 is an order of magnitude lower than the others, but still lead to a high fold change.
 </P><SECTION id="phaseIII"><H1>Phase III Demonstration of the Heat Sensor Function</H1><P><B>Triplicate measurements of the best 3 variants</B></P><P>A final experiment was performed according to the <A href="https://static.igem.org/mediawiki/2017/9/96/T--ETH_Zurich--protocolfd1.pdf">protocol</A> with the TlpA RBS library variants H1, A9, C12 and D9 in biological triplicates (Figure 8).
    </P><FIGURE class="fig-nonfloat" style="width:800px;"><FIGCAPTION>Figure 8. TlpA-regulated GFP expression. <B>A</B> Fluorescence of the variants C12 (C), A9 (A), H1 (H) and the parent variant. <B>B</B> Fold changes of the different variants of the TlpA RBS. The samples were non-induced (37 °C) and induced (45 °C) in biological triplicates.</FIGCAPTION></FIGURE><P>
The thermoswitch was now tight enough to repress the toxic protein E to enable transformant colonies to grow. In the next step we transformed it together with a protein E RBS library containing plasmid, with the aim to find protein E RBS library variants with enough reduced translation initiation rate to survive.
</P><P>We could improve the leakiness of the TlpA promotor by simply enhancing the translation initiation rate of the repressor protein TlpA. High GFP expression fold changes of up to 200 were obtained.</P></SECTION></SECTION><SECTION class="references"><H1>References</H1><OL><LI id="bib1">Hurme, R., Berndt, K.D., Namork, E. &amp; Rhen, M. &quot;DNA binding exerted by a bacterial gene regulator with an extensive coiled-coil domain.&quot; <CITE>J. Biol. Chem.</CITE> 271 (1996): 12626–12631. <A href="https://doi.org/10.1074/jbc.271.21.12626">doi: 10.1074/jbc.271.21.12626</A></LI><LI id="bib2">Piraner, Dan I., et al. &quot;Tunable thermal bioswitches for in vivo control of microbial therapeutics.&quot;<CITE>Nature chemical biology</CITE> 13.1 (2017): 75-80. <A href="https://doi.org/10.1038/nchembio.2233">doi: 10.1038/nchembio.2233</A></LI><LI id="bib3">Espah Borujeni, Amin, Anirudh S. Channarasappa, and Howard M. Salis. &quot;Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites.&quot; <CITE>Nucleic acids research</CITE> 42.4 (2013): 2646-2659. <A href="https://doi.org/10.1093/nar/gkt1139">doi: 10.1093/nar/gkt1139</A></LI><LI id="bib4">Jeschek, Markus, Daniel Gerngross, and Sven Panke. &quot;Rationally reduced libraries for combinatorial pathway optimization minimizing experimental effort.&quot; <CITE>Nature communications</CITE> 7 (2016). <A href="https://doi.org/10.1038/ncomms11163">doi: 10.1038/ncomms11163</A></LI></OL></SECTION></MAIN><FOOTER id="footerN"><BUTTON onclick="goto_top()" id="goto-top" title="Go to top">Top</BUTTON><A class="sponsor-a" href="/Team:ETH_Zurich/Sponsors">Sponsors</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>