<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_ETH_Zurich_Model_Environment_Sensing_model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:ETH Zurich/Model/Environment Sensing/model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="eth-main-contents" class="eth-main-contents"><HEADER id="header"><NAV><UL class="main-menu col-to-row enlarge-line-height custom_cursor"><LI onclick="return true"><SPAN class="menu-entry">Project Idea</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Description">Story of CATE</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Applied_Design">CATE in Action</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Circuit">Circuit</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Design">Design</A></LI></UL><LI onclick="return true"><SPAN class="menu-entry">Engineering</SPAN></LI><LI onclick="return true"><SPAN class="dry">Dry Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Environment_Sensing">Tumor Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/Heat_Sensor">Heat Sensor</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Vivo">3D Model</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Model/In_Silico_Final">Final Integration</A></LI><UL class="subsubmenu"><LI onclick="return true"><SPAN class="wet">Wet Lab</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Experiments">Experiments</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Notebook">Lab Journal</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Protocols">Protocols</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Safety">Safety</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sequences">Sequences</A></LI></UL></NAV></HEADER></DIV><LI onclick="return true"><SPAN class="menu-entry">Achievements</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Results">Results</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Parts">Parts</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Medal_Criteria">Medal Criteria</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/InterLab">InterLab</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Collaborations">Collaborations</A></LI><LI onclick="return true"><SPAN class="menu-entry">Human Practices</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Human_Practices">Overview</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Gold_Integrated">Integrated Practices</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/HP/Silver">Public Engagement</A></LI><LI onclick="return true"><SPAN class="menu-entry">Our Team</SPAN></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Attributions">Attributions</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Team">Members</A></LI><LI onclick="return true"><A href="/Team:ETH_Zurich/Sponsors">Sponsors</A></LI><MAIN role="main"><H1 class="headline">Full description of the model of the AND-gate</H1><SECTION class="abstract"><H1 class="goal">Building of our analytical model</H1><P>To be able to understand the behavior of our model, and later how to tune it <A href="https://2017.igem.org/wiki/index.php?title=Team:ETH_Zurich/Model/Environment_Sensing/parameter_space">via genetic engineering</A>, we first need to establish the equations that govern its responses. Here is a detailed description of our model.
</P><P>The tumor sensing circuit is composed of several proteins that interact with small molecules (AHL and lactate) and DNA (at the transcription factors binding sites). To establish a model describing the behavior of our circuit, we first had to understand the way these interactions are happening inside of the cell. Building on the description of the <A href="https://2017.igem.org/Team:ETH_Zurich/Circuit/Fa_Tumor_Sensor">Tumor Sensor circuit</A>, here is a more detailed overview:</P><H1>Simplification of the lactate sensing</H1><P>Let us first focus on the lactate sensing part of the circuit. In the cell, two proteins are produced:</P><UL><LI>LldP: a transmembrane protein enabling the transport of extracellular lactate into E. coli.</LI><LI>LldR: a transcription factor, repressing the activity of the hybrid promoter when not bound to lactate. Lldr releases repression once it binds to lactate.</LI></UL><P>To model precisely the regulation of the hybrid promoter by lactate, it would be necessary to take into account all the following points:</P><UL><LI>How the intracellular lactate concentration behaves in regard to the expression level of LldP and the extracellular lactate concentration</LI><LI>What is the binding constant between LldR and the lactate</LI><LI>What is the binding dynamics of Lldr to the operon, and how it affects the transcription rate downstream</LI></UL><P>In an effort to simplify our model to reduce it to the most meaningful parameters, and because it has already extensively been studied and characterized by previous iGEM teams, we have chosen not to take into account the complexity of the lactate sensing pathway and rather use a phenomenological model to describe its influence. We rely on the characterization of the lactate sensor using several expression regulation sequences <A href="https://2015.igem.org/Team:ETH_Zurich/Results#Characterization_of_synthetic_promoter_library">done by the ETH 2015 iGEM team</A>. We consider therefore that lactate sensing follows a Hill function as following:</P><P><SPAN class="math">\[
        P_{\text{Lac}} \simeq \frac{\left(\frac{[\text{Lac}]}{K_{\text{Lac}}} \right)^{n_{\text{Lac}}}}{1 + \left(\frac{[\text{Lac}]}{K_{\text{Lac}}} \right)^{n_{\text{Lac}}}}\]</SPAN></P><P>As a result, the schematics of the circuit can be simplified this way:</P></SECTION><SECTION><H1>Quorum Sensing sensor modelization</H1><P>The sensing of the bacterial cell density is done via a quorum sensing circuit. The principles behind quorum sensing is that, via the expression of the enzyme LuxI, each bacteria produces a basal amount of a small molecule (here N-acyl homoserine lactone, AHL) that diffuses in the environment and into neighboring cells. When AHL is present in sufficient quantity, it binds to the intracellular LuxR and induces the production of more LuxI, which in turn results in the production of more AHL. This positive feedback loop results in the activation of the operon containing the LuxI gene when the cell density reaches a critical threshold.

    </P><H2 id="concerning-luxr">Concerning LuxR</H2><P>LuxR is under a constitutive promoter of strength <SPAN class="math">\( a_{\text{luxR}} \)</SPAN> and its degradation rate is <SPAN class="math">\( d_{\text{luxR}} \)</SPAN>. <SPAN style="font-variant: small-caps;">AHL</SPAN> binds and stabilizes LuxR; LuxR-<SPAN style="font-variant: small-caps;">AHL</SPAN> molecules can only act as transcription factors when they form a tetramer (2*<SPAN style="font-variant: small-caps;">AHL</SPAN>+2*LuxR). Since we are modeling the steady state, the following simplifications apply:</P><UL><LI><P>We consider that the total amount of LuxR present in the cell is constant, and only depends on its constitutive expression and degradation rate.</P></LI><LI><P>We consider the global binding equilibrium between LuxR and <SPAN style="font-variant: small-caps;">AHL</SPAN> without taking into account the intermediary dimers.</P></LI></UL><P>We can therefore write the following equations:</P><P><SPAN class="math">\[\begin{aligned}
        [\text{LuxR}]_0 &amp;= \frac{a_{\text{LuxR}}}{d_{\text{LuxR}}} &amp; \text{steady state concentration} \\
        [\text{LuxR-AHL}] &amp;= K_{LuxRAHL} [\text{LuxR}]^2 [\text{AHL}]^2 &amp; \text{rapid binding equilibrium} \\
        [\text{LuxR}] &amp;= [\text{LuxR}]_0 - 2 [\text{LuxR-AHL}] &amp; \text{mass conservation}\end{aligned}\]</SPAN></P><H2>Hybrid Lux-Lac promoter</H2><P>The expression of the main operon containing the LuxI, Bfr and Azurin genes is regulated by a hybrid promoter activated by the quorum sensing and repressed by the lactate sensing (the repression being released in presence of lactate). This hybrid promoter should behave as a AND-gate: mathematically, this corresponds to multiplying the Hill functions describing their behavior.</P><P>Along with the lactate concentration, the intracellular levels of LuxR-<SPAN style="font-variant: small-caps;">AHL</SPAN> complexes affect LuxI expression. With <SPAN class="math">\( a_{\text{LuxI}} \)</SPAN> being the maximal production rate of LuxI, <SPAN class="math">\( d_{\text{LuxI}} \)</SPAN> the degradation rate, <SPAN class="math">\( k_{\text{LuxI}} \)</SPAN> the leakiness of the promoter and <SPAN class="math">\( P_{\text{Lux-Lac}} \)</SPAN> the combined effect of the <SPAN class="math">\( P_{\text{Lux}} \)</SPAN> and <SPAN class="math">\( P_{\text{Lac}} \)</SPAN> regulating sequences behavior, the ODE governing the production of <SPAN class="math">\( [\text{LuxI}] \)</SPAN> can be written as following:</P><P><SPAN class="math">\[\frac{\mathrm{d} [\text{luxI}]}{\mathrm{d} t} = a_{\text{LuxI}} (k_{\text{LuxI}} + (1 - k_{\text{LuxI}}) P_{\text{Lux-Lac}}) - d_{\text{LuxI}} [\text{luxI}]\]</SPAN></P><P>where</P><P><SPAN class="math">\[\begin{aligned}
        P_{\text{Lux-Lac}}
        &amp;= P_{\text{Lux}} \, P_{\text{Lac}} \\
        \\
        P_{\text{Lux}}
        &amp;= \frac{\left(\frac{[\text{LuxR-AHL}]}{K_{\text{LuxR}}} \right)^{n_{\text{LuxR}}}}{1 + \left(\frac{[\text{LuxR-AHL}]}{K_{\text{LuxR}}} \right)^{n_{\text{LuxR}}}}\end{aligned}\]</SPAN></P><P>Solving the above at steady state, we get:</P><P><SPAN class="math">\[\begin{aligned}
        \frac{\mathrm{d} [\text{luxI}]}{\mathrm{d} t} &amp;= 0 \\
        [\text{luxI}] &amp;= \frac{a_{\text{luxI}}}{d_{\text{luxI}}} (k_{\text{luxI}} + (1 - k_{\text{luxI}}) P_{\text{Lux-Lac}})\end{aligned}\]</SPAN></P><H2>Production of AHL</H2><P>AHL is produced intracellularly by LuxI and diffuses then freely through the membrane <A href="#bib1" class="forward-ref">[1]</A>. Modeling the production of AHL is quite straightforward: it is proportional to the amount of LuxI present intracellularly. To describe the production per unit of volume though, we have to take into account the bacterial cell density present locally and take it as a dilution coefficient (for instance, if the cells occupy locally half of the volume, then the intracellularly produced AHL would be instantly diluted two times as it diffuses into the surrounding environment).</P><P><SPAN class="math">\[\begin{aligned}
        P_{\text{AHL}}
        &amp;= d_{\text{cell}} a_{\text{AHL}} [\text{luxI}] \end{aligned}\]</SPAN></P></SECTION><SECTION><H1>The missing link between AHL production and its concentration: DIFFUSION MODEL</H1><P>To be able to close mathematically the feedback loop, we still miss an equation: we need to know how the intracellular production of AHL translates into the AHL concentration in the environment. For that, we have to consider the diffusion of AHL around the colonized area of the tumor. By solving the equation governing AHL transport, we can, under certain assumptions which will be detailed below, get the relationship between AHL production and its local concentration around bacterial cells.</P><H1>Assumptions</H1><H2 id="assumption-negligible-ahl-degradation-in-the-layer">Assumption: negligible AHL degradation in the layer</H2><P>The colony produces <SPAN style="font-variant: small-caps;">AHL</SPAN> which diffuses out of the layer. Because of the symmetry of the problem, we only consider diffusion in the radial direction, being characterized by the diffusion constant <SPAN class="math">\( D = 4.9e-6 \text{cm}^2 \text{s}^{-1} \)</SPAN> (diffusion constant of <SPAN style="font-variant: small-caps;">AHL</SPAN> in water) <A href="#bib4" class="forward-ref">[4]</A>. We also take into account extracellular degradation of <SPAN style="font-variant: small-caps;">AHL</SPAN>, described by the degradation constant <SPAN class="math">\( k_{\text{deg}} = 5e-4 \text{min}^{-1} \)</SPAN><A href="#bib4" class="forward-ref">[4]</A>. Using the relationship between the mean square distance and time in a Brownian movement, we can estimate the average time needed for <SPAN style="font-variant: small-caps;">AHL</SPAN> to diffuse out of the layer:</P><P><SPAN class="math">\[\Delta t = \frac{w^2}{D} = \frac{0.05^2}{4.9 \cdot 10^{-6}} \simeq 510 \text{s}\]</SPAN></P><P>During this time <SPAN class="math">\( \Delta t \)</SPAN>, we can estimate the magnitude of the degradation of <SPAN style="font-variant: small-caps;">AHL</SPAN>. The proportion of AHL that gets degraded before diffusing out of the layer can be estimated as:</P><P><SPAN class="math">\[\begin{aligned}
        \Delta t \, k_{\text{deg}} &amp;= \frac{510}{60}\cdot 5 \cdot 10^{-14} \simeq 0.4 \%\end{aligned}\]</SPAN></P><P>Therefore, we can consider for further work that the degradation of AHL happening in the layer is negligible.</P><H2 id="assumption-only-care-about-local-ahl">Assumption: AHL doesn't diffuse far from where it is produced</H2><P>To assess whether we would have to consider in a given point of the tumor the AHL coming from every part of the tumor or only the closest area, we have to estimate how far <SPAN style="font-variant: small-caps;">AHL</SPAN> diffuses before being degraded. Considering the half-life of AHL and the mean distance covered in this given time by diffusion:</P><P><SPAN class="math">\[\begin{aligned}
        t_{1/2} &amp;= \frac{\ln(2)}{k_{\text{deg}}} \simeq 1400 \text{min} \\
        \\
        d &amp;= \sqrt{D t_{1/2}} \simeq 6 \text{mm}\end{aligned}\]</SPAN></P><P>Since <SPAN class="math">\( d &lt; r_1 = 10 \text{mm} \)</SPAN>, <SPAN style="font-variant: small-caps;">AHL</SPAN> will be substantially degraded before it reaches a colonized area far from where it is produced. We will thus assume that we can restrict the study of the diffusion of AHL to a local one, without considering the AHL that could come from the other side of the tumor.</P></SECTION><SECTION><H1 id="simplified-model-of-ahl-diffusion">Model of AHL diffusion</H1><P>Given the latter assumption, and given <SPAN class="math">\( w \ll r_1 \)</SPAN>, we assume that the colonized area appears locally as an infinite sheet of width <SPAN class="math">\( w \)</SPAN>. According to the first assumption stated above, we consider only <EM>production and diffusion</EM> to be significant within the sheet, and <EM>diffusion and degradation</EM> to be significant outside the sheet. We model the sheet as perpendicular to the <SPAN class="math">\( x \)</SPAN> axis, centered at <SPAN class="math">\( 0 \)</SPAN>, covering the interval <SPAN class="math">\( [-w/2, w/2 ] \)</SPAN>.</P><P>For a small parallelepiped of surface <SPAN class="math">\( \mathrm{d} S \)</SPAN> and width <SPAN class="math">\( \mathrm{d} x \)</SPAN> perpendicular to <SPAN class="math">\( x \)</SPAN> axis (see figure below), we can use <EM>Fick’s law</EM> to model diffusion of <SPAN style="font-variant: small-caps;">AHL</SPAN>.</P><P>The flux of the protein is:</P><P><SPAN class="math">\[\Phi(x) = - D \left. \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} x} \right|_{x}\]</SPAN></P><P>When inside (<SPAN class="math">\( x &lt; |w/2| \)</SPAN>) of the layer, the change of protein amount within time length <SPAN class="math">\( \mathrm{d} t \)</SPAN> is equal to diffusive transports plus production:</P><P><SPAN class="math">\[\begin{aligned}
        \mathrm{d} n &amp;= (( \Phi(x) - \Phi(x + \mathrm{d} x) ) \mathrm{d} S + P_{\text{AHL}} \mathrm{d} V) \, \mathrm{d} t \\
        \frac{\mathrm{d} n}{\mathrm{d} V \mathrm{d} t} &amp;= D \frac{\mathrm{d}^2 [\text{AHL}]}{\mathrm{d} x^2} + P_{\text{AHL}} \\     \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} t} &amp;= D \frac{\mathrm{d}^2 [\text{AHL}]}{\mathrm{d} x^2} + P_{\text{AHL}}\end{aligned}\]</SPAN></P><P>where <SPAN class="math">\( P_{\text{AHL}} \)</SPAN> is the <EM>volumic production</EM> of <SPAN style="font-variant: small-caps;">AHL</SPAN> in <SPAN class="math">\( \text{mol L}^{-1} \text{s}^{-1} \)</SPAN>.</P><P>Outside (<SPAN class="math">\( x &gt; |w/2| \)</SPAN>) of the layer, where there is diffusion and degradation, we get:</P><P><SPAN class="math">\[\begin{aligned}
        \mathrm{d} n &amp;= (( \Phi(x) - \Phi(x + \mathrm{d} x) ) \mathrm{d} S - k_{\text{deg}} \mathrm{d} n) \, \mathrm{d} t \\
        \frac{\mathrm{d} n}{\mathrm{d} V \mathrm{d} t} &amp;= D \frac{\mathrm{d}^{2} [\text{AHL}]}{\mathrm{d} x^2} - k_{\text{deg}} \frac{\mathrm{d} n}{\mathrm{d} V} \\
        \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} t} &amp;= D \frac{\mathrm{d}^{2} [\text{AHL}]}{\mathrm{d} x^2} - k_{\text{deg}} [\text{AHL}]\end{aligned}\]</SPAN></P></SECTION><SECTION><H2 id="diffusion-inside-the-layer">Solving for diffusion inside the layer</H2><P>We are interested in the concentration profile of <SPAN style="font-variant: small-caps;">AHL</SPAN> at steady state. Indeed, we assume that diffusion happens faster than the colonization of the bacteria (happening over 2 days <A href="#bib1" class="forward-ref">[1]</A>), so <SPAN class="math">\( P_{\text{AHL}} \)</SPAN> is considered constant in this quasi steady state assumption (QSSA), and we proceed as follows for <SPAN class="math">\( x \in (-w/2, w/2) \)</SPAN>:</P><P><SPAN class="math">\[\begin{aligned}
        \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} t} &amp;= 0 \\
        \frac{\mathrm{d}^2 [\text{AHL}]}{\mathrm{d} x^2} &amp;= - \frac{P_{\text{AHL}}}{D} \\
        [\text{AHL}](x) &amp;= - \frac{P_{\text{AHL}}}{2D} x^2 + c_0 x + c_1  \\
        [\text{AHL}](x) &amp;= - \frac{P_{\text{AHL}}}{2D} x^2 + c_1 &amp; \text{\( c_0 = 0\) because of symmetry}\end{aligned}\]</SPAN></P><P>The concentration profile has a parabolic shape.</P></SECTION><SECTION><H2 id="diffusion-outside-the-layer">Solving for diffusion outside the layer</H2><P>Applying a similar QSSA, we solve the differential equation as follows for <SPAN class="math">\( x \in (-\infty, -w/2) \cup (w/2, +\infty) \)</SPAN>:</P><P><SPAN class="math">\[\begin{aligned}
        \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} t} &amp;= 0 \\
        \frac{\mathrm{d}^2 [\text{AHL}]}{\mathrm{d} x^2} - \frac{k_{\text{deg}}}{D} [\text{AHL}] &amp;= 0 \\
        \frac{\mathrm{d}^2 [\text{AHL}]}{\mathrm{d} x^2} - 1/\kappa^2 [\text{AHL}] &amp;= 0 \ \  \  \  \  \kappa = \sqrt{D/k_{\text{deg}}} \simeq 7.7 \text{mm} \space \text{is the characteristic length of diffusion}\\
        [\text{AHL}](x) &amp;= c_2 \exp{(-x/\kappa)} + c_3 \exp{(x/\kappa)} + c_4 \end{aligned}\]</SPAN></P><P>We have <SPAN class="math">\( c_2 = c_3 := c \)</SPAN> due to symmetry of the problem.</P></SECTION><SECTION><H2 id="boundary-conditions">Boundary conditions</H2><P><SPAN style="font-variant: small-caps;">AHL</SPAN> concentration is <SPAN class="math">\( 0 \)</SPAN> at <SPAN class="math">\( \pm \infty \)</SPAN>. This implies <SPAN class="math">\( c_4 = 0 \)</SPAN> and:</P><P><SPAN class="math">\[\begin{aligned}
        [\text{AHL}] =
        \begin{cases}
        c \exp{(x/\kappa)} &amp; for \space x &lt; -w/2 \\
        c \exp{(-x/\kappa)} &amp; for \space x &gt; w/2 \\
        \end{cases}\end{aligned}\]</SPAN></P><P>The flux of AHL should be continuous at the boundary of the colonized area:</P><P><SPAN class="math">\[\begin{aligned}
        \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} x}(w^{+}/2) &amp;= \frac{\mathrm{d} [\text{AHL}]}{\mathrm{d} x}(w^{-}/2) \\
        -(1/\kappa) c \exp{(- w/2\kappa)} &amp;= -\frac{P_{\text{AHL}}}{D} \frac{w}{2} \\
        c &amp;= \frac{P_{\text{AHL}}w}{2D\kappa} \exp{(w/2\kappa)}\end{aligned}\]</SPAN></P><P>Plus, due to continuity of the concentration of AHL:</P><P><SPAN class="math">\[\begin{aligned}
        [\text{AHL}](w^{+}/2) &amp;= [\text{AHL}](w^{-}/2) \\
        c \exp{(-\kappa w/2)} &amp;= -\frac{P_{\text{AHL}}}{2D} \frac{w^2}{4} + c_1 \\
        c_1 &amp;= c \exp{(-w/2\kappa)} + \frac{P_{\text{AHL}}}{2D} \frac{w^2}{4} \\
        c_1 &amp;= \frac{P_{\text{AHL}}w\kappa}{2D} + \frac{P_{\text{AHL}}w^2}{8D} \\
        c_1 &amp;= \frac{P_{\text{AHL}}w}{2D}(\kappa + w/4)\end{aligned}\]</SPAN></P><P>If we assess the order of magnitude of each term, we notice that one can simplify this result since <SPAN class="math">\( \kappa \simeq 7.7 \text{mm} \)</SPAN> and <SPAN class="math">\( w/4 \simeq 0.125 \text{mm} \)</SPAN>:</P><P><SPAN class="math">\[c_1 \simeq \frac{P_{\text{AHL}}w\kappa}{2D}\]</SPAN></P></SECTION><SECTION><H2 id="full-solution">Full solution</H2><P>The final concentration of <SPAN style="font-variant: small-caps;">AHL</SPAN> is:</P><P><SPAN class="math">\[[\text{AHL}] =
        \frac{P_{\text{AHL}} w \kappa}{2 D} \cdot
        \begin{cases}
        \exp{(\frac{1}{\kappa} (x + w/2))} &amp; for \space x &lt; -w/2 \\
        \ 1+\frac{1}{w \kappa}\left(\frac{w^2}{4} - x^2 \right) &amp; for \space -w/2 &lt; x &lt; w/2 \\
        \exp{(\frac{1}{\kappa} (w/2 - x))} &amp; for \space x &gt; w/2
        \end{cases}\]</SPAN></P><P>A dimensional analysis can confirm that [AHL] is a concentration in M, as <SPAN class="math">\(P_{\text{AHL}}\)</SPAN> is a production rate in M.min<SUP>-1</SUP>, w and <SPAN class="math">\(\kappa\)</SPAN> two lengths in m, and D a diffusion coefficient in m<SUP>2</SUP>.min<SUP>-1</SUP> (the remaining expressions are dimensionless).
    </P></SECTION><SECTION><H1>AHL concentration inside of the layer</H1><P>To complete our model at the bacterial circuit level, we only need to know the AHL concentration inside the colonization layer. This is the AHL concentration the bacteria will be exposed to and to which they will react. If we compute the concentration of AHL at x=0, which is where the concentration is maximal, we get:</P><P><SPAN class="math">\[\begin{aligned}\text{AHL}(x=0) =
        \frac{P_{\text{AHL}} w \kappa}{2 D}(1+\frac{w}{4 \kappa} )
        \end{aligned}\]</SPAN></P><P>With the numerical values of w and <SPAN class="math">\(\kappa\)</SPAN> applying to our problem, since we have <SPAN class="math">\(\frac{w}{4 \kappa} \simeq 0.02\)</SPAN>, we can neglect this component of the equations, which amounts to ignore the intra-layer variation. The intuition is that this simplification is allowed to us because of the very fast diffusion compared to the width of the colonization layer (high <SPAN class="math">\(\kappa\)</SPAN> and small w), which results in a high homogenization of AHL concentration into the layer.</P><P>We therefore get the following equation relating AHL concentration to its volumetric production:</P><P><SPAN class="math">\[[\text{AHL}] =
        \frac{P_{\text{AHL}} w \kappa}{2 D}
        \]</SPAN></P></SECTION><SECTION><H1>Initial test of our model</H1><P>Our final in-vivo model comprises the following equations:</P><P><SPAN class="math">\[\begin{aligned}
        [\text{LuxR}]_0 &amp;= \frac{a_{\text{LuxR}}}{d_{\text{LuxR}}} &amp; \text{LuxR steady state concentration} \\        
        [\text{LuxR-AHL}] &amp;= K_{LuxRAHL} [\text{LuxR}]^2 [\text{AHL}]^2 &amp; \text{rapid binding equilibrium} \\
        [\text{LuxR}] &amp;= \frac{a_{\text{LuxR}}}{d_{\text{LuxR}}} - 2 [\text{LuxR-AHL}] &amp; \text{mass conservation}\\
        [\text{luxI}] &amp;= \frac{a_{\text{luxI}}}{d_{\text{luxI}}} (k_{\text{luxI}} + (1 - k_{\text{luxI}}) P_{\text{Lux-Lac}})  &amp; \text{LuxI concentration} \\
        [\text{AHL}] &amp;= \frac{d_{\text{cell}} a_{\text{AHL}} [\text{luxI}] w \kappa}{2 D} &amp; \text{AHL concentration}\\
\end{aligned}\]</SPAN></P><P>To simulate responses of our system under different values of lactate and bacterial cell density input, we solve this system using the fzero function of Matlab. Here is an example of the response of our system with typical values (see tables in the following part) to see how it behaves:</P><P>The white lines correspond to low levels of lactate and bacterial cell density (in healthy tissues) and the black lines represent the high levels (in tumor tissues). We can see that our system behaves well like an AND-gate as expected, but that the levels at which the transitions happen are not the right ones for our application. To find under which circumstances the system behaves as we need it to, we have to proceed to a <A href="https://2017.igem.org/wiki/index.php?title=Team:ETH_Zurich/Model/Environment_Sensing/parameter_space">parameter search.</A></P></SECTION><SECTION class="references"><H1>References</H1><OL><LI id="bib1"><A href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC219261/?page=4">Kaplan HB, Greenberg EP. Diffusion of autoinducer is involved in regulation of the Vibrio fischeri luminescence system. <SOURCE>Journal of Bacteriology.</SOURCE> 1985;163(3):1210-1214.</A></LI><LI id="bib2"><A href="http://www.pnas.org/content/101/30/10955.full">Jordi Garcia-Ojalvo, Michael B. Elowitz, and Steven H. Strogatz Modeling a synthetic multicellular clock: Repressilators coupled by quorum sensing <SOURCE>PNAS</SOURCE> 2004 101 (30) 10955-10960</A></LI><LI id="bib3"><A href="https://www.ncbi.nlm.nih.gov/pubmed/20100181">Fekete, A., Kuttler, C., Rothballer, M., Hense, B. A., Fischer, D., Buddrus-Schiemann, K., Lucio, M., Müller, J., Schmitt-Kopplin, P. and Hartmann, A. (2010), Dynamic regulation of N-acyl-homoserine lactone production and degradation in Pseudomonas putida IsoF. <SOURCE>FEMS Microbiology Ecology</SOURCE>, 72: 22–34. doi:10.1111/j.1574-6941.2009.00828.x</A></LI><LI id="bib4"><A href="http://www.sciencedirect.com/science/article/pii/S0303264705001267">A.B. Goryachev, D.J. Toh T.Lee, Systems analysis of a quorum sensing network: Design constraints imposed by the functional requirements, network topology and kinetic constants <SOURCE>Biosystems</SOURCE>, Volume 83, Issues 2–3, February–March 2006, Pages 178-187</A></LI><LI id="bib5"><A href="http://www.nature.com/nature/journal/v434/n7037/full/nature03461.html">A synthetic multicellular system for programmed pattern formation Subhayu Basu, Yoram Gerchman, Cynthia H. Collins, Frances H. Arnold &amp; Ron Weiss<SOURCE>Nature</SOURCE> 434, 1130-1134 (28 April 2005) | doi:10.1038/nature03461</A></LI><LI id="bib6"><A href="http://gingproc.iim.csic.es/meigo.html">Egea JA, Henriques D, Cokelaer T, Villaverde AF, MacNamara A, Danciu DP, Banga JR and Saez-Rodriguez J. (2014) MEIGO: an open-source software suite based on metaheuristics for global optimization in systems biology and bioinformatics. <SOURCE>BMC Bioinformatics</SOURCE> 15:136.</A></LI><LI id="bib7"><A href="https://gitlab.com/csb.ethz/HYPERSPACE">Zamora-Sillero E, Hafner M, Ibig A, Stelling J, Wagner A (2011) Efficient characterization of high-dimensional parameter spaces for systems biology. <SOURCE>BMC Syst Biol</SOURCE> 5: 142.http://doi.org/10.1186/1752-0509-5-142</A></LI><LI id="bib8"><A href="https://www.nature.com/nbt/journal/v27/n10/full/nbt.1568.html">Salis, Howard M., Ethan A. Mirsky, and Christopher A. Voigt. &quot;Automated design of synthetic ribosome binding sites to control protein expression.&quot;<SOURCE>Nature biotechnology</SOURCE> 27.10 (2009): 946-950. http://dx.doi.org/10.1038/nbt.1568</A></LI></OL></SECTION></MAIN><FOOTER id="footerN"><BUTTON onclick="goto_top()" id="goto-top" title="Go to top">Top</BUTTON><A class="sponsor-a" href="/Team:ETH_Zurich/Sponsors">Sponsors</A></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>