Team:ETH Zurich/Experiments/Cell Lysis
Project Idea
Story of CATE
CATE in Action
Circuit
Design
Engineering
Dry Lab
Overview
Tumor Sensor
Heat Sensor
3D Model
Final Integration
Wet Lab
Experiments
Lab Journal
Protocols
Safety
Sequences
Achievements
Results
Parts
Medal Criteria
InterLab
Collaborations
Human Practices
Overview
Integrated Practices
Public Engagement
Our Team
Attributions
Members
Sponsors
Cell Lysis Experiments
This is a detailed experiment page dedicated to an individual function. To access other experiments, go to our Experiments page . To get a quick glimpse at all of our achievements, check out Results .
Achievements
We rationally designed an RBS library in order to tune protein E expression strength . We managed to decrease expression levels at uninduced state far enough to successfully co-transform protein E with the thermosensing system.
We show that it is possible for our engineered bacteria to grow at 37 °C when transformed with the heat-inducible cell-lysis system. After inducation at 45 °C, we can show that the cells lyse and release their protein-content into the environment .
Introduction
CATE needs to release the previously accumulated Anti-Cancer Toxin to the extracellular space. We implemented the cell lysis mechanism from bacteriophage Phi X174. It is initiated upon recognition of the heat signal by the Heat Sensor . Development and test of the heat sensor is shown on the Heat Sensor Experiments page. Here we show the induction of cell lysis with the heat sensor and subsequent GFP release to the supernatant.
Figure 1. The genetic circuit of our TlpA heat sensor . TlpA represses the PTlpA Promoter. A temperature of 45 °C releases the repression leading to induction of protein E. Protein E molecules interfere with cell wall synthesis and lead to cell lysis. Previously accumulated toxins get released.
For more details about the lysis mechanism, go to Cell Lysis .
Phase I: Initial System Design
The cell lysis mechanism of phage Phi X174 needs a single gene called protein E to get activated. In the initial design we placed protein E under an inducible promotor called PLux. A ribosome binding site with large translation initiation rate was calculated with the Salis Lab RBS calculator and placed in front of the protein E coding sequence.
Figure 2. Cell Lysis test plamids. AHL inducible protein E is placed on a pSEVA291 vector. PF and SF are abbreviations for BioBrick Prefix and BioBrick Suffix restriction sites. RS1-RS4 are restriction sites that we introduced for subsequent cloning.
Initially, no transformants could be obtained. This was probably due to a high leakiness of the PTlpA, which lead to enough expression of protein E to lyse all successful transformants.
CONCLUSION: We knew now that the protein E must be regulated by a very tight promotor. We used this knowledge to engineer the Heat Sensor to a low base level expression of the regulated gene. Read here how we engineered leakiness of the PTlpA.
Phase II - Optimization of co-transformation efficiency
Initially, it was not possible to transform a protein E, regulated by the
Heat Sensor into E. coli. Therefore we reduced the translation initiation rate of the protein E RBS.
Protein E RBS library creation
We constructed a ribosome binding site library to find variants expressing little enough protein E to avoid lysis from leaky expression, but still enough to lyse the cells when the expression of protein E is induced. The RedLibs algorithm [1] was used.
We used the RedLibs algorithm to design a RBS library that would allow us vary protein expression and screen for improved variants. Therefore, we initially calculated the TIR values for a fully degenerate RBS library with 8 times N (fully degenerate base) at the positions -13 to -5 upstream of the ATG start codon using the Salis RBS calculator. [2] [3] This library with 65’536 variants would be too large to efficiently screen and contain too many unfunctional RBS sequences. Therefore, we used the RedLibs algorithm to reduce the library to smaller size and distribute it’s values uniformly[Rationally reduced libraries for combinatorial pathway optimization minimizing experimental effort https://www.nature.com/articles/ncomms11163]. The algorithm then provided us with a partially degenerate sequence, that could be implemented by a single cloning step and codes for an as uniform as possible distribution of TIR values.
Distribution of TIR values calculated with RedLibs. Most sequences have a low TIR and should therefore enable transformants to grow despite some leakiness of the promotor.
The degenerate primers were ordered and the library was created with a PCR amplification and subsequent Gibson assembly and transformation. The plasmid was designed in a way that transformants with correct insert produce GFP constitutively and the protein E is controlled by the heat sensor.
Figure 5. Heat inducible cell lysis test device. The plasmids contained RBS libraries of protein E (above) and TlpA (below).
The double transformation with the plasmids (Figure 5) grown at 37 °C, yielded green fluorescent colonies. This shows successfull inhibition of protein E at 37 °C (colonies are not lysed) and successful insertion of the protein E gene (+ const. promotor) between PLux and gfp (constitutive green at 37 °C)
Figure 6. Double transformation of the heat inducible cell lysis test device (protein E RBS library and TlpA RBS library).
Protein E RBS library variant selection
All fluorescent colonies were picked and inoculated to a 96 well plate and grown overnight (16 h) to stationary phase at 37°C. Continuing with the 96 well format, the samples were inoculated into a fresh 96 well culture plate (OD 0.1) and grown to OD600 0.4. At this point the cultures were split to fresh plates (flat transparent bottom) and incubated at 37 °C and 45 °C for 3 h. The OD600 was measured from the beginning of the OD600 0.1 culture to track the growth curve during induction. Four variants were selected for the subsequent experiment. They were restreaked to obtain multiple single clones for triplicate measurements.
Phase III - Heat-Induction of GFP Release by TlpA-controlled Protein E Expression
We tested the function of heat-induced cell lysis by inducing E. coli Top10 for 3 h at 45 °C in culture tubes in a shaking incubator. Because the fluorescence of each sample was different due to intrinsic noise, only the ratio of total fluorescence and fluorescence in the supernatant can give a cue about the lysis efficiency. Therefore we measured the total fluorescence and the supernatant fluorescence every hour for the induction period (Figure 7).
Figure 7. Protein E RBS library variants fluorescence ratio (supernatant/total). Four library variants were selected and induced with a heat shock of 45 °C (lower row). The negative control consists of a constitutively expressed GFP without protein E.
We performed the experiment according to the protocol with three protein E and TlpA RBS library variants and one protein E RBS library and improved TlpA RBS variant. (Read here how we produced it). The protein E RBS variants were sequenced and compared to the predicted translation initiation rates:
Table 1. Protein E RBS library variants -13 to -5 upstream sequences with their corresponding calculated translation initiation rates.
We showed that the Heat Sensor effectively induces protein E expression with 3 h of induction at 45 °C. The variant C (protein E RBS: CGGGGGGG, Table 1) has a tight repression because it was cotransformed with the engineered TlpA RBS (Figure 7, D).  CONCLUSION: We showed that heat induced cell lysis happens and about 70% of the total GFP is found in the supernatant after 3 h induction at 45 °C. This value underestimates the effective amount of released protein, because cell lysis might continue after the measurement period of 3 h and release even more protein. We also showed effective inhibition of cell lysis at 37 °C if the protein E is regulated by a highly expressed TlpA such as our engineered TlpA RBS variant (Figure 7, D).
References
Jeschek, Markus, Daniel Gerngross, and Sven Panke. "Rationally reduced libraries for combinatorial pathway optimization minimizing experimental effort."
Nature communications
7 (2016). doi: 10.1038/ncomms11163
Espah Borujeni, Amin, Anirudh S. Channarasappa, and Howard M. Salis. "Translation rate is controlled by coupled trade-offs between site accessibility, selective RNA unfolding and sliding at upstream standby sites."
Nucleic acids research
42.4 (2013): 2646-2659. doi: 10.1093/nar/gkt1139
Salis, Howard M., Ethan A. Mirsky, and Christopher A. Voigt. "Automated design of synthetic ribosome binding sites to control protein expression."
Nature biotechnology
27.10 (2009): 946-950. doi: 10.1038/nbt.1568
Top
Sponsors
