Team:TAS Taipei/Experiments
TAS_Taipei
Attributions About Us Safety Human Practices Modeling Prototype Experiments Project
EXPERIMENTS
ALDH2*1
DNA Design and Protein
Expression
IS OUR ALDH2*1
FUNCTIONAL?
ALDH2*2 DNA Design
Does ALDH2*1 Metabolize
Acetaldehyde Faster?
PURIFICATION OF
ALDH2*1
Design & Purification of
HIS-tagged Proteins
Does Purified ALDH2*1
Metabolize Acetaldehyde
Faster?
REGULATION OF
ALDH2*1 EXPRESSION
Ethanol-Sensitive Promoter
Is PalcA Activated by Ethanol
and alcR?
REFERENCES
LAB NOTEBOOK
SUMMARY
We genetically engineered Escherichia coli (E. coli) to constitutively produce normal ALDH2*1, and experimentally show that our recombinant ALDH2*1 is more efficient at breaking down acetaldehyde compared to the mutant ALDH2*2. To better characterize enzymatic activity, we also designed, purified, and tested both HIS-tagged ALDH2*1 and ALDH2*2 enzymes. Finally, we designed and tested an ethanol-induced promoter to regulate the production of ALDH2*1.
.
ALDH2*1
DNA Design and Protein Expression
ALDH2 is a mitochondrial enzyme responsible for oxidizing acetaldehyde, a toxic intermediate of ethanol metabolism, into harmless acetic acid (Figure 2-1) (Larson et al., 2005; Farrés et al., 1994). Our goal is to produce recombinant normal ALDH2*1 enzyme to help prevent the toxic buildup of acetaldehyde in ALDH2 deficient individuals.
Figure 2-1. Acetaldehyde accumulates in ALDH2 deficient individuals. Ethanol is first converted to a toxic intermediate, acetaldehyde, by alcohol dehydrogenase (ADH). For individuals with wild type ALDH2, acetaldehyde is converted to acetate by ALDH2*1. However, people who are ALDH2 deficient carry the mutant ALDH2*2, which cannot fully convert acetaldehyde into acetate, and acetaldehyde accumulates as a result. (Figure: Caroline C)
We obtained the DNA sequence of ALDH2*1, which has also been documented on the registry (BBa_M36520; NCBI: NM_001204889.1), and modified it to remove an internal PstI site and lower the guanine and cytosine content (ALDH2*1 basic part: BBa_K2539150, Figure 2-2), as recommended by Integrated DNA Technology (IDT). The sequence was then flanked by an upstream strong promoter and strong ribosome binding site (RBS) combination (BBa_K880005), and a downstream double terminator (BBa_B0015) to maximize expression of ALDH2*1 enzyme (Figure 2-2). This final construct was ordered from IDT and cloned into the standard pSB1C3 Biobrick backbone (BBa_K2539100; Figure 2-2). PCR check and sequencing results from Tri-I Biotech indicate that our final construct is correct.
Figure 2-2. ALDH2*1 expression construct and basic part. (Left) The expression construct BBa_K2539100 includes a constitutive strong promoter, strong RBS, the human ALDH2*1 open reading frame, and a double terminator. PCR check for BBa_K2539100 using VF2 and VR primers produced a band at the expected size of 2.1 kb (yellow box). (Right) BBa_K2539150 contains just the sequence of ALDH2*1. PCR check using VF2 and VR primers produced a band at the expected size of 1.8 kb (yellow box). (Cloning & Figures: Catherine C, Charlotte C)
We used SDS-PAGE to check for ALDH2*1 expression in E. coli carrying our construct. Bacterial cultures expressing either ALDH2*1 or GFP (control) were grown overnight at 37°C, lysed, and prepped. ALDH2*1 is approximately 56 kDa (Chen et al., 2014), and we observed a much darker band, which was slightly higher than 50 kDa, in the ALDH2*1 lysate sample (Figure 2-3), suggesting that recombinant ALDH2*1 is being expressed in the transformed E. coli.
Figure 2-3. SDS-PAGE results show that E. coli carrying BBa_K2539100 produce ALDH2*1. Bacterial cultures were grown overnight at 37°C, lysed, and prepped for SDS-PAGE. The expected size for ALDH2*1 is 56 kDa, and we observed a prominent band in E. coli carrying BBa_K2539100 around 50 kDa (arrowhead), but not in E. coli expressing GFP (used as a control). (Protein Gel & Figure: Leona T, Catherine C)
.
IS OUR ALDH2*1 FUNCTIONAL?
ALDH2*2 DNA Design
The ALDH2*2 gene consists of a single G to A point mutation; this leads to a change in codon 487, where GAA (coding for glutamic acid) is mutated to AAA (coding for lysine) (Figure 2-4; Enomoto et al., 1991). The nucleotide was changed in our DNA designs, and this ALDH2*2 sequence was similarly placed between the strong promoter and strong RBS combination (BBa_K880005) and a downstream double terminator (BBa_B0015) to maximize expression (Figure 2-4). This final construct was ordered from IDT and cloned into the standard pSB1C3 Biobrick backbone (BBa_K2539200; Figure 2-4). PCR check and sequencing results from Tri-I Biotech indicate that our final construct is correct.
Figure 2-4. ALDH2*2 expression construct and basic part. (A) The mutation in ALDH2*2 leads to a change in protein structure, which decreases its acetaldehyde metabolism efficiency. Figure taken from Li et al., 2016. (B, Left) The expression construct BBa_K2539200 includes a constitutive strong promoter, strong RBS, the human ALDH2*2 open reading frame, and a double terminator. PCR check for BBa_K2539200 using VF2 and VR primers produced a band at the expected size of 2.1 kb (yellow box). (B, Right) BBa_K2539250 contains just the sequence of ALDH2*2. PCR check using VF2 and VR primers produced a band at the expected size of 1.8 kb (yellow box). (Cloning & Figures: Catherine C, Charlotte C)
.
Does ALDH2*1 Metabolize Acetaldehyde Faster?
With both forms of ALDH2 prepared, we could compare the enzymatic activity of wild type ALDH2*1 to the mutant ALDH2*2. When ALDH2 converts acetaldehyde into acetate, NADH is produced (Figure 2-5, A). To test the ability of our recombinant ALDH2 to metabolize acetaldehyde, we used reagents from a kit (Megazyme, K-ACHYD) to quantify the amount of NADH produced by taking absorbance readings at 340 nm. This wavelength is highly absorbed by the reduced form, NADH, but not the oxidized form, NAD+ (Harimech et al., 2015; McComb et al., 1976). We expected to see both forms of ALDH2 metabolize acetaldehyde, and as a result, produce NADH, but ALDH2*1 should be more efficient compared to ALDH2*2.
Figure 2-5. ALDH2*1 converts acetaldehyde at a faster rate than ALDH2*2. (A) The conversion of acetaldehyde to acetate by ALDH2 uses NAD+ and produces NADH. (B) Experimental setup. The supernatant from E. coli cell lysates was mixed with acetaldehyde and NAD+ to initiate the reaction at 25°C. NADH concentration was measured by taking absorbance readings at 340 nm. (C) Relative activity of lysates containing either ALDH2*1, ALDH2*2, or inactive ALDH2*1 (boiled to denature proteins; negative control). Over a 40-minute period, E. coli carrying our ALDH2*1 construct (BBa_K2539100) produced more NADH than the mutant form (BBa_K2539200), while the negative control (boiled BBa_K2539100) did not change significantly. Error bars represent standard error. (Experiment & Figure: Justin W, Yvonne W, Catherine C)
Bacterial liquid cultures were prepared, and the populations were standardized by measuring the OD600 of each culture, and then diluting cultures appropriately to achieve the same population in all samples. Next, the cultures were spun down, and the pellets were resuspended in xTractor Lysis Buffer (Takara Bio, 635671) in order to lyse the cells. After lysis, the supernatant containing proteins was transferred into quartz cuvettes containing a pH 9.0 buffer, distilled water, and excess NAD+ (all solutions were provided by the kit) (Figure 2-5, B). Acetaldehyde was added to initiate the reaction, and absorbance values at 340 nm were recorded for 40 minutes. As a negative control, cell lysates of E. coli expressing ALDH2*1 were boiled to denature all proteins.
Over a 40 minute period, the absorbance at 340 nm increased faster for ALDH2*1 compared to ALDH2*2, but did not significantly change for the negative control (inactive ALDH2*1) (Figure 2-5, C). Our results show that functional ALDH2*1 and ALDH2*2 are being produced, and there is a significant difference in the acetaldehyde metabolism rate (the error bars do not overlap, Figure 2-5, C). However, the error bars were close, so we decided to purify the proteins to see if we could observe a greater difference in enzyme activity between the wild type and mutant ALDH2.
.
PURIFICATION OF ALDH2*1
Since these tests were run using cell extracts, we hypothesized that there must be numerous other E. coli enzymes present which also reduce NAD+ into NADH, or oxidize NADH back into NAD+. To remove these other factors that might affect NADH concentration in the E. coli, we decided to directly test the activity of purified ALDH2*1 and ALDH2*2 enzymes.
.
Design & Purification of HIS-tagged Proteins
We added a HIS-tag (6xHIS) to the N-terminus of both ALDH2*1 and ALDH2*2 sequences. Like the other constructs, this was flanked by a strong promoter and strong RBS combination (BBa_K880005) and a downstream double terminator (BBa_B0015) to maximize expression (Figure 2-6). The final sequence was ordered from IDT and cloned into the standard Biobrick backbone (pSB1C3). PCR check and sequencing results from Tri-I Biotech indicate that our constructs are correct.
Figure 2-6. PCR check results for HIS-tagged ALDH2 constructs (BBa_K2539101 and BBa_K2539201) using VF2 and VR primers. The construct contains a strong promoter, strong RBS, a start codon with a 6x HIS tag, either the ALDH2*1 or ALDH2*2 sequence, and ends with a double terminator. The expected size of both PCR products are 2.1 kb (yellow box). (Figure and Cloning: Catherine C)
To purify the HIS-tagged proteins, we grew cultures of E. coli expressing HIS-tagged ALDH2*1 and ALDH2*2. The bacterial cultures were centrifuged, lysed, and filtered before the cell extract sample was run through a nickel column (GE Healthcare, 11-0033-99). HIS-tagged proteins should bind to the column, which contained nickel ions. Then, we ran a low-concentration imidazole buffer through the column to wash out any non-specific proteins. Finally, we eluted and collected the HIS-ALDH2 proteins by running a high-concentration imidazole concentration buffer through the column (imidazole competes with the HIS tag to bind with nickel ions in the column) (Bornhorst et al., 2000). We used SDS-PAGE to check protein content at different steps of the purification process: lysed cell sample, flow-through after the wash buffer, and final eluate containing the purified protein (Figure 2-7). HIS-tagged ALDH2*1 should be around 56 kDa. In the initial cell lysate lane, there is a band around 56 kDa (Figure 2-7). This band disappears in the wash buffer flow-through lane, and reappears in the eluate. A similar band around 50 kDa was also seen in the eluate when we purified ALDH2*2 (Figure 2-7). For both HIS-tagged samples, there are also some other proteins in the eluate, but our gels show that we now have much purer samples of both HIS-tagged ALDH2*1 and ALDH2*2, which should give clearer results in our functional test.
Figure 2-7. SDS-PAGE results show protein content at different steps of protein purification. (Left) A band around 56 kDa in the cell extract (green) and the eluate (red), but not present in the wash buffer flow through lane (yellow), matches our expected HIS-tagged ALDH2*1. (Right) The same band can be seen in the eluate lane for HIS-tagged ALDH2*2 (red). (Protein Purification/Gel & Figure: Leona T, Justin W, Catherine C)
.
Does Purified ALDH2*1 Metabolize Acetaldehyde Faster?
Figure 2-8. Purified ALDH2*1 has a much higher activity level compared to purified ALDH2*2. Using the same procedure described in Figure 2-5, here, the enzymatic activity of purified ALDH2*1 and ALDH2*2 were tested at 25°C. A negative control containing only elution buffer (from the protein purification process) was also included (gray). There is a significant and much clearer difference in enzymatic activity between purified ALDH2*1 and ALDH2*2, compared to when bacterial cell extracts were tested. ALDH2*1 steadily metabolized more acetaldehyde compared to both ALDH2*2 and the negative control, both of which did not seem to have any effect. The error bars represent standard error, and are bolded for ALDH2*1 and ALDH2*2 to highlight the difference between the two groups (Experiment & Figure: Justin W)
We repeated our functional test using purified enzymes (Figure 2-8), and saw a much clearer difference between ALDH2*1 and ALDH2*2, compared to when we used bacterial cell extracts. When cell extracts were tested, ALDH2*1 seemed to work slightly better, but NADH concentrations increased for both ALDH2*1 and ALDH2*2. When the cell extracts were removed, however, purified ALDH2*2 did not have any effect on NADH, while purified ALDH2*1 significantly increased NADH levels (Figure 2-8). This result seems logical and matches research which show that when only the mutant ALDH2*2 is present, enzymatic activity can drop down to a mere 4% compared to normal ALDH2*1 (Gross et al., 2015). This also shows that the increase seen with ALDH2*2 cell extracts (Figure 2-5) was likely due to other enzymes in the bacteria that also reduce NAD+ and produce NADH.
.
REGULATION OF ALDH2*1 EXPRESSION
Ethanol-Sensitive Promoter
When we surveyed the public about their preferences for treatment, most people indicated that they prefer treatment only when they drink (instead of daily, weekly, etc.). We had the same thought--we wanted to regulate expression so that ALDH2 is only made in the presence of ethanol.
Figure 2-9. The combination of transcription factor, alcR, and ethanol activates the PalcA promoter. BBa_K2539300 contains a strong constitutive promoter which regulates the expression of alcR. After alcR is made, it can bind to ethanol and activate PalcA, which controls the expression of ALDH2*1 in BBa_K2539400. (Figure: Charlotte C, Catherine C)
PalcA (BBa_K2092001) is an ethanol-induced promoter that also requires the binding of a transcription factor, alcR, in order to be activated (Figure 2-9). (Li et al., 2005; Homa et al., 2015) We designed ALDH2*1 to be placed after the PalcA promoter and a strong RBS (BBa_B0034), and before a double terminator (BBa_B0015) (PalcA_ALDH2*1: BBa_K2539400). Since PalcA is active only in the presence of both ethanol and alcR, we also overexpressed alcR by placing the alcR sequence between a strong constitutive promoter and RBS combination (BBa_K880005), and before a double terminator (BBa_B0015) (Pconst_alcR: BBa_K2539300; Figure 2-10). These sequences were ordered from IDT, and cloned together into one pSB1C3 backbone (BBa_K2539450). All sequences were confirmed by PCR check and Tri-I Biotech.
Figure 2-10. PCR check for ethanol-regulation constructs (BBa_K2539300, BBa_K2539500, BBa_K2539550, BBa_K2539400, BBa_K2539450) using VF2 and VR primers. The expected PCR product size of each construct is indicated. (Cloning & Figures: Catherine C, Charlotte C)
.
Is PalcA Activated by Ethanol and alcR?
We used green fluorescence protein (GFP) as a reporter to test the function of PalcA. The GFP sequence was placed behind the PalcA promoter, a strong RBS (BBa_B0034), and before a double terminator (BBa_B0015) to express GFP in the presence of ethanol and alcR (PalcA_GFP: BBa_K2539500; Figure 2-10). We then cloned PalcA_GFP (BBa_K2539500) and Pconst_alcR (BBa_K2539300) together into pSB1C3 to make BBa_K2539550. All sequences were confirmed by PCR check (Figure 2-10) and Tri-I Biotech.
We tested E. coli carrying Pconst_alcR+PalcA_GFP (BBa_K2539550) using different concentrations of ethanol (Figure 2-11). Bacterial cultures with different plasmids were grown overnight. In addition to Pconst_alcR+PalcA_GFP (BBa_K2539550), cultures of Pconst_alcR (BBa_K2539300), PalcA_GFP (BBa_K2539500), and a positive control expressing GFP were also prepared. An initial fluorescence reading was taken using a 96 well plate reader. Next, different amounts of ethanol were added to each tube, the tubes were sealed to prevent evaporation, and left in a shaking incubator for 24 hours. A final measurement was taken, again using a 96 well plate reader, to detect any changes in fluorescence.  If PalcA is functional, then we should see Pconst_alcR+PalcA_GFP (BBa_K2539550) cultures glow green in the presence of ethanol.
Figure 2-11. Relative fluorescence of E. coli cultures carrying Pconst_alcR+PalcA_GFP (BBa_K2539550) with varying amounts of ethanol. Over 24 hours, the sample with 0.3 g ethanol (EtOH) had the biggest increase in fluorescence. Pconst refers to the constitutive promoter we used (in BBa_K880005). Lysogeny broth (LB), Pconst_alcR, and PalcA_GFP were used as negative controls for this experiment. (Experiment & Figure: Catherine C, Jake Y, Justin W)
Our results matched the general expected trend (Figure 2-11). Addition of 0.3 g of ethanol to Pconst_alcR+PalcA_GFP (BBa_K2539550) seemed to yield the highest fluorescence, and lower amounts of  0.01 g and 0.1 g ethanol were not significantly different than the controls (LB only, Pconst_alcR, and PalcA_GFP). Over 0.5 g of ethanol seemed to kill the bacteria (3 mL of culture), which matches literature thresholds of ethanol tolerance (Chong et al., 2013). In our tests, 0.3 g of ethanol yielded the greatest amount of GFP; however, this effect was extremely weak and the measured fluorescence was about 500 times lower than the positive control (constitutively expressed GFP; not shown here). In summary, the ethanol promoter, PalcA, is functional in the presence of alcR and a specific concentration of ethanol, but is very inefficient. Thus, this promoter is not applicable for our goal of regulating ALDH2 production.
.
REFERENCES
Bornhorst JA, Falke JJ. (2000). Purification of proteins using polyhistidine affinity tags. Methods Enzymol. 326:245-54.
Chen CH, Ferreira JCB, Gross ER, Mochly-Rosen D. (2014). Targeting Aldehyde Dehydrogenase 2: New Therapeutic Opportunities. Physiol Rev. 94(1):1-34.
Chong H, Huang L, Yeow J, Wang I, Zhang H, Song H, Jiang R. (2013). Improving Ethanol Tolerance of Escherichia coli by Rewiring Its Global Regulator cAMP Receptor Protein (CRP). PLoS ONE, 8(2), e57628.
Enomoto N, Takada A, Date T. (1991). Genotyping of the aldehyde dehydrogenase 2 (ALDH2) gene using the polymerase chain reaction: evidence for single point mutation in the ALDH2 gene of ALDH2-deficiency. Gastroenterol Jpn. 26(4): 440–447.
Farrés J, Wang X, Takahashi K, Cunningham SJ, Wang TT, Weiner H. (1994). Effects of changing glutamate 487 to lysine in rat and human liver mitochondrial aldehyde dehydrogenase. A model to study human (Oriental type) class 2 aldehyde dehydrogenase. J Biol Chem. 13;269(19):13854-60.
Gross ER, Zambelli VO, Small BA, Ferreira JCB, Chen CH, Mochly-Rosen D. (2015). A personalized medicine approach for Asian Americans with the aldehyde dehydrogenase 2*2 variant. Annu Rev Pharmacol Toxicol. 55:107-27.
Harimech PK, Hartmann R, Rejman R, del Pino P, Rivera-Gila P, Parak WJ. (2015). Encapsulated enzymes with integrated fluorescence-control of enzymatic activity. J. Mater. Chem. B. 3, 2801-2807.
Hemmati H, Basu C. (2015). Transcriptional analyses of an ethanol inducible promoter in Escherichia coli and tobacco for production of cellulase and green fluorescent protein. Biotechnol Biotechnol Equip. 29(6):1043-1052.
Larson HN, Weiner H, Hurley TD. (2005). Disruption of the Coenzyme Binding Site and Dimer Interface Revealed in the Crystal Structure of Mitochondrial Aldehyde Dehydrogenase “Asian” Variant. J Biol Chem. 280(34):30550-6.
Li R, Jia X, Mao X. (2005). Ethanol-inducible gene expression system and its applications in plant functional genomics. Plant Science. 169(3):463-469.
Li R, Zhao Z, Sun M, Luo J, Xiao Y. (2016). ALDH2 gene polymorphism in different types of cancers and its clinical significance. Life Sci. 147:59-66.
McComb RB, Bond LW, Burnett RW, Keech RC, Bowers, GN Jr. (1976). Determination of the molar absorptivity of NADH. Clin Chem. 22(2): 141–150.
