<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TAS_Taipei_Model skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TAS Taipei/Model</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TAS_Taipei</TITLE><NAV class="toolbar"><A class="att nav-tab tab8" href="https://2018.igem.org/Team:TAS_Taipei/Attributions">Attributions</A><A class="as nav-tab tab7" href="https://2018.igem.org/Team:TAS_Taipei/Team">About Us</A><A class="safe nav-tab tab6" href="https://2018.igem.org/Team:TAS_Taipei/Safety">Safety</A><A class="hp nav-tab tab5" href="https://2018.igem.org/Team:TAS_Taipei/Human_Practices" style="width: 15% !important;">Human Practices</A><A class="mod nav-tab tab4" href="https://2018.igem.org/Team:TAS_Taipei/Model">Modeling</A><A class="prot nav-tab tab3" href="https://2018.igem.org/Team:TAS_Taipei/Applied_Design">Prototype</A><A class="exp nav-tab tab2" href="https://2018.igem.org/Team:TAS_Taipei/Experiments">Experiments</A><A class="pro nav-tab tab1" href="https://2018.igem.org/Team:TAS_Taipei/Description">Project</A></NAV></P><DIV class="row full"><DIV class="jumbo col-lg-12"><H1 class="meet center">MODELING</H1></DIV></DIV><DIV class="row sidenavback"><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#ModelLink" class="snfont overview">MODEL PARAMETERS</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#EthanolLink" .="" class="snfont F13">ETHANOL &amp; ACETALDEHYDE CONCENTRATIONS</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#EnzymeLink" class="snfont EnzAct">ENZYME ACTIVITIES OF ADH &amp; ALDH2</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#ADHLink" class="snfont ADH">ADH</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#ALDH2Link" class="snfont ALDH2">ALDH2</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#EliminateLink" class="snfont Eliminate">HOW MUCH ACETALDEHYDE TO ELIMINATE?</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#ModSeqLink" class="snfont ModSeq">Modeling SequentialEnzymatic Reactions</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#SoftwareLink" class="snfont Soft">Software</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#CalculateLink" class="snfont Calculat">Calculating Required ALDH2Levels</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#DetermineLink" class="snfont Determine">Determining the Number ofALDH2 Molecules and Bacterial Cells Needed in the Candy</A></H1></DIV><DIV class="col-lg-2 fixed paddingremoval"><H1><A href="#RefLink" class="snfont References">REFERENCES</A></H1></DIV></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly" id="t1">MODELING</H1></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">The two main objectives of our mathematical model are to:</H6><DIV class="col-lg-8 col-lg-offset-3"><OL class="a"><LI>Predict how much ALDH2 to consume in order to reduce acetaldehyde to match wild type ALDH2 levels.
  </LI><LI>Inform manufacturers how much ALDH2 needs to be put into the candies.
</LI></OL></DIV><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">To answer both of these questions, we started by modeling the kinetics of alcohol metabolism in the mouth.</H6></DIV><DIV class="row"><P class="dote col-lg-offset-3" id="ModelLink">.</P></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly" id="members">OVERVIEW OF MODEL PARAMETERS</H1></DIV><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">Our model treats the alcohol metabolism process as a series of steps mediated by two enzymes, alcohol dehydrogenase (ADH) and aldehyde dehydrogenase 2 (ALDH2). ADH converts ethanol to acetaldehyde (a toxic intermediate), and ALDH2 converts acetaldehyde to acetate (Figure 4-1).</H6><DIV class="row col-lg-offset-3 image_containercenter col-lg-3 imagebox"><H5 class="subtitle"><B>Figure 4-1: Alcohol Metabolism.</B> Normally, ethanol is first converted to acetaldehyde (a toxic intermediate) by the enzyme alcohol dehydrogenase (ADH). Then, acetaldehyde is converted to acetate, which can be safely metabolized in the body, by the enzyme aldehyde dehydrogenase 2 (ALDH2). ALDH2*1 refers to wild type ALDH2. (Figure: Caroline C)</H5></DIV><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">Since salivary acetaldehyde is the toxic intermediate we want to eliminate, we needed to first understand how fast acetaldehyde is made and metabolized in the mouth. In the ethanol metabolism pathway (Figure 4-1), four factors affect the acetaldehyde levels in the mouth after alcohol consumption.</H6><DIV class="col-lg-8 col-lg-offset-3"><OL class="a"><LI>Concentration of <B>ethanol</B> in the mouth
  </LI><LI>How fast <B>ADH</B> converts ethanol into acetaldehyde
  </LI><LI>Concentration of <B>acetaldehyde</B> in the mouth
  </LI><LI>How fast <B>ALDH2*1</B> converts acetaldehyde into acetate
</LI></OL></DIV><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly" id="EthanolLink">We obtained literature values for factors 1-3, and used our own experimental data to determine factor 4. </H6></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly" id="members">FACTORS 1 &amp; 3:Determining Ethanol &amp; Acetaldehyde Concentrationsin the Mouth</H1></DIV><DIV class="row col-lg-offset-3 image_container_table col-lg-3 imagebox"><H5 class="subtitle"><B>Table 4-2. Salivary acetaldehyde levels after consuming one mouthful of different alcoholic beverages.</B> Ethanol concentrations after 0.5 minutes were calculated as proportional to the reduction in acetaldehyde concentration after 0.5 minutes. (Lachenmeier <I>et al.</I>, 2011)</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">We researched the ethanol and acetaldehyde concentrations found in different alcoholic beverages. In Lachenmeier (2011), acetaldehyde concentrations were measured in the alcoholic beverage, as well as in the mouth 30 seconds after consumption. Immediately after consumption, we assume that most of the alcohol is swallowed. Hence, <B>we set the salivary ethanol and acetaldehyde levels 30 seconds after consumption (what remains in the mouth) as the initial concentrations for our model.</B></H6></DIV><DIV class="row"><P class="dote col-lg-offset-3" id="EnzymeLink">.</P></DIV></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly">FACTORS 2 &amp; 4:Determining Enzyme Activities of ADH and ALDH2
</H1></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">We modeled both the ADH and ALDH2 reaction rates using Michaelis-Menten kinetics.</H6></DIV><DIV class="row col-lg-offset-3 image_container42 col-lg-3 imagebox"><H5 class="subtitle3"><B>Equation 1.</B> The Michaelis-Menten Equation above models the rate of substrate elimination (or rate of product formation) of a certain concentration of enzymes at varying concentrations of substrates. V<SUB>0</SUB> is the enzymatic activity for a given substrate concentration. S is the substrate (either ethanol or acetaldehyde, depending on the enzyme); V<SUB>max</SUB> represents the maximum reaction rate (dP/dt); K<SUB>m</SUB> is the Michaelis-Menten constant and represents the substrate concentration at which the reaction rate is half of V<SUB>max</SUB>.</H5></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly" id="ADHLink">FACTOR 2:Determining Enzyme Activity of AlcoholDehydrogenase (ADH1B*2)
</H1></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">There are two main forms of alcohol dehydrogenase (ADH) involved in ethanol metabolism: ADH1B*1 (slower form) and ADH1B*2 (faster form). ALDH2 deficiency is an issue that mainly exists among East Asians, and we found that over 90% of East Asians have at least one ADH1B*2 allele (Eng <I>et al.</I>, 2007). In our model, we made an assumption that all patients are heterozygous for ADH1B (*1/*2). From literature, we obtained the K<SUB>m</SUB> value for ADH1B*2 (which we assume to be the same for a slower form), and an effective rate of ethanol metabolism by ADH1B*1/*2 (0.82 uM/sec; unit conversion calculations are shown below; rate adapted from Yokoyama <I>et al.</I>, 2016).</H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">We then created a Michaelis-Menten curve modeling the enzymes kinetics of the heterozygous ADH1B*1/*2 genotype.</H6></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Figure 4-3. Michaelis-Menten curve modeling the enzymes kinetics of the heterozygous ADH1B*1/*2 genotype.</B> This curve reflects the effective reaction rate of ADH1B*1/*2 at various concentrations of ethanol. (Figure: Justin L)</H5></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly" id="ALDH2Link">FACTOR 4:Determining Enzyme Activity of AldehydeDehydrogenase (ALDH2*1)
</H1></DIV><DIV class="row"><H3 class="col-lg-12 col-lg-offset-3 paddingremovaly indent" id="">Purified ALDH2*1 Activity at 37°C</H3></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">In <A href="https://2018.igem.org/Team:TAS_Taipei/Applied_Design" target="_blank">Prototype</A>, we tested the enzymatic activity of purified ALDH2*1 at 37°C (Figure 3-2). In these experiments, we measured changes in NADH absorbance at 340 nm. To convert NADH absorbance into changes in concentration of NADH, we used the Beer-Lambert Law.</H6></DIV><DIV class="row col-lg-offset-3 image_container42 col-lg-3 imagebox"><H5 class="subtitle35"><B>Equation 2. Conversion of absorbance values to changes in NADH concentration. (Left)</B> The variable <B>A</B> denotes NADH absorbance at 340 nm; the variable <B>ε</B> is a constant that denotes the NADH molar extinction factor at the same wavelength of light; the variable <B>b</B> is a constant that denotes the pathlength of the light; and <B>C</B> denotes the NADH concentration. <B>(Right)</B> The Beer-Lambert Law in the form of a differential equation, with changes in <B>A</B> relating to changes in <B>C</B> over time. We used this equation to determine the corresponding change in NADH concentration using a change in ∆A<SUB>NADH</SUB> from our functional test. (Figure: Justin L) </H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">From the graph in Figure 3-2, we calculated the change in OD340 of NADH over a time period of 40 minutes. We first corrected the absorbance by subtracting the elution buffer control from the purified enzyme at t = 40 min. We then used the difference in absorbance from t = 0 min to t = 40 min to calculate <B>dA<SUB>NADH</SUB>/dt</B>. By dividing dA/dt with the extinction coefficient <B>ε<SUB>NADH</SUB></B>(6220/M<SUP>-1</SUP>Cm<SUP>-1</SUP>, as indicated on the Megazyme kit manual) and the pathlength <B>b</B> (1 cm, which is the length of the cuvette), we calculated the rate of change in [NADH] over time (Figure 4-4). Because acetaldehyde, acetate, and NADH all react in a one-to-one ratio in the oxidation of acetaldehyde (Figure 4-1), a change in NADH concentration is equivalent to changes in acetaldehyde and acetate concentrations.  </H6></DIV><DIV class="row col-lg-offset-3 image_container39 col-lg-3 imagebox"><H5 class="subtitle"><B>Figure 4-4. Decrease in acetaldehyde concentration as a result of purified ALDH2*1 in artificial saliva at 37°C.</B> The acetaldehyde concentrations were calculated from the raw absorbance values and the Beer-Lambert Law. The calculated acetaldehyde metabolism rate is 0.270 uM/min or 4.50 nM/sec. (Figure: Justin W)</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">0.222 nM of purified ALDH2*1 was used in this experiment, where we determined the dA<SUB>NADH</SUB>/dt at 37°C to be 0.067/40 min. We then used the Beer-Lambert Law to determine that <B>at 37°C, 0.222 nM of purified ALDH2*1 enzymes converts acetaldehyde at a rate of 4.50 nM/sec</B> (Figure 4-4).</H6><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">Using a nanodrop spectrophotometer, the experimental concentration of purified ALDH2*1 was measured to be 0.222 nM. To determine if there was any enzyme degradation during the reaction, we used a rate equation based on the Michaelis-Menten model. The constant k<SUB>cat</SUB> represents the conversion rate of the ALDH2*1-acetaldehyde complex to acetate (1180/min; from Rashkovetsky <I>et al</I>., 1994). We used 4.50 nM/sec from the experiment above as the V<SUB>max</SUB> for ALDH2*1.  We solved for [E<SUB>0</SUB>], the concentration of reacted ALDH2 enzymes, which turned out to be 0.229 nM. Because the measured and reacted ALDH2*1 concentrations have a 3% standard error, we concluded that <B>enzyme degradation is negligible</B>.  
</H6><H5 class="subtitle3"><B>Equation 3. Rate equation derived from the Michaelis-Menten model.</B> The maximum enzymatic activity, V<SUB>max</SUB>, is equal to the product of the constant k<SUB>cat</SUB>  and the initial concentration of ALDH2*1, the enzymes. K<SUB>cat</SUB> has a unit of min<SUP>-1</SUP>. This model shows that enzymatic degradation is negligible throughout the course of the reaction. 
</H5></DIV><DIV class="row"><H3 class="col-lg-12 col-lg-offset-3 paddingremovaly indent" id="">ALDH2*1-<I>Ec</I>N Activity at 37°C</H3></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">We next determined the rate of ALDH2*1-<I>Ec</I>N activity at 37°C.
</H6></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Figure 4-5. Decrease in acetaldehyde concentration as a result of ALDH2*1 in 4.79*10<SUP>8</SUP> cells <I>Ec</I>N at 37°C.</B> Using experimental data from Figure 3-8 (see <A href="https://2018.igem.org/Team:TAS_Taipei/Applied_Design" target="_blank">Prototype</A> for details), absorbance values from ALDH2*2 in <I>Ec</I>N were subtracted from ALDH2*1 in <I>Ec</I>N. Following our assumption that bacteria cells contain other enzymes that may also convert NAD+ to NADH, this subtraction takes away the effects of other enzymes, and represents only the NADH converted by ALDH2*1. Absorbance values were then converted to acetaldehyde concentration, so we could determine the rate of acetaldehyde metabolism of the ALDH2*1 in our <I>Ec</I>N probiotic. (Experiment &amp; Figure: Justin W)
</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">In our Prototype section, we documented functional tests for ALDH2*1-carrying <I>Ec</I>N in artificial saliva at 37°C. Using experimental data from Figure 3-8 (in the <A href="https://2018.igem.org/Team:TAS_Taipei/Applied_Design" target="_blank">Prototype</A> section), absorbance values from ALDH2*2 in <I>Ec</I>N were subtracted from ALDH2*1 in <I>Ec</I>N. Because bacteria cells contain other enzymes that may also convert NAD+ to NADH, this subtraction allows us to determine the effects of other enzymes, and represents only the NADH converted by ALDH2*1. This correction was performed under the assumption that the enzymatic activity of ALDH2*2 was negligible. We then converted absorbance values to acetaldehyde concentration to determine the rate of acetaldehyde metabolism of the ALDH2*1 in our <I>Ec</I>N bacteria. Here, we determined that our <B>ALDH2*1-<I>Ec</I>N bacteria converts acetaldehyde at a rate of 6.37 nM/sec at 37°C</B> in artificial saliva.
Knowing the enzyme activity of purified ALDH2*1 at 37°C (0.222 nM of purified ALDH2*1 converts 4.50 nM acetaldehyde/sec), and the rate of conversion in our ALDH2*1-<I>Ec</I>N (6.37 nM acetaldehyde/sec), we set up a ratio to obtain the concentration of ALDH2*1, <B>X</B>, present in the <I>Ec</I>N from our experiment (Figure 4-5).
</H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly" id="EliminateLink">Through this ratio, we calculated <B>X</B> to be <B>0.314 nM of ALDH2*1 (carried in the <I>Ec</I>N from our functional tests), which converted 6.37 nM acetaldehyde/sec.</B></H6></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly">HOW MUCH ACETALDEHYDE SHOULD BEELIMINATED?</H1></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly" id="ModSeqLink">After understanding the variables involved in acetaldehyde regulation, we can now model how fast acetaldehyde is being made and metabolized in the mouth after alcohol consumption.</H6></DIV><DIV class="row"><H3 class="col-lg-12 col-lg-offset-3 paddingremovaly indent" id="">Modeling Sequential Enzymatic Reactions</H3></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Figure 4-6. A schematic diagram of a two-step sequential enzymatic reaction.</B> In the metabolic pathway of ethanol to acetate, Substrate 1 represents ethanol; Enzyme 1 represents ADH1B*2; Substrate 2 represents acetaldehyde; Enzyme 2 represents ALDH2; Product represents acetate.</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">We constructed a system of two differential equations to model the change in concentrations of ethanol and acetaldehyde in a human’s mouth over time based on the pathway illustrated in Figure 4-6.</H6></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Equation 4</B> models the change in ethanol concentration over time. We set it as equal to the enzymatic activity of ADH1B*2 enzymes (Figure 4-3). The symbol [E] represents ethanol concentration. This differential equation corresponds to the conversion rate of substrate 1 to substrate 2 in Figure 4-6.</H5></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Equation 5</B> models the change in acetaldehyde concentration over time. We set it as equal to the conversion rate of ethanol to acetaldehyde (d Ethanol/dt) minus the enzymatic activity of ALDH2 enzymes. The acetaldehyde elimination rate (AER) and k<SUB>m</SUB> are constant values that vary for different ALDH2 genotypes. For example, a person who is homozygous wild type (ALDH2*1/*1) has an AER of 2.1 µM/sec and a K<SUB>m</SUB> of 0.2 µM (Rashkovetsky <I>et al</I>., 1994).  The symbol [A] represents acetaldehyde concentration. This differential equation corresponds to the conversion rate of Substrate 2 to the Product in Figure 4-6.
</H5><P class="dote col-lg-offset-3" id="SoftwareLink">.</P></DIV><DIV class="row"><H3 class="col-lg-12 col-lg-offset-3 paddingremovaly indent">Software</H3></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">We then created a Python program (see source code on our <A href="https://2018.igem.org/Team:TAS_Taipei/Software" target="_blank">Software</A> page!) to numerically solve our two-step sequential enzymatic reaction with the LSODA method. This program can actually be used to help researchers easily model reaction rates involving any two-step sequential enzymatic reactions, where a different enzyme catalyzes each step. Researchers only have to input initial enzyme and substrate concentrations and the enzymatic activity in order to create an easy-to-visualize plot of substrate concentrations vs. time. Researchers can follow the documented directions on our wiki to change or add variables specific to their enzymatic reactions, in order to get the graphs of substrate concentrations over time they wish to model. We licensed our program with the Open Source MIT License. 
</H6><P class="dote col-lg-offset-3" id="CalculateLink">.</P></DIV><DIV class="row"><H3 class="col-lg-12 col-lg-offset-3 paddingremovaly indent">Calculating Required ALDH2 Levels</H3></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">Through the program, we graphed acetaldehyde concentrations in the mouth over time after consuming different alcoholic beverages. There are three different ALDH2 genotypes (homozygous wild type ALDH2*1/*1; heterozygous ALDH2*1/*2; homozygous mutant ALDH2*2/*2), which process acetaldehyde at different rates (different V<SUB>max</SUB> values). Based on literature, we know that the heterozygous functions at 20-40% efficiency compared to homozygous wild type, while the homozygous mutant functions at 1-4% efficiency. The graphs below reflect the different efficiencies (20% and 1% were used to assume the worst case scenarios for the two mutant genotypes) (Gross <I>et al</I>., 2014; Chen <I>et al</I>., 2014).</H6></DIV><DIV class="row col-lg-offset-3 image_container20 col-lg-3"><H5 class="subtitle4"><B>Figure 4-7. Acetaldehyde concentration over time for three different types of alcohol (beer, wine, or spirits).</B> The three curves in each graph represent acetaldehyde concentration over time for the different ALDH2 genotypes. Labels for each curve are provided in the legend.</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">As seen in Figure 4-7, homozygous wild-type ALDH2*1/*1 metabolizes acetaldehyde the fastest, while the homozygous mutant ALDH2*2/*2 metabolizes acetaldehyde the slowest. <B>Our goal is to help homozygous mutant individuals metabolize acetaldehyde at the same rate as wild type individuals.</B> We included the enzymatic activity for our purified ALDH2*1, which, when subtracted from Equation 5, should return acetaldehyde concentrations to wild type levels. This enzymatic activity can be adjusted by changing the enzyme concentration so that the acetaldehyde concentration matches levels found in wild type individuals.</H6></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Equation 6 models the rate of change in acetaldehyde concentration over time after additional ALDH2*1 enzymes are supplied to a homozygous mutant individual.</B> The example enzymatic activity presented in the equation is the activity of 0.222 µM ALDH2*1 (from our purified enzyme functional test). The activity can be adjusted by adjusting the enzyme concentration.</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">Because ethanol concentration is in great excess during the first few hours after consumption, we can assume that the rate of change in ethanol concentration is essentially constant (Jones <I>et al</I>., 2011). Thus, we assume the rate of acetaldehyde production from the conversion of ethanol is constant for all three types of alcohol beverage (Figure 4-7). As a result, the rate of change in acetaldehyde concentration is also the same across the three types of alcohol beverage. </H6><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">Then, we plotted graphs to investigate the impact of different concentrations of additional ALDH2*1 on acetaldehyde concentrations for different genotypes (Figure 4-8).</H6></DIV><DIV class="row col-lg-offset-3 image_container20 col-lg-3 imagebox"><H5 class="subtitle4"><B>Figure 4-8. Acetaldehyde concentration over time after supplying various concentrations of ALDH2*1 enzymes.</B> Here, concentrations are shown after consumption of spirits. Changing the concentration of additional ALDH2*1 changes the acetaldehyde levels in the mouth. An additional 0.103 µM ALDH2*1 reduces acetaldehyde levels in homozygous mutants to match wild type levels. For other types of alcohol (not shown here, but can be found in Modeling <A href="https://2018.igem.org/Team:TAS_Taipei/Notebook" target="_blank">Lab Notebook</A>), the same concentration of 0.103 µM ALDH2*1 also adjusts acetaldehyde levels in homozygous mutants to match those found in wild type people.</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly" id="DetermineLink">As seen in Figure 4-8, the concentration of ALDH2*1 (and therefore the enzymatic activity of ALDH2*1), can be adjusted so that the mutant genotype acetaldehyde levels can match wild type levels. For example, we discovered that 0.103 µM ALDH2*1 is the approximate concentration needed to cover the difference in reaction rates between people who are homozygous mutant and wild type every minute (Figure 4-8).</H6></DIV><DIV class="row"><H3 class="col-lg-12 col-lg-offset-3 paddingremovaly indent" id="">Determining the Number of ALDH2 Molecules and Bacterial Cells Needed in the Candy</H3></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">In order to provide a quantifiable amount of ALDH2*1 for manufacturers to produce, we converted the concentration of ALDH2*1 to the number of molecules needed for production. For example, using the hypothetical spirits example above, by multiplying 0.103 µM of ALDH2*1 enzyme with the average volume of saliva in a person’s mouth, which is 1.19 mL, we found that 1.226*10<SUP>-10</SUP> mol ALDH2*1 are required.  

 </H6><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">We previously determined that one candy should completely dissolve in the mouth in about 15 minutes (“Product Testing” experiment in <A href="https://2018.igem.org/Team:TAS_Taipei/Applied_Design" target="_blank">Prototype</A>), so we multiplied the moles of ALDH2*1 needed in one minute by 15 to ensure that, on average, 1.226*10<SUP>-10</SUP> mol of ALDH2*1 are released every minute that our candy is used. So, in this example,  1.839*10<SUP>-9</SUP> mol (1.226*10<SUP>-10</SUP> x 15 min) of purified ALDH2*1 enzymes would be packaged into one candy. Here, we made the assumption that one candy’s volume changes (dissolves) in a linear constant rate fashion. 
</H6><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">Furthermore, to determine the number of <I>Ec</I>N bacteria cells (probiotic) needed to produce the specified moles of ALDH2*1 above, we used values from the <I>Ec</I>N bacteria functional test (0.314 nM of ALDH2*1 is produced by 4.79*10<SUP>8</SUP> cells in 16 hours) to calculate the moles of ALDH2*1 enzymes produced by each cell. We then calculated the number of bacteria cells needed to produce 1.839*10<SUP>-9</SUP> moles of enzymes in 16 hours. The manufacturers would need 2.81*10<SUP>9</SUP> bacteria cells in one candy. From literature, we figured that one <I>E. coli</I> bacterium cell weighs about 0.95*10<SUP>-12</SUP> g. Hence, one candy would contain about 2.67 mg of bacteria cells (Glazyrina <I>et al.</I>, 2010). 
 </H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description indent paddingremovaly">Each candy allows an ALDH2 deficient patient to match normal wild type ALDH2 activity. <B>For consumers, the safest way to prevent the increased cancer risks is to keep a piece of candy in the mouth when they are drinking.</B> For those who are unwilling to eat the candy while drinking, they can still process acetaldehyde at wild type rates if they consume the candy <I>after</I> drinking; however, they would be exposed to the high acetaldehyde levels that were present before the candy was eaten. It is possible that the bacteria may also persist in the mouth after the candy has been consumed. This would provide a protective effect, allowing the candy to be used before drinking, but further experiments would need to be conducted to determine whether the probiotic can remain in the mouth.<P class="dote col-lg-offset-3" id="RefLink">.</P></H6></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly">REFERENCES</H1></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">
Chen CH, Ferreira JCB, Gross ER, Mochly-Rosen D. (2014). Targeting Aldehyde Dehydrogenase 2: New Therapeutic Opportunities. Physiological Reviews. 94(1):1-34. 

Edenberg HJ. (2007). The Genetics of Alcohol Metabolism: Role of Alcohol Dehydrogenase and Aldehyde Dehydrogenase Variants. Alcohol Research &amp; Health, 30(1), 5–13.

Eng MY, Luczak SE, Wall TL. (2007). <I>ALDH2, ADH1B,</I> and <I>ADH1C</I> Genotypes in Asians: A Literature Review. <I>Alcohol Research &amp; Health, 30</I>(1), 22–27.

Glazyrina J, Materne EM, Dreher T, Storm D, Junne S, Adams T, Greller G, Neubauer P. (2010). High cell density cultivation and recombinant protein production with <I>Escherichia coli</I> in a rocking-motion-type bioreactor. Microb Cell Fact. 9: 42. Doi: 10.1186/1475-2859-9-42. p.4 left column 5th paragraph.

Jones AW. Pharmacokinetics of Ethanol - Issues of Forensic Importance. (2011). Forensic Sci Rev. 23(2):91-136.

Lachenmeier DW, Monakhova YB. (2011). Short-term salivary acetaldehyde increase due to direct exposure to alcoholic beverages as an additional cancer risk factor beyond ethanol metabolism. J Exp Clin Cancer Res.; 30(1): 3.

Yokoyama A, Kamada Y, Imazeki H, Hayashi E, Murata S, Kinoshita K, Yokoyama T, Kitagawa Y. (2016). Effects of ADH1B and ALDH2 Genetic Polymorphisms on Alcohol Elimination Rates and Salivary Acetaldehyde Levels in Intoxicated Japanese Alcoholic Men. Alcohol Clin Exp Res. 40(6):1241-50.</H6></DIV></DIV></DIV></DIV></BODY></HTML>