<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_TAS_Taipei_Basic_Part skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:TAS Taipei/Basic Part</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>TAS_Taipei</TITLE></P><DIV class="navbar"><A class="pro" href="https://2018.igem.org/Team:TAS_Taipei/Description">Project</A><A class="exp" href="https://2018.igem.org/Team:TAS_Taipei/Experiments">Experiments</A><A class="prot" href="https://2018.igem.org/Team:TAS_Taipei/Applied_Design">Prototype</A><A class="mod" href="https://2018.igem.org/Team:TAS_Taipei/Model">Modeling</A><A class="hp" href="https://2018.igem.org/Team:TAS_Taipei/Human_Practices">Human Practices</A><A class="safe" href="https://2018.igem.org/Team:TAS_Taipei/Safety">Safety</A><A class="as" href="https://2018.igem.org/Team:TAS_Taipei/Team">About Us</A><A class="att" href="https://2018.igem.org/Team:TAS_Taipei/Attributions">Attributions</A></DIV><DIV class="row"><H1 class="col-lg-12 col-lg-offset-3 paddingremovaly" id="aboutourschool">BEST NEW BASIC PART</H1></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">
Our best new basic part is <A href="http://parts.igem.org/Part:BBa_K2539150" target="_blank">BBa_K2539150</A>: the protein-coding gene ALDH2*1. This gene codes for human mitochondrial acetaldehyde dehydrogenase (ALDH2), ALDH2*1 is the wild type form, which is responsible for oxidizing acetaldehyde, a toxic intermediate of ethanol metabolism, into harmless acetic acid (Larson <I>et al.</I>, 2005; Farrés <I>et al.</I>, 1994). We were able to test and validate ALDH2*1’s ability to metabolize acetaldehyde. The design and characterization of BBa_K2539150 are documented below.
   </H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 bigthannormal paddingremovaly"><B>Part Design</B></H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">We modified the ALDH2*1 sequence (NCBI: NM_001204889.1) to remove an internal PstI site.</H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 bigthannormal paddingremovaly"><B>PCR Check Results</B></H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">The part was confirmed by PCR using the primers VF2 and VR, as well as sequencing by Tri-I Biotech.</H6></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>BBa_K2539150 contains just the sequence of ALDH2*1.</B> PCR check using VF2 and VR primers produced a band at the expected size of 1.8 kb (yellow box).(Cloning &amp; Figures: Catherine C)</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 bigthannormal paddingremovaly"><B>Characterization</B></H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">We tested the enzyme function of ALDH2*1 (BBa_K2539150) by placing it under the control of a constitutive promoter (BBa_K2539100). We compared ALDH2*1 (the wild type) activity to ALDH2*2 (the mutant) activity. ALDH2*1 should convert acetaldehyde to acetate faster than the mutant form.</H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly"><B>Testing enzyme function in cell lysates:</B></H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">When ALDH2 converts acetaldehyde into acetate, NADH is produced. To test the ability of recombinant ALDH2*1 to metabolize acetaldehyde, we used reagents from a kit (Megazyme, K-ACHYD) to quantify the amount of NADH produced by taking absorbance readings at 340 nm. This wavelength is highly absorbed by the reduced form, NADH, but not the oxidized form, NAD+ (Harimech <I>et al.</I>, 2015; McComb <I>et al.</I>, 1976). High absorbance values would indicate more conversion of acetaldehyde into acetate. </H6><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>The wild type ALDH2*1 converts acetaldehyde at a faster rate than mutant ALDH2*2. (A)</B> The conversion of acetaldehyde to acetate by ALDH2 uses NAD+ and produces NADH. <B>(B)</B> Experimental setup. The supernatant from ALDH2-expressing <I>E. coli</I> cell lysates was mixed with acetaldehyde and NAD+ to initiate the reaction at 25°C. NADH concentration was measured by taking absorbance readings at 340 nm. <B>(C)</B> Relative activity of lysates containing either ALDH2*1, ALDH2*2, or inactive ALDH2*1 (boiled to denature proteins; negative control). Error bars represent standard error. (Experiment &amp; Figure: Justin W, Yvonne W, Catherine C)</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">Over a 40 minute period, <I>E. coli</I> carrying the ALDH2*1-expressing construct (BBa_K2539100) produced more NADH than the mutant form (BBa_K2539200), while the negative control (boiled BBa_K2539100) did not change significantly. This shows that wild type ALDH2*1 is more efficient at metabolizing acetaldehyde compared to the mutant ALDH2*2.</H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly"><B>Testing enzyme function of purified samples:</B></H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">To purify ALDH2*1 and ALDH2*2 enzymes, we added HIS-tags to the DNA designs (BBa_K2539101 and BBa_K2539201). We repeated our functional test using purified enzymes (results shown below), and saw a much clearer difference between ALDH2*1 (BBa_K2539150) and ALDH2*2 (BBa_K2539250), compared to when we used bacterial cell extracts. In our tests, purified ALDH2*2 did not have any effect on NADH, while purified ALDH2*1 significantly increased NADH levels.</H6></DIV><DIV class="row col-lg-offset-3 image_container col-lg-3 imagebox"><H5 class="subtitle"><B>Purified ALDH2*1 has a much higher activity level compared to purified ALDH2*2.</B> Using the same procedure, we tested the enzymatic activity of purified ALDH2*1 and ALDH2*2 at 25°C. A negative control containing only elution buffer (from the protein purification process) was also included (gray). ALDH2*1 steadily metabolized more acetaldehyde compared to both ALDH2*2 and the negative control, both of which did not seem to have any effect. The error bars represent standard error, and are bolded for ALDH2*1 and ALDH2*2 to highlight the difference between the two groups. (Experiment &amp; Figure: Justin W)</H5></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 bigthannormal paddingremovaly"><B>References</B></H6></DIV><DIV class="row"><H6 class="col-lg-8 col-lg-offset-3 description paddingremovaly">
Farrés J, Wang X, Takahashi K, Cunningham SJ, Wang TT, Weiner H. (1994). Effects of changing glutamate 487 to lysine in rat and human liver mitochondrial aldehyde dehydrogenase. A model to study human (Oriental type) class 2 aldehyde dehydrogenase. J Biol Chem. 13;269(19):13854-60. 

Harimech PK, Hartmann R, Rejman R, del Pino P, Rivera-Gila P, Parak WJ. (2015). Encapsulated enzymes with integrated fluorescence-control of enzymatic activity. J. Mater. Chem. B. 3, 2801-2807. 

Larson HN, Weiner H, Hurley TD. (2005). Disruption of the Coenzyme Binding Site and Dimer Interface Revealed in the Crystal Structure of Mitochondrial Aldehyde Dehydrogenase “Asian” Variant. J Biol Chem. 280(34):30550-6. 

McComb RB, Bond LW, Burnett RW, Keech RC, Bowers, GN Jr. (1976). Determination of the molar absorptivity of NADH. Clin Chem. 22(2): 141–150. 

   </H6></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>