"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn State/Daily Notebook
From 2014.igem.org
Bidirectional Promoters Overview
WELCOME TO PENN STATE iGEM 2014!
Click here  to edit this page!
HOME
JUDGING FORM
OFFICIAL PROFILE
TEAM
PROJECTS
PARTS
WETLAB
SAFETY
HUMAN PRACTICES
ATTRIBUTIONS
Penn State iGEM 2014 Notebook Page
Below is our detailed, day-to-day Laboratory Notebook.
IMPORTANT LINKS:
Click HERE to return to the weekly summaries
Protocols
Laboratory Notebook
Biodetoxification
Codon Optimization
Tuesday, May 20, 2014
First iGEM meeting with Dr. Richard and Dr. Salis.
First iGEM meeting with Dr. Richard and Dr. Salis.
Wednesday, May 21, 2014
Emily's first experience with cloning! Ashlee led Emily through several practice experiments from designs made earlier in the year: making a gel, loading samples, gel purifying DNA. Ashlee introduced Emily to her research the previous semester. She had inserted the HMF-ABCDE pathway on the pSEVA251 KanR plasmid into P. putida and had validated its function, but was having difficulty inserting the dCas9 system into the pSEVA251 KanR HMF-ABCDE plasmid. The plasmid size would be close to 18 kb and this, among other affects, was thought to increase the difficulty of the cloning. Another hypothesis was that the dCas9 pathway did not have a strong terminator for the trans-acting RNA. Ashlee and Emily began work to insert the terminator. They performed a ligation of the pSEVA251/HMF plasmid and the terminator.
Clay and Sam worked on a program in Excel to codon optimize GFPs. Sucess. Unfortunately, program is clunky and requires a lot of user input for any optimization. Decision made to attempt the same task in MATLAB.
Thursday, May 22, 2014
Ashlee and Emily finished the ligation, transformed the ligation into DH10B electrocompetent Escherichia coli cells using electroporation, incubated at 37 degrees C for 1 hour, and plated on Kanamycin plates.
RBS design begun. Library calculator run using 34 N's as a constraint in the "Constraints" field. Nothing used in pre sequence field. First 60 bp of original superfolder GFP used as "coding sequence". Clay started working on MATLAB program for codon optimization.
Friday, May 23, 2014
The transformed colonies took over 24 hours to grow - something unusual for this strain but something Ashlee had observed since after she inserted the HMF pathway and attempted to insert dCas9. We picked six colonies for overnight growth to do more cloning tomorrow.
Design of GFPs continues as Clay works on program to optimize genes. Question asked: which GFP should be optimized? GFP mut3b and superfolder GFP both present advantages and disadvantages. Met with Chiam Yu to discuss effects of codon optimization on translation. Acquired data from previous codon optimization project, which will serve as the basis for our fast/slow codon optimization.
Saturday, May 24, 2014
Memorial Day Weekend? How about lab cloning weekend! Only one of the 5 terminator/HMF/pSEVA251 ligation colonies grew. Ashlee conducted plasmid preparation of the terminator/HMF/pSEVA251 vector and digested this (the backbone) and the dCas9 system (the insert) with restriction enzymes AatII and AflII.
More RBS library calculations run. Problem: since TIR is dependent on the first 60 bp of a coding sequence and our variant GFPs will differ in this region, how can we ensure that an accurate library is developed?
Sunday, May 25, 2014
The 5 colonies still did not grow. Ashlee prepared a gel and ran the dCas9 and backbone digestions and gel purified them. Colonies had not been growing well for Ashlee in the latter half of the semester as the plasmid size increased above 12 kb, so a new strategy would have to be pursued.
Back to Top
Monday, May 26, 2014
Due to a string of failed clonings with the broadhost vector pSEVA251 and the large inserts, new designs are evaluated! Instead of creating a plasmid with the HMF pathway (7.5 kb) and dCas9 system (5.5 kb), we shall add the HMF pathway and dCas9 to the P. putida genome using homologous recombination.
Decision reached to optimize Superfolder GFP based on its superior post translational modification, ensuring that translation elongation remains the rate limiting step. Papers on codon optimization downloaded to Mendeley Desktop. New hypothesis for existence of rare codons developed: Perhaps they function as a molecular "brake" to slow down translation and prevent "ribosome traffic jams."
Tuesday, May 27, 2014
Inoculated LB broth with ampicillin and dCas9 plasmid from cryogenic storage; inoculated Lb broth with chloramphenicol and FTV vector from Ashlee's past experiment; streaked the HMF vector on a kanamycin plate.
Decided to use homogeneous "leader sequence" upstream of each variant GFP to ensure Ribosome Binding Site (RBS)library creates accurate range of translation initiation rate (TIR)
Wednesday, May 28, 2014
Emily and Ashlee made cryogenic storage of the dCas9 plasmid; plasmid prepared the FTV and dCas9 vectors; digested FTV vector; inoculated LB broth with a colony from the HMF plate for overnight growth and plasmid preparation tomorrow.
Optimized leader sequence from previous project for our use. Also explored options of creating novel synthetic leader as well as using first 60 base pairs of GFP mut3b. Met with Dr. Salis, decided to use a new synthetic leader. Created leader based on several design considerations. Ran RBS library calculator using a "pre sequence" of 20 bp upstream of RBS.
Thursday, May 29, 2014
Prepared plasmid containing the HMF pathway; inoculated LB broth with Lambda Red Recombinase plasmid from cryogenic storage. Ashlee, Emily, and graduate student Iman Farasat ordered primers for three plasmids that will be constructed via Gibson Chew-Back and Annealing Assembly, two of which will be inserted into the genome by homologous recombination.
Diagrammed "Cloning Strategy" for the project, including all steps from receiving synthetic DNA to characterization. This is a work in progress! Chose pFTV as vector, decided to order variant GFPs as gblocks through IDT.
Friday, May 30, 2014
Ashlee and Emily made cryogenic storage of the Lambda Red Recombinase plasmid.
Created a "Plan B" for cloning that details fallback plans and options that we will pursue if cloning is not successful. Simplified Outline: Ligate one GFP into pFTV, ligate in the dRBS. Transform cells, plasmid prep and sequence to determine which RBS was taken by each. Swap out GFPs, then sequence again to ensure that variant GFPs were successfully introduced.
Back to Top
Monday, June 2, 2014
Constructed dCas9 gene cassette and plasmid backbone with replication origin ColE1 via PCR Rescue. Gel purified dCas9 and ColE1 cassettes. 1 out of 4 dCas9 PCR's were successful, and 2 out of 4 colE1's were successful, all of which were Ashlee's. We attributed this to Emily's lack of cloning experience.
Sam prepared electrocompetent cells for use later on, and began process of primer design. Clay designed the synthetic leader sequence and finalized the program in MATLAB that optimizes GFPs at codon level.Met with Dr. Salis and decided to also optimize a GFP for slow insertion time, based on a model created by Iman Farasat.
Tuesday, June 3, 2014
Conducted Colony PCR using P. putida KT2440 strain as DNA template to construct two ~1 kb genome overlaps. Plasmid prepared the Lambda Red Recombinase plasmid, DH10B-PKD46, FTV-ptac-LacI-CmR plasmid, and NoHP_15A_Plmra_CmR plasmid containing RFP with a strong, unique promoter.  Stock of NoHP_15A_Pkmra_CmR and FTV_ptac_LacI_CmR for cryogenic storage was also made. Lambda Red Recombinase cassette was amplified using PCR Rescue and gel purified.
All constructs (variant GFPs in vector pFTV checked for enzyme restriction sites, enzymes picked to be used in the cloning process. gblocks designed using format: junk DNA- restriction site- CDS- restriction site- junk DNA. Sam designed rescue primers to be used for amplifying the gblocks. They will be expensive and we don't want to leave any chance of running out of stock once we have them. Primers for rescue PCR redesigned when it was realized that Clay accidentally truncated the GFPs by incorrectly copying the coding sequence of original superfolder GFP from its Ape file.
Wednesday, June 4, 2014
We made ampicillin agar plates and ampicillin antibiotic stock for cloning. The PCR Rescue of Lambda Red Recombinase was also gel purified.
Sam made Chloramphenicol plates for use later on.Redesigned leader sequence to be a full 60 bp, redesigned rescue primers again. Sequencing primers designed. MATLAB program updated to optimize for slow insertion time GFP. Script also created to total the insertion times of each GFP. Primers updated again as enzymes were re chosen, due to the presence of one of them in the CDS of slow insertion time GFP.
Thursday, June 5, 2014
We conducted PCR Rescue to amplify the kanamycin resistance cassette (specifically the neomycin cassette, which also confers resistance to kanamycin) from pSEVA251 KanR plasmid. Two different sets of primers for kanamycin were tested, and the first set was successful - all 4 PCR's were correct. The second set of primers all failed. However, Emily had her first PCR success! Kanamycin cassette was gel purified.
Project Plan updated. Five GFPs will be ligated into pFTV separately, then dRBS will be ligated in. Data will be collected and sequencing will show which RBS was used by each colony. Cryogenic stock of cells harboring pFTV grown.
Friday, June 6, 2014
Conducted colony PCR using P. putida KT2440 strain as the DNA template to construct 1 kb overlaps for homologous recombination. All four of the first genome overlaps with gene PP_0747 were successful; only 2 overlaps with upp gene were successful. These were gel purified. We learned Gibson Chew-Back Annealing Assembly (CBA) protocol.
Gblocks arrived. Rescue PCR conducted to increase stocks of gblocks. Samples run in gel and purified. Plasmids harvested from cells harboring pFTV. Clay left early to rebuild the deck at his house. Bastard.
Sunday, June 8, 2014
One 4-part, two 3-part, and two 2-part Gibson CBA's were conducted to assemble the kanamycin resistance cassette, two genome overlaps, and the colE1 replication origin. This completed plasmid will be termed "plasmid 1".
Inverse PCR conducted on pFTV backbone. Shows very faint bands in gel, decided to increase number of cycles from 30 to 35.
Back to Top
Monday, June 9, 2014
The two 2-part and two 3-part CBA's were amplified using PCR Rescue and gel purified. The original 4-part CBA was transformed into E. coli electrocompetent cells using electroporation and plated on kanamycin antibiotic agar plates. The 4-part CBA was repeated to ensure accuracy. Because the CBA parts contained no plasmid DNA, the 4-part CBA could be digested by restriction enzyme Dpn1. Dpn1 binds and cuts methylated DNA sites, thus destroying any plasmid DNA remaining as a contaminant.
More issues with RBS library design, as it seems very difficult to find sequences with sufficiently high TIR. Decided to use an initial condition for the calculator, which should speed it up and also ensure higher TIR is reached. More calculations ran.
Tuesday, June 10, 2014
The original 4-part CBA worked! Many colonies appeared on the plate after incubation at 37 degrees C for 18 hours, and 12 colonies were selected for plasmid preparation. These were digested with AatII and XbaI, two restriction sites that are only both contained in the final assembled 4-part plasmid. 6/12 colonies showed the correct bands on the gel. We also prepared more 1 kb ladder from concentrate.
gblocks digested to ready them for ligation into pFTV. Not enough stock of pFTV was present, so inverse PCR ran again, this time with 3 tubes.
Wednesday, June 11, 2014
3 successful colonies were sent for sequencing. In order to insert the dCas9 system into plasmid 1, dCas9 was digested with XhoI and ClaI. 4 successful colonies were digested with ClaI for 3 hours, heat inactivated at 65 degrees C, and then digested with SalI-HF restriction enzyme. SalI and XhoI are compatible sites. These digestions were gel purified, resulting in low concentrations of plasmid DNA. Only two colonies were used to continue further. We met with Leah Bug and Matthew Johnson from the Penn State Center for Science and the Schools .
pFTV digested. Not enough Cla1 enzyme was present, so this step will be suspect if there are issues in the future. More Cla1 ordered. Inverse PCR ran again, this time with added extension time (2:30 instead of 1:30). Inverse PCR shows very faint bands again and gel not excised.
Thursday, June 12, 2014
The plasmid backbone was digested with phosphotase enzyme. These backbones were ligated to dCas9 over 18 hours at 16 degrees C to ensure maximum ligation product.
Considerable time spent working with RBS library calculator. Calculations ran to determine max TIR for the leader sequence that was designed as well as with the enzyme restriction site (Pst1) downstream of the RBS.
Friday, June 13, 2014
Emily purified the ligation product.
Calculations from yesterday yield very high TIR. Still no good libraries, though. Libraries ran using new initial conditions.
Sunday June 15, 2014
Ashlee transformed the ligation into E. coli DH10B electrocompetent cells via electroporation, and plated them on kanamycin antibiotic agar plates to grow overnight.
gblocks and pFTV ligated together. Cells transformed.
Back to Top
Monday, June 16, 2014
The ligation failed. We amplified more of the dCas9 system using PCR Rescue, in which 2 out of 4 PCR's were successful - both Emily's!. We are evaluating this difficult PCR and will be altering the annealing temperature. PCR Rescue for dCas9 were repeated at 58 degrees C and 62 degrees C annealing temperature. We inoculated LB broth with ampicillin resistance and dCas9 from cryogenic stock. We also conducted colony PCR of the second set of genome overlaps. We received our sequencing results and two out of three colonies have the correct sequence.
Overnight cultures of transformed cells streaked on plates.
Tuesday, June 17, 2014
PCR Rescue RFP cassette and gel purified - all 4 RFP PCR's worked. Gel purified dCas9 Rescue from yesterday. PCR Rescue colE1 origin and chloramphenicol resistance cassette to construct plasmid 2, which will contain ColE1, CmR, RFP, HMF pathway, and two P. putida genome overlaps. We plasmid prepared new dCas9 to use as a template for PCR. New and old dCas9 templates were used for PCR Rescue of the third genome overlap. See the schematic here for more information.
Plates show only two colonies, one each from "slow" GFP and "slow insertion time" GFP. These colonies sampled, grown in cultures.
Wednesday, June 18, 2014
Digested the HMF pathway with EcoRI-HF and PstI-HF restriction enzymes. All PCR's of the third genome overlap containing dCas9 failed, and we realized we must complete the first plasmid by inserting dCas9 and use that as a template instead of the original dCas9 plasmid. This points the failure of the ligation to either the dCas9 PCR's or the ligase buffer has expired. We made new aliquots of fresh ligase buffer to test whether this was the case. We have run out of ClaI and cannot digest dCas9 until this arrives. Our strategy now is to Gibson assembly the Lambda Red Recombinase system and dCas9, then PCR Rescue and ligate into plasmid 1 using XhoI/SalI-HF and XbaI to mitigate the lack of ClaI.
Plasmid prep on cultures from colonies that grew after transformation. New hypothesis develope: perhaps the higher expression GFPs killed the cells due to their extremely rapid translation elongation, leaving only the less efficiently translated GFP carrying cells to live. Online check using website Genscript and their free gene analysis tool shows that the common GFP scores a perfect 1.0 and the rare GFP a perfect 0.0 on their scale (from 0 being not optimized at all to 1 being perfectly optimized). This reveals that the algorithm used by Genscript corresponds directly to the codon usage profile for E.coli over the entire genome.
Thursday, June 19, 2014
Conducted 2-part Gibson CBA to assemble the Lambda Red Recombinase system and the dCas9 system.
Many more RBS library calculations run, some using a method where the Shine Dalgarno (SD) sequence is mutated, but not the rest of the initial condition. Hopefully this will speed up the calculations and finally yield some high TIR libraries that evenly span the range we are looking for. Another overnight culture of slow GFP/slow insertion time GFP cells innoculated.
Friday, June 20, 2014
PCR Rescue to amplify Lambda Red Recombinase and dCas9 cassette and gel purified. Gel bands reflect failed Gibson CBA.
Plasmid harvest on culture from yesterday. Plasmids digested using Xho1 and Xmal1, expecting to see two bands, one at 1.1 kb and one at 1.7 kb. Ran in gel. Gel displays the correct bands. This shows that our construct is basically correct and gives us confidence to send for sequencing to confirm.
Back to Top
Monday, June 23, 2014
Repeat Gibson CBA of Lambda Red Recombinase system and dCas9 cassette using 25 femtomole DNA and 50 femtomole DNA. Emily and Ashlee worked on the presentation to the teachers for the Center for Science and the Schools .
Plasmids prepared and sent for sequencing to Quintara Bio.
Tuesday, June 24, 2014
Ashlee and Emily PCR Rescued PAK-HMF and ran the results on a gel for later purification.  Continued work on the presentation and a demonstration to the teachers for the Center for Science and the Schools .
Inverse PCR attempted again, in case we will need to go back to it. Five tubes run, this time with added cycles and extension time. Still, bands are very faint. Need to find a way to get this reaction to be successful.
Wednesday, June 25, 2014
Emily PCR Rescued the FTV vector using crRNA primers.  The PCR results were gel purified.  More work was done to update the Wiki and brainstorm design ideas for it.  Emily also did some finishing touches on the presentation for the Center for Science and the Schools . Ashlee updated the Notebook and Team information on the website. She is slowly learning HTML and CSS.
Sequencing Data arrived, showing that the vectors did not pick up an insert at all, and re-circularized without one. Seems as if inverse PCR was successful. Tail sequence of forward primer shown perfectly, tail sequence of reverse primer shown partially by sequencing. Plan is to re-do the digestion of gblocks and pFTV and then ligate, transform again. Inverse PCR completed again, this time shows extremely faint bands. We know that this was successful at least once because of the sequencing data, but we haven't been able to replicate it. Suspicion that the origional plasmid prep of pFTV may have been faulty due to the low concentration that was gathered makes us want to repeat this process. t
Thursday, June 26, 2014
Ashlee and Emily met with Dr. Richard to discuss research using his biomass hydrolyzer and using an Aspen or HYSYS model of a biorefinery plant to evaluate the economic benefits of increasing the threshold of furfural/HMF toxin bacteria can tolerate by inserting the HMF-ABCDE catabolism pathway. We will begin manipulating the Aspen Plus model and Economic analysis from the National Renewable Energy Laboratory (NREL) .
Plan for continuing codon optimization project is supported by the advisers. Clay updated notebook online as well as the document diagramming the design process of the project.More RBS calculations run. Problem with RBS library calculator at this point is that using no initial condition (only N's in the  constraints field, followed by Pst1) the calculator does not mutate the Pst1 site that needs to be part of the dRBS, but yields very low TIR. Using a strong initial condition, followed by Pst1, the calculator yields acceptable TIR but mutates Pst1. Inserting Pst1 into these calculations after the fact, and then using the "evaluate" function of the calculator shows that TIR drops precipitously. Calculations were run in single RBS forward engineering mode to determine max TIR that can be found using only N's and Pst1. Culture of pFTV harboring cells inoculated.
Friday, June 27, 2014
Ashlee performed a transformation of Plasmid 1 into DH10B Escherichia coli electrocompetent cells via electroporation and plated this on Kanamycin plates. This was done to regenerate our Plasmid 1 so we can digest it and insert the dCas9 and Lambda Red Recombinase systems. She updated the Notebook and Project pages, and is constructing figures for her design process. She is having trouble with "div" functions in HTML and is consequently frustrated. We met with Dr. Salis and Dr. Richard for our biweekly iGEM meeting and Ashlee presented the work on the website, presentation and Emily's demonstration idea for the Center for Science and the Schools, and current progress on the project.
Culture of pFTV harboring cells plasmid harvested, much better concentration of pFTV measured than before. Inverse PCR conducted again, this time using optimized protocol and 3 tubes. Lab meeting. Sam did gel extraction, purification. Seems like strong bands were observed, and at the correct places. This is a big success.
Saturday, June 28, 2014
The transformation did not grow. We must pick new colonies and sequence confirm they have the right Gibson CBA parts in order to resupply our stock of Plasmid 1. This will be used as the backbone for the insertion of dCas9 and Lambda Red Recombinase.
Sunday, June 29, 2014
Picked 10 colonies for overnight growth from the second 4-part CBAR of Plasmid 1, which was digested with Dpn1 to ensure no contamination by original plasmid DNA.
Sam digested gblocks and pFTV again using Cla1 and Pst1, purified, then ligated them together, purified again, then transformed cells and plated them.
Back to Top
Monday, June 30, 2014
Plasmid prepared 10 colonies. Digested them with AatII and XbaI, which are restriction sites only contained in the final Plasmid 1. These 10 digests were run on a gel, of which 4 showed the correct bands at approximately 1 kb and 3 kb.
No colonies observed on plates from Sunday. Attempted to use ligation product from Sunday to transform another batch of cells, because of sudden fear that our electrocompetent cells may not be good. Not enough ligated product remains to attempt this.
Tuesday, July 1, 2014
Ashlee and Emily prepared tubes of primers and four purified digestion products to send for sequencing.  They continued their work on the website and tried to find more html help for certain aspects.
Goal of the day is to transform some more cells, and use different ratio of insert to backbone during ligation in hopes of getting colonies to grow. Attempted to re-ligate digested GFPs and pFTV but not enough remains from Sunday to attempt this. Calculations ran for re-doing digestions and ligation. Digestions completed. RBS calculations from 6/26 analyzed, found that even using no initial condition (only N's followed by Pst1), sufficiently high TIR can be discovered. Need to use calculator algorithm on either Dr. Salis or Iman's computer so that a dRBS can be constructed using an initial condition that is followed by a non-mutable Pst1 site. Digested gblocks and pFTV ligated and cells transformed, then plated on antibiotic selective plates.
Wednesday, July 2, 2014
Ashlee worked remotely on the website and is awaiting sequencing results. Plasmids with all four junctions correct (ColE1, KanR, Overlap 1, and Overlap 2) will be used to re-attempt the insertion of dCas9.
Colonies observed on plates with G1 and G4 transformed cells. More plates made, more centrifuge tubes autoclaved. Rescue PCR ran to increase stocks of gblocks. Unfortunately, very faint bands are noticed. Decision to lower annealing temperature next time to increase output. Salis lab manual updated with protocol for making antibiotic plates. Six colonies from each plate of cells that grew used to inoculate a culture.
Thursday, July 3, 2014
Ashlee updated the Notebook and Project pages. She added shortcuts to the Notebook page to make it more user-friendly. She is also constructing more figures to display the design process and "Plasmid 1", "Plasmid 2", and "Plasmid 3" constructs.
Realized mistake of adding six individual colonies to the same culture. Never, ever do that again. That was really stupid. Inoculated six individual cultures with one colony each, as it should have been done the first time.Prepared more electrocompetent cells. Plasmid prepped the 12 tubes (six each of GFP1 and GFP4, the cells that grew from transformation on 7/1. Streaked plates with media from the 12 cultures.
Friday, July 4, 2014
'MERICA. Ashlee began to update the Safety page.
Clay exercised his right to do synthetic biology by digesting the plasmids prepped from 7/3 using two enzymes, then running them in a gel to determine if the expected bands were present. One of the six samples had the right bands.
Saturday, July 5, 2014
Ashlee updated the project page and worked through a CSS tutorial to better organize all the pages. She is still working to improve the Biodetoxification figures.
Worked on updating the lab notebook and website.
Sunday, July 6, 2014
Sam started another rescue PCR, using an annealing temp lowered by 3 degrees, to increase stocks of gblocks.Clay and Sam met to discuss the state of the project.
Back to Top
Monday, July 7, 2014
Emily worked on the presentation for the Center for Science and the Schools; Ashlee and Emily collaborated on the demonstration for the teachers. Ashlee and Emily digested Plasmid 1 in order to insert the dCas9 system. We first digested the backbone for ~4 hours with ClaI restriction enzyme, heat inactivated the enzyme at 65 degrees C for 20 minutes, cleaned the digestion, then digested for ~4 hours with SalI-HF. We have to do this because of the high fidelity binding of SalI and the fact that the two restriction sites are very close together. HF enzymes could remain bound tightly and block ClaI from binding.
Cleaned up PCR from Sunday. Concentration of DNA was very high (>1000 ng/ul. Sent plasmids that displayed correct band on 7/4 for sequencing with Quintara. Prepared cryo stock of cells harboring these plasmids.
Tuesday, July 8, 2014
Ashlee and Emily worked on finalizing the presentation for the Center for Science and the Schools. We also ran the digest of plasmid 1 on a gel and PCR Rescued the dCas9 system. Many dCas9 PCRs were not very successful, and we did more troubleshooting. We increased the elongation time to 3 minutes (~30 seconds per kb of DNA plus an extra 30 seconds - dCas9 is 5.5 kb) and we set the annealing temperature to 58 degrees Celsius. We tested changing the annealing temperature previously, and 58 degrees C had the most success. We ran our 8 PCR reactions on a gel and found that 7 of them worked! We have solved the problem of poor dCas9 PCRs! Ashlee also updated the website logo and several other spots around the website.
Designed primers to be used to introduce 40 bp homology with pFTV backbone into gblocks, in case sequencing results came back negative. In this event, cbar could be used to introduce GFPs into pFTV.
Wednesday, July 9, 2014
Ashlee and Emily continued to work on the presentation and the demonstration for the Center for Science and the Schools and we met with Matt Johnson to discuss our presentation. He had some good suggestions for us and we began to restructure the presentation. We also digested dCas9 with XhoI and ClaI restriction enzymes and ligated it into two of our Plasmid 1 backbones. This is an overnight ligation, so we will transform it tomorrow. We also picked new Plasmid 1 colonies to plasmid prep and try to improve our yield after the two serial digestions with ClaI and SalI-HF and gel extraction. We made cryogenic stock of Plasmid 1 constructs #1 and #9, two of our best sequenced plasmids.
Sequencing results arrive. There is an insert in the backbone, but it is not the one that was expected. Expected GFP4, received GFP2. This must be due to a contamination or mislabeling. Just the thought of this gave Clay the heeby jeebies, which were so bad that God knows he might still have them. Because only two of the four primers annealed in the sequencing reaction (the two that anneal into the backbone), it was decided to retry the reaction using the other two primers (that anneal in the CDS) and send for sequencing again. This was attempted, but the sequencing center on campus responded that the plasmid (template) DNA concentration was too low. Six cultures inoculated with cells on plate streaked on 7/1 to be plasmid prepped tomorrow.
Thursday, July 10, 2014
Ashlee and Emily plasmid prepped 6 Plasmid 1 constructs - 3 from #1 and 3 from #9. We changed the plasmid preparation protocol to a 30 minute centrifuge spin after the cell had been lysed and digested with RNase because we were not seeing clean and compact cell pellets after 20 minutes. 30 minutes improved the pellets significantly and we got higher DNA concentrations. We then digested these with ClaI for 7 hours (instead of the previous 4), heat inactivated for 25 minutes, then PCR cleaned the digest. We then digested with SalI-HF overnight for 7 hours. We hope that this additional digest time coupled with increased centrifuge time during plasmid preparation would result in greater amounts of digested backbone. We also performed two transformations of the dCas9-Plasmid 1 ligations.
Plasmid prep of six tubes inoculated yesterday. Samples re-sent for sequencing on campus with higher DNA concentration. (76.5 ng/ul today vs 24.1 ng/ul yesterday). Several improvements made to plasmid prep protocol. Unfortunately, sequencing center returns message kindly explaining that concentration of DNA was still insufficient. Three cultures inoculated for plasmid prep tomorrow. Safety form updated. Lab notebook and weekly update section updated. Plan is tentatively to send for sequencing tomorrow morning by 9 after doing some tricks in plasmid prep to increase DNA concentration. Then, if sequencing results confirm presence of successfully ligated G2, scale up plasmid stock, digest out G2, replace in four ligations with the other GFPs, then transform cells and plate. After this, plasmisd prep and send for sequencing to confirm presence of all five GFPs. If sequencing fails to confirm, investigate other methods of introducing GFP into pFTV (cbar). Inoculated three cultures of pFTV+G2 to plasmid prep tomorrow.
Friday, July 11, 2014
One of our dCas9 ligations, with backbone Plasmid 1 #1 had colonies grow on the plate. The ligation to backbone #2 was not successful. #2 did not appear to sequence well, so perhaps that plasmid is not 100% correct. Emily made a large gel and we ran the digested backbones on it. All 6 plasmid digests ran well.
Plasmid prepped three cultures from yesterday and sent for sequencing on campus, this time with high enough plasmid concentration. Prepared SOC media. Tested electrocompetent cells by transforming one tube with original pFTV and the other with nothing. Plated on Cm plates. Realized problems with Reverse inverse PCR primer. The annealing section is correct but the tail is complementary to what it should be. This is resulting in the correct sequence containing Sac1, a spacer, and Pst1 in the bottom strand instead of in the strand that we want it to be in. Met to talk about the design of the dRBS. It was decided that we would order two oligos with the ends as if the restriction enzyme has already cut them. The plan is to anneal these oligos and we can then skip the digestion step and just ligated them directly into the digested cells. Fingers crossed that this will work. Dr. Salis thinks it should be okay, but Iman has doubts because annealing the two strands (each is a dRBS) is really 48 ligations proceeding simultaneously in the same tube and is bound to have low efficiency. Perhaps we can linearly increase our transformation output by transforming two or three tubes at a time.
Saturday, July 12, 2014
Ashlee picked 12 colonies from the dCas9-Plasmid 1 ligation for overnight.
Designed new Reverse inverse PCR primer and put it into the cart to order.
Sunday, July 13, 2014
Ashlee plasmid prepped the 12 dCas9-Plasmid 1 colonies and digested them with AclI and EcoRI-HF. These two restriction sites are only present in the dCas9 system, and so only successful ligations will result in two bands on a gel. Any self-ligations should result with only one band on a gel.
Back to Top
Monday, July 14, 2014
Ashlee ran the dCas9-Plasmid 1 digestions on a gel to confirm that dCas9 was inserted. We were expecting bands around 8 kb and 1 kb, but instead all 12 colonies showed bands around 8 kb and 3 kb. This is very odd, and we are hoping that there is some other restriction site that is not reflected on our plasmid map or the backbone is larger than we originally thought, as plasmid maps are often inaccurate. We talked with Dr. Salis and he advised us to proceed and digest the plasmid with the restriction enzymes we will use to excise the cassette containing dCas9, run an analytical gel to determine whether our cassette is the right size, and then attempt homologous recombination in P. putida. We digested the plasmid with AvrII and PacI. We will send 7 colonies for sequencing tomorrow. Ashlee and Emily edited the presentation for Center for Science and the Schools.
Investigated method of using PCR assembly to create a strand of ds DNA containing the dRBS. This method requires a long top strand and a primer, which would be used in PCR to create the ds DNA. Problem is that the top strand is longer than 50 bp and very expensive. Designs are completed, but tentative decision reached to stick with the annealed oligos method unless it proves to be difficult.
Tuesday, July 15, 2014
We sent 7 colonies for sequencing to determine whether the dCas9 system is present in Plasmid 1. We ran the digestions from Monday on an analytical gel and got two bands: <1 kb and ~ 5 kb. We were expecting bands of <1 kb and ~9 kb, so we are wondering if there is some contamination. We had our biweekly meeting with Dr. Salis and Dr. Richard and discussed the meaning of these conflicting bands.
Rescue PCR was done again to make sure that we had enough gBlock since we have been using them to re-do failed reactions. Updated website attributions page. Project page created for codon optimization project. Inverse PCR on pFTV accomplished using new reverse inverse PCR primer. Inverse PCR and Rescue PCR products ran in gel, excised and purified. Digestions with Pst1 and Cla1 ran overnight.
Wednesday, July 16, 2014
We PCR Rescued dCas9 from two Plasmid 1-dCas9 ligation colonies to see if dCas9 is present in our vector, and if it is, how much of it is present. We ran the PCR product on a gel and observed that...  Emily and Ashlee continued to finalize the presentation and created the activity we will demonstrate to the teachers.
DNA purification of digestion products, ligation, DNA purification of ligation product, transformation of cells using all 5 GFPs. Clay left town to have sinus surgery.
Thursday, July 17, 2014
Sequencing results were supposed to arrive but had to be re-run. Emily and Ashlee finished the demo and updated the Prezi. Iman ordered new dCas9 primers, truncating them to remove the CBAR overlaps. Hopefully if this was the problem we will have more success.
Sam inoculated thirty (30) cultures, six from each GFP variant, so that they could be miniprepped the next day.
Friday, July 18, 2014
Ashlee finalized the Prezi. Sequencing results are still delayed.
All of the cultures grew. Each culture was streaked on a plate to save it for further analysis if needed. Three rounds of miniprep were done in order to ensure that the lysis reaction did not occur for longer than 2-3 minutes per sample.
Saturday, July 19, 2014
Sequencing results arrived for dCas9-Plasmid 1. Ashlee and Emily rehearsed their presentation and practiced the demo and activity.
All of the plasmids were nanodropped for concentration. All were in the range of 100-300 ng per µL. Since this was acceptable concentration and after a long evening of miniprepping and camp, Sam decided to treat himself to Chipotle.
Sunday, July 20, 2014
All of the samples were digested with Xma1 and Sac 1 to see if they would show the correct bands before the samples were sent for sequencing. Out of the thirty samples, 2 from each GFP showed the correct or almost correct band. These were samples 1.4, 1.6, 2.1, 2.3, 3.1, 3.3, 4.1, 4.4, 5.3, and 5.4. This will be the samples that would be sent out this upcoming week.
Back to Top
Monday, July 21, 2014
We received new PCR primers for the dCas9 system and PCR Rescued them. We sent new Plasmid 1 colonies #1, #6, and #9 for sequencing and digested them with ClaI and SalI-HF for 10 hours (5 hour digestion with ClaI, 65 degrees C heat inactivation of ClaI, 5 hour digestion with SalI-HF). Ashlee ran the PCR products on a gel and 3/4 were successful. She digested dCas9 PCR with ClaI and XhoI restriction enzymes. She digested the backbones with phosphotase so they would not self-ligate, and ligated in the dCas9 system (16 hour ligation). Ashlee also used the new backbones to CBAR the old dCas9 PCR product, just to test whether our old dCas9 system was correct.
Tuesday, July 22, 2014
Ashlee performed DNA clean-up on the 2 CBA plasmids and transformed them into DH10B cells. She plated them on Kan plates and let them grow overnight at 37 degrees C. She also cleaned the three ligations, transformed them into DH10B cells, and plated on Kan plates and let them grow overnight at 37 degrees C.
Samples sent for sequencing that displayed correct bands after digestion 7/20. All samples sent using F1 and R2 primers. These primers anneal in the backbone and are thus non specific to any GFP. Transformed cells with original pFTV to create a cryo stock.
Wednesday, July 23, 2014
Ashlee and Emily presented to the teachers for the Center for the Science and the Schools early in the morning.  The presentation was an hour long and was great practice for the Giant Jamboree.  Videos and pictures can be found on the Human Practices page.  Only ligation products with the backbone #6 grew - sequencing results arrived and luckily #6 was the only correct plasmid. #1 and #9 both had point mutations. We picked 8 ligation colonies to be plasmid prepared and sent for sequencing tomorrow. #6 and #9 CBA colonies grew and we picked 3 colonies from #6, as it was the only correct backbone.
Made Cm plates.Created cryo stock of original pFTV cells.
Thursday, July 24, 2014
8 ligation colonies were plasmid prepared and sent for sequencing, along with three CBA colonies from plasmid #6. Met with Dr. Richard to discuss project updates, practice jamborees, and future deadlines for iGEM.
iGEM meeting to discuss all of the updates for our projects and to talk about potential practice jamborees with other schools. Observed colonies on plate with cells transformed using original pFTV. Brainstormed ideas for Science U presentation, Sam began developing a "Central Dogma Relay Race". Annealed oligos containing the dRBS. Inoculated cultures with cells from each streak of cells shown to display correct bands, in anticipation that sequencing results will show at least some of them are correct.
Friday, July 25, 2014
Ashlee and Emily left on vacations.
Cleaned up annealed oligos that cooled overnight. Analyzed sequencing results from Tuesday. G1.4 has no insert. G1.6 has common GFP, possibly with a point mutation. G2.1 has a definite mutation. G2.3 appears correct, has common GFP, but should sequence one more time. G3.1 and G3.3 have fast GFP, should be sequenced again. G4.1 has some garbled insert, G4.4 also has common GFP. G5.3 has no insert. G5.4 has slow insertion time GFP (correct. Should be sequenced again). All reactions with R2 primer failed. This primer anneals in the terminator and probably has too much secondary structure. It will be moved downstream and re-ordered. Plasmid prep on G2.3 and G3.3 yields good concentration. Rescue PCR on G1,3,5, appears successful. G1,3,5 digested along with pFTV overnight.
Saturday, July 26, 2014
Spent time today working on all of the necessary materials for the ScienceU presentation that will be on Tuesday of next week. Start to put together different activities that would introduce the high schoolers to genetic engineering, our project, and current ethical issues in synthetic biology. Cleanup of digestion product from yesterday, ligation, purification, transformation of cells with G1,3,5. 30 Minute incubation, plating.
Sunday, July 27, 2014
Worked a majority of the day on making a PowerPoint and developing content and examples that kids would understand without a heavy science background. Lots of CM 50 plates were made so that we could demonstrate antibiotic resistance in our presentation and also because we needed some more.
Back to Top
Monday, July 28, 2014
Sequencing results arrive - only one colony from the ligation, arbitrarily termed "L6" had the dCas9 system. The rest of the ligation and CBA plasmids did not appear to have dCAs9 inserted. Ashlee transformed L6 into DH10B cells and plated for overnight growth on Kan at 37 degreees C.
The "Central Dogma Relay" was put together and cut out so that we could use it as a way to check and see if the kids were able to understand transcription and translation. PENN STATE and SCIENCE were used as examples that the kids will need to transcribe so that they can keep their cell "alive". Transformation from yesterday yields very few colonies, cells re-transformed using yesterday's ligation product, plated. Samples G2.3 and G3.3 sent for sequencing using primers common R1 and F2 and fast F2 and R1 (these primers all anneal in the CDS). Four colonies picked from re-transformation of G5, used to inoculate cultures. Culture inoculated from streak of G5.3 cells (plated 7/18). Six colonies picked from G1 re-transformation. Redesigned R2 Sequencing primer outside the terminator. Rescue PCR done on G4, failed first time (yielded weird smears). Rescue PCR redone, this time appears to work. Gel purification, digested overnight. Plasmid prep of cultures inoculated earlier.
Tuesday, July 29, 2014
Ashlee videotaped Clay and Sam's presentation to the Science-U highschoolers.
Worked on presentation and put finishing touches on the slides. Prepared activities and set up room in 11 Life Sciences. The presentation went extremely well. We had a well behaved group of kids and it was nice to be able to see some of the faces that I (Sam) have been working with all summer. They really enjoyed it and we got free t-shirts for our time. I think we may have inspired a few people to become future scientists! Purification of yesterday's digestion product, ligation, transformation, plating.
Wednesday, July 30, 2014
Ashlee picked 2 colonies from the L6 plate for overnight growth to make cryogenic storage and to plasmid prepare. We are going to save the original L6 plasmid as long as possible because it was sequence confirmed. Emily digested the Lambda Red Recombinase system with XbaI and ClaI restriction enzymes.
No colonies observed again, so the transformation was redone using the initial ligated G4 (7/16) and the one that was prepared previously this week. Sequencing results from Monday confirm G2.3 and G3.3 as correct.
Thursday, July 31, 2014
Ashlee and Emily plasmid prepped one of the previously picked colonies.  This was then digested with XbaI and ClaI so the Lambda Red Recombinase system could be inserted.  The other picked colony was used to make cryogenic stock for potential use later.  A Rescue PCR was done on overlap region 3 and ran on a gel for purification. All 4 PCR reactions failed.
There were several colonies on the plate that were transformed with the old G4 ligation, but only 2 on the plate that had the new G4 ligation. Two (2) colonies from each plate were chosen and inoculated so that could be mini-prepped tomorrow. Cultures inoculated with streaks from G2.3 and G3.3 for preparing cryo stocks. Four cultures inoculated with G5.4 (from 7/18 plating) to be plasmid prepped tomorrow. Digestion of G2.3,G3.3 started with Sac1. Double digest not possible because 12 nucleotide spacer isn't sufficient to use bulky and tightly binding HF enzymes. Solution is to do one digestion, purify, then digest again with the other enzyme.
Friday, August 1, 2014
Ashlee sent the L6 prepared plasmid for sequencing, just to check whether the latter part of dCas9 was inserted. If the complete system was not inserted, we would not be able to use this plasmid as a template to create the overlap 3 region. We ran the digestion of the L6 backbone on a gel and it was the correct size, 9.5 kb. We also made Kan 30 plates and made new DNA ladder aliquot. Ashlee ligated the Lambda Red Recombinase system into the L6 backbone for an overnight, 18 hour ligation.
Cultures from yesterday were plasmid prepped: 4 cultures of G5.4 (7/18 plating), 2 cultures from new G4 (7/29 ligation product), 2 cultures from old G4 (7/18 plating), one culture G2.3 and one culture G3.3 (both 7/18 plating). All plasmid prep successful. Analytical digestion of G5.4 (7/18 plating), G4.1 and G4.2 (7/30 plating), G4.1 and G4.2 (7/18 plating), G5.3 (from 7/18), G1.1,1.2,1.3,1.4,1.5,1.6 (from 7/27 plating), G5.4 (7/18 plating). Analytical digestion is shorter (~3 hours with HF enzymes) and will be used to check for correct bands. G2.3, G3.3 digested with longer digestion protocol to prepare for dRBS insertion. This isn’t possible with a double digest because of the short spacer between the two sites, so a sequential approach is used. Then, G2.3 and G3.3 ligated with the annealed dRBS oligos. DNA purified, then cells transformed.
Saturday, August 2, 2014
Ashlee performed a gradient PCR for the overlap containing the dCas9 system. Because it failed at 60 degree C annealing temperature, we want to test a range of annealing temperatures from 56 degrees C to 66 degrees C. In order to do a CBA of this overlap and the HMF system in Plasmid 2, we needed a long (90 bp) reverse primer, which could be causing our PCR issues. She transformed the ligation of Lambda Red Recombinase and the L6-dCas9 plasmid and plated it on Kan plates to grow overnight at 37 degrees C.
No colonies observed on plates from yesterday’s transformation. Retransformed with less ligation product.
Sunday, August 3, 2014
Colonies from yesterday’s transformation yield 2-3 colonies each. Digestion products (after the second sequential digestion) of G2.3 and G3.3 re-ligated, purified, hten transformed again. Observed ~6 colonies per plate by the late evening.
Back to Top
Monday, August 4, 2014
Ashlee and Emily made new Kanamycin plates and preformed a PCR Rescue of Overlap 3.  This PCR failed as it seems that the Reverse Primer is too large for an efficient PCR reaction.  New primers were designed.
Clay out for sinus checkup.
Tuesday, August 5, 2014
The L6 plasmid containing dCAS9 was plasmid prepared and digested to excise bits of the genome.
Redid sequential digests, using longer time (6 hrs each). Oligos re-annealed. Rescue PCR performed on all 5 gblocks, gel run and all bands are in correct position as well as very bright. Gel purified and final DNA concentration is between 37 and 52 ng/ul. Digestion products of plasmid with G2.3 and G3.3 purified, then ligated to annealed oligos containing dRBS. DNA purified, concentration not measured. Cells transformed.
Wednesday, August 6, 2014
The digest from yesterday was cleaned to excise dCas9.  We also picked PAK and FTV backbones from cryo storage for overnight growth.
Cultures inoculated with glowing colonies on plates in fridge for cryo stock preparation. Inoculated cultures with G5.41 (7/18), G2.3 (7/18), G3.3 (7/18), G1.3 (7/28). Re-annealed oligos again, using improvements to the protocol suggested by Amin.
Thursday, August 7, 2014
We plasmid prepared the PAK backbone for the HMF pathway as well as the FTV backbone for the crRNA system.
Sent for sequencing the results of plasmid prep from overnight cultures. Transformed more cells with G4+pFTV plasmid.
Friday, August 8, 2014
Inoculated cultures with colonies that frew on G4 plates that Sam made from re-digestion/transformation yesterday (5 total). Also streaked plate with these. Began to re-clone G4 by digesting inverse PCR product and G4 with Cla1 and pst1.
Saturday, August 9, 2014
Cleaned up digestions, ligated, purified DNA transformed cells. Forgot to plasmid prep cultures from yesterday so re-inoculated them for prep tomorrow. Re-ligated G2.3 and G3.3 with the dRBS using improved protocol suggested by Amin as well as dRBS annealed oligos that were created using the improved protocol for annealing. Transformed cells.
Sunday, August 10, 2014
Plasmid prepped cultures from yesterday. Observed lots of colonies on plate with G3.3 + dRBS, at least 20 on plate G2.3  and at least 100 on plate G4 from yesterday.
Back to Top
Monday, August 11, 2014
Plasmid prepped cultures (5 from plate G4 transformed 8/9) and ran in analytical digestion along with five from plate G4 transformed 8/7. Ran in gel and it seems that all have a GFP variant, although it cannot be said for certain that the correct one is present.
Tuesday, August 12, 2014
A PCR Rescue was preformed fr both the FTV and PAK backbones.  These were gel purified as all reactions worked.
Using DNA from 8/10 plasmid prep, all sent for sequencing.
Wednesday, August 13, 2014
Concentrations for the PAK PCR Rescue were low, so this reaction was redone and gel purified to yield a better DNA concentration.  We also performed a PCR reaction of Overlap 3 with new primers.  Next, we did a Gibson CBA of PAK and FTV backbones.  The product was purified via a PCR clean method.
TECAN run with G3.3 (confirmed earlier).  Set up very sensitive PCR to attempt to rescue G4 gblock from original tube.
Thursday, August 14, 2014
Ran products of PCR in gel, strong band in correct location recorded, gel purified. TECAN run finished. Overnight culture plate placed into cryo storage. EC cells prepared, media prepared, plates prepared.  Cryo stocks prepared and old plates thrown out. Clay leaves for Spain in three days, returns home.
Friday, August 15, 2014
Emily and Ashlee were out of lab due to moving into new apartments.
Saturday, August 16, 2014
Ashlee tested new P. putida cells by plating them on different antibiotic resistant plates.  No colonies were grown which was a good indication of the electrocompetence of the cells.
Sunday, August 17, 2014
Many of the PCR reactions and CBA reactions were redone to ensure their quality.  The CBA of the PAK backbone and crRNA was digested with Dpn1 and provided many colonies for us to work with.
Back to Top
Monday, August 18, 2014
A colony PCR reaction of overlap 4 was done and was gel purified.  Seven PAK-crRNA CBA colonies were picked and grown in liquid media with Kanamycin.  Ashlee picked RFP from Amin's cryogenic storage and grew it over night.
Tuesday, August 19, 2014
Emily plasmid prepared the PAK-crRNA plasmids, two different RFP-FTV plasmids and the HMF plasmid.  The RFP-FTV segments were rescue PCR reacted and gel purified.  One set of samples that should produce the RFP segment did not work while the other samples producing the FTV segment did and were further purified.  The HMF plasmid was digested with EcoR1 and Pst1 overnight.
Wednesday, August 20, 2014
We sent the seven plasmid prepared samples of PAK-crRNA to Quintara Bio for sequencing.  The HMF digest was run on a gel.
Thursday, August 21, 2014
Sam returned to school. Move in took almost all day.
Friday, August 22, 2014
Ashlee took the GRE.
Learned about project updates. Discussed what needed to be done moving forward. Innoculated media with G1 and G5.
Saturday, August 23, 2014
Took 14 colonies from the TECAN plate that had different fluorescence readings and innoculated them. G1 and G5 from yesterday were plasmid prepped.
Sunday, August 24, 2014
Of the selected TECAN cells from yesterday were plasmid prepped.
Back to Top
Monday, August 25, 2014
Happy first day of school, Penn State!  The HMF plasmid was digested again with EcoR1 and PstI.
Plasmid prepped cells were sent for sequencing. Hoping to get results before the end of the weeks so they can start to be characterized.
Tuesday, August 26, 2014
The HMF digest was run on a gel and purified yielding only a small amount of DNA.  This will be redone in the coming days.
G4 was digested.
Wednesday, August 27, 2014
Four lambda red-dCas9-L6 digestion reactions were run.  We digested the plasmid with AvrII and PacI.  These were PCR cleaned, instead of gel purified, and were then transformed into P. putida competent cells.
G4 was attempted to be ligated into the backbone. Cells were transformed and plated with the G4 ligation.
Thursday, August 28, 2014
The same digestion was repeated today as we did not get the DNA concentrations we anticipated.  This made it hard to transform, so we will do that again tomorrow.
At this point, G4 has not been successful transformed into the backbone. It has been decided to stop pursuing this GFP variant because it is being too problematic.
Friday, August 29, 2014
The digestion was PCR cleaned again using an additional wash cycle.  This still did not yield better DNA concentrations and we were unable to move forward with the transformation.  Some thought into why this was happening was brought up, but no conclusion was reached and we decided we would try again.
Sam left for retreat with the Homecoming Executive Committee
Saturday, August 30, 2014
Sunday, August 31, 2014
Sam got back from retreat!
Back to Top
Monday, September 1, 2014
Ashlee and Emily started taking some information and pictures from the Prezi that was used in the NEWBio presentation and adding them into our new presentation for both projects.
Happy Labor Day!
Tuesday, September 2, 2014
We continued adding information to the presentation and preparing what we would discuss with the other teams that would be present at Carnegie Mellon in a few short days.
Wednesday, September 3, 2014
Started to prepare slides for the presentation at CMU.
Thursday, September 4, 2014
Met with Ashlee and Emily to discuss more of what we will be presenting at CMU.
Friday, September 5, 2014
The team read over the slides made and came up with a good presentation for the Carnegie Mellon trip tomorrow.
Saturday, September 6, 2014
Traveled to Carnegie Mellon University for a practice Jamboree and Workshop
Sunday, September 7, 2014
Our lab is currently  undergoing a large move to a new building.  In the next few weeks not much wet lab work could be done because we had limited access to incubated shakers and other lab materials that are always necessary.
We gained a lot of positive and constructive feedback from the practice Jamboree. Hopefully we can work on having a more cohesive presentation for the giant jamboree!
Back to Top
Monday, September 8, 2014
Lab Move!
Sam was in charge of planning the Color Run for Penn State. He was in meetings the entire week with different university officials.
Tuesday, September 9, 2014
Wednesday, September 10, 2014
Thursday, September 11, 2014
Friday, September 12, 2014
Saturday, September 13, 2014
Sunday, September 14, 2014
The Color Run Penn State!
Back to Top
Monday, September 15, 2014
Lab Move!
For the next few weeks, Sam's primary focus was on Homecoming because he had major responsibilities to the University.
Tuesday, September 16, 2014
Wednesday, September 17, 2014
Thursday, September 18, 2014
Friday, September 19, 2014
Saturday, September 20, 2014
Media, plates, and tips were autoclaved to prepare for the upcoming weeks.
Sunday, September 21, 2014
New P. putida cells were picked from cryogenic stock and tested on varying Kanamycin resistances.  We have been struggling to get transformed cells to grow and these tests could help us find a better way to grow them on in the presence of Kanamycin.
G1 and G5 were digested using Sac1 HF and Pst1 HF
Back to Top
Monday, September 22, 2014
Homecoming week; Sam was out of lab.
Tuesday, September 23, 2014
Homecoming week; Sam was out of lab
Wednesday, September 24, 2014
Homecoming week; Sam was out of lab
Thursday, September 25, 2014
More P. putida cells were picked for overnight growth.
Homecoming week; Sam was out of lab
Friday, September 26, 2014
While Sam was busy working on Homecoming events, Ashlee took over some of his work with the G blocks.
Homecoming week; Sam was out of lab
Saturday, September 27, 2014
Homecoming week; Sam was out of lab
Sunday, September 28, 2014
Homecoming week; Sam was out of lab
Back to Top
Monday, September 29, 2014
Ashlee assisted Sam in digestions
dRBS 1 and 2 were digested using Sac1 HF and Pst1 HF.
Tuesday, September 30, 2014
G1, G3, and G5 were transformed because supply of plasmids was running low.
Wednesday, October 1, 2014
Thursday, October 2, 2014
Ashlee and Emily met with Dr. Salis to discuss some parts of the project that are really struggling.  It was decided to use a slightly different media, LB Lennox, due to its low salt content, in hopes that our transformed cells would grow better. We also discussed new protocol for using the Lambda Red Recombinase.
Media and CM 50 plates were prepared for the upcoming weeks.
Friday, October 3, 2014
Ashlee and Emily made LB Lennox liquid media and poured LB Lennox-Kan 7.5 plates.  A transformation of Plasmid 1 into E. coli was done before Ashlee had to leave for the weekend.  Emily picked lambda red cells from the cryogenic storage and plated them on an Ampicillin plate to grow overnight.
Saturday, October 4, 2014
Emily saw a good amount of growth on the transformation plates and the lambda red plate.  10 colonies were picked from the transformation for overnight growth and two colonies were picked from the lambda red plate.
Sunday, October 5, 2014
Emily plasmid prepared all 12 colonies that were picked the day before.  She also spent some time working on the Wiki.
Back to Top
Monday, October 6, 2014
Ashlee assisted Sam.
G1, G3, and G5 was transformed and plated so that we could start working with a fresh culture. The cultures were then used to innoculate media.
Tuesday, October 7, 2014
G1, G3, and G5 were miniprepped and started to be digested with Sac 1 HF
Wednesday, October 8, 2014
dRBS 1 and 2 were cleaned as well as G1, G3, and G5.
Thursday, October 9, 2014
Friday, October 10, 2014
The whole team was gone for the weekend.
Saturday, October 11, 2014
Sunday, October 12, 2014
Back to Top
Monday, October 13, 2014
We have been focusing on updating the wiki and making it look a lot better before the wiki freeze at the end of the week. Emily spent a lot of time editing the notebook, weekly summaries, and the human practices page. Ashlee digested G1 and G3 for Sam.
Tuesday, October 14, 2014
Ashlee digested G1 and G3 for Sam.
Wednesday, October 15, 2014
G1, G2, were ligated with dRBS 1 and 2 while G3 was ligated with dRBS 2. These plasmids were then used to transform cells and they were plated. Here's to hoping that there will be colonies tomorrow!
Thursday, October 16, 2014
Friday, October 17, 2014
We all worked to finish the Wiki! Yay!
We all worked to finish the Wiki! Yay!
Projected work after the Wiki Freeze
Focus on codon optimization.
Get data for common, rare, slow GFP's.
Retrieved from " http://2014.igem.org/Team:Penn_State/Daily_Notebook "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
