<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_Synbio"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn/Synbio">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn/Synbio&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/Synbio&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn/Synbio&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn/Synbio" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn/Synbio</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><UL id="nav"><LI><A href="https://2014.igem.org/Team:Penn">Home</A></LI><LI><A style="hover:" href="https://2014.igem.org/Team:Penn/Overview">Project</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Overview">Overview</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Magnetism">Magnetism</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Microbio">Microbiology</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Synbio">SynBio in AMB-1</A></LI><LI><A href="https://2014.igem.org/Team:Penn/CdTolerance">Cadmium Tolerance</A></LI></UL><LI><A href="https://2014.igem.org/Team:Penn/Specsheet">Human Practices</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Specsheet">Spec Sheet</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Outreach">Outreach</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Biomeme">Biomeme</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Notebook">Notebook</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Notebook">Timeline</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Safety">Safety</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Protocol">Protocols</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Supplement">Supplementary Materials</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Team">About</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Team">Team</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Achievements">BioBricks</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Sponsors">Sponsors</A></LI><LI><A href="https://2014.igem.org/Team:Penn/Resources">References</A></LI></P><TITLE>University of Pennsylvania iGEM</TITLE><DIV id="redbox"><DIV id="textbox"><H3 style="text-align: left"><A name="Overview">Overview</A></H3><P style="text-align: left; text-indent:0px">After we had gleaned enough information about the microbiology of AMB-1, we began focus more on its applicability in synthetic biology. We began to explore the literature for transformation protocols. However, in our search, we found that very little transformation data or clear procedures existed for this strain of bacteria. During the course of our work with AMB-1, we consolidated various protocols and developed methods of our own in order to form a clear, optimized transformation protocol for future research teams to use. 
</P><P style="text-align: left;text-indent:0px"> Upon developing this “master-protocol”, we attempted transformation of the AMB-1 strain with the <B>PYMB essentials BioBrick.</B></P><P style="text-align: left;text-indent:0px"> This vector contains an AMB-1 origin of replication, a replication gene and an AMB-1 specific promoter (pmsp1), on a PSB1C3 backbone. The backbone contains chloramphenicol resistance. If we were to successfully transform the vector into AMB-1 and grow it on chloramphenicol coated plates, this would indicate that the AMB-1 had received the shuttle vector and that the transformation was successful.</P><P style="text-align: left;text-indent:0px"> Eventually, after attempting to transform AMB-1, we found that our transformations with the shuttle vector PYMB essentials had key components missing from the sequence. <B>We designed a new shuttle vector to transform AMB-1. We are introducing this plasmid to the iGEM community with the hope that it spurs future exploration of AMB-1 and that it inspires other teams to work with uncharacterized strains.</B></P><P style="text-align: left;text-indent:0px"> As we worked to transform AMB-1, we developed a more optimized transformation protocol. The protocol we developed is outlined here: </P><A href="https://static.igem.org/mediawiki/2014/7/77/Penn_iGEM_Transformation_Protocol.pdf" style="font-size: 15px; margin:0 auto; color:#fff"> Penn iGEM Transformation Protocol </A><P style="text-align: left;text-indent:0px"> Our additions/findings to the original protocols are: </P><OL><LI>We outlined a protocol for the preparation of an aerobically grown cell culture of AMB-1 from an anaerobic culture. This is the first step of transformation. <A href="https://static.igem.org/mediawiki/2014/8/8c/Inoculation.pdf" style="font-size: 15px; margin:0 auto; color:#fff"> Innoculation of Aerobic Cultures </A></LI><LI>We determined the optimal electroporation settings and buffer for transformation.</LI><LI>The growth of bacteria on plates after recovery using the procedure outlined in the protocols found in literature are reported to take 2 weeks. We found that this technique can be optimized if one of two plating techniques outlined below are used to reduce oxygen exposure:</LI><OL type="A"><LI>	Plating the colonies on 1% agar directly, and then wrapping the plates in parafilm to reduce produces colonies in 20-24hours. </LI><LI>	Plating cells and growing the cultures in an anaerobic chamber produces colonies in 10-12 hours. </LI></OL></OL><H3 style="text-align: left; font-size: 18px;"><A name="PYMB">PYMB Transformation Issues </A></H3><P style="text-align: left; text-indent:0px">As we proceeded in our attempts to transform AMB-1 with PYMB essentials, we found that despite our scrupulous adherence to the protocol, our transformation was unsuccessful.</P><P style="text-align: left; text-indent:0px">We attempted to design an experiment that would address why AMB-1 was not growing on plates with antibiotic. We plated the bacteria at every step of transformation in order to address the success of each step. (Table 3A)</P><H3 style="text-align: left; font-size: 18px;"><A name="PYMB"> Figure 3A </A></H3><DIV id="figureBox" style="margin-left: auto; margin-right:auto; width: 700px; text-align:center;"><P style="color:#fff"> The above figure is a picture of the plates of transformed AMB-1 with and without DNA. The transformed cells containing the plasmid do not survive on plates with chloramphenicol which suggests that transformation was unsuccessful. </P></DIV><P>From this experiment, we were able to determine that AMB-1 cells are viable are viable in aerobic cultures, after suspension in TES buffer, after electroporation, and after recovery. Therefore, in our next steps in troubleshooting transformation, we attempted to address the following two possibilities for why it was not successful.</P><P style="text-align: left; text-indent:0px"><B> 1) Plasmid Sequence Problems </B></P><P style="text-align: left; text-indent:0px">We obtained the PYMB shuttle vector from a previous BioBrick that attempted to only retain key components of an original shuttle vector in an effort to reduce its length. The plasmid was constructed by an estimation of the length of the replication origin. Also, other parts of the plasmid expect for the replication gene were omitted. The original plasmid included pUC19 ligated on BamHI site with pMGT, an endogenous plasmid found in Magnetospirillum magneticum MGT-1. However, the BioBrick only retained the putative essentials of this large vector. <SUP>[5]</SUP></P><P style="text-align: left; text-indent:0px">Due to this, we proposed that one reason for the failure of transformation is that this BioBrick is missing key components essential to its function that have been eliminated from the original pMGT sequence. It was possible that an essential part of the origin of replication was removed.</P><P style="text-align: left; text-indent:0px">Another concern is that the chloramphenicol resistance was part of the PSBIC3 backbone and did not have an AMB-1 specific promoter driving its expression.</P><P style="text-align: left; text-indent:0px"><B>2)  Error in the electroporation technique </B></P><P>We also wanted to ensure that the error was not in the electroporation itself, before we started to explore the possibility of a faulty plasmid. We designed an experiment to eliminate the possibility that a faulty electroporation machine or a flawed protocol was impacting our results. </P><H3 style="text-align: left; font-size: 18px;"><A name="Redesign">Troubleshooting the Shuttle Vector in E.coli</A></H3><P style="text-align: left; text-indent:0px">In order to eliminate any doubt about electroporation, we chose to use the same protocol (substituting SOC broth for TES buffer) to transform E.Coli with the same plasmid. This is possible because the PYMB Essentials BioBrick is on the PSBIC3 backbone which has the E.Coli origin of replication and replication gene. We plated the E.Coli at every step of transformation keeping as many variables constant as possible. (Table 2.2)</P><P style="text-align: left; text-indent:0px">The results of this experiment indicated to us that E.Coli cells are viable in aerobic cultures, after suspension in TES buffer, after electroporation, and after recovery. The electroporation was successful and the results imply that:</P><OL><LI> The device is effective</LI><LI> E. Coli rep gene is functional in PYMB</LI><LI> Chloramphenicol resistance is expressible in E. Coli</LI><LI> The protocol is feasible</LI></OL><P style="text-align: left; text-indent:0px">The proposed next step after this experiment was:</P><OL><LI>Redesign a shuttle vector BioBrick because the problem is most likely that the PYMB essentials plasmid is defective.</LI></OL><P style="text-align: left; text-indent:0px">From this experiment, we were able to conclude that the problem with transformation was most likely due to a faulty shuttle vector. </P><H3>Supporting literature</H3><P>We also investigated the paper that isolated a native plasmid in Magnetospirillum magneticum MGT-1 and created a shuttle vector of AMB-1. This paper concluded that the shortest region required for replication in AMB-1 was more than 2 kB and included the AMB-1 origin of replication and rep gene. This region also included a long sequence of nucleotides that was essential for replication for the replication of the plasmid. In addition to this, the exact sequence of the origin of replication has not been isolated. The PYMB essentials biobrick only contains 80 nucleotides of what could be the origin of replication. This gives us more reason to believe that this part is not functional. <SUP>[5]</SUP></P><H3>Redesign is necessary because the plasmid is not available</H3><P>The same paper described the construction a shuttle vector called pUMG. This includes the entire sequence of a native plasmid (pMGT) found in a closely related strain to AMB-1, called Magnetospirillum magneticum MGT-1. pUMG was able to successfully replicate in AMB-1. <SUP>[5]</SUP> Unfortunately, this shuttle vector is not available to any research teams in North America and has not been used extensively outside of Japan and no shareable AMB-1 shuttle vectors like pUMG exist on AddGene or any other plasmid sharing database. </P><H3 style="text-align: left; font-size: 18px;">Redesigning the Shuttle Vector</H3><OL><LI>Chloramphenicol resistance doesn’t have AMB-1 promoter before it</LI><LI>Origin of replication is non-functional in PYMB</LI></OL><P style="text-align: left; text-indent:0px">We worked to redesign the shuttle vector to eliminate these problems and create a working plasmid to successfully transform the rare strain of bacteria.</P><P style="text-align: left; text-indent:0px"><B>pMAGMA1: ori(AMB-1) + rep(AMB-1)</B></P><P style="text-align: left; text-indent:0px">This minimal shuttle vector is a modified version of pSB1C3 with an AMB-1 specific origin of replication and replication gene. We are using this design for experiments to test if these two AMB-1 specific parts are sufficient for growth in AMB-1. The antibiotic resistance is the chloramphenicol resistance gene found in pSB1C3, driven by a promoter known to express in E.coli.  </P><P style="text-align: left; text-indent:0px"><B> pMAGMA2: ori(AMB-1) + rep(AMB-1) + pmsp3 + chloramphenicol </B></P><P style="text-align: left; text-indent:0px">pMAGMA2, our second version of an AMB-1 shuttle vector, includes the chloramphenicol resistance gene driven by the AMB-1 specific pmsp3 promoter. This system eradicates the concern that the chloramphenicol gene is not being appropriately expressed because it is not near an AMB-1 promoter, but rather in the pSBIC3 backbone in the original construct. </P><P style="text-align: left; text-indent:0px"><B> pMAGMA3: Final shuttle vector: ori(AMB-1) + rep(AMB-1) + pmsp3 + chloramphenicol + pmsp3 + BioBrick multiple-cloning site</B></P><P style="text-align: left; text-indent:0px">This third and final plasmid, pMAGMA3, is our final envisioned shuttle vector to engineer AMB-1. It is similar to the last two plasmids as it contains the AMB-1 replication cassette. However, it has a more versatile BioBrick multiple cloning site downstream of another pmsp3 promoter for gene expression specifically in AMB-1. This module can be found downstream of the chloramphenicol resistance gene, also driven by pmsp3. </P><P style="text-align: left; text-indent:0px">This final construct is therefore instrumental in the use of AMB-1 as a chassis. This final construct therefore optimized for the use of AMB-1 as a chassis.  It allows researchers to grow their constructs in E.coli, for initial testing and mini-prepping, and then to transform into AMB-1. </P><DIV style="position:absolute;right:5px;bottom:50px"><A href="https://2014.igem.org/Team:Penn/CdTolerance">;
</A></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Penn/Synbio">http://2014.igem.org/Team:Penn/Synbio</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn/Synbio" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn/Synbio" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn/Synbio&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn/Synbio&amp;oldid=401463" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></DIV></BODY></HTML>