"
Page
Discussion
View source
History
teams
Log in
Â 
Team:Penn State
From 2014.igem.org
WELCOME TO PENN STATE iGEM 2014!
Click here  to edit this page!
HOME
JUDGING FORM
OFFICIAL PROFILE
TEAM
PROJECTS
PARTS
WETLAB
SAFETY
HUMAN PRACTICES
ATTRIBUTIONS
Welcome!
You have reached the 2014 Penn State iGEM page.
Here you will find information about our projects, daily and weekly summaries of our wet laboratory activities, and information about our community outreach initiatives.
IMPORTANT LINKS:
Judging Form
Projects
Weekly Lab Summaries
Protocols
Meet the Team!
Our Projects
Click on the pictures for more information!
Engineering a Biodetoxification Pathway
for Lignocellulosic Feedstock
Codon Optimization
Abstract
Biofuels can be produced from fermenting biomass by bacteria. However, biomass is tough to break down and requires costly pretreatment processes before it can be converted to fuel. Pretreatment produces toxic byproducts, including furfural and 5-hydroxymethyl furfural (HMF), which will kill microbes. To solve this problem, we intend to engineer bacteria with a recently discovered metabolic pathway that consumes furfural and HMF. Koopman et. al. identified the six enzyme pathway from Cupriavidus basilensis and showed that it functions in Pseudomonas putida. In C. basilensis or P. putida, HMF can be used as the sole carbon source instead of costly sugar. However, this pathway does not function in Esherichia coli. Based on our recent experiments, the pathway also does not function in Pseudomonas fluorescens, a microbial relative of P. putida. We want to determine the genomic differences that allow the pathway to function in one organism versus another. We intend to do this using a novel approach, combinatorial dCas9 gene knockdown. The final objective of this research is to engineer the HMF pathway in E. coli and bring us one step closer to sustainable biofuels produced by bacteria.
Abstract
Codon level optimization of genes allows for fine tuning of expression due to differences in translational efficiency between degenerate codons. It is theorized that through this technique expression ceilings due to translation becoming the rate limiting step in protein synthesis can be lifted.  Traditional methods in E. coli rely on the preference for certain codons across the entire genome, yet this is not the only possible approach.  In this project, novel criteria for codon optimization were employed to design and create variants of a reporter gene that was then characterized in vivo. Results show that the new criteria for codon optimization, for example the statistical correlation between a degenerate codon and its presence in highly translated parts of the genome, are feasible for use in future projects. This leads to new theories about the mechanics of translation, and will allow researchers to optimize genes at the codon level with greater fidelity.
Retrieved from " http://2014.igem.org/Team:Penn_State "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
