"
Page
Discussion
View source
History
teams
Log in
 
Team:Penn State/Biodetox
From 2014.igem.org
WELCOME TO PENN STATE iGEM 2014!
(Page under construction)
Click here  to edit this page!
Home
Team
Official Team Profile
Projects
Biodetoxification Codon Optimization
Parts
Modeling
Notebook
Safety
Human Practices
Attributions
PROJECTS
Project Description
tell about project - give background, essentially write the abstract. (1-2 paragraphs)
references
iGEM teams are encouraged to record references you use during the course of your research. They should be posted somewhere on your wiki so that judges and other visitors can see how you though about your project and what works inspired you.
It's also important to clearly describe your achievements so that judges will know what you tried to do and where you succeeded. Please write your project page such that what you achieved is easy to distinguish from what you attempted.
INFO TO INCLUDE:
Overall project summary
Project Details
Materials and Methods
The Experiments
Results
Data analysis
Conclusions
Projects
Biodetoxification
ENGINEERING A BIODETOXIFICATION PATHWAY FOR LIGNOCELLULOSIC BIOMASS FEEDSTOCK
Project Summary
Abstract will go here....
Why is this important?
If we can determine the genes essential to the function of the pathway responsible for catabolizing furfural and HMF toxins, we can in principle engineer any bacterium to have these genes. Engineering E. coli to have the HMF pathway will allow this well-studied, easily manipulated and fast-growing bacterium to live in the presence of toxins released during biomass pretreatment, and even use furfural and HMF toxins as fuel!
Therefore, industry and academic researchers can subject biomass to more intense pretreatment processes – which release more toxins. Engineered bacteria will be able to live in the presence of these toxins and more sugars will be available for hydrolysis and fermentation, increasing biofuel yields from the same initial amount of biomass.
Background
Greenhouse gas emissions and dwindling fossil fuel reserves have pushed developed countries like the United States to explore renewable fuel sources. “Biofuels” are an attractive sustainable energy technology because they can be produced from plant biomass, which includes wood, grasses, and agricultural waste. Bioethanol and biodiesel can be blended or used as automobile fuel, among other uses. One way to produce biofuels from biomass is by using bacteria to ferment the sugars in plant matter to fuel alcohols. However, the bioenergy industry faces problems converting this inexpensive plant matter into high value fuels. Biomass is tough to break down and requires costly pretreatment processes before it can be converted to fuel. Pretreatment produces toxic byproducts, including furfural and 5-hydroxymethyl furfural (HMF), which will kill cell cultures and inhibit the conversion of biomass to usable sugars.
To solve this problem, we intend to engineer bacteria with a recently discovered metabolic pathway that consumes furfural and HMF. Koopman et. al. identified the six enzyme pathway from Cupriavidus basilensis and showed that it functions in Pseudomonas putida.  In C. basilensis or P. putida, HMF can be used as the sole carbon source. Engineering bacteria with this pathway would allow them to survive and produce biofuels but also use the toxic HMF as an energy source. However, this pathway does not function in Esherichia coli is commonly used to manufacture valuable chemicals, including fuels, because it is fast growing, easily manipulated, and well-studied.  Based on our recent experiments, the pathway also does not function in Pseudomonas fluorescens, a microbial relative of P. putida.
The first and foremost objective of our research is to identify the genomic differences that allow the pathway to function in one organism, P. putida, but not E. coli or P. fluorescens.  We will do this using a novel approach, combinatorial dCas9 gene knockdown.  Cas9 is an RNA-guided DNA endonuclease that cleaves DNA at precise locations.  Cas9 binds CRISPR RNAs (clustered regularly spaced short palindromic repeats) in order to bind corresponding DNA sties.  The deactivated form of Cas9, dCas9, contains two mutations that block its endonuclease activity, but it still can stably bind DNA and block transcription.  Gene expression can be lowered 100- to 1000-fold. To carry out combinatorial gene knockdown, we will construct CRISPR RNA libraries containing all possible 3-gene combinations for the 19 target genes we have identified. These libraries and the dCas9 system will be co-transformed into P. putida construct containing the HMF pathway. These colonies will be grown for the furoic acid assay and we can determine from absorbance measurements whether the HMF pathway is functioning.  We will sequence the genomes of non-functioning strains and identify which genes were turned off via dCas9 knockdown.
The 19 target genes in P. putida were identified through manual genome comparison with the help of graduate student Iman Farasat.  These are genes that are likely involved in furfural catabolism and are present in P. putida but not present in E. coli.  Many of these genes encode cofactors, chaperone proteins, ATPases, and several other possible transport proteins.  Our hypothesis is that a key oxidoreductase in the HMF pathway requires a molybdenum-containing cofactor, which is produced by a separate pathway and inserted with a chaperone protein.   Manual comparison of genomes is time consuming, and another objective of our research is to develop a program that can optimize genome comparison. This program would employ “BLAST”, Basic Local Alignment Search Tool from the National Center for Biotechnology Information, to identify homologs between species and potential target genes that are contained in one genome but not another.  Optimizing genome comparison would allow industrial and academic researchers to identify the likely missing genes in any pathway.
The final objective of this research is to engineer the HMF pathway in E. coli.  This is a late-stage goal, providing the missing ingredients of the HMF pathway are identified.  But if this objective is completed, it could be one step closer to sustainable fuels produced by bacteria.
Our Objectives
1) Validate the gene knockdown strategy using dCas9
This is our first objective.  Once we insert the dCas9 system, HMF pathway, and RFP cassette into the P. putida genome, we will use a crRNA targeting RFP to attempt to knockdown RFP expression. If we see white colonies, we will know that transcription of the RFP gene is blocked and the system works.
2) Validate the functioning of the HMF pathway
Along with our first objective, we will concurrently evaluate the functioning of the HMF pathway inserted into the P. putida genome using a 2-furoic acid assay. We will inoculate LB broth with P. putida colonies that have grown in the presence of 5-fluorouracil and add 3 g/L 2-furoic acid. Colonies that show growth and consumption of 2-furoic acid reflect a correct functioning of the HMF pathway.
3) Identify the essential genes in the HMF pathway
After we complete the RFP knockdown, we will begin our combinatorial knockdown of the 19 HMF pathway target genes previously identified by graduate student Iman Farasat. Using double and triple gene knockdowns, we will be able to analyze which gene or combination of genes is responsible for the functioning of the pathway. We will do this using a 2-furoic acid assay, where we will incubate all constructs in 2-furoic acid (approximately 3 g/L) and monitor their growth and consumption of 2-furoic acid over a 24 hour period. Colonies that display reduced 2-furoic acid concentration demonstrate that the pathway is still functioning – we will look for colonies that do not deplete the 2-furoic acid concentration.  The latter colonies reflect that the pathway has failed to function, and we will use these to determine which gene combinations result in this phenotype.
4) Develop a program for automated genome comparisons
5) Engineer the pathway to function in P. fluorescens and E. coli
This is a late-stage goal that will most likely not be completed before the Jamboree. Once we identify the gene(s) that are essential for the functioning of the HMF pathway, we will insert them into the E. coli genome and evaluate the action of the pathway using the 2-furoic acid assay. If this objective is completed, industry and academic researchers will be able to use our E. coli strain to produce biofuels without the toxic effects from furfural and 5-hydroxymethyl furfural killing cell cultures. They will be able to subject biomass to more intense pretreatment procedures, increasing sugar yields and the amount of biofuels that can be fermented from these sugars.
Design Methods
Insert the dCas9 system and HMF pathway into the P. putida genome
Retrieved from " http://2014.igem.org/Team:Penn_State/Biodetox "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
