<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_State_Biodetoxification"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn_State/Biodetoxification">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn_State/Biodetoxification&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/Biodetoxification&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/Biodetoxification&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn_State/Biodetoxification" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn State/Biodetoxification</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1><FONT color="white"> WELCOME TO PENN STATE iGEM 2014! </FONT></H1><P style="color:white"><A href="https://2014.igem.org/wiki/index.php?title=Team:Penn_State/Biodetoxification&amp;action=edit" style="color:#00008B"> Click here  to edit this page!</A></P><TABLE width="100%"><TBODY><TR height="50px"><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>HOME</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/2014_Judging_Form?id=1506" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>JUDGING FORM</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>OFFICIAL PROFILE</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Team" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>TEAM</B></FONT></FONT></A></TD><TD style="border:2px solid navy" align="center"><A href="https://2014.igem.org/Team:Penn_State/Project" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> PROJECTS</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Parts" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>PARTS</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Notebook" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> WETLAB</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Safety" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B> SAFETY</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/HumanPractices2" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B>HUMAN PRACTICES</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Attributions" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> ATTRIBUTIONS</B></FONT></FONT></A></TD></TR></TBODY></TABLE><CENTER><H1>ENGINEERING A BIODETOXIFICATION PATHWAY FOR  LIGNOCELLULOSIC BIOMASS FEEDSTOCK</H1></CENTER><P>Click <A href="https://2014.igem.org/Team:Penn_State/Project">HERE</A> to return to the Projects page.</P><H3>Project Summary</H3><P>The goal for this project is to identify the essential genes required for the function of the 5-(hydroxymethyl)furfural (HMF) catabolism pathway found in <I>Cupriavidus basilensis</I>.(1) Furfual and its derivatives are toxic byproducts of the pretreatment process used to produce biofuels from biomass. They inhibit microbial growth and fermentation yields. This pathway also functions in <I>Pseudomonas putida</I> but does not function in an industrially relevant organism like <I>Escherichia coli</I>. Determining the essential genes associated with this pathway would allow researchers to engineer <I>E. coli</I> to produce biofuels from biomass and has the potential to make biofuel production cost effective. Sugar is an expensive feedstock, but with the HMF pathway, fermentation bacteria could grow off of the toxins produced in pretreatment. This would also allow for harsher pretreatment conditions and result in higher biofuel yields. We planned to used a combinatorial gene knockdown method using deactivated Cas9 endonuclease (dCas9) and the CRISPR system. We would use a furfural assay to determine which constructs had stopped catabolizing furfural. Sequencing of these constructs would tell us which genes are being turned off, and which need to stay on for the pathway to continue functioning.  </P><H3> Current Status</H3><P>Due to the large size of the dCas9 system (5.5 kb) and the HMF pathway (7.5 kb), we had significant difficulty producing plasmids with these components. The HMF pathway was successfully integrated to a broad host vector, but the dCas9 would not insert. We decided to insert these large systems into the genome of <I>P. putida</I> to alleviate the problems to due high base pair cloning and because <I>P. putida</I> is naturally resistant to several antibiotics, which made it unfeasible to insert multiple plasmids containing the dCas9 and HMF pathway. It is resistant to ampicillin and chloramphenicol and carbenicillin was tested and had no effect on growth. Most recently, we are trying to insert the dCas9 system into the genome using homologous recombination, but that has also proved difficult. We decided to suspend this project until after the Jamboree and focus on the Codon Optimization project, which was also experiencing cloning difficulties.</P><H3>What We Have Accomplished</H3><P>Ashlee and Emily have managed to construct the first plasmid, termed plasmid 1.2, as shown below. This plasmid contains the dCas9 system (5.5 kb), the Lambda Red Recombinase system (3 kb) and the Kanamycin resistance marker (1 kb). It also contains two 1 kb genome overlaps with the <I>P. putida upp</I> gene. This gene encodes for the uracil phosphoribosyltransferase (UPRTase), which is sensitive to 5-fluorouracil. This would allow us to use a counterselection method to insert our second plasmid containing the HMF pathway. However, to construct the second plasmid, we needed the Kanamycin resistance marker to be inserted into the genome. Since this has not happened, we could not continue making plasmids. </P><P>The figure below displays the plan to insert the dCas9 and HMF gene cluster into the genome of <I>Pseudomonas putida</I>. 
</P><P><STRONG>Insert the dCas9 system and HMF pathway into the <I>P. putida</I> genome</STRONG></P><P><FIG caption="">Note: Genes not to scale</FIG></P><P>We prepared a plasmid containing the dCas9 system, kanamycin resistance marker, and Lambda Red Recombinase system. This plasmid was transformed into <I>E. coli</I> due to its greater cloning efficiency, and was excised using restriction enzymes AflII and PacI in order to attempt homologous recombination.</P><P>First, we created what we termed &quot;Plasmid 1&quot; via Gibson Chew-Back Assembly. The individual parts contained in this plasmid are approximately 1 kb genome overlaps &quot;Overlap 1&quot; and &quot;Overlap 2&quot;, ColE1 replication origin so the plasmid can be replicated in <I>E. coli</I>, and the kanamycin resistance marker &quot;KanR&quot; to select for cells that have the correct plasmid.</P><P>Next, we added dCas9 system via an overnight ligation. Using the restriction sites ClaI and XhoI, we can digest the dCas9 PCR product. The Plasmid 1 backbone was digested with ClaI and SalI-HF restriction enzymes. A SalI site is already present in the dCas9 system, but XhoI and SalI sites are compatible. This plasmid was termed &quot;Plasmid 1.1&quot; for ease of understanding, shown in the figure below.</P><P>The final step was to insert the Lambda Red Recombinase system using ClaI and XbaI restriction digest sites. This plasmid was termed &quot;Plasmid 1.2&quot; shown below.</P><H3> Future Plans</H3><P>We will continue trying to insert the Plasmid 1.2 into the <I>P. putida</I> genome using homologous recombination. Before Ashlee graduates, she hopes to construct the second plasmid, containing the HMF pathway and the RFP reporter gene, and insert that into the genome as well. She will use a CRISPR RNA targetting RFP to validate that the dCas9 system can knock out genes. Emily will continue this project after Ashlee graduates in December. She hopes to identify the essential genes of the pathway and engineer it to function in <I>E. coli</I>! </P><H2>More Detailed Information</H2><H3>Why is this important?</H3><P>If we can determine the genes essential to the function of the pathway responsible for catabolizing furfural and HMF toxins, we can in principle engineer any bacterium to have these genes. Engineering <I>E. coli</I> to have the HMF pathway will allow this well-studied, easily manipulated and fast-growing bacterium to live in the presence of toxins released during biomass pretreatment, and even use furfural and HMF toxins as fuel!</P><P>Therefore, industry and academic researchers can subject biomass to more intense pretreatment processes â which release more toxins. Engineered bacteria will be able to live in the presence of these toxins and more sugars will be available for hydrolysis and fermentation, increasing biofuel yields from the same initial amount of biomass.</P><H3>Background</H3><P>Greenhouse gas emissions and dwindling fossil fuel reserves have pushed developed countries like the United States to explore renewable fuel sources. âBiofuelsâ are an attractive sustainable energy technology because they can be produced from plant biomass, which includes wood, grasses, and agricultural waste. Bioethanol and biodiesel can be blended or used as automobile fuel, among other uses. One way to produce biofuels from biomass is by using bacteria to ferment the sugars in plant matter to fuel alcohols. However, the bioenergy industry faces problems converting this inexpensive plant matter into high value fuels. Biomass is tough to break down and requires costly pretreatment processes before it can be converted to fuel. Pretreatment produces toxic byproducts, including furfural and 5-hydroxymethyl furfural (HMF), which will kill cell cultures and inhibit the conversion of biomass to usable sugars.</P><P>To solve this problem, we intend to engineer bacteria with a recently discovered metabolic pathway that consumes furfural and HMF. Koopman et. al. identified the six enzyme pathway from Cupriavidus basilensis and showed that it functions in Pseudomonas putida.  In C. basilensis or P. putida, HMF can be used as the sole carbon source. Engineering bacteria with this pathway would allow them to survive and produce biofuels but also use the toxic HMF as an energy source. However, this pathway does not function in Esherichia coli is commonly used to manufacture valuable chemicals, including fuels, because it is fast growing, easily manipulated, and well-studied.  Based on our recent experiments, the pathway also does not function in Pseudomonas fluorescens, a microbial relative of P. putida.</P><P>The first and foremost objective of our research is to identify the genomic differences that allow the pathway to function in one organism, P. putida, but not E. coli or P. fluorescens.  We will do this using a novel approach, combinatorial dCas9 gene knockdown.  Cas9 is an RNA-guided DNA endonuclease that cleaves DNA at precise locations.  Cas9 binds CRISPR RNAs (clustered regularly spaced short palindromic repeats) in order to bind corresponding DNA sties.  The deactivated form of Cas9, dCas9, contains two mutations that block its endonuclease activity, but it still can stably bind DNA and block transcription.  Gene expression can be lowered 100- to 1000-fold. To carry out combinatorial gene knockdown, we will construct CRISPR RNA libraries containing all possible 3-gene combinations for the 19 target genes we have identified. These libraries and the dCas9 system will be co-transformed into P. putida construct containing the HMF pathway. These colonies will be grown for the furoic acid assay and we can determine from absorbance measurements whether the HMF pathway is functioning.  We will sequence the genomes of non-functioning strains and identify which genes were turned off via dCas9 knockdown.</P><P>The 19 target genes in P. putida were identified through manual genome comparison with the help of graduate student Iman Farasat.  These are genes that are likely involved in furfural catabolism and are present in P. putida but not present in E. coli.  Many of these genes encode cofactors, chaperone proteins, ATPases, and several other possible transport proteins.  Our hypothesis is that a key oxidoreductase in the HMF pathway requires a molybdenum-containing cofactor, which is produced by a separate pathway and inserted with a chaperone protein.  
Manual comparison of genomes is time consuming, and another objective of our research is to develop a program that can optimize genome comparison. This program would employ âBLASTâ, Basic Local Alignment Search Tool from the National Center for Biotechnology Information, to identify homologs between species and potential target genes that are contained in one genome but not another.  Optimizing genome comparison would allow industrial and academic researchers to identify the likely missing genes in any pathway.</P><P>The final objective of this research is to engineer the HMF pathway in E. coli.  This is a late-stage goal, providing the missing ingredients of the HMF pathway are identified.  But if this objective is completed, it could be one step closer to sustainable fuels produced by bacteria.</P><H3>Our Objectives</H3><P><STRONG>1) Validate the gene knockdown strategy using dCas9</STRONG></P><P>This is our first objective.  Once we insert the dCas9 system, HMF pathway, and RFP cassette into the P. putida genome, we will use a crRNA targeting RFP to attempt to knockdown RFP expression. If we see white colonies, we will know that transcription of the RFP gene is blocked and the system works.</P><P><STRONG>2) Validate the functioning of the HMF pathway</STRONG></P><P>Along with our first objective, we will concurrently evaluate the functioning of the HMF pathway inserted into the P. putida genome using a 2-furoic acid assay. We will inoculate LB broth with P. putida colonies that have grown in the presence of 5-fluorouracil and add 3 g/L 2-furoic acid. Colonies that show growth and consumption of 2-furoic acid reflect a correct functioning of the HMF pathway.</P><P><STRONG>3) Identify the essential genes in the HMF pathway</STRONG></P><P> After we complete the RFP knockdown, we will begin our combinatorial knockdown of the 19 HMF pathway target genes previously identified by graduate student Iman Farasat. Using double and triple gene knockdowns, we will be able to analyze which gene or combination of genes is responsible for the functioning of the pathway. We will do this using a 2-furoic acid assay, where we will incubate all constructs in 2-furoic acid (approximately 3 g/L) and monitor their growth and consumption of 2-furoic acid over a 24 hour period. Colonies that display reduced 2-furoic acid concentration demonstrate that the pathway is still functioning â we will look for colonies that do not deplete the 2-furoic acid concentration.  The latter colonies reflect that the pathway has failed to function, and we will use these to determine which gene combinations result in this phenotype.</P><P><STRONG>4) Develop a program for automated genome comparisons</STRONG></P><P><STRONG>5) Engineer the pathway to function in <I>P. fluorescens</I> and <I>E. coli</I></STRONG></P><P>This is a late-stage goal that will most likely not be completed before the Jamboree. Once we identify the gene(s) that are essential for the functioning of the HMF pathway, we will insert them into the E. coli genome and evaluate the action of the pathway using the 2-furoic acid assay. If this objective is completed, industry and academic researchers will be able to use our E. coli strain to produce biofuels without the toxic effects from furfural and 5-hydroxymethyl furfural killing cell cultures. They will be able to subject biomass to more intense pretreatment procedures, increasing sugar yields and the amount of biofuels that can be fermented from these sugars.</P><H3>References</H3><P> (1) Koopman, Frank, et al. &quot;Identification and characterization of the furfural and 5-(hydroxymethyl) furfural degradation pathways of Cupriavidus basilensis HMF14.&quot; Proceedings of the National Academy of Sciences 107.11 (2010): 4919-4924.</P><P> (2) Graf, Nadja, and Josef Altenbuchner. &quot;Development of a method for markerless gene deletion in Pseudomonas putida.&quot; Applied and environmental microbiology 77.15 (2011): 5549-5552.</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Penn_State/Biodetoxification">http://2014.igem.org/Team:Penn_State/Biodetoxification</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn_State/Biodetoxification" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn_State/Biodetoxification" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn_State/Biodetoxification&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn_State/Biodetoxification&amp;oldid=370622" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>