<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_State_Safety"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn_State/Safety">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn_State/Safety&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/Safety&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/Safety&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn_State/Safety" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn State/Safety</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><H1><FONT color="white"> WELCOME TO PENN STATE iGEM 2014! </FONT></H1><P style="color:#E7E7E7"><A href="https://2014.igem.org/wiki/index.php?title=Team:Penn_State/Safety&amp;action=edit" style="color:#00008B"> Click here  to edit this page!</A></P><TABLE width="100%"><TBODY><TR heigth="50px"><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>HOME</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/2014_Judging_Form?id=1506" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>JUDGING FORM</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>OFFICIAL PROFILE</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Team" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>TEAM</B></FONT></FONT></A></TD><TD style="border:2px solid navy" align="center"><A href="https://2014.igem.org/Team:Penn_State/Project" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> PROJECTS</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Parts" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>PARTS</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Notebook" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> WETLAB</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Safety" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B> SAFETY</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/HumanPractices2" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B>HUMAN PRACTICES</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Attributions" style="color:#000000"><FONT color="white"><FONT face="castellar"><B>ATTRIBUTIONS</B></FONT></FONT></A></TD></TR></TBODY></TABLE><TABLE align="center" width="1000px"><TBODY><TR><TD colspan="6"><CENTER><H1>Safety</H1></CENTER></TD></TR></TBODY></TABLE><TABLE align="left" width="600px"><TBODY><TR><TD><H3><CENTER>Our Lab Information</CENTER></H3></TD></TR><TR><TD>We are a BSL-2 laboratory, but all organisms the 2014 iGEM team works with are BSL-1</TD></TR><TR><TD><UL><LI><I>Escherichia coli, DH10B</I></LI><LI><I>Pseudomonas putida, KT2440</I></LI><LI><I>Pseudomonas fluorescens, B2</I></LI></UL></TD></TR><TR><TD>All laboratory members are trained through the Environmental Health and Safety program at Penn State</TD></TR><TR><TD><A href="http://www.research.psu.edu/orp/biohazardous/policies-and-guidelines/policies-and-guidelines">Penn State Biosafety Guidelines</A></TD></TR><TR><TD><A href="http://www.research.psu.edu/orp/biohazardous/policies-and-guidelines/federal-policies">Federal Policies Governing Research Laboratories</A></TD></TR><TR><TD><H4>Sam demonstrating BSL-1 laboratory PPE</H4></TD></TR><TR><TD><H3>The Risks of Our Projects Now</H3></TD></TR><TR><TD><H4>What are the risks to the safety and health of lab members?</H4></TD></TR><TR><TD>To minimize risk to team members working in the lab, BSL-1 facility standard operating procedures are followed. http://www.cdc.gov/training/quicklearns/biosafety/ The lab is kept clean and organized and inspected regularly. All members of the lab group have received current instruction on safety procedures.</TD></TR><TR><TD><H4>What are the risks to the safety and health of the general public (if any biological materials escaped from your lab)?</H4></TD></TR><TR><TD>All strains used are considered nonpathogenic and engineered to be unable to survive outside laboratory conditions. To minimize accidental exposure, access to the laboratory by unauthorized persons is not permitted. There is no planned release of engineered bacteria. In order to reduce risk, cultures are grown in small quantities and micro technique is followed for all laboratory procedures as to minimize ecological impact and potential for toxicity to humans.</TD></TR><TR><TD><H4>What are the risks to the environment (from waste disposal, or from materials escaping from your lab)?</H4></TD></TR><TR><TD>All of the strains we use are engineered to be unable to grow outside of a laboratory environment. Waste disposal should pose no threat as the modified bacteria cannot live on pipet tips or dried up agar plates, and liquid cultures are killed with bleach before disposal. Our lab is locked, with access to lab members only, and any materials that escape should not be able to survive without the correct additives in the media.</TD></TR><TR><TD><H4>What are the risks to security through malicious mis-use by individuals, groups, or countries</H4></TD></TR><TR><TD>None of the strains used are on the Select Agent list and pose no threats to biosecurity.</TD></TR><TR><TD><H4>What measures are you taking to reduce these risks? (For example: safe lab practices, choices of which organisms to use.)</H4></TD></TR><TR><TD>We kill bacterial waste using 10% bleach solution before it is disposed of.  We are also using nonpathogenic baterial strains: E. coli DH10B, P. putida KT2440, and P. fluorescens B2.  These strains cannot grow outside the laboratory environment and are non-infectious BSL 1 agents. Our iGEM projects are designed to use only small scale techniques, reducing the amount of materials used and bacteria grown.</TD></TR><TR><TD><H3>The Risks of Our Project in the Future</H3></TD></TR><TR><TD><H4>What new risks might arise from your project's growth? (Consider the categories of risk listed in parts a-d of the previous question: lab workers, the general public, the environment, and malicious mis-uses.) Also, what risks might arise if the knowledge you generate or the methods you develop became widely available?</H4></TD></TR><TR><TD><P>CODON OPTIMIZATION PROJECT:</P><P>A great success in the field of codon optimization would allow for the optimization of genes at the codon level that were previously considered difficult to repress, tune, or even express at all. The ability to break these barriers would allow researchers, industry specialists, and possibly unauthorized individuals the ability to tune the expression of genes expressed by altering them at the codon level. It is foreseeable that companies generating biological products such as medicines that are coded by weakly expressed genes could find a way to generate stronger genes that code for the exact same products. Because any gene could be optimized in this manner, it is also theoretically possible that the production of toxins or other dangerous biologically produced molecules could be increased - as with any synthetic biology technique. This scenario is not likely, and does not carry significant risk for several reasons. FIrst, alternative methods of increasing output of metabolic pathways are already extant, and the use of codon optimization as a tool for doing so would not be the easiest, quickest, or most cost effective method. Second, to codon optimize a gene, large scale DNA synthesis equipment and know how is required, and any group sending a dangerous genetic element for codon optimization would have to go through the additional layer of security of a commercial entity, and would thus be even less likely to use our method.</P><P>An interesting application of our project, if it were to truly allow for much more efficient optimization of existing genetic systems, is that scale up of biomolecule production could be done much more quickly than is now possible. This is significant because it is foreseeable that in the future a rare chemical or medicine derived from biological sources could require rapid scale up, and the ability to quickly optimize coding sequences could be extremely valuable.</P><P>BIODETOXIFICATION PATHWAY PROJECT:</P><P>Our strains of Escherichia coli and Pseudomonas putida are engineered to be unable to survive outside of the laboratory and we use antibiotics to prevent other bacteria from growing. However, if our materials should ever become contaminated with wild-type bacteria, they could potentially transfer antibiotic resistance markers between hosts. We are inserting the neomycin resistance marker into the genome of P. putida and intend to remove it via homologous recombination in a second round of cloning. If that sample becomes contaminated, it is theoretically possible that other bacteria could acquire this antibiotic resistance. The late stage goal of this project is to engineer E. coli with a pathway to degrade the furfural toxin, which would allow it to survive in the presence of furfural and also metabolize it for energy. At a larger scale we could unknowingly be selecting for a more robust bacteria if any wild-type E. coli was ever able to acquire the pathway or antibiotic resistance markers. </P></TD></TR><TR><TD><H4>Does your project currently include any design features to reduce risks? Or, if you did all the future work to make your project grow into a popular product, would you plan to design any new features to minimize risks? (For example: auxotrophic chassis, physical containment, etc.) Such features are not required for an iGEM project, but many teams choose to explore them.</H4></TD></TR><TR><TD><P>CODON OPTIMIZATION:</P><P>Since transgenic organisms almost always grow slower than their wild type counterparts, our hosts, shackled with a metabolically demanding reporter gene such as GFP would stand little chance at reproduction in the wild. In addition, by knowing the sequence of DNA in the plasmid harboring the fluorescent gene, measures could be taken to disable the transcriptive or translational machinery in these organisms. We are committed to making this data available to anyone who would require it.</P><P>BIODETOXIFICATION:</P><P>The biodetoxification project currently uses an auxotrophic strain of E. coli, in which it cannot grow without leucine. In the future we would intend for others to use similarly auxotrophic strains for biofuel production. If a strain of E. coli was engineered that it could live on furfural, this facility would have to be kept separate from other facilities producing furfural and other furan derivatives. E. coli could survive and grow in the presence of furfural and could contaminate stocks for other applications. </P></TD></TR></TBODY></TABLE><TABLE align="right" width="600px"><TBODY><TR><TD><H3><CENTER>Our Equipment</CENTER></H3></TD></TR><TR><TD>Here is a sample of our equipment that we use almost every day:</TD></TR><TR><TD><H4>Thermocycler for Digestions and PCR's</H4></TD></TR><TR><TD><H4>Incubator Shakers</H4></TD></TR><TR><TD><H4>Centrifuge</H4></TD></TR><TR><TD><H4>Table Top Centrifuge</H4></TD></TR><TR><TD><H4>Tecan</H4></TD></TR><TR><TD><H4>Nanodrop</H4></TD></TR><TR><TD><H4>Electroporator</H4></TD></TR></TBODY></TABLE><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Penn_State/Safety">http://2014.igem.org/Team:Penn_State/Safety</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn_State/Safety" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn_State/Safety" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn_State/Safety&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn_State/Safety&amp;oldid=398852" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>