<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Penn_State_DailyNotebook"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Penn_State/DailyNotebook">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Penn_State/DailyNotebook&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/DailyNotebook&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Penn_State/DailyNotebook&amp;action=history">History               </A></LI><LI style="color:white;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Penn_State/DailyNotebook" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Penn State/DailyNotebook</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2014.igem.org</H3><P><TITLE>Bidirectional Promoters Overview</TITLE></P><H1><FONT color="white"> WELCOME TO PENN STATE iGEM 2014! </FONT></H1><P><FONT color="white"> (Page under construction) </FONT></P><P style="color:#E7E7E7"><A href="https://2014.igem.org/wiki/index.php?title=Team:Penn_State/Notebook&amp;action=edit" style="color:#FFFFFF"> Click here  to edit this page!</A></P><TABLE width="100%"><TBODY><TR height="50px"><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Home </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Team" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Team </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://igem.org/Team.cgi?year=2014&amp;team_name=Penn_State" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Official Team Profile </B></FONT></FONT></A></TD><TD style="border:2px solid navy" align="center"><A href="https://2014.igem.org/Team:Penn_State/Project" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Projects</B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Parts" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Parts </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Modeling" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Modeling </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Notebook" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Notebook </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Safety" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B> Safety </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/HumanPractices2" style=" color:#000000"><FONT color="white"><FONT face="castellar"><B> Human Practices </B></FONT></FONT></A></TD><TD style="border:2px solid navy;" align="center"><A href="https://2014.igem.org/Team:Penn_State/Attributions" style="color:#000000"><FONT color="white"><FONT face="castellar"><B> Attributions </B></FONT></FONT></A></TD></TR></TBODY></TABLE><H1>Penn State iGEM 2014 Notebook Page</H1><P>Below is our detailed, day-to-day <A href="https://2014.igem.org/Team:Penn_State/DailyNotebook">Laboratory Notebook</A>.</P><H2>Weekly Summaries</H2><DIV id="wkshortcut" style="background-color:#e7e7e7;height:375px;width:75px;float:left;" class="sidebar-box"><H4><CENTER>Shortcuts</CENTER></H4><P><CENTER><A href="#Week 1"><FONT color="black">Week 1</FONT></A></CENTER></P><P><CENTER><A href="#Week 2"><FONT color="black">Week 2</FONT></A></CENTER></P><P><CENTER><A href="#Week 3"><FONT color="black">Week 3</FONT></A></CENTER></P><P><CENTER><A href="#Week 4"><FONT color="black">Week 4</FONT></A></CENTER></P><P><CENTER><A href="#Week 5"><FONT color="black">Week 5</FONT></A></CENTER></P><P><CENTER><A href="#Week 6"><FONT color="black">Week 6</FONT></A></CENTER></P><P><CENTER><A href="#Week 7"><FONT color="black">Week 7</FONT></A></CENTER></P><P><CENTER><A href="#Week 8"><FONT color="black">Week 8</FONT></A></CENTER></P><P><CENTER><A href="#Week 9"><FONT color="black">Week 9</FONT></A></CENTER></P><P><CENTER><A href="#Week 10"><FONT color="black">Week 10</FONT></A></CENTER></P><P><CENTER><A href="#Week 11"><FONT color="black">Week 11</FONT></A></CENTER></P><P><CENTER><A href="#Week 12"><FONT color="black">Week 12</FONT></A></CENTER></P><P><CENTER><A href="#Week 13"><FONT color="black">Week 13</FONT></A></CENTER></P></DIV><DIV id="dailynotebook"><H2><A name="Laboratory Notebook"><FONT color="black">Laboratory Notebook</FONT></A></H2><TABLE border="1" style="width:1100px"><TBODY><TR><TD><FONT size="5" face="castellar"><CENTER><B>Biodetoxification</B></CENTER></FONT></TD><TD><FONT size="5" face="castellar"><CENTER><B>Codon Optimization</B></CENTER></FONT></TD></TR><TR><TD><CENTER><B><A name="NB wk1"><FONT color="black">Tuesday, May 20, 2014</FONT></A></B></CENTER></TD><TD><B>First iGEM meeting with Dr. Richard and Dr. Salis.</B></TD><TD>First iGEM meeting with Dr. Richard and Dr. Salis.</TD></TR><TR><TD><CENTER><B>Wednesday, May 21, 2014</B></CENTER></TD><TD><B>Emily's first experience with cloning!</B> Ashlee led Emily through several practice experiments from designs made earlier in the year: making a gel, loading samples, gel purifying DNA. Ashlee introduced Emily to her research the previous semester. She had inserted the HMF-ABCDE pathway on the pSEVA251 KanR plasmid into <I>P. putida</I> and had validated its function, but was having difficulty inserting the dCas9 system into the pSEVA251 KanR HMF-ABCDE plasmid. The plasmid size would be close to 18 kb and this, among other affects, was thought to increase the difficulty of the cloning. Another hypothesis was that the dCas9 pathway did not have a strong terminator for the trans-acting RNA. Ashlee and Emily began work to insert the terminator. They performed a ligation of the pSEVA251/HMF plasmid and the terminator.</TD><TD>Clay and Sam worked on a program in Excel to codon optimize GFPs. Sucess. Unfortunately, program is clunky and requires a lot of user input for any optimization. Decision made to attempt the same task in MATLAB.</TD></TR><TR><TD><CENTER><B>Thursday, May 22, 2014</B></CENTER></TD><TD>Ashlee and Emily finished the ligation, transformed the ligation into DH10B electrocompetent <I>Escherichia coli</I> cells using electroporation, incubated at 37 degrees C for 1 hour, and plated on Kanamycin plates.</TD><TD>RBS design begun. Library calculator run using 34 N's as a constraint in the &quot;Constraints&quot; field. Nothing used in pre sequence field. First 60 bp of original superfolder GFP used as &quot;coding sequence&quot;. Clay started working on MATLAB program for codon optimization.  </TD></TR><TR><TD><CENTER><B>Friday, May 23, 2014</B></CENTER></TD><TD>The transformed colonies took over 24 hours to grow - something unusual for this strain but something Ashlee had observed since after she inserted the HMF pathway and attempted to insert dCas9. We picked six colonies for overnight growth to do more cloning tomorrow.</TD><TD>Design of GFPs continues as Clay works on program to optimize genes. Question asked: which GFP should be optimized? GFP mut3b and superfolder GFP both present advantages and disadvantages. Met with Chiam Yu to discuss effects of codon optimization on translation. Acquired data from previous codon optimization project, which will serve as the basis for our fast/slow codon optimization.</TD></TR><TR><TD><CENTER><B>Saturday, May 24, 2014</B></CENTER></TD><TD>Memorial Day Weekend? How about lab cloning weekend! Only one of the 5 terminator/HMF/pSEVA251 ligation colonies grew. Ashlee conducted plasmid preparation of the terminator/HMF/pSEVA251 vector and digested this (the backbone) and the dCas9 system (the insert) with restriction enzymes AatII and AflII.</TD><TD>More RBS library calculations run. Problem: since TIR is dependent on the first 60 bp of a coding sequence and our variant GFPs will differ in this region, how can we ensure that an accurate library is developed?</TD></TR><TR><TD><CENTER><B>Sunday, May 25, 2014</B></CENTER></TD><TD>The 5 colonies still did not grow. Ashlee prepared a gel and ran the dCas9 and backbone digestions and gel purified them. Colonies had not been growing well for Ashlee in the latter half of the semester as the plasmid size increased above 12 kb, so a new strategy would have to be pursued.</TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B><A name="NB wk2"><FONT color="black">Monday, May 26, 2014</FONT></A></B></CENTER></TD><TD>Due to a string of failed clonings with the broadhost vector pSEVA251 and the large inserts, new designs are evaluated! Instead of creating a plasmid with the HMF pathway (7.5 kb) and dCas9 system (5.5 kb), we shall add the HMF pathway and dCas9 to the <I>P. putida</I> genome using homologous recombination. </TD><TD>Decision reached to optimize Superfolder GFP based on its superior post translational modification, ensuring that translation elongation remains the rate limiting step. Papers on codon optimization downloaded to Mendeley Desktop. New hypothesis for existence of rare codons developed: Perhaps they function as a molecular &quot;brake&quot; to slow down translation and prevent &quot;ribosome traffic jams.&quot;</TD></TR><TR><TD><CENTER><B>Tuesday, May 27, 2014</B></CENTER></TD><TD>Inoculated LB broth with ampicillin and dCas9 plasmid from cryogenic storage; inoculated Lb broth with chloramphenicol and FTV vector from Ashlee's past experiment; streaked the HMF vector on a kanamycin plate.</TD><TD>Decided to use homogeneous &quot;leader sequence&quot; upstream of each variant GFP to ensure Ribosome Binding Site (RBS)library creates accurate range of translation initiation rate (TIR)</TD></TR><TR><TD><CENTER><B>Wednesday, May 28, 2014</B></CENTER></TD><TD>Emily and Ashlee made cryogenic storage of the dCas9 plasmid; plasmid prepared the FTV and dCas9 vectors; digested FTV vector; inoculated LB broth with a colony from the HMF plate for overnight growth and plasmid preparation tomorrow.</TD><TD>Optimized leader sequence from previous project for our use. Also explored options of creating novel synthetic leader as well as using first 60 base pairs of GFP mut3b. Met with Dr. Salis, decided to use a new synthetic leader. Created leader based on several design considerations. Ran RBS library calculator using a &quot;pre sequence&quot; of 20 bp upstream of RBS.</TD></TR><TR><TD><CENTER><B>Thursday, May 29, 2014</B></CENTER></TD><TD>Prepared plasmid containing the HMF pathway; inoculated LB broth with Lambda Red Recombinase plasmid from cryogenic storage. Ashlee, Emily, and graduate student Iman Farasat ordered primers for three plasmids that will be constructed via Gibson Chew-Back and Annealing Assembly, two of which will be inserted into the genome by homologous recombination.</TD><TD>Diagrammed &quot;Cloning Strategy&quot; for the project, including all steps from receiving synthetic DNA to characterization. This is a work in progress! Chose pFTV as vector, decided to order variant GFPs as gblocks through IDT.</TD></TR><TR><TD><CENTER><B>Friday, May 30, 2014</B></CENTER></TD><TD>Ashlee and Emily made cryogenic storage of the Lambda Red Recombinase plasmid.</TD><TD>Created a &quot;Plan B&quot; for cloning that details fallback plans and options that we will pursue if cloning is not successful. Simplified Outline: Ligate one GFP into pFTV, ligate in the dRBS. Transform cells, plasmid prep and sequence to determine which RBS was taken by each. Swap out GFPs, then sequence again to ensure that variant GFPs were successfully introduced. </TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B><A name="NB wk3"><FONT color="black">Monday, June 2, 2014</FONT></A></B></CENTER></TD><TD>Constructed dCas9 gene cassette and plasmid backbone with replication origin ColE1 via PCR Rescue. Gel purified dCas9 and ColE1 cassettes. 1 out of 4 dCas9 PCR's were successful, and 2 out of 4 colE1's were successful, all of which were Ashlee's. We attributed this to Emily's lack of cloning experience.</TD><TD> Sam prepared electrocompetent cells for use later on, and began process of primer design. Clay designed the synthetic leader sequence and finalized the program in MATLAB that optimizes GFPs at codon level.Met with Dr. Salis and decided to also optimize a GFP for slow insertion time, based on a model created by Iman Farasat.</TD></TR><TR><TD><CENTER><B>Tuesday, June 3, 2014</B></CENTER></TD><TD>Conducted Colony PCR using <I>P. putida</I> KT2440 strain as DNA template to construct two ~1 kb genome overlaps. Plasmid prepared the Lambda Red Recombinase plasmid, DH10B-PKD46, FTV-ptac-LacI-CmR plasmid, and NoHP_15A_Plmra_CmR plasmid containing RFP with a strong, unique promoter.  Stock of NoHP_15A_Pkmra_CmR and FTV_ptac_LacI_CmR for cryogenic storage was also made. Lambda Red Recombinase cassette was amplified using PCR Rescue and gel purified.</TD><TD>All constructs (variant GFPs in vector pFTV checked for enzyme restriction sites, enzymes picked to be used in the cloning process. gblocks designed using format: junk DNA- restriction site- CDS- restriction site- junk DNA. Sam designed rescue primers to be used for amplifying the gblocks. They will be expensive and we don't want to leave any chance of running out of stock once we have them. Primers for rescue PCR redesigned when it was realized that Clay accidentally truncated the GFPs by incorrectly copying the coding sequence of original superfolder GFP from its Ape file.</TD></TR><TR><TD><CENTER><B>Wednesday, June 4, 2014</B></CENTER></TD><TD>We made ampicillin agar plates and ampicillin antibiotic stock for cloning. The PCR Rescue of Lambda Red Recombinase was also gel purified.</TD><TD>Sam made Chloramphenicol plates for use later on.Redesigned leader sequence to be a full 60 bp, redesigned rescue primers again. Sequencing primers designed. MATLAB program updated to optimize for slow insertion time GFP. Script also created to total the insertion times of each GFP. Primers updated again as enzymes were re chosen, due to the presence of one of them in the CDS of slow insertion time GFP.  </TD></TR><TR><TD><CENTER><B>Thursday, June 5, 2014</B></CENTER></TD><TD>We conducted PCR Rescue to amplify the kanamycin resistance cassette (specifically the neomycin cassette, which also confers resistance to kanamycin) from pSEVA251 KanR plasmid. Two different sets of primers for kanamycin were tested, and the first set was successful - all 4 PCR's were correct. The second set of primers all failed. However, <B>Emily had her first PCR success!</B> Kanamycin cassette was gel purified.</TD><TD>Project Plan updated. Five GFPs will be ligated into pFTV separately, then dRBS will be ligated in. Data will be collected and sequencing will show which RBS was used by each colony. Cryogenic stock of cells harboring pFTV grown.</TD></TR><TR><TD><CENTER><B>Friday, June 6, 2014</B></CENTER></TD><TD>Conducted colony PCR using <I>P. putida</I> KT2440 strain as the DNA template to construct 1 kb overlaps for homologous recombination. All four of the first genome overlaps with gene PP_0747 were successful; only 2 overlaps with <I>upp</I> gene were successful. These were gel purified. We learned Gibson Chew-Back Annealing Assembly (CBA) protocol. </TD><TD>  Gblocks arrived. Rescue PCR conducted to increase stocks of gblocks. Samples run in gel and purified. Plasmids harvested from cells harboring pFTV. Clay left early to rebuild the deck at his house. Bastard. </TD></TR><TR><TD><CENTER><B>Sunday, June 8, 2014</B></CENTER></TD><TD>One 4-part, two 3-part, and two 2-part Gibson CBA's were conducted to assemble the kanamycin resistance cassette, two genome overlaps, and the colE1 replication origin. This completed plasmid will be termed &quot;plasmid 1&quot;. </TD><TD>Inverse PCR conducted on pFTV backbone. Shows very faint bands in gel, decided to increase number of cycles from 30 to 35. </TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B><A name="NB wk4"><FONT color="black">Monday, June 9, 2014</FONT></A></B></CENTER></TD><TD>The two 2-part and two 3-part CBA's were amplified using PCR Rescue and gel purified. The original 4-part CBA was transformed into <I>E. coli</I> electrocompetent cells using electroporation and plated on kanamycin antibiotic agar plates. The 4-part CBA was repeated to ensure accuracy. Because the CBA parts contained no plasmid DNA, the 4-part CBA could be digested by restriction enzyme Dpn1. Dpn1 binds and cuts methylated DNA sites, thus destroying any plasmid DNA remaining as a contaminant.</TD><TD>More issues with RBS library design, as it seems very difficult to find sequences with sufficiently high TIR. Decided to use an initial condition for the calculator, which should speed it up and also ensure higher TIR is reached. More calculations ran.</TD></TR><TR><TD><CENTER><B>Tuesday, June 10, 2014</B></CENTER></TD><TD>The original 4-part CBA worked! Many colonies appeared on the plate after incubation at 37 degrees C for 18 hours, and 12 colonies were selected for plasmid preparation. These were digested with AatII and XbaI, two restriction sites that are only both contained in the final assembled 4-part plasmid. 6/12 colonies showed the correct bands on the gel. We also prepared more 1 kb ladder from concentrate.</TD><TD> gblocks digested to ready them for ligation into pFTV. Not enough stock of pFTV was present, so inverse PCR ran again, this time with 3 tubes.</TD></TR><TR><TD><CENTER><B>Wednesday, June 11, 2014</B></CENTER></TD><TD>3 successful colonies were sent for sequencing. In order to insert the dCas9 system into plasmid 1, dCas9 was digested with XhoI and ClaI. 4 successful colonies were digested with ClaI for 3 hours, heat inactivated at 65 degrees C, and then digested with SalI-HF restriction enzyme. SalI and XhoI are compatible sites. These digestions were gel purified, resulting in low concentrations of plasmid DNA. Only two colonies were used to continue further. We met with Leah Bug and Matthew Johnson from the <A href="http://csats.psu.edu/">Penn State Center for Science and the Schools</A>.</TD><TD>pFTV digested. Not enough Cla1 enzyme was present, so this step will be suspect if there are issues in the future. More Cla1 ordered. Inverse PCR ran again, this time with added extension time (2:30 instead of 1:30). Inverse PCR shows very faint bands again and gel not excised.</TD></TR><TR><TD><CENTER><B>Thursday, June 12, 2014</B></CENTER></TD><TD>The plasmid backbone was digested with phosphotase enzyme. These backbones were ligated to dCas9 over 18 hours at 16 degrees C to ensure maximum ligation product.</TD><TD>Considerable time spent working with RBS library calculator. Calculations ran to determine max TIR for the leader sequence that was designed as well as with the enzyme restriction site (Pst1) downstream of the RBS.</TD></TR><TR><TD><CENTER><B>Friday, June 13, 2014</B></CENTER></TD><TD>Emily purified the ligation product.</TD><TD>Calculations from yesterday yield very high TIR. Still no good libraries, though. Libraries ran using new initial conditions.</TD></TR><TR><TD><CENTER><B>Sunday June 15, 2014</B></CENTER></TD><TD>Ashlee transformed the ligation into <I>E. coli</I> DH10B electrocompetent cells via electroporation, and plated them on kanamycin antibiotic agar plates to grow overnight.</TD><TD>gblocks and pFTV ligated together. Cells transformed.</TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B><A name="NB wk5"><FONT color="black">Monday, June 16, 2014</FONT></A></B></CENTER></TD><TD>The ligation failed. We amplified more of the dCas9 system using PCR Rescue, in which 2 out of 4 PCR's were successful - <B>both Emily's!</B>. We are evaluating this difficult PCR and will be altering the annealing temperature. PCR Rescue for dCas9 were repeated at 58 degrees C and 62 degrees C annealing temperature. We inoculated LB broth with ampicillin resistance and dCas9 from cryogenic stock. We also conducted colony PCR of the second set of genome overlaps. We received our sequencing results and two out of three colonies have the correct sequence.  </TD><TD>Overnight cultures of transformed cells streaked on plates.</TD></TR><TR><TD><CENTER><B>Tuesday, June 17, 2014</B></CENTER></TD><TD>PCR Rescue RFP cassette and gel purified - all 4 RFP PCR's worked. Gel purified dCas9 Rescue from yesterday. PCR Rescue colE1 origin and chloramphenicol resistance cassette to construct plasmid 2, which will contain ColE1, CmR, RFP, HMF pathway, and two <I>P. putida</I> genome overlaps. We plasmid prepared new dCas9 to use as a template for PCR. New and old dCas9 templates were used for PCR Rescue of the third genome overlap. See the schematic here <FONT color="red" here=""> for more information. </FONT></TD><TD>Plates show only two colonies, one each from &quot;slow&quot; GFP and &quot;slow insertion time&quot; GFP. These colonies sampled, grown in cultures. </TD></TR><TR><TD><CENTER><B>Wednesday, June 18, 2014</B></CENTER></TD><TD>Digested the HMF pathway with EcoRI-HF and PstI-HF restriction enzymes. All PCR's of the third genome overlap containing dCas9 failed, and we realized we must complete the first plasmid by inserting dCas9 and use that as a template instead of the original dCas9 plasmid. This points the failure of the ligation to either the dCas9 PCR's or the ligase buffer has expired. We made new aliquots of fresh ligase buffer to test whether this was the case. We have run out of ClaI and cannot digest dCas9 until this arrives. Our strategy now is to Gibson assembly the Lambda Red Recombinase system and dCas9, then PCR Rescue and ligate into plasmid 1 using XhoI/SalI-HF and XbaI to mitigate the lack of ClaI.</TD><TD> Plasmid prep on cultures from colonies that grew after transformation. New hypothesis develope: perhaps the higher expression GFPs killed the cells due to their extremely rapid translation elongation, leaving only the less efficiently translated GFP carrying cells to live. Online check using website Genscript and their free gene analysis tool shows that the common GFP scores a perfect 1.0 and the rare GFP a perfect 0.0 on their scale (from 0 being not optimized at all to 1 being perfectly optimized). This reveals that the algorithm used by Genscript corresponds directly to the codon usage profile for E.coli over the entire genome. </TD></TR><TR><TD><CENTER><B>Thursday, June 19, 2014</B></CENTER></TD><TD>Conducted 2-part Gibson CBA to assemble the Lambda Red Recombinase system and the dCas9 system.</TD><TD>Many more RBS library calculations run, some using a method where the Shine Dalgarno (SD) sequence is mutated, but not the rest of the initial condition. Hopefully this will speed up the calculations and finally yield some high TIR libraries that evenly span the range we are looking for. Another overnight culture of slow GFP/slow insertion time GFP cells innoculated.</TD></TR><TR><TD><CENTER><B>Friday, June 20, 2014</B></CENTER></TD><TD>PCR Rescue to amplify Lambda Red Recombinase and dCas9 cassette and gel purified. Gel bands reflect failed Gibson CBA.</TD><TD>Plasmid harvest on culture from yesterday. Plasmids digested using Xho1 and Xmal1, expecting to see two bands, one at 1.1 kb and one at 1.7 kb. Ran in gel. Gel displays the correct bands. This shows that our construct is basically correct and gives us confidence to send for sequencing to confirm.</TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B><A name="NB wk6"><FONT color="black">Monday, June 23, 2014</FONT></A></B></CENTER></TD><TD>Repeat Gibson CBA of Lambda Red Recombinase system and dCas9 cassette using 25 femtomole DNA and 50 femtomole DNA. Emily and Ashlee worked on the presentation to the teachers for the <A href="http://csats.psu.edu/"> Center for Science and the Schools</A>. </TD><TD>Plasmids prepared and sent for sequencing to Quintara Bio.</TD></TR><TR><TD><CENTER><B>Tuesday, June 24, 2014</B></CENTER></TD><TD>Ashlee and Emily PCR Rescued PAK-HMF and ran the results on a gel for later purification.  Continued work on the presentation and a demonstration to the teachers for the <A href="http://csats.psu.edu/"> Center for Science and the Schools</A>.</TD><TD>Inverse PCR attempted again, in case we will need to go back to it. Five tubes run, this time with added cycles and extension time. Still, bands are very faint. Need to find a way to get this reaction to be successful.</TD></TR><TR><TD><CENTER><B>Wednesday, June 25, 2014</B></CENTER></TD><TD>Emily PCR Rescued the FTV vector using crRNA primers.  The PCR results were gel purified.  More work was done to update the Wiki and brainstorm design ideas for it.  Emily also did some finishing touches on the presentation for the <A href="http://csats.psu.edu/"> Center for Science and the Schools</A>. Ashlee updated the Notebook and Team information on the website. She is slowly learning HTML and CSS.</TD><TD>Sequencing Data arrived, showing that the vectors did not pick up an insert at all, and re-circularized without one. Seems as if inverse PCR was successful. Tail sequence of forward primer shown perfectly, tail sequence of reverse primer shown partially by sequencing. Plan is to re-do the digestion of gblocks and pFTV and then ligate, transform again. Inverse PCR completed again, this time shows extremely faint bands. We know that this was successful at least once because of the sequencing data, but we haven't been able to replicate it. Suspicion that the origional plasmid prep of pFTV may have been faulty due to the low concentration that was gathered makes us want to repeat this process. t</TD></TR><TR><TD><CENTER><B>Thursday, June 26, 2014</B></CENTER></TD><TD>Ashlee and Emily met with Dr. Richard to discuss research using his biomass hydrolyzer and using a HYSYS model of a biorefinery plant to evaluate the economic benefits of increasing the threshold of furfural/HMF toxin bacteria can tolerate by inserting the HMF-ABCDE catabolism pathway. We will begin manipulating the Aspen Plus model and Economic analysis from the <A href="http://www.nrel.gov/">National Renewable Energy Laboratory (NREL)</A>. </TD><TD>Plan for continuing codon optimization project is supported by the advisers. Clay updated notebook online as well as the document diagramming the design process of the project.More RBS calculations run. Problem with RBS library calculator at this point is that using no initial condition (only N's in the  constraints field, followed by Pst1) the calculator does not mutate the Pst1 site that needs to be part of the dRBS, but yields very low TIR. Using a strong initial condition, followed by Pst1, the calculator yields acceptable TIR but mutates Pst1. Inserting Pst1 into these calculations after the fact, and then using the &quot;evaluate&quot; function of the calculator shows that TIR drops precipitously. Calculations were run in single RBS forward engineering mode to determine max TIR that can be found using only N's and Pst1. Culture of pFTV harboring cells inoculated.  </TD></TR><TR><TD><CENTER><B>Friday, June 27, 2014</B></CENTER></TD><TD>Ashlee performed a transformation of Plasmid 1 into DH10B <I>Escherichia coli</I> electrocompetent cells via electroporation and plated this on Kanamycin plates. This was done to regenerate our Plasmid 1 so we can digest it and insert the dCas9 and Lambda Red Recombinase systems. She updated the Notebook and Project pages, and is constructing figures for her design process. She is having trouble with &quot;div&quot; functions in HTML and is consequently frustrated. We met with Dr. Salis and Dr. Richard for our biweekly iGEM meeting and Ashlee presented the work on the website, presentation and Emily's demonstration idea for the Center for Science and the Schools, and current progress on the project.</TD><TD>Culture of pFTV harboring cells plasmid harvested, much better concentration of pFTV measured than before. Inverse PCR conducted again, this time using optimized protocol and 3 tubes. Lab meeting. Sam did gel extraction, purification. Seems like strong bands were observed, and at the correct places. This is a big success.</TD></TR><TR><TD><CENTER><B>Saturday, June 28, 2014</B></CENTER></TD><TD>The transformation did not grow. We must pick new colonies and sequence confirm they have the right Gibson CBA parts in order to resupply our stock of Plasmid 1. This will be used as the backbone for the insertion of dCas9 and Lambda Red Recombinase.</TD></TR><TR><TD><CENTER><B>Sunday, June 29, 2014</B></CENTER></TD><TD>Picked 10 colonies for overnight growth from the second 4-part CBAR of Plasmid 1, which was digested with Dpn1 to ensure no contamination by original plasmid DNA.</TD><TD>Sam digested gblocks and pFTV again using Cla1 and Pst1, purified, then ligated them together, purified again, then transformed cells and plated them. </TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B><A name="NB wk7"><FONT color="black">Monday, June 30, 2014</FONT></A></B></CENTER></TD><TD>Plasmid prepared 10 colonies. Digested them with AatII and XbaI, which are restriction sites only contained in the final Plasmid 1. These 10 digests were run on a gel, of which 4 showed the correct bands at approximately 1 kb and 3 kb.</TD><TD>No colonies observed on plates from Sunday. Attempted to use ligation product from Sunday to transform another batch of cells, because of sudden fear that our electrocompetent cells may not be good. Not enough ligated product remains to attempt this. </TD></TR><TR><TD><CENTER><B>Tuesday, July 1, 2014</B></CENTER></TD><TD>Ashlee and Emily prepared tubes of primers and four purified digestion products to send for sequencing.  They continued their work on the website and tried to find more html help for certain aspects.</TD><TD>Goal of the day is to transform some more cells, and use different ratio of insert to backbone during ligation in hopes of getting colonies to grow. Attempted to re-ligate digested GFPs and pFTV but not enough remains from Sunday to attempt this. Calculations ran for re-doing digestions and ligation. Digestions completed. RBS calculations from 6/26 analyzed, found that even using no initial condition (only N's followed by Pst1), sufficiently high TIR can be discovered. Need to use calculator algorithm on either Dr. Salis or Iman's computer so that a dRBS can be constructed using an initial condition that is followed by a non-mutable Pst1 site. </TD></TR><TR><TD><CENTER><B>Wednesday, July 2, 2014</B></CENTER></TD><TD>Ashlee worked remotely on the website and is awaiting sequencing results. Plasmids with all four junctions correct (ColE1, KanR, Overlap 1, and Overlap 2) will be used to re-attempt the insertion of dCas9.</TD></TR><TR><TD><CENTER><B>Thursday, July 3, 2014</B></CENTER></TD><TD>Ashlee updated the Notebook and Project pages. She added shortcuts to the Notebook page to make it more user-friendly. She is also constructing more figures to display the design process and &quot;Plasmid 1&quot;, &quot;Plasmid 2&quot;, and &quot;Plasmid 3&quot; constructs.</TD></TR><TR><TD><CENTER><B>Friday, July 4, 2014</B></CENTER></TD><TD>'MERICA. Ashlee began to update the Safety page. </TD></TR><TR><TD><CENTER><B>Saturday, July 5, 2014</B></CENTER></TD><TD>Ashlee updated the project page and worked through a CSS tutorial to better organize all the pages. She is still working to improve the Biodetoxification figures.</TD></TR><TR><TD><CENTER><B>Sunday, July 6, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, July 7, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, July 8, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Wednesday, July 9, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Thursday, July 10, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Friday, July 11, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Saturday, July 12, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Sunday, July 13, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, July 14, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, July 15, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Wednesday, July 16, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Thursday, July 17, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Friday, July 18, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Saturday, July 19, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Sunday, July 20, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, July 21, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, July 22, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Wednesday, July 23, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Thursday, July 24, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Friday, July 25, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Saturday, July 26, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Sunday, July 27, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, July 28, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, July 29, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Wednesday, July 30, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Thursday, July 31, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Friday, August 1, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Saturday, August 2, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Sunday, August 3, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, August 4, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, August 5, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Wednesday, August 6, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Thursday, August 7, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Friday, August 8, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Saturday, August 9, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Sunday, August 10, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, August 11, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, August 12, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Wednesday, August 13, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Thursday, August 14, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Friday, August 15, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Saturday, August 16, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Sunday, August 17, 2014</B></CENTER></TD></TR><TR><TD><A href="#Top">Back to Top</A></TD></TR><TR><TD><CENTER><B>Monday, August 18, 2014</B></CENTER></TD></TR><TR><TD><CENTER><B>Tuesday, August 19, 2014</B></CENTER></TD></TR></TBODY></TABLE></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2014.igem.org/Team:Penn_State/DailyNotebook">http://2014.igem.org/Team:Penn_State/DailyNotebook</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Penn_State/DailyNotebook" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Penn_State/DailyNotebook" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Penn_State/DailyNotebook&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Penn_State/DailyNotebook&amp;oldid=28466" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2014.igem.org:Privacy_policy" title="2014.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2014.igem.org:General_disclaimer" title="2014.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>