<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UCL_London_Project_Approach"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:UCL_London/Project/Approach">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UCL_London/Project/Approach&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL_London/Project/Approach&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UCL_London/Project/Approach&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UCL_London/Project/Approach" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UCL London/Project/Approach</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><DIV id="ucltop_nav"><UL><LI><A href="https://2009.igem.org/Team:UCL_London"><SPAN>Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Team"><SPAN>The Team</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Notebook"><SPAN>Wet Lab</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Ethics"><SPAN>Ethics</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Project/References"><SPAN>References</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Acknowledgement"><SPAN>Acknowledgements</SPAN></A></LI></UL></DIV><DIV id="uclmain_container"><DIV id="uclleft_col"><UL id="uclleft_nav"><LI><A href="https://2009.igem.org/Team:UCL_London"><SPAN>Home</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Project/Description"><SPAN>Description</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Project/Objectives"><SPAN>Objectives</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Project/Approach"><SPAN>Approach</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Project/Vision"><SPAN>Vision</SPAN></A></LI><LI><A href="https://2009.igem.org/Team:UCL_London/Applications"><SPAN>Applications</SPAN></A></LI></UL><P style="margin:20px;"> </P><H2>Sponsors</H2></DIV><DIV id="ucla1"><H2><SPAN class="mw-headline" id="Approach">Approach</SPAN></H2><P><B>The Idea</B></P><P>After brain storming project ideas during the spring the team had come up with three ideas we considered not only very interesting but also manageable for a summer project. The three main ideas were: 
</P><P>1.	Incorporation of some photosynthetic function into e.coli or characterisation of a small library of photosynthetic related biobricks for prokaryotes.
</P><P>2.	Tightly inducible and well controlled cell membrane disruption for drug release to replace mechanical homogenisation in large scale bioprocessing.
</P><P>3.	Traffic-light stress sensor for bio-processing application. For use within bioprocessing, process design or optimisation we would want examine stresses such as; low oxygen level, shear stress, acetate accumulation, high/low pH, high/low temperature, misfolded pharmaceutical proteins in cytoplasm/periplasm and growth phases. 
</P><P>
We decided to make the Traffic-light stress sensor in <I>E.coli</I> and also to narrow our detection range to:
</P><P><B>Low oxygen levels</B></P><P><B>Acetate accumulation</B></P><P><B>Misfolded proteins</B></P><P>The reason for calibrating our biosensor for the three stresses above is that these stresses seemed to be the most crucial ones for improving general bioprocessing.
Literature review introduced us to a possible way to detect misfolded proteins in the periplasm via the two component signal transduction systems CpxAR or BaeSR and the Extracytoplasmic function Sigma-E. Low oxygen levels we thought we should be able to detect via activation of the FNR molecule in anaerobic conditions. Acetate seemed to be just too elusive to be detected in <I>E.coli</I> in a simple way. Long term exposure to acetate can increase transcription of certain flagellar subunits in <I>E.coli</I> but it is unknown how this is controlled. Acetate can also impact the rotation of a flagellum to make the bacteria tumble and change direction. However, we could not come up with a solution of how to translate this into something that can be detected more easily. 
A eukaryotic DNA binding protein; FacB from ``Aspergillus nidulance`` could possibly be used for acetate detection. At the end we decided that the quest for successfully incorporating a not very well characterised eukaryotic signalling system into a prokaryote would have added to many new layers of complexity into our project. We settled for making a biosensor that could detect low oxygen levels, misfolded proteins in the periplasm and possibly also shear stress.
</P><P><B>Devise for detection of low dissolved oxygen tension DOT</B></P><P><I>E.coli</I> has at least two global regulatory systems to adapt itself for anaerobic growth. These systems are the one component Fnr protein and the two-component Arc system. 
After reviewing in the literature what had been done before this project we decided to base our oxygen sensor system on a promoter being activated by Fnr under anaerobic conditions. We based the sequence of this promoter on the promoter sequence of NarK which has been showed to be induced more than 100-fold under anaerobic conditions. When analysing the upstream sequence of NarK and comparing this with the binding region for Fnr we decided to exchange on basepair in an attempt to optimise the affinity for Fnr. 
The biobricks are called; BBa_K239005 for the native NarK promoter and BBa_239006 for the modified and shorter modified NarK promoter.
However, initial experimental results have not been able to confirm that the promoters are working as expectedly. Before the experiments the promoters have been incorporated into testing devices by ligateing  them upstream of with RBS, GFP and terminator to form the biobricks BBa_K239010 and BBa_K239011. These devises have then been cultivated without access to air and oxygen or in nitrogen sparged media. The native NarK promoter should also be weakly activated by nitrate due to binding sites for the nitrate activated protein NarL. We have not been able to detect this nitrate activation either when cultivating our strains in different concentrations of Sodium nitrate.   
In the writing moment we have still not confirmed the sequence of these devices so there is still a risk that something is wrong with the sequence. 
Before designing our devise we made a model for oxygen levels in Excel to determine in within what time frame our oxygen sensor would have to respond. The time frame would depend on which oxygen concentration or DOT level the sensor could detect together with other factors such as geometry of fermenter or growth rate depending on strain, media e.tc.  
The input table of the model is shown below with the calculated necessary respond time in blue colour. The Excel model can be found in the link below the table.
</P><P><A href="https://2009.igem.org/Image:UCL_London_DOT_model.xls#filelinks" class="external text" rel="nofollow">Model for required DOT response time</A> (for biosensor) in order to detect lower oxygen level, before it has droped to levels inhibiting for the process (e.g. 30 or 40% DOT).
</P><H2><SPAN class="mw-headline" id="Background_summary_review_on_periplasmic_Misfolded_Proteins">Background summary review on periplasmic Misfolded Proteins</SPAN></H2><P><B>General introduction to stress responses and global regulation in bacteria</B></P><P>Many bacteria have evolved a large amount of more or less independent mechanisms to respond to a wide range of different environmental conditions in order to adapt themselves to changing environments. Fast and accurate response to changing conditions can often be crucial for survival or present an important evolutionary advantage; e.g. when plentiful nutrients become available and fast adaptation to higher growth rate can lead to a larger share of the total organism population in the habitat.
Some regulatory systems respond to major changes in the environment by regulating a vast amount of operons. These systems are defined as Global regulatory mechanisms. If one protein is regulating many operons, these operons are then classified as part of the same Regulon. A group of regulons that respond to the same change in environmental conditions are all defined as part of the same Stimulon.
Sigma factors can often play an important role in global regulatory mechanisms. In E.coli examples of this class of sigma factors are: sigma factor 54 (ammonia limitation), sigma factor S (nutritional deprivation and general stress), sigma factor H/32 (abruptly elevated temperature) and sigma factor E/24 (Misfolded outer membrane proteins). Sigma factor 70 is the major sigma factor in E.coli responsive for direction of RNA polymerase to the majority of promoter sequences. When another environmentally induced sigma factor gets activated the cell is rapidly able redirect its transcription to better adapt to the new conditions.  
Sigma factors and their function vary widely in different bacteria and often two sigma factors with the same name can be completely different, without any similarity in function. E.g. in the gram positive bacteria B.subtilis sigma factor H has a function involved with sporulation and has an independent structure from sigma factor H in E.coli (involved in heat shock response). In E.coli there are 6 inducible specialised sigma factors in addition to the housekeeping sigma-70. 
Sigma factors are unique to bacteria and bacteria phages and have so far not been found in eukaryotes.   
</P><P><B>Envelope stress responses</B></P><P>When stress sensing has to be transmitted trough membranes in order activate response another level of complexity is added to task of sensing stress. This transmission of stress signals is necessary for gram negative bacteria such as E.coli when responding to stresses induced in the periplasm, outer membrane and the peptidoglycan layer. (Hayden and Ades., 2008) Being the outer layer of the bacteria and in contact with the media, the bacterial envelope is the initial target for physical, chemical and biological (e.g. infection) stress. The envelope is also vital and involved in crucial functions such as respiration, adhesion, secretion, nutrient transport, osmotic pressure and bacterial integrity maintenance. (Bury-Moné et al., 2009) The inner and outer membrane maintaining the structure and many of the functions of the envelope are also extra susceptical to stresses such as hydrophobic toxins or abrupt changes in osmolarity, etc. Therefore, many stress responses of gram negative bacteria are dedicated primary or exclusively to preserving the integrity and function its envelope. 
A resent publication by Bury-Moné et al. (Sept 2009) has summarised the envelope stress signalling pathways in E.coli into five main individual pathways: Sigma-E, Psp, Cpx, Bae and Rcs. 
The Psp (Phage Shock Protein) pathway is not completely clarified but it is similar to the Sigma-E pathway in that both are regulated via sequestration/release of transcriptional factors depending on specific signals. PspF is an enhancer binding protein for Sigma-54. In the absence of stress signals, its function is inhibited by binding to PspA. It is believed that one or both proteins PspB and PspC bind to PspA to disrupt its inhibition of PspF following a not fully clarified stress signal.  
The Rcs pathway is comprised of a complex phosporelay signalling system. It was first discovered as a Regulator of colanic acid Capsule Synthesis, which is also the function that has given the regulatory system its name. It has later been discovered that the Rcs regulon include operons for envelope composition and that it also is activated by stresses affecting the peptidoglycan layer, whereby it contributes to antibiotic resistance. (Bury-Moné et al. 2009)    
</P><P><B>CpxA-CpxR envelope stress response</B></P><P>The CpxA-CpxR (Conjugative Plasmid eXpression) response functions as a two-component signal transduction system.
This type of sensor systems is among the most common in bacteria. Two component signal transduction systems have been found in all bacteria and also in some plants but are so far unknown of in animals. The general function of the class is made up of an auto-phosphorylating sensor histidine kinase, which phosphorylates a conserved histidine residue of its transmitter domain. Then the phosphate is transferred to a conserved aspartate of the receiver domain on the response regulator. The response regulator is then subsequently able to perform its particular function in the cell. 
The CpxA is the system’s sensor kinase which auto-phosphorylates when proteins designated for the outer membrane or secretion (e.g. pili or curli fibers) are mounting in the periplams. The concentration of proteins can be due to problems in transport or damage to the outer membrane. (Snyder and Champess 2007)
The phosphorylated CpxA  transfers its phosphate to CpxR which becomes activated as a DNA binding protein activating or increasing the transcription of more than 100 different genes. The genes activated are mostly chaperones and proteases involved in the refolding or degradation of proteins in the periplasm. DiGuiseppe and Silhavy (2003) have showed that of all the genes being induced by CpxR the operon cpxP (part of the cpx operon and involved in feedback inhibition) promoter is the strongest induced of them all. 
The CpxA-CpxR system also regulates the pore size in the membrane by increasing the transcription of ompC and repressing the transcription ompF with the effect that fewer toxins can get in through the smaller porin OmpC. OmpC and OmpF are the two major porin proteins in E.coli and their main function is to balance the osmotic pressure of the cell. They are otherwise regulated due to changes in the osmolarity via EnvZ and OmpR.  (Snyder and Champess 2007)
Yamamoto and Ishihama (2005) have demonstrated that CpxAR is being activated as a response to external copper.  
An example of an additional protein regulating the activity of CpxA is NlpE (an outer membrane lipoprotein), which increases activity following adhesion. (Bury-Moné et al., 2009)
GTAAANNNNNGTAAA has been proposed as the CpxR binding site. (Wulf et al. 2002) However, an examination of a large amount of CpxR induced operons by Price and Ravio (2009) indicates that the level of consensus with the proposed sequence or its orientation seems to play a minor role when determining the strength of induction by CpxR for a particular promoter. Of larger importance seems to be the location of the binding site. 
</P><P><B>BaeS-BaeR envelope stress response</B></P><P>The BaeSR (Bacterial Adaptative rEsponse) is another example of a two component signal transduction system and its signal pathway is the same as for CpxAR system.
It was first discovered as an envelope stress signal transduction pathway in E.coli by Raffa and Ravio (2002). BaeR was discovered as an additional transcription factor of the uncharacterized Spy gene, in addition to the previously known CpxAR system. BaeSR was showed to partly share stimuli with CpxAR as the two systems both are induced by PapG overexpression, spheroplast formation and indole. Indole is regarded as a putative inducer of the BaeSR system and as such it has also been used for investigations of how the BaeSR system responds and influences gene expression. (Nishino et al., 2005)
BaeSR stress induced activation in E.coli is less well characterized than the activation of the Cpx system and only 4 operons have been showed to be induced by BaeR. No negative regulation by the system has been found. BaeR is binding upstream of spy, arcD, ycaC and the mtd-bae operon. (Bury-Moné et al., 2009)
Baranova and Nikaido (2002) have showed that BaeR activates transcription of yegMNOB /mdtABCD (multidrug transporters) which increases e.coli’s resistance to Novobiocin and Deoxycholate. BaeSR seem to have a conserved function in at least one more gram negative bacteria as experiments by Nishino et al. (2007) have showed that the BaeSR complex is responsible for multidrug and metal resistance in Salmonella enterica. Yamamoto et al. (2008) has identified BaeRS two component system as a fifth regulon member of the zinc-responsive stimulon (adding to previously existing Zur, ZraSR, ZntR and NhaR). 
Analysis of the upstream region of spy, mdtA and arcD, by Nishino et al. (2005), has led to the identification of a 18bp long binding sequence for BaeR. It is located between -156 to -54 bp before initial transcription point and reads: 5’-TTTTTCTCCATDATTGGC-3’. Where D is any nucleotide G, A or T. 
Experiments on the spy promoter, by Yamamoto et al. (2008), identify the sequence TCTNCANAA as the BaeR-box. 
</P><P><B>The Extracytoplasmic function sigma, Sigma-E (24)</B></P><P>Sigma-E was first discovered as an alternative sigma factor recognised by one of the promoters for RpoH. However, it is mostly involved in transcription of genes with function in the periplasm, many of which are also part of the CpxA-CpxR regulon.  
Nevertheless, inactivation of the gene rpoE which encodes Sigma-E are lethal to cell at any temperatures and hence it is essential for the cell’s viability in general. The reason for this life determining role is still unknown.
The pathway of Sigma-E is activated when the periplasmic sensor protease DegS cleaves the anti-sigma factor RseA. This occurs after exposure of DegS to misfolded outer membrane proteins. Following the cleavage of RseA in the periplasm another inner-membrane bound protease YaeL (also named RseP) degrades the trans-membrane domain of RseA. The latter event releases the Sigma-E transcription factor in the cytoplasm and activates its large regulon, see illustration below. (Rhodius et al., 2005) Another periplasmic protein RseB is bound to the periplasmic domain of RseA to protect it from proteolysis. (Hayden and Ades., 2008)
</P><P>Sigma E can also be activated independently of misfolded OMPs by cytoplasmic alarmone ppGpp, which is dependent on nutrient availability. (Costanzo and Ades 2006) This mechanism is not illustrated on the graphical view above but is further explained below. 
</P><P><B>ppGpp</B></P><P>Costanzo and Ades (2006) have demonstrated that Sigma-E, independently of misfolded proteins, also is activated by the stringent response mediated by ppGpp.
ppGpp had previously only been known to activate Sigma-S and Sigma-54 following nutritional stress. 
When the cell is starved of an amino acid this nutritional stress is sensed and mediated by the stringent response. The stringent response has an impact on a vast amount of operons and including a strong down regulation of rRNA and tRNA to save energy. During exponential growth more than half of the RNA polymerases are involved in transcription of rRNA and tRNA so down regulation of these genes will save a lot of transcriptional power for alternative operons. 
If an uncharged tRNA binds to the A site in the ribosome this will cause the activation of ribosome-bound enzyme RelA (RELaxed control gene A). Activated RelA enzymes will synthesize ppGpp which then binds to two active sites of RNA polymerase. ppGpp’s function is not fully understood but the RNAP preferences for different promoter sequences change upon binding to ppGpp. 
In the presence of a sufficient amount of amino acids the protein SpoT normally degrades ppGpp an hence stops the stringent response.   
</P><P><B>DegP</B></P><P>DegP is dependent of Sigma E and can not be induced only by over expression of CpxR, in comparison the Spy promoter, which is dependent on sigma factor 70, can be induced up to 40 fold by over expression of CpxR. (Bury-Moné et al. 2009)
</P><P>(to be written)
</P><P><B>Spy</B></P><P>Spy (Spheroplast protein Y) was first discovered by Hagenmaier et al. (1997) as a protein being expressed exclusively in cells being subjected to spheroplasting stress. Raffa and Ravio (2002) demonstrated that Spy is, in addition to CpxAR, also being induced by the much smaller regulon BaeSR. BaeSR and CpxAR are also sharing some stimulon related to the periplasm, Screening of spy inducing metals by Yamamoto et al. (2008) has showed that Zn2+ is the metal causing the strongest induction of the operon over longer exposure, acting via the BaeSR system. However, the copper shock induction through CpxAR seems to mediate a quicker response. Bury-Moné et al. (2009) are showing that also RcsB (Regulator of colanic acid Capsule Synthesis) could induce expression of the Spy gene. 
Experiments, by Yamamoto et al. (2008), in which spy promoter fragments have been exposed to purified BaeR and acetyl phosphate have revealed a promoter-distal site between -162 and -137 and a promoter-proximal site between -109 and -79. The BaeR-box TCTNCANAA is present as a direct repeat in the promoter-distal site. The promoter-proximal site includes one single copy of the BaeR-box.
</P></DIV></DIV><DIV id="uclfooter"><P style="float:left; margin:30px 0px 20px 300px; font-size:12px; text-decoration: none;"><A href="#top">Back to top of Page</A></P><P style="width: 25%; font-size: 70%; text-align: right;float: left; color:#7e7e7e; padding-top: 55px; padding-right: 15px;">UCL London IGEM Wiki 2009</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:UCL_London/Project/Approach">http://2009.igem.org/Team:UCL_London/Project/Approach</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UCL_London/Project/Approach" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UCL_London/Project/Approach" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UCL_London/Project/Approach&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UCL_London/Project/Approach&amp;oldid=167001" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></BODY></HTML>