<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Tec-Monterrey_EKAM_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Tec-Monterrey_EKAM/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Tec-Monterrey_EKAM/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey_EKAM/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Tec-Monterrey_EKAM/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Tec-Monterrey_EKAM/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Tec-Monterrey EKAM/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><TABLE style="background-color:#white; white-space: pre-wrap;" align="center" cellpadding="3" cellspacing="1"><TBODY><TR><TH style="color:#463F70;padding: 10px; border:1px solid #fff; font-size:20pt;" rowspan="3">EKAM
</TH></TR><TR><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM" title="Team:Tec-Monterrey EKAM">Home</A></TH><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM/Team" title="Team:Tec-Monterrey EKAM/Team">Team</A></TH><TH style="background-color:#F1F2F9;"><A href="https://igem.org/Team.cgi?year=2012&amp;team_name=Tec-Monterrey_EKAM" class="external text" rel="nofollow">Profile</A></TH><TH style="background-color:#9E93DB; color:white"><STRONG class="selflink">Project</STRONG></TH><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM/Parts" title="Team:Tec-Monterrey EKAM/Parts">Parts</A></TH><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM/Modeling" title="Team:Tec-Monterrey EKAM/Modeling">Modeling</A></TH><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM/Notebook" title="Team:Tec-Monterrey EKAM/Notebook">Notebook</A></TH><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM/Safety" title="Team:Tec-Monterrey EKAM/Safety">Safety</A></TH><TH style="background-color:#F1F2F9;"><A href="/Team:Tec-Monterrey_EKAM/Attributions" title="Team:Tec-Monterrey EKAM/Attributions">Attributions</A></TH></TR></TBODY></TABLE><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1 tocsection-1"><A href="#Modular_Terpenoid_Biofactory"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Modular Terpenoid Biofactory</SPAN></A></LI><LI class="toclevel-2 tocsection-2"><A href="#Software_Tool"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Software Tool</SPAN></A></LI></UL><LI class="toclevel-1 tocsection-3"><A href="#Implementation_on_a_Platform_for_Terpenoid_Production"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Implementation on a Platform for Terpenoid Production</SPAN></A></LI><LI class="toclevel-1 tocsection-4"><A href="#Promoters_for_P._pastoris"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Promoters for P. pastoris</SPAN></A></LI><LI class="toclevel-1 tocsection-5"><A href="#Human_Practices"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Human Practices</SPAN></A></LI><LI class="toclevel-2 tocsection-6"><A href="#SyBRE"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">SyBRE</SPAN></A></LI><LI class="toclevel-2 tocsection-7"><A href="#iGEM_H.S."><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">iGEM H.S.</SPAN></A></LI><LI class="toclevel-2 tocsection-8"><A href="#DETECT"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">DETECT</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="Modular_Terpenoid_Biofactory"><B>Modular Terpenoid Biofactory</B></SPAN></H2><P>Ever since the potential of genetic engineering became apparent, we have seen a rise in the use of synthetic biology to develop full-fledged systems working for the very interests of mankind. The idea that a basic system could potentially be used in the production of a whole range of molecules suggests that this kind of multi-directed manufacturing could be designed to optimally synthesize any given product at a given time. This is when the idea of a modular biofactory comes up, and the proper strategies for its design and execution were the focus of the project.
</P><P>Starting from the assumption that a synthetic biology project is founded on the design of the proper genetic construct being transformed into an organism, it follows that most of the versatility of the project rests on the proper planning strategies applied to the DNA molecule. This would allow for distinct parts of the construct to be exchanged, added, or eliminated, depending on the development and focus that is ultimately given to the entire project.
</P><P>Considering restriction enzymes as the basic tool for molecular cloning, it is possible to strategically locate the proper restriction sites in a genetic construct for the entire cloning process to be optimized. The proper design can come up with a master molecule from which a great number of sequence combinations can be derived using a limited number of enzymes. If certain restriction site pairs are assigned to each specific module, it would enable the future manipulation of each of these specific parts separately in the most efficient way possible.
</P><P>The ability to manipulate specific portions of the genetic constructs opens up possibilities in the project execution that allow it to be more dynamic and adaptable. When working with a synthesized construct, one of the barriers for its proper realization is the high costs of producing long nucleotide sequences by oligonucleotide synthesis. This modular design strategy, by allowing to make multiple sequence combinations of the already synthesized construct, enables to study different sequence setups without having to synthesize each specific iteration separately, thus cutting a great amount of costs from the project.
</P><P>Obviously, this can and has been achieved by working with digestion and ligation reactions, but multiple reactions convey an additional expense on reactants, mostly enzymes. Because a desired sequence exchange could be done by a single digestion and ligation, the modular strategy also promises a reduction on costs in this area. Even further, oligonucleotide synthesis has a limited sequence length that can be reached without errors, so larger constructs require the use of subcloning, which further expands the project’s cost. This is when the In-Fusion tool comes in handy, as it makes it possible, through flanking site recombination, to construct precise DNA molecules by joining several fragments. This allows for the synthesis of separate sequences to be ordered and then these fragments to be used in a single reaction that will end up with the final desired construct, with the modular design strategy implemented.
</P><P>Having the option to easily manipulate different sections of the genetic construct makes it possible to debug a genetically engineered system by switching the involved components and using trial and error, so malfunctioning parts can be edited. This property also makes it easier to characterize certain parts by enabling the use of different expression assemblies. A module can be constituted by a switchable promoter or reporter gene, for example, offering the opportunity to reuse systems that have been proved useful to study new single parts.
</P><H3><SPAN class="mw-headline" id="Software_Tool"><B>Software Tool</B></SPAN></H3><P>As a tool for the implementation of this construct design strategy, a software application was developed to aid in the appropriate consideration and placing of restriction sites in the master molecule, enabling a simultaneous consideration of the multiple factors affecting the efficiency of the final design. Enzyme compatibility, including digestion reaction conditions and cohesive end affinity, as well as their availability and other characteristics, are taken into account as the user points out the limits they wish to define for each module, so a future manipulation of the fragment can be carried out with as little enzyme as possible.
</P><H2><SPAN class="mw-headline" id="Implementation_on_a_Platform_for_Terpenoid_Production"><B>Implementation on a Platform for Terpenoid Production</B></SPAN></H2><P>The modular design strategy obviously poses a greater advantage to systems aimed at manufacturing related products, such as families of biomolecules, related to each other by common metabolic pathways of production. One such family of biomolecules is terpenoids, also known as isoprenoids, which are a kind of hydrocarbon made up of five-carbon isoprene units, arranged and modified in such a way that all of them share common physical and chemical characteristics and yet also exhibit a wide range of biological properties. 
</P><P>Well-known cases are artemisinin, the drug with the most effective mechanism against malaria-causing <I>P. falciparum</I>, and taxol, a mitotic inhibitor used in chemotherapeutic cancer treatments. Both of them are produced in plants (a fern-like shrub and the bark of a tree, respectively), but their yields are usually low and the demand for their medical use is significantly high. Other examples include lycopene, the precursor in tomatoes for beta-carotene, and limonene, which is present in most cosmetic products distributed nowadays.
</P><P>The biological pathway through which terpenoids are synthesized in eukaryotes is the mevalonate pathway, which includes a series of molecular transformations from Acetyl-CoA to isopentenyl-5-pyrophosphate and dimethylallyl-pyrophosphate. These isomeric molecules are the common precursors for all terpenoid molecules. A bioengineering approach for terpenoid production involves using the mevalonate pathway and the appropriate synthases in order to end up with the desired molecule. The yeast <I>P. pastoris</I> already uses the mevalonate pathway, but it has been reported that the HMG-CoA reductase involved in its rate-limiting step is heavily regulated. This can be bypassed by a reported N-terminus truncation that renders it still functional but increases the total yield of the process.
</P><P>Applying the aforementioned modular biofactory design strategy to this model, it is possible to implement a biological platform for the synthesis of terpenoids. Using the production of lycopene as a proof of concept, the separate genes required include the truncated HMG-CoA reductase, GGPP synthase (CrtE), phytoene synthase (CrtB), and phytoene desaturase (CrtI), each with a transcription promoter, RBS and transcription terminator.
</P><P>Additionally, another module may be implemented to further optimize the platform for terpenoid production. P. pastoris utilizes a great part of the molecules produced in the mevalonate pathway to feed its sterol pathway, starting with the production of squalene. Silencing the squalene synthase gene, ERG9, would enable the use of precursors for terpenoid production.
</P><P>Although <I>P. pastoris</I> has no known RNAi mechanism, it has been shown that introducing genes for Dicer and Argonaute from <I>Saccharomyces castellii</I> successfully provides <I>S. cerevisiae</I> with the silencing mechanism. Introducing the AGO1 and DCR1 genes, together with an antisense sequence for the ERG9 gene, a separate module for the platform is generated.
</P><H2><SPAN class="mw-headline" id="Promoters_for_P._pastoris"> Promoters for <I>P. pastoris</I></SPAN></H2><P>When working with <I>P. pastoris</I> as an expression system, a need for a controlled expression evidences a lack of related parts in the registry. Therefore, the characterization of four inducible promoters is presented.
The registry’s insufficiency of expression tools for alternative systems like this yeast is thus addressed. <I>P. pastoris</I> offers an expression system that has proven useful in substantial protein production, especially in molecules requiring post-translational modifications. Cultures of this yeast are able to achieve relatively large cell densities, allowing greater amounts of biomass to be used for production. Also, this organism is non-pathogenic and regarded as biologically safe, making it a relevant alternative for expression systems.
</P><P>Using GFP as a reporter for the strength of each promoter, the detected fluorescence is registered as proportional to the level of expression of the gene. The emitted radiation can be quantified by a fluorescence microplate reader, as well as by fluorescence microscopy, using specialized software (ImageJ) for an objective appraisal of the images.
The results here presented reveal the successful implementation of the promoters in the system and the functionality of this product of gene expression. Further characterization includes the detailed study of the effect of inducer concentration in the medium over time.
</P><P>We are aware of the characteristics of GFP that make it a non-ideal reporter chromophore. The availability of molecular oxygen in its surroundings is a source of variability in its fluorescent activity. Also, the correct folding of the protein takes a certain amount of time and also depends on the medium’s conditions, inserting noise into the results. However, GFP provides an easy way to track gene expression in simpler systems. The data to be collected on promoter strength is purely relative and specific for its setup, providing a general baseline for their usefulness on other projects.
</P><P>The availability of inducible promoters for a biosystem is useful when a tight grip on expression of the genes of interest is desired. Applying the modular biofatory design strategy, they come extremely handy if the purpose is for each module to be expressed at different times. It would be possible to construct a mathematical model quantifying the effect of each of the separate modules using distinct promoters for each group of genes associated to a specific function of the genetic construct.
</P><H2><SPAN class="mw-headline" id="Human_Practices"><B>Human Practices</B></SPAN></H2><H3><SPAN class="mw-headline" id="SyBRE"><A href="/Team:Tec-Monterrey_EKAM/Sybre" title="Team:Tec-Monterrey EKAM/Sybre"><B>SyBRE</B></A></SPAN></H3><P>We decided to institutionalize the school’s participation in iGEM and further promote synthetic biology in our university through an official student group: SyBRE (Synthetic Biology Research and Engineering).
</P><H3><SPAN class="mw-headline" id="iGEM_H.S."><A href="/Team:Tec-Monterrey_EKAM/iGEMJunior" title="Team:Tec-Monterrey EKAM/iGEMJunior"><B>iGEM H.S.</B></A></SPAN></H3><P>We took on the task to promote the iGEM High School competition on our university’s high school campus, <A href="https://2012hs.igem.org/Team:PrepaTec_GarzaSadaMx" class="external text" rel="nofollow">Preparatoria Eugenio Garza Sada</A>.
</P><H3><SPAN class="mw-headline" id="DETECT"><A href="/Team:Tec-Monterrey_EKAM/Detect" title="Team:Tec-Monterrey EKAM/Detect"><B>DETECT</B></A></SPAN></H3><P>Through SyBRE, we’ve developed a community service called DETECT (Spanish acronym for ‘Detection of Talented Students’), aimed to detect gifted children and bring them to the Biology and Chemistry areas.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:Tec-Monterrey_EKAM/Project">http://2012.igem.org/Team:Tec-Monterrey_EKAM/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Tec-Monterrey_EKAM/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Tec-Monterrey_EKAM/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Tec-Monterrey_EKAM/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Tec-Monterrey_EKAM/Project&amp;oldid=268483" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>