<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCAU-China_Demonstrate skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCAU-China/Demonstrate</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-default"><DIV class="container"><DIV class="navbar-header"><A href="https://2018.igem.org/Team:SCAU-China" class="navbar-brand">SCAU-2018</A></DIV><DIV class="navbar"><UL style="float: left;" class="nav"><LI class="dropdown"><A href="javascript:void(0)">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="javascript:void(0)">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/ProjectOverview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/SRK">Synergistic Recombination Kit</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/MM">Mathematical Model of Biological Intrinsic Regulation System</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Type">Type II CRISPR/Cas 9 Kit</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/MOM">Method for Optimizing Microbial Cell Culture</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Outlook">Outlook</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A href="javascript:void(0)">LAB WORK</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Measurement">Measurement</A></LI><LI class="dropdown"><A href="javascript:void(0)">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model/HAWNA">HAWNA</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model/PPIBoost">PPIBoost</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model/CultrueCondition">Cultrue Condition</A></LI><UL style="float: right;" class="nav"><LI class="dropdown"><A href="javascript:void(0)">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Safety">Safety</A></LI></UL><LI class="dropdown"><A href="javascript:void(0)">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Integrated">Integrated</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Public_Engagement">Public Engagement &amp; Education</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV id="mainDiv"><DIV class="heart"><DIV id="mask"><DIV class="DBoard" id="title">    
       Demonstrate
      </DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:left;"> 
       1.Genetic transformation of cyanobacteria</DIV><DIV class="DBoard" id="article1"><P>We have successfully over-produced the cellulose in the transgenic cyanobacteria. The glucose (bacterial cellulose digestion produce) content of the transgenic is much higher that the WT (Figure 2). There was significant differences between and wild-type. Unfortunately, the growth of the transgenic stain was much slower than the WT (Figure 1). 
</P></DIV><DIV class="DBoard" id="article2" align="center"><P style="font-size:16px; text-align: center;">Figure 1 Transferred cyanobacteria (the left) and the wail type (the right).The transferred cyanobacteria is inhibited in the growth (in yellow color) but not in cellulose production.
Here is the measurement of cyanobacteria glucose (3 repeats). The same of amount of transgenic cyanobacteria with bcsZH-ABCD-bglX genes and wild-type were treated with lysozyme to break the cells. 
</P></DIV><DIV class="DBoard" id="article1"><P>The differences between the red column and blue column indicated that the content of bacteria cellulose.</P></DIV><DIV class="DBoard" id="article2" align="center" style="height:535px;"><P style="font-size:16px; text-align: center;">Figure 2 The measurement of cellulose content in transgenic cyanobacteria with bcs gene.</P><P>These results indicate that the bacterial cellulose can be over-expressed and accumulated in the cyanobacteria. We need to reduce the negative impact of gene over-expression on the normal metabolism in cyanobacteria and meanwhile further increase the production levels of bacterial cellulose.</P></DIV><DIV class="DBoard" id="title" style="height:40px;width:660px;font-size:20px;line-height:40px;text-align:center;">    
     2.Mathematical Model of Biological Intrinsic Regulation System (HAWNA)
      </DIV><DIV class="DBoard" id="article1"><P>To better address the problems we encountered above when over-producing cellulose in cyanobacteria, we took advantage of computer modeling and system biology and apply them into our synthetic biology experiments. We aim to mimic gene expression networks to identify genes involved in the regulation of cellulose gene expression. However, the gene regulatory network in living organisms is extremely complex, and any modification of the genetic node can result in changes in the entire metabolic network. Therefore, we have developed a system biology algorithms called HAWNA (Hierarchical and Weighted Network Analysis). It can simulates the gene expression network of an organism based on gene expression profiling datas. By discovering the interaction in the exogenous expression system and classis organisms gene regulatory network,we may can ues HAWNA to predict the relatively known gene regulatory networks ,to some extent,can provide some help to in-depth study in the nosiy expression network.
</P></DIV><DIV class="DBoard" id="article2" align="center"><P style="font-size:16px; text-align: center;">Figure 3 The structure of HAWNA </P></DIV><DIV class="DBoard" id="title" style="height:40px;width:660px;font-size:20px;line-height:40px;text-align:center;">    
     3. PPI network prediction
      </DIV><DIV class="DBoard" id="article1"><P>We proposed a novel protein-protein interaction prediction method based on the research of Shen et al. 2007 published on PNAS. This method integrates the functional annotation data, transcriptome(microarray), and protein sequences information, to predict PPI of PCC6803 in high performance.10-fold validation demonstrate that our model is out perform than S kernel SVM proposed by Shen</P></DIV><DIV class="DBoard" id="title" style="height:40px;width:660px;font-size:20px;line-height:40px;text-align:center;">    
     4.   Measurement
      </DIV><DIV class="DBoard" id="article1"><P>
           Our team proposed a new computational method to predict the intensity of promoters in E.coil, and efficiently identify their most importance sequences or units which acted as the core element to determine the activities of promoters.For this purpose, we combined sequence data and a new feature encoding method with machine learning algorithm to predict the promoter strength and consensus unit or core site may have a strong correlation to the activities of the promoter. Our GFP experiment demonstrate our model can achieve reasonable prediction of promoter intensity.
       </P></DIV><DIV class="DBoard" id="title" style="height:40px;width:660px;font-size:20px;line-height:40px;text-align:center;">    
    5.  Method for optimizing Microbial cell culture
      </DIV><DIV class="DBoard" id="article1"><P style="font-size:16px; text-align: center;">In vitro, based on measured data tracing Synechocystis growth, we constructed a double layer of model to characterize the main parameters influencing the behavior of our system. We simulated about 1875 combinations, and found the most suitable conditions which have the best K and R value compare with other schemes.</P></DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:center;">    
     6. Synergistic Recombination and Type-II CRISPR/Cas9 Kit for Synechocystis and Microcoleus 
      </DIV><DIV class="DBoard" id="article1"><P>We verified through experiments that this Synergistic Recombination works well in Synechocystis and Microcoleus, with Prbc, Trbc and slr0168. It is conceptual and in dynamic change, expected to be applied to a wider range of fields. 
 
</P></DIV><DIV class="DBoard" id="article2" align="center"><P style="font-size:16px; text-align: center;">Figure 9 PCR amplifying of homologous genes slr0168 in Synechocystis and Microcoleus </P><P>In addition, we designed a type II CRISPR/Cas9 kit based on the Synergistic Recombination kit to knock out slr0163/slr0753/slr0897/slr0977/slr127/cesA
 </P></DIV><DIV class="DBoard" id="article2" align="center"><P style="font-size:16px; text-align: center;">Figure 10 Enzyme digestion to conform the successful construction of cas 9 knocking out kit.</P></DIV><DIV class="DBoard" id="article2" align="center"><P style="font-size:16px; text-align: center;">Figure 11 Colony PCR result of slr0163/slr0753/slr0897/slr0977/slr127/cesA in the Type-II CRISPR/Cas9 Kit</P></DIV><DIV class="DBoard" id="article1"><P>Taken together, we achieved bacterial cellulose expression in Synechocystis sp.PCC6803 and Microcoleus vaginatus FACHB-253 by a synergistic recombination kit, which can be further used as a brand new strategy for desertification control from synthetic biology aspect of view. In order to increase the cellulose production we put forward a Mathematical Model of Biological Intrinsic Regulation System and a Menthod for Optimizing Microbial Cell Culture. Meanwhile,we modified the synergistic recombination kit into the type-II CRISPR/Cas9 kit for Synechocystis and Microcoleus, expected to be applied to a wider range of fields. Cyanobacteria can be regarded as an ideal bioreactor for water retention in desert for desertification control. 
</P></DIV><P>School's name:SCAU</P><P>Member's name:SCAU</P><P>Designed by:SCAU</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>