<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_SCAU-China_Experiments skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:SCAU-China/Experiments</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV class="navbar-default"><DIV class="container"><DIV class="navbar-header"><A href="https://2018.igem.org/Team:SCAU-China" class="navbar-brand">SCAU-2018</A></DIV><DIV class="navbar"><UL style="float: left;" class="nav"><LI class="dropdown"><A href="javascript:void(0)">TEAM</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Members">Members</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Attributions">Attributions</A></LI></UL><LI class="dropdown"><A href="javascript:void(0)">PROJECT</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/ProjectOverview">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Background">Background</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Design">Design</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/SRK">Synergistic Recombination Kit</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/MM">Mathematical Model of Biological Intrinsic Regulation System</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Type">Type II CRISPR/Cas 9 Kit</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/MOM">Method for Optimizing Microbial Cell Culture</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Outlook">Outlook</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Demonstrate">Demonstrate</A></LI><LI class="dropdown"><A href="javascript:void(0)">LAB WORK</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Experiments">Experiments</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Parts">Parts</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Improve">Improve</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/InterLab">Interlab</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Measurement">Measurement</A></LI><LI class="dropdown"><A href="javascript:void(0)">MODEL</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model/HAWNA">HAWNA</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model/PPIBoost">PPIBoost</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Model/CultrueCondition">Cultrue Condition</A></LI><UL style="float: right;" class="nav"><LI class="dropdown"><A href="javascript:void(0)">SAFETY</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Safety">Safety</A></LI></UL><LI class="dropdown"><A href="javascript:void(0)">HUMAN PRACTICES</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Human_Practices">Overview</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/silver">Silver</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Integrated">Integrated</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Public_Engagement">Public Engagement &amp; Education</A></LI><LI><A href="https://2018.igem.org/Team:SCAU-China/Collaborations">Collaborations</A></LI></DIV></DIV></DIV><DIV id="mainDiv"><DIV class="heart"><DIV id="mask"><DIV class="DBoard" id="title">    
       Experiments
      </DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:center;">    
    Acetobacter xylinus.Protocols
      </DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:center;">    
   Acetobacter xylinus. culturing media and culturing
      </DIV><DIV class="DBoard" id="article1" style="height:700px;"><P style="font-size: 22px;color:black;">Materials：</P><P style="font-size: 18px">For 500ml media:</P><P style="font-size: 18px">10g glucose (2%w/v)</P><P style="font-size: 18px">2.5g yeast extract (0.5% w/v)</P><P style="font-size: 18px">2.5g peptone (0.5% w/v)</P><P style="font-size: 18px">1.35g Na2HPO4 (0.27% w/v)</P><P style="font-size: 18px">0.75g citric acid (0.15% w/v)</P><P style="font-size: 18px">500ml distilled H20</P><P style="font-size: 18px">7.5g of agar if making agar plates</P><P style="font-size: 18px">Add 250ml dH20 to glucose in one bottle and 250ml dH20 to the rest in a second bottle. In incompletely distilled water, glucose will form a solid mass, so stir vigorously immediately after adding water. Autoclave both bottles to sterilize media and pour glucose solution in sterile conditions (next to a Bunsen burner or in a flow hood) into the second bottle. Autoclaving glucose separately from amino acids avoids Maillard reaction, which can result in the formation of toxic byproducts in the media.</P><P style="font-size: 18px">Streak/inoculate Acetobacter xylinus. to plates or into media.</P><P style="font-size: 18px">Incubate plates at 30°C inverted. Colonies will appear in 48-72 hours.</P><P style="font-size: 18px">Incubate liquid cultures at 30 °C standing. For quick growth, grow with shaking at 180rpm. </P></DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:center;">    
  CTAB Genomic DNA extraction from Acetobacter xylinus.
      </DIV><DIV class="DBoard" id="article1" style="height:1400px;"><P style="font-size: 22px;color:black;">Materials：</P><P style="font-size: 18px">CTAB  solution</P><P style="font-size: 18px">- 100mM Tris-HCl (pH 8.0) </P><P style="font-size: 18px">- 1.4M NaCl </P><P style="font-size: 18px">- 20mM EDTA </P><P style="font-size: 18px">- 2% CTAB (w/v) </P><P style="font-size: 18px">- 1% PVP-360 (optional) </P><P style="font-size: 18px">- 1% PVP-360 (optional) </P><P style="font-size: 18px">24-:1 CHCl3 : isoamyl alcohol (100ml: 96ml CHCl3 + 4ml isoamyl alcohol)</P><P style="font-size: 18px">Liquid nitrogen </P><P style="font-size: 18px">1. Transfer 2 ml xxx culture into 2ml eppendorf tube</P><P style="font-size: 18px">2. Centrifuge at 1200rpm for 90sat 4°C , remove the supernatant as much as possible</P><P style="font-size: 18px">3. Freeze in liquid nitrogen</P><P style="font-size: 18px">4. Add 5 drops CTAB solution with β-mer (0.2% v/v) and a pinch of glass powder </P><P style="font-size: 18px">5. Grind the freeze bacteria into homogenate with glass rod </P><P style="font-size: 18px">6. 500μl CTAB solution with β-mer </P><P style="font-size: 18px">7. Incubate 60°C 30-40mins with periodic mixing </P><P style="font-size: 18px">8. 500μl 24:1 CHCl3 : isoamyl alcohol, mix thoroughly but gently for 5mins </P><P style="font-size: 18px">9. Spin 1200rpm at 4°C for 15mins </P><P style="font-size: 18px">10. Transfer aqueous phase to new tube </P><P style="font-size: 18px">11. 500μl cold isopropanol, mix and incubate at -20°C for 1hr </P><P style="font-size: 18px">12. Spin 1200rpm at 4°C for 20mins  </P><P style="font-size: 18px">13. Remove supernatant, wash pellet with 70% EtOH </P><P style="font-size: 18px">14. Spin 1200rpm at 4°C for 10mins  </P><P style="font-size: 18px">15. Remove supernatant, dry the pellet under vacuum </P><P style="font-size: 18px">16. Dissolve in TE / ddH20 with 10μg/ml RNaseA at 37°C for 1hr </P><P style="font-size: 18px">17. Incubate at 65°C for 10min to kill the DNase </P><P style="font-size: 18px">18. Store at 4 / -20°C  </P></DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:center;">    
   E. coli Protocols
      </DIV><DIV class="DBoard" id="article1" style="height:4900px;"><P style="font-size: 22px; color:black;">Molecular cloning workflow</P><P style="font-size: 18px">1. PCR was often used to generate the insert fragments, for example to add eGFP fragment to pSB1C3 plasmid, or to clone out a desired coding sequence for a new biobrick. In each case primers were designed with overhangs containing the desired prefix and suffix sequences. PCR reactions were set up using KOD plus or Q5 (NEB) polymerases. PCR programs varied but typically used a   protocol which improves accuracy of primer binding at the beginning of the reaction due to the higher temperatures. Ran ~10% of the PCR product on agarose gel to confirm it's success, then replicates were pooled and PCR purified (Axygen kit)</P><P style="font-size: 18px">2. Insert fragment and vector’s backbone DNA were digested with the appropriate restriction enzymes</P><P style="font-size: 18px">3. Products were Gel-extracted or PCR purified</P><P style="font-size: 18px">4. Vector backbone may be de-phosphorylated to prevent background re-ligation, setting up reaction as per manufacturers recommendation, then heat-kill the enzyme
Optional: PCR purify to remove enzyme and buffer
</P><P style="font-size: 18px">5. Ligation reactions for the desired combinations of insert and backbone were set up, including negative control reactions which contain no insert DNA. </P><P style="font-size: 18px">6. Transformed with ~2-4 ul ligation reaction into  DH5-alpha cells using the general heat shock protocol as explained previously</P><P style="font-size: 18px">7. Picked colonies and inoculated mini-prep cultures, instead of picking cultures for mini-prepping, colony PCR was used when large numbers of colonies needed to be screened. And we used the forward primer binding at the backbone and reverse primer binding at inserting DNA to run PCR program and confirm it's success. Then only the positive clones were mini-prepped.</P><P style="font-size: 18px">8. Positive clones were sent for sequencing of the insert using appropriate primers.
Add DNA to 50 ul cells on ice (no more than 5 ul, i.e. no more than 10% volume of cells)
</P><P style="font-size: 18px" style="color:black;">To autoclave:</P><P style="font-size: 18px">500 mL LB</P><P style="font-size: 18px">500 mL Water</P><P style="font-size: 18px">50 mL 10% glycerol</P><P style="font-size: 22px; color:black;">Conical Flasks</P><P style="font-size: 18px">1. To grow: 5 mL overnight culture containing the required antibiotic, grow under shaking conditions at 37 degC</P><P style="font-size: 18px">2. Prepare Eppendorf tubes and keep in the -80 °C freezer until required</P><P style="font-size: 18px">3. Inoculate the autoclaved flasks with 50 mL LB</P><P style="font-size: 18px">4. Add 500 ul of overnight culture into 50 the conical flasks and provide specific antibiotic, if required</P><P style="font-size: 18px">5. Grow for ~1 h and then start taking OD 600 nm readings every half hour. When OD reaches 0.5, proceed to the next step.</P><P style="font-size: 18px">6. Pour culture into falcon tube</P><P style="font-size: 18px">7. Centrifuge for 10 minutes at 4000 rpm and at 4°C </P><P style="font-size: 18px">8. Discard supernatant and use blue roll remove any left overs.</P><P style="font-size: 18px">9. Add 800 ul of previously chilled, autoclaved water, resuspend cells, then add 9.2 mL to make it up to 10 mL</P><P style="font-size: 18px">10. Centrifuge for 10 minutes at 4000 rpm and at 4 °C </P><P style="font-size: 18px">11. Place tubes on ice. Remove supernate. Gently resuspend each cell pellet in 1ml of ice-cold 10% glycerol. Final OD600 of resuspended cells is about 200-250.</P><P style="font-size: 18px">11. Place tubes on ice. Remove supernate. Gently resuspend each cell pellet in 1ml of ice-cold 10% glycerol. Final OD600 of resuspended cells is about 200-250.</P><P style="font-size: 22px; color:black;">Preparation of chemically competent E. coli cells</P><P style="font-size: 18px">1. Inoculate 2 ml LB broth with an aliquot (~50 ul) of the desired E. coli from the -80degC freezer stock of cells.</P><P style="font-size: 18px">2. Incubate for 2h at 37°C</P><P style="font-size: 18px">3. Add the 2 ml seed culture to 250 ml LB broth and grow at 37degC, shaking (~200 rpm) until OD 600 of 0.3 (~5 hours)</P><P style="font-size: 18px">4. Centrifuge at 4°C, (in our case 3000 rpm in Heraeus megafuge, Thermo) for 10 minutes</P><P style="font-size: 18px">5. Discard supernatant, then resuspend in 80 ml ice cold CCMB80 buffer (it is easier to resuspend in 1 ml first using a Gilson pipette, then add buffer to the required volume)</P><P style="font-size: 18px">6. Place in ice for 20 minutes</P><P style="font-size: 18px">7. Centrifuge 4°C C and discard supernatant</P><P style="font-size: 18px">8. Resuspend in 10 ml CCMB80 buffer</P><P style="font-size: 18px">9. Test OD 600 of 200 ul SOC media with 50 ul resuspended cells and based on this calculate the amount of CCMB80 buffer needed to add to the resuspended cells to achieve a final yield of OD 600 1.0-1.5.</P><P style="font-size: 18px">10. Aliquot in volumes as desired (for us ~250 ul) then store at -80 °C</P><P style="font-size: 22px; color:black;">Gibson Assembly</P><P style="font-size: 18px">1.prepare 5 X ISO buffer.Six mL of this buffer can be prepared by combining the following:</P><P style="font-size: 18px">1M Tris-HCL(Ph7.5)   3mL</P><P style="font-size: 18px">2M MgCl2   150uL</P><P style="font-size: 18px">100mM dNTP   600uL</P><P style="font-size: 18px">1M DTT     300uL</P><P style="font-size: 18px">PEG-8000    1.5g</P><P style="font-size: 18px">100mM NAD    300uL</P><P style="font-size: 18px">demineralised  water  to 6ml</P><P style="font-size: 18px">Aliquot 100 ul and store at -20°C</P><P style="font-size: 18px">2.prepare a  2 X assembly master mixture.This can be prepared by combing the following:</P><P style="font-size: 18px">5 x ISO      320uL</P><P style="font-size: 18px">T5 exonuclease    0.64uL</P><P style="font-size: 18px">Phusion polymerase 20uL</P><P style="font-size: 18px">Taq ligase    160uL</P><P style="font-size: 18px">demineralised  water to 1.2mL</P><P style="font-size: 18px">Aliquot 15uL and store at -20°C.This assembly mixture is ideal for the assembly of DNA molecules with20-150bp overlaps.</P><P style="font-size: 18px">3. Assembly Reation</P><P style="font-size: 18px">The DNA should be in equimolar amounts.Use 10-100ng for each ~6kbDNA fragment.For larger DNA segments, increasingly proportionate amounts of DNA should be added (eg.250ng of each 150kb DNA segment)  </P><P style="font-size: 18px">Incubate at 50°C for 40 to 60min（60 min is optimal）.If cloning is desired，electroporate 1 uL of the assembly reation into 30 uL electrocompetent E.coli.</P><P style="font-size: 22px; color:black;">Heat –Shock transformation</P><P style="font-size: 18px">1. Incubate on ice 15-30 min</P><P style="font-size: 18px">2. Heat shock 42°C，  45 s </P><P style="font-size: 18px">3. Place samples back on ice for 2 minutes </P><P style="font-size: 18px">4. Add 200 ul LB, or up to 10x volume of the cells</P><P style="font-size: 18px">5. Incubate at 37°C for 60 minutes, shaking </P><P style="font-size: 18px">6. Optional: Spin down cells, discard supernatant and resuspend in 100-200 ul LB to concentrate</P><P style="font-size: 18px">7. Plate out cells on LB agar, maximum 200 ul</P><P style="font-size: 18px">8. Incubate at 37°C overnight.</P><P style="font-size: 22px; color:black;">LB media:</P><P style="font-size: 18px">1L demineralised water add:</P><P style="font-size: 18px">Tryptone         10g</P><P style="font-size: 18px">Yeast extract             5g</P><P style="font-size: 18px">Sodium Chloride      10g</P><P style="font-size: 18px">5g of agar if making agar plates</P><P style="font-size: 18px">Autoclave</P><P style="font-size: 22px; color:black;">10X TBE Buffer:</P><P style="font-size: 18px">For 1L :</P><P style="font-size: 18px">Tris                         108g</P><P style="font-size: 18px">Acetic acid             55g</P><P style="font-size: 18px">0.5M EDTA            20ml</P><P style="font-size: 22px; color:black;">1% Agarose Gel:</P><P style="font-size: 22px; color:black;">Material:</P><P style="font-size: 18px">1g Agarose</P><P style="font-size: 18px">100mL 1X TAE buffer</P><P style="font-size: 18px">8uL SYBR Safe</P><P style="font-size: 18px">1. Mix Agarose and 1x TAE buffer</P><P style="font-size: 18px">2. Heat up until Agarose is dissolved</P><P style="font-size: 18px">3. Add SYBR Safe DNA stain</P><P style="font-size: 18px">4. Pour into gel tray and let cool</P><P style="font-size: 22px; color:black;">Agarose Gel Electrophoresis</P><P style="font-size: 22px; color:black;">Material:</P><P style="font-size: 18px">1% Agarose gel DNA ladder</P><P style="font-size: 18px">6x loading dye</P><P style="font-size: 18px">Electrophoresis cuvette</P><P style="font-size: 18px">1. Set gel tray into cuvette, filled with 1x TAE buffer</P><P style="font-size: 18px">1. Set gel tray into cuvette, filled with 1x TAE buffer</P><P style="font-size: 18px">3. Run gel at 120V for 20-30min</P></DIV><DIV class="DBoard" id="title" style="height:40px;width:400px;font-size:20px;line-height:40px;text-align:center;">    
   Cyanobacteria Protocols
      </DIV><DIV class="DBoard" id="article1" style="height:3400px;"><P style="font-size: 22px; color:black;">Cyanobacteria culture</P><P style="font-size: 18px">We use BG-11 Medium for Blue Green Algae.The BG-11 recipe is as follow:</P><P style="font-size: 18px">1.Prepare 100 X Citrate mixture and 1000X A5 mixture.</P><P style="font-size: 18px">The prescripition formula are as follow:</P><P style="font-size: 18px">2、Dosing the reagent according to the following sheet</P><P style="font-size: 22px; color:black;">Cyanobacteria Transformation</P><P style="font-size: 22px; color:green;">1.Synechocystis sp. </P><P style="font-size: 18px">Stable transformation of Synechocystis sp. is achieved via the uptake of DNA and incorporation into the host genome by homologous double recombination.Before transformation,our team use a special menthod to make the Synechocystis sp.  permeable.</P><P style="font-size: 22px; color:black;">Reagents preparation</P><P style="font-size: 18px">Prepare the Tes buffer(PH7.3)、permeable solution(PH7.3)、fresh BG11, The permeable solution composition is as follow.</P><P style="font-size: 22px; color:black;">Preparation of permeable cells</P><P style="font-size: 18px">Inoculated 10 ml of fresh Synechocystis sp.   cells into 100 ml of BG11 medium, and adjusted OD730 to 0.2 ，light overnight. On the next day, the cells were collected by centrifugation（7000rpm/min，10min）. Wash the sediment with 50 mmol/L Tes (N-tris(hydroxymethyl)methyl-2-aminoethanesulfonic acid) (pH 7.3) buffer, and resuspend it in 10 ml of 50 mmol Tes (pH 7.3) buffer which containing 1 mmol/LEDTA and 2 mg/ml lysozyme. And then，place it in a 100ml flask, shake on a shaker at 60r/min, control the temperature at 36 °C, and after 2 hours of light, take 2ml sample and transfer 20ml ice cold 50mmol / In L Tes (pH 7.3), centrifuge immediately. The pellet was resuspended in approximately 10 ml of fresh BG11 medium to give a solution OD730 = 0.5 (2 x 10^5 cells/ml). This prepared permeabilized cells were used for transformation.</P><P style="font-size: 22px; color:black;">Transformation</P><P style="font-size: 18px">Transformation 1 ml of donor DNA in lml permeable cells, placed in a 50ml flask, illuminated at 30 ° C，incubate for 24 to 32 hours in the incubator. Before applying the permeabilized cells to the solid medium containing antibiotics (spectinomycin10ng/mL), they should be inoculated into a liquid medium containing half of the antibiotics for 2 to 3days and then transferred to a completely resistant solid medium. </P><P style="font-size: 22px; color:green;">2.Microcolus vaginatus</P><P style="font-size: 18px">Microcolus vaginatus have been widely used in algae crusts but have not been used as a chassis. Due to the presence of a colloidal sheath outside the cell wall of the micro-sphingidae, the natural transformation efficiency is low in Microcolus vaginatus . Therefore, our team modified  “Transformation of the Cyanobacterium Leptolyngbya boryana by Electroporation. Bio-protocol”（Tsujimoto, R.）, and successfully obtained the transformant of Microcolus vaginatus. The modified protocpl is as follows:</P><P style="font-size: 22px; color:pink;">Preparation of Microcolus vaginatus single cells</P><P style="font-size: 18px">1. Take 10 mL of algae solution and centrifuge at 10 ° C, 2000 rpm for 10 min.</P><P style="font-size: 18px">2. Ultrasound treatment (350w, 2min, 5s surgery, 10s rest) treatment of algae into single cells</P><P style="font-size: 18px">3. The algae solution was treated with cellulase and pectinase and placed on a shaker overnight, protected from light.</P><P style="font-size: 18px">1. Single cell cells were harvested by centrifugation of a single cell algae solution at 1000 rpm for 10 minutes at 4 °C.</P><P style="font-size: 18px">2. The algal cells were washed three times with HEPES shock buffer (pH = 7.5) and centrifuged at 1000 rpm for 10 minutes at 10 ° C to collect algae cells.</P><P style="font-size: 22px; color:pink;">Electric shock conversion</P><P style="font-size: 18px">1. Take 70 ul of algal single cells and mix 30 ul of 100 ng / ul of recombinant plasmid DNA with a pre-cooled 0.2 cm electric shock cup.</P><P style="font-size: 18px">2. Put the electric shock cup into the shock converter for electric shock conversion, the electric field strength is 4kV / cm, and the electric shock is twice.</P><P style="font-size: 18px">3. Immediately add 0.9 mL of pre-cooled BG-11 liquid medium, transfer the mixture to a sterile centrifuge tube and use an ice bath for 10 minutes.</P><P style="font-size: 18px">4. The culture was recovered at 1000-2000 LUX at 25 ° C for 6 hours, added to 9 mL of liquid for one week, transferred to an anti-Spe plate, and cultured at 1000-2000 LUX, and the transformation results were observed.</P></DIV><DIV class="DBoard" id="title" style="height:40px;width:430px;font-size:20px;line-height:40px;text-align:center;">    
   Genomic DNA extraction from Cyanobacteria 
      </DIV><DIV class="DBoard" id="article1" style="height:100px;"><P style="font-size: 18px">We use TIANamp Bacteria DNA Kit to extract the genome of Synechocystis sp.   and Microcolus vaginatus ，you can see the whole protocol in <A href="https://static.igem.org/mediawiki/2018/3/3b/T--SCAU-China--Experiment.pdf">this link</A></P></DIV><DIV class="DBoard" id="title" style="height:40px;width:420px;font-size:20px;line-height:40px;text-align:center;">    
  Bacterial cellulose content measurement
      </DIV><DIV class="DBoard" id="article1" style="height:3400px;"><P style="font-size: 22px; color:green;">Step1:Reagent preparation:</P><P style="font-size: 18px">1.10% sodium tungstate solution: 10 g of sodium tungstate dissolved in 100 ml of water</P><P style="font-size: 18px">2.alkaline copper reagent: 4.5g copper sulfate crystals dissolved in 200ml water, 40g anhydrous sodium carbonate dissolved in 400ml water, 7.5g tartaric acid dissolved in 300ml water, first mixed with tartaric acid and sodium carbonate, and finally added copper sulfate, mixed Constant volume to 1000ml (long-term storage at room temperature)</P><P style="font-size: 18px">3. Phosphomolybdic acid reagent: 70 g of molybdic acid was dissolved in 400 ml of 10% sodium hydroxide solution, 10 g of sodium tungstate was added, and the furnace was boiled for 20 minutes. After cooling, add 250 ml of 85% phosphoric acid, mix well, dilute to 1000 ml, and protect from light.</P><P style="font-size: 18px">4. 0.25% benzoic acid solution: 2.5 g of benzoic acid was accurately weighed and dissolved in distilled water to a volume of 1000 ml.</P><P style="font-size: 18px">5. Standard glucose solution: Weigh 1.000 g of glucose. Dissolve with 0.25% benzoic acid and dilute to 1000 ml. At this time, the concentration of the glucose solution was 1 mg / ml. Store in a refrigerator at 4℃. Dilute 10 times with 0.25% benzoic acid before use.</P><P style="font-size: 18px">6.0.33mol / l sulfuric acid solution: Pipette 18.3ml of concentrated sulfuric acid, slowly put into distilled water, while stirring, to 1000ml volume</P><P style="font-size: 18px">7..lysozyme solution: take a small amount of lysozyme dissolved in 5ml distilled water, stored at 4℃</P><P style="font-size: 18px">8.cellulase solution: add a small amount of cellulase in 5ml of distilled water, stored at 4℃</P><P style="font-size: 22px; color:green;">Step2:Enzyme treatment</P><P style="font-size: 18px">1. Take 2 15ml centrifuge tubes, take 2ml bacterial solution in the ultra-clean platform, labeled as A, B;</P><P style="font-size: 18px">2. Add 0.5 ml of lysozyme and 0.5 ml of cellulase to the group A centrifuge tube, and add 0.5 ml of lysozyme to the group B, and treat at 37℃;</P><P style="font-size: 18px">Protein-free bacterial filtrate preparation</P><P style="font-size: 18px">3. Transfer 0.5 ml of 10% sodium tungstate solution into each centrifuge tube and shake well;</P><P style="font-size: 18px">4. Pipette 0.5ml 0.33mol / l sulfuric acid, slowly drip into a 15ml centrifuge tube, shake well, add 0.5ml cellulase solution to the B group centrifuge tube and mix;</P><P style="font-size: 18px">5. Dispense 2 ml of each liquid in a 15 ml centrifuge tube into four 2 ml centrifuge tubes;</P><P style="font-size: 18px">6. Centrifuge at 13000r for 10 minutes, and take the supernatant for use;</P><P style="font-size: 18px">7, according to the table processing, take 3 blood glucose tubes labeled 0, A, B</P><P style="font-size: 22px; color:green;">Step3: Measurement</P><P style="font-size: 18px">Blanking with a blank tube, measuring the absorbance of each sample at a wavelength of 620 nm with a spectrophotometerdata analysis</P><P style="font-size: 18px">Glucose content = (measuring tube A value / standard tube A value) * (c / 0.1) * 100</P><P style="font-size: 18px">Cellulose yield per 100 ml of bacterial liquid = tube glucose content - B tube glucose production</P><P style="font-size: 18px">Note: Cellulose yield varies with the growth state of the bacteria/cyanobacteria, so it is necessary to measure the growth curve and cellulose yield by a correlation function.</P><P style="font-size: 22px; color:green;">Calcofluor white fluorescent staining of cellulose</P><P style="font-size: 18px">Transfer a drop of algae droplets into a clean centrifuge tube, then drop a drop of 10% KOH and Calcofluor white dye solution, mix and pour a drop of the mixture onto a clean glass slide, cover the coverslip, and absorb excess with absorbent paper. liquid. Observe the slide under the fluorescence microscope.</P><P style="font-size: 22px; color:green;">Optimal cultivation conditions explor</P><P style="font-size: 18px">Exploring conditions and levels:</P><P style="font-size: 18px">Lighting（lux）1000   2000  4000   6000   9000  </P><P style="font-size: 18px">Nitrogen source (sodium nitrate concentration 1.5g/L）0   0.5   1.5   3   4.5</P><P style="font-size: 18px">Phosphorus potassium (dipotassium hydrogen phosphate concentration 0.04g/L）0   0.02  0.03  0.06   0.09</P><P style="font-size: 18px">Carbon source (sodium carbonate concentration 0.02g/L）0   0.02  0.03  0.06   0.09</P><P style="font-size: 18px">Iron (ammonium ferric citrate 0.006g/L）0     0.003   0.006   0.009   0.015</P><P style="font-size: 18px">Sodium chloride（g/L）0  10  20  40  60 </P><P style="font-size: 18px"><A href="https://static.igem.org/mediawiki/2018/f/f3/T--SCAU-China--Exper.docx">Click here to see the the orthogonal list </A></P></DIV><P>School's name:SCAU</P><P>Member's name:SCAU</P><P>Designed by:SCAU</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>