"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Project/Golden
From 2012.igem.org
Home
Team
Our Team
Photo Gallery
Collaborations
Official team profile
Project
Overview
Introduction
Golden Gate Standard
The TAL Vector
GATE Assembly Kit
Using the Toolkit
The Future of TAL
Experiments and Results
Notebook
Labbook Toolkit
Labbook TAL Vector
Methods
Parts
Safety
Modeling
Companel|DNA
Download
Human Practices
Overview
Philosophical Analysis
Educational Outreach
Public Outreach
Attributions
Golden Gate Standard
On this page, we introduce the Golden Gate Standard to the Registry of Standard Biological parts. We explain in detail, how Golden Gate Cloning works and how it can be made compatible with existing standards. Moreover, we provide step-by-step protocols for using this new standard.
Introduction
Although BioBrick assembly is a powerful tool for the synbio community because it allows standardized and simple construction of complex genetic constructs from basic genetic modules, it is not the best option when it comes to assembling larger numbers of modules in a short period of time. Furthermore, BioBrick assembly leaves scars between assembled parts, which is not optimal for protein fusion constructs. One popular method which overcomes these obstacles is Gibson Cloning (see figure 1).  This method uses an exonuclease to produce sticky ends on overlapping PCR products, a polymerase to fill up single stranded gaps after annealing and a ligase to connect the different parts.
Gibson cloning allows for assembling whole constructs in one reaction and has been used by many iGEM teams over the past years. However, this technique is not compatible with parts provided by the Registry, unless they are PCR-amplified in order to linearize them and to add overlapping sequences. Furthermore, Gibson cloning requires three different enzymes and can be very tricky. We therefore propose another method called Golden Gate Cloning1 (or its derivatives MoClo2 and GoldenBraid3). Golden Gate Cloning (GGC) can be used to assemble many fragments with very high efficiency in one reaction. Importantly, insert fragments can be cut out of amplification vectors (such as iGEM standard vectors) and assembled in one single reaction.
Figure 1: Gibson Cloning
Mechanism
In conventional cloning, restriction enzymes bind to and cut at the exact same spot. Consequently, one conventional restriction enzyme only produces one type of sticky ends. That is the reason why in conventional cloning, only two DNA parts can be assembled in one step. Golden Gate Cloning overcomes this restriction by exploiting the ability of type IIs restriction enzymes (such as BsaI, BsmBI or BbsI) to produce 4 bp sticky ends right next to their binding sites, irrespective of the adjacent nucleotide sequence. Thus, these enzymes are capable of producing multiple sticky ends at different DNA fragments in one reaction. Importantly, binding sites of type IIs restriction enzymes are not palindromic and therefore are oriented towards the cutting site (figure 2).
Figure 2: BsmB1 restriction mechanism
So, if a part is flanked by 4 bp overlaps and two binding sites of a type IIs restriction enzyme, which are oriented towards the centre of the part, digestion will lead to predefined sticky ends at each side of the part. In case multiple parts are designed this way and overlaps at both ends of the parts are chosen carefully, the parts align in a predefined order (figure 3).
Figure 3 : Golden Gate mechanism
In case a destination vector is added, that contains type IIs restriction sites pointing in opposite directions, the intermediate piece gets replaced by the assembled parts – magic! After transformation, the antibiotic resistance of the destination vector selects for the right clones.
Golden Gate Cloning is typically performed as an all-in-one-pot reaction. This means that all DNA parts, the type IIs restriction enzyme and a ligase are mixed in a PCR tube and put into a thermocycler. By cycling back and forth 10 to 50 times between 37°C and 20°C, the DNA parts get digested and ligated over and over again.  Digested DNA fragments are either religated  into their plasmids or get assembled with other parts as described above. Since assembled parts lack restriction sites for the type IIs enzyme, the parts get “trapped” in the desired construct. This is the reason why Golden Gate Cloning assembles DNA fragments with such exceptional efficiency. We successfully used this approach to assemble whole TAL effector expression vectors from six different parts – all in one reaction.
Merging BioBrick Standard and Golden Gate Cloning
As described above, the overlaps flanking a part determine the position of the particular part within the construct after GGC. We therefore propose the following two strategies for implementing Golden Gate Cloning within the Registry of standard biological parts.
Strategy 1
In most cases, iGEM teams seek to assemble so called protein generators, which consist of one part of each of the following categories:
- Promoters
- Ribosome binding sites (RBS)
- Protein coding regions
- and terminators
Since the order of these parts in a protein generator is always the same (promoter first,  RBS second, etc.), we can attribute a particular pair of overlaps to each of these categories and thereby define the order of the corresponding parts. From our experience with GGC, we propose the following overlaps which have shown no mispairing in our experiments:
Figure 4 : Overlaps of GGC
We believe that new standards should only be introduced to the Registry of Standard Biological Parts if they are compatible with the existing standards (most importantly with RFC10). Otherwise, the registry would get functionally split up into smaller part libraries and teams using one standard could not collaborate with teams using another. We therefore were very careful with introducing type IIs restriction sites into iGEM backbones. In this context, choosing the optimal relative position of the type IIs binding site to the BioBrick prefix and suffix restriction sites is essential for preserving idempotency of the RFC10 standard. Idempotency in this context means that assembling BioBricks results in higher order constructs that meet BioBrick standard requirements (i.e. they are flanked by the four standard restriction sites and do not contain any of them). The type IIs restriction site can principally be placed in three different positions:
1. between prefix/suffix restriction sites and the actual part
2. between the two restriction sites of the prefix and suffix (EcoRI and XbaI or SpeI and PstI, respectively)
3. distal from both RFC10 restriction sites.
Figure 5 : Restriction sites
As illustrated in figure 5, only placing the type IIs site between the RFC10 standard restriction sites maintains idempotency of the BioBrick standard. In each of the other cases, constructs assembled by Golden Gate Cloning are not iGEM standard compatible because they contain RFC10 standard restriction sites. We actually built 96 BioBricks using this Golden Gate Standard and successfully applied both RFC10 or "Golden Gate standard". We therefore propose the following Protocol:
For creating new BioBricks by PCR amplifying the corresponding DNA sequences, we propose the following primers:
1. For Promoters:
Pro fo: GATGAATTCGCGGCCGCTTCTAGAGAAGAC AT CCTG +  appr. 17 bp overlap
Pro re: GATCTGCAGCGGCCGCTACTAGTAGAAGAC TA GAGC + appr. 17 bp overlap (reverse complement)
2. For RBS:
RBS fo: GATGAATTCGCGGCCGCTTCTAGAGAAGAC AT GCTC + appr. 17 bp overlap
RBS re: GATCTGCAGCGGCCGCTACTAGTAGAAGAC TA GTCA + appr. 17 bp overlap (reverse complement)
3. For ORF:
ORF fo: GATGAATTCGCGGCCGCTTCTAGAGAAGAC AT TGAC + appr. 17 bp overlap
ORF re: GATACTGCAGCGGCCGCTACTAGTAGAAGAC TA GAGC + appr. 17 bp overlap (reverse complement)
4. For Terminators:
Ter fo: GATGAATTCGCGGCCGCTTCTAGAGAAGAC AT GCTT + appr. 17 bp overlap
Ter re: GATCTGCAGCGGCCGCTACTAGTAGAAGAC TA GAGT + appr. 17 bp overlap (reverse complement)
After purification of the PCR product, you can digest your linearized iGEM vector and your part with EcoRI and PstI using the following Protocol:
Component
Amount (μl)
Purified PCR product
 30
EcoRI
 1
PstI
 1
BsaI
 0,5
NEB buffer 4 (10x)
 4
ddH2O
 3,5
Total Volume
 40
Thermocycler programm:
1.     37°C, 12 hours
2.     80°C, 20 minutes
We very much advise you to digest the vector for 12 hours and purify the product on a gel. This significantly reduces the risk of religation of you vector. We usually had no colonies on our negative control plate after ligation with T4 ligase and transformation into DH10B cells.
Since most standard iGEM plasmids contain binding sites of the most common two type IIs restriction enzymes (namely BsaI/Eco31I and BsmBI/Esp3I), we propose using BbsI/BpiI. We have tested this enzyme in various reaction conditions with many different reaction additives (such as ATP or DTT). Although ligase buffer worked best with other type IIs restriction enzymes (in those cases, ligase activity probably was the bottleneck), we had best results with G Buffer (Fermantas) plus several additives using BbsI.
For assembling parts that are in Golden Gate standard, we recommend the following protocol:
Component
Amount (μl)
BpiI/BbsI (15 U)
 0,75
T4 Ligase (400 U)
 1
DTT (10 mM)
 1
ATP (10 mM)
 1
G-Buffer (10x, Fermentas)
 1
parts
 40 fmoles each
ddH2O
 Fill up to 10
Total Volume
 10
Thermocycler programm:
1.     37°C, 5 min
2.     20°C, 5 min
repeat (1. and 2.) 50 times
3.     50°C, 10 min
4.     80°C, 10 min
We always used this 8:40 hour thermocycler program to obtain best results. However you can also reduce the number of cycles.
Strategy 2
The Golden Gate Standard described above is very efficient, however, it does not exploit the exceptional advantage of GGC to assemble parts in-frame and without a scar.  In the past iGEM competitions, several attempts to clone protein modules in-frame have been proposed (see http://partsregistry.org/Help:Standards/Assembly#Registry_Supported_Assembly_Standards ), but no standard allows for scarless products (which is crucial for many applications, such as protein domain assembly). For scarless cloning of BioBricks, we therefore propose the following strategy:
Step 1: Define the sequences of DNA that you want to assemble without a scar. In case the sequences contain protein-coding sequences, make sure your sequences are in frame (e.g. the last three bp of the upstream part form a codon and the first three bp of the downstream part form a codon).
Step 2: Choose your 4 bp overlaps: In most cases, you can define the last 4 bp of every part as your overlaps.
Exceptions:
1. Overlaps are palindromic (don’t worry, the chance of a palindromic 4 bp sequence is less than 7 %). In this case, the part not only aligns with the downstream part but also with itself.
2. Several parts end with the same 4 bp sequence.
3. Three out of four base pairs of different parts are similar. In this case, mispairing may occur. However, we tried several 4 bp overhangs that overlap in 3 of 4 bp and haven’t had any false cloning product yet.
In case you encounter one of these exceptions, try one of the following overlaps:
1. Use the first 4 bp of the downstream part as overlap.
2. Use 2 bp of the upstream and 2 bp of the downstream part as overlap.
Usually, you should be able to define your overlaps now.
Step 3: Design your primers:
Forward primer: GAT GAAGAC CG XXXX + first appr. 17 bp of the part (xxxx represents the overlap for the upstream part)
Reverse primer: GATCA GAAGAC CG + reverse complement of the last appr. 17 bp of the part
Step 4: Perform PCR using a high-fidelity polymerase to amplify the BioBricks with the corresponding primers.
Step 5: Load the entire PCR product on a 1,5 % agarose gel and check whether your PCR product has the right size.
Step 6: Excise corresponding band and perform gel purification.
Step 7: Perform Golden Gate Cloning as described in strategy 1.
We used this approach to built the BioBricks for assembling multiple protein domains of our TAL proteins in one single reaction without forming scars.
References
1. Engler, C., Gruetzner, R., Kandzia, R. & Marillonnet, S. Golden Gate Shuffling: A One-Pot DNA Shuffling Method Based on Type IIs Restriction Enzymes. PLoS ONE 4, e5553 (2009).
2. Werner, S., Engler, C., Weber, E., Gruetzner, R. & Marillonnet, S. Fast track assembly of multigene constructs using golden gate cloning and the MoClo system. Bioengineered Bugs 3, 38–43 (2012).
3. Sarrion-Perdigones, A. et al. GoldenBraid: An Iterative Cloning System for Standardized Assembly of Reusable Genetic Modules. PLoS ONE 6, e21622 (2011).
Back to top
Retrieved from " http://2012.igem.org/Team:Freiburg/Project/Golden "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
