"
Page
Discussion
View source
History
teams
Log in
 
Team:Freiburg/Project/Vektor
From 2012.igem.org
Home
Team
Our Team
Photo Gallery
Collaborations
Official team profile
Project
Overview
Introduction
Golden Gate Standard
The TAL Vector
GATE Assembly Kit
Using the Toolkit
The Future of TAL
Experiments and Results
Notebook
Labbook Toolkit
Labbook TAL Vector
Methods
Parts
Safety
Modeling
Companel|DNA
Download
Human Practices
Overview
Philosophical Analysis
Educational Outreach
Public Outreach
Attributions
Creating the TAL Mammobrick vector
Introduction:
In nature, TALEs are injected into the host cells by plant pathogenic bacteria in order to modulate their gene expression. From the synthetic biologist’s point of view, this is very convenient because it implies that TALEs can be expressed in bacteria but also function in a eukaryotic system. We therefore provide plasmids for expression in either human cell lines or in bacteria.
Eukaryotic expression vector:
Since we wanted to express our TAL effectors in Human Embryonic Kidney (HEK) cells, we needed a eukaryotic expression vector.  Unfortunately, the registry does not offer such a vector, so we decided to build one our own. In order to avoid intellectual property rights violations, we ordered the vector pTALEN (v2) NG    (along with the Zhang Lab TALE Toolbox) from the open source plasmid repository Addgene . The Zhang Lab at MIT has constructed this plasmid for TAL effector expression, so we decided that it would be a good template for our own vector. Converting pTALEN (v2) NG into a RFC10 compatible vector would have taken more mutagenesis PCRs than we would have been able to perform over the summer, so we chose the following two-step vector assembly strategy:
Step 1: Mammobrick
In the first step, we wanted to built a universal mammalian expression vector (called MammoBrick), which would allows future iGEM students to express any gene in human cell lines simply by cloning the open reading frame into the MammoBrick using the BioBrick assembly protocol. We assembled the MammoBrick from the following four parts, essentially, using the protocol described here :
Part 1: BACKBONE
We have cut the backbone out of pTALEN (v2) NG with Ngo MIV and AfiII and purified the corresponding 2234bp band from a gel. Since both enzymes produce 5’ overhangs, they were compatible with overhangs produced by BsaI digestion. This backbone contains a SV 40 polyadenylation signal, an ampicillin resistance gene and an origin of replication.
Part2: CMV promoter
At first, we tried to use the CMV promotor that was included in the 2012 distribution kit. Part BBa_J52034 was submitted to the registry by Team Slovenia in 2006 and has been on the distribution kit since then (although sequencing was inconsistent every year). After numerous attempts to use this part, we sequenced it and found out that it was not a CMV promotor, but a part of the lacI gene. Reading the part’s review, we noticed that Team Munich 2010 had already pointed out that it was a lacI fragment. Interestingly, Team DTU Denmark was able to induce fluorescent protein expression with this bacterial gene fragment- magic. Since no other mammalian promoter was available on this year’s distribution kit, we designed the following primers and amplified the CMV promoter from the vector pPhi-Yellow-C:
GTTACCGGTCTCGTTAAGAATTCGCGGCCGCTTCTAGAGATAGTAATCAATTACGGGGTC
CTAGAGGTCTCGCTGCCTGCAGCGGCCGCTACTAGTAGATCTGACGGTTCACTAAAC
After amplifying the CMV promoter with these primers, the promoter is not only flanked by the iGEM prefix and suffix, but also by distal BsaI restriction sites. This way, we were able to directly assemble the PCR product with the other MammoBrick parts.
Part 3: PuroORF
We replaced the hygromycin resistance gene in pTALEN (v2) NG for two reasons: Firstly, it contained multiple iGEM restriction sites and secondly, selection via hygromycin takes much longer than selection with puromycin. Since we also didn’t find a puromycin ORF without illegal restriction sites, we decided to make silent mutations in the PuroORF to remove these sites and get it synthesized, flanked by BsaI restriction sites and appropriate overlaps for subsequent Golden Gate cloning.
Part 4: PostORF
We called the region between the stop codon of the TAL ORF and the start codon of the antibiotic resistance gene PostORF. We wanted to use this part in our vector because it contains the SV40 promoter and enhancer for expression of the antibiotic selection marker. So we used PCR to “excise the fragment and add BsaI sites and appropriate overlaps to it.
After every single part had been purified, we used Golden Gate cloning to assemble them in one step. After quite some testing, we came up with the following protocol:
Component
Amount (μl)
pTALEN (v2) NG backbone (56 ng)
 1
CMV promoter (17 ng)
 1
Post ORF (17,5 ng)
 1
ddH2O
 11,5
T4 Ligase (400 U)
 1
BsaI  (15 U)
 1
T4 Ligase buffer
 2
Total
 20
Thermocycler programm:
1.     37°C, 5 min
2.     20 °C, 5 min
go to 1. 50 times
4.     50°C, 10 min
5.     80°C, 10 min
So we assembled the whole MammoBrick vector in one single reaction:
Step 2: Eukaryotic TALE expression vector:
Once the MammoBrick was ready, we inserted the TAL open reading frame and thereby evaluated, how easy it would be for future iGEM students to expression any desired ORF in eukaryotic cells.
Designing the TAL open reading frame:
For this purpose, we designed a TAL ORF by adding the following modifications to the TAL open reading frame in pTALEN (v2) NG:
1. We removed all EcoRI, XbaI, SpeI, PstI, BsmBI, BbsI and PmeI restriction sites.
2. We replaced the BsaI restriction sites for inserting direpeats by BsmBI sites, because – according to the manufacturer - BsmBI is better suited for digest over one hour.
3.  We added a consensus RBS in front of the ORF for expression in bacteria
4. We added a His-Tag to the n-terminal end to allow protein purification.
5. We flanked the whole sequence with the iGEM prefix and suffix.
6. Most importantly, we replaced the FokI nuclease at the C-terminal end of the protein by one of our inventions: The Plug and Play Effector Cassette.
This whole construct was synthesized by Genscript.
Plug and Play Effecor Cassette: Our project was designed to enable future iGEM teams to easily use the powerful TALE technology. On top of that, we wanted to built a TALE platform which allows iGEM students to develop their own TAL constructs. We therefore invented the easy-to-use Plug and Play Effector Cassette (PPEC), which can be used to fuse BioBricks, that are in the Golden Gate standard , to the c-terminus of the TAL protein.
The PPEC consists of two BbsI binding sites that point in opposite directions. Digestion with BbsI leads to removal of the PPEC and to the formation of sticky ends at which the upstream sticky end (GGCA) is the last 4 nucleotides of the TAL protein and the downstream sticky end (TAAA) contains the stop codon. When an equimolar amount of the effector containing plasmid (flanked also by BbsI sites and the same overlaps) is added to the GGC mix, the effector is cut out of the iGEM vector and ligated into the eukaryotic TAL expression vector in-frame and without a scar. We have optimized this reaction by systematically testing different reaction buffers and thermocycler programs and came up with the following protocol:
Component
Amount (μl)
BpiI/BbsI (15 U)
 0,75
T4 Ligase (400 U)
 1
DTT (10 mM)
 1
ATP (10 mM)
 11,5
G-Buffer (10x, Fermentas)
 1
parts
 40 fmoles each
ddH2O
 Fill up to 10
Total
 10
Thermocycler programm:
1.     37°C, 5 min
2.     20 °C, 5 min
go back to 1. 20 times
4.     50°C, 10 min
5.     80°C, 10 min
5.     4°C, ∞
But even Golden Gate cloning is not 100 % efficient. In order to remove those plasmids that did not take up a vector insert, we added the restriction site of the blunt end cutter PmeI (MssI) to the PPEC. We chose PmeI because it has a 8 bp binding site, which is very unlikely to occur in the gene of an effector that you would like to fuse with the TAL gene.
So after performing the Golden Gate reaction described above, we digested with MssI fast digest (fermentas) according to the following protocol:
Component
Amount (μl)
GGC-Product
 10
PmeI/MssI FastDigest
 1
Fast Digest Buffer (10x)
 1,5
ddH2O
 2,5
Total
 15
Thermocycler programm:
1.     37°C, 1h
2.     80 °C, 20 min
5.     4°C, ∞
This linearizes all vectors that do not contain the effector (at least, we do not see colonies on the negative control plate). To be sure, these linearized vectors do not religate, perform the following digest with T5 exonuclease, which specifically removes linearized DNA:
Component
Amount (μl)
Product of PmeI digest
 7,5
T5 Exonuclease
 1
Total
 8,5
Thermocycler programm:
1.     37°C, 1h
2.     80 °C, 20 min
5.     4°C, ∞
The efficiency of our own little invention – the PPEC – actually surprised us a little bit, for details, see the results section .
Insertion of the TAL ORF into the MammoBrick vector:
Since we wanted to put the TAL ORF under the control of the CMV promoter, we digested both the MammoBrick vector (with SpeI and PstI) and the TAL ORF (with XbaI and PstI), ligated them and transformed into a ccdB-cassette resistant E.coli strain. The resulting clones were verified by sequencing and contained the eukaryotic TAL expression vector:
Prokaryotic TAL expression vector:
Although for the most part, TAL effectors have been used in eukaryotic organisms, we wanted to enable future iGEM teams to also use this exciting technology in bacteria. So we used BioBrick assembly to construct the following protein generator using TAL ORF, Part:BBa_J04500 (IPTG inducible promoter with RBS) and BBa_B0015 (double terminator):
Back to top
Retrieved from " http://2012.igem.org/Team:Freiburg/Project/Vektor "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
