"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Chile2/Cyanolux/Results
From 2012.igem.org
Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012
Team »
Roster
PSB Lab
Photo gallery
Luxilla »
Project
Results
Modelling
Future Prospects
Biolamp
Biobricks
Notebook
Side projects »
Bactomithril
Gibson-small parts
Syne growth curve
Int_C debugging
LuxBrick char.
Methodologies »
Protocols
Arduino
DIY
Software
Human Practices »
Synthetic Sociology
Biosafety
SynBio Workshop
More »
Collaborations
Judging Criteria
Attribution
Contents
1Construction of plasmids
1.1pSB1C3_IntK
1.1.1sfGFP with LVA tag for describing circadian behaviour
1.1.2Luciferase(s)
1.2pSB1A3_IntC (From 2010 Utah's iGEM team)
1.3pSB1C3_IntS
1.3.1Substrate production under Pcaa3
1.3.2Substrate production under PsigE
2Synechocystis
2.1Standardizing growth conditions
2.2Transforming Synechocystis PCC 6803
2.2.1BBa_K743009
2.3Reestreaking
2.4Confirming DNA
2.5Conclusions
3Revealing phenotypes
3.1Microscopy
3.2Luminometer readings
3.3Conclusions
Construction of plasmids
pSB1C3_IntK
Our first attempts to build the construct ([ BBa_K743004 ]) through Gibson Assemly were unsuccessful due to problems involving high compexity of the reaction, however we were able to Biobrick the recombination sites ([ | Part:BBa_K743000 ] and | Part:BBa_K743001 ). Afterwards we decided to build a simple backbone first before continuing with more complex assemblies.
Through standard assembly we managed to build K743006 which led us to continue assembling our constructs through simpler Gibson Assemblies.
sfGFP with LVA tag for describing circadian behaviour
To describe the circadian behaviour of the promoter we built a fast-degrading reporter consisting of sfGPF I746916 with a LVA degradation tag in the C-terminal end of the protein. This construct will serve as a real-time reporter of promoter activity, you may find more information about the half-life of proteins with the LVA tag | here . The construct has been verified by digestion and corroborated by sequencing.
300px
Luciferase(s)
In short time we were able to build our final constructs for the luciferase using 2 different versions of it available in the registry ( | From Photorhabdus luminiscent, BBa_K743014 and | from Vibrio fisherii, BBa_K743015 ) under an endogenous  Synechocystis's promoter (transaldolase Reference???). All constructs and parts have been verified by digestion and corroborated by sequencing.
pSB1A3_IntC (From 2010 Utah's iGEM team)
We decided to use this plasmid to place the LuxCDEG part of the Lux operon in this plasmid, however after various attempts to transform Synechocystis without success we reconsidered (see below SOMEWHERE!!! [[]]) due to problems regarding the design of this plasmid backbone. After analyzing the sequence of the vector (CODE HERE) we blasted the sequences of the recombination sites and we found out that position of the sequences of each recombination site are swapped, leaving the backbone part of the plasmid to be integrated into the genome. Furthermore, the recombination sites are separated by aproximately 2.2 kilobases, knocking 1 gene.
You can see their original construct design here .
300px
pSB1C3_IntS
After we reconsidered about using pSB1A3_IntC we designed a new plasmid backbone to place the LuxCDEG (substrate regeneration part of the Lux operon) under the Synechocystis's promoters we choose through our modelling (LINK HERE). We have obtained the plasmid backbone including all parts through Gibson Assembly and the plasmid has validated through digestion and was corroborated through sequencing.
300px
Substrate production under Pcaa3
We have assembled all parts of the substrate production pathway ( LuxCD and LuxEG ) for the final construct under the Pcaa3 promoter (which has also been Biobricked with code K743002 and we are now verifying for positive assembles.
Substrate production under PsigE
As this assembly was unsuccessful previously, we are re-amplifying the LuxCD part (to obtain higher concentrations of the insert) and we will retry the assembly after the wiki freeze.
Synechocystis
Standardizing growth conditions
While we managed to acquire an inoculum of Synechocystis PCC 6803, no conditions were set for working with Synechocystis PCC 6803 in our faculty. We proceeded to standardize our own growth conditions by building a custom made cyanobacterial incubator based on growth conditions described in the literature (References?)
Here we show our growth curve for Synechocystis PCC 6803 using the conditions set in our DIY ) shaking incubator (40uE/s/m²), 90 RPM, 30°C.
Methods for the growth curve characterization of Synechocystis PCC 6803 over here .
Transforming Synechocystis PCC 6803
After corroborating our constructs we proceded with the [Team:UC_Chile/Protocols#Transformation_of_Synechocystis_PCC._6803| Synechocystis transformation] for each of the constructs as soon as we obtained sufficient concentration (10ug) of the construct.
Transformation in Synechocystis takes about 2 weeks to reveal transformant colonies, which in turn have to restreaked again before PCR verification. This is necesary due to the high remaining amount of plasmid in the Millipore membrane which may serve as template for the colony PCR, inducing false positive verification bands.
BBa_K743009
This picture shows a transformation after 11 days. On the left there is the control transformation plate (no DNA) on antibiotic and in the right a positive transformation of K743009 with its corresponding Synechocystis colonies growing througout the Millipore membrane on the same antibiotic.
Reestreaking
Confirming DNA
Conclusions
Revealing phenotypes
Microscopy
Luminometer readings
Conclusions
Retrieved from " http://2012.igem.org/Team:UC_Chile2/Cyanolux/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
