<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Chile2_General_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Chile2/General_Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Chile2/General_Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile2/General_Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile2/General_Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Chile2/General_Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Chile2/General Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012</TITLE></P><DIV id="banner"><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Team/Members">Team <SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Members">Roster</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/PSB_Lab">PSB Lab</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Gallery">Photo gallery</A></LI></UL><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Luxilla<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Project</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Results_short">Results</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Future">Future Prospects</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biolamp">Biolamp</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biobricks">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Notepad">Notebook</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Side projects<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Bactomithril</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Gibson">Gibson-small parts</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Growthcurve">Syne growth curve </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Int_C">Int_C debugging</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/LuxBrick">LuxBrick char.</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Protocols">Methodologies<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Arduino">Arduino</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/DIY">DIY</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Other">Software</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Human Practices<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Synthetic Sociology</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Biosafety">Biosafety</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Workshop">SynBio Workshop</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Collaborations">More<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Collaborations">Collaborations</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Accomplishments">Judging Criteria </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Attribution">Attribution</A></LI></DIV><DIV id="Growth_Media"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Growth_Media"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Growth Media</SPAN></A></LI><LI class="toclevel-2"><A href="#LB_media"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">LB media</SPAN></A></LI><LI class="toclevel-2"><A href="#SOB_media"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">SOB media</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Buffers"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Buffers</SPAN></A></LI><LI class="toclevel-2"><A href="#5X_Gibson_Assembly_Isothermal_Buffer"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">5X Gibson Assembly Isothermal Buffer</SPAN></A></LI><LI class="toclevel-2"><A href="#1.33X_Gibson_Assembly_Master_Mix"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">1.33X Gibson Assembly Master Mix</SPAN></A></LI><LI class="toclevel-1"><A href="#DNA_assembly_protocols"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">DNA assembly protocols</SPAN></A></LI><LI class="toclevel-2"><A href="#Standard_Assembly"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Standard Assembly</SPAN></A></LI><LI class="toclevel-3"><A href="#Digestion_reaction"><SPAN class="tocnumber">3.1.1</SPAN><SPAN class="toctext">Digestion reaction</SPAN></A></LI><LI class="toclevel-3"><A href="#Ligation_reaction"><SPAN class="tocnumber">3.1.2</SPAN><SPAN class="toctext">Ligation reaction</SPAN></A></LI><LI class="toclevel-2"><A href="#Gibson_Assembly"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-3"><A href="#Design_primers"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">Design primers</SPAN></A></LI><LI class="toclevel-3"><A href="#Obtaining_parts"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">Obtaining parts</SPAN></A></LI><LI class="toclevel-3"><A href="#Assembly_reaction"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">Assembly reaction</SPAN></A></LI><LI class="toclevel-3"><A href="#Checking_assembly"><SPAN class="tocnumber">3.2.4</SPAN><SPAN class="toctext">Checking assembly</SPAN></A></LI><LI class="toclevel-1"><A href="#E.coli_Transformation"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">E.coli Transformation</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Growth_Media">Growth Media</SPAN></H1></DIV><H2><SPAN class="mw-headline" id="LB_media">LB media</SPAN></H2><UL><P>Final concentrations:</P><LI>1% (w/v) tryptone</LI><LI>0.5% (w/v) yeast extract</LI><LI>86 mM NaCl</LI><P>Per liter:</P><LI>10g Tryptone</LI><LI>5g Yeast extract</LI><LI>5g NaCl</LI><LI>15g Agar -Optional-</LI><LI>Adjust pH to 7.5 with NaOH 1M</LI></UL><H2><SPAN class="mw-headline" id="SOB_media">SOB media</SPAN></H2><UL><P>Final concentrations:</P><LI>0.5% (w/v) yeast extract</LI><LI>2% (w/v) tryptone</LI><LI>10 mM NaCl</LI><LI>2.5 mM KCl</LI><LI>20 mM MgSO4</LI><P>Per liter:</P><LI>5 g yeast extract</LI><LI>20 g tryptone</LI><LI>0.584 g NaCl</LI><LI>0.186 g KCl</LI><LI>2.4 g MgSO4</LI><LI>15g Agar -Optional-</LI><LI>Adjust pH to 7.5 with NaOH 1M</LI></UL><DIV id="Buffers"><H1><SPAN class="mw-headline" id="Buffers">Buffers</SPAN></H1></DIV><H2><SPAN class="mw-headline" id="5X_Gibson_Assembly_Isothermal_Buffer">5X Gibson Assembly Isothermal Buffer</SPAN></H2><UL><P>Final Concentrations:</P><LI>25% PEG MW 8000</LI><LI>500mM Tris HCl pH 7.5</LI><LI>50mM CaCl</LI><LI>50mM DTT</LI><LI>1mM dATP</LI><LI>1mM dTTP</LI><LI>1mM dGTP</LI><LI>1mM dCTP</LI><LI>5mM NAD+</LI><LI>H20 nuclease free</LI><P>For a 2 mL stock (equivalent to 20 100uL 5X isothermal buffers)</P><LI>0.5g PEG MW 8000</LI><LI>1.5mL Tris-HCl pH 7.5 1M</LI><P>Swivel for 30 minutes in a orbital oscillator for PEG to dissolve fully</P><LI>50uL CaCl 2M</LI><LI>100uL DTT 1M</LI><LI>20uL dATP 100mM</LI><LI>20uL dTTP 100mM</LI><LI>20uL dGTP 100mM</LI><LI>20uL dCTP 100mM</LI><LI>200uL NAD+ 200mM</LI><LI>H20 nuclease free up to 2mL</LI><P>Alicuot into 100uL stocks</P><P><B>Store at -80°C</B></P></UL><H2><SPAN class="mw-headline" id="1.33X_Gibson_Assembly_Master_Mix">1.33X Gibson Assembly Master Mix</SPAN></H2><UL><P>For a 375ul 1.33X Gibson Assembly Master Mix</P><LI>100uL 5X Gibson Assembly Isothermal Buffer</LI><LI>6.25uL Phusion Polymerase 2 U/uL (cat N° F-350S) from Thermo Scientific </LI><LI>2uL T5 Exonuclease 1U/uL (cat N° T5E4111K) from Epicentre</LI><P>It is important to dilute the T5 Exonuclease stock from 10U/uL to 1U/ul to measure the volume correctly</P><LI>50uL Taq DNA Ligase 2000 U/uL (cat N° M208S) from NEB</LI><LI>216.75uL of nuclease free H20</LI><P><B>Alicuot 9uL in 0.2mL PCR tubes. This will yield about 42 reactions</B></P></UL><DIV id="DNA_assembly_protocols"><H1><SPAN class="mw-headline" id="DNA_assembly_protocols">DNA assembly protocols</SPAN></H1></DIV><H2><SPAN class="mw-headline" id="Standard_Assembly">Standard Assembly</SPAN></H2><P>The standard assembly reaction relies on the standarization of Biobricks to join different DNA parts. Plasmids from the registry of standard parts allow joining of DNA parts by using a combination of specific enzymes to cut a Upstream Biobrick with the restriction enzymes EcoRI and SpeI, and a Downstream Biobrick with the restriction enzymes XbaI and PstI into a Destination plasmid which has been cut with the restriction enzymes EcoRI and PstI. The reaction yields the Upstream and Downstream part joined by a mixed restriction site and allows further elongation of the construct by the same strategy. It is important to notice that such a reaction requires purification of the digested parts if any of the plasmids (Upstream, Downstream or Destination) share a resistance marker.</P><UL><LI>Upstream part enzymes: EcoRI &amp; SpeI</LI><LI>Downstream part enzymes: XbaI &amp; PstI</LI><LI>Destination backbone enzymes: EcoRI &amp; PstI</LI></UL><H3><SPAN class="mw-headline" id="Digestion_reaction">Digestion reaction</SPAN></H3><UL><LI>X volume of DNA to 500ng of plasmid</LI><LI>(42.5 - X)uL of nuclease free water</LI><LI>5uL of NEB buffer 2</LI><LI>0.5uL of BSA 100X</LI><LI>1uL of Enzyme 1</LI><LI>1uL of Enzyme 2</LI><LI>Incubate at 37°C for 2 hours</LI><LI>Heat inactivate enzymes at 80°C for 20 minutes</LI></UL><H3><SPAN class="mw-headline" id="Ligation_reaction">Ligation reaction</SPAN></H3><UL><LI>2uL of digested Upstream part</LI><LI>2uL of digested Downstream part</LI><LI>2uL of digested Destination plasmid</LI><LI>2uL of T4 DNA ligase buffer</LI><LI>11uL of nuclease free water</LI><LI>1uL of T4 DNA ligase</LI><LI>Incubate at room temperature for 10 minutes</LI><LI>Heat inactivate at 80°C for 20 minutes</LI><LI><A href="#E.coli_Transformation">Transform</A></LI></UL><H2><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H2><P>Gibson Assembly is a DNA assembly method created by Daniel Gibson during the development of the first Synthetic Genome (Synthia) (reference YYY). Its adaptation to a cloning method allows fast and accurate production of increasingly complex constructions. The strategy behind the method relies on PCR to obtain different parts which share a 40bp homology region, and a 3 enzyme reaction which produces cohesive ends, fills the gaps between the parts and ligates the resulting construct into a scarless assembly of various (&gt;2) parts.</P><H3><SPAN class="mw-headline" id="Design_primers">Design primers</SPAN></H3><P>The easiest way to design primers to obtain amplicons with the required overlaps (40bp final overlaps) is to make an <I>in sillico</I> design of the final construct</P><UL><LI>Design forward primer of right amplicon of joint in 5'-&gt;3' direction</LI><LI>Calculate length of annealing part of primer as to reach a Tm of approximately 63°C</LI><LI>Add 20 bp of overlap</LI><LI>Design reverse primer of left amplicon of joint in 5'-&gt;3' direction</LI><LI>To do this, select full joint and &quot;reverse complement&quot; it (be sure to be able to discriminate between right and left parts of joint)</LI><LI>Calculate length of annealing part of primer as to reach a Tm of approximately 63°C</LI><LI>Add 20 bp of overlap</LI><LI>Apply same principle in all joints</LI></UL><H3><SPAN class="mw-headline" id="Obtaining_parts">Obtaining parts</SPAN></H3><UL><LI>PCR parts using Phusion Polymerase datasheet indications. It is important to use a low ammount of template plasmid (10pg) as to reduce possibility of carry-over during band purification</LI><LI>Run agarose gel electrophoresis on a adecuate gel (50bp to 200bp parts should be run on a 3% w/v agarose gel, larger parts should be run on a 1% w/v to 1.5% w/v agarose gel) until clear distinction of bands is achieved. It is recommended that gels should be exposed as little as possible to UV transilluminators as UV light damages DNA.</LI><LI>Cut band and proceed with band purification. Elute in smallest volume as possible according to your kit specification</LI><LI>Quantify purified DNA</LI></UL><H3><SPAN class="mw-headline" id="Assembly_reaction">Assembly reaction</SPAN></H3><P>The assembly reaction is composed of 9uL of 1.33X Gibson assembly master mix + 3uL of purified parts DNA</P><UL><LI>The calculation of the ratio at which your DNA parts are should be in correspondence to the level of competence of your cells. We have found that with our cells (5*10^⁸ colonies/ug of pUC19 DNA) the following ratios work well</LI><LI>Calculate the amount of pmoles of each purified DNA part using the following equation: pmoles of DNA = weigth in ng * 1000 (conversion factor from nano to pico) / (650 Daltons * base pair length of part)</LI><LI>We have seen that using 0.01 picomoles of template (part with the selection resistance) and the rest of parts at 0.03 picomoles and above, yield a high ratio of true positive transformants and a decent number of colonies to check</LI><LI>Thoroughly mix the 3uL of purified parts DNA with the 9uL of 1.33X Gibson assembly master mix in ice</LI><LI>Directly incubate the reaction at 50°C for 1 hour</LI><LI><A href="#E.coli_Transformation"> Transform</A> competent cells with 8uL of assembled DNA (that leaves 4uL to check in an agarose gel if the reaction fails)</LI></UL><H3><SPAN class="mw-headline" id="Checking_assembly">Checking assembly</SPAN></H3><UL><LI>Check transformant colonies by colony PCR using primers for whole insert</LI><LI>Grow positive colonies in media with corresponding antibiotic</LI><LI>Miniprep colonies and digest plasmid with EcoRI &amp; PstI restriction enzymes</LI><LI>If checking protocols validate until now, proceed by sequencing to corroborate</LI></UL><DIV id="E.coli_Transformation"><H1><SPAN class="mw-headline" id="E.coli_Transformation">E.coli Transformation</SPAN></H1></DIV><UL><LI>Keep 50uL of <A href="http://openwetware.org/wiki/TOP10_chemically_competent_cells" class="external text" rel="nofollow">chemically competent cells</A> in ice (for no more than 10 minutes until transformation)</LI><LI>Mix 0.01 to 1 total ng of plasmid with competent cells (depending on size of plasmid and competence of your cells)</LI><P>Flick very softly to mix (competent cells are very fragile)</P><LI>Leave on ice for 30 minutes</LI><LI>Heat shock at 42°C for 60 seconds</LI><LI>Add 250uL of <A href="#Growth_Media"> LB media</A></LI><LI>Incubate at 37°C in a rotating shaker for 1 hour (Ampicillin) or 2 hours (Chloramphenicol and Kanamycin)</LI><LI>Plate 50uL of transformed cells to a LB agar petri dish with corresponding antibiotic (30ug/mL Kanamycin or 30ug/mL Chloramphenicol or 100ug/mL Ampicillin)</LI></UL><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UC_Chile2/General_Protocols">http://2012.igem.org/Team:UC_Chile2/General_Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Chile2/General_Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Chile2/General_Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Chile2/General_Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Chile2/General_Protocols&amp;oldid=133154" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>