<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Chile2_Cyanolux_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Chile2/Cyanolux/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Chile2/Cyanolux/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile2/Cyanolux/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile2/Cyanolux/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Chile2/Cyanolux/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Chile2/Cyanolux/Project</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012</TITLE></P><DIV id="banner"><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Team/Members">Team <SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Members">Roster</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/PSB_Lab">PSB Lab</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Gallery">Photo gallery</A></LI></UL><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Luxilla<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Project</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Results_short">Results</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Future">Future Prospects</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biolamp">Biolamp</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biobricks">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Notepad">Notebook</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Side projects<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Bactomithril</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Gibson">Gibson-small parts</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Growthcurve">Syne growth curve </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Int_C">Int_C debugging</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/LuxBrick">LuxBrick char.</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Protocols">Methodologies<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Arduino">Arduino</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/DIY">DIY</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Other">Software</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Human Practices<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Synthetic Sociology</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Biosafety">Biosafety</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Workshop">SynBio Workshop</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Collaborations">More<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Collaborations">Collaborations</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Accomplishments">Judging Criteria </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Attribution">Attribution</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Main_Goal"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Main Goal</SPAN></A></LI><LI class="toclevel-1"><A href="#Rationale"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Rationale</SPAN></A></LI><LI class="toclevel-2"><A href="#Synechocystis_PCC_6803"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Synechocystis PCC 6803</SPAN></A></LI><LI class="toclevel-2"><A href="#Lux_Operon"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Lux Operon</SPAN></A></LI></UL><LI class="toclevel-1"><A href="#Experimental_Strategy"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experimental Strategy</SPAN></A></LI><LI class="toclevel-2"><A href="#Splitting_the_Lux_operon_and_choosing_promoters"><SPAN class="tocnumber">3.1</SPAN><SPAN class="toctext">Splitting the Lux operon and choosing promoters</SPAN></A></LI><LI class="toclevel-2"><A href="#Building_constructs"><SPAN class="tocnumber">3.2</SPAN><SPAN class="toctext">Building constructs</SPAN></A></LI><LI class="toclevel-3"><A href="#pSB1C3_IntK"><SPAN class="tocnumber">3.2.1</SPAN><SPAN class="toctext">pSB1C3_IntK</SPAN></A></LI><LI class="toclevel-3"><A href="#pSB1A3_IntC_.28Utah_2010_iGEM_Team_integration_plasmid.29"><SPAN class="tocnumber">3.2.2</SPAN><SPAN class="toctext">pSB1A3_IntC (Utah 2010 iGEM Team integration plasmid)</SPAN></A></LI><LI class="toclevel-3"><A href="#pSB1C3_IntS"><SPAN class="tocnumber">3.2.3</SPAN><SPAN class="toctext">pSB1C3_IntS</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Main_Goal">Main Goal</SPAN></H1><P>Natural cycles have always fascinated mankind, probably due to the mysterious mechanisms involved in them and the power they exert in our everyday life. Since the dawn of synthetic biology, engineering oscillatory systems has been a recurrent topic, being Ellowitz's represillator a classical example. Nevertheless, to date no iGEM team has accomplished the implementation of a robust oscillatory system. That will be our challenge for this year's iGEM project.</P><P>To reach our goal we designed a synthethic circuit that links to the endogenous circadian rhythm of <I>Synechocystis PCC6803.</I> As a proof of concept we are going to engineer the first light-rechargeable biological lamp: <I>Synechocystis PCC 6803</I> cells that emit light only by night while recovering and producing the substrates in the day.<B> We strongly believe</B> this will serve as an enabling tool to any project requiring time control over a biological behaviour independently of the user's input.</P><P>Furthermore, the characterization of this chassis is a fundamental step to explore new systems with minimal inputs to replace <I>E. coli</I>, for example, in the biotechnological industry in order to achieve greener processes.</P><H1><SPAN class="mw-headline" id="Rationale">Rationale</SPAN></H1><H2><SPAN class="mw-headline" id="Synechocystis_PCC_6803">Synechocystis PCC 6803</SPAN></H2><P>Cyanobacteria are prokaryotic photoautotrophs and they are believed to be the only group of organisms to
evolve oxygenic photosynthesis about 2.4 billion years ago <A href="#1">1</A>. Although this biochemical breakthrough can’t
be understated, several cyanobacteria species also play a crucial role in the planet nitrogen cycle as marine
diazotrophs <A href="#2">2</A>. Cyanobacteria are found almost in every environment in earth´s surface and interestingly, they
are one of the few prokaryotes known to have circadian rhythms accounting for their photosyntethic
lifestyle <A href="#3">3</A>.</P><P><I>Synechocystis PCC6803</I> is a gram negative non-filamentous cyanobacteria and it was the third
prokaryote and the first photoautotroph whose genome was sequenced. Consequently, is has become a model organism as its genetic background has been widely studied.</P><P>Given the reasons mentioned above, it is of no surprise that <I>Synechocystis PCC6803</I> (among other
cyanobacteria) has been extensively used for biotechnological applications and proposed as the “green <I>E.coli</I>”<A href="#4">4</A>.</P><P>With the dawn of Synthetic Biology, research has made use of <I>Synechocystis</I> for
commodity chemicals production and detection of water soluble pollutants among other applications<A href="#5">5</A>,<A href="#6">6</A>.</P><P>While aware of these practical applications, we (UC_Chile) are particulary interested in the well
characterized circadian behavior of this chassis <A href="#7">7</A> and its implications. There are several
genes known to oscillate in a daily basis, most of them related to respiration, photosynthesis and energy
metabolism <A href="#8">8</A>, and it has been shown that reporter systems using these promoters show a
similar expression pattern <A href="#9">9</A>.</P><P>Lastly, there are a lot of biobricks designed especially for <I>Synechocystis</I> or from its genome´s
sequence by previous iGEM teams (references) but sadly no one has ever characterized them in this chassis
and the registry lacks a set of tools for its transformation with standard biological parts. Moreover, to our
knowledge, no iGEM team has ever accomplished a successful direct <I>Synechocystis</I> transformation with naked DNA.</P><H2><SPAN class="mw-headline" id="Lux_Operon">Lux Operon </SPAN></H2><P>The lux operon is a group of genes that are responsible for density-dependent bioluminescent behavior
in various prokariotic organisms such as <I>Vibrio fischeri</I> and <I>Photorabdus luminescens</I><A href="#12">12</A>. In <I>V. fischeri</I>, the operon is composed of 8 genes: LuxA and LuxB encode for the monomers of a heterodimeric luciferase; LuxC, LuxD and LuxE code for fatty acid reductases enzymes and LuxR and LuxI are responsible for the regulation of the whole operon <A href="#13">13</A>.</P><P>Lastly LuxG is believed to act as a FMNH2 dependent FADH reductase, although luminescence is barely affected
in its absence <A href="#14">14</A>. The n-decanal ( n= 9 to 14) substrate oxidization to n-decanoic acid by the LuxAB heterodimer is coupled with the reduction of FMNH to FMNH2 and the releasing of oxygen and light.</P><P>The carboxylic group of the product is then reduced to aldehyde by CDE proteins allowing the reaction to
start over <A href="#15">15</A>.</P><P>LuxAB genes have been widely used as reporters dependent on the addition of n-decanal to the culture
media <A href="#16">16</A> and in 2010, the <A href="https://2010.igem.org/Team:Cambridge" class="external text" rel="nofollow">Cambridge iGEM team</A> engineered LuxABCDEG to an <I>E. coli</I>-optimized biobrick
format, uncoupling it from the LuxR and LuxI regulation.</P><P>As a team we decided to work with this operon for a number of reasons, first of all, the luminescence
produced by this pathway is much more visually appealing than other systems from the registry (i.e XFPs),
moreover, the light production doesn´t depend on a single peptide but on a whole pathway involving several genes, which makes it much more tunable, for instance, decoupling in time the substrate recovery from the luciferase reaction itself.</P><H1><SPAN class="mw-headline" id="Experimental_Strategy">Experimental Strategy</SPAN></H1><P>We have devised different strategies to achieve bioluminescence controlled under circadian rhythms. Here we describe the strategies used for building the constructs to reach our goals.</P><H2><SPAN class="mw-headline" id="Splitting_the_Lux_operon_and_choosing_promoters">Splitting the Lux operon and choosing promoters</SPAN></H2><P>We decided that we would separate LuxAB (the luciferase part of the operon) and LuxCDEG (the substrate producing enzymes LuxCDE with LuxG the FMNH2/FMN reducing enzyme) to allow phase-dependent expression of the parts. Using specific promoters of Synechocystis PCC. 6803 we can have fine-tunning of the production of bioluminescence. Recent work on global gene expression in Synechocystis aided on finding adecuate promoters<A href="#7"> 7</A>, <A href="#8"> 8</A> . (Images at right from cited papers.)</P><P>To try our approach, we selected various promoters which could serve the purpose.  Our rational for selecting candidate promoters involved amplitude of oscillation, peak activity, hour, absence of restriction sites, predicted strength of promoter according to the role of the gene and reproducibility between experiments (based on the literature available).  We looked for promoters which would have peak expression nearby dusk hours and that were slightly out of phase to optimize production of bioluminescence according to our mathematical models <B>(LINK OVER HERE!)</B>. We prioritized promoters from genes that would be involved in central energetic metabolism as we believe that their expression would be most robust and reliable.</P><P>We choose the transaldolase promoter <B>(specific name here and code in Synechocystis Genome)</B> to direct the expression of the LuxAB genes and we found a couple of other promoters which filled the other requirements from above. Pcaa3 (NAME HERE AND DESCRIPTION OF ENDOGENOUS ACTIVITY) and PsigE (NAME HERE AND DESCRIPTION OF ENDOGENOUS ACTIVITY), the former being already in Biobrick format (courtesy from the Utah team iGEM 2010).
</P><H2><SPAN class="mw-headline" id="Building_constructs">Building constructs</SPAN></H2><P>
When confronted with the different available strategies to express the genes from the Lux operon in Synechocystis, we concluded that the one that best suits our need is by using integration plasmids. The reason for this is that the available plasmids that replicate in Synechocystis are very large (8 Kb) without even considering the genes we need to include in the constructs (that would sum up to a final 16 Kb aproximately). Such a large plasmid would prove very difficult to handle through molecular biology techniques, let alone transform Synechocystis.

Using integration plasmids also proposes an additional advantage, that is to produce successive integrations which allow accumulation of different desirable elements in the genome. Integration in Synechocystis is undergone through double recombination of homologous DNA which also allows interruption of genes if wanted. In our case we have designed our system to produce suceptibility to copper as a biosafety measure to have further control over our recombinant Synechocystis.
</P><P>
We designed two constructs that have different recombination locations in the Synechocystis chromosome. We named them according to what Utah iGEM team from 2010 proposed for <A href="https://2010.igem.org/Construction_usu.html#Integration_Plasmid_Construction" class="external text" rel="nofollow">naming conventions</A>:
</P><H3><SPAN class="mw-headline" id="pSB1C3_IntK">pSB1C3_IntK</SPAN></H3><A href="http://partsregistry.org/Part:BBa_K743006" class="external text" rel="nofollow">BBa_K743006</A><P>This construct is an integrative plasmid which targets neutral recombination sites (slr0370 and sll0337). We selected this locus because it has been extensively used in the literature (<A href="#11"> 11</A>) and it shown to have no deleterious effects on Synechocystis viability. We selected Kanamycin resistance as our selectable marker. [PUT LINK TO CONSTRUCT HERE].</P><P>
Using this backbone we have decided to put LuxAB under the transaldolase promoter.We have choose the transaldolase promoter to express the luciferase part of the operon, as in the literature the promoter is described as having a peak of expression at 2 hours past dusk, which we believe is just the right timing to &quot;turn on the lamp&quot;.
</P><P>We've found 2 versions of the bacterial luciferase which we will use on this construct. The first one is of <I>Photorhabdus luminiscent</I> K216008 from the 2009 Edinburgh iGEM team and the second one is part from the LuxBrick (K325909 from the 2010 Cambridge iGEM team) and originally comes from <I>Vibrio fisherii</I> but has been &quot;E.coli optimized&quot;.
</P><H3><SPAN class="mw-headline" id="pSB1A3_IntC_.28Utah_2010_iGEM_Team_integration_plasmid.29">pSB1A3_IntC (Utah 2010 iGEM Team integration plasmid)</SPAN></H3><P>We plan on using this plasmid to express the LuxCDEG contructs under the regulation of the Pcaa3 and PsigE promoters mentioned above. <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K390200" class="external text" rel="nofollow">pSB1A3_IntC</A>.</P><H3><SPAN class="mw-headline" id="pSB1C3_IntS">pSB1C3_IntS</SPAN></H3><P>Due to issues mentioned in the results page (PUT LINK HERE) we designed a new plasmid backbone.
This is an integration plasmid which makes Synechocystis susceptible to copper concentrations higher than 0.75 uM <A href="#10">10</A> by disrupting the CopS gene. We believe that this strategy serves as a biosafety measure to avoid the possibility of having a leakage of recombinant DNA to the environment. The plasmid uses Spectynomycin as a selectable marker. [PUT LINK TO CONSTRUCT HERE]
</P><P>We plan on expressing LuxCDEG under the control of the promoters Pcaa3 and PsigE (mentioned above). These promoters have peak activities 1 hour before dusk. We believe that we might enhance bioluminescence yield initially by setting the substrate production/regeneration part of the operon prior to the expression of the luciferase.(LINK TO MODELLING?)
</P><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><DIV id="1"><P>(1) Hohmann-Marriott MF, Blankenship.(2011). Evolution of photosynthesis. Annual Review of Plant Biology Vol. 62: 515-548
</P></DIV><DIV id="2"><P>(2) Jonathan P. Zehr. (2011) Nitrogen fixation by marine cyanobacteria. Trends in microbiology, Vol. 19, 162–17
</P></DIV><DIV id="3"><P>(3) Carl Hirschie Johnson and Susan S. Golden.(1999). CIRCADIAN PROGRAMS IN CYANOBACTERIA: Adaptiveness
and Mechanism. Annual Review of Microbiology, Vol. 53, 389-409
</P></DIV><DIV id="4"><P>(4) Ducat, D. C., Way, J. C., &amp; Silver, P. a. (2011). Engineering cyanobacteria to generate high-value products.
Trends in biotechnology, Vol. 29, 95-103.
</P></DIV><DIV id="5"><P>(5) Huang, H.-H., Camsund, D., Lindblad, P., &amp; Heidorn, T. (2010). Design and characterization of molecular
tools for a Synthetic Biology approach towards developing cyanobacterial biotechnology. Nucleic acids
research, 38, 2577-93.
</P></DIV><DIV id="6"><P>(6) Peca, L., Kós, P. B., Máté, Z., Farsang, A., &amp; Vass, I. (2008). Construction of bioluminescent cyanobacterial
reporter strains for detection of nickel, cobalt and zinc. FEMS microbiology letters, 289, 258-64.
</P></DIV><DIV id="7"><P>(7) Kucho, K.-ichi, Okamoto, K., Tsuchiya, Y., Nomura, S., Nango, M., Kanehisa, M., Ishiura, M., et al. (2005).
Global Analysis of Circadian Expression in the Cyanobacterium Synechocystis sp . Global Analysis of Circadian
Expression in the Cyanobacterium. Society.
</P></DIV><DIV id="8"><P>(8) Layana, C., &amp; Diambra, L. (2011). Time-course analysis of cyanobacterium transcriptome: detecting
oscillatory genes. PloS one, 6, e26291.
</P></DIV><DIV id="9"><P>(9) Kunert, a, Hagemann, M., &amp; Erdmann, N. (2000). Construction of promoter probe vectors for
Synechocystis sp. PCC 6803 using the light-emitting reporter systems Gfp and LuxAB. Journal of
microbiological methods, 41, 185-94.
</P></DIV><DIV id="10"><P>(10)Giner-Lamia, J., Lopez-Maury, L., Reyes, J. C., &amp; Florencio, F. J. (2012). The CopRS two-component system is responsible for resistance to copper in the cyanobacterium Synechocystis sp. PCC 6803. Plant physiology, 159, 1806-1818.
</P></DIV><DIV id="11"><P>(11)Kucho, K., Aoki, K., Itoh, S., &amp; Ishiura, M., (2005). Improvement of the bioluminescence reporter system for real-time monitoring of circadian rhythms in the cyanobacterium Synechocystis sp. strain PCC 6803. Genes Genet. Syst.  80, p. 19–23
</P></DIV><DIV id="12"><P>(12)Meighen, E. a. (1991). Molecular biology of bacterial bioluminescence. Microbiological reviews, 55(1), 123-42. 
</P></DIV><DIV id="13"><P>(13)Dunlap, P. V. (1999). Quorum regulation of luminescence in Vibrio fischeri. Journal of molecular microbiology and biotechnology, 1(1), 5-12. 
</P></DIV><DIV id="14"><P>(14)Luciferase, R. (2001). Differential Transfers of Reduced Flavin Cofactor and Product by Bacterial Flavin. Society, 1749-1754.
</P></DIV><DIV id="15"><P>(15) Kelly, C. J., Hsiung, C.-J., &amp; Lajoie, C. a. (2003). Kinetic analysis of bacterial bioluminescence. Biotechnology and bioengineering, 81(3), 370-8. doi:10.1002/bit.10475
</P></DIV><DIV id="16"><P>(16)Tehrani, G. A., Mirzaahmadi, S., Bandehpour, M., &amp; Laloei, F. (2011). Molecular cloning and expression of the luciferase coding genes of Vibrio fischeri. Journal of Biotechnology, 10(20), 4018-4023. doi:10.5897/AJB10.2363
</P></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UC_Chile2/Cyanolux/Project">http://2012.igem.org/Team:UC_Chile2/Cyanolux/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Chile2/Cyanolux/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Chile2/Cyanolux/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Chile2/Cyanolux/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Chile2/Cyanolux/Project&amp;oldid=182853" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>