"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Chile2/General Protocols
From 2012.igem.org
Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012
Team »
Roster
PSB Lab
Photo gallery
Luxilla »
Project
Results
Modelling
Future Prospects
Biolamp
Biobricks
Notebook
Side projects »
Bactomithril
Gibson-small parts
Syne growth curve
Int_C debugging
LuxBrick char.
Methodologies »
Protocols
Arduino
DIY
Software
Human Practices »
Synthetic Sociology
Biosafety
SynBio Workshop
More »
Collaborations
Judging Criteria
Attribution
Contents
1Growth Media
1.1LB media
1.2SOB media
2Buffers
2.15X Gibson Assembly Isothermal Buffer
2.21.33X Gibson Assembly Master Mix
3DNA assembly protocols
3.1Standard Assembly
3.1.1Digestion reaction
3.1.2Ligation reaction
3.2Gibson Assembly
3.2.1Design primers
3.2.2Obtaining parts
3.2.3Assembly reaction
3.2.4Checking assembly
4E.coli Transformation
Growth Media
LB media
Final concentrations:
1% (w/v) tryptone
0.5% (w/v) yeast extract
86 mM NaCl
Per liter:
10g Tryptone
5g Yeast extract
5g NaCl
15g Agar -Optional-
Adjust pH to 7.5 with NaOH 1M
SOB media
Final concentrations:
0.5% (w/v) yeast extract
2% (w/v) tryptone
10 mM NaCl
2.5 mM KCl
20 mM MgSO4
Per liter:
5 g yeast extract
20 g tryptone
0.584 g NaCl
0.186 g KCl
2.4 g MgSO4
15g Agar -Optional-
Adjust pH to 7.5 with NaOH 1M
Buffers
5X Gibson Assembly Isothermal Buffer
Final Concentrations:
25% PEG MW 8000
500mM Tris HCl pH 7.5
50mM CaCl
50mM DTT
1mM dATP
1mM dTTP
1mM dGTP
1mM dCTP
5mM NAD+
H20 nuclease free
For a 2 mL stock (equivalent to 20 100uL 5X isothermal buffers)
0.5g PEG MW 8000
1.5mL Tris-HCl pH 7.5 1M
Swivel for 30 minutes in a orbital oscillator for PEG to dissolve fully
50uL CaCl 2M
100uL DTT 1M
20uL dATP 100mM
20uL dTTP 100mM
20uL dGTP 100mM
20uL dCTP 100mM
200uL NAD+ 200mM
H20 nuclease free up to 2mL
Alicuot into 100uL stocks
Store at -80°C
1.33X Gibson Assembly Master Mix
For a 375ul 1.33X Gibson Assembly Master Mix
100uL 5X Gibson Assembly Isothermal Buffer
6.25uL Phusion Polymerase 2 U/uL (cat N° F-350S) from Thermo Scientific
2uL T5 Exonuclease 1U/uL (cat N° T5E4111K) from Epicentre
It is important to dilute the T5 Exonuclease stock from 10U/uL to 1U/ul to measure the volume correctly
50uL Taq DNA Ligase 2000 U/uL (cat N° M208S) from NEB
216.75uL of nuclease free H20
Alicuot 9uL in 0.2mL PCR tubes. This will yield about 42 reactions
DNA assembly protocols
Standard Assembly
The standard assembly reaction relies on the standarization of Biobricks to join different DNA parts. Plasmids from the registry of standard parts allow joining of DNA parts by using a combination of specific enzymes to cut a Upstream Biobrick with the restriction enzymes EcoRI and SpeI, and a Downstream Biobrick with the restriction enzymes XbaI and PstI into a Destination plasmid which has been cut with the restriction enzymes EcoRI and PstI. The reaction yields the Upstream and Downstream part joined by a mixed restriction site and allows further elongation of the construct by the same strategy. It is important to notice that such a reaction requires purification of the digested parts if any of the plasmids (Upstream, Downstream or Destination) share a resistance marker.
Upstream part enzymes: EcoRI & SpeI
Downstream part enzymes: XbaI & PstI
Destination backbone enzymes: EcoRI & PstI
Digestion reaction
X volume of DNA to 500ng of plasmid
(42.5 - X)uL of nuclease free water
5uL of NEB buffer 2
0.5uL of BSA 100X
1uL of Enzyme 1
1uL of Enzyme 2
Incubate at 37°C for 2 hours
Heat inactivate enzymes at 80°C for 20 minutes
Ligation reaction
2uL of digested Upstream part
2uL of digested Downstream part
2uL of digested Destination plasmid
2uL of T4 DNA ligase buffer
11uL of nuclease free water
1uL of T4 DNA ligase
Incubate at room temperature for 10 minutes
Heat inactivate at 80°C for 20 minutes
Transform
Gibson Assembly
Gibson Assembly is a DNA assembly method created by Daniel Gibson during the development of the first Synthetic Genome (Synthia) (reference YYY). Its adaptation to a cloning method allows fast and accurate production of increasingly complex constructions. The strategy behind the method relies on PCR to obtain different parts which share a 40bp homology region, and a 3 enzyme reaction which produces cohesive ends, fills the gaps between the parts and ligates the resulting construct into a scarless assembly of various (>2) parts.
Design primers
The easiest way to design primers to obtain amplicons with the required overlaps (40bp final overlaps) is to make an in sillico design of the final construct
Design forward primer of right amplicon of joint in 5'->3' direction
Calculate length of annealing part of primer as to reach a Tm of approximately 63°C
Add 20 bp of overlap
Design reverse primer of left amplicon of joint in 5'->3' direction
To do this, select full joint and "reverse complement" it (be sure to be able to discriminate between right and left parts of joint)
Calculate length of annealing part of primer as to reach a Tm of approximately 63°C
Add 20 bp of overlap
Apply same principle in all joints
Obtaining parts
PCR parts using Phusion Polymerase datasheet indications. It is important to use a low ammount of template plasmid (10pg) as to reduce possibility of carry-over during band purification
Run agarose gel electrophoresis on a adecuate gel (50bp to 200bp parts should be run on a 3% w/v agarose gel, larger parts should be run on a 1% w/v to 1.5% w/v agarose gel) until clear distinction of bands is achieved. It is recommended that gels should be exposed as little as possible to UV transilluminators as UV light damages DNA.
Cut band and proceed with band purification. Elute in smallest volume as possible according to your kit specification
Quantify purified DNA
Assembly reaction
The assembly reaction is composed of 9uL of 1.33X Gibson assembly master mix + 3uL of purified parts DNA
The calculation of the ratio at which your DNA parts are should be in correspondence to the level of competence of your cells. We have found that with our cells (5*10^⁸ colonies/ug of pUC19 DNA) the following ratios work well
Calculate the amount of pmoles of each purified DNA part using the following equation: pmoles of DNA = weigth in ng * 1000 (conversion factor from nano to pico) / (650 Daltons * base pair length of part)
We have seen that using 0.01 picomoles of template (part with the selection resistance) and the rest of parts at 0.03 picomoles and above, yield a high ratio of true positive transformants and a decent number of colonies to check
Thoroughly mix the 3uL of purified parts DNA with the 9uL of 1.33X Gibson assembly master mix in ice
Directly incubate the reaction at 50°C for 1 hour
Transform competent cells with 8uL of assembled DNA (that leaves 4uL to check in an agarose gel if the reaction fails)
Checking assembly
Check transformant colonies by colony PCR using primers for whole insert
Grow positive colonies in media with corresponding antibiotic
Miniprep colonies and digest plasmid with EcoRI & PstI restriction enzymes
If checking protocols validate until now, proceed by sequencing to corroborate
E.coli Transformation
Keep 50uL of chemically competent cells in ice (for no more than 10 minutes until transformation)
Mix 0.01 to 1 total ng of plasmid with competent cells (depending on size of plasmid and competence of your cells)
Flick very softly to mix (competent cells are very fragile)
Leave on ice for 30 minutes
Heat shock at 42°C for 60 seconds
Add 250uL of LB media
Incubate at 37°C in a rotating shaker for 1 hour (Ampicillin) or 2 hours (Chloramphenicol and Kanamycin)
Plate 50uL of transformed cells to a LB agar petri dish with corresponding antibiotic (30ug/mL Kanamycin or 30ug/mL Chloramphenicol or 100ug/mL Ampicillin)
Retrieved from " http://2012.igem.org/Team:UC_Chile2/General_Protocols "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
