<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Chile2_Protocols"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Chile2/Protocols">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Chile2/Protocols&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile2/Protocols&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile2/Protocols&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Chile2/Protocols" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Chile2/Protocols</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012</TITLE></P><DIV id="banner"><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Team/Members">Team <SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Members">Roster</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/PSB_Lab">PSB Lab</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Gallery">Photo gallery</A></LI></UL><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Luxilla<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Project</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Results_short">Results</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Future">Future Prospects</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biolamp">Biolamp</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biobricks">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Notepad">Notebook</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Side projects<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Bactomithril</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Gibson">Gibson-small parts</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Growthcurve">Syne growth curve </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Int_C">Int_C debugging</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/LuxBrick">LuxBrick char.</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Protocols">Methodologies<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Arduino">Arduino</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/DIY">DIY</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Other">Software</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Human Practices<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Synthetic Sociology</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Biosafety">Biosafety</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Workshop">SynBio Workshop</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Collaborations">More<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Collaborations">Collaborations</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Accomplishments">Judging Criteria </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Attribution">Attribution</A></LI></DIV><P>Here you will find various protocols we have used during the development of our project. We have separated them into 4 main sections: <A href="#General_Protocols">#General Protocols</A><A href="#Cyanobacteria_Protocols">#Cyanobacteria Protocols</A><A href="#Bactomithril_Protocols">#Bactomithril Protocols</A><A href="#Characterization_Protocols">#Characterization Protocols</A></P><DIV id="General_Protocols"><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#General_Protocols"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">General Protocols</SPAN></A></LI><LI class="toclevel-2"><A href="#Growth_media"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Growth media</SPAN></A></LI><LI class="toclevel-3"><A href="#LB_media"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">LB media</SPAN></A></LI><LI class="toclevel-3"><A href="#SOB_media"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">SOB media</SPAN></A></LI><LI class="toclevel-3"><A href="#BG-11"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">BG-11</SPAN></A></LI><LI class="toclevel-3"><A href="#BG-110"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">BG-110</SPAN></A></LI><LI class="toclevel-3"><A href="#BG-11C"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">BG-11C</SPAN></A></LI><LI class="toclevel-3"><A href="#Concentrated_salts_solution_100X"><SPAN class="tocnumber">1.1.6</SPAN><SPAN class="toctext">Concentrated salts solution 100X</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Buffers"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Buffers</SPAN></A></LI><LI class="toclevel-3"><A href="#5X_Gibson_Assembly_Isothermal_Buffer"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">5X Gibson Assembly Isothermal Buffer</SPAN></A></LI><LI class="toclevel-3"><A href="#1.33X_Gibson_Assembly_Master_Mix"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">1.33X Gibson Assembly Master Mix</SPAN></A></LI><LI class="toclevel-2"><A href="#DNA_assembly_protocols"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">DNA assembly protocols</SPAN></A></LI><LI class="toclevel-3"><A href="#Standard_Assembly"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">Standard Assembly</SPAN></A></LI><LI class="toclevel-4"><A href="#Digestion_reaction"><SPAN class="tocnumber">1.3.1.1</SPAN><SPAN class="toctext">Digestion reaction</SPAN></A></LI><LI class="toclevel-4"><A href="#Ligation_reaction"><SPAN class="tocnumber">1.3.1.2</SPAN><SPAN class="toctext">Ligation reaction</SPAN></A></LI><LI class="toclevel-3"><A href="#Gibson_Assembly"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Gibson Assembly</SPAN></A></LI><LI class="toclevel-4"><A href="#Primer_design"><SPAN class="tocnumber">1.3.2.1</SPAN><SPAN class="toctext">Primer design</SPAN></A></LI><LI class="toclevel-4"><A href="#Obtaining_parts"><SPAN class="tocnumber">1.3.2.2</SPAN><SPAN class="toctext">Obtaining parts</SPAN></A></LI><LI class="toclevel-4"><A href="#Assembly_reaction"><SPAN class="tocnumber">1.3.2.3</SPAN><SPAN class="toctext">Assembly reaction</SPAN></A></LI><LI class="toclevel-4"><A href="#Checking_assembly"><SPAN class="tocnumber">1.3.2.4</SPAN><SPAN class="toctext">Checking assembly</SPAN></A></LI><LI class="toclevel-2"><A href="#E.coli_Transformation"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">E.coli Transformation</SPAN></A></LI><LI class="toclevel-2"><A href="#Chemically_Competent_cells"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Chemically Competent cells</SPAN></A></LI><LI class="toclevel-1"><A href="#Cyanobacteria_Protocols"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Cyanobacteria Protocols</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_of_Synechocystis_PCC._6803"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Transformation of Synechocystis PCC. 6803</SPAN></A></LI><LI class="toclevel-3"><A href="#Materials"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-3"><A href="#To_be_prepared_previously"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">To be prepared previously</SPAN></A></LI><LI class="toclevel-3"><A href="#Transformation"><SPAN class="tocnumber">2.1.3</SPAN><SPAN class="toctext">Transformation</SPAN></A></LI><LI class="toclevel-3"><A href="#Plate_preparation"><SPAN class="tocnumber">2.1.4</SPAN><SPAN class="toctext">Plate preparation</SPAN></A></LI><LI class="toclevel-2"><A href="#Reinoculation"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Reinoculation</SPAN></A></LI><LI class="toclevel-2"><A href="#Cyanobacterial_DNA_extraction"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Cyanobacterial DNA extraction</SPAN></A></LI><LI class="toclevel-1"><A href="#Bactomithril_Protocols"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Bactomithril Protocols</SPAN></A></LI><LI class="toclevel-1"><A href="#Characterization_Protocols"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Characterization Protocols</SPAN></A></LI><LI class="toclevel-2"><A href="#Methods_for_the_characterization_of_Synechocystis_PCC_6803_growth_curve"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Methods for the characterization of Synechocystis PCC 6803 growth curve</SPAN></A></LI><LI class="toclevel-3"><A href="#Materials_2"><SPAN class="tocnumber">4.1.1</SPAN><SPAN class="toctext">Materials</SPAN></A></LI><LI class="toclevel-2"><A href="#Methods_for_BBa_K325909_.28LuxBrick.29_characterization"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Methods for BBa_K325909 (LuxBrick) characterization</SPAN></A></LI><LI class="toclevel-2"><A href="#Methods_for_Gibson_Assembly_for_small_parts_characterization"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Methods for Gibson Assembly for small parts characterization</SPAN></A></LI><LI class="toclevel-1"><A href="#References"><SPAN class="tocnumber">5</SPAN><SPAN class="toctext">References</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="General_Protocols">General Protocols</SPAN></H1></DIV><DIV id="Growth_media"><H2><SPAN class="mw-headline" id="Growth_media">Growth media</SPAN></H2></DIV><H3><SPAN class="mw-headline" id="LB_media">LB media</SPAN></H3><UL><P>Final concentrations:</P><LI>1% (w/v) tryptone</LI><LI>0.5% (w/v) yeast extract</LI><LI>86 mM NaCl</LI><P>Per liter:</P><LI>10g Tryptone</LI><LI>5g Yeast extract</LI><LI>5g NaCl</LI><LI>15g Agar -Optional-</LI><LI>Adjust pH to 7.5 with NaOH 1M</LI></UL><H3><SPAN class="mw-headline" id="SOB_media">SOB media</SPAN></H3><UL><P>Final concentrations:</P><LI>0.5% (w/v) yeast extract</LI><LI>2% (w/v) tryptone</LI><LI>10 mM NaCl</LI><LI>2.5 mM KCl</LI><LI>20 mM MgSO4</LI><P>Per liter:</P><LI>5 g yeast extract</LI><LI>20 g tryptone</LI><LI>0.584 g NaCl</LI><LI>0.186 g KCl</LI><LI>2.4 g MgSO4</LI><LI>15g Agar -Optional-</LI><LI>Adjust pH to 7.5 with NaOH 1M</LI></UL><H3><SPAN class="mw-headline" id="BG-11">BG-11</SPAN></H3><P>For 1L:
</P><UL><LI>1.5gr NaNO3</LI><LI>1L distilled H2O </LI><LI>10mL Concentrated salts solution 100X (4°C)</LI><LI>200µL H2PO4 1M (Room T°)</LI><LI>10gr BactoAgar -optional-</LI></UL><H3><SPAN class="mw-headline" id="BG-110">BG-110</SPAN></H3><P>For 1L:
</P><UL><LI>1L distilled H2O </LI><LI>10mL Concentrated salts solution 100X (4°C)</LI><LI>200µL H2PO4 1M (Room T°)</LI><LI>10gr BactoAgar -optional-</LI></UL><H3><SPAN class="mw-headline" id="BG-11C">BG-11C</SPAN></H3><P>
For 1L:
</P><UL><LI>1.5gr NaNO3</LI><LI>1L distilled H2O </LI><LI>10mL Concentrated salts solution 100X (4°C)</LI><LI>200µL H2PO4 1M (Room T°)</LI><LI>0.84gr NaHCO3</LI><LI>10gr BactoAgar -opcional-</LI></UL><H3><SPAN class="mw-headline" id="Concentrated_salts_solution_100X">Concentrated salts solution 100X</SPAN></H3><P>For 1L:
</P><UL><LI>7.5gr SO4Mg * 7H2O</LI><LI>3.6gr CaCl2 * 2H2O</LI><LI>0.6gr Citric Acid</LI><LI>0.6gr Ammonium Ferric Citrate</LI><LI>0.093gr EDTA-Na2 </LI><LI>2.0gr CO3 Na2 </LI><LI>0.286gr BO3H3</LI><LI>0.181gr MnCl2 * 4H2O</LI><LI>0.022gr ZnSO4 * 7H2O</LI><LI>0.039gr NaMoO4 * 2H2O</LI><LI>0.0079gr CuSO4 * 5H2O</LI><LI>0.0049gr CoCl2 * 6H2O</LI><LI>1L distilled H2O </LI></UL><P>Autoclave and store at 4°C.
</P><DIV id="Buffers"><H2><SPAN class="mw-headline" id="Buffers">Buffers</SPAN></H2></DIV><H3><SPAN class="mw-headline" id="5X_Gibson_Assembly_Isothermal_Buffer">5X Gibson Assembly Isothermal Buffer</SPAN></H3><UL><P>Final Concentrations:</P><LI>25% PEG MW 8000</LI><LI>500mM Tris HCl pH 7.5</LI><LI>50mM CaCl</LI><LI>50mM DTT</LI><LI>1mM dATP</LI><LI>1mM dTTP</LI><LI>1mM dGTP</LI><LI>1mM dCTP</LI><LI>5mM NAD+</LI><LI>H20 nuclease free</LI><P>For a 2 mL stock (equivalent to 20 100uL 5X isothermal buffers)</P><LI>0.5g PEG MW 8000</LI><LI>1.5mL Tris-HCl pH 7.5 1M</LI><P>Swivel for 30 minutes in a orbital oscillator for PEG to dissolve fully</P><LI>50uL CaCl 2M</LI><LI>100uL DTT 1M</LI><LI>20uL dATP 100mM</LI><LI>20uL dTTP 100mM</LI><LI>20uL dGTP 100mM</LI><LI>20uL dCTP 100mM</LI><LI>200uL NAD+ 200mM</LI><LI>H20 nuclease free up to 2mL</LI><P>Alicuot into 100uL stocks</P><P><B>Store at -80°C</B></P></UL><H3><SPAN class="mw-headline" id="1.33X_Gibson_Assembly_Master_Mix">1.33X Gibson Assembly Master Mix</SPAN></H3><UL><P>For a 375ul 1.33X Gibson Assembly Master Mix</P><LI>100uL 5X Gibson Assembly Isothermal Buffer</LI><LI>6.25uL Phusion Polymerase 2 U/uL (cat N° F-350S) from Thermo Scientific </LI><LI>2uL T5 Exonuclease 1U/uL (cat N° T5E4111K) from Epicentre</LI><P>It is important to dilute the T5 Exonuclease stock from 10U/uL to 1U/ul to measure the volume correctly</P><LI>50uL Taq DNA Ligase 2000 U/uL (cat N° M208S) from NEB</LI><LI>216.75uL of nuclease free H20</LI><P><B>Alicuot 9uL in 0.2mL PCR tubes. This will yield about 42 reactions</B></P></UL><DIV id="DNA_assembly_protocols"><H2><SPAN class="mw-headline" id="DNA_assembly_protocols">DNA assembly protocols</SPAN></H2></DIV><H3><SPAN class="mw-headline" id="Standard_Assembly">Standard Assembly</SPAN></H3><P>The standard assembly reaction relies on the standarization of Biobricks to join different DNA parts. Plasmids from the registry of standard parts allow joining of DNA parts by using a combination of specific enzymes to cut a Upstream Biobrick with the restriction enzymes EcoRI and SpeI, and a Downstream Biobrick with the restriction enzymes XbaI and PstI into a Destination plasmid which has been cut with the restriction enzymes EcoRI and PstI. The reaction yields the Upstream and Downstream part joined by a mixed restriction site and allows further elongation of the construct by the same strategy. It is important to notice that such a reaction requires purification of the digested parts if any of the plasmids (Upstream, Downstream or Destination) share a resistance marker.</P><UL><LI>Upstream part enzymes: EcoRI &amp; SpeI</LI><LI>Downstream part enzymes: XbaI &amp; PstI</LI><LI>Destination backbone enzymes: EcoRI &amp; PstI</LI></UL><H4><SPAN class="mw-headline" id="Digestion_reaction">Digestion reaction</SPAN></H4><UL><LI>X volume of DNA to 500ng of plasmid</LI><LI>(42.5 - X)uL of nuclease free water</LI><LI>5uL of NEB buffer 2</LI><LI>0.5uL of BSA 100X</LI><LI>1uL of Enzyme 1</LI><LI>1uL of Enzyme 2</LI><LI>Incubate at 37°C for 2 hours</LI><LI>Heat inactivate enzymes at 80°C for 20 minutes</LI></UL><H4><SPAN class="mw-headline" id="Ligation_reaction">Ligation reaction</SPAN></H4><UL><LI>2uL of digested Upstream part</LI><LI>2uL of digested Downstream part</LI><LI>2uL of digested Destination plasmid</LI><LI>2uL of T4 DNA ligase buffer</LI><LI>11uL of nuclease free water</LI><LI>1uL of T4 DNA ligase</LI><LI>Incubate at room temperature for 10 minutes</LI><LI>Heat inactivate at 80°C for 20 minutes</LI><LI><A href="#E.coli_Transformation">Transform</A></LI></UL><H3><SPAN class="mw-headline" id="Gibson_Assembly">Gibson Assembly</SPAN></H3><P>Gibson Assembly is a DNA assembly method created by Daniel Gibson during the development of the first Synthetic Genome (Synthia) <A href="#2">2</A>. Its adaptation to a cloning method allows fast and accurate production of increasingly complex constructions. The strategy behind the method relies on PCR to obtain different parts which share a 40bp homology region, and a 3 enzyme reaction which produces cohesive ends, fills the gaps between the parts and ligates the resulting construct into a scarless assembly of various (&gt;2) parts.</P><H4><SPAN class="mw-headline" id="Primer_design">Primer design</SPAN></H4><P>The easiest way to design primers to obtain amplicons with the required overlaps (40bp final overlaps) is to make an <I>in sillico</I> design of the final construct</P><UL><LI>Design forward primer of right amplicon of joint in 5'-&gt;3' direction</LI><LI>Calculate length of annealing part of primer as to reach a Tm of approximately 63°C</LI><LI>Add 20 bp of overlap</LI><LI>Design reverse primer of left amplicon of joint in 5'-&gt;3' direction</LI><LI>To do this, select full joint and &quot;reverse complement&quot; it (be sure to be able to discriminate between right and left parts of joint)</LI><LI>Calculate length of annealing part of primer as to reach a Tm of approximately 63°C</LI><LI>Add 20 bp of overlap</LI><LI>Apply same principle in all joints</LI></UL><H4><SPAN class="mw-headline" id="Obtaining_parts">Obtaining parts</SPAN></H4><UL><LI>PCR parts using Phusion Polymerase datasheet indications. It is important to use a low ammount of template plasmid (10pg) as to reduce possibility of carry-over during band purification</LI><LI>Run agarose gel electrophoresis on a adecuate gel (50bp to 200bp parts should be run on a 3% w/v agarose gel, larger parts should be run on a 1% w/v to 1.5% w/v agarose gel) until clear distinction of bands is achieved. It is recommended that gels should be exposed as little as possible to UV transilluminators as UV light damages DNA.</LI><LI>Cut band and proceed with band purification. Elute in smallest volume as possible according to your kit specification</LI><LI>Quantify purified DNA</LI></UL><H4><SPAN class="mw-headline" id="Assembly_reaction">Assembly reaction</SPAN></H4><P>The assembly reaction is composed of 9uL of 1.33X Gibson assembly master mix + 3uL of purified parts DNA</P><UL><LI>The calculation of the ratio at which your DNA parts are should be in correspondence to the level of competence of your cells. We have found that with our cells (5*10^⁸ colonies/ug of pUC19 DNA) the following ratios work well</LI><LI>Calculate the amount of pmoles of each purified DNA part using the following equation: pmoles of DNA = weigth in ng * 1000 (conversion factor from nano to pico) / (650 Daltons * base pair length of part)</LI><LI>We have seen that using 0.01 picomoles of template (part with the selection resistance) and the rest of parts at 0.03 picomoles and above, yield a high ratio of true positive transformants and a decent number of colonies to check</LI><LI>Thoroughly mix the 3uL of purified parts DNA with the 9uL of 1.33X Gibson assembly master mix in ice</LI><LI>Directly incubate the reaction at 50°C for 1 hour</LI><LI><A href="#E.coli_Transformation"> Transform</A> competent cells with 8uL of assembled DNA (that leaves 4uL to check in an agarose gel if the reaction fails)</LI></UL><H4><SPAN class="mw-headline" id="Checking_assembly">Checking assembly</SPAN></H4><UL><LI>Check transformant colonies by colony PCR using primers for whole insert</LI><LI>Grow positive colonies in media with corresponding antibiotic</LI><LI>Miniprep colonies and digest plasmid with EcoRI &amp; PstI restriction enzymes</LI><LI>If checking protocols validate until now, proceed by sequencing to corroborate</LI></UL><DIV id="E.coli_Transformation"><H2><SPAN class="mw-headline" id="E.coli_Transformation">E.coli Transformation</SPAN></H2></DIV><UL><LI>Keep 50uL of <A href="#Competent_Cells"> chemically competent cells</A> in ice (for no more than 10 minutes until transformation)</LI><LI>Mix 0.01 to 1 total ng of plasmid with competent cells (depending on size of plasmid and competence of your cells)</LI><P>Flick very softly to mix (competent cells are very fragile)</P><LI>Leave on ice for 30 minutes</LI><LI>Heat shock at 42°C for 60 seconds</LI><LI>Add 250uL of <A href="#Growth_Media"> LB media</A></LI><LI>Incubate at 37°C in a rotating shaker for 1 hour (Ampicillin) or 2 hours (Chloramphenicol and Kanamycin)</LI><LI>Plate 50uL of transformed cells to a LB agar petri dish with corresponding antibiotic (30ug/mL Kanamycin or 30ug/mL Chloramphenicol or 100ug/mL Ampicillin)</LI></UL><P>&lt;/div&gt;</P><DIV id="Competent_Cells"><H2><SPAN class="mw-headline" id="Chemically_Competent_cells">Chemically Competent cells</SPAN></H2><P>To prepare our competent cells we used a protocol we found in openwetware.org. Starting from commercial TOP10 cells as the starting inoculus for the whole protocol, we get competence levels close to 5*10⁸ cfu/ug pUC19 DNA</P><P><A href="http://openwetware.org/wiki/TOP10_chemically_competent_cells" class="external text" rel="nofollow">Openwetware protocol here</A></P><DIV id="Cyanobacteria_Protocols"><H1><SPAN class="mw-headline" id="Cyanobacteria_Protocols">Cyanobacteria Protocols</SPAN></H1></DIV><DIV id="Transformation_of_Synechocystis"><H2><SPAN class="mw-headline" id="Transformation_of_Synechocystis_PCC._6803">Transformation of Synechocystis PCC. 6803</SPAN></H2></DIV><P>By iGEM - UC_Chile Team 2012 (Based on <A href="#1"> 1</A>)
</P><H3><SPAN class="mw-headline" id="Materials">Materials</SPAN></H3><UL><LI>Liquid BG-11 Media</LI><LI>Solid BG-11 Media</LI><LI>Sterile 250 mL Erlenmeyer flask</LI><LI>Sterile 50 mL Falcon tube (2)</LI><LI>Sterile 1.5ml Eppendorf  tubes</LI><LI>Petri Dishes</LI><LI>Millipore Membrane Filters, 0.45um HA (HAWP09000) </LI><LI>Pasteur pipette</LI><LI>Antibiotics (according to your selection resistance)</LI><LI>DNA for transformation</LI></UL><H3><SPAN class="mw-headline" id="To_be_prepared_previously">To be prepared previously</SPAN></H3><P>
Concentrate the DNA to be transformed into Synechocystis to 1µg/µL in 10µL for each transformation (A total of 10 ug of the transformation plasmid). 

Inoculate Synechocystis PCC 6803 into a sterile 250mL Erlenmeyer flask with 150mL* of liquid BG-11 and grow culture until you reach an OD730nm  of 0.8 – 1.0 (That should take between 6 to 10 days dependending on the amount of initial inoculum). 
</P><UL><LI> For each 50 mL of Synechocystis PCC 6803 you will have 5 individual transformations. 
</LI></UL><P>
Once you have reached the OD730nm and have enough DNA, proceed to transform.
</P><H3><SPAN class="mw-headline" id="Transformation">Transformation</SPAN></H3><P>All subsequent actions are to be realized under sterile conditions.
</P><UL><LI>1.-Transfer Synechocystis cells into 50 mL Falcon tubes and centrifuge 10 minutes at 2760g (Room temperature)</LI><LI>2.- Wash cells: </LI><P>Discard supernatant and resuspend pellet in the following manner:
</P><UL>- Add 1mL of liquid BG-11 and softly pipet cells until resuspended.</UL><UL>- Add 4mL of liquid BG-11 and pipet cells until completely mixed.</UL><UL>- Add 5ml of BG-11 and pipet cells.</UL><UL><LI> It is important that cells are throroughly resuspended so that recieve fresh media and do not form aggregates.
</LI></UL><LI>3.- Centrifuge 10 minutos at 2760g (Room temperature).</LI><LI>4.- Discard supernatant and extract any remaining media by pipeting.</LI><LI>5.- Resuspend pellet with 1mL of BG-11 media. Once resuspended, add 0.5 mL more of BG-11. </LI><LI>6.- Distribuite 300 uL of resuspended cells into sterile 1,5 mL Eppendorf tube.</LI><LI>* Make sure to pipet into the bottom of the tube so that none of the cells remain in the walls of the tube.</LI><LI>7.- Add 10 μL of plasmid DNA (1μg/μL) to the tube containing the 300 uL of the Synechocystis cells.</LI><LI>Softly mix contents with the point of the pipet. Repeat the procedure to all but the transformation control tube (without DNA). </LI><LI>8.- Leave the tubes in a light chamber (50 uE/s²/m²) at 30°C for 5 hours.  Flick the tubes at 2.5 hours. </LI></UL><H3><SPAN class="mw-headline" id="Plate_preparation">Plate preparation</SPAN></H3><P>
Prepare 50 mL BG-11 plates with no antibiotic for the transformation day.
Put Millipore membrane filters on BG-11 plates. 
</P><UL><LI>9.- After the 5 hours of recuperation, take the celular suspension and spread over the Millipore membrane. The suspension is to be spreaded carefully over the whole membrane using a sterile bended Pasteur pipete.</LI><LI>10.- Let the membranes dry before transfering the plates to the growth chamber.</LI><LI>11.- Transfer plates to the growth chamber putting the membrane facing up and leave at 30°C for 12 hours in light.</LI><LI>12.- Transfer membranes to BG-11 plates that contain half of the concentration of the antibiotic (12.5 ug/mL of antibiotic) and leave growing for 3 days at 30°C in light. </LI><LI>13.- After the 3 days have passed, transfer the membrane to BG-11 plates with full concentration of antibiotic (25ug/mL).</LI></UL><P><I>Colonies should appear between 1 to 2 weeks after the transformation. First you will see most of the cellular suspension dissapear and slowly small colonies should appear in the plate.</I></P><DIV id="Reinoculation"><H2><SPAN class="mw-headline" id="Reinoculation">Reinoculation</SPAN></H2><P>Protocol right here!
</P></DIV><DIV id="Cyanobacterial_DNA_extraction"><H2><SPAN class="mw-headline" id="Cyanobacterial_DNA_extraction">Cyanobacterial DNA extraction</SPAN></H2></DIV><P>Protocol goes here!</P><DIV id="Bactomithril_Protocol"><H1><SPAN class="mw-headline" id="Bactomithril_Protocols">Bactomithril Protocols</SPAN></H1></DIV><P>You can find a complete protocol to produce spider-silk fibers in the following paper:

Teulé, F., Cooper, A. R., Furin, W. a, Bittencourt, D., Rech, E. L., Brooks, A., &amp; Lewis, R. V. (2009). A protocol for the production of recombinant spider silk-like proteins for artificial fiber spinning. Nature protocols, 4(3), 341-55. doi:10.1038/nprot.2008.250 

Other easier protocols are in the following adresses:<A href="http://www.pnas.org/content/107/32/14059.long" class="external free" rel="nofollow">http://www.pnas.org/content/107/32/14059.long</A><A href="http://onlinelibrary.wiley.com/doi/10.1002/jbm.a.34353/full" class="external free" rel="nofollow">http://onlinelibrary.wiley.com/doi/10.1002/jbm.a.34353/full</A></P><DIV id="Characterization_Protocols"><H1><SPAN class="mw-headline" id="Characterization_Protocols">Characterization Protocols</SPAN></H1></DIV><DIV id="SyneGrowth"><H2><SPAN class="mw-headline" id="Methods_for_the_characterization_of_Synechocystis_PCC_6803_growth_curve">Methods for the characterization of Synechocystis PCC 6803 growth curve</SPAN></H2><P>This are the methods we used for setting up the growth curve experiment of our Synechocystis PCC 6803 which is further described <A href="/Team:UC_Chile2/Characterization" title="Team:UC Chile2/Characterization"> here </A>.
</P><H3><SPAN class="mw-headline" id="Materials_2">Materials</SPAN></H3><UL><LI>4 sterile (autoclaved) 250 mL Erlenmeyer flasks</LI><LI>1 liter of BG-11</LI><LI>250 mL of BG-110</LI><LI>Axenic culture of Synechocystis PCC 6803</LI><LI>Sterile filtered 1 mL tips</LI><LI>Spectrophotometer</LI><LI>5 1 mL Cuvettes</LI></UL><P>All procedures should be realized under sterile conditions, preferably in a laminar flow hood.
</P><UL><LI>Measure 150 mL of BG-11 with a sterile 50 mL Falcon tube and put into each of the 3 Erlenmeyer flasks.</LI><LI>Measure 150 mL of BG-110 with a sterile 50 mL Falcon tube and put into the remaining Erlenmeyer flask.</LI><LI>Inoculate 1 mL of axenic Synechocystis PCC 6803 with and OD730 of X to each flask (including the one with BG-110). </LI><LI>Place flasks in rotating incubator and try to avoid leaving them out of the incubator for long when taking measurements</LI><LI>Measure OD600 and OD730 twice a day (preferably with at least 8 hours of difference between measurements)</LI><LI>Repeat measurements for 14 days</LI></UL></DIV><DIV id="LuxBrick_Characterization"><H2><SPAN class="mw-headline" id="Methods_for_BBa_K325909_.28LuxBrick.29_characterization">Methods for <A href="http://partsregistry.org/Part:BBa_K325909" class="external text" rel="nofollow">BBa_K325909 (LuxBrick)</A> characterization</SPAN></H2></DIV><UL><LI> Pick a single colony from a transformed plate of K325909. Grow in LB media with chloramphenicol 30ug/mL O/N at 30°C.</LI><LI>Measure OD at 600nm, then dilute to OD600 = 0.6 with growth media following direct C*V ratio to obtain bacterial stock for experiments.</LI><LI>Induce with 3mM (L)-Arabinose (final concentration)</LI><LI> Alicuot 600uL of bacteria and add LB media with chloramphenicol 30ug/mL + additives and inducers up to 1.5mL. This will yield a 0.24 OD600nm.</LI><LI> Grow at 30°C for 1.5 hours (or at chosen T° condition for characterization).</LI><LI> Alicuot 100uL into 4 wells of a 96 well falcon plaque.</LI><LI> Measure luminescence in a ThermoScientific Luminoskan Ascent Microplate Luminometer.</LI></UL><DIV id="Gibson_Assembly_for_small_parts_Characterization"><H2><SPAN class="mw-headline" id="Methods_for_Gibson_Assembly_for_small_parts_characterization">Methods for Gibson Assembly for small parts characterization</SPAN></H2></DIV><P>The experiment for the characterization of T5 exonuclease concentration in Gibson Assembly for small parts was set as follows:
</P><UL><LI>After the design of the segment to be assembled as described in the <A href="/Team:UC_Chile2/Results/Gibson#Gibson_Assembly_for_small_parts" title="Team:UC Chile2/Results/Gibson"> Characterization section</A>, amplification of the parts was done using standard Phusion Polymerase PCR conditions but with a total amount of template of 0.05ng</LI><LI>Bands where cut and purification of DNA from agarose gel was realized using a Promega's Wizard SV Gel and PCR Clean-Up System, final elution volume of 20 ul</LI><LI>Purified DNA was quantified using a Nanodrop 2000</LI><LI>According to concentration of DNA, the amount of pmoles/uL of each part was calculated using the following formulae:</LI><P>pmoles of DNA = weigth in ng * 1000 (conversion factor from nano to pico) / (650 Daltons * base pair length of part)
</P><LI>Volumes of each part were calculated to obtain a final amount of 0.0125 pmoles of the small part and 0.0025 pmoles of the backbone in a final volume of 3 uL (add nuclease-free water to reach 3 uL) for each reaction</LI><LI>For the preparation of Gibson Assembly Master Mixes, different volumes of a 10X T5 exonuclease 1.33X Gibson Assembly Master Mix were mixed with T5 exonuclease-free 1.33X Gibson Assembly Master Mix until appropriate concentrations were reached. If mixing volumes of any of the Master Mixes was lower than 1 uL, then one of the diluted T5 exonuclease 1.33X Master Mixes was used for the mix.</LI><LI>Gibson Assembly was done as described <A href="#Assembly_reaction"> here</A>, and transformation proceeded as described <A href="#E.coli_Transformation"> here</A></LI><LI>Each of the concentration points of the experiment was done in triplicate</LI><LI>Red colonies and white colonies were counted 20 hours post-transformation</LI></UL><H1><SPAN class="mw-headline" id="References">References</SPAN></H1><DIV id="1"><P>[1]Julian J. Eaton-Rye , “Construction of Gene Interruptions and Gene Deletions in the Cyanobacterium Synechocystis sp. Strain PCC 6803 ”, Photosynthesis Research Protocols, Methods in Molecular Biology, vol. 684, DOI 10.1007/978-1-60761-925-3_22 
</P></DIV><DIV id="2"><P>[2]Gibson DG, Young L, Chuang RY, Venter JC, Hutchison CA 3rd, Smith HO. (2009). &quot;Enzymatic assembly of DNA molecules up to several hundred kilobases&quot;. Nature Methods 6 (5): 343–345. doi:10.1038/nmeth.1318
</P></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UC_Chile2/Protocols">http://2012.igem.org/Team:UC_Chile2/Protocols</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Chile2/Protocols" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Chile2/Protocols" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Chile2/Protocols&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Chile2/Protocols&amp;oldid=168576" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>