"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Chile2/Cyanolux/Strategy
From 2012.igem.org
Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012
Team »
Roster
PSB Lab
Photo gallery
Luxilla »
Project
Results
Modelling
Future Prospects
Biolamp
Biobricks
Notebook
Side projects »
Bactomithril
Gibson-small parts
Syne growth curve
Int_C debugging
LuxBrick char.
Methodologies »
Protocols
Arduino
DIY
Software
Human Practices »
Synthetic Sociology
Biosafety
SynBio Workshop
More »
Collaborations
Judging Criteria
Attribution
Contents
1About
1.1Splitting the Lux operon and choosing promoters
1.2Building constructs for Synechocystis PCC 6803
1.2.1pSB1C3_IntK
1.2.2pSB1C3_IntS
1.2.3pSB1A2_IntC (Utah 2010 iGEM Team integration plasmid)
About
We have devised different strategies to achieve bioluminescence controlled under circadian rhythms. Here we describe the strategies used for building the constructs to reach our goals.
Splitting the Lux operon and choosing promoters
We decided that we would separate LuxAB (the luciferase part of the operon) and LuxCDEG (the substrate producing enzymes LuxCDE with LuxG the FMNH2/FMN reducing enzyme) to allow phase-dependent expression of the parts. Using specific promoters of Synechocystis PCC. 6803 we can have fine-tunning of the production of bioluminescence. Recent work on global gene expression in Synechocystis aided on finding adecuate promoters (Kucho K. , et al., (2005) Global Analysis of Circadian Expression in the Cyanobacterium Synechocystis sp. Strain PCC 6803., J. Bacteriol. , 187(6):2190. DOI: 10.1128/JB.187.6.2190-2199; and Layana C, Diambra L (2011) Time-Course Analysis of Cyanobacterium Transcriptome: Detecting Oscillatory Genes. PLoS ONE 6(10): e26291. DOI:10.1371/journal.pone.0026291. Images at right from cited papers.)  We choose various promoters to try our approach. We looked for promoters which would have peak expression nearby dusk hours and that were slightly out of phase to optimize production of bioluminescence according to our mathematical models. (LINK OVER HERE!). We choose the transaldolase promoter (specific name here) to direct the expression of the LuxAB genes and we found a couple of other promoters which filled the requirement from above. Pcaa3 (NAME HERE AND DESCRIPTION) and PsigE (NAME HERE AND DESCRIPTION), one of which already was in the registry (PsigE from Hawaii iGEM 2007 DOUBLECHECKTHIS).
Building constructs for Synechocystis PCC 6803
As transformation in Synechocystis is undergone through double recombination of DNA strands, we designed two constructs that have different recombination locations in the Synechocystis chromosome.
pSB1C3_IntK
The first of our constructs is an integrative plasmid which targets neutral recombination sites (slr0370 and slr03770 DOUBLECHECK this). We selected this locus because it has been extensively used in the literature (CAPAZ EXAGERE?) and it shown to have no deleterious effects on Synechocystis viability. We selected Kanamycin resistance as our transformation marker. [PUT LINK TO CONSTRUCT HERE].
pSB1C3_IntS
We designed another construct that besides serving as a double recombination plasmid it makes Synechocystis susceptible to copper concentrations higher than X uM (Giner-Lamia, J., Lopez-Maury, L., Reyes, J. C., & Florencio, F. J. (2012). The CopRS two-component system is responsible for resistance to copper in the cyanobacterium Synechocystis sp. PCC 6803. Plant physiology, 159(August), 1806-1818.). We have designed this construct to interrupt the CopS gene as a biosafety measure to avoid the possibility of having a leakage of recombinant DNA to the environment.  This plasmid has Spectynomycin resistance as the transformation marker. [PUT LINK TO CONSTRUCT HERE]
pSB1A2_IntC (Utah 2010 iGEM Team integration plasmid)
Alternatively, we designed our LuxCDEG contructs for the Utah 2010 iGEM Team plasmid backbone pSB1A2_IntC. [PUT LINK TO CONSTRUCT HERE].
Retrieved from " http://2012.igem.org/Team:UC_Chile2/Cyanolux/Strategy "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
