<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_UC_Chile_Cyanolux_Results"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:UC_Chile/Cyanolux/Results">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:UC_Chile/Cyanolux/Results&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile/Cyanolux/Results&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:UC_Chile/Cyanolux/Results&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:UC_Chile/Cyanolux/Results" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:UC Chile/Cyanolux/Results</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2012.igem.org</H3><P><TITLE> Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012</TITLE></P><DIV id="banner"><DIV id="menucontainer"><UL class="sf-menu sf-navbar"><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Team/Members">Team <SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Members">Roster</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/PSB_Lab">PSB Lab</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Team/Gallery">Photo gallery</A></LI></UL><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Luxilla<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Project_short">Project</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Results_short">Results</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Modelling">Modelling</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Future">Future Prospects</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biolamp">Biolamp</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Biobricks">Biobricks</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Cyanolux/Notepad">Notebook</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Side projects<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Bactomithril">Bactomithril</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Gibson">Gibson-small parts</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Growthcurve">Syne growth curve </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/Int_C">Int_C debugging</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Results/LuxBrick">LuxBrick char.</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Protocols">Methodologies<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Protocols">Protocols</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Arduino">Arduino</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/DIY">DIY</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Other">Software</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Human Practices<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Community_outreach">Synthetic Sociology</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Biosafety">Biosafety</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Workshop">SynBio Workshop</A></LI><LI><A class="sf-with-ul" href="https://2012.igem.org/Team:UC_Chile/Collaborations">More<SPAN class="sf-sub-indicator">»</SPAN></A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Collaborations">Collaborations</A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Accomplishments">Judging Criteria </A></LI><LI><A href="https://2012.igem.org/Team:UC_Chile/Attribution">Attribution</A></LI></DIV><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Plasmid_Construction"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Plasmid Construction</SPAN></A></LI><LI class="toclevel-2"><A href="#Neutral_recombination_plasmid"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">Neutral recombination plasmid</SPAN></A></LI><LI class="toclevel-3"><A href="#Insertion_of_LuxAB_genes"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Insertion of LuxAB genes</SPAN></A></LI></UL><LI class="toclevel-2"><A href="#Suceptibility_construct"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">Suceptibility construct</SPAN></A></LI><LI class="toclevel-3"><A href="#Insertion_of_Lux_substrate_biosynthesis_genes_under_twilight_promoters"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">Insertion of Lux substrate biosynthesis genes under twilight promoters</SPAN></A></LI><LI class="toclevel-1"><A href="#Synechocystis_PCC_6803"><SPAN class="tocnumber">2</SPAN><SPAN class="toctext">Synechocystis PCC 6803</SPAN></A></LI><LI class="toclevel-2"><A href="#Transforming_Synechocystis_PCC_6803"><SPAN class="tocnumber">2.1</SPAN><SPAN class="toctext">Transforming Synechocystis PCC 6803</SPAN></A></LI><LI class="toclevel-3"><A href="#LuxABxl_under_transaldolase_promoter_transformation"><SPAN class="tocnumber">2.1.1</SPAN><SPAN class="toctext">LuxABxl under transaldolase promoter transformation</SPAN></A></LI><LI class="toclevel-3"><A href="#LuxABvf_under_transaldolase_promoter_transformation"><SPAN class="tocnumber">2.1.2</SPAN><SPAN class="toctext">LuxABvf under transaldolase promoter transformation</SPAN></A></LI><LI class="toclevel-2"><A href="#Corroboration_of_integration"><SPAN class="tocnumber">2.2</SPAN><SPAN class="toctext">Corroboration of integration</SPAN></A></LI><LI class="toclevel-3"><A href="#Integration_of_LuxABxl_under_transaldolase_promoter"><SPAN class="tocnumber">2.2.1</SPAN><SPAN class="toctext">Integration of LuxABxl under transaldolase promoter</SPAN></A></LI><LI class="toclevel-3"><A href="#Integration_of_LuxABvf_under_transaldolase_promoter"><SPAN class="tocnumber">2.2.2</SPAN><SPAN class="toctext">Integration of LuxABvf under transaldolase promoter</SPAN></A></LI><LI class="toclevel-2"><A href="#Testing_constructions_in_Synechocystis"><SPAN class="tocnumber">2.3</SPAN><SPAN class="toctext">Testing constructions in Synechocystis</SPAN></A></LI><LI class="toclevel-2"><A href="#sfGFP_with_degradation_tag_to_characterize_transaldolase_promoter"><SPAN class="tocnumber">2.4</SPAN><SPAN class="toctext">sfGFP with degradation tag to characterize transaldolase promoter</SPAN></A></LI><LI class="toclevel-3"><A href="#High-sensitive_camera"><SPAN class="tocnumber">2.4.1</SPAN><SPAN class="toctext">High-sensitive camera</SPAN></A></LI><LI class="toclevel-1"><A href="#Experimental_Highlights"><SPAN class="tocnumber">3</SPAN><SPAN class="toctext">Experimental Highlights</SPAN></A></LI><LI class="toclevel-1"><A href="#Conclusions"><SPAN class="tocnumber">4</SPAN><SPAN class="toctext">Conclusions</SPAN></A></LI><LI class="toclevel-2"><A href="#Transformation_in_Synechocystis_PCC._6803"><SPAN class="tocnumber">4.1</SPAN><SPAN class="toctext">Transformation in Synechocystis PCC. 6803</SPAN></A></LI><LI class="toclevel-2"><A href="#Verification_of_integrations"><SPAN class="tocnumber">4.2</SPAN><SPAN class="toctext">Verification of integrations</SPAN></A></LI><LI class="toclevel-2"><A href="#Promoters"><SPAN class="tocnumber">4.3</SPAN><SPAN class="toctext">Promoters</SPAN></A></LI><LI class="toclevel-2"><A href="#Luciferases"><SPAN class="tocnumber">4.4</SPAN><SPAN class="toctext">Luciferases</SPAN></A></LI><LI class="toclevel-2"><A href="#Susceptibility_construct"><SPAN class="tocnumber">4.5</SPAN><SPAN class="toctext">Susceptibility construct</SPAN></A></LI></TD></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Plasmid_Construction">Plasmid Construction</SPAN></H1><P>
Following our experimental strategy we built two plasmid backbones on which to insert the Lux constructs.
</P><H2><SPAN class="mw-headline" id="Neutral_recombination_plasmid">Neutral recombination plasmid</SPAN></H2><DIV id="boxright"><P>
1st lane is 1Kb Ladder. 4th lane is C4; Band at 2300: Digestion intermediate, band at 2000: backbone, band at 1700: RS1-KanR-RS2partial, band at 500: RS2partial.
</P></DIV><P>This plasmid backbone was constructed by standard assembly techniques. Recombination sites (<A href="http://partsregistry.org/Part:BBa_K743000" class="external text" rel="nofollow">BBa_K743000</A> and <A href="http://partsregistry.org/Part:BBa_K743001" class="external text" rel="nofollow">Part:BBa_K743001</A>) were Biobricked by Gibson assembly and were consenquently ligated to neighboring parts and  KanRB0015 in parallel assembly. Afterwards, we ligated both composites through a standard assembly reaction. Final construct was validated by digestion (see gel image at right) and corroborated through sequencing.
</P><P>
After obtaining <A href="http://partsregistry.org/Part:BBa_K743006" class="external text" rel="nofollow">BBa_K743006</A>, we proceeded assembling our constructs through Gibson assemblies.
</P><P><I>If you are wondering why the plasmid was constructed by standard assembly and not by gibson assembly from scratch, please refer to </I><A href="/Team:UC_Chile/Results/Gibson" title="Team:UC Chile/Results/Gibson"> Gibson assembly for small parts </A></P><H3><SPAN class="mw-headline" id="Insertion_of_LuxAB_genes">Insertion of LuxAB genes</SPAN></H3><P>
Starting from <A href="http://partsregistry.org/Part:BBa_K743006|" class="external text" rel="nofollow">K743006</A> as a plasmid backbone, we were able to build our LuxAB constructs for the bacterial luciferase using two different versions available at the registry (from <A href="http://partsregistry.org/Part:BBa_K743014" class="external text" rel="nofollow">Photorhabdus luminiscent, BBa_K743014</A> and <A href="http://partsregistry.org/Part:BBa_K743015" class="external text" rel="nofollow">Vibrio fisherii, BBa_K743015</A>) under an endogenous Synechocystis's promoter (transaldolase Reference???). Resulting constructs were verified by digestion (see gel images below) and corroborated by sequencing.
</P><DIV id="boxright"><P>Lane 1 = Ladder 1Kb, Lane 6 = K743014. Band at 4000: RS1-Pta-LuxABvf-Kan-RS2partial, band at 2000: pSB1C3, band at 500: RS2partial. 
</P></DIV><DIV id="boxleft" style="width: 366px;"><P>Lane 1 = Ladder 1Kb, Lane 3 = K743014. Band at 4000: RS1-Pta-LuxABxl-Kan-RS2partial, band at 2000: pSB1C3, band at 500: RS2partial. 
</P></DIV><H2><SPAN class="mw-headline" id="Suceptibility_construct">Suceptibility construct</SPAN></H2><P>pSB1C3_IntS was constructed in two successive Gibson assemblies. The first assembly inserted the CopS region (the entire RS5 &amp; RS6 recombination locus) to a chopped-down pSB1C3 lacking the whole VF2 to VR region. The second assembly inserted the Spectynomycin resistance selection marker and the whole VF to VR region of plasmid pSB4K5-J04450 right in between of the RS5 &amp; RS6 CopS region. This construct yields red colonies if correctly assembled (see image at left). Red colonies were colony PCRed, verificated by digestion (see image down right) and corroborated by sequencing.
</P><DIV id="boxleft"><P>A couple of picked red colonies are shown in this plate. 
</P></DIV><DIV id="boxright"><P>Digestion profile for <A href="http://partsregistry.org/Part:BBa_K743018" class="external text" rel="nofollow">K743018</A>&amp;<A href="http://partsregistry.org/Part:BBa_K743010" class="external text" rel="nofollow">K743010 </A></P></DIV><H3><SPAN class="mw-headline" id="Insertion_of_Lux_substrate_biosynthesis_genes_under_twilight_promoters">Insertion of Lux substrate biosynthesis genes under twilight promoters</SPAN></H3><P>We have amplified all the parts of the substrate production pathway (<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K325902" class="external text" rel="nofollow">LuxCD</A> and <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K325903" class="external text" rel="nofollow">LuxEG</A>) for the final construct under the Pcaa3 promoter (which has also been Biobricked with code [<A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K743002" class="external text" rel="nofollow">K743002</A>]. Also, we are amplifying one remaining part (LuxCD with 5' homology to PsigE) to do Gibson assembly of the LuxCDEG construct under PsigE. Gibson reaction will be done along with the substrate production construct under Pcaa3 in pSB1C3_IntS for both constructs.
</P><H1><SPAN class="mw-headline" id="Synechocystis_PCC_6803">Synechocystis PCC 6803</SPAN></H1><H2><SPAN class="mw-headline" id="Transforming_Synechocystis_PCC_6803">Transforming Synechocystis PCC 6803</SPAN></H2><P>Once constructs were corroborated by sequencing, we proceeded to <A href="/Team:UC_Chile/Protocols#Transformation_of_Synechocystis_PCC._6803" title="Team:UC Chile/Protocols"> transform synechocystis</A>. Prior to transforming, we verified that our Synechocystis cultures were in exponential growth by comparison to our <A href="/Team:UC_Chile/Results/Growthcurve" title="Team:UC Chile/Results/Growthcurve"> growth curve characterization</A>.
</P><P>Transformation in Synechocystis takes about two weeks to reveal transformant colonies, which in turn have to be restreaked again before PCR verification. This makes working in Synechocystis pressing as mistakes <A href="/Team:UC_Chile/Results/Int_C" title="Team:UC Chile/Results/Int C"> can take long to unravel</A>.
</P><DIV style="width: 600px; padding: 10px; margin-left: auto; margin-right: auto;"><P>This image shows various time points during a Synechocystis PCC. 6803 transformations. Be advised that it is only to be used as a reference of what to expect during the first week of transformation.  
</P></DIV><H3><SPAN class="mw-headline" id="LuxABxl_under_transaldolase_promoter_transformation">LuxABxl under transaldolase promoter transformation</SPAN></H3><DIV style="width: 600px; padding: 10px; margin-left: auto; margin-right: auto;"><P>Transformation of Synechocystis PCC 6803 with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K743014" class="external text" rel="nofollow">K743014</A> on the 14th day of transformation. The left plate is the control transformation plate (no DNA) on Kanamycin 25ug/mL and on the right, successfully transformed Synechocystis colonies growing throughout the Millipore membrane on the same antibiotic. 
</P></DIV><H3><SPAN class="mw-headline" id="LuxABvf_under_transaldolase_promoter_transformation">LuxABvf under transaldolase promoter transformation</SPAN></H3><DIV style="width: 600px; padding: 10px; margin-left: auto; margin-right: auto;"><P>Transformation of Synechocystis PCC 6803 with <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_K743015" class="external text" rel="nofollow">K743015</A> on the 14th day of transformation. The left plate is the control transformation plate (no DNA) on Kanamycin 25ug/mL and on the right, successfully transformed Synechocystis colonies growing throughout the Millipore membrane on the same antibiotic. 
</P></DIV><H2><SPAN class="mw-headline" id="Corroboration_of_integration">Corroboration of integration</SPAN></H2><P>To corroborate if the transformant colonies had integrated the actual insert, we did colony PCR for multiple parts contained in the insert.
</P><H3><SPAN class="mw-headline" id="Integration_of_LuxABxl_under_transaldolase_promoter">Integration of LuxABxl under transaldolase promoter</SPAN></H3><P>We did a massive colony PCR to corroborate the presence of the parts which composed the original K743014 integration plasmid, with corresponding negative controls to avoid confusing a genomic amplicon with an insert amplicon. 
</P><P>Regretably we had many negative controls which had amplicons and also lots of amplicons of the insert which we could not obtain... We are not certain about the repercussions of this result so we will try the experiment after the wiki freeze.
</P><H3><SPAN class="mw-headline" id="Integration_of_LuxABvf_under_transaldolase_promoter">Integration of LuxABvf under transaldolase promoter</SPAN></H3><DIV style="width: 400px; padding: 10px; margin-left: auto; margin-right: auto;"><P>This PCR was done using DNA from transformant LuxABvf colonies to amplify parts contained in the insert in lanes 2,4,6,8 and DNA from wild-type Synechocystis PCC 6803, used as a negative control in lanes 3,5,7,9. Ladder 1Kb is placed in lane 1 and 10.
</P></DIV><P style="font-size:150%">We would like to make notice that to our knowledge we are the first iGEM team to do a succesful direct transformation of Synechocystis PCC. 6803 with naked DNA!</P><H2><SPAN class="mw-headline" id="Testing_constructions_in_Synechocystis">Testing constructions in Synechocystis</SPAN></H2><P>To test our constructions, we managed to get access to a Luminometer which also has dispensing capabilities. To test the activity of the transformed Synechocystis with the luciferases we used decanal and dodecanal exogenously, using incremental concentrations of substrate. 
</P><P>
According to these results we are inclined to think that our the transaldolase promoter that we are using is not driving the expression of the luciferases. Furthermore, we have received recent advice to use much larger (up to 1 Kb) of promoter upstream of the +1 transcription site of the transaldolase gene.
</P><H2><SPAN class="mw-headline" id="sfGFP_with_degradation_tag_to_characterize_transaldolase_promoter">sfGFP with degradation tag to characterize transaldolase promoter</SPAN></H2><P>To describe the circadian behaviour of the transaldolase promoter without having the dificulties of equipment (luminometer and substrate), we built a fast-degrading reporter consisting of sfGPF(I746916) with a LVA degradation tag in the C-terminal end of the protein,  <A href="http://partsregistry.org/Part:BBa_K743019" class="external text" rel="nofollow">BBa_K743019</A>. This construct will serve as a real-time reporter of promoter activity. As it is a reporter plasmid with fast-degradation tags, verificating if sfGFP is being expressed circadianly by Synechocystis should be effortless. 
</P><P>You may find more information about the half-life of proteins with the LVA tag  <A href="http://partsregistry.org/wiki/index.php?title=Part:BBa_M0050" class="external text" rel="nofollow">| here</A>. The construct has been verified by digestion and corroborated by sequencing. Check construct digestion <A href="/Team:UC_Chile/Cyanolux/Results#C10" title="Team:UC Chile/Cyanolux/Results"> here</A></P><H3><SPAN class="mw-headline" id="High-sensitive_camera">High-sensitive camera</SPAN></H3><P>
During the Latin America Jamboree, we had a chat with a couple of judges and a student from the Universidad de los Andes, David Olarte, on how to induce Synechocystis with n-decanal.
</P><P>David sent us some work he had done on luminescence assays in cyanobacteria. Following his methods we finally were able to see light emittion, confirming prescence of catalytically active luciferase.
</P><H1><SPAN class="mw-headline" id="Experimental_Highlights">Experimental Highlights</SPAN></H1><P>
- Constructed two functional integration plasmids for <I>Synechocystis</I> PCC. 6803

- Transformed <I>Synechocystis</I> PCC. 6803 with an optimized transformation protocol, available <A href="https://2012.igem.org/Team:UC_Chile/Protocols#Transformation_of_Synechocystis_PCC._6803" class="external text" rel="nofollow">here</A>
- Verified integration of constructs

- Confirmed expression of genes and activity of luciferase through a bioluminescence assay with exogenous substrate.
</P><H1><SPAN class="mw-headline" id="Conclusions">Conclusions</SPAN></H1><H2><SPAN class="mw-headline" id="Transformation_in_Synechocystis_PCC._6803">Transformation in Synechocystis PCC. 6803</SPAN></H2><P>We have successfully transformed Synechocystis PCC. 6803. We have been able to standarize growth conditions and transformation protocols. We believe that we are the first iGEM team to transform  Synechocystis PCC. 6803 directly with naked DNA.
</P><H2><SPAN class="mw-headline" id="Verification_of_integrations">Verification of integrations</SPAN></H2><P>We have been able to verify the integration of the construct into Synechocystis's genome.
</P><H2><SPAN class="mw-headline" id="Promoters">Promoters</SPAN></H2><P>We have been able to demonstrate that the transaldolase is driving the expression of the constructs but we have no certainty as if it is behaving circadianly. With our sfGFP construct with degradation tag we will be able to determinate if that is so.
</P><H2><SPAN class="mw-headline" id="Luciferases">Luciferases</SPAN></H2><P>With our new results with the High-sensitivity CCD camera we have been able to observe bioluminescence from our transformed Synechocystis. The luciferase is catalytically active, however we need to see if we can enhance light output by changing the length of the promoter.
</P><H2><SPAN class="mw-headline" id="Susceptibility_construct"> Susceptibility construct</SPAN></H2><P>We have been unable to obtain a correct assembly of the LuxCDEG (substrate production) into our suceptibility construct. Due to repetitive failures, we will try a new strategy of cloning, by ordering new primers with longer overlapping ends.
</P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2012.igem.org/Team:UC_Chile/Cyanolux/Results">http://2012.igem.org/Team:UC_Chile/Cyanolux/Results</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:UC_Chile/Cyanolux/Results" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:UC_Chile/Cyanolux/Results" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:UC_Chile/Cyanolux/Results&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:UC_Chile/Cyanolux/Results&amp;oldid=282930" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2012.igem.org:Privacy_policy" title="2012.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2012.igem.org:General_disclaimer" title="2012.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></DIV></BODY></HTML>