"
Page
Discussion
View source
History
teams
Log in
 
Team:UC Chile/Cyanolux/Results
From 2012.igem.org
Project: Luxilla - Pontificia Universidad Católica de Chile, iGEM 2012
Team »
Roster
PSB Lab
Photo gallery
Luxilla »
Project
Results
Modelling
Future Prospects
Biolamp
Biobricks
Notebook
Side projects »
Bactomithril
Gibson-small parts
Syne growth curve
Int_C debugging
LuxBrick char.
Methodologies »
Protocols
Arduino
DIY
Software
Human Practices »
Synthetic Sociology
Biosafety
SynBio Workshop
More »
Collaborations
Judging Criteria
Attribution
Contents
1Plasmid Construction
1.1Neutral recombination plasmid
1.1.1Insertion of LuxAB genes
1.2Suceptibility construct
1.2.1Insertion of Lux substrate biosynthesis genes under twilight promoters
2Synechocystis PCC 6803
2.1Transforming Synechocystis PCC 6803
2.1.1LuxABxl under transaldolase promoter transformation
2.1.2LuxABvf under transaldolase promoter transformation
2.2Corroboration of integration
2.2.1Integration of LuxABxl under transaldolase promoter
2.2.2Integration of LuxABvf under transaldolase promoter
2.3Testing constructions in Synechocystis
2.4sfGFP with degradation tag to characterize transaldolase promoter
2.4.1High-sensitive camera
3Experimental Highlights
4Conclusions
4.1Transformation in Synechocystis PCC. 6803
4.2Verification of integrations
4.3Promoters
4.4Luciferases
4.5Susceptibility construct
Plasmid Construction
Following our experimental strategy we built two plasmid backbones on which to insert the Lux constructs.
Neutral recombination plasmid
1st lane is 1Kb Ladder. 4th lane is C4; Band at 2300: Digestion intermediate, band at 2000: backbone, band at 1700: RS1-KanR-RS2partial, band at 500: RS2partial.
This plasmid backbone was constructed by standard assembly techniques. Recombination sites ( BBa_K743000 and Part:BBa_K743001 ) were Biobricked by Gibson assembly and were consenquently ligated to neighboring parts and  KanRB0015 in parallel assembly. Afterwards, we ligated both composites through a standard assembly reaction. Final construct was validated by digestion (see gel image at right) and corroborated through sequencing.
After obtaining BBa_K743006 , we proceeded assembling our constructs through Gibson assemblies.
If you are wondering why the plasmid was constructed by standard assembly and not by gibson assembly from scratch, please refer to Gibson assembly for small parts
Insertion of LuxAB genes
Starting from K743006 as a plasmid backbone, we were able to build our LuxAB constructs for the bacterial luciferase using two different versions available at the registry (from Photorhabdus luminiscent, BBa_K743014 and Vibrio fisherii, BBa_K743015 ) under an endogenous Synechocystis's promoter (transaldolase Reference???). Resulting constructs were verified by digestion (see gel images below) and corroborated by sequencing.
Lane 1 = Ladder 1Kb, Lane 6 = K743014. Band at 4000: RS1-Pta-LuxABvf-Kan-RS2partial, band at 2000: pSB1C3, band at 500: RS2partial.
Lane 1 = Ladder 1Kb, Lane 3 = K743014. Band at 4000: RS1-Pta-LuxABxl-Kan-RS2partial, band at 2000: pSB1C3, band at 500: RS2partial.
Suceptibility construct
pSB1C3_IntS was constructed in two successive Gibson assemblies. The first assembly inserted the CopS region (the entire RS5 & RS6 recombination locus) to a chopped-down pSB1C3 lacking the whole VF2 to VR region. The second assembly inserted the Spectynomycin resistance selection marker and the whole VF to VR region of plasmid pSB4K5-J04450 right in between of the RS5 & RS6 CopS region. This construct yields red colonies if correctly assembled (see image at left). Red colonies were colony PCRed, verificated by digestion (see image down right) and corroborated by sequencing.
A couple of picked red colonies are shown in this plate.
Digestion profile for K743018 & K743010
Insertion of Lux substrate biosynthesis genes under twilight promoters
We have amplified all the parts of the substrate production pathway ( LuxCD and LuxEG ) for the final construct under the Pcaa3 promoter (which has also been Biobricked with code [ K743002 ]. Also, we are amplifying one remaining part (LuxCD with 5' homology to PsigE) to do Gibson assembly of the LuxCDEG construct under PsigE. Gibson reaction will be done along with the substrate production construct under Pcaa3 in pSB1C3_IntS for both constructs.
Synechocystis PCC 6803
Transforming Synechocystis PCC 6803
Once constructs were corroborated by sequencing, we proceeded to transform synechocystis . Prior to transforming, we verified that our Synechocystis cultures were in exponential growth by comparison to our growth curve characterization .
Transformation in Synechocystis takes about two weeks to reveal transformant colonies, which in turn have to be restreaked again before PCR verification. This makes working in Synechocystis pressing as mistakes can take long to unravel .
This image shows various time points during a Synechocystis PCC. 6803 transformations. Be advised that it is only to be used as a reference of what to expect during the first week of transformation.
LuxABxl under transaldolase promoter transformation
Transformation of Synechocystis PCC 6803 with K743014 on the 14th day of transformation. The left plate is the control transformation plate (no DNA) on Kanamycin 25ug/mL and on the right, successfully transformed Synechocystis colonies growing throughout the Millipore membrane on the same antibiotic.
LuxABvf under transaldolase promoter transformation
Transformation of Synechocystis PCC 6803 with K743015 on the 14th day of transformation. The left plate is the control transformation plate (no DNA) on Kanamycin 25ug/mL and on the right, successfully transformed Synechocystis colonies growing throughout the Millipore membrane on the same antibiotic.
Corroboration of integration
To corroborate if the transformant colonies had integrated the actual insert, we did colony PCR for multiple parts contained in the insert.
Integration of LuxABxl under transaldolase promoter
We did a massive colony PCR to corroborate the presence of the parts which composed the original K743014 integration plasmid, with corresponding negative controls to avoid confusing a genomic amplicon with an insert amplicon.
Regretably we had many negative controls which had amplicons and also lots of amplicons of the insert which we could not obtain... We are not certain about the repercussions of this result so we will try the experiment after the wiki freeze.
Integration of LuxABvf under transaldolase promoter
This PCR was done using DNA from transformant LuxABvf colonies to amplify parts contained in the insert in lanes 2,4,6,8 and DNA from wild-type Synechocystis PCC 6803, used as a negative control in lanes 3,5,7,9. Ladder 1Kb is placed in lane 1 and 10.
We would like to make notice that to our knowledge we are the first iGEM team to do a succesful direct transformation of Synechocystis PCC. 6803 with naked DNA!
Testing constructions in Synechocystis
To test our constructions, we managed to get access to a Luminometer which also has dispensing capabilities. To test the activity of the transformed Synechocystis with the luciferases we used decanal and dodecanal exogenously, using incremental concentrations of substrate.
According to these results we are inclined to think that our the transaldolase promoter that we are using is not driving the expression of the luciferases. Furthermore, we have received recent advice to use much larger (up to 1 Kb) of promoter upstream of the +1 transcription site of the transaldolase gene.
sfGFP with degradation tag to characterize transaldolase promoter
To describe the circadian behaviour of the transaldolase promoter without having the dificulties of equipment (luminometer and substrate), we built a fast-degrading reporter consisting of sfGPF(I746916) with a LVA degradation tag in the C-terminal end of the protein, BBa_K743019 . This construct will serve as a real-time reporter of promoter activity. As it is a reporter plasmid with fast-degradation tags, verificating if sfGFP is being expressed circadianly by Synechocystis should be effortless.
You may find more information about the half-life of proteins with the LVA tag | here . The construct has been verified by digestion and corroborated by sequencing. Check construct digestion here
High-sensitive camera
During the Latin America Jamboree, we had a chat with a couple of judges and a student from the Universidad de los Andes, David Olarte, on how to induce Synechocystis with n-decanal.
David sent us some work he had done on luminescence assays in cyanobacteria. Following his methods we finally were able to see light emittion, confirming prescence of catalytically active luciferase.
Experimental Highlights
- Constructed two functional integration plasmids for Synechocystis PCC. 6803
- Transformed Synechocystis PCC. 6803 with an optimized transformation protocol, available here
- Verified integration of constructs
- Confirmed expression of genes and activity of luciferase through a bioluminescence assay with exogenous substrate.
Conclusions
Transformation in Synechocystis PCC. 6803
We have successfully transformed Synechocystis PCC. 6803. We have been able to standarize growth conditions and transformation protocols. We believe that we are the first iGEM team to transform  Synechocystis PCC. 6803 directly with naked DNA.
Verification of integrations
We have been able to verify the integration of the construct into Synechocystis's genome.
Promoters
We have been able to demonstrate that the transaldolase is driving the expression of the constructs but we have no certainty as if it is behaving circadianly. With our sfGFP construct with degradation tag we will be able to determinate if that is so.
Luciferases
With our new results with the High-sensitivity CCD camera we have been able to observe bioluminescence from our transformed Synechocystis. The luciferase is catalytically active, however we need to see if we can enhance light output by changing the length of the promoter.
Susceptibility construct
We have been unable to obtain a correct assembly of the LuxCDEG (substrate production) into our suceptibility construct. Due to repetitive failures, we will try a new strategy of cloning, by ordering new primers with longer overlapping ends.
Retrieved from " http://2012.igem.org/Team:UC_Chile/Cyanolux/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
