"
Page
Discussion
View source
History
teams
Log in
 
Team:HokkaidoU Japan/Projects/Length
From 2014.igem.org
Projects
Length Variation
Top
RNA in Love
Overview
Background
Achievements
H-stem System
Overview
Stabilization
Modelling
Construct
How To Use
Anti-sense B0034
Overview
Method
Results
Conclusion
Length Variation
Overview
Method
Results
Conclusion
Extra Materials
Parts
Notebook
Safety
Outreach
Univ. Festival
Education
Survey
High School
Education
Survey
Discussion
Background
Evaluation
About Us
About Us
Acknowledgements
Team
Overview
Using anti-sense RNA (asRNA) is one of the methods to repress gene expression. It is known that the length of anti-sense sequence is related to its repression efficiency (N. Nakashima et al., 2006 [1] ), but the details of the relation are still unclear. In this project, we made different lengths of anti-sense sequence (Fig. 1).
Fig. 1 Each anti-sense represses mRNA.
Our experiments will be a clue for other iGEMers who want to design their own anti-sense sequence.
Introduction
In repressing gene by anti-sense RNA, it is important to determine the length of anti-sense. However, it is difficult to do it. Theoretically, if the length is too long, it doesn’t repress target RNA effectively. The reason is because the RNA polymerase takes a lot of time to synthesize them, and the diffusion rate of them also gets low. However, too short asRNA also has some problems. The short anti-sense cannot bind to the specific part of mRNA because it has too short complementary sequences of target RNA. In the industrial and academic fields, people hope to use anti-sense that has suitable repression efficiency. For example, you can create knock down recombinant organisms easily by using strong anti-sense, and in iGEM, you can make bio-devices which have a complicated gene network and require fine-tuned gene expression. Gene expression is not only ON or OFF. As stated above, each case needs each repression efficiency. Researchers currently tried to change anti-sense repression efficiency by changing anti-sense’s binding sequence. However, it is known that this method is difficult.
Fig. 2 Determining a repression region is difficult.
In this project, we made many kinds of anti-sense which repress mRFP, but these anti-sense constructs themselves are not useful for you. However, in our method, you can create the anti-sense that you desire. We expect this project will help you decide anti-sense sequences. We’d like to tell scientists and iGEMers how easy and accurate to repress the gene expression by anti-sense RNA.
Methods
We made anti-sense constructs that repress mRFP. The target construct are composed of Ptet ( BBa_R0040 ), B0034 ( BBa_B0034 ), mRFP ( BBa_E1010 ) and double terminator ( BBa_B0015 ).   Insert fragments were synthesized based on BioBrick by PCR.  As forward primer, ”XhoI-Ptet (-10)”was used for making all fragments. The primer binds to -10 region of Ptet, and its end has XhoI restriction enzyme site. To change the downstream seqeunce, each reverse primer is designed differently (as90 NcoI, as120 NcoI) (Fig. 3). These primers bind to each specific part of mRFP, and their ends have NcoI restriction enzyme site. By that way, we can get various length of insert fragments, as90 and as120. As90 is the anti-sense that covers 90 bp of mRNA, and as120 is the anti-sense that covers 120 bp of mRNA (complement RBS and a part of mRFP sequence.) Of course, the edges of insert fragments have restriction enzymes XhoI, NcoI sites.
Fig. 3 Synthesizing insert fragments by PCR
Fig. 4 Ligate the insert fragment with H-stem vector
After we finished synthesizing insert fragments, we inserted them into our H-stem vector (anti-sense expression vector BBa_K1524100 ) by XhoI and NcoI.    Then, we measured their repression efficiencies. In the same way, we made as30, as60 on H-stem vector in other experiment (as30 ( Anti-sense B0034 ) and as60 ( H-stem system )). We performed repression experiment by using their four anti-sense constructs.
How to assay
We performed RT-PCR to confirm the transcription of anti-sense RNA (asRNA) constructs.
Cultivated the colony in 4 mL LB medium for 16 hours.
Centrifuged the 4 mL of culture at 10,000 rpm / for 2 min / at 25°C
Removed the supernatant and add M9ZB medium then voltex the pelet.
Performed RT-PCR
Measured absorbance of 260 nm about cDNA.
Results
Though we measured absorbance of 260 nm about cDNA, we could not get any cDNA. After RNA extraction, we confirmed absorbance of 260 nm (this is the absorbance of nucleic acid). However, after RT-PCR of that products, we could not confirm the existence of nucleic acid.
It is presumable that there are some problems in RNA extraction. First, we might have lost RNA during the experiment operation. RNA is degraded easily than DNA because generally RNase can be easily contaminated.
Secondly, the incomplete deactivation of DNase can be considered to be the reason of our failure. If DNase was not deactivated sufficiently, it will result in the degradation of cDNA produced in RT-PCR.
Though we could not make it by wiki freeze, we are going to retry this experiment.
Developmental experiment
We theoretically estimated that the repression efficiency of asRNA is related to its length. To find the optimum length, we must design many lengths of asRNA, and measure their efficiency. However, that takes a lot of time and labor. So, as a future work, we propose an efficient method to synthesize various length of anti-sense.
Methods
Preparation for randomizing
Fig. 5 PrePCR for randomizing
