<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_Project_Overview_Protocols skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/Project/Overview/Protocols</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="mainNav"><UL><LI class="first"><A href="https://2015.igem.org/Team:Glasgow">Project</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/UVA">UVA</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Repressors">Repressors</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Bistable">Bistable Switch</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Bioluminesence">Bioluminescence</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Description">RBS Library</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Terminator">Terminator</A></LI></UL><LI><A href="https://2015.igem.org/Team:Glasgow/Team">About Us</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Basic_Part">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Part_Collection">Collection</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Design">Application</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Design">Product Design</A></LI></NAV></P><P style="padding:0; margin:0; margin-top:0.8vh;">Policy &amp;</P><P style="padding:0; margin:0;"> Practice</P><UL><LI><A href="https://2015.igem.org/Team:Glasgow/AzureA">Azure A Staining</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Safety">Lab Safety</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Practices/Bedtimestories">Bedtime Stories</A></LI></UL><LI><A href="https://2015.igem.org/Team:Glasgow/Interlab">Measurement</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI><DIV id="bodybackground"><DIV class="container3"><DIV class="sign"><DIV class="neon-blue" id="title">Glas<SPAN id="fade">glow</SPAN></DIV><DIV class="neon-blue"><SPAN class="neon-purple"><STRONG>Protocols</STRONG></SPAN></DIV></DIV></DIV></DIV><P class="mainText" style="margin-top:3vh;"><A href="https://2015.igem.org/Team:Glasgow">Home</A> &gt; <A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Protocols">Protocols</A></P><DIV class="containerRight"><H1 style="margin-top:13vh;">E. coli strains</H1><P class="mainText">TOP10:
•	F- mcrA Δ(mrr-hsdRMS-mcrBC) φ80lacZΔM15 ΔlacX74 nupG recA1 araD139 Δ(ara-leu)7697 galE15 galK16 rpsL(StrR) endA1 λ-
•	Used for cloning of unmethylated DNA (such as from the distribution plate, or DNA synthesised by IDT)

DH5α:
•	F- endA1 glnV44 thi-1 recA1 relA1 gyrA96 deoR nupG Φ80dlacZΔM15 Δ(lacZYA-argF)U169, hsdR17(rK- mK+), λ–
•	Used for assembly of plasmids, and to methylate DNA from TOP10 in order to transform into DS941

DS941:
•	an MG1655 derivative
•	AB1157, recF, lacZM15, lacIq
•	Used as chassis

DS941.Z1:
•	an MG1655 derivative
•	AB1157, recF, lacZM15, lacIq
•	extra copy of TetR and LacI
•	Used for measurements when extra Lac repressor was required

ER2925:
•	ara-14 leuB6 fhuA31 lacY1 tsx78 glnV44 galK2 galT22 mcrA dcm-6 hisG4 rfbD1 R(zgb210::Tn10)TetS endA1 rpsL136 dam13::Tn9 xylA-5 mtl-1 thi-1 mcrB1 hsdR2
•	Used when dam methylation was not required 
</P><H1>Preparation of CaCl<SUB>2</SUB> Competent Cells</H1><P class="mainText">1.	Dilute 400μl of overnight liquid culture into 20ml of broth with any necessary antibiotics to select for any plasmids already transformed into the cells
2.	Incubate at  37⁰C, shaking at 225rpm for 90 minutes
3.	Spin down for 2 minutes at 7000rpm at 4⁰C
4.	Discard supernatant, resuspend pellet in 10ml of 50 mM CaCl2,  keep on ice
5.	Repeat centrifugation for 2 minutes at 7000rpm at 4⁰C
6.	Discard supernatant and the resuspend pellet in 1ml of 50 mM CaCl<SUB>2</SUB>, keep on ice
7.	CaCl2 competent cells can be kept on ice in the fridge overnight</P><H1>Transformation of CaCl<SUB>2</SUB> Competent Cells</H1><P class="mainText">1.	1μl of plasmid DNA was added to 100μl of competent cells 
2.	Samples were incubated on ice for 20 minutes
3.	Heat shock was cried out at 37⁰C for 5 minutes
4.	Cells were kept on ice for 2 minutes
5.	200μl of broth was added and the cells incubated at 37⁰C for expression step (time varies dependent on antibiotic resistance gene in the plasmid that has just been transformed into the cells):
•	Chloramphenicol resistance = 90 minutes
•	Kanamycin resistance = 60 minutes
•	Ampicillin resistance = 30 minutes
6.	100-200μl of transformed cells was spread on dried L-agar plates with required antibiotic(s) to select for plasmid(s)
7.	Plates were incubated at 37°C overnight</P><H1>Preparation of Electrocompetent Cells</H1><P class="mainText">1.	Inoculate 400ml L-broth with 4ml culture 
2.	Incubate at  37⁰C, shaking at 250rpm until mid-log phase (OD600=0.5-0.7)  
3.	Split culture into 2 x 200ml samples
4.	Chill on ice for 20mins
•	Spin 4000 x g for 15minutes at 40C
5.	Re-suspend each in 200ml ice cold 10% glycerol
•	Spin 4000 x g for 15minutes at 40C
6.	Re-suspend each in 100ml ice cold 10% glycerol
•	Spin 4000 x g for 15minutes at 40C
7.	Re-suspend each in 10ml ice cold 10% glycerol
•	Spin 4000 x g for 15minutes at 40C
8.	Re-suspend each in 500l ice cold 10% glycerol
9.	Aliquot 60l into small eppendorf tubes and store at -700C</P><H1>Transformation of Electrocompetent Cells</H1><P class="mainText">1.	Add 1μl of plasmid DNA to 30μl of competent cells 
2.	Leave on ice for 20 minutes
3.	Transfer to pre-cooled electroporation cuvette
4.	An electrical pulse was delivered by a Biorad Micropulser and 1ml L-broth immediately added
5.	Culture was then transferred to a 2ml nunc tube and incubated at 37⁰C for expression step (time varies dependent on antibiotic resistance gene in the plasmid that has just been transformed into the cells):
•	Chloramphenicol resistance = 90 minutes
•	Kanamycin resistance = 60 minutes
•	Ampicillin resistance = 30 minutes
6.	Spread 100-200μl of transformed cells on dried L-agar plates with required antibiotic(s) to select for plasmid(s)
7.	Incubate plates at 37°C overnight</P><H1>Plasmid DNA preparation using Qiagen Miniprep kit</H1><P class="mainText">1.	Spin down 3-5ml of overnight liquid culture
2.	Add 250µl of Buffer P1 and resuspend the cell pellet thoroughly by pipetting up and down - it is important to have no lumps of cells at this stage
3.	Add 250µl of Buffer P2 , mix by inverting the tube about 6 times
4.	Within less than 5 minutes, add 350µl of Buffer N3 , mix by inverting the tube about 6 times
5.	Spin for 10 minutes at 13,000rpm
6.	Transfer the supernatant from the eppendorf into a QIAprep column in a 2ml collection tube
7.	Spin the column and collection tube for 1 minute at 13,500rpm and discard the flow-through
8.	Wash the column by adding 500μl of Buffer PB to the column, and spin for 1 minute at 13,000rpm, discard the flowthrough
9.	Wash the column by adding 750μl of Buffer PE to the column, and spin for 1 minute at 13,000rpm, discard the flowthrough, and spin again for 1 minute at 13,500rpm, to get rid of the last traces of PE wash buffer
10.	Transfer column to eppendorf, and discard the collection tube
11.	Add 50μl of Buffer PE to the column, and leave to stand for 1 minute
12.	Spin for 1 minute at 13,000rpm, keep the flowthrough in the eppendorf and discard the column</P><H1>Restriction Digests</H1><P class="mainText">1.	A 20μl reaction typically contained:
•	2μl of buffer
•	4μl* of miniprep (or 8μl G-Block)
•	Make up to 20μl with ddH20
•	0.5-1.0μl of each restriction enzyme
2.	Vortex
3.	Incubate at 37⁰C for at least 60 minutes
4.	*adjust volume dependant on concentration of miniprep
5.	Heat inactivate restriction enzymes</P><H1>Gel Electrophoresis</H1><P class="mainText">1.	Make 1% agarose gel by dissolving 1g agarose in TAE buffer, melting in microwave, then cooling to 55⁰C before pouring gel and leaving to set
2.	Load the gel with DNA with loading dye (30% glycerol, 1% bromophenol blue, 0.5%  sodium dodecyl sulfate, diluted in TE buffer)
3.	Run gel at 1.5 Amp, 100V for approximately 1 hour
4.	Stain gel for detection of DNA bands:
•	Ethidium bromide – stain in (concentration) for 40 minutes, destain for 40 minutes, image under UV light
•	For gel extraction: Azure A – stain in 0.04%/20% ethanol for 15 minutes, destain for 15 minutes, image under visible light (more detail on our <A href="https://2015.igem.org/Team:Glasgow/AzureA">Azure A staining</A> page)</P><H1>Gel Extraction usng a Qiagen Gel Extraction Kit</H1><P class="mainText">1.	Cut out DNA band from gel, and weigh in eppendorf
2.	Add 3 volumes of Buffer QG to 1 volume of gel (100μg = ~100μl)
3.	Incubate at 50⁰C for 10 minutes (until agarose has dissolved), mix by vortex every 2-3 minutes
4.	Add 1 volume of isopropanol to sample and mix
5.	Transfer the sample from the eppendorf into a QIAprep spin column in a 2ml collection tube, centrifuge 1 minute at 13,000rpm and discard flowthrough
6.	Add 500μl Buffer QG to column and centrifuge for 1 minute at 13,000rpm
7.	Add 750μl Buffer PE to column, let it stand for 2-5 min and centrifuge for 1 minute at 13,000rpm, discard flowthrough and centrifuge again for 1 minute at 13,000rpm
8.	Transfer column to eppendorf, and discard the collection tube
9.	Add 30μl Buffer EB to column, leave to stand for 1 minute and centrifuge for 1 minute at 13,000rpm, keep the flowthrough in the eppendorf and discard the column</P><H1>Ligation</H1><P class="mainText">1.	For 10μl reaction:
•	3μl* of vector
•	5μl* of insert (or 1μl of 1/100 dilution of annealed oligos)
•	1μl of 10x ligase buffer
•	Make up to 10μl with ddH20 
•	0.5μl of T4 ligase
2.	Vortex, and leave overnight at room temperature, then heat inactivate at 65°C
3.	*adjust volume dependant on concentration of gel purified DNA</P><H1>Annealing Oligos</H1><P class="mainText">1.	Dissolve dried oligo in TE buffer to make 100μM solution and leave for 10 minutes
2.	For 100μl reaction:
•	10μl of top strand 100μM oligo
•	10μl of bottom strand 100μM oligo
•	80μl of TE buffer
3.	Heat in 86°C waterbath in small polystyrene float for 5 minutes
4.	Scoop out eppendorfs in float and approximately 100ml 86°C water with beaker
5.	Allow to cool down slowly to  approximately 30C
6.	Dilute annealed oligos 1/100 </P><H1>PCR</H1><P class="mainText">1.	50μl reaction volume:
•	10μl of 5x HF Buffer
•	5μl of 2mM dNTPs
•	1μl of 50mM MgCl2
•	1.5μl of DMSO
•	5μl of 5μM forward primer
•	5μl of 5μM reverse primer
•	1μl of 1/100 dilution of template DNA
•	Make up to 50μl (22μl) ddH2O
•	0.5μl of Phusion polymerase
2.	Programme Settings:
a)	98⁰C – 1 minute
b)	98⁰C – 20 seconds
c)	~50-60⁰C – 30 seconds 
•	Annealing temperature depends on primers, should be ~5°C lower than Tm of primers
b)	72⁰C – 30 seconds (Back to step b) for 30 cycles)
c)	72⁰C – 10 minutes
d)	4⁰C – Hold</P><H1>Purification of Oligos</H1><P class="mainText">1.	Mix 30 l oligo with 30 l formamide loading buffer
2.	Heat to 80C for 5 min
3.	Run samples on acrylamide gel (pre-run for 30 min at 450 V) for 90 min at 450 V
4.	Following electrophoresis, cut oligo DNA out of the gel with minimal acrylimide
5.	Crush gel chip in eppendorf with pipette tip and add 500 l TE buffer
6.	Shake overnight at 37C, 1150 rpm
7.	Cut off tip of a 1 ml pipette tip to allow pipetting of TE-gel mix and pipette into a 0.22 m costar column
8.	Centrifuge at 3,000 rpm for 10 min, discard column
9.	Dry samples at 40C in vacuum centrifuge to a volume of 200 l (~45 min drying time)
10.	Add 1/9 x V of 3M Ammonium acetate and 2.2 x V of 100% EtOH and vortex to mix
11.	Place in freezer for 40 min – 1 hr
12.	Centrifuge at 13,000 rpm, 4C for 45-60 min
13.	Remove supernatant and add 1 ml of 80% EtOH   
14.	Centrifuge at 13,000 rpm, 4C for 10-45min
15.	Remove supernatant and dry pellet with lid open on bench for 10-15 min
16.	Resuspend pellet in 30 l TE buffer </P><FOOTER><DIV class="jumbotron"><DIV class="container"><DIV class="col-md-4"><H4><STRONG>Location</STRONG></H4><DIV class="footerinfo">
                    Bower Building, Wilkins Teaching Laboratory
                    University of Glasgow  
                    University Avenue 
                    G12 8QQ</DIV></DIV><DIV class="col-md-4"><H4><STRONG>Follow Us On</STRONG></H4></DIV></DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></DIV></BODY></HTML>