Team:Glasgow/Project/Overview/Repressors
Project
Overview
UVA
Repressors
Bistable Switch
Bioluminescence
RBS Library
Terminator
About Us
Team
Attributions
Sponsors
Notebook
Protocols
Parts
Basic Parts
Composite
Collection
Application
Product Design
Policy &
Practice
Azure A Staining
Lab Safety
Bedtime Stories
Measurement
Interlab Study
Collaborations
Home > Project > Repressors
Summary
Introduction
Methods
Results
Conclusion
References
Top
Summary
Aim: To characterise PhlF and SrpR repressors and their respective repressible promoters for submission to the iGEM registry. Results Overview: • K1725000 (PhlF repressible promoter) and K1725020 (SrpR repressible promoter) successfully drive expression of GFP in E. coli. • K1725042 (phlF expression driven by a lac regulated promoter) represses K1725001 (GFP expression driven by PhlF repressible promoter) GFP expression about 83-fold. • K1725042 (phlF expression driven by a lac regulated promoter) is orthongonal. Basic Parts submitted: • BBa_K1725000 – PhlF repressible promoter (PphlF) • BBa_K1725020 – SrpR repressible promoter (PsrpR) • BBa_K1725040 – phlF encoding PhlF repressor • BBa_K1725060 – srpR encoding SrpR repressor • BBa_K1725080 – Promoter (lacI regulated, lambda pL hybrid - pL-lac) with extra NheI site
Introduction
Our genetic circuit needed an inverter, as our UVA sensor turns on transcription, but our circuit needed to turn off transcription when UVA was present. There are suitable several repressor protein/repressible promoter pairs in the iGEM registry such as TetR or LacI, however, they usually come from experimental studies based on existing systems. Recently, Stanton et al. (2014) have identified sixteen prokaryotic TetR-like repressors by genomic mining and designed synthetic repressible promoters (Figure 1A), as with more independently functioning parts, it will be possible to build increasingly complex genetic circuits . By characterising and submitting two new repressors to the registry, based on Stanton et al. (2014) designs, we aimed to demonstrate that this approach could yield efficient parts, thus widening the access to this synthetic technique, which has the potential  to create a much larger range of functions for genetic circuits. To understand how they were able to design synthetic repressible promoters, it is important to understand how promoters and repressors work.Transcription is the process where RNA polymerase binds to DNA to make mRNA; a promoter tells RNA polymerase where to bind to the DNA, so a promoter is found upstream of a gene. Promoters have -10 and -35 sites that the sigma transcription facter binds to and attracts RNA polymerase, as shown in Figure 1B, transcription starts at +1. If one or both of these sites are bound by another protein, sigma factor cannot recognise the promoter, and transcription does not take place. Transcriptional repressors are proteins that bind to DNA at a specific sequence called the operator sequence. Stanton et al. (2014) overlapped the operator sequence for each repressor over the -10 and/or -35 sites of BBa_J23119, a strong, constitutive Anderson family promoter, meaning when the repressor binds to its operator sequence sigma factor cannot recognise the promoter and transcription cannot start, as shown in Figure 1B.
Figure 1 Synthetic Repressible Promoter Design. A – Synthetic repressible promoters with PhlF and SrpR operator sequences, with -35 and -10 sites shown relative to BBa_J23119, and operator sequence in red capital letters. BioBricks of these promoters include sequence from 5’ end to 3’ end of operator sequence. Image reproduced from (Stanton et al., 2014) B – (i) sturcture of a typical repressible promtoer; (ii) sigma factor binding to -35 and -10 sites to attract RNA polymerase; (iii) repressor protein binding to operator sequence preventing sigma factor binding. Altered from (Alberts et al, 2008)
For a repressor to be useful in a genetic circuit, it must be specific so as not to interfere with another part of the circuit or have unwanted interactions within the cell. Repressors that do this are called orthogonal. Repressor A binds to promoter A; repressor B binds to promoter B; but repressor A cannot bind to promoter B, and vice versa. The sixteen prokaryotic TetR-like repressors Stanton et al., (2014) identified are orthogonal, as shown in Figure 2. In particular, TetR, PhlF, and SrpR do not show significant repression of the other’s repressible promoters.
Figure 2 Orthogonal Repressors. Most orthogonal starting top left corner, decreasing towards bottom right corner. TetR, SrpR, and PhlF indicated by arrows. Image reproduced from (Stanton et al., 2014)
It was decided to make BioBricks of two of the sixteen repressors and characterise them for the iGEM registry. The first repressor we decided to submit as a BioBrick was the PhlF repressor from Pseudomonas protegens Pf-5. In P. protegens PhlF is involved in regulation of the phlACBD operon which synthesises an antifungal metabolite 2,4-diacetylphloroglucinol (PHL). (Sheehan et al., 2000, Abbas et al., 2002) The second repressor was the SrpR repressor from Pseudomonas putida S12. In P. putida SrpR is a regulator of the srpABC operon which is involved in organic solvent tolerance. (Wery et al., 2001, Sun et al., 2011) The aim was to submit and characterise both phlF and srpR and their respective repressible promoters.
Methods
E. coli strains used: TOP10, DH5α, and DS941.  Plasmids in Table 1 constructed by BioBrick Standard Assembly , and checked by restriction digest before confirming by sequencing.
Table 1 Composite Parts Assembled. K1725041, K1725061, and R0011.B0032 (for assembly into K1725083) construct synthesised by IDT. K1725062 sequencing showed a deletion in K1725080.
As part of our fluorescence measurements, approximate molecules of GFP per cell were calculated (figure 3). Equations 1-3 convert A600 measurement to number of cells (in the well of the 96-well plate when fluorescence measurements were taken). After the fluorescence measurements, serial dilutions (10-4, 10-5, 10-6, and 10-7) from three of the samples were spotted on an agar plate and grown overnight. It was assumed that one cell grew into one colony. Spots with individual colonies colonies were counted. and used in the calculations. Equations 4-6 convert the fluorescence measurement, in arbitrary units, to approximate GFP molecules. The Typhoon FLA 9000 was calibrated with iLOV fluorescent protein, and GFP is approximately 11.5 times brighter than iLOV (Buckley et al. 2015).
Figure 3 Calculations for approximate molecules of GFP per cell. For these measurements, the target A600 was 0.5. Avagadro's constant (6.02x1023) is the number of molecules per mole.
Protocols for CaCl2 competent cells, transformation, miniprep, restriction digest, gel electrophoresis, ethidium bromide staining, Azure A staining, gel extraction, oligo annealing, and ligation available on our Protocols page. More detail on fluorescence measurements, the spectrophotometer and Typhoon FLA 900 calibrations and settings on our Interlab Study page.
Results
GFP fluorescence driven by PphlF, PsrpR, and PtetR each with a weak and strong ribosome binding site (RBS) (B0032 and B0034, respectively) with plasmid backbone pSB3K3 was measured to compare the relative strengths of promoters K1725000 (PphlF) and K1725020 (PsrpR) to a promoter already well documented in the registry, R0040 (PtetR). Figure 4 shows the fluorescence scan image and a graph of approximate molecules of GFP per cell. These results indicated that PphlF is a significantly stronger promoter than PtetR or PsrpR. It also confirms that B0034 is a stronger ribosome binding site that B0032. Due to this, promoters driving expression of GFP with B0034 ribosome binding site were used for characterisation of the repressors.
Figure 4 Characterising Promoters. All constructs with pSB3K3 plasmid backbone, in DH5α cells. Replicates of constructs and controls from three colonies, under the same conditions. Mean and standard deviation of replicates were calculated to give value and error bars. Values were normalised against a cells only control (not shown) so cells only = 0, therefore any negative values where the error bars overlapping 0, are essentially 0 (as a negative number of molecules is not possible).
To test if our repressors were orthogonal, a smaller version of the cross-talk table Stanton et al., (2014) created was emulated, and our expected results are shown in Figure 5. For example, the PhlF repressor (K1725040) is expected to repress GFP expression from PphlF so these cells should not fluoresce, however, it is not expected to repress GFP expression from PsrpR so these cells should fluoresce green. In order to transform two different plasmids into the same cell, they must have different origins of replication, and different antibiotic resistances so both plasmids can be selected for. It was decided to keep the repressor gene expressed from the lac regulated promoter (pL-lac) in the high copy number pSB1C3 backbone, and move the repressible promoters driving GFP expression into the lower copy number pSB3K3 backbone (hence why the relative strengths of PphlF, PsrpR, and PtetR were measured in pSB3K3).
Figure 5 Expected GFP fluorescence of cells containing plasmids of repressor BioBricks with pSB1C3 backbone, and promoter BioBricks with pSB3K3 backbone. Cells with bold border shown in detail below: PhlF represses PphlF; PhlF does not repress PsrpR
As shown in Figure 6, TetR represses GFP expression from PtetR, but not from PphlF or PsrpR. Similarly, PhlF represses GFP expression from PphlF, but not from PtetR or PsrpR, as expected. SrpR, however, was not observed to repress any of the three promoter constructs. Sequencing of SrpR driven by pL-lac showed a deletion in the promoter pL-Lac (K1725080), suggesting SrpR was not being expressed. Furthermore, SrpR driven by PphlF (K1725063) has been observed to repress expression driven by PsrpR (more detail on our Bistable Switch page). This supports the hypothesis that SrpR driven by pL-lac should be capable of repressing GFP expression from PsrpR. Unfortunately, due to time constraints, it was not possible to reconstruct K1725062 (SrpR driven by pL-lac) and repeat this experiment.
Figure 6 Characterising Repressors. Repressor driven by pL-lac in pSB1C3 backbone; promoter driving GFP constructs in pSB3K3 backbone; in DS941 cells. The DS941 genotype can be found on our Protocols page. Cells were grown overnight in 100μM IPTG, to induce expression of the repressor proteins. Three replicates of the sample were diluted and tested under the same conditions for each sample. Mean and standard deviation of replicates were calculated to give value and error bars.
In addition to showing that PhlF and TetR were capable of repressing PphlF and PtetR, respectively, quantification of repression of GFP expression was calculated. Figure 7 shows that TetR represses PtetR GFP expression by about 33-fold, whereas PhlF represses PphlF GFP expression by about 83-fold. However, TetR ( C0040 has been modified with an LVA tail for rapid degradation while PhlF has not, meaning the fold repression is not comparable between them.
Figure 7 Fold Repression. Repressor protein expression induced with 100μM IPTG. Values and error bars from experiments described above.
To further characterise PhlF and TetR, the concentration of IPTG used to induce repressor expression was reduced to investigate the range of regulation of GFP expression. Figure 8 shows that TetR has a wider range of regulation, likely due to quicker repressor regradation, whereas PhlF shows no significant difference between 100μM and 10μM IPTG.
Figure 8. Repressor constructs with pSB1C3 backbone; promoter driving GFP constructs with pSB3K3 backbone; in DS941 cells. Cells were grown overnight in 100μM, 30 μM, 10 μM, 3 μM, and 0 μM IPTG, to induce expression of the repressor proteins. Three replicates of the sample were diluted and tested under the same conditions for each sample. Mean and standard deviation of replicates were calculated to give value and error bars.
Conclusion
New Basic Parts, BBa_K1725000 (PphlF), BBa_K1725020 (PsrpR), and BBa_K1725040 (phlF encoding PhlF repressor) were successfully characterised, and submitted to the iGEM registry. BBa_K1725060 – (srpR encoding SrpR repressor) was characterised on our Bistable Switch page. PphlF (K1725000) drives expression of GFP in the BioBricks K1725001 and K1725002, and is significantly stronger than our control promoter, PtetR (R0040). PsrpR (K1725020) drives expression of GFP in the BioBricks K1725021 and K1725022, and is of equivalent strength to our control promoter, PtetR, when both were coupled with the strong Ribosome Binding Site, B0034. PhlF (K1725040) successfully represses PphlF driven GFP expression about 83-fold, and does not repress promoters PsrpR or PtetR. The next steps in characterisation of our repressor BioBricks would be to construct the BioBrick K1725062 (by insertion of the lacI regulated promoter, K1725080, upstream of K1725061, in order to express the SrpR repressor) then to repeat our fluorescence experiments to characterise SrpR (K1725060) further. Additionally, GFP purifed from E. coli would be used in place of iLOV fluorescent protein to calibrate the Typhoon FLA 9000, so the molecules of GFP per cell calculations would give a value closer to an absolute value for fluorescence.
References
Abbas, A., Morrissey, J.P., Marquez, P.C., Sheehan, M.M., Delany, I.R., and O’Gara, F. (2002). Characterization of Interactions between the Transcriptional Repressor PhlF and Its Binding Site at the phlA Promoter in Pseudomonas fluorescens F113. J. Bacteriol. 184, 3008–3016. Alberts et al (2008). Molecular Biology of the Cell (Garland Science, Taylor and Francis Group). Chapter 7, p337-434 Buckley, A. Petersen, J. Roe, A. Douce, G. Christie, J. (2015). LOV-based reporters for fluorescence imaging. Current Opinion in Chemical Biology. 27 (1), p39–45. Sheehan, M.M., Delany, I., Fenton, A., Bardin, S., O’Gara, F., and Aarons, S. (2000). Regulation of production of the antifungal metabolite 2,4-diacetylphloroglucinol in Pseudomonas fluorescens F113: genetic analysis of phlF as a transcriptional repressor. Microbiology 146, 537–546. Stanton, B.C., Nielsen, A.A.K., Tamsir, A., Clancy, K., Peterson, T., and Voigt, C.A. (2014). Genomic mining of prokaryotic repressors for orthogonal logic gates. Nat. Chem. Biol. 10, 99–105. Sun, X., Zahir, Z., Lynch, K.H., and Dennis, J.J. (2011). An Antirepressor, SrpR, Is Involved in Transcriptional Regulation of the SrpABC Solvent Tolerance Efflux Pump of Pseudomonas putida S12. J. Bacteriol. 193, 2717–2725. Wery, J., Hidayat, B., Kieboom, J., and Bont, J.A.M. de (2001). An Insertion Sequence Prepares Pseudomonas putida S12 for Severe Solvent Stress. J. Biol. Chem. 276, 5700–5706.
Read More!
Protocols
Bioluminesence
Terminator
UVA
Location
Bower Building, Wilkins Teaching Laboratory
University of Glasgow
University Avenue
G12 8QQ
Follow Us On
