<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Glasgow_Project_Overview_UVA skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Glasgow/Project/Overview/UVA</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><NAV class="mainNav"><UL><LI class="first"><A href="https://2015.igem.org/Team:Glasgow">Project</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview">Overview</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/UVA">UVA</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Repressors">Repressors</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Bistable">Bistable Switch</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Bioluminesence">Bioluminescence</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Description">RBS Library</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Terminator">Terminator</A></LI></UL><LI><A href="https://2015.igem.org/Team:Glasgow/Team">About Us</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Team">Team</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Attributions">Attributions</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Sponsors">Sponsors</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Notebook">Notebook</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Protocols">Protocols</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Basic_Part">Parts</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Basic_Part">Basic Parts</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Composite_Part">Composite</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Part_Collection">Collection</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Design">Application</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Design">Product Design</A></LI></NAV></P><P style="padding:0; margin:0; margin-top:0.8vh;">Policy &amp;</P><P style="padding:0; margin:0;"> Practice</P><UL><LI><A href="https://2015.igem.org/Team:Glasgow/AzureA">Azure A Staining</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Safety">Lab Safety</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Practices/Bedtimestories">Bedtime Stories</A></LI></UL><LI><A href="https://2015.igem.org/Team:Glasgow/Interlab">Measurement</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Interlab">Interlab Study</A></LI><LI><A href="https://2015.igem.org/Team:Glasgow/Collaborations">Collaborations</A></LI></DIV><P class="links scrollOverview"><A style="color:blue;" href="https://2015.igem.org/Team:Glasgow"> Home</A> &gt; <A style="color:blue;" href="https://2015.igem.org/Team:Glasgow/Project/Overview"> Project</A> &gt; <A style="color:blue;" href="https://2015.igem.org/Team:Glasgow/Project/Overview/UVA">UVA</A></P><DIV id="sidebar" class="widget widget-categories"><TABLE><TBODY><TR><TD class="overview">Summary</TD></TR><TR><TD class="sensor">Overview</TD></TR><TR><TD class="survivability">Sensor</TD></TR><TR><TD class="conclusion">Testing <I>P<SUB>lsiR</SUB></I></TD></TR><TR><TD class="results">Survivability</TD></TR><TR><TD class="top">Top</TD></TR></TBODY></TABLE></DIV><DIV class="containerRight"><H2 style="margin-top:13vh;">Summary</H2><P class="mainText">
Aims:
<UL><LI>To characterise the three components of a UV-A sensor system: <I>uirS</I>, <I>uirR</I>, <I>P<SUB>lsiR</SUB></I>.</LI><LI>To investigate <I>E. coli</I> survival rates in UV-A and sunlight.</LI></UL>
Results:
<UL><LI>Successfully isolated the components of the <I>UirS</I> Two-Component UVA sensor system from <I>Synechocystis</I> genomic DNA and produced biobrick compatible parts in pSB1C3
</LI><LI>Assembled composite parts using the Anderson library of Promoters to generate a selection of expression strengths of UVA system components for testing purposes
</LI><LI>Assembled UVA sensor composite parts in a two plasmid system utilising the schematic of the optimised green-light sensor <I>ccaS</I></LI><LI>Assayed strains of <I>E. coli</I> for UVA light survivability
</LI><LI>Tested UVA system assembled to drive GFP expression and theorised reasons for failure
</LI></UL>
Parts:
<UL><LI>Essential:

•	K1725400 (<I>P<SUB>lsiR</SUB></I>)

•	K1725410 (UirS)

•	K1725411 (UirS with RBS)

•	K1725420 (UirR)

•	K1725421 (UirR with RBS)

</LI><LI>Others:

•	K1725401 (<I>P<SUB>lsiR</SUB></I>.I13500)

•	K1725402 (<I>P<SUB>lsiR</SUB></I>.E5501)

•	K1725422 (J23101.B0032.UirR)

•	K1725423 (J23110.B0032.UirR)

•	K1725424 (J23114.B0032.UirR)

•	K1725425 (J23116.B0032.UirR)

•	K1725426 (J23101.B0032.UirR.B0015)

•	K1725427 (J23110.B0032.UirR.B0015)

•	K1725428 (J23114.B0032.UirR.B0015)

•	K1725429 (J23116.B0032.UirR.B0015)

•	K1725430 (J23101.B0032.UirR.B0015.<I>P<SUB>lsiR</SUB></I>.I13500)

•	K1725431 (J23110.B0032.UirR.B0015.<I>P<SUB>lsiR</SUB></I>.I13500)

•	K1725432 (J23114.B0032.UirR.B0015.<I>P<SUB>lsiR</SUB></I>.I13500)

•	K1725433 (J23116.B0032.UirR.B0015.<I>P<SUB>lsiR</SUB></I>.I13500)
</LI></UL></P><H2>Overview</H2><P class="mainText">
In order to prevent cells permanently shifting their metabolism towards one that favours light production, we decided to repress the bioluminescence genes (our optimised <I>lux</I> operon) in the presence of sunlight. The chosen system recognises unidirectional UV-A light and comes from the cyanobacterium <I>Synechocystis</I> sp. PCC6803. The system has not yet been characterized by an iGEM team for the Parts Registry. Three components are required to produce a response to UV-A: <I>uirS</I> (UV intensity response Sensor), <I>uirR</I> (UV intensity response Regulator), and <I>P<SUB>lsiR</SUB></I> (Promoter of the light and stress integrating response Regulator) (Song et al., 2011). Expression of a chromophore molecule, phycocyanobillin (PCB), is also required for functional light sensing. Phycocyanobillin is synthesised in a two step pathway from Heme within cyanobacterium. Several biobricks encoding the two phycocyanobillin synthesis pathway enzymes already existed within the Registry of Parts due to utilisation of PCB in other previously characterised <I>Synechocystis</I> sp. PCC6803 light sensors such as the <I>ccaS</I> green-light sensor system.  

We obtained <I>uirS</I> and <I>uirR</I> from <I>Synechocystis</I> genomic DNA via PCR. The forward primer used was flanked with the BioBrick prefix as well as the medium strength ribosome-binding site B0032, with the biobrick suffix flanking the reverse. We also utilised the reverse primer for <I>uirS</I> to modify the stop codon from TAG (suboptimal in <I>E. coli</I>) to TAA. 

A G-Block was ordered from IDT of the 535bp genomic region identified in Song et al. between the stop codon of <I>uirR</I> and start of <I>lsiR</I>, which was demonstrated to contain the <I>uirR</I> transcriptional activator binding region for the promoter of <I>lsiR</I>. The G-block was synthesised with the biobrick prefix and suffix attached, and a SpeI site within mutated away by one base (t58a). 

The <I>uirR</I> gene and <I>P<SUB>lsiR</SUB></I> promoter region were both biobrick compatible and were therefore ligated directly into the pSB1C3 plasmid; whereas <I>uirS</I> possessed three BioBrick incompatible restriction sites, preventing ligation into pSB1C3 and submission to the registry. 

In order to make <I>uirS</I> biobrick compatible we first had to clone it into a vector which would allow us to alter the  incompatible restriction sites by PCR mutagenesis. We chose to use the TOPO® TA Cloning® Kit from Invitrogen (<A href="https://www.thermofisher.com/order/catalog/product/450641">450641</A>) and followed the protocol provided (See protocols). Through a diagnostic digest of 5 miniprepped transformants it was observed that <I>uirS</I> inserted into TOPO vector reversed in all 5 cases. TOPO cloning is a non-directional process, but it was hypothesised at this stage that the combination of the lac promoter and very high copy number resulting from the pUC origin of replication present in pCR 2.1 TOPO vector lead to selection against expression of <I>uirS</I>.

Once successfully cloned into a vector we began the process of PCR mutagenizing the BioBrick incompatible restriction sites in a stepwise manner using the QuikChange II Site-Directed Mutagenesis Kit from Agilent (<A href="http://www.genomics.agilent.com/en/product.jsp?cid=AG-PT-175&amp;tabId=AG-PR-1161&amp;_requestid=221619">200523</A>) (See protocols). The Agilent QuikChange Primer Design tool was used to design mutagenic primers to modify a single base in the three illegal restriction sites without altering the amino acid sequence of <I>uirS</I> and attempting to match codon frequency. Although we were modifying <I>uirS</I> in its reversed conformation within TOPO vector, the following sequence labels were numbered in the forward direction from the start codon.
<UL><LI>XbaI site (1): 571-576
</LI><LI style="margin-left: 50px;">TCTAGA to TCTcGA = a573c
</LI><LI style="margin-left: 50px;">CTA (Leucine) to CTC (Leucine)
</LI><LI>EcoRI site (2): 1638-1643
</LI><LI style="margin-left: 50px;">GAATTC to GAATcC = t1641c
</LI><LI style="margin-left: 50px;">ATT (Isoleucine) to ATC (Isoleucine)
</LI><LI>XbaI site (3): 1790-1795
</LI><LI style="margin-left: 50px;">TCTAGA to TCTcGA = a1792c
</LI><LI style="margin-left: 50px;">AGA (Arginine) to CGA (Arginine)
</LI></UL><CENTER><FIGCAPTION>
Figure 1: Plasmid map of uirS after ligating into pCR 2.1 TOPO vector reversed. Generated by ApE (A Plasmid Editor).
</FIGCAPTION></CENTER>
The final step prior to beginning mutagenesis was performing polyacrylamide gel purification of the primer oligonucleotides and quantifying them with a spectrophotometer (See protocols).

The first illegal site modified using the Quikchange protocol was a1792c, followed by t1641c, then finally a573c. In order to track our samples the initial <I>uirS</I>.TOPO colonies were numbered 1 to 5, and DNA from colony 1 was used in the mutagenesis of a1792c. Following each mutagenesis several transformant colonies were given letter designations e.g. A-D and checked by diagnostic digest to confirm successful silencing of the incompatible site, then the DNA of one correct colony was chosen to move to the next step.

Interestingly, it transpired that the XbaI site of a573c was protected by Dam methylation; <I>E. coli</I> Dam-methylase methylates the Adenines of GATC sequences. XbaI sequence is TCTAGA, and at this position in <I>uirS</I> was flanked by TC, leading to methylation in the centre of the restriction site and inability for XbaI to bind and cut at the location. In practice this would also have meant that <I>uirS</I> would be biobrick compatible without removing this XbaI site, as there would not be interference with use of XbaI for plasmid assembly, however the Parts Registry would still reject the part as  BioBrick incompatible under the RFC10 standard as it does not consider methylation when processing part DNA sequence. Additionally, in order to visualise the silencing of a573c by restriction digest the use of a Dam Methylase knockout (Dam-) strain of <I>E. coli</I> (NEB: ER2925) was required. Plasmid DNA that is purified by miniprep from ER2925 <I>E. coli</I> lacks Dam methylation and thus the XbaI site can be cut and the fragments visualised. The other option to confirm successful mutagenesis of the site would have been DNA sequencing, but we had a sample of ER2925 available to use.

We used the program <A href="http://biologylabs.utah.edu/jorgensen/wayned/ape/">ApE</A> to create plasmid maps and digest predictions, the following figure shows the expected digest results at each stage of our PCR mutagenesis, if the miniprep DNA was cut with EcoRI and XbaI restriction enzymes.
<CENTER><FIGCAPTION>
Figure 2: Predictive digest of stepwise removal of biobrick incompatible restriction sites. Generated by ApE: A Plasmid Editor.
</FIGCAPTION></CENTER>
Miniprep DNA from each step (<I>uirS</I>1, <I>uirS</I>1A, <I>uirS</I>1AA, and <I>uirS</I>1AAA) was transformed into the ER2925 Dam- strain, a colony was cultured, miniprepped, and digested with EcoRI-HF and XbaI-HF enzymes. Undigested DNA from each sample was run in lanes 1-4.
<CENTER><FIGCAPTION>
Figure 3:  Restriction digest. Lanes 1-4 ; Uncut <I>uirS</I>1-1AAA. Lanes 6-9 <I>uirS</I>1-1AAA digested with EcoRI and XbaI. DNA miniprepped from Dam- methylation strain ER2925 so methylation protected XbaI site at position 523 can be checked by digest. 1% agarose gel stained with Ethidium bromide and photographed on a Bio-Rad GelDoc. Figure illustrates the removal of restriction sites in a sequential manner, with the final <I>uirS</I>1AAA sample being biobrick compatible.
</FIGCAPTION></CENTER>
Once <I>uirS</I> was biobrick compatible it was digested from pCR 2.1 TOPO plasmid and ligated into pSB1C3, then further light-sensor plasmid assembly could take place.

</P><H2> Sensor</H2><P class="mainText">
In <I>Synechocystis</I>, the <I>uirS</I> two-component system is involved in a negative phototactic response to unidirectional UV-A light (Song et al., 2011). The proposed mechanism places <I>uirS</I>, a transmembrane protein, as the protein that photosenses UV light. It is suggested that a phosphotransfer from <I>uirS</I> to <I>uirR</I> occurs upon UVA-light excitation of Phycocyanobillin chromophore bound within <I>uirS</I>; <I>uirR</I> is released from the transmembrane protein and locates to a binding region upstream of the lsiR promoter. <I>uirR</I> bound upstream of lsiR is shown to act as a transcriptional activator of lsiR (Song et al., 2011). <B>Figure 4</B> is the illustration of the <I>uirS</I> mechanism taken from Song et al.
<CENTER><FIGCAPTION>
Figure 4: Figure taken from Song et al., 2011. Illustrating the proposed mechanism of the <I>uirS</I> two-component UV-A photosensory system in <I>Synechocystis</I> sp. PCC6803.
</FIGCAPTION></CENTER>
The <I>uirS</I> system possesses structural and functional homologies with the ccaS green-light photosensor system, also from <I>Synechocystis</I> sp. PCC6803, which has been characterised by several previous iGEM teams (Edinburgh 2010, Wash U St Louis 2014). Both sensors are two-component systems where a bound Phycocyanobillin chromophore detects light within a transmembrane sensor protein (CcaS, UirS), leading to phosphorylation of a DNA binding regulatory protein (CcaR, UirR), which binds upstream of a promoter region (P<SUB>cpc</SUB>G2, P<SUB>lsiR</SUB>) inducing transcriptional activation. A 2014 paper by Schmid et al. “Refactoring and Optimization of Light-Switchable <I>Escherichia coli</I> Two-Component Systems”, detailed the optimisation of the <I>ccaS</I> green-light sensor system. We hoped that the investigations into optimised plasmid assembly for <I>ccaS</I>  could be applied to our <I>uirS</I> UV-A light sensor system due to their homologies.

The paper reports that for the <I>ccaS</I>  sensor, the most effective assembly is a two plasmid arrangement; where the sensor and PCB synthesis genes are placed on a low copy number p15A origin of replication plasmid, while the response protein, associated promoter, and reporter genes are placed on a high copy number ColE1 ori plasmid. Figure 5 is taken from Schmid et al and shows their optimised green light sensor assembly.

<CENTER><FIGCAPTION>Figure 5. Optimised <I>ccaS</I> green light sensor plasmid maps, taken from Schmid et al. 2014.
</FIGCAPTION></CENTER>
We replicated as closely as possible the two-plasmid assemblies with our <I>uirS</I> system biobricks, creating two composite parts: K1725430 (J23101.B0032.<I>uirR</I>.B0015.<I>P<SUB>lsiR</SUB></I>.GFP) and K1725445 (J23106.B0032.<I>uirS</I>.K322122(ho1 and pcyA)). To emulate copy number, in testing we utilised the pSB1C3 high copy number pMB1 plasmid for the K1725430 response-reporter plasmid, and pSB3K3 p15A low copy for the K1725445 sensor-chromophore plasmid.

As our chassis for UV-A testing we chose the <I>E. coli</I> strain DS941, which exhibited increased survivability under UV illumination when compared to both TOP10 and DH5α in our UVA illumination survival study. DS941 cells carrying K1725430.pSB1C3 plasmid were made chemically competent (Protocols) and transformed with the K1725445.pSB3K3 plasmid. Resulting transformants were cultured, miniprepped, and checked with a diagnostic digest to check presence of both plasmids. Three of the transformants checked appeared correct under diagnostic restriction digest.
We then designed an experiment to determine whether sunlight UVA illumination would result in expression of GFP from our assembled <I>uirS</I> two plasmid system. 

Six <I>E. coli</I> strains were tested:
<UL><LI>3 different DS941 transformants expressing both sensor and response plasmids 
</LI><LI>DS941 with K1725430 Response plasmid alone
</LI><LI>DS941 with K1725445 Sensor plasmid alone
</LI><LI>DS941 cells only
</LI></UL>
Cells were cultured overnight then 1 ml of o/n was added to 20 ml of fresh broth in a 100 ml conical flask, and incubated with shaking @ 37c for 2 hours. Then 10 ml of each cell culture was added to two empty Petri Dishes, one for sunlight illumination and one to be kept in darkness. Sunlight plates were placed on a tray by a window, and darkness plates were wrapped in tinfoil. After 5 hours 1ml of cells from each plate was pippetted into fresh eppendorf tubes, spun down to a pellet in a microfuge, and then resuspended in Phosphate binding solution (PBS: see interlab study protocol for full methodology). 200 µl of each cell type was placed in a microwell plate along with blank PBS as a control, and  scanned with a Typhoon fluorescence reader (see interlab study protocol for full methodology). 

No fluorescence above that exhibited by the cells-only control wells was observed, indicating that our assembly of the <I>uirS</I> two component system did not function. At this point there was not sufficient time remaining prior to the jamboree to do further testing to discern the reason for the failure of the system, however the two reasons we believe to be the most likely cause of failure are:
<UL><LI><I>P<SUB>lsiR</SUB></I> promoter from <I>Synechocystis</I> does not function as a promoter in <I>E. coli</I>. It may not be able to recruit the sigma-70 factor required for binding of <I>E. coli</I> RNA-polymerase to initiate transcription
</LI><LI>The Phycocyanobillin synthesis biobrick used <A href="//parts.igem.org/wiki/index.php?title=Part:BBa_K322122">K322122</A> may be supplied with a mutated promoter, leading to lack of expression of PCB. We have documented this error in sequencing of its promoter on the experience page on the part K322122
</LI></UL></P><H2> Testing <I>P<SUB>lsiR</SUB></I></H2><P class="mainText">
A crucial part of testing the <I>uirS</I> system in <I>E. coli</I> was ensuring that the <I>P<SUB>lsiR</SUB></I> promoter did not act as a constitutive promoter, thus negating the use of the system as a conditional transcriptional activator. We have confirmed through the use of a Typhoon FLA 9500 laser scanner that <I>P<SUB>lsiR</SUB></I> is not active when <I>uirS</I> and <I>uirR</I> are absent. 
<I>P<SUB>lsiR</SUB></I> was ligated to GFP with two ribosome binding sites of different strengths and no fluorescence was observed (the parts  used for this experiment were K1725401 and K1725402) (<B>Figure 6</B>). Placing <I>uirR</I> upstream of the strong constitutive promoter J23101 did not have an adverse effect on cell health or its ability to reproduce. 
<CENTER><FIGCAPTION>Figure 6: Average Fluorescence/A600 in DH5α Strains. Fluorescence and absorbance were measured at one-hour intervals and the ratio of the two was averaged. Fluorescence was measured in arbitrary units. Little to no additional fluorescence was observed in strains containing <I>P<SUB>lsiR</SUB></I>.I13500 or <I>P<SUB>lsiR</SUB></I>.E5501, when compared to DH5α cells containing no GFP or GFP without a promoter (E5501).</FIGCAPTION></CENTER><I>P<SUB>lsiR</SUB></I> was submitted as a separate BioBrick (K1725400) after the removal of the SpeI restriction site starting at the 53<SUP>rd</SUP> nucleotide. Assuming UirS and UirR are present, this BioBrick allows future teams to create a system that elicits a response to UV-A. Through BioBrick assembly two additional parts were created that allow for a system based on these three members to be tested by fluorescence measurements: K1725401 (which contains <I>P<SUB>lsiR</SUB></I> upstream of the RBS B0034 and GFP) and K1725402 (which contains <I>P<SUB>lsiR</SUB></I> upstream of the RBS B0032 and GFP).
<I>uirR</I> was put downstream of a B0032 RBS and was submitted as part K1725421. Future teams may find it useful to choose promoters expressing UirR different than the ones we have submitted. We ligated four promoters of varied strength (J23101, J23110, J23114, and J23114) as annealed oligos to <I>uirR</I> (the corresponding parts are K1725422, K1725423, K1725424, and K1725425). These parts can be tested in the future together with <I>P<SUB>lsiR</SUB></I> upstream of GFP to determine in what way levels of UirR influence light response. To further make future work on this system easier, we have also submitted parts that contain the double terminator B0015 after the constructs with one of the aforementioned promoters and <I>uirR</I> (K1725426, K1725427, K1725428, and K1725429). Thus, an iGEM team which wants a BioBrick that needs to be more strictly regulated downstream of <I>uirR</I> can add it effortlessly.

Finally, we made four parts that are made of K1725426, K1725427, K1725428, or K1725429 attached to K1725401.By combining <I>uirR</I>  that has a promoter and a terminator to <I>P<SUB>lsiR</SUB></I> upstream of GFP, we provide accessible BioBricks for the further characterisation of the UirS/UirR/ <I>P<SUB>lsiR</SUB></I> system via fluorescence measurements (K1725430, K1725431, K1725432, and K1725433).


</P><H2> Survivability </H2><DIV class="box"><H5>Introduction</H5><DIV class="text">
As our final Glow in the Dark genetic circuit would contain a UV-A light sensor, we decided to test how UV-A exposure affected different strains of <I>E. coli</I> in order to decide on the most UVA resistant strain to use as a chassis. It is well known that prolonged exposure to high doses of UVA can be deleterious to <I>E. coli</I>, but strains might be able to withstand the dose of UV-A required to activate out UVA sensor (50µM/m<SUP>2</SUP>/sec).</DIV></DIV><DIV class="box"><H5>Initial aims</H5><DIV class="text">
We decided to use a survivability assay to choose a chassis strain. TOP10, DH5α, DS941 and MG1655 strains were our possibilities. UV light causes mutations, so the strain must be able to repair it's DNA. <I>E. coli</I> has a group of <I>Rec</I> proteins which are involved in DNA repair during recombination, and have been modified in laboratory strains to allow for better plasmid transformation and growth. 

We predicted that the <I>recA-</I> mutants (TOP10 and DH5α) would be adversely affected by UVA exposure as it is central to the DNA repair mechanism that UVA damage would require, whereas DS941 is a <I>recF</I> mutant, which is a less crucial mechanism so cells may be less adversely affected by UVA exposure. MG1655 (wild-type for recombination) was used a control strain. 
</DIV></DIV><DIV class="box"><H5>Method</H5><DIV class="text">
1ml 10x serial dilutions were made up to 10-6 from a 5ml overnight of each strain. 10µl spots of each dilution were then spotted onto LB agar plates. 
These plates were then exposed to 50µM/m<SUP>2</SUP>/sec of UVA at room temperature in illumination cabinets. Time points were then taken by removing plates from the illumination cabinet.

After illumination, plates were incubated at 37°C, under the assumption that every single viable cell will form a colony. The length of incubation is irrelevant, provided that every cell is given enough time to form a visible colony. This also forms the basis of our counting system, where a colony is assumed to have come from a single cell. We also make the assumption that cell division at the temperature and time we were running the experiment was negligible/nonexistent.

After incubation, the colonies on each spot/dilution were counted. The number of colonies from the lowest visible dilution (some dilutions formed a lawn of growth) were then multiplied by the dilution factor to approximate how many cells would be in 10µl of undiluted culture.
</DIV></DIV><DIV class="box"><H5>Results</H5><DIV class="text"><CENTER>
As shown in Figure 7, TOP10 and DH5α cell counts drop dramatically after 20 minutes. DS941 and MG1655 show some variation, but do not appear to be as adversely affected as the <I>recA-</I> mutants. Therefore DS941 was chosen as our chasis.
<CENTER><FIGCAPTION>Figure 7: Estimated  cell count per 10ul of a 5ml overnight in lb broth over time of exposure to 50µM/m<SUP>2</SUP>/sec to UVA</FIGCAPTION></CENTER></CENTER></DIV></DIV><H2 class="readMore">Read More!</H2><DIV class="monsterContainer"><DIV class="monster"><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Protocols"><H3><SPAN class="monsterSpan">Protocols</SPAN></H3></A></DIV><DIV class="monster"><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Bioluminesence"><H3 style="left:2.5%;"><SPAN class="monsterSpan" style="font-size:20px;">Bioluminescence</SPAN></H3></A></DIV><DIV class="monster"><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Terminator"><H3><SPAN class="monsterSpan">Terminator</SPAN></H3></A></DIV><DIV class="monster"><A href="https://2015.igem.org/Team:Glasgow/Project/Overview/Repressors"><H3><SPAN class="monsterSpan">Repressors</SPAN></H3></A></DIV></DIV></DIV><DIV class="jumbotron"><DIV class="container"><DIV class="col-md-4"><H4><STRONG>Location</STRONG></H4><DIV class="footerinfo">
                    Bower Building, Wilkins Teaching Laboratory
                    University of Glasgow  
                    University Avenue 
                    G12 8QQ</DIV></DIV><DIV class="col-md-4"><H4><STRONG>Follow Us On</STRONG></H4></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>