Team:Glasgow/Project/Overview/Bioluminesence
Project
Overview
UVA
Repressors
Bistable Switch
Bioluminescence
RBS Library
Terminator
About Us
Team
Attributions
Sponsors
Notebook
Protocols
Parts
Basic Parts
Composite
Collection
Application
Product Design
Policy &
Practice
Azure A Staining
Lab Safety
Bedtime Stories
Measurement
Interlab Study
Collaborations
Home > Project > Bioluminescence
Introduction
Our Aims
Getting Started
Creating luxABG
Creating luxCDE
Creating Final Products
Top
Introduction
In this section of our project we aimed to engineer bacteria that are capable of producing bioluminescence. We were greatly inspired by the work of the Cambridge iGEM team from 2010 , and used some of their research as a starting point for our own. Their project featured the Lux Operon K325909 , originally from Vibrio fischeri, comprising of six lux genes that when expressed are capable of producing bioluminescence. These genes were luxC, luxD and luxE encoding three enzymes that act together to produce tetradecanal in the cell, and luxA and luxB which when expressed can use the decanal to produce luciferase and therefore bioluminescence. The final gene in the operon is luxG, which is thought to enhance the brightness of the bioluminescence, by producing reduced flavin mononucleotide (FMN) which is required for bacterial luminescence.
Our Aims
In 2010, the Cambridge iGEM team isolated the lux operon from V.fischeri, it was decided that our team would characterise the lux genes individually for submission to the registry. The Cambridge 2010 iGEM team used the native V. fischeri lux operon; each lux gene in the operon had its own (V. fisheri) ribosome binding site and the genes were in the order luxCDABEG (Figure 1). We aimed to reconstruct the lux operon for optimal performance using Biobrick assembly, changing the order of the lux genes from luxCDABEG (Figure 1) to luxABGCDE (Figure 2), and using a defined biobrick ribosome binding site ( B0032 ) for each gene in the operon. Our team originally aimed to engineer a luxABGCDE operon biobrick with no promoter, so that we could drive expression from the promoter in our inverter, or future iGEM teams could add their own promoter. We also wanted to make a test operon controlled by the arabinose regulated pBAD promoter ( BBa_I0500 ) to directly compare to K325909, and separately a construct where tetradecanal production was regulated independently from the light production enzymes LuxA, LuxB and LuxG. To do this we aimed to place luxABG under the control of the BBa_I0500 and luxCDE under the control of the IPTG-regulated pL-lac promoter, BBa_K1725080.
Figure 1 – Biobrick K325909 in pSB1C3 showing Gene Order of luxCDABEG. This biobrick constructed by the Cambridge 2010 iGEM team contains the native operon isolated directly from V. fischeri.
Figure 2 – A Diagram showing the final biobrick which we aimed to create, with genes order luxABCDEG.
Getting Started
To begin, we created primers for the individual lux genes. All of the forward primers were made to contain the BioBrick prefix, the B0032 RBS and a BioBrick Scar preceding the ATG of the open reading frame. All of the reverse primers incorporated the BioBrick suffix sequence. Our primers were used to carry out PCR using biobrick K325909 as a template, yielding the six individual lux genes (Figure 3). Once each of the genes was successfully isolated by PCR, they were digested with EcoRI and PstI and ligated individually into the pSB1C3 plasmid backbone and transformed individually into Top10 E. coli cells. The plasmids isolated from these ligations went on to become our first biobricks submitted for the Bioluminescence section: • K1725206 – B0032.luxA • K1725207 – B0032.luxB • K1725208 – B0032.luxC • K1725209 – B0032.luxD • K1725210 – B0032.luxE • K1725211 – B0032.luxG
Creating K1725216 (luxABG)
(TOP10 cells were used for all transformations using these genes)  The luxA gene was inserted upstream of luxB using AlwNI, SpeI and XbaI, different restriction enzymes from BioBrick Standard Assembly method, this method utilised the compatibility of SpeI and XbaI restriction sites (figure 4).    DNA extracted from transformants from this ligation   were tested by restriction digest and agarose gel electrophoresis to confirm the presence of the desired construct K1725212 (B0032,luxA, B0032, luxB). The arabinose-regulated promoter (BBa_I0500)   was inserted into K1725212 upstream of luxA to  demonstrate that when luxA and luxB were expressed and decanal was sprayed on the lid of the plate and allowed to diffuse across the plate, the cells produce bioluminescence (Figure  5). This was done again using AlwNI and SpeI to cut pSB1C3 (BBa_I0500), and AlwNI and XbaI to cut pSB1C3 (BBa_K1725212), creating pSB1C3 (BBa_K1725214; pBAD/araC, B0032, luxA, B0032, luxB) . Once transformed, 4 separate colonies were re-streaked onto plates containing arabinose, and the same 4 colonies re-streaked onto plates without arabinose. Non-arabinose plates were used as a control to show that luxA and luxB were both switched off when the promoter is not induced.
Figure 3: Assembly of LuxAB vector: pSP1C3 containing BBa¬¬_K1725206 (luxA) was cut with AlwNI and SpeI and ligated with pSP1C3 containing BBa_K1725207 (luxB) digested with AlwNI and XbaI. This ligation is possible due to the compatibility of the SpeI and XbaI restriction sites; however both sites are destroyed in the construction.
The next step for making the desired construct was to insert luxG into pSB1C3 (BBa_K1725212) and into pSB1C3 (BBa_K1725214) downstream of luxB using the biobrick standard assembly method. However, ligation of the luxG gene was unsuccessful, and after two attempts, due to time constraints, it was decided that luxG would be omitted from the final construct.
Figure 4: On the left, TOP10 cells containing pSB1C3 (BBa_K1725214), grown on nutrient agar containing arabinose and exposed to decanal diluted 1/10 and inoculated onto the lid of the plate. On the right, TOP10 cells containing pSB1C3-I0500-luxAB, grown on nutrient agar without arabinose and exposed to decanal.  Bioluminescence is being  produced only  by cells where luxA and luxB are expressed, left bottom image.
Creating K1725222 (luxCDE)
For assembly of luxCD,  it was decided to insert luxC upstream of luxD in the same way as for LuxAB,  using  AlwNI and SpeI to cut  pSP1C3 containing BBa¬¬_K1725208 (luxC), and AlwN1 and Xba1 to cut pSP1C3 containing BBa_K1725209 (luxD) (figure 5). The ligation was successfully transformed into TOP10.
Figure 5: Assembly of K1725222 luxCD vector: pSP1C3 containing BBa¬¬_K1725208 (luxC) was cut with AlwNI and SpeI and ligated with pSP1C3 containing BBa_K1725209 (luxD) digested with AlwNI and XbaI. This ligation is possible due to the compatibility of the SpeI and XbaI restriction sites; however both sites are destroyed in the construction.
The next step was to insert a promoter upstream of luxCD, as we had planned to measure the levels of tetradecanal produced when luxC, luxD and luxE were expressed. We decided to use the  IPTG-inducible pL-lac promoter (BBa_K1725080), which was inserted upstream of luxCD and transformed into TOP10 cells. However, no colonies were obtained. It was hypothesised that due to the absence of a lac repressor in Top10, the luxCD plasmid was being constitutively expressed. Constitutive expression of this plasmid may be harmful to the cell and could be preventing colony formation.  To overcome this, we decided to transform luxCD without a promoter into DS941.Z1 (genotype on our Protocols page) as this strain highly expresses the lac repressor protein. However, DNA from TOP10 is unmethlyated, so if it was transformed directly into DS941.Z1, the restriction system would digest the DNA and it would not tranform. Therefore, the DNA from TOP10 was first transformed into DH5α (which has the same methylation system as DS941.Z1, but doesn’t have the restriction system) which methylated the DNA for subsequent transformation into DS941.Z1. IPTG-inducible pL-lac promoter (BBa_K1725080) was inserted upstream of the luxCD plasmid (BBa_K175215), using the restriction enzymes AlwNI, XbaI and SpeI as previously described, creating (BBa_K1725215). In a separate construction, luxE was inserted downstream of luxCD, creating luxCDE (BBa_K1725222).   A final ligation was set up to insert luxE downstream of luxCD under the control of the IPTG-inducible pL-lac promoter (BBa_K1725215), creating luxCDE under control of the pL-lac promoter (BBa_K1725219).  Plasmid constructs were transformed into DH5 cells.
Creating Final Products
For the final product all that was left to do was to ligate luxAB to luxCDE, bearing in mind luxG was intended to have been included, but proved too difficult. luxCDE was inserted downstream of LuxAB and also I0500luxAB so that we could compare this construct to our BioBrick of the lux operon with optimised RBS (more detail on our RBS library page): • K1725223 - luxABCDE (Figure 6) • K1725224 - I0500.luxABCDE
Figure 6. Final construct K1725223 in pSB1C3.
We tested K1725224 (I0500.luxABCDE) compared to the original K325909 (luxCDABEG), three colonies with different brightness from our RBS library bioluminescence Biobricks, and K1725223 (luxABCDE) as a negative control, as shown in Figure 7.
Figure 7. Left to right: Bright colony from our RBS library (K1725350 - I0500.luxABG - in pSB1C3 and K1725351 - K1725080.luxCDE - in pSB3K3); K325909 in pSB1C3; K1725224 (I0500.luxABCDE) in pSB1C3; medium colony from RBS library (plasmids identical to bright colony, except different RBS); dim colony from RBS library (plasmids identical to bright colony, except different RBS); K1725223 (luxABCDE). All in TOP10 cells, and induced with 1% arabinose.
Evidently, the absence of luxG reduces the brightness of the bioluminescence. This supports our hypothesis that expression of luxG is important for optimising the brightness of the lux operon in E. coli.
Read More!
Protocols
UVA
Terminator
Repressors
Location
Bower Building, Wilkins Teaching Laboratory
University of Glasgow
University Avenue
G12 8QQ
Follow Us On
