Team:Glasgow/Description
Project
Overview
UVA
Repressors
Bistable Switch
Bioluminescence
RBS Library
Terminator
About Us
Team
Attributions
Sponsors
Notebook
Protocols
Parts
Basic Parts
Composite
Collection
Application
Product Design
Policy &
Practice
Azure A Staining
Lab Safety
Bedtime Stories
Measurement
Interlab Study
Collaborations
Home > Project > RBS Library
Summary
Introduction
Design
Strategy and Approaches
Discussion
Top
Ribosome Binding Site Library Summary
Aim
To optimise bioluminescence in Escherichia coli by creating a range of Ribosome Binding Sites (RBS) for each of the six genes in the luxCDABEG operon from Aliivibrio fisheri, originally submitted to the registry as a single BioBrick ( K325909 ) in 2010 by the Cambridge team.
Results
Designed RBS library with 32 variants for each lux gene.
Made luxABG and luxCDE constructs from the RBS library – over 32000 RBS variants for each construct.
Showed that cells are able to uptake decanal from the environment and produce light when luxAB or luxABG are expressed.
Visualisy screened bioluminescence of RBS libraries for luxAB, luxABG and luxCDE constructs and determined optimal RBS arrangements for light production in E. coli.
Showed that E. coli can communicate by decanal movement between cells in a 2-cell communication system
Optimised the lux operon for improved bioluminescence in E. coli when compared to previous attempts.
Biobricks
Composite Parts
BBa_K1725350: 721 I0500-K1725305-luxA-K1725309-luxB-K1725302-luxG
BBa_K1725351: K1725080-K1725307-luxC-K1725302-luxD-K1725301-luxE
BBa_K1725352: I0500-K1725305-luxA-K1725309-luxB-K1725302-luxG-K1725080-RBS-luxC-RBS-luxD-RBS-luxE
Basic Parts: RBS library
BBa_K1725301
BBa_K1725302
BBa_K1725303
BBa_K1725304
BBa_K1725305
BBa_K1725306
BBa_K1725307
BBa_K1725308
BBa_K1725309
BBa_K1725310
BBa_K1725311
BBa_K1725312
BBa_K1725313
BBa_K1725314
BBa_K1725315
BBa_K1725316
BBa_K1725317
BBa_K1725318
BBa_K1725319
BBa_K1725320
BBa_K1725321
BBa_K1725322
BBa_K1725323
BBa_K1725324
BBa_K1725325
BBa_K1725326
BBa_K1725327
BBa_K1725328
BBa_K1725329
BBa_K1725330
BBa_K1725331
BBa_K1725332
Introduction
For the Bioluminescence part of our project we used the luxCDABEG operon from A. fischeri introduced to iGEM by the Cambridge team in 2010. Five lux genes are known to be essential for bioluminescence production: luxA and luxB encoding bacterial luciferase and luxC, luxD and luxE encoding an enzyme complex that synthesises tetradecanal, a substrate for the luciferase. A sixth gene, luxG encodes a flavin reductase that provides reduced flavin mononucleotide for the bioluminescence reaction resulting in an enhanced light output. Initially we decided to optimise bioluminescence in E. coli by rearranging the whole Lux operon and placing a defined relatively-weak ribosome binding site – B0032 – upstream of each of the six lux genes, as described on our Bioluminescence page. Taking this approach further, we thought of adjusting bioluminescence in E. coli by creating a B0032-derived Ribosome Binding Site library for each lux gene. The idea behind this was to create a range of RBS combinations in a lux operon and therefore, generate E. coli strains of different bioluminescence intensity. We assumed that the most favourable RBS arrangements in lux operon should be observed in the E. coli colonies emitting the most light.
Design
For the construction of the RBS library, we used a master sequence based on the RBS B0032 (Figure 1). 4 nucleotides within the actual ribosome binding site were randomised giving 32 different B0032-derived RBS variants. The predicted efficiency of each RBS library member was estimated using RBS Library Calculator (ref: http://msb.embopress.org/content/10/6/731) for every lux gene. Theoretically, with 32 different RBS variants for each of the six lux genes, the final RBS library for the complete lux operon would have over a billion different RBS combinations.
Figure 1. Design of the RBS library.
Strategy and approaches
Randomised PCR and Cloning, Cloning, Cloning
For construction of the RBS library, each lux gene was amplified by PCR using primers with a 5’ extension containing a BioBrick prefix and the B0032-derived degenerate RBS sequence. PCR products were cleaved with restriction enzymes (EcoRI and PstI), ligated into the plasmid pSB1C3, and transformed to E. coli strain TOP10 which is a recA- mutant meaning that any unwanted gene rearrangements between repeated DNA sequences on the plasmid can be avoided. In order to induce expression of luxABG and luxCDE, luxA and luxC PCR products were inserted downstream the pBAD ( BBA_ I0500 ) and BBa_K1725080 promoters, respectively. pBAD is a widely used promoter inducible by L-arabinose, in our assays we have used 0.2% arabinose to activate the promoter. The BBa_K1725080 promoter is identical to the IPTG-regulated, LacI-repressed R0011 promoter ( BBa_R0011 ) but contains an NheI site upstream of the promoter to make Biobrick insertion identification easier in the restriction digests. Since we have used lacI- Top10 cells for all our transformations, R0011N was constitutively active. Colonies from the transformation plates were then washed from the plate and plasmid DNA was purified from mixed pool of colonies. The pooled plasmid DNA was and sequenced to ensure that all 32 RBS library members were present in the sample (Figure 2). A similar approach was applied to the subsequent ligations in the assemblies of pBAD.luxABG and R0011N.luxCDE.
Figure 2. Sequencing results of luxA amplified with a primer containing randomised RBS B0032 sequence. Four peaks at the same position are observed where sequence was randomised to N (A, C, G and T) and two peaks (A and G) are observed where nucleotide location was randomised to R.
Testing pBAD.luxAB
Once we have assembled luxA and luxB together with a pBAD promoter upstream, we wanted to determine if the construct allows cells to produce light. To do this we had to supply exogenous aldehyde as substrate for bacterial luciferase. In addition to that, we also aimed to visualise RBS library in the construct as RBS of different strengths should cause variability in bioluminescence intensity between colonies. To start, we grew transformed cells on the L-agar with 0.2% arabinose and then exposed them to the decanal vapour. The method that we developed to supply decanal involved placing 6 10μl of 10% decanal in ethanol on the lid of the Petri dish, and then immediately putting lids on the plates. After 3-4 minutes of exposure, we found that bioluminescence had started, and we could photograph the plates without the lids in the dark room with a 30s exposeure at ISO 64000 on Canon EOS D650 digital Camera. We have also found out that leaving the lids with decanal on the plates result in much more intense bioluminescence visible by naked eye.
Figure 3. Pictures of plates with pBAD.luxAB colonies in  white light and in the dark. Plate on the left side contains L-arabinose, plate on the right is a control with no arabinose.
As seen in the Figure 3, colonies that grew on the plate containing arabinose (ara+) show bioluminescence activity in the dark while control plate with no arabinose (ara-) did not show any bioluminescence. More importantly, in the ara+ plate we observe that, on the photograph, colonies varied in bioluminescence intensity from very bright to absolutely blank. Therefore, here we show that E. coli is able to uptake decanal from the environment and produce light when the expression of luxAB is turned on. Moreover, we demonstrate that some of the RBS arrangements in the pBAD.luxAB construct are more efficient than others in terms of stimulating translation initiation. For further testing, we have picked up several colonies that exhibited different bioluminescence intensity: a range from very bright to dim or blank colonies. We tested all chosen colonies by restriction digests for the plasmid size in order to test if any of the genes are missing from the dim colonies and all colonies were confirmed to be similar in plasmid size. In addition, we have made short streaks of each colony on ara+ plate in order to compare their brightness on a bigger patch of cells (Figure 4). From the picture we can clearly see colony F being the brightest colony on the plate and some of the patches producing very little bioluminescence. This again supports our hypothesis about some RBS combinations being more favourable for optimal bioluminescence production  by the E. coli translation machinery.
Figure 4. pBAD.luxAB colonies A-J. Initial transformation plate with marked locations where colonies A-J were picked (left)and short streaks of colonies A-J. Yellow: bright colonies, Orange: medium-brightness colonies, RED: Dim colonies (classified from their initial appearances on the transformation plate)
Mr. Bright: luxG
As mentioned before, luxG is a known flavin reductase and provides reduced flavin mononucleotide to luciferase luxAB resulting in increased bioluminescent activity. Therefore, having the pBAD.luxAB construct ready we have inserted luxG gene downstream of luxAB RBS libraries in order to increase the bioluminescence in cells. Transformed cells were grown on both ara+ and ara- plates and treated with decanal as described earlier.
Figure 5. Pictures of plates with pBAD.luxABG colonies in white light and in the dark. Plate on the left side contains L-arabinose, plate on the right is a control.
As can be seen in the Figure 5, only colonies that have grown on the ara+ plates exhibit luminescence while control plates remain blank in the dark. Similarly to the results observed in pBAD.luxAB testing, here we also see a range of luminescence between colonies on ara+ plate most likely corresponding to the different ribosome binding sites for luxG. However, on the ara+ plates we observe only half as many colonies as on the ara- plates suggesting luxG expression from some RBS sequences were detrimental to cell growth and prevented colony formation. To test this, we have picked 136 random colonies from the ara- plate and streaked them on new ara+ and ara-plates to see difference in survivability. Results are presented in the Figure 6 where short streaks appear to be similar on both ara+ and ara- plates with no bacterial growth disruption on the ara+ plate. Therefore, we assume that the negative luxG effect is not very severe and may be only observable on the transformation plates where colonies are formed from a single cell.
Figure 6. Pictures of pBAD.luxABG colonies streaked on ara+ and ara- plates in white light and in the dark. Plate on the left side contains L-arabinose, plate on the right is a control.
Additionally, in order to compare bioluminescence between cells where luxG is expressed and in cells where it is not, we also streaked previously described 12 pBAD.luxAB colonies and 12 brightest pBAD.luxABG colonies on the same ara+ plate. As can be seen from the Figure 7, most of the pBAD.luxABG colonies appear to exhibit brighter luminescence than the pBAD.luxAB colonies which is what we expect to see.
Figure 7. Picture of pBAD.luxAB and pBAD.luxABG colonies streaked on ara+ plate taken in the dark. Left bottom corner: 12 pBAD.luxAB colonies with different RBSs for luxA and luxB; right bottom corner 12 'bright' pBAD.luxABG colonies; top and middle: 136 random pBAD.luxABG colonies.
We have picked up the brightest pBAD.luxABG colony, determined RBS for each gene by sequencing and submitted the construct as the biobrick BBa_K17252340.
The luxCDE story
Simultaneously with the assembly of pBAD.luxABG we have also assembled K1725080.luxCDE construct using similar approaches (with our IPTG inducible promoter driving expression of luxCDE). In order to test our RBS library in luxCDE we employed two strategies. Firstly, we wanted to test if two strains, one expressing luxABG and other expressing luxCDE, are able to produce luminescence when mixed. The idea was that the strain expressing luxCDE would produce aldehydes such as tetradecanal, and the strain expressing luxABG would use it to produce light. We hoped that the bioluminescence level would depend on the amount of tetradecanal produced. We mixed equal volumes of strains carrying 24 different K1725080.luxCDE variants with different RBSs with our best pBAD.luxABG (BBa_K17252340) overnight culture in a 96-well plate (the pBAD.luxABG overnight culture was grown for two hours in 1% L-arabinose before addition to the plate). However, we could not observe any bioluminescence in the wells. We repeated the assay, this time growing both K1725080.luxCDE and pBAD.luxABG overnight cultures together in 1% L-arabinose for several time intervals. We only tested two random K1725080.luxCDE colonies and after 3hrs we observed a very low level of bioluminescence which was different between two samples (Figure 8). This biofluorescence was also confirmed using a luminometer. This was a first proof that aldehyde is released by cells expressing luxCDE and can then be taken up by cells with the luxABG genes. However, due to the lack of time we did not study this process in more detail leaving it for the possible future investigations.
Figure 8. Testing 2-cell communication system. Flasks with the same pBAD.luxABG (BBa_K17252340) and  different K1725080.luxCDE colonies in each flask photographed in white light and in the dark. The culture on the left demonstrated a low level of bioluminescence, seen as a faint green tint in the broth.
The second strategy that we have developed for testing our RBS library in luxCDE assembly involved generating E. coli strain with two plasmids: pSB1C3 with pBAD.luxABG and pSB3K3 with K7125080.luxCDE. We have used pBAD.luxABG plasmid DNA (BBa_K1725340) for the first transformation and then transformed resulting cells with K7125080.luxCDE RBS library which was first ligated into pSB3K3 (plasmid with kanamycin resistance). In terms of ribosome binding sites, we aimed to have the same type of luxABG in all cells and different type of luxCDE in each cell in order to ensure that any differences in bioluminescence arise from ribosome binding sites in the luxCDE genes. Transformed cells were grown on an ara+ plate over night at 37°C and kept for an hour at room temperature before being photographed (Figure 9). Once again, we observed a range of bioluminescent colonies, and this time light output was seen by naked eye: we were able to see single colonies emitting different amounts of light. As previously observed, bioluminescence increased in all of our assays if the cells were kept below 30°C for some time (Figure 9 right).
Figure 9. Plates of cells transformed with pBAD.luxABG (BBa_K17252340) and RBS library of R0011N.luxCDE in PSB3K3. Left: Picture taken after 1hr at room temperature; right: picture taken after 5hrs at the room temperature
We picked the brightest colony again, purified the pSB3K3 plasmid and then the ligated K1725080.luxCDE (BBa_K7125351) construct downstream of the pBAD.luxABG (BBa_K7125350) generating the new biobrick pBAD.luxABG.R0011N.luxCDE (BBa_K7125352).
Discussion
Decanal
During the construction of the RBS library for lux genes we have demonstrated that E. coli is able to uptake decanal present in the environment and produce bioluminescence when the expression of luxAB or luxABG is induced. In addition, we have also shown that 60μl of 5% decanal solution equally dispersed on the plate lid can successfully induce bioluminescence in cells expressing luxAB or luxABG. Finally, we have demonstrated that leaving the lid with decanal on the plate for a longer time results in much more intense luminescence visible by naked eye (Figure 10).
Figure 10. Plates with pBAD.luxAB colonies exposed to decanal. Left: plates without the lids; right: lids with decanal kept on the plates.
Cell communication
When testing the RBS for the luxCDE genes we have shown that E. coli cells expressing luxCDE might be leaking aldehyde (presumably tetradecanal) to the environment to some extent as mixing them with cells expressing luxABG resulted in a weak bioluminescence. Therefore, we propose that E. coli cells might be able to communicate by tetradecanal movement between cells.
Comparison to luxCDABEG (BBa_K325909)
We have compared our RBS-optimised luxABGCDE operon to the luxCDABEG operon generated by Cambridge team in 2010. Results show our strain being brighter than Cambridge team's strain and producing more intense bioluminescence (Figure 11).
Figure 11. Comparison to luxCDABEG (BBa_K325909) Left: flask with BBa_K1725352 colony (GlasGlow); right: flask with BBa_K325909 colony (E. glowli)
RBS optimisation
The main goal of our RBS library was to optimise bioluminescence in E. coli by generating strains with a variety of ribosome binding sites for lux genes and then pick up the brightest colonies. We managed to create RBS libraries for luxABG and luxCDE assemblies and test them separately for the translation initiation efficiency. We then generated strain with luxABGCDE (Bba_ K1725352) in one plasmid consisting of luxABG (Bba_ K1725350) and luxCDE (Bba_ K1725351) constructs that gave colonies the highest intensity of bioluminescence. RBS sequence for every lux gene was then determined and their estimated strength compared to the B0032 (Figure 12).
Figure 12. Relative RBS strengths for B0032 and RBS of luxABGCDE (Bba_ K1725352) as estimated by RBS library calculator. Red circle: estimated RBS strength of the optimised RBS in luxABGCDE for each lux gene; arrow: estimated RBS B0032 strength for each lux gene.
From the calculations given by the RBS library calculator, we can clearly see that almost all of our RBS sequences (except RBS of luxB and luxC) from our brightest strain are estimated to be more efficient than B0032 in translation initiation. Therefore, we conclude that we have increased bioluminescence in E. coli by successfully adjusting ribosome binding sites of lux genes to the translation mechanism in E. coli. In conclusion, we successfully incorporated 32 different BioBrick RBS variants upstream of each of the lux genes by PCR, and then combined the different libraries together by BioBrick assembly. This method could potentially produce over 1 billion different variants of the lux operon. The method was relatively quick, and could easily be used to optimize other metabolic pathways where a suitable screen or selection for optimal performance is available.
Location
Bower Building, Wilkins Teaching Laboratory
University of Glasgow
University Avenue
G12 8QQ
Follow Us On
