<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_WLC-Milwaukee_Modeling skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:WLC-Milwaukee/Modeling</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wlcHeader" class="navLinks" style="z-index:5;top:14px;left:0px;position:fixed;height:90px;width:100%;"><TABLE width="100%" align="center" cellspacing="0"><TBODY><TR height="90px"><TD id="HomeBox"><A id="homeLink" href="https://2015.igem.org/Team:WLC-Milwaukee" style="text-decoration:none">Home</A></TD><TD id="ProjectBox"><A id="projectLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Project" style="text-decoration:none">Project</A></TD><TD id="PartsBox"><A id="partsLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Parts" style="text-decoration:none">Parts</A></TD><TD id="NotebookBox"><A id="notebookLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Notebook" style="text-decoration:none">Documentation</A></TD><TD id="ModelingBox"><A id="modelingLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Modeling" style="text-decoration:none">Modeling</A></TD><TD id="practBox"><A id="practLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Practices">Policy</A></TD><TD id="teamBox"><A id="teamLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Attributions">People</A></TD></TR></TBODY></TABLE></DIV><DIV id="container"><DIV id="element"><P>Modeling</P></DIV></DIV><DIV style="padding-left:10%;padding-right:10%;font-size:11pt"><LABEL for="hider"><H1 id="heading_1"><FONT color="#610B0B"><B> OVERVIEW </B></FONT></H1></LABEL><P>Click the arrow above for the summary of the rationale behind our experiments and our results: </P><DIV id="content_area_1"><P>In order to probe further into the feasibility of our project, we addressed the fact that bacteria evolve in response to bacteriophages. The basis of our project is that gram-negative bacteria without a working <I>tolC</I> gene (be it knocked-out, misfolding, or incomplete) will be unable to form a complete antibiotic-efflux protein complex. As a result bacteria without a functioning <I>tolC</I> gene will show an increased sensitivity to antibiotics normally resisted through efflux[1]; we demonstrated this experimentally using a Kirby-Bauer assay and the antibiotics Novobiocin and Erythromycin (See Documentation &gt; Results &gt; Kirby-Bauer Assays). </P><P>	Since evolution is driven by selection of random mutations, we decided to imitate this mechanism. In order to simulate this, we would need a way to simulate the insertion of random non-silent mutations into a <I>tolC</I> gene/protein, a way to predict whether or not these mutations were damaging to the function of the TolC protein, and a way to predict whether or not these would affect TolC-mediated phage binding. For the sake of simplicity we considered only point mutations to the coding-region of the DNA. We divided the mutations into 4 main types: </P><UL><LI>A nonsense mutation which turns an amino acid into a stop codon; this will prevent a full TolC protein from being translated, preventing both antibiotic efflux and phage binding. </LI><LI>A missense mutation which prevents the TolC monomers from forming a trimer. In these we included strongly polar or charged amino acids being mutated into the beta-barrel regions, the insertion of Valine, Threonine, Isoleucine, Serine, Aspartate, Asparagine, or Proline in the alpha helices (these amino acids are not accommodated in an alpha helix[2]) which are critical for TolC trimerization, and in addition 1/4 of any other mutations in the alpha helices (we assume the amino acids on one face of each helix are critical for interaction with other monomers; there are ~3.6 amino acids per turn). </LI><LI>A base-pair substitution in the extracellular loops of the TolC protein. These mutations do not affect the assembly or function and therefore do not affect antibiotic resistance[1]. We assumed any of these would result in phage-resistance.</LI><LI>Any mutation not fitting into the above three categories we assumed would not affect assembly/function or the ability of bacteriophages to recognize the protein. Therefore these show no effect on phage resistance or antibiotic sensitivity.</LI></UL><DIV style="float:right;margin:10px 10px;">
Example of mutation program input (E. coli tolC sequence)</DIV><P>With these parameters, we constructed a program to simulate random point mutations. C++ was chosen for its universality as well as our familiarity with it. At the base of the program is a codon class; this contains room for 3 characters to be filled with 3 nucleotides from the DNA sequence. When filled or altered, the class analyzes itself to determine which amino acid it represents, its polarity, and its charge. Codons have the ability to mutate themselves (pick one of their three constituent bases and change it to one of the other three nucleotides), as well as the ability to return their contents, polarity, charge, and amino acid. Codons were organized and used in a single-linked list controlled by a polypeptide class. The polypeptide class stored a string of higher-level information about each codon in a parallel single-linked list; the information stored here was used to store what secondary structure the specific amino acid was in. When told to mutate, the Polypeptide class used a random number generator (rand() from stdio.h) seeded with the current time to pick which amino acid will be mutated; the randomly selected amino acid is then told to mutate. </P><DIV style="float:right;margin:10px 10px;">
Example of the file used to associate secondary structures with codons</DIV><P>Handling Input/Output as well as analysis of the mutations (for us, this included predicting function destruction and if it affected phage binding) was done in a main class, so the underlying classes could easily be adapted for other projects.  Input of the DNA sequence was done by parsing a plain-text sequence from a file. The amino acid properties is input from a plain text file and parsed as an integer and a string in which the string is the property of the amino acid and the integer designates how many amino acids it applies to. The user inputs how many mutations are desired through the CLI. Information about each mutation is outputted into a csv file (comma separated values) and at the end the program prints the frequency of amino acids changing their charge (or lack of it), changing their polarity, turning into premature stops as well as the percentage of mutations that are predicted to destroy function of the TolC and the percentage of mutations which would interfere with phage binding.  Assigning the <I>tolC</I> codons to a specific secondary structure was based on previous research [3].</P></DIV><LABEL for="hider2"><H1 id="heading_2"><FONT color="#610B0B"><B> RESULTS &amp; LIMITATIONS </B></FONT></H1></LABEL><P>Click the arrow above for the results obtained from out experiments and assessment of the simplifications and limitations of our method: </P><DIV id="content_area_2"><B>Limitations</B><P>Our model is fairly simplistic in its nature. It is limited in the scope of mutations, only handling missense and nonsense mutations. Insertions and deletions/frameshift mutations would add considerable layer of complexity in programming both how to generate these mutations as well as analyzing the effects of these mutations. Our model also limited itself to the coding portions of the genes. The effect of mutations on any control sequences (promoter, RBS, ect…) would be more difficult to predict because their effects are not due to amino acids, but rather their interactions with other molecules in the cell. One could hypothesize that a mutation either upregulating or downregulating gene expression could work in favor of our project; downregulation (or stopping transcription) would likely make the bacteria hypersensistive to the antibiotics, and an upregulation could increase the likely hood that that individual bacterium would be infected by a bacteriophage (more receptors = more binding opportunities) <CITE phage="" book="" page="" on="" factors="" in="" phage="" binding=""> . We, however, are not prepared to make that claim. </CITE></P><B>Results</B><P>Results are output as a .csv file which can be opened using a program like Microsoft Excel or Apple’s Numbers for further analysis. On a run of 1000 mutants on <I>E. coli’s tolC</I> gene we got the following numbers:
<TABLE><TBODY><TR><TD width="10%">Changes of charge</TD><TD> 358/1000</TD></TR><TR><TD>Polarity changes</TD><TD> 338/1000 </TD></TR><TR><TD>Premature stops added</TD><TD> 46/1000 </TD></TR><TR><TD>Predicted Function Destroying Mutations</TD><TD> 395.25/1000 </TD></TR><TR><TD>Predicted mutations preventing phage binding</TD><TD> 446.25/1000</TD></TR></TBODY></TABLE></P><P>Meaning out of every 1000 <I>tolC</I> mutants we would predict approximately 446.25 to be resistant to a TolC-binding bacteriophage, and 553.75 to be infectable. Of the phage-insensitive mutants, we would predict 395.5 (88.6%) to be hypersensitive to (what would be under normal circumstances) efflux-resisted antibiotics, and 50.75 (11.4%) to be both phage insensitive as well as possessing the normal antibiotic tolerance.</P><DIV style="inline;margin:10px 10px;">
Screenshot of the output .csv file opened in Excel</DIV></DIV><LABEL for="hider3"><H1 id="heading_3"><FONT color="#610B0B"><B> EXPERIMENTAL VERIFICATION </B></FONT></H1></LABEL><P>Click the arrow above to see the methods used to verify our results: </P><DIV id="content_area_3"><DIV style="float:right;margin:10px 10px;">
Example of a TLS phage-Ampicillin-ΔtolC <I>E. coli</I> transformed with <I>E. coli</I> K12 <I>tolC</I> plate</DIV><P>A model is only as informative as it is accurate. Knowing this we had a plan to test whether or not our model was actually giving us something resembling the reality. We planned a week-long protocol for evolving and analyzing mutants resistant to the TolC-binding TLS bacteriophage (referred to as TLS phage). The protocol is outlined is as follows:
<TABLE><TBODY><TR><TD>Day 0</TD><TD>
Take 250µL of overnight grown culture of ΔtolC <I>E. coli</I> (knockout-<I>tolC E. coli</I>) with the pUC57 plasmid containing the <I>E. coli</I> K12 <I>tolC</I> gene mix with 50 µL of TLS phage stock and plate in 4mL 50µg/mL Ampicillin (Luria Broth) Top Agar on an LB-Amp plate (Make 1 per person).</TD></TR><TR><TD>Day 1</TD><TD>
Pick 10-15 mutant colonies and streak them onto a 1/6 or 1/8 sectioned LB-Amp plate. </TD></TR><TR><TD>Day 2</TD><TD>
Pick 1 mutant colony from each colony streaked and grow them overnight in a LB-Amp liquid culture. 
Grow enough ΔtolC <I>E. coli</I> cells for 10-15 transformations. (Electroporation or Heat Shock) in a LB liquid culture.</TD></TR><TR><TD>Day 3</TD><TD>
Miniprep the 10-15 cultures grown from picked colonies, carefully labeling the resulting plasmids so you can distinguish one from another. 
Prepare the ΔtolC <I>E. coli</I> cells as competent cells (Electrocompetent or Thermocompetent for whichever method you prefer).
Transform with the 10-15 plasmids you just miniprepped, being extra sure that you know which transformation is from which plasmid.Plate transformation on LB-Amp plates after the resting period. </TD></TR><TR><TD>Day 4</TD><TD>
Make freezer stocks of successfully transformed bacteria, making sure you know which stock goes with which plasmid. 
Perform cross-streaks on Amp plates with TLS phage crossing every successfully transformed sample.</TD></TR><TR><TD>Day 5</TD><TD>
Make stocks of the samples whose bacteria are NOT killed by cross-streaks (these ones we are now sure have a TLS-inhibiting mutation in the <I>E. coli tolC</I> gene) 
Throw out other samples and their corresponding minipreps.</TD></TR></TBODY></TABLE></P><DIV style="float:right;margin:10px 10px;">
8 streaks of picked colonies after 4 days of incubation</DIV><P>The idea is that we would take our knockout <I>tolC</I> (ΔtolC) <I>E. coli</I> strain and transform them with a plasmid containing the <I>E. coli tolC</I> gene (behind  a pBAD promoter and a strong <I>E. coli</I> RBS) and an ampicillin-resistance gene. We would then grow them as if we were making a lawn, but mix in an abundance of TLS phage and a working concentration of ampicillin. It was hoped that the ampicillin would prevent any bacteria without the plasmid from growing. The TLS phage would provide the selective pressure to select for phage resistant mutants. One of these plates is pictured. </P><P>It was immediately noted that there was considerable variability in the size of colonies (initially presumed to be caused by different mutations. Twelve colonies were picked making sure to get a variety of colony sizes; none of these grew on LB-Amp. Fifteen colonies were picked from a similar plate with identical results. It is highly likely that the colonies picked were satellite colonies that had abandoned their plasmids under selective pressure from the TLS phage. </P><P>Attempting the experiment with 2, 4, and 6 times the working concentration of ampicillin resulted in approximately 10% of picked colonies growing. Due to time constraints  further experimentation was halted.</P></DIV>
Bibliography:
[1] Anne Marie Augustus; Teresa Celaya; Fasahath Husain; Matthew Humbard; Rajeev Misra; “Antibiotic-Sensitive TolC Mutants and Their Suppressors.” <I>Journal of Bacteriology</I> Vol. 186 No. 6, Mar. 2004, p. 1851–1860. 
[2] Tymoczko, John L.; Berg, Jeremy M.; Stryer, Lubert (2011-12-23). <I>Biochemistry</I> (Page 51). W. H. Freeman. Kindle Edition.
[3] Koronakis V.; Sharff A.; Koronakis E.; Luisi B, Hughes C.; “Crystal structure of the bacterial membrane protein TolC central to multidrug efflux and protein export.” <I>Nature</I>. 2000 Jun 22;405(6789):914-9.</DIV></DIV></DIV></DIV></DIV></BODY></HTML>