Team:WLC-Milwaukee/Notebook
Home
Project
Parts
Documentation
Modeling
Policy
People
Documentation
MEDALING QUALIFICATIONS
What medaling qualifications did we meet this year? Click the arrow above to find out.
Bronze  
Our team has satisfied the following departments: 
Registered for iGEM, had a great summer, and attended the Giant Jamboree.
Completed the Judging Form.
Created and shared a Description of the team's project using the iGEM wiki, and documented the team's parts using the Registry of Standard Biological Parts.
Will have presented a poster and a talk at the iGEM Jamboree.
Created a page on our team wiki with clear attribution of each aspect of our project. This page clearly attributes work done by the members of our team and distinguishes it from work done by others.
Documented at least one new standard BioBrick Part or Device central to our project and submitted it to the iGEM Registry. We have also documented a new application of a BioBrick part from the 2014 iGEM year, which was the pBAD promoter plus strong RBS.
Silver  
In addition to the Bronze Medal requirements, our team has also satisfied these 3 following departments:
Experimentally validated that our new BioBrick Part or Device of our own design and construction works as expected. Yersinia pestis tolC, Pseudomonas tolC, and Proteus tolC are the new BioBrick parts that we designed. We documented the characterization of each part in the Main Page section of the Registry entry for that Part/Device.
Submitted these new parts to the iGEM Parts Registry. This part was different from the part we documented in Bronze medal criterion #6.
iGEM projects involved important questions beyond the bench, for example relating to ethics, sustainability, social justice, safety, security, and intellectual property rights. We referred to these activities as Human Practices in iGEM. Demonstrate how your team has identified, investigated and addressed one or more of these issues in the context of your project.
Surveys of high school teachers, college students, and our summer campers.
Reached out and interviewed physicians regarding antibiotic resistance.
An important issue that the WLC-Milwaukee team has addressed in this year’s project are the ethical questions surrounding the use of genetically modified organisms (GMOs). Many people within our faith consider genetic manipulation to be within man’s grasp, yet perhaps we haven’t fully understood the implications of such abilities. In an attempt to better understand current perception of the ethicality of GMOs in our faith, we surveyed both teachers and students of Wisconsin Evangelical Lutheran high schools.
In order to educate both the faculty, and by extension the students of those in the Wisconsin Evangelical Lutheran synod, the WLC-Milwaukee team has put together a Biotechnology Curriculum. It poses topics for discussion with both teachers and faculty and asks how work such as genetic modification coincides with our faith. The curriculum also suggests that concepts such as genetic engineering and synthetic biology be explored to attain a better understanding of the scientific methods in light of our faith.
Gold  
In addition to the Bronze and Silver Medal requirements, our team also achieved these following goals: 
Help any registered iGEM team from a high-school, different track, another university, or institution in a significant way by, for example, mentoring a new team, characterizing a part, debugging a construct, modeling/simulating their system or helping validate a software/hardware solution to a synbio problem.
We collaborated with various teams, providing them feedback and contributing to their human practices project. Through our surveys, we also began to lay the ground work of starting an iGEM team from surrounding high schools in Milwaukee.
Expanded on our silver medal Human Practices activity by demonstrating how we have integrated the investigated issues into the design and executed into our project. We also demonstrated an innovative Human Practices activity that related to our project (this typically involves educational, public engagement, and/or public perception activities; see the Human Practices Hub for information and examples of innovative activities from previous teams).
Bioengineering Summer Camp
Biotechnology Information Night
Wisconsin Evangelical Wisconsin Synod Education Plan
WLC Biotechnology Statement
Our team investigation into the ethicality of genetic modification was integrated into our project in two ways. Over the summer the team hosted the 2015 WLC Bioengineering summer camp, a week long camp designed to educate high school students about Biotechnology. Throughout the week activities such as lab protocols and lectures from campus professors were the focus of those attending. In the afternoon students had time to work on a project with their groups which focused on creating to presentation of a past iGEM project to present to the rest of the students on campus. At the end of the week, both instructors and the students in attendance deemed the camp a success.
While the Bioengineering camp was aimed at high school students, the Wisconsin Lutheran College – Milwaukee team also wanted to create a means of education for students currently attending WLC. To meet this need this year’s team introduced Biotechnology Information Night. The evening consisted of presentations from a few of the iGEM seniors. A total of 17 students were in attendance. Topics such as iGEM, GMOs, and other applications of biotechnology were presented and the evening ended with discussion concerning the topics covered. Most importantly there was also discussion concerning the ethicality of genetic engineering and what it means to our faith. Many students participated in the latter part of this discussion and a few of the senior iGEM members had the opportunity to share their thoughts as well as the conclusions they had come to over their years of study and participating in iGEM. It was a great discussion and we look forward to hosting these again in the future!
Improve the function OR characterization of a previously existing BioBrick Part or Device (created by another team, or by your own team in in a previous year of iGEM)
Our 2014 iGEM team generated a biobrick K1406000 that contained an arabinose-inducible promoter and ribosomal binding site (RBS). While the sequence was correct, we were unable to demonstrate that the promoter and RBS were functional. This year’s team has taken that part and added tolC genes from several bacterial species with the expectation that the arabinose promoter will express these tolC genes.
Our results demonstrate that we were able to express TolC protein from each of the clones that were made (Figures 1 and 2). Western blot analysis was performed with TolC-specific antibodies that show that when cultures were grown containing these clones, TolC protein was abundantly produced when arabinose was present. When no arabinose was present there was very little expression of the tolC genes. This is what is expected of a gene that is controlled by an arabinose promoter. Therefore, the arabinose-inducible promoter and RBS that was created last year has been better characterized by showing that it does work to allow regulated expression of TolC and we have improved this part by adding tolC genes that are necessary for our project this year.
Demonstrate a functional prototype of your project. Your prototype can derive from a previous project (that was not demonstrated to work) by your team or by another team. Show this system working under real-world conditions that you simulate in the lab. (Remember, biological materials may not be taken outside the lab.)
Our project attempted to create a method to isolate bacteriophages specific for the TolC protein from several pathogenic bacterial strains. We would use clones of the tolC gene from these pathogenic strains and express them in a nonpathogenic E. coli strains and attempt to isolate these specific phages. This would allow us to find phages against these dangerous pathogens without having to handle the pathogenic bacteria.
While we were successful in generating the E. coli strains that contained the tolC genes from pathogenic bacteria, we were unable to actually isolate any bacteriophage specific for these genes. We were able to isolate several bacteriophages but they did not appear to have specificity for the TolC proteins. There are several reasons for why this may not have worked such as phage binding conditions, ability of phages from other pathogens to replicate in E. coli, and knowing for certain that there are phages that are specific for these pathogens in the sample we were testing. These are all avenues of research that we will pursue if we continue to work on this project.
RESULTS
Click the arrow to learn more about the results of our project.
Western Blots
In order to look for phages for specific organism’s TolC proteins, we had to first make sure the proteins were being produced. In all, we isolated the tolC genes for Salmonella, Vibrio cholera, Yersenia pestis, Proteus, Pseudomonas, Klebsiella, and E. coli K12. These were put under the control of the pBAD promoter and a strong E. coli RBS, and the E. coli tolC signaling sequence was put upstream the different organisms’ tolC genes.  Using the Western Blots, we were able to show that the proteins were being transcribed, and transcription levels were raised in the presence of the pBAD promoter’s inducer- arabinose (at 0.1% working solution). Note: basal levels of transcription in Luria Broth are sufficient for function (see Kirby-Bauer assays which were done without the inducer) and phage binding (see the example phage plate in Modeling > Experimental Verification, which was also done without any inducer).
Kirby-Bauer Assays
Pictured are Example Kirby-Bauer assays.  Note that the zones of inhibition are small for the WT E. coli (top left)
and much larger for ΔtolC E. coli (top right).
Some transgenic TolC proteins can restore efflux function to ΔtolC E. coli,like that
of Proteus (bottom left), but not all work- for an example see Pseudomonas's tolC (bottom right)
The Kirby-Bauer assays were the second part of our expression tests. While the western blots tested for the production of the proteins, the Kirby-Bauer assay is testing whether or not the expressed TolC is having an impact on sensitivity to two efflux-resisted antibiotics: Novobiocin and Erythromycin. The Kirby-Bauer assay is a simple test that involves plating what would become a lawn of bacteria (using Top Agar); on top of this filter disks containing an antibiotic are placed. The antibiotics diffuse from the disks and inhibit the growth of bacteria within a certain distance based on the concentration of antibiotic at that distance and the minimum inhibitory concentration (MIC) of the antibiotic against the specific bacteria plated.
In the Kirby-Bauer assays, wild type (WT) E. coli, ΔtolC E. coli, and a ΔtolC E. coli with the pUC57 plasmid containing the pBAD promoter and the strong RBS (everything but a gene) were used as controls. The WT E. coli had smaller zones of clearing, indicating reduced sensitivity to the antibiotics compared to the two other controls, as expected. The rest of the samples were ΔtolC E. coli expressing a transgenic tolC gene contained on a pUC57 plasmid behind the same pBAD promoter-strong RBS combination. A Kirby-bauer showing zones of clearing similar in size to the WT E. coli was  interpreted as TolC proteins being successfully assembled, inserted into the membrane, and successfully interfacing with other membrane proteins to form a functional trans-periplasmic efflux pump. Zones of clearing similar to the ΔtolC control were interpreted as a failure interface with other membrane proteins to form a fully functional trans-periplasmic efflux pump. It is impossible to say based on the Kirby-Bauer assays if these failures were due to failure to polymerize, insert into the membrane, or interacting with the other membrane proteins in the efflux pump.
Kirby-Bauer assays were analyzed by taking a picture of the plates from the same height (using a stand) and keeping a ruler in the image. The pictures were analyzed using the ImageJ software package. A conversion was set between pixels and centimeters using the “Set Scale” function. A circle was made to reflect the zone of clearing and its properties were exported as a .csv. Excel was used to convert from the circumference to the diameter of the zone of clearing, and each diameter was expressed as a percentage of the ΔtolC E. coli’s zone of clearing’s diameter.
Results of Kirby-Bauer assays done on ΔtolC E. coli with our transgenic molecules.
Green signifies results that indicate a restoration of efflux function in ΔtolC E. coli.
Phage Solution Enhancement
Our project aims to act as a chassis for collecting and isolating new bacteriophages specific to the TolC proteins of pathogenic strains of bacteria, using a safe lab-strain of E. coli. To be used in such a way, we modified a preexisting protocol for isolating bacteriophages from the environment. In doing so, we added three rounds of what we referred to as “phage solution enhancement.” The idea behind this is that the environmental phage source is incubated with a larger volume of overnight-grown ΔtolC E. coli for ~20 minutes; this should be enough time for any bacteriophages capable of infecting these knockout-tolC cells to have an opportunity to bind a host bacterium, but not enough time to replicate and release new phage particles into the environment. These incubations are then chilled and centrifuged. The pellet created at the bottom contains the ΔtolC E. coli and any non-TolC specific bacteriophages which bound/infected the ΔtolC E. coli. The supernatant, which contains any bacteriophages which did not bind, is moved into a new tube, treated with chloroform to kill any remaining bacteria, and moved into another new tube. This becomes the new phage solution and the process is repeated twice more. The protocols are included in the table below.
Examples of plaque-assay results. These pictured plates were the countable plates from the phage solution enhancement quantification experiment. For all these experiments a phage solution dilution was plated 100 µL with 250 µL overnight-grown WT E. coli, and incubated overnight at 37°C
Top Left: unenhanced non-TolC specific "Phage #4" solution diluted to 1E-9.
Top Right: enhanced non-TolC specific "Phage #4" solution diluted to 1E-9.
Bottom Left: unenhanced TolC specific TLS Phage solution diluted to 1E-9.
Bottom Right: enhanced non-TolC speific TLS Phage solution diluted to 1E-7
Bacteria-Specific Phage Isolation Screen
TolC Specific Bacteriophage Screen
Pipette 5 mL of your environmental sample into a 50 mL conical tube
Add 35 mL LB broth to conical tube
Add 1 ml overnight grown E. coli culture (or any bacteria of interest)
Incubate overnight shaking at 37°C
Centrifuge the 50 mL conical tube for 10 minutes at top speed
Pour supernatant into new conical tube (throw tube with pellet in biohazard bin)
Add 2 ml Chloroform to tube containing supernatant
Vortex vigorously for 1 minute
Centrifuge 50 mL conical tube for 10 minutes at top speed
After centrifugation chloroform will form a layer on the bottom and supernatant on the top
Remove 25 mL supernatant and put in new 50 mL conical tube
Attempt to filter-sterilize the supernatant into a new conical tube using a 45µm filter (often there is still too much debris in the solution so the filters become clogged very quickly.)
Perform plaque assay with this phage sample (see plaque assay protocol)
Pipette 5 mL of your environmental sample into a 50 mL conical tube
Add 10 mL culture of ΔtolC E. coli
Mix and incubate standing at 37°C for 20 minutes
Centrifuge tube for 10 minutes at 4500rpm and 4°C
Pour supernatant in new 50 mL conical tube (dispose of pellet in biohazard bin)
Add 2 ml Chloroform to supernatant
Vortex vigorously for 1 minute
Centrifuge for 10 minutes at top speed
Repeat steps 2-8 a total of 3 times
Remove 10 mL of this now phage solution into new conical tube (Store tube in refrigerator after use)
Pipette 1 mL of this phage solution into new 15 mL conical tube
Add 8 mL LB broth to conical tube
Add 1 mL wild-type E. coli culture or  ΔtolC E. coli expressing another type of TolC (add antibiotics to the mixture to a working concentration if needed).
Incubate mixture overnight shaking at 37°C
Next day, centrifuge the conical tube for 10 minutes at top speed
Pour the supernatant into a new conical tube
Add 1 mL chloroform to supernatant
Vortex vigorously for 1 minute
Centrifuge tube for 10 minutes at high speed
Transfer phage solution into new conical tube
Perform plaque assay with the same wild-type E. coli strain or ΔtolC E. coli strain with plasmid tolC used previously
After plaques have been isolated on plates, purify the phage from several plaques and test whether they use TolC as a receptor using a cross-streak of wild-type E. coli, the ΔtolC E. coli with a plasmid containing another tolC (if used), and the ΔtolC E. coli
In order to justify these steps, we attempt to quantify the effect of our process of phage solution enhancement. In order to test the protocol’s effectiveness, our experiment made use of two phages. One TolC-binding phage, TLS phage, and one non-TolC-binding phage we isolated ourselves called Phage #4 (we found that mipA, fhuA, fecA, and flgH knocokout E. colis, were insensitive to this phage). To begin the experiment we infected bacterial cultures (separately) in order to proliferate our phages; 1 mL of overnight grown WT E. coli (wild type) was mixed with 10 mL of LB and 250 µL of isolated phage solution. After incubating overnight at 37°C, these tubes were centrifuged. The supernatants were moved to new tubes; these were treated with chloroform, centrifuged, and the supernatants were again moved to new tubes (these are our starting phage solutions). From there four new tubes were labeled (TLS+, TLS-, 4+, and 4-). The “+” tubes became the “enhanced” tubes, and the “-“ tubes were left “unenhanced”; both “TLS” tubes were started with 5 mL of the TLS starting phage solution, and both “4” tubes were started with 5 mL of the Phage #4 starting phage solution. The “+” tubes were enhanced (see steps 2-9 under “TolC Specific Bacteriophage Screen” above) adding 30 mL of volume to the original 5mL of phage solution. To compensate for the gained volume, the “-“ tubes had 30 mL of sterile LB added.
Expecting high phage concentrations in some of our new phage solutions, we performed 1:10 serial dilutions (of which 100µL of the total 1000µL of each dilution would be used) on all of our new phage solutions down to a concentration of 10^-12. All these dilutions were mixed with 250 mL of overnight grown WT E. coli culture and 4 mL of molten LB top agar; the mixtures were immediately plated. After ~24 hours of incubation at 37°C, the plates were analyzed. Plates with a countable number of plaques (between 30 and 300) were chosen for each type of phage solution. These were used to calculate the number of plaque forming units per milliliter (PFU/mL). The “+” solutions’ PFU/mL’s were divided by those from the matching “-“ solution, to obtain a fractional change in phage concentration from the enhancement process. We expected a fractional change of approximately 1 from TLS+/TLS- and a number significantly less than 1 for Phage #4. Results are below. These results demonstrate the efficacy of the phage enhancement process.
TIMELINE
Want to explore a breakdown of our project week to week? Click on the arrow now!
April 19th - April 25th
General Project:
This week we assembled our team for the summer, and ended the week out with a BBQ to get to know each other!
April 26th - May 2nd
General Project:
A brainstorming meeting occurred and we had a great time formulating our lab work, wiki, and policy & practices parts of our projects. Committees will be important for us to accomplish everything.
May 10th – May 16th
Speakers for our summer camp were finalized and rooms reserved! XXX
May 17th – May 23rd
General Project:
We had a final meeting in our committees before breaking for summer. Projects were started and last details decided upon so we could make the most of our summer!
Laboratory Progress:
Prepared Amp, Arabinose, and LB plates to begin lab work. Began isolation of tolC specific phages using the enhancement method. First Mini Preps of Sal tolC on puc57, Vch tolC puc57 and J04450 on pSB1C3. Dilution plated the four types of enhanced phage solutions from earlier in the week. These phages were matched with their respective bacteria: WT, ΔtolC, Sal tolC in ΔtolC E. coli and Vch tolC in ΔtolC E coli.
May 24th - May 30th
General Project:
Bioengineering Summer Camp lead coordinators, Matt and Sierra met with MSOE to discuss what activities and models we would use for our students at the summer camp.
Laboratory Progress:
From last week’s dilution plates 6 plaques were picked, 3 from WT, 2 from Vch, and 1 from Sal and used for another dilution plating experiment. The WT phages were plated with WT E. coli, Vch in ΔtolC E. coli, and Sal tolC in ΔtolC E. coli. DH5 E. coli were grown for a potential transformation via heat shock. Restriction enzyme digest with last week’s mini preps (Sal tolC, Vch tolC, J04450). Gel electrophoresis of Vch tolC, Vch tolC cut plasmid, Sal tolC, and Sal tolC cut plasmid. The weeks earlier dilution plating was not dilute enough. Dilutions extended to 10E-7 for all 6 phages. Mini prep of J04450 on pSB1C3. Ran Gel of J04450 plasmid, Vch tolC puc57 plasmid, Sal tolC puc57 plasmid. Performed heat shock transformation of Vch tolC puc57 and Sal tolC puc57 in DH5 cells. Plated on Amp LB plates. Picked colonies from previous day’s transformation. Mini prep Vch tolC puc57 and Sal tolC puc57. Ran gel of Sal tolC puc57 and Vch tolC puc57.
May 31st – June 6th
Laboratory Progress:
Preparations were made for the first Western Blot. Kirby assay prepared; antibiotics used were Novobiocin, Erythromycin, and water. Bacteria used were WT, ΔtolC E. coli, pBAD+RBS in ΔtolC E. coli, Sal tolC, and Vch in ΔtolC. Six plaques from 6 different types of dilution plates made. Finalized the 6 picked phages: WT-1, 2, 3. Vch 1,2. Sal 1. Performed 6 sets of cross streaks. 96 well plate of knockout test or the other Werner method.
June 7th – June 13th
Laboratory Progress: Phage solution preparation. Prepared 3 batches of phage solution. Performed dilution with 3 phages from earlier in the week as well as the previous 6. We analyzed these with knockouts to determine the proteins necessary for binding.
June 14th – June 20th
Laboratory Progress: Attempted heat shock transformation of measurement parts. Redid 96 well knockout test. Attempted mini prep of transformed parts. Determined 3 were not successful.
June 21st – June 27th
General Project:
Matt and Sierra proposed a final wiki design which the team agreed upon. Onto coding!
Laboratory Progress: Began designing and implementing phage solution enhancement quantification experiment. Retransformed the Measurement parts successfully.
June 28th – July 4th
Laboratory Progress:
Attempted assembly / ligation of Measurement constructs; one success, two failures. Attempted isolation of phages from river samples.
July 5th – July 11th
General Project:
The student lecturers, Matt and Sierra, at our summer camp began formulating their presentations about viruses and antibiotic resistance! We are pumped to share our knowledge to help high schoolers understand antibiotics and use them appropriately in the “real world”.
Laboratory Progress:
Attempted variants of phage solution enhancement. Reattempted assembly of two previously unsuccessful Measurement parts. One out of two successful.
July 12th – July 18th
General Project:
This week was all about finding questions to ask our experts: Sierra focused on doctors, Christa focused on industry specialists, and Anna focused on foundations that all work with antibiotics, antibiotic resistance, or bacteriophages.
Laboratory Progress:
Made broth, top agar, and plates from minimal arabinose free media for expression tests. Made standard OD curves for later-abandoned, proposed modeling experiment.
July 19th – July 25th
General Project:
Preparations were made for our summer camp next week! Food, name tags, lab manuals, and binders are waiting for our campers and staff members! We are so excited to meet our students and work with them over the week!
Laboratory Progress:
Troubleshooted failed attempt to move Vch and Sal tolCs to iGEM plasmid. Determined it was because we left non bio brick compatible cut sites in the gene. Designed site directed mutagenesis primers to remedy this.
July 26th – August 1st
General Project:
Summer Camp week! Read about the blast that we had with our students here . Unfortunately, lab progressed fairly slow due to our staff dedicating a majority of their time to the summer camp, but no regrets, as our students had a wonderful time.
August 2nd – August 8th
General Project:
This was a major background research week. Sierra, Christa, and Matt focused on source gathering, journal reading, and article writing galore! Phage therapy and bacteriophages are fascinating to read about, find our sources around the rest of the wiki!
Laboratory Progress:
Received and began transforming more tolC genes such as E. coli K12, Yersinia Pestis, Proteus Mirabilis.
August 9th – August 15th
General Project:
Jordyn proposed, and Sierra finalized drafts of surveys for our WELS high school teachers and WLC students. Now onto sending them out and collecting responses and data.
Laboratory Progress:
Received, transformed, and began experimenting with more tolC genes such as Klebciella and Pseudomonas. We began the process of moving these to the BioBrick shipping plasmid. Attempted one last attempt at assembly of Measurement part; this failed.
August 16th – August 22nd
Laboratory Progress:
Attempted to assemble BioBricks.
August 23rd – August 29th
We welcomed new freshman on our campus this week! Sierra and Harrison had a table at "Org Smorg" to recruit for next year's iGEM team as well as survey the freshmen about their knowledge and opinions regarding GMOs. Jack made sure we looked great with a new poster board made up for the event. Matt, Sierra, and Dr. Werner also decided on the program we would pursue.
August 30th – September 5th
We were back on campus this week! It was great to see the entire team again. We had a team meeting on September 1st and got caught up with our tasks ahead of us. We prepared for Biotechnology Info Night, designing the coffee lids and deciding the content of our session. We also prepared for having lunch tables to gather survey data.
September 6th – September 12th
General Project:
This week was a big week for Policy & Practices. We had lunch tables at our school for students to have conversations with us regarding biotech and take our survey. We also encouraged them to come to Biotechnology Info Night.
Laboratory Progress:
Further attempt at modeling experimental verification; attempts failed. Successfully moved Yersina Pestis, Proteus, and Pseudomonas to the pSB1C3 plasmid.
September 13th – September 19th
This week, unfortunately, was a large wiki building week. We focused a lot as a team on uploading content and finalizing all of our Policy & Practices data. Zach, Anna, and Jack analyzed surveys while Sierra coordinated who was responsible for what content. Christa focused on wiki proof reading, and Ryan contributed to graphic design. Jordyn and Harrison ensured that all of our requirements were fulfilled and documented. Matt, the wiki master coder, made sure that we had pages to upload content to. We also prepped our BioBricks and shipped them!.
