<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_WLC-Milwaukee_Project skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:WLC-Milwaukee/Project</SPAN></H1><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><DIV id="wlcHeader" class="navLinks" style="z-index:5;top:14px;left:0px;position:fixed;height:90px;width:100%;"><TABLE width="100%" align="center" cellspacing="0"><TBODY><TR height="90px"><TD id="HomeBox"><A id="homeLink" href="https://2015.igem.org/Team:WLC-Milwaukee" style="text-decoration:none">Home</A></TD><TD id="ProjectBox"><A id="projectLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Project" style="text-decoration:none">Project</A></TD><TD id="PartsBox"><A id="partsLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Parts" style="text-decoration:none">Parts</A></TD><TD id="NotebookBox"><A id="notebookLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Notebook" style="text-decoration:none">Documentation</A></TD><TD id="ModelingBox"><A id="modelingLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Modeling" style="text-decoration:none">Modeling</A></TD><TD id="practBox"><A id="practLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Practices">Policy</A></TD><TD id="teamBox"><A id="teamLink" href="https://2015.igem.org/Team:WLC-Milwaukee/Attributions">People</A></TD></TR></TBODY></TABLE></DIV><DIV id="container"><DIV id="element"><P>Project</P></DIV></DIV><DIV style="padding-left:10%;padding-right:10%;font-size:11pt"><CENTER><H1 id="heading_1"><FONT color="#610B0B"><B>PROBLEM</B></FONT></H1></CENTER><P><B>Problem:</B> Bacteria are becoming increasingly resistant to antibiotics at an alarming rate.


</P><LABEL for="hider1"><H1 id="heading_1"><FONT size="4" color="#610B0B"><B>Disease Causing Bacteria</B></FONT></H1></LABEL><P> 
This is an overview of several gram-negative bacteria that cause diseases in humans. Included are their mechanisms of infection and their virulence factors. 
</P><P>
Click the arrow above for more information.

</P><DIV id="content_area_1"><FONT size="4" color="#610B0B"><B><I>Pseudomonas aeruginosa</I> {BioBrick BBa_K1683004}</B></FONT><B>Quick Facts:</B>
Gram (-)
Aerobic
Extracellular infection<B>Types of infections that it causes:</B><UL><LI> Infecting burned tissue and causing lung infections in people with cystic fibrosis. </LI><FONT size="2">In burned tissue, the wound seeps plasma, rich in nutrients. <I>P. aeruginosa</I> then takes the opportunity of this nutrient rich “media”. We must fight these infections quickly, because this strain is notorious for its antibiotic resistance. Once it spreads to the bloodstream, it becomes even more challenging to fight, and this is what can cause death in patients. 
	In cystic fibrosis, infection is slower. Infection occurs in 80% of cystic fibrosis patients within the first few years of their life. [bacterial pathogenesis]. Once it exists in the lungs, antibiotics can help slow the infection, but eventually bacteria develop physiological traits that increase resistance, such as the alginate capsule.</FONT><LI>Skin infections</LI><LI>Bloodstream infections</LI><LI>Urinary tract infections</LI><LI>Wound infections</LI><LI>Nosocomial (hospital acquired)</LI></UL>
Infections are rare even though it has a widespread distribution. It inhabits wet surfaces, especially hospital surfaces, which contributes to its nosocomial traits. <I>P. aeruginosa</I> is originally a soil microorganism. <B>Virulence Factors:</B><I>P. aeruginosa</I> has many virulence factors. They are categorized into physical characteristics, such as nonpilus adhesions and flagella. There are also enzymes that function as toxins. These virulence factors work to damage the surface and membranes of eukaryotic membranes. Pyocyanin is a cytotoxin that interferes with the a normal redox reaction in host cells, creating reactive oxygen. This damages our cells and causes tissue necrosis. In addition, it produces rhamnolipids, which form a barrier around the cell known as “shielding”. This barrier protects the cell against leukocytes, further barring our immune system from fighting the infection. The most important virulence factors of this opportunistic pathogen are the type three secretion system and the effector proteins/toxins it injects into cells, ExoS, ExoT, ExoU, and ExoY.  These protein toxins heavily modify the cytoskeleton of the cell through modifications to the small GTPases Rho, Rac, and CDC42 (ExoS, ExoT), and increasing cellular adenylate cyclase levels, leading to cell necrosis, cytoskeletal rearrangements, and cell death through apoptosis.  <I>P. aeruginosa</I><B>Reference:</B>
[1] Wilson. (2011). <I>Bacterial Pathogenesis</I> A Molecular Approach.

<FONT size="4" color="#610B0B"><B><I>Yersinia pestis </I> {BioBrick BBa_K1683000}</B></FONT><B>Quick Facts:</B>
Gram (-)
Nonmotile
Anaerobic
Obligate Parasite<B>Course of Infection</B><I>Yersinia pestis</I> is the cause of what is colloquially known as the Plague. Primarily found in rodents, which contract the bacteria from their frequent contact with fleas, humans play no role in the long term survival of Y. pestis. (Perry 51) Transmission of the bacteria occur when fleas feed on the blood of rodents, some of which is regurgitated into the mammal. The spread then continues through predators and other mammals that either come into contact with infected rodents or directly ingest the infected rodent. Today, domesticated cats are the primary cause of most <I>Y. pestis</I> infections, prompting concerns over veterinarian health and creating a resurgence of attention for the bacterium.
<B>Infection Types</B>
There are three primary forms that human plague cases will occur in, these being bubonic, septicemic, and pneumonic. “Over the past 24 years, the majority of patients in the United States have had either bubonic or septicemic plague.” (Perry 57) Pneumonic plague is far more rare and deadly. It is spread by respiratory droplets and its incubation period can be as little as one to three days. Symptoms of the bubonic form include painful and swollen lymph nodes and fever. The symptoms of the other two forms are hard to detect. Septicemic plague symptoms resemble that of other gram-negative septicemias (Perry 58) and pneumonic plague can be misdiagnosed as other pulmonary infections since chills, headaches, and gastrointestinal disturbances are common symptoms.<B>Treatment</B>
All three forms are treatable with antibiotics such as Streptomycin, Gentamicin, Oxytetracycline, Tetracycline, Chloramphenicol, and Doxycyline. Streptomycin is the antibiotic of choice for most treatments but due to its high toxicity its use is usually stopped after five to ten days and switched with less toxic antibiotics. Upon diagnosis, immediate isolation is required for at least two days to allow antibiotic treatment to take effect.<B>Virulence Factors</B><UL><LI>Some of the most common virulence factors of <I>Y. pestis</I> include serum resistance, the Fraction 1 capsule, the Ysc type three secretion system, and the secreted toxin/effector suite of proteins called YOPs.</LI><LI>Serum resistance is the ability of <I>Y. pestis</I> to resist complement-mediated lysis since it must be able to survive (and/or grow) in blood; the primary mean of infection. An integrated pMT1 gene allows for complete resistance to complement lysis. <I>Y. pestis</I> is resistant even in the presence of antibodies and is chromosomally encoded. The rough or short lipopolysaccharide of <I>Y. pestis</I> appears to mediate this resistance.</LI><LI>Fraction 1 is the name given to a potent antigen of the human blood stream. Unfortunately, <I>Y. pestis</I> possesses the ability to form a large gel like capsule around itself to prevent the antigen from binding to it. This form of resistance also coincides with <I>Y. pestis’</I> ability to resist phagocytosis.</LI><LI> The Ysc type three secretion, evolutionarily based off modifications from the flagular membrane proteins, injects unfolded Yop factors into cells attempting phagocytosis of the bacteria.  These factors then fold into active configurations and specifically interact with multiple targets throughout the cell.  Some of the known actions of the Yops are to disrupt or alter signalling pathways, disrupt phagocytosis, and cause apoptosis of the cell.</LI></UL><B>Reference:</B>
Robert D. Perry, J. D. (1997). <I>Yersinia pestis</I>-Etiologic Agent of Plague. Clinical Microbiology Reviews, 35-59. Retrieved 2015, from http://www.ncbi.nlm.nih.gov/pmc/articles/PMC172914/pdf/100035.pdf


<FONT size="4" color="#610B0B"><B><I>Proteus mirabilis</I> {BioBrick BBa_K1683002}</B></FONT><P><B>Quick Facts: </B>
Gram (-)
Swarming Motility
Facultative Anaerobe</P><P><B>Course of Infection</B></P><P>Humans are the most common host of <I>Proteus mirabilis</I>. This bacterium generally colonizes the gastrointestinal tract, the kidneys, and the bladder. Infections will most commonly occur if there is a loss of functionality in the urinary tract. [1] When this happens the incredibly motile <I>P. mirabilis</I> will be able move up the urethra to the bladder and eventually to the kidneys. Once in the bladder it can produce fimbriae which will bind to the surfaces of the urinary tract. </P><P>At this stage in the course of infection the bacterium is generally harmless unless kidney stones begin to form. This occurs as mineral buildups form in the vicinity of the bacterial colonies. Further complications occur if the kidneys become infected, which can lead to septicemia. [2] Once this happens the resulting blood infection can be fatal and it is for this reason that <I>P. mirabilis</I> is one of the most widely studied bacterium that cause urinary tract infections. </P><P><B>Infection Types:</B></P><P><I>P. mirabilis</I> is the primary cause of urinary tract infections in mammals, especially those with preexisting functional problems in the urinary tract. [3] This strain causes 90% of urinary tract infections out of the various <I>Proteus</I>species. The initial urinary tract infections are rarely a cause for concern as they can be treated relatively easily with antibiotics. </P><P><B>Treatment</B></P><P><I>P. mirabilis</I> can be treated with a wide variety of antibiotics including quinolone, trimethoprim or sulfamethoxazole, ceftriaxone and gentamicin. [4] If the infection is relatively minor, the antibiotics can be prescribed for 10-14 days to destroy the pathogens. In more severe cases of infection, antibiotics will be prescribed for as many as 21 days. Complications can ensue if kidney stones begin to form. As the stones form they will be become heavily contaminated with the bacteria. When the minerals forming the stones coat the bacteria, antibiotics will become less and less effective against bacteria. [3] This can lead to a resurgence in the bacteria. </P><P><B>Virulence Factors</B></P><P><I>P. mirabilis</I> possesses a large variety of virulence factors, making it difficult to prevent its spread once an infection has started.  Known virulence factors include urease, fimbriae, and flagella. </P><P>Urease is an enzyme responsible for hydrolyzing urea into ammonia and carbonic acid. [4] The buildup of this ammonia is fatal for the cell. When this urea builds up it can combine with minerals such as magnesium and calcium, forming kidney stones. The buildup of the ammonia and carbonic acid act as a buffer from the lytic acid of phagolysosomes, further prolonging the length of time that <I>P. mirabilis</I> can survive in the urinary tract. </P><P>The fimbriae of <I>P. mirabilis</I> are what allow the bacteria to attach to the walls of the urinary tract, making it difficult for urine to flush out bacteria. [1] Once these bacteria have attached here, they can form flagella that allow them to move further up the UT towards the bladder and kidney. </P><P><B>References:</B></P><P>[1] Burall, L. S. (2004). <I>Proteus mirabilis Genes That Contribute to Pathogenesis of Urinary Tract Infection</I>. Retrieved September 2015, from US National Library of Medicine National Institutes of Health: http://www.ncbi.nlm.nih.gov/pmc/articles/PMC387873/</P><P>[2] Gonzalez, G. (2014, October 30). <I>Proteus Infections Treatment &amp; Management</I>. Retrieved September 2015, from Medscape: http://emedicine.medscape.com/article/226434-treatment</P><P>[3] Gould, S. (2014, July 20). <I>Urease: an anti-microbial target in bacteria and fungi</I>. Retrieved September 2015, from Scientific American: http://blogs.scientificamerican.com/lab-rat/urease-an-anti-microbial-target-in-bacteria-and-fungi/</P><P>[4] <I>Proteus Mirabilis</I>. (n.d.). Retrieved September 2015, from Encyclopedia of Life:
http://eol.org/pages/972768/details</P><FONT size="4" color="#610B0B"><B><I>Vibrio cholerae</I></B></FONT><B>Quick facts:</B>
Gram (-)
Facultative anaerobic 
Extracellular <B>Type of infection that it causes:</B><I>Vibrio cholerae</I> causes the diarrheal disease, cholera, which has a mortality rate of 100,000 deaths a year.  This bacterium resides in aquatic areas and gets transmitted to its human host via food or water that it has contaminated.  Once consumed the bacteria make their way to the epithelium of the small intestine (SI) to start colonization.   
<B>Virulence factors:</B> <I>V. cholerae</I> have several virulence factors that allow them to survive the process of getting to the SI epithelium.  The first obstacle is the low pH of the stomach.  Unless there are 10^11 free-living <I>V. cholerae</I>  cells (which is perhaps unrealistic) entering the host, not enough will survive in order to infect.  Alternatively, the most successful <I>V. cholerae</I> enter as microcolonies or as part of a biofilm.  Bacteria that are part of a biofilm produce an exopolysaccharide matrix that will protect them from the low pH.  Once the <I>V. cholerae</I> have made it to the SI they need to travel through the mucus layer covering the epithelium.  Therefore motility is an important trait for <I>V. cholerae</I>.  Transportation of the bacteria through the mucus layer is facilitated by the expression of hapA.  HapA encodes for haemagglutinin/protease (Hap), a mucinase, which is produced by <I>V. cholerae</I>.  After making their way through the mucus layer, <I>V. cholerae</I> will attach to the epithelium.  Initially, the bacteria produce nonspecific adhesins that allow themselves to attach temporarily to the epithelium while determining the suitability of the substrate.  Some of the important adhesions are Mam7 and GbpA.  Mam7 creates protein-protein and protein-lipid interactions, while GbpA binds to GlcNac molecules that are attached to epithelial cells.  Once <I>V. cholerae</I> are attached to the SI epithelium they produce cholera toxin (CT) an adenylate cyclase ribosylating toxin attached to a pentameric receptor-binding subunit.  Once inside intestinal cells, the toxin subunit causes large increases in adenylate cyclase, which causes chloride ion pumps to increase the extracellular concentration of chloride ions, resulting in a huge out-rush of water into the intestinal lumen.  This causes watery diarrhea that leads to dehydration and death in the host.  
<B>Reference:</B> 
Almagro-Moreno S, Pruss K, Taylor RK (2015) Intestinal Colonization Dynamics of Vibrio cholerae.  <FONT size="4" color="#610B0B"><B><I>Salmonella</I></B></FONT><B>Quick facts:</B>
Gram (-)
Facultative anaerobic 
Intracellular infection<B>Type of infection that it causes:</B>
Salmonella resides in the intestines of humans and animals as a pathogen.  They are also present in the feces of infected animals or food that they come in contact with. The bacteria can enter a host orally during ingestion of contaminated food or water and cause a disease called salmonellosis.   This results in the inflammation of the small and large intestines, which is manifested in symptoms of watery diarrhea, abdominal pain, and a fever.  Salmonellosis will normally subside in four to seven days on its own, but antibiotics can be taken to help destroy the bacteria in the digestive tract. 
 
Once ingested the Salmonella colonize the intestinal epithelium.  It enters enterocytes, M cells, and dendritic cells.  If the Salmonella reach the submucosa then they can internalize in macrophages, which become their hosts.  Salmonella can live in many different types of host cells and this improves its survival.  The bacteria invade cells using the type III secretion system Spi1. This is a highly specific process that relies on the expression of several bacterial factors.  Effector proteins rearrange the actin cytoskeleton causing ruffles in the cell membrane allowing the Salmonella to be internalized.  In addition, adhesins on the surface of Salmonella assist in attachment and internalization.  Once internalized, the Salmonella replicate in the modified phagosome utilizing a second type three secretion system, Spi2.   <B>Virulence Factors: </B><UL><LI>Type I secretion systems BapBCD and SiiCDF secrete the surface proteins BapA and SiiE, which assist in adhesion and invasion of the host cell. </LI><LI>Flagella give the Salmonella a motile function, which can assist in infection. </LI><LI>Flagellin causes inflammation of the intestinal epithelium while preventing cell apoptosis.  </LI><LI>The environment of the host cell contains insufficient resources for the survival of the pathogen, so Salmonella utilizes several ion transporters to acquire the necessary resources. </LI><LI>Host cell proteins bind most of the available iron, so Salmonella produce siderophores, enterobactin and salmochelin, to ensure that the pathogen receives the required amount of the mineral.   </LI><LI>CorA, MgtA, and MgtB are Mg2+ uptake systems. </LI><LI>Zn2+ is collected by the ZnuABC high affinity Zn2+ uptake system. </LI><LI>K+ is gained by the Trk system. </LI><LI>Salmonella uses SodCI, a superoxide dismutase in order to combat reactive oxygen species in the host cell that would otherwise kill the intracellular pathogen.</LI></UL><B>References:</B>  
http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774479/ http://www.healthhype.com/salmonella-and-shigella-differences-causes-disease-symptoms.html 
http://www.cdc.gov/salmonella/general/technical.html
</DIV><LABEL for="hider2"><H1 id="heading_2"><FONT size="4" color="#610B0B"><B>Antibiotic Resistance </B></FONT></H1></LABEL><P>
How antibiotics target bacteria and how the bacteria resist these mechanisms.
</P><P>
Click the arrow above for more information.

</P><DIV id="content_area_2"><B>Antibiotic definition:</B><P>	Antibiotics literally mean “agents against life.” Thus they are compounds which inhibit the growth of (bacteriostatic) or kill bacteria (bactericidal) under appropriate concentrations. Some are produced naturally by various organisms while others are produced synthetically in the laboratory. Ideally an antibiotic causes minor, if any, side effects to humans and animals, has a broad spectrum of activity which will be effective against many kinds of bacteria, and bioavailability to effectively reach and act in the area of bacterial infection.[1]</P><B>Cell membrane differences:</B><P>	Bacteria are commonly classified according to the structure of their cell membranes: gram-positive, gram-negative, acid-fast, and mycoplasma. Most bacteria fall into either the gram-positive or gram-negative category.[2]</P><P>	For a gram-positive bacterium, the interior of the cell is surrounded by a typical phospholipid bilayer. Outside this cell membrane is a thick layer of peptidoglycan, or murein, which is a polymer of sugars and amino acids. This murein covering is responsible for giving structure and shape to the bacterium, gives resilience to osmotic pressures, and provides an effective defense mechanism against antibiotics or other harmful chemicals. Techoic acids are other polymers which adhere to the outside surface of the murein layer of gram-positive bacteria. Some bacteria also have an outer capsule encompassing the whole cell.[2]</P><P>	The structure of the gram-negative cell membrane is more complex. Rather than simply constructing the thick murein layer typical of gram-positive bacteria, gram-negative bacteria utilize a thin layer of murein surrounded by another outer membrane. The interior of the cell is surrounded by the typical phospholipid, inner membrane. A relatively thin layer of murein lies outside the inner membrane and a unique outer membrane envelopes the murein. The resulting space between the two membranes is called the periplasm, or periplasmic space. The fluid of the periplasm contains a variety of proteins. Many of these proteins are binding proteins or enzymes for metabolism. Some gram negative bacteria also have β–lactamases to inhibit antibiotics in the periplasmic fluid. The outer membrane is bilayered, but is distinctive for the lipopolysaccharide (LPS) molecules found in the outermost layer. The cell membranes of gram-negative bacteria must also have porins, or channels, to allow the entry of small (700 daltons or less) essential hydrophilic compounds while larger ones require specialized transport mechanisms. Thus gram negative cells have a defense system of degradative enzymes in the periplasm and porins and specialized transport mechanisms exclusively for essential hydrophilic molecules.[2] </P><DIV style="float:right;margin:10px 10px;">
http://www.nature.com/nrd/journal/v2/n8/images/nrd1153-f1.jpg</DIV><P><B>Antibiotic mechanisms:</B></P><P>	The majority of utilized antibiotics target one of five vital processes in bacterial growth: cell wall synthesis, protein synthesis, DNA synthesis, RNA synthesis, or tetrahydrofolate synthesis.[1,3] </P><P>	Cell wall synthesis inhibitors include β-lactam antibiotics, glycopeptides, and others. The β-lactam antibiotics are so named for the four-member ring structures that they have in common. Examples of β-lactam antibiotics include the penicillins, cephalosporins, carbapenems, and monobactams. These β-lactam antibiotics function to inhibit cell wall synthesis by preventing the transpeptidation reaction responsible for linking the peptidoglycan, or murein, side chains together. β-lactams may also covalently bond with the transpeptidase enzyme which similarly inhibits cell wall synthesis and triggers autolysins which normally degrade peptidoglycan during bacterial growth and division. The glycopeptides include vancomycin, daptomycin, and teichoplanin which are especially effective against gram-positive bacteria. This is because they inhibit the final steps of peptidoglycan synthesis by binding to the D-Ala-D-Ala muropeptides thereby preventing the essential cross-linking of the peptidoglycan. Other antibiotics like fosfomycin and bacitracin inhibit cell wall synthesis at more initial stages, specifically the conversion of N-acetylglucosamine to N-acetylmuramic acid. Without intact peptidoglycan, the bacterium is more likely to lyse from osmotic pressure with its environment.[1]  </P><P>	Protein synthesis inhibiting antibiotics include aminoglycosides, tetracyclines, macrolides, and lincosamides. These antibiotics exclusively interrupt bacterial translation by taking advantage of the structural differences between prokaryotic and eukaryotic ribosomes. Aminoglycosides such as kanamycin and gentamicin are trisaccharides with amino groups. They bind to sites of the 30S subunit of the prokaryotic ribosome thus preventing its union with the 50S subunit to create a fully functional ribosome. Tetracyclines similarly bind to the 30S subunit of the ribosome, but they distort the A site such that aminoacylated tRNA cannot interact properly with the mRNA transcript. The macrolides, such as erythromycin, impede translation by binding to the 50S ribosomal subunit such that the E site is blocked. The lincosamides which include lincomycin and clindamycin function similarly to the macrolides but have different structures. Inhibition of protein synthesis tends to be bactericidal.[1]  </P><P>	DNA synthesis inhibitors include the quinolones such as nalidixic acid and the fluoroquinolones such as norfloxacin and ciprofloxacin. These antibiotics interfere with DNA gyrase which is essential for relaxing the positive supercoils that results during DNA replication, recombination, and transcription. Without functioning DNA gyrase, DNA becomes irreparably broken and dysfunctional. Another antibiotic, metronidazole, requires activation by flavodoxin but can then causes damage directly by nicking DNA. Intact DNA is essential for continued growth and life of an organism.[1,3]</P><P>	   Rifampin exemplifies antibiotics that inhibit RNA synthesis. It inhibits transcription by binding to the β subunit of RNA polymerase. It is analogous to macrolides in that rifampin sterically blocks the exit of the RNA polymerase as macrolides block the exit site of the ribosome during translation. RNA is the essential intermediate in the central dogma of molecular biology; without it proteins cannot be synthesized and the organism will die.[1]</P><P>	The antibiotics trimethoprim and sulfonamides inhibit the tetrahydrofolate biosynthesis pathway in bacteria which is necessary for the synthesis of amino acids including fMet. These antibiotics inhibit the enzymes required to produce tetrahydrofolic acid which is necessary for the reactions that result in the formation of nucleic acids and fMet. Mammals do not synthesize tetrahydrofolic acid which decreases the risk of unwanted side effects in their application. These antibiotics are generally only bacteriostatic because synthesis of new amino acids is not absolutely necessary.[1]</P><P>	This is not an exhaustive list, but merely an overview of the main groups of antibiotics. There is a need for new antibiotics, both discovered in nature and synthesized in the lab, to effectively combat infectious bacteria which continue to evolve mechanisms of defense.
 </P><DIV style="float:right;margin:10px 10px;">
https://upload.wikimedia.org/wikipedia/commons/0/08/Antibiotics_Mechanisms_of_action.png</DIV><P><B>Modes of resistance:</B></P><P>	Bacteria have evolved methods to be resistant to various antibiotics. Resistance differs from tolerance in that it causes the bacterium to stop growing while the concentrations of antibiotic are high, it does not require mutations or additional genes, and it is reversible. A resistant bacterium will grow even in the presence of antibiotic, gained resistance from mutation or from a resistance gene, and is not reversible.[1]</P><P>	Mechanisms of antibiotic resistance tend to fall into one of four general areas: preventing the antibiotic from reaching its target (often by efflux), chemically modifying the antibiotic so it becomes harmless, modifying the antibiotic’s target so it is unaffected by the antibiotic, or preventing the antibiotic from becoming activated.[1]</P><P>	There are several ways that bacteria prevent an antibiotic from reaching its target and taking effect. Merely the structure of the cell membrane, particularly in gram-negative cells forms the first line of defense. Antibiotics must pass through the outer membrane and the inner membrane to reach the cytoplasm. There are outer membrane porins which allow for transport of some molecules, but they selectively permit certain molecules based on size. Large, bulky molecules are unable to enter the cell through these porins. However mutations in the genes for the porin proteins can act either to broaden or narrow the opening of these porins thus changing the selection of which molecules can enter the cell. Antibiotics may also be prevented from crossing the cytoplasmic membrane. A more common resistance mechanism is the active efflux pump. Efflux pumps are ubiquitous in bacteria and work to prevent the accumulation of an antibiotic in the cell so that it never reaches an inhibitory concentration. These pumps may be specific or compatible with many substrates and are found in both gram-positive and gram-negative bacteria. Efflux pumps have been reported for every group of antibiotic, causing a significant challenge in combating bacterial infection.[1] </P><P>	Bacteria may also enzymatically inactivate an antibiotic, rendering it harmless. Gram-negative bacteria commonly have β-lactamases in the periplasm which hydrolyze the antibiotic to cleave C-N bond in the ring structure. Sensitivity to β-lactams antibiotics has been partially regained by combining the β-lactam antibiotic with a β-lactamase inhibitor such as clavulanic acid or sulbactam. Unfortunately some bacteria have already become resistant to this method as well. An additional technique bacteria use to combat antibiotics is to modify them with the addition of a chemical group. Phosphorylation, adenylation, and acetylation of a hydroxyl or amino group may be sufficient to interfere with the activity of the antibiotic. This is the primary form of aminoglycoside resistance which would otherwise act to prevent protein synthesis. Similarly acetylation of chloramphenicol and streptogramin render these antibiotics ineffective. Tetracyclines can also be inactivated by an oxidation reaction.[1] </P><P>	Another strategy to resist antibiotics is the altering or defending the target of the antibiotic usually by mutations that change the antibiotic’s target or by chemically altering the target with the addition of chemical groups. β-lactams are also sensitive to this form of resistance when mutations cause the binding specificity of the penicillin binding proteins to change. Even resistance to vancomycin which targets the D-Ala-D-Ala dipeptides at the ends of the muropeptides of peptidoglycan has emerged. The resistance takes the form of changing the D-Ala-D-Ala substrate to D-Ala-D-lactate which still allows the essential crosslinking of the peptidoglycan while remaining unaffected by the antibiotic. Vancomycin-resistant Enterococcus (VRE) has three different enzyme mutations to accomplish this. The first enzyme is a lactate dehydrogenase to synthesize D-lactate from pyruvate. The second synthesizes D-Ala-D-lactate from the components D-Ala and D-lactate. The third selectively cleaves D-Ala-D-Ala leaving only the D-Ala-D-lactate which is not susceptible to antibiotic action. Macrolides, streptogramins, and lincosamides can be inhibited by methylation of the A2058 adenine in the 23S rRNA. This modification prevents the antibiotic from fully binding to the ribosomal E site such that protein synthesis proceeds. Mutations in either the A or B subunit of DNA gyrase offers resistance to quinolones. Mutations in the β-subunit of RNA polymerase can offer resistance to rifampin by reducing the binding efficiency of the antibiotic to the exit of RNA polymerase such that transcription proceeds normally. Similarly a mutation in the 30S subunit of the bacterial ribosome confers resistance to streptomycin. Mutations in the enzymes which are active in tetrahydrofolate synthesis also occur and offer resistance to trimethoprim and sulfonamides. Bacteria have evolved to render antibiotics ineffective by changing the target of antibiotic activity, either the primary structure of the protein or by the addition of chemical groups.[1]</P><P>	 Finally, certain antibiotics require activation and resistance can be conferred by preventing this from occurring. This is true of metronidazole and isoniazid used to treat ulcers and tuberculosis respectively.[1]
	Given enough time, bacteria evolve mechanisms of resistance to practically any antibiotic.</P><DIV style="float:right;margin:10px 10px;">
http://groundupstrength.wdfiles.com/local--files/health%3Aantibiotic-resistance-questions-and-answers/antibiotic-resistance.jpg</DIV><B>Transmission of antibiotic resistance:</B><P>	Bacteria utilize a variety of methods to propagate antibiotic resistance. Genetic transfer of resistance genes may take the form of transposons, integrons, and conjugative transposons. Transposons are pieces of DNA flanked by insertion sequences coding for the enzyme transposase that can insert themselves into the genome or plasmid of a cell. These can potentially carry, and thus spread a resistance gene. Integrons are transposons that also have an integrase gene and an attachment site. This allows circular pieces of DNA, like plasmids, to be integrated into the genome. Conjugative transposons (CTn) differ in that they transmit themselves from the genome of one cell to the genome of a second cell. The CTn excises itself, circularizes, is passed by conjugation to another cell, and integrates itself into the genome.[1]</P><P>While random mutations can and sometimes do confer resistance to an antibiotic, transmission of resistance genes already present in the population from natural selection and genetic drift is far more time and energy efficient for a bacterium. Plasmids are one mode of transmission of antibiotic resistance genes. When two cells conjugate, one which has a plasmid with an antibiotic resistance gene with a second bacteria that lacks it, the second cell gains a copy of that plasmid such that both bacteria are antibiotic resistant. It is also possible for plasmids to have two or more resistance genes. Selection for one resistance gene simultaneously selects for all the others on the same plasmid. Thus while the bacterium may not currently need these additional resistance genes, they persist in the cell and the population. Horizontal gene transfer of plasmids via conjugation which occurs even between bacteria of different species is a real threat. Combined with selective pressure from an antibiotic and short generation times, it does not take long for a high percentage of the population to become resistant to any given antibiotic.[1]</P><P><B>Bibliography:</B></P>
[1] Wilson, Brenda A.; Salyers, Abigail A.; Whitt, Dixie D.; Winkler, Malcolm E.;  <I>Bacterial Pathogenesis A Molecular Approach</I>. Ed. 3. ASM Press: Washington D.C., 2011. p321, 330-340, 350-367.
[2] Schaechter, Moselio; Ingraham, John L.; Neidhardt, Frederick C.; <I>Microbe.</I> ASM Press: Washington, D.C., 2006. p23-27, 398-99, 453-454.
[3] Kohanski, Michael A.; Dwyer, Daniel J.; Collins, James J.; &quot;How antibiotics kill bacteria: from targets to networks.&quot; <I>Nat Rev Microbiol</I>. 2010 Jun; 8(6): 423–435.</DIV><CENTER><H1 id="heading_1"><FONT color="#610B0B"><B>OUR SOLUTION</B></FONT></H1></CENTER><P><B>Our Solution:</B> Use phage therapy as an alternative to antibiotics  
</P><LABEL for="hider3"><H1 id="heading_3"><FONT size="4" color="#610B0B"><B>Phage Therapy in the Past </B></FONT></H1></LABEL><P>
The beginnings of phage therapy.

</P><P>
Click the arrow above for more information.

</P><DIV id="content_area_3"><P><B>Phage Therapy: History, Research, &amp; Cultural Implications</B>
Lakaysha Blacksher
Human Practices, iGEM 2015</P><P><B>Introduction</B></P><P>Bacteriophage (“phage”) therapy, as a study, was initially developed in 1916 but then became nearly obsolete in the West until the 1940's and again after the fall of the Soviet Union. In the East, phage therapy had been and is still currently used in Georgia and Poland among other countries. Their success has thus rekindled interest and research in the West. This therapy is increasingly cited as the best alternative to antibiotics, the overuse of which has led to increased bacterial resistance. </P><P><B>Purpose and Function </B></P><P>The purpose of phage therapy, according to the Phage Therapy Center in Tbilisi, Georgia, is to provide “an effective treatment solution for patients who have bacterial infections that do not respond to conventional antibiotic therapies”[1]. Phage therapy has been used to cure acne, some symptoms of cystic fibrosis (CF), urinary tract infections (UTIs), and countless other conditions. Bacteriophages function by infecting a bacterium, replicating inside of the bacterium, and causing the bacterium to lyse (bursting the cell membrane, spilling the cytoplasm, and killing the bacterium).  </P><P><B>History </B></P><P>The first use of phage therapy is relatively unknown, given that several scientists have made the claim of discovering its usefulness. In 1896, Ernest Hankin, a British scientist, reported the presence of “marked antibacterial activity (against <I>Vibrio cholerae</I>), which he observed in the waters of the Ganges and Jumna rivers in India...”[2]. In 1898, Russian bacteriologist Nikolay Gamaleya observed a similar function with Bacillus subtilis. Neither of these scientists developed their observations further. However, the British scientist Frederick Twort “reintroduced the subject almost twenty years after Hankin’s observation by...advancing the hypothesis that it may have been due to...a virus”[2]. While Twort did not further the research due to lack of finances, a French-Canadian named Felix d’Herelle managed to further the  Afterward, d’Herelle used his discovery to successfully treat dysentery in a 12-year-old boy. This became the first attempt to use bacteriophages therapeutically.  </P><P><B>Importance </B></P><P>Phage therapy, while not seen as a “be-all, end-all” for infections, is offered as an alternative treatment in the fight against antibiotic resistance. Antibiotic resistance has become an ever-increasing difficulty and microbiologists have not been able to keep up with the mutations conferring resistance and the transmission of resistance genes (often occurring by horizontal DNA transfer from one resistant bacterium to a nonresistant one). One “superbug”, MRSA (Methicillin-Resistant Staphylococcus Aureus), has become common place in hospitals. While it is often not lethal, it can still pose a threat to immunocompromised patients. </P><P><B>Role of iGEM </B></P><P>This year’s iGEM team from Wisconsin Lutheran College is working with bacteriophages to combat antibiotic resistance. The TLS Phage used binds to TolC, an antibiotic efflux pump commonly found in resistant bacteria. When antibiotics are used in tandem with phage therapy, the antibiotic will eliminate the bacteria lacking a resistance gene while the TLS Phage will kill the rest.  </P><P><B>Cultural Implications </B></P><P>Historically phage therapy has not been prominent in Western cultures, but it has the potential to gain popularity in the future. One must, therefore, question past and current practices: Why do we “over-medicate” with antibiotics? Is there a culture of excess? Are there consequences to overusing phage therapy as happened with antibiotics? What then?  </P><P>The West maintains a culture of excess, and most American/European media portrays opulence as total success. In battle there is the need to annihilate the enemy in effective, though  destructive ways, be it an army, a competing corporate entity, or a microscopic organism. While this phenomenon of antibiotic abuse has spread across the globe, it is not as ingrained in non-Western cultures.  </P><P>The culture of over-medicating in the US began almost as quickly as antibiotics were developed. Farmers began feeding animals antibiotics because they were often infected by bacteria present in fecal matter, near or in grazing pastures, and contaminated water. Patients in hospitals also often neglected to take their full courses of antibiotics due to miseducation and/or personal laxness. Therefore, what few bacteria survived grew resistant, so much so, that any formerly used antibiotic was rendered  useless. </P><P>According to the Wall Street Journal, “Doctors in some hospitals prescribed up to 3 times as many antibiotics as doctors other hospitals.”[3] While this is not indicative of abuse, out of 250,000, at least 14,000[4] people die from <I>Clostridium dificile</I>, which is commonly found in hospitals, and is an antibiotic-resistant bacterium. </P><P>Should phage therapy become widely used in the West by 2020, given this culture of excess, there are concerns that this technology may be abused. Is it possible that phage therapy can be abused? For now, it does not appear so, given that antibiotics function in a very different manner than bacteriophages, are found naturally, and are essentially nontoxic, given they “consist of a core of genetic material (nucleic acid) surrounded by a protein capsid”[5]. This is promising, especially for rural areas that may not be capable of developing new antibiotics to fight infections, and also for bacteria that simply cannot be killed by conventional means.   </P><P><B>Advantages and Drawbacks </B></P><P>As with any scientific development, there are possibilities that phage therapy may not work as well as hoped or have unexpected side effects. Thus, it is always wise to review possible disadvantages and risks.   </P><UL><P>While the advantages include non-toxicity, no possibility of cross-resistance, relative cost-effectiveness, and potential for single dosing. Some disadvantages include the following: 
 
<LI>Phages may kill gut flora, removing any probiotics necessary for healthy digestion, and render the patient more ill. </LI><LI>They also pose a danger to immunocompromised individuals by way of infection—“many protein-based pharmaceuticals can stimulate immune systems, antibiotics that lyse bacteria will release bacterial toxins in situ, and live-attenuated vaccines both actively replicate and evolve including within the context of infecting body tissues”[6]. It should be noted that phages are viruses and should be handled accordingly. </LI><LI>There is a potential for ineffectiveness if the phage lacks lytic capability (reproducing and destroying the membrane of affected cells at once) or virulence.  </LI>
 
This may seem discouraging, but it should be noted that pharmaceutically-based phages go through a rigorous selection via the FDA, EPA, and the USDA after intense monitoring by pharmaceutical companies. They are selected by high virulence (so long as it does not subject patients to worse illness), non-toxicity, non-temperance (does not undergo lysogenesis), and other specifics.  </P><P><B>Conclusion:</B></P><P>It may be advisable to use phage therapy in tandem with antibiotics until the alternate becomes a necessity. The decision to use them largely rests with facilitators of medicine, such as pharmaceutical companies and hospitals, but as this technology has seen marked success in the former Soviet Union, it is increasingly evident that it is a viable weapon in the fight against antibiotic resistance by virtue of its design and by the advancement of the study in post-Soviet Europe. </P><P><B>Bibliography </B>
[1] Phage Therapy Center, About Phage Therapy Center. 2013. http://www.phagetherapycenter.com/pii/PatientServlet?command=static_about 
[2] Alavidze, Zemphira; Morris, J. Glenn, Jr.; Sulakvelidze, Alexander, Bacteriophage Therapy. March 2001. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC90351/ 
[3] Wall Street Journal, CDC: Antibiotic Overuse Can be Lethal. 2014. 
http://www.wsj.com/articles/SB10001424052702304585004579419493198620498 
[4] LiveScience, Antibiotic Misuse in Hospitals Raises Patient Infection Risk. 2014. 
http://www.livescience.com/43845-antibiotics-hospitals-infections.html 
[5] Encyclopedia Britannica, Bacteriophage. 
http://www.britannica.com/science/bacteriophage 
[6] Abedon, Stephen T., Loc-Carrillo, Catherine, Pros and cons of phage therapy. http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3278648/ 
[7] iGEM 2015, https://2015.igem.org/Team:WLC-Milwaukee/ 
[8] Madhusoodanan, Jyoti, Bacteriophage Boom? 29 September 2014. http://www.the-scientist.com/?articles.view/articleNo/41097/title/Bacteriophage-Boom-/ </P><LABEL for="hider4"><H1 id="heading_4"><FONT size="4" color="#610B0B"><B>Phage Therapy Now </B></FONT></H1></LABEL><P>

Modern advances in phage therapy and how phages infect bacteria.
</P><P>
Click the arrow above for more information.
</P><DIV id="content_area_4"><B>Introduction:</B><P>According to the Phage Therapy Center, bacteriophage therapy is defined as “the therapeutic use of lytic bacteriophages to treat pathogenic bacterial infection.” (1) And while this technique was discovered prior to antibiotic treatments, inconsistent results and a lack of understanding caused it to be almost completely overshadowed by antibiotics. (2) The ever increasing issue of antibiotic resistance has led to a phage research revival. Modern knowledge of bacteriophages gives hope that phage therapy will not only work with but surpass antibiotic treatment. </P><P>Bacteriophages are viruses that target bacteria to reproduce. The phages do this by injecting their DNA into the host. The bacteria can then enter one of two phases, the first of which is the lysogenic phase. In the lysogenic phase, the viral genome integrates into the bacterial DNA and goes into a hibernation allowing the cell to replicate on its own until it enters the lytic cycle. In the lytic cycle, the bacterium is used to create parts of the phage which are then assembled. (3) The phages create two enzymes, hollins, which perforate the plasma membrane, and lysins, which attack the cell wall. The enzymes ultimately cause the bacteria to lyse. (2) Phage therapy works based on the bacteriophages’ ability to lyse specific bacteria while not harming eukaryotic cells. </P><B>Early Drawbacks:</B><P>The early founders of phage therapy made many mistakes that ultimately led to antibiotic treatments being favored over phage therapy. The main issue was a lack of information about bacteriophages. While some at the time believed that they were viruses, many of the studies hinted at phages being an enzyme that was created by the bacteria. (4) The lack of knowledge also led to very inconsistent reports causing Eaton, Bayne, and Jones to write a report which deemed the technique no better than prior treatments.(4) Also, phages are very specific, making it difficult to kill the correct bacteria with the correct phages using the technology of the time. (5) The lack of research and understanding also led to impurities being injected into patients, further infecting them, and denaturing the bacteriophages during isolation leaving them inactive. Finally, the researchers faced the issue of the phages being rapidly eliminated by the body and were unable to prevent it. </P><B>Solutions</B><P>
Due to the overuse of antibiotics, alternative treatments like phage therapy have been sought. Fortunately, modern technology has had more success with phages than the discoverers. Thanks to scientific advances, many of the original problems have been solved, and phages are now even superior to antibiotics in many ways.  First, phages can be very specific and can be programmed to target only the virulent bacteria so as to not damage the microbiomes or probiotics. (1) Modern technology has also allowed for the easy and effective isolation of phages without disrupting their function. In addition, while the body metabolizes antibiotics, lytic bacteriophages are constantly replicating in the host bacteria and thus grow exponentially in number. (1) Phages have also been identified with coat proteins that are harder for the body to identify as foreign. This allows the phages to stay in circulation longer and until they reach the lytic cycle and quickly kill the bacteria. (5) There is also much less risk of resistant bacteria forming as only the target bacteria will be infected while the others remain untouched. This is very different from antibiotics that cause drug-resistant strains of numerous species. If, and when, a bacteria does become resistant to a phage, it is much easier and faster to find a new bacteriophage than it is to develop a new antibiotic. (1) Finally, while a phage’s ability to mutate could be seen as a safety threat, it also provides a built-in mechanism to continue to infect bacteria that have adapted a resistance to the original strain. (5)</P><B>Conclusion:</B><P>
While phage therapy is not a new technique, it has only recently started a comeback as a viable antibacterial treatment. Bacteriophages are viruses that can be selected to target and lyse bacteria of choice. Phage therapy has overcome many of the obstacles that cleared the way for antibiotics to take over, and is now in many ways superior to antibiotics. The new surge of research being done on phage therapy continues to reveal exciting new options for the technique. </P><B>References:</B>

[1] &quot;What Is Phage Therapy?&quot; Phage Therapy. Phage Therapy Center, 2013. Web. 20 Aug. 2015. <HTTP:>. 

[2] Potera, Carol. &quot;Phage Renaissance: New Hope against Antibiotic Resistance.&quot; Environmental Health Perspectives. 1 Feb. 2013. Web. 20 Aug. 2015. . 

[3] &quot;Lytic Cycle.&quot; New World Encyclopedia. 22 Dec. 2008. Web. 20 Aug. 2015. <HTTP:>. 

[4] Sulakvelidze, Alexander, Zemphira Alavidze, and Glenn J. Morris. &quot;Bacteriophage Therapy.&quot; Antimicrobial Agents Chemotherapy. National Center for Biotechnology Information, 1 Mar. 2001. Web. 20 Aug. 2015. . 

[5] Carlton, Richard M. &quot;Phage Therapy: Past History and Future Prospects.&quot; Archivum Immunologiae Et Therapiae Experimentalis, 1999. Web. 20 Aug. 2015. <HTTP:>. </HTTP:></HTTP:></HTTP:></DIV><LABEL for="hider5"><H1 id="heading_5"><FONT size="4" color="#610B0B"><B>TolC </B></FONT></H1></LABEL><P>

Discover the structure, functions, and applications of TolC by clicking the arrow above.

</P><DIV id="content_area_5"><B>Structure</B>
TolC is a protein comprised of 471 amino acids to compose a homotrimer that forms a tapered hollow cylinder 140Å in length: a 40Å long outer membrane barrel [the channel domain] that anchors a 100Å long helical barrel that projects across the periplasmic space [1]. In review, the trimer forms a single beta barrel for the passing of proteins. Each monomer gives four antiparallel beta strands and four antiparallel alpha helical strands. This forms the channel and tunnel domains, respectively [1]. There are also small loops present on the sides of TolC that allow for phage binding or colicin attachment. It is thought that the tunnel assembly is supported by hydrogen bonds and salt bridges. 


TolC is interesting because unlike other efflux pumps, TolC bypasses the periplasm. To bypass the periplasm, instead of using a multiprotein assembly to span the membrane, TolC only requires the outer membrane protein. This protein interacts with the inner membrane translocases traffic ATPase and an adaptor protein. There is not just one TolC and interacting translocase, but one cell may have more than one TolC homolog with multiple compatible translocases. This contributes to antibiotic resistance: multidrug resistant <I>Pseudomonas aeruginosa</I> has four major efflux systems. <B>Function</B>

TolC is known as an efflux pump; it provides an exit for large proteins. TolC is used for protein export for proteins such as “heat-stable enterotoxin, cationic antimicrobial peptides, and microcins” [1]. For interest of our project it also effluxes antibiotics, contributing to antibiotic resistance. The protein is exported through translocase, such as AcrAB, then through the TolC periplasmic entrance [1]. <B>In Antibiotic Efflux</B>

TolC’s role in antibiotic efflux is what interests us as a target for fighting multidrug-resistant bacterial infections. Koronakas expressed interest in finding a way to block the TolC pump. Our iGEM team aimed at finding bacteriophage that use this pump to bind and kill the bacterium. <B>References</B>

[1] Koronakis, V., Eswaran, J. &amp; Hughes, C. STRUCTURE AND FUNCTION OF TOLC: The Bacterial Exit Duct for Proteins and Drugs. Annu. Rev. Biochem. 73,467–489 [2004]. 
NOTE: Figures also from Koronakis paper.

</DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>