"
Page
Discussion
View source
History
teams
Log in
 
Team:Missouri Miners/Project
From 2012.igem.org
Home
Team
Meet the Missouri Miners!
Official Team Profile
Project
Parts Submitted to the Registry
Abstract
Inspiration
Background
Project Description
Project Goals
Modeling
Making our Parts
Testing
Next Steps
Notebook
Ethics & Safety
Ethics
Safety
Community
Outreach and Involvement
Public Opinion Research
Acknowledgements
Attributions
Sponsors
Collaboration
Resources
Contact Us
Adjustable Multi-Enzyme to Cell Surface Anchoring Protein
Figure: "Structure of the C. thermocellum CipA scaffoldin CohI9–X-DocII trimodular fragment in complex with the SdbA CohII module. The backbone ribbon representation of the complex depicts SdbA CohII in blue, DocII in green, X module in rose, and CohI9 in yellow. The calcium ions and chloride ion appear as orange and cyan spheres, respectively. The modules are identified, and the N and C termini are labeled accordingly."
(Adams, Currie, Ali, Bayer, Jia, and Smith 833-839)
Abstract
There are a plethora of enzymes that occur in the natural world which perform reactions that could be immensely useful to humans. Unfortunately, the efficiency of some of these reactions may render their applications impractical. The cellulosome scaffolding protein produced by Clostridium thermocellum has been shown to significantly increase the efficiency of cellulose degradation (Kataeva 617-624). It is possible that the scaffolding protein can be reduced in size and adapted for the cell surface of Escherichia coli. Different cohesion sites on the new cell surface display protein can also be introduced to allow attachment of desired enzymes. Future applications would include producing a collection of distinct versions of the scaffolding protein for unique arrangements and concentrations of enzymes, enabling construction of an extra-cellular assembly line for a variety of multi-enzymatic reactions. This would lay the foundation for making previously infeasible applications of enzymes possible through increased efficiency.
Inspiration
Tuberculosis is caused by bacterial infections of Mycobacterium tuberculosis. The use of antibiotics in the treatment of tuberculosis is a double-edged sword: while curing the disease in some cases, it can also cause resistant mutants to emerge. Proper treatment of tuberculosis is typically a multi-drug regimen using first line anti-TB drugs: isoniazid, rifampin, pyrazinamide, ethambutol, and streptomycin (Long 425-428). However, if the regimen is not prescribed correctly or is not followed diligently by the patient, naturally drug-resistant mutants in the patient’s population of tubercle bacilli can become a larger percentage of the population (Long 425-428). Tuberculosis is highly contagious and the spread of resistant mutants is causing more and more drug-resistant tuberculosis cases every year (World Health Organization).
M. tuberculosis’s natural drug resistance  is  due to the mycolic acid (a complex fatty acid) biofilm that protects it from the host’s immune system and makes drug delivery extremely difficult (Ojha, and et al 164-174). The original idea for our project was to help combat drug-resistant tuberculosis infections by starting the first steps toward creating an anti-mycobacteria microbe. This microbe would be capable of breaking down the mycolic acid biofilm around the bacilli to allow drugs and the host’s immune system to eliminate the infection.
Background
This proposal means that standardized fatty acid degradation enzymes and an easy implementation of the degrading enzymes needed to be created. E. coli's fatty acid oxidation pathway breaks down a couple different fatty acids including long chain and complex fatty acids, which is carried out by a multi-subunit enzyme that has an alpha2beta2 conformation (Binstock, Pramanik, and Schulz 492-495). This multi-enzyme pathway can be isolated and over expressed in E. coli along with a cell surface display system engineered from Clostridium thermocellum's cellulosome. C. thermocellum, among other cellulose-degrading organisms, naturally produces and utilizes a scaffolding protein that contain binding sites for cellulose degrading enzymes. This structure has been shown to significantly increase the efficiency of the organism's ability to degrade extracellular cellulose (Kataeva 617-624). The structure itself is composed of a number of smaller parts listed below and illustrated in the diagram.
Cellulose-degrading enzymes - the enzymatic subunits of the cellulosome
Type 1 dockerin regions - attached to the cellulose-degrading enzymes
Type 1 cohesin regions - attached to the cellulosome scaffoldin protein and bind to type 1 dockerin regions
Cellulose binding domain - attaches to the cellulose substrate and further increases the efficiency of cellulose degradation process
Type 2 dockerin region - attached to cellulosome scaffoldin protein
Type 2 cohesin region - binds to type 2 dockerin region and the S-layer binding module that anchors the cellulosome to the surface of the cell
Figure 2 - Diagram of the cellulosome as it appears in C. thermocellum naturally
The C. thermocellum cellulosome is a good candidate for this project for the following reasons:
The entirety of the cellulosome is located outside of the cell. This means that substrates do not have to be taken into the cell before reactions can occur.
The cellulosome keeps enzymes within close proximity to the cell. This would be ideal for applications in sensitive environments (i.e. within another organism).
The cellulosome significantly increases the efficiency of cellulose-degrading enzymes. The system may do the same for other multiple enzyme processes.
The cohesin/dockerin interactions of a given species are specific to that species; the cohesin of one species will not bind to the dockerin of another.
Project Description
Project Goals
The purpose of this project is to create a BioBrick tool kit that would allow teams to anchor multiple enzymes to the external surface of Escherichia coli. Not only would the use of this kit allow users to anchor their parts to the surface of E. coli but also adjust the anchoring construct to fit their needs. To execute this project, the following goals must be accomplished:
Replace the S-layer binding module of C. thermocellum with a trans-membrane protein that is compatible with gram negative bacteria like E. coli
Reduce the size of the cellulosome scaffoldin gene (roughly 7kb long) to make it easier to work with during PCR and in plasmids
Compile a variety of cohesin and dockerin regions (from other organisms) to give the user more control over how the enzymatic subunits bind to the scaffoldin
Standardize a variety of cohesin regions to allow for easy attachment of enzymes of the user’s choice to the scaffoldin
Project Overview
In order to fulfill the above project goals, the following steps need to be taken:
Step 1: Combine CtCoh2 with the E. coli transmembrane protein LPP-OmpA (BBa_K103006) , submitted by the 2008 Warsaw iGEM team
Step 2: Isolate both miniA1 and miniA2 (see Figure 4) fragment sequences from C. thermocellum's scaffoldin CipA gene
Step 3: Combine miniA1 and miniA2 by means of a single sac1 restriction site
Step 4: Isolate the coding sequence for the type one cohesin region, CtCoh1, from C. thermocellum's genome
Step 1
Our team addressed the S-layer binding module issue by hybridizing the type 2 cohesin of C. thermocellum with the LPP-OmpA, BBa_K103006 , part previously submitted to the parts registry by the 2008 Warsaw iGEM team. The LPP-OmpA part is a transmembrane protein with an extracellular amino acid string that allowed us to attach the type 2 cohesin of Clostridium's sdbA anchoring module without affecting the protein's structure or functionality. With LPP-OmpA acting as the new anchoring module, the cellulosome should successfully bind to the cell surface. Unfortunately the final transformation of this part failed and the team was unable to submit this part; however, it is documented in the Registry as BBa_K877001 .
Figure 3 - A diagram of our MiniA part ( BBa_K877000 ) and LPP-OmpA/coh2 part ( BBa_K877001 )
Steps 2 and 3
To address the difficult size of the cipA cellulosome gene, the team developed an abbreviated version of the cellulosome scaffoldin by removing from the coding domain six of the nine cohesion regions as well as the cellulose binding domain. The second fragment termed miniA2 codes for the last two cohesin modules and a signaling peptide for exocytosis. The miniA1 fragment begins with the iGEM standard prefix and ends with a blunt restriction site, sac1, while the miniA2 fragment begins with sac1 and ends with the standard iGEM suffix. Figure 4 depicts the portions of the scaffolding protein that were termed miniA1 and miniA2.
Figure 4 - Comparison between wild type cipA scaffoldin and our miniA scaffoldin
Step 4
Due to complications in the lab, our team did not reach this step of the project; however, in the future, it would be necessary to isolate the coding sequence for the type one cohesin region, CtCoh1, from C. thermocellum's genome. These fragments would be hybridized with enzymes of interest in order to create the desired custom extracellular assembly line. Our team planned to first hybridize these parts with a variety of fluorescent proteins to test the system. See Testing for more information.
Modeling
Enzymatic modeling provided by BYU iGEM
Making Our Parts
XXX Figure 5 - Replacement of the native s-layer binding module with the LPP-OmpA part to create our LPP-OmpA/CtCoh2 part
The portion of the sdbA gene that codes for the type 2 cohesin module of the sdbA protien was PCR amplified and given the iGEM suffix and a SacI prefix. The SacI prefix matches the same restriction site on LPP-OmpA's suffix. The two were ligated together after being cut with SacI. This part was entered into the Registry as BBa_K877001 .
Figure 6 - Assembly of our miniA part from portions of the cipA gene
The portion of the cipA gene which codes for the last two type 1 cohesin modules, along with the portion that codes for the first type 1 cohesin module and type 2 dockerin module, where PCR amplified. The fragments were named miniA1 and miniA2 respectively. MiniA1 was given the iGEM prefix and a SacI suffix while miniA2 was given a SacI prefix and the iGEM suffix. These two fragments were ligated together at the blunt restriction sites, inserted into the iGEM chloramphenicol-resistant backbone, and submitted to the Registry as BBa_K877000 .
Testing
In the future the team could test the system via the following methods:
To determine whether or not the type 2 cohesin module is successfully anchored to the surface of the cells, the team must first create a fluorescent protein/type 2 dockerin module combination. Cells will be made to express both the LPP-OmpA-CtcohII part as well as the fluorescent dockerin module. If the cells fluoresce, they will be washed to ensure that any secreted fluorescent protein is not responsible. If the cells still fluoresce, the membranes will be isolated and tested alone for fluorescence. If the part works, the membrane will fluoresce indicating that the fluorescent protein is anchored to the cell membrane.
To determine whether or not the scaffoldin module works, the same experiment will be performed but with a different hybrid. In this case, the fluorescent protein would be combined with a type 1 dockerin module. It would also be ideal if more than one color of fluorescent protein was successfully anchored to the cell membrane. This would show that the construct is capable of anchoring more than one part.
It may also be possible to asses the effects of changing the ratios between the cohesin sites compatible with one fluorescence and those compatible with another. For instance if cohesins from two species were incorporated into the scaffoldin and if two different fluorescent proteins were each hybridized to a corresponding dockerin module, the fluorescent protein hybrids would only be able to bind to the cohesin that originated from the same organism as the dockerin module that protein has. In theory if one protein's dockerin had a single cohesin on the scaffolding and the other had two, the effective concentration of the later protein would be twice that of the former.
The team may also attempt to test the system using a naturally occurring multiple enzyme process (possibly fatty acid oxidation). The efficiency of such a system could be measured with and without the modified scaffoldin and anchoring protein.
Next Steps
In the future, a library of cohesin and dockerin modules could be built for simplified assembly of customized scaffodins. To do this, the team would isolate cohesin and dockerin modules of a variety of types from a variety of cellulosome-producing species. These modules would then be combined by iGEM non-standard restriction sites flanking the gene sequence, allowing for almost complete customization to the user's needs.
References
Adams, J.J., M.A. Currie, S. Ali, E.A. Bayer, Z. Jia, and S.P. Smith. " Insights into Higher-Order Organization
of the Cellulosome Revealed by a Dissect-and-Build Approach: Crystal Structure of Interacting "Clostridium thermocellum" Multimodular Components
." Journal of Molecular Biology. 369. (2010): 833-839. Web. 1 Oct. 2012.
Binstock, J. F., A. Pramanik, and H. Schulz. " Isolation of a Multi-enzyme Complex of Fatty Acid Oxidation from
"Escherichia coli
"." Biochemistry. 74.2 (1977): 492-495. Web. 29 Sep. 2012.
Chao, Tiffany. "Tuberculosis Becoming More Drug-Resistant Worldwide." ABC News. ABC News Medical Unit, 30 August
2012. Web. 28 Sep 2012. < http://abcnews.go.com/Health/tuberculosis-drug-resistant-worldwide/story?id=17107153&gt ;.
Kataeva, I., G. Guglielmi, and P. Beguin. 1997. Interaction between Clostridium thermocellum endoglucanase CelD and polypeptides
derived from the cellulosome-integrating protein CipA: stoichiometry and cellulolytic activity of the complexes
. Biochem. J. 326:617-624.
Long, Robert. "Drug-resistant tuberculosis." Canadian Medical Association Journal. 163.4 (2000): 425-428. Web. 28
Sep. 2012. < http://www.ecmaj.ca/content/163/4/425.full.pdf&gt ;.
Ojha, et al. "Molecular Microbiology." Growth of "Mycobacterium tuberculosis" Biofilms Containing Free Mycolic Acids
and Harbouring Drug-tolerant Bacteria. 69.1 (2008): 164-174. Web. 29 Sep. 2012. < http://onlinelibrary.wiley.com/doi/10.1111/j.1365-2958.2008.06274.x/pdf ;.
"Tuberculosis." World Health Organization. N.p., March 2012. Web. 28 Sep 2012. < http://www.who.int/mediacentre/factsheets/fs104/en/ .
Retrieved from " http://2012.igem.org/Team:Missouri_Miners/Project "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
