<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UC_Davis_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UC Davis/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM</TITLE><UL id="topnav"><LI class="dropdown"><DIV class="dropdown-main">PROJECT</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Design">Design</A><A href="https://2018.igem.org/Team:UC_Davis/Measurement">Measurement</A><A href="https://2018.igem.org/Team:UC_Davis/Demonstrate">Results</A><A href="https://2018.igem.org/Team:UC_Davis/Modeling">Modeling</A><A href="https://2018.igem.org/Team:UC_Davis/InterLab">InterLab</A></DIV></LI><LI class="dropdown"><DIV class="dropdown-main">HUMAN PRACTICES</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Human_Practices">Integrating Human Practices</A><A href="https://2018.igem.org/Team:UC_Davis/Education">Education &amp; Outreach</A></DIV></LI><LI class="dropdown"><A href="https://2018.igem.org/Team:UC_Davis/Parts" class="nav-item" class="dropdown-main">PARTS</A></LI><LI id="navlogo"><A class="navlogo" href="https://2018.igem.org/Team:UC_Davis"><B class="logo-word">Cenozoic</B></A></LI><LI class="dropdown"><DIV class="dropdown-main">LAB WORK</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UC_Davis/Notebook">Lab Notebook</A><A href="https://2018.igem.org/Team:UC_Davis/SupplementalMaterials">Supplementary Material</A><A href="https://2018.igem.org/Team:UC_Davis/Safety">Safety</A></DIV></LI><LI class="dropdown"><DIV class="nav-item">TEAM</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Team">Meet the Team</A><A href="https://2018.igem.org/Team:UC_Davis/Attributions">Attributions</A><A href="https://2018.igem.org/Team:UC_Davis/Sponsors">Sponsors</A></DIV></LI><LI class="dropdown"><DIV class="nav-item">ACHIEVEMENTS</DIV><DIV class="dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Medal_Requirements">Medal Requirements</A><A href="https://2018.igem.org/Team:UC_Davis/Collaborations">Collaborations</A></DIV></LI><LI class="burger"><LABEL for="b-control">☰</LABEL><DIV class="burger-content"><DIV class="b-dropdown"><DIV class="b-dropdown-main">Project</DIV><DIV class="b-dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Design">Design</A><A href="https://2018.igem.org/Team:UC_Davis/Measurement">Measurement</A><A href="https://2018.igem.org/Team:UC_Davis/Demonstrate">Results</A><A href="https://2018.igem.org/Team:UC_Davis/Modeling">Modeling</A><A href="https://2018.igem.org/Team:UC_Davis/InterLab">InterLab</A></DIV></DIV><DIV class="b-dropdown"><DIV class="b-dropdown-main">Human Practices</DIV><DIV class="b-dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Human_Practices">Integrating Human Practices</A><A href="https://2018.igem.org/Team:UC_Davis/Education">Education &amp; Outreach</A></DIV></DIV><DIV class="b-dropdown"><A href="https://2018.igem.org/Team:UC_Davis/Parts" class="b-dropdown-main">Parts</A></DIV></DIV><DIV class="b-dropdown"><DIV class="b-dropdown-main">Lab Work</DIV><DIV class="b-dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Experiments">Experiments</A><A href="https://2018.igem.org/Team:UC_Davis/Notebook">Lab Notebook</A><A href="https://2018.igem.org/Team:UC_Davis/SupplementalMaterials">Supplementary Material</A><A href="https://2018.igem.org/Team:UC_Davis/Safety">Safety</A></DIV></DIV><DIV class="b-dropdown"><DIV class="b-dropdown-main">Team</DIV><DIV class="b-dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Team">Meet the Team</A><A href="#">Advisors</A><A href="https://2018.igem.org/Team:UC_Davis/Sponsors">Sposors</A></DIV></DIV><DIV class="b-dropdown"><DIV class="b-dropdown-main">Achievements</DIV><DIV class="b-dropdown-content"><A href="https://2018.igem.org/Team:UC_Davis/Medal_Requirements">Medal Requirements</A><A href="https://2018.igem.org/Team:UC_Davis/Collaborations">Collaborations</A></DIV></DIV></LI></UL><TITLE>Education</TITLE></P><DIV class="parentContent"><DIV class="sidenav" style="padding-top: 250px;"><A href="#1">Bioassay Design</A><A href="#2">Promoter Constructs</A><A href="#3">Host Strains</A><A href="#4">Chemicals of Concern</A><A href="#5">Plasmid</A><A href="#6">Measurement</A><A href="#7">Extensions</A><A href="#8">Project Design Notebook</A><A href="#9">References</A></DIV><DIV class="container"><DIV class="scrollable" style="padding-left:10%;padding-right: 7%;"><CENTER><DIV style="padding-top:100px;padding-bottom: 60px; font-size: 30px; color: #d9a900"><B>Project Design </B></DIV></CENTER><DIV class="clearfix"><DIV id="1" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Bioassay Design</B></DIV><DIV style="padding-left:7%;padding-right: 7%">
      We designed a mammalian cell-based bioassay that reports activation of specific stress pathways via fluorescence, for use in environmental toxicology. To do this, we selected transcriptionally regulated target genes which are present in mammalian cells and are involved in stress pathways. We isolated the promoters with transcription factor binding sites from these target genes and coupled them to a fluorescent reporter gene. We selected EGFP, a variant of green fluorescent protein (GFP). GFP is ubiquitous in synthetic biology due to its reliability and ease of measurement [1]. EGFP is derived from GFP and has been optimized for use in mammalian systems. When a chemical of concern is screened using our assay, if it triggers a specific stress response, the reporter gene will be expressed, causing the assay to fluoresce. The fluorescence of the assay can be quantitatively measured and analyzed. This assay will provide data on the effect of chemicals of concern on the physiological health of mammalian cells; measurements may be easily taken a range of concentrations, duration of exposure, salinities, pH, temperatures, nutrient availabilities, and other conditions. This also allows for measurement of synergistic or interfering effects due to multiple chemicals of concern present simultaneously.

<P>We selected 8 promoter constructs derived from 5 target genes (see Figure 1 below) and coupled them to EGFP. This promoter and reporter gene construct were inserted into a plasmid and transfected into two cell lines (see Figure 2 below). The resulting bioassays were exposed to different chemicals of concern at a variety of concentrations and conditions (see Figure 3 below).
</P></DIV><DIV style="padding-left:10%;padding-bottom: 22px; font-size: 20px; color: #d9a900"><B><I>Why not use whole organisms?</I></B></DIV><DIV style="padding-left:10%;padding-right: 7%">A cell-based approach cannot replace in vivo toxicology studies. However, these studies require extensive funding, time, and other resources. By developing a relatively low-cost, cell-based bioassay, preliminary data may be quickly gathered, allowing for more informed decision making as to which in vivo studies are necessary. By using a cell-based preliminary assay, it is our hope that researchers will be able to quickly gather data, make more informed decisions, and save resources. Our cell-based bioassay may also be used to add to the body of knowledge concerning the effect of specific chemicals of concern on the physiological health of mammalian cells and the mechanism of stress. </DIV><DIV style="padding-left:10%;padding-bottom: 22px; font-size: 20px; color: #d9a900"><B><I>Why not use cell-free biochemical assays, such as ELISA?</I></B></DIV><DIV style="padding-left:10%;padding-right: 7%">
  Antibody-based assays, such as Enzyme-Linked Immunosorbent Assay (ELISA), have been very successful in biomedical research, environmental toxicology, and other fields [2]. These cell-free methods involve selective binding of a target molecule to a prepared antibody. Such methods are very successful at identifying single, known compounds in an environmental sample, but do not provide any data regarding the effect of the chemical of concern on the health of a living cell. Similarly, the tools of analytical chemistry and organic chemistry may be used to great success when identifying molecules, but do not provide any data regarding the actual effect on a living cell.
</DIV><DIV style="padding-left:10%;padding-bottom: 22px; font-size: 20px; color: #d9a900"><B><I>Why use synthetic biology?</I></B></DIV><DIV style="padding-left:10%;padding-right: 7%">

The goal of our bioassay is to create a tool that can be used to better understand the effect on the physiological health of mammalian cells of environmental toxins. An alternative way to achieve this knowledge is to expose cells to the chemicals of concern, lyse the cells, isolate the RNA, and run a quantitative-real-time-reverse-transcription-PCR in order to characterize and quantify the mRNAs present in the cell. However, this approach has limitations. It requires lysing the cells and cannot be used to gather real-time data about the behavior of the same cell over time. By using a fluorescent reporter gene, we can measure the induction of the reporter gene over time without lysing cells and can more easily take a large number of data points across different chemicals of concern, concentrations, and other variables. A fluorescent bioassay also reduces the amount of work required to measure many data points, compared to PCR based methods. Our bioassay also makes possible the future study of the behavior of a single cell over time after exposure to a chemical of concern, with the aid of microfluidics.
</DIV><DIV style="padding-left:10%;padding-bottom: 22px; font-size: 20px; color: #d9a900"><B><I>Why use mammalian cells?</I></B></DIV><DIV style="padding-left:10%;padding-right: 7%">
  We chose to use mammalian cells because they make much more accurate models for human health than bacteria or yeast. Furthermore, within the iGEM competition and the field of synthetic biology as a whole, there has been relatively little work with mammalian systems, compared to bacteria, yeast, and algae. Working with mammalian cells brings a variety of new challenges and opportunities to iGEM: they are more difficult and expensive to culture than bacteria, they require specialized equipment and safety training, they can be used to produce proteins suitable for use in human therapeutics (due to similar post-translational modifications), and they can be used for more complicated circuits and pathways utilizing spatial/temporal differentiation [3].

</DIV><DIV style="padding-left:10%;padding-bottom: 22px; font-size: 20px; color: #d9a900"><B><I>Why not use human cells?</I></B></DIV><DIV style="padding-left:10%;padding-right: 7%">
  Although human cell lines would make a superior model for human disease, compared to cell lines derived from hamsters and mice, for our project we chose not to use human cells. Work involving human cells requires specialized facilities, equipment, resources, and safety training. Human cells require a BSL-2 lab, which would have been more difficult for our team to use than our regular wetlab space, which is BSL-1. Additionally, as our project took place in the context of the iGEM competition, we wanted for other teams to be able to easily reproduce our findings and expand them. By using human cell lines, many teams, which lack access to a BSL-2 lab, would have had more difficulty in expanding upon our project. It would be relatively straightforward to insert our genetic constructs to a human cell line, and this presents an opportunity to extend our project.
</DIV><DIV id="2" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Promoter Constructs</B></DIV><DIV style="padding-left:7%;padding-right: 7%">

Table 1 shows the promoter constructs we used and the target genes from which they were derived. Full FASTA sequences for our promoters are available <A href="https://2018.igem.org/Team:UC_Davis/SupplementalMaterials">here.</A></DIV><CENTER><DIV style="line-height: 25px">
    Table 1. Promoter Constructs
</DIV><TABLE class="tg"><TBODY><TR><TH class="tg-9hbo">Construct</TH><TH class="tg-9hbo">Target Gene</TH><TH class="tg-9hbo">Species of Origin</TH><TH class="tg-9hbo">Stress Pathway</TH><TH class="tg-9hbo">Size</TH><TH class="tg-9hbo">Further Reading</TH></TR><TR><TD class="tg-yw4l">MT1</TD><TD class="tg-yw4l">Metallothionein 1</TD><TD class="tg-yw4l">Mus musculus</TD><TD class="tg-yw4l">Oxidative, heavy metal</TD><TD class="tg-yw4l">305 nucleotides</TD><TD class="tg-yw4l">[4]</TD></TR><TR><TD class="tg-yw4l">MT2_1</TD><TD class="tg-yw4l">Metallothionein 2</TD><TD class="tg-yw4l">Homo sapiens</TD><TD class="tg-yw4l">Oxidative, heavy metal</TD><TD class="tg-yw4l">377 nucleotides</TD><TD class="tg-yw4l">[5]</TD></TR><TR><TD class="tg-yw4l">MT2_2</TD><TD class="tg-yw4l">Metallothionein 2</TD><TD class="tg-yw4l">Homo sapiens</TD><TD class="tg-yw4l">Oxidative, heavy metal</TD><TD class="tg-yw4l">59 nucleotides</TD><TD class="tg-yw4l">[5]</TD></TR><TR><TD class="tg-yw4l">MT2_3</TD><TD class="tg-yw4l">Metallothionein 2</TD><TD class="tg-yw4l">Homo sapiens</TD><TD class="tg-yw4l">Oxidative, heavy metal</TD><TD class="tg-yw4l">60 nucleotides; construct MT2_2 plus one base at 5’ end</TD><TD class="tg-yw4l">[5]</TD></TR><TR><TD class="tg-yw4l">MT2_4</TD><TD class="tg-yw4l">Metallothionein 2</TD><TD class="tg-yw4l">Homo sapiens</TD><TD class="tg-yw4l">Oxidative, heavy metal</TD><TD class="tg-yw4l">60 nucleotides; construct MT2_2 plus one base at 3’ end</TD><TD class="tg-yw4l">[5]</TD></TR><TR><TD class="tg-yw4l">FGF</TD><TD class="tg-yw4l">FGF21</TD><TD class="tg-yw4l">Homo sapiens</TD><TD class="tg-yw4l">Unfolded protein response (UPR), endoplasmic reticulum stress</TD><TD class="tg-yw4l">695 nucleotides</TD><TD class="tg-yw4l">[6]</TD></TR><TR><TD class="tg-yw4l">GD153</TD><TD class="tg-yw4l">GADD153</TD><TD class="tg-yw4l">Cricetulus griseus</TD><TD class="tg-yw4l">Organochlorine biocides, genotoxins</TD><TD class="tg-yw4l">811 nucleotides</TD><TD class="tg-yw4l">[7], [8]</TD></TR><TR><TD class="tg-yw4l">GD45</TD><TD class="tg-yw4l">GADD45α</TD><TD class="tg-yw4l">Homo sapiens</TD><TD class="tg-yw4l">Genotoxins, mechanical stress</TD><TD class="tg-yw4l">1006 nucleotides</TD><TD class="tg-yw4l">[9]</TD></TR></TBODY></TABLE></CENTER><DIV id="3" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Host Strains</B></DIV><CENTER><DIV style="line-height: 25px">
        Table 2. Host Strains
    </DIV><TABLE class="tg"><TBODY><TR><TH class="tg-9hbo">Host Strain</TH><TH class="tg-9hbo">Description</TH><TH class="tg-9hbo">Supplier</TH></TR><TR><TD class="tg-yw4l">CHO-DG44</TD><TD class="tg-yw4l">An immortal, adherent cell line derived from Chinese hamster (Cricetulus griseus) ovary cells. The strain we used is dihydrofolate reductase deficient.</TD><TD class="tg-yw4l">ATCC: CRL-9096 [10]</TD></TR><TR><TD class="tg-yw4l">AML-12</TD><TD class="tg-yw4l">An immortal, adherent cell line derived from mouse (Mus musculus) liver cells.</TD><TD class="tg-yw4l">ATCC: CRL-2254 [11]</TD></TR></TBODY></TABLE></CENTER><DIV id="4" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Chemicals of Concern</B></DIV><CENTER><DIV style="line-height: 25px">
        Table 3. Chemicals of Concern
    </DIV><TABLE class="tg"><TBODY><TR><TH class="tg-j2xt">Chemical name</TH><TH class="tg-j2xt">Occupational limit</TH><TH class="tg-j2xt">LD50</TH><TH class="tg-j2xt">Klamath Environmental concentration</TH><TH class="tg-j2xt">Cytotoxicology (In micromoles per liter)</TH></TR><TR><TD class="tg-0lax">Hydrogen Peroxide</TD><TD class="tg-0lax">75 mg/L</TD><TD class="tg-0lax">2000 mg/kg</TD><TD class="tg-0lax">--</TD><TD class="tg-0lax">Relevant dose is 60 uM</TD></TR><TR><TD class="tg-0lax">Warfarin</TD><TD class="tg-0lax">0.1 ug/L</TD><TD class="tg-0lax">374 mg/kgmice186 mg/kg rats</TD><TD class="tg-0lax">Qualitatively detected by Tom Young’s lab; unpublished data</TD><TD class="tg-0lax">Relevant dose is 0.0316 – 31.6 μM</TD></TR><TR><TD class="tg-0lax">2,4-D</TD><TD class="tg-0lax">10 ug/L</TD><TD class="tg-0lax">2019 ppm (min)12979 ppm(max)</TD><TD class="tg-0lax">0.424 - 7.49 ppm</TD><TD class="tg-0lax">-</TD></TR><TR><TD class="tg-0lax">Copper sulfate</TD><TD class="tg-0lax">1 ppb by inhalation</TD><TD class="tg-0lax">30 mg/kg</TD><TD class="tg-0lax">Copper measured at concentration of 19-67 ppm (112uM - 420uM)</TD><TD class="tg-0lax">Relevant dose is &amp; lt;1.3ppm (6.25uM)</TD></TR><TR><TD class="tg-0lax">Metam Sodium</TD><TD class="tg-0lax">Limit of irritation 22 ug/L</TD><TD class="tg-0lax">781 mg/kg (rat)1836 ppm (min)</TD><TD class="tg-0lax">131309.21 kg</TD><TD class="tg-0lax">-</TD></TR></TBODY></TABLE></CENTER><DIV id="5" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Plasmid</B></DIV><DIV style="padding-left:7%;padding-right: 7%">
  We used pcDNA3-EGFP as our plasmid [12]. The original plasmid is shown in Figure 1 below.

</DIV><CENTER><DIV style="line-height: 25px">
    Figure 1. Map of pcDNA3-EGFP [13]

</DIV></CENTER><DIV style="padding-left:7%;padding-right: 7%">
  To prepare our constructs, we used restriction enzymes to remove the CMV enhancer, CMV promoter, and the T7 promoter. Promoter sequences were inserted using either restriction digest or Sequence and Ligation Independent Cloning (SLIC). Figure 2 below shows the complete plasmid for construct MT2_1.

</DIV><CENTER><DIV style="line-height: 25px">
    Figure 2. Map of pcDNA-EGFP_MT2_2 [13]

</DIV></CENTER><DIV id="6" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Measurement</B></DIV><DIV style="padding-left:7%;padding-right: 7%">
  Our bioassay was constructed on 96 well plates. Plates were seeded with 30,000 cells/well for AML12 cells and 20,000 cells/well for CHO cells and allowed to adhere for 24 hours. The cells were seeded such that 24 hours after reaching they would be at 70%-90% confluence. At this time, we transfected our cells with the construct containing the target promoter with reporter gene and allowed to sit for another 24 hours. Next, the complete growth media present in the wells were aspirated and replaced with clear media containing chemical of concern. We experimented with use of both a Tecan plate reader and a fluorescent microscope for measurement. For a full report on our measurement practices, please visit our  <A style="color:  #d9a900;" href="https://2018.igem.org/Team:UC_Davis/Measurement">measurement page</A> and our  <A style="color:  #d9a900;" href="https://2018.igem.org/Team:UC_Davis/Experiments">experiments page.</A></DIV><DIV id="7" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Extensions</B></DIV><DIV style="padding-left:7%;padding-right: 7%">
  Our project opens up new opportunities for work with mammalian cells in the iGEM competition. We have created comprehensive protocols for working with two mammalian cell lines, AML-12, and CHO-DG44, which may be downloaded on our Experiments Page. By adding new mammalian parts to the registry, future teams will have the ability to easily access useful mammalian regulatory elements for use in their own constructs. Future teams may also benefit from our protocol for measuring fluorescence of adherent mammalian cells. One extension of our work is to transfect our construct into human cell lines. By using human cells, the bioassay will be a more accurate model for human health.
</DIV><CENTER><DIV style="padding-top:100px;padding-bottom: 60px; font-size: 30px; color: #d9a900"><B>Project Design Notebook </B></DIV></CENTER></DIV>

The Project Design Notebook available for download below contains our thought and research as well as work space, as we designed our project. Within, we grapple with aspects of human practices, molecular biology, environmental toxicology, and the logistics of planning our experiments.

<DIV id="8" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>Project Design Notebook</B></DIV><DIV id="9" style="padding-left:7%;padding-bottom: 22px; font-size: 25px; color: #667d9d"><B>References</B></DIV><DIV style="padding-left:7%;padding-right: 7%">
  [1] “PDB101: Molecule of the Month: Green Fluorescent Protein (GFP).” PDB-101, RCSB PDB, June 2003, pdb101.rcsb.org/motm/42.
       
       [2] Enzyme Immunoassay (EIA)/Enzyme-Linked Immunosorbent Assay (ELISA)
       Rudolf M. Lequin
       Clinical Chemistry Dec 2005, 51 (12) 2415-2418; DOI: 10.1373/clinchem.2005.051532
       
       [3]Kis Z, Pereira HSA, Homma T, Pedrigi RM, Krams R. 2015 Mammalian synthetic biology: emerging medical applications. J. R. Soc. Interface 12 : 20141000. http://dx.doi.org/10.1098/rsif.2014.1000
       
       [4] Larochelle, Olivier &amp; Labbé, Simon &amp; Harrisson, Jean-François &amp; Simard, Carl &amp; Tremblay, Véronique &amp; St-Gelais, Geneviève &amp; Govindan, Manjapra Variath &amp; Seguin, Carl. (2008). Nuclear Factor-1 and Metal Transcription Factor-1 Synergistically Activate the Mouse Metallothionein-1 Gene in Response to Metal Ions. The Journal of biological chemistry. 283. 8190-201. 10.1074/jbc.M800640200.
       
[5] Santos, Anderson K. et al. &quot;Expression System Based on An MtIIa Promoter to Produce Hpsa In Mammalian Cell Cultures&quot;. Frontiers in Microbiology, vol 7, 2016. Frontiers Media SA, doi:10.3389/fmicb.2016.01280.
       
       [6] F.G. Schaap, A.E. Kremer, W.H. Lamers, P.L. Jansen, I.C. Gaemers. Fibroblast growth factor 21 is induced by endoplasmic reticulum stress
       Biochimie, 95 (2013), pp. 692-699, 10.1016/j.biochi.2012.10.019
       
       [7]  Li, Dahui et al. &quot;Genotoxic Evaluation Of The Insecticide Endosulfan Based On The Induced GADD153-GFP Reporter Gene Expression&quot;. Environmental Monitoring And Assessment, vol 176, no. 1-4, 2010, pp. 251-258. Springer Nature, doi:10.1007/s10661-010-1580-7.
       
       [8]  Park, Jong Sung et al. &quot;Isolation, Characterization And Chromosomal Localization Of The Human GADD153 Gene&quot;. Gene, vol 116, no. 2, 1992, pp. 259-267. Elsevier BV, doi:10.1016/0378-1119(92)90523-r.
       
       [9] Mitra, Sumegha et al. &quot;Gadd45a Promoter Regulation By A Functional Genetic Variant Associated With Acute Lung Injury&quot;. Plos ONE, vol 9, no. 6, 2014, p. e100169. Public Library Of Science (Plos), doi:10.1371/journal.pone.0100169.
       
       [10] Cell line available from ATCC at: https://www.atcc.org/Products/All/CRL-9096.aspx
       
       [11] Cell line available from ATCC at: https://www.atcc.org/products/all/CRL-2254.aspx
       
       [12] More detailed information regarding this plasmid is available from Addgene here: https://www.addgene.org/13031/ . pcDNA3-EGFP was a gift from Doug Golenbock (Addgene plasmid # 13031)
       
       [13] Plasmid map created with software from SnapGene. Software is available at: snapgene.com

</DIV></DIV></DIV></DIV></DIV><TITLE>iGEM-Footer</TITLE><FOOTER><DIV class="centered clearfix"><DIV class="footer-contact"><H2 style="font-size: 25px;color: #d9a900; padding-left: 10px">Contact</H2><P style="padding-left: 10px">530-752-1139</P><P style="padding-left: 10px">2240 Academic SurgeDavis,Ca 95616</P></DIV><DIV style="font-size: 25px; padding-right: 130px; padding-left: 130px;padding-top: 50px;">
UC Davis iGEM 2018</DIV></DIV></FOOTER></DIV></DIV></DIV></DIV></BODY></HTML>