Team:Aachen/Notebook
iGEM Team Aachen 2017
Home
Project
Description
Design
Applied Design
Biobricks
Safety
Results
Demonstrate
Outlook
Human Practices
Overview
Human Practices - Silver
Integrated Human Practices - Gold
Collaborations
Lab
Workflow
Protocols
Notebook
InterLab
Team
Team
Attributions
Special Awards
Hardware
Plants
Model
Engagement
Week
01
02
03
04
05
06
07
08
09
10
11
12
13
14
15
16
17
18
19
20
21
22
<div class="white100"></div>
Week 1
05/29
AVP1: For the Integration of AVP1 we are using the easy clone plasmid pCfB391       which contains the homologous regions, the terminator and a Histidin auxotrophy marker. We received       E.coli DH5alpha carrying pCfb391 and made overnights in LB+Ampicillin (To be used by AVP1 Group Jotho)
LAB: Isolation of yeast genome (quick and dirty protocol)
ENA1: PCR of left, right homologous arm from the previously isolated yeast       genome and kanMX from pUG6 plasmid were performed, but the gel showed failure.       The PCRs failed repeatedly, with the gels looking strange; our troubleshooting       concluded that the error was in a wrongly prepared agarose solution.
05/30
LAB: For further use we created sterile 50% Glycerol stock.
AVP1: We created Cryos of the overnights of E.coli carrying pCfB391, Storage at -80°C
ENA1: The production PCR for the left and right homologous arm for the       ENA1 knockout finally worked.
Gel right arm:
Gel left arm:
05/31
LAB: Overnight cultures of Escherichia coli carrying       the plasmids pUG6, Easy clone plasmid and our Gal1 plasmid were made.
ENA1: A gradient PCR was performed for the kanMX cassette,       to determine the annealing temperature of our primers.
06/01
LAB:Cryo cultures of Escherichia coli carrying the plasmids       pUG6, Easy clone plasmid and our Gal1 plasmid were made to secure the plasmids.
ENA1:The kanMX cassette was amplified, with a pUG6 plasmid as template.       The PCR worked fine.
The ENA1 knockout transformation was performed with the three previously       amplified fragments into BY4742 wildtype.
kanmx product:
AVP1:We prepared the saving of the AVP1-gene in      Escherichia coli. We made four overnights, two of E. coli pCfB 391       (containing the easy clone plasmid) and another two of E. coli       containing pUC19. They were incubated at 37°C.
06/02
NHX1: The 4 fragments for the overexpression transformation of       NHX1 were amplified: the left homologous sequence “X-Up”, the Gal1       promotor from the pUG6 plasmid,  the NHX1 gene from BY4742 genome and       the right homologous sequence from an easy clone plasmid,       including the marker genes for yeast and e.coli, “X-Down”.
AVP1: We performed a plasmid preparation of pUC19.
<div class="white100"></div>
Week 2
06/06
mCherry Labelling: To check the correct       intracellular position (vacuolar membrane) of the foreign       genes we are transforming into S.cerevisiae BY4742 we want       to label the native vacuolar protein       encoded by the gene ATG22 with mCherry (Fusion Protein).       On the other hand, we will label the foreign genes with       eGFP and colocalize it with ATG22-mCherry to the vacuolar       membrane.
In the following weeks we worked on transforming S.cerevisiae BY4742 with mCherry.       Steps included:
I. Isolation of Plasmids carrying the genes for mCherry,        eGFP and a Kanaymcin-Cassette (Selection with Geniticin) (Week 2)
II. Isolation of Homologous Regions from S.cerevisiae       BY4742 for integration into BY4742 Genome via        homologous recombination (Week 2)
III. PCRs to amplifiy the 4 Fragments for the        mCherry Transformation: Homologous Region Left,        KanMX, mCherry, Homologous Region Right (Week 2-5)
IV. Transformation of S.cerevisiae BY4742 with the        4 amplified fragments (Week 5)
V. Verification PCR of transformed colonies which        grew on selection Media (Week 5-6)
VI. Microscopy with Fluorescence Microscope (Week 6)
Results of these steps can be seen at 07/07 (Week 6)
AVP1: We successfully performed a       single-digest of our prepped pUC19 with EcoR1.
AVP1 Fragment Preparation: AVP1 Fragment Preparation:       We inoculated 2 overnight cultures of e.coli to prepare fragments for       the integration of AVP1. Overnight 1 contained an e.coli strain       carrying the EasyClone Plasmid (pO391). The other strain contained       the pGREGG566 Plasmid with the Gal1 Promoter that is part of one       fragment for the AVP1 integration.
ENA1: The plates of the transformation of ENA1 in BY4742 showed growth.       The wildtype control plate with G418 antibiotic showed growth too, though –       which led to the suspicion that the plates weren’t produced correctly, for       example with too little G418. Transformed cells, a control transformation      with a plasmid with kanMX and wildtype BY4742 were plated again to test this.      We also produced new G418 plates.
NHX1: A PCR-cleanup was performed with the four previously       amplified fragments for the NHX1 transformation.
06/07
ENA1: It was discovered that our strain was wrongly labelled and       already contained a G418 resistance – which explained the problems       with growth of the negative control. The plates turned out not to       be the problem.
NHX1: The transformation of NHX1 into BY4742 was performed       with the previously amplified fragments according to protocol.       This transformation had to be repeated later due to the discoveries       about our strain. To be able to repeat he transformation, the fragments       for it were to be amplified again. After the Gel, PCR cleanup was performed.
NHX1 PCR product:
AVP1 Fragment Preparation: We isolated the 2 plasmids from       the overnights (06/06) and determined the concentration (pGREGG566,       carrying Gal1 Promoter: 740 ng/µL; pCfB 391 EasyClone: 208,5 ng/µL).
LAB: We mixed 1L of Tris-HCl buffer and 1L of TE buffer.
06/08
AVP1 Fragment Preparation: We used the pGREGG566 plasmid      isolated the day before to amplify the Gal1 Promoter linear fragment.       Therefore, we carried out a PCR and Gelelectrophoresis afterwards.
All 3 PCR tubes showed bands at 500 bp, which is the expected length       for our Gal1 Promoter Fragment (462 + 40 bases overhang from the reverse       Primer -> 502 bases long. The negative control tube 4 with no template       (plasmid) showed no band at 500 bp.
We cleaned up the product and determined the concentration: 25,5 ng/L.
06/09
LAB:An overnight culture of E.coli containing a plasmid with eGFP was made.
ENA1:: The kanMX fragment for the transformation was amplified       again, as our product from the previous PCR was depleted. The transformation       was repeated because of the strain mix-up. Overnight culture of the E.coli       containing the plasmid with kanMX was made.
AVP1 Fragment Preparation: We carried out 2 PCRs and       2 Gelelectrophoresis to amplify the Upper Homologous Region +       Terminator (TADH1) = Fragment 1, and the Lower Homologous Region      + loxP + Marker + loxP = Fragment 4 from the EasyClonePlasmid pCfB 391.
Bought Fragments are necessary for the integration of AVP1       into the yeast genome.
Gel Fragment 1:
all 4 PCR tubes showed bands at around 900 bp, which is       what we expected (our Fragment 1 (Up and TADH1 Terminator)       is 896 bp). The PCR worked in the 50 µL (1,2) as well as in       the 25 µL (M,3) tubes, but not in the negative control. The       negative control tube K with no insert (Easy Clone Plasmid      p0391) showed no band at around 900 bp.
The PCR Product was cleaned up and concentration was determined afterwards.
Gel Fragment 4:
All 4 PCR tubes showed bands at around 2000+ bp, which is what we expected       (our Fragment 4 (Down loxP-Marker-loxP) is 2111 bp long). The PCR worked in       the 50 µL (1,2) as well as in the 25 µL (M,3) tubes, but not in the negative      control. The negative control tube K with no insert (Easy Clone Plasmid p0391)      showed no band at around 2000 bp.
The PCR Product was cleaned up and concentration was determined afterwards.
AVP1: We dephosphorylated puC19 and performed a       digestion of the AVP1-gene with EcoR1.
<div class="white100"></div>
Week 3
06/12
LAB: Plasmid Prep was done for the pUG6 and eGFP plasmids from the overnight cultures.
AVP1: After preparing our parts for the AVP1 back-up in       E. coli we finally started the transformation. First our insert       (AVP1) and our plasmid (pUC19) were ligated with T4 ligase,       we prepared two tubes: the ligation and a negative-control.
Ligation-Mix composition:
Gene:
AV
P1
Components
Ligation
Negative Control
Insert (AVP1)
6,56 µL
-
Backbone (pUC19)
4 µL
4 µL
T4 Ligase
2 µL
2 µL
Buffer
2 µL
2 µL
ddH2O
5,44 µL
12 µL
total
20 µL
20 µL
We incubated the two Eppis for 1h at room temperature and       performed a heat shock at 65°C afterwards.
For the transformation, we used E. coli BL21. We used the       transformation-protocol from NEB.
The transformed cells were spread on Amp-Plates. In addition,       we made a negative and a positive control.
BL21 with pUC19 + AVP1
BL21 with pUC19 (without insert)
wildtype
dilutions from BL21 with insert
06/13
ENA1: The kanMX fragment was amplified from the previously       prepared pUG6 plasmid. The gel shows two bands, the lower one being our long       primers (long due to overhangs).
kanMX production PCR:
AVP1: There was no growth on       the plates from the day before, so either the       ligation or the transformation didn't work.       We asked one of our advisors and they gave us       the tip to use a higher insert/backbone-ratio (9:1) and extend the       ligation time.
For our second ligation, we used the same parts as before.      We prepared 3 tubes.
Ligation-Mix composition:
Gene:
AVP1
Components
Ligation (3:1 ratio)
Ligation (9:1 ratio)
Negative Control
Insert (AVP1)
6,56 µL
9,84 µL
-
Backbone (pUC19)
4 µL
2 µL
4 µL
T4 ligase
2 µL
2 µL
2 µL
Buffer
2 µL
2 µL
2 µL
ddH2O
5,44 µL
4,16 µL
12 µL
After the ligation we performed the transformation and spread the cells on Amp-Plates.
06/14
LAB: A gradient PCR was done for eGFP to determine the annealing temperature.       Although an error was made when creating the primers, the PCR worked quite well.
Annealing Test:
AVP1: The transformation seemed successful.       There grew c20 colonies on the 9:1 ratio plate and c5 on the 3:1 ratio plate.
06/15
LAB:Now that the annealing temperature for the eGFP fragments was       determined, a production PCR could be performed. The product was cleaned.
production PCR result:
To test the E.coli with pUG6 and eGFP, LB plates with Aminopenicillin       for pUG6 and Kanamycin for eGFP were produced, and the bacteria plated       on one plate each.
06/16
LAB: In order to produce cryo cultures       for pUG6, eGFP and BY4742, we first have to produce overnights from only       one colony of each. We used the plates we produced on the day before and       the one we received from our advisor (Sandra) (BY4742).
Our overnights contained the following:
1. 1 pUG6 colony + 25 ml LB medium + 25 ul Amp (100%)
2. 1 eGFP colony + 25 ml LB medium + 25 ul Kan (100%)
3. 1 BY4742 colony + 25 ml YPD medium
4. 1 BY4742 colony + 25 ml YPD medium
AVP1: We checked the transformation with a colony-PCR. Therefore, we used the protocol 1.3 Check PCR from the script "Praktikum Molekulare Biotechnologie".
We checked 8 colonies from our 9:1 plate.
Check PCR composition:
Gene:
AVP1
Components
Volume for one Tube
Volume for ten Tubes
Final Concentration
ddH2O
15 µL
150 µL
-
5x Green GoTag Buffer
4 µL
40 µL
1x
Primer f (10µM)
0,2 µL
2 µL
0,1 µM
Primer r (10µM)
0,2 µL
2 µL
0,1 µM
dNTP-mix (10mM)
0,4 µL
4 µL
200 µM
GoTag Polymerase
0,2 µL
2 µL
2,5 U
Due to the different melting temperatures, we performed a gradient PCR (T: 57,5-62,0°C).
Gene:
AV
P1
PCR Step
Time
Temperature
ID Initial denaturation
10 min
94°C
D Denaturation
30 sec
94°C
A Annealing
30 sec
gradient (57,5-62,0°C)
E Extention
60 sec
72°C
FE Final Elongation
5 min
72°C
S Storage
unlimited
8°C
After the PCR we made a gel-electrophoresis.
Unfortunately, the results didn't meet our expectations,       there are bands between 250bp and 500bp, all at a different       height. If the transformation had been successful there       would be a 2300bp band.
<div class="white100"></div>
Week 4
06/19
LAB: Overnight cultures from 16.06.17 where used to       create cryo cultures. Cryo tubes of pUG6 and eGFP (both in E.coli)       were filled with 500µl culture + 500µl 50% glycerin and the cryo of       BY4742 with 500µl culture + 500µl 30% glycerin.
Competent BY4742 cryos were created according to the McClean protocol.
AVP1: To check if AVP1 has any other unexpected EcoR1-restriction sites,      we performed a digestion and a gel-electrophoresis afterwards. The       result was one single band with the expected size of c2300bp.
Because of our ambiguous results of the colony PCR we       decided to check our cloning via restriction analysis and       ordered the restriction enzyme Nde1 (cuts in AVP1 & pUC19).
06/20
NHX1: Production PCR for NHX1 was made following the program       from the 08.06.17.  The gel shows that the PCR worked out well. The bands fit       the length of the fragment, which was 1899bp.
Production PCR NHX1:
NHX1 with eGFP: PCR Products for NHX1 with eGFP overhangs and      terminator+marker+homologous sequence with eGFP overhangs were produced and PCR      cleanup was done.
ENA1: Transformation for the ENA1 knockout was performed.       Transformed cells and control were plated on YPD with G418.
AVP1: The restriction analysis showed, that the transformation       failed.      We talked to our instructor and she recommended us to use the       "CloneJET PCR Cloning-Kit" (blunt-end ligation) because we       had the fear, that the end digestion of AVP1 didn't work.      We performed a new transformation of AVP1 in E. coli DH5a.       Ligation mixture:
Gene:
AVP1
Component
Amount
2x Reaction Buffer
10 µL
Insert
7 µL
pJET 1.2
1µL
T4 DNA Ligase
1 µL
ddH2O
1 µL
20 µL
06/21
LAB: The G418 plates were tested by plating BY4742 wildtype on our YPD+G418 plates.
To test our strains, two overnight cultures were made with BY4742       wildtype, one with G418 and one without.
NHX1: The transformation for the overexpression of NHX1 was performed.       The competent cells were tested using a plasmid with kanMX G418 resistance.
ENA1: With a rest of PCR product, PCR cleanup was performed.
AVP1: We transformed the ligation-product       and a control-ligation in DH5a, using the protocol from the      NEB-Kit "NEB 5-alpha competent E. coli".
06/22
LAB: We made 3 liters of 1,2% Agarose Gel for use in Gel Electrophoresis
NHX1: Product PCR of NHX1, X4-Up homologous sequence,       Gal1 promotor and x4-Down homologous seuqnce with selection marker was      repeated to replenish our stock of the fragments.
The gel showed that the PCR was successful:
PCR-cleanup was performed with the products.
AVP1: The transformation didn't succeed.
06/23
NHX1: The transformation of NHX1 into BY4742 wildtype       seemed to work, as colonies started to show on the plates       and the control transformation showed no growth. The cells       looked very wet and yellow-ish, wish led us to the conclusion       that they weren’t yeast cells. Microscoping the cells and comparing       them to reference yeast cells confirmed our suspicion that our       transformation was contaminated.
AVP1: To perform another transformation,       we amplified the AVP1 gene via PCR.
Composition of 100µL master mix:
PCR-programm:
Gene:
AVP1
Component
Amount
B2 PfuS Buffer
10 µL
Primer f
8 µL
Primer r
8 µL
dNTP-mix
2 µL
Polymerase (PfuS)
1 µL
Template (AVP1)
2,2 µL
ddH2O
71 µL
Gene:
AV
P1
PCR Step
Time
Temperature
ID Initial denaturation
2 min 20 sec
98°C
D Denaturation
20 sec
98°C
A Annealing
30 sec
61°C
E Extention
2 min 30 sec
72°C
FE Final Elongation
3 min
72°C
S Storage
unlimited
8°C
The PCR was successful. [DNA]= 18ng/µL
<div class="white100"></div>
Week 5
06/26
NHX1: To have some backup material, we decided to make some       cryo cultures for our transformed NHX1 overexpression cells. We made 5 cryo       cultures in 30% glycerin solutions and stored them in a -80°C container.
We suspected that our transformation was contaminated because of a rather       unusual smell for a yeast culture. Using a microscope, we found out that our       cultures were in fact e. colis. Several colonies were tested, each with the       same misfortune.
Our cryo cultures were discarded.
ENA1: After several days, no growth was observed on our       transformation plate. We decided to use a method called “Abschwämmen”.       A small amount of YPD+G418 medium is added to the plate. Using a sterile       toothpick, the colonies are carefully picked and dissolved in the medium.       This medium was then pipetted into a new YPD+G418 plate.
We tested our SC-Histidine free plates and concluded they were functional.
AVP1 Fragment Preparation:To refill our stock of       Gal1 promoter we repeated the PCR carried out on 06/07:
We used the pGREGG566 plasmid isolated the day before to      amplify the Gal1 Promoter linear fragment. Therefore, we carried out a       PCR, which failed.
AVP1: Because all teams had problems with the PCRs       we performed a PCR-test series to check the polymerases. Our suspicions       were confirmed and we had to throw some of the tubes away.
Simultaneously to the PCRs we performed a second transformation with AVP1       ligated in pJET. this time we used a AVP1 with a higher       concentration (18 ng/µL instead of 9 ng/µL).
06/27
NHX1: Due to the contaminated plates we had to discard,       we made a new transformation. In order to test our plates, we did a test       transformation using a KanMX plasmid.
LAB: New SC plates were produced, histidine free.
AVP1: This time 5 colonies grew on the plate.       We performed a colony PCR to check the transformation success.
Result:
We prepared overnight cultures for a restriction analysis.
06/28
AtHENA: We produced three different overnights; two different       knockout strains and one wild type.
AVP1: We performed a second colony PCR and changed       the annealing temperature to 60°C.
In addition, we performed a restriction analysis with Nde1.
Both methods showed, that the transformation failed. Because       the saving of AVP1 in E. coli didn't seemed very promising in the       future we decided to skip this part and begin with the transformation in yeast.
06/29
AVP1: We performed another PCR to amplify all 4 fragments for the       integration of AVP1 into S.cerevisiae BY4742 which failed.
LAB: Selection media without histidine and       without uracil were produced.
ATHENA: We resuspended our new primers for KanMX extraction       in TE – Buffer. Overnights from our plate #2 were made. The antibiotic G418       was added as selective marker.
06/30
<div class="white100"></div>
Week 6
07/03
AVP1 Fragment Preparation: To refill our stock of Gal1       promoter we repeated the PCR carried out on 06/07 :
We used the pGREGG566 plasmid isolated the day before (      to amplify the Gal1 Promoter linear fragment. Therefore, we       carried out a PCR, which failed.
NHA1: We dissolved our new primers in DMSO with a 25 mMolar       solution. We produced new YPD Medium because of a contamination. These new       primers were used in a gradient PCR to produce the right and left arms beside       the NHA1 gene. A verification PCR for ENA1 was also introduced in our gradient       PCR.
This where our results:
The NHA1 PCRs did not work.
ENA1: We decided to prepare some overnight cultures. Overnight       #1 contained G418 (antibiotic) and the ENA1 transformation, overnight #2       contained G418 (antibiotic) and wild type and overnight #3 with contained       YPD and the ENA1 transformation.
07/04
AVP1 Fragment Preparation: We repeated the PCR frp, 07/03.       This time the gel showed 1 intense band at around 500 bp, which fits the Gal1       fragment length (502 bp). The product was therefore cleaned up and the       concentration was determined.
07/05
AVP1 Fragment Preparation: We carried out PCR to amplify      the codon optimized gene from IDT. The test gel showed 1 intense band at       around 2350 kb, which fits the length of the AVP1 Gene (2375 bp in length).       The product was cleaned up using PCR clean-up and concentration was determined      via NanoVue afterwards (123 ng/µL).
Furthermore, we repeated the PCRs from 06/09 to refill our stock of       Homologous Region Fragments for the genome integration.
LAB:We wanted to use a FACS Machine to do a cell analysis. To do this,       we needed to produce PBS medium for this analysis and as a medium for       fluorescent microscopy.The pH was set to 7.4 using NaCl. After this the       overall volume was increased to 1000 ml using cold ddH2O and autoclaved.
NHA1:We found the results from yesterday´s gradient PCR       somewhat strange. We decided to rapidly test the template DNA with an       electrophoresis. This were our results:
No template was found in our NHA1 left and right arms. That wild type BY4742       genome was disposed of and new genome was isolated from a new overnight culture.      Another reason why our gradient PCR wasn´t working was the solution in which       the primers were dissolved in, DMSO. A primer solution, whichever the       concentration is, should never include more than 3.5 % DMSO, otherwise       the polymerase will react with said solute and denaturate.
07/07
mCherry Integration Results: The last weeks we worked on transforming mCherry into S.cerevisiae       BY4742 to create the fusion protein ATG22-mCherry. Amplification of the 4 fragments (Homologous Region       Left, KanMX, mCherry, Homologous Region Right) was successful, the Transformation as well. Verification       PCRs of the integrated fragments showed, that mCherry was integrated correctly, however microscopy did not       show fluorescence at all (Used Laser Wavelength was: 561). Control of Codon Usage showed, that mCherry is       Codon Optimized for E.coli and therefore not expressed correctly. We will withdraw mCherry and use a       S.cerevisiae codon optimized mRFP Gene from now on.
<div class="white100"></div>
Week 7
07/10
ENA1:Since we had many different mutant colonies where our       transformation did seem to work, we decided to store them in case something       went wrong along the way. We produced 10 different overnights for ENA1 K.O       transformations and wild type BY4742.
Interlab Study:We transformed samples of competent E. coli       cells. We transformed with three different controls; one negative, one positive       and another positive with a RFP marker. We produced 30 LB plates containing       chloramphenicol from which 18 plates were used.
07/11
AVP1: We created his-drop out medium (plates)       for our transformation of AVP1 in yeast.
NHA1:We produced some YPD medium flasks for later use.       We proceeded to new gradient PCRs (10 samples for each) containing XII-Up       (left arm), XII-Down (right arm) and the URA3 + loxP marker.
The following gel electrophoresis shows our results:
The XII-Up PCR did not work. The temperatures that barely seem to work for       X-Down were 49.3 °C, 51.2 °C, 53.3 °C and 57.2 °C.
Interlab Study:On all our plates, some E. coli colony       growth was observed. However, we could only count the colonies on 4 of       these plates. Because of this we decided to replate on LB + Chloramphenicol       most of the E. colis on new plates to obtain single colonies.
07/12
AVP1: For the transformation, we first performed a       PCR of fragment 3 (Gal1-Promotor). After that we transformed our       fragments into Saccharomyces cerevisiae. 3 tubes were made:
yeast + fragments 1-4
yeast + control plasmid
yeast from the other group for testing + control plasmid
Gene:
AVP1
Component
Tube 1
Tube 2
Tube 3
Fragment 1 (TADH1)
11,8 µL
-
-
Fragment 2 (AVP1)
12,5 µL
-td>
-
Fragment 3 (Gal1)
13,9 µL
-
-
Fragment 4 (his-aux-marker)
7,3 µL
-
-
plasmid p0391
-
0,96 µL
0,96 µL
ddH2O
-
33,04 µL
33,04 µL
The transformation-mixture contained 720µL 50% PEG 350;       108 µL 1M LiOAc and 150µL carrier DNA (salmon sperm). For       the transformation, we used the protocol: "Heatshock transformation S.       cerevisiae (modified after Joska al. 2014)" from Sandra.
NHA1:A gradient PCR was made for the XII-Up, XII-Down and       URA3+ loxP exactly as yesterday. The results of the gel electrophoresis       show that no product was present in our mixtures.
Interlab Study:We wanted to be sure our LB plates weren’t       working well so we tested them. We plated E. coli dH5d and E. coli with       PuG6 on our plates.
07/13
mRFP Labelling: Since the expression of mCherry failed due to wrong Codon usage        we are repeating the labeling with a codon optimized mRFP.
We performed PCRs of the for the Transformation needed fragments 1 (Left Homologous Region),        3 (mRFP) and 4 (Right Homologous Region). Fragment 2 did not change.
PCR Mix:
Gene:
Left Homologous Fragment
mRFP
Right Homologous Fragment
Component
µl
µl
µl
Template
1,314
0,348
1,104
Buffer
15
15
15
Primer f
12
12
12
Primer r
12
12
12
dNTP
3
3
3
Polymerase (PfuS)
1,5
1,5
1,5
ddH2O
105,2
106,15
105,4
PCR Programm:
Gene:
Left Homolo
gous Fragment
mR
FP
Right Homolo
gous Fragment
PCR Step
Time
Temperature
Time
Temperature
Time
Temperature
ID Initial denaturation
180 sec
98°C
180 sec
98°C
180 sec
98°C
D Denaturation
20 sec
98°C
20 sec
98°C
20 sec
98°C
A Annealing
40 sec
58°C
40 sec
63°C
40 sec
53°C
E Extention
60 sec
72°C
60 sec
72°C
60 sec
72°C
FE Final Elongation
5 min
72°C
5 min
72°C
5 min
72°C
S Storage
unlimited
8°C
unlimited
8°C
unlimited
8°C
Afterwards we analyzed the PCRs via a Gel Electrophoresis,        performed a PCR Clean-Up and determined the DNA-concentration via NanoVue.
Gel:
ENA1: We did a verification PCR using the isolated genome        of our ENA1 mutated colonies. We then followed this with a gelelectrophoresis.
NHA1: Due to the crazy results from the past days,        we decided to do 2 separate PCRs. One production PCR for the XII-Down        and URA3 + loxP fragments and one gradient PCR for the XII-Up fragment.        We also used the new primers (now diluted in TE buffer to a 25-mM        concentration) for every PCR. The following gel shows our results:
URA3 + loxP and XII-Down seem to work fine. Nonetheless XII-Up gradient PCR showed        now promising bands. Both successful products were cleaned up and stored.
AVP1: We created competent S. cerevisiae BY4742,       using the protocol "McClean: Making and Using Frozen Yeast Competent Cells".
07/14
mRRP Labelling: We transformed S.cerevisiae BY4742 with the 4 mRFP Fragments       to integrate mRFP into the genome. YPD-Plates containing Genticin       were used as a selection marker.
DNA-Mix:
Gene:
mR
FP
Fragment
m needed [ng]
V needed [µl]
Homologous Region Left
1000
23,81
KanMX
500
10
mRFP
13,42
1000
Homologous Region Right
1000
23,81
H2O
0
0
Trafo-Mix:
Trafo:
mRFP into Wildtype (WT)
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
AVP1: Transformation results (07/12): Because there was growth      on the plates with the transformed yeast, the transformation seemed to be      successful.
We isolated the genome of 5 colonies from plate 1 for further tests.
NHA1: The only fragment missing to continue was XII-Up.       We followed a production PCR for XII-Up with a PCR clean up while we waited       for our gel electrophoresis.
Our products did contain the wished fragment size.
<div class="white100"></div>
Week 8
07/17
mRFP Labelling: Check of Transformation done at 07/14. 4 colonies grew. No fluorescence under UV-Lamp. Verficiation PCR needed.
AVP1: Our transformed yeast cells were contaminated       with E. coli, so we performed a new transformation. This time we used       the competent yeast cells we made the week before. The amounts of the       four DNA-fragments were the same as last time (tube 1). Again, we used       the protocol "Heat shock transformation S. cerevisiae".
NHA1: We produced 2 liters x1 PBS-Buffer (100 ml x10 PBS per 900 ml ddH2O).
07/18
mRFP Labelling: Check of Transformation done at 07/14. 4 colonies grew. No fluorescence under UV-Lamp. Verficiation PCR needed.
NHA1: We finally started with our transformation tests.       Our NHA1 transformed cells and wild type BY4742 were then plated on a URA-       plates. We also produced some PEG 50% 3350 solution for further yeast       transformations.
ENA1: We did a verification PCR for our ENA1 K.O followed       by a gel electrophoresis. No bands were observed.
NHX1: To check if our NHX1 overexpression transformation       we did a verification PCR with our isolated genome. None of our products worked.
AVP1: We successfully amplified our 4 fragments for further transformations.
07/19
mRFP Labelling: Gel Electrophoresis of the PCR Products was unsuccessful.       We did another PCR with the right and left parts of the integrated fragments,       which failed as well. Therefore, mRFP was not integrated into BY4742 during       Transformation.
AVP1: We made competent Ena1-kock-out BY4742       mutants and performed a transformation with these cells to       integrate AVP1. We used the same amounts of fragment as before.
NHX1: Since our verification PCR did not work yesterday       we decided to isolate a new genome and use it for a new verification PCR.       This again did not work.
07/20
mRFP Labelling: We performed another Transformation of BY4742 with the 4 mRFP Fragments       (Left Homologous Fragment, KanMX, mRFP, Right Homologous Fragment) as done at 07/14.
DNA-Mix:
Gene:
mR
FP
Fragment
m needed [ng]
V needed [µl]
Left Homologous Region
500
7,1
KanMX
250
5
mRFP
500
6,71
Homologous Region Right
500
11,9
H2O
4
Trafo-Mix:
Trafo:
mRFP into WT
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
AVP1: To check the success of our transformation (07/17)       we performed a genome-isolation and a check-PCR with 5 of the c50 colonies.       For the genome-isolation we used the protocol "Quick and Dirty       (Mans et. al 2015)" from our advisor Sandra.
(The check-PCR failed.)
NHA1: We needed some XII-Up fragments again. Instead       of doing a gradient PCR, we decided to set our temperature to 47.7 °C       and make larger mixtures. Our gel electrophoresis showed the result of said PCR.
Our products were cleaned up and used for a new NHA1 transformation       (transformation #2). The transformed yeast cells were again plated       (along with a wild type) on URA+ plates.
07/21
NHA1:After not seeing any culture growth in the past days,       we decided to do a third transformation just to be sure. We plated our       transformation and wild type on URA- plates.
Our XII-Down and XII-Up fragments were accidentally thrown away,       therefore we needed to produce more. Since we already knew the       ideal annealing temperatures for our fragments, we decided to do      a production PCR. We analyzed our products with a gel electrophoresis.       The results showed that none of the PCR products worked.
<div class="white100"></div>
Week 9
07/24
mRFP Labelling: 40 colonies grew on the Transformation from 07/20. We isolated       the Genomes and performed Verification PCRs       of 5 colonies. Primers included both left and right part of the integrated       fragments and additional 100bp left and right of the integration sites.
PCR Mix:
Strain:
BY4742+mRFP
Component
µl
Template
1,5 per Tube
Buffer
5
Primer f
2
Primer r
2
dNTP
1
Polymerase (PfuS)
0,5
ddH2O
37,2
PCR Programm:
Strain:
BY474
2+mRFP
PCR Step
Time
Temperature
ID Initial denaturation
180 sec
98°C
D Denaturation
15 sec
98°C
A Annealing
30 sec
57°C
E Extention
50 sec
72°C
FE Final Elongation
5 min
72°C
S Storage
unlimited
8°C
AtNHXS1: We want to integrate the Arabidopsis Thaliana vacuolar Sodium/Proton Antiporter AtNHXS1 into         S.cerevisiae. Therefore, we use the Easy Clone Plasmid pCfb257, carrying the TCYC1 Terminator, Integrationsides         for Chromsome X-3 and a Leucin Marker. We received E.coli DH5alpha carrying pCfb257 on a plate containing Ampicillin         and incubated it. Additionally we performed PCRs to amplify the Fragments 2 (Gal1 Promoter) and 3         (AtNHXS1 synthesized by IDT) for Transformation, which failed.
AVP1: We repeated the check-PCR. It failed again.
We isolated another five genomes from different colonies (BY4742+AVP1 & Ena1-kock-out + AVP1).
07/25
mRFP Labelling: Gel Electrophoresis of the Verification PCRs from 07/24. Colonies 2 showed       correct bands. We microscoped colonies 2 and 4 other colonies (6, 7, 8 and 9) which had not       been PCR verified to check fluorescence (Laser wavelength: 561nm). Whereas colony 2, 7, 8 and       9 showed no to small fluorescence, colony 6 showed high fluorescence. 20 other microscoped       colonies showed fluorescence as well. We will go on working with colony 6.
AtNHXS1: We prepared an overnight culture of E.coli DH5alpha carrying pCfB257 to amplify the plasmid.       Additionally, we performed another PCR to amplify Fragments 2 and 3, which failed.
AVP1: We performed another cPCR. It failed again.
Because all our PCRs failed, we received a different polymerase (GoTag)       from our advisor and repeated the cPCR. We had 12 tubes.
Gelelectrophoresis:
1, 8: ladder
2, 3, 4: cPCR AVP1
5, 6, 7: cPCR Ena1
9-14: control
07/26
AtNHXS1: We performed a plasmid prep of the E.coli+pCfB257 Overnight to isolate pCfB257.       We performed PCRs of all 4 Fragments (Integrationside Up, Gal1, AtNHXS1,       Terminator-Leucin Marker-Integrationside Down)
PCR Mix:
Gene:
Fragment 1: Up
Fragment 2: Gal1
Fragment 3: AtNHXS1
Fragment 4: T-Down
Component
µl
µl
µl
µl
Template
0,52
0,34
1,5
0,52
Buffer
25
25
25
25
Primer f
20
20
20
20
Primer r
20
20
20
20
dNTP
5
5
5
5
Polymerase (PfuS)
2,5
2,5
2,5
2,5
ddH2O
177
177,2
176
177
PCR Programm:
Gene:
Fragme
nt1:Up
Fragment
2:Gal1
Fragment3
:AtNHXS1
Fragment
4:T-Down
PCR Step
Time
Temperature
Time
Temperature
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
98°C
30
98°C
30 sec
98°C
30 sec
98°C
D Denaturation
8 sec
98°C
8 sec
98°C
8 sec
98°C
8 sec
98°C
A Annealing
20 sec
59°C
20 sec
61°C
30 sec
63°C
30 sec
58°C
E Extention
20 sec
72°C
20 sec
72°C
60 sec
72°C
30 sec
72°C
FE Final Elongation
5 min
72°C
5 min
72°C
5 min
72°C
5 min
72°C
S Storage
unlimited
8°C
unlimited
8°C
unlimited
8°C
unlimited
8°C
Afterwards we performed a Gel Electrophoresis, a PCR-Clean Up and determined the DNA-concentrations of the fragments.
Gel:
AVP1: We checked the other homologies recombination       locus of the tested Ena1-mutants and both loci of the AVP1-mutant.
Gel-electrophoresis
1, 16: ladder
2, 3, 4, 5: posterior homologues region
6, 7, 8 ,9: AVP1+Gal
10-13: control
14, 15: anterior homologues region
Both strains integrated the homologues DNA-sequences.
We performed another PCR to prove the right order of the integrated fragments.
Gel-electrophoresis
07/27
mRFP: We performed Verification PCRs to verify the integration of mRFP into colony 6        which was fluorescing on 07/25. The PCRs failed. We made an overnight of colony 6 in YPD containing G418.
AtNHXS1: We transformed the 4 amplified AtNHXS1 Fragments into BY4742 WT and BY4742 containing        Ena1 Knock-Out. We used SC-Plates without Leucin as selection media.
DNA Mix:
Gene:
AtNH
XS1
Fragment
m needed [ng]
V needed [µl]
Up
500
4,46
Gal1
500
4,33
AtNHXS1
500
3,38
T-loxP-Leu-loxP-Down
350
4,76
H2O
17,07
Trafo Mix:
Trafo:
AtNHXS1 into WT & Ena-Mutant
Component
Amount [µl]
50% PEG 3350
480
1M LiAC
72
Carrier DNA
100
AVP1: The integration of AVP1 in the Ena1-knockout-mutant was successful, the integration of AVP1 in the wildtype failed.
We isolated the genome of 8 other colonies and performed a cPCR.
Gelelectrophoresis:
07/28
mRFP: We performed new PCRs to verify the integration of mRFP into BY4742. We checked the left        and right part of the fragments and additionally 100bp on each side of the fragment of the genome.
PCR Mix:
Gene:
Left Part
Right Part
Component
µl
µl
Template
1
1
Primer f
0,2
0,2
Primer r
0,2
0,2
One Taq 2x Mastermix
5
5
ddH2O
3,4
3,4
PCR Programm:
Gene:
Left
Part
Right
Part
PCR Step
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
94°C
30 sec
94°C
D Denaturation
15 sec
94°C
15 sec
94°C
A Annealing
25 sec
57°C
25 sec
59°C
E Extention
75 sec
68°C
120 sec
68°C
FE Final Elongation
5 min
68°C
5 min
68°C
S Storage
unlimited
8°C
unlimited
8°C
Afterwards we performed a Gel Electrophoresis.
Gel:
We made competent cells of BY4742 + mRFP from the overnights from 07/27.
AVP1: We checked the 5 colonies that contained the upper        homologues sequence. They didn't integrated the other fragments.
<div class="white100"></div>
Week 10
07/31
AtNHXS1: Colonies grew on AtNHXS1 Transformations from 07/27. We performed a Genome Isolation and verification PCR (Left and Part of Fragments, 100bp of Genome on left side) of 5 colonies.
PCR Mix:
Gene:
AtNHXS1
Component
µl
Template
1 per Tube
Primer f
2
Primer r
2
OneTap 2x Mastermix
50
ddH2O
36
PCR Programm:
Gene:
AtNH
XS1
PCR Step
Time
Temperature
ID Initial denaturation
30 sec
94°C
D Denaturation
15 sec
94°C
A Annealing
25 sec
56°C
E Extention
110 sec
68°C
FE Final Elongation
5 min
68°C
S Storage
unlimited
8°C
We performed a Gel Electrophoresis (WT left, Ena middle, Positive Control right)
We prepared Overinghts of WT+AtNHXS1 (Colony 1) and Ena+AtNHXS1 (Colony 3) to make cryos and competent cells.
AVP1: None of the colonies we checked the day before integrated all of the four fragments.
Therefore, we isolated 12 others from the same plate and performed a check      -PCR.
08/01
AtNHXS1: We performed vericifaction PCRs of the left and right       side of WT+AtNHXS1 (Colony 1) and Ena+AtNHXS1 (Colony 3). Primers       included whole integrationsite and 100bp Genome on each side.
PCR Mix:
Gene:
AVP1
Component
µl
Template
1 per Tube
Primer f
0,4
Primer r
0,4
One Taq 2x Mastermix
10
ddH2O
7,2
PCR Programm:
Gene:
Up-At
NHXS1
T-D
own
PCR Step
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
94°C
30 sec
94°C
D Denaturation
15 sec
94°C
15 sec
94°C
A Annealing
25 sec
56°C
25 sec
57°C
E Extention
110 sec
68°C
200 sec
68°C
FE Final Elongation
5 min
68°C
5 min
68°C
S Storage
unlimited
8°C
unlimited
8°C
Gel Electrophoresis was successful for both strains (Left WT, middle Ena, right positive control)
Gel:
We made 25 competent cryos per strain (WT+AtNHXS1 and Ena+AtNHXS1) from the overnights made 07/31.
AtNHXS1-eGFP: We are planning to label AtNHXS1 with eGFP to colocalize it with ATG22-mRFP to the       vacuolar membrane. We performed a PCR to amplify Fragment 3 (AtNHXS1 with      Overhang for eGFP) and Fragment 5 (Failed).
PCR Mix:
Gene:
AtNHXS1-eGFP
Component
µl
Template
0,6
Buffer
10
Primer f
4
Primer r
4
dNTP
2
Polymerase (PfuS)
1
ddH2O
70,8
PCR Programm:
Gene:
AtNH
XS1
PCR Step
Time
Temperature
ID Initial denaturation
30 sec
98°C
D Denaturation
8 sec
98°C
A Annealing
30 sec
63°C
E Extention
60 sec
72°C
FE Final Elongation
5 min
72°C
S Storage
unlimited
8°C
We performed a Gel Electroophoresis (Fragment 3 left, 5 right), Clean-Up and determined the DNA-concentration.
AVP1: Because the last cPCR didn’t work, we performed a       ramp PCR (AVP1 in Ena1 mutant). We prepared two mastermixes, split it in two       tubes and added 1uL template DNA each.
Gene:
AVP1
Component
Mix 1
Mix 2
Template
1 µL
1 µL
OneTag Mix
10 µL
10 µL
Primer f
0,4 µL
0,4 µL
Primer r
0,4 µL
0,4 µL
ddH2O
9,2 µL
9,2 µL
PCR-program:
Gene:
AV
P1
PCR Step
Time
Temperature
ID Initial denaturation
30 sec
94°C
D Denaturation
20 sec
94°C
A Annealing
30 sec
ramp 45-55°C
E Extention
2 min 50 sec
68°C
FE Final Elongation
5 min
68°C
S Storage
unlimited
8°C
Gelelectrophoresis:
The gelelectrophoresis showed that the transformation was sucessful.
In addition, we performed a cPCR with the AVP1 in wildtype genome.       It showed that this transformation failed.
08/02
AtNHXS1: We made cryos of WT+AtNHXS1 and Ena+AtNHXS1 from the Overnights made 07/31.
AtNHXS1-eGFP: We performed another PCR to amplify Fragment 5 (T-Down with eGFP Overhang).
PCR Mix:
Gene:
T-Down
Component
µl
Template
0,1
Buffer
10
Primer f
5
Primer r
5
dNTP
2
Q5 High-Fidelity DNA Polymerase
1
ddH2O
69,9
PCR Programm:
Gene:
T-D
own
PCR Step
Time
Temperature
ID Initial denaturation
30 sec
98°C
D Denaturation
8 sec
98°C
A Annealing
25 sec
66°C
E Extention
90 sec
72°C
FE Final Elongation
2 min
72°C
S Storage
unlimited
8°C
We performed a Gel Electrophoresis, Clean-Up and determined the DNA-concentration.
Gel:
AVP1: We performed a new transformation to integrate AVP1       in wildtype using the protocol “Heat shock transformation S. cerevisiae”       from Sandra.
08/03
AtNHXS1-eGFP: We performed a Transformation of AtNHXS-eGFP (5 Fragments) into BY4742 mRFP. No colonies grew, the Transformation failed.
LAB: We prepared the following Buffers for isolation of vacuoles: YPD-Media, Spheroplasting Buffer, DEAE-dextran solution, PS-Buffer with 4%, 8% and 15& Ficoll.
AVP1: We performed three PCRs to amplify our fragments for the transformation.
08/04
LAB: We performed a PCR to amplify the homologous region Down of the BY4742 genome for the Knock-Out of NHA1. The PCR failed.
AVP1: Transformation results (08/02): 21 colonies had grown on our plate. We isolated the genome and performed a check-PCR
<div class="white100"></div>
Week 11
08/07
AtNHXS1-eGFP: We performed another Transformation of AtNHXS1-eGFP into BY4742 mRFP.
DNA Mix:
Gene:
AtNHXS
1-eGFP
Fragment
m needed [ng]
V needed [µl]
Up
500
4,46
Gal1
500
4,33
AtNHXS1
500
7,87
eGFP
500
7,75
T-Down
350
5,98
H2O
3,61
Transformation Mix:
Trafo:
AtNHXS1-eGFP into BY4742 mRFP
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
AtNHXS1: We performed a Transformation of AtNHXS1 into AVENA (WT+Ena+AVP1)
DNA Mix:
Gene:
AtNH
XS1
Fragment
m needed [ng]
V needed [µl]
Up
500
4,46
Gal1
500
4,33
AtNHXS1
500
3,38
T-Down
350
4,76
H2O
17,07
Transformation Mix:
Trafo:
AtNHXS1 into BY4742 AVENA
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
08/10
Biobrick Assembly AtNHXS1: We will integrate AtNHXS1 as a part into the Biobrick Plasmid pSB1C3. We performed a Restriction Digest of AtNHXS1 and pSB1C3 with EcoRI and PStI.
Digestion Mix:
Gene:
pSB1C3
AtNHXS1
Component
µl
µl
Template
4
2
Buffer Cutsmart (19x)
1
1
EcoRI-HF
0,5
0,5
PstI-HF
0,5
0,5
ddH2O
4
6
We performed a PCR Clean-Up, determined the DNA-concentrations performed a Ligation of Backbone and Insert.
Ligation Mix:
Gene:
AtNHXS1 into pSB1C3
Component
µl
Backbone (25ng)
5
Insert (90ng)
18
Buffer
1
T4 DNA Ligase
1
We transformed the Ligation Mixture into E.coli DH5alpha and transformed the backbone as a negative control. Selection on LB-Plares with Cloramphenicol.
08/11
AtNHXS1-eGFP: We microscoped the transformed colonies of AtNHXS1-eGFP into BY4742 mRFP       (Transformation: 08/07). Only around 1% of the cells fluoresced red. Every cell that fluoresced red, fluoresced green as well.       Resolution was too low to colocalize transporters.
<div class="white100"></div>
Week 12
08/14
AtNHXS1: We performed verification PCRs       of the left part (+100bp from Genome) of 6 transformed       colonies BY4742 AVENA+AtNHXS1 (ATAVENA).
PCR Mix:
PCR Program:
Gene:
ATAVENA
Component
µl
Template
1 per Tube
Primer f
1,2
Primer r
1,2
One Taq 2x Mastermix
30
ddH2O
21,6
Gene:
ATAV
ENA
PCR Step
Time
Temperature
ID Initial denaturation
30 sec
94°C
D Denaturation
15 sec
94°C
A Annealing
25 sec
56°C
E Extention
110 sec
68°C
FE Final Elongation
5 min
68°C
S Storage
unlimited
8°C
Gel Electrophoresis was successful for colonies 1 and 2.
AtNHXS1-eGFP: AtNHXS1-eGFP: Verification       PCRs of 7 growing colonies failed.
AVP1: We prepared an overnight-culture to create competent       cells, using the protocol McClean: Making Yeast competent cells.
08/15
AtNHXS1: We performed Verifiecation PCRs of both the left       and right part of the Fragments (+100bp of the Genome on       each side) of colonies 1 and 2.
PCR Mix:
PCR Program:
Gene:
Left
Right
Component
µl
µl
Template
1 per Tube
1 per Tube
Primer f
0,4
0,4
Primer r
0,4
0,4
One Taq 2x Mastermix
10
10
ddH2O
7,2
7,2
Gene:
Le
ft
Rig
ht
PCR Step
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
94°C
30 sec
94°C
D Denaturation
15 sec
94°C
15 sec
94°C
A Annealing
25 sec
56°C
25 sec
58°C
E Extention
110 sec
68°C
200 sec
68°C
FE Final Elongation
5 min
68°C
5 min
68°C
S Storage
unlimited
8°C
unlimited
8°C
Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
AtNHXS1-eGFP: We performed another verification PCR of the       7 growing colonies, which failed, the Transformation was unsuccessful.
BioBrick AtNHXS1-eGFP: We made Overnights of the 8 grown       colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
AVP1: We completed the making of the competent cells.
08/16
AtNHXS1-eGFP: We performed another Transformation of       AtNHXS1+eGFP into BY4742 mRFP with a new eGFP Fragment.
DNA Mix:
Trafo Mix:
Gene:
AtNHXS
1-eGFP
Fragment
m needed [ng]
V needed [µl]
Up
500
4,46
Gal1
500
4,33
AtNHXS1
500
7,87
eGFP
500
5,56
T-Down
350
5,98
H2O
1,42
Trafo:
AtNHXS1-eGFP into BY4742 mRFP
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
Biobrick AtNHXS1: We made a Plasmid Prep and Cryos       of 8 colonies with the Overnights from 08/15. Afterwards we performed      a Single Digest of the Plasmids with EcoRI.
Digestion Mix:
Gene:
AtNHXS1-pSB1C3
Component
µl
Cut Smart Buffer
1
Template DNA
5
EcoRI
1
ddH2O
3
We performed a PCR Clean-Up. Gel Electrophoresis       was successful for colonies 1-5 and 8.
LAB:We made Overnights of several mutants       (WT,Ena,Avp,Avena,AtNHXS1,AtEna,AtAvena) for Cryos, Galactose      Growth Check, Genome Isolation and making competent cells.
08/17
AtNHXS1: We performed a Transformation of AtNHXS1 into AVP (AVPA).
DNA Mix:
Trafo Mix:
Gene:
AtNH
XS1
Fragment
m needed [ng]
V needed [µl]
Up
500
4,46
Gal1
500
4,33
AtNHXS1
500
3,38
T-Down
350
4,76
H2O
17,07
Trafo:
AtNHXS1 into BY4742 AVP
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
We made 10 eppis competent cells of ATAVENA from the Overnight of 08/16.
AVP1: We performed a transformation to integrate AVP1 together with eGFP in our mRFP mutant. We used the same protocol as always.
After the plate was incubated at 28°C for 4 days, 6 colonies grew.
08/18
Galactose Growth Check: We plated 6 different       strains (WT, AVP, AVENA, AtNHXS1, AtENA, AtAVENA)       on YPD-Agar containing 60%, 80% and 100% Galactose as carbon source.
LAB: We made Cryos of 7 BY4742 strains (WT, ENA,       AVP, AVENA, AtNHXS1, AtENA, AtAVENA). We isolated the genome of 6       BY strains (AVP, AVENA, AtNHXS1, AtENA, AtAVENA, mRFP) and dterermined       the DNA-concentrations.
AtNHXS1: We performed PCRs to amplify template DNA for       Sequencing of AtNHXS1 of the strains AtNHXS1, AtENA, AtAVENA (Promoter –       40bp of Terminator) and additionally BY mRFP. We want to sequence 500bp       of ATG22 and the whole mRFP gene. The PCRs failed.
<div class="white100"></div>
Week 13
08/21
AtNHXS1: We performed new PCRs to amplify the       templates for sequencing of the AtNHXS1 part of AtNHXS1, AtENA,       AtAVENA and additionally BY mRFP. The PCRs failed.
AVP1: (We checked our transformation of the mRFP mutant. It failed.)
08/22
AtNHXS1: We performed new PCRs to amplify the       templates for sequencing of the AtNHXS1 part of AtNHXS1, AtENA,       AtAVENA and additionally BY mRFP. The PCRs failed.
AVP1: We send the genome fragments of our transformed wildtype      and the Ena1-knock-out mutant to eurofins to get it sequenced.
We repeated the transformation with AVP1 + eGFP with a higher AVP1       concentration. The used protocol we used was the same as always.
08/24
AVP1: We got our sequencing results: the integration of AVP1 in      wildtype was successful, the AVP1 gene in the Ena1-kock-out had a point-mutation. (The transformation of the mRFP mutant with AVP1 + eGFP failed.)
<div class="white100"></div>
Week 14
08/28
AVP1: We performed a PCR to amplify our fragments.
08/29
AVP1: We integrated the AVP1 fragment into four different mutants form the other teams:
ATENA (AtNHXS1 + Ena1-knockout)
Ena1 (Ena1 knockout)
HATENA (ATENA + NHA1)
NHNA (NHA1 + Ena1-kockout
The protocol we used was the same as always.
08/31
AtNHXS1: We performed Verifiecation PCRs of both the left       and right part of the Fragments (+100bp of the Genome on       each side) of colonies 1 and 2.
PCR Mix:
PCR Program:
Gene:
Left
Right
Component
µl
µl
Template
1 per Tube
1 per Tube
Primer f
0,4
0,4
Primer r
0,4
0,4
One Taq 2x Mastermix
10
10
ddH2O
7,2
7,2
Gene:
Le
ft
Rig
ht
PCR Step
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
94°C
30 sec
94°C
D Denaturation
15 sec
94°C
15 sec
94°C
A Annealing
25 sec
56°C
25 sec
58°C
E Extention
110 sec
68°C
200 sec
68°C
FE Final Elongation
5 min
68°C
5 min
68°C
S Storage
unlimited
8°C
unlimited
8°C
Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
AtNHXS1-eGFP: We performed another verification PCR of the       7 growing colonies, which failed, the Transformation was unsuccessful.
BioBrick AtNHXS1-eGFP: We made Overnights of the 8 grown       colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
09/01
AVP1: There was growth on all plates. We performed a check-PCR with 5 colonies of each mutant.
ATENA (AtNHXS1 + Ena1-knockout): colony 5
Ena1 (Ena1 knockout): colony 3
HATENA (ATENA + NHA1): colony 3
NHNA (NHA1 + Ena1-kockout: colony 1
We send the fragments containing our integrated genes to eurofins for sequencing.
<div class="white100"></div>
Week 15
09/05
AVP1: We amplified the fragments 1, 2, 3 & 4 via PCR and       repeated the transformations of Ena1 and ATENA, using the same protocol       as always.
09/06
mRFP: We performed a new Transformation of mRFP into BY4742 WT. Selection with Geniticin.
DNA Mix:
Gene:
mR
FP
Fragment
m needed [ng]
V needed [µl]
Homologous Region Left
500
1,48
KanMX
250
5
mRFP
500
1,03
Homologous Region Right
500
4
H2O
22,49
Trafo Mix:
Trafo:
mRFP into BY WT
Component
Amount [µl]
50% PEG 3350
240
1M LiAC
36
Carrier DNA
50
AtSultr1;2: We performed a Transformation of AtSultr1;2 (4 Fragments) into BY4742 WT. Selection with       SC-Plates without Lysin. The Transformation failed because no colonies grew.
AVP1: Due to low concentrations, we amplified the fragments 1, 2 & 3 again.
09/07
AVP1:We integrated the fragments in the genome of the mutants Ena1, ATENA, HATENA and NHNA, using the same protocol as always.
Results after two days: growth on all plates.
<div class="white100"></div>
Week 16
09/11
AVP1: We performed a cPCR of the colonies transformed       09/08. Five colonies per mutant were checked.
Gel-electrophoresis:
The bands were at the expected height.
We send the desired gene-fragments to eurofins to get it sequenced.
09/12
AVP1: Sequencing results: two of the mutants (AVENA & HATAVENA) contained point mutations, the other two (ATAVENA & PHA) couldn't be sequenced due to too low DNA-concentrations.
(We performed a new PCR to amplify the desired fragment for the sequencing. It failed.)
09/13
AVP1: To check the expression of our protein AVP1 we decided to perform a SDS-page. See protocol.
09/14
AVP1: We performed the SDS-page. See protocol.
09/15
AVP1: We performed the SDS-page. See protocol.
<div class="white100"></div>
Week 17
09/18
AVP1: We integrated the AVP1 fragment into four different mutants form the other teams:
ATENA (AtNHXS1 + Ena1-knockout)
Ena1 (Ena1 knockout)
HATENA (ATENA + NHA1)
NHNA (NHA1 + Ena1-kockout
The protocol we used was the same as always.
09/20
AtNHXS1-eGFP: We microscoped the transformed colonies of AtNHXS1-eGFP into BY4742 mRFP       (Transformation: 08/07). Only around 1% of the cells fluoresced red. Every cell that fluoresced red, fluoresced green as well.       Resolution was too low to colocalize transporters.
09/22
AtNHXS1: We performed Verifiecation PCRs of both the left       and right part of the Fragments (+100bp of the Genome on       each side) of colonies 1 and 2.
PCR Mix:
PCR Program:
Gene:
Left
Right
Component
µl
µl
Template
1 per Tube
1 per Tube
Primer f
0,4
0,4
Primer r
0,4
0,4
One Taq 2x Mastermix
10
10
ddH2O
7,2
7,2
Gene:
Le
ft
Rig
ht
PCR Step
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
94°C
30 sec
94°C
D Denaturation
15 sec
94°C
15 sec
94°C
A Annealing
25 sec
56°C
25 sec
58°C
E Extention
110 sec
68°C
200 sec
68°C
FE Final Elongation
5 min
68°C
5 min
68°C
S Storage
unlimited
8°C
unlimited
8°C
Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
AtNHXS1-eGFP: We performed another verification PCR of the       7 growing colonies, which failed, the Transformation was unsuccessful.
BioBrick AtNHXS1-eGFP: We made Overnights of the 8 grown       colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
<div class="white100"></div>
Week 18
09/25
Since the labelling of the vacuole with mRFP and mCherry did not showed the expected results (gene was integrated correct, but there was no fluorescence) we decided to use the fluorescenceprotein mOrange Part BBa_E2050 and create a fusionprotein with the VNX-channel, that only appears in the vacuoles. Therefore we designed and ordered the respective primers.
09/26
We performed Cryos of ATAVENA to back up the transformated cells, after we recieved the sequencing results that showed, that the gene is correct.
We sequenced new colonies of HATAVENA (col 5, 16 and 18) from the transformation that we performed on 08.09.2017. Therefore we performed PCRs with the genomes to isolate the AVP1-cassette for the sequencing.
Furthermore we prepared a RNA-isolation and q-PCR that we want to perform with our AVP1-mutante and the wildtype to check the transcription rate. Therfore we prepared Overnights: a) 2 Overnights of AVP1-mutante in YPD+Galactose b)  2 Overnights of AVP1-mutante in YPD+Glucose and c) 2 Overnights of BY4742 wildtype. We also prepared the buffers and solution we will need for the RNA-Isolation (see protocol!).
09/27
We isolated the RNA of AVP1-mutante and the BY4742 wildtype and performed the reverse transcription by following the protocol “qPCR”. The isolation was successful, the samples were freezed at -80°C.
The sequencing results of HATAVENA showed, that colony 5 and 18 integrated the AVP1-cassette correctly.
09/28
We performed PCRs to amplify the fragments for the integration of AtSultr1;2.
(Biotech-Day)
09/29
(Biotech-Day)
We performed a transformation of AtSult1;2 in BY AVP1 and a transformation of AVP1 and AKT1 in BY XENA (-Cre).
We performed PCRs to amplify the 5 fragments for the integration of the fusion protein VNX+mOrange.
<div class="white100"></div>
Week 19
10/02
We performed the qPCR using our cDNA which we generated from our isolated RNA (see 27.09.2017) to verify the transcription of the integrated AVP1-gene. The results showed, that the wildtype transcribate no AVP1, the RNA-amount of the induced AVP1-mutante was about 3 time higher than the amount of uninduced AVP1-mutante. (Check our result-page!)
We performed a transformation of mOrange and VNX (fusionprotein) in BY 4742 wildtype.
10/04
We isolated the genomes of the colonies containing mOrange-VNX (transformation see 02.10.2017) for verification.
We performed verification PCRs of AVP1 and AKT1 with the genomes of the transformated colonies (see transformation 20.09.2017). The results showed that only AVP1 was integrated successfully. We performed PCRs for sequencing.
10/05
The sequencing results of AVP1 in BY XENA showed that the gene was integrated correctly. The new mutants are named BY XAVENA.
We performed further verification PCRs of the genomes containing VNX+mOrange, because the PCRs of 04.10.17 showed unexpected results.
We isolated genomes of new colonies containing VNX+mOrange of the transformation see 02.10.2017 and performed verification PCRs.
<div class="white100"></div>
Week 20
10/11
We performed several transformations: we transformed AKT1-gene into BY NHNA ( a)with the transformation kit and also b) by following the transformation protocol), we transformed AtSult1;2 into BY 4742 wildtype and we transformed mOrange+VNX also in BY2742 wildtype.
10/12
ENA1:Since we had many different mutant colonies where our       transformation did seem to work, we decided to store them in case something       went wrong along the way. We produced 10 different overnights for ENA1 K.O       transformations and wild type BY4742.
Interlab Study:We transformed samples of competent E. coli       cells. We transformed with three different controls; one negative, one positive       and another positive with a RFP marker. We produced 30 LB plates containing       chloramphenicol from which 18 plates were used.
10/13
AVP1: We integrated the AVP1 fragment into four different mutants form the other teams:
ATENA (AtNHXS1 + Ena1-knockout)
Ena1 (Ena1 knockout)
HATENA (ATENA + NHA1)
NHNA (NHA1 + Ena1-kockout
The protocol we used was the same as always.
<div class="white100"></div>
Week 21
10/16
AtNHXS1: We performed Verifiecation PCRs of both the left       and right part of the Fragments (+100bp of the Genome on       each side) of colonies 1 and 2.
PCR Mix:
PCR Program:
Gene:
Left
Right
Component
µl
µl
Template
1 per Tube
1 per Tube
Primer f
0,4
0,4
Primer r
0,4
0,4
One Taq 2x Mastermix
10
10
ddH2O
7,2
7,2
Gene:
Le
ft
Rig
ht
PCR Step
Time
Temperature
Time
Temperature
ID Initial denaturation
30 sec
94°C
30 sec
94°C
D Denaturation
15 sec
94°C
15 sec
94°C
A Annealing
25 sec
56°C
25 sec
58°C
E Extention
110 sec
68°C
200 sec
68°C
FE Final Elongation
5 min
68°C
5 min
68°C
S Storage
unlimited
8°C
unlimited
8°C
Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
AtNHXS1-eGFP: We performed another verification PCR of the       7 growing colonies, which failed, the Transformation was unsuccessful.
BioBrick AtNHXS1-eGFP: We made Overnights of the 8 grown       colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
10/17
AtNHXS1: We performed verification PCRs       of the left part (+100bp from Genome) of 6 transformed       colonies BY4742 AVENA+AtNHXS1 (ATAVENA).
PCR Mix:
PCR Program:
Gene:
ATAVENA
Component
µl
Template
1 per Tube
Primer f
1,2
Primer r
1,2
One Taq 2x Mastermix
30
ddH2O
21,6
Gene:
ATAV
ENA
PCR Step
Time
Temperature
ID Initial denaturation
30 sec
94°C
D Denaturation
15 sec
94°C
A Annealing
25 sec
56°C
E Extention
110 sec
68°C
FE Final Elongation
5 min
68°C
S Storage
unlimited
8°C
Gel Electrophoresis was successful for colonies 1 and 2.
AtNHXS1-eGFP: AtNHXS1-eGFP: Verification       PCRs of 7 growing colonies failed.
10/20
ENA1:Since we had many different mutant colonies where our       transformation did seem to work, we decided to store them in case something       went wrong along the way. We produced 10 different overnights for ENA1 K.O       transformations and wild type BY4742.
Interlab Study:We transformed samples of competent E. coli       cells. We transformed with three different controls; one negative, one positive       and another positive with a RFP marker. We produced 30 LB plates containing       chloramphenicol from which 18 plates were used.
<div class="white100"></div>
Week 22
10/23
We performed two double digests of a) the AKT1-gene b) the AtSultr1;2 and c) the pSB1C3-backbone with EcoRI and PstI. Afterwards we ligated a) AKT1 with pSB1C3-backbone and b) the AtSultr1;2 with pSB1C3-backbone and finally performed the transformation of the ligated plasmids into competent E.coli DH5α.
10/25
We performed Overnights of the grown colonies of the transformation with the pSB1C3-AtSultr1;2-plasmid.
10/27
ENA1:Since we had many different mutant colonies where our       transformation did seem to work, we decided to store them in case something       went wrong along the way. We produced 10 different overnights for ENA1 K.O       transformations and wild type BY4742.
Interlab Study:We transformed samples of competent E. coli       cells. We transformed with three different controls; one negative, one positive       and another positive with a RFP marker. We produced 30 LB plates containing       chloramphenicol from which 18 plates were used.
