Team:Aachen/Biobricks
iGEM Team Aachen 2017
Home
Project
Description
Design
Applied Design
Biobricks
Safety
Results
Demonstrate
Outlook
Human Practices
Overview
Human Practices - Silver
Integrated Human Practices - Gold
Collaborations
Lab
Workflow
Protocols
Notebook
InterLab
Team
Team
Attributions
Special Awards
Hardware
Plants
Model
Engagement
Home
Project
Description
Design
Biobricks
Safety
Results
Demonstrate
Human Practices
Silver Human Practices
Integrated/Gold
Collaborations
Lab
Workflow
Protocols
Notebook
Interlab
Team
Attributions
Special Awards
Hardware
Plants
Model
Biobricks
One of the key missions of iGEM is getting better, as a community, at solving problems. In order to be a part of this great initiative, we isolated genes which we thought showed a great potential to be analyzed and used for future projects.
Basic Part - AtNHXS1
This biobrick produces a S. cerevisiae codon usage optimized Na+, K+ / H+ vacuolar antiporter protein from Arabidopsis thaliana which can be used to accumulate cations in the vacuole and increase salt tolerance in S. cerevisiae.
Usage and Biology:
The native Na+, K+ / H+ antiporter protein AtNHX1 located in the vacuolar membrane consists of 9 transmembrane helices and 3 hydrophobic regions. This protein plays an important role in the accumulation of cations and therefore cellular salt / osmotic stress tolerance. When an Arabidopsis thaliana calmodulin-like protein 15 (AtCaM15), found in the vacuolar lumen, binds to intracellular Ca+ it interacts with the AtNHX1 C-Terminus, regulating the selectivity to Na+ (K+ selectivity isn’t affected significantly), thus decreasing the Vmax of the Na+ / H+ exchange activity.       The mutated AtNHXS1 has been optimized for S. cerevisiae codon usage and consists of only 4 transmembrane helices from which the CaM regulated C-Terminus has been cut out. This eliminates the regulatory process and increases the Na+ / H+ exchange activity by 1-fold while maintaining the K+ / H+ which may be responsible for the increased sodium transport activity when compared to the native AtNHX1 protein.
Basic Part - AtSultr1;2
AtSultr1;2, a high affinity sulfate transporter, found naturally in Arabidopsis thaliana, was optimized for the S. cerevisiae codon usage. Between the genes AtSultr1;1 and AtSultr1;2, the second one proved to be easier to analyze due to the phenotypical differences to the wildtype.
Usage and Biology:
In nature, AtSultr1;2 plays an essential role in the Arabidopsis thaliana growth, especially when sulfur is limited. Being a membrane transporter, AtSultr1;2 can be used to accumulate sulfate in the cell’s cytosol to further use it in the production of cysteine and methionine. Because S. cerevisiae lacks the genes that control the expression of this protein, any wished promoter can be used.
Composite Part - AtNHXS1
This composite biobrick integrates a vacuolar membrane (tonoplast) H+ - pyrophosphate originating from Arabidopsis thaliana optimized for S. cerevisiae codon usage into the vacuolar membrane. The vacuolar H+ - ATPase, found naturally in yeast, and AVP1 proteins generate a biochemical proton gradient which can be used by different vacuolar transporters as an energy source.
Usage and Biology:
The vacuolar membrane (tonoplast) bound H+ - pyrophosphatase AVP1 found in Arabidopsis thaliana hydrolyses cytosolic inorganic pyrophosphate (PPi) to orthophosphate (Pi) in order to use the resulting energy to pump protons into the vacuole, acidifying it. The proton gradient generated through the vacuolar H+ - ATPase and AVP1 can be used by vacuolar transporters, e.g. AtNHXS1, to pump ions into the vacuole. The protein is optimized for S. cerevisiae codon usage and consists of only one subunit, making its overexpression and correlation to the results easier. AVP1 expression is regulated through a Gal1 promoter. The sides of the cassette consist of two homologue fragments which integrate into highly conserved genome regions in the chromosome XI of the S. cerevisiae genome.
Sadly, our cassette did not fit the iGEM’s parameters because of the many restriction sites present due to its long sequence.
