<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Team Aachen 2017</TITLE></P><DIV class="container"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="active"><A href="https://2017.igem.org/Team:Aachen">Home</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/Project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Biobricks">Biobricks</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/HP" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/HP">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/HP/Silver">Human Practices - Silver</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/HP/Gold_Integrated">Integrated Human Practices - Gold</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/Lab" class="dropdown-toggle" data-toggle="dropdown">Lab</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Workflow">Workflow</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/InterLab">InterLab</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/Team" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#">Special Awards</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Plant">Plants</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Engagement">Engagement</A></LI></DIV></DIV><DIV class="container-fluid"><DIV class="blackgrad navbar-fixed-top" style="margin-top:50px"><DIV class="container"><DIV class="row"><DIV class="col-md-12"><UL class="ul1"><LI class="li2">Week</LI><LI class="li1"><A href="#toWeek1">01</A></LI><LI class="li1"><A href="#toWeek2">02</A></LI><LI class="li1"><A href="#toWeek3">03</A></LI><LI class="li1"><A href="#toWeek4">04</A></LI><LI class="li1"><A href="#toWeek5">05</A></LI><LI class="li1"><A href="#toWeek6">06</A></LI><LI class="li1"><A href="#toWeek7">07</A></LI><LI class="li1"><A href="#toWeek8">08</A></LI><LI class="li1"><A href="#toWeek9">09</A></LI><LI class="li1"><A href="#toWeek10">10</A></LI><LI class="li1"><A href="#toWeek11">11</A></LI><LI class="li1"><A href="#toWeek12">12</A></LI><LI class="li1"><A href="#toWeek13">13</A></LI><LI class="li1"><A href="#toWeek14">14</A></LI><LI class="li1"><A href="#toWeek15">15</A></LI><LI class="li1"><A href="#toWeek16">16</A></LI><LI class="li1"><A href="#toWeek17">17</A></LI><LI class="li1"><A href="#toWeek18">18</A></LI><LI class="li1"><A href="#toWeek19">19</A></LI><LI class="li1"><A href="#toWeek20">20</A></LI><LI class="li1"><A href="#toWeek21">21</A></LI><LI class="li1"><A href="#toWeek22">22</A></LI></UL></DIV></DIV></DIV></DIV></DIV><A name="toWeek1">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 1</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">05/29</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> For the Integration of AVP1 we are using the easy clone plasmid pCfB391 
					which contains the homologous regions, the terminator and a Histidin auxotrophy marker. We received 
					E.coli DH5alpha carrying pCfb391 and made overnights in LB+Ampicillin (To be used by AVP1 Group Jotho)
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>Isolation of yeast genome (quick and dirty protocol)
				</P><P class="NoteText"><STRONG>ENA1:</STRONG> PCR of left, right homologous arm from the previously isolated yeast 
					genome and kanMX from pUG6 plasmid were performed, but the gel showed failure. 
					The PCRs failed repeatedly, with the gels looking strange; our troubleshooting 
					concluded that the error was in a wrongly prepared agarose solution.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">05/30</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG> For further use we created sterile 50% Glycerol stock.
				</P><P class="NoteText"><STRONG>AVP1:</STRONG> We created Cryos of the overnights of E.coli carrying pCfB391, Storage at -80°C
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG> The production PCR for the left and right homologous arm for the 
					ENA1 knockout finally worked.
				</P><P class="NoteText" style="float: left">
					Gel right arm:
				</P><P class="NoteText" style="float: right; margin-right:30%;">
					Gel left arm:
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">05/31</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>Overnight cultures of Escherichia coli carrying 
					the plasmids pUG6, Easy clone plasmid and our Gal1 plasmid were made.
				</P><P class="NoteText"><STRONG>ENA1:</STRONG> A gradient PCR was performed for the kanMX cassette, 
					to determine the annealing temperature of our primers.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/01</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG>Cryo cultures of Escherichia coli carrying the plasmids 
					pUG6, Easy clone plasmid and our Gal1 plasmid were made to secure the plasmids.
				</P><P class="NoteText"><STRONG>ENA1:</STRONG>The kanMX cassette was amplified, with a pUG6 plasmid as template. 
					The PCR worked fine.
					The ENA1 knockout transformation was performed with the three previously 
					amplified fragments into BY4742 wildtype.
				</P><P class="NoteText">
					kanmx product:
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG>We prepared the saving of the AVP1-gene in
					Escherichia coli. We made four overnights, two of E. coli pCfB 391 
					(containing the easy clone plasmid) and another two of E. coli 
					containing pUC19. They were incubated at 37°C.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/02</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1:</STRONG> The 4 fragments for the overexpression transformation of 
					NHX1 were amplified: the left homologous sequence “X-Up”, the Gal1 
					promotor from the pUG6 plasmid,  the NHX1 gene from BY4742 genome and 
					the right homologous sequence from an easy clone plasmid, 
					including the marker genes for yeast and e.coli, “X-Down”.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We performed a plasmid preparation of pUC19.
				</P></DIV></DIV></DIV></DIV><A name="toWeek2">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 2</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/06</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mCherry Labelling:</STRONG> To check the correct 
					intracellular position (vacuolar membrane) of the foreign 
					genes we are transforming into S.cerevisiae BY4742 we want 
					to label the native vacuolar protein 
					encoded by the gene ATG22 with mCherry (Fusion Protein). 
					On the other hand, we will label the foreign genes with 
					eGFP and colocalize it with ATG22-mCherry to the vacuolar 
					membrane. 
				</P><P class="NoteText">
					In the following weeks we worked on transforming S.cerevisiae BY4742 with mCherry. 
					Steps included:
				</P><UL style="list-style-type:none; font-size:16px;"><LI style="padding-left:20px;"><STRONG>I.</STRONG>	Isolation of Plasmids carrying the genes for mCherry, 
						eGFP and a Kanaymcin-Cassette (Selection with Geniticin) (Week 2)
					</LI><LI style="padding-left:20px;"><STRONG>II.</STRONG>	Isolation of Homologous Regions from S.cerevisiae
						BY4742 for integration into BY4742 Genome via 
						homologous recombination (Week 2)
					</LI><LI style="padding-left:20px;"><STRONG>III.</STRONG>	PCRs to amplifiy the 4 Fragments for the 
						mCherry Transformation: Homologous Region Left, 
						KanMX, mCherry, Homologous Region Right (Week 2-5)
					</LI><LI style="padding-left:20px;"><STRONG>IV.</STRONG>
						Transformation of S.cerevisiae BY4742 with the 
						4 amplified fragments (Week 5)
					</LI><LI style="padding-left:20px;"><STRONG>V.</STRONG>	Verification PCR of transformed colonies which 
						grew on selection Media (Week 5-6)
					</LI><LI style="padding-left:20px;"><STRONG>VI.</STRONG>	Microscopy with Fluorescence Microscope (Week 6)
					</LI></UL><P class="NoteText">
					Results of these steps can be seen at 07/07 (Week 6)
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We successfully performed a 
					single-digest of our prepped pUC19 with EcoR1.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1 Fragment Preparation:</STRONG> AVP1 Fragment Preparation: 
					We inoculated 2 overnight cultures of e.coli to prepare fragments for 
					the integration of AVP1. Overnight 1 contained an e.coli strain 
					carrying the EasyClone Plasmid (pO391). The other strain contained 
					the pGREGG566 Plasmid with the Gal1 Promoter that is part of one 
					fragment for the AVP1 integration. 
			</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG> The plates of the transformation of ENA1 in BY4742 showed growth. 
					The wildtype control plate with G418 antibiotic showed growth too, though – 
					which led to the suspicion that the plates weren’t produced correctly, for 
					example with too little G418. Transformed cells, a control transformation
					with a plasmid with kanMX and wildtype BY4742 were plated again to test this.
					We also produced new G418 plates.
				</P><P class="NoteText"><STRONG>NHX1:</STRONG> A PCR-cleanup was performed with the four previously 
					amplified fragments for the NHX1 transformation.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/07</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG> It was discovered that our strain was wrongly labelled and 
					already contained a G418 resistance – which explained the problems 
					with growth of the negative control. The plates turned out not to 
					be the problem. 
				</P><P class="NoteText"><STRONG>NHX1:</STRONG> The transformation of NHX1 into BY4742 was performed 
					with the previously amplified fragments according to protocol. 
					This transformation had to be repeated later due to the discoveries 
					about our strain. To be able to repeat he transformation, the fragments 
					for it were to be amplified again. After the Gel, PCR cleanup was performed. 
				</P><P class="NoteText">
					NHX1 PCR product: 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG> AVP1 Fragment Preparation: </STRONG>We isolated the 2 plasmids from 
					the overnights (06/06) and determined the concentration (pGREGG566, 
					carrying Gal1 Promoter: 740 ng/µL; pCfB 391 EasyClone: 208,5 ng/µL). 
				</P><P class="NoteText"><STRONG>LAB: </STRONG> We mixed 1L of Tris-HCl buffer and 1L of TE buffer.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/08</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG> AVP1 Fragment Preparation: </STRONG>We used the pGREGG566 plasmid
					isolated the day before to amplify the Gal1 Promoter linear fragment. 
					Therefore, we carried out a PCR and Gelelectrophoresis afterwards.
				</P><P class="NoteText">
					All 3 PCR tubes showed bands at 500 bp, which is the expected length 
					for our Gal1 Promoter Fragment (462 + 40 bases overhang from the reverse 
					Primer -&gt; 502 bases long. The negative control tube 4 with no template 
					(plasmid) showed no band at 500 bp. 
					We cleaned up the product and determined the concentration: 25,5 ng/L.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/09</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG>An overnight culture of E.coli containing a plasmid with eGFP was made.
				</P><P class="NoteText"><STRONG>ENA1:</STRONG>: The kanMX fragment for the transformation was amplified 
					again, as our product from the previous PCR was depleted. The transformation 
					was repeated because of the strain mix-up. Overnight culture of the E.coli 
					containing the plasmid with kanMX was made.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG> AVP1 Fragment Preparation: </STRONG>We carried out 2 PCRs and 
					2 Gelelectrophoresis to amplify the Upper Homologous Region + 
					Terminator (TADH1) = Fragment 1, and the Lower Homologous Region
					+ loxP + Marker + loxP = Fragment 4 from the EasyClonePlasmid pCfB 391.
					Bought Fragments are necessary for the integration of AVP1 
					into the yeast genome.
				</P><P class="NoteText">
					Gel Fragment 1:
				</P><P class="NoteText">
					all 4 PCR tubes showed bands at around 900 bp, which is 
					what we expected (our Fragment 1 (Up and TADH1 Terminator) 
					is 896 bp). The PCR worked in the 50 µL (1,2) as well as in 
					the 25 µL (M,3) tubes, but not in the negative control. The 
					negative control tube K with no insert (Easy Clone Plasmid
					p0391) showed no band at around 900 bp.
					The PCR Product was cleaned up and concentration was determined afterwards.
				</P><P class="NoteText">
					Gel Fragment 4:
				</P><P class="NoteText">
					All 4 PCR tubes showed bands at around 2000+ bp, which is what we expected 
					(our Fragment 4 (Down loxP-Marker-loxP) is 2111 bp long). The PCR worked in 
					the 50 µL (1,2) as well as in the 25 µL (M,3) tubes, but not in the negative
					control. The negative control tube K with no insert (Easy Clone Plasmid p0391)
					showed no band at around 2000 bp.
					The PCR Product was cleaned up and concentration was determined afterwards.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We dephosphorylated puC19 and performed a 
					digestion of the AVP1-gene with EcoR1.
				</P></DIV></DIV></DIV></DIV><A name="toWeek3">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 3</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/12</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>Plasmid Prep was done for the pUG6 and eGFP plasmids from the overnight cultures.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> After preparing our parts for the AVP1 back-up in 
					E. coli we finally started the transformation. First our insert 
					(AVP1) and our plasmid (pUC19) were ligated with T4 ligase, 
					we prepared two tubes: the ligation and a negative-control.
				</P><P class="NoteText">
					Ligation-Mix composition:
				</P><TABLE id="Notebooktable"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AV</TH><TH style="border-right: 2px solid grey">P1</TH></TR><TR><TD style="text-align: center"><STRONG>Components</STRONG></TD><TD>Ligation</TD><TD>Negative Control</TD></TR><TR><TD><STRONG>Insert (AVP1)</STRONG></TD><TD>6,56 µL</TD><TD>-</TD></TR><TR><TD><STRONG>Backbone (pUC19)</STRONG></TD><TD>4 µL</TD><TD>4 µL</TD></TR><TR><TD><STRONG>T4 Ligase</STRONG></TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD><STRONG>ddH2O</STRONG></TD><TD>5,44 µL</TD><TD>12 µL</TD></TR><TR><TD><STRONG>total</STRONG></TD><TD>20 µL</TD><TD>20 µL</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					We incubated the two Eppis for 1h at room temperature and 
					performed a heat shock at 65°C afterwards. 
					For the transformation, we used E. coli BL21. We used the 
					transformation-protocol from NEB.
					The transformed cells were spread on Amp-Plates. In addition, 
					we made a negative and a positive control.
				</P><UL style="font-size:16px; margin-left:2%;"><LI>BL21 with pUC19 + AVP1</LI><LI>BL21 with pUC19 (without insert)</LI><LI>wildtype</LI><LI>dilutions from BL21 with insert</LI></UL></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/13</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG> The kanMX fragment was amplified from the previously 
					prepared pUG6 plasmid. The gel shows two bands, the lower one being our long 
					primers (long due to overhangs).
				</P><P class="NoteText">
					kanMX production PCR:
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>There was no growth on 
					the plates from the day before, so either the 
					ligation or the transformation didn't work. 
					We asked one of our advisors and they gave us 
					the tip to use a higher insert/backbone-ratio (9:1) and extend the 
					ligation time.
					For our second ligation, we used the same parts as before.
					We prepared 3 tubes.
				</P><P class="NoteText">
					Ligation-Mix composition:
				</P><TABLE id="Notebooktable"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: middle">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Components</STRONG></TD><TD>Ligation (3:1 ratio)</TD><TD>Ligation (9:1 ratio)</TD><TD>Negative Control</TD></TR><TR><TD><STRONG>Insert (AVP1)</STRONG></TD><TD>6,56 µL</TD><TD>9,84 µL</TD><TD>-</TD></TR><TR><TD><STRONG>Backbone (pUC19)</STRONG></TD><TD>4 µL</TD><TD>2 µL</TD><TD>4 µL</TD></TR><TR><TD><STRONG>T4 ligase</STRONG></TD><TD>2 µL</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>2 µL</TD><TD>2 µL</TD><TD>2 µL</TD></TR><TR><TD><STRONG>ddH2O</STRONG></TD><TD>5,44 µL</TD><TD>4,16 µL</TD><TD>12 µL</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					After the ligation we performed the transformation and spread the cells on Amp-Plates.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/14</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>A gradient PCR was done for eGFP to determine the annealing temperature. 
					Although an error was made when creating the primers, the PCR worked quite well.
				</P><P class="NoteText">
					Annealing Test:
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>The transformation seemed successful. 
					There grew c20 colonies on the 9:1 ratio plate and c5 on the 3:1 ratio plate.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/15</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG>Now that the annealing temperature for the eGFP fragments was 
					determined, a production PCR could be performed. The product was cleaned.
				</P><P class="NoteText">
					production PCR result:
				</P><P class="NoteText" style="margin-top:10px;">
					To test the E.coli with pUG6 and eGFP, LB plates with Aminopenicillin 
					for pUG6 and Kanamycin for eGFP were produced, and the bacteria plated 
					on one plate each.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/16</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>In order to produce cryo cultures 
					for pUG6, eGFP and BY4742, we first have to produce overnights from only 
					one colony of each. We used the plates we produced on the day before and 
					the one we received from our advisor (Sandra) (BY4742). 
				</P><P class="NoteText">
					Our overnights contained the following:
						1.	1 pUG6 colony + 25 ml LB medium + 25 ul Amp (100%)
						2.	1 eGFP colony + 25 ml LB medium + 25 ul Kan (100%)
						3.	1 BY4742 colony + 25 ml YPD medium
						4.	1 BY4742 colony + 25 ml YPD medium</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We checked the transformation with a colony-PCR. Therefore, we used the protocol 1.3 Check PCR from the script &quot;Praktikum Molekulare Biotechnologie&quot;.
					We checked 8 colonies from our 9:1 plate. 
				</P><P class="NoteText">
					Check PCR composition:
				</P><TABLE id="Notebooktable"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: middle">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Components</STRONG></TD><TD>Volume for one Tube</TD><TD>Volume for ten Tubes</TD><TD>Final Concentration</TD></TR><TR><TD><STRONG>ddH2O</STRONG></TD><TD>15 µL</TD><TD>150 µL</TD><TD>-</TD></TR><TR><TD><STRONG>5x Green GoTag Buffer</STRONG></TD><TD>4 µL</TD><TD>40 µL</TD><TD>1x</TD></TR><TR><TD><STRONG>Primer f (10µM)</STRONG></TD><TD>0,2 µL</TD><TD>2 µL</TD><TD>0,1 µM</TD></TR><TR><TD><STRONG>Primer r (10µM)</STRONG></TD><TD>0,2 µL</TD><TD>2 µL</TD><TD>0,1 µM</TD></TR><TR><TD><STRONG>dNTP-mix (10mM)</STRONG></TD><TD>0,4 µL</TD><TD>4 µL</TD><TD>200 µM</TD></TR><TR><TD><STRONG>GoTag Polymerase</STRONG></TD><TD>0,2 µL</TD><TD>2 µL</TD><TD>2,5 U</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					Due to the different melting temperatures, we performed a gradient PCR (T: 57,5-62,0°C).
				</P><TABLE id="Notebooktable"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AV</TH><TH style="border-right: 2px solid grey">P1</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>10 min</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>30 sec</TD><TD>gradient (57,5-62,0°C)</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>60 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>72°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					After the PCR we made a gel-electrophoresis.
				</P><P class="NoteText" style="margin-top:10px;">
					Unfortunately, the results didn't meet our expectations, 
					there are bands between 250bp and 500bp, all at a different 
					height. If the transformation had been successful there 
					would be a 2300bp band.
				</P></DIV></DIV></DIV></DIV><A name="toWeek4">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 4</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/19</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>Overnight cultures from 16.06.17 where used to 
					create cryo cultures. Cryo tubes of pUG6 and eGFP (both in E.coli) 
					were filled with 500µl culture + 500µl 50% glycerin and the cryo of 
					BY4742 with 500µl culture + 500µl 30% glycerin.
				</P><P class="NoteText">
					Competent BY4742 cryos were created according to the McClean protocol.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>To check if AVP1 has any other unexpected EcoR1-restriction sites,
					we performed a digestion and a gel-electrophoresis afterwards. The 
					result was one single band with the expected size of c2300bp. 
					Because of our ambiguous results of the colony PCR we 
					decided to check our cloning via restriction analysis and 
					ordered the restriction enzyme Nde1 (cuts in AVP1 &amp; pUC19).
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/20</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1:</STRONG> Production PCR for NHX1 was made following the program 
					from the 08.06.17.  The gel shows that the PCR worked out well. The bands fit 
					the length of the fragment, which was 1899bp.
				</P><P class="NoteText">
					Production PCR NHX1:
				</P><P class="NoteText"><STRONG>NHX1 with eGFP:</STRONG> PCR Products for NHX1 with eGFP overhangs and
					terminator+marker+homologous sequence with eGFP overhangs were produced and PCR
					cleanup was done.
				</P><P class="NoteText"><STRONG>ENA1:</STRONG> Transformation for the ENA1 knockout was performed. 
					Transformed cells and control were plated on YPD with G418.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> The restriction analysis showed, that the transformation 
					failed.
					We talked to our instructor and she recommended us to use the 
					&quot;CloneJET PCR Cloning-Kit&quot; (blunt-end ligation) because we 
					had the fear, that the end digestion of AVP1 didn't work.
					We performed a new transformation of AVP1 in E. coli DH5a. 
					Ligation mixture:
				</P><TABLE id="Notebooktable" style="margin-top:10px;margin-bottom:10px;"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>Amount</TD></TR><TR><TD><STRONG>2x Reaction Buffer</STRONG></TD><TD>10 µL</TD></TR><TR><TD><STRONG>Insert</STRONG></TD><TD>7 µL</TD></TR><TR><TD><STRONG>pJET 1.2</STRONG></TD><TD>1µL</TD></TR><TR><TD><STRONG>T4 DNA Ligase</STRONG></TD><TD>1 µL</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>1 µL</TD></TR><TR><TD>20 µL</TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/21</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB: </STRONG>The G418 plates were tested by plating BY4742 wildtype on our YPD+G418 plates.
					To test our strains, two overnight cultures were made with BY4742 
					wildtype, one with G418 and one without.
				</P><P class="NoteText"><STRONG>NHX1:</STRONG> The transformation for the overexpression of NHX1 was performed. 
					The competent cells were tested using a plasmid with kanMX G418 resistance.
				</P><P class="NoteText"><STRONG>ENA1:</STRONG> With a rest of PCR product, PCR cleanup was performed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG> We transformed the ligation-product 
					and a control-ligation in DH5a, using the protocol from the
					NEB-Kit &quot;NEB 5-alpha competent E. coli&quot;.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/22</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG> We made 3 liters of 1,2% Agarose Gel for use in Gel Electrophoresis
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1: </STRONG>Product PCR of NHX1, X4-Up homologous sequence, 
					Gal1 promotor and x4-Down homologous seuqnce with selection marker was
					repeated to replenish our stock of the fragments.
					The gel showed that the PCR was successful:
				</P><P class="NoteText" style="margin-top:10px;">
					PCR-cleanup was performed with the products.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>The transformation didn't succeed. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/23</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1:</STRONG> The transformation of NHX1 into BY4742 wildtype 
					seemed to work, as colonies started to show on the plates 
					and the control transformation showed no growth. The cells 
					looked very wet and yellow-ish, wish led us to the conclusion 
					that they weren’t yeast cells. Microscoping the cells and comparing 
					them to reference yeast cells confirmed our suspicion that our 
					transformation was contaminated.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>To perform another transformation, 
					we amplified the AVP1 gene via PCR.  
				</P><P class="NoteText" style="float:left"> 
					Composition of 100µL master mix:
				</P><P class="NoteText" style="float:right; margin-right:33%;"> 
					PCR-programm:
				</P><TABLE id="Notebooktable" style="float:left; width:40%;"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>Amount</TD></TR><TR><TD><STRONG>B2 PfuS Buffer</STRONG></TD><TD>10 µL</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>8 µL</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>8 µL</TD></TR><TR><TD><STRONG>dNTP-mix</STRONG></TD><TD>2 µL</TD></TR><TR><TD><STRONG>Polymerase (PfuS)</STRONG></TD><TD>1 µL</TD></TR><TR><TD><STRONG>Template (AVP1)</STRONG></TD><TD>2,2 µL</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>71 µL</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AV</TH><TH style="border-right: 2px solid grey">P1</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>2 min 20 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>20 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>30 sec</TD><TD>61°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>2 min 30 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>3 min</TD><TD>72°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:325px; position:relative"> 
					The PCR was successful. [DNA]= 18ng/µL
				</P></DIV></DIV></DIV></DIV><A name="toWeek5">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 5</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/26</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1:</STRONG> To have some backup material, we decided to make some 
					cryo cultures for our transformed NHX1 overexpression cells. We made 5 cryo 
					cultures in 30% glycerin solutions and stored them in a -80°C container.
				</P><P class="NoteText">
					We suspected that our transformation was contaminated because of a rather 
					unusual smell for a yeast culture. Using a microscope, we found out that our 
					cultures were in fact e. colis. Several colonies were tested, each with the 
					same misfortune.
					Our cryo cultures were discarded. 
				</P><P class="NoteText"><STRONG>ENA1:</STRONG> After several days, no growth was observed on our 
					transformation plate. We decided to use a method called “Abschwämmen”. 
					A small amount of YPD+G418 medium is added to the plate. Using a sterile 
					toothpick, the colonies are carefully picked and dissolved in the medium. 
					This medium was then pipetted into a new YPD+G418 plate.
				</P><P class="NoteText">
					We tested our SC-Histidine free plates and concluded they were functional.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1 Fragment Preparation:</STRONG>To refill our stock of 
					Gal1 promoter we repeated the PCR carried out on 06/07: 
					We used the pGREGG566 plasmid isolated the day before to
					amplify the Gal1 Promoter linear fragment. Therefore, we carried out a 
					PCR, which failed.

				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>Because all teams had problems with the PCRs 
					we performed a PCR-test series to check the polymerases. Our suspicions 
					were confirmed and we had to throw some of the tubes away. 
					Simultaneously to the PCRs we performed a second transformation with AVP1 
					ligated in pJET. this time we used a AVP1 with a higher 
					concentration (18 ng/µL instead of 9 ng/µL).
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/27</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1:</STRONG> Due to the contaminated plates we had to discard, 
					we made a new transformation. In order to test our plates, we did a test 
					transformation using a KanMX plasmid.
				</P><P class="NoteText"><STRONG>LAB:</STRONG> New SC plates were produced, histidine free. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>This time 5 colonies grew on the plate. 
					We performed a colony PCR to check the transformation success. 
				</P><P class="NoteText">
					Result:
				</P><P class="NoteText" style="margin-top:10px;">
					We prepared overnight cultures for a restriction analysis. 
				</P></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/28</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtHENA:</STRONG> We produced three different overnights; two different 
					knockout strains and one wild type. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We performed a second colony PCR and changed 
					the annealing temperature to 60°C.
					In addition, we performed a restriction analysis with Nde1. 
					Both methods showed, that the transformation failed. Because 
					the saving of AVP1 in E. coli didn't seemed very promising in the 
					future we decided to skip this part and begin with the transformation in yeast. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/29</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We performed another PCR to amplify all 4 fragments for the 
					integration of AVP1 into S.cerevisiae BY4742 which failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG> Selection media without histidine and 
					without uracil were produced.
				</P><P class="NoteText"><STRONG>ATHENA:</STRONG> We resuspended our new primers for KanMX extraction 
					in TE – Buffer. Overnights from our plate #2 were made. The antibiotic G418 
					was added as selective marker.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">06/30</H1></DIV></DIV></DIV></DIV><A name="toWeek6">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 6</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/03</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1 Fragment Preparation:</STRONG>  To refill our stock of Gal1 
					promoter we repeated the PCR carried out on 06/07 :
					We used the pGREGG566 plasmid isolated the day before (
					to amplify the Gal1 Promoter linear fragment. Therefore, we 
					carried out a PCR, which failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1:</STRONG> We dissolved our new primers in DMSO with a 25 mMolar 
					solution. We produced new YPD Medium because of a contamination. These new 
					primers were used in a gradient PCR to produce the right and left arms beside 
					the NHA1 gene. A verification PCR for ENA1 was also introduced in our gradient 
					PCR.
					 This where our results:
				</P><P class="NoteText">The NHA1 PCRs did not work.
				</P><P class="NoteText"><STRONG>ENA1:</STRONG> We decided to prepare some overnight cultures. Overnight 
					#1 contained G418 (antibiotic) and the ENA1 transformation, overnight #2 
					contained G418 (antibiotic) and wild type and overnight #3 with contained 
					YPD and the ENA1 transformation.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/04</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1 Fragment Preparation:</STRONG> We repeated the PCR frp, 07/03. 
					This time the gel showed 1 intense band at around 500 bp, which fits the Gal1 
					fragment length (502 bp). The product was therefore cleaned up and the 
					concentration was determined. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/05</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1 Fragment Preparation: </STRONG>We carried out PCR to amplify
					the codon optimized gene from IDT. The test gel showed 1 intense band at 
					around 2350 kb, which fits the length of the AVP1 Gene (2375 bp in length). 
					The product was cleaned up using PCR clean-up and concentration was determined
					via NanoVue afterwards (123 ng/µL). 
					Furthermore, we repeated the PCRs from 06/09 to refill our stock of 
					Homologous Region Fragments for the genome integration. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG>We wanted to use a FACS Machine to do a cell analysis. To do this, 
					we needed to produce PBS medium for this analysis and as a medium for 
					fluorescent microscopy.The pH was set to 7.4 using NaCl. After this the 
					overall volume was increased to 1000 ml using cold ddH2O and autoclaved.
				</P><P class="NoteText"><STRONG>NHA1:</STRONG>We found the results from yesterday´s gradient PCR 
					somewhat strange. We decided to rapidly test the template DNA with an 
					electrophoresis. This were our results:
				</P><P class="NoteText">
					No template was found in our NHA1 left and right arms. That wild type BY4742 
					genome was disposed of and new genome was isolated from a new overnight culture.
					Another reason why our gradient PCR wasn´t working was the solution in which 
					the primers were dissolved in, DMSO. A primer solution, whichever the 
					concentration is, should never include more than 3.5 % DMSO, otherwise 
					the polymerase will react with said solute and denaturate.
				</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/07</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mCherry Integration Results:</STRONG> The last weeks we worked on transforming mCherry into S.cerevisiae 
					BY4742 to create the fusion protein ATG22-mCherry. Amplification of the 4 fragments (Homologous Region 
					Left, KanMX, mCherry, Homologous Region Right) was successful, the Transformation as well. Verification 
					PCRs of the integrated fragments showed, that mCherry was integrated correctly, however microscopy did not 
					show fluorescence at all (Used Laser Wavelength was: 561). Control of Codon Usage showed, that mCherry is 
					Codon Optimized for E.coli and therefore not expressed correctly. We will withdraw mCherry and use a 
					S.cerevisiae codon optimized mRFP Gene from now on.
				</P></DIV></DIV></DIV></DIV><A name="toWeek7">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 7</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/10</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG>Since we had many different mutant colonies where our 
					transformation did seem to work, we decided to store them in case something 
					went wrong along the way. We produced 10 different overnights for ENA1 K.O 
					transformations and wild type BY4742.
				</P><P class="NoteText"><STRONG>Interlab Study:</STRONG>We transformed samples of competent E. coli 
					cells. We transformed with three different controls; one negative, one positive 
					and another positive with a RFP marker. We produced 30 LB plates containing 
					chloramphenicol from which 18 plates were used.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/11</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We created his-drop out medium (plates) 
					for our transformation of AVP1 in yeast.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1:</STRONG>We produced some YPD medium flasks for later use. 
					We proceeded to new gradient PCRs (10 samples for each) containing XII-Up 
					(left arm), XII-Down (right arm) and the URA3 + loxP marker. 
					The following gel electrophoresis shows our results:
				</P><P class="NoteText">
					The XII-Up PCR did not work. The temperatures that barely seem to work for 
					X-Down were 49.3 °C, 51.2 °C, 53.3 °C and 57.2 °C.
				</P><P class="NoteText"><STRONG>Interlab Study:</STRONG>On all our plates, some E. coli colony 
					growth was observed. However, we could only count the colonies on 4 of 
					these plates. Because of this we decided to replate on LB + Chloramphenicol 
					most of the E. colis on new plates to obtain single colonies.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/12</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>For the transformation, we first performed a 
					PCR of fragment 3 (Gal1-Promotor). After that we transformed our 
					fragments into Saccharomyces cerevisiae. 3 tubes were made:
				</P><UL style="margin-left:3%; font-size:16px;"><LI>yeast + fragments 1-4</LI><LI>yeast + control plasmid</LI><LI>yeast from the other group for testing + control plasmid</LI></UL><TABLE id="Notebooktable"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>Tube 1</TD><TD>Tube 2</TD><TD>Tube 3</TD></TR><TR><TD><STRONG>Fragment 1 (TADH1)</STRONG></TD><TD>11,8 µL </TD><TD>-</TD><TD>-</TD></TR><TR><TD><STRONG>Fragment 2 (AVP1)</STRONG></TD><TD>12,5 µL</TD><TD>-td&gt;
						</TD><TD>-</TD></TR><TR><TD><STRONG>Fragment 3 (Gal1)</STRONG></TD><TD>13,9 µL</TD><TD>-</TD><TD>-</TD></TR><TR><TD><STRONG>Fragment 4 (his-aux-marker)</STRONG></TD><TD>7,3 µL</TD><TD>-</TD><TD>-</TD></TR><TR><TD><STRONG>plasmid p0391</STRONG></TD><TD>-</TD><TD>0,96 µL</TD><TD>0,96 µL</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>-</TD><TD>33,04 µL</TD><TD>33,04 µL</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					The transformation-mixture contained 720µL 50% PEG 350; 
					108 µL 1M LiOAc and 150µL carrier DNA (salmon sperm). For 
					the transformation, we used the protocol: &quot;Heatshock transformation S. 
					cerevisiae (modified after Joska al. 2014)&quot; from Sandra. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1:</STRONG>A gradient PCR was made for the XII-Up, XII-Down and 
					URA3+ loxP exactly as yesterday. The results of the gel electrophoresis 
					show that no product was present in our mixtures.
				</P><P class="NoteText"><STRONG>Interlab Study:</STRONG>We wanted to be sure our LB plates weren’t 
					working well so we tested them. We plated E. coli dH5d and E. coli with 
					PuG6 on our plates. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/13</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> Since the expression of mCherry failed due to wrong Codon usage 
						we are repeating the labeling with a codon optimized mRFP.
						
						We performed PCRs of the for the Transformation needed fragments 1 (Left Homologous Region), 
						3 (mRFP) and 4 (Right Homologous Region). Fragment 2 did not change.
					</P><DIV class="row row-eq-height"><DIV class="col-md-5"><P class="NoteText">
								PCR Mix:
							</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey; ">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Left Homologous Fragment</TH><TH class="thr" style="border-right: 2px solid grey">mRFP</TH><TH class="thr" style="border-right: 2px solid grey">Right Homologous Fragment</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1,314</TD><TD>0,348</TD><TD>1,104</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>15</TD><TD>15</TD><TD>15</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 12</TD><TD> 12</TD><TD> 12</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>12</TD><TD>12</TD><TD>12</TD></TR><TR><TD><STRONG>dNTP</STRONG></TD><TD>3</TD><TD>3</TD><TD>3</TD></TR><TR><TD><STRONG>Polymerase (PfuS)</STRONG></TD><TD>1,5</TD><TD>1,5</TD><TD>1,5</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>105,2</TD><TD>106,15</TD><TD>105,4</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
								PCR Programm:
							</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Left Homolo</TH><TH style="border-right: 2px solid grey">gous Fragment</TH><TH style="text-align: right">mR</TH><TH style="text-align:left; border-right: 2px solid grey;">FP</TH><TH style="text-align: right">Right Homolo</TH><TH style="text-align:left">gous Fragment</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>180 sec</TD><TD>98°C</TD><TD>180 sec</TD><TD>98°C</TD><TD>180 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>20 sec</TD><TD>98°C</TD><TD>20 sec</TD><TD>98°C</TD><TD>20 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>40 sec</TD><TD>58°C</TD><TD>40 sec</TD><TD>63°C</TD><TD>40 sec</TD><TD>53°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>60 sec</TD><TD>72°C</TD><TD>60 sec</TD><TD>72°C</TD><TD>60 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>72°C</TD><TD>5 min</TD><TD>72°C</TD><TD>5 min</TD><TD>72°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px;">
						Afterwards we analyzed the PCRs via a Gel Electrophoresis, 
						performed a PCR Clean-Up and determined the DNA-concentration via NanoVue.
					</P><P class="NoteText">
						 Gel:
					</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1: </STRONG>We did a verification PCR using the isolated genome 
						of our ENA1 mutated colonies. We then followed this with a gelelectrophoresis.
					</P><P class="NoteText"><STRONG>NHA1: </STRONG>Due to the crazy results from the past days, 
						we decided to do 2 separate PCRs. One production PCR for the XII-Down 
						and URA3 + loxP fragments and one gradient PCR for the XII-Up fragment. 
						We also used the new primers (now diluted in TE buffer to a 25-mM 
						concentration) for every PCR. The following gel shows our results:
					</P><P class="NoteText">
						URA3 + loxP and XII-Down seem to work fine. Nonetheless XII-Up gradient PCR showed 
						now promising bands. Both successful products were cleaned up and stored.
					</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We created competent S. cerevisiae BY4742, 
					using the protocol &quot;McClean: Making and Using Frozen Yeast Competent Cells&quot;.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/14</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRRP Labelling: </STRONG>We transformed S.cerevisiae BY4742 with the 4 mRFP Fragments 
					to integrate mRFP into the genome. YPD-Plates containing Genticin 
					were used as a selection marker.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText"><STRONG>DNA-Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%;"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">mR</TH><TH style="border-right: 2px solid grey">FP</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Homologous Region Left</STRONG></TD><TD>1000</TD><TD>23,81</TD></TR><TR><TD><STRONG>KanMX</STRONG></TD><TD>500</TD><TD>10</TD></TR><TR><TD><STRONG>mRFP</STRONG></TD><TD>13,42</TD><TD>1000</TD></TR><TR><TD><STRONG>Homologous Region Right</STRONG></TD><TD>1000</TD><TD>23,81</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>0</TD><TD>0</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-6"><P class="NoteText"><STRONG>Trafo-Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">mRFP into Wildtype (WT)</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row" style="margin-top:20px;"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>Transformation results (07/12): Because there was growth
					on the plates with the transformed yeast, the transformation seemed to be
					successful. 
					We isolated the genome of 5 colonies from plate 1 for further tests.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1: </STRONG>The only fragment missing to continue was XII-Up. 
					We followed a production PCR for XII-Up with a PCR clean up while we waited 
					for our gel electrophoresis.
				</P><P class="NoteText">
					Our products did contain the wished fragment size.
				</P></DIV></DIV></DIV></DIV><A name="toWeek8">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 8</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/17</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> Check of Transformation done at 07/14. 4 colonies grew. No fluorescence under UV-Lamp. Verficiation PCR needed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>Our transformed yeast cells were contaminated 
					with E. coli, so we performed a new transformation. This time we used 
					the competent yeast cells we made the week before. The amounts of the 
					four DNA-fragments were the same as last time (tube 1). Again, we used 
					the protocol &quot;Heat shock transformation S. cerevisiae&quot;. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1: </STRONG>We produced 2 liters x1 PBS-Buffer (100 ml x10 PBS per 900 ml ddH2O). 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/18</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> Check of Transformation done at 07/14. 4 colonies grew. No fluorescence under UV-Lamp. Verficiation PCR needed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1: </STRONG>We finally started with our transformation tests. 
					Our NHA1 transformed cells and wild type BY4742 were then plated on a URA- 
					plates. We also produced some PEG 50% 3350 solution for further yeast 
					transformations.	
				</P><P class="NoteText"><STRONG>ENA1: </STRONG>We did a verification PCR for our ENA1 K.O followed 
					by a gel electrophoresis. No bands were observed.
				</P><P class="NoteText"><STRONG>NHX1: </STRONG>To check if our NHX1 overexpression transformation 
					we did a verification PCR with our isolated genome. None of our products worked. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We successfully amplified our 4 fragments for further transformations. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/19</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> Gel Electrophoresis of the PCR Products was unsuccessful. 
					We did another PCR with the right and left parts of the integrated fragments, 
					which failed as well. Therefore, mRFP was not integrated into BY4742 during 
					Transformation.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We made competent Ena1-kock-out BY4742 
					mutants and performed a transformation with these cells to 
					integrate AVP1. We used the same amounts of fragment as before.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHX1: </STRONG>Since our verification PCR did not work yesterday 
					we decided to isolate a new genome and use it for a new verification PCR. 
					This again did not work.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/20</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> We performed another Transformation of BY4742 with the 4 mRFP Fragments 
					(Left Homologous Fragment, KanMX, mRFP, Right Homologous Fragment) as done at 07/14.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText"><STRONG>DNA-Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">mR</TH><TH style="border-right: 2px solid grey">FP</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Left Homologous Region</STRONG></TD><TD>500</TD><TD>7,1</TD></TR><TR><TD><STRONG>KanMX</STRONG></TD><TD>250</TD><TD>5</TD></TR><TR><TD><STRONG>mRFP</STRONG></TD><TD>500</TD><TD>6,71</TD></TR><TR><TD><STRONG>Homologous Region Right</STRONG></TD><TD>500</TD><TD>11,9</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>4</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText"><STRONG>Trafo-Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">mRFP into WT</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row" style="margin-top:20px;"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>To check the success of our transformation (07/17) 
					we performed a genome-isolation and a check-PCR with 5 of the c50 colonies. 
					For the genome-isolation we used the protocol &quot;Quick and Dirty 
					(Mans et. al 2015)&quot; from our advisor Sandra. 
					(The check-PCR failed.)

				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1: </STRONG>We needed some XII-Up fragments again. Instead 
					of doing a gradient PCR, we decided to set our temperature to 47.7 °C 
					and make larger mixtures. Our gel electrophoresis showed the result of said PCR.
				</P><P class="NoteText">
					Our products were cleaned up and used for a new NHA1 transformation 
					(transformation #2). The transformed yeast cells were again plated 
					(along with a wild type) on URA+ plates.
				</P></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/21</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>NHA1:</STRONG>After not seeing any culture growth in the past days, 
					we decided to do a third transformation just to be sure. We plated our 
					transformation and wild type on URA- plates.
				</P><P class="NoteText">
					Our XII-Down and XII-Up fragments were accidentally thrown away, 
					therefore we needed to produce more. Since we already knew the 
					ideal annealing temperatures for our fragments, we decided to do
					a production PCR. We analyzed our products with a gel electrophoresis. 
					The results showed that none of the PCR products worked.
				</P></DIV></DIV></DIV></DIV><A name="toWeek9">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 9</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/24</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> 40 colonies grew on the Transformation from 07/20. We isolated 
					the Genomes and performed Verification PCRs 
					of 5 colonies. Primers included both left and right part of the integrated 
					fragments and additional 100bp left and right of the integration sites.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText"><STRONG>PCR Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Strain:</TH><TH class="thr" style="border-right: 2px solid grey">BY4742+mRFP</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1,5 per Tube</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>5</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>2</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>2</TD></TR><TR><TD><STRONG>dNTP</STRONG></TD><TD>1</TD></TR><TR><TD><STRONG>Polymerase (PfuS)</STRONG></TD><TD>0,5</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>37,2</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText"><STRONG>PCR Programm:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Strain:</TH><TH style="text-align: right">BY474</TH><TH style="border-right: 2px solid grey">2+mRFP</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>180 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>30 sec</TD><TD>57°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>50 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>72°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><DIV class="row" style="margin-top:15px;"><DIV class="col-md-12"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We want to integrate the Arabidopsis Thaliana vacuolar Sodium/Proton Antiporter AtNHXS1 into 
							S.cerevisiae. Therefore, we use the Easy Clone Plasmid pCfb257, carrying the TCYC1 Terminator, Integrationsides 
							for Chromsome X-3 and a Leucin Marker. We received E.coli DH5alpha carrying pCfb257 on a plate containing Ampicillin 
							and incubated it. Additionally we performed PCRs to amplify the Fragments 2 (Gal1 Promoter) and 3 
							(AtNHXS1 synthesized by IDT) for Transformation, which failed. 
						</P></DIV></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We repeated the check-PCR. It failed again.
					We isolated another five genomes from different colonies (BY4742+AVP1 &amp; Ena1-kock-out + AVP1).
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/25</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP Labelling:</STRONG> Gel Electrophoresis of the Verification PCRs from 07/24. Colonies 2 showed 
					correct bands. We microscoped colonies 2 and 4 other colonies (6, 7, 8 and 9) which had not 
					been PCR verified to check fluorescence (Laser wavelength: 561nm). Whereas colony 2, 7, 8 and 
					9 showed no to small fluorescence, colony 6 showed high fluorescence. 20 other microscoped 
					colonies showed fluorescence as well. We will go on working with colony 6.
				</P><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We prepared an overnight culture of E.coli DH5alpha carrying pCfB257 to amplify the plasmid. 
					Additionally, we performed another PCR to amplify Fragments 2 and 3, which failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We performed another cPCR. It failed again.
					Because all our PCRs failed, we received a different polymerase (GoTag) 
					from our advisor and repeated the cPCR. We had 12 tubes.
				</P><P class="NoteText">Gelelectrophoresis: </P><UL class="NoteText" style="float:right; margin-right:38%;"><LI>1, 8: ladder</LI><LI>2, 3, 4: cPCR AVP1</LI><LI>5, 6, 7: cPCR Ena1</LI><LI>9-14: control</LI></UL></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/26</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed a plasmid prep of the E.coli+pCfB257 Overnight to isolate pCfB257. 
					We performed PCRs of all 4 Fragments (Integrationside Up, Gal1, AtNHXS1, 
					Terminator-Leucin Marker-Integrationside Down)
				</P><P class="NoteText"><STRONG>PCR Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Fragment 1: Up</TH><TH class="thr" style="border-right: 2px solid grey">Fragment 2: Gal1</TH><TH class="thr" style="border-right: 2px solid grey">Fragment 3: AtNHXS1</TH><TH class="thr" style="border-right: 2px solid grey">Fragment 4: T-Down</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>0,52</TD><TD>0,34</TD><TD>1,5</TD><TD>0,52</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>25</TD><TD>25</TD><TD>25</TD><TD>25</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>20</TD><TD>20</TD><TD>20</TD><TD>20</TD></TR><TR><TD><STRONG>dNTP</STRONG></TD><TD>5</TD><TD>5</TD><TD>5</TD><TD>5</TD></TR><TR><TD><STRONG>Polymerase (PfuS)</STRONG></TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD><TD>2,5</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>177</TD><TD>177,2</TD><TD>176</TD><TD>177</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px"><STRONG>PCR Programm:</STRONG></P><TABLE id="Notebooktable" style="width:80%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Fragme</TH><TH style="border-right: 2px solid grey">nt1:Up</TH><TH style="text-align: right">Fragment</TH><TH style="text-align:left; border-right: 2px solid grey;">2:Gal1</TH><TH style="text-align: right">Fragment3</TH><TH style="border-right: 2px solid grey">:AtNHXS1</TH><TH style="text-align: right">Fragment</TH><TH style="text-align:left">4:T-Down</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>98°C</TD><TD>30</TD><TD>98°C</TD><TD>30 sec</TD><TD>98°C</TD><TD>30 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>8 sec</TD><TD>98°C</TD><TD>8 sec</TD><TD>98°C</TD><TD>8 sec</TD><TD>98°C</TD><TD>8 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>20 sec</TD><TD>59°C</TD><TD>20 sec</TD><TD>61°C</TD><TD>30 sec</TD><TD>63°C</TD><TD>30 sec</TD><TD>58°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>20 sec</TD><TD>72°C</TD><TD>20 sec</TD><TD>72°C</TD><TD>60 sec</TD><TD>72°C</TD><TD>30 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>72°C</TD><TD>5 min </TD><TD>72°C </TD><TD>5 min</TD><TD>72°C</TD><TD>5 min </TD><TD>72°C </TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px">
					Afterwards we performed a Gel Electrophoresis, a PCR-Clean Up and determined the DNA-concentrations of the fragments.
				</P><P class="NoteText" style="margin-top:10px; margin-left:0%"><STRONG>Gel:</STRONG></P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We checked the other homologies recombination 
					locus of the tested Ena1-mutants and both loci of the AVP1-mutant.
				</P><P class="NoteText">
				Gel-electrophoresis 
				</P><UL class="NoteText" style="float:right; margin-right:25%;"><LI>1, 16: ladder</LI><LI>2, 3, 4, 5: posterior homologues region</LI><LI>6, 7, 8 ,9: AVP1+Gal</LI><LI>10-13: control</LI><LI>14, 15: anterior homologues region</LI></UL><P class="NoteText">
					Both strains integrated the homologues DNA-sequences.
					We performed another PCR to prove the right order of the integrated fragments. 
				</P><P class="NoteText">
				Gel-electrophoresis 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/27</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP:</STRONG> We performed Verification PCRs to verify the integration of mRFP into colony 6 
						which was fluorescing on 07/25. The PCRs failed. We made an overnight of colony 6 in YPD containing G418.
					</P><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We transformed the 4 amplified AtNHXS1 Fragments into BY4742 WT and BY4742 containing 
						Ena1 Knock-Out. We used SC-Plates without Leucin as selection media.
					</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText"><STRONG>DNA Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNH</TH><TH style="border-right: 2px solid grey">XS1</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Up</STRONG></TD><TD>500</TD><TD>4,46</TD></TR><TR><TD><STRONG>Gal1</STRONG></TD><TD>500</TD><TD>4,33</TD></TR><TR><TD><STRONG>AtNHXS1</STRONG></TD><TD>500</TD><TD>3,38</TD></TR><TR><TD><STRONG>T-loxP-Leu-loxP-Down</STRONG></TD><TD>350</TD><TD>4,76</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>17,07</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText"><STRONG>Trafo Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1 into WT &amp; Ena-Mutant</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>480</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>72</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>100</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV><DIV class="row" style="margin-top:10px;"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> The integration of AVP1 in the Ena1-knockout-mutant was successful, the integration of AVP1 in the wildtype failed. 
						We isolated the genome of 8 other colonies and performed a cPCR.
					</P><P class="NoteText">
					Gelelectrophoresis:
					</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/28</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP:</STRONG> We performed new PCRs to verify the integration of mRFP into BY4742. We checked the left 
						and right part of the fragments and additionally 100bp on each side of the fragment of the genome.
					</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText"><STRONG>PCR Mix:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Left Part</TH><TH class="thr" style="border-right: 2px solid grey">Right Part</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1</TD><TD>1</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 0,2</TD><TD> 0,2</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,2</TD><TD>0,2</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>5</TD><TD>5</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>3,4</TD><TD>3,4</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText"><STRONG>PCR Programm:</STRONG></P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Left </TH><TH style="border-right: 2px solid grey">Part</TH><TH style="text-align: right">Right</TH><TH style="text-align:left"> Part</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>57°C</TD><TD>25 sec</TD><TD>59°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>75 sec</TD><TD>68°C</TD><TD>120 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px;">
						Afterwards we performed a Gel Electrophoresis.
					</P><P class="NoteText" style="margin-top:10px;"><STRONG>Gel:</STRONG></P><P class="NoteText" style="margin-top:10px;">
						We made competent cells of BY4742 + mRFP from the overnights from 07/27.
					</P></DIV></DIV><DIV class="row" style="margin-top:10px;"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We checked the 5 colonies that contained the upper 
						homologues sequence. They didn't integrated the other fragments. 
					</P></DIV></DIV></DIV></DIV><A name="toWeek10">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 10</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">07/31</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> Colonies grew on AtNHXS1 Transformations from 07/27. We performed a Genome Isolation and verification PCR (Left and Part of Fragments, 100bp of Genome on left side) of 5 colonies.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							PCR Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>2</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>2</TD></TR><TR><TD><STRONG>OneTap 2x Mastermix</STRONG></TD><TD>50</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>36</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							PCR Programm:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNH</TH><TH style="border-right: 2px solid grey">XS1</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px">
						We performed a Gel Electrophoresis (WT left, Ena middle, Positive Control right)
				</P><P class="NoteText" style="margin-top:10px">
						We prepared Overinghts of WT+AtNHXS1 (Colony 1) and Ena+AtNHXS1 (Colony 3) to make cryos and competent cells.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>None of the colonies we checked the day before integrated all of the four fragments. 
					Therefore, we isolated 12 others from the same plate and performed a check
					-PCR.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/01</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed vericifaction PCRs of the left and right 
					side of WT+AtNHXS1 (Colony 1) and Ena+AtNHXS1 (Colony 3). Primers 
					included whole integrationsite and 100bp Genome on each side.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							PCR Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 0,4</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,4</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>10</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>7,2</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							PCR Programm:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Up-At</TH><TH style="border-right: 2px solid grey">NHXS1</TH><TH style="text-align: right">T-D</TH><TH style="text-align:left">own</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD><TD>25 sec</TD><TD>57°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD><TD>200 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px;">
					Gel Electrophoresis was successful for both strains (Left WT, middle Ena, right positive control)
				</P><P class="NoteText" style="margin-top:10px;">
					Gel:
				</P><P class="NoteText" style="margin-top:10px;">
					We made 25 competent cryos per strain (WT+AtNHXS1 and Ena+AtNHXS1) from the overnights made 07/31.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>AtNHXS1-eGFP:</STRONG> We are planning to label AtNHXS1 with eGFP to colocalize it with ATG22-mRFP to the 
					vacuolar membrane. We performed a PCR to amplify Fragment 3 (AtNHXS1 with
					Overhang for eGFP) and Fragment 5 (Failed).
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							PCR Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1-eGFP</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>0,6</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>10</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>4</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>4</TD></TR><TR><TD><STRONG>dNTP</STRONG></TD><TD>2</TD></TR><TR><TD><STRONG>Polymerase (PfuS)</STRONG></TD><TD>1</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>70,8</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							PCR Programm:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNH</TH><TH style="border-right: 2px solid grey">XS1</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>8 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>30 sec</TD><TD>63°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>60 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>72°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px;">
					We performed a Gel Electroophoresis (Fragment 3 left, 5 right), Clean-Up and determined the DNA-concentration.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>Because the last cPCR didn’t work, we performed a 
					ramp PCR (AVP1 in Ena1 mutant). We prepared two mastermixes, split it in two 
					tubes and added 1uL template DNA each.
				</P><TABLE id="Notebooktable"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-left: 2px solid grey">AVP1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>Mix 1</TD><TD>Mix 2</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 µL</TD><TD>1 µL</TD></TR><TR><TD><STRONG>OneTag Mix </STRONG></TD><TD>10 µL</TD><TD>10 µL</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 0,4 µL</TD><TD> 0,4 µL</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,4 µL</TD><TD>0,4 µL</TD></TR><TR><TD><STRONG>ddH2O</STRONG></TD><TD>9,2 µL</TD><TD>9,2 µL</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">PCR-program:</P><TABLE id="Notebooktable" style="width:40%;"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AV</TH><TH style="border-right: 2px solid grey">P1</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>20 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>30 sec</TD><TD>ramp 45-55°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>2 min 50 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					Gelelectrophoresis:
				</P><P class="NoteText" style="margin-top:10px;">
					The gelelectrophoresis showed that the transformation was sucessful.
					In addition, we performed a cPCR with the AVP1 in wildtype genome. 
					It showed that this transformation failed.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/02</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We made cryos of WT+AtNHXS1 and Ena+AtNHXS1 from the Overnights made 07/31.
				</P><P class="NoteText"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another PCR to amplify Fragment 5 (T-Down with eGFP Overhang). 
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							PCR Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">T-Down</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>0,1</TD></TR><TR><TD><STRONG>Buffer</STRONG></TD><TD>10</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 5</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>5</TD></TR><TR><TD><STRONG>dNTP</STRONG></TD><TD>2</TD></TR><TR><TD><STRONG>Q5 High-Fidelity DNA Polymerase</STRONG></TD><TD>1</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>69,9</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							PCR Programm:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">T-D</TH><TH style="border-right: 2px solid grey">own</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>8 sec</TD><TD>98°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>66°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>90 sec</TD><TD>72°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>2 min</TD><TD>72°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px">
					We performed a Gel Electrophoresis, Clean-Up and determined the DNA-concentration.
				</P><P class="NoteText" style="margin-top:10px">
					Gel:
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We performed a new transformation to integrate AVP1 
					in wildtype using the protocol “Heat shock transformation S. cerevisiae” 
					from Sandra.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/03</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1-eGFP:</STRONG> We performed a Transformation of AtNHXS-eGFP (5 Fragments) into BY4742 mRFP. No colonies grew, the Transformation failed.
				</P><P class="NoteText"><STRONG>LAB:</STRONG> We prepared the following Buffers for isolation of vacuoles: YPD-Media, Spheroplasting Buffer, DEAE-dextran solution, PS-Buffer with 4%, 8% and 15&amp; Ficoll.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We performed three PCRs to amplify our fragments for the transformation.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/04</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>LAB:</STRONG> We performed a PCR to amplify the homologous region Down of the BY4742 genome for the Knock-Out of NHA1. The PCR failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>Transformation results (08/02): 21 colonies had grown on our plate. We isolated the genome and performed a check-PCR 
				</P></DIV></DIV></DIV></DIV><A name="toWeek11">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 11</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/07</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another Transformation of AtNHXS1-eGFP into BY4742 mRFP.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							DNA Mix:
						</P><TABLE id="Notebooktable" style="margin-top:10px; width:100%"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNHXS</TH><TH style="border-right: 2px solid grey">1-eGFP</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Up</STRONG></TD><TD>500</TD><TD>4,46</TD></TR><TR><TD><STRONG>Gal1</STRONG></TD><TD>500</TD><TD>4,33</TD></TR><TR><TD><STRONG>AtNHXS1</STRONG></TD><TD>500</TD><TD>7,87</TD></TR><TR><TD><STRONG>eGFP</STRONG></TD><TD>500</TD><TD>7,75</TD></TR><TR><TD><STRONG>T-Down</STRONG></TD><TD>350</TD><TD>5,98</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>3,61</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							Transformation Mix:
						</P><TABLE id="Notebooktable" style="margin-top:10px; width:100%"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1-eGFP into BY4742 mRFP</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px;"><STRONG>AtNHXS1:</STRONG> We performed a Transformation of AtNHXS1 into AVENA (WT+Ena+AVP1)
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							DNA Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNH</TH><TH style="border-right: 2px solid grey">XS1</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Up</STRONG></TD><TD>500</TD><TD>4,46</TD></TR><TR><TD><STRONG>Gal1</STRONG></TD><TD>500</TD><TD>4,33</TD></TR><TR><TD><STRONG>AtNHXS1</STRONG></TD><TD>500</TD><TD>3,38</TD></TR><TR><TD><STRONG>T-Down</STRONG></TD><TD>350</TD><TD>4,76</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>17,07</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							Transformation Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1 into BY4742 AVENA</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></DIV></DIV></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/10</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>Biobrick Assembly AtNHXS1:</STRONG> We will integrate AtNHXS1 as a part into the Biobrick Plasmid pSB1C3. We performed a Restriction Digest of AtNHXS1 and pSB1C3 with EcoRI and PStI.
				</P><P class="NoteText">
					Digestion Mix:
				</P><TABLE id="Notebooktable"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">pSB1C3</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>4</TD><TD>2</TD></TR><TR><TD><STRONG>Buffer Cutsmart (19x)</STRONG></TD><TD>1</TD><TD>1</TD></TR><TR><TD><STRONG>EcoRI-HF</STRONG></TD><TD> 0,5</TD><TD> 0,5</TD></TR><TR><TD><STRONG>PstI-HF</STRONG></TD><TD>0,5</TD><TD>0,5</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>4</TD><TD>6</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					We performed a PCR Clean-Up, determined the DNA-concentrations performed a Ligation of Backbone and Insert.
				</P><P class="NoteText">
					Ligation Mix:
				</P><TABLE id="Notebooktable"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1 into pSB1C3</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Backbone (25ng)</STRONG></TD><TD>5</TD></TR><TR><TD><STRONG>Insert (90ng)</STRONG></TD><TD>18</TD></TR><TR><TD><STRONG>Buffer </STRONG></TD><TD>1</TD></TR><TR><TD><STRONG>T4 DNA Ligase</STRONG></TD><TD>1</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					We transformed the Ligation Mixture into E.coli DH5alpha and transformed the backbone as a negative control. Selection on LB-Plares with Cloramphenicol.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/11</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1-eGFP:</STRONG>  We microscoped the transformed colonies of AtNHXS1-eGFP into BY4742 mRFP 
					(Transformation: 08/07). Only around 1% of the cells fluoresced red. Every cell that fluoresced red, fluoresced green as well. 
					Resolution was too low to colocalize transporters.
				</P></DIV></DIV></DIV></DIV><A name="toWeek12">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 12</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/14</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1: </STRONG>We performed verification PCRs 
					of the left part (+100bp from Genome) of 6 transformed 
					colonies BY4742 AVENA+AtNHXS1 (ATAVENA). 
				</P><P class="NoteText" style="float:left">
					PCR Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					PCR Program: 
				</P><TABLE id="Notebooktable" style="float:left; width:40%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">ATAVENA</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 1,2</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>1,2</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>30</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>21,6</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right; width:50%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">ATAV</TH><TH style="border-right: 2px solid grey">ENA</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:300px;">
					Gel Electrophoresis was successful for colonies 1 and 2.
				</P><P class="NoteText"><STRONG>AtNHXS1-eGFP: </STRONG>AtNHXS1-eGFP: Verification 
					PCRs of 7 growing colonies failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We prepared an overnight-culture to create competent 
					cells, using the protocol McClean: Making Yeast competent cells.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/15</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed Verifiecation PCRs of both the left 
					and right part of the Fragments (+100bp of the Genome on 
					each side) of colonies 1 and 2.
				</P><P class="NoteText" style="float:left">
					PCR Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					PCR Program: 
				</P><TABLE id="Notebooktable" style="float:left; width:40%;"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Left</TH><TH class="thr" style="border-right: 2px solid grey">Right</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>10</TD><TD>10</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>7,2</TD><TD>7,2</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right; width:50%;"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Le</TH><TH style="border-right: 2px solid grey">ft</TH><TH style="text-align: right">Rig</TH><TH style="text-align:left">ht</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD><TD>25 sec</TD><TD>58°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD><TD>200 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:320px;">
					Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another verification PCR of the 
					7 growing colonies, which failed, the Transformation was unsuccessful.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>BioBrick AtNHXS1-eGFP: </STRONG>We made Overnights of the 8 grown 
					colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We completed the making of the competent cells. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/16</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another Transformation of 
					AtNHXS1+eGFP into BY4742 mRFP with a new eGFP Fragment.
				</P><P class="NoteText" style="float:left;">
					DNA Mix:
				</P><P class="NoteText" style="float:right; margin-right:30%;">
					Trafo Mix:
				</P><TABLE id="Notebooktable" style="float:left; width:40%;"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNHXS</TH><TH style="border-right: 2px solid grey">1-eGFP</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Up</STRONG></TD><TD>500</TD><TD>4,46</TD></TR><TR><TD><STRONG>Gal1</STRONG></TD><TD>500</TD><TD>4,33</TD></TR><TR><TD><STRONG>AtNHXS1</STRONG></TD><TD>500</TD><TD>7,87</TD></TR><TR><TD><STRONG>eGFP</STRONG></TD><TD>500</TD><TD>5,56</TD></TR><TR><TD><STRONG>T-Down</STRONG></TD><TD>350</TD><TD>5,98</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>1,42</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="width:40%; float:right"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1-eGFP into BY4742 mRFP</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:270px;"><STRONG>Biobrick AtNHXS1:</STRONG> We made a Plasmid Prep and Cryos 
					of 8 colonies with the Overnights from 08/15. Afterwards we performed
					a Single Digest of the Plasmids with EcoRI.
				</P><P class="NoteText" style="">
					Digestion Mix:
				</P><TABLE id="Notebooktable"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1-pSB1C3</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Cut Smart Buffer </STRONG></TD><TD>1</TD></TR><TR><TD><STRONG>Template DNA</STRONG></TD><TD>5</TD></TR><TR><TD><STRONG>EcoRI</STRONG></TD><TD>1</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>3</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:10px;">
					We performed a PCR Clean-Up. Gel Electrophoresis 
					was successful for colonies 1-5 and 8.
				</P><P class="NoteText" style="margin-top:20px;"><STRONG>LAB:</STRONG>We made Overnights of several mutants 
					(WT,Ena,Avp,Avena,AtNHXS1,AtEna,AtAvena) for Cryos, Galactose
					Growth Check, Genome Isolation and making competent cells.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/17</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed a Transformation of AtNHXS1 into AVP (AVPA).
				</P><P class="NoteText" style="float:left">
					DNA Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					Trafo Mix: 
				</P><TABLE id="Notebooktable" style="width:40%; float:left;"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">AtNH</TH><TH style="border-right: 2px solid grey">XS1</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Up</STRONG></TD><TD>500</TD><TD>4,46</TD></TR><TR><TD><STRONG>Gal1</STRONG></TD><TD>500</TD><TD>4,33</TD></TR><TR><TD><STRONG>AtNHXS1</STRONG></TD><TD>500</TD><TD>3,38</TD></TR><TR><TD><STRONG>T-Down</STRONG></TD><TD>350</TD><TD>4,76</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>17,07</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="width:50%; float:right;"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">AtNHXS1 into BY4742 AVP</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE><P class="NoteText" style=" float:left; margin-top:10px;">
					We made 10 eppis competent cells of ATAVENA from the Overnight of 08/16. 
				</P></DIV></DIV><DIV class="row" style="margin-top:10px;"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We performed a transformation to integrate AVP1 together with eGFP in our mRFP mutant. We used the same protocol as always.
					After the plate was incubated at 28°C for 4 days, 6 colonies grew.

				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/18</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>Galactose Growth Check:</STRONG> We plated 6 different 
					strains (WT, AVP, AVENA, AtNHXS1, AtENA, AtAVENA) 
					on YPD-Agar containing 60%, 80% and 100% Galactose as carbon source.
				</P><P class="NoteText"><STRONG>LAB: </STRONG>We made Cryos of 7 BY4742 strains (WT, ENA, 
					AVP, AVENA, AtNHXS1, AtENA, AtAVENA). We isolated the genome of 6 
					BY strains (AVP, AVENA, AtNHXS1, AtENA, AtAVENA, mRFP) and dterermined 
					the DNA-concentrations.
				</P><P class="NoteText"><STRONG>AtNHXS1: </STRONG>We performed PCRs to amplify template DNA for 
					Sequencing of AtNHXS1 of the strains AtNHXS1, AtENA, AtAVENA (Promoter – 
					40bp of Terminator) and additionally BY mRFP. We want to sequence 500bp 
					of ATG22 and the whole mRFP gene. The PCRs failed.
				</P></DIV></DIV></DIV><A name="toWeek13">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 13</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/21</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed new PCRs to amplify the 
					templates for sequencing of the AtNHXS1 part of AtNHXS1, AtENA, 
					AtAVENA and additionally BY mRFP. The PCRs failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>(We checked our transformation of the mRFP mutant. It failed.)

				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/22</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed new PCRs to amplify the 
					templates for sequencing of the AtNHXS1 part of AtNHXS1, AtENA, 
					AtAVENA and additionally BY mRFP. The PCRs failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We send the genome fragments of our transformed wildtype
					and the Ena1-knock-out mutant to eurofins to get it sequenced.
					We repeated the transformation with AVP1 + eGFP with a higher AVP1 
					concentration. The used protocol we used was the same as always. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/24</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We got our sequencing results: the integration of AVP1 in
					wildtype was successful, the AVP1 gene in the Ena1-kock-out had a point-mutation. 
					(The transformation of the mRFP mutant with AVP1 + eGFP failed.)
				</P></DIV></DIV></DIV></DIV><A name="toWeek14">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 14</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/28</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We performed a PCR to amplify our fragments. 
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/29</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We integrated the AVP1 fragment into four different mutants form the other teams: 
				</P><UL class="NoteText" style="margin-left:4%;"><LI>ATENA (AtNHXS1 + Ena1-knockout)</LI><LI>Ena1 (Ena1 knockout)</LI><LI>HATENA (ATENA + NHA1)</LI><LI>NHNA (NHA1 + Ena1-kockout</LI></UL><P class="NoteText">
					The protocol we used was the same as always. 
				</P></DIV></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">08/31</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed Verifiecation PCRs of both the left 
					and right part of the Fragments (+100bp of the Genome on 
					each side) of colonies 1 and 2.
				</P><P class="NoteText" style="float:left">
					PCR Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					PCR Program: 
				</P><TABLE id="Notebooktable" style="float:left; width:40%;"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Left</TH><TH class="thr" style="border-right: 2px solid grey">Right</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>10</TD><TD>10</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>7,2</TD><TD>7,2</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right; width:50%;"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Le</TH><TH style="border-right: 2px solid grey">ft</TH><TH style="text-align: right">Rig</TH><TH style="text-align:left">ht</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD><TD>25 sec</TD><TD>58°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD><TD>200 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:320px;">
					Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another verification PCR of the 
					7 growing colonies, which failed, the Transformation was unsuccessful.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>BioBrick AtNHXS1-eGFP: </STRONG>We made Overnights of the 8 grown 
					colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/01</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>There was growth on all plates. We performed a check-PCR with 5 colonies of each mutant.
				</P><UL class="NoteText" style="margin-left:4%;"><LI>ATENA (AtNHXS1 + Ena1-knockout): colony 5</LI><LI>Ena1 (Ena1 knockout): colony 3</LI><LI>HATENA (ATENA + NHA1): colony 3</LI><LI>NHNA (NHA1 + Ena1-kockout: colony 1</LI></UL><P class="NoteText" style="margin-top:10px;">
					We send the fragments containing our integrated genes to eurofins for sequencing.
				</P></DIV></DIV></DIV><A name="toWeek15">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 15</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/05</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We amplified the fragments 1, 2, 3 &amp; 4 via PCR and 
					repeated the transformations of Ena1 and ATENA, using the same protocol 
					as always.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/06</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>mRFP:</STRONG> We performed a new Transformation of mRFP into BY4742 WT. Selection with Geniticin.
				</P><DIV class="row"><DIV class="col-md-5"><P class="NoteText">
							DNA Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegelb"><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">mR</TH><TH style="border-right: 2px solid grey">FP</TH></TR><TR><TD style="text-align: center"><STRONG>Fragment</STRONG></TD><TD>m needed [ng]</TD><TD>V needed [µl]</TD></TR><TR><TD><STRONG>Homologous Region Left</STRONG></TD><TD>500</TD><TD>1,48</TD></TR><TR><TD><STRONG>KanMX</STRONG></TD><TD>250</TD><TD>5</TD></TR><TR><TD><STRONG>mRFP</STRONG></TD><TD>500</TD><TD>1,03</TD></TR><TR><TD><STRONG>Homologous Region Right</STRONG></TD><TD>500</TD><TD>4</TD></TR><TR><TD><STRONG>H<SUB>2</SUB>O</STRONG></TD><TD>22,49</TD></TR></TBODY></TABLE></DIV><DIV class="col-md-5"><P class="NoteText">
							Trafo Mix:
						</P><TABLE id="Notebooktable" style="width:100%"><TBODY><TR id="Notebooktablegrün"><TH style="border-right: 2px solid grey">Trafo:</TH><TH class="thr" style="border-right: 2px solid grey">mRFP into BY WT</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD> Amount [µl]</TD></TR><TR><TD><STRONG>50% PEG 3350</STRONG></TD><TD>240</TD></TR><TR><TD><STRONG>1M LiAC</STRONG></TD><TD>36</TD></TR><TR><TD><STRONG>Carrier DNA</STRONG></TD><TD>50</TD></TR></TBODY></TABLE></DIV></DIV><P class="NoteText" style="margin-top:10px;"><STRONG>AtSultr1;2:</STRONG>
					We performed a Transformation of AtSultr1;2 (4 Fragments) into BY4742 WT. Selection with 
					SC-Plates without Lysin. The Transformation failed because no colonies grew.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> Due to low concentrations, we amplified the fragments 1, 2 &amp; 3 again.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/07</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG>We integrated the fragments in the genome of the mutants Ena1, ATENA, HATENA and NHNA, using the same protocol as always. 
					Results after two days: growth on all plates. 

				</P></DIV></DIV></DIV></DIV><A name="toWeek16">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 16</H1></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/11</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We performed a cPCR of the colonies transformed 
					09/08. Five colonies per mutant were checked.
				</P><P class="NoteText">
					Gel-electrophoresis:
				</P><P class="NoteText">
					The bands were at the expected height. 
					We send the desired gene-fragments to eurofins to get it sequenced.

				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/12</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> Sequencing results: two of the mutants (AVENA &amp; HATAVENA) contained point mutations, the other two (ATAVENA &amp; PHA) couldn't be sequenced due to too low DNA-concentrations.
					(We performed a new PCR to amplify the desired fragment for the sequencing. It failed.)
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/13</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> To check the expression of our protein AVP1 we decided to perform a SDS-page. See protocol.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/14</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We performed the SDS-page. See protocol.
				</P></DIV></DIV></DIV><DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/15</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1:</STRONG> We performed the SDS-page. See protocol.
				</P></DIV></DIV></DIV></DIV><A name="toWeek17">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 17</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/18</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We integrated the AVP1 fragment into four different mutants form the other teams: 
				</P><UL class="NoteText" style="margin-left:4%;"><LI>ATENA (AtNHXS1 + Ena1-knockout)</LI><LI>Ena1 (Ena1 knockout)</LI><LI>HATENA (ATENA + NHA1)</LI><LI>NHNA (NHA1 + Ena1-kockout</LI></UL><P class="NoteText">
					The protocol we used was the same as always. 
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/20</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1-eGFP:</STRONG>  We microscoped the transformed colonies of AtNHXS1-eGFP into BY4742 mRFP 
					(Transformation: 08/07). Only around 1% of the cells fluoresced red. Every cell that fluoresced red, fluoresced green as well. 
					Resolution was too low to colocalize transporters.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/22</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed Verifiecation PCRs of both the left 
					and right part of the Fragments (+100bp of the Genome on 
					each side) of colonies 1 and 2.
				</P><P class="NoteText" style="float:left">
					PCR Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					PCR Program: 
				</P><TABLE id="Notebooktable" style="float:left; width:40%;"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Left</TH><TH class="thr" style="border-right: 2px solid grey">Right</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>10</TD><TD>10</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>7,2</TD><TD>7,2</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right; width:50%;"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Le</TH><TH style="border-right: 2px solid grey">ft</TH><TH style="text-align: right">Rig</TH><TH style="text-align:left">ht</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD><TD>25 sec</TD><TD>58°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD><TD>200 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:320px;">
					Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another verification PCR of the 
					7 growing colonies, which failed, the Transformation was unsuccessful.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>BioBrick AtNHXS1-eGFP: </STRONG>We made Overnights of the 8 grown 
					colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
				</P></DIV></DIV></DIV><A name="toWeek18">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 18</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/25</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">Since the labelling of the vacuole with mRFP and mCherry did not showed the expected results (gene was integrated correct, but there was no fluorescence) we decided to use the fluorescenceprotein mOrange Part BBa_E2050 and create a fusionprotein with the VNX-channel, that only appears in the vacuoles. Therefore we designed and ordered the respective primers.</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/26</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">
				We performed Cryos of ATAVENA to back up the transformated cells, after we recieved the sequencing results that showed, that the gene is correct.
				</P><P class="NoteText">
				
				We sequenced new colonies of HATAVENA (col 5, 16 and 18) from the transformation that we performed on 08.09.2017. Therefore we performed PCRs with the genomes to isolate the AVP1-cassette for the sequencing.
				</P><P class="NoteText">Furthermore we prepared a RNA-isolation and q-PCR that we want to perform with our AVP1-mutante and the wildtype to check the transcription rate. Therfore we prepared Overnights: a) 2 Overnights of AVP1-mutante in YPD+Galactose b)  2 Overnights of AVP1-mutante in YPD+Glucose and c) 2 Overnights of BY4742 wildtype. We also prepared the buffers and solution we will need for the RNA-Isolation (see protocol!).
</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/27</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">
					We isolated the RNA of AVP1-mutante and the BY4742 wildtype and performed the reverse transcription by following the protocol “qPCR”. The isolation was successful, the samples were freezed at -80°C.
					</P><P class="NoteText">
				The sequencing results of HATAVENA showed, that colony 5 and 18 integrated the AVP1-cassette correctly.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/28</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">
					We performed PCRs to amplify the fragments for the integration of AtSultr1;2.
</P><P class="NoteText">(Biotech-Day)</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">09/29</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">
				(Biotech-Day)
				</P><P class="NoteText">We performed a transformation of AtSult1;2 in BY AVP1 and a transformation of AVP1 and AKT1 in BY XENA (-Cre).
</P><P class="NoteText">We performed PCRs to amplify the 5 fragments for the integration of the fusion protein VNX+mOrange.

				</P></DIV></DIV></DIV><A name="toWeek19">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 19</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/02</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">
					We performed the qPCR using our cDNA which we generated from our isolated RNA (see 27.09.2017) to verify the transcription of the integrated AVP1-gene. The results showed, that the wildtype transcribate no AVP1, the RNA-amount of the induced AVP1-mutante was about 3 time higher than the amount of uninduced AVP1-mutante. (Check our result-page!)
					</P><P class="NoteText">We performed a transformation of mOrange and VNX (fusionprotein) in BY 4742 wildtype.

				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/04</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">
					We isolated the genomes of the colonies containing mOrange-VNX (transformation see 02.10.2017) for verification. </P><P class="NoteText">We performed verification PCRs of AVP1 and AKT1 with the genomes of the transformated colonies (see transformation 20.09.2017). The results showed that only AVP1 was integrated successfully. We performed PCRs for sequencing.

				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/05</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">The sequencing results of AVP1 in BY XENA showed that the gene was integrated correctly. The new mutants are named BY XAVENA. 
</P><P class="NoteText">We performed further verification PCRs of the genomes containing VNX+mOrange, because the PCRs of 04.10.17 showed unexpected results.
</P><P class="NoteText">We isolated genomes of new colonies containing VNX+mOrange of the transformation see 02.10.2017 and performed verification PCRs.
</P></DIV></DIV></DIV><A name="toWeek20">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 20</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/11</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">We performed several transformations: we transformed AKT1-gene into BY NHNA ( a)with the transformation kit and also b) by following the transformation protocol), we transformed AtSult1;2 into BY 4742 wildtype and we transformed mOrange+VNX also in BY2742 wildtype.</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/12</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG>Since we had many different mutant colonies where our 
					transformation did seem to work, we decided to store them in case something 
					went wrong along the way. We produced 10 different overnights for ENA1 K.O 
					transformations and wild type BY4742.
				</P><P class="NoteText"><STRONG>Interlab Study:</STRONG>We transformed samples of competent E. coli 
					cells. We transformed with three different controls; one negative, one positive 
					and another positive with a RFP marker. We produced 30 LB plates containing 
					chloramphenicol from which 18 plates were used.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/13</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AVP1: </STRONG>We integrated the AVP1 fragment into four different mutants form the other teams: 
				</P><UL class="NoteText" style="margin-left:4%;"><LI>ATENA (AtNHXS1 + Ena1-knockout)</LI><LI>Ena1 (Ena1 knockout)</LI><LI>HATENA (ATENA + NHA1)</LI><LI>NHNA (NHA1 + Ena1-kockout</LI></UL><P class="NoteText">
					The protocol we used was the same as always. 
				</P></DIV></DIV></DIV><A name="toWeek21">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 21</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/16</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1:</STRONG> We performed Verifiecation PCRs of both the left 
					and right part of the Fragments (+100bp of the Genome on 
					each side) of colonies 1 and 2.
				</P><P class="NoteText" style="float:left">
					PCR Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					PCR Program: 
				</P><TABLE id="Notebooktable" style="float:left; width:40%;"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">Left</TH><TH class="thr" style="border-right: 2px solid grey">Right</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>0,4</TD><TD>0,4</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>10</TD><TD>10</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>7,2</TD><TD>7,2</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right; width:50%;"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">Le</TH><TH style="border-right: 2px solid grey">ft</TH><TH style="text-align: right">Rig</TH><TH style="text-align:left">ht</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD><TD>25 sec</TD><TD>58°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD><TD>200 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:320px;">
					Afterwards we performed a Gel Electrophoresis, both colonies show correct results.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>AtNHXS1-eGFP:</STRONG> We performed another verification PCR of the 
					7 growing colonies, which failed, the Transformation was unsuccessful.
				</P><P class="NoteText" style="margin-top:10px;"><STRONG>BioBrick AtNHXS1-eGFP: </STRONG>We made Overnights of the 8 grown 
					colonies from the Transformation from 08/11. Purpose: Plasmid Prep.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/17</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>AtNHXS1: </STRONG>We performed verification PCRs 
					of the left part (+100bp from Genome) of 6 transformed 
					colonies BY4742 AVENA+AtNHXS1 (ATAVENA). 
				</P><P class="NoteText" style="float:left">
					PCR Mix: 
				</P><P class="NoteText" style="float:right; margin-right:40%;">
					PCR Program: 
				</P><TABLE id="Notebooktable" style="float:left; width:40%"><TBODY><TR id="Notebooktablepink"><TH style="border-right: 2px solid grey">Gene:</TH><TH class="thr" style="border-right: 2px solid grey">ATAVENA</TH></TR><TR><TD style="text-align: center"><STRONG>Component</STRONG></TD><TD>µl</TD></TR><TR><TD><STRONG>Template</STRONG></TD><TD>1 per Tube</TD></TR><TR><TD><STRONG>Primer f</STRONG></TD><TD> 1,2</TD></TR><TR><TD><STRONG>Primer r</STRONG></TD><TD>1,2</TD></TR><TR><TD><STRONG>One Taq 2x Mastermix</STRONG></TD><TD>30</TD></TR><TR><TD><STRONG>ddH<SUB>2</SUB>O</STRONG></TD><TD>21,6</TD></TR></TBODY></TABLE><TABLE id="Notebooktable" style="float:right; width:50%"><TBODY><TR><TH style="border-right: 2px solid grey">Gene:</TH><TH style="text-align: right">ATAV</TH><TH style="border-right: 2px solid grey">ENA</TH></TR><TR><TD style="text-align: center"><STRONG>PCR Step</STRONG></TD><TD>Time</TD><TD>Temperature</TD></TR><TR><TD><STRONG>ID Initial denaturation</STRONG></TD><TD>30 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>D Denaturation</STRONG></TD><TD>15 sec</TD><TD>94°C</TD></TR><TR><TD><STRONG>A Annealing</STRONG></TD><TD>25 sec</TD><TD>56°C</TD></TR><TR><TD><STRONG>E Extention</STRONG></TD><TD>110 sec</TD><TD>68°C</TD></TR><TR><TD><STRONG>FE Final Elongation</STRONG></TD><TD>5 min</TD><TD>68°C</TD></TR><TR><TD><STRONG>S Storage</STRONG></TD><TD>unlimited</TD><TD>8°C</TD></TR></TBODY></TABLE><P class="NoteText" style="margin-top:300px;">
					Gel Electrophoresis was successful for colonies 1 and 2.
				</P><P class="NoteText"><STRONG>AtNHXS1-eGFP: </STRONG>AtNHXS1-eGFP: Verification 
					PCRs of 7 growing colonies failed.
				</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/20</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG>Since we had many different mutant colonies where our 
					transformation did seem to work, we decided to store them in case something 
					went wrong along the way. We produced 10 different overnights for ENA1 K.O 
					transformations and wild type BY4742.
				</P><P class="NoteText"><STRONG>Interlab Study:</STRONG>We transformed samples of competent E. coli 
					cells. We transformed with three different controls; one negative, one positive 
					and another positive with a RFP marker. We produced 30 LB plates containing 
					chloramphenicol from which 18 plates were used.
				</P></DIV></DIV></DIV></DIV></DIV><A name="toWeek22">&lt;div class=&quot;white100&quot;&gt;&lt;/div&gt;</A><DIV class="container"><DIV class="row"><DIV class="col-md-3"><H1 class="NoteWeek">Week 22</H1></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/23</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">We performed two double digests of a) the AKT1-gene b) the AtSultr1;2 and c) the pSB1C3-backbone with EcoRI and PstI. Afterwards we ligated a) AKT1 with pSB1C3-backbone and b) the AtSultr1;2 with pSB1C3-backbone and finally performed the transformation of the ligated plasmids into competent E.coli DH5α.</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/25</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><P class="NoteText">We performed Overnights of the grown colonies of the transformation with the pSB1C3-AtSultr1;2-plasmid.</P></DIV></DIV><DIV class="row"><DIV class="col-md-2"><H1 class="NoteDay">10/27</H1></DIV></DIV><DIV class="row"><DIV class="col-md-11"><DIV class="row"><DIV class="col-md-11"><P class="NoteText"><STRONG>ENA1:</STRONG>Since we had many different mutant colonies where our 
					transformation did seem to work, we decided to store them in case something 
					went wrong along the way. We produced 10 different overnights for ENA1 K.O 
					transformations and wild type BY4742.
				</P><P class="NoteText"><STRONG>Interlab Study:</STRONG>We transformed samples of competent E. coli 
					cells. We transformed with three different controls; one negative, one positive 
					and another positive with a RFP marker. We produced 30 LB plates containing 
					chloramphenicol from which 18 plates were used.
				</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>