<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_Aachen_Design skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:Aachen/Design</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>iGEM Team Aachen 2017</TITLE></P><DIV class="container"><DIV class="collapse navbar-collapse" id="myNavbar"><UL class="nav navbar-nav"><LI class="active"><A href="https://2017.igem.org/Team:Aachen">Home</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/Project" class="dropdown-toggle" data-toggle="dropdown">Project</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Design">Design</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Applied_Design">Applied Design</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Biobricks">Biobricks</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Safety">Safety</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Results">Results</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Demonstrate">Demonstrate</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Outlook">Outlook</A></LI></UL><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/HP" class="dropdown-toggle" data-toggle="dropdown">Human Practices</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/HP">Overview</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/HP/Silver">Human Practices - Silver</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/HP/Gold_Integrated">Integrated Human Practices - Gold</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Collaborations">Collaborations</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/Lab" class="dropdown-toggle" data-toggle="dropdown">Lab</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Workflow">Workflow</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Protocols">Protocols</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Notebook">Notebook</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/InterLab">InterLab</A></LI><LI class="dropdown"><A href="https://2017.igem.org/Team:Aachen/Team" class="dropdown-toggle" data-toggle="dropdown">Team</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Attributions">Attributions</A></LI><LI class="dropdown"><A href="#">Special Awards</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Hardware">Hardware</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Plant">Plants</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Model">Model</A></LI><LI><A href="https://2017.igem.org/Team:Aachen/Engagement">Engagement</A></LI></DIV></DIV><DIV class="container-fluid subboxb" style="margin-top:50px;"><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DIV class="subboxSaf" style="font-size:28px;padding:12px;background-color:#aa0044;border-color:#aa0044; color:white;">Design</DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DIV class="SafText1"><DIV><STRONG>Introduction</STRONG></DIV><P style="text-align:justify;">
				The Design of a project is the product of the first big planning phase. It changes often throughout the project due to first findings in experimental data and a never-ending engineering cycle. 
				Opposit to our project description, this page contains information on all the transporters and channels we got in contact with throughout our project. Further down this page we also give some 
				information about why we chose S.cerevisiae as our chassis and why we integrated genes from Arabiddopsis thaliana. In addition to that, we also hand information about why we choose homologous 
				recombination, the loxP system, Easy Clone integration sites and an inducible promoter.
			</P><DIV class="row" style="margin-left:6%;margin-top:20px;"><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#cerervisiae">S.cerervisiae</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#thaliana">A.thaliana</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#homologous">Recombination</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#Easy">EasyClone</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#loxP">loxP System</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;"><P class="SafText" style="color:white;padding:5px;margin-bottom:0px;margin-left:40px;"><A style="text-decoration:none;color:white" href="#GAL1">GAL1</A></P></DIV></DIV><P style="text-align:justify;margin-top:40px;">
				In the very beginning of the planning- process we collect all kind of ideas of how we could design our project. Which organism is the best? Which molecular tools should be used? Which 
				thermodynamically efficiencies do the several transporters have? We read a lot of literature, design a first plan and consult our advisors to concretize our scheme. Afterwards, we start with 
				the modelling and the work in the laboratory. We have failures and successes regarding the aims of our project at first, which made us shift the design from time to time. The modelling and 
				the first lab- results cause that some parts were adapted and the project itself is adjusted in the direction the modeling and the lab- results guide us. Finally, after repeating this cycle a 
				few times, we get our latest project design and performed several tests to proof the results.  
			</P><P style="text-align:justify;">	
				Check out how we constructed our mutants!	
			</P><DIV style="border-bottom-style:solid;border-bottom-color:#003559;border-bottom-width:5px;margin-bottom:60px;"><P style="text-align:justify;">
				To help you understand better how we worked out the whole thing, we assembled a graphic description of the various genetic modifications.
			</P></DIV><P style="text-align:justify;">
				The native cell. Pictured transporters and channels are making the cell capable of a certain salt tolerance, but also in pumping ions out of the cell into the extracellular room again, 
				which is of course not in our interest. ENA1 and NHA1 code the main efflux mechanisms resulting in salt efflux as a reaction to salt stress (Figure 1).
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:100%;margin-left:%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.1: ENA1 and NHA1 code the 2 most important efflux mechanisms in 
			the plasma membrane of S.cerevisiae. They use ATP (ENA1) or the H+ gradient (AVP1) over the plasma membrane to active pump sodium out of the cytosol.</STRONG></P><P style="text-align:justify;">	
				The native transporters of higher interest to our project are highlighted in the graphic below (Figure 2). 
				How we alter these and which additionals genes we integrate will be explained in the following graphic.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:100%;margin-left:%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.2: The left graphic contains a selection of transporters 
			that are connected to the salt stress response and ion homeostasis in general. We alter the three most important parts of this system: </STRONG>ENA1: the primary active sodium pump is responsible 
			for most of the sodium/potassium efflux over the plasma membranNHA1: the secondary active antiporter uses the proton gradient over the plasma membrane to pump out sodium and is the second 
			most important part of the sodium efflux mechanism.NHX1 is responsible for the sequestration of sodium in the prevacuolar compartsments which convers salt tolerance.</P><P style="text-align:justify;">	 
				Our approach to increase the intracellular sequestration of ions in S. cerevisiae involves the alteration of the native salt- response and the implementation of the salt- tolerance mechanism 
				of the plant A. thaliana. Therefore, we modified our yeast in the following three ways: 
			</P><P style="text-align:justify;"><STRONG style="color:#008080">-	1.</STRONG><STRONG> The knockout of the native efflux mechanisms for sodium</STRONG><STRONG style="color:#008080">-	2. </STRONG><STRONG>The overexpression of the native mechanism responsible for the sequestration of ions in the prevacuolar compartments</STRONG><STRONG style="color:#008080">-	3. </STRONG><STRONG>The integration of the efficient mechanism that causes the sequestration of ions in the vacuole of Arabidopsis thaliana into the 	vacuolar membrane of S. cerevisiae</STRONG></P><P style="text-align:justify;">
				Beginning with the explanation of the alteration of the native- salt response we want to explain our general design. Figure 3 shows the native yeast cell with the three important transporters before alteration.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:37%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.3: native yeast cell with ENA1, NHA1 and NHX1</STRONG></P><P style="text-align:justify;">
				- The primary pathway for the extrusion of sodium ions in the plasma membrane is the P-type Na+ -ATPase encoded by Ena1 [1,2]. Knocking this gene out is the first big step to avert the efflux 
				of sodium out of the cell. Ena1 is crucial for the cell´s sodium efflux ability[3]. Therefore, the cell gets sensitive for salty media, because it loses a big amount of its natural capability 
				for salt tolerance [4,3].
			</P><P style="text-align:justify;">
				- The H+/Na+ antiporter, encoded by the Nha1 gene can extrude Na+, Li+ and K+ using the proton gradient over the plasma membrane [5,6]. The Nha1 gene knockout should have a similar effect to 
				the Ena1 knockout, denying the efflux of ions out of the cell.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:37%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.4: yeast cell with ENA1 and NHA1 knocked out</STRONG></P><P style="text-align:justify;">
				- Through overexpression of natural yeast Na+, K+/H+ exchanger we aim to strengthen and accelerate the ion- influx from the cytosol into intracellular compartments like the vacuole. 
				The protein encoded by the NHX1 gene in S. cerevisiae naturally transports sodium into prevacuolar compartments [7,8]. The VNX1- antiporter exchanges Na+ and K+ with H+ from the vacuole 
				and is located in the vacuolar membrane [9]. 
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:30%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.5: yeast cell with ENA1 and NHA1 knocked out and VNX1 und NHX1 overexpressed.</STRONG></P><P style="text-align:justify;">
				As previously stated, we want to sequestrate ions using the native salt response of our cells. In order to support and reinforce this system, we introduced parts of the salt-tolerance 
				mechanism of Arabidopsis Thaliana. 
			</P><P style="text-align:justify;">
				- In Arabidopsis thaliana, the proton electrochemical gradient across the vacuolar membrane (tonoplast) is partially generated by a vacuolar membrane bound H+-pyrophosphatase, which 
				is only encoded by a single gene, AVP1 [10]. The expression in yeast elevates the translocation of protons into the cellular vacuole [8]. This increased proton-influx in general leads 
				to a higher ion accumulation through H+-syn- and antiporters [11–13].
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:30%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.6: yeast cell with ENA1 and NHA1 knocked out and VNX1 und NHX1 overexpressed.AVP1 is integrated into the vacuolar membrane and increases the proton gradient over the membrane.</STRONG></P><P style="text-align:justify;">
				- In Arabidopsis thaliana the Na+/H+ antiporter AtNHX1 mediates the import of not only Na+, but also K+ into the vacuole. The transporter probably contributes to the accumulation of 
				cations significantly [12,13]. AtNHXS1 is a through DNA- shuffling improved version with an increased Na+/H+ Vmax [14]. We introduce the AtNHXS1- gene into our strain genome, so it 
				increases the sequestration of ions into the vacuole.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:30%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.7: yeast cell with ENA1 and NHA1 knocked out and VNX1 und NHX1 
			overexpressed.AVP1 is integrated into the vacuolar membrane and increases the proton gradient over the membrane. AtNHX1 is integrated into the vacuolar membrane vaciliating the seqeustration of 
			sodium using the protongradient that is increased by AVP1.</STRONG></P><P style="text-align:justify;">
				In Arabidopsis thaliana AtCLCc, a Cl-/H+ antiporter, localized in the vacuolar membrane, is responsible for the transport of Cl- across the vacoular membrane and therefore important regarding 
				salt tolerance [14,15]. We integrate the AtClCc gene in the yeast genome to increase accumulation of chloride in the vacuolar lumen, using the proton gradient generated by AVP1.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:30%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.8: yeast cell with ENA1 and NHA1 knocked out and VNX1 und NHX1 
			overexpressed.AVP1 is integrated into the vacuolar membrane and increases the proton gradient over the membrane. AtNHX1 is integrated into the vacuolar membrane vaciliating the seqeustration of 
			sodium using the protongradient that is increased by AVP1.AtCLCc was integrated to further increase the accumulation of NaCl into the vacuole.</STRONG></P><P style="text-align:justify;">
				As we decided, in result of our Human Practices work, to show the adaptability of the concept to other ions, we designed more cells which should be capable of sequestrating potassium.
			</P><P style="text-align:justify;">
				The AtAKT1 gene from Arabiddopsis thaliana mediates potassium uptake over a wide range of potassium concentration in the extracellular room [24–27]. AtAKT1 specifically is a high affinity 
				transporter in the plasma membrane [28–30]. Since sodium has a negative effect on the efficiency of AtAKT1 [31], we did not combine a sodium accumulating mutant with this channel but rather 
				designed a separate mutant called NAKT. Therefore, we integrated the channel in the plasma membrane of a mutant with ENA1 already knocked out. We did this, because the gene not only mediates 
				efflux of sodium but also transports potassium back into the extracellular room. 
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:30%;margin-top:5px;margin-bottom:60px;"><STRONG>Fig.9: The NAKT mutant: NHA1 and ENA1 are knocked out since they are also increase potassium efflux. AtAKT1 is integrated into the plasma membrane to actively import potassium into the cytosol. AtAKT1 consumes ATP. 

			</STRONG></P><DIV class="row" style="margin-left:6%;margin-top:90px;"><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#cerervisiae">S.cerervisiae</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#thaliana">A.thaliana</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#homologous">Recombination</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#Easy">EasyClone</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;border:"><P class="SafText" style="color:white;padding:5px;margin:0px;"><A style="text-decoration:none;color:white" href="#loxP">loxP System</A></P></DIV><DIV class="col-md-2" style="width:15%;backgroud-style:solid;background-color:#415a77;"><P class="SafText" style="color:white;padding:5px;margin-bottom:0px;margin-left:40px;"><A style="text-decoration:none;color:white" href="#GAL1">Promoter</A></P></DIV></DIV><DIV id="cerervisiae"><STRONG>Why <I>Saccharomyces cerervisiae?</I></STRONG></DIV><P style="text-align:justify;">
				Why did we choose S. cerevisiae as the organism to work with? There are a few very striking arguments:
			</P><P style="text-align:justify;"><STRONG style="color:#008080;">1.</STRONG> Our project aims to remove salts from industrial process water. It is therefore crucial that our organism can survive and be <STRONG style="color:#008080;">cultivated in liquid media</STRONG>. Since we rely on a eukaryotic organism 
				(see 2) and plants are unable to grow in pure water, we decided to work with a yeast.
			</P><P style="text-align:justify;"><STRONG style="color:#008080;">2.</STRONG> We intend to accumulate as much NaCl inside the cell as possible. Sodium is toxic and at higher concentrations becomes lethal [16].  It substitutes potassium as a cofactor in 
				enzymes and inhibits specific cellular targets [17]. That is why it is essential to remove sodium from the cytosol, making <STRONG style="color:#008080;">storage compartments like the vacuole</STRONG> of S. cerevisiae 
				irreplaceable for our project.
			</P><P style="text-align:justify;"><STRONG style="color:#008080;">3.</STRONG> Since we expose our mutants to a variation of significantly increased concentrations of salts, a certain <STRONG style="color:#008080;">native salt tolerance</STRONG> is necessary. S.cerevisiae is able to grow in a 
				wide range of environments conditions and can tolerate high external concentrations of potassium and sodium [16]. Additionally, S.cerervisiae does not struggle to grow at 
				concentrations found in industrial process water where we see the application of our project.
			</P><P style="text-align:justify;"><STRONG style="color:#008080;">4.</STRONG> S.cerevisiae is a well described <STRONG style="color:#008080;">model organism</STRONG> in general. Fast growth and a broad range of genetic tools to prepare mutants enable the fast progress in laboratory work, 
				which is necessary for iGEM. Being the first eukaryotic organism with a fully annotated and sequenced genome [18] it also became the model organism to study alkali metal 
				cation homeostasis and lead to identification of most transporters included in this complexly regulated system. All these transporters have been predicted in silico [19], 
				which helped us in modelling our system. 
			</P><DIV id="thaliana"><STRONG>Why <I>Arabidopsis thaliana?</I></STRONG></DIV><P style="text-align:justify;">
				Why did we use the seqeustration mechanism of A. thaliana for the integration into the vacuolar membrane. 
			</P><P style="text-align:justify;">
				Arabidopsis thaliana is one of the best described plants and is well known as model organism in plant physiology. A lot of research has been done in the past to decipher 
				the different genes of A. thaliana [22,23]. Moreover, the main mechanisms occurring salt tolerance, for example the ion channels responsible for a fast efflux of Na+ out 
				of the cell, are well described. Main cellular metabolic processes in A. thaliana and yeast are similar, so genes can be transferred easily from one organism to the other. 
			</P><P style="text-align:justify;">
				Allgemeiner Fact: müssen wir noch irgendwo einbringen.
			</P><P style="text-align:justify;">
				In yeast cells, transport systems exist at both the plasma and organellar membranes, which mediates alkali metal cation fluxes with different substrate specificities and using 
				diverse mechanisms (e.g. primary active ATPases, secondary active symporters and antiporters, and passive channels).
			</P><DIV id="homologous"><STRONG>Homologous Recombination</STRONG></DIV><P style="text-align:justify;">
				Homologous recombination is a powerful tool to make genome editing. It is a well-established technique for S. cerevisiae and generates integrations into the genome, that are stable 
				through generations. Such a characteristic is a big advantage in comparison to plasmid-systems. It also prevents horizontal gene transfer, which would be an issue with the plasmid 
				system. Especially when looking at antibiotic- resistances, but also at other genes, this is a big argument for us in terms of safety and responsibility to use homologous recombination in the lab.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:50%;margin-left:25%;margin-top:5px;margin-bottom:70px;"><STRONG>(A) Homologous Recombination results in the ingetration 
			of our gene casette into the genome (B)</STRONG></P><P style="text-align:justify;">
				For the application it is incontrovertible that a stable integration into the genome is the right choice. The lower chance for horizontal gene transfer is in application even more 
				important than in the lab. High conservation of introduced genes is significant here, because loss or mutation of any introduced gene could easily lead to dysfunction. Moreover, 
				there is no need to maintain selection pressure with e.g. use of antibiotics in combination with antibiotic resistant cells. Another advantage is that we can use our lab strains 
				directly for application- testing.
			</P><DIV id="Easy"><STRONG>Easy Clone Recombination Sites</STRONG></DIV><P style="text-align:justify;">
				Genome editing using homologous recombination needs good recombination sites. The sites have great influence on the level of conservation an introduced gene- fragment has and the success- rate of a gene introduction in the laboratory.
			</P><P style="text-align:justify;">
				Therefore, we concluded the recombination sites given by the Easy Clone system [20] as very promising. The authors pledge that the easy clone recombination sites are located in 
				highly conserved regions on the chromosomes. Additionally, they validated them in their studies in terms of successrate of gene introductions.
			</P><DIV><STRONG>&quot;LoxP&quot; - System</STRONG></DIV><P id="loxP" style="text-align:justify;">
				For lab, safety and responsibility reasons we decided to remove all marker genes we transformed completely. To do so without removing our transformed genes, we used the CreLox recombination system.
			</P><P class="SafText" style="text-align:justify;font-size:100%;width:40%;margin-left:30%;margin-top:5px;margin-bottom:70px;"><STRONG>The loxP system is used to remove antibiotic resistances from the genome.The GOI outside of the loxP casette remains intact while the removed antibiotic resistance is degraded.</STRONG></P><P style="text-align:justify;">
				We placed a loxP site on each side in between the marker gene and the homologous regions. Once the transformation and selection had worked, we removed the resistance using the CreLox system. In the genomic integration cassette, the loxP sites were directly beside the marker gene. What we ended up with were completely transformed cells without any of the antibiotic resistance genes used to mark them. 
			</P><DIV id="GAL1"><STRONG>GAL1</STRONG></DIV><P style="text-align:justify;"> 
				For detailed information about the promoter we used you can visit the <A href="http://parts.igem.org/Part:BBa_J63006 " style="text-decoration:none;color:#aa0044;"><STRONG>part page</STRONG></A> in the iGEM registry.
			</P><P style="text-align:justify;">			In all integrations we carried out (overexpressions of native genes, overexpression and integration of foreign genes), the genes were equipped with a GAL1 promotor upstream. The GAL1 promotor allows sensitive adjustability of the expression rate through the Glucose/Galactose ratio in the cultivation media [21]. Thus, it is possible to regulate to the best expression rate of the introduced proteins by experimenting. Just strongly overexpressing the membrane- bound transporter proteins, as we are working with, can get adversely for cell in terms of survivability in salty media.
			</P></DIV></DIV></DIV></DIV><DIV class="container-fluid subboxb" style="margin-top:50px; margin-bottom:20px;"><DIV class="container"><DIV class="row"><DIV class="col-xs-12"><DIV class="subboxSaf" style="background-color:grey;border-color:grey; color:white;">References</DIV></DIV></DIV></DIV></DIV><DIV class="container"><DIV class="row"><DIV class="col-xs-12 SafText1"><P style="text-align:justify;">
				[1]	A. Ruiz and J. Ariño, “Function and regulation of the Saccharomyces cerevisiae ENA sodium ATPase system,” Eukaryotic cell, vol. 6, no. 12, pp. 2175–2183, 2007.
			</P><P style="text-align:justify;">
				[2]	R. Haro, B. Garciadeblas, and A. Rodríguez-Navarro, “A novel P-type ATPase from yeast involved in sodium transport,” FEBS letters, vol. 291, no. 2, pp. 189–191, 1991.
			</P><P style="text-align:justify;">	
				[3]	R. A. Gaxiola, R. Rao, A. Sherman et al., “The Arabidopsis thaliana proton transporters, AtNhx1 and Avp1, can function in cation detoxification in yeast,” Proceedings of the National Academy of Sciences, vol. 96, no. 4, pp. 1480–1485, 1999.
			</P><P style="text-align:justify;">	
				[4]	B. Garciadeblas, F. Rubio, F. J. Quintero et al., “Differential expression of two genes encoding isoforms of the ATPase involved in sodium efflux in Saccharomyces cerevisiae,” Molecular and General Genetics MGG, vol. 236, no. 2, pp. 363–368, 1993.
			</P><P style="text-align:justify;">	
				[5]	M. A. Bañuelos, H. Sychrová, C. Bleykasten-Grosshans et al., “The Nha1 antiporter of Saccharomyces cerevisiae mediates sodium and potassium efflux,” Microbiology (Reading, England), 144 (Pt 10), pp. 2749–2758, 1998.
			</P><P style="text-align:justify;">	
				[6]	C. Prior, S. Potier, J. L. Souciet et al., “Characterization of the NHA1 gene encoding a Na+/H+-antiporter of the yeast Saccharomyces cerevisiae,” FEBS letters, vol. 387, no. 1, pp. 89–93, 1996.
			</P><P style="text-align:justify;">
				[7]	R. Nass and R. Rao, “Novel Localization of a Na + /H + Exchanger in a Late Endosomal Compartment of Yeast,” Journal of Biological Chemistry, vol. 273, no. 33, pp. 21054–21060, 1998.
			</P><P style="text-align:justify;">	
				[8]	R. Nass, K. W. Cunningham, and R. Rao, “Intracellular Sequestration of Sodium by a Novel Na + /H + Exchanger in Yeast Is Enhanced by Mutations in the Plasma Membrane H + -ATPase,” Journal of Biological Chemistry, vol. 272, no. 42, pp. 26145–26152, 1997.
			</P><P style="text-align:justify;">
				[9]	O. Cagnac, M. Leterrier, M. Yeager et al., “Identification and characterization of Vnx1p, a novel type of vacuolar monovalent cation/H+ antiporter of Saccharomyces cerevisiae,” The Journal of biological chemistry, vol. 282, no. 33, pp. 24284–24293, 2007.
			</P><P style="text-align:justify;">
				[10]	H. Sze, “Energization of Plant Cell Membranes by H+-Pumping ATPases: Regulation and Biosynthesis,” THE PLANT CELL ONLINE, vol. 11, no. 4, pp. 677–690, 1999.
			</P><P style="text-align:justify;">	
				[11]	R. A. Gaxiola, R. Rao, A. Sherman et al., “The Arabidopsis thaliana proton transporters, AtNhx1 and Avp1, can function in cation detoxification in yeast,” Proceedings of the National Academy of Sciences of the United States of America, vol. 96, no. 4, pp. 1480–1485, 1999.
			</P><P style="text-align:justify;">	
				[12]	R. A. Gaxiola, J. Li, S. Undurraga et al., “Drought- and salt-tolerant plants result from overexpression of the AVP1 H+-pump,” Proceedings of the National Academy of Sciences of the United States of America, vol. 98, no. 20, pp. 11444–11449, 2001.
			</P><P style="text-align:justify;">
				[13]	X.-G. Duan, A.-F. Yang, F. Gao et al., “Heterologous expression of vacuolar H(+)-PPase enhances the electrochemical gradient across the vacuolar membrane and improves tobacco cell salt tolerance,” Protoplasma, vol. 232, 1-2, pp. 87–95, 2007.
			</P><P style="text-align:justify;">	
				[14]	M. Jossier, L. Kroniewicz, F. Dalmas et al., “The Arabidopsis vacuolar anion transporter, AtCLCc, is involved in the regulation of stomatal movements and contributes to salt tolerance,” The Plant journal : for cell and molecular biology, vol. 64, no. 4, pp. 563–576, 2010.
			</P><P style="text-align:justify;">	
				[15]	D. Geelen, C. Lurin, D. Bouchez et al., “Disruption of putative anion channel gene AtCLC-a in Arabidopsis suggests a role in the regulation of nitrate content,” The Plant Journal, vol. 21, no. 3, pp. 259–267, 2000.
			</P><P style="text-align:justify;">	
				[16]	J. Ariño, J. Ramos, and H. Sychrová, “Alkali metal cation transport and homeostasis in yeasts,” Microbiology and molecular biology reviews : MMBR, vol. 74, no. 1, pp. 95–120, 2010.
			</P><P style="text-align:justify;">	
				[17]	“Salt Tolerance in Plants and Microorganisms: Toxicity Targets and Defense Responses,” (Keine Angabe), vol. 165, pp. 1–52, 1996.
			</P><P style="text-align:justify;">	
				[18]	A. Goffeau, B. G. Barrell, H. Bussey et al., “Life with 6000 genes,” Science (New York, N.Y.), vol. 274, no. 5287, 546, 563-7, 1996.
			</P><P style="text-align:justify;">	
				[19]	B. Nelissen, R. de Wachter, and A. Goffeau, “Classification of all putative permeases and other membrane plurispanners of the major facilitator superfamily encoded by the complete genome of Saccharomyces cerevisiae,” FEMS microbiology reviews, vol. 21, no. 2, pp. 113–134, 1997.
			</P><P style="text-align:justify;">	
				[20]	N. B. Jensen, T. Strucko, K. R. Kildegaard et al., “EasyClone: method for iterative chromosomal integration of multiple genes in Saccharomyces cerevisiae,” FEMS yeast research, vol. 14, no. 2, pp. 238–248, 2014.
			</P><P style="text-align:justify;">	
				[21] http://parts.igem.org/Part:BBa_J63006
			</P><P style="text-align:justify;">	
				[22]	J. M. van Norman and P. N. Benfey, “Arabidopsis thaliana as a model organism in systems biology,” Wiley interdisciplinary reviews. Systems biology and medicine, vol. 1, no. 3, pp. 372–379, 2009.
			</P><P style="text-align:justify;">	
				[23]	J.-K. Zhu, “Genetic Analysis of Plant Salt Tolerance Using Arabidopsis: Fig. 1,” Plant Physiology, vol. 124, no. 3, pp. 941–948, 2000.
			</P><P style="text-align:justify;">
				[24]    R. E. Hirsch, B. D. Lewis, E. P. Spalding et al., “A role for the AKT1 potassium channel in plant nutrition,” Science (New York, N.Y.), vol. 280, no. 5365, pp. 918–921, 1998.
			</P><P style="text-align:justify;">	
				[25]    D. Lagarde, M. Basset, M. Lepetit et al., “Tissue-specific expression of Arabidopsis AKT1 gene is consistent with a role in K+ nutrition,” The Plant journal : for cell and molecular biology, vol. 9, no. 2, pp. 195–203, 1996.
			</P><P style="text-align:justify;">	
				[26]    H. Sentenac, N. Bonneaud, M. Minet et al., “Cloning and expression in yeast of a plant potassium ion transport system,” Science (New York, N.Y.), vol. 256, no. 5057, pp. 663–665, 1992.
			</P><P style="text-align:justify;">	
				[27]    E. P. Spalding, R. E. Hirsch, D. R. Lewis et al., “Potassium uptake supporting plant growth in the absence of AKT1 channel activity: Inhibition by ammonium and stimulation by sodium,” The Journal of general physiology, vol. 113, no. 6, pp. 909–918, 1999.
			</P><P style="text-align:justify;">	
				[28]    H. H. Fu and S. Luan, “AtKuP1: A dual-affinity K+ transporter from Arabidopsis,” The Plant cell, vol. 10, no. 1, pp. 63–73, 1998.
			</P><P style="text-align:justify;">	
				[29]    G. E. Santa-María, F. Rubio, J. Dubcovsky et al., “The HAK1 gene of barley is a member of a large gene family and encodes a high-affinity potassium transporter,” The Plant cell, vol. 9, no. 12, pp. 2281–2289, 1997.
			</P><P style="text-align:justify;">	
				[30]    F. Rubio, W. Gassmann, and J. I. Schroeder, “Response: High-affinity potassium uptake in plants,” Science (New York, N.Y.), vol. 273, no. 5277, pp. 978–979, 1996.
			</P><P style="text-align:justify;">
				[31]    J. K. Zhu, “Regulation of ion homeostasis under salt stress,” Current opinion in plant biology, vol. 6, no. 5, pp. 441–445, 2003.
			</P><P style="text-align:justify;">
				[32]    N. Yoshimoto, E. Inoue, A. Watanabe-Takahashi et al., “Posttranscriptional regulation of high-affinity sulfate transporters in Arabidopsis by sulfur nutrition,” Plant physiology, vol. 145, no. 2, pp. 378–388, 2007.
			</P><P style="text-align:justify;">	
				[33]    M. Barberon, P. Berthomieu, M. Clairotte et al., “Unequal functional redundancy between the two Arabidopsis thaliana high-affinity sulphate transporters SULTR1;1 and SULTR1;2,” The New phytologist, vol. 180, no. 3, pp. 608–619, 2008.
			</P><P style="text-align:justify;">	
				[34]    T. Kataoka, A. Watanabe-Takahashi, N. Hayashi et al., “Vacuolar Sulfate Transporters Are Essential Determinants Controlling Internal Distribution of Sulfate in Arabidopsis,” The Plant cell, vol. 16, no. 10, pp. 2693–2704, 2004.
			</P></DIV></DIV></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>