Team:Aachen/Protocols
iGEM Team Aachen 2017
Home
Project
Description
Design
Applied Design
Biobricks
Safety
Results
Demonstrate
Outlook
Human Practices
Overview
Human Practices - Silver
Integrated Human Practices - Gold
Collaborations
Lab
Workflow
Protocols
Notebook
InterLab
Team
Team
Attributions
Special Awards
Hardware
Plants
Model
Engagement
Protocols
Protocol mRNA-isolation and RT-PCR
General:
- This protocol was designed to isolate mRNA from a yeast culture and perform a RT-qPCR for proofing gene expression of a specific gene.
- When working with mRNA only use nuclease-free chemicals and materials. Wear gloves and change them frequently. Do not talk to prevent the mRNA from RNases
- Protocol for mRNA isolation was taken from “RNaesy Mini Handbook” (Qiagen). Respective chemicals and buffers are included in this kit.
Specific for our use:
- Following strains were tested:
o Wildtype
o AVP1-mutant with not induced promotor (grown in YPD with glucose)
o AVP1-mutant with induced promotor (grown in YPD with galactose)
- mRNA-isolation and all steps except of RT-qPCR were performed in duplicates per strain
- RT-qPCR itself was performed in triplicates per sample
- For reference we used a housekeeping gen. We chose the yeast gene ACT1.
- Primer pairs used in the RT-qPCR are designed to amplify a gene fragment which is 100 bases in length within the AVP1 gene  und the housekeeping gene ACT1 each
Preparation of cells:
- Inoculate 10 ml YPD containing 100% galactose and 10 ml normal YPD for reference with a cryo-culture of yeast cells and let it growth at 28 C°, 150 rpm for 12-16 hours (preculture)
- Inoculate 50 ml YPD containing 100% galactose or normal YPD with 2 ml of preculture and let it grow until OD600 0,4
Cell lysis and mRNA-isolation
- Media complementation right before cell lysis
o Add 0,1% β-mercaptoethanol and zymolyase to prepared Y1-buffer-stock-solution
 Recommendations: For cell lysis of 50 ml culture (OD600 0,4) you will need 2ml Y1-stock-solution, 2 µl β-mercaptoethanol and 5 mg zymolyase (20800 units)
- Centrifuge cells in 50 ml flask at 1000xg, 4C° for 5 min
- Discard supernatant
- Resuspend cells in 100 µl nuclease-free-water and transfer to a 2 ml Eppendorf-reaction-tube
- Centrifuge at 1000xg for 1 min
- Discard supernatant
- Resuspend cells in 2 ml Y-buffer and incubate at 35 C°, 300 rpm for 20 min (conditions depending on the enzyme which is used)
- Centrifuge cells at 300 rpm for 5 min to sediment originated spheroplasts
- Discard supernatant
- Add 350µl buffer RLT and vortex vigorously to destroy spheroplasts
- Add 350µl ethanol (70% (v/v)) and mix while pipetting
- Transfer solution to an RNeasy-column siting in 2ml tube
- Close lid and centrifuge at 8000xg for 15 s
- Discard supernatant
- Add 700µl buffer RW1 to the column, centrifuge at 8000xg for 15s
- Discard supernatant
- Add 500µl buffer RPE and centrifuge at 8000xg for 15s
- Discard supernatant
- Add 500µl RPE and centrifuge at 8000xg for 2min
- Place column in a new tube and centrifuge at full speed for 1min
- Discard supernatant
- Place column in a new tube and add 40µl nuclease-free-water
- Centrifuge at 10000 rpm for 1min
- Discard column
Preparation of RT-qPCR
- Measure mRNA concentration in eluate
- Extract 3 µg of mRNA (respective µl) and place in nuclease-free Eppendorf reaction tube for DNA digestion
- Add 3µl DNaseI, 3 µl Dnase buffer and fill up to 30µl with nuclease-free water
- Incubate mixture at 37 C° for 30min
- Add 3 µl EDTA (25 mM)
- Incubate at 75 C° for 10min
- Extract 11 µl (=1µg) mRNA-mixture and place in a nuclease-free Eppendorf reaction tube, add 14 µl nuclease-free water and 2 µl primer (0,2µM; dNTP-primer for binding at polyadenylation-sequence of mRNA; here olio dTprimer PVX)
- Incubate at 80 C° for 10min
- Cool on ice shortly
- Transfer mixture to a pcr-tube
- Add 1 µl M-MLV (100 Units, reverse transcriptase), 4µl 5X M-MLV-buffer, 0,5mM dNTPs and fill up to 50µl with nuclease-free water
- Perform a pcr with following conditions:
o 1. 40 C° for 30min
o 2. 45 C° for 20min
o 3. 50 C° for 20min
o 4. 55 C° for 20min
o 5. 75 C° for 15min
o Conditions depend on length of mRNA
o No repetitions
- Add 1µl RNaseI and incubate at 37 C° for 1hour to digest RNA
- Prepare mixture for RT-qPCR
o Extract 5µl of mRNA mixture and transfer to Eppendorf reaction tube
o Add 300nM of each primer pair (primers for housekeeping gen and primers for gen to be verified)
o Add 10µl SYBr-Green
o Fill up to 20µl with ddH20
o Transfer your samples to 96 well-plate for RT-qPCR
- Perform a pcr with following conditions
o 1. 95 C° for 3min
o 2. 95 C° for 15s
o 3. 55 C° for 30s              repeat 40 times
o 4. 72 C° for 30s
o Conditions depend on the properties of the used primers and on the length of the amplified sequence
- Perform a melting curve of the originated cDNA product to check the right amplification (conditions depend on the machine you are using)
Media:
- Buffer Y1-stock solution:
o 1M Sorbitol
o 0,1M EDTA
o Adjust the pH value to pH = 7,4 with HCL and NaOH and autoclave
 Example: To produce 50ml Y1-stock-solution:
9,109g Sorbit ( n = 1mol/L * 0,05 L = 0,05mol ; m = n * M = 0,05mol * 182,18g/mol = 9,109g)
1,8612g EDTA ( n = 0,1mol/L * 0,05 L = 0,005mol ; m = n * M = 0,005mol * 372,24g/mol = 1,8612g)
- Buffer Y1-working-solution
o Y1-stock-solution
o 0,1% β-mercaptoethanol and zymolyase (calculate 5mg zymolyase per 50 ml cell culture with OD600 0,4)
References:
- Script “Real Time PCR, Applications Guide” (Biorad)
- RNeasy Mini Handbook (Qiagen)
- Dissertation of Christina Dickmeis (RTWH Aachen)
Cell preparation and SDS-PAGE
To proof the transcription of our integrated AVP1-DNA, we performed a SDS-PAGE gel electrophoresis.
Protocol for preparation of cells:
- Prepare an overnight-culture of your cells in 10 ml YPD-media and incubate at 28 C° and 250 rpm
o We prepared four overnight-cultures:
 Wildtype (S. cerevisiae BY4742) in YPD-media
 AVP1-mutant in YPD-media
 AVP1-mutant in YPD-media
 AVP1-mutant in YPD-media with galactose instead of glucose
- After 12 hours of incubation inoculate 50 ml of YPD-media with the entire overnight-culture and let it grow at 28 C°, 250 rpm until OD600 0,4 is reached
o We inoculated as followed:
 Overnight-culture (1) we inoculated in 50 ml YPD
 Overnight-cuture (2) we inoculated in 50  ml YPD
 Overnight-culture (3) we inoculated in 50 ml YPD with galactose instead of glucose
 Overnight-culture (4) we inoculated in 50 ml YPD with galactose instead of glucose
- Calculate total OD-values: Multiple the measured with the culture volume (is needed later)
- Centrifuge cells in 50 ml flasks at 1000 xg, 4 C° for 5 min, discard supernatant and resuspend pellet in 50 mml cold water
- Centrifuge cells again (1000 xg, 5 min, 5 min) and discard supernatant
- Freeze cells at -80 C° or continue with “Protocol cell disruption and gel loading”
Protocol cell disruption and gel loading
- Heat “cracking buffer” at 60 C°
- Resuspend (frozen) cell pellet in cracking buffer (100 µl of cracking buffer per 7,5 total OD600)
- Place mixture in 1,5 ml screw cap centrifuge tubes, filled with 80 µl glass beads (0,5 mm diameter) per 7,5 total OD 600
- Heat sample at 70 C° for 10 min
- Vortex for 1 min
- Centrifuge cells at 14000 rpm, 4 C° for 5 min
- Place supernatant (1) on ice and continue protocol with pellet
- Heat pellet at 100 C° for 3-5 min (eventually add new cracking buffer if necessary)
- Vortex for min
- Centrifuge cells at 14000 rpm, 4 C° for 5 min
- Place supernatant on ice (2)
- Boil both supernatants (1 and 2) shortly (2 min) in water bath and immediately load SDS-gel
o We received eight samples and loaded on gel
Notes:
- Important: Add 100 x PMSF every 7 minutes during the procedure to inhibit proteases (Use 1 µl PMSF per 100 µl Cracking buffer
- All centrifugation steps were conducted in a benchtop centrifuge
- All heating steps were conducted in a heat block
Supplements of cracking buffer:
- Cracking buffer stock solution:
Chemicals - Final concentration - To prepare 100 ml
Urea - 8 M - 48g
SDS - 5 % m/v - 5g
Tris HCl (pH 6,8) - 40 mM - 4 ml of 1M stock solution
EDTA - 0,1 mM - 20 µl of 0,5 M stock solution
Bromphenol Blau - 0,4 mg/ml - 40 mg
ddH2O - To final volume of 100 ml
- Cracking buffer (used for protocol):
Cracking buffer stock solution - 1 ml
Beta-mercaptoethanol - 10µl
PMSF - 50µl
ddH2O - 70 µl
- 0,0348 g PMSF (solid) was solved in 2 ml isopropanol
Reference for this Protocol: “Yeast Protocol Handbook”
Plasmid DNA Purification
This protocol is designed for purification of up to 20 µg of high-copy plasmid DNA from 1-5ml overnight cultures of E.coli in LB medium. Here a purification kit from “QIAGEN” is used.
Note: All protocol steps should be carried out at room temperature (15-25*C).
Procedure:
1. Resusped pelleted bacterial cells in 250µl Buffer P1 and transfer to a microcentrifuge tube.
- Ensure that RNase A has been added to Buffer P1. No cell clumps should be visible after resuspension of the pellet.
- If LyseBlue reagent has been added to Buffer P1, vigorously shake the buffer bottle to ensure LyseBlue particles are completely dissolved. The bacteria should be resuspended completely by vortexing or pipetting up and down until no cell clumps remain.
2. Add 250 µl Buffer P2 and mix thorougly by inverting the tube 4-6 times.
- Mix gently by inverting the tube. Do not vortex, as this will result in shearing of genomic DNA. If necessary, continue inverting the tube until the solution becomes viscous an slightly clear. Do not allow the lysis reaction to proceed for more than 5 min.
- If LyseBlue has been added to Buffer P1 the cell suspension will turn blue after addition of Buffer P2. Mixing should result in a homogeneously colored suspension.
- If the suspension contains localized colorless regions of if brownish cell clumps are still visible, continue mixing the solution until a homogeneously colored suspension is achieved.
3. Add 350 µl Buffer N3 and mix immediately and thoroughly by inverting the tube 4-6 times.
- To avoid localized precipitation, mix the solution thoroughly, immediately after addition of Buffer N3. Large culture volumes (e.g. > 5ml) may require inverting up to 10 times. The solution should become cloudy.
- If LyseBLue reagent has been used, the suspension should be mixed until all trace of blue has gone and the suspension is colorless. A homogeneous colorless suspension incated that the SDS has been effectively precipitated.
4. Centrifuge for 10 min at 13,000 rpm (~17,900 x g) in a table-top microcentrifuge.
- A compact white pellet will form
5. Apply 800 µl pf the supernatant from step 4 to the QIAprep 2.0 spin column by pipetting.
6.Centrifuge for 30-60 s. Discard the flow-through.
7. Recommended: Wash the QIAprep 2.0 spin column by adding 0,5 ml Buffer PB and centrifuging for 30-60 s. Discard the flow-through.
- This step is necessary to remove trace nuclease activity when using endA+ strains such as the JM series, HB101 and its derivatives, or any wild-type strain, which have high levels of nuclease activity or high carbohydrate content. Host strains such as XL-1 Blue and DH5a do not require this additional wash step.
8. Wash QIAprep 2.0 spin column by adding 0,75 ml Buffer PE and centrifuging for 30-60 sec.
9. Discard the flow-through, and centrifuge at full speed for additional 1 min to remove residual wash buffer.
- important: Residual wash buffer will not be completely removed unless the flow-through is discarded before this additional centrifugation. Residual ethanol from Buffer PE max inhibit subsequent enzymatic reactions.
10. Place the QIAprep 2.0 column in a clean 1,5 ml microcentrifuge tube to elute DNA, add 50 µl Buffer EB (10mM Tris-Cl, pH 8.5) or water to the center of each QIAprep 2.0 spin column, let stand for 1 min, and centrifuge for 1 min.
PCR Clean-Up
1. Adjust DNA binding conditions
- For very small sample volumes <30 µl adjust the volume of the reaction mixture to 50-100 µl with water. It is not necessary to remove mineral oil.
- Mix 1 volume of sample with 2 volumes of Buffer NTI (e.g., mix 100 µl PCR reaction and 200 µl Buffer NTI).
- Note: For removal of small fragments like primer dimers dilutions of Buffer NTI can be used instead of 100% Buffer NTI. Please refer to section 2.3.
2. Bind DNA
- Place a NucleoSpin Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 µl sample.
- Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube.
- Load remaining sample if necessary and repeat the centrifugation step.
3. Wash silica membrane
- Add 700 µl Buffer NT3 to the NucleoSpin Gel and PCR Clean-up Column. Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube.
- Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and improve A(260)/A(230) values (see section 2.7 for detailed information).
4. Dry silica membrane
- Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely. Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube.
- Note: Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2-5 min at 70°C prior to elution.
5. Elute DNA
- Place the NucleoSpin Gel and PCR Clean-up Column into a new 1,5 mL microentrifuge tube (not provided). Add 15-30 µl Buffer NE and incubate at room temperature (18-25°C) for 1 min at 11,000 x g.
- Note: DNA recovery of larger fragments (>1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min. See section 3,6 for detailed information.
Agarose Gel Clean-Up
PCR-cleanup kit from “NucleoSpin” is used.
1. Excise DNA fragment/solubilize gel slice
Note: Minimize UV exposure time to avoid damaging the DNA.
Take a clean scalpel to excise the DNA fragment from an agarose gel. Remove all excess agarose.
Determine the weight of the gel slice and transfer it to a clean tube
For each 100 mg of agarose gel < 2% add 200 µL Buffer NTI.
For gels containing <2% agarose, double the volume of Buffer NTI.
Incubate sample for 5-10 min at 50°C. Vortex the sample briefly every 2-3 min until the gel slice is completely dissolved!
2. Bind DNA
Place a NucleoSpin Gel and PCR Clean-up Column into a Collection Tube (2 ml) and load up to 700 µl sample.
Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube.
Load remaining sample if necessary and repeat the centrifugation step.
3. Wash silica membrane            Add 700 µl Buffer NT3 to the NucleoSpin Gel and PCR Clean-up Column. Centrifuge for 30 s at 11,000 x g. Discard flow-through and place the column back into the collection tube.
Recommended: Repeat previous washing step to minimize chaotropic salt carry-over and improve A(260)/A(230) values (see section 2.7 for detailed information).
4. Dry silica membrane
Centrifuge for 1 min at 11,000 x g to remove Buffer NT3 completely. Make sure the spin column does not come in contact with the flow-through while removing it from the centrifuge and the collection tube.
Note: Residual ethanol from Buffer NT3 might inhibit enzymatic reactions. Total removal of ethanol can be achieved by incubating the columns for 2-5 min at 70°C prior to elution.
5. Elute DNA
Place the NucleoSpin Gel and PCR Clean-up Column into a new 1,5 mL microentrifuge tube (not provided). Add 15-30 µl Buffer NE and incubate at room temperature (18-25°C) for 1 min at 11,000 x g.
Note: DNA recovery of larger fragments (>1000 bp) can be increased by multiple elution steps with fresh buffer, heating to 70°C and incubation for 5 min. See section 3,6 for detailed information.
Transformation of E.coli
work space: clean bench
conservation: 4°C (after 16-18 hours 37°C)
1. Thaw 50 µL of E. coli Omp8 / 100 µL E. coli DH5α or E. coli BL21 (DE3) Gold on ice
2. Add DNA (2-400 ng/µL: 50 ng if Plasmid DNA; 2 µL (~20ng) ligation mixture; 1-4 µL PLICing-reaction) to the competent cells and swirl gently
3. Incubate for 15-30 min on ice.
4. Perform heatshock of the competent cells using a preheated water bath at 42°C for 90 s (Omp 8) or 45s (DH5α, BL21).
5. Samples were immediately cooled down 2 min on ice.
6. Fill up the transformation mix to 1 mL with LB media or SOC media and incubate at 37°C at 250rpm for 45 min (45-60 min) for recovery of the cells.
7. Plate cells on LB agar plate with antibiotic respectively and dry them under the clean bench.
1. 200 µL
2. resuspended pellet (from centrifugation of the leftover)
8. Incubate the agar plates at 37°C overnight (16-18 hours).
9. Count single colonies of each agar plate on the next day.
Overnight culture of E.coli
work space: clean bench
conservation: 37°C (for E. coli) for 16-18 hours
1. pipette 5 mL of autoclaved LB-Medium into a sterile glass
2. add 5 µL corresponding antibiotics (1000x)
3. add one colony from the plate of your strain with a toothpick/ pipette tip and throw it in the glass
4. leave it shaking in the 37°C room (for E. coli) over night
Overnight culture of S.cerevisiae
work space: clean bench
conservation: 28°C (for S.cerevisiae) for 1-2 days
1. pipette 5 mL of autoclaved YEP-Medium into a sterile glass
2. (add 5 µL corresponding antibiotics (1000x))
3. add one colony from the plate of your strain with a toothpick/ pipette tip and throw it in the glass
4. leave it shaking in the 28°C room (for for S.cerevisiae) over 1 or 2 nights
Perform a cry-culture
work space: clean bench
conservation: -80°C freezer
1. pipette 500 µL of the glycerol stock (creation of 50% glycerol stock) in a cryo tube
2. add 500 µL of the over night culture overnight culture S.cerevisiae or overnight culture e.coli
3. store it at -80°C to conserve it
You can transfer the ratio to every other size of the solution but 1 mL is a good and normal size for a cryo culture.
Determination of DNA concentration with NanoVue
1.: Firstly, clean the device with water by pipetting a few drops on the surface and wipe it twice.
2.: Turn the device on and select the option LIFE SCIENCE → DNA.
3.: Adjust the pipette to 2 µl, place a drop of the used buffer (as reference) in the center of the cross and close the lid.
4.: Clean the surface and repeat step 3 in order to get a second reference.
5.: Wipe the surface and apply a drop of our sample, close the lid and write the result down
6.: Clean the device in the end properly
Resuspension of IDT Oligo primer
 Resuspend oligos in TE (Tris-EDTA) buffer (10mM Tris; 0.1 mM EDTA; pH 8.0) as this buffer will maintain a constant pH. Alternatively, use nuclease-free water (Not DEPC).
 Resuspend dry oligonucleotides to a storage stock concentration of 100 μM and then dilute a portion of this to create working stock solutions.
storage stock concentration
 suspend the primers in the IDT tubes
 To make a 100 μM concentration stock solution (as backup in the IDT tube): Take the number of nmoles in the tube (from the tube-label) and multiply that by 10. This will be the number of μL buffer to add to get a 100 μM solution.
 incubate for at least 15 minutes
 the tube can be stored at -20°C
PCR working stock
 take a new tube for this stock
 To get 100 µL 5 µM solution, mix 5 µL of 100 µM oligo stock with 95 µL of water or TE buffer
 use the 5 µM stock for the PCR Mastermix and store the rest at -20°C
PfuS PCR setup and cycling conditions
1. Setup on ice in a 0.2 ml tube the mastermix for PCR according to the table below. Thaw all non-enzyme components at RT, mix by short vortex and collect by short centrifugation. It is recommended to add PhuS DNA Polymerase last in order to prevent any primer degradation caused by the 3`→5`exonuclease activity. Polymerase may be diluted in 1 x Buffer just prior to use in order to reduce pipetting errors.
Mastermix: (50 µL reaction)
Plasmid template (sequencing or column grade plasmid preparation) - X µL (10-15 ng, max. 50 ng)
Deionized H20 (PCR quality) 35.5 µL - X µL of plasmid template
10 x PfuS buffer B2 - 5 µL (final 1x)
X_fwd primer [1] (5 µM) - 4 µL (final 400 nM)
x_rev primer (5 µM) - 4 µL (final 400 nM)
dNTP-mix (10mM each) - 05 1 µL (final 0.2 mM each nucleotide)
PfuS (2 U/µl [2] E2 - 0,5 µL (final 0.02 U/µL)
PCR programm: (25 cycles, tube control with 50 µL, 105 °C heated lid)
ID - Initial denaturation - 30 sec - 98°C
D - denaturation - 5-10 sec [3] 98°C - 24 cycles (D,A,E)
A - annealing - 15-30 sec - X °C (45-72°C) [4]
E - extension - 15-30 sec/Kb [5] - 72 °C
FE - final elongation [6] - 5-10 min - 72 °C
S - Storage - forever - 8°C
 optional 2.Load all samples on a 0,8% TEA agarose-gel (100V; 35 min). Check for optimal annealing temperature and if enough PCR product is present gel-extract the bands of column purifiy is with the PCR purification-kit. (Macherey-Nagel NucleoSpin Extract II; 50 µL Ne-buffer).
Notes
 [1] Primers: Oligonucleotides primers are generally 20-40 nucleotides in length and ideally have a GC content of 40-60%. The final concentration of each in a PCR reaction may be 0.2-1 µM. while 0.4 µM is recommended.
 [2] PfuS DNA polymerase concentration: recommended use of 1 U/50 µL reaction. However, the optimal concentration of PfuS DNA polymerase may vary from 0.5-2 U/50 µL reaction depending on amplicon length and difficulty. Do not exceed 2 U/50 µL reaction, especially for amplicons longer than 5 kb (https://www.neb.com).
 [3] Keep the denaturation as short as possible. Usually 5-10 seconds at 98°C is enough for most templates. Note: the denaturation time and temperature may vary depending on the ramp rate and temperature control mode of the cycler.
 [4] The PfuS DNA polymerase has the ability to stabilize primer-template hybridization. As a basic rule, for primers > 20 nt, anneal for 10-30 sec at a Tm + 3 °C of the lower primer. The Tm´s should be calculated with the nearest neighbor method (calculate tm for matching sequence with http://eu.idtdna/analyzer/applications/oligoanalyer/default.aspx) as results from primer Tm calculations can vary significantly depending on the method used. For primers <20 nt, use a annealing temperature equal to the Tm of the lower Tm primer.
 [5] Extension time depends on amplicon length and complexity. For low complexity DNA (e.g. plasmid, lambda or BAC DNA) use extension time of 15 sec per 1 kb. For high complexity genomic DNA 30 sec per 1 kb is recommended.
 [6] Final extension is an optimal step, usually does not influence the outcome and can by default be skipped.
OneTaq Quick-Load 2X Mastermix PCR conditions
reaction setup       We recommend assembling all reaction components on ice and quickly transferring the reactions to a thermocycler preheated to the denaturation temperature (94°C)
Component 25 µL reaction 50 µL reaction Final Concentration
10 µM Forward Primer 0,5 µL 1 µL 0,2 µM
10 µM Reverse Primer 0,5 µL 1 µL 0,2 µM
Template DNA variable variable <1,000 ng
One Taq Quick-Load 2X Master Mix with Standard Buffer 12,5 µL 25 µL 1X
Nuclease-free water to 25 µL to 50 µL
 Notes: Gently mix the reaction. Collect all liquid to the bottom of the tube by a quick spin if necessary.
Overlay the sample with mineral oil if using a PCR machine without a heated lid.
 Tranfer PCR tubes to a PCR machine and begin thermocycling:
Thermocycling conditions for a routine PCR:
STEP TEMP TIME
Initial Denaturation 94°C 30 seconds
30 Cycles: 94°C 15-30 seconds; 45-68°C 15-60 seconds; 68°C 1 minute per kb
Final Extension 68°C 5 minutes
Hold 4-10°C
general guidelines
1.Template:
 Use of high quality, purified DNA templates greatly enhances the success of PCR. Recommended amounts of DNA template of a 50 µL reaction are as follows:
DNA Amount
genomic 1 ng - 1 µg
plasmid or viral 1 pg - 1 ng
2.Primers:
 Oligonucleotide primers are generally 20-40 nucleotides in length and ideally have a GC content of 40-60%. Computer programs such as Primer3 can be used to design or analyze primers. The final concentration of each primer in a PCR may be 0.05-1 µM. typically 0.2 µM.
3.Mg++ and Additives: Mg++ concentration of 1.5-2.0 mM is optimal for most PCR products generated with OneTaq DNA Polymerase. The final Mg++ concentration in 1X OneTaq Quick-Load Master Mix with Standard Buffer is 1.8 mM. This supports satisfactory amplification of most amplicons. Howeoever, Mg++ can be further optimized in 0.2 mM increments using MgCl2.
 For amplification of difficult targets, like GC-rich sequences, we recommend OneTaq Quick Load 2X Master Mix with GC Buffer. Alternatively, DMSO or formamide may be used.
4.Denaturation:
 An initial denaturation of 30 seconds at 94°C is sufficient to amplify most targets from pure DNA templates. For difficult templates such as GC-rich sequences, a longer denaturation of 2-4 minutes at 94°C is recommended prior to PCR cycling to fully denature the template. Alternatively, use OneTaq Quick-Load 2x Master Mix with GC Buffer. With colony PCR, an initial 2-5 minute denaturation at 94°C is recommended to lyse cells.
 During thermocycling a 15-30 second denaturation at 94°C is recommended.
5.Annealing:
 The annealing step is typically 15-60 seconds. Annealing temperature is based on the Tm of the primer pair and is typically 45-68°C. Annealing temperatures can be optimized by doing a temperature gradient PCR starting 5°C below the calculated Tm. We recommend using NEB`s Tm Calculator to determine appropriate annealing temperature for PCR.
 Tm Calculator: http://tmcalculator.neb.com/#!/
6.Extension:
 The recommended extension temperature is 68°C. Extension times are generally 1 minute per kb. A final extension of 5 minutes at 68°C is recommended.
7. Cycle Number:
 Generally, 25-35 cycles yields sufficient product. Up to 45 ycles may be required to detect low copy number targets.
8. 2-step PCR:
 When primers with annealing temperatuers of 68°C or above are used, a 2-step thermocycling potocol (combining annealing and extension into one step) is possible.
9. PCR Product: The majority of the PCR products generated using OneTaq DNA Polymerase contain dA overhangs at the 3`end; therefore the PCR products ca be ligated to dT/dU-overhang vectors.
Production of Agarose Gel
For 1000ml of Agarose Gel:
 12g of NEEO Agarose
 20g of 50x TAE Buffer
 980g of ddH20
 Mix ingedients in a 1000ml Schott-Bottles
 Put the bottle into the autoclave and set the program Agarose
 After autoclavation put the bottles into the 60°C closet
 Write your Name and the used programm in the autoclave book
 Attention: Do not fill bottles with EtBr yet! Do that after opening a new bottle
Production of TSS-media
Preparation
To make 50 mL:
 5g PEG 8000       1.5 mL 1M MgCl2 (or 0.30g MgCl2*6H20)       2.5 mL DMSO       Add LB to 50 mL
=>Filter sterilize (0.22 μm filter)
Notes
 PEG 3350 can be used instead of PEG 8000. According to the original CT Chung paper, PEG 3350 produces better efficiency, and other sized PEGs can be used as well albeit with a slight loss in efficiency. pH from original Chung CT paper calls for pH to be acidic (pH=6.5)... be sure to pH solution before sterilization.       If using non-chemically resistant filters (e.g., cellulose nitrate), add DMSO after sterilization. DMSO should be sterile in and of itself, so it may be prudent it add it afterwards if you are unsure about the compatibility of your filters.
Storage
 Between +4 °C and -20 °C
Performing of competent cells
Materials
 Plate of cells to be made competent
 TSS-medium
 LB Medium
 Ice
Glassware & Equipment
 Falcon tubes
 500μl Eppendorf tubes, on ice
 200ml conical flask
 200μl pipetman or repeating pipettor
 5ml pipette
Preparation
1. Grow a 5ml overnight culture of cells in LB media. In the morning, dilute this culture back into 25-50ml of fresh LB media in a 200ml conical flask. You should aim to dilute the overnight culture by at least 1/100.
2. Grow the diluted culture to an OD600 of 0.2 - 0.5. (You will get a very small pellet if you grow 25ml to OD600 0.2)
3. Put eppendorf tubes on ice now so that they are cold when cells are aliquoted into them later. If your culture is X ml, you will need X tubes. At this point you should also make sure that your TSS is being chilled (it should be stored at 4oC but if you have just made it fresh then put it in an ice bath).
4. Split the culture into two 50ml falcon tubes and incubate on ice for 10 min.
All subsequent steps should be carried out at 4oC and the cells should be kept on ice wherever possible
1. Centrifuge for 10 minutes at 3000 rpm and 4oC.
2. Remove supernatant. The cell pellets should be sufficiently solid that you can just pour off the supernatant if you are careful. Pipette out any remaining media.
3. Resuspend in chilled TSS buffer. The volume of TSS to use is 10% of the culture volume that you spun down. You may need to vortex gently to fully resuspend the culture, keep an eye out for small cell aggregates even after the pellet is completely off the wall.
4. Add 100 μl aliquots to your chilled eppendorfs and store at − 80oC.
5. Place the cells at -80°C
Note:
 If you run a control every time you clone (i.e. a vector-only ligation), you can as well freeze cells in 200 μl aliquots.
 It is a good idea to run a positive control on the cells.
Transformation of competent E.coli cells (performed with Tss-protocol)
Method
1. Thaw TSS cells on ice.
2. Add DNA, pipette gently to mix (1μl of prepped plasmid is more than enough).
3. Let sit for 30 minutes on ice.
4. Incubate cells for 30-45 seconds at 42oC.
5. Incubate cells on ice for 2 min.
6. Add 1 mL SOC (2XYT and LB are also suitable) at room temp.
7. Incubate for 45 minutes - 1 hour at 37oC on shaker.
8. Spread 100-300 μl onto a plate made with appropriate antibiotic.
9. Grow overnight at 37 °C.
 Save the rest of the transformants in liquid culture at 4 °C. If nothing appears on your plate, you can spin this down, resuspend in enough medium to spread on one plate and plate it all. This way you will find even small numbers of transformants.
Notes
 With SOC-medium you have a 3-folded better transformation efficiency of E. coli cells
 If you are adding small volumes of plasmid or transormation DNA (~1μl), be careful to mix the culture well. Diluting the plasmid back into a larger volume can also help
 If you are in a rush, you can shorten incubation time on ice to 5-10 min.(step 3)
 According to the original TSS paper, heat shock step is completely optional and may actually reduce transformation efficiency.With DH5a Z1 and pUC19 a heat shock at 42°C for 30 sec improved transformation efficiency 10-fold
Preparation of Biobricks
 the instruction for preparing the biobrick: http://parts.igem.org/Help:Protocols/Linearized_Plasmid_Backbones
• 1 Restriction Digest of the gene you want to integrate and pSB1C3
• 2 PCR clean up
• 3 NanoVue
• 4 Dephosphorylation
• 5 Ligation
• 6 Transformation in DH5α
Restriction Digest of the gene you want to integrate and pSB1C3
 We carried out two Double Digests:
 Gen with EcoRI and PstI (we integrated the sequences for this enzymes around our gen before ordering it)
 pSB1C3 linearized plasmid backbone with EcoRI and PstI (this linearized fragment is in the iGEM-Kit)
 digest mix
component backbone pSB1C3 insert
buffer cutsmart (10x) "NBCS" - 1µl - 1µl
EcoRI-HF "N8" - 0,5µl - 0,5µl
PstI-HF "N22" - 0,5µl - 0,5µl
template DNA final c= 10ng/µl - 4µl - ? µl
H20 - 4µl - 6µl
 * amount of DNA:
a) c(0) of backbone: 25ng/µl → to get a final concentration of 10ng/µl we have to use 4 µl backbone-DNA. (because: 25ng/µl * 4µl = 100ng ; 100ng / 10µl = 10ng/µl)
b) c(0) of insert: ___ ng/µl → to get a final concentration of 10ng/µl we have to use ___ µl insert-DNA. (because: __ng/µl * __µl = 100ng ; 100ng / 10µl = 10ng/µl)
 Digest 37°C/30 min, heat kill 80°C/20 min
PCR clean up
 performe a PCR-clean up to remove the enzymes. Use less elution-buffer, because your concentrations will be very low! (Remember: you just use 100ng DNA → the maximal concentration you could get when using 10µl elution buffer is 10ng/µl)
NanoVue
 Performe a NanoVue to determine the concentration of the digested DNA.
Dephosphorylation
It is an optional step which is used when only one restriction enzymes is used. See link: http://igem.rwth-aachen.de/2016/index.php/Protocols/dephosphorylation.
Ligation
 (protocol from iGEM.org)
 ligation mix
digested backbone - 25ng - ___ µl
digested insert - * - µl
T4 DNA ligase buffer - - 1µl
T4 DNA ligase - - 1µl
H20 - - up to 10 µl *
 Ligate 16°C/30 min, heat kill 80°C/20 min
 Transform with 1-2 µl of product
 Note: For linearized plasmid backbones provided by iGEM HQ, a plasmid backbone with an insert of BBa_J04450 was used as template. As a result any red colonies that appear during your ligation may be due to the template as a background. Digesting with Dpn1 before use should reduce this occurrence.
 *calculation of the amount of insert in ng: insert(ng) = plasmid (ng) x [lenght of insert (bp) / lenght of backbone (bp)] x ratio of insert/backbone
 insert ___ (ng) = _ng x [_ bp / _ bp] x 10/1 = ___ ng
 ratio: insert:backbone is 10:1
 concentration of backbone: _ ng/µl ; we want to add 25ng so we have to use 25ng/ __ ng/µl = __ µl
 concentration of insert: _ ng/µl ; we want to add about _ ng so we have to use __ ng/ __ ng/µl = __ µl
Transformation in DH5α
 work space: clean bench
1. Thaw 100 µL E. coli DH5α on ice
2. Add DNA (2µL (~20ng) ligation mixture) to the competent cells and swirl gently
3. Incubate for 15-30 min on ice.
4. Perform heatshock of the competent cells (using a heat-block) at 42°C for 45s (DH5α).
5. Samples were immediately cooled down 2 min on ice.
6. Fill up the transformation mix to 1 mL with LB media or SOC media and incubate at 37°C at 250rpm for 45 min (45-60 min) for recovery of the cells.
7. Plate cells on LB agar plate with antibiotic respectively and dry them under the clean bench.
a. 200 µL
b. resuspended pellet (from centrifugation of the leftover)
8. Incubate the agar plates at 37°C overnight (16-18 hours).
9. Count single colonies of each agar plate on the next day.
Single tube transformation for BioBricks
Before You Start
Estimated bench time: 1 hour Estimated total time: 2 hours (plus 14-18 hour incubation)
When transforming competent cells, both timing and temperature are very important. Use a lab timer, follow the incubation temperatures closely, and keep materials on ice when required. Read through the entire protocol before starting!
Materials
 Resuspended DNA to be transformed
 10pg/µl Positive transformation control DNA (e.g. pSB1C3 w/ BBa_J04450, RFP on high-copy chloramphenicol resistant plasmid. Located in the *Competent Cell Test Kit.)
 competent cells (50µl per sample)
 1.5mL Microtubes
 SOC Medium (950µL per sample)
 LB Agar + Chloramphenicol(2 per sample.)
Biobricks contain CHL-resistance
Equipment
 Floating Foam Tube Rack
 Ice & ice bucket
 Lab Timer
 42°C water bath
 37°C incubator
 Sterile spreader or glass beads (/ also "dreigalsky-spatel" works)
 Pipettes and Tips (10µl, 20µl, 200µl recommended)
 Microcentrifuge
Method
1. Resuspend DNA in selected wells in the Distribution Kit with 10µl dH20. Pipet up and down several times, let sit for a few minutes. Resuspension will be red from cresol red dye.
2. Label 1.5ml tubes with part name or well location. Fill lab ice bucket with ice, and pre-chill 1.5ml tubes (one tube for each transformation, including your control) in a floating foam tube rack.
3. Thaw competent cells on ice: This may take 10-15min for a 260µl stock. Dispose of unused competent cells. Do not refreeze unused thawed cells, as it will drastically reduce transformation efficiency.
4. Pipette 50µl of competent cells into 1.5ml tube: 50µl in a 1.5ml tube per transformation. Tubes should be labeled, pre-chilled, and in a floating tube rack for support. Keep all tubes on ice. Don’t forget a 1.5ml tube for your control.
5. Pipette 1µl of resuspended DNA into 1.5ml tube: Pipette from well into appropriately labeled tube. Gently pipette up and down a few times. Keep all tubes on ice.
6. Pipette 1µl of control DNA into 2ml tube: Pipette 1µl of 10pg/µl control into your control transformation. Gently pipette up and down a few times. Keep all tubes on ice.
7. Close 1.5ml tubes, incubate on ice for 30min: Tubes may be gently agitated/flicked to mix solution, but return to ice immediately.
8. Heat shock tubes at 42°C for 45 sec: 1.5ml tubes should be in a floating foam tube rack. Place in water bath to ensure the bottoms of the tubes are submerged. Timing is critical.
9. Incubate on ice for 5min: Return transformation tubes to ice bucket.
10. Pipette 950µl SOC media to each transformation: SOC should be stored at 4°C, but can be warmed to room temperature before use. Check for contamination.
11. Incubate at 37°C for 1 hours, shaking at 200-300rpm
12. Pipette 100µL of each transformation onto petri plates Spread with sterilized spreader or glass beads immediately(/"Dreigalsky-Spatel"). This helps ensure that you will be able to pick out a single colony.
13. Spin down cells at 6800g for 3mins and discard 800µL of the supernatant. Resuspend the cells in the remaining 100µL, and pipette each transformation onto petri plates Spread with sterilized spreader or glass beads immediately. This increases the chance of getting colonies from lower concentration DNA samples.
14. Incubate transformations overnight (14-18hr) at 37°C: Incubate the plates upside down (agar side up). If incubated for too long, colonies may overgrow and the antibiotics may start to break down; un-transformed cells will begin to grow.
15. Pick single colonies: Pick single colonies from transformations: do a colony PCR to verify part size, make glycerol stocks, grow up cell cultures and miniprep.
16. Count colonies for control transformation: Count colonies on the 100μl control plate and calculate your competent cell efficiency. Competent cells should have an efficiency of 1.5x10^8 to 6x10^8 cfu/µg DNA.
Creation of carrier DNA for transformation
Material
salmon sperm: 4mg ddH2O: 2 ml
Method
1. you need to resuspend 4mg of salmon sperm in 400ul of ddH2O      2. deanturate DNA at 95°C for 5 minutes      3. store carrier DNA solution at -20°C
Notes  salmon sperm should be autoclaved before use, however denaturation at 95°C is enough.       salmon could be not easy to resuspend.       salmon sperm to weight is very difficult. It isn´t a problem if you cannot precisly weight the quantity you need.
Cre-loxP treatment of yeast cells
needed material
 YPD+hygromycin (conc. 250mg/L) plates       YPD plates       auxotrophie-media-plates of the auxotrophie you want to remove       liquid induce-media, with the GAL1-Promotor 100%Galactose YPD-Media       plasmid containing cre- protein and an inducible promoter upstream (e.g. pSH69 of E.coli Stammnummer 2757 (iAMB))
Procedure
1. Transformation of the cells that have to be treated with the plasmid that contains the gene for the cre-protein. For example the plasmid pSH69 from E.coli Stammnummer 2757 (iAMB). The protocol will be described in usage of this plasmid, which contains a GAL1 Promoter upstream of the cre-protein-DNA and a hygromycin Marker Gene.      2. The Transformants get plated on YPD + hygromycin (conc. 250mg/L) plates.      3. When the colonies on the plate grew, use the method "Abschwämmen" to dilute the Colonies on Medium. That means you pour some induce- media onto the plate, so all colonies are covered with media. Than you remove all the colonies from the plate into the media with a drigalski-spatel or something else.      4.Take the media from the plates with all the cells from the colonies in it and pour it into a induce- media, in this case Galactose+YPD -media (because of the GAL1 Promoter).      5.Leave the cells in the induce- media for 4-5h at 28-30°C. Now the cre-protein gets expressed and it deletes all the DNA sequences which are between LOXp- sites.      6.Plate the cells on a YPD- plate but before prepare a dilution-row (Verdünnungsreihe) of the cells in the induce media to get single colonies on the plate.      7.After grwoth of colonies on the YPD-plate, check 5 colonies or more for success of the cre treatment: You mark 5 colonies or more on the YPD- plate and take half of each to spread them on auxotrophie-media-plates of exactly the auxotrophie you want to remove from your cells. Only if the cells can NOT grow on the auxotrophie-media-plate, the cre-treatment was successful!
Creation of competent yeast cells
Materials
 Frozen Competent Cell (FCC) Solution → Solution is 5% (v/v) glycerol and 10% (v/v) DMSO. Be sure to filter-sterilize.
 Liquid YPD media → about 1L
 50 mL Falcon
 styrofoam-box for freezer
 A 250 mL sterile culture flask for each strain
 A 2 L sterile flask for each strain
Cell Preparation
 Day 1
 Inoculate 25ml of YPD with your strain (for an overnight cuture) and grow at 30°C and 200 RPM overnight or for 12-16 hours.
 Day 2
 Pre-warm 500 mL YPD in the 2 L flask at 30°C for later use.
 Take the overnight culture and find the OD600 to determine cell count (OD 0.1 ≈ 1 x 10^6 cells/mL.) An OD close to 1 is desired. (dilute the culture with YPD-medium until you get an OD close to 1)
 With this culture, inoculate the pre-warmed flask of YPD with the entire 25mL and shake at 30°C for about 4 hours.
 NOTE that adding cells according to their titers as described in Geitz et. al will not be possible since the cell densities achieved were not *even close to what was described (try using 2X YPD if this is the route you need to go in.)
 centrifuge at 3,000xg for 5 minutes using 50 mL conical tubes. I split 250 mL of the culture among 5 tubes, spin them down, discard the supernatant, and add the remaining culture to the same tubes and repeat.
 Wash the cells in 0.5 volumes (25 mL in each of these tubes) of sterile water. Resuspend in 0.01 volumes (500μL each) sterile water and transfer to microfuge tubes. Pellet cells at 3,000xg for 5 minutes.
 Resuspend in 500μL FCC solution and aliquot 50μL each into microfuge tubes.
 Place these tubes into the styrofoam racks and store at -80°C. Since the racks are to promote slow freezing, after the tubes are brought to -80°C you may transfer them to whatever your container of choice is in the -80.
Using These Cells for Transformation
 Take out a tube for each set of transformations and warm between your hands for 15 to 30 seconds. Pellet the cells at 3,000xg for 2 minutes and discard the supernatant. Take these cells and add them to your transformation master mix and follow the procedure as described in McClean:Yeast Transformation day 2 step 7. You may need to use more than one tube of cells if your yield was low.
Transformation of yeast cells
Materials
 1 M lithium acetate (LiAc) → autoclave for 15 min       0.1 M LiAc       50% PEG 3350 in water → autoclave for 15 min       single stranded carrier DNA → fish sperm DNA, prepare 2 mg/mL solution in dH2O and denature for 5 min at 95°C
transformation:
 preparation of DNA-solution: Combine 0.5-1 µg each of desired fragments (ratio of 1:2 for fragments containing marker elements to fragments without marker element), bring to a total volume of 34 µL with dH20 (for plasmids, use 100-300 ng)       prepare transformation-mixture (can be scaled up and prepared as master mix):       240 µL 50% PEG 3350       36 µL 1M LiAc       50 µL carrier DNA       centrifuge aliquot of competent cells (16,000xg, 15 s) → e.g. cryos of competent cells       resuspend cells in 34 µL (plasmid-)DNA- solution       add transformation-mixture and vortex gently       incubate at 30°C for 30 min       invert tube, then incubate for precisely 30 min at 42°C       centrifuge cells (16,000xg, 15 s)       A) if antibiotic marker is used: resuspend cells in 500 µL YPD and incubate at 30°C and 200 rpm for 3-4 h, afterwards spread on selection medium       B) if there is no antibiotic marker: resuspend pellet in 100 µL A. bidest, spread on selection medium
Yeast genome purification
 pick a large colony and resuspend in 100 µL 0.2 M LiOAC containing 1% SDS
 Solution: mix 1,3196mg LiOAC, 0,67mg SDS and 100 µl ddH2O (calculation from Niels)
 incubate 10 min at 75°C
 add 300 µL 100% EtOH, vortex
 centrifuge 3 min at max. rpm, discard supernatant
 resuspend pellet in 150 µL 70 % EtOH (→ for 70% EtOH mix 105µl of 100%-EtOH with 45µl H2O)
 centrifuge 3 min at max. rpm, discard supernatant
 dry pellet at 37 °C (place tube upside down)
 add 50 µL ddH2O to the dry pellet and vortex shortly
 centrifuge 1 min at max. rpm, use 2 µL supernatant for PCR
Creation of competent yeast cells and transformation following protocol from "zymoresearch"
Preparation of Competent Cells
1. Grow yeast cells at 30°C in 10 ml YPD broth until mid-log phase (~5 x 10^6- 2 x 10^7cells/ml or OD600 of 0.8-1.0).
Note
If you don´t know when your mid-log phase is, you can use a fully grown S. cerevisiae overnight culture, dilute it 1/10 and then let it grown for another 4-6 h at 30°C.
The following steps are accomplished at room temperature:
1. Pellet the cells at 500 x g for 4 minutes and discard the supernatant.
2. Add 10 ml EZ 1 solution to wash the pellet. Repellet the cells and discard the supernatant.
3. Add 1 ml EZ 2 solution to resuspend the pellet.
At this point, the competent cells can be used for transformations directly or stored frozen at or below -70°C for future use.
Note:
It is important to freeze the cells slowly. To accomplish this, either wrap the aliquotted cells in 2-6 layers of paper towels or place in a Styrofoam box before placing in the freezer.
DO NOT use liquid nitrogen to snap-freeze the cells.
Transformation
This part of the procedure is the same for both frozen stored (thawed at room temperature) and freshly prepared competent yeast cells.
1. Mix 50 µl of competent cells with 0.2-1 µg DNA (in less than 5 µl volume); add 500 µl EZ 3 solution and mix thoroughly.
2. Incubate at 30°C for 45 minutes. Mix vigorously by flicking with finger or vortexing (if appropriate for your DNA) 2-3 times during this incubation.
3. Spread 50-150 µl of the above transformation mixture on an appropriate plate. It is unnecessary to pellet and wash the cells before spreading.
4. Incubate the plates at 30°C for 2-4 days to allow for growth of transformants.
Attention:
For transformations of C. albicans, use freshly prepared competent cells; frozen cells sometimes give poor results.
Creation of LB-media
for 1000mL
 1.mix:
10g Peptone/Trypton
10g NaCl
5g Yeast extract
for plates
20g Agar (-> leave the stirring bar in the flask)
up to 1000mL ddH2O
 2. autoclave with bottle not tightly closed
 3. add 1000uL antibiotics (1000x) when bottle is slightly cooled down
for plates
 4. make plates under cleanbench when -70°C is reached
Creation of LB-media with chloramphenicol
 1.: mix: 8g LB-Medium + 8g Agar + 400ml water
 2.: autoclave
 3.: at 50°C: mix with 0,4ml Chloramphenicol (to get 1:1000(Antibiotica:Medium)) with the concentration 25mg/ml (mix 10mg Chloramph. with 0,4ml water)
Creation of SOC-media
 1.: 10g Trypton + 1g NaCl + 2,5 yeast extract + 1,25 KCl
 2.: fill up with 480ml water
 3.: autoclave
 4.:add
 5ml 1M MgCl2 x 6H2O (1,0165g + 5ml ddH2O)
 5ml 1M MgSO4 x 7H2O (1,2321g + 5ml ddH2O)
 10ml 1M glucose monohydrate (1,9817g + 10ml ddH2O)
Creation of VYP-media
for 1000mL
 1.mix
10g Yeast extract
20g Pepton/ Trypton
100mL 20% D-Glucose (-> autoclave seperatly or filter-sterelize)
for plates
20g Agar (-> leave the stirring bar in the flask)
up to 1000mL ddH2O
 2. autoclave it with bottle not tightly closed
 3. add 1000uL antibiotcs when bottle is slightly cooled down
for plates
 4. make plates under clean bench when -70°C is reached
Creation of SC minimal media
SC is synthetic minimal defined medium for yeast
1000 ml
Mix
 0,67% yeast nitrogen base (without amino acids)
 2% cabon source (i.e glucose or raffinose)
 0,01% (adenine, arginine, cysteine, leucine, lysine, threonine, tryptophan, uracil)
 0,005% (aspartic acid, histidine, isoleucine, methionine, phenylalanine, proline, serine, tyrosine, valine)
 1,5% agar(for plates)
1. Dissolve the following reagents in 900 ml deionized water (800 ml if preparing medium containing raffinose).
Note: We make medium and plates as we need them and weigh out each amino acid. Many researchers prepare 100X solutions of each amino acid that they need.
2. If you are making plates, add the agar after dissolving the reagents above.
3. Autoclave at 15 psi, 121°C for 20 minutes.
4. Cool to 50°C and add 100 ml of filter-sterilized 20% glucose or 200 ml of filter-sterilized 10% raffinose.
5. Pour plates and allow to harden. Invert the plates and store at 4°C. Plates are stable for 6 months.
Induction Medium: If you are making induction medium, follow Steps 1–3 above except dissolve the reagents in 800 ml of deionized water. Cool the medium to 50°C and add 100 ml of filter-sterilized 20% galactose and 100 ml of filter-sterilized 10% raffinose to the medium.
Creation of URA minimal media
[URA-]-medium with synthetic drop-out complement
Composition:
 960ml ddH2O
 6.7g nitrogen base without AA
 40ml 50% glucose solution
 1.92g synthetic drop-out complement ([URA-])
Note:
Glucose should only be sterile filtered added to the mixture. It should be done after autoclaving the mixture. Glucose should not be autoclaved because of the possible caramelisation of it due to the high temperature. Storage at room temperature.
[URA-]-plates with synthetic drop-out complement
Composition:
 960ml ddH2O
 6.7g nitrogen base without AA
 40ml 50% glucose solution
 1.92g synthetic drop-out complement ([URA-])
 20g Agarose
Note:
Glucose should only be sterile filtered added to the mixture. It should be done after autoclaving the mixture. Glucose should not be autoclaved because of the possible caramelisation of it due to the high temperature. storage at 4°C.
[HIS-]-medium with synthetic drop-out complement
Composition:
 960ml ddH2O
 6.7g nitrogen base without AA
 40ml 50% glucose solution
 1.92g synthetic drop-out complement ([URA-])
alternative to 1.92g synthetic drop-out complement ([HIS-]:
50ul of 1% uracyl, thryptophan and leucin + synthetic drop-out complement ([HIS-],[URA-],[TRP-] and [LEU-])
Note:
Glucose should only be sterile filtered added to the mixture. It should be done after autoclaving the mixture. Glucose should not be autoclaved because of the possible caramelisation of it due to the high temperature. Storage at room temperature.
[URA-]-plates with synthetic drop-out complement
Composition:
 960ml ddH2O
 6.7g nitrogen base without AA
 40ml 50% glucose solution
 1.92g synthetic drop-out complement ([URA-])
alternative to 1.92g synthetic drop-out complement ([HIS-]:       50ul of 1% uracyl, thryptophan and leucin + synthetic drop-out complement ([HIS-],[URA-],[TRP-] and [LEU-])
 20g Agarose
Note:      Glucose should only be sterile filtered added to the mixture. It should be done after autoclaving the mixture. Glucose should not be autoclaved because of the possible caramelisation of it due to the high temperature. storage at 4°C.
all mentioned synthetic drop-out complements are avaible on the internet. We used syntheticdrop-out complements of sigma-aldrich, wich are in Schwaneberg        laboratory avaible
Creation of YPD-galactose-media
Creation of YPD-galactose-media
YPD-galactose-medium:(1000ml)
 900ml ddH2O       20g tryptone/peptone       10g Yeast extract       100ml 20% galactose solution
Note:
All components of the medium were autoclaved, with exception of galactose which was sterile filter added to the medium after autoclaving it. This because we wanted to  avoid the formation of caramel in our medium.      For plates:
 900ml ddH2O       20g tryptone/peptone       10g Yeast extract       100ml 20% galactose solution       20g of Agarose
Note:
All components of the medium were autoclaved, with exception of galactose which was sterile filter added to the medium after autoclaving it. This because we wanted to  avoid the formation of caramel in our medium.      Galactose must be added to the medium before it cooled down.
Creation of 50% glycerol stock
work space: scale room & autoclave room
conservation: room temperature
100 mL solution
50 mL Glycerin (liquid in the scale room)
50 mL ddH2O (from the autoclave room)
You can transfer the ratio to every other size of the solution
Creation of Tris-HCL buffer
 to optain 1L Tris - HCl with a pH of 8,0
1. 121,1 g Tris are soluted in 800mL ddH20
2. HCl is titrated until a pH of 8,0 is obtained
3. Solution is than filled up to 1L with ddH20
 storage at room temperature
Creation of TE buffer
 TE buffer = Tris - EDTA buffer
TE buffer: 10mM Tris; 0.1 mM EDTA; pH 8.0
 10 mL Tris-HCl 1M pH = 8,0 mixed with 0,2 mL 0,5 M EDTA       this 10,2 mL are soluted in 989,8 mL ddH20
