<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_MIT_gateway"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:MIT_gateway">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:MIT_gateway&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT_gateway&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:MIT_gateway&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:MIT_gateway" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:MIT gateway</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2010.igem.org</H3><DIV id="header"><DIV id="whitespace" style="background-color: transparent; height: 60px; font-size: large; color: white;">      <A href="http://mit.edu"><B style="vertical-align:-15px;">iGEM 2010</B></A></DIV><DIV id="navwrapper"><UL id="topnav"><LI class="about"><TABLE style="padding: 5px; background-color: transparent; color:white;"><TBODY><TR><TD>Home</TD></TR></TBODY></TABLE></LI><LI class="last"><A href="https://2010.igem.org/Team:MIT">Home</A></LI></UL><LI class="people"><TABLE style="padding: 5px; background-color: transparent; color:white;"><TBODY><TR><TD>Team</TD></TR></TBODY></TABLE></LI><LI><A href="https://2010.igem.org/Team:MIT_undergraduates">Undergraduates</A></LI><LI><A href="https://2010.igem.org/Team:MIT_instructors">Instructors</A></LI><LI><A href="https://2010.igem.org/Team:MIT_advisors">Advisors</A></LI><LI><A href="https://2010.igem.org/Team:MIT_fun">Fun</A></LI><LI><A href="https://2010.igem.org/Team:MIT_gallery">Gallery</A></LI><LI class="last"><A href="https://2010.igem.org/Team:MIT_sponsors">Sponsors</A></LI><LI class="academics"><TABLE style="padding: 5px; background-color: transparent; color:white;"><TBODY><TR><TD>Project</TD></TR></TBODY></TABLE></LI><LI><A href="https://2010.igem.org/Team:MIT_overview">Overview</A></LI><LI><A href="https://2010.igem.org/Team:MIT_toggle">Bacteria</A></LI><LI><A href="https://2010.igem.org/Team:MIT_phage">Phage</A></LI><LI><A href="https://2010.igem.org/Team:MIT_mammalian">Mammalian</A></LI><LI><A href="https://2010.igem.org/Team:MIT_results">Results</A></LI><LI><A href="https://2010.igem.org/Team:MIT_summary">Summary</A></LI><LI class="last"><A href="https://2010.igem.org/Team:MIT_acknowledgements">Acknowledgments</A></LI><LI class="notebook"><TABLE style="padding: 5px; background-color: transparent; color:white;"><TBODY><TR><TD>Notebook</TD></TR></TBODY></TABLE></LI><LI><A href="https://2010.igem.org/Team:MIT_mm">Materials &amp; Methods</A></LI><LI><A href="https://2010.igem.org/Team:MIT_parts">Parts</A></LI><LI class="last"><A href="https://2010.igem.org/Team:MIT_safety">Safety</A></LI><LI class="contributions"><TABLE style="padding: 5px; background-color: transparent; color:white;"><TBODY><TR><TD>Beyond Biobrick</TD></TR></TBODY></TABLE></LI><LI><A href="https://2010.igem.org/Team:MIT_mammalian_Standard">New Mammalian Standard</A></LI><LI class="last"><A href="https://2010.igem.org/Team:MIT_bioethics">Bioethics</A></LI></DIV></DIV><DIV class="header"><DIV style="width:250px; margin: 10px; position: relative; top: -4px; left:-11px; display: block; float:right; padding: 7px; background-color: white;"><DL id="nav"><DT><B>Bacterial Protocol</B></DT><DD><UL><LI><A href="https://2010.igem.org/Team:MIT_bconst">Biobrick Construction</A></LI><LI><A href="https://2010.igem.org/Team:MIT_bexp">Bacterial Experiments</A></LI></UL></DD><DT><B>Mammalian Protocol</B></DT><DD><UL><LI><A href="https://2010.igem.org/Team:MIT_mmethods">Microfluidics</A></LI><LI><A href="https://2010.igem.org/Team:MIT_mge">Genetic Engineering</A></LI><LI><A href="https://2010.igem.org/Team:MIT_gateway">Gateway Cloning</A></LI></UL></DD><DT><B>Phage Protocol</B></DT><DD><UL><LI><A href="https://2010.igem.org/Team:MIT_phageprot">Basic Protocol</A></LI></UL></DD></DL></DIV><DIV id="unique" style="padding:0px; font-size: 14px; border: 1px solid black; margin:0px; background-color:transparent;"><TABLE width="650px" style="background-color: white; margin-top:5px; padding: 10px;"><TBODY><TR><TD><DIV class="bodybaby">Gateway Cloning</DIV></TD></TR><TR><TD>The Mammalian team used Gateway cloning to assemble its composite parts.<DIV class="outline"><A href="#gateway">1 Gateway Cloning</A><A href="#commented">2 Commented Protocol</A>    <A href="#design">2.1 Design PCR-Primers</A>    <A href="#gel">2.2 Gel Purify PCR Product</A>    <A href="#confirm">2.3 Confirm PCR Product</A>    <A href="#measure">2.4 Measure DNA</A>    <A href="#calculate">2.5 Calculate PCR Product Needed</A>    <A href="#volume">2.6 Calculate volume of DONRTM  needed</A>    <A href="#prepare">2.7 Prepare Gateway Reaction</A>    <A href="#retrieve">2.8 Retrieve 5x BP Clonase II</A>    <A href="#add">2.9 Add BP-Clonase to Gateway Reaction</A>    <A href="#incubate">2.10 Incubate</A>    <A href="#proteinasek">2.11 Add ProteinaseK</A>    <A href="#transform">2.12 Transform Bacteria</A>    <A href="#plate">2.13 Plate Bacteria</A><A href="#known">3 Known Issues</A><A href="#cite">4 Citation</A></DIV></TD></TR><TR><TD><DIV class="bodybaby" id="gateway">Gateway Cloning</DIV><DIV class="bodybaby" id="commented">Commented Protocol</DIV><B class="bolded" id="miniprep">Design PCR-Primers</B>
The Gateway®  clones have a reading frame which should be kept. Design primers that the PCR product starts with a ATG and ends with a STOP-codon or the last aminoacid (if you want to make a fusion protein). Primer3plus  is a powerful tool helping you to pick primers with the right annealing temperature which should be 60°C. Try to avoid self similarity and other things as usual, but because you are very limited in the position of the primers (its start and stop), I only care about annealing temperature and give it a try. Then just add to the primer which binds the start codon the attB1.1-sequence at his 5' End . To the primer which binds the stop codon or the last aminoacid add the attB2.1-sequence at his 5' End . The open reading frame is indicated and you should change the last two NN to code for an aminoacid of your choice. Good luck for the PCR! Because of the long 5' overhang and the restrictions on picking the primers, getting the PCR to work can be tricky.<U>Improved and more efficient att sites used to recombine into pDONR 221:</U>
attB1.1  GGG-GCA-ACT-TTg-tac-aaa-aaa-gtt-gNN
attB2.1  GG-GGC-AAC-TTT-GTA-CAA-Caa-agt-tgN<U>The original att sites used to recombine into pDONR 221:</U>
attB1  GGGG-ACA-AGT-TTg-tac-aaa-aaa-gca-ggc-tNN
attB2  GGG-GAC-CAC-TTT-GTA-CAA-Gaa-agc-tgg-gtN<U>The att sites used to recombine into pDONR P4-P1R:</U>
attB4  GGGG-ACA-ACT-TTg-tat-aga-aaa-gtt-gNN
attB1  GGG-GAC-TGC-TTT-TTT-GTA-Caa-act-tgN<U>The att sites used to recombine into pDONR P2R-P3:</U>
attB2  GGGG-ACA-GCT-TTc-ttg-tac-aaa-gtg-gNN
attB3  GGG-GAC-AAC-TTT-GTA-TAA-Taa-agt-tgN <B class="bolded" id="gel">Gel Purify PCR Product</B>
Purification of the PCR-product is needed to get rid of smaller side-products, which remove primer-dimers which can result in false positive colonies. Remember that you want to clone DNA, so the cutting should be made on the weakest UV-light available and as fast as possible. And of course you NEVER make a picture of the gel before. Use the kid for gel-purification available in your lab. <B class="bolded" id="confirm">Confirm PCR Product</B>
You need a PCR product with the  attB1.1 and attB2.2 and the DONRTM vector MUST have attP1 and attP2 sites, or it will not work.

The amount of plasmids is not soo important as in a multiple Gateway® reaction, because it is more efficient. If you want to optimize you can calculate equimolar amounts of both plasmids as described in the How to measure DNA. Here we use double the amount of DEST-vector, because most of the ones we use are round and about double the size of the ENTRTM clones. <B class="bolded" id="measure">Measure DNA</B>
The amount of ENTRTM is not so important as in a multiple Gateway® reaction, because it is more efficient. If you want to optimize you can calculate equimolar amounts of both plasmids as described in the multiple Gateway® protocol. Here we use double the amount of DEST-vector, because most of the ones we use are round and about double the size of the ENTRTM clones. <B class="bolded" id="calculate">Calculate PCR Product Needed</B>
ng needed = (length of the PCR product in bp) x 0.0165  <B class="bolded" id="volume">Calculate volume of DONTR(TM) needed</B>
µl needed = 75 ng needed / (concentration in ng/µl)

The DONR-vector should be tested for low background colonies (due to a mutated ccdB-gene) when transferred in DH5alpha-bacteria. <B class="bolded" id="prepare">Prepare Gateway Reaction</B>
PCR-product             ( ? ng)
pDONRTM-vector     (75 ng)
add water to a total volume of<B> 4µl </B><B class="bolded" id="retrieve">Retrieve 5x BP-Clonase II</B>
...From -20°C. It is most efficiently mixed by pipetting up and down, do not vortex. <B class="bolded" id="add">Add BP-Clonase to Gateway Reaction</B>
The enzymes looses 50% activity after 15 freeze-thaw cycles. The advantage of BP-ClonaseTMII is that it can be stored at -20 °C because it contains already the buffer. DO NOT LEAVE OUT.<B class="bolded" id="incubate">Incubate</B>
Incubation over-night will enhance the reaction ca. 5-10 fold. This is especially important for PCR products over 5.000 bp. <B class="bolded" id="proteinasek">Add ProteinaseK</B>
This step will enhance the reaction ca. 100 fold!!!!. This is different to the LR-reactions which are only enhanced 2 fold by adding the proteinase K!!!! 

<B class="bolded" id="transform">Transform Bacteria</B>
For electro competent cells use 1-2 µl, for chemical competent all. <B class="bolded" id="plate">Plate Bacteria</B>
The resulting ENTRTM-vectors are kanamycin resistant. <DIV class="bodybaby" id="known">Known Issues</DIV><UL><LI>  The reaction is very efficient. You can obtain about 200 colonies of which about 95 % are correct.</LI><LI> BP reactions work better with linear templates like PCR-products. If you want to use plasmids, linearize them first with a suitable restriction enzyme.</LI><LI> The obtained plasmids are big. To check for correct clones digest with Sty I and in parallel with Eco RI and Hind III. Compare the pattern of bands with the predicted band size to find the correct clones.</LI></UL><DIV class="bodybaby" id="cite">Citation</DIV>
1. Untergasser A. “Cloning – Gateway BP-Reaction II” Untergasser's Lab. Summer 2006. (include here the date when you accessed these page).
<HTTP:>. 

</HTTP:></TD></TR></TBODY></TABLE></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2010.igem.org/Team:MIT_gateway">http://2010.igem.org/Team:MIT_gateway</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:MIT_gateway" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:MIT_gateway" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:MIT_gateway&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:MIT_gateway&amp;oldid=188466" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2010.igem.org:Privacy_policy" title="2010.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2010.igem.org:General_disclaimer" title="2010.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>