"
Page
Discussion
View source
History
teams
Log in
 
Team:MIT gateway
From 2010.igem.org
      
iGEM 2010
Home
Home
Team
Undergraduates
Instructors
Advisors
Fun
Gallery
Sponsors
Project
Overview
Bacteria
Phage
Mammalian
Results
Summary
Acknowledgments
Notebook
Materials & Methods
Parts
Safety
Beyond Biobrick
New Mammalian Standard
Bioethics
Bacterial Protocol
Biobrick Construction
Bacterial Experiments
Mammalian Protocol
Microfluidics
Genetic Engineering
Gateway Cloning
Phage Protocol
Basic Protocol
Gateway Cloning
The Mammalian team used Gateway cloning to assemble its composite parts.
1 Gateway Cloning
2 Commented Protocol
     2.1 Design PCR-Primers
     2.2 Gel Purify PCR Product
     2.3 Confirm PCR Product
     2.4 Measure DNA
     2.5 Calculate PCR Product Needed
     2.6 Calculate volume of DONRTM  needed
     2.7 Prepare Gateway Reaction
     2.8 Retrieve 5x BP Clonase II
     2.9 Add BP-Clonase to Gateway Reaction
     2.10 Incubate
     2.11 Add ProteinaseK
     2.12 Transform Bacteria
     2.13 Plate Bacteria
3 Known Issues
4 Citation
Gateway Cloning
Commented Protocol
Design PCR-Primers
The Gateway®  clones have a reading frame which should be kept. Design primers that the PCR product starts with a ATG and ends with a STOP-codon or the last aminoacid (if you want to make a fusion protein). Primer3plus  is a powerful tool helping you to pick primers with the right annealing temperature which should be 60°C. Try to avoid self similarity and other things as usual, but because you are very limited in the position of the primers (its start and stop), I only care about annealing temperature and give it a try. Then just add to the primer which binds the start codon the attB1.1-sequence at his 5' End . To the primer which binds the stop codon or the last aminoacid add the attB2.1-sequence at his 5' End . The open reading frame is indicated and you should change the last two NN to code for an aminoacid of your choice. Good luck for the PCR! Because of the long 5' overhang and the restrictions on picking the primers, getting the PCR to work can be tricky.
Improved and more efficient att sites used to recombine into pDONR 221:
attB1.1  GGG-GCA-ACT-TTg-tac-aaa-aaa-gtt-gNN
attB2.1  GG-GGC-AAC-TTT-GTA-CAA-Caa-agt-tgN
The original att sites used to recombine into pDONR 221:
attB1  GGGG-ACA-AGT-TTg-tac-aaa-aaa-gca-ggc-tNN
attB2  GGG-GAC-CAC-TTT-GTA-CAA-Gaa-agc-tgg-gtN
The att sites used to recombine into pDONR P4-P1R:
attB4  GGGG-ACA-ACT-TTg-tat-aga-aaa-gtt-gNN
attB1  GGG-GAC-TGC-TTT-TTT-GTA-Caa-act-tgN
The att sites used to recombine into pDONR P2R-P3:
attB2  GGGG-ACA-GCT-TTc-ttg-tac-aaa-gtg-gNN
attB3  GGG-GAC-AAC-TTT-GTA-TAA-Taa-agt-tgN
Gel Purify PCR Product
Purification of the PCR-product is needed to get rid of smaller side-products, which remove primer-dimers which can result in false positive colonies. Remember that you want to clone DNA, so the cutting should be made on the weakest UV-light available and as fast as possible. And of course you NEVER make a picture of the gel before. Use the kid for gel-purification available in your lab.
Confirm PCR Product
You need a PCR product with the  attB1.1 and attB2.2 and the DONRTM vector MUST have attP1 and attP2 sites, or it will not work.
The amount of plasmids is not soo important as in a multiple Gateway® reaction, because it is more efficient. If you want to optimize you can calculate equimolar amounts of both plasmids as described in the How to measure DNA. Here we use double the amount of DEST-vector, because most of the ones we use are round and about double the size of the ENTRTM clones.
Measure DNA
The amount of ENTRTM is not so important as in a multiple Gateway® reaction, because it is more efficient. If you want to optimize you can calculate equimolar amounts of both plasmids as described in the multiple Gateway® protocol. Here we use double the amount of DEST-vector, because most of the ones we use are round and about double the size of the ENTRTM clones.
Calculate PCR Product Needed
ng needed = (length of the PCR product in bp) x 0.0165
Calculate volume of DONTR(TM) needed
µl needed = 75 ng needed / (concentration in ng/µl)
The DONR-vector should be tested for low background colonies (due to a mutated ccdB-gene) when transferred in DH5alpha-bacteria.
Prepare Gateway Reaction
PCR-product             ( ? ng)
pDONRTM-vector     (75 ng)
add water to a total volume of 4µl
Retrieve 5x BP-Clonase II
...From -20°C. It is most efficiently mixed by pipetting up and down, do not vortex.
Add BP-Clonase to Gateway Reaction
The enzymes looses 50% activity after 15 freeze-thaw cycles. The advantage of BP-ClonaseTMII is that it can be stored at -20 °C because it contains already the buffer. DO NOT LEAVE OUT.
Incubate
Incubation over-night will enhance the reaction ca. 5-10 fold. This is especially important for PCR products over 5.000 bp.
Add ProteinaseK
This step will enhance the reaction ca. 100 fold!!!!. This is different to the LR-reactions which are only enhanced 2 fold by adding the proteinase K!!!! Transform Bacteria
For electro competent cells use 1-2 µl, for chemical competent all.
Plate Bacteria
The resulting ENTRTM-vectors are kanamycin resistant.
Known Issues
The reaction is very efficient. You can obtain about 200 colonies of which about 95 % are correct.
BP reactions work better with linear templates like PCR-products. If you want to use plasmids, linearize them first with a suitable restriction enzyme.
The obtained plasmids are big. To check for correct clones digest with Sty I and in parallel with Eco RI and Hind III. Compare the pattern of bands with the predicted band size to find the correct clones.
Citation
1. Untergasser A. “Cloning – Gateway BP-Reaction II” Untergasser's Lab. Summer 2006. (include here the date when you accessed these page).
.
Retrieved from " http://2010.igem.org/Team:MIT_gateway "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
