<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_USTC_Project"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu"><UL><LI class="selected"><A href="/Team:USTC/Project">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:USTC/Project&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC/Project&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:USTC/Project&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:USTC/Project" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls"><FORM action="/Special:Search" id="searchform">Â </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:USTC/Project</H1><DIV id="bodyContent"><H3 id="siteSub">From 2009.igem.org</H3><TABLE style="color:#ffffff;background-color:#66b3ff;" cellpadding="3" cellspacing="2" width="960" align="center"><TBODY><TR><TH width="50" align="center"><A href="/Team:USTC" title="Team:USTC">Home</A></TH><TH width="50" align="center"><A href="/Team:USTC/Team" title="Team:USTC/Team">Team</A></TH><TH width="50" align="center"><STRONG class="selflink">Project</STRONG></TH><TH width="50" align="center"><A href="/Team:USTC/Modeling" title="Team:USTC/Modeling">Modeling</A></TH><TH width="50" align="center"><A href="/Team:USTC/Parts" title="Team:USTC/Parts">Parts</A></TH><TH width="50" align="center"><A href="/Team:USTC/Standard_%26_Protocol" title="Team:USTC/Standard &amp; Protocol">Standard &amp; Protocol</A></TH><TH width="50" align="center"><A href="/Team:USTC/Tool" title="Team:USTC/Tool">Software Tool</A></TH><TH width="50" align="center"><A href="/Team:USTC/Human_Practice" title="Team:USTC/Human Practice">Human Practice</A></TH><TH width="50" align="center"><A href="/Team:USTC/Notebook" title="Team:USTC/Notebook">Notebook</A></TH></TR></TBODY></TABLE><H1><SPAN class="mw-headline" id="Team:USTC.2FProject"> Team:USTC/Project </SPAN></H1><TABLE id="toc" class="toc"><TBODY><TR><TD><DIV id="toctitle"><H2>Contents</H2></DIV><UL><LI class="toclevel-1"><A href="#Team:USTC.2FProject"><SPAN class="tocnumber">1</SPAN><SPAN class="toctext">Team:USTC/Project</SPAN></A></LI><LI class="toclevel-2 tocsection-1"><A href="#The_Background"><SPAN class="tocnumber">1.1</SPAN><SPAN class="toctext">The Background</SPAN></A></LI><LI class="toclevel-3 tocsection-2"><A href="#Evolution_vs._Design"><SPAN class="tocnumber">1.1.1</SPAN><SPAN class="toctext">Evolution vs. Design</SPAN></A></LI><LI class="toclevel-4 tocsection-3"><A href="#Evolution_and_design_are_two_sides_of_the_coin."><SPAN class="tocnumber">1.1.1.1</SPAN><SPAN class="toctext">Evolution and design are two sides of the coin.</SPAN></A></LI><LI class="toclevel-4 tocsection-4"><A href="#Why.3F_All_the_complexity_is_solved_by_this_simple_process_without_any_input_of_knowledge.3F"><SPAN class="tocnumber">1.1.1.2</SPAN><SPAN class="toctext">Why? All the complexity is solved by this simple process without any input of knowledge?</SPAN></A></LI></UL><LI class="toclevel-3 tocsection-5"><A href="#Directed_Evolution"><SPAN class="tocnumber">1.1.2</SPAN><SPAN class="toctext">Directed Evolution</SPAN></A></LI><LI class="toclevel-3 tocsection-6"><A href="#Design_.26_Evolutionary_Approaches_in_iGEM_Projects"><SPAN class="tocnumber">1.1.3</SPAN><SPAN class="toctext">Design &amp; Evolutionary Approaches in iGEM Projects</SPAN></A></LI><LI class="toclevel-3 tocsection-7"><A href="#Evolutionary_Biology.2C_Population_Genetics_.26_Evolutionary_Algorithm"><SPAN class="tocnumber">1.1.4</SPAN><SPAN class="toctext">Evolutionary Biology, Population Genetics &amp; Evolutionary Algorithm</SPAN></A></LI><LI class="toclevel-3 tocsection-8"><A href="#Problems_to_Be_Solved"><SPAN class="tocnumber">1.1.5</SPAN><SPAN class="toctext">Problems to Be Solved</SPAN></A></LI><LI class="toclevel-2 tocsection-9"><A href="#The_Blueprint"><SPAN class="tocnumber">1.2</SPAN><SPAN class="toctext">The Blueprint</SPAN></A></LI><LI class="toclevel-3 tocsection-10"><A href="#The_Goal"><SPAN class="tocnumber">1.2.1</SPAN><SPAN class="toctext">The Goal</SPAN></A></LI><LI class="toclevel-3 tocsection-11"><A href="#Modules_.26_Flow_Chart_of_the_System"><SPAN class="tocnumber">1.2.2</SPAN><SPAN class="toctext">Modules &amp; Flow Chart of the System</SPAN></A></LI><LI class="toclevel-3 tocsection-12"><A href="#Scoring_Function"><SPAN class="tocnumber">1.2.3</SPAN><SPAN class="toctext">Scoring Function</SPAN></A></LI><LI class="toclevel-3 tocsection-13"><A href="#Self-Adaptive_Controller"><SPAN class="tocnumber">1.2.4</SPAN><SPAN class="toctext">Self-Adaptive Controller</SPAN></A></LI><LI class="toclevel-3 tocsection-14"><A href="#Variation_Function"><SPAN class="tocnumber">1.2.5</SPAN><SPAN class="toctext">Variation Function</SPAN></A></LI><LI class="toclevel-3 tocsection-15"><A href="#Selection_Function"><SPAN class="tocnumber">1.2.6</SPAN><SPAN class="toctext">Selection Function</SPAN></A></LI><LI class="toclevel-3 tocsection-16"><A href="#Reporter"><SPAN class="tocnumber">1.2.7</SPAN><SPAN class="toctext">Reporter</SPAN></A></LI><LI class="toclevel-2 tocsection-17"><A href="#The_Prototype"><SPAN class="tocnumber">1.3</SPAN><SPAN class="toctext">The Prototype</SPAN></A></LI><LI class="toclevel-3 tocsection-18"><A href="#What_to_Do_First.3F"><SPAN class="tocnumber">1.3.1</SPAN><SPAN class="toctext">What to Do First?</SPAN></A></LI><LI class="toclevel-3 tocsection-19"><A href="#Constitutive_Promoter_Family_as_Stimulus_Signals"><SPAN class="tocnumber">1.3.2</SPAN><SPAN class="toctext">Constitutive Promoter Family as Stimulus Signals</SPAN></A></LI><LI class="toclevel-3 tocsection-20"><A href="#Design_of_the_Self-Adaptive_Controller"><SPAN class="tocnumber">1.3.3</SPAN><SPAN class="toctext">Design of the Self-Adaptive Controller</SPAN></A></LI><LI class="toclevel-3 tocsection-21"><A href="#Principle_of_Operation"><SPAN class="tocnumber">1.3.4</SPAN><SPAN class="toctext">Principle of Operation</SPAN></A></LI><LI class="toclevel-3 tocsection-22"><A href="#Vector_.26_Chassis"><SPAN class="tocnumber">1.3.5</SPAN><SPAN class="toctext">Vector &amp; Chassis</SPAN></A></LI><LI class="toclevel-3 tocsection-23"><A href="#Assembly_Road_Map"><SPAN class="tocnumber">1.3.6</SPAN><SPAN class="toctext">Assembly Road Map</SPAN></A></LI><LI class="toclevel-2 tocsection-24"><A href="#The_Progress"><SPAN class="tocnumber">1.4</SPAN><SPAN class="toctext">The Progress</SPAN></A></LI><LI class="toclevel-2 tocsection-25"><A href="#Beyond_iGEM"><SPAN class="tocnumber">1.5</SPAN><SPAN class="toctext">Beyond iGEM</SPAN></A></LI></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="The_Background"> The Background </SPAN></H2><H3><SPAN class="mw-headline" id="Evolution_vs._Design"> Evolution vs. Design </SPAN></H3><H4><SPAN class="mw-headline" id="Evolution_and_design_are_two_sides_of_the_coin."> Evolution and design are two sides of the coin. </SPAN></H4><P>Until 150 years ago, people believed that we are designed and created by the God, a god, gods or other <A href="http://en.wikipedia.org/wiki/Intelligent_designer" class="external text" rel="nofollow">intelligent designer</A>s. After <A href="http://en.wikipedia.org/wiki/Charles_Darwin" class="external text" rel="nofollow">Charles Darwin</A>'s <I><A href="http://en.wikipedia.org/wiki/On_the_Origin_of_Species" class="external text" rel="nofollow">On the Origin of Species</A></I> in 1859, more and more people began to realize that a god is not necessary. During the process of evolution, we can be created automatically, without any intelligence.
</P><P><A href="http://en.wikipedia.org/wiki/Design" class="external text" rel="nofollow">Design</A> and <A href="http://en.wikipedia.org/wiki/Engineering" class="external text" rel="nofollow">engineering</A> are processes with the need of knowledge. Before we can design a machine, we have to know everything about the possible parts. If any information is unknown, we have to learn it. If nobody knows it, we have to measure it by ourselves. If all the knowledge needed are too much, we have to collaborate with others. <A href="http://en.wikipedia.org/wiki/Computer-aided_design" class="external text" rel="nofollow">Computer-aided design</A> have to be used to accomplish more complex tasks. <A href="http://en.wikipedia.org/wiki/Space_Shuttle" class="external text" rel="nofollow">Space shuttle</A> and <A href="http://en.wikipedia.org/wiki/Microprocessor" class="external text" rel="nofollow">microprocessor</A> are seen as the most complex systems engineered, but they are far too simple when compared with the <A href="http://en.wikipedia.org/wiki/Complexity" class="external text" rel="nofollow">complexity</A> of biological systems created by evolution.
</P><P>On the other side, <A href="http://en.wikipedia.org/wiki/Evolution" class="external text" rel="nofollow">evolution</A> is a completely different process. Variation is random, selection is directional based on the fitness to the environment, that is all. However, all the amazing things on our planet is emerged from this simple process, no more thing is needed.
</P><H4><SPAN class="mw-headline" id="Why.3F_All_the_complexity_is_solved_by_this_simple_process_without_any_input_of_knowledge.3F"> Why? All the complexity is solved by this simple process without any input of knowledge? </SPAN></H4><P>Yes. The answer is on the scale. The evolution process is so simple that it can be scaled up infinitely. The complex problem can be solved in the distributed evolution system. Any success in the distributed system can be amplified by the selection process. Therefore, although the variation is random, it will be powerful enough to search for the solutions, when the scale of the system is big enough.
</P><P>In the design process, the variation is directional based on the knowledge, but the process is not scalable, because of its requirement of intelligence. As a result, it is much more difficult to solve complex problems by design.
</P><H3><SPAN class="mw-headline" id="Directed_Evolution"> Directed Evolution </SPAN></H3><P>Although Darwin's theory was published 150 years ago, people have been using the power of evolution for more than ten thousand years. <A href="http://en.wikipedia.org/wiki/Domestication" class="external text" rel="nofollow">Domestication</A> of plants and animals is done by <A href="http://en.wikipedia.org/wiki/Artificial_selection" class="external text" rel="nofollow">artificial selection</A>, which is the process of intentional breeding for certain traits, therefore changing the direction of evolution.
</P><P><A href="http://en.wikipedia.org/wiki/Directed_evolution" class="external text" rel="nofollow">Directed evolution</A> is a method using the similar principle to create new biomolecules with desired properties. The targets of directed evolution include enzymes, antibodies, <A href="http://en.wikipedia.org/wiki/Aptamer" class="external text" rel="nofollow">aptamer</A>s, <A href="http://en.wikipedia.org/wiki/Ribozyme" class="external text" rel="nofollow">ribozyme</A>s, biosynthetic pathways, and synthetic genetic circuits. More information can be found on <A href="http://www.che.caltech.edu/groups/fha/home.html" class="external text" rel="nofollow">the Frances H. Arnold research group</A>, <A href="http://ellingtonlab.org" class="external text" rel="nofollow">the Ellington lab</A>, and <A href="http://genetics.mgh.harvard.edu/szostakweb/" class="external text" rel="nofollow">the Szostak lab</A>.
</P><P>The directed evolution experiment contains several rounds of 3 steps: variation, selection, and amplification. Variation is the mutation or recombination of the information encoded in the DNA, usually by error-prone PCR and <A href="http://en.wikipedia.org/wiki/DNA_shuffling" class="external text" rel="nofollow">DNA shuffling</A> respectively. Selection is the process of separating the variants with desired phenotypes from others, it can either refer to screening (isolate good variants) or selection (eliminate bad variants), either <I>in vivo</I> or <I>in vitro</I>. Amplification is the replication of the variants after selection, which recovers the population size for the new round of directed evolution.
</P><P>New function of biomolecules or biological systems is difficult to be rationally designed, but directed evolution is successfully used in solving these problems.
</P><H3><SPAN class="mw-headline" id="Design_.26_Evolutionary_Approaches_in_iGEM_Projects"> Design &amp; Evolutionary Approaches in iGEM Projects </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Flow Chart of Engineering a Genetically Engineered Machine</DIV></DIV></DIV><P>In the iGEM competition, teams specify, design, build, and test simple biological systems made from standard, interchangeable biological parts. Most projects engineer the systems by modeling and measurement, including this project itself. This engineering approach is proved to be feasible but difficult in the design of biological systems during this years.
</P><P>Most BioBrick parts have never been characterized, because characterization and documentation of the parts usually takes a lot of time. Further more, the properties of BioBrick parts and devices are very sensitive to the conditions they work in, such as strain, plasmid, other parts used together, culture medium, growth rate, temperature, pH, shaking rate, light, and so on. It is very difficult to characterize all the parameters, so the parts often have to be remeasured in different projects before the behavior of the system can be predicted.
</P><P>There are also many projects using evolutionary approaches. These approaches use selection or screening to find the needed part, eliminate the steps of measurement or constructing new parts with desired properties.
</P><TABLE class="sortable" style="border:3px solid black; width: 90%" align="center"><CAPTION><B>Some of the teams using evolutionary approaches in their project:</B></CAPTION><TBODY><TR><TH align="center" style="background:#f0f0f0;">Team Name / People
</TH><TH align="center" style="background:#f0f0f0;">Competition / Lab
</TH><TH align="center" style="background:#f0f0f0;">Project
</TH><TH align="center" style="background:#f0f0f0;">Approach
</TH><TH align="center" style="background:#f0f0f0;">Wiki Link
</TH></TR><TR><TD>Davidson</TD><TD>iGEM 2006</TD><TD>Solving the Pancake Problem with an E. coli Computer</TD><TD>in vivo Recombination; Selection</TD><TD><A href="http://parts2.mit.edu/wiki/index.php/Davidson_2006" class="external text" rel="nofollow">Davidson</A></TD></TR><TR><TD>Missouri_Western</TD><TD>iGEM 2006</TD><TD>Solving the Pancake Problem with an E. coli Computer</TD><TD>in vivo Recombination; Selection</TD><TD><A href="http://parts2.mit.edu/wiki/index.php/Missouri_Western_State_University_2006" class="external text" rel="nofollow">Missouri_Western</A></TD></TR><TR><TD>Alberta</TD><TD>iGEM 2007</TD><TD>Plan B: Building Butanol BioBricks</TD><TD>Mutagenic Compound; Selection</TD><TD><A href="https://2007.igem.org/Alberta" class="external text" rel="nofollow">Alberta</A></TD></TR><TR><TD>Boston_University</TD><TD>iGEM 2007</TD><TD>Increasing the Current Output of Microbial Fuel Cells Through the Directed Evolution of Shewanella oneidensis</TD><TD>Error-Prone PCR; Screening</TD><TD><A href="https://2007.igem.org/Boston_University" class="external text" rel="nofollow">Boston_University</A></TD></TR><TR><TD>Davidson_Missouri_W</TD><TD>iGEM 2007</TD><TD>Living Hardware to Solve the Hamiltonian Path Problem</TD><TD>in vivo Recombination; Screening</TD><TD><A href="https://2007.igem.org/Davidson_Missouri_W" class="external text" rel="nofollow">Davidson_Missouri_W</A></TD></TR><TR><TD>Harvard</TD><TD>iGEM 2007</TD><TD>Cling-E. coli: Bacteria on target</TD><TD>Screening</TD><TD><A href="https://2007.igem.org/Harvard" class="external text" rel="nofollow">Harvard</A></TD></TR><TR><TD>USTC</TD><TD>iGEM 2007</TD><TD>Extensible Logic Circuit in Bacteria</TD><TD>Error-Prone PCR; Screening</TD><TD><A href="https://2007.igem.org/USTC" class="external text" rel="nofollow">USTC</A></TD></TR><TR><TD>Calgary_Software</TD><TD>iGEM 2008</TD><TD>EvoGEM: An Evolutionary Approach to Designing Life</TD><TD>Evolutionary Algorithm</TD><TD><A href="https://2008.igem.org/Team:Calgary_Software" class="external text" rel="nofollow">Calgary_Software</A></TD></TR><TR><TD>ETH_Zurich</TD><TD>iGEM 2008</TD><TD>Make yourself simpler, stupid! Or how engineering a self-minimizing cell leads to the Minimal Genome</TD><TD>Controlled expression of restriction enzymes and ligases in vivo; Selection</TD><TD><A href="https://2008.igem.org/Team:ETH_Zurich" class="external text" rel="nofollow">ETH_Zurich</A></TD></TR><TR><TD>IIT_Madras</TD><TD>iGEM 2008</TD><TD>StressKit: A BioBrick library of Lac-repressed Ï24, Ï28, Ï32 and Ï38 promoters for Escherichia coli</TD><TD>Screening</TD><TD><A href="https://2008.igem.org/Team:IIT_Madras" class="external text" rel="nofollow">IIT_Madras</A></TD></TR><TR><TD>Illinois</TD><TD>iGEM 2008</TD><TD>Cell-based and in vitro antigenic sensors for medical diagnostics</TD><TD>Error-Prone PCR; Screening</TD><TD><A href="https://2008.igem.org/Team:Illinois" class="external text" rel="nofollow">Illinois</A></TD></TR><TR><TD>Newcastle_University</TD><TD>iGEM 2008</TD><TD>A Computational Intelligence Approach to Developing a Diagnostic Biosensor: The Newcastle BugBusters Project</TD><TD>Evolutionary Algorithm</TD><TD><A href="https://2008.igem.org/Team:Newcastle_University" class="external text" rel="nofollow">Newcastle_University</A></TD></TR><TR><TD>Peking_University</TD><TD>iGEM 2008</TD><TD>A genetic circuit to direct evolution of proteins in vivo</TD><TD>in vivo Mutagenesis; Screening</TD><TD><A href="https://2008.igem.org/Team:Peking_University" class="external text" rel="nofollow">Peking_University</A></TD></TR><TR><TD>Tokyo_Tech</TD><TD>iGEM 2008</TD><TD>Coli.Touch â implementation of a pressure-responsive genetic circuit in E. Coli</TD><TD>Error-Prone PCR; Screening</TD><TD><A href="https://2008.igem.org/Team:Tokyo_Tech" class="external text" rel="nofollow">Tokyo_Tech</A></TD></TR><TR><TD>Warsaw</TD><TD>iGEM 2008</TD><TD>Bacterial device for creating and production of interactors for any given bait protein</TD><TD>in vivo Mutagenesis; Selection</TD><TD><A href="https://2008.igem.org/Team:Warsaw" class="external text" rel="nofollow">Warsaw</A></TD></TR><TR><TD>Jason R. Kelly</TD><TD>Endy Lab</TD><TD>Screening plasmid</TD><TD>Screening</TD><TD><A href="http://openwetware.org/wiki/Endy:Screening_plasmid" class="external text" rel="nofollow">Endy:Screening_plasmid</A></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Evolutionary_Biology.2C_Population_Genetics_.26_Evolutionary_Algorithm"> Evolutionary Biology, Population Genetics &amp; Evolutionary Algorithm </SPAN></H3><P>The process of evolution is as simple as the repeat of variation and selection, but it is also more complex than everything we can imagine. While an organism is evolving to fit the environment, the environment containing other organisms are also evolving. The variation rate of the organism and the selection pressure of the environment themselves are also evolving to keep the survival of the entire ecosystem.
</P><P>There are many theoretical researches of evolution following the work of Darwin. <A href="http://en.wikipedia.org/wiki/Portal:Evolutionary_biology" class="external text" rel="nofollow">Evolutionary biology</A>, <A href="http://en.wikipedia.org/wiki/Population_genetics" class="external text" rel="nofollow">population genetics</A> and <A href="http://en.wikipedia.org/wiki/Evolutionary_algorithm" class="external text" rel="nofollow">evolutionary algorithm</A> are 3 research fields that are closely related to the research of the dynamics of evolution. The first two fields are on the biology side, while evolutionary algorithm is an engineering approach widely used in optimization problems.
</P><P><A href="http://en.wikipedia.org/wiki/Fitness_landscape" class="external text" rel="nofollow">Fitness landscape</A> is often used to characterize evolution. Mountain peaks are high ability to survive, while valleys represent low fitness. Similar genotypes are close to each other, while different ones are far from each other. The population takes an adaptive walk across such a landscape. The paths of evolution is determined by the fitness landscape.
</P><H3><SPAN class="mw-headline" id="Problems_to_Be_Solved"> Problems to Be Solved </SPAN></H3><P>Evolutionary algorithm have been used for more than 50 years, many strategies have been and are being developed to improve the efficiency of the algorithm. These strategies deal with the control of evolution parameters. There are deterministic control, adaptive control (feedback control), and self-adaptive control (the evolution of evolution).
</P><P>Directed evolution experiments are usually carried out by hand, the parameters are optimized according to experience. Therefore, it will take a lot of time to optimize conditions, especially for new targets. When the condition is not optimized, different fitness may not be well separated, and the success rate will be low.
</P><P>If the conditions can be controlled automatically according to theories of evolutionary algorithm, it is expected to greatly increase the success rate and reduce the time, labor and cost. However, automation of the experiments is not practicable under today's technology.
</P><P>Fortunately, the rapid development of synthetic biology has opened another possibility. Biological systems become programmable now. It is possible to automate the process in the cell, to make an <I>E. coli</I> cell an automatic directed evolution machine.
</P><H2><SPAN class="mw-headline" id="The_Blueprint"> The Blueprint </SPAN></H2><H3><SPAN class="mw-headline" id="The_Goal"> The Goal </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Automation of Engineering a Genetically Engineered Machine</DIV></DIV></DIV><P>We designed the <I>E. coli</I> Automatic Directed Evolution Machine (E.ADEM) project.
</P><P>The ultimate goal is to make E.ADEM a universal framework for evolutionary approaches in synthetic biology. Anything we want in synthetic biology can be automatically created, from promoters, RBS, regulators, receptors, binding partners, aptamers, enzymes and ribozymes, to sensors, logic devices, reporters, metabolic pathways, entire genomes, and even solutions of mathematic problems.
</P><P>To each evolution object you want it to evolve, a scoring function can be designed to output PoPS as the fitness score to your demand. After that, you can ligate the scoring function device into the E.ADEM plasmid, transform E. coli, culture the cells and wait for them to evolve automatically and robustly, and get what you want at last.
</P><H3><SPAN class="mw-headline" id="Modules_.26_Flow_Chart_of_the_System"> Modules &amp; Flow Chart of the System </SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Modules &amp; Flow Chart of E.ADEM</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Scoring_Function"> Scoring Function </SPAN></H3><P><B>Possible Design:</B></P><P><B>For Transcription Repressor:</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Scoring Function Design for Transcription Repressor</DIV></DIV></DIV></DIV><P><B>For Device (e.g. Sensor, Logic Gate):</B></P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Supervised Learning Scoring Function Design for Device (e.g. Sensor, Logic Gate)</DIV></DIV></DIV></DIV><P><B>For Enzyme:</B></P><P>If there is a sensor for the product or substrate, use it as the scoring function.
</P><P>Else, perform evolution for a sensor first.
</P><P><B>For Binding Partner:</B></P><P>Use <I>E. coli</I> two-hybrid systems.
</P><H3><SPAN class="mw-headline" id="Self-Adaptive_Controller"> Self-Adaptive Controller </SPAN></H3><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Demand Analysis of the Controller</DIV></DIV></DIV><P>Function: adjust variation rate and selection pressure, base on the fitness score, the population size and the average fitness score calculated by a quorum sensing device.
</P><P>
Problems:
</P><UL><LI>Variation rate
</LI><LI>Too low: Slow
</LI><LI>Too high: Not Stable
</LI></UL><LI>Selection pressure
</LI><LI>Too low: Not Directional
</LI><LI>Too high: Die out
</LI><P>Solutions:
</P><UL><LI>Control strategies in evolutionary algorithm
</LI></UL><P>
See below for our design.
</P><H3><SPAN class="mw-headline" id="Variation_Function"> Variation Function </SPAN></H3><P><B>Possible Design:</B></P><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">An Idea for Avoiding Non-Specific Mutation</DIV></DIV></DIV><UL><LI>Targeted Mutagenesis
</LI><LI>Activation induced cytidine deaminase (AID)
</LI><LI>iGEM 2008 Peking_University
</LI><LI>iGEM 2008 Warsaw
</LI></UL><LI>Multiplex Automated Genome Engineering (MAGE) [<A href="#bibkey_Church">1</A>]
</LI><LI>Error-prone DNA polymerase I
</LI><LI>Bacteriophage
</LI><LI>Error-prone reverse transcription
</LI><LI>Recombination
</LI><LI>Site-specific recombination
</LI><LI>Including inversion, excision/integration and translocation
</LI><LI>Homologous recombination 
</LI><LI>Transposition
</LI><P><OL><LI id="bibkey_Church" value="1"><A href="http://dx.doi.org/10.1038/nature08187" title="View or buy article from publisher" class="extiw" style="color:Black; text-decoration:none">Wang HH, Isaacs FJ, Carr PA, Sun ZZ, Xu G, Forest CR, and Church GM. <I>Programming cells by multiplex genome engineering and accelerated evolution.</I> Nature 2009 Aug 13; 460(7257) 894-8.</A> doi:10.1038/nature08187 pmid:19633652.<SPAN class="noprint"><A href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=19633652" class="extiw" title="PMID 19633652" rel="nofollow">PubMed</A><A href="http://www.hubmed.org/display.cgi?uids=19633652" class="extiw" title="PMID 19633652" rel="nofollow">HubMed</A></SPAN><SPAN class="noprint"><SPAN style="color:#aaa">[Church]</SPAN></SPAN></LI></OL><SPAN class="noprint">All Medline abstracts: <A href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=pubmed&amp;dopt=Abstract&amp;list_uids=19633652" class="extiw" title="All abstracts at PubMed" rel="nofollow">PubMed</A><A href="http://www.hubmed.org/display.cgi?uids=19633652" class="extiw" title="All abstracts at HubMed" rel="nofollow">HubMed</A></SPAN></P><H3><SPAN class="mw-headline" id="Selection_Function"> Selection Function </SPAN></H3><P>Kill or survive the cell:
</P><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Selection Function Design</DIV></DIV></DIV></DIV><H3><SPAN class="mw-headline" id="Reporter"> Reporter </SPAN></H3><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Reporter Design</DIV></DIV></DIV></DIV><H2><SPAN class="mw-headline" id="The_Prototype"> The Prototype </SPAN></H2><H3><SPAN class="mw-headline" id="What_to_Do_First.3F"> What to Do First? </SPAN></H3><P>The Self-Adaptive Controller is the core of the machine. We must finish it first in the prototype machine.
</P><P>The Selection Function is important to test the Self-Adaptive Controller.
</P><P>The Variation Function can be made later.
</P><P>The Scoring Function can be made later. Constitutive Promoter Family can be used as Stimulus Signals for the Self-Adaptive Controller.
</P><H3><SPAN class="mw-headline" id="Constitutive_Promoter_Family_as_Stimulus_Signals"> Constitutive Promoter Family as Stimulus Signals </SPAN></H3><P>We choose to use constitutive promoters instead of the conditional operon impressions (represented by  <A href="http://en.wikipedia.org/wiki/Lac_operon" class="external text" rel="nofollow">IPTG-induced expression</A>) as the stimulus signals to test the system. The stimulus signals in a testing system are supposed to be definite and stable. However, the IPTG-induced signals are susceptible to many environmental factors. The process of inducible expression involves a series of dynamic actions in physical chemistry: the diffusion process of IPTG molecules, and the equilibrium between the attachment and disattachment of IPTG to the promoter. That way, the expression signals would fluctuate in a large scope in experiments and the mathematic analyses would be very complicate.</P><P>Comparatively, the stimulus signals based on a series of constitutive promoters of different levels are far stabler since the process are relatively direct.
<UL><LI>The signals produced by the constitutive promoters can maintain at the steady state during the measurement.
</LI><LI>Constitutive-promoter expressions can give out several different stimulus signals in one system without any disturbation among them. That is, several constitutive promoters can work independently in one system to produce double or triple stimulus signals. Instead, the IPTG-induced testing system can imput only one signal at a time corresponding to the concentration of the IPTG. 
</LI><LI>Two or more systems with different stimulus signals can grow in the same nurture if the signals are produced by the constitutive promoters. For example, in our project, several kinds of E.coli with different imputs signals can grow in the same culture medium preparing for the screen.
</LI></UL>
We have characterized the constitutive promoter family in detail and the measurements and the results are described in the registered parts<A href="https://2009.igem.org/Team:USTC/Parts" class="external autonumber" rel="nofollow">[1]</A>.</P><H3><SPAN class="mw-headline" id="Design_of_the_Self-Adaptive_Controller"> Design of the Self-Adaptive Controller </SPAN></H3><TABLE><TBODY><TR><TD align="right" width="300" height="200"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Design of the self-Adaptive Controller</DIV></DIV></DIV></DIV></TD><TD><P>The self-adaptive controller consists of two quorom sensing parts one tetR parts and a hybrid promoter.
</P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="right" width="300" height="200"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">Two quorom sensing systems</DIV></DIV></DIV></DIV></TD><TD><P>There are two quorom sensing parts in the self adaptive controller. LuxR is expressed by a constitutive promoter and interacts with AHL produced by both LuxI in two parts. The complex then activates hybrid promoter. LuxI in different parts perform different functions. LuxI's expression in Quorom sensing 1 depends on the population density and take effect in evolution in the aspect of cell density. LuxI in part 2, on the other hand, is expressed correspondent to the evolution score directly. Thus it takes part in the evolution process directly.
</P></TD></TR></TBODY></TABLE><TABLE><TBODY><TR><TD align="right" width="300" height="200"><DIV class="center"><DIV class="thumb tnone"><DIV class="thumbinner" style="width:502px;"><DIV class="thumbcaption">TetR parts</DIV></DIV></DIV></DIV></TD><TD><P>TetR is expressed correspondent to the evolution score, just like the LuxI from quorom sensing part 2. However tetR represses the hybrid promoter. High score means high tetR level. High tetR level means low death rate.
</P></TD></TR></TBODY></TABLE><H3><SPAN class="mw-headline" id="Principle_of_Operation"> Principle of Operation </SPAN></H3><UL><LI>Properties of selection system
</LI></UL><P>The selection system consists of hybrid promoter and ccdB. AHL-LuxR complex and tetR input perform contrast effect. We perform several experiments to identify how the hybrid promoter work. The property is as shown below:
</P><UL><LI>How different &quot;scores&quot; result in evolution
</LI></UL><P>When high score and low score E.coli are grown seperately:
</P><P>Low score produce little AHL and little tetR. High score produce more AHL and more tetR at the same time. Two kinds of E.coli will survive in their own way. They will both reach high population density.
</P><P>What if we mix them together?
</P><P>At first, two kinds of E.coli were mixed 1:1. Two kinds of E.coli maintain their own density state.
</P><P>Average AHL level begin to change.Low score E.coli in the high level AHL circumustance will express more ccdB. Then the high score E.coli has a advantage over the low score.More high score ones will survive. The difference between score finally results in the evolution.
</P><H3><SPAN class="mw-headline" id="Vector_.26_Chassis"> Vector &amp; Chassis </SPAN></H3><P>Vector:
</P><P>PSB1A3: high copy number, therefore less sensitive to mutation.
</P><P>
Chassis:
</P><P>Top10: common used strain.
</P><P>MDSâ¢ 42 recA Blue:
</P><P><A href="http://www.scarabgenomics.com/" class="external free" rel="nofollow">http://www.scarabgenomics.com/</A></P><P>Scarab Genomics has bioengineered the Clean GenomeÂ® E. coli by deleting over 15% of the E. coli K-12 genome. Using synthetic biology methods, the K-12 genome was rationally designed by making a series of precise deletions, which included the elimination of non-essential genes, recombinogenic or mobile DNA, and cryptic, virulent genes. This genome reduction optimizes the E. coli strain as a biological factory, providing enhanced genetic stability and improved metabolic efficiency. These properties make the Clean GenomeÂ® the E. coli strain of choice for a wide spectrum of applications ranging from routine cloning to production of biological material for therapeutic purposes.
</P><P>Scarab Genomics was established in 2002. The Clean GenomeÂ® E. coli is the result of Dr. Fred Blattner's research at the University of Wisconsin, Madison. <B>Scarab Genomics has licensed the patented reduced genome technology from Wisconsin Alumni Research Foundation on an exclusive, worldwide basis.</B></P><H3><SPAN class="mw-headline" id="Assembly_Road_Map"> Assembly Road Map </SPAN></H3><P>In order to keep the whole process in perspective, we designed maps to direct our work in wet lab.
</P><TABLE><TBODY><TR><TD align="right" width="300" height="200"><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Assembly Road Map_1: Milestone1.</DIV></DIV></DIV></TD><TD align="justify"><DIV class="thumb tright"><DIV class="thumbinner" style="width:402px;"><DIV class="thumbcaption">Assembly Road Map_2: Milestone2.</DIV></DIV></DIV></TD></TR></TBODY></TABLE><H2><SPAN class="mw-headline" id="The_Progress"> The Progress </SPAN></H2><UL><LI>Assembly work
</LI></UL><P>As shown in the roadmap of our work, we finished assemblying most of the designed system except the pNOT parts. 
We submitted 167 biobricks finally.
</P><UL><LI>Measument workÂ : Most our parts are measured to prove their utility and get useful parameters.
</LI><LI>6 constitutive promoter are measured and marked with barcode system.
</LI><LI>Hybrid promoter and related tetR,LuxI,LuxR are proved to work well in <A href="https://2009.igem.org/Team:USTC/Modeling/Model-2" class="external text" rel="nofollow">our models</A>.
</LI><LI>Several models were established and fitted our data pretty well.
</LI><LI>Density control parts(ccdB related) do not work stably and need further improvements.We tried to add LVA to ccdB to improve its stability ,but it did not work.
</LI></UL><H2><SPAN class="mw-headline" id="Beyond_iGEM"> Beyond iGEM </SPAN></H2><P>The E.ADEM project will continue on OWW:
</P><P><A href="http://openwetware.org/wiki/Synthetic_Biology:Collaborative_Projects" class="external free" rel="nofollow">http://openwetware.org/wiki/Synthetic_Biology:Collaborative_Projects</A></P><P><A href="http://openwetware.org/wiki/E._coli_Automatic_Directed_Evolution_Machine_project" class="external free" rel="nofollow">http://openwetware.org/wiki/E._coli_Automatic_Directed_Evolution_Machine_project</A></P><DIV class="printfooter">
Retrieved from &quot;<A href="http://2009.igem.org/Team:USTC/Project">http://2009.igem.org/Team:USTC/Project</A>&quot;</DIV></DIV></DIV><DIV id="footer-box"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:USTC/Project" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:USTC/Project" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:USTC/Project&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:USTC/Project&amp;oldid=169555" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2009.igem.org:Privacy_policy" title="2009.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2009.igem.org:General_disclaimer" title="2009.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>