"
Page
Discussion
View source
History
teams
Log in
 
Team:Linkoping Sweden/Results
From 2014.igem.org
PROJECT
OVERVIEW
CONTEXT
CULPRIT
CHALLENGE
SOLUTION
BIOLOGY
RESULTS
OVERVIEW
PARTS
POLICY & PRACTICES
OVERVIEW
SURVEYS
MEDIA
COLLABORATION
OUTREACH
SAFETY
NOTEBOOK
TEAM
OUR TEAM
GALLERY
ACKNOWLEDGEMENTS
Our vision is to create a Biobrick which includes the sequence of the Ara h1 protein linked to a red fluorescent protein. We want to accomplish this so that we in turn can express the protein and thus provide an interaction of this protein complex with the antibodies. To ensure that the epitope-red fluorescent protein complex will bind to the Ara h1 specific IgG antibodies several ideas for Biobrick-design was brought to mind. Since we use both monoclonal antibodies specific for epitope 2 on Ara h1 and polyclonal antibodies specific for several epitopes on Ara h1 we decided to create a Biobrick consisting of epitope 2 of Ara h1 linked to a red fluorescent protein as well as a biobrick consisting of five wisely chosen epitopes (epitope 22, epitope 1, epitope 3, epitope 4, and epitope 17) from Ara h1 linked to a red fluorescent protein. However, since there are two different mutants of red fluorescent protein (called RFP and MCherry respectively) we decided to create two setups of every Biobrick combination to ensure that the best possible detection by FRET is attained, as RFP and MCherry differ slightly in their excitation and emission wavelengths. The main idea is to practically test this FRET effect in both epitope-RFP and epitope-MCherry combinations by fluorescence. The data from these tests will show us which mutant is best suited to use and thus prove that our theory works.
Biobricks
These discussed combinations resulted in 4 different Biobricks which are the two combinations of RFP linked to either epitope 2 (Fig. 1) or all 5 epitopes (Fig. 2) as well as the two combinations of MCherry with either epitope 2 or all 5 epitopes.  Protein expressed from the Biobrick including epitope 2 linked to RFP/MCherry will thus be used in combination with the monoclonal antibodies which are specific for epitope 2 as mentioned previously. The 5 different epitopes linked to RFP/MCherry will be used in combination with the polyclonal antibodies. Worth mentioning as well is that all the 4 biobricks contain a sequence for Promotor + RBS as well as a sequence for His-TEV. Furthermore, we designed an additional Biobrick including a sequence for Promotor + RBS and a sequence for His-TEV (Fig. 3).
Figure 1. Schematic illustration of the designed sequence of our biobrick including epitope 2. We have created two identical sequences, one containing RFP (E1010) and one containing MCherry (J06504). However, in this figure one can only see the sequence containing RFP.
Figure 1. Schematic illustration of the designed sequence of our biobrick including epitope 2. We have created two identical sequences, one containing RFP (E1010) and one containing MCherry (J06504). However, in this figure one can only see the sequence containing RFP.
Figure 2. Schematic illustration of the designed sequence of our biobrick including all five epitopes (epitope 22, epitope 1, epitope 3, epitope 4 and epitope 17). We have created two identical sequences, one containing RFP (E1010) and one containing MCherry (J06504). However, in this figure one can only see the sequence containing RFP.
Figure 2. Schematic illustration of the designed sequence of our biobrick including all five epitopes (epitope 22, epitope 1, epitope 3, epitope 4 and epitope 17). We have created two identical sequences, one containing RFP (E1010) and one containing MCherry (J06504). However, in this figure one can only see the sequence containing RFP.
Figure 3. Schematic illustration of our designed biobrick including the sequence of His-TEV.
Figure 3. Schematic illustration of our designed biobrick including the sequence of His-TEV.
Approach
Firstly, we designed and ordered two different pUC57 plasmids from Genscript, one including the sequence of (EcoR1, Xba1, Promotor + RBS, His-TEV, epitope 2, Spe1 and Pst1) and the other including the sequence of (EcoR1, Xba1, Promotor + RBS, His-TEV and epitope 22, epitope 1, epitope 3, epitope 4, and epitope 17, Spe1 and Pst1).  These sequences were digested by using EcoR1 and Spe1.
Furthermore, we used both Biobricks coding for red fluorescent protein which are named E1010 (RFP) and J06504 (MCherry). These were digested with Xba1 and Pst1. Finally, our linearized pSB1C3 backbone was digested with EcoR1 and Pst1 and 4 different ligations were created in the combinations (1) Promotor + RBS, His-TEV, epitope 2 and RFP, (2) Promotor + RBS, His-TEV, epitope 2 MCherry, (3) Promotor + RBS, His-TEV and epitope 22, epitope 1, epitope 3, epitope 4, and epitope 17 and RFP as well as (4) Promotor + RBS, His-TEV and epitope 22, epitope 1, epitope 3, epitope 4, and epitope 17 and MCherry.
We also decided to create another Biobrick which had no connection to our project idea but will be of great use for other iGEM participants in the future. This Biobrick consists of a His-TEV sequence and the idea is to use this Biobrick when you have a plasmid containing a protein sequence which you later on would want to express and purify, using for example a nickel-bead column. (Fig. 3) This Biobrick was created by using our linearized pSB1C3 backbone and one of our ordered pUC57 plasmids. We designed primers specific only for the His-TEV sequence in our pUC57 plasmids and an overnight PCR experiment was set up for the amplification of our His-TEV sequence. The result from the PCR was controlled on an agarose-gel which indicated strongly that our PCR experiment was successful (Fig. 4). Furthermore, our His-TEV sequence was digested with EcoR1 and Pst1 as well as our linearized pSB1C3 backbone and finally ligated.
All 5 of our Biobrick constructs were analyzed by transforming our plasmids into electro-competent E.Coli cells followed by a PCR colony screening.  The following PCR result was controlled by running an agarose-gel which indicated plasmid weights around > 3000 bp for all the epitope and RFP/MCherry constructs as well as a weight around > 2000 bp for our His-TEV biobrick (Fig. 5). Furthermore, all biobricks were analyzed by PCR screening in combination with our His-TEV sequence specific primers. This was done to prove that the ligation was successful by the appearance of a band at the weight for His-TEV on another agarose gel      (Fig. 6). All 5 biobricks were sent to sequencing to further prove that the cloning of our Biobricks had worked. However, unfortunately there was a problem when we were about to send our samples for sequencing. This means that we have not received any response regarding the results yet. Furthermore, since we lack information regarding the sequencing result, we have no other choice than to wait until we can begin to express proteins.
Figure 4. PCR amplification using His-TEV specific primers. (1) DNA ladder (From bottom base-pairs: 250 bp, 500 bp, 750 bp,1k bp, 1.5k bp, 2k bp, 2.5k bp, 3k bp, 3.5k bp, 4k bp, 5k bp, 6k bp, 8k bp, 10k bp). The strongly visible band around 125 base-pairs is indicating that our His-TEV sequence has been amplified correctly.
Figure 4. PCR amplification using His-TEV specific primers. (1) DNA ladder (From bottom base-pairs: 250 bp, 500 bp, 750 bp,1k bp, 1.5k bp, 2k bp, 2.5k bp, 3k bp, 3.5k bp, 4k bp, 5k bp, 6k bp, 8k bp, 10k bp). The strongly visible band around 125 base-pairs is indicating that our His-TEV sequence has been amplified correctly.
Figure 5. Control of the colony-screening experiment performed on all 5 different biobricks. (1) DNA ladder (From bottom base-pairs: 250 bp, 500 bp, 750 bp,1k bp, 1.5k bp, 2k bp, 2.5k bp, 3k bp, 3.5k bp, 4k bp, 5k bp, 6k bp, 8k bp, 10k bp).  (2) His-TEV Biobrick, colony 2 (3) RFP + all 5 epitopes, colony 1 (4) RFP + all 5 epitopes, colony 2 (5) RFP + all 5 epitopes, colony 3 (6) MCherry + all 5 epitopes, colony 2 (7) MCherry + epitope 2, colony 2 (8) RFP + epitope 2, colony 1 (9) RFP (E1010) Biobrick (10) Control (11) Empty super-competent cells (12) DNA ladder.
Figure 5. Control of the colony-screening experiment performed on all 5 different biobricks. (1) DNA ladder (From bottom base-pairs: 250 bp, 500 bp, 750 bp,1k bp, 1.5k bp, 2k bp, 2.5k bp, 3k bp, 3.5k bp, 4k bp, 5k bp, 6k bp, 8k bp, 10k bp).  (2) His-TEV Biobrick, colony 2 (3) RFP + all 5 epitopes, colony 1 (4) RFP + all 5 epitopes, colony 2 (5) RFP + all 5 epitopes, colony 3 (6) MCherry + all 5 epitopes, colony 2 (7) MCherry + epitope 2, colony 2 (8) RFP + epitope 2, colony 1 (9) RFP (E1010) Biobrick (10) Control (11) Empty super-competent cells (12) DNA ladder.
Figure 6. Control of the PCR screening experiment performed using His-TEV sequence specific primers. (1) DNA ladder (From bottom base-pairs: 250 bp, 500 bp, 750 bp,1k bp, 1.5k bp, 2k bp, 2.5k bp, 3k bp, 3.5k bp, 4k bp, 5k bp, 6k bp, 8k bp, 10k bp). (2) His-TEV Biobrick, colony 1 (3) His-TEV Biobrick, colony 2 (4) His-TEV Biobrick, colony 3 (5) RFP + all 5 epitopes, colony 1 (6) RFP + all 5 epitopes, colony 2 (7) RFP + all 5 epitopes, colony 3 (8) RFP + all 5 epitopes, colony 4 (9) MCherry + all 5 epitopes, colony 1 (10) MCherry + all 5 epitopes, colony 2 (11) MCherry + all 5 epitopes, colony 3 (12) MCherry + all 5 epitopes, colony 4 (13) RFP + epitope 2, colony 1 (14) RFP + epitope 2, colony 2 (15) MCherry + epitope 2, colony 1 (16) MCherry + epitope 2, colony 2 (17) DNA ladder.
Figure 6. Control of the PCR screening experiment performed using His-TEV sequence specific primers. (1) DNA ladder (From bottom base-pairs: 250 bp, 500 bp, 750 bp,1k bp, 1.5k bp, 2k bp, 2.5k bp, 3k bp, 3.5k bp, 4k bp, 5k bp, 6k bp, 8k bp, 10k bp). (2) His-TEV Biobrick, colony 1 (3) His-TEV Biobrick, colony 2 (4) His-TEV Biobrick, colony 3 (5) RFP + all 5 epitopes, colony 1 (6) RFP + all 5 epitopes, colony 2 (7) RFP + all 5 epitopes, colony 3 (8) RFP + all 5 epitopes, colony 4 (9) MCherry + all 5 epitopes, colony 1 (10) MCherry + all 5 epitopes, colony 2 (11) MCherry + all 5 epitopes, colony 3 (12) MCherry + all 5 epitopes, colony 4 (13) RFP + epitope 2, colony 1 (14) RFP + epitope 2, colony 2 (15) MCherry + epitope 2, colony 1 (16) MCherry + epitope 2, colony 2 (17) DNA ladder.
Antibody labeling
Our project idea is also based on the labeling of our antibodies with a green fluorescent probe called FITC. The labeling was performed by following the labeling protocol from Thermo Scientific and the result was analyzed by measuring the absorbance. In (Fig. 7) one can see the result from the FITC labeled polyclonal antibodies in which the concentration of FITC at Amax (494 nm) was calculated to be 11.79 microM as well as the calculated protein concentration at A280 (277.5) turned out to be 5.6 microM, indicating a 2:1 ratio of labeling. The FITC labeled monoclonal antibodies was calcultated to have a concentration of 5.0 microM  FITC at Amax (493 nm) and the calculated protein concentration at A280 (278.5) turned out to be 4.7 microM, indicating a 1:1 ratio of labeling. In (Fig. 8) one can see a schematic illustration of an FITC labeled antibody. Our goal is to test our FRET effect by expressing our RFP/MCherry linked epitope protein before entering the competition in Boston. Hopefully we will be able to perform this before the deadline, otherwise we will continue with our project after the competition.
Figure 7. Labeling of monoclonal and polyclonal antibodies with a green fluorescent probe (FITC).
Figure 7. Labeling of monoclonal and polyclonal antibodies with a green fluorescent probe (FITC).
Figure 8. Schematic illustration of an antibody labeled with FITC.
Figure 8. Schematic illustration of an antibody labeled with FITC.
Linköping University 581 83 Linköping, Sweden liuigemgroup@gmail.com
Copyright (c) 2014 igem.org. All rights reserved.
Retrieved from " http://2014.igem.org/Team:Linkoping_Sweden/Results "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
