<HTML lang="en" dir="ltr" class="client-nojs">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_UPMC_PARIS_Notebook skin-igem action-view"><DIV id="globalWrapper"><DIV id="content" class="mw-body" role="main"><DIV id="top_title"><H1 id="firstHeading" class="firstHeading"><SPAN dir="auto">Team:UPMC PARIS/Notebook</SPAN></H1></DIV><DIV id="HQ_page"><DIV id="bodyContent"><DIV id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><P><TITLE>Impact UPMC</TITLE></P><DIV id="navbar-main"><DIV class="navbar navbar-inverse navbar-fixed-top"><DIV class=""><DIV class="navbar-collapse collapse"><UL class="nav navbar-nav"><LI><A href="https://2017.igem.org/Team:UPMC_PARIS">Home</A></LI><LI><DIV class="btn-group show-on-hover"><A href="#">
                    Team </A></DIV></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Team">Team</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Collaborations">Collaborations</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Sponsors_team">Sponsors</A></LI></UL></DIV><LI><DIV class="btn-group show-on-hover"><A href="#">
                    Project </A></DIV></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Description">Description</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Design_engineering">Design</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Experiments">Experiments</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Notebook">Notebook</A></LI></DIV><LI><DIV class="btn-group show-on-hover"><A href="#">
                    Parts </A></DIV></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Parts">Parts</A></LI></DIV><LI><DIV class="btn-group show-on-hover"><A href="#">
                    Manufacturing </A></DIV></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/State_of_art">State of art</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Entrepreneurship_manufacturing">Entrepreneurship</A></LI></DIV><LI><DIV class="btn-group show-on-hover"><A href="#">
                    Human practices </A></DIV></LI><LI class="dropdown-submenu"><A tabindex="-1" href="#">Silver HP</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Education">Education</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Laws_and_regulation">Laws and regulations</A></LI><LI class="dropdown-submenu"><A tabindex="-1" href="#">Gold HP</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/HP/Gold_Integrated">Integrated and Gold</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Gold_HP_Biology">Biology</A></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Engagement">Public engagement</A></LI></DIV><LI><DIV class="btn-group show-on-hover"><A href="#blog">
                    Awards </A></DIV></LI><LI><A href="https://2017.igem.org/Team:UPMC_PARIS/Entrepreneurship">Entrepreneurship</A></LI></DIV><LI><A href="https://igem.org/2017_Judging_Form?team=UPMC_PARIS"> Judging form</A></LI></DIV></DIV></DIV><DIV class=""><SECTION class="fullSize" id="about"><DIV class="row"><DIV class="col-lg-8 col-lg-offset-2 text-center"><H1 class="section-heading content-title">Notebook</H1><H3 class="content-title">Protocols iGEM Team 2017 UPMC-Paris</H3><H4>Bacterial competentiation</H4><P style="text-align: justify;">
We started from frozen E. coli DH5α which we thaw on ice and then streak on Lysogeny Broth
(LB) agar plates (autoclaved 20 g sterile agar for 1L distilled water) that we incubated
overnight at 37°C. The next day we inoculated one colony on 4 mL LB (autoclaved 25g LB in
distilled water), overnight at 37°C and 220 rpm. The following day, we inoculated 500µL of
these pre-culture in 50 mL of LB and incubated them at 37°C until OD600: 0,5. At this stage,
bacterial cells are centrifuged (10mn, 4°C, 4200 rpm), and pellets were gently resuspended in
2 mL of T1 solution (30mM acétate K, 100mm KCI, 10mM CacI2, 15% glycerol and 50mM
MnCI2 added extemporary) and we completed to 15mL. We incubated cells in this solution
for one hour and centrifuge them 10mn at 4°C/ 2300 rpm. We resuspended in 2 mL of solution
T2 (MOPS pH7 10mM, KCI 10mM, CaCI2 75mM, Glycérol 15%) and was aliquoted into
Eppendorf carefully frozen on dry ice with ethanol.

</P><H4>Transformation</H4><P style="text-align: justify;">
DNA was resuspended with 10 µL of water and 1 µL of DNA incubated with 50 µL of the
competent cells DH5α on ice for 30 minutes. We realized heat-shock at 42°C for 45s and
transferred quickly tubes on ice for 5mn. We added 950 µL of SOC media per tube, and
incubated them at 37°C for 1 hour shaking at 200-300rpm. We collected 100 µL from each
tubes and spread in triplicate the mixture evenly across the LB agar plates with
chloramphenycol (25 µg/mL). We incubated them with the agar side at the top at 37°C
overnight for approximately 16 hours, because growing with chloramphenycol is slower than
with other antibiotics. The next day we evaluated the number of colonies on a light field based.
</P><H4>Plasmid DNA extraction</H4><P style="text-align: justify;">
We used the Monarch Plasmid Miniprep kit from NEB.
We realized all steps of centrifugation at 16000g. 5 mL of culture was pelleted by
centrifugation during 30 seconds and the supernatant was discarded. Cells was totally
resuspended in 200μl of Plasmid Resuspension Buffer (B1) and lysed with 200 μl Plasmid Lysis
Buffer (B2) and 5-6 time inversion tubes. After one minute of incubation and in order to avoid
plasmid irreversible denaturation, we added to neutralize lysate 400 μl of Plasmid
Neutralization Buffer (B3). After 2 minutes, we centrifuge and carefully transfer supernatant
to the spin column and repeat centrifugation for 1mn. We removed RNA, protein and
endotoxin with 200 μl of Plasmid Wash Buffer 1 and then after centrifugation with 400 μl of
Plasmid Wash Buffer2. To finish, we made elution with 30μl of DNA elution buffer. Plasmid
DNA concentration was measured with nanodrop.
</P><H4>Polymerisation Chain Reaction</H4><P style="text-align: justify;">
We realized DNA amplification with Q5® High-Fidelity 2X Master Mix (NEB #M0492S). To make
25µL reaction, we added 12.5µL of Q5 High-Fidelity 2X Master Mix, 1.25µL of each forward/
reverse primers (10µM), 250 ng of DNA and we completed with nuclease free water. We 
gently mixed reaction and transferred PCR tubes from the ice to a PCR machine with the block
preheated to 98°C and began thermocycling.
We have designed our primers to amplify the desired DNA sequence, adding restriction
recognition sites of the enzymes that will be used in cloning at 5’ ends of the primers, with a
few extra nucleotides prior to the restriction site to increase restriction efficiency. To do so,
we used a Tm calculator tool from <A href="https://tmcalculator.neb.com/#!/"> NEB</A> to design primer pair
with no more difference of temperature than 5°C.


The thermocycling program was:
<UL><LI>98°C/ 30s for initial denaturation</LI><LI>98°C/ 10s for denaturation, TM°C/ 20s for hybridation, 72°C/ 20s for elongation (X30)</LI><LI>72°C/ 2mn for final extension</LI><LI>hold 4°C</LI></UL></P><H4>Purification products PCR :</H4><P style="text-align: justify;">
We used the Monarch DNA Cleanup extraction Kit from NEB. We realized all steps of
centrifugation at 13K RPM. First, we resuspended PCR products with 80μl of DNA Cleanup
Binding Buffer and transfer mix into spin column. After one minute of spinning we discarded
flow-through. We added 200 μl of DNA Wash Buffer was added before 1 minute of spinning
(these steps were repeated twice time). We transferred columns into a clean 1.5 ml microfuge
tube before to add 20μl of DNA Elution Buffer to the center of the matrix. After 1 minute, we
eluted DNA by one minute of centrifugation. Concentration and purity of DNA was assessed
with nanodrop.
</P><H4>Resuspention of synthetized DNA by IDT
</H4><P style="text-align: justify;">
DNA was centrifuge 5s/ 300g and TE 1X (1M Tris-HCL pH 7.5-8; 0.5M EDTA pH8; sterile H20)
was added in order to reach a final concentration of 10 ng/uL. Resuspended DNA was vortexed
and incubated 20mn at 50°C before centrifugation 3/5s (300g mini).
</P><H4>Cloning and part assembly </H4><P style="text-align: justify;">
We used the Biobrick assembly kit from NEB.

First, we digested purified PCR products with prefix and suffix restriction enzymes. 500 ng of
upstream parts were individually digested with EcoRI and SpeI in a digestion mix containing
5µL of 10X digestion Buffer 2.1, 1 µL of each enzyme and sterile water to complete the mix
until a final volume of 50µL. 500 ng of downstream parts were individually digested by XbaI
and PstI and 100 ng of pSB1C3 plasmid by EcoRI and PstI enzymes in the same conditions that
upstream parts. Digestion mix were incubated at 37°C for 10 minutes and then heat
inactivated at 80°C for 20 minutes. Then, we gather our devices by using the T4 DNA ligase
from NEB. 2 µl of EcoRI/SpeI digested upstream parts were mixed with 2 µl of XbaI/PstI
digested downstream part, 2µL of EcoRI/PstI digested plasmid, 2 µl of 10X T4 DNA ligase
Buffer, 1 µl of ligase and 11 µL of sterile water. Then, ligation mix was incubated at room
temperature for 10 minutes and heat inactivated at 80°C for 20 minutes. The assembly
construction was used for the transformation of E. coli DH5α. After transformation, cells were
spreaded in LB culture overnight in LB+ Chloramphenycol.
</P><H4>GFP and RFP quantification:</H4><P style="text-align: justify;">
After transformation, we selected recombinant clones directly by visualization of GFP and RFP
fluorescence through epifluorescent microscope Zeiss Axiovision. We inoculated 4 mL of LB +
Chloramphenycol with colonies emitting GFP and RFP fluorescence and incubated then
overnight at 37°C. The next day, bacterial cells were all diluted in LB+ Chloramphenycol at a
final OD600 of 0,2. We quantified GFP and RFP production of bacteria in 96-well microwell
plate during their exponential stage (OD600 of 0.5). by using the spectrophotometer
Molecular Device SpectraMax M4 with the following set-up.

Transformed cells were synchronized by dilution on LB+ Chloramphenycol at OD600: 0,2.
Cultures were aliquoted (150 µl) into a 96-well plate (Greiner Bio-One) in which OD (600 nm)
and fluorescence for GFP (BBa_I13500) and RFP (BBa_I13507) were read using a SpectraMax
M4 fluorescence plate reader (Molecular Devices) along exponential stage.

Finaly, we made ratio between protein synthesis (fluorescence) and growth rate (optical
density) of our different transformed strains and measured each sample in triplicates.
</P><P style="text-align: left;"><B class="center">GFP (BBa_I13500) :</B><LI>Excitation wavelength : 485nm</LI><LI>Emission wavelenght : 530nm</LI><LI>Integration time : 500 ms</LI><B>RFP (BBa_I13507) : </B><LI>Excitation wavelength : 584 nm</LI><LI>Emission wavelenght : 607nm</LI><LI>Integration time : 500 ms</LI></P><H4>SKP cytoplasmic genomic integration:</H4><P style="text-align: justify;">
The DH5a competent strain was transformed with 1uL of the construction and select on
LB+ampicillin (100ug/ml) at 32°C in triplicate. After two days we count few colonies
(approximately 10) resistant to ampicillin. We picked 3 colonies and streaked out once
overnight in LB+ampicillin, 32°C. We started cultures overnight of 3 colonies in LB without
ampicillin at 32°C (this step allows for some loss of plasmid) and grow overnight. We diluted
cultures appropriately to recover single cells (I dilute 10-7 and plate 50 uL of culture) and plate
on LB plates. Grow at 42°C to block replication of the plasmid. Streak colonies once on LB at
42°C to ensure the complete loss of the plasmid.

After this step of integration and before characterization of the effect of SKP in protein
synthesis we made verifications:

A simple checking was to pick the next day 3 colonies to put them in culture in LB+/- ampicillin.
</P><H4>Electrophoresis</H4><P style="text-align: justify;">
We realized an agarose gel electrophoresis (1.5%) using agarose powder with 100ml of 1x TAE
buffer, heated until all agarose is dissolved Agar Low Melting… dissolved in TAE 1X before to
add 6μl of Bet. Mix 1μl of 1kbp DNA ladder with 6μl of loading dye (bromophenol blue) and
4μl of 1x TAE buffer (total 6ul) and load onto first well. We Mixed 10μl of PCR products with
2μl of loading dye 6X before loading into wells. Running gel electrophoresis was made at 10V
for 30 minutes approximately and photograph gels under UV light
</P><H4>Lysis of bacteria:</H4><P style="text-align: justify;">
To produce the proteins to purify, we used DH5α containing PET21 GST-X10HIS 
These bacteria were cultivated from a glycerol stock maintained at -80°C.
Volumes vary depending on the purification system. So, for the sake of clarity the volumes are presented in Table.1. 

The lysis of the bacteria took place at the middle of the exponential phase at t = 6h. The experiments were carried out at 4°C. We centrifuged the bacteria at 3000 RPM for 10 min at 4°C. The bacterial pellet was then resuspended in PBS containing lysosyme (1 mg/ml) for 30 min then mechanical lysis was made by vortexing for 10 min. A centrifugation (12000 rpm during 10 min) was then performed and the supernatant was recovered. 60 μL of the supernatant were taken for an SDS-PAGE (unpurified fraction) and a 15 μL fraction was used to study the enzymatic activity.
</P><H4>Purification with Gluthation Agarose Resin</H4><P style="text-align: justify;">
From the bacterial lysis supernatant, we purified the GST-10XHIS through a 0.5 ml Glutathione-agarose resin. To do so, we added our sample and centrifuged the resin four times and retrieved the supernatant. Then we added 100µL of GSH 100mM and centrifugated 2times. We recuparated the surpernant which represent the elution sample. 
We also took  60 µL and 10 µL of aliquot from it to perform a SDS-PAGE and determine the GST enzymatic activity respectively.
</P><H4>Purification HiTrap Column</H4><P style="text-align: justify;">

We used a HiTrap 1ml column to perform this purification.
We used a siringe to file our proteins into the column.

First, we created the binding buffer and the elution buffer.
-Binding buffer (Imidazole 20 mM) :  3.75 ml of Phosphate buffer solution 
                                                                0.3 ml of Imidazole 2M 
                                                                qs water to 30 ml
-Elution buffer (Imidazole 500 mM) : 1 ml of Phosphate buffer solution
                                                                 2 ml of Imidazole 2M
                                                                 qs water to 8 ml


Before purifying our proteins sample, we preteated it by adding Imidazole so that it has the same concentration as the binding buffer (20 mM). 
-Steps for the purification :
1.	Fill the syringe with distilled water. Remove the stopper and connect the column to the syringe tubing « drop to drop » to avoid introducing air to the system.
2.	Remove the twist-off end.
3.	Wash the column with 5 column volumes of distilled water.
4.	Equilibrate the column with 5 column volumes of binding buffer with a flow rate of 1ml/min.
5.	Apply the pretreated sample.
6.	Wash with 5 column volumes of binding buffer.
7.	Eluate with 5 column volumes of elution buffer.

</P><H4>Mesurement of the GST enzymatic activity </H4><P style="text-align: justify;">
We took aliquots from the supernatant before and after purification in order to measure the enzymatic activity of the GST. 
The measurement of this enzymatic activity allows us to obtain a quantitative analysis of the GST and thus a final purification yield.

The measurement of the enzymatic activity of the GST was done through the following reaction:


                                              GST                     
                                  GSH + CDNB ---&gt; GS-DNB + HCl           


Glutathione (GSH) and CDNB are substrates of GST.
The absorbance of GS-DNB at 340 nm allowed us to quantify the enzymatic activity of GST.
Note that the reaction between GSH and CDNB can be done without the presence of GSH. This is the chemical activity. So, we standardized our OD measurements according to this absorbance values.
From this enzymatic activity, a yield can be obtained. Yield = Final total activity / Initial total activity
Initial total activity = Enzyme activity for 5 l * Total Volume of the supernatant bacterial lysis
Total final activity = Enzymatic activity for 5 l * purification supernatant Total Volume

</P></DIV></DIV></SECTION></DIV></BODY></HTML>