Team:UPMC PARIS/Experiments
Impact UPMC
Home
Team
Team
Collaborations
Sponsors
Project
Description
Design
Experiments
Notebook
Parts
Parts
Manufacturing
State of art
Entrepreneurship
Human practices
Silver HP
Education
Laws and regulations
Gold HP
Integrated and Gold
Biology
Public engagement
Awards
Entrepreneurship
Judging form
Experiments
Part  I - Characterization  of  the  relative  strength  of  some  promoters  with  polycistronism effect
Introduction :
The insertion of chaperone genes in bacterial chromosome is an efficient system to ensure   the  stability of  our  optimized  E.  coli strain  for  the  expression  of  eukaryotes  recombinant   proteins. However,  the  transposition one hampers system  that we used  to  target a unique   insertion site. To ensure an adequate production of chaperone for the proper folding of all   recombinant  proteins,  we  decided  to  insert  two  copies  of  our  chaperone  gene  through  a   polycistronic construct under the control of a strong promoter.
The Anderson Promoters include a large panel of constitutive promoters family with different levels of strengths. They are mainly including into  the  J61002 backbone expressing RFP  for color screening. Lot of team projects are based on these types of promoters and they were previously well characterized. However, their ability to control multiple gene expression was never determined.
We characterized  strength  of  several  promoters  from  the  iGEM  community,  which  are BBa_J23101,  BBa_J23102,  BBa_J23104,  BBa_J23107  and  BBa_J23110.  In  other  hand,  we wanted  to  evaluate  their  level  of  strength  in  the  context  of  polycistronism,  with  GFP (BBa_I13500) and RFP  (BBa_I13507)  reporters downstream. We  think  that it could be very interesting to assess the ability of these promoters to control several gene expression, in order to  choose  which  one  of  them  is  more  appropriate  for  us  to  induce  an  high  chaperone expression.
Achievements :
- We  documented  at  least  one  new  substantial  contribution  to  the  iGEM  community  that showcases a project made with BioBrick.
- We measured the strength variation of 5 differents promoters in the context of a simple or multiple gene expression, allowing us to selected for our project the promoter BBa_J23107, showing high level of activity.
Method
In  order  to  assess  the  strength  of  various  promoters  from  the  Anderson  family  we  built  a polycistronic  reporter  including  GFP  (BBa_I13500)  and  RFP  (BBa_I13507)  which  we  places under the control of characterized promoters.
Figure 1 Global strategy about our study of Anderson’s Promoter characterization
Amplification of iGEM parts
We  transformed E. coli DH5α competent bacteria with DNA  from  the iGEM distribution kit 2017.  The  Anderson’s  promoters  BBa_J23101,  BBa_J23102,  BBa_J23104,  BBa_J23107, BBa_J23110, GFP coding sequence BBa_I13500 and RFP coding sequence BBa_I13507.
DNA amplification of parts and cloning of devices
After transformation of bacterial cells we proceed to a plasmid DNA extraction with the  the Monarch Plasmid Miniprep kit from NEB. Then we amplified each part by PCR using the Q5® High-Fidelity 2X Master Mix (NEB #M0492S) with following primers.
Primer Foward: VF 5’tgccacctgacgtctaagaa 3’
Primer Revers : VF2 5’attaccgcctttgagtgagc 3’ primers
(Tm: 60°C)
We used the Monarch DNA Cleanup extraction Kit from NEB to purify the PCR products.
We used the Biobrick assembly kit from NEB to digest GFP BBa_I13500 upstream part and RFP reporter downstream part and cloning the new device GFP-RFP in the pSB1C3 plasmid.
The next day we follow the same scheme in order to assemble our final polycistronic device Characterized promoter - GFP - RFP.
After transformation, we selected recombinant clones directly by visualization of GFP and RFP fluorescence through epifluorescent microscope Zeiss Axiovision.
We  quantify  GFP  and  RFP  production  in  bacteria in  96-well  microwell  plate  during  their exponential phase (OD600 of 0.5) using the spectrophotometer Molecular Device SpectraMax M4. Finally, we made ratio between protein synthesis (fluorescence) and growth rate (optical density) of our different transformed strains and measured each sample in triplicates.
Results
Figure 2 Quantification of GFP fluorescence and RFP in the context of polycistronic reporter under the control of Anderson’s promoters
Promoter strength was assessed through the quantification of specific fluorescence from GFP   (A) and RFP (B) normalized with OD600 to obtain relative fluorescent units (RFU).
Discussion
The  specific  relative  fluorescence  of  the  original Anderson  Promoters  in  backbone  pSB1C3 shows  strength variations with GFP  reporter BBa_I13500. The  strongest promoter of  the 5 measured ones  is  BBa_J23107  (15796,8U)  followed  by  BBa_J23104  promoter  (9443,2  U). BBa_J23102  and  BBa_J23110  exhibit  a  comparable  activity  than  BBa_J23101  promoter (approximately  2500  U).  Our  system  of  polycistronic  reporter  show  that  all  of  the  five promoters tested allow the a polycistronic production of proteins. Surprisingly, the level of RFP fluorescence is not affected by the strength of these different promoters. These results suggest  in  our  study  that  the  level  of  activity  in  the  context  of  polycistron  is  not  directly affected  by  the  strength of  the  promoter. One  hypothesis of  these observation is  that  the distance between the sequence GFP and RFP was not enough important to look for the effect on the second reporter expression, and it could be more reliable to use Lux genes in order to avoid  the  overlap  between  fluorescences.  To  conclude,  our  work  convince  us  to  use BBa_J23107 promoter in the second part of our work.
Part  II-  SKP  integration  into  bacterial  chromosome
Introduction :
Our  purpose  was  to  produce  antibodies  through  our  transportable  factory  "The  BioMaker  Factory".  As  many  eukaryotic  proteins  it  depends  a  lot  of  various  post-traductional  modification  and  a  precise  tridimensional  architecture.  In  this  way,  it’s  appears  important  for  us  to  coexpress  our  recombinant  protein  with  chaperones.  However,  WHO  legislations  demand  a  total  safety  of  sanitary  product  exempt  from  any  dangerous  substance.    So  il  was  a  necessity  for  us  to  avoid  usage  of  antibiotics  during  cell  growth.  The  direct  insertion  of  chaperone  gene  within  the  bacterial  genome  appeared  to  be  the  best  way  to  ensure  a  stable  expression  of  additional  chaperone  proteins  into  the  genome  of  our  designed  bacteria  allowing  a  proper  folding  of  eukaryotic  therapeutical  proteins.
Method:
To  do  so,  we  used  pGRG25  integration  vector  obtained  from  Addgene  in  transformed  DH5α  E.  coli.  This  vector  was  designed  by  McKenzie  et  al  to  insert  DNA  sequence  in  bacterial  chromosome  without  inducing  drug  resistance  of  the  host.  This  system  uses  Tn7  to  insert  transgenes  at  a  defined  neutral  site  in  the  chromosome  (attTn7).  The  site  is  highly  conserved  and  is  known  to  work  as  a  Tn7  attachment  site  in  E.  coli  and  its  relatives.  The  attTn7  sequence  is  conserved  in  most  (all)  bacteria.
Construction  of  cytosolic  Skp  gene :
We  had  synthesized  a  cytosolic  form  of  Skp  under  the  control  of  T7  promoter    and  ended  by  a  T7  terminator  by  IDT.
We  amplified  by  Polymerisation  Chain  Reaction  the  Skp  CDS  preceded  by  its  RBS  and  ended  by  the  T7  promoter  with  the  following  primers,  introducing  XbaI  restriction  site  and  SpeI  NotI  restrictions  sites.
Primer  forward  :  5’GCGGCCGCTACTAGTATTATTTAACCTG3’ Primer  Reverse  :  5’CTTCTAGAGCCATGGCTGACAAAA3’
TM  :67,3°C
We  used  the  Biobrick  Assembly  kit  to  place  Skp  under  the  control  of  the  promoter  from  the  iGEM  distribution  kit  2017  BBa_J2310  we  used  this  new  construct  to  transformed  E.  coli  competent  DH5α.
Cloning  of  Skp  in  pGRG25  :
Culture  of  these  strain  was  made  at  30°C.  Indeed,  the  plasmid  backbone  is  the  easily  curable  temperature  sensitive  mutant  of  pSC101,  carrying  the  pSC101  temperature  sensitive  origin,  which  must  be  grown  at  30-32°C  to  allow  replication.  So  in  order  to  extract  plasmids,  we  inoculate  these  bacterial  cells  overnight  in  culture  with  ampicillin.  We  inserted  the  Skp  sequence  into  the  NotI  restriction  site  located  on  the  MCS  of  pGRG25  and  then  we  transformed  E.  coli  competent  DH5α  with  the  cloned  vector.
Integration  of  the  cytoplasmic  form  of  Skp  in  E.  coli  DH5α  genome
Cultures  of  the  transformed  cloned  cells  was  made  overnight  without  antibiotics  at  32°C  (this  step  allows  for  some  loss  of  plasmid)  and  then  at  42°C  to  block  replication  of  the  plasmid.  Colonies  were  streaked  once  on  LB  at  42°C  to  ensure  the  complete  loss  of  the  plasmid  and  DNA  integration  into  the  genome.
After  this  step  of  integration  and  before  characterization  of  the  effect  of  SKP  in  protein  synthesis  we  made  verifications:
A  simple  checking  was  to  pick  the  next  day  3  colonies  to  put  them  in  culture  in  LB+/-  ampicillin.
Verification  of  the  genomic  insertion  of  cytosolic  Skp
We  check  out  the  proper  insertion  of  our  Skp  by  performing  a  PCR  on  colonies  allowing  the  specific  amplification  of  cytosolic  Skp  with  the  following  primers:
Primer  forward  :  5’GCGGCCGCTACTAGTATTATTTAACCTG3’ Primer  Reverse  :  5’CTTCTAGAGCCATGGCTGACAAAA3’
TM  :67,3°C
Transposition  was  verified  by  the  absence  by  PCR  amplifying  sequences  which  flank  the  attTn7  site  with  the  following  primers.
Primer  Foward    5'GATGCTGGTGGCGAAGCTGT3’  and  5'GATGACGGTTTGTCACATGGA3’
Then,  we  purified  DNA  from  PCR  and  we  visualized  PCR  product  on  an  agarose  gel  electrophoresis  (1.5%).
Characterization  of  Skp  insertion
In  order  to  assess  the  effect  related  to  the  SKP  chromosomique  intégration,  we  transformed  our  engineered  E.  coli  with  our  composite  part  BBa_I13500-BBa_I13507  created  for  the  promoter  caracterisation  under  the  BBa_J231107  promoter  and  then  we  quantified  the  production  of  GFP  and  RFP.
Results Verification  of  the  genomic  insertion  of  cytosolic  Skp
After  amplification  by  polymerisation  chain  reaction  of  RBS-SKP-Terminateur  sequence,  which  represent  a  fragment  of  622pB  we  realized  a  comparison  between  non  integrated  DH5α  cells  and  bacterial  cells  with  DNA  insertion  (at  42°C).  Here  on  these  electrophoresis  gel  we  can  see  that  there  is  a  band  with  a  very  high  level  of  intensity  and  some  SMIRS.  We  made  a  mistake  between  preparation  of  PCR  and  used  primers  10  times  more  concentrated  but  this  observation  confirm  the  fact  that  we  can  improve  the  level  of  SKP  production  directly  into  the  genome.
Characterization  of  the  SKP  insertion:
Promoter  strength  was  assessed  through  the  quantification  of  specific  fluorescence  from  GFP  (A)  and  RFP  (B)  normalized  with  OD600  to  obtain  relative  fluorescent  units  (RFU).      After  transformation  we  compared  the  level  of  GFP  and  RFP  expression  between  DH5α  with  and  without  SKP  integration.  Firstly,  we  can  find  with  the  same  condition  of  transformation  a  same  range  of  fluorescence  for  both  reporters  compared  to  the  characterisation  of  Anderson’s  promoters.  Here  we  can  see  that  the  level  of  GFP  expression  (around  20000U)  is  not  significantly  change  when  we  introduce  SKP  into  the  genome,  even  if  there  is  a  very  weak  increase  in  the  emission  level  of  GFP  and  RFP.  Our  supposition  is  that  GFP  is  a  protein  which  is  relatively  simple,  which  don’t  need  chaperon  proteins  or  an  high  level  of  post  traductional  modifications,  and  we  didn’t  had  the  time  to  realize  other  tests  with  more  complex  proteins
Part III - Determination of the best purification system
Introduction
The goal is to test and compare the Gluthation purification system and the hisTrap one to systems to see which one would be the most effective for our box.  To do so, we have purified GST-10XHIS proteins with both of the purification system. We have determined the GST enzymatic activity before and after the purifications to obtain the purification yields. Finally, we have done SDS-PAGE chromatographies to semi-quantify the proteins and compare the results with those obtained with the enzymatic activity.
Results
Figure.11: Growth curve of the DH5α containing the PET 21 GST-10HIS
All the points data come from a mean done by data collected from 5 cultures. Cultures were incubated in LB at37°C. The DO(600nm) was calculated every hour.
We cultured DH5α containing plasmid pet21 GST-10XHIS. We measured the OD every hour in order to follow the turbidity and thus determine the time corresponding to the exponential phase of growth(fig.11).
Analysis of GST enzymatic activity:
We calculated the GST activity of the sample taken before and after the purification. We took 5µL of the aliquotes to do the reaction. We then calculated the total reaction for the all volume before and after lysis that can be found in table.1.   The table.2 presents the the activities and purification yields obtained for both of the purification.
For the HisTrap column, we observe a purification yield equal to 72.1 %.
For the Gluthation Agarose Resin we observe a purification yield equal to 32%.
Fraction
Enzymatic activity From 5 µL (Δabs/min)
Total enzymatic activity (Δabs/min)
Purification Yield
F1 HisTrap column
0.13
208
72.1%
F2 HisTrap column
0.15
150
F1 Gluthation Agarose Resin
0.75
120
32%
F2 Gluthation Agarose Resin
0.77
38.5
Table.2 GST Enzymatic activity obtained and purification yield
Analysis of SDS-PAGE gel:
After bacterial lysis, we prepared aliquots from the solutions before and after purification and we performed SDS-PAGE for both types of purification. This is done in order to check the presence of GST-10XHIS and to compare these results with those obtained with the enzymatic analysis.
12% polyacrylamide gel. 10 µL deposited with Laemli 1X for each well.  M: Kb Ladder. 1:  eluted and purified proteins using a Gluthation-Agarose column. 2: expressed proteins before purification.
Figure.13: SDS-PAGE analyse of purification of GST-10his proteins using a HisTrap column
Before purification, mutliple bands can be seen corresponding to bacterial proteins. However, an intense band at about 27 kDa, which correspond to the GST-10His is observed and suggest a good expression of the protein(fig.12.2). After purification with Gluthation Agarose Resin, we can only see one band corresponding to the GST-10His but smaller than the one seen before purification(fig12.1).
SDS-PAGE of HisTrap column purification : Similarly to what was observed in the previous SDSPAGE, multiple bands and one more intense can be seen at about 27kDa corresponding to the GST-10His, before purification(fig13.2). After purification, only one the GST-10His band is observed at 27kDa. The band looks as intense as the one before purification(fig13.1).
Comparing the results with the SDS-PAGE and the GST enzymatic activity, we can say that the purification yields are consistent with the intensity of the bands observed with the SDS-PAGE. Indeed, with the Gluthation-Agarose column, we calculated a purification yield of 32% which is consistent with the diminution of the band size after purification. Likewise, the purification yield of 73% observed with the HisTrap column is consistent with the size of the band. However, the size of the band after HisTrap purification seems to be the as the one before purifcation so the purification yield should be closer to 100%. We have probably lost GST enzymatic activity during the experiments.
To conclude, the HisTrap column seems to be more efficient at purifing the GST-10His protein than the GST-Trap one. Yet, we should not forget that the HisTrap purification System require buffer containing imidazole. Imidazole is irritating and can cause allergies, so we will have to be even more careful when choosing our following Sephadex exclusion chromaztography to exclude small undesired composants. We should maybe try other types of column like the one capable of binding the SUMO protein for example.  For the box modelisation, we still decided to use the data obtained with the HisTrap column.
