"
Page
Discussion
View source
History
teams
Log in
 
Team:Grenoble/Notebook Rawhide
From 2011.igem.org
Grenoble 2011, Mercuro-Coli iGEM
Home
Le Projet
Introduction
Device
The Genetic Network
Modelling
Results
Biobricks
Human Practice
Overview
A communication problem
Developing a solution
Promoting synthetic biology
Collaboration
Safety
Notebook
June
July
August
September
October
About us
The Team
Grenoble
Data Page
Sponsors
July 28th to 3st
Team MarmottesMarion
According to the results from the sequencing order will have to start over constructions since the 1st-steps. A Purification step will be add for all 1st-step construction in order to  get round our resistance issues.
Cloning session :
-> 1st-step construction: RBS-CinI
-> 2nd-step constructions:
TetR-CinR
MerR-CinR
MerR-LuxR
LacI-LuxI
-> 3rd-step construction
pLac + TetR-LuxR
Digestions: Because we are having issues to get 2nd-step constructions. We tried both 3A Assembly and Standard protocols.
Cheking and purification gels: We add a cheking step, before performing ligations on Standard Protocol. To maximise our chance to get 2nd-step constructions, we tried to purified the inserts.
On purification gel, plasmids seem to be digested but inserts is missing. On checking gel, cinI wasn't digested correctly.
Sequencing Order
->RESULTS
RBS-TetR RBS-LuxI RBS-CinR RBS-LacI pLac-GFP pCin-GFP pMerT-GFP pLux-Lyco
6 out of 8 sequence alignments failed.
To perform this constructions, we used standard protocol: we kept the plasmid of the shortest biobrick which we inserted the longest biobrick.
The problem was identified as coming from a wrong identification of antibiotic resistants. In most constructions below, vector and inserts had the same antibiotic resistance. So, even wrong constructions were selected on Petri Dishes.
Enzyme checking
Because we had no results with the latest cloning session, we tested our enzymes.
We tried to digest each site individually. We obtained linearized plasmid. So, enzymes are well working.
Cloning session:
-> Biobricks which require an RBS and have the same antibiotic resistance as RBS plasmid, were transfered on pSB1AC3 plasmid with chloramphenicol resistance:
TetR
LuxR
LuxI
LacI
GFP
Lycopen
-> Fha1 was also transfered on pSB1AT3 plasmid to simplify selection of the right constructions.
Digestions: a gel checking of the restriction results were performed.
Spreading over Petri dish
On the Restriction Gel, We obtain the right size for all inserts.
Because pSB1AC3 is a double resistance plasmid (ampicillin and chloramphenicol) and RBS plasmid is on ampicillin, as previously the right constructions won't be selected.
Cloning session:
-> 3rd-step construction:
pLac + MerR-CinR
pConst + MerR-CinR
-> 2nd-step construction:
RBS-CinI
Preculture: To perform this new cloning session, we first selected 5 colonies from the Petri dish of MerR-CinR construction.
Digestions: a gel checking of the restriction results were performed.
Purification: In order to check and extract the wright insert.
Ligations : Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids.
Spreading over Petri dish
Cloning Session:
-> 1st-step construction START OVER:
RBS-LacI        RBS-TetR        RBS-LuxI        Fha-LacI        Fha-LuxI        Fha-LuxR        Fha-CinI        Fha-CinR        Fha-TetR        pLux-Lycopene        pCin-Lycopene
-> Construction Test
pCin-GFP        pLux-GFP        pTet-GFP        pConst-GFP        pLac-GFP        pConst-RBS-CinR        pConst-RBS-LuxR
Digestions: a gel checking of the restriction results were performed.
Purification: In order to check and extract the wright insert.
Ligations : Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids.
Spreading over Petri dish
On Puritication gel, 4 biobricks were not correctly digested: CinI, LuxI, LuxR, RBS-LuxR. All the other were apparently well purified.
Ligations were performed as usual.
Results on Petri Dishes were unsatifying: we obtained an agglomerate of colonies. And this in all Petri Dishes. Does the issue come from too old Petri Dishes? Or LB pre culture?
So, we tried a 2nd-spreding on Petri Dishes and the result was the same.
We also checked by PCR if we could work anyway with those colonies. But the results were not conclusive.
New stock on Petri Dishes
We spread again -80°C stock in order to start new miniprep of the troubling biobricks:
Lycopene
RBS
Fha
TetR
LuxR
LuxI
CinI
CinR
pTet
pMerT
pCin
pConst
pLux
Cloning Session:
We started over constructions which couldn't be performed during the previous cloning because of wrong digestion.
Fha-LuxR
Fha-LuxI
Fha-CinR
Fha-CinI
pCin-Lycopene
pLux-Lycopene
pConst-RBS-LuxR
RBS-LuxI
RBS-LuxR
Digestions: a gel checking of the restriction results were performed.
Purification: In order to check and extract the wright insert.
Ligations : Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids. We add a control for the ligations: plasmids without its inserts (pLux, pCin, Fha, RBS).
Spreading over Petri dish
Team GodlikeGeoffrey
It's all about patience
Models
Stochastic program is implemented for the toggle switch and works fine. We adapted the program for the whole system so we can implement the quorum sensing stochastic model very quickly when we have the parameters (about 1/2 day of work)
Parameters
We have coherent sets of parameters, but we still can't compare our models to the real system and are waiting for the results of characterization experiments.
July 22nd to 27th
Stock of electro competent cells
Team GodlikeJB
Stochastic model done
Models
Stochastic program is implemented for the toggle switch and works fine. We adapted the program for the whole system so we can implement the quorum sensing stochastic model very quickly when we have the parameters (about 1/2 day of work). The current model and parameters prove that we do have a bi-modal distribution when IPTG and pollutant concentrations are equal :
On the X-axis we can see the concentration of lacI (neg) or merR (positive) in cells. On the Y-axis we can see the amount of cells in one way or another.
Parameters
We have coherent sets of parameters, but we still can't compare our models to the real system and are waiting for the results of characterization experiments.
Team MarmottesMarion
Moving in to the CIME, a biology teaching platform, free for the summer vacation. Preparation of new storage. PCR results from last week cloning.
Cloning session :
-> every 1st-step constructions that failed last time, so 6 constructions.
-> 2 2nd-step constructions:
MerR-LuxR
TetR-LuxR
Ligations:
Because we still have a low cloning rate we tried different proportions between vectors and inserts:
1 vector/1 insert
1 vector/2 inserts
1 vector/3 inserts
Transformations with electrocompetent cells
Spreading over Petri Dish
RESULTS :
Best results on Petri Dishes with the proportion 1/3. All constructions seem to have the right size except MerR-LuxR.
Sequencing Order
RBS-TetR RBS-LuxI RBS-CinR RBS-LacI pLac-GFP pCin-GFP pMerT-GFP pLux-Lyco
Products Order
For ligations, digestions, PCR, MiniPreps, MidiPreps
Cloning Session :
-> 5 constructions with Fha1
-> 3 2nd-step constructions
LacI-LuxI
MerR-CinR
TetR-CinR
-> 1 construction for the tests
pLux-GFP
-> 1st-step construction
RBS-CinI
The final RBS (Fha1) which allows to keep the toggle off has just been delivered. So, all first steps constructions including RBS were made once again.
Because the efficiency of Fha1 hasn't been completly demonstrated, we keep on cloning with the standard RBS. So, RBS and Fha1 constructions are made in parallel.
RESULTS :
Every constructions gave colonies on Petri Dishes.
But PCR cheking showed that nothing were amplified on every Fha1 constructions with VF2 and VR primers. Thus, Fha1 was provided in a PCR blent plasmid instead of an iGEM's plasmid.
The insert of the test construction is having the right size.
2 out of 3 2nd step constructions had the right size.
We still have issues to clone the 1st step construction: RBS-CinI.
July 14th to 21st
Team MarmottesMorgane
Better clonings rate: Clonings performed with recommended enzymes by iGEM.
Stock of electro competent cells
RESULTS : Great Transformation Rate
New stock of Biobrick Miniprep
DO checking PCR cheking
Cloning of every first steps of our construction: (Standard Assembly Method)
-> 11 clonings
We changed our strategy: clonings were performed with enzymes recommended by iGEM (New England Biolabs).
One Petri Dish was free of colony. We performed PCRs on 5 colonies from each Petri Dish. According to PCR results, 5 constructions out of 11 seem to have the right size.
Cloning session :
-> every 1st-step constructions that failed last time, so 6 constructions.
-> 2 2nd-step constructions:
MerR-LuxR
TetR-LuxR
Only digestions were performed because of a lack of equipments.
We decided to use the Standard Protocol for the first step constructions because the assembly combinates a short and a long gene. The long gene is inserted in the plasmid of the small gene.
But the Second step constructions were performed with the 3A Assembly Protocol.
Team GodlikeMaxime
Stochastic model done
Models
First results with the stochastic model. Our whole model does work like we expected it. But parameters are needed more than ever now.
Parameters
We got incomplete sets of parameters from either litterature or iGem teams. Some very useful sets can be found in Aberdeen 09, Brown 10, ETHZ 07, BCCS Bristol 08
July 7th to 13th
Team MarmottesMarion
Moving in to the CIME, a biology teaching platform, free for the summer vacation. Preparation of new storage. PCR results from last week cloning.
Moving in of the whole team to a new lab
We are taking advantage that the school year is over to move in the CIME, a teaching platform of biology.
Cheking of DNA concentration into our Biobrick Miniprep
OD cheking was performed on all our Biobrick stock to have an idea of the DNA rate into our preparation. We also made a PCR cheking to make sure that the DNA rate really corresponds to our Biobrick.
We got an average rate of 100 ng/µl. Which is a much lower than expect, so 10 times more DNA products would had been necessary to get great constructions.
Order of new enzymes -> New England Biolabs Company
We hope to increase the digestion efficiency.
PCR Checking of cloning from the last week
As previously, inserts were shorter than expected, except the construction RBS-TetR.
Making new storage
Petri Dishes with Antibiotic (Amp, Kan, Cm, Tet). LB Culture medium
RBS-TetR Miniprep
In order to send it to the sequencing.
Team GodlikeGeoffrey
Learning Gillespie and looking for parameters.
Models
Stochastic model is on its way. We base it on Gillespie's algorithm. It's mainly about getting familiar with the math for now. Also working on Hysteresis and isoclines subparts of the deterministic models.
Parameters
Still a lot of work going on there. The problem is we can not compare our sets to experimental results as the whole system is not finished yet. We will then try to characterize some of the promoters we use and figure out how to obtain some degradation rates as well
Retrieved from " http://2011.igem.org/Team:Grenoble/Notebook_Rawhide "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
