<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grenoble_Notebook_July"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grenoble/Notebook/July">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grenoble/Notebook/July&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Notebook/July&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Notebook/July&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grenoble/Notebook/July" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grenoble/Notebook/July</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Grenoble 2011, Mercuro-Coli iGEM</TITLE></P><DIV class="left"><DIV class="navigation"><DIV class="centering"><UL class="topnav"><UL><LI><A href="/Team:Grenoble">Home</A></LI><LI><A href="/Team:Grenoble/Projet/Intro"><SPAN class="scriptina"><I>Le Projet</I></SPAN></A></LI><LI><A href="/Team:Grenoble/Projet/Intro">Introduction</A></LI><LI><A href="/Team:Grenoble/Projet/Device">Device</A></LI><LI><A href="/Team:Grenoble/Projet/Design">The Genetic Network</A></LI><LI><A href="/Team:Grenoble/Projet/Modelling">Modelling</A></LI><LI><A href="/Team:Grenoble/Projet/Results">Results</A></LI><LI><A href="/Team:Grenoble/Projet/Biobricks">Biobricks</A></LI></UL><LI><A href="/Team:Grenoble/HumanPractice">Human Practice</A></LI><LI><A href="/Team:Grenoble/HumanPractice">Overview</A></LI><LI><A href="/Team:Grenoble/HumanPractice/identifying">A communication problem</A></LI><LI><A href="/Team:Grenoble/HumanPractice/developing">Developing a solution</A></LI><LI><A href="/Team:Grenoble/HumanPractice/promoting">Promoting synthetic biology</A></LI><LI><A href="/Team:Grenoble/HumanPractice/collaboration">Collaboration</A></LI></UL><LI><A href="/Team:Grenoble/Safety">Safety</A></LI><LI><A href="/Team:Grenoble/Notebook/June">Notebook</A></LI><LI><A href="/Team:Grenoble/Notebook/June">June</A></LI><LI><A href="/Team:Grenoble/Notebook/July">July</A></LI><LI><A href="/Team:Grenoble/Notebook/August">August</A></LI><LI><A href="/Team:Grenoble/Notebook/September">September</A></LI><LI><A href="/Team:Grenoble/Notebook/October">October</A></LI><LI><A href="/Team:Grenoble/The_Team/">About us</A></LI><LI><A href="/Team:Grenoble/The_Team/">The Team</A></LI><LI><A href="/Team:Grenoble/Grenoble">Grenoble</A></LI><LI><A href="/Team:Grenoble/Data_Page">Data Page</A></LI><LI><A href="/Team:Grenoble/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV class="body"><DIV class="left"><DIV class="blocbackground"><H1 id="week1">July 7<SUP>th</SUP> to 13<SUP>th</SUP></H1></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					Moving in to the CIME, a biology teaching platform, free for the summer vacation. Preparation of new storage. PCR results from last week cloning.
				</P><UL><LI>Moving in of the whole team to a new lab</LI><P>
						We are taking advantage that the school year is over to move in the CIME, a teaching platform of biology.
					</P><LI>Cheking of DNA concentration into our Biobrick Miniprep :</LI><P>
						OD cheking was performed on all our Biobrick stock to have an idea of the DNA rate into our preparation. We also made a PCR cheking to make sure that the DNA rate really corresponds to our Biobrick. We got an average rate of 100 ng/µl. Which is a much lower than expect, so 10 times more DNA products would had been necessary to get great constructions.
					</P><LI>Order of new enzymes from New England Biolabs Company</LI><P>We hope to increase the digestion efficiency.</P><LI>PCR Checking of cloning from the last week</LI><P>As previously, inserts were shorter than expected, except the construction RBS-TetR.</P><LI>Making new storage</LI><P>Petri Dishes with Antibiotic (Amp, Kan, Cm, Tet). LB Culture medium</P><LI>RBS-TetR Miniprep</LI><P>In order to send it to the sequencing.</P></UL></DIV></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				Miniprep of pTOPO clones to check for the presence of insert. Digestion with EcoRI and PstI. One clone is sent to sequencing for : fhaSL ; rsmA protein coding sequence, rsmY regulatory RNA. Adaptation of a new transformation protocol (by electroporation), given by some other members of our group.
			</P><P>
				Second try of inserting our sequences into an Igem plasmid from PCR products, and first 3A ligation: 
			</P><UL><LI>fhaLS + gfp on PSB1C3</LI><LI>rposLS + gfp on PSB1C3</LI></UL></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>Learning Gillespie and looking for parameters.</P></DIV><UL><LI>Models :</LI><P>
					Stochastic model is on its way. We base it on Gillespie's algorithm. It's mainly about getting familiar with the math for now. Also working on Hysteresis and isoclines subparts of the deterministic models.
				</P><LI>Parameters :</LI><P>
					Still a lot of work going on there. The problem is we can not compare our sets to experimental results as the whole system is not finished yet. We will then try to characterize some of the promoters we use and figure out how to obtain some degradation rates as well.
				</P></UL></DIV><DIV class="blocbackground"><H1 id="week2">July 14<SUP>th</SUP> to 21<SUP>st</SUP></H1></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>Better clonings rate: Clonings performed with recommended enzymes by iGEM.</P><UL><LI>Stock of electro competent cells :</LI><UL><LI>Results :</LI><P>Great transformation rate</P></UL><LI>New stock of Biobrick Miniprep :</LI><P>DO checking, PCR cheking</P><LI>Cloning (Standard Assembly Method) of every first steps of our construction :</LI><UL><LI>11 clonings</LI><P>
							We changed our strategy: clonings were performed with enzymes recommended by iGEM ().
						</P><P>
							One Petri Dish was free of colony. We performed PCRs on 5 colonies from each Petri Dish. According to PCR results, 5 constructions out of 11 seem to have the right size.
						</P></UL><LI>Cloning session :</LI><UL><LI>Every 1<SUP>st</SUP>-step constructions that failed last time, so 6 constructions.</LI><LI>2 2<SUP>nd</SUP>-step constructions :</LI><UL><LI>MerR-LuxR</LI><LI>TetR-LuxR</LI></UL></UL><P>
						Only digestions were performed because of a lack of equipments. We decided to use the Standard Protocol for the first step constructions because the assembly combinates a short and a long gene. The long gene is inserted in the plasmid of the small gene. But the Second step constructions were performed with the 3A Assembly Protocol.
					</P></UL></DIV></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>Stochastic model done.</P></DIV><UL><LI>Models :</LI><P>
					First results with the stochastic model. Our whole model does work like we expected it. But parameters are needed more than ever now.
				</P><LI>Parameters :</LI><P>
					We got incomplete sets of parameters from either litterature or iGem teams. Some very useful sets can be found in Aberdeen 09, Brown 10, ETHZ 07, BCCS Bristol 08.
				</P></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				Production of Bba_J23119, a constitutive promotor needed in our construction : transformation of cells, selection of some clones, digestion, migration on accrylamid (better vizualisation of small fragment) agarose gel to check for the presence of insert.
			</P><P>
				Second try to transform ligation from 28th June in Top10 super competent cells. Constructs were fha1SL-I13401, rposSL-I13401, fhaLS, rpoSLS, rsmA and rsmY ligated into pSB1C3. A first screen was performed on white colonies by colony PCR. Plasmid purification from positive clones and screen by EcoR1 and Pst1 restriction digest was done.
			</P></DIV><DIV class="blocbackground"><H1 id="week3">July 22<SUP>nd</SUP> to 27<SUP>th</SUP></H1></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>Stochastic model done.</P></DIV><UL><LI>Models :</LI><P>
					Stochastic program is implemented for the toggle switch and works fine. We adapted the program for the whole system so we can implement the quorum sensing stochastic model very quickly when we have the parameters (about 1/2 day of work). The current model and parameters prove that we do have a bi-modal distribution when IPTG and pollutant concentrations are equal :
				</P><DIV class="noindent"><P>
						On the X-axis we can see the concentration of lacI (neg) or merR (positive) in cells. On the Y-axis we can see the amount of cells in one way or another.
					</P></DIV><LI>Parameters :</LI><P>
					We have coherent sets of parameters, but we still can't compare our models to the real system and are waiting for the results of characterization experiments.
				</P></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				The first sequencing results are received: fha1LS and rsmY on pTOPO are OK. The rsmA coding sequence has a deletion. Realization of glycerol stock for the positives clones. Amplification with high fidelity Taq polymerase of rsmA and rpoSLS. Digestion and ligation of PCR products with I13401 (gfp) in PSB1C3 linearized plasmid. Three A assembly for :
			</P><UL><LI>rpoSLS- I1340- pSB1C31</LI><LI>fhaLS-I13401-pSB1C3</LI></UL><P>
				When screening transformants by PCR using VF and VR, DNA fragments appear to have the wrong size. Plasmid purification from positive clones and screen by EcoR1 and Pst1 restriction digests was done. 10 clones for construct 15 and nine for construct 14 were selected to check if leader sequences are upstream the I13401 (Leader sequences are too small to be seen on agarose gel.)
			</P><P>
				Flow cytometry comparison of the biobrick parts to our constructions:
			</P><UL><LI>Bba_K256003 (Pcons-rbs-gfp)</LI><LI>Bba_K176005 (Pcons-rbs-gfp) the twin of Bba_K256003</LI><LI>Bba_E0840 (rbs-gfp), negative control</LI><LI>Bba_13401 (gfp), negative control</LI></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					Moving in to the CIME, a biology teaching platform, free for the summer vacation. Preparation of new storage. PCR results from last week cloning.
				</P><UL><LI>Cloning session :</LI><UL><LI>every 1<SUP>st</SUP>-step constructions that failed last time, so 6 constructions.</LI><LI>2, 2<SUP>nd</SUP>-step constructions :</LI><UL><LI>MerR-LuxR</LI><LI>TetR-LuxR</LI></UL><LI>Ligations :</LI><P>Because we still have a low cloning rate we tried different proportions between vectors and inserts:</P><UL><LI>1 vector/1 insert</LI><LI>1 vector/2 inserts</LI><LI>1 vector/3 inserts</LI></UL><LI>Transformations with electrocompetent cells</LI><LI>Spreading over Petri Dish</LI><UL><LI>Results :</LI><P>Best results on Petri Dishes with the proportion 1/3. All constructions seem to have the right size except MerR-LuxR.</P></UL></UL><LI>Sequencing Order</LI><P>RBS-TetR RBS-LuxI RBS-CinR RBS-LacI pLac-GFP pCin-GFP pMerT-GFP pLux-Lyco</P><LI>Products Order</LI><P>For ligations, digestions, PCR, MiniPreps, MidiPreps</P><LI>Cloning Session :</LI><UL><LI>5 constructions with Fha1</LI><LI>3, 2<SUP>nd</SUP>-step constructions</LI><UL><LI>LacI-LuxI</LI><LI>MerR-CinR</LI><LI>TetR-CinR</LI></UL><LI>1 construction for the tests</LI><UL><LI>pLux-GFP</LI></UL><LI>1st-step construction</LI><UL><LI>RBS-CinI</LI></UL></UL><P>
						The final RBS (Fha1) which allows to keep the toggle off has just been delivered. So, all first steps constructions including RBS were made once again. Because the efficiency of Fha1 hasn't been completly demonstrated, we keep on cloning with the standard RBS. So, RBS and Fha1 constructions are made in parallel.
					</P><UL><LI>Results :</LI><P>
							Every constructions gave colonies on Petri Dishes. But PCR cheking showed that nothing were amplified on every Fha1 constructions with VF2 and VR primers. Thus, Fha1 was provided in a PCR blent plasmid instead of an iGEM's plasmid. The insert of the test construction is having the right size. 2 out of 3 2nd step constructions had the right size. We still have issues to clone the 1st step construction: RBS-CinI.
						</P></UL></UL></DIV></DIV><DIV class="blocbackground"><H1 id="week4">July 28<SUP>th</SUP> to 3<SUP>rd</SUP></H1></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				Constructs 14 and 15 are digested by Nco1 (cut into rpoSLS) and Ara1 (cut into fhaLS) in order to check for the presence of leader sequences. Negative results.
			</P><P>
				New PCR of rsmA, ligation in pTOPO and transformation. Screen of many colonies by PCR and mini prep. Test of different minipreping methods. The primers used for the screen are the same ones as the ones initially used to produce the amplicon. Incoherent results are given by the two methods. We deduce that the use of same primer set for amplify a sequence is not reliable for screening this same sequence on a clone in this case. Indeed, it is very likely to gives false positives.
			</P></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					According to the results from the sequencing order will have to start over constructions since the 1st-steps. A Purification step will be add for all 1st-step construction in order to  get round our resistance issues.
				</P><UL><LI>Cloning session :</LI><LI>1<SUP>st</SUP>-step construction: RBS-CinI</LI><LI>2<SUP>nd</SUP>-step constructions:</LI><UL><LI>TetR-CinR</LI><LI>MerR-CinR</LI><LI>MerR-LuxR</LI><LI>LacI-LuxI</LI></UL><LI>3<SUP>rd</SUP>-step construction</LI><UL><LI>pLac + TetR-LuxR</LI></UL></UL><P>
						Digestions: Because we are having issues to get 2<SUP>nd</SUP>-step constructions. We tried both 3A Assembly and Standard protocols.
					</P><P>
						Cheking and purification gels: We add a cheking step, before performing ligations on Standard Protocol. To maximise our chance to get 2<SUP>nd</SUP>-step constructions, we tried to purified the inserts. On purification gel, plasmids seem to be digested but inserts is missing. On checking gel, cinI wasn't digested correctly.
					</P><LI>Sequencing Order :</LI><UL><LI>Results :</LI><P>
							RBS-TetR RBS-LuxI RBS-CinR RBS-LacI pLac-GFP pCin-GFP pMerT-GFP pLux-Lyco
						</P><P>
							6 out of 8 sequence alignments failed.
						</P><P>
							To perform this constructions, we used standard protocol: we kept the plasmid of the shortest biobrick which we inserted the longest biobrick. The problem was identified as coming from a wrong identification of antibiotic resistants. In most constructions below, vector and inserts had the same antibiotic resistance. So, even wrong constructions were selected on Petri Dishes.
						</P></UL><LI>Enzyme checking :</LI><P>
						Because we had no results with the latest cloning session, we tested our enzymes. We tried to digest each site individually. We obtained linearized plasmid. So, enzymes are well working.
					</P><LI>Cloning session :</LI><UL><LI>Biobricks which require an RBS and have the same antibiotic resistance as RBS plasmid, were transfered on pSB1AC3 plasmid with chloramphenicol resistance :</LI><UL><LI>TetR</LI><LI>LuxR</LI><LI>LuxI</LI><LI>LacI</LI><LI>GFP</LI><LI>Lycopen</LI></UL><LI>Fha1 was also transfered on pSB1AT3 plasmid to simplify selection of the right constructions.</LI><LI>Digestions :</LI><P>A gel checking of the restriction results were performed.</P><P>Spreading over Petri dish</P><P>
							On the Restriction Gel, We obtain the right size for all inserts. Because pSB1AC3 is a double resistance plasmid (ampicillin and chloramphenicol) and RBS plasmid is on ampicillin, as previously the right constructions won't be selected.
						</P></UL><LI>Cloning session :</LI><UL><LI>3<SUP>rd</SUP>-step construction :</LI><UL><LI>pLac + MerR-CinR</LI><LI>pConst + MerR-CinR</LI></UL><LI>2<SUP>nd</SUP>-step construction:</LI><UL><LI>RBS-CinI</LI></UL><LI>Preculture :</LI><P>
							To perform this new cloning session, we first selected 5 colonies from the Petri dish of MerR-CinR construction.
						</P><LI>Digestions :</LI><P> a gel checking of the restriction results were performed.</P><LI>Purification :</LI><P>In order to check and extract the wright insert.</P><LI>Ligations :</LI><P>Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids.</P><LI>Spreading over Petri dish</LI></UL><LI>Cloning Session :</LI><UL><LI>1<SUP>st</SUP>-step construction START OVER :</LI><UL><LI>RBS-LacI</LI><LI>RBS-TetR</LI><LI>RBS-LuxI</LI><LI>Fha-LacI</LI><LI>Fha-LuxI</LI><LI>Fha-LuxR</LI><LI>Fha-CinI</LI><LI>Fha-CinR</LI><LI>Fha-TetR</LI><LI>pLux-Lycopene</LI><LI>pCin-Lycopene</LI></UL><LI>Construction Test :</LI><UL><LI>pCin-GFP</LI><LI>pLux-GFP</LI><LI>pTet-GFP</LI><LI>pConst-GFP</LI><LI>pLac-GFP</LI><LI>pConst-RBS-CinR</LI><LI>pConst-RBS-LuxR</LI></UL><LI>Digestions :</LI><P>A gel checking of the restriction results were performed.</P><LI>Purification :</LI><P>In order to check and extract the wright insert.</P><LI>Ligations :</LI><P>Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids.</P><LI>Spreading over Petri dish</LI></UL><P>
						On Puritication gel, 4 biobricks were not correctly digested: CinI, LuxI, LuxR, RBS-LuxR. All the other were apparently well purified. Ligations were performed as usual.
					</P><P>
						Results on Petri Dishes were unsatifying: we obtained an agglomerate of colonies. And this in all Petri Dishes. Does the issue come from too old Petri Dishes? Or LB pre culture ? So, we tried a 2nd-spreding on Petri Dishes and the result was the same. We also checked by PCR if we could work anyway with those colonies. But the results were not conclusive.
					</P><LI>New stock on Petri Dishes :</LI><UL><LI>We spread again -80°C stock in order to start new miniprep of the troubling biobricks :</LI><UL><LI>Lycopene</LI><LI>RBS</LI><LI>Fha</LI><LI>TetR</LI><LI>LuxR</LI><LI>LuxI</LI><LI>CinI</LI><LI>CinR</LI><LI>pTet</LI><LI>pMerT</LI><LI>pCin</LI><LI>pConst</LI><LI>pLux</LI></UL></UL><LI>Cloning Session :</LI><UL><LI> We started over constructions that couldn't be performed during the previous cloning because of wrong digestion :</LI><UL><LI>Fha-LuxR</LI><LI>Fha-LuxI</LI><LI>Fha-CinR</LI><LI>Fha-CinI</LI><LI>pCin-Lycopene</LI><LI>pLux-Lycopene</LI><LI>pConst-RBS-LuxR</LI><LI>RBS-LuxI</LI><LI>RBS-LuxR</LI></UL><LI>Digestions :</LI><P>A gel checking of the restriction results were performed.</P><LI>Purification :</LI><P>In order to check and extract the wright insert.</P><LI>Ligations :</LI><P>
							Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids. We add a control for the ligations: plasmids without its inserts (pLux, pCin, Fha, RBS).
						</P><LI>Spreading over Petri dish</LI></UL></DIV></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>It's all about patience</P></DIV><UL><LI>Models</LI><P>
					Stochastic program is implemented for the toggle switch and works fine. We adapted the program for the whole system so we can implement the quorum sensing stochastic model very quickly when we have the parameters (about 1/2 day of work).
				</P><LI>Parameters</LI><P>
					We have coherent sets of parameters, but we still can't compare our models to the real system and are waiting for the results of characterization experiments.
				</P></UL></DIV></DIV></DIV><DIV class="body"><DIV class="left"><DIV class="blocbackground"><TABLE class="nobordure"><TBODY><TR><TD><A href="https://2011.igem.org/Main_Page" title="iGEM 2011 Main Page">iGEM 2011 Main Page</A><H3><SPAN class="vert">Contact</SPAN> Us:</H3><H3><A title="Geoffrey BOUCHAGE" href="mailto:geoffrey.bouchage@phelma.grenoble-inp.fr"> Clic here !</A></H3></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Grenoble/Notebook/July">http://2011.igem.org/Team:Grenoble/Notebook/July</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grenoble/Notebook/July" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grenoble/Notebook/July" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grenoble/Notebook/July&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grenoble/Notebook/July&amp;oldid=257712" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>