<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grenoble_Notebook_Rawhide"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grenoble/Notebook_Rawhide">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grenoble/Notebook_Rawhide&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Notebook_Rawhide&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Notebook_Rawhide&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grenoble/Notebook_Rawhide" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grenoble/Notebook Rawhide</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Grenoble 2011, Mercuro-Coli iGEM</TITLE></P><DIV class="left"><DIV class="navigation"><DIV class="centering"><UL class="topnav"><UL><LI><A href="/Team:Grenoble">Home</A></LI><LI><A href="/Team:Grenoble/Projet/Intro"><SPAN class="scriptina"><I>Le Projet</I></SPAN></A></LI><LI><A href="/Team:Grenoble/Projet/Intro">Introduction</A></LI><LI><A href="/Team:Grenoble/Projet/Device">Device</A></LI><LI><A href="/Team:Grenoble/Projet/Design">The Genetic Network</A></LI><LI><A href="/Team:Grenoble/Projet/Modelling">Modelling</A></LI><LI><A href="/Team:Grenoble/Projet/Results">Results</A></LI><LI><A href="/Team:Grenoble/Projet/Biobricks">Biobricks</A></LI></UL><LI><A href="/Team:Grenoble/HumanPractice">Human Practice</A></LI><LI><A href="/Team:Grenoble/HumanPractice">Overview</A></LI><LI><A href="/Team:Grenoble/HumanPractice/identifying">A communication problem</A></LI><LI><A href="/Team:Grenoble/HumanPractice/developing">Developing a solution</A></LI><LI><A href="/Team:Grenoble/HumanPractice/promoting">Promoting synthetic biology</A></LI><LI><A href="/Team:Grenoble/HumanPractice/collaboration">Collaboration</A></LI></UL><LI><A href="/Team:Grenoble/Safety">Safety</A></LI><LI><A href="/Team:Grenoble/Notebook/June">Notebook</A></LI><LI><A href="/Team:Grenoble/Notebook/June">June</A></LI><LI><A href="/Team:Grenoble/Notebook/July">July</A></LI><LI><A href="/Team:Grenoble/Notebook/August">August</A></LI><LI><A href="/Team:Grenoble/Notebook/September">September</A></LI><LI><A href="/Team:Grenoble/Notebook/October">October</A></LI><LI><A href="/Team:Grenoble/The_Team/">About us</A></LI><LI><A href="/Team:Grenoble/The_Team/">The Team</A></LI><LI><A href="/Team:Grenoble/Grenoble">Grenoble</A></LI><LI><A href="/Team:Grenoble/Data_Page">Data Page</A></LI><LI><A href="/Team:Grenoble/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV class="body"><DIV class="left"><H1>July 28<SUP>th</SUP> to 3<SUP>st</SUP></H1><DIV class="blocbackground"><H2>Team Marmottes<SPAN>Marion</SPAN></H2><P>According to the results from the sequencing order will have to start over constructions since the 1st-steps. A Purification step will be add for all 1st-step construction in order to  get round our resistance issues.</P><UL><LI>Cloning session :
	    -&gt; 1st-step construction: RBS-CinI
	     -&gt; 2nd-step constructions:
	        TetR-CinR
	        MerR-CinR
	        MerR-LuxR
	        LacI-LuxI
	     -&gt; 3rd-step construction
	        pLac + TetR-LuxR</LI><P>Digestions: Because we are having issues to get 2nd-step constructions. We tried both 3A Assembly and Standard protocols. </P><P>Cheking and purification gels: We add a cheking step, before performing ligations on Standard Protocol. To maximise our chance to get 2nd-step constructions, we tried to purified the inserts.</P><P>On purification gel, plasmids seem to be digested but inserts is missing. On checking gel, cinI wasn't digested correctly. </P><LI>Sequencing Order
	    -&gt;RESULTS</LI><P>RBS-TetR RBS-LuxI RBS-CinR RBS-LacI pLac-GFP pCin-GFP pMerT-GFP pLux-Lyco</P><P>6 out of 8 sequence alignments failed.
	   To perform this constructions, we used standard protocol: we kept the plasmid of the shortest biobrick which we inserted the longest biobrick. 
	   The problem was identified as coming from a wrong identification of antibiotic resistants. In most constructions below, vector and inserts had the same antibiotic resistance. So, even wrong constructions were selected on Petri Dishes.</P><LI>Enzyme checking</LI><P>Because we had no results with the latest cloning session, we tested our enzymes.</P><P>We tried to digest each site individually. We obtained linearized plasmid. So, enzymes are well working.</P><LI>Cloning session:
	    -&gt; Biobricks which require an RBS and have the same antibiotic resistance as RBS plasmid, were transfered on pSB1AC3 plasmid with chloramphenicol resistance:
	    TetR
	    LuxR
	    LuxI
	    LacI
	    GFP
	    Lycopen
	      -&gt; Fha1 was also transfered on pSB1AT3 plasmid to simplify selection of the right constructions.</LI><P>Digestions: a gel checking of the restriction results were performed.</P><P>Spreading over Petri dish</P><P>On the Restriction Gel, We obtain the right size for all inserts.
	    Because pSB1AC3 is a double resistance plasmid (ampicillin and chloramphenicol) and RBS plasmid is on ampicillin, as previously the right constructions won't be selected.</P><LI>Cloning session:
	      -&gt; 3rd-step construction:
	      pLac + MerR-CinR
	      pConst + MerR-CinR
	      -&gt; 2nd-step construction:
	      RBS-CinI</LI><P>Preculture: To perform this new cloning session, we first selected 5 colonies from the Petri dish of MerR-CinR construction.</P><P>Digestions: a gel checking of the restriction results were performed.</P><P>Purification: In order to check and extract the wright insert.</P><P>Ligations : Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids.</P><P>Spreading over Petri dish</P><LI>Cloning Session:
	      -&gt; 1st-step construction START OVER:
	      RBS-LacI
	      RBS-TetR
	      RBS-LuxI
	      Fha-LacI
	      Fha-LuxI
	      Fha-LuxR
	      Fha-CinI
	      Fha-CinR
	      Fha-TetR
	      pLux-Lycopene
	      pCin-Lycopene
	      
  	      -&gt; Construction Test
	      pCin-GFP
	      pLux-GFP
	      pTet-GFP
	      pConst-GFP
	      pLac-GFP
	      pConst-RBS-CinR
	      pConst-RBS-LuxR</LI><P>Digestions: a gel checking of the restriction results were performed.</P><P>Purification: In order to check and extract the wright insert.</P><P>Ligations : Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids.</P><P>Spreading over Petri dish</P><P>On Puritication gel, 4 biobricks were not correctly digested: CinI, LuxI, LuxR, RBS-LuxR. All the other were apparently well purified.
	      Ligations were performed as usual.
	      Results on Petri Dishes were unsatifying: we obtained an agglomerate of colonies. And this in all Petri Dishes. Does the issue come from too old Petri Dishes? Or LB pre culture?
	      So, we tried a 2nd-spreding on Petri Dishes and the result was the same.
	      We also checked by PCR if we could work anyway with those colonies. But the results were not conclusive.</P><LI>New stock on Petri Dishes</LI><P>We spread again -80°C stock in order to start new miniprep of the troubling biobricks:
		  Lycopene
		  RBS
		  Fha
		  TetR
		  LuxR
		  LuxI
		  CinI
		  CinR
		  pTet
		  pMerT
		  pCin
		  pConst
		  pLux</P><LI>Cloning Session:
		  We started over constructions which couldn't be performed during the previous cloning because of wrong digestion.
		  Fha-LuxR
		  Fha-LuxI
		  Fha-CinR
		  Fha-CinI
		  pCin-Lycopene
		  pLux-Lycopene
		  pConst-RBS-LuxR
		  RBS-LuxI
		  RBS-LuxR</LI><P>Digestions: a gel checking of the restriction results were performed.</P><P>Purification: In order to check and extract the wright insert.</P><P>Ligations : Standard Protocol was performed. MerR-CinR was inserted into pLac and pConst plasmids. We add a control for the ligations: plasmids without its inserts (pLux, pCin, Fha, RBS).</P><P>Spreading over Petri dish</P></UL></DIV><DIV class="blocbackground"><H2>Team Godlike<SPAN>Geoffrey</SPAN></H2><P>It's all about patience</P><UL><LI>Models</LI><P>Stochastic program is implemented for the toggle switch and works fine. We adapted the program for the whole system so we can implement the quorum sensing stochastic model very quickly when we have the parameters (about 1/2 day of work)</P><LI>Parameters</LI><P>We have coherent sets of parameters, but we still can't compare our models to the real system and are waiting for the results of characterization experiments.</P></UL></DIV><H1>July 22<SUP>nd</SUP> to 27<SUP>th</SUP></H1>

Stock of electro competent cells 

  <DIV class="blocbackground"><H2>Team Godlike<SPAN>JB</SPAN></H2><P>Stochastic model done</P><UL><LI>Models</LI><P>Stochastic program is implemented for the toggle switch and works fine. We adapted the program for the whole system so we can implement the quorum sensing stochastic model very quickly when we have the parameters (about 1/2 day of work). The current model and parameters prove that we do have a bi-modal distribution when IPTG and pollutant concentrations are equal :</P><P>On the X-axis we can see the concentration of lacI (neg) or merR (positive) in cells. On the Y-axis we can see the amount of cells in one way or another.</P><LI>Parameters</LI><P>We have coherent sets of parameters, but we still can't compare our models to the real system and are waiting for the results of characterization experiments.</P></UL></DIV><DIV class="blocbackground"><H2>Team Marmottes<SPAN>Marion</SPAN></H2><P>Moving in to the CIME, a biology teaching platform, free for the summer vacation. Preparation of new storage. PCR results from last week cloning.</P><UL><LI>Cloning session :
	    -&gt; every 1st-step constructions that failed last time, so 6 constructions.
	    -&gt; 2 2nd-step constructions:
	    MerR-LuxR
	    TetR-LuxR</LI><P>Ligations:
	    Because we still have a low cloning rate we tried different proportions between vectors and inserts:
	    1 vector/1 insert
	    1 vector/2 inserts
	    1 vector/3 inserts
	    Transformations with electrocompetent cells 
	    Spreading over Petri Dish
	    RESULTS :
	    Best results on Petri Dishes with the proportion 1/3. All constructions seem to have the right size except MerR-LuxR.</P><LI>Sequencing Order</LI><P>RBS-TetR RBS-LuxI RBS-CinR RBS-LacI pLac-GFP pCin-GFP pMerT-GFP pLux-Lyco</P><LI>Products Order</LI><P>For ligations, digestions, PCR, MiniPreps, MidiPreps</P><LI>Cloning Session :
	      -&gt; 5 constructions with Fha1
	       -&gt; 3 2nd-step constructions
	            LacI-LuxI
	            MerR-CinR
	            TetR-CinR
	       -&gt; 1 construction for the tests
	            pLux-GFP
 	      -&gt; 1st-step construction 
	            RBS-CinI</LI><P>The final RBS (Fha1) which allows to keep the toggle off has just been delivered. So, all first steps constructions including RBS were made once again.Because the efficiency of Fha1 hasn't been completly demonstrated, we keep on cloning with the standard RBS. So, RBS and Fha1 constructions are made in parallel.</P><P>RESULTS : 
	    Every constructions gave colonies on Petri Dishes.
	    But PCR cheking showed that nothing were amplified on every Fha1 constructions with VF2 and VR primers. Thus, Fha1 was provided in a PCR blent plasmid instead of an iGEM's plasmid. 
	    The insert of the test construction is having the right size.
	    2 out of 3 2nd step constructions had the right size.
	    We still have issues to clone the 1st step construction: RBS-CinI.</P></UL></DIV><H1 id="week2">July 14<SUP>th</SUP> to 21<SUP>st</SUP></H1><DIV class="blocbackground"><H2>Team Marmottes<SPAN>Morgane</SPAN></H2><P>Better clonings rate: Clonings performed with recommended enzymes by iGEM. </P><UL><LI>Stock of electro competent cells </LI><P>RESULTS : Great Transformation Rate</P><LI>New stock of Biobrick Miniprep </LI><P>DO checking PCR cheking</P><LI>Cloning of every first steps of our construction: (Standard Assembly Method)
	    -&gt; 11 clonings</LI><P>We changed our strategy: clonings were performed with enzymes recommended by iGEM (New England Biolabs). </P><P>One Petri Dish was free of colony. We performed PCRs on 5 colonies from each Petri Dish. According to PCR results, 5 constructions out of 11 seem to have the right size.</P><LI>Cloning session :
	    -&gt; every 1st-step constructions that failed last time, so 6 constructions.
	    -&gt; 2 2nd-step constructions:
	         MerR-LuxR
	         TetR-LuxR</LI><P>Only digestions were performed because of a lack of equipments.
	   We decided to use the Standard Protocol for the first step constructions because the assembly combinates a short and a long gene. The long gene is inserted in the plasmid of the small gene.
	   But the Second step constructions were performed with the 3A Assembly Protocol.</P></UL></DIV><DIV class="blocbackground"><H2>Team Godlike<SPAN>Maxime</SPAN></H2><P>Stochastic model done</P><UL><LI>Models</LI><P>First results with the stochastic model. Our whole model does work like we expected it. But parameters are needed more than ever now.</P><LI>Parameters</LI><P>We got incomplete sets of parameters from either litterature or iGem teams. Some very useful sets can be found in Aberdeen 09, Brown 10, ETHZ 07, BCCS Bristol 08</P></UL></DIV><H1 id="week1">July 7<SUP>th</SUP> to 13<SUP>th</SUP></H1><DIV class="blocbackground"><H2>Team Marmottes<SPAN>Marion</SPAN></H2><P>Moving in to the CIME, a biology teaching platform, free for the summer vacation. Preparation of new storage. PCR results from last week cloning.</P><UL><LI>Moving in of the whole team to a new lab</LI><P>We are taking advantage that the school year is over to move in the CIME, a teaching platform of biology.</P><LI>Cheking of DNA concentration into our Biobrick Miniprep</LI><P>OD cheking was performed on all our Biobrick stock to have an idea of the DNA rate into our preparation. We also made a PCR cheking to make sure that the DNA rate really corresponds to our Biobrick.
	  We got an average rate of 100 ng/µl. Which is a much lower than expect, so 10 times more DNA products would had been necessary to get great constructions.</P><LI>Order of new enzymes -&gt; New England Biolabs Company</LI><P>We hope to increase the digestion efficiency.</P><LI>PCR Checking of cloning from the last week</LI><P>As previously, inserts were shorter than expected, except the construction RBS-TetR.</P><LI>Making new storage</LI>Petri Dishes with Antibiotic (Amp, Kan, Cm, Tet). LB Culture medium<LI>RBS-TetR Miniprep</LI><P>In order to send it to the sequencing.</P><DIV class="blocbackground"><H2>Team Godlike<SPAN>Geoffrey</SPAN></H2><P>Learning Gillespie and looking for parameters.</P><UL><LI>Models</LI><P>Stochastic model is on its way. We base it on Gillespie's algorithm. It's mainly about getting familiar with the math for now. Also working on Hysteresis and isoclines subparts of the deterministic models.</P><LI>Parameters</LI><P> Still a lot of work going on there. The problem is we can not compare our sets to experimental results as the whole system is not finished yet. We will then try to characterize some of the promoters we use and figure out how to obtain some degradation rates as well </P></UL></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Grenoble/Notebook_Rawhide">http://2011.igem.org/Team:Grenoble/Notebook_Rawhide</A>&quot;</DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grenoble/Notebook_Rawhide" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grenoble/Notebook_Rawhide" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grenoble/Notebook_Rawhide&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grenoble/Notebook_Rawhide&amp;oldid=68903" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></UL></DIV></DIV></DIV></DIV></DIV></DIV></BODY></HTML>