<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grenoble_Safety"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grenoble/Safety">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grenoble/Safety&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Safety&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Safety&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grenoble/Safety" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grenoble/Safety</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Grenoble 2011, Mercuro-Coli iGEM</TITLE></P><DIV class="left"><DIV class="navigation"><DIV class="centering"><UL class="topnav"><UL><LI><A href="/Team:Grenoble">Home</A></LI><LI><A href="/Team:Grenoble/Projet/Intro"><SPAN class="scriptina"><I>Le Projet</I></SPAN></A></LI><LI><A href="/Team:Grenoble/Projet/Intro">Introduction</A></LI><LI><A href="/Team:Grenoble/Projet/Device">Device</A></LI><LI><A href="/Team:Grenoble/Projet/Design">The Genetic Network</A></LI><LI><A href="/Team:Grenoble/Projet/Modelling">Modelling</A></LI><LI><A href="/Team:Grenoble/Projet/Results">Results</A></LI><LI><A href="/Team:Grenoble/Projet/Biobricks">Biobricks</A></LI></UL><LI><A href="/Team:Grenoble/HumanPractice">Human Practice</A></LI><LI><A href="/Team:Grenoble/HumanPractice">Overview</A></LI><LI><A href="/Team:Grenoble/HumanPractice/identifying">A communication problem</A></LI><LI><A href="/Team:Grenoble/HumanPractice/developing">Developing a solution</A></LI><LI><A href="/Team:Grenoble/HumanPractice/promoting">Promoting synthetic biology</A></LI><LI><A href="/Team:Grenoble/HumanPractice/collaboration">Collaboration</A></LI></UL><LI><A href="/Team:Grenoble/Safety">Safety</A></LI><LI><A href="/Team:Grenoble/Notebook/June">Notebook</A></LI><LI><A href="/Team:Grenoble/Notebook/June">June</A></LI><LI><A href="/Team:Grenoble/Notebook/July">July</A></LI><LI><A href="/Team:Grenoble/Notebook/August">August</A></LI><LI><A href="/Team:Grenoble/Notebook/September">September</A></LI><LI><A href="/Team:Grenoble/Notebook/October">October</A></LI><LI><A href="/Team:Grenoble/The_Team/">About us</A></LI><LI><A href="/Team:Grenoble/The_Team/">The Team</A></LI><LI><A href="/Team:Grenoble/Grenoble">Grenoble</A></LI><LI><A href="/Team:Grenoble/Data_Page">Data Page</A></LI><LI><A href="/Team:Grenoble/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV class="body"><DIV class="left"><DIV class="blocbackground"><H1>Safety issues</H1><P>
				In general, the work in a laboratory requires the use of complex equipment or it implies performing delicate operations. The material, that could be a machine, chemicals or biological material involves the existence of risks. Risks for the goods and for people in the room, but also risks for the environment and people outside the lab. The safety rules and procedures as well as the personal and collective protective equipment are made to minimize the risks by decreasing the probability of  an incident to happen.
			</P></DIV><DIV class="blocbackground"><H3 id="general">General considerations</H3><P>
				Half of our team made an internship at the CEA Grenoble. The CEA has a specific department working on safety issues. There is also a special team in charge of security and safety called FLS: Formation locale de sécurité, we may translate: Local Group of Security and Safety. They ensure the safety of the people who are working in the center and the visitors and also of the goods and the material. The members of our team who made their internship in CEA have attended a compulsory safety training organised by FLS. 
			</P><P>
				The whole team is now working together in another lab, the CIME (centre inter-universitaire de microélectronique), next to the CEA labs and the Phelma school buildings. All team members have met the safety engineer of the labs where we conduct the experiments. He explained us the safety rules to be followed.
			</P><P>
				At CEA some researchers work on microsystems to detect and quantify pollutants like heavy metals. They shared their experience and knowledge with us about the way to conduct safe experiments with very toxic chemicals like mercury. 
			</P></DIV><DIV class="blocbackground"><H3 id="instru">Instruments</H3><P>
				During our experiments, we only performed commonly used protocols and instrumentation for microbiology and common laboratory strains of E.Coli. We have used basic devices that we find in every molecular biological laboratory:
				<DL><DT>Ultra violet lamp: </DT><DD>
						There is a risk for the eyes and the skin, but the UV lamp is only used to take a picture of our gel after electrophoresis, so we are never directly exposed because there is a protective cover and we wear a mask that shields from UV.
					</DD><DT>Centrifuge:</DT><DD>
						The centrifuges have to be perfectly balanced. All the centrifuges in our lab have detectors that warn the operator in case of imbalance. 
					</DD><DT>Autoclave:</DT><DD>
						The operation of the autoclave require a specific training and has only be performed by trained people.
					</DD></DL></P></DIV><DIV class="blocbackground"><H3 id="chemical">Chemical risk-assessment</H3><P>
				A toxic chemical, the EtBr (ethidium bromide) is commonly used to stain DNA. We do not use EtBr solution while making our gel but we dip the gel in an EtBr bath after the electrophoresis. Due to the hazardous nature of this product, a hood is specially dedicated to its usage. The EtBr and all material that got in contact with it is stored in a special trash in the hood. 
			</P><P>
				We design a biosensor to measure a pollutant (like heavy metals) concentration in water. We are actually working on two versions of this biosensor. One of them involves the use of the MerR sensor for mercury, and the second alternative one, TetR for tetracycline. We therefore need to use mercury to test this system. This raises questions about safety for the researcher but also for the public and the environment. The teracycline is a safer alternative to test our system.
			</P><P>
				For the tests we will have to use mercury in a water solution. It is the ionic form Hg2+ that will be used. The mercury is very toxic and mutagenic. To limit the risks in terms of probability of incident and in terms of hazards, we will use a stock solution. We will only have to dilute it to the wanted concentrations. Moreover a chemical hood will be dedicated to the preparation of these solutions and for the test of our system. A specific trash will also be dedicated to store all the wastes in contact with mercury.
			</P><P>
				Concerning the toxic waste management, a firm specialized in the processing of hazardous wastes recovers the barrels of toxic chemicals. A tracking sheet is associated to each toxic barrel. The lab receive then a document that certifies the appropriate waste treatment.
			</P></DIV><DIV class="blocbackground"><H3 id="bio">Biological risks, biosafety rules</H3><P>
				However we performed common experiments of molecular biology and biochemistry for which the risks are well known nowadays. The most uncertain part of our project is the genetic modification of living organism. Hence we are presenting the different Biobrick and confronting them to their related safety issues as well as a scenario where we discuss the different hazards and their possibility.
			</P><P>
				In our bio-safety analysis, we try to take into consideration : 
				<UL><LI>The risk of the chassis bacteria</LI><LI>The one of each biobrick as well as their combinations in the whole device</LI><LI>The robustness of the whole device</LI></UL></P><H4 id="microorg">Microorganism chassis</H4><P>	
				After each experiment, all biological wastes are collected in a special bin and autoclaved before leaving the lab, to prevent environmental contamination.
			</P><P>
				We work with a strain of E.Coli designed for lab work : BW25113. This strain is commonly used by students and researchers. It has no virulence genes, and is therefore a riskless chassis. Furthermore, it has got several genetic modifications that will limit its development if ever it was to make it out of the lab. Those modifications are :
				<UL><LI>An inactivated lacZ, ara and rha genes : the bacteria can use neither lactose, arabinose or rhamnose as sources of energy.</LI><LI>A deletion into a gene coding for an enzyme (pyr E) involved in the synthesis of Thymine and Cytosine nucleotides. This deletion however does not make the strain auxotroph for theses bases.</LI></UL></P><P>
				These mutations are a disadvantage and limit the development of this strain on a minimal medium. 
		</P><H4 id="bioparts">Biobricks parts used</H4><P>
				The genetic device we develop is a toggle switch which includes two quorum sensing biobricks: cin I and cinR genes. We also use a reporter gene coding for a pigment, the lycopen as an output. We include a post transcriptional regulation mechanism extracted from Pseudomonas aeruginosa, rsma.  We present here some details about safet of these biobricks. Moreover we made an event tree to analyse a scenario where one of our bacteria went out of the lab.
			</P><H5 id="toggle">Toggle switch</H5><P>
				Bio-safety is not only about the nature of the basic parts, but also about their combinations. The toggle switch mechanism is a “man-made” specific combination of inoffensive genetic sequences. It has no hazard on its own but activates the transcription of downstream genes, in our case cinI or cinR (see here after). In an uncontaminated environment (no mercury) only cinR is transcribed and no quorum sensing molecule is synthesized.
			</P><H5 id="qs">Quorum sensing</H5><P>
				We used CinI as quorum sensing, share by many species of legume-nodulating rhizobia <A href="#1">(1)</A>⁠, a genus of soil bacteria that fix nitrogen. The cin quorum sensing molecule regulates growth inhibition, expression of nodulation genes, but no harmful response has been noticed so far. It is a Rhizobium-specific communication system. These bacteria colonise plant cells within root nodules and have never shown any pathogenicity towards humans and their environment.
			</P><H5 id="tomato">The lycopen</H5><P>
				We use a combination of three genes that codes for lycopen. This pigment is naturally found in tomato and has no toxicity.
			</P><H5 id="rsma">The rsma regulation system</H5><P>
				The Rsma translational regulation system we extracted from an opportunistic bacterium called Pseudomonas aeruginosa <A href="#2">(2)</A>⁠. It is very similar to others regulation systems like “Csra” that can be found in many eubacteria <A href="#3">(3)</A>. The Rsma and csra systems both control a large variety of physiological processes such as central carbon metabolism, motility, biofilm formation, virulence, pathogenesis <A href="#4">(4)</A> ⁠and many more <A href="#5">(5)</A>⁠...
			</P><P>
				Basically, when CsrA or RsmA proteins are expressed, they bind to a mRNA leader sequence and act as translational repressors by inducing their degradation. Alternatively, when a small RNA molecule is expressed (called rsmy in the Pseudomonas system) it titres and sequesters the protein, allowing the expression of targeted genes <A href="#5">(5)</A>. In most systems, the binding site on the RNA leader sequence is a stem-loop containing repeats of GGA nucleotides <A href="#4">(4)</A>.
			</P><P>
				In Pseudomonas, the protein RsmA negatively regulates the type VI secretion system, which has been implicated in the P.aeruginosa chronic infections <A href="#5">(5)</A>⁠. In specific environment conditions, rsmY/rsmZ are transcribed, which produces a syringe base plate <A href="#4">(4)</A>⁠. That is used to inject proteins into a target cell. 
			</P><H5 id="outlab">Analysis of a catastrophic scenario</H5><P>
				Being originally implicated in the activation of virulence genes, this rsma regulation system implies safety issues. For instance, the RsmA system could somehow interfere with the CsrA system of E. coli, which is highly homologous. 
				<UL><LI>So what would happen if our strain was to make it out of the lab ?</LI><LI>Could our genetic device activate genes in a wild type E.coli strains, or in any other bacteria ?</LI><LI>Would it, then, become harmful to human or any other organism in the environment ?</LI></UL></P><P>
				The worst situations we could imagine with an organism containing these bricks are:
				<UL><LI>an over development of bacteria in any specific environment</LI><LI>a health threat to human or any other organism</LI></UL></P><P>
				We tried to think about what would be the series of event to cause such a disaster. For each of them we tried to think of how probable it is to occur, and what is know about the possible hazard. 
			</P><P>
				In order to illustrate our vision of the risk, we made an event tree analysis composed of three colours representing the level of probability (yellow - orange - red).
			</P><TABLE><TBODY><TR><TH>Hazard</TH><TH>Likelyhood of Hazard to occur</TH></TR><TR><TD>1. Some bacteria containing our genetic circuit get out of the lab.</TD><TD><UL><LI>So far the project is experimental : all components are confined to  the laboratory. </LI><LI>All the biological waste are collected in a special bin and autoclaved before leaving the lab.</LI></UL></TD></TR><TR><TD>2. These bacteria grow in the environment</TD><TD>Our strain has  several mutations that limit its development outside the favourable conditions of the lab</TD></TR><TR><TD>3. A living E.coli from the lab tranfers genes to a wild type E.coli by conjugation</TD><TD>This strain is F- so it cannot build up any pili that are necessary for gene transfer. It could however become competent if it would acquire the necessary genes in contact with  F+ bacteria.</TD></TR><TR><TD>4. Wild-type bacteria of any species incorporate genetic material from a lysed cell released from the laboratory</TD><TD>The sequence we use comes originally from Pseudomonas. The risk to transfer this DNA sequence therefore already exists potentially in nature.</TD></TR><TR><TD>5. The transferred RsmA protein or rsmy RNA interact with the host regulation system</TD><TD>This is likely to occur since homologs exist in many species.</TD></TR><TR><TD>6. The transferred DNA can activate the transcription of virulence genes</TD><TD>Several check points exist in a cell to control such a global mechanism. A response is triggered only if all checkpoints are coherently functioning.</TD></TR><TR><TD>7. Virulence factors are expressed in the bacteria which  has incoporated the brick</TD><TD>In a normal condition a virulent cell would express its virulence only when a targeted host is close and triggers it. This might not be the case. Then the expression of virulence factors would be useless, or even a disadvantage.</TD></TR><TR><TD>8. Virulence factors are expressed and threat a specific target </TD><TD>As far as we know about this system, it  is used by opportunistic bacteria that can become pathogenic only to immune-depressive individuals. </TD></TR></TBODY></TABLE><H4 id="robustus">Robustness of the device</H4><DIV class="legend">
					Click on the icon above to see the result of mathematical study of the robustness of the device.
				</DIV><P>
				In this section, we consider how the device would behave if some of the components stop working properly. Mutations can prevent the functionning of parts of the genetic circuit, with predictible consequences.
			</P><P>
				In our project, we can divide the system into four components:
				<UL><LI>the toggle switch component</LI><LI>the communication component based on quorum sensing genes</LI><LI>the output signaling component</LI><LI>a translational regulation component based on rsma</LI></UL></P><P>
				Several situations were considered and tested using the numerical model we developped. Here are a few examples of unwanted behaviours of components and their expected consequences.
			</P><P><UL><LI>If the degradation tags of the repressors in the toggle switch do not work, the toggle switch would become very slow in switching and therefore unable to sense external molecule concentration. The response of the device will be very slow.</LI><LI>If any branch of the toggle switch is not functional, then no activation of the output signaling component can occur. The plate would remain white. </LI><LI>If the cinI promotor leaks or is not sensitive enough to AHL, the output signal will be produced whatever the external molecule concentrations. The entire device will be red. </LI><LI>If the post-transcriptional system doesn't work, the toggle switch would be less sensitive but still working. The red band will be larger.</LI></UL></P><P>
				This last result suggests that the RsmA/rsmY regulation system is not critical but modelling show it improves dramatically the sensitivity of our device. Given that it is a global regulation system, it might interfere with cinI (see section biobrick safety). Before integration into our final genetic network we will have to test this possibility.
			</P><CENTER><DIV class="legend">
					Click on the icon above to know why RsmA is important.
				</DIV></CENTER></DIV><DIV class="blocbackground"><H2 id="bibli">References</H2><DIV id="1">1.</DIV> Wisniewski-dy, F. and Downie, J.A. Quorum-sensing in Rhizobium. Antonie van Leeuwenhoek 397-407(2002).
			<DIV id="2">2.</DIV> Brencic, A. and Lory, S. Determination of the regulon and identification of novel mRNA targets of Pseudomonas aeruginosa RsmA. Molecular Microbiology 72, 612-632(2009).
			<DIV id="3">3.</DIV> Timmermans, J. and Melderen, L.V. Post-transcriptional global regulation by CsrA in bacteria. Cellular and Molecular Life Sciences 2897-2908(2010).doi:10.1007/s00018-010-0381-z
			<DIV id="4">4.</DIV> Mercante, J. et al. Molecular Geometry of CsrA ( RsmA ) Binding to RNA and Its Implications for Regulated Expression. Journal of Molecular Biology 392, 511-528(2009).
			<DIV id="5">5.</DIV> Bernard, C.S. et al. MINIREVIEW Nooks and Crannies in Type VI Secretion Regulation . Society 192, 3850-3860(2010).
			</DIV><DIV class="blocbackground"><H2>OPTIONAL QUESTION:</H2><STRONG>Do you have other ideas on how to deal with safety or security issues that could be useful for future iGEM competitions? How could parts, devices and systems be made even safer through biosafety engineering?</STRONG><P>
				Manipulation of living organism allows producing artificial form of life and metabolism. These modifications, although well controlled, require application of the precautionary principle. Even if in our project we do not plan to take our work out of the lab, engineered bacteria might be accidentally or on purpose released in the environment.
			</P><P>
				In many projects, the main issue is that bacteria may leave the lab. In such a case, efficient methods should be used to limit or prevent their development ouside. Synthetic biology project should employ living organisms unnable to survive outside the lab. For exemple microorganisms forced to use rare carbon sources. Another possibility is to have a suicide gene repressed by an artificial molecule no tpresent in nature. Another way is to make bacteria weaker so that they cannot face natural selection. 
			</P><P>	
				To increase the safety of iGEM competition, we propose to use auxotrophic bacteria that cannot grow in the absence of a given amino acid, for instance. 
			</P><P>
				A team of researcher from CEA (IG/Genoscope – Évry), Institut für Biologie (Freie Universität, Berlin), CNRS, University of Evry, Katholieke Universiteit (Leuven) and Heurisko company (USA) worked on <A href="http://www.cea.fr/le_cea/actualites/evolution_d_un_genome_bacterien-60034">Chemical evolution of a bacterial genome.</A> They forced the bacteria to use a synthetic chemical instead of Thymine.
			</P></DIV></DIV></DIV><DIV class="body"><DIV class="left"><DIV class="blocbackground"><TABLE class="nobordure"><TBODY><TR><TD><A href="https://2011.igem.org/Main_Page" title="iGEM 2011 Main Page">iGEM 2011 Main Page</A><H3><SPAN class="vert">Contact</SPAN> Us:</H3><H3><A title="Geoffrey BOUCHAGE" href="mailto:geoffrey.bouchage@phelma.grenoble-inp.fr"> Clic here !</A></H3></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Grenoble/Safety">http://2011.igem.org/Team:Grenoble/Safety</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grenoble/Safety" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grenoble/Safety" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grenoble/Safety&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grenoble/Safety&amp;oldid=255910" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>