"
Page
Discussion
View source
History
teams
Log in
 
Team:Grenoble/Projet/regulation
From 2011.igem.org
Grenoble 2011, Mercuro-Coli iGEM
Home
Le Projet
Introduction
Device
The Genetic Network
Modelling
Results
Biobricks
Human Practice
Overview
A communication problem
Developing a solution
Promoting synthetic biology
Collaboration
Safety
Notebook
June
July
August
September
October
About us
The Team
Grenoble
Data Page
Sponsors
Two new translational regulation mechanisms!
A post-transcriptional regulation system for our toggle switch
The toggle we built, switch the bacteria to a sender or receiver phenotype depending on the relative amounts of two inducers: mercury (or tetracycline) in the sample and IPTG, our reference which is comprised as a linear gradient in our sensor. Bacteria are thus exposed to IPTG before they encounter the inducer contained in the sample and hence all cells will be in the receiver phenotype induced by IPTG. To avoid this bias, we want to keep the amount of LacI repressor as low as possible until the sample to analyse is added.
To achieve this, we decided to develop a translational regulation system that allows to control the onset of the synthesis of both repressors (LacI and MerR/TetR). This regulation mechanism should allow, upon triggering, to rapidly increase the amount of a protein within a cell.
We investigated two mechanisms that are well documented in the literature and that can be extracted from different microorganisms.The first one is from Pseudomonas aeruginosa and comprises two RNA sequences and a protein, RsmA. The second one, the RpoS regulation system, is from E. coli, and it involves a hairpin leader sequence and an inducible regulatory small RNA.
We isolated and cloned the RsmA translational regulation system from Pseudomonas aeruginosa (BBa_K545005, BBa_K545006, BBa_K545007, BBa_K545008), and     part of the RpoS system from E. coli (BBa_K545666).
The RsmA translational regulation system
How does it work?
The RsmA regulation system of Pseudomonas has homologs in many other bacteria, like CsrA of Escherichia coli1, for example. It is basically composed of:
A leader sequence at the 5’ end of the mRNAs of the genes to be regulated. Many different sequences exist depending on the gene to regulate.
A regulatory protein named RsmA that binds to a GGA motif within the stem-loop structure of the transcribed leader sequences2. When RsmA is bound to the mRNA, the latter cannot be translated and is degraded.
An inducible small RNA – the one we use is called rsmY – which sequesters the RsmA protein, having a greater affinity for it than the transcribed gene leader sequences.
Using this system, the cell transcribes genes of which the translation is more or less repressed by RsmA, depending on their leader sequence (Fig 1). The strength of the repression depends on the stem-loop conformation of the leader sequence as well as on the number of GGA repeats that constitute binding sites for RsmA (see also Fig 3 + 4)
Figure 1 : When no trigger comes from the environment, the translation of genes carrying a leader sequence (LS) containing stem-loops and GGA motifs is repressed by RsmA. The ribosome cannot bind on the RBS and the mRNA is not translated.
When the transcription of rsmY is triggered, the rsmY RNA acts as an activator by sequestering the RsmA repressor and allowing the ribosome access to the messenger to be translated (see Fig 2).
Figure 2 : When the transcription of rsmY is triggered, the RsmA protein is sequestered, which allows the translation of genes carrying an RsmA-controlled leader Sequence.
Fha1 and magA operon leader sequences
A microarray analysis revealed that RsmA regulates about 60 genes from two to more than one hundred fold3! Most of those genes are involved in secretion, or pili biogenesis. We decided to work on the leader sequences of magA and fha1. They are not strongly inhibited by RsmA, but are well documented, and the biobricks we made will be useful for our host lab.
Figure 3 : Secondary structure of the leader sequence of fha1, identified as a direct target for RsmA regulation. The downstream sequence codes for a scaffold protein of the “type 6” secretion system. Highlighted are the ribosome-binding site (RBS) and the GGA motifs (Brenic and Lory, 2009). PA0081 is the sequence ID of the Pseudomonas genome project website ( http://www.pseudomonas.com/ )
Figure 4 : Secondary srtucture of the magA operon leader sequence, also identified as a direct target of RsmA regulation. The operon contains a macrobuline-like protein (Brenic and Lory, 2009). The ribosome-binding site (RBS) is highlighted. PA4492 is the sequence ID of the Pseudomonas genome project website ( http://www.pseudomonas.com/ )
rsmA translational regulation system influence the initial condition of our system. It allows us to control       these initial conditions. Without this system, the accuracy of our device would be degraded as shown here . You can also find characterization       of rsmA components here .
The rpoS regulation system
When nutrients become scarce, bacteria need to quickly shut down the expression of many genes an activate others. A major global regulator of this growth transition is RpoS, an alternative sigma factor (also called sigmaS). The RNA polymerase holoenzyme containing RpoS recognizes a new set of promoters and thus changes the global transcriptional program in an appropriate manner.
Because of the central role of RpoS, its expression is tightly regulated. Much of this regulation is exerted at the level of translation. The mechanism has been intensely studied and we can therefore exploit the system to create a new biobrick that provides an on-off switch for the translation of target genes.
The 5'-untranslated RNA (5'-UTR) of the rpoS gene adopts a particular secondary structure that places the ribosome binding site into a double-stranded region and therefore prevents recognition by the ribosome5. A small RNA, called dsrA, is produced when the cells enter starvation. This RNA interacts with the 5'-UTR of the rpoS RNA and induces a change in its secondary structure that liberates the RBS and thus stimulates the translation of rpoS 6⁠.
We have amplified the rpoS leader sequence by PCR and cloned it into PSB1C3.
1. Timmermans, J. & Melderen, L.V. Post-transcriptional global regulation by CsrA in bacteria. Cellular and Molecular Life Sciences 2897-2908(2010).doi:10.1007/s00018-010-0381-z
2. Mercante, J. et al. Molecular Geometry of CsrA ( RsmA ) Binding to RNA and Its Implications for Regulated Expression. Journal of Molecular Biology 392, 511-528(2009).
3. Brencic, A. & Lory, S. Determination of the regulon and identification of novel mRNA targets of Pseudomonas aeruginosa RsmA. Molecular Microbiology 72, 612-632(2009).
iGEM 2011 Main Page
Contact Us:
Clic here !
Retrieved from " http://2011.igem.org/Team:Grenoble/Projet/regulation "
Recent changes
What links here
Related changes
Special pages
My preferences
Printable version
Permanent link
Privacy policy
Disclaimers
