<HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
<style type="text/css">
A:before { content:' '; } 
A:after { content:' '; } 
SPAN:before { content:' '; } 
SPAN:after { content:' '; } 
</style>
<BODY class="mediawiki  ltr ns-0 ns-subject page-Team_Grenoble_Notebook_June"><DIV id="globalWrapper"><DIV id="top-section"><DIV id="p-logo"><A href="/Main_Page" title="Main Page">&quot;
	    </A></DIV><DIV id="menubar" class="left-menu noprint"><UL><LI class="selected"><A href="/Team:Grenoble/Notebook/June">Page               </A></LI><LI class="new"><A href="/wiki/index.php?title=Talk:Team:Grenoble/Notebook/June&amp;action=edit&amp;redlink=1">Discussion               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Notebook/June&amp;action=edit">View source               </A></LI><LI><A href="/wiki/index.php?title=Team:Grenoble/Notebook/June&amp;action=history">History               </A></LI><LI style="color:#808080;cursor:default">teams</LI></UL></DIV><DIV class="right-menu noprint" id="menubar"><UL><LI id="pt-login"><A href="/wiki/index.php?title=Special:UserLogin&amp;returnto=Team:Grenoble/Notebook/June" title="You are encouraged to log in; however, it is not mandatory [o]" accesskey="o">Log in</A></LI></UL></DIV><DIV id="search-controls" class="noprint"><FORM action="/Special:Search" id="searchform"> </FORM></DIV></DIV><DIV id="content"><H1 class="firstHeading">Team:Grenoble/Notebook/June</H1><DIV id="bodyContent"><H3 id="siteSub" class="noprint">From 2011.igem.org</H3><P><TITLE>Grenoble 2011, Mercuro-Coli iGEM</TITLE></P><DIV class="left"><DIV class="navigation"><DIV class="centering"><UL class="topnav"><UL><LI><A href="/Team:Grenoble">Home</A></LI><LI><A href="/Team:Grenoble/Projet/Intro"><SPAN class="scriptina"><I>Le Projet</I></SPAN></A></LI><LI><A href="/Team:Grenoble/Projet/Intro">Introduction</A></LI><LI><A href="/Team:Grenoble/Projet/Device">Device</A></LI><LI><A href="/Team:Grenoble/Projet/Design">The Genetic Network</A></LI><LI><A href="/Team:Grenoble/Projet/Modelling">Modelling</A></LI><LI><A href="/Team:Grenoble/Projet/Results">Results</A></LI><LI><A href="/Team:Grenoble/Projet/Biobricks">Biobricks</A></LI></UL><LI><A href="/Team:Grenoble/HumanPractice">Human Practice</A></LI><LI><A href="/Team:Grenoble/HumanPractice">Overview</A></LI><LI><A href="/Team:Grenoble/HumanPractice/identifying">A communication problem</A></LI><LI><A href="/Team:Grenoble/HumanPractice/developing">Developing a solution</A></LI><LI><A href="/Team:Grenoble/HumanPractice/promoting">Promoting synthetic biology</A></LI><LI><A href="/Team:Grenoble/HumanPractice/collaboration">Collaboration</A></LI></UL><LI><A href="/Team:Grenoble/Safety">Safety</A></LI><LI><A href="/Team:Grenoble/Notebook/June">Notebook</A></LI><LI><A href="/Team:Grenoble/Notebook/June">June</A></LI><LI><A href="/Team:Grenoble/Notebook/July">July</A></LI><LI><A href="/Team:Grenoble/Notebook/August">August</A></LI><LI><A href="/Team:Grenoble/Notebook/September">September</A></LI><LI><A href="/Team:Grenoble/Notebook/October">October</A></LI><LI><A href="/Team:Grenoble/The_Team/">About us</A></LI><LI><A href="/Team:Grenoble/The_Team/">The Team</A></LI><LI><A href="/Team:Grenoble/Grenoble">Grenoble</A></LI><LI><A href="/Team:Grenoble/Data_Page">Data Page</A></LI><LI><A href="/Team:Grenoble/Sponsors">Sponsors</A></LI></DIV></DIV></DIV><DIV class="body"><DIV class="left"><DIV class="blocbackground"><H1 id="week1">June 1<SUP>st</SUP> to 7<SUP>th</SUP></H1></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>No manipulation, we just planned the best way to work. </P></DIV><UL><LI>Plasmid Mapping</LI><UL><LI>Toggle Switch : (4 final plasmids)</LI><P>
						1 plasmid was considered for each toggle switch way (MerR or LacI), so 2 final plasmids. But those 2 plasmids will include the 2 different couples cinI/cinR and luxI/luxR. The most efficient construction will be used.
					</P><LI>Coloration Generator : (4 plasmids)</LI><P>
						Coloration is induced by the activation of the promoter pLux or pCin depending on which couple cinI/cinR or luxI/luxR picked. As previously, the best coloring agent hasn't been decided yet. So, 4 plasmids will be produced: 1 for pCin, 1 for pLux and both followed either by GFP or Lycopène.
					</P><LI>Tests : (6 plasmids)</LI><P>
						The test of the two ways (MerR and LacI) of our toggle will be achieved by replacing pLac or pMerT by a constitutive promoter. So, 4 plasmids: 2 ways of the toggle switch, 2 different couples (luxI/luxR, cinI/cinR). Efficiency of both promoters will be tested separately by associating them to GFP.
					</P></UL><LI>Biobrick Listing :</LI><UL><LI>21 Biobricks will be necessary to achieve only the toggle switch and the coloration generator :</LI><UL><LI>14 Biobricks</LI><LI>7 plasmid backbones</LI></UL><P>
						This includes biobricks which will be used for the tests (for example for the promoters) and as intermediate constructions.
					</P></UL><LI>Manipulation schedule :</LI><P>
					Steps were defined for each construction depending on the level of assembly required. For example the assembly of two existing biobricks corresponds to a first level. Whereas the assembly of two newly obtained corresponds to a 2nd our 3rd level. Each level should be achieve at the time.
				</P></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				Preparation of chemically competent cells. Transformation with 20 ng of a reference plasmid in order to know the transformation efficiency of or the cells. Biobricks research and BioBricks characterization researchs.
			</P><P>
				Realization of glycerol stocks from our competent cells. Transformation trainingg,... . Fabrication of new competent cells. Preparation of PCR for making new biobricks. Search for PAO1 Pseudomonas genome project website, reading about biobricks prefix, suffix, and assemble process. Bibliography. Get information about prefix-suffix, digestion, spring 2011 Dna distribution. Chemically competent cells with a better transformation rate.
			</P></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>Essentially worked on the models to use and how to simulate them</P></DIV><UL><LI>Models</LI><P>
					Toggle Switch : Worked on the equations that would modelize our system the best. We now have the equations for the toggle switch and a first Matlab script that we can base our work on. <I>reference : Gardner, T.R., Cantor, C.R. &amp; Collins, J.J., Construction of a genetic toggle switch in Escherichia coli,  403, 339 - 342 (2000)</I></P><LI>Programs</LI><P>
					Simulations : We are mainly working on a deterministic model for our network, we use basic Matlab ODE solvers for now. Our plan for the final device is a plate with our bacteria. We will have to adapt our code for this particular device
				</P></UL></DIV><DIV class="blocbackground"><H1 id="week2">June 8<SUP>th</SUP> to 14<SUP>th</SUP></H1></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>
				Still worked on model equations
			</P><UL><LI>Model equations :</LI><P>
					We deducted our equations from chemical and physical mechanisms and worked on simplifications.
				</P><LI>Parameters :</LI><P>
					One of the most importants part of our work is finding parameters for our model in order to match real behaviour of our cells as precisely as possible. Half of the needed parameters obtained up to now.
				</P><LI>Programs :</LI><P>
					Minor  on the algorithm, minor bugs fixed and some plotting features added.
				</P></UL></DIV></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				Isolation of a plasmid from the host lab, which contains the  rsmA gene under the inducible promoter plac. The same plasmid without rsmA is also extracted, to be used as a negative control. 
			</P><P>
				Preparation of a new stock of competent cells. Realization of the first test, in order to know whether the RsmA protein of Pseudomonas inhibits the growth of Escherichia coli. Transformation of competent cells with the pUC plasmid that contains the rsmA coding sequence under a promoter inducible by IPTG , and the same plasmid without the rsmA sequence. One clone for each transformation is selected and cultured in triplicate with three different concentrations of IPTG : 0 ; 0,5 and 1 mM. The growth of E coli cells expressing the rsmA gene is identical to the growth of cells carrying the control plasmid.
			</P><P>
				Amplification of sequences involved in the rsmA system: fha1 Leader Sequence (fhaLS), rsmA, and rsmY. Amplification of the rpoS Leader Sequence (rposLS), involved in the translational regulation of the rpoS sigma factor. Sequences of interest were checked on online databases: “Pseudomonas genome project” for the RsmA system constituents, and on “colibri” for the rpoS leader sequence of Escherichia.  “Primer 3 website” is used to design all the primers, which are supplemented with specific iGEM sequences, and checked for their ΔG. Bacterial colonies are the matrices for all of those amplicons, except for rsma  that was on a Pbad vector. All of these PCR products carry the Biobrick prefixes and suffixes, which allow to work with the iGEM protocols. Migration on agarose gel. 
			</P><P>
				Several attempts are made to amplify the magA Leader Sequence (magALS). The later being very short, PCR amplification failed; it is finally ordered at a gene synthesis company.
			</P></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					Computer sequencing, first manipulations: transformation of the Biobrick just arrived.
				</P><UL><LI>Computer Sequencing :</LI><P>
						Every intermediate, final and test constructions were listed on DNA Workbench in order to compare PCR and Sequencing Results with this data bank.
					</P><LI>Biobricks Transformation :</LI><P>
						To achieve it, the Standard Transformation protocole was performed. 3 out 21 transformation gave colonies on Petri dishes:
					</P><UL><LI>MerR transformation</LI><LI>2 plasmid backbone</LI></UL><P>
						Red colonies resulted from all plasmid backbones, looks odd!!
					</P><LI>Culture on liquid media :</LI><P>
						All transformation that gave colonnies were resuspended except plasmid backbones. A stock of &quot; to use biobricks&quot; is waiting at 4°C and an other one is remaining at -80°C.
					</P></UL></DIV></DIV><DIV class="blocbackground"><H1 id="week3">June 15<SUP>th</SUP> to 21<SUP>st</SUP></H1></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>
					Working on parameters
				</P></DIV><UL><LI>Parameters :</LI><P>
						We now have enough parameters for simulating our toggle switch system. But still many parameters missing for a complete model. However, with such parameters we can at least start working on the final device main features.
					</P><LI>Simulations :</LI><P>
						Obtained our first (meaningful) curves ! The simulated genetical network does switch indeed :
						</P></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					Still working on Biobrick Transformations. MerR transformations is still giving nothing. pTet/TetR has been considered as a substitut.
				</P><UL><LI>Plasmids and MerR Transformation :</LI><P>
					In case our colony issues come from a manipulation mistake, we tried again to Transform our Biobricks with the Standard protocole. MerR transformation gave nothing. From plasmid backbones resulted Red colonies. But after looking on iGEM website plasmid backbones appears to have a RFP-gene as default Biobrick.
				</P><LI>Miniprep :</LI><P>
					The kit Macherey-Nagel NucleoSpin Extract II was used.
				</P><LI>MerR Electroporation Transformation :</LI><P>
					Because the efficiency of the Standard Transformation is much lower than the Electroporation Transformation, we performed the latter. As previously, we  nothing on our Petri dish.
				</P><LI>MerR PCR :</LI><P>
					To verify if there is something in the well. Nothing on the gel.
				</P><LI>pTet/TetR Transformation :</LI><P>
					Because we still have nothing with merR, we are looking for alternative to the couple pMerT/MerR. As usual, the Standard protocole was used. Petri dishes full of colonies.
				</P><LI>MerR order:</LI><P>
					After many trial, merR seems to be absent from the well 7C of the plate 4.
				</P></UL></DIV></DIV><DIV class="blocbackground"><H2>Biology</H2><P>
				Production of backbone plasmids and bricks of interest. We transform the following plasmids into E coli cells : psb1A3, psb1AC3, psb1AK3, psb1AT3, psb1C3, psb1K3, psb1T3, and the bricks that we need : I13401 ( gfp, terminator), J23119 ( a strong constitutive promoter), and E840 ( medium RBS, gfp, terminator). Digestion and migration on agarose gels. All plasmids except psb1T3, and the brick  J23119 are successfully obtained et our first attempt.
			</P><P>
				Production of  DNA of the brick J23119, digestion and migration on gel. Digestion of the PCR amplicons fha1 and cloning into psb1C3 in order to be sent to the registry. Digestion of the PCR amplicons fha1 by an other couple of enzymes, in order to clone it 5’ to the part I13401 ( gfp-terminator). Ligation of fha1 leader sequence to I13401 and selection on media containing a new antibiotic. No sequence could be inserted into IGEM plasmids, which were prepared by mini-preping and restriction digests without purification. 
			</P><P>
				Blunt ligation of our PCR products (fha1LS, rsmYLS, rsmA, rposLS) containing the biobrick prefixes and suffixes into  pTopo plasmids.
	</P></DIV><DIV class="blocbackground"><H1 id="week4">June 22<SUP>nd</SUP> to 28<SUP>th</SUP></H1></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>
					Working on Quorum Sensing.
				</P></DIV><UL><LI>Parameters :</LI><P>
						We now have a complete set of parameters for our whole network. However physical parameters such as viscosity of AHL are still missing. Accurate predictions are therefore still not possible for the whole system.
					</P><LI>Models and results :</LI><P>
						We have worked on the whole set of equations for the Quorum Sensing part of our network. We also demonstrated that our system can switch from one way to another with sufficient amount of IPTG/pollutant.
					</P></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					3A-Assembly were carried out for all first steps of our constructions but results were inconclusive. Alternative methods should be thought off like inactivate enzymes before mixing them together.
				</P><UL><LI>Stock of electro competent cells: Great Colony Rate on Petri dishes</LI><LI>Cloning training with RBS-CinI (3A Assembly Method):</LI><UL><LI>Digestion of the 3 biobricks :</LI><UL><LI>RBS (E-S)</LI><LI>CinI (X-P)</LI><LI>pSB1AC3 (E-P)</LI></UL></UL><LI>Ligation :</LI><P>
						Digestion results were mixed all together and then enzymes were heat-inactived at 80°C. Growth of red and white colonies. PCR checkings were performed on 3 white colonies. The results demonstrate that the insert is shorter than expected.
					</P><LI>Cloning of every first steps of our construction (3A Assembly Method) :</LI><UL><LI>10 clonings :</LI><P>
							Same process as above, growth of red and white colonies for 6 out of 10 ligation results. PCR checks were performed as above for the 6 valid ligations. Only 2 appeared to have the right size. The sequencing of the 2 valid constructions should corroborate our results.
						</P></UL><LI>Sequencing order of the two valid constructions :</LI><UL><LI>pMerT-GFP</LI><LI>RBS-CinR</LI></UL><P>We ordered on GATC company.</P></UL></DIV></DIV><DIV class="blocbackground"><H1 id="week5">June 29<SUP>th</SUP> to July 6<SUP>th</SUP></H1></DIV><DIV class="blocbackground"><H2>Modelling</H2><DIV class="noindent"><P>
					Working on parameters.
				</P></DIV><UL><LI>Models:</LI><P>
					Our deterministic model is now finished. The whole system is properly simulated and we simplificated it. We will now work on the stochastic approach. Stochastic is very important for our system as it will define the precision of the whole measure.
				</P><LI>Parameters:</LI><P>
					This is now the main part of our work. We have to look for the best set of parameters for our system to properly modelize it. We dissect litterature as well as other and former teams results to get a whole set. For the time being we do not have proper set.
				</P><P>
					The parameters we obtained are often contradictory from one set to another. We will try to characterize some of the parameters ourselves. We are working with biologists on experiments to characterize those parameters.
				</P></UL></DIV><DIV class="blocbackground"><H2>Biology</H2><DIV class="noindent"><P>
					Thanks to the delivery of new MerR material, we will finally be able to include MerR to our System. Because former clonings gave no results, we carried out the Standard Assembly but results are not significant.We are having to many problem with cloning, we have to check every steps (Minipreps concentration, digestion results by PCR , purification of the insert, proportion of vector (X1)/insert(X3) during the ligation...) 
				</P><UL><LI>New cloning trial CinI-RBS (Standard Assembly)</LI><UL><LI>2 different assemblies were achieved</LI><LI>Digestion of the 2 biobricks :</LI><UL><LI>RBS (S-P), the plasmid of RBS remains.</LI><LI>CinI (X-P) is inserted into the plasmid of RBS.</LI></UL><LI>Digestion of the 2 biobricks :</LI><UL><LI>CinI (S-P), the plasmid of CinI remains.</LI><LI>RBS (X-P) is inserted into the plasmid of CinI.</LI></UL><LI>Ligation :</LI><P>
							First digestion results were individually heat-inactivated at 80°C to eliminate enzymes remaining and then mixed all together.
						</P><LI>Spreading over Petri dish :</LI><P>
							Colonies grown only on the Petri dish containing the construction of CinI inserted into RBS plasmid. PCRs were performed on 4 colonies from this dish. None of them gave a conclusive result. All inserts are much shorter than expected (400 bps instead of 1040 bps)
						</P></UL><LI>pTet/TetR Miniprep : Petri dishes full of colonies.</LI><LI>MerR receipt: MerR was delivered into a small flask containing bacteria already transformed with MerR.</LI><LI>MerR culture and PCR checking : The insert amplified by PCR had the expected length.</LI><LI>Cloning of every first steps of our construction : (Standard Assembly Method)</LI><UL><LI>8 clonings</LI><P>
							We used the same process as above inserting the biggest Biobrick into the plasmid of the shortest. Thus, the risk to loose the shortest piece is avoided.
						</P></UL><LI>Analysis of Sequencing Results :</LI><UL><LI>pMerT-GFP</LI><LI>RBS-CinR</LI></UL><P>
						Sequences from GATC and Computer Sequencing were compared. No match between both sequences from GATC and Computer Sequencing for pMerT-GFP construction. Whereas sequences comparison of RBS-CinR demonstrates a strong similitude in the CinR region. But, RBS region seems to be absent or to have received mutations.
					</P></UL></DIV></DIV></DIV></DIV><DIV class="body"><DIV class="left"><DIV class="blocbackground"><TABLE class="nobordure"><TBODY><TR><TD><A href="https://2011.igem.org/Main_Page" title="iGEM 2011 Main Page">iGEM 2011 Main Page</A><H3><SPAN class="vert">Contact</SPAN> Us:</H3><H3><A title="Geoffrey BOUCHAGE" href="mailto:geoffrey.bouchage@phelma.grenoble-inp.fr"> Clic here !</A></H3></TD></TR></TBODY></TABLE></DIV></DIV></DIV><DIV class="printfooter">
Retrieved from &quot;<A href="http://2011.igem.org/Team:Grenoble/Notebook/June">http://2011.igem.org/Team:Grenoble/Notebook/June</A>&quot;</DIV></DIV></DIV><DIV id="footer-box" class="noprint"><DIV id="footer"><UL id="f-list"><LI id="t-recentchanges"><A href="/Special:RecentChanges" title="Recent changes">Recent changes</A></LI><LI id="t-whatlinkshere"><A href="/Special:WhatLinksHere/Team:Grenoble/Notebook/June" title="List of all wiki pages that link here [j]" accesskey="j">What links here</A></LI><LI id="t-recentchangeslinked"><A href="/Special:RecentChangesLinked/Team:Grenoble/Notebook/June" title="Recent changes in pages linked from this page [k]" accesskey="k">Related changes</A></LI><LI id="t-specialpages"><A href="/Special:SpecialPages" title="List of all special pages [q]" accesskey="q">Special pages</A></LI><LI><A href="/Special:Preferences">My preferences</A></LI></UL></DIV><DIV id="footer"><UL id="f-list"><LI id="t-print"><A href="/wiki/index.php?title=Team:Grenoble/Notebook/June&amp;printable=yes" title="Printable version of this page [p]" accesskey="p">Printable version</A></LI><LI id="t-permalink"><A href="/wiki/index.php?title=Team:Grenoble/Notebook/June&amp;oldid=257655" title="Permanent link to this revision of the page">Permanent link</A></LI><LI id="privacy"><A href="/2011.igem.org:Privacy_policy" title="2011.igem.org:Privacy policy">Privacy policy</A></LI><LI id="disclaimer"><A href="/2011.igem.org:General_disclaimer" title="2011.igem.org:General disclaimer">Disclaimers</A></LI></UL></DIV></DIV></DIV></BODY></HTML>